# Genetic Variation in ABCB1, ADRB1, CYP3A4, CYP3A5, NEDD4L and NR3C2 Confers Differential Susceptibility to Resistant Hypertension among South Africans

## Metadata
**Authors:** Jonathan N Katsukunya, Erika Jones, Nyarai D Soko, Dirk Blom, Phumla Sinxadi, Brian Rayner, Collet Dandara
**Journal:** Journal of Personalized Medicine
**Date:** 2024 Jun 21
**DOI:** [10.3390/jpm14070664](https://doi.org/10.3390/jpm14070664)
**PMID:** 39063918
**PMCID:** PMC11277774
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11277774/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC11277774/pdf/jpm-14-00664.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC11277774/pdf/jpm-14-00664.pdf)

## Abstract

Resistant hypertension (RHTN) prevalence ranges from 4 to 19% in Africa. There is a paucity of data on the role of genetic variation on RHTN among Africans. We set out to investigate the role of polymorphisms in ABCB1, ADRB1, CYP3A4, CYP3A5, NEDD4L, and NR3C2, on RHTN susceptibility among South Africans. Using a retrospective matched case–control study, 190 RHTN patients (cases: blood pressure (BP) ≥ 140/90 mmHg on ≥3 anti-hypertensives or BP < 140/90 mmHg on >3 anti-hypertensives) and 189 non-RHTN patients (controls: <3 anti-hypertensives, BP < 140/90 or ≥140/90 mmHg), 12 single nucleotide polymorphisms were genotyped using polymerase chain reaction–restriction fragment length polymorphism (PCR-RFLP), quantitative PCR and Sanger sequencing. Genetic association analyses were conducted using the additive model and multivariable logistic regression. Homozygosity for CYP3A5 rs776746C/C genotype (p = 0.02; OR: 0.44; CI: 0.22–0.89) was associated with reduced risk for RHTN. Homozygous ADRB1 rs1801252G/G (p = 0.02; OR: 3.30; CI: 1.17–10.03) and NEDD4L rs4149601A/A genotypes (p = 0.001; OR: 3.82; CI: 1.67–9.07) were associated with increased risk for RHTN. Carriers of the of ADRB1 rs1801252—rs1801253 G–C haplotype had 2.83-fold odds of presenting with RHTN (p = 0.04; OR: 2.83; CI: 1.05–8.20). These variants that are associated with RHTN may have clinical utility in the selection of antihypertensive drugs in our population.

Keywords: pharmacogenomics, hypertension, resistant hypertension, Africans, antihypertensive drugs

## 1. Introduction

More than 1 billion people globally are living with hypertension [[1](#B1-jpm-14-00664)] and between 10 and 20% progress to resistant hypertension (RHTN) [[2](#B2-jpm-14-00664)]. RHTN affects between 4 and 19% and 13% of people with hypertension in Africa and South Africa, respectively [[3](#B3-jpm-14-00664),[4](#B4-jpm-14-00664)]. RHTN is characterized by poor response to antihypertensive medication and is a relevant outcome of interest in pharmacogenomic research. RHTN is defined as either a blood pressure (BP) that remains above target (≥140/90 mmHg) while taking more than three antihypertensive drugs including a diuretic or a controlled BP (<140/90 mmHg) while taking four or more antihypertensive drugs in adherent patients [[5](#B5-jpm-14-00664)].

Genetic factors have been implicated in the development of RHTN [[6](#B6-jpm-14-00664)], accounting for up to 50% [[7](#B7-jpm-14-00664)] of poor BP treatment responses [[8](#B8-jpm-14-00664),[9](#B9-jpm-14-00664)]. Genetic variation may play a significant role in the way patients respond to six classes of commonly prescribed antihypertensive drugs: calcium channel blockers (CCBs), diuretics, angiotensin-converting enzyme inhibitors, angiotensin receptor blockers, mineralocorticoid receptor antagonists, and β-blockers [[10](#B10-jpm-14-00664),[11](#B11-jpm-14-00664)].

In South Africa, hydrochlorothiazide (HCTZ), amlodipine, enalapril, losartan, atenolol, and spironolactone are examples of commonly used antihypertensive drugs [[12](#B12-jpm-14-00664)]. Studies have shown that single nucleotide polymorphisms (SNPs) in *ABCB1*, *ADRB1*, *CYP3A4*, *CYP3A5*, *NEDD4L,* and *NR3C2* genes influence treatment outcomes. ABCB1 (or P-glycoprotein) is involved in the clearance of amlodipine and SNPs in the *ABCB1* gene, c.3435T>C (rs1045642, p.Ile1145=) and c.2677C>A (rs2032582, p.Ser893Ala) have been shown to influence clearance rates of amlodipine in Asian populations [[13](#B13-jpm-14-00664)]. CYP3A4 and CYP3A5 enzymes are also involved in the clearance of amlodipine, and SNPs have been found to affect both the pharmacokinetics of amlodipine and BP response in Asian and African American populations [[9](#B9-jpm-14-00664),[13](#B13-jpm-14-00664)]. Among Europeans, for the *CYP3A5* c.219-237T>C (rs774756 or *3) SNP, the T allele carriers were found to have an increased likelihood of switching from dihydropyridine CCBs to other antihypertensive medications [[14](#B14-jpm-14-00664)]. ADRB1 (or β1-adrenergic receptor) is the target site for β-blockers. SNPs have been reported to affect the sensitivity of the receptors to β-blockers thereby influencing their efficacy. For example, Guerra et al. (2022), reported on the association of the *ADRB1* c.145A>G (rs1801252, p.Ser49Gly) and c.1165G>C (rs1801253, p.Gly389Arg) polymorphisms with differential response in heart failure patients taking β-blockers [[15](#B15-jpm-14-00664)]. NEDD4L is the principal regulator of the activity of sodium ion channels in the kidney [[16](#B16-jpm-14-00664)] and SNPs have been shown to affect response to HCTZ. For example, *NEDD4L* c.49-16229G>A (rs4149601) and c.-300G>C (rs292449) SNPs were associated with BP response or adverse cardiovascular outcomes in European patients on treatment with HCTZ [[17](#B17-jpm-14-00664),[18](#B18-jpm-14-00664)]. The mineralocorticoid receptors (or NR3C2) are target sites for spironolactone and SNPs have been reported to directly affect spironolactone response and enalapril response indirectly. For example, *NR3C2* c.-2-358C>G (rs2070950) and c.538G>A (rs5522, p.Val180Ile) have been found to be associated with BP and heart failure responses in European [[19](#B19-jpm-14-00664)] and Egyptian patients [[20](#B20-jpm-14-00664)].

Little is known about the impact of these polymorphisms on Africans. Although important strides have been made in hypertension pharmacogenomics, such as the development of the International Consortium for Antihypertensive Pharmacogenomics Studies (ICAPS), findings from pharmacogenomic studies of hypertension still generate conflicting results across populations. In addition, according to the Pharmacogenomics Knowledgebase (PharmGKB^®^, Stanford, CA, USA) [[13](#B13-jpm-14-00664)] and ICAPS, there is a lack of data on which genetic variants influence how African patients respond to antihypertensive treatment. Consequently, no antihypertensives have been annotated in the Clinical Pharmacogenetics Implementation Consortium (CPIC^®^, Stanford, CA, USA) pharmacogenetic guidelines or incorporated into dosing algorithms. Therefore, this highlights the need for harmonization across populations and extensive genomic characterization of African populations. Aggressive forms of hypertension, such as RHTN, are more common in people of African origin (i.e., Black African or Mixed Ancestry) [[21](#B21-jpm-14-00664)], and pharmacogenes may play important roles in its development. In this study, we set out to investigate the role of genetic variation in six pharmacogenes, *ABCB1*, *ADRB1*, *CYP3A4*, *CYP3A5*, *NEDD4L*, and *NR3C2* in RHTN in South African patients.

## 2. Materials and Methods

### 2.1. Ethics Statement and Study Approval

This study is a sub-study to a larger project titled, “Pharmacogenomics of Cardiovascular Diseases: Focusing on Dyslipidemia and Hypertension (PRECODE)”, whose protocol was approved by the University of Cape Town Human Research Ethics Committee (UCT-HREC) with ethics clearance, “HREC 694/2020”, and was granted permission to recruit consenting participants for the study from the Hypertension Clinic, Groote Schuur Hospital, Cape Town, South Africa. This sub-study also has its own ethical approval which was granted by the UCT-HREC with ethics clearance, “HREC 141/2022”. The rights to privacy were respected and all medical information of the participants was kept confidential during the study.

### 2.2. Study Design and Setting

This was a retrospective matched case–control study. Patients attending the Hypertension Clinic at Groote Schuur Hospital, Cape Town, South Africa, from January 2006 to December 2016 and August 2021 to July 2022 who consented to DNA studies and access to their medical records were recruited into the study. Patients were matched 1:1 according to age, sex, and ethnicity utilizing a case–control design. All patients had hypertension and cases or controls were differentiated based on either having resistant hypertension (RHTN) or no RHTN. Cases were patients having RHTN, defined as (i) BP ≥ 140/90 mmHg on 3 or more antihypertensive drugs including a diuretic in optimal doses, or (ii) BP < 140/90 mmHg on 4 or more antihypertensive drugs. Adherence to antihypertensive therapy was confirmed by therapeutic drug monitoring (TDM) of amlodipine levels where possible (i.e., plasma amlodipine levels > 7.0 ng/mL) [[22](#B22-jpm-14-00664)]. Controls were patients without RHTN, defined as BP < 140/90 mmHg or BP ≥ 140/90 mmHg on less than 3 antihypertensive drugs. The study included participants (i) with a confirmed diagnosis of primary or essential hypertension, (ii) of Black African and Mixed Ancestry (i.e., admixed population of South Africans resulting from intermarriages between Black African, European, San or Asian populations) descent, (iii) on at least one antihypertensive drug for at least a year prior, and (iv) >18 years at the time of recruitment. The study excluded participants (i) with a confirmed diagnosis of secondary hypertension, (ii) who were pregnant at the time of recruitment, (iii) with confirmed white coat hypertension or non-adherence, and (iv) not on treatment with antihypertensive drugs.

### 2.3. Sample Size Calculation

The study sample size was determined according to Naing et al. (2006) [[23](#B23-jpm-14-00664)], which assumed the prevalence of RHTN to be 12.6% [[4](#B4-jpm-14-00664)] and an alpha level of 0.05. According to the calculation, a sample size of at least 163 patients with RHTN would be required to provide at least 80% power. However, due to the study’s case–control nature, 163 more patients without RHTN would be required to serve as controls. This results in a total sample size of 326 patients. The Hypertension Clinic at Groote Schuur Hospital, South Africa, serves patients referred for specialist hypertension care and most have RHTN. Therefore, since there is no recent study estimating the true prevalence of RHTN in our setting, we acknowledge that a prevalence of 12.6% used in our calculation may be lower. As such, to account for this, our analysis included 379 patients (190 cases and 189 controls).

### 2.4. Genetic Characterization

Genomic DNA was extracted from 2 mL of venous blood from consenting participants using the Chemagic 360^®^ automated nucleic acid system (Chemagen Technologies, GMBH/Perkin Elmer Incorporated, Shelton, CT, USA) according to the manufacturer’s protocol. The quality of the extracted genomic DNA was assessed by electrophoresis using a 1% (*w*/*v*) agarose gel and the quantity of DNA was determined using spectrophotometry on a NanoDrop C1000 UV spectrophotometer (Thermofischer Scientific Corporation, Waltham, MA, USA). Genomic DNA of adequate quality and quantity was used for genotyping the 12 SNPs in six genes using polymerase chain reaction–restriction fragment length polymorphism (PCR-RFLP), TaqMan^®^ allelic discrimination assays, and Sanger sequencing.

Forward and reverse primers flanking regions of interest containing the SNPs were either designed or obtained from previous studies ([Table S1](#app1-jpm-14-00664)). Each PCR was carried out in a 25 µL reaction volume containing 100 ng genomic DNA; 1X Green GoTaq Flexi^®^ Reaction Buffer (Promega Corporation, Madison, WI, USA); 0.4 mM of deoxynucleotide triphosphates (dNTPs) (Promega Corporation, Madison, WI, USA); 1.5 mM magnesium chloride (Promega Corporation, Madison, WI, USA); 0.2 mM of forward and reverse primers each (Inqaba Biotechnical industries (Pty) Ltd., Pretoria, South Africa), 1 U of GoTaq Flexi^®^ DNA Polymerase (Promega Corporation, Madison, WI, USA) and the volume made up to 25 uL with nuclease-free water. All PCRs were carried out using a SimpliAmp^™^ Thermal Cycler (Applied Biosystems, ThermoFisher Scientific, Life Technologies, Singapore). The reaction conditions were initial denaturation at 94 °C for 3 min, followed by 35 cycles of further denaturation at 94 °C for 30 s, annealing at temperatures specific for each SNP for 30 s ([Table S1](#app1-jpm-14-00664)), initial extension at 72 °C for 30 s, and final extension at 72 °C for 10 min.

PCR products for *ABCB1* rs1045642, rs2032582, *ADRB1* rs1801252, rs1801253, *CYP3A5* rs776746, rs10264272, and rs41303343 SNPs were digested using *MboI*, *BSeYI*, *Sau96I*, *BstNI*, *SspI* and *DdeI* (for both rs10264272 and rs41303343) restriction enzymes, respectively (New England BioLabs^®^, Ipswich, UK). Each digest reaction was carried out in a 30 µL reaction volume containing, 10 µL of PCR product, 1X CutSmart™ Buffer (New England BioLabs^®^, Ipswich, UK), 3U restriction enzyme specific for each SNP, and nuclease-free water. All restriction enzyme digest reactions were incubated and inactivated on a SimpliAmp^™^ Thermal Cycler (Applied Biosystems, ThermoFisher Scientific, Life Technologies, Singapore) at temperatures and periods specific for each enzyme according to the manufacturer’s protocol ([https://nebcloner.neb.com/#!/redigest](https://nebcloner.neb.com/#!/redigest) (accessed on 16 May 2024)). Restriction enzyme digests were resolved using electrophoresis on a 3% (*w*/*v*) agarose gel stained with SafeView^TM^ Classic (Applied Biological Materials Inc., Richmond, BC, Canada) nucleic acid stain. Visualization was conducted on a UV transilluminator (UVITEC, Avebury House, Cambridge, UK) using the Fire Reader^®^ software for Windows (D-56-26.MX, France).

Validated TaqMan^®^ allelic discrimination genotyping assay kits, designed and synthesized by ThermoFisher Scientific Corporation (Waltham, MA, USA), were used for genotyping the *CYP3A4* rs2740574 (C___1837671_50), *NR3C2* rs5522 (C__12007869_20), rs2070950 (C___1594391_1_), *NEDD4L* rs4149601 (C___1424558_10) and rs292449 (C___1132327_10) SNPs. The reactions were set up in a total reaction volume of 10 µL containing 5 µL of 2X TaqPath^™^ ProAmp^™^ Master Mix (ThermoFisher Scientific Corporation Waltham, MA, USA), 0.5 µL of 20X TaqMan^®^ genotyping assay specific for each SNP and 45 ng of genomic DNA. All qPCR reactions were carried out on a CFX96 Touch Real-Time PCR Thermal Cycler (Bio-Rad Laboratories, Inc., CA, USA) and the reaction conditions were initial denaturation at 95 °C for 5 min followed by 50 cycles of further denaturation and annealing at 95 °C for 15 s and 60 °C for 1 min, respectively, then final elongation at 60 °C for 30 s.

Sanger sequencing was used to validate genotyping for the 12 SNPs. For each SNP, 10% of samples were selected making sure that all the representative genotypes were included. This approach ensured that the primary genotyping methods could accurately resolve each genotype group and provide robust validation across different genotypes without the need to sequence all the samples. Post-PCR clean-up using exonuclease I (ExoI) and thermosensitive alkaline phosphatase (FastAP^™^) enzymes from ThermoFisher Scientific (Waltham, MA, USA) was completed as per manufacturer’s protocol prior to sequencing. Each sequencing reaction was set up in a reaction volume of 10 µL, containing PCR product, BigDye™ Terminator v3.1 sequencing buffer, BigDye™ Terminator v3.1 Ready Reaction Mix (ThermoFisher Scientific, Waltham, MA, USA), and sequencing primer ([Table S1](#app1-jpm-14-00664)). Sequencing reactions were carried out on a SimpliAmp™ Thermal Cycler (Applied Biosystems, Singapore) and the reaction conditions were initial denaturation at 96 °C for 1 min followed by 25 cycles of further denaturation at 96 °C for 10 s, annealing at 50 °C for 5 s and elongation at 60 °C for 4 min. Post-sequencing clean-up was performed according to the manufacturer’s protocol using the EDTA/ethanol precipitation method (ThermoFisher Scientific, Waltham, MA, USA). All sequencing reactions were resolved using capillary electrophoresis on a SeqStudio^®^ Genetic Analyzer (ThermoFisher Scientific, Waltham, MA, USA) and analyzed using DNAStar^®^-SeqMan Pro Sequence Assembly software (Lasergene^®^, version 17.4, Madison, WI, USA).

### 2.5. Statistical Analysis

Statistical analyses were performed using R statistical software (version 4.2.1, 23 June 2022, Vienna, Austria). Genotype, allele, and haplotype frequencies were computed using SHEsis online software [[24](#B24-jpm-14-00664)]. Categorical data are presented as *N* (%) [where *N* = the number of individuals in that category and % = the frequency of individuals within that category]. Continuous data are presented as mean ± standard deviation (SD) or median ± interquartile range (IQR) [25th–75th percentile] depending on the distribution of the data. The Shapiro–Wilk test was used to assess for any deviation from normality and the Hardy–Weinberg equilibrium (HWE) was determined for each SNP for Black African and Mixed Ancestry population groups using the chi-square test with one degree of freedom. Continuous data were compared between cases and controls either using the T-test or the Mann–Whitney U-test (Wilcoxon rank-sum test) for normally and non-normally distributed data, respectively. Categorical data were compared between cases and controls using Pearson’s chi-square test or Fisher’s exact test. The main outcome of interest was RHTN. The additive model of inheritance was used to estimate associations with RHTN. Multivariable logistic regression including genetic (genotypes and haplotypes) and non-genetic variables (clinical or demographic) was used to adjust for any confounding effects. Odds ratios (ORs) described the strength of associations, and 95% confidence intervals (CI) were used as precision estimates. For all analyses, the level of statistical significance was set at *p* < 0.05.

## 3. Results

### 3.1. Demographic and Clinical Characteristics

A total of 379 participants with hypertension (29% Black African and 71% Mixed Ancestry) were successfully recruited. In 2% of the participants, adherence to antihypertensive therapy was confirmed using therapeutic drug monitoring of amlodipine levels. [Table 1](#jpm-14-00664-t001) compares the distribution of participant characteristics between cases and controls. The participants comprised 190 cases (with RHTN) and 189 hypertensive controls (but with no RHTN). In each group, 54% were female and the median age for cases and controls was 43.8 (20–54.2) and 42.2 (26–52.6) years, respectively (*p* = 0.05). There were statistically significant differences in the presence of comorbidities and hypertension-mediated organ damage of cases and controls. Left ventricular hypertrophy (LVH) (*p* < 0.001), chronic kidney disease (CKD) (*p* < 0.001), diabetes mellitus (*p* = 0.03), and dyslipidemia (*p* = 0.03) were more prevalent in the cases compared to controls, and aldosterone levels were significantly higher (*p* = 0.002). Use of statin or lipid-lowering and antihypertensive drugs (amlodipine, hydrochlorothiazide, atenolol, enalapril, spironolactone, and losartan) was significantly higher in cases than in controls (*p* < 0.001).

### Table 1.

| Variable(s) | Cases (N = 190) | Controls (N = 189) | p-Value |
| --- | --- | --- | --- |
| Age years | 43.8 (20–54.2) | 42.2 (26–52.6) | 0.05 |
| Female sex | 103 (54.2%) | 102 (54.0%) | 0.99 |
| Ethnicity |   |   |   |
| Black African | 58 (30.5%) | 52 (27.5%) | 0.57 |
| Mixed ancestry | 132 (69.5%) | 137 (72.5%) |   |
| Alcohol consumption (frequent/occasional) | 45 (23.7%) | 47 (24.9%) | 0.90 |
| Smoking (active/past) | 63 (33.2%) | 64 (33.9%) | 0.99 |
| Positive familyHistory of hypertension | 123 (64.7%) | 121 (64.0%) | 0.91 |
| Blood pressure (mmHg) |   |   |   |
| Uncontrolled (≥140/90) | 149 (78.4%) | 68 (36.0%) | <0.001 |
| Controlled (<140/90) | 41 (21.6%) | 121 (64.0%) |   |
| Left ventricular hypertrophy a |   |   |   |
| Yes | 90 (47.4%) | 52 (27.5%) | <0.001 |
| No/unknown | 100 (52.6%) | 189 (72.5%) |   |
| Aldosterone(pmol/L) | 276 (31–435) | 200 (2.3–313) | 0.002 |
| Comorbidities |   |   |   |
| Chronic kidney disease (CKD) b | 36 (19.1%) | 10 (5.3%) | <0.001 |
| Diabetes mellitus (DM) | 33 (17.6%) | 18 (9.6%) | 0.03 |
| Dyslipidemia | 30 (15.8%) | 15 (7.9%) | 0.03 |
| Ischemic heart disease (IHD) | 17 (8.9%) | 9 (4.8%) | 0.11 |
| Previous stroke | 13 (6.8%) | 5 (2.6%) | 0.09 |
| Concomitant drugs |   |   |   |
| Statin or lipid-lowering therapy | 76 (40.0%) | 45 (23.8%) | <0.001 |
| Diabetic treatment c | 23 (12.1%) | 15 (7.9%) | 0.08 |
| Analgesics | 12 (6.0%) | 10 (5.2%) | 0.69 |
| Antihypertensives drugs |   |   |   |
| Amlodipine | 172 (90.5%) | 116 (61.3%) | <0.001 |
| Hydrochlorothiazide | 148 (77.9%) | 108 (57.1%) | <0.001 |
| Atenolol | 123 (64.7%) | 27 (14.2%) | <0.001 |
| Enalapril | 150 (78.9%) | 88 (46.6%) | <0.001 |
| Spironolactone | 52 (27.3%) | 7 (3.7%) | <0.001 |
| Losartan | 39 (20.5%) | 14 (7.4%) | <0.001 |

Table 1 Caption: Demographic and clinical characteristics of study participants.

### 3.2. Genotype or Allele Frequency Distributions and Associations with Resistant Hypertension

Genotype and allele frequency distributions are shown in [Table 2](#jpm-14-00664-t002) and [Table S2](#app1-jpm-14-00664), respectively. All genotypes were in Hardy–Weinberg equilibrium (HWE) except for *ABCB1* rs2032582 and *CYP3A4* rs2740574 (*p* < 0.05) polymorphisms. We report for the first time, frequencies of *ADRB1* rs1801252 and rs1801253 polymorphisms in an African population. Allele frequencies for the SNPs under study are comparable to other African populations and variable when compared to non-African populations such as Asians and Europeans [[25](#B25-jpm-14-00664)].

### Table 2.

| Single Nucleotide Polymorphism | Genotype | Genotype Frequencies, N (freq) |  |  | p-Value | OR [95% CI] |
| --- | --- | --- | --- | --- | --- | --- |
|   | Combined | Cases | Controls |  |  |  |
| ABCB1 rs1045642 (c.3435C>T) | C/C | 191 (0.50) | 99 (0.52) | 92 (0.49) | 1 |   |
| C/T | 142 (0.37) | 70 (0.37) | 72 (0.38) | 0.66 | 0.90 [0.57–1.42] |  |
| T/T | 46 (0.12) | 21 (0.11) | 25 (0.13) | 0.51 | 0.78 [0.39–1.57] |  |
| ABCB1 rs2032582 (c.2677C>A) | C/C | 219 (0.59) | 112 (0.60) | 107 (0.58) | 1 |   |
| A/C | 109 (0.29) | 54 (0.29) | 55 (0.30) | 0.82 | 0.93 [0.58–1.52] |  |
| A/A | 46 (0.13) | 22 (0.11) | 24 (0.13) | 0.74 | 0.88 [0.44–1.74] |  |
| ADRB1 rs1801252 (c.145A>G) | A/A | 190 (0.51) | 92 (0.48) | 107 (0.57) | 1 |   |
| A/G | 149 (0.40) | 79 (0.42) | 70 (0.37) | 0.23 | 1.31 [0.84–2.05] |  |
| G/G | 31 (0.08) | 19 (0.10) | 12 (0.06) | 0.13 | 1.83 [0.80–4.39] |  |
| ADRB1 rs1801253 (c.1165G>C) | C/C | 195 (0.51) | 100 (0.53) | 95 (0.50) | 1 |   |
| C/G | 153 (0.40) | 74 (0.39) | 79 (0.42) | 0.67 | 0.89 [0.57–1.39] |  |
| G/G | 31 (0.08) | 16 (0.08) | 15 (0.08) | 0.98 | 1.01 [0.44–2.34] |  |
| CYP3A4 rs2740574(c.-392C>T, CYP3A4*1B) | T/T | 136 (0.36) | 59 (0.32) | 77 (0.41) | 1 |   |
| C/T | 132 (0.35) | 74 (0.40) | 58 (0.31) | 0.05 | 1.30 [0.78–2.17] |  |
| C/C | 105 (0.28) | 52 (0.28) | 53 (0.28) | 0.36 | 0.78 [0.47–1.30] |  |
| CYP3A5rs776746(c.219-237T>C, CYP3A5*3) | T/T | 144 (0.38) | 78 (0.41) | 66 (0.35) | 1 |   |
| C/T | 146 (0.39) | 77 (0.41) | 69 (0.37) | 0.82 | 0.94 [0.58–1.54] |  |
| C/C | 89 (0.23) | 35 (0.18) | 54 (0.29) | 0.03 | 0.54 [0.31–0.97] |  |
| CYP3A5 rs10264272(c.624C>T, CYP3A5*6) | C/C | 285 (0.75) | 141 (0.74) | 144 (0.76) | 1 |   |
| C/T | 86 (0.22) | 45 (0.24) | 41 (0.22) | 0.71 | 1.12 [0.67–1.87] |  |
| T/T | 8 (0.10) | 4 (0.02) | 4 (0.02) | 0.99 | 1.02 [0.18–5.60] |  |
| CYP3A5rs41303343(insT, CYP3A5*7) | A/A | 322 (0.85) | 158 (0.83) | 164 (0.87) | 1 |   |
| A/T | 52 (0.14) | 30 (0.16) | 22 (0.12) | 0.29 | 1.41 [0.75–2.69] |  |
| T/T | 3 (0.01) | 1 (0.01) | 2 (0.01) | 0.99 | 0.51 [0.01–10.08] |  |
| NEDD4Lrs4149601(c.49-16229G>A) | G/G | 134 (0.36) | 59 (0.31) | 75 (0.40) | 1 |   |
| A/G | 179 (0.47) | 95 (0.50) | 84 (0.45) | 0.14 | 1.43 [0.89–2.31] |  |
| A/A | 64 (0.17) | 36 (0.19) | 28 (0.15) | 0.13 | 1.63 [0.86–3.11] |  |
| NEDD4L rs292449(c.-300G>C) | G/G | 100 (0.26) | 52 (0.28) | 48 (0.25) | 1 |   |
| C/G | 190 (0.50) | 89 (0.47) | 101 (0.53) | 0.25 | 0.73 [0.42–1.26] |  |
| C/C | 88 (0.23) | 48 (0.25) | 40 (0.21) | 0.77 | 0.90 [0.49–1.67] |  |
| NR3C2rs5522(c.538G>A) | T/T | 286 (0.76) | 145 (0.76) | 141 (0.75) | 1 |   |
| C/T | 90 (0.24) | 45 (0.24) | 45 (0.24) | 0.99 | 0.97 [0.59–1.61] |  |
| C/C | 1 (0.003) | 0 (0) | 1 (0.01) | 0.49 | 0 [0–38.2] |  |
| NR3C2 rs2070950(c.-2-358C>G) | G/G | 171 (0.45) | 88 (0.47) | 83 (0.44) | 1 |   |
| C/G | 145 (0.38) | 71 (0.38) | 74 (0.39) | 0.73 | 0.91 [0.57–1.44] |  |
| C/C | 62 (0.16) | 30 (0.16) | 32 (0.17) | 0.77 | 0.88 [0.47–1.65] |  |

Table 2 Caption: Genotype frequency distributions between cases and controls.

The *CYP3A5* rs776746C/C genotype (i.e., *CYP3A5*3/*3*) was statistically significantly more frequent in controls (29%) compared to cases (18%) (*p* = 0.03). Homozygous *CYP3A5* rs776746C/C genotype was significantly associated with reduced risk of RHTN (OR: 0.54; CI: 0.31–0.97) ([Table 2](#jpm-14-00664-t002)). There was a trend toward a significant difference (*p* = 0.06) in the distribution of *ADRB1* rs1801252G allele carriers between cases (31%) and controls (25%). Similarly, there was a trend toward a significant difference (*p* = 0.07) in the distribution of the *NEDD4L* rs4149601A allele between cases (44%) and controls (37%). Carriers of *NEDD4L* rs4149601A allele showed a trend of more prevalence among the RHTN group (OR: 1.31; CI: 0.98–1.75) ([Table S2](#app1-jpm-14-00664)).

### 3.3. Haplotype Frequency Distributions and Associations with Resistant Hypertension

The frequency distributions in cases or controls and associations with RHTN of haplotypes constructed from the studied SNPs, namely *ADRB1* rs1801252—rs1801253; *CYP3A5* rs776746—rs10264272—rs4130334 and *NEDD4L* rs4149601—rs292449 are shown in [Table 3](#jpm-14-00664-t003). Carriers of the *CYP3A5* rs776746—rs10264272—rs4130334 C–C–A haplotype were significantly (*p* = 0.03) more frequent in controls (46%) than cases (38%) and were significantly associated with reduced risk of RHTN (OR: 0.73; CI: 0.54–0.97). There was a trend towards significance (*p* = 0.06) in the distribution of the *ADRB1* rs1801252—rs1801253 G–C haplotype between cases (31%) and controls (25%) with an increased risk of RHTN (OR: 1.34; CI: 0.98–1.85). There was a trend towards significance (*p* = 0.07) in the distribution of the *NEDD4L* rs4149601—rs292449 A–C haplotype between cases (25%) and controls (19%) with an increased risk of RHTN (OR: 1.37; CI: 0.97–1.94).

### Table 3.

| SNP Combination | Haplotype | Cases, N (Freq) | Controls, N (Freq) | p-Value | OR [95% CI] |
| --- | --- | --- | --- | --- | --- |
| ADRB1rs1801252—rs1801253 | A-C | 157 (0.41) | 175 (0.46) | 0.17 | 0.82 [0.61–1.09] |
| A-G | 106 (0.28) | 109 (0.29) | 0.77 | 0.96 [0.69–1.31] |  |
| G-C | 117 (0.31) | 94 (0.25) | 0.06 | 1.34 [0.98–1.85] |  |
| CYP3A5rs776746—rs10264272—rs41303343 | C-C-A | 144 (0.38) | 172 (0.46) | 0.03 | 0.73 [0.54–0.97] |
| T-C-A | 150 (0.40) | 130 (0.35) | 0.14 | 1.25 [0.93–1.68] |  |
| T-C-T | 30 (0.08) | 25 (0.07) | 0.43 | 1.25 [0.72–2.16] |  |
| T-T-A | 50 (0.13) | 46 (0.12) | 0.65 | 1.10 [0.72–1.70] |  |
| NEDD4Lrs4149601—rs292449 | A-C | 93 (0.25) | 72 (0.19) | 0.07 | 1.37 [0.97–1.94] |
| A-G | 73 (0.19) | 68 (0.18) | 0.70 | 1.07 [0.75–1.55] |  |
| G-C | 100 (0.26) | 123 (0.33) | 0.05 | 0.73 [0.54–1.00] |  |
| G-G | 112 (0.30) | 111 (0.30) | 0.99 | 0.99 [0.73–1.37] |  |

Table 3 Caption: Haplotype frequency distributions between cases and controls.

### 3.4. Effect of Confounding Variables on Associations of Genetic Variables with Resistant Hypertension

Aldosterone levels, dyslipidemia, diabetes mellitus, and use of statins or lipid-lowering therapy were identified as predictor variables potentially confounding the main outcome–RHTN, at univariate analysis (*p* < 0.05) ([Table 1](#jpm-14-00664-t001)) and prior evidence from the literature as major risk factors [[26](#B26-jpm-14-00664),[27](#B27-jpm-14-00664),[28](#B28-jpm-14-00664)]. Associations of genotypes or haplotypes in *ADRB1*, *CYP3A5*, and *NEDD4L* polymorphisms (i.e., those with *p* < 0.05 and trended towards significance at univariate analysis) with RHTN after adjusting for aldosterone levels, dyslipidemia, diabetes mellitus and use of statins or lipid-lowering therapy are shown in [Table 4](#jpm-14-00664-t004). *ADRB1* rs1801252G/G genotype (*p* = 0.02; OR: 3.30; CI: 1.17–10.03), *ADRB1* rs1801252—rs1801253 G-C haplotype (*p* = 0.04; OR: 2.83; CI: 1.05–8.20) and *NEDD4L* rs4149601A/A genotype (*p* = 0.001; OR: 3.82 CI: 1.67–9.07), were significantly associated with increased risk of RHTN. The *NEDD4L* rs4149601—rs292449 A–C haplotype tended towards significant association with increased risk of RHTN (*p* = 0.08; OR: 3.14; CI: 0.88–12.9). The *CYP3A5* rs776746C/C genotype was significantly associated with reduced risk of RHTN (*p* = 0.02; OR: 0.44; CI: 0.22–0.89) while no significant association was observed for the *CYP3A5* rs776746—rs10264272—rs41303343 C–C–A haplotype (*p* = 0.59; OR: 0.86; CI: 0.49–1.50).

### Table 4.

| SNP/SNP Combination | Genotype/Haplotype | Coefficient | p-Value | OR [95% CI] a |
| --- | --- | --- | --- | --- |
| ADRB1 rs1801252 | GG | 1.19 | 0.02 | 3.30 [1.17–10.03] b |
| ADRB1rs1801252—rs1801253 | G-C | 1.04 | 0.04 | 2.83 [1.05–8.20] b |
| CYP3A5 rs776746 | CC | −0.80 | 0.02 | 0.44 [0.22–0.89] b |
| CYP3A5rs776746—rs1026427—rs41303343 | C-C-A | −0.14 | 0.59 | 0.86 [0.49–1.50] |
| NEDD4L rs4149601 | AA | 1.34 | 0.001 | 3.82 [1.67–9.07] b |
| NEDD4Lrs4149601—rs292449 | A-C | 1.14 | 0.08 | 3.14 [0.88–12.9] |

Table 4 Caption: Multivariable logistic regression analysis adjusting for confounding variables.

### 3.5. Association of Genetic Variables with Resistant Hypertension Related Adverse Outcomes: Chronic Kidney Disease (CKD) and Left Ventricular Hypertrophy (LVH)

Patients with RHTN have been shown to be at higher risk of adverse cardiovascular outcomes and target organ damage, such as CKD and LVH than patients without RHTN [[29](#B29-jpm-14-00664)]. We hypothesized that variants associated with RHTN may be associated with adverse outcomes: CKD and LVH. [Table 5](#jpm-14-00664-t005) shows associations of *ADRB1* rs1801252, *CYP3A5* rs776746, and *NEDD4L* rs4149601 SNPs with CKD and LVH in unadjusted and adjusted (adjusted for the effect of aldosterone levels, dyslipidemia, diabetes mellitus and use of statins or lipid-lowering therapy) logistic regression models. The *CYP3A5* rs776746C/C genotype was significantly associated with reduced risk of CKD (unadjusted: *p* = 0.03; OR: 0.37; CI: 0.13–0.90 and adjusted: *p* = 0.04; OR: 0.29; CI: 0.08–0.93). There were no significant associations observed with LVH.

### Table 5.

|   |  | Chronic Kidney Disease |  |  |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- |
|   |   | Unadjusted |  |  | Adjusted a |  |  |
| Variable | Genotype | Coefficient | p-Value | OR [95% CI] | Coefficient | p-Value | OR [95% CI] |
| ADRB1 rs1801252 | G/G | −0.11 | 0.87 | 0.90 [0.20–2.83] | −0.09 | 0.83 | 0.91 [0.39–2.08] |
| CYP3A5 rs776746 | C/C | −0.99 | 0.03 | 0.37 [0.13–0.90] | −1.22 | 0.04 | 0.29 [0.08–0.93] b |
| NEDD4L rs4149601 | A/A | 0.19 | 0.71 | 1.20 [0.46–3.54] | −0.25 | 0.72 | 0.78 [0.16–3.02] |
|   |   | Left Ventricular Hypertrophy |  |  |  |  |  |
| ADRB1 rs1801252 | G/G | 0.26 | 0.12 | 1.29 [0.83–2.00] | 0.55 | 0.25 | 1.13 [0.57–2.25] |
| CYP3A5 rs776746 | C/C | −0.08 | 0.76 | 0.92 [0.53–1.59] | 0.12 | 0.72 | 1.74 [0.67–4.45] |
| NEDD4L rs4149601 | A/A | −0.05 | 0.85 | 0.95 [0.50–1.74] | −0.17 | 0.69 | 0.85 [0.37–1.89] |

Table 5 Caption: Association with chronic kidney disease (CKD) and left ventricular hypertrophy (LVH).

### 3.6. Assessing Blood Pressure (BP) Variation according to Genotypes in ADRB1, CYP3A5, and NEDD4L Polymorphisms in Hypertension

Resistant hypertension is a phenotype defined by the number of medications and BP levels. As a secondary analysis, we also hypothesized that genetic variation in *ADRB1, CYP3A5*, and *NEDD4L* genes may explain BP variation in hypertension. [Figure 1](#jpm-14-00664-f001) and [Figure 2](#jpm-14-00664-f002) shows median systolic blood pressure (SBP) and diastolic blood pressure (DBP) variation across genotypes for *ADRB1* c.145A>G (rs1801252, p.Ser49Gly), *CYP3A5* c.219-237T>C (rs774756) and *NEDD4L* c.49-16229G>A (rs4149601) polymorphisms. Median SBP and DBP did not seem to vary significantly (*p* > 0.05) across genotype groups.

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4811/11277774/90e5c40c087f/jpm-14-00664-g001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=11277774_jpm-14-00664-g001.jpg)

Median systolic blood pressure (SBP) variation according to genotypes. There were no significant differences in median SBP across genotypes for ADRB1 c.145A>G (rs1801252, p.Ser49Gly), CYP3A5 c.219-237T>C (rs774756) and NEDD4L c.49-16229G>A (rs4149601) polymorphisms (ADRB1 rs1801252: A/A vs. A/G: p = 0.18; A/A vs. G/G: p = 0.91; CYP3A5 rs774756: T/T vs. CT: p = 0.46; T/T vs. CC: p = 0.49; and NEDD4L rs4149601: G/G vs. A/G: p = 0.52; G/G vs. A/A: p = 0.29).

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4811/11277774/26dc04b88e9e/jpm-14-00664-g002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=11277774_jpm-14-00664-g002.jpg)

Median diastolic blood pressure (DBP) variation according to genotypes. There were no significant differences in median DBP across genotypes for ADRB1 c.145A>G (rs1801252, p.Ser49Gly), CYP3A5 c.219-237T>C (rs774756) and NEDD4L c.49-16229G>A (rs4149601) polymorphisms (ADRB1 rs1801252: A/A vs. A/G: p = 0.77; A/A vs. G/G: p = 0.19; CYP3A5 rs774756: T/T vs. CT: p = 0.55; T/T vs. CC: p = 0.96; and NEDD4L rs4149601: G/G vs. A/G: p = 0.73; G/G vs. A/A: p = 0.46).

## 4. Discussion

This study set out to evaluate the role of genetic variation in *ABCB1*, *ADBR1*, *CYP3A4*, *CYP3A5*, *NEDD4L,* and *NR3C2* among South African hypertensive patients, comparing the frequency of genetic variants between RHTN and non-RHTN patients (controls). *CYP3A5* c.219-237T>C (rs774756 or *CYP3A5*3*) polymorphism was found to be associated with reduced risk of RHTN and protection against adverse outcomes such as CKD. *ADRB1* c.145A>G (rs1801252, p.Ser49Gly) and *NEDD4L* c.49-16229G>A (rs4149601) polymorphisms were associated with increased risk of RHTN. No significant association with RHTN was observed for polymorphisms in *ABCB1*, *CYP3A4,* and *NR3C2*.

According to Rodriguez-Antona et al. (2022), *CYP3A5* c.219-237T>C (rs774756) nucleotide change in intron 3 causes a disruption of the translational reading frame and skipping of exon 9, leading to a premature stop codon and production of a non-functional, truncated protein. At the biochemical level, the enzyme produced lacks key functional domains essential for its catalytic activity [[30](#B30-jpm-14-00664)]. Thus, *CYP3A5* rs776746C/C genotype carriers are non-expressors while T/T genotype carriers express higher levels of CYP3A5. CYP3A5 partially metabolizes and clears CCBs such as amlodipine; some ACE inhibitors, such as enalapril; and some mineralocorticoid receptor antagonists, such as spironolactone [[31](#B31-jpm-14-00664)]. Therefore, in non-expressors, exposure to the BP-lowering effect of CYP3A5 substrate drugs is increased as there is reduced metabolic clearance. As such, it is possible that *CYP3A5* rs776746C allele carriers may derive greater benefit from antihypertensive drugs that are CYP3A5 substrates, which also results in reduced occurrence of hypertension-related adverse outcomes such as CKD. Furthermore, carriage of the *CYP3A5* rs776746C allele has been shown to be associated with reduced clearance and increased exposure to drugs metabolized by CYP3A5 in several pharmacokinetic studies [[13](#B13-jpm-14-00664)] further supporting findings from this study.

CYP3A4 and CYP3A5 have broad overlapping substrate specificities and according to Soria-Chacartegui et al. (2023), CYP3A4 contributes mainly to the metabolic clearance of amlodipine [[32](#B32-jpm-14-00664)]. However, we observed no associations between the studied *CYP3A4* polymorphism and RHTN. To our knowledge, there have not been reports of the association of *CYP3A4* polymorphisms with RHTN, although *CYP3A4* rs2740574 polymorphism has been shown to be associated with BP response to amlodipine in African Americans [[33](#B33-jpm-14-00664)].

The *ADRB1* rs1801252G/G genotype and *ADRB1* rs1801252–1801253 G–C haplotype were associated with an increased likelihood of RHTN. Beta-blockers, such as atenolol, metoprolol, or bisoprolol, result in a lowering of BP through inhibition of the downstream effects of endogenous catecholamines such as epinephrine and norepinephrine. Beta-blockers achieve this by blocking beta-adrenergic receptors, which in turn reduces heart rate, cardiac output, and the release of renin from the kidneys [[34](#B34-jpm-14-00664)]. However, polymorphisms such as *ADRB1* rs1801252 and *ADRB1* rs1801253 can stunt this effect. For example, the *ADRB1* rs1801252G allele has been reported to increase agonist-promoted downregulation due to altered N-glycosylation, affecting receptor stability and function. Conversely, the *ADRB1* rs1801253C allele alters G-protein coupling, leading to decreased adenylyl cyclase activity. This results in a reduced conversion of ATP to cAMP, a critical messenger in the signal transduction pathway that mediates the effects of beta-adrenergic stimulation. Consequently, the generation of cAMP from ATP is diminished, leading to reduced physiological responses to beta-blockers [[35](#B35-jpm-14-00664)]. While there have been inconsistent findings across populations on associations of the *ADRB1* polymorphisms and response to beta-blockers, previous studies indicate that *ADRB1* rs1801252A allele carriers respond better to beta-blocker treatment compared to G allele carriers, and individuals carrying the G–C haplotype (*ADRB1* rs1801252–1801253) exhibit a poor response [[35](#B35-jpm-14-00664)]. This was consistent with our study’s outcomes, so it is possible that observed associations with RHTN are due to poor response to beta-blockers.

The carriers of the *NEDD4L* rs4149601A/A genotype were observed to have an increased risk of RHTN. Under normal physiological functions, NEDD4L serves as a principal regulator of sodium ion transporters in the kidney, which are targeted by antihypertensive medications such as hydrochlorothiazide (HCTZ) and amiloride. For example, NEDD4L is a specific ubiquitin ligase of the epithelial sodium channel (ENaC). NEDD4L has been reported to be composed of a C2-WW(x4)-HECT domain which facilitates binding to the PY motif of the ENaC leading to channel ubiquitination and endocytosis. This in turn suppresses ENaC function and reduces sodium reabsorption in the kidney, which lowers blood volume and blood pressure [[16](#B16-jpm-14-00664)]. The presence of the *NEDD4L* rs4149601A allele causes alternative splicing, resulting in a non-functional NEDD4L [[18](#B18-jpm-14-00664)]. The *NEDD4L* rs4149601A allele has been linked to adverse cardiovascular outcomes due to poor response in patients on HCTZ in several studies. For example, according to McDonough et al., (2016), carriers of the *NEDD4L* rs4149601A allele were found to not respond well to treatment with HCTZ, had less BP reduction, and were at increased risk of adverse CVD outcomes compared to G allele carriers. Therefore, these prior associations with poor response to HCTZ resulting in adverse outcomes due to *NEDD4L* rs4149601, are also consistent with findings from this study, although this study establishes a link with RHTN.

The genes selected for analysis in this study encode key enzymes involved in the pharmacokinetics and pharmacodynamics of antihypertensive drugs. We observed that selected polymorphisms in *ABCB1*, *CYP3A4,* and *NR3C2* genes were not significantly associated with RHTN, despite prior evidence of their association with antihypertensive drug response in other populations [[9](#B9-jpm-14-00664)]. We speculate that this lack of significance could be due to genetic heterogeneity that exists between population groups, meaning that the genetic makeup affecting drug response might vary significantly among different ethnicities. It is possible that the specific variants selected for this study may not be the most relevant ones for South Africans (i.e., Black Africans and Mixed Ancestry in this instance).

Our study has limitations. First, the accuracy of our findings was dependent on being able to rule out non-adherence to antihypertensive therapy. The adherence was self-reported, which is the least reliable method. Where applicable, amlodipine concentrations were measured when non-adherence was suspected. Second, all participants recruited for the study had hypertension including the controls. The absence of a normotensive control group could potentially affect the interpretation and generalizability of the study findings, as there is no reference point for comparison to assess the impact of hypertension on the variables under investigation. Third, we did not adjust for multiple testing. Fourth, RHTN is a polygenic disorder, and this study was a targeted and hypothesis-driven approach, focusing on a few selected genes or variants. It is possible that our approach could have fallen short in detecting other variants in genes with unknown biological links to hypertension, necessitating the need for extensive research.

In conclusion, RHTN is a phenotype resulting from poor antihypertensive drug response. It appears that *ADRB1* c.145A>G (rs1801252, p.Ser49Gly), *CYP3A5* c.219-237T>C (rs774756 or *CYP3A5*3*) and *NEDD4L* c.49-16229G>A (rs4149601) polymorphisms may be of pharmacogenomic importance in African populations. Our findings suggest that screening for these variants may help in predicting patients who are likely to have good or poor responses to antihypertensive drugs among Africans. For example, curating these variants in the development of rapid diagnostic tests (i.e., bedside tests) and dosing algorithms, may assist clinicians in making rapid decisions about the selection and dosage of antihypertensive therapy, thereby improving daily medical practice. In addition to aiding in diagnosis, variants may potentially aid in the development of efficacious alternative antihypertensive drugs other than the ones affected by these polymorphisms. However, further research is still needed. It is important that our findings be replicated in larger prospective cohort or pharmacokinetic studies to establish causality. Additionally, research in other African populations is necessary, as our study was limited to Black African and Mixed Ancestry individuals from South Africa. Furthermore, non-targeted approaches such as genome-wide association studies (GWAS) or whole-exome sequencing may aid in the possible discovery of African-specific variants associated with refractory response to antihypertensive therapy.

## Acknowledgments

The research reported in this publication was supported by the South African Medical Research Council (SAMRC) with funds received from the South African Department of Science and Innovation. Thank you to the study participants and all the members of the Pharmacogenomics and Drug Metabolism Research group.

## Supplementary Materials

The following supporting information can be downloaded at: [https://www.mdpi.com/article/10.3390/jpm14070664/s1](https://www.mdpi.com/article/10.3390/jpm14070664/s1), Table S1: Primer sequences and annealing temperatures; Table S2: Allele frequency distributions between cases and controls. References [[36](#B36-jpm-14-00664),[37](#B37-jpm-14-00664),[38](#B38-jpm-14-00664),[39](#B39-jpm-14-00664),[40](#B40-jpm-14-00664)] are cited in the Supplementary Materials.

## Author Contributions

Conceptualization, J.N.K., E.J., N.D.S., D.B., P.S., B.R. and C.D.; methodology, J.N.K., N.D.S. and C.D.; validation, E.J., N.D.S., D.B., P.S., B.R. and C.D.; formal analysis, J.N.K. and C.D.; investigation, J.N.K., E.J., N.D.S., D.B., P.S., B.R. and C.D.; resources, P.S. and C.D.; writing—original draft preparation, J.N.K. and C.D.; writing—review and editing, E.J., N.D.S., D.B., P.S., B.R. and C.D.; supervision, E.J., N.D.S., D.B., P.S., B.R. and C.D.; project administration, P.S. and C.D.; funding acquisition, P.S. and C.D. All authors have read and agreed to the published version of the manuscript.

## Institutional Review Board Statement

The study was conducted in accordance with the Declaration of Helsinki and approved by the Institutional Review Board of the University of Cape Town (UCT-HREC 694/2020 and UCT-HREC 141/2022, 4 March 2022).

## Informed Consent Statement

Informed consent was obtained from all subjects involved in the study.

## Data Availability Statement

The data that support the findings of this study are available from the corresponding author [C.D] upon reasonable request.

## Conflicts of Interest

The authors declare no conflicts of interest. The funders had no role in the study’s design, the data collection, analyses, or interpretation of data, the writing of the manuscript, or the decision to publish the results.

## Funding Statement

The research reported in this publication was supported by the South African Medical Research Council (SAMRC) with funds received from the South African Department of Science and Innovation. The Pharmacogenomics and Drug Metabolism Research group is also funded by the SAMRC through an extramural grant.

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

The data that support the findings of this study are available from the corresponding author [C.D] upon reasonable request.

### Supplementary Materials

### Data Availability Statement

The data that support the findings of this study are available from the corresponding author [C.D] upon reasonable request.

## References

1. WHO More Than 700 Million People with Untreated Hypertension: Number of People Living with Hypertension Has Doubled to 1.28 Billion Since 1990. [(accessed on 5 April 2024)]. Available online: https://www.who.int/news/item/25-08-2021-more-than-700-million-people-with-untreated-hypertension.  [https://www.who.int/news/item/25-08-2021-more-than-700-million-people-with-untreated-hypertension](https://www.who.int/news/item/25-08-2021-more-than-700-million-people-with-untreated-hypertension)

2. Brant L.C.C., Passaglia L.G., Pinto-Filho M.M., de Castilho F.M., Ribeiro A.L.P., Nascimento B.R. The Burden of Resistant Hypertension Across the World. Curr. Hypertens. Rep. 2022;24:55–66. doi: 10.1007/s11906-022-01173-w.  [DOI](https://doi.org/10.1007/s11906-022-01173-w) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35118612/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr.%20Hypertens.%20Rep.&title=The%20Burden%20of%20Resistant%20Hypertension%20Across%20the%20World&author=L.C.C.%20Brant&author=L.G.%20Passaglia&author=M.M.%20Pinto-Filho&author=F.M.%20de%20Castilho&author=A.L.P.%20Ribeiro&volume=24&publication_year=2022&pages=55-66&pmid=35118612&doi=10.1007/s11906-022-01173-w&)

3. Nansseu J.R.N., Noubiap J.J.N., Mengnjo M.K., Aminde L.N., Essouma M., Jingi A.M., Bigna J.J.R. The highly neglected burden of resistant hypertension in Africa: A systematic review and meta-analysis. BMJ Open. 2016;6:e011452. doi: 10.1136/bmjopen-2016-011452.  [DOI](https://doi.org/10.1136/bmjopen-2016-011452) | [PMC free article](/articles/PMC5051381/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27650760/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMJ%20Open&title=The%20highly%20neglected%20burden%20of%20resistant%20hypertension%20in%20Africa:%20A%20systematic%20review%20and%20meta-analysis&author=J.R.N.%20Nansseu&author=J.J.N.%20Noubiap&author=M.K.%20Mengnjo&author=L.N.%20Aminde&author=M.%20Essouma&volume=6&publication_year=2016&pages=e011452&pmid=27650760&doi=10.1136/bmjopen-2016-011452&)

4. Moosa M.S., Kuttschreuter L.S., Rayner B.L. Evaluation and management of patients referred to a tertiary-level hypertension clinic in Cape Town, South Africa. S. Afr. Med. J. 2016;106:797. doi: 10.7196/SAMJ.2016.v106i8.9610.  [DOI](https://doi.org/10.7196/SAMJ.2016.v106i8.9610) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27499407/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=S.%20Afr.%20Med.%20J.&title=Evaluation%20and%20management%20of%20patients%20referred%20to%20a%20tertiary-level%20hypertension%20clinic%20in%20Cape%20Town,%20South%20Africa&author=M.S.%20Moosa&author=L.S.%20Kuttschreuter&author=B.L.%20Rayner&volume=106&publication_year=2016&pages=797&pmid=27499407&doi=10.7196/SAMJ.2016.v106i8.9610&)

5. Flack J.M., Buhnerkempe M.G., Moore K.T. Resistant Hypertension: Disease Burden and Emerging Treatment Options. Curr. Hypertens. Rep. 2024;26:183–199. doi: 10.1007/s11906-023-01282-0.  [DOI](https://doi.org/10.1007/s11906-023-01282-0) | [PMC free article](/articles/PMC11533979/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/38363454/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr.%20Hypertens.%20Rep.&title=Resistant%20Hypertension:%20Disease%20Burden%20and%20Emerging%20Treatment%20Options&author=J.M.%20Flack&author=M.G.%20Buhnerkempe&author=K.T.%20Moore&volume=26&publication_year=2024&pages=183-199&pmid=38363454&doi=10.1007/s11906-023-01282-0&)

6. Champaneria M.K., Patel R.S., Oroszi T.L. When blood pressure refuses to budge: Exploring the complexity of resistant hypertension. Front. Cardiovasc. Med. 2023;10:1211199. doi: 10.3389/fcvm.2023.1211199.  [DOI](https://doi.org/10.3389/fcvm.2023.1211199) | [PMC free article](/articles/PMC10322223/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/37416924/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front.%20Cardiovasc.%20Med.&title=When%20blood%20pressure%20refuses%20to%20budge:%20Exploring%20the%20complexity%20of%20resistant%20hypertension&author=M.K.%20Champaneria&author=R.S.%20Patel&author=T.L.%20Oroszi&volume=10&publication_year=2023&pages=1211199&pmid=37416924&doi=10.3389/fcvm.2023.1211199&)

7. Rysz J., Franczyk B., Rysz-Górzyńska M., Gluba-Brzózka A. Pharmacogenomics of Hypertension Treatment. Int. J. Mol. Sci. 2020;21:4709. doi: 10.3390/ijms21134709.  [DOI](https://doi.org/10.3390/ijms21134709) | [PMC free article](/articles/PMC7369859/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32630286/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Mol.%20Sci.&title=Pharmacogenomics%20of%20Hypertension%20Treatment&author=J.%20Rysz&author=B.%20Franczyk&author=M.%20Rysz-G%C3%B3rzy%C5%84ska&author=A.%20Gluba-Brz%C3%B3zka&volume=21&publication_year=2020&pages=4709&pmid=32630286&doi=10.3390/ijms21134709&)

8. Oliveira-Paula G.H., Pereira S.C., Tanus-Santos J.E., Lacchini R. Pharmacogenomics and Hypertension: Current Insights. Pharmgenom. Pers. Med. 2019;12:341–359. doi: 10.2147/pgpm.S230201.  [DOI](https://doi.org/10.2147/pgpm.S230201) | [PMC free article](/articles/PMC6878918/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31819590/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmgenom.%20Pers.%20Med.&title=Pharmacogenomics%20and%20Hypertension:%20Current%20Insights&author=G.H.%20Oliveira-Paula&author=S.C.%20Pereira&author=J.E.%20Tanus-Santos&author=R.%20Lacchini&volume=12&publication_year=2019&pages=341-359&pmid=31819590&doi=10.2147/pgpm.S230201&)

9. Johnson R., Dludla P., Mabhida S., Benjeddou M., Louw J., February F. Pharmacogenomics of amlodipine and hydrochlorothiazide therapy and the quest for improved control of hypertension: A mini review. Heart Fail. Rev. 2019;24:343–357. doi: 10.1007/s10741-018-09765-y.  [DOI](https://doi.org/10.1007/s10741-018-09765-y) | [PMC free article](/articles/PMC6476827/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30645721/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Heart%20Fail.%20Rev.&title=Pharmacogenomics%20of%20amlodipine%20and%20hydrochlorothiazide%20therapy%20and%20the%20quest%20for%20improved%20control%20of%20hypertension:%20A%20mini%20review&author=R.%20Johnson&author=P.%20Dludla&author=S.%20Mabhida&author=M.%20Benjeddou&author=J.%20Louw&volume=24&publication_year=2019&pages=343-357&pmid=30645721&doi=10.1007/s10741-018-09765-y&)

10. Unger T., Borghi C., Charchar F., Khan N.A., Poulter N.R., Prabhakaran D., Ramirez A., Schlaich M., Stergiou G.S., Tomaszewski M., et al. 2020 International Society of Hypertension Global Hypertension Practice Guidelines. Hypertension. 2020;75:1334–1357. doi: 10.1161/hypertensionaha.120.15026.  [DOI](https://doi.org/10.1161/hypertensionaha.120.15026) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32370572/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hypertension&title=2020%20International%20Society%20of%20Hypertension%20Global%20Hypertension%20Practice%20Guidelines&author=T.%20Unger&author=C.%20Borghi&author=F.%20Charchar&author=N.A.%20Khan&author=N.R.%20Poulter&volume=75&publication_year=2020&pages=1334-1357&pmid=32370572&doi=10.1161/hypertensionaha.120.15026&)

11. Seeley A., Prynn J., Perera R., Street R., Davis D., Etyang A.O. Pharmacotherapy for hypertension in Sub-Saharan Africa: A systematic review and network meta-analysis. BMC Med. 2020;18:75. doi: 10.1186/s12916-020-01530-z.  [DOI](https://doi.org/10.1186/s12916-020-01530-z) | [PMC free article](/articles/PMC7099775/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32216794/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Med.&title=Pharmacotherapy%20for%20hypertension%20in%20Sub-Saharan%20Africa:%20A%20systematic%20review%20and%20network%20meta-analysis&author=A.%20Seeley&author=J.%20Prynn&author=R.%20Perera&author=R.%20Street&author=D.%20Davis&volume=18&publication_year=2020&pages=75&pmid=32216794&doi=10.1186/s12916-020-01530-z&)

12. Onwukwe S.C., Ngene N.C. Blood pressure control in hypertensive patients attending a rural community health centre in Gauteng Province, South Africa: A cross-sectional study. S. Afr. Fam. Pract. (2004) 2022;64:e1–e9. doi: 10.4102/safp.v64i1.5403.  [DOI](https://doi.org/10.4102/safp.v64i1.5403) | [PMC free article](/articles/PMC8991089/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35384677/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=S.%20Afr.%20Fam.%20Pract.%20(2004)&title=Blood%20pressure%20control%20in%20hypertensive%20patients%20attending%20a%20rural%20community%20health%20centre%20in%20Gauteng%20Province,%20South%20Africa:%20A%20cross-sectional%20study&author=S.C.%20Onwukwe&author=N.C.%20Ngene&volume=64&publication_year=2022&pages=e1-e9&pmid=35384677&doi=10.4102/safp.v64i1.5403&)

13. Whirl-Carrillo M., Huddart R., Gong L., Sangkuhl K., Thorn C.F., Whaley R., Klein T.E. An Evidence-Based Framework for Evaluating Pharmacogenomics Knowledge for Personalized Medicine. Clin. Pharmacol. Ther. 2021;110:563–572. doi: 10.1002/cpt.2350.  [DOI](https://doi.org/10.1002/cpt.2350) | [PMC free article](/articles/PMC8457105/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34216021/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=An%20Evidence-Based%20Framework%20for%20Evaluating%20Pharmacogenomics%20Knowledge%20for%20Personalized%20Medicine&author=M.%20Whirl-Carrillo&author=R.%20Huddart&author=L.%20Gong&author=K.%20Sangkuhl&author=C.F.%20Thorn&volume=110&publication_year=2021&pages=563-572&pmid=34216021&doi=10.1002/cpt.2350&)

14. Türkmen D., Masoli J.A., Delgado J., Kuo C.L., Bowden J., Melzer D., Pilling L.C. Calcium-channel blockers: Clinical outcome associations with reported pharmacogenetics variants in 32 000 patients. Br. J. Clin. Pharmacol. 2023;89:853–864. doi: 10.1111/bcp.15541.  [DOI](https://doi.org/10.1111/bcp.15541) | [PMC free article](/articles/PMC10091789/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36134646/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Clin.%20Pharmacol.&title=Calcium-channel%20blockers:%20Clinical%20outcome%20associations%20with%20reported%20pharmacogenetics%20variants%20in%2032%20000%20patients&author=D.%20T%C3%BCrkmen&author=J.A.%20Masoli&author=J.%20Delgado&author=C.L.%20Kuo&author=J.%20Bowden&volume=89&publication_year=2023&pages=853-864&pmid=36134646&doi=10.1111/bcp.15541&)

15. Guerra L.A., Lteif C., Arwood M.J., McDonough C.W., Dumeny L., Desai A.A., Cavallari L.H., Duarte J.D. Genetic polymorphisms in ADRB2 and ADRB1 are associated with differential survival in heart failure patients taking β-blockers. Pharmacogenom. J. 2022;22:62–68. doi: 10.1038/s41397-021-00257-1.  [DOI](https://doi.org/10.1038/s41397-021-00257-1) | [PMC free article](/articles/PMC8799480/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34642472/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenom.%20J.&title=Genetic%20polymorphisms%20in%20ADRB2%20and%20ADRB1%20are%20associated%20with%20differential%20survival%20in%20heart%20failure%20patients%20taking%20%CE%B2-blockers&author=L.A.%20Guerra&author=C.%20Lteif&author=M.J.%20Arwood&author=C.W.%20McDonough&author=L.%20Dumeny&volume=22&publication_year=2022&pages=62-68&pmid=34642472&doi=10.1038/s41397-021-00257-1&)

16. Ishigami T., Kino T., Minegishi S., Araki N., Umemura M., Ushio H., Saigoh S., Sugiyama M. Regulators of Epithelial Sodium Channels in Aldosterone-Sensitive Distal Nephrons (ASDN): Critical Roles of Nedd4L/Nedd4-2 and Salt-Sensitive Hypertension. Int. J. Mol. Sci. 2020;21:3871. doi: 10.3390/ijms21113871.  [DOI](https://doi.org/10.3390/ijms21113871) | [PMC free article](/articles/PMC7312533/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32485919/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Mol.%20Sci.&title=Regulators%20of%20Epithelial%20Sodium%20Channels%20in%20Aldosterone-Sensitive%20Distal%20Nephrons%20(ASDN):%20Critical%20Roles%20of%20Nedd4L/Nedd4-2%20and%20Salt-Sensitive%20Hypertension&author=T.%20Ishigami&author=T.%20Kino&author=S.%20Minegishi&author=N.%20Araki&author=M.%20Umemura&volume=21&publication_year=2020&pages=3871&pmid=32485919&doi=10.3390/ijms21113871&)

17. Cunningham P.N., Chapman A.B. The Future of Pharmacogenetics in the Treatment of Hypertension. Pharmacogenomics. 2019;20:129–132. doi: 10.2217/pgs-2018-0191.  [DOI](https://doi.org/10.2217/pgs-2018-0191) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30808251/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=The%20Future%20of%20Pharmacogenetics%20in%20the%20Treatment%20of%20Hypertension&author=P.N.%20Cunningham&author=A.B.%20Chapman&volume=20&publication_year=2019&pages=129-132&pmid=30808251&doi=10.2217/pgs-2018-0191&)

18. McDonough C.W., Burbage S.E., Duarte J.D., Gong Y., Langaee T.Y., Turner S.T., Gums J.G., Chapman A.B., Bailey K.R., Beitelshees A.L., et al. Association of variants in NEDD4L with blood pressure response and adverse cardiovascular outcomes in hypertensive patients treated with thiazide diuretics. J. Hypertens. 2013;31:698–704. doi: 10.1097/HJH.0b013e32835e2a71.  [DOI](https://doi.org/10.1097/HJH.0b013e32835e2a71) | [PMC free article](/articles/PMC3756535/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23353631/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Hypertens.&title=Association%20of%20variants%20in%20NEDD4L%20with%20blood%20pressure%20response%20and%20adverse%20cardiovascular%20outcomes%20in%20hypertensive%20patients%20treated%20with%20thiazide%20diuretics&author=C.W.%20McDonough&author=S.E.%20Burbage&author=J.D.%20Duarte&author=Y.%20Gong&author=T.Y.%20Langaee&volume=31&publication_year=2013&pages=698-704&pmid=23353631&doi=10.1097/HJH.0b013e32835e2a71&)

19. Dumeny L., Vardeny O., Edelmann F., Pieske B., Duarte J.D., Cavallari L.H. NR3C2 genotype is associated with response to spironolactone in diastolic heart failure patients from the Aldo-DHF trial. Pharmacotherapy. 2021;41:978–987. doi: 10.1002/phar.2626.  [DOI](https://doi.org/10.1002/phar.2626) | [PMC free article](/articles/PMC8688239/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34569641/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacotherapy&title=NR3C2%20genotype%20is%20associated%20with%20response%20to%20spironolactone%20in%20diastolic%20heart%20failure%20patients%20from%20the%20Aldo-DHF%20trial&author=L.%20Dumeny&author=O.%20Vardeny&author=F.%20Edelmann&author=B.%20Pieske&author=J.D.%20Duarte&volume=41&publication_year=2021&pages=978-987&pmid=34569641&doi=10.1002/phar.2626&)

20. Sarhan N.M., Shahin M.H., El Rouby N.M., El-Wakeel L.M., Solayman M.H., Langaee T., Khorshid H., Schaalan M.F., Sabri N.A., Cavallari L.H. Effect of Genetic and Nongenetic Factors on the Clinical Response to Mineralocorticoid Receptor Antagonist Therapy in Egyptians with Heart Failure. Clin. Transl. Sci. 2020;13:195–203. doi: 10.1111/cts.12702.  [DOI](https://doi.org/10.1111/cts.12702) | [PMC free article](/articles/PMC6951455/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31560448/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Transl.%20Sci.&title=Effect%20of%20Genetic%20and%20Nongenetic%20Factors%20on%20the%20Clinical%20Response%20to%20Mineralocorticoid%20Receptor%20Antagonist%20Therapy%20in%20Egyptians%20with%20Heart%20Failure&author=N.M.%20Sarhan&author=M.H.%20Shahin&author=N.M.%20El%20Rouby&author=L.M.%20El-Wakeel&author=M.H.%20Solayman&volume=13&publication_year=2020&pages=195-203&pmid=31560448&doi=10.1111/cts.12702&)

21. Zilbermint M., Hannah-Shmouni F., Stratakis C.A. Genetics of Hypertension in African Americans and Others of African Descent. Int. J. Mol. Sci. 2019;20:1081. doi: 10.3390/ijms20051081.  [DOI](https://doi.org/10.3390/ijms20051081) | [PMC free article](/articles/PMC6429313/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30832344/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Mol.%20Sci.&title=Genetics%20of%20Hypertension%20in%20African%20Americans%20and%20Others%20of%20African%20Descent&author=M.%20Zilbermint&author=F.%20Hannah-Shmouni&author=C.A.%20Stratakis&volume=20&publication_year=2019&pages=1081&pmid=30832344&doi=10.3390/ijms20051081&)

22. Peeters L.E.J., Feyz L., Boersma E., Daemen J., van Gelder T., Koch B.C.P., Versmissen J. Clinical Applicability of Monitoring Antihypertensive Drug Levels in Blood. Hypertension. 2020;76:80–86. doi: 10.1161/hypertensionaha.120.15038.  [DOI](https://doi.org/10.1161/hypertensionaha.120.15038) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32418497/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hypertension&title=Clinical%20Applicability%20of%20Monitoring%20Antihypertensive%20Drug%20Levels%20in%20Blood&author=L.E.J.%20Peeters&author=L.%20Feyz&author=E.%20Boersma&author=J.%20Daemen&author=T.%20van%20Gelder&volume=76&publication_year=2020&pages=80-86&pmid=32418497&doi=10.1161/hypertensionaha.120.15038&)

23. Naing L., Winn T., Nordin R. Pratical Issues in Calculating the Sample Size for Prevalence Studies. Arch. Orofac. Sci. 2006;1:9–14.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arch.%20Orofac.%20Sci.&title=Pratical%20Issues%20in%20Calculating%20the%20Sample%20Size%20for%20Prevalence%20Studies&author=L.%20Naing&author=T.%20Winn&author=R.%20Nordin&volume=1&publication_year=2006&pages=9-14&)

24. Shi Y.Y., He L. Publisher Correction: SHEsis, a powerful software platform for analyses of linkage disequilibrium, haplotype construction, and genetic association at polymorphism loci. Cell Res. 2023;1:1. doi: 10.1038/s41422-023-00805-3.  [DOI](https://doi.org/10.1038/s41422-023-00805-3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/37165067/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Res.&title=Publisher%20Correction:%20SHEsis,%20a%20powerful%20software%20platform%20for%20analyses%20of%20linkage%20disequilibrium,%20haplotype%20construction,%20and%20genetic%20association%20at%20polymorphism%20loci&author=Y.Y.%20Shi&author=L.%20He&volume=1&publication_year=2023&pages=1&pmid=37165067&doi=10.1038/s41422-023-00805-3&)

25. Harrison P.W., Amode M.R., Austine-Orimoloye O., Andrey G.A., Barba M., Barnes I., Becker A., Bennett R., Berry A., Bhai J., et al. Ensembl 2024. Nucleic Acids Res. 2023;52:D891–D899. doi: 10.1093/nar/gkad1049.  [DOI](https://doi.org/10.1093/nar/gkad1049) | [PMC free article](/articles/PMC10767893/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/37953337/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&title=Ensembl%202024&author=P.W.%20Harrison&author=M.R.%20Amode&author=O.%20Austine-Orimoloye&author=G.A.%20Andrey&author=M.%20Barba&volume=52&publication_year=2023&pages=D891-D899&pmid=37953337&doi=10.1093/nar/gkad1049&)

26. Lin Y.H., Liu Y.H., Wu D.W., Su H.M., Chen S.C. Dyslipidemia Increases the Risk of Incident Hypertension in a Large Taiwanese Population Follow-Up Study. Nutrients. 2022;14:3277. doi: 10.3390/nu14163277.  [DOI](https://doi.org/10.3390/nu14163277) | [PMC free article](/articles/PMC9416084/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36014784/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nutrients&title=Dyslipidemia%20Increases%20the%20Risk%20of%20Incident%20Hypertension%20in%20a%20Large%20Taiwanese%20Population%20Follow-Up%20Study&author=Y.H.%20Lin&author=Y.H.%20Liu&author=D.W.%20Wu&author=H.M.%20Su&author=S.C.%20Chen&volume=14&publication_year=2022&pages=3277&pmid=36014784&doi=10.3390/nu14163277&)

27. Leopold J.A., Ingelfinger J.R. Aldosterone and Treatment-Resistant Hypertension. N. Engl. J. Med. 2023;388:464–467. doi: 10.1056/NEJMe2213559.  [DOI](https://doi.org/10.1056/NEJMe2213559) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36724334/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%20Engl.%20J.%20Med.&title=Aldosterone%20and%20Treatment-Resistant%20Hypertension&author=J.A.%20Leopold&author=J.R.%20Ingelfinger&volume=388&publication_year=2023&pages=464-467&pmid=36724334&doi=10.1056/NEJMe2213559&)

28. Jia G., Sowers J.R. Hypertension in Diabetes: An Update of Basic Mechanisms and Clinical Disease. Hypertension. 2021;78:1197–1205. doi: 10.1161/hypertensionaha.121.17981.  [DOI](https://doi.org/10.1161/hypertensionaha.121.17981) | [PMC free article](/articles/PMC8516748/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34601960/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hypertension&title=Hypertension%20in%20Diabetes:%20An%20Update%20of%20Basic%20Mechanisms%20and%20Clinical%20Disease&author=G.%20Jia&author=J.R.%20Sowers&volume=78&publication_year=2021&pages=1197-1205&pmid=34601960&doi=10.1161/hypertensionaha.121.17981&)

29. Mehta N.D., Battle S.J., DiPette D.J. Cardiac structure and function in resistant hypertension: The beneficial role of blood pressure control. J. Clin. Hypertens. 2023;25:426–428. doi: 10.1111/jch.14626.  [DOI](https://doi.org/10.1111/jch.14626) | [PMC free article](/articles/PMC10184487/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36861493/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Hypertens.&title=Cardiac%20structure%20and%20function%20in%20resistant%20hypertension:%20The%20beneficial%20role%20of%20blood%20pressure%20control&author=N.D.%20Mehta&author=S.J.%20Battle&author=D.J.%20DiPette&volume=25&publication_year=2023&pages=426-428&pmid=36861493&doi=10.1111/jch.14626&)

30. Rodriguez-Antona C., Savieo J.L., Lauschke V.M., Sangkuhl K., Drögemöller B.I., Wang D., van Schaik R.H.N., Gilep A.A., Peter A.P., Boone E.C., et al. PharmVar GeneFocus: CYP3A5. Clin. Pharmacol. Ther. 2022;112:1159–1171. doi: 10.1002/cpt.2563.  [DOI](https://doi.org/10.1002/cpt.2563) | [PMC free article](/articles/PMC9399309/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35202484/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=PharmVar%20GeneFocus:%20CYP3A5&author=C.%20Rodriguez-Antona&author=J.L.%20Savieo&author=V.M.%20Lauschke&author=K.%20Sangkuhl&author=B.I.%20Dr%C3%B6gem%C3%B6ller&volume=112&publication_year=2022&pages=1159-1171&pmid=35202484&doi=10.1002/cpt.2563&)

31. Lee F.Y., Islahudin F., Abdul Gafor A.H., Wong H.S., Bavanandan S., Mohd Saffian S., Md Redzuan A., Makmor-Bakry M. Adverse Drug Reactions of Antihypertensives and CYP3A5*3 Polymorphism Among Chronic Kidney Disease Patients. Front. Pharmacol. 2022;13:848804. doi: 10.3389/fphar.2022.848804.  [DOI](https://doi.org/10.3389/fphar.2022.848804) | [PMC free article](/articles/PMC8963814/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35359836/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front.%20Pharmacol.&title=Adverse%20Drug%20Reactions%20of%20Antihypertensives%20and%20CYP3A5*3%20Polymorphism%20Among%20Chronic%20Kidney%20Disease%20Patients&author=F.Y.%20Lee&author=F.%20Islahudin&author=A.H.%20Abdul%20Gafor&author=H.S.%20Wong&author=S.%20Bavanandan&volume=13&publication_year=2022&pages=848804&pmid=35359836&doi=10.3389/fphar.2022.848804&)

32. Soria-Chacartegui P., Zubiaur P., Ochoa D., Villapalos-García G., Román M., Matas M., Figueiredo-Tor L., Mejía-Abril G., Calleja S., de Miguel A., et al. Genetic Variation in CYP2D6 and SLC22A1 Affects Amlodipine Pharmacokinetics and Safety. Pharmaceutics. 2023;15:404. doi: 10.3390/pharmaceutics15020404.  [DOI](https://doi.org/10.3390/pharmaceutics15020404) | [PMC free article](/articles/PMC9959242/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36839726/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmaceutics&title=Genetic%20Variation%20in%20CYP2D6%20and%20SLC22A1%20Affects%20Amlodipine%20Pharmacokinetics%20and%20Safety&author=P.%20Soria-Chacartegui&author=P.%20Zubiaur&author=D.%20Ochoa&author=G.%20Villapalos-Garc%C3%ADa&author=M.%20Rom%C3%A1n&volume=15&publication_year=2023&pages=404&pmid=36839726&doi=10.3390/pharmaceutics15020404&)

33. Bhatnagar V., Garcia E.P., O’Connor D.T., Brophy V.H., Alcaraz J., Richard E., Bakris G.L., Middleton J.P., Norris K.C., Wright J., et al. CYP3A4 and CYP3A5 polymorphisms and blood pressure response to amlodipine among African-American men and women with early hypertensive renal disease. Am. J. Nephrol. 2010;31:95–103. doi: 10.1159/000258688.  [DOI](https://doi.org/10.1159/000258688) | [PMC free article](/articles/PMC2853591/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19907160/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Nephrol.&title=CYP3A4%20and%20CYP3A5%20polymorphisms%20and%20blood%20pressure%20response%20to%20amlodipine%20among%20African-American%20men%20and%20women%20with%20early%20hypertensive%20renal%20disease&author=V.%20Bhatnagar&author=E.P.%20Garcia&author=D.T.%20O%E2%80%99Connor&author=V.H.%20Brophy&author=J.%20Alcaraz&volume=31&publication_year=2010&pages=95-103&pmid=19907160&doi=10.1159/000258688&)

34. Oliver E., Mayor F., Jr., D’Ocon P. Beta-blockers: Historical Perspective and Mechanisms of Action. Rev. Esp. Cardiol. (Engl. Ed.) 2019;72:853–862. doi: 10.1016/j.rec.2019.04.006.  [DOI](https://doi.org/10.1016/j.rec.2019.04.006) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31178382/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Rev.%20Esp.%20Cardiol.%20(Engl.%20Ed.)&title=Beta-blockers:%20Historical%20Perspective%20and%20Mechanisms%20of%20Action&author=E.%20Oliver&author=F.%20Mayor&author=P.%20D%E2%80%99Ocon&volume=72&publication_year=2019&pages=853-862&pmid=31178382&doi=10.1016/j.rec.2019.04.006&)

35. Thomas C.D., Johnson J.A. Pharmacogenetic factors affecting β-blocker metabolism and response. Expert Opin. Drug Metab. Toxicol. 2020;16:953–964. doi: 10.1080/17425255.2020.1803279.  [DOI](https://doi.org/10.1080/17425255.2020.1803279) | [PMC free article](/articles/PMC7606773/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32726152/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Expert%20Opin.%20Drug%20Metab.%20Toxicol.&title=Pharmacogenetic%20factors%20affecting%20%CE%B2-blocker%20metabolism%20and%20response&author=C.D.%20Thomas&author=J.A.%20Johnson&volume=16&publication_year=2020&pages=953-964&pmid=32726152&doi=10.1080/17425255.2020.1803279&)

36. Turgut S., Yaren A., Kursunluoglu R., Turgut G. MDR1 C3435T polymorphism in patients with breast cancer. Arch. Med. Res. 2007;38:539–544. doi: 10.1016/j.arcmed.2007.02.005.  [DOI](https://doi.org/10.1016/j.arcmed.2007.02.005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17560460/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arch.%20Med.%20Res.&title=MDR1%20C3435T%20polymorphism%20in%20patients%20with%20breast%20cancer&author=S.%20Turgut&author=A.%20Yaren&author=R.%20Kursunluoglu&author=G.%20Turgut&volume=38&publication_year=2007&pages=539-544&pmid=17560460&doi=10.1016/j.arcmed.2007.02.005&)

37. Rhodes K.E., Zhang W., Yang D., Press O.A., Gordon M., Vallböhmer D., Schultheis A.M., Lurje G., Ladner R.D., Fazzone W., et al. ABCB1, SLCO1B1 and UGT1A1 gene polymorphisms are associated with toxicity in metastatic colorectal cancer patients treated with first-line irinotecan. Drug Metab. Lett. 2007;1:23–30. doi: 10.2174/187231207779814328.  [DOI](https://doi.org/10.2174/187231207779814328) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19356014/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab.%20Lett.&title=ABCB1,%20SLCO1B1%20and%20UGT1A1%20gene%20polymorphisms%20are%20associated%20with%20toxicity%20in%20metastatic%20colorectal%20cancer%20patients%20treated%20with%20first-line%20irinotecan&author=K.E.%20Rhodes&author=W.%20Zhang&author=D.%20Yang&author=O.A.%20Press&author=M.%20Gordon&volume=1&publication_year=2007&pages=23-30&pmid=19356014&doi=10.2174/187231207779814328&)

38. Moriyama A., Nishizawa D., Kasai S., Hasegawa J., Fukuda K.-i., Nagashima M., Katoh R., Ikeda K. Association between Genetic Polymorphisms of the β1-Adrenergic Receptor and Sensitivity to Pain and Fentanyl in Patients Undergoing Painful Cosmetic Surgery. J. Pharmacol. Sci. 2013;121:48–57. doi: 10.1254/jphs.12159fp.  [DOI](https://doi.org/10.1254/jphs.12159fp) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23257656/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Pharmacol.%20Sci.&title=Association%20between%20Genetic%20Polymorphisms%20of%20the%20%CE%B21-Adrenergic%20Receptor%20and%20Sensitivity%20to%20Pain%20and%20Fentanyl%20in%20Patients%20Undergoing%20Painful%20Cosmetic%20Surgery&author=A.%20Moriyama&author=D.%20Nishizawa&author=S.%20Kasai&author=J.%20Hasegawa&author=K.-i.%20Fukuda&volume=121&publication_year=2013&pages=48-57&pmid=23257656&doi=10.1254/jphs.12159fp&)

39. Tavira B., Coto E., Diaz-Corte C., Alvarez V., López-Larrea C., Ortega F. A search for new CYP3A4 variants as determinants of tacrolimus dose requirements in renal-transplanted patients. Pharmacogenet. Genom. 2013;23:445–448. doi: 10.1097/FPC.0b013e3283636856.  [DOI](https://doi.org/10.1097/FPC.0b013e3283636856) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23778326/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet.%20Genom.&title=A%20search%20for%20new%20CYP3A4%20variants%20as%20determinants%20of%20tacrolimus%20dose%20requirements%20in%20renal-transplanted%20patients&author=B.%20Tavira&author=E.%20Coto&author=C.%20Diaz-Corte&author=V.%20Alvarez&author=C.%20L%C3%B3pez-Larrea&volume=23&publication_year=2013&pages=445-448&pmid=23778326&doi=10.1097/FPC.0b013e3283636856&)

40. van Schaik R.H., van der Heiden I.P., van den Anker J.N., Lindemans J. CYP3A5 variant allele frequencies in Dutch Caucasians. Clin. Chem. 2002;48:1668–1671. doi: 10.1093/clinchem/48.10.1668.  [DOI](https://doi.org/10.1093/clinchem/48.10.1668) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12324482/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Chem.&title=CYP3A5%20variant%20allele%20frequencies%20in%20Dutch%20Caucasians&author=R.H.%20van%20Schaik&author=I.P.%20van%20der%20Heiden&author=J.N.%20van%20den%20Anker&author=J.%20Lindemans&volume=48&publication_year=2002&pages=1668-1671&pmid=12324482&doi=10.1093/clinchem/48.10.1668&)
