# Association of DROSHA Variants with Susceptibility and Outcomes in Childhood Acute Lymphoblastic Leukemia

## Metadata
**Authors:** Ioannis Kyriakidis, Iordanis Pelagiadis, Charalampos Pontikoglou, Helen A Papadaki, Eftichia Stiakaki
**Journal:** Current Issues in Molecular Biology
**Date:** 2025 Jun 19
**DOI:** [10.3390/cimb47060473](https://doi.org/10.3390/cimb47060473)
**PMID:** 40699872
**PMCID:** PMC12191901
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC12191901/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC12191901/pdf/cimb-47-00473.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC12191901/pdf/cimb-47-00473.pdf)

## Abstract

MicroRNAs are key regulators of lymphoid differentiation, exhibiting a pivotal role in acute lymphoblastic leukemia (ALL) biology and prognosis. The initial steps of canonical miRNA biogenesis involve the microprocessor complex processing the primary miRNA transcripts into precursor miRNAs via Drosha. DROSHA polymorphisms have been implicated in pediatric ALL and linked with cancer risk. This study investigated the role of rs642321, rs3805500, and rs10035440 DROSHA polymorphisms in ALL susceptibility, relapse, and outcomes in children and adolescents of Greek descent. The study included 252 children and adolescents (115 ALL cases and 137 controls). Genotyping was performed using RT-qPCR and the TaqMan Genotyping Assay. Homozygotes for the minor allele in DROSHA rs642321 were nominally associated with ALL susceptibility (TT vs. CC+CT; OR 4.5; 95% CI: 1.2–21.2; padj = 0.034). Likewise, homozygotes for the minor allele in rs3805500 were linked with ALL risk (GG vs. AA+AG; OR 2.7; 95% CI: 1.3–6.1; padj = 0.012). A suggestive association was observed between the rs3805500 AG genotype and both relapsed (OR 5.8; 95% CI: 1.6–24.3; padj = 0.011) and deceased cases (OR 5; 95% CI: 1.1–26.3; padj = 0.038). Patients with the rs3805500 AG and GG genotypes showed a trend toward poorer overall survival rates. In summary, certain haplotypes of DROSHA polymorphisms may be modestly associated with the occurrence of childhood ALL and its outcomes, although these findings require validation in larger, independent cohorts.

Keywords: acute lymphoblastic leukemia, childhood, Drosha, microprocessor, microRNA, polymorphism, disease susceptibility, survival, relapse, prognosis

## 1. Introduction

Acute lymphoblastic leukemia (ALL) is the neoplasia with the highest incidence rates among children and adolescents [[1](#B1-cimb-47-00473)]. Overall survival (OS) in childhood ALL exceeded 90%, mainly due to modern multiagent chemotherapy protocols with balanced intensity based on the stratification according to risk estimators of relapse: age and white blood cell (WBC) count at diagnosis, extramedullary disease, unfavorable risk genetics, and the measurable residual disease (MRD) reflecting the response to therapy [[2](#B2-cimb-47-00473)]. Despite the advancements in risk-adapted protocols, relapse is a significant burden affecting up to 20% of cases, and two-thirds of these patients are expected to succumb to the disease [[3](#B3-cimb-47-00473)]. Beyond the well-characterized genetic defects found in the blasts of specific childhood ALL subtypes, there have been identified inherited genetic variants that play a crucial role in ALL predisposition, drug responsiveness, and therapy-related toxicities [[4](#B4-cimb-47-00473)]. In the same context, various single-nucleotide polymorphisms (SNPs) in microRNA and miRNA-processing genes have been implicated in childhood ALL susceptibility and prognosis [[5](#B5-cimb-47-00473),[6](#B6-cimb-47-00473),[7](#B7-cimb-47-00473)].

Over the last three decades, microRNAs (miRNAs or miR) have been in the spotlight due to their role in regulating gene expression, mainly at the post-transcriptional level, by silencing or destabilizing target transcripts. These small non-coding RNA molecules, about 22 nucleotides in length in their mature form, display distinct biological features and target specific genes, and depending on their expression, they act as oncogenes or tumor-suppressive genes [[6](#B6-cimb-47-00473),[8](#B8-cimb-47-00473)]. Dysregulated miRNA expression contributes to childhood ALL pathogenesis by affecting the cell cycle control balance, apoptosis, and lymphoid differentiation [[9](#B9-cimb-47-00473),[10](#B10-cimb-47-00473)]. Aberrant miRNA expression profiles have been linked to poor prognosis and drug resistance in ALL. Moreover, miRNAs interact with critical signaling pathways, such as Notch and PI3K/Akt, essential in lymphocyte development and frequently altered in leukemia. Therefore, understanding miRNA involvement in ALL could offer new diagnostic and therapeutic strategies, improving treatment outcomes for affected children [[6](#B6-cimb-47-00473),[11](#B11-cimb-47-00473)]. The concentration of miRNA levels is regulated in six different levels: (i) production of the primary miRNA transcripts (pri-miRNA); (ii) cleavage of the primary transcript into precursor miRNAs by the Drosha–DGCR8 microprocessor complex (pre-miRNA); (iii) transport of the precursor miRNA from the nucleus to the cytoplasm; (iv) cleavage of the precursor miRNA into its mature functional form by Dicer; (v) interaction with target transcripts, leading to subsequent inhibition or cleavage mediated by the formation of the RNA-induced silencing complex or RISC; and (vi) processes that control the stability and degradation of mature miRNAs [[12](#B12-cimb-47-00473)].

Beyond its role in miRNA biogenesis, the role of the ribonuclease Drosha in childhood ALL has been recently under investigation. Significantly higher levels of *DROSHA* mRNA have been documented in pediatric patients with ALL compared to controls [[13](#B13-cimb-47-00473)]. Moreover, several studies associated *DROSHA* (formerly known as nuclear ribonuclease III or *RNASEN*) variants with childhood ALL susceptibility [[14](#B14-cimb-47-00473),[15](#B15-cimb-47-00473),[16](#B16-cimb-47-00473)]. The role of Drosha is essential in the canonical miRNA pathway, and *DROSHA*-knockout has a detrimental effect on miRNA levels (∼16-fold reduction in mature miRNA expression), allowing only the production of Drosha-independent miRNAs and noncanonical miRNAs [[17](#B17-cimb-47-00473),[18](#B18-cimb-47-00473)]. Drosha independent noncanonical miRNA biogenesis may sustain a low-level miRNA production but requires an intact Dicer RNase [[19](#B19-cimb-47-00473)].

Single-nucleotide polymorphisms (SNPs) affecting miRNA biogenesis or mRNA:miRNA interactions are important risk factors in the development of different types of cancer [[20](#B20-cimb-47-00473)]. The present study aims to investigate *DROSHA* rs642321, rs3805500, and rs10035440 in children and adolescents with ALL and healthy controls from Greece. The selection of the SNPs was based on their location (3′-UTR and intron variants), their functional annotations, and previously published studies. This study is linked to the MICRO-CALLS project (MicroRNAs and Childhood ALL Survival) and was conducted, among other aims, to explore the potential effects of *DROSHA* SNPs on measured miRNA levels.

## 2. Materials and Methods

In this case–control study, 252 subjects were enrolled: 115 children and adolescents with ALL diagnosed and treated in the host institution’s tertiary Pediatric Hematology-Oncology Department, and 137 healthy controls referred for routine checkups. Incident ALL cases were not subjected to exclusion criteria, whereas controls were required to have no family history of cancer. Specimens from the pediatric population with ALL were obtained at remission or at the end of the induction phase in deceased patients. All samples were examined twice. [Table 1](#cimb-47-00473-t001) describes the characteristics of cases and controls. An a priori power analysis was conducted using G*Power 3.1 Statistical Power Analyses for Mac (Heinrich-Heine-Universität Düsseldorf, Düsseldorf, Germany; [https://bit.ly/3ubiGF8](https://bit.ly/3ubiGF8), accessed on 16 June 2025). The analysis indicated that a total of 206 participants would be sufficient to detect a statistically significant difference with an odds ratio (OR) of 2.0, α = 0.05, critical z = 1.96, and an actual power of 95.1%.

### Table 1.

| Trait | Patients with ALL(n = 115) | Control Group(n = 137) |
| --- | --- | --- |
| Median age (Range) | 4.9 (1.2–18.1) | 4.2 (0.5–18.5) |
| Sex |   |   |
| Female | 48 (41.7%) | 69 (50.4%) |
| Male | 67 (58.3%) | 68 (49.6%) |
| Relapse | 15 (13%) |   |
| Overall survival | 106 (92.2%) |   |
| Median WBC at diagnosis | 13,000/μL |   |
| (Range) | (800–498,000) |   |
| Lineage |   |   |
| B | 103 (89.6%) |   |
| T | 12 (10.4%) |   |
| Cytogenetics |   |   |
| Hyperdiploidy | 36 (31.3%) |   |
| ETV6::RUNX1 | 34 (29.5%) |   |
| Monosomies and near-diploid ALL | 21 (18.3%) |   |
| BCR::ABL1 | 4 (3.5%) |   |
| TCF3::PBX1 | 3 (2.6%) |   |
| ΚΜΤ2A-r | 1 (0.9%) |   |
| Other | 16 (13.9%) |   |
| High-risk stratification | 43 (37.4%) |   |
| Positive MRD on day 33 | 31 (27%) |   |

Table 1 Caption: Participant characteristics.

The legal guardians of each participant signed informed consent forms. Adolescents were also informed at recruitment and asked to sign the respective consent forms. The protocol of this study was approved by the Research Ethics Committee of the University of Crete (number 128/07.11.2023).

Peripheral blood specimens were collected in vials with ethylenediaminetetraacetic acid (EDTA). Genomic DNA was extracted using the PureLink Genomic DNA Mini Kit (Invitrogen, Thermo Fisher Scientific Inc., Waltham, MA, USA). SNPs *DROSHA* rs642321, rs3805500, and rs10035440 were genotyped by TaqMan SNP Genotyping Assays (C___3197426_20, C___3197387_20 and C__11277968_10, respectively; Applied Biosystems, Thermo Fisher Scientific Inc., Life Technologies Corporation, Carlsbad, CA, USA) on Bio-Rad CFX96 Real-Time PCR Detection System (Bio-Rad Laboratories, Hercules, CA, USA), adhering to the manufacturer instructions. A 100% concordance rate was observed among the duplicate sets for each SNP.

The goodness-of-fit chi-squared test assessed the deviation from Hardy–Weinberg equilibrium (HWE) for each SNP in the control group. The two-sided Fisher–Freeman–Halton Exact Test and Pearson Chi-Square test were utilized to evaluate the statistical significance of differences in genotype frequencies between the control group and patients with ALL, as well as the differences in demographic and clinical variables. Logistic regression analysis was performed to calculate ORs and the corresponding 95% confidence intervals (CIs), which were used to estimate the strength of association between the SNPs and ALL risk, adjusted for age and gender. IBM SPSS Statistics version 29.0.1.0 (IBM Corp., Armonk, NY, USA) was employed for the statistical analyses. SNPStats was used to verify the linkage disequilibrium (LD) results, genotype/haplotype distributions, and associations for binary and quantitative responses ([https://www.snpstats.net/start.htm](https://www.snpstats.net/start.htm), accessed on 16 June 2025; Catalan Institute of Oncology). SNPstats also calculated the Akaike Information (AIC) and the Bayesian Information (BIC) criteria, which determine the goodness-of-fit of an estimated statistical model. The lowest AIC and BIC values identified the recessive model as the best fitting model for all three SNPs [[21](#B21-cimb-47-00473),[22](#B22-cimb-47-00473)]. LD between *DROSHA* SNP pairs was evaluated utilizing Lewontin’s standardized coefficient D′, and Haploview was used for the presentation of the results [[23](#B23-cimb-47-00473)]. The regulatory motifs altered by each variant, meaning the allele-specific binding affinity changes based on the motifs of key transcriptional regulators, were extracted from VannoPortal ([http://www.mulinlab.org/vportal](http://www.mulinlab.org/vportal), accessed on 16 June 2025). The influence of each *DROSHA* SNP genotype on gene expression levels was assessed using the Genotype-Tissue Expression (GTEx) Portal ([http://www.gtexportal.org](http://www.gtexportal.org), accessed on 16 June 2025). Data meta-analysis was performed using Review Manager version 5.4.1 (RevMan, The Cochrane Collaboration, London, UK). All *p* values are two-sided, and the significance level was set at *p* < 0.05.

## 3. Results

All three polymorphisms in the control samples were in accordance with HWE. [Table 2](#cimb-47-00473-t002) lists the respective p_HWE_ along with the VannoPortal regulatory effects noted for each polymorphism. The slight male predilection that was calculated for children with ALL (58.3%) was expected [[24](#B24-cimb-47-00473)]. Sex and age did not significantly differ between cases and controls. [Table 3](#cimb-47-00473-t003) presents the distribution of *DROSHA* rs642321, rs3805500, and rs10035440 genotypes among pediatric patients with ALL and controls and demonstrates the optimal genetic model for each significant association [which, in this case, differs from the additive model typically assessed in genome-wide association studies (GWAS)]. Investigations for all genetic models, the allelic distribution, and crude ORs can be accessed through [Table S1](#app1-cimb-47-00473).

### Table 2.

| Variant | Position GRCh38.p14 | Global Frequency | European Frequency | pHWE | Allele-SpecificBinding Affinity Changes | Location |
| --- | --- | --- | --- | --- | --- | --- |
| rs642321 | chr5:31400896 | T = 21.1%C = 78.9% | T = 19.5%C = 80.5% | 0.929 | SOX6, CEBPA, THAP1, CDK2, MAFK, MEF2B, CUX1, RUNX2, SRF, FOXA1, SMARCA1, SPI1, FOXA3, HOXA1 | 3′-UTRvariant |
| rs3805500 | chr5:31462870 | G = 41%A = 59% | G = 32.9%A = 67.1% | 0.469 | TAL1, SPI1, NR4A1, IRF2, HEY1, MYC, SOX2, ESRRA, FOXA2, CTCF, IRF5, NR2C1, AR, POU2F2, ZNF274, BRF1 | Intronvariant |
| rs10035440 | chr5:31539356 | T = 82%C = 18% | T = 81.5%C = 18.5% | 0.682 | CEBPA, HSF2, NANOG, BCL2, IRF2, CUX1 | C5orf22intronvariant |

Table 2 Caption: HWE and regulatory effects of the SNPs in the DROSHA gene.

### Table 3.

| SNP | Genotype | Controls | Cases | p | Best Model | OR (95% CI) | p |
| --- | --- | --- | --- | --- | --- | --- | --- |
| rs642321 | CC | 83 (60.6%) | 63 (67.7%) | 0.037 | RecessiveTT vs. CC+CT | 4.467(1.202–21.216) | 0.034 |
|   | CT | 51 (37.2%) | 23 (24.7%) |   |  |  |  |
|   | TT | 3 (2.2%) | 7 (7.5%) |   |  |  |  |
| rs3805500 | AA | 54 (46.2%) | 43 (47.3%) | 0.035 | RecessiveGG vs. AA+AG | 2.734(1.269–6.118) | 0.012 |
|   | AG | 51 (43.6%) | 28 (30.8%) |   |  |  |  |
|   | GG | 12 (10.3%) | 20 (22%) |   |  |  |  |
| rs10035440 | TT | 73 (62.9%) | 67 (72.8%) | NS | NS results |  |  |
|   | CT | 36 (31%) | 22 (23.9%) |  |  |  |  |
|   | CC | 7 (6%) | 3 (3.3%) |  |  |  |  |

Table 3 Caption: Summary of nominal genotype associations between DROSHA SNPs rs642321, rs3805500, and rs10035440 and susceptibility to ALL in children from Crete, Greece.

Homozygosity for the minor allele in *DROSHA* rs642321 was nominally associated with ALL susceptibility (TT vs. CC+CT; OR 4.5; 95% CI: 1.2–21.2; p_adj_ = 0.034). Likewise, homozygosity for the minor allele in rs3805500 was linked with ALL risk (GG vs. AA+AG; OR 2.7; 95% CI: 1.3–6.1; p_adj_ = 0.012). No significant association with childhood ALL risk was calculated for rs10035440 genotypes.

The linkage disequilibrium analysis of *DROSHA* rs642321, rs3805500, and rs10035440 using Haploview did not indicate strong LD among these variants in patients with ALL, suggesting that rs642321 and rs3805500 may exert independent effects on disease risk ([Figure 1](#cimb-47-00473-f001)). Both D′ and r^2^ values were close to zero, suggesting that none of the SNPs strongly predict each other.

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/388a/12191901/c9f9d308b82b/cimb-47-00473-g001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=12191901_cimb-47-00473-g001.jpg)

LD plot of DROSHA SNPs in patients with ALL (left) and controls (right), generated using Haploview. Each diamond represents a pairwise LD comparison, with the value inside indicating 100 × D′ (unless D′ = 1.0, in which case the value is omitted). The color gradient reflects the strength of LD: darker red indicates higher D′ values and stronger LD, while white indicates weaker LD.

Further analysis of genotypes in-between phenotypes (B- and T-cell) and B-ALL genetic subtypes calculated insignificant correlations. Notably, children with T-ALL had a significantly higher prevalence of homozygosity for the minor TT allele in rs642321 than patients with B-ALL (37.5% vs. 5.9%; *p* = 0.036). The combined TT/GG haplotype (for rs642321 and rs3805500, respectively) did not associate with pediatric ALL diagnosis. The TT/GG haplotype was identified in only one individual from the control group and three patients. The latter finding suggests that the simultaneous heterozygosity for the minor alleles in both SNPs does not likely confer an additional risk for ALL, as supported by the LD analysis above. In addition, haplotype analysis suggested a potentially stronger association with childhood ALL risk for the rs3805500 G allele when chaperoned by the rs642321 C allele, although this finding requires further validation. [Table S3](#app1-cimb-47-00473) presents the haplotype analysis results for rs642321 and rs3805500. Investigations among patients with ALL indicated significantly higher WBC at diagnosis for GG carriers of rs3805500 (68,540 vs. 31,681; *p* = 0.037). In the same context, homozygotes for the minor allele in rs642321 were diagnosed at an older age (10.95 vs. 6.24; *p* = 0.002). When focusing on ALL subtypes, no patient with hyperdiploidy and all patients with *TCF3*::*PBX1* were homozygotes for the minor allele in rs642321 (Fisher–Freeman–Halton exact test; *p* = 0.051).

Regarding outcomes, the rs3805500 AG genotype was nominally associated with both relapsed (OR 5.8; 95% CI: 1.6–24.3; p_adj_ = 0.011) and deceased cases (OR 5; 95% CI: 1.1–26.3; p_adj_ = 0.038). Noteworthily, children and adolescents with ALL carrying the rs3805500 AG and GG genotypes showed a trend toward lower OS (84.4%) than those homozygous for the major allele A (98.2%; *p* = 0.014). In the same context, relapse-free survival (RFS) appeared lower in children and adolescents with the rs3805500 AG+GG genotypes (81.8%) than in the AA carriers (94.5%, *p* = 0.048). [Figure 2](#cimb-47-00473-f002) illustrates the lower 5-year RFS and OS in rs3805500 heterozygotes observed in our cohort. To further evaluate the prognostic impact of rs3805500, a multivariable Cox proportional-hazards model was developed to assess the occurrence of relapse, adjusting for established clinical factors including age and white blood cell count at diagnosis, cytogenetics, and MRD status at the end of induction (day 33). The AG genotype remained an independent predictor of relapse (HR 8.3, 95% CI: 1.2–58.7, *p* = 0.035; detailed description of the model can be found in [Table S5](#app1-cimb-47-00473)), but, as largely anticipated, positive MRD33 was by far the strongest predictor of relapse (HR 15.3; 95% CI: 1.6–143.2; *p* = 0.017). No relapsed or deceased case was found with the CC genotype in rs10035440, but this association failed to reach the significance levels. Haplotype investigations did not reveal significant associations with ALL prognosis.

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/388a/12191901/72dca8d0d0ca/cimb-47-00473-g002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=12191901_cimb-47-00473-g002.jpg)

Association of DROSHA rs3805500 heterozygotes with RFS (A) and OS (B) in the childhood ALL cohort. A trend toward lower RFS and OS was observed in AG vs. AA+GG (illustrated here) and AG+GG vs. AA comparisons. These associations remained significant after FDR correction, but did not survive Bonferroni adjustment for multiple testing.

## 4. Discussion

Regarding *DROSHA* SNPs in whole blood, the investigation of the expression quantitative trait loci (eQTL) with GTEx revealed significant effect sizes. The relevant data can be accessed in [Table S2 and Figure S1](#app1-cimb-47-00473), both located in the [Supplementary Materials](#app1-cimb-47-00473). In agreement with previous reports that supported up-regulation of *DROSHA* mRNA in childhood ALL compared to controls, higher predicted Drosha expression was observed among rs642321 TT carriers and rs3805500 GG carriers, genotypes that were suggestively associated with increased susceptibility in this study [[13](#B13-cimb-47-00473)]. Among the transcription factors predicted to exhibit allele-specific binding at the studied DROSHA SNPs, SPI1, TAL1, MYC, and CEBPA stand out because of their established roles in hematopoietic differentiation and leukemogenesis, particularly in ALL [[25](#B25-cimb-47-00473)]. Notably, the T-cell immunophenotype was nominally associated with the risk TT genotype in rs642321, which in turn seems to be associated with a 6-fold increase in affinity change for SPI1 ([Table 2](#cimb-47-00473-t002)).

All three SNPs examined in this study have been previously investigated in the context of childhood ALL occurrence, but published results are inconsistent [[14](#B14-cimb-47-00473),[15](#B15-cimb-47-00473),[16](#B16-cimb-47-00473)]. A meta-analysis was therefore performed to assess consistency across populations, integrating our findings with those of earlier studies, as illustrated in the forest plots of [Figure 3](#cimb-47-00473-f003).

### Figure 3.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/388a/12191901/b322822d54ba/cimb-47-00473-g003.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=12191901_cimb-47-00473-g003.jpg)

Forest plots of studies including results for DROSHA SNPs. (A) rs642321 dominant model CC vs. CT+TT; (B) rs3805500 dominant model AA vs. AG+GG; and (C) rs10035440 dominant model TT vs. CT+CC. Each plot shows the OR and 95% CI for individual studies, including the current Greek cohort [14,15,16]. The summary OR is represented by a diamond. Heterogeneity across studies (I2) is indicated. The plots were generated using a random-effects model.

Regarding rs642321, a tetra-amplification refractory mutation system (T-ARMS) PCR study from southeast Iran, involving 75 children with ALL and 115 age- and sex-matched healthy children, reported an association with the CC genotype and the dominant model (*p* = 0.037) [[14](#B14-cimb-47-00473)]. In contrast, our study identified a nominal association under the recessive and overdominant models (TT vs. CC+CT, and CC+TT vs. CT, respectively). Despite differing model assumptions, the meta-analysis confirmed a significant overall association with the CC genotype, suggesting that variation in this locus may influence ALL risk ([Figure 3](#cimb-47-00473-f003)A). The low I^2^ value supports cross-study consistency, although dbSNP data indicate that C and T allele frequencies may vary significantly between Iranian and Greek populations ([https://www.ncbi.nlm.nih.gov/snp/rs642321](https://www.ncbi.nlm.nih.gov/snp/rs642321), accessed on 16 June 2025).

As regards rs3805500, a study from Brazil genotyped 74 children with B-ALL and 159 adult controls (aged 66 ± 16 years old) and reported a threefold increased risk of ALL for AA carriers (OR 2.9; 95% CI: 1.415–5.998; *p* = 0.004), employing the TaqMan OpenArray Genotyping technology (Real-Time PCR) [[15](#B15-cimb-47-00473)]. However, the latter study’s results warrant interpretation with caution, because they only refer to B-ALL cases, subtype prevalence is far beyond expected (no hyperdiploid cases, 21% with *TCF3*::*PBX1*, 14% with *BCR*::*ABL1*, and only 7% with *ETV6*::*RUNX1*), genotyping failed in 17–19% of cases, and there were significant differences between patients and controls as regards age, sex, and ancestry. In our study, we did not replicate this association ([Figure 3](#cimb-47-00473-f003)B); in fact, we observed a nominal association with the GG genotype instead. Furthermore, when our analysis was restricted to B-ALL cases, no significant association was observed for the AA genotype.

As for the distribution of rs10035440 genotypes, both our study and the previous study from Brazil found no significant genotype distribution differences between patients and controls [[15](#B15-cimb-47-00473)]. However, a study from Spain involving 198 pediatric patients with B-ALL and 339 adult unrelated healthy individuals (aged 51 ± 8 years old), also using TaqMan OpenArray Genotyping technology, reported a significant association of rs10035440 genotypes with susceptibility to B-ALL under the additive genetic model (OR 1.38; 95% CI: 1.04–1.83; *p* = 0.027) [[16](#B16-cimb-47-00473)]. The latter study also examined rs3805500, but did not report any significant findings. Our meta-analysis did not support an association between rs10035440 genotypes and childhood ALL risk ([Figure 3](#cimb-47-00473-f003)C). Taken together, these findings highlight the complexity of genetic contributions to ALL susceptibility and underscore the need for further large-scale, population-specific investigations.

The three studied *DROSHA* SNPs in this study have also been investigated in other types of neoplasia, including chronic lymphoblastic leukemia (CLL). Regarding rs642321, there were no significant associations of any genotype with the risk for breast, lung, or bladder cancer or with lung cancer survival [[26](#B26-cimb-47-00473),[27](#B27-cimb-47-00473),[28](#B28-cimb-47-00473)]. As regards rs3805500, there have been indications that the GG genotype could be protective against treatment-related hepatic toxicity experienced by the pediatric ALL population (GG vs. AA+AG; OR 0.12; 95% CI: 0.02–0.98; *p* = 0.009; p_adj_ = 0.239) [[29](#B29-cimb-47-00473)]. Carriers of the GG genotype in rs3805500 may be at greater risk for ALL, but the same population is expected to encounter fewer liver toxicities when diagnosed with the disease. The rs3805500 A allele has also been associated with susceptibility to CLL, while another report associated the rs3805500 minor allele with head and neck cancer prognosis [[30](#B30-cimb-47-00473),[31](#B31-cimb-47-00473)]. Notably, rs3805500 is in LD with rs640831, a polymorphism previously associated with reduced *DROSHA* mRNA expression and expression changes in 56 miRNAs out of 199 analyzed [[28](#B28-cimb-47-00473)]. Regarding rs10035440, a study in pediatric patients with ALL from Spain reported significantly lower rates of hyperbilirubinemia during induction for the CT+CC carriers (vs. TT; OR 0.23; 95% CI: 0.08–0.68; *p* = 0.0041; p_adj_ = 0.1886) [[29](#B29-cimb-47-00473)]. The rs10035440 has also been correlated with ovarian cancer survival, but results failed to remain significant after false discovery rate (FDR) adjustment [[32](#B32-cimb-47-00473)].

As shown above ([Table 2](#cimb-47-00473-t002), [Table S2, and Figure S1](#app1-cimb-47-00473)), these three SNPs appear to have putative roles in transcriptional regulation and may influence *DROSHA* expression. Consistent with this hypothesis, higher Drosha levels in rs642321 and rs3805500 homozygotes for the minor allele are anticipated. Significantly higher Drosha expression levels have been previously documented in childhood ALL and CLL specimens compared to healthy controls [[13](#B13-cimb-47-00473),[33](#B33-cimb-47-00473)]. Deregulated Drosha levels—whether due to regulatory SNPs, structural variants, or deletions—may impair miRNA biogenesis and disrupt the tightly regulated balance between oncogenic and tumor-suppressor miRNAs. This dysregulation may contribute to leukemogenesis by promoting the unchecked proliferation, impaired apoptosis, and altered differentiation of lymphoid progenitor cells. Similar mechanisms have been implicated in other malignancies, including lung, breast, skin, bladder, and head and neck cancers [[16](#B16-cimb-47-00473),[31](#B31-cimb-47-00473),[34](#B34-cimb-47-00473)]. Moreover, the examined intronic and 3′-UTR variants may also influence alternative splicing, potentially affecting the subcellular localization of Drosha and impacting non-canonical miRNA processing pathways [[35](#B35-cimb-47-00473)].

Possible limitations of the present case–control study are as follows: (i) its sample size, although a priori analysis showed that the study population is sufficient to deduce significant correlations with actual power exceeding 95%; (ii) the descent of the population from Crete, a confined island region of Greece, although no deviations from the HWE were observed in the control group; (iii) analytical type limitations, referring to whether observed significant differences can explain differences between patients and controls. The study lacks the rigor to determine whether findings are causal or affected by confounding variables, although the respective statistical analyses detected no such issues. After applying the Bonferroni correction for multiple testing (α = 0.0056), none of the associations between the three *DROSHA* SNPs and ALL remained statistically significant. Furthermore, no relevant GWAS study indicated a potential role for *DROSHA* variants ([https://www.ebi.ac.uk/gwas/efotraits/MONDO_0000870](https://www.ebi.ac.uk/gwas/efotraits/MONDO_0000870), accessed on 16 June 2025). However, when applying a less stringent FDR correction with Benjamini–Hochberg procedure, five associations—including rs642321 (recessive and overdominant models) and rs3805500 (recessive, homozygote, and relapse risk)—remained statistically significant (*q* < 0.05; details in [Table S4](#app1-cimb-47-00473)), supporting the potential relevance of these findings. Moreover, the associations observed for rs3805500 and rs642321 in reports from similar case–control studies may justify further investigation in larger cohorts.

Future research could benefit from expression analysis of Drosha in these specimens and further investigations using RNA-sequencing. Quantifying microRNAs and other microRNA-processing partners may also provide deeper insights into the involved molecular mechanisms [[36](#B36-cimb-47-00473)]. In the context of studying the canonical miRNA biogenesis pathway, recent data support the association of minor G allele carriers in *DICER1* rs3742330 and minor allele A homozygotes of *AGO1* rs636832 with childhood ALL risk [[37](#B37-cimb-47-00473)].

## 5. Conclusions

Nominal associations were observed between *DROSHA* rs642321 and rs3805500 and ALL susceptibility in children and adolescents from Crete, Greece. The present case–control study is the first to investigate these SNPs in a population of Greek descent and only the fourth to examine them in childhood ALL. A trend toward increased ALL risk was observed in rs642321 TT carriers, and rs3805500 GG carriers also showed a potential increase in susceptibility. A meta-analysis incorporating previous studies supported a significant overall association for rs642321, suggesting a possible role in ALL predisposition. Statistical analysis disclosed the increased prevalence of high-risk ALL features among the homozygotes for the minor allele in rs642321 (higher age at diagnosis and T-cell immunophenotype) and rs3805500 (higher WBC at diagnosis). Furthermore, the alternative allele in rs3805500 was more prevalent among relapsed and deceased cases. These preliminary findings may reflect a modest contribution of miRNA-processing gene polymorphisms to ALL biology. Further validation in different ethnic groups, larger and independent cohorts, is essential to establish the clinical relevance of these associations.

## Abbreviations

| AIC | Akaike information criteria |
| --- | --- |
| BIC | Bayesian information criteria |
| ALL | Acute lymphoblastic leukemia |
| ChIP-seq | Chromatin immunoprecipitation sequencing |
| CI | Confidence interval |
| EDTA | Ethylenediaminetetraacetic acid |
| FDR | False discovery rate |
| GTEx | Genotype-Tissue Expression |
| GWAS | Genome-wide association study |
| HWE | Hardy–Weinberg equilibrium |
| LD | Linkage disequilibrium |
| MRD | Measurable residual disease |
| NS | Not significant |
| OR | Odds ratio |
| OS | Overall survival |
| PCR | Polymerase chain reaction |
| RFS | Relapse-free survival |
| SNP | Single-nucleotide polymorphism |
| UTR | Untranslated region |
| WBC | White blood cell |
## Supplementary Materials

The supporting information can be downloaded at: [https://www.mdpi.com/article/10.3390/cimb47060473/s1](https://www.mdpi.com/article/10.3390/cimb47060473/s1).

## Author Contributions

Conceptualization: C.P., H.A.P. and E.S.; data curation: I.K. and I.P.; formal analysis: I.K.; funding acquisition: I.K. and E.S.; investigation: I.K. and I.P.; methodology: I.K., I.P. and E.S.; project administration: I.P., C.P., H.A.P. and E.S.; resources: I.K., I.P. and E.S.; software: I.K.; supervision: C.P., H.A.P. and E.S.; validation: I.P. and E.S.; visualization: I.K.; writing—original draft: I.K.; writing—review and editing: I.P., C.P., H.A.P. and E.S. All authors have read and agreed to the published version of the manuscript.

## Institutional Review Board Statement

The study was conducted in accordance with the Declaration of Helsinki, and the protocol of this study was approved by the Research Ethics Committee of the University of Crete (number 128/07.11.2023) on 7 November 2023.

## Informed Consent Statement

Informed consent was obtained from all subjects involved in the study.

## Data Availability Statement

The data presented in this study are available upon request from the corresponding author for privacy reasons.

## Conflicts of Interest

The authors declare no conflicts of interest.

## Funding Statement

This work was supported by the Special Account for Research Funds of the University of Crete (grant number 11515).

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

The data presented in this study are available upon request from the corresponding author for privacy reasons.

### Supplementary Materials

### Data Availability Statement

The data presented in this study are available upon request from the corresponding author for privacy reasons.

## References

1. Tang X.-W., Jiang J., Huang S., Shi X.-M., Xu H., Xu J., Peng J.-Y., Zhang W., Shi L., Zhong X.-L., et al. Long-Term Trends in Cancer Incidence and Mortality among U.S. Children and Adolescents: A SEER Database Analysis from 1975 to 2018. Front. Pediatr. 2024;12:1357093. doi: 10.3389/fped.2024.1357093.  [DOI](https://doi.org/10.3389/fped.2024.1357093) | [PMC free article](/articles/PMC11258622/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/39035461/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front.%20Pediatr.&title=Long-Term%20Trends%20in%20Cancer%20Incidence%20and%20Mortality%20among%20U.S.%20Children%20and%20Adolescents:%20A%20SEER%20Database%20Analysis%20from%201975%20to%202018&author=X.-W.%20Tang&author=J.%20Jiang&author=S.%20Huang&author=X.-M.%20Shi&author=H.%20Xu&volume=12&publication_year=2024&pages=1357093&pmid=39035461&doi=10.3389/fped.2024.1357093&)

2. DelRocco N.J., Loh M.L., Borowitz M.J., Gupta S., Rabin K.R., Zweidler-McKay P., Maloney K.W., Mattano L.A., Larsen E., Angiolillo A., et al. Enhanced Risk Stratification for Children and Young Adults with B-Cell Acute Lymphoblastic Leukemia: A Children’s Oncology Group Report. Leukemia. 2024;38:720–728. doi: 10.1038/s41375-024-02166-1.  [DOI](https://doi.org/10.1038/s41375-024-02166-1) | [PMC free article](/articles/PMC10997503/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/38360863/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Leukemia&title=Enhanced%20Risk%20Stratification%20for%20Children%20and%20Young%20Adults%20with%20B-Cell%20Acute%20Lymphoblastic%20Leukemia:%20A%20Children%E2%80%99s%20Oncology%20Group%20Report&author=N.J.%20DelRocco&author=M.L.%20Loh&author=M.J.%20Borowitz&author=S.%20Gupta&author=K.R.%20Rabin&volume=38&publication_year=2024&pages=720-728&pmid=38360863&doi=10.1038/s41375-024-02166-1&)

3. Samardžić-Predojević J., Đurđević-Banjac B., Malčić-Zanić D. Influence of Minimal Residual Disease at Day 15 of Induction Therapy on Survival of Children with Acute Lymphoblastic Leukemia. Acta Med. Acad. 2023;52:153–160. doi: 10.5644/ama2006-124.427.  [DOI](https://doi.org/10.5644/ama2006-124.427) | [PMC free article](/articles/PMC10945324/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/38407081/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Acta%20Med.%20Acad.&title=Influence%20of%20Minimal%20Residual%20Disease%20at%20Day%2015%20of%20Induction%20Therapy%20on%20Survival%20of%20Children%20with%20Acute%20Lymphoblastic%20Leukemia&author=J.%20Samard%C5%BEi%C4%87-Predojevi%C4%87&author=B.%20%C4%90ur%C4%91evi%C4%87-Banjac&author=D.%20Mal%C4%8Di%C4%87-Zani%C4%87&volume=52&publication_year=2023&pages=153-160&pmid=38407081&doi=10.5644/ama2006-124.427&)

4. Moriyama T., Relling M.V., Yang J.J. Inherited Genetic Variation in Childhood Acute Lymphoblastic Leukemia. Blood. 2015;125:3988–3995. doi: 10.1182/blood-2014-12-580001.  [DOI](https://doi.org/10.1182/blood-2014-12-580001) | [PMC free article](/articles/PMC4481591/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25999454/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Inherited%20Genetic%20Variation%20in%20Childhood%20Acute%20Lymphoblastic%20Leukemia&author=T.%20Moriyama&author=M.V.%20Relling&author=J.J.%20Yang&volume=125&publication_year=2015&pages=3988-3995&pmid=25999454&doi=10.1182/blood-2014-12-580001&)

5. Navabi A., Aznab M., Heydarpour F. The Association between microRNA Polymorphisms and the Risk of Childhood Acute Lymphoblastic Leukemia: A Meta-Analysis. Cancer Epidemiol. 2022;81:102285. doi: 10.1016/j.canep.2022.102285.  [DOI](https://doi.org/10.1016/j.canep.2022.102285) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36343468/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Epidemiol.&title=The%20Association%20between%20microRNA%20Polymorphisms%20and%20the%20Risk%20of%20Childhood%20Acute%20Lymphoblastic%20Leukemia:%20A%20Meta-Analysis&author=A.%20Navabi&author=M.%20Aznab&author=F.%20Heydarpour&volume=81&publication_year=2022&pages=102285&pmid=36343468&doi=10.1016/j.canep.2022.102285&)

6. Kyriakidis I., Kyriakidis K., Tsezou A. MicroRNAs and the Diagnosis of Childhood Acute Lymphoblastic Leukemia: Systematic Review, Meta-Analysis and Re-Analysis with Novel Small RNA-Seq Tools. Cancers. 2022;14:3976. doi: 10.3390/cancers14163976.  [DOI](https://doi.org/10.3390/cancers14163976) | [PMC free article](/articles/PMC9406077/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36010971/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancers&title=MicroRNAs%20and%20the%20Diagnosis%20of%20Childhood%20Acute%20Lymphoblastic%20Leukemia:%20Systematic%20Review,%20Meta-Analysis%20and%20Re-Analysis%20with%20Novel%20Small%20RNA-Seq%20Tools&author=I.%20Kyriakidis&author=K.%20Kyriakidis&author=A.%20Tsezou&volume=14&publication_year=2022&pages=3976&pmid=36010971&doi=10.3390/cancers14163976&)

7. da Silva Menezes E., de Moraes F.C.A., de Nazaré Cohen-Paes A., Wanderley A.V., Pereira E.E.B., Pastana L.F., Modesto A.A.C., de Assumpção P.P., Burbano R.M.R., Dos Santos S.E.B., et al. Influence of Genetic Variations in miRNA and Genes Encoding Proteins in the miRNA Synthesis Complex on Toxicity of the Treatment of Pediatric B-Cell ALL in the Brazilian Amazon. Int. J. Mol. Sci. 2023;24:4431. doi: 10.3390/ijms24054431.  [DOI](https://doi.org/10.3390/ijms24054431) | [PMC free article](/articles/PMC10003057/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36901860/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Mol.%20Sci.&title=Influence%20of%20Genetic%20Variations%20in%20miRNA%20and%20Genes%20Encoding%20Proteins%20in%20the%20miRNA%20Synthesis%20Complex%20on%20Toxicity%20of%20the%20Treatment%20of%20Pediatric%20B-Cell%20ALL%20in%20the%20Brazilian%20Amazon&author=E.%20da%20Silva%20Menezes&author=F.C.A.%20de%20Moraes&author=A.%20de%20Nazar%C3%A9%20Cohen-Paes&author=A.V.%20Wanderley&author=E.E.B.%20Pereira&volume=24&publication_year=2023&pages=4431&pmid=36901860&doi=10.3390/ijms24054431&)

8. Shang R., Lee S., Senavirathne G., Lai E.C. microRNAs in Action: Biogenesis, Function and Regulation. Nat. Rev. Genet. 2023;24:816–833. doi: 10.1038/s41576-023-00611-y.  [DOI](https://doi.org/10.1038/s41576-023-00611-y) | [PMC free article](/articles/PMC11087887/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/37380761/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Genet.&title=microRNAs%20in%20Action:%20Biogenesis,%20Function%20and%20Regulation&author=R.%20Shang&author=S.%20Lee&author=G.%20Senavirathne&author=E.C.%20Lai&volume=24&publication_year=2023&pages=816-833&pmid=37380761&doi=10.1038/s41576-023-00611-y&)

9. O’Connell R.M., Rao D.S., Chaudhuri A.A., Baltimore D. Physiological and Pathological Roles for microRNAs in the Immune System. Nat. Rev. Immunol. 2010;10:111–122. doi: 10.1038/nri2708.  [DOI](https://doi.org/10.1038/nri2708) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20098459/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Immunol.&title=Physiological%20and%20Pathological%20Roles%20for%20microRNAs%20in%20the%20Immune%20System&author=R.M.%20O%E2%80%99Connell&author=D.S.%20Rao&author=A.A.%20Chaudhuri&author=D.%20Baltimore&volume=10&publication_year=2010&pages=111-122&pmid=20098459&doi=10.1038/nri2708&)

10. Santulli G. MicroRNA, Basic Science: From Molecular Biology to Clinical Practice. Volume 887. Springer; Cham, Switzerland: 2015. Advances in Experimental Medicine and Biology.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=MicroRNA,%20Basic%20Science:%20From%20Molecular%20Biology%20to%20Clinical%20Practice&author=G.%20Santulli&publication_year=2015&isbn=978-3-319-22379-7&)

11. Deng W. Advancements in the Regulatory Role of microRNAs in Childhood Acute Lymphoblastic Leukemia: Mechanisms and Clinical Implications. Technol. Cancer Res. Treat. 2024;23:15330338241273143. doi: 10.1177/15330338241273143.  [DOI](https://doi.org/10.1177/15330338241273143) | [PMC free article](/articles/PMC11301750/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/39099455/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Technol.%20Cancer%20Res.%20Treat.&title=Advancements%20in%20the%20Regulatory%20Role%20of%20microRNAs%20in%20Childhood%20Acute%20Lymphoblastic%20Leukemia:%20Mechanisms%20and%20Clinical%20Implications&author=W.%20Deng&volume=23&publication_year=2024&pages=15330338241273143&pmid=39099455&doi=10.1177/15330338241273143&)

12. Bofill-De Ros X., Vang Ørom U.A. Recent Progress in miRNA Biogenesis and Decay. RNA Biol. 2024;21:1–8. doi: 10.1080/15476286.2023.2288741.  [DOI](https://doi.org/10.1080/15476286.2023.2288741) | [PMC free article](/articles/PMC10761092/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/38031325/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=RNA%20Biol.&title=Recent%20Progress%20in%20miRNA%20Biogenesis%20and%20Decay&author=X.%20Bofill-De%20Ros&author=U.A.%20Vang%20%C3%98rom&volume=21&publication_year=2024&pages=1-8&pmid=38031325&doi=10.1080/15476286.2023.2288741&)

13. Hajirostamlou M., Ghorbian S. Evaluation of the Clinical Significance of RNase III Enzyme DROSHA in Pediatrics Acute Lymphocytic Leukemia. Mol. Biol. Rep. 2021;48:451–456. doi: 10.1007/s11033-020-06072-4.  [DOI](https://doi.org/10.1007/s11033-020-06072-4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33389538/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Biol.%20Rep.&title=Evaluation%20of%20the%20Clinical%20Significance%20of%20RNase%20III%20Enzyme%20DROSHA%20in%20Pediatrics%20Acute%20Lymphocytic%20Leukemia&author=M.%20Hajirostamlou&author=S.%20Ghorbian&volume=48&publication_year=2021&pages=451-456&pmid=33389538&doi=10.1007/s11033-020-06072-4&)

14. Hashemi M., Hasani S.-S., Naderi M. DROSHA Rs642321 Polymorphism Influence Susceptibility to Childhood Acute Lymphoblastic Leukemia: A Preliminary Report. Indian. J. Med. Paediatr. Oncol. 2017;38:416–419. doi: 10.4103/ijmpo.ijmpo_4_15.  [DOI](https://doi.org/10.4103/ijmpo.ijmpo_4_15) | [PMC free article](/articles/PMC5759056/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29333004/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Indian.%20J.%20Med.%20Paediatr.%20Oncol.&title=DROSHA%20Rs642321%20Polymorphism%20Influence%20Susceptibility%20to%20Childhood%20Acute%20Lymphoblastic%20Leukemia:%20A%20Preliminary%20Report&author=M.%20Hashemi&author=S.-S.%20Hasani&author=M.%20Naderi&volume=38&publication_year=2017&pages=416-419&pmid=29333004&doi=10.4103/ijmpo.ijmpo_4_15&)

15. de Souza T.P., de Carvalho D.C., Wanderley A.V., Fernandes S.M., Rodrigues J.C.G., Cohen-Paes A., Fernandes M.R., Mello Junior F.A.R., Pastana L.F., Vinagre L.W.M.S., et al. Influence of Variants of the Drosha, Mir499a, and Mir938 Genes on Susceptibility to Acute Lymphoblastic Leukemia in an Admixed Population from the Brazilian Amazon. Am. J. Transl. Res. 2020;12:8216–8224.  [PMC free article](/articles/PMC7791525/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33437394/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Transl.%20Res.&title=Influence%20of%20Variants%20of%20the%20Drosha,%20Mir499a,%20and%20Mir938%20Genes%20on%20Susceptibility%20to%20Acute%20Lymphoblastic%20Leukemia%20in%20an%20Admixed%20Population%20from%20the%20Brazilian%20Amazon&author=T.P.%20de%20Souza&author=D.C.%20de%20Carvalho&author=A.V.%20Wanderley&author=S.M.%20Fernandes&author=J.C.G.%20Rodrigues&volume=12&publication_year=2020&pages=8216-8224&pmid=33437394&)

16. Gutierrez-Camino A., Lopez-Lopez E., Martin-Guerrero I., Piñan M.A., Garcia-Miguel P., Sanchez-Toledo J., Carbone Bañeres A., Uriz J., Navajas A., Garcia-Orad A. Noncoding RNA-Related Polymorphisms in Pediatric Acute Lymphoblastic Leukemia Susceptibility. Pediatr. Res. 2014;75:767–773. doi: 10.1038/pr.2014.43.  [DOI](https://doi.org/10.1038/pr.2014.43) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24618566/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pediatr.%20Res.&title=Noncoding%20RNA-Related%20Polymorphisms%20in%20Pediatric%20Acute%20Lymphoblastic%20Leukemia%20Susceptibility&author=A.%20Gutierrez-Camino&author=E.%20Lopez-Lopez&author=I.%20Martin-Guerrero&author=M.A.%20Pi%C3%B1an&author=P.%20Garcia-Miguel&volume=75&publication_year=2014&pages=767-773&pmid=24618566&doi=10.1038/pr.2014.43&)

17. Kim Y.-K., Kim B., Kim V.N. Re-Evaluation of the Roles of DROSHA, Export in 5, and DICER in microRNA Biogenesis. Proc. Natl. Acad. Sci. USA. 2016;113:E1881–E1889. doi: 10.1073/pnas.1602532113.  [DOI](https://doi.org/10.1073/pnas.1602532113) | [PMC free article](/articles/PMC4822641/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26976605/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&title=Re-Evaluation%20of%20the%20Roles%20of%20DROSHA,%20Export%20in%205,%20and%20DICER%20in%20microRNA%20Biogenesis&author=Y.-K.%20Kim&author=B.%20Kim&author=V.N.%20Kim&volume=113&publication_year=2016&pages=E1881-E1889&pmid=26976605&doi=10.1073/pnas.1602532113&)

18. Johnson K.C., Johnson S.T., Liu J., Chu Y., Arana C., Han Y., Wang T., Corey D.R. Consequences of Depleting TNRC6, AGO, and DROSHA Proteins on Expression of microRNAs. RNA. 2023;29:1166–1184. doi: 10.1261/rna.079647.123.  [DOI](https://doi.org/10.1261/rna.079647.123) | [PMC free article](/articles/PMC10351893/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/37169394/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=RNA&title=Consequences%20of%20Depleting%20TNRC6,%20AGO,%20and%20DROSHA%20Proteins%20on%20Expression%20of%20microRNAs&author=K.C.%20Johnson&author=S.T.%20Johnson&author=J.%20Liu&author=Y.%20Chu&author=C.%20Arana&volume=29&publication_year=2023&pages=1166-1184&pmid=37169394&doi=10.1261/rna.079647.123&)

19. Treiber T., Treiber N., Meister G. Regulation of microRNA Biogenesis and Its Crosstalk with Other Cellular Pathways. Nat. Rev. Mol. Cell Biol. 2019;20:5–20. doi: 10.1038/s41580-018-0059-1.  [DOI](https://doi.org/10.1038/s41580-018-0059-1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30728477/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Mol.%20Cell%20Biol.&title=Regulation%20of%20microRNA%20Biogenesis%20and%20Its%20Crosstalk%20with%20Other%20Cellular%20Pathways&author=T.%20Treiber&author=N.%20Treiber&author=G.%20Meister&volume=20&publication_year=2019&pages=5-20&pmid=30728477&doi=10.1038/s41580-018-0059-1&)

20. Elshazli R.M., Toraih E.A., Hussein M.H., Ruiz E.M., Kandil E., Fawzy M.S. Pan-Cancer Study on Variants of Canonical miRNA Biogenesis Pathway Components: A Pooled Analysis. Cancers. 2023;15:338. doi: 10.3390/cancers15020338.  [DOI](https://doi.org/10.3390/cancers15020338) | [PMC free article](/articles/PMC9856462/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36672288/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancers.&title=Pan-Cancer%20Study%20on%20Variants%20of%20Canonical%20miRNA%20Biogenesis%20Pathway%20Components:%20A%20Pooled%20Analysis&author=R.M.%20Elshazli&author=E.A.%20Toraih&author=M.H.%20Hussein&author=E.M.%20Ruiz&author=E.%20Kandil&volume=15&publication_year=2023&pages=338&pmid=36672288&doi=10.3390/cancers15020338&)

21. Posada D., Buckley T.R. Model Selection and Model Averaging in Phylogenetics: Advantages of Akaike Information Criterion and Bayesian Approaches over Likelihood Ratio Tests. Syst. Biol. 2004;53:793–808. doi: 10.1080/10635150490522304.  [DOI](https://doi.org/10.1080/10635150490522304) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15545256/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Syst.%20Biol.&title=Model%20Selection%20and%20Model%20Averaging%20in%20Phylogenetics:%20Advantages%20of%20Akaike%20Information%20Criterion%20and%20Bayesian%20Approaches%20over%20Likelihood%20Ratio%20Tests&author=D.%20Posada&author=T.R.%20Buckley&volume=53&publication_year=2004&pages=793-808&pmid=15545256&doi=10.1080/10635150490522304&)

22. Horita N., Kaneko T. Genetic Model Selection for a Case–Control Study and a Meta-Analysis. Meta Gene. 2015;5:1–8. doi: 10.1016/j.mgene.2015.04.003.  [DOI](https://doi.org/10.1016/j.mgene.2015.04.003) | [PMC free article](/articles/PMC4443430/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26042205/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Meta%20Gene&title=Genetic%20Model%20Selection%20for%20a%20Case%E2%80%93Control%20Study%20and%20a%20Meta-Analysis&author=N.%20Horita&author=T.%20Kaneko&volume=5&publication_year=2015&pages=1-8&pmid=26042205&doi=10.1016/j.mgene.2015.04.003&)

23. Barrett J.C., Fry B., Maller J., Daly M.J. Haploview: Analysis and Visualization of LD and Haplotype Maps. Bioinformatics. 2005;21:263–265. doi: 10.1093/bioinformatics/bth457.  [DOI](https://doi.org/10.1093/bioinformatics/bth457) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15297300/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=Haploview:%20Analysis%20and%20Visualization%20of%20LD%20and%20Haplotype%20Maps&author=J.C.%20Barrett&author=B.%20Fry&author=J.%20Maller&author=M.J.%20Daly&volume=21&publication_year=2005&pages=263-265&pmid=15297300&doi=10.1093/bioinformatics/bth457&)

24. Siegel R.L., Miller K.D., Jemal A. Cancer Statistics, 2020. CA Cancer J. Clin. 2020;70:7–30. doi: 10.3322/caac.21590.  [DOI](https://doi.org/10.3322/caac.21590) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31912902/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=CA%20Cancer%20J.%20Clin.&title=Cancer%20Statistics,%202020&author=R.L.%20Siegel&author=K.D.%20Miller&author=A.%20Jemal&volume=70&publication_year=2020&pages=7-30&pmid=31912902&doi=10.3322/caac.21590&)

25. Seki M., Kimura S., Isobe T., Yoshida K., Ueno H., Nakajima-Takagi Y., Wang C., Lin L., Kon A., Suzuki H., et al. Recurrent SPI1 (PU.1) Fusions in High-Risk Pediatric T Cell Acute Lymphoblastic Leukemia. Nat. Genet. 2017;49:1274–1281. doi: 10.1038/ng.3900.  [DOI](https://doi.org/10.1038/ng.3900) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28671687/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet.&title=Recurrent%20SPI1%20(PU.1)%20Fusions%20in%20High-Risk%20Pediatric%20T%20Cell%20Acute%20Lymphoblastic%20Leukemia&author=M.%20Seki&author=S.%20Kimura&author=T.%20Isobe&author=K.%20Yoshida&author=H.%20Ueno&volume=49&publication_year=2017&pages=1274-1281&pmid=28671687&doi=10.1038/ng.3900&)

26. Yuan L., Chu H., Wang M., Gu X., Shi D., Ma L., Zhong D., Du M., Li P., Tong N., et al. Genetic Variation in DROSHA 3’UTR Regulated by Hsa-miR-27b Is Associated with Bladder Cancer Risk. PLoS ONE. 2013;8:e81524. doi: 10.1371/journal.pone.0081524.  [DOI](https://doi.org/10.1371/journal.pone.0081524) | [PMC free article](/articles/PMC3842954/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24312312/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&title=Genetic%20Variation%20in%20DROSHA%203%E2%80%99UTR%20Regulated%20by%20Hsa-miR-27b%20Is%20Associated%20with%20Bladder%20Cancer%20Risk&author=L.%20Yuan&author=H.%20Chu&author=M.%20Wang&author=X.%20Gu&author=D.%20Shi&volume=8&publication_year=2013&pages=e81524&pmid=24312312&doi=10.1371/journal.pone.0081524&)

27. Sung H., Jeon S., Lee K.-M., Han S., Song M., Choi J.-Y., Park S.K., Yoo K.-Y., Noh D.-Y., Ahn S.-H., et al. Common Genetic Polymorphisms of microRNA Biogenesis Pathway Genes and Breast Cancer Survival. BMC Cancer. 2012;12:195. doi: 10.1186/1471-2407-12-195.  [DOI](https://doi.org/10.1186/1471-2407-12-195) | [PMC free article](/articles/PMC3487887/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22639842/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Cancer&title=Common%20Genetic%20Polymorphisms%20of%20microRNA%20Biogenesis%20Pathway%20Genes%20and%20Breast%20Cancer%20Survival&author=H.%20Sung&author=S.%20Jeon&author=K.-M.%20Lee&author=S.%20Han&author=M.%20Song&volume=12&publication_year=2012&pages=195&pmid=22639842&doi=10.1186/1471-2407-12-195&)

28. Rotunno M., Zhao Y., Bergen A.W., Koshiol J., Burdette L., Rubagotti M., Linnoila R.I., Marincola F.M., Bertazzi P.A., Pesatori A.C., et al. Inherited Polymorphisms in the RNA-Mediated Interference Machinery Affect microRNA Expression and Lung Cancer Survival. Br. J. Cancer. 2010;103:1870–1874. doi: 10.1038/sj.bjc.6605976.  [DOI](https://doi.org/10.1038/sj.bjc.6605976) | [PMC free article](/articles/PMC3008605/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21102586/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Cancer&title=Inherited%20Polymorphisms%20in%20the%20RNA-Mediated%20Interference%20Machinery%20Affect%20microRNA%20Expression%20and%20Lung%20Cancer%20Survival&author=M.%20Rotunno&author=Y.%20Zhao&author=A.W.%20Bergen&author=J.%20Koshiol&author=L.%20Burdette&volume=103&publication_year=2010&pages=1870-1874&pmid=21102586&doi=10.1038/sj.bjc.6605976&)

29. López-López E., Gutiérrez-Camino Á., Piñán M.Á., Sánchez-Toledo J., Uriz J.J., Ballesteros J., García-Miguel P., Navajas A., García-Orad Á. Pharmacogenetics of MicroRNAs and MicroRNAs Biogenesis Machinery in Pediatric Acute Lymphoblastic Leukemia. PLoS ONE. 2014;9:e91261. doi: 10.1371/journal.pone.0091261.  [DOI](https://doi.org/10.1371/journal.pone.0091261) | [PMC free article](/articles/PMC3948785/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24614921/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&title=Pharmacogenetics%20of%20MicroRNAs%20and%20MicroRNAs%20Biogenesis%20Machinery%20in%20Pediatric%20Acute%20Lymphoblastic%20Leukemia&author=E.%20L%C3%B3pez-L%C3%B3pez&author=%C3%81.%20Guti%C3%A9rrez-Camino&author=M.%C3%81.%20Pi%C3%B1%C3%A1n&author=J.%20S%C3%A1nchez-Toledo&author=J.J.%20Uriz&volume=9&publication_year=2014&pages=e91261&pmid=24614921&doi=10.1371/journal.pone.0091261&)

30. Martin-Guerrero I., Gutierrez-Camino A., Lopez-Lopez E., Bilbao-Aldaiturriaga N., Pombar-Gomez M., Ardanaz M., Garcia-Orad A. Genetic Variants in miRNA Processing Genes and Pre-miRNAs Are Associated with the Risk of Chronic Lymphocytic Leukemia. PLoS ONE. 2015;10:e0118905. doi: 10.1371/journal.pone.0118905.  [DOI](https://doi.org/10.1371/journal.pone.0118905) | [PMC free article](/articles/PMC4368096/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25793711/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&title=Genetic%20Variants%20in%20miRNA%20Processing%20Genes%20and%20Pre-miRNAs%20Are%20Associated%20with%20the%20Risk%20of%20Chronic%20Lymphocytic%20Leukemia&author=I.%20Martin-Guerrero&author=A.%20Gutierrez-Camino&author=E.%20Lopez-Lopez&author=N.%20Bilbao-Aldaiturriaga&author=M.%20Pombar-Gomez&volume=10&publication_year=2015&pages=e0118905&pmid=25793711&doi=10.1371/journal.pone.0118905&)

31. Zhang X., Yang H., Lee J.J., Kim E., Lippman S.M., Khuri F.R., Spitz M.R., Lotan R., Hong W.K., Wu X. MicroRNA-Related Genetic Variations as Predictors for Risk of Second Primary Tumor and/or Recurrence in Patients with Early-Stage Head and Neck Cancer. Carcinogenesis. 2010;31:2118–2123. doi: 10.1093/carcin/bgq177.  [DOI](https://doi.org/10.1093/carcin/bgq177) | [PMC free article](/articles/PMC3105587/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20819778/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Carcinogenesis&title=MicroRNA-Related%20Genetic%20Variations%20as%20Predictors%20for%20Risk%20of%20Second%20Primary%20Tumor%20and/or%20Recurrence%20in%20Patients%20with%20Early-Stage%20Head%20and%20Neck%20Cancer&author=X.%20Zhang&author=H.%20Yang&author=J.J.%20Lee&author=E.%20Kim&author=S.M.%20Lippman&volume=31&publication_year=2010&pages=2118-2123&pmid=20819778&doi=10.1093/carcin/bgq177&)

32. Liang D., Meyer L., Chang D.W., Lin J., Pu X., Ye Y., Gu J., Wu X., Lu K. Genetic Variants in MicroRNA Biosynthesis Pathways and Binding Sites Modify Ovarian Cancer Risk, Survival, and Treatment Response. Cancer Res. 2010;70:9765–9776. doi: 10.1158/0008-5472.CAN-10-0130.  [DOI](https://doi.org/10.1158/0008-5472.CAN-10-0130) | [PMC free article](/articles/PMC5373101/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21118967/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res.&title=Genetic%20Variants%20in%20MicroRNA%20Biosynthesis%20Pathways%20and%20Binding%20Sites%20Modify%20Ovarian%20Cancer%20Risk,%20Survival,%20and%20Treatment%20Response&author=D.%20Liang&author=L.%20Meyer&author=D.W.%20Chang&author=J.%20Lin&author=X.%20Pu&volume=70&publication_year=2010&pages=9765-9776&pmid=21118967&doi=10.1158/0008-5472.CAN-10-0130&)

33. Gelmez M.Y., Coskunpinar E., Saracoglu B., Deniz G., Aktan M. Investigation of AID, Dicer, and Drosha Expressions in Patients with Chronic Lymphocytic Leukemia. Immunol. Invest. 2017;46:433–446. doi: 10.1080/08820139.2017.1288241.  [DOI](https://doi.org/10.1080/08820139.2017.1288241) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28388279/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Immunol.%20Invest.&title=Investigation%20of%20AID,%20Dicer,%20and%20Drosha%20Expressions%20in%20Patients%20with%20Chronic%20Lymphocytic%20Leukemia&author=M.Y.%20Gelmez&author=E.%20Coskunpinar&author=B.%20Saracoglu&author=G.%20Deniz&author=M.%20Aktan&volume=46&publication_year=2017&pages=433-446&pmid=28388279&doi=10.1080/08820139.2017.1288241&)

34. Czubak K., Lewandowska M.A., Klonowska K., Roszkowski K., Kowalewski J., Figlerowicz M., Kozlowski P. High Copy Number Variation of Cancer-Related microRNA Genes and Frequent Amplification of DICER1 and DROSHA in Lung Cancer. Oncotarget. 2015;6:23399–23416. doi: 10.18632/oncotarget.4351.  [DOI](https://doi.org/10.18632/oncotarget.4351) | [PMC free article](/articles/PMC4695126/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26156018/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncotarget&title=High%20Copy%20Number%20Variation%20of%20Cancer-Related%20microRNA%20Genes%20and%20Frequent%20Amplification%20of%20DICER1%20and%20DROSHA%20in%20Lung%20Cancer&author=K.%20Czubak&author=M.A.%20Lewandowska&author=K.%20Klonowska&author=K.%20Roszkowski&author=J.%20Kowalewski&volume=6&publication_year=2015&pages=23399-23416&pmid=26156018&doi=10.18632/oncotarget.4351&)

35. Nguyen L.X.T., Zhang B., Hoang D.H., Zhao D., Wang H., Wu H., Su Y.-L., Dong H., Rodriguez-Rodriguez S., Armstrong B., et al. Cytoplasmic DROSHA and Non-Canonical Mechanisms of MiR-155 Biogenesis in FLT3-ITD Acute Myeloid Leukemia. Leukemia. 2021;35:2285–2298. doi: 10.1038/s41375-021-01166-9.  [DOI](https://doi.org/10.1038/s41375-021-01166-9) | [PMC free article](/articles/PMC8973317/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33589748/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Leukemia&title=Cytoplasmic%20DROSHA%20and%20Non-Canonical%20Mechanisms%20of%20MiR-155%20Biogenesis%20in%20FLT3-ITD%20Acute%20Myeloid%20Leukemia&author=L.X.T.%20Nguyen&author=B.%20Zhang&author=D.H.%20Hoang&author=D.%20Zhao&author=H.%20Wang&volume=35&publication_year=2021&pages=2285-2298&pmid=33589748&doi=10.1038/s41375-021-01166-9&)

36. Kyriakidis I., Pelagiadis I., Stiakaki E. Pre-Analytical Considerations for microRNA Quantification in Childhood Leukemia Research. J. Biol. Methods. 2025;12:e99010062. doi: 10.14440/jbm.2025.0001.  [DOI](https://doi.org/10.14440/jbm.2025.0001) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Methods&title=Pre-Analytical%20Considerations%20for%20microRNA%20Quantification%20in%20Childhood%20Leukemia%20Research&author=I.%20Kyriakidis&author=I.%20Pelagiadis&author=E.%20Stiakaki&volume=12&publication_year=2025&pages=e99010062&doi=10.14440/jbm.2025.0001&)

37. Kyriakidis I., Pelagiadis I., Katzilakis N., Stratigaki M., Keklikoglou I., Tragiannidis A., Stiakaki E. DICER1 Rs3742330 and AGO1 Rs636832 Polymorphisms and Acute Lymphoblastic Leukemia in Greek Children and Adolescents: A Case-Control Study. Gene Rep. 2024;37:102043. doi: 10.1016/j.genrep.2024.102043.  [DOI](https://doi.org/10.1016/j.genrep.2024.102043) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gene%20Rep.&title=DICER1%20Rs3742330%20and%20AGO1%20Rs636832%20Polymorphisms%20and%20Acute%20Lymphoblastic%20Leukemia%20in%20Greek%20Children%20and%20Adolescents:%20A%20Case-Control%20Study&author=I.%20Kyriakidis&author=I.%20Pelagiadis&author=N.%20Katzilakis&author=M.%20Stratigaki&author=I.%20Keklikoglou&volume=37&publication_year=2024&pages=102043&doi=10.1016/j.genrep.2024.102043&)
