# Effect of the NFIB rs28379954 T>C polymorphism on CYP2D6‐catalyzed metabolism of solanidine

## Metadata
**Authors:** Robert Løvsletten Smith, Birgit M Wollmann, Elisabet Størset, Hasan Çağın Lenk, Kevin S O'Connell, Marianne Kringen Kristiansen, Magnus Ingelman‐Sundberg, Espen Molden
**Journal:** Clinical and Translational Science
**Date:** 2024 Feb 22
**DOI:** [10.1111/cts.13743](https://doi.org/10.1111/cts.13743)
**PMID:** 38385986
**PMCID:** PMC10883345
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10883345/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC10883345/pdf/CTS-17-e13743.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC10883345/pdf/CTS-17-e13743.pdf)

## Abstract

Cytochrome P450 2D6 (CYP2D6) is important for metabolism of 20%–25% of all clinically used drugs. Many known genetic variants contribute to the large interindividual variability in CYP2D6 metabolism, but much is still unexplained. We recently described that nuclear factor 1B (NFIB) regulates hepatic CYP2D6 expression with the minor allele of NFIB rs28379954 T>C significantly increasing CYP2D6‐mediated risperidone metabolism. In this study, we investigated the effect of NFIB T>C on metabolism of solanidine, a dietary CYP2D6 substrate. Analyses of solanidine and metabolites (M414, M416, and M444) were performed by ultra‐high performance liquid chromatography‐high‐resolution mass spectrometry in a cohort of 463 CYP2D6‐genotyped patients of which with 58 (12.5%) carried NFIB TC (n = 56) or CC (n = 2). Increased metabolism of solanidine was found in CYP2D6 normal metabolizers (NMs; n = 258, 55.7%) carrying the NFIB C variant (n = 27, 5.8%) with 2.83‐ and 3.38‐fold higher M416‐to‐solanidine (p = 0.039) and M444‐to‐solanidine (p = 0.046) ratios, respectively, whereas this effect was not significant among intermediate metabolizers (n = 166, 35.9%) (p ≥ 0.09). Importantly, no effect of the NFIB polymorphism on solanidine metabolism was seen in TC or CC carriers lacking CYP2D6 activity (poor metabolizers, n = 30, 6.5%, p ≥ 0.74). Furthermore, the NFIB polymorphism significantly explained variability in solanidine metabolism (M414 p = 0.013, M416 p = 0.020, and M416 and M444 p = 0.009) in multiple linear regression models for each metabolic ratio in the entire population, correcting for covariates (including CYP2D6 genotypes). Thus, the study confirms the effect of NFIB in regulating CYP2D6 activity, suggesting an about 200% increase in CYP2D6‐mediated clearance in NMs being NFIB CT or CC carriers, comprising around 6% of Europeans.

The interindividual variability in CYP2D6 activity is high, particularly in patients who are classified as *CYP2D6* normal metabolizers (NMs). Recently, it was showed that CYP2D6 NM patients carrying the *NFIB rs28379954 C* variant have increased risperidone metabolism, but studies on additional CYP2D6 substrates are necessary to strengthen the evidence of an association between *NFIB* genotype and CYP2D6 metabolism, as well as estimate the effect size of the *NFIB* C variant on CYP2D6 clearance.

We investigated the effect of the *NFIB* rs28379954 *T*>*C* polymorphism on metabolism of the dietary CYP2D6 biomarker solanidine in patients stratified by *CYP2D6* genotype.

We found that CYP2D6 NMs carrying *NFIB C* had significantly increased CYP2D6‐mediated metabolism of solanidine by around 200%, whereas no significant effect of *NFIB* genotype was seen in CYP2D6 intermediate metabolizers and poor metabolizers (PMs). The significant effect in NMs and absent effect in PMs provided conclusive evidence that *NFIB* modulates CYP2D6 activity. Further studies should investigate the clinical impact of the increased metabolism in CYP2D6 NMs carrying *NFIB TC* or *CC* genotypes, who comprise around 6% of Europeans.

The study contributes with increased understanding of genetic factors causing the extensive variability in enzyme activity among CYP2D6 NMs assigned by *CYP2D6* genotype. Furthermore, it opens for new studies investigating the clinical impact of increased metabolism in CYP2D6 NMs carrying the *NFIB TC* or *CC* genotype. The studies should investigate the potential importance of the *NFIB T*>*C* polymorphism for precision dosing of CYP2D6 drugs in CYP2D6 NMs as well as clinical outcomes.

## INTRODUCTION

CYP2D6 is a highly polymorphic enzyme involved in the metabolism of a great variety of therapeutic drugs.[1](#cts13743-bib-0001) Knowledge of the *CYP2D6* genotype is used for prediction of drug clearance and optimal dosing in vivo. However, the interindividual variability in CYP2D6 activity is very high between patients having the same *CYP2D6* genotype‐predicted phenotype.[2](#cts13743-bib-0002), [3](#cts13743-bib-0003), [4](#cts13743-bib-0004) This indicates the influence of other, unknown genetic variants affecting CYP2D6‐mediated drug metabolism.

Recently, we showed that nuclear factor 1B (NFIB) plays a role in the regulation of several drug‐metabolizing enzymes, including cytochrome P450 1A2 (CYP1A2) and 2D6 (CYP2D6) as well UGTs and GST in human liver spheroids.[5](#cts13743-bib-0005) Furthermore, we found that CYP2D6 normal metabolizers (NMs) carrying the *C* variant allele of the *NFIB* rs28379954*T*>*C* polymorphism exhibited significantly increased metabolism of the CYP2D6 biomarker drug risperidone compared to *NFIB TT* carriers, but this effect was importantly not observed in CYP2D6 poor metabolizers (PMs).[5](#cts13743-bib-0005) Transfection of in vitro reporter constructs encompassing −1915 to −413 bp of the 5′ upstream *CYP2D6* sequences in Huh7 cells suggested that NFIB appears to cluster with proteins interfering with a regulatory element −413 to −534 bp upstream of the *CYP2D6* gene thereby inhibiting gene expression.[5](#cts13743-bib-0005) In 3D liver spheroids, we found that the variant form of the *NFIB* gene, *NFIB C*, causes less expression of nuclear NFIB in hepatocytes, which would then result in less NFIB‐mediated lowering of CYP2D6 expression and increased rate of CYP2D6 metabolism.[5](#cts13743-bib-0005)

Although these novel findings on risperidone metabolism provided evidence for the role of NFIB in regulating CYP2D6 metabolism, data on additional substrates are needed to reproduce the NFIB effect on CYP2D6 metabolism. Recent studies from Magliocco et al.[6](#cts13743-bib-0006) and Behrle et al.[7](#cts13743-bib-0007) identified solanidine and metabolites, originating from potatoes (*S. tuberosum*),[6](#cts13743-bib-0006), [8](#cts13743-bib-0008), [9](#cts13743-bib-0009) as promising dietary biomarkers of CYP2D6 activity. After potato intake, solanidine is present in human serum in nanomolar (nM) concentrations up to 3 weeks.[10](#cts13743-bib-0010) The aim of the present study was to investigate the effect of the *NFIB C* variant on solanidine metabolism in a large population stratified for *CYP2D6* genotype.

## METHODS

### Subjects

This study included *CYP2D6*‐genotyped patients, who had performed routine pharmacogenetic testing and therapeutic drug monitoring (TDM) of psychiatric drug(s) at the Center for Psychopharmacology, Diakonhjemmet Hospital (Oslo, Norway), during the period March 2019 to April 2022. The patients partly overlapped with previous studies investigating the impact of *NFIB* genotype on risperidone and clozapine metabolism.[5](#cts13743-bib-0005), [11](#cts13743-bib-0011)

Information about *CYP2D6* genotypes was retrieved from the laboratory database, whereas *NFIB* genotype was carried out using remaining DNA samples from the original routine analyses. Measurements of solanidine and its metabolites were drawn from existing high‐resolution mass spectrometry (HRMS) data files of TDM measurements. Exclusion criteria were age less than 18 or greater than or equal to 65 years and co‐medicated with the CYP2D6 inhibitors bupropion, fluoxetine, paroxetine, or levomepromazine. Because the routine genotyping assays could not discriminate among duplication *CYP2D6*1*, *CYP2D6*4*, or *CYP2D6*41*, patients with change in copy number with presence of *CYP2D6*4* or *CYP2D6*41* were excluded.

The use of historical data in this study was approved by the Regional Committee for Medical and Health Research Ethics and the Hospital Investigational Review Board.

### CYP2D6 and NFIB rs28379954 T>C genotyping

Although ethnicity is not allowed to register when performing laboratory analysis in Norway, most inhabitants are of European Ancestry. The routine panel for *CYP2D6* genotyping therefore comprises the most relevant *CYP2D6* variants found in Europeans. Analyses of *CYP2D6* variant alleles were performed using TaqMan‐based real‐time polymerase chain reaction (PCR) assays implemented and validated for routine pharmacogenetic testing at the Center for Psychopharmacology, Diakonhjemmet Hospital, Oslo, Norway, described in detail elsewhere.[12](#cts13743-bib-0012) The *CYP2D6* pharmacogenetic panel includes (i) non‐functional (*Null*) alleles *CYP2D6*3*, *CYP2D6*4*, *CYP2D6*6*, (ii) decreased function (*Decr*) alleles *CYP2D6*9*, *CYP2D6*10*, and *CYP2D6*41*, as well as (iii) copy number analysis to identify *CYP2D6*5* (*Null*, whole gene deletion) allele and duplication of functional alleles (*CYP2D6*1/*1x2*). Absence of detected variants in the routine pharmacogenetic analyses was interpreted as wild type (*CYP2D6*1*). Copy number variation was analyzed using a predesigned TaqMan assay in exon 9 of *CYP2D6* (Thermo Fisher Scientific) with RnaseP as reference assay, which is present on two copies in humans. All the analyses of the *CYP2D6* genotyping panel, where data were included from, are accredited for clinical use.

The patients were categorized according to the Clinical Pharmacogenetics Implementation Consortium (CPIC) guidelines: CYP2D6 ultrarapid metabolizers (UMs; *CYP2D6*1*1x2*), CYP2D6 NMs (*CYP2D6*1/*1 or CYP2D6*1/Decr*), intermediate metabolizers (IMs; *CYP2D6*1/Null*, *CYP2D6Decr/Decr* or *CYP2D6Decr/Null*), or PMs (*CYP2D6Null/Null*).[13](#cts13743-bib-0013) As some *CYP2D6* variants, such as *CYP2D6*2*, *CYP2D6*17*, and *CYP2D6*35*, were not included in the routine *CYP2D6* pharmacogenetic panel, this may for some patients affect the accuracy of assignments to CYP2D6 IM and NM groups.

Genotyping of the *NFIB* rs28379954 *T>C* polymorphism is described in detail elsewhere.[11](#cts13743-bib-0011) Briefly, it was performed in a similar way as for *CYP2D6* genotyping with TaqMan single‐nucleotide polymorphism genotyping assay from Thermo Fisher Scientific (C_59359617_10). The PCR assays were run on QuantStudio12K Flex or QuantStudio Dx Real‐Time PCR Systems (Thermo Fischer Scientific).

### Ultra‐high performance liquid chromatography‐HRMS assay

The serum samples had previously been analyzed on an ultra‐high performance liquid chromatography‐HRMS instrument in routine TDM of any psychiatric drugs,[14](#cts13743-bib-0014) but, in this case, applied for identification and quantification of solanidine and metabolites through retrospective reprocessing of the non‐selective full scan HRMS data, as described in detail elsewhere.[8](#cts13743-bib-0008), [9](#cts13743-bib-0009) Solanidine and metabolites (i.e., M414, M416, and M444) were identified in the HRMS data files by accurate mass (*m/z*, ±5 ppm) detection at the fourth decimal and isotope ratio. The identity of solanidine was confirmed using retention time and matched tandem mass spectrometry spectrum by analyzing a reference standard purchased from Phytolab (Vestenbergsgreuth, Germany). TraceFinder 5.1 (Thermo Fisher Scientific) was used for data processing. All peaks were integrated automatically. Undetectable levels of solanidine and solanidine metabolites were truncated to the lower limit of detection (LLOD; i.e., M414, 1564; M416, 2041; M444, 1257; and solanidine, 2556) found for the corresponding analyte to enable proper calculations of metabolic ratios.

### Outcome measure and statistical analyses

The outcome measures were the various CYP2D6‐determined metabolite‐to‐solanidine ratios (MRs) in *NFIB TC/CC* versus *TT* carriers. The MRs were ln‐transformed prior to statistical analyses. Undetectable levels of metabolites and solanidine were truncated to the LLOD to enable proper calculations of MRs. Samples with undetectable serum levels of solanidine and metabolite(s) were excluded from the corresponding analysis. Initially, the various MRs distributions were compared between *NFIB TC/CC* versus *TT* carriers within each of the specified CYP2D6 phenotype subgroups using Student's *t*‐test. The 95% confidence intervals were calculated for each of the solanidine metabolic ratios to compare distributions of *NFIB C* versus *TT* carriers within each CYP2D6 group. Mean estimates of metabolite ratios for the respective NFIB subgroups were back transformed to describe the quantitative differences in solanidine metabolism between *NFIB TC/CC* and *TT* carriers in the various CYP2D6 groups. CYP2D6 UMs were left out of the statistical analyses due to small sample size. Subsequently, multiple linear regression analyses using ln‐transformed MRs were carried out to evaluate the significance degree of *NFIB* genotype explaining the overall interindividual variability in the various solanine MRs when including age, sex, and CYP2D6 phenotype (NM, IM, PM, or UM) as covariates.

All statistical analyses were performed in SPSS, version 27.0 (IBM SPSS). GraphPad version 9 (GraphPad Software) was used for graphical presentations. The significance level was set to 0.05. Unstandardized beta (B) values with standard errors (SEs) were back transformed to linear scales in the results presentation to illustrate the quantitative effects of *NFIB* genotype on the various solanidine metabolic ratios.

## RESULTS

Initially, 608 patients were screened for study inclusion. After excluding patients with inconclusive *CYP2D6* genotypes (i.e., possible duplication of null or decreased‐function alleles, *n* = 15), use of CYP2D6 inhibitors (*n* = 40), or age less than 18 or greater than or equal to 64 years (*n* = 90), a total of 463 subjects (59.4% men, mean age = 41.4 years, SD: 11.2) were included. In line with expected values in Europeans, the frequencies of genotype predicted CYP2D6 UMs, NMs, IMs, and PMs were 1.9% (*n* = 9), 56% (*n* = 258), 36% (*n* = 166), and 6.5% (*n* = 30), respectively. Among the patients, 56 heterozygous and two homozygous *NFIB C* carriers were identified. Thus, 12.1% of the population carried the *NFIB C* variant, whereof 27 were CYP2D6 NMs (5.8%; Table [S1](#cts13743-supitem-0001)).

The sensitivity of detecting solanidine and the metabolites in the serum samples varied from 97.8% (M414) to 91.1% (M444). There were no significant differences in age, sex, or proportions of undetectable levels of solanidine and metabolites in *NFIB C* versus *TT* carriers (*p* ≥ 0.19). Figure [1](#cts13743-fig-0001) shows the distributions of solanidine MRs in *NFIB TC/CC* versus *TT* carriers within each CYP2D6 subgroup. Significantly increased M416‐ (*p* = 0.039) and M444‐to‐solanidine (*p* = 0.046) metabolic ratios were observed for *NFIB C* versus *TT* carriers within CYP2D6 NMs, with 2.84‐ and 3.37‐fold increased formations of M416 and M444, respectively (Figure [1](#cts13743-fig-0001); Table [S2](#cts13743-supitem-0002)). No statistically significant differences were found between *NFIB TC/CC* and *TT* carriers within CYP2D6 IMs or PMs, but a tendency of increased formation of M414 was observed in IMs carrying *NFIB C* (*p* = 0.089). The lack of significant effect of *NFIB C* in CYP2D6 PMs (*p* ≥ 0.35), where enzyme activity is absent, provided strong evidence that *NFIB* genotype impacts CYP2D6 metabolism. Pairwise comparisons of solanidine MRs in *NFIB TC/CC* versus *TT* carriers in CYP2D6 UMs could not be performed due to the small sample size (Figure [1](#cts13743-fig-0001)).

### FIGURE 1.

![FIGURE 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1124/10883345/7d52e18aeb62/CTS-17-e13743-g001.jpg)

Distributions of (a) M444‐, (b) M414‐, and (c) M416‐to‐solanidine metabolic ratios in NFIB C (red dots) versus TT carriers (black dots) among various CYP2D6 phenotype subgroups. The two blue‐square dots represent the two homozygous NFIB C carriers detected in the population. Horizontal lines represent mean values of the respective metabolic ratios (±95% confidence interval). Statistical analyses were performed by Student's t‐test comparing NFIB C versus TT carriers within each CYP2D6 phenotype subgroups. IM intermediate metabolizers; NM, normal metabolizers; PM, poor metabolizers; UM, ultra‐rapid metabolizers.

In multiple linear regression analyses, *NFIB TC* and *CC* carriers significantly explaining variabilities of all solanidine MRs, that is, *p* = 0.013 for M414, *p* = 0.020 for M416, and *p* = 0.009 for M444 (Table [1](#cts13743-tbl-0001)), supporting the effect of NFIB on the regulation of CYP2D6 metabolism. As expected, the CYP2D6 PM phenotype was the strongest predictor of solanidine MRs (Table [1](#cts13743-tbl-0001)). In the multivariate analysis, including CYP2D6 PMs, *NFIB* genotype represented 2.7% of the model's overall explained variability of the M444 ratio (most sensitive to *NFIB C*). When the analysis was performed without CYP2D6 PMs, *NFIB* genotype represented 15.4% of the model's overall explained variability (data not shown).

### TABLE 1.

| Variables | M414‐to‐solanidine ratio |  | M416‐to‐solanidine ratio |  | M444‐to‐solanidine ratio |  |
| --- | --- | --- | --- | --- | --- | --- |
| B (SE) | p | B (SE) | p | B (SE) | p |  |
| No. of patients | 445 |  | 440 |  | 422 |  |
| Intercept | 2.86 (0.14) | <0.001 | 2.62 (0.22) | <0.001 | 0.73 (0.26) | 0.006 |
| NFIB TC or CC | 0.57 (0.23) | 0.013 | 0.83 (0.36) | 0.020 | 1.12 (0.43) | 0.009 |
| Covariates |  |  |  |  |  |  |
| Age a | 0.002 (0.007) | 0.74 | 0.008 (0.010) | 0.43 | 0.005 (0.013) | 0.66 |
| Male sex | 0.14 (0.15) | 0.35 | 0.18 (0.24) | 0.46 | 0.28 (0.29) | 0.33 |
| CYP2D6 IM | −0.89 (0.16) | <0.001 | −1.71 (0.25) | <0.001 | −2.38 (0.30) | <0.001 |
| CYP2D6 PM | −10.2 (0.31) | <0.001 | −9.88 (0.47) | <0.001 | −8.67 (0.56) | <0.001 |
| Model R 2 | 0.722 |  | 0.513 |  | 0.398 |  |

Table 1 Caption: The effects of NFIB rs28379954 T>C allele (NFIB C) carriers on M414‐, M416‐, and M444‐to‐solanidine ratios with age, sex, and CYP2D6 phenotype subgroups as covariates.

## DISCUSSION

The present study shows that the *NFIB C* variant is significantly associated with increased CYP2D6 metabolism of solanidine. A significant effect on solanidine metabolism was only seen in CYP2D6 NMs, which is compatible with the findings that the *NFIB C* variant modulates *CYP2D6* gene expression and activity, which cannot occur in CYP2D6 PMs.

We recently showed that the *NFIB C* variant is associated with increased metabolism of risperidone,[5](#cts13743-bib-0005) another CYP2D6 substrate, where a significant effect of *NFIB* genotype was also exclusively seen in CYP2D6 NMs (no significant effect in IMs and PMs). Together, these findings on two different CYP2D6 substrates indicate that the effect of *NFIB* genotype on CYP2D6 metabolism is generic, and not substrate dependent. This supports that the *NFIB C* is a genetic marker of CYP2D6 metabolism, although studies with additional CYP2D6 substrates are still required.

An issue which needs to be considered when assessing the potential value of solanidine metabolism predicting exposure of CYP2D6 substrates, is the substantial interindividual variability within and between *NFIB* genotype subgroups. This variability reflects other factors, genetic and non‐genetic, affecting CYP2D6 metabolism. However, this issue is the same for *CYP2D6* genotype in predicting metabolizer phenotype, where the inter‐ and intra‐genotype variability in CYP2D6 metabolism is also extensive. Nevertheless, the significant associations of *NFIB* genotype with risperidone[5](#cts13743-bib-0005) and solanidine metabolism in CYP2D6 NMs contribute to the understanding of the unexplained metabolic variability of this highly polymorphic enzyme.

In the present study, which mainly included individuals of European ancestry, the overall proportion of patients carrying the *NFIB C* variant was 12.1%, and as much as 5.8% were CYP2D6 NMs. Thus, this subgroup comprising individuals with increased CYP2D6 metabolism probably represents a substantial part of White populations. In other ethnic groups, frequencies of the minor rs28379954 *T>C* allele are low, ranging from 0.2% in Africans to 4% in South‐Asians.[15](#cts13743-bib-0015) Therefore, the *NFIB C* variant is mainly of relevance for individual variability in CYP2D6 metabolism among people of European ancestry.

*NFIB C* carriers showed significantly 2.84‐ (M416) to 3.37‐fold (M444) increased formation of metabolites in CYP2D6 NMs. This indicates an average increase in CYP2D6 activity of around 200% in CYP2D6 NMs carrying the *NFIB C* variant. An important point though is that the effect of the *NFIB C* variant among CYP2D6 NMs to a considerable extent was driven by absence of patients showing decreased CYP2D6 metabolism. This may indicate that the presence of *NFIB C* counteracts downregulation of CYP2D6 activity. However, as the overlaps in individual solanidine metabolic ratios were high between CYP2D6 NMs carrying versus not carrying *NFIB C*, the apparent power for classifying the former subgroup as “UM” is currently not sufficient. Furthermore, current evidence does not indicate whether the impact of the *NFIB C* variant on CYP2D6 metabolism is large enough to warrant prescription changes of CYP2D6 substrates, including risperidone.

A typical feature supported by the present study, is that generally the presence of more than two functional *CYP2D6* genes has less impact on serum or plasma concentrations of CYP2D6 substrates as compared to the difference between NMs and IMs. This has been shown in studies on several psychiatric drugs, where, however, a significantly increased risk of therapeutic failure in CYP2D6 UMs versus NMs is still observed.[2](#cts13743-bib-0002), [4](#cts13743-bib-0004), [16](#cts13743-bib-0016) Additionally, a noticeable distinction exists in the discriminatory capacity between the two substrates here used for identifying CYP2D6 NMs and UMs. Specifically, solanidine metabolic ratios exhibit a greater ability to differentiate individuals with gene variants associated with decreased metabolism compared to those with higher activity variants. Conversely, risperidone proves to be more effective in distinguishing subjects carrying increased activity CYP2D6 variants than solanidine.[2](#cts13743-bib-0002), [8](#cts13743-bib-0008) Additional studies on other CYP2D6 substrates than risperidone and solanidine are necessary to highlight this issue, and it would also be valuable to replicate the findings of *NFIB C* on risperidone and solanidine metabolism in other cohorts.

The naturalistic nature of the current study is associated with some limitations, such as missing information about comorbidity and organ function that may have altered CYP2D6 activity. Further, the amount of potato intake was unknown, and differences in time after solanidine consumption probably contributed to some of the variability in solanidine MRs beyond that attributable to differences in CYP2D6 activity. However, the solanidine MRs are expected to be relatively stable over time due to solanidine's long elimination half‐life.[10](#cts13743-bib-0010) Another potential issue worth mentioning is that the routine *CYP2D6* panel at the point of data collection only included the most relevant variants present in Norwegians, which was the basis for CYP2D6 subgroups assignments of the patients. If the panel had included variants such as *CYP2D6*2*, *CYP2D6*17*, and *CYP2D6*35*, which are now being implemented, this would have increased the precision of CYP2D6 phenotype subgroup assignments. Furthermore, lacking information on potential disease factors and unknown *CYP2D6* variant alleles may have influenced the variability in solanidine metabolic ratios. However, the ability to include a large number of subjects allowed for robust comparisons between *NFIB TC* or *CC* versus *TT* carriers within the CYP2D6 subgroups, except from CYP2D6 UMs. An additional strength was the ability to exclude patients using strong CYP2D6 inhibitors reducing bias of the main non‐genetic factor causing variability in CYP2D6 metabolism in naturalistic settings. Regarding significant *p* values from univariate tests and potential adjustment for multiple testing, the additional multiple linear regression models for each solanidine metabolic ratio provided very low *p* values of the *NFIB* genotype in explaining variability in solanidine metabolism, even when including CYP2D6 PMs in the models. Thus, the overall evidence for the associations between *NFIB TC* or *CC* and CYP2D6 metabolism is robust.

In conclusion, the present study shows a significant association between the *NFIB C* allele variant and metabolism of solanidine, a dietary CYP2D6 biomarker, in CYP2D6 NMs. *NFIB* genotype had no effect in CYP2D6 PMs, which is rational because CYP2D6 activity is absent in these patients. The estimated increase in CYP2D6 activity in heterozygous or homozygous carriers of the *NFIB C* variant versus the *TT* genotype is around 200% in CYP2D6 NMs. Individuals who are CYP2D6 NMs and carry the *NFIB C* comprise around 6% of Europeans and represent a group where additional studies should be performed to clarify whether the impact of *NFIB C* on CYP2D6 metabolism is large enough to affect clinical response and warrant potential prescription changes of drugs being CYP2D6 substrates.

## AUTHOR CONTRIBUTIONS

R.L.S. and E.M. wrote the manuscript with input from all co‐authors. R.L.S., B.M.W., H.C.L., K.O., M.K.K., M.I.‐S., E.S., and E.M. designed the research. R.L.S. and B.M.W. performed the research. R.L.S. and B.M.W. analyzed the data. B.M.W and M.K.K. contributed with analytical tools.

## FUNDING INFORMATION

This work was financially supported by the European Union's Horizon 2020 research and innovation program (Grant Agreement 964874/REALMENT) and by the South‐Eastern Norway Regional Health Authority (grant no. 2020019). E.S. and M.I.‐S. received funding from the Swedish Research Council (grant no. 2021‐02732).

## CONFLICT OF INTEREST STATEMENT

M.I.‐S. is a co‐founder and co‐owner of HepaPredict AB. All other authors declared no competing interests for this work.

## Supporting information

## ACKNOWLEDGMENTS

The authors would like to thank the laboratory technicians at Center for Psychopharmacology, Diakonhjemmet Hospital, Oslo, Norway, for helping during data collection and preparation.

Smith RL, Wollmann BM, Størset E, et al. Effect of the *NFIB* rs28379954 *T*>*C* polymorphism on CYP2D6‐catalyzed metabolism of solanidine. Clin Transl Sci. 2024;17:e13743. doi: 10.1111/cts.13743

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Kehinde O, Ramsey LB, Gaedigk A, Oni‐Orisan A. Advancing CYP2D6 pharmacogenetics through a pharmacoequity lens. Clin Pharmacol Ther. 2023;114:69‐76.  [DOI](https://doi.org/10.1002/cpt.2890) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36924260/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Advancing%20CYP2D6%20pharmacogenetics%20through%20a%20pharmacoequity%20lens&author=O%20Kehinde&author=LB%20Ramsey&author=A%20Gaedigk&author=A%20Oni%E2%80%90Orisan&volume=114&publication_year=2023&pages=69-76&pmid=36924260&doi=10.1002/cpt.2890&)

2. Jukic MM, Smith RL, Haslemo T, Molden E, Ingelman‐Sundberg M. Effect of CYP2D6 genotype on exposure and efficacy of risperidone and aripiprazole: a retrospective, cohort study. Lancet Psychiatry. 2019;6:418‐426.  [DOI](https://doi.org/10.1016/S2215-0366(19)30088-4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31000417/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet%20Psychiatry&title=Effect%20of%20CYP2D6%20genotype%20on%20exposure%20and%20efficacy%20of%20risperidone%20and%20aripiprazole:%20a%20retrospective,%20cohort%20study&author=MM%20Jukic&author=RL%20Smith&author=T%20Haslemo&author=E%20Molden&author=M%20Ingelman%E2%80%90Sundberg&volume=6&publication_year=2019&pages=418-426&pmid=31000417&doi=10.1016/S2215-0366(19)30088-4&)

3. Jukic MM, Smith RL, Molden E, Ingelman‐Sundberg M. Evaluation of the CYP2D6 haplotype activity scores based on metabolic ratios of 4,700 patients treated with three different CYP2D6 substrates. Clin Pharmacol Ther. 2021;110:750‐758.  [DOI](https://doi.org/10.1002/cpt.2246) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33792048/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Evaluation%20of%20the%20CYP2D6%20haplotype%20activity%20scores%20based%20on%20metabolic%20ratios%20of%204,700%20patients%20treated%20with%20three%20different%20CYP2D6%20substrates&author=MM%20Jukic&author=RL%20Smith&author=E%20Molden&author=M%20Ingelman%E2%80%90Sundberg&volume=110&publication_year=2021&pages=750-758&pmid=33792048&doi=10.1002/cpt.2246&)

4. Frederiksen T, Smith RL, Jukic MM, Molden E. Association between CYP2D6 metabolizer status and vortioxetine exposure and treatment switching: a retrospective, naturalistic cohort study using therapeutic drug monitoring data from 640 patients. J Clin Psychopharmacol. 2022;42:396‐399.  [DOI](https://doi.org/10.1097/JCP.0000000000001563) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35703273/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Psychopharmacol&title=Association%20between%20CYP2D6%20metabolizer%20status%20and%20vortioxetine%20exposure%20and%20treatment%20switching:%20a%20retrospective,%20naturalistic%20cohort%20study%20using%20therapeutic%20drug%20monitoring%20data%20from%20640%20patients&author=T%20Frederiksen&author=RL%20Smith&author=MM%20Jukic&author=E%20Molden&volume=42&publication_year=2022&pages=396-399&pmid=35703273&doi=10.1097/JCP.0000000000001563&)

5. Lenk HC, Klöditz K, Johansson I, et al. The polymorphic nuclear factor NFIB regulates hepatic CYP2D6 expression and influences risperidone metabolism in psychiatric patients. Clin Pharmacol Ther. 2022;111:1165‐1174.  [DOI](https://doi.org/10.1002/cpt.2571) | [PMC free article](/articles/PMC9314634/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35253216/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=The%20polymorphic%20nuclear%20factor%20NFIB%20regulates%20hepatic%20CYP2D6%20expression%20and%20influences%20risperidone%20metabolism%20in%20psychiatric%20patients&author=HC%20Lenk&author=K%20Kl%C3%B6ditz&author=I%20Johansson&volume=111&publication_year=2022&pages=1165-1174&pmid=35253216&doi=10.1002/cpt.2571&)

6. Magliocco G, Desmeules J, Matthey A, et al. Metabolomics reveals biomarkers in human urine and plasma to predict cytochrome P450 2D6 (CYP2D6) activity. Br J Pharmacol. 2021;178:4708‐4725.  [DOI](https://doi.org/10.1111/bph.15651) | [PMC free article](/articles/PMC9290485/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34363609/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Pharmacol&title=Metabolomics%20reveals%20biomarkers%20in%20human%20urine%20and%20plasma%20to%20predict%20cytochrome%20P450%202D6%20(CYP2D6)%20activity&author=G%20Magliocco&author=J%20Desmeules&author=A%20Matthey&volume=178&publication_year=2021&pages=4708-4725&pmid=34363609&doi=10.1111/bph.15651&)

7. Behrle AC, Douglas J, Leeder JS, van Haandel L. Isolation and identification of 3,4‐seco‐solanidine‐3,4‐dioic acid (SSDA) as a urinary biomarker of cytochrome P450 2D6 (CYP2D6) activity. Drug Metab Dispos. 2022;50:1342‐1351.  [DOI](https://doi.org/10.1124/dmd.122.000957) | [PMC free article](/articles/PMC9513856/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35878926/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Isolation%20and%20identification%20of%203,4%E2%80%90seco%E2%80%90solanidine%E2%80%903,4%E2%80%90dioic%20acid%20(SSDA)%20as%20a%20urinary%20biomarker%20of%20cytochrome%20P450%202D6%20(CYP2D6)%20activity&author=AC%20Behrle&author=J%20Douglas&author=JS%20Leeder&author=L%20van%20Haandel&volume=50&publication_year=2022&pages=1342-1351&pmid=35878926&doi=10.1124/dmd.122.000957&)

8. Wollmann B, Smith RL, Kringen MK, Ingelman‐Sundberg M, Molden E, Storset E. Evidence for solanidine as a dietary CYP2D6 biomarker: significant correlation with risperidone metabolism. Br J Clin Pharmacol. 2023. Online ahead of print. doi: 10.1111/bcp.15721  [DOI](https://doi.org/10.1111/bcp.15721) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36960588/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=Evidence%20for%20solanidine%20as%20a%20dietary%20CYP2D6%20biomarker:%20significant%20correlation%20with%20risperidone%20metabolism&author=B%20Wollmann&author=RL%20Smith&author=MK%20Kringen&author=M%20Ingelman%E2%80%90Sundberg&author=E%20Molden&publication_year=2023&pmid=36960588&doi=10.1111/bcp.15721&)

9. Wollmann BM, Storset E, Kringen MK, Molden E, Smith RL. Prediction of CYP2D6 poor metabolizers by measurements of solanidine and metabolites—a study in 839 patients with known CYP2D6 genotype. Eur J Clin Pharmacol. 2023;79:523‐531.  [DOI](https://doi.org/10.1007/s00228-023-03462-y) | [PMC free article](/articles/PMC10038974/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36806969/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Clin%20Pharmacol&title=Prediction%20of%20CYP2D6%20poor%20metabolizers%20by%20measurements%20of%20solanidine%20and%20metabolites%E2%80%94a%20study%20in%20839%20patients%20with%20known%20CYP2D6%20genotype&author=BM%20Wollmann&author=E%20Storset&author=MK%20Kringen&author=E%20Molden&author=RL%20Smith&volume=79&publication_year=2023&pages=523-531&pmid=36806969&doi=10.1007/s00228-023-03462-y&)

10. Harvey MH, Morris BA, McMillan M, Marks V. Measurement of potato steroidal alkaloids in human serum and saliva by radioimmunoassay. Hum Toxicol. 1985;4:503‐512.  [DOI](https://doi.org/10.1177/096032718500400506) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/4054913/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Toxicol&title=Measurement%20of%20potato%20steroidal%20alkaloids%20in%20human%20serum%20and%20saliva%20by%20radioimmunoassay&author=MH%20Harvey&author=BA%20Morris&author=M%20McMillan&author=V%20Marks&volume=4&publication_year=1985&pages=503-512&pmid=4054913&doi=10.1177/096032718500400506&)

11. Smith RL, O'Connell K, Athanasiu L, et al. Identification of a novel polymorphism associated with reduced clozapine concentration in schizophrenia patients‐a genome‐wide association study adjusting for smoking habits. Transl Psychiatry. 2020;10:198. doi: 10.1038/s41398-020-00888-1  [DOI](https://doi.org/10.1038/s41398-020-00888-1) | [PMC free article](/articles/PMC7303159/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32555152/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Transl%20Psychiatry&title=Identification%20of%20a%20novel%20polymorphism%20associated%20with%20reduced%20clozapine%20concentration%20in%20schizophrenia%20patients%E2%80%90a%20genome%E2%80%90wide%20association%20study%20adjusting%20for%20smoking%20habits&author=RL%20Smith&author=K%20O'Connell&author=L%20Athanasiu&volume=10&publication_year=2020&pages=198&pmid=32555152&doi=10.1038/s41398-020-00888-1&)

12. Rudberg I, Mohebi B, Hermann M, Refsum H, Molden E. Impact of the ultrarapid CYP2C19*17 allele on serum concentration of escitalopram in psychiatric patients. Clin Pharmacol Ther. 2008;83:322‐327.  [DOI](https://doi.org/10.1038/sj.clpt.6100291) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17625515/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Impact%20of%20the%20ultrarapid%20CYP2C19*17%20allele%20on%20serum%20concentration%20of%20escitalopram%20in%20psychiatric%20patients&author=I%20Rudberg&author=B%20Mohebi&author=M%20Hermann&author=H%20Refsum&author=E%20Molden&volume=83&publication_year=2008&pages=322-327&pmid=17625515&doi=10.1038/sj.clpt.6100291&)

13. Caudle KE, Sangkuhl K, Whirl‐Carrillo M, et al. Standardizing CYP2D6 genotype to phenotype translation: consensus recommendations from the clinical pharmacogenetics implementation consortium and Dutch pharmacogenetics working group. Clin Transl Sci. 2020;13:116‐124.  [DOI](https://doi.org/10.1111/cts.12692) | [PMC free article](/articles/PMC6951851/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31647186/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Transl%20Sci&title=Standardizing%20CYP2D6%20genotype%20to%20phenotype%20translation:%20consensus%20recommendations%20from%20the%20clinical%20pharmacogenetics%20implementation%20consortium%20and%20Dutch%20pharmacogenetics%20working%20group&author=KE%20Caudle&author=K%20Sangkuhl&author=M%20Whirl%E2%80%90Carrillo&volume=13&publication_year=2020&pages=116-124&pmid=31647186&doi=10.1111/cts.12692&)

14. Smith RL, Wollmann BM, Kylleso L, Tran TTA, Tveito M, Molden E. Effect of valproic acid on the metabolic Spectrum of clozapine in patients with schizophrenia. J Clin Psychopharmacol. 2022;42:43‐50.  [DOI](https://doi.org/10.1097/JCP.0000000000001507) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34928560/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Psychopharmacol&title=Effect%20of%20valproic%20acid%20on%20the%20metabolic%20Spectrum%20of%20clozapine%20in%20patients%20with%20schizophrenia&author=RL%20Smith&author=BM%20Wollmann&author=L%20Kylleso&author=TTA%20Tran&author=M%20Tveito&volume=42&publication_year=2022&pages=43-50&pmid=34928560&doi=10.1097/JCP.0000000000001507&)

15. NIH . National Library of Medicine. 2023. [cited 2023 17th October]. www.ncbi.nlm.nih.gov/snp  [www.ncbi.nlm.nih.gov/snp](http://www.ncbi.nlm.nih.gov/snp)

16. Smith RL, Molden E, Bernard JP. Effect of CYP2D6 and CYP2C19 genotypes on atomoxetine serum levels: a study based on therapeutic drug monitoring data. Br J Clin Pharmacol. 2023;89:2246‐2253.  [DOI](https://doi.org/10.1111/bcp.15706) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36851891/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=Effect%20of%20CYP2D6%20and%20CYP2C19%20genotypes%20on%20atomoxetine%20serum%20levels:%20a%20study%20based%20on%20therapeutic%20drug%20monitoring%20data&author=RL%20Smith&author=E%20Molden&author=JP%20Bernard&volume=89&publication_year=2023&pages=2246-2253&pmid=36851891&doi=10.1111/bcp.15706&)
