# The polygenic implication of clopidogrel responsiveness: Insights from platelet reactivity analysis and next-generation sequencing

## Metadata
**Authors:** Omar Echeverría, Mariana Angulo-Aguado, Ricardo Vela, Carlos Calderón-Ospina, Katherine Parra, Nora Contreras, Adrien Morel, Rodrigo Cabrera, Carlos Restrepo, Carolina Ramírez-Santana, Oscar Ortega-Recalde, Manuel Eduardo Rojas-Quintana, Luisa Murcia, Cristian Camilo Gaviria-Sabogal, Nattaly Valero, Dora Janeth Fonseca-Mendoza
**Journal:** PLOS ONE
**Date:** 2024 Jul 11
**DOI:** [10.1371/journal.pone.0306445](https://doi.org/10.1371/journal.pone.0306445)
**PMID:** 38991024
**PMCID:** PMC11239111
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11239111/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC11239111/pdf/pone.0306445.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC11239111/pdf/pone.0306445.pdf)

## Abstract

Clopidogrel is widely used worldwide as an antiplatelet therapy in patients with acute coronary disease. Genetic factors influence interindividual variability in response. Some studies have explored the polygenic contributions in the drug response, generating pharmacogenomic risk scores (PgxPRS). Importantly, these factors are less explored in underrepresented populations, such as Latin-American countries. Identifying patients at risk of high-on-treatment platelet reactivity (HTPR) is highly valuable in translational medicine. In this study we used a custom next-generation sequencing (NGS) panel composed of 91 single nucleotide polymorphisms (SNPs) and 28 genes related to clopidogrel metabolism, to analyze 70 patients with platelet reactivity values, assessed through closure time (CT). Our results demonstrated the association of SNPs with HTPR and non-HTPR, revealing the strongest associations with rs2286823 (OR: 5,0; 95% CI: 1,02–24,48; p: 0,03), rs2032582 (OR: 4,41; 95% CI: 1,20–16,12; p: 0,019), and rs1045642 (OR: 3,38; 95% CI: 0,96–11,9; p: 0,05). Bivariate regression analysis demonstrated the significant association of several SNPs with the CT value, a “surrogate” biomarker of clopidogrel response. Exploratory results from the LASSO regression model showed a high discriminatory capacity between HTPR and non-HTPR patients (AUC: 0,955), and the generated PgxPRS demonstrated a significant negative association between the risk score, CT value, and the condition of HTPR and non-HTPR. To our knowledge, our study addresses for the first time the analysis of the polygenic contribution in platelet reactivity using NGS and establishes PgxPRS derived from the LASSO model. Our results demonstrate the polygenic implication of clopidogrel response and offer insights applicable to the translational medicine of antiplatelet therapy in an understudied population.

## Introduction

Clopidogrel is an antiplatelet therapy widely prescribed worldwide for antiplatelet therapy in the prevention of atherothrombotic events in patients with coronary heart disease. Clopidogrel is a prodrug that requires biotransformation into an active metabolite by cytochrome P-450 (CYP-450) enzymes. The P2Y12 receptor is inhibited by the active metabolite of clopidogrel, playing a role in adenosine diphosphate (ADP)-induced platelet activation and aggregation [[1](#pone.0306445.ref001)]. Clopidogrel response can be evaluated by determining platelet reactivity through ADP-induced light transmittance aggregometry (LTA), the Verify Now P2Y12 assay, and the INNOVANCE PFA-200 system, which can discriminate between patients with high on-treatment platelet reactivity (HTPR) or low on-treatment platelet reactivity (LTPR) [[2](#pone.0306445.ref002)]. Importantly, there is significant interindividual variability in the response, with some patients experiencing HTPR and an increased risk of adverse effects [[3](#pone.0306445.ref003)].

The variability in clopidogrel efficacy has been linked to non-genetic factors such as age, sex, comorbidities, and drug-drug interactions, among others. Platelet response to clopidogrel is highly heritable h(2) = 0,73 and has been associated with molecular variants in genes related to drug pharmacokinetics, and pharmacodynamics [[4](#pone.0306445.ref004)]. The primary worldwide genetic determinant is the single nucleotide polymorphism (SNP) *CYP2C19*2* (rs4244285). Individuals carrying this loss-of-function allele convert clopidogrel to its active metabolite less efficiently, leading to reduced platelet inhibition. This variant, however, explains only about 12% of the variation in HTPR in Europeans, leaving a significant percentage of the variability in the response to clopidogrel unexplained [[5](#pone.0306445.ref005)].

Numerous studies have been conducted to identify genetic determinants of clopidogrel response, primarily through genome-wide association studies (GWAS) and case-control studies with specific variants, proposing candidate genes such as *CYP2C19*, *CES1*, *CYP2B6*, *CYP2C9*, *P2Y12*, *PON1*, among others [[5](#pone.0306445.ref005)–[7](#pone.0306445.ref007)]. These findings have demonstrated the polygenic effect on clopidogrel response raising proposals for pharmacogenomic polygenic risk scores (PgxRS) applicable to identifying patients with HTPR [[5](#pone.0306445.ref005), [8](#pone.0306445.ref008)]. Polygenic risk scores represent effective approaches to improve the precision of risk-based screening programs for disease, but they have been less explored in pharmacogenomics [[9](#pone.0306445.ref009)]. Most studies to date have predominantly focused on individuals of European ancestry, and, except for analyses conducted in Caribbean Hispanic populations, there remains a notable knowledge gap for the Latin-American population, including Colombia [[10](#pone.0306445.ref010)].

Pharmacogenomic analyses to identify molecular variants of interest have primarily been developed by analyzing common variants through SNP microarrays. However, recently, the implementation of next-generation sequencing (NGS) has demonstrated high versatility in identifying both rare and common variants that explain the missing heritability associated with drug response phenotypes [[11](#pone.0306445.ref011)]. In this study, we employed an innovative strategy by developing a custom panel for the analysis of 91 SNPs (derived from previous GWAS studies) and 28 genes associated with clopidogrel pharmacokinetics and pharmacodynamics through NGS.

To the best of our knowledge, our study is the first to utilize the LASSO regression model and construct a PgxPRS for discriminating patients with an inadequate response to clopidogrel, determined through platelet reactivity measured as closure time (CT). The exploratory LASSO model demonstrated a high discriminatory power (Area Under the Curve- AUC:0,955), and the PgxPRS score was statistically associated with clopidogrel response. Our results hold significant implications for translational medicine in the field of antiplatelet therapy.

## Materials and methods

### Patients

This study included 70 patients admitted to the Hospital Universitario Mayor-Méderi with a diagnosis of acute coronary syndrome and treated with a loading dose of 300 mg/75 mg of clopidogrel for at least seven consecutive days. Eligible patients, aged 18 years or older, were invited to participate in the study and provided written informed consent. Exclusion criteria comprised individuals receiving oral anticoagulants and glycoprotein IIa/IIb receptor inhibitors, hematocrit values below 25% or above 52%, platelet count <100x10^9^/L, creatinine >15 mg/dL, and clinical evidence of hepatic damage or platelet dysfunction.

Non-genetic variables, such as sex, body mass index, comorbidities (diabetes, hypertension, and dyslipidemia), smoking, alcohol consumption, type of acute coronary syndrome, type of intervention, stent usage, family history, use of CYP2C19 inhibitors, and inducers, among others, were collected. The 70 patients in our study were enrolled from a previous study described by Angulo-Aguado et al. in 2021 [[12](#pone.0306445.ref012)]. For the current research, access was granted to DNA samples stored as of March 30, 2022. Throughout the project development, the anonymity of the samples was maintained, adhering to the initially assigned coding.

All experimental procedures were approved by the ethics committee of the Universidad del Rosario and followed the Helsinki Declaration guidelines (Approval DVO005 1785-CV1486).

### Platelet function test

As previously described [[12](#pone.0306445.ref012)], platelet function was evaluated using the INNOVANCE PFA-200 P2Y system (Siemens Healthcare). Blood samples were obtained four hours after the initial clopidogrel loading dose in tubes containing 3,2% sodium citrate. Platelet function assessment was performed using the PFA-200 P2Y system, which simulates the *in vitro* process of primary hemostasis and detects ADP receptor blockade using a membrane coated with ADP (20 μg), PGE-1 (5 ng), and calcium (125 μg). The result is expressed in terms of closure time (CT) measured in seconds, where values <106 s represent HTPR (Siemens Healthcare. 2010. Innovance PFA P2Y.). This value was utilized to define patients into HTPR and non-HTPR.

### Comprehensive custom NGS panel

In this study, we designed a comprehensive custom NGS panel that involved the analysis of 91 SNPs and 28 genes related to clopidogrel metabolism ([S1 Table](#pone.0306445.s003)). The SNPs were selected from previous GWAS studies conducted mainly in Europeans significantly associated with adenosine diphosphate-stimulated platelet aggregation [[7](#pone.0306445.ref007), [8](#pone.0306445.ref008), [13](#pone.0306445.ref013)]. The genes were extracted from the pharmacokinetic and pharmacodynamic pathways described in the PharmGKB database ([https://www.pharmgkb.org/](https://www.pharmgkb.org/)).

Agilent NGS custom target enrichment probes were used to design a customized hybrid capture panel to target both SNPs and genes. The design of custom hybridization capture probes for targeted sequencing was conducted using the SureDesign software. To select sequences of interest within an NGS library, we applied hybridization capture-based target enrichment using the SureSelect Custom Tier1 DNA Target Enrichment Probes (Agilent, Santa Clara, CA, USA).

Library preparation was carried out using the SureSelect XT HS2 target enrichment system (Agilent, Santa Clara, CA, USA). For sequencing, the DNA was circularized, and the library underwent denaturation after split oligo ligation. Subsequently, digestion and purification were performed using specific beads. The circularized DNA served as the template for generating DNBs (nanoballs) through the rolling circle amplification process (MGI Tech Co, 2022). DNBs were quantified and then sequenced on the DNBSeqG400 platform. Library preparation and sequencing were conducted by GencellPharma (Bogotá, Colombia).

### Genetic data quality analysis

The sequencing data obtained were analyzed using the hg19 reference genome (GRCh37). We assessed sequencing coverage for a set of 28 genes associated with the drug pathway. For each of these genes, we utilized the ENST codes obtained through Biomart-Ensembl. Exons lacking coding regions were filtered out, and the coordinates of exons to be covered were obtained. To calculate depth and coverage we used the bedcov tool from samtools, using as threshold for coverage 10X, 20X, 50X, 100X, 200X, and 500X. The same procedure was applied to the 91 SNPs with corresponding coordinates, aiming to derive the total read count, which is equivalent to the sum of depths per base. With the goal of achieving a 100% genotyping rate for the SNPs selected in all samples, variant calls with less than 10X were validated through Sanger Sequencing. Primer design was conducted using the Primer-BLAST tool for the amplification via Polymerase Chain Reaction (PCR) of the following SNPs: rs150117487, rs10935838, rs4709967, rs3732759, rs2046934, rs4709968, rs11724226 and rs6831212 ([S2 Table](#pone.0306445.s004)). The PCR was standardized, and the amplified products were visualized on 1.2% agarose gels. The amplicons were analyzed through Sanger sequencing, and the presence of the SNP was determined using FinchTV by comparing it to the hg19 reference genome (GRCh37).

### Bioinformatic analysis

Variant Call Format (VCF) files were used for the genotyping of SNPs and for prioritizing variants in the analyzed genes. Variants were prioritized using two filters: a) Clinically validated variants reported in the PharmGKB database (evidence levels 1, 2, and 3) and ClinVar and b) Rare and novel variants (MAF < 2%). For downstream analysis, all variants included in filter a were considered. For filter b, only variants with potential pathogenicity were selected, including Loss-of-Function (LoF) variants (nonsense and frameshift), missense variants, and splicing variants. To predict potential splicing alterations, Adaptive Boosting (ADA) Random Forest (TF) scores (cutoff ≥ 0.6) were used for splice site prediction. To determine the pathogenicity of missense variants, scores from in silico predictors within VarSeq software were considered. These predictors included Sorting Intolerant from Tolerant (SIFT), Polyphen 2, Mutation Taster, Mutation Assessor, and Functional Analysis Through Hidden Markov Models (FATHMM and FATHMM-MKL). Missense variants with 3 out of 6 positive predictors were considered as pathogenic. For the final analysis, variants obtained from filters a and b with a total depth of 8X, allelic representation in heterozygotes of ≥0.25 (minimum 25% of the polymorphic allele), and relevance to clopidogrel response as reported in the PharmGKB database were considered.

### Population genetic analysis and linkage disequilibrium

Allele and genotype frequencies as well as Hardy–Weinberg equilibrium (HWE) were calculated using the SNP-Stats software ([https://www.snpstats.net/start.htm](https://www.snpstats.net/start.htm)) and Haploview 4.2 ([https://www.broadinstitute.org/](https://www.broadinstitute.org/). Deviation from HWE was assessed using a χ2 goodness-of-fit test with 1°of freedom. Allelic frequencies for all molecular variants identified following bioinformatic analysis in the Latin America population were obtained from the genomAD public database version 2.1.1 ([https://gnomad.broadinstitute.org/](https://gnomad.broadinstitute.org/)). These frequencies were compared with our data using Chi-square analysis and Yates correction. Statistical significance was determined at p<0,05.

### Statistical analysis

A bivariate analysis was performed to assess the association between clinical and genetic factors and closure time (CT), which was utilized as an indicator of the response to clopidogrel. Patients were categorized into two groups, those with CT<106 seconds were classified as HTPR and those with CT>106 seconds were classified non-HTPR. For categorical variables, Chi-square test was applied, and Odds ratios (OR) and their respective 95% confidence intervals (CI) were calculated. Additionally, a logistic regression model was constructed using the least absolute shrinkage and selection operator (LASSO) regularization method. This model was built using samples from individuals with CT < 106s and individuals with CT > 106s. K-fold cross-validation was employed, dividing the data into three equal-sized folds or subsets. The LASSO model was estimated using the glmnet package. The outcomes of the model were depicted as receiver-operating-characteristic (ROC) curves, with their respective area under the curve (AUC) values. ROC and AUC estimation were performed using the pROC package. Data analysis was conducted using R (v4.1.2). To explore the potential association between SNPs and Mean CT values, comparisons were conducted to assess the potential association between single nucleotide polymorphisms (SNPs) and the numerical variable closure time (CT) using the Mann-Whitney U test with a significance threshold of p < 0,05. For the genetic variants, all statistical analyses were conducted by categorizing genotypes into three groups: a) if they carried 0,1 or 2 polymorphic alleles; b) if they carried 0 or 1 polymorphic allele, and c) if they carried 0 or 2 polymorphic alleles. This context considers an additive model for the association analyses. The co-dominant model (grouping a) proposes that genotypes with 0, 1, or 2 polymorphic alleles are associated with the lowest, intermediate, and highest risk, respectively [[14](#pone.0306445.ref014)].

### Pharmacogenomic polygenic risk estimation (PgxPRS)

The estimation of PgxPRS was performed using the **β** values obtain from the LASSO analysis. The individual risk score for each patient was determined as described previously and following the standard equation [[15](#pone.0306445.ref015)].

| PGxPRS=∑iNβj*dosageij |
| --- |
N represents the number of SNPs in the score, **β***i* is the effect size (or beta) of variant *i* and *dosage*_ij_ is the number of copies of SNP *i* in the genotype of individual **j**.

A linear correlation analysis was conducted between the PgxPRS data and the CT values, establishing the value of r^2^ and the statistical significance of the correlation (Spearman´s rho, p<0,05). The capacity of PGxPRS to differentiate patients HTPR from non-HTPR was evaluated (Mann-Whitney U test, p<0,05).

## Results

### Clinical and demographic characteristics

The study included a total of 70 patients. However, one of the patients was excluded from the final analysis due to the inability to obtain genotyping data, possibly due to low sample quality. Regarding gender distribution, males predominated in the study, accounting for 62.3% of the participants. The most common comorbidities among the patients were hypertension, affecting 63,8% of the participants, followed by dyslipidemia, which affected 56,5% of the patients. Regarding the types of acute coronary syndrome, acute myocardial infarction with ST-segment elevation (STEMI) was the most common, affecting 68,1% of the patients. Furthermore, a significant proportion of the patients exhibited CT<106s categorizing them as HTPR (clopidogrel non-responders), with this category representing 79,7% of the patients in the study. [Table 1](#pone.0306445.t001) summarizes the clinical and sociodemographic characteristics of the study sample.

### Table 1. Clinical and sociodemographic characteristics.

| Characteristic | (n) | % |
| --- | --- | --- |
| Sex |  |  |
| Male | 43 | 62,3 |
| Female | 26 | 37,7 |
| BMI type |  |  |
| Normal | 27 | 39,1 |
| Overweight | 27 | 39,1 |
| Obesity | 15 | 21,7 |
| Diabetes |  |  |
| No | 46 | 66,7 |
| Yes | 23 | 33,3 |
| Hypertension |  |  |
| No | 25 | 36,2 |
| Yes | 44 | 63,8 |
| Dyslipidemia |  |  |
| No | 29 | 42 |
| Yes | 39 | 56,5 |
| Smoking |  |  |
| Yes | 5 | 7,2 |
| No | 64 | 92,8 |
| Alcohol intake |  |  |
| No | 67 | 97,1 |
| Yes | 2 | 2,9 |
| ACS type |  |  |
| NSTEMI | 47 | 68,1 |
| STEMI | 14 | 20,3 |
| Unstable angina | 8 | 11,6 |
| Intervention type |  |  |
| PCI | 30 | 43,5 |
| Bypass | 18 | 26,1 |
| Medical management | 21 | 30,4 |
| Stent |  |  |
| No | 44 | 63,8 |
| Yes | 25 | 36,2 |
| Family history of ACS |  |  |
| No | 44 | 63,8 |
| Yes | 25 | 36,2 |
| Statins |  |  |
| No | 11 | 15,9 |
| Yes | 58 | 84,1 |
| Enoxaparin |  |  |
| No | 19 | 27,5 |
| Yes | 50 | 72,5 |
| ASA |  |  |
| No | 62 | 89,9 |
| Yes | 7 | 10,1 |
| CYP2C19 inducers |  |  |
| No | 69 | 100 |
| Yes | 0 | 0 |
| CYP2C19 inhibitors |  |  |
| No | 12 | 17,4 |
| Yes | 57 | 82,6 |
| PFA interpretation |  |  |
| Non-HTPR | 14 | 20,3 |
| HTPR | 55 | 79,7 |
| Outcome |  |  |
| Absent adverse effects | 59 | 85,5 |
| Bleeding | 10 | 14,5 |
| Reinfarction/thrombosis | 0 | 0 |
| PIatelet count |  |  |
| < = 200000 | 17 | 24,6 |
| >200000 | 52 | 75,4 |

Table 1 Caption: BMI type: Body Mass Index; ACS type: Acute Coronary Syndrome type; NSTEMI: Non-ST Elevation Myocardial Infarction; STEMI: ST Elevation Myocardial Infarction; PCI: Percutaneous Coronary Intervention; ASA: Acetyl Salicylic Acid; PFA: Platelet Function Analyzer; Non-HTPR: Non-High on Treatment Platelet Reactivity; HTPR: High on Treatment Platelet Reactivity

### Quality analysis of next generation sequencing

For SNPs, the average percentage of samples covered at thresholds of at least 10x, 20x, 50x, 100x, and 200x was consistently above 95,6%, >94,5%, >90,5%, >83.5%, and >78,9%, respectively. Notably, 87% of the analyzed SNPs were genotyped for all patients at a minimum of 10x depth. 96,7% of the SNPs were genotyped for more than 85% samples with a minimum depth of 10x. For the SNPs, rs10935838 and rs150117487, 68,6% and 8,6% patients, respectively, were genotyped at a minimum of 10X ([S3 Table](#pone.0306445.s005)). For genes, the average fraction of the target regions covered with at least 10X, 20X, 50X, and 100X was >95%, >92%, >88%, >82%, and >73%, respectively ([S4](#pone.0306445.s006) and [S5](#pone.0306445.s007) Tables). It is important to highlight that for the *CES1* gene, only three exonic regions satisfying the required 10x depth; therefore, this gene was excluded from the study.

### Genomic data obtained from NGS and population genetic analysis

Of the 91 SNPs included in the custom panel analyzed via NGS, 6 were excluded from the final analysis (rs140410716, rs201441480, rs28365085, rs3749187, rs72558186, and rs74569896) as 100% of the alleles represented in the study population corresponded to wild type. Regarding the analysis of the 28 genes involved in the pharmacokinetics and pharmacodynamics of clopidogrel, a total of 3,241,327 molecular variants were identified in the 69 patients analyzed. After applying the filtering criteria (filter a) described in the methodology, 18 molecular variants were identified ([Fig 1](#pone.0306445.g001)). Five of these variants rs2242480, rs41273215, rs57731889, rs6785930 and rs6809699, were common with the SNPs selected from GWAS studies. Following the additional filter (filter b, our analysis has revealed 26 molecular alterations in 14 genes. Among these, 21 (80,7%) are missense, 3 (11,5%) involve splicing, and 2 (7,7%) are insertion-deletion variants. Three of these variants are novel and not cataloged in public databases such as gnomAD v3.1.2 (*ABCB1* [NM_000927.5](https://www.ncbi.nlm.nih.gov/nuccore/NM_000927.5):c.2732C>T, *PTGS1* [NM_000962.4](https://www.ncbi.nlm.nih.gov/nuccore/NM_000962.4):c.94+198_94+338del, *ITGB3* [NM_000212.3](https://www.ncbi.nlm.nih.gov/nuccore/NM_000212.3):c.841C>T) ([S6 Table](#pone.0306445.s008)). Given the high sequencing depth achieved through NGS of the customized panel (ranging between 227,9x and 1912,7x), we did not perform Sanger sequencing to confirm any of the rare variants we identified.

### Fig 1. Bioinformatic pipeline.

![Fig 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/65f0/11239111/c2dc9f3d2e98/pone.0306445.g001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=11239111_pone.0306445.g001.jpg)

A total of 98 variants were included, with 85 of them extracted from previous GWAS studies and 13 obtained from gene analysis after applying bioinformatic filter A as described in the methodology. Population genetic analysis indicated that 96% of the variants are in Hardy-Weinberg equilibrium and exhibited allelic frequencies between 0,007 and 0,9420 ([S7 Table](#pone.0306445.s009)). The comparison of the allelic frequencies for the variants in our sample to those reported in the Latin American population in the gnomAD public database indicated that 54 of them (55,1%) presented significant differences (p < 0,05). Interestingly, 63% of these polymorphisms occurred at significantly higher frequencies in the sampled population. Most of them (85,3%) are variants in genes related to the pharmacokinetics and pharmacodynamics of clopidogrel. Notably, rs10306114 in the *PTGS1* had the lowest allele frequency in our population, and rs6809699 in the *P2RY12* gene presented the highest allele frequency, compared with Latin American population allele frequencies, these differences were statistically significant (p<0,05) ([S7 Table](#pone.0306445.s009)). Disequilibrium analysis revealed linkage in 14 chromosomes ([S1 Fig](#pone.0306445.s001)).

### Clinical-genetic association analysis with HTPR

Analysis of the association between clinical and genetic factors and the response to clopidogrel (established by the closure time-CT) revealed that 5,1% (5/98) of variants were significantly associated (rs1045642, rs2254638, rs2286823, rs2032582 and rs6798347). These SNPs are located within *ABCB1*, *N6AMT1*, *POR*, *ABCB1* and *MED12L* genes respectively. The SNP rs2286823 exhibited the strongest positive association strength (OR: 5,0; 95% CI: 1,02–24,48; p: 0,03) related to carrying 1 or 2 polymorphic alleles (GA+AA). Strong associations were also evidenced for the SNPs rs2032582 (OR: 4,41; 95% CI: 1,20–16,12; p: 0,019) and rs1045642 (OR: 3,38; 95% CI: 0,96–11,9; p: 0,05). A negative association was observed for SNP rs6798347 (OR: 0,15; 95% CI: 0,02–1,08; p: 0,039) related to the genotype carrying 2 polymorphic alleles. Regarding the association of HTPR/Non-HTPR status with non-genetic factors, the only significant association, which was negative, was identified for the stent usage condition (OR: 0,23; 95% CI: 0,07–0,79; p: 0,014). All results are detailed in [Table 2](#pone.0306445.t002).

### Table 2. Association between genetic factors and clopidogrel response.

| Gen | Variable | Genotype | Non-HTPR | HTPR | p.value | OR | IC_lower | IC_Upper |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
| ABCB1 | rs1045642 | AA | 6 (42,9%) | 10 (18,2%) | 0,039 | NA | NA | NA |
| AG | 8 (57,1%) | 31 (56,4%) |  |  |  |  |  |  |
| GG | 0 (0,0%) | 14 (25,5%) |  |  |  |  |  |  |
| ABCB1 | rs1045642_1 | AA | 6 (42,9%) | 10 (18,2%) | 0,051 | 3,38 | 0,96 | 11,90 |
| AG + AA | 8 (57,1%) | 45 (81,8%) |  |  |  |  |  |  |
| ABCB1 | rs2032582 | AA | 6 (42,9%) | 8 (14,5%) | 0,029 | NA | NA | NA |
| AC | 7 (50,0%) | 29 (52,7%) |  |  |  |  |  |  |
| CC | 1 (7,1%) | 18 (32,7%) |  |  |  |  |  |  |
| ABCB1 | rs2032582_1 | AA | 6 (42,9%) | 8 (14,5%) | 0,019 | 4,41 | 1,20 | 16,12 |
| AC + CC | 8 (57,1%) | 47 (85,5%) |  |  |  |  |  |  |
| N6AMT1 | rs2254638 | AA | 2 (14,3%) | 7 (12,7%) | 0,012 | NA | NA | NA |
| AG | 11 (78,6%) | 21 (38,2%) |  |  |  |  |  |  |
| GG | 1 (7,1%) | 27 (49,1%) |  |  |  |  |  |  |
| POR | rs2286823_1 | GG | 12 (85,7%) | 30 (54,5%) | 0,033 | 5,00 | 1,02 | 24,48 |
| GA + AA | 2 (14,3%) | 25 (45,5%) |  |  |  |  |  |  |
| MED12L | rs6798347_3 | GG | 7 (70%) | 31 (93,9%) | 0,039 | 0,15 | 0,02 | 1,08 |
| AA | 3 (30%) | 2 (6,1%) |  |  |  |  |  |  |
| - | Stent | Yes | 5 (35,7%) | 39 (70,9%) | 0,014 | 0,23 | 0,07 | 0,79 |
| No | 9 (64,3%) | 16 (29,1%) |  |  |  |  |  |  |

Table 2 Caption: HTPR High on-Treatment Platelet Reactivity. Non-HTPRT Non- High on-Treatment Platelet Reactivity. NA Not AppIicable.

### Genetic association analyses with the mean CT value

We evaluated the association of the 98 prioritized molecular variants with the mean CT values. This approach allowed us to identify 22 SNPs statistically associated (p<0,05). Some of them exhibited a high linkage disequilibrium (genes *NR3C2*, *PDE10A*, *POR* and *ABCB*1) ([Table 3](#pone.0306445.t003) and [S1 Fig](#pone.0306445.s001)). It was observed that being a carrier of at least one polymorphic allele resulted in a statistically significant lower CT value compared to the homozygous genotype for the reference allele. Conversely, for SNP rs1330344, non-carriers exhibited decreased CT values ([Table 3](#pone.0306445.t003) and [S2 Fig](#pone.0306445.s002)). The polymorphisms were observed at high allelic frequencies in the analyzed population (3,1% to 78,9%), which is relevant given their implication in closure time (CT), an indicator of the response to clopidogrel ([S7 Table](#pone.0306445.s009)).

### Table 3. Association between mean CT values and genetic factors.

| Gene | SNP | Genotype | N° of patients | CT mean value |
| --- | --- | --- | --- | --- |
| NR3C2 | rs72729980_3 | TT | 51 | 96,84 |
| CC | 3 | 60,67 |  |  |
| rs907620_3 | TT | 51 | 96,84 |  |
| CC | 3 | 60,67 |  |  |
| PDE10A | rs12055399 | GG | 64 | 94,89 |
| GA | 5 | 57,2 |  |  |
| rs4709966 | GG | 65 | 94,62 |  |
| AA | 4 | 52,25 |  |  |
| rs4709967 | TT | 64 | 94,89 |  |
| TC | 5 | 57,2 |  |  |
| rs7747284 | GG | 65 | 94,62 |  |
| GA | 4 | 52,25 |  |  |
| rs9348035 | AA | 64 | 94,89 |  |
| AG | 5 | 57,2 |  |  |
| rs9348036 | GG | 64 | 94,89 |  |
| GA | 5 | 57,2 |  |  |
| rs9356381 | GG | 64 | 94,89 |  |
| GT | 5 | 57,2 |  |  |
| rs9364810 | AA | 64 | 94,89 |  |
| AG | 5 | 57,2 |  |  |
| rs9365909 | AA | 64 | 94,89 |  |
| AG | 5 | 57,2 |  |  |
| rs9365910 | CC | 65 | 94,62 |  |
| CA | 4 | 52,25 |  |  |
| CYP3A5 | rs776746_3 | CC | 46 | 88,37 |
| TT | 5 | 64,6 |  |  |
| POR | rs2286823_1 | GG | 42 | 104 |
| GA + AA | 27 | 73,74 |  |  |
| rs2302429_3 | GG | 28 | 95,82 |  |
| AA | 3 | 62,33 |  |  |
| ABCB1 | rs1045642_3 | AA | 16 | 107,75 |
| GG | 14 | 64,07 |  |  |
| rs2032582_3 | AA | 14 | 109,29 |  |
| CC | 15 | 69,2 |  |  |
| PTGS1 | rs907620_1 | CC | 5 | 67,6 |
| CT + TT | 64 | 94,08 |  |  |
| CYP2C19 | rs12248560_1 | CC | 56 | 98,36 |
| CT + TT | 13 | 65,46 |  |  |
| CDH15 | rs72819363_1 | AA | 56 | 96,75 |
| AC + CC | 13 | 72,38 |  |  |
| CYP4F2 | rs2108622_3 | CC | 39 | 95,67 |
| TT | 3 | 65,67 |  |  |
| N6AMT1 | rs2254638_3 | AA | 9 | 100,67 |
| GG | 28 | 65,89 |  |  |

Table 3 Caption: * Statistical significance. p-value<0,05. NA Not AppIicable.

### Predictive model based on the Least Absolute Shrinkage and Selection Operator algorithm (LASSO)

To further construct a predictive model without redundant clinical information and SNPs, we used LASSO regression to narrow down the range of candidate variables. According to mean-square error ([Fig 2](#pone.0306445.g002)) and coefficients ([Fig 3](#pone.0306445.g003)), we opted for the former λ as it resulted in a better prediction efficiency than the latter λ in the estimated model. The AUC for the model on the training set was 0,955 ([Fig 4](#pone.0306445.g004)). The analysis showed that 2 clinical variables (Stent and statins) and 7 SNPs (rs12456693, rs2254638, rs342293, rs57731889, rs6798347, rs6801273 and rs2032582) were the most important for differentiating HTPR from Non-HTPR ([Table 4](#pone.0306445.t004)). These SNPs are located within *SLC14A2*, *N6AMT1*, *CTB-30L5*.*1*, *PEAR1*, *MED12L*, *P2RY12* and *ABCB1* respectively.

### Fig 2. Cross-validation plot for the penalty term λ based on differentially expressed metabolites.

![Fig 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/65f0/11239111/270ae0862c2e/pone.0306445.g002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=11239111_pone.0306445.g002.jpg)

Vertical bars represent acceptable maximum and minimum λ values with corresponding mean-squared error and the number of covariates for each model.

### Fig 3. Plots for lasso regression coefficients.

![Fig 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/65f0/11239111/ebe647acbffe/pone.0306445.g003.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=11239111_pone.0306445.g003.jpg)

Over different values of the penalty parameter λ for each model.

### Fig 4. ROC curves for training datasets for the estimated model.

![Fig 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/65f0/11239111/c052050b28d1/pone.0306445.g004.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=11239111_pone.0306445.g004.jpg)

### Table 4. Lasso model summary data.

| Genotype | Non-HTPR | HTPR | β value |
| --- | --- | --- | --- |
| NA | 5 (35,7%) | 39 (70,9%) | -0,0446 |
| 9 (64,3%) | 16 (29,1%) |  |  |
| NA | 0 (0,0%) | 11 (20,0%) | -0,0355 |
| 0 (0,0%) | 3 (5,5%) |  |  |
| 14 (100,0%) | 41 (74,5%) |  |  |
| CC | 11 (78,6%) | 42 (76,4%) | -0,1160 |
| CT | 2 (14,3%) | 13 (23,6%) |  |
| TT | 1 (7,1%) | 0 (0,0%) |  |
| AA | 2 (14,3%) | 7 (12,7%) | 0,1787 |
| AG | 11 (78,6%) | 21 (38,2%) |  |
| GG | 1 (7,1%) | 27 (49,1%) |  |
| CC | 4 (28,6%) | 27 (49,1%) | -0,0764 |
| CG | 7 (50,0%) | 24 (43,6%) |  |
| GG | 3 (21,4%) | 4 (7,3%) |  |
| CC | 9 (64,3%) | 34 (61,8%) | -0,0188 |
| CT | 3 (21,4%) | 19 (34,5%) |  |
| TT | 2 (14,3%) | 2 (3,6%) |  |
| GG | 7 (50,0%) | 31 (56,4%) | -0,1389 |
| GA | 4 (28,6%) | 22 (40,0%) |  |
| AA | 3 (21,4%) | 2 (3,6%) |  |
| TT | 7 (50,0%) | 18 (32,7%) | 0,1893 |
| TC | 3 (21,4%) | 29 (52,7%) |  |
| CC | 4 (28,6%) | 8 (14,5%) |  |
| AA | 6 (42,9%) | 8 (14,5%) | 0,1293 |
| AC + CC | 8 (57,1%) | 47 (85,5%) |  |

Table 4 Caption: NA Not AppIicable.

The beta values obtained from LASSO model, representing the coefficients associated with the predictor variables in the linear regression model, demonstrated that three of them (rs2254638, rs6801273 and rs2032582) exhibited positive values, whereas remaining ones were negatives ([Table 4](#pone.0306445.t004)). Interestingly, one of them (rs2032582) also displayed a strong association with the HTPR status (OR: 4,41; 95% CI: 1,20–16,12; p: 0,019) as previously described. The SNPs selected by the LASSO model were observed in the study population with high allelic frequencies and two of them (rs342293 and rs6801273) exhibit significantly higher frequencies in our population compared to those in Latin American populations described in the gnomAD database ([S7 Table](#pone.0306445.s009)). None of them exhibit linkage disequilibrium.

### Pharmacogenomics polygenic risk score (PgxPRS)

We developed a PgXPRS model considering individual effect sizes (β coefficients) and genotypes of the SNPs identified as predictor variables in the linear regression model (LASSO). The beta coefficients of each SNP used in PGxPRS determination are described in [Table 4](#pone.0306445.t004). Individual scores were determined for the 69 patients included in the study, with values ranging from -0,21119257 to 0,78887658 ([S8 Table](#pone.0306445.s010)).

The comparison of PGxPRS values between HTPR and non-HTPR patients demonstrated a statistically significant difference, thus highlighting the model’s ability for distinguishing these two groups (p-value: 0,0002962) ([Fig 5](#pone.0306445.g005)). Finally, a negative correlation was observed between PGxPRS and CT (p<0,001). The coefficient of determination (R^2^) was 0,13, indicating that approximately 13% of the variability in CT values can be attributed to variability in PgxPRS ([Fig 6](#pone.0306445.g006)).

### Fig 5. Boxplot PGxPRS vs HTPR condition.

![Fig 5](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/65f0/11239111/2bc51aacdff9/pone.0306445.g005.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=11239111_pone.0306445.g005.jpg)

Boxplot illustrating the relationship between responders and non-responders with respect to PGxPRS. The Mann-Whitney test reveals a statistically significant difference (p<0.0003).

### Fig 6. Scatterplot between polygenic risk score (PGxPRS) and obturation time (CT) with the added regression line illustrating the Spearman’s rho correlation (p<0.05).

![Fig 6](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/65f0/11239111/a5d55f8a70ea/pone.0306445.g006.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=11239111_pone.0306445.g006.jpg)

## Discussion

Clopidogrel continues to be the most extensively used antiplatelet drug globally [[3](#pone.0306445.ref003)], however, there is documented high variability in drug response, with an estimated 70% of this variability attributed to genetic factors [[4](#pone.0306445.ref004)]. It has been established that *CYP2C19*2*, a well-studied loss-of-function allele, account for only 12% of the variability in Europeans, implying that unexplored genetic variant impact clopidogrel response [[16](#pone.0306445.ref016)–[19](#pone.0306445.ref019)]. Based on findings of SNPs associated with clopidogrel response evaluated through GWAS [[5](#pone.0306445.ref005), [7](#pone.0306445.ref007), [8](#pone.0306445.ref008), [13](#pone.0306445.ref013)] and the analysis of the entire coding region and adjacent introns of drug metabolism-related genes ([https://www.pharmgkb.org/](https://www.pharmgkb.org/)), we designed a comprehensive pharmacogenetic panel (PGx) analyzed through NGS. To our knowledge, this approach has not been previously explored. The customized panel offers great flexibility in analysis, allowing the study of predefined gene sets and SNPs for specific phenotypes, with significant advantages such as the ability to simultaneously sequence multiple loci with great depth of coverage [[20](#pone.0306445.ref020)]. These findings support the use of current molecular methodologies based on targeted sequencing, which provide the potential to identify common, novel, and rare variants in pharmacogenetic interest genes [[21](#pone.0306445.ref021)].

Our designed PGx panel enabled the genotyping of 91 SNPs previously suggested to be associated with clopidogrel response by GWAS, six of them were monomorphic and therefore unsuitable for analysis. It is not surprising to identify these differences when considering that GWAS studies involve different populations than the underrepresented Latin American population [[8](#pone.0306445.ref008), [13](#pone.0306445.ref013)]. It is of interest that genetic architecture differences among populations determine that some markers are non-polymorphic or rare, and thus are related to the observed lack of replication in transethnic studies involving non-European ancestral populations [[22](#pone.0306445.ref022)].

Statistically significant differences were evident among the allele frequencies of 55,1% of the analyzed SNPs in our population (ranging from 0,7% to 94,2%) compared to those reported in the Latin American population in the gnomAD database, 63% of them appeared at higher frequencies (p<0,05) ([S7 Table](#pone.0306445.s009)). Notably, we have identified these population differences in other studies conducted with samples from Colombian population, where genes related to the absorption, distribution, metabolism, and excretion (ADME) of drugs, response to anesthetics, and the coding region of *CYP2C19* have been evaluated [[11](#pone.0306445.ref011), [12](#pone.0306445.ref012), [23](#pone.0306445.ref023)]. In all these studies, allele frequency comparisons have been made with the gnomAD database, Latino/Admixed American, which has determined that the described population consists of individuals with a wide range of admixture and significant variations in frequencies compared to other populations ([https://gnomad.broadinstitute.org/](https://gnomad.broadinstitute.org/)). The population differences found in this, and other studies may result from the lack of representation of all Hispanic groups in this database, highlighting the current need for genetic analysis in our Latin American populations.

We conducted analyses to determine the impact of genetic variables on clopidogrel responsiveness, utilizing the CT value, that has been described as a potential predictive biomarker [[24](#pone.0306445.ref024)–[26](#pone.0306445.ref026)]. Bivariate analyses indicated that five SNPs were significantly associated. The SNPs rs2286823 (*POR-*c.1239+20G>A), rs2032582 (*ABCB1*-c.2677G>T), and rs1045642 (*ABCB1*-c.3435C>T) exhibited the strongest positive association ([Table 2](#pone.0306445.t002)). Previous studies have shown that the *POR* gene plays a crucial role in interindividual differences in drug metabolism by affecting the activity of CYP enzymes [[27](#pone.0306445.ref027)]. Analyses in knockout mouse models have demonstrated that the loss of POR significantly reduces the metabolic capability of CYP enzymes, including CYP2C19, responsible for approximately 45% of clopidogrel conversion to its active metabolite [[28](#pone.0306445.ref028)]. In this context, the SNP *POR*- rs2286823, associated with decreased protein expression, can potentially diminish catalysis by CYP2C19 to barely detectable levels, like the effect caused by variants related to lower POR activity, such as A287P and R457H [[29](#pone.0306445.ref029)]. Our results are consistent with Shiraska et al., 2016, who proposed the significance of *POR* variants in individualizing clopidogrel therapy. In our population, carrying polymorphic alleles is associated with a fourfold risk of developing HTPR [[30](#pone.0306445.ref030)].

Regarding the implication of *ABCB1* gene variants rs2032582 and rs1045642 in the response to clopidogrel, our findings are in concordance with several literature reports indicating their association with a poor response to clopidogrel. Our results indicate that carriers of polymorphic alleles for *ABCB1* c.2677G>T or *ABCB1* c.3435C>T confer a 3,4- and 2,3-times higher risk of an inadequate response to clopidogrel (HTPR). For carriers of the *ABCB1*- 3435T variant, lower peak plasma concentration (Cmax) and total area under the plasma concentration-time curve (AUC) values of clopidogrel and its active metabolite were determined compared to non-carriers, suggesting that the *ABCB1* C3435T genotype influences clopidogrel absorption and active metabolite formation [[31](#pone.0306445.ref031)–[33](#pone.0306445.ref033)]. Similarly, carriers of *ABCB1* rs2032582 were also correlated with a decreased response to clopidogrel due to higher-than-expected platelet function compared to non-carriers [[10](#pone.0306445.ref010)]. In summary, this evidence supports our findings and demonstrates the functional impact of these variants on platelet reactivity, which was noteworthy in our population.

In addition, less-studied variants have demonstrated significant implications for HTPR, one of these corresponds to the SNP rs2254638 in the *N6AMT1* gene, for which a high proportion of homozygotes for the polymorphic allele (49,1%) was associated with a poor antiplatelet response ([Table 2](#pone.0306445.t002)). *N6AMT1* encodes for N (6)-adenine-specific DNA methyltransferase involved in the methylation of the release factor I [[34](#pone.0306445.ref034)] and has been previously linked to increased clopidogrel resistance [[35](#pone.0306445.ref035)].

Our analysis revealed that rs6798347 (*P2RY12* c.−281−3614C>T) constitutes a protective allele (OR: 0,15; 95% CI: 0,02–1,08; p: 0,039), in accordance with previous reports. This SNP, located in the promotor region of the gene, has been associated with a significantly lower incidence of HTPR compared to non-carriers [[36](#pone.0306445.ref036), [37](#pone.0306445.ref037)].

Given the potential association between reduced CT and a poorer prognosis in terms of clopidogrel response, we decided to investigate the relationship of genetic polymorphism with the mean CT value. This approach enabled us to identify statistically significant differences with 22 SNPs ([Table 3](#pone.0306445.t003)). Some of the SNPs that exhibited a positive association (p<0,05) were also identified in the previously discussed analysis (HTPR vs. non-HTPR condition) (*N6AMT1*: rs2254638; *POR*: rs2286823; *ABCB1*: rs1045642, rs2032582), supporting their significant influence on clopidogrel responsiveness.

An innovative discovery is the identification of several SNPs in the *PDE10A* gene to be associated with lower CT values (p: 0,0012), which, to our knowledge, had not been previously described in the literature. PDE10A is a member of the PDE10 protein family that catalyzes the hydrolysis of cAMP and cGMP. While the literature has focused on its role in psychiatric and neurological disorders, its involvement in smooth muscle cell (SMC)-like cell proliferation, injury-induced intimal thickening, and accelerated atherosclerosis has been recently documented [[38](#pone.0306445.ref038)]. It is estimated that the regulation of *PDE10A* gene expression is influenced by highly conserved putative intronic transcriptional sequences ([https://grantome.com/grant/NIH/R01-HL134910-01](https://grantome.com/grant/NIH/R01-HL134910-01)). Therefore, the molecular variations identified in our patients could be related to this up-regulation. In terms of clopidogrel response, it is plausible to hypothesize that intimal hyperplasia may affect treatment efficacy due to its association with a higher load of activated platelets and, consequently, increased resistance to platelet inhibition, suggesting that patients with *PDE10A* SNPs may require adjustments in antiplatelet therapy. Notably, the molecular variants of the PDE10A gene associated with the CT mean value are in LD ([Table 3](#pone.0306445.t003) and [S1 Fig](#pone.0306445.s001)). This finding enables the translational application of our results by allowing the selection of a single nucleotide polymorphism (SNP) that optimizes the information retained in a genomic region while reducing the genotyping effort and simplifying the analysis. It has been estimated that with sample sizes ranging from 50 to 100 individuals, consistent results can be achieved to identify these LD patterns. This enables genotyping fewer markers that yield redundant information due to extensive LD between nearby SNPs [[39](#pone.0306445.ref039)].

Other SNPs related to lower CT values have previously been linked to clopidogrel response, such as the case of *CYP3A5*-rs776746, and carriers of the polymorphic allele are estimated to have a high hepatic content of CYP3A, potentially affecting the drug [[40](#pone.0306445.ref040)]. We observed that rs2108622 in the *CYP4F2* gene showed lower CT values in TT homozygous patients, an allele associated with loss of function. This is explained by the interaction of a metabolite resulting from CYP4F2 monooxygenase-mediated W-hydroxylation of arachidonic acid with TxA2-induced platelet aggregation, thereby reducing platelet activation and aggregation through the platelet TxA2 receptor [[41](#pone.0306445.ref041)]. It is noteworthy that these genes, *CYP3A5* and *CYP4F2*, play a significant role in the response to tacrolimus and warfarin, respectively. However, our study, along with other literature reports, highlight their involvement in the response to clopidogrel. This emphasizes the importance of considering these genes in the monitoring of this antiplatelet agent.

For findings derived from case-control association analyses, it is essential to consider the influence of population stratification due to undetected population structure differences in ancestry that may lead to spurious associations [[42](#pone.0306445.ref042)]. Although our study did not involve ancestry informative markers (AIMs) analysis, we estimate that our patients exhibit a similar genetic background without the influence of factors such as genetic isolation or non-random mating, which are determinants of population stratification. The sample analyzed originates from the Andean region, where high inter-individual variation typical of other regions in the country has not been identified [[43](#pone.0306445.ref043)]. Additionally, it has been suggested that approaches such as family-based designs or genomic control are necessary when the identified genetic effect is very small (OR < 1,20) [[42](#pone.0306445.ref042)]. However, our findings demonstrated much higher odds ratios (ORs) ranging from 3.8 to 5.0 ([Table 2](#pone.0306445.t002)).

Despite the benefits derived from bivariate analyses, the relevance of analyzing models that include all genetic and non-genetic variables simultaneously has been emphasized. With this perspective, we conducted a combined analysis involving 30 clinical variables and 85 SNPs to explore a predictive model for HTPR and non-HTPR patients. This combined effect was evaluated using the machine learning approach, The Least Absolute Shrinkage and Selection Operator penalized regression model (LASSO).

LASSO regression is commonly employed to reduce model complexity by applying a penalty factor to emphasize more important variables. This model can reduce the effect of variables to zero, thereby decreasing the number of variables in a model [[44](#pone.0306445.ref044)]. LASSO has the advantage of avoiding overfitting and selecting only one feature from a group of correlated features [[45](#pone.0306445.ref045)].

This supervised machine learning approach is a regression method that has been used in pharmacogenomic prediction models for Warfarin dosage, demonstrating superior performance compared to algorithms based on multivariate linear regression. These findings, along with those of other authors, demonstrate that in pharmacogenetics, machine learning approaches show satisfactory performance in predicting drug responses in various fields such as cancer, depression, and anticoagulant therapy [[45](#pone.0306445.ref045)–[47](#pone.0306445.ref047)]. Results derived from the AUC value ([Fig 4](#pone.0306445.g004)) demonstrate that our model exhibits a high predictive capability (0,955) in distinguishing between HTPR and non-HTPR patients. This elevated AUC value, indicating that the LASSO model effectively separates phenotypic classes even with penalization and variable selection, suggests that the two non-genetic characteristics (stent and statins) and the 7 SNPs (rs12456693, rs2254638, rs342293, rs57731889, rs6798347, rs6801273, and rs2032582) selected by LASSO are relevant for the classification of HTPR and non-HTPR ([Table 4](#pone.0306445.t004)).

The data derived from the LASSO model enabled the generation of a PgxPRS related to the HTPR or non-HTPR condition. While several literature reports use effect size estimation derived from GWAS data, our estimation from the LASSO model is innovative and has recently been successfully applied in generating risk scores for patient stratification in susceptibility to ovarian cancer and Alzheimer’s disease [[48](#pone.0306445.ref048), [49](#pone.0306445.ref049)]. While drug dosing guidelines have been based on the analysis of one or a few SNPs, polygenic risk scores have emerged as a promising tool given the complex interaction and polygenic nature of drug response [[9](#pone.0306445.ref009), [50](#pone.0306445.ref050), [51](#pone.0306445.ref051)]. The PgxPRS developed by us enabled us to establish that HTPR patients with low CT values (Figs [5](#pone.0306445.g005) and [6](#pone.0306445.g006)) exhibited the highest risk score values ([Fig 5](#pone.0306445.g005)) (p<0,05). Given the relationship between CT value and inadequate clopidogrel response, our model represents a potential tool in clinical practice for patient risk stratification.

Significantly, our genomic approach using a custom panel evaluated through NGS has not been addressed in the mentioned studies or others in the literature. In our population, SNPs not described in previous studies developing PgxPRS were identified (rs12456693, rs2254638, rs342293, rs57731889, rs6798347, rs6801273). The majority of PGxPRS have involved predominantly European populations, so extrapolation to populations of different ethnic origins should be done with caution [[8](#pone.0306445.ref008)].

Surprisingly, for our population, the impact of *CYP2C19*2* was not evident in any of the analyses derived from the study. This finding can be primarily explained by the allele frequency in the study population (7,9%), which is significantly lower compared to other populations such as European (14.6%), Asian (30%), or African American (17,5%) populations [[52](#pone.0306445.ref052)]. This result is consistent with other study conducted by us in larger cohorts of individuals from the Colombian population [[11](#pone.0306445.ref011)]. The impact of SNPs may be influenced by their frequency across populations, emphasizing the need to incorporate comprehensive genomic studies involving a larger number of genetic markers potentially related to clopidogrel response.

Regarding the analysis of genes related to clopidogrel metabolism, the prioritization of molecular variants established by the Filter A identified 13 pharmacogenetically actionable polymorphisms in response to clopidogrel, as reported by the ClinVar and PharmKGB databases (evidence levels 1, 2, and 3). These variants were presented with allele frequencies ranging from 2,1% to 47,8%, and 76,9% of them showed statistically significant differences compared to those reported for the Latin American population in the gnomAD database, highlighting the importance of conducting specific population characterization and contributing to fill this knowledge gap in our population. These variants were predominantly represented by missense variants (80,7%), in accordance with previous studies, where an enrichment of pharmacogenetic variation by rare missense variants has been observed. This enrichment is primarily explained by low evolutionary constraint assumed for this type of genomic variation [[53](#pone.0306445.ref053)]. A previously reported study by our group characterizing ADME (absorption, distribution, metabolism, and excretion) genes through NGS revealed that rare missense variants are 2,1 times more frequent than common ones in our population. This emphasizes that such variants serve as a significant source of variability in pharmacogenes and likely account for unexplained interindividual variability in drug response [[11](#pone.0306445.ref011)]. The impact of these variants should be analyzed in larger patient cohorts or through functional validation, and their clinical application should be considered carefully.

Our work presents the first study using a custom panel of SNPs and genes related to clopidogrel response to develop models that establish associations and generate PgxPRS with molecular variants related to CT, a biomarker for clopidogrel response. This exploratory study proposes a PgxPRS that, when validated, becomes an invaluable tool in translational medicine applicable to stratifying patients at risk of not responding appropriately to antiplatelet treatment.

### Study limitations

The present study has several noteworthy limitations. Firstly, the limited sample size did not allow for a division to achieve training and validation through the LASSO model. For this reason, our results, demonstrating a high discriminatory power between HTPR and non-HTPR patients, should be considered exploratory. Secondly, our sample size may affect the accuracy of PgxPRS prediction, as a large discovery sample is required to determine the expected contribution of each SNP to the polygenic score for clopidogrel response. Thirdly, no replication data were included, so the conclusions derived from our study, which is the first to use a customized panel approach and LASSO analysis, should be interpreted with caution. The functional impact of rare variants was determined using an I*n-Silico* approach, therefore, functional validation or analysis in large patient cohorts is necessary. Finally, our results are derived from clopidogrel response determined through platelet reactivity analysis by assessing CT, so follow-up for the verification of major adverse cardiac events may be of great clinical utility.

## Conclusions

We have conducted the first pharmacogenomic study based on the analysis through NGS of multiple SNPs and genes related to platelet reactivity in patients treated with clopidogrel. Through case-control association analysis, we determined statistically significant associations within our population with SNPs rs1045642, rs2032582, and rs2286823. Similarly, molecular variants of the genes *NR3C2*, *PDE10A*, *CYP3A5*, *POR*, *ABCB1*, *PTGS1*, *CYP2C29*, *CDH15*, *CYP4F2*, and *N6AMT1* show association with CT values, a “surrogate” biomarker of clopidogrel response. The generation of a PgxPRS for the discrimination of HTPR and non-HTPR support the polygenic implication of the response to clopidogrel. Enhancing our understanding of the genetic architecture influencing variability in response to antiplatelet pharmacotherapy is crucial for addressing the challenge of inadequate responses to clopidogrel and exploring alternative therapeutic options. The exploration of such alternatives needs advancements in research. This pursuit aims not only to optimize current antiplatelet strategies, but also to pave the way for personalized medicine, where treatment decisions can be tailored based on an individual’s genetic profile.

## Supporting information

## Acknowledgments

We would like to thank Camilo Peralta for his support in data organization and the patients for their participation in this research.

## Data Availability

All vcf files and table_genotypes.xlxs are available from the Proyecto-genetica/Clopidogrel-repository database accession number(s): [https://github.com/Proyecto-genetica/Clopidogrel-repository.git](https://github.com/Proyecto-genetica/Clopidogrel-repository.git).

## Funding Statement

We would like to thank Camilo Peralta for his support in data organization and the patients for their participation in this research. Hospital Universitario Mayor Méderi- Universidad del Rosario GRANT QAN-BG278.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

All vcf files and table_genotypes.xlxs are available from the Proyecto-genetica/Clopidogrel-repository database accession number(s): [https://github.com/Proyecto-genetica/Clopidogrel-repository.git](https://github.com/Proyecto-genetica/Clopidogrel-repository.git).

### Supplementary Materials

### Data Availability Statement

All vcf files and table_genotypes.xlxs are available from the Proyecto-genetica/Clopidogrel-repository database accession number(s): [https://github.com/Proyecto-genetica/Clopidogrel-repository.git](https://github.com/Proyecto-genetica/Clopidogrel-repository.git).

## References

1. Zou X, Deng X li, Wang Y meng, Li J hua, Liu L, Huang X, et al. Genetic polymorphisms of high platelet reactivity in Chinese patients with coronary heart disease under clopidogrel therapy. Int J Clin Pharm. 2020. Feb;42(1):158–66. doi: 10.1007/s11096-019-00953-w  [DOI](https://doi.org/10.1007/s11096-019-00953-w) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32253660/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Clin%20Pharm&title=Genetic%20polymorphisms%20of%20high%20platelet%20reactivity%20in%20Chinese%20patients%20with%20coronary%20heart%20disease%20under%20clopidogrel%20therapy&author=X%20Zou&author=X%20li%20Deng&author=Y%20meng%20Wang&author=J%20hua%20Li&author=L%20Liu&volume=42&issue=1&publication_year=2020&pages=158-66&pmid=32253660&doi=10.1007/s11096-019-00953-w&)

2. Zwart B, Godschalk TC, Zheng K, Denteneer J, Kelder JC, Ten Berg JM. Measuring high on-treatment platelet reactivity in clinical practice; should we use a panel of platelet function tests? Blood Coagulation & Fibrinolysis. 2019. Sep;30(6):263–9.  [DOI](https://doi.org/10.1097/MBC.0000000000000831) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31259779/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood%20Coagulation%20&%20Fibrinolysis&title=Measuring%20high%20on-treatment%20platelet%20reactivity%20in%20clinical%20practice;%20should%20we%20use%20a%20panel%20of%20platelet%20function%20tests?&author=B%20Zwart&author=TC%20Godschalk&author=K%20Zheng&author=J%20Denteneer&author=JC%20Kelder&volume=30&issue=6&publication_year=2019&pages=263-9&pmid=31259779&doi=10.1097/MBC.0000000000000831&)

3. Castrichini M, Luzum JA, Pereira N. Pharmacogenetics of Antiplatelet Therapy. Annu Rev Pharmacol Toxicol. 2023. Jan 20;63(1):211–29. doi: 10.1146/annurev-pharmtox-051921-092701  [DOI](https://doi.org/10.1146/annurev-pharmtox-051921-092701) | [PMC free article](/articles/PMC9868113/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35914768/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annu%20Rev%20Pharmacol%20Toxicol&title=Pharmacogenetics%20of%20Antiplatelet%20Therapy&author=M%20Castrichini&author=JA%20Luzum&author=N%20Pereira&volume=63&issue=1&publication_year=2023&pages=211-29&pmid=35914768&doi=10.1146/annurev-pharmtox-051921-092701&)

4. Shuldiner AR. Association of Cytochrome P450 2C19 Genotype With the Antiplatelet Effect and Clinical Efficacy of Clopidogrel Therapy. JAMA. 2009. Aug 26;302(8):849. doi: 10.1001/jama.2009.1232  [DOI](https://doi.org/10.1001/jama.2009.1232) | [PMC free article](/articles/PMC3641569/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19706858/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA&title=Association%20of%20Cytochrome%20P450%202C19%20Genotype%20With%20the%20Antiplatelet%20Effect%20and%20Clinical%20Efficacy%20of%20Clopidogrel%20Therapy&author=AR%20Shuldiner&volume=302&issue=8&publication_year=2009&pages=849&pmid=19706858&doi=10.1001/jama.2009.1232&)

5. Zhong W, Wu H, Chen J, Li X, Lin H, Zhang B, et al. Genomewide Association Study Identifies Novel Genetic Loci That Modify Antiplatelet Effects and Pharmacokinetics of Clopidogrel. Clin Pharma and Therapeutics. 2017. Jun;101(6):791–802. doi: 10.1002/cpt.589  [DOI](https://doi.org/10.1002/cpt.589) | [PMC free article](/articles/PMC5485718/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27981573/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharma%20and%20Therapeutics.&title=Genomewide%20Association%20Study%20Identifies%20Novel%20Genetic%20Loci%20That%20Modify%20Antiplatelet%20Effects%20and%20Pharmacokinetics%20of%20Clopidogrel.&author=W%20Zhong&author=H%20Wu&author=J%20Chen&author=X%20Li&author=H%20Lin&volume=101&issue=6&publication_year=2017&pages=791-802&pmid=27981573&doi=10.1002/cpt.589&)

6. Strisciuglio T, Franco D, Di Gioia G, De Biase C, Morisco C, Trimarco B, et al. Impact of genetic polymorphisms on platelet function and response to anti platelet drugs. Cardiovasc Diagn Ther. 2018. Oct;8(5):610–20. doi: 10.21037/cdt.2018.05.06  [DOI](https://doi.org/10.21037/cdt.2018.05.06) | [PMC free article](/articles/PMC6232354/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30498685/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cardiovasc%20Diagn%20Ther.&title=Impact%20of%20genetic%20polymorphisms%20on%20platelet%20function%20and%20response%20to%20anti%20platelet%20drugs.&author=T%20Strisciuglio&author=D%20Franco&author=G%20Di%20Gioia&author=C%20De%20Biase&author=C%20Morisco&volume=8&issue=5&publication_year=2018&pages=610-20&pmid=30498685&doi=10.21037/cdt.2018.05.06&)

7. Backman JD, O’Connell JR, Tanner K, Peer CJ, Figg WD, Spencer SD, et al. Genome-wide analysis of clopidogrel active metabolite levels identifies novel variants that influence antiplatelet response. Pharmacogenetics and Genomics. 2017. Apr;27(4):159–63. doi: 10.1097/FPC.0000000000000272  [DOI](https://doi.org/10.1097/FPC.0000000000000272) | [PMC free article](/articles/PMC5346037/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28207573/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics%20and%20Genomics&title=Genome-wide%20analysis%20of%20clopidogrel%20active%20metabolite%20levels%20identifies%20novel%20variants%20that%20influence%20antiplatelet%20response&author=JD%20Backman&author=JR%20O%E2%80%99Connell&author=K%20Tanner&author=CJ%20Peer&author=WD%20Figg&volume=27&issue=4&publication_year=2017&pages=159-63&pmid=28207573&doi=10.1097/FPC.0000000000000272&)

8. Lewis JP, Backman JD, Reny JL, Bergmeijer TO, Mitchell BD, Ritchie MD, et al. Pharmacogenomic polygenic response score predicts ischaemic events and cardiovascular mortality in clopidogrel-treated patients. European Heart Journal—Cardiovascular Pharmacotherapy. 2020. Jul 1;6(4):203–10. doi: 10.1093/ehjcvp/pvz045  [DOI](https://doi.org/10.1093/ehjcvp/pvz045) | [PMC free article](/articles/PMC7363022/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31504375/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=European%20Heart%20Journal%E2%80%94Cardiovascular%20Pharmacotherapy&title=Pharmacogenomic%20polygenic%20response%20score%20predicts%20ischaemic%20events%20and%20cardiovascular%20mortality%20in%20clopidogrel-treated%20patients&author=JP%20Lewis&author=JD%20Backman&author=JL%20Reny&author=TO%20Bergmeijer&author=BD%20Mitchell&volume=6&issue=4&publication_year=2020&pages=203-10&pmid=31504375&doi=10.1093/ehjcvp/pvz045&)

9. Johnson D, Wilke MAP, Lyle SM, Kowalec K, Jorgensen A, Wright GEB, et al. A Systematic Review and Analysis of the Use of Polygenic Scores in Pharmacogenomics. Clin Pharma and Therapeutics. 2022. Apr;111(4):919–30. doi: 10.1002/cpt.2520  [DOI](https://doi.org/10.1002/cpt.2520) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34953075/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharma%20and%20Therapeutics&title=A%20Systematic%20Review%20and%20Analysis%20of%20the%20Use%20of%20Polygenic%20Scores%20in%20Pharmacogenomics.&author=D%20Johnson&author=MAP%20Wilke&author=SM%20Lyle&author=K%20Kowalec&author=A%20Jorgensen&volume=111&issue=4&publication_year=2022&pages=919-30&pmid=34953075&doi=10.1002/cpt.2520&)

10. Duconge J, Santiago E, Hernandez‐Suarez DF, Moneró M, López‐Reyes A, Rosario M, et al. Pharmacogenomic polygenic risk score for clopidogrel responsiveness among Caribbean Hispanics: A candidate gene approach. Clinical Translational Sci. 2021. Nov;14(6):2254–66. doi: 10.1111/cts.13124  [DOI](https://doi.org/10.1111/cts.13124) | [PMC free article](/articles/PMC8604227/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34415683/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clinical%20Translational%20Sci&title=Pharmacogenomic%20polygenic%20risk%20score%20for%20clopidogrel%20responsiveness%20among%20Caribbean%20Hispanics:%20A%20candidate%20gene%20approach.&author=J%20Duconge&author=E%20Santiago&author=DF%20Hernandez%E2%80%90Suarez&author=M%20Moner%C3%B3&author=A%20L%C3%B3pez%E2%80%90Reyes&volume=14&issue=6&publication_year=2021&pages=2254-66&pmid=34415683&doi=10.1111/cts.13124&)

11. Silgado-Guzmán DF, Angulo-Aguado M, Morel A, Niño-Orrego MJ, Ruiz-Torres DA, Contreras Bravo NC, et al. Characterization of ADME Gene Variation in Colombian Population by Exome Sequencing. Front Pharmacol. 2022. Jun 30;13:931531. doi: 10.3389/fphar.2022.931531  [DOI](https://doi.org/10.3389/fphar.2022.931531) | [PMC free article](/articles/PMC9280300/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35846994/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front%20Pharmacol&title=Characterization%20of%20ADME%20Gene%20Variation%20in%20Colombian%20Population%20by%20Exome%20Sequencing.&author=DF%20Silgado-Guzm%C3%A1n&author=M%20Angulo-Aguado&author=A%20Morel&author=MJ%20Ni%C3%B1o-Orrego&author=DA%20Ruiz-Torres&volume=13&publication_year=2022&pages=931531&pmid=35846994&doi=10.3389/fphar.2022.931531&)

12. Angulo-Aguado M, Panche K, Tamayo-Agudelo CA, Ruiz-Torres DA, Sambracos-Parrado S, Niño-Orrego MJ, et al. A Pharmacogenetic Study of CYP2C19 in Acute Coronary Syndrome Patients of Colombian Origin Reveals New Polymorphisms Potentially Related to Clopidogrel Therapy. JPM. 2021. May 12;11(5):400. doi: 10.3390/jpm11050400  [DOI](https://doi.org/10.3390/jpm11050400) | [PMC free article](/articles/PMC8150782/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34065778/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JPM&title=A%20Pharmacogenetic%20Study%20of%20CYP2C19%20in%20Acute%20Coronary%20Syndrome%20Patients%20of%20Colombian%20Origin%20Reveals%20New%20Polymorphisms%20Potentially%20Related%20to%20Clopidogrel%20Therapy.&author=M%20Angulo-Aguado&author=K%20Panche&author=CA%20Tamayo-Agudelo&author=DA%20Ruiz-Torres&author=S%20Sambracos-Parrado&volume=11&issue=5&publication_year=2021&pages=400&pmid=34065778&doi=10.3390/jpm11050400&)

13. Verma SS, Bergmeijer TO, Gong L, Reny J, Lewis JP, Mitchell BD, et al. Genomewide Association Study of Platelet Reactivity and Cardiovascular Response in Patients Treated With Clopidogrel: A Study by the International Clopidogrel Pharmacogenomics Consortium. Clin Pharma and Therapeutics. 2020. Nov;108(5):1067–77. doi: 10.1002/cpt.1911  [DOI](https://doi.org/10.1002/cpt.1911) | [PMC free article](/articles/PMC7689744/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32472697/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharma%20and%20Therapeutics&title=Genomewide%20Association%20Study%20of%20Platelet%20Reactivity%20and%20Cardiovascular%20Response%20in%20Patients%20Treated%20With%20Clopidogrel:%20A%20Study%20by%20the%20International%20Clopidogrel%20Pharmacogenomics%20Consortium.&author=SS%20Verma&author=TO%20Bergmeijer&author=L%20Gong&author=J%20Reny&author=JP%20Lewis&volume=108&issue=5&publication_year=2020&pages=1067-77&pmid=32472697&doi=10.1002/cpt.1911&)

14. Horita N, Kaneko T. Genetic model selection for a case–control study and a meta-analysis. Meta Gene. 2015. Sep;5:1–8. doi: 10.1016/j.mgene.2015.04.003  [DOI](https://doi.org/10.1016/j.mgene.2015.04.003) | [PMC free article](/articles/PMC4443430/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26042205/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Meta%20Gene&title=Genetic%20model%20selection%20for%20a%20case%E2%80%93control%20study%20and%20a%20meta-analysis&author=N%20Horita&author=T%20Kaneko&volume=5&publication_year=2015&pages=1-8&pmid=26042205&doi=10.1016/j.mgene.2015.04.003&)

15. Collister JA, Liu X, Clifton L. Calculating Polygenic Risk Scores (PRS) in UK Biobank: A Practical Guide for Epidemiologists. Front Genet. 2022. Feb 18;13:818574. doi: 10.3389/fgene.2022.818574  [DOI](https://doi.org/10.3389/fgene.2022.818574) | [PMC free article](/articles/PMC8894758/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35251129/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front%20Genet.&title=Calculating%20Polygenic%20Risk%20Scores%20(PRS)%20in%20UK%20Biobank:%20A%20Practical%20Guide%20for%20Epidemiologists.&author=JA%20Collister&author=X%20Liu&author=L%20Clifton&volume=13&publication_year=2022&pages=818574&pmid=35251129&doi=10.3389/fgene.2022.818574&)

16. Pereira NL, Rihal CS, So DYF, Rosenberg Y, Lennon RJ, Mathew V, et al. Clopidogrel Pharmacogenetics: State-of-the-Art Review and the TAILOR-PCI Study. Circ: Cardiovascular Interventions. 2019. Apr;12(4):e007811.  [DOI](https://doi.org/10.1161/CIRCINTERVENTIONS.119.007811) | [PMC free article](/articles/PMC6581205/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30998396/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circ:%20Cardiovascular%20Interventions&title=Clopidogrel%20Pharmacogenetics:%20State-of-the-Art%20Review%20and%20the%20TAILOR-PCI%20Study.&author=NL%20Pereira&author=CS%20Rihal&author=DYF%20So&author=Y%20Rosenberg&author=RJ%20Lennon&volume=12&issue=4&publication_year=2019&pages=e007811&pmid=30998396&doi=10.1161/CIRCINTERVENTIONS.119.007811&)

17. Sun Y, Lu Q, Tao X, Cheng B, Yang G. Cyp2C19*2 Polymorphism Related to Clopidogrel Resistance in Patients With Coronary Heart Disease, Especially in the Asian Population: A Systematic Review and Meta-Analysis. Front Genet. 2020. Dec 22;11:576046. doi: 10.3389/fgene.2020.576046  [DOI](https://doi.org/10.3389/fgene.2020.576046) | [PMC free article](/articles/PMC7783419/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33414804/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front%20Genet.&title=Cyp2C19*2%20Polymorphism%20Related%20to%20Clopidogrel%20Resistance%20in%20Patients%20With%20Coronary%20Heart%20Disease,%20Especially%20in%20the%20Asian%20Population:%20A%20Systematic%20Review%20and%20Meta-Analysis.&author=Y%20Sun&author=Q%20Lu&author=X%20Tao&author=B%20Cheng&author=G%20Yang&volume=11&publication_year=2020&pages=576046&pmid=33414804&doi=10.3389/fgene.2020.576046&)

18. Yu L, Wang T, Bai H, Zhu W, Li Y, Wu J, et al. Association between cytochrome P450 2C19 polymorphism and clinical outcomes in clopidogrel-treated Uygur population with acute coronary syndrome: a retrospective study. BMC Cardiovasc Disord. 2021. Dec;21(1):391. doi: 10.1186/s12872-021-02201-4  [DOI](https://doi.org/10.1186/s12872-021-02201-4) | [PMC free article](/articles/PMC8359105/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34384383/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Cardiovasc%20Disord&title=Association%20between%20cytochrome%20P450%202C19%20polymorphism%20and%20clinical%20outcomes%20in%20clopidogrel-treated%20Uygur%20population%20with%20acute%20coronary%20syndrome:%20a%20retrospective%20study&author=L%20Yu&author=T%20Wang&author=H%20Bai&author=W%20Zhu&author=Y%20Li&volume=21&issue=1&publication_year=2021&pages=391&pmid=34384383&doi=10.1186/s12872-021-02201-4&)

19. Lin J, Mo Y, Cai D, Mao D, Fu H, Wei D. CYP2C19 polymorphisms and clopidogrel efficacy in the secondary prevention of ischemic stroke: a retrospective observational study. Ann Palliat Med. 2021. Dec;10(12):12171–80. doi: 10.21037/apm-21-2905  [DOI](https://doi.org/10.21037/apm-21-2905) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35016407/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Palliat%20Med&title=CYP2C19%20polymorphisms%20and%20clopidogrel%20efficacy%20in%20the%20secondary%20prevention%20of%20ischemic%20stroke:%20a%20retrospective%20observational%20study&author=J%20Lin&author=Y%20Mo&author=D%20Cai&author=D%20Mao&author=H%20Fu&volume=10&issue=12&publication_year=2021&pages=12171-80&pmid=35016407&doi=10.21037/apm-21-2905&)

20. Castellanos E, Gel B, Rosas I, Tornero E, Santín S, Pluvinet R, et al. A comprehensive custom panel design for routine hereditary cancer testing: preserving control, improving diagnostics and revealing a complex variation landscape. Sci Rep. 2017. Jan 4;7(1):39348. doi: 10.1038/srep39348  [DOI](https://doi.org/10.1038/srep39348) | [PMC free article](/articles/PMC5209725/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28051113/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sci%20Rep.&title=A%20comprehensive%20custom%20panel%20design%20for%20routine%20hereditary%20cancer%20testing:%20preserving%20control,%20improving%20diagnostics%20and%20revealing%20a%20complex%20variation%20landscape.&author=E%20Castellanos&author=B%20Gel&author=I%20Rosas&author=E%20Tornero&author=S%20Sant%C3%ADn&volume=7&issue=1&publication_year=2017&pages=39348&pmid=28051113&doi=10.1038/srep39348&)

21. Gulilat M, Lamb T, Teft WA, Wang J, Dron JS, Robinson JF, et al. Targeted next generation sequencing as a tool for precision medicine. BMC Med Genomics. 2019. Dec;12(1):81. doi: 10.1186/s12920-019-0527-2  [DOI](https://doi.org/10.1186/s12920-019-0527-2) | [PMC free article](/articles/PMC6547602/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31159795/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Med%20Genomics&title=Targeted%20next%20generation%20sequencing%20as%20a%20tool%20for%20precision%20medicine&author=M%20Gulilat&author=T%20Lamb&author=WA%20Teft&author=J%20Wang&author=JS%20Dron&volume=12&issue=1&publication_year=2019&pages=81&pmid=31159795&doi=10.1186/s12920-019-0527-2&)

22. Li YR, Keating BJ. Trans-ethnic genome-wide association studies: advantages and challenges of mapping in diverse populations. Genome Med. 2014. Oct;6(10):91. doi: 10.1186/s13073-014-0091-5  [DOI](https://doi.org/10.1186/s13073-014-0091-5) | [PMC free article](/articles/PMC4254423/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25473427/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Med&title=Trans-ethnic%20genome-wide%20association%20studies:%20advantages%20and%20challenges%20of%20mapping%20in%20diverse%20populations&author=YR%20Li&author=BJ%20Keating&volume=6&issue=10&publication_year=2014&pages=91&pmid=25473427&doi=10.1186/s13073-014-0091-5&)

23. Corredor-Orlandelli D, Sambracos-Parrado S, Mantilla-García S, Tovar-Tirado J, Vega-Ramírez V, Mendoza-Ayús SD, et al. Association between Paraoxonase-1 p.Q192R Polymorphism and Coronary Artery Disease susceptibility in the Colombian Population. VHRM. 2021. Nov;Volume 17:689–99. doi: 10.2147/VHRM.S330766  [DOI](https://doi.org/10.2147/VHRM.S330766) | [PMC free article](/articles/PMC8573264/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34764653/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=VHRM&title=Association%20between%20Paraoxonase-1%20p.Q192R%20Polymorphism%20and%20Coronary%20Artery%20Disease%20susceptibility%20in%20the%20Colombian%20Population.&author=D%20Corredor-Orlandelli&author=S%20Sambracos-Parrado&author=S%20Mantilla-Garc%C3%ADa&author=J%20Tovar-Tirado&author=V%20Vega-Ram%C3%ADrez&volume=17&publication_year=2021&pages=689-99&pmid=34764653&doi=10.2147/VHRM.S330766&)

24. Lim HH, Li S, An GD, Woo KS, Kim KH, Kim JM, et al. Platelet Function Analyzer-200 P2Y Results Are Predictive of the Risk of Major Adverse Cardiac Events in Korean Patients Receiving Clopidogrel Therapy Following Acute Coronary Syndrome. Ann Lab Med. 2018. Sep 28;38(5):413–9. doi: 10.3343/alm.2018.38.5.413  [DOI](https://doi.org/10.3343/alm.2018.38.5.413) | [PMC free article](/articles/PMC5973914/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29797810/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Lab%20Med.&title=Platelet%20Function%20Analyzer-200%20P2Y%20Results%20Are%20Predictive%20of%20the%20Risk%20of%20Major%20Adverse%20Cardiac%20Events%20in%20Korean%20Patients%20Receiving%20Clopidogrel%20Therapy%20Following%20Acute%20Coronary%20Syndrome.&author=HH%20Lim&author=S%20Li&author=GD%20An&author=KS%20Woo&author=KH%20Kim&volume=38&issue=5&publication_year=2018&pages=413-9&pmid=29797810&doi=10.3343/alm.2018.38.5.413&)

25. Koessler J, Kobsar AL, Rajkovic MS, Schafer A, Flierl U, Pfoertsch S, et al. The new INNOVANCE® PFA P2Y cartridge is sensitive to the detection of the P2Y 12 receptor inhibition. Platelets. 2011. Feb;22(1):19–25.  [DOI](https://doi.org/10.3109/09537104.2010.514967) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20873965/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Platelets&title=The%20new%20INNOVANCE%C2%AE%20PFA%20P2Y%20cartridge%20is%20sensitive%20to%20the%20detection%20of%20the%20P2Y%2012%20receptor%20inhibition&author=J%20Koessler&author=AL%20Kobsar&author=MS%20Rajkovic&author=A%20Schafer&author=U%20Flierl&volume=22&issue=1&publication_year=2011&pages=19-25&pmid=20873965&doi=10.3109/09537104.2010.514967&)

26. Li S, Choi JL, Guo LZ, Goh RY, Kim BR, Woo KS, et al. Correlation Between the CYP2C19 Phenotype Status and the Results of Three Different Platelet Function Tests in Cardiovascular Disease Patients Receiving Antiplatelet Therapy: An Emphasis on Newly Introduced Platelet Function Analyzer-200 P2Y Test. Ann Lab Med. 2016. Jan 1;36(1):42–8. doi: 10.3343/alm.2016.36.1.42  [DOI](https://doi.org/10.3343/alm.2016.36.1.42) | [PMC free article](/articles/PMC4697342/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26522758/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Lab%20Med.&title=Correlation%20Between%20the%20CYP2C19%20Phenotype%20Status%20and%20the%20Results%20of%20Three%20Different%20Platelet%20Function%20Tests%20in%20Cardiovascular%20Disease%20Patients%20Receiving%20Antiplatelet%20Therapy:%20An%20Emphasis%20on%20Newly%20Introduced%20Platelet%20Function%20Analyzer-200%20P2Y%20Test.&author=S%20Li&author=JL%20Choi&author=LZ%20Guo&author=RY%20Goh&author=BR%20Kim&volume=36&issue=1&publication_year=2016&pages=42-8&pmid=26522758&doi=10.3343/alm.2016.36.1.42&)

27. Pandey AV, Sproll P. Pharmacogenomics of human P450 oxidoreductase. Front Pharmacol [Internet]. 2014. May 9 [cited 2024 Jan 2];5. Available from: http://journal.frontiersin.org/article/10.3389/fphar.2014.00103/abstract  [http://journal.frontiersin.org/article/10.3389/fphar.2014.00103/abstract](http://journal.frontiersin.org/article/10.3389/fphar.2014.00103/abstract) | [DOI](https://doi.org/10.3389/fphar.2014.00103) | [PMC free article](/articles/PMC4023047/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24847272/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front%20Pharmacol%20%5BInternet%5D.&title=Pharmacogenomics%20of%20human%20P450%20oxidoreductase.&author=AV%20Pandey&author=P%20Sproll&volume=5&publication_year=2014&pmid=24847272&doi=10.3389/fphar.2014.00103&)

28. Xue X, Gong L, Qi X, Wu Y, Xing G, Yao J, et al. Knockout of hepatic P450 reductase aggravates triptolide-induced toxicity. Toxicology Letters. 2011. Aug;205(1):47–54. doi: 10.1016/j.toxlet.2011.05.003  [DOI](https://doi.org/10.1016/j.toxlet.2011.05.003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21596114/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Toxicology%20Letters&title=Knockout%20of%20hepatic%20P450%20reductase%20aggravates%20triptolide-induced%20toxicity&author=X%20Xue&author=L%20Gong&author=X%20Qi&author=Y%20Wu&author=G%20Xing&volume=205&issue=1&publication_year=2011&pages=47-54&pmid=21596114&doi=10.1016/j.toxlet.2011.05.003&)

29. Agrawal V, Huang N, Miller WL. Pharmacogenetics of P450 oxidoreductase: effect of sequence variants on activities of CYP1A2 and CYP2C19. Pharmacogenetics and Genomics. 2008. Jul;18(7):569–76. doi: 10.1097/FPC.0b013e32830054ac  [DOI](https://doi.org/10.1097/FPC.0b013e32830054ac) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18551037/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics%20and%20Genomics.&title=Pharmacogenetics%20of%20P450%20oxidoreductase:%20effect%20of%20sequence%20variants%20on%20activities%20of%20CYP1A2%20and%20CYP2C19.&author=V%20Agrawal&author=N%20Huang&author=WL%20Miller&volume=18&issue=7&publication_year=2008&pages=569-76&pmid=18551037&doi=10.1097/FPC.0b013e32830054ac&)

30. Shirasaka Y, Chaudhry AS, McDonald M, Prasad B, Wong T, Calamia JC, et al. Interindividual variability of CYP2C19-catalyzed drug metabolism due to differences in gene diplotypes and cytochrome P450 oxidoreductase content. Pharmacogenomics J. 2016. Aug;16(4):375–87. doi: 10.1038/tpj.2015.58  [DOI](https://doi.org/10.1038/tpj.2015.58) | [PMC free article](/articles/PMC4775436/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26323597/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Interindividual%20variability%20of%20CYP2C19-catalyzed%20drug%20metabolism%20due%20to%20differences%20in%20gene%20diplotypes%20and%20cytochrome%20P450%20oxidoreductase%20content&author=Y%20Shirasaka&author=AS%20Chaudhry&author=M%20McDonald&author=B%20Prasad&author=T%20Wong&volume=16&issue=4&publication_year=2016&pages=375-87&pmid=26323597&doi=10.1038/tpj.2015.58&)

31. Taubert D, Vonbeckerath N, Grimberg G, Lazar A, Jung N, Goeser T, et al. Impact of P-glycoprotein on clopidogrel absorption. Clinical Pharmacology & Therapeutics. 2006. Nov;80(5):486–501. doi: 10.1016/j.clpt.2006.07.007  [DOI](https://doi.org/10.1016/j.clpt.2006.07.007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17112805/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clinical%20Pharmacology%20&%20Therapeutics&title=Impact%20of%20P-glycoprotein%20on%20clopidogrel%20absorption&author=D%20Taubert&author=N%20Vonbeckerath&author=G%20Grimberg&author=A%20Lazar&author=N%20Jung&volume=80&issue=5&publication_year=2006&pages=486-501&pmid=17112805&doi=10.1016/j.clpt.2006.07.007&)

32. Simon T, Verstuyft C, Mary-Krause M, Quteineh L, Drouet E, Méneveau N, et al. Genetic Determinants of Response to Clopidogrel and Cardiovascular Events. N Engl J Med. 2009. Jan 22;360(4):363–75. doi: 10.1056/NEJMoa0808227  [DOI](https://doi.org/10.1056/NEJMoa0808227) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19106083/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Genetic%20Determinants%20of%20Response%20to%20Clopidogrel%20and%20Cardiovascular%20Events&author=T%20Simon&author=C%20Verstuyft&author=M%20Mary-Krause&author=L%20Quteineh&author=E%20Drouet&volume=360&issue=4&publication_year=2009&pages=363-75&pmid=19106083&doi=10.1056/NEJMoa0808227&)

33. Wang X qin, Shen C lin, Wang B ning, Huang X hui, Hu Z le, Li J. Genetic polymorphisms of CYP2C19*2 and ABCB1 C3435T affect the pharmacokinetic and pharmacodynamic responses to clopidogrel in 401 patients with acute coronary syndrome. Gene. 2015. Mar;558(2):200–7. doi: 10.1016/j.gene.2014.12.051  [DOI](https://doi.org/10.1016/j.gene.2014.12.051) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25542807/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gene&title=Genetic%20polymorphisms%20of%20CYP2C19*2%20and%20ABCB1%20C3435T%20affect%20the%20pharmacokinetic%20and%20pharmacodynamic%20responses%20to%20clopidogrel%20in%20401%20patients%20with%20acute%20coronary%20syndrome.&author=X%20qin%20Wang&author=C%20lin%20Shen&author=B%20ning%20Wang&author=X%20hui%20Huang&author=Z%20le%20Hu&volume=558&issue=2&publication_year=2015&pages=200-7&pmid=25542807&doi=10.1016/j.gene.2014.12.051&)

34. Liu P, Nie S, Li B, Yang ZQ, Xu ZM, Fei J, et al. Deficiency in a Glutamine-Specific Methyltransferase for Release Factor Causes Mouse Embryonic Lethality. Molecular and Cellular Biology. 2010. Sep 1;30(17):4245–53. doi: 10.1128/MCB.00218-10  [DOI](https://doi.org/10.1128/MCB.00218-10) | [PMC free article](/articles/PMC2937546/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20606008/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Molecular%20and%20Cellular%20Biology&title=Deficiency%20in%20a%20Glutamine-Specific%20Methyltransferase%20for%20Release%20Factor%20Causes%20Mouse%20Embryonic%20Lethality&author=P%20Liu&author=S%20Nie&author=B%20Li&author=ZQ%20Yang&author=ZM%20Xu&volume=30&issue=17&publication_year=2010&pages=4245-53&pmid=20606008&doi=10.1128/MCB.00218-10&)

35. Zhong Z, Hou J, Zhang Q, Li B, Li C, Liu Z, et al. Effect of cytochrome P450 2C19 polymorphism on adverse cardiovascular events after drug-eluting stent implantation in a large Hakka population with acute coronary syndrome receiving clopidogrel in southern China. Eur J Clin Pharmacol. 2018. Apr;74(4):423–31. doi: 10.1007/s00228-017-2393-1  [DOI](https://doi.org/10.1007/s00228-017-2393-1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29243114/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Clin%20Pharmacol&title=Effect%20of%20cytochrome%20P450%202C19%20polymorphism%20on%20adverse%20cardiovascular%20events%20after%20drug-eluting%20stent%20implantation%20in%20a%20large%20Hakka%20population%20with%20acute%20coronary%20syndrome%20receiving%20clopidogrel%20in%20southern%20China&author=Z%20Zhong&author=J%20Hou&author=Q%20Zhang&author=B%20Li&author=C%20Li&volume=74&issue=4&publication_year=2018&pages=423-31&pmid=29243114&doi=10.1007/s00228-017-2393-1&)

36. Rudež G, Bouman HJ, Van Werkum JW, Leebeek FWG, Kruit A, Ruven HJT, et al. Common Variation in the Platelet Receptor P2RY12 Gene Is Associated With Residual On-Clopidogrel Platelet Reactivity in Patients Undergoing Elective Percutaneous Coronary Interventions. Circ Cardiovasc Genet. 2009. Oct;2(5):515–21.  [DOI](https://doi.org/10.1161/CIRCGENETICS.109.861799) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20031628/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circ%20Cardiovasc%20Genet.&title=Common%20Variation%20in%20the%20Platelet%20Receptor%20P2RY12%20Gene%20Is%20Associated%20With%20Residual%20On-Clopidogrel%20Platelet%20Reactivity%20in%20Patients%20Undergoing%20Elective%20Percutaneous%20Coronary%20Interventions.&author=G%20Rude%C5%BE&author=HJ%20Bouman&author=JW%20Van%20Werkum&author=FWG%20Leebeek&author=A%20Kruit&volume=2&issue=5&publication_year=2009&pages=515-21&pmid=20031628&doi=10.1161/CIRCGENETICS.109.861799&)

37. Nie X yan, Li J lei, Zhang Y, Xu Y, Yang X li, Fu Y, et al. Haplotype of platelet receptor P2RY12 gene is associated with residual clopidogrel on-treatment platelet reactivity. J Zhejiang Univ Sci B. 2017. Jan;18(1):37–47. doi: 10.1631/jzus.B1600333  [DOI](https://doi.org/10.1631/jzus.B1600333) | [PMC free article](/articles/PMC5260476/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28070995/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Zhejiang%20Univ%20Sci%20B&title=Haplotype%20of%20platelet%20receptor%20P2RY12%20gene%20is%20associated%20with%20residual%20clopidogrel%20on-treatment%20platelet%20reactivity&author=X%20yan%20Nie&author=J%20lei%20Li&author=Y%20Zhang&author=Y%20Xu&author=X%20li%20Yang&volume=18&issue=1&publication_year=2017&pages=37-47&pmid=28070995&doi=10.1631/jzus.B1600333&)

38. Raheem IT, Schreier JD, Fuerst J, Gantert L, Hostetler ED, Huszar S, et al. Discovery of pyrazolopyrimidine phosphodiesterase 10A inhibitors for the treatment of schizophrenia. Bioorganic & Medicinal Chemistry Letters. 2016. Jan;26(1):126–32. doi: 10.1016/j.bmcl.2015.11.013  [DOI](https://doi.org/10.1016/j.bmcl.2015.11.013) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26602277/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioorganic%20&%20Medicinal%20Chemistry%20Letters&title=Discovery%20of%20pyrazolopyrimidine%20phosphodiesterase%2010A%20inhibitors%20for%20the%20treatment%20of%20schizophrenia&author=IT%20Raheem&author=JD%20Schreier&author=J%20Fuerst&author=L%20Gantert&author=ED%20Hostetler&volume=26&issue=1&publication_year=2016&pages=126-32&pmid=26602277&doi=10.1016/j.bmcl.2015.11.013&)

39. Meng Z, Zaykin DV, Xu CF, Wagner M, Ehm MG. Selection of Genetic Markers for Association Analyses, Using Linkage Disequilibrium and Haplotypes. The American Journal of Human Genetics. 2003. Jul;73(1):115–30. doi: 10.1086/376561  [DOI](https://doi.org/10.1086/376561) | [PMC free article](/articles/PMC1180574/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12796855/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=The%20American%20Journal%20of%20Human%20Genetics&title=Selection%20of%20Genetic%20Markers%20for%20Association%20Analyses,%20Using%20Linkage%20Disequilibrium%20and%20Haplotypes&author=Z%20Meng&author=DV%20Zaykin&author=CF%20Xu&author=M%20Wagner&author=MG%20Ehm&volume=73&issue=1&publication_year=2003&pages=115-30&pmid=12796855&doi=10.1086/376561&)

40. Kim H, Lim Y, Oh M, Ghim J, Kim E, Kim D, et al. The pharmacokinetic and pharmacodynamic interaction of clopidogrel and cilostazol in relation to CYP2C19 and CYP3A5 genotypes. Brit J Clinical Pharma. 2016. Feb;81(2):301–12. doi: 10.1111/bcp.12794  [DOI](https://doi.org/10.1111/bcp.12794) | [PMC free article](/articles/PMC4833149/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26426352/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Brit%20J%20Clinical%20Pharma.&title=The%20pharmacokinetic%20and%20pharmacodynamic%20interaction%20of%20clopidogrel%20and%20cilostazol%20in%20relation%20to%20CYP2C19%20and%20CYP3A5%20genotypes.&author=H%20Kim&author=Y%20Lim&author=M%20Oh&author=J%20Ghim&author=E%20Kim&volume=81&issue=2&publication_year=2016&pages=301-12&pmid=26426352&doi=10.1111/bcp.12794&)

41. Tatarunas V, Kupstyte N, Giedraitiene A, Skipskis V, Jakstas V, Zvikas V, et al. The impact of CYP2C19 *2, CYP4F2 * 3, and clinical factors on platelet aggregation, CYP4F2 enzyme activity, and 20-hydroxyeicosatetraenoic acid concentration in patients treated with dual antiplatelet therapy. Blood Coagulation & Fibrinolysis. 2017. Dec;28(8):658–64.  [DOI](https://doi.org/10.1097/MBC.0000000000000658) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28806186/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood%20Coagulation%20&%20Fibrinolysis.&title=The%20impact%20of%20CYP2C19%20*2,%20CYP4F2%20*%203,%20and%20clinical%20factors%20on%20platelet%20aggregation,%20CYP4F2%20enzyme%20activity,%20and%2020-hydroxyeicosatetraenoic%20acid%20concentration%20in%20patients%20treated%20with%20dual%20antiplatelet%20therapy.&author=V%20Tatarunas&author=N%20Kupstyte&author=A%20Giedraitiene&author=V%20Skipskis&author=V%20Jakstas&volume=28&issue=8&publication_year=2017&pages=658-64&pmid=28806186&doi=10.1097/MBC.0000000000000658&)

42. Little J, Higgins JPT, Ioannidis JPA, Moher D, Gagnon F, von Elm E, et al. STrengthening the REporting of Genetic Association Studies (STREGA): an extension of the STROBE statement. PLoS Med. 2009. Feb 3;6(2):e22. doi: 10.1371/journal.pmed.1000022  [DOI](https://doi.org/10.1371/journal.pmed.1000022) | [PMC free article](/articles/PMC2634792/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19192942/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Med.&title=STrengthening%20the%20REporting%20of%20Genetic%20Association%20Studies%20(STREGA):%20an%20extension%20of%20the%20STROBE%20statement.&author=J%20Little&author=JPT%20Higgins&author=JPA%20Ioannidis&author=D%20Moher&author=F%20Gagnon&volume=6&issue=2&publication_year=2009&pages=e22&pmid=19192942&doi=10.1371/journal.pmed.1000022&)

43. Ossa H, Aquino J, Pereira R, Ibarra A, Ossa RH, Pérez LA, et al. Outlining the Ancestry Landscape of Colombian Admixed Populations. Chiang TY, editor. PLoS ONE. 2016. Oct 13;11(10):e0164414. doi: 10.1371/journal.pone.0164414  [DOI](https://doi.org/10.1371/journal.pone.0164414) | [PMC free article](/articles/PMC5063461/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27736937/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE.&title=Outlining%20the%20Ancestry%20Landscape%20of%20Colombian%20Admixed%20Populations&author=H%20Ossa&author=J%20Aquino&author=R%20Pereira&author=A%20Ibarra&author=RH%20Ossa&volume=11&issue=10&publication_year=2016&pages=e0164414&pmid=27736937&doi=10.1371/journal.pone.0164414&)

44. Choi SW, Mak TSH, O’Reilly PF. Tutorial: a guide to performing polygenic risk score analyses. Nat Protoc. 2020. Sep 1;15(9):2759–72. doi: 10.1038/s41596-020-0353-1  [DOI](https://doi.org/10.1038/s41596-020-0353-1) | [PMC free article](/articles/PMC7612115/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32709988/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Protoc.&title=Tutorial:%20a%20guide%20to%20performing%20polygenic%20risk%20score%20analyses.&author=SW%20Choi&author=TSH%20Mak&author=PF%20O%E2%80%99Reilly&volume=15&issue=9&publication_year=2020&pages=2759-72&pmid=32709988&doi=10.1038/s41596-020-0353-1&)

45. Cilluffo G, Fasola S, Ferrante G, Malizia V, Montalbano L, La Grutta S. Machine Learning: An Overview and Applications in Pharmacogenetics. Genes. 2021. Sep 26;12(10):1511. doi: 10.3390/genes12101511  [DOI](https://doi.org/10.3390/genes12101511) | [PMC free article](/articles/PMC8535911/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34680905/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genes&title=Machine%20Learning:%20An%20Overview%20and%20Applications%20in%20Pharmacogenetics.&author=G%20Cilluffo&author=S%20Fasola&author=G%20Ferrante&author=V%20Malizia&author=L%20Montalbano&volume=12&issue=10&publication_year=2021&pages=1511&pmid=34680905&doi=10.3390/genes12101511&)

46. Liu R, Li X, Zhang W, Zhou HH. Comparison of Nine Statistical Model Based Warfarin Pharmacogenetic Dosing Algorithms Using the Racially Diverse International Warfarin Pharmacogenetic Consortium Cohort Database. Hernandez-Lemus E, editor. PLoS ONE. 2015. Aug 25;10(8):e0135784. doi: 10.1371/journal.pone.0135784  [DOI](https://doi.org/10.1371/journal.pone.0135784) | [PMC free article](/articles/PMC4549222/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26305568/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE.&title=Comparison%20of%20Nine%20Statistical%20Model%20Based%20Warfarin%20Pharmacogenetic%20Dosing%20Algorithms%20Using%20the%20Racially%20Diverse%20International%20Warfarin%20Pharmacogenetic%20Consortium%20Cohort%20Database&author=R%20Liu&author=X%20Li&author=W%20Zhang&author=HH%20Zhou&author=E%20Hernandez-Lemus&volume=10&issue=8&publication_year=2015&pages=e0135784&pmid=26305568&doi=10.1371/journal.pone.0135784&)

47. Redenšek S, Jenko Bizjan B, Trošt M, Dolžan V. Clinical-Pharmacogenetic Predictive Models for Time to Occurrence of Levodopa Related Motor Complications in Parkinson’s Disease. Front Genet. 2019. May 16;10:461. doi: 10.3389/fgene.2019.00461  [DOI](https://doi.org/10.3389/fgene.2019.00461) | [PMC free article](/articles/PMC6532453/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31156712/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front%20Genet.&title=Clinical-Pharmacogenetic%20Predictive%20Models%20for%20Time%20to%20Occurrence%20of%20Levodopa%20Related%20Motor%20Complications%20in%20Parkinson%E2%80%99s%20Disease.&author=S%20Reden%C5%A1ek&author=B%20Jenko%20Bizjan&author=M%20Tro%C5%A1t&author=V%20Dol%C5%BEan&volume=10&publication_year=2019&pages=461&pmid=31156712&doi=10.3389/fgene.2019.00461&)

48. Dareng EO, Tyrer JP, Barnes DR, Jones MR, Yang X, Aben KKH, et al. Polygenic risk modeling for prediction of epithelial ovarian cancer risk. Eur J Hum Genet. 2022. Mar;30(3):349–62. doi: 10.1038/s41431-021-00987-7  [DOI](https://doi.org/10.1038/s41431-021-00987-7) | [PMC free article](/articles/PMC8904525/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35027648/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Hum%20Genet&title=Polygenic%20risk%20modeling%20for%20prediction%20of%20epithelial%20ovarian%20cancer%20risk&author=EO%20Dareng&author=JP%20Tyrer&author=DR%20Barnes&author=MR%20Jones&author=X%20Yang&volume=30&issue=3&publication_year=2022&pages=349-62&pmid=35027648&doi=10.1038/s41431-021-00987-7&)

49. Zhou X, Chen Y, Ip FCF, Lai NCH, Li YYT, Jiang Y, et al. Genetic and polygenic risk score analysis for Alzheimer’s disease in the Chinese population. Alzheimer’s & Dementia: Diagnosis, Assessment & Disease Monitoring [Internet]. 2020. Jan [cited 2024 Jan 2];12(1). Available from: https://onlinelibrary.wiley.com/doi/10.1002/dad2.12074  [https://onlinelibrary.wiley.com/doi/10.1002/dad2.12074](https://onlinelibrary.wiley.com/doi/10.1002/dad2.12074) | [DOI](https://doi.org/10.1002/dad2.12074) | [PMC free article](/articles/PMC7403835/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32775599/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Alzheimer%E2%80%99s%20&%20Dementia:%20Diagnosis,%20Assessment%20&%20Disease%20Monitoring%20%5BInternet%5D.&title=Genetic%20and%20polygenic%20risk%20score%20analysis%20for%20Alzheimer%E2%80%99s%20disease%20in%20the%20Chinese%20population&author=X%20Zhou&author=Y%20Chen&author=FCF%20Ip&author=NCH%20Lai&author=YYT%20Li&volume=12&issue=1&publication_year=2020&pmid=32775599&doi=10.1002/dad2.12074&)

50. Lanfear DE, Luzum JA, She R, Gui H, Donahue MP, O’Connor CM, et al. Polygenic Score for β-Blocker Survival Benefit in European Ancestry Patients With Reduced Ejection Fraction Heart Failure. Circ: Heart Failure. 2020. Dec;13(12):e007012.  [DOI](https://doi.org/10.1161/CIRCHEARTFAILURE.119.007012) | [PMC free article](/articles/PMC7738407/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33012170/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circ:%20Heart%20Failure&title=Polygenic%20Score%20for%20%CE%B2-Blocker%20Survival%20Benefit%20in%20European%20Ancestry%20Patients%20With%20Reduced%20Ejection%20Fraction%20Heart%20Failure.&author=DE%20Lanfear&author=JA%20Luzum&author=R%20She&author=H%20Gui&author=MP%20Donahue&volume=13&issue=12&publication_year=2020&pages=e007012&pmid=33012170&doi=10.1161/CIRCHEARTFAILURE.119.007012&)

51. Simona A, Song W, Bates DW, Samer CF. Polygenic risk scores in pharmacogenomics: opportunities and challenges—a mini review. Front Genet. 2023. Jun 15;14:1217049. doi: 10.3389/fgene.2023.1217049  [DOI](https://doi.org/10.3389/fgene.2023.1217049) | [PMC free article](/articles/PMC10311496/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/37396043/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front%20Genet&title=Polygenic%20risk%20scores%20in%20pharmacogenomics:%20opportunities%20and%20challenges%E2%80%94a%20mini%20review.&author=A%20Simona&author=W%20Song&author=DW%20Bates&author=CF%20Samer&volume=14&publication_year=2023&pages=1217049&pmid=37396043&doi=10.3389/fgene.2023.1217049&)

52. Ionova Y, Ashenhurst J, Zhan J, Nhan H, Kosinski C, Tamraz B, et al. CYP2C19 Allele Frequencies in Over 2.2 Million Direct‐to‐Consumer Genetics Research Participants and the Potential Implication for Prescriptions in a Large Health System. Clinical Translational Sci. 2020. Nov;13(6):1298–306.  [DOI](https://doi.org/10.1111/cts.12830) | [PMC free article](/articles/PMC7719394/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32506666/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clinical%20Translational%20Sci&title=CYP2C19%20Allele%20Frequencies%20in%20Over%202.2%20Million%20Direct%E2%80%90to%E2%80%90Consumer%20Genetics%20Research%20Participants%20and%20the%20Potential%20Implication%20for%20Prescriptions%20in%20a%20Large%20Health%20System.&author=Y%20Ionova&author=J%20Ashenhurst&author=J%20Zhan&author=H%20Nhan&author=C%20Kosinski&volume=13&issue=6&publication_year=2020&pages=1298-306&pmid=32506666&doi=10.1111/cts.12830&)

53. Nelson MR, Wegmann D, Ehm MG, Kessner D, St. Jean P, Verzilli C, et al. An Abundance of Rare Functional Variants in 202 Drug Target Genes Sequenced in 14,002 People. Science. 2012. Jul 6;337(6090):100–4. doi: 10.1126/science.1217876  [DOI](https://doi.org/10.1126/science.1217876) | [PMC free article](/articles/PMC4319976/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22604722/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=An%20Abundance%20of%20Rare%20Functional%20Variants%20in%20202%20Drug%20Target%20Genes%20Sequenced%20in%2014,002%20People.&author=MR%20Nelson&author=D%20Wegmann&author=MG%20Ehm&author=D%20Kessner&author=P%20St.%20Jean&volume=337&issue=6090&publication_year=2012&pages=100-4&pmid=22604722&doi=10.1126/science.1217876&)
