# Enhancer RNA transcription pinpoints functional genetic variants linked to asthma

## Metadata
**Authors:** Sarah K Sasse, Amber Dahlin, Lynn Sanford, Margaret A Gruca, Arnav Gupta, Fabienne Gally, Ann Chen Wu, Carlos Iribarren, Robin D Dowell, Scott T Weiss, Anthony N Gerber
**Journal:** Nature Communications
**Date:** 2025 Mar 31
**DOI:** [10.1038/s41467-025-57693-x](https://doi.org/10.1038/s41467-025-57693-x)
**PMID:** 40164603
**PMCID:** PMC11958640
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11958640/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC11958640/pdf/41467_2025_Article_57693.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC11958640/pdf/41467_2025_Article_57693.pdf)

## Abstract

Bidirectional enhancer RNA (eRNA) transcription is a widespread response to environmental signals and glucocorticoids. We investigated whether single nucleotide polymorphisms (SNPs) within dynamically regulated eRNA-transcribing regions contribute to genetic variation in asthma. Through applying multivariate regression modeling with permutation-based significance thresholding to a large clinical cohort, we identified novel associations between asthma and 35 SNPs located in eRNA-transcribing regions implicated in regulating cellular processes relevant to asthma, including rs258760 (mean allele frequency = 0.34, asthma odds ratio = 0.95; P = 5.04E-03). We show that rs258760 disrupts an active aryl hydrocarbon receptor (AHR) response element linked to transcriptional regulation of the glucocorticoid receptor gene by AHR ligands, which are commonly found in combusted air pollution. The role of rs258760 as a protective variant for asthma was independently validated using UK Biobank data. Our findings establish eRNA signatures as a tool for discovery of functional genetic variants and define a novel association between air pollution, glucocorticoid signaling and asthma.

Subject terms: Functional genomics, Genetics research

## Introduction

Asthma is a highly prevalent disease for which pathogenesis and treatment responsiveness are governed by complex stochastic interactions between genetic background and the environment^1^. Genome-wide association studies (GWAS) have defined numerous single nucleotide polymorphisms (SNPs) and associated pathways linked to risk of developing asthma^2–4^. For example, a large-scale GWAS of asthma in multiple ethnic populations identified 16 high-confidence SNP associations, annotated to *HLA-DQA1* and *IL1RL1*/*IL18R1*, which have established functional roles for asthma^5^. However, the sum of heritability explained by these SNPs is estimated to account for only 2.5% of the genetic risk of asthma^6–8^. Moreover, the majority of asthma-associated SNPs defined through GWAS are in non-protein coding portions of the genome^9^, and direct regulatory effects of most such SNPs have not been established^10^. These relative weaknesses of GWAS-based approaches are likely due to several underlying factors, including unaccounted effects of environment on asthma risk, linkage disequilibrium, and the very stringent *p*-values needed to identify significant genetic associations on a genome-wide basis^11^. These drawbacks have also limited the discovery of genetic variants relevant to specific asthma phenotypes, such as responsiveness to therapeutics. For example, clinical heterogeneity in responses to glucocorticoids in asthma is well described, however, relatively few genetic variants have been discovered that directly modify transcriptional responses to glucocorticoids or that directly link glucocorticoid targets to asthma risk^12^.

Specialized regions within the genome that control gene expression, frequently referred to as cis-regulatory elements or enhancers, are enriched for functional genetic variants that are linked to disease heritability^13,14^. As a method to reduce the *p*-value burden of traditional GWAS and increase the likelihood of identifying putative functional variants, strategies that incorporate enhancer features into genetic association studies have been developed and applied to various diseases^15^. However, enhancers, as defined using standard assays for chromatin accessibility or histone modifications, can span hundreds of kilobases^16^, which impacts resolution and statistical power. Moreover, these approaches often fail to define relationships between enhancers and regulation of specific genes^17^, especially in the context of disease-associated signaling pathways that reshape both enhancer activity and gene transcription.

Sequencing of non-polyadenylated or nascent transcripts has revealed that RNA is transcribed from active enhancers^18,19^, and that concomitant with changes in gene expression, significant changes in enhancer RNA (eRNA) transcription^20,21^ occur in response to a range of stimuli, including steroid hormones, inflammatory signals and environmental pollutants. Although the biological functions of enhancer RNAs remain poorly understood, eRNA transcription, which can be identified based on sites of bidirectional transcription exclusive of annotated gene transcription start sites, has emerged as a very sensitive marker of enhancer activity^22^. Moreover, discrete sites of RNA polymerase II (RNAPII) loading and transcription initiation within enhancers, which frequently serve as a nidus for the dynamic binding of regulatory transcription factors^19,23^, including the glucocorticoid receptor, can be bioinformatically inferred based on bidirectional transcription signatures^23^. We hypothesized that regions localized to sites of RNAPII loading within eRNA-transcript signatures that change dynamically in response to anti-inflammatory glucocorticoids or pro-inflammatory signals in airway epithelial cells harbor functional genetic variants associated with asthma risk. Here, we combine multivariate logistic regression modeling with permutation-based significance testing to filter SNPs within a large asthma cohort based on localization within dynamic eRNA-transcript signatures, and we perform fine mapping, annotation and functional validation of a subset of these novel asthma-associated SNPs.

## Results

### Localization of SNPs within dynamically regulated enhancers and asthma associations in a large clinical cohort

We previously employed Global Run-on Sequencing (GRO-seq) to analyze nascent RNA transcription on a genome-wide basis in BEAS-2B cells in response to 30-minute treatment with the prototypical anti-inflammatory glucocorticoid, dexamethasone (dex), and/or TNF, a canonical inducer of NFkB signaling, widely used to model therapeutic glucocorticoid receptor (GR) cross-talk with inflammatory signals^20^. Revised analysis of these data with updated algorithms (see Methods) defined, exclusive of transcription start sites (TSSs), a set of 1714 unique bidirectional enhancer RNAs (eRNAs) that responded significantly to dex (672 upregulated, 207 downregulated) or TNF (752 upregulated, 83 downregulated), and 137 eRNAs that responded to more than one condition (Supplementary Data [1](#MOESM3); p_adj_ < 0.05 vs vehicle). This compares to over 8000 sites of inducible GR or NFkB (the p65 subunit) occupancy identified in our published ChIP-seq analysis in the same cell type^24^.

Next, we used Tfit modeling to systematically infer and annotate the sites of RNAPII loading and bidirectional transcription initiation^25^, frequently referred to as Mu (μ), within the dex- and/or TNF-regulated eRNA set. To investigate potential relationships between these eRNA-transcribing genomic regions and asthma risk in the non-Hispanic white population within the Genetic Epidemiology Research in Adult Health and Aging (GERA) clinical cohort^5,26^, we leveraged regression modeling, permutation testing and fine mapping approaches^27^ to create an analysis pipeline (Fig. [1A](#Fig1)) for discovery of genetic variants associated with asthma based on proximity to μ. Based on this pipeline, we first screened the GERA dataset to filter SNPs that were annotated within the dynamically regulated enhancer regions, for each treatment separately (dex or TNF). A total of 143,869 SNPs were mapped to dex-responsive enhancer regions and 137,339 were mapped to TNF-responsive enhancer regions. Next, since RNAPII loading sites within eRNA transcribing regions are known to be enriched for functional transcription factor binding motifs^23^, we filtered these enhancer-SNP sets to include only those that were annotated to regions within 50 bp (+/-) of the Tfit-predicted RNAPII loading site, μ (μ-localized SNPs, or μ-SNPs). Within dex-responsive regions, we identified 866 total annotated μ-SNPs (661 upregulated; 205 downregulated), and within TNF-responsive regions we found 826 μ-SNPs (743 upregulated; 83 downregulated). The lower number of μ-SNPs observed within downregulated regions follows the predicted biological function of enhancers, i.e., dex or TNF perturbation are increasingly recognized as directly affecting gene transcription through upregulating enhancer site function^28,29^.

### Fig. 1. Pipeline for discovery of novel asthma-SNP associations based on proximity to μ and examples of μ-SNP colocalization.

![Fig. 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6545/11958640/fe8c7797eadf/41467_2025_57693_Fig1_HTML.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=11958640_41467_2025_57693_Fig1_HTML.jpg)

A Schematic of pipeline and filtering approaches to identify high confidence μ-SNPs associated with asthma. B–C GRO-seq tracks from BEAS-2B cells treated as indicated for 30 min and visualized in the Integrative Genomics Viewer (IGV) genome browser based on counts per million mapped reads (vertical scales). Blue indicates reads annotated to the sense strand while red indicates reads annotated to the antisense strand. The transcription start site (TSS) and direction of transcription are marked by arrows at the top of each screenshot. Magnified regions show the locations of μ originally calculated by Tfit (green) and following manual refinement (gray) relative to the indicated SNP within dynamically regulated enhancers containing high (B) or low (C) confidence μ-SNPs.

The subsets of μ-SNPs grouped by changes in eRNA transcription with treatment were then tested for association with asthma status using multivariate regression modeling, and the significance of associations was determined using a permutation-based approach implemented in the statistical genetic analysis software, PLINK (v. 1.9)^30^. Across the four treatment groups, representing 1692 total annotated μ-SNPs, 75 μ-SNPs were initially identified as significantly associated with asthma status (25, dex upregulated; 15, dex downregulated; 32, TNF upregulated; 3, TNF downregulated). SNPs within these regions were further characterized using in silico fine mapping to investigate possible functional impacts and prioritize candidate hits for follow-up study. Ultimately, this analysis pipeline output 39 significantly associated μ-SNPs with compelling evidence for functional roles based on robustness of association with asthma (Table [1](#Tab1)) and proximity to μ (Supplementary Table [1](#MOESM1)). Of these, there were 36 unique μ-SNPs, with 3 μ-SNPs identified within an eRNA region that was regulated by both TNF and dex (*rs78553489*, *rs186724791*, and *rs150808839*). Correlation with our published ChIP-seq analysis, which was performed in BEAS-2B cells^24^, identified sites of inducible GR or p65 occupancy located within ~ 1 kb of 25 of the 36 unique μ−SNPs (Supplementary Table [1](#MOESM1)).

### Table 1.

| SNP rsIDa | Minor Allele Frequency (GERA) | Odds Ratio for Asthma Risk (95% CI)b | Differentially Regulated Enhancer Dataset Assoc. with μ-SNPc | Permutation P valued |
| --- | --- | --- | --- | --- |
| rs117861858 | 4.92 × 10−05 | 10.4 (1.60 −67.7) | dex-repressed | 1.13 × 10−04 |
| rs141870749 | 2.95 × 10−05 | 9.83 (0.55 −176) | dex-induced | 1.58 × 10−04 |
| rs150872583 | 2.95 × 10−05 | 6.95 (0.63 − 77.1) | dex-induced | 1.59 × 10−04 |
| rs3129975 | 2.14 × 10−01 | 1.08 (1.04 − 1.12) | dex-repressed | 1.65 × 10−04 |
| rs113433891 | 1.97 × 10−05 | 5.07 (0.31 − 82.2) | dex-induced | 2.34 × 10−04 |
| rs80028982 | 9.83 × 10−06 | 3.09 (0.19 − 50.2) | dex-induced | 2.35 × 10−04 |
| rs149411423 | 2.95 × 10−05 | 13.2 (0.48 − 364) | dex-induced | 2.36 × 10−04 |
| rs187844646 | 1.97 × 10−05 | 12.6 (0.60 − 265) | dex-induced | 2.36 × 10−04 |
| rs141937580 | 1.97 × 10−05 | 5.30 (0.11 − 253) | dex-induced | 2.40 × 10−04 |
| rs182978446 | 1.97 × 10−05 | 10.8 (0.55 − 210) | dex-induced | 2.41 × 10-04 |
| rs186639917 | 1.57 × 10−04 | 5.43 (1.57 − 18.8) | dex-induced | 1.61 × 10-03 |
| rs78553489 | 2.07 × 10−04 | 4.12 (1.36 − 12.5) | TNF-repressed | 4.71 × 10-03 |
|   |   |   | dex-induced | 6.21 × 10-03 |
| rs258760 | 3.40 × 10−01 | 0.95 (0.92 − 0.99) | dex-repressed | 5.04 × 10-03 |
| rs189641630 | 6.59 × 10−04 | 2.16 (1.23 − 3.79) | dex-induced | 6.00 × 10-03 |
| rs141243502 | 2.56 × 10−04 | 0.15 (0.031 − 0.71) | dex-induced | 6.50 × 10-03 |
| rs12506479 | 2.71 × 10−01 | 1.05 (1.01 − 1.09) | TNF-induced | 6.58 × 10-03 |
| rs114363579 | 3.34 × 10−04 | 2.47 (1.18 − 5.13) | dex-repressed | 9.30 × 10−03 |
| rs61289882 | 2.08 × 10−01 | 1.05 (1.01 − 1.09) | TNF-induced | 9.38 × 10−03 |
| rs73731205 | 1.42 × 10−03 | 0.55 (0.34 − 0.88) | dex-repressed | 1.02 × 10−02 |
| rs76528672 | 7.87 × 10−05 | 5.51 (1.33 − 22.9) | dex-repressed | 1.06 × 10−02 |
| rs76717004 | 3.09 × 10−03 | 0.67 (0.49 − 0.913) | dex-repressed | 1.15 × 10−02 |
| rs58128597 | 7.54 × 10−02 | 0.93 (0.87 − 0.98) | TNF-repressed | 1.33 × 10−02 |
| rs78292063 | 1.60 × 10−03 | 1.55 (1.08 − 2.22) | dex-induced | 1.41 × 10−02 |
| rs57722273 | 2.38 × 10−03 | 1.43 (1.05 − 1.94) | dex-induced | 2.13 × 10−02 |
| rs186724791 | 1.57 × 10−04 | 0.08 (0.01 − 0.84) | dex-induced | 2.24 × 10−02 |
|   |   |   | TNF-induced | 2.24 × 10−02 |
| rs1454917 | 3.40 × 10−01 | 1.04 (1.00 − 1.07) | dex-repressed | 2.83 × 10−02 |
| rs186542091 | 8.92 × 10−03 | 0.82 (0.69 − 0.98) | dex-repressed | 3.29 × 10−02 |
| rs1409773 | 6.30 × 10−02 | 1.07 (1.00 − 1.14) | TNF-induced | 3.64 × 10−02 |
| rs3094117 | 2.72 × 10−01 | 1.04 (1.00 − 1.07) | dex-repressed | 3.69 × 10−02 |
| rs3924229 | 8.19 × 10−02 | 0.94 (0.89 − 1.00) | dex-induced | 3.92 × 10−02 |
| rs78124271 | 1.48 × 10−03 | 1.50 (1.01 − 2.23) | dex-induced | 4.03 × 10−02 |
| rs188187807 | 1.77 × 10−04 | 2.73 (0.97 − 7.66) | dex-induced | 4.20 × 10−02 |
| rs150808839 | 1.17 × 10−02 | 0.86 (0.74 − 0.99) | dex-induced | 4.29 × 10−02 |
|   |   |   | TNF-induced | 4.48 × 10−02 |
| rs60249091 | 1.82 × 10−01 | 1.04 (1.00 − 1.08) | TNF-induced | 4.33 × 10−02 |
| rs6018492 | 8.64 × 10−02 | 0.95 (0.89 − 1.00) | dex-induced | 4.78 × 10−02 |
| rs184589179 | 2.72 × 10−03 | 0.72 (0.53 − 1.00) | dex-induced | 4.98 × 10−02 |

Table 1 Caption: Initial set of unique μ-SNPs (n = 36) significantly associated with asthma risk in GERA cohort, sorted by permutation P-value

As a further step in validating the pertinence of this μ-SNP set in relationship to asthma, searches of the NHGRI-GWAS catalog, ClinVar, and PubMed databases were performed. None of the SNPs were reported as previously associated with asthma by GWAS, suggesting these could represent novel asthma associations based on our methods. Since our analysis is not driven by identifying GWAS signals, but rather as a tool to define functional variants and associated target genes, we alternately interrogated for prior associations at the gene level, specifying overlapping or nearest genes to each μ-SNP as the target. Using published data available in the NHGRI-GWAS catalog and a literature search of PubMed, we identified multiple associations with asthma and/or asthma treatment response, including *SLC16A12*^31^ (*rs78124271*), *CXCL8*^32,33^ (annotated as the closest gene downstream of *rs12506479*), *DUSP4*^34^ (annotated as the closest gene downstream of *rs186639917* and *rs80028982*), and *CLDN1*^35,36^ (annotated as the closest gene downstream of *rs60249091*). In aggregate, based on proximity, this preliminary analysis revealed that at least 47% (17/36) of our reported loci were previously related to respiratory disease and relevant immune response phenotypes. These data are summarized in Supplementary Data [2](#MOESM4).

### Refined mapping and analysis of μ-SNP associations

Confidence in predicting RNAPII loading sites based on Tfit-annotated enhancers depends on the complexity and magnitude of the bidirectional eRNA signature. To focus on SNPs that co-localize with high confidence sites of RNAPII loading (μ) and are thus more likely to be directly associated with enhancer function, we visually inspected the GRO-seq tracks within each enhancer region of interest to manually annotate and refine the bidirectional calls (Supplementary Table [2](#MOESM1)). One non-unique μ-SNP (*rs150808839*) was excluded here due to lack of visual confirmation of a clear bidirectional signature in its surrounding region. The remaining 35 regions were passed to a custom script that calculates the predicted location of μ based on each manually refined bidirectional annotation. A total of 24 μ-SNPs (69%) were located within 500 bp (+/-) of the refined μ sites; these were classified as high confidence μ-SNPs and prioritized for further analysis (Supplementary Data [3](#MOESM5)). Examples of high confidence and lower confidence sites of μ-SNP co-localization are depicted in Fig. [1B, C](#Fig1), respectively. Notably, of the 24 high confidence μ-SNPs, 22 (92%) resided in enhancers that, based on similar transcriptional patterns in response to dex and/or TNF (Supplementary Data [4](#MOESM6) and [5](#MOESM7); p_adj_ < 0.05 vs vehicle), could be linked to regulation of at least one proximal gene within a one megabase (Mb) radius (Supplementary Data [3](#MOESM5)). Gene functions linked to individual SNPs in this manner encompass signaling processes (e.g., *BMP1*^37^ and *PLK2*^38^), ubiquitin ligase activity (*HERC4*^39^) and transcriptional regulation (*CEBPB*^40^).

To further assess the regulatory function of the high confidence μ-SNP regions, we cloned ~450-1100 bp spanning 9 of the high confidence μ-SNP regions into a minimal promoter luciferase reporter (Supplementary Data [6](#MOESM8)). We assayed responses of these reporters to dex and TNF in BEAS-2B cells in comparison to two previously described control glucocorticoid-responsive reporters (p*FKBP5*^41^, p*TNFAIP3*^42^) and two reporters derived from low confidence μ-SNP regions (p*COL8A1*, p*CFLAR*). We found that 7 of the 9 high confidence enhancer reporters (78%) exhibited dynamic changes in activity (Fig. [2A](#Fig2)) consistent with regulation of the corresponding endogenous eRNAs (Fig. [2B](#Fig2); representative examples). These data indicate that the high confidence μ-SNPs are likely to confer biologically relevant regulatory function.

### Fig. 2. High confidence μ-SNP regions recapitulate dynamic eRNA regulation patterns in reporter assays.

![Fig. 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6545/11958640/42cade8b18ab/41467_2025_57693_Fig2_HTML.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=11958640_41467_2025_57693_Fig2_HTML.jpg)

A Mean (±SD) normalized luciferase activity of indicated enhancer reporter constructs or empty vector (EV) control in BEAS-2B cells treated as indicated for 8 hr (n = 6 technical replicates per treatment). pFKBP5 and pTNFAIP3 were included as canonical positive controls for dex and TNF responses. *Padj < 0.05 vs same reporter+veh; one-way ANOVA corrected for multiple comparisons using Bonferroni’s method (pFKBP5: Pdex < 0.0001, PTNF+dex = 0.0017; pTNFAIP3: Pdex = 0.0008, PTNF < 0.0001, PTNF+dex < 0.0001; pCOL8A1: PTNF < 0.0001, PTNF+dex < 0.0001; pCFLAR: PTNF < 0.0001, PTNF+dex < 0.0001; pRHOB: Pdex < 0.0001, PTNF+dex < 0.0001; pKCMF1: Pdex < 0.0001, PTNF < 0.0001, PTNF+dex < 0.0001; pANKRD1: Pdex = 0.0087; pCEBPB: Pdex < 0.0001, PTNF < 0.0001; pERRFI1: PTNF < 0.0001, PTNF+dex < 0.0001; pIER3: Pdex = 0.0002, PTNF < 0.0001, PTNF+dex < 0.0001; pOSR1: Pdex < 0.0001, PTNF < 0.0001, PTNF+dex < 0.0001; pNR3C1: Pdex < 0.0001, PTNF < 0.0001, PTNF+dex < 0.0001; pGCNT1: Pdex < 0.0001, PTNF = 0.0003, PTNF+dex < 0.0001.) Source data are provided as a Source Data file. B GRO-seq tracks, as described for Fig. 1, for select regions interrogated by enhancer reporter assay.

### Functional analysis characterizes novel SNPs relevant to GR signaling

To identify high confidence μ-SNPs that are most likely to exert a functional role in controlling gene expression, we used MatInspector^43^ to annotate canonical transcription factor binding motifs within 30 bp (+/-) of each SNP site. Through comparing motifs identified for both the major and variant alleles, *rs149411423* and *rs258760* emerged as strong candidates for functional regulation. *rs149411423* is a C > T transversion (minor allele frequency (MAF) in GERA = 2.95E-05; Table [1](#Tab1)) located in an enhancer region within *PRUNE2* that is ~175 kb from the TSS of *GCNT1*, which encodes a glycosyltransferase implicated in pulmonary immune responses and allergy^44,45^. As shown in Fig. [3A](#Fig3), both *GCNT1* (log_2_FC = 2.19, p_adj_ = 3.93E-09) and the enhancer harboring *rs149411423* (log_2_FC = 2.57, p_adj_ = 0.003) show increased transcription after dex treatment in BEAS-2B cells, whereas *PRUNE2* expression was unchanged (log_2_FC = 0.27, p_adj_ = 0.999). GR ChIP-seq analysis in BEAS-2B^20,24^ (Fig. [3B](#Fig3), *top*) and primary human airway smooth muscle (HASM) cells^46^ (Fig. [3B](#Fig3), *bottom*), another critical effector cell type in asthma, revealed dex-inducible GR occupancy near the *rs149411423*-μ-SNP region, supporting a direct regulatory role for this enhancer in the induction of *GCNT1* transcription and protein expression (see Supplementary Fig [1](#MOESM1)) by glucocorticoids.

### Fig. 3. rs149411423 disrupts a functional glucocorticoid response element that regulates induction of GCNT1.

![Fig. 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6545/11958640/ab9dbb820be9/41467_2025_57693_Fig3_HTML.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=11958640_41467_2025_57693_Fig3_HTML.jpg)

A GRO-seq tracks, as described for Fig.1, for the GCNT1 locus. Scaling applies through panel (D). B Glucocorticoid receptor (GR) ChIP-seq tracks from BEAS-2B (gray; top) and primary human airway smooth muscle (HASM) cells (black; bottom) ±1 hr dex visualized in the IGV browser based on counts per million mapped reads (vertical scales). C Micro-C chromatin contacts in BEAS-2B cells with arcs connecting 1 kb interacting regions across genomic space; darker color indicates higher contact frequency; blue indicates contacts of interest. Arc height is proportional to distance. D Heat map of chromatin contacts in publicly available Micro-C data generated in human embryonic stem cells39; arrows indicate contact between regions of interest. E Magnified view of GRO-seq tracks within rs149411423-μ-SNP region shown in panel (A). Scaling applies through Panel (F). F Aligned ATAC-seq tracks from primary human airway epithelial cells cultured at air-liquid interface (ALI; green), based on counts per million mapped reads (vertical scales), and magnified view of GR ChIP-seq peaks described in panel (B). G Consensus binding logo (MatBase) for GR and sequence of match identified by MatInspector within the GCNT1 enhancer harboring rs149411423. H Mean (±SD) normalized luciferase activity of parent or point mutation GCNT1 reporter constructs in cells treated ±dex for 8 hr (BEAS-2B; n = 6 technical replicates except for pFKBP5 + dex, n = 5) or 24 hr (primary HASM; n = 6 technical replicates except for HASM1: pGCNT1+dex, n = 5 and HASM2: pFKBP5+veh, n = 5). *Padj < 0.05 vs same reporter+veh; unpaired two-sided t-tests corrected for multiple comparisons using Holm-Sidak method (BEAS-2B: all P’s<0.000001; HASM1: P = 0.0028 for pFKBP5, P = 0.00005 for pGCNT1; HASM2: P = 0.0109 for pFKBP5, P = 0.0027 for pGCNT1, P = 0.0047 for pGCNT1 C > T), $Padj < 0.05 vs EV+veh; one-way ANOVA corrected for multiple comparisons using Bonferroni’s test (BEAS-2B: P < 0.0001 for pGCNT1 and pGCNT1 C > T; HASM1: P = 0.0003 for pGCNT1; HASM2: P < 0.0001 for pGCNT1), #Padj < 0.05 vs parent construct+dex; one-way ANOVA corrected for multiple comparisons using Bonferroni’s test (BEAS-2B: P < 0.0001 for pGCNT1 C > T; HASM1: P < 0.0001 for pGCNT1 C > T; HASM2: P < 0.0001 for pGCNT1 C > T). Source data are provided as a Source Data file.

To further assess putative regulation of *GCNT1* transcription by the *rs149411423*-μ-SNP enhancer region, we analyzed three-dimensional chromatin contact maps in BEAS-2B cells using Micro-C data we generated previously^47^. This was visualized as arcs connecting interacting genomic regions across the locus of interest (Fig. [3C](#Fig3)). These data identified three-dimensional contacts between the *rs149411423*-μ-SNP region and the *GCNT1* TSS. Moreover, publicly available Micro-C data generated in human embryonic stem cells^48^ and visualized in the UCSC Genome Browser defined a topologically associated domain (TAD) that contains both the *rs149411423*-μ-SNP and the *GCNT1* TSS (Fig. [3D](#Fig3)).

Further inspection the *rs149411423*-μ-SNP enhancer (Fig. [3E](#Fig3)) illustrates clear overlap of the eRNA bidirectional center with a chromatin accessibility peak in differentiated primary human airway epithelial cells within ATAC-seq data we generated previously^49^ (Fig. [3F](#Fig3), *top*) and the dex-inducible GR ChIP-seq peak observed in both BEAS-2B and HASM cells (Fig. [3F](#Fig3), *bottom*), supporting physiological relevance of the region. MatInspector analysis of the sequence surrounding *rs149411423* indicated that the transversion resides within a putative glucocorticoid response element (Fig. [3G](#Fig3)). Through applying a customized position weight matrix we created to predict relative GR binding affinities^50^, we found that the predicted affinity of the sequence containing the major C allele for binding GR is ~2-fold greater than that containing the variant T allele, suggesting the SNP may affect enhancer function. To assess whether *rs149411423* directly alters GR-regulated transcription, we generated a reporter harboring the variant allele using site-directed mutagenesis and assayed its activity compared to the parent construct with the major C allele in BEAS-2B and primary HASM cells. Relative to the parent construct, the mutant reporter exhibited dramatically attenuated induction by dex in all tested cell types (Fig. [3H](#Fig3)).

Based on these data, we used a CRISPR-based method to target the *rs149411423*-μ-SNP region (see Supplemental Methods) for deletion in Beas-2B cells^51^. PCR testing of bulk transfected cells indicated successful editing of the region (Supp Fig. [2](#MOESM1)) at an estimated efficiency range of 30–75% in 3 independent experiments. Analysis of *GCNT1* expression in relationship to estimated editing efficiency indicated that dex-mediated induction of *GCNT1* expression is largely mediated through the *rs149411423*-μ-SNP enhancer (Supp Fig. [2](#Fig2)). Taken together, our data implicate *rs149411423* as reducing GR-mediated induction of an enhancer that regulates transcription of *GCNT1*, strongly supporting a functional role for this SNP.

The second strong candidate SNP, *rs258760*, is a C > T transversion (MAF in GERA = 3.40E-01; Table [1](#Tab1)) within an *ARHGAP26* intron. This genomic location is approximately 200 kb downstream of the TSS for *NR3C1*, which encodes the glucocorticoid receptor. To assess whether the enhancer region harboring *rs258760* is associated with transcriptional control of *ARHGAP26* or *NR3C1*, we compared eRNA transcription patterns to nascent transcription of both genes after dex or TNF+dex treatment. Figure [4A](#Fig4) (GRO-seq tracks) illustrates that eRNA transcription from the *rs258760* region is reduced by dex (log_2_FC = -1.8, p_adj_ = 0.035) and TNF+dex treatment (log_2_FC = -2.01, p_adj_ = 0.011). Whereas *ARHGAP26* transcription is increased by dex (log_2_FC = 0.76, p_adj_ = 0.035) and TNF+dex (log_2_FC = 1.41, p_adj_ = 1.04E-07), *NR3C1* transcription is significantly reduced by treatment with TNF+dex (log_2_FC = -1.36, p_adj_ = 0.0008). As no other gene within a 1 Mb window of the *rs258760* region showed significant expression changes with either dex or dex+TNF treatment, these data suggest that *rs258760* resides in a regulatory element that controls expression of *NR3C1*. Further supporting this notion, in our published analysis of nascent transcriptional responses to wood smoke particles (WSP) using Precision Run-on Sequencing (PRO-seq)^52^, eRNA transcription from the same +200 kb enhancer was significantly increased after 30 minutes of WSP exposure in BEAS-2B cells (log_2_FC = 1.99, p_adj_ = 6.47E-62), with a congruent increase in transcription of *NR3C1* (log_2_FC = 0.90, p_adj_ = 9.66E-43), which we validated previously using RT-qPCR^52^. Expression of *ARHGAP26* (log_2_FC = -0.11, p_adj_ = 0.071; Fig. [4A](#Fig4), PRO-seq tracks), however, was unchanged by WSP treatment. Together, these data strongly suggest that the enhancer harboring *rs258760* controls transcription of *NR3C1*, rather than *ARHGAP26*. This μ-SNP region also aligns with an ATAC-seq peak in differentiated human-derived airway epithelial cells^49^, providing additional support for the physiologic relevance of this enhancer (Fig. [4B](#Fig4)). Furthermore, we previously showed that this enhancer region is occupied by the aryl hydrocarbon receptor (AHR)^52^, and MatInspector analysis of transcription factor binding motifs in this region in the context of the major C vs variant T alleles revealed that the C allele is a critical nucleotide within a canonical binding motif for AHR. This AHR binding motif is lost in the context of the T allele (Fig. [4C](#Fig4)), raising the possibility that AHR ligands, which are found at significant concentrations in wood and wildfire smoke^53–55^, control the activity of this enhancer in an allele-specific manner.

### Fig. 4. Expression and function of NR3C1 are regulated by aryl hydrocarbon receptor (AHR) ligands through an AHR response element disrupted by rs258760.

![Fig. 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6545/11958640/53cb31367912/41467_2025_57693_Fig4_HTML.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=11958640_41467_2025_57693_Fig4_HTML.jpg)

A IGV screenshots of BEAS-2B GRO-seq tracks aligned with PRO-seq tracks (sense strand light blue; antisense light purple) from BEAS-2B cells ±30 min wood smoke particles (WSP) treatment. B ATAC-seq tracks from primary human airway epithelial cells cultured at ALI. C Consensus binding logo (MatBase) for AHR and sequence match identified within the region containing rs258760. D Mean (±SD) normalized luciferase activity of indicated reporters (6 technical replicates) in BEAS-2B cells treated ±TCDD for 8 hr. ^Padj = 0.05 vs EV+veh and *Padj < 0.05 vs same reporter+veh; unpaired two-sided t-tests corrected here and throughout figure for multiple comparisons using Holm-Sidak test (P = 0.0039 for EV; P = 0.0003 for pMT2A, P < 0.0001 for pNR3C1, P = 0.0085 for pNR3C1 C > T), #Padj < 0.05 vs parent construct+TCDD; one-way ANOVA corrected for multiple comparisons using Bonferroni method (P < 0.0001 for pNR3C1 C > T). E qPCR analysis of dex-mediated gene regulation following 4 hr TCDD pre-treatment in BEAS-2B cells (4 technical replicates). Bars depict mean (±SD) CT values on a log2 scale relative to DMSO+veh-treated cells. *Padj < 0.05 vs same gene + DMSO+dex (unpaired two-sided t-tests corrected P = 0.0003 for FKBP5; P = 0.0006 for TNFAIP3; P = 0.0002 for NR3C1). F Magnified view of PRO-seq tracks from (A). G BEAS-2B Micro-C contacts as described for Fig. 3C (blue indicates relevant contacts). H Micro-C interactions39 as described for Fig. 3D; arrows indicate relevant contacts. I Visualization of SNPs by GWAS p-values for associations with forced vital capacity (FVC)57 across the chr5:143,100,000-143,400,000 (NR3C1) topologically-associated domain in the Lung Disease Knowledge Portal; association strength indicated by the color scale. J Mean (±SD) normalized luciferase activity of indicated reporters in BEAS-2B cells treated ±WSP for 8 hr (6 technical replicates). ^Padj < 0.05 vs EV+veh and *Padj < 0.05 vs same reporter+veh (unpaired two-sided t-tests corrected P = 0.0005 for EV, P < 0.0001 for pMT2A, P < 0.0001 for pNR3C1, P = 0.0015 for pNR3C1 C > T, P = 0.0005 for prs258753, P = 0.0081 for prs258753 T > C, P = 0.0005 for prs864354, P = 0.0002 for prs864354 G > T), $Padj < 0.05 vs EV+veh; one-way ANOVA corrected for multiple comparisons using Bonferonni test (all P’s<0.0001), #p < 0.05 vs parent construct+WSP; one-way ANOVA Bonferonni corrected (P < 0.0001 for pNR3C1 C > T). Source data are provided as a Source Data file.

To determine whether AHR and *rs258760* regulate activity of this presumptive *NR3C1* enhancer, we used site-directed mutagenesis to create a reporter with the minor, asthma-protective T allele, and assayed its activity compared to the parent construct in BEAS-2B cells. As illustrated in Fig. [4D](#Fig4), the parent C allele (*pNR3C1*) reporter shows robust induction following exposure to the dioxin, TCDD, a potent AHR ligand responsible for toxic effects of Agent Orange^56^. This induction was markedly reduced in the mutant reporter harboring the minor T allele, with p*MT2A* serving as a positive control for TCDD responsiveness. Thus, *rs258760* disrupts a functional AHR response element linked to regulating expression of *NR3C1* in airway epithelial cells in response to AHR ligands. Further, pre-treatment of BEAS-2B cells with TCDD prior to dex exposure potentiates induction of canonical GR transcriptional targets (Fig. [4E](#Fig4)), such as *FKBP5*^41^. These findings implicate the major C allele, which is associated with increased risk of asthma, as integral to an active AHR response element controlling a functional AHR-NR3C1 signaling axis in airway epithelial cells.

To further probe the relevance of the *rs258760*-μ-SNP in controlling *NR3C1* transcriptional regulation, we visualized our BEAS-2B Micro-C data across this region, which indicated three-dimensional physical contacts between the *rs258760*-μ-SNP locus and the *NR3C1* TSS (Fig. [4F, G](#Fig4)). These findings were corroborated by Micro-C data generated in human embryonic stem cells^48^ (Fig. [4H](#Fig4)), indicating that the *rs258760*-μ-SNP and the *NR3C1* TSS reside within a TAD roughly encompassing chr5:143,100,000-143,400,000 (hg38). To determine whether *rs258760* and/or other SNPs within the chr5:143,100,000-143,400,000 TAD have been independently associated with measures of lung function relevant to asthma, we used the Lung Disease Knowledge Portal ([https://lung.hugeamp.org/](https://lung.hugeamp.org/)) to visualize the TAD in genomic space along the x-axis relative to GWAS *p*-values for associations with lung forced vital capacity (FVC)^57^, which has been previously defined as relevant to air pollution and asthma^58–61^, on the y-axis (Fig. [4I](#Fig4)). While *rs258760* did not achieve statistical significance, there were significant associations between a number of other SNPs within the chr5:143,100,000-143,400,000 TAD and FVC, suggesting genetic variation within this TAD influences lung function. Of these, the GWAS-defined lead variant in the region, *rs864354* (FVC, GWAS *p*-value = 3.00E-11), lies within a genomic area that has no features of enhancer activity in our nascent sequencing data (see Fig. [4F](#Fig4)). In contrast, rs*258753* (FVC, GWAS *p*-value = 1.74E-9) is located within an intergenic enhancer in which eRNA transcription increases in response to WSP (see Fig. [4F](#Fig4)). To further explore putative functional relevance of *rs864354* and rs*258753*, we created luciferase reporters driven by the ~500–800 bp genomic regions spanning each transversion site and used site-directed mutagenesis to introduce the relevant variant allele into each reporter context. We directly compared the activities of these reporters to the *rs258760*-μ-SNP reporter series in response to WSP treatment (Fig. [4J](#Fig4)). The reporter spanning the *rs864354* SNP did not exhibit appreciable activity relative to the empty vector control, suggesting the region is not regulatory and the SNP is not likely a functional variant. In contrast, the *rs258753* reporter showed basal enhancer activity that increased in response to WSP, though not as robustly as the *rs258760*-μ-SNP reporter, consistent with an absence of AHR/ARNT binding motifs in the vicinity of *rs25875* identified using FIMO^62^ (Supplementary Data [7](#MOESM9)). And while the *rs258753* variant allele reduced responses to WSP relative to the parent construct, the difference was not statistically significant. In aggregate, these data implicate genetic variation in regulatory elements for *NR3C1* that respond to WSP as influencing asthma risk and lung function, and establish a novel connection between asthma, combustion-derived pollutants, and control of *NR3C1* expression.

To establish generalizability of the eRNA-SNP filtering method and further extend the relevance of the μ-SNP set, we performed a replication study using the UK Biobank (see Supplementary Methods). Demographic data on the 14,302 subject asthma cohort we defined within the UK Biobank are shown in Supplementary Table [3](#MOESM1). Of the 11 μ-SNPs with MAF ≥ 5%, 3 high confidence μ-SNPs, including *rs258753*, replicated with *p* values from the UK Biobank of < 0.05 and with joint *p*-values < 0.05, based on the method of Liptak^63^ (Table [2](#Tab2)). Moreover, 7/8 of the remaining μ-SNPs with MAF ≥ 5% exhibited odds ratios that were directionally (e.g., > 1 or < 1) congruent with the odds ratios defined in GERA (aggregate *p*-value < 0.006). Although replication of the low allele frequency (0.5% ≤ MAF < 5%) and rare (MAF < 0.5%) μ-SNPs was not anticipated due to obligate power constraints associated with less common variants, one rare variant, *rs78124271*, also replicated as associated with an asthma diagnosis in the UK Biobank (odds ratio 1.45, UK Biobank *p*-value = 0.027; UK Biobank/GERA combined *p*-value = 0.00341). Thus, filtering SNPs based on proximity to μ identifies replicable genetic variants associated with asthma.

### Table 2.

| SNP rsIDa | Major Allele | Minor Allele | Minor Allele Frequency | Odds Ratio (95% CI)b | P-value (UKBB)c | Combined P value (GERA + UKB) |
| --- | --- | --- | --- | --- | --- | --- |
| rs3129975 | T | C | 23.06% | 1.04 (1.00 − 1.08) | 0.040 | 1.66 × 10−05 |
| rs258760 | C | T | 32.37% | 0.97 (0.93 − 1.01) | 0.047 | 5.56 × 10−04 |
| rs1409773 | A | G | 6.45% | 1.06 (0.99 − 1.14) | 0.041 | 3.92 × 10−03 |
| rs1454917 | T | C | 34.94% | 0.99 (0.96 − 1.03) | 0.320 | - |
| rs3094117 | A | C | 28.88% | 1.03 (0.99 − 1.06) | 0.090 | - |
| rs12506479 | T | C | 26.90% | 1.01 (0.97 − 1.04) | 0.399 | - |
| rs61289882 | A | T | 20.22% | 1.01 (0.97 − 1.05) | 0.372 | - |
| rs60249091 | T | A | 18.56% | 1.03 (0.99 − 1.08) | 0.069 | - |
| rs3924229 | A | G | 11.95% | 1.02 (0.97 − 1.07) | 0.273 | - |
| rs6018492 | A | G | 7.86% | 0.97 (0.91 – 1.03) | 0.169 | - |
| rs58128597 | T | C | 7.32% | 0.97 (0.91 – 1.04) | 0.188 | - |

Table 2 Caption: μ-SNPs with significant p-values for odds ratio influencing likelihood of asthma diagnosis in the UK Biobank, sorted by combined p value integrating GERA and UK Biobank data

## Discussion

Nascent transcript sequencing is a powerful method for quantifying enhancer activity and has been applied previously to study eQTLs^64,65^, yet had not been systematically leveraged to identify novel SNP-disease associations. Our results indicate that constraining genetic association analysis based on SNP proximity to bidirectional eRNA transcription initiation sites can facilitate discovery of genetic variants that influence asthma risk, including the identification of disease-associated low frequency and rare variants, which are difficult to associate with disease using traditional GWAS. SNP filtering and analysis based on nascent eRNA transcription signatures offers other unique advantages including enrichment for functional regions within enhancers, which can span thousands of base pairs when defined using other methods, and in facilitating the identification of presumptive target genes for specific SNP-enhancer regions. Moreover, although we focused here on SNPs within eRNA-centered RNAPII loading regions that respond dynamically to glucocorticoids or TNF, this approach is generalizable to eRNAs regulated by other stimuli relevant to asthma, such as Type II cytokines or particulate pollution. Thus, our data provide a systematic method for discovery of functional variants that influence asthma or other airway disease risk in the context of specific endotypes or environmental exposures.

Our experimental approach was empirically designed to address specific limitations of current methodologies. For example, as gene TSSs are well annotated by many methods^66^, and SNPs located at TSSs do not pose an assignment problem in which the identification of the target gene for a putative regulatory SNP is uncertain, we specifically excluded sites of bidirectional transcription that overlap with gene TSSs from our filtering pipeline. In addition, since sites of bidirectional eRNA transcription are enriched for transcription factor binding^19,23^, we used narrow 100 bp regions centered on bidirectional eRNA signatures to filter SNPs prior to permutation analysis. This also served to limit the genomic search space, thereby increasing statistical power. However, when we refined the mapping of bidirectional eRNA transcription in relationship to high priority SNPs identified through our pipeline, we expanded the constraint for μ-SNP proximity to 1 kb regions centered on each refined μ, which better reflects the average size of typical enhancers^67^. Future investigations are needed to determine the optimal size for initial filtering in relationship to μ and to identify centers of bidirectional transcription more efficiently and using scalable bioinformatics methods, with a goal of obviating the manual refinement step we used in this work. Utilizing additional information on enhancer architecture and function, such as ChIP-seq data, may also help inform sequence size optimization and would help clarify potential advantages or drawbacks of applying eRNA signatures as tools for genetic discovery.

We used congruent transcriptional responses between eRNA-transcribing regions and genes located within a 1 Mb region to empirically assign relevant enhancer-gene interactions, with corroborative support provided by chromatin conformation data. This system expands on methods that have used cap analysis gene expression data from different cell types^68,69^, which can also define enhancer activity, to facilitate enhancer-promoter assignments, and our 1 Mb size window for assignments is derived from these and other data^70^. A key strength of our system in relationship to defining functional SNP-gene-disease relationships is the use of dynamic eRNA and gene transcription data in response to disease-relevant stimuli, rather than reliance on static expression comparisons across different cell types^71^. Additional genomic editing using CRISPR-based approaches to directly assess functional interactions between putative enhancer-reporter pairs, as we performed to definitively link the enhancer harboring *rs149411423* with control of *GCNT1* expression in response to dex, would further validate our approach, although enhancer redundancy can complicate such efforts^72^.

We undertook several approaches to establish the general relevance of our findings to asthma. First, we performed in silico analysis, which indicated that almost half of the novel μ-SNPs are in proximity to genes previously linked to asthma or immune function in other studies, establishing biologic plausibility of our findings. Second, we tested genomic regions harboring μ-SNPs in reporter assays, and we found that 7 of 9 tested regions conferred regulatory activity in this context, indicating frequent colocalization of μ-SNPs with genomic regulatory elements. Third, we performed a replication study using the UK Biobank, in which 3/11 μ-SNPs with MAF ≥ 5% replicated with *p*-values of < 0.05%. Moreover, the odds ratios in the UK Biobank for risk of an asthma diagnosis were directionally congruent with the odds ratios in GERA for 10 of the 11 μ-SNPs with MAF ≥ 5%, providing additional statistical support for our method (*p* < 0.006). These data exceed replication rates reported by others using these cohorts^73^. Both GERA and the UK biobank use physician-based diagnoses as a criterion for asthma, and standards for physiologic confirmation may differ between the cohorts, potentially adding variability. Although we focused on the non-Hispanic white subgroup within GERA, the ethnic makeup of this subgroup and the UK Biobank are not identical. This serves to further reduce the a priori power of the replication analysis, as prior studies have found that associations between SNPs and an asthma diagnosis are generally dependent on the underlying ethnicity^74–76^, even accounting for ethnicity-related differences in allele frequencies. Similarly, even within ethnically similar cohorts, modest differences in allele frequency can result in opposite allelic effects on disease risk between cohorts^77^, as we observed here with replication of the rare variant, *rs76717004*. Future investigations using and/or combining additional cohorts to increase power may provide further confidence in the novel associations we report in this study that have yet to be replicated.

A further limitation of our study is the reliance on BEAS-2B cells as a system to define relevant eRNA regions that respond to dex or TNF. Accordingly, SNPs residing within eRNAs regulated by these signaling pathways in other cell types may not have been interrogated for associations with asthma in our study. In addition, the regulatory and functional associations we defined between eRNA-SNPs and presumptive target genes need to be further validated in additional cell types directly relevant to asthma, including primary airway epithelial cells, airway smooth muscle, and immune cells.

Despite these limitations, our detailed analysis of two variants, *rs149411423* and *rs258760*, directly illustrates the effectiveness of our experimental design for discovery of functional SNPs and extends our understanding of GR signaling in the context of asthma risk. *rs149411423*, a high risk, rare allele, disrupts a functional GRE that regulates *GCNT1*. Although *rs149411423* did not replicate with UK Biobank data, the very low frequency of *rs149411423* significantly reduces the statistical power of such replication studies. However, the functional effects of *rs149411423* and the CRISPR-confirmed regulatory association with *GCNT1*, a glycosyltransferase linked to IgE levels, immune signaling and the pulmonary immune response to infection^44,78,79^, provide strong biologic support for the relevance of *rs149411423* to asthma. Furthermore, *GCNT1* is expressed in airway epithelial cells, where it is believed to play a role in mucin glycosylation^80^, and our data show that the *rs149411423* enhancer resides in open chromatin in differentiated primary human airway epithelial cells (Fig. [3](#Fig3)). Thus, although it remains to be determined whether *rs149411423* directly influences the therapeutic effects of GCs in asthma, we elucidate a novel and physiologically plausible GR-GCNT1 axis that is relevant to asthma risk and highlights GCNT1 as a potential therapeutic target.

Interestingly, *rs149411423* was present at very low minor allele frequencies in all populations genotyped in GERA, and imputation of genotypes in GERA was robust^26^. However, *rs149411423* was reported at a notably higher frequency in the African-American population genotyped in the 1000 Genomes Project (MAF in 1000 Genomes = 0.0106). Studies that utilize multi-ethnic cohorts with sufficient sample sizes to provide the statistical power to detect robust associations with asthma are needed to further clarify the roles of this and other low frequency and rare variants we identified in this study. While we limited our analysis to SNPs in GERA with imputation quality scores > 0.9, it is possible that the imputation of the rare SNPs may be subject to errors. For *rs149411423*, our extensive mechanistic analysis links this SNP directly to altered induction of an asthma-relevant target gene, *GCNT1*, by glucocorticoids, providing additional support for a link between *rs149411423* and asthma. Mechanistic studies of the other rare SNPs may similarly strengthen the genetic findings we report here.

We also discovered a genetic variant, *rs258760*, a low risk, high frequency allele that regulates expression of the glucocorticoid receptor itself in relationship to asthma risk. In this case the variant allele is the most common allele, and minor allele frequencies were again found to vary by ethnicity in other multi-ethnic cohorts such as 1000 Genomes: (*rs258760* MAF in 1000 Genomes = 0.34 for European, 0.117 for African-American and 0.008 for East Asian populations). *rs258760* was also one of the 3 common alleles (MAF > 5%) that replicated using UK Biobank data. The more common C allele, associated with increased risk of an asthma diagnosis, is a required nucleotide within a functional binding site for AHR. Our data implicate this AHR response element in controlling increased transcription of airway epithelial GR in response to AHR ligands, which are present at physiologically relevant concentrations in particulate matter air pollution^81,82^. Our results are supported by evidence that *rs258760* and *NR3C1* reside within a TAD and physically associate with each other in three dimensions. Moreover, within this TAD, a second SNP, *rs258753*, which achieved genome-wide significance in a GWAS examining lung forced vital capacity (FVC), is located within a distinct wood smoke-induced enhancer. However, this SNP was not predicted to alter binding site preferences for AHR, and accordingly, our data show that only *rs258760* significantly influenced enhancer responses to wood smoke. Thus, we establish an unexpected genetic link between combustion-related air pollution, such as wood smoke, regulation of GR expression, and lung function.

Intriguingly, as the loss of a functional AHR element caused by the *rs258760* variant allele was associated with a decreased risk of an asthma diagnosis in both the GERA and UK Biobank cohorts, our data support a model in which local inflammatory repression via GR in the setting of air pollution may have deleterious effects. This raises the possibility that empiric use of inhaled steroids in the context of air pollution, a problem of increasing importance given the malign effects of climate change on air quality^83–85^, may entail unappreciated risks dependent on genetic context. This notion aligns with several studies in which the clinical effectiveness of inhaled corticosteroid use in the setting of air pollution was brought into question^86–89^, and supports further investigation of genetic associations between the *NR3C1* locus and asthma, and also consideration of a powered clinical trial to directly determine whether inhaled corticosteroids are harmful or beneficial in the context of air pollution. While further exploration and validation are underway, particularly in appropriately powered multi-ethnic cohorts that account for environmental factors, in aggregate, we have defined novel genetic links between GR signaling and asthma risk, and we have established a powerful new application of eRNA transcription signatures for discovery of SNP-disease associations.

## Methods

### Ethical Standards

This research complies with all relevant ethical standards, and the data used from GERA is derived from studies originally approved by Institutional Review Boards of Kaiser Permanente.

### Nascent transcription and cistromic datasets

We previously published the BEAS-2B GRO-seq nascent transcript dataset^20^ used in the current study with linked Gene Expression Omnibus (GEO) accession number [GSE124916](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE124916) (30 min datasets). Duplicate samples for four treatment conditions [vehicle (ethanol), TNF (20 ng/ml), dexamethasone (100 nM) or TNF+dex] at two time points (10 or 30 minutes) were sequenced in the original study. Raw sequencing data in fastq format as well as non-normalized, strand-differentiated coverage files of hg38-mapped reads in bedGraph format are available for download under this accession series. For the current study, computational analysis to identify regions with bidirectional transcriptional activity (eRNAs) was performed exactly as described^20^ with the following modification: a newly available version ([https://github.com/Dowell-Lab/FStitch](https://github.com/Dowell-Lab/FStitch)) of the Fast Read Stitcher (FStitch) program (v. 1.1.1), with improved efficiency and accuracy impacts on downstream processing, was used for file pre-processing prior to application of Tfit (v. 1.0) to annotate the bidirectional eRNA signatures. Revised eRNA annotations were then subjected to differential expression analysis using DESeq2 to identify differentially transcribed eRNAs (Supplementary Data [1](#MOESM3)) and their inferred sites of RNAPII loading (μ). The GEO accession number for the primary human airway epithelial cell ATAC-seq data (Norm-1 and Norm-2 datasets), described previously^49^, is [GSE157606](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE157606). BEAS-2B ChIP-seq data were published previously^24^, with GEO accession number [GSE79803](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE79803). HASM ChIP-seq data were also described previously^46^, with GEO accession number [GSE95632](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE95632). Here, to create the screenshots depicted in Fig. [3](#Fig3), we remapped the BEAS-2B and HASM ChIP-seq datasets to hg38 using our standardized ChIP-seq pipeline. Briefly, reads are trimmed for adapters, minimum length, and minimum quality using the bbduk tool from the BBMap Suite (v. 38.05). Trimmed reads are then mapped to hg38 using hisat2 (v. 2.1.0). Resulting SAM files are converted to sorted BAM files using samtools (v. 1.9) and to bedGraph coverage format using genomeCoverageBed from the BEDTools suite (v. 2.29.2). Read coverage is normalized to reads per million mapped using a custom python script and files converted to TDF format using igvtools (v. 2.5.3) for visualization in IGV. The BEAS-2B 30-minute wood smoke particle exposure PRO-seq data depicted in Fig. [4](#Fig4) were published previously^52^, with GEO accession number [GSE167371](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE167371).

### Genomic data

We utilized genotype (SNP) and phenotype information (asthma diagnosis, age, sex, body mass index (BMI), length of follow up, smoking status (current, former, never), and the first six ancestral principal components) from the Genetic Epidemiology Research on Adult Health and Aging (GERA) cohort, which includes over 110,000 subjects (68,623 total asthma cases across four ethnic groups) with extensive electronic medical records (EMR) information and genome-wide SNP genotypes (>8 million typed and imputed markers). The QC methods for GERA genomic data were previously described^26^. Briefly, extensive QC was performed on genotype data prior to imputation. The Affymetrix Axiom chip was used for genotyping and the Axiom Analysis Suite and Array Power Tools include a Rare Heterozygous Adjusted Algorithm^90^. Genotypes were pre-phased with Shape-IT v. 2.5 and genetic markers were imputed to 1000 Genomes Project (phase 1 integrated release) reference CEU panels using IMPUTE2. For the eRNA-SNP analysis, we applied a threshold for imputed SNPs to exclude SNPs with imputation quality scores (info r^2^) < 0.9, significantly higher than the commonly applied lower threshold of 0.3-0.5^91^. As GWAS-based associations of genetic risk factors with disease susceptibility can vary significantly by ethnic group within large multi-ethnic cohorts such as GERA^5^, we limited the current study to a subset of the cohort with the highest percentage of asthma cases (16,274 non-Hispanic white asthmatic cases and 38,269 controls).

### μ-SNP co-localization and statistical association with asthma risk

Unless otherwise indicated, R statistical computing software and PLINK^30^ (v. 1.9; [http://pngu.mgh.harvard.edu/purcell/plink](http://pngu.mgh.harvard.edu/purcell/plink)) were used for all analyses.

### Statistical methods

To identify genetic predictors for asthma that were also present in dynamically regulated enhancer regions, we first screened the GERA subset (54,543 samples) to filter SNPs that overlapped the Tfit-annotated enhancer regions significantly regulated by dex or TNF (Supplementary Data [1](#MOESM3)), for each treatment separately. Next, we filtered these SNP sets to include only those that were annotated to regions within 50 bp (+/-) of the Tfit-predicted RNAPII loading site (μ), referred to as μ-SNPs. After sorting by minor allele frequency (MAF), μ-SNPs were categorized as either common (MAF ≥ 0.05) or uncommon (MAF < 0.05) variants and tested independently. Tests for association of additive genotype (categorical variable) with asthma status (dichotomized outcome variable) were conducted using multivariate logistic regression models, adjusted for covariates (listed in the “Genomic data” section above). The significance of associations was determined using a permutation-based approach. One method to improve feature prediction is to combine model-based approaches with permutation. Since each set of dynamically regulated enhancer regions contained relatively few (< 1000) SNPs genotyped in GERA, yielding a small number of tests, far fewer SNPs were interrogated overall as compared to a GWAS, which tests millions of SNPs. In this instance, multiple correction methods traditionally applied in GWAS (e.g., Bonferroni) are overly conservative, or over-correct for Type I error. In contrast to the Chi-square test, permutation-based approaches are appropriate for analysis of rare alleles or smaller numbers of samples and preserve the correlational structure between SNPs, requiring a less stringent multiple test correction threshold. Max(T) permutation was applied with 5000 permutations to generate two sets of empirical (pointwise and familywise) *P* values. The max(T) permutation approach is advantageous for candidate SNP-based approaches, as it is typically sufficient to perform a much smaller number of tests. Following permutation and multiple testing correction using a false-discovery rate (FDR) threshold of 0.05, associations were prioritized based on their level of significance. PLINK v. 1.9 was used to conduct permutation-based analyses while R was employed for SNP filtering, analysis of summary statistics, and prioritization.

### Fine mapping and candidate SNP selection

An in silico fine mapping approach was applied using a web-based annotation tool, SNPnexus^92^ (v. 4; [https://www.snp-nexus.org/v4/](https://www.snp-nexus.org/v4/)), to determine the SNP genomic localization, predicted functions (coding, non-coding, etc.), predicted consequences on protein/gene expression or function, relationship to regulatory elements (e.g., transcription factor motifs, CpG windows), and impact on structural variation (e.g., CNVs). Prior reports of associations with asthma, lung function measures, glucocorticoid response or related phenotypes were also determined through literature search using the NCBI PubMed database. After fine mapping, SNPs were grouped according to whether their respective enhancer regions were up- or down-regulated and compared across groups. The resulting set of SNPs with the strongest predictive evidence for causality and closest proximity to μ were prioritized for functional characterization in vitro.

### Manual refinement of μ and classification of high confidence μ -SNPs and associated target genes

For each of the 36 unique candidate μ-SNPs prioritized for follow-up, we visually examined the GRO-seq tracks within the surrounding genomic region in order to manually refine the bidirectional calls originally made by Tfit. We passed these manually refined regions of bidirectional transcription (Supplementary Table [2](#MOESM1)) to a custom script that calculates the midpoint between the peak signal intensity in each half of the region, resulting in a new estimate of μ for each bidirectional relying both on manual refinement and computational analysis. We then averaged the μ values for any sets of replicates in which the specified bidirectional had robust signal and compared the refined μ estimate to the SNP location. SNPs located within 500 bp (+/-) of the refined μ locations were classified as high confidence μ-SNPs and examined further (Supplementary Data [3](#MOESM5)). A custom python script, in conjunction with bedtools (v. 2.28.0), was used to identify genes within 1 Mb (+/-) of each SNP and append results from differential gene expression analysis (Supplementary Data [4](#MOESM6)) for easy comparison with the relevant μ-SNP bidirectional signature. To account for the short 30-minute GRO-seq treatment timepoint potentially missing true signal changes in longer gene transcripts, a second, “truncated” differential gene expression analysis was performed. Assuming a conservative polymerase processivity rate of 3 kb/min^93^, gene transfer format (GTF) annotation files were generated corresponding to the first 90 kb downstream of all UCSC RefGene-annotated gene TSSs (downloaded August 2019); genes smaller than 90 kb were retained in full. Reads were then counted across these GTF files using the featureCounts algorithm within the R (v. 3.6.1) package Rsubread (v. 3.10) and analyzed with DESeq2 (v. 1.26.0) for differential expression (Supplementary Data [5](#MOESM7)). Genes for which significant expression changes were primarily evident in the truncated DESeq2 analysis, including *NR3C1*, which spans ~150 kb, are asterisked in Supplementary Data [3](#MOESM5); data presented in the Results section for *NR3C1* are from this truncated analysis.

### Cell culture and reagents

BEAS-2B transformed normal human airway epithelial cells (ATCC) were grown in DMEM with L-glutamine and 4.5 g/L glucose (Corning) containing 10% FBS (VWR) and 1% penicillin/streptomycin (pen/strep; Corning). Primary human airway smooth muscle (HASM) cells (provided by Dr. Reynold Panettieri, Rutgers Biomedical Health Sciences) were cultured in Ham’s F12 Medium with L-glutamine (Corning) containing 10% FBS and 1% pen/strep. All cells were maintained in 5% CO_2_ at 37 °C.

Dexamethasone (dex; Sigma-Aldrich) was dissolved in sterile 100% ethanol (vehicle) and used at a final concentration of 100 nM. Recombinant Human TNF-alpha Protein (TNF) purchased from R&D Systems was diluted in sterile 1X Dulbecco’s phosphate buffered saline (DPBS) containing 0.1% bovine serum albumin (BSA) and used at a final concentration of 20 ng/ml. 2,3,7,8-tetrachlorodibenzo-p-dioxin (TCDD) obtained from Cambridge Isotope Laboratories was diluted in sterile DMSO (vehicle) and used at a final concentration of 10 nM. Wood smoke particles were obtained and prepared as described previously^52^ and used at a final concentration of 1 mg/ml.

### Micro-C computational analysis

Paired-end Micro-C data were analyzed using the HiC-Pro^94^(v. 2.11.4) pipeline, which independently aligns R1 vs R2 reads to the hg38 reference genome using bowtie2 (v. 2.2.9) in end-to-end mode. Prior to inputting raw fastq data into HiC-Pro, we merged R1 and R2 reads from each replicate, respectively. HiC-Pro then filters for valid interactions within specified bins of the genome using samtools (v. 1.1), python (v. 2.7), and the python packages bx-python (v. 0.8.9), scipy (v. 1.2.3), pysam (v. 0.16.0.1) and pandas (v. 0.24.2) to generate raw contact counts between corresponding pairs of loci, output as a single coverage file. We configured HiC-Pro to remove all singleton, multimapped, and duplicate reads, and binned reads at 1 kb intervals, in accordance with the highest suggested resolution for Micro-C. To account for systematic biases inherent in chromatin conformation assays, contact counts were normalized using the iced normalization package (v. 0.5.7). Contacts were then filtered to remove those with a normalized count < 1.0 and were visualized as contact maps using the plotgardener library (v. 1.4.2) in R (v. 4.2.3). Micro-C data is deposited under [GSE241294](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE241294).

### Cloning, transfection and luciferase assays

Enhancer reporter constructs were amplified from BEAS-2B genomic DNA by PCR, cloned into pCR2.1-TOPO (Life Technologies), and subsequently ligated into the pGL3-Promoter vector backbone (Life Technologies) using SacI/XhoI (p*COL8A1*) or KpnI/XhoI (all others). p*FKBP5*, p*TNFAIP3*, and p*MT2A* positive control constructs have been described^41,42,50^. The QuikChange II Site-directed Mutagenesis (SDM) Kit from Agilent Technologies was used as instructed by the manufacturer to generate reporter constructs harboring variant alleles (p*GCNT1* C > T, p*NR3C1* C > T, p*rs258753* T > C, p*rs864354* G > T). PCR primer sequences used for cloning and SDM and detailed information pertaining to each reporter construct and related μ-SNP are available in Supplementary Data [6](#MOESM8).

For transfection experiments, BEAS-2B cells were seeded on 96-well tissue culture plates at 20 K cells/well in complete medium. The next day, cells were transfected with a total of 200 ng DNA/well (10:1 ratio of firefly reporter construct to *Renilla* luciferase internal control (pSV40-RL; Promega)) using Lipofectamine 2000 (0.5 ul/well) from Life Technologies. Transfection complexes were removed the following day and cells were treated in fresh complete medium for 8 hr prior to luciferase assay. HASM1 cells were seeded on 96-well tissue culture plates at 50 K cells/well in complete medium. Medium was replaced two days later with fresh complete and the following day, cells were transfected with a total of 125 ng DNA/well (10:1:1 ratio of firefly luciferase to renilla luciferase to mCherry plasmid, respectively) using Lipofectamine 3000 (0.3 ul/well) and P3000 Reagent (0.8 ul/well) from Life Technologies. Transfection complexes were removed the following day and cells were treated in fresh complete medium for 24 hr prior to luciferase assay. HASM2 cells were seeded on 96-well plates at 40 K cells/well in antibiotic-free medium. The next day, cells were transfected with 200 ng total DNA/well (10:1:1 ratio as described for HASM1 cells) using FuGENE6 (1 ul/well) from Promega. Transfection complexes were replaced the following day with fresh complete medium and cells allowed to recover an additional 24 hr prior to 24 hr treatment and luciferase assay.

Luciferase assays were performed using the Dual-Luciferase Reporter Assay System (Promega) and Infinite M1000 Plate Reader (Tecan) as previously described^95^, with the following modifications for cells grown in 96-well plates: each well was lysed in 25 ul 1X lysis reagent and luminescence detected from 5 ul lysate combined with 40 ul of each assay reagent. Each experiment was performed in technical sextuplet using independent wells (except for those employing TCDD, which were run in technical quadruplicate to reduce waste generation) and repeated at least once with qualitatively similar results.

### RNA purification and quantitative RT-PCR (qRT-PCR)

BEAS-2B cells were grown to confluence in 6-well tissue culture plates and treated with vehicle (DMSO) or TCDD (10 nM) for 4 hr. Wells were rinsed 3X in DPBS and then fresh medium containing 100 nM dex was added to all wells for an additional 4 hr. Cells were harvested in TRIzol and RNA purified using the PureLink RNA Mini Kit from Life Technologies prior to qRT-PCR, performed with normalization to *RPL19* as previously described^95^. Primers used for qRT-PCR are described in Supplementary Table [4](#MOESM1).

### Statistical notes

Statistical tests for comparing reporter assays and RT-qPCR data are described in the Figure Legends. In all cases, tests were performed under the assumption of normality of the underlying distributions.

### Reporting summary

Further information on research design is available in the [Nature Portfolio Reporting Summary](#MOESM10) linked to this article.

## Supplementary information

## Source data

## Acknowledgements

Funding was provided as follows: NHLBI R01 HL109557 (to A.N.G.); NHLBI R01 HL152244 (to A.D.); NIH R01 GM125871 (to R.D.D.).

## Author contributions

S.K.S. performed, designed, and analyzed experiments, prepared figures, and cowrote the manuscript. A.D. performed, designed, and analyzed experiments, prepared tables, and co-wrote the manuscript. L.S. and M.A.G. analyzed data and reviewed the manuscript. F.G. and A.G. performed, designed, and analyzed experiments and edited the manuscript. A.C.W. and C.I. reviewed the manuscript and curated data. R.D.D. and S.T.W. edited the manuscript, interpreted data, and designed experimental methods. A.N.G. designed and coordinated the study, interpreted data, and cowrote the manuscript.

## Peer review

### Peer review information

*Nature Communications* thanks Klaus Bønnelykke and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. A peer review file is available.

## Data availability

All genomics data used in this manuscript are publicly available, with accession numbers identified above in the relevant methods sections. All computer tools used are available publicly and are referenced. Information for access to GERA data can be found at: [https://divisionofresearch.kaiserpermanente.org/research/research-program-on-genes-environment-and-health/](https://divisionofresearch.kaiserpermanente.org/research/research-program-on-genes-environment-and-health/). [Source data](#Sec22) are provided with this paper.

## Competing interests

A.N.G. and S.K.S. consult for and own shares in Psammiad Therapeutics; R.D.D. is a cofounder of Arpeggio Biosciences; S.T.W. receives royalties from UpToDate and is on the Scientific Board of Histolix. These external interests are not directly relevant to the research presented herein, and did not influence the presented experiments or their interpretation. All other authors declare no competing interests.

## Footnotes

## Supplementary information

The online version contains supplementary material available at 10.1038/s41467-025-57693-x.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

All genomics data used in this manuscript are publicly available, with accession numbers identified above in the relevant methods sections. All computer tools used are available publicly and are referenced. Information for access to GERA data can be found at: [https://divisionofresearch.kaiserpermanente.org/research/research-program-on-genes-environment-and-health/](https://divisionofresearch.kaiserpermanente.org/research/research-program-on-genes-environment-and-health/). [Source data](#Sec22) are provided with this paper.

### Supplementary Materials

### Data Availability Statement

All genomics data used in this manuscript are publicly available, with accession numbers identified above in the relevant methods sections. All computer tools used are available publicly and are referenced. Information for access to GERA data can be found at: [https://divisionofresearch.kaiserpermanente.org/research/research-program-on-genes-environment-and-health/](https://divisionofresearch.kaiserpermanente.org/research/research-program-on-genes-environment-and-health/). [Source data](#Sec22) are provided with this paper.

## References

1. Bønnelykke, K. & Ober, C. Leveraging gene-environment interactions and endotypes for asthma gene discovery. J. Allergy Clin. Immunol.137, 667–679 (2016).  [DOI](https://doi.org/10.1016/j.jaci.2016.01.006) | [PMC free article](/articles/PMC5004762/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26947980/) | [Google Scholar](https://scholar.google.com/scholar_lookup?B%C3%B8nnelykke,%20K.%20&%20Ober,%20C.%20Leveraging%20gene-environment%20interactions%20and%20endotypes%20for%20asthma%20gene%20discovery.%20J.%20Allergy%20Clin.%20Immunol.137,%20667%E2%80%93679%20(2016).)

2. Kim, K. W. & Ober, C. Lessons learned from GWAS of asthma. Allergy Asthma Immunol. Res. 11, 170–187 (2019).  [DOI](https://doi.org/10.4168/aair.2019.11.2.170) | [PMC free article](/articles/PMC6340805/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30661310/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Kim,%20K.%20W.%20&%20Ober,%20C.%20Lessons%20learned%20from%20GWAS%20of%20asthma.%20Allergy%20Asthma%20Immunol.%20Res.%2011,%20170%E2%80%93187%20(2019).)

3. Sleiman, P. M. et al. Variants of DENND1B associated with asthma in children. N. Engl. J. Med. 362, 36–44 (2010).  [DOI](https://doi.org/10.1056/NEJMoa0901867) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20032318/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Sleiman,%20P.%20M.%20et%20al.%20Variants%20of%20DENND1B%20associated%20with%20asthma%20in%20children.%20N.%20Engl.%20J.%20Med.%20362,%2036%E2%80%9344%20(2010).)

4. Bønnelykke, K. et al. A genome-wide association study identifies CDHR3 as a susceptibility locus for early childhood asthma with severe exacerbations. Nat. Genet46, 51–55 (2014).  [DOI](https://doi.org/10.1038/ng.2830) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24241537/) | [Google Scholar](https://scholar.google.com/scholar_lookup?B%C3%B8nnelykke,%20K.%20et%20al.%20A%20genome-wide%20association%20study%20identifies%20CDHR3%20as%20a%20susceptibility%20locus%20for%20early%20childhood%20asthma%20with%20severe%20exacerbations.%20Nat.%20Genet46,%2051%E2%80%9355%20(2014).)

5. Dahlin, A. et al. Large-scale, multiethnic genome-wide association study identifies novel loci contributing to asthma susceptibility in adults. J. Allergy Clin. Immunol.143, 1633–1635 (2019).  [DOI](https://doi.org/10.1016/j.jaci.2018.11.037) | [PMC free article](/articles/PMC6451881/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30578877/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Dahlin,%20A.%20et%20al.%20Large-scale,%20multiethnic%20genome-wide%20association%20study%20identifies%20novel%20loci%20contributing%20to%20asthma%20susceptibility%20in%20adults.%20J.%20Allergy%20Clin.%20Immunol.143,%201633%E2%80%931635%20(2019).)

6. Long, A., Bunning, B., Sampath, V., DeKruyff, R. H. & Nadeau, K. C. Epigenetics and the Environment in Airway Disease: Asthma and Allergic Rhinitis. Adv. Exp. Med Biol.1253, 153–181 (2020).  [DOI](https://doi.org/10.1007/978-981-15-3449-2_6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32445095/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Long,%20A.,%20Bunning,%20B.,%20Sampath,%20V.,%20DeKruyff,%20R.%20H.%20&%20Nadeau,%20K.%20C.%20Epigenetics%20and%20the%20Environment%20in%20Airway%20Disease:%20Asthma%20and%20Allergic%20Rhinitis.%20Adv.%20Exp.%20Med%20Biol.1253,%20153%E2%80%93181%20(2020).)

7. Augustine, T., Al-Aghbar, M. A., Al-Kowari, M., Espino-Guarch, M. & van Panhuys, N. Asthma and the Missing Heritability Problem: Necessity for Multiomics Approaches in Determining Accurate Risk Profiles. Front Immunol.13, 822324 (2022).  [DOI](https://doi.org/10.3389/fimmu.2022.822324) | [PMC free article](/articles/PMC9174795/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35693821/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Augustine,%20T.,%20Al-Aghbar,%20M.%20A.,%20Al-Kowari,%20M.,%20Espino-Guarch,%20M.%20&%20van%20Panhuys,%20N.%20Asthma%20and%20the%20Missing%20Heritability%20Problem:%20Necessity%20for%20Multiomics%20Approaches%20in%20Determining%20Accurate%20Risk%20Profiles.%20Front%20Immunol.13,%20822324%20(2022).)

8. Vicente, C. T., Revez, J. A. & Ferreira, M. A. R. Lessons from ten years of genome-wide association studies of asthma. Clin. Transl. Immunol.6, e165 (2017).  [DOI](https://doi.org/10.1038/cti.2017.54) | [PMC free article](/articles/PMC5750453/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29333270/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Vicente,%20C.%20T.,%20Revez,%20J.%20A.%20&%20Ferreira,%20M.%20A.%20R.%20Lessons%20from%20ten%20years%20of%20genome-wide%20association%20studies%20of%20asthma.%20Clin.%20Transl.%20Immunol.6,%20e165%20(2017).)

9. Nasser, J. et al. Genome-wide enhancer maps link risk variants to disease genes. Nature593, 238–243 (2021).  [DOI](https://doi.org/10.1038/s41586-021-03446-x) | [PMC free article](/articles/PMC9153265/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33828297/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Nasser,%20J.%20et%20al.%20Genome-wide%20enhancer%20maps%20link%20risk%20variants%20to%20disease%20genes.%20Nature593,%20238%E2%80%93243%20(2021).)

10. Alsheikh, A. J. et al. The landscape of GWAS validation; systematic review identifying 309 validated non-coding variants across 130 human diseases. BMC Med Genomics15, 74 (2022).  [DOI](https://doi.org/10.1186/s12920-022-01216-w) | [PMC free article](/articles/PMC8973751/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35365203/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Alsheikh,%20A.%20J.%20et%20al.%20The%20landscape%20of%20GWAS%20validation;%20systematic%20review%20identifying%20309%20validated%20non-coding%20variants%20across%20130%20human%20diseases.%20BMC%20Med%20Genomics15,%2074%20(2022).)

11. Chen, Z., Boehnke, M., Wen, X. & Mukherjee, B. Revisiting the genome-wide significance threshold for common variant GWAS. G3 (Bethesda)11, jkaa056 (2021).  [DOI](https://doi.org/10.1093/g3journal/jkaa056) | [PMC free article](/articles/PMC8022962/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33585870/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Chen,%20Z.,%20Boehnke,%20M.,%20Wen,%20X.%20&%20Mukherjee,%20B.%20Revisiting%20the%20genome-wide%20significance%20threshold%20for%20common%20variant%20GWAS.%20G3%20(Bethesda)11,%20jkaa056%20(2021).)

12. Ortega, V. E., Meyers, D. A. & Bleecker, E. R. Asthma pharmacogenetics and the development of genetic profiles for personalized medicine. Pharmgenomics Pers. Med. 8, 9–22 (2015).  [DOI](https://doi.org/10.2147/PGPM.S52846) | [PMC free article](/articles/PMC4325626/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25691813/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Ortega,%20V.%20E.,%20Meyers,%20D.%20A.%20&%20Bleecker,%20E.%20R.%20Asthma%20pharmacogenetics%20and%20the%20development%20of%20genetic%20profiles%20for%20personalized%20medicine.%20Pharmgenomics%20Pers.%20Med.%208,%209%E2%80%9322%20(2015).)

13. Hnisz, D. et al. Super-enhancers in the control of cell identity and disease. Cell155, 934–947 (2013).  [DOI](https://doi.org/10.1016/j.cell.2013.09.053) | [PMC free article](/articles/PMC3841062/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24119843/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Hnisz,%20D.%20et%20al.%20Super-enhancers%20in%20the%20control%20of%20cell%20identity%20and%20disease.%20Cell155,%20934%E2%80%93947%20(2013).)

14. Demenais, F. et al. Multiancestry association study identifies new asthma risk loci that colocalize with immune-cell enhancer marks. Nat. Genet50, 42–53 (2018).  [DOI](https://doi.org/10.1038/s41588-017-0014-7) | [PMC free article](/articles/PMC5901974/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29273806/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Demenais,%20F.%20et%20al.%20Multiancestry%20association%20study%20identifies%20new%20asthma%20risk%20loci%20that%20colocalize%20with%20immune-cell%20enhancer%20marks.%20Nat.%20Genet50,%2042%E2%80%9353%20(2018).)

15. Cano-Gamez, E. & Trynka, G. From GWAS to function: using functional genomics to identify the mechanisms underlying complex diseases. Front Genet11, 424 (2020).  [DOI](https://doi.org/10.3389/fgene.2020.00424) | [PMC free article](/articles/PMC7237642/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32477401/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Cano-Gamez,%20E.%20&%20Trynka,%20G.%20From%20GWAS%20to%20function:%20using%20functional%20genomics%20to%20identify%20the%20mechanisms%20underlying%20complex%20diseases.%20Front%20Genet11,%20424%20(2020).)

16. Grosveld, F., van Staalduinen, J. & Stadhouders, R. Transcriptional regulation by (super)enhancers: from discovery to mechanisms. Annu Rev. Genomics Hum. Genet22, 127–146 (2021).  [DOI](https://doi.org/10.1146/annurev-genom-122220-093818) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33951408/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Grosveld,%20F.,%20van%20Staalduinen,%20J.%20&%20Stadhouders,%20R.%20Transcriptional%20regulation%20by%20(super)enhancers:%20from%20discovery%20to%20mechanisms.%20Annu%20Rev.%20Genomics%20Hum.%20Genet22,%20127%E2%80%93146%20(2021).)

17. Currin, K. W. et al. Genetic effects on liver chromatin accessibility identify disease regulatory variants. Am. J. Hum. Genet108, 1169–1189 (2021).  [DOI](https://doi.org/10.1016/j.ajhg.2021.05.001) | [PMC free article](/articles/PMC8323023/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34038741/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Currin,%20K.%20W.%20et%20al.%20Genetic%20effects%20on%20liver%20chromatin%20accessibility%20identify%20disease%20regulatory%20variants.%20Am.%20J.%20Hum.%20Genet108,%201169%E2%80%931189%20(2021).)

18. Kim, T. K. et al. Widespread transcription at neuronal activity-regulated enhancers. Nature465, 182–187 (2010).  [DOI](https://doi.org/10.1038/nature09033) | [PMC free article](/articles/PMC3020079/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20393465/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Kim,%20T.%20K.%20et%20al.%20Widespread%20transcription%20at%20neuronal%20activity-regulated%20enhancers.%20Nature465,%20182%E2%80%93187%20(2010).)

19. Dukler, N. et al. Nascent RNA sequencing reveals a dynamic global transcriptional response at genes and enhancers to the natural medicinal compound celastrol. Genome Res. 27, 1816–1829 (2017).  [DOI](https://doi.org/10.1101/gr.222935.117) | [PMC free article](/articles/PMC5668940/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29025894/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Dukler,%20N.%20et%20al.%20Nascent%20RNA%20sequencing%20reveals%20a%20dynamic%20global%20transcriptional%20response%20at%20genes%20and%20enhancers%20to%20the%20natural%20medicinal%20compound%20celastrol.%20Genome%20Res.%2027,%201816%E2%80%931829%20(2017).)

20. Sasse, S. K. et al. Nascent transcript analysis of glucocorticoid crosstalk with TNF defines primary and cooperative inflammatory repression. Genome Res. 29, 1753–1765 (2019).  [DOI](https://doi.org/10.1101/gr.248187.119) | [PMC free article](/articles/PMC6836729/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31519741/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Sasse,%20S.%20K.%20et%20al.%20Nascent%20transcript%20analysis%20of%20glucocorticoid%20crosstalk%20with%20TNF%20defines%20primary%20and%20cooperative%20inflammatory%20repression.%20Genome%20Res.%2029,%201753%E2%80%931765%20(2019).)

21. Franco, H. L., Nagari, A. & Kraus, W. L. TNFalpha signaling exposes latent estrogen receptor binding sites to alter the breast cancer cell transcriptome. Mol. Cell58, 21–34 (2015).  [DOI](https://doi.org/10.1016/j.molcel.2015.02.001) | [PMC free article](/articles/PMC4385449/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25752574/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Franco,%20H.%20L.,%20Nagari,%20A.%20&%20Kraus,%20W.%20L.%20TNFalpha%20signaling%20exposes%20latent%20estrogen%20receptor%20binding%20sites%20to%20alter%20the%20breast%20cancer%20cell%20transcriptome.%20Mol.%20Cell58,%2021%E2%80%9334%20(2015).)

22. Mikhaylichenko, O. et al. The degree of enhancer or promoter activity is reflected by the levels and directionality of eRNA transcription. Genes Dev.32, 42–57 (2018).  [DOI](https://doi.org/10.1101/gad.308619.117) | [PMC free article](/articles/PMC5828394/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29378788/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Mikhaylichenko,%20O.%20et%20al.%20The%20degree%20of%20enhancer%20or%20promoter%20activity%20is%20reflected%20by%20the%20levels%20and%20directionality%20of%20eRNA%20transcription.%20Genes%20Dev.32,%2042%E2%80%9357%20(2018).)

23. Azofeifa, J. G. et al. Enhancer RNA profiling predicts transcription factor activity. Genome Res. 28, 334–344 (2018).  [DOI](https://doi.org/10.1101/gr.225755.117) | [PMC free article](/articles/PMC5848612/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29449408/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Azofeifa,%20J.%20G.%20et%20al.%20Enhancer%20RNA%20profiling%20predicts%20transcription%20factor%20activity.%20Genome%20Res.%2028,%20334%E2%80%93344%20(2018).)

24. Kadiyala, V. et al. Cistrome-based cooperation between airway epithelial glucocorticoid receptor and NF-kappaB orchestrates anti-inflammatory effects. J. Biol. Chem.291, 12673–12687 (2016).  [DOI](https://doi.org/10.1074/jbc.M116.721217) | [PMC free article](/articles/PMC4933445/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27076634/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Kadiyala,%20V.%20et%20al.%20Cistrome-based%20cooperation%20between%20airway%20epithelial%20glucocorticoid%20receptor%20and%20NF-kappaB%20orchestrates%20anti-inflammatory%20effects.%20J.%20Biol.%20Chem.291,%2012673%E2%80%9312687%20(2016).)

25. Azofeifa, J. G. & Dowell, R. D. A generative model for the behavior of RNA polymerase. Bioinformatics33, 227–234 (2017).  [DOI](https://doi.org/10.1093/bioinformatics/btw599) | [PMC free article](/articles/PMC5942361/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27663494/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Azofeifa,%20J.%20G.%20&%20Dowell,%20R.%20D.%20A%20generative%20model%20for%20the%20behavior%20of%20RNA%20polymerase.%20Bioinformatics33,%20227%E2%80%93234%20(2017).)

26. Kvale, M. N. et al. Genotyping Informatics and quality control for 100,000 subjects in the genetic epidemiology research on adult health and aging (GERA) cohort. Genetics200, 1051–1060 (2015).  [DOI](https://doi.org/10.1534/genetics.115.178905) | [PMC free article](/articles/PMC4574249/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26092718/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Kvale,%20M.%20N.%20et%20al.%20Genotyping%20Informatics%20and%20quality%20control%20for%20100,000%20subjects%20in%20the%20genetic%20epidemiology%20research%20on%20adult%20health%20and%20aging%20(GERA)%20cohort.%20Genetics200,%201051%E2%80%931060%20(2015).)

27. Yoosefzadeh-Najafabadi, M., Eskandari, M., Belzile, F. & Torkamaneh, D. Genome-wide association study statistical models: a review. Methods Mol. Biol.2481, 43–62 (2022).  [DOI](https://doi.org/10.1007/978-1-0716-2237-7_4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35641758/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Yoosefzadeh-Najafabadi,%20M.,%20Eskandari,%20M.,%20Belzile,%20F.%20&%20Torkamaneh,%20D.%20Genome-wide%20association%20study%20statistical%20models:%20a%20review.%20Methods%20Mol.%20Biol.2481,%2043%E2%80%9362%20(2022).)

28. Gerber, A. N., Newton, R. & Sasse, S. K. Repression of transcription by the glucocorticoid receptor: a parsimonious model for the genomics era. J. Biol. Chem.296, 100687 (2021).  [DOI](https://doi.org/10.1016/j.jbc.2021.100687) | [PMC free article](/articles/PMC8141881/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33891947/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Gerber,%20A.%20N.,%20Newton,%20R.%20&%20Sasse,%20S.%20K.%20Repression%20of%20transcription%20by%20the%20glucocorticoid%20receptor:%20a%20parsimonious%20model%20for%20the%20genomics%20era.%20J.%20Biol.%20Chem.296,%20100687%20(2021).)

29. Schmidt, S. F., Larsen, B. D., Loft, A. & Mandrup, S. Cofactor squelching: artifact or fact? Bioessays38, 618–626 (2016).  [DOI](https://doi.org/10.1002/bies.201600034) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27273739/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Schmidt,%20S.%20F.,%20Larsen,%20B.%20D.,%20Loft,%20A.%20&%20Mandrup,%20S.%20Cofactor%20squelching:%20artifact%20or%20fact?%20Bioessays38,%20618%E2%80%93626%20(2016).)

30. Purcell, S. et al. PLINK: a tool set for whole-genome association and population-based linkage analyses. Am. J. Hum. Genet81, 559–575 (2007).  [DOI](https://doi.org/10.1086/519795) | [PMC free article](/articles/PMC1950838/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17701901/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Purcell,%20S.%20et%20al.%20PLINK:%20a%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses.%20Am.%20J.%20Hum.%20Genet81,%20559%E2%80%93575%20(2007).)

31. Granell, R. et al. A meta-analysis of genome-wide association studies of childhood wheezing phenotypes identifies ANXA1 as a susceptibility locus for persistent wheezing. Elife12, e84315 (2023).  [DOI](https://doi.org/10.7554/eLife.84315) | [PMC free article](/articles/PMC10292845/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/37227431/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Granell,%20R.%20et%20al.%20A%20meta-analysis%20of%20genome-wide%20association%20studies%20of%20childhood%20wheezing%20phenotypes%20identifies%20ANXA1%20as%20a%20susceptibility%20locus%20for%20persistent%20wheezing.%20Elife12,%20e84315%20(2023).)

32. Gras, D. et al. Human bronchial epithelium orchestrates dendritic cell activation in severe asthma. Eur. Respir. J.49, 1602399 (2017).  [DOI](https://doi.org/10.1183/13993003.02399-2016) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28275176/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Gras,%20D.%20et%20al.%20Human%20bronchial%20epithelium%20orchestrates%20dendritic%20cell%20activation%20in%20severe%20asthma.%20Eur.%20Respir.%20J.49,%201602399%20(2017).)

33. Qiu, Y., Zhu, J., Bandi, V., Guntupalli, K. K. & Jeffery, P. K. Bronchial mucosal inflammation and upregulation of CXC chemoattractants and receptors in severe exacerbations of asthma. Thorax62, 475–482 (2007).  [DOI](https://doi.org/10.1136/thx.2006.066670) | [PMC free article](/articles/PMC2117215/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17234659/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Qiu,%20Y.,%20Zhu,%20J.,%20Bandi,%20V.,%20Guntupalli,%20K.%20K.%20&%20Jeffery,%20P.%20K.%20Bronchial%20mucosal%20inflammation%20and%20upregulation%20of%20CXC%20chemoattractants%20and%20receptors%20in%20severe%20exacerbations%20of%20asthma.%20Thorax62,%20475%E2%80%93482%20(2007).)

34. Kobayashi, Y. et al. Impaired dual-specificity protein phosphatase DUSP4 reduces corticosteroid sensitivity. Mol. Pharm.91, 475–481 (2017).  [DOI](https://doi.org/10.1124/mol.116.107656) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28283554/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Kobayashi,%20Y.%20et%20al.%20Impaired%20dual-specificity%20protein%20phosphatase%20DUSP4%20reduces%20corticosteroid%20sensitivity.%20Mol.%20Pharm.91,%20475%E2%80%93481%20(2017).)

35. Xia, Y., Cao, H., Zheng, J. & Chen, L. Claudin-1 mediated tight junction dysfunction as a contributor to atopic march. Front Immunol.13, 927465 (2022).  [DOI](https://doi.org/10.3389/fimmu.2022.927465) | [PMC free article](/articles/PMC9277052/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35844593/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Xia,%20Y.,%20Cao,%20H.,%20Zheng,%20J.%20&%20Chen,%20L.%20Claudin-1%20mediated%20tight%20junction%20dysfunction%20as%20a%20contributor%20to%20atopic%20march.%20Front%20Immunol.13,%20927465%20(2022).)

36. Fujita, H. et al. Claudin-1 expression in airway smooth muscle exacerbates airway remodeling in asthmatic subjects. J. Allergy Clin. Immunol.127, 1612–21.e8 (2011).  [DOI](https://doi.org/10.1016/j.jaci.2011.03.039) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21624620/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Fujita,%20H.%20et%20al.%20Claudin-1%20expression%20in%20airway%20smooth%20muscle%20exacerbates%20airway%20remodeling%20in%20asthmatic%20subjects.%20J.%20Allergy%20Clin.%20Immunol.127,%201612%E2%80%9321.e8%20(2011).)

37. Hopkins, D. R., Keles, S. & Greenspan, D. S. The bone morphogenetic protein 1/Tolloid-like metalloproteinases. Matrix Biol.26, 508–523 (2007).  [DOI](https://doi.org/10.1016/j.matbio.2007.05.004) | [PMC free article](/articles/PMC2722432/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17560775/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Hopkins,%20D.%20R.,%20Keles,%20S.%20&%20Greenspan,%20D.%20S.%20The%20bone%20morphogenetic%20protein%201/Tolloid-like%20metalloproteinases.%20Matrix%20Biol.26,%20508%E2%80%93523%20(2007).)

38. Cozza, G. & Salvi, M. The acidophilic kinases PLK2 and PLK3: structure, substrate targeting and inhibition. Curr. Protein Pept. Sci.19, 728–745 (2018).  [DOI](https://doi.org/10.2174/1389203719666180124095405) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29366414/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Cozza,%20G.%20&%20Salvi,%20M.%20The%20acidophilic%20kinases%20PLK2%20and%20PLK3:%20structure,%20substrate%20targeting%20and%20inhibition.%20Curr.%20Protein%20Pept.%20Sci.19,%20728%E2%80%93745%20(2018).)

39. Hochrainer, K. et al. The human HERC family of ubiquitin ligases: novel members, genomic organization, expression profiling, and evolutionary aspects. Genomics85, 153–164 (2005).  [DOI](https://doi.org/10.1016/j.ygeno.2004.10.006) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15676274/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Hochrainer,%20K.%20et%20al.%20The%20human%20HERC%20family%20of%20ubiquitin%20ligases:%20novel%20members,%20genomic%20organization,%20expression%20profiling,%20and%20evolutionary%20aspects.%20Genomics85,%20153%E2%80%93164%20(2005).)

40. Poghosyan, A., Patel, J. K., Clifford, R. L. & Knox, A. J. Epigenetic dysregulation of interleukin 8 (CXCL8) hypersecretion in cystic fibrosis airway epithelial cells. Biochem Biophys. Res Commun.476, 431–437 (2016).  [DOI](https://doi.org/10.1016/j.bbrc.2016.05.140) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27240956/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Poghosyan,%20A.,%20Patel,%20J.%20K.,%20Clifford,%20R.%20L.%20&%20Knox,%20A.%20J.%20Epigenetic%20dysregulation%20of%20interleukin%208%20(CXCL8)%20hypersecretion%20in%20cystic%20fibrosis%20airway%20epithelial%20cells.%20Biochem%20Biophys.%20Res%20Commun.476,%20431%E2%80%93437%20(2016).)

41. Hubler, T. R. & Scammell, J. G. Intronic hormone response elements mediate regulation of FKBP5 by progestins and glucocorticoids. Cell Stress Chaperones9, 243–252 (2004).  [DOI](https://doi.org/10.1379/CSC-32R.1) | [PMC free article](/articles/PMC1065283/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15544162/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Hubler,%20T.%20R.%20&%20Scammell,%20J.%20G.%20Intronic%20hormone%20response%20elements%20mediate%20regulation%20of%20FKBP5%20by%20progestins%20and%20glucocorticoids.%20Cell%20Stress%20Chaperones9,%20243%E2%80%93252%20(2004).)

42. Altonsy, M. O., Sasse, S. K., Phang, T. L. & Gerber, A. N. Context-dependent cooperation between nuclear factor kappaB (NF-kappaB) and the glucocorticoid receptor at a TNFAIP3 intronic enhancer: a mechanism to maintain negative feedback control of inflammation. J. Biol. Chem.289, 8231–8239 (2014).  [DOI](https://doi.org/10.1074/jbc.M113.545178) | [PMC free article](/articles/PMC3961651/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24500711/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Altonsy,%20M.%20O.,%20Sasse,%20S.%20K.,%20Phang,%20T.%20L.%20&%20Gerber,%20A.%20N.%20Context-dependent%20cooperation%20between%20nuclear%20factor%20kappaB%20(NF-kappaB)%20and%20the%20glucocorticoid%20receptor%20at%20a%20TNFAIP3%20intronic%20enhancer:%20a%20mechanism%20to%20maintain%20negative%20feedback%20control%20of%20inflammation.%20J.%20Biol.%20Chem.289,%208231%E2%80%938239%20(2014).)

43. Cartharius, K. et al. MatInspector and beyond: promoter analysis based on transcription factor binding sites. Bioinformatics21, 2933–2942 (2005).  [DOI](https://doi.org/10.1093/bioinformatics/bti473) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15860560/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Cartharius,%20K.%20et%20al.%20MatInspector%20and%20beyond:%20promoter%20analysis%20based%20on%20transcription%20factor%20binding%20sites.%20Bioinformatics21,%202933%E2%80%932942%20(2005).)

44. Fonseca, K. L. et al. Deficiency in the glycosyltransferase Gcnt1 increases susceptibility to tuberculosis through a mechanism involving neutrophils. Mucosal Immunol.13, 836–848 (2020).  [DOI](https://doi.org/10.1038/s41385-020-0277-7) | [PMC free article](/articles/PMC7434595/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32203062/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Fonseca,%20K.%20L.%20et%20al.%20Deficiency%20in%20the%20glycosyltransferase%20Gcnt1%20increases%20susceptibility%20to%20tuberculosis%20through%20a%20mechanism%20involving%20neutrophils.%20Mucosal%20Immunol.13,%20836%E2%80%93848%20(2020).)

45. Recto, K. A. et al. Transcriptome-wide association study of circulating IgE levels identifies novel targets for asthma and allergic diseases. Front Immunol.14, 1080071 (2023).  [DOI](https://doi.org/10.3389/fimmu.2023.1080071) | [PMC free article](/articles/PMC9922991/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36793728/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Recto,%20K.%20A.%20et%20al.%20Transcriptome-wide%20association%20study%20of%20circulating%20IgE%20levels%20identifies%20novel%20targets%20for%20asthma%20and%20allergic%20diseases.%20Front%20Immunol.14,%201080071%20(2023).)

46. Sasse, S. K. et al. Glucocorticoid receptor ChIP-seq Identifies PLCD1 as a KLF15 target that represses airway smooth muscle hypertrophy. Am. J. Respir. Cell Mol. Biol.57, 226–237(2017).  [DOI](https://doi.org/10.1165/rcmb.2016-0357OC) | [PMC free article](/articles/PMC5576583/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28375666/)

47. El Kharbili, M. et al. Noncoding SNPs decrease expression of FABP5 during COPD exacerbations. J. Clin. Invest134, e175626 (2023).  [DOI](https://doi.org/10.1172/JCI175626) | [PMC free article](/articles/PMC10849757/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/38113113/) | [Google Scholar](https://scholar.google.com/scholar_lookup?El%20Kharbili,%20M.%20et%20al.%20Noncoding%20SNPs%20decrease%20expression%20of%20FABP5%20during%20COPD%20exacerbations.%20J.%20Clin.%20Invest134,%20e175626%20(2023).)

48. Krietenstein, N. et al. Ultrastructural details of mammalian chromosome architecture. Mol. Cell78, 554–565.e7 (2020).  [DOI](https://doi.org/10.1016/j.molcel.2020.03.003) | [PMC free article](/articles/PMC7222625/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32213324/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Krietenstein,%20N.%20et%20al.%20Ultrastructural%20details%20of%20mammalian%20chromosome%20architecture.%20Mol.%20Cell78,%20554%E2%80%93565.e7%20(2020).)

49. Gally, F. et al. The MUC5B-associated variant rs35705950 resides within an enhancer subject to lineage- and disease-dependent epigenetic remodeling. JCI Insight6, e144294 (2021).  [DOI](https://doi.org/10.1172/jci.insight.144294) | [PMC free article](/articles/PMC7934873/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33320836/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Gally,%20F.%20et%20al.%20The%20MUC5B-associated%20variant%20rs35705950%20resides%20within%20an%20enhancer%20subject%20to%20lineage-%20and%20disease-dependent%20epigenetic%20remodeling.%20JCI%20Insight6,%20e144294%20(2021).)

50. Sasse, S. K. et al. Response Element Composition Governs Correlations between Binding Site Affinity and Transcription in Glucocorticoid Receptor Feed-forward Loops. J. Biol. Chem.290, 19756–19769 (2015).  [DOI](https://doi.org/10.1074/jbc.M115.668558) | [PMC free article](/articles/PMC4528137/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26088140/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Sasse,%20S.%20K.%20et%20al.%20Response%20Element%20Composition%20Governs%20Correlations%20between%20Binding%20Site%20Affinity%20and%20Transcription%20in%20Glucocorticoid%20Receptor%20Feed-forward%20Loops.%20J.%20Biol.%20Chem.290,%2019756%E2%80%9319769%20(2015).)

51. Jacobi, A. M. et al. Simplified CRISPR tools for efficient genome editing and streamlined protocols for their delivery into mammalian cells and mouse zygotes. Methods121-122, 16–28 (2017).  [DOI](https://doi.org/10.1016/j.ymeth.2017.03.021) | [PMC free article](/articles/PMC5761324/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28351759/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Jacobi,%20A.%20M.%20et%20al.%20Simplified%20CRISPR%20tools%20for%20efficient%20genome%20editing%20and%20streamlined%20protocols%20for%20their%20delivery%20into%20mammalian%20cells%20and%20mouse%20zygotes.%20Methods121-122,%2016%E2%80%9328%20(2017).)

52. Gupta, A. et al. Deconvolution of multiplexed transcriptional responses to wood smoke particles defines rapid aryl hydrocarbon receptor signaling dynamics. J. Biol. Chem.297, 101147 (2021).  [DOI](https://doi.org/10.1016/j.jbc.2021.101147) | [PMC free article](/articles/PMC8517214/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34520756/)

53. Young, T. M. et al. Identifying toxicologically significant compounds in urban wildfire ash using in vitro bioassays and high-resolution mass spectrometry. Environ. Sci. Technol.55, 3657–3667 (2021).  [DOI](https://doi.org/10.1021/acs.est.0c06712) | [PMC free article](/articles/PMC8351470/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33647203/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Young,%20T.%20M.%20et%20al.%20Identifying%20toxicologically%20significant%20compounds%20in%20urban%20wildfire%20ash%20using%20in%20vitro%20bioassays%20and%20high-resolution%20mass%20spectrometry.%20Environ.%20Sci.%20Technol.55,%203657%E2%80%933667%20(2021).)

54. Ghetu, C. C. et al. Wildfire impact on indoor and outdoor PAH Air quality. Environ. Sci. Technol.56, 10042–10052 (2022).  [DOI](https://doi.org/10.1021/acs.est.2c00619) | [PMC free article](/articles/PMC9301925/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35803593/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Ghetu,%20C.%20C.%20et%20al.%20Wildfire%20impact%20on%20indoor%20and%20outdoor%20PAH%20Air%20quality.%20Environ.%20Sci.%20Technol.56,%2010042%E2%80%9310052%20(2022).)

55. Wentworth, G. R., Aklilu, Y. A., Landis, M. S. & Hsu, Y. M. Impacts of a large boreal wildfire on ground level atmospheric concentrations of PAHs, VOCs and ozone. Atmos. Environ.178, 19–30 (2018). (1994).  [DOI](https://doi.org/10.1016/j.atmosenv.2018.01.013) | [PMC free article](/articles/PMC5906807/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29681759/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Wentworth,%20G.%20R.,%20Aklilu,%20Y.%20A.,%20Landis,%20M.%20S.%20&%20Hsu,%20Y.%20M.%20Impacts%20of%20a%20large%20boreal%20wildfire%20on%20ground%20level%20atmospheric%20concentrations%20of%20PAHs,%20VOCs%20and%20ozone.%20Atmos.%20Environ.178,%2019%E2%80%9330%20(2018).%20(1994).)

56. Sorg, O. AhR signalling and dioxin toxicity. Toxicol. Lett.230, 225–233 (2014).  [DOI](https://doi.org/10.1016/j.toxlet.2013.10.039) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24239782/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Sorg,%20O.%20AhR%20signalling%20and%20dioxin%20toxicity.%20Toxicol.%20Lett.230,%20225%E2%80%93233%20(2014).)

57. Shrine, N. et al. New genetic signals for lung function highlight pathways and chronic obstructive pulmonary disease associations across multiple ancestries. Nat. Genet.51, 481–493 (2019).  [DOI](https://doi.org/10.1038/s41588-018-0321-7) | [PMC free article](/articles/PMC6397078/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30804560/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Shrine,%20N.%20et%20al.%20New%20genetic%20signals%20for%20lung%20function%20highlight%20pathways%20and%20chronic%20obstructive%20pulmonary%20disease%20associations%20across%20multiple%20ancestries.%20Nat.%20Genet.51,%20481%E2%80%93493%20(2019).)

58. Castelluccio, J. F. et al. Study of correlation between forced vital capacity and demand for healthcare services in severe asthmatics. Multidiscip. Respiratory Med.10, 22 (2015).  [DOI](https://doi.org/10.1186/s40248-015-0020-5) | [PMC free article](/articles/PMC4508817/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26199726/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Castelluccio,%20J.%20F.%20et%20al.%20Study%20of%20correlation%20between%20forced%20vital%20capacity%20and%20demand%20for%20healthcare%20services%20in%20severe%20asthmatics.%20Multidiscip.%20Respiratory%20Med.10,%2022%20(2015).)

59. Ackermann-Liebrich, U. et al. Lung function and long term exposure to air pollutants in Switzerland. Study on Air Pollution and Lung Diseases in Adults (SAPALDIA) Team. Am. J. Respir. Crit. Care Med155, 122–129 (1997).  [DOI](https://doi.org/10.1164/ajrccm.155.1.9001300) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9001300/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Ackermann-Liebrich,%20U.%20et%20al.%20Lung%20function%20and%20long%20term%20exposure%20to%20air%20pollutants%20in%20Switzerland.%20Study%20on%20Air%20Pollution%20and%20Lung%20Diseases%20in%20Adults%20(SAPALDIA)%20Team.%20Am.%20J.%20Respir.%20Crit.%20Care%20Med155,%20122%E2%80%93129%20(1997).)

60. Miller, A. & Palecki, A. Restrictive impairment in patients with asthma. Respir. Med. 101, 272–276 (2007).  [DOI](https://doi.org/10.1016/j.rmed.2006.05.008) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16797172/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Miller,%20A.%20&%20Palecki,%20A.%20Restrictive%20impairment%20in%20patients%20with%20asthma.%20Respir.%20Med.%20101,%20272%E2%80%93276%20(2007).)

61. Abbey, D. E. et al. Long-term particulate and other air pollutants and lung function in nonsmokers. Am. J. Respir. Crit. Care Med. 158, 289–298 (1998).  [DOI](https://doi.org/10.1164/ajrccm.158.1.9710101) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9655742/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Abbey,%20D.%20E.%20et%20al.%20Long-term%20particulate%20and%20other%20air%20pollutants%20and%20lung%20function%20in%20nonsmokers.%20Am.%20J.%20Respir.%20Crit.%20Care%20Med.%20158,%20289%E2%80%93298%20(1998).)

62. Grant, C. E., Bailey, T. L. & Noble, W. S. FIMO: scanning for occurrences of a given motif. Bioinformatics27, 1017–1018 (2011).  [DOI](https://doi.org/10.1093/bioinformatics/btr064) | [PMC free article](/articles/PMC3065696/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21330290/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Grant,%20C.%20E.,%20Bailey,%20T.%20L.%20&%20Noble,%20W.%20S.%20FIMO:%20scanning%20for%20occurrences%20of%20a%20given%20motif.%20Bioinformatics27,%201017%E2%80%931018%20(2011).)

63. Lipták, T. On the combination of independent tests. Magy. Tud. Akad. Mat. Kut. Int Kozl3, 171–197 (1958).  [Google Scholar](https://scholar.google.com/scholar_lookup?Lipt%C3%A1k,%20T.%20On%20the%20combination%20of%20independent%20tests.%20Magy.%20Tud.%20Akad.%20Mat.%20Kut.%20Int%20Kozl3,%20171%E2%80%93197%20(1958).)

64. Hauberg, M. E. et al. Differential activity of transcribed enhancers in the prefrontal cortex of 537 cases with schizophrenia and controls. Mol. Psychiatry24, 1685–1695 (2019).  [DOI](https://doi.org/10.1038/s41380-018-0059-8) | [PMC free article](/articles/PMC6222027/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29740122/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Hauberg,%20M.%20E.%20et%20al.%20Differential%20activity%20of%20transcribed%20enhancers%20in%20the%20prefrontal%20cortex%20of%20537%20cases%20with%20schizophrenia%20and%20controls.%20Mol.%20Psychiatry24,%201685%E2%80%931695%20(2019).)

65. Danko, C. G. et al. Identification of active transcriptional regulatory elements from GRO-seq data. Nat. Methods12, 433–438 (2015).  [DOI](https://doi.org/10.1038/nmeth.3329) | [PMC free article](/articles/PMC4507281/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25799441/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Danko,%20C.%20G.%20et%20al.%20Identification%20of%20active%20transcriptional%20regulatory%20elements%20from%20GRO-seq%20data.%20Nat.%20Methods12,%20433%E2%80%93438%20(2015).)

66. Forrest, A. R. R. et al. A promoter-level mammalian expression atlas. Nature507, 462–470 (2014).  [DOI](https://doi.org/10.1038/nature13182) | [PMC free article](/articles/PMC4529748/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24670764/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Forrest,%20A.%20R.%20R.%20et%20al.%20A%20promoter-level%20mammalian%20expression%20atlas.%20Nature507,%20462%E2%80%93470%20(2014).)

67. Panigrahi, A. & O’Malley, B. W. Mechanisms of enhancer action: the known and the unknown. Genome Biol.22, 108 (2021).  [DOI](https://doi.org/10.1186/s13059-021-02322-1) | [PMC free article](/articles/PMC8051032/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33858480/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Panigrahi,%20A.%20&%20O%E2%80%99Malley,%20B.%20W.%20Mechanisms%20of%20enhancer%20action:%20the%20known%20and%20the%20unknown.%20Genome%20Biol.22,%20108%20(2021).)

68. Thurman, R. E. et al. The accessible chromatin landscape of the human genome. Nature489, 75–82 (2012).  [DOI](https://doi.org/10.1038/nature11232) | [PMC free article](/articles/PMC3721348/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22955617/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Thurman,%20R.%20E.%20et%20al.%20The%20accessible%20chromatin%20landscape%20of%20the%20human%20genome.%20Nature489,%2075%E2%80%9382%20(2012).)

69. Sanyal, A., Lajoie, B. R., Jain, G. & Dekker, J. The long-range interaction landscape of gene promoters. Nature489, 109–113 (2012).  [DOI](https://doi.org/10.1038/nature11279) | [PMC free article](/articles/PMC3555147/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22955621/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Sanyal,%20A.,%20Lajoie,%20B.%20R.,%20Jain,%20G.%20&%20Dekker,%20J.%20The%20long-range%20interaction%20landscape%20of%20gene%20promoters.%20Nature489,%20109%E2%80%93113%20(2012).)

70. Furlong, E. E. M. & Levine, M. Developmental enhancers and chromosome topology. Science361, 1341–1345 (2018).  [DOI](https://doi.org/10.1126/science.aau0320) | [PMC free article](/articles/PMC6986801/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30262496/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Furlong,%20E.%20E.%20M.%20&%20Levine,%20M.%20Developmental%20enhancers%20and%20chromosome%20topology.%20Science361,%201341%E2%80%931345%20(2018).)

71. Andersson, R. et al. An atlas of active enhancers across human cell types and tissues. Nature507, 455–461 (2014).  [DOI](https://doi.org/10.1038/nature12787) | [PMC free article](/articles/PMC5215096/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24670763/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Andersson,%20R.%20et%20al.%20An%20atlas%20of%20active%20enhancers%20across%20human%20cell%20types%20and%20tissues.%20Nature507,%20455%E2%80%93461%20(2014).)

72. Cannavò, E. et al. Shadow enhancers are pervasive features of developmental regulatory networks. Curr. Biol.26, 38–51 (2016).  [DOI](https://doi.org/10.1016/j.cub.2015.11.034) | [PMC free article](/articles/PMC4712172/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26687625/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Cannav%C3%B2,%20E.%20et%20al.%20Shadow%20enhancers%20are%20pervasive%20features%20of%20developmental%20regulatory%20networks.%20Curr.%20Biol.26,%2038%E2%80%9351%20(2016).)

73. Edris, A. et al. Asthma exacerbations and eosinophilia in the UK Biobank: a genome-wide association study. ERJ Open Res.10, 00566–02023 (2024).  [DOI](https://doi.org/10.1183/23120541.00566-2023) | [PMC free article](/articles/PMC10772900/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/38196893/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Edris,%20A.%20et%20al.%20Asthma%20exacerbations%20and%20eosinophilia%20in%20the%20UK%20Biobank:%20a%20genome-wide%20association%20study.%20ERJ%20Open%20Res.10,%2000566%E2%80%9302023%20(2024).)

74. Kantor, D. B. et al. Replication and fine mapping of asthma-associated loci in individuals of African ancestry. Hum. Genet132, 1039–1047 (2013).  [DOI](https://doi.org/10.1007/s00439-013-1310-7) | [PMC free article](/articles/PMC3975655/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23666277/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Kantor,%20D.%20B.%20et%20al.%20Replication%20and%20fine%20mapping%20of%20asthma-associated%20loci%20in%20individuals%20of%20African%20ancestry.%20Hum.%20Genet132,%201039%E2%80%931047%20(2013).)

75. Baye, T. M. et al. Differences in candidate gene association between european ancestry and african american asthmatic children. PLOS ONE6, e16522 (2011).  [DOI](https://doi.org/10.1371/journal.pone.0016522) | [PMC free article](/articles/PMC3046166/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21387019/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Baye,%20T.%20M.%20et%20al.%20Differences%20in%20candidate%20gene%20association%20between%20european%20ancestry%20and%20african%20american%20asthmatic%20children.%20PLOS%20ONE6,%20e16522%20(2011).)

76. Chang, X. et al. Genetic architecture of asthma in African American patients. J. Allergy Clin. Immunol.151, 1132–1136 (2023).  [DOI](https://doi.org/10.1016/j.jaci.2022.09.001) | [PMC free article](/articles/PMC9992439/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36089080/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Chang,%20X.%20et%20al.%20Genetic%20architecture%20of%20asthma%20in%20African%20American%20patients.%20J.%20Allergy%20Clin.%20Immunol.151,%201132%E2%80%931136%20(2023).)

77. Greene, C. S., Penrod, N. M., Williams, S. M. & Moore, J. H. Failure to replicate a genetic association may provide important clues about genetic architecture. PLoS One4, e5639 (2009).  [DOI](https://doi.org/10.1371/journal.pone.0005639) | [PMC free article](/articles/PMC2685469/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19503614/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Greene,%20C.%20S.,%20Penrod,%20N.%20M.,%20Williams,%20S.%20M.%20&%20Moore,%20J.%20H.%20Failure%20to%20replicate%20a%20genetic%20association%20may%20provide%20important%20clues%20about%20genetic%20architecture.%20PLoS%20One4,%20e5639%20(2009).)

78. Perkey, E. et al. GCNT1-mediated O-glycosylation of the sialomucin CD43 Is a sensitive indicator of notch signaling in activated T Cells. J. Immunol.204, 1674–1688 (2020).  [DOI](https://doi.org/10.4049/jimmunol.1901194) | [PMC free article](/articles/PMC7306398/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32060138/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Perkey,%20E.%20et%20al.%20GCNT1-mediated%20O-glycosylation%20of%20the%20sialomucin%20CD43%20Is%20a%20sensitive%20indicator%20of%20notch%20signaling%20in%20activated%20T%20Cells.%20J.%20Immunol.204,%201674%E2%80%931688%20(2020).)

79. Kathryn, R. et al. Transcriptome-wide association study of circulating IgE levels identifies novel targets for asthma and allergic diseases. medRxiv08, 20176479 (2020). 2020.  [DOI](https://doi.org/10.3389/fimmu.2023.1080071) | [PMC free article](/articles/PMC9922991/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36793728/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Kathryn,%20R.%20et%20al.%20Transcriptome-wide%20association%20study%20of%20circulating%20IgE%20levels%20identifies%20novel%20targets%20for%20asthma%20and%20allergic%20diseases.%20medRxiv08,%2020176479%20(2020).%202020.)

80. Chen, G. et al. SPDEF is required for mouse pulmonary goblet cell differentiation and regulates a network of genes associated with mucus production. J. Clin. Invest119, 2914–2924 (2009).  [DOI](https://doi.org/10.1172/JCI39731) | [PMC free article](/articles/PMC2752084/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19759516/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Chen,%20G.%20et%20al.%20SPDEF%20is%20required%20for%20mouse%20pulmonary%20goblet%20cell%20differentiation%20and%20regulates%20a%20network%20of%20genes%20associated%20with%20mucus%20production.%20J.%20Clin.%20Invest119,%202914%E2%80%932924%20(2009).)

81. Weng, C. M. et al. Aryl hydrocarbon receptor activation by diesel exhaust particles mediates epithelium-derived cytokines expression in severe allergic asthma. Allergy73, 2192–2204 (2018).  [DOI](https://doi.org/10.1111/all.13462) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29672862/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Weng,%20C.%20M.%20et%20al.%20Aryl%20hydrocarbon%20receptor%20activation%20by%20diesel%20exhaust%20particles%20mediates%20epithelium-derived%20cytokines%20expression%20in%20severe%20allergic%20asthma.%20Allergy73,%202192%E2%80%932204%20(2018).)

82. Mason, G. G. Dioxin-receptor ligands in urban air and vehicle exhaust. Environ. Health Perspect.102, 111–116 (1994).  [DOI](https://doi.org/10.1289/ehp.94102s4111) | [PMC free article](/articles/PMC1566942/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7529698/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Mason,%20G.%20G.%20Dioxin-receptor%20ligands%20in%20urban%20air%20and%20vehicle%20exhaust.%20Environ.%20Health%20Perspect.102,%20111%E2%80%93116%20(1994).)

83. Perera, F. & Nadeau, K. Climate change, fossil-fuel pollution, and children’s health. N. Engl. J. Med. 386, 2303–2314 (2022).  [DOI](https://doi.org/10.1056/NEJMra2117706) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35704482/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Perera,%20F.%20&%20Nadeau,%20K.%20Climate%20change,%20fossil-fuel%20pollution,%20and%20children%E2%80%99s%20health.%20N.%20Engl.%20J.%20Med.%20386,%202303%E2%80%932314%20(2022).)

84. Xu, R. et al. Wildfires, global climate change, and human health. N. Engl. J. Med.383, 2173–2181 (2020).  [DOI](https://doi.org/10.1056/NEJMsr2028985) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33034960/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Xu,%20R.%20et%20al.%20Wildfires,%20global%20climate%20change,%20and%20human%20health.%20N.%20Engl.%20J.%20Med.383,%202173%E2%80%932181%20(2020).)

85. Guarnieri, M. & Balmes, J. R. Outdoor air pollution and asthma. Lancet383, 1581–1592 (2014).  [DOI](https://doi.org/10.1016/S0140-6736(14)60617-6) | [PMC free article](/articles/PMC4465283/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24792855/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Guarnieri,%20M.%20&%20Balmes,%20J.%20R.%20Outdoor%20air%20pollution%20and%20asthma.%20Lancet383,%201581%E2%80%931592%20(2014).)

86. Lewis, T. C. et al. Air pollution and respiratory symptoms among children with asthma: vulnerability by corticosteroid use and residence area. Sci. Total Environ.448, 48–55 (2013).  [DOI](https://doi.org/10.1016/j.scitotenv.2012.11.070) | [PMC free article](/articles/PMC4327853/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23273373/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Lewis,%20T.%20C.%20et%20al.%20Air%20pollution%20and%20respiratory%20symptoms%20among%20children%20with%20asthma:%20vulnerability%20by%20corticosteroid%20use%20and%20residence%20area.%20Sci.%20Total%20Environ.448,%2048%E2%80%9355%20(2013).)

87. Ierodiakonou, D. et al. Ambient air pollution, lung function, and airway responsiveness in asthmatic children. J. Allergy Clin. Immunol.137, 390–399 (2016).  [DOI](https://doi.org/10.1016/j.jaci.2015.05.028) | [PMC free article](/articles/PMC4742428/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26187234/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Ierodiakonou,%20D.%20et%20al.%20Ambient%20air%20pollution,%20lung%20function,%20and%20airway%20responsiveness%20in%20asthmatic%20children.%20J.%20Allergy%20Clin.%20Immunol.137,%20390%E2%80%93399%20(2016).)

88. Zora, J. E. et al. Associations between urban air pollution and pediatric asthma control in El Paso, Texas. Sci. Total Environ.448, 56–65 (2013).  [DOI](https://doi.org/10.1016/j.scitotenv.2012.11.067) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23312496/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Zora,%20J.%20E.%20et%20al.%20Associations%20between%20urban%20air%20pollution%20and%20pediatric%20asthma%20control%20in%20El%20Paso,%20Texas.%20Sci.%20Total%20Environ.448,%2056%E2%80%9365%20(2013).)

89. Gomez, L. et al. Modification of asthma clinical trial treatment efficacy by social and environmental exposures. ISEE Conference Abstracts2021. 10.1289/isee.2021.O-TO-084 (2021).

90. Mizrahi-Man, O. et al. Novel genotyping algorithms for rare variants significantly improve the accuracy of Applied Biosystems™ Axiom™ array genotyping calls: retrospective evaluation of UK biobank array data. PLoS One17, e0277680 (2022).  [DOI](https://doi.org/10.1371/journal.pone.0277680) | [PMC free article](/articles/PMC9671364/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36395175/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Mizrahi-Man,%20O.%20et%20al.%20Novel%20genotyping%20algorithms%20for%20rare%20variants%20significantly%20improve%20the%20accuracy%20of%20Applied%20Biosystems%E2%84%A2%20Axiom%E2%84%A2%20array%20genotyping%20calls:%20retrospective%20evaluation%20of%20UK%20biobank%20array%20data.%20PLoS%20One17,%20e0277680%20(2022).)

91. Roshyara, N. R., Kirsten, H., Horn, K., Ahnert, P. & Scholz, M. Impact of pre-imputation SNP-filtering on genotype imputation results. BMC Genet.15, 88 (2014).  [DOI](https://doi.org/10.1186/s12863-014-0088-5) | [PMC free article](/articles/PMC4236550/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25112433/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Roshyara,%20N.%20R.,%20Kirsten,%20H.,%20Horn,%20K.,%20Ahnert,%20P.%20&%20Scholz,%20M.%20Impact%20of%20pre-imputation%20SNP-filtering%20on%20genotype%20imputation%20results.%20BMC%20Genet.15,%2088%20(2014).)

92. Oscanoa, J. et al. SNPnexus: a web server for functional annotation of human genome sequence variation (2020 update). Nucleic Acids Res. 48, W185–w192 (2020).  [DOI](https://doi.org/10.1093/nar/gkaa420) | [PMC free article](/articles/PMC7319579/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32496546/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Oscanoa,%20J.%20et%20al.%20SNPnexus:%20a%20web%20server%20for%20functional%20annotation%20of%20human%20genome%20sequence%20variation%20(2020%20update).%20Nucleic%20Acids%20Res.%2048,%20W185%E2%80%93w192%20(2020).)

93. Jonkers, I., Kwak, H. & Lis, J. T. Genome-wide dynamics of Pol II elongation and its interplay with promoter proximal pausing, chromatin, and exons. eLife3, e02407 (2014).  [DOI](https://doi.org/10.7554/eLife.02407) | [PMC free article](/articles/PMC4001325/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24843027/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Jonkers,%20I.,%20Kwak,%20H.%20&%20Lis,%20J.%20T.%20Genome-wide%20dynamics%20of%20Pol%20II%20elongation%20and%20its%20interplay%20with%20promoter%20proximal%20pausing,%20chromatin,%20and%20exons.%20eLife3,%20e02407%20(2014).)

94. Servant, N. et al. HiC-Pro: an optimized and flexible pipeline for Hi-C data processing. Genome Biol.16, 259 (2015).  [DOI](https://doi.org/10.1186/s13059-015-0831-x) | [PMC free article](/articles/PMC4665391/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26619908/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Servant,%20N.%20et%20al.%20HiC-Pro:%20an%20optimized%20and%20flexible%20pipeline%20for%20Hi-C%20data%20processing.%20Genome%20Biol.16,%20259%20(2015).)

95. Sasse, S. K. et al. The glucocorticoid receptor and KLF15 regulate gene expression dynamics and integrate signals through feed-forward circuitry. Mol. Cell Biol.33, 2104–2115 (2013).  [DOI](https://doi.org/10.1128/MCB.01474-12) | [PMC free article](/articles/PMC3648070/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23508109/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Sasse,%20S.%20K.%20et%20al.%20The%20glucocorticoid%20receptor%20and%20KLF15%20regulate%20gene%20expression%20dynamics%20and%20integrate%20signals%20through%20feed-forward%20circuitry.%20Mol.%20Cell%20Biol.33,%202104%E2%80%932115%20(2013).)
