# PDE4 Gene Family Variants Are Associated with Response to Apremilast Treatment in Psoriasis

## Metadata
**Authors:** Kalliopi Liadaki, Efterpi Zafiriou, Themistoklis Giannoulis, Sofia Alexouda, Kleoniki Chaidaki, Polyxeni Gidarokosta, Angeliki-Viktoria Roussaki-Schulze, Sotirios G Tsiogkas, Athina Daponte, Zissis Mamuris, Dimitrios P Bogdanos, Nicholas K Moschonas, Theologia Sarafidou
**Journal:** Genes
**Date:** 2024 Mar 17
**DOI:** [10.3390/genes15030369](https://doi.org/10.3390/genes15030369)
**PMID:** 38540428
**PMCID:** PMC10970167
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10970167/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC10970167/pdf/genes-15-00369.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC10970167/pdf/genes-15-00369.pdf)

## Abstract

Moderate-to-severe psoriasis (Ps) treatment includes systemic drugs and biological agents. Apremilast, a small molecule primarily metabolized by cytochrome CYP3A4, modulates the immune system by specifically inhibiting phosphodiesterase type 4 (PDE4) isoforms and is currently used for the treatment of Ps and psoriatic arthritis (PsA). Clinical trials and real-world data showed variable efficacy in response among Ps patients underlying the need for personalized therapy. This study implements a candidate-gene and a network-based approach to identify genetic markers associated with apremilast response in forty-nine Greek Ps patients. Our data revealed an association of sixty-four SNPs within or near PDE4 and CYP3A4 genes, four SNPs in ncRNAs ANRIL, LINC00941 and miR4706, which influence the abundance or function of PDE4s, and thirty-three SNPs within fourteen genes whose protein products either interact directly with PDE4 proteins or constitute components of the cAMP signaling pathway which is modulated by PDE4s. Notably, fifty-six of the aforementioned SNPs constitute eQTLs for the respective genes in relevant to psoriasis tissues/cells implying that these variants could be causal. Our analysis provides a number of novel genetic variants that, upon validation in larger cohorts, could be utilized as predictive markers regarding the response of Ps patients to apremilast treatment.

Keywords: apremilast, psoriasis, association study, SNP, PDE4, CYP3A4, protein interactome, pathway analysis

## 1. Introduction

Psoriasis (Ps) is a chronic autoimmune inflammatory disease with a multifactorial etiology and a prevalence estimate of 2–3% worldwide [[1](#B1-genes-15-00369)]. The management of moderate-to-severe psoriasis involves various treatment approaches, i.e., first-line systemic drugs, small molecules, and biological agents [[2](#B2-genes-15-00369)]. Although these treatments are generally effective, they show considerable variation in response rates among patients [[3](#B3-genes-15-00369)]. It is estimated that 30–50% of the patients present an insufficient response to various treatments, which depends to a significant degree on the genetic background [[4](#B4-genes-15-00369)]. Furthermore, an appreciable number of patients, who exhibit an initial response to treatment, experience a secondary loss of response, while others discontinue treatment due to side effects [[5](#B5-genes-15-00369),[6](#B6-genes-15-00369)]. Hence, the identification and implementation of valid prognostic biomarkers in the context of precision medicine are of high importance for the protection of patients’ quality of life as well as for cost reduction in healthcare.

Apremilast, a small molecule which specifically inhibits the 3′,5′ cyclic adenosine monophosphate (cAMP) specific phosphodiesterase type 4 (PDE4) and modulates the immune system, has been added to the armamentarium for the efficacious treatment of moderate-to-severe plaque Ps, psoriatic arthritis (PsA) and oral ulcers of Behcet’s disease in adults after FDA approval in 2014 [[7](#B7-genes-15-00369)]. Due to its low molecular weight, apremilast is easily absorbed from the gut and transferred into the cells where it hinders PDE4 [[8](#B8-genes-15-00369)]. PDE4, a major subtype of PDEs expressed by immune or inflammatory cells, is a well conserved enzyme encoded by four genes, *PDE4A*, *B*, *C* and *D* mapped to chromosomes 19p13.2, 1p31.3, 19p13.11, and 5p11.2-q12.1, respectively. Due to extensive alternative splicing and/or the use of different promoters, they generate multiple alternatively spliced transcripts producing many different functional isoforms [[9](#B9-genes-15-00369)]. It has been shown by in vitro experiments that apremilast specifically inhibits the PDE4B1, PDE4C1, PDE4B2, PDE4D2 and PDE4A1A isoforms without targeting any cell surface receptors and kinases [[10](#B10-genes-15-00369)]. Inhibition of PDE4s results in an increase in the intracellular second messenger cAMP as its hydrolysis to 5′-AMP is performed exclusively by these enzymes. Consequently, various signaling pathways are modified in immune cells, resulting in the decrease in proinflammatory cytokines and the increase in anti-inflammatory mediators [[11](#B11-genes-15-00369)]. Apremilast is primarily metabolized by cytochrome 450 (CYP-450) 3A4, whose expression and activity in the liver and small intestine is highly variable in part due to genetic factors [[12](#B12-genes-15-00369)].

Although apremilast has advantages, i.e., oral administration, lack of immunogenicity, acceptable levels of safety and tolerance, and relatively low cost, different response rates have been reported. Earlier phase III trials have shown that after 16 weeks of treatment 29–41% of the patients showed at least a 75% reduction in the Psoriasis Area and Severity Index (PASI) score [[13](#B13-genes-15-00369),[14](#B14-genes-15-00369),[15](#B15-genes-15-00369)]. Accordingly, subsequent real-world data showed variable efficacy, underlying the need for personalized therapy [[16](#B16-genes-15-00369),[17](#B17-genes-15-00369),[18](#B18-genes-15-00369)]. In this context, there is only one published pharmacogenomics study examining the association between SNPs and apremilast response outcome concerning 34 Russian patients with Ps [[19](#B19-genes-15-00369)]. The authors performed a genome-wide association study (GWAS) in combination with an analysis of pre-selected SNPs, based on their association with Ps or PsA, and identified SNPs associated with apremilast response which map in four genomic regions of unknown function and in *IL-1β*, *IL-4*, *IL-23R*, and *TNF-α* genes.

Typical pharmacogenetic analyses aiming to determine genetic markers for drug response primarily focus on polymorphisms on either direct drug targets or drug metabolizing enzymes. In this aspect, we performed an association study on polymorphisms inside or adjacent to *PDE4A*, *B*, *C*, *D* and *CYP3A4* and extend the analysis to ncRNAs and proteins that may affect PDE4 function and therefore the response to apremilast treatment, in a cohort of 49 Greek patients.

## 2. Materials and Methods

### 2.1. Patients

Forty-nine adult patients diagnosed with Ps vulgaris who had been under treatment with apremilast (trade name Otezla, 30 mg taken orally twice/day) for at least 6 months were included in the study. The patients were recruited and followed-up at the Dermatology Department of the General University Hospital of Larissa (GUHL), Greece. Patients’ response was evaluated based on the decrease in the PASI score at 6 and 9 months of treatment and a reduction greater than 75% classified them as responders. Responder patients receiving apremilast in combination with another treatment as well as patients who exhibited a marginal positive or negative response to apremilast were excluded from the study. This study was conducted after informed consent of the patients with the approval of the Research Ethics Committee of GUHL (11/2nd/08-02-2022) in accordance with the Declaration of Helsinki. Patients’ peripheral blood and DNA samples were semi-anonymized with a code number and treated as confidential.

### 2.2. Samples, Genotyping, Quality Control and Association Analysis

DNA extraction was performed from peripheral whole blood using the PureLink Genomic DNA Kit (Thermo Fisher Scientific, Waltham, MA, USA) according to the manufacturer’s instructions. DNA integrity/quality was assessed through agarose gel electrophoresis and the quantification was performed with a Quawell spectrophotometer. DNA concentration was adjusted to 50 ng/μL. Genotype screening was performed at the Human Genomics Facility (HuGe-F) of Erasmus MC (University Medical Centre, Rotterdam, The Netherlands) using 200 ng DNA per sample and the Illumina Infinium Global Screening Array MD v3.0 which contains approximately 730,000 SNP variants.

The genotyping data were obtained in .ped and .map files. Quality control steps were applied that filtered out individuals and SNPs with call rate < 99%, SNPs with minor allele frequency (MAF) < 5% and linkage disequilibrium (LD) > 0.8 based on r^2^ values. The genomic coordinates for all the genes tested for association with apremilast response were retrieved from the Ensembl database ([https://www.ensembl.org/index.html](https://www.ensembl.org/index.html) accessed on 15 December 2023), ref. [[20](#B20-genes-15-00369)] using the GRCh37 assembly in which all the genotyping data derived from HuGe-F were available. Based on these coordinates, an association analysis for allelic, dominant, recessive and genotypic models was implemented. The odds ratio was calculated with 95% confidence interval based on the comparative risk of non-response in the presence of the minor allele. All the above analyses were performed using the PLINK software (PLINK 1.9) [[21](#B21-genes-15-00369)]. Furthermore, the clinical characteristics of the patients were assessed for association with the response to apremilast applying the appropriate statistics using R Studio.

### 2.3. ncRNA Analysis

The LncRNA2Target v3 database [[22](#B22-genes-15-00369)], which contains exclusively experimentally validated data, was used to retrieve all the human lncRNAs/mRNA pairs and the respective Ensembl IDs. SNPs located within these lncRNAs were tested for association with the treatment response as described in [Section 2.2](#sec2dot2-genes-15-00369).

Τhe miRTarBase [[23](#B23-genes-15-00369)], which includes only experimentally verified miRNA-mRNA interactions, was searched to track down all the miRNAs that target *PDE4* or *CYP3A4* mRNAs as well as the miRNA-related SNPs. The miRNA identifiers were used as queries in the Ensembl database and those that did not correspond to a unique gene or were discontinued were excluded from further analysis. Furthermore, miRNA-related variants that did not map to a unique genomic location according to the Ensembl database were also excluded. The remaining miRNA-related SNPs were investigated for LD data in all available populations using the Linkage Disequilibrium Calculator tool provided by the Ensembl database, setting the window size at 0.5 Mb and the threshold for r^2^ and D’ at 0.7. The list of SNPs that either map within the miRNAs or display high LD with them were examined for association with the response to apremilast as described in [Section 2.2](#sec2dot2-genes-15-00369).

### 2.4. Protein-Network and Pathway Analysis

Protein–protein interaction (PPI) data for human PDE4 proteins were extracted from the PICKLE meta-database (version 3.3) [[24](#B24-genes-15-00369),[25](#B25-genes-15-00369)], using the “first neighbors” network setup and the “cross-checking” filtering method which includes only the experimentally validated with high confidence direct PPI output. The Cytoscape software platform ([[26](#B26-genes-15-00369)], version 3.8.2, [http://www.cytoscape.org/](http://www.cytoscape.org/) accessed on 10 January 2024) was used for the reconstruction of the protein interactome. The Kyoto Encyclopedia of Genes and Genomes (KEGG) database was used to retrieve the cAMP signaling pathway (hsa04024) which is regulated by PDE4 proteins. We created a simplified version of the pathway focusing on the signal transduction relevant to inflammation psoriasis. The SNPs within the genes encoding the proteins of the PPI network and the modified KEGG pathway were tested for association with the response to apremilast as described in [Section 2.2](#sec2dot2-genes-15-00369).

### 2.5. Expression Quantitative Trait Loci (eQTL) Analysis

The data for the eQTL analysis were derived from the Ensembl database by setting > 1.3 as the threshold of the *p* value (−log10). The *p* values were converted to non-log values. From the list of tissue/cells in which the variant of interest was found to be correlated with the respective gene expression, only the skin, blood and immune cells were selected for all the genes except for *CYP3A4*, for which the liver and small intestine were selected. The effect size of the eQTLs refers to the effect of the alternative versus the reference allele.

## 3. Results

### 3.1. Clinical Characteristics of the Study Population

Forty-nine patients, 32 males and 17 females, with Ps vulgaris who were treated with apremilast were enrolled in this study, and their clinical characteristics were analyzed. Their mean age was 55.4, and the mean disease duration was 18.4 years. At baseline, the mean PASI was 10.3, the mean BMI was 31.6, and the mean weight was 92.8 kg. In addition to BMI, the baseline weight was also included in the analysis due to the body-weight-independent treatment dosing of apremilast. Comorbidities were reported in approximately 70% of the patients. In accordance with real-world data, the proportion of the patients that achieved PASI score improvement greater than 75% after 24–36 weeks of treatment was 45% [[16](#B16-genes-15-00369),[17](#B17-genes-15-00369),[18](#B18-genes-15-00369)]. Thus, the responder and non-responder groups include 22 and 27 patients, respectively. [Table 1](#genes-15-00369-t001) shows the main characteristics of the study population. Their sex, age, disease duration, baseline BMI and weight are not correlated with the efficacy of apremilast. However, a higher absolute baseline PASI is associated with lower treatment response.

### Table 1.

|   | All Patients (n = 49) | Responders (n = 22) | Non-Responders (n = 27) | p Value |
| --- | --- | --- | --- | --- |
| Male/Female (M/F) | 32/17 | 13/9 | 19/8 | 0.601 |
| Mean baseline age (mean ± s.d.) | 55.4 (±15) | 54.8 (±13.2) | 55.9 (±16.4) | 0.954 |
| Disease duration (years) | 18.4 (±13.1) | 17.1 (±10.5) | 19.2 (±14.6) | 0.634 |
| Baseline weight (kg), M/F | 92.8 (±19.4), 98/83.2 | 90.5 (±20.3), 95.9/82.4 | 94.7 (±18.9), 93.3/84.1 | 0.469 |
| Baseline BMI | 31.6 (±5.7) | 32.3 (±7.3) | 31.1 (±4.3) | 0.513 |
| Baseline PASI | 10.3 (±6.6) | 8.5 (±3.2) | 11.8 (±8.3) | 0.036 |
| PASI at 6 months | 6.1 (±6.4) | 1.7 (±1.4) | 9.8 (±6.6) | 0.003 |
| PASI at 9 months | 4.7 (±4.5) | 1.3 (±0.9) | 7.6 (±4.2) | 0.007 |

Table 1 Caption: Patients’ physical and clinical characteristics.

### 3.2. PDE4 and CYP3A4 Variants Are Associated with Response to Treatment

Following the genotyping data quality control, 3892 SNPs which are mapped within a physical distance of approximately ±1 Mb of PDE4A, PDE4B, PDE4C, PDE4D and CYP3A4 genes’ chromosomal location were selected. A subsequent filter removed 2628 SNPs with MAF < 5% and pruned for LD > 0.8 based on r^2^ values which further excluded 280 SNPs. Thus, for the association analysis regarding apremilast response, 984 SNPs were finally employed. Allele-based association tests detected 64 SNPs displaying nominal *p* values < 0.05 ([Table 2](#genes-15-00369-t002)) including 10 SNPs with *p* values ≤ 0.01. Apart from the allelic association, SNP rs1045895 also shows association with the response under the dominant model of inheritance ([Table 2](#genes-15-00369-t002)). Interestingly, 15 SNPs associated with the apremilast response map within intronic regions of PDE4B, PDE4D and CYP3A4.

### Table 2.

| Locus | SNP | Location, HGVS Name | F_R | F_NR | MAF R + NR | MAF (EUR) | p Value | OR |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
| u. PDE4A | rs12979813 (A/G) | Intergenic, chr19,NC_000019.10:g.11232027 | 0.4545 | 0.2593 | 0.35 | 0.14 | 4.3 × 10−2 | 0.42 |
| rs2305795 * (A/G) | Intergenic, chr19,NC_000019.10:g.10115376 | 0.5909 | 0.3889 | 0.48 | 0.40 | 4.6 × 10−2 | 0.44 |  |
| d. PDE4A | rs76966440 (C/T) | Intergenic, chr19, NC_000019.10:g.10687087 | 0.1136 | 0 | 0.05 | 0.02 | 1.1 × 10−2 | 0 |
| rs892085 * (A/G) | Intergenic, chr19, NC_000019.10:g.10707416 | 0.6136 | 0.3889 | 0.49 | 0.48 | 2.7 × 10−2 | 0.40 |  |
| rs74942924 (G/A) | Intergenic, chr19,NC_000019.10:g.10952989 | 0 | 0.0926 | 0.05 | 0.03 | 3.8 × 10−2 | NA |  |
| rs11085752 * (A/C) | Intergenic, chr19,NC_000019.10:g.10954733 | 0.6136 | 0.3889 | 0.49 | 0.42 | 2.7 × 10−2 | 0.40 |  |
| rs7935 (T/C) | Intergenic, chr19,NC_000019.10:g.10994932 | 0.5227 | 0.3148 | 0.41 | 0.35 | 3.7 × 10−2 | 0.42 |  |
| rs12980863 (C/T) | Intergenic, chr19, NC_000019.10:g.11199195 | 0.2955 | 0.5000 | 0.41 | 0.39 | 4 × 10−2 | 2.39 |  |
| rs12979813 (A/G) | Intergenic, chr19, NC_000019.10:g.11232027 | 0.0682 | 0.2222 | 0.15 | 0.14 | 3.5 × 10−2 | 3.91 |  |
| rs420703 (C/A) | Intergenic, chr19, NC_000019.10:g.11301192 | 0.3409 | 0.5741 | 0.47 | 0.30 | 2.1 × 10−2 | 2.61 |  |
| rs322151 * (C/T) | Intergenic, chr19, NC_000019.10:g.11306524 | 0.2727 | 0.4815 | 0.39 | 0.23 | 3.5 × 10−2 | 2.48 |  |
| rs322144 (G/C) | Intergenic, chr19, NC_000019.10:g.11313027 | 0.3409 | 0.6296 | 0.50 | 0.56 | 4 × 10−3 | 3.29 |  |
| rs10424035 (G/A) | Intergenic, chr19, NC_000019.10:g.11400322 | 0 | 0.0926 | 0.05 | 0.01 | 3.8 × 10−2 | NA |  |
| u. PDE4B | rs883824 (G/A) | Intergenic, chr1, NC_000001.11:g.64830174 | 0.0455 | 0.1852 | 0.12 | 0.18 | 3.6 × 10−2 | 4.77 |
| rs1045895 (G/A) | Intergenic, chr1, NC_000001.11:g.65432298 | 0.3409 | 0.5556 | 0.46 | 0.40 | 3.4 × 10−2 | 2.42 |  |
| rs1045895 (G/A) dom † |   | 10 GA + AA/12 GG | 21 GA + AA/6 GG |   |   | 2 × 10−2 |   |  |
| PDE4B | rs61799396 (C/T) | Intron, chr1, NC_000001.11:g.65956761 | 0.3409 | 0.1667 | 0.24 | 0.25 | 4.6 × 10−2 | 0.39 |
| rs1937457 (C/T) | Intron, chr1,NC_000001.11:g.65981104 | 0.4318 | 0.2222 | 0.32 | 0.29 | 2.6 × 10−2 | 0.38 |  |
| rs12757542 (A/G) | Intron, chr1, NC_000001.11:g.66030425 | 0.3864 | 0.1667 | 0.27 | 0.28 | 1.4 × 10−2 | 0.32 |  |
| rs12406476 (T/C) | Intron, chr1, NC_000001.11:g.66090109 | 0.3409 | 0.5556 | 0.46 | 0.52 | 3.4 × 10−2 | 2.42 |  |
| rs12745871 (C/T) | Intron, chr1, NC_000001.11:g.66121693 | 0.6136 | 0.3148 | 0.45 | 0.34 | 3 × 10−3 | 0.29 |  |
| rs2503174 (A/G) | Intron, chr1, NC_000001.11:g.66143471 | 0.0909 | 0.2593 | 0.18 | 0.32 | 3.2 × 10−2 | 3.50 |  |
| rs2485381 (A/G) | Intron, chr1, NC_000001.11:g.66162049 | 0.0227 | 0.1481 | 0.09 | 0.17 | 3.2 × 10−2 | 7.48 |  |
| rs1890196 (T/C) | Intron, chr 1,NC_000001.11:g.66337397 | 0.3182 | 0.5741 | 0.46 | 0.48 | 1.1 × 10−2 | 2.89 |  |
| d. PDE4B | rs11208847 (T/C) | Intergenic, chr1, NC_000001.11:g.66400358 | 0.0909 | 0.2593 | 0.18 | 0.22 | 3.2 × 10−2 | 3.50 |
| rs12118088 (A/G) | Intergenic, chr1, NC_000001.11:g.66479505 | 0 | 0.2037 | 0.11 | 0.09 | 1 × 10−3 | NA |  |
| rs765685 (G/A) | Intergenic, chr1, NC_000001.11:g.66499792 | 0.5455 | 0.3333 | 0.43 | 0.48 | 3.5 × 10−2 | 0.42 |  |
| rs11809759 (A/G) | Intergenic, chr1, NC_000001.11:g.66804222 | 0.4773 | 0.2778 | 0.37 | 0.41 | 4.2 × 10−2 | 0.42 |  |
| rs2815378 (C/T) | Intergenic, chr1, NC_000001.11:g.67044015 | 0.0682 | 0.3148 | 0.20 | 0.30 | 3 × 10−3 | 6.28 |  |
| rs77040148 (A/G) | Intergenic, chr1, NC_000001.11:g.67138150 | 0 | 0.1296 | 0.07 | 0.05 | 1.3 × 10−2 | NA |  |
| d. PDE4C | rs62128111 (C/T) | Intergenic, chr19, NC_000019.10:g.17352405 | 0.1591 | 0.4074 | 0.30 | 0.27 | 7 × 10−3 | 3.63 |
| rs12972417 (C/T) | Intergenic, chr19, NC_000019.10:g.17414505 | 0.0909 | 0.2778 | 0.19 | 0.22 | 2 × 10−2 | 3.85 |  |
| rs10410487 (C/T) | Intergenic, chr19,NC_000019.10:g.17718799 | 0.3636 | 0.5741 | 0.48 | 0.43 | 3.8 × 10−2 | 2.36 |  |
| rs12151113 (T/C) | Intergenic, chr19, NC_000019.10:g.17748561 | 0.5909 | 0.3333 | 0.45 | 0.51 | 1.1 × 10−2 | 0.35 |  |
| rs2161489 (C/T) | Intergenic, chr19,NC_000019.10:g.17753643 | 0 | 0.1296 | 0.07 | 0.07 | 1.3 × 10−2 | NA |  |
| rs8108865 (C/T) | Intergenic, chr19,NC_000019.10:g.17761260 | 0.2500 | 0.0556 | 0.14 | 0.19 | 6 × 10−3 | 0.18 |  |
| rs4808096 (A/G) | Intergenic, chr19,NC_000019.10:g.17771371 | 0 | 0.0926 | 0.05 | 0.05 | 3.8 × 10−2 | NA |  |
| rs897753 (A/C) | Intergenic, chr19,NC_000019.10:g.18043766 | 0.3409 | 0.1481 | 0.23 | 0.20 | 2.5 × 10−2 | 0.34 |  |
| u. PDE4C | rs4808781 (G/A) | Intergenic, chr 19,NC_000019.10:g.18306162 | 0.0682 | 0.3333 | 0.21 | 0.23 | 1 × 10−3 | 6.83 |
| rs76580597 (C/T) | Intergenic, chr 19,NC_000019.10:g.18317822 | 0 | 0.0926 | 0.05 | 0.05 | 3.8 × 10−2 | NA |  |
| rs10423674 (C/A) | Intergenic, chr 19,NC_000019.10:g.18707093 | 0.4773 | 0.2778 | 0.37 | 0.32 | 4.2 × 10−2 | 0.42 |  |
| rs2023878 (C/T) | Intergenic, chr 19,NC_000019.10:g.18723314 | 0.3182 | 0.1481 | 0.22 | 0.21 | 4.5 × 10−2 | 0.37 |  |
| rs4808907 (T/C) | Intergenic, chr 19,NC_000019.10:g.19025732 | 0.1818 | 0.4074 | 0.31 | 0.23 | 1.6 × 10−2 | 3.09 |  |
| d. PDE4D | rs11960699 (A/C) | Intergenic, chr 5,NC_000005.10:g.58013923 | 0.2955 | 0.5185 | 0.42 | 0.40 | 2.6 × 10−2 | 2.57 |
| rs11741819 (T/C) | Intergenic, chr 5,NC_000005.10:g.58014172 | 0.1364 | 0.3333 | 0.24 | 0.23 | 2.4 × 10−2 | 3.17 |  |
| rs72753331 (G/A) | Intergenic, chr 5,NC_000005.10:g.58044825 | 0.0682 | 0.2222 | 0.15 | 0.07 | 3.5 × 10−2 | 3.91 |  |
| rs16887824 (C/T) | Intergenic, chr 5,NC_000005.10:g.58109683 | 0.3409 | 0.1111 | 0.21 | 0.18 | 6 × 10−3 | 0.24 |  |
| rs9283718 (G/A) | Intergenic, chr 5,NC_000005.10:g.58115743 | 0.3182 | 0.5185 | 0.43 | 0.51 | 4.6 × 10−2 | 2.31 |  |
| rs11742503 (A/G) | Intergenic, chr 5,NC_000005.10:g.58234354 | 0.3182 | 0.1481 | 0.22 | 0.18 | 4.5 × 10−2 | 0.37 |  |
| rs115791568 (A/G) | Intergenic, chr 5,NC_000005.10:g.58742066 | 0.1136 | 0 | 0.05 | 0.04 | 1.1 × 10−2 | 0 |  |
| rs963443 (G/A) | Intergenic, chr 5,NC_000005.10:g.58775972 | 0.3409 | 0.5926 | 0.48 | 0.41 | 1.3 × 10−2 | 2.81 |  |
| PDE4D | rs78103527 (T/C) | Intron, chr 5,NC_000005.10:g.59381162 | 0 | 0.0926 | 0.05 | 0.02 | 3.8 × 10−2 | NA |
| rs697076 (C/T) | Intron, chr 5,NC_000005.10:g.59555985 | 0.1364 | 0.3519 | 0.26 | 0.23 | 1.5 × 10−2 | 3.44 |  |
| rs295943 (C/T) | Intron, chr 5,NC_000005.10:g.59591047 | 0 | 0.1481 | 0.08 | 0.11 | 8 × 10−3 | NA |  |
| rs177077 (C/T) | Intron, chr 5,NC_000005.10:g.59643033 | 0.0909 | 0.2778 | 0.19 | 0.24 | 2 × 10−2 | 3.85 |  |
| rs16890078 (T/C) | Intron, chr 5,NC_000005.10:g.59661496 | 0.1364 | 0.0185 | 0.07 | 0.08 | 2.4 × 10−2 | 0.12 |  |
| rs2963821 (A/C) | Intron, chr 5,NC_000005.10:g.59813075 | 0.3182 | 0.5185 | 0.43 | 0.49 | 4.6 × 10−2 | 2.31 |  |
| d. CYP3A4 | rs73159483 (G/A) | Intergenic, chr 7, NC_000007.14:g.98806461 | 0 | 0.0926 | 0.05 | 0.03 | 3.8 × 10−2 | NA |
| rs1203844 (T/C) | Intergenic, chr 7, NC_000007.14:g.98850871 | 0.5000 | 0.2407 | 0.36 | 0.24 | 8 × 10−3 | 0.32 |  |
| rs2395022 * (C/A) | Intergenic, chr 7, NC_000007.14:g.99152756 | 0.1136 | 0 | 0.05 | 0.06 | 1.1 × 10−2 | 0 |  |
| CYP3A4 | rs35599367 (C/T) | Intron, chr 7,NC_000007.14:g.99768693 | 0.1136 | 0.0185 | 0.06 | 0.05 | 5 × 10−2 | 0.15 |
| u. CYP3A4 | rs472660 (G/A) | Intergenic, chr 7, NC_000007.14:g.99862484 | 0.1591 | 0.0370 | 0.09 | 0.12 | 3.7 × 10−2 | 0.20 |
| rs7786877 (A/G) | Intergenic, chr 7, NC_000007.14:g.100616392 | 0.2273 | 0.4259 | 0.34 | 0.26 | 3.8 × 10−2 | 2.52 |  |
| rs4729597 (T/C) | Intergenic, chr 7, NC_000007.14:g.100624226 | 0.2727 | 0.5000 | 0.40 | 0.37 | 2.2 × 10−2 | 2.67 |  |
| rs117406702 (G/A) | Intergenic, chr 7, NC_000007.14:g.100748471 | 0 | 0.0926 | 0.05 | 0.03 | 3.8 × 10−2 | NA |  |

Table 2 Caption: Statistically significant SNPs associated with the response to apremilast within or near PDE4 and CYP3A4 genes.

### 3.3. SNPs within ncRNAs That Target PDE4 mRNAs Are Associated with Apremilast Response

Differential response to apremilast treatment may be affected by various biological factors, including lncRNAs and miRNAs that target *PDE4* and *CYP3A4* genes influencing their stability, level of abundance or function. Therefore, all human lncRNAs/mRNA pairs were retrieved from LncRNA2Target v3 database, and SNPs located within the lncRNA genes were examined for association with response to apremilast. In total, 18 lncRNAs were correlated with PDE4A, B and C expression levels. Interestingly, two lncRNAs, i.e., *ANRIL* and *LINC00941*, were identified to contain SNPs that are associated with the response to apremilast ([Table 3](#genes-15-00369-t003)). The genomic coordinates of lncRNAs, their SNP content for which genotyping data are available in our analysis and the corresponding *PDE4* gene(s) are presented in [Table S1](#app1-genes-15-00369).

### Table 3.

| Locus | SNP | Location, HGVS Name | F_R | F_NR | MAF R + NR | MAFEUR | p Value | OR |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
| ANRIL | rs1063192 (A/G) | 3′ UTR, chr9,NM_004936.4:c.*2619 | 0.568 | 0.315 | 0.43 | 0.43 | 1.2 × 10−2 | 0.35 |
| ANRIL | rs10120688 (A/G) | Intron, chr9,NC_000009.12:g.22056500 | 0.636 | 0.407 | 0.51 | 0.50 | 2.4 × 10−2 | 0.39 |
| LINC00941 | rs12297445 (G/A) | Intron, chr12,NC_000012.12:g.30789344 | 0 | 0.093 | 0.05 | 0.09 | 3.8 × 10−2 | NA |
| miR4706 | rs2296316 (T/C) | Intron, chr14,NC_000014.9:g.65053528 | 0.591 | 0.333 | 0.45 | 0.46 | 1.1 × 10−2 | 0.35 |

Table 3 Caption: Statistically significant SNPs in ncRNAs associated with the response to apremilast.

Similarly to lncRNAs, SNPs which could potentially affect miRNA-mRNA pairing may result in modified transcription or translation of their target transcripts. Τhe miRTarBase lists 201 entries for miRNAs which contain 1276 SNPs ([Table S2](#app1-genes-15-00369)) and target *PDE4* or *CYP3A4* transcripts. However, only one of those SNPs is included in the genotyping platform used in our analysis. Thus, we tracked down all the SNPs that exhibit high linkage disequilibrium values (r^2^ > 0.7) with the miRNAs-related SNPs and examined for those having available genotypic data. In total, 3305 SNPs were identified with 15 of them been genotyped in our analysis. Out of these, rs2296316 (A/G) shows a positive association between the presence of the alternative allele (A) and the response to the apremilast therapy ([Table 3](#genes-15-00369-t003)). The aforementioned SNP is in complete LD (r2 = 1, D’ = 1) with rs2296319 in miR4706 which targets *PDE4C* mRNA.

### 3.4. Protein Network and Pathway Analysis

Τhe response to a drug treatment is generally the result of several functionally interconnected biological factors. Consequently, it is anticipated that a network-based approach may be a reliable source for the identification of predictive biomarkers in addition to those determined by the candidate gene approach, regarding the response to apremilast treatment. Accordingly, we reconstructed the interactome of PDE4A, B, C and D including their first protein neighbors ([Figure 1](#genes-15-00369-f001)). The network consists of 65 protein-nodes with 69 interactions. PDE4A, B, C and D are interconnected with six proteins. Namely, PDE4D, the most central node of the network having 39 first protein neighbors, is connected with both PDE4A and PDE4B, by three (AKAP6, LYN, PIAS4) and two (LIS1 and KAPCA) proteins, respectively, whereas PDE4A is connected with PDE4C by one protein (OSGI1). The association analysis of the response to apremilast treatment with the SNPs located within the genes encoding the protein nodes of the network detected 33 SNPs in 11 genes with statistically significant *p* values ([Table 4](#genes-15-00369-t004)). The respective proteins are mainly first interactors of the PDE4D protein ([Figure 1](#genes-15-00369-f001)A). Along the same lines, a simplified version of the cAMP-signaling KEGG pathway focusing on the signal transduction relevant to psoriasis was created ([Figure 1](#genes-15-00369-f001)B), and its analysis revealed SNPs within *CBP*, *ATF1* and *NFKBIA* genes to be associated with the response to apremilast.

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/33b4/10970167/90c2390dbfc6/genes-15-00369-g001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=10970167_genes-15-00369-g001.jpg)

(A) The protein interactome of PDE4A, B, C and D with their interconnecting protein neighbors. PDE4A, B, C and D (yellow nodes) are interconnected with six proteins (AKAP6, PIAS4, LYN, KAPCA, LIS1, OSGI1). PDE4D is the most central node, whereas PDE4B is the most peripheral PDE4 protein of the network. SNPs located within eleven PDE4-first neighbor protein (light blue nodes) coding genes showed statistically significant association with the response to apremilast treatment (Table 4). Forty-six additional protein nodes (white nodes) constitute the entire PDE4 protein network. (B) Part of the cAMP signaling pathway showing apremilast’s mechanism of action. SNPs within IκΒα, CBP and ATF-1 (light blue colored) coding genes are associated with apremilast treatment response (Table 4).

### Table 4.

| Protein (Gene) Name | SNP | Location, HGVS Name | F_R | F_NR | MAF R + NR | MAFEUR | p Value | OR |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
| AKAP6 (AKAP6) | rs2031106 (G/A) | Intron, chr 14,NC_000014.9:g.32392503 | 0.182 | 0.056 | 0.11 | 0.12 | 4.9 × 10−2 | 0.26 |
|   | rs7142538 (A/G) | Intron, chr 14,NC_000014.9:g.32404235 | 0.114 | 0 | 0.05 | 0.08 | 1.1 × 10−2 | 0 |
|   | rs3742926 (C/T) | Missense, chr 14,NM_004274.5:c.1010 | 0.068 | 0.222 | 0.15 | 0.10 | 3.5 × 10−2 | 3.91 |
|   | rs7150894 (A/G) | Synonymous, chr 14,NM_004274.5:c.1155 | 0.523 | 0.296 | 0.40 | 0.34 | 2.3 × 10−2 | 0.38 |
|   | rs1956204 (C/T) | Intron, chr 14,NC_000014.9:g.32549256 | 0.136 | 0.315 | 0.23 | 0.20 | 3.8 × 10−2 | 2.91 |
|   | rs11624518 (G/A) | Intron, chr 14,NC_000014.9:g.32711145 | 0.205 | 0.389 | 0.31 | 0.29 | 4.9 × 10−2 | 2.48 |
|   | rs11622943 (A/G) | Intron, chr 14,NC_000014.9:g.32776891 | 0.159 | 0.352 | 0.27 | 0.39 | 3.2 × 10−2 | 2.87 |
|   | rs34628134 (A/G) | Intron, chr 14,NC_000014.9:g.32778457 | 0.455 | 0.241 | 0.34 | 0.23 | 2.6 × 10−2 | 0.38 |
| A4 (APP) | rs380417 (C/T) | Intron, chr 21,NC_000021.9:g.25899847 | 0.023 | 0.185 | 0.11 | 0.22 | 1.1 × 10−2 | 9.77 |
|   | rs2829981 (A/G) | Intron, chr 21,NC_000021.9:g.25908384 | 0 | 0.093 | 0.05 | 0.09 | 3.8 × 10−2 | NA |
|   | rs1783024 (C/T) | Intron, chr 21,NC_000021.9:g.25966822 | 0.614 | 0.407 | 0.50 | 0.42 | 4.2 × 10−2 | 0.43 |
|   | rs128648 (C/T) | Intron, chr 21,NC_000021.9:g.25970679 | 0.136 | 0.315 | 0.23 | 0.28 | 3.8 × 10−2 | 2.91 |
|   | rs58908134 (G/T) | Intron, chr 21,NC_000021.9:g.25989938 | 0.068 | 0.222 | 0.15 | 0.18 | 3.5 × 10−2 | 3.91 |
| SHAN2 (SHANK2) | rs10899236 (A/G) | Intron, chr 11,NC_000011.10:g.70606029 | 0.614 | 0.407 | 0.50 | 0.45 | 4.2 × 10−2 | 0.43 |
|   | rs12274337 (C/T) | Intron, chr 11,NC_000011.10:g.70643910 | 0 | 0.093 | 0.05 | 0.03 | 3.8 × 10−2 | NA |
|   | rs35961474 (C/T) | Intron, chr 11, NC_000011.10:g.70901955 | 0.136 | 0 | 0.06 | 0.05 | 5.1 × 10−3 | 0 |
| ITSN1 (ITSN1) | rs2251854 (A/G) | Intron, chr 21, NC_000021.9:g.33683942 | 0.205 | 0.426 | 0.33 | 0.32 | 2 × 10−2 | 2.89 |
|   | rs2256797 (T/C) | Intron, chr 21,NC_000021.9:g.33785207 | 0.114 | 0.333 | 0.23 | 0.23 | 1.1 × 10−2 | 3.90 |
|   | rs2834269 (T/C) | Intron, chr 21, NC_000021.9:g.33828070 | 0.455 | 0.241 | 0.34 | 0.25 | 2.6 × 10−2 | 0.38 |
|   | rs2834287 (A/G) | 3′ UTR, chr 21,NM_003024.3:c.*357 | 0.091 | 0.315 | 0.21 | 0.17 | 7.2 × 10−3 | 4.60 |
| P85A (PIK3R1) | rs831122 (G/A) | Intron, chr 5, NC_000005.10:g.68267102 | 0.023 | 0.185 | 0.11 | 0.15 | 1.1 × 10−2 | 9.77 |
| AKA12 (AKAP12) | rs10499266 (G/A) | Intron, chr 6,NC_000006.12:g.151251388 | 0.136 | 0.019 | 0.07 | 0.08 | 2.4 × 10−2 | 0.12 |
|   | rs73613423 (T/G) | Intron, chr 6,NC_000006.12:g.151263516 | 0.114 | 0 | 0.05 | 0.05 | 1.1 × 10−2 | 0 |
|   | rs9397389 (C/T) | Intron, chr 6,NC_000006.12:g.151308730 | 0.205 | 0.389 | 0.31 | 0.26 | 4.9 × 10−2 | 2.48 |
| MEOX2 (MEOX2) | rs10270030 (T/C) | Intron, chr 7, NC_000007.14:g.15648375 | 0.318 | 0.148 | 0.22 | 0.35 | 4.5 × 10−2 | 0.37 |
| ABL1 (ABL1) | rs75764711 (G/A) | Intron, chr 9,NC_000009.12:g.130794660 | 0 | 0.111 | 0.06 | 0.02 | 2.2 × 10−2 | NA |
| ARRB1 (ARRB1) | rs512797 (G/A) | Intron, chr 11,NC_000011.10:g.75285037 | 0 | 0.167 | 0.09 | 0.16 | 4.5 × 10−3 | NA |
|   | rs616714 (G/T) | Intron, chr 11,NC_000011.10:g.75333596G | 0.068 | 0.259 | 0.17 | 0.18 | 1.3 × 10−2 | 4.78 |
| NKG2D (KLRK1) | rs2617151 (G/A) | Intron, chr 12,NC_000012.12:g.10379835 | 0.136 | 0.315 | 0.23 | 0.18 | 3.8 × 10−2 | 2.91 |
| ZBTB1 (ZBTB1) | rs74056445 (A/G) | Intron, chr 14, NC_000014.9:g.64511474 | 0.023 | 0.148 | 0.09 | 0.09 | 3.2 × 10−2 | 7.48 |
| CREBBP (CBP) | rs2239316 * (A/G) | Intron, chr16,NC_000016.10:g.3863994 | 0.091 | 0.259 | 0.18 | 0.25 | 3.2 × 10−2 | 3.5 |
| ATF1 (ATF1) | rs1129406 (C/T) | Synonymous, chr 12,NM_005171.5:c.327 | 0.409 | 0.222 | 0.31 | 0.39 | 4.6 × 10−2 | 0.41 |
| IκBα (NFKBIA) | rs696 *(G/A) | 3′ UTR, chr 14,NM_020529.3:c.*126 | 0.568 | 0.352 | 0.45 | 0.39 | 3.2 × 10−2 | 0.41 |

Table 4 Caption: Statistically significant SNPs identified through the protein network and pathway analysis associated with the response to apremilast.

### 3.5. eQTL Analysis

The majority of the SNPs associated with the response to apremilast ([Table 2](#genes-15-00369-t002), [Table 3](#genes-15-00369-t003) and [Table 4](#genes-15-00369-t004)) are mapped either in intronic regions or in the chromosomal vicinity of the respective genes. Thus, they may contribute to the regulation of the corresponding gene expression or may be in high LD with other regulatory variants not included in the genotyping platform used in our analysis. In order to identify putative causal variants, we examined whether these SNPs constitute significant cis-eQTLs in tissues or cells relevant to the examined phenotype. In total, 56 eQTLs were determined corresponding to 55,4% of the total SNPs identified with the complementary approaches described previously ([Figure 2](#genes-15-00369-f002)). [Table S3](#app1-genes-15-00369) lists the eQTLs whose alternative alleles display an effect size ranging from 0.04 to 1.95 and the tissues in which the correlation with the expression of the respective gene was identified. The majority of the eQTLs were identified in immune cells, whereas twenty-two were also detected in the skin. Two SNPs located within and upstream of the CYP3A4 gene constitute eQTLs in the liver and the small intestine, respectively.

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/33b4/10970167/29afe05edb98/genes-15-00369-g002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=10970167_genes-15-00369-g002.jpg)

A Venn diagram of the SNPs associated with the apremilast response.

## 4. Discussion

In this pharmacogenetic study, we have implemented a candidate-gene approach at three different levels, i.e., (i) the apremilast direct target and its metabolizing enzyme genes, (ii) the ncRNAs that are associated or influence their expression and (iii) the PDE4 protein network and pathway analysis gene products. The genotypic data of the 49 patients propose an association with the response to apremilast of (i) sixty-four SNPs within or near *PDE4* and *CYP3A4* genes, (ii) three SNPs within the *ANRIL* and *LINC00941* lncRNAs and one SNP inside *miR4706* and (iii) thirty-three SNPs within genes encoding protein components of the PDE4 interactome or cAMP signaling pathway relevant to psoriasis.

In order to evaluate the identified associations (listed in [Table 2](#genes-15-00369-t002), [Table 3](#genes-15-00369-t003) and [Table 4](#genes-15-00369-t004)) and their potential clinical application, we explored the literature for previous associations with autoimmune diseases or diseases with an autoimmune component. Indeed, previous studies have shown that SNPs associated with the response to Ps treatment are also associated with Ps susceptibility [[27](#B27-genes-15-00369),[28](#B28-genes-15-00369)]. Specifically, rs892085 and rs322151, located downstream of *PDE4A*, have been associated with Ps and PsA [[29](#B29-genes-15-00369),[30](#B30-genes-15-00369),[31](#B31-genes-15-00369)] and systemic sclerosis, respectively [[32](#B32-genes-15-00369)]. In addition, rs2395022, mapped 0.6 Mb downstream of *CYP3A4* constitutes a susceptibility locus for inflammatory bowel disease [[33](#B33-genes-15-00369)]. Furthermore, rs2305795 and rs11085752, upstream and downstream of *PDE4A*, respectively, have been shown to contribute to the genetic risk of neurological diseases, i.e., narcolepsy and Alzheimer’s [[34](#B34-genes-15-00369),[35](#B35-genes-15-00369)]. The rs2239316 of *CBP* gene is associated with *IL32* CpG methylation in CD4+ and CD8+ T cells, which in turn is associated with juvenile idiopathic arthritis [[36](#B36-genes-15-00369)]. Finally, rs696, located in the 3′ UTR of *NFKBIA* gene, was associated with Guillain–Barré syndrome [[37](#B37-genes-15-00369)], Behcet disease [[38](#B38-genes-15-00369)] and anti-TNF response in patients with Crohn’s disease and ulcerative colitis [[39](#B39-genes-15-00369)].

The analysis of lncRNAs showed that SNPs within *ANRIL* and *LINC00941*, which are correlated with their expression in immune cells and skin (eQTLs listed in [Table S3](#app1-genes-15-00369)), are also associated with the response to apremilast ([Table 3](#genes-15-00369-t003)). The association with *ANRIL* lncRNA is of particular importance since *ANRIL* has been previously associated with autoimmune disease susceptibility, including Ps [[40](#B40-genes-15-00369),[41](#B41-genes-15-00369),[42](#B42-genes-15-00369)]. Previous studies in T-Rex 293 HEK cell lines or epidermal tissue cultures have demonstrated that following *ANRIL* or *LINC00941* knockdown, *PDE4B* expression was down or up regulated, respectively [[22](#B22-genes-15-00369),[43](#B43-genes-15-00369)]. Although this correlation could be indirect and may be detected due to the sequestration of specific miRNAs, it provides evidence for the putative cause of the observed genetic association. This, in combination with the findings of the involvement of *LINC00941* in human epidermal homeostasis [[44](#B44-genes-15-00369)] as well as the regulation of TGF-β/Smad signaling by ANRIL [[45](#B45-genes-15-00369)], also suggests the molecular mechanisms that are linked with the response.

The applied protein network analysis for the detection of putative biomarkers identified SNPs in 11 genes associated with apremilast treatment response ([Figure 1](#genes-15-00369-f001)A), four of which have established roles related to PDE4s function. Specifically, AKAP6, a common first neighbor of PDE4A and PDE4D, belongs to the family of A-kinase anchoring proteins (AKAPs) and, among other functions, is involved in cAMP-mediated signaling, forming a complex which includes protein kinase A (PKA), Rap guanine nucleotide exchange factor 3 (PRGF3/EPAC1) and different phosphodiesterases including PDE4D3 [[46](#B46-genes-15-00369),[47](#B47-genes-15-00369)]. AKA12, also an AKAP family member, functions as an anchoring protein mediating the subcellular compartmentation of PKA [[48](#B48-genes-15-00369)]. ARRB1 is involved in the degradation of cAMP by recruiting cAMP phosphodiesterases to ligand-activated receptors [[49](#B49-genes-15-00369)] and ZBTB1 is a transcription factor that represses cAMP-responsive element (CRE)-mediated transcriptional activation [[50](#B50-genes-15-00369)]. KEGG pathway analysis identified three genes ([Figure 1](#genes-15-00369-f001)B) with pivotal roles in the cAMP signaling pathway, i.e., *CBP*, *ATF1* and *NFKBIA*. The CBP protein, through specific binding to phosphorylated CREB, enhances its transcriptional activity towards cAMP-responsive genes [[51](#B51-genes-15-00369)]. ATF1 is activated by PKA phosphorylation and stimulates CRE-containing gene expression [[52](#B52-genes-15-00369)]. NFKBIA inhibits the activity of NF-kappa-B, modulating immune and pro-inflammatory responses [[53](#B53-genes-15-00369)]. The aforementioned functional roles provide meaningful information for the detected genetic associations. On the other hand, the rest of the associated genes ([Figure 1](#genes-15-00369-f001)A, [Table 4](#genes-15-00369-t004)) have reported functional roles not related to PDE4s, so far. Therefore, their association with apremilast response may contribute to the identification of novel molecules participating in apremilast’s therapeutic action.

Generally, the colocalization of association signals with cis-eQTLs strengthens the validity of the association and implies that the identified variants could be causal. In our study, more than half of the SNPs associated with the apremilast response constitute eQTLs for the respective genes in relevant-to-Ps tissues and cells. The majority of the eQTLs were identified in immune cells, thus they may modulate the inflammatory signaling of PDE4s interfering with apremilast response. It is worth mentioning the case of the intronic SNP rs35599367 in *CYP3A4* which has been functionally characterized clarifying its identification as an eQTL in the liver ([Table S3](#app1-genes-15-00369)). Its minor T allele has been demonstrated to alter the *CYP3A4* splicing pattern in HepG2 cells producing a nonfunctional variant that yields reduced enzyme expression [[54](#B54-genes-15-00369)], thus increasing drug bioavailability. In such cases, dose adjustment may be required to reduce side effects, as it has been suggested for rs35599367 and statin usage [[55](#B55-genes-15-00369)]. Notably, in our cohort, the T allele of this SNP is associated with a positive response to apremilast since its frequency is 11.4% in the responder group versus 1.85% in the non-responders.

Besides the identification of genetic markers, our study highlights that a higher absolute baseline PASI was associated with lower treatment response. This is in accordance with a previous pooled analysis of Phase 3 and Phase 4 clinical trials which concluded that apremilast may be particularly beneficial in more moderate disease, early in the treatment [[56](#B56-genes-15-00369)]. In contrast, the efficacy of apremilast treatment was not correlated with sex as has been previously suggested for systemic drugs and biological agents in Ps [[57](#B57-genes-15-00369),[58](#B58-genes-15-00369)]. This difference in sex was partly attributed to the lower weight of women vs. men in combination with the weigh-independent standard dosing of apremilast. This men/women weight difference was also detected in our study; although, we found no difference in the weight of women in the responder and non-responder group ([Table 1](#genes-15-00369-t001)). Thus, the correlation of treatment response with body weight could be further investigated.

This study comprises the second pharmacogenetic study of apremilast in Ps, though in a larger number of patients compared to the first one [[19](#B19-genes-15-00369)], and the first association study of apremilast’s target genes. The results of our analysis should be discussed within the context of the number of samples tested; however, they offer an appreciable start towards the precision medicine goal aiming Ps. Our results have to be validated in larger cohorts of various populations. This could offer stronger associations throughout the genome to be utilized for estimating the polygenic risk score of Ps patients. Nevertheless, this integrated candidate-gene approach revealed promising biomarkers that could predict apremilast response and complies with the initiative for further investigation of the field.

## Acknowledgments

We are grateful to all the patients that participated in the study. This work was supported by the postgraduate programs “Biotechnology-Quality Assessment in Nutrition and the Environment”, “Molecular Biology and Genetics Applications -Diagnostic Markers” and “Bio-entrepreneurship” of the Department of Biochemistry and Biotechnology, University of Thessaly, Greece.

## Supplementary Materials

The following supporting information can be downloaded at: [https://www.mdpi.com/article/10.3390/genes15030369/s1](https://www.mdpi.com/article/10.3390/genes15030369/s1), Table S1: LncRNAs that target PDE4 transcripts; Table S2: miRNAs that target PDE4 and CYP3A4 transcripts; Table S3: eQTLs associated with the response to apremilast.

## Author Contributions

Conceptualization, K.L., E.Z., Z.M., D.P.B. and T.S.; Clinical investigation, E.Z., K.C., P.G., A.-V.R.-S., S.G.T., A.D. and D.P.B.; Investigation, S.A., K.L. and T.S.; Data analysis, K.L., T.G., N.K.M. and T.S.; Writing—original draft preparation, K.L. and T.S., Writing—Review and Editing, K.L., E.Z., Z.M., D.P.B., N.K.M. and T.S.; Supervision, K.L. and T.S.; Funding Acquisition, K.L. and T.S. All authors have read and agreed to the published version of the manuscript.

## Institutional Review Board Statement

The study was conducted in accordance with the Declaration of Helsinki and approved by the Research Ethics Committee of the General University Hospital of Larissa, Greece (11/2nd/08-02-22).

## Informed Consent Statement

Informed consent was obtained from all patients involved in the study.

## Data Availability Statement

The original contributions presented in the study are included in the article supplementary material, further inquiries can be directed to the corresponding author.

## Conflicts of Interest

The authors declare no conflicts of interest.

## Funding Statement

This research received no external funding.

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

The original contributions presented in the study are included in the article supplementary material, further inquiries can be directed to the corresponding author.

### Supplementary Materials

### Data Availability Statement

The original contributions presented in the study are included in the article supplementary material, further inquiries can be directed to the corresponding author.

## References

1. Boehncke W.-H., Schön M.P. Psoriasis. Lancet. 2015;386:983–994. doi: 10.1016/S0140-6736(14)61909-7.  [DOI](https://doi.org/10.1016/S0140-6736(14)61909-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26025581/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Psoriasis&author=W.-H.%20Boehncke&author=M.P.%20Sch%C3%B6n&volume=386&publication_year=2015&pages=983-994&pmid=26025581&doi=10.1016/S0140-6736(14)61909-7&)

2. Lee H.-J., Kim M. Challenges and Future Trends in the Treatment of Psoriasis. Int. J. Mol. Sci. 2023;24:13313. doi: 10.3390/ijms241713313.  [DOI](https://doi.org/10.3390/ijms241713313) | [PMC free article](/articles/PMC10487560/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/37686119/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Mol.%20Sci.&title=Challenges%20and%20Future%20Trends%20in%20the%20Treatment%20of%20Psoriasis&author=H.-J.%20Lee&author=M.%20Kim&volume=24&publication_year=2023&pages=13313&pmid=37686119&doi=10.3390/ijms241713313&)

3. Wang C.-Y., Wang C.-W., Chen C.-B., Chen W.-T., Chang Y.-C., Hui R.C.-Y., Chung W.-H. Pharmacogenomics on the Treatment Response in Patients with Psoriasis: An Updated Review. Int. J. Mol. Sci. 2023;24:7329. doi: 10.3390/ijms24087329.  [DOI](https://doi.org/10.3390/ijms24087329) | [PMC free article](/articles/PMC10138383/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/37108492/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Mol.%20Sci.&title=Pharmacogenomics%20on%20the%20Treatment%20Response%20in%20Patients%20with%20Psoriasis:%20An%20Updated%20Review&author=C.-Y.%20Wang&author=C.-W.%20Wang&author=C.-B.%20Chen&author=W.-T.%20Chen&author=Y.-C.%20Chang&volume=24&publication_year=2023&pages=7329&pmid=37108492&doi=10.3390/ijms24087329&)

4. Ovejero-Benito M.C., Muñoz -Aceituno E., Reolid A., Saiz- Rodríguez M., Abad-Santos F., Daudén E. Pharmacogenetics and Pharmacogenomics in Moderate-to-Severe Psoriasis. Am. J. Clin. Dermatol. 2018;19:209–222. doi: 10.1007/s40257-017-0322-9.  [DOI](https://doi.org/10.1007/s40257-017-0322-9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28921458/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Clin.%20Dermatol.&title=Pharmacogenetics%20and%20Pharmacogenomics%20in%20Moderate-to-Severe%20Psoriasis&author=M.C.%20Ovejero-Benito&author=E.%20Mu%C3%B1oz%20-Aceituno&author=A.%20Reolid&author=M.%20Saiz-%20Rodr%C3%ADguez&author=F.%20Abad-Santos&volume=19&publication_year=2018&pages=209-222&pmid=28921458&doi=10.1007/s40257-017-0322-9&)

5. Owczarczyk-Saczonek A., Owczarek W., Osmola-Mańkowska A., Adamski Z., Placek W., Rakowska A. Secondary failure of TNF-α inhibitors in clinical practice. Dermatol. Ther. 2019;32:e12760. doi: 10.1111/dth.12760.  [DOI](https://doi.org/10.1111/dth.12760) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30288872/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Dermatol.%20Ther.&title=Secondary%20failure%20of%20TNF-%CE%B1%20inhibitors%20in%20clinical%20practice&author=A.%20Owczarczyk-Saczonek&author=W.%20Owczarek&author=A.%20Osmola-Ma%C5%84kowska&author=Z.%20Adamski&author=W.%20Placek&volume=32&publication_year=2019&pages=e12760&pmid=30288872&doi=10.1111/dth.12760&)

6. Elberdín L., Fernández-Torres R.M., Paradela S., Mateos M., Blanco E., Balboa-Barreiro V., Gómez-Besteiro M.I., Outeda M., Martín-Herranz I., Fonseca E. Biologic Therapy for Moderate to Severe Psoriasis. Real-World Follow-up of Patients Who Initiated Biologic Therapy at Least 10 Years Ago. Dermatol. Ther. 2022;12:761–770. doi: 10.1007/s13555-022-00693-2.  [DOI](https://doi.org/10.1007/s13555-022-00693-2) | [PMC free article](/articles/PMC8941069/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35226302/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Dermatol.%20Ther.&title=Biologic%20Therapy%20for%20Moderate%20to%20Severe%20Psoriasis.%20Real-World%20Follow-up%20of%20Patients%20Who%20Initiated%20Biologic%20Therapy%20at%20Least%2010%20Years%20Ago&author=L.%20Elberd%C3%ADn&author=R.M.%20Fern%C3%A1ndez-Torres&author=S.%20Paradela&author=M.%20Mateos&author=E.%20Blanco&volume=12&publication_year=2022&pages=761-770&pmid=35226302&doi=10.1007/s13555-022-00693-2&)

7. Nassim D., Alajmi A., Jfri A., Pehr K. Apremilast in dermatology: A review of literature. Dermatol. Ther. 2020;33:e14261. doi: 10.1111/dth.14261.  [DOI](https://doi.org/10.1111/dth.14261) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32876993/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Dermatol.%20Ther.&title=Apremilast%20in%20dermatology:%20A%20review%20of%20literature&author=D.%20Nassim&author=A.%20Alajmi&author=A.%20Jfri&author=K.%20Pehr&volume=33&publication_year=2020&pages=e14261&pmid=32876993&doi=10.1111/dth.14261&)

8. Chiricozzi A., Caposiena D., Garofalo V., Cannizzaro M.V., Chimenti S., Saraceno R. A new therapeutic for the treatment of moderate-to-severe plaque psoriasis: Apremilast. Expert Rev. Clin. Immunol. 2016;12:237–249. doi: 10.1586/1744666X.2016.1134319.  [DOI](https://doi.org/10.1586/1744666X.2016.1134319) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26692125/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Expert%20Rev.%20Clin.%20Immunol.&title=A%20new%20therapeutic%20for%20the%20treatment%20of%20moderate-to-severe%20plaque%20psoriasis:%20Apremilast&author=A.%20Chiricozzi&author=D.%20Caposiena&author=V.%20Garofalo&author=M.V.%20Cannizzaro&author=S.%20Chimenti&volume=12&publication_year=2016&pages=237-249&pmid=26692125&doi=10.1586/1744666X.2016.1134319&)

9. Johnson K.R., Nicodemus-Johnson J., Danziger R.S. An evolutionary analysis of cAMP-specific Phosphodiesterase 4 alternative splicing. BMC Evol. Biol. 2010;10:247. doi: 10.1186/1471-2148-10-247.  [DOI](https://doi.org/10.1186/1471-2148-10-247) | [PMC free article](/articles/PMC2929239/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20701803/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Evol.%20Biol.&title=An%20evolutionary%20analysis%20of%20cAMP-specific%20Phosphodiesterase%204%20alternative%20splicing&author=K.R.%20Johnson&author=J.%20Nicodemus-Johnson&author=R.S.%20Danziger&volume=10&publication_year=2010&pages=247&pmid=20701803&doi=10.1186/1471-2148-10-247&)

10. Schafer P.H., Parton A., Capone L., Cedzik D., Brady H., Evans J.F., Man H.-W., Muller G.W., Stirling D.I., Chopra R. Apremilast is a selective PDE4 inhibitor with regulatory effects on innate immunity. Cell Signal. 2014;26:2016–2029. doi: 10.1016/j.cellsig.2014.05.014.  [DOI](https://doi.org/10.1016/j.cellsig.2014.05.014) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24882690/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Signal.&title=Apremilast%20is%20a%20selective%20PDE4%20inhibitor%20with%20regulatory%20effects%20on%20innate%20immunity&author=P.H.%20Schafer&author=A.%20Parton&author=L.%20Capone&author=D.%20Cedzik&author=H.%20Brady&volume=26&publication_year=2014&pages=2016-2029&pmid=24882690&doi=10.1016/j.cellsig.2014.05.014&)

11. Keating G.M. Apremilast: A Review in Psoriasis and Psoriatic Arthritis. Drugs. 2017;77:459–472. doi: 10.1007/s40265-017-0709-1.  [DOI](https://doi.org/10.1007/s40265-017-0709-1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28213862/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drugs&title=Apremilast:%20A%20Review%20in%20Psoriasis%20and%20Psoriatic%20Arthritis&author=G.M.%20Keating&volume=77&publication_year=2017&pages=459-472&pmid=28213862&doi=10.1007/s40265-017-0709-1&)

12. Zanger U.M., Schwab M. Cytochrome P450 enzymes in drug metabolism: Regulation of gene expression, enzyme activities, and impact of genetic variation. Pharmacol. Ther. 2013;138:103–141. doi: 10.1016/j.pharmthera.2012.12.007.  [DOI](https://doi.org/10.1016/j.pharmthera.2012.12.007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23333322/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacol.%20Ther.&title=Cytochrome%20P450%20enzymes%20in%20drug%20metabolism:%20Regulation%20of%20gene%20expression,%20enzyme%20activities,%20and%20impact%20of%20genetic%20variation&author=U.M.%20Zanger&author=M.%20Schwab&volume=138&publication_year=2013&pages=103-141&pmid=23333322&doi=10.1016/j.pharmthera.2012.12.007&)

13. Papp K., Reich K., Leonardi C.L., Kircik L., Chimenti S., Langley R.G., Hu C., Stevens R.M., Day R.M., Gordon K.B., et al. Apremilast, an oral phosphodiesterase 4 (PDE4) inhibitor, in patients with moderate to severe plaque psoriasis: Results of a phase III, randomized, controlled trial (Efficacy and Safety Trial Evaluating the Effects of Apremilast in Psoriasis [ESTEEM] 1) J. Am. Acad. Dermatol. 2015;73:37–49. doi: 10.1016/j.jaad.2015.03.049.  [DOI](https://doi.org/10.1016/j.jaad.2015.03.049) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26089047/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Am.%20Acad.%20Dermatol.&title=Apremilast,%20an%20oral%20phosphodiesterase%204%20(PDE4)%20inhibitor,%20in%20patients%20with%20moderate%20to%20severe%20plaque%20psoriasis:%20Results%20of%20a%20phase%20III,%20randomized,%20controlled%20trial%20(Efficacy%20and%20Safety%20Trial%20Evaluating%20the%20Effects%20of%20Apremilast%20in%20Psoriasis%20%5BESTEEM%5D%201)&author=K.%20Papp&author=K.%20Reich&author=C.L.%20Leonardi&author=L.%20Kircik&author=S.%20Chimenti&volume=73&publication_year=2015&pages=37-49&pmid=26089047&doi=10.1016/j.jaad.2015.03.049&)

14. Paul C., Cather J., Gooderham M., Poulin Y., Mrowietz U., Ferrandiz C., Crowley J., Hu C., Stevens R.M., Shah K., et al. Efficacy and safety of apremilast, an oral phosphodiesterase 4 inhibitor, in patients with moderate-to-severe plaque psoriasis over 52 weeks: A phase III, randomized controlled trial (ESTEEM 2) Br. J. Dermatol. 2015;173:1387–1399. doi: 10.1111/bjd.14164.  [DOI](https://doi.org/10.1111/bjd.14164) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26357944/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Dermatol.&title=Efficacy%20and%20safety%20of%20apremilast,%20an%20oral%20phosphodiesterase%204%20inhibitor,%20in%20patients%20with%20moderate-to-severe%20plaque%20psoriasis%20over%2052%20weeks:%20A%20phase%20III,%20randomized%20controlled%20trial%20(ESTEEM%202)&author=C.%20Paul&author=J.%20Cather&author=M.%20Gooderham&author=Y.%20Poulin&author=U.%20Mrowietz&volume=173&publication_year=2015&pages=1387-1399&pmid=26357944&doi=10.1111/bjd.14164&)

15. Reich K., Gooderham M., Bewley A., Green L., Soung J., Petric R., Marcsisin J., Cirulli J., Chen R., Piguet V. Safety and efficacy of apremilast through 104 weeks in patients with moderate to severe psoriasis who continued on apremilast or switched from etanercept treatment: Findings from the LIBERATE study. J. Eur. Acad. Dermatol. Venereol. 2018;32:397–402. doi: 10.1111/jdv.14738.  [DOI](https://doi.org/10.1111/jdv.14738) | [PMC free article](/articles/PMC5873268/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29220542/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Eur.%20Acad.%20Dermatol.%20Venereol.&title=Safety%20and%20efficacy%20of%20apremilast%20through%20104%20weeks%20in%20patients%20with%20moderate%20to%20severe%20psoriasis%20who%20continued%20on%20apremilast%20or%20switched%20from%20etanercept%20treatment:%20Findings%20from%20the%20LIBERATE%20study&author=K.%20Reich&author=M.%20Gooderham&author=A.%20Bewley&author=L.%20Green&author=J.%20Soung&volume=32&publication_year=2018&pages=397-402&pmid=29220542&doi=10.1111/jdv.14738&)

16. Wong T.H., Sinclair S., Smith B., Fraser C., Morton C.A. Real-world, single-centre experience of apremilast for the treatment of moderate to severe psoriasis. Clin. Exp. Dermatol. 2017;42:674–685. doi: 10.1111/ced.13150.  [DOI](https://doi.org/10.1111/ced.13150) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28621002/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Exp.%20Dermatol.&title=Real-world,%20single-centre%20experience%20of%20apremilast%20for%20the%20treatment%20of%20moderate%20to%20severe%20psoriasis&author=T.H.%20Wong&author=S.%20Sinclair&author=B.%20Smith&author=C.%20Fraser&author=C.A.%20Morton&volume=42&publication_year=2017&pages=674-685&pmid=28621002&doi=10.1111/ced.13150&)

17. Papadavid E., Rompoti N., Theodoropoulos K., Kokkalis G., Rigopoulos D. Real-world data on the effectiveness and safety of apremilast in patients with moderate-to-severe plaque psoriasis. J. Eur. Acad. Dermatol. Venereol. 2018;32:1173–1179. doi: 10.1111/jdv.14832.  [DOI](https://doi.org/10.1111/jdv.14832) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29388335/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Eur.%20Acad.%20Dermatol.%20Venereol.&title=Real-world%20data%20on%20the%20effectiveness%20and%20safety%20of%20apremilast%20in%20patients%20with%20moderate-to-severe%20plaque%20psoriasis&author=E.%20Papadavid&author=N.%20Rompoti&author=K.%20Theodoropoulos&author=G.%20Kokkalis&author=D.%20Rigopoulos&volume=32&publication_year=2018&pages=1173-1179&pmid=29388335&doi=10.1111/jdv.14832&)

18. Ighani A., Georgakopoulos J.R., Zhou L.L., Walsh S., Shear N., Yeung J. Efficacy and safety of apremilast monotherapy for moderate to severe psoriasis: Retrospective study. J. Cutan. Med. Surg. 2018;22:290–296. doi: 10.1177/1203475418755982.  [DOI](https://doi.org/10.1177/1203475418755982) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29373924/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Cutan.%20Med.%20Surg.&title=Efficacy%20and%20safety%20of%20apremilast%20monotherapy%20for%20moderate%20to%20severe%20psoriasis:%20Retrospective%20study&author=A.%20Ighani&author=J.R.%20Georgakopoulos&author=L.L.%20Zhou&author=S.%20Walsh&author=N.%20Shear&volume=22&publication_year=2018&pages=290-296&pmid=29373924&doi=10.1177/1203475418755982&)

19. Verbenko D.A., Karamova A.E., Artamonova O.G., Deryabin D.G., Rakitko A., Chernitsov A., Krasnenko A., Elmuratov A., Solomka V.S., Kubanov A.A. Apremilast Pharmacogenomics in Russian Patients with Moderate-to-Severe and Severe Psoriasis. J. Pers. Med. 2020;11:20. doi: 10.3390/jpm11010020.  [DOI](https://doi.org/10.3390/jpm11010020) | [PMC free article](/articles/PMC7823747/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33383665/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Pers.%20Med.&title=Apremilast%20Pharmacogenomics%20in%20Russian%20Patients%20with%20Moderate-to-Severe%20and%20Severe%20Psoriasis&author=D.A.%20Verbenko&author=A.E.%20Karamova&author=O.G.%20Artamonova&author=D.G.%20Deryabin&author=A.%20Rakitko&volume=11&publication_year=2020&pages=20&pmid=33383665&doi=10.3390/jpm11010020&)

20. Martin F.J., Amode M.R., Aneja A., Austine-Orimoloye O., Azov A.G., Barnes I., Becker A., Bennett R., Berry A., Bhai J., et al. Ensembl 2023. Nucleic Acids Res. 2023;51:D933–D941. doi: 10.1093/nar/gkac958.  [DOI](https://doi.org/10.1093/nar/gkac958) | [PMC free article](/articles/PMC9825606/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36318249/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&title=Ensembl%202023&author=F.J.%20Martin&author=M.R.%20Amode&author=A.%20Aneja&author=O.%20Austine-Orimoloye&author=A.G.%20Azov&volume=51&publication_year=2023&pages=D933-D941&pmid=36318249&doi=10.1093/nar/gkac958&)

21. Purcell S., Neale B., Todd-Brown K., Thomas L., Ferreira M.A., Bender D., Maller J., Sklar P., de Bakker P.I., Daly M.J., et al. PLINK: A tool set for whole-genome association and population-based linkage analyses. Am. J. Hum. Genet. 2007;81:559–575. doi: 10.1086/519795.  [DOI](https://doi.org/10.1086/519795) | [PMC free article](/articles/PMC1950838/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17701901/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Hum.%20Genet.&title=PLINK:%20A%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses&author=S.%20Purcell&author=B.%20Neale&author=K.%20Todd-Brown&author=L.%20Thomas&author=M.A.%20Ferreira&volume=81&publication_year=2007&pages=559-575&pmid=17701901&doi=10.1086/519795&)

22. Cheng L., Wang P., Tian R., Wang S., Guo Q., Luo M., Zhou W., Liu G., Jiang H., Jiang Q. LncRNA2Target v2.0: A comprehensive database for target genes of lncRNAs in human and mouse. Nucleic Acids Res. 2019;47:D140–D144. doi: 10.1093/nar/gky1051.  [DOI](https://doi.org/10.1093/nar/gky1051) | [PMC free article](/articles/PMC6323902/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30380072/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&title=LncRNA2Target%20v2.0:%20A%20comprehensive%20database%20for%20target%20genes%20of%20lncRNAs%20in%20human%20and%20mouse&author=L.%20Cheng&author=P.%20Wang&author=R.%20Tian&author=S.%20Wang&author=Q.%20Guo&volume=47&publication_year=2019&pages=D140-D144&pmid=30380072&doi=10.1093/nar/gky1051&)

23. Huang H.-Y., Lin Y.-C.-D., Cui S., Huang Y., Tang Y., Xu J., Bao J., Li Y., Wen J., Zuo H., et al. miRTarBase update 2022: An informative resource for experimentally validated miRNA–target interactions. Nucleic Acids Res. 2022;50:D222–D230. doi: 10.1093/nar/gkab1079.  [DOI](https://doi.org/10.1093/nar/gkab1079) | [PMC free article](/articles/PMC8728135/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34850920/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&title=miRTarBase%20update%202022:%20An%20informative%20resource%20for%20experimentally%20validated%20miRNA%E2%80%93target%20interactions&author=H.-Y.%20Huang&author=Y.-C.-D.%20Lin&author=S.%20Cui&author=Y.%20Huang&author=Y.%20Tang&volume=50&publication_year=2022&pages=D222-D230&pmid=34850920&doi=10.1093/nar/gkab1079&)

24. Dimitrakopoulos G.N., Klapa M.I., Moschonas N.K. PICKLE 3.0: Enriching the human meta-database with the mouse protein interactome extended via mouse–human orthology. Bioinformatics. 2020;37:145–146. doi: 10.1093/bioinformatics/btaa1070.  [DOI](https://doi.org/10.1093/bioinformatics/btaa1070) | [PMC free article](/articles/PMC8034533/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33367505/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=PICKLE%203.0:%20Enriching%20the%20human%20meta-database%20with%20the%20mouse%20protein%20interactome%20extended%20via%20mouse%E2%80%93human%20orthology&author=G.N.%20Dimitrakopoulos&author=M.I.%20Klapa&author=N.K.%20Moschonas&volume=37&publication_year=2020&pages=145-146&pmid=33367505&doi=10.1093/bioinformatics/btaa1070&)

25. Dimitrakopoulos G.N., Klapa M.I., Moschonas N.K. How Far Are We from the Completion of the Human Protein Interactome Reconstruction? Biomolecules. 2022;12:140. doi: 10.3390/biom12010140.  [DOI](https://doi.org/10.3390/biom12010140) | [PMC free article](/articles/PMC8774112/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35053288/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biomolecules&title=How%20Far%20Are%20We%20from%20the%20Completion%20of%20the%20Human%20Protein%20Interactome%20Reconstruction?&author=G.N.%20Dimitrakopoulos&author=M.I.%20Klapa&author=N.K.%20Moschonas&volume=12&publication_year=2022&pages=140&pmid=35053288&doi=10.3390/biom12010140&)

26. Shannon P., Markiel A., Ozier O., Baliga N.S., Wang J.T., Ramage D., Amin N., Schwikowski B., Ideker T. Cytoscape: A software environment for integrated models of biomolecular interaction networks. Genome Res. 2003;13:2498–2504. doi: 10.1101/gr.1239303.  [DOI](https://doi.org/10.1101/gr.1239303) | [PMC free article](/articles/PMC403769/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14597658/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res.&title=Cytoscape:%20A%20software%20environment%20for%20integrated%20models%20of%20biomolecular%20interaction%20networks&author=P.%20Shannon&author=A.%20Markiel&author=O.%20Ozier&author=N.S.%20Baliga&author=J.T.%20Wang&volume=13&publication_year=2003&pages=2498-2504&pmid=14597658&doi=10.1101/gr.1239303&)

27. Berna-Rico E., Perez-Bootello J., Abbad-Jaime de Aragon C., Gonzalez-Cantero A. Genetic Influence on Treatment Response in Psoriasis: New Insights into Personalized Medicine. Int. J. Mol. Sci. 2023;24:9850. doi: 10.3390/ijms24129850.  [DOI](https://doi.org/10.3390/ijms24129850) | [PMC free article](/articles/PMC10298473/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/37372997/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Mol.%20Sci.&title=Genetic%20Influence%20on%20Treatment%20Response%20in%20Psoriasis:%20New%20Insights%20into%20Personalized%20Medicine&author=E.%20Berna-Rico&author=J.%20Perez-Bootello&author=C.%20Abbad-Jaime%20de%20Aragon&author=A.%20Gonzalez-Cantero&volume=24&publication_year=2023&pages=9850&pmid=37372997&doi=10.3390/ijms24129850&)

28. Morelli M., Galluzzo M., Madonna S., Scarponi C., Scaglione G.L., Galluccio T., Andreani M., Pallotta S., Girolomoni G., Bianchi L., et al. HLA-Cw6 and other HLA-C alleles, as well as MICBDT, DDX58, and TYK2 genetic variants associate with optimal response to anti-IL-17A treatment in patients with psoriasis. Expert Opin. Biol. Ther. 2021;21:259–270. doi: 10.1080/14712598.2021.1862082.  [DOI](https://doi.org/10.1080/14712598.2021.1862082) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33297781/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Expert%20Opin.%20Biol.%20Ther.&title=HLA-Cw6%20and%20other%20HLA-C%20alleles,%20as%20well%20as%20MICBDT,%20DDX58,%20and%20TYK2%20genetic%20variants%20associate%20with%20optimal%20response%20to%20anti-IL-17A%20treatment%20in%20patients%20with%20psoriasis&author=M.%20Morelli&author=M.%20Galluzzo&author=S.%20Madonna&author=C.%20Scarponi&author=G.L.%20Scaglione&volume=21&publication_year=2021&pages=259-270&pmid=33297781&doi=10.1080/14712598.2021.1862082&)

29. Stuart P.E., Nair R.P., Tsoi L.C., Tejasvi T., Das S., Kang H.M., Ellinghaus E., Chandran V., Callis-Duffin K., Ike R., et al. Genome-wide Association Analysis of Psoriatic Arthritis and Cutaneous Psoriasis Reveals Differences in Their Genetic Architecture. Am. J. Hum. Genet. 2015;97:816–836. doi: 10.1016/j.ajhg.2015.10.019.  [DOI](https://doi.org/10.1016/j.ajhg.2015.10.019) | [PMC free article](/articles/PMC4678416/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26626624/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Hum.%20Genet.&title=Genome-wide%20Association%20Analysis%20of%20Psoriatic%20Arthritis%20and%20Cutaneous%20Psoriasis%20Reveals%20Differences%20in%20Their%20Genetic%20Architecture&author=P.E.%20Stuart&author=R.P.%20Nair&author=L.C.%20Tsoi&author=T.%20Tejasvi&author=S.%20Das&volume=97&publication_year=2015&pages=816-836&pmid=26626624&doi=10.1016/j.ajhg.2015.10.019&)

30. Bowes J., Budu-Aggrey A., Huffmeier U., Uebe S., Steel K., Hebert H.L., Wallace C., Massey J., Bruce I.N., Bluett J., et al. Dense genotyping of immune-related susceptibility loci reveals new insights into the genetics of psoriatic arthritis. Nat. Commun. 2015;6:6046. doi: 10.1038/ncomms7046.  [DOI](https://doi.org/10.1038/ncomms7046) | [PMC free article](/articles/PMC4327416/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25651891/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&title=Dense%20genotyping%20of%20immune-related%20susceptibility%20loci%20reveals%20new%20insights%20into%20the%20genetics%20of%20psoriatic%20arthritis&author=J.%20Bowes&author=A.%20Budu-Aggrey&author=U.%20Huffmeier&author=S.%20Uebe&author=K.%20Steel&volume=6&publication_year=2015&pages=6046&pmid=25651891&doi=10.1038/ncomms7046&)

31. Yin X., Low H.Q., Wang L., Li Y., Ellinghaus E., Han J., Estivill X., Sun L., Zuo X., Shen C., et al. Genome-wide meta-analysis identifies multiple novel associations and ethnic heterogeneity of psoriasis susceptibility. Nat. Commun. 2015;6:6916. doi: 10.1038/ncomms7916.  [DOI](https://doi.org/10.1038/ncomms7916) | [PMC free article](/articles/PMC4423213/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25903422/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&title=Genome-wide%20meta-analysis%20identifies%20multiple%20novel%20associations%20and%20ethnic%20heterogeneity%20of%20psoriasis%20susceptibility&author=X.%20Yin&author=H.Q.%20Low&author=L.%20Wang&author=Y.%20Li&author=E.%20Ellinghaus&volume=6&publication_year=2015&pages=6916&pmid=25903422&doi=10.1038/ncomms7916&)

32. Allanore Y., Saad M., Dieudé P., Avouac J., Distler J.H., Amouyel P., Matucci-Cerinic M., Riemekasten G., Airo P., Melchers I., et al. Genome-wide scan identifies TNIP1, PSORS1C1, and RHOB as novel risk loci for systemic sclerosis. PLoS Genet. 2011;7:e1002091. doi: 10.1371/journal.pgen.1002091.  [DOI](https://doi.org/10.1371/journal.pgen.1002091) | [PMC free article](/articles/PMC3131285/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21750679/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet.&title=Genome-wide%20scan%20identifies%20TNIP1,%20PSORS1C1,%20and%20RHOB%20as%20novel%20risk%20loci%20for%20systemic%20sclerosis&author=Y.%20Allanore&author=M.%20Saad&author=P.%20Dieud%C3%A9&author=J.%20Avouac&author=J.H.%20Distler&volume=7&publication_year=2011&pages=e1002091&pmid=21750679&doi=10.1371/journal.pgen.1002091&)

33. Liu J.Z., van Sommeren S., Huang H., Ng S.C., Alberts R., Takahashi A., Ripke S., Lee J.C., Jostins L., Shah T., et al. Association analyses identify 38 susceptibility loci for inflammatory bowel disease and highlight shared genetic risk across populations. Nat. Genet. 2015;47:979–986. doi: 10.1038/ng.3359.  [DOI](https://doi.org/10.1038/ng.3359) | [PMC free article](/articles/PMC4881818/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26192919/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet.&title=Association%20analyses%20identify%2038%20susceptibility%20loci%20for%20inflammatory%20bowel%20disease%20and%20highlight%20shared%20genetic%20risk%20across%20populations&author=J.Z.%20Liu&author=S.%20van%20Sommeren&author=H.%20Huang&author=S.C.%20Ng&author=R.%20Alberts&volume=47&publication_year=2015&pages=979-986&pmid=26192919&doi=10.1038/ng.3359&)

34. Miyagawa T., Tokunaga K. Genetics of narcolepsy. Hum. Genome Var. 2019;6:4. doi: 10.1038/s41439-018-0033-7.  [DOI](https://doi.org/10.1038/s41439-018-0033-7) | [PMC free article](/articles/PMC6325123/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30652006/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum.%20Genome%20Var.&title=Genetics%20of%20narcolepsy&author=T.%20Miyagawa&author=K.%20Tokunaga&volume=6&publication_year=2019&pages=4&pmid=30652006&doi=10.1038/s41439-018-0033-7&)

35. Wang H., Yang J., Schneider J.A., De Jager P.L., Bennett D.A., Zhang H.-Y. Genome-wide interaction analysis of pathological hallmarks in Alzheimer’s disease. Neurobiol. Aging. 2020;93:61–68. doi: 10.1016/j.neurobiolaging.2020.04.025.  [DOI](https://doi.org/10.1016/j.neurobiolaging.2020.04.025) | [PMC free article](/articles/PMC9795865/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32450446/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neurobiol.%20Aging&title=Genome-wide%20interaction%20analysis%20of%20pathological%20hallmarks%20in%20Alzheimer%E2%80%99s%20disease&author=H.%20Wang&author=J.%20Yang&author=J.A.%20Schneider&author=P.L.%20De%20Jager&author=D.A.%20Bennett&volume=93&publication_year=2020&pages=61-68&pmid=32450446&doi=10.1016/j.neurobiolaging.2020.04.025&)

36. Meyer B., Chavez R.A., Munro J.E., Chiaroni-Clarke R.C., Akikusa J.D., Allen R.C., Craig J.M., Ponsonby A.-L., Saffery R., Ellis J.A. DNA methylation at IL32 in juvenile idiopathic arthritis. Sci. Rep. 2015;5:11063. doi: 10.1038/srep11063.  [DOI](https://doi.org/10.1038/srep11063) | [PMC free article](/articles/PMC4603785/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26057774/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&title=DNA%20methylation%20at%20IL32%20in%20juvenile%20idiopathic%20arthritis&author=B.%20Meyer&author=R.A.%20Chavez&author=J.E.%20Munro&author=R.C.%20Chiaroni-Clarke&author=J.D.%20Akikusa&volume=5&publication_year=2015&pages=11063&pmid=26057774&doi=10.1038/srep11063&)

37. Dutta D., Nagappa M., Sreekumaran Nair B.V., Das S.K., Wahatule R., Sinha S., Vasanthapuram R., Taly A.B., Debnath M. Variations within Toll-like receptor (TLR) and TLR signaling pathway-related genes and their synergistic effects on the risk of Guillain-Barré syndrome. J. Peripher. Nerv. Syst. 2022;27:131–143. doi: 10.1111/jns.12484.  [DOI](https://doi.org/10.1111/jns.12484) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35138004/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Peripher.%20Nerv.%20Syst.&title=Variations%20within%20Toll-like%20receptor%20(TLR)%20and%20TLR%20signaling%20pathway-related%20genes%20and%20their%20synergistic%20effects%20on%20the%20risk%20of%20Guillain-Barr%C3%A9%20syndrome&author=D.%20Dutta&author=M.%20Nagappa&author=B.V.%20Sreekumaran%20Nair&author=S.K.%20Das&author=R.%20Wahatule&volume=27&publication_year=2022&pages=131-143&pmid=35138004&doi=10.1111/jns.12484&)

38. Yenmis G., Oner T., Cam C., Koc A., Kucuk O.S., Yakicier M.C., Dizman D., Kanigur Sultuybek G. Association of NFKB1 and NFKBIA polymorphisms in relation to susceptibility of Behçet’s disease. Scand. J. Immunol. 2015;81:81–86. doi: 10.1111/sji.12251.  [DOI](https://doi.org/10.1111/sji.12251) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25367031/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Scand.%20J.%20Immunol.&title=Association%20of%20NFKB1%20and%20NFKBIA%20polymorphisms%20in%20relation%20to%20susceptibility%20of%20Beh%C3%A7et%E2%80%99s%20disease&author=G.%20Yenmis&author=T.%20Oner&author=C.%20Cam&author=A.%20Koc&author=O.S.%20Kucuk&volume=81&publication_year=2015&pages=81-86&pmid=25367031&doi=10.1111/sji.12251&)

39. Bank S., Julsgaard M., Abed O.K., Burisch J., Broder Brodersen J., Pedersen N.K., Gouliaev A., Ajan R., Nytoft Rasmussen D., Honore Grauslund C., et al. Polymorphisms in the NFkB, TNF-alpha, IL-1beta, and IL-18 pathways are associated with response to anti-TNF therapy in Danish patients with inflammatory bowel disease. Aliment. Pharmacol. Ther. 2019;49:890–903. doi: 10.1111/apt.15187.  [DOI](https://doi.org/10.1111/apt.15187) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30811631/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Aliment.%20Pharmacol.%20Ther.&title=Polymorphisms%20in%20the%20NFkB,%20TNF-alpha,%20IL-1beta,%20and%20IL-18%20pathways%20are%20associated%20with%20response%20to%20anti-TNF%20therapy%20in%20Danish%20patients%20with%20inflammatory%20bowel%20disease&author=S.%20Bank&author=M.%20Julsgaard&author=O.K.%20Abed&author=J.%20Burisch&author=J.%20Broder%20Brodersen&volume=49&publication_year=2019&pages=890-903&pmid=30811631&doi=10.1111/apt.15187&)

40. Rezazadeh M., Gharesouran J., Moradi M., Noroozi R., Omrani M.D., Taheri M., Ghafouri-Fard S. Association study of ANRIL genetic variants and multiple sclerosis. J. Mol. Neurosci. 2018;65:54–59. doi: 10.1007/s12031-018-1069-3.  [DOI](https://doi.org/10.1007/s12031-018-1069-3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29713948/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Mol.%20Neurosci.&title=Association%20study%20of%20ANRIL%20genetic%20variants%20and%20multiple%20sclerosis&author=M.%20Rezazadeh&author=J.%20Gharesouran&author=M.%20Moradi&author=R.%20Noroozi&author=M.D.%20Omrani&volume=65&publication_year=2018&pages=54-59&pmid=29713948&doi=10.1007/s12031-018-1069-3&)

41. Rakhshan A., Zarrinpour N., Moradi A., Ahadi M., Omrani M.D., Ghafouri-Fard S., Taheri M. Genetic variants within ANRIL (antisense non-coding RNA in the INK4 locus) are associated with risk of psoriasis. Int. Immunopharmacol. 2020;78:106053. doi: 10.1016/j.intimp.2019.106053.  [DOI](https://doi.org/10.1016/j.intimp.2019.106053) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31812071/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int.%20Immunopharmacol.&title=Genetic%20variants%20within%20ANRIL%20(antisense%20non-coding%20RNA%20in%20the%20INK4%20locus)%20are%20associated%20with%20risk%20of%20psoriasis&author=A.%20Rakhshan&author=N.%20Zarrinpour&author=A.%20Moradi&author=M.%20Ahadi&author=M.D.%20Omrani&volume=78&publication_year=2020&pages=106053&pmid=31812071&doi=10.1016/j.intimp.2019.106053&)

42. Hao S., Zhou K., Yu P., Tian J., Zhang Y., Dang L., Tong N., Li Y. ANRIL polymorphisms in psoriasis vulgaris patients in northern China. Eur. J. Dermatol. 2022;32:259–268. doi: 10.1684/ejd.2022.4240.  [DOI](https://doi.org/10.1684/ejd.2022.4240) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35866908/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Dermatol.&title=ANRIL%20polymorphisms%20in%20psoriasis%20vulgaris%20patients%20in%20northern%20China&author=S.%20Hao&author=K.%20Zhou&author=P.%20Yu&author=J.%20Tian&author=Y.%20Zhang&volume=32&publication_year=2022&pages=259-268&pmid=35866908&doi=10.1684/ejd.2022.4240&)

43. Bochenek G., Häsler R., El Mokhtari N.E., König I.R., Loos B.G., Jepsen S., Rosenstiel P., Schreiber S., Schaefer A.S. The large non-coding RNA ANRIL, which is associated with atherosclerosis, periodontitis and several forms of cancer, regulates ADIPOR1, VAMP3 and C11ORF10. Hum. Mol. Genet. 2013;22:4516–4527. doi: 10.1093/hmg/ddt299.  [DOI](https://doi.org/10.1093/hmg/ddt299) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23813974/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum.%20Mol.%20Genet.&title=The%20large%20non-coding%20RNA%20ANRIL,%20which%20is%20associated%20with%20atherosclerosis,%20periodontitis%20and%20several%20forms%20of%20cancer,%20regulates%20ADIPOR1,%20VAMP3%20and%20C11ORF10&author=G.%20Bochenek&author=R.%20H%C3%A4sler&author=N.E.%20El%20Mokhtari&author=I.R.%20K%C3%B6nig&author=B.G.%20Loos&volume=22&publication_year=2013&pages=4516-4527&pmid=23813974&doi=10.1093/hmg/ddt299&)

44. Ziegler C., Graf J., Faderl S., Schedlbauer J., Strieder N., Förstl B., Spang R., Bruckmann A., Merkl R., Hombach S., et al. The long non-coding RNA LINC00941 and SPRR5 are novel regulators of human epidermal homeostasis. EMBO Rep. 2019;20:e46612. doi: 10.15252/embr.201846612.  [DOI](https://doi.org/10.15252/embr.201846612) | [PMC free article](/articles/PMC6362352/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30622217/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=EMBO%20Rep.&title=The%20long%20non-coding%20RNA%20LINC00941%20and%20SPRR5%20are%20novel%20regulators%20of%20human%20epidermal%20homeostasis&author=C.%20Ziegler&author=J.%20Graf&author=S.%20Faderl&author=J.%20Schedlbauer&author=N.%20Strieder&volume=20&publication_year=2019&pages=e46612&pmid=30622217&doi=10.15252/embr.201846612&)

45. Adylova A., Mukhanbetzhanovna A.A., Attar R., Yulaevna I.M., Farooqi A.A. Regulation of TGFβ/SMAD signaling by long non-coding RNAs in different cancers: Dark Knight in the Castle of molecular oncology. Noncoding RNA Res. 2021;6:23–28. doi: 10.1016/j.ncrna.2020.12.003.  [DOI](https://doi.org/10.1016/j.ncrna.2020.12.003) | [PMC free article](/articles/PMC7814108/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33511320/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Noncoding%20RNA%20Res.&title=Regulation%20of%20TGF%CE%B2/SMAD%20signaling%20by%20long%20non-coding%20RNAs%20in%20different%20cancers:%20Dark%20Knight%20in%20the%20Castle%20of%20molecular%20oncology&author=A.%20Adylova&author=A.A.%20Mukhanbetzhanovna&author=R.%20Attar&author=I.M.%20Yulaevna&author=A.A.%20Farooqi&volume=6&publication_year=2021&pages=23-28&pmid=33511320&doi=10.1016/j.ncrna.2020.12.003&)

46. Dodge-Kafka K.L., Soughayer J., Pare G.C., Carlisle Michel J.J., Langeberg L.K., Kapiloff M.S., Scott J.D. The protein kinase A anchoring protein mAKAP coordinates two integrated cAMP effector pathways. Nature. 2005;437:574–578. doi: 10.1038/nature03966.  [DOI](https://doi.org/10.1038/nature03966) | [PMC free article](/articles/PMC1636584/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16177794/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=The%20protein%20kinase%20A%20anchoring%20protein%20mAKAP%20coordinates%20two%20integrated%20cAMP%20effector%20pathways&author=K.L.%20Dodge-Kafka&author=J.%20Soughayer&author=G.C.%20Pare&author=J.J.%20Carlisle%20Michel&author=L.K.%20Langeberg&volume=437&publication_year=2005&pages=574-578&pmid=16177794&doi=10.1038/nature03966&)

47. Mauban J.R.H., O’ Donnell M., Warrier S., Manni S., Bond M. AKAP-scaffolding proteins and regulation of cardiac physiology. Physiology. 2009;24:78–87. doi: 10.1152/physiol.00041.2008.  [DOI](https://doi.org/10.1152/physiol.00041.2008) | [PMC free article](/articles/PMC2751630/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19364910/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Physiology&title=AKAP-scaffolding%20proteins%20and%20regulation%20of%20cardiac%20physiology&author=J.R.H.%20Mauban&author=M.%20O%E2%80%99%20Donnell&author=S.%20Warrier&author=S.%20Manni&author=M.%20Bond&volume=24&publication_year=2009&pages=78-87&pmid=19364910&doi=10.1152/physiol.00041.2008&)

48. Qasim H., McConnell B.K. AKAP12 Signaling Complex: Impacts of Compartmentalizing cAMP-Dependent Signaling Pathways in the Heart and Various Signaling Systems. J. Am. Heart Assoc. 2020;9:e016615. doi: 10.1161/JAHA.120.016615.  [DOI](https://doi.org/10.1161/JAHA.120.016615) | [PMC free article](/articles/PMC7670535/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32573313/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Am.%20Heart%20Assoc.&title=AKAP12%20Signaling%20Complex:%20Impacts%20of%20Compartmentalizing%20cAMP-Dependent%20Signaling%20Pathways%20in%20the%20Heart%20and%20Various%20Signaling%20Systems&author=H.%20Qasim&author=B.K.%20McConnell&volume=9&publication_year=2020&pages=e016615&pmid=32573313&doi=10.1161/JAHA.120.016615&)

49. Baillie G.S., Scott J.D., Houslay M.D. Compartmentalisation of phosphodiesterases and protein kinase A: Opposites attract. FEBS Lett. 2005;579:3264–3270. doi: 10.1016/j.febslet.2005.03.089.  [DOI](https://doi.org/10.1016/j.febslet.2005.03.089) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15943971/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=FEBS%20Lett.&title=Compartmentalisation%20of%20phosphodiesterases%20and%20protein%20kinase%20A:%20Opposites%20attract&author=G.S.%20Baillie&author=J.D.%20Scott&author=M.D.%20Houslay&volume=579&publication_year=2005&pages=3264-3270&pmid=15943971&doi=10.1016/j.febslet.2005.03.089&)

50. Liu Q., Yao F., Wang M., Zhou B., Cheng H., Wang W., Jin L., Lin Q., Wang J.-C. Novel human BTB/POZ domain-containing zinc finger protein ZBTB1 inhibits transcriptional activities of CRE. Mol. Cell Biochem. 2011;357:405–414. doi: 10.1007/s11010-011-0911-5.  [DOI](https://doi.org/10.1007/s11010-011-0911-5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21706167/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Cell%20Biochem.&title=Novel%20human%20BTB/POZ%20domain-containing%20zinc%20finger%20protein%20ZBTB1%20inhibits%20transcriptional%20activities%20of%20CRE&author=Q.%20Liu&author=F.%20Yao&author=M.%20Wang&author=B.%20Zhou&author=H.%20Cheng&volume=357&publication_year=2011&pages=405-414&pmid=21706167&doi=10.1007/s11010-011-0911-5&)

51. Wen A.Y., Sakamoto K.M., Miller L.S. The role of the transcription factor CREB in immune function. J. Immunol. 2010;185:6413–6419. doi: 10.4049/jimmunol.1001829.  [DOI](https://doi.org/10.4049/jimmunol.1001829) | [PMC free article](/articles/PMC5519339/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21084670/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Immunol.&title=The%20role%20of%20the%20transcription%20factor%20CREB%20in%20immune%20function&author=A.Y.%20Wen&author=K.M.%20Sakamoto&author=L.S.%20Miller&volume=185&publication_year=2010&pages=6413-6419&pmid=21084670&doi=10.4049/jimmunol.1001829&)

52. Servillo G., Della Fazia M.A., Sassone-Corsi P. Coupling cAMP signaling to transcription in the liver: Pivotal role of CREB and CREM. Exp. Cell Res. 2002;275:143–154. doi: 10.1006/excr.2002.5491.  [DOI](https://doi.org/10.1006/excr.2002.5491) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11969286/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Exp.%20Cell%20Res.&title=Coupling%20cAMP%20signaling%20to%20transcription%20in%20the%20liver:%20Pivotal%20role%20of%20CREB%20and%20CREM&author=G.%20Servillo&author=M.A.%20Della%20Fazia&author=P.%20Sassone-Corsi&volume=275&publication_year=2002&pages=143-154&pmid=11969286&doi=10.1006/excr.2002.5491&)

53. Ferreiro D.U., Komives E.A. Molecular mechanisms of system control of NF-kappaB signaling by IkappaBalpha. Biochemistry. 2010;49:1560–1567. doi: 10.1021/bi901948j.  [DOI](https://doi.org/10.1021/bi901948j) | [PMC free article](/articles/PMC2865148/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20055496/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochemistry&title=Molecular%20mechanisms%20of%20system%20control%20of%20NF-kappaB%20signaling%20by%20IkappaBalpha&author=D.U.%20Ferreiro&author=E.A.%20Komives&volume=49&publication_year=2010&pages=1560-1567&pmid=20055496&doi=10.1021/bi901948j&)

54. Wang D., Sadee W. CYP3A4 intronic SNP rs35599367 (CYP3A4*22) alters RNA splicing. Pharmacogenet. Genom. 2016;26:40–43. doi: 10.1097/FPC.0000000000000183.  [DOI](https://doi.org/10.1097/FPC.0000000000000183) | [PMC free article](/articles/PMC4674354/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26488616/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet.%20Genom.&title=CYP3A4%20intronic%20SNP%20rs35599367%20(CYP3A4*22)%20alters%20RNA%20splicing&author=D.%20Wang&author=W.%20Sadee&volume=26&publication_year=2016&pages=40-43&pmid=26488616&doi=10.1097/FPC.0000000000000183&)

55. Wang D., Guo Y., Wrighton S.A., Cooke G.E., Sadee W. Intronic polymorphism in CYP3A4 affects hepatic expression and response to statin drugs. Pharmacogenom. J. 2011;11:274–286. doi: 10.1038/tpj.2010.28.  [DOI](https://doi.org/10.1038/tpj.2010.28) | [PMC free article](/articles/PMC3248744/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20386561/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenom.%20J.&title=Intronic%20polymorphism%20in%20CYP3A4%20affects%20hepatic%20expression%20and%20response%20to%20statin%20drugs&author=D.%20Wang&author=Y.%20Guo&author=S.A.%20Wrighton&author=G.E.%20Cooke&author=W.%20Sadee&volume=11&publication_year=2011&pages=274-286&pmid=20386561&doi=10.1038/tpj.2010.28&)

56. Reich K., Mrowietz U., Menter A., Griffiths C.E.M., Bagel J., Strober B., Nunez Gomez N., Shi R., Guerette B., Lebwohl M. Effect of baseline disease severity on achievement of treatment target with apremilast: Results from a pooled analysis. J. Eur. Acad. Dermatol. Venereol. 2021;35:2409–2414. doi: 10.1111/jdv.17520.  [DOI](https://doi.org/10.1111/jdv.17520) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34255891/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Eur.%20Acad.%20Dermatol.%20Venereol.&title=Effect%20of%20baseline%20disease%20severity%20on%20achievement%20of%20treatment%20target%20with%20apremilast:%20Results%20from%20a%20pooled%20analysis&author=K.%20Reich&author=U.%20Mrowietz&author=A.%20Menter&author=C.E.M.%20Griffiths&author=J.%20Bagel&volume=35&publication_year=2021&pages=2409-2414&pmid=34255891&doi=10.1111/jdv.17520&)

57. Wolf P. Systemic antipsoriatic treatment: Do women respond better than men and if so, why? Br. J. Dermatol. 2021;185:1088–1089. doi: 10.1111/bjd.20710.  [DOI](https://doi.org/10.1111/bjd.20710) | [PMC free article](/articles/PMC9293209/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34617589/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Dermatol.&title=Systemic%20antipsoriatic%20treatment:%20Do%20women%20respond%20better%20than%20men%20and%20if%20so,%20why?&author=P.%20Wolf&volume=185&publication_year=2021&pages=1088-1089&pmid=34617589&doi=10.1111/bjd.20710&)

58. Guillet C., Seeli C., Nina M., Maul L.V., Maul J.-T. The impact of gender and sex in psoriasis: What to be aware of when treating women with psoriasis. Int. J. Womens Dermatol. 2022;8:e010. doi: 10.1097/JW9.0000000000000010.  [DOI](https://doi.org/10.1097/JW9.0000000000000010) | [PMC free article](/articles/PMC9112394/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35619672/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Womens%20Dermatol.&title=The%20impact%20of%20gender%20and%20sex%20in%20psoriasis:%20What%20to%20be%20aware%20of%20when%20treating%20women%20with%20psoriasis&author=C.%20Guillet&author=C.%20Seeli&author=M.%20Nina&author=L.V.%20Maul&author=J.-T.%20Maul&volume=8&publication_year=2022&pages=e010&pmid=35619672&doi=10.1097/JW9.0000000000000010&)
