# Whole-Exome sequencing analysis identified TMSB10/TRABD2A locus to be associated with carfilzomib-related cardiotoxicity among patients with multiple myeloma

## Metadata
**Authors:** Marwa Tantawy, Guang Yang, Raghunandan Reddy Algubelli, Gabriel DeAvila, Samuel M Rubinstein, Robert F Cornell, Michael G Fradley, Erin M Siegel, Oliver A Hampton, Ariosto S Silva, Daniel Lenihan, Kenneth H Shain, Rachid C Baz, Yan Gong
**Journal:** Frontiers in Cardiovascular Medicine
**Date:** 2023 Jun 20
**DOI:** [10.3389/fcvm.2023.1181806](https://doi.org/10.3389/fcvm.2023.1181806)
**PMID:** 37408649
**PMCID:** PMC10319068
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10319068/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC10319068/pdf/fcvm-10-1181806.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC10319068/pdf/fcvm-10-1181806.pdf)

## Abstract

**Background:** 
Proteasome inhibitor Carfilzomib (CFZ) is effective in treating patients with refractory or relapsed multiple myeloma (MM) but has been associated with cardiovascular adverse events (CVAE) such as hypertension, cardiomyopathy, and heart failure. This study aimed to investigate the contribution of germline genetic variants in protein-coding genes in CFZ-CVAE among MM patients using whole-exome sequencing (WES) analysis.

**Methods:** 
Exome-wide single-variant association analysis, gene-based analysis, and rare variant analyses were performed on 603,920 variants in 247 patients with MM who have been treated with CFZ and enrolled in the Oncology Research Information Exchange Network (ORIEN) at the Moffitt Cancer Center. Separate analyses were performed in European Americans and African Americans followed by a trans-ethnic meta-analysis.

**Results:** 
The most significant variant in the exome-wide single variant analysis was a missense variant rs7148 in the thymosin beta-10/TraB Domain Containing 2A (TMSB10/TRABD2A) locus. The effect allele of rs7148 was associated with a higher risk of CVAE [odds ratio (OR) = 9.3 with a 95% confidence interval of 3.9—22.3, p = 5.42*10−7]. MM patients with rs7148 AG or AA genotype had a higher risk of CVAE (50%) than those with GG genotype (10%). rs7148 is an expression quantitative trait locus (eQTL) for TRABD2A and TMSB10. The gene-based analysis also showed TRABD2A as the most significant gene associated with CFZ-CVAE (p = 1.06*10−6).

**Conclusions:** 
We identified a missense SNP rs7148 in the TMSB10/TRABD2A as associated with CFZ-CVAE in MM patients. More investigation is needed to understand the underlying mechanisms of these associations.

Keywords: cardio-oncology, proteosome inhibitors, Multiple Myeloma, carfilzomib, cardiotoxcity, whole exome sequencing

### Background

Proteasome inhibitor Carfilzomib (CFZ) is effective in treating patients with refractory or relapsed multiple myeloma (MM) but has been associated with cardiovascular adverse events (CVAE) such as hypertension, cardiomyopathy, and heart failure. This study aimed to investigate the contribution of germline genetic variants in protein-coding genes in CFZ-CVAE among MM patients using whole-exome sequencing (WES) analysis.

### Methods

Exome-wide single-variant association analysis, gene-based analysis, and rare variant analyses were performed on 603,920 variants in 247 patients with MM who have been treated with CFZ and enrolled in the Oncology Research Information Exchange Network (ORIEN) at the Moffitt Cancer Center. Separate analyses were performed in European Americans and African Americans followed by a trans-ethnic meta-analysis.

### Results

The most significant variant in the exome-wide single variant analysis was a missense variant rs7148 in the thymosin beta-10/TraB Domain Containing 2A (*TMSB10/TRABD2A*) locus. The effect allele of rs7148 was associated with a higher risk of CVAE [odds ratio (OR) = 9.3 with a 95% confidence interval of 3.9—22.3, *p* = 5.42*10^−7^]. MM patients with rs7148 AG or AA genotype had a higher risk of CVAE (50%) than those with GG genotype (10%). rs7148 is an expression quantitative trait locus (eQTL) for *TRABD2A* and *TMSB10*. The gene-based analysis also showed *TRABD2A* as the most significant gene associated with CFZ-CVAE (*p* = 1.06*10^−6^).

### Conclusions

We identified a missense SNP rs7148 in the *TMSB10/TRABD2A* as associated with CFZ-CVAE in MM patients. More investigation is needed to understand the underlying mechanisms of these associations.

## 1. Introduction

Multiple Myeloma (MM) is a malignancy of the plasma cells. According to yearly incidence rates and prevalence figures, it ranks third among hematologic cancers in the United States ([1](#B1)). Proteasome inhibitors (PIs) are one of the most effective drugs for the treatment of MM and are the backbone therapies for MM treatment ([2](#B2)). Three PIs have been approved by the United States Food and Drug Administration: bortezomib, carfilzomib, and ixazomib ([3](#B3)). Carfilzomib (Kyprolis®) is a second-generation, irreversible PI approved for treating relapsed and refractory MM due to its survival benefit and overall response rate in refractory MM patients ([4](#B4), [5](#B5)). The National Comprehensive Cancer Network (NCCN) Guideline recommends carfilzomib-based therapy for newly diagnosed MM patients with pre-existing neuropathy or high-risk patients and patients with relapsed and refractory disease ([6](#B6)). Despite its effectiveness, carfilzomib (CFZ) has been shown to have significant cardiovascular adverse events (CVAE) (20%–25%), including 7.2% incident HF, in the clinical trials that excluded patients with pre-existing cardiovascular disorders ([7](#B7)). In the previous two meta-analyses studies, CFZ was associated with a high incidence of CVAE (8%–18%) ([2](#B2), [3](#B3)) including hypertension, heart failure, cardiomyopathy, and arrhythmia ([8](#B8), [9](#B9)). The rate of CVAE was even higher (∼50%) in an observational study when MM patients with pre-existing cardiovascular conditions were not excluded ([10](#B10)). A severe clinical implication developed from this cardiotoxicity is treatment interruption, which could lead to disease progression ([11](#B11)).

Studies have shown that early detection of and early intervention for cardiotoxicity induced by other therapies (i.e., anthracyclines, Trastuzumab) can improve long-term outcomes ([12](#B12), [13](#B13)). Therefore, stratifying patients before the carfilzomib treatment might provide opportunities for early intervention to optimize patient outcomes. Pharmacogenomics, or the identification of genetic determinants of drug response and adverse effects, is a tool that has been useful in individualizing medication therapy ([14](#B14), [15](#B15)). This study aims to identify germline genetic variants in protein-coding genes associated with CFZ-CVAE in MM patients using whole-exome sequencing (WES) analysis.

## 2. Material and methods

### 2.1. Patients

Patients included in this study were admitted to the Moffitt Cancer Center's Total Cancer Care (TCC) Protocol with IRB approval (MCC#14690; Advarra IRB Pro00014441) ([16](#B16)). Patients consented to contribute blood specimens and medical information for research purposes in collaboration with the Oncology Research Information Exchange Network (ORIEN), a network of seventeen cancer centers that have agreed to implement a common TCC biospecimen collection protocol to follow patients throughout their lifetime ([17](#B17)). Clinical and epidemiological data were collected for select TCC-consented patients, and the molecular data were produced as described below. This study included a total of 247 patients who have been diagnosed with MM and treated with carfilzomib at the H. Lee Moffitt Cancer Center and have germline DNA WES data available through ORIEN. This study was also approved by the University of Florida Institutional Review Board (IRB202003031).

### 2.2. Cardiovascular adverse events

We queried the electronic health records data of Moffitt's health research informatics platform to determine if a CVAE had occurred after initiation of carfilzomib treatment. We used the International Classification of Disease (ICD) revisions 9 and 10 to define cardiovascular events. A complete list of the ICD-9 and -10 codes used is listed in the supplementary materials [Supplementary Table S1](#s10). Moffitt's Pentecost Myeloma Research Center clinical database was utilized to identify if the eligible TCC patients receiving carfilzomib as part of their treatment had one of these cardiac events between the start and end date. We performed a manual chart review on 10% of patients to verify the accuracy of the billing records in terms of the definition of the CVAE. All records reviewed for cardiovascular adverse events matched the determination by the billing codes.

### 2.3. Whole-Exome sequencing and quality control

Germline DNA was extracted from peripheral blood samples with buffy coat using the QIAsymphony SP instrument (QIAGEN, Hilden, Germany) following standard protocols implemented across the ORIEN. The WES of germline DNA was performed for each patient using SeqCap EZ Exome Enrichment Kit v3.0 (Roche NimbleGen, Pleasanton, CA) or xGen Exome Hybridization Panel with supplement probes (integrated Data Technologies, Inc., Coralville, IA), with 100 × coverage. Capture kits covered variants for limited regions; each captured library was loaded onto Illumina-HiSeq 4,000 (Illumina, San Diego, CA). Over 26,000 protein-coding genes were sequenced. The raw sequencing data underwent a rigorous analysis pipeline for alignment, variant calling, quality control steps, and annotation algorithms ([18](#B18)–[20](#B20)). Before the genetic association analysis, the WES data underwent additional quality control steps: variant call rate > 95%, sample call rate > 95%, sex check, and Hardy Weinberg Equilibrium analysis. Principle component analysis was performed on a subset of variants after more stringent quality control steps [variant rate and sample call rate > 99% and minor allele frequency (MAF) >10%] to evaluate the genetic ancestry of these patients.

Germline WES data was available on 605,446 variants in 247 patients. Of the 247 patients, 228 were genetically clustered with individuals of European ancestry, and 19 were clustered with African ancestry. A total of 603,920 variants passed the quality control steps and were included in the WES analysis.

### 2.4. Whole-Exome sequencing data analysis

All association analyses were performed in genetically clustered European Americans and African Americans separately, adjusting for age, gender, and principal components for ancestry. Trans-ethnic meta-analyses were then performed to combine the results from both groups.

Exome-wide association analysis of single variants with a MAF ≥ 1% was performed to estimate the odds ratio (OR) and 95% confidence interval (CI) for each variant on chromosomes 1–22 for the development of CFZ-CVAE using multivariable logistic regression assuming an additive model of inheritance using PLINK ([21](#B21)). All variants with *p* < 5*10^−8^ were considered statistical significance. Variants with *p* < 5*10^−4^ were considered suggestive ([22](#B22)).

Following the exome-wide association analysis, the summary statistics were functionally annotated using Functional Mapping and Annotation of Genome-Wide Association Studies (FUMA GWAS) for a gene-based and gene set analysis to recognize potential genes of interest ([23](#B23)). The Genotype-Tissue Expression (GTEx) database was queried to identify tissue-specific gene expression and regulation. ANNOVAR ([24](#B24)) was used to annotate the genetic variants appropriately.

Rare variants analysis was performed using the sequence kernel association test (SKAT) using the SKAT package ([25](#B25), [26](#B26)) to evaluate the association of the joint effect of multiple rare variants (MAF < 1%) with CFZ-CVAE.

### 2.5. Ingenuity pathway analysis (IPA)

Functional assignment and pathway analysis of the association results was performed on the top variants from the WES analyses (*p* < 0.001). IPA uses a network generation algorithm to create multiple networks and uses hypergeometric distribution to create scores for each network ([27](#B27)). The statistical significance level is generated using Fisher's Exact test. Any pathway enriched by genes more than by chance would be statistically significant.

## 3. Results

### 3.1. Patients characteristics

Overall, 247 MM patients were included in the analysis. The mean age was ∼59 years, and 57% were men. Thirty-eight (15.5%) developed CVAE after initiation and during carfilzomib-based therapy. [Table 1](#T1) summarizes the characteristics of the 38 patients who developed CVAE and the 209 who did not. The baseline demographics and medical history were similar between the two groups of patients. A total of 228 (92.3%) patients were genetically clustered with individuals of European ancestry (EA), and 19 clustered with individuals of African ancestry (AA). The baseline characteristics and medical history of the 228 EA MM patients were summarized in [Supplementary Table S2](#s10).

### Table 1.

| Characteristics | Overall (n = 247) | CVAE (n = 38) | No CVAE (n = 209) | P |
| --- | --- | --- | --- | --- |
| Age (years) | 58.9 ± 9.8 | 60.6 ± 9.4 | 58.6 ± 9.8 | 0.27 |
| Sex (male) | 141 (57.1%) | 21 (55.3%) | 120 (57.4%) | 0.81 |
| Race |  |  |  |  |
| European American | 228 (92.3%) | 35 (92.1%) | 193 (92.3%) | 0.96 |
| African American | 19 (7.7%) | 3 (7.9%) | 16 (7.7%) |  |
| Medical History |  |  |  |  |
| Hyperlipidemia | 78 (31.6%) | 15 (39.4%) | 63 (30.1%) | 0.26 |
| Hypertension | 130 (52.6%) | 25 (65.8%) | 105 (50.2%) | 0.077 |
| Diabetes | 22 (8.9%) | 6 (15.8%) | 16 (7.7%) | 0.11 |
| Ischemic Heart Disease | 5 (2.0%) | 1 (2.6%) | 4 (1.9%) | 0.77 |
| Myocardial Infarction | 12 (4.9%) | 3 (7.9%) | 9 (4.3%) | 0.34 |

Table 1 Caption: Demographics and clinical characteristics of patients.

### 3.2. Exome-wide common variant analysis

The results of the exome-wide association analysis of the common variants in the EA patients are summarized in the Manhattan plot ([Figure 1A](#F1)) and the QQ plot ([Figure 1B](#F1)). None of the variants were genome-wide significant. However, eleven SNPs from five loci reached the suggestive significance level with *p* < 5*10^−4^ ([Table 2](#T2)). The top SNP rs7148 is a missense variant in the thymosin beta-10 (*TMSB10)* gene, with OR of 9.33% and 95% CI of 3.90 – 22.35 (*p* = 5.42*10^−07^) ([Table 2](#T2), [Figure 2](#F2)). The minor allele frequency rs7148 was ∼7% in EA patients. Amongst patients with the rs7148 variant, 50% (99/198) of patients with the AG or AA genotype developed a CVAE compared with 10.1% (20/198) with the GG genotype (p < 0.0001). GTEx analysis revealed that the rs7148 A allele was associated with higher TraB Domain Containing 2A (TRABD2A) gene expression in the left ventricle tissue with a *p*-value 1.9 × 10^−7^ ([Supplementary Figure S1](#s10)). The second most significant SNP, rs12471929, is an intronic variant in the *TMSB10* gene that is in perfect linkage disequilibrium (LD) (*r*^2^ = 1, D'=1) with rs7148, with OR of 8.95 (3.78–21.18), *p* = 6.17*10^−7^ ([Table 2](#T2), [Figure 2](#F2)). A few other SNPs in LD with rs7148 in the 1,000 genomes database are shown in [Supplementary Table S3](#s10).

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b51a/10319068/7b3c03109500/fcvm-10-1181806-g001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=10319068_fcvm-10-1181806-g001.jpg)

Results of exome-wide association analysis in European Americans. (A) Manhattan plot representing the association between single-nucleotide polymorphism (SNP) genotype and CVAE in MM patients. (B) The quantile-quantile (QQ) plot of association results.

### Table 2.

| CHR | SNP | BP | Gene | A1 | A2 | MAF | OR | 95% CI | P | dbSNP functional annotation | MAF CVAE |  | MAC CVAE |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| No | Yes | No | Yes |  |  |  |  |  |  |  |  |  |  |  |
| 2 | rs7148 | 85,13,3216 | TMSB10 | A | G | 0.068 | 9.33 | 3.90–22.35 | 5.42E-07 | Missense | 0.039 | 0.229 | 15 | 16 |
| 2 | rs12,47,1929 | 85,13,3320 | TMSB10 | T | C | 0.070 | 8.95 | 3.78–21.18 | 6.17E-07 | Intronic | 0.039 | 0.243 | 15 | 17 |
| 5 | rs28,45,0841 | 32,09,3210 | PDZD2 | T | C | 0.077 | 5.25 | 2.38–11.58 | 3.91E-05 | Intronic | 0.052 | 0.214 | 20 | 15 |
| 5 | rs22,91,113 | 32,07,4509 | PDZD2 | A | G | 0.057 | 6.17 | 2.52–15.15 | 7.05E-05 | Synonymous | 0.036 | 0.171 | 14 | 12 |
| 4 | rs17,38,7037 | 15,99,2783 | PROM1 | C | A | 0.057 | 5.84 | 2.44–139.5 | 7.32E-05 | Intronic | 0.036 | 0.171 | 14 | 12 |
| 2 | rs75,34,8511 | 11,73,8951 | GREB1 | T | C | 0.037 | 8.30 | 2.90–23.77 | 8.08E-05 | Synonymous | 0.021 | 0.129 | 8 | 9 |
| 8 | rs22,72,682 | 12,60,49443 | WASHC5 | C | T | 0.094 | 3.83 | 1.95–7.49 | 9.26E-05 | Intronic | 0.065 | 0.257 | 25 | 18 |
| 8 | rs11,54,2889 | 12,60,44527 | WASHC5/SQLE | T | C | 0.059 | 5.01 | 2.23–11.24 | 9.50E-05 | Synonymous | 0.034 | 0.200 | 13 | 14 |
| 5 | rs37,33,720 | 32,08,7808 | PDZD2 | C | G | 0.072 | 4.90 | 2.20–10.87 | 9.55E-05 | Synonymous | 0.049 | 0.200 | 19 | 14 |
| 5 | rs10,06,6063 | 32,09,0294 | PDZD2 | A | G | 0.072 | 4.90 | 2.20–10.87 | 9.55E-05 | Missense | 0.049 | 0.200 | 19 | 14 |
| 5 | rs16,88,9442 | 32,09,3070 | PDZD2 | A | G | 0.072 | 4.90 | 2.20–10.87 | 9.55E-05 | Intronic | 0.049 | 0.200 | 19 | 14 |

Table 2 Caption: Top SNPs in the WES analysis of CFZ-CVAE in the European American patients.

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b51a/10319068/c680009f3b09/fcvm-10-1181806-g002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=10319068_fcvm-10-1181806-g002.jpg)

Regional association plot for variant rs7148. The X-axis represents a 1Mb region, 500 kb on either side of the variant, and the y-axis shows –the log10 P-value for individual SNPs. Pairwise LD (r2) with the variant is based on 1,000 Genome phase 3 v5 European reference samples and described using the color scale in the bar. The bottom panel shows the genes located within the region.

Among the other SNPs with a suggestive level of significance were: five SNPs in the *PDZD2* gene, which encodes PDZ Domain Containing Protein 2; an intronic variant in the Prominin 1, CD133 (*PROM1*) gene; a synonymous variant on *GREB1* (Growth Regulating Estrogen Receptor Binding 1) gene on Chromosome 2, and two variants in *WASHC5/SQLE* gene on Chromosome 8 ([Table 2](#T2)). The minor allele frequencies, allele counts, and Hardy Weinberg Equilibrium test results by CVAE status are summarized in [Supplementary Table S4](#s10).

We also performed an exploratory analysis on AA patients. No SNPs reached statistical significance as a result of the small sample size. The four SNPs with nominal significance (*p* < 0.05) are shown in [Supplementary Table S5](#s10). The two most frequent SNPs in EA (rs7148, rs1247192) were not observed in AA patients (MAF = 0).

The top SNPs in the trans-ethnic meta-analysis combing EA and AA are listed in [Table 3](#T3). Only two of these eleven SNPs were observed in AA. Therefore, the results of these SNPs in the meta-analysis were almost identical to those in the EA analysis. The only top EA SNPs observed in AA were the Chromosome 8 SNPs in the *WASCH5/SQLE* locus. While these two SNPs had minor allele frequencies of 6%–9% in EA, the frequencies were much higher in AA (42%–45%). The directions of associations with CVAE were consistent in AA patients compared to those in EA patients ([Table 3](#T3)).

### Table 3.

| CHR | SNP | BP | Gene | A1 | A2 | EA |  |  |  | AA |  |  |  | Meta-analysis |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| MAF | OR | 95% CI | P | MAF | OR | 95% CI | P | MAF | meta P | Direction |  |  |  |  |  |  |
| 2 | rs7148 | 85,13,3216 | TMSB10 | A | G | 0.068 | 9.33 | 3.90–22.35 | 5.42E-07 | 0 | – | – | – | 0.068 | 5.43E-07 | +? |
| 2 | rs12,47,1929 | 85,13,3320 | TMSB10 | T | C | 0.070 | 8.95 | 3.78–21.18 | 6.17E-07 | 0 | – | – | – | 0.070 | 6.18E-07 | +? |
| 5 | rs28,45,0841 | 32,09,3210 | PDZD2 | T | C | 0.077 | 5.25 | 2.38–11.58 | 3.91E-05 | 0 | – | – | – | 0.077 | 3.90E-05 | +? |
| 8 | rs11,54,2889 | 12,60,44527 | WASHC5/SQLE | T | C | 0.059 | 5.01 | 2.23–11.24 | 9.50E-05 | 0.42 | 20.76 | 0.10–4300 | 0.27 | 0.067 | 5.67E-05 | ++ |
| 8 | rs22,72,682 | 12,60,49443 | WASHC5 | C | T | 0.094 | 3.83 | 1.95–7.49 | 9.26E-05 | 0.45 | 3.48 | 0.17–70.85 | 0.42 | 0.110 | 6.55E-05 | -- |
| 5 | rs22,91,113 | 32,07,4509 | PDZD2 | A | G | 0.057 | 6.17 | 2.52–15.15 | 7.05E-05 | 0 | – | – | – | 0.057 | 7.06E-05 | +? |
| 4 | rs17,38,7037 | 15,99,2783 | PROM1 | C | A | 0.057 | 5.84 | 2.44–139.5 | 7.32E-05 | 0 | – | – | – | 0.943 | 7.31E-05 | -? |
| 2 | rs75,34,8511 | 11,73,8951 | GREB1 | T | C | 0.037 | 8.30 | 2.90–23.77 | 8.08E-05 | 0 | – | – | – | 0.037 | 8.08E-05 | +? |
| 5 | rs37,33,720 | 32,08,7808 | PDZD2 | C | G | 0.072 | 4.90 | 2.20–10.87 | 9.55E-05 | 0 | – | – | – | 0.072 | 9.55E-05 | +? |
| 5 | rs10,06,6063 | 32,09,0294 | PDZD2 | A | G | 0.072 | 4.90 | 2.20–10.87 | 9.55E-05 | 0 | – | – | – | 0.072 | 9.55E-05 | +? |
| 5 | rs16,88,9442 | 32,09,3070 | PDZD2 | A | G | 0.072 | 4.90 | 2.20–10.87 | 9.55 × 10−5 | 0 | – | – | – | 0.072 | 9.55E-05 | +? |

Table 3 Caption: Top SNPs in the trans-ethnic meta-analysis of 247 ORIEN patients.

### 3.3. Gene-based analysis

The gene-based analysis in EA patients using FUMA revealed that the *TRABD2A* gene that encodes TraB Domain Containing 2A is significantly associated with CFZ-CVAE (*p* = 1.06*10^−6^) ([Figure 3](#F3)).

### Figure 3.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b51a/10319068/f9df4bdb3d5a/fcvm-10-1181806-g003.jpg)

The gene-based association testing results from the FUMA analysis of CFZ-CVAE yielded TRABD2A as a significant gene (p = 1.06*10−6).

### 3.4. Rare variant analysis performed using SKAT

SKAT analysis was performed on 40,969 gene sets and 620,661 SNPs, and a significant association was determined by comparing CFZ-CVAE after correcting for multiple testing. The genes—Chromosome 1 Open Reading Frame 116 (C1orf116), LOC102724084 (DYNLRB2 antisense RNA1), Diphosphoinositol pentakisphosphate kinase 2 (PPIP5K2 (NM_0013)), and Transmembrane Protein 183A (*TMEM183A*) were statistically significantly associated with CFZ-CVAE (*p*-value = 1.1*10^−5^, 4.1*10^−5^, 6.2*10^−5^, and 6.2*10^−5^, respectively) ([Supplementary Table S6](#s10)**)**.

### 3.5. IPA analysis of WES results

Using IPA, the pathway enrichment analysis showed that the lowest *p*-value and most significant genes overlapped with cardiotoxicity. The functional toxicity annotation of genes related to cardiotoxicity implicated cardiac arteriopathy with three variants from WES results: rs3750765 located in leucine-rich repeat-containing 20 (*LRRC20*) gene, rs72713436 in sterile alpha motif domain-containing 4A (*SAMD4A*) gene, and rs75454001 in CUB-Sushi multiple domains 1 (*CSMD1*) gene. All of these genes were associated with human coronary artery disease (*P* < 0.001) ([28](#B28)).

## 4. Discussion

In this first genetic association analysis of CFZ-related CVAE in MM patients, we conducted a WES of germline DNA samples from patients who received CFZ in the ORIEN network. In this retrospective cohort study of patients in a real-world clinical setting, we identified a missense SNP rs7148 in the *TMSB10/TRABD2A* gene locus on chromosome 2 to be associated with a higher risk for CVAE in MM patients treated with CFZ.

In order to reflect real-world practices, all MM patients treated with CFZ were included in this study regardless of their cardiovascular disease history. We found that 15.4% of the MM patients treated with CFZ developed CVAE, which is in line with the event rates of 8%–18% reported in prior meta-analyses ([2](#B2), [3](#B3)). Due to the retrospective nature of this study and a lack of clinical guidelines on proteome inhibitor monitoring at the time of this study, these patients were not actively monitored for cardiotoxicity by cardiologists. Not surprisingly, the event rate we observed is lower than the CFZ-CVAE event rate of 50% reported in a prospective study, the Prospective Observation of Cardiac Safety with Proteasome Inhibitor (PROTECT) study ([10](#B10)). The recently published 2022 Cardio-Oncology guideline by the European Society of Cardiology ([29](#B29)) recommends testing and surveillance of MM patients at baseline and every cycle during the first 6 cycles under proteasome inhibitor treatment based on the risk. The guideline recommends first stratifying MM patients into low, moderate, high, or very high-risk groups based on clinical factors prior to the treatment. Considering genetic variants may improve the risk stratification of these patients.

To our knowledge, this is the first genetic association study of CFZ-CVAE in humans. We found that the variant carriers of missense SNP rs7148 in the *TMSB10* gene were at higher risk for CFZ-CVAE. We also identified an intronic variant rs12471929 which is in perfect LD with rs7148 in EA. Both of these SNPs are also eQTLs for the nearby gene *TRABD2A* in that the variant alleles are associated with higher expression of TRABD2A in the left ventricle heart tissue ([30](#B30)).

Thymosins are a family of small peptides initially identified from the thymus and consist of three groups, alpha-, beta-, and gamma-thymosins, according to their isoelectric points ([31](#B31)). The beta-thymosins (TMSB4, TMSB10, TMSB15), found in the cytoplasm, interact with G-actin and produces a large pool of actin monomers ([32](#B32)). TMSB10 has been reported to function in cytoskeleton organization, cell morphology, proliferation, motility, and interaction with Ras and angiogenesis, cell growth, and apoptosis ([33](#B33)). The two isoforms, TMSB10 and TMSB4, are identified as significant actin monomer sequestering proteins that may regulate actin filament assembly ([34](#B34)). A recent study showed that the actin-binding protein TMSB10 was upregulated in dysfunctional endothelial dysfunction in acute myocardial infarction (AMI), and endothelial dysfunction is considered one of the primary factors in the progression of atherothrombosis in AMI ([35](#B35)). Another study on mice found that TMSB10 can inhibit vascular endothelial growth factor (VEGF) expression by inhibiting the Ras-ERK signaling pathway leading to the suppression of vascular formation, especially in tumor formation ([36](#B36)).

*TRABD2A* gene enables Wnt-protein binding activity and metalloendopeptidase activity ([37](#B37)). A recent genome-wide association study on cardiac troponin T (cTnT) in large cohorts identified a genetic variant rs548487604 near the *TRABD2A* gene to be associated with the elevation of cTnT level ([38](#B38)), which is related to the incidence of cardiovascular diseases, cardiovascular death, and heart failure in a general population ([39](#B39), [40](#B40)).

A previous study on mice proposed a pathway associated with CFZ-CVAE. This study found that treatment with CFZ induced the apoptosis pathway by activation of PP2A (protein phosphatase 2A), which inactivates AMPK*α* and the downstream signaling related to autophagy phosphoinositide 3′-kinase (PI3K)-Akt-endothelial nitric oxide synthase (eNOS) (PI3K/Akt/eNOS) axis. This axis is responsible for myocardial cell growth and survival and plays a vital role the cardiac dysfunction ([41](#B41)). Previous studies have identified a specific subtype of hematopoietic stem cells in peripheral blood called endothelial progenitor cells (EPCs) that express numerous combinations of antigens associated with hematopoietic stem and endothelial cells and play an essential role in neovascularization of ischemic tissue and reversal of endothelial dysfunction ([42](#B42), [43](#B43)), and *in vivo* studies showed that TMSB4 increases EPC migration and decreases EPC apoptosis under serum deprivation *via* the (PI3K/Akt/eNOS) signal transduction pathway ([44](#B44), [45](#B45)), and several studies showed that the telomerase length and telomerase activity of circulating EPCs and decreased in patients with coronary artery disease ([46](#B46), [47](#B47)). In a recent study that included 48 patients with relapsed/refractory MM and received CFZ, the brachial artery flow-mediated dilation (FMD) and 26s proteasome activity were detected to evaluate the endothelial function. This study concluded that patients who received CFZ and with low potential for proteasome activity recovery may suffer from both acute and long-term endothelial dysfunction ([48](#B48)).

Endothelial cell homeostasis depends on the ubiquitin-proteasome system, which induces oxidative stress in the cells and regulates the expression of endothelial nitric oxide synthase ([49](#B49)). The proteasome inhibitor CFZ causes the plasma of cancer cells, cardiomyocytes, and endothelial cells to accumulate with unfolded, dysfunctional proteins, which may lead to impaired vasodilation, excessive oxidative stress, inflammation, cell apoptosis, and autophagy ([50](#B50), [51](#B51)). Other studies have shown that the endothelial dysfunction caused by CFZ's inhibition of proteasome activity may result in CVAE or other endothelial dysfunction-related events like hypertension, heart failure, and coronary artery disease ([51](#B51)–[53](#B53)). In light of the literature, our finding of the genetic variants in the *TMSB10/TRABD2A* locus appears to support the role of endothelial dysfunction in CFZ-CVAE.

Among the other SNPs with a suggestive significance level, five are located on the PDZ domain-containing protein 2 (*PDZD2)* gene, which contains six PDZ domains and shares sequence similarity with pro-interleukin-16 (pro-IL-16). SNPs in the *PDZD2* gene have been associated with heart rate in heart failure patients with reduced ejection fraction ([54](#B54)). *PROM1* has a role in cell differentiation, proliferation, and apoptosis. PR1P is a small peptide derived from the extracellular domain of PROM1-derived peptide and improves cardiac function following ischemia. SQLE encodes squalene epoxidase, which catalyzes the first oxygenation step in sterol biosynthesis and is one of the rate-limiting enzymes in this pathway.

It is important to recognize some limitations of our study. Firstly, the patient population is predominantly European Americans. The sample size of patients of African descent was too small to have enough statistical power for any meaningful discovery. Further investigation is required to explore this phenotype and outcomes in MM patients of African ancestry. Secondly, using WES means we may have missed critical genetic variants outside the coding regions of the genome. Thirdly, using ICD codes to identify CVAE has its limitations. Even though our manual chart review of 10% of patients indicated 100% of CVAE were confirmed, it would have been ideal to review all the charts to confirm CVAE status. Fourthly, due to the small sample size, we had to combine all the CVAEs. Lastly, our study findings need to be replicated in an independent study before these genetic variants can be incorporated into the risk stratification of MM patients.

In summary, in this WES study of MM patients in a real-world clinical setting, we identified a missense variant in the *TMSB10/TRABD2A* locus to be associated with CFZ-CVAE among MM patients. Once validated, this association could provide the basis for a Precision Medicine approach to identify MM patients at high risk for CFZ-CVAE.

## Acknowledgments

An earlier version of this study was presented in part as a poster presentation at the American Society of Clinical Pharmacology and Therapeutics (ASCPT) 2022 Virtual meeting ([55](#B55)).

## Funding Statement

Part of the funding for this study came from the NIH/NHLBI under grant number R01HL151659 and the UF CTSA under award number UL1TR001427. In addition to the Total Cancer Care Protocol, the H Lee Moffitt Cancer Center & Research Institute's Collaborative Data Services Core Facility, Tissue Core Facility, Molecular Genomics Core Facility, and Biostatistics and Bioinformatics Shared Resource were also involved in this research; an NCI designated Comprehensive Cancer Center (P30-CA076292). The WES data delivered by the ORIEN Avatar Project is managed and funded partially by M2Gen®, which received partial funding from third-party partners for data generated in the Avatar Project. This work was also supported by the Pentecost Family Myeloma Research Center (PMRC) at the H. Lee Moffitt Cancer Center & Research Institute.

## Data availability statement

The data presented in the study are deposited in the database of Genotypes and Phenotypes (dbGaP) repository, accession number phs003308.v1.p1.

## Ethics statement

The studies involving human participants were reviewed and approved by the University of Florida Institutional Review Board (IRB202003031). The patients/participants provided their written informed consent to participate in this study.

## Author contributions

MT and YG wrote the manuscript, and RA extracted the data from ORIEN, RCB manually reviewed the charts to validate the CVAE. GD, EMS, and KHS directed sample identification and contributed to the TCC and ORIEN project. MT, GY, and YG performed statistical analyses. YG secured the fund for this study. All authors contributed to the article and approved the submitted version.

## Conflict of interest

MGF: research funding from Medtronic; Advisory board/consulting fees: Abbott, AstraZeneca, Myovant, Takeda, and Zoll. RCB: research funding from Janssen, Karyopharm, BMS, and Abbvie; Advisory board member for Janssen, Karyopharm, BMS, Shattuck labs, GSK, Pfizer, and Abbvie. RFC: Employed at AbbVie and stock shareholder. SMR: Advisory board member for Janssen, Sanofi, Roche Diagnostics, and EUSA pharma.

The remaining authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.

## Publisher's note

All claims expressed in this article are solely those of the authors and do not necessarily represent those of their affiliated organizations, or those of the publisher, the editors and the reviewers. Any product that may be evaluated in this article, or claim that may be made by its manufacturer, is not guaranteed or endorsed by the publisher.

## Supplementary material

The Supplementary Material for this article can be found online at: [https://www.frontiersin.org/articles/10.3389/fcvm.2023.1181806/full#supplementary-material](https://www.frontiersin.org/articles/10.3389/fcvm.2023.1181806/full#supplementary-material)

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

The data presented in the study are deposited in the database of Genotypes and Phenotypes (dbGaP) repository, accession number phs003308.v1.p1.

### Supplementary Materials

### Data Availability Statement

The data presented in the study are deposited in the database of Genotypes and Phenotypes (dbGaP) repository, accession number phs003308.v1.p1.

## References

1. Siegel RL, Miller KD, Fuchs HE, Jemal A. Cancer statistics, 2022. CA Cancer J Clin. (2022) 72:7–33. 10.3322/caac.21708  [DOI](https://doi.org/10.3322/caac.21708) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35020204/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=CA%20Cancer%20J%20Clin&title=Cancer%20statistics,%202022&author=RL%20Siegel&author=KD%20Miller&author=HE%20Fuchs&author=A%20Jemal&volume=72&publication_year=2022&pages=7-33&pmid=35020204&doi=10.3322/caac.21708&)

2. Gandolfi S, Laubach JP, Hideshima T, Chauhan D, Anderson KC, Richardson PG. The proteasome and proteasome inhibitors in multiple myeloma. Cancer Metastasis Rev. (2017) 36:561–84. 10.1007/s10555-017-9707-8  [DOI](https://doi.org/10.1007/s10555-017-9707-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29196868/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Metastasis%20Rev&title=The%20proteasome%20and%20proteasome%20inhibitors%20in%20multiple%20myeloma&author=S%20Gandolfi&author=JP%20Laubach&author=T%20Hideshima&author=D%20Chauhan&author=KC%20Anderson&volume=36&publication_year=2017&pages=561-84&pmid=29196868&doi=10.1007/s10555-017-9707-8&)

3. Teicher BA, Tomaszewski JE. Competitive landscape report. Biochem Pharmacol. (2015) 96:1–9. 10.1016/j.bcp.2015.04.008  [DOI](https://doi.org/10.1016/j.bcp.2015.04.008) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25935605/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20Pharmacol&title=Competitive%20landscape%20report&author=BA%20Teicher&author=JE%20Tomaszewski&volume=96&publication_year=2015&pages=1-9&pmid=25935605&doi=10.1016/j.bcp.2015.04.008&)

4. Hasinoff BB, Patel D, Wu X. Molecular mechanisms of the cardiotoxicity of the proteasomal-targeted drugs bortezomib and carfilzomib. Cardiovasc Toxicol. (2017) 17:237–50. 10.1007/s12012-016-9378-7  [DOI](https://doi.org/10.1007/s12012-016-9378-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27388042/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cardiovasc%20Toxicol&title=Molecular%20mechanisms%20of%20the%20cardiotoxicity%20of%20the%20proteasomal-targeted%20drugs%20bortezomib%20and%20carfilzomib&author=BB%20Hasinoff&author=D%20Patel&author=X%20Wu&volume=17&publication_year=2017&pages=237-50&pmid=27388042&doi=10.1007/s12012-016-9378-7&)

5. Dimopoulos MA, Goldschmidt H, Niesvizky R, Joshua D, Chng WJ, Oriol A, et al. Carfilzomib or bortezomib in relapsed or refractory multiple myeloma (ENDEAVOR): an interim overall survival analysis of an open-label, randomised, phase 3 trial. Lancet Oncol. (2017) 18:1327–37. 10.1016/S1470-2045(17)30578-8  [DOI](https://doi.org/10.1016/S1470-2045(17)30578-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28843768/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet%20Oncol&title=Carfilzomib%20or%20bortezomib%20in%20relapsed%20or%20refractory%20multiple%20myeloma%20(ENDEAVOR):%20an%20interim%20overall%20survival%20analysis%20of%20an%20open-label,%20randomised,%20phase%203%20trial&author=MA%20Dimopoulos&author=H%20Goldschmidt&author=R%20Niesvizky&author=D%20Joshua&author=WJ%20Chng&volume=18&publication_year=2017&pages=1327-37&pmid=28843768&doi=10.1016/S1470-2045(17)30578-8&)

6. Callander NS, Baljevic M, Adekola K, Anderson LD, Campagnaro E, Castillo JJ, et al. NCCN Guidelines® Insights: Multiple Myeloma, Version 3.2022. J Natl Compr Canc Netw. (2022) 20(1):8–19. 10.6004/jnccn.2022.0002  [DOI](https://doi.org/10.6004/jnccn.2022.0002) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34991075/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Natl%20Compr%20Canc%20Netw&title=NCCN%20Guidelines%C2%AE%20Insights:%20Multiple%20Myeloma,%20Version%203.2022&author=NS%20Callander&author=M%20Baljevic&author=K%20Adekola&author=LD%20Anderson&author=E%20Campagnaro&volume=20&issue=1&publication_year=2022&pages=8-19&pmid=34991075&doi=10.6004/jnccn.2022.0002&)

7. Siegel D, Martin T, Nooka A, Harvey RD, Vij R, Niesvizky R, et al. Integrated safety profile of single-agent carfilzomib: experience from 526 patients enrolled in 4 phase II clinical studies. Haematologica. (2013) 98(11):1753–61. 10.3324/haematol.2013.089334  [DOI](https://doi.org/10.3324/haematol.2013.089334) | [PMC free article](/articles/PMC3815177/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23935022/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Haematologica&title=Integrated%20safety%20profile%20of%20single-agent%20carfilzomib:%20experience%20from%20526%20patients%20enrolled%20in%204%20phase%20II%20clinical%20studies&author=D%20Siegel&author=T%20Martin&author=A%20Nooka&author=RD%20Harvey&author=R%20Vij&volume=98&issue=11&publication_year=2013&pages=1753-61&pmid=23935022&doi=10.3324/haematol.2013.089334&)

8. Kistler KD, Kalman J, Sahni G, Murphy B, Werther W, Rajangam K, et al. Incidence and risk of cardiac events in patients with previously treated multiple myeloma versus matched patients without multiple myeloma: an observational, retrospective, cohort study. Clin Lymphoma Myeloma Leuk. (2017) 17:89–96.e3. 10.1016/j.clml.2016.11.009  [DOI](https://doi.org/10.1016/j.clml.2016.11.009) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28025038/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Lymphoma%20Myeloma%20Leuk&title=Incidence%20and%20risk%20of%20cardiac%20events%20in%20patients%20with%20previously%20treated%20multiple%20myeloma%20versus%20matched%20patients%20without%20multiple%20myeloma:%20an%20observational,%20retrospective,%20cohort%20study&author=KD%20Kistler&author=J%20Kalman&author=G%20Sahni&author=B%20Murphy&author=W%20Werther&volume=17&publication_year=2017&pages=89-96.e3&pmid=28025038&doi=10.1016/j.clml.2016.11.009&)

9. Atrash S, Tullos A, Panozzo S, Bhutani M, Van Rhee F, Barlogie B, et al. Cardiac complications in relapsed and refractory multiple myeloma patients treated with carfilzomib. Blood Cancer J. (2015) 5:e272. 10.1038/bcj.2014.93  [DOI](https://doi.org/10.1038/bcj.2014.93) | [PMC free article](/articles/PMC4314456/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25594159/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood%20Cancer%20J&title=Cardiac%20complications%20in%20relapsed%20and%20refractory%20multiple%20myeloma%20patients%20treated%20with%20carfilzomib&author=S%20Atrash&author=A%20Tullos&author=S%20Panozzo&author=M%20Bhutani&author=F%20Van%20Rhee&volume=5&publication_year=2015&pages=e272&pmid=25594159&doi=10.1038/bcj.2014.93&)

10. Cornell RF, Ky B, Weiss BM, Dahm CN, Gupta DK, Du L, et al. Prospective study of cardiac events during proteasome inhibitor therapy for relapsed multiple myeloma. J Clin Oncol. (2019) 37:1946–55. 10.1200/JCO.19.00231  [DOI](https://doi.org/10.1200/JCO.19.00231) | [PMC free article](/articles/PMC9798911/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31188726/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Prospective%20study%20of%20cardiac%20events%20during%20proteasome%20inhibitor%20therapy%20for%20relapsed%20multiple%20myeloma&author=RF%20Cornell&author=B%20Ky&author=BM%20Weiss&author=CN%20Dahm&author=DK%20Gupta&volume=37&publication_year=2019&pages=1946-55&pmid=31188726&doi=10.1200/JCO.19.00231&)

11. Yu AF, Yadav NU, Lung BY, Eaton AA, Thaler HT, Hudis CA, et al. Trastuzumab interruption and treatment-induced cardiotoxicity in early HER2-positive breast cancer. Breast Cancer Res Treat. (2015) 149:489–95. 10.1007/s10549-014-3253-7  [DOI](https://doi.org/10.1007/s10549-014-3253-7) | [PMC free article](/articles/PMC4970316/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25552363/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Breast%20Cancer%20Res%20Treat&title=Trastuzumab%20interruption%20and%20treatment-induced%20cardiotoxicity%20in%20early%20HER2-positive%20breast%20cancer&author=AF%20Yu&author=NU%20Yadav&author=BY%20Lung&author=AA%20Eaton&author=HT%20Thaler&volume=149&publication_year=2015&pages=489-95&pmid=25552363&doi=10.1007/s10549-014-3253-7&)

12. Koutsoukis A, Ntalianis A, Repasos E, Kastritis E, Dimopoulos MA, Paraskevaidis I. Cardio-oncology: a focus on cardiotoxicity. European Cardiology Review. (2018) 13:64–9. 10.15420/ecr.2017:17:2  [DOI](https://doi.org/10.15420/ecr.2017:17:2) | [PMC free article](/articles/PMC6159462/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30310475/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=European%20Cardiology%20Review&title=Cardio-oncology:%20a%20focus%20on%20cardiotoxicity&author=A%20Koutsoukis&author=A%20Ntalianis&author=E%20Repasos&author=E%20Kastritis&author=MA%20Dimopoulos&volume=13&publication_year=2018&pages=64-9&pmid=30310475&doi=10.15420/ecr.2017:17:2&)

13. Henry ML, Niu J, Zhang N, Giordano SH, Chavez-MacGregor M. Cardiotoxicity and cardiac monitoring among chemotherapy-treated breast cancer patients. JACC Cardiovasc Imaging. (2018) 11:1084–93. 10.1016/j.jcmg.2018.06.005  [DOI](https://doi.org/10.1016/j.jcmg.2018.06.005) | [PMC free article](/articles/PMC6149535/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30092967/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JACC%20Cardiovasc%20Imaging&title=Cardiotoxicity%20and%20cardiac%20monitoring%20among%20chemotherapy-treated%20breast%20cancer%20patients&author=ML%20Henry&author=J%20Niu&author=N%20Zhang&author=SH%20Giordano&author=M%20Chavez-MacGregor&volume=11&publication_year=2018&pages=1084-93&pmid=30092967&doi=10.1016/j.jcmg.2018.06.005&)

14. Shuldiner AR, O’Connell JR, Bliden KP, Gandhi A, Ryan K, Horenstein RB, et al. Association of cytochrome P450 2C19 genotype with the antiplatelet effect and clinical efficacy of clopidogrel therapy. JAMA. (2009) 302:849–57. 10.1001/jama.2009.1232  [DOI](https://doi.org/10.1001/jama.2009.1232) | [PMC free article](/articles/PMC3641569/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19706858/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA&title=Association%20of%20cytochrome%20P450%202C19%20genotype%20with%20the%20antiplatelet%20effect%20and%20clinical%20efficacy%20of%20clopidogrel%20therapy&author=AR%20Shuldiner&author=JR%20O%E2%80%99Connell&author=KP%20Bliden&author=A%20Gandhi&author=K%20Ryan&volume=302&publication_year=2009&pages=849-57&pmid=19706858&doi=10.1001/jama.2009.1232&)

15. Krynetski EY, Tai HL, Yates CR, Fessing MY, Loennechen T, Schuetz JD, et al. Genetic polymorphism of thiopurine S-methyltransferase: clinical importance and molecular mechanisms. Pharmacogenetics. (1996) 6:279–90. 10.1097/00008571-199608000-00001  [DOI](https://doi.org/10.1097/00008571-199608000-00001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8873214/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Genetic%20polymorphism%20of%20thiopurine%20S-methyltransferase:%20clinical%20importance%20and%20molecular%20mechanisms&author=EY%20Krynetski&author=HL%20Tai&author=CR%20Yates&author=MY%20Fessing&author=T%20Loennechen&volume=6&publication_year=1996&pages=279-90&pmid=8873214&doi=10.1097/00008571-199608000-00001&)

16. Fenstermacher DA, Wenham RM, Rollison DE, Dalton WS. Implementing personalized medicine in a cancer center. Cancer J. (2011) 17:528–36. 10.1097/PPO.0b013e318238216e  [DOI](https://doi.org/10.1097/PPO.0b013e318238216e) | [PMC free article](/articles/PMC3240816/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22157297/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20J&title=Implementing%20personalized%20medicine%20in%20a%20cancer%20center&author=DA%20Fenstermacher&author=RM%20Wenham&author=DE%20Rollison&author=WS%20Dalton&volume=17&publication_year=2011&pages=528-36&pmid=22157297&doi=10.1097/PPO.0b013e318238216e&)

17. Dalton WS, Sullivan D, Ecsedy J, Caligiuri MA. Patient enrichment for precision-based cancer clinical trials: using prospective cohort surveillance as an approach to improve clinical trials. Clin Pharmacol Ther. (2018) 104:23–6. 10.1002/cpt.1051  [DOI](https://doi.org/10.1002/cpt.1051) | [PMC free article](/articles/PMC6032821/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29570791/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Patient%20enrichment%20for%20precision-based%20cancer%20clinical%20trials:%20using%20prospective%20cohort%20surveillance%20as%20an%20approach%20to%20improve%20clinical%20trials&author=WS%20Dalton&author=D%20Sullivan&author=J%20Ecsedy&author=MA%20Caligiuri&volume=104&publication_year=2018&pages=23-6&pmid=29570791&doi=10.1002/cpt.1051&)

18. Li H. A statistical framework for SNP calling, mutation discovery, association mapping and population genetical parameter estimation from sequencing data. Bioinformatics (2011) 27:2987–93. 10.1093/bioinformatics/btr509  [DOI](https://doi.org/10.1093/bioinformatics/btr509) | [PMC free article](/articles/PMC3198575/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21903627/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=A%20statistical%20framework%20for%20SNP%20calling,%20mutation%20discovery,%20association%20mapping%20and%20population%20genetical%20parameter%20estimation%20from%20sequencing%20data&author=H%20Li&volume=27&publication_year=2011&pages=2987-93&pmid=21903627&doi=10.1093/bioinformatics/btr509&)

19. Depristo MA, Banks E, Poplin R, Garimella K V, Maguire JR, Hartl C, et al. A framework for variation discovery and genotyping using next-generation DNA sequencing data. Nat Genet. (2011) 43:491–501. 10.1038/ng.806  [DOI](https://doi.org/10.1038/ng.806) | [PMC free article](/articles/PMC3083463/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21478889/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=A%20framework%20for%20variation%20discovery%20and%20genotyping%20using%20next-generation%20DNA%20sequencing%20data&author=MA%20Depristo&author=E%20Banks&author=R%20Poplin&author=V%20Garimella%20K&author=JR%20Maguire&volume=43&publication_year=2011&pages=491-501&pmid=21478889&doi=10.1038/ng.806&)

20. Van der Auwera GA, Carneiro MO, Hartl C. Poplin R, Del Angel G, Levy-Moonshine A, et al. From FastQ data to high confidence variant calls: the Genome Analysis Toolkit best practices pipeline. Curr Protoc Bioinformatics. (2013) 43(1110):11.10.1–11.10.33. 10.1002/0471250953.bi1110s43  [DOI](https://doi.org/10.1002/0471250953.bi1110s43) | [PMC free article](/articles/PMC4243306/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25431634/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Protoc%20Bioinformatics&title=From%20FastQ%20data%20to%20high%20confidence%20variant%20calls:%20the%20Genome%20Analysis%20Toolkit%20best%20practices%20pipeline&author=GA%20Van%20der%20Auwera&author=MO%20Carneiro&author=C%20Hartl&author=R%20Poplin&author=G%20Del%20Angel&volume=43&issue=1110&publication_year=2013&pages=11.10.1-11.10.33&pmid=25431634&doi=10.1002/0471250953.bi1110s43&)

21. Purcell S, Neale B, Todd-Brown K, Thomas L, Ferreira MAR, Bender D, et al. PLINK: a tool set for whole-genome association and population-based linkage analyses. Am J Hum Genet. (2007) 81:559–75. 10.1086/519795  [DOI](https://doi.org/10.1086/519795) | [PMC free article](/articles/PMC1950838/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17701901/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=PLINK:%20a%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses&author=S%20Purcell&author=B%20Neale&author=K%20Todd-Brown&author=L%20Thomas&author=MAR%20Ferreira&volume=81&publication_year=2007&pages=559-75&pmid=17701901&doi=10.1086/519795&)

22. Fadista J, Manning AK, Florez JC, Groop L. The (in)famous GWAS P-value threshold revisited and updated for low-frequency variants. Eur J Hum Genet. (2016) 24:1202–5. 10.1038/ejhg.2015.269  [DOI](https://doi.org/10.1038/ejhg.2015.269) | [PMC free article](/articles/PMC4970684/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26733288/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Hum%20Genet&title=The%20(in)famous%20GWAS%20P-value%20threshold%20revisited%20and%20updated%20for%20low-frequency%20variants&author=J%20Fadista&author=AK%20Manning&author=JC%20Florez&author=L%20Groop&volume=24&publication_year=2016&pages=1202-5&pmid=26733288&doi=10.1038/ejhg.2015.269&)

23. Watanabe K, Taskesen E, Van Bochoven A, Posthuma D. Functional mapping and annotation of genetic associations with FUMA. Nat Commun. (2017) 8:1826. 10.1038/s41467-017-01261-5  [DOI](https://doi.org/10.1038/s41467-017-01261-5) | [PMC free article](/articles/PMC5705698/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29184056/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&title=Functional%20mapping%20and%20annotation%20of%20genetic%20associations%20with%20FUMA&author=K%20Watanabe&author=E%20Taskesen&author=A%20Van%20Bochoven&author=D%20Posthuma&volume=8&publication_year=2017&pages=1826&pmid=29184056&doi=10.1038/s41467-017-01261-5&)

24. Wang K, Li M, Hakonarson H. ANNOVAR: functional annotation of genetic variants from high-throughput sequencing data. Nucleic Acids Res. (2010) 38:e164. 10.1093/NAR/GKQ603  [DOI](https://doi.org/10.1093/NAR/GKQ603) | [PMC free article](/articles/PMC2938201/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20601685/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=ANNOVAR:%20functional%20annotation%20of%20genetic%20variants%20from%20high-throughput%20sequencing%20data&author=K%20Wang&author=M%20Li&author=H%20Hakonarson&volume=38&publication_year=2010&pages=e164&pmid=20601685&doi=10.1093/NAR/GKQ603&)

25. Lee S, Abecasis GR, Boehnke M, Lin X. Rare-variant association analysis: study designs and statistical tests. Am J Hum Genet. (2014) 95:5–23. 10.1016/j.ajhg.2014.06.009  [DOI](https://doi.org/10.1016/j.ajhg.2014.06.009) | [PMC free article](/articles/PMC4085641/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24995866/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Rare-variant%20association%20analysis:%20study%20designs%20and%20statistical%20tests&author=S%20Lee&author=GR%20Abecasis&author=M%20Boehnke&author=X%20Lin&volume=95&publication_year=2014&pages=5-23&pmid=24995866&doi=10.1016/j.ajhg.2014.06.009&)

26. Wu MC, Lee S, Cai T, Li Y, Boehnke M, Lin X. Rare-variant association testing for sequencing data with the sequence kernel association test. Am J Hum Genet. (2011) 89:82–93. 10.1016/j.ajhg.2011.05.029  [DOI](https://doi.org/10.1016/j.ajhg.2011.05.029) | [PMC free article](/articles/PMC3135811/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21737059/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Rare-variant%20association%20testing%20for%20sequencing%20data%20with%20the%20sequence%20kernel%20association%20test&author=MC%20Wu&author=S%20Lee&author=T%20Cai&author=Y%20Li&author=M%20Boehnke&volume=89&publication_year=2011&pages=82-93&pmid=21737059&doi=10.1016/j.ajhg.2011.05.029&)

27. Krämer A, Green J, Pollard J Jr, Tugendreich S. Causal analysis approaches in Ingenuity Pathway Analysis. Bioinformatics. (2014) 30(4):523–30. 10.1093/bioinformatics/btt703  [DOI](https://doi.org/10.1093/bioinformatics/btt703) | [PMC free article](/articles/PMC3928520/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24336805/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=Causal%20analysis%20approaches%20in%20Ingenuity%20Pathway%20Analysis&author=A%20Kr%C3%A4mer&author=J%20Green&author=J%20Jr%20Pollard&author=S%20Tugendreich&volume=30&issue=4&publication_year=2014&pages=523-30&pmid=24336805&doi=10.1093/bioinformatics/btt703&)

28. Burton PR, Clayton DG, Cardon LR, Craddock N, Deloukas P, Duncanson A, et al. Genome-wide association study of 14,000 cases of seven common diseases and 3,000 shared controls. Nature. (2007) 447:661–78. 10.1038/nature05911  [DOI](https://doi.org/10.1038/nature05911) | [PMC free article](/articles/PMC2719288/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17554300/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Genome-wide%20association%20study%20of%2014,000%20cases%20of%20seven%20common%20diseases%20and%203,000%20shared%20controls&author=PR%20Burton&author=DG%20Clayton&author=LR%20Cardon&author=N%20Craddock&author=P%20Deloukas&volume=447&publication_year=2007&pages=661-78&pmid=17554300&doi=10.1038/nature05911&)

29. Lyon AR, López-Fernández T, Couch LS, Asteggiano R, Aznar MC, Bergler-Klein J, et al. 2022 ESC guidelines on cardio-oncology developed in collaboration with the European hematology association (EHA), the European society for therapeutic radiology and oncology (ESTRO) and the international cardio-oncology society (IC-OS). Eur Heart J. (2022) 43:4229–361. 10.1093/eurheartj/ehac244  [DOI](https://doi.org/10.1093/eurheartj/ehac244) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36017568/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20Heart%20J&title=2022%20ESC%20guidelines%20on%20cardio-oncology%20developed%20in%20collaboration%20with%20the%20European%20hematology%20association%20(EHA),%20the%20European%20society%20for%20therapeutic%20radiology%20and%20oncology%20(ESTRO)%20and%20the%20international%20cardio-oncology%20society%20(IC-OS)&author=AR%20Lyon&author=T%20L%C3%B3pez-Fern%C3%A1ndez&author=LS%20Couch&author=R%20Asteggiano&author=MC%20Aznar&volume=43&publication_year=2022&pages=4229-361&pmid=36017568&doi=10.1093/eurheartj/ehac244&)

30. Carithers LJ, Ardlie K, Barcus M, Branton PA, Britton A, Buia SA, et al. A novel approach to high-quality postmortem tissue procurement: the GTEx project. Biopreserv Biobank. (2015) 13:311–7. 10.1089/bio.2015.0032  [DOI](https://doi.org/10.1089/bio.2015.0032) | [PMC free article](/articles/PMC4675181/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26484571/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biopreserv%20Biobank&title=A%20novel%20approach%20to%20high-quality%20postmortem%20tissue%20procurement:%20the%20GTEx%20project&author=LJ%20Carithers&author=K%20Ardlie&author=M%20Barcus&author=PA%20Branton&author=A%20Britton&volume=13&publication_year=2015&pages=311-7&pmid=26484571&doi=10.1089/bio.2015.0032&)

31. Santelli G, Califano D, Chiappetta G, Vento MT, Bartoli PC, Zullo F, et al. Thymosin beta-10 gene overexpression is a general event in human carcinogenesis. Am J Pathol. (1999) 155:799–804. 10.1016/s0002-9440(10)65178-4  [DOI](https://doi.org/10.1016/s0002-9440(10)65178-4) | [PMC free article](/articles/PMC1866886/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10487837/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Pathol&title=Thymosin%20beta-10%20gene%20overexpression%20is%20a%20general%20event%20in%20human%20carcinogenesis&author=G%20Santelli&author=D%20Califano&author=G%20Chiappetta&author=MT%20Vento&author=PC%20Bartoli&volume=155&publication_year=1999&pages=799-804&pmid=10487837&doi=10.1016/s0002-9440(10)65178-4&)

32. Chen C, Li M, Yang H, Chai H, Fisher W, Yao Q. Roles of thymosins in cancers and other organ systems. World J Surg. (2005) 29:264–70. 10.1007/S00268-004-7817-2  [DOI](https://doi.org/10.1007/S00268-004-7817-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15706436/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=World%20J%20Surg&title=Roles%20of%20thymosins%20in%20cancers%20and%20other%20organ%20systems&author=C%20Chen&author=M%20Li&author=H%20Yang&author=H%20Chai&author=W%20Fisher&volume=29&publication_year=2005&pages=264-70&pmid=15706436&doi=10.1007/S00268-004-7817-2&)

33. Sribenja S, Li M, Wongkham S, Wongkham C, Yao Q, Chen C. Cancer investigation advances in thymosin β10 research: differential expression, molecular mechanisms, and clinical implications in cancer and other conditions advances in thymosin β10 research: differential expression, molecular mechanisms, and clinical implications in cancer and other conditions. Cancer Invest. (2009) 27:1016–22. 10.3109/07357900902849640  [DOI](https://doi.org/10.3109/07357900902849640) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19909017/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Invest&title=Cancer%20investigation%20advances%20in%20thymosin%20%CE%B210%20research:%20differential%20expression,%20molecular%20mechanisms,%20and%20clinical%20implications%20in%20cancer%20and%20other%20conditions%20advances%20in%20thymosin%20%CE%B210%20research:%20differential%20expression,%20molecular%20mechanisms,%20and%20clinical%20implications%20in%20cancer%20and%20other%20conditions.&author=S%20Sribenja&author=M%20Li&author=S%20Wongkham&author=C%20Wongkham&author=Q%20Yao&volume=27&publication_year=2009&pages=1016-22&pmid=19909017&doi=10.3109/07357900902849640&)

34. Yu FX, Lin SC, Morrison-Bogorad M, Yin HL. Effects of thymosin beta 4 and thymosin beta 10 on actin structures in living cells. Cell Motil Cytoskeleton. (1994) 27:13–25. 10.1002/cm.970270103  [DOI](https://doi.org/10.1002/cm.970270103) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8194107/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Motil%20Cytoskeleton&title=Effects%20of%20thymosin%20beta%204%20and%20thymosin%20beta%2010%20on%20actin%20structures%20in%20living%20cells&author=FX%20Yu&author=SC%20Lin&author=M%20Morrison-Bogorad&author=HL%20Yin&volume=27&publication_year=1994&pages=13-25&pmid=8194107&doi=10.1002/cm.970270103&)

35. Nukala SB, Regazzoni L, Aldini G, Zodda E, Tura-Ceide O, Mills NL, et al. Differentially expressed proteins in primary endothelial cells derived from patients with acute myocardial infarction. Hypertension. (2019) 74:947–56. 10.1161/HYPERTENSIONAHA.119.13472  [DOI](https://doi.org/10.1161/HYPERTENSIONAHA.119.13472) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31446798/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hypertension&title=Differentially%20expressed%20proteins%20in%20primary%20endothelial%20cells%20derived%20from%20patients%20with%20acute%20myocardial%20infarction&author=SB%20Nukala&author=L%20Regazzoni&author=G%20Aldini&author=E%20Zodda&author=O%20Tura-Ceide&volume=74&publication_year=2019&pages=947-56&pmid=31446798&doi=10.1161/HYPERTENSIONAHA.119.13472&)

36. Lee SH, Son MJ, Oh SH, Rho SB, Park K, Kim YJ, Park MS, Lee JH. Thymosin β10 inhibits angiogenesis and tumor growth by interfering with ras function. Cancer Res (2005) 65:137–47. 10.1158/0008-5472.137.65.1  [DOI](https://doi.org/10.1158/0008-5472.137.65.1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15665289/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=Thymosin%20%CE%B210%20inhibits%20angiogenesis%20and%20tumor%20growth%20by%20interfering%20with%20ras%20function&author=SH%20Lee&author=MJ%20Son&author=SH%20Oh&author=SB%20Rho&author=K%20Park&volume=65&publication_year=2005&pages=137-47&pmid=15665289&doi=10.1158/0008-5472.137.65.1&)

37. Liu M, Dong J, Ouyang J, Zhao L, Liang G, Shang H. Metalloprotease TRABD2A restriction of HIV-1 production in monocyte-derived dendritic cells. AIDS Res Hum Retroviruses. (2019) 35:887–9. 10.1089/AID.2019.0140  [DOI](https://doi.org/10.1089/AID.2019.0140) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31282173/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=AIDS%20Res%20Hum%20Retroviruses&title=Metalloprotease%20TRABD2A%20restriction%20of%20HIV-1%20production%20in%20monocyte-derived%20dendritic%20cells&author=M%20Liu&author=J%20Dong&author=J%20Ouyang&author=L%20Zhao&author=G%20Liang&volume=35&publication_year=2019&pages=887-9&pmid=31282173&doi=10.1089/AID.2019.0140&)

38. Nasu T, Satoh M, Hachiya T, Sutoh Y, Ohmomo H, Hitomi S, et al. A genome-wide association study for highly sensitive cardiac troponin T levels identified a novel genetic variation near a RBAK-ZNF890P locus in the Japanese general population. Int J Cardiol. (2021) 329:186–91. 10.1016/j.ijcard.2020.12.019  [DOI](https://doi.org/10.1016/j.ijcard.2020.12.019) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33321125/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Cardiol&title=A%20genome-wide%20association%20study%20for%20highly%20sensitive%20cardiac%20troponin%20T%20levels%20identified%20a%20novel%20genetic%20variation%20near%20a%20RBAK-ZNF890P%20locus%20in%20the%20Japanese%20general%20population&author=T%20Nasu&author=M%20Satoh&author=T%20Hachiya&author=Y%20Sutoh&author=H%20Ohmomo&volume=329&publication_year=2021&pages=186-91&pmid=33321125&doi=10.1016/j.ijcard.2020.12.019&)

39. Welsh P, Preiss D, Hayward C, Shah AS, McAllister D, Briggs A, et al. Cardiac troponin T and troponin I in the general population. Circulation. (2019) 139:2754–64. 10.1161/CIRCULATIONAHA.118.038529  [DOI](https://doi.org/10.1161/CIRCULATIONAHA.118.038529) | [PMC free article](/articles/PMC6571179/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31014085/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation&title=Cardiac%20troponin%20T%20and%20troponin%20I%20in%20the%20general%20population&author=P%20Welsh&author=D%20Preiss&author=C%20Hayward&author=AS%20Shah&author=D%20McAllister&volume=139&publication_year=2019&pages=2754-64&pmid=31014085&doi=10.1161/CIRCULATIONAHA.118.038529&)

40. Saunders JT, Nambi V, de Lemos JA, Chambless LE, Virani SS, Boerwinkle E, et al. Cardiac troponin T measured by a highly sensitive assay predicts coronary heart disease, heart failure, and mortality in the atherosclerosis risk in communities study. Circulation. (2011) 123:1367–76. 10.1161/CIRCULATIONAHA.110.005264  [DOI](https://doi.org/10.1161/CIRCULATIONAHA.110.005264) | [PMC free article](/articles/PMC3072024/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21422391/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation&title=Cardiac%20troponin%20T%20measured%20by%20a%20highly%20sensitive%20assay%20predicts%20coronary%20heart%20disease,%20heart%20failure,%20and%20mortality%20in%20the%20atherosclerosis%20risk%20in%20communities%20study&author=JT%20Saunders&author=V%20Nambi&author=JA%20de%20Lemos&author=LE%20Chambless&author=SS%20Virani&volume=123&publication_year=2011&pages=1367-76&pmid=21422391&doi=10.1161/CIRCULATIONAHA.110.005264&)

41. Efentakis P, Kremastiotis G, Varela A, Nikolaou PE, Papanagnou ED, Davos CH, et al. Molecular mechanisms of carfilzomib-induced cardiotoxicity in mice and the emerging cardioprotective role of metformin. Blood. (2019) 133:710–23. 10.1182/blood-2018-06-858415  [DOI](https://doi.org/10.1182/blood-2018-06-858415) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30482794/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Molecular%20mechanisms%20of%20carfilzomib-induced%20cardiotoxicity%20in%20mice%20and%20the%20emerging%20cardioprotective%20role%20of%20metformin&author=P%20Efentakis&author=G%20Kremastiotis&author=A%20Varela&author=PE%20Nikolaou&author=ED%20Papanagnou&volume=133&publication_year=2019&pages=710-23&pmid=30482794&doi=10.1182/blood-2018-06-858415&)

42. Shi Q, Rafii S, Wu Hong-De M, Wijelath ES, Yu C, Ishida A, et al. Evidence for circulating bone marrow-derived endothelial cells. Blood. (1998) 92:362–7. 10.1182/blood.v92.2.362  [DOI](https://doi.org/10.1182/blood.v92.2.362) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9657732/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Evidence%20for%20circulating%20bone%20marrow-derived%20endothelial%20cells&author=Q%20Shi&author=S%20Rafii&author=M%20Wu%20Hong-De&author=ES%20Wijelath&author=C%20Yu&volume=92&publication_year=1998&pages=362-7&pmid=9657732&doi=10.1182/blood.v92.2.362&)

43. Asahara T, Murohara T, Sullivan A, Silver M, Van Der Zee R, Li T, et al. Isolation of putative progenitor endothelial cells for angiogenesis. Science. (1997) 275:964–7. 10.1126/science.275.5302.964  [DOI](https://doi.org/10.1126/science.275.5302.964) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9020076/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Isolation%20of%20putative%20progenitor%20endothelial%20cells%20for%20angiogenesis&author=T%20Asahara&author=T%20Murohara&author=A%20Sullivan&author=M%20Silver&author=R%20Van%20Der%20Zee&volume=275&publication_year=1997&pages=964-7&pmid=9020076&doi=10.1126/science.275.5302.964&)

44. Zhao Y, Qiu F, Xu S, Yu L, Fu G. Thymosin β4 activates integrin-linked kinase and decreases endothelial progenitor cells apoptosis under serum deprivation. J Cell Physiol. (2011) 226:2798–806. 10.1002/jcp.22624  [DOI](https://doi.org/10.1002/jcp.22624) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21935929/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Cell%20Physiol&title=Thymosin%20%CE%B24%20activates%20integrin-linked%20kinase%20and%20decreases%20endothelial%20progenitor%20cells%20apoptosis%20under%20serum%20deprivation&author=Y%20Zhao&author=F%20Qiu&author=S%20Xu&author=L%20Yu&author=G%20Fu&volume=226&publication_year=2011&pages=2798-806&pmid=21935929&doi=10.1002/jcp.22624&)

45. Qiu FY, Song XX, Zheng H, Zhao YB, Fu GS. Thymosin β4 induces endothelial progenitor cell migration via PI3K/akt/eNOS signal transduction pathway. J Cardiovasc Pharmacol. (2009) 53:209–14. 10.1097/FJC.0b013e318199f326  [DOI](https://doi.org/10.1097/FJC.0b013e318199f326) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19247195/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Cardiovasc%20Pharmacol&title=Thymosin%20%CE%B24%20induces%20endothelial%20progenitor%20cell%20migration%20via%20PI3K/akt/eNOS%20signal%20transduction%20pathway&author=FY%20Qiu&author=XX%20Song&author=H%20Zheng&author=YB%20Zhao&author=GS%20Fu&volume=53&publication_year=2009&pages=209-14&pmid=19247195&doi=10.1097/FJC.0b013e318199f326&)

46. Murasawa S, Llevadot J, Silver M, Isner JM, Losordo DW, Asahara T. Constitutive human telomerase reverse transcriptase expression enhances regenerative properties of endothelial progenitor cells. Circulation. (2002) 106:1133–9. 10.1161/01.CIR.0000027584.85865.B4  [DOI](https://doi.org/10.1161/01.CIR.0000027584.85865.B4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12196341/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation&title=Constitutive%20human%20telomerase%20reverse%20transcriptase%20expression%20enhances%20regenerative%20properties%20of%20endothelial%20progenitor%20cells&author=S%20Murasawa&author=J%20Llevadot&author=M%20Silver&author=JM%20Isner&author=DW%20Losordo&volume=106&publication_year=2002&pages=1133-9&pmid=12196341&doi=10.1161/01.CIR.0000027584.85865.B4&)

47. Vasa M, Fichtlscherer S, Aicher A, Adler K, Urbich C, Martin H, et al. Number and migratory activity of circulating endothelial progenitor cells inversely correlate with risk factors for coronary artery disease. Circ Res. (2001) 89:E1–7. 10.1161/hh1301.093953  [DOI](https://doi.org/10.1161/hh1301.093953) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11440984/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circ%20Res&title=Number%20and%20migratory%20activity%20of%20circulating%20endothelial%20progenitor%20cells%20inversely%20correlate%20with%20risk%20factors%20for%20coronary%20artery%20disease&author=M%20Vasa&author=S%20Fichtlscherer&author=A%20Aicher&author=K%20Adler&author=C%20Urbich&volume=89&publication_year=2001&pages=E1-7&pmid=11440984&doi=10.1161/hh1301.093953&)

48. Kastritis E, Laina A, Georgiopoulos G, Gavriatopoulou M, Papanagnou E-D, Eleutherakis-Papaiakovou E, et al. Carfilzomib-induced endothelial dysfunction, recovery of proteasome activity, and prediction of cardiovascular complications: a prospective study. Leukemia. (2021) 35:1418–27. 10.1038/s41375-021-01141-4  [DOI](https://doi.org/10.1038/s41375-021-01141-4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33589757/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Leukemia&title=Carfilzomib-induced%20endothelial%20dysfunction,%20recovery%20of%20proteasome%20activity,%20and%20prediction%20of%20cardiovascular%20complications:%20a%20prospective%20study&author=E%20Kastritis&author=A%20Laina&author=G%20Georgiopoulos&author=M%20Gavriatopoulou&author=E-D%20Papanagnou&volume=35&publication_year=2021&pages=1418-27&pmid=33589757&doi=10.1038/s41375-021-01141-4&)

49. de Carvalho JE R, Verwoert MT, Vogels IMC, Reits EA, van Noorden CJF, Klaassen I, et al. Involvement of the ubiquitin-proteasome system in the expression of extracellular matrix genes in retinal pigment epithelial cells. Biochem Biophys Rep. (2018) 13:83–92. 10.1016/j.bbrep.2018.01.005  [DOI](https://doi.org/10.1016/j.bbrep.2018.01.005) | [PMC free article](/articles/PMC5789218/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29387813/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20Biophys%20Rep&title=Involvement%20of%20the%20ubiquitin-proteasome%20system%20in%20the%20expression%20of%20extracellular%20matrix%20genes%20in%20retinal%20pigment%20epithelial%20cells&author=R%20de%20Carvalho%20JE&author=MT%20Verwoert&author=IMC%20Vogels&author=EA%20Reits&author=CJF%20van%20Noorden&volume=13&publication_year=2018&pages=83-92&pmid=29387813&doi=10.1016/j.bbrep.2018.01.005&)

50. Meiners S, Ludwig A, Stangl V, Stangl K. Proteasome inhibitors: poisons and remedies. Med Res Rev. (2008) 28:309–27. 10.1002/med.20111  [DOI](https://doi.org/10.1002/med.20111) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17880010/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Med%20Res%20Rev&title=Proteasome%20inhibitors:%20poisons%20and%20remedies&author=S%20Meiners&author=A%20Ludwig&author=V%20Stangl&author=K%20Stangl&volume=28&publication_year=2008&pages=309-27&pmid=17880010&doi=10.1002/med.20111&)

51. Brandes RP. Endothelial dysfunction and hypertension. Hypertension. (2014) 64:924–8. 10.1161/HYPERTENSIONAHA.114.03575  [DOI](https://doi.org/10.1161/HYPERTENSIONAHA.114.03575) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25156167/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hypertension&title=Endothelial%20dysfunction%20and%20hypertension&author=RP%20Brandes&volume=64&publication_year=2014&pages=924-8&pmid=25156167&doi=10.1161/HYPERTENSIONAHA.114.03575&)

52. Lerman A, Zeiher AM. Endothelial function: cardiac events. Circulation. (2005) 111:363–8. 10.1161/01.CIR.0000153339.27064.14  [DOI](https://doi.org/10.1161/01.CIR.0000153339.27064.14) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15668353/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation&title=Endothelial%20function:%20cardiac%20events&author=A%20Lerman&author=AM%20Zeiher&volume=111&publication_year=2005&pages=363-8&pmid=15668353&doi=10.1161/01.CIR.0000153339.27064.14&)

53. Chen-Scarabelli C, Corsetti G, Pasini E, Dioguardi FS, Sahni G, Narula J, et al. Spasmogenic effects of the proteasome inhibitor carfilzomib on coronary resistance, vascular tone and reactivity. EBioMedicine. (2017) 21:206–12. 10.1016/j.ebiom.2017.05.024  [DOI](https://doi.org/10.1016/j.ebiom.2017.05.024) | [PMC free article](/articles/PMC5514385/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28587834/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=EBioMedicine&title=Spasmogenic%20effects%20of%20the%20proteasome%20inhibitor%20carfilzomib%20on%20coronary%20resistance,%20vascular%20tone%20and%20reactivity&author=C%20Chen-Scarabelli&author=G%20Corsetti&author=E%20Pasini&author=FS%20Dioguardi&author=G%20Sahni&volume=21&publication_year=2017&pages=206-12&pmid=28587834&doi=10.1016/j.ebiom.2017.05.024&)

54. Evans KL, Wirtz HS, Li J, She R, Maya J, Gui H, et al. Genetics of heart rate in heart failure patients (GenHRate). Hum Genomics. (2019) 13:22. 10.1186/s40246-019-0206-6  [DOI](https://doi.org/10.1186/s40246-019-0206-6) | [PMC free article](/articles/PMC6528282/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31113495/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Genomics&title=Genetics%20of%20heart%20rate%20in%20heart%20failure%20patients%20(GenHRate)&author=KL%20Evans&author=HS%20Wirtz&author=J%20Li&author=R%20She&author=J%20Maya&volume=13&publication_year=2019&pages=22&pmid=31113495&doi=10.1186/s40246-019-0206-6&)

55. Tantawy M, Guang Y, Algubelli RR, Rubinstein SM, Fradley MG, Lu Q, et al. TMSB10/TRABD2A Locus associated with carfilzomib-related cardiotoxicity in multiple myeloma patients: a whole-exome sequencing analysis. Clin Pharmacol Ther. (2022) 111:S69:PW-004. 10.1002/cpt.2521  [DOI](https://doi.org/10.1002/cpt.2521) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=TMSB10/TRABD2A%20Locus%20associated%20with%20carfilzomib-related%20cardiotoxicity%20in%20multiple%20myeloma%20patients:%20a%20whole-exome%20sequencing%20analysis&author=M%20Tantawy&author=Y%20Guang&author=RR%20Algubelli&author=SM%20Rubinstein&author=MG%20Fradley&volume=111&publication_year=2022&pages=S69:PW-004&doi=10.1002/cpt.2521&)
