# The NUDIX hydrolase NUDT5 regulates thiopurine metabolism and cytotoxicity

## Metadata
**Authors:** Maud Maillard, Rina Nishii, Hieu S Vu, Kashi R Bhattarai, Wenjian Yang, Jing Li, Ute Hofmann, Daniel Savic, Smita Bhatia, Matthias Schwab, Min Ni, Jun J Yang
**Journal:** The Journal of Clinical Investigation
**Date:** 2025 Jul 15
**DOI:** [10.1172/JCI190443](https://doi.org/10.1172/JCI190443)
**PMID:** 40662362
**PMCID:** PMC12259257
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC12259257/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC12259257/pdf/jci-135-190443.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC12259257/pdf/jci-135-190443.pdf)

## Abstract

Thiopurines are anticancer agents used for the treatment of leukemia and autoimmune diseases. These purine analogs are characterized by a narrow therapeutic index because of the risk of myelosuppression. With the discovery of NUDIX hydrolase 15 (NUDT15) as a major modulator of thiopurine metabolism and toxicity, we sought to comprehensively examine all members of the NUDIX hydrolase family for their effect on the pharmacologic effects of thiopurine. By performing a NUDIX-targeted CRISPR/Cas9 screen in leukemia cells, we identified NUDT5, whose depletion led to drastic thiopurine resistance. NUDT5 deficiency resulted in a nearly complete depletion of active metabolites of thiopurine and the loss of thioguanine incorporation into DNA. Mechanistically, NUDT5 deletion resulted in substantial alteration in purine nucleotide biosynthesis, as determined by steady-state metabolomics profiling. Stable isotope tracing demonstrated that the loss of NUDT5 was linked to a marked suppression of the purine salvage pathway but with minimal effects on purine de novo synthesis. Finally, we comprehensively identified germline genetic variants in NUDT5 associated with thiopurine-induced myelosuppression in 582 children with acute lymphoblastic leukemia. Collectively, these results pointed to NUDT5 as a key regulator of the thiopurine response primarily through its effects on purine homeostasis, highlighting its potential to inform individualized thiopurine therapy.

Keywords: Genetics, Oncology, Therapeutics

Keywords: Pharmacogenetics

## Introduction

Thiopurines, including 6-thioguanine (TG), 6-mercaptopurine (MP), and its prodrug azathioprine, are widely used therapeutic agents for treating cancers such as acute lymphoblastic leukemia (ALL), as well as autoimmune diseases like inflammatory bowel disease (IBD) ([1](#B1), [2](#B2)). However, these synthetic guanine analogs have a narrow therapeutic index due to severe hematopoietic toxicity that can lead to treatment discontinuation and life-threatening infections ([3](#B3), [4](#B4)).

Thiopurines exert their antileukemic activity only after being extensively metabolized into thioguanine nucleotides (TGNs). MP and TG are first converted by hypoxanthine-guanine phosphoribosyl transferase (HPRT) into thioinosine monophosphate (TIMP) and thioguanosine monophosphate (TGMP), respectively, and subsequently into active cytotoxic thioguanosine triphosphate (TGTP). The incorporation of TGTP into DNA (as DNA-TG) triggers post-replication mismatch repair (MMR), DNA strand breaks, and, ultimately, apoptosis ([5](#B5)–[7](#B7)). Thiopurine methyltransferase (TPMT) and NUDIX hydrolase 15 (NUDT15) are 2 major metabolizing enzymes of thiopurines, both acting to prevent the excessive build-up of TGNs. More important, germline loss-of-function variants in these 2 genes have been associated with thiopurine toxicity ([8](#B8)–[18](#B18)). This led to the recommendations of a *TPMT*/*NUDT15* genotype–guided dose adjustment by the Clinical Pharmacogenetics Implementation Consortium (CPIC) ([19](#B19)), which have since been added to the drug labels issued by the US FDA, the European Medicines Agency, and other regulatory agencies globally.

NUDT15 belongs to a family of 22 hydrolases that dephosphorylate a wide variety of naturally occurring nucleotides as well as synthetic analogs that are used in cancer or infectious disease therapies ([20](#B20), [21](#B21)). With the exception of NUDT15, the role of these enzymes in thiopurine disposition has not been systematically examined. Yet, the promiscuity in their substrate recognition suggests that other NUDIX enzymes may also be involved in regulating thiopurine metabolism and/or response.

In this study, we performed a *NUDIX*-targeted CRISPR/Cas9 screen in human B-ALL cells treated with thiopurine and found that NUDIX hydrolase 5–deficient cells were resistant to thiopurine. We performed extensive metabolomics profiling followed by stable isotope tracing of the purine synthesis pathway to identify the molecular mechanisms supporting this phenotype. Finally, we systematically evaluated the association of germline genetic variants in *NUDT5* with thiopurine-related myelosuppression in 582 children with ALL.

## Results

### NUDIX-targeted CRISPR/Cas9 screen for modulators of thiopurine cytotoxicity.

To identify which of the 22 NUDIX hydrolases regulate the cytotoxic effects of thiopurines, we performed a *NUDIX*-targeted CRISPR/Cas9 screen in 2 human B-ALL cell lines, Nalm6 (*DUX4-IGH* rearranged B-ALL) and 697 (*TCF3-PBX1* fusion B-ALL) ([Figure 1A](#F1)). A library of 46 sgRNAs was assembled with 2 sgRNAs per gene, except *NUDT4*, for which 2 pairs of flanking sgRNAs were designed because of the small exon size. Following a 7-day exposure to TG, sgRNAs targeting *NUDT15* were significantly depleted, confirming the loss of this gene linked to thiopurine sensitivity (fold-change [FC] vs. no treatment control: 0.41 and 0.52, *P* = 0.006 and 0.011, in Nalm6 and 697 cells, respectively) ([Figure 1B](#F1)). Conversely, *NUDT5* sgRNAs were significantly enriched in B-ALL cells that survived thiopurine treatment (FC = 18.4 and 3.2, respectively, *P* = 0.002), indicating that NUDT5 deficiency drives drug resistance. The depletion of *NUDT21* in the control condition (without thiopurine) suggested it is essential for ALL survival (FC = 0.35 and 0.25, *P* = 0.006 and 0.002, respectively) ([22](#B22)) ([Figure 1C](#F1)). Using a FC of 2 or less or greater than 2 and a *P* value of less than 0.05 as the threshold of statistical significance, we identified no other *NUDIX* genes with significant effects on the thiopurine response.

### Figure 1. NUDIX-targeted CRISPR/Cas9 screen for modulators of thiopurine cytotoxicity.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a3f4/12259257/5e44629b3fcd/jci-135-190443-g066.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=12259257_jci-135-190443-g066.jpg)

(A) Workflow for NUDIX-targeted CRISPR/Cas9 screen in B-ALL cell lines. Illustration was created with BioRender.com. (B) Volcano plot showing the enrichment or depletion of the sgRNAs targeting the respective NUDIX genes in CRISPR/Cas9-transduced cells after 7 days of TG (TGDay7) compared with control (No treatDay7). (C) Volcano plot showing enrichment or depletion of the NUDIX genes in CRISPR/Cas9-transduced cells after 7 days of culturing (No treatDay7) compared with day 0 of culturing (i.e., immediately after lentiviral transduction [No treatDay0]), without drug exposure. The x and y axes represent the FC (in logarithmic scale) and the nominal P value of the enrichment or depletion, respectively. The vertical dotted line indicates a log2(FC) of –1 or 1, and the horizontal dotted line represents a P value of 0.05.

### NUDT5 deletion results in marked thiopurine drug resistance.

To characterize the molecular mechanism by which NUDT5 modulates the thiopurine response, we first generated *NUDT5*^KO^ B-ALL cell lines using one of the sgRNAs from the original CRISPR library and confirmed protein depletion by Western blotting ([Figure 2A](#F2)). In the absence of thiopurine drugs, *NUDT5* deletion did not alter ALL cell proliferation ([Figure 2B](#F2)). After exposing the cells to increasing concentrations of TG or MP, *NUDT5*-KO (*NUDT5*^KO^) Nalm6 cells showed minimal cell death, whereas both drugs were highly cytotoxic in parental cells in a dose-dependent manner ([Figure 2, C and D](#F2)). These results were replicated in the 697 cell line ([Figure 2, E and F](#F2)). Reexpression of *NUDT5* in the *NUDT5*^KO^ cells largely restored drug sensitivity ([Supplemental Figure 1](#sd); supplemental material available online with this article; [https://doi.org/10.1172/JCI190443DS1](https://doi.org/10.1172/JCI190443DS1)). As expected, the MMR-dependent DNA damage signaling pathway was activated in parental cells exposed to MP, e.g., histone H2AX phosphorylation and checkpoint kinase Chk1/Chk2 activation ([23](#B23)). However, no change was observed when *NUDT5* was knocked out, similar to the parental cells without MP treatment ([Supplemental Figure 2](#sd)).

### Figure 2. NUDT5 is required for thiopurine-induced apoptosis.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a3f4/12259257/c8616d3098be/jci-135-190443-g067.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=12259257_jci-135-190443-g067.jpg)

(A) Western blots confirming NUDT5 KO in the Nalm6 and 697 B-ALL cell lines. (B) Proliferation assay showing no difference in cell growth between NUDT5KO and parental cell lines. (C and D) NUDT5KO Nalm6 cell sensitivity to TG and MP, respectively, after 72 hours of treatment. (E and F) NUDT5KO 697 cell sensitivity to TG and MP, respectively, after 72 hours of treatment. Data are presented as the mean ± SD. n = 3 replicates. **P < 0.01, by 2-tailed, unpaired t test for comparisons between groups. LC50, lethal concentration 50.

### KO of NUDT5 compromises thiopurine biotransformation.

To determine whether thiopurine metabolism was altered in *NUDT5*^KO^ cells, we measured cytosolic concentrations of 12 thiopurine metabolites (namely, thioguanine mono-, di-, and triphosphate [TGMP, TGDP, and TGTP, respectively], thioinosine mono-, di-, and triphosphate [TIMP, TIDP, and TITP, respectively], and their respective methylated forms [meTGMP, meTGDP, meTGTP, meTIMP, meTIDP, and meTITP]), plus the degree of thioguanine incorporation into DNA (DNA-TG) ([Figure 3A](#F3)). The first product of MP biotransformation, TIMP, was significantly decreased in both *NUDT5*^KO^ cell lines (Nalm6: FC = 0.31, *P* < 0.001, 697: FC = 0.005, *P* < 0.001), as was its methylated species, meTIMP (FC = 0.23 and 0.017, *P* < 0.0001) ([Figure 3, B and C](#F3)). Similarly, the final product of thiopurine activation, TGTP, was significantly reduced in both *NUDT5*^KO^ cell lines (Nalm6: FC = 0.35, *P* = 0.0016, 697: not detected in *NUDT5*^KO^, *P* = 0.0002). Last, DNA-TG was almost completely absent in cells without NUDT5 (Nalm6: FC = 0.009, *P* < 0.001, 697: not detected in *NUDT5*^KO^, *P* < 0.0001). This depletion in DNA-TG was also observed when these cells were treated with TG, which utilizes a similar but not identical biotransformation pathway ([Supplemental Figure 3](#sd)). Together, these data indicate that NUDT5 is essential for the activation of thiopurines to exert their cytotoxicity.

### Figure 3. NUDT5 deletion impairs intracellular metabolism of thiopurines.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a3f4/12259257/20d093e9640a/jci-135-190443-g068.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=12259257_jci-135-190443-g068.jpg)

(A) Thiopurine metabolism pathway. Illustration was created with BioRender.com. (B and C) Cytosolic metabolites were measured in parental and NUDT5KO Nalm6 (B) and 697 (C) cell lines after treatment with 10 μM MP for 24 hours. Nuclear DNA-TG was measured after treatment with 5 μM MP for 24 hours. Data are presented as the mean ± SD. n = 3 replicates. **P < 0.01. ***P < 0.001, and ****P < 0.0001, by 2-tailed, unpaired t test.

### Depletion of NUDT5 inhibits the purine salvage pathway.

As a phosphatase, NUDT5 has been linked to the hydrolysis of adenosine diphosphate ribose (ADP-R) to generate ribose-5-phosphate (R-5-P) ([Figure 4A](#F4)), the precursor of phosphoribosyl pyrophosphate (PRPP) that is used for nucleotide biosynthesis, including both de novo and salvage purine synthesis ([Figure 4B](#F4)) ([24](#B24)). Building upon this, we first subjected both parental and *NUDT5*^KO^ cell lines to a broad metabolomics analysis examining 166 cellular metabolites involved in nucleotide, amino acid, and carbohydrate biosynthesis and degradation ([Supplemental Table 1](#sd)). The most significant change was seen with hypoxanthine and guanosine, 400-fold and 16-fold higher in *NUDT5*^KO^ compared with parental cells, respectively (*P* = 0.0004 and 0.003, [Figure 4, C and D](#F4)). Of the 23 purine-related metabolites detected, we also observed a significant increase of 5-aminoimidazole-4-carboxamide ribonucleotide (AICAR) and inosine monophosphate (IMP) in *NUDT5*^KO^ cells (2-fold, *P* = 0.0007, and 1.6-fold, *P* = 0.02, respectively). These changes pointed to the profound effect of *NUDT5* expression on purine metabolism.

### Figure 4. Targeted metabolomics profiling identifies the effects of NUDT5 on purine nucleotide homeostasis.

![Figure 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a3f4/12259257/26bb189b1535/jci-135-190443-g069.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=12259257_jci-135-190443-g069.jpg)

(A) NUDT5 hydrolyzes ADP-R into AMP and R-5-P. (B) Illustration of de novo and salvage pathways for purine synthesis. (C) Volcano plot generated after data transformation and normalization of the concentrations of metabolites measured in Nalm6 parental and NUDT5KO cells. (D) Levels of the purine intermediates hypoxanthine, guanosine, AICAR, and IMP in Nalm6 parental and NUDT5KO cells. The concentrations (conc.) of the metabolites were normalized by the protein concentration in the sample. Data are represented as the mean ± SD. n = 3 replicates. *P < 0.05, **P < 0.01, and ***P < 0.001, by 2-tailed, unpaired t test. Illustrations in A and B were created with BioRender.com.

To more precisely determine the role of NUDT5 in purine metabolism, we performed a series of isotope-tracing experiments focusing on metabolites related to either de novo purine synthesis (DNPS) or purine salvage pathways. Using nitrogen-labeled hypoxanthine ([^15^N_4_]hypoxanthine), we sought to track the ^15^N-labeled purine ring into IMP, guanosine monophosphate (GMP), and adenosine monophosphate (AMP) (m+4 labeling) through the salvage pathway ([Figure 5A](#F5)). As shown in [Figure 5B](#F5), the loss of NUDT5 markedly reduced m+4 labeling of all purine nucleotides (all *P* < 0.05), consistent with downregulation of the salvage pathway. Exposure to MP had a minimal effect on these hypoxanthine-derived purine nucleotides in either WT (*P* = 0.14, 0.26, and 0.20, for IMP, GMP, and AMP, respectively) or *NUDT5*^KO^ (*P* = 0.11, 0.02, and 0.04, for IMP, GMP and AMP, respectively) cells. These results suggested that the first step of purine salvage, i.e., HPRT-catalyzed conversion of hypoxanthine to IMP, was impaired in *NUDT5*^KO^ cells. This was likely due to the loss of HPRT catalytic activity (or enzymatic inhibition), whereas *HPRT1* expression was unaffected in the absence of NUDT5 ([Supplemental Figure 4](#sd)).

### Figure 5. Stable isotope tracing of purine synthesis pathways highlights the contribution of NUDT5 to purine metabolism.

![Figure 5](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a3f4/12259257/669d2c734ea6/jci-135-190443-g070.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=12259257_jci-135-190443-g070.jpg)

(A) Schematic illustrating 15N labeling of purine nucleotides from [15N4]hypoxanthine during the purine salvage pathway. (B) Fractional enrichment of m+4 IMP, AMP, and GMP from [15N4]hypoxanthine in parental and NUDT5KO Nalm6 cells during the purine salvage pathway, with and without MP. (C) Schematic illustrating 15N labeling of purine nucleotides from [amide-15N]glutamine during the de novo purine synthesis pathway. (D) Fractional enrichment of m+2 IMP and AMP, and m+3 GMP from [amide-15N]glutamine in parental and NUDT5KO Nalm6 cells during the purine de novo synthesis, with and without MP. Data are presented as the mean ± SD. n = 2 replicates. *P < 0.05, **P < 0.01, and ***P < 0.001, by 2-tailed, unpaired t test. Panels A and C were created with BioRender.com.

In parallel, we examined DNPS using [amide-^15^N]glutamine, which is used by phosphoribosyl pyrophosphate amidotransferase (PPAT) to convert PRPP to phosphoribosylamine (PRA), the first precursor of purine nucleotides during DNPS ([Figure 5C](#F5)). Labeled glutamine provides two ^15^N atoms for the de novo synthesis of IMP, which is further converted to AMP (m+2 labeling) or to GMP using a second molecule of glutamine (m+3 labeling). As shown in [Figure 5D](#F5), there was no change in m+2 labeling of IMP or AMP in *NUDT5*^KO^ cells relative to parental control cells, although we observed a modest decrease in m+3 labeling of GMP (*P* = 0.006), suggesting a minimal effect of NUDT5 on DNPS at baseline. In the presence of MP, we found that DNPS was greatly inhibited in parental cells, as reflected by the reduction of IMP, AMP, and GMP, whereas *NUDT5*^KO^ cells showed no MP-induced changes in DNPS.

Therefore, we reason that NUDT5 deficiency blunted HPRT activity, downregulating purine salvage and hampering the conversion of thiopurine to cytotoxic metabolites. By contrast, NUDT5 may have a minimal role in DNPS under physiological conditions but is required for DNPS inhibition by thiopurines.

### Association of NUDT5 germline variants with hematopoietic toxicity in patients receiving thiopurine therapy.

To evaluate the clinical relevance of the NUDT5*-*mediated thiopurine response, we comprehensively characterized genetic variations of *NUDT5* in 582 children with ALL enrolled on the Children’s Oncology Group AALL03N1 trial, for whom systematic data on MP-related myelosuppression were collected ([13](#B13)). Patients with known pharmacogenetic variants in *TPMT* and *NUDT15* were excluded to avoid confounding effects. A total of 2,108 variants were identified within *NUDT5* exons and 100 kb upstream of the gene ([Figure 6A](#F6)). Among 416 common variants (minor allele frequency [MAF] >0.03), the only coding variant in *NUDT5* was a synonymous variant, rs6686 (c.609A>G, p.A203A), in exon 10, which was not associated with MP dose intensity (DI) (nominal *P* = 0.472).

### Figure 6. Characterization of NUDT5 germline variants associated with thiopurine-induced myelosuppression in children with ALL.

![Figure 6](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a3f4/12259257/d73bc0d30e93/jci-135-190443-g071.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=12259257_jci-135-190443-g071.jpg)

(A) NUDT5 sequencing and genotyping data were retrieved for 582 children with ALL, who were enrolled in the Children’s Oncology Group AALL03N1 clinical trial. All patients were TPMT and NUDT15 WT. Common variants (MAF >0.03) were considered for a multiple regression association analysis with MP sensitivity. HWE, Hardy-Weinberg Equilibrium. (B) The intergenic variant rs55713253 C>T on chromosome 10 was associated with a significant decrease in MP DI during maintenance therapy. Shown is the nominal P value of the multiple regression model, assuming an additive genetic model. (C) View of the genomic environment of rs55713253 C>T from the UCSC Genome Browser (https://genome.ucsc.edu/). CLP, common lymphoid progenitor; CMP, common myeloid progenitor; DHS, DNase hypersensitivity site; GMP, granulocyte-monocyte progenitor; HSC, hematopoietic stem cell; LMPP, lymphoid-primed multipotent progenitor cell; MEP, megakaryocyte-erythrocyte progenitor; MMP, multipotent progenitor. (D) Luciferase reporter gene assay in Nalm6 cells expressing the NUDT5 rs55713253-mutant T allele showed increased expression of the reporter gene. Ref, reference allele; Alt, alternative allele. n = 3 replicates. Data are presented as the mean ± SD. **P < 0.01, by 2-tailed, unpaired t test.

Because *NUDT5* expression was linked to the thiopurine response, we hypothesized that *cis*-regulatory variants at this locus (i.e., expression quantitative trait loci, [eQTL], of *NUDT5*) are associated with MP dose intensity (DI, in %), a proxy marker for drug-induced myelosuppression. Among the 415 common noncoding variants, 32 were associated with MP DI during the maintenance therapy according to the multivariable regression model adjusting for genetic ancestry (*P* < 0.05), and 4 of them were related to *NUDT5* transcription in blood cells according to the public eQTLGen gene expression database (*P* < 0.0001) ([Supplemental Table 2](#sd)).

Only the intergenic variant rs55713253 (C>T, Chr10:12,305,358, MAF = 0.06 in the cohort) remained significant in a permutation-based analysis to account for multiple testing (nominal *P* = 0.0019 and adjusted *P* = 0.036). Patients with the WT genotype (CC, *n* = 514) tolerated a median DI of 89% (IQR, 72%–97.2%), which was higher than for patients heterozygous for rs55713253 (CT, *n* = 64, 79.2%, [63.1–92.7]), and the DI was the lowest in homozygous patients (TT, *n* = 4, DI = 52.7%, [37.4–90.0], [Figure 6B](#F6)). Consistent with our observation that NUDT5 potentiated thiopurine cytotoxicity, the T allele at this locus was also associated with elevated *NUDT5* expression in blood (*P* = 5.0 × 10^–19^, *z* score = 9.0, per eQTLGen) ([25](#B25)). We queried the RegulomeDB public database and found that rs55713253 was likely to affect *NUDT5* transcription in hematopoietic cells (probability score of 0.97 based on chromatin accessibility and transcription factor binding data); it occurs 67 kb upstream of the *NUDT5* transcription start site, in a region enriched in histone 3 methylation marks (H3K4Me1 and H3K4Me3) ([Figure 6C](#F6)). This variant overlaps with an YY1 ChIP-Seq peak in GM12878 lymphoblastoid cells according to the Encyclopedia of DNA Elements (ENCODE) database. We experimentally tested the effect of rs55713253 on enhancer activity with a reporter gene assay and found that the T allele significantly increased luciferase expression, confirming its potential to influence *NUDT5* expression (FC = 1.29, *P* = 0.01, [Figure 6D](#F6)).

## Discussion

In this study, we comprehensively explored the role of NUDIX hydrolases on thiopurine metabolism, efficacy, and toxicity. Our *NUDIX*-targeted CRISPR/Cas9 screen confirmed NUDT15 as a negative regulator of cytotoxicity of this class of drugs, but, surprisingly, we discovered that NUDT5 was required for thiopurine-induced cell death. In fact, several prior studies have reported similar findings, so much so that NUDT5 was the top thiopurine resistance gene in a genome-wide fashion ([26](#B26)), although the underlying biology is not understood. Physiologically, NUDT5 plays a critical role in metabolic homeostasis by hydrolyzing cytosolic ADP-R into AMP and R-5-P ([24](#B24), [27](#B27)–[29](#B29)). Because R-5-P is the precursor of PRPP, which is required for the conversion of MP to TIMP or TG to TGMP ([24](#B24), [28](#B28)), we initially hypothesized that NUDT5 deletion causes the loss of PRPP and thus impairs thiopurine biotransformation ([Figure 4A](#F4)). A few lines of evidence from us and others seem to refute this hypothesis: (a) leukemia cells generally favor the pentose phosphate pathway to produce R-5-P from glucose ([30](#B30)), and NUDT5-mediated hydrolysis of ADP-R is probably a minor source for this metabolite; (b) NUDT5 deletion specifically resulted in drastic reduction of purine salvage without affecting DNPS ([Figure 5](#F5)), even though both pathways depend on PRPP. Instead, our data suggest that the loss of NUDT5 impaired HPRT function (directly or indirectly), which limited the cell’s ability to convert thiopurine prodrug into active metabolites (TIMP, TGTP, and DNA-TG) and, in turn, reduced drug cytotoxicity. However, the exact molecular process by which NUDT5 modulates HPRT activity and purine salvage in this context remains unclear.

It should not go unnoticed that NUDT5 deletion also led to nearly complete depletion of meTIMP, the thiopurine metabolite responsible for DNPS inhibition ([31](#B31), [32](#B32)). These data raise the question of whether thiopurines exert their cytotoxic effects by being incorporated into nucleic acids and causing DNA mismatch (historically considered the major toxic pathway), or by inhibiting DNPS, previously deemed to be a minor pathway ([33](#B33)). The fact that *NUDT5* KO also led to resistance to TG (which does not affect DNPS) points to DNA damage as the main cause of thiopurine-induced cell death in our model systems. Nonetheless, NUDT5 clearly contributed to thiopurine activation, regardless of the exact pathways leading to cytotoxicity. Thiopurine cytotoxicity and the rate of DNPS both vary substantially by ALL subtype ([34](#B34), [35](#B35)), but this is not likely to be related to *NUDT5* because its expression is relatively consistent across ALL subtypes (St. Jude Cloud, ALL expression dataset of 925 cases: [https://pecan.stjude.cloud/expression/gene-expression](https://pecan.stjude.cloud/expression/gene-expression)), However, we cannot completely rule out the subtype-dependent effects of NUDT5 on thiopurine metabolism.

Our comprehensive analysis of *NUDT5* genetics in children with ALL suggests that increased expression of *NUDT5* may be linked to reduced tolerance to thiopurines. In this context, higher enzymatic activity would be associated with increased thiopurine activation, leading to elevated TGNs and a higher risk of hematopoietic toxicity. For example, 3 of the 4 patients homozygous at rs55713253 (TT genotype) required a greater than 50% dose reduction, with a median DI lower than that in patients with the CT genotype, although this difference did not reach statistical significance plausibly because of our limited sample size. Therefore, we reason that *NUDT5* variants may have a sizable effect on thiopurine tolerance but that the effects seem more variable than what we have seen with *NUDT15* and *TPMT* variants. Assuming an additive allelic effect, *TPMT* variants (rs1142345, rs1800462, rs1800460) were estimated to explain 4.9% of the interpatient variability in DI in our study cohort, whereas *NUDT15* rs116855232 would explain 6.1% of it. By comparison, the *NUDT5* variant (rs55713253) explains an additional 1.8% variability in this phenotype. Larger cohorts will be needed in the future to define the precise clinical relevance of the *NUDT5* genotype in thiopurine dosing. Finally, our findings also suggest that somatic loss-of-function mutations in *NUDT5* could lead to thiopurine resistance, potentially reducing the efficacy of ALL therapy. However, when we queried whole-genome or whole-exome sequencing data on 2,754 ALL cases, we found no *NUDT5* mutations in leukemia genomes ([36](#B36)).

In conclusion, we identified NUDT5 as a key modulator of thiopurine metabolism and thus of efficacy and/or toxicity, and our mechanistic studies provided insights into NUDT5-mediated regulation of purine homeostasis. Our findings indicate that the efficacy of thiopurines may be compromised in patients with *NUDT5* deficiency, whereas *NUDT5* activation can result in excessive toxicity, pointing to the importance of *NUDT5* in thiopurine pharmacogenomics.

## Methods

### Sex as a biological variable.

Our study examined male and female patients. Details about the cohort have been previously published ([37](#B37)).

### NUDIX-targeted CRISPR/Cas9-KO screen.

A library of 46 sgRNAs targeting 22 *NUDIX* genes was designed and purchased from MilliporeSigma, with each gene covered by 2 guides, except for *NUDT4*, for which 2 pairs of flanking sgRNAs were designed due to the small exon size ([Supplemental Table 3](#sd)). The Cas9-expressing human B-ALL cell lines Nalm6 and 697 were used for lentiviral transduction. Cells were cultured in RPMI-1640 (Gibco, Thermo Fisher Scientific, 11875093) containing 10% FBS (Cytiva, SH30071.03) at 37°C with 5% CO_2_. Transduced cells with sgRNAs were selected with puromycin and challenged with 1.5 μM TG for 7 days. The sgRNA sequences were recovered by genomic PCR and detected by deep sequencing performed on a MiSeq system (Illumina) ([38](#B38)). Sequencing data were analyzed with MAGeCK, version 0.5.9.5 ([39](#B39)). Details on the CRISPR/Cas9 screen can be found in the [Supplemental Methods](#sd).

### CRISPR/Cas9 genome editing of NUDT5.

One of the sgRNAs of the pair targeting *NUDT5* in the CRISPR/Cas9-KO library (sgRNA 1, 5′-CACCGGAAGTGTTCTCTGCAGCACG-3′) was subcloned into the pSpCas9(BB)-2A-GFP (px458) vector (Addgene, 48138). The B-ALL cell lines Nalm6 (American Type Culture Collection [ATCC], CRL-3273) and 697 (DSMZ, ACC42) were a gift from William E. Evans (St. Jude Children’s Research Hospital, Memphis, Tennessee, USA). The cell lines were transfected by electroporation using the Cell Line Nucleofector Kit R (Lonza, VCA-1001) and the Nucleofector II device (Lonza). Single cells were sorted by flow cytometry 48 hours after transfection on the basis of size, viability, and GFP^+^ expression using a Bigfoot Spectral Cell Sorter (Invitrogen, Thermo Fisher Scientific) and the LSRFortessa Cytometer (BD Biosciences). Flow Jo software, version 10.8.1, was used for data analysis. Single clones were selected and cultured using semi-solid ClonaCell-TCS Medium (STEMCELL Technologies, 03814). KO of *NUDT5* was verified by Western blotting (see corresponding section for details).

### NUDT5 overexpression in KO cells.

*NUDT5* cDNA was amplified by PCR and subcloned into a cl20c-IRES-mCherry lentiviral plasmid ([40](#B40)) using the NEBuilder HiFi DNA Assembly Cloning kit (New England Biolabs [NEB], E5520) ([Supplemental Table 4](#sd)). Purified empty vector and cl20c-NUDT5-IRES-mCherry plasmids were respectively transfected together with packaging vectors into Lenti-X 293T cells (Takara, 632180). After 48 hours of transfection, the culture media containing lentivirus particles were collected to infect the Nalm6 *NUDT5*^KO^ cell line. Transduction efficiency was determined after 48 hours using flow cytometry, and mCherry^+^ cells were sorted, as described above.

### In vitro thiopurine sensitivity assessment.

Sensitivity to thiopurine was tested in *NUDT5*^KO^ cell lines and *NUDT5*-overexpressing Nalm6 cells. Briefly, cells were seeded into a 384-well microplate with 1,250 cells per well, and gradient-diluted MP or TG was added to the microwells. After 72 hours of incubation, cell viability was measured by the CellTiter-Glo assay (Promega, G8461), and luminescence was detected with the Synergy H4 hybrid reader (Agilent BioTek).

### Western blotting.

A total of 1 million cells were harvested and washed once with cold PBS (Gibco, Thermo Fisher Scientific, 10010023). Cell pellets were lysed with 100 μL RIPA lysis buffer (Thermo Fisher Scientific, J63306.AP). The protein concentration in the lysates was measured using the BCA assay kit (Thermo Fisher Scientific, 23227). Protein (30 μg) for each sample was subjected to SDS-PAGE, followed by membrane transfer and incubation with antibodies. The following monoclonal antibodies for the detection of proteins of interest and control proteins were purchased from Abcam or Cell Signaling Technology (CST): β-actin (CST, 4967), GAPDH (CST, 14C10), HPRT (Abcam, EPR5299), NUDT5 (Abcam, EPR7734), pATM (CST, D6H9), pATR (CST, D5K8W), pChk1 (CST, 133D3), pChk2 (CST, E8Q1A), and γH2AX (CST, 2577). The detection was revealed using a secondary antibody conjugated to horseradish peroxidase (HRP) (Anti-rabbit IgG, Amersham ECL HRP-linked Antibody NA9340, Cytiva).

### Measurement of thiopurine cytosolic and nucleic metabolites.

Twelve thiopurine cytosolic metabolites (thioguanine nucleotides: TGMP, TGDP, TGTP; methyl thioguanine nucleotides: meTGMP, meTGDP, meTGTP; thioinosine nucleotides: TIMP, TIDP, TITP; methylthioinosine nucleotides: meTIMP, meTIDP, meTITP) were extracted from Nalm6 and 697 *NUDT5*^KO^ cells after 24 hours of MP (10 μM) and analyzed by liquid chromatography coupled to tandem mass spectrometry (LC-MS/MS) according to a previously published method ([41](#B41)). For the quantification of nuclear DNA-TG, genomic DNA was extracted from cells similarly exposed to thiopurines using the DNeasy Blood and Tissue kit (Qiagen, 69506). Eluted DNA was digested for 24 hours at 37°C with a cocktail of enzymes (all purchased from MilliporeSigma) containing benzonase (E1014), phosphodiesterase I (P3243), and alkaline phosphatase (P7923) mixed in a solution of Tris-HCl (20 mM, pH 7.9) with 100 mM NaCl and 20 mM MgCl_2_ (Invitrogen). DNA-TG was detected and quantified by LC-MS/MS following a procedure adapted from previously published methods ([42](#B42), [43](#B43)).

### Metabolomics analysis.

To evaluate the effect of NUDT5 deletion on endogenous metabolite homeostasis, we performed steady-state metabolomics profiling with Nalm6 parental and *NUDT5*^KO^ cell lines. Briefly, 5 million cells were harvested by centrifugation and washed twice with ice-cold PBS, and the pellets were snap-frozen in liquid nitrogen to quench intracellular metabolism. Endogenous metabolites were extracted according to a previously described method ([44](#B44)). Cell extracts were subjected to LC/MS-MS–based targeted metabolomics analyses. Metabolite concentrations were normalized to the protein content of each sample. The metabolomics data were analyzed with Metaboanalyst, version 6.0 ([45](#B45)). For the univariate analysis, metabolite concentrations were normalized to the median of replicates and were log transformed. For the enrichment analysis, metabolite abundance was compared with the Kyoto Encyclopedia of Genes and Genomes (KEGG) human metabolic pathways.

### Stable isotope tracing in B-ALL cells.

Stable isotope tracing assays were performed in Nalm6 parental and Nalm6 *NUDT5*^KO^ cells. All stable isotopes were purchased from Cambridge Isotope Laboratories. For the hypoxanthine tracing assay, 10 million cells were cultured in base RPMI (MilliporeSigma, 11875-093) with 10% of dialyzed FBS (GeminiBio, 100-108) and supplemented with 10 μM [^15^N_4_]hypoxanthine (NLM-8500-0.1, Cambridge Isotope Laboratories). For the glutamine tracing experiment, 10 million cells were incubated in glutamine-free RPMI (MilliporeSigma, R0883) with 10% dialyzed FBS and supplemented with 2 mM [amide-^15^N]glutamine (NLM-557-1, Cambridge Isotope Laboratories). Metabolite extraction was performed after 4 hours of incubation in the respective labeled medium, according to a previously published method ([46](#B46)). Briefly, cell pellets were rinsed with ice-cold saline twice and quenched by 80% cold methanol. Samples were subjected to three freeze-thaw cycles using liquid nitrogen and a 37°C water bath and then homogenized using a vortex mixer. Samples were then centrifuged at 20,000*g* for 15 minutes at 4°C, and the supernatants were collected into new tubes and dried overnight in a SpeedVac concentrator. Dried metabolites were reconstituted in analytical-grade water containing 0.1% formic acid. Samples were analyzed using reverse-phase liquid chromatography coupled with a high-resolution MS Q-TOF mass spectrometer ([46](#B46)).

### Patients and thiopurine therapy.

Clinical data and biological specimens were collected for 582 patients with ALL enrolled in the Children’s Oncology Group’s AALL03N1 clinical trial ([37](#B37)). All patients received daily thiopurine for the maintenance phase of their ALL. Dose adjustments were based on the monitoring of hematopoietic function and the occurrence of severe infections. MP DI (in percentages) was used as a surrogate marker of thiopurine-related toxicity, as previously described ([13](#B13)).

### NUDT5 sequencing and genotyping in patients.

Germline DNA was extracted from peripheral blood or bone marrow obtained during clinical remission, and samples were submitted for targeted sequencing of exonic regions of *TPMT*, *NUDT15*, and *NUDT5*, in addition to genotyping by genome-wide mapping array (Affymetrix SNP6 Mapping array), as previously described ([13](#B13), [47](#B47), [48](#B48)). Additional genotypes were imputed using the TOPMed imputation server. Only patients with *TPMT* and *NUDT15* WT genotypes were included in this study.

### Association of NUDT5 variants with thiopurine toxicity.

A univariate linear regression model was used to describe the association between patients’ demographics and MP DI. A multiple regression model was applied to evaluate the association between *NUDT5* genetic variants and MP DI, with population structure included as a covariate ([13](#B13)). Age at diagnosis, WBC count, and sex were not statistically significantly associated with MP DI (*P* > 0.05) and thus not adjusted for in the regression model. We assumed an additive genetic model, in which genotype was treated as an ordinal variable on the basis of the number of copies of the alternative allele. Associations with a nominal *P* value of less than 0.05 were considered statistically significant, and permutation was used to adjust for multiple testing ([Supplemental Methods](#sd)) to account for linkage disequilibrium among SNPs. The eQTLGen public gene expression database was used to perform the *cis*-eQTL analysis ([49](#B49)). Only variants located at a putative enhancer region according to the HaploReg, version 4, database were included in the permutation-based analysis to account for multi-testing ([50](#B50)). In each permutation, DIs were randomly assigned to patients, and the association between the permuted DIs and genotypes was assessed using the same regression model for each candidate SNP. The smallest *P* value was recorded for each permutation. The adjusted *P* value was the number of permuted *P* values less than or equal to the observed *P* value. A total of 1,000 permutations were performed. The effect of the variants of interest on gene transcription was assessed using the RegulomeDB database, version 2.2 ([51](#B51)).

### Dual-luciferase reporter assay.

A 300 bp sequence centered on the reference or alternative allele of intergenic variant rs55713253 (chromosome 10:12305258, hg19; C>T) was subcloned upstream of the minimal promoter into the pGL4.23-basic vector using EcoRV restriction sites and validated by Sanger sequencing (see [Supplemental Table 4](#sd) for primers). pGL4.23 luciferase plasmid DNA (60 μg) and a pRL-TK *Renilla* control vector (6 μg) were introduced into Nalm6 cells (10 million cells per replicate) using the Neon Transfection system (100 μL tip, Thermo Fisher Scientific, MPK5000). The transfection parameters were: 1,600 V, a width of 20 ms, and 1 pulse. The activity of firefly luciferase and *Renilla* was determined 48 hours after transfection using the Dual Luciferase Reporter Assay system (Promega, E1960) on a BioTek Cytation1 cell imaging multimode reader (Agilent Technologies). Luciferase activity was determined by normalizing Firefly luciferase signal to *Renilla* signal.

### Statistics.

All statistical tests were 2 sided and chosen as appropriate according to data distribution. A *P* value of less 0.05 was considered statistically significant. All *P* values are presented as nominal unless otherwise specified. MP DI was compared between *NUDT5* variant genotypes with the nonparametric Wilcoxon rank-sum test. The association study with *NUDT5* variants was performed with the rvtest package ([52](#B52)) for R software, version 4.1.3. Data were plotted with GraphPad Prism, version 10.0.0 (GraphPad Software). Data are presented as the mean ± SD.

### Study approval.

The human subject research was approved by the respective IRBs, those of St. Jude Children’s Research Hospital and the Children’s Oncology Group–affiliated institutions. Written informed consent (or assent when appropriate) were obtained from the parents, guardians, and/or patients. This study was approved by the IRB of St. Jude Children’s Research Hospital and was conducted in accordance with Declaration of Helsinki principles.

### Data availability.

Values for all data points in graphs are reported in the [Supporting Data Values](#sd) file. Genomics data and dose intensity data for the AALL03N1 cohort were deposited in the Database of Genotypes and Phenotypes (dbGaP) (study accession: phs002846.v1.p1).

## Author contributions

MM and RN are co–first authors, as they both contributed to the acquisition, analysis, and interpretation of data, with MM listed first since she drafted and revised the manuscript. JJY conceived and designed the project. MM, RN, HSV, KRB, JL, and UH performed the experiments. WY analyzed the genomics data. MM, RN, WY, DS, SB, MS, MN, and JJY interpreted the data. MM, WY, MN, and JJY contributed to the writing of the manuscript. All authors reviewed and agreed to the definitive version of the manuscript.

## Supplementary Material

## Acknowledgments

The authors wish to thank the patients and parents who took part in this study. Cas9-expressing Nalm6 and 697 cell lines were a gift from William E. Evans (St. Jude Children’s Research Hospital, Memphis, Tennessee, USA). The authors also wish to thank members at the Center for Advanced Genome Engineering at St. Jude Children’s Research Hospital for their technical support in performing the CRISPR experiments included in this study. This work was supported in part by the NIH and National Cancer Institute (NCI) grants R35 GM141947 (to JJY), R01 CA096670 (to SB), and P30 CA021765 (St. Jude Children’s Research Hospital Cancer Center Support Grant) and by the American Lebanese Syrian Associated Charities. The funders had no role in the study’s design; the collection, analysis, or interpretation of the data; or the writing of the manuscript and decision to submit it for publication. MS and UH are in part supported by the Robert Bosch Stiftung (Stuttgart, Germany). MS was supported by the Deutsche Forschungsgemeinschaft (DFG) under Germany’s Excellence Strategy-EXC 2180–390900677.

### Version 1. 07/15/2025

### Version 1. 07/15/2025

Electronic publication

## Funding Statement

No grant number.

## Footnotes

## Contributor Information

Maud Maillard, Email: maud.maillard@stjude.org.

Rina Nishii, Email: rina.nishii@bms.com.

Wenjian Yang, Email: wenjian.yang@stjude.org.

Jing Li, Email: lijing@wayne.edu.

Ute Hofmann, Email: ute.hofmann@ikp-stuttgart.de.

Daniel Savic, Email: Daniel.Savic@STJUDE.ORG.

Smita Bhatia, Email: smitabhatia@uabmc.edu.

Matthias Schwab, Email: Matthias.Schwab@ikp-stuttgart.de.

Min Ni, Email: min.ni@stjude.org.

Jun J. Yang, Email: jun.yang@stjude.org.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

Values for all data points in graphs are reported in the [Supporting Data Values](#sd) file. Genomics data and dose intensity data for the AALL03N1 cohort were deposited in the Database of Genotypes and Phenotypes (dbGaP) (study accession: phs002846.v1.p1).

### Supplementary Materials

### Data Availability Statement

Values for all data points in graphs are reported in the [Supporting Data Values](#sd) file. Genomics data and dose intensity data for the AALL03N1 cohort were deposited in the Database of Genotypes and Phenotypes (dbGaP) (study accession: phs002846.v1.p1).

## References

1. Dubinsky MC. Azathioprine, 6-mercaptopurine in inflammatory bowel disease: pharmacology, efficacy, and safety. Clin Gastroenterol Hepatol. 2004;2(9):731–743. doi: 10.1016/S1542-3565(04)00344-1.  [DOI](https://doi.org/10.1016/S1542-3565(04)00344-1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15354273/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Gastroenterol%20Hepatol&title=Azathioprine,%206-mercaptopurine%20in%20inflammatory%20bowel%20disease:%20pharmacology,%20efficacy,%20and%20safety&author=MC%20Dubinsky&volume=2&issue=9&publication_year=2004&pages=731-743&pmid=15354273&doi=10.1016/S1542-3565(04)00344-1&)

2. Pui CH, et al. Childhood acute lymphoblastic leukemia: progress through collaboration. J Clin Oncol. 2015;33(27):2938–2948. doi: 10.1200/JCO.2014.59.1636.  [DOI](https://doi.org/10.1200/JCO.2014.59.1636) | [PMC free article](/articles/PMC4567699/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26304874/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Childhood%20acute%20lymphoblastic%20leukemia:%20progress%20through%20collaboration&author=CH%20Pui&volume=33&issue=27&publication_year=2015&pages=2938-2948&pmid=26304874&doi=10.1200/JCO.2014.59.1636&)

3. Connell WR, et al. Bone marrow toxicity caused by azathioprine in inflammatory bowel disease: 27 years of experience. Gut. 1993;34(8):1081–1085. doi: 10.1136/gut.34.8.1081.  [DOI](https://doi.org/10.1136/gut.34.8.1081) | [PMC free article](/articles/PMC1374358/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8174958/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gut&title=Bone%20marrow%20toxicity%20caused%20by%20azathioprine%20in%20inflammatory%20bowel%20disease:%2027%20years%20of%20experience&author=WR%20Connell&volume=34&issue=8&publication_year=1993&pages=1081-1085&pmid=8174958&doi=10.1136/gut.34.8.1081&)

4. Levinsen M, et al. Myelotoxicity after high-dose methotrexate in childhood acute leukemia is influenced by 6-mercaptopurine dosing but not by intermediate thiopurine methyltransferase activity. Cancer Chemother Pharmacol. 2015;75(1):59–66. doi: 10.1007/s00280-014-2613-7.  [DOI](https://doi.org/10.1007/s00280-014-2613-7) | [PMC free article](/articles/PMC4446052/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25347948/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Chemother%20Pharmacol&title=Myelotoxicity%20after%20high-dose%20methotrexate%20in%20childhood%20acute%20leukemia%20is%20influenced%20by%206-mercaptopurine%20dosing%20but%20not%20by%20intermediate%20thiopurine%20methyltransferase%20activity&author=M%20Levinsen&volume=75&issue=1&publication_year=2015&pages=59-66&pmid=25347948&doi=10.1007/s00280-014-2613-7&)

5. Karran P, Attard N. Thiopurines in current medical practice: molecular mechanisms and contributions to therapy-related cancer. Nat Rev Cancer. 2008;8(1):24–36. doi: 10.1038/nrc2292.  [DOI](https://doi.org/10.1038/nrc2292) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18097462/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Cancer&title=Thiopurines%20in%20current%20medical%20practice:%20molecular%20mechanisms%20and%20contributions%20to%20therapy-related%20cancer&author=P%20Karran&author=N%20Attard&volume=8&issue=1&publication_year=2008&pages=24-36&pmid=18097462&doi=10.1038/nrc2292&)

6. Schmiegelow K, et al. Mercaptopurine/Methotrexate maintenance therapy of childhood acute lymphoblastic leukemia: clinical facts and fiction. J Pediatr Hematol Oncol. 2014;36(7):503–517. doi: 10.1097/MPH.0000000000000206.  [DOI](https://doi.org/10.1097/MPH.0000000000000206) | [PMC free article](/articles/PMC4222610/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24936744/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pediatr%20Hematol%20Oncol&title=Mercaptopurine/Methotrexate%20maintenance%20therapy%20of%20childhood%20acute%20lymphoblastic%20leukemia:%20clinical%20facts%20and%20fiction&author=K%20Schmiegelow&volume=36&issue=7&publication_year=2014&pages=503-517&pmid=24936744&doi=10.1097/MPH.0000000000000206&)

7. Nielsen SN et al. DNA-thioguanine nucleotide concentration and relapse-free survival during maintenance therapy of childhood acute lymphoblastic leukaemia (NOPHO ALL2008): a prospective substudy of a phase 3 trial. Lancet Oncol. 2017;18(4):515–524. doi: 10.1016/S1470-2045(17)30154-7.  [DOI](https://doi.org/10.1016/S1470-2045(17)30154-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28258828/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet%20Oncol&title=DNA-thioguanine%20nucleotide%20concentration%20and%20relapse-free%20survival%20during%20maintenance%20therapy%20of%20childhood%20acute%20lymphoblastic%20leukaemia%20(NOPHO%20ALL2008):%20a%20prospective%20substudy%20of%20a%20phase%203%20trial&author=al%20Nielsen%20SN%20et&volume=18&issue=4&publication_year=2017&pages=515-524&pmid=28258828&doi=10.1016/S1470-2045(17)30154-7&)

8. Krynetski EY, et al. A single point mutation leading to loss of catalytic activity in human thiopurine S-methyltransferase. Proc Natl Acad Sci U S A. 1995;92(4):949–953. doi: 10.1073/pnas.92.4.949.  [DOI](https://doi.org/10.1073/pnas.92.4.949) | [PMC free article](/articles/PMC42614/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7862671/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&title=A%20single%20point%20mutation%20leading%20to%20loss%20of%20catalytic%20activity%20in%20human%20thiopurine%20S-methyltransferase&author=EY%20Krynetski&volume=92&issue=4&publication_year=1995&pages=949-953&pmid=7862671&doi=10.1073/pnas.92.4.949&)

9. Relling MV, et al. Prognostic importance of 6-mercaptopurine dose intensity in acute lymphoblastic leukemia. Blood. 1999;93(9):2817–2823. doi: 10.1182/blood.V93.9.2817.409k04_2817_2823.  [DOI](https://doi.org/10.1182/blood.V93.9.2817.409k04_2817_2823) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10216075/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Prognostic%20importance%20of%206-mercaptopurine%20dose%20intensity%20in%20acute%20lymphoblastic%20leukemia&author=MV%20Relling&volume=93&issue=9&publication_year=1999&pages=2817-2823&pmid=10216075&doi=10.1182/blood.V93.9.2817.409k04_2817_2823&)

10. Relling MV, et al. Thiopurine methyltransferase in acute lymphoblastic leukemia. Blood. 2006;107(2):843–844. doi: 10.1182/blood-2005-08-3379.  [DOI](https://doi.org/10.1182/blood-2005-08-3379) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16401827/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Thiopurine%20methyltransferase%20in%20acute%20lymphoblastic%20leukemia&author=MV%20Relling&volume=107&issue=2&publication_year=2006&pages=843-844&pmid=16401827&doi=10.1182/blood-2005-08-3379&)

11. Hartford C, et al. Differential effects of targeted disruption of thiopurine methyltransferase on mercaptopurine and thioguanine pharmacodynamics. Cancer Res. 2007;67(10):4965–4972. doi: 10.1158/0008-5472.CAN-06-3508.  [DOI](https://doi.org/10.1158/0008-5472.CAN-06-3508) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17510427/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=Differential%20effects%20of%20targeted%20disruption%20of%20thiopurine%20methyltransferase%20on%20mercaptopurine%20and%20thioguanine%20pharmacodynamics&author=C%20Hartford&volume=67&issue=10&publication_year=2007&pages=4965-4972&pmid=17510427&doi=10.1158/0008-5472.CAN-06-3508&)

12. Yang SK, et al. A common missense variant in NUDT15 confers susceptibility to thiopurine-induced leukopenia. Nat Genet. 2014;46(9):1017–1020. doi: 10.1038/ng.3060.  [DOI](https://doi.org/10.1038/ng.3060) | [PMC free article](/articles/PMC4999337/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25108385/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=A%20common%20missense%20variant%20in%20NUDT15%20confers%20susceptibility%20to%20thiopurine-induced%20leukopenia&author=SK%20Yang&volume=46&issue=9&publication_year=2014&pages=1017-1020&pmid=25108385&doi=10.1038/ng.3060&)

13. Yang JJ, et al. Inherited NUDT15 variant is a genetic determinant of mercaptopurine intolerance in children with acute lymphoblastic leukemia. J Clin Oncol. 2015;33(11):1235–1242. doi: 10.1200/JCO.2014.59.4671.  [DOI](https://doi.org/10.1200/JCO.2014.59.4671) | [PMC free article](/articles/PMC4375304/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25624441/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Inherited%20NUDT15%20variant%20is%20a%20genetic%20determinant%20of%20mercaptopurine%20intolerance%20in%20children%20with%20acute%20lymphoblastic%20leukemia&author=JJ%20Yang&volume=33&issue=11&publication_year=2015&pages=1235-1242&pmid=25624441&doi=10.1200/JCO.2014.59.4671&)

14. Moriyama T, et al. NUDT15 polymorphisms alter thiopurine metabolism and hematopoietic toxicity. Nat Genet. 2016;48(4):367–373. doi: 10.1038/ng.3508.  [DOI](https://doi.org/10.1038/ng.3508) | [PMC free article](/articles/PMC5029084/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26878724/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=NUDT15%20polymorphisms%20alter%20thiopurine%20metabolism%20and%20hematopoietic%20toxicity&author=T%20Moriyama&volume=48&issue=4&publication_year=2016&pages=367-373&pmid=26878724&doi=10.1038/ng.3508&)

15. Nishii R, et al. Preclinical evaluation of NUDT15-guided thiopurine therapy and its effects on toxicity and antileukemic efficacy. Blood. 2018;131(22):2466–2474. doi: 10.1182/blood-2017-11-815506.  [DOI](https://doi.org/10.1182/blood-2017-11-815506) | [PMC free article](/articles/PMC5981167/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29572377/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Preclinical%20evaluation%20of%20NUDT15-guided%20thiopurine%20therapy%20and%20its%20effects%20on%20toxicity%20and%20antileukemic%20efficacy&author=R%20Nishii&volume=131&issue=22&publication_year=2018&pages=2466-2474&pmid=29572377&doi=10.1182/blood-2017-11-815506&)

16. Toksvang LN, et al. Maintenance therapy for acute lymphoblastic leukemia: basic science and clinical translations. Leukemia. 2022;36(7):1749–1758. doi: 10.1038/s41375-022-01591-4.  [DOI](https://doi.org/10.1038/s41375-022-01591-4) | [PMC free article](/articles/PMC9252897/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35654820/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Leukemia&title=Maintenance%20therapy%20for%20acute%20lymphoblastic%20leukemia:%20basic%20science%20and%20clinical%20translations&author=LN%20Toksvang&volume=36&issue=7&publication_year=2022&pages=1749-1758&pmid=35654820&doi=10.1038/s41375-022-01591-4&)

17. Maillard M, et al. Additive effects of TPMT and NUDT15 on thiopurine toxicity in children with acute lymphoblastic leukemia across multiethnic populations. J Natl Cancer Inst. 2024;116(5):702–710. doi: 10.1093/jnci/djae004.  [DOI](https://doi.org/10.1093/jnci/djae004) | [PMC free article](/articles/PMC11077315/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/38230823/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Natl%20Cancer%20Inst&title=Additive%20effects%20of%20TPMT%20and%20NUDT15%20on%20thiopurine%20toxicity%20in%20children%20with%20acute%20lymphoblastic%20leukemia%20across%20multiethnic%20populations&author=M%20Maillard&volume=116&issue=5&publication_year=2024&pages=702-710&pmid=38230823&doi=10.1093/jnci/djae004&)

18. Schaeffeler E, et al. Impact of NUDT15 genetics on severe thiopurine-related hematotoxicity in patients with European ancestry. Genet Med. 2019;21(9):2145–2150. doi: 10.1038/s41436-019-0448-7.  [DOI](https://doi.org/10.1038/s41436-019-0448-7) | [PMC free article](/articles/PMC6752748/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30728528/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genet%20Med&title=Impact%20of%20NUDT15%20genetics%20on%20severe%20thiopurine-related%20hematotoxicity%20in%20patients%20with%20European%20ancestry&author=E%20Schaeffeler&volume=21&issue=9&publication_year=2019&pages=2145-2150&pmid=30728528&doi=10.1038/s41436-019-0448-7&)

19. Relling MV, et al. Clinical pharmacogenetics implementation consortium guideline for thiopurine dosing based on TPMT and NUDT15 genotypes: 2018 update. Clin Pharmacol Ther. 2019;105(5):1095–1105. doi: 10.1002/cpt.1304.  [DOI](https://doi.org/10.1002/cpt.1304) | [PMC free article](/articles/PMC6576267/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30447069/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Clinical%20pharmacogenetics%20implementation%20consortium%20guideline%20for%20thiopurine%20dosing%20based%20on%20TPMT%20and%20NUDT15%20genotypes:%202018%20update&author=MV%20Relling&volume=105&issue=5&publication_year=2019&pages=1095-1105&pmid=30447069&doi=10.1002/cpt.1304&)

20. Carreras-Puigvert J, et al. A comprehensive structural, biochemical and biological profiling of the human NUDIX hydrolase family. Nat Commun. 2017;8(1):1541. doi: 10.1038/s41467-017-01642-w.  [DOI](https://doi.org/10.1038/s41467-017-01642-w) | [PMC free article](/articles/PMC5688067/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29142246/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&title=A%20comprehensive%20structural,%20biochemical%20and%20biological%20profiling%20of%20the%20human%20NUDIX%20hydrolase%20family&author=J%20Carreras-Puigvert&volume=8&issue=1&publication_year=2017&pages=1541&pmid=29142246&doi=10.1038/s41467-017-01642-w&)

21. Szklarczyk D, et al. The STRING database in 2023: protein-protein association networks and functional enrichment analyses for any sequenced genome of interest. Nucleic Acids Res. 2023;51(d1):D638–D646. doi: 10.1093/nar/gkac1000.  [DOI](https://doi.org/10.1093/nar/gkac1000) | [PMC free article](/articles/PMC9825434/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36370105/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=The%20STRING%20database%20in%202023:%20protein-protein%20association%20networks%20and%20functional%20enrichment%20analyses%20for%20any%20sequenced%20genome%20of%20interest&author=D%20Szklarczyk&volume=51&issue=d1&publication_year=2023&pages=D638-D646&pmid=36370105&doi=10.1093/nar/gkac1000&)

22. Brumbaugh J, et al. Nudt21 controls cell fate by connecting alternative polyadenylation to chromatin signaling. Cell. 2018;172(1-2):106–120. doi: 10.1016/j.cell.2017.11.023.  [DOI](https://doi.org/10.1016/j.cell.2017.11.023) | [PMC free article](/articles/PMC5766360/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29249356/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell&title=Nudt21%20controls%20cell%20fate%20by%20connecting%20alternative%20polyadenylation%20to%20chromatin%20signaling&author=J%20Brumbaugh&volume=172&issue=1-2&publication_year=2018&pages=106-120&pmid=29249356&doi=10.1016/j.cell.2017.11.023&)

23. Karran P. Thiopurines, DNA damage, DNA repair and therapy-related cancer. Br Med Bull. 2006;79-80:153–170. doi: 10.1093/bmb/ldl020.  [DOI](https://doi.org/10.1093/bmb/ldl020) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17277075/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20Med%20Bull&title=Thiopurines,%20DNA%20damage,%20DNA%20repair%20and%20therapy-related%20cancer&author=P%20Karran&volume=79-80&publication_year=2006&pages=153-170&pmid=17277075&doi=10.1093/bmb/ldl020&)

24. Zha M et al. Molecular mechanism of ADP-ribose hydrolysis by human NUDT5 from structural and kinetic studies. J Mol Biol. 2008;379(3):568–578. doi: 10.1016/j.jmb.2008.04.006.  [DOI](https://doi.org/10.1016/j.jmb.2008.04.006) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18462755/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Mol%20Biol&title=Molecular%20mechanism%20of%20ADP-ribose%20hydrolysis%20by%20human%20NUDT5%20from%20structural%20and%20kinetic%20studies&author=al%20Zha%20M%20et&volume=379&issue=3&publication_year=2008&pages=568-578&pmid=18462755&doi=10.1016/j.jmb.2008.04.006&)

25. Lepik K, et al. C-reactive protein upregulates the whole blood expression of CD59 - an integrative analysis. PLoS Comput Biol. 2017;13(9):e1005766. doi: 10.1371/journal.pcbi.1005766.  [DOI](https://doi.org/10.1371/journal.pcbi.1005766) | [PMC free article](/articles/PMC5609773/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28922377/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Comput%20Biol&title=C-reactive%20protein%20upregulates%20the%20whole%20blood%20expression%20of%20CD59%20-%20an%20integrative%20analysis&author=K%20Lepik&volume=13&issue=9&publication_year=2017&pages=e1005766&pmid=28922377&doi=10.1371/journal.pcbi.1005766&)

26. Oshima K, et al. Mutational and functional genetics mapping of chemotherapy resistance mechanisms in relapsed acute lymphoblastic leukemia. Nat Cancer. 2020;1(11):1113–1127. doi: 10.1038/s43018-020-00124-1.  [DOI](https://doi.org/10.1038/s43018-020-00124-1) | [PMC free article](/articles/PMC8011577/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33796864/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Cancer&title=Mutational%20and%20functional%20genetics%20mapping%20of%20chemotherapy%20resistance%20mechanisms%20in%20relapsed%20acute%20lymphoblastic%20leukemia&author=K%20Oshima&volume=1&issue=11&publication_year=2020&pages=1113-1127&pmid=33796864&doi=10.1038/s43018-020-00124-1&)

27. Arimori T, et al. Diverse substrate recognition and hydrolysis mechanisms of human NUDT5. Nucleic Acids Res. 2011;39(20):8972–8983. doi: 10.1093/nar/gkr575.  [DOI](https://doi.org/10.1093/nar/gkr575) | [PMC free article](/articles/PMC3203587/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21768126/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=Diverse%20substrate%20recognition%20and%20hydrolysis%20mechanisms%20of%20human%20NUDT5&author=T%20Arimori&volume=39&issue=20&publication_year=2011&pages=8972-8983&pmid=21768126&doi=10.1093/nar/gkr575&)

28. Ito R, et al. Cleavage of oxidized guanine nucleotide and ADP sugar by human NUDT5 protein. J Biochem. 2011;149(6):731–738. doi: 10.1093/jb/mvr028.  [DOI](https://doi.org/10.1093/jb/mvr028) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21389046/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biochem&title=Cleavage%20of%20oxidized%20guanine%20nucleotide%20and%20ADP%20sugar%20by%20human%20NUDT5%20protein&author=R%20Ito&volume=149&issue=6&publication_year=2011&pages=731-738&pmid=21389046&doi=10.1093/jb/mvr028&)

29. Page BDG, et al. Targeted NUDT5 inhibitors block hormone signaling in breast cancer cells. Nat Commun. 2018;9(1):250. doi: 10.1038/s41467-017-02293-7.  [DOI](https://doi.org/10.1038/s41467-017-02293-7) | [PMC free article](/articles/PMC5772648/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29343827/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&title=Targeted%20NUDT5%20inhibitors%20block%20hormone%20signaling%20in%20breast%20cancer%20cells&author=BDG%20Page&volume=9&issue=1&publication_year=2018&pages=250&pmid=29343827&doi=10.1038/s41467-017-02293-7&)

30. Soltani M, et al. The importance of cellular metabolic pathways in pathogenesis and selective treatments of hematological malignancies. Front Oncol. 2021;11:767026. doi: 10.3389/fonc.2021.767026.  [DOI](https://doi.org/10.3389/fonc.2021.767026) | [PMC free article](/articles/PMC8636012/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34868994/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front%20Oncol&title=The%20importance%20of%20cellular%20metabolic%20pathways%20in%20pathogenesis%20and%20selective%20treatments%20of%20hematological%20malignancies&author=M%20Soltani&volume=11&publication_year=2021&pages=767026&pmid=34868994&doi=10.3389/fonc.2021.767026&)

31. Stet EH, et al. Reversal of 6-mercaptopurine and 6-methylmercaptopurine ribonucleoside cytotoxicity by amidoimidazole carboxamide ribonucleoside in Molt F4 human malignant T-lymphoblasts. Biochem Pharmacol. 1993;46(3):547–550. doi: 10.1016/0006-2952(93)90534-4.  [DOI](https://doi.org/10.1016/0006-2952(93)90534-4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8347177/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20Pharmacol&title=Reversal%20of%206-mercaptopurine%20and%206-methylmercaptopurine%20ribonucleoside%20cytotoxicity%20by%20amidoimidazole%20carboxamide%20ribonucleoside%20in%20Molt%20F4%20human%20malignant%20T-lymphoblasts&author=EH%20Stet&volume=46&issue=3&publication_year=1993&pages=547-550&pmid=8347177&doi=10.1016/0006-2952(93)90534-4&)

32. Coulthard SA, et al. Increased sensitivity to thiopurines in methylthioadenosine phosphorylase-deleted cancers. Mol Cancer Ther. 2011;10(3):495–504. doi: 10.1158/1535-7163.MCT-10-0798.  [DOI](https://doi.org/10.1158/1535-7163.MCT-10-0798) | [PMC free article](/articles/PMC3059482/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21282358/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Cancer%20Ther&title=Increased%20sensitivity%20to%20thiopurines%20in%20methylthioadenosine%20phosphorylase-deleted%20cancers&author=SA%20Coulthard&volume=10&issue=3&publication_year=2011&pages=495-504&pmid=21282358&doi=10.1158/1535-7163.MCT-10-0798&)

33. Dervieux T, et al. Differing contribution of thiopurine methyltransferase to mercaptopurine versus thioguanine effects in human leukemic cells. Cancer Res. 2001;61(15):5810–5816.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/11479220/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=Differing%20contribution%20of%20thiopurine%20methyltransferase%20to%20mercaptopurine%20versus%20thioguanine%20effects%20in%20human%20leukemic%20cells&author=T%20Dervieux&volume=61&issue=15&publication_year=2001&pages=5810-5816&pmid=11479220&)

34. Lee SHR, et al. Pharmacotypes across the genomic landscape of pediatric acute lymphoblastic leukemia and impact on treatment response. Nat Med. 2023;29(1):170–179. doi: 10.1038/s41591-022-02112-7.  [DOI](https://doi.org/10.1038/s41591-022-02112-7) | [PMC free article](/articles/PMC9873558/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36604538/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Med&title=Pharmacotypes%20across%20the%20genomic%20landscape%20of%20pediatric%20acute%20lymphoblastic%20leukemia%20and%20impact%20on%20treatment%20response&author=SHR%20Lee&volume=29&issue=1&publication_year=2023&pages=170-179&pmid=36604538&doi=10.1038/s41591-022-02112-7&)

35. Zaza G, et al. Acute lymphoblastic leukemia with TEL-AML1 fusion has lower expression of genes involved in purine metabolism and lower de novo purine synthesis. Blood. 2004;104(5):1435–1441. doi: 10.1182/blood-2003-12-4306.  [DOI](https://doi.org/10.1182/blood-2003-12-4306) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15142881/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Acute%20lymphoblastic%20leukemia%20with%20TEL-AML1%20fusion%20has%20lower%20expression%20of%20genes%20involved%20in%20purine%20metabolism%20and%20lower%20de%20novo%20purine%20synthesis&author=G%20Zaza&volume=104&issue=5&publication_year=2004&pages=1435-1441&pmid=15142881&doi=10.1182/blood-2003-12-4306&)

36. Brady SW, et al. The genomic landscape of pediatric acute lymphoblastic leukemia. Nat Genet. 2022;54(9):1376–1389. doi: 10.1038/s41588-022-01159-z.  [DOI](https://doi.org/10.1038/s41588-022-01159-z) | [PMC free article](/articles/PMC9700506/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36050548/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=The%20genomic%20landscape%20of%20pediatric%20acute%20lymphoblastic%20leukemia&author=SW%20Brady&volume=54&issue=9&publication_year=2022&pages=1376-1389&pmid=36050548&doi=10.1038/s41588-022-01159-z&)

37. Bhatia S, et al. Systemic exposure to thiopurines and risk of relapse in children with acute lymphoblastic leukemia: a Children’s Oncology Group Study. JAMA Oncol. 2015;1(3):287–295. doi: 10.1001/jamaoncol.2015.0245.  [DOI](https://doi.org/10.1001/jamaoncol.2015.0245) | [PMC free article](/articles/PMC4561178/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26181173/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA%20Oncol&title=Systemic%20exposure%20to%20thiopurines%20and%20risk%20of%20relapse%20in%20children%20with%20acute%20lymphoblastic%20leukemia:%20a%20Children%E2%80%99s%20Oncology%20Group%20Study&author=S%20Bhatia&volume=1&issue=3&publication_year=2015&pages=287-295&pmid=26181173&doi=10.1001/jamaoncol.2015.0245&)

38. Zhang H, et al. Functional interrogation of HOXA9 regulome in MLLr leukemia via reporter-based CRISPR/Cas9 screen. Elife. 2020;9:e57858. doi: 10.7554/eLife.57858.  [DOI](https://doi.org/10.7554/eLife.57858) | [PMC free article](/articles/PMC7599066/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33001025/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Elife&title=Functional%20interrogation%20of%20HOXA9%20regulome%20in%20MLLr%20leukemia%20via%20reporter-based%20CRISPR/Cas9%20screen&author=H%20Zhang&volume=9&publication_year=2020&pages=e57858&pmid=33001025&doi=10.7554/eLife.57858&)

39. Li W, et al. MAGeCK enables robust identification of essential genes from genome-scale CRISPR/Cas9 knockout screens. Genome Biol. 2014;15(12):554. doi: 10.1186/s13059-014-0554-4.  [DOI](https://doi.org/10.1186/s13059-014-0554-4) | [PMC free article](/articles/PMC4290824/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25476604/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Biol&title=MAGeCK%20enables%20robust%20identification%20of%20essential%20genes%20from%20genome-scale%20CRISPR/Cas9%20knockout%20screens&author=W%20Li&volume=15&issue=12&publication_year=2014&pages=554&pmid=25476604&doi=10.1186/s13059-014-0554-4&)

40. Li Y, et al. PAX5 epigenetically orchestrates CD58 transcription and modulates blinatumomab response in acute lymphoblastic leukemia. Sci Adv. 2022;8(50):eadd6403. doi: 10.1126/sciadv.add6403.  [DOI](https://doi.org/10.1126/sciadv.add6403) | [PMC free article](/articles/PMC9750140/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36516256/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sci%20Adv&title=PAX5%20epigenetically%20orchestrates%20CD58%20transcription%20and%20modulates%20blinatumomab%20response%20in%20acute%20lymphoblastic%20leukemia&author=Y%20Li&volume=8&issue=50&publication_year=2022&pages=eadd6403&pmid=36516256&doi=10.1126/sciadv.add6403&)

41. Hofmann U, et al. Simultaneous quantification of eleven thiopurine nucleotides by liquid chromatography-tandem mass spectrometry. Anal Chem. 2012;84(3):1294–1301. doi: 10.1021/ac2031699.  [DOI](https://doi.org/10.1021/ac2031699) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22220820/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anal%20Chem&title=Simultaneous%20quantification%20of%20eleven%20thiopurine%20nucleotides%20by%20liquid%20chromatography-tandem%20mass%20spectrometry&author=U%20Hofmann&volume=84&issue=3&publication_year=2012&pages=1294-1301&pmid=22220820&doi=10.1021/ac2031699&)

42. Quinlivan EP, Gregory JF. DNA digestion to deoxyribonucleoside: a simplified one-step procedure. Anal Biochem. 2008;373(2):383–385. doi: 10.1016/j.ab.2007.09.031.  [DOI](https://doi.org/10.1016/j.ab.2007.09.031) | [PMC free article](/articles/PMC2239294/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18028864/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anal%20Biochem&title=DNA%20digestion%20to%20deoxyribonucleoside:%20a%20simplified%20one-step%20procedure&author=EP%20Quinlivan&author=JF%20Gregory&volume=373&issue=2&publication_year=2008&pages=383-385&pmid=18028864&doi=10.1016/j.ab.2007.09.031&)

43. Jacobsen JH, et al. Liquid chromatography-tandem mass spectrometry quantification of 6-thioguanine in DNA using endogenous guanine as internal standard. J Chromatogr B Analyt Technol Biomed Life Sci. 2012;881-882:115–118. doi: 10.1016/j.jchromb.2011.11.032.  [DOI](https://doi.org/10.1016/j.jchromb.2011.11.032) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22178190/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Chromatogr%20B%20Analyt%20Technol%20Biomed%20Life%20Sci&title=Liquid%20chromatography-tandem%20mass%20spectrometry%20quantification%20of%206-thioguanine%20in%20DNA%20using%20endogenous%20guanine%20as%20internal%20standard&author=JH%20Jacobsen&volume=881-882&publication_year=2012&pages=115-118&pmid=22178190&doi=10.1016/j.jchromb.2011.11.032&)

44. Moriyama T, et al. Mechanisms of NT5C2-mediated thiopurine resistance in acute lymphoblastic leukemia. Mol Cancer Ther. 2019;18(10):1887–1895. doi: 10.1158/1535-7163.MCT-18-1112.  [DOI](https://doi.org/10.1158/1535-7163.MCT-18-1112) | [PMC free article](/articles/PMC6774896/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31358663/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Cancer%20Ther&title=Mechanisms%20of%20NT5C2-mediated%20thiopurine%20resistance%20in%20acute%20lymphoblastic%20leukemia&author=T%20Moriyama&volume=18&issue=10&publication_year=2019&pages=1887-1895&pmid=31358663&doi=10.1158/1535-7163.MCT-18-1112&)

45. Pang Z, et al. MetaboAnalyst 6.0: towards a unified platform for metabolomics data processing, analysis and interpretation. Nucleic Acids Res. 2024;52(W1):W398–W406. doi: 10.1093/nar/gkae253.  [DOI](https://doi.org/10.1093/nar/gkae253) | [PMC free article](/articles/PMC11223798/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/38587201/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=MetaboAnalyst%206.0:%20towards%20a%20unified%20platform%20for%20metabolomics%20data%20processing,%20analysis%20and%20interpretation&author=Z%20Pang&volume=52&issue=W1&publication_year=2024&pages=W398-W406&pmid=38587201&doi=10.1093/nar/gkae253&)

46. Wu Z, et al. Electron transport chain inhibition increases cellular dependence on purine transport and salvage. Cell Metab. 2024;36(7):1504–1520. doi: 10.1016/j.cmet.2024.05.014.  [DOI](https://doi.org/10.1016/j.cmet.2024.05.014) | [PMC free article](/articles/PMC11240302/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/38876105/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Metab&title=Electron%20transport%20chain%20inhibition%20increases%20cellular%20dependence%20on%20purine%20transport%20and%20salvage&author=Z%20Wu&volume=36&issue=7&publication_year=2024&pages=1504-1520&pmid=38876105&doi=10.1016/j.cmet.2024.05.014&)

47. DePristo MA, et al. A framework for variation discovery and genotyping using next-generation DNA sequencing data. Nat Genet. 2011;43(5):491–498. doi: 10.1038/ng.806.  [DOI](https://doi.org/10.1038/ng.806) | [PMC free article](/articles/PMC3083463/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21478889/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=A%20framework%20for%20variation%20discovery%20and%20genotyping%20using%20next-generation%20DNA%20sequencing%20data&author=MA%20DePristo&volume=43&issue=5&publication_year=2011&pages=491-498&pmid=21478889&doi=10.1038/ng.806&)

48. Moriyama T, et al. Comprehensive characterization of pharmacogenetic variants in TPMT and NUDT15 in children with acute lymphoblastic leukemia. Pharmacogenet Genomics. 2022;32(2):60–66. doi: 10.1097/FPC.0000000000000453.  [DOI](https://doi.org/10.1097/FPC.0000000000000453) | [PMC free article](/articles/PMC8702453/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34412101/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Comprehensive%20characterization%20of%20pharmacogenetic%20variants%20in%20TPMT%20and%20NUDT15%20in%20children%20with%20acute%20lymphoblastic%20leukemia&author=T%20Moriyama&volume=32&issue=2&publication_year=2022&pages=60-66&pmid=34412101&doi=10.1097/FPC.0000000000000453&)

49. Võsa U, et al. Large-scale cis- and trans-eQTL analyses identify thousands of genetic loci and polygenic scores that regulate blood gene expression. Nat Genet. 2021;53(9):1300–1310. doi: 10.1038/s41588-021-00913-z.  [DOI](https://doi.org/10.1038/s41588-021-00913-z) | [PMC free article](/articles/PMC8432599/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34475573/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Large-scale%20cis-%20and%20trans-eQTL%20analyses%20identify%20thousands%20of%20genetic%20loci%20and%20polygenic%20scores%20that%20regulate%20blood%20gene%20expression&author=U%20V%C3%B5sa&volume=53&issue=9&publication_year=2021&pages=1300-1310&pmid=34475573&doi=10.1038/s41588-021-00913-z&)

50. Ward LD, Kellis M. HaploReg v4: systematic mining of putative causal variants, cell types, regulators and target genes for human complex traits and disease. Nucleic Acids Res. 2016;44(d1):D877–D881. doi: 10.1093/nar/gkv1340.  [DOI](https://doi.org/10.1093/nar/gkv1340) | [PMC free article](/articles/PMC4702929/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26657631/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=HaploReg%20v4:%20systematic%20mining%20of%20putative%20causal%20variants,%20cell%20types,%20regulators%20and%20target%20genes%20for%20human%20complex%20traits%20and%20disease&author=LD%20Ward&author=M%20Kellis&volume=44&issue=d1&publication_year=2016&pages=D877-D881&pmid=26657631&doi=10.1093/nar/gkv1340&)

51. Dong S, et al. Annotating and prioritizing human non-coding variants with RegulomeDB v.2. Nat Genet. 2023;55(5):724–726. doi: 10.1038/s41588-023-01365-3.  [DOI](https://doi.org/10.1038/s41588-023-01365-3) | [PMC free article](/articles/PMC10989417/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/37173523/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Annotating%20and%20prioritizing%20human%20non-coding%20variants%20with%20RegulomeDB%20v.2&author=S%20Dong&volume=55&issue=5&publication_year=2023&pages=724-726&pmid=37173523&doi=10.1038/s41588-023-01365-3&)

52. Zhan X, et al. RVTESTS: an efficient and comprehensive tool for rare variant association analysis using sequence data. Bioinformatics. 2016;32(9):1423–1426. doi: 10.1093/bioinformatics/btw079.  [DOI](https://doi.org/10.1093/bioinformatics/btw079) | [PMC free article](/articles/PMC4848408/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27153000/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=RVTESTS:%20an%20efficient%20and%20comprehensive%20tool%20for%20rare%20variant%20association%20analysis%20using%20sequence%20data&author=X%20Zhan&volume=32&issue=9&publication_year=2016&pages=1423-1426&pmid=27153000&doi=10.1093/bioinformatics/btw079&)
