# Novel Genetic Variants Explaining Severe Adverse Drug Events after Clinical Implementation of DPYD Genotype-Guided Therapy with Fluoropyrimidines: An Observational Study

## Metadata
**Authors:** Xando Díaz-Villamarín, María Martínez-Pérez, María Teresa Nieto-Sánchez, Gabriela Ruiz-Tueros, Emilio Fernández-Varón, Alicia Torres-García, Beatriz González Astorga, Isabel Blancas, Antonio J Iáñez, José Cabeza-Barrera, Rocío Morón
**Journal:** Pharmaceutics
**Date:** 2024 Jul 19
**DOI:** [10.3390/pharmaceutics16070956](https://doi.org/10.3390/pharmaceutics16070956)
**PMID:** 39065653
**PMCID:** PMC11280107
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11280107/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC11280107/pdf/pharmaceutics-16-00956.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC11280107/pdf/pharmaceutics-16-00956.pdf)

## Abstract

Fluoropyrimidines (FPs) are commonly prescribed in many cancer streams. The EMA and FDA-approved drug labels for FPs recommend genotyping the DPYD*2A (rs3918290), *13 (rs55886062), *HapB3 (rs56038477), alleles, and DPYD rs67376798 before treatment starts. We implemented the DPYD genotyping in our daily clinical routine, but we still found patients showing severe adverse drug events (ADEs) to FPs. We studied among these patients the DPYD rs1801265, rs17376848, rs1801159, rs1801160, rs1801158, and rs2297595 as explanatory candidates of the interindividual differences for FP-related toxicities, examining the association with the response to FPs . We also studied the impact of DPYD testing for FP dose tailoring in our clinical practice and characterized the DPYD gene in our population. We found a total acceptance among physicians of therapeutic recommendations translated from the DPYD test, and this dose tailoring does not affect the treatment efficacy. We also found that the DPYD*4 (defined by rs1801158) allele is associated with a higher risk of ADEs (severity grade ≥ 3) in both the univariate (O.R. = 5.66; 95% C.I. = 1.35–23.67; p = 0.014) and multivariate analyses (O.R. = 5.73; 95% C.I. = 1.41–28.77; p = 0.019) among FP-treated patients based on the DPYD genotype. This makes it a candidate variant for implementation in clinical practice.

Keywords: fluoropyrimidines, DPYD, personalized medicine, pharmacogenetics, clinical implementation

## 1. Introduction

Fluorouracil and its oral prodrug capecitabine, both fluoropyrimidines (FPs), are anticancer drugs used in the treatment of many solid tumors, such as breast and gastric, but especially in colorectal cancer. Despite the wide use of these drugs, among FP-treated patients with standard doses, up to 30% have severe (grade ≥ 3) treatment-related toxicity [[1](#B1-pharmaceutics-16-00956),[2](#B2-pharmaceutics-16-00956)], which can lead to treatment-related death in up to 1% of patients [[2](#B2-pharmaceutics-16-00956),[3](#B3-pharmaceutics-16-00956),[4](#B4-pharmaceutics-16-00956)]. The most common severe treatment-related toxicity includes diarrhea, oropharyngeal mucositis, hand–foot syndrome, and myelosuppression [[2](#B2-pharmaceutics-16-00956),[5](#B5-pharmaceutics-16-00956)].

Fluorouracil is administered intravenously, while capecitabine is an oral prodrug metabolized by carboxylesterase, cytidine deaminase, and thymidine phosphorylase resulting, respectively, in 5′-deoxy-5-fluorocytidin (5′dFCR), 5′-deoxy-5-fluorouridine (5′dFUR), and finally 5-fluorouracil (5-FU) [[6](#B6-pharmaceutics-16-00956)] ([Figure 1](#pharmaceutics-16-00956-f001)). After this, around 80% of fluorouracil is inactivated, and the remaining 20% is converted into cytotoxic metabolites or excreted in the urine [[5](#B5-pharmaceutics-16-00956)]. FPs are catabolized by different enzymes, leading to their inactivation, and dihydropyrimidine dehydrogenase (DPD) is the main enzyme responsible for this [[7](#B7-pharmaceutics-16-00956)].

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e1af/11280107/ae1b28efdb40/pharmaceutics-16-00956-g001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=11280107_pharmaceutics-16-00956-g001.jpg)

Capecitabine/fluorouracil pathway. Carboxylesterase 1 (CES1); carboxylesterase 2 (CES2); cytidine deaminase (CDA); thymidine phosphorylase (TYMP); 5′-deoxy-5-fluorocytidin (5′dFCR); 5′-deoxy-5-fluorouridine (5′dFUR); uridine-5-monophosphate synthase (UMPS); amidophosphoribosyltransferase (PPAT); dihydropyrimidine dehydrogenase (DPD); fluorouridine monophosphate (FUMP); fluorouridine diphosphate (FUDP); ribonucleoside diphosphate reductase (RRM1 y 2); thymidine kinase (TKI); fluorodeoxyuridine monophosphate (FdUMP); fluorodeoxyuridine diphosphate (FdUDP); fluorodeoxyuridine triphosphate (FdUTP); deoxyuridine monophosphate (dUMP); deoxythymidine monophosphate (dTMP); 5,10 methylene tetrahydrofolate (5–10 methylene THF); dihidrofolato (DHF); dihydrofluorouracil (DHFU); dihidropirimidinasa (DPYS); 5-fluoro-ureidopropionic acid (FUPA); β-ureidopropionase (UPB1); α-fluoro-β-alanina (FBAL).

Patients with reduced DPD activity catabolize less fluorouracil, leading to more than 20% of cytotoxic/therapeutic metabolites, thus increasing the risk of supratherapeutic drug availability toxicity. In European and North American patients, reduced and complete lack of DPD activity are present in 3–5% and 0.01–0.1%, respectively [[2](#B2-pharmaceutics-16-00956)].

### 1.1. The Dihydropyrimidine Dehydrogenase Gene (DPYD)

Many genetic variants in the gene encoding the DPD enzyme (*DPYD* gene) partially explained the interindividual differences in the toxicity of FPs. The most relevant genetic variants in this regard are the *DPYD* rs67376798 (c.2846A>T, D949V), the rs3918290 (c.1905+1G>A, IVS14+1G>A) defining the *DPYD**2A allele, the rs55886062 (c.1679T>G, I560S) defining the *DPYD**13, and the rs75017182 (c.1129-5923C>G) and rs56038477 (c.1236G>A, E412E) characterizing the *DPYD**haplotype (Hap) B3. All of them have been associated with a reduction in DPD activity, resulting in higher concentrations of fluorouracil cytotoxic metabolites and an increased risk of severe or fatal toxicities.

The *DPYD**2A (rs3918290) was the first allele known to have a functional impact on DPD activity [[8](#B8-pharmaceutics-16-00956),[9](#B9-pharmaceutics-16-00956)]. It was stated that homozygous carriers of this variant have a complete loss of function of DPD while heterozygous carriers showed a significant reduction in mean DPD activity [[9](#B9-pharmaceutics-16-00956),[10](#B10-pharmaceutics-16-00956)]. Several studies reported a significant association of this allele with the toxicity of FPs, even though the FP dose tailoring based on *DPYD**2A genotype was demonstrated to be an effective strategy for the prevention of severe or fatal toxicity events [[11](#B11-pharmaceutics-16-00956)]. In their study, Deene et al. showed in a cohort of n = 1631 patients that a 50% dose reduction in *DPYD**2A (rs3918290) carriers may decrease the FP-related toxicity from 73% to 28% (*p* < 0.001). Thereafter, when studying *DPYD**2A (defined by rs3918290) combined with *DPYD**13 (defined by rs55886062), *DPYD**Hap B3 (defined by rs56038477), and rs67376798, a 25.6-fold increased risk of death in FP-treated patients was found [[12](#B12-pharmaceutics-16-00956)], and a systematic review and meta-analysis including n = 7365 patients concluded that *DPYD* genotyping for these four *DPYD* variants can identify a significant ratio of patients who are DPD deficient [[13](#B13-pharmaceutics-16-00956)]. Finally, Henricks L.M. et al. [[2](#B2-pharmaceutics-16-00956)] found the prospective *DPYD* genotyping feasible in routine clinical practice, and dose reductions based on *DPYD* genotype improved patient safety in FP-treated patients. In this prospective multicenter study, they tailored the FP treatment dose based on *DPYD**2A (defined by rs3918290), *DPYD**13 (defined by rs55886062), *DPYD**HapB3 (defined by rs56038477), and rs67376798 in n = 1103 patients. *DPYD**2A and *13 carriers received a 50% dose reduction, while rs67376798 and *DPYD**HapB3 carriers received a dose reduction of 25%. Results indicated that FP-related severe toxicity (grade ≥ 3) was more prevalent among *DPYD* variant carriers compared to wild-type patients (*p* = 0.0013), and a significantly lower relative risk for severe FP-related toxicity for *DPYD* genotype-guided dosing was observed compared with historical cohorts for each genetic variant carrier.

On the other hand, DPYD rs3918290, rs55886062, and rs56038477, characterizing the DPYD*2A, *13, and *HapB3 alleles, respectively, and rs67376798, are not the only DPYD variants associated with the toxicity of FPs. There are many others that showed the highest level of evidence according to the PharmGKB [[14](#B14-pharmaceutics-16-00956)] with this association ([Table 1](#pharmaceutics-16-00956-t001)).

### Table 1.

| *Allele | DPYDVariant | Major Nucleotide Variation | MAF |  |  | Molecules | Toxicity ^ |
| --- | --- | --- | --- | --- | --- | --- | --- |
| Ibs | Europe | Global |  |  |  |  |  |
| - | rs115232898 | c.557A>G | 100/0 | 100/0 | 99/1 | Fluorouracil | x |
| - | rs148994843 | c.1543G>A | No data | 100/0 | 100/0 | Fluorouracil |   |
| - | rs17376848 | c.1896T>C | 98/2 | 96/4 | 94/5 | Capecitabine/fluorouracil | x |
| *4 | rs1801158 | c.1601G>A | 94/6 | 97/3 | 99/1 | Capecitabine/fluorouracil | x |
| *5 | rs1801159 | c.1627A>G | 79/21 | 81/19 | 82/18 | Capecitabine/fluorouracil | x |
| *6 | rs1801160 | c.2194G>A | 95/5 | 95/5 | 96/4 | Capecitabine/fluorouracil | x |
| *9A | rs1801265 | c.85T>C | 79/21 | 79/21 | 74/26 | Capecitabine/fluorouracil | x |
| *8 | rs1801266 | c.703C>T | No data | 100/0 | 100/0 | Fluorouracil |   |
| *10 | rs1801268 | c.2983G>T | No data | 100/0 | 100/0 | Fluorouracil |   |
| - | rs2297595 | c.496A>G | 88/12 | 88/12 | 94/6 | Capecitabine/fluorouracil | x |
| *2A | rs3918290 | c.1905+1G>A | 100/0 | 99/1 | 99/1 | Capecitabine/fluorouracil/tegafur | x |
| *13 | rs55886062 | c.1679T>G | 100/0 | 100/0 | 100/0 | Capecitabine/fluorouracil/tegafur | x |
| - | rs56005131 | c.2303C>A | 100/0 | 100/0 | 100/0 | Fluorouracil |   |
| *HapB3 | rs56038477 | c.1236G>A | 98/2 | 98/2 | 99/1 | Capecitabine/fluorouracil | x |
| - | rs59086055 | c.1774C>T | 100/0 | 100/0 | 100/0 | Fluorouracil | x |
| - | rs67376798 | c.2846A>T | 100/0 | 99/1 | 99/1 | Capecitabine/fluorouracil/tegafur | x |
| *3 | rs72549303 | c.1898del | No data | 100/0 | 100/0 | Fluorouracil |   |
| *11 | rs72549306 | c.1003G>T | No data | 100/0 | 100/0 | Fluorouracil |   |
| *7 | rs72549309 | c.299_302del | No data | 100/0 | 100/0 | Fluorouracil |   |
| *HapB3 | rs75017182 | c.1129-5923C>G | 98/2 | 98/2 | 99/1 | Capecitabine/fluorouracil | x |
| *12 | rs78060119 | c.1156G>T | No data | 100/0 | 100/0 | Fluorouracil | x |

Table 1 Caption: DPYD variants associated with any fluoropyrimidine phenotype with a 1A level of evidence (obtained from PharmGKB [14]).

### 1.2. Pharmacogenetics of Fluoropyrimidines

All the evidence regarding the association of *DPYD* variants with FP-related toxicity, including the reduction in toxicity with dose tailoring based on the *DPYD* genotype and the clinical impact of this practice, has led to updates for the drug labels approved by the Food and Drug Administration (FDA) and the European Medicines Agency (EMA), incorporating this information, as well as the development of pharmacogenetic (PGx) dosing guidelines for these drugs.

The European Public Assessment Report (EPAR) [[16](#B16-pharmaceutics-16-00956)] for capecitabine by the EMA states that capecitabine is contraindicated in patients with no DPD activity and recommends a reduction in starting dose in reduced DPD activity patients to avoid serious toxicity. The EPAR also reports that the reduced DPD activity may be stated considering the *DPYD* genotype and highlights *DPYD**2A (defined by rs3918290), *DPYD**13 (defined by rs55886062), *DPYD**HapB3 (defined by rs56038477), and rs67376798 (also known as c.2846A>T) as being mainly responsible for DPD reduced/complete absence activity, while considering that there may be other *DPYD* variants associated with an increased risk of life-threatening toxicity. The FDA table of PGx [[17](#B17-pharmaceutics-16-00956)] includes information about the capecitabine and fluorouracil/*DPYD* drug–gene interactions, reporting that genotype-translated DPD intermediate or poor metabolizer (IM or PM) phenotypes result in higher severe or life-threatening risk toxicities. It considers that there is no dosage safe in PMs and recommends withholding or discontinuing treatment in the presence of early-onset or unusually severe toxicity.

Carrying these *DPYD* variants alone or in combination results in different translated phenotypes for DPD activity, thus different dosing recommendations. As commented above, there are available PGx dosing guidelines including this information. Both the Clinical Pharmacogenetics Implementation Consortium (CPIC) [[18](#B18-pharmaceutics-16-00956)] and Dutch Pharmacogenomics Working Group (DPWG) guidelines [[19](#B19-pharmaceutics-16-00956)] categorize patients’ *DPYD* gene activity score (GAS) depending on *DPYD* genotype, as shown in [Table 2](#pharmaceutics-16-00956-t002). Furthermore, CPIC guidelines refer to the DPD metabolizing status and categorize patients as normal metabolizers (NM) if they are not carrying any of these variants (GAS = 2), as IM if *DPYD* GAS = 1 or 1.5, and as PM when the GAS is lower than 1.

### Table 2.

| DPYD Allele | Wild-Type | DPYD*2A | DPYD*13 | DPYD*HapB3 | c.2846A>T |
| --- | --- | --- | --- | --- | --- |
| Wild-type | GAS = 2 | GAS = 1 | GAS = 1 | GAS = 1.5 | GAS = 1.5 |
| DPYD*2A |   | GAS = 0 | GAS = 0 | GAS = 0.5 | GAS = 0.5 |
| DPYD*13 |   |   | GAS = 0 | GAS = 0.5 | GAS = 0.5 |
| DPYD*HapB3 |   |   |   | GAS = 1 | GAS = 1 |
| c.2846A>T |   |   |   |   | GAS = 1 |

Table 2 Caption: Phenotype translation from the considered DPYD allele combinations for fluoropyrimidine dose tailoring.

Also, both the DPWG and CPIC guidelines ([Table 3](#pharmaceutics-16-00956-t003)), recommend an alternative drug, if possible, in patients with a *DPYD* GAS lower than 1 (DPD PMs, in CPIC guidelines), and to start with 50% of the standard dose in patients with a GAS of 1 or 1.5 (DPD IMs). Also, the CPIC guideline reports that IM patients carrying the c.[2846A>T]/[2846A>T] genotype (GAS = 1) may require a dose reduction higher than 50%. The main difference between the CPIC and DPWG guidelines is the *DPYD* alleles considered for analysis. While the DPWG considers only the *DPYD**2A (defined by rs3918290), *DPYD**13 (defined by rs55886062), *DPYD**HapB3 (defined by rs56038477), and rs67376798 (also known as c.2846A>T), the CPIC guideline for FPs and *DPYD* provides a list with n = 83 variants, resulting in diplotypes translated into IM or PM (GAS < 2) phenotypes for reduced DPD activity and FP dose tailoring.

### Table 3.

| PGx Guideline | DPYD Variant | Phenotype | Recommendation |
| --- | --- | --- | --- |
| CPIC | *2Ac.2846A>T*13*HapB3Others ^ | GAS = 2 (NM) | Use the standard dose |
| GAS = 1.5 (IM) | 50% dose reduction followed by dose titration, based on clinical judgment and ideally on therapeutic drug monitoring |  |  |
| GAS = 1 (IM) | 50% dose reduction followed by dose titration, based on clinical judgment and ideally on therapeutic drug monitoring. In patients homozygous for c.2846A>T, a dose reduction of more than 50% may be required |  |  |
| GAS = 0.5 (PM) | Alternative drug. If no other therapeutic option is available, strongly reduce the dose with early therapeutic drug monitoring |  |  |
| GAS = 0 (PM) | Alternative drug |  |  |
| DPWG | *2Ac.2846A>T*13*HapB3 | GAS = 2 | Use the standard dose |
| GAS = 1.5 | Start with 50% of the standard dose or avoid fluorouracil and capecitabine. Adjust the subsequent doses guided by toxicity and effectiveness |  |  |
| GAS = 1 | Start with 50% of the standard dose or avoid fluorouracil and capecitabine. Adjust the subsequent doses guided by toxicity and effectiveness |  |  |
| GAS = 0 | Avoid fluorouracil and capecitabine. If not possible, determine DPD activity and adjust the dose |  |  |

Table 3 Caption: CPIC and DPWG PGx guidelines recommendations for DPYD/fluoropyrimidine drug–gene interactions.

### 1.3. Hypothesis and Objectives

The EMA, AEMPS, SEFF, SEOM, and the drug labels for capecitabine and 5-FU recommend *DPYD* genotyping to state the DPD metabolizing status before treatment starts with these drugs. Furthermore, there are available dosing guidelines based on PGx information, such as those from the DPWG [[19](#B19-pharmaceutics-16-00956)] and CPIC [[18](#B18-pharmaceutics-16-00956)].

We have implemented *DPYD* genotyping in our daily clinical practice before treatment starts with FPs. Despite this, many patients still show severe adverse drug events.

With this study, we aim to state the impact in our clinical routine of capecitabine and 5-FU dose tailoring based on *DPYD**2A (defined by rs3918290), *DPYD**13 (defined by rs55886062), *DPYD**HapB3 (defined by rs56038477), and rs67376798 (also known as c.2846A>T), studying its association with the treatment efficacy and toxicity.

Also, we aim to explain the toxicity events in the cohort of patients receiving the FP treatment tailored by *DPYD* genotype. In this regard, we studied the association with the FP toxicity and efficacy of other relevant *DPYD* variants, previously associated with FP response, but not recommended by the scientific societies and sanitary authorities.

Finally, we aimed to characterize the *DPYD* gene in our population, studying its genotypes and phenotypes distribution, the minor allele frequency (MAF), Hardy–Weinberg (H-W) equilibrium of included variants, and the possible linkage disequilibrium (LD).

In Spain, the Spanish Agency for Medicine and Health Products (*Agencia Española del Medicamento y Productos Sanitarios*, AEMPS), Spanish Society of Pharmacogenetics and Pharmacogenomics (*Sociedad Española de Farmacogenética y Farmacogenómica,* SEFF), and Spanish Society of Medical Oncology (*Sociedad Española de Oncología Médica*, SEOM), recommend genotyping the *DPYD**2A (defined by rs3918290), *DPYD**13 (defined by rs55886062), *DPYD**HapB3 (defined by rs56038477), and rs67376798 (also known as c.2846A>T) prior to FP treatment, avoiding its use in DPD PM patients (GAS < 1), and using a 50% dose reduction in DPD IM patients (1 < GAS < 2).

Since 2021, we have implemented the PGx test of *DPYD**2A (defined by rs3918290), *DPYD**13 (defined by rs55886062), *DPYD**HapB3 (defined by rs56038477), and rs67376798 (also known as c.2846A>T) in our hospital before capecitabine and fluorouracil treatment initiation, as well as dose tailoring based on guidelines from the DPWG. Despite this implementation in clinical practice and its acceptance by sanitary authorities and physicians in daily routine, we still have patients experiencing severe or life-threatening FP-related toxicities.

## 2. Materials and Methods

### 2.1. Study Design

The observational retrospective study including patients treated with capecitabine or fluorouracil and tested for *DPYD**2A (defined by rs3918290), *DPYD**13 (defined by rs55886062), *DPYD**HapB3 (defined by rs56038477), and rs67376798 (also known as c.2846A>T) before treatment began in our hospital between 1 March 2021 and 31 December 2021.

The inclusion criteria were patients diagnosed with cancer and prescribed capecitabine or fluorouracil, those tested for *DPYD**2A (defined by rs3918290), *DPYD**13 (defined by rs55886062), *DPYD**HapB3 (defined by rs56038477), and rs67376798, with a capecitabine or fluorouracil dose tailored based on DPWG guidelines, with an available 6-month follow-up period based on medical records, and non-previously treated with FPs. Patients who did not sign the informed consent form or who asked for to withdraw from the study were excluded.

The Research Ethics Committee of Granada approved the study (Code: 1605-N_22; date of approval: 14 September 2022) and the principles of the Declaration of Helsinki were followed.

The main endpoints were the toxicity and efficacy of capecitabine or 5-FU. These data was obtained from electronic medical records and confirmed by physicians in case of discrepancies.

The toxicity endpoint was adverse drug events (ADEs) to capecitabine or 5-FU. The causality and severity of ADEs were stated using the Liverpool Causality Assessment Tool (LCAT) [[20](#B20-pharmaceutics-16-00956)], and Common Terminology Criteria for Adverse Events (CTCAE) [[21](#B21-pharmaceutics-16-00956)], respectively. Only ADEs categorized as probable or definite and showing a severity grade 3 or higher were considered for the study.

The efficacy endpoint was achieved if patients received a positive clinical assessment from the oncologist regarding the progression of the illness, as recorded in the medical records, and the non-discontinuation of FP treatment during follow-up if it was not because of an ADE. For the positive clinical assessment of metastatic patients by the oncologists, the Response Evaluation Criteria in Solid Tumors (RECIST 1.1) measure was used within 6 months of initiating therapy [[22](#B22-pharmaceutics-16-00956)]. Radiological and imaging techniques, such as computed tomography (CT), were employed to monitor the evolution of both target and non-target lesions in treated patients. These assessments were conducted every 3 to 4 months (1–2 times during follow-up), unless patients exhibited clinical signs of progression earlier (e.g., pain, dyspnea, sweating, elevated tumor markers in blood tests).

### 2.2. Procedures for the Inclusion of DPYD Variants in the Study

To include candidate *DPYD* variants as explanatory factors of remaining toxicity events in our study population, who were receiving dose-tailored FP treatment based on *DPYD**2A (defined by rs3918290), *DPYD**13 (defined by rs55886062), *DPYD**HapB3 (defined by rs56038477), and rs67376798, we searched in PharmGKB [[14](#B14-pharmaceutics-16-00956)] for genetic variants of *DPYD* reported as clinical annotations with the highest level of evidence (1A) concerning their association with any phenotype for capecitabine, 5-FU, or tegafur ([Table 1](#pharmaceutics-16-00956-t001)). We included those genetic variants related to the toxicity of these drugs with MAF values higher than 1% in the Iberian Peninsula population according to the 1000 Genomes project [[15](#B15-pharmaceutics-16-00956)].

### 2.3. Management of Patients

As part of the clinical practice at our hospital, whenever a doctor considers prescribing capecitabine or 5-FU in patients diagnosed with whatever cancer, they may request a *DPYD* PGx test from the hospital pharmacy.

Once the request for the test is received, a nurse takes a saliva sample with sterile cotton swabs, the DNA is extracted, and *DPYD**2A (defined by rs3918290), *DPYD**13 (defined by rs55886062), *DPYD**HapB3 (defined by rs56038477), and rs67376798 are tested less than 48 h after the saliva sample collection. The pharmacists upload a PGx report in the electronic medical records, including the therapeutic recommendation translated from the PGx result, within 72 h of sample collection.

The genotype–phenotype–therapeutic recommendation translation process (dose tailoring) is performed following the instructions in the DPWG guidelines and is shown in [Table 2](#pharmaceutics-16-00956-t002) and [Table 3](#pharmaceutics-16-00956-t003). This means that patients not carrying any of the tested variants are assigned a *DPYD* GAS = 2 and treated with the standard dose. Those carrying a single mutated allele are assigned GAS = 1–1.5 and start the treatment with a 50% reduction in the standard dose. Patients with two mutated alleles are assigned GAS = 0, and the treatment is switched.

The remaining DNA and saliva samples are stored as a private biosamples collection registered with the Carlos III Health Institute (C.0007322). Once the follow-up of patients was finished, we retrospectively tested, using this remaining DNA, the *DPYD* variants meeting the criteria that are explained in [Section 2.2](#sec2dot2-pharmaceutics-16-00956).

Those patients not treated with capecitabine or fluorouracil after the PGx test were excluded from the analysis. The included patients were followed up for six months.

### 2.4. Data Management, Statistical Analysis, and Genotyping

First, a descriptive analysis of the population included in the study was performed. The impact and usefulness of the *DPYD* PGx test for capecitabine and 5-FU dose tailoring in our clinical practice was assessed by studying the association with the toxicity and efficacy endpoints of the dose tailoring based on *DPYD**2A (defined by rs3918290), *DPYD**13 (defined by rs55886062), *DPYD**HapB3 (defined by rs56038477), and rs67376798.

After that, to explain ADEs that occurred during follow-up in patients receiving an FP-*DPYD*-dose tailored treatment, we performed a genotype association study with the toxicity and efficacy endpoint.

Moreover, we carried out a multivariate analysis to discard possible confounding factors on the association of genetic variants with the toxicity and efficacy endpoints.

Finally, we characterized the *DPYD* variants in our population, including both those used in our daily clinical practice and those studied for their association with the toxicity endpoint after PGx FP dose tailoring. In this regard, the distribution (number, n, and percentage, %) of genotypes, phenotypes, and MAFs were calculated, an LD analysis was carried out, and the H-W equilibrium was tested.

The descriptive analysis, MAFs, genotype/phenotype distribution, and multivariate analysis were conducted using R commander. For the association studies of genetic variants with the endpoints, LD analysis, and H–W equilibrium analyses, we used the SNPstats online tool [[23](#B23-pharmaceutics-16-00956)].

For the multivariate analysis, we considered for inclusion all the study variables recorded, including clinical parameters, genetic variants, and concomitant treatments. We built different multivariate models using the backward, forward, and stepwise methods for the association with the efficacy and toxicity endpoints. These models were upgraded and compared using the Akaike information criterion (AIC). The final model showing the lower AIC was chosen.

The chi-square test or Fisher exact test were used, and odds ratios (OR) and *p*-values were calculated, considering *p* < 0.05 to be statistically significant.

### 2.5. DNA Extraction and Genotyping

For genotyping, DNA was isolated from saliva samples using standard procedures. DNA extraction was carried out following the method by Freeman et al. [[24](#B24-pharmaceutics-16-00956)], a non-organic (proteinase K and salting out) protocol including modifications from the method described by Gomez-Martín A. et al. [[25](#B25-pharmaceutics-16-00956)]. The included genetic variants were genotyped using Taqman assay technology (Thermo Fisher Scientific, Waltham, MA, USA) and analyzed with QuantStudio 12K Flex de Applied Biosystems (Foster City, CA, USA).

## 3. Results

Between Mar/01/2021 and Dec/31/2021, N = 569 patients were prescribed capecitabine or 5-FU at the *Hospital Universitario Clínico San Cecilio* (Granada, Spain). In total, n = 190 *DPYD* PGx tests were requested, and n = 167 patients were finally treated with FPs ([Figure 2](#pharmaceutics-16-00956-f002)). This means n = 402 patients were prescribed capecitabine or 5-FU in our hospital without being *DPYD* tested, and n = 23 were *DPYD* tested but not treated with FPs.

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e1af/11280107/8c340243e934/pharmaceutics-16-00956-g002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=11280107_pharmaceutics-16-00956-g002.jpg)

Recruitment and management of patients. PGx: pharmacogenetic; NM: normal metabolizer; IM: intermediate metabolizer; PM: poor metabolizer; GAS: Gene Activity Score; ADE: adverse drug event.

Among the n = 167 *DPYD*-tested patients finally treated with capecitabine or 5-FU, we found n = 161 *DPYD**1/* 1 (wild-type) patients, translated into a GAS = 2 phenotype receiving normal doses of FPs. We also found n = 1 *DPYD**1/*13, n = 3 *1/*HapB3, and n = 2 *1/c.2846A>T genotypes, translated into n = 5 GAS = 1.5, and n = 1 GAS = 1, who were recommended to be treated with 50% of the standard dose. All these patients were dose-adjusted based on our recommendation ([Figure 2](#pharmaceutics-16-00956-f002)). Among those n = 23 non-treated patients, we found n = 21 *DPYD**1/*1 (wild-type) and n = 2 *1/*HapB3.

All the n = 167 FP-treated patients after dose tailoring based on *DPYD* genotyping were prescribed treatment because of digestive tumors, including colon (n = 81), stomach (n = 13), pancreas (n = 10), duodenum (n = 1), esophageal (n = 1), and rectal cancer (n = 45), although breast cancer (n = 12) and other kind of tumors (n = 4) affected some patients. All the patients received capecitabine or 5-FU as a first-line treatment, except the n = 12 breast cancer patients who were prescribed it after cytotoxic chemotherapy failure and a locally advanced tumor or metastasis. The mean age was 64.26 ± 10.89 years old, and 37.13% were women ([Table 4](#pharmaceutics-16-00956-t004)).

### Table 4.

| Parameter | N = 167n (%) or Mean ± sd |
| --- | --- |
| Women | 62 (37.13) |
| Age | 64.26 ± 10.89 |
| BMI | 26.69 ± 4.92 |
| BS | 1.80 ± 0.19 |
| Ethnicity (European) | 167 (100) |
| Tumor Location |   |
| Colorectal | 126 (75.45) |
| Gastric | 13 (7.78) |
| Pancreas | 10 (5.99) |
| Breast | 12 (7.19) |
| Others | 6 (3.59) |
| Tumor stage |   |
| I | 3 (1.80) |
| II | 29 (17.36) |
| III | 64 (38.32) |
| IV | 71 (42.51) |
| Chemotherapy treatment |  |
| Capecitabine (monotherapy) | 72 (43.11) |
| XELOX | 39 (23.35) |
| FOLFOX | 40 (23.95) |
| FLOT | 6 (3.59) |
| FOLFIRINOX | 10 (5.99) |
| Initial doses (1000 mg/m2)% |   |
| 100 | 143 (85.63) |
| <100 and >50 | 18 (10.78) |
| 50 | 6 (3.59) |
| Associated antibody | 16 (9.58) |
| Toxicity endpoint | 48 (28.74) |
| Efficacy endpoint | 127 (71.86) |
| DPYD genotype |  |
| DPYD*1/*1 | 161 (96.40) |
| DPYD*1/*HapB3 | 3 (1.80) |
| DPYD*1/rs67376798 | 2 (1.20) |
| DPYD*1/*13 | 1 (0.60) |
| DPD phenotype |  |
| NM (GAS: 2) | 161 (96.40) |
| IM (GAS: 1.5 or GAS:1) | 6 (3.60) |
| PM (GAS: 0) | 0 (0.00) |

Table 4 Caption: Baseline characteristics of fluoropyrimidine-treated patients.

### 3.1. Association Study of DPYD Variants with the Response

#### 3.1.1. Association with Response of Dose Tailoring Based on DPYD

We found no association between the *DPYD* variants used for FP dose tailoring and the toxicity endpoint. We also found no association with the efficacy endpoint ([Table 5](#pharmaceutics-16-00956-t005)). This means that carrying a genotype translated into a GAS lower than 2 (DPD IM or PM) and receiving adjusted doses is not related to variable response to FPs.

|   |  | ADE Severity ≥ 3 |  | OR (95% CI) | p-Value |
| --- | --- | --- | --- | --- | --- |
| Yes n (%) | NO n (%) |  |  |  |  |
| DPYD GAS < 2 | YES n (%) | 2 (4.3) | 4 (3.3) | 1.29 (0.11–9.34) | 0.674 |
| NO n (%) | 45 (95.7) | 116 (96.7) |  |  |  |
|   |  | Efficacy |  | OR (95% CI) | p-Value |
| Yes n (%) | NO n (%) |  |  |  |  |
| DPYD GAS < 2 | YES n (%) | 3 (2.3) | 3 (7.9) | 0.28 (0.04–2.19) | 0.132 |
| NO n (%) | 126 (97.7) | 35 (92.1) |  |  |  |

Table Caption: Association study of DPYD variants used for fluoropyrimidine dose tailoring with the response.

#### 3.1.2. Association with the Response of New DPYD Variants

Among treated patients based on *DPYD**2A (defined by rs3918290), *DPYD**13 (defined by rs55886062), *DPYD**HapB3 (defined by rs56038477), and rs67376798, we still found n = 47 patients meeting the toxicity endpoint.

As commented above, we retrospectively studied n = 6 *DPYD* variants that had been related to the toxicity of FPs and with an MAF higher than 1% in the Iberian Peninsula population according to the 1000 Genomes project.

We found that carrying the *DPYD* rs1801158 is associated with a higher risk of ADEs with a severity grade ≥ 3 (OR = 5.66; 95% C.I. = 1.35–23.67; *p* = 0.014) ([Table 6](#pharmaceutics-16-00956-t006)). No other *DPYD* variant was associated with the toxicity endpoint.

|   |  | ADE Severity ≥ 3 |  | OR (95% CI) | p-Value |
| --- | --- | --- | --- | --- | --- |
| Yes n (%) | NO n (%) |  |  |  |  |
| DPYD rs1801265 | A/G–G/G | 15 (31.9) | 38 (31.9) | 1.00 (0.48–2.06) | 1 |
| A/A | 32 (68.1) | 81 (68.1) |  |  |  |
| DPYD rs17376848 | A/G | 0 (0) | 6 (5) | 0.00 (0–NA) | 0.043 |
| A/A | 47 (100) | 113 (95) |  |  |  |
| DPYD rs1801159 | C/T–T/T | 18 (38.3) | 53 (44.5) | 0.77 (0.39–1.54) | 0.46 |
| C/C | 29 (61.7) | 66 (55.5) |  |  |  |
| DPYD rs1801160 | C/T | 6 (12.8) | 15 (12.6) | 1.01 (0.37–2.80) | 0.98 |
| C/C | 41 (87.2) | 104 (87.4) |  |  |  |
| DPYD rs1801158 | C/T | 6 (12.8) | 3 (2.5) | 5.66 (1.35–23.67) | 0.014 |
| C/C | 41 (87.2) | 116 (97.5) |  |  |  |
| DPYD rs2297595 | C/T | 11 (23.4) | 24 (20.2) | 1.21 (0.54–2.72) | 0.65 |
| C/C | 36 (76.6) | 95 (79.8) |  |  |  |
|   |  | Efficacy |  | OR (95% CI) | p-Value |
| Yes n (%) | NO n (%) |  |  |  |  |
| DPYD rs1801265 | A/G–G/G | 41 (31.8) | 12 (32.4) | 0.97 (0.44–2.12) | 0.94 |
| A/A | 88 (68.2) | 25 (67.6) |  |  |  |
| DPYD rs17376848 | A/G | 6 (4.7) | 0 (0) | NA (0.00-NA) | 0.079 |
| A/A | 123 (95.3) | 37 (100) |  |  |  |
| DPYD rs1801159 | C/T–T/T | 56 (43.4) | 15 (40.5) | 1.13 (0.54–2.37) | 0.76 |
| C/C | 73 (56.6) | 22 (59.5) |  |  |  |
| DPYD rs1801160 | C/T | 16 (12.4) | 5 (13.5) | 0.91 (0.31–2.66) | 0.86 |
| C/C | 113 (87.6) | 32 (86.5) |  |  |  |
| DPYD rs1801158 | C/T | 6 (4.7) | 3 (8.1) | 0.55 (0.13–2.33) | 0.43 |
| C/C | 123 (95.3) | 34 (91.9) |  |  |  |
| DPYD rs2297595 | C/T | 28 (21.7) | 7 (18.9) | 1.19 (0.47–2.99) | 0.71 |
| C/C | 101 (78.3) | 30 (81.1) |  |  |  |

Table Caption: Association study with the efficacy/toxicity endpoints of DPYD variants not used for FP dose tailoring.

We found no association of any of the included SNPs with the efficacy endpoint.

#### 3.1.3. Multivariate Analysis

The following variables were included as possible explanatory parameters of the efficacy and toxicity endpoints in the multivariate analysis: clinical variables, including age, sex, and body surface area; concomitant drugs in the therapeutic scheme, including monoclonal antibodies, oxaliplatin, irinotecan, and radiotherapy; and all the *DPYD* genetic variants included in the study, i.e., those used and not used for FP dose tailoring.

For the multivariate analysis to explain the toxicity endpoint, the model using the stepwise (backward/forward) method showed the lower AIC, and this model was finally included. After adjustment, the model showed that the *DPYD* rs1801158 is associated with ADEs (severity grade ≥ 3) for FPs (OR = 5.73; 95% CI = 1.41–28.77; *p* = 0.019) in *DPYD* dose-tailored patients based on *DPYD**2A (defined by rs3918290), *DPYD**13 (defined by rs55886062), *DPYD**HapB3 (defined by rs56038477), and rs67376798 ([Table 7](#pharmaceutics-16-00956-t007)).

| Toxicity Endpoint |  |  |
| --- | --- | --- |
| Variable | OR (95% CI) | p-Value |
| DPYD rs1801158 (CT or TT) | 5.73 (1.41–28.77) | 0.019 |
| Irinotecan | 2.32 (0.92–5.81) | 0.071 |
| Age | NA | 0.090 |
| DPYD rs56038477 (CT or TT) | 6.99 (0.64–155.45) | 0.120 |
| Monoclonal antibody | 0.00 (NA–Inf) | 0.990 |
| DPYD rs17376848 (AG or GG) | 0.00 (NA–Inf) | 0.992 |
| Efficacy Endpoint |  |  |
| Variable | OR (95% CI) | p-Value |
| Irinotecan | 0.15 (0.06–0.35) | <0.001 |
| DPYD rs56038477 (CT or TT) | 0.096 (0.01–1.05) | 0.062 |
| Radiotherapy | 2.31 (0.79–8.48) | 0.16 |

Table Caption: Association study with the toxicity and efficacy endpoint of study variables included in the multivariate model after adjustment.

In the multivariate analysis to explain the efficacy endpoint we found that the concomitant treatment with irinotecan is associated with lower rates of efficacy (*p* ≤ 0.001).

### 3.2. DPYD Characterization

In total, n = 190 patients were requested to be tested and dose-tailored based on *DPYD**2A (defined by rs3918290), *DPYD**13 (defined by rs55886062), *DPYD**HapB3 (defined by rs56038477), and rs67376798. Among them, n = 189 were tested for *DPYD* variants considered candidates to explain differences in the response to FPs. This means that n = 1 patient could not be tested for these variants because we did not have enough stored DNA.

Among all the tested variants in our population, we found no differences with the MAF for the Iberian Peninsula population reported by the 1000 Genomes Project [[15](#B15-pharmaceutics-16-00956)], and all the SNPs were in the Hardy–Weinberg equilibrium ([Table 8](#pharmaceutics-16-00956-t008)).

### Table 8.

| *Allele | DPYDVariant | Major Nucleotide Variation | GenotypeN = 190 ^ |  |  | H-W | MAF | MAF Ibs | Comparison with 1000 Genomes |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Wt | Het | Hom |  |  |  |  |  |  |  |
| - | rs17376848 | c.1896T>C | 181 (95.77) | 8 (4.23) | 0 (0) | 1 | 0.021 | 98/2 | 1 |
| *4 | rs1801158 | c.1601G>A | 177 (93.65) | 12 (6.35) | 0 (0) | 1 | 0.032 | 94/6 | 0.149 |
| *5 | rs1801159 | c.1627A>G | 109 (57.67) | 69 (36.51) | 11 (5.82) | 1 | 0.241 | 79/21 | 0.475 |
| *6 | rs1801160 | c.2194G>A | 168 (88.89) | 21 (11.11) | 0 (0) | 1 | 0.056 | 95/5 | 0.643 |
| *9A | rs1801265 | c.85T>C | 128 (67.73) | 58 (30.69) | 3 (1.59) | 0.3 | 0.169 | 79/21 | 0.170 |
| - | rs2297595 | c.496A>G | 152 (80.42) | 37 (10.58) | 0 (0) | 0.2 | 0.098 | 88/12 | 0.470 |
| *2A | rs3918290 | c.1905+1G>A | 190 (100) | 0 (0) | 0 (0) | - | 0 | 100/0 | 1 |
| *13 | rs55886062 | c.1679T>G | 189 (99.47) | 1 (0.53) | 0 (0) | 1 | 0.003 | 100/0 | 1 |
| *HapB3 | rs56038477 | c.1236G>A | 185 (97.37) | 5 (2.63) | 0 (0) | 1 | 0.013 | 98/2 | 0.729 |
| - | rs67376798 | c.2846A>T | 188 (98.95) | 2 (1.05) | 0 (0) | 1 | 0.011 | 100/0 | 0.538 |

Table 8 Caption: DPYD characterization in our population.

We found many significant differences in the LD analysis. The *DPYD* rs1801265 was linked to the rs56038477 (r = 0.214; *p* < 0.001) and the rs2297595 (r = 0.604; *p* < 0.001) in our population ([Table 9](#pharmaceutics-16-00956-t009)). In more detail, we found that all the patients carrying rs56038477 also carried rs1801265, and n = 31 (83.8%) of patients carrying rs2297595 also carried rs1801265. We found other *p*-values lower than 0.05 in the LD analyses but showing r values close to r = 1.

### Table 9.

| Linkage Disequilibrium |  |  |  |  |  |  |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| n = 190 ^ |  |  |  |  |  |  |  |  |  |
|   | rs1801265 (*9A) | rs17376848 | rs1801159 (*5) | rs1801160 (*6) | rs1801158 (*4) | rs2297595 | rs55886062 (*13) | rs67376798 | rs56038477 (*HapB3) |
| rs1801265 (*9A) | - | 0.9463 | 0.6052 | 0.0369 | 0.1844 | 0 | 0.0272 | 0.5667 | 1 × 10−4 |
| 2 × 10−4 | −0.0045 | 0.0103 | −0.0044 | 0.0684 | 0.0024 | −9 × 10−4 | 0.0074 |  |  |
| 0.012 | 0.1142 | 0.1943 | 0.9901 | 0.7748 | 0.9751 | 0.9191 | 0.9903 |  |  |
| 0.0037 | −0.0284 | 0.1145 | −0.0728 | 0.6035 | 0.1212 | −0.0314 | 0.2139 |  |  |
| rs17376848 | - | - | 0.8753 | 0.5391 | 0.7111 | 0.6804 | 0.9834 | 0.9584 | 0.897 |
| 5 × 10−4 | −0.0011 | −4 × 10−4 | −9 × 10−4 | 0 | 0 | −1 × 10−4 |  |  |  |
| 0.0358 | 0.9562 | 0.8977 | 0.4854 | 0.0028 | 0.2721 | 0.5501 |  |  |  |
| 0.0086 | −0.0337 | −0.0203 | −0.0226 | 0.0011 | −0.0029 | −0.0071 |  |  |  |
| rs1801159 (*5) | - | - | - | 0.9568 | 0.0887 | 0.0141 | 0.0848 | 0.9315 | 0.2636 |
| −3 × 10−4 | −0.0065 | −0.0177 | 0.0022 | 2 × 10−4 | 0.0025 |  |  |  |  |
| 0.0203 | 0.9933 | 0.6966 | 0.9725 | 0.0342 | 0.3631 |  |  |  |  |
| −0.003 | −0.0934 | −0.1347 | 0.0946 | 0.0047 | 0.0613 |  |  |  |  |
| rs1801160 (*6) | - | - | - | - | 0.4411 | 0.6994 | 0.8619 | 0.0191 | 0.1098 |
| 0.0017 | 0.0016 | −1 × 10−4 | 0.0024 | 0.002 |  |  |  |  |  |
| 0.9746 | 0.2376 | 0.6704 | 0.4308 | 0.2396 |  |  |  |  |  |
| −0.0423 | −0.0212 | −0.0095 | 0.1287 | 0.0878 |  |  |  |  |  |
| rs1801158 (*4) | - | - | - | - | - | 0.3038 | 0.9693 | 0.9033 | 0.8394 |
| −0.0028 | 0 | −1 × 10−4 | −2 × 10−4 |  |  |  |  |  |  |
| 0.9848 | 0.231 | 0.5147 | 0.7001 |  |  |  |  |  |  |
| −0.0564 | −0.0021 | −0.0067 | −0.0111 |  |  |  |  |  |  |
| rs2297595 | - | - | - | - | - | - | 0.7833 | 0.6709 | 0.4257 |
| −3 × 10−4 | −6 × 10−4 | 0.0013 |  |  |  |  |  |  |  |
| 0.8023 | 0.8752 | 0.1577 |  |  |  |  |  |  |  |
| −0.0151 | −0.0233 | 0.0437 |  |  |  |  |  |  |  |
| rs55886062 (*13) | - | - | - | - | - | - | - | 0.8471 | 0.8998 |
| 0 | 0 |  |  |  |  |  |  |  |  |
| 0.0149 | 0.012 |  |  |  |  |  |  |  |  |
| 0.0106 | 0.0069 |  |  |  |  |  |  |  |  |
| rs67376798 | - | - | - | - | - | - | - | - | 0.95 |
| 0 |  |  |  |  |  |  |  |  |  |
| 0.0042 |  |  |  |  |  |  |  |  |  |
| 0.0034 |  |  |  |  |  |  |  |  |  |
| rs56038477 (*HapB3) | - | - | - | - | - | - | - | - | p-value |
| D |  |  |  |  |  |  |  |  |  |
| D’ |  |  |  |  |  |  |  |  |  |
| r |  |  |  |  |  |  |  |  |  |

Table 9 Caption: Linkage disequilibrium analysis.

## 4. Discussion

FPs, including 5-FU and the oral prodrug capecitabine, are commonly prescribed antimetabolite chemotherapies utilized across many cancer streams.

Among FP-treated patients with standard doses, up to 30% show severe (grade ≥ 3) treatment-related toxicity. Many genetic variants in the *DPYD* gene encoding the DPD enzyme partially explained these toxicities. The *DPYD**2A (defined by rs3918290), *13 (defined by rs55886062), *HapB3 (defined by rs56038477), alleles, and *DPYD* rs67376798 had shown the highest level of evidence about their association with FPs response. The EMA- and FDA-approved drug labels for FPs, and the SEFF and SEOM in Spain, recommend genotyping these variants before treatment starts.

Depending on *DPYD* genotype, patients may be categorized as DPD NM, IM, or PM and receive a PGx dose-tailored treatment (50% of standard doses in DPD IM patients, and alternative therapies in DPD PMs).

Moreover, different studies concluded that dose adjustments based on *DPYD* genotype do not influence the treatment efficacy. Deenen M.J. et al. did not find a relationship between *DPYD* variants and progression-free survival or overall survival despite a 50% dose reduction in *DPYD**2A carriers [[26](#B26-pharmaceutics-16-00956)], and Lam S.W. et al. [[27](#B27-pharmaceutics-16-00956)] observed no differences in response in seven more clinical studies examining *DPYD* polymorphisms with a dose reduction, time to progression, progression-free survival, and/or overall survival, concluding that there is no evidence that a priori dose adjustments for *DPYD* carriers decrease FP efficacy, and low-activity variant carriers treated with standard of care appear to have similar efficacy once an acceptable dose is found.

We implemented *DPYD* genotyping in our daily clinical routine, but we still find patients showing severe toxicities to FPs. Thus, we hypothesized that there might be other variants influencing the FP-related toxicities.

We identified six *DPYD* variants (rs1801265, rs17376848, rs1801159, rs1801160, rs1801158, and rs2297595) as explanatory candidates of the interindividual differences for the FP-related toxicities, since these had been related to the toxicity of FPs with the highest level of evidence, and they have an MAF higher than 1% in the Iberian Peninsula population.

In this study, we assessed the association with response to FPs of these novel candidate variants to explain suboptimal patient response for the first time in a cohort that received FP treatment based on *DPYD**2A (defined by rs3918290), *13 (defined by rs55886062), *HapB3 (defined by rs56038477), and rs67376798.

This way, we could determine whether these new *DPYD* variants explain the remaining toxicities despite a PGx dose-tailored treatment and whether they would be potentially useful in daily clinical practice.

This study may also be used as a guide for the clinical implementation of the FP-*DPYD* drug–gene interaction.

### 4.1. Association of Genetic Variants with Response to Fluoropyrimidines

In the study of the association of FP dose tailoring according to the *DPYD* genotype with the toxicity and efficacy endpoints, we found no significant differences ([Table 5](#pharmaceutics-16-00956-t005)). These results make sense, since significant differences would have meant an underestimation of dose modifications resulting from the presence of the *DPYD**2A (defined by rs3918290), *DPYD**13 (defined by rs55886062), *DPYD**HapB3 (defined by rs56038477), and rs67376798 variants.

In the association study with the toxicity and efficacy endpoints of the new variants, which are not currently being used to guide treatment with FPs in our population, we have observed the following.

The *DPYD**4 (defined by rs1801158) allele is associated with a higher risk of severe ADEs (severity grade ≥ 3) in both the univariate (OR = 5.66; 95% CI = 1.35–23.67; *p* = 0.014) and multivariate analyses (OR= 5.73; 95% CI= 1.41–28.77; *p* = 0.019) after adjusting the model.

This variant had been previously associated with FP response. The *DPYD**4 CT genotype was associated with decreased catalytic activity of DPD [[28](#B28-pharmaceutics-16-00956)], and an increased risk of drug toxicity when treated with capecitabine or fluorouracil in colorectal cancer patients [[29](#B29-pharmaceutics-16-00956)], as compared to CC genotype. On the other hand, many other studies showed contradictory results [[13](#B13-pharmaceutics-16-00956)] in this regard. An interesting study by André B. P. van Kuilenburg et al. [[30](#B30-pharmaceutics-16-00956)] found that *DPYD**4 allele T is associated with decreased activity of DPD when expressed in mammalian cells (HEK293 Flp-In) as compared to allele C, while highlighting the conflicting data about this association, as they found no association when DPD activity was assessed within a healthy cohort of n = 100 individuals.

These findings, and the results described below, reveal the need for further studies, especially considering the expression of genes, and not just categorizing patients as carriers/non-carriers of single or combined *DPYD* variants.

We found an association of chemotherapy schemes including irinotecan with a lower efficacy in the multivariate analysis ([Table 7](#pharmaceutics-16-00956-t007)). We observed that the n = 10 patients treated with FOLFIRINOX (including irinotecan) are the same n = 10 patients with pancreatic cancer, with the worse baseline condition and prognosis among recruited patients.

Also, regarding our results, no other variants were significantly associated with either toxicity or efficacy endpoints. The *DPYD* rs17376848 showed a confounding association with the response. According to the significance criteria based on *p*-values, carriers of this variant (Genotype AG or GG vs. AA) were associated with a lower risk of the toxicity endpoint (*p* = 0.043) and a certain trend toward the efficacy of FPs (*p* = 0.079). However, upon closer examination of these results, we find that only n = 6 patients carry the *DPYD* rs17376848 variant, and none of them experienced an ADE (severity grade ≥ 3) during follow-up, preventing us from confirming this association. Furthermore, this statistical significance is lost in the multivariate analysis (*p* = 0.992; [Table 7](#pharmaceutics-16-00956-t007)) when considering the influence of concomitant treatments, other clinical variables, and interactions with other *DPYD* variants.

As happens with *DPYD**4 (defined by rs1801158), those other five variants suggested as candidate variants to explain the remaining toxicity events in our population showed inconclusive results in previous studies [[13](#B13-pharmaceutics-16-00956),[31](#B31-pharmaceutics-16-00956)]. In any case, further studies with larger cohorts and treatment guidance based on these variants are necessary to confirm their utility or lack of influence in clinical practice.

Anyway, despite of the baseline characteristics of recruited patients, stage of the tumor, different fluoropyrimidines schemes, interactions between clinical and PGx variables, and receiving a PGx dose tailored treatment, the *DPYD**4 (defined by rs1801158) showed an association with a higher risk of severe ADEs with FPs in our population. This makes it a candidate variant for potential implementation in clinical practice.

### 4.2. Insights on Clinical Practice

We have observed that in the clinical practice of our hospital, there is an important association between the profile of the health professional and the degree of acceptance and requests for PGx tests. As also happened to us in previous studies [[32](#B32-pharmaceutics-16-00956)], there is a bias between the hospital departments prescribing the drugs and those requesting the PGx tests. In this case, we have observed that 96.3% (n = 183) of the n = 190 requests for the *DPYD* test were made by the medical oncology department, and n = 7 (3.7%) were made by other units of our hospital. Furthermore, we observed that, of the n = 170 patients who were prescribed capecitabine or 5-FU in the medical oncology unit during the time of recruitment, n = 167 (98.2%) received the treatment as guided by the PGx test. On the other hand, n = 402 patients in total in our hospital received treatment with FPs without having been genotyped for the variants recommended on the drug label by the EMA, FDA, SEFF, and SEOM. Among these, only n = 3 were treated by doctors assigned to the medical oncology service, and n = 399 were treated by other units.

As we can see, despite the level of evidence regarding the *DPYD*–FP interaction and the recommendations on the drug label by health authorities and scientific societies, the degree of implementation of these tests is strongly linked to the profile of the healthcare professional and the internal procedures or protocols of their hospital department.

On the other hand, it is true that until a few months ago, PGx tests, including the *DPYD*–FP interaction described in this study, had not been included in the portfolio of common services of the national health system in Spain, which is the set of health procedures that must be available to any citizen. The recent update of the service portfolio institutionally supports the performance of PGx tests and invites health professionals to perform them.

Another aspect that we observed in our results is that the level of acceptance of the therapeutic recommendations that emerge from the PGx results is total, and that dose reductions do not translate into a decrease in the treatment’s efficacy. Furthermore, based on our results and, as mentioned above, *DPYD* rs1801158 could be implemented in clinical practice. This variant has a relatively high frequency (MAF = 0.032; 6% carriers) and was the only genetic or clinical variant that was associated with an increased risk of toxicity secondary to FPs in the univariate and multivariate analyses.

The *DPYD*–FP interaction, regardless of many contradictory results, has been demonstrated to be useful in clinical practice in different studies. Furthermore, recent studies have demonstrated the cost-effectiveness of PGx tests [[33](#B33-pharmaceutics-16-00956),[34](#B34-pharmaceutics-16-00956)]. The potential benefit, actual related costs, and support by sanitary authorities should be enough reasons for the large-scale implementation of PGx tests, especially the *DPYD*–FP interaction.

### 4.3. Limitations

This is not a comparative clinical trial; rather, it is an observational cohort study. We did not consider all the variants in the *DPYD* gene. However, for the recruited cohort, it was not useful to study further variants. In fact, we examined all variants with a MAG higher than 1% in our population. Rare variants do not make sense in a cohort of n = 167 patients.

Nevertheless, it is necessary to investigate rare variants in larger cohorts to study the different interactions between them in detail. In particular, conducting an association study between *DPYD* haplotypes and their response to FPs would be especially interesting. Also, studying the epigenetics of the *DPYD* gene as a key gene in the PGx of FPs would be also interesting. This might elucidate discrepancies between association studies of *DPYD* variants with their FP response.

We did not study all the clinical variables that influence the response to FPs, especially the baseline condition of recruited patients, as commented above. Furthermore, this study is based on real-world data obtained from daily clinical practice, and we are limited by the reliable information collected in the patients’ medical records.

In the same regard, we recruited a wide range of different patients, including metastatic/adjuvant-treated patients, different stages of tumors, chemotherapy schemes, etc. Anyway, the aim was to perform the study considering real-world data based on our daily clinical practice, and we found significant differences concerning the influence of the *DPYD**4 alle on FP toxicities.

Also, considering the inclusion criteria of candidate *DPYD* variants as explanatory factors of FP-related toxicities, we should study the *DPYD* rs75017182 variant ([Table 1](#pharmaceutics-16-00956-t001)). It was not genotyped because it has a lower MAF thanthe *DPYD* rs56038477,and both variants charactere the *DPYD**HapB3.

## 5. Conclusions

Based on our results, we concluded that FP dose lowering based on the *DPYD* genotype does not affect the treatment efficacy. *DPYD**4 (defined by the rs1801158) is associated with FP toxicity in patients receiving a PGx dose-tailored treatment based on *DPYD**2A (defined by rs3918290), *DPYD**13 (defined by rs55886062), *DPYD**HapB3 (defined by rs56038477), and rs67376798. Based on this, *DPYD**4 (defined by the rs1801158) is an explanatory factor of remaining ADEs among FP-treated and PGx dose-tailored patients, and its genotyping should be implemented in daily clinical practice.

## Acknowledgments

We want to kindly thank patients participating in the study and sanitary personnel of the Hospital Universitario San Cecilio collaborating in the recruitment of patients.

## Author Contributions

X.D.-V. and M.M.-P.: conceptualization, methodology, visualization, formal analysis, writing—original draft; E.F.-V. and A.J.I.: methodology, data curation, formal analysis, writing—review and editing; M.T.N.-S., G.R.-T., and A.T.-G.: software, investigation, resources; B.G.A. and I.B.: validation, data curation, writing—review and editing. J.C.-B.: supervision, project administration; R.M.: conceptualization, methodology, visualization, formal analysis, writing—original draft. All authors have read and agreed to the published version of the manuscript.

## Institutional Review Board Statement

The study was conducted in accordance with the Declaration of Helsinki and approved by the Research Ethics Committee of Granada (protocol code 1605-N_22 and date of approval: 14 September 2022).

## Informed Consent Statement

Informed consent was obtained from all subjects involved in the study.

## Data Availability Statement

The data presented in this study are available on request from the corresponding author. The data are not publicly available due to containing clinical and personal information.

## Conflicts of Interest

The authors declare no conflicts of interest. The funders had no role in the design of the study; in the collection, analyses, or interpretation of data; in the writing of the manuscript; or in the decision to publish the results.

## Funding Statement

This research was funded by the “*Fundación Andaluza de Farmacia Hospitalaria*”, grant number 74/2024.

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Data Availability Statement

The data presented in this study are available on request from the corresponding author. The data are not publicly available due to containing clinical and personal information.

### Data Availability Statement

The data presented in this study are available on request from the corresponding author. The data are not publicly available due to containing clinical and personal information.

## References

1. Mikhail S.E., Sun J.F., Marshall J.L. Safety of capecitabine: A review. Expert. Opin. Drug Saf. 2010;9:831–841. doi: 10.1517/14740338.2010.511610.  [DOI](https://doi.org/10.1517/14740338.2010.511610) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20722491/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Expert.%20Opin.%20Drug%20Saf.&title=Safety%20of%20capecitabine:%20A%20review&author=S.E.%20Mikhail&author=J.F.%20Sun&author=J.L.%20Marshall&volume=9&publication_year=2010&pages=831-841&pmid=20722491&doi=10.1517/14740338.2010.511610&)

2. Henricks L.M., Lunenburg C.A.T.C., de Man F.M., Meulendijks D., Frederix G.W.J., Kienhuis E., Creemers G.J., Baars A., Dezentjé V.O., Imholz A.L.T., et al. DPYD genotype-guided dose individualisation of fluoropyrimidine therapy in patients with cancer: A prospective safety analysis. Lancet Oncol. 2018;19:1459–1467. doi: 10.1016/S1470-2045(18)30686-7.  [DOI](https://doi.org/10.1016/S1470-2045(18)30686-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30348537/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet%20Oncol.&title=DPYD%20genotype-guided%20dose%20individualisation%20of%20fluoropyrimidine%20therapy%20in%20patients%20with%20cancer:%20A%20prospective%20safety%20analysis&author=L.M.%20Henricks&author=C.A.T.C.%20Lunenburg&author=F.M.%20de%20Man&author=D.%20Meulendijks&author=G.W.J.%20Frederix&volume=19&publication_year=2018&pages=1459-1467&pmid=30348537&doi=10.1016/S1470-2045(18)30686-7&)

3. Henricks L.M., Opdam F.L., Beijnen J.H., Cats A., Schellens J.H.M. DPYD genotype-guided dose individualization to improve patient safety of fluoropyrimidine therapy: Call for a drug label update. Ann. Oncol. 2017;28:2915–2922. doi: 10.1093/annonc/mdx411.  [DOI](https://doi.org/10.1093/annonc/mdx411) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29045513/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann.%20Oncol.&title=DPYD%20genotype-guided%20dose%20individualization%20to%20improve%20patient%20safety%20of%20fluoropyrimidine%20therapy:%20Call%20for%20a%20drug%20label%20update&author=L.M.%20Henricks&author=F.L.%20Opdam&author=J.H.%20Beijnen&author=A.%20Cats&author=J.H.M.%20Schellens&volume=28&publication_year=2017&pages=2915-2922&pmid=29045513&doi=10.1093/annonc/mdx411&)

4. Van Cutsem E., Twelves C., Cassidy J., Allman D., Bajetta E., Boyer M., Bugat R., Findlay M., Frings S., Jahn M., et al. Oral capecitabine compared with intravenous fluorouracil plus leucovorin in patients with metastatic colorectal cancer: Results of a large phase III study. J. Clin. Oncol. 2001;19:4097–4106. doi: 10.1200/JCO.2001.19.21.4097.  [DOI](https://doi.org/10.1200/JCO.2001.19.21.4097) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11689577/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Oncol.&title=Oral%20capecitabine%20compared%20with%20intravenous%20fluorouracil%20plus%20leucovorin%20in%20patients%20with%20metastatic%20colorectal%20cancer:%20Results%20of%20a%20large%20phase%20III%20study&author=E.%20Van%20Cutsem&author=C.%20Twelves&author=J.%20Cassidy&author=D.%20Allman&author=E.%20Bajetta&volume=19&publication_year=2001&pages=4097-4106&pmid=11689577&doi=10.1200/JCO.2001.19.21.4097&)

5. White C., Scott R.J., Paul C., Ziolkowski A., Mossman D., Fox S.B., Michael M., Ackland S. Dihydropyrimidine Dehydrogenase Deficiency and Implementation of Upfront DPYD Genotyping. Clin. Pharmacol. Ther. 2022;112:791–802. doi: 10.1002/cpt.2667.  [DOI](https://doi.org/10.1002/cpt.2667) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35607723/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Dihydropyrimidine%20Dehydrogenase%20Deficiency%20and%20Implementation%20of%20Upfront%20DPYD%20Genotyping&author=C.%20White&author=R.J.%20Scott&author=C.%20Paul&author=A.%20Ziolkowski&author=D.%20Mossman&volume=112&publication_year=2022&pages=791-802&pmid=35607723&doi=10.1002/cpt.2667&)

6. Miwa M., Ura M., Nishida M., Sawada N., Ishikawa T., Mori K., Shimma N., Umeda I., Ishitsuka H. Design of a novel oral fluoropyrimidine carbamate, capecitabine, which generates 5 fluorouracil selectively in tumours by enzymes concentrated in human liver and cancer tissue. Eur. J. Cancer. 1998;34:1274–1281. doi: 10.1016/S0959-8049(98)00058-6.  [DOI](https://doi.org/10.1016/S0959-8049(98)00058-6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9849491/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Cancer&title=Design%20of%20a%20novel%20oral%20fluoropyrimidine%20carbamate,%20capecitabine,%20which%20generates%205%20fluorouracil%20selectively%20in%20tumours%20by%20enzymes%20concentrated%20in%20human%20liver%20and%20cancer%20tissue&author=M.%20Miwa&author=M.%20Ura&author=M.%20Nishida&author=N.%20Sawada&author=T.%20Ishikawa&volume=34&publication_year=1998&pages=1274-1281&pmid=9849491&doi=10.1016/S0959-8049(98)00058-6&)

7. van Kuilenburg A.B., Meinsma R., Zonnenberg B.A., Zoetekouw L., Baas F., Matsuda K., Tamaki N., van Gennip A.H. Dihydropyrimidinase deficiency and severe 5-fluorouracil toxicity. Clin. Cancer Res. 2003;9:4363–4367.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/14555507/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Cancer%20Res.&title=Dihydropyrimidinase%20deficiency%20and%20severe%205-fluorouracil%20toxicity&author=A.B.%20van%20Kuilenburg&author=R.%20Meinsma&author=B.A.%20Zonnenberg&author=L.%20Zoetekouw&author=F.%20Baas&volume=9&publication_year=2003&pages=4363-4367&pmid=14555507&)

8. Meinsma R., Fernandez Salguero P., Van Kuilenburg A.B.P., Van Gennip A.H., Gonzalez F.J. Human polymorphism in drug metabolism: Mutation in the dihydropyrimidine dehydrogenase gene results in exon skipping and thymine uracilurea. DNA Cell Biol. 1995;14:1–6. doi: 10.1089/dna.1995.14.1.  [DOI](https://doi.org/10.1089/dna.1995.14.1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7832988/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=DNA%20Cell%20Biol.&title=Human%20polymorphism%20in%20drug%20metabolism:%20Mutation%20in%20the%20dihydropyrimidine%20dehydrogenase%20gene%20results%20in%20exon%20skipping%20and%20thymine%20uracilurea&author=R.%20Meinsma&author=P.%20Fernandez%20Salguero&author=A.B.P.%20Van%20Kuilenburg&author=A.H.%20Van%20Gennip&author=F.J.%20Gonzalez&volume=14&publication_year=1995&pages=1-6&pmid=7832988&doi=10.1089/dna.1995.14.1&)

9. Van Kuilenburg A.B.P., Blom M.J., Van Lenthe H., Mul E., Van Gennip A.H. The activity of dihydropyrimidine dehydrogenase in human blood cells. J. Inherit. Metab. Dis. 1997;20:331–334. doi: 10.1023/a:1005305323052.  [DOI](https://doi.org/10.1023/a:1005305323052) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9266348/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Inherit.%20Metab.%20Dis.&title=The%20activity%20of%20dihydropyrimidine%20dehydrogenase%20in%20human%20blood%20cells&author=A.B.P.%20Van%20Kuilenburg&author=M.J.%20Blom&author=H.%20Van%20Lenthe&author=E.%20Mul&author=A.H.%20Van%20Gennip&volume=20&publication_year=1997&pages=331-334&pmid=9266348&doi=10.1023/a:1005305323052&)

10. Wei X., McLeod H.L., McMurrough J., Gonzalez F.J., Fernandez-Salguero P. Molecular basis of the human dihydropyrimidine dehydrogenase deficiency and 5-fluorouracil toxicity. J. Clin. Investig. 1996;98:610–615. doi: 10.1172/JCI118830.  [DOI](https://doi.org/10.1172/JCI118830) | [PMC free article](/articles/PMC507468/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8698850/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Investig.&title=Molecular%20basis%20of%20the%20human%20dihydropyrimidine%20dehydrogenase%20deficiency%20and%205-fluorouracil%20toxicity&author=X.%20Wei&author=H.L.%20McLeod&author=J.%20McMurrough&author=F.J.%20Gonzalez&author=P.%20Fernandez-Salguero&volume=98&publication_year=1996&pages=610-615&pmid=8698850&doi=10.1172/JCI118830&)

11. Deenen M.J., Meulendijks D., Cats A., Sechterberger M.K., Severens J.L., Boot H., Smits P.H., Rosing H., Mandigers C.M., Soesan M., et al. Upfront Genotyping of DPYD*2A to Individualize Fluoropyrimidine Therapy: A Safety and Cost Analysis. J. Clin. Oncol. 2016;34:227–234. doi: 10.1200/JCO.2015.63.1325.  [DOI](https://doi.org/10.1200/JCO.2015.63.1325) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26573078/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Oncol.&title=Upfront%20Genotyping%20of%20DPYD*2A%20to%20Individualize%20Fluoropyrimidine%20Therapy:%20A%20Safety%20and%20Cost%20Analysis&author=M.J.%20Deenen&author=D.%20Meulendijks&author=A.%20Cats&author=M.K.%20Sechterberger&author=J.L.%20Severens&volume=34&publication_year=2016&pages=227-234&pmid=26573078&doi=10.1200/JCO.2015.63.1325&)

12. Sharma B.B., Rai K., Blunt H., Zhao W., Tosteson T.D., Brooks G.A. Pathogenic DPYD Variants and Treatment-Related Mortality in Patients Receiving Fluoropyrimidine Chemotherapy: A Systematic Review and Meta-Analysis. Oncologist. 2021;26:1008–1016. doi: 10.1002/onco.13967.  [DOI](https://doi.org/10.1002/onco.13967) | [PMC free article](/articles/PMC8649021/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34506675/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncologist&title=Pathogenic%20DPYD%20Variants%20and%20Treatment-Related%20Mortality%20in%20Patients%20Receiving%20Fluoropyrimidine%20Chemotherapy:%20A%20Systematic%20Review%20and%20Meta-Analysis&author=B.B.%20Sharma&author=K.%20Rai&author=H.%20Blunt&author=W.%20Zhao&author=T.D.%20Tosteson&volume=26&publication_year=2021&pages=1008-1016&pmid=34506675&doi=10.1002/onco.13967&)

13. Meulendijks D., Henricks L.M., Sonke G.S., Deenen M.J., Froehlich T.K., Amstutz U., Largiadèr C.R., Jennings B.A., Marinaki A.M., Sanderson J.D., et al. Clinical relevance of DPYD variants c.1679T>G, c.1236G>A/HapB3, and c.1601G>A as predictors of severe fluoropyrimidine-associated toxicity: A systematic review and meta-analysis of individual patient data. Lancet Oncol. 2015;16:1639–1650. doi: 10.1016/S1470-2045(15)00286-7.  [DOI](https://doi.org/10.1016/S1470-2045(15)00286-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26603945/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet%20Oncol.&title=Clinical%20relevance%20of%20DPYD%20variants%20c.1679T>G,%20c.1236G>A/HapB3,%20and%20c.1601G>A%20as%20predictors%20of%20severe%20fluoropyrimidine-associated%20toxicity:%20A%20systematic%20review%20and%20meta-analysis%20of%20individual%20patient%20data&author=D.%20Meulendijks&author=L.M.%20Henricks&author=G.S.%20Sonke&author=M.J.%20Deenen&author=T.K.%20Froehlich&volume=16&publication_year=2015&pages=1639-1650&pmid=26603945&doi=10.1016/S1470-2045(15)00286-7&)

14. Whirl-Carrillo M., Huddart R., Gong L., Sangkuhl K., Thorn C.F., Whaley R., Klein T.E. An Evidence-Based Framework for Evaluating Pharmacogenomics Knowledge for Personalized Medicine. Clin. Pharmacol. Ther. 2021;110:563–572. doi: 10.1002/cpt.2350.  [DOI](https://doi.org/10.1002/cpt.2350) | [PMC free article](/articles/PMC8457105/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34216021/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=An%20Evidence-Based%20Framework%20for%20Evaluating%20Pharmacogenomics%20Knowledge%20for%20Personalized%20Medicine&author=M.%20Whirl-Carrillo&author=R.%20Huddart&author=L.%20Gong&author=K.%20Sangkuhl&author=C.F.%20Thorn&volume=110&publication_year=2021&pages=563-572&pmid=34216021&doi=10.1002/cpt.2350&)

15. 1000 Genomes Project Consortium. Abecasis G.R., Auton A., Brooks L.D., DePristo M.A., Durbin R.M., Handsaker R.E., Kang H.M., Marth G.T., McVean G.A. An integrated map of genetic variation from 1,092 human genomes. Nature. 2012;491:56–65. doi: 10.1038/nature11632.  [DOI](https://doi.org/10.1038/nature11632) | [PMC free article](/articles/PMC3498066/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23128226/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=An%20integrated%20map%20of%20genetic%20variation%20from%201,092%20human%20genomes&author=G.R.%20Abecasis&author=A.%20Auton&author=L.D.%20Brooks&author=M.A.%20DePristo&author=R.M.%20Durbin&volume=491&publication_year=2012&pages=56-65&pmid=23128226&doi=10.1038/nature11632&)

16. European Public Assessment Report (EPAR) for Capecitabine. Feb 16, 2012. [(accessed on 1 March 2024)]. Available online: https://www.ema.europa.eu/en/documents/assessment-report/capecitabine-teva-epar-public-assessment-report_en.pdf.  [https://www.ema.europa.eu/en/documents/assessment-report/capecitabine-teva-epar-public-assessment-report_en.pdf](https://www.ema.europa.eu/en/documents/assessment-report/capecitabine-teva-epar-public-assessment-report_en.pdf)

17. Food and Drug Administration (FDA) Table of Pharmacogenetic Associations. [(accessed on 1 March 2024)];2022 October 26; Available online: https://www.fda.gov/medical-devices/precision-medicine/table-pharmacogenetic-associations.  [https://www.fda.gov/medical-devices/precision-medicine/table-pharmacogenetic-associations](https://www.fda.gov/medical-devices/precision-medicine/table-pharmacogenetic-associations)

18. Amstutz U., Henricks L.M., Offer S.M., Barbarino J., Schellens J.H.M., Swen J.J., Klein T.E., McLeod H.L., Caudle K.E., Diasio R.B., et al. Clinical Pharmacogenetics Implementation Consortium (CPIC) Guideline for Dihydropyrimidine Dehydrogenase Genotype and Fluoropyrimidine Dosing: 2017 Update. Clin. Pharmacol. Ther. 2018;103:210–216. doi: 10.1002/cpt.911.  [DOI](https://doi.org/10.1002/cpt.911) | [PMC free article](/articles/PMC5760397/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29152729/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Clinical%20Pharmacogenetics%20Implementation%20Consortium%20(CPIC)%20Guideline%20for%20Dihydropyrimidine%20Dehydrogenase%20Genotype%20and%20Fluoropyrimidine%20Dosing:%202017%20Update&author=U.%20Amstutz&author=L.M.%20Henricks&author=S.M.%20Offer&author=J.%20Barbarino&author=J.H.M.%20Schellens&volume=103&publication_year=2018&pages=210-216&pmid=29152729&doi=10.1002/cpt.911&)

19. Lunenburg C.A.T.C., van der Wouden C.H., Nijenhuis M., Crommentuijn-van Rhenen M.H., de Boer-Veger N.J., Buunk A.M., Houwink E.J.F., Mulder H., Rongen G.A., van Schaik R.H.N., et al. Dutch Pharmacogenetics Working Group (DPWG) guideline for the gene-drug interaction of DPYD and fluoropyrimidines. Eur. J. Hum. Genet. 2020;28:508–517. doi: 10.1038/s41431-019-0540-0.  [DOI](https://doi.org/10.1038/s41431-019-0540-0) | [PMC free article](/articles/PMC7080718/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31745289/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Hum.%20Genet.&title=Dutch%20Pharmacogenetics%20Working%20Group%20(DPWG)%20guideline%20for%20the%20gene-drug%20interaction%20of%20DPYD%20and%20fluoropyrimidines&author=C.A.T.C.%20Lunenburg&author=C.H.%20van%20der%20Wouden&author=M.%20Nijenhuis&author=M.H.%20Crommentuijn-van%20Rhenen&author=N.J.%20de%20Boer-Veger&volume=28&publication_year=2020&pages=508-517&pmid=31745289&doi=10.1038/s41431-019-0540-0&)

20. Gallagher R.M., Kirkham J.J., Mason J.R., Bird K.A., Williamson P.R., Nunn A.J., Turner M.A., Smyth R.L., Pirmohamed M. Development and inter-rater reliability of the Liverpool adverse drug reaction causality assessment tool. PLoS ONE. 2011;6:e28096. doi: 10.1371/journal.pone.0028096.  [DOI](https://doi.org/10.1371/journal.pone.0028096) | [PMC free article](/articles/PMC3237416/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22194808/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&title=Development%20and%20inter-rater%20reliability%20of%20the%20Liverpool%20adverse%20drug%20reaction%20causality%20assessment%20tool&author=R.M.%20Gallagher&author=J.J.%20Kirkham&author=J.R.%20Mason&author=K.A.%20Bird&author=P.R.%20Williamson&volume=6&publication_year=2011&pages=e28096&pmid=22194808&doi=10.1371/journal.pone.0028096&)

21. U.S. Department of Health and Human Services Common Terminology Criteria for Adverse Events (CTCAE) [(accessed on 1 March 2024)];2017 Available online: https://ctep.cancer.gov/protocoldevelopment/electronic_applications/docs/ctcae_v5_quick_reference_5x7.pdf.  [https://ctep.cancer.gov/protocoldevelopment/electronic_applications/docs/ctcae_v5_quick_reference_5x7.pdf](https://ctep.cancer.gov/protocoldevelopment/electronic_applications/docs/ctcae_v5_quick_reference_5x7.pdf)

22. Eisenhauer E.A., Therasse P., Bogaerts J., Schwartz L.H., Sargent D., Ford R., Dancey J., Arbuck S., Gwyther S., Mooney M., et al. New response evaluation criteria in solid tumours: Revised RECIST guideline (version 1.1) Eur. J. Cancer. 2009;45:228–247. doi: 10.1016/j.ejca.2008.10.026.  [DOI](https://doi.org/10.1016/j.ejca.2008.10.026) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19097774/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Cancer&title=New%20response%20evaluation%20criteria%20in%20solid%20tumours:%20Revised%20RECIST%20guideline%20(version%201.1)&author=E.A.%20Eisenhauer&author=P.%20Therasse&author=J.%20Bogaerts&author=L.H.%20Schwartz&author=D.%20Sargent&volume=45&publication_year=2009&pages=228-247&pmid=19097774&doi=10.1016/j.ejca.2008.10.026&)

23. Solé X., Guinó E., Valls J., Iniesta R., Moreno V. SNPStats: A web tool for the analysis of association studies. Bioinformatics. 2006;22:1928–1929. doi: 10.1093/bioinformatics/btl268.  [DOI](https://doi.org/10.1093/bioinformatics/btl268) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16720584/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=SNPStats:%20A%20web%20tool%20for%20the%20analysis%20of%20association%20studies&author=X.%20Sol%C3%A9&author=E.%20Guin%C3%B3&author=J.%20Valls&author=R.%20Iniesta&author=V.%20Moreno&volume=22&publication_year=2006&pages=1928-1929&pmid=16720584&doi=10.1093/bioinformatics/btl268&)

24. Freeman B., Smith N., Curtis C., Huckett L., Mill J., Craig I.W. DNA from buccal swabs recruited by mail: Evaluation of storage effects on long-term stability and suitability for multiplex polymerase chain reaction genotyping. Behav. Genet. 2003;33:67–72. doi: 10.1023/A:1021055617738.  [DOI](https://doi.org/10.1023/A:1021055617738) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12645823/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Behav.%20Genet.&title=DNA%20from%20buccal%20swabs%20recruited%20by%20mail:%20Evaluation%20of%20storage%20effects%20on%20long-term%20stability%20and%20suitability%20for%20multiplex%20polymerase%20chain%20reaction%20genotyping&author=B.%20Freeman&author=N.%20Smith&author=C.%20Curtis&author=L.%20Huckett&author=J.%20Mill&volume=33&publication_year=2003&pages=67-72&pmid=12645823&doi=10.1023/A:1021055617738&)

25. Gómez-Martín A., Hernández A.F., Martínez-González L.J., González-Alzaga B., Rodríguez-Barranco M., López-Flores I., Aguilar-Garduno C., Lacasana M. Polymorphisms of pesticide-metabolizing genes in children living in intensive farming communities. Chemosphere. 2015;139:534–540. doi: 10.1016/j.chemosphere.2015.07.079.  [DOI](https://doi.org/10.1016/j.chemosphere.2015.07.079) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26318115/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chemosphere&title=Polymorphisms%20of%20pesticide-metabolizing%20genes%20in%20children%20living%20in%20intensive%20farming%20communities&author=A.%20G%C3%B3mez-Mart%C3%ADn&author=A.F.%20Hern%C3%A1ndez&author=L.J.%20Mart%C3%ADnez-Gonz%C3%A1lez&author=B.%20Gonz%C3%A1lez-Alzaga&author=M.%20Rodr%C3%ADguez-Barranco&volume=139&publication_year=2015&pages=534-540&pmid=26318115&doi=10.1016/j.chemosphere.2015.07.079&)

26. Deenen M.J., Tol J., Burylo A.M., Doodeman V.D., de Boer A., Vincent A., Guchelaar H.J., Smits P.H., Beijnen J.H., Punt C.J., et al. Relationship between single nucleotide polymorphisms and haplotypes in DPYD and toxicity and efficacy of capecitabine in advanced colorectal cancer. Clin. Cancer Res. 2011;17:3455–3468. doi: 10.1158/1078-0432.CCR-10-2209.  [DOI](https://doi.org/10.1158/1078-0432.CCR-10-2209) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21498394/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Cancer%20Res.&title=Relationship%20between%20single%20nucleotide%20polymorphisms%20and%20haplotypes%20in%20DPYD%20and%20toxicity%20and%20efficacy%20of%20capecitabine%20in%20advanced%20colorectal%20cancer&author=M.J.%20Deenen&author=J.%20Tol&author=A.M.%20Burylo&author=V.D.%20Doodeman&author=A.%20de%20Boer&volume=17&publication_year=2011&pages=3455-3468&pmid=21498394&doi=10.1158/1078-0432.CCR-10-2209&)

27. Lam S.W., Guchelaar H.J., Boven E. The role of pharmacogenetics in capecitabine efficacy and toxicity. Cancer Treat. Rev. 2016;50:9–22. doi: 10.1016/j.ctrv.2016.08.001.  [DOI](https://doi.org/10.1016/j.ctrv.2016.08.001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27569869/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Treat.%20Rev.&title=The%20role%20of%20pharmacogenetics%20in%20capecitabine%20efficacy%20and%20toxicity&author=S.W.%20Lam&author=H.J.%20Guchelaar&author=E.%20Boven&volume=50&publication_year=2016&pages=9-22&pmid=27569869&doi=10.1016/j.ctrv.2016.08.001&)

28. Seck K., Riemer S., Kates R., Ullrich T., Lutz V., Harbeck N., Schmitt M., Kiechle M., Diasio R., Gross E. Analysis of the DPYD gene implicated in 5-fluorouracil catabolism in a cohort of Caucasian individuals. Clin. Cancer Res. 2005;11:5886–5892. doi: 10.1158/1078-0432.CCR-04-1784.  [DOI](https://doi.org/10.1158/1078-0432.CCR-04-1784) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16115930/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Cancer%20Res.&title=Analysis%20of%20the%20DPYD%20gene%20implicated%20in%205-fluorouracil%20catabolism%20in%20a%20cohort%20of%20Caucasian%20individuals&author=K.%20Seck&author=S.%20Riemer&author=R.%20Kates&author=T.%20Ullrich&author=V.%20Lutz&volume=11&publication_year=2005&pages=5886-5892&pmid=16115930&doi=10.1158/1078-0432.CCR-04-1784&)

29. Madi A., Fisher D., Maughan T.S., Colley J.P., Meade A.M., Maynard J., Humphreys V., Wasan H., Adams R.A., Idziaszczyk S., et al. Pharmacogenetic analyses of 2183 patients with advanced colorectal cancer; potential role for common dihydropyrimidine dehydrogenase variants in toxicity to chemotherapy. Eur. J. Cancer. 2018;102:31–39. doi: 10.1016/j.ejca.2018.07.009.  [DOI](https://doi.org/10.1016/j.ejca.2018.07.009) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30114658/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Cancer&title=Pharmacogenetic%20analyses%20of%202183%20patients%20with%20advanced%20colorectal%20cancer;%20potential%20role%20for%20common%20dihydropyrimidine%20dehydrogenase%20variants%20in%20toxicity%20to%20chemotherapy&author=A.%20Madi&author=D.%20Fisher&author=T.S.%20Maughan&author=J.P.%20Colley&author=A.M.%20Meade&volume=102&publication_year=2018&pages=31-39&pmid=30114658&doi=10.1016/j.ejca.2018.07.009&)

30. Kuilenburg A.B.P.V., Meijer J., Tanck M.W.T., Dobritzsch D., Zoetekouw L., Dekkers L.L., Roelofsen J., Meinsma R., Wymenga M., Kulik W., et al. Phenotypic and clinical implications of variants in the dihydropyrimidine dehydrogenase gene. Biochim. Biophys. Acta. 2016;1862:754–762. doi: 10.1016/j.bbadis.2016.01.009.  [DOI](https://doi.org/10.1016/j.bbadis.2016.01.009) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26804652/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochim.%20Biophys.%20Acta&title=Phenotypic%20and%20clinical%20implications%20of%20variants%20in%20the%20dihydropyrimidine%20dehydrogenase%20gene&author=A.B.P.V.%20Kuilenburg&author=J.%20Meijer&author=M.W.T.%20Tanck&author=D.%20Dobritzsch&author=L.%20Zoetekouw&volume=1862&publication_year=2016&pages=754-762&pmid=26804652&doi=10.1016/j.bbadis.2016.01.009&)

31. Lee A.M., Shi Q., Pavey E., Alberts S.R., Sargent D.J., Sinicrope F.A., Berenberg J.L., Goldberg R.M., Diasio R.B. DPYD variants as predictors of 5-fluorouracil toxicity in adjuvant colon cancer treatment (NCCTG N0147) J. Natl. Cancer Inst. 2014;106:dju298. doi: 10.1093/jnci/dju298.  [DOI](https://doi.org/10.1093/jnci/dju298) | [PMC free article](/articles/PMC4271081/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25381393/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Natl.%20Cancer%20Inst.&title=DPYD%20variants%20as%20predictors%20of%205-fluorouracil%20toxicity%20in%20adjuvant%20colon%20cancer%20treatment%20(NCCTG%20N0147)&author=A.M.%20Lee&author=Q.%20Shi&author=E.%20Pavey&author=S.R.%20Alberts&author=D.J.%20Sargent&volume=106&publication_year=2014&pages=dju298&pmid=25381393&doi=10.1093/jnci/dju298&)

32. Díaz-Villamarín X., Fernández-Varón E., Rojas Romero M.C., Callejas-Rubio J.L., Cabeza-Barrera J., Rodríguez-Nogales A., Gálvez J., Morón R. Azathioprine dose tailoring based on pharmacogenetic information: Insights of clinical implementation. Biomed. Pharmacother. 2023;168:115706. doi: 10.1016/j.biopha.2023.115706.  [DOI](https://doi.org/10.1016/j.biopha.2023.115706) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/37857254/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biomed.%20Pharmacother.&title=Azathioprine%20dose%20tailoring%20based%20on%20pharmacogenetic%20information:%20Insights%20of%20clinical%20implementation&author=X.%20D%C3%ADaz-Villamar%C3%ADn&author=E.%20Fern%C3%A1ndez-Var%C3%B3n&author=M.C.%20Rojas%20Romero&author=J.L.%20Callejas-Rubio&author=J.%20Cabeza-Barrera&volume=168&publication_year=2023&pages=115706&pmid=37857254&doi=10.1016/j.biopha.2023.115706&)

33. Fragoulakis V., Roncato R., Bignucolo A., Patrinos G.P., Toffoli G., Cecchin E., Mitropoulou C. Cost-utility analysis and cross-country comparison of pharmacogenomics-guided treatment in colorectal cancer patients participating in the U-PGx PREPARE study. Pharmacol. Res. 2023;197:106949. doi: 10.1016/j.phrs.2023.106949.  [DOI](https://doi.org/10.1016/j.phrs.2023.106949) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/37802427/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacol.%20Res.&title=Cost-utility%20analysis%20and%20cross-country%20comparison%20of%20pharmacogenomics-guided%20treatment%20in%20colorectal%20cancer%20patients%20participating%20in%20the%20U-PGx%20PREPARE%20study&author=V.%20Fragoulakis&author=R.%20Roncato&author=A.%20Bignucolo&author=G.P.%20Patrinos&author=G.%20Toffoli&volume=197&publication_year=2023&pages=106949&pmid=37802427&doi=10.1016/j.phrs.2023.106949&)

34. Koufaki M.I., Fragoulakis V., Díaz-Villamarín X., Karamperis K., Vozikis A., Swen J.J., Dávila-Fajardo C.L., Vasileiou K.Z., Patrinos G.P., Mitropoulou C. Economic evaluation of pharmacogenomic-guided antiplatelet treatment in Spanish patients suffering from acute coronary syndrome participating in the U-PGx PREPARE study. Hum. Genom. 2023;17:51. doi: 10.1186/s40246-023-00495-3.  [DOI](https://doi.org/10.1186/s40246-023-00495-3) | [PMC free article](/articles/PMC10249170/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/37287029/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum.%20Genom.&title=Economic%20evaluation%20of%20pharmacogenomic-guided%20antiplatelet%20treatment%20in%20Spanish%20patients%20suffering%20from%20acute%20coronary%20syndrome%20participating%20in%20the%20U-PGx%20PREPARE%20study&author=M.I.%20Koufaki&author=V.%20Fragoulakis&author=X.%20D%C3%ADaz-Villamar%C3%ADn&author=K.%20Karamperis&author=A.%20Vozikis&volume=17&publication_year=2023&pages=51&pmid=37287029&doi=10.1186/s40246-023-00495-3&)
