# AKT1 Functional Polymorphism (rs2494750) Influences Clinical Outcomes in Patients With Advanced Lung Cancer Treated With EGFR Inhibitors

## Metadata
**Authors:** Sun Ha Choi, Jin Eun Choi, Mi Jeong Hong, Jang Hyuck Lee, Hyo-Gyoung Kang, Sook Kyung Do, Won Kee Lee, Ji Eun Park, Yong Hoon Lee, Hyewon Seo, Jaehee Lee, Shin Yup Lee, Seung Ick Cha, Chang Ho Kim, Jae Yong Park, Seung Soo Yoo
**Journal:** Journal of Korean Medical Science
**Date:** 2025 Jun 9
**DOI:** [10.3346/jkms.2025.40.e184](https://doi.org/10.3346/jkms.2025.40.e184)
**PMID:** 40795341
**PMCID:** PMC12339898
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC12339898/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC12339898/pdf/jkms-40-e184.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC12339898/pdf/jkms-40-e184.pdf)

## Abstract

**Background:** 
Epidermal growth factor receptor (EGFR)-tyrosine kinase inhibitors (TKIs) are the standard first-line treatment for patients with advanced lung cancer harboring activating EGFR mutations. This study aimed to investigate the association between genetic polymorphisms in the EGFR signaling pathway and clinical outcomes in patients receiving EGFR-TKI therapy.

**Methods:** 
We enrolled 266 patients with advanced lung cancer treated with EGFR-TKIs and examined 30 putative functional polymorphisms across 11 genes involved in the EGFR signaling pathway. Associations between these polymorphisms and clinical outcomes were assessed.

**Results:** 
Among the polymorphisms analyzed, AKT1 rs2494750G>C was significantly associated with improved chemotherapy response (codominant model: odds ratio, 1.78; 95% confidence interval [CI], 1.05–2.99; P = 0.031) and prolonged progression-free survival (recessive model: hazard ratio, 0.62; 95% CI, 0.39–0.97; P = 0.037). In the luciferase reporter assays, the rs2494750C allele exhibited significantly lower promoter activity than the rs2494750G allele in H522 and A549 lung cancer cell lines (P = 0.033 and P < 0.001, respectively). AKT1 expression levels were also significantly reduced in individuals with the CC genotype compared to those with GG or GC genotypes (P = 0.034).

**Conclusion:** 
The AKT1 rs2494750G>C polymorphism reduces promoter activity and gene expression of AKT1, potentially contributing to improved clinical outcomes in patients with advanced lung cancer treated with EGFR-TKIs.

Keywords: AKT1 Polymorphism, EGFR-TKI, Non-Small Cell Lung Cancer, Chemotherapy Response, Progression-Free Survival

### Background

Epidermal growth factor receptor (EGFR)-tyrosine kinase inhibitors (TKIs) are the standard first-line treatment for patients with advanced lung cancer harboring activating *EGFR* mutations. This study aimed to investigate the association between genetic polymorphisms in the EGFR signaling pathway and clinical outcomes in patients receiving EGFR-TKI therapy.

### Methods

We enrolled 266 patients with advanced lung cancer treated with EGFR-TKIs and examined 30 putative functional polymorphisms across 11 genes involved in the EGFR signaling pathway. Associations between these polymorphisms and clinical outcomes were assessed.

### Results

Among the polymorphisms analyzed, *AKT1* rs2494750G>C was significantly associated with improved chemotherapy response (codominant model: odds ratio, 1.78; 95% confidence interval [CI], 1.05–2.99; *P* = 0.031) and prolonged progression-free survival (recessive model: hazard ratio, 0.62; 95% CI, 0.39–0.97; *P* = 0.037). In the luciferase reporter assays, the rs2494750C allele exhibited significantly lower promoter activity than the rs2494750G allele in H522 and A549 lung cancer cell lines (*P* = 0.033 and *P* < 0.001, respectively). AKT1 expression levels were also significantly reduced in individuals with the CC genotype compared to those with GG or GC genotypes (*P* = 0.034).

### Conclusion

The *AKT1* rs2494750G>C polymorphism reduces promoter activity and gene expression of AKT1, potentially contributing to improved clinical outcomes in patients with advanced lung cancer treated with EGFR-TKIs.

## INTRODUCTION

Lung cancer remains the leading cause of cancer-related mortality worldwide.[1](#B1) The identification of driver mutations in cancer-related genes (*EGFR* mutation, *ALK* and *ROS1* rearrangements, *BRAF* V600E, *KRAS* G12C mutations) and consequently, the emergence of targeted therapies for each mutation, has significantly transformed the therapeutic landscape of lung cancer.[2](#B2),[3](#B3) The development of targeted therapies for these genetic alterations, including epidermal growth factor receptor (EGFR)-tyrosine kinase inhibitors (TKIs) and *ALK* inhibitors, has led to marked improvements in survival outcomes and reduced adverse reactions compared to conventional cytotoxic chemotherapy. Among the various actionable driver mutations, *EGFR* mutations are the most common, and mutation prevalence varies by racial background, occurring in approximately 40–60% of Asian patients, compared to 10–15% in Western populations.[2](#B2)

EGFR-TKIs have become the standard first-line treatment for patients with advanced non-small cell lung cancer (NSCLC) harboring activating *EGFR* mutations.[4](#B4) First-generation EGFR-TKIs (e.g., gefitinib and erlotinib) significantly improved response rates and progression-free survival (PFS) compared to platinum-based chemotherapy.[5](#B5),[6](#B6) Second-generation agents (e.g., afatinib and dacomitinib) further extended PFS in comparison to their first-generation counterparts.[7](#B7),[8](#B8) More recently, third-generation EGFR-TKIs (e.g., osimertinib and lazertinib) are being developed and are now adopted in clinical practice.[9](#B9),[10](#B10) Nevertheless, resistance to EGFR-TKIs remains an inevitable challenge in most cases. Mechanisms of resistance may be EGFR-dependent, such as the T790M mutation or EGFR amplification, or EGFR-independent, involving alterations in HER-2, MET, FGFR, or even histological transformation to small cell lung cancer.[11](#B11) Additionally, patients with exon 19 deletions tend to have a more favorable outcomes with EGFR-TKI therapy compared to those with exon 21 L858R mutations.[4](#B4),[12](#B12) Despite these observations, significant variability in treatment response and outcome persists even among patients with the same *EGFR* mutation, underscoring the need for additional predictive biomarkers.

Single nucleotide polymorphisms (SNPs), the most common form of genetic variation, represent a single base change in the genome. Certain functional SNPs can influence cancer susceptibility, treatment response, or prognosis.[13](#B13),[14](#B14),[15](#B15) Based on this rationale, we hypothesized that SNPs in genes involved in the EGFR signaling pathway may influence the clinical efficacy of EGFR-TKIs in patients with NSCLC harboring *EGFR* mutations, thereby impacting treatment response or PFS. Therefore, we aimed to identify putative functional SNPs in EGFR signaling pathway–related genes and investigate their potential association with clinical outcomes in this patient population.

## METHODS

### Study population

This retrospective study included patients diagnosed with stage III or IV NSCLC who were not candidates for curative surgical resection and received first-line therapy with EGFR-TKIs at Kyungpook National University Hospital (KNUH) between April 2010 and April 2017. Patients with stage IV disease who experienced recurrence after curative surgery and subsequently received EGFR-TKIs as first-line chemotherapy were also included. A total of 266 patients who provided informed consent and biological samples were enrolled. Among them, 148 had *EGFR* exon 19 deletions, 111 had the exon 21 L858R mutation, and 7 had uncommon *EGFR* mutations. All patients were Korean and had adenocarcinoma histology. The EGFR-TKIs used during the study period were gefitinib, erlotinib, and afatinib; second-generation dacomitinib and the third-generation osimertinib and lazertinib had not yet been approved in Korea for first-line use as of April 2017.

Genomic DNA was extracted from whole blood samples obtained prior to treatment. Samples were provided by the National Biobank of Korea-KNUH, supported by the Ministry of Health, Welfare and Family Affairs.

### Polymorphism selection and genotyping

A total of 614,701 polymorphisms in 16 genes related to the EGFR signaling pathway were retrieved from the NCBI database (https://www.ncbi,nlm,nih,gov/SNP): *EGFR, ERBB2, ERBB3, ERBB4, KRAS, NRAS, BRAF, PTEN, PIK3CA, LKB1, AKT1, mTOR, NF1, RASA1, TSC1*, and *TSC2*. From these, 843 putative functional polymorphisms were selected using the SNP functional prediction model (https://snpinfo.niehs,nih.gov/). SNPs with a minor allele frequency of < 0.1 in HapMap Japanese population in Tokyo (n = 761) and those in strong linkage disequilibrium (LD) (*r*^2^ > 0.8) (n = 46) were excluded using the TagSNP utility for LD tag SNP selection. This filtering resulted in 36 candidate SNPs. Genotyping was performed using the Sequenom MassARRAY iPLEX assay (Sequenom, San Diego, CA, USA) according to the manufacturer’s instructions. One SNP failed genotyping and five SNPs were excluded due to deviation from Hardy-Weinberg equilibrium (*P* < 0.05) or low call rates (< 95%). Ultimately, 30 SNPs in 11 EGFR signaling pathway–related genes were analyzed for associations with clinical outcomes in the 266 enrolled patients.

### Promoter-luciferase constructs and luciferase assay

To evaluate whether the rs2494750G>C variant affects promoter activity of the AKT serine/threonine kinase 1 (*AKT1*) gene, we performed a luciferase assay. *AKT1* promoter fragments encompassing the rs2494750 site and pGL3-basic plasmid were synthesized by polymerase chain reaction (PCR) using the following primers: *AKT1* forward, 5′-cttggcattcGGAGACAGCCAGTGCAAAATAAG-3′; *AKT1* reverse, 5′-aacagtaccgCAGAGGAGGAGCGGTGTCTAG-3′, pGL3-basic forward: 5′-tcctcctctgCGGTACTGTTGGTAAAGCCAC-3′ and pGL3-basic reverse, 5′-ggctgtctccGAATGCCAAGCTTACTTAGATC-3′ (overlapping sequences are indicated in lowercase). The PCR products were assembled into the pGL3-basic-AKT1 construct containing the rs2494750 G or C allele using the NEBuilder™ HiFi DNA Assembly Master Mix Kit, according to the manufacturer's instructions (New England Biolabs, Frankfurt, Germany). All constructs were verified by DNA sequencing. Human NSCLC cell lines H522 and A549 were transfected the constructs and pRL-SV40 vector (Promega, Madison, WI, USA) using Effectene transfection reagent (Qiagen, Hilden, Germany). The cells were collected 48 hours post-transfection, and luciferase activity was measured using the Dual-Luciferase® Reporter Assay System (Promega), with results normalized based on the activity of Renilla luciferase. Results are presented as the mean ± standard deviation from three independent experiments.

### RNA preparation and quantitative reverse transcription-PCR

To assess ATK1 expression in tumor versus normal lung tissue, we analyzed mRNA levels in paired tumor and adjacent non-tumorous lung tissue samples from 109 patients with NSCLC who underwent surgery for stage I-IIIA disease. ATK1 mRNA expression was examined by quantitative reverse transcription-PCR (qRT-PCR). Total RNA was extracted using Trizol (Invitrogen, Carlsbad, CA, USA). Real-time PCR was performed using a LightCycler 480 (Roche Applied Science, Mannheim, Germany) and QuantiFast SYBR Green PCR Master Mix (Qiagen). The primers for *AKT1* and β-actin genes were: *AKT1* forward, 5′-CCACCTGACCAAGATGACAG-3′; AKT1 reverse, 5′-TCTCCATCCCTCCAAGCTATC-3′; β-actin forward, 5′-TTGTTACAGGAAGTCCCTTGCC-3′; β-actin reverse, 5′- ATGCTATCACCTCCCCTGTGT-3′. Each sample was analyzed in duplicate. Relative AKT1 mRNA expression levels were normalized to β-actin and calculated using the 2^−ΔΔCt^ method.

### Statistical analysis

Patients were categorized as responders (complete or partial response), or non-responders (stable or progressive disease) based on their clinical response to EGFR-TKI therapy. PFS was estimated using the Kaplan-Meier method, and survival differences were assessed using the log-rank test. Multivariable Cox proportional hazards models were used to calculate hazard ratios (HRs) and 95% confidence intervals (CIs), adjusting for age, sex, smoking status, disease stage, and Eastern Cooperative Oncology Group performance status. Comparison was conducted in dominant, recessive, and codominant models. A *P* value < 0.05 was considered statistically significant. All analyses were conducted using the Statistical Analysis System for Windows, version 9.1 (SAS Institute, Cary, NC, USA). Box plots were generated using the ggplot2 package in R program.

### Ethics statement

The study protocol was approved by the Institutional Review Board (IRB) of Kyungpook National University Hospital (IRB approval No. KNUMCBIO_14-1009), and written informed consent was obtained from all participants.

## RESULTS

### Clinical characteristics of the study population

The overall response rate to EGFR-TKIs was 83.8%, and the median PFS was 16.6 months ([Table 1](#T1)). Patients harboring *EGFR* exon 19 deletions tended to have a better response to therapy than those with the exon 21 L8585R mutation (*P* = 0.063; [Table 1](#T1)). Similarly, patients treated with second-generation EGFR-TKIs had a trend toward longer PFS compared to those treated with first-generation agents (*P* = 0.067; [Table 1](#T1)).

### Table 1. Univariate analysis for chemotherapy response and progression-free survival by clinical variables.

| Variables |  | No. of cases | Response to chemotherapy |  |  |  | Progression-free survival |  |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Responders (CR+PR) | Non-responders (SD+PD) | OR (95% CI) | P | MST, mon | 95% CI | Log-rank P | HR (95% CI) | P |  |  |  |
| Overall |  | 266 | 223 (83.8) | 43 (16.2) |   |   | 16.6 | 15.4–18.5 |   |   |   |
| Age, yr |  |   |   |   |   |   |   |   |   |   |   |
|   | < 65 | 121 | 101 (83.5) | 20 (16.5) | 1.00 |   | 15.4 | 12.4–8.2 |   | 1.00 |   |
|   | ≥ 65 | 145 | 122 (84.1) | 23 (15.9) | 1.05 (0.55–2.02) | 0.883 | 18.2 | 15.9–21.6 | 0.145 | 0.81 (0.61–1.08) | 0.146 |
| Gender |  |   |   |   |   |   |   |   |   |   |   |
|   | Male | 93 | 78 (83.9) | 15 (16.1) | 1.00 |   | 15.4 | 11.6–18.3 |   | 1.00 |   |
|   | Female | 173 | 145 (83.8) | 28 (16.2) | 0.99 (0.50–1.98) | 0.991 | 17.4 | 16.0–19.6 | 0.099 | 0.78 (0.57–1.05) | 0.100 |
| Smoking status |  |   |   |   |   |   |   |   |   |   |   |
|   | Never | 179 | 148 (82.7) | 31 (17.3) | 1.00 |   | 17.4 | 15.9–19.4 |   | 1.00 |   |
|   | Ever | 87 | 75 (86.2) | 12 (13.8) | 1.31 (0.63–2.69) | 0.464 | 15.4 | 11.6–18.3 | 0.484 | 1.16 (0.82–1.52) | 0.484 |
| Clinical stage |  |   |   |   |   |   |   |   |   |   |   |
|   | Ⅲ | 19 | 14 (73.7) | 5 (26.3) | 1.00 |   | 23.6 | 7.59–N.D. |   | 1.00 |   |
|   | Ⅳ | 247 | 209 (84.6) | 38 (15.4) | 1.96 (0.67–5.77) | 0.219 | 16.4 | 15.1–18.3 | 0.096 | 1.71 (0.90–3.24) | 0.100 |
| PS ECOG |  |   |   |   |   |   |   |   |   |   |   |
|   | 0–1 | 86 | 74 (86.1) | 12 (13.9) | 1.00 |   | 19.4 | 16.6–23.6 |   | 1.00 |   |
|   | 2 | 180 | 149 (82.8) | 31 (17.2) | 0.78 (0.38–1.61) | 0.499 | 15.5 | 12.5–17.4 | 0.011 | 1.49 (1.09–2.03) | 0.011 |
| Weight loss |  |   |   |   |   |   |   |   |   |   |   |
|   | No | 166 | 136 (81.9) | 30 (73.7) | 1.00 |   | 16.4 | 13.0–18.5 |   | 1.00 |   |
|   | Yes | 19 | 14 (18.1) | 5 (26.3) | 0.62 (0.21–1.85) | 0.388 | 15.4 | 6.9–29.4 | 0.837 | 1.06 (0.62–1.80) | 0.837 |
| EGFR mutation |  |   |   |   |   |   |   |   |   |   |   |
|   | Exon 19 deletion | 148 | 129 (87.2) | 19 (12.8) | 1.00 |   | 17.4 | 15.5–19.4 |   | 1.00 |   |
|   | Exon 21 L858R | 111 | 87 (78.4) | 24 (21.6) | 0.53 (0.28–1.03) | 0.063 | 15.2 | 10.6–18.5 |   | 1.21 (0.91–1.62) | 0.193 |
|   | Othersa | 7 | 7 (100.0) | 0 (0.0) | – | 0.984 | 32.1 | 8.6–32.1 | 0.286 | 0.65 (0.21–2.05) | 0.463 |
| EGFR-TKI |  |   |   |   |   |   |   |   |   |   |   |
|   | 1st generation | 166 | 135 (81.3) | 31 (18.7) | 1.00 |   | 15.9 | 12.4–17.4 |   | 1.00 |   |
|   | 2nd generation | 100 | 88 (88.0) | 12 (12.0) | 1.68 (0.82–3.45) | 0.155 | 18.5 | 15.6–24.1 | 0.065 | 0.75 (0.56–1.02) | 0.067 |

Table 1 Caption: CR = complete response, PR = partial response, SD = stable disease, PD = progressive disease, OR = odds ratio, MST = median survival time, CI = confidence interval, HR = hazard ratio, PS = performance status, ECOG = Eastern Cooperative Oncology Group, N.D. = not determined. aUncommon EGFR mutations; two G719A, three G719X and two S768I.

### Association between polymorphisms and clinical outcomes

Among the 30 SNPs analyzed, only *AKT1* rs2494750G>C was significantly associated with both treatment response and PFS. The remaining 29 SNPs showed weak associations with either response or PFS, or no association at all ([Supplementary Table 1](#S1)). The rs2494750G>C polymorphism was significantly associated with improved treatment response under a codominant model (GG vs. GC vs. CC = 76.4% vs. 89.4% vs. 86.8%; odds ratio [OR] = 1.78; 95% CI: 1.05–2.99; *P* = 0.031; [Table 2](#T2)). Patients with the rs2494750 CC genotype had significantly longer PFS compared to those with the rs2494750 GG or GC genotypes under a recessive model (CC vs. GG+GC = 20.4 months vs. 16.0 months; HR, 0.62; 95% CI, 0.39–0.97; *P* = 0.037; [Table 2](#T2) and [Fig. 1](#F1)). In subgroup analyses, the association between *ATK1* rs2494750G>C and both treatment response and PFS was significant only in patients treated with first-generation EGFR-TKIs ([Supplementary Table 2](#S2)). No significant difference in *AKT1* rs2494750 genotype distribution was observed between patients with or without the T790M mutation ([Supplementary Table 3](#S3)).

### Table 2. Genotypes of rs2494750 and their associations with clinical outcomes.

| Gene/ID No. |  | No. of cases (%)a | Response to chemotherapy |  |  |  | Progression-free survival |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Respondersb | Non-respondersb | OR (95% CI)c | P c | MST (95% CI) | Log-Rank P | HR (95% CI)d | P d |  |  |  |
| AKT1 rs2494750 |  |   |   |   |   |   |   |   |   |   |
|   | GG | 110 (42.1) | 84 (76.4) | 26 (23.6) | 1.00 |   | 14.5 (12.2–18.6) |   | 1.00 |   |
|   | GC | 113 (43.3) | 101 (89.4) | 12 (10.6) | 2.61 (1.22–5.55) | 0.013 | 16.4 (15.1–18.5) |   | 1.01 (0.74–1.38) | 0.952 |
|   | CC | 38 (14.6) | 33 (86.8) | 5 (13.2) | 2.15 (0.75–6.18) | 0.156 | 20.4 (18.2–30.1) |   | 0.62 (0.39–1.00) | 0.052 |
| Dominant |  |   |   |   | 1.57 (1.12–2.22) | 0.009 | 18.2 (16.0–18.7) | 0.440 | 0.95 (0.82–1.10) | 0.454 |
| Recessive |  |   |   |   | 1.41 (0.51–3.89) | 0.510 | 16.0 (13.3–18.0) | 0.047 | 0.62 (0.39–0.97) | 0.037 |
| Codominant |  |   |   |   | 1.78 (1.05–2.99) | 0.031 |   | 0.139 | 0.84 (0.69–1.04) | 0.109 |

Table 2 Caption: Values are presented as number (%). OR = odds ratio, CI = confidence interval, MST = median survival time (months), HR = hazard ratio. aColumn percentage. bRow percentage. cOR, 95% CI, and their corresponding P values were calculated by multivariate regression analysis, adjusted for age, gender, smoking status, stage, Eastern Cooperative Oncology Group performance status. dHR, 95% CI and their corresponding P values were calculated using multivariate Cox proportional hazard models, adjusted for age, gender, smoking status, stage, Eastern Cooperative Oncology Group performance status.

### Fig. 1. Kaplan-Meier curves for progression-free survival stratified by AKT1 rs2494750 genotype under a recessive model. A P value was calculated using the multivariate Cox proportional model.

![Fig. 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6207/12339898/42153eee8ff9/jkms-40-e184-g001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=12339898_jkms-40-e184-g001.jpg)

### Functional effect of rs2494750 on AKT1 promoter activity

The rs2494750G>C SNP is located in the 5′ untranslated region UTR of *AKT1* and may influence promoter activity. To test this, we performed a luciferase reporter assay to investigate the effect of this polymorphism on *AKT1* promoter activity. The rs2494750 C allele showed significantly reduced promoter activity compared to the G allele in H522 and A549 lung cancer cell lines (*P* = 0.033 and *P* < 0.001, respectively; [Fig. 2](#F2)). Additionally, *AKT1* mRNA expression was significantly higher in tumor samples compared to paired non-malignant lung tissues from 109 patients with NSCLC (*P* = 0.001; [Fig. 3A](#F3)). When stratified by genotype, *AKT1* expression was significantly lower in patients with the CC genotype compared to those with the GG or GC genotypes (*P* = 0.034; [Fig. 3B](#F3)).

### Fig. 2. Luciferase assay showing the effect of the rs2494750 G and C alleles on AKT1 promoter activity in H522 and A549 lung cancer cell lines.

![Fig. 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6207/12339898/3a18ab6acc5e/jkms-40-e184-g002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=12339898_jkms-40-e184-g002.jpg)

### Fig. 3. AKT1 mRNA expression. (A) Comparison of AKT1 mRNA expression between 109 paired tumor and adjacent non-malignant lung tissues. (B) AKT1 mRNA expression stratified by rs2494750 genotypes.

![Fig. 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6207/12339898/41b54d797d48/jkms-40-e184-g003.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=12339898_jkms-40-e184-g003.jpg)

## DISCUSSION

This study investigated the association between polymorphisms in EGFR pathway–related genes and clinical outcomes in advanced NSCLC patients treated with EGFR-TKIs as first-line therapy. Among the 30 analyzed SNPs, *AKT1* rs2494750G>C was significantly associated with better chemotherapy response and prolonged PFS. Functional assays further demonstrated that this SNP modulates *AKT1* expression.

Protein kinases are key regulators of cellular processes such as signaling, metabolism, proliferation, survival, and migration. The phosphoinositide 3-kinase (PI3K)-AKT signaling pathway is one of the most frequently activated pathways in human cancers.[16](#B16),[17](#B17) AKT, also known as protein kinase B, is a serine/threonine kinase with three isoforms (AKT1, AKT2, and AKT3).[18](#B18) Hyperactivation of AKT has been observed across multiple cancer types,[19](#B19),[20](#B20),[21](#B21),[22](#B22) as is associated with tumor progression, metastasis, and treatment resistance.[23](#B23),[24](#B24),[25](#B25) Therefore, the development of targeted therapies against AKT has been a major focus of precision oncology.[26](#B26),[27](#B27),[28](#B28) In lung cancer, particularly in *EGFR-*mutant subtypes, AKT activation is frequently observed,[29](#B29),[30](#B30) and has been linked to resistance against EGFR-TKIs.[30](#B30),[31](#B31)

Moreover, genetic variants in *AKT* are associated with susceptibility to various cancer types.[32](#B32),[33](#B33),[34](#B34) Dysregulation in the PI3K-AKT signaling pathway can affect cancer proliferation, invasion, metastasis, and drug resistance.[35](#B35) Alterations in the PI3K-AKT signaling may influence apoptosis, angiogenesis, which may have an impact on cancer prognosis.[36](#B36) Our study found that the *AKT1* rs2494750G>C polymorphism, located in the 5′ UTR of the gene, was significantly associated with improved chemotherapy response and PFS. Sequence alignment predicts this region to be a transcription factor binding site, and may influence *AKT1* transcription and translation. Functional studies confirmed that the rs2494750 C allele exhibited significantly reduced level of AKT1 activation compared to the G allele in H522 and A549 lung cancer cell lines ([Fig. 2](#F2)). Furthermore, AKT1 expression was significantly lower in tumor samples with the CC genotype compared to GG or GC genotypes ([Fig. 3B](#F3)). Higher AKT1 expression in tumor tissue compared to the normal tissue suggests that AKT1 plays a role in the development of cancer, while reduced AKT1 levels in established tumors may be associated with improved prognosis. Accordingly, the *AKT1* rs2494750 G to C substitution may modulate AKT1 expression, potentially affecting tumor metabolism, proliferation, or apoptosis, and thereby influencing the prognosis of lung cancer patients. These results are supported by findings reported by Lee et al.[37](#B37) that inhibiting siRNA-mediated *AKT1* inhibition in NSCLC cells reduced cancer cell growth and migration and enhanced apoptosis in response to cisplatin treatment. Similarly, FitzGerald et al.[38](#B38) found that *AKT1* rs2494750G>C polymorphism was associated with improved prostate cancer–specific survival.

In our study, *AKT1* rs2494750G>C was significantly associated with improved chemotherapy response and prolonged PFS, but not with overall survival ([Supplementary Table 4](#S4)). This discrepancy may be attributed to the variability in overall survival among patients treated with EGFR-TKIs, which can depend on resistance mechanisms, particularly the emergence of the T790M mutation, and the use of third-generation EGFR-TKIs or subsequent lines of therapy. Subgroup analysis revealed a significant association between *AKT1* rs2494750G>C and both response to chemotherapy and PFS in patients with first-generation EGFR-TKIs, but not in those treated with second-generation agents ([Supplementary Table 2](#S2)). This may be due to the limited sample size in the second-generation EGFR-TKI group (n = 100) compared to the first-generation group (n = 166), reducing statistical power. Moreover, the potential role of *AKT1* rs2494750G>C in patients treated with third-generation EGFR-TKIs (e.g., osimertinib and lazertinib) remains unclear, as these drugs were not included in this study due to their unavailability as first-line treatments during the study period. Further research is warranted to assess the prognostic and predictive value of *AKT1* rs2494750G>C in patients receiving third-generation EGFR-TKIs.

Additionally, although this study focused on EGFR-TKI therapy, *AKT1* rs2494750G>C may also influence the efficacy of other treatment modalities such as conventional chemotherapy or immunotherapy, or may serve as an independent prognostic biomarker for lung cancer. To establish this, further investigations in cohorts receiving non-EGFR-TKI therapies as first-line treatment are needed.

In summary, our study identified *AKT1* rs2494750G>C as a potential predictive biomarker for improved response and PFS in patients with NSCLC treated with EGFR-TKIs as first-line therapy. Functional analyses demonstrated that this polymorphism may impact *AKT1* promoter activity and gene expression, potentially influencing clinical outcomes.

## Footnotes

## SUPPLEMENTARY MATERIALS

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Bray F, Laversanne M, Sung H, Ferlay J, Siegel RL, Soerjomataram I, et al. Global cancer statistics 2022: GLOBOCAN estimates of incidence and mortality worldwide for 36 cancers in 185 countries. CA Cancer J Clin. 2024;74(3):229–263. doi: 10.3322/caac.21834.  [DOI](https://doi.org/10.3322/caac.21834) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/38572751/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=CA%20Cancer%20J%20Clin&title=Global%20cancer%20statistics%202022:%20GLOBOCAN%20estimates%20of%20incidence%20and%20mortality%20worldwide%20for%2036%20cancers%20in%20185%20countries&author=F%20Bray&author=M%20Laversanne&author=H%20Sung&author=J%20Ferlay&author=RL%20Siegel&volume=74&issue=3&publication_year=2024&pages=229-263&pmid=38572751&doi=10.3322/caac.21834&)

2. Tan AC, Tan DSW. Targeted therapies for lung cancer patients with oncogenic driver molecular alterations. J Clin Oncol. 2022;40(6):611–625. doi: 10.1200/JCO.21.01626.  [DOI](https://doi.org/10.1200/JCO.21.01626) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34985916/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Targeted%20therapies%20for%20lung%20cancer%20patients%20with%20oncogenic%20driver%20molecular%20alterations&author=AC%20Tan&author=DSW%20Tan&volume=40&issue=6&publication_year=2022&pages=611-625&pmid=34985916&doi=10.1200/JCO.21.01626&)

3. Araghi M, Mannani R, Heidarnejad Maleki A, Hamidi A, Rostami S, Safa SH, et al. Recent advances in non-small cell lung cancer targeted therapy; an update review. Cancer Cell Int. 2023;23(1):162. doi: 10.1186/s12935-023-02990-y.  [DOI](https://doi.org/10.1186/s12935-023-02990-y) | [PMC free article](/articles/PMC10416536/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/37568193/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Cell%20Int&title=Recent%20advances%20in%20non-small%20cell%20lung%20cancer%20targeted%20therapy;%20an%20update%20review&author=M%20Araghi&author=R%20Mannani&author=A%20Heidarnejad%20Maleki&author=A%20Hamidi&author=S%20Rostami&volume=23&issue=1&publication_year=2023&pages=162&pmid=37568193&doi=10.1186/s12935-023-02990-y&)

4. Hayashi H, Nadal E, Gray JE, Ardizzoni A, Caria N, Puri T, et al. Overall treatment strategy for patients with metastatic NSCLC with activating EGFR mutations. Clin Lung Cancer. 2022;23(1):e69–e82. doi: 10.1016/j.cllc.2021.10.009.  [DOI](https://doi.org/10.1016/j.cllc.2021.10.009) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34865963/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Lung%20Cancer&title=Overall%20treatment%20strategy%20for%20patients%20with%20metastatic%20NSCLC%20with%20activating%20EGFR%20mutations&author=H%20Hayashi&author=E%20Nadal&author=JE%20Gray&author=A%20Ardizzoni&author=N%20Caria&volume=23&issue=1&publication_year=2022&pages=e69-e82&pmid=34865963&doi=10.1016/j.cllc.2021.10.009&)

5. Maemondo M, Inoue A, Kobayashi K, Sugawara S, Oizumi S, Isobe H, et al. Gefitinib or chemotherapy for non-small-cell lung cancer with mutated EGFR. N Engl J Med. 2010;362(25):2380–2388. doi: 10.1056/NEJMoa0909530.  [DOI](https://doi.org/10.1056/NEJMoa0909530) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20573926/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Gefitinib%20or%20chemotherapy%20for%20non-small-cell%20lung%20cancer%20with%20mutated%20EGFR&author=M%20Maemondo&author=A%20Inoue&author=K%20Kobayashi&author=S%20Sugawara&author=S%20Oizumi&volume=362&issue=25&publication_year=2010&pages=2380-2388&pmid=20573926&doi=10.1056/NEJMoa0909530&)

6. Zhou C, Wu YL, Chen G, Feng J, Liu XQ, Wang C, et al. Erlotinib versus chemotherapy as first-line treatment for patients with advanced EGFR mutation-positive non-small-cell lung cancer (OPTIMAL, CTONG-0802): a multicentre, open-label, randomised, phase 3 study. Lancet Oncol. 2011;12(8):735–742. doi: 10.1016/S1470-2045(11)70184-X.  [DOI](https://doi.org/10.1016/S1470-2045(11)70184-X) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21783417/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet%20Oncol&title=Erlotinib%20versus%20chemotherapy%20as%20first-line%20treatment%20for%20patients%20with%20advanced%20EGFR%20mutation-positive%20non-small-cell%20lung%20cancer%20(OPTIMAL,%20CTONG-0802):%20a%20multicentre,%20open-label,%20randomised,%20phase%203%20study&author=C%20Zhou&author=YL%20Wu&author=G%20Chen&author=J%20Feng&author=XQ%20Liu&volume=12&issue=8&publication_year=2011&pages=735-742&pmid=21783417&doi=10.1016/S1470-2045(11)70184-X&)

7. Park K, Tan EH, O’Byrne K, Zhang L, Boyer M, Mok T, et al. Afatinib versus gefitinib as first-line treatment of patients with EGFR mutation-positive non-small-cell lung cancer (LUX-Lung 7): a phase 2B, open-label, randomised controlled trial. Lancet Oncol. 2016;17(5):577–589. doi: 10.1016/S1470-2045(16)30033-X.  [DOI](https://doi.org/10.1016/S1470-2045(16)30033-X) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27083334/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet%20Oncol&title=Afatinib%20versus%20gefitinib%20as%20first-line%20treatment%20of%20patients%20with%20EGFR%20mutation-positive%20non-small-cell%20lung%20cancer%20(LUX-Lung%207):%20a%20phase%202B,%20open-label,%20randomised%20controlled%20trial&author=K%20Park&author=EH%20Tan&author=K%20O%E2%80%99Byrne&author=L%20Zhang&author=M%20Boyer&volume=17&issue=5&publication_year=2016&pages=577-589&pmid=27083334&doi=10.1016/S1470-2045(16)30033-X&)

8. Wu YL, Cheng Y, Zhou X, Lee KH, Nakagawa K, Niho S, et al. Dacomitinib versus gefitinib as first-line treatment for patients with EGFR-mutation-positive non-small-cell lung cancer (ARCHER 1050): a randomised, open-label, phase 3 trial. Lancet Oncol. 2017;18(11):1454–1466. doi: 10.1016/S1470-2045(17)30608-3.  [DOI](https://doi.org/10.1016/S1470-2045(17)30608-3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28958502/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet%20Oncol&title=Dacomitinib%20versus%20gefitinib%20as%20first-line%20treatment%20for%20patients%20with%20EGFR-mutation-positive%20non-small-cell%20lung%20cancer%20(ARCHER%201050):%20a%20randomised,%20open-label,%20phase%203%20trial&author=YL%20Wu&author=Y%20Cheng&author=X%20Zhou&author=KH%20Lee&author=K%20Nakagawa&volume=18&issue=11&publication_year=2017&pages=1454-1466&pmid=28958502&doi=10.1016/S1470-2045(17)30608-3&)

9. Ramalingam SS, Vansteenkiste J, Planchard D, Cho BC, Gray JE, Ohe Y, et al. Overall survival with osimertinib in untreated, EGFR-mutated advanced NSCLC. N Engl J Med. 2020;382(1):41–50. doi: 10.1056/NEJMoa1913662.  [DOI](https://doi.org/10.1056/NEJMoa1913662) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31751012/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Overall%20survival%20with%20osimertinib%20in%20untreated,%20EGFR-mutated%20advanced%20NSCLC&author=SS%20Ramalingam&author=J%20Vansteenkiste&author=D%20Planchard&author=BC%20Cho&author=JE%20Gray&volume=382&issue=1&publication_year=2020&pages=41-50&pmid=31751012&doi=10.1056/NEJMoa1913662&)

10. Cho BC, Ahn MJ, Kang JH, Soo RA, Reungwetwattana T, Yang JC, et al. Lazertinib versus gefitinib as first-line treatment in patients with EGFR-mutated advanced non-small-cell lung cancer: results from LASER301. J Clin Oncol. 2023;41(26):4208–4217. doi: 10.1200/JCO.23.00515.  [DOI](https://doi.org/10.1200/JCO.23.00515) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/37379502/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Lazertinib%20versus%20gefitinib%20as%20first-line%20treatment%20in%20patients%20with%20EGFR-mutated%20advanced%20non-small-cell%20lung%20cancer:%20results%20from%20LASER301&author=BC%20Cho&author=MJ%20Ahn&author=JH%20Kang&author=RA%20Soo&author=T%20Reungwetwattana&volume=41&issue=26&publication_year=2023&pages=4208-4217&pmid=37379502&doi=10.1200/JCO.23.00515&)

11. Reita D, Pabst L, Pencreach E, Guérin E, Dano L, Rimelen V, et al. Molecular mechanism of EGFR-TKI resistance in EGFR-mutated non-small cell lung cancer: application to biological diagnostic and monitoring. Cancers (Basel) 2021;13(19):4926. doi: 10.3390/cancers13194926.  [DOI](https://doi.org/10.3390/cancers13194926) | [PMC free article](/articles/PMC8507869/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34638411/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancers%20(Basel)&title=Molecular%20mechanism%20of%20EGFR-TKI%20resistance%20in%20EGFR-mutated%20non-small%20cell%20lung%20cancer:%20application%20to%20biological%20diagnostic%20and%20monitoring&author=D%20Reita&author=L%20Pabst&author=E%20Pencreach&author=E%20Gu%C3%A9rin&author=L%20Dano&volume=13&issue=19&publication_year=2021&pages=4926&pmid=34638411&doi=10.3390/cancers13194926&)

12. Batra U, Biswas B, Prabhash K, Krishna MV. Differential clinicopathological features, treatments and outcomes in patients with Exon 19 deletion and Exon 21 L858R EGFR mutation-positive adenocarcinoma non-small-cell lung cancer. BMJ Open Respir Res. 2023;10(1):e001492. doi: 10.1136/bmjresp-2022-001492.  [DOI](https://doi.org/10.1136/bmjresp-2022-001492) | [PMC free article](/articles/PMC10277533/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/37321664/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMJ%20Open%20Respir%20Res&title=Differential%20clinicopathological%20features,%20treatments%20and%20outcomes%20in%20patients%20with%20Exon%2019%20deletion%20and%20Exon%2021%20L858R%20EGFR%20mutation-positive%20adenocarcinoma%20non-small-cell%20lung%20cancer&author=U%20Batra&author=B%20Biswas&author=K%20Prabhash&author=MV%20Krishna&volume=10&issue=1&publication_year=2023&pages=e001492&pmid=37321664&doi=10.1136/bmjresp-2022-001492&)

13. Savas S, Liu G. Studying genetic variations in cancer prognosis (and risk): a primer for clinicians. Oncologist. 2009;14(7):657–666. doi: 10.1634/theoncologist.2009-0042.  [DOI](https://doi.org/10.1634/theoncologist.2009-0042) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19581524/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncologist&title=Studying%20genetic%20variations%20in%20cancer%20prognosis%20(and%20risk):%20a%20primer%20for%20clinicians&author=S%20Savas&author=G%20Liu&volume=14&issue=7&publication_year=2009&pages=657-666&pmid=19581524&doi=10.1634/theoncologist.2009-0042&)

14. Yoo SS, Lee S, Choi JE, Hong MJ, Do SK, Lee JH, et al. Promoter-specific variants in neurod1 and H3K4me3 coincident regions and clinical outcomes of small cell lung cancer. J Korean Med Sci. 2023;38(45):e381. doi: 10.3346/jkms.2023.38.e381.  [DOI](https://doi.org/10.3346/jkms.2023.38.e381) | [PMC free article](/articles/PMC10659920/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/37987107/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Korean%20Med%20Sci&title=Promoter-specific%20variants%20in%20neurod1%20and%20H3K4me3%20coincident%20regions%20and%20clinical%20outcomes%20of%20small%20cell%20lung%20cancer&author=SS%20Yoo&author=S%20Lee&author=JE%20Choi&author=MJ%20Hong&author=SK%20Do&volume=38&issue=45&publication_year=2023&pages=e381&pmid=37987107&doi=10.3346/jkms.2023.38.e381&)

15. Minnai F, Noci S, Esposito M, Schneider MA, Kobinger S, Eichhorn M, et al. Germline polymorphisms associated with overall survival in lung adenocarcinoma: genome-wide analysis. Cancers (Basel) 2024;16(19):3264. doi: 10.3390/cancers16193264.  [DOI](https://doi.org/10.3390/cancers16193264) | [PMC free article](/articles/PMC11475969/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/39409885/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancers%20(Basel)&title=Germline%20polymorphisms%20associated%20with%20overall%20survival%20in%20lung%20adenocarcinoma:%20genome-wide%20analysis&author=F%20Minnai&author=S%20Noci&author=M%20Esposito&author=MA%20Schneider&author=S%20Kobinger&volume=16&issue=19&publication_year=2024&pages=3264&pmid=39409885&doi=10.3390/cancers16193264&)

16. Hoxhaj G, Manning BD. The PI3K-AKT network at the interface of oncogenic signalling and cancer metabolism. Nat Rev Cancer. 2020;20(2):74–88. doi: 10.1038/s41568-019-0216-7.  [DOI](https://doi.org/10.1038/s41568-019-0216-7) | [PMC free article](/articles/PMC7314312/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31686003/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Cancer&title=The%20PI3K-AKT%20network%20at%20the%20interface%20of%20oncogenic%20signalling%20and%20cancer%20metabolism&author=G%20Hoxhaj&author=BD%20Manning&volume=20&issue=2&publication_year=2020&pages=74-88&pmid=31686003&doi=10.1038/s41568-019-0216-7&)

17. Katso R, Okkenhaug K, Ahmadi K, White S, Timms J, Waterfield MD. Cellular function of phosphoinositide 3-kinases: implications for development, homeostasis, and cancer. Annu Rev Cell Dev Biol. 2001;17(1):615–675. doi: 10.1146/annurev.cellbio.17.1.615.  [DOI](https://doi.org/10.1146/annurev.cellbio.17.1.615) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11687500/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annu%20Rev%20Cell%20Dev%20Biol&title=Cellular%20function%20of%20phosphoinositide%203-kinases:%20implications%20for%20development,%20homeostasis,%20and%20cancer&author=R%20Katso&author=K%20Okkenhaug&author=K%20Ahmadi&author=S%20White&author=J%20Timms&volume=17&issue=1&publication_year=2001&pages=615-675&pmid=11687500&doi=10.1146/annurev.cellbio.17.1.615&)

18. Datta SR, Brunet A, Greenberg ME. Cellular survival: a play in three Akts. Genes Dev. 1999;13(22):2905–2927. doi: 10.1101/gad.13.22.2905.  [DOI](https://doi.org/10.1101/gad.13.22.2905) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10579998/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genes%20Dev&title=Cellular%20survival:%20a%20play%20in%20three%20Akts&author=SR%20Datta&author=A%20Brunet&author=ME%20Greenberg&volume=13&issue=22&publication_year=1999&pages=2905-2927&pmid=10579998&doi=10.1101/gad.13.22.2905&)

19. Malik SN, Brattain M, Ghosh PM, Troyer DA, Prihoda T, Bedolla R, et al. Immunohistochemical demonstration of phospho-Akt in high Gleason grade prostate cancer. Clin Cancer Res. 2002;8(4):1168–1171.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/11948129/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=Immunohistochemical%20demonstration%20of%20phospho-Akt%20in%20high%20Gleason%20grade%20prostate%20cancer&author=SN%20Malik&author=M%20Brattain&author=PM%20Ghosh&author=DA%20Troyer&author=T%20Prihoda&volume=8&issue=4&publication_year=2002&pages=1168-1171&pmid=11948129&)

20. Roy HK, Olusola BF, Clemens DL, Karolski WJ, Ratashak A, Lynch HT, et al. AKT proto-oncogene overexpression is an early event during sporadic colon carcinogenesis. Carcinogenesis. 2002;23(1):201–205. doi: 10.1093/carcin/23.1.201.  [DOI](https://doi.org/10.1093/carcin/23.1.201) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11756242/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Carcinogenesis&title=AKT%20proto-oncogene%20overexpression%20is%20an%20early%20event%20during%20sporadic%20colon%20carcinogenesis&author=HK%20Roy&author=BF%20Olusola&author=DL%20Clemens&author=WJ%20Karolski&author=A%20Ratashak&volume=23&issue=1&publication_year=2002&pages=201-205&pmid=11756242&doi=10.1093/carcin/23.1.201&)

21. Zhou X, Tan M, Stone Hawthorne V, Klos KS, Lan KH, Yang Y, et al. Activation of the Akt/mammalian target of rapamycin/4E-BP1 pathway by ErbB2 overexpression predicts tumor progression in breast cancers. Clin Cancer Res. 2004;10(20):6779–6788. doi: 10.1158/1078-0432.CCR-04-0112.  [DOI](https://doi.org/10.1158/1078-0432.CCR-04-0112) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15501954/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=Activation%20of%20the%20Akt/mammalian%20target%20of%20rapamycin/4E-BP1%20pathway%20by%20ErbB2%20overexpression%20predicts%20tumor%20progression%20in%20breast%20cancers&author=X%20Zhou&author=M%20Tan&author=V%20Stone%20Hawthorne&author=KS%20Klos&author=KH%20Lan&volume=10&issue=20&publication_year=2004&pages=6779-6788&pmid=15501954&doi=10.1158/1078-0432.CCR-04-0112&)

22. Stahl JM, Sharma A, Cheung M, Zimmerman M, Cheng JQ, Bosenberg MW, et al. Deregulated Akt3 activity promotes development of malignant melanoma. Cancer Res. 2004;64(19):7002–7010. doi: 10.1158/0008-5472.CAN-04-1399.  [DOI](https://doi.org/10.1158/0008-5472.CAN-04-1399) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15466193/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=Deregulated%20Akt3%20activity%20promotes%20development%20of%20malignant%20melanoma&author=JM%20Stahl&author=A%20Sharma&author=M%20Cheung&author=M%20Zimmerman&author=JQ%20Cheng&volume=64&issue=19&publication_year=2004&pages=7002-7010&pmid=15466193&doi=10.1158/0008-5472.CAN-04-1399&)

23. Qiao M, Sheng S, Pardee AB. Metastasis and AKT activation. Cell Cycle. 2008;7(19):2991–2996. doi: 10.4161/cc.7.19.6784.  [DOI](https://doi.org/10.4161/cc.7.19.6784) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18818526/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Cycle&title=Metastasis%20and%20AKT%20activation&author=M%20Qiao&author=S%20Sheng&author=AB%20Pardee&volume=7&issue=19&publication_year=2008&pages=2991-2996&pmid=18818526&doi=10.4161/cc.7.19.6784&)

24. Liu LZ, Zhou XD, Qian G, Shi X, Fang J, Jiang BH. AKT1 amplification regulates cisplatin resistance in human lung cancer cells through the mammalian target of rapamycin/p70S6K1 pathway. Cancer Res. 2007;67(13):6325–6332. doi: 10.1158/0008-5472.CAN-06-4261.  [DOI](https://doi.org/10.1158/0008-5472.CAN-06-4261) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17616691/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=AKT1%20amplification%20regulates%20cisplatin%20resistance%20in%20human%20lung%20cancer%20cells%20through%20the%20mammalian%20target%20of%20rapamycin/p70S6K1%20pathway&author=LZ%20Liu&author=XD%20Zhou&author=G%20Qian&author=X%20Shi&author=J%20Fang&volume=67&issue=13&publication_year=2007&pages=6325-6332&pmid=17616691&doi=10.1158/0008-5472.CAN-06-4261&)

25. Shi H, Hong A, Kong X, Koya RC, Song C, Moriceau G, et al. A novel AKT1 mutant amplifies an adaptive melanoma response to BRAF inhibition. Cancer Discov. 2014;4(1):69–79. doi: 10.1158/2159-8290.CD-13-0279.  [DOI](https://doi.org/10.1158/2159-8290.CD-13-0279) | [PMC free article](/articles/PMC3893054/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24265152/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Discov&title=A%20novel%20AKT1%20mutant%20amplifies%20an%20adaptive%20melanoma%20response%20to%20BRAF%20inhibition&author=H%20Shi&author=A%20Hong&author=X%20Kong&author=RC%20Koya&author=C%20Song&volume=4&issue=1&publication_year=2014&pages=69-79&pmid=24265152&doi=10.1158/2159-8290.CD-13-0279&)

26. Garcia-Echeverria C, Sellers WR. Drug discovery approaches targeting the PI3K/Akt pathway in cancer. Oncogene. 2008;27(41):5511–5526. doi: 10.1038/onc.2008.246.  [DOI](https://doi.org/10.1038/onc.2008.246) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18794885/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncogene&title=Drug%20discovery%20approaches%20targeting%20the%20PI3K/Akt%20pathway%20in%20cancer&author=C%20Garcia-Echeverria&author=WR%20Sellers&volume=27&issue=41&publication_year=2008&pages=5511-5526&pmid=18794885&doi=10.1038/onc.2008.246&)

27. Mundi PS, Sachdev J, McCourt C, Kalinsky K. AKT in cancer: new molecular insights and advances in drug development. Br J Clin Pharmacol. 2016;82(4):943–956. doi: 10.1111/bcp.13021.  [DOI](https://doi.org/10.1111/bcp.13021) | [PMC free article](/articles/PMC5137819/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27232857/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=AKT%20in%20cancer:%20new%20molecular%20insights%20and%20advances%20in%20drug%20development&author=PS%20Mundi&author=J%20Sachdev&author=C%20McCourt&author=K%20Kalinsky&volume=82&issue=4&publication_year=2016&pages=943-956&pmid=27232857&doi=10.1111/bcp.13021&)

28. Nitulescu GM, Margina D, Juzenas P, Peng Q, Olaru OT, Saloustros E, et al. Akt inhibitors in cancer treatment: The long journey from drug discovery to clinical use (Review) Int J Oncol. 2016;48(3):869–885. doi: 10.3892/ijo.2015.3306.  [DOI](https://doi.org/10.3892/ijo.2015.3306) | [PMC free article](/articles/PMC4750533/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26698230/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Oncol&title=Akt%20inhibitors%20in%20cancer%20treatment:%20The%20long%20journey%20from%20drug%20discovery%20to%20clinical%20use%20(Review)&author=GM%20Nitulescu&author=D%20Margina&author=P%20Juzenas&author=Q%20Peng&author=OT%20Olaru&volume=48&issue=3&publication_year=2016&pages=869-885&pmid=26698230&doi=10.3892/ijo.2015.3306&)

29. Balsara BR, Pei J, Mitsuuchi Y, Page R, Klein-Szanto A, Wang H, et al. Frequent activation of AKT in non-small cell lung carcinomas and preneoplastic bronchial lesions. Carcinogenesis. 2004;25(11):2053–2059. doi: 10.1093/carcin/bgh226.  [DOI](https://doi.org/10.1093/carcin/bgh226) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15240509/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Carcinogenesis&title=Frequent%20activation%20of%20AKT%20in%20non-small%20cell%20lung%20carcinomas%20and%20preneoplastic%20bronchial%20lesions&author=BR%20Balsara&author=J%20Pei&author=Y%20Mitsuuchi&author=R%20Page&author=A%20Klein-Szanto&volume=25&issue=11&publication_year=2004&pages=2053-2059&pmid=15240509&doi=10.1093/carcin/bgh226&)

30. Dobashi Y, Koyama S, Kanai Y, Tetsuka K. Kinase-driven pathways of EGFR in lung carcinomas: perspectives on targeting therapy. Front Biosci (Landmark Ed) 2011;16(5):1714–1732. doi: 10.2741/3815.  [DOI](https://doi.org/10.2741/3815) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21196258/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front%20Biosci%20(Landmark%20Ed)&title=Kinase-driven%20pathways%20of%20EGFR%20in%20lung%20carcinomas:%20perspectives%20on%20targeting%20therapy&author=Y%20Dobashi&author=S%20Koyama&author=Y%20Kanai&author=K%20Tetsuka&volume=16&issue=5&publication_year=2011&pages=1714-1732&pmid=21196258&doi=10.2741/3815&)

31. Sos ML, Koker M, Weir BA, Heynck S, Rabinovsky R, Zander T, et al. PTEN loss contributes to erlotinib resistance in EGFR-mutant lung cancer by activation of Akt and EGFR. Cancer Res. 2009;69(8):3256–3261. doi: 10.1158/0008-5472.CAN-08-4055.  [DOI](https://doi.org/10.1158/0008-5472.CAN-08-4055) | [PMC free article](/articles/PMC2849653/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19351834/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=PTEN%20loss%20contributes%20to%20erlotinib%20resistance%20in%20EGFR-mutant%20lung%20cancer%20by%20activation%20of%20Akt%20and%20EGFR&author=ML%20Sos&author=M%20Koker&author=BA%20Weir&author=S%20Heynck&author=R%20Rabinovsky&volume=69&issue=8&publication_year=2009&pages=3256-3261&pmid=19351834&doi=10.1158/0008-5472.CAN-08-4055&)

32. Crezee T, Petrulea M, Piciu D, Jaeger M, Smit JWA, Plantinga TS, et al. Akt1 genetic variants confer increased susceptibility to thyroid cancer. Endocr Connect. 2020;9(11):1065–1074. doi: 10.1530/EC-20-0311.  [DOI](https://doi.org/10.1530/EC-20-0311) | [PMC free article](/articles/PMC7774771/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33112820/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Endocr%20Connect&title=Akt1%20genetic%20variants%20confer%20increased%20susceptibility%20to%20thyroid%20cancer&author=T%20Crezee&author=M%20Petrulea&author=D%20Piciu&author=M%20Jaeger&author=JWA%20Smit&volume=9&issue=11&publication_year=2020&pages=1065-1074&pmid=33112820&doi=10.1530/EC-20-0311&)

33. Zhu J, Wang M, He J, Zhu M, Wang JC, Jin L, et al. Polymorphisms in the AKT1 and AKT2 genes and oesophageal squamous cell carcinoma risk in an Eastern Chinese population. J Cell Mol Med. 2016;20(4):666–677. doi: 10.1111/jcmm.12750.  [DOI](https://doi.org/10.1111/jcmm.12750) | [PMC free article](/articles/PMC5126231/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26828791/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Cell%20Mol%20Med&title=Polymorphisms%20in%20the%20AKT1%20and%20AKT2%20genes%20and%20oesophageal%20squamous%20cell%20carcinoma%20risk%20in%20an%20Eastern%20Chinese%20population&author=J%20Zhu&author=M%20Wang&author=J%20He&author=M%20Zhu&author=JC%20Wang&volume=20&issue=4&publication_year=2016&pages=666-677&pmid=26828791&doi=10.1111/jcmm.12750&)

34. Zhang Z, Chen Q, Zhang J, Wang Y, Hu X, Yin S, et al. Associations of genetic polymorphisms in pTEN/AKT/mTOR signaling pathway genes with cancer risk: a meta-analysis in Asian population. Sci Rep. 2017;7(1):17844. doi: 10.1038/s41598-017-17250-z.  [DOI](https://doi.org/10.1038/s41598-017-17250-z) | [PMC free article](/articles/PMC5736732/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29259266/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sci%20Rep&title=Associations%20of%20genetic%20polymorphisms%20in%20pTEN/AKT/mTOR%20signaling%20pathway%20genes%20with%20cancer%20risk:%20a%20meta-analysis%20in%20Asian%20population&author=Z%20Zhang&author=Q%20Chen&author=J%20Zhang&author=Y%20Wang&author=X%20Hu&volume=7&issue=1&publication_year=2017&pages=17844&pmid=29259266&doi=10.1038/s41598-017-17250-z&)

35. Rascio F, Spadaccino F, Rocchetti MT, Castellano G, Stallone G, Netti GS, et al. The pathogenic role of PI3K/AKT pathway in cancer onset and drug resistance: an updated review. Cancers (Basel) 2021;13(16):3949. doi: 10.3390/cancers13163949.  [DOI](https://doi.org/10.3390/cancers13163949) | [PMC free article](/articles/PMC8394096/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34439105/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancers%20(Basel)&title=The%20pathogenic%20role%20of%20PI3K/AKT%20pathway%20in%20cancer%20onset%20and%20drug%20resistance:%20an%20updated%20review&author=F%20Rascio&author=F%20Spadaccino&author=MT%20Rocchetti&author=G%20Castellano&author=G%20Stallone&volume=13&issue=16&publication_year=2021&pages=3949&pmid=34439105&doi=10.3390/cancers13163949&)

36. Khezri MR, Jafari R, Yousefi K, Zolbanin NM. The PI3K/AKT signaling pathway in cancer: Molecular mechanisms and possible therapeutic interventions. Exp Mol Pathol. 2022;127:104787. doi: 10.1016/j.yexmp.2022.104787.  [DOI](https://doi.org/10.1016/j.yexmp.2022.104787) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35644245/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Exp%20Mol%20Pathol&title=The%20PI3K/AKT%20signaling%20pathway%20in%20cancer:%20Molecular%20mechanisms%20and%20possible%20therapeutic%20interventions&author=MR%20Khezri&author=R%20Jafari&author=K%20Yousefi&author=NM%20Zolbanin&volume=127&publication_year=2022&pages=104787&pmid=35644245&doi=10.1016/j.yexmp.2022.104787&)

37. Lee MW, Kim DS, Lee JH, Lee BS, Lee SH, Jung HL, et al. Roles of AKT1 and AKT2 in non-small cell lung cancer cell survival, growth, and migration. Cancer Sci. 2011;102(10):1822–1828. doi: 10.1111/j.1349-7006.2011.02025.x.  [DOI](https://doi.org/10.1111/j.1349-7006.2011.02025.x) | [PMC free article](/articles/PMC11158224/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21722267/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Sci&title=Roles%20of%20AKT1%20and%20AKT2%20in%20non-small%20cell%20lung%20cancer%20cell%20survival,%20growth,%20and%20migration&author=MW%20Lee&author=DS%20Kim&author=JH%20Lee&author=BS%20Lee&author=SH%20Lee&volume=102&issue=10&publication_year=2011&pages=1822-1828&pmid=21722267&doi=10.1111/j.1349-7006.2011.02025.x&)

38. FitzGerald LM, Zhao S, Leonardson A, Geybels MS, Kolb S, Lin DW, et al. Germline variants in IL4, MGMT and AKT1 are associated with prostate cancer-specific mortality: an analysis of 12,082 prostate cancer cases. Prostate Cancer Prostatic Dis. 2018;21(2):228–237. doi: 10.1038/s41391-017-0029-2.  [DOI](https://doi.org/10.1038/s41391-017-0029-2) | [PMC free article](/articles/PMC6026113/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29298992/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Prostate%20Cancer%20Prostatic%20Dis&title=Germline%20variants%20in%20IL4,%20MGMT%20and%20AKT1%20are%20associated%20with%20prostate%20cancer-specific%20mortality:%20an%20analysis%20of%2012,082%20prostate%20cancer%20cases&author=LM%20FitzGerald&author=S%20Zhao&author=A%20Leonardson&author=MS%20Geybels&author=S%20Kolb&volume=21&issue=2&publication_year=2018&pages=228-237&pmid=29298992&doi=10.1038/s41391-017-0029-2&)
