# Haptoglobin Gene Expression and Anthracycline-Related Cardiomyopathy in Childhood Cancer Survivors: A COG-ALTE03N1 Report

## Metadata
**Authors:** Purnima Singh, David K Crossman, Liting Zhou, Xuexia Wang, Noha Sharafeldin, Lindsey Hageman, Javier G Blanco, Paul W Burridge, Saro H Armenian, Frank M Balis, Douglas S Hawkins, Frank G Keller, Melissa M Hudson, Joseph P Neglia, A Kim Ritchey, Jill P Ginsberg, Wendy Landier, Smita Bhatia
**Journal:** JACC: CardioOncology
**Date:** 2023 Feb 7
**DOI:** [10.1016/j.jaccao.2022.09.009](https://doi.org/10.1016/j.jaccao.2022.09.009)
**PMID:** 37397079
**PMCID:** PMC10308004
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10308004/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC10308004/pdf/main.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC10308004/pdf/main.pdf)

## Abstract

**Background:** 
Anthracycline-related cardiomyopathy is a leading cause of premature death in childhood cancer survivors. The high interindividual variability in risk suggests the need to understand the underlying pathogenesis.

**Objectives:** 
The authors interrogated differentially expressed genes (DEGs) to identify genetic variants serving regulatory functions or genetic variants not easily identified when using genomewide array platforms. Using leads from DEGs, candidate copy number variants (CNVs) and single-nucleotide variants (SNVs) were genotyped.

**Methods:** 
Messenger RNA sequencing was performed on total RNA from peripheral blood of 40 survivors with cardiomyopathy (cases) and 64 matched survivors without cardiomyopathy (control subjects). Conditional logistic regression analysis adjusting for sex, age at cancer diagnosis, anthracycline dose, and chest radiation was used to assess the associations between gene expression and cardiomyopathy and between CNVs and SNVs and cardiomyopathy.

**Results:** 
Haptoglobin (HP) was identified as the top DEG. Participants with higher HP gene expression had 6-fold greater odds of developing cardiomyopathy (OR: 6.4; 95% CI: 1.4-28.6). The HP2-specific allele among the HP genotypes (HP1-1, HP1-2, and HP2-2) had higher transcript levels, as did the G allele among SNVs previously reported to be associated with HP gene expression (rs35283911 and rs2000999). The HP1-2 and HP2-2 genotypes combined with the G/G genotype for rs35283911 and/or rs2000999 placed the survivors at 4-fold greater risk (OR: 3.9; 95% CI: 1.0-14.5) for developing cardiomyopathy.

**Conclusions:** 
These findings provide evidence of a novel association between HP2 allele and cardiomyopathy. HP binds to free hemoglobin to form an HP-hemoglobin complex, thereby preventing oxidative damage from free heme iron, thus providing biological plausibility to the mechanistic basis of the present observation.

Key Words: allele-specific expression, anthracycline-related cardiomyopathy, childhood cancer survivors, ferroptosis, gene expression, haptoglobin, iron overload

### Background

Anthracycline-related cardiomyopathy is a leading cause of premature death in childhood cancer survivors. The high interindividual variability in risk suggests the need to understand the underlying pathogenesis.

### Objectives

The authors interrogated differentially expressed genes (DEGs) to identify genetic variants serving regulatory functions or genetic variants not easily identified when using genomewide array platforms. Using leads from DEGs, candidate copy number variants (CNVs) and single-nucleotide variants (SNVs) were genotyped.

### Methods

Messenger RNA sequencing was performed on total RNA from peripheral blood of 40 survivors with cardiomyopathy (cases) and 64 matched survivors without cardiomyopathy (control subjects). Conditional logistic regression analysis adjusting for sex, age at cancer diagnosis, anthracycline dose, and chest radiation was used to assess the associations between gene expression and cardiomyopathy and between CNVs and SNVs and cardiomyopathy.

### Results

Haptoglobin (*HP*) was identified as the top DEG. Participants with higher *HP* gene expression had 6-fold greater odds of developing cardiomyopathy (OR: 6.4; 95% CI: 1.4-28.6). The *HP2*-specific allele among the *HP* genotypes (HP1-1, HP1-2, and HP2-2) had higher transcript levels, as did the G allele among SNVs previously reported to be associated with *HP* gene expression (rs35283911 and rs2000999). The HP1-2 and HP2-2 genotypes combined with the G/G genotype for rs35283911 and/or rs2000999 placed the survivors at 4-fold greater risk (OR: 3.9; 95% CI: 1.0-14.5) for developing cardiomyopathy.

### Conclusions

These findings provide evidence of a novel association between *HP2* allele and cardiomyopathy. HP binds to free hemoglobin to form an HP-hemoglobin complex, thereby preventing oxidative damage from free heme iron, thus providing biological plausibility to the mechanistic basis of the present observation.

## Methods

### Study design

Study participants were drawn from a Children’s Oncology Group study (COG-ALTE03N1 [Genetic Analysis in Identifying Late-Occurring Complications in Childhood Cancer Survivors]; [NCT00082745](https://clinicaltrials.gov/ct2/show/NCT00082745)) that used a matched case-control design to understand the pathogenesis of cardiomyopathy in childhood cancer survivors. Children’s Oncology Group member institutions enrolled patients after obtaining approval from local Institutional Review Boards. Written informed consent or assent was obtained from patients, parents, or legal guardians. Cases and control subjects were identified from individuals diagnosed with cancer at ≤21 years of age. Cases consisted of childhood cancer survivors who developed cardiomyopathy after exposure to anthracyclines. For each case, patients who had no signs or symptoms of cardiomyopathy after anthracycline exposure were randomly selected as control subjects from the same Children’s Oncology Group childhood cancer survivor cohort, matched on primary cancer diagnosis, year of diagnosis (±5 years), and race/ethnicity. The selected control subjects also needed to have a longer duration of cardiomyopathy-free follow-up compared with time from cancer diagnosis to cardiomyopathy for the corresponding case. Participants provided blood samples in PAXgene blood RNA tubes for RNA and blood samples in dipotassium ethylenediaminetetraacetic acid tubes or saliva in Oragene kits (DNA Genotek) for DNA.

Cases fulfilled the American Heart Association criteria for cardiac compromise by presenting with signs and/or symptoms (dyspnea, orthopnea, fatigue, edema, hepatomegaly, and/or rales). In the absence of signs or symptoms, cases had echocardiographic features of left ventricular dysfunction (ejection fraction ≤40% and/or fractional shortening ≤ 28%). Lifetime anthracycline exposure was calculated by multiplying the cumulative dose (milligrams per square meter) of individual anthracyclines by a factor that reflects the drug’s cardiotoxic potential[12](#bib12) and then summing the results. Radiation to the chest with heart in the field was captured as a yes-or-no variable.

### RNA isolation, library construction, and sequencing

RNA was isolated using the PAXgene whole-blood RNA kit (Qiagen). RNA concentration was measured using a Nanodrop ND-1000 Spectrophotometer (Thermo Fisher Scientific). RNA quality was checked on Bioanalyzer Nanochip (Agilent Technologies) and samples with RNA integrity numbers >7 were submitted to the Genomic Services Laboratory at the HudsonAlpha Institute for Biotechnology in Huntsville, Alabama. Polyadenylated RNAs were isolated using NEBNext Magnetic Oligo d(T)25 beads. Libraries were prepared using the TruSeq RNA Sample Preparation Kit (Illumina). Each library was pair-end sequenced (100 bp) by using the TruSeq SBS Kit v4-HS (Illumina), on a HiSeq 2500 platform. Raw reads were demultiplexed using bcl2fastq Conversion Software (Illumina) with default settings.

### Differential gene expression analysis

Trim Galore! ([https://www.bioinformatics.babraham.ac.uk/projects/trim_galore/](https://www.bioinformatics.babraham.ac.uk/projects/trim_galore/)) was used to trim off primer adapter sequences found in the raw FASTQ files. STAR was used to align the trimmed RNA-seq FASTQ reads to the human reference genome from GENCODE (GRCh38 p7 release 25).[13](#bib13) HTSeq-count was used to count the number of reads mapping to each gene from the STAR alignments.[14](#bib14) Normalization and differential expression was then applied to the count files using DESeq2.[15](#bib15) The *q* values (adjusted *P* values) from DESeq2 are adjusted for multiple testing using the Benjamini-Hochberg procedure, which controls false discovery rate.

### Transcript expression analysis

Following mapping of the reads to the reference genome(GRCh38 p13 release 39), the mapped reads were assembled using Cufflinks.[16](#bib16) Cuffnorm was used to estimate expression levels of individual transcripts assembled by Cufflinks. Cuffnorm output contains normalized expression levels for all gene-specific transcripts. Transcripts with fragments per kilobase of transcript per million mapped reads (FPKM) ≥1 were used to estimate allele-specific transcript abundance.

### DNA isolation and genotyping

DNA was isolated using the Gentra Puregene blood kit (Qiagen) or prepIT•L2P reagent (saliva). DNA concentration was measured using a Nanodrop ND-1000 Spectrophotometer. Copy number variants (CNVs) were determined using polymerase chain reaction (PCR) amplification devised by Koch et al[17](#bib17) with minor modifications.[18](#bib18) PCR products from DNA samples were run on 1% and 2% agarose gel (E-Gel agarose gels, Invitrogen, Thermo Fisher Scientific) using an E-Gel Power Snap Electrophoresis System. Genotyping for single-nucleotide variants (SNVs) was performed on the Juno system (Fluidigm) and 96.96 Genotyping IFCs. Endpoint fluorescence values were measured on the BioMark HD system, and Fluidigm SNP Genotyping Analysis software was used to generate genotyping calls for each sample.

### Statistical analysis

Statistical analyses were conducted using JMP 16 Pro (JMP Statistical Discovery) and SAS version 9.4 (SAS Institute). We summarized the characteristics of the study participants according to their case-control status and presented as mean ± SD and median with IQR for continuous variables. Categorical variables are shown as absolute numbers and percentages. Continuous and categorical variables were examined using Wilcoxon rank sum or Kruskal-Wallis tests and Fisher exact or chi-square tests, respectively. Multivariable conditional logistic regression analysis (adjusting for age at diagnosis of primary cancer [continuous variable], sex [male vs female]), anthracycline dose [<250 mg/m^2^ vs ≥250 mg/m^2^], and chest radiation [yes vs no]) was used to determine the association between gene-specific RNA counts and gene expression–related genetic variants with cardiomyopathy. Model results are presented as ORs and corresponding 95% CIs.

## Results

### Patient characteristics

The median ages at primary cancer diagnosis for the 40 cases and 64 matched control subjects were 8.2 years (IQR: 3.6-13.9 years) and 9.7 years (IQR: 3.3-14.4 years), respectively ([Table 1](#tbl1)). Cases had received higher cumulative anthracycline exposure compared with controls (≥250 mg/m^2^, 62.5% vs 35.9%; *P* = 0.008) and were more likely to have received chest radiation (47.5% vs 20.3%; *P* = 0.003). The median time between cancer diagnosis and cardiomyopathy for cases was 5.3 years (IQR: 0.8-12.8 years); control subjects were followed for a significantly longer period (median 10.1 years; IQR: 7.1-14.5 years; *P* < 0.001). The phenotypic characteristics of the cases are presented in [Supplemental Table 1](#mmc1).

### Table 1.

|   | Cases (n = 40) | Controls (n = 64) | P Valuea |
| --- | --- | --- | --- |
| Age at primary cancer diagnosis, y | 8.2 (3.6-13.9) | 9.7 (3.3-14.4) | 0.97 |
| Sex |   |   |   |
| Female | 24 (60.0) | 34 (53.1) | 0.49 |
| Cumulative anthracycline exposure |   |   |   |
| ≥250 mg/m2 | 25 (62.5) | 23 (35.9) | 0.008 |
| Chest radiation |   |   |   |
| Yes | 19 (47.5) | 13 (20.3) | 0.003 |
| Race/ethnicity |   |   |   |
| Non-Hispanic White | 23 (57.5) | 37 (57.8) | Matched |
| Hispanic | 9 (22.5) | 16 (25.0) |   |
| Black/African American | 5 (12.5) | 7 (10.9) |   |
| Asian | 3 (7.5) | 3 (4.7) |   |
| Mixed race/ethnicity | 0 (0.0) | 1 (1.6) |   |
| Primary diagnosis |   |   |   |
| Acute lymphoblastic leukemia | 9 (22.5) | 16 (25.0) | Matched |
| Acute myeloid leukemia | 2 (5.0) | 3 (4.7) |   |
| Ewing sarcoma | 4 (10.0) | 8 (12.5) |   |
| Hodgkin lymphoma | 7 (17.5) | 10 (15.6) |   |
| Kidney tumors | 2 (5.0) | 2 (3.1) |   |
| Neuroblastoma | 5 (12.5) | 8 (12.5) |   |
| Non-Hodgkin lymphoma | 5 (12.5) | 8 (12.5) |   |
| Osteosarcoma | 4 (10.0) | 7 (10.9) |   |
| Soft tissue sarcoma | 2 (5.0) | 2 (3.1) |   |
| Time from diagnosis to cardiac event for cases or time to enrollment for control subjects, y | 5.3 (0.8-12.8) | 10.1 (7.1-14.5) | <0.001 |
| HP RNA count | 135.3 (63.7-300.2) | 70.3 (47.5-105.8) | <0.001 |

Table 1 Caption: Demographic and Clinical Characteristics by Case-Control Status

### Gene expression and anthracycline-related cardiomyopathy

Overall, 43,198 genes (protein coding, long noncoding RNA, processed pseudogenes, unprocessed pseudogenes, miscellaneous RNA, small nuclear RNA, small nucleolar RNA, and transcribed pseudogenes) were expressed in total RNA from peripheral blood. We filtered 28,026 low-expressed genes (RNA counts <10 in ≥70% of the samples). The remaining 15,172 genes included protein-coding genes (82%), long noncoding RNA, pseudogenes, and immunoglobulin genes. Using DESeq2, we identified 36 DEGs with adjusted *P* values <0.05 and absolute fold changes >2 ([Supplemental Table 2](#mmc1)); 35 were up-regulated among cases, and 1 was down-regulated. Prioritization of these 36 genes with >50% of matched sets showing differential expression between the cases and their matched control subjects, as well as previously published association with heart disease, identified haptoglobin (*HP*) as the top differentially expressed gene, with 25 of 40 cases (62.5%) having higher expression compared with control subjects. In the present report we focus on *HP* gene expression and its association with anthracycline-related cardiomyopathy.

### HP gene expression and anthracycline-related cardiomyopathy

[Figure 1](#fig1) shows the normalized aggregate read RNA counts of matched case-control sets plotted side by side for the *HP* gene. Overall, median expression of *HP* gene was higher among cases compared with control subjects (median RNA count 135.3 [IQR: 63.7-300.2] vs 70.3 [IQR: 47.5-105.8]; *P* < 0.001). Dichotomizing *HP* RNA counts at the median level for the control subjects, into low vs high gene expression (≤70 vs >70), we found that 75% of cases had high *HP* expression compared with 50% of control subjects (*P* = 0.011) Multivariable conditional logistic regression analysis, adjusted for age at diagnosis of primary cancer, sex, anthracycline exposure, and chest radiation, showed that childhood cancer survivors with *HP* RNA counts >70 (higher gene expression) had 6-fold higher odds of developing cardiomyopathy (OR: 6.4; 95% CI: 1.4-28.6; *P* = 0.014) compared with those with *HP* RNA counts ≤70 (lower gene expression).

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/13b3/10308004/153e7065e5c1/gr1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=10308004_gr1.jpg)

HP Gene Expression The normalized aggregate read RNA counts of haptoglobin (HP) gene are shown from RNA sequencing in anthracycline-related cardiomyopathy. The matched case-control sets are plotted side by side. Matched cases are indicated in red, and control subjects are indicated in green.

### HP genotype and HP gene expression levels

*HP* gene exists as 2 CNVs (*HP1* and *HP2*), yielding 3 genotypes: HP1-1, HP1-2, and HP2-2.[19](#bib19) Of the 104 participants in this study, we had genotype data for only 98 because of either unavailability of DNA (n = 4) or inconclusive PCR results (n = 2). *HP* genotypes showed the following distribution in 98 genotyped samples: 16.3% for HP1-1, 78.5% for HP1-2, and 5% for HP2-2. *HP*^del^ and [NM_001126102.1](https://www.ncbi.nlm.nih.gov/nuccore/NM_001126102.1):c.190 + 1 (splice donor mutation) were not found. The median RNA counts did not differ significantly among the 3 genotypes (HP1-1, 91.0; HP1-2, 84.2; and HP2-2, 73.4; *P* = 0.86) ([Table 2](#tbl2)). However, to gain insight into *HP1* and *HP2* allele-specific transcripts, we assessed 40 cases matched 1:1 with control subjects (n = 80) and finally analyzed (n = 74) those with genotyping data ([Supplemental Results](#mmc1), [Supplemental Table 3](#mmc1)). FPKM represented transcript abundance in the paired-end RNA-seq data. *HP2* allele had higher transcript levels compared with *HP1* allele (median FPKM level 3.6 [IQR: 1.6-7.3] vs 0 [IQR: 0-3.4]), regardless of case-control status ([Table 2](#tbl2)). Furthermore, as shown in [Table 3](#tbl3), cases had higher median *HP2* cumulative FPKM transcript levels compared with control subjects (median 5.1 [IQR: 2.1-11.8] vs 2.9 [IQR: 1.4-4.4]; *P* = 0.027).

### Table 2.

| HP RNA counts by HP genotype |  |  |  |  |
| --- | --- | --- | --- | --- |
| HP RNA Counts | HP1-1 (n = 16) | HP1-2 (n = 77) | HP2-2 (n = 5) | P Valuea |
| Median (IQR) | 91.0 (24.0-155.4) | 84.2 (55.8-166.4) | 73.4 (48.7-483.5) | 0.86 |
| ≤70, n (%) | 7 (43.7) | 31 (40.3) | 2 (40.0) | 0.96 |
| >70, n (%) | 9 (56.2) | 46 (59.7) | 3 (60.0) |  |

Table 2 Caption: Genetic Determinants of HP RNA Expression in Childhood Cancer Survivors

### Table 3.

|   | Cases | Control Subjects | P Valuea |
| --- | --- | --- | --- |
| HP2 allele (4 transcripts in HP1-2 and HP2-2) |  |  |  |
| Cumulative FPKM | (n = 31) | (n = 32) |   |
| Median (IQR) | 5.1 (2.1-11.8) | 2.9 (1.4-4.4) | 0.027 |
| HP1 allele (2 transcripts in HP1-1 and HP1-2) |  |  |  |
| Cumulative FPKM | (n = 35) | (n = 34) |   |
| Median (IQR) | 3.9 (1.2-7.9) | 2.2 (0-4.3) | 0.11 |

Table 3 Caption: HP Allele-Specific Transcript Abundance in Cases and Control Subjects

### Additional genetic determinants of HP gene expression levels

SNVs rs35283911 and rs2000999 are known to be associated with *HP* gene expression (independent of *HP* CNV status).[20](#bib20), [21](#bib21), [22](#bib22) Genotyping assays for SNVs rs2000999, G>A, rs35283911 Del, G>-, and [NM_001126102.1](https://www.ncbi.nlm.nih.gov/nuccore/NM_001126102.1):c.190 + 1 SNV, G>C were designed using Fluidigm D3 assay design ([Supplemental Figure 1](#mmc1)). As shown in [Table 2](#tbl2), the major allele (G) for both rs35283911 and rs2000999 was associated with higher *HP* expression. rs35283911 [G] was associated with higher *HP* gene expression (median RNA count G/G, 94.5; G/-, 68.8; and -/-, 25.7; *P* = 0.033). Similarly, rs2000999 [G] was also associated with higher *HP* expression (median RNA count G/G, 95.4; G/A, 63.8; and A/A, 25.7; *P* = 0.008).

### Genetic determinants of HP gene expression and risk for anthracycline-related cardiomyopathy

The *HP* genotype (model A) and the 2 SNVs taken together (rs35283911 and rs2000999) (model B) demonstrated 3.2-fold and 1.8-fold higher odds of cardiomyopathy, respectively, but these associations did not reach statistical significance ([Table 4](#tbl4)). We then examined the combined effect of HP genotype and the 2 SNVs, (rs35283911 and rs2000999), by creating a combined high-risk and low-risk group. The high-risk group consisted of patients with *HP2* allele (HP1-2 or HP2-2) who also had rs35283911 [G/G] or rs2000999 [G/G] genotype; the low-risk group included all others (model C). The high-risk group showed significantly higher *HP* expression (median RNA count 96.1 vs 68.8; *P* = 0.006). Multivariable conditional logistic regression analysis adjusted for age at diagnosis of primary cancer, sex, and anthracycline and chest radiation showed that patients in the high-risk group were 4 times more likely to develop cardiomyopathy (OR: 3.9; 95% CI: 1.0-14.5; *P* = 0.045) compared with those in the low-risk group ([Table 4](#tbl4)). The unadjusted conditional logistic regression analysis containing only the genetic variables did not show a statistically significant association.

### Table 4.

|   | Unadjusteda |  | Adjustedb |  |
| --- | --- | --- | --- | --- |
| OR (95% CI) | P Value | OR (95% CI) | P Value |  |
| HP CNV status (model A) |  |  |  |  |
| HP1-1 | Reference | 0.54 | Reference | 0.17 |
| HP1-2 or HP2-2 | 1.5 (0.42-5.2) | 3.2 (0.61-17.0) |  |  |
| Combined HP SNV status (model B) |  |  |  |  |
| rs2000999 [A/A] or [G/A] and rs35283911 [-/-] or [G/-] | Reference | 0.36 | Reference | 0.33 |
| rs2000999 [G/G] or rs35283911 [G/G] | 1.6 (0.58-4.5) | 1.8 (0.5-6.2) |  |  |
| Composite HP CNV and SNV status (model C) |  |  |  |  |
| HP1-1 and rs2000999 [A/A] or [G/A] and rs35283911 [-/-] or [G/-] | Reference | 0.14 | Reference | 0.045 |
| HP1-2 or HP2-2 and rs2000999 [G/G] or rs35283911 [G/G] | 2.0 (0.79-5.1) | 3.9 (1.0-14.5) |  |  |

Table 4 Caption: Unadjusted and Adjusted Models for Genetic Determinants of HP Gene Expression and Anthracycline-Related Cardiomyopathy

## Discussion

We show that childhood cancer survivors with anthracycline-related cardiomyopathy have significantly higher levels of whole blood *HP* gene expression compared with survivors without cardiomyopathy. Those with *HP* RNA counts >70 have 6-fold higher adjusted odds of developing cardiomyopathy. Cancer survivors with cardiomyopathy have higher *HP2* cumulative transcript levels compared with control subjects. Finally, patients with the HP1-2 or HP2-2 genotype and rs35283911 [G/G] and rs2000999 [G/G] genotypes are more likely to have higher *HP* gene expression and 3.9 times greater odds of developing cardiomyopathy, compared with those with HP1 allele or rs35283911 [delG].

Cell-free hemoglobin (HB) promotes accumulation of hydroxyl radicals and reactive oxygen species.[23](#bib23) HP is a mammalian plasma protein that binds to free HB to form the HP-HB complex that is phagocytosed by monocyte-macrophages by binding to receptor CD163,[24](#bib24), [25](#bib25), [26](#bib26) thereby preventing oxidative damage from free heme iron.[27](#bib27) The *HP* locus is polymorphic in humans. The homozygous allelic deletion for *HP* (*HP*^del^) is also known to occur, albeit at a very low frequency.[28](#bib28) An intragenic duplication of exons 3 and 4 of the ancestral *HP* gene produced *HP2*,[29](#bib29) yielding 3 genotypes: HP1-1, HP1-2, and HP2-2.[19](#bib19) *HP1* and *HP2* differ in their ability to clear HB and differ functionally in their ability to protect from HB-induced oxidative stress.[30](#bib30) *HP1* carriers have a higher HP plasma protein concentration compared with *HP2* carriers.[21](#bib21)^,^[24](#bib24)^,^[27](#bib27)^,^[31](#bib31), [32](#bib32), [33](#bib33) HP1-1 linear polymer has the highest affinity for cell-free HB and thereby superior antioxidant capacity compared with HP2-2-ring polymer,[33](#bib33), [34](#bib34), [35](#bib35), [36](#bib36) by inducing a rapid clearance of HB from circulation, indicating that the potential for tissue damage is highest for individuals with the HP2-2 genotype and lowest for those with the HP1-1 genotype. Furthermore, CD163-mediated endocytosis into the macrophages is altered in patients with the HP2-2 genotype.

Preclinical models support this observation. The amount of myocardial injury after radiation was Hp genotype dependent in mice containing the Hp2 allele (generated by introducing the human *HP2* allele, as the HP2 allele does not exist in mice).[37](#bib37) Although the Hp2 mice demonstrated increased production of several lipid peroxidation products in the myocardium after radiation, the Hp1 mice demonstrated increased production of the anti-inflammatory and antioxidant cytokine interleukin-10. Pharmacologic administration of an antioxidant reduced myocardial infarct size in this model. In childhood cancer survivors, the association between anthracycline exposure and cardiomyopathy is potentiated by chest radiation. In the present study, 47.5% of survivors with cardiomyopathy had received chest radiation compared with 20% of survivors without. Unfortunately, we could not evaluate the sole effect of radiation to the heart, as all patients were anthracycline exposed. HP CNVs (HP1-1, HP1-2, and HP2-2) have been identified as prognostic biomarkers of vascular disease, myocardial infarction, hypertension, heart failure, and angiogenesis in nononcology populations.[38](#bib38) Free HB causes vascular injury as it intercalates into cell membranes. Once inside the cell, heme is broken down to release redox-active iron, which oxidizes low-density lipoproteins, promoting foam cell formation and vascular endothelial cell apoptosis and exacerbating plaque instability, resulting in plaque rupture. Following intraplaque hemorrhage, large amounts of cell-free HB accumulate within the atheroma because of impaired clearance by the HP-CD163 scavenging system. HP binds free HB, thus preventing this cascade of events. The potential for plaque instability is accentuated in patients with the HP2-2 genotype.[39](#bib39), [40](#bib40), [41](#bib41) The presence of the HP2-2 genotype and low plasma levels of HP (<1.4 g/L) were associated with detrimental outcomes following acute myocardial infarction.[42](#bib42) Increased intraplaque iron deposition in patients with the HP2-2 genotype may be responsible for increased oxidative stress and instability of the carotid plaque.[41](#bib41)

These observations in the nononcology population align with our observations that cancer survivors with cardiomyopathy have higher *HP2* cumulative transcript levels compared with control subjects. There is evidence to suggest that iron plays a role in anthracycline-related cardiomyopathy.[43](#bib43) Heme iron catalyzes lipid peroxidation, which is a hallmark of ferroptosis.[44](#bib44) Iron-dependent ferroptosis has been shown to have a role in anthracycline-induced cardiomyopathy.[45](#bib45) Inhibition of ferroptosis by ferrostatin-1 significantly reduces cardiomyopathy risk. The cardioprotective properties of dexrazoxane, a compound that chelates iron, supports this hypothesis.[46](#bib46) We propose that the alteration of iron homeostasis by anthracyclines and the associated heart damage due to iron accumulation[43](#bib43)^,^[47](#bib47)^,^[48](#bib48) are further exacerbated by ineffective clearance of HB because of the presence of the *HP2* allele ([Central Illustration](#undfig2)), thus predisposing the patient to anthracycline-induced cardiomyopathy.

### Central Illustration.

![Central Illustration](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/13b3/10308004/6fe5f51f8ac0/gr2.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=10308004_gr2.jpg)

HP and Anthracycline-Related Cardiotoxicity The figure shows differences in the clearance of hemoglobin (HB) by HP1 and HP2 carriers. On the left is the efficient clearance of HB by CD163 scavenger receptor in HP1-1 carriers, whereas HB:HP2-2 is not effectively cleared by CD163 receptor because of lower levels of CD163 and inefficient endocytosis. The inefficient clearance of HB-HP2-2 leads to increased iron and lipid peroxidation that when compounded by iron overload from anthracycline exposure renders the HP2 allele carriers more susceptible to ferroptosis and cardiomyopathy. HP = haptoglobin; ROS = reactive oxygen species.

Determining whether dysregulated gene expression is the direct result of the outcome (cardiomyopathy) or plays a role in the pathogenesis can be difficult. SNVs such as rs35283911, rs2000999, and [NM_001126102.1](https://www.ncbi.nlm.nih.gov/nuccore/NM_001126102.1):c.190 + 1G>C are associated with *HP* gene expression.[20](#bib20) rs35283911 is a single-base-pair deletion (delG, intron variant in *TXNL4B*) upstream of *HP*, occurs in the background of *HP2*, and influences *HP* RNA and protein levels.[21](#bib21) rs2000999 (intron variant in *TXNL4B* and *HPR*) is downstream of the *HP* gene, and its effect is likely mediated via linkage disequilibrium with rs35283911.[22](#bib22) Up to 45% of the variance in serum HP is explained by rs2000999.[22](#bib22) In our study of childhood cancer survivors, we found that the major allele (G) for both rs35283911 and rs2000999 was associated with higher *HP* expression. A previous study showed that rs2000999 and *HP* CNVs influence HP levels independently.[49](#bib49) Our findings are consistent with these observation; patients with the HP1-2 or HP2-2 genotype and with the rs35283911 [G/G] and rs2000999 [G/G] genotypes have higher *HP* gene expression and 3.9 times greater odds of developing cardiomyopathy. The association between the SNVs or CNVs and HP gene expression and cardiomyopathy provides us with evidence to state that initial exposure to anthracyclines triggers heart damage because of iron accumulation and that this is compounded by ineffective clearance of cell-free HB because of the presence of the *HP2* allele, thus predisposing the patient to anthracycline-induced cardiomyopathy.

### Study limitations

Ideally, gene expression should be measured in the affected tissue (ie, cardiac tissue). HP binds to cell-free HB and prevents iron-mediated formation of free reactive oxygen species that cause myocardial injury and vascular damage. Thus, the ideal experiment would be to examine the extent of iron overload and ferroptosis in the myocardium from patients with high *HP* expression. However such experiments were not logistically feasible. Finally, radiation dose to the heart was not available to us; instead, we relied on receipt of radiation to the chest (with the heart in the field) as a yes-or-no variable.

The present study is the first to identify an association between *HP* RNA expression in peripheral blood in the context of genetic determinants of *HP* expression and the subsequent risk for anthracycline-related cardiomyopathy in childhood cancer survivors. Because of the structure of HP1 and HP2 CNVs, it is difficult to identify them using classic genomewide association studies, presenting the need to conduct studies such as ours to identify novel variants using differential gene expression analysis. It is important to note that the *HP* CNV was genotyped using a gold-standard PCR approach and not determined by genotype imputation. The biologic plausibility of the association between *HP* and anthracycline-related cardiomyopathy, as well as the previous demonstration of the association between HP and cardiometabolic traits that potentiate the risk for cardiomyopathy provide credence to this association in childhood cancer survivors with anthracycline-related cardiomyopathy.

## Conclusions

The present study demonstrates that *HP* gene expression in blood is associated with anthracycline-related cardiomyopathy in childhood cancer survivors and that key genetic determinants contribute to *HP* expression and the risk for anthracycline-related cardiomyopathy. These findings provide evidence for the role of genetic variants associated with *HP* RNA levels in anthracycline-related cardiac dysfunction as well as a possible mechanistic explanation of the role of oxidative damage due to anthracycline exposure. Upon validation of these observations in an independent population, these findings could result in a blood-based screening test with limited invasiveness and high acceptance. Furthermore, patients carrying the HP2-2 genotype could derive more benefit from dexrazoxane, given that it binds free iron and/or removes iron from the doxorubicin-iron complex, thereby preventing oxygen free radical formation.

**COMPETENCY IN PATIENT CARE AND PROCEDURAL SKILLS:***HP* gene expression in peripheral blood is associated with anthracycline-related cardiomyopathy in childhood cancer survivors, and key genetic determinants contribute to *HP* expression and the risk for cardiomyopathy.

**TRANSLATIONAL OUTLOOK:** These findings provide a possible mechanistic explanation of the role of oxidative damage due to anthracycline exposure and iron-dependent ferroptosis in anthracycline-related cardiomyopathy. The ability to demonstrate these findings using peripheral blood could result in a blood-based screening test with limited invasiveness and high acceptance.

**COMPETENCY IN PATIENT CARE AND PROCEDURAL SKILLS:***HP* gene expression in peripheral blood is associated with anthracycline-related cardiomyopathy in childhood cancer survivors, and key genetic determinants contribute to *HP* expression and the risk for cardiomyopathy.

**TRANSLATIONAL OUTLOOK:** These findings provide a possible mechanistic explanation of the role of oxidative damage due to anthracycline exposure and iron-dependent ferroptosis in anthracycline-related cardiomyopathy. The ability to demonstrate these findings using peripheral blood could result in a blood-based screening test with limited invasiveness and high acceptance.

## Funding Support and Author Disclosures

This research is supported by the National Cancer Institute (R35CA220502; principal investigator [PI], S. Bhatia), Leukemia and Lymphoma Society (6563-19; PI, S. Bhatia), and the V Foundation (DT2019-010; PI, S. Bhatia). The Children’s Oncology Group study (COG-ALTE03N1 [[NCT00082745](https://clinicaltrials.gov/ct2/show/NCT00082745)]; PI, S. Bhatia) reported here is supported by the National Clinical Trials Network Operations Center Grant (U10CA180886; PI, D.S. Hawkins), the National Clinical Trials Network Statistics & Data Center Grant (U10CA180899; PI, Alonzo), the Children’s Oncology Group Chair’s Grant (U10CA098543; PI, Adamson), the Children’s Oncology Group Statistics & Data Center Grant (U10CA098413; PI, Anderson), the National Cancer Institute Community Oncology Research Program Grant (UG1CA189955; PI, Pollock), and the Community Clinical Oncology Program Grant (U10CA095861; PI, Pollock), and the St. Baldrick’s Foundation through an unrestricted grant. The content is solely the responsibility of the authors and does not necessarily represent the official views of the National Institutes of Health. The authors have reported that they have no relationships relevant to the contents of this paper to disclose.

## Footnotes

## Appendix

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Armstrong G.T., Chen Y., Yasui Y., et al. Reduction in late mortality among 5-year survivors of childhood cancer. N Engl J Med. 2016;374:833–842. doi: 10.1056/NEJMoa1510795.  [DOI](https://doi.org/10.1056/NEJMoa1510795) | [PMC free article](/articles/PMC4786452/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26761625/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%C2%A0Engl%20J%20Med&title=Reduction%20in%20late%20mortality%20among%205-year%20survivors%20of%20childhood%20cancer&author=G.T.%20Armstrong&author=Y.%20Chen&author=Y.%20Yasui&volume=374&publication_year=2016&pages=833-842&pmid=26761625&doi=10.1056/NEJMoa1510795&)

2. Bhatia S. Genetics of anthracycline cardiomyopathy in cancer survivors: JACC: CardioOncology state-of-the-art review. J Am Coll Cardiol CardioOnc. 2020;2:539–552. doi: 10.1016/j.jaccao.2020.09.006.  [DOI](https://doi.org/10.1016/j.jaccao.2020.09.006) | [PMC free article](/articles/PMC7757557/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33364618/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%C2%A0Am%20Coll%20Cardiol%20CardioOnc&title=Genetics%20of%20anthracycline%20cardiomyopathy%20in%20cancer%20survivors:%20JACC:%20CardioOncology%20state-of-the-art%20review&author=S.%20Bhatia&volume=2&publication_year=2020&pages=539-552&pmid=33364618&doi=10.1016/j.jaccao.2020.09.006&)

3. Wang X., Sun C.L., Quinones-Lombrana A., et al. CELF4 variant and anthracycline-related cardiomyopathy: a Children’s Oncology Group genome-wide association study. J Clin Oncol. 2016;34:863–870. doi: 10.1200/JCO.2015.63.4550.  [DOI](https://doi.org/10.1200/JCO.2015.63.4550) | [PMC free article](/articles/PMC5070560/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26811534/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%C2%A0Clin%20Oncol&title=CELF4%20variant%20and%20anthracycline-related%20cardiomyopathy:%20a%20Children%E2%80%99s%20Oncology%20Group%20genome-wide%20association%20study&author=X.%20Wang&author=C.L.%20Sun&author=A.%20Quinones-Lombrana&volume=34&publication_year=2016&pages=863-870&pmid=26811534&doi=10.1200/JCO.2015.63.4550&)

4. Magdy T., Jouni M., Kuo H.H., et al. Identification of drug transporter genomic variants and inhibitors that protect against doxorubicin-induced cardiotoxicity. Circulation. 2022;145:279–294. doi: 10.1161/CIRCULATIONAHA.121.055801.  [DOI](https://doi.org/10.1161/CIRCULATIONAHA.121.055801) | [PMC free article](/articles/PMC8792344/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34874743/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation&title=Identification%20of%20drug%20transporter%20genomic%20variants%20and%20inhibitors%20that%20protect%20against%20doxorubicin-induced%20cardiotoxicity&author=T.%20Magdy&author=M.%20Jouni&author=H.H.%20Kuo&volume=145&publication_year=2022&pages=279-294&pmid=34874743&doi=10.1161/CIRCULATIONAHA.121.055801&)

5. Leong S.L., Chaiyakunapruk N., Lee S.W. Candidate gene association studies of anthracycline-induced cardiotoxicity: a systematic review and meta-analysis. Sci Rep. 2017;7:39. doi: 10.1038/s41598-017-00075-1.  [DOI](https://doi.org/10.1038/s41598-017-00075-1) | [PMC free article](/articles/PMC5428315/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28232737/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sci%20Rep&title=Candidate%20gene%20association%20studies%20of%20anthracycline-induced%20cardiotoxicity:%20a%20systematic%20review%20and%20meta-analysis&author=S.L.%20Leong&author=N.%20Chaiyakunapruk&author=S.W.%20Lee&volume=7&publication_year=2017&pages=39&pmid=28232737&doi=10.1038/s41598-017-00075-1&)

6. Umans B.D., Battle A., Gilad Y. Where are the disease-associated eQTLs? Trends Genet. 2021;37:109–124. doi: 10.1016/j.tig.2020.08.009.  [DOI](https://doi.org/10.1016/j.tig.2020.08.009) | [PMC free article](/articles/PMC8162831/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32912663/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Trends%20Genet&title=Where%20are%20the%20disease-associated%20eQTLs?&author=B.D.%20Umans&author=A.%20Battle&author=Y.%20Gilad&volume=37&publication_year=2021&pages=109-124&pmid=32912663&doi=10.1016/j.tig.2020.08.009&)

7. Yao D.W., O’Connor L.J., Price A.L., Gusev A. Quantifying genetic effects on disease mediated by assayed gene expression levels. Nat Genet. 2020;52:626–633. doi: 10.1038/s41588-020-0625-2.  [DOI](https://doi.org/10.1038/s41588-020-0625-2) | [PMC free article](/articles/PMC7276299/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32424349/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Quantifying%20genetic%20effects%20on%20disease%20mediated%20by%20assayed%20gene%20expression%20levels&author=D.W.%20Yao&author=L.J.%20O%E2%80%99Connor&author=A.L.%20Price&author=A.%20Gusev&volume=52&publication_year=2020&pages=626-633&pmid=32424349&doi=10.1038/s41588-020-0625-2&)

8. Verlouw J.A.M., Clemens E., de Vries J.H., et al. A comparison of genotyping arrays. Eur J Hum Genet. 2021;29:1611–1624. doi: 10.1038/s41431-021-00917-7.  [DOI](https://doi.org/10.1038/s41431-021-00917-7) | [PMC free article](/articles/PMC8560858/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34140649/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Hum%20Genet&title=A%C2%A0comparison%20of%20genotyping%20arrays&author=J.A.M.%20Verlouw&author=E.%20Clemens&author=J.H.%20de%20Vries&volume=29&publication_year=2021&pages=1611-1624&pmid=34140649&doi=10.1038/s41431-021-00917-7&)

9. Singh P., Wang X., Hageman L., et al. Association of GSTM1 null variant with anthracycline-related cardiomyopathy after childhood cancer—a Children’s Oncology Group ALTE03N1 report. Cancer. 2020;126:4051–4058. doi: 10.1002/cncr.32948.  [DOI](https://doi.org/10.1002/cncr.32948) | [PMC free article](/articles/PMC7423633/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32413235/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer&title=Association%20of%20GSTM1%20null%20variant%20with%20anthracycline-related%20cardiomyopathy%20after%20childhood%20cancer%E2%80%94a%20Children%E2%80%99s%20Oncology%20Group%20ALTE03N1%20report&author=P.%20Singh&author=X.%20Wang&author=L.%20Hageman&volume=126&publication_year=2020&pages=4051-4058&pmid=32413235&doi=10.1002/cncr.32948&)

10. Wang Z., Gerstein M., Snyder M. RNA-seq: a revolutionary tool for transcriptomics. Nat Rev Genet. 2009;10:57–63. doi: 10.1038/nrg2484.  [DOI](https://doi.org/10.1038/nrg2484) | [PMC free article](/articles/PMC2949280/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19015660/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Genet&title=RNA-seq:%20a%20revolutionary%20tool%20for%20transcriptomics&author=Z.%20Wang&author=M.%20Gerstein&author=M.%20Snyder&volume=10&publication_year=2009&pages=57-63&pmid=19015660&doi=10.1038/nrg2484&)

11. Zhao W., He X., Hoadley K.A., Parker J.S., Hayes D.N., Perou C.M. Comparison of RNA-seq by poly (A) capture, ribosomal RNA depletion, and DNA microarray for expression profiling. BMC Genomics. 2014;15:419. doi: 10.1186/1471-2164-15-419.  [DOI](https://doi.org/10.1186/1471-2164-15-419) | [PMC free article](/articles/PMC4070569/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24888378/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Genomics&title=Comparison%20of%20RNA-seq%20by%20poly%20(A)%20capture,%20ribosomal%20RNA%20depletion,%20and%20DNA%20microarray%20for%20expression%20profiling&author=W.%20Zhao&author=X.%20He&author=K.A.%20Hoadley&author=J.S.%20Parker&author=D.N.%20Hayes&volume=15&publication_year=2014&pages=419&pmid=24888378&doi=10.1186/1471-2164-15-419&)

12. Feijen E.A.M., Leisenring W.M., Stratton K.L., et al. Derivation of anthracycline and anthraquinone equivalence ratios to doxorubicin for late-onset cardiotoxicity. JAMA Oncol. 2019;5:864–871. doi: 10.1001/jamaoncol.2018.6634.  [DOI](https://doi.org/10.1001/jamaoncol.2018.6634) | [PMC free article](/articles/PMC6490232/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30703192/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA%20Oncol&title=Derivation%20of%20anthracycline%20and%20anthraquinone%20equivalence%20ratios%20to%20doxorubicin%20for%20late-onset%20cardiotoxicity&author=E.A.M.%20Feijen&author=W.M.%20Leisenring&author=K.L.%20Stratton&volume=5&publication_year=2019&pages=864-871&pmid=30703192&doi=10.1001/jamaoncol.2018.6634&)

13. Dobin A., Davis C.A., Schlesinger F., et al. STAR: ultrafast universal RNA-seq aligner. Bioinformatics. 2013;29:15–21. doi: 10.1093/bioinformatics/bts635.  [DOI](https://doi.org/10.1093/bioinformatics/bts635) | [PMC free article](/articles/PMC3530905/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23104886/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=STAR:%20ultrafast%20universal%20RNA-seq%20aligner&author=A.%20Dobin&author=C.A.%20Davis&author=F.%20Schlesinger&volume=29&publication_year=2013&pages=15-21&pmid=23104886&doi=10.1093/bioinformatics/bts635&)

14. Anders S., Pyl P.T., Huber W. HTSeq—a Python framework to work with high-throughput sequencing data. Bioinformatics. 2015;31:166–169. doi: 10.1093/bioinformatics/btu638.  [DOI](https://doi.org/10.1093/bioinformatics/btu638) | [PMC free article](/articles/PMC4287950/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25260700/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=HTSeq%E2%80%94a%20Python%20framework%20to%20work%20with%20high-throughput%20sequencing%20data&author=S.%20Anders&author=P.T.%20Pyl&author=W.%20Huber&volume=31&publication_year=2015&pages=166-169&pmid=25260700&doi=10.1093/bioinformatics/btu638&)

15. Love M.I., Huber W., Anders S. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. Genome Biol. 2014;15:550. doi: 10.1186/s13059-014-0550-8.  [DOI](https://doi.org/10.1186/s13059-014-0550-8) | [PMC free article](/articles/PMC4302049/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25516281/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Biol&title=Moderated%20estimation%20of%20fold%20change%20and%20dispersion%20for%20RNA-seq%20data%20with%20DESeq2&author=M.I.%20Love&author=W.%20Huber&author=S.%20Anders&volume=15&publication_year=2014&pages=550&pmid=25516281&doi=10.1186/s13059-014-0550-8&)

16. Trapnell C., Roberts A., Goff L., et al. Differential gene and transcript expression analysis of RNA-seq experiments with TopHat and Cufflinks. Nat Protoc. 2012;7:562–578. doi: 10.1038/nprot.2012.016.  [DOI](https://doi.org/10.1038/nprot.2012.016) | [PMC free article](/articles/PMC3334321/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22383036/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Protoc&title=Differential%20gene%20and%20transcript%20expression%20analysis%20of%20RNA-seq%20experiments%20with%20TopHat%20and%20Cufflinks&author=C.%20Trapnell&author=A.%20Roberts&author=L.%20Goff&volume=7&publication_year=2012&pages=562-578&pmid=22383036&doi=10.1038/nprot.2012.016&)

17. Koch W., Latz W., Eichinger M., et al. Genotyping of the common haptoglobin Hp 1/2 polymorphism based on PCR. Clin Chem. 2002;48:1377–1382.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/12194911/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Chem&title=Genotyping%20of%20the%20common%20haptoglobin%20Hp%201/2%20polymorphism%20based%20on%20PCR&author=W.%20Koch&author=W.%20Latz&author=M.%20Eichinger&volume=48&publication_year=2002&pages=1377-1382&pmid=12194911&)

18. Cupaioli F.A., Mosca E., Magri C., et al. Assessment of haptoglobin alleles in autism spectrum disorders. Sci Rep. 2020;10:7758. doi: 10.1038/s41598-020-64679-w.  [DOI](https://doi.org/10.1038/s41598-020-64679-w) | [PMC free article](/articles/PMC7210291/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32385356/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sci%20Rep&title=Assessment%20of%20haptoglobin%20alleles%20in%20autism%20spectrum%20disorders&author=F.A.%20Cupaioli&author=E.%20Mosca&author=C.%20Magri&volume=10&publication_year=2020&pages=7758&pmid=32385356&doi=10.1038/s41598-020-64679-w&)

19. Maeda N., Yang F., Barnett D.R., Bowman B.H., Smithies O. Duplication within the haptoglobin Hp2 gene. Nature. 1984;309:131–135. doi: 10.1038/309131a0.  [DOI](https://doi.org/10.1038/309131a0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/6325933/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Duplication%20within%20the%20haptoglobin%20Hp2%20gene&author=N.%20Maeda&author=F.%20Yang&author=D.R.%20Barnett&author=B.H.%20Bowman&author=O.%20Smithies&volume=309&publication_year=1984&pages=131-135&pmid=6325933&doi=10.1038/309131a0&)

20. Kazmi N., Koda Y., Ndiaye N.C., et al. Genetic determinants of circulating haptoglobin concentration. Clin Chim Acta. 2019;494:138–142. doi: 10.1016/j.cca.2019.03.1617.  [DOI](https://doi.org/10.1016/j.cca.2019.03.1617) | [PMC free article](/articles/PMC6541910/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30898509/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Chim%20Acta&title=Genetic%20determinants%20of%20circulating%20haptoglobin%20concentration&author=N.%20Kazmi&author=Y.%20Koda&author=N.C.%20Ndiaye&volume=494&publication_year=2019&pages=138-142&pmid=30898509&doi=10.1016/j.cca.2019.03.1617&)

21. Bjornsson E., Helgason H., Halldorsson G., et al. A rare splice donor mutation in the haptoglobin gene associates with blood lipid levels and coronary artery disease. Hum Mol Genet. 2017;26:2364–2376. doi: 10.1093/hmg/ddx123.  [DOI](https://doi.org/10.1093/hmg/ddx123) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28398513/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=A%C2%A0rare%20splice%20donor%20mutation%20in%20the%20haptoglobin%20gene%20associates%20with%20blood%20lipid%20levels%20and%20coronary%20artery%20disease&author=E.%20Bjornsson&author=H.%20Helgason&author=G.%20Halldorsson&volume=26&publication_year=2017&pages=2364-2376&pmid=28398513&doi=10.1093/hmg/ddx123&)

22. Froguel P., Ndiaye N.C., Bonnefond A., et al. A genome-wide association study identifies rs2000999 as a strong genetic determinant of circulating haptoglobin levels. PLoS ONE. 2012;7 doi: 10.1371/journal.pone.0032327.  [DOI](https://doi.org/10.1371/journal.pone.0032327) | [PMC free article](/articles/PMC3293812/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22403646/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&title=A%C2%A0genome-wide%20association%20study%20identifies%20rs2000999%20as%20a%20strong%20genetic%20determinant%20of%20circulating%20haptoglobin%20levels&author=P.%20Froguel&author=N.C.%20Ndiaye&author=A.%20Bonnefond&volume=7&publication_year=2012&pmid=22403646&doi=10.1371/journal.pone.0032327&)

23. Sadrzadeh S.M., Graf E., Panter S.S., Hallaway P.E., Eaton J.W. Hemoglobin. A biologic fenton reagent. J Biol Chem. 1984;259:14354–14356.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/6094553/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%C2%A0Biol%20Chem&title=Hemoglobin.%20A%20biologic%20fenton%20reagent&author=S.M.%20Sadrzadeh&author=E.%20Graf&author=S.S.%20Panter&author=P.E.%20Hallaway&author=J.W.%20Eaton&volume=259&publication_year=1984&pages=14354-14356&pmid=6094553&)

24. Andersen C.B.F., Stodkilde K., Saederup K.L., et al. Haptoglobin. Antioxid Redox Signal. 2017;26:814–831. doi: 10.1089/ars.2016.6793.  [DOI](https://doi.org/10.1089/ars.2016.6793) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27650279/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Antioxid%20Redox%20Signal.&title=Haptoglobin&author=C.B.F.%20Andersen&author=K.%20Stodkilde&author=K.L.%20Saederup&volume=26&publication_year=2017&pages=814-831&pmid=27650279&doi=10.1089/ars.2016.6793&)

25. Thomsen J.H., Etzerodt A., Svendsen P., Moestrup S.K. The haptoglobin-CD163-heme oxygenase-1 pathway for hemoglobin scavenging. Oxid Med Cell Longev. 2013;2013 doi: 10.1155/2013/523652.  [DOI](https://doi.org/10.1155/2013/523652) | [PMC free article](/articles/PMC3678498/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23781295/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oxid%20Med%20Cell%20Longev&title=The%20haptoglobin-CD163-heme%20oxygenase-1%20pathway%20for%20hemoglobin%20scavenging&author=J.H.%20Thomsen&author=A.%20Etzerodt&author=P.%20Svendsen&author=S.K.%20Moestrup&volume=2013&publication_year=2013&pmid=23781295&doi=10.1155/2013/523652&)

26. Sadrzadeh S.M., Bozorgmehr J. Haptoglobin phenotypes in health and disorders. Am J Clin Pathol. 2004;121(suppl):S97–S104. doi: 10.1309/8GLX5798Y5XHQ0VW.  [DOI](https://doi.org/10.1309/8GLX5798Y5XHQ0VW) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15298155/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Clin%20Pathol&title=Haptoglobin%20phenotypes%20in%20health%20and%20disorders&author=S.M.%20Sadrzadeh&author=J.%20Bozorgmehr&volume=121&issue=suppl&publication_year=2004&pages=S97-S104&pmid=15298155&doi=10.1309/8GLX5798Y5XHQ0VW&)

27. Langlois M.R., Delanghe J.R. Biological and clinical significance of haptoglobin polymorphism in humans. Clin Chem. 1996;42:1589–1600.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/8855140/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Chem&title=Biological%20and%20clinical%20significance%20of%20haptoglobin%20polymorphism%20in%20humans&author=M.R.%20Langlois&author=J.R.%20Delanghe&volume=42&publication_year=1996&pages=1589-1600&pmid=8855140&)

28. Soejima M., Agusa T., Iwata H., et al. Haptoglobin genotyping of Vietnamese: global distribution of HP del, complete deletion allele of the HP gene. Leg Med (Tokyo) 2015;17:14–16. doi: 10.1016/j.legalmed.2014.08.004.  [DOI](https://doi.org/10.1016/j.legalmed.2014.08.004) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25212669/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Leg%20Med%20(Tokyo)&title=Haptoglobin%20genotyping%20of%20Vietnamese:%20global%20distribution%20of%20HP%20del,%20complete%20deletion%20allele%20of%20the%20HP%20gene&author=M.%20Soejima&author=T.%20Agusa&author=H.%20Iwata&volume=17&publication_year=2015&pages=14-16&pmid=25212669&doi=10.1016/j.legalmed.2014.08.004&)

29. Boettger L.M., Salem R.M., Handsaker R.E., et al. Recurring exon deletions in the HP (haptoglobin) gene contribute to lower blood cholesterol levels. Nat Genet. 2016;48:359–366. doi: 10.1038/ng.3510.  [DOI](https://doi.org/10.1038/ng.3510) | [PMC free article](/articles/PMC4811681/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26901066/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Recurring%20exon%20deletions%20in%20the%20HP%20(haptoglobin)%20gene%20contribute%20to%20lower%20blood%20cholesterol%20levels&author=L.M.%20Boettger&author=R.M.%20Salem&author=R.E.%20Handsaker&volume=48&publication_year=2016&pages=359-366&pmid=26901066&doi=10.1038/ng.3510&)

30. Bulters D., Gaastra B., Zolnourian A., et al. Haemoglobin scavenging in intracranial bleeding: biology and clinical implications. Nat Rev Neurol. 2018;14:416–432. doi: 10.1038/s41582-018-0020-0.  [DOI](https://doi.org/10.1038/s41582-018-0020-0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29925923/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Neurol&title=Haemoglobin%20scavenging%20in%20intracranial%20bleeding:%20biology%20and%20clinical%20implications&author=D.%20Bulters&author=B.%20Gaastra&author=A.%20Zolnourian&volume=14&publication_year=2018&pages=416-432&pmid=29925923&doi=10.1038/s41582-018-0020-0&)

31. Kasvosve I., Gomo Z.A., Gangaidzo I.T., et al. Reference range of serum haptoglobin is haptoglobin phenotype-dependent in blacks. Clin Chim Acta. 2000;296:163–170. doi: 10.1016/s0009-8981(00)00225-4.  [DOI](https://doi.org/10.1016/s0009-8981(00)00225-4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10807979/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Chim%20Acta&title=Reference%20range%20of%20serum%20haptoglobin%20is%20haptoglobin%20phenotype-dependent%20in%20blacks&author=I.%20Kasvosve&author=Z.A.%20Gomo&author=I.T.%20Gangaidzo&volume=296&publication_year=2000&pages=163-170&pmid=10807979&doi=10.1016/s0009-8981(00)00225-4&)

32. Carter K., Worwood M. Haptoglobin: a review of the major allele frequencies worldwide and their association with diseases. Int J Lab Hematol. 2007;29:92–110. doi: 10.1111/j.1751-553X.2007.00898.x.  [DOI](https://doi.org/10.1111/j.1751-553X.2007.00898.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17474882/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Lab%20Hematol&title=Haptoglobin:%20a%20review%20of%20the%20major%20allele%20frequencies%20worldwide%20and%20their%20association%20with%20diseases&author=K.%20Carter&author=M.%20Worwood&volume=29&publication_year=2007&pages=92-110&pmid=17474882&doi=10.1111/j.1751-553X.2007.00898.x&)

33. Naryzny S.N., Legina O.K. Haptoglobin as a biomarker. Biochem Mosc Suppl B Biomed Chem. 2021;15:184–198. doi: 10.1134/S1990750821030069.  [DOI](https://doi.org/10.1134/S1990750821030069) | [PMC free article](/articles/PMC8365284/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34422226/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20Mosc%20Suppl%20B%20Biomed%20Chem&title=Haptoglobin%20as%20a%20biomarker&author=S.N.%20Naryzny&author=O.K.%20Legina&volume=15&publication_year=2021&pages=184-198&pmid=34422226&doi=10.1134/S1990750821030069&)

34. Melamed-Frank M., Lache O., Enav B.I., et al. Structure-function analysis of the antioxidant properties of haptoglobin. Blood. 2001;98:3693–3698. doi: 10.1182/blood.v98.13.3693.  [DOI](https://doi.org/10.1182/blood.v98.13.3693) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11739174/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Structure-function%20analysis%20of%20the%20antioxidant%20properties%20of%20haptoglobin&author=M.%20Melamed-Frank&author=O.%20Lache&author=B.I.%20Enav&volume=98&publication_year=2001&pages=3693-3698&pmid=11739174&doi=10.1182/blood.v98.13.3693&)

35. Van Vlierberghe H., Langlois M., Delanghe J. Haptoglobin polymorphisms and iron homeostasis in health and in disease. Clin Chim Acta. 2004;345:35–42. doi: 10.1016/j.cccn.2004.03.016.  [DOI](https://doi.org/10.1016/j.cccn.2004.03.016) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15193975/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Chim%20Acta&title=Haptoglobin%20polymorphisms%20and%20iron%20homeostasis%20in%20health%20and%20in%20disease&author=H.%20Van%20Vlierberghe&author=M.%20Langlois&author=J.%20Delanghe&volume=345&publication_year=2004&pages=35-42&pmid=15193975&doi=10.1016/j.cccn.2004.03.016&)

36. Buehler P.W., Abraham B., Vallelian F., et al. Haptoglobin preserves the CD163 hemoglobin scavenger pathway by shielding hemoglobin from peroxidative modification. Blood. 2009;113:2578–2586. doi: 10.1182/blood-2008-08-174466.  [DOI](https://doi.org/10.1182/blood-2008-08-174466) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19131549/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Haptoglobin%20preserves%20the%20CD163%20hemoglobin%20scavenger%20pathway%20by%20shielding%20hemoglobin%20from%20peroxidative%20modification&author=P.W.%20Buehler&author=B.%20Abraham&author=F.%20Vallelian&volume=113&publication_year=2009&pages=2578-2586&pmid=19131549&doi=10.1182/blood-2008-08-174466&)

37. Blum S., Asaf R., Guetta J., et al. Haptoglobin genotype determines myocardial infarct size in diabetic mice. J Am Coll Cardiol. 2007;49:82–87. doi: 10.1016/j.jacc.2006.08.044.  [DOI](https://doi.org/10.1016/j.jacc.2006.08.044) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17207726/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%C2%A0Am%20Coll%20Cardiol&title=Haptoglobin%20genotype%20determines%20myocardial%20infarct%20size%20in%20diabetic%20mice&author=S.%20Blum&author=R.%20Asaf&author=J.%20Guetta&volume=49&publication_year=2007&pages=82-87&pmid=17207726&doi=10.1016/j.jacc.2006.08.044&)

38. Graves K.L., Vigerust D.J. Hp: an inflammatory indicator in cardiovascular disease. Future Cardiol. 2016;12:471–481. doi: 10.2217/fca-2016-0008.  [DOI](https://doi.org/10.2217/fca-2016-0008) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27203711/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Future%20Cardiol&title=Hp:%20an%20inflammatory%20indicator%20in%20cardiovascular%20disease&author=K.L.%20Graves&author=D.J.%20Vigerust&volume=12&publication_year=2016&pages=471-481&pmid=27203711&doi=10.2217/fca-2016-0008&)

39. Purushothaman K.-R., Purushothaman M., Levy A.P., Sharma S.K., Fuster V., Moreno P.R. In: Therapeutic Angiogenesis for Vascular Diseases: Molecular Mechanisms and Targeted Clinical Approaches for the Treatment of Angiogenic Disease. Slevin M., editor. Springer Netherlands; Dordrecht, the Netherlands: 2011. Neovascularization and intra-plaque hemorrhage: role of haptoglobin, macrophages, and heme-oxygenase-1 pathway; pp. 237–256.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Therapeutic%20Angiogenesis%20for%20Vascular%20Diseases:%20Molecular%20Mechanisms%20and%20Targeted%20Clinical%20Approaches%20for%20the%20Treatment%20of%20Angiogenic%20Disease&author=K.-R.%20Purushothaman&author=M.%20Purushothaman&author=A.P.%20Levy&author=S.K.%20Sharma&author=V.%20Fuster&publication_year=2011&)

40. Purushothaman K.R., Purushothaman M., Levy A.P., et al. Increased expression of oxidation-specific epitopes and apoptosis are associated with haptoglobin genotype: possible implications for plaque progression in human atherosclerosis. J Am Coll Cardiol. 2012;60:112–119. doi: 10.1016/j.jacc.2012.04.011.  [DOI](https://doi.org/10.1016/j.jacc.2012.04.011) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22766337/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%C2%A0Am%20Coll%20Cardiol&title=Increased%20expression%20of%20oxidation-specific%20epitopes%20and%20apoptosis%20are%20associated%20with%20haptoglobin%20genotype:%20possible%20implications%20for%20plaque%20progression%20in%20human%20atherosclerosis&author=K.R.%20Purushothaman&author=M.%20Purushothaman&author=A.P.%20Levy&volume=60&publication_year=2012&pages=112-119&pmid=22766337&doi=10.1016/j.jacc.2012.04.011&)

41. Lioupis C., Barbatis C., Drougou A., et al. Association of haptoglobin genotype and common cardiovascular risk factors with the amount of iron in atherosclerotic carotid plaques. Atherosclerosis. 2011;216:131–138. doi: 10.1016/j.atherosclerosis.2011.01.028.  [DOI](https://doi.org/10.1016/j.atherosclerosis.2011.01.028) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21316675/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Atherosclerosis&title=Association%20of%20haptoglobin%20genotype%20and%20common%20cardiovascular%20risk%20factors%20with%20the%20amount%20of%20iron%20in%20atherosclerotic%20carotid%20plaques&author=C.%20Lioupis&author=C.%20Barbatis&author=A.%20Drougou&volume=216&publication_year=2011&pages=131-138&pmid=21316675&doi=10.1016/j.atherosclerosis.2011.01.028&)

42. Torabi A., Cleland J.G., Khan N.K., et al. The timing of development and subsequent clinical course of heart failure after a myocardial infarction. Eur Heart J. 2008;29:859–870. doi: 10.1093/eurheartj/ehn096.  [DOI](https://doi.org/10.1093/eurheartj/ehn096) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18353754/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20Heart%20J&title=The%20timing%20of%20development%20and%20subsequent%20clinical%20course%20of%20heart%20failure%20after%20a%20myocardial%20infarction&author=A.%20Torabi&author=J.G.%20Cleland&author=N.K.%20Khan&volume=29&publication_year=2008&pages=859-870&pmid=18353754&doi=10.1093/eurheartj/ehn096&)

43. Ichikawa Y., Ghanefar M., Bayeva M., et al. Cardiotoxicity of doxorubicin is mediated through mitochondrial iron accumulation. J Clin Invest. 2014;124:617–630. doi: 10.1172/JCI72931.  [DOI](https://doi.org/10.1172/JCI72931) | [PMC free article](/articles/PMC3904631/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24382354/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%C2%A0Clin%20Invest&title=Cardiotoxicity%20of%20doxorubicin%20is%20mediated%20through%20mitochondrial%20iron%20accumulation&author=Y.%20Ichikawa&author=M.%20Ghanefar&author=M.%20Bayeva&volume=124&publication_year=2014&pages=617-630&pmid=24382354&doi=10.1172/JCI72931&)

44. Gutteridge J.M. Lipid peroxidation and antioxidants as biomarkers of tissue damage. Clin Chem. 1995;41:1819–1828.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/7497639/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Chem&title=Lipid%20peroxidation%20and%20antioxidants%20as%20biomarkers%20of%20tissue%20damage&author=J.M.%20Gutteridge&volume=41&publication_year=1995&pages=1819-1828&pmid=7497639&)

45. Fang X., Wang H., Han D., et al. Ferroptosis as a target for protection against cardiomyopathy. Proc Natl Acad Sci U S A. 2019;116:2672–2680. doi: 10.1073/pnas.1821022116.  [DOI](https://doi.org/10.1073/pnas.1821022116) | [PMC free article](/articles/PMC6377499/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30692261/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&title=Ferroptosis%20as%20a%20target%20for%20protection%20against%20cardiomyopathy&author=X.%20Fang&author=H.%20Wang&author=D.%20Han&volume=116&publication_year=2019&pages=2672-2680&pmid=30692261&doi=10.1073/pnas.1821022116&)

46. Ganatra S., Nohria A., Shah S., et al. Upfront dexrazoxane for the reduction of anthracycline-induced cardiotoxicity in adults with preexisting cardiomyopathy and cancer: a consecutive case series. Cardiooncology. 2019;5:1. doi: 10.1186/s40959-019-0036-7.  [DOI](https://doi.org/10.1186/s40959-019-0036-7) | [PMC free article](/articles/PMC7048095/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32154008/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cardiooncology&title=Upfront%20dexrazoxane%20for%20the%20reduction%20of%20anthracycline-induced%20cardiotoxicity%20in%20adults%20with%20preexisting%20cardiomyopathy%20and%20cancer:%20a%20consecutive%20case%20series&author=S.%20Ganatra&author=A.%20Nohria&author=S.%20Shah&volume=5&publication_year=2019&pages=1&pmid=32154008&doi=10.1186/s40959-019-0036-7&)

47. Gammella E., Maccarinelli F., Buratti P., Recalcati S., Cairo G. The role of iron in anthracycline cardiotoxicity. Front Pharmacol. 2014;5:25. doi: 10.3389/fphar.2014.00025.  [DOI](https://doi.org/10.3389/fphar.2014.00025) | [PMC free article](/articles/PMC3935484/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24616701/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front%20Pharmacol&title=The%20role%20of%20iron%20in%20anthracycline%20cardiotoxicity&author=E.%20Gammella&author=F.%20Maccarinelli&author=P.%20Buratti&author=S.%20Recalcati&author=G.%20Cairo&volume=5&publication_year=2014&pages=25&pmid=24616701&doi=10.3389/fphar.2014.00025&)

48. Salazar-Mendiguchia J., Gonzalez-Costello J., Roca J., Ariza-Sole A., Manito N., Cequier A. Anthracycline-mediated cardiomyopathy: basic molecular knowledge for the cardiologist. Arch Cardiol Mex. 2014;84:218–223. doi: 10.1016/j.acmx.2013.08.006.  [DOI](https://doi.org/10.1016/j.acmx.2013.08.006) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25001055/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arch%20Cardiol%20Mex&title=Anthracycline-mediated%20cardiomyopathy:%20basic%20molecular%20knowledge%20for%20the%20cardiologist&author=J.%20Salazar-Mendiguchia&author=J.%20Gonzalez-Costello&author=J.%20Roca&author=A.%20Ariza-Sole&author=N.%20Manito&volume=84&publication_year=2014&pages=218-223&pmid=25001055&doi=10.1016/j.acmx.2013.08.006&)

49. Soejima M., Sagata N., Komatsu N., et al. Genetic factors associated with serum haptoglobin level in a Japanese population. Clin Chim Acta. 2014;433:54–57. doi: 10.1016/j.cca.2014.02.029.  [DOI](https://doi.org/10.1016/j.cca.2014.02.029) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24613938/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Chim%20Acta&title=Genetic%20factors%20associated%20with%20serum%20haptoglobin%20level%20in%20a%20Japanese%20population&author=M.%20Soejima&author=N.%20Sagata&author=N.%20Komatsu&volume=433&publication_year=2014&pages=54-57&pmid=24613938&doi=10.1016/j.cca.2014.02.029&)
