# Investigation of Genomic and Transcriptomic Risk Factors of Clopidogrel Response in African Americans

## Metadata
**Authors:** Guang Yang, Cristina Alarcon, Catherine Chanfreau, Norman H Lee, Paula Friedman, Edith Nutescu, Matthew Tuck, Travis O'Brien, Li Gong, Teri E Klein, Kyong‐Mi Chang, Philip S Tsao, David O Meltzer, Julie A Lynch, Million Veteran Program, Sony Tuteja, Minoli A Perera
**Journal:** Clinical Pharmacology and Therapeutics
**Date:** 2025 Jan 27
**DOI:** [10.1002/cpt.3552](https://doi.org/10.1002/cpt.3552)
**PMID:** 39868839
**PMCID:** PMC11993291
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11993291/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC11993291/pdf/CPT-117-1313.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC11993291/pdf/CPT-117-1313.pdf)

## Abstract

Clopidogrel, an anti‐platelet drug, is used to prevent thrombosis after percutaneous coronary intervention. Clopidogrel resistance results in recurring ischemic events, with African Americans (AA) suffering disproportionately. The aim of this study was to discover novel biomarkers of clopidogrel resistance in African Americans using genome and transcriptome data. We conducted a genome‐wide association study (GWAS), including local ancestry adjustment, in 141 AA on clopidogrel to identify genetic associations with high on‐treatment platelet reactivity (HTPR), with validation of genome‐wide significant and suggestive loci in an independent cohort of AA clopidogrel patients (N = 823) from the Million Veteran's Program (MVP) along with in vitro functional analysis. We performed differential gene expression (DGE) analysis in whole blood to identify transcriptomic predictors of response, followed by functional validation in MEG‐01 cells. GWAS identified one signal on Chromosome 7 as significantly associated with increasing risk of HTPR. The lead single‐nucleotide polymorphism (SNP), rs7807369, within thrombospondin 7A (THSD7A) was associated with an increased risk of HTPR (odds ratio (OR) = 4.02, P = 4.56 × 10−9). Higher THSD7A gene expression was associated with HTPR in an independent cohort of clopidogrel‐treated patients (P = 0.004) and carrying a risk allele showed increased gene expression in primary human endothelial cells. Notably, the CYP2C19*2 variants showed no association with clopidogrel response in the discovery or MVP cohorts. DGE analysis identified an association with decreased LAIR1 and AP3B2 expression to HTPR. LAIR1 knockdown in MEG‐01 cells resulted in increased expression of SYK and AKT1, suggesting an inhibitory role of LAIR1 in the Glycoprotein VI pathway. In summary, these findings suggest that other variants and genes outside of CYP2C19 star alleles play an important role in clopidogrel response in AA.

Currently knowledge in this area is limited to the association of coding variants in *CYP2C19* (known as *CYP2C19* star alleles) to clopidogrel response. The original discovery of these associated star allele variants was conducted in European populations. No GWAS or transcriptome‐wide study to date include discovery efforts in African Americans.

Our study aims to answer if these known variants, previously unknown variants, or genes contribute to clopidogrel response in AAs.

We found SNPs within *THSD7A* were associated with clopidogrel resistance in African Americans. *THSD7A* gene expression is higher in patients with clopidogrel resistance and in endothelial cells carrying the risk variants. Both African American cohorts showed no association with HTPR, clopidogrel resistance, or major adverse cardiac with *CYP2C19* star alleles. Additionally, decreased expression of *LAIR1* may lead to increased activation of the GPVI pathway via increased expression of SYK and AKT.

Guidance currently does not consider variants outside of the *CYP2C19* star alleles when making decisions on anti‐platelet therapies. However, our study calls into question if additional variants should be considered for people of African ancestry.

Clopidogrel is an anti‐platelet agent commonly prescribed to prevent ischemic events in patients with coronary artery disease (CAD), acute coronary syndrome (ACS), and in patients undergoing percutaneous coronary intervention (PCI).[1](#cpt3552-bib-0001) However, the response to clopidogrel can vary widely between individuals, with both pharmacodynamic response (P2Y12 reaction units [PRU]) or clinical response showing significant interindividual differences.[2](#cpt3552-bib-0002) High on‐treatment platelet reactivity (HTPR),[3](#cpt3552-bib-0003) has been linked to a greater incidence of major adverse cardiovascular events (MACE) in clinical trials.[4](#cpt3552-bib-0004)

This variability in response to clopidogrel is, in part, heritable, with genetic factors accounting for up to 70% of variability.[5](#cpt3552-bib-0005), [6](#cpt3552-bib-0006) Inadequate clopidogrel response (also known as clopidogrel resistance) is associated with an increased risk of recurrence of MACE, such as myocardial infarction, stroke, and even death.[7](#cpt3552-bib-0007), [8](#cpt3552-bib-0008), [9](#cpt3552-bib-0009)

The first pharmacogenomics study of clopidogrel response was published by Mega. *et al*. showed that carriers of *CYP2C19* loss‐of‐function (LOF) variants were associated with significantly decreased clopidogrel response by reducing the active metabolite of clopidogrel.[10](#cpt3552-bib-0010) The first genome‐wide association study (GWAS) of clopidogrel response in a European‐ancestry cohort similarly identified LOF alleles in *CYP2C19* (*CYP2C19*2*) as significantly decreasing clopidogrel activity, which resulted in increased risk of post‐PCI thrombotic events.[6](#cpt3552-bib-0006) This finding has been validated by other studies.[11](#cpt3552-bib-0011), [12](#cpt3552-bib-0012), [13](#cpt3552-bib-0013)

It is important to note that most pharmacogenomic studies of clopidogrel have largely been performed in populations of European‐ancestry, and it remains unclear whether the findings are generalizable to other populations. Additionally, while *CYP2C19* has been shown to explain a significant portion (~12%) of the variability in clopidogrel response, heritability studies suggest that up to ~70% of the variance in clopidogrel response is genetic in nature,[6](#cpt3552-bib-0006) indicating that other genetic variants likely play a role. Therefore, there is still a need to identify additional genetic variants that contribute to clopidogrel response to improve the efficacy and safety of clopidogrel treatment.

The African American (AA) population is an admixed population that contains more than one ancestry . A study done in 24 USA hospitals showed that AA have a much higher one‐year mortality rate (~7.2%) compared with European‐ancestry (~3.6%) who were treated with clopidogrel,[14](#cpt3552-bib-0014) with apparent association to the poor anti‐platelet response.[15](#cpt3552-bib-0015), [16](#cpt3552-bib-0016) Current GWAS studies often exclude individuals of African ancestry,[17](#cpt3552-bib-0017) which can lead to a lack of knowledge regarding population‐specific genetic variants that may be associated with clopidogrel response in this population. This knowledge gap may limit our ability to fully understand the genetic factors that influence drug response and develop personalized treatment strategies for a broader array of genetically diverse patients.

The aim of this study is to identify novel biomarkers of clopidogrel response in AA through both GWAS and transcriptomics analysis followed by *in vitro* functional analysis of implicated variants and genes.

## MATERIALS AND METHODS

The study overview is illustrated in **Figure** [1](#cpt3552-fig-0001). Our study looked at both genotype and gene expression associations with clopidogrel response. This study included 154 subjects from the ACCOuNT cohort with available genotype data, with 99 of these subjects also having corresponding RNA‐seq data. Validation of findings was done in independent cohorts as well as through *in vitro* studies, as shown.

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7c04/11993291/bfe3d1412bda/CPT-117-1313-g005.jpg)

Overview of study. Infographic showing the multiple stages of this study including discovery cohort analysis, functional in vitro follow‐up and validation cohorts for both the genomic (blue), and transcriptomic (green) findings.Created by BioRender. 18

### ACCOuNT discovery cohort and sample collection

African American participants on clopidogrel were recruited through African American Cardiovascular Pharmacogenomics Consortium (ACCOuNT) as previously described.[19](#cpt3552-bib-0019) The study protocol was approved by the University of Chicago, University of Illinois, Chicago, Northwestern University, George Washington University, and the DC Veterans Administration Hospital Institutional Review Boards. All participants provided written consent before the start of the study procedures. In this study, 154 self‐identified AA participants were administered a daily dose of 75 mg of clopidogrel for a minimum of 15 days. Blood samples were collected into sodium citrate‐vacutainers and analyzed within 4 hours of collection at Northwestern Memorial Hospital diagnostic laboratory, the Washington, DC Veterans Affairs Medical Center, or at George Washington University using the VerifyNow platform, which provided a P_2_Y_12_ reaction unit (PRU) value. Participants with a PRU value of 230 or higher were classified as having high on‐treatment platelet reactivity (HTPR) as this threshold has previously been establishing as associated with an increased risk of MACE.[20](#cpt3552-bib-0020) EDTA‐vacutainers were used for DNA isolation, and PAXgene‐vacutainers were used for RNA isolation.

### Genotyping and imputation

DNA was isolated from a blood sample using the Gentra Puregene Bood kit from Qiagen (Maryland, USA). DNA samples were genotyped using the Illumina Infinium Multi‐Ethnic Global Kit (MEGA) at the University of Chicago Functional Genomics Core using standard protocols. Genotyping calls were created by Genome Studio using a 0.15 GenCall score cutoff. Standard quality control (QC) procedures were implemented as described in **Supplementary Methods** [S1](#cpt3552-supitem-0001). After the quality control measures, genotyping data was imputed using the Trans‐Omics of Precision Medicine Program (TOPMed) imputation server ([https://imputation.biodatacatalyst.nhlbi.nih.gov/#!](https://imputation.biodatacatalyst.nhlbi.nih.gov/#!)), using the TOPMed imputation panel.[21](#cpt3552-bib-0021) SNPs with minor allele frequencies (MAF) < 0.05 and imputation scores <0.8 were removed. This resulted in the inclusion of 8,599,814 SNPs in the analysis. Visual inspection of the principal components analysis (PCA) plot was used to confirm that all participants cluster with the ASW AA population.

### Genome‐wide association studies (GWAS) analyses

Multivariable logistic regression GWAS was performed to estimate the odds ratios (ORs) and 95% confidence intervals (CI) of each variant for the risk of HTPR (case/control analysis) adjusting for age, sex, PC1, and PC2 using SNPTEST v2 software. Additionally, multivariable linear regression GWAS was performed to identify the association of SNPs and PRU value as a continuous phenotype. We also conducted local ancestry inference (LAI) GWAS using Tractor. Detailed methods can be found in **Supplementary Methods** [S1](#cpt3552-supitem-0001). The LAI GWAS was performed using PLINK. All clinical covariates that showed association with PRU or HTPR were included as covariates in the analysis. The deconvoluted model within TRACTOR was used in this analysis.[22](#cpt3552-bib-0022) Meta‐analyses on the deconvoluted AFR and EUR summary statistics were conducted using METAL using a random‐effect model. SNPs with *P*‐value ≤5 × 10^−8^ were considered as genome‐wide significance, whereas SNPs with *P*‐value ≤5 × 10^−6^ were considered as suggestive level of significance.

### Validation in million veterans program clopidogrel cohort

To validate our significant and suggestive loci, we used data collected from the Million Veteran's Program (MVP)[23](#cpt3552-bib-0023) for AA patients treated with clopidogrel after a PCI performed at VA catheterization facilities (*n* = 823). Unlike the ACCOuNT cohort, the only available phenotype for clopidogrel response was the Major Adverse Cardiovascular Events (MACE) composite phenotype defined as all‐cause death, recurrent myocardial infarction (MI), and stroke following the index PCI, as recorded in electronic medical records data as previously described.[24](#cpt3552-bib-0024) We prioritized SNPs found in either the GWAS or the LAI GWAS analysis by (i) those showing several SNPs within the same locus in high LD and (ii) by the lead variant at each locus. We first ran an allele association test to determine the univariate association to MACE. Only SNPs that show *P* < 0.05 were tested in the Cox regression models. Genotypes were extracted from the MVP Release 4 imputed genomic data. Imputation was performed using the African Genomic Resources (AGR) reference panel.[25](#cpt3552-bib-0025) We tested the association of the carrier status of candidate loci SNPs (coded as carrying two copies of the effect allele vs. 1 or 0 copy as the reference) using a Cox proportional hazard model adjusted for age at procedure, type of stent received (bare metal vs. drug‐eluting), presence of chronic kidney disease, prior history of PCI, and timing of PCI relative to enrollment in the MVP study (before vs. after) with censoring at the end of clopidogrel treatment (defined by last refill date plus day supply of medication). As a sensitivity analysis, we included an indicator for PCI indication (acute coronary syndrome [ACS] vs. stable ischemic heart disease) in the model. The effect of ACS was not significant in the model and was therefore, not retained in the final model. *CYP2C19* LOF SNPs were also tested for association with MACE with LOF categorized by the number of *CYP2C19*2* or **3* SNPs carried [Correction added on 7 February 2025, after first online publication: The gene name CYP2C9 has been corrected as CYP2C19 in this version.]. Statistical analyses were performed in R using the packages “survival” and “survminer.”[24](#cpt3552-bib-0024)

### HUVEC cell culture, sample collection, and analysis of SNP associations to THSD7A gene expression

HUVEC cell lines from one white donor and seven AA donors were purchased from several vendors: Lonza (Walkersville, MD, USA), ACGT (Manassas, VA, USA), and PromoCell (Heidelberg, Germany). All HUVEC cell lines were cultured in EGM‐Plus medium with supplements (Lonza Walkersville) at 37°C in a 5% CO_2_/95% air cell culture incubator.

DNA was isolated from these 8 cell lines using the PureLink™ Genomic DNA Mini Kit (ThermoFisher). DNA was amplified using the Platinum™ SuperFi II Green PCR Master Mix (ThermoFisher) and the PCR product sequenced by ACGT Inc. (Wheeling, IL, USA). The primers used for both PCR amplification and sequencing of *THSD7A* were: 5′‐tgtgtcaaaacccaaaaggc‐3′ (forward) and 5′‐tcccaccatcaactttgggta‐3′ (reverse). For protein and RNA isolation, cells were cultured in 10‐cm plates. Half of the cells from one plate were used for protein extraction, while the other half were used for total RNA extraction.

### Differential gene expression analysis of THSD7A in GEO data from patients on clopidogrel

To investigate the association between *THSD7A* gene expression and the risk of HTPR, the [GSE32226](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE32226)[26](#cpt3552-bib-0026) gene expression dataset was downloaded from NCBI GEO ([www.ncbi.nlm.nih.gov/geo](http://www.ncbi.nlm.nih.gov/geo)).[27](#cpt3552-bib-0027) This dataset includes exon array data from peripheral blood cells from 26 male CAD patients treated with clopidogrel (75 mg/day) including PRU value. All samples were normalized by robust multiarray average (RMA) procedure. We reclassified samples with a PRU value of ≥230 as HTPR, consistent with our study's definition (12 cases and 14 controls). The GEO2R interactive web tool was used for differential gene expression analysis. Raw CEL data were analyzed using Linear Models for Microarray Analysis (limma),[28](#cpt3552-bib-0028) and expression data specifically for the *THSD7A* gene were extracted.

### RNA‐sequencing and quality control analysis (QC)

To perform differential gene expression analysis, total RNA was isolated from whole blood samples from the ACCOuNT cohort using the PAXgene Blood RNA kit from PreAnalytiX (Qiagen). Globin mRNA was depleted from the RNA samples using the GLOBINclear kit (Invitrogen, Thermo Fisher), and RNA quality was measured by Bioanalyzer or TapeStation (Agilent, Santa Clara, CA, USA). Samples with RNA integrity number (RIN) score >7 were selected for sequencing. Libraries were prepared using the TruSeq Stranded mRNA Library Prep Kits, and TruSeq RNA Indexes plate for 96 samples (Illumina, San Diego, CA, USA). Sequencing was performed on the NovaSeq platform (Illumina) by Novogene (Sacramento, CA, USA) following the manufacturer's instructions, generating paired‐end 50 bp reads with approximately 50 million reads per sample.

Sequencing data quality was assessed using FastQC. Trimmomatic (version 0.39) was used to trim the sequencing reads, removing adaptor‐related sequences and reads with ≥3 consecutive bases with an aggregate quality score of <20.[29](#cpt3552-bib-0029) The high‐quality reads were then quantified at the transcript level against the human reference transcriptome (Homo_sapiens.GRCh38.cdna.all.fa) using Salmon.[30](#cpt3552-bib-0030)

### Differential gene expression analysis in the ACCOuNT cohort

Differential gene expression (DGE) analysis was performed using the R package “DESeq2”.[31](#cpt3552-bib-0031) Gene expression values were filtered based on expression thresholds of >10 reads in at least 20% of the samples. Only protein‐coding genes and lncRNAs were included in the analysis. Expression values for each gene were normalized using regularized log transformation. Surrogate variable analysis (SVA) was used to account for unmeasured covariates, and all relevant surrogate variables were incorporated into DESeq2. Generalized linear regression models were then fit for each gene.[32](#cpt3552-bib-0032) Genes with a fold change >2.5 and a false discovery rate (FDR) < 0.05 between HTPR and non‐HTPR groups were considered differentially expressed.

### shRNA knockdown of LAIR1 in MEG‐01 cells

Based on our findings in the DGE analysis, we sought to further validate the top signal via *in vitro* knockdown of *LAIR1* to understand the pathways affected and their relationship to platelet function. MEG‐01 cells were cultured in RPMI 1640 medium supplemented with 10% fetal bovine serum (FBS), 2 mM l‐glutamine, 100 U/mL penicillin, and 100 μg/mL streptomycin, in a 37°C humidified atmosphere containing 5% CO_2_/95% air. *LAIR1* shRNA lentivirus clones were purchased from the RNAi Consortium (Sigma‐Aldrich). MEG‐01 cells were infected by LAIR1 shRNA lentivirus clones. A pLKO.1‐puro non‐Mammalian control transduction particle (SHC002V) served as a negative control. Puromycin (2 μg/mL) was used to select the infected cells. After selection with puromycin for 4 days, a real‐time quantitative polymerase chain reaction (RT‐qPCR) was performed to test the knockdown efficiency.

### RT‐qPCR analysis

Total RNA was isolated from MEG‐01 cells and HUVECs using the miRNeasy Mini kit (Qiagen, Valencia, CA, USA). RT‐qPCR analysis was performed using the Power SYBR® Green RNA‐to‐CT™ 1‐Step Kit (Thermo Fisher, Waltham, MA, USA) on an Applied Biosystems QuantStudio 7 system (Thermo Fisher). PCR primers for human genes *LAIR1* (leukocyte‐associated immunoglobulin like receptor 1), *SRC* (SRC proto‐oncogene, non‐receptor tyrosine kinase), *SYK* (spleen‐associated tyrosine kinase), and *AKT1* (AKT serine/threonine kinase 1) were obtained from Sigma‐Aldrich (Saint Louis, MO, USA). The primer sequences for each gene are shown in **Table** [S2](#cpt3552-supitem-0001). Pre‐designed PCR primers for the human *THSD7A* gene were obtained (Bio‐Rad, qHsaCID0007366). Relative gene expression was quantified using the comparative Ct method (2^−ΔΔCT^), with glyceraldehyde 3‐phosphate dehydrogenase (GAPDH) serving as an internal control.

### Western blotting and ELISA analyses

We performed protein quantification studies to assess the changes in proteins resulting from both the knockdown of *LAIR1* and the carriage of the risk allele in HUVECs. Total protein was extracted following established procedures. Briefly, total protein from *LAIR1* knockdown and negative control MEG‐01 cells was isolated from RIPA buffer containing protease and phosphatase inhibitors, while protein from HUVECs was extracted with RIPA buffer supplemented with protease inhibitors. Protein concentration was quantified using the Bradford protein assay kit (ThermoFisher Scientific). Lysates were loaded into NuPAGE™ 4–12%, 1.5 mm, 10 well Mini Protein Bis‐Tris gels (ThermoFisher Scientific; NP0335BOX), and electrophoresed at 100 V for 90 minutes. Following gel electrophoresis, proteins were transferred to Immun‐Blot® PVDF Membrane (Bio‐Rad, Hercules, CA, USA) using NuPAGE™ transfer buffer (NP00061; ThermoFisher Scientific) with 10% methanol for 1 hour. Membranes were then blocked for 1 hour at room temperature in a Blotting‐Grade Blocker Buffer (Bio‐Rad, Hercules, CA, USA).

Primary antibodies were diluted in their respective blocking buffers and incubated overnight at 4°C. After incubation, membranes were washed with TBS 0.1% Tween‐20 (TBST), followed by incubation with secondary antibodies for 1 hour at room temperature. Washes were performed with 1× TBST before imaging using the Odyssey Fc system. Protein detection was performed using Image Studio Ver 5.2.

Most primary antibodies were purchased from Santa Cruz Biotechnology (Dallas, TX), including LAIR‐1, Src, Phospho‐Src (Tyr419), ERK ½, Phospho‐ERK ½, and Cell Signaling (Danvers, MA, USA), including phospho‐ Akt (Ser473), Akt, phospho‐Syk (Tyr525/526), Syk, and GAPDH. THSD7A protein expression in HUVECs was also confirmed by ELISA (MyBioSource, San Diego, CA, USA).

All values are represented as mean ± standard deviation (SD) from at least three independent experiments. Statistical differences between groups were measured using the Student's *t*‐test. *P*‐values in all experiments were considered significant at <0.05.

## RESULTS

Following QC for both genotyping and RNA‐seq, 141 samples passed genotyping QC (38 cases, 103 Controls, as detailed in **Table** [S1](#cpt3552-supitem-0001)). Among 99 RNA‐seq data, 93 included PRU values (22 cases and 71 controls). Additionally, we validated our finding in a cohort of 823 AA participants from the MVP cohort who were treated with clopidogrel after a PCI. The characteristics of both patient cohorts are summarized in **Table** [S3](#cpt3552-supitem-0001).

### GWAS identified an association at the THSD7A gene locus

Our logistic regression GWAS identified one locus that reached a genome‐wide significance level (*P* < 5* 10^−8^). The top SNP, rs7807369, located on chromosome 7, was associated with a significantly increased risk of HTPR (OR: 4.02, 95% confidence interval (CI): 2.30–7.01, *P* = 4.56 * 10^−9^, MAF: 0.31) (**Figure** [2](#cpt3552-fig-0002), **Table** [1](#cpt3552-tbl-0001)). **Table** [1](#cpt3552-tbl-0001) shows the top associations for the lead variant at each locus which reached suggestive significance, with all associations shown in **Table** [S1](#cpt3552-supitem-0001). In contrast, linear regression GWAS analyses of PRU as a continuous phenotype did not identify any genome‐wide significant associations. However, one signal, located on chromosome 18, reached suggestive significance (*P* < 1 × 10^−6^). The top SNP rs2226859 was associated with decreasing the PRU value (**β** = −0.61, 95%CI: −0.72 to −0.49, *P* = 8.41 × 10^−7^, MAF: 0.36). This SNP is in an intergenic region and is not eQTL to any gene in the GTEx. Our LAI GWAS for HTPR did not find any genome‐wide significant loci. However, 10 SNPs across 5 loci reached nominal significance (**Table** [1](#cpt3552-tbl-0001)).

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7c04/11993291/1de9c1a1339c/CPT-117-1313-g001.jpg)

Miami and QQ plots from GWAS. Miami plot representing the association between single‐nucleotide polymorphisms (SNPs) and high on‐treatment platelet reactivity (HTPR) (Top) and the association between SNPs and PRU value (Bottom). The x‐axis shows the position of each SNP on the chromosome, while the y‐axis displays the association (−log10 P‐value) between the SNP and the trait of interest. The red line represents the threshold for genome‐wide significance (P = 5*10−8), and the blue line represents the suggestive level of significance (P = 1*10−6). The QQ plot displays the observed P‐values of the genome‐wide association study (GWAS) summary data on the y‐axis, compared to the expected P‐values under the null hypothesis of no association on the x‐axis. Deviations from the diagonal line indicate departures from the null hypothesis, where points above the line suggest more significant associations than expected, while points below the line suggest fewer significant associations than expected. (a) The GWAS identified one locus (THSD7A on chromosome 7) to be associated with a higher risk of HTPR at a genome‐wide significance level (P < 5* 10−8). (b) QQ plot for GWAS. The genomic control value was 1.06 for logistic regression model GWAS (Left) and 1.00 for linear regression model GWAS.

### Table 1.

| SNPID | Chr | Position | Non‐effect allele | Effect allele | MAF | OR | 95% CI | SE | P value |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| GWAS association to HTPR |  |  |  |  |  |  |  |  |  |
| rs7807369 | 7 | 11,465,339 | C | G | 0.31 | 4.02 | 2.30–7.01 | 0.46 | 4.56E‐09 |
| rs7792678a | 7 | 11,459,151 | A | C | 0.36 | 3.16 | 1.84–5.45 | 0.41 | 1.11E‐06 |
| rs7811849a | 7 | 11,459,290 | T | A | 0.36 | 3 | 1.74–5.16 | 0.41 | 2.67E‐06 |
| rs6013452 | 20 | 52,407,102 | A | T | 0.46 | 3.53 | 2.02–6.17 | 0.4 | 2.65E‐07 |
| rs58975724 | 16 | 8,656,828 | A | G | 0.24 | 3.84 | 2.15–6.88 | 0.42 | 3.09E‐07 |
| rs10094297 | 8 | 18,524,850 | T | C | 0.36 | 0.21 | 0.11–0.43 | 0.4 | 3.64E‐07 |
| rs58075729 | 4 | 26,841,202 | C | A | 0.25 | 3.82 | 2.15–6.81 | 0.4 | 5.41E‐07 |
| rs4845158 | 1 | 190,008,728 | G | A | 0.50 | 0.29 | 0.17–0.52 | 0.36 | 9.74E‐07 |
| rs10760187 | 9 | 121,803,541 | T | C | 0.18 | 10.83 | 2.64–44.51 | 0.80 | 1.24E‐06 |
| rs75274719 | 2 | 13,813,090 | C | A | 0.075 | 8.23 | 3.06–22.13 | 0.59 | 1.35E‐06 |
| rs4350879 | 3 | 127,255,613 | T | G | 0.34 | 3.41 | 1.97–5.91 | 0.35 | 1.53E‐06 |
| rs1149515 | 7 | 16,736,755 | T | C | 0.14 | 15.61 | 2.36–103.04 | 1.05 | 2.74E‐06 |
| LAI GWAS association to HTPR |  |  |  |  |  |  |  |  |  |
| rs6577698 | 8 | 137,007,398 | T | C | 0.27 | 0.19 | 0.10–0.38 | 0.34 | 1.55E‐06 |
| rs11723866 | 4 | 136,777,689 | A | G | 0.37 | 9.37 | 3.67–23.91 | 0.48 | 2.82E‐06 |
| rs191786 | 6 | 82,497,480 | T | C | 0.72 | 6.33 | 2.84–14.09 | 0.41 | 6.20E‐06 |
| rs977413 | 21 | 24,008,267 | A | G | 0.41 | 5.71 | 2.73–11.92 | 0.38 | 3.53E‐06 |
| rs1480155 | 5 | 146,431,164 | A | G | 0.22 | 0.11 | 0.05–0.29 | 0.48 | 4.52E‐06 |

Table 1 Caption: Top association of HPTR GWAS analysis

The regional plot shows that the top SNP, along with variants in high linkage disequilibrium (LD), identified through the logistic regression analysis, are located at the intronic region of *THSD7A* gene. Based on the data from the UCSC Genome browser, two SNPs are in high LD (*r* ^2^ ≥ 0.8) with rs7807369 and are located within a potential enhancer/promoter region (**Figures** [S1](#cpt3552-supitem-0001) **and** [S2](#cpt3552-supitem-0001)). Hi‐C interactions analysis from Vanno Portal also shows that SNPs, rs7792678 and rs7811849, affect the *THSD7A* expression in Human umbilical vein endothelial (HUVEC) cell line by regulating promoter activation (**Figure** [S3](#cpt3552-supitem-0001)).

### External validation of top GWAS associations in the MVP cohort

We examined the association between the candidate loci (variants from either GWAS, which reached suggestive threshold level *P* < 5 × 10^−6^) and the occurrence of MACE in the AA participants from MVP who received clopidogrel after a PCI. Independent loci were tested for genotypic association with MACE (**Table** [S4](#cpt3552-supitem-0001)). Those showing nominal association were then tested in a Cox regression model. We restricted this analysis to 823 participants under the age of 65 at the time of the PCI due to the loss of outcomes data in older individuals who are more likely to receive care outside of the VA healthcare system. We performed a time‐to‐event analysis for the occurrence of MACE using a Cox proportional hazard model, adjusted for age at procedure, type of stent used, existence of chronic kidney disease, prior history of PCI, and timing of PCI and enrollment in the MVP study. Our analysis validated rs191786, identified in the LAI GWAS of HTPR (**Table** [2](#cpt3552-tbl-0002)) (Hazard Ratio (HR) = 1.77, 95% CI: 1.002–3.14, *P* = 0.049, MAF: 0.23), with an elevated risk of MACE for the homozygous carriers of the C allele. Homozygous carriers of the C allele for rs1149515 were found a significantly lower risk of MACE (HR = 0.54, 95% CI: 0.33–0.90, *P* = 0.0179, MAF: 0.16). Neither of these associations met the Bonferroni correct *P* > 0.003. No association was found for rs7807369. A significantly elevated risk of MACE was seen with rs977413; however, the direction of effect was opposite that seen in the LAI GWAS. Of note, the association of *CYP2C19* loss‐of‐function with MACE (*P* = 0.738) did not reach nominal significance in the MVP AA cohort, in contrast to many of the European studies which show significant associations between *CYP2C19* LOF alleles and MACE (**Table** [2](#cpt3552-tbl-0002)).

### Table 2.

| SNPID | Chr | Position | Non‐effect allele | Effect allele | EAF | Hazard ratio | 95% CI | Pval |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
| rs7807369 | 7 | 11,465,339 | C | G | 0.34 | 1.41 | 0.73–2.71 | 0.308 |
| rs1149515 | 7 | 16,736,755 | T | C | 0.17 | 1.82 | 3.03–1.11 | 0.0179 |
| rs191786a | 6 | 82,497,480 | T | C | 0.78 | 1.77 | 1.002–3.14 | 0.0490 |
| rs977413a | 21 | 24,008,267 | G | A | 0.68 | 2.09 | 1.147–3.796 | 0.016 |
| CYP2C19b LOF | 9 |   |   |   |   | 0.91 | 0.53–1.57 | 0.738 |

Table 2 Caption: Validation of top SNPs in the MVP cohort using an adjusted Cox model for the risk of Major Adverse Cardiac Event (MACE) among African American patients receiving clopidogrel after a percutaneous coronary intervention (PCI).

### Validation of THSD7A expression and functional validation of rs7792678 and rs7811849 in HUVECs

To further explore the association between *THSD7A* gene expression and the risk of HTPR, we analyzed the difference in gene expression of *THSD7A* using public gene expression data ([GSE32226](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE32226))[26](#cpt3552-bib-0026) comparing HTPR and non‐HTPR groups, as defined in our GWAS cohort. This analysis revealed that higher expression of *THSD7A* was associated with an increased risk of HTPR (*P* = 0.004) (**Figure** [3 a](#cpt3552-fig-0003)).

### Figure 3.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7c04/11993291/0fe74e858c90/CPT-117-1313-g004.jpg)

THSD7A gene and protein expression. (a) The violin box plot of the gene expression of THSD7A in HTPR (PRU >230) and non‐HTPR groups from the GEO dataset GSE32226. The THSD7A gene expression is significantly higher in the HTPR group compared with the non‐HTPR group (P = 0.004). (b) Fold‐change in the expression of THSD7A gene in heterozygote carriers of rs7792678 and rs7811849 (r 2 LD = 1) compared with homozygous common allele carriers in HUVEC cell cultures. Bar heights indicate the mean expression of the THSD7A gene in genotype groups. Error bars indicate 95% confidence interval estimates of the mean expressions. (c) Quantified differences in protein expression from the THSD7A ELISA are shown in the bar chart for the same groups. **P < 0.005, ***P < 0.0005.

To investigate the potential function of our top associations in *THSD7A* gene expression, we sequenced seven HUVEC (primary endothelial) cell cultures from AA donors and one from a European‐ancestry donor. The European and 2 of the AA HUVEC cultures carried the homozygous common allele at both rs7792678 and rs7811849 (SNP shown to affect expression in *in silico* analysis and high LD with lead GWAS variants), while 5 AA HUVEC cultures were heterozygous for the risk variants. RT‐qPCR found that the cultures carrying the effect variants showed a 2.2‐fold increase in *THSD7A* expression compared to the homozygous common allele carriers (*P* = 0.0006) (**Figure** [3 b](#cpt3552-fig-0003)). In addition, ELISA demonstrated that THSD7A protein levels were significantly increased by 50% (*P* = 0.004) in cultures with the effect alleles as compared to those with the common alleles (**Figure** [3 c](#cpt3552-fig-0003)). All associations remained significant for analysis with rs7807369 (2.0‐fold change, *P* = 0.043). However, only 6 samples had high‐quality genotype calls for this variant.

### Differential gene expression analysis identified lower expression of LAIR1 associated with HTPR

Differential gene expression analysis using RNA‐seq data from whole blood samples revealed that lower expression of *LAIR1* and *AP3B2* was associated with an increased risk of HTPR (**Figure** [4 a](#cpt3552-fig-0004)). As *LAIR1* has known expression in endothelial cells, we pursued further validation of this association. To explore the role of *LAIR1* in clopidogrel response, shRNA knockdown of LAIR1 was performed in the MEG‐01 cell (megakaryocyte cell line). The knockdown significantly reduced *LAIR1* expression and led to a notable increased expression of *SYK*, *AKT1* but not *SRC* (**Figure** [5 a–d](#cpt3552-fig-0005)).

### Figure 4.

![Figure 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7c04/11993291/66691daa4bb9/CPT-117-1313-g002.jpg)

Volcano plot of the differential gene expression analysis. The volcano plot illustrates the statistical significance and fold change for each gene in the RNA‐seq dataset from whole blood. Each point represents a gene, with the x axis representing the log2 fold change and the y‐axis representing the negative logarithm of the P‐value. The red color point represents the gene with significant P‐value (<4*10−6) and log2 fold change.

### Figure 5.

![Figure 5](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7c04/11993291/62a540ff0560/CPT-117-1313-g003.jpg)

In vitro assays of LAIR1 knockdown. Fold change expression of target genes LAIR1 (a), SRC (b), SYK (c), and AKT (d), between MEG‐01 cells after LAIR1 knockdown and wild‐type cells. Bar heights indicate the mean expression of relative genes. Error bars indicate 95% confidence interval estimates of the mean expressions. Decreased LAIR1 expression increased the expression of SYK and AKT significantly with no change in SRC gene expression. Quantification of western blotting images for (e) LAIR1, (f) phosphor‐Src, (g) phospho‐Syk, and (h) phospho Akt. Bar heights indicate the mean value of relative proteins. Error bars indicate 95% confidence interval estimates of the mean values. The LAIR1 knockdown MEG‐01 cells show significantly lower LAIR1 protein levels compared with wild‐type MEG‐01 cells. The LAIR1 knockdown increased the phosphorylation levels of Syk and Akt proteins. *P < 0.05, **P < 0.01, ***P < 0.001, ****P < 0.0001, ns = non‐significant.

The effect of *LAIR1* knockdown on protein expression was further assessed through Western blot analysis. The results showed that the *LAIR1* knockdown cells exhibited a significant decrease in LAIR1 protein levels compared to the control group. Moreover, the knockdown of LAIR1 led to an increase in the protein levels of Syk and Akt. There is no significant difference in the protein level of SRC between LAIR1 knockdown and control groups. Taken together, these findings demonstrate that LAIR1 inhibits Syk expression and reduces activity in the downstream AKT signaling pathway (**Figure** [5 e–h](#cpt3552-fig-0005), **Figure** [S4](#cpt3552-supitem-0001)).

## DISCUSSION

This is the first study to investigate both genomic and transcriptomic biomarkers of clopidogrel response in AAs. We identified potential functional variants for *THSD7A* that are associated with a higher risk of HTPR. According to the gnomAD browser, the frequency of the associated variants is 0.414 among AA populations, compared to 0.135 among European populations. However, within the ICPC European clopidogrel study, this SNP was found at a frequency of 0.105 and was not associated with PRU.[33](#cpt3552-bib-0033) The ICPC European clopidogrel study did not investigate associations to HTPR. While we were unable to validate the association of *THSD7A* variants with MACE in the AA MVP cohort, we identified an association with a second SNP, rs1149515, located at a nearby locus in Chromosome 7, which did show an association to MACE in the AA MVP cohort (**Table** [2](#cpt3552-tbl-0002)). rs1149515 is an eQTL for *TSPAN13*, though its precise function remains unknown.

Using HEUVEC cells, we were able to validate the effect of the associated variants with higher expression of thrombospondin 7A (*THSD7A*). The GTEx consortium showed that rs7807369 is an eQTL for *THSD7A* in thyroid tissue with a concordant direction on effect. Additionally, the gene expression data from within a cohort of clopidogrel patients showed higher expression of *THSD7A* in patients with HTPR as compared with the non‐HTPR group. These validation efforts point to a potential role for *THSD7A* in clopidogrel response. *THSD7A* has been implicated in several disease processes, including venous thromboembolism,[34](#cpt3552-bib-0034) and CAD,[35](#cpt3552-bib-0035) both of which are linked to platelet function. *THSD7A* encodes a protein that play a crucial role in endothelial cell migration and angiogenesis.[36](#cpt3552-bib-0036) Studies have shown that *THSD7A* can inhibit primary endothelial cell migration and tube formation,[37](#cpt3552-bib-0037) which would result in the dysfunction of the endothelial cells. Previous studies have shown endothelial dysfunction was significantly associated with HTPR in patients who were treated with clopidogrel.[38](#cpt3552-bib-0038), [39](#cpt3552-bib-0039), [40](#cpt3552-bib-0040)

Notable, we did not detect an association of clopidogrel response with the *CYP2C19*2* allele. *CYP2C19*2* was present in our cohort (MAF = 0.177) [Correction added on 7 February 2025, after first online publication: The gene name CYP2C9 has been corrected as CYP2C19 in this version.]. However, it showed no association with either HTPR or PRU (*P* = 0.60 and 0.62). At this MAF, we should have been able to detect at least nominal significance with this variant. *CYP2C19*2* carriers, a LOF variant of *CYP2C19*, were reported to have a decreasing response to clopidogrel in European and Asian populations.[6](#cpt3552-bib-0006), [41](#cpt3552-bib-0041) In contrast, a recent study in AA did find a significant association between *CYP2C19* intermediate and poor CYP2C19 metabolizer status and MACE.[42](#cpt3552-bib-0042) While it is clear that the *CYP2C19*2* variant has functional consequences on CYP2C19 enzyme function,[43](#cpt3552-bib-0043) our lack of association suggests that additional factors (including additional SNPs), study composition, or difference in effect size for this allele may modify this effect. Moreover, the Cox regression analysis also showed no association of *CYP2C19* LOF alleles to MACE in an independent AA cohort. More in‐depth sequencing is needed to better understand the role of these clinically actionable SNPs on clopidogrel response in African ancestry populations.

This study identified *LAIR1* expression as associated with a decreased risk of HTPR. *LAIR1* is a transmembrane glycoprotein that is expressed on hematopoietic and megakaryocytes cells (the precursor cell to platelets).[44](#cpt3552-bib-0044), [45](#cpt3552-bib-0045) GPVI is a platelet‐specific collagen receptor that plays a crucial role in platelet activation and aggregation.[46](#cpt3552-bib-0046) Both *LAIR1* and the gene encoding GPVI (glycoprotein VI) are located on chromosome 19—making up the leukocyte receptor complex (LRC).[47](#cpt3552-bib-0047) The collagen‐binding site on LAIR1 is similar to that of GPVI, a well‐known platelet receptor that initiates platelet activation and aggregation.[44](#cpt3552-bib-0044) Unlike GPVI, LAIR1 is an inhibitory receptor that binds collagen.[48](#cpt3552-bib-0048) A recent study by Smith *et al*.[49](#cpt3552-bib-0049) demonstrated that mice with *LAIR1* deficiency showed a 25% increase in platelet counts, a longer platelet half‐life, and increased platelet formation. Our study provides evidence that the knockdown of *LAIR1* in human megakaryoblast cells results in increased activation of the GPVI pathway. Specifically, we found that SYK and AKT gene and protein expression levels were significantly increased with *LAIR1*‐knockdown. SYK and AKT are two important factors in the platelet aggregation pathway.[50](#cpt3552-bib-0050) These findings suggest that LAIR1 and GPVI may co‐regulate platelet aggregation in human cells. Based on the abovementioned evidence, LAIR1 is a promising target gene that regulates platelet aggregation.

## LIMITATIONS

One of the major limitations of this study is the small sample size used for analysis. The limited sample size may have reduced the statistical power to detect small to medium effect sizes and rare variants, potentially limiting the generalizability and precision of the findings. We have tried to remedy this limitation through functional validation of our findings *in vitro* models as well as in independent data sets. The second limitation is the lack of a replication study to ensure the robustness of our GWAS results. Replication is unfortunately not possible as no other independent AA clopidogrel GWAS dataset exists. The ICPC cohort is a European cohort, which showed that most of our significant SNPs were below 5% allele frequency in their cohort and, hence, not included in the analysis. Of the SNPs that were also tested in this cohort, none were associated with PRU. They may be due to differences in LD structure between European and African populations as well as differences in phenotyping (no HTPR associations were tested in the ICPC cohort). Future studies with larger AA and multi‐ancestry cohorts are warranted to validate and expand upon the findings reported here. Finally, we were only able to conduct transcriptome‐wide analysis on samples gathered after clopidogrel exposure. This means that differences in gene expression between response groups may be due to underlying biology associated with platelet reactivity or changes to the transcriptome caused by drug exposure.

In conclusion, we have performed the first multi‐omic study of clopidogrel response in an AA cohort. These data will be useful as we move toward in the implementation of genotypes‐guided clopidogrel therapy in diverse clinical populations. Our work shows that the effect of known variants differs by population and potentially by ancestry.

## FUNDING

Funding for ACCOuNT was provided by 1U54MD010723 from NIHMD (G.Y., C.A., N.H.L., E.N., M.T., T.O., L.G., T.K., D.M., M.A.P). This research is based on data from the Million Veteran Program, Office of Research and Development, Veterans Health Administration, and was supported by award #I01‐BX003362 for MVP‐003/028 to K.M.C., P.S.T. This work was supported using resources and facilities of the Department of Veterans Affairs (VA) Informatics and Computing Infrastructure (VINCI), VA ORD 24‐VINCI‐01.

## CONFLICT OF INTEREST

The authors declared no competing interests for this work.

## AUTHOR CONTRIBUTIONS

G.Y., C.A., C.C., S.T., and M.A.P. wrote the manuscript. G.Y., C.A., N.H.L., P.F., E.N., M.T., T.O., L.G., T.E.K., K.M.C., P.S.T., S.T., and M.A.P. designed the research. G.Y., C.A., C.C., N.H.L., P.F., E.N., M.T., T.O., K.M.C., P.S.T., D.O.M., S.T. and M.A.P. preformed the research. G.Y., C.A., C.C., L.G., T.E.K., K.M.C., P.S.T., J.A.L., S.T., and M.A.P. analyzed the data.

## DISCLAIMER

This publication does not represent the views of the Department of Veterans Affairs or the United States Government.

## Supporting information

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Nguyen, T.A. , Diodati, J.G. & Pharand, C. Resistance to clopidogrel: a review of the evidence. J. Am. Coll. Cardiol. 45, 1157–1164 (2005).  [DOI](https://doi.org/10.1016/j.jacc.2005.01.034) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15837243/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Am.%20Coll.%20Cardiol.&title=Resistance%20to%20clopidogrel:%20a%20review%20of%20the%20evidence&author=T.A.%20Nguyen&author=J.G.%20Diodati&author=C.%20Pharand&volume=45&publication_year=2005&pages=1157-1164&pmid=15837243&doi=10.1016/j.jacc.2005.01.034&)

2. Nagareddy, P. & Smyth, S.S. Inflammation and thrombosis in cardiovascular disease. Curr. Opin. Hematol. 20, 457–463 (2013).  [DOI](https://doi.org/10.1097/MOH.0b013e328364219d) | [PMC free article](/articles/PMC4086917/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23892572/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr.%20Opin.%20Hematol.&title=Inflammation%20and%20thrombosis%20in%20cardiovascular%20disease&author=P.%20Nagareddy&author=S.S.%20Smyth&volume=20&publication_year=2013&pages=457-463&pmid=23892572&doi=10.1097/MOH.0b013e328364219d&)

3. Spiliopoulos, S. & Pastromas, G. Current status of high on‐treatment platelet reactivity in patients with coronary or peripheral arterial disease: mechanisms, evaluation and clinical implications. World J. Cardiol. 7, 912–921 (2015).  [DOI](https://doi.org/10.4330/wjc.v7.i12.912) | [PMC free article](/articles/PMC4691818/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26730297/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=World%20J.%20Cardiol.&title=Current%20status%20of%20high%20on%E2%80%90treatment%20platelet%20reactivity%20in%20patients%20with%20coronary%20or%20peripheral%20arterial%20disease:%20mechanisms,%20evaluation%20and%20clinical%20implications&author=S.%20Spiliopoulos&author=G.%20Pastromas&volume=7&publication_year=2015&pages=912-921&pmid=26730297&doi=10.4330/wjc.v7.i12.912&)

4. Sharifi, H. , Habibi, V. & Emami, Z.A. Platelet reactivity unit (PRU) in patients undergoing elective PCI: rethinking the optimal cut point. Anatol. J. Cardiol. 18, 164 (2017).  [DOI](https://doi.org/10.14744/AnatolJCardiol.2017.7939) | [PMC free article](/articles/PMC5731270/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28766514/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anatol.%20J.%20Cardiol.&title=Platelet%20reactivity%20unit%20(PRU)%20in%20patients%20undergoing%20elective%20PCI:%20rethinking%20the%20optimal%20cut%20point&author=H.%20Sharifi&author=V.%20Habibi&author=Z.A.%20Emami&volume=18&publication_year=2017&pages=164&pmid=28766514&doi=10.14744/AnatolJCardiol.2017.7939&)

5. Lewis, J.P. & Shuldiner, A.R. Clopidogrel pharmacogenetics: beyond candidate genes and genome‐wide association studies. Clin. Pharmacol. Ther. 101, 323–325 (2017).  [DOI](https://doi.org/10.1002/cpt.519) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27649515/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Clopidogrel%20pharmacogenetics:%20beyond%20candidate%20genes%20and%20genome%E2%80%90wide%20association%20studies&author=J.P.%20Lewis&author=A.R.%20Shuldiner&volume=101&publication_year=2017&pages=323-325&pmid=27649515&doi=10.1002/cpt.519&)

6. Shuldiner, A.R. , O'Connell, J.R. , Bliden, K.P. et al. Association of cytochrome P450 2C19 genotype with the antiplatelet effect and clinical efficacy of clopidogrel therapy. JAMA 302, 849–857 (2009).  [DOI](https://doi.org/10.1001/jama.2009.1232) | [PMC free article](/articles/PMC3641569/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19706858/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA&title=Association%20of%20cytochrome%20P450%202C19%20genotype%20with%20the%20antiplatelet%20effect%20and%20clinical%20efficacy%20of%20clopidogrel%20therapy&author=A.R.%20Shuldiner&author=J.R.%20O'Connell&author=K.P.%20Bliden&volume=302&publication_year=2009&pages=849-857&pmid=19706858&doi=10.1001/jama.2009.1232&)

7. Geisler, T. et al. Low response to clopidogrel is associated with cardiovascular outcome after coronary stent implantation. Eur. Heart J. 27, 2420–2425 (2006).  [DOI](https://doi.org/10.1093/eurheartj/ehl275) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17005534/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur.%20Heart%20J.&title=Low%20response%20to%20clopidogrel%20is%20associated%20with%20cardiovascular%20outcome%20after%20coronary%20stent%20implantation&author=T.%20Geisler&volume=27&publication_year=2006&pages=2420-2425&pmid=17005534&doi=10.1093/eurheartj/ehl275&)

8. Zhang, Y.J. et al. Influence of genetic polymorphisms in P2Y12 receptor signaling pathway on antiplatelet response to clopidogrel in coronary heart disease. BMC Cardiovasc. Disord. 22, 575 (2022).  [DOI](https://doi.org/10.1186/s12872-022-02988-w) | [PMC free article](/articles/PMC9801627/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36581799/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Cardiovasc.%20Disord.&title=Influence%20of%20genetic%20polymorphisms%20in%20P2Y12%20receptor%20signaling%20pathway%20on%20antiplatelet%20response%20to%20clopidogrel%20in%20coronary%20heart%20disease&author=Y.J.%20Zhang&volume=22&publication_year=2022&pages=575&pmid=36581799&doi=10.1186/s12872-022-02988-w&)

9. Wisman, P.P. et al. Platelet‐reactivity tests identify patients at risk of secondary cardiovascular events: a systematic review and meta‐analysis. J. Thromb. Haemost. 12, 736–747 (2014).  [DOI](https://doi.org/10.1111/jth.12538) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24612413/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Thromb.%20Haemost.&title=Platelet%E2%80%90reactivity%20tests%20identify%20patients%20at%20risk%20of%20secondary%20cardiovascular%20events:%20a%20systematic%20review%20and%20meta%E2%80%90analysis&author=P.P.%20Wisman&volume=12&publication_year=2014&pages=736-747&pmid=24612413&doi=10.1111/jth.12538&)

10. Mega, J.L. et al. Cytochrome p‐450 polymorphisms and response to clopidogrel. N. Engl. J. Med. 360, 354–362 (2009).  [DOI](https://doi.org/10.1056/NEJMoa0809171) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19106084/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%20Engl.%20J.%20Med.&title=Cytochrome%20p%E2%80%90450%20polymorphisms%20and%20response%20to%20clopidogrel&author=J.L.%20Mega&volume=360&publication_year=2009&pages=354-362&pmid=19106084&doi=10.1056/NEJMoa0809171&)

11. Pereira, N.L. et al. Clopidogrel pharmacogenetics. Circ Cardiovasc Interv. 12, e007811 (2019).  [DOI](https://doi.org/10.1161/CIRCINTERVENTIONS.119.007811) | [PMC free article](/articles/PMC6581205/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30998396/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circ%20Cardiovasc%20Interv.&title=Clopidogrel%20pharmacogenetics&author=N.L.%20Pereira&volume=12&publication_year=2019&pages=e007811&pmid=30998396&doi=10.1161/CIRCINTERVENTIONS.119.007811&)

12. Stone, G.W. et al. Platelet reactivity and clinical outcomes after coronary artery implantation of drug‐eluting stents (ADAPT‐DES): a prospective multicentre registry study. Lancet 382, 614–623 (2013).  [DOI](https://doi.org/10.1016/S0140-6736(13)61170-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23890998/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Platelet%20reactivity%20and%20clinical%20outcomes%20after%20coronary%20artery%20implantation%20of%20drug%E2%80%90eluting%20stents%20(ADAPT%E2%80%90DES):%20a%20prospective%20multicentre%20registry%20study&author=G.W.%20Stone&volume=382&publication_year=2013&pages=614-623&pmid=23890998&doi=10.1016/S0140-6736(13)61170-8&)

13. Lee, C.R. et al. Clinical pharmacogenetics implementation consortium guideline for CYP2C19 genotype and clopidogrel therapy: 2022 update. Clin. Pharmacol. Ther. 112, 959–967 (2022).  [DOI](https://doi.org/10.1002/cpt.2526) | [PMC free article](/articles/PMC9287492/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35034351/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Clinical%20pharmacogenetics%20implementation%20consortium%20guideline%20for%20CYP2C19%20genotype%20and%20clopidogrel%20therapy:%202022%20update&author=C.R.%20Lee&volume=112&publication_year=2022&pages=959-967&pmid=35034351&doi=10.1002/cpt.2526&)

14. Cresci, S. et al. Cytochrome p450 gene variants, race, and mortality among clopidogrel‐treated patients after acute myocardial infarction. Circ. Cardiovasc. Genet. 7, 277–286 (2014).  [DOI](https://doi.org/10.1161/CIRCGENETICS.113.000303) | [PMC free article](/articles/PMC4104276/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24762860/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circ.%20Cardiovasc.%20Genet.&title=Cytochrome%20p450%20gene%20variants,%20race,%20and%20mortality%20among%20clopidogrel%E2%80%90treated%20patients%20after%20acute%20myocardial%20infarction&author=S.%20Cresci&volume=7&publication_year=2014&pages=277-286&pmid=24762860&doi=10.1161/CIRCGENETICS.113.000303&)

15. Pendyala, L.K. et al. Racial disparity with on‐treatment platelet reactivity in patients undergoing percutaneous coronary intervention. Am. Heart J. 166, 266–272 (2013).  [DOI](https://doi.org/10.1016/j.ahj.2013.04.008) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23895809/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20Heart%20J.&title=Racial%20disparity%20with%20on%E2%80%90treatment%20platelet%20reactivity%20in%20patients%20undergoing%20percutaneous%20coronary%20intervention&author=L.K.%20Pendyala&volume=166&publication_year=2013&pages=266-272&pmid=23895809&doi=10.1016/j.ahj.2013.04.008&)

16. Long, Y. , Gracely, E.J. , Newschaffer, C.J. & Liu, L. Analysis of the prevalence of cardiovascular disease and associated risk factors for European‐American and African‐American populations in the state of Pennsylvania 2005‐2009. Am. J. Cardiol. 111(1), 68–72 (2013).  [DOI](https://doi.org/10.1016/j.amjcard.2012.08.045) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23040600/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Cardiol.&title=Analysis%20of%20the%20prevalence%20of%20cardiovascular%20disease%20and%20associated%20risk%20factors%20for%20European%E2%80%90American%20and%20African%E2%80%90American%20populations%20in%20the%20state%20of%20Pennsylvania%202005%E2%80%902009&author=Y.%20Long&author=E.J.%20Gracely&author=C.J.%20Newschaffer&author=L.%20Liu&volume=111&issue=1&publication_year=2013&pages=68-72&pmid=23040600&doi=10.1016/j.amjcard.2012.08.045&)

17. Coram, M.A. , Fang, H. , Candille, S.I. , Assimes, T.L. & Tang, H. Leveraging multi‐ethnic evidence for risk assessment of quantitative traits in minority populations. Am. J. Hum. Genet. 101, 638 (2017).  [DOI](https://doi.org/10.1016/j.ajhg.2017.09.005) | [PMC free article](/articles/PMC5630193/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28985498/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Hum.%20Genet.&title=Leveraging%20multi%E2%80%90ethnic%20evidence%20for%20risk%20assessment%20of%20quantitative%20traits%20in%20minority%20populations&author=M.A.%20Coram&author=H.%20Fang&author=S.I.%20Candille&author=T.L.%20Assimes&author=H.%20Tang&volume=101&publication_year=2017&pages=638&pmid=28985498&doi=10.1016/j.ajhg.2017.09.005&)

18. Clark, C. , (2025) https://BioRender.com/h97a316  [https://BioRender.com/h97a316](https://BioRender.com/h97a316)

19. Friedman, P.N. et al. The ACCOuNT consortium: a model for the discovery, translation, and implementation of precision medicine in African Americans. Clin. Transl. Sci. 12, 209–217 (2019).  [DOI](https://doi.org/10.1111/cts.12608) | [PMC free article](/articles/PMC6510376/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30592548/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Transl.%20Sci.&title=The%20ACCOuNT%20consortium:%20a%20model%20for%20the%20discovery,%20translation,%20and%20implementation%20of%20precision%20medicine%20in%20African%20Americans&author=P.N.%20Friedman&volume=12&publication_year=2019&pages=209-217&pmid=30592548&doi=10.1111/cts.12608&)

20. Collet, J.P. Current concepts in the clinical utility of platelet reactivity testing. Interv. Cardiol. 8, 100–106 (2013).  [DOI](https://doi.org/10.15420/icr.2013.8.2.100) | [PMC free article](/articles/PMC5808737/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29588760/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Interv.%20Cardiol.&title=Current%20concepts%20in%20the%20clinical%20utility%20of%20platelet%20reactivity%20testing&author=J.P.%20Collet&volume=8&publication_year=2013&pages=100-106&pmid=29588760&doi=10.15420/icr.2013.8.2.100&)

21. Kowalski, M.H. et al. Use of >100,000 NHLBI trans‐omics for precision medicine (TOPMed) consortium whole genome sequences improves imputation quality and detection of rare variant associations in admixed African and Hispanic/Latino populations. PLoS Genet. 15, e1008500 (2019).  [DOI](https://doi.org/10.1371/journal.pgen.1008500) | [PMC free article](/articles/PMC6953885/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31869403/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet.&title=Use%20of%20>100,000%20NHLBI%20trans%E2%80%90omics%20for%20precision%20medicine%20(TOPMed)%20consortium%20whole%20genome%20sequences%20improves%20imputation%20quality%20and%20detection%20of%20rare%20variant%20associations%20in%20admixed%20African%20and%20Hispanic/Latino%20populations&author=M.H.%20Kowalski&volume=15&publication_year=2019&pages=e1008500&pmid=31869403&doi=10.1371/journal.pgen.1008500&)

22. Atkinson, E.G. et al. Tractor uses local ancestry to enable the inclusion of admixed individuals in GWAS and to boost power. Nat. Genet. 53, 195–204 (2021).  [DOI](https://doi.org/10.1038/s41588-020-00766-y) | [PMC free article](/articles/PMC7867648/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33462486/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet.&title=Tractor%20uses%20local%20ancestry%20to%20enable%20the%20inclusion%20of%20admixed%20individuals%20in%20GWAS%20and%20to%20boost%20power&author=E.G.%20Atkinson&volume=53&publication_year=2021&pages=195-204&pmid=33462486&doi=10.1038/s41588-020-00766-y&)

23. Gaziano, J.M. et al. Million veteran program: a mega‐biobank to study genetic influences on health and disease. J. Clin. Epidemiol. 70, 214–223 (2016).  [DOI](https://doi.org/10.1016/j.jclinepi.2015.09.016) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26441289/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Epidemiol.&title=Million%20veteran%20program:%20a%20mega%E2%80%90biobank%20to%20study%20genetic%20influences%20on%20health%20and%20disease&author=J.M.%20Gaziano&volume=70&publication_year=2016&pages=214-223&pmid=26441289&doi=10.1016/j.jclinepi.2015.09.016&)

24. Chanfreau‐Coffinier, C. et al. Abstract 16710: CYP2C19 Polymorphisms and clinical outcomes following percutaneous coronary intervention (PCI) in the million veterans program (MVP). Circulation 142(Suppl_3):A16710–A16710 (2020).  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation&title=Abstract%2016710:%20CYP2C19%20Polymorphisms%20and%20clinical%20outcomes%20following%20percutaneous%20coronary%20intervention%20(PCI)%20in%20the%20million%20veterans%20program%20(MVP)&author=C.%20Chanfreau%E2%80%90Coffinier&volume=142(Suppl_3)&publication_year=2020&pages=A16710-A16710&)

25. O'Connell, J. , Yun, T. , Moreno, M. et al. A population‐specific reference panel for improved genotype imputation in African Americans. Communications Biology 4, 1269 (2021).  [DOI](https://doi.org/10.1038/s42003-021-02777-9) | [PMC free article](/articles/PMC8571350/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34741098/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Communications%20Biology&title=A%20population%E2%80%90specific%20reference%20panel%20for%20improved%20genotype%20imputation%20in%20African%20Americans&author=J.%20O'Connell&author=T.%20Yun&author=M.%20Moreno&volume=4&publication_year=2021&pages=1269&pmid=34741098&doi=10.1038/s42003-021-02777-9&)

26. Luchessi, A.D. , Silbiger, V.N. , Hirata, R.D. et al. Pharmacogenomics of anti‐platelet therapy focused on peripheral blood cells of coronary arterial disease patients. Clin. Chim. Acta 425, 9–17 (2013).  [DOI](https://doi.org/10.1016/j.cca.2013.06.021) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23832067/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Chim.%20Acta&title=Pharmacogenomics%20of%20anti%E2%80%90platelet%20therapy%20focused%20on%20peripheral%20blood%20cells%20of%20coronary%20arterial%20disease%20patients&author=A.D.%20Luchessi&author=V.N.%20Silbiger&author=R.D.%20Hirata&volume=425&publication_year=2013&pages=9-17&pmid=23832067&doi=10.1016/j.cca.2013.06.021&)

27. Barrett, T. et al. NCBI GEO: archive for functional genomics data sets—update. Nucleic Acids Res. 41, D991–D995 (2013).  [DOI](https://doi.org/10.1093/nar/gks1193) | [PMC free article](/articles/PMC3531084/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23193258/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&title=NCBI%20GEO:%20archive%20for%20functional%20genomics%20data%20sets%E2%80%94update&author=T.%20Barrett&volume=41&publication_year=2013&pages=D991-D995&pmid=23193258&doi=10.1093/nar/gks1193&)

28. Ritchie, M.E. et al. Limma powers differential expression analyses for RNA‐sequencing and microarray studies. Nucleic Acids Res. 43, e47 (2015).  [DOI](https://doi.org/10.1093/nar/gkv007) | [PMC free article](/articles/PMC4402510/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25605792/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&title=Limma%20powers%20differential%20expression%20analyses%20for%20RNA%E2%80%90sequencing%20and%20microarray%20studies&author=M.E.%20Ritchie&volume=43&publication_year=2015&pages=e47&pmid=25605792&doi=10.1093/nar/gkv007&)

29. Bolger, A.M. , Lohse, M. & Usadel, B. Trimmomatic: a flexible trimmer for Illumina sequence data. Bioinformatics 30, 2114–2120 (2014).  [DOI](https://doi.org/10.1093/bioinformatics/btu170) | [PMC free article](/articles/PMC4103590/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24695404/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=Trimmomatic:%20a%20flexible%20trimmer%20for%20Illumina%20sequence%20data&author=A.M.%20Bolger&author=M.%20Lohse&author=B.%20Usadel&volume=30&publication_year=2014&pages=2114-2120&pmid=24695404&doi=10.1093/bioinformatics/btu170&)

30. Patro, R. , Duggal, G. , Love, M.I. , Irizarry, R.A. & Kingsford, C. Salmon provides fast and bias‐aware quantification of transcript expression. Nat. Methods 14, 417–419 (2017).  [DOI](https://doi.org/10.1038/nmeth.4197) | [PMC free article](/articles/PMC5600148/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28263959/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Methods&title=Salmon%20provides%20fast%20and%20bias%E2%80%90aware%20quantification%20of%20transcript%20expression&author=R.%20Patro&author=G.%20Duggal&author=M.I.%20Love&author=R.A.%20Irizarry&author=C.%20Kingsford&volume=14&publication_year=2017&pages=417-419&pmid=28263959&doi=10.1038/nmeth.4197&)

31. Love, M.I. , Huber, W. & Anders, S. Moderated estimation of fold change and dispersion for RNA‐seq data with DESeq2. Genome Biol. 15, 550 (2014).  [DOI](https://doi.org/10.1186/s13059-014-0550-8) | [PMC free article](/articles/PMC4302049/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25516281/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Biol.&title=Moderated%20estimation%20of%20fold%20change%20and%20dispersion%20for%20RNA%E2%80%90seq%20data%20with%20DESeq2&author=M.I.%20Love&author=W.%20Huber&author=S.%20Anders&volume=15&publication_year=2014&pages=550&pmid=25516281&doi=10.1186/s13059-014-0550-8&)

32. Leek, J.T. & Storey, J.D. Capturing heterogeneity in gene expression studies by surrogate variable analysis. PLoS Genet. 3, 1724–1735 (2007).  [DOI](https://doi.org/10.1371/journal.pgen.0030161) | [PMC free article](/articles/PMC1994707/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17907809/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet.&title=Capturing%20heterogeneity%20in%20gene%20expression%20studies%20by%20surrogate%20variable%20analysis&author=J.T.%20Leek&author=J.D.%20Storey&volume=3&publication_year=2007&pages=1724-1735&pmid=17907809&doi=10.1371/journal.pgen.0030161&)

33. Verma, S.S. , Bergmeijer, T.O. , Gong, L. et al. Genomewide association study of platelet reactivity and cardiovascular response in patients treated with clopidogrel: a study by the international clopidogrel pharmacogenomics consortium. Clin. Pharmacol. Ther. 108, 1067–1077 (2020).  [DOI](https://doi.org/10.1002/cpt.1911) | [PMC free article](/articles/PMC7689744/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32472697/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Genomewide%20association%20study%20of%20platelet%20reactivity%20and%20cardiovascular%20response%20in%20patients%20treated%20with%20clopidogrel:%20a%20study%20by%20the%20international%20clopidogrel%20pharmacogenomics%20consortium&author=S.S.%20Verma&author=T.O.%20Bergmeijer&author=L.%20Gong&volume=108&publication_year=2020&pages=1067-1077&pmid=32472697&doi=10.1002/cpt.1911&)

34. de Visser, M.C. , van Minkelen, R. , van Marion, V. et al. Genome‐wide linkage scan in affected sibling pairs identifies novel susceptibility region for venous thromboembolism: genetics in familial thrombosis study. J. Thromb. Haemost. 11, 1474–1484 (2013).  [DOI](https://doi.org/10.1111/jth.12313) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23742623/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Thromb.%20Haemost.&title=Genome%E2%80%90wide%20linkage%20scan%20in%20affected%20sibling%20pairs%20identifies%20novel%20susceptibility%20region%20for%20venous%20thromboembolism:%20genetics%20in%20familial%20thrombosis%20study&author=M.C.%20de%20Visser&author=R.%20van%20Minkelen&author=V.%20van%20Marion&volume=11&publication_year=2013&pages=1474-1484&pmid=23742623&doi=10.1111/jth.12313&)

35. Li, Y. et al. Genome‐wide association and functional studies identify. Arterioscler. Thromb. Vasc. Biol. 38, 964–975 (2018).  [DOI](https://doi.org/10.1161/ATVBAHA.117.310594) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29472232/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arterioscler.%20Thromb.%20Vasc.%20Biol.&title=Genome%E2%80%90wide%20association%20and%20functional%20studies%20identify&author=Y.%20Li&volume=38&publication_year=2018&pages=964-975&pmid=29472232&doi=10.1161/ATVBAHA.117.310594&)

36. Kuo, M.W. , Wang, C.H. , Wu, H.C. , Chang, S.J. & Chuang, Y.J. Soluble THSD7A is an N‐glycoprotein that promotes endothelial cell migration and tube formation in angiogenesis. PLoS One 6, e29000 (2011).  [DOI](https://doi.org/10.1371/journal.pone.0029000) | [PMC free article](/articles/PMC3237571/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22194972/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Soluble%20THSD7A%20is%20an%20N%E2%80%90glycoprotein%20that%20promotes%20endothelial%20cell%20migration%20and%20tube%20formation%20in%20angiogenesis&author=M.W.%20Kuo&author=C.H.%20Wang&author=H.C.%20Wu&author=S.J.%20Chang&author=Y.J.%20Chuang&volume=6&publication_year=2011&pages=e29000&pmid=22194972&doi=10.1371/journal.pone.0029000&)

37. Wang, C.H. , Su, P.T. , Du, X.Y. et al. Thrombospondin type I domain containing 7A (THSD7A) mediates endothelial cell migration and tube formation. J. Cell. Physiol. 222, 685–694 (2010).  [DOI](https://doi.org/10.1002/jcp.21990) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20020485/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Cell.%20Physiol.&title=Thrombospondin%20type%20I%20domain%20containing%207A%20(THSD7A)%20mediates%20endothelial%20cell%20migration%20and%20tube%20formation&author=C.H.%20Wang&author=P.T.%20Su&author=X.Y.%20Du&volume=222&publication_year=2010&pages=685-694&pmid=20020485&doi=10.1002/jcp.21990&)

38. Boehlen, F. & Clemetson, K.J. Platelet chemokines and their receptors: what is their relevance to platelet storage and transfusion practice? Transfus. Med. 11, 403–417 (2001).  [DOI](https://doi.org/10.1046/j.1365-3148.2001.00340.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11851938/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Transfus.%20Med.&title=Platelet%20chemokines%20and%20their%20receptors:%20what%20is%20their%20relevance%20to%20platelet%20storage%20and%20transfusion%20practice?&author=F.%20Boehlen&author=K.J.%20Clemetson&volume=11&publication_year=2001&pages=403-417&pmid=11851938&doi=10.1046/j.1365-3148.2001.00340.x&)

39. Schober, A. et al. Deposition of platelet RANTES triggering monocyte recruitment requires P‐selectin and is involved in neointima formation after arterial injury. Circulation 106, 1523–1529 (2002).  [DOI](https://doi.org/10.1161/01.cir.0000028590.02477.6f) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12234959/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation&title=Deposition%20of%20platelet%20RANTES%20triggering%20monocyte%20recruitment%20requires%20P%E2%80%90selectin%20and%20is%20involved%20in%20neointima%20formation%20after%20arterial%20injury&author=A.%20Schober&volume=106&publication_year=2002&pages=1523-1529&pmid=12234959&doi=10.1161/01.cir.0000028590.02477.6f&)

40. Reiner, M.F. et al. Ticagrelor, but not clopidogrel, reduces arterial thrombosis via endothelial tissue factor suppression. Cardiovasc. Res. 113, 61–69 (2017).  [DOI](https://doi.org/10.1093/cvr/cvw233) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28028070/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cardiovasc.%20Res.&title=Ticagrelor,%20but%20not%20clopidogrel,%20reduces%20arterial%20thrombosis%20via%20endothelial%20tissue%20factor%20suppression&author=M.F.%20Reiner&volume=113&publication_year=2017&pages=61-69&pmid=28028070&doi=10.1093/cvr/cvw233&)

41. Sun, Y. , Lu, Q. , Tao, X. , Cheng, B. & Yang, G. *2 polymorphism related to clopidogrel resistance in patients with coronary heart disease, especially in the Asian population: a systematic review and meta‐analysis. Front. Genet. 11, 576046 (2020).  [DOI](https://doi.org/10.3389/fgene.2020.576046) | [PMC free article](/articles/PMC7783419/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33414804/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front.%20Genet.&title=*2%20polymorphism%20related%20to%20clopidogrel%20resistance%20in%20patients%20with%20coronary%20heart%20disease,%20especially%20in%20the%20Asian%20population:%20a%20systematic%20review%20and%20meta%E2%80%90analysis&author=Y.%20Sun&author=Q.%20Lu&author=X.%20Tao&author=B.%20Cheng&author=G.%20Yang&volume=11&publication_year=2020&pages=576046&pmid=33414804&doi=10.3389/fgene.2020.576046&)

42. Tunehag, K.R. et al. CYP2C19 genotype is associated with adverse cardiovascular outcomes in black patients treated with clopidogrel undergoing percutaneous coronary intervention. J. Am. Heart Assoc. 13, e033791 (2024).  [DOI](https://doi.org/10.1161/JAHA.123.033791) | [PMC free article](/articles/PMC11255757/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/38874073/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Am.%20Heart%20Assoc.&title=CYP2C19%20genotype%20is%20associated%20with%20adverse%20cardiovascular%20outcomes%20in%20black%20patients%20treated%20with%20clopidogrel%20undergoing%20percutaneous%20coronary%20intervention&author=K.R.%20Tunehag&volume=13&publication_year=2024&pages=e033791&pmid=38874073&doi=10.1161/JAHA.123.033791&)

43. Jeong, Y.H. et al. Effect of CYP2C19*2 and *3 loss‐of‐function alleles on platelet reactivity and adverse clinical events in east Asian acute myocardial infarction survivors treated with clopidogrel and aspirin. Circ. Cardiovasc. Interv. 4, 585–594 (2011).  [DOI](https://doi.org/10.1161/CIRCINTERVENTIONS.111.962555) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22045970/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circ.%20Cardiovasc.%20Interv.&title=Effect%20of%20CYP2C19*2%20and%20*3%20loss%E2%80%90of%E2%80%90function%20alleles%20on%20platelet%20reactivity%20and%20adverse%20clinical%20events%20in%20east%20Asian%20acute%20myocardial%20infarction%20survivors%20treated%20with%20clopidogrel%20and%20aspirin&author=Y.H.%20Jeong&volume=4&publication_year=2011&pages=585-594&pmid=22045970&doi=10.1161/CIRCINTERVENTIONS.111.962555&)

44. Steevels, T.A. , Westerlaken, G.H. , Tijssen, M.R. et al. Co‐expression of the collagen receptors leukocyte‐associated immunoglobulin‐like receptor‐1 and glycoprotein VI on a subset of megakaryoblasts. Haematologica 95, 2005–2012 (2010).  [DOI](https://doi.org/10.3324/haematol.2010.026120) | [PMC free article](/articles/PMC2995557/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20713462/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Haematologica&title=Co%E2%80%90expression%20of%20the%20collagen%20receptors%20leukocyte%E2%80%90associated%20immunoglobulin%E2%80%90like%20receptor%E2%80%901%20and%20glycoprotein%20VI%20on%20a%20subset%20of%20megakaryoblasts&author=T.A.%20Steevels&author=G.H.%20Westerlaken&author=M.R.%20Tijssen&volume=95&publication_year=2010&pages=2005-2012&pmid=20713462&doi=10.3324/haematol.2010.026120&)

45. Verbrugge, A. , de Ruiter, T. , Geest, C. , Coffer, P.J. & Meyaard, L. Differential expression of leukocyte‐associated Ig‐like receptor‐1 during neutrophil differentiation and activation. J. Leukoc. Biol. 79, 828–836 (2006).  [DOI](https://doi.org/10.1189/jlb.0705370) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16461736/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Leukoc.%20Biol.&title=Differential%20expression%20of%20leukocyte%E2%80%90associated%20Ig%E2%80%90like%20receptor%E2%80%901%20during%20neutrophil%20differentiation%20and%20activation&author=A.%20Verbrugge&author=T.%20de%20Ruiter&author=C.%20Geest&author=P.J.%20Coffer&author=L.%20Meyaard&volume=79&publication_year=2006&pages=828-836&pmid=16461736&doi=10.1189/jlb.0705370&)

46. Perrella, G. , Nagy, M. , Watson, S.P. & Heemskerk, J.W.M. Platelet GPVI (glycoprotein VI) and thrombotic complications in the venous system. Arterioscler. Thromb. Vasc. Biol. 41, 2681–2692 (2021).  [DOI](https://doi.org/10.1161/ATVBAHA.121.316108) | [PMC free article](/articles/PMC9653110/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34496636/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arterioscler.%20Thromb.%20Vasc.%20Biol.&title=Platelet%20GPVI%20(glycoprotein%20VI)%20and%20thrombotic%20complications%20in%20the%20venous%20system&author=G.%20Perrella&author=M.%20Nagy&author=S.P.%20Watson&author=J.W.M.%20Heemskerk&volume=41&publication_year=2021&pages=2681-2692&pmid=34496636&doi=10.1161/ATVBAHA.121.316108&)

47. Schwartz, J.C. , Sanderson, N.D. , Bickhart, D.M. , Smith, T.P.L. & Hammond, J.A. The structure, evolution, and gene expression within the caprine leukocyte receptor complex. Front. Immunol. 10, 2302 (2019).  [DOI](https://doi.org/10.3389/fimmu.2019.02302) | [PMC free article](/articles/PMC6775213/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31616444/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front.%20Immunol.&title=The%20structure,%20evolution,%20and%20gene%20expression%20within%20the%20caprine%20leukocyte%20receptor%20complex&author=J.C.%20Schwartz&author=N.D.%20Sanderson&author=D.M.%20Bickhart&author=T.P.L.%20Smith&author=J.A.%20Hammond&volume=10&publication_year=2019&pages=2302&pmid=31616444&doi=10.3389/fimmu.2019.02302&)

48. Brondijk, T.H. , de Ruiter, T. , Ballering, J. et al. Crystal structure and collagen‐binding site of immune inhibitory receptor LAIR‐1: unexpected implications for collagen binding by platelet receptor GPVI. Blood 115, 1364–1373 (2010).  [DOI](https://doi.org/10.1182/blood-2009-10-246322) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20007810/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Crystal%20structure%20and%20collagen%E2%80%90binding%20site%20of%20immune%20inhibitory%20receptor%20LAIR%E2%80%901:%20unexpected%20implications%20for%20collagen%20binding%20by%20platelet%20receptor%20GPVI&author=T.H.%20Brondijk&author=T.%20de%20Ruiter&author=J.%20Ballering&volume=115&publication_year=2010&pages=1364-1373&pmid=20007810&doi=10.1182/blood-2009-10-246322&)

49. Smith, C.W. et al. Mice lacking the inhibitory collagen receptor LAIR‐1 exhibit a mild thrombocytosis and hyperactive platelets. Arterioscler. Thromb. Vasc. Biol. 37, 823–835 (2017).  [DOI](https://doi.org/10.1161/ATVBAHA.117.309253) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28336561/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arterioscler.%20Thromb.%20Vasc.%20Biol.&title=Mice%20lacking%20the%20inhibitory%20collagen%20receptor%20LAIR%E2%80%901%20exhibit%20a%20mild%20thrombocytosis%20and%20hyperactive%20platelets&author=C.W.%20Smith&volume=37&publication_year=2017&pages=823-835&pmid=28336561&doi=10.1161/ATVBAHA.117.309253&)

50. Rayes, J. , Watson, S.P. & Nieswandt, B. Functional significance of the platelet immune receptors GPVI and CLEC‐2. J. Clin. Invest. 129, 12–23 (2019).  [DOI](https://doi.org/10.1172/JCI122955) | [PMC free article](/articles/PMC6307936/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30601137/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Invest.&title=Functional%20significance%20of%20the%20platelet%20immune%20receptors%20GPVI%20and%20CLEC%E2%80%902&author=J.%20Rayes&author=S.P.%20Watson&author=B.%20Nieswandt&volume=129&publication_year=2019&pages=12-23&pmid=30601137&doi=10.1172/JCI122955&)
