# CYP3A Genotype Is Associated With Variability in the Exposure and Clearance of the Novel Oncogenic Transcription Inhibitor Lurbinectedin

## Metadata
**Authors:** Rubin Lubomirov, Laura Pérez‐Ramos, Salvador Fudio, Eduardo Asín‐Prieto, Laura Ibarra‐Gómez, Pablo Zubiaur
**Journal:** Clinical and Translational Science
**Date:** 2025 Mar 27
**DOI:** [10.1111/cts.70173](https://doi.org/10.1111/cts.70173)
**PMID:** 40146606
**PMCID:** PMC11949125
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11949125/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC11949125/pdf/CTS-18-e70173.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC11949125/pdf/CTS-18-e70173.pdf)

## Abstract

Lurbinectedin is an oncogenic transcription inhibitor indicated for the treatment of small cell lung cancer (SCLC), which has also shown activity against other malignancies. In this work, two independent cohorts of 180 (discovery cohort) and 719 (validation cohort) cancer patients receiving lurbinectedin in Phases I, II, or III clinical trials were enrolled. Using a population pharmacokinetic (popPK) model of the discovery cohort, patients with extremely high (n = 10, cohort 1) and low (n = 10, cohort 2) etaCL values (i.e., a variable used as a surrogate of unexplained CL interindividual variability) were identified. They were sequenced for 42 candidate genes involved in lurbinectedin pharmacokinetics. A total of 34 variants located in 20 genes were significantly associated with lurbinectedin etaCL; the best nine hits (located in CYP3A5, CYP3A4, ABCB1, ARNT, NR5A2, NR1H4, and FOXA3) were subsequently genotyped in the validation cohort. A strong additive association between CYP3A4 and CYP3A5 genotypes (informed as a CYP3A activity score [AS] variable) and lurbinectedin clearance (CL) and exposure was confirmed, for example, patients with an AS of 3, 2, or 1 showed a 2.3‐, 1.6‐, and 1.5‐fold higher total lurbinectedin CL compared to those with an AS of 0 and 2.3‐, 1.8‐, and 1.6‐fold higher unbound lurbinectedin CL. In conclusion, preemptive CYP3A genotyping may offer a valuable approach for personalizing treatment with lurbinectedin in cancer patients.

Keywords: cancer, CYP3A4, CYP3A5, lurbinectedin, pharmacogenetics, pharmacokinetics

## 1. Introduction

Lurbinectedin (Zepzelca) is an innovative tetrahydroisoquinoline compound and chemotherapy agent that impedes oncogenic transcription. It targets specific sequences within the DNA minor groove, creating adducts that result in the formation of double‐strand breaks (DSBs) [[1](#cts70173-bib-0001), [2](#cts70173-bib-0002), [3](#cts70173-bib-0003)]. Moreover, it promotes the selective degradation of transcribing RNA Pol II and the displacement of transcription factors from the promoters of genes that are being actively transactivated [[4](#cts70173-bib-0004)]. The induction of DSBs initiates a prolonged delay in transitioning through the S phase of the cell cycle, causing arrest at the GSM transition, culminating in apoptosis‐induced tumor cell death [[5](#cts70173-bib-0005)]. Lurbinectedin was granted accelerated approval by the US Food and Drug Administration (FDA) in June 2020 [[6](#cts70173-bib-0006), [7](#cts70173-bib-0007)] followed by approvals in various other countries for treating metastatic small‐cell lung cancer (SCLC) in adult patients experiencing disease progression on or after platinum‐based chemotherapy. Recently, lurbinectedin combined with atezolizumab as a first‐line maintenance therapy for extensive‐stage small cell lung cancer (ES‐SCLC) demonstrated significantly and clinically meaningful improvements in overall survival (OS) and progression‐free survival (PFS) compared to atezolizumab alone [[8](#cts70173-bib-0008)]. Moreover, recent studies have reported activity of lurbinectedin in patients with endometrial cancer, germline *BRCA1/2* metastatic breast cancer, and neuroendocrine tumors, and in patients with relapsed Ewing sarcoma [[9](#cts70173-bib-0009), [10](#cts70173-bib-0010), [11](#cts70173-bib-0011), [12](#cts70173-bib-0012)].

Lurbinectedin can cause adverse drug reactions (ADRs); some of them are mild but occur very frequently (e.g., fatigue, nausea, musculoskeletal pain, constipation) [[13](#cts70173-bib-0013)]; other ADRs can be serious in approximately 3%–5% of the patients (i.e., febrile neutropenia, neutropenia, infection of the respiratory tract, anemia, dyspnea, and thrombocytopenia). The efficacy and safety of lurbinectedin as a first‐line treatment in SCLC as well as in patients with other types of malignancies, alone or in combination with other anti‐cancer drugs, are currently being assessed in late phase studies (e.g., in patients with leiomyosarcoma).

Administered intravenously over 1 h at the dose of 3.2 mg/m^2^, lurbinectedin exhibits a mean (CV%) area under the time‐concentration curve from *t* = 0 to infinite (AUC_∞_) of 551 μg*h/L (94%), and a C_max_ of 107 μg/L (79%). It is highly bound to plasma proteins (> 99%), with a high affinity for α‐1‐acid glycoprotein (AAG). Lurbinectedin shows an elimination half‐life (t_1/2_) of 51 h, with a total plasma clearance of 11 L/h. It undergoes nearly complete metabolism, with 89% of the administered dose excreted as metabolites through biliary excretion, while urinary excretion is minor. The cytochrome P450 isoform 3A4 (CYP3A4) metabolizes lurbinectedin in vitro; the concomitant use of CYP3A inhibitors or inducers is contraindicated [[13](#cts70173-bib-0013), [14](#cts70173-bib-0014)].

Optimal dosing of cancer patients should aim to achieve effective and safe therapeutic responses. This requires sufficient drug exposure to allow the medication to exert its mechanism of action and therapeutic effect, while minimizing harmful side effects. Moreover, improved treatment tolerability should result in a lower rate of treatment discontinuation or dose reductions, leading to greater efficacy and, presumably, higher progression‐free survival (PFS) and/or overall survival (OS). Given prior knowledge of lurbinectedin pharmacokinetics, and the absence of studies assessing the impact of genetic variation on it, an observational retrospective pharmacogenetic study was conducted to identify genetic biomarkers associated with lurbinectedin clearance and exposure.

## 2. Materials and Methods

### 2.1. Study Population

Data from 180 cancer patients enrolled in Phase I and II clinical trials, receiving lurbinectedin either as monotherapy or in combination, was used to identify the 20 patients in the discovery stage of this work; for the validation stage, an independent population of 719 patients was used with patients having similar characteristics, which also included Phase III clinical trials (Table [S1](#cts70173-supitem-0001)). Figure [S1](#cts70173-supitem-0001) shows the flow of patients in the different study stages. These trials adhered to the principles outlined in the Declaration of Helsinki, Good Clinical Practice guidelines, and pertinent local regulations. Approval for all study protocols was obtained from the Independent Local Ethics Committees (IECs) of each participating study site. Prior to any intervention, each patient provided written informed consent for their participation in the clinical trial and for genetic testing. Additional details pertaining to the clinical trials are available in Table [S1](#cts70173-supitem-0001) ([[15](#cts70173-bib-0015), [16](#cts70173-bib-0016), [17](#cts70173-bib-0017), [18](#cts70173-bib-0018), [19](#cts70173-bib-0019), [20](#cts70173-bib-0020), [21](#cts70173-bib-0021), [22](#cts70173-bib-0022), [23](#cts70173-bib-0023), [24](#cts70173-bib-0024)]).

### 2.2. Pharmacokinetic Analyses

Lurbinectedin total plasma concentrations were pooled from patients in the discovery stage and fitted to a previously published PopPK model [[25](#cts70173-bib-0025)] using non‐linear mixed‐effects modeling with NONMEM v7.3. The interpatient variability on drug clearance (etaCL) was used as a surrogate of CL differences unexplained by model covariates that could be due to genetic variations. More specifically, etaCL values are the individual deviations from the population‐predicted CL values after accounting for the effect of covariates included in the model. In the PopPK model [[25](#cts70173-bib-0025)], any demographic, clinical, or laboratory variable potentially related to lurbinectedin CL was evaluated, and those with a demonstrated effect on CL were retained in the model (e.g., albumin or C‐reactive protein [CRP] serum levels or body surface area [BSA]).

For the validation stage, the effect of genetic variation on AUC_∞_ (hereinafter, AUC refers to AUC_∞_ for clarity) and CL values was analyzed. Moreover, the unbound lurbinectedin exposure metrics were analyzed, which were estimated as previously reported (AUC_u_ and CL_u_, respectively) [[26](#cts70173-bib-0026)]. Briefly, the individual total CL, which is dose‐independent, is estimated with the PopPK model. This model incorporates multiple dosing levels: 5 mg fixed dose, 2.0 mg/m^2^, and primarily 3.2 mg/m^2^ (Table [S1](#cts70173-supitem-0001)). Total AUC values were calculated as the dose divided by total CL. Using the AUC value, albumin levels and AAG, and their respective dissociation constants calculated in vitro [[26](#cts70173-bib-0026)], the AUC_u_ was calculated. CL_u_ was subsequently calculated for each patient as dose divided by AUC_u_. Since the dose varied across studies, all AUC and AUC_u_ values were predicted assuming a unique 3.2 mg/m^2^ dose as a single agent, based on individual BSA values and total CL/CL_u_ values.

Histogram plots and probit distribution were used to identify subpopulations based on their etaCL values, showing patients with clearance values higher than estimated by the model (positive etaCL values) or patients with clearance values lower than estimated by the model. The 10 patients with the lowest etaCL values (i.e., closer to −1) were classified as potentially having variants that could decrease lurbinectedin metabolism (cohort 1), while the 10 patients with the highest etaCL values were classified as potentially having variants that could accelerate lurbinectedin metabolism (Cohort 2).

### 2.3. Safety

The safety analysis was conducted exclusively with data from a subgroup of patients from the validation stage, corresponding to clinical trial [NCT02454972](https://clinicaltrials.gov/ct2/show/NCT02454972) (*n* = 252), in which patients with selected solid tumors and SCLC were treated with lurbinectedin as a single agent at the full dose of 3.2 mg/m^2^ [[22](#cts70173-bib-0022)]. The occurrence of the adverse drug reactions (ADRs) neutropenia and thrombocytopenia was documented throughout the treatment period and for up to 30 days after the final lurbinectedin infusion, or until the patient's death or initiation of a new antitumor therapy, whichever occurred first. ADRs were classified according to the United States National Cancer Institute's Common Terminology Criteria for Adverse Events (NCI‐CTCAE) based on type and severity. Patients were closely monitored for safety, and any ADR was tracked until the patient either fully recovered, symptoms improved to at least NCI‐CTCAE Grade 1, or a new antitumor therapy was started. Neutropenia and thrombocytopenia were analyzed according to drug exposure and genetic polymorphism.

### 2.4. Genotyping

Next Generation Sequencing (NGS) for the discovery stage and TaqMan genotyping for the validation stage were outsourced to GENOMICA S.A.U. (Madrid, Spain). The germinal DNA (gDNA) was extracted from peripheral blood mononuclear cells with the DNeasy Blood & Tissue kit, according to the manufacturer's instructions (Qiagen). DNA quality and concentration were assessed with a Qubit instrument (ThermoFisher, USA). Targeted exome sequencing of 42 candidate genes (Table [1](#cts70173-tbl-0001)) involved in lurbinectedin metabolism and transport was performed in an Ion PGM System for Next‐Generation Sequencing (NGS) using an Ion 318 chip v2 for the 20 samples classified as Cohort 1 (*n* = 10) and Cohort 2 (*n* = 10) (discovery stage) and a customized NGS panel. The Ion Reporter v5.2 software and Annotate Variants Single v5.2 workflow were used for variant analysis, specific for GRCh38_human_5.0. In a subsequent validation stage, 719 independent patients were genotyped for nine variants with TaqMan probes (Table [2](#cts70173-tbl-0002)) and a QuantStudio 5 with a 384‐well thermal block (ThermoFisher, USA); the TaqMan genotyper software was used for result analysis.

### TABLE 1.

| Family of genes | Gene |
| --- | --- |
| DMEs | CYP3A4, CYP3A5, CYP3A7, CYP3A43, CYP2D6 |
| DME electron donors | POR, CYB5A, PGRMC1, PGRMC2 |
| Regulators of DME expression: ligand‐dependent nuclear receptors | NR1I2, NR1I3, VDR, NR1H4, PPARA, RARA, RXRA, NR1H3, NR1H2, NR3C1, ARNT, AHR, NR5A2, NR0B2 |
| Regulators of DME expression: Constitutive transcription factors | HNF4A, NR2F2, CEBPB, CEBPA, USF1, HNF1A, FOXA3, FOXA2, PPARGC1A, NCOA1 |
| Regulators of DME expression: Transcription factors signaling regulators | NFKB1, NFKB2, NFKBIA, IL6R, IL6ST |
| Drug transporters | ABCB1 |
| Plasma binding proteins | ALB, ORM1, ORM2 |

Table 1 Caption: Family of genes and genes targeted with exome sequencing (discovery stage).

### TABLE 2.

| Gene | Chr | Variant | Nucleotide change (GRCh38.p14) | Nucleotide change (RefSeq) | Prevalence in the low etaCL group | Prevalence in the high etaCL group | p value | TaqMan ID/s | Prevalence in the validation phase |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Variants selected from discovery phase |  |  |  |  |  |  |  |  |  |
| ARNT | 3 | rs3894771 | A>T | T>A | 5% | 45% | < 0.01 | C___1498674_10 | 51% |
| NR5A2 | 1 | rs1060060 | C>T | C>T | 45% | 5% | < 0.01 | C__25595682_10 | 29% |
| NR1H4 | 12 | rs369219711 | T>A | T>A | 5% | 45% | < 0.01 | AN2XHVF | 0% |
| FOXA3 | 19 | rs11671106 | C>T | C>T | 60% | 20% | < 0.01 | AN33DFD | 31% |
| CYP3A5 | 7 | rs15524 | A>G | T>C | 30% | 0% | N/A | C___8303531_40 | 10% |
| Candidate variants |  |  |  |  |  |  |  |  |  |
| ABCB1 | 7 | rs1045642 | A>G | T>C | 20% | 0% | 0.04 | C___7586657_20 | 52% |
| CYP3A5 | 7 | rs776746 | T>C | A>G | 95% | 70% | 0.0375 | C__26201809_30 | 90% |
| ABCB1 | 7 | rs2032582 | A>C/T | T>G/A | 40% | 40% | 1.0 | C_11711720C_30 | 42% |
| C_11711720D_40 |  |  |  |  |  |  |  |  |  |
| CYP3A4 | 7 | rs35599367 | G>A | C>T | Not detected in the discovery phase |  |  | C__59013445_10 | 4% |

Table 2 Caption: Summary of genes and variants identified from the discovery stage, including a description, statistical comparison of variant prevalence across etaCL groups, additional candidate variants selected for the validation stage, and TaqMan probe IDs used with corresponding observed variant prevalence in the validation stage.

### 2.5. Statistical Analysis

For the discovery stage, variant frequencies between Cohort 1 and 2 were compared using PLINK 2.0 [[27](#cts70173-bib-0027)]. A Manhattan plot was employed to display hits (values with a significance *p* value lower than 0.05 after correction for multiple comparisons) from the statistical analyses.

For the validation stage, a descriptive and statistical analysis of the exposure metrics was conducted. AUC_∞_, CL, AUC_u_, and CL_u_ were analyzed across genotypes. Based on the large sample size and the central limit theorem, parametric tests were used, that is, *t*‐test or ANOVA tests. Mean (SD) values are shown. In this stage, for pharmacogenes that are well characterized structurally and functionally, variant information was used to define gene haplotypes according to PharmVar [[14](#cts70173-bib-0014)] star‐allele definitions (accessed April 17, 2024). The genotype‐informed pharmacogenetic phenotype was informed when possible, that is, when a therapeutic guideline is available. For the CYP3A4 phenotype, the Dutch Pharmacogenetic Working Group (DPWG) guideline for quetiapine prescription was used [[28](#cts70173-bib-0028)]; for the CYP3A5 phenotype, the Clinical Pharmacogenetics Implementation Consortium Guideline (CPIC) for tacrolimus prescription was used [[29](#cts70173-bib-0029)]. A combined *CYP3A* activity score variable was explored as reported previously [[30](#cts70173-bib-0030)], where the number of active *CYP3A* alleles is summed (e.g., poor metabolizers [PM] for both enzymes [i.e., *CYP3A5*3/*3 + CYP3A4*22/*22*] have an score of 0 and normal metabolizers [NM] for both enzymes [i.e., *CYP3A5*1/*1 + CYP3A4*1/*1*] have an score of 4); the remaining variants were individually analyzed.

For the safety analyses, the same approach described above was used to compare drug exposure (in terms of AUC_u_), according to each ADR group (e.g., patients with Grade 4 neutropenia vs. the remaining patients, or patients with grade 2 or higher thrombocytopenia vs. the remaining patients). For the comparison of ADR occurrence across genotypes (e.g., Grade 4 neutropenia according to CYP3A5 genotype groups), a Chi‐squared or a Fisher exact test was used when applicable.

## 3. Results

### 3.1. Discovery Stage

A three‐compartment mammillary model with linear distribution and elimination from the central compartment was suitable to describe the data. The model estimated a volume of distribution at steady state of 438 L caused by the distribution to deep tissues and a low central volume of 11.6 L, while CL was 11.2 L/h. Figure [1](#cts70173-fig-0001) shows the histogram plot and the probit distribution of etaCL in the 180 patients from the discovery stage cohort. All patients classified as Cohort 1 showed etaCL values between −1 and − 0.6, while patients classified as Cohort 2 showed all etaCL values between 0.6 and 1.

### FIGURE 1.

![FIGURE 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1716/11949125/d5e91dbed2d0/CTS-18-e70173-g002.jpg)

Histogram plot and probit distribution of etaCL in the exploratory stage cohort.

The comparison of variant frequencies between Cohort 1 and 2 identified 34 significant hits, involving variants in 20 genes with nominally significant differences in frequencies across the two cohorts; no variant showed a statistically significant difference in prevalence between the two groups after correction for multiple comparisons. All variants with nominal *p* values lower than 0.05 are shown as follows: *ARNT* rs3894771 (*p* = 0.003), rs7517566 (*p* = 0.035); *IL6R* rs3887104 (*p* = 0.035); *NR5A2* rs1060060 (*p* = 0.003), rs1060061 (*p* = 0.025), rs2690034 (*p* = 0.008), rs2816912 (*p* = 0.003), rs2246210 (*p* = 0.038), rs2246209 (*p* = 0.038), rs2690033 (*p* = 0.038), rs2690032 (*p* = 0.003); *PPARGC1A* rs2279525 (*p* = 0.047); *ALB* rs770341325 (*p* = 0.028); *NFKB1* rs964820610 (*p* = 0.038), rs72696119 (*p* = 0.038); *IL6ST* rs1373998 (*p* = 0.037); *NR3C1* rs6198 (*p* = 0.037); *POR* rs72553972 (*p* = 0.037); *ABCB1* rs1258246940 (*p* = 0.035); *CYP3A5* rs15524 (*p* = 0.008), rs776746 (*p* = 0.037); *CYP3A7* rs10211 (*p* = 0.028), rs12360 (*p* = 0.028), rs2257401 (*p* = 0.028); *CYP3A4* rs2242580 (*p* = 0.028); *VDR* rs2544043 (*p* = 0.017); *NR1H4* rs369219711 (*p* = 0.003); *HNF1A* rs41279096 (*p* = 0.035); *NFKBIA* rs1957106 (*p* = 0.038); *FOXA3* rs11671106 (*p* = 0.010); *NR1H2* rs2303044 (*p* = 0.035); *CYP2D6* rs1985842 (*p* = 0.047), rs543725218 (*p* = 0.037).

Based on the *p* values, the following variants were selected for the validation stage: *ARNT* rs3894771, *NR5A2* rs1060060, *NR1H4* rs369219711, *FOXA3* rs11671106, and *CYP3A5* rs15524 and rs776746 (*CYP3A5*3*) (Table [3](#cts70173-tbl-0003)). These were added to three additional candidate variants, that is, *ABCB1* rs1045642 and rs2032582, and *CYP3A4* rs35599367 (*CYP3A4*22*) (Table [2](#cts70173-tbl-0002)).

### TABLE 3.

| Genotype/score | n | AUCu (ng × h/L) | SD | p | Clu (kL/h) | SD | p |
| --- | --- | --- | --- | --- | --- | --- | --- |
| CYP3A score |  |  |  |  |  |  |  |
| 4 | 10 | 936.22 | 398.44 | < 0.001 | 6.03 | 2.47 | < 0.001 |
| 3 | 106 | 1302.62 | 1270.77 | 4.86 | 1.98 |  |  |
| 2 | 547 | 1516.33 | 907.28 | 3.81 | 1.76 |  |  |
| 1 | 39 | 1695.8 | 880.44 | 3.45 | 1.89 |  |  |
| 0 | 2 | 3542.05 | 2717.23 | 2.1 | 1.44 |  |  |
| CYP3A5 |  |  |  |  |  |  |  |
| *1/*1 (NM) | 10 | 936.22 | 398.44 | 0.012 | 6.03 | 2.47 | < 0.001 |
| *1/*3 (IM) | 113 | 1301.65 | 1235.9 | 4.83 | 1.94 |  |  |
| *3/*3 (PM) | 581 | 1538.11 | 922.5 | 3.77 | 1.78 |  |  |
| CYP3A4 |  |  |  |  |  |  |  |
| *1/*1 (NM) | 657 | 1476.71 | 977.18 | 0.007 | 4 | 1.87 | 0.128 |
| *1/*22 (IM) | 46 | 1633.58 | 842.78 | 3.59 | 1.83 |  |  |
| *22/*22 (PM) | 2 | 3542.05 | 2717.23 | 2.1 | 1.44 |  |  |

Table 3 Caption: Unbound lurbinectedin pharmacokinetic parameters according to genotypes or phenotypes in the validation cohort.

### 3.2. Validation Stage

#### 3.2.1. Pharmacokinetics

A three‐compartment mammillary model with linear distribution and elimination from the central compartment was suitable to describe the time course of total plasma concentrations of lurbinectedin after i.v. administration in the validation stage cohort. The model estimated a volume of distribution at steady state of 460 L, a low central volume of 11.5 L, while total plasma CL was 9.65 L/h. Mean (arithmetic) (CV%) estimated AUC was 631.62 (99%) μg*h/L for a 3.2 mg/m [[2](#cts70173-bib-0002)] dose; when accounting for the lurbinectedin unbound fraction, these values were 1492.81 (66%) ng*h/L(AUC_u_) and 3.97 (47%) kL/h (CL_u_), respectively. The estimated lurbinectedin unbound fraction, based on the AUC_u_/AUC_∞_ ratio, was 0.24%, and 0.27% based on the CL_u_/CL ratio.

*CYP3A4* and *CYP3A5* were individually related to variability in the analyzed exposure metrics (Table [3](#cts70173-tbl-0003), Table [S2](#cts70173-supitem-0001)). Most patients (78%) showed a CYP3A activity score of 2, mainly consistent with patients with *CYP3A4*1/*1* + *CYP3A5*3/*3* genotypes and, to a minor extent, with patients with *CYP3A4*1/*22* + *CYP3A5*1/*3* genotypes. Patients with an activity score of 1 represented 6% of the study population (i.e., *CYP3A4*1/*22* + *CYP3A5*3/*3*) and those with a score of 3 represented 15% (i.e., *CYP3A4 *1/*1* + *CYP3A5*1/*3*). *CYP3A5*3*, with a prevalence of 91% in the study population, was related to drug accumulation (higher AUC and AUC_u_) and lower CL/CL_u_; the same effect was observed for *CYP3A4*22*, which was significantly less prevalent (7%) (Table [3](#cts70173-tbl-0003), Table [S2](#cts70173-supitem-0001)). When combining both genotypes in a CYP3A activity score variable, the effects were sharper and the significance was higher (i.e., *p* values were lower) (Table [3](#cts70173-tbl-0003), Table [S2](#cts70173-supitem-0001)). Similarly, the effects were higher for unbound exposure metrics compared to total exposure metrics (Figure [2](#cts70173-fig-0002)). No other association was replicated for the remaining genes and variants (Table [S3](#cts70173-supitem-0001)).

![FIGURE 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1716/11949125/a145612a6582/CTS-18-e70173-g001.jpg)

Lurbinectedin unbound area under the concentration‐time (AUCu) (a–c) and clearance (CLu) (d–f) values according to CYP3A activity score, CYP3A4 and CYP3A5 genotypes. AUC and AUCu correspond to the area under the concentration‐time curve from 0 to infinity (AUC∞) for the parent drug and the unbound moiety, respectively, estimated for a unique dose of 3.2 mg/m2, calculated based on CL and CLu. Outliers are excluded from the figures for clarity.

#### 3.2.2. Safety

ADR occurrence was significantly associated with AUC_u_ levels. Patients with Grade 4 neutropenia showed significantly higher AUC_u_ mean [SD] values compared to those without neutropenia Grade 4 (2785 [1934] ng × h/L and 1578 [820] ng × h/L, respectively, *p* < 0.0001). When comparing patients with neutropenia (any grade) with those without neutropenia, the values were 2291 (1607) ng × h/L and 1567 (856) ng × h/L, respectively, *p* < 0.0001. Moreover, patients with Grade 3 and 4 thrombocytopenia showed significantly higher AUCu mean (SD) values compared to those without thrombocytopenia Grade 3 and 4 (4395 [2842] ng × h/L and 1671 [909] ng × h/L, respectively, *p* < 0.0001). When comparing patients with thrombocytopenia grade 2–4 with those without or with Grade 1 thrombocytopenia, the values were 3081 (2372) ng × h/L and 1660 (856,911 ng × h/L, respectively, *p* < 0.0015).

Despite not reaching the level of statistical significance, the following tendencies were observed: no cases of Grade 4 neutropenia were reported in *CYP3A5*1/*1* patients (NMs) (0 of 4), while 9 of 36 (25%) *CYP3A5*1/*3* patients (IMs) and 29 of 180 (16%) *CYP3A5*3/*3* patients (PMs) suffered from this ADR (*p* = 0.285). Moreover, neutropenia grade ≥ 3 was reported in 6 of 15 (40%) *CYP3A4*1/*22* (IMs) patients compared to 60 of 205 (29%) CYP3A4*1/*1 (NMs) patients (*p* = 0.38).

## 4. Discussion

Preemptive genotyping prior to the prescription of certain cytotoxic drugs is an increasingly common practice, aimed at minimizing drug toxicity while maximizing effectiveness. For example, preemptive genotyping of *DPYD* for the prescription of fluoropyrimidine drugs (e.g., capecitabine or 5‐fluoruracil) can be considered the standard of care in some European countries; in 2020, the European Medicines Agency (EMA) and equivalent National Agencies (e.g., the Spanish Drugs Agency, AEMPS) jointly recommended this testing, emphasizing that “patients completely lacking DPD should not receive any fluorouracil‐based medications” [[31](#cts70173-bib-0031)] (accessed April 17, 2024). Fluoropyrimidine drugs are extensively metabolized by the DPD enzyme. Individuals with DPD PM phenotype exhibit unacceptable levels of toxicity, necessitating avoidance of these drugs. Conversely, DPD IMs may respond positively to fluoropyrimidines with appropriate dose adjustments. A simple preemptive genotyping test can identify decreased function or no function variants thereby predicting DPD IM and PM phenotypes, enabling a personalized precision pharmacotherapy approach. Another notable example is the relationship between UGT1A1 and irinotecan. A 30% dose reduction in UGT1A1 PMs significantly decreases the likelihood of ADRs [[32](#cts70173-bib-0032), [33](#cts70173-bib-0033)]. However, this biomarker has not yet received such a robust regulatory endorsement like *DPYD*.

Patients with relapsed metastatic SCLC with disease progression have a poor prognosis. The use of lurbinectedin in these patients slows disease progression and prolongs life by several months [[22](#cts70173-bib-0022)]. To our knowledge, this is the first report on lurbinectedin pharmacogenetics published to date.

The estimated AUC_∞_ value reported in this study of approximately 632 μg*h/L aligns closely with the value of 551 μg*h/L documented in the lurbinectedin drug label [[13](#cts70173-bib-0013)], both corresponding to a 3.2 mg/m [[2](#cts70173-bib-0002)] single agent dose, with CV values exceeding 90% in both cases; the estimated CL was almost identical to the value reported in the lurbinectedin drug label [[13](#cts70173-bib-0013)], revealing a high consistency with historical lurbinectedin data. Lurbinectedin shows an exposure–response relationship for both toxicity and efficacy, with AUC_u_ values of 1400 ng × h/L (1000–1700 ng × h/L) providing the greatest benefit/risk ratio [[26](#cts70173-bib-0026)].

A strong association was observed between *CYP3A4* and *CYP3A5* genotypes and lurbinectedin CL and exposure, both total and unbound, which was even stronger when consolidated into a CYP3A activity score variable. Patients carrying *CYP3A5*1* alleles (i.e., with a CYP3A activity score higher than 2) showed significantly higher drug CL and lower exposure compared to patients with the *CYP3A5*3/*3* genotype (most patients in our study population). Conversely, patients carrying *CYP3A4*22* alleles showed significantly lower drug clearance and higher exposure. Europeans, however, are mainly PMs (> 80%) for CYP3A5 due to the high prevalence of the *CYP3A5*3* allele, whose core variant is 6981A>G, rs776746, which produces a splice defect and the complete loss of enzymatic activity. The clinical trials that have led to the clinical development of lurbinectedin were mainly conducted in Europeans and, therefore, the optimal dose of 3.2 mg/m [[2](#cts70173-bib-0002)] is most suitable for CYP3A5 PM patients, as they account for the majority in this cohort. In contrast, patients carrying *CYP3A5*1* alleles (i.e., with a CYP3A activity score higher than 2) may not be appropriately dosed at this strength; in fact, in this work, they showed suboptimal exposure metrics due to their higher metabolic capacity.

To compensate for the reduced AUC_u_ and increased CL_u_, a dose increase in patients with *CYP3A5*1/*3* genotype (intermediate metabolizers, IM) and in those with *CYP3A5*1/*1* genotype (NMs) may be required. A subset of patients with European ancestry could potentially benefit from these adjustments and, importantly, the majority of patients in certain ethnic groups (e.g., African Americans, Afro‐Caribbean or Sub‐Saharan Africans), where the *CYP3A5*3* allele is less prevalent than *CYP3A5*1*.

As previously mentioned, CYP3A4 is reported as a key metabolizing enzyme of lurbinectedin. CYP3A5 and CYP3A4 exhibit similar substrate specificity, with over 85% overlap in their amino acid sequences [[34](#cts70173-bib-0034)], supporting the association observed for the *CYP3A5* genotype. Interestingly, *CYP3A4*22* (defined by 15389C>T, rs35599367, also a splice defect variant) was also related to impaired lurbinectedin metabolism (i.e., to lower CL_u_ and higher AUC_u_). A dose reduction in patients with *CYP3A4*1/*22* (IM) and *CYP3A4*22/*22* (PM) genotypes (phenotypes) could be beneficial to normalize drug exposure in these patients. The benefit–risk ratio of this adjustment would be more debatable compared to a potential adjustment based on *CYP3A5* genotype; while the risk of toxicity is generally more tolerable in oncology, the risk of insufficient efficacy could be considered questionable. Furthermore, the effect of *CYP3A4*22* was of somewhat smaller magnitude and statistical significance compared to that of *CYP3A5*3*, likely due to the following reasons: (a) the variant is considered to reduce CYP3A4 activity by 25%–50%, whereas *CYP3A5*3* is classified as a no‐function variant, (b) the lower prevalence of the *CYP3A4*22* allele compared to *CYP3A5*3* reduces the statistical power (e.g., 113 patients carried the *CYP3A5*1/*3* genotype compared to 46 with *CYP3A4*1/*22*).

A CYP3A activity score variable was calculated, consistent with our findings and the biology. This variable was even more strongly associated with lurbinectedin pharmacokinetic variability compared to *CYP3A4* and *CYP3A5* genotypes when analyzed individually. This strongly suggests that both variables are true predictors of lurbinectedin clearance and exposure and may potentially be considered together to inform routine clinical practice.

Notably, the co‐administration of bosentan, a strong *CYP3A4* inducer, decreased lurbinectedin exposure by 20% and increased CL by 21% [[35](#cts70173-bib-0035)]. Moreover, the co‐administration of itraconazole, a strong CYP3A4 inhibitor, increased lurbinectedin exposure 2.7‐fold [[36](#cts70173-bib-0036)]. Therefore, any genotype‐informed dose adjustment should account for drug–drug interactions and the patient's specific clinical and demographic information.

The lack of effect from *ARNT, NR5A2, NR1H4, FOXA3*, and *ABCB1* variants during the validation stage suggests that the initial associations observed in the discovery stage may have been spurious. In contrast, *CYP3A4* and *CYP3A5* variants, which are better candidates from a pharmacogenetic perspective a priori, did have an effect in the validation stage. However, again, the translation of any dose adjustment into clinical practice should be carefully evaluated based on well‐designed clinical trials supporting such adjustments.

Finally, a clear exposure‐response relationship was observed between AUC_u_ and ADR occurrence, which is consistent with lurbinectedin's clinical utility index and therapeutic window, as reported previously [[26](#cts70173-bib-0026)]. Unfortunately, due to the restriction of the sample size, no statistically significant relationships between *CYP3A* genotypes and response were observed, yet two congruent tendencies were observed: first, no individual with CYP3A5 NM phenotype showed Grade 4 neutropenia, while 16%–25% of IMs or PMs did; second, patients with *CYP3A4*1/*22* genotype (IMs) showed a numerically higher percentage of grade ≥ 3 neutropenia compared to NMs.

### 4.1. Caveats

The clinical benefit derived from genotype‐informed dose adjustments is unproven and merely speculative. Other variants, namely *CYP3A4*20*, may explain an additional percentage of pharmacokinetic variability. Variants in other genes, such as *CYP3A7*, which is reported to show overlapping substrate specificity with CYP3A4, could also explain part of the unexplained pharmacokinetic variability (e.g., *CYP3A7*1C*) [[37](#cts70173-bib-0037), [38](#cts70173-bib-0038)]. Additionally, no efficacy variable was analyzed because efficacy endpoints were either not collected or not consistent across clinical trials, which nevertheless are beyond the scope and objective of this work. However, the association between *CYP3A* genotype groups and drug exposure was strong, which indirectly relates to therapy effectiveness and safety, and lays the foundations for future upcoming studies. Furthermore, the impact of *CYP3A* genetic variation on drug tolerability is to date unknown. Additional research is currently underway to incorporate the CYP3A activity score into pharmacokinetic/pharmacodynamic (PK/PD) and/or exposure‐response studies. These studies will assess how CYP3A activity score (and/or *CYP3A4* and *CYP3A5* genotypes) influence the effectiveness and safety of lurbinectedin, which ultimately would justify dose adjustments.

## 5. Conclusion

In the exploratory phase of this observational study, 32 variants in 20 genes were related to lurbinectedin unexplained interindividual variability in CL; the best nine, located in *CYP3A5, CYP3A4, ABCB1, ARNT, NR5A2, NR1H4*, and *FOXA3*, were subsequently genotyped in a validation cohort. The association between the *CYP3A5*3* allele (defined by the presence of 6981A>G, rs776746, a splice defect variant that defines a non‐functional allele) and *CYP3A4*22* (defined by the presence of 15389C>T, rs35599367, another splice defect variant that defines a reduced function allele) was confirmed. A strong additive association between *CYP3A4* and *CYP3A5* genotypes was observed towards the analyzed exposure metrics; when consolidating *CYP3A4* and *CYP3A5* genotype information into a CYP3A activity score variable, the association was stronger. Additional studies are required to confirm the impact of CYP3A activity score (and/or *CYP3A4* and *CYP3A5* genotypes) on lurbinectedin pharmacokinetic variability and safety, and whether *CYP3A* genotype‐informed dose adjustments would be clinically useful.

## Author Contributions

Rubin Lubomirov, Salvador Fudio, and Pablo Zubiaur wrote the manuscript; Rubin Lubomirov, Salvador Fudio, and Pablo Zubiaur designed the research; Rubin Lubomirov, Salvador Fudio, Pablo Zubiaur, Laura Pérez‐Ramos, Eduardo Asín‐Prieto, and Laura Ibarra‐Gómez. performed the research; Rubin Lubomirov, Salvador Fudio, and Pablo Zubiaur analyzed the data.

## Conflicts of Interest

The authors declare no conflicts of interest.

## Supporting information

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Bueren‐Calabuig J. A., Giraudon C., Galmarini C. M., Egly J. M., and Gago F., “Temperature‐Induced Melting of Double‐Stranded DNA in the Absence and Presence of Covalently Bonded Antitumour Drugs: Insight From Molecular Dynamics Simulations,” Nucleic Acids Research 39, no. 18 (2011): 8248–8257, 10.1093/nar/gkr512.  [DOI](https://doi.org/10.1093/nar/gkr512) | [PMC free article](/articles/PMC3185422/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21727089/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Research&title=Temperature%E2%80%90Induced%20Melting%20of%20Double%E2%80%90Stranded%20DNA%20in%20the%20Absence%20and%20Presence%20of%20Covalently%20Bonded%20Antitumour%20Drugs:%20Insight%20From%20Molecular%20Dynamics%20Simulations&author=J.%20A.%20Bueren%E2%80%90Calabuig&author=C.%20Giraudon&author=C.%20M.%20Galmarini&author=J.%20M.%20Egly&author=F.%20Gago&volume=39&issue=18&publication_year=2011&pages=8248-8257&pmid=21727089&doi=10.1093/nar/gkr512&)

2. Harlow M. L., Maloney N., Roland J., et al., “Lurbinectedin Inactivates the Ewing Sarcoma Oncoprotein EWS‐FLI1 by Redistributing It Within the Nucleus,” Cancer Research 76, no. 22 (2016): 6657–6668, 10.1158/0008-5472.CAN-16-0568.  [DOI](https://doi.org/10.1158/0008-5472.CAN-16-0568) | [PMC free article](/articles/PMC5567825/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27697767/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Research&title=Lurbinectedin%20Inactivates%20the%20Ewing%20Sarcoma%20Oncoprotein%20EWS%E2%80%90FLI1%20by%20Redistributing%20It%20Within%20the%20Nucleus&author=M.%20L.%20Harlow&author=N.%20Maloney&author=J.%20Roland&volume=76&issue=22&publication_year=2016&pages=6657-6668&pmid=27697767&doi=10.1158/0008-5472.CAN-16-0568&)

3. Jimeno A., Sharma M. R., Szyldergemajn S., et al., “Phase I Study of Lurbinectedin, a Synthetic Tetrahydroisoquinoline That Inhibits Activated Transcription, Induces DNA Single‐ and Double‐Strand Breaks, on a Weekly × 2 Every‐3‐Week Schedule,” Investigational New Drugs 35, no. 4 (2017): 471–477, 10.1007/s10637-017-0427-2.  [DOI](https://doi.org/10.1007/s10637-017-0427-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28105566/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Investigational%20New%20Drugs&title=Phase%20I%20Study%20of%20Lurbinectedin,%20a%20Synthetic%20Tetrahydroisoquinoline%20That%20Inhibits%20Activated%20Transcription,%20Induces%20DNA%20Single%E2%80%90%20and%20Double%E2%80%90Strand%20Breaks,%20on%20a%20Weekly%E2%80%89%C3%97%E2%80%892%20Every%E2%80%903%E2%80%90Week%20Schedule&author=A.%20Jimeno&author=M.%20R.%20Sharma&author=S.%20Szyldergemajn&volume=35&issue=4&publication_year=2017&pages=471-477&pmid=28105566&doi=10.1007/s10637-017-0427-2&)

4. Santamaria Nunez G., Robles C. M., Giraudon C., et al., “Lurbinectedin Specifically Triggers the Degradation of Phosphorylated RNA Polymerase II and the Formation of DNA Breaks in Cancer Cells,” Molecular Cancer Therapeutics 15, no. 10 (2016): 2399–2412, 10.1158/1535-7163.MCT-16-0172.  [DOI](https://doi.org/10.1158/1535-7163.MCT-16-0172) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27630271/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Molecular%20Cancer%20Therapeutics&title=Lurbinectedin%20Specifically%20Triggers%20the%20Degradation%20of%20Phosphorylated%20RNA%20Polymerase%20II%20and%20the%20Formation%20of%20DNA%20Breaks%20in%20Cancer%20Cells&author=G.%20Santamaria%20Nunez&author=C.%20M.%20Robles&author=C.%20Giraudon&volume=15&issue=10&publication_year=2016&pages=2399-2412&pmid=27630271&doi=10.1158/1535-7163.MCT-16-0172&)

5. Leal J. F., Martinez‐Diez M., Garcia‐Hernandez V., et al., “PM01183, a New DNA Minor Groove Covalent Binder With Potent In Vitro and In Vivo Anti‐Tumour Activity,” British Journal of Pharmacology 161, no. 5 (2010): 1099–1110, 10.1111/j.1476-5381.2010.00945.x.  [DOI](https://doi.org/10.1111/j.1476-5381.2010.00945.x) | [PMC free article](/articles/PMC2998690/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20977459/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=British%20Journal%20of%20Pharmacology&title=PM01183,%20a%20New%20DNA%20Minor%20Groove%20Covalent%20Binder%20With%20Potent%20In%C2%A0Vitro%20and%20In%C2%A0Vivo%20Anti%E2%80%90Tumour%20Activity&author=J.%20F.%20Leal&author=M.%20Martinez%E2%80%90Diez&author=V.%20Garcia%E2%80%90Hernandez&volume=161&issue=5&publication_year=2010&pages=1099-1110&pmid=20977459&doi=10.1111/j.1476-5381.2010.00945.x&)

6. Markham A., “Lurbinectedin: First Approval,” Drugs 80, no. 13 (2020): 1345–1353, 10.1007/s40265-020-01374-0.  [DOI](https://doi.org/10.1007/s40265-020-01374-0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32816202/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drugs&title=Lurbinectedin:%20First%20Approval&author=A.%20Markham&volume=80&issue=13&publication_year=2020&pages=1345-1353&pmid=32816202&doi=10.1007/s40265-020-01374-0&)

7. Singh S., Jaigirdar A. A., Mulkey F., et al., “FDA Approval Summary: Lurbinectedin for the Treatment of Metastatic Small Cell Lung Cancer,” Clinical Cancer Research 27, no. 9 (2021): 2378–2382, 10.1158/1078-0432.CCR-20-3901.  [DOI](https://doi.org/10.1158/1078-0432.CCR-20-3901) | [PMC free article](/articles/PMC9532454/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33288660/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clinical%20Cancer%20Research&title=FDA%20Approval%20Summary:%20Lurbinectedin%20for%20the%20Treatment%20of%20Metastatic%20Small%20Cell%20Lung%20Cancer&author=S.%20Singh&author=A.%20A.%20Jaigirdar&author=F.%20Mulkey&volume=27&issue=9&publication_year=2021&pages=2378-2382&pmid=33288660&doi=10.1158/1078-0432.CCR-20-3901&)

8. Jazz Pharmaceuticals , “Jazz Pharmaceuticals Announces Statistically Significant Overall Survival and Progression‐Free Survival Results for Zepzelca (Lurbinectedin) and Atezolizumab Combination in First‐Line Maintenance Therapy for Extensive‐Stage Small Cell Lung Cancer,” 2024.

9. Boni V., Pistilli B., Brana I., et al., “Lurbinectedin, a Selective Inhibitor of Oncogenic Transcription, in Patients With Pretreated Germline BRCA1/2 Metastatic Breast Cancer: Results From a Phase II Basket Study,” ESMO Open 7, no. 5 (2022): 100571, 10.1016/j.esmoop.2022.100571.  [DOI](https://doi.org/10.1016/j.esmoop.2022.100571) | [PMC free article](/articles/PMC9588879/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36037567/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=ESMO%20Open&title=Lurbinectedin,%20a%20Selective%20Inhibitor%20of%20Oncogenic%20Transcription,%20in%20Patients%20With%20Pretreated%20Germline%20BRCA1/2%20Metastatic%20Breast%20Cancer:%20Results%20From%20a%20Phase%20II%20Basket%20Study&author=V.%20Boni&author=B.%20Pistilli&author=I.%20Brana&volume=7&issue=5&publication_year=2022&pages=100571&pmid=36037567&doi=10.1016/j.esmoop.2022.100571&)

10. Longo‐Munoz F., Castellano D., Alexandre J., et al., “Lurbinectedin in Patients With Pretreated Neuroendocrine Tumours: Results From a Phase II Basket Study,” European Journal of Cancer 172 (2022): 340–348, 10.1016/j.ejca.2022.06.024.  [DOI](https://doi.org/10.1016/j.ejca.2022.06.024) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35830841/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=European%20Journal%20of%20Cancer&title=Lurbinectedin%20in%20Patients%20With%20Pretreated%20Neuroendocrine%20Tumours:%20Results%20From%20a%20Phase%20II%20Basket%20Study&author=F.%20Longo%E2%80%90Munoz&author=D.%20Castellano&author=J.%20Alexandre&volume=172&publication_year=2022&pages=340-348&pmid=35830841&doi=10.1016/j.ejca.2022.06.024&)

11. Kristeleit R., Leary A., Delord J. P., et al., “Lurbinectedin in Patients With Pretreated Endometrial Cancer: Results From a Phase 2 Basket Clinical Trial and Exploratory Translational Study,” Investigational New Drugs 41, no. 5 (2023): 677–687, 10.1007/s10637-023-01383-2.  [DOI](https://doi.org/10.1007/s10637-023-01383-2) | [PMC free article](/articles/PMC10560193/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/37556023/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Investigational%20New%20Drugs&title=Lurbinectedin%20in%20Patients%20With%20Pretreated%20Endometrial%20Cancer:%20Results%20From%20a%20Phase%202%20Basket%20Clinical%20Trial%20and%20Exploratory%20Translational%20Study&author=R.%20Kristeleit&author=A.%20Leary&author=J.%20P.%20Delord&volume=41&issue=5&publication_year=2023&pages=677-687&pmid=37556023&doi=10.1007/s10637-023-01383-2&)

12. Subbiah V., Brana I., Longhi A., et al., “Antitumor Activity of Lurbinectedin, a Selective Inhibitor of Oncogene Transcription, in Patients With Relapsed Ewing Sarcoma: Results of a Basket Phase II Study,” Clinical Cancer Research 28, no. 13 (2022): 2762–2770, 10.1158/1078-0432.CCR-22-0696.  [DOI](https://doi.org/10.1158/1078-0432.CCR-22-0696) | [PMC free article](/articles/PMC9306456/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35486638/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clinical%20Cancer%20Research&title=Antitumor%20Activity%20of%20Lurbinectedin,%20a%20Selective%20Inhibitor%20of%20Oncogene%20Transcription,%20in%20Patients%20With%20Relapsed%20Ewing%20Sarcoma:%20Results%20of%20a%20Basket%20Phase%20II%20Study&author=V.%20Subbiah&author=I.%20Brana&author=A.%20Longhi&volume=28&issue=13&publication_year=2022&pages=2762-2770&pmid=35486638&doi=10.1158/1078-0432.CCR-22-0696&)

13. Food and Drug Administration US , “ZEPZELCA Drug Label. Drug Label. 6/2020”.

14. Aviles P., Altares R., van Andel L., et al., “Metabolic Disposition of Lurbinectedin, a Potent Selective Inhibitor of Active Transcription of Protein‐Coding Genes, in Nonclinical Species and Patients,” Drug Metabolism and Disposition 50, no. 4 (2022): 327–340, 10.1124/dmd.121.000668.  [DOI](https://doi.org/10.1124/dmd.121.000668) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35042701/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metabolism%20and%20Disposition&title=Metabolic%20Disposition%20of%20Lurbinectedin,%20a%20Potent%20Selective%20Inhibitor%20of%20Active%20Transcription%20of%20Protein%E2%80%90Coding%20Genes,%20in%20Nonclinical%20Species%20and%20Patients&author=P.%20Aviles&author=R.%20Altares&author=L.%20van%20Andel&volume=50&issue=4&publication_year=2022&pages=327-340&pmid=35042701&doi=10.1124/dmd.121.000668&)

15. Benton C. B., Rodriguez Diaz ‐Pavon J., Maiti A., et al., “Phase I Study of Lurbinectedin (PM11083) in Patients With Advanced AML and MDS,” Journal of Clinical Oncology 35, no. 15_suppl (2017): e18521, 10.1200/JCO.2017.35.15_suppl.e18521.  [DOI](https://doi.org/10.1200/JCO.2017.35.15_suppl.e18521) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Clinical%20Oncology&title=Phase%20I%20Study%20of%20Lurbinectedin%20(PM11083)%20in%20Patients%20With%20Advanced%20AML%20and%20MDS&author=C.%20B.%20Benton&author=J.%20Rodriguez%20Diaz%20%E2%80%90Pavon&author=A.%20Maiti&volume=35&issue=15_suppl&publication_year=2017&pages=e18521&doi=10.1200/JCO.2017.35.15_suppl.e18521&)

16. Sauri T., Awada A., Calvo E., et al., “Lurbinectedin (PM01183) Administered Once (C1) Every 3 Weeks (q3w) in Combiantion With Capecitabine (XEL) in Patients (Pts) With Metastatic Breast (MBC), Colorectal (CRC) or Pancreatic (PaC) Cancer,” European Journal of Cancer 27, no. Suppl. 6 (2016): 392P.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=European%20Journal%20of%20Cancer&title=Lurbinectedin%20(PM01183)%20Administered%20Once%20(C1)%20Every%203%E2%80%89Weeks%20(q3w)%20in%20Combiantion%20With%20Capecitabine%20(XEL)%20in%20Patients%20(Pts)%20With%20Metastatic%20Breast%20(MBC),%20Colorectal%20(CRC)%20or%20Pancreatic%20(PaC)%20Cancer&author=T.%20Sauri&author=A.%20Awada&author=E.%20Calvo&volume=27&issue=Suppl.%206&publication_year=2016&pages=392P&)

17. Drilon A., Garralda E., Stathis A., et al., “Lurbinectedin (PM01183) Plus Paclitaxel (P), Recommended Dose (RD) Expansion Results With or Without the Addition of Bevacizumab (Bev) in Patients (Pts) With Selected Solid Tumors,” Annals of Oncology 27 (2016): vi125.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annals%20of%20Oncology&title=Lurbinectedin%20(PM01183)%20Plus%20Paclitaxel%20(P),%20Recommended%20Dose%20(RD)%20Expansion%20Results%20With%20or%20Without%20the%20Addition%20of%20Bevacizumab%20(Bev)%20in%20Patients%20(Pts)%20With%20Selected%20Solid%20Tumors&author=A.%20Drilon&author=E.%20Garralda&author=A.%20Stathis&volume=27&publication_year=2016&pages=vi125&)

18. Metaxas Y., Kahatt C., Alfaro V., et al., “A Phase I Trial of Lurbinectedin in Combination With Cisplatin in Patients With Advanced Solid Tumors,” Investigational New Drugs 40, no. 1 (2022): 91–98, 10.1007/s10637-021-01142-1.  [DOI](https://doi.org/10.1007/s10637-021-01142-1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34453241/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Investigational%20New%20Drugs&title=A%20Phase%20I%20Trial%20of%20Lurbinectedin%20in%20Combination%20With%20Cisplatin%20in%20Patients%20With%20Advanced%20Solid%20Tumors&author=Y.%20Metaxas&author=C.%20Kahatt&author=V.%20Alfaro&volume=40&issue=1&publication_year=2022&pages=91-98&pmid=34453241&doi=10.1007/s10637-021-01142-1&)

19. Takahashi S., Shimizu T., Doi T., et al., “551P Phase I Study of Lurbinectedin in Japanese Patients With Pretreated Advanced Tumours: Final Results,” Annals of Oncology 31 (2020): S478, 10.1016/j.annonc.2020.08.665.  [DOI](https://doi.org/10.1016/j.annonc.2020.08.665) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annals%20of%20Oncology&title=551P%20Phase%20I%20Study%20of%20Lurbinectedin%20in%20Japanese%20Patients%20With%20Pretreated%20Advanced%20Tumours:%20Final%20Results&author=S.%20Takahashi&author=T.%20Shimizu&author=T.%20Doi&volume=31&publication_year=2020&pages=S478&doi=10.1016/j.annonc.2020.08.665&)

20. Falcon Gonzalez A., Paz‐Ares L. G., Cote G. M., et al., “Lurbinectedin (LUR) in Combination With Irinotecan (IRI) in Patients (Pts) With Advanced Endometrial Carcinoma,” Journal of Clinical Oncology 39, no. 15_suppl (2021): 5586, 10.1200/JCO.2021.39.15_suppl.5586.  [DOI](https://doi.org/10.1200/JCO.2021.39.15_suppl.5586) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Clinical%20Oncology&title=Lurbinectedin%20(LUR)%20in%20Combination%20With%20Irinotecan%20(IRI)%20in%20Patients%20(Pts)%20With%20Advanced%20Endometrial%20Carcinoma&author=A.%20Falcon%20Gonzalez&author=L.%20G.%20Paz%E2%80%90Ares&author=G.%20M.%20Cote&volume=39&issue=15_suppl&publication_year=2021&pages=5586&doi=10.1200/JCO.2021.39.15_suppl.5586&)

21. Fudio S., Tabernero J., Subbiah V., et al., “Effect of Lurbinectedin on the QTc Interval in Patients With Advanced Solid Tumors: An Exposure‐Response Analysis,” Cancer Chemotherapy and Pharmacology 87, no. 1 (2021): 113–124, 10.1007/s00280-020-04153-6.  [DOI](https://doi.org/10.1007/s00280-020-04153-6) | [PMC free article](/articles/PMC7801313/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33108504/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Chemotherapy%20and%20Pharmacology&title=Effect%20of%20Lurbinectedin%20on%20the%20QTc%20Interval%20in%20Patients%20With%20Advanced%20Solid%20Tumors:%20An%20Exposure%E2%80%90Response%20Analysis&author=S.%20Fudio&author=J.%20Tabernero&author=V.%20Subbiah&volume=87&issue=1&publication_year=2021&pages=113-124&pmid=33108504&doi=10.1007/s00280-020-04153-6&)

22. Trigo J., Subbiah V., Besse B., et al., “Lurbinectedin as Second‐Line Treatment for Patients With Small‐Cell Lung Cancer: A Single‐Arm, Open‐Label, Phase 2 Basket Trial,” Lancet Oncology 21, no. 5 (2020): 645–654, 10.1016/S1470-2045(20)30068-1.  [DOI](https://doi.org/10.1016/S1470-2045(20)30068-1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32224306/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet%20Oncology&title=Lurbinectedin%20as%20Second%E2%80%90Line%20Treatment%20for%20Patients%20With%20Small%E2%80%90Cell%20Lung%20Cancer:%20A%20Single%E2%80%90Arm,%20Open%E2%80%90Label,%20Phase%202%20Basket%20Trial&author=J.%20Trigo&author=V.%20Subbiah&author=B.%20Besse&volume=21&issue=5&publication_year=2020&pages=645-654&pmid=32224306&doi=10.1016/S1470-2045(20)30068-1&)

23. Farago A. F., Drapkin B. J., Lopez‐Vilarino de Ramos J. A., et al., “ATLANTIS: A Phase III Study of Lurbinectedin/Doxorubicin Versus Topotecan or Cyclophosphamide/Doxorubicin/Vincristine in Patients With Small‐Cell Lung Cancer Who Have Failed One Prior Platinum‐Containing Line,” Future Oncology 15, no. 3 (2019): 231–239, 10.2217/fon-2018-0597.  [DOI](https://doi.org/10.2217/fon-2018-0597) | [PMC free article](/articles/PMC6331752/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30362375/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Future%20Oncology&title=ATLANTIS:%20A%20Phase%20III%20Study%20of%20Lurbinectedin/Doxorubicin%20Versus%20Topotecan%20or%20Cyclophosphamide/Doxorubicin/Vincristine%20in%20Patients%20With%20Small%E2%80%90Cell%20Lung%20Cancer%20Who%20Have%20Failed%20One%20Prior%20Platinum%E2%80%90Containing%20Line&author=A.%20F.%20Farago&author=B.%20J.%20Drapkin&author=J.%20A.%20Lopez%E2%80%90Vilarino%20de%20Ramos&volume=15&issue=3&publication_year=2019&pages=231-239&pmid=30362375&doi=10.2217/fon-2018-0597&)

24. Leary A., Oaknin A., Trigo J. M., et al., “Pooled Safety Analysis of Single‐Agent Lurbinectedin in Patients With Advanced Solid Tumours,” European Journal of Cancer 192 (2023): 113259, 10.1016/j.ejca.2023.113259.  [DOI](https://doi.org/10.1016/j.ejca.2023.113259) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/37634282/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=European%20Journal%20of%20Cancer&title=Pooled%20Safety%20Analysis%20of%20Single%E2%80%90Agent%20Lurbinectedin%20in%20Patients%20With%20Advanced%20Solid%20Tumours&author=A.%20Leary&author=A.%20Oaknin&author=J.%20M.%20Trigo&volume=192&publication_year=2023&pages=113259&pmid=37634282&doi=10.1016/j.ejca.2023.113259&)

25. Fernandez‐Teruel C., Gonzalez I., Troconiz I. F., Lubomirov R., Soto A., and Fudio S., “Population‐Pharmacokinetic and Covariate Analysis of Lurbinectedin (PM01183), a New RNA Polymerase II Inhibitor, in Pooled Phase I/II Trials in Patients With Cancer,” Clinical Pharmacokinetics 58, no. 3 (2019): 363–374, 10.1007/s40262-018-0701-2.  [DOI](https://doi.org/10.1007/s40262-018-0701-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30090974/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clinical%20Pharmacokinetics&title=Population%E2%80%90Pharmacokinetic%20and%20Covariate%20Analysis%20of%20Lurbinectedin%20(PM01183),%20a%20New%20RNA%20Polymerase%20II%20Inhibitor,%20in%20Pooled%20Phase%20I/II%20Trials%20in%20Patients%20With%20Cancer&author=C.%20Fernandez%E2%80%90Teruel&author=I.%20Gonzalez&author=I.%20F.%20Troconiz&author=R.%20Lubomirov&author=A.%20Soto&volume=58&issue=3&publication_year=2019&pages=363-374&pmid=30090974&doi=10.1007/s40262-018-0701-2&)

26. Fernandez‐Teruel C., Fudio S., and Lubomirov R., “Integrated Exposure‐Response Analysis of Efficacy and Safety of Lurbinectedin to Support the Dose Regimen in Small‐Cell Lung Cancer,” Cancer Chemotherapy and Pharmacology 89, no. 5 (2022): 585–594, 10.1007/s00280-021-04366-3.  [DOI](https://doi.org/10.1007/s00280-021-04366-3) | [PMC free article](/articles/PMC9054899/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34739582/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Chemotherapy%20and%20Pharmacology&title=Integrated%20Exposure%E2%80%90Response%20Analysis%20of%20Efficacy%20and%20Safety%20of%20Lurbinectedin%20to%20Support%20the%20Dose%20Regimen%20in%20Small%E2%80%90Cell%20Lung%20Cancer&author=C.%20Fernandez%E2%80%90Teruel&author=S.%20Fudio&author=R.%20Lubomirov&volume=89&issue=5&publication_year=2022&pages=585-594&pmid=34739582&doi=10.1007/s00280-021-04366-3&)

27. Purcell S., Neale B., Todd‐Brown K., et al., “PLINK: A Tool Set for Whole‐Genome Association and Population‐Based Linkage Analyses,” American Journal of Human Genetics 81, no. 3 (2007): 559–575, 10.1086/519795.  [DOI](https://doi.org/10.1086/519795) | [PMC free article](/articles/PMC1950838/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17701901/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=American%20Journal%20of%20Human%20Genetics&title=PLINK:%20A%20Tool%20Set%20for%20Whole%E2%80%90Genome%20Association%20and%20Population%E2%80%90Based%20Linkage%20Analyses&author=S.%20Purcell&author=B.%20Neale&author=K.%20Todd%E2%80%90Brown&volume=81&issue=3&publication_year=2007&pages=559-575&pmid=17701901&doi=10.1086/519795&)

28. Beunk L., Nijenhuis M., Soree B., et al., “Dutch Pharmacogenetics Working Group (DPWG) Guideline for the Gene‐Drug Interaction Between CYP2D6, CYP3A4 and CYP1A2 and Antipsychotics,” European Journal of Human Genetics 32, no. 3 (2024): 278–285, 10.1038/s41431-023-01347-3.  [DOI](https://doi.org/10.1038/s41431-023-01347-3) | [PMC free article](/articles/PMC10923774/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/37002327/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=European%20Journal%20of%20Human%20Genetics&title=Dutch%20Pharmacogenetics%20Working%20Group%20(DPWG)%20Guideline%20for%20the%20Gene%E2%80%90Drug%20Interaction%20Between%20CYP2D6,%20CYP3A4%20and%20CYP1A2%20and%20Antipsychotics&author=L.%20Beunk&author=M.%20Nijenhuis&author=B.%20Soree&volume=32&issue=3&publication_year=2024&pages=278-285&pmid=37002327&doi=10.1038/s41431-023-01347-3&)

29. Birdwell K. A., Decker B., Barbarino J. M., et al., “Clinical Pharmacogenetics Implementation Consortium (CPIC) Guidelines for CYP3A5 Genotype and Tacrolimus Dosing,” Clinical Pharmacology and Therapeutics 98, no. 1 (2015): 19–24, 10.1002/cpt.113.  [DOI](https://doi.org/10.1002/cpt.113) | [PMC free article](/articles/PMC4481158/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25801146/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clinical%20Pharmacology%20and%20Therapeutics&title=Clinical%20Pharmacogenetics%20Implementation%20Consortium%20(CPIC)%20Guidelines%20for%20CYP3A5%20Genotype%20and%20Tacrolimus%20Dosing&author=K.%20A.%20Birdwell&author=B.%20Decker&author=J.%20M.%20Barbarino&volume=98&issue=1&publication_year=2015&pages=19-24&pmid=25801146&doi=10.1002/cpt.113&)

30. Saiz‐Rodriguez M., Almenara S., Navares‐Gomez M., et al., “Effect of the Most Relevant CYP3A4 and CYP3A5 Polymorphisms on the Pharmacokinetic Parameters of 10 CYP3A Substrates,” Biomedicine 8, no. 4 (2020): 94, 10.3390/biomedicines8040094.  [DOI](https://doi.org/10.3390/biomedicines8040094) | [PMC free article](/articles/PMC7235792/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32331352/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biomedicine&title=Effect%20of%20the%20Most%20Relevant%20CYP3A4%20and%20CYP3A5%20Polymorphisms%20on%20the%20Pharmacokinetic%20Parameters%20of%2010%20CYP3A%20Substrates&author=M.%20Saiz%E2%80%90Rodriguez&author=S.%20Almenara&author=M.%20Navares%E2%80%90Gomez&volume=8&issue=4&publication_year=2020&pages=94&pmid=32331352&doi=10.3390/biomedicines8040094&)

31. Agency EM , “European Medicines Agency Fluorouracil and Fluorouracil Related Substances (Capecitabine, Tegafur and Flucytosine) Containing Medicinal Products—Referral,” 2019.

32. Karas S. and Innocenti F., “All You Need to Know About UGT1A1 Genetic Testing for Patients Treated With Irinotecan: A Practitioner‐Friendly Guide,” JCO Oncology Practice 18, no. 4 (2022): 270–277, 10.1200/OP.21.00624.  [DOI](https://doi.org/10.1200/OP.21.00624) | [PMC free article](/articles/PMC9014426/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34860573/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JCO%20Oncology%20Practice&title=All%20You%20Need%20to%20Know%20About%20UGT1A1%20Genetic%20Testing%20for%20Patients%20Treated%20With%20Irinotecan:%20A%20Practitioner%E2%80%90Friendly%20Guide&author=S.%20Karas&author=F.%20Innocenti&volume=18&issue=4&publication_year=2022&pages=270-277&pmid=34860573&doi=10.1200/OP.21.00624&)

33. Hulshof E. C., Deenen M. J., Nijenhuis M., et al., “Dutch Pharmacogenetics Working Group (DPWG) Guideline for the Gene–Drug Interaction Between UGT1A1 and Irinotecan,” European Journal of Human Genetics 31, no. 9 (2022): 982–987, 10.1038/s41431-022-01243-2.  [DOI](https://doi.org/10.1038/s41431-022-01243-2) | [PMC free article](/articles/PMC10474017/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36443464/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=European%20Journal%20of%20Human%20Genetics&title=Dutch%20Pharmacogenetics%20Working%20Group%20(DPWG)%20Guideline%20for%20the%20Gene%E2%80%93Drug%20Interaction%20Between%20UGT1A1%20and%20Irinotecan&author=E.%20C.%20Hulshof&author=M.%20J.%20Deenen&author=M.%20Nijenhuis&volume=31&issue=9&publication_year=2022&pages=982-987&pmid=36443464&doi=10.1038/s41431-022-01243-2&)

34. Tseng E., Walsky R. L., R. A. Luzietti, Jr. , et al., “Relative Contributions of Cytochrome CYP3A4 Versus CYP3A5 for CYP3A‐Cleared Drugs Assessed In Vitro Using a CYP3A4‐Selective Inactivator (CYP3cide),” Drug Metabolism and Disposition 42, no. 7 (2014): 1163–1173, 10.1124/dmd.114.057000.  [DOI](https://doi.org/10.1124/dmd.114.057000) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24737844/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metabolism%20and%20Disposition&title=Relative%20Contributions%20of%20Cytochrome%20CYP3A4%20Versus%20CYP3A5%20for%20CYP3A%E2%80%90Cleared%20Drugs%20Assessed%20In%C2%A0Vitro%20Using%20a%20CYP3A4%E2%80%90Selective%20Inactivator%20(CYP3cide)&author=E.%20Tseng&author=R.%20L.%20Walsky&author=R.%20A.%20Luzietti&volume=42&issue=7&publication_year=2014&pages=1163-1173&pmid=24737844&doi=10.1124/dmd.114.057000&)

35. Moreno I., Hernandez T., Calvo E., et al., “Impact of a Moderate CYP3A4 Inducer (Bosentan) on Lurbinectedin Pharmacokinetics and Safety in Patients With Advanced Solid Tumors: An Open‐Label, Two‐Way, Crossover, Phase Ib Drug‐Drug Interaction Study,” Pharmaceuticals (Basel) 17, no. 2 (2024): 182, 10.3390/ph17020182.  [DOI](https://doi.org/10.3390/ph17020182) | [PMC free article](/articles/PMC10892269/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/38399397/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmaceuticals%20(Basel)&title=Impact%20of%20a%20Moderate%20CYP3A4%20Inducer%20(Bosentan)%20on%20Lurbinectedin%20Pharmacokinetics%20and%20Safety%20in%20Patients%20With%20Advanced%20Solid%20Tumors:%20An%20Open%E2%80%90Label,%20Two%E2%80%90Way,%20Crossover,%20Phase%20Ib%20Drug%E2%80%90Drug%20Interaction%20Study&author=I.%20Moreno&author=T.%20Hernandez&author=E.%20Calvo&volume=17&issue=2&publication_year=2024&pages=182&pmid=38399397&doi=10.3390/ph17020182&)

36. Moreno I., Hernandez T., Calvo E., et al., “Pharmacokinetics and Safety of Lurbinectedin Administrated With Itraconazole in Cancer Patients: A Drug‐Drug Interaction Study,” Marine Drugs 22, no. 4 (2024): 178, 10.3390/md22040178.  [DOI](https://doi.org/10.3390/md22040178) | [PMC free article](/articles/PMC11050816/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/38667795/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Marine%20Drugs&title=Pharmacokinetics%20and%20Safety%20of%20Lurbinectedin%20Administrated%20With%20Itraconazole%20in%20Cancer%20Patients:%20A%20Drug%E2%80%90Drug%20Interaction%20Study&author=I.%20Moreno&author=T.%20Hernandez&author=E.%20Calvo&volume=22&issue=4&publication_year=2024&pages=178&pmid=38667795&doi=10.3390/md22040178&)

37. Moreira R. P., Jorge A. A., Gomes L. G., et al., “Pharmacogenetics of Glucocorticoid Replacement Could Optimize the Treatment of Congenital Adrenal Hyperplasia due to 21‐Hydroxylase Deficiency,” Clinics (São Paulo, Brazil) 66, no. 8 (2011): 1361–1366, 10.1590/s1807-59322011000800009.  [DOI](https://doi.org/10.1590/s1807-59322011000800009) | [PMC free article](/articles/PMC3161212/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21915484/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clinics%20(S%C3%A3o%20Paulo,%20Brazil)&title=Pharmacogenetics%20of%20Glucocorticoid%20Replacement%20Could%20Optimize%20the%20Treatment%20of%20Congenital%20Adrenal%20Hyperplasia%20due%20to%2021%E2%80%90Hydroxylase%20Deficiency&author=R.%20P.%20Moreira&author=A.%20A.%20Jorge&author=L.%20G.%20Gomes&volume=66&issue=8&publication_year=2011&pages=1361-1366&pmid=21915484&doi=10.1590/s1807-59322011000800009&)

38. Johnson N., Maguire S., Morra A., et al., “CYP3A7*1C Allele: Linking Premenopausal Oestrone and Progesterone Levels With Risk of Hormone Receptor‐Positive Breast Cancers,” British Journal of Cancer 124, no. 4 (2021): 842–854, 10.1038/s41416-020-01185-w.  [DOI](https://doi.org/10.1038/s41416-020-01185-w) | [PMC free article](/articles/PMC7884683/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33495599/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=British%20Journal%20of%20Cancer&title=CYP3A7*1C%20Allele:%20Linking%20Premenopausal%20Oestrone%20and%20Progesterone%20Levels%20With%20Risk%20of%20Hormone%20Receptor%E2%80%90Positive%20Breast%20Cancers&author=N.%20Johnson&author=S.%20Maguire&author=A.%20Morra&volume=124&issue=4&publication_year=2021&pages=842-854&pmid=33495599&doi=10.1038/s41416-020-01185-w&)
