# Impact of Pharmacogenetics on Pharmacokinetics of First-Line Antituberculosis Drugs in the HIRIF Trial

## Metadata
**Authors:** Elizabeth Mackay, Gareth Platt, Charles A Peloquin, Meredith B Brooks, Julia M Coit, Gustavo E Velásquez, Henry Pertinez, Dante Vargas, Epifanio Sánchez, Roger I Calderón, Judith Jiménez, Karen Tintaya, Dalicxa Garcia, Elna Osso, Leo Lecca, Carole Mitnick, Geraint R Davies
**Journal:** The Journal of Infectious Diseases
**Date:** 2025 Apr 17
**DOI:** [10.1093/infdis/jiaf195](https://doi.org/10.1093/infdis/jiaf195)
**PMID:** 40239986
**PMCID:** PMC12349936
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC12349936/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC12349936/pdf/jiaf195.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC12349936/pdf/jiaf195.pdf)

## Abstract

**Background:** 
Variability in the pharmacokinetics (PK) of first-line antituberculosis drugs (rifampicin [RIF], isoniazid [INH], and pyrazinamide [PZA]) is high and may be influenced by pharmacogenetic polymorphism. We performed a pharmacogenetic substudy in 90 participants with PK data from the HIRIF trial in Peru.

**Methods:** 
Relevant single-nucleotide polymorphisms (SNPs) in the NAT2, SLCO1B1, AADAC, and AOX1 loci were genotyped using real-time polymerase chain reaction (PCR).

**Results:** 
The proportions of slow, intermediate, and fast acetylators predicted by a conventional 6-SNP NAT2 panel were 32.5%, 48.2%, and 19.2%, respectively. A single NAT2 tag SNP (rs1495741) agreed with the panel-predicted phenotype in 91% and was a better predictor of INH area under the curve (AUC). Accounting for discrepancies possibly caused by rare alleles not represented in the panel or that could be unequivocally resolved using observed AUC, sensitivity of the tag SNP was 97.7%. A previously described SNP in SLCO1B1 (rs4149032) was present at an allele frequency of 0.31 and appeared to influence RIF AUC and maximum concentration (Cmax) at a dose of 20 mg/kg, despite an extreme distribution of alleles across the randomized arms. The AADAC SNP (rs1803155) predominated in the study population and was not linked to RIF PK, although an effect could have been missed due to sample size and allele frequency. There was no association between PZA PK and a common SNP in AOX1 (rs55754655).

**Conclusions:** 
A tag SNP approach may offer simpler and cheaper prediction of INH PK. Further exploration of the impact of SLCO1B1 SNPs on RIF PK is required in this and other populations.

Clinical Trials Registration. NCT01408914.

Keywords: tuberculosis, pharmacokinetics, pharmacogenetics, rifampicin, isoniazid, NAT2, SLCO1B1

### Background

Variability in the pharmacokinetics (PK) of first-line antituberculosis drugs (rifampicin [RIF], isoniazid [INH], and pyrazinamide [PZA]) is high and may be influenced by pharmacogenetic polymorphism. We performed a pharmacogenetic substudy in 90 participants with PK data from the HIRIF trial in Peru.

### Methods

Relevant single-nucleotide polymorphisms (SNPs) in the *NAT2, SLCO1B1*, *AADAC,* and *AOX1* loci were genotyped using real-time polymerase chain reaction (PCR).

### Results

The proportions of slow, intermediate, and fast acetylators predicted by a conventional 6-SNP *NAT2* panel were 32.5%, 48.2%, and 19.2%, respectively. A single *NAT2* tag SNP (rs1495741) agreed with the panel-predicted phenotype in 91% and was a better predictor of INH area under the curve (AUC). Accounting for discrepancies possibly caused by rare alleles not represented in the panel or that could be unequivocally resolved using observed AUC, sensitivity of the tag SNP was 97.7%. A previously described SNP in *SLCO1B1* (rs4149032) was present at an allele frequency of 0.31 and appeared to influence RIF AUC and maximum concentration (C_max_) at a dose of 20 mg/kg, despite an extreme distribution of alleles across the randomized arms. The *AADAC* SNP (rs1803155) predominated in the study population and was not linked to RIF PK, although an effect could have been missed due to sample size and allele frequency. There was no association between PZA PK and a common SNP in *AOX1* (rs55754655).

### Conclusions

A tag SNP approach may offer simpler and cheaper prediction of INH PK. Further exploration of the impact of *SLCO1B1* SNPs on RIF PK is required in this and other populations.

**Clinical Trials Registration**. [NCT01408914](https://clinicaltrials.gov/ct2/show/NCT01408914).

## METHODS

HIRIF was a phase 2B randomized trial comparing RIF doses of 10, 15, and 20 mg/kg/day (delivered with standard weight-based doses of companion drugs) during the intensive phase of tuberculosis treatment ([NCT01408914](https://clinicaltrials.gov/ct2/show/NCT01408914)). Adults with newly diagnosed, previously untreated, smear positive (≥ 2+) pulmonary tuberculosis were recruited in Lima, Peru. All study procedures, including the separate PK and pharmacogenetics informed consent processes, were approved by the sponsors’ and local ethical oversight bodies [[21](#jiaf195-B21)].

Plasma concentrations of RIF, INH, and PZA were obtained using a hybrid intensive/sparse sampling scheme after 2 weeks of dosing. Drugs were assayed using validated Liquid Chromatography-Mass Spectrometry (LC-MS) methods at the University of Florida and summary PK parameters including the peak plasma concentration (C_max_) and area under the plasma concentration time curve (AUC_all_) were obtained by noncompartmental methods as previously described [[22](#jiaf195-B22)].

Whole blood was collected during the PK visit and frozen at −80°C. Genomic DNA was extracted at the University of Liverpool using the QIAamp Blood Midi kit (Qiagen). Genotyping was performed in duplicate by real-time polymerase chain reaction (PCR) for the following SNPs and loci: *NAT2*5* (rs1801280), *NAT2*6* (rs1799930), *NAT2*7* (rs1799931), *NAT2*12* (rs1208), *NAT2*13* (rs1041983), *NAT2*14* (rs1801279), a tag SNP located 14 kb 3′ of the *NAT2* exon (rs1495741), *SLCO1B1* (rs4149032), *AADAC* (rs1803155), and *AOX1* (rs55754655). Briefly, 23 μL of PCR master mix was placed in each of the central wells of a 96-well white PCR plate (MJ) comprising the following: 12.5 μL ABgene qPCR mix MM, 9.25 μL double-distilled water (Sigma-Aldrich), and 1.25 μL primer/probe mix. Then, 2 μL of sample genomic DNA at 20 ng/mL (or 2 μL of double distilled water for blanks) was added to each well and the plate sealed with a clear membrane before spinning for 10 seconds at 1000*g*. PCR amplification was performed in an Opticon 2.0 thermal cycler (Bio-Rad Laboratories) with an initial cycle of 95°C for 10 minutes (denaturation/enzyme activation step) followed by 49 cycles of 15 seconds at 92°C and 90 seconds at 60°C (combined annealing/extension step). Genotypes were analyzed using MJ Opticon Monitor analysis software using the cycle threshold values for the dyes FAM and VIC corresponding to the different alleles.

Predictions of acetylator phenotype were based on genetic diplotype according to the number of mutant loss of function alleles present (*5, *6, *7, or *14): fast (no mutant alleles), intermediate (1 mutant allele), and slow (2 mutant alleles). Linkage analysis and analysis of variance models were performed in R version 3.4.4/Bioconductor. PK parameters were analyzed on a logarithmic scale and nested linear models using a weight-adjusted dose metric and different genetic models were compared using the Akaike information criterion (AIC), likelihood ratio test and Wald tests of the coefficients.

## RESULTS

Genotyping was performed on 90 participants in the HIRIF trial who had consented to genetic analysis and had paired PK data available unaffected by study halts. All SNPs genotyped were in Hardy-Weinberg equilibrium.

The mutant SNP allele frequencies for the 6 SNP *NAT2* panel were as follows: *NAT2*5* (rs1801280) 0.23, *NAT2*6* (rs1799930) 0.09, *NAT2*7* (rs1799931) 0.27, *NAT2*12* (rs1208) 0.22, *NAT2*13* (rs1041983) 0.37, and *NAT2*14* (rs1801279) 0.00. The allele frequency for the *NAT2* tag SNP (rs1495741) was 0.41. As expected, the 7 *NAT2* SNPs were in strong linkage disequilibrium, particularly the tag SNP which had a D′ value of 0.91 or greater for all other *NAT2* SNPs ([Figure 1](#jiaf195-F1)). Because no mutants were detected for the *NAT2*14* (rs1801279) allele in the study population, it was not considered further in our analysis.

### Figure 1.

![Figure 1.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/57a6/12349936/d2f6b728e304/jiaf195f1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=12349936_jiaf195f1.jpg)

Linkage disequilibrium (LD) heatmap for NAT2 alleles.

The distribution of INH AUC exhibited at least 2 prominent modes as expected ([Figure 1](#jiaf195-F1)). Conventional genotypic predictions of the proportions of slow, intermediate, and fast INH acetylators using the 6-SNP panel were 32.5%, 48.2%, and 19.2%, respectively. Concordance between the 6-SNP panel and the tag SNP was generally high with disagreement in only 8 participants (8.9%) ([Table 1](#jiaf195-T1)). Notably, 3 of these discrepancies were predicted as fast acetylators by the 6-SNP panel but as slow acetylators by the tag SNP. In all of these participants, the observed INH AUC was greater than 12 μg/mL*h, clustering close to the mean AUC of the slow acetylator group (13.7 μg/mL*h) rather than that of the fast acetylator group (5.2 μg/mL*h) and lying beyond the predicted distribution of AUC for fast acetylators obtained by either genotyping method ([Figure 2](#jiaf195-F2)). These pharmacokinetic exposures appear clearly consistent with a slow acetylator phenotype, suggesting that the panel prediction was incorrect. The other 5 discordant predictions involved discrepancies only of a single allele and were distributed symmetrically in the confusion matrix, although 3 could also be resolved unequivocally in favor of the tag SNP based on their AUC, giving an overall sensitivity of 97.7%.

### Table 1.

|   | Fast | Intermediate | Slow |
| --- | --- | --- | --- |
| AA | 3 | 1 | 25 |
| AG | 0 | 37 | 2 |
| GG | 13 | 2 | 0 |

Table 1 Caption: Confusion Matrix for Tag SNP (rs1495741) Versus 6-SNP Panel Predicted Acetylator Phenotype

### Figure 2.

![Figure 2.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/57a6/12349936/6970658726a9/jiaf195f2.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=12349936_jiaf195f2.jpg)

Distribution of isoniazid (INH) area under the curve (AUC) with superimposed nonparametric scaled density estimates for rs1495741 genotypes (NAT2 tag single-nucleotide polymorphism).

In analysis of variance models, the tag SNP predicted the INH AUC significantly better than the conventional 6-SNP method in the study population ([Table 2](#jiaf195-T2)) and this advantage remained even when the 3 widely discrepant predictions were excluded from the model (AIC −2.176 and 9.004 for the tag SNP and panel, respectively). Both variables were much stronger predictors of INH AUC than the weight-adjusted INH dose itself. The conclusions of the analysis using C_max_ as a measure of PK exposure instead of AUC were similar.

### Table 2.

| Model | Point Estimate | 95% CI | Wald Test P Value | AIC |
| --- | --- | --- | --- | --- |
| AUCall, μg/mL*h |  |  |  |  |
| Base model | Intercept 6.711 | 5.665–7.950 | <.001 | 60.436 |
| SNP panel | Fast 3.721 | 2.725–5.081 | <.001 | 26.841 |
| Intermediate 5.487 | 3.795–7.932 | .042 |   |  |
| Slow 12.829 | 8.659–19.007 | <.001 |   |  |
| rs1495741 | AA 13.962 | 11.544–16.887 | <.001 | −5.726 |
| AG 5.479 | 4.262–7.043 | <.001 |   |  |
| GG 2.760 | 1.992–3.821 | <.001 |   |  |
| INH dose, mg/kg | per mg/kg −0.231 | −.161 to −.331 | .870 | 63.093 |
| Cmax, μg/mL |  |  |  |  |
| Base model | Intercept 2.157 | 1.902–2.447 | <.001 | 11.204 |
| SNP panel | Fast 1.430 | 1.127–1.813 | .004 | −17.841 |
| Intermediate 1.866 | 1.408–2.473 | .067 |   |  |
| Slow 3.412 | 2.527–4.607 | <.001 |   |  |
| rs1495741 | AA 3.628 | 3.122–4.215 | <.001 | −45.067 |
| AG 1.863 | 1.528–2.271 | <.001 |   |  |
| GG 1.156 | .894–1.495 | <.001 |   |  |
| INH dose, mg/kg | Per mg/kg 0.180 | .051–.253 | .222 | 11.666 |

Table 2 Caption: One-Way ANOVA of Performance of the Tag SNP (rs1495741) Versus 6-SNP Panel in Predicted INH AUC0–6 and Cmax

The mutant allele frequency for *SLCO1B1* (rs4149032) was 0.31. Although this SNP was relatively common, it was not distributed evenly across the trial arms. The frequencies of the mutant allele in the 10, 15, and 20 mg/kg dose groups were 0.31, 0.48, and 0.80, respectively, with 50% of those carrying mutant alleles in the highest dose group. The probability of this distribution of alleles having occurred during randomization by chance was 1 in 416 (Fisher exact test *P* = .002) but seems the only explanation, given the lack of any possible causal association between assignment to trial arm and *SLCO1B1* genotype. This correlation created substantial collinearity with trial arm and therefore with RIF dose in exploring the impact of the 2 variables.

Exploratory analysis supported empirical choice of a dominant genetic model for this locus and suggested that the relationship of RIF AUC and C_max_ to dose was much stronger in an analysis stratified by the rs4149032 SNP, with a possible threshold effect for doses above 10 mg/kg ([Figure 3](#jiaf195-F3)). Indeed, the impact of RIF weight-adjusted dose was inconsistent and of borderline statistical significance in wild-type participants but was consistent and strongly significant in participants with any *SLCO1B1* mutation. While in univariate analysis both trial arm and rs4149032 were associated with measures of RIF PK exposure, only the former was retained in the final model. However, when the dose was instead categorized at a threshold of 20 mg/kg there was some evidence of an impact of rs4149032 mutants, which was comparable in magnitude to a doubling of RIF dose. Although this analysis was thus supportive of an independent effect of the SNP at higher doses of RIF, all full interaction models fitted resulted in a substantial rise in AIC, often with unstable coefficients suggesting over-parameterization ([Table 3](#jiaf195-T3)).

### Figure 3.

![Figure 3.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/57a6/12349936/a8ee386836e7/jiaf195f3.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=12349936_jiaf195f3.jpg)

Rifampicin (RIF) plasma area under the curve (AUC) by dose level, stratified by SLCO1B1 genotype (rs4149032), wild type (left) and mutant (right), using a dominant genetic model. Boxplots show median, interquartile range, range (whiskers) and any outliers.

### Table 3.

| Model | Point Estimate | 95% CI | Wald Test P Value | AIC |
| --- | --- | --- | --- | --- |
| AUCall, μg/mL*h |  |  |  |  |
| Base model | Intercept 41.805 | 36.247– 48.215 | <.001 | 34.628 |
| Trial arm | 10 mg/kg 26.298 | 21.151–32.695 | <.001 | 14.089 |
| 15 mg/kg 47.970 | 35.436–64.936 | <.001 |   |  |
| 20 mg/kg 58.028 | 45.536–79.182 | <.001 |   |  |
| Dose 20 mg/kg | <20 mg/kg 35.887 | 30.468–42.270 | <.001 | 26.488 |
| 20 mg/kg 58.028 | 43.400–77.586 | .002 |   |  |
| rs4149032, dominant model | Wild type 34.459 | 28.162–42.163 | <.001 | 30.153 |
| Mutant 50.200 | 37.984–66.342 | .010 |   |  |
| Trial arm + rs4149032 | 10 mg/kg 25.223 | 20.003–31.805 | <.001 | 15.235 |
| 15 mg/kg 45.186 | 33.063–61.756 | <.001 |   |  |
| 20 mg/kg 51.796 | 36.885–72.736 | <.001 |   |  |
| Mutant (increase) 3.966 | 2.169–6.623 | .301 |   |  |
| Dose 20 mg/kg + rs4149032 | 10 mg/kg/WT 32.531 | 26.598–39.788 | <.001 | 26.226 |
| 20 mg/kg/WT 47.751 | 35.053–65.049 | .017 |   |  |
| Mutant (increase) 9.266 | 4.707–16.019 | .093 |   |  |
| Cmax, μg/mL) |  |  |  |  |
| Base model | Intercept 8.946 | 7.858–10.185 | <.001 | 18.42 |
| Trial arm | 10 mg/kg 6.003 | 4.909–7.340 | <.001 | 0.614 |
| 15 mg/kg 10.026 | 7.634–13.167 | <.001 |   |  |
| 20 mg/kg 11.922 | 8.947–15.887 | <.001 |   |  |
| Dose 20 mg/kg | <20 mg/kg 7.826 | 6.740–9.089 | <.001 | 11.049 |
| 20 mg/kg 11.922 | 9.145–15.543 | .003 |   |  |
| rs4149032 | Wild type 7.384 | 6.153–8.861 | <.001 | 13.164 |
| Mutant 10.679 | 8.301–13.740 | .005 |   |  |
| Trial arm + rs4149032 | 10 mg/kg 5.706 | 4.278–7.610 | <.001 | 1.548 |
| 15 mg/kg 9.221 | 6.743–12.609 | .002 |   |  |
| 20 mg/kg 10.386 | 8.060–13.383 | <.001 |   |  |
| Mutant (increase) 1.107 | 1.009–1.169 | .174 |   |  |
| Dose 20 mg/kg + rs4149032 | 10 mg/kg 7.035 | 5.859–8.448 | <.001 | 10.065 |
| 20 mg/kg 9.713 | 7.336–12.862 | .027 |   |  |
| Mutant (increase) 2.121 | 1.182–3.457 | .053 |   |  |

Table 3 Caption: Two-Way ANOVA of RIF dose and the SLCO1B1 SNP rs4149032

The *AADAC* rs1803155 SNP was present at an allele frequency of 0.86 . It was not associated with changes in RIF C_max_ and AUC and there was no significant statistical interaction with RIF dose or with the *SLCO1B1* rs4149032 SNP. The allele frequency for the *AOX1* SNP (rs55754655) in the study population was low at 0.03 and was not associated with changes in the pharmacokinetic parameters of PZA.

## DISCUSSION

While the impact of the highly polymorphic *NAT2* locus on the pharmacokinetics of INH has long been recognized, there are relatively few data on the distribution of *NAT2* alleles in patients with tuberculosis in Latin America [[23–25](#jiaf195-B23)]. By contrast, there have been fewer and apparently conflicting reports on the pharmacogenetic basis of the high interindividual PK variability of RIF, with both the *SLCO1B1* and *AADAC* loci possibly implicated in studies from different geographical areas [[9–19](#jiaf195-B9)]. We sought to address these questions in this substudy of Peruvian patients with tuberculosis randomized to 3 dose levels of RIF during first-line treatment in a clinical trial and who also underwent PK sampling [[21](#jiaf195-B21), [22](#jiaf195-B22)].

Mutant *NAT2* alleles associated with slower acetylation were common in our cohort resulting in a predicted proportion of fast acetylators of 19.2%, similar to recent reports in adult patients with tuberculosis from Peru [[25](#jiaf195-B25)], Bolivia and Argentina [[24](#jiaf195-B24)], and in children with tuberculosis in Venezuela [[23](#jiaf195-B23)]. This proportion is lower than studies in various Native American populations [[24](#jiaf195-B24)] and likely reflects the overall impact of historical genetic admixture from Europe and Asia in the Peruvian population.

The diverse *NAT2* alleles are known to form a haploblock with the tag SNP rs1495741, located 14 kb 3′ of the *NAT2* exon, suggesting that this single SNP may be capable of adequately representing the genetic variation captured by the traditional 6- or 7-SNP panel [[26](#jiaf195-B26)]. We found the tag SNP to be highly concordant with the predictions of the 6-SNP panel and to be more accurate in cases of extreme discrepancy, which could reflect the presence in the population of rare mutant alleles not captured by the panel. These findings are in agreement with those of previous studies, including that of Chamorro et al who claimed a sensitivity of 97% and specificity of 99.3% of the tag SNP for prediction of slow acetylator phenotype in a Latin American context [[27](#jiaf195-B27)]. This is the group of patients at highest risk of drug-induced liver injury whilst on tuberculosis treatment [[28](#jiaf195-B28)] who may benefit most from stratified approaches to dosing of INH. While it may be possible to improve performance using more sophisticated bioinformatic approaches to SNP panels [[29](#jiaf195-B29)] or targeted sequencing, this simpler approach may be sufficiently accurate for clinical purposes and worthy of further evaluation.

The metabolism of RIF is complex and dynamic due to saturable autoinduction of metabolism and nonlinear pharmacokinetics at doses higher than 10 mg/kg [[3](#jiaf195-B3), [6](#jiaf195-B6)]. RIF is a substrate for the hepatic organic anion uptake transporter OATP1B1 [[30](#jiaf195-B30)] but may also directly inhibit uptake of other substrates [[31](#jiaf195-B31)], possibly through competitive mechanisms. OATP1B1 is coded by the gene locus *SLCO1B1* in which at least 14 nonsynonymous SNPs have been identified, 6 of which have been characterized as conferring loss of function [[32](#jiaf195-B32)]. Such mutations would be expected to reduce accessibility of RIF to hepatic clearance and to be associated with higher plasma PK exposures.

rs4149032 is an SNP in *SLCO1B1* previously linked to lower RIF exposure in some African populations [[9–11](#jiaf195-B9)]. However, this association has not been confirmed in other studies from Africa [[13–16](#jiaf195-B13)] and India [[12](#jiaf195-B12)]. We did not observe such an association in our cohort in Peru. This may be because rs4149032 is an intronic SNP and in variable linkage disequilibrium with nonsynonymous SNPs such as rs11045819 (*4) and rs4149056 (*5) in different populations. The haplotypic structure of *SLCO1B1* is complex with relatively large haploblocks, which may be variably captured by proposed tag SNPs in different populations [[33](#jiaf195-B33)]. Hence the performance of rs4149032 as a tag SNP may be population dependent, varying with the local allele frequencies of the different SNPs and their correlation. Few studies to date have simultaneously characterized all the relevant SNPs in *SLCO1B1*, focusing on previously identified candidate SNPs in isolation. However, other factors might also have an impact, including the relatively small sample size in most of the studies, precision of the PK parameter estimates obtained, and timing of samples in relation to autoinduction of metabolism.

Our study is the first to our knowledge to examine the impact of *SLCO1B1* SNPs in the context of higher doses of RIF than are conventionally used. Plasma AUC of RIF appears to be nonlinear with doses above 10 mg/kg [[3](#jiaf195-B3)] and saturability of hepatic uptake mechanisms is one possible explanation of this observation, a hypothesis on which the HIRIF trial could potentially provide information. Although we observed a greater increase in RIF AUC in the presence of any *SLCO1B1* mutation at higher doses of RIF, interpretation of this finding was complicated by the chance occurrence of an extreme distribution of mutant alleles across the treatment arms. Hence, we can only speculate that, while participants with wild-type OATP1B1 may have retained additional capacity for RIF transport at higher doses, those with *SLCO1B1* loss-of-function mutations might be expected to have no such reserve and thus greater increases in plasma exposure of RIF. Hence, the nonlinearity and interindividual variability of RIF PK at higher doses could be influenced importantly by polymorphism in *SLCO1B1,* and should be studied in more detail in a wider range of populations.

Although it has long been suggested that the primary metabolic pathway for rifamycins in humans involves hepatic esterases, more recent detailed in vitro studies have proposed that they are substrates for arylacetamide deacetylase (AADAC), a microsomal serine deacetylase, rather than the better characterized carboxylesterases (CES-1/2), which are responsible for activation of numerous ester prodrugs [[34](#jiaf195-B34)]. Two studies have associated lower clearance and higher plasma exposure of the related rifamycin rifapentine with the rs1803155 SNP in *AADAC* at 2 different dose levels [[17](#jiaf195-B17), [18](#jiaf195-B18)] and another study reported similar findings for RIF [[19](#jiaf195-B19)] at the standard dose [[12](#jiaf195-B12)]. We did not confirm any change in RIF exposure with this SNP across any of the doses studied and this analysis did not suffer from the difficulties caused by the extreme distribution of rs4149032 alleles. The primary metabolites of RIF and rifapentine are formed by esterase attack at the same functional group (25 position) so this finding is perhaps unexpected. It is notable that these prior studies almost exclusively included people of African heritage and that in the few non-African participants AADAC was not associated with rifapentine exposure [[18](#jiaf195-B18)]. However, in agreement with another recent study from Peru [[25](#jiaf195-B25)], the rs1803155 mutant allele frequency was higher in our study population (0.86) than in these previous studies, which ranged from 0.70 to 0.82, and the sample size was smaller. These factors would make it difficult to detect any effect, particularly under a dominant genetic model, because of the rarity of wild-type homozygotes. Larger studies in other populations, where the mutant allele frequency is lower, may be required to clarify the impact of *AADAC* polymorphism on PK of rifamycins.

Although the xanthine oxidase pathway has been implicated in the metabolic schema of PZA the *AOX1* locus has not previously been explored as a possible source of PK variability of the drug. We examined only one common candidate SNP, which proved to be uncommon in the trial population so this question may be worthy of further study in other populations.

Although our study was conducted under clinical trial conditions and high-quality pharmacokinetic and genotyping data were available at a range of RIF doses, our conclusions were limited by the relatively small sample size of the substudy and our inability to extensively characterize the haplotypic structure of the *SLCO1B1* and *AADAC* loci.

We have shown that the proportion of fast acetylators is relatively low in patients with tuberculosis in Lima and that a single common tag SNP may offer a reliable and simple test to determine acetylator status in this population. We also suggest that SNPs in *SLCO1B1* may be implicated in the nonlinearity of RIF PK at higher doses and should be further studied in a wider range of settings.

## Contributor Information

Elizabeth Mackay, Department of Clinical Infection, Microbiology and immunology, University of Liverpool, Liverpool, United Kingdom.

Gareth Platt, Department of Clinical Infection, Microbiology and immunology, University of Liverpool, Liverpool, United Kingdom.

Charles A Peloquin, Department of Pharmacotherapy and Translational Research, University of Florida, Gainesville, Florida, USA.

Meredith B Brooks, Department of Global Health, Boston University School of Public Health, Boston, Massachusetts, USA.

Julia M Coit, Deparment of Global Health and Social Medicine, Harvard Medical School, Boston, Massachusetts, USA.

Gustavo E Velásquez, Department of Medicine, University of California, San Francisco, California, USA.

Henry Pertinez, Department of Clinical Infection, Microbiology and immunology, University of Liverpool, Liverpool, United Kingdom.

Dante Vargas, Department of Pneumonology, Hospital Nacional Hipolito Unanue, Lima, Peru.

Epifanio Sánchez, Department of Pneumonology, Hospital Nacional Sergio Bernales, Lima, Peru.

Roger I Calderón, Advanced Research and Health, Lima, Peru.

Judith Jiménez, Socios en Salud, Lima, Peru.

Karen Tintaya, Socios en Salud, Lima, Peru.

Dalicxa Garcia, Socios en Salud, Lima, Peru.

Elna Osso, Deparment of Global Health and Social Medicine, Harvard Medical School, Boston, Massachusetts, USA.

Leo Lecca, Deparment of Global Health and Social Medicine, Harvard Medical School, Boston, Massachusetts, USA; Socios en Salud, Lima, Peru.

Carole Mitnick, Deparment of Global Health and Social Medicine, Harvard Medical School, Boston, Massachusetts, USA.

Geraint R Davies, Department of Clinical Infection, Microbiology and immunology, University of Liverpool, Liverpool, United Kingdom.

## Notes

** Acknowledgment. ** Rifampicin for the HIRIF trial was donated by Sanofi-Aventis.

** Author contributions. ** E. M. and G. P. performed the genotyping and C. P. the pharmacokinetic analysis. G. D. and H. P. carried out the statistical analysis. C. M., L. L., G. V., D. V. V., E. G. S., M. B., J. C., E. O., D. G., K. T., J. J., and R. D. were members of the study team who executed the trial and prepared the data. All authors read and approved the final manuscript.

** Financial support. ** This work was supported by the US National Institutes of Health (grant numbers U01AI091429 to C. M. for the HIRIF trial, and K08AI141740 to G. V. for analysis).

## References

1. Tiberi S, du Plessis N, Walzl G, et al. Tuberculosis: progress and advances in development of new drugs, treatment regimens, and host-directed therapies. Lancet Infect Dis 2018; 18:e183–98.  [DOI](https://doi.org/10.1016/S1473-3099(18)30110-5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29580819/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet%20Infect%20Dis&title=Tuberculosis:%20progress%20and%20advances%20in%20development%20of%20new%20drugs,%20treatment%20regimens,%20and%20host-directed%20therapies&volume=18&publication_year=2018&pages=e183-98&pmid=29580819&doi=10.1016/S1473-3099(18)30110-5&)

2. Sturkenboom MGG, Märtson AG, Svensson EM, et al. Population pharmacokinetics and Bayesian dose adjustment to advance TDM of anti-TB drugs. Clin Pharmacokinet 2021; 60:685–710.  [DOI](https://doi.org/10.1007/s40262-021-00997-0) | [PMC free article](/articles/PMC7935699/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33674941/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacokinet&title=Population%20pharmacokinetics%20and%20Bayesian%20dose%20adjustment%20to%20advance%20TDM%20of%20anti-TB%20drugs&volume=60&publication_year=2021&pages=685-710&pmid=33674941&doi=10.1007/s40262-021-00997-0&)

3. Te Brake LHM, de Jager V, Narunsky K, et al. PanACEA consortium. Increased bactericidal activity but dose-limiting intolerability at 50 mg·kg-1 rifampicin. Eur Respir J 2021; 58:2000955.  [DOI](https://doi.org/10.1183/13993003.00955-2020) | [PMC free article](/articles/PMC8411896/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33542056/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20Respir%20J&title=PanACEA%20consortium.%20Increased%20bactericidal%20activity%20but%20dose-limiting%20intolerability%20at%2050%E2%80%85mg%C2%B7kg-1%20rifampicin&volume=58&publication_year=2021&pages=2000955&pmid=33542056&doi=10.1183/13993003.00955-2020&)

4. Boeree MJ, Heinrich N, Aarnoutse R, et al. PanACEA consortium.high-dose rifampicin, moxifloxacin, and SQ109 for treating tuberculosis: a multi-arm, multi-stage randomised controlled trial. Lancet Infect Dis 2017; 17:39–49.  [DOI](https://doi.org/10.1016/S1473-3099(16)30274-2) | [PMC free article](/articles/PMC5159618/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28100438/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet%20Infect%20Dis&title=PanACEA%20consortium.high-dose%20rifampicin,%20moxifloxacin,%20and%20SQ109%20for%20treating%20tuberculosis:%20a%20multi-arm,%20multi-stage%20randomised%20controlled%20trial&volume=17&publication_year=2017&pages=39-49&pmid=28100438&doi=10.1016/S1473-3099(16)30274-2&)

5. Hong BL, D'Cunha R, Li P, et al. A systematic review and meta-analysis of isoniazid pharmacokinetics in healthy volunteers and patients with tuberculosis. Clin Ther 2020; 42:e220–41.  [DOI](https://doi.org/10.1016/j.clinthera.2020.09.009) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33032843/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Ther&title=A%20systematic%20review%20and%20meta-analysis%20of%20isoniazid%20pharmacokinetics%20in%20healthy%20volunteers%20and%20patients%20with%20tuberculosis&volume=42&publication_year=2020&pages=e220-41&pmid=33032843&doi=10.1016/j.clinthera.2020.09.009&)

6. Azuma J, Ohno M, Kubota R, et al. Pharmacogenetics-based tuberculosis therapy research group. NAT2 genotype guided regimen reduces isoniazid-induced liver injury and early treatment failure in the 6-month four-drug standard treatment of tuberculosis: a randomized controlled trial for pharmacogenetics-based therapy. Eur J Clin Pharmacol 2013; 69:1091–101.  [DOI](https://doi.org/10.1007/s00228-012-1429-9) | [PMC free article](/articles/PMC3641305/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23150149/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Clin%20Pharmacol&title=Pharmacogenetics-based%20tuberculosis%20therapy%20research%20group.%20NAT2%20genotype%20guided%20regimen%20reduces%20isoniazid-induced%20liver%20injury%20and%20early%20treatment%20failure%20in%20the%206-month%20four-drug%20standard%20treatment%20of%20tuberculosis:%20a%20randomized%20controlled%20trial%20for%20pharmacogenetics-based%20therapy&volume=69&publication_year=2013&pages=1091-101&pmid=23150149&doi=10.1007/s00228-012-1429-9&)

7. Yoo H, Ji SC, Cho JY, et al. A pilot study to investigate the utility of NAT2 genotype-guided isoniazid monotherapy regimens in NAT2 slow acetylators. Pharmacogenet Genomics 2021; 31:68–73.  [DOI](https://doi.org/10.1097/FPC.0000000000000423) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33165168/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=A%20pilot%20study%20to%20investigate%20the%20utility%20of%20NAT2%20genotype-guided%20isoniazid%20monotherapy%20regimens%20in%20NAT2%20slow%20acetylators&volume=31&publication_year=2021&pages=68-73&pmid=33165168&doi=10.1097/FPC.0000000000000423&)

8. Stott KE, Pertinez H, Sturkenboom MGG, et al. Pharmacokinetics of rifampicin in adult TB patients and healthy volunteers: a systematic review and meta-analysis. J Antimicrob Chemother 2018; 73:2305–13.  [DOI](https://doi.org/10.1093/jac/dky152) | [PMC free article](/articles/PMC6105874/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29701775/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Antimicrob%20Chemother&title=Pharmacokinetics%20of%20rifampicin%20in%20adult%20TB%20patients%20and%20healthy%20volunteers:%20a%20systematic%20review%20and%20meta-analysis&volume=73&publication_year=2018&pages=2305-13&pmid=29701775&doi=10.1093/jac/dky152&)

9. Weiner M, Peloquin C, Burman W, et al. Effects of tuberculosis, race, and human gene SLCO1B1 polymorphisms on rifampin concentrations. Antimicrob Agents Chemother 2010; 54:4192–200.  [DOI](https://doi.org/10.1128/AAC.00353-10) | [PMC free article](/articles/PMC2944564/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20660695/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Antimicrob%20Agents%20Chemother&title=Effects%20of%20tuberculosis,%20race,%20and%20human%20gene%20SLCO1B1%20polymorphisms%20on%20rifampin%20concentrations&volume=54&publication_year=2010&pages=4192-200&pmid=20660695&doi=10.1128/AAC.00353-10&)

10. Chigutsa E, Visser ME, Swart EC, et al. The SLCO1B1 rs4149032 polymorphism is highly prevalent in South Africans and is associated with reduced rifampin concentrations: dosing implications. Antimicrob Agents Chemother 2011; 55:4122–7.  [DOI](https://doi.org/10.1128/AAC.01833-10) | [PMC free article](/articles/PMC3165308/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21709081/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Antimicrob%20Agents%20Chemother&title=The%20SLCO1B1%20rs4149032%20polymorphism%20is%20highly%20prevalent%20in%20South%20Africans%20and%20is%20associated%20with%20reduced%20rifampin%20concentrations:%20dosing%20implications&volume=55&publication_year=2011&pages=4122-7&pmid=21709081&doi=10.1128/AAC.01833-10&)

11. Gengiah TN, Botha JH, Soowamber D, Naidoo K, Karim SSA. Low rifampicin concentrations in tuberculosis patients with HIV infection. J Infect Dev Ctries 2014; 8:987–93.  [DOI](https://doi.org/10.3855/jidc.4696) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25116663/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Infect%20Dev%20Ctries&title=Low%20rifampicin%20concentrations%20in%20tuberculosis%20patients%20with%20HIV%20infection&volume=8&publication_year=2014&pages=987-93&pmid=25116663&doi=10.3855/jidc.4696&)

12. Ramesh K, Kumar AKH, Kannan T, et al. SLCO1B1 gene polymorphisms do not influence plasma rifampicin concentrations in a South Indian population. Int J Tuberc Lung Dis 2016; 20:1231–5.  [DOI](https://doi.org/10.5588/ijtld.15.1007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27510251/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Tuberc%20Lung%20Dis&title=SLCO1B1%20gene%20polymorphisms%20do%20not%20influence%20plasma%20rifampicin%20concentrations%20in%20a%20South%20Indian%20population&volume=20&publication_year=2016&pages=1231-5&pmid=27510251&doi=10.5588/ijtld.15.1007&)

13. Hennig S, Naiker S, Reddy T, et al. Effect of SLCO1B1 polymorphisms on rifabutin pharmacokinetics in African HIV-infected patients with tuberculosis. Antimicrob Agents Chemother 2015; 60:617–20.  [DOI](https://doi.org/10.1128/AAC.01195-15) | [PMC free article](/articles/PMC4704238/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26482301/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Antimicrob%20Agents%20Chemother&title=Effect%20of%20SLCO1B1%20polymorphisms%20on%20rifabutin%20pharmacokinetics%20in%20African%20HIV-infected%20patients%20with%20tuberculosis&volume=60&publication_year=2015&pages=617-20&pmid=26482301&doi=10.1128/AAC.01195-15&)

14. Sloan DJ, McCallum AD, Schipani A, et al. Genetic determinants of the pharmacokinetic variability of rifampin in Malawian adults with pulmonary tuberculosis. Antimicrob Agents Chemother 2017; 61:e00210-7.  [DOI](https://doi.org/10.1128/AAC.00210-17) | [PMC free article](/articles/PMC5487625/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28461315/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Antimicrob%20Agents%20Chemother&title=Genetic%20determinants%20of%20the%20pharmacokinetic%20variability%20of%20rifampin%20in%20Malawian%20adults%20with%20pulmonary%20tuberculosis&volume=61&publication_year=2017&pages=e00210-7&pmid=28461315&doi=10.1128/AAC.00210-17&)

15. Calcagno A, Cusato J, Sekaggya-Wiltshire C, et al. The influence of pharmacogenetic variants in HIV/tuberculosis coinfected patients in Uganda in the SOUTH study. Clin Pharmacol Ther 2019; 106:450–7.  [DOI](https://doi.org/10.1002/cpt.1403) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30779340/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=The%20influence%20of%20pharmacogenetic%20variants%20in%20HIV/tuberculosis%20coinfected%20patients%20in%20Uganda%20in%20the%20SOUTH%20study&volume=106&publication_year=2019&pages=450-7&pmid=30779340&doi=10.1002/cpt.1403&)

16. Mukonzo JK, Kengo A, Kutesa B, et al. Role of pharmacogenetics in rifampicin pharmacokinetics and the potential effect on TB-rifampicin sensitivity among Ugandan patients. Trans R Soc Trop Med Hyg 2020; 114:107–14.  [DOI](https://doi.org/10.1093/trstmh/trz108) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31789383/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Trans%20R%20Soc%20Trop%20Med%20Hyg&title=Role%20of%20pharmacogenetics%20in%20rifampicin%20pharmacokinetics%20and%20the%20potential%20effect%20on%20TB-rifampicin%20sensitivity%20among%20Ugandan%20patients&volume=114&publication_year=2020&pages=107-14&pmid=31789383&doi=10.1093/trstmh/trz108&)

17. Francis J, Zvada SP, Denti P, et al. A population pharmacokinetic analysis shows that arylacetamide deacetylase (AADAC) gene polymorphism and HIV infection affect the exposure of rifapentine. Antimicrob Agents Chemother 2019; 63:e01964-18.  [DOI](https://doi.org/10.1128/AAC.01964-18) | [PMC free article](/articles/PMC6437540/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30670438/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Antimicrob%20Agents%20Chemother&title=A%20population%20pharmacokinetic%20analysis%20shows%20that%20arylacetamide%20deacetylase%20(AADAC)%20gene%20polymorphism%20and%20HIV%20infection%20affect%20the%20exposure%20of%20rifapentine&volume=63&publication_year=2019&pages=e01964-18&pmid=30670438&doi=10.1128/AAC.01964-18&)

18. Weiner M, Gelfond J, Johnson-Pais TL, et al. Pharmacokinetics/pharmacodynamics group of tuberculosis trials consortium. Decreased plasma rifapentine concentrations associated with AADAC single nucleotide polymorphism in adults with tuberculosis. J Antimicrob Chemother 2021; 76:582–6.  [DOI](https://doi.org/10.1093/jac/dkaa490) | [PMC free article](/articles/PMC7879139/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33374006/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Antimicrob%20Chemother&title=Pharmacokinetics/pharmacodynamics%20group%20of%20tuberculosis%20trials%20consortium.%20Decreased%20plasma%20rifapentine%20concentrations%20associated%20with%20AADAC%20single%20nucleotide%20polymorphism%20in%20adults%20with%20tuberculosis&volume=76&publication_year=2021&pages=582-6&pmid=33374006&doi=10.1093/jac/dkaa490&)

19. Verma R, da Silva KE, Rockwood N, et al. A nanopore sequencing-based pharmacogenomic panel to personalize tuberculosis drug dosing. Am J Crit Care Med 2024; 209:1486–96.  [DOI](https://doi.org/10.1164/rccm.202309-1583OC) | [PMC free article](/articles/PMC11208962/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/38647526/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Crit%20Care%20Med&title=A%20nanopore%20sequencing-based%20pharmacogenomic%20panel%20to%20personalize%20tuberculosis%20drug%20dosing&volume=209&publication_year=2024&pages=1486-96&pmid=38647526&doi=10.1164/rccm.202309-1583OC&)

20. Lacroix C, Hoang TP, Nouveau J, et al. Pharmacokinetics of pyrazinamide and its metabolites in healthy subjects. Eur J Clin Pharmacol 1989; 36:395–400.  [DOI](https://doi.org/10.1007/BF00558302) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2737233/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Clin%20Pharmacol&title=Pharmacokinetics%20of%20pyrazinamide%20and%20its%20metabolites%20in%20healthy%20subjects&volume=36&publication_year=1989&pages=395-400&pmid=2737233&doi=10.1007/BF00558302&)

21. Velásquez GE, Brooks MB, Coit JM, et al. Efficacy and safety of high-dose rifampin in pulmonary tuberculosis. A randomized controlled trial. Am J Respir Crit Care Med 2018; 198:657–66.  [DOI](https://doi.org/10.1164/rccm.201712-2524OC) | [PMC free article](/articles/PMC6118011/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29954183/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Respir%20Crit%20Care%20Med&title=Efficacy%20and%20safety%20of%20high-dose%20rifampin%20in%20pulmonary%20tuberculosis.%20A%20randomized%20controlled%20trial&volume=198&publication_year=2018&pages=657-66&pmid=29954183&doi=10.1164/rccm.201712-2524OC&)

22. Peloquin CA, Velásquez GE, Lecca L, et al. Pharmacokinetic evidence from the HIRIF trial to support increased doses of rifampin for tuberculosis. Antimicrob Agents Chemother 2017; 61:e00038-17.  [DOI](https://doi.org/10.1128/AAC.00038-17) | [PMC free article](/articles/PMC5527578/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28559269/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Antimicrob%20Agents%20Chemother&title=Pharmacokinetic%20evidence%20from%20the%20HIRIF%20trial%20to%20support%20increased%20doses%20of%20rifampin%20for%20tuberculosis&volume=61&publication_year=2017&pages=e00038-17&pmid=28559269&doi=10.1128/AAC.00038-17&)

23. Verhagen LM, Coenen MJ, López D, et al. Full-gene sequencing analysis of NAT2 and its relationship with isoniazid pharmacokinetics in Venezuelan children with tuberculosis. Pharmacogenomics 2014; 15:285–96.  [DOI](https://doi.org/10.2217/pgs.13.230) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24533708/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Full-gene%20sequencing%20analysis%20of%20NAT2%20and%20its%20relationship%20with%20isoniazid%20pharmacokinetics%20in%20Venezuelan%20children%20with%20tuberculosis&volume=15&publication_year=2014&pages=285-96&pmid=24533708&doi=10.2217/pgs.13.230&)

24. Bisso-Machado R, Ramallo V, Paixão-Côrtes VR, et al. NAT2 gene diversity and its evolutionary trajectory in the Americas. Pharmacogenomics J 2016; 16:559–65.  [DOI](https://doi.org/10.1038/tpj.2015.72) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26503810/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=NAT2%20gene%20diversity%20and%20its%20evolutionary%20trajectory%20in%20the%20Americas&volume=16&publication_year=2016&pages=559-65&pmid=26503810&doi=10.1038/tpj.2015.72&)

25. Levano KS, Jaramillo-Valverde L, Tarazona DD, et al. Allelic and genotypic frequencies of NAT2, CYP2E1, and AADAC genes in a cohort of Peruvian tuberculosis patients. Mol Genet Genomic Med 2021; 9:e1764.  [DOI](https://doi.org/10.1002/mgg3.1764) | [PMC free article](/articles/PMC8580101/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34510815/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Genet%20Genomic%20Med&title=Allelic%20and%20genotypic%20frequencies%20of%20NAT2,%20CYP2E1,%20and%20AADAC%20genes%20in%20a%20cohort%20of%20Peruvian%20tuberculosis%20patients&volume=9&publication_year=2021&pages=e1764&pmid=34510815&doi=10.1002/mgg3.1764&)

26. Garcia-Closas M, Hein DW, Silverman D, et al. A single nucleotide polymorphism tags variation in the arylamine N-acetyltransferase 2 phenotype in populations of European background. Pharmacogenet Genomics 2011; 21:231–6.  [DOI](https://doi.org/10.1097/FPC.0b013e32833e1b54) | [PMC free article](/articles/PMC3003749/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20739907/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=A%20single%20nucleotide%20polymorphism%20tags%20variation%20in%20the%20arylamine%20N-acetyltransferase%202%20phenotype%20in%20populations%20of%20European%20background&volume=21&publication_year=2011&pages=231-6&pmid=20739907&doi=10.1097/FPC.0b013e32833e1b54&)

27. Chamorro JG, Castagnino JP, Musella RM, et al. tagSNP rs1495741 as a useful molecular marker to predict antituberculosis drug-induced hepatotoxicity. Pharmacogenet Genomics 2016; 26:357–61.  [DOI](https://doi.org/10.1097/FPC.0000000000000224) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27104815/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=tagSNP%20rs1495741%20as%20a%20useful%20molecular%20marker%20to%20predict%20antituberculosis%20drug-induced%20hepatotoxicity&volume=26&publication_year=2016&pages=357-61&pmid=27104815&doi=10.1097/FPC.0000000000000224&)

28. Richardson M, Kirkham J, Dwan K, Sloan DJ, Davies G, Jorgensen AL. NAT2 variants and toxicity related to anti-tuberculosis agents: a systematic review and meta-analysis. Int J Tuberc Lung Dis 2019; 23:293–305.  [DOI](https://doi.org/10.5588/ijtld.18.0324) | [PMC free article](/articles/PMC6421944/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30871660/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Tuberc%20Lung%20Dis&title=NAT2%20variants%20and%20toxicity%20related%20to%20anti-tuberculosis%20agents:%20a%20systematic%20review%20and%20meta-analysis&volume=23&publication_year=2019&pages=293-305&pmid=30871660&doi=10.5588/ijtld.18.0324&)

29. Verma R, Patil S, Zhang N, et al. A rapid pharmacogenomic assay to detect NAT2 polymorphisms and guide isoniazid dosing for tuberculosis treatment. Am J Crit Care Med 2021; 204:1317–26.  [DOI](https://doi.org/10.1164/rccm.202103-0564OC) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34375564/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Crit%20Care%20Med&title=A%20rapid%20pharmacogenomic%20assay%20to%20detect%20NAT2%20polymorphisms%20and%20guide%20isoniazid%20dosing%20for%20tuberculosis%20treatment&volume=204&publication_year=2021&pages=1317-26&pmid=34375564&doi=10.1164/rccm.202103-0564OC&)

30. Litjens CHC, van den Heuvel JJMW, Russel FGM, Aarnoutse RE, Te Brake LHM, Koenderink JB. Rifampicin transport by OATP1B1 variants. Antimicrob Agents Chemother 2020; 64:e00955-20.  [DOI](https://doi.org/10.1128/AAC.00955-20) | [PMC free article](/articles/PMC7508598/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32690641/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Antimicrob%20Agents%20Chemother&title=Rifampicin%20transport%20by%20OATP1B1%20variants&volume=64&publication_year=2020&pages=e00955-20&pmid=32690641&doi=10.1128/AAC.00955-20&)

31. Vavricka SR, Van Montfoort J, Ha HR, Meier PJ, Fattinger K. Interactions of rifamycin SV and rifampicin with organic anion uptake systems of human liver. Hepatology 2002; 36:164–72.  [DOI](https://doi.org/10.1053/jhep.2002.34133) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12085361/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hepatology&title=Interactions%20of%20rifamycin%20SV%20and%20rifampicin%20with%20organic%20anion%20uptake%20systems%20of%20human%20liver&volume=36&publication_year=2002&pages=164-72&pmid=12085361&doi=10.1053/jhep.2002.34133&)

32. Kiander W, Vellonen K, Malinen MM, et al. The effect of single nucleotide variations in the transmembrane domain of OATP1B1 on in vitro functionality. Pharm Res 2021; 38:1663–75.  [DOI](https://doi.org/10.1007/s11095-021-03107-8) | [PMC free article](/articles/PMC8602229/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34647232/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharm%20Res&title=The%20effect%20of%20single%20nucleotide%20variations%20in%20the%20transmembrane%20domain%20of%20OATP1B1%20on%20in%20vitro%20functionality&volume=38&publication_year=2021&pages=1663-75&pmid=34647232&doi=10.1007/s11095-021-03107-8&)

33. Jada SR, Xiaochen S, Yan LY, et al. Pharmacogenetics of SLCO1B1: haplotypes, htSNPs and hepatic expression in three distinct Asian populations. Eur J Clin Pharm 2007; 63:555–63.  [DOI](https://doi.org/10.1007/s00228-007-0285-5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17415554/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Clin%20Pharm&title=Pharmacogenetics%20of%20SLCO1B1:%20haplotypes,%20htSNPs%20and%20hepatic%20expression%20in%20three%20distinct%20Asian%20populations&volume=63&publication_year=2007&pages=555-63&pmid=17415554&doi=10.1007/s00228-007-0285-5&)

34. Nakajima A, Fukami T, Kobayashi Y, Watanabe A, Nakajima M, Yokoi T. Human arylacetamide deacetylase is responsible for deacetylation of rifamycins: rifampicin, rifabutin, and rifapentine. Biochem Pharmacol 2011; 82:1747–56.  [DOI](https://doi.org/10.1016/j.bcp.2011.08.003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21856291/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20Pharmacol&title=Human%20arylacetamide%20deacetylase%20is%20responsible%20for%20deacetylation%20of%20rifamycins:%20rifampicin,%20rifabutin,%20and%20rifapentine&volume=82&publication_year=2011&pages=1747-56&pmid=21856291&doi=10.1016/j.bcp.2011.08.003&)
