# Modelling the influence of MDR1 polymorphism on digoxin pharmacokinetic parameters

## Metadata
**Authors:** Emmanuelle Comets, Céline Verstuyft, Marc Lavielle, Patrice Jaillon, Laurent Becquemont, France Mentré
**Journal:** European Journal of Clinical Pharmacology
**Date:** 2007 Mar 13
**DOI:** [10.1007/s00228-007-0269-5](https://doi.org/10.1007/s00228-007-0269-5)
**PMID:** 17404720
**PMCID:** PMC1963422
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1963422/

## Abstract

**Objectives:** 
Digoxin is a well-known probe for the activity of P-glycoprotein. The objective of this work was to apply different methods for covariate selection in non-linear mixed effect models to study the relationship between the pharmacokinetic parameters of digoxin and the genotype for two major exons located on the MDR-1 gene coding for P-glycoprotein.

**Methods:** 
Thirty-two healthy volunteers were recruited in three pharmacokinetic drug interaction studies. The data after a single oral administration of digoxin alone were pooled. All subjects were genotyped for the MDR1 C3435T and G2677T/A genotypes. The concentration-time profile of digoxin was established using 12 to 16 blood samples taken 15 minutes to 72 hours after administration.

We modelled the pharmacokinetics of digoxin using non-linear mixed effect models. Parameter estimation was performed using the stochastic approximation EM method (SAEM). We used three methods to select the covariate model: selection from a full model using Wald tests, forward inclusion using the log-likelihood ratio test and model selection using the Bayesian Information Criteria.

**Results:** 
The three covariate inclusion methods led to the same final model. Carriers of two T alleles for the C3435T polymorphism in exon 26 of MDR1 had a lower apparent volume of distribution than carriers of a C allele. The only other covariate effect was a shorter absorption time-lag in women.

**Conclusion:** 
The apparent volume of distribution of digoxin is lower in TT subjects, probably reflecting differences in bioavailability. Non-linear mixed effect models can be useful to detect the influence of covariates on pharmacokinetic parameters.

Keywords: Adult; Alleles; Bayes Theorem; Biological Availability; Digoxin; pharmacokinetics; Female; Genotype; Humans; Male; Models, Statistical; Nonlinear Dynamics; P-Glycoprotein; genetics; metabolism; Pharmacogenetics; Polymorphism, Genetic; Sex Factors; Stochastic Processes; Tissue Distribution

Keywords: digoxin, population pharmacokinetics, pharmacogenetics, P-glycoprotein

### Objectives

Digoxin is a well-known probe for the activity of P-glycoprotein. The objective of this work was to apply different methods for covariate selection in non-linear mixed effect models to study the relationship between the pharmacokinetic parameters of digoxin and the genotype for two major exons located on the MDR-1 gene coding for P-glycoprotein.

### Methods

Thirty-two healthy volunteers were recruited in three pharmacokinetic drug interaction studies. The data after a single oral administration of digoxin alone were pooled. All subjects were genotyped for the MDR1 C3435T and G2677T/A genotypes. The concentration-time profile of digoxin was established using 12 to 16 blood samples taken 15 minutes to 72 hours after administration.

We modelled the pharmacokinetics of digoxin using non-linear mixed effect models. Parameter estimation was performed using the stochastic approximation EM method (SAEM). We used three methods to select the covariate model: selection from a full model using Wald tests, forward inclusion using the log-likelihood ratio test and model selection using the Bayesian Information Criteria.

### Results

The three covariate inclusion methods led to the same final model. Carriers of two T alleles for the C3435T polymorphism in exon 26 of MDR1 had a lower apparent volume of distribution than carriers of a C allele. The only other covariate effect was a shorter absorption time-lag in women.

### Conclusion

The apparent volume of distribution of digoxin is lower in TT subjects, probably reflecting differences in bioavailability. Non-linear mixed effect models can be useful to detect the influence of covariates on pharmacokinetic parameters.

## Introduction

Pharmacogenetics is a recent field of research investigating the variability in drug effect due to genetic factors. Genetic variation occurs at many levels: drug absorption, distribution and metabolism, receptors for drug action, and drug elimination. Single-Nucleotide Polymorphisms (SNPs) have been identified which induce modifications of the pharmacokinetics (drug course through the body) or pharmacodynamics (drug efficacy and safety). SNPs have namely been shown to modify bioavailability [[1](#R1), [4](#R4)], decrease excretion [[25](#R25)] sometimes inducing severe toxicity [[10](#R10)] and have been linked to drug efficacy [[6](#R6), [13](#R13)]. Thus, pharmacogenetics are the next step to provide individualised treatments.

The studies including pharmacogenetic data have become more numerous over the last few years. In an overwhelming majority of these studies, non-compartmental analysis (NCA) is used to compare pharmacokinetic measurements such as AUC or maximum concentration between groups. This technique requires a large number of sampling points for every subject. On the other hand, modelling approaches can take advantage of sparse individual designs and can be used in patients with routine clinical data [[26](#R26)], but these more sophisticated approaches are seldom used. One issue with these approaches is the method used for covariate selection and hypothesis testing, since detecting a gene effect can be thought of as a model selection problem. A wide variety of approaches have been proposed. The mainstream method consists in stepwise selection [[23](#R23), [17](#R17)], possibly following prior screening of relevant candidate covariates. The criterion for model selection is usually the likelihood ratio test, which is widely used to compare nested mixed effect models. Tests assessing the statistical significance of the final parameters in the final model, such as the Wald test, can also be used as a selection criterion [[26](#R26)]. Other criteria can be used in model selection, such as the Akaike (AIC) or the Bayesian Information Criterion (BIC) [[22](#R22)]. Regardless of the method used, the clinical relevance is sometimes also assessed by examining the magnitude of the effects found.

In a previous paper, Verstuyft et al. estimated the AUC of digoxin, a probe for the activity of P-glycoprotein, in healthy volunteers using non-compartmental analysis and showed an increase in subjects carrying the TT genotype for the C3435T polymorphism of MDR-1 [[38](#R38)]. The objective of the present paper was to reanalyse the data in [[38](#R38)] by a modelling approach, using three covariate model selection methods: likelihood ratio tests, backwards selection from a full model using Wald tests, which take into account potential correlations between covariates, and model selection using the BIC, which considers all the potential models. A related problem in covariate selection is that the false positive rate (type I error) of the tests has been shown to increase when the estimation methods rely on linear approximations to the likelihood [[7](#R7), [40](#R40)]. In this work, we therefore use a recent estimation method, the stochastic EM algorithm SAEM [[20](#R20)]. Although the three methods can be applied with other estimation algorithms, SAEM allows estimation of the likelihood without approximation, via stochastic simulation, and has been shown to have better statistical properties than linearised methods [[33](#R33)].

## Materials and methods

### Data

Pharmacokinetic data was collected from 32 healthy volunteers included in three pharmacokinetic interaction studies dealing with oral digoxin [[38](#R38)]. Seven subjects participated in a macrogol-digoxin interaction study [[30](#R30)], 12 in a grapefruit juice-digoxin interaction study [[2](#R2)], and 13 in a dipyridamole-digoxin interaction study [[39](#R39)]. The three studies were performed in accordance with the Declaration of Helsinki and its amendments. Protocols were approved by the Ethics Committee of the Pitié-Salpêtrière Hospital (CCPPRB), Paris, France, and written informed consent was obtained from all subjects. The 3 studies took place in the same clinical unit under the supervision of the same research team.

All subjects received a 0.5 mg oral dose of digoxin with a glass of water after an overnight fast. Pharmacokinetic samples were obtained at times 15, 30, 45 minutes, 1, 1.5, 2, 4, 6, 8, 12, 24 and 48 hours after the dose for two of the studies [[2](#R2), [39](#R39)]. For the last study [[30](#R30)], samples were taken at 15, 30, 45 minutes, 1, 1.5, 2, 2.5, 3, 4, 6, 9, 12, 16, 24, 48 and 72 hours.

The three studies included 23 men and 9 women, with a mean age of 25.8±5.2 years (range 19–35). Patients were genotyped for two MDR1 polymorphisms, C3435T polymorphism in exon 26 and G2677T/A polymorphism in exon 21. In study [[39](#R39)], patients were genotyped prior to inclusion to balance the genotypes for the C3435T polymorphism while in the 2 other studies, genotyping was performed after inclusion. As a result, the genotypes of the 32 patients for this polymorphism included 10 TT (mutant homozygotes, 31%), 8 CT (heterozygotes, 25%) and 14 CC (wild-type homozygotes, 44%). G2677T/A genotyping revealed 12 GG (38%), 11 GT (34%), 7 TT (22%), 1 GA (3%) and 1 AA (3%) subjects, with a linkage disequilibrium between the two polymorphisms (Somer’s D’=0.72).

Digoxin was measured using a modified enzyme multiplied digoxin immunoassay (EMIT 2000, Dade Behring, Calif., USA), with a quantification limit of 0.1 ng/ml. MDR1 C3435T and G2677T/A genotypes were determined by TaqMan allelic discrimination. More details concerning the analytical methods can be found in [[38](#R38), [37](#R37)].

### Statistical methods

#### Pharmacokinetic model

The pharmacokinetics of digoxin were described using a two-compartment model [[15](#R15)] with first-order absorption and elimination, and an absorption time-lag, using the analytical form of the model. We assumed a proportional variance model for the residual error. This model included six parameters: k_a_, k_el_, V_c_/F, T_lag_ and the two transfer rate constants k_1,2_ and k_2,1_. Interindividual variability was estimated for the first four parameters, with no covariance between them (diagonal variance matrix Ω).

Denoting *f* the function describing this model, the statistical model for concentration *yi j* in individual *i* at time *ti j* is:

| yi j=f(θi,ti j)+εi j | (1) |
| --- | --- |
θ*i* denotes the vector of parameters for individual *i* and its components are assumed to follow a log-normal distribution:

| θi=θ0eηi | (2) |
| --- | --- |
where ηi∼N(0,Ω) is the vector of individual random effects.

The residual errors ε*i j* are assumed to be independent, with distribution N(0,σi j2), where the variance of the error is modelled using a proportional error model: σi j2=σ2f(θi,ti j)2.

The model for covariate effect describes the relationship between the individual pharmacokinetic parameters and a given covariate. The effect of polymorphism in exon 26 on a component θ^(^*k*^)^ of the vector of parameters θ was modelled as:

| θi(k)=θ0(k)(1+βCT(k))CT(1+βTT(k))TTeηi | (3) |
| --- | --- |
Thus, the expected value of θi(k) is θ0(k) for subjects with genotype CC, θ0(k)(1+βCT(k)) for subjects with genotype CT and θ0(k)(1+βTT(k)) for subjects with genotype TT. This model was used for the 4 parameters with variability (k_a_, k_el_, V_c_/F, T_lag_). In the following, we will drop the superscript ^(^*k*^)^ for simplicity. For each parameter in the model, there are 5 possible models for the gene-parameter relationship: the full model with three classes as in [equation 3](#FD3) (denoted H_1_ in the following), three intermediate models with two classes that we denote H_0_*a*: {β*CT* = 0}, H_0_*b*: {β*TT* = 0} and H_0_*c*:{β*CT* − β*TT* = 0} and the model with no gene effect H_0_: {β*CT* = β*TT* = 0}. In the following, we first illustrate the three covariate selection approaches using the polymorphism in exon 26, then we apply these methods considering all the available covariates.

#### Backward covariate selection using the Wald test

One approach to select the covariate model is to estimate the parameters of a full model and perform a significance test using the Wald statistics to select which parameters should be kept in the model [[26](#R26)]. The advantage of this method is that model selection is performed in one step, and that interactions between covariates are taken into account in the estimation of the parameters. Given the model described in [equation 3](#FD3), we test if the three parameters β*CT*, β*TT* and (β*CT* − β*TT*) are significantly different from zero by comparing the corresponding Wald statistics to the critical value of a χ^2^ with one degree of freedom.

A screening step is often performed to eliminate candidate covariates which have a very small probability of influencing the parameters, to improve the estimation of the remaining parameters in the model. We choose an arbitrary value of 0.25 as the significance threshold, and we eliminate the covariates for which the p-values of the 3 tests corresponding to the 3 null hypotheses H_0_*a*, H_0_*b* and H_0_*c* are higher than 0.25. This yields a simplified model where some parameters are modelled according to model H_1_ and some parameters are the same regardless of the genotype. This step eliminates relationships that are totally irrelevant from the model and increases the precision of estimation of the other, possibly meaningful, parameters.

In the next stage, we estimate again the parameters and their standard errors using the simplified model. For each parameter modelled using H_1_, the p-values of the three Wald tests are used to select the appropriate relationship, after correction for multiple tests by applying the Simes procedure [[35](#R35), [3](#R3)]. This method allows to control the family-wise error rate for the three simultaneous tests performed. For a given parameter, the final model for the gene-parameter relationship depends on which hypotheses are rejected. For example, if H_0_*a* and H_0_*c* are simultaneously rejected for a parameter, [equation 3](#FD3) simplifies to:

| θi=θ0(1+βCT)CTeηi | (4) |
| --- | --- |
This procedure leads to the final model.

#### Forward covariate selection using the log-likelihood ratio test

Convergence problems and non-identifiability may occurr when trying to estimate the parameters of a full model with many covariates. The alternative is to build the model using forward selection. Different forms of this approach are used in most studies using nonlinear mixed effect models [[23](#R23), [17](#R17)].

For forward selection, we start from a model without covariates (basic model) and compute empirical Bayes estimates (EBE) of the individual parameters. One-way analysis of variance (ANOVA) is used to test for a difference between the three genotypes for each parameter [[23](#R23)]. As previously, we begin by a screening step: candidate relationships are selected as those where the p-value of the ANOVA is less than 0.25. We then model the candidate relationships as in [equation 3](#FD3) one at a time, starting with the most significant according to the LRT. We stop when none of the remaining relationships provide a significant improvement in the model according to a LRT. We then test for all parameter-gene relationships the three submodels H_0_*a*, H_0_*b* and H_0_*c* using the LRT again, correcting the p-values using the Simes procedure. The best model for the corresponding relationship is selected as in the previous strategy, based on the p-values for the three corresponding tests.

#### Covariate selection using the Bayesian Information Criterion

We compared the two previous selection methods with model selection using the Bayes Information criterion (BIC) given by:

| BIC=−2LL+Plog(ntot) | (5) |
| --- | --- |
where P is the number of parameters (fixed and variance) in the model and *ntot* is the total number of observations.

The best model is defined as the model with the lowest BIC. For model selection using the BIC, we also consider models close to the lowest BIC. From the definition of Bayes factor as a ratio of posterior to prior odds used in Bayesian model selection, Raftery shows that the strength of evidence of one model versus the other is limited when models are within 3 points of BIC while a larger difference provides positive evidence [[29](#R29), [18](#R18)].

A practical problem is the number of models to test. For each parameter in the model, there are 5 possible models when considering the genotype for exon 26 alone. To test all possible combinations for the 4 parameters with variability would require generating and fitting 625 models. Although technically feasible here, this would soon become impractical with more covariates or more parameters, therefore we propose a simplified approach. In a first step, for each parameter, we keep the model with the lowest BIC, as well as models within 3 points of BIC to the lowest. The model without covariate (H_0_) is also added to this list of possible models. In a second step, we build combined models where the possible models for one parameter are combined with each of the models for the other parameters. We estimate the corresponding BIC, and the best model s selected as the model having the lowest BIC overall. Again, we also examine models with BIC close to the lowest value.

#### Estimation method

The parameters are estimated using maximum likelihood approaches. Because the regression function is nonlinear with respect to the random effects, the likelihood function has no closed form. The most commonly used estimation methods rely on approximations of the likelihood function through first-order Taylor expansions, and have been implemented for instance in the nlme package in R/Splus [[27](#R27)], and in the NONMEM software [[34](#R34)]. To avoid this approximation, Bayesian approaches have been proposed which integrate the likelihood using Monte-Carlo Markov chains (MCMC) [[36](#R36)]. An alternative approach is to consider random effects as missing data and to use the EM algorithm [[9](#R9)]. An algorithm called SAEM has been recently developed using the EM approach: stochastic approximation combined with MCMC methods to simulate the random effect in the E-step provides a convergent algorithm and consistent estimates of the population parameters [[8](#R8)]. This method has better statistical properties since no linearisation is involved in the computation of the likelihood and hence the statistical tests based on the results have better properties [[20](#R20)]. It has also been recently applied in two applications, the study the pharmacokinetics of saquinavir in HIV patients [[21](#R21)] and the modelling of the viral load decrease to compare two treatments in a clinical trial [[32](#R32)].

The SAEM algorithm is implemented in the MATLAB language in the software MONOLIX, available on the author’s website ([http://www.math.u-psud.fr/~lavielle/monolix/logiciels.html](http://www.math.u-psud.fr/~lavielle/monolix/logiciels.html)). We used version 1.1 of MONOLIX, in a Linux environment (Red Hat 9.0, GNU Fortran compiler), with MATLAB version 7. The analysis of the results was handled using the R statistical and graphical environment [[28](#R28)]. MONOLIX provides an estimate of the parameters (fixed effects and variance of the random effects) as well as an estimate of the estimation error via the Fisher information matrix [[20](#R20)].

The likelihood is computed by an importance sampling procedure [[31](#R31)]. Since a good estimate of the log-likelihood was required to perform likelihood ratio tests, we used the average of five successive estimations of the likelihood to obtain a more stable estimate.

#### Model building

The three strategies described above were applied to the digoxin data considering the exon 26 polymorphism. We then performed the same analysis for exon 21. For the G2677T/A polymorphism in exon 21, 5 different genotypes were found in the population (GG, GT, GA, TT and TA). We performed first an analysis taking them all into account, and second an analysis where we regrouped the mutant alleles, yielding 3 groups (group 1: GG, group 2: GT or GA, group 3: TT or TA). The influence of the polymorphism in exon 21 was analysed first independently from the results of the analysis including exon 26, then including the model developed for exon 26 alone. We also considered the homozygous wild-type diplotype (combined genotype) CC-GG, combining the GG genotype at position 2677 in exon 21 and the CC genotype at position 3435 in exon 26, versus all other diplotypes. The functional haplotype has previously been shown to influence the AUC of digoxin [[16](#R16)]. Other haplotype analyses were not performed since the number of subjects was too small. Finally, full covariate analysis was performed; the following covariates were available in the study in addition to gene effect: gender, age, weight, body mass index and smoking status. Renal function was not evaluated in these subjects.

We examined the following plots to evaluate the goodness of fit of the final model provided by each approach: scatterplots of predictions (population and individual) versus individual observations; population weighted residuals versus predictions and versus independent variable (time); absolute individual weighted residuals versus individual predictions. In addition, model validation was performed using prediction distribution errors [[5](#R5)], which are computed as the quantiles of the observations in the predicted distribution. The predicted distribution for each observation was obtained through 1000 simulations of the data set given the final model. The prediction distribution errors were decorrelated as proposed in [[5](#R5)] to take into account the correlation induced by the multiple observations within one subject. If the model is adequate, the distribution of the prediction distribution errors is expected to follow a uniform distribution over the interval [0–1], and we used a Kolmogorov-Smirnov test to test this assumption.

## Results

### Backward covariate selection using the Wald test

A full model including the effect of exon 26 genotype on all parameters was fit. The volume of distribution was the only parameter for which at least one of the p-value of the Wald tests for the gene effect was lower than 0.25. The results are shown in [figure 1](#F1): for each parameter, we show the estimate of β_CT_, β_TT_ and the difference β_CT_ − β_TT_ as well as the corresponding confidence interval. The horizontal line represents the expected value of 0 in the absence of effect. As seen from this figure, only β_TT_ and β_CT_ − β_TT_ for parameter V_c_/F were found to be significantly different from zero using Wald tests.

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2707/1963422/aa276bcfdfca/halms146888f1.jpg)

Estimates of the genetic fixed effect for the different parameters in the model.

The model was then re-run with only V_c_/F, yielding the following estimates for the gene effects: β_CT_ = 0.065 (NS), β_TT_ = −0.164 (p<0.01), β_CT_ − β_TT_ = 0.229 (p<0.02). A final model was therefore run, including only a different V_c_/F for TT subjects.

### Forward covariate selection using the log-likelihood ratio test

[Figure 2](#F2) displays the empirical Bayes estimates of the four parameters with intraindividual variability (k_a_, k_el_, V_c_/F, and T_lag_), separated according to the genotype for exon 26. As with the Wald test, only V_c_/F was found to have a significant relationship with the MDR-1 polymorphism on exon 26 (p<0.017 according to the ANOVA), the three other tests yielding p-values larger than 0.4. Including the full gene effect in the model for V_c_/F led to an improvement in the model (p=0.007 according to a LRT, df=2).

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2707/1963422/6faa27813b9f/halms146888f2.jpg)

Empirical Bayes estimates of ka, kel, Vc/F and Tlag with the model without covariate.

In the next and final stage, we then tested the three submodels versus H_1_ using LRT yielding the following p-values: p=0.003 for H_0_*a* ={β*CT* = 0}, p=0.29 for H_0_*b* ={β*TT* = 0}, and p=0.049 for H_0_*c* =(β*CT* − β*TT* = 0}. Using the Simes procedure, the final model selected was the model where TT subjects have different V_c_/F from the two other groups. For the effect of exon 26 polymorphism, the model selected by this strategy was therefore the same as for the selection based on Wald tests.

### Covariate selection using the Bayesian Information Criterion

The selection for each parameter separately yielded the following results: for V_c_/F, the best model was a model with different population mean for TT subjects; for the other parameters, the best model was a model without covariates and there was no model within 3 points of BIC of the lowest model. The results are illustrated in [figure 3](#F3), which shows the BIC of the five models tested for each parameter. For each parameter, the model with the lowest BIC is shown as a full circle.

### Figure 3.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2707/1963422/012a505a59a6/halms146888f3.jpg)

BIC for the five models tested for each parameter: basic model (no gene effect), submodels with βCT = βTT, βCC = βTT or βCC = βCT, full model with βCC and βTT

The models were then combined, and again, the best model overall was here the model with different population mean for V_c_/F in TT subjects.

### Final model

For the analysis of exon 26 alone, the three methods led to the same final model, a model where the carriers of the TT genotype have a different population mean for V_c_/F.

The same analyses were done considering the genotype for exon 21. We found no significant parameter-genotype relationship when considering the five genotype group for exon 21 but some genotypes were present in few subjects, suggesting a lack of power. When regrouping the subjects in three groups according to the number of mutant alleles, the estimate of the volume of distribution was slightly lower in group 3 (TT or TA) versus the other groups (p<0.04). However, when the model for exon 26 was taken into account, this relationship disappeared, showing that the difference is accounted for by exon 26 because of the strong linkage between the two exons (Somers D’=0.72). The three approaches presented above therefore gave the same results for the selection of genetic covariates in the model.

In addition, 11 subjects carried the CC-GG diplotype, and a slight difference was found between the estimates of the volume of distribution for these patients when using a Wald test (p=0.045). However, the two other methods (BIC and EBE) did not pick this difference up.

In the final step, we then added the other covariates to the model. Because of the strong linkage disequilibrium between the two exons, the full covariate model included only exon 26. Using the Wald test approach, the final model included different population mean for V_c_/F in TT subjects, as previously, as well as smaller absorption time-lag in women. Using the LRT approach, a small increase in T_lag_ (2.5%) was also found for smoking patients, but the size of the effect was not clinically significant and thus the final model was the same as with the Wald approach. The BIC approach was not implemented for the full covariate selection because of time constraints.

The parameter estimates and estimates of the standard errors are given in [table I](#T1). The parameters were all well estimated, with standard errors lower than 20% except for the two covariate effects, for which it was less than 40%. The residual (intraindividual) error was also small (17%). In this model, subjects with the TT genotype have a volume of distribution lower by 17% relative to carriers of at least a C allele, and women have a 54% shorter absorption time relative to men. The within-subject variability was largest for the absorption rate constant k*a*.

### Table I.

| Parameter | Population mean (SE as %) | Variability as % (SE on ω2 as %) |
| --- | --- | --- |
| ka (hr−1) | 3.15 (20) | 85 (32) |
| kel (L.hr−1) | 0.09 (6) | 17 (63) |
| Vc/F (L) | 172.70 (5) | 15 (30) |
| βV,TT(−) | −0.17 (31) | - |
| Tlag (hr) | 0.21 (8) | 28 (42) |
| βTlag,women (−) | −0.43 (27) | - |
| k1,2 (hr−1) | 0.32 (7) | - |
| K2,1 (hr−1) | 0.10 (7) | - |
| σ(%) | 0.18 (4) | - |

Table Caption: Estimates of the population pharmacokinetic parameters for the final model.

A plot of the concentrations of digoxin as a function of time for the three genotypes for exon 26 is shown in [figure 4](#F4). Overlayed is the corresponding population predictions for the group. Diagnostic graphs for this model are shown in [figure 5](#F5). The two upper graphs show respectively the population (left) and individual (right) predictions versus observed concentrations. The two bottom graphs show the individual predictions for the first two subjects in the dataset. The graphs show a satisfactory fit, and the absorption phase is well described. A slight underestimation can be seen around 24 hours, as the model does not capture a small rebound at that time. We tested two alternative models, one with a double absorption phase and one assuming enterohepatic recycling, but both encountered numerical difficulties and unphysiological estimates, and the bias in the model was not improved. Therefore, the two-compartment model was kept. We performed model validation for the final model; using prediction discrepancies, we did not reject the hypothesis that the data observed could have been obtained under the model (NS, p=0.49) and considered the model to be adequately qualified.

### Figure 4.

![Figure 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2707/1963422/5e6a407ab2c8/halms146888f4.jpg)

Concentration versus time data for digoxin, for the three genotype classes for exon 26 polymorphism (in log-scale). Overlayed is the line corresponding to the predictions using the population parameters in each group, for men.

### Figure 5.

![Figure 5](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2707/1963422/faf363e340ca/halms146888f5.jpg)

Goodness of fit plots for the final model. Top: population predicted concentrations versus observed concentrations (left); individual predicted concentrations versus observed concentrations (right). Bottom: predicted concentrations (line) overlayed on observed concentrations (dots) for the first man (left) and the first woman (right) in the dataset.

## Discussion

With the recent availability of cheaper genotyping methods, it is now possible to collect genetic information related to drug transporters, metabolic complexes or receptor structure on a routine basis in clinical trials or before a patient is given a new treatment. In pharmacokinetics (PK) and pharmacodynamics (PD), the time course of drug concentrations or effects is described using models with a small number of parameters, and pharmacogenetic data is being increasingly used to characterise their variability. There are now reports of pharmacogenetics studies for a large variety of drug classes, confirming the widespread interest and potential applications of pharmacogenetics.

The statistical analysis in these studies however is usually limited to using non-compartmental approaches to study the influence of genotype on AUC, apparent clearance or maximum or trough concentration. Only a few papers report the use of more sophisticated methods such as mixed effect models or Bayesian analysis, despite the fact that these approaches can be more informative. They can take advantage of sparse designs, which could be useful to design studies for screening genetic factors or during therapeutic monitoring. Here we present the first pharmacokinetic population model for digoxin including pharmacogenetics.

In this study, we used three different methods to explore the relationships between the pharmacokinetic parameters and the genetic covariates: forward step wise selection, Wald test-based selection, and criteria-based selection. Although in the present application, they led to the same final model, the three methods all have different characteristics and strengths.

The tests for the three approaches are asymptotic, that is, they assume that the number of subjects as well as the number of points per subject is large enough. All three methods require good estimates of the likelihood, and the Wald test requires in addition good estimates of the standard errors. The only approximation in the computation of the estimated standard errors of estimation involved in SAEM lies in the asymptotic approximation applied to the finite dataset [[20](#R20)], so that we expect better statistical properties of the tests based on estimates obtained by SAEM relative to more traditional methods based on first-order linearisation such as are implemented in NONMEM [[34](#R34)] or in the library nlme for R [[28](#R28)]. Indeed, the standard errors of estimation of the parameters estimated using the SAEM software have been shown to be accurately predicted [[33](#R33)].

Genetic covariates (genotypes or haplotypes) are usually modelled as categorical covariates, except for some genes such as CYP2D6 where a numeric variable representing the number of mutant alleles has been used as the genetic covariate [[19](#R19)]. Categorical covariates bring specific challenges. We need to estimate one parameter for each possible genotype and the number of possible covariate models increases exponentially with the number of genotypes. Also, the dataset is often unbalanced, with sometimes a very small number of patients for the rarer genotypes, which can generate problems for parameter estimation.

Given these specific challenges, being able to select the covariate model in one step with the Wald test is appealing, and has been proposed by Panhard [[26](#R26)]. All the potential relationships are included in the model and a simultaneous estimation of the significance of all the parameters is provided. This approach could be most interesting in sparse data settings where the empirical Bayes estimates (EBE) do not contain as much information as they do in our example where the pharmacokinetic sampling was rich. The three methods described above can be applied regardless of the estimation method, and have been used for instance in NONMEM [[17](#R17)] and nlme [[26](#R26)]. Using the new algorithm SAEM, we can obtain good estimates of the parameters and their estimation error, allowing us to select the covariate model by backwards deletion from a full model. Compared to the two other methods, the Wald test requires an additional assumption, in that the confidence interval for the estimated parameters is assumed to be symmetrical, which makes it less robust than the LRT.

The likelihood ratio test, by contrast, does not require any additional hypothesis beyond that of the asymptotic. Stepwise inclusion is therefore the main method used for covariate model selection in PK/PD models. However, it suffers from a number of known problems: inflation of type I error due to multiple testing during the building process, selection bias, collinear variables and no guarantee that the final model selected using these methods is the correct model [[41](#R41)]. Inflation of the type I error is also inherent to the first-order linearisation of the log-likelihood used by NONMEM [[7](#R7)], while the stochastic approximation of the log-likelihood performed by SAEM retains a type I error closer to the nominal value, as shown in simulation studies [[14](#R14)]. Variants of stepwise methods include building generalised additive models using the empirical Bayes estimates of a model without covariate [[24](#R24)], however they do not address the issues mentionned above. An interesting combination of the LRT approach and the Wald approach could be outlined as follows: first, build a full model with all potential covariates included, and keep as candidate covariates those for which the Wald test is significant; finally, build the covariate model using LRT-based forward or backward selection from these candidate covariates. This would reduce the number of models to test in the selection while allowing for combination of covariates to enter the model.

Finally, the advantage of criterion-based strategies lies in their systematic exploration of all possible models. The use of model selection criteria such as AIC or BIC is more frequent in the Bayesian litterature [[29](#R29)] and has solid theoretical background in information theory. In practice however, AIC often proves anti-conservative and has been shown to be non-consistent [[42](#R42)], and here we use the BIC. Criterion-based strategies have two main drawbacks. The first drawback is that the number of possible models increases exponentially with the number of covariates, although we can simplify the number of possible models by considering prior physiological knowledge to eliminate unlikely parameter-genotype relationships. The second drawback is that there is no formal test of the relative performance of two models. Kass and Raftery propose to use the difference in BIC as a measure of the strength of evidence of one model versus another [[18](#R18)], but one can be left with several competing models of similar strength using that approach.

In summary, despite known problems we recall here, stepwise selection strategies are less computationally cumbersome than criteria-based selection, while more robust to poor estimations of the standard errors than selection based on the Wald test. However, it can be useful to explore candidate relationships using this last method, especially in the presence of a large number of covariates, because as shown here it can provide reliable estimates in one step and because effects due to a combination of several covariates may be missed by stepwise approaches.

The strategies outlined in this work can be used for all types of covariates (demographic data, clinical characteristics, biological measurements…), as well as for building the structural model.

Our main finding, the difference in volume of distribution found for TT subjects, explains the higher AUC observed for these subjects in the previous non-compartmental analysis performed using this data [[38](#R38)]. It can be interpreted as a higher bioavailability in TT subjects relative to CC or CT subjects. This result should be confirmed in patients receiving digoxin, and probably does not warrant dose adjustment for digoxin, especially considering the high variability in absorption. A possible exception would be to adjust dosage in certain populations such as elderly patients or patients receiving other comedications. The proportion of digoxin-treated patients experiencing therapeutic drug monitoring has been shown to increase with the number of PgP inhibitors received [[11](#R11)], which could make it useful to determine the genotype governing PgP activity [[12](#R12)].

In conclusion, we modelled the pharmacokinetics of digoxin including pharmacogenetic data, using nonlinear mixed effect models. Our main finding was that carriers of the TT genotype for the C3435T polymorphism in exon 26 of the MDR-1 gene have lower apparent volume of distribution. Several methods can be used to test for genetic effects. In addition to the usual stepwise selection method, we recommend using the Wald test to screen candidate covariates.

## Acknowledgments

This study complies with the current laws of France, where they were performed, and the protocols were approved by the Ethics Committee of the Pitié-Salpêtrière Hospital (CCPPRB), Paris, France.

## Appendix

The SAEM algorithm is implemented the MATLAB language in the software MONOLIX, available on the author’s website ([http://www.math.u-psud.fr/~lavielle/monolix/logiciels.html](http://www.math.u-psud.fr/~lavielle/monolix/logiciels.html)). We used MONOLIX version 1.1.

The dataset was prepared in R as a two-dimensional array, with columns representing subject ID, time and observed concentrations. A column representing the dose was also added (with the same value at all times and for all subjects). To code for the categorical covariates representing the genotypes of MDR1, we used dummy variables. For example, to code for the exon 26 polymorphism, we defined 3 dummy variables, one with value 1 for the subjects with CC genotype and 0 for the other two genotypes, one with value 1 for the subjects with CT genotype and 0 otherwise, and one with value 1 for the subjects with TT genotype and 0 otherwise. Each dummy variable was entered as an additional column in the dataset. Exemples of datasets used with MONOLIX are included in the Zip file containing the program.

The following code was used to define the pharmacokinetic model (lines beginning with the symbol % are comments), using the explicit analytical equation:

The program MONOLIX is run from within MATLAB. A window opens in which the user specifies the dataset, the model function and the number of covariates to include in the analysis. In our analysis, the variance-covariance matrix was set to diagonal and the variance for parameters k_1,2_ and k_2,1_ was set to 0. The covariate model was also specified via the graphical interface as a linear combination of the dummy covariates defined above.

Version 1.1 of the software requires some tuning of the numerical procedure to ensure convergence of the Markov chain during the stochastic approximation step (see the user manual on the website). We used the following sequence of four stepsizes in the algorithm:

| {a1 =0during K1=500 iterationsa2 =0.5during K2=100 iterationsa3 =0.8during K3=100 iterationsa4 =1during K4=2000 iterations | (6) |
| --- | --- |
The output from MONOLIX consists in a series of graphs as well as a table of parameter estimates with their associated standard errors. Hypothesis testing opens a new window in which the two models compared are specified and the corresponding criteria (AIC, BIC, log-likelihood) are shown after the fit of each model is performed. Empirical Bayes Estimates (EBE) of the individual parameters are obtained as the mean of the posterior distribution and the standard errors on these parameters (the standard deviations of the posterior distribution) are also reported.

## References

1. Anglicheau D, Verstuyft C, Laurent-Puig P, Becquemont L, Schlageter MH, Cassinat B, et al. Association of the Multidrug Resistance-1 gene single-nucleotide polymorphisms with the tacrolimus dose requirements in renal transplant recipients. J Am Soc Nephrol. 2003;14:1889–1896. doi: 10.1097/01.asn.0000073901.94759.36.  [DOI](https://doi.org/10.1097/01.asn.0000073901.94759.36) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12819250/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Am%20Soc%20Nephrol&title=Association%20of%20the%20Multidrug%20Resistance-1%20gene%20single-nucleotide%20polymorphisms%20with%20the%20tacrolimus%20dose%20requirements%20in%20renal%20transplant%20recipients&author=D%20Anglicheau&author=C%20Verstuyft&author=P%20Laurent-Puig&author=L%20Becquemont&author=MH%20Schlageter&volume=14&publication_year=2003&pages=1889-1896&pmid=12819250&doi=10.1097/01.asn.0000073901.94759.36&)

2. Becquemont L, Verstuyft C, Kerb R, Brinkmann U, Lebot M, Jaillon P, et al. Effect of grapefruit juice on digoxin pharmacokinetics in humans. Clin Pharmacol Ther. 2001;70:311–316.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/11673746/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Effect%20of%20grapefruit%20juice%20on%20digoxin%20pharmacokinetics%20in%20humans&author=L%20Becquemont&author=C%20Verstuyft&author=R%20Kerb&author=U%20Brinkmann&author=M%20Lebot&volume=70&publication_year=2001&pages=311-316&pmid=11673746&)

3. Benjamini Y, Hochberg B. Controlling the false discovery rate: a practical and powerful approach to multiple testing. JRSS B. 1995;57:289–300.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JRSS%20B&title=Controlling%20the%20false%20discovery%20rate:%20a%20practical%20and%20powerful%20approach%20to%20multiple%20testing&author=Y%20Benjamini&author=B%20Hochberg&volume=57&publication_year=1995&pages=289-300&)

4. Bonhomme-Faivre L, Devocelle A, Saliba F, Chatled S, Maccario J, Farinotti R, et al. MDR-1 C3435T polymorphism influences cyclosporine a dose requirement in liver-transplant recipients. Transplantation. 2004;78:21–25. doi: 10.1097/01.tp.0000130981.55654.78.  [DOI](https://doi.org/10.1097/01.tp.0000130981.55654.78) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15257034/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Transplantation&title=MDR-1%20C3435T%20polymorphism%20influences%20cyclosporine%20a%20dose%20requirement%20in%20liver-transplant%20recipients&author=L%20Bonhomme-Faivre&author=A%20Devocelle&author=F%20Saliba&author=S%20Chatled&author=J%20Maccario&volume=78&publication_year=2004&pages=21-25&pmid=15257034&doi=10.1097/01.tp.0000130981.55654.78&)

5. Brendel K, Comets E, Laffont C, Laveille C, Mentré F. Metrics for external model evaluation with an application to the population pharmacokinetics of gliclazide. Pharm Res. 2006;23:2036–9. doi: 10.1007/s11095-006-9067-5.  [DOI](https://doi.org/10.1007/s11095-006-9067-5) | [PMC free article](/articles/PMC2124466/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16906454/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharm%20Res&title=Metrics%20for%20external%20model%20evaluation%20with%20an%20application%20to%20the%20population%20pharmacokinetics%20of%20gliclazide&author=K%20Brendel&author=E%20Comets&author=C%20Laffont&author=C%20Laveille&author=F%20Mentr%C3%A9&volume=23&publication_year=2006&pages=2036-9&pmid=16906454&doi=10.1007/s11095-006-9067-5&)

6. Cartron G, Dacheux L, Salles G, Solal-Celigny P, Bardos P, Colombat P, et al. Therapeutic activity of humanized anti-CD20 monoclonal antibody and polymorphism in IgG Fc receptor FcgammaRIIIa gene. Blood. 2002;99:754–758. doi: 10.1182/blood.v99.3.754.  [DOI](https://doi.org/10.1182/blood.v99.3.754) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11806974/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Therapeutic%20activity%20of%20humanized%20anti-CD20%20monoclonal%20antibody%20and%20polymorphism%20in%20IgG%20Fc%20receptor%20FcgammaRIIIa%20gene&author=G%20Cartron&author=L%20Dacheux&author=G%20Salles&author=P%20Solal-Celigny&author=P%20Bardos&volume=99&publication_year=2002&pages=754-758&pmid=11806974&doi=10.1182/blood.v99.3.754&)

7. Comets E, Mentré F. Evaluation of tests based on individual versus population modelling to compare dissolution curves. J Biopharm Stat. 2001;11:107–123. doi: 10.1081/BIP-100107652.  [DOI](https://doi.org/10.1081/BIP-100107652) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11725926/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biopharm%20Stat&title=Evaluation%20of%20tests%20based%20on%20individual%20versus%20population%20modelling%20to%20compare%20dissolution%20curves&author=E%20Comets&author=F%20Mentr%C3%A9&volume=11&publication_year=2001&pages=107-123&pmid=11725926&doi=10.1081/BIP-100107652&)

8. Delyon B, Lavielle M, Moulines E. Convergence of a stochastic approximation version of the EM algorithm. Ann Stat. 1999;27:94–128.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Stat&title=Convergence%20of%20a%20stochastic%20approximation%20version%20of%20the%20EM%20algorithm&author=B%20Delyon&author=M%20Lavielle&author=E%20Moulines&volume=27&publication_year=1999&pages=94-128&)

9. Dempster A, Laird N, Rubin D. Maximum likelihood from incomplete data via the EM algorithm. JRSS B. 1977;39:1–38.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JRSS%20B&title=Maximum%20likelihood%20from%20incomplete%20data%20via%20the%20EM%20algorithm&author=A%20Dempster&author=N%20Laird&author=D%20Rubin&volume=39&publication_year=1977&pages=1-38&)

10. Diasio R, Johnson M. The role of pharmacogenetics and pharmacogenomics in cancer chemotherapy with 5-fluorouracil. Pharmacology. 2000;61:199–203. doi: 10.1159/000028401.  [DOI](https://doi.org/10.1159/000028401) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10971206/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacology&title=The%20role%20of%20pharmacogenetics%20and%20pharmacogenomics%20in%20cancer%20chemotherapy%20with%205-fluorouracil&author=R%20Diasio&author=M%20Johnson&volume=61&publication_year=2000&pages=199-203&pmid=10971206&doi=10.1159/000028401&)

11. Englund G, Hallberg P, Artursson P, Michaelsson K, Melhus H. Association between the number of coadministered P-glycoprotein inhibitors and serum digoxin levels in patients on therapeutic drug monitoring. BMC Med. 2004;2:8. doi: 10.1186/1741-7015-2-8.  [DOI](https://doi.org/10.1186/1741-7015-2-8) | [PMC free article](/articles/PMC411066/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15061868/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Med&title=Association%20between%20the%20number%20of%20coadministered%20P-glycoprotein%20inhibitors%20and%20serum%20digoxin%20levels%20in%20patients%20on%20therapeutic%20drug%20monitoring&author=G%20Englund&author=P%20Hallberg&author=P%20Artursson&author=K%20Michaelsson&author=H%20Melhus&volume=2&publication_year=2004&pages=8&pmid=15061868&doi=10.1186/1741-7015-2-8&)

12. Ensom M, Chang T, Patel P. Pharmacogenetics: the therapeutic drug monitoring of the future? Clin Pharmacokinet. 2001;40:783–802. doi: 10.2165/00003088-200140110-00001.  [DOI](https://doi.org/10.2165/00003088-200140110-00001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11735602/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacokinet&title=Pharmacogenetics:%20the%20therapeutic%20drug%20monitoring%20of%20the%20future?&author=M%20Ensom&author=T%20Chang&author=P%20Patel&volume=40&publication_year=2001&pages=783-802&pmid=11735602&doi=10.2165/00003088-200140110-00001&)

13. Fabris M, Tolusso B, Di Poi E, Assaloni R, Sinigaglia L, Ferraccioli G. Tumor necrosis factor-alpha receptor II polymorphism in patients from southern Europe with mild-moderate and severe rheumatoid arthritis. J Rheumatol. 2002;29:1847–1850.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/12233877/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Rheumatol&title=Tumor%20necrosis%20factor-alpha%20receptor%20II%20polymorphism%20in%20patients%20from%20southern%20Europe%20with%20mild-moderate%20and%20severe%20rheumatoid%20arthritis&author=M%20Fabris&author=B%20Tolusso&author=E%20Di%20Poi&author=R%20Assaloni&author=L%20Sinigaglia&volume=29&publication_year=2002&pages=1847-1850&pmid=12233877&)

14. Girard P, Mentré F. A comparison of estimation methods in nonlinear mixed effects models using a blind analysis. Population Approach Group in Europe (Abstract 384), Pamplona (Spain) 2005 Jun–17;  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Population%20Approach%20Group%20in%20Europe%20(Abstract%20384),%20Pamplona%20(Spain)&title=A%20comparison%20of%20estimation%20methods%20in%20nonlinear%20mixed%20effects%20models%20using%20a%20blind%20analysis&author=P%20Girard&author=F%20Mentr%C3%A9&publication_year=2005&)

15. Hornestam B, Jerling M, Karlsson MO, Held P. Intravenously administered digoxin in patients with acute atrial fibrillation: a population pharmacokinetic/pharmacodynamic analysis based on the Digitalis in Acute Atrial Fibrillation trial. Eur J Clin Pharmacol. 2003;58:747–755. doi: 10.1007/s00228-002-0553-3.  [DOI](https://doi.org/10.1007/s00228-002-0553-3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12634981/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Clin%20Pharmacol&title=Intravenously%20administered%20digoxin%20in%20patients%20with%20acute%20atrial%20fibrillation:%20a%20population%20pharmacokinetic/pharmacodynamic%20analysis%20based%20on%20the%20Digitalis%20in%20Acute%20Atrial%20Fibrillation%20trial&author=B%20Hornestam&author=M%20Jerling&author=MO%20Karlsson&author=P%20Held&volume=58&publication_year=2003&pages=747-755&pmid=12634981&doi=10.1007/s00228-002-0553-3&)

16. Johne A, Kopke K, Gerloff T, Mai I, Rietbrock S, Meisel C, et al. Modulation of steady-state kinetics of digoxin by haplotypes of the P-glycoprotein MDR1 gene. Clin Pharmacol Ther. 2002;72:584–594. doi: 10.1067/mcp.2002.129196.  [DOI](https://doi.org/10.1067/mcp.2002.129196) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12426522/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Modulation%20of%20steady-state%20kinetics%20of%20digoxin%20by%20haplotypes%20of%20the%20P-glycoprotein%20MDR1%20gene&author=A%20Johne&author=K%20Kopke&author=T%20Gerloff&author=I%20Mai&author=S%20Rietbrock&volume=72&publication_year=2002&pages=584-594&pmid=12426522&doi=10.1067/mcp.2002.129196&)

17. Jonsson E, Karlsson M. Automated covariate model building with NONMEM. Pharm Res. 1998;15:1463–1468. doi: 10.1023/a:1011970125687.  [DOI](https://doi.org/10.1023/a:1011970125687) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9755901/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharm%20Res&title=Automated%20covariate%20model%20building%20with%20NONMEM&author=E%20Jonsson&author=M%20Karlsson&volume=15&publication_year=1998&pages=1463-1468&pmid=9755901&doi=10.1023/a:1011970125687&)

18. Kass R, Raftery A. Bayes factors. J Am Stat Assoc. 1995;90:773–795.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Am%20Stat%20Assoc&title=Bayes%20factors&author=R%20Kass&author=A%20Raftery&volume=90&publication_year=1995&pages=773-795&)

19. Kirchheiner J, Heesch C, Bauer S, Meisel C, Seringer A, Goldammer M, et al. Impact of the ultrarapid metabolizer genotype of cytochrome P450 2D6 on metoprolol pharmacokinetics and pharmacodynamics. Clin Pharmacol Ther. 2004;76:302–312. doi: 10.1016/j.clpt.2004.07.002.  [DOI](https://doi.org/10.1016/j.clpt.2004.07.002) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15470329/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Impact%20of%20the%20ultrarapid%20metabolizer%20genotype%20of%20cytochrome%20P450%202D6%20on%20metoprolol%20pharmacokinetics%20and%20pharmacodynamics&author=J%20Kirchheiner&author=C%20Heesch&author=S%20Bauer&author=C%20Meisel&author=A%20Seringer&volume=76&publication_year=2004&pages=302-312&pmid=15470329&doi=10.1016/j.clpt.2004.07.002&)

20. Kuhn E, Lavielle M. Maximum likelihood estimation in nonlinear mixed effects models. Comput Stat Data Analysis. 2005;49:1020–1038.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Comput%20Stat%20Data%20Analysis&title=Maximum%20likelihood%20estimation%20in%20nonlinear%20mixed%20effects%20models&author=E%20Kuhn&author=M%20Lavielle&volume=49&publication_year=2005&pages=1020-1038&)

21. Lavielle M, Mentré F. Estimation of population pharmacokinetic parameters of saquinavir in HIV patients with the MONOLIX software. J Pharmacokinet Pharmacodyn. 2006 doi: 10.1007/s10928-006-9043-z. in press.  [DOI](https://doi.org/10.1007/s10928-006-9043-z) | [PMC free article](/articles/PMC1974848/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17211713/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacokinet%20Pharmacodyn&title=Estimation%20of%20population%20pharmacokinetic%20parameters%20of%20saquinavir%20in%20HIV%20patients%20with%20the%20MONOLIX%20software&author=M%20Lavielle&author=F%20Mentr%C3%A9&publication_year=2006&pmid=17211713&doi=10.1007/s10928-006-9043-z&)

22. Ludden T, Beal S, Sheiner L. Comparison of the Akaike Information Criterion, the Schwarz criterion and the F test as guides to model selection. J Pharmacokinet Biopharm. 1994;22:431–45. doi: 10.1007/BF02353864.  [DOI](https://doi.org/10.1007/BF02353864) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7791040/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacokinet%20Biopharm&title=Comparison%20of%20the%20Akaike%20Information%20Criterion,%20the%20Schwarz%20criterion%20and%20the%20F%20test%20as%20guides%20to%20model%20selection&author=T%20Ludden&author=S%20Beal&author=L%20Sheiner&volume=22&publication_year=1994&pages=431-45&pmid=7791040&doi=10.1007/BF02353864&)

23. Maitre P, Buhrer M, Thomson D, Stanski D. A three-step approach combining Bayesian regression and NONMEM population analysis: application to midazolam. J Pharmacokinet Biopharm. 1991;19:377–378. doi: 10.1007/BF01061662.  [DOI](https://doi.org/10.1007/BF01061662) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1920085/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacokinet%20Biopharm&title=A%20three-step%20approach%20combining%20Bayesian%20regression%20and%20NONMEM%20population%20analysis:%20application%20to%20midazolam&author=P%20Maitre&author=M%20Buhrer&author=D%20Thomson&author=D%20Stanski&volume=19&publication_year=1991&pages=377-378&pmid=1920085&doi=10.1007/BF01061662&)

24. Mandema J, Verotta D, Sheiner L. Building population pharmacokinetic-pharmacodynamic models. I. Models for covariate effects. J Pharmacokin Biopharm. 1992;20:511–528. doi: 10.1007/BF01061469.  [DOI](https://doi.org/10.1007/BF01061469) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1287200/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacokin%20Biopharm&title=Building%20population%20pharmacokinetic-pharmacodynamic%20models.%20I.%20Models%20for%20covariate%20effects&author=J%20Mandema&author=D%20Verotta&author=L%20Sheiner&volume=20&publication_year=1992&pages=511-528&pmid=1287200&doi=10.1007/BF01061469&)

25. McLeod H, Yu J. Cancer pharmacogenomics: SNPs, chips, and the individual patient. Cancer Invest. 2003;21:630–640. doi: 10.1081/cnv-120022384.  [DOI](https://doi.org/10.1081/cnv-120022384) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14533451/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Invest&title=Cancer%20pharmacogenomics:%20SNPs,%20chips,%20and%20the%20individual%20patient&author=H%20McLeod&author=J%20Yu&volume=21&publication_year=2003&pages=630-640&pmid=14533451&doi=10.1081/cnv-120022384&)

26. Panhard X, Mentré F. Evaluation by simulation of tests based on non-linear mixed-effects models in pharmacokinetic interaction and bioequivalence cross-over trials. Stat Med. 2005;10:1509–1524. doi: 10.1002/sim.2047.  [DOI](https://doi.org/10.1002/sim.2047) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15761916/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Stat%20Med&title=Evaluation%20by%20simulation%20of%20tests%20based%20on%20non-linear%20mixed-effects%20models%20in%20pharmacokinetic%20interaction%20and%20bioequivalence%20cross-over%20trials&author=X%20Panhard&author=F%20Mentr%C3%A9&volume=10&publication_year=2005&pages=1509-1524&pmid=15761916&doi=10.1002/sim.2047&)

27. Pinheiro J, Bates D. Mixed-Effects Models in S and S-PLUS. Springer-Verlag; New York: 2000.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Mixed-Effects%20Models%20in%20S%20and%20S-PLUS&author=J%20Pinheiro&author=D%20Bates&publication_year=2000&)

28. R Development Core Team. R: A language and environment for statistical computing. R Foundation for Statistical Computing; Vienna, Austria: 2004.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=R:%20A%20language%20and%20environment%20for%20statistical%20computing&publication_year=2004&)

29. Raftery A. Markov chain Monte Carlo in practice. Chapman & Hall; London: 1996.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Markov%20chain%20Monte%20Carlo%20in%20practice&author=A%20Raftery&publication_year=1996&)

30. Ragueneau I, Poirier J, Radembino N, Sao A, Funck-Brentano C, Jaillon P. Pharmacokinetic and pharmacodynamic drug interactions between digoxin and macrogol. Br J Clin Pharmacol. 1999;48:453–56. doi: 10.1046/j.1365-2125.1999.00025.x.  [DOI](https://doi.org/10.1046/j.1365-2125.1999.00025.x) | [PMC free article](/articles/PMC2014333/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10510161/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=Pharmacokinetic%20and%20pharmacodynamic%20drug%20interactions%20between%20digoxin%20and%20macrogol&author=I%20Ragueneau&author=J%20Poirier&author=N%20Radembino&author=A%20Sao&author=C%20Funck-Brentano&volume=48&publication_year=1999&pages=453-56&pmid=10510161&doi=10.1046/j.1365-2125.1999.00025.x&)

31. Robert C, Casella G. Monte Carlo Statistical Methods. Springer-Verlag; New York: 1999.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Monte%20Carlo%20Statistical%20Methods&author=C%20Robert&author=G%20Casella&publication_year=1999&)

32. Samson A, Lavielle M, Mentré F. Extension of the SAEM algorithm to left-censored data in nonlinear mixed-effects model: Application to HIV dynamics model. Comput Stat Data Analysis. 2006;51:1562–74.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Comput%20Stat%20Data%20Analysis&title=Extension%20of%20the%20SAEM%20algorithm%20to%20left-censored%20data%20in%20nonlinear%20mixed-effects%20model:%20Application%20to%20HIV%20dynamics%20model&author=A%20Samson&author=M%20Lavielle&author=F%20Mentr%C3%A9&volume=51&publication_year=2006&pages=1562-74&)

33. Samson A, Mentré F, Lavielle M. Using SAEM, a new maximum likelihood estimation method in nolinear mixed-effects models, for comparison of longitudinal responses. 5th International Meeting of Statistical Methods in Biopharmacy; Paris (France). Sep–27. 2005.  [Google Scholar](https://scholar.google.com/scholar_lookup?Samson%20A,%20Mentr%C3%A9%20F,%20Lavielle%20M.%20Using%20SAEM,%20a%20new%20maximum%20likelihood%20estimation%20method%20in%20nolinear%20mixed-effects%20models,%20for%20comparison%20of%20longitudinal%20responses.%205th%20International%20Meeting%20of%20Statistical%20Methods%20in%20Biopharmacy;%20Paris%20(France).%20Sep%E2%80%9327.%202005.)

34. Sheiner L, Beal S. NONMEM Version 5.1; University of California, NONMEM Project Group; San Francisco. 1998.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=NONMEM%20Version%205.1&author=L%20Sheiner&author=S%20Beal&publication_year=1998&)

35. Simes R. An improved Bonferroni procedure for multiple tests of significance. Biometrika. 1986;73:751–754.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biometrika&title=An%20improved%20Bonferroni%20procedure%20for%20multiple%20tests%20of%20significance&author=R%20Simes&volume=73&publication_year=1986&pages=751-754&)

36. Spiegelhalter DJ, Thomas A, Best NG. WinBUGS version 1.3 User Manual. Imperial College; London, United Kingdom: 2000.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=WinBUGS%20version%201.3%20User%20Manual&author=DJ%20Spiegelhalter&author=A%20Thomas&author=NG%20Best&publication_year=2000&)

37. Verstuyft C, Morin S, Yang J, Loriot MA, Barbu V, Kerb R, et al. A new, rapid and robust genotyping method for CYP2C9 and MDR1. Ann Biol Clin (Paris) 2003;61:305–309.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/12805007/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Biol%20Clin%20(Paris)&title=A%20new,%20rapid%20and%20robust%20genotyping%20method%20for%20CYP2C9%20and%20MDR1&author=C%20Verstuyft&author=S%20Morin&author=J%20Yang&author=MA%20Loriot&author=V%20Barbu&volume=61&publication_year=2003&pages=305-309&pmid=12805007&)

38. Verstuyft C, Schwab M, Schaeffeler E, Kerb R, Brinkmann U, Jaillon P, et al. Digoxin pharmacokinetics and MDR1 genetic polymorphisms. Eur J Clin Pharmacol. 2003;58:809–812. doi: 10.1007/s00228-003-0567-5.  [DOI](https://doi.org/10.1007/s00228-003-0567-5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12698307/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Clin%20Pharmacol&title=Digoxin%20pharmacokinetics%20and%20MDR1%20genetic%20polymorphisms&author=C%20Verstuyft&author=M%20Schwab&author=E%20Schaeffeler&author=R%20Kerb&author=U%20Brinkmann&volume=58&publication_year=2003&pages=809-812&pmid=12698307&doi=10.1007/s00228-003-0567-5&)

39. Verstuyft C, Strabach S, El Morabet H, Kerb R, Brinkmann U, Dubert L, et al. Dipyridamole enhances digoxin bioavailability via P-glycoprotein inhibition. Clin Pharmacol Ther. 2003;73:51–60. doi: 10.1067/mcp.2003.8.  [DOI](https://doi.org/10.1067/mcp.2003.8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12545143/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Dipyridamole%20enhances%20digoxin%20bioavailability%20via%20P-glycoprotein%20inhibition&author=C%20Verstuyft&author=S%20Strabach&author=H%20El%20Morabet&author=R%20Kerb&author=U%20Brinkmann&volume=73&publication_year=2003&pages=51-60&pmid=12545143&doi=10.1067/mcp.2003.8&)

40. Wählby U, Bouw M, Niclas Jonsson E, Karlsson M. Assessment of type I error rates for the statistical sub-model in NONMEM. J Pharmacokinet Pharmacodyn. 2002;29:251–269. doi: 10.1023/a:1020254823597.  [DOI](https://doi.org/10.1023/a:1020254823597) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12449498/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacokinet%20Pharmacodyn&title=Assessment%20of%20type%20I%20error%20rates%20for%20the%20statistical%20sub-model%20in%20NONMEM&author=U%20W%C3%A4hlby&author=M%20Bouw&author=E%20Niclas%20Jonsson&author=M%20Karlsson&volume=29&publication_year=2002&pages=251-269&pmid=12449498&doi=10.1023/a:1020254823597&)

41. Wählby U, Niclas Jonsson E, Karlsson M. Comparison of stepwise covariate model building strategies in population pharmacokinetic-pharmacodynamic analysis. AAPS PharmSci. 2002;4:E27. doi: 10.1208/ps040427.  [DOI](https://doi.org/10.1208/ps040427) | [PMC free article](/articles/PMC2751316/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12645999/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=AAPS%20PharmSci&title=Comparison%20of%20stepwise%20covariate%20model%20building%20strategies%20in%20population%20pharmacokinetic-pharmacodynamic%20analysis&author=U%20W%C3%A4hlby&author=E%20Niclas%20Jonsson&author=M%20Karlsson&volume=4&publication_year=2002&pages=E27&pmid=12645999&doi=10.1208/ps040427&)

42. Yang Y. Can the strengths of AIC and BIC be shared? A conflict between model identification and regression estimation. Biometrika. 2005;92:937–950.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biometrika&title=Can%20the%20strengths%20of%20AIC%20and%20BIC%20be%20shared?%20A%20conflict%20between%20model%20identification%20and%20regression%20estimation&author=Y%20Yang&volume=92&publication_year=2005&pages=937-950&)
