# Association of DNA methylation, polymorphism and mRNA level of ALAS1 with antituberculosis drug-induced liver injury

## Metadata
**Authors:** Zhuolu Hao, Bing Han, Xinyue Zhou, Hongkai Jian, Xiaomin He, Lihuan Lu, Meiling Zhang, Hongqiu Pan, Honggang Yi, Shaowen Tang
**Journal:** Pharmacogenomics
**Date:** 2024 Sep 12
**DOI:** [10.1080/14622416.2024.2392480](https://doi.org/10.1080/14622416.2024.2392480)
**PMID:** 39263813
**PMCID:** PMC11492648
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11492648/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC11492648/pdf/IPGS_25_2392480.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC11492648/pdf/IPGS_25_2392480.pdf)

## Abstract

Aim: To investigate the association of DNA methylation, genetic polymorphisms and mRNA level of aminolevulinate synthase 1 (ALAS1) with antituberculosis drug-induced liver injury (AT-DILI) risk.

Methods: Based on a 1:1 matched case-control study with 182 cases and 182 controls, one CpG island and three single nucleotide polymorphisms (SNPs) were detected. ALAS1 mRNA level was detected in 34 samples.

Results: Patients with methylation status were at high risk of AT-DILI (odds ratio: 1.567, 95% CI: 1.015–2.421, p = 0.043) and SNP rs352169 was associated with AT-DILI risk (GA vs. GG, odds ratio: 1.770, 95% CI: 1.101–2.847, p = 0.019). ALAS1 mRNA level in the cases was significantly lower than that in the controls (0.75 ± 0.34 vs. 1.00 ± 0.42, p = 0.021).

Conclusion: The methylation status and SNP rs352169 of ALAS1 were associated with AT-DILI risk.

Keywords: : ALAS1, AT-DILI, DNA methylation, SNP, synergistic effect

#### Introduction

#### Introduction

#### Materials & methods

#### Results

#### Discussion

## 1. Introduction

Tuberculosis (TB) is still a major public health issue worldwide. The Global Tuberculosis Report 2023 showed that approximately 10.6 million people developed TB in 2022, an increase of 4.5% from 10.3 million in 2021, with an estimated 1.3 million TB deaths among them [[1](#CIT00001)]. TB patients in China contributes to approximately 7.1% of TB patients in the whole world, with 748,000 new patients and China is one of 30 countries with a high TB burden identified by the WHO [[1](#CIT00001)]. The first-line regimen for treating TB recommended by the WHO was a combination of isoniazid (INH), rifampicin (RIF), pyrazinamide, ethambutol and/or streptomycin for a period of 6–9 months [[2](#CIT00002)]. However, adverse drug reactions (ADRs) are the leading causes of treatment interruption in TB patients. Antituberculosis drug-induced liver injury (AT-DILI) is the most frequent ADR of first-line anti-TB therapy, with an estimated incidence of 5.4% in a large-scale prospective Chinese cohort [[3](#CIT00003)]. Patients with mild AT-DILI show increased transaminases, while patients with severe AT-DILI may develop liver failure. At the same time, patients may face drug withdrawal or replacement with second-line anti-TB drugs, which increases the risk of drug resistance and relapse [[4](#CIT00004)]. Therefore, effectively predicting the occurrence of AT-DILI and reducing ADRs are essential to control and treat TB.

At present, the pathogenesis of AT-DILI has been explored from multiple perspectives, especially regarding the hepatotoxicity due to INH and RIF, but the precise mechanism is still unclear [[5](#CIT00005)]. Recently, pregnane X receptor (PXR)-aminolevulinic synthase-1 (ALAS1) axis activation leading to abnormal metabolism of protoporphyrin IX (PPIX) and hepatic heme accumulation has become recognized as one of the possible mechanisms of AT-DILI [[6](#CIT00006),[7](#CIT00007)]. Animal experiments found that the combination of INH and RIF mediated PXR activation and caused accumulation of PPIX in the liver through the heme biosynthesis pathway [[8](#CIT00008)]. PPIX, an endogenous hepatotoxin, is the final intermediate in heme biosynthesis [[9](#CIT00009)]. ALAS1 is the first rate-limiting enzyme in the heme biosynthesis pathway [[10](#CIT00010)]. RIF has been proven to be a human PXR-specific activator [[11](#CIT00011)]. INH-RIF cotherapy causes PXR-mediated transcriptional activation of ALAS1 and ALAS1 catalyzes the condensation of glycine and succinyl-CoA to 5-aminolevulinate, which further synthesizes PPIX [[6](#CIT00006)]. However, the ALAS1 level is under negative feedback regulation by excess heme, maintaining a low baseline activation state [[12](#CIT00012)]. ALAS1 is upregulated by INH-RIF therapy both by direct transcriptional activation and through derepression of negative feedback due to heme incorporation into the cytochrome P450 apoprotein [[6](#CIT00006)]. This dual mechanism leads to an increase in hepatotoxic PPIX.

Recently, different studies have shown the important role of genetic and epigenetic factors in the pathogenesis of AT-DILI and DNA methylation is a hot spot to explore the epigenetic mechanism of AT-DILI [[13](#CIT00013),[14](#CIT00014)]. Studies have found that the methylation of drug metabolic enzymes (cytochrome P450 1A1 and glutathione S-transferase P1) is associated with the risk of AT-DILI [[15](#CIT00015)]. However, to the best of our knowledge, no study has yet explored the epigenetic mechanism of ALAS1 in the occurrence of AT-DILI. Previous studies have explored the relationship between *ALAS1* gene polymorphisms and AT-DILI, single nucleotide polymorphism (SNP) rs352168, rs104903, rs352169 and rs164640 polymorphisms of *ALAS1* gene were not associated with AT-DILI, but no statistical association was found in sex subgroup [[16](#CIT00016)], the interaction between gene polymorphisms and methylation in AT-DILI has not been analyzed. Therefore, in this study, we aimed to investigate the impact of CpG island DNA methylation in the promoter region and genetic polymorphisms of *ALAS1* on AT-DILI in an eastern Chinese population.

## 2. Methods

### 2.1. Study design & patients

This study was designed as a 1:1 matched case-control study. All anti-TB treatment patients were recruited from four designated infectious hospitals in Jiangsu Province from December 2017 to October 2023. Each patient signed an informed consent form and the study was approved by the Ethics Committee of Nanjing Medical University.

Patients who met the following criteria were included: (1) newly registered pulmonary TB patients; (2) ready to receive standard short-course anti-TB treatment regimens according to the WHO guidelines; and (3) prepared to be subjected to at least three liver function tests during anti-TB treatment. Patients with any of the following were excluded from the present study: (1) mental illness; (2) abnormal liver dysfunction (alanine transaminase [ALT], aspartate transaminase [AST], alkaline phosphatase [ALP] or total bilirubin [TBil]) before anti-TB treatment; (3) basic hepatic diseases, such as viral hepatitis, cirrhosis, liver cancer and nonalcoholic liver disease; (4) liver injury induced by other drugs or from unknown reasons; (5) serious diseases of vital organs; and (6) HIV or syphilis. The patient flow was shown in [Figure 1](#F0001).

### Figure 1.

![Figure 1.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ab29/11492648/a6cad046cda3/IPGS_A_2392480_F0001_B.jpg)

Flow chart of study design. AT-DILI: Antituberculosis drug-induced liver injury; HIV: Human immunodeficiency virus; mRNA: messenger RNA; TB: Tuberculosis.

AT-DILI cases were defined in accordance with the modified diagnostic criteria established by the Chinese Medical Association [[17](#CIT00017)]. Briefly, AT-DILI was defined according to the following criteria: (1) ALT ≥5 upper limit of normal (ULN); (2) ALP ≥2 ULN; (3) ALT ≥3 ULN combined with TBil ≥2 ULN; and (4) causality score ≥3 based on the updated Roussel Uclaf Causality Assessment Method (RUCAM) [[18](#CIT00018)]. Patients who fulfilled the criterion of AT-DILI were assigned to the case group and each case was matched with one normal serum liver function control by age at baseline (±5 years) and sex.

### 2.2. Epidemiological data & blood specimen collection

The treatment and follow-up of TB patients during the treatment period were in accordance with the previously published research protocol (Home-based Anti-Tuberculosis Treatment Adverse Reactions study) [[19](#CIT00019)]. Patient information was collected, which included sex, age, ethnicity, smoking and drinking history, previous medical history and conditions related to anti-TB treatment (such as prophylactic use of hepatoprotective drugs). In addition, ALT, AST, TBil, direct bilirubin (DBil), ALP, γ-glutamyl transferase (GGT), total protein, albumin, globulin, albumin/globulin, glucose, hemoglobin and white blood cell were collected from hospital healthcare systems. The residual blood samples for liver function tests of each patient were preserved at -80°C and were used for genomic DNA extraction. In addition, 2 ml of fasting venous blood was collected to the anticoagulant tube and the pretreatment of blood sample was completed within 8 h.

### 2.3. DNA extraction & bisulfite modification

The column method was used to extract blood genomic DNA (manufacturer: Vazyme, Nanjing, China; model: DC112) and two micrograms of isolated genomic DNA was used for bisulfite treatment according to the manufacturer's protocol of the EZ DNA Methylation-Gold™ Kit (manufacturer: ZYMO Research, CA, USA; model: D5006) [[20](#CIT00020)]. The treated DNA was dissolved in 10 μl of water and stored at -20°C for further experiments. The above operations were carried out in standard laboratories by uniformly trained professionals who strictly followed the operating instructions.

### 2.4. DNA methylation analysis

Bisulfite-treated DNA was subjected to methylation-specific PCR using primers targeting the CpG island in the promoter region of the *ALAS1* gene by ABI Prism 9700 PCR (Applied Biosystems, USA) [[21](#CIT00021)]. The primer sequences were listed in Supplementary Table S1. After an initial denaturation at 95°C for 5 min, 40 cycles of 95°C for 30 s, 56°C for 30 s and 72°C for 30 s and a final extension period of 7 min at 72°C, were performed. The PCR products, after gelred staining, were separated on a 3% agarose gel and visualized under ultraviolet illumination. The samples with bands amplified by the methylated-specific primers were recorded as completely methylated; the samples with bands amplified by unmethylated-specific primers were recorded as completely unmethylated; samples with bands amplified by both methylated-specific and unmethylated-specific primers were recorded as incompletely methylated; and both incomplete and complete methylation belonged to the methylation status. A total of 10% of DNA samples were randomly selected for repeated tests and the concordance rate was 100%.

### 2.5. SNP selection & genotyping

All eligible SNPs in the *ALAS1* gene regions, which included the 2 kb upstream and downstream of *ALAS1* gene, were downloaded from the China Han population of the 1000 Genomes Project database. The SNPs were screened and evaluated by Haploview 4.2 software (Broad Institute of MIT and Harvard, MA, USA) based on the following conditions: the minor allele frequency was ≥10% and the correlation coefficient R^2^ was ≥0.8. Additionally, bioinformatics analysis of potentially functional SNPs using GETx Portal database ([https://gtexportal.org/](https://gtexportal.org/)) and SNP Function Prediction ([https://snpinfo.niehs.nih.gov/](https://snpinfo.niehs.nih.gov/)). Meanwhile, the SNPs reported in previous studies were used for reference [[16](#CIT00016),[22](#CIT00022)]. Finally, three SNPs (rs352163, rs352169 and rs104903) in the *ALAS1* gene were selected for genotyping using TaqMan allelic discrimination technology. Primers and probes were designed and synthesized by company (manufacturer: Ji Ao Biotechnology Company Ltd, Nanjing, China). The primer and probes sequences were listed in Supplementary Table S1. Taqman allelic discrimination technology was conducted using QuantStudio 7 Flex Real-Time PCR System (Thermo, MA, USA). After an initial denaturation at 50°C for 2 min and 95°C for 10 min, 45 cycles of 95°C for 15 s, 60°C for 1 min and a final extension period of 10 min at 15°C, were performed. Case and control samples were randomly tested in the genotyping process to ensure the quality of the experiment. The overall success rate of SNP genotyping was 100%.

### 2.6. RNA extraction & real-time quantitative PCR

Venous blood was collected from January 2023 to October 2023 for mRNA detection. The magnetic bead method was used to extract RNA from peripheral blood mononuclear cells (manufacturer: Yuan, Jiangsu, China; model: Yu-R01) and RNA samples were reverse-transcribed into complementary DNA (cDNA) using a HiScript III RT SuperMix for qPCR kit (manufacturer: Vazyme, Nanjing, China; model: R232). The expression level of *ALAS1* mRNA was measured using a quantitative real-time PCR system with SYBR qPCR Master Mix (manufacturer: Vazyme, Nanjing, China; model: Q712-02) by QuantStudio 7 Flex Real-Time PCR System (Thermo). After an initial denaturation at 95°C for 3 min, 40 cycles of 95°C for 10 s, 60°C for 20 s and 72°C for 10 s and a final extension period of 95°C for 15 s, 60°C for 1 min and 95°C for 15 s, were performed. The relative expression level of *ALAS1* mRNA was calculated using the 2^-ΔΔCt^ formula [[23](#CIT00023)]. The sequences of primers were shown in Supplementary Table S1.

### 2.7. Statistical analysis

R software for Windows version 4.2.1 was used to conduct statistical analysis. Normally distributed data were described as the mean ± standard deviation and analyzed by the independent samples *t*-test, while non-normally distributed data were described as the median with interquartile range and analyzed by the Wilcoxon rank-sum test. Categorized variables were expressed as numbers and percentages and analyzed using a conditional logistic regression model. Deviations from Hardy–Weinberg equilibrium for each SNP among the controls were analyzed with the χ^2^ test. Univariate conditional logistic regression model was used to estimate the association of methylation status and SNPs with AT-DILI as well as the interaction between methylation status and SNPs with AT-DILI, multivariate analysis adjusted the potential confounding factors (smoking, drinking and the use of preventive liver protection drugs) as covariates [[24–26](#CIT00024)]. The corresponding odds ratios (ORs) and their 95% CIs were calculated. Three genetic models (dominant, recessive and additive) were used to comprehensively analyze the effect of SNPs. And the false discovery rate was used for multiple corrections. Haplotype blocks were structured using HaploView 4.2 software considering the linkage disequilibrium between SNPs. A two-tailed *p*-value < 0.05 was considered statistically significant.

## 3. Results

### 3.1. Characteristics of patients

A total of 364 anti-TB treatment patients (182 AT-DILI cases and 182 controls) were included in the study. The AT-DILI cases consisted of 131 males and 51 females with an average age of 53.2 ± 17.2 years. Among 182 AT-DILI cases, 22 cases (12.09%) were judged as possible (3–5 scores), 114 cases (62.64%) were judged as probable (6–8 scores) and 46 cases (25.27%) were judged as highly probable (>8 scores) based on the updated RUCAM. The matched controls contained 131 males and 51 females with a mean age of 52.9 ± 17.4 years. The characteristics and clinical indicators for the cases and controls were shown in [Table 1](#T0001) and there was no significant difference between the cases and controls regarding the baseline demographics (*p* > 0.05). For clinical characteristics, the distribution of TBil, DBil, ALT, AST, GGT, AST/ALT and white blood cell levels at baseline and TBil, DBil, ALT, AST, ALP, GGT, AST/ALT, DBil/TBil, total protein, albumin, albumin/globulin and glucose levels at peak were significantly different between the two groups (*p* < 0.05) ([Table 1](#T0001)).

### Table 1.

| Characteristic | AT-DILI cases (N = 182) | Controls (N = 182) | p-value |
| --- | --- | --- | --- |
| Sex (male/female) | 131/51 | 131/51 | – |
| Age (year)a | 53.2 ± 17.2 | 52.9 ± 17.4 | 0.839c |
| Hepatoprotectant (use/not use) | 109/73 | 94/88 | 0.061d |
| Smoking history (yes/no) | 24/158 | 21/161 | 0.631d |
| Drinking history (yes/no) | 13/169 | 16/166 | 0.578d |
| Liver diseases history (yes/no) | 22/160 | 28/154 | 0.378d |
| Diabetes (yes/no) | 35/147 | 29/153 | 0.345d |
| High blood pressure (yes/no) | 131/51 | 131/51 | 0.839d |
| Baseline value |  |  |  |
| ALT (U/l)b | 15.0 (10.1–21.4) | 12.0 (8.0–16.0) | <0.001e |
| AST (U/l)b | 20.0 (15.8–25.0) | 18.0 (15.0–22.3) | 0.014e |
| TBil (μmol/l)b | 10.6 (8.3–16.0) | 9.0 (6.7–12.3) | <0.001e |
| DBil (μmol/l)b | 3.7 (2.7–5.1) | 3.0 (2.2–4.1) | <0.001e |
| ALP (U/l)b | 80.0 (64.2–97.7) | 77.0 (65.2–93.3) | 0.414e |
| GGT (U/l)b | 29.5 (19.7–48.8) | 23.0 (18.0–32.0) | <0.001e |
| AST/ALTb | 1.4 (1.0–1.8) | 1.6 (1.2–2.0) | 0.002e |
| DBil/TBilb | 0.3 (0.3–0.4) | 0.3 (0.3–0.4) | 0.656e |
| TP (g/l)b | 70.6 (66.5–74.3) | 70.4 (66.2–73.8) | 0.395e |
| ALB (g/l)b | 40.1 (36.6–43.6) | 40.4 (36.9–43.2) | 0.553e |
| GLB (g/l)b | 30.5 (27.2–32.6) | 30.0 (26.6–32.9) | 0.496e |
| A/Gb | 1.3 (1.2–1.5) | 1.4 (1.2–1.5) | 0.587e |
| GLU (mmol/l)b | 5.32 (4.99–5.64) | 5.27 (5.04–5.56) | 0.089e |
| HB (g/l)b | 132.0 (125.0–141.0) | 130.0 (120.0–140.0) | 0.071e |
| WBC (109/l)b | 5.89 (5.09–6.59) | 6.25 (5.03–7.38) | 0.039e |
| During treatment (peak value) |  |  |  |
| ALT (U/l)b | 259.0 (195.0–412.0) | 16.8 (11.7–22.8) | <0.001e |
| AST (U/l)b | 204.0 (126.0–368.0) | 22.0 (18.0–27.7) | <0.001e |
| TBil (μmol/l)b | 16.0 (11.2–25.6) | 8.1 (6.3–10.1) | <0.001e |
| DBil (μmol/l)b | 6.0 (4.0–11.6) | 2.5 (2.1–3.4) | <0.001e |
| ALP (U/l)b | 112.0 (82.2–177.0) | 77.0 (67.2–92.1) | <0.001e |
| GGT (U/l)b | 108.0 (61.2–173.0) | 28.5 (22.0–40.2) | <0.001e |
| AST/ALTb | 0.8 (0.6–1.3) | 1.4 (1.0–1.8) | <0.001e |
| DBil/TBilb | 0.4 (0.3–0.6) | 0.3 (0.3–0.4) | <0.001e |
| TP (g/l)b | 70.6 (66.5–74.3) | 70.4 (66.2–73.8) | 0.004e |
| ALB (g/l)b | 40.8 (37.9–43.5) | 42.8 (41.2–44.8) | <0.001e |
| GLB (g/l)b | 30.4 (26.6–33.0) | 29.5 (27.4–31.0) | 0.094e |
| A/Gb | 1.3 (1.2–1.6) | 1.5 (1.3–1.6) | 0.004e |
| GLU (mmol/l)b | 5.51 (5.23–5.73) | 5.38 (5.34–5.51) | <0.001e |
| HB (g/l)b | 137.0 (127.0–143.0) | 134.0 (126.0–142.0) | 0.283e |
| WBC (109/l)b | 5.4 (4.54–6.54) | 5.57 (4.67–6.58) | 0.197e |

Table 1 Caption: Characteristics of patients among AT-DILI cases and controls.

### 3.2. Association of methylation status with AT-DILI

[Table 2](#T0002) showed the methylation detection results based on the paired four-cell table, where each case-control pair could be classified into one of the four basic combinations of methylation status (both case and control methylated, case methylated but control unmethylated, case unmethylated but control methylated, both case and control unmethylated). For example, in [Table 2](#T0002), ‘32’ indicated that there were 32 matched case-control pairs with methylation status. Compared with the controls, the frequency of methylation status in the promoter region of the *ALAS1* gene was higher in the AT-DILI cases (47.25% vs. 36.26%). Multivariate conditional logistic regression analysis indicated that patients with methylation status were at a higher risk of AT-DILI than those with unmethylated status (OR: 1.567, 95% CI: 1.015–2.421, *p* = 0.043) ([Table 2](#T0002)).

### Table 2.

| Controls | AT-DILI cases |  | OR (95% CI)a | p-valuea |
| --- | --- | --- | --- | --- |
| Methylation | Unmethylation |  |  |  |
| Methylation | 32 | 34 | 1.567 (1.015–2.421) | 0.043 |
| Unmethylation | 54 | 62 |   |   |

Table 2 Caption: Association analysis of CpG island methylation of ALAS1 gene with AT-DILI risk.

### 3.3. Association of genotypes & haplotypes with AT-DILI

Hardy-Weinberg equilibrium tests demonstrated no significant deviation from the expected values for three SNPs (rs352163, rs352169 and rs104903) among the controls (Supplementary Table S2). Univariate conditional logistic regression analysis indicated that patients carrying the GA genotype of rs352169 were at a higher risk of AT-DILI than those with the GG genotype (OR: 1.744, 95% CI: 1.094–2.780, *p* = 0.019) and significant differences were also found under the dominant genetic model and additive model (OR: 1.800, 95% CI: 1.152–2.813, *p* = 0.010; OR: 1.538, 95% CI: 1.097–2.156, *p* = 0.013) ([Table 3](#T0003)). After false discovery rate multiple correction, the above statistical difference still existed (corrected *p* = 0.032). And multivariate conditional logistic regression model still showed the same results. A reconstructed linkage disequilibrium plot of the *ALAS1* gene in controls is shown in Supplementary Figure S1. One haplotype (rs352163-rs352169-rs104903) was further incorporated into the multivariate conditional logistic regression model and no significant association with AT-DILI susceptibility was found (*p* > 0.05) ([Table 4](#T0004)).

### Table 3.

| SNPs | AT-DILI cases (N = 182) |  | Controls (N = 182) |  | Univariate analysis |  |  | Multivariate analysis |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| n | % | n | % | OR (95% CI)a | p-valuea | p-valueb | OR (95% CI)a | p-valuea | p-valueb |  |
| ALAS1-rs352163 (G>A) |  |  |  |  |  |  |  |  |  |  |
| GG | 69 | 37.9 | 82 | 45.0 | 1.000 | – |   | 1.000 | – |   |
| GA | 88 | 48.4 | 80 | 44.0 | 1.359 (0.856, 2.159) | 0.194 |   | 1.380 (0.862, 2.208) | 0.180 |   |
| AA | 25 | 13.7 | 20 | 11.0 | 1.555 (0.782, 3.089) | 0.208 |   | 1.590 (0.790, 3.197) | 0.194 |   |
| Dom |   |   |   |   | 1.394 (0.891, 2.180) | 0.145 |   | 1.417 (0.900, 2.231) | 0.133 |   |
| Rec |   |   |   |   | 1.278 (0.690, 2.368) | 0.436 |   | 1.294 (0.692, 2.420) | 0.420 |   |
| Add |   |   |   |   | 1.275 (0.921, 1.764) | 0.144 |   | 1.290 (0.927, 1.796) | 0.131 |   |
| ALAS1-rs352169 (G>A) |  |  |  |  |  |  |  |  |  |  |
| GG | 69 | 37.9 | 93 | 51.1 | 1.000 | – |   | 1.000 | – |   |
| GA | 91 | 50.0 | 74 | 40.7 | 1.744 (1.094, 2.780) | 0.019 | 0.032 | 1.770 (1.101, 2.847) | 0.019 | 0.032 |
| AA | 22 | 12.1 | 15 | 8.2 | 2.066 (0.977, 4.370) | 0.058 |   | 2.078 (0.963, 4.482) | 0.062 |   |
| Dom |   |   |   |   | 1.800 (1.152, 2.813) | 0.010 | 0.032 | 1.823 (1.157, 2.873) | 0.010 | 0.032 |
| Rec |   |   |   |   | 1.538 (0.765, 3.093) | 0.227 |   | 1.535 (0.754, 3.125) | 0.238 |   |
| Add |   |   |   |   | 1.538 (1.097, 2.156) | 0.013 | 0.032 | 1.550 (1.097, 2.192) | 0.013 | 0.032 |
| ALAS1-rs104903 (T>A) |  |  |  |  |  |  |  |  |  |  |
| TT | 68 | 37.4 | 83 | 45.6 | 1.000 | – |   | 1.000 | – |   |
| TA | 90 | 49.5 | 80 | 44.0 | 1.422 (0.898, 2.250) | 0.133 |   | 1.484 (0.927, 2.377) | 0.100 |   |
| AA | 24 | 13.1 | 19 | 10.4 | 1.610 (0.802, 3.235) | 0.181 |   | 1.706 (0.835, 3.485) | 0.143 |   |
| Dom |   |   |   |   | 1.455 (0.934, 2.266) | 0.098 |   | 1.522 (0.966, 2.399) | 0.070 |   |
| Rec |   |   |   |   | 1.294 (0.687, 2.437) | 0.425 |   | 1.331 (0.700, 2.530) | 0.383 |   |
| Add |   |   |   |   | 1.310 (0.945, 1.816) | 0.105 |   | 1.353 (0.968, 1.893) | 0.077 |   |

Table 3 Caption: Genotypes distribution of ALAS1 gene in two groups and the risks of AT-DILI.

### Table 4.

| Haplotype | AT-DILI cases (%) | Controls (%) | OR (95% CI)a | p-valuea |
| --- | --- | --- | --- | --- |
| G-G-A | 57.69 | 64.56 | 1.000 |   |
| A-A-T | 32.42 | 25.82 | 1.394 (0.994–1.955) | 0.054 |
| A-G-T | 4.12 | 5.77 | 0.818 (0.402–1.664) | 0.579 |
| G-A-A | 3.02 | 1.65 | 1.847 (0.669–5.098) | 0.237 |

Table 4 Caption: Distribution of ALAS1 rs352163-rs352169-rs104903 haplotype in two groups and the risks of AT-DILI.

### 3.4. Association of ALAS1 mRNA expression level with AT-DILI

The expression level of *ALAS1* mRNA in 17 AT-DILI cases and 17 matched controls was measured. The characteristics and clinical indicators for the cases and controls are shown in Supplementary Table S3. The results showed that the relative expression level of *ALAS1* mRNA in the cases was significantly lower than that in the controls (cases vs. controls, 0.75 ± 0.34 vs. 1.00 ± 0.42, *p* = 0.021) (Supplementary Figure S2). However, no significant differences of *ALAS1* mRNA level were observed in three SNPs and methylation status (Supplementary Figure S3).

### 3.5. Interaction between methylation & polymorphism in AT-DILI

Multivariate conditional logistic regression model was performed for further crossover analysis according to the genotype and methylation status ([Table 5](#T0005)). Compared with patients with the GG genotype and unmethylated status, patients with the GA + AA genotype and methylation status had a higher risk of AT-DILI (OR: 2.991, 95% CI: 1.330–6.726, *p* = 0.008) ([Table 5](#T0005)).

### Table 5.

| Genotype | Methylation | AT-DILI cases (N = 182) | Controls (N = 182) | OR (95% CI)a | p-valuea |
| --- | --- | --- | --- | --- | --- |
| GG | No | 37 | 59 | 1.000 | – |
| GA + AA | No | 59 | 57 | 2.192 (1.041–4.614) | 0.039 |
| GG | Yes | 32 | 34 | 2.007 (0.925–4.357) | 0.078 |
| GA + AA | Yes | 54 | 32 | 2.991 (1.330–6.726) | 0.008 |

Table 5 Caption: Crossover analysis of CpG island methylation in the promoter region and rs352169 polymorphism of ALAS1 gene.

## 4. Discussion

This study mainly found that patients with methylation status in the promoter region of *ALAS1* were at a higher risk of AT-DILI than those with unmethylated status (OR: 1.567, 95% CI: 1.034–2.421, *p* = 0.043) and patients with a polymorphism at rs352169 were at an increased risk of AT-DILI (GA vs. GG, OR: 1.770, 95% CI: 1.101–2.847, *p* = 0.019). In addition, our findings indicated that *ALAS1* mRNA expression was significantly reduced in AT-DILI cases compared with that in controls in the validation phase (*p* = 0.021). Furthermore, patients with the GA + AA genotype and methylation status in the *ALAS1* gene had a higher risk of AT-DILI than patients with the GG genotype and unmethylated status (OR: 2.991, 95% CI: 1.330–6.726, *p* = 0.008). To date, this study was the first to explore the relationship between *ALAS1* CpG island DNA methylation in the promoter region, genetic polymorphisms and the risk of AT-DILI and this study provided theoretical evidence for early AT-DILI screening and intervention.

The human *ALAS1* gene on chromosome 3p21.2 comprises 12 exons spanning approximately 16 kilobases and contains 4729 SNPs ([https://www.ncbi.nlm.nih.gov/](https://www.ncbi.nlm.nih.gov/)). ALAS1 is the first rate-limiting enzyme of the heme biosynthetic pathway in mammalian cells [[10](#CIT00010)] and can induce acute attacks of hepatic porphyria [[27](#CIT00027)] and hepatotoxicity [[6](#CIT00006),[28](#CIT00028)]. To date, three studies have explored the relationship between polymorphisms in the *ALAS1* gene and the risk of AT-DILI and the most studied SNP was rs352169 [[16](#CIT00016),[22](#CIT00022),[29](#CIT00029)]. Two studies with 118 AT-DILI cases and 628 controls from West China Hospital in Sichuan Province [[22](#CIT00022),[29](#CIT00029)] and one study with 202 AT-DILI cases and 808 controls from four hospitals in Jiangsu Province [[16](#CIT00016)] did not show any correlation between rs352169 and susceptibility to AT-DILI. However, in this study, the association between SNP rs352169 and the risk of AT-DILI still existed after adjusting for multiple different variables. Further bioinformatics analysis revealed that the RegulomeDB score was 1f, indicating that SNP rs352169 may affect transcription factor-binding sites and DNase peaks ([https://www.regulomedb.org/](https://www.regulomedb.org/)) [[30](#CIT00030)]. Two SNPs (rs352163 and rs104903) are also potentially functional, but they are not meant to be associated with AT-DILI risk. The relationship between genotype and phenotype and between genotype and disease is not always univocal [[31](#CIT00031)]. Our study is inconsistent with some previous reports. This discrepancy could have several reasons. First, the diagnosis of AT-DILI cases was different: two were based on the Common Terminology Criteria for Adverse Events Version 5.0 criteria (ALT and/or AST ≥3 ULN with hepatitis symptoms or ≥5 ULN with or without symptoms or TBil ≥1.5 ULN) [[22](#CIT00022),[29](#CIT00029)], one was based on Chinese guidelines (ALT ≥3 ULN) [[16](#CIT00016)] and the present study was based on the Drug Hepatotoxicity Steering Committee (ALT ≥5 ULN). Second, the case-control study designs were different; two were nonmatched [[22](#CIT00022),[29](#CIT00029)] and one was 1:4 matched [[16](#CIT00016)]. Third, covariates adjusted during analysis were different, one adjusting for age and sex [[22](#CIT00022)], one adjusting for liver disease history, hepatoprotectant use, smoking history and drinking history [[16](#CIT00016)] and another without description [[29](#CIT00029)] and potential confounding factors (smoking history, drinking history and hepatoprotectant use) were adjusted in the present study to improve efficiency. Supplementary Table S4 showed allele frequency distribution of the three SNPs in different populations. The major allele frequencies of rs352163, rs352169 and rs104903 were generally higher in our study population than in other regions except African. All these factors may account for the differences in results between studies. Therefore, genotyping of different populations is necessary and whether the results of this study can be verified in population in other regions remains to be studied.

In addition to genetic polymorphisms, interindividual epigenetic variation is likely to be an important mechanism contributing to interindividual variability in the expression and function of ALAS1, which may lead to the development and progression of liver diseases [[32](#CIT00032)]. One previous study indicated that *HIF-1*α gene CpG island methylation was associated with a higher risk of AT-DILI (OR: 1.92, 95% CI: 1.32–2.63) [[33](#CIT00033)]. The present study also showed that methylation of the CpG island in the *ALAS1* gene was a risk factor for AT-DILI. In general, DNA methylation can reduce gene expression and eventually change phenotypic modifications by recruiting proteins involved in inhibiting gene expression or inhibiting the binding of transcription factors to DNA [[34](#CIT00034)]. Our study also indicated that the interaction between the *ALAS1* gene rs352169 polymorphism and CpG island methylation in the promoter region is associated with the risk of AT-DILI. In recent years, many studies have focused on the interaction between DNA methylation and polymorphism. A large number of *cis*-meQTL associations identified in one study have suggested that DNA methylation levels were under strong genetic influence and two SNPs (rs4970774 and rs3814309) negatively regulated the GSTM1 and GSTM4 by negatively controlling seven methylated sites collectively and methylation sites were typically associated with SNPs in close proximity [[35](#CIT00035)]. Another study clearly proved that there was a rs886205-genotype-dependent difference in the epigenetic regulation of the analyzed CpGs in the presence of alcohol [[36](#CIT00036)]. Moreover, whether and how the SNPs influences the methylation structure and then regulates gene expression is unclear yet. Our study found that patients with the GA + AA genotype and methylation status in the *ALAS1* gene had a higher risk of AT-DILI than patients with the GG genotype and unmethylated status (OR: 2.991, 95% CI: 1.330–6.726, *p* = 0.008). There may be a synergistic effect between the rs352169 polymorphism and methylation, which jointly affects the occurrence of AT-DILI. In the course of AT-DILI, RIF and INH upregulate the expression of ALAS1 through the PXR pathway and induce an increase in hepatotoxic PPIX [[6](#CIT00006)]. At the same time, multiple mechanisms could inhibit ALAS1 levels, such as methylation in the promoter region of *ALAS1* and a heme-dependent negative feedback mechanism [[12](#CIT00012)], which keep ALAS1 at low levels. One animal experiment also showed that ALAS1 levels briefly increased and then decreased after liver injury in rats [[7](#CIT00007)]. The regulatory mechanisms of ALAS1 are complex and still need to be further explored.

To the best of our knowledge, this is the first study attempting to evaluate the relationship between CpG island DNA methylation in the promoter region, genetic polymorphism of *ALAS1* and the risk of AT-DILI in the Chinese population and it provides further insight into the potential mechanisms of AT-DILI in terms of both epigenetics and genetics. The present study was designed as a 1:1 matched case-control study based on stricter criteria for AT-DILI (ALT ≥5 ULN) and all cases were strictly assessed by the RUCAM score system to reduce misclassification bias. However, two potential limitations cannot be ignored. First, the methylation-specific PCR was used for DNA methylation analysis, which is well established and widely used until now and can rapidly detect the methylation status of CpG island in the promoter region. Although this method has the advantages of stable results and low cost, only qualitative results can be obtained and accurate DNA methylation levels cannot be obtained. Second, the sample sizes of real-time quantitative PCR experiment were small and caution is needed for interpreting statistically different results and the methylation degree and mRNA levels of the *ALAS1* gene need to be proven in future studies with large samples.

## 5. Conclusion

Based on this 1:1 matched case-control study, the methylation status in the promoter region and polymorphism at rs352169 in *ALAS1* may be associated with susceptibility to AT-DILI. Moreover, there may be a synergistic effect between methylation and polymorphism in the occurrence of AT-DILI.

## Supplementary Material

## Acknowledgments

We are grateful to all cooperating organizations and their staff whose hard work made this study possible.

## Funding Statement

This study was funded by a grant from the National Natural Science Foundation of China (grant no.: 82073614).

## Supplemental material

Supplemental data for this article can be accessed at [https://doi.org/10.1080/14622416.2024.2392480](https://doi.org/10.1080/14622416.2024.2392480)

## Author contributions

Z Hao and B Han contributed equally to this work. Z Hao was the principal investigator who conceived the study. X He, L Lu, M Zhang and H Pan collected the clinical data and sample. Z Hao, B Han and X Zhou performed laboratory experiments. B Han analyzed laboratory data and H Jian checked the data. Z Hao wrote the manuscript. H Yi examined and analyzed the data and S Tang conceived and designed the study, interpreted the data, revised the manuscript and supervised the quality of the study throughout the conduct of the project. All authors have approved the final draft submitted.

## Financial disclosure

This study was funded by a grant from the National Natural Science Foundation of China (grant no.: 82073614). The authors have no other relevant affiliations or financial involvement with any organization or entity with a financial interest in or financial conflict with the subject matter or materials discussed in the manuscript apart from those disclosed.

## Competing interests disclosure

The authors have no competing interests or relevant affiliations with any organization or entity with the subject matter or materials discussed in the manuscript. This includes employment, consultancies, honoraria, stock ownership or options, expert testimony, grants or patents received or pending, or royalties.

## Writing disclosure

No writing assistance was utilized in the production of this manuscript.

## Ethical conduct of research

The study was approved by the Ethics Committee of Nanjing Medical University (The approval number: [2018]579) and conducted in accordance with the Declaration of Helsinki principles. Written informed consent was obtained from every participant before enrolment.

## Data availability statement

The data supporting the findings of this study are included within the main manuscript and the supplemental files. All data generated and analyzed in the study are available from the corresponding author upon reasonable request.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

The data supporting the findings of this study are included within the main manuscript and the supplemental files. All data generated and analyzed in the study are available from the corresponding author upon reasonable request.

### Supplementary Materials

### Data Availability Statement

The data supporting the findings of this study are included within the main manuscript and the supplemental files. All data generated and analyzed in the study are available from the corresponding author upon reasonable request.

## References

1. Bagcchi S. WHO's global tuberculosis report 2022. Lancet Microbe. 2023;4(1):e20. doi: 10.1016/S2666-5247(22)00359-7  [DOI](https://doi.org/10.1016/S2666-5247(22)00359-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36521512/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet%20Microbe.&title=WHO's%20global%20tuberculosis%20report%202022&volume=4&issue=1&publication_year=2023&pages=e20&pmid=36521512&doi=10.1016/S2666-5247(22)00359-7&)

2. Grace AG, Mittal A, Jain S, et al. Shortened treatment regimens versus the standard regimen for drug-sensitive pulmonary tuberculosis. Cochrane Database Syst Rev. 2019;12(12):CD012918. doi: 10.1002/14651858.CD012918.pub2  [DOI](https://doi.org/10.1002/14651858.CD012918.pub2) | [PMC free article](/articles/PMC6953336/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31828771/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cochrane%20Database%20Syst%20Rev.&title=Shortened%20treatment%20regimens%20versus%20the%20standard%20regimen%20for%20drug-sensitive%20pulmonary%20tuberculosis&volume=12&issue=12&publication_year=2019&pages=CD012918&pmid=31828771&doi=10.1002/14651858.CD012918.pub2&)

3. Jiang FR, Yan HD, Liang LL, et al. Incidence and risk factors of anti-tuberculosis drug induced liver injury (DILI): large cohort study involving 4652 Chinese adult tuberculosis patients. Liver Int. 2021;41(7):1565–1575. doi: 10.1111/liv.14896  [DOI](https://doi.org/10.1111/liv.14896) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33866661/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Liver%20Int.&title=Incidence%20and%20risk%20factors%20of%20anti-tuberculosis%20drug%20induced%20liver%20injury%20(DILI):%20large%20cohort%20study%20involving%204652%20Chinese%20adult%20tuberculosis%20patients&volume=41&issue=7&publication_year=2021&pages=1565-1575&pmid=33866661&doi=10.1111/liv.14896&)

4. Alsayed SSR, Gunosewoyo H. Tuberculosis: pathogenesis, current treatment regimens and new drug targets. Int J Mol Sci. 2023;24(6):5202. doi: 10.3390/ijms24065202  [DOI](https://doi.org/10.3390/ijms24065202) | [PMC free article](/articles/PMC10049048/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36982277/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Mol%20Sci.&title=Tuberculosis:%20pathogenesis,%20current%20treatment%20regimens%20and%20new%20drug%20targets&volume=24&issue=6&publication_year=2023&pages=5202&pmid=36982277&doi=10.3390/ijms24065202&)

5. Ezhilarasan D. Antitubercular drugs induced liver injury: an updated insight into molecular mechanisms. Drug Metab. Rev. 2023;55(3):239–253. doi: 10.1080/03602532.2023.2215478  [DOI](https://doi.org/10.1080/03602532.2023.2215478) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/37218081/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab.%20Rev.&title=Antitubercular%20drugs%20induced%20liver%20injury:%20an%20updated%20insight%20into%20molecular%20mechanisms&volume=55&issue=3&publication_year=2023&pages=239-253&pmid=37218081&doi=10.1080/03602532.2023.2215478&)

6. Lyoumi S, Lefebvre T, Karim Z, et al. PXR-ALAS1: a key regulatory pathway in liver toxicity induced by isoniazid-rifampicin antituberculosis treatment. Clin Res Hepatol Gastroenterol. 2013;37(5):439–441. doi: 10.1016/j.clinre.2013.06.010  [DOI](https://doi.org/10.1016/j.clinre.2013.06.010) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23916555/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Res%20Hepatol%20Gastroenterol.&title=PXR-ALAS1:%20a%20key%20regulatory%20pathway%20in%20liver%20toxicity%20induced%20by%20isoniazid-rifampicin%20antituberculosis%20treatment&volume=37&issue=5&publication_year=2013&pages=439-441&pmid=23916555&doi=10.1016/j.clinre.2013.06.010&)

7. Tanioka N, Shimizu H, Omori E, et al. Role of the transcription factor BTB and CNC homology 1 in a rat model of acute liver injury induced by experimental endotoxemia. Acta Med Okayama. 2021;75(3):363–372.  [DOI](https://doi.org/10.18926/AMO/62232) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34176941/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Acta%20Med%20Okayama.&title=Role%20of%20the%20transcription%20factor%20BTB%20and%20CNC%20homology%201%20in%20a%20rat%20model%20of%20acute%20liver%20injury%20induced%20by%20experimental%20endotoxemia&volume=75&issue=3&publication_year=2021&pages=363-372&pmid=34176941&doi=10.18926/AMO/62232&)

8. Li F, Lu J, Cheng J, et al. Human PXR modulates hepatotoxicity associated with rifampicin and isoniazid co-therapy. Nat Med. 2013;19(4):418–420. doi: 10.1038/nm.3104  [DOI](https://doi.org/10.1038/nm.3104) | [PMC free article](/articles/PMC3618537/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23475203/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Med.&title=Human%20PXR%20modulates%20hepatotoxicity%20associated%20with%20rifampicin%20and%20isoniazid%20co-therapy&volume=19&issue=4&publication_year=2013&pages=418-420&pmid=23475203&doi=10.1038/nm.3104&)

9. Sachar M, anderson KE, Ma X. Protoporphyrin IX: the good, the bad and the ugly. J Pharmacol Exp Ther. 2016;356(2):267–275. doi: 10.1124/jpet.115.228130  [DOI](https://doi.org/10.1124/jpet.115.228130) | [PMC free article](/articles/PMC4727154/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26588930/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacol%20Exp%20Ther.&title=Protoporphyrin%20IX:%20the%20good,%20the%20bad%20and%20the%20ugly&volume=356&issue=2&publication_year=2016&pages=267-275&pmid=26588930&doi=10.1124/jpet.115.228130&)

10. Kubota Y, Nomura K, Katoh Y, et al. Novel mechanisms for heme-dependent degradation of ALAS1 protein as a component of negative feedback regulation of heme biosynthesis. J Biol Chem. 2016;291(39):20516–20529. doi: 10.1074/jbc.M116.719161  [DOI](https://doi.org/10.1074/jbc.M116.719161) | [PMC free article](/articles/PMC5034046/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27496948/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem.&title=Novel%20mechanisms%20for%20heme-dependent%20degradation%20of%20ALAS1%20protein%20as%20a%20component%20of%20negative%20feedback%20regulation%20of%20heme%20biosynthesis&volume=291&issue=39&publication_year=2016&pages=20516-20529&pmid=27496948&doi=10.1074/jbc.M116.719161&)

11. Wang J, Bwayi M, Gee RRF, et al. PXR-mediated idiosyncratic drug-induced liver injury: mechanistic insights and targeting approaches. Expert Opin Drug Metab Toxicol. 2020;16(8):711–722. doi: 10.1080/17425255.2020.1779701  [DOI](https://doi.org/10.1080/17425255.2020.1779701) | [PMC free article](/articles/PMC7429329/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32500752/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Expert%20Opin%20Drug%20Metab%20Toxicol.&title=PXR-mediated%20idiosyncratic%20drug-induced%20liver%20injury:%20mechanistic%20insights%20and%20targeting%20approaches&volume=16&issue=8&publication_year=2020&pages=711-722&pmid=32500752&doi=10.1080/17425255.2020.1779701&)

12. Peoc'h K, Nicolas G, Schmitt C, et al. Regulation and tissue-specific expression of delta-aminolevulinic acid synthases in non-syndromic sideroblastic anemias and porphyrias. Mol Genet Metab. 2019;128(3):190–197. doi: 10.1016/j.ymgme.2019.01.015  [DOI](https://doi.org/10.1016/j.ymgme.2019.01.015) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30737140/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Genet%20Metab.&title=Regulation%20and%20tissue-specific%20expression%20of%20delta-aminolevulinic%20acid%20synthases%20in%20non-syndromic%20sideroblastic%20anemias%20and%20porphyrias&volume=128&issue=3&publication_year=2019&pages=190-197&pmid=30737140&doi=10.1016/j.ymgme.2019.01.015&)

13. Huai C, Wei Y, Li M, et al. Genome-wide analysis of DNA methylation and antituberculosis drug-induced liver injury in the Han Chinese population. Clin Pharmacol Ther. 2019;106(6):1389–1397. doi: 10.1002/cpt.1563  [DOI](https://doi.org/10.1002/cpt.1563) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31247120/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther.&title=Genome-wide%20analysis%20of%20DNA%20methylation%20and%20antituberculosis%20drug-induced%20liver%20injury%20in%20the%20Han%20Chinese%20population&volume=106&issue=6&publication_year=2019&pages=1389-1397&pmid=31247120&doi=10.1002/cpt.1563&)

14. Chen R, Wang J, Zhang Y, et al. Key factors of susceptibility to anti-tuberculosis drug-induced hepatotoxicity. Arch Toxicol. 2015;89(6):883–897. doi: 10.1007/s00204-015-1473-1  [DOI](https://doi.org/10.1007/s00204-015-1473-1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25693865/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arch%20Toxicol.&title=Key%20factors%20of%20susceptibility%20to%20anti-tuberculosis%20drug-induced%20hepatotoxicity&volume=89&issue=6&publication_year=2015&pages=883-897&pmid=25693865&doi=10.1007/s00204-015-1473-1&)

15. He L, Gao L, Shi Z, et al. Involvement of cytochrome P450 1A1 and glutathione S-transferase P1 polymorphisms and promoter hypermethylation in the progression of anti-tuberculosis drug-induced liver injury: a case-control study. PLOS ONE. 2015;10(3):e0119481. doi: 10.1371/journal.pone.0119481  [DOI](https://doi.org/10.1371/journal.pone.0119481) | [PMC free article](/articles/PMC4370371/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25798582/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLOS%20ONE.&title=Involvement%20of%20cytochrome%20P450%201A1%20and%20glutathione%20S-transferase%20P1%20polymorphisms%20and%20promoter%20hypermethylation%20in%20the%20progression%20of%20anti-tuberculosis%20drug-induced%20liver%20injury:%20a%20case-control%20study&volume=10&issue=3&publication_year=2015&pages=e0119481&pmid=25798582&doi=10.1371/journal.pone.0119481&)

16. Zhang ML, Zhu J, Wang NN, et al. The role of the genetic variant FECH rs11660001 in the occurrence of anti-tuberculosis drug-induced liver injury. J Clin Pharm Ther. 2022;47(8):1276–1283. doi: 10.1111/jcpt.13672  [DOI](https://doi.org/10.1111/jcpt.13672) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35470464/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Pharm%20Ther.&title=The%20role%20of%20the%20genetic%20variant%20FECH%20rs11660001%20in%20the%20occurrence%20of%20anti-tuberculosis%20drug-induced%20liver%20injury&volume=47&issue=8&publication_year=2022&pages=1276-1283&pmid=35470464&doi=10.1111/jcpt.13672&)

17. Aithal GP, Watkins PB, andrade RJ, et al. Case definition and phenotype standardization in drug-induced liver injury. Clin Pharmacol Ther. 2011;89(6):806–815. doi: 10.1038/clpt.2011.58  [DOI](https://doi.org/10.1038/clpt.2011.58) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21544079/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther.&title=Case%20definition%20and%20phenotype%20standardization%20in%20drug-induced%20liver%20injury&volume=89&issue=6&publication_year=2011&pages=806-815&pmid=21544079&doi=10.1038/clpt.2011.58&)

18. Danan G, Teschke R. RUCAM in drug and herb induced liver injury: the update. Int J Mol Sci. 2015;17(1):14. doi: 10.3390/ijms17010014  [DOI](https://doi.org/10.3390/ijms17010014) | [PMC free article](/articles/PMC4730261/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26712744/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Mol%20Sci.&title=RUCAM%20in%20drug%20and%20herb%20induced%20liver%20injury:%20the%20update&volume=17&issue=1&publication_year=2015&pages=14&pmid=26712744&doi=10.3390/ijms17010014&)

19. Yang M, Pan H, Lu L, et al. Home-based anti-tuberculosis treatment adverse reactions (HATTAR) study: a protocol for a prospective observational study. BMJ Open. 2019;9(3):e027321. doi: 10.1136/bmjopen-2018-027321  [DOI](https://doi.org/10.1136/bmjopen-2018-027321) | [PMC free article](/articles/PMC6475199/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30928962/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMJ%20Open.&title=Home-based%20anti-tuberculosis%20treatment%20adverse%20reactions%20(HATTAR)%20study:%20a%20protocol%20for%20a%20prospective%20observational%20study&volume=9&issue=3&publication_year=2019&pages=e027321&pmid=30928962&doi=10.1136/bmjopen-2018-027321&)

20. Sun P, Shen Y, Wang T, et al. Distinct clinical and genetic mutation characteristics in sporadic and Lynch syndrome-associated endometrial cancer in a Chinese population. Cancer Epidemiol. 2021;73:101934. doi: 10.1016/j.canep.2021.101934  [DOI](https://doi.org/10.1016/j.canep.2021.101934) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34000661/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Epidemiol.&title=Distinct%20clinical%20and%20genetic%20mutation%20characteristics%20in%20sporadic%20and%20Lynch%20syndrome-associated%20endometrial%20cancer%20in%20a%20Chinese%20population&volume=73&publication_year=2021&pages=101934&pmid=34000661&doi=10.1016/j.canep.2021.101934&)

21. Pajares MJ, Palanca-Ballester C, Urtasun R, et al. Methods for analysis of specific DNA methylation status. Methods. 2021;187:3–12. doi: 10.1016/j.ymeth.2020.06.021  [DOI](https://doi.org/10.1016/j.ymeth.2020.06.021) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32640317/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Methods.&title=Methods%20for%20analysis%20of%20specific%20DNA%20methylation%20status&volume=187&publication_year=2021&pages=3-12&pmid=32640317&doi=10.1016/j.ymeth.2020.06.021&)

22. Zhang JW, Jiao L, Song JJ, et al. Genetic and functional evaluation of the role of FOXO1 in antituberculosis drug-induced hepatotoxicity. Evid-Based Compl Alt. 2021;2021:3185874. doi: 10.1155/2021/3185874  [DOI](https://doi.org/10.1155/2021/3185874) | [PMC free article](/articles/PMC8238576/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34249128/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Evid-Based%20Compl%20Alt.&title=Genetic%20and%20functional%20evaluation%20of%20the%20role%20of%20FOXO1%20in%20antituberculosis%20drug-induced%20hepatotoxicity&volume=2021&publication_year=2021&pages=3185874&pmid=34249128&doi=10.1155/2021/3185874&)

23. Chen YT, Wu Y, Yang H, et al. DNA methylation and mRNA expression of B7-H3 gene in ankylosing spondylitis: a case-control study. Immunol Invest. 2022;51(7):2025–2034. doi: 10.1080/08820139.2022.2095285  [DOI](https://doi.org/10.1080/08820139.2022.2095285) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35786112/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Immunol%20Invest.&title=DNA%20methylation%20and%20mRNA%20expression%20of%20B7-H3%20gene%20in%20ankylosing%20spondylitis:%20a%20case-control%20study&volume=51&issue=7&publication_year=2022&pages=2025-2034&pmid=35786112&doi=10.1080/08820139.2022.2095285&)

24. Abbara A, Chitty S, Roe JK, et al. Drug-induced liver injury from antituberculous treatment: a retrospective study from a large TB centre in the UK. BMC Infect Dis. 2017;17(1):231. doi: 10.1186/s12879-017-2330-z  [DOI](https://doi.org/10.1186/s12879-017-2330-z) | [PMC free article](/articles/PMC5366108/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28340562/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Infect%20Dis.&title=Drug-induced%20liver%20injury%20from%20antituberculous%20treatment:%20a%20retrospective%20study%20from%20a%20large%20TB%20centre%20in%20the%20UK&volume=17&issue=1&publication_year=2017&pages=231&pmid=28340562&doi=10.1186/s12879-017-2330-z&)

25. Chen Q, Hu A, Ma A, et al. Effectiveness of prophylactic use of hepatoprotectants for tuberculosis drug-induced liver injury: a population-based cohort analysis involving 6,743 Chinese patients. Front Pharmacol. 2022;13:813682. doi: 10.3389/fphar.2022.813682  [DOI](https://doi.org/10.3389/fphar.2022.813682) | [PMC free article](/articles/PMC9065346/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35517815/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front%20Pharmacol.&title=Effectiveness%20of%20prophylactic%20use%20of%20hepatoprotectants%20for%20tuberculosis%20drug-induced%20liver%20injury:%20a%20population-based%20cohort%20analysis%20involving%206,743%20Chinese%20patients&volume=13&publication_year=2022&pages=813682&pmid=35517815&doi=10.3389/fphar.2022.813682&)

26. Zaverucha-do-Valle C, Monteiro SP, El-Jaick KB, et al. The role of cigarette smoking and liver enzymes polymorphisms in anti-tuberculosis drug-induced hepatotoxicity in Brazilian patients. Tuberculosis (Edinb). 2014;94(3):299–305. doi: 10.1016/j.tube.2014.03.006  [DOI](https://doi.org/10.1016/j.tube.2014.03.006) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24793319/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Tuberculosis%20(Edinb).&title=The%20role%20of%20cigarette%20smoking%20and%20liver%20enzymes%20polymorphisms%20in%20anti-tuberculosis%20drug-induced%20hepatotoxicity%20in%20Brazilian%20patients&volume=94&issue=3&publication_year=2014&pages=299-305&pmid=24793319&doi=10.1016/j.tube.2014.03.006&)

27. Manceau H, Gouya L, Puy H. Acute hepatic and erythropoietic porphyrias: from ALA synthases 1 and 2 to new molecular bases and treatments. Curr Opin Hematol. 2017;24(3):198–207. doi: 10.1097/MOH.0000000000000330  [DOI](https://doi.org/10.1097/MOH.0000000000000330) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28118224/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Opin%20Hematol.&title=Acute%20hepatic%20and%20erythropoietic%20porphyrias:%20from%20ALA%20synthases%201%20and%202%20to%20new%20molecular%20bases%20and%20treatments&volume=24&issue=3&publication_year=2017&pages=198-207&pmid=28118224&doi=10.1097/MOH.0000000000000330&)

28. Fujii H, Takahashi T, Matsumi M, et al. Increased heme oxygenase-1 and decreased delta-aminolevulinate synthase expression in the liver of patients with acute liver failure. Int J Mol Med. 2004;14(6):1001–1005.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/15547665/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Mol%20Med.&title=Increased%20heme%20oxygenase-1%20and%20decreased%20delta-aminolevulinate%20synthase%20expression%20in%20the%20liver%20of%20patients%20with%20acute%20liver%20failure&volume=14&issue=6&publication_year=2004&pages=1001-1005&pmid=15547665&)

29. Zhang JW, Zhou W, Ma SJ, et al. Combined electronic medical records and gene polymorphism characteristics to establish an anti-tuberculosis drug-induced hepatic injury (ATDH) prediction model and evaluate the prediction value. Ann Transl Med. 2022;10(20):1114. doi: 10.21037/atm-22-4551  [DOI](https://doi.org/10.21037/atm-22-4551) | [PMC free article](/articles/PMC9652536/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36388795/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Transl%20Med.&title=Combined%20electronic%20medical%20records%20and%20gene%20polymorphism%20characteristics%20to%20establish%20an%20anti-tuberculosis%20drug-induced%20hepatic%20injury%20(ATDH)%20prediction%20model%20and%20evaluate%20the%20prediction%20value&volume=10&issue=20&publication_year=2022&pages=1114&pmid=36388795&doi=10.21037/atm-22-4551&)

30. Boyle AP, Hong EL, Hariharan M, et al. Annotation of functional variation in personal genomes using RegulomeDB. Genome Res. 2012;22(9):1790–1797. doi: 10.1101/gr.137323.112  [DOI](https://doi.org/10.1101/gr.137323.112) | [PMC free article](/articles/PMC3431494/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22955989/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res.&title=Annotation%20of%20functional%20variation%20in%20personal%20genomes%20using%20RegulomeDB&volume=22&issue=9&publication_year=2012&pages=1790-1797&pmid=22955989&doi=10.1101/gr.137323.112&)

31. Picard N, Boyer JC, Etienne-Grimaldi MC, et al. Pharmacogenetics-based personalized therapy: levels of evidence and recommendations from the French Network of Pharmacogenetics (RNPGx). Therapie. 2017;72(2):185–192. doi: 10.1016/j.therap.2016.09.014  [DOI](https://doi.org/10.1016/j.therap.2016.09.014) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28237406/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Therapie.&title=Pharmacogenetics-based%20personalized%20therapy:%20levels%20of%20evidence%20and%20recommendations%20from%20the%20French%20Network%20of%20Pharmacogenetics%20(RNPGx)&volume=72&issue=2&publication_year=2017&pages=185-192&pmid=28237406&doi=10.1016/j.therap.2016.09.014&)

32. Botello-Manilla AE, Chavez-Tapia NC, Uribe M, et al. Genetics and epigenetics purpose in nonalcoholic fatty liver disease. Expert Rev Gastroent. 2020;14(8):733–747. doi: 10.1080/17474124.2020.1780915  [DOI](https://doi.org/10.1080/17474124.2020.1780915) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32552211/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Expert%20Rev%20Gastroent.&title=Genetics%20and%20epigenetics%20purpose%20in%20nonalcoholic%20fatty%20liver%20disease&volume=14&issue=8&publication_year=2020&pages=733-747&pmid=32552211&doi=10.1080/17474124.2020.1780915&)

33. Chong YZ, Zhu HY, Ren Q, et al. Interaction between the HIF-1 alpha gene rs1957757 polymorphism and CpG island methylation in the promoter region is associated with the risk of anti-tuberculosis drug-induced liver injury in humans: a case-control study. J Clin Pharm Ther. 2022;47(7):948–955. doi: 10.1111/jcpt.13625  [DOI](https://doi.org/10.1111/jcpt.13625) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35218216/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Pharm%20Ther.&title=Interaction%20between%20the%20HIF-1%20alpha%20gene%20rs1957757%20polymorphism%20and%20CpG%20island%20methylation%20in%20the%20promoter%20region%20is%20associated%20with%20the%20risk%20of%20anti-tuberculosis%20drug-induced%20liver%20injury%20in%20humans:%20a%20case-control%20study&volume=47&issue=7&publication_year=2022&pages=948-955&pmid=35218216&doi=10.1111/jcpt.13625&)

34. Ichino L, Boone BA, Strauskulage L, et al. MBD5 and MBD6 couple DNA methylation to gene silencing through the J-domain protein SILENZIO. Science. 2021;372:1434–1439. doi: 10.1126/science.abg6130  [DOI](https://doi.org/10.1126/science.abg6130) | [PMC free article](/articles/PMC8639832/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34083448/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science.&title=MBD5%20and%20MBD6%20couple%20DNA%20methylation%20to%20gene%20silencing%20through%20the%20J-domain%20protein%20SILENZIO&volume=372&publication_year=2021&pages=1434-1439&pmid=34083448&doi=10.1126/science.abg6130&)

35. Lu YH, Wang BH, Jiang F, et al. Multi-omics integrative analysis identified SNP-methylation-mRNA: interaction in peripheral blood mononuclear cells. J Cellular Molecular Medicine. 2019;23(7):4601–4610. doi: 10.1111/jcmm.14315  [DOI](https://doi.org/10.1111/jcmm.14315) | [PMC free article](/articles/PMC6584519/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31106970/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Cellular%20Molecular%20Medicine.&title=Multi-omics%20integrative%20analysis%20identified%20SNP-methylation-mRNA:%20interaction%20in%20peripheral%20blood%20mononuclear%20cells&volume=23&issue=7&publication_year=2019&pages=4601-4610&pmid=31106970&doi=10.1111/jcmm.14315&)

36. Pathak H, Frieling H, Bleich S, et al. Promoter polymorphism rs886205 genotype interacts with DNA methylation of the ALDH2 regulatory region in alcohol dependence. Alcohol Alcoholism (Oxford, Oxfordshire). 2017;52(3):269–276. doi: 10.1093/alcalc/agw106  [DOI](https://doi.org/10.1093/alcalc/agw106) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28430929/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Alcohol%20Alcoholism%20(Oxford,%20Oxfordshire).&title=Promoter%20polymorphism%20rs886205%20genotype%20interacts%20with%20DNA%20methylation%20of%20the%20ALDH2%20regulatory%20region%20in%20alcohol%20dependence&volume=52&issue=3&publication_year=2017&pages=269-276&pmid=28430929&doi=10.1093/alcalc/agw106&)
