# The association between HLA-B variants and amoxicillin-induced severe cutaneous adverse reactions in Chinese han population

## Metadata
**Authors:** Ting Wang, Jin Yang, Fanping Yang, Ye Cheng, Zichong Huang, Bei Li, Linlin Yang, Qinghe Xing, Xiaoqun Luo
**Journal:** Frontiers in Pharmacology
**Date:** 2024 May 28
**DOI:** [10.3389/fphar.2024.1400239](https://doi.org/10.3389/fphar.2024.1400239)
**PMID:** 38863977
**PMCID:** PMC11165025
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11165025/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC11165025/pdf/fphar-15-1400239.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC11165025/pdf/fphar-15-1400239.pdf)

## Abstract

**Background:** 
Amoxicillin (AMX) is among the most prescribed and the best tolerated antimicrobials worldwide. However, it can occasionally trigger severe cutaneous adverse reactions (SCAR) with a significant morbidity and mortality. The genetic factors that may be relevant to AMX-induced SCAR (AMX-SCAR) remain unclear. Identification of the genetic risk factor may prevent patients from the risk of AMX exposure and resume therapy with other falsely implicated drugs.

**Methodology:** 
Four patients with AMX-SCAR, 1,000 population control and 100 AMX-tolerant individuals were enrolled in this study. Both exome-wide and HLA-based association studies were conducted. Molecular docking analysis was employed to simulate the interactions between AMX and risk HLA proteins.

**Results:** 
Compared with AMX-tolerant controls, a significant association of HLA-B*15:01 with AMX-SCAR was validated [odds ratio (OR) = 22.9, 95% confidence interval (CI): 1.68–1275.67; p = 7.34 × 10−3]. Moreover, 75% carriers of HLA-B*15:01 in four patients with AMX-SCAR, and the carrier frequency of 10.7% in 1,000 control individuals and 11.0% in 100 AMX-tolerant controls, respectively. Within HLA-B protein, the S140 present in all cases and demonstrated the strongest association with AMX-SCAR [OR = 53.5, p = 5.18 × 10−4]. Molecular docking results also confirmed the interaction between AMX and S140 of the HLA-B protein, thus eliminating the false-positive results during in association analysis.

**Conclusion:** 
Our findings suggest that genetic susceptibility may be involved in the development of AMX-SCAR in Han Chinese. However, whether the HLA-B variants observed in this study can be used as an effective genetic marker of AMX-induced SCAR still needs to be further explored in larger cohort studies and other ethnic populations.

Keywords: amoxicillin, severe cutaneous adverse reactions, HLA-B, variants, whole-exome sequencing

### Background

Amoxicillin (AMX) is among the most prescribed and the best tolerated antimicrobials worldwide. However, it can occasionally trigger severe cutaneous adverse reactions (SCAR) with a significant morbidity and mortality. The genetic factors that may be relevant to AMX-induced SCAR (AMX-SCAR) remain unclear. Identification of the genetic risk factor may prevent patients from the risk of AMX exposure and resume therapy with other falsely implicated drugs.

### Methodology

Four patients with AMX-SCAR, 1,000 population control and 100 AMX-tolerant individuals were enrolled in this study. Both exome-wide and HLA-based association studies were conducted. Molecular docking analysis was employed to simulate the interactions between AMX and risk HLA proteins.

### Results

Compared with AMX-tolerant controls, a significant association of *HLA-B*15:01* with AMX-SCAR was validated [odds ratio (OR) = 22.9, 95% confidence interval (CI): 1.68–1275.67; *p* = 7.34 × 10^−3^]. Moreover, 75% carriers of *HLA-B*15:01* in four patients with AMX-SCAR, and the carrier frequency of 10.7% in 1,000 control individuals and 11.0% in 100 AMX-tolerant controls, respectively. Within HLA-B protein, the S140 present in all cases and demonstrated the strongest association with AMX-SCAR [OR = 53.5, *p* = 5.18 × 10^−4^]. Molecular docking results also confirmed the interaction between AMX and S140 of the HLA-B protein, thus eliminating the false-positive results during in association analysis.

### Conclusion

Our findings suggest that genetic susceptibility may be involved in the development of AMX-SCAR in Han Chinese. However, whether the HLA-B variants observed in this study can be used as an effective genetic marker of AMX-induced SCAR still needs to be further explored in larger cohort studies and other ethnic populations.

## Introduction

Amoxicillin (AMX) is among the most prescribed and the best tolerated antimicrobials worldwide ([Sharland et al., 2018](#B25)). However, AMX is also one of the most common drug allergy labels in the electronic health record, it can occasionally trigger severe cutaneous adverse reactions (SCAR), such as Stevens-Johnson syndrome (SJS), toxic epidermal necrolysis (TEN), drug reactions with eosinophilia and systemic symptoms (DRESS), and drug-induced acute erythroderma, with a significant morbidity and mortality ([Aouam et al., 2010](#B1); [Girelli et al., 2013](#B10); [Pejcic et al., 2023](#B21)). In time of the development of SCAR in combination medicines and antibiotic resistant settings, future exposure to all concurrently dosed drugs is contraindicated, seriously restricting the choice of candidate drugs for the patients ([Sodhi et al., 2023](#B26)). To date, SCAR is considered as immune-mediated delayed hypersensitivity reactions, due to drug-meditated immune response caused by cytotoxic T lymphocytes are specifically activated that occur until at least 3–4 days after of exposure to the causative drug ([Chessman et al., 2008](#B6); [Wei et al., 2012](#B31)).

Advances in pharmacogenetics and pharmacogenomics have revealed the relationship between genetics and SCAR and excavated important molecules involved in the pathogenesis of drug hypersensitivity. A growing body of evidence suggested that HLA alleles are strongly associated with the drug-specific SCARs, and pretherapy screening for the risk HLA alleles could effectively reduce the occurrence of related SCARs. Notably, previous studies have identified the strong genetic associations between *HLA-B*58:01* and allopurinol-induced SCAR([Hung et al., 2005](#B11)) and *HLA-B*15:02* and carbamazepine-induced SJS/TEN ([Chung et al., 2004](#B8)). Besides, *HLA-B*57:01* ([Mallal et al., 2008](#B18)) and *HLA-B*13:01* ([Zhang et al., 2013](#B33)) have also been found strong associations with abacavir hypersensitivity and dapsone hypersensitivity syndrome, respectively. The findings of these strong genetic associations have been applied to clinical screening to prevent the cases of severe drug hypersensitivity ([Wang et al., 2021](#B28)).

However, the genetic factors that may be relevant to AMX-induced SCAR (AMX-SCAR) remain unclear. The associations between genetic markers and beta-lactam antibiotics-induced adverse drug reactions have been reported. Yang et al. previously reported that *HLA-DRB* allele was found to be weakly associated with IgE-mediated-type hypersensitivity to penicillin in Chinese population ([Yang et al., 2006](#B32)). *HLA-DRB1*10:01* allele was found to be associated with penicillin-induced immediate reactions ([Nicoletti et al., 2021](#B20))and *HLA-B*55:01* was related to penicillin-induced delayed reactions by genome-wide association study ([Krebs et al., 2020](#B16)) in European population. Romano et al. reported that *HLA-DRB3*02:02* allele was strongly associated with penicillin-induced delayed hypersensitivity compared to immediate reaction in Italian population ([Romano et al., 2022](#B23)). Besides, Wattanachai et al. recently reported that six HLA alleles including *HLA-A*01:01, HLA-B*50:01, HLA-C*06:02, HLA-DRB1*15:01, HLA-DQA1*03:01,* and *HLA-DQB1*03:02*, were associated with the increased risk of beta-lactam antibiotics-related SCARs in Thai population ([Wattanachai et al., 2023](#B30)). To investigate the genetic predisposition to hypersensitivity reactions induced by AMX antibiotic to prevent patients from the risk of AMX exposure and resume therapy with other falsely implicated drugs in Chinese population, we carried out a case-control association study of patients with AMX-induced SCAR by using whole-exome sequencing (WES) and then further validated the results in AMX-tolerant population. Furthermore, molecular docking modeling was employed to elucidate the potential key amino acid residues of HLA molecules bound to AMX and the specific conformation pattern of ligand-receptor binding.

## Materials and methods

### Subject enrollment

Four cases with AMX-induced SCAR were recruited from Huashan Hospital of Fudan University in Shanghai, China from January 2006 to December 2022. Phenotypes of SCAR were classified according to the consensus definitions of RegiSCAR study criteria ([Auquier-Dunant et al., 2002](#B2); [Cacoub et al., 2011](#B5); [Wang et al., 2018](#B29)). The drug causality for each enrolled case was determined by the dermatologist using the algorithm of drug causality for epidermal necrolysis (ALDEN) score published by the RegiSCAR study group or the Naranjo algorithm ([Naranjo et al., 1981](#B19); [Sassolas et al., 2010](#B24); [Kardaun et al., 2013](#B14)). Data on patients’ demography, clinical patterns, causative drugs, treatments, and clinical outcomes were collected from patients’ case notes. Patients’ files with incomplete data were excluded. For general population and drug-tolerant controls, we collected clinical data and DNA samples of 1,000 individuals in general population without history of adverse drug reactions as the WES control group as well as 100 drug-tolerant patients who had received amoxicillin for several times and the cumulative duration was for more than 3 months without evidence of adverse reactions were enrolled as tolerant controls ([Supplementary Table S1](#s11)). The latency period between the intake of the drug and the onset of the skin eruption was recorded. Written informed consent has been collected from all subjects, and the study protocol was approved by the institutional review board of Department of Dermatology, Huashan Hospital.

### HLA genotyping

The whole-exome sequencing (WES) method include steps such as DNA sample extraction, library building, and sequencing, following the methods reported previously by ([Jiang et al., 2022](#B13)). Typing of HLA alleles were performed using HLA-HD version 1.3.0 ([Kawaguchi et al., 2017](#B15)) with sequence mapped to the reference sequence of six-digit HLA alleles from the IMGT/HLA database in default settings ([Barker et al., 2023](#B3)). High-resolution four-digit HLA-B allele genotyping was performed by Weihe Biotechnology (Jiangsu, China) using the polymerase chain reaction-sequence based typing (PCR-SBT) method.

### Molecular modeling

We conducted molecular docking studies via the Maestro (version 2019) software to investigate the interaction patterns between AMX and HLA protein. The protein and nucleotide sequences of HLA-B*15:01, HLA-B*46:01, HLA-B*15:02, and HLA-B*35:05 were obtained from the IMGT/HLA database. The structure of these four HLA protein molecules were obtained from SWISS-MODEL server, and HLA-B*15:01, HLA-B*15:02, HLA-B*35:05 and HLA-B*46:01 were modeled based on the following crystal structures, respectively: 5TXS (100% identity), 6UZM (100% identity), 7SIF (100% identity) and 4LCY (100% identity). AMX molecule was used as ligands and HLA molecule was used as receptors for docking modeling. The receptor molecule obtained the dominant conformation after repairing the missing residues and minimizing the energy using Maestro software. Ligprep module was used to generate 32 3D conformations of AMX molecules, and the optimal conformation was selected for modeling. The above receptor and ligand molecules were protonated at human physiological PH value. All parameters were set to the software default parameters. The docking simulation of the optimized ligand receptor was performed, and the docking models of the top 20 outputs were displayed. According to the docking scores and results, the optimal binding mode of AMX docking with each HLA molecule was selected.

### Statistical analysis

The statistical analyses were carried out using R version 4.2.3 (The R Foundation for Statistical Computing, Vienna, Austria). Individuals who tested positive for at least one copy of a specific allele were considered as carriers. Fisher exact test was applied to compare the carrier frequency between specific HLA allele for AMX-SCAR patients and population controls as well as AMX-tolerant controls. Overall odd ratios (ORs) with corresponding 95% confidence intervals (CIs) were calculated to quantify the association. When zero cell counts were included, ORs were calculated using Haldane’s modification, which adds 0.5 to all cells to accommodate possible zero counts to reduce bias in estimating ORs. In the HLA protein sequence analysis, the χ^2^ test and Fisher’s exact test were used to analyse the polymorphic position and amino acid residue between the AMX-SCAR group and two control groups, respectively. A two-tailed *p*-value of <0.05 was considered as statistically significant.

## Results

### Demographic and clinical characteristics of patients

Herein, we described four patients with AMX-SCAR recruited from the dermatology ward from Huashan Hospital of Fudan University between 2010 and 2021, who have been diagnosed with SJS, TEN, DRESS and Acute Erythroderma, respectively ([Table 1](#T1)). One out of four patients underwent oral AMX monotherapy, and the remaining three were on oral AMX combined with other drugs, for which we have ruled out other possible combined drugs as the culprit based on the medical history and ALDEN score or Naranjo algorithm. Latency period ranged from 1 to 9 days, with an average of 6.5 ± 2.6 days. Among them, three out of four patients had fever (≥37.5°C), 2 cases presented increased white blood cell count (≥0.5×10^9^/L), 1 case exhibited drug induced liver injury, and 2 cases had a history of penicillin allergy implying cross-reactivity with other penicillins.

### TABLE 1.

| Gender/age | SCAR | Fever | WBC | EP | AST | ALT | ALP | GGT | Latency (days) | Allergic history | Combined drug | Hospitalization time (days) | HLA-B |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| M/60 | DRESS | 38°C | 31.7↑ | 3.6↑ | 735↑ | 786↑ | 1,448↑ | 1,021↑ | 5 | - | Niuhuang Jiedu Tablet | 11 | 13:01/ 15:01 |
| F/28 | TEN | 39.5°C | 13.0 ↑ | 0.5 | 17 | 35 | 27 | 11 | 9 | - | Acetaminophen | 14 | 15:01 /51:01 |
| F/49 | Acute Erythroderma | - | 9.9 | 0 | 14 | 22 | 46 | 13 | 1 | Penicillin | - | 6 | 15:01 /37:01 |
| F/56 | SJS | 37.5°C | 12.1 ↑ | 0.8 ↑ | 11 | 12 | 28 | 7 | 4 | Penicillin | chlorphenazine | 9 | 39:01/35:05 |

Table 1 Caption: Clinical characteristics and HLA-B genotypes of patients with amoxicillin-induced severe cutaneous adverse reactions.

### Associations of HLA alleles and amino acids with AMX-SCAR

The HLA analysis detected three carriers of *HLA-B*15:01* in four patients with AMX-SCAR, and the carrier frequency of *HLA-B*15:01* is 10.7% in 1,000 population controls and 11.0% in 100 AMX-tolerant controls, implying a significant association of HLA-B with AMX-SCAR ([Supplementary Table S2](#s11), [Table 2](#T2)). Previous studies suggested that amino acid motifs rather than classical alleles were the major drives conferring disease risk ([Illing et al., 2012](#B12); [Raychaudhuri et al., 2012](#B22)). In the present study, three out of four patients harbored *HLA-B*15:01*, suggesting it may play an important role in the pathogenesis of AMX-SCAR ([Table 1](#T1)). Then, we further analyzed the frequencies of amino acids in HLA-B proteins in cases and 100 AMX-tolerant controls, detecting 28 amino acids significantly enriched in patients (*p* < 0.05). Among them, S140 (*p* = 5.18 × 10^−4^) demonstrated the strongest association, followed by A48 (*p* = 4.45 × 10^−3^), L187 (*p* = 4.45 × 10^−3^), R121 (*p* = 6.84 × 10^−3^), W14 (*p* = 6.84 × 10^−3^), V17 (*p* = 6.84 × 10^−3^), and W180 (*p* = 7.34 × 10^−3^) ([Supplementary Table S3](#s11)).

### TABLE 2.

| HLA-B genotype | Carrier, No. (%) |  | P | OR (95% CI) |
| --- | --- | --- | --- | --- |
| AMX-SCAR (4) | AMX-tolerant controls (100) |  |  |  |
| B*13:01 | 1 (25) | 10 (10) | 0.3650 | 2.95 (0.05–41.08) |
| B*15:01 | 3 (75) | 11 (11) | 7.34 × 10 −3 | 22.9 (1.68–1275.67) |
| B*35:05 | 1 (25) | 2 (2) | 0.1120 | 14.99 (0.21–375.00) |
| B*37:01 | 1 (25) | 4 (4) | 0.1810 | 7.64 (0.12–125.78) |
| B*39:01 | 1 (25) | 2 (2) | 0.1120 | 14.99 (0.21–375.00) |
| B*51:01 | 1 (25) | 11 (11) | 0.3920 | 2.66 (0.05–36.68) |

Table 2 Caption: Frequencies of HLA alleles in patients with amoxicillin-induced severe cutaneous adverse reactions and amoxicillin-tolerant controls.

### In silico analysis and molecular docking

Given the strong associations of AMX-SCAR with specific amino acids of HLA-B, molecular modeling was conducted to further confirm the interaction between AMX and HLA-B proteins to eliminate false-positive results. The docking results revealed that amino acid at position 140 and 180 in HLA-B play a crucial role in binding to AMX. In HLA-B*15:01, S140, K170, W171, and E176 can form hydrogen bonds, R121 and W180 formed cation-π hydrophobic interactions with AMX. These combined effects gave rise to relatively low steric hindrance, which was conducive for AMX binding in the pocket with a low-energy conformation ([Figure 1A](#F1)). In HLA-B*15:02 (belongs to the same serotype and differs from HLA-B*15:01 by only 5 amino acids), W180 became L180, which greatly increased steric hindrance ([Figure 1B](#F1)). In HLA-B*35:05 (presents in patient 4), although the conversion of W180 to L180 slightly weakens the binding to the AMX molecule, S140 remains to be a key amino acid in forming hydrogen bonds. In addition, W171 and N104 also form hydrogen bonds, and this binding mode allows AMX to be surrounded by polar amino acids, promoting the formation of stable complexes in the dynamic binding process ([Figure 1C](#F1)). In HLA-B*46:01 (shows a high frequency in tolerant controls), AMX’s phenolic head is surrounded by polar amino acids such as E87, and the increased steric hindrance prevents AMX from remaining in a low-energy conformation in the pocket. Consequently, AMX loses the hydrophobic effect with W180 and the polar effect with S140 ([Figure 1D](#F1)). Overall, AMX tends to bind with S140 in the antigen-binding cleft of HLA-B, differing from unrelated HLA molecules HLA-B*46:01, which indicates that predisposition to AMX-SCAR is related to one essential amino acid residue in the peptide binding pocket of HLA-B.

### FIGURE 1.

![FIGURE 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/86f7/11165025/539282f31be2/fphar-15-1400239-g001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=11165025_fphar-15-1400239-g001.jpg)

Binding modes of Amoxicillin in four HLA proteins and the sequence alignment. The binding modes of amoxicillin (AMX) in HLA-B*15:01 (A), HLA-B*15:02 (B), HLA-B*35:05 (C) and HLA -B*46:01 (D) are shown in the green cartoon representation. AMX (yellow carbons) and HLA residues (green carbons or pink carbons) that involve ligand bindings are shown in the stick representation. Hydrogen bonds are displayed as red dashed lines, π-π hydrophobic interactions and cation-π hydrophobic interactions are shown as blue dashed lines. For clarity, two-dimensional schematic representations of the AMX-HLA interactions are shown next to the three-dimensional models. In the sequence alignment (E) results, identical residues are shown with a red background; partially conserved residues are shown as white letter with a red background; Non-conserved residues are displayed with a green background, with the first type residue colored in yellow and the second type residue colored in black, and these residues are denoted as red stars at the bottom.

### Genetic predictors of AMX-induced SCAR

For HLA-B proteins, amino acid residues S140, L187, A48, R121, W14, V17, I306, T329, and W180 are encoded by coding sequence variants C419, CT559-560, G142, G363, G41, G49, A916, A985, and T538, respectively. The distribution of these genetic variants in the AMX-SCAR and AMX-tolerant groups is consistent with amino acids. From the perspective of genetic variants as predictors of AMX-SCAR, where C419, CT559-560, G142, G363, G41, G49, A916, and A985 showed the highest sensitivity (up to 100%), followed by T538 with 75%. In terms of specificity, variant T538 exhibited the highest value (up to 89%), followed by C419 (87%), CT559-560 and G142 (76%), G363, G41, and G49 (73%), and A916 and A985 (72%). According to the current results, S140 of HLA-B or its genetic allele C419 (also known as rs4997052), present in all 4 subjects with AMX-SCAR but only 13 of the 100 AMX-tolerant controls, can serve as a clinical predictor of AMX-induced SCAR with a sensitivity of 100% and specificity of 87% ([Table 3](#T3)).

### TABLE 3.

| Predictor | Carrier, No. (%) |  | P | OR (95% CI) | Sensitivity (%) | Specificity (%) |
| --- | --- | --- | --- | --- | --- | --- |
| AMX-SCAR (4) | AMX-tolerant controls (100) |  |  |  |  |  |
| HLA-B*15:01 | 3 (75) | 11 (11) | 7.34 × 10−3 | 22.9 (1.64–1262.54) | 75 | 89 |
| T538 | 3 (75) | 11 (11) | 7.34 × 10−3 | 22.9 (1.68–1275.67) | 75 | 89 |
| C419 | 4 (100) | 13 (13) | 5.18 × 10−4 | 53.5 (2.67–1072.29) | 100 | 87 |

Table 3 Caption: Predictive efficiency of pharmacogenetic predictors for amoxicillin-induced severe cutaneous adverse reactions.

## Discussion

Although the incidence of SCAR is low, it may cause high mortality and permanent disability ([Duong et al., 2017](#B9)). Amoxicillin (AMX) is the most widely used penicillin all over the world, and is a World Health Organization (WHO)-designated core access antibiotic ([Sharland et al., 2018](#B25)). However, AMX is one of the leading causative drugs for life-threating SCAR. The results of the clinical characteristics of patients with AMX-SCAR showed that most of the patients were female, which was consistent with previous studies of the frequency of drug allergies ([Blumenthal et al., 2019](#B4)). Patients with the DRESS phenotype in this study had a latency period of 8 days, which is shorter than the typically reported for DRESS syndrome (3–6 weeks). Um et al. found that the latency period in antibiotic-induced DRESS was significantly shorter than in anticonvulsant-induced DRESS([Um et al., 2010](#B27)). Drug-related skin changes generally begin to appear after 1–3 weeks of continuous drug use, and the latent period for cutaneous adverse drug reactions with SJS/TEN phenotype in this study was 8.5 days on average, which was consistent with recent reports ([Lee et al., 2023](#B17); [Pejcic et al., 2023](#B21)).

Previous studies on the association between HLA and various diseases have suggested that amino acid variants, rather than classical alleles, are the main drivers of disease risk. Illing et al. reported that Asp114 and Ser116 in HLA-B*57:01 are key amino acids involved in the pathogenesis of abacavir hypersensitivity syndrome (AHS) ([Illing et al., 2012](#B12)). Raychaudhuri et al. revealed that five amino acids in three HLA proteins could explain most of the association between HLA and the risk of seropositive rheumatoid arthritis ([Raychaudhuri et al., 2012](#B22)). More recently, Chu et al. found that four amino acid positions could account for the associations of HLA with Graves’ disease (GD) in Han Chinese ([Chu et al., 2018](#B7)). In present study, key residue S140 in the antigen-binding pocket of HAL-B was the main factor interacting with AMX and had a notably higher risk of developing AMX-SCAR.

To date, genetic susceptibilities to AMX antibiotic-induced hypersensitivity remain unclear. There still lacks a strong and applicable genetic marker to prevent these adverse events. Through genetic association and molecular docking analysis, we identified one amino acid S140 of HLA-B or its corresponding genetic allele C419 as a potential predictor of AMX-SCAR. Prospective screening of the variant C419, coupled with an alternative drug treatment for carriers, may significantly decrease the incidence of AMX-SCAR in Han Chinese. The novelty of our work is also manifested in the first discovery of the causal and mechanistic linkage between S140 of HLA-B and AMX-SCAR, which can provide important clues for subsequent research. Nevertheless, the sample size of this study was limited due to challenges in recruiting SCAR cases with a clear cause, future studies with a larger sample size are required to confirm the fidelity of this genetic marker and explore the underlying mechanism.

## Acknowledgments

We also thank the support of National Super Computing Center in Zhengzhou for providing computational resources for this study. The funder had no role in study design, data collection, data analysis, interpretation, or writing of this report.

## Funding Statement

The author(s) declare that financial support was received for the research, authorship, and/or publication of this article. This work was supported by the National Natural Science Foundation of China (grant numbers 82173422, 81872525, 81472873, 31972880, 31371274); National Key Research and Development Program (grant numbers 2021YFC2700800); Shanghai Municipal Commission of Science and Technology Research Project (grant numbers 19JC1411001); and National Key Clinical Specialty Construction Project Department of Allergy and Immunology, Huashan Hospital.

## Data availability statement

The data presented in the study are deposited in the NCBI Sequence Read Archive (SRA) repository: [https://www.ncbi.nlm.nih.gov/sra/](https://www.ncbi.nlm.nih.gov/sra/), accession number PRJNA1110861.

## Ethics statement

The studies involving humans were approved by Department of Dermatology, Huashan Hospital. The studies were conducted in accordance with the local legislation and institutional requirements. The participants provided their written informed consent to participate in this study.

## Author contributions

TW: Data curation, Formal Analysis, Software, Validation, Writing–original draft. JY: Data curation, Investigation, Writing–original draft. FY: Data curation, Investigation, Writing–original draft. YC: Methodology, Software, Visualization, Writing–original draft. ZH: Data curation, Methodology, Visualization, Writing–original draft. BL: Validation, Writing–original draft. LY: Funding acquisition, Resources, Writing–review and editing. QX: Formal Analysis, Funding acquisition, Project administration, Supervision, Writing–review and editing. XL: Funding acquisition, Validation, Writing–review and editing.

## Conflict of interest

The authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.

## Publisher’s note

All claims expressed in this article are solely those of the authors and do not necessarily represent those of their affiliated organizations, or those of the publisher, the editors and the reviewers. Any product that may be evaluated in this article, or claim that may be made by its manufacturer, is not guaranteed or endorsed by the publisher.

## Supplementary material

The Supplementary Material for this article can be found online at: [https://www.frontiersin.org/articles/10.3389/fphar.2024.1400239/full#supplementary-material](https://www.frontiersin.org/articles/10.3389/fphar.2024.1400239/full#supplementary-material)

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

The data presented in the study are deposited in the NCBI Sequence Read Archive (SRA) repository: [https://www.ncbi.nlm.nih.gov/sra/](https://www.ncbi.nlm.nih.gov/sra/), accession number PRJNA1110861.

### Supplementary Materials

### Data Availability Statement

The data presented in the study are deposited in the NCBI Sequence Read Archive (SRA) repository: [https://www.ncbi.nlm.nih.gov/sra/](https://www.ncbi.nlm.nih.gov/sra/), accession number PRJNA1110861.

## References

1. Aouam K., Fredj Nadia B., Amel C., Naceur B. (2010). Amoxicillin-induced hypersensitivity after DRESS to carbamazepine. World Allergy Organ J. 3, 220–222. 10.1097/WOX.0b013e3181eab930  [DOI](https://doi.org/10.1097/WOX.0b013e3181eab930) | [PMC free article](/articles/PMC3651104/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23282653/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=World%20Allergy%20Organ%20J.&title=Amoxicillin-induced%20hypersensitivity%20after%20DRESS%20to%20carbamazepine&author=K.%20Aouam&author=B.%20Fredj%20Nadia&author=C.%20Amel&author=B.%20Naceur&volume=3&publication_year=2010&pages=220-222&pmid=23282653&doi=10.1097/WOX.0b013e3181eab930&)

2. Auquier-Dunant A., Mockenhaupt M., Naldi L., Correia O., Schroder W., Roujeau J. C., et al. (2002). Correlations between clinical patterns and causes of erythema multiforme majus, Stevens-Johnson syndrome, and toxic epidermal necrolysis: results of an international prospective study. Arch. Dermatol 138, 1019–1024. 10.1001/archderm.138.8.1019  [DOI](https://doi.org/10.1001/archderm.138.8.1019) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12164739/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arch.%20Dermatol&title=Correlations%20between%20clinical%20patterns%20and%20causes%20of%20erythema%20multiforme%20majus,%20Stevens-Johnson%20syndrome,%20and%20toxic%20epidermal%20necrolysis:%20results%20of%20an%20international%20prospective%20study&author=A.%20Auquier-Dunant&author=M.%20Mockenhaupt&author=L.%20Naldi&author=O.%20Correia&author=W.%20Schroder&volume=138&publication_year=2002&pages=1019-1024&pmid=12164739&doi=10.1001/archderm.138.8.1019&)

3. Barker D. J., Maccari G., Georgiou X., Cooper M. A., Flicek P., Robinson J., et al. (2023). The IPD-IMGT/HLA database. Nucleic Acids Res. 51, D1053–D1060. 10.1093/nar/gkac1011  [DOI](https://doi.org/10.1093/nar/gkac1011) | [PMC free article](/articles/PMC9825470/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36350643/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&title=The%20IPD-IMGT/HLA%20database&author=D.%20J.%20Barker&author=G.%20Maccari&author=X.%20Georgiou&author=M.%20A.%20Cooper&author=P.%20Flicek&volume=51&publication_year=2023&pages=D1053-D1060&pmid=36350643&doi=10.1093/nar/gkac1011&)

4. Blumenthal K. G., Peter J. G., Trubiano J. A., Phillips E. J. (2019). Antibiotic allergy. Lancet 393, 183–198. 10.1016/S0140-6736(18)32218-9  [DOI](https://doi.org/10.1016/S0140-6736(18)32218-9) | [PMC free article](/articles/PMC6563335/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30558872/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Antibiotic%20allergy&author=K.%20G.%20Blumenthal&author=J.%20G.%20Peter&author=J.%20A.%20Trubiano&author=E.%20J.%20Phillips&volume=393&publication_year=2019&pages=183-198&pmid=30558872&doi=10.1016/S0140-6736(18)32218-9&)

5. Cacoub P., Musette P., Descamps V., Meyer O., Speirs C., Finzi L., et al. (2011). The DRESS syndrome: a literature review. Am. J. Med. 124, 588–597. 10.1016/j.amjmed.2011.01.017  [DOI](https://doi.org/10.1016/j.amjmed.2011.01.017) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21592453/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Med.&title=The%20DRESS%20syndrome:%20a%20literature%20review&author=P.%20Cacoub&author=P.%20Musette&author=V.%20Descamps&author=O.%20Meyer&author=C.%20Speirs&volume=124&publication_year=2011&pages=588-597&pmid=21592453&doi=10.1016/j.amjmed.2011.01.017&)

6. Chessman D., Kostenko L., Lethborg T., Purcell A. W., Williamson N. A., Chen Z., et al. (2008). Human leukocyte antigen class I-restricted activation of CD8+ T cells provides the immunogenetic basis of a systemic drug hypersensitivity. Immunity 28, 822–832. 10.1016/j.immuni.2008.04.020  [DOI](https://doi.org/10.1016/j.immuni.2008.04.020) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18549801/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Immunity&title=Human%20leukocyte%20antigen%20class%20I-restricted%20activation%20of%20CD8+%20T%20cells%20provides%20the%20immunogenetic%20basis%20of%20a%20systemic%20drug%20hypersensitivity&author=D.%20Chessman&author=L.%20Kostenko&author=T.%20Lethborg&author=A.%20W.%20Purcell&author=N.%20A.%20Williamson&volume=28&publication_year=2008&pages=822-832&pmid=18549801&doi=10.1016/j.immuni.2008.04.020&)

7. Chu X., Yang M., Song Z. J., Dong Y., Li C., Shen M., et al. (2018). Fine mapping MHC associations in Graves' disease and its clinical subtypes in Han Chinese. J. Med. Genet. 55, 685–692. 10.1136/jmedgenet-2017-105146  [DOI](https://doi.org/10.1136/jmedgenet-2017-105146) | [PMC free article](/articles/PMC6161647/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29987165/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Med.%20Genet.&title=Fine%20mapping%20MHC%20associations%20in%20Graves'%20disease%20and%20its%20clinical%20subtypes%20in%20Han%20Chinese&author=X.%20Chu&author=M.%20Yang&author=Z.%20J.%20Song&author=Y.%20Dong&author=C.%20Li&volume=55&publication_year=2018&pages=685-692&pmid=29987165&doi=10.1136/jmedgenet-2017-105146&)

8. Chung W. H., Hung S. I., Hong H. S., Hsih M. S., Yang L. C., Ho H. C., et al. (2004). Medical genetics: a marker for Stevens-Johnson syndrome. Nature 428, 486. 10.1038/428486a  [DOI](https://doi.org/10.1038/428486a) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15057820/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Medical%20genetics:%20a%20marker%20for%20Stevens-Johnson%20syndrome&author=W.%20H.%20Chung&author=S.%20I.%20Hung&author=H.%20S.%20Hong&author=M.%20S.%20Hsih&author=L.%20C.%20Yang&volume=428&publication_year=2004&pages=486&pmid=15057820&doi=10.1038/428486a&)

9. Duong T. A., Valeyrie-Allanore L., Wolkenstein P., Chosidow O. (2017). Severe cutaneous adverse reactions to drugs. Lancet 390, 1996–2011. 10.1016/S0140-6736(16)30378-6  [DOI](https://doi.org/10.1016/S0140-6736(16)30378-6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28476287/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Severe%20cutaneous%20adverse%20reactions%20to%20drugs&author=T.%20A.%20Duong&author=L.%20Valeyrie-Allanore&author=P.%20Wolkenstein&author=O.%20Chosidow&volume=390&publication_year=2017&pages=1996-2011&pmid=28476287&doi=10.1016/S0140-6736(16)30378-6&)

10. Girelli F., Bernardi S., Gardelli L., Bassi B., Parente G., Dubini A., et al. (2013). A new case of DRESS syndrome induced by sulfasalazine and triggered by amoxicillin. Case Rep. Rheumatol. 2013, 409152. 10.1155/2013/409152  [DOI](https://doi.org/10.1155/2013/409152) | [PMC free article](/articles/PMC3723001/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23936716/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Case%20Rep.%20Rheumatol.&title=A%20new%20case%20of%20DRESS%20syndrome%20induced%20by%20sulfasalazine%20and%20triggered%20by%20amoxicillin&author=F.%20Girelli&author=S.%20Bernardi&author=L.%20Gardelli&author=B.%20Bassi&author=G.%20Parente&volume=2013&publication_year=2013&pages=409152&pmid=23936716&doi=10.1155/2013/409152&)

11. Hung S. I., Chung W. H., Liou L. B., Chu C. C., Lin M., Huang H. P., et al. (2005). HLA-B*5801 allele as a genetic marker for severe cutaneous adverse reactions caused by allopurinol. Proc. Natl. Acad. Sci. U. S. A. 102, 4134–4139. 10.1073/pnas.0409500102  [DOI](https://doi.org/10.1073/pnas.0409500102) | [PMC free article](/articles/PMC554812/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15743917/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A.&title=HLA-B*5801%20allele%20as%20a%20genetic%20marker%20for%20severe%20cutaneous%20adverse%20reactions%20caused%20by%20allopurinol&author=S.%20I.%20Hung&author=W.%20H.%20Chung&author=L.%20B.%20Liou&author=C.%20C.%20Chu&author=M.%20Lin&volume=102&publication_year=2005&pages=4134-4139&pmid=15743917&doi=10.1073/pnas.0409500102&)

12. Illing P. T., Vivian J. P., Dudek N. L., Kostenko L., Chen Z., Bharadwaj M., et al. (2012). Immune self-reactivity triggered by drug-modified HLA-peptide repertoire. Nature 486, 554–558. 10.1038/nature11147  [DOI](https://doi.org/10.1038/nature11147) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22722860/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Immune%20self-reactivity%20triggered%20by%20drug-modified%20HLA-peptide%20repertoire&author=P.%20T.%20Illing&author=J.%20P.%20Vivian&author=N.%20L.%20Dudek&author=L.%20Kostenko&author=Z.%20Chen&volume=486&publication_year=2012&pages=554-558&pmid=22722860&doi=10.1038/nature11147&)

13. Jiang M., Yang F., Zhang L., Xu D., Jia Y., Cheng Y., et al. (2022). Unique motif shared by HLA-B*59:01 and HLA-B*55:02 is associated with methazolamide-induced Stevens-Johnson syndrome and toxic epidermal necrolysis in Han Chinese. J. Eur. Acad. Dermatol Venereol. 36, 873–880. 10.1111/jdv.17980  [DOI](https://doi.org/10.1111/jdv.17980) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35122707/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Eur.%20Acad.%20Dermatol%20Venereol.&title=Unique%20motif%20shared%20by%20HLA-B*59:01%20and%20HLA-B*55:02%20is%20associated%20with%20methazolamide-induced%20Stevens-Johnson%20syndrome%20and%20toxic%20epidermal%20necrolysis%20in%20Han%20Chinese&author=M.%20Jiang&author=F.%20Yang&author=L.%20Zhang&author=D.%20Xu&author=Y.%20Jia&volume=36&publication_year=2022&pages=873-880&pmid=35122707&doi=10.1111/jdv.17980&)

14. Kardaun S. H., Sekula P., Valeyrie-Allanore L., Liss Y., Chu C. Y., Creamer D., et al. (2013). Drug reaction with eosinophilia and systemic symptoms (DRESS): an original multisystem adverse drug reaction. Results from the prospective RegiSCAR study. Br. J. Dermatol 169, 1071–1080. 10.1111/bjd.12501  [DOI](https://doi.org/10.1111/bjd.12501) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23855313/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Dermatol&title=Drug%20reaction%20with%20eosinophilia%20and%20systemic%20symptoms%20(DRESS):%20an%20original%20multisystem%20adverse%20drug%20reaction.%20Results%20from%20the%20prospective%20RegiSCAR%20study&author=S.%20H.%20Kardaun&author=P.%20Sekula&author=L.%20Valeyrie-Allanore&author=Y.%20Liss&author=C.%20Y.%20Chu&volume=169&publication_year=2013&pages=1071-1080&pmid=23855313&doi=10.1111/bjd.12501&)

15. Kawaguchi S., Higasa K., Shimizu M., Yamada R., Matsuda F. (2017). HLA-HD: an accurate HLA typing algorithm for next-generation sequencing data. Hum. Mutat. 38, 788–797. 10.1002/humu.23230  [DOI](https://doi.org/10.1002/humu.23230) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28419628/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum.%20Mutat.&title=HLA-HD:%20an%20accurate%20HLA%20typing%20algorithm%20for%20next-generation%20sequencing%20data&author=S.%20Kawaguchi&author=K.%20Higasa&author=M.%20Shimizu&author=R.%20Yamada&author=F.%20Matsuda&volume=38&publication_year=2017&pages=788-797&pmid=28419628&doi=10.1002/humu.23230&)

16. Krebs K., Bovijn J., Zheng N., Lepamets M., Censin J. C., Jurgenson T., et al. (2020). Genome-wide study identifies association between HLA-B(*)55:01 and self-reported penicillin allergy. Am. J. Hum. Genet. 107, 612–621. 10.1016/j.ajhg.2020.08.008  [DOI](https://doi.org/10.1016/j.ajhg.2020.08.008) | [PMC free article](/articles/PMC7536643/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32888428/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Hum.%20Genet.&title=Genome-wide%20study%20identifies%20association%20between%20HLA-B(*)55:01%20and%20self-reported%20penicillin%20allergy&author=K.%20Krebs&author=J.%20Bovijn&author=N.%20Zheng&author=M.%20Lepamets&author=J.%20C.%20Censin&volume=107&publication_year=2020&pages=612-621&pmid=32888428&doi=10.1016/j.ajhg.2020.08.008&)

17. Lee E. Y., Knox C., Phillips E. J. (2023). Worldwide prevalence of antibiotic-associated stevens-johnson syndrome and toxic epidermal necrolysis: a systematic review and meta-analysis. JAMA Dermatol 159, 384–392. 10.1001/jamadermatol.2022.6378  [DOI](https://doi.org/10.1001/jamadermatol.2022.6378) | [PMC free article](/articles/PMC9932945/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36790777/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA%20Dermatol&title=Worldwide%20prevalence%20of%20antibiotic-associated%20stevens-johnson%20syndrome%20and%20toxic%20epidermal%20necrolysis:%20a%20systematic%20review%20and%20meta-analysis&author=E.%20Y.%20Lee&author=C.%20Knox&author=E.%20J.%20Phillips&volume=159&publication_year=2023&pages=384-392&pmid=36790777&doi=10.1001/jamadermatol.2022.6378&)

18. Mallal S., Phillips E., Carosi G., Molina J. M., Workman C., Tomazic J., et al. (2008). HLA-B*5701 screening for hypersensitivity to abacavir. N. Engl. J. Med. 358, 568–579. 10.1056/NEJMoa0706135  [DOI](https://doi.org/10.1056/NEJMoa0706135) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18256392/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%20Engl.%20J.%20Med.&title=HLA-B*5701%20screening%20for%20hypersensitivity%20to%20abacavir&author=S.%20Mallal&author=E.%20Phillips&author=G.%20Carosi&author=J.%20M.%20Molina&author=C.%20Workman&volume=358&publication_year=2008&pages=568-579&pmid=18256392&doi=10.1056/NEJMoa0706135&)

19. Naranjo C. A., Busto U., Sellers E. M., Sandor P., Ruiz I., Roberts E. A., et al. (1981). A method for estimating the probability of adverse drug reactions. Clin. Pharmacol. Ther. 30, 239–245. 10.1038/clpt.1981.154  [DOI](https://doi.org/10.1038/clpt.1981.154) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7249508/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=A%20method%20for%20estimating%20the%20probability%20of%20adverse%20drug%20reactions&author=C.%20A.%20Naranjo&author=U.%20Busto&author=E.%20M.%20Sellers&author=P.%20Sandor&author=I.%20Ruiz&volume=30&publication_year=1981&pages=239-245&pmid=7249508&doi=10.1038/clpt.1981.154&)

20. Nicoletti P., Carr D. F., Barrett S., McEvoy L., Friedmann P. S., Shear N. H., et al. (2021). Beta-lactam-induced immediate hypersensitivity reactions: a genome-wide association study of a deeply phenotyped cohort. J. Allergy Clin. Immunol. 147, 1830–1837 e15. 10.1016/j.jaci.2020.10.004  [DOI](https://doi.org/10.1016/j.jaci.2020.10.004) | [PMC free article](/articles/PMC8100096/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33058932/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Allergy%20Clin.%20Immunol.&title=Beta-lactam-induced%20immediate%20hypersensitivity%20reactions:%20a%20genome-wide%20association%20study%20of%20a%20deeply%20phenotyped%20cohort&author=P.%20Nicoletti&author=D.%20F.%20Carr&author=S.%20Barrett&author=L.%20McEvoy&author=P.%20S.%20Friedmann&volume=147&publication_year=2021&pages=1830-1837%20e15&pmid=33058932&doi=10.1016/j.jaci.2020.10.004&)

21. Pejcic A. V., Milosavljevic M. N., Folic M., Fernandes D., Bentes J., Djesevic M., et al. (2023). Amoxicillin-associated Stevens-Johnson syndrome or toxic epidermal necrolysis: systematic review. J. Chemother. 35, 75–86. 10.1080/1120009X.2022.2051128  [DOI](https://doi.org/10.1080/1120009X.2022.2051128) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35285784/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Chemother.&title=Amoxicillin-associated%20Stevens-Johnson%20syndrome%20or%20toxic%20epidermal%20necrolysis:%20systematic%20review&author=A.%20V.%20Pejcic&author=M.%20N.%20Milosavljevic&author=M.%20Folic&author=D.%20Fernandes&author=J.%20Bentes&volume=35&publication_year=2023&pages=75-86&pmid=35285784&doi=10.1080/1120009X.2022.2051128&)

22. Raychaudhuri S., Sandor C., Stahl E. A., Freudenberg J., Lee H. S., Jia X., et al. (2012). Five amino acids in three HLA proteins explain most of the association between MHC and seropositive rheumatoid arthritis. Nat. Genet. 44, 291–296. 10.1038/ng.1076  [DOI](https://doi.org/10.1038/ng.1076) | [PMC free article](/articles/PMC3288335/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22286218/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet.&title=Five%20amino%20acids%20in%20three%20HLA%20proteins%20explain%20most%20of%20the%20association%20between%20MHC%20and%20seropositive%20rheumatoid%20arthritis&author=S.%20Raychaudhuri&author=C.%20Sandor&author=E.%20A.%20Stahl&author=J.%20Freudenberg&author=H.%20S.%20Lee&volume=44&publication_year=2012&pages=291-296&pmid=22286218&doi=10.1038/ng.1076&)

23. Romano A., Oussalah A., Chery C., Gueant-Rodriguez R. M., Gaeta F., Cornejo-Garcia J. A., et al. (2022). Next-generation sequencing and genotype association studies reveal the association of HLA-DRB3*02:02 with delayed hypersensitivity to penicillins. Allergy 77, 1827–1834. 10.1111/all.15147  [DOI](https://doi.org/10.1111/all.15147) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34687232/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Allergy&title=Next-generation%20sequencing%20and%20genotype%20association%20studies%20reveal%20the%20association%20of%20HLA-DRB3*02:02%20with%20delayed%20hypersensitivity%20to%20penicillins&author=A.%20Romano&author=A.%20Oussalah&author=C.%20Chery&author=R.%20M.%20Gueant-Rodriguez&author=F.%20Gaeta&volume=77&publication_year=2022&pages=1827-1834&pmid=34687232&doi=10.1111/all.15147&)

24. Sassolas B., Haddad C., Mockenhaupt M., Dunant A., Liss Y., Bork K., et al. (2010). ALDEN, an algorithm for assessment of drug causality in Stevens-Johnson Syndrome and toxic epidermal necrolysis: comparison with case-control analysis. Clin. Pharmacol. Ther. 88, 60–68. 10.1038/clpt.2009.252  [DOI](https://doi.org/10.1038/clpt.2009.252) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20375998/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=ALDEN,%20an%20algorithm%20for%20assessment%20of%20drug%20causality%20in%20Stevens-Johnson%20Syndrome%20and%20toxic%20epidermal%20necrolysis:%20comparison%20with%20case-control%20analysis&author=B.%20Sassolas&author=C.%20Haddad&author=M.%20Mockenhaupt&author=A.%20Dunant&author=Y.%20Liss&volume=88&publication_year=2010&pages=60-68&pmid=20375998&doi=10.1038/clpt.2009.252&)

25. Sharland M., Pulcini C., Harbarth S., Zeng M., Gandra S., Mathur S., et al. (2018). Classifying antibiotics in the WHO essential medicines list for optimal use-be AWaRe. Lancet Infect. Dis. 18, 18–20. 10.1016/S1473-3099(17)30724-7  [DOI](https://doi.org/10.1016/S1473-3099(17)30724-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29303731/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet%20Infect.%20Dis.&title=Classifying%20antibiotics%20in%20the%20WHO%20essential%20medicines%20list%20for%20optimal%20use-be%20AWaRe&author=M.%20Sharland&author=C.%20Pulcini&author=S.%20Harbarth&author=M.%20Zeng&author=S.%20Gandra&volume=18&publication_year=2018&pages=18-20&pmid=29303731&doi=10.1016/S1473-3099(17)30724-7&)

26. Sodhi K. K., Singh C. K., Kumar M., Singh D. K. (2023). Whole-genome sequencing of Alcaligenes sp. strain MMA: insight into the antibiotic and heavy metal resistant genes. Front. Pharmacol. 14, 1144561. 10.3389/fphar.2023.1144561  [DOI](https://doi.org/10.3389/fphar.2023.1144561) | [PMC free article](/articles/PMC10213877/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/37251338/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front.%20Pharmacol.&title=Whole-genome%20sequencing%20of%20Alcaligenes%20sp.%20strain%20MMA:%20insight%20into%20the%20antibiotic%20and%20heavy%20metal%20resistant%20genes&author=K.%20K.%20Sodhi&author=C.%20K.%20Singh&author=M.%20Kumar&author=D.%20K.%20Singh&volume=14&publication_year=2023&pages=1144561&pmid=37251338&doi=10.3389/fphar.2023.1144561&)

27. Um S. J., Lee S. K., Kim Y. H., Kim K. H., Son C. H., Roh M. S., et al. (2010). Clinical features of drug-induced hypersensitivity syndrome in 38 patients. J. Investig. Allergol. Clin. Immunol. 20, 556–562.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/21313995/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Investig.%20Allergol.%20Clin.%20Immunol.&title=Clinical%20features%20of%20drug-induced%20hypersensitivity%20syndrome%20in%2038%20patients&author=S.%20J.%20Um&author=S.%20K.%20Lee&author=Y.%20H.%20Kim&author=K.%20H.%20Kim&author=C.%20H.%20Son&volume=20&publication_year=2010&pages=556-562&pmid=21313995&)

28. Wang C. W., Tassaneeyakul W., Chen C. B., Chen W. T., Teng Y. C., Huang C. Y., et al. (2021). Whole genome sequencing identifies genetic variants associated with co-trimoxazole hypersensitivity in Asians. J. Allergy Clin. Immunol. 147, 1402–1412. 10.1016/j.jaci.2020.08.003  [DOI](https://doi.org/10.1016/j.jaci.2020.08.003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32791162/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Allergy%20Clin.%20Immunol.&title=Whole%20genome%20sequencing%20identifies%20genetic%20variants%20associated%20with%20co-trimoxazole%20hypersensitivity%20in%20Asians&author=C.%20W.%20Wang&author=W.%20Tassaneeyakul&author=C.%20B.%20Chen&author=W.%20T.%20Chen&author=Y.%20C.%20Teng&volume=147&publication_year=2021&pages=1402-1412&pmid=32791162&doi=10.1016/j.jaci.2020.08.003&)

29. Wang C. W., Yang L. Y., Chen C. B., Ho H. C., Hung S. I., Yang C. H., et al. (2018). Randomized, controlled trial of TNF-α antagonist in CTL-mediated severe cutaneous adverse reactions. J. Clin. Invest. 128, 985–996. 10.1172/JCI93349  [DOI](https://doi.org/10.1172/JCI93349) | [PMC free article](/articles/PMC5824923/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29400697/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Invest.&title=Randomized,%20controlled%20trial%20of%20TNF-%CE%B1%20antagonist%20in%20CTL-mediated%20severe%20cutaneous%20adverse%20reactions&author=C.%20W.%20Wang&author=L.%20Y.%20Yang&author=C.%20B.%20Chen&author=H.%20C.%20Ho&author=S.%20I.%20Hung&volume=128&publication_year=2018&pages=985-996&pmid=29400697&doi=10.1172/JCI93349&)

30. Wattanachai P., Amornpinyo W., Konyoung P., Purimart D., Khunarkornsiri U., Pattanacheewapull O., et al. (2023). Association between HLA alleles and beta-lactam antibiotics-related severe cutaneous adverse reactions. Front. Pharmacol. 14, 1248386. 10.3389/fphar.2023.1248386  [DOI](https://doi.org/10.3389/fphar.2023.1248386) | [PMC free article](/articles/PMC10546186/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/37795024/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front.%20Pharmacol.&title=Association%20between%20HLA%20alleles%20and%20beta-lactam%20antibiotics-related%20severe%20cutaneous%20adverse%20reactions&author=P.%20Wattanachai&author=W.%20Amornpinyo&author=P.%20Konyoung&author=D.%20Purimart&author=U.%20Khunarkornsiri&volume=14&publication_year=2023&pages=1248386&pmid=37795024&doi=10.3389/fphar.2023.1248386&)

31. Wei C. Y., Chung W. H., Huang H. W., Chen Y. T., Hung S. I. (2012). Direct interaction between HLA-B and carbamazepine activates T cells in patients with Stevens-Johnson syndrome. J. Allergy Clin. Immunol. 129, 1562–1569. 10.1016/j.jaci.2011.12.990  [DOI](https://doi.org/10.1016/j.jaci.2011.12.990) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22322005/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Allergy%20Clin.%20Immunol.&title=Direct%20interaction%20between%20HLA-B%20and%20carbamazepine%20activates%20T%20cells%20in%20patients%20with%20Stevens-Johnson%20syndrome&author=C.%20Y.%20Wei&author=W.%20H.%20Chung&author=H.%20W.%20Huang&author=Y.%20T.%20Chen&author=S.%20I.%20Hung&volume=129&publication_year=2012&pages=1562-1569&pmid=22322005&doi=10.1016/j.jaci.2011.12.990&)

32. Yang J., Qiao H. L., Zhang Y. W., Jia L. J., Tian X., Gao N. (2006). HLA-DRB genotype and specific IgE responses in patients with allergies to penicillins. Chin. Med. J. Engl. 119, 458–466. 10.1097/00029330-200603020-00005  [DOI](https://doi.org/10.1097/00029330-200603020-00005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16584643/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chin.%20Med.%20J.%20Engl.&title=HLA-DRB%20genotype%20and%20specific%20IgE%20responses%20in%20patients%20with%20allergies%20to%20penicillins&author=J.%20Yang&author=H.%20L.%20Qiao&author=Y.%20W.%20Zhang&author=L.%20J.%20Jia&author=X.%20Tian&volume=119&publication_year=2006&pages=458-466&pmid=16584643&doi=10.1097/00029330-200603020-00005&)

33. Zhang F. R., Liu H., Irwanto A., Fu X. A., Li Y., Yu G. Q., et al. (2013). HLA-B*13:01 and the dapsone hypersensitivity syndrome. N. Engl. J. Med. 369, 1620–1628. 10.1056/NEJMoa1213096  [DOI](https://doi.org/10.1056/NEJMoa1213096) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24152261/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%20Engl.%20J.%20Med.&title=HLA-B*13:01%20and%20the%20dapsone%20hypersensitivity%20syndrome&author=F.%20R.%20Zhang&author=H.%20Liu&author=A.%20Irwanto&author=X.%20A.%20Fu&author=Y.%20Li&volume=369&publication_year=2013&pages=1620-1628&pmid=24152261&doi=10.1056/NEJMoa1213096&)
