# Germline cis variant determines epigenetic regulation of the anti-cancer drug metabolism gene dihydropyrimidine dehydrogenase (DPYD)

## Metadata
**Authors:** Ting Zhang, Alisa Ambrodji, Huixing Huang, Kelly J Bouchonville, Amy S Etheridge, Remington E Schmidt, Brianna M Bembenek, Zoey B Temesgen, Zhiquan Wang, Federico Innocenti, Deborah Stroka, Robert B Diasio, Carlo R Largiadèr, Steven M Offer
**Journal:** eLife
**Date:** 2024 Apr 30
**DOI:** [10.7554/eLife.94075](https://doi.org/10.7554/eLife.94075)
**PMID:** 38686795
**PMCID:** PMC11060711
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11060711/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC11060711/pdf/elife-94075.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC11060711/pdf/elife-94075.pdf)

## Abstract

Enhancers are critical for regulating tissue-specific gene expression, and genetic variants within enhancer regions have been suggested to contribute to various cancer-related processes, including therapeutic resistance. However, the precise mechanisms remain elusive. Using a well-defined drug-gene pair, we identified an enhancer region for dihydropyrimidine dehydrogenase (DPD, DPYD gene) expression that is relevant to the metabolism of the anti-cancer drug 5-fluorouracil (5-FU). Using reporter systems, CRISPR genome-edited cell models, and human liver specimens, we demonstrated in vitro and vivo that genotype status for the common germline variant (rs4294451; 27% global minor allele frequency) located within this novel enhancer controls DPYD transcription and alters resistance to 5-FU. The variant genotype increases recruitment of the transcription factor CEBPB to the enhancer and alters the level of direct interactions between the enhancer and DPYD promoter. Our data provide insight into the regulatory mechanisms controlling sensitivity and resistance to 5-FU.

Research organism: Human

## Introduction

Therapeutic resistance has been reported for nearly all anti-cancer drugs ([Ramos et al., 2021](#bib47)), and as many as 90% of mortalities in cancer can be linked to drug resistance ([Mansoori et al., 2017](#bib34)). Enhancer-mediated regulation of gene expression has been increasingly implicated in multiple cancer-related processes, including resistance, response, and toxicity to cancer therapies ([Koutsi et al., 2022](#bib26); [Lauschke et al., 2019](#bib27)). However, the specific molecular mechanisms through which epigenetic changes drive these processes remain mostly elusive.

Enhancers regulate gene expression by recruiting transcription factors (TFs) and subsequently interacting with the promoter region of a target gene to drive expression. The activity of enhancers can be affected by localized epigenetic state and genetic variations that affect TF binding. More than 90% of disease-associated genetic variants identified in genome-wide association studies (GWAS) lie in the non-coding portion of the genome ([Manolio et al., 2009](#bib33); [Maurano et al., 2012](#bib35)). Many of these GWAS variants map to putative enhancers ([Boix et al., 2021](#bib7)). Regardless, translating enhancer GWAS variants into disease mechanisms remains a largely unmet challenge ([Claringbould and Zaugg, 2021](#bib10)).

In previous studies, we identified a *cis* expression quantitative trait locus (eQTL) for the chemotherapeutic metabolism gene dihydropyrimidine dehydrogenase enzyme (DPD, encoded by the *DPYD* gene) ([Etheridge et al., 2020](#bib13)). DPD is the initial and rate-defining enzyme for the conversion of the commonly used chemotherapeutic 5-FU into inactive metabolites. Hepatic DPD eliminates approximately 85% of circulating 5-FU within minutes of administration ([Sommadossi et al., 1982](#bib51); [Heggie et al., 1987](#bib19)), and deficiency of DPD is associated with severe (clinical grade ≥3) toxicity to 5-FU ([Amstutz et al., 2018](#bib4)). Clinical and preclinical studies have identified missense and splicing variants of *DPYD* that consistently correlate with reduced DPD function and increased 5-FU toxicity risk ([Amstutz et al., 2018](#bib4); [Lee et al., 2014](#bib28); [Shrestha et al., 2018](#bib50); [Offer et al., 2014b](#bib44); [Offer et al., 2013a](#bib41); [Offer et al., 2013b](#bib42); [Nie et al., 2017](#bib40)). However, these variants explain <20% of reported cases of grade ≥3 5 FU toxicities ([Amstutz et al., 2018](#bib4); [Meulendijks et al., 2015](#bib38)), and the mechanisms contributing to 5-FU resistance, and therefore means to overcome resistance, remain elusive.

In the present study, we used human liver tissues and cellular models to identify and characterize a novel *cis*-enhancer element capable of modulating *DPYD* expression. We additionally provide mechanistic data demonstrating that transcription-factor driven expression of *DPYD* is dependent on allele status for a common germline genetic variant located within this enhancer, suggesting that the variant could be a valuable biomarker of 5-FU toxicity risk. Furthermore, the genotype-dependent regulation of DPD expression offers a potential mechanism for overcoming resistance in patients carrying the risk variant.

## Results

### Identification of DPYD enhancer regions

Candidate *cis* enhancer regions for *DPYD* ([NM_000110.4](https://www.ncbi.nlm.nih.gov/nuccore/NM_000110.4)) were identified using GeneHancer ([Fishilevich et al., 2017](#bib14)) and Ensembl Regulatory Build ([Zerbino et al., 2015](#bib55)) data mapped to GRCh37/hg19. We defined the upstream boundary of the target region using enhancer data supported by multiple methods and gene-enhancer links supported by multiple methods as implemented within GeneHancer (i.e. putative enhancer regions with ‘double-elite’ status; [Figure 1A](#fig1)). We then mapped annotated transcription factor binding sites in the target region using ChIP-seq data from the Ensembl Regulatory Build as an additional layer of evidence ([Figure 1A](#fig1)). In all, three candidate proximal upstream enhancer regions were identified, which will be referred to herein as E9, E16, and E20 based on the relevant location upstream of the *DPYD* transcription start site ([Figure 1A](#fig1)).

### Figure 1. Identification of a novel dihydropyrimidine dehydrogenase (DPYD) enhancer.

![Figure 1.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0fa4/11060711/f0ece7ea4c43/elife-94075-fig1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=11060711_elife-94075-fig1.jpg)

(A) Candidate DPYD enhancer regions were selected for further study using data from GeneHancer and Ensembl Regulatory Build. Coordinates are based on GRCh37/hg19. Regions are termed E9, E16, and E20 based on the approximate distance upstream of the DPYD transcription start site. For CRISPR inhibition (CRISPRi), DPYD expression was measured in HepG2 cells expressing dCas9-KRAB following transfection with guide-RNAs specific to the E9 (B), E16 (C), and E20 (D) regions. For CRISPR activation (CRISPRa), DPYD expression was measured in HepG2 cells expressing dCas9-P300 following transfection with guide-RNAs specific to E9 (E), E16 (F), and E20 (G). Data represent the mean of three independent biological replicates ± SD. *p<0.05; **p<0.005. p-values were calculated using a two-tailed Student’s t-test comparing results to those from lentiguide controls.

CRISPR-interference (CRISPRi) and -activation (CRISPRa) were used to determine which of these regions were capable of regulating *DPYD* expression. For CRISPRi, we generated cell lines that stably expressed the Krüppel-associated box (KRAB) domain of Kox1 fused to nuclease-deficient Cas9 protein (dCas9-KRAB; [Yeo et al., 2018](#bib54)). For CRISPRa, we created cell lines that stably expressed dCas9 fused to the histone acetyltransferase E1A binding protein P300 (dCas9-P300; [Hilton et al., 2015](#bib21)). Six guide RNAs (gRNAs) were used to target dCas9 fusion proteins to each region of interest, and quantitative reverse transcription PCR (qRT-PCR) was used to monitor changes in *DPYD* expression following transfection.

Targeting of dCas9-KRAB (CRISPRi) to the E9 region significantly decreased *DPYD* expression relative to control for four of six gRNAs ([Figure 1B](#fig1)). Targeting of dCas9-KRAB to E16 resulted in a significant decrease in *DPYD* expression for one of six gRNAs ([Figure 1C](#fig1)), and no significant reductions in *DPYD* expression were noted following transfection of gRNAs for E20 in dCas9-expressing cells; however, two gRNAs increased *DPYD* expression ([Figure 1D](#fig1)). For reciprocal CRISPRa experiments, targeting dCas9-P300 to E9 significantly increased *DPYD* expression for four of six gRNAs tested ([Figure 1E](#fig1)), whereas gRNAs specific to E16 and E20 did not elicit any significant changes in *DPYD* expression ([Figure 1F and G](#fig1)). Consistent with the results in HepG2 cells, CRISPRi and CRISPRa targeted to the E9 region significantly altered *DPYD* expression in HCT116 cells ([Figure 1—figure supplement 1A and D](#fig1s1)), whereas no significant changes were noted when targeted to E16 or E20 ([Figure 1—figure supplement 1B, C, E, F](#fig1s1)).

To confirm that the expected epigenetic changes were induced at the E9 region, we performed chromatin immunoprecipitation (ChIP) coupled with quantitative PCR (ChIP-qPCR) in dCas9-KRAB and -P300 expressing cells transfected with E9 gRNAs. As expected, targeting dCas9-KRAB to E9 resulted in a localized increase in H3K9me3 compared to controls, indicating a shift from active to inactive chromatin at E9 ([Figure 2A](#fig2); the primer nearest E9 is denoted by the gray box). Targeting of dCas-P300 to E9 caused changes consistent with epigenetic activation of the region, including a localized gain of H3K27ac-marked chromatin ([Figure 2B](#fig2)). Similar changes in localized histone modifications were noted when dCas9-KRAB and dCas9-P300 HCT116 cells were directed to E9 ([Figure 2C–D](#fig2)). Collectively, these results indicate that the E9 region might act as a *cis*-regulatory element modulating *DPYD* expression.

### Figure 2. Epigenetic changes at the E9 region induced by CRISPR interference (CRISPRi)/CRISPR activation (CRISPRa).

![Figure 2.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0fa4/11060711/e47dc2266b72/elife-94075-fig2.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=11060711_elife-94075-fig2-figsupp1.jpg)

Lentiguide vectors encoding guide-RNAs targeting the E9 region (E9 gRNAs) or empty vector control (Lentiguide) were transduced into HepG2 cells expressing dCas9-KRAB (A) or dcas9-P300 (B) and HCT116 cells expressing dCas9-KRAB (C) or dcas9-P300 (D). Chromatin immunoprecipitation (ChIP) was performed using antibodies specific to H3K9me3 (A, C) or H3K27ac (B, D). Quantitative PCR using primers centered at the indicated regions (E) was used to measure the relative abundance of H3K9me3 and H3K27ac. Data are presented relative to input DNA control and are further normalized to IgG control. Error bars represent the SD of three independent biological replicate experiments.

### Identification of rs4294451 as a putative regulatory SNP within E9

In an expression quantitative trait loci (eQTL) genome-wide association study (GWAS) conducted in human liver specimens, Etheridge et al. previously identified three independent haplotype blocks that significantly associated with altered *DPYD* expression ([Etheridge et al., 2020](#bib13)). One of the blocks identified in that study spanned the E9 region, prompting us to investigate if genetic variants in E9 could potentially perturb *DPYD* regulation. Based on the biological role of enhancers, we postulated that a causal functional single nucleotide polymorphism (SNP) could interfere with transcription factor binding. Within the Ensembl Regulatory Build ([Zerbino et al., 2015](#bib55)), we identified a region of approximately 150 bp showing evidence for binding by multiple transcription factors in HepG2 cells, but not in other cell types tested ([Figure 2—figure supplement 1](#fig2s1)). Within this region, we identified the variant rs4294451, which is a tag SNP for the previously identified eQTL block ([Etheridge et al., 2020](#bib13)), prompting us to hypothesize that this variant might be the causal eQTL SNP.

To assess the contribution of rs4294451 alleles to *DPYD* regulation, we first performed reporter assays by cloning the *DPYD* promoter and E9 enhancer region containing either the reference or variant allele for rs4294451 (T or A, respectively) into a luciferase reporter vector ([Figure 3A](#fig3)). Reporter vectors containing the T-allele yielded significantly higher luciferase activity compared to vectors containing only the *DPYD* promoter (p=0.00054), whereas those containing the A allele within E9 showed a more modest increase (p=0.070, [Figure 3B](#fig3)). Directly comparing E9-containing reporters, vectors containing the T allele showed significantly higher activity than those with the A allele (p=0.046, [Figure 3B](#fig3)), suggesting that the variant could impact *DPYD* expression.

### Figure 3. The rs4294451 T allele is associated with elevated dihydropyrimidine dehydrogenase (DPYD) expression.

![Figure 3.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0fa4/11060711/0ead00a8fdba/elife-94075-fig3.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=11060711_elife-94075-fig3.jpg)

(A) Luciferase reporter constructs were generated by cloning the E9 region containing the reference rs4294451 T allele or the variant rs4294451 A allele into reporter vectors containing the DPYD promoter. (B) Luciferase reporter activity was measured for vectors shown in panel A. Error bars represent the SD of three independent biological replicates. (C) Relative expression of endogenous DPYD measured via RT-qPCR in HCT116 cells engineered using CRISPR-mediated genome editing to contain the depicted genotypes for rs4294451. Chromatin enrichment of H3K27ac (D) and H3K9me3 (E) was measured using chromatin immunoprecipitation (ChIP) coupled with quantitative PCR (ChIP-qPCR) in HCT116 cells engineered to contain the indicated genotypes at rs4294451. (F) DPYD allele-specific expression was measured in human liver tissues using the C and T alleles at position c.85. (G) Allele-specific expression was measured using the c.496-A and G alleles. All panels: *p<0.05; **p<0.005; ***p<0.0005. p-values were calculated as pairwise comparisons between the indicated groups using a two-tailed Student’s t-tests. Error bars represent SD.

### Impact of rs4294451 genotype on DPYD expression and regulation

To directly characterize the impact of rs4294451 alleles on *DPYD* expression at the endogenous locus, we used CRISPR-mediated genome editing to create matched isogenic HCT116 knock-in cell models for each genotype. Consistent with the results from the reporter assay ([Figure 3B](#fig3)), cells homozygous for the A allele had significantly lower *DPYD* expression compared to cells that were homozygous for the T allele (*P*=0.0011, [Figure 3C](#fig3)). Heterozygous rs4294451 A/T cells displayed intermediate expression compared to cells with homozygous A/A and T/T genotypes (p=0.00013 and p=0.028, respectively; [Figure 3C](#fig3)). At the chromatin level, cells homozygous for the A allele showed reductions in H3K27ac and accumulation of H3K9me at E9 compared to cells that were homozygous or heterozygous for the T allele ([Figure 3D–E](#fig3)). This finding is consistent with the A allele being associated with a less active epigenetic state at E9. Cells carrying a heterozygous genotype for rs4294451 displayed an intermediate epigenetic state, which is consistent with one epigenetically active and one inactive copy of E9 ([Figure 3D–E](#fig3)). Similar epigenetic differences between rs4294451 genotypes were noted at E9 in liver specimens obtained from human donors ([Figure 3—figure supplement 1](#fig3s1)).

### Allele-specific expression of DPYD transcripts

To precisely determine the impact of rs4294451 alleles on *DPYD* expression in vivo, we measured allele-specific expression using reverse transcriptase digital droplet PCR (RT-ddPCR) using donor liver tissues. Because rs4294451 falls outside of the coding region, we used coding region variants as proxies to measure strand-specific expression. These coding variants consisted of *DPYD*-c.85T>C (rs1801265) and c.496A>G (rs2297595), both of which are in linkage disequilibrium (LD) with the rs4294451 T allele (c.85C: D′=0.92, R^2^=0.80; c.496G: D′=0.67, R^2^=0.21). Imperfect LD between rs4294451 and proxies allowed us to use samples that are heterozygous for the coding region SNP, but homozygous for rs429441 genotype, as an additional level of control. In samples measured using the c.85 genotype, the fractional abundance of the C-allele ranged from 0.542 to 0.602 in samples heterozygous for rs4294451 ([Figure 3F](#fig3)). Expressed in terms of relative expression, the C-allele was expressed at levels 18–51% higher than the T-allele (i.e. 0.542/0.458=1.18 and 0.602/0.398=1.51). In contrast, the sample that was homozygous for the rs4294451 T allele showed allelic expression at the c.85 locus that was indistinguishable from the DNA control ([Figure 3F](#fig3)), indicating equal expression of both transcripts. In measurements using c.496, a significant allelic-imbalance of 0.565 (CI: 0.546–0.584) in favor of the variant c.496 G-allele compared to the DNA control (CI: 0.488–0.511) was observed ([Figure 3G](#fig3)). This again indicated higher expression of the linked rs4294451 T allele. Notably, liver specimens that were homozygous at the rs4294451 locus again showed c.496 allelic expression that was indistinguishable from the DNA control ([Figure 3G](#fig3)). These data indicate that rs4294451 significantly affects the expression of *DPYD* in the liver.

### Rs4294451 genotype affects interactions between the DPYD promoter and E9

We next investigated the impact of rs4294451 genotype on intra-chromatin interactions between the E9 region and the *DPYD* promoter using quantitative analysis of chromatin conformation capture (3C-qPCR) in knock-in cells. The schematic in [Figure 4A](#fig4) shows the location of primers used for 3C-qPCR relative to the *DPYD* transcription start site (TSS) and E9 region, as well as the location of *Hin*dIII digestion sites. When using primers anchored at the *DPYD* promoter, the E9 region showed a significantly stronger interaction with the promoter in cells carrying the T allele compared to those homozygous for the A allele (p=8.9 × 10^–4^ comparing TT to AA; ANOVA p=3.0 × 10^–4^ across all three genotypes; [Figure 4C](#fig4)). With primers anchored to the E9 region, cells homozygous for the rs4294451 T allele showed stronger promoter interaction than those carrying the A allele (p=4.9 × 10^–4^ comparing TT to AA; ANOVA p=1.2 × 10^–4^ across all three genotypes; [Figure 4D](#fig4)).

### Figure 4. The rs4294451 T allele is associated with increased interaction between E9 and the dihydropyrimidine dehydrogenase (DPYD) promoter.

![Figure 4.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0fa4/11060711/6dd71d5a3241/elife-94075-fig4.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=11060711_elife-94075-fig4.jpg)

(A) Schematic of HindIII restriction enzyme sites (vertical bars) and primers (arrows) used for chromatin conformation capture (3 C) relative to the DPYD transcription start site (TSS) and E9 region. (B) Legend for panels C–E. (C) 3 C of chromatin interactions in rs4294451 knock-in HCT116 cells using anchor primer positioned within the digestion fragment containing the DPYD promoter. A, location of anchor primer; N, location of primer used for data normalization. (D) 3 C of knock-in cells using anchor primer positioned within the fragment containing the E9 region. For panels C–D: one-way ANOVA p: *p<0.01; **p<0.001; all other data points, p>0.01. (E) Knock-in cells were treated with dilutions of 5-fluorouracil (5-FU) and viability was assessed using real-time cell analysis (RTCA). Cell index is a measure of impedance between electrodes that are arrayed at the bottom of the RTCA plate and is representative of the number of live cells attached to the culture plate. Data are from 48 hr of 5-FU treatment. For all plotted data, the mean ± SD of three independent replicates is presented.

### Rs4294451 genotype confers differential sensitivity to 5-FU

Having demonstrated that the rs4294451 genotype impacts *DPYD* expression, we next evaluated the impact on 5-FU sensitivity using real-time cellular analysis (RTCA). RTCA provides a measure of the number of viable cells present in a culture. We have previously demonstrated the utility of this technology for measuring differences in drug-sensitivity to 5-FU and that those differences directly correlate with DPD enzyme function within cells ([Shrestha et al., 2018](#bib50); [Offer et al., 2013b](#bib42)). Consistent with lower expression of *DPYD*, knock-in cells homozygous for the rs4294451 A allele were significantly more sensitive to 5-FU than cells homozygous for the T allele (IC_50_ concentrations were 9.1 and 95.0 μM 5-FU, respectively; p<0.0001; [Figure 4E](#fig4)). Heterozygous T/A cells showed an intermediate IC_50_ value of 29.9 µM 5-FU ([Figure 4E](#fig4)).

### Transcription factor binding within E9 is affected by rs4294451 status

Based on ENCODE data, rs4294451 is localized to a region that previously has been shown to be bound by multiple transcription factors ([Figure 2—figure supplement 1](#fig2s1)). To determine if known transcription factor binding sites could be affected by allele status at rs4294451, we computed binding scores for JUND, MYBL2, HNF4A, CEBPB, FOXA1, and FOXA2 for sequences comprising the E9 region containing the T and A alleles at rs4294451. CEBPB, FOXA1, and FOXA2 showed differential predicted binding scores between the rs4294451 A- and T-containing query sequences (data not shown). For CEBPB, the differential binding site had a higher score than other sites in the queried region that did not overlap with rs4294451, suggesting that the SNP could affect CEBPB binding in the region. While predicted binding in the region varied for FOXA1 and FOXA2 depending on rs4294451 genotype, other binding sites that did not overlap with the variant site showed stronger predicted binding, indicating that the SNP was likely not affecting the critical binding site in the region. Binding scores did not differ by genotype for JUND or HNF4A, and no binding site above the threshold score was detected in the region for MYBL2.

Based on the above, we sought to determine if CEBPB could regulate *DPYD* expression through the E9 region and if regulation was affected by rs4294451 genotype. We first used reporter assays in conjunction with CEBPB. CEBPB expression significantly increased reporter activity relative to GFP control for luciferase vectors containing the *DPYD* promoter, indicating that the promoter contains CEBPB recognition sites (p=9.1 × 10^–4^; [Figure 5A](#fig5)). In cells overexpressing CEBPB, plasmids containing the promoter and E9 region resulted in higher luciferase activity compared to those containing only the *DPYD* promoter, regardless of the presence of the A or T allele (p=0.0036 and p=0.0017, respectively), suggesting that rs4294451 genotype does not completely disrupt CEBPB regulation through E9 ([Figure 5A](#fig5)). Comparing results for the A and T alleles at rs4294451 in the presence of overexpressed CEBPB indicates that the T allele is likely more responsive to CEBPB ([Figure 5A](#fig5); p=0.036).

### Figure 5. T allele is associated with higher occupancy of CEBPB at E9 and the dihydropyrimidine dehydrogenase (DPYD) promoter.

![Figure 5.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0fa4/11060711/a3d676c31f3a/elife-94075-fig5.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=11060711_elife-94075-fig5.jpg)

(A) Expression plasmids for CEBPB or GFP (control) were co-transfected into HEK293T cells with the luciferase reporter plasmids depicted in Figure 3A. (B) Chromatin immunoprecipitation (ChIP) coupled with quantitative PCR (ChIP-qPCR) was performed to examine the relative CEBPB enrichment at the DNA fragments within the E9 region in knock-in HCT116 cells for the rs4294451 genotypes indicated. Primers used for E9 are centered on the position at 9201 nucleotides upstream of the DPYD TSS. (C) ChIP-qPCR was used to measure CEBPB occupancy surrounding the DPYD promoter region. *p<0.05; **p<0.005; ***p<0.0005. p-values were calculated as pairwise comparisons between the indicated groups using a two-tailed Student’s t-tests. For all panels, error bars represent the SD of three independent replicates.

To directly examine CEBPB binding to the E9 region and to determine if binding affinity differs between the rs4294451 A and T alleles, we performed ChIP with CEBPB antibodies in isogenic knock-in HCT116 cells for the rs4294451 A/A, A/T, and T/T genotypes. Cells homozygous for the reference T allele showed significantly higher CEBPB occupancy at both E9 and the *DPYD* promoter than A/T or A/A cells ([Figure 5B–C](#fig5)). These results suggest that rs4294451 genotype determined the binding potential for CEBPB at the E9 enhancer region, which, in turn, affects CEBPB-driven expression of *DPYD* from the promoter.

### Upregulation of DPYD expression by CEBPB is dependent on the rs4294451-T allele

To further characterize the role of CEBPB in regulating *DPYD*, we disrupted CEBPB expression using two independent shRNAs (denoted as ‘sh1’ and ‘sh2’) in HCT116 knock-in cells carrying rs4294451 A/A, A/T, and T/T genotypes. Knockdown of CEBPB was confirmed at the protein ([Figure 6A](#fig6)) and mRNA ([Figure 6B–D](#fig6)) levels. Knockdown of CEBPB significantly reduced *DPYD* expression in rs4294451 T/A and T/T cells, but not in rs4294451 A/A cells ([Figure 6E–F](#fig6)). CEBPB knockdown also reduced occupancy of the transcription factor at both E9 and the *DPYD* TSS in rs4294451 A/T and T/T cells, but not A/A cells, and the level of CEBPB occupancy at both regions under CEBPB knockdown conditions is similar to that in control shRNA-treated A/A cells ([Figure 6—figure supplement 1](#fig6s1)).

### Figure 6. CEBPB-mediated upregulation of dihydropyrimidine dehydrogenase (DPYD) is dependent on rs4294451 T allele.

![Figure 6.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0fa4/11060711/c3e2d9e686f8/elife-94075-fig6.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=11060711_elife-94075-fig6.jpg)

(A) Immunoblot showing knockdown of CEBPB expression in HCT116 cells carrying different rs4294451 genotypes transduced with lentiviral particles encoding two independent shRNAs against CEBPB (sh1 and sh2) or a scrambled control shRNA (scr) (see also Figure 6—source data 1, Figure 6—source data 2, and Figure 6—source data 3). CEBPB expression was measured by RT-qPCR in HCT116 A/A (B), T/A (C) and T/T cells (D) transduced with the indicated shRNA lentiviral particles. DPYD expression was measured in shCEBPB and scramble control HCT116 A/A (E), T/A (F), and T/T (G) cells. The effect of CEBPB knockdown on cell viability in HCT116 A/A cells (H) and HCT116 T/T cells (I) was measured by RTCA. Data shown are from 48 hr of 5-fluorouracil (5-FU) treatment at the indicated concentrations. *p<0.05, calculated as a pairwise two-sided Student’s t-test comparing the indicated data to that of the associated scr control. Error bars represent the SD from three independent replicates. Figure 6—source data 1. Original file for western blot analysis in Figure 6A (anti-CEBPB). elife-94075-fig6-data1.tif (4.5MB, tif) Figure 6—source data 2. Original file for western blot analysis in Figure 6A (anti-tubulin). elife-94075-fig6-data2.tif (4.1MB, tif) Figure 6—source data 3. PDF containing Figure 6A and annotated western blots used to make figure, including highlighted bands and sample labels. elife-94075-fig6-data3.pdf (1MB, pdf)

We next determined the extent to which CEBPB contributed to 5-FU chemoresistance in cells with the T allele of rs4294451 ([Figure 4E](#fig4)). In rs4294451 A/A cells, knockdown of CEBPB did not affect 5-FU sensitivity ([Figure 6H](#fig6)). In contrast, CEBPB knockdown significantly reduced the IC_50_ for 5-FU in rs4294451 T/T cells (p*<*0.0001; [Figure 6I](#fig6)).

### The rs4294451-T allele is enriched in individuals of African ancestry

Data from the Genome Aggregation Database (gnomAD [Chen et al., 2022](#bib9)) were used to estimate the frequency of the rs4294451-T allele in global populations ([Figure 6—figure supplement 2A](#fig6s2)). The highest minor allele frequency (MAF) was noted for African/African American individuals (40% MAF), where the MAF was lowest in East Asian individuals (7%). For comparison, individuals of European (Non-Finnish) ancestry, the population with the highest number of individuals reported in gnomAD, had a MAF of 23%. Within Latino-Admixed American gnomAD subjects, similar differences in MAFs were noted in local ancestry-informed frequency data ([Figure 6—figure supplement 2B](#fig6s2)).

## Discussion

The antitumor efficacy and risk of severe adverse events associated with 5-FU are determined by the overall bioavailability of the drug in plasma. As the critical determinant of 5-FU pharmacokinetics, liver DPD expression is pivotal to both the risk of severe adverse events and therapeutic resistance in 5-FU chemotherapy at opposite ends of the exposure spectrum. This is underscored by the narrow therapeutic window for 5-FU, with toxicity and efficacy occurring at partially overlapping drug exposure levels ([Beumer et al., 2019](#bib6)). Deleterious germline coding-region *DPYD* variants have been linked to severe 5-FU toxicity [Amstutz et al., 2018](#bib4); [Lee et al., 2014](#bib28); however, these variants are responsible for only a small fraction of severe adverse events in 5-FU use and are unlikely to contribute to drug resistance ([Henricks et al., 2018](#bib20)). Elevated expression of DPD in tumor cells is known to confer 5-FU resistance ([Jiang et al., 1997](#bib23); [Kikuchi et al., 2015](#bib25)), and upregulation of hepatic DPD expression has been shown to reduce drug exposures and promote the development of 5-FU–resistant tumors ([Li et al., 2013](#bib30)). However, the mechanisms regulating DPD expression are not well characterized, nor is it understood how the regulatory processes can be altered to support the development of 5-FU resistance.

In the current study, we identified a novel *cis*-enhancer region for *DPYD* that is located approximately 9 kb upstream from the gene’s transcription start site. We additionally provide evidence that the E9 region directly interacts with the *DPYD* promoter, supporting E9 as a functional enhancer for *DPYD* expression ([Figure 4](#fig4)). We demonstrated that CEBPB is a critical transcription factor for *DPYD* that binds to this enhancer region, termed E9, promoting enhancer-promoter interactions and increasing *DPYD* expression. We also showed that the allele status of the germline variant rs4294451, located within the E9 region, can affect CEBPB-driven *DPYD* expression and sensitivity/resistance to 5-FU, making it a strong candidate biomarker for 5-FU toxicity risk and potentially tumor resistance to 5-FU–based cancer therapy. These findings are consistent with the recent identification of a haplotype block linked to the rs4294451 T allele that was significantly associated with elevated *DPYD* expression in human liver tissues ([Etheridge et al., 2020](#bib13)). In the present manuscript, we show that the rs4294451 T allele is enriched for active chromatin marks in both human liver ([Figure 3—figure supplement 1](#fig3s1)) and in cellular knock-in models ([Figure 3D–E](#fig3)). Furthermore, using an innovative digital droplet RT-qPCR–based approach, we demonstrate that the rs4294451 T allele is associated with elevated expression of the *cis DPYD* transcript in human liver ([Figure 3F–G](#fig3)).

Allele frequency data retrieved from gnomAD suggest that a majority (65–70%, estimated from allele frequencies) of individuals of African ancestry carry the rs4294451-T allele ([Figure 6—figure supplement 2](#fig6s2)), whereas only about 35% of individuals of European ancestry are predicted to be carriers of the T allele. African American patients have worse overall survival in colorectal cancer compared to white patients, owing to biological and non-biological factors. While differential access to healthcare, treatment bias, and socioeconomic factors have been shown to contribute to the poorer prognosis ([Mayberry et al., 1995](#bib36)), other unrecognized factors also contribute to this difference ([Govindarajan et al., 2003](#bib15); [Alexander et al., 2007](#bib2)). Our data support a hypothesis that higher systemic 5-FU catabolism to due elevated liver DPD expression in carriers of the rs4294451-T allele results in lower exposure to active anti-tumor metabolites of 5-FU. Additionally, we demonstrate that colorectal cancer cell lines likely retain the enhancer functions associated with the rs4294451-T allele, suggesting that tumor cells carrying this variant could more readily inactive 5-FU via increased DPD expression. The higher likelihood of carrying the T-allele in individuals of African ancestry would, therefore, place them at greater risk. Additional studies will be needed to investigate the degree to which rs4294451-T contributes to survival and progression in colorectal cancer and in other solid cancers frequently treated with 5-FU.

The transcription factor CEBPB is a member of the CCAAT Enhancer Binding Protein family, a group of transcription factors that contain basic leucine zipper (bZIP) domains and is highly expressed in liver ([Akira et al., 1990](#bib1); [Jakobsen et al., 2013](#bib22)). Multiple CEBPB isoforms have been detected, with some acting as transcriptional activators and others as inhibitors. The data presented herein indicate that active isoforms of CEBPB are up-regulating *DPYD* through binding to the E9 enhancer region. Inhibitory isoforms of CEBPB have also been shown to be important for certain physiological processes including tumorigenesis and liver regeneration ([Bégay et al., 2015](#bib5); [Li et al., 2008](#bib29)). Our over-expression studies used the full-length active isoform. Therefore, we cannot rule out a role for the inactive isoform participating in the regulation of *DPYD* expression. In addition to acting as a homodimer, CEBPB can also heterodimerize with other CEBP family proteins and interact with other transcription factors, including P300/CBP, CREB, NFKB, AP1, and NFAT, to co-regulate gene expression ([Miller, 2016](#bib39); [Seo et al., 2021](#bib49)). Additional studies are underway to characterize the role of additional regulatory factors within the *DPYD* enhancer region identified in this manuscript.

In silico analyses suggested that the rs4294451 A allele created a stronger binding site for CEBPB within the E9 enhancer region. However, our data demonstrate that the T allele is associated with higher reporter activity ([Figure 3B](#fig3)), higher DPD expression in both knock-in cells ([Figure 3C](#fig3)) and human liver tissues ([Figure 3F–G](#fig3)), and localized epigenetic activation ([Figure 3D–E](#fig3)). Enrichment of CEBPB to the E9 region was likewise shown to be higher in cells with the rs4294451 T allele ([Figure 5](#fig5)) and higher levels of interaction were also noted between the E9 region with the T allele and the *DPYD* promoter ([Figure 4](#fig4)). These findings are also consistent with previously reported eQTL results for the haplotype block linked to rs4294451, where the T allele was associated with higher levels of *DPYD* expression in human liver specimens ([Etheridge et al., 2020](#bib13)), and our observation that the T allele is associated with cellular resistance to 5-FU ([Figure 4E](#fig4)). The binding of CEBPB to specific motifs has recently been shown to be cell-type specific and to rely on non-consensus binding motifs with binding strengths that can be modulated by the sequence and structure of surrounding DNA regions ([Cohen et al., 2018](#bib11); [Lountos et al., 2022](#bib31)), providing a possible explanation for the discrepancy between predicted and observed results.

To our knowledge, this is the first report directly linking CEBPB to 5-FU metabolism and the first mechanistic data demonstrating the *cis* effects of a regulatory variant on *DPYD* expression. The role of CEBPB in modulating 5-FU resistance and toxicity is not without precedent. For example, mir-191 is abnormally expressed in several cancers and has been associated with both 5-FU resistance and the regulation of CEBPB expression [Zhang et al., 2015](#bib56); however, the CEBPB–*DPYD* regulatory axis has not previously been recognized. Furthermore, CEBPB signaling has been shown to be activated in colorectal cancer cells following treatment with 5-FU ([Wang et al., 2019](#bib53)), suggesting that CEBPB-mediated activation of *DPYD* expression might represent a dynamic response to therapy.

While the contributions of regulatory variants to 5-FU metabolism have not been widely studied to date, previous studies have explored DPD regulation. Our laboratories previously characterized a *trans*-acting regulatory variant for DPD located within the microRNA mir-27a that was subsequently shown to further increase 5-FU toxicity risk in individuals that carried deleterious nonsynonymous *DPYD* variants ([Offer et al., 2014a](#bib43); [Amstutz et al., 2015](#bib3)). The variant at rs4294451 is in LD with the *DPYD* variants c.85T>C and c.496G>A, which served as coding region proxies for allele-specific expression in our present study ([Figure 3F–G](#fig3)). The haplotypes defined by these two coding-region variants together with a third variant (c.1129–5923C>G/rs75017182) previously associated with varied levels of systemic DPD activity ([Hamzic et al., 2021](#bib18)). A subsequent retrospective analysis indicated that these haplotype differences translate to a differential risk of severe 5-FU toxicity ([Medwid et al., 2023](#bib37)); however, additional studies are needed that are powered to evaluate more than the most common haplotypes. Taken together, these results suggest that the differential regulatory effects of rs4294451 alleles could further impact overall DPD activity and, by extension, modulate the risk of severe 5-FU–related toxicity conferred by coding or splice-variants in *DPYD*. Further work is also needed to identify interactions between the linked variants that impact DPD enzyme activity and the regulatory variant to define systemic DPD function and 5-FU toxicity risk.

## Methods

### Cells

HEK293T, HCT116, and HepG2 cell lines were obtained from the American Tissue Collection Center (ATCC, Manassas, VA). Aliquots of low passage cells were cryopreserved within 2 weeks of receipt. Cells were cultured for no longer than 10 total passages or 2 months, whichever occurred first. All cell lines were monitored for mycoplasma using Hoechst staining (Sigma-Aldrich, St. Louis, MO) at a minimum of one time per month and found to be negative. Culture health and identity were monitored by microscopy to evaluate morphology and by comparing population doubling times to baseline values that were recorded at time of receipt. Additional authentication of cell lines above that described was not performed.

All cell lines were cultured in Dulbecco’s modified Eagle’s medium (DMEM; Corning, Corning, NY) supplemented with 10% fetal bovine serum (FBS; MilliporeSigma, Billerica, MA), 2 mM L-glutamate (Corning), and 1 x penicillin/streptomycin solution (MilliporeSigma). Cells were maintained at 37 °C with 5% CO_2_. To support cell attachment and expansion, HepG2 cells were grown on plates coated with 5% Matrigel (Corning).

### Liver tissues

Human liver tissues used for ChIP analyses were processed through Dr. Mary Relling’s laboratory at St. Jude Children’s Research Hospital, part of the Pharmacogenetics of Anticancer Agents Research (PAAR) Group, and were provided by the Liver Tissue Cell Distribution System funded by NIH Contract #N01-DK-7–0004/HHSN267200700004C and by the Cooperative Human Tissue Network. The acquisition and use of specimens for this manuscript was conducted with the approval of the University of North Carolina at Chapel Hill IRB (study number 10–2253), which has designated the use of these livers for the current analyses as nonhuman subject research and the need for direct consent for use in this study was waived. Human liver tissues used for allele-specific expression were obtained from the University Clinic of Visceral Surgery and Medicine, Inselspital, Bern, Switzerland. Specimens were from donated material from patients who had undergone liver surgery at Inselspital and signed a written consent form for remnant tissues to be used in research (KEK-BE:2016–02202). Patients with impaired liver function due to cirrhosis or other conditions were excluded from analyses.

### Quantitative RT-PCR (RT-qPCR)

Total RNA was extracted from cells using the Direct-zol RNA Kit (Zymo Research, Tustin, CA) following the manufacturer’s protocol. Reverse transcription into cDNA was performed using the Transcriptor Reverse Transcriptase kit (Roche, Indianapolis, IN) and random hexamer primers (Roche) according to the manufacturer’s instructions. Quantitative PCR (qPCR) was performed on a LightCycler 480 System (Roche) using LightCycler 480 SYBR Green I Master Mix reagents (Roche). Primers used for RT-qPCR are listed in [Supplementary file 1](#supp1). RNA expression was normalized to the reference gene L32, and relative gene expression was calculated using the 2^−ΔΔCT^ method. For all analyses, three independent experiments were performed; for each experiment, gene expression was assessed in triplicate.

### Western blot analysis

Whole-cell lysates were extracted using the RIPA lysis buffer system (Santa Cruz Biotechnology, Dallas, TX), separated by SDS-PAGE, and transferred to PVDF membrane (MilliporeSigma). Blots were blocked using Odyssey Blocking Buffer (LI-COR Biosciences, Lincoln, NE) and incubated with primary antibody at 4 °C overnight. Primary antibodies consisted of anti-CEBPB (PA5-27244; 1:1,000 dilution; Thermo Fisher Scientific, Waltham, MA), anti-alpha Tubulin (ab4074; 1:7,500 dilution; Abcam, Waltham, MA), and anti-cas9 (sc-517386; 1:1,000 dilution; Santa Cruz Biotechnology, Dallas, TX). Membranes were washed and incubated with anti-mouse and anti-rabbit secondary antibodies (#926–32212 and #926–68073; both 1:5,000 dilution; LI-COR Biosciences) for 1 hr at room temperature. Blots were imaged using the Odyssey infrared imaging system (LI-COR Biosciences).

### Plasmids

The gRNA expression empty vector lentiGuide-Puro was a gift from Feng Zhang (Addgene_52963). pCMV-FLAG LAP2 was a gift from Joan Massague (Addgene_15738). The oligonucleotides targeting enhancer regions (E9, E16, and E20) were designed using GuideScan ([Perez et al., 2017](#bib45)), hybridized, phosphorylated, and cloned into lentiGuide-Puro via *Bsm*BI sites. The luciferase expression vector pGL4.10 was purchased from Promega (Madison, WI). Lentivirus vectors expressing shRNAs targeting CEBPB were obtained from the University of Minnesota Genomics Center (Sh1: TRCN0000007440 and Sh2: TRCN0000007442). Plasmids generated for these studies are available from the authors upon request.

### CRISPR inactivation (CRISPRi) and activation (CRISPRa)

HepG2 cell lines that overexpress dCas9-KRAB and dCas9-P300 were generated by lentiviral transduction using lenti-EF1a-dCas9-KRAB-Puro and pLV-dCas9-p300-P2A-PuroR, respectively. To generate lentiviral particles, HEK293T cells were co-transfected with lenti-EF1a-dCas9-KRAB-Puro plasmid (a gift from Kristen Brennand; Addgene_99372) or pLV-dCas9-p300-P2A-PuroR plasmid (a gift from Charles Gersbach; Addgene_83889), psPAX2 (a gift from Didier Trono; Addgene_12260), and pMD2.G (a gift from Didier Trono; Addgene_12259) using TransIT-Lenti Transfection Reagent (Mirus Bio, Madison, WI). A 3:1 ratio of transfection reagent to total plasmid was used for all transfections. For all transfections, medium was changed 14 hr after transfection, and viral supernatants were collected 34 hr later. Supernatants were filtered using 0.45 µm PVDF filters (MilliporeSigma) to remove debris/cells and used directly for transductions. For transductions, HCT116 cells or HepG2 cells were seeded at a density of 4 × 10^5^ cells per well in 6-well plates and incubated with 500 μL virus-containing supernatant, 12.5 μg/mL polybrene (MilliporeSigma), and 1.5 mL fresh DMEM culture medium. Medium was changed after 24 hr. Cells were treated with 1 μg/mL puromycin to initiate selection for transduced cells 48 hr after transduction. Expression of dCas9-KRAB and dCas9-P300 was confirmed by western blotting. Guide RNAs (gRNAs) for each target region were identified and designed using GuideScan ([Perez et al., 2017](#bib45)). Oligonucleotides corresponding to each gRNA were obtained from IDT (Coralville, IA), hybridized, phosphorylated using T4 polynucleotide kinase (New England Biolabs, Ipswich, MA), and ligated into digested BsmBI-digested (New England Biolabs) lentiGuide-Puro vector (a gift from Feng Zhang; Addgene_52963). Cell lines stably expressing dCas9-KRAB or dCas9-P300 were transfected with plasmids encoding gRNAs or lentiGuide empty vector using TransIT-X2 (Mirus Bio). RNA was extracted 2 days after transfection, and *DPYD* expression was measured by RT-qPCR.

### Chromatin immunoprecipitation coupled with quantitative PCR (ChIP-qPCR)

ChIP assays were performed using the ChIP-IT Express Enzymatic Kit (Active Motif, Carlsbad, CA) following the manufacturer’s directions. One million cells were harvested, washed, and crossed-linked using 1% formaldehyde (Thermo Fisher Scientific) in serum-free medium for 10 min followed by quenching with 125 mM glycine for 5 min at room temperature. The chromatin was digested using an enzymatic shearing cocktail provided by the kit to an average size of 200–1000 bp. Two percent of the sheared chromatin was retained as input control. Approximately 25 μg sheared chromatin was incubated with 2 μg H3K27ac antibody (ab4729; Abcam), 2 μg H3K9me3 antibody (ab8898; Abcam), 2 μg CEBPB antibody (PA5-27244; Thermo Fisher Scientific), or 2 μg control normal Rabbit IgG antibody (antibody 2729; Cell Signaling Technology, Danvers, MA), in the presence of protein G magnetic beads, ChIP buffer, and protease inhibitor cocktail (all Active Motif) for 4 hr at 4 °C. Magnetic beads were washed, chromatin was eluted, cross-linking was reversed, and proteinase K treatment was performed using reagents provided in the kit following the manufacturer’s directions. DNA was purified using the QIAquick PCR Purification Kit (Qiagen, Germantown, MD). Purified DNA was used for subsequent qPCR reactions using SYBR Green I Master Mix on a LightCycler 480 System (Roche). Enrichment was calculated using the following formula: (a) % ChIP = 2^(Input Ct - ChIP Ct)^ * (dilution factor) (100); (b) % IgG = 2^(Input Ct - IgG Ct)^ * (dilution factor) (100); (c) Fold Enrichment = % ChIP ÷ % IgG. Primers used for ChIP-qPCR are listed in [Supplementary file 1](#supp1).

### Luciferase reporter assays

The *DPYD* promoter region, consisting of the 1154 bp of genomic DNA directly upstream of the *DPYD* TSS, was amplified by PCR, digested with EcoRV and HindIII (New England Biolabs), and cloned into compatible sites on the pGL4.10 vector (Promega). The 1392 bp region comprising the E9 region was PCR amplified from genomic DNA and cloned upstream of the *DPYD* promoter using KpnI and SacI sites (New England Biolabs). A vector containing the A allele of rs4294451 within E9 was confirmed by Sanger sequencing. The vector containing the rs4294451 T allele was generated by site-directed mutagenesis and confirmed by sequencing. All primers used in vector construction are listed in [Supplementary file 1](#supp1). For reporter assays, 10^5^ HEK293T cells were seeded into 24-well plates and co-transfected with pGL4.10-based plasmids and pRL-SV40 Renilla luciferase plasmid (Promega). After 48 hr, luciferase activity was measured using the Dual-Glo Luciferase Assay (Promega) following the manufacturer’s recommendations on a Synergy HTX Multimode Plate Reader (Agilent Technologies, Santa Clara, CA).

### Knock-in cell lines for rs4294451 genotypes

Knock-in cell lines were generated using CRISPR/Cas9 gene editing. Homology-directed repair (HDR) donor templates and target-specific Alt-R crRNA were designed using the Alt-R HDR Design Tool (IDT). Equimolar amounts of crRNA (IDT) and common Alt-R tracrRNA (IDT) were annealed to form the gRNA duplex. RNP complexes were formed by combining gRNA with Alt-R S.p. Cas9 Nuclease V3 (IDT) to a final Cas9:gRNA ratio of 4:4.8. RNA complex and Alt-R HDR Donor Oligos were transfected into HCT116 cells by electroporation (Lonza Nucleofector 96-well Shuttle System; Lonza, Bend, OR) using parameters provided by the manufacturer. Seventy-two hours after transfection, serial dilutions were performed to obtain single-cell clones. Clones were expanded, genomic DNA was isolated, and rs42944551 genotype was determined by rhAmp Genotyping (assay ID: Hs.GT.rs4294451.A.1; IDT) using rhAmp Genotyping Master Mix and universal probe Reporter Mix (both IDT). Cell morphology was compared to the parental cell lines and confirmed to be consistent.

Upon derivation of confirmed knock-in clones, low passage cells were cryopreserved. Cell morphology was compared to that of parental cultures and found to be consistent. Consistent population doubling times were also noted. Cells were cultured for no longer than 10 total passages following the establishment of a confirmed clone. All cell lines were monitored for mycoplasma using Hoechst staining (Sigma-Aldrich) at a minimum of one time per month and found to be negative. Additional authentication of cell lines above that described was not performed.

### Allele-specific gene expression

Allele-specific expression of *DPYD* was measured using reverse transcriptase droplet digital PCR (RT-ddPCR) by targeting variants in the coding region of *DPYD* (c.85T>C and c.496A>G). Linkage disequilibrium (LD) between variants was calculated using LDpair implemented within LDlink ([Machiela and Chanock, 2015](#bib32)). DNA was extracted from donor liver tissues using the QIAamp DNA Mini Kit (Qiagen). For RNA extraction, tissues were lysed in QIAzol (Qiagen), and RNA was extracted using the miRneasy Kit (Qiagen) with on-column DNA digestion using RNase-free DNase (Qiagen). RNA quality was assessed using an Agilent 2100 Bioanalyzer running 2100 Expert Software v.B.02.10 using Agilent RNA 6000 Nano kits.

DNA samples were genotyped for c.85, c.496, and rs4294451 loci using TaqMan SNP Genotyping assays (Applied Biosystems, Waltham, MA). Tissues that were heterozygous for at least one of the coding region SNPs (i.e. c.85T>C and/or c.496A>G) were suitable for allele-specific expression analysis because expression from both alleles could be discriminated using the coding-region markers in mRNA. Allele-specific expression was measured using RT-ddPCR with the One-Step RT-ddPCR Advanced Kit for Probes (Qiagen) on a QX200 ddPCR Droplet Reader (BioRad, Hercules, CA). Fractional abundance was calculated using QuantaSoft software (BioRad). Poisson distributions were determined using Quantasoft and were used to define 95% confidence intervals. To address possible biases associated with differing probe efficacies caused by differential probe binding affinities or specificities, fractional abundances are also reported relative to those measured in DNA.

### Quantitative analysis of chromatin conformation capture (3C-qPCR)

Quantitative analysis of chromatin conformation capture (3C-qPCR) was performed as described by [Hagège et al., 2007](#bib16) with minor modifications. Cells were trypsinized using 0.25% w/v trypsin-EDTA (Thermo Fisher Scientific) and resuspended in 1% FBS (MilliporeSigma) in DMEM (Corning) for counting by flow cytometry using a NovoCyte 3000 RYB system (Agilent Technologies). Cells (10^7^) were pelleted by centrifugation at 300×*g* at 22 °C for 5 min. Supernatant was discarded, and cell pellets were resuspended in 500 µL of 10% FBS in DMEM. Single-cell suspensions were obtained by filtration through a 40 µm cell strainer (Corning). Crosslinking was performed by adding 9.5 mL of 1% formaldehyde in 10% FBS in PBS per 10^7^ cells. Reactions were incubated at 22 °C while tumbling for 10 min. Reactions were transferred to ice and crosslinking was quenched by the addition of ice-cold glycine (MilliporeSigma) to achieve a final concentration of 0.125 M. Samples were centrifuged at 300×*g* at 4 °C for 5 min. Supernatant was removed and discarded. Crosslinked cell pellets were lysed in 5 mL cold lysis buffer (10 mM Tris-HCl, pH7.5; 10 mM NaCl; 5 mM MgCl; 0.1 mM EGTA; 1 x Roche Complete protease inhibitor cocktail). Lysis was allowed to proceed for 10 min on ice with intermittent gentle pipetting to obtain homogeneous suspensions of nuclei. Nuclei were pelleted at 500×*g* at 4 °C for 5 min and resuspended in 0.5 mL of 1.2 x NEBuffer r2.1 (New England Biolabs) containing 3% sodium dodecyl sulfate (SDS, MilliporeSigma). Samples were incubated at 37 °C for 1 hr with shaking. Following incubation, Triton X-100 (MilliporeSigma) was added to a final concentration of 2%. Reactions were incubated at 37 °C for 1 hr with shaking. To digest DNA, 600 U of HindIII (New England Biolabs) was added, and reactions were incubated for 16 hr at 37 °C with shaking. To inactivate digestion, SDS was added to a final concentration of 1.6%, and reactions were incubated at 65 °C for 25 min with shaking. Excess SDS was sequestered by the addition of Triton X-100 to a final concentration of 1%. Samples were divided into two aliquots, one for ligation and the other for non-ligation control. Ligation was performed on a sevenfold dilution of HindIII-digested chromatin using 100 units of Quick T4 DNA ligase (New England Biolabs) at 16 °C for 16 hr, followed by 1 hr at 22 °C. Proteinase K (300 µg; New England Biolabs) was added to ligation mixtures and non-ligated controls, and samples were incubated at 65 °C for 16 hr to reverse crosslinking. DNA was subsequently purified by adding 300 µg RNase A (Thermo Fisher) followed by a 30 min incubation at 37 °C and subsequent phenol-chloroform extraction as described ([Hagège et al., 2007](#bib16)). DNA pellets were washed with 70% ethanol and dissolved in 10 mM Tris pH 7.5.

For qPCR, directional primers were designed within each fragment as depicted in [Figure 4A](#fig4). Two anchors, one localized to the fragment containing the *DPYD* promoter and the other to the E9 region, were selected and paired with primers designed to ‘walk’ across the length of the surrounding region. Reactions were carried out in 10 µL reaction volumes, consisting of 5 pmol of each primer and 1 µL of a 1:50 dilution of each 3 C sample. Amplification was performed using LightCycler 480 SYBR Green I Master Mix (Roche) on a LightCycler 480 thermocycler (Roche). PCR conditions were 95 °C for 10 min, and 40 cycles of 95 °C for 15 s, 65 °C for 1 min, and 72 °C for 15 s. Enrichment was calculated as 2^-(cp (ligated DNA) - cp (non-ligated DNA))^. Enrichment with the fragment containing the nearest primer was used as the control for interaction frequency for further normalization between replicate experiments. Specifically, when primer 5 was used as the anchor, data were normalized to the average of values obtained from using primers 5 and 4. When primer 7 was used as the anchor, data were normalized to the average of values from primers 7 and 8. All experiments were performed in triplicate. Primer sequences and positions relative to the *DPYD* transcription start site (TSS) for qPCR are listed in [Supplementary file 2](#supp2).

### Cellular sensitivity to 5-FU

Cell viability was monitored using the xCELLigence MP Real-Time Cell Analysis (RTCA) system (Agilent) as previously described ([Shrestha et al., 2018](#bib50)). Background impedance values for RTCA E-View plates (Agilent) were obtained using complete DMEM prior to plating cells. Cells were seeded at a density of 5000 cells per well and incubated for 20 hr, at which time medium was removed and replaced with medium containing serial dilutions of 5-FU ranging from 1.25 μM to 100 μM. Impedance values (expressed as cell index, CI, units) were recorded every 15 min over the course of the experiment to monitor proliferation. Results represent the average of three independent cultures. To account for minute differences in plating and potential cell loss during drug addition, relative CI was calculated as the CI measured 48 hr after 5-FU divided by the CI recorded immediately after treatment. Four parameter logistic non-linear regression analysis was used to determine IC_50_ concentrations (GraphPad Prism version 9, GraphPad Software, San Diego, CA).

### CEBPB knockdown cells

Lentiviral particles for CEBPB knockdown were generated by transfecting shRNA plasmids (TRCN0000007440 and TRCN0000007442) or scramble shRNA control (a gift from David Sabatini; Addgene_1864 [Sarbassov et al., 2005](#bib48)), psPAX2 (a gift from Didier Trono; Addgene_12260), and pMD2.G (a gift from Didier Trono; Addgene_12259) into HEK293T cells using TransIT-Lenti (Mirus Bio, Madison, WI). A 3:1 ratio of transfection reagent to total plasmid was used for all transfections. For all transfections, medium was changed 14 hr after transfection, and viral supernatants were collected 34 hr later. Supernatants were filtered using 0.45 µm PVDF filters (Millipore) to remove debris/cells and used directly for transductions. For transductions, target cells were seeded at a density of 4 × 10^5^ cells per well in six-well plates and incubated with 500 μL virus-containing supernatant, 12.5 μg/mL polybrene (MilliporeSigma), and 1.5 mL fresh DMEM culture medium.

### Analysis of transcription factor binding motifs

Potential transcription factor binding sites within the E9 region were determined using TFBSTools ([Tan and Lenhard, 2016](#bib52)). DNA sequence corresponding to the 101-nucleotide region centered on rs4294451 was retrieved from genome build GRCh38.p13 ([NC_000001.11](https://www.ncbi.nlm.nih.gov/nuccore/NC_000001.11)). A second DNA string was created to mimic the sequence corresponding to the rs4294451 A allele. Position frequency matrices for each transcription factor were retrieved from JASPAR CORE 2022 ([Castro-Mondragon et al., 2022](#bib8)) and converted to log-scale position weight matrices using the toPWM method implemented in TFBSTools. JASPAR includes binding site information from multiple sources. ENCODE data ([ENCODE Project Consortium, 2012](#bib12)) were available for CEBPB (matrix ID MA04661) and JUND (MA0491.1). In the absence of ENCODE data, alternatives including PAZAR ([Portales-Casamar et al., 2007](#bib46)) (HNF4A, MA0114.2; FOXA1, MA0148.3), REMAP ([Hammal et al., 2022](#bib17)) (FOXA2, MA0047.3), and data from an individual publication ([Jolma et al., 2013](#bib24)) (MYBL2, MA0777.1) were used. EP300 is a cofactor that does not recognize a specific DNA motif on its own; instead, it interacts with various DNA-binding factors to modify chromatin and facilitate the activation of target genes. As such, P300 does not have a DNA-binding motif and was not included in analysis. Nucleotide sequences were scanned using the patterns presented in the position weight matrices to identify putative transcription factor binding sites. Forward and reverse strands were searched, and the 70^th^ percentile between the minimum and maximum possible value for a matrix was used as the minimum threshold score. Empirical p-values for each score were calculated by an exact method using the TFMPvalue R package. R version 4.2.2 was used for analyses.

### Statistical analyses

Significance was defined as *P*<0.05 unless otherwise noted in the text. Pairwise comparisons were performed using unpaired two-tailed Student’s t-tests calculated using GraphPad Prism version 9. One-way ANOVA statistics were calculated using GraphPad Prism. Summary statistics pertaining to allele-specific expression were calculated using Quantasoft software as described in the Allele-specific gene expression section. Transcription factor binding predictions and associated analyses were performed in R version 4.2.2 as described above.

## Acknowledgements

This work was supported by the National Cancer Institute of the National Institutes of Health under award number R01CA251065 (SMO, PI). Allele-specific expression experiments were supported by the Swiss National Science Foundation under award number 320030_212583 (CRL, PI). The funding bodies did not contribute to the design of the study, the collection, analysis, and interpretation of data, or in writing the manuscript.

## Funding Statement

The funders had no role in study design, data collection and interpretation, or the decision to submit the work for publication.

## Contributor Information

Steven M Offer, Email: soffer@uiowa.edu.

Arduino A Mangoni, Flinders Medical Centre and Flinders University, Australia.

Eduardo L Franco, McGill University, Canada.

## Funding Information

This paper was supported by the following grants:

## Additional information

### Competing interests

### Author contributions

### Competing interests

### Author contributions

## Additional files

## Data availability

All data generated or analysed during this study are included in the manuscript and supporting files. All uses of previously published data within this manuscript are noted in the text and/or figure legends. Data were retrieved from GeneHancer ([http://www.genecards.org/](http://www.genecards.org/), [https://genecards.weizmann.ac.il/geneloc/index.shtml](https://genecards.weizmann.ac.il/geneloc/index.shtml)), the Ensembl Regulatory Build ([https://grch37.ensembl.org/info/genome/funcgen/regulatory_build.html](https://grch37.ensembl.org/info/genome/funcgen/regulatory_build.html)) and the Genome Aggregation Database (gnomAD v3.1.2 and gnomAD v3.1, [https://gnomad.broadinstitute.org/](https://gnomad.broadinstitute.org/)).

## References

1. Akira S, Isshiki H, Sugita T, Tanabe O, Kinoshita S, Nishio Y, Nakajima T, Hirano T, Kishimoto T. A nuclear factor for IL-6 expression (NF-IL6) is A member of A C/EBP family. The EMBO Journal. 1990;9:1897–1906. doi: 10.1002/j.1460-2075.1990.tb08316.x.  [DOI](https://doi.org/10.1002/j.1460-2075.1990.tb08316.x) | [PMC free article](/articles/PMC551896/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2112087/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=The%20EMBO%20Journal&title=A%20nuclear%20factor%20for%20IL-6%20expression%20(NF-IL6)%20is%20A%20member%20of%20A%20C/EBP%20family&author=S%20Akira&author=H%20Isshiki&author=T%20Sugita&author=O%20Tanabe&author=S%20Kinoshita&volume=9&publication_year=1990&pages=1897-1906&pmid=2112087&doi=10.1002/j.1460-2075.1990.tb08316.x&)

2. Alexander DD, Waterbor J, Hughes T, Funkhouser E, Grizzle W, Manne U. African-American and Caucasian disparities in colorectal cancer mortality and survival by data source: an epidemiologic review. Cancer Biomarkers. 2007;3:301–313. doi: 10.3233/cbm-2007-3604.  [DOI](https://doi.org/10.3233/cbm-2007-3604) | [PMC free article](/articles/PMC2667694/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18048968/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Biomarkers&title=African-American%20and%20Caucasian%20disparities%20in%20colorectal%20cancer%20mortality%20and%20survival%20by%20data%20source:%20an%20epidemiologic%20review&author=DD%20Alexander&author=J%20Waterbor&author=T%20Hughes&author=E%20Funkhouser&author=W%20Grizzle&volume=3&publication_year=2007&pages=301-313&pmid=18048968&doi=10.3233/cbm-2007-3604&)

3. Amstutz U, Offer SM, Sistonen J, Joerger M, Diasio RB, Largiadèr CR. Polymorphisms in MIR27A Associated with Early-Onset Toxicity in Fluoropyrimidine-Based Chemotherapy. Clinical Cancer Research. 2015;21:2038–2044. doi: 10.1158/1078-0432.CCR-14-2817.  [DOI](https://doi.org/10.1158/1078-0432.CCR-14-2817) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25655103/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clinical%20Cancer%20Research&title=Polymorphisms%20in%20MIR27A%20Associated%20with%20Early-Onset%20Toxicity%20in%20Fluoropyrimidine-Based%20Chemotherapy&author=U%20Amstutz&author=SM%20Offer&author=J%20Sistonen&author=M%20Joerger&author=RB%20Diasio&volume=21&publication_year=2015&pages=2038-2044&pmid=25655103&doi=10.1158/1078-0432.CCR-14-2817&)

4. Amstutz U, Henricks LM, Offer SM, Barbarino J, Schellens JHM, Swen JJ, Klein TE, McLeod HL, Caudle KE, Diasio RB, Schwab M. Clinical Pharmacogenetics Implementation Consortium (CPIC) Guideline for Dihydropyrimidine Dehydrogenase Genotype and Fluoropyrimidine Dosing: 2017 Update. Clinical Pharmacology and Therapeutics. 2018;103:210–216. doi: 10.1002/cpt.911.  [DOI](https://doi.org/10.1002/cpt.911) | [PMC free article](/articles/PMC5760397/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29152729/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clinical%20Pharmacology%20and%20Therapeutics&title=Clinical%20Pharmacogenetics%20Implementation%20Consortium%20(CPIC)%20Guideline%20for%20Dihydropyrimidine%20Dehydrogenase%20Genotype%20and%20Fluoropyrimidine%20Dosing:%202017%20Update&author=U%20Amstutz&author=LM%20Henricks&author=SM%20Offer&author=J%20Barbarino&author=JHM%20Schellens&volume=103&publication_year=2018&pages=210-216&pmid=29152729&doi=10.1002/cpt.911&)

5. Bégay V, Smink JJ, Loddenkemper C, Zimmermann K, Rudolph C, Scheller M, Steinemann D, Leser U, Schlegelberger B, Stein H, Leutz A. Deregulation of the endogenous C/EBPβ LIP isoform predisposes to tumorigenesis. Journal of Molecular Medicine. 2015;93:39–49. doi: 10.1007/s00109-014-1215-5.  [DOI](https://doi.org/10.1007/s00109-014-1215-5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25401168/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Molecular%20Medicine&title=Deregulation%20of%20the%20endogenous%20C/EBP%CE%B2%20LIP%20isoform%20predisposes%20to%20tumorigenesis&author=V%20B%C3%A9gay&author=JJ%20Smink&author=C%20Loddenkemper&author=K%20Zimmermann&author=C%20Rudolph&volume=93&publication_year=2015&pages=39-49&pmid=25401168&doi=10.1007/s00109-014-1215-5&)

6. Beumer JH, Chu E, Allegra C, Tanigawara Y, Milano G, Diasio R, Kim TW, Mathijssen RH, Zhang L, Arnold D, Muneoka K, Boku N, Joerger M. Therapeutic drug monitoring in oncology: International association of therapeutic drug monitoring and clinical toxicology recommendations for 5-fluorouracil therapy. Clinical Pharmacology and Therapeutics. 2019;105:598–613. doi: 10.1002/cpt.1124.  [DOI](https://doi.org/10.1002/cpt.1124) | [PMC free article](/articles/PMC6309286/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29923599/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clinical%20Pharmacology%20and%20Therapeutics&title=Therapeutic%20drug%20monitoring%20in%20oncology:%20International%20association%20of%20therapeutic%20drug%20monitoring%20and%20clinical%20toxicology%20recommendations%20for%205-fluorouracil%20therapy&author=JH%20Beumer&author=E%20Chu&author=C%20Allegra&author=Y%20Tanigawara&author=G%20Milano&volume=105&publication_year=2019&pages=598-613&pmid=29923599&doi=10.1002/cpt.1124&)

7. Boix CA, James BT, Park YP, Meuleman W, Kellis M. Regulatory genomic circuitry of human disease loci by integrative epigenomics. Nature. 2021;590:300–307. doi: 10.1038/s41586-020-03145-z.  [DOI](https://doi.org/10.1038/s41586-020-03145-z) | [PMC free article](/articles/PMC7875769/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33536621/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Regulatory%20genomic%20circuitry%20of%20human%20disease%20loci%20by%20integrative%20epigenomics&author=CA%20Boix&author=BT%20James&author=YP%20Park&author=W%20Meuleman&author=M%20Kellis&volume=590&publication_year=2021&pages=300-307&pmid=33536621&doi=10.1038/s41586-020-03145-z&)

8. Castro-Mondragon JA, Riudavets-Puig R, Rauluseviciute I, Berhanu Lemma R, Turchi L, Blanc-Mathieu R, Lucas J, Boddie P, Khan A, Manosalva Pérez N, Fornes O, Leung TY, Aguirre A, Hammal F, Schmelter D, Baranasic D, Ballester B, Sandelin A, Lenhard B, Vandepoele K, Wasserman WW, Parcy F, Mathelier A. JASPAR 2022: the 9th release of the open-access database of transcription factor binding profiles. Nucleic Acids Research. 2022;50:D165–D173. doi: 10.1093/nar/gkab1113.  [DOI](https://doi.org/10.1093/nar/gkab1113) | [PMC free article](/articles/PMC8728201/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34850907/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Research&title=JASPAR%202022:%20the%209th%20release%20of%20the%20open-access%20database%20of%20transcription%20factor%20binding%20profiles&author=JA%20Castro-Mondragon&author=R%20Riudavets-Puig&author=I%20Rauluseviciute&author=R%20Berhanu%20Lemma&author=L%20Turchi&volume=50&publication_year=2022&pages=D165-D173&pmid=34850907&doi=10.1093/nar/gkab1113&)

9. Chen S, Francioli LC, Goodrich JK, Collins RL, Kanai M, Wang Q, Alföldi J, Watts NA, Vittal C, Gauthier LD, Poterba T, Wilson MW, Tarasova Y, Phu W, Yohannes MT, Koenig Z, Farjoun Y, Banks E, Donnelly S, Gabriel S, Gupta N, Ferriera S, Tolonen C, Novod S, Bergelson L, Roazen D, Ruano-Rubio V, Covarrubias M, Llanwarne C, Petrillo N, Wade G, Jeandet T, Munshi R, Tibbetts K, O’Donnell-Luria A, Solomonson M, Seed C, Martin AR, Talkowski ME, Rehm HL, Daly MJ, Tiao G, Neale BM, MacArthur DG, Karczewski KJ, gnomAD Project Consortium A Genome-Wide Mutational Constraint Map Quantified from Variation in 76,156 Human Genomes. bioRxiv. 2022 doi: 10.1101/2022.03.20.485034.  [DOI](https://doi.org/10.1101/2022.03.20.485034)

10. Claringbould A, Zaugg JB. Enhancers in disease: molecular basis and emerging treatment strategies. Trends in Molecular Medicine. 2021;27:1060–1073. doi: 10.1016/j.molmed.2021.07.012.  [DOI](https://doi.org/10.1016/j.molmed.2021.07.012) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34420874/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Trends%20in%20Molecular%20Medicine&title=Enhancers%20in%20disease:%20molecular%20basis%20and%20emerging%20treatment%20strategies&author=A%20Claringbould&author=JB%20Zaugg&volume=27&publication_year=2021&pages=1060-1073&pmid=34420874&doi=10.1016/j.molmed.2021.07.012&)

11. Cohen DM, Lim H-W, Won K-J, Steger DJ. Shared nucleotide flanks confer transcriptional competency to bZip core motifs. Nucleic Acids Research. 2018;46:8371–8384. doi: 10.1093/nar/gky681.  [DOI](https://doi.org/10.1093/nar/gky681) | [PMC free article](/articles/PMC6144830/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30085281/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Research&title=Shared%20nucleotide%20flanks%20confer%20transcriptional%20competency%20to%20bZip%20core%20motifs&author=DM%20Cohen&author=H-W%20Lim&author=K-J%20Won&author=DJ%20Steger&volume=46&publication_year=2018&pages=8371-8384&pmid=30085281&doi=10.1093/nar/gky681&)

12. ENCODE Project Consortium An integrated encyclopedia of DNA elements in the human genome. Nature. 2012;489:57–74. doi: 10.1038/nature11247.  [DOI](https://doi.org/10.1038/nature11247) | [PMC free article](/articles/PMC3439153/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22955616/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=An%20integrated%20encyclopedia%20of%20DNA%20elements%20in%20the%20human%20genome&volume=489&publication_year=2012&pages=57-74&pmid=22955616&doi=10.1038/nature11247&)

13. Etheridge AS, Gallins PJ, Jima D, Broadaway KA, Ratain MJ, Schuetz E, Schadt E, Schroder A, Molony C, Zhou Y, Mohlke KL, Wright FA, Innocenti F. A new liver expression quantitative trait locus map from 1,183 individuals provides evidence for novel expression quantitative trait loci of drug response, metabolic, and sex-biased phenotypes. Clinical Pharmacology and Therapeutics. 2020;107:1383–1393. doi: 10.1002/cpt.1751.  [DOI](https://doi.org/10.1002/cpt.1751) | [PMC free article](/articles/PMC7816646/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31868224/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clinical%20Pharmacology%20and%20Therapeutics&title=A%20new%20liver%20expression%20quantitative%20trait%20locus%20map%20from%201,183%20individuals%20provides%20evidence%20for%20novel%20expression%20quantitative%20trait%20loci%20of%20drug%20response,%20metabolic,%20and%20sex-biased%20phenotypes&author=AS%20Etheridge&author=PJ%20Gallins&author=D%20Jima&author=KA%20Broadaway&author=MJ%20Ratain&volume=107&publication_year=2020&pages=1383-1393&pmid=31868224&doi=10.1002/cpt.1751&)

14. Fishilevich S, Nudel R, Rappaport N, Hadar R, Plaschkes I, Iny Stein T, Rosen N, Kohn A, Twik M, Safran M, Lancet D, Cohen D. GeneHancer: genome-wide integration of enhancers and target genes in GeneCards. Database. 2017;2017:bax028. doi: 10.1093/database/bax028.  [DOI](https://doi.org/10.1093/database/bax028) | [PMC free article](/articles/PMC5467550/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28605766/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Database&title=GeneHancer:%20genome-wide%20integration%20of%20enhancers%20and%20target%20genes%20in%20GeneCards&author=S%20Fishilevich&author=R%20Nudel&author=N%20Rappaport&author=R%20Hadar&author=I%20Plaschkes&volume=2017&publication_year=2017&pages=bax028&pmid=28605766&doi=10.1093/database/bax028&)

15. Govindarajan R, Shah RV, Erkman LG, Hutchins LF. Racial differences in the outcome of patients with colorectal carcinoma. Cancer. 2003;97:493–498. doi: 10.1002/cncr.11067.  [DOI](https://doi.org/10.1002/cncr.11067) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12518374/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer&title=Racial%20differences%20in%20the%20outcome%20of%20patients%20with%20colorectal%20carcinoma&author=R%20Govindarajan&author=RV%20Shah&author=LG%20Erkman&author=LF%20Hutchins&volume=97&publication_year=2003&pages=493-498&pmid=12518374&doi=10.1002/cncr.11067&)

16. Hagège H, Klous P, Braem C, Splinter E, Dekker J, Cathala G, de Laat W, Forné T. Quantitative analysis of chromosome conformation capture assays (3C-qPCR) Nature Protocols. 2007;2:1722–1733. doi: 10.1038/nprot.2007.243.  [DOI](https://doi.org/10.1038/nprot.2007.243) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17641637/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature%20Protocols&title=Quantitative%20analysis%20of%20chromosome%20conformation%20capture%20assays%20(3C-qPCR)&author=H%20Hag%C3%A8ge&author=P%20Klous&author=C%20Braem&author=E%20Splinter&author=J%20Dekker&volume=2&publication_year=2007&pages=1722-1733&pmid=17641637&doi=10.1038/nprot.2007.243&)

17. Hammal F, de Langen P, Bergon A, Lopez F, Ballester B. ReMap 2022: a database of Human, Mouse, Drosophila and Arabidopsis regulatory regions from an integrative analysis of DNA-binding sequencing experiments. Nucleic Acids Research. 2022;50:D316–D325. doi: 10.1093/nar/gkab996.  [DOI](https://doi.org/10.1093/nar/gkab996) | [PMC free article](/articles/PMC8728178/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34751401/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Research&title=ReMap%202022:%20a%20database%20of%20Human,%20Mouse,%20Drosophila%20and%20Arabidopsis%20regulatory%20regions%20from%20an%20integrative%20analysis%20of%20DNA-binding%20sequencing%20experiments&author=F%20Hammal&author=P%20de%20Langen&author=A%20Bergon&author=F%20Lopez&author=B%20Ballester&volume=50&publication_year=2022&pages=D316-D325&pmid=34751401&doi=10.1093/nar/gkab996&)

18. Hamzic S, Schärer D, Offer SM, Meulendijks D, Nakas C, Diasio RB, Fontana S, Wehrli M, Schürch S, Amstutz U, Largiadèr CR. Haplotype structure defines effects of common DPYD variants C.85T > C (rs1801265) and C.496A > G (rs2297595) on dihydropyrimidine dehydrogenase activity: Implication for 5-fluorouracil toxicity. British Journal of Clinical Pharmacology. 2021;87:3234–3243. doi: 10.1111/bcp.14742.  [DOI](https://doi.org/10.1111/bcp.14742) | [PMC free article](/articles/PMC8359980/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33491253/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=British%20Journal%20of%20Clinical%20Pharmacology&title=Haplotype%20structure%20defines%20effects%20of%20common%20DPYD%20variants%20C.85T%20>%20C%20(rs1801265)%20and%20C.496A%20>%20G%20(rs2297595)%20on%20dihydropyrimidine%20dehydrogenase%20activity:%20Implication%20for%205-fluorouracil%20toxicity&author=S%20Hamzic&author=D%20Sch%C3%A4rer&author=SM%20Offer&author=D%20Meulendijks&author=C%20Nakas&volume=87&publication_year=2021&pages=3234-3243&pmid=33491253&doi=10.1111/bcp.14742&)

19. Heggie GD, Sommadossi JP, Cross DS, Huster WJ, Diasio RB. Clinical pharmacokinetics of 5-fluorouracil and its metabolites in plasma, urine, and bile. Cancer Research. 1987;47:2203–2206.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/3829006/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Research&title=Clinical%20pharmacokinetics%20of%205-fluorouracil%20and%20its%20metabolites%20in%20plasma,%20urine,%20and%20bile&author=GD%20Heggie&author=JP%20Sommadossi&author=DS%20Cross&author=WJ%20Huster&author=RB%20Diasio&volume=47&publication_year=1987&pages=2203-2206&pmid=3829006&)

20. Henricks LM, Lunenburg CATC, de Man FM, Meulendijks D, Frederix GWJ, Kienhuis E, Creemers G-J, Baars A, Dezentjé VO, Imholz ALT, Jeurissen FJF, Portielje JEA, Jansen RLH, Hamberg P, Ten Tije AJ, Droogendijk HJ, Koopman M, Nieboer P, van de Poel MHW, Mandigers CMPW, Rosing H, Beijnen JH, Werkhoven E, van Kuilenburg ABP, van Schaik RHN, Mathijssen RHJ, Swen JJ, Gelderblom H, Cats A, Guchelaar H-J, Schellens JHM. DPYD genotype-guided dose individualisation of fluoropyrimidine therapy in patients with cancer: a prospective safety analysis. The Lancet. Oncology. 2018;19:1459–1467. doi: 10.1016/S1470-2045(18)30686-7.  [DOI](https://doi.org/10.1016/S1470-2045(18)30686-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30348537/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=The%20Lancet.%20Oncology&title=DPYD%20genotype-guided%20dose%20individualisation%20of%20fluoropyrimidine%20therapy%20in%20patients%20with%20cancer:%20a%20prospective%20safety%20analysis&author=LM%20Henricks&author=CATC%20Lunenburg&author=FM%20de%20Man&author=D%20Meulendijks&author=GWJ%20Frederix&volume=19&publication_year=2018&pages=1459-1467&pmid=30348537&doi=10.1016/S1470-2045(18)30686-7&)

21. Hilton IB, D’Ippolito AM, Vockley CM, Thakore PI, Crawford GE, Reddy TE, Gersbach CA. Epigenome editing by a CRISPR-Cas9-based acetyltransferase activates genes from promoters and enhancers. Nature Biotechnology. 2015;33:510–517. doi: 10.1038/nbt.3199.  [DOI](https://doi.org/10.1038/nbt.3199) | [PMC free article](/articles/PMC4430400/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25849900/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature%20Biotechnology&title=Epigenome%20editing%20by%20a%20CRISPR-Cas9-based%20acetyltransferase%20activates%20genes%20from%20promoters%20and%20enhancers&author=IB%20Hilton&author=AM%20D%E2%80%99Ippolito&author=CM%20Vockley&author=PI%20Thakore&author=GE%20Crawford&volume=33&publication_year=2015&pages=510-517&pmid=25849900&doi=10.1038/nbt.3199&)

22. Jakobsen JS, Waage J, Rapin N, Bisgaard HC, Larsen FS, Porse BT. Temporal mapping of CEBPA and CEBPB binding during liver regeneration reveals dynamic occupancy and specific regulatory codes for homeostatic and cell cycle gene batteries. Genome Research. 2013;23:592–603. doi: 10.1101/gr.146399.112.  [DOI](https://doi.org/10.1101/gr.146399.112) | [PMC free article](/articles/PMC3613577/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23403033/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Research&title=Temporal%20mapping%20of%20CEBPA%20and%20CEBPB%20binding%20during%20liver%20regeneration%20reveals%20dynamic%20occupancy%20and%20specific%20regulatory%20codes%20for%20homeostatic%20and%20cell%20cycle%20gene%20batteries&author=JS%20Jakobsen&author=J%20Waage&author=N%20Rapin&author=HC%20Bisgaard&author=FS%20Larsen&volume=23&publication_year=2013&pages=592-603&pmid=23403033&doi=10.1101/gr.146399.112&)

23. Jiang W, Lu Z, He Y, Diasio RB. Dihydropyrimidine dehydrogenase activity in hepatocellular carcinoma: implication in 5-fluorouracil-based chemotherapy. Clinical Cancer Research. 1997;3:395–399.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/9815697/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clinical%20Cancer%20Research&title=Dihydropyrimidine%20dehydrogenase%20activity%20in%20hepatocellular%20carcinoma:%20implication%20in%205-fluorouracil-based%20chemotherapy&author=W%20Jiang&author=Z%20Lu&author=Y%20He&author=RB%20Diasio&volume=3&publication_year=1997&pages=395-399&pmid=9815697&)

24. Jolma A, Yan J, Whitington T, Toivonen J, Nitta KR, Rastas P, Morgunova E, Enge M, Taipale M, Wei G, Palin K, Vaquerizas JM, Vincentelli R, Luscombe NM, Hughes TR, Lemaire P, Ukkonen E, Kivioja T, Taipale J. DNA-binding specificities of human transcription factors. Cell. 2013;152:327–339. doi: 10.1016/j.cell.2012.12.009.  [DOI](https://doi.org/10.1016/j.cell.2012.12.009) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23332764/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell&title=DNA-binding%20specificities%20of%20human%20transcription%20factors&author=A%20Jolma&author=J%20Yan&author=T%20Whitington&author=J%20Toivonen&author=KR%20Nitta&volume=152&publication_year=2013&pages=327-339&pmid=23332764&doi=10.1016/j.cell.2012.12.009&)

25. Kikuchi O, Ohashi S, Nakai Y, Nakagawa S, Matsuoka K, Kobunai T, Takechi T, Amanuma Y, Yoshioka M, Ida T, Yamamoto Y, Okuno Y, Miyamoto S, Nakagawa H, Matsubara K, Chiba T, Muto M. Novel 5-fluorouracil-resistant human esophageal squamous cell carcinoma cells with dihydropyrimidine dehydrogenase overexpression. American Journal of Cancer Research. 2015;5:2431–2440.  [PMC free article](/articles/PMC4568778/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26396918/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=American%20Journal%20of%20Cancer%20Research&title=Novel%205-fluorouracil-resistant%20human%20esophageal%20squamous%20cell%20carcinoma%20cells%20with%20dihydropyrimidine%20dehydrogenase%20overexpression&author=O%20Kikuchi&author=S%20Ohashi&author=Y%20Nakai&author=S%20Nakagawa&author=K%20Matsuoka&volume=5&publication_year=2015&pages=2431-2440&pmid=26396918&)

26. Koutsi MA, Pouliou M, Champezou L, Vatsellas G, Giannopoulou AI, Piperi C, Agelopoulos M. Typical enhancers, super-enhancers, and cancers. Cancers. 2022;14:4375. doi: 10.3390/cancers14184375.  [DOI](https://doi.org/10.3390/cancers14184375) | [PMC free article](/articles/PMC9496678/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36139535/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancers&title=Typical%20enhancers,%20super-enhancers,%20and%20cancers&author=MA%20Koutsi&author=M%20Pouliou&author=L%20Champezou&author=G%20Vatsellas&author=AI%20Giannopoulou&volume=14&publication_year=2022&pages=4375&pmid=36139535&doi=10.3390/cancers14184375&)

27. Lauschke VM, Zhou Y, Ingelman-Sundberg M. Novel genetic and epigenetic factors of importance for inter-individual differences in drug disposition, response and toxicity. Pharmacology & Therapeutics. 2019;197:122–152. doi: 10.1016/j.pharmthera.2019.01.002.  [DOI](https://doi.org/10.1016/j.pharmthera.2019.01.002) | [PMC free article](/articles/PMC6527860/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30677473/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacology%20&%20Therapeutics&title=Novel%20genetic%20and%20epigenetic%20factors%20of%20importance%20for%20inter-individual%20differences%20in%20drug%20disposition,%20response%20and%20toxicity&author=VM%20Lauschke&author=Y%20Zhou&author=M%20Ingelman-Sundberg&volume=197&publication_year=2019&pages=122-152&pmid=30677473&doi=10.1016/j.pharmthera.2019.01.002&)

28. Lee AM, Shi Q, Pavey E, Alberts SR, Sargent DJ, Sinicrope FA, Berenberg JL, Goldberg RM, Diasio RB. DPYD variants as predictors of 5-fluorouracil toxicity in adjuvant colon cancer treatment (NCCTG N0147) Journal of the National Cancer Institute. 2014;106:dju298. doi: 10.1093/jnci/dju298.  [DOI](https://doi.org/10.1093/jnci/dju298) | [PMC free article](/articles/PMC4271081/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25381393/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20the%20National%20Cancer%20Institute&title=DPYD%20variants%20as%20predictors%20of%205-fluorouracil%20toxicity%20in%20adjuvant%20colon%20cancer%20treatment%20(NCCTG%20N0147)&author=AM%20Lee&author=Q%20Shi&author=E%20Pavey&author=SR%20Alberts&author=DJ%20Sargent&volume=106&publication_year=2014&pages=dju298&pmid=25381393&doi=10.1093/jnci/dju298&)

29. Li Y, Bevilacqua E, Chiribau C-B, Majumder M, Wang C, Croniger CM, Snider MD, Johnson PF, Hatzoglou M. Differential Control of the CCAAT/Enhancer-binding Protein β (C/EBPβ) Products Liver-enriched Transcriptional Activating Protein (LAP) and Liver-enriched Transcriptional Inhibitory Protein (LIP) and the Regulation of Gene Expression during the Response to Endoplasmic Reticulum Stress. Journal of Biological Chemistry. 2008;283:22443–22456. doi: 10.1074/jbc.M801046200.  [DOI](https://doi.org/10.1074/jbc.M801046200) | [PMC free article](/articles/PMC2504880/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18550528/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Biological%20Chemistry&title=Differential%20Control%20of%20the%20CCAAT/Enhancer-binding%20Protein%20%CE%B2%20(C/EBP%CE%B2)%20Products%20Liver-enriched%20Transcriptional%20Activating%20Protein%20(LAP)%20and%20Liver-enriched%20Transcriptional%20Inhibitory%20Protein%20(LIP)%20and%20the%20Regulation%20of%20Gene%20Expression%20during%20the%20Response%20to%20Endoplasmic%20Reticulum%20Stress&author=Y%20Li&author=E%20Bevilacqua&author=C-B%20Chiribau&author=M%20Majumder&author=C%20Wang&volume=283&publication_year=2008&pages=22443-22456&pmid=18550528&doi=10.1074/jbc.M801046200&)

30. Li LH, Dong H, Zhao F, Tang J, Chen X, Ding J, Men HT, Luo WX, Du Y, Ge J, Tan BX, Cao D, Liu JY. The upregulation of dihydropyrimidine dehydrogenase in liver is involved in acquired resistance to 5-fluorouracil. European Journal of Cancer. 2013;49:1752–1760. doi: 10.1016/j.ejca.2012.12.013.  [DOI](https://doi.org/10.1016/j.ejca.2012.12.013) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23313143/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=European%20Journal%20of%20Cancer&title=The%20upregulation%20of%20dihydropyrimidine%20dehydrogenase%20in%20liver%20is%20involved%20in%20acquired%20resistance%20to%205-fluorouracil&author=LH%20Li&author=H%20Dong&author=F%20Zhao&author=J%20Tang&author=X%20Chen&volume=49&publication_year=2013&pages=1752-1760&pmid=23313143&doi=10.1016/j.ejca.2012.12.013&)

31. Lountos GT, Cherry S, Tropea JE, Wlodawer A, Miller M. Structural basis for cell type specific DNA binding of C/EBPβ: The case of cell cycle inhibitor p15INK4b promoter. Journal of Structural Biology. 2022;214:107918. doi: 10.1016/j.jsb.2022.107918.  [DOI](https://doi.org/10.1016/j.jsb.2022.107918) | [PMC free article](/articles/PMC9909937/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36343842/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Structural%20Biology&title=Structural%20basis%20for%20cell%20type%20specific%20DNA%20binding%20of%20C/EBP%CE%B2:%20The%20case%20of%20cell%20cycle%20inhibitor%20p15INK4b%20promoter&author=GT%20Lountos&author=S%20Cherry&author=JE%20Tropea&author=A%20Wlodawer&author=M%20Miller&volume=214&publication_year=2022&pages=107918&pmid=36343842&doi=10.1016/j.jsb.2022.107918&)

32. Machiela MJ, Chanock SJ. LDlink: a web-based application for exploring population-specific haplotype structure and linking correlated alleles of possible functional variants. Bioinformatics. 2015;31:3555–3557. doi: 10.1093/bioinformatics/btv402.  [DOI](https://doi.org/10.1093/bioinformatics/btv402) | [PMC free article](/articles/PMC4626747/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26139635/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=LDlink:%20a%20web-based%20application%20for%20exploring%20population-specific%20haplotype%20structure%20and%20linking%20correlated%20alleles%20of%20possible%20functional%20variants&author=MJ%20Machiela&author=SJ%20Chanock&volume=31&publication_year=2015&pages=3555-3557&pmid=26139635&doi=10.1093/bioinformatics/btv402&)

33. Manolio TA, Collins FS, Cox NJ, Goldstein DB, Hindorff LA, Hunter DJ, McCarthy MI, Ramos EM, Cardon LR, Chakravarti A, Cho JH, Guttmacher AE, Kong A, Kruglyak L, Mardis E, Rotimi CN, Slatkin M, Valle D, Whittemore AS, Boehnke M, Clark AG, Eichler EE, Gibson G, Haines JL, Mackay TFC, McCarroll SA, Visscher PM. Finding the missing heritability of complex diseases. Nature. 2009;461:747–753. doi: 10.1038/nature08494.  [DOI](https://doi.org/10.1038/nature08494) | [PMC free article](/articles/PMC2831613/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19812666/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Finding%20the%20missing%20heritability%20of%20complex%20diseases&author=TA%20Manolio&author=FS%20Collins&author=NJ%20Cox&author=DB%20Goldstein&author=LA%20Hindorff&volume=461&publication_year=2009&pages=747-753&pmid=19812666&doi=10.1038/nature08494&)

34. Mansoori B, Mohammadi A, Davudian S, Shirjang S, Baradaran B. The different mechanisms of cancer drug resistance: A brief review. Advanced Pharmaceutical Bulletin. 2017;7:339–348. doi: 10.15171/apb.2017.041.  [DOI](https://doi.org/10.15171/apb.2017.041) | [PMC free article](/articles/PMC5651054/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29071215/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Advanced%20Pharmaceutical%20Bulletin&title=The%20different%20mechanisms%20of%20cancer%20drug%20resistance:%20A%20brief%20review&author=B%20Mansoori&author=A%20Mohammadi&author=S%20Davudian&author=S%20Shirjang&author=B%20Baradaran&volume=7&publication_year=2017&pages=339-348&pmid=29071215&doi=10.15171/apb.2017.041&)

35. Maurano MT, Humbert R, Rynes E, Thurman RE, Haugen E, Wang H, Reynolds AP, Sandstrom R, Qu H, Brody J, Shafer A, Neri F, Lee K, Kutyavin T, Stehling-Sun S, Johnson AK, Canfield TK, Giste E, Diegel M, Bates D, Hansen RS, Neph S, Sabo PJ, Heimfeld S, Raubitschek A, Ziegler S, Cotsapas C, Sotoodehnia N, Glass I, Sunyaev SR, Kaul R, Stamatoyannopoulos JA. Systematic localization of common disease-associated variation in regulatory DNA. Science. 2012;337:1190–1195. doi: 10.1126/science.1222794.  [DOI](https://doi.org/10.1126/science.1222794) | [PMC free article](/articles/PMC3771521/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22955828/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Systematic%20localization%20of%20common%20disease-associated%20variation%20in%20regulatory%20DNA&author=MT%20Maurano&author=R%20Humbert&author=E%20Rynes&author=RE%20Thurman&author=E%20Haugen&volume=337&publication_year=2012&pages=1190-1195&pmid=22955828&doi=10.1126/science.1222794&)

36. Mayberry RM, Coates RJ, Hill HA, Click LA, Chen VW, Austin DF, Redmond CK, Fenoglio-Preiser CM, Hunter CP, Haynes MA. Determinants of black/white differences in colon cancer survival. Journal of the National Cancer Institute. 1995;87:1686–1693. doi: 10.1093/jnci/87.22.1686.  [DOI](https://doi.org/10.1093/jnci/87.22.1686) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7473817/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20the%20National%20Cancer%20Institute&title=Determinants%20of%20black/white%20differences%20in%20colon%20cancer%20survival&author=RM%20Mayberry&author=RJ%20Coates&author=HA%20Hill&author=LA%20Click&author=VW%20Chen&volume=87&publication_year=1995&pages=1686-1693&pmid=7473817&doi=10.1093/jnci/87.22.1686&)

37. Medwid S, Wigle TJ, Kim RB. Fluoropyrimidine-associated toxicity and DPYD variants C.85T>C, C.496A>G, and C.1236G>A: impact of haplotype. Cancer Chemotherapy and Pharmacology. 2023;91:97–102. doi: 10.1007/s00280-022-04491-7.  [DOI](https://doi.org/10.1007/s00280-022-04491-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36357798/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Chemotherapy%20and%20Pharmacology&title=Fluoropyrimidine-associated%20toxicity%20and%20DPYD%20variants%20C.85T>C,%20C.496A>G,%20and%20C.1236G>A:%20impact%20of%20haplotype&author=S%20Medwid&author=TJ%20Wigle&author=RB%20Kim&volume=91&publication_year=2023&pages=97-102&pmid=36357798&doi=10.1007/s00280-022-04491-7&)

38. Meulendijks D, Henricks LM, Sonke GS, Deenen MJ, Froehlich TK, Amstutz U, Largiadèr CR, Jennings BA, Marinaki AM, Sanderson JD, Kleibl Z, Kleiblova P, Schwab M, Zanger UM, Palles C, Tomlinson I, Gross E, van Kuilenburg ABP, Punt CJA, Koopman M, Beijnen JH, Cats A, Schellens JHM. Clinical relevance of DPYD variants c.1679T>G, c.1236G>A/HapB3, and c.1601G>A as predictors of severe fluoropyrimidine-associated toxicity: A systematic review and meta-analysis of individual patient data. The Lancet. Oncology. 2015;16:1639–1650. doi: 10.1016/S1470-2045(15)00286-7.  [DOI](https://doi.org/10.1016/S1470-2045(15)00286-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26603945/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=The%20Lancet.%20Oncology&title=Clinical%20relevance%20of%20DPYD%20variants%20c.1679T>G,%20c.1236G>A/HapB3,%20and%20c.1601G>A%20as%20predictors%20of%20severe%20fluoropyrimidine-associated%20toxicity:%20A%20systematic%20review%20and%20meta-analysis%20of%20individual%20patient%20data&author=D%20Meulendijks&author=LM%20Henricks&author=GS%20Sonke&author=MJ%20Deenen&author=TK%20Froehlich&volume=16&publication_year=2015&pages=1639-1650&pmid=26603945&doi=10.1016/S1470-2045(15)00286-7&)

39. Miller M. Interactions of CCAAT/enhancer-binding protein β with transcriptional coregulators. Postepy Biochemii. 2016;62:343–348.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/28132489/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Postepy%20Biochemii&title=Interactions%20of%20CCAAT/enhancer-binding%20protein%20%CE%B2%20with%20transcriptional%20coregulators&author=M%20Miller&volume=62&publication_year=2016&pages=343-348&pmid=28132489&)

40. Nie Q, Shrestha S, Tapper EE, Trogstad-Isaacson CS, Bouchonville KJ, Lee AM, Wu R, Jerde CR, Wang Z, Kubica PA, Offer SM, Diasio RB. Quantitative Contribution of rs75017182 to Dihydropyrimidine Dehydrogenase mRNA Splicing and Enzyme Activity. Clinical Pharmacology and Therapeutics. 2017;102:662–670. doi: 10.1002/cpt.685.  [DOI](https://doi.org/10.1002/cpt.685) | [PMC free article](/articles/PMC6138243/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28295243/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clinical%20Pharmacology%20and%20Therapeutics&title=Quantitative%20Contribution%20of%20rs75017182%20to%20Dihydropyrimidine%20Dehydrogenase%20mRNA%20Splicing%20and%20Enzyme%20Activity&author=Q%20Nie&author=S%20Shrestha&author=EE%20Tapper&author=CS%20Trogstad-Isaacson&author=KJ%20Bouchonville&volume=102&publication_year=2017&pages=662-670&pmid=28295243&doi=10.1002/cpt.685&)

41. Offer SM, Lee AM, Mattison LK, Fossum C, Wegner NJ, Diasio RB. A DPYD variant (Y186C) in individuals of african ancestry is associated with reduced DPD enzyme activity. Clinical Pharmacology and Therapeutics. 2013a;94:158–166. doi: 10.1038/clpt.2013.69.  [DOI](https://doi.org/10.1038/clpt.2013.69) | [PMC free article](/articles/PMC3821392/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23588312/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clinical%20Pharmacology%20and%20Therapeutics&title=A%20DPYD%20variant%20(Y186C)%20in%20individuals%20of%20african%20ancestry%20is%20associated%20with%20reduced%20DPD%20enzyme%20activity&author=SM%20Offer&author=AM%20Lee&author=LK%20Mattison&author=C%20Fossum&author=NJ%20Wegner&volume=94&publication_year=2013a&pages=158-166&pmid=23588312&doi=10.1038/clpt.2013.69&)

42. Offer SM, Wegner NJ, Fossum C, Wang K, Diasio RB. Phenotypic profiling of DPYD variations relevant to 5-fluorouracil sensitivity using real-time cellular analysis and in vitro measurement of enzyme activity. Cancer Research. 2013b;73:1958–1968. doi: 10.1158/0008-5472.CAN-12-3858.  [DOI](https://doi.org/10.1158/0008-5472.CAN-12-3858) | [PMC free article](/articles/PMC3602211/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23328581/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Research&title=Phenotypic%20profiling%20of%20DPYD%20variations%20relevant%20to%205-fluorouracil%20sensitivity%20using%20real-time%20cellular%20analysis%20and%20in%20vitro%20measurement%20of%20enzyme%20activity&author=SM%20Offer&author=NJ%20Wegner&author=C%20Fossum&author=K%20Wang&author=RB%20Diasio&volume=73&publication_year=2013b&pages=1958-1968&pmid=23328581&doi=10.1158/0008-5472.CAN-12-3858&)

43. Offer SM, Butterfield GL, Jerde CR, Fossum CC, Wegner NJ, Diasio RB. microRNAs miR-27a and miR-27b directly regulate liver dihydropyrimidine dehydrogenase expression through two conserved binding sites. Molecular Cancer Therapeutics. 2014a;13:742–751. doi: 10.1158/1535-7163.MCT-13-0878.  [DOI](https://doi.org/10.1158/1535-7163.MCT-13-0878) | [PMC free article](/articles/PMC3954441/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24401318/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Molecular%20Cancer%20Therapeutics&title=microRNAs%20miR-27a%20and%20miR-27b%20directly%20regulate%20liver%20dihydropyrimidine%20dehydrogenase%20expression%20through%20two%20conserved%20binding%20sites&author=SM%20Offer&author=GL%20Butterfield&author=CR%20Jerde&author=CC%20Fossum&author=NJ%20Wegner&volume=13&publication_year=2014a&pages=742-751&pmid=24401318&doi=10.1158/1535-7163.MCT-13-0878&)

44. Offer SM, Fossum CC, Wegner NJ, Stuflesser AJ, Butterfield GL, Diasio RB. Comparative functional analysis of DPYD variants of potential clinical relevance to dihydropyrimidine dehydrogenase activity. Cancer Research. 2014b;74:2545–2554. doi: 10.1158/0008-5472.CAN-13-2482.  [DOI](https://doi.org/10.1158/0008-5472.CAN-13-2482) | [PMC free article](/articles/PMC4012613/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24648345/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Research&title=Comparative%20functional%20analysis%20of%20DPYD%20variants%20of%20potential%20clinical%20relevance%20to%20dihydropyrimidine%20dehydrogenase%20activity&author=SM%20Offer&author=CC%20Fossum&author=NJ%20Wegner&author=AJ%20Stuflesser&author=GL%20Butterfield&volume=74&publication_year=2014b&pages=2545-2554&pmid=24648345&doi=10.1158/0008-5472.CAN-13-2482&)

45. Perez AR, Pritykin Y, Vidigal JA, Chhangawala S, Zamparo L, Leslie CS, Ventura A. GuideScan software for improved single and paired CRISPR guide RNA design. Nature Biotechnology. 2017;35:347–349. doi: 10.1038/nbt.3804.  [DOI](https://doi.org/10.1038/nbt.3804) | [PMC free article](/articles/PMC5607865/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28263296/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature%20Biotechnology&title=GuideScan%20software%20for%20improved%20single%20and%20paired%20CRISPR%20guide%20RNA%20design&author=AR%20Perez&author=Y%20Pritykin&author=JA%20Vidigal&author=S%20Chhangawala&author=L%20Zamparo&volume=35&publication_year=2017&pages=347-349&pmid=28263296&doi=10.1038/nbt.3804&)

46. Portales-Casamar E, Kirov S, Lim J, Lithwick S, Swanson MI, Ticoll A, Snoddy J, Wasserman WW. PAZAR: a framework for collection and dissemination of cis-regulatory sequence annotation. Genome Biology. 2007;8:R207. doi: 10.1186/gb-2007-8-10-r207.  [DOI](https://doi.org/10.1186/gb-2007-8-10-r207) | [PMC free article](/articles/PMC2246282/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17916232/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Biology&title=PAZAR:%20a%20framework%20for%20collection%20and%20dissemination%20of%20cis-regulatory%20sequence%20annotation&author=E%20Portales-Casamar&author=S%20Kirov&author=J%20Lim&author=S%20Lithwick&author=MI%20Swanson&volume=8&publication_year=2007&pages=R207&pmid=17916232&doi=10.1186/gb-2007-8-10-r207&)

47. Ramos A, Sadeghi S, Tabatabaeian H. Battling chemoresistance in cancer: Root causes and strategies to uproot them. International Journal of Molecular Sciences. 2021;22:9451. doi: 10.3390/ijms22179451.  [DOI](https://doi.org/10.3390/ijms22179451) | [PMC free article](/articles/PMC8430957/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34502361/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=International%20Journal%20of%20Molecular%20Sciences&title=Battling%20chemoresistance%20in%20cancer:%20Root%20causes%20and%20strategies%20to%20uproot%20them&author=A%20Ramos&author=S%20Sadeghi&author=H%20Tabatabaeian&volume=22&publication_year=2021&pages=9451&pmid=34502361&doi=10.3390/ijms22179451&)

48. Sarbassov DD, Guertin DA, Ali SM, Sabatini DM. Phosphorylation and regulation of Akt/PKB by the rictor-mTOR complex. Science. 2005;307:1098–1101. doi: 10.1126/science.1106148.  [DOI](https://doi.org/10.1126/science.1106148) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15718470/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Phosphorylation%20and%20regulation%20of%20Akt/PKB%20by%20the%20rictor-mTOR%20complex&author=DD%20Sarbassov&author=DA%20Guertin&author=SM%20Ali&author=DM%20Sabatini&volume=307&publication_year=2005&pages=1098-1101&pmid=15718470&doi=10.1126/science.1106148&)

49. Seo J, Koçak DD, Bartelt LC, Williams CA, Barrera A, Gersbach CA, Reddy TE. AP-1 subunits converge promiscuously at enhancers to potentiate transcription. Genome Research. 2021;31:538–550. doi: 10.1101/gr.267898.120.  [DOI](https://doi.org/10.1101/gr.267898.120) | [PMC free article](/articles/PMC8015846/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33674350/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Research&title=AP-1%20subunits%20converge%20promiscuously%20at%20enhancers%20to%20potentiate%20transcription&author=J%20Seo&author=DD%20Ko%C3%A7ak&author=LC%20Bartelt&author=CA%20Williams&author=A%20Barrera&volume=31&publication_year=2021&pages=538-550&pmid=33674350&doi=10.1101/gr.267898.120&)

50. Shrestha S, Zhang C, Jerde CR, Nie Q, Li H, Offer SM, Diasio RB. Gene-Specific Variant Classifier (DPYD-Varifier) to Identify Deleterious Alleles of Dihydropyrimidine Dehydrogenase. Clinical Pharmacology and Therapeutics. 2018;104:709–718. doi: 10.1002/cpt.1020.  [DOI](https://doi.org/10.1002/cpt.1020) | [PMC free article](/articles/PMC6043412/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29327356/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clinical%20Pharmacology%20and%20Therapeutics&title=Gene-Specific%20Variant%20Classifier%20(DPYD-Varifier)%20to%20Identify%20Deleterious%20Alleles%20of%20Dihydropyrimidine%20Dehydrogenase&author=S%20Shrestha&author=C%20Zhang&author=CR%20Jerde&author=Q%20Nie&author=H%20Li&volume=104&publication_year=2018&pages=709-718&pmid=29327356&doi=10.1002/cpt.1020&)

51. Sommadossi JP, Gewirtz DA, Diasio RB, Aubert C, Cano JP, Goldman ID. Rapid catabolism of 5-fluorouracil in freshly isolated rat hepatocytes as analyzed by high performance liquid chromatography. The Journal of Biological Chemistry. 1982;257:8171–8176.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/7085662/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=The%20Journal%20of%20Biological%20Chemistry&title=Rapid%20catabolism%20of%205-fluorouracil%20in%20freshly%20isolated%20rat%20hepatocytes%20as%20analyzed%20by%20high%20performance%20liquid%20chromatography&author=JP%20Sommadossi&author=DA%20Gewirtz&author=RB%20Diasio&author=C%20Aubert&author=JP%20Cano&volume=257&publication_year=1982&pages=8171-8176&pmid=7085662&)

52. Tan G, Lenhard B. TFBSTools: an R/bioconductor package for transcription factor binding site analysis. Bioinformatics. 2016;32:1555–1556. doi: 10.1093/bioinformatics/btw024.  [DOI](https://doi.org/10.1093/bioinformatics/btw024) | [PMC free article](/articles/PMC4866524/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26794315/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=TFBSTools:%20an%20R/bioconductor%20package%20for%20transcription%20factor%20binding%20site%20analysis&author=G%20Tan&author=B%20Lenhard&volume=32&publication_year=2016&pages=1555-1556&pmid=26794315&doi=10.1093/bioinformatics/btw024&)

53. Wang D, Yang L, Yu W, Wu Q, Lian J, Li F, Liu S, Li A, He Z, Liu J, Sun Z, Yuan W, Zhang Y. Colorectal cancer cell-derived CCL20 recruits regulatory T cells to promote chemoresistance via FOXO1/CEBPB/NF-κB signaling. Journal for Immunotherapy of Cancer. 2019;7:215. doi: 10.1186/s40425-019-0701-2.  [DOI](https://doi.org/10.1186/s40425-019-0701-2) | [PMC free article](/articles/PMC6688336/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31395078/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20for%20Immunotherapy%20of%20Cancer&title=Colorectal%20cancer%20cell-derived%20CCL20%20recruits%20regulatory%20T%20cells%20to%20promote%20chemoresistance%20via%20FOXO1/CEBPB/NF-%CE%BAB%20signaling&author=D%20Wang&author=L%20Yang&author=W%20Yu&author=Q%20Wu&author=J%20Lian&volume=7&publication_year=2019&pages=215&pmid=31395078&doi=10.1186/s40425-019-0701-2&)

54. Yeo NC, Chavez A, Lance-Byrne A, Chan Y, Menn D, Milanova D, Kuo C-C, Guo X, Sharma S, Tung A, Cecchi RJ, Tuttle M, Pradhan S, Lim ET, Davidsohn N, Ebrahimkhani MR, Collins JJ, Lewis NE, Kiani S, Church GM. An enhanced CRISPR repressor for targeted mammalian gene regulation. Nature Methods. 2018;15:611–616. doi: 10.1038/s41592-018-0048-5.  [DOI](https://doi.org/10.1038/s41592-018-0048-5) | [PMC free article](/articles/PMC6129399/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30013045/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature%20Methods&title=An%20enhanced%20CRISPR%20repressor%20for%20targeted%20mammalian%20gene%20regulation&author=NC%20Yeo&author=A%20Chavez&author=A%20Lance-Byrne&author=Y%20Chan&author=D%20Menn&volume=15&publication_year=2018&pages=611-616&pmid=30013045&doi=10.1038/s41592-018-0048-5&)

55. Zerbino DR, Wilder SP, Johnson N, Juettemann T, Flicek PR. The ensembl regulatory build. Genome Biology. 2015;16:56. doi: 10.1186/s13059-015-0621-5.  [DOI](https://doi.org/10.1186/s13059-015-0621-5) | [PMC free article](/articles/PMC4407537/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25887522/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Biology&title=The%20ensembl%20regulatory%20build&author=DR%20Zerbino&author=SP%20Wilder&author=N%20Johnson&author=T%20Juettemann&author=PR%20Flicek&volume=16&publication_year=2015&pages=56&pmid=25887522&doi=10.1186/s13059-015-0621-5&)

56. Zhang X-F, Li K, Gao L, Li S-Z, Chen K, Zhang J-B, Wang D, Tu R-F, Zhang J-X, Tao K-X, Wang G, Zhang X-D. miR-191 promotes tumorigenesis of human colorectal cancer through targeting C/EBPβ. Oncotarget. 2015;6:4144–4158. doi: 10.18632/oncotarget.2864.  [DOI](https://doi.org/10.18632/oncotarget.2864) | [PMC free article](/articles/PMC4414178/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25784653/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncotarget&title=miR-191%20promotes%20tumorigenesis%20of%20human%20colorectal%20cancer%20through%20targeting%20C/EBP%CE%B2&author=X-F%20Zhang&author=K%20Li&author=L%20Gao&author=S-Z%20Li&author=K%20Chen&volume=6&publication_year=2015&pages=4144-4158&pmid=25784653&doi=10.18632/oncotarget.2864&)
