# Effect of NAT2, GSTM1 and CYP2E1 genetic polymorphisms on plasma concentration of isoniazid and its metabolites in patients with tuberculosis, and the assessment of exposure-response relationships

## Metadata
**Authors:** Viktorija Ulanova, Agnija Kivrane, Anda Viksna, Leonora Pahirko, Lauma Freimane, Darja Sadovska, Iveta Ozere, Andra Cirule, Eduards Sevostjanovs, Solveiga Grinberga, Dace Bandere, Renate Ranka
**Journal:** Frontiers in Pharmacology
**Date:** 2024 Mar 22
**DOI:** [10.3389/fphar.2024.1332752](https://doi.org/10.3389/fphar.2024.1332752)
**PMID:** 38584604
**PMCID:** PMC10995391
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10995391/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC10995391/pdf/fphar-15-1332752.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC10995391/pdf/fphar-15-1332752.pdf)

## Abstract

Objectives: Isoniazid is a key drug in the chemotherapy of tuberculosis (TB), however, interindividual variability in pharmacokinetic parameters and drug plasma levels may affect drug responses including drug induced hepatotoxicity. The current study investigated the relationships between isoniazid exposure and isoniazid metabolism-related genetic factors in the context of occurrence of drug induced hepatotoxicity and TB treatment outcomes.

Methods: Demographic characteristics and clinical information were collected in a prospective TB cohort study in Latvia (N = 34). Time to sputum culture conversion (tSCC) was used as a treatment response marker. Blood plasma concentrations of isoniazid (INH) and its metabolites acetylisoniazid (AcINH) and isonicotinic acid (INA) were determined at three time points (pre-dose (0 h), 2 h and 6 h after drug intake) using liquid chromatography-tandem mass spectrometry. Genetic variations of three key INH-metabolizing enzymes (NAT2, CYP2E1, and GSTM1) were investigated by application PCR- and Next-generation sequencing-based methods. Depending on variables, group comparisons were performed by Student’s t-test, one-way ANOVA, Mann-Whitney-Wilcoxon, and Kruskal-Wallis tests. Pearson correlation coefficient was calculated for the pairs of normally distributed variables; model with rank transformations were used for non-normally distributed variables. Time-to-event analysis was performed to analyze the tSCC data. The cumulative probability of tSCC was obtained using Kaplan-Meier estimators. Cox proportional hazards models were fitted to estimate hazard rate ratios of successful tSCC.

Results: High TB treatment success rate (94.1%) was achieved despite the variability in INH exposure. Clinical and demographic factors were not associated with either tSCC, hepatotoxicity, or INH pharmacokinetics parameters. Correlations between plasma concentrations of INH and its metabolites were NAT2 phenotype-dependent, while GSTM1 genetic variants did not showed any effects. CYP2E1*6 (T > A) allelic variant was associated with INH pharmacokinetic parameters. Decreased level of AcINH was associated with hepatotoxicity, while decreased values of INA/INH and AcINH/INH were associated with month two sputum culture positivity.

Conclusion: Our findings suggest that CYP2E1, but not GSTM1, significantly affects the INH pharmacokinetics along with NAT2. AcINH plasma level could serve as a biomarker for INH-related hepatotoxicity, and the inclusion of INH metabolite screening in TB therapeutic drug monitoring could be beneficial in clinical studies for determination of optimal dosing strategies.

Keywords: NAT2, GSTM1, CYP2E1, pharmacogenomics, isoniazid, tuberculosis

## 1 Introduction

Tuberculosis (TB) is one of the leading infectious causes of death worldwide. The World Health Organization estimates that 10.6 million people developed TB in 2021 ([World Health Organization, 2022](#B60)). Treatment success rate of 85% for drug-susceptible TB is achieved with 2-month short-course chemotherapy containing isoniazid (INH), rifampicin (RIF), ethambutol (EMB), and pyrazinamide (PZA), followed by INH and RIF for additional 4 months ([World Health Organization, 2020](#B59)). INH is a key drug in the chemotherapy of TB, however, interindividual variability in pharmacokinetic parameters and drug plasma levels may affect drug responses ([Ellard and Gammon, 1976](#B14); [Aarnoutse, 2011](#B1)). Therapeutic drug monitoring, which utilizes a technique using plasma drug concentration measurements in clinical specimens, was highlighted as a personized treatment approach for the evaluation of the effectiveness of anti-TB treatment and achieving therapeutic goals ([Alsultan and Peloquin, 2014](#B3); [Perumal et al., 2020](#B37)). However, the current data correlating INH plasma concentrations, adverse drug events and TB treatment outcome are still limited, and the involvement of contributing factors is not sufficiently studied.

Several enzymes are known to metabolize INH, and a detailed model of candidate enzymes in the metabolism of INH in liver cells is described elsewhere ([Klein et al., 2016](#B28)). The main metabolic pathway of INH is acetylation by arylamine N-acetyltransferase 2 (NAT2) to N-acetylisoniazid (AcINH) ([Ellard and Gammon, 1976](#B14); [Daly, 2021](#B12)). Other INH metabolites, such as hydrazine (Hz), isonicotinic acid (INA), acetylhydrazine (AcHz), and diacetylhydrazine (diAcHz) are generated by hydrolysis, glycine conjugation, hydrazone formation, and further acetylation. None of the INH metabolites are active, while Hz and AcHz are considered to be hepatotoxic ([Becker et al., 2007](#B6)). In turn, INH, AcHz, and Hz are likely oxidized, in part, by cytochrome P450 2E1 (CYP2E1) enzyme into potentially hepatotoxic intermediates, which could be further conjugated by glutathione S-transferase (GST) enzyme family with glutathione and effectively removed by excretion ([Klein et al., 2016](#B28)). Currently, *NAT2* is the most extensively studied pharmacogenetic risk factor for the development of INH-related drug induced hepatotoxicity (DIH) ([Daly, 2021](#B12)). It is well-known that the NAT2 enzyme activity exhibits substantial interindividual variability resulting in different INH plasma half-lives ([Ellard and Gammon, 1976](#B14); [Aarnoutse, 2011](#B1)). Genetic polymorphisms of *NAT2* gene are responsible for the variations in the acetylation capacity, and the recent meta-analysis study has shown that NAT2 deficiency was significantly associated with the likelihood of experiencing DIH ([Richardson et al., 2019](#B40)). Also, it was proposed that INH dose adjustment based on the *NAT2* genotype before prescribing TB treatment could improve clinical outcome and reduce the risk of INH-induced liver injury, however, the current level of evidence is not sufficient to recommend dose optimization ([Azuma et al., 2013](#B4); [McIlleron and Chirehwa, 2018](#B32); [Khan and Das, 2022](#B24)). A number of risk factors for the development of INH-related liver injury have been identified, including age and some environmental factors, such as alcohol consumption, and genetic variations in other drug metabolizing enzymes, such as *CYP2E1* and *GSTM1*, could also be related to the alterations of the INH metabolism and accumulation of toxic substances, which, subsequently, can lead to adverse drug events and affect treatment outcome ([Ellard and Gammon, 1976](#B14); [Yang et al., 2019](#B64)).

This study has two objectives: (1) to examine the impact of exposure to INH and its metabolites on TB treatment outcome and DIH; (2) to investigate factors which influence variability in INH pharmacokinetics. Towards these objectives we performed pharmacokinetic profiling of INH and its major metabolites (AcINH, INA) in patients with pulmonary TB, and investigated genetic variations of three key INH-metabolizing enzymes (NAT2, CYP2E1, and GSTM1) along with an array of patient-related factors. The obtained data provide a better insight into INH pharmacogenetics thus contributing to future implementation of personalized approach for enhancing the efficacy, tolerability and limiting the toxicity of existing anti-TB medication.

## 2 Materials and methods

### 2.1 Study participants and data collection

Study cohort was comprised of 34 patients with diagnosed drug susceptible pulmonary TB admitted to the Riga East University Hospital (REUH), Centre of Tuberculosis and Lung Diseases. In order to minimize the impact of other health conditions on the study results, the following exclusion criteria were applied: age ≤ 18 years, pregnancy or lactation, history of cancer, other infectious diseases (e.g., HVC, HVB, HIV, and AIDS), and renal failure.

The diagnosis of TB was confirmed radiologically, symptomatically, and/or bacteriologically on Löwenstein–Jensen (L-J) solid medium. Xpert/RIF Ultra test was used for detection of TB and resistance to RIF with additional Line probe assay for resistance-conferring mutation/INH resistance detection before phenotypic drug susceptibility testing. Sputum smear/culture findings, laboratory test values and treatment outcomes were recorded. Time to sputum culture conversion (tSCC) was determined by the number of days to the first negative inoculation of Löwenstein-Jensen (L-J) medium (including the last day), indicating the treatment outcome. “Treatment success” was defined as the sum of “cured” and “treatment completed” ([World Health Organization, 2022](#B61)). All patients received the WHO-recommended treatment regimen for drug-susceptible TB which consistent of a 2-month chemotherapy containing isoniazid INH (4–6 mg/kg), RIF (8–12 mg/kg), PZA (20–30 mg/kg) and EMB (15–25 mg/kg) followed by INH and RIF for additional 4 months. The anti-TB drugs were all single drug products rather than fixed-dose combinations, administered once daily.

Clinical biochemistry tests of liver and renal function were performed at baseline and on day 10–12 of treatment. The reference ranges for transaminases [alanine aminotransferase (ALAT), aspartate aminotransferase (ASAT)], direct and total bilirubin were adopted from specific reference ranges of REUH Laboratory Service laboratory “Gaiļezers” to determine the upper limit of normal range (ULN); the reference range were as follows: alanine aminotransferase (ALAT), 8–41 U/L; aspartate aminotransferase (ASAT), 8–40 U/L; direct (conjugated) bilirubin, < 3.4 µ mol/L; total bilirubin, 5–21 µ mol/L. The DIH case was confirmed based on the increase of ALAT/ASAT values above the ULN on day 10–12 of TB treatment with/without increase of total and conjugated bilirubin levels: mild hyperfermentemia was characterized by elevated serum ALAT/ASAT levels 1.5–5 times above the ULN; moderate hyperfermentemia: 5–10 times above the ULN, and severe hyperfermentemia: >10 times above ULN.

Patient data [age, biological sex, height, weight, sputum microscopy results, time to sputum culture conversion (tSCC), laboratory findings, treatment outcome] were obtained from medical records; smoking status, self-reported alcohol intake were obtained from patients’ questionnaires. Patient’s nutritional status was reported as body mass index (BMI), which was calculated based on height and body weight. According to the TB treatment guidelines, TB treatment was initiated following the directly observed treatment (DOT) strategy for health facility-based patients, and the video observed directly observed treatment (VDOT) strategy was implemented for outpatient department patients.

Patients were enrolled in the study after providing written informed consent for the use of their phenotypic and genetic data. The study was approved by the Central Medical Ethics Committee of Latvia (approval No. 01-29.1/1 and approval No. 01-29.1.2/1736), the Ethics Committee of the Riga East University Hospital (approval No 24-A/15 and approval No. A1/1.1–07.1/20/1809), and the Riga Stradins University Research Ethics Committee (approval No. 6-3/1/6 and approval No. 105/28.01.2016.). This study was conducted in accordance with the Declaration of Helsinki.

### 2.2 DNA and blood plasma samples

Blood sample collection was carried on 10–12 days after the treatment onset. Blood samples were collected by medical personnel using vacutainers with EDTA at three time points: pre-dose (0 h), 2 h, and 6 h after drug intake. Blood samples were immediately centrifuged at 4,000 rpm (3,488 × g) for 15 min at 4°C to separate the plasma; plasma aliquots were immediately frozen and stored at −70°C until further analysis. Genomic DNA was extracted from the peripheral white blood cells using the standard phenol-chloroform method described elsewhere ([Sambrook et al., 1989](#B43)). DNA sample concentrations were measured using a spectrophotometer (ND-1000 UV-VIS Spectrophotometer, NanoDrop Technologies, United States). DNA samples were stored at −20°C, until it was used as a template for PCR amplification.

### 2.3 NAT2 genotyping

A 1211-bp fragment, which contains the entire coding region of *NAT2* was amplified by PCR using primers P100 5′-GTC​ACA​CGA​GGA​AAT​CAA​ATG​C and P56 5′-GTT​TTC​TAG​CAT​GAA​TCA​CTC​TGC-3’ ([Cascorbi et al., 1996](#B8)). The PCR was performed in a final volume of 25 μL. The master mix contained (per reaction): 1x Phusion Buffer with 7.5 mM MgCl_2_, 200 mM of each dNTPs, 0.3 mM of each primer, 0.4 U of Phusion Hot Start II High-Fidelity DNA polymerase (Thermo Fisher Scientific, United States) and 5 ng of DNA template. The PCR assays were performed under the following conditions: an initial denaturation at 98°C 30 s; 40 cycles of denaturation at 98°C for 10s, primer annealing at 65°C for 30 s, and elongation at 72°C for 30 s; and a final elongation step at 72°C for 5 min. PCR products were purified using the ExoI and FastAP enzymes (Thermo Fisher Scientific, United States) and subsequently sequenced on both DNA strands using P100 and three additional internal primers (P90, 5′-ACA​CAA​GGG​TTT​ATT​TTG​TTC​C-3’; NAT2_R_for, 5′ TAA​TTC​TAG​AGG​CTG​CCA​CAT​C-3’; NAT2_S_for, 5′-GAA​TAC​ATA​CCT​GCA​GAC​GTC​TC-3′) ([Cascorbi et al., 1996](#B8); [Igumnova et al., 2016](#B21)). Sequencing was performed using the ABI PRISM BigDye Terminator v3.1 Cycle Sequencing Kit (Applied Biosystems) according to the manufacturer’s recommendations on an ABI Prism 3,100 Genetic Analyzer (Perkin-Elmer, United States).

For *NAT2* gene analysis of heterozygous samples, subsequent cloning of 1211-bp PCR fragment into the pJET1.2/blunt cloning vector was performed using the CloneJET PCR Cloning Kit (Thermo Fisher Scientific, United States) according to the manufacturer’s instructions. Plasmids with confirmed *NAT2* gene inserts were purified from bacterial cultures using GeneJET Plasmid Miniprep Kit (Thermo Fisher Scientific, United States), and the inserts were sequenced.

Sequencing data analysis was performed using CodonCode Aligner software (CodonCode Corporation, United States) with the sequence of human gene for arylamine N-acetyltransferase (EC 2.3.1.5) (GenBank: [X14672.1](https://www.ncbi.nlm.nih.gov/nuccore/X14672.1)) as the reference. A possible NAT2 phenotype was assigned based on the obtained genotyping data of conventional 7-SNP signature, as the presence/absence of signature mutations were linked to the resultant protein activity ([Sim et al., 2014](#B48)). In addition, a database of human *NAT2* alleles was used ([http://nat.mbg.duth.gr/Human%20NAT2%20alleles_2013.htm](http://nat.mbg.duth.gr/Human%20NAT2%20alleles_2013.htm)). According to ([Stanley and Sim, 2008](#B52)) individuals were classified as rapid (carrying two rapid *NAT2* alleles), intermediate (one rapid and one slow allele) or slow (two slow alleles) acetylators.

### 2.4 GSTM1 genotyping

The *GSTM1* gene double deletion genotype (0/0) was detected based on the comparative duplex PCR assay as described previously ([Igumnova et al., 2016](#B21)). Briefly, the PCR master mix contained (per reaction) 1X Taq Buffer with (NH_4_)_2_SO_4_, 2.5 mM MgCl_2_, 100 mM of each dNTPs, 0.2 mM of each primer Beta1 (5′-GGT​TGG​CCA​ATC​TAC​TCC​CAG​G-3′), Beta2 (5′-GCT​CAC​TCA​GTG​TGG​CAA​AG-3′), M1 (5′-CTG​CCC​TAC​TTG​ATT​GAT​GGG-3′) and M2 (5′-CTG​GAT​TGT​AGC​AGA​TCA​TGC-3′), 1.25 U of Taq DNA polymerase and 28 ng of DNA template. PCR assays were performed under the following conditions: an initial denaturation at 95°C for 10 min; 35 cycles of denaturation at 95°C for 30 s, primer annealing at 64°C for 30 s, and elongation at 72°C for 40 s; and a final elongation step at 72°C for 2 min. The PCR products were separated and visualized using electrophoresis on a 1.5% agarose gel containing 0.2 mg of ethidium bromide/ml with transillumination under UV light. The homozygous GSTM1 null genotype (0/0) was detected by the absence of the 273-bp. Individuals who were presented with the 273-bp fragment were classified as GSTM1 plus genotype (A/A or A/0) and contained at least one wild-type allele.

### 2.5 Full-length CYP2E1 gene analysis

For the full-length *CYP2E1* gene analysis the next-generation sequencing (NGS)-based assay targeting all 9 gene exons with interleaving introns, exonic, untranslated (UTR), upstream and intergenic regions was used. The full-length *CYP2E1* gene sequencing assay, including PCR amplification and NGS paired-end libraries preparation workflow was carried out according to the published protocol ([Igumnova et al., 2022](#B22)) with a few minor modifications in the sequencing data analysis. Specifically, functional annotation of the identified single nucleotide polymorphisms (SNPs) and Indels was performed using Snpeff eff (vGalaxy Version 4.3 + T.galaxy1) and online-based wANNOVAR tool ([http://wannovar.wglab.org/](http://wannovar.wglab.org/)) ([Wang et al., 2010](#B56)). Reference SNP reports accumulated in dbSNP database ([https://www.ncbi.nlm.nih.gov/snp/](https://www.ncbi.nlm.nih.gov/snp/)), Varsome ([https://varsome.com/](https://varsome.com/)), PharmGKB ([https://www.pharmgkb.org/](https://www.pharmgkb.org/)) and the American College of Medical Genetics and Genomics (ACMG) were used for the detected variant identification and annotation ([Whirl-Carrillo et al., 2012](#B58)).

Sequencing data analysis was performed on the Galaxy online-based platform using the public server at [https://usegalaxy.org](https://usegalaxy.org/). ([Jalili et al., 2020](#B23)). Trimmomatic (v0.38) was used to trim adapter sequences and low quality read ends (Phred quality score < 20). Reads were mapped against the human reference genome (GRCh38.p13, GCF 000001405.39) using Map with BWA-MEM (v0.7.17.1). BAM filter (v0.5.9) was applied to keep only mapped reads, discard reads shorter than 50 bp, but Mark duplicates (v2.18.2.2) was used to finalize deduplication. Samtools depth (v1.9) was used to compute the depth at each position in genomic coordinates Chr10 ([NC_000010.11](https://www.ncbi.nlm.nih.gov/nuccore/NC_000010.11), 133524920–133539123). FreeBayes (v1.3.1) for genomic variant calling using BED file in specific target region ([NC_000010.11](https://www.ncbi.nlm.nih.gov/nuccore/NC_000010.11), 133524920–133539123) was implemented. Detected variants were filtered using the following arguments: minimum coverage to process a site ≥10, minimum mapping quality ≥10, minimum base quality ≥20, minimum alternate fracture 0.2, minimum alternate count of observations ≥6 supporting an alternate allele within a single individual in order to evaluate the position. Variants with quality <5 were filtered out to reduce false positive calls. All detected genomic variants were visually inspected using Integrative Genome Viewer, v2.8.9 ([Robinson et al., 2011](#B41)). The sequence analysis and SNV identification were performed using FinchTV and MEGA software ([Erasmus, 2021](#B15); [Taiwo et al., 2022](#B54)) with the sequence of human *CYP2E1* gene (E.C.1.14.13.n7) (GenBank: [NG_008383.1](https://www.ncbi.nlm.nih.gov/nuccore/NG_008383.1)) as the reference.

### 2.6 Liquid chromatography-tandem mass spectrometry (LC-MS/MS)

Blood plasma concentrations of INH and its main metabolites AcINH and INA were determined at three time points (pre-dose (0 h), 2 h and 6 h after drug intake) using liquid chromatography-tandem mass spectrometry (LC-MS/MS) method as previously described ([Kivrane et al., 2021](#B27)). Briefly, plasma **s**amples were analysed using An Acquity UPLC H-Class chromatographic system (Waters, Milford, MA, United States) coupled to a XEVO TQ-S tandem mass spectrometer (Waters, Milford, MA, United States). The chromatographic separation was achieved on the Waters Acquity UPLC BEH C8 column (2.1 mm × 75 mm; 1.7 μm). The multiple reaction monitoring mode (MRM) was used for detection and quantification of the analytes. Data acquisition and analysis were performed using MassLynx software and TargetLynx module (version 4.1., Waters, Milford, MA, United States). Calibration ranges were 0.16–10.0 μg/mL for INH, AcINH and INA, respectively; the lower limits of quantification (LLOQ) were 0.16 μg/mL for INH and both two primary metabolites. The within- and between-run accuracy in all concentration levels ranged from 87.2% to 113.6%, but within- and between-run precision was between 1.6% and 14.9% (at the LLOQ level CV<20%).

### 2.7 Determination of INH pharmacokinetic parameters

The following INH pharmacokinetic parameters were evaluated: (1) INH plasma peak concentration at 2 h (C_max_ 2 h); (2) area under the time–concentration curve 0–6 h (AUC_0–6h_) of INH and its two major metabolites (AcINH and INA); (3) metabolic ratio (MR) AcINH/INH and MR INA/INH (calculated as the ratio of metabolite to the parent drug in 2 h plasma and expressed as a range) ([Chen et al., 2017](#B9); [Sekaggya-Wiltshire, Lamorde, et al., 2018](#B44)). Therapeutic range for INH was defined as 3–6 μg/mL approximately 2 h after ingestion ([Märtson et al., 2021](#B30)).

### 2.8 Statistical analysis

All statistical analysis were conducted using R statistical software version 4.2.1. (R Core Team (2022), R Foundation for Statistical Computing, Vienna, Austria; [https://r-project.org](https://r-project.org)).

Categorical variables were summarized as frequencies and percentages; equality of proportions and association between two-categorical variables were tested using Chi-square tests. When Chi-square test were not applicable, Fisher’s exact test was used instead for association test. Shapiro-Wilk test was used to verify the normality assumption, and, where possible, log-transformation was used to achieve the normal distribution in the data. For normally distributed variables, two or more group comparisons were performed by Student’s *t*-test and one-way ANOVA, respectively. Models with rank transformations were used for non-normally distributed variables, two and three or more group comparisons were performed by Mann-Whitney-Wilcoxon and Kruskal-Wallis test, respectively.

Pearson correlation coefficient with the corresponding 95% confidence interval and *p*-values were calculated for the pairs of normally distributed variables; model with rank transformations were used for non-normally distributed variables (i.e., AcINH/INH and INA/INH ratios). When applicable, during the correlation analysis, the six pharmacokinetic parameters of INH were adjusted for age, sex, and INH dose (mg/kg); AcINH and INA (AUC0-6 h) values were log-transformed and the correlations were also stratified by NAT2 acetylator status and *GSTM1* genotypes.

Time-to-event analysis was performed to analyse the tSCC data. Patients whose treatment outcomes were “treatment completed” or “lost to follow-up” were considered as censored. The cumulative probability of tSCC, stratified by *GSTM1* genotypes and *NAT2* phenotypes, was obtained using Kaplan-Meier estimators. Log-rank tests were performed to assess differences between the groups. Median tSCC were calculated for both GSTM1 and NAT2 groups. Additionally, the mean tSCC, restricted at time point 60 days, was compared between subgroups of *GSTM1* and *NAT2* (R library survRM2). Confidence intervals were constructed for the group medians and for the differences of restricted mean tSCC. Cox proportional hazards models were fitted to estimate hazard rate (HR) ratios of successful tSCC. In a case of proportional hazards, the HR of successful tSCC can be interpreted as the odds for an individual in the group with higher hazard to reach the endpoint first ([Spruance et al., 2004](#B51)). In the Cox regression models the following predictors were included for the whole study cohort (*n* = 34): (1) age, sex, smear microscopy, and standardized values of AcINH/INH ratio; (2) age, sex, smear microscopy, and standardized values of INA/INH ratio; (3) *GSTM1* genotype, NAT2 phenotype, CYP2E1*6 (T > A) allelic variant (rs6413432), and sputum smear microscopy.

Linkage disequilibrium (LD) was assessed by Ensembl Linkage Disequilibrium Calculator ([https://www.ensembl.org/Homo_sapiens/Tools/LD?db=core](https://www.ensembl.org/Homo_sapiens/Tools/LD?db=core)) utilizing genotyping data from the 1000 Genomes Project database for SNP pairwise comparisons through the r^2^ coefficient. All the SNPs were filtered using a pairwise tagging algorithm with high LD r^2^ measure ≥ 0.8 and minor allele frequency (MAF) > 0.05. All detected SNPs genotypes were tested for agreement with Hardy-Weinberg equilibrium (HWE) by Fisher exact probability test based on the genotype distributions in this study. As a result, seven SNPs were chosen as the representatives to analyse the association with patients’ clinical characteristics. *p*-values ≤ 0.05 were considered statistically significant.

## 3 Results

### 3.1 TB treatment outcome and DIH cases

Patients’ characteristics are presented in [Table 1](#T1) and [Supplementary Table S1](#s12). In total, 34 patients with pulmonary TB were included. All patients were Caucasians. The mean patient age was 46.3 (±12.2) years, and the majority of the patients were male (82.4%, 28/34), current smokers (73.5%, 25/34), and with normal BMI (61.8%, 21/34) ([Table 1](#T1)). One-third of the patients (32.4%, 11/34) reported daily alcohol intake. In 55.9% (19/34) cases smear microscopy result was positive.

### TABLE 1.

| Characteristic |  |  |
| --- | --- | --- |
| Categorical data. no/total (%) |  |  |
| Biological sex | Male | 28 (82.4) |
| Female | 6 (17.7) |  |
| Smoking status | Non-smoker | 9 (26.5) |
| Smoker | 25 (73.5) |  |
| BMI* | Underweight | 8 (23.5) |
| Normal | 21 (61.8) |  |
| Overweight | 5 (14.7) |  |
| Smear microscopy | Negative | 15 (44.1) |
| Positive | 19 (55.9) |  |
| Self-reported alcohol intake | Yes | 11 (32.4) |
| No | 23 (67.7) |  |
| Non-categorical data. mean (±SD) |  |  |
| Age |   | 46.3 (±12.2) |
| INH dose (mg/kg) | 4.64 (±0.83) |  |

Table 1 Caption: Study participants’ characteristics (patients with pulmonary tuberculosis, n = 34).

TB treatment outcome was successful for all but two patients (treatment success rate = 94.1%): the majority of the patients were cured (28/34, 82.4%), and four (11.8%) patients completed the treatment ([Table 2](#T2)). Two patients (5.9%) were lost to follow-up. Overall, tSCC data were available for 28 patients (i.e., those with “cured” treatment outcome). Median tSCC was 52 (11–197) days; 16 patients (57.1%) became *M. tuberculosis* culture-negative in less than 60 days, while for 12 patients (42.9%) tSCC was > 60 days.

### TABLE 2.

| Characteristics |  | Treatment outcome* |  | tSCC (n = 28**) |  | Hepatotoxicity |  |
| --- | --- | --- | --- | --- | --- | --- | --- |
| Cured | Treatment completed | <60 days | >60 days | Yes | No |  |  |
| All patients. no/total (%) |  | 28/34 (82.4) | 4/34 (11.8) | 16/28 (57.1) | 12/28 (42.9) | 4/34 (11.8) | 30/34 (88.2) |
| Categorical data. no/total (%) |  |   |   |   |   |   |   |
| Biological sex | Male | 24/28 (85.7) | 3/28 (10.7) | 13/24 (54.2) | 11/24 (45.8) | 2/28 (7.1) | 26/28 (92.9) |
| Female | 4/6 (66.7) | 1/6 (16.7) | 3/4 (75.0) | 1/4 (25.0) | 2/6 (33.3) | 4/6 (66.7) |  |
| p-value | 0.51 |   | 0.44 |   | 0.071 |   |  |
| OR (95% CI) | 2.0 (0.16, 24.3) |   | 0.39 (0.04, 4.35) |   | 0.15 (0.02, 1.42) |   |  |
| Smoking status | Non-smoker | 5/9 (55.6) | 3/9 (33.3) | 3/5 (60.0) | 2/5 (40.0) | 2/9 (22.2) | 7/9 (77.8) |
| Smoker | 23/25 (92.0) | 1/25 (4.0) | 13/23 (56.5) | 10/23 (43.5) | 2/25 (8.0) | 23/25 (92.0) |  |
| p-value | 0.039 |   | 0.89 |   | 0.26 |   |  |
| OR (95% CI) | 13.8 (1.2, 161.7) |   | 0.87 (0.12, 6.2) |   | 0.30 (0.04, 2.6) |   |  |
| BMI*** | Underweight | 5/8 (62.5) | 2/8 (25.0) | 3/5 (60.0) | 2/5 (40%) | 1/8 (12.5) | 7/8 (87.5) |
| Normal | 19/21 (90.5) | 1/21 (4.8) | 10/19 (52.6) | 9/19 (47.4) | 2/21 (9.5) | 19/21 (90.5) |  |
| Overweight | 4/5 (80.0) | 1/5 (20.0) | 3/4 (75.0) | 1/4 (25%) | 1/5 (20.0) | 4/5 (80.0) |  |
| p-value | 0.21 |   | 0.71 |   | 0.81 |   |  |
| Smear microscopy | Negative | 9/15 (60.0) | 4/15 (26.7) | 7/9 (77.8) | 2/9 (22.2) | 2/15 (13.3) | 13/15 (86.7) |
| Positive | 19/19 (100.0) | 0/19 (0.0) | 9/19 (47.4) | 10/19 (52.6) | 2/19 (10.5) | 17/19 (89.5) |  |
| p-value | 0.02 |   | 0.13 |   | 0.8 |   |  |
| OR (95% CI) |   |   | 0.26 (0.04, 1.57) |   | 0.76 (0.09, 6.2) |   |  |
| Daily alcohol intake | Yes | 10/11 (90.9) | 1/11 (9.1) | 6/10 (60.0) | 4/10 (40.0) | 2/11 (18.1) | 9/11 (81.8) |
| No | 18/23 (78.3) | 3/23 (13.0) | 10/18 (55.6) | 8/18 (44.4) | 2/23 (8.7) | 21/23 (91.3) |  |
| p-value | 1 |   | 0.82 |   | 0.42 |   |  |
| OR (95% CI) | 1.6 (0.15, 18.2) |   | 1.2 (0.25, 5.8) |   | 2.33 (0.28, 19.2) |   |  |
| Non-categorical data. mean (±SD) |  |  |  |  |  |  |  |
| Age |   | 48.1 (±10.8) | 43.0 (±17.1) | 47.4 (±12.2) | 48.9 (±9.1) | 52.5 (±9.0) | 45.5 (±12.5) |
| p-value | 0.6 |   | 0.72 |   | 0.24 |   |  |
| INH dose (mg/kg) |   | 4.67 (±0.74) | 4.24 (±1.41) | 4.68 (±0.78) | 4.66 (±0.71) | 4.93 (±0.91) | 4.60 (±0.82) |
| p-value | 0.58 |   | 0.94 |   | 0.53 |   |  |
| AUC 0–6h, µg/mL·h | INH | 10.8 (±5.5) | 12.7 (±6.8) | 10.1 (±5.9) | 11.8 (±5.1) | 13.8 (±4.7) | 10.9 (±5.6) |
| p-value | 0.63 |   | 0.4 |   | 0.31 |   |  |
| AcINH | 7.5 (±4.0) | 4.42 (±1.7) | 8.6 (±4.0) | 6.1 (±3.7) | 4.6 (±0.7) | 7.4 (±3.9) |  |
| p-value | 0.088 |   | 0.102 |   | 0.019 |   |  |
| INA | 2.1 (±0.9) | 1.5 (±0.5) | 2.3 (±1.0) | 1.9 (±0.8) | 1.7 (±0.6) | 2.1 (±0.9) |  |
| p-value | 0.18 |   | 0.2 |   | 0.43 |   |  |
| Cmax 2h, µg/mL | INH | 2.6 (±1.5) | 3.0 (±1.6) | 2.5 (±1.5) | 2.8 (±1.5) | 2.7 (±1.5) | 2.7 (±1.5) |
| p-value | 0.67 |   | 0.51 |   | 0.99 |   |  |
| Non-categorical data. median (IQR) |  |  |  |  |  |  |  |
| Metabolic ratio | AcINH/INH | 0.46 (0.35–1.64) | 0.27 (0.20–1.36) | 0.91 (0.42–1.74) | 0.39 (0.33–0.50) | 0.40 (0.32–0.44) | 0.47 (0.34–1.66) |
| p-value | 0.12 |   | 0.053 |   | 0.26 |   |  |
| INA/INH | 0.21 (0.13–0.28) | 0.12 (0.08–0.31) | 0.23 (0.16–0.31) | 0.14 (0.10–0.21) | 0.12 (0.10–0.15) | 0.19 (0.12–0.28) |  |
| p-value | 0.19 |   | 0.026 |   | 0.12 |   |  |

Table 2 Caption: Treatment outcome, response and hepatotoxicity in patients with TB (n = 34) depending on various factors.

In total, after 10–12 days of therapy, transaminase levels increased in 4 patients (11.76%): mild and moderate DIH developed in one patient each, and severe DIH—in two patients. In two of these patients ALAT/ASAT levels were increased before therapy onset, and in 1 patient simultaneous changes in total and conjugated bilirubin levels were recorded ([Table 2](#T2), [Supplementary Table S1](#s12); [Supplementary Figure S1](#s12)).

### 3.2 INH pharmacokinetic data

Mean plasma values of INH and both metabolites are presented in [Table 3](#T3). Overall, 64.7% (22/34) of the patients had INH C_max_ 2 h value below the therapeutic range window (<3.0 μg/mL; [Supplementary Table S1](#s12)); the mean INH C_max_ 2 h of the cohort was 2.7 (±1.4) µg/mL.

### TABLE 3.

| Total |  | Pharmacokinetic parameters AUC0–6h, µg/mL·h; mean (±SD) |  |  | Cmax, 2 h, µg/mL; mean (±SD) | Metabolic ratio; median (IQR) |  |
| --- | --- | --- | --- | --- | --- | --- | --- |
| INH | AcINH | INA | INH | AcINH/INH | INA/INH |  |  |
|   |   | 11.2 (±5.5) | 7.0 (±3.8) | 2.0 (±0.9) | 2.7 (±1.4) | 0.46 (0.34–1.56) | 0.16 (0.12–0.28) |
| Biological sex | Male (n = 28) | 11.2 (±5.5) | 7.4 (±4.0) | 2.1 (±0.9) | 2.8 (±1.5) | 0.46 (0.34–1.64) | 0.19 (0.13–0.28) |
| Female (n = 6) | 11.3 (±5.9) | 5.5 (±2.2) | 1.6 (±0.6) | 2.5 (±1.5) | 0.45 (0.35–0.49) | 0.11 (0.10–0.20) |  |
| p-value | 0.98 | 0.28 | 0.22 | 0.66 | 0.84 | 0.26 |  |
| Smoking status | Non-smoker (n = 9) | 11.2 (±5.9) | 6.7 (±3.2) | 1.7 (±0.6) | 2.6 (±1.4) | 0.43 (0.35–1.70) | 0.15 (0.10–0.22) |
| Smoker (n = 25) | 11.3 (±5.4) | 7.2 (±4.1) | 2.1 (±1.0) | 2.8 (±1.5) | 0.46 (0.33–1.34) | 0.20 (0.12–0.28) |  |
| p-value | 0.99 | 0.91 | 0.19 | 0.69 | 0.79 | 0.36 |  |
| BMI* | Underweight (n = 8) | 11.8 (±4.1) | 7.8 (±4.0) | 2.3 (±1.3) | 2.9 (±0.9) | 0.47 (0.40–1.32) | 0.21 (0.13–0.27) |
| Normal (n = 21) | 11.7 (±6.2) | 7.0 (±4.2) | 1.9 (±0.8) | 2.9 (±1.7) | 0.36 (0.32–1.34) | 0.15 (0.11–0.24) |  |
| Overweight (n = 5) | 8.6 (±3.9) | 6.2 (±1.9) | 1.8 (±0.6) | 1.9 (±0.9) | 0.49 (0.43–1.66) | 0.23 (0.17–0.36) |  |
| p-value | 0.51 | 0.79 | 0.62 | 0.42 | 0.65 | 0.62 |  |
| Daily alcohol intake | Yes (n = 11) | 11.1 (±5.0) | 6.7 (±4.1) | 1.9 (±0.7) | 2.7 (±1.4) | 0.46 (0.35–1.0) | 0.20 (0.14–0.27) |
| No (n = 23) | 11.3 (±5.8) | 7.2 (±3.8) | 2.3 (±1.2) | 2.8 (±1.5) | 0.46 (0.34–1.67) | 0.16 (0.10–0.26) |  |
| p-value | 0.91 | 0.54 | 0.49 | 0.8 | 0.72 | 0.42 |  |

Table 3 Caption: Pharmacokinetic parameters for isoniazid stratified by patient characteristics.

### 3.3 Genetic variability of INH-metabolizing enzymes

#### 3.3.1 NAT2 and GSTM1

During the genotyping analysis, *NAT2* SA phenotype-related genotypes were detected in 67.7% (23/34), and *NAT2* IA—in 32.4% (11/34) of the patients, respectively ([Table 4](#T4)). *GSTM1*-null genotype was present in 58.8% (20/34), and *GSTM1*-plus genotype—in 41.2% (14/34) of the patients, respectively.

| Total |  | AUC0–6h, µg/mL·h; EMM, SE, CI 95% |  |  | Cmax, 2 h, µg/mL | Metabolic ratio; median (IQR) |  |
| --- | --- | --- | --- | --- | --- | --- | --- |
| INH | AcINH | INA | INH | AcINH/INH | INA/INH |  |  |
|   |   | 11.3, SE = 1.26, CI=(8.7, 13.9) | 5.7, SE = 1.14, CI=(4.39, 7.39) | 1.69, SE = 1.1, CI=(1.39, 2.06) | 2.64, SE = 0.33, CI=(1.97, 3.31) | 0.46 (0.34–1.56) | 0.16 (0.12–0.28) |
| NAT2 phenotype | SA (n = 23) | 13.22, SE = 1.07, CI=(11.03, 15.40) | 4.48, SE = 1.09, CI=(3.78, 5.37) | 1.59, SE = 1.11, CI=(1.29, 1.96) | 3.01, SE = 0.32, CI=(2.35, 3.66) | 0.35 (0.31–0.46) | 0.13 (0.10–0.16) |
| IA (n = 11) | 5.87, SE = 1.54, CI=(2.72, 9.02) | 11.02, SE = 1.13, CI=(8.58, 14.30) | 2.02, SE = 1.16, CI=(1.50, 2.74) | 1.62, SE = 0.46, CI=(0.68, 2.56) | 1.71 (1.65–2.11) | 0.38 (0.28–0.43) |  |
| p-value | <0.001 | <0.001 | 0.13 | 0.007 | <0.001 | <0.001 |  |
| GSTM1 genotype | null (n = 20) | 11.2, SE = 1.59, CI=(7.94, 14.5) | 5.64, SE = 1.17, CI=(4.06, 7.85) | 1.87, SE = 1.10, CI=(1.54, 2.28) | 2.68, SE = 0.41, CI=(1.84, 3.52) | 0.46 (0.35–1.64) | 0.18 (0.13–0.30) |
| plus (n = 14) | 11.4, SE = 1.69, CI=(7.91, 14.8) | 5.75, SE = 1.192, CI=(4.06, 8.17) | 1.78, SE = 1.12, CI=(1.40, 2.24) | 2.60, SE = 0.44, CI=(1.70, 3.49) | 0.42 (0.34–1.16) | 0.15 (0.11–0.26) |  |
| p-value | 0.93 | 0.93 | 0.72 | 0.87 | 0.77 | 0.74 |  |
| CYP2E1 SNP ID | Presence |   |   |   |   |   |   |
| rs6413432 | No (n = 30) | 10.4, SE = 1.27, CI= (7.85, 13) | 6.05, SE = 1.14, CI=(4.62, 7.92) | 1.77, SE = 1.11, CI=(1.43, 2.17) | 2.43, SE = 0.33, CI= (1.76, 3.11) | 0.468 (0.353–1.658) | 0.207 (0.132–0.281) |
| Yes (n = 4) | 16.4, SE = 2.73, CI= (10.78, 22) | 3.97, SE = 1.33, CI=(2.21, 7.10) | 1.32, SE = 1.24, CI=(0.85, 2.06) | 3.91, SE = 0.712, CI= (2.46, 5.37) | 0.275 (0.212–0.335) | 0.084 (0.078–0.091) |  |
| p-value | 0.047 | 0.166 | 0.214 | 0.057 | 0.011 | 0.003 |  |
| rs2515641 | No (n = 25) | 10.9, SE = 1.52, CI= (7.76, 14.0) | 5.75, SE = 1.17, CI=(4.18, 7.85) | 1.68, SE = 1.12, CI=(1.32, 2.13) | 2.52, SE = 0.393, CI= (1.71, 3.32) | 0.457 (0.338–1.665) | 0.204 (0.12–0.282) |
| Yes (n = 9) | 12.0, SE = 1.98, CI= (7.98, 16.1) | 5.64, SE = 1.22, CI=(3.74, 8.5) | 1.71, SE = 1.16, CI=(1.26, 2.34) | 2.87, SE = 0.514, CI= (1.82, 3.92) | 0.428 (0.348–0.487) | 0.159 (0.114–0.211) |  |
| p-value | 0.619 | 0.928 | 0.91 | 0.558 | 0.673 | 0.748 |  |
| rs41299398 | No (n = 25) | 10.8, SE = 1.42, CI = 7.92, 13.7) | 5.47, SE = 1.15, CI=(4.06, 7.32) | 1.56, SE = 1.11, CI=(1.26, 1.93) | 2.53, SE = 0.368, CI= (1.78, 3.28) | 0.428 (0.334–1.636) | 0.148 (0.105–0.279) |
| Yes (n = 9) | 12.5, SE = 2.06, CI= (8.25, 16.7) | 6.36, SE = 1.23, CI=(4.18, 9.78) | 2.10, SE = 1.16, CI=(1.54, 2.85) | 2.93, SE = 0.534, CI= (1.84, 4.02) | 0.479 (0.424–0.604) | 0.217 (0.165–0.263) |  |
| p-value | 0.472 | 0.502 | 0.08 | 0.501 | 0.41 | 0.275 |  |
| rs915908 | No (n = 22) | 10.1, SE = 1.39, CI= (7.26, 13.0) | 5.7, SE = 1.16, CI=(4.22, 7.69) | 1.60, SE = 1.12, CI=(1.28, 2.01) | 2.33, SE = 0.360, CI= (1.6, 3.07) | 0.468 (0.348–1.611) | 0.182 (0.122–0.268) |
| Yes (n = 12) | 13.5, SE = 1.78, CI= (9.91, 17.2) | 5.7, SE = 1.21, CI=(3.9, 8.41) | 1.88, SE = 1.15, CI=(1.41, 2.5) | 3.24, SE = 0.459, CI= (2.3, 4.18) | 0.423 (0.307–0.888) | 0.152 (0.104–0.292) |  |
| p-value | 0.09 | 0.991 | 0.321 | 0.085 | 0.245 | 0.701 |  |
| rs8192766 | No (n = 30) | 10.9, SE = 1.33, CI= (8.20, 13.6) | 5.87, SE = 1.14, CI=(4.44, 7.69) | 1.69, SE = 1.11, CI=(1.37, 2.09) | 2.51, SE = 0.339, CI= (1.81, 3.21) | 0.455 (0.341–1.658) | 0.162 (0.115–0.281) |
| Yes (n = 4) | 13.8, SE = 3.01, CI= (7.64, 20.0) | 4.76, SE = 1.36, CI=(2.55, 8.94) | 1.70, SE = 1.26, CI=(1.06, 2.74) | 3.54, SE = 0.770, CI= (1.97, 5.12) | 0.352 (0.211–0.67) | 0.178 (0.129–0.216) |  |
| p-value | 0.365 | 0.522 | 0.978 | 0.206 | 0.278 | 0.396 |  |
| rs41299422 | No (n = 30) | 11.0, SE = 1.27, CI= (8.42, 13.6) | 5.81, SE = 1.14, CI=(4.45, 7.54) | 1.70, SE = 1.10, CI=(1.39, 2.08) | 2.58, SE = 0.332, CI= (1.91, 3.26) | 0.468 (0.338–1.658) | 0.207 (0.116–0.281) |
| Yes (n = 4) | 14.7, SE = 3.09, CI= (8.38, 21.0) | 4.53, SE = 1.37, CI=(2.37, 8.58) | 1.59, SE = 1.27, CI=(0.97, 2.6) | 3.41, SE = 0.805, CI= (1.76, 5.05) | 0.385 (0.33–0.43) | 0.139 (0.115–0.161) |  |
| p-value | 0.236 | 0.423 | 0.782 | 0.307 | 0.222 | 0.211 |  |
| rs7081484 | No (n = 29) | 11.7, SE = 1.42, CI= (8.76, 14.5) | 5.75, SE = 1.15, CI=(4.26, 7.69) | 1.69, SE = 1.12, CI=(1.35, 2.10) | 2.69, SE = 0.369, CI= (1.94, 3.45) | 0.455 (0.339–1.636) | 0.159 (0.105–0.279) |
| Yes (n = 5) | 9.9, SE = 2.60, CI= (4.58, 15.2) | 5.58, SE = 1.30, CI=(3.25, 9.58) | 1.71, SE = 1.22 CI=(1.14, 2.58) | 2.46, SE = 0.677, CI= (1.07, 3.84) | 0.487 (0.319–1.219) | 0.211 (0.145–0.232) |  |
| p-value | 0.546 | 0.93 | 0.94 | 0.754 | 0.993 | 0.565 |  |

Table Caption: Pharmacokinetic parameters for isoniazid stratified by NAT2 phenotype, GSTM1 genotype, and CYP2E1 SNP (rs) variants.

#### 3.3.2 Selection of tagging CYP2E1 SNPs

Sequencing data quality for the study samples set (*n* = 34) after the applied quality filters are summarized in [Supplementary Table S2](#s12). All detected SNPs were grouped according to the region of location and functional classification. The majority of detected SNPs were intronic (*n* = 60); three were exonic (missense), three variants located in a 3′ untranslated region (3 UTR), five deletion-insertion variants, eight intergenic, one multiple nucleotide variation, and five - upstream SNPs ([Supplementary Table S3](#s12)). All but nine SNPs conformed to HWE (*p* > 0.05). In the sample set, four allelic variants, i.e., *CY2E1*1B* (rs2070676, G>C), *CYP2E1*4* (rs6413419 G>A), *CYP2E1*6* (rs6413432 T>A), and *CYP2E1*7A* (rs2070673 A>T) were identified. Of all identified SNPs, the variants rs2515641 (T>C), *CY2E1*1B* (G>C) (rs2070676), and *CYP2E1*6* (T>A) (rs6413432) were of clinical significance level 3 ([Khrunin et al., 2010](#B26); [Iacobucci et al., 2013](#B20); [Richardson et al., 2018](#B39); [Wu et al., 2019](#B62); [Yu et al., 2019](#B65)).

When selecting tagging SNPs, priority was given to variants found to be associated with impaired drug metabolism, i.e., with potential effects on CYP2E1 enzyme activity and/or potentially leading to adverse drug reactions, including DIH. Based on the MAF and HWE results, as well as previously reported possible effects on CYP2E1 enzyme activity, 7 SNPs were selected for further analysis: rs8192766, rs6413432, rs2515641, rs41299422, rs7081484, rs41299398, and rs915908.

### 3.4 Association of non-genetic and pharmacokinetic factors with TB treatment outcome and tSCC

No significant associations of age, BMI group, self-reported alcohol intake, or INH dose (mg/kg) with treatment outcome or tSCC were found ([Table 2](#T2)). There was a significant association between smoking status and treatment outcome, where smokers in comparison to non-smokers had 13.8 times higher odds of treatment outcome “cured” in comparison to the “treatment completed” and “lost to follow up” outcomes (*p* = 0.039); however, we did not observe significant associations of smoking status and the overall treatment success ([Table 2](#T2)). Similarly, there was a significant association between sputum microscopy results and treatment outcome: the treatment outcome “cured” could be assigned to 100% of microscopy-positive patients in comparison to the microscopy-negative patients (outcome “cured”: 60.0%, “treatment completed”: 26.7%, “lost to follow up”: 13.3%) (*p* = 0.02). However, again, sputum microscopy result was not associated with the overall treatment success in our cohort. Further analysis indicated that smoking was significantly associated with positive sputum microscopy (*p* = 0.025) ([Supplementary Table S4](#s12)).

Within the INH pharmacokinetic parameters, median INA/INH ratio was significantly higher in patients with tSCC < 60 days in comparison to those with tSCC > 60 days (*p* = 0.026) ([Table 2](#T2)).

Global log-rank *p*-value of the Cox regression model incorporating age, sex, smear microscopy, and standardized values of AcINH/INH ratio was *p* = 0.227 and concordance index was 0.6; the estimated hazard rate (HR) ratios are illustrated ([Figure 1A](#F1)). Based on the preceding information, the following observations were found for the whole cohort: the odds of an individual with a given standardized MR (AcINH/INH) value to reach SCC first compared to an individual with one unit lower standardized MR value were 1.4 (95% CI, 1.00–2.1), *p* = 0.052; a one unit increase of the standardized AcINH/INH ratio value gave HR/(1 + HR) = 58% probability of healing first. There was no significant effect of other predictors on HR of treatment success.

### FIGURE 1.

![FIGURE 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ce9a/10995391/6149f0cc6d1b/fphar-15-1332752-g001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=10995391_fphar-15-1332752-g001.jpg)

Forest plot demonstrating the hazard rate ratios of successful time to sputum culture conversion obtained from fitted Cox proportional hazards model. Statistically significant p-value is marked with asterisk (*). (A). Interactions between age, sex and sputum microscopy results were used along with the AcINH/INH metabolic ratio. (B) Interactions between age, sex and sputum microscopy results were used along with the INA/INH metabolic ratio (C) Interactions between NAT2 acetylator status, GSTM1 genotype and the presence of CYP2E1*6 (T>A) allelic variant (rs6413432) were used along with the sputum microscopy results. NAT2, N-acetyltransferase 2; GSTM1, glutathione S-transferase mu 1 class; MR, metabolic ratio; INH, isoniazid; AcINH, N-acetylisoniazid; INA, isonicotinic acid.

Global log-rank *p*-value of the Cox regression model incorporating age, sex, smear microscopy, and standardized values of INA/INH ratio was *p* = 0.145 and concordance index was 0.65; the estimated hazard rate (HR) ratios are illustrated ([Figure 1B](#F1)). The odds of an individual with a given standardized MR (INA/INH) value to reach SCC first compared to an individual with one unit lower standardized MR value were 1.56 (95% CI, 1.07–2.3), *p* = 0.02; a one unit increase of the standardized INA/INH ratio value gave HR/(1+HR) = 61% probability of healing first.

### 3.5 Association of non-genetic and pharmacokinetic factors with DIH

No significant associations of age, BMI group, self-reported alcohol intake, or INH dose (mg/kg) with hepatotoxicity were found ([Table 2](#T2)). Biological sex was slightly associated with DIH, with males having 85% lower odds of hepatotoxicity than women, however, statistical significance was not reached (*p* = 0.071). Within the INH pharmacokinetic parameters, AcINH AUC0-6 h was significantly lower in patients with DIH (*p* = 0.019).

### 3.6 Association of factors with INH pharmacokinetic parameters

Further, we analysed the possible impact of various non-genetic and genetic factors on the variability in INH pharmacokinetics.

None of the analysed pharmacokinetic parameters differed significantly between patients’ groups when stratified by biological sex, smoking status, BMI, and self-reported daily alcohol intake ([Table 2](#T2)). Also, neither age, nor INH dose (mg/kg) correlated with any of INH pharmacokinetics parameters ([Supplementary Figures S2, S3](#s12)).

In the whole cohort, no correlations were observed between INH (AUC_0–6h_) and either AcINH (AUC_0–6h_) or INA (AUC_0–6h_) ([Figures 2A, E](#F2)); also, no correlation was observed between INH C_max_ and AcINH (AUC_0–6h_) ([Figure 2B](#F2)). However, a significant correlation was observed between INH C_max_ and INA (AUC_0–6h_) (*p* = 0.0099) ([Figure 2F](#F2)).

### FIGURE 2.

![FIGURE 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ce9a/10995391/4bc7a1b34b2c/fphar-15-1332752-g002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=10995391_fphar-15-1332752-g002.jpg)

XY plots showing correlation analysis results for isoniazid and two metabolites in blood plasma of patients with tuberculosis. (A) Acetylisoniazid (AUC0–6h µg·h/mL) versus isoniazid (AUC0–6h µg·h/mL) in the whole study cohort. (B) Acetylisoniazid (AUC0–6h µg·h/mL) versus isoniazid Cmax (2 h, µg/mL) in the whole study cohort. (C) Acetylisoniazid (AUC0–6h µg·h/mL) versus isoniazid (AUC0–6h µg·h/mL), patient samples were stratified based on NAT2 SA and IA acetylator status. (D) Acetylisoniazid (AUC0–6h µg·h/mL) versus isoniazid Cmax (2 h, µg/mL), patient samples were stratified based on NAT2 SA and IA acetylator status. (E) Isonicotinic acid (AUC0–6h µg·h/mL) versus isoniazid (AUC0–6h µg·h/mL) in the whole study cohort. (F) Isonicotinic acid (AUC0–6h µg·h/mL) versus isoniazid Cmax (2 h, µg/mL) in the whole study cohort. (G) Isonicotinic acid (AUC0–6h µg·h/mL) versus isoniazid (AUC0–6h µg·h/mL), patient samples were stratified based on NAT2 SA and IA acetylator status. (H) Isonicotinic acid (AUC0–6h µg·h/mL) versus isoniazid Cmax (2 h, µg/mL), patient samples were stratified based on NAT2 SA and IA acetylator status. AcINH and INA AUC0–6h values were log-trans formed before running the tests. Shaded areas represent the 95% confidence interval of the observed concentration mean values. The solid lines represent the Pearson correlation coefficient. Dots represent the observed data for a single patient. INH, isoniazid; AcINH, acetylisoniazid; INA, izonicotinic acid; AUC, area under the concentration-time curve; Cmax, maximum concentration; NAT2, N-acetyltransferase 2; SA, slow acetylator; IA, intermediate acetylator.

Correlations between the pharmacokinetic parameters became significantly stronger when the patients were stratified based on the *NAT2* acetylator status; the majority of the observed correlation coefficients were statistically significant, and clear separation between *NAT2* groups was observed ([Figures 2C, D, G, H](#F2)). On the other hand, stratification of the patients based on the *GSTM1* genotype did not improve the correlation values for the pharmacokinetic parameters, and clear group separation was not observed ([Supplementary Figure S4](#s12)).

In the whole cohort, significant correlation was observed between both INH metabolites INA (AUC_0–6h_) and AcINH (AUC_0–6h_), as well as between AcINH/INH and INA/INH ratios ([Figures 3A, D](#F3)). Again, group separation was observed for *NAT2* SA and IA acetylators, but not for the *GSTM1* genotypes ([Figures 3B, C, E, F](#F3)).

### FIGURE 3.

![FIGURE 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ce9a/10995391/07629b4a1834/fphar-15-1332752-g003.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=10995391_fphar-15-1332752-g003.jpg)

XY plots showing correlation analysis results for isoniazid metabolites and metabolic ratios in blood plasma of patients with tuberculosis. (A) Acetylisoniazid (AUC0–6h µg·h/mL) versus isonicotinic acid (AUC0–6h µg·h/mL) in the whole study cohort. (B) Acetylisoniazid (AUC0–6h µg·h/mL) versus isonicotinic acid (AUC0–6h µg·h/mL), patient samples were stratified based on NAT2 SA and IA acetylator status. (C) Acetylisoniazid (AUC0–6h µg·h/mL) versus isonicotinic acid (AUC0–6h µg·h/mL), patient samples were stratified based on GSTM1 genotypes. (D) Correlation between INA/INH and AcINH/INH ratios in the whole study cohort. (E) Correlation between INA/INH and AcINH/INH ratios, patient samples were stratified based on NAT2 SA and IA acetylator status. (F) Correlation between INA/INH and AcINH/INH ratios, patient samples were stratified based on GSTM1 genotypes. The solid lines represent the Pearson correlation coefficient. AcINH and INA (AUC0–6h µg·h/mL) values were log-trans formed before running the tests. Spearman correlation was used for non-normally distributed variables (i.e., AcINH/INH and INA/INH ratios). Shaded areas represent the 95% confidence interval of the observed concentration mean values. Dots represent the observed data for a single patient. INH, isoniazid; AcINH, acetylisoniazid; INA, izonicotinic acid; AUC, area under the concentration-time curve; NAT2, N-acetyltransferase 2; SA, slow acetylator; IA, intermediate acetylator; GSTM1, glutathione S-transferase mu 1 class.

After adjustment by sex, age and INH dose, the mean and median values of the pharmacokinetics parameters differed significantly between *NAT2* SA and IA groups, except for INA AUC_0–6h_ ([Table 4](#T4)). More specifically, INH AUC_0–6h_ and INH C_max_ values were significantly higher, while AcINH AUC_0–6h_ values were significantly lower in *NAT2* slow acetylators. Also, median values of both AcINH/INH and INA/INH ratios were significantly lower in the *NAT2* SA group. None of pharmacokinetic parameters differed significantly between *GSTM1*-plus and -null genotypes.

Similarly, pharmacokinetic parameters were compared between patient groups divided according to the presence/absence of the each *CYP2E1* SNPs. Overall, only rs6413432 claimed a statistically significant difference for three INH pharmacokinetic parameters: INH AUC_0–6h_, values were significantly higher, while median values of both AcINH/INH MR and INA/INH MR were significantly lower in patients with rs6413432 ([Table 4](#T4)).

### 3.7 Association of genetic factors with treatment outcome, tSCC and DIH

As some INH pharmacokinetic parameters were associated with tSCC and DIH, and the impact of genetic factors on INH pharmacokinetics was observed, we further investigated possible association of genetic factors with treatment outcome, tSCC and DIH.

Based on the univariate analysis, there were no significant associations between treatment outcome and any of genetic factors studied ([Table 5](#T5)). DIH occurred only in the patients with NAT2 SA phenotype (*N* = 4); however, statistically significant difference was not observed between NAT2 SA and IA groups ([Table 5](#T5)). A trend of association between NAT2 phenotypes and tSCC was detected, with SA having 80% lower odds of tSCC within 60 days in comparison to IA (OR = 0.2; 95% CI, 0.03–1.22); however, statistical significance was not reached (*p* = 0.069) ([Table 5](#T5)).

### TABLE 5.

| Total |  |  | Treatment outcome* |  | tSCC (n = 28**) |  | Hepatotoxicity |   |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Cured | Treatment completed | <60 days | >60 days | Yes | No |  |  |  |
| NAT2 phenotype | SA | 23/34 (67.7) | 18/23 (78.3) | 3/23 (13.0) | 8/18 (44.4) | 10/18 (55.6) | 4/23 (17.4) | 19/23 (82.6) |
| IA | 11/34 (32.4) | 10/11 (90.9) | 1/11 (9.1) | 8/10 (80.0) | 2/10 (20.0) | 0/11 (0.0) | 11/11 (100.0) |  |
| p-value | 0.059 | 1.0 |   | 0.069 |   | 0.14 |   |  |
| OR1 (95% CI) | na | 0.6 (0.05, 6.6) |   | 0.2 (0.03, 1.22) |   | na |   |  |
| GSTM1 genotype | null | 20/34 (58.8) | 17/20 (85.0) | 2/20 (10.0) | 9/17 (52.9) | 8/17 (47.1) | 2/20 (10.0) | 18/20 (90.0) |
| plus | 14/34 (41.2) | 11/14 (78.6) | 2/14 (14.3) | 7/11 (63.6) | 4/11 (36.4) | 2/20 (14.3) | 12/14 (85.7) |  |
| p-value | 0.39 | 0.68 |   | 0.58 |   | 0.7 |   |  |
| OR2 (95% CI) | na | 1.6 (0.19, 12.6) |   | 0.64 (0.14, 3.0) |   | 0.67 (0.08, 5.4) |   |  |
| CYP2E1 SNP ID | Presence |   |   |   |   |   |   |   |
| rs6413432 | No | 30/34 (88.2) | 26/30 (86.7) | 2/30 (6.7) | 16/26 (61.5) | 10/26 (38.5) | 3/30 (10.0) | 27/30 (90.0) |
| Yes | 4/34 (11.8) | 2/4 (50.0) | 2/4 (50.0) | 0/0 (0.0) | 2/2 (100.0) | 1/4 (25.0) | 3/4 (75.0) |  |
| p-value | <0.001 | 0.106 |   | 0.34 |   | 0.961 |   |  |
| OR (95% CI) | na | 0.08 (0.01, 0.87) |   | na |   | 3 (0.23, 38.74) |   |  |
| rs2515641 | No | 25/34 (73.5) | 20/25 (80.0) | 3/25 (12.0) | 12/20 (60.0) | 8/20 (40.0) | 3/25 (12.0) | 22/25 (88.0) |
| Yes | 9/34 (26.5) | 8/9 (88.9) | 1/9 (11.1) | 4/8 (50.0) | 4/8 (50.0) | 1/9 (11.1) | 8/9 (88.9) |  |
| p-value | <0.001 | 1.0 |   | 0.952 |   | 1.0 |   |  |
| OR (95% CI) | na | 1.2 (0.11, 13.32) |   | 0.67 (0.13, 3.47) |   | 0.92 (0.08, 10.14) |   |  |
| rs41299398 | No | 25/34 (73.5) | 20/25 (80.0) | 4/25 (16.0) | 10/20 (50.0) | 10/20 (50.0) | 3/25 (12.0) | 22/25 (88.0) |
| Yes | 9/34 (26.5) | 8/9 (88.8) | 0/9 (0.0) | 6/8 (75.0) | 2/8 (25.0) | 1/9 (11.1) | 8/9 (88.9) |  |
| p-value | <0.001 | 0.537 |   | 0.432 |   | 1.0 |   |  |
| OR (95% CI) | na |   |   | 3 (0.48, 18.6) |   | 0.92 (0.08, 10.14) |   |  |
| rs915908 | No | 22/34 (64.7) | 17/22 (77.3) | 3/22 (13.6) | 11/17 (64.7) | 6/17 (35.3) | 2/22 (9.1) | 20/22 (90.9) |
| Yes | 12/34 (35.3) | 11/12 (91.7) | 1/12 (8.3) | 5/11 (45.5) | 6/11 (54.6) | 2/12 (16.7) | 10/12 (83.3) |  |
| p-value | 0.12 | 1.0 |   | 0.539 |   | 0.922 |   |  |
| OR (95% CI) | na | 1.94 (0.18, 21.12) |   | 0.45 (0.1, 2.14) |   | 2 (0.24, 16.36) |   |  |
| rs8192766 | No | 30/34 (88.2) | 25/30 (83.3) | 3/30 (10.0) | 14/25 (56.0) | 11/25 (44.0) | 4/30 (13.3) | 26/30 (86.7) |
| Yes | 4/34 (11.8) | 3/4 (75.0) | 1/4 (25.0) | 2/3 (66.7) | 1/3 (33.3) | 0/4 (0.0) | 4/4 (100.0) |  |
| p-value | <0.001 | 1.0 |   | 1.0 |   | 1.0 |   |  |
| OR (95% CI) | na | 0.36 (0.03, 4.66) |   | 1.57 (0.13, 19.67) |   | na |   |  |
| rs41299422 | No | 30/34 (88.2) | 24/30 (80.0) | 4/30 (13.3) | 15/24 (62.5) | 9/24 (37.5) | 4/30 (13.3) | 26/30 (86.7) |
| Yes | 4/34 (11.8) | 4/4 (100.0) | 0/4 (0.0) | 1/4 (25.0) | 3/4 (75.0) | 0/4 (0.0) | 4/4 (100.0) |  |
| p-value | <0.001 | 1.0 |   | 0.391 |   | 1.0 |   |  |
| OR (95% CI) | na |   |   | 0.2 (0.02, 2.23) |   | na |   |  |
| rs7081484 | No | 29/34 (85.3) | 24/29 (82.8) | 3/29 (10.3) | 13/24 (54.2) | 11/24 (45.8) | 4/29 (13.8) | 25/29 (86.2) |
| Yes | 5/34 (14.7) | 4/5 (80.0) | 1/5 (20.0) | 3/4 (75.0) | 1/4 (25.0) | 0/5 (0.0) | 5/5 (100.0) |  |
| p-value | <0.001 | 1.0 |   | 0.815 |   | 0.894 |   |  |
| OR (95% CI) | na | 0.5 (0.04, 6.08) |   | 2.54 (0.23, 28.02) |   | na |   |  |

Table 5 Caption: Treatment outcome and hepatotoxicity in patients with TB (n = 34) stratified by NAT2 phenotype, GSTM1 genotype, and CYP2E1 SNP (rs) variants.

No significant association between *GSTM1* groups and tSCC was detected, and no significant associations were found for any of 7 analysed SNPs of *CYP2E1* gene with either treatment outcome, tSCC, or hepatotoxicity ([Table 5](#T5)).

Cumulative probabilities of the sputum culture conversion event were estimated as the complements of Kaplan-Meier curves ([Figure 4](#F4)). According to the log-rank test, there was no significant difference in tSCC between the genotypes of either NAT2 (*p* = 0.16), or *GSTM1* (*p* = 0.33). Based on the *GSTM1* stratification, the median treatment times were 64 (95% CI, 42–85) and 66 (95% CI, 46–197) days for null vs. plus genotype, respectively. The mean tSCC, restricted at time 60 days, were 46.7 (95% CI, 38.9–54.5) vs. 50 (95% CI, 43.6–56.5) days for null vs. plus genotype, respectively; the difference between plus and null groups was 3.4 days (95% CI, 13.5–6.7), *p* = 0.51. Based on the NAT2 phenotype stratification, the median treatment times were 46 (95% CI, 27–197) and 71.5 (95% CI, 57–112) days for NAT2 IA and SA individuals, respectively. The mean tSCC, restricted at time 60 days, were 39.6 (95% CI, 29.3–49.9) vs. 52.4 (95% CI, 47.2–57.5) days for NAT2 IA and SA individuals, respectively; the difference between NAT2 IA and SA groups was 12.7 days (95% CI, 1.2–24.3), *p* = 0.03 ([Figure 4](#F4)).

### FIGURE 4.

![FIGURE 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ce9a/10995391/36f3cb4b5688/fphar-15-1332752-g004.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=10995391_fphar-15-1332752-g004.jpg)

Kaplan-Meier curves of cumulative probability of the sputum culture conversion time along with the table of cumulative number of events. (A) Data were stratified based on GSTM1 genotype. (B) Data were stratified based on NAT2 SA and IA acetylator status. The log-rank test was used for comparison of the curves, the corresponding p-values are displayed. Median treatment times in subgroups are indicated by dashed lines. NAT2, N-acetyltransferase 2; SA, slow acetylator; IA, intermediate acetylator; GSTM1, glutathione S-transferase mu 1 class.

However, in the Cox regression model for the whole study cohort (*n* = 34) incorporating *GSTM1* genotype, NAT2 phenotype, CYP2E1*6 (T > A) allelic variant (rs6413432), and smear microscopy, none of the predictors contributed significantly on HR for the respond to treatment (i.e., tSCC); global log-rank *p*-value of the model was *p* = 0.081 and concordance index was 0.64 ([Figure 1C](#F1)).

## 4 Discussion

This is the prospective study aimed to decipher the impact of pharmacokinetic, pharmacogenetic and patient-related factors on the response to TB treatment, TB treatment outcome and the risk of DIH, and to investigate possible links between these variables. For this task, along with the array of patient- and disease-related factors, we evaluated six pharmacokinetic parameters of the parent drug INH and its two metabolites, AcINH and INA, for more accurate assessments of the data obtained ([Akkerman et al., 2014](#B2); [Sturkenboom et al., 2015](#B53)). In our study, a high TB treatment success rate was achieved (94.1%). However, we did not observe any associations between patient-related factors with either DIH or treatment outcome. Molla and colleagues have reported that males were significantly less likely to develop hepatotoxicity than females ([Molla et al., 2021](#B34)); in our study similar trend was observed, however, statistical significance was not reached (*p* = 0.071). Smoking status was considered to be a baseline risk factor associated with negative impact on the TB treatment success, delaying conversion of the sputum culture, and extending the time of TB treatment ([Silva et al., 2018](#B47); [World Health Organization, 2022](#B60)). Here, smoking was highly associated with positive sputum microscopy results confirming the negative smoking impact. However, both smokers and patients with positive sputum microscopy results were more likely to achieve treatment outcome “cured” in comparison to the “treatment completed” or “lost to follow up” outcomes (*p* < 0.05). This result could be related to the fact that patients with positive sputum microscopy result were treated in the hospital until the negative sputum microscopy was confirmed, rather than received ambulatory treatment for TB, thus achieving better treatment compliance. These findings could have some implications for TB eradication programmes. NAT2 enzyme plays a key role in biotransformation of xenobiotics including first-line anti-TB drug INH ([Mthiyane et al., 2020](#B35)). The *NAT2* gene is highly polymorphic, and more than 65 variants containing one or more SNPs in the 870 bp long coding region have previously been reported ([Hein and Doll, 2012](#B18)); in turn, the state of NAT2 activity in an individual depends on combinations of rapid and slow alleles ([Hein and Doll, 2012](#B18); [Mthiyane et al., 2020](#B35)). Worldwide, significant differences in the distribution and frequency of NAT2 alleles between ethnic populations exist; the overall proportions of the acetylator phenotypes in our study cohort was 68% for SA and 32% for IA carriers, similar to those reported in Caucasian populations in Europe and the United States ([Sabbagh et al., 2011](#B42); [Bisso-Machado et al., 2016](#B7)).

Data from our study confirmed previous reports reviewed in ([Hong et al., 2020](#B19)) showing that INH and its metabolites AcINH and INA blood plasma levels were significantly different between the NAT2 slow and intermediate acetylators: INH AUC_0–6h_ and INH C_max_ were significantly higher, and AUC_0–6h_ values for both metabolites were significantly lower in the SA group in comparison to the IA group; consequently, the median AcINH/INH and INA/INH ratio values were also significantly lower in the SA group. Also, significant NAT2 inter-phenotypic differences in INH and metabolites correlation patterns were detected.

According to the proposed INH metabolism pathway, INH, Hz and AcHz are metabolized by NAT2 enzyme and thus are subjects to the same acetylation polymorphism ([Balhara et al., 2021](#B5)). Three metabolites have been proposed to be responsible for INH-induced liver injury, AcHz, Hz and a metabolite resulting from the bioactivation of INH itself ([Metushi et al., 2011](#B33)). In our study, AcINH AUC_0–6h_ values were significantly lower in patients with DIH suggesting impaired INH acetylation in these cases; in addition, slow acetylation of toxicologically active AcHz to inactive diAcHz could be expected.The frequency of DIH observed in our cohort (11.76%), as well as the association of a slow acetylation profile with DIH, are consistent with the data reported previously ([Sistanizad et al., 2011](#B49); [Perwitasari et al., 2018](#B38); [Khan et al., 2019](#B25)). In our small cohort, statistical significance was reached for this INH no other INH pharmacogenetic parameters reached statistical significance, thus these results indicated that AcINH plasma level could serve as a biomarker for DIH.GSTs are important phase II detoxification isozymes that catalyse the conjugation of reduced form of glutathione to toxic metabolites including in the INH metabolic pathway, thereby reducing their reactivity toward cellular macromolecules ([Sodhi et al., 1996](#B50); [Eaton and Bammler, 1999](#B13)). GST genetic polymorphisms, especially the variants of *GSTM1* gene, have been extensively studied, and the absence of *GSTM1* activity due to null mutation was associated with accumulation of toxic intermediates leading to DIH in some studies ([Wang et al., 2016](#B57)). The proportion of the *GSTM1*-null genotype in our Caucasian cohort was 59%; however, we did not observe any effects of *GSTM1* genotype on DIH, TB treatment outcome, or any of the six INH pharmacokinetic parameters studied.

CYP2E1 is another enzyme involved in the Phase I biotransformation reactions of INH ([Cheng et al., 2013](#B10)). Previous studies on the pharmacogenetic variability of *CYP2E1* have resulted in evidence-based clinical annotations of specific genetic polymorphisms and corresponding allelic variants with consequent different phenotypes in regulatory pathways that effect the pharmacokinetics and treatment response ([https://cpicpgx.org/guidelines](https://cpicpgx.org/guidelines)), but controversial associations of *CYP2E1* genotype with risk of INH-related hepatotoxicity were reported ([Lei et al., 2021](#B29)). In our study, statistically significant associations between *CYP2E1**6 (T > A) allelic variant (rs6413432) and three INH pharmacokinetic parameters, i.e., increase in INH AUC_0–6h_, and decrease in both AcINH/INH and INA/INH ratios, were detected; however, no direct proofs for contribution in either DIH or treatment outcome were observed. None of other detected SNPs, including two allelic variants *CYP2E1**4 (rs6413419) and *CYP2E1**7A (rs2070673) which were previously associated with potential effects on CYP2E1 enzymatic activity, showed any statistically significant associations.

*CYP2E1**6 allelic variant, recognized by *Dra* I restriction enzyme, is located in intron 6 of the gene ([Fang et al., 2017](#B16)). Previously, it was suggested that this polymorphism is associated with the *CYP2E1* gene expression at mRNA level ([Uematsu et al., 1994](#B55)). Our data indicate that *CYP2E1**6 was related to the decreased amounts of AcINH and INA; however, the mechanism of interaction between *CYP2E1* genetic polymorphisms and INH plasma levels is unclear. Clinical studies have reported the interactions of INH with CY2E1 substrates including observations of interphenotypic difference for INH-related inhibition of *CYP2E1* activity, which was more extensive in slow acetylators ([O’Shea et al., 1997](#B36)). However, *in vitro* studies did not observe any significant inhibition of CYP2E1 by INH or its metabolites ([Balhara et al., 2021](#B5)). Also, treatment of drug-susceptible TB involves several drugs, including RIF, which is a CYP2E1 inducer, thus drug-drug interactions could potentially interfere with INH metabolism or manifestation of INH-related adverse drug reactions through increased production of reactive metabolites, especially in slow acetylators ([Shen et al., 2008](#B46); [Yamada et al., 2009](#B63)). Additional studies are required to determine the possible functional implications of the *CYP2E1* SNVs by assessing the drug response and clinical manifestations of the identified variants.

Another objective of our study was to determine the possible relationship between INH pharmacokinetic data and TB treatment outcome. First, it should be noted that, in our study, INH C_max_ at 2 h time point was below the therapeutic reference range in the majority of the patients (64.7%). Considering the overall good treatment success rates, our findings support the opinion that the current therapeutic ranges are probably set too high ([Perumal et al., 2020](#B37)). Further, our results showed that INA/INH ratios were significantly higher in patients with SCC time less than 60 days. This assertion was further supported by the findings showing that a one unit increase of the standardized AcINH/INH ratio value gave 58% probability to achieve SCC first. Thus, it could be concluded, that for a patient to have a probability of healing first, the metabolic rate of the standard dose INH should be sufficiently high, while non-efficient INH metabolic rate is not favourable. Also, quite surprisingly, our results from the time-to-event analysis showed that intermediate acetylators recovered more rapidly than slow acetylators; however, the results of univariate analysis did not reach statistically significant difference between NAT2 phenotypes and tSCC. Somewhere controversially, several studies have shown that low INH C_max_ and/or AUC values in rapid acetylators were associated with longer tSCC and a higher likelihood of failure or relapse; however, other studies, and also this study, found no association between the INH concentrations and treatment outcomes ([Sekaggya-Wiltshire, von Braun, et al., 2018](#B45)). INH plasma half-life is approximately 70 min in fast and 2–5 h in slow acetylators ([Goodman et al., 2006](#B17)), thus INH C_max_ measurements at a single 2 h time point may not fully reflect INH exposure for all patients. Thus, in our study, plasma concentrations of AcINH and INA were measured, and INH metabolic ratio values were used in addition to INH C_max_ and AUC values, which could provide more accurate assessments of the data.

On the other hand, inconsistency of the results may be due to the variability of the study population, the involvement of different factors, and the study design. TB treatment outcome should be considered as multifactorial event, considering multiple factors such as bacillary load, *M. tuberculosis* strain, individual measurement or local population estimate of the infecting strain minimum inhibitory concentration (MIC) values, treatment adherence, insufficient drug absorption, drug-drug interactions, drug concentration at the site of TB localization, age, gender, comorbid conditions, and immune and nutritional status of the patient ([Colangeli et al., 2018](#B11); [Perumal et al., 2020](#B37)). Importantly, it was reported recently that INH concentrations were 14.6-fold higher in lung/airway epithelial lining fluid than blood plasma, thus accumulation of drugs at the site of disease may explain the success of the first-line regimen ([McCallum et al., 2021](#B31)).

Our study had limitations. First, the number of cohort participants was relatively small, and we were not able to include individuals with fast NAT2 phenotype. Second, the study was conducted in the Latvian population with European ancestry, and the obtained data were not extrapolated to other ethnicities. Also, we were not able to include all possible patient-, disease-, and treatment-related factors, and there is a possibility that the statistical power of the analysis to identify all genetic polymorphisms/SNPs was not sufficient.

## 5 Conclusion

In conclusion, the results of our study brought closer the research on anti-TB drug pharmacology and its application in clinical practice. Our findings suggest that *CYP2E1*, but not *GSTM1*, significantly affects the INH pharmacokinetics along with *NAT2*. Also, significant associations between INH pharmacokinetic parameters and both incidence of hepatotoxicity and month two sputum culture positivity were observed. AcINH plasma level could serve as a biomarker for INH-related hepatotoxicity. These findings could have further implications in promotion of the concept of precision medicine in TB management. The inclusion of INH metabolite screening in TB therapeutic drug monitoring could be beneficial in clinical studies for determination of optimal dosing strategies.

## Acknowledgments

We would like to express our gratitude to the staff members of Centre of Tuberculosis and Lung Diseases, Riga East University Hospital for their assistance in collecting blood and plasma samples and data used in this study. We also extend our appreciation to all the study participants for generously donating their blood samples and time to support this research.

## Funding Statement

The author(s) declare financial support was received for the research, authorship, and/or publication of this article. This study was supported by the FLP Program/Ministry of Education and Science, Republic of Latvia, project No lzp 2020/1-0050.

## Data availability statement

The original contributions presented in the study are publicly available. This data can be found here: European Nucleotide Archive, accession number PRJEB69677.

## Ethics statement

The studies involving humans were approved by the Central Medical Ethics Committee of Latvia. The studies were conducted in accordance with the local legislation and institutional requirements. The participants provided their written informed consent to participate in this study.

## Author contributions

VU: Writing–original draft, Visualization, Validation, Methodology, Investigation, Data curation, Writing–review and editing, Conceptualization. AK: Writing–review and editing, Validation, Methodology, Investigation, Data curation. AV: Writing–review and editing, Methodology, Investigation, Data curation. LP: Visualization, Software, Formal Analysis, Writing–review and editing, Methodology, Investigation, Data curation. LF: Writing–review and editing, Methodology, Investigation, Data curation. DS: Writing–review and editing, Investigation, Data curation. IO: Conceptualization, Writing–review and editing, Investigation. AC: Supervision, Writing–review and editing. ES: Validation, Methodology, Investigation, Data curation, Writing–review and editing. SG: Resources, Funding acquisition, Conceptualization, Writing–review and editing, Validation, Methodology, Investigation, Data curation. DB: Project administration, Writing–review and editing, Resources, Funding acquisition. RR: Supervision, Conceptualization, Writing–review and editing, Resources, Project administration, Funding acquisition.

## Conflict of interest

The authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.

## Publisher’s note

All claims expressed in this article are solely those of the authors and do not necessarily represent those of their affiliated organizations, or those of the publisher, the editors and the reviewers. Any product that may be evaluated in this article, or claim that may be made by its manufacturer, is not guaranteed or endorsed by the publisher.

## Supplementary material

The Supplementary Material for this article can be found online at: [https://www.frontiersin.org/articles/10.3389/fphar.2024.1332752/full#supplementary-material](https://www.frontiersin.org/articles/10.3389/fphar.2024.1332752/full#supplementary-material)

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

The original contributions presented in the study are publicly available. This data can be found here: European Nucleotide Archive, accession number PRJEB69677.

### Supplementary Materials

### Data Availability Statement

The original contributions presented in the study are publicly available. This data can be found here: European Nucleotide Archive, accession number PRJEB69677.

## References

1. Aarnoutse R. (2011). Chapter 18: pharmacogenetics of antituberculosis drugs. Antituberc. Chemother. 40, 176–190. 10.1159/000330320  [DOI](https://doi.org/10.1159/000330320) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Antituberc.%20Chemother.&title=Chapter%2018:%20pharmacogenetics%20of%20antituberculosis%20drugs&author=R.%20Aarnoutse&volume=40&publication_year=2011&pages=176-190&doi=10.1159/000330320&)

2. Akkerman O. W., van Altena R., Bolhuis M. S., van der Werf T. S., Alffenaar J.-W. (2014). Strategy to limit sampling of antituberculosis drugs instead of determining concentrations at two hours postingestion in relation to treatment response. Antimicrob. Agents Chemother. 58 (1), 628. 10.1128/AAC.01535-13  [DOI](https://doi.org/10.1128/AAC.01535-13) | [PMC free article](/articles/PMC3910774/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24363390/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Antimicrob.%20Agents%20Chemother.&title=Strategy%20to%20limit%20sampling%20of%20antituberculosis%20drugs%20instead%20of%20determining%20concentrations%20at%20two%20hours%20postingestion%20in%20relation%20to%20treatment%20response&author=O.%20W.%20Akkerman&author=R.%20van%20Altena&author=M.%20S.%20Bolhuis&author=T.%20S.%20van%20der%20Werf&author=J.-W.%20Alffenaar&volume=58&issue=1&publication_year=2014&pages=628&pmid=24363390&doi=10.1128/AAC.01535-13&)

3. Alsultan A., Peloquin C. A. (2014). Therapeutic drug monitoring in the treatment of tuberculosis: an update. Drugs 74 (8), 839–854. 10.1007/s40265-014-0222-8  [DOI](https://doi.org/10.1007/s40265-014-0222-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24846578/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drugs&title=Therapeutic%20drug%20monitoring%20in%20the%20treatment%20of%20tuberculosis:%20an%20update&author=A.%20Alsultan&author=C.%20A.%20Peloquin&volume=74&issue=8&publication_year=2014&pages=839-854&pmid=24846578&doi=10.1007/s40265-014-0222-8&)

4. Azuma J., Ohno M., Kubota R., Yokota S., Nagai T., Tsuyuguchi K., et al. (2013). NAT2 genotype guided regimen reduces isoniazid-induced liver injury and early treatment failure in the 6-month four-drug standard treatment of tuberculosis: a randomized controlled trial for pharmacogenetics-based therapy. Eur. J. Clin. Pharmacol. 69 (5), 1091–1101. 10.1007/s00228-012-1429-9  [DOI](https://doi.org/10.1007/s00228-012-1429-9) | [PMC free article](/articles/PMC3641305/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23150149/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Clin.%20Pharmacol.&title=NAT2%20genotype%20guided%20regimen%20reduces%20isoniazid-induced%20liver%20injury%20and%20early%20treatment%20failure%20in%20the%206-month%20four-drug%20standard%20treatment%20of%20tuberculosis:%20a%20randomized%20controlled%20trial%20for%20pharmacogenetics-based%20therapy&author=J.%20Azuma&author=M.%20Ohno&author=R.%20Kubota&author=S.%20Yokota&author=T.%20Nagai&volume=69&issue=5&publication_year=2013&pages=1091-1101&pmid=23150149&doi=10.1007/s00228-012-1429-9&)

5. Balhara A., Kumar A., Kumar S., Samiulla D. S., Giri S., Singh S. (2021). Exploration of inhibition potential of isoniazid and its metabolites towards CYP2E1 in human liver microsomes through LC-MS/MS analysis. J. Pharm. Biomed. Analysis 203, 114223. 10.1016/j.jpba.2021.114223  [DOI](https://doi.org/10.1016/j.jpba.2021.114223) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34214766/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Pharm.%20Biomed.%20Analysis&title=Exploration%20of%20inhibition%20potential%20of%20isoniazid%20and%20its%20metabolites%20towards%20CYP2E1%20in%20human%20liver%20microsomes%20through%20LC-MS/MS%20analysis&author=A.%20Balhara&author=A.%20Kumar&author=S.%20Kumar&author=D.%20S.%20Samiulla&author=S.%20Giri&volume=203&publication_year=2021&pages=114223&pmid=34214766&doi=10.1016/j.jpba.2021.114223&)

6. Becker C., Dressman J. B., Amidon G. L., Junginger H. E., Kopp S., Midha K. K., et al. (2007). Biowaiver monographs for immediate release solid oral dosage forms: isoniazid. J. Pharm. Sci. 96 (3), 522–531. 10.1002/jps.20765  [DOI](https://doi.org/10.1002/jps.20765) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17117431/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Pharm.%20Sci.&title=Biowaiver%20monographs%20for%20immediate%20release%20solid%20oral%20dosage%20forms:%20isoniazid&author=C.%20Becker&author=J.%20B.%20Dressman&author=G.%20L.%20Amidon&author=H.%20E.%20Junginger&author=S.%20Kopp&volume=96&issue=3&publication_year=2007&pages=522-531&pmid=17117431&doi=10.1002/jps.20765&)

7. Bisso-Machado R., Ramallo V., Paixão-Côrtes V. R., Acuña-Alonzo V., Demarchi D. A., Sandoval J. R. S., et al. (2016). NAT2 gene diversity and its evolutionary trajectory in the Americas. Pharmacogenomics J. 16 (6), 559–565. 10.1038/tpj.2015.72  [DOI](https://doi.org/10.1038/tpj.2015.72) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26503810/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J.&title=NAT2%20gene%20diversity%20and%20its%20evolutionary%20trajectory%20in%20the%20Americas&author=R.%20Bisso-Machado&author=V.%20Ramallo&author=V.%20R.%20Paix%C3%A3o-C%C3%B4rtes&author=V.%20Acu%C3%B1a-Alonzo&author=D.%20A.%20Demarchi&volume=16&issue=6&publication_year=2016&pages=559-565&pmid=26503810&doi=10.1038/tpj.2015.72&)

8. Cascorbi I., Brockmöller J., Mrozikiewicz P. M., Bauer S., Loddenkemper R., Roots I. (1996). Homozygous rapid arylamine N-acetyltransferase (NAT2) genotype as a susceptibility factor for lung cancer. Cancer Res. 56 (17), 3961–3966.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/8752164/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res.&title=Homozygous%20rapid%20arylamine%20N-acetyltransferase%20(NAT2)%20genotype%20as%20a%20susceptibility%20factor%20for%20lung%20cancer&author=I.%20Cascorbi&author=J.%20Brockm%C3%B6ller&author=P.%20M.%20Mrozikiewicz&author=S.%20Bauer&author=R.%20Loddenkemper&volume=56&issue=17&publication_year=1996&pages=3961-3966&pmid=8752164&)

9. Chen R., Zheng X., Hu P. (2017). CYP2D6 phenotyping using urine, plasma, and saliva metabolic ratios to assess the impact of CYP2D6∗10 on interindividual variation in a Chinese population. Front. Pharmacol. 8, 239. 10.3389/fphar.2017.00239  [DOI](https://doi.org/10.3389/fphar.2017.00239) | [PMC free article](/articles/PMC5411458/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28512430/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front.%20Pharmacol.&title=CYP2D6%20phenotyping%20using%20urine,%20plasma,%20and%20saliva%20metabolic%20ratios%20to%20assess%20the%20impact%20of%20CYP2D6%E2%88%9710%20on%20interindividual%20variation%20in%20a%20Chinese%20population&author=R.%20Chen&author=X.%20Zheng&author=P.%20Hu&volume=8&publication_year=2017&pages=239&pmid=28512430&doi=10.3389/fphar.2017.00239&)

10. Cheng J., Krausz K. W., Li F., Ma X., Gonzalez F. J. (2013). CYP2E1-dependent elevation of serum cholesterol, triglycerides, and hepatic bile acids by isoniazid. Toxicol. Appl. Pharmacol. 266 (2), 245–253. 10.1016/j.taap.2012.10.024  [DOI](https://doi.org/10.1016/j.taap.2012.10.024) | [PMC free article](/articles/PMC3661416/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23142471/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Toxicol.%20Appl.%20Pharmacol.&title=CYP2E1-dependent%20elevation%20of%20serum%20cholesterol,%20triglycerides,%20and%20hepatic%20bile%20acids%20by%20isoniazid&author=J.%20Cheng&author=K.%20W.%20Krausz&author=F.%20Li&author=X.%20Ma&author=F.%20J.%20Gonzalez&volume=266&issue=2&publication_year=2013&pages=245-253&pmid=23142471&doi=10.1016/j.taap.2012.10.024&)

11. Colangeli R., Jedrey H., Kim S., Connell R., Ma S., Chippada Venkata U. D., et al. (2018). Bacterial factors that predict relapse after tuberculosis therapy. N. Engl. J. Med. 379 (9), 823–833. 10.1056/NEJMoa1715849  [DOI](https://doi.org/10.1056/NEJMoa1715849) | [PMC free article](/articles/PMC6317071/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30157391/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%20Engl.%20J.%20Med.&title=Bacterial%20factors%20that%20predict%20relapse%20after%20tuberculosis%20therapy&author=R.%20Colangeli&author=H.%20Jedrey&author=S.%20Kim&author=R.%20Connell&author=S.%20Ma&volume=379&issue=9&publication_year=2018&pages=823-833&pmid=30157391&doi=10.1056/NEJMoa1715849&)

12. Daly A. K. (2021). “Genetic polymorphism of drug‐metabolizing enzymes and drug transporters in drug toxicity,” in Transporters and drug‐metabolizing enzymes in drug toxicity. Editor Li A. P. (Wiley; ), Hoboken, NJ, USA, 139–165. 10.1002/9781119171003.ch5  [DOI](https://doi.org/10.1002/9781119171003.ch5) | [Google Scholar](https://scholar.google.com/scholar_lookup?title=Transporters%20and%20drug%E2%80%90metabolizing%20enzymes%20in%20drug%20toxicity&author=A.%20K.%20Daly&publication_year=2021&)

13. Eaton D. L., Bammler T. K. (1999). Concise review of the glutathione S-transferases and their significance to toxicology. Toxicol. Sci. An Official J. Soc. Toxicol. 49 (2), 156–164. 10.1093/toxsci/49.2.156  [DOI](https://doi.org/10.1093/toxsci/49.2.156) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10416260/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Toxicol.%20Sci.%20An%20Official%20J.%20Soc.%20Toxicol.&title=Concise%20review%20of%20the%20glutathione%20S-transferases%20and%20their%20significance%20to%20toxicology&author=D.%20L.%20Eaton&author=T.%20K.%20Bammler&volume=49&issue=2&publication_year=1999&pages=156-164&pmid=10416260&doi=10.1093/toxsci/49.2.156&)

14. Ellard G. A., Gammon P. T. (1976). Pharmacokinetics of isoniazid metabolism in man. J. Pharmacokinet. Biopharm. 4, 83–113. 10.1007/BF01086149  [DOI](https://doi.org/10.1007/BF01086149) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/950592/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Pharmacokinet.%20Biopharm.&title=Pharmacokinetics%20of%20isoniazid%20metabolism%20in%20man&author=G.%20A.%20Ellard&author=P.%20T.%20Gammon&volume=4&publication_year=1976&pages=83-113&pmid=950592&doi=10.1007/BF01086149&)

15. Erasmus D. J. (2021). DNA barcoding: a different perspective to introducing undergraduate students to DNA sequence analysis. Biochem. Mol. Biol. Educ. A Bimon. Publ. Int. Union Biochem. Mol. Biol. 49 (3), 416–421. 10.1002/bmb.21492  [DOI](https://doi.org/10.1002/bmb.21492) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33529467/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem.%20Mol.%20Biol.%20Educ.%20A%20Bimon.%20Publ.%20Int.%20Union%20Biochem.%20Mol.%20Biol.&title=DNA%20barcoding:%20a%20different%20perspective%20to%20introducing%20undergraduate%20students%20to%20DNA%20sequence%20analysis&author=D.%20J.%20Erasmus&volume=49&issue=3&publication_year=2021&pages=416-421&pmid=33529467&doi=10.1002/bmb.21492&)

16. Fang Z., Wu Y., Zhang N. (2017). Association between CYP2E1 genetic polymorphisms and urinary cancer risk: a meta-analysis. Oncotarget 8 (49), 86853–86864. 10.18632/oncotarget.20993  [DOI](https://doi.org/10.18632/oncotarget.20993) | [PMC free article](/articles/PMC5689730/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29156840/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncotarget&title=Association%20between%20CYP2E1%20genetic%20polymorphisms%20and%20urinary%20cancer%20risk:%20a%20meta-analysis&author=Z.%20Fang&author=Y.%20Wu&author=N.%20Zhang&volume=8&issue=49&publication_year=2017&pages=86853-86864&pmid=29156840&doi=10.18632/oncotarget.20993&)

17. Goodman L. S., Gilman A., Brunton L. L., Lazo J. S., Parker K. L. (2006). (McGraw-Hill; ).Goodman and Gilman’s the pharmacological basis of therapeutics, New York, NY, USA,  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Goodman%20and%20Gilman%E2%80%99s%20the%20pharmacological%20basis%20of%20therapeutics&publication_year=2006&)

18. Hein D. W., Doll M. A. (2012). Accuracy of various human NAT2 SNP genotyping panels to infer rapid, intermediate and slow acetylator phenotypes. Pharmacogenomics 13 (1), 31–41. 10.2217/pgs.11.122  [DOI](https://doi.org/10.2217/pgs.11.122) | [PMC free article](/articles/PMC3285565/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22092036/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Accuracy%20of%20various%20human%20NAT2%20SNP%20genotyping%20panels%20to%20infer%20rapid,%20intermediate%20and%20slow%20acetylator%20phenotypes&author=D.%20W.%20Hein&author=M.%20A.%20Doll&volume=13&issue=1&publication_year=2012&pages=31-41&pmid=22092036&doi=10.2217/pgs.11.122&)

19. Hong B.-L., D’Cunha R., Li P., Al-Shaer M. H., Alghamdi W. A., An G., et al. (2020). A systematic review and meta-analysis of isoniazid pharmacokinetics in healthy volunteers and patients with tuberculosis. Clin. Ther. 42 (11), e220–e241. 10.1016/j.clinthera.2020.09.009  [DOI](https://doi.org/10.1016/j.clinthera.2020.09.009) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33032843/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Ther.&title=A%20systematic%20review%20and%20meta-analysis%20of%20isoniazid%20pharmacokinetics%20in%20healthy%20volunteers%20and%20patients%20with%20tuberculosis&author=B.-L.%20Hong&author=R.%20D%E2%80%99Cunha&author=P.%20Li&author=M.%20H.%20Al-Shaer&author=W.%20A.%20Alghamdi&volume=42&issue=11&publication_year=2020&pages=e220-e241&pmid=33032843&doi=10.1016/j.clinthera.2020.09.009&)

20. Iacobucci I., Lonetti A., Candoni A., Sazzini M., Papayannidis C., Formica S., et al. (2013). Profiling of drug-metabolizing enzymes/transporters in CD33+ acute myeloid leukemia patients treated with Gemtuzumab-Ozogamicin and Fludarabine, Cytarabine and Idarubicin. Pharmacogenomics J. 13 (4), 335–341. 10.1038/tpj.2012.13  [DOI](https://doi.org/10.1038/tpj.2012.13) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22584460/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J.&title=Profiling%20of%20drug-metabolizing%20enzymes/transporters%20in%20CD33+%20acute%20myeloid%20leukemia%20patients%20treated%20with%20Gemtuzumab-Ozogamicin%20and%20Fludarabine,%20Cytarabine%20and%20Idarubicin&author=I.%20Iacobucci&author=A.%20Lonetti&author=A.%20Candoni&author=M.%20Sazzini&author=C.%20Papayannidis&volume=13&issue=4&publication_year=2013&pages=335-341&pmid=22584460&doi=10.1038/tpj.2012.13&)

21. Igumnova V., Capligina V., Krams A., Cirule A., Elferts D., Pole I., et al. (2016). Genotype and allele frequencies of isoniazid-metabolizing enzymes NAT2 and GSTM1 in Latvian tuberculosis patients. J. Infect. Chemother. Official J. Jpn. Soc. Chemother. 22 (7), 472–477. 10.1016/j.jiac.2016.04.003  [DOI](https://doi.org/10.1016/j.jiac.2016.04.003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27236516/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Infect.%20Chemother.%20Official%20J.%20Jpn.%20Soc.%20Chemother.&title=Genotype%20and%20allele%20frequencies%20of%20isoniazid-metabolizing%20enzymes%20NAT2%20and%20GSTM1%20in%20Latvian%20tuberculosis%20patients&author=V.%20Igumnova&author=V.%20Capligina&author=A.%20Krams&author=A.%20Cirule&author=D.%20Elferts&volume=22&issue=7&publication_year=2016&pages=472-477&pmid=27236516&doi=10.1016/j.jiac.2016.04.003&)

22. Igumnova V., Kivrane A., Viksna A., Norvaisa I., Ranka R. (2022). Next-generation sequencing and bioinformatics-based protocol for the full-length CYP2E1 gene polymorphism analysis. Pharmacogenomics Personalized Med. 15, 959–965. 10.2147/PGPM.S371709  [DOI](https://doi.org/10.2147/PGPM.S371709) | [PMC free article](/articles/PMC9653044/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36393979/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20Personalized%20Med.&title=Next-generation%20sequencing%20and%20bioinformatics-based%20protocol%20for%20the%20full-length%20CYP2E1%20gene%20polymorphism%20analysis&author=V.%20Igumnova&author=A.%20Kivrane&author=A.%20Viksna&author=I.%20Norvaisa&author=R.%20Ranka&volume=15&publication_year=2022&pages=959-965&pmid=36393979&doi=10.2147/PGPM.S371709&)

23. Jalili V., Afgan E., Gu Q., Clements D., Blankenberg D., Goecks J., et al. (2020). The Galaxy platform for accessible, reproducible and collaborative biomedical analyses: 2020 update. Nucleic Acids Res. 48 (W1), W395-W402–W402. 10.1093/nar/gkaa434  [DOI](https://doi.org/10.1093/nar/gkaa434) | [PMC free article](/articles/PMC7319590/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32479607/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&title=The%20Galaxy%20platform%20for%20accessible,%20reproducible%20and%20collaborative%20biomedical%20analyses:%202020%20update&author=V.%20Jalili&author=E.%20Afgan&author=Q.%20Gu&author=D.%20Clements&author=D.%20Blankenberg&volume=48&issue=W1&publication_year=2020&pages=W395-W402-W402&pmid=32479607&doi=10.1093/nar/gkaa434&)

24. Khan N., Das A. (2022). Time for isoniazid pharmacogenomic-guided therapy of tuberculosis based on NAT2 acetylation profiles in India. Eur. J. Drug Metab. Pharmacokinet. 47 (4), 443–447. 10.1007/s13318-022-00764-x  [DOI](https://doi.org/10.1007/s13318-022-00764-x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35349098/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Drug%20Metab.%20Pharmacokinet.&title=Time%20for%20isoniazid%20pharmacogenomic-guided%20therapy%20of%20tuberculosis%20based%20on%20NAT2%20acetylation%20profiles%20in%20India&author=N.%20Khan&author=A.%20Das&volume=47&issue=4&publication_year=2022&pages=443-447&pmid=35349098&doi=10.1007/s13318-022-00764-x&)

25. Khan S., Mandal R. K., Elasbali A. M., Dar S. A., Jawed A., Wahid M., et al. (2019). Pharmacogenetic association between NAT2 gene polymorphisms and isoniazid induced hepatotoxicity: trial sequence meta-analysis as evidence. Biosci. Rep. 39 (1), BSR20180845. 10.1042/BSR20180845  [DOI](https://doi.org/10.1042/BSR20180845) | [PMC free article](/articles/PMC6331676/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30509962/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biosci.%20Rep.&title=Pharmacogenetic%20association%20between%20NAT2%20gene%20polymorphisms%20and%20isoniazid%20induced%20hepatotoxicity:%20trial%20sequence%20meta-analysis%20as%20evidence&author=S.%20Khan&author=R.%20K.%20Mandal&author=A.%20M.%20Elasbali&author=S.%20A.%20Dar&author=A.%20Jawed&volume=39&issue=1&publication_year=2019&pages=BSR20180845&pmid=30509962&doi=10.1042/BSR20180845&)

26. Khrunin A. V., Moisseev A., Gorbunova V., Limborska S. (2010). Genetic polymorphisms and the efficacy and toxicity of cisplatin-based chemotherapy in ovarian cancer patients. Pharmacogenomics J. 10 (1), 54–61. 10.1038/tpj.2009.45  [DOI](https://doi.org/10.1038/tpj.2009.45) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19786980/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J.&title=Genetic%20polymorphisms%20and%20the%20efficacy%20and%20toxicity%20of%20cisplatin-based%20chemotherapy%20in%20ovarian%20cancer%20patients&author=A.%20V.%20Khrunin&author=A.%20Moisseev&author=V.%20Gorbunova&author=S.%20Limborska&volume=10&issue=1&publication_year=2010&pages=54-61&pmid=19786980&doi=10.1038/tpj.2009.45&)

27. Kivrane A., Grinberga S., Sevostjanovs E., Igumnova V., Pole I., Viksna A., et al. (2021). LC-MS/MS method for simultaneous quantification of the first-line anti-tuberculosis drugs and six primary metabolites in patient plasma: implications for therapeutic drug monitoring. J. Chromatogr. B, Anal. Technol. Biomed. Life Sci. 1185, 122986. 10.1016/j.jchromb.2021.122986  [DOI](https://doi.org/10.1016/j.jchromb.2021.122986) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34688197/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Chromatogr.%20B,%20Anal.%20Technol.%20Biomed.%20Life%20Sci.&title=LC-MS/MS%20method%20for%20simultaneous%20quantification%20of%20the%20first-line%20anti-tuberculosis%20drugs%20and%20six%20primary%20metabolites%20in%20patient%20plasma:%20implications%20for%20therapeutic%20drug%20monitoring&author=A.%20Kivrane&author=S.%20Grinberga&author=E.%20Sevostjanovs&author=V.%20Igumnova&author=I.%20Pole&volume=1185&publication_year=2021&pages=122986&pmid=34688197&doi=10.1016/j.jchromb.2021.122986&)

28. Klein D. J., Boukouvala S., McDonagh E. M., Shuldiner S. R., Laurieri N., Thorn C. F., et al. (2016). PharmGKB summary: isoniazid pathway, pharmacokinetics. Pharmacogenetics Genomics 26 (9), 436–444. 10.1097/FPC.0000000000000232  [DOI](https://doi.org/10.1097/FPC.0000000000000232) | [PMC free article](/articles/PMC4970941/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27232112/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics%20Genomics&title=PharmGKB%20summary:%20isoniazid%20pathway,%20pharmacokinetics&author=D.%20J.%20Klein&author=S.%20Boukouvala&author=E.%20M.%20McDonagh&author=S.%20R.%20Shuldiner&author=N.%20Laurieri&volume=26&issue=9&publication_year=2016&pages=436-444&pmid=27232112&doi=10.1097/FPC.0000000000000232&)

29. Lei S., Gu R., Ma X. (2021). Clinical perspectives of isoniazid-induced liver injury. Liver Res. 5 (2), 45–52. 10.1016/j.livres.2021.02.001  [DOI](https://doi.org/10.1016/j.livres.2021.02.001) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Liver%20Res.&title=Clinical%20perspectives%20of%20isoniazid-induced%20liver%20injury&author=S.%20Lei&author=R.%20Gu&author=X.%20Ma&volume=5&issue=2&publication_year=2021&pages=45-52&doi=10.1016/j.livres.2021.02.001&)

30. Märtson A.-G., Burch G., Ghimire S., Alffenaar J.-W. C., Peloquin C. A. (2021). Therapeutic drug monitoring in patients with tuberculosis and concurrent medical problems. Expert Opin. Drug Metabolism Toxicol. 17 (1), 23–39. 10.1080/17425255.2021.1836158  [DOI](https://doi.org/10.1080/17425255.2021.1836158) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33040625/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Expert%20Opin.%20Drug%20Metabolism%20Toxicol.&title=Therapeutic%20drug%20monitoring%20in%20patients%20with%20tuberculosis%20and%20concurrent%20medical%20problems&author=A.-G.%20M%C3%A4rtson&author=G.%20Burch&author=S.%20Ghimire&author=J.-W.%20C.%20Alffenaar&author=C.%20A.%20Peloquin&volume=17&issue=1&publication_year=2021&pages=23-39&pmid=33040625&doi=10.1080/17425255.2021.1836158&)

31. McCallum A. D., Pertinez H. E., Else L. J., Dilly-Penchala S., Chirambo A. P., Sheha I., et al. (2021). Intrapulmonary pharmacokinetics of first-line anti-tuberculosis drugs in Malawian patients with tuberculosis. Clin. Infect. Dis. 73 (9), e3365–e3373. 10.1093/cid/ciaa1265  [DOI](https://doi.org/10.1093/cid/ciaa1265) | [PMC free article](/articles/PMC8563277/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32856694/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Infect.%20Dis.&title=Intrapulmonary%20pharmacokinetics%20of%20first-line%20anti-tuberculosis%20drugs%20in%20Malawian%20patients%20with%20tuberculosis&author=A.%20D.%20McCallum&author=H.%20E.%20Pertinez&author=L.%20J.%20Else&author=S.%20Dilly-Penchala&author=A.%20P.%20Chirambo&volume=73&issue=9&publication_year=2021&pages=e3365-e3373&pmid=32856694&doi=10.1093/cid/ciaa1265&)

32. McIlleron H., Chirehwa M. T. (2018). Current research toward optimizing dosing of first-line antituberculosis treatment. Expert Rev. Anti-Infective Ther. 17 (1), 27–38. 10.1080/14787210.2019.1555031  [DOI](https://doi.org/10.1080/14787210.2019.1555031) | [PMC free article](/articles/PMC6364307/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30501530/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Expert%20Rev.%20Anti-Infective%20Ther.&title=Current%20research%20toward%20optimizing%20dosing%20of%20first-line%20antituberculosis%20treatment&author=H.%20McIlleron&author=M.%20T.%20Chirehwa&volume=17&issue=1&publication_year=2018&pages=27-38&pmid=30501530&doi=10.1080/14787210.2019.1555031&)

33. Metushi I. G., Cai P., Zhu X., Nakagawa T., Uetrecht J. P. (2011). A fresh look at the mechanism of isoniazid-induced hepatotoxicity. Clin. Pharmacol. Ther. 89 (6), 911–914. 10.1038/clpt.2010.355  [DOI](https://doi.org/10.1038/clpt.2010.355) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21412230/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=A%20fresh%20look%20at%20the%20mechanism%20of%20isoniazid-induced%20hepatotoxicity&author=I.%20G.%20Metushi&author=P.%20Cai&author=X.%20Zhu&author=T.%20Nakagawa&author=J.%20P.%20Uetrecht&volume=89&issue=6&publication_year=2011&pages=911-914&pmid=21412230&doi=10.1038/clpt.2010.355&)

34. Molla Y., Wubetu M., Dessie B. (2021). Anti-tuberculosis drug induced hepatotoxicity and associated factors among tuberculosis patients at selected hospitals, Ethiopia. Hepatic Med. Evid. Res. 13, 1–8. 10.2147/HMER.S290542  [DOI](https://doi.org/10.2147/HMER.S290542) | [PMC free article](/articles/PMC7850419/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33536799/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hepatic%20Med.%20Evid.%20Res.&title=Anti-tuberculosis%20drug%20induced%20hepatotoxicity%20and%20associated%20factors%20among%20tuberculosis%20patients%20at%20selected%20hospitals,%20Ethiopia&author=Y.%20Molla&author=M.%20Wubetu&author=B.%20Dessie&volume=13&publication_year=2021&pages=1-8&pmid=33536799&doi=10.2147/HMER.S290542&)

35. Mthiyane T., Millard J., Adamson J., Balakrishna Y., Connolly C., Owen A., et al. (2020). N-acetyltransferase 2 genotypes among Zulu-speaking South Africans and isoniazid and N-Acetyl-Isoniazid pharmacokinetics during antituberculosis treatment. Antimicrob. Agents Chemother. 64 (4), 2366–2419. 10.1128/AAC.02376-19  [DOI](https://doi.org/10.1128/AAC.02376-19) | [PMC free article](/articles/PMC7179278/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31964788/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Antimicrob.%20Agents%20Chemother.&title=N-acetyltransferase%202%20genotypes%20among%20Zulu-speaking%20South%20Africans%20and%20isoniazid%20and%20N-Acetyl-Isoniazid%20pharmacokinetics%20during%20antituberculosis%20treatment&author=T.%20Mthiyane&author=J.%20Millard&author=J.%20Adamson&author=Y.%20Balakrishna&author=C.%20Connolly&volume=64&issue=4&publication_year=2020&pages=2366-2419&pmid=31964788&doi=10.1128/AAC.02376-19&)

36. O’Shea D., Kim R. B., Wilkinson G. R. (1997). Modulation of CYP2E1 activity by isoniazid in rapid and slow N-acetylators. Br. J. Clin. Pharmacol. 43 (1), 99–103. 10.1111/j.1365-2125.1997.tb00039.x  [DOI](https://doi.org/10.1111/j.1365-2125.1997.tb00039.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9056059/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Clin.%20Pharmacol.&title=Modulation%20of%20CYP2E1%20activity%20by%20isoniazid%20in%20rapid%20and%20slow%20N-acetylators&author=D.%20O%E2%80%99Shea&author=R.%20B.%20Kim&author=G.%20R.%20Wilkinson&volume=43&issue=1&publication_year=1997&pages=99-103&pmid=9056059&doi=10.1111/j.1365-2125.1997.tb00039.x&)

37. Perumal R., Naidoo K., Naidoo A., Ramachandran G., Requena-Mendez A., Sekaggya-Wiltshire C., et al. (2020). A systematic review and meta-analysis of first-line tuberculosis drug concentrations and treatment outcomes. Int. J. Tuberc. Lung Dis. Official J. Int. Union Against Tuberc. Lung Dis. 24 (1), 48–64. 10.5588/ijtld.19.0025  [DOI](https://doi.org/10.5588/ijtld.19.0025) | [PMC free article](/articles/PMC10622255/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32005307/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Tuberc.%20Lung%20Dis.%20Official%20J.%20Int.%20Union%20Against%20Tuberc.%20Lung%20Dis.&title=A%20systematic%20review%20and%20meta-analysis%20of%20first-line%20tuberculosis%20drug%20concentrations%20and%20treatment%20outcomes&author=R.%20Perumal&author=K.%20Naidoo&author=A.%20Naidoo&author=G.%20Ramachandran&author=A.%20Requena-Mendez&volume=24&issue=1&publication_year=2020&pages=48-64&pmid=32005307&doi=10.5588/ijtld.19.0025&)

38. Perwitasari D. A., Darmawan E., Mulyani U. A., Vlies P. V. D., Alffenaar J.-W. C., Atthobar J., et al. (2018). Polymorphisms of NAT2, CYP2E1, GST, and HLA related to drug-induced liver injury in Indonesian tuberculosis patients. Int. J. Mycobacteriology 7 (4), 380–386. 10.4103/ijmy.ijmy_143_18  [DOI](https://doi.org/10.4103/ijmy.ijmy_143_18) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30531039/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Mycobacteriology&title=Polymorphisms%20of%20NAT2,%20CYP2E1,%20GST,%20and%20HLA%20related%20to%20drug-induced%20liver%20injury%20in%20Indonesian%20tuberculosis%20patients&author=D.%20A.%20Perwitasari&author=E.%20Darmawan&author=U.%20A.%20Mulyani&author=P.%20V.%20D.%20Vlies&author=J.-W.%20C.%20Alffenaar&volume=7&issue=4&publication_year=2018&pages=380-386&pmid=30531039&doi=10.4103/ijmy.ijmy_143_18&)

39. Richardson M., Kirkham J., Dwan K., Sloan D. J., Davies G., Jorgensen A. L. (2018). CYP genetic variants and toxicity related to anti-tubercular agents: a systematic review and meta-analysis. Syst. Rev. 7 (1), 204. 10.1186/s13643-018-0861-z  [DOI](https://doi.org/10.1186/s13643-018-0861-z) | [PMC free article](/articles/PMC6247669/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30458875/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Syst.%20Rev.&title=CYP%20genetic%20variants%20and%20toxicity%20related%20to%20anti-tubercular%20agents:%20a%20systematic%20review%20and%20meta-analysis&author=M.%20Richardson&author=J.%20Kirkham&author=K.%20Dwan&author=D.%20J.%20Sloan&author=G.%20Davies&volume=7&issue=1&publication_year=2018&pages=204&pmid=30458875&doi=10.1186/s13643-018-0861-z&)

40. Richardson M., Kirkham J., Dwan K., Sloan D. J., Davies G., Jorgensen A. L. (2019). NAT2 variants and toxicity related to anti-tuberculosis agents: a systematic review and meta-analysis. Int. J. Tuberc. Lung Dis. 23 (3), 293–305. 10.5588/ijtld.18.0324  [DOI](https://doi.org/10.5588/ijtld.18.0324) | [PMC free article](/articles/PMC6421944/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30871660/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Tuberc.%20Lung%20Dis.&title=NAT2%20variants%20and%20toxicity%20related%20to%20anti-tuberculosis%20agents:%20a%20systematic%20review%20and%20meta-analysis&author=M.%20Richardson&author=J.%20Kirkham&author=K.%20Dwan&author=D.%20J.%20Sloan&author=G.%20Davies&volume=23&issue=3&publication_year=2019&pages=293-305&pmid=30871660&doi=10.5588/ijtld.18.0324&)

41. Robinson J. T., Thorvaldsdóttir H., Winckler W., Guttman M., Lander E. S., Getz G., et al. (2011). Integrative genomics viewer. Nat. Biotechnol. 29 (1), 24–26. 10.1038/nbt.1754  [DOI](https://doi.org/10.1038/nbt.1754) | [PMC free article](/articles/PMC3346182/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21221095/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Biotechnol.&title=Integrative%20genomics%20viewer&author=J.%20T.%20Robinson&author=H.%20Thorvaldsd%C3%B3ttir&author=W.%20Winckler&author=M.%20Guttman&author=E.%20S.%20Lander&volume=29&issue=1&publication_year=2011&pages=24-26&pmid=21221095&doi=10.1038/nbt.1754&)

42. Sabbagh A., Darlu P., Crouau-Roy B., Poloni E. S. (2011). Arylamine N-acetyltransferase 2 (NAT2) genetic diversity and traditional subsistence: a worldwide population survey. PloS One 6 (4), e18507. 10.1371/journal.pone.0018507  [DOI](https://doi.org/10.1371/journal.pone.0018507) | [PMC free article](/articles/PMC3071824/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21494681/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PloS%20One&title=Arylamine%20N-acetyltransferase%202%20(NAT2)%20genetic%20diversity%20and%20traditional%20subsistence:%20a%20worldwide%20population%20survey&author=A.%20Sabbagh&author=P.%20Darlu&author=B.%20Crouau-Roy&author=E.%20S.%20Poloni&volume=6&issue=4&publication_year=2011&pages=e18507&pmid=21494681&doi=10.1371/journal.pone.0018507&)

43. Sambrook J., Fritsch E. F., Maniatis T. (1989). Molecular cloning: a lab manual. A Lab. Man. 3.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=A%20Lab.%20Man.&title=Molecular%20cloning:%20a%20lab%20manual&author=J.%20Sambrook&author=E.%20F.%20Fritsch&author=T.%20Maniatis&volume=3&publication_year=1989&)

44. Sekaggya-Wiltshire C., Lamorde M., Kiragga A. N., Dooley K. E., Kamya M. R., Kambugu A., et al. (2018). The utility of pharmacokinetic studies for the evaluation of exposure-response relationships for standard dose anti-tuberculosis drugs. Tuberc. Edinb. Scotl. 108, 77–82. 10.1016/j.tube.2017.11.004  [DOI](https://doi.org/10.1016/j.tube.2017.11.004) | [PMC free article](/articles/PMC5846119/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29523331/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Tuberc.%20Edinb.%20Scotl.&title=The%20utility%20of%20pharmacokinetic%20studies%20for%20the%20evaluation%20of%20exposure-response%20relationships%20for%20standard%20dose%20anti-tuberculosis%20drugs&author=C.%20Sekaggya-Wiltshire&author=M.%20Lamorde&author=A.%20N.%20Kiragga&author=K.%20E.%20Dooley&author=M.%20R.%20Kamya&volume=108&publication_year=2018&pages=77-82&pmid=29523331&doi=10.1016/j.tube.2017.11.004&)

45. Sekaggya-Wiltshire C., von Braun A., Lamorde M., Ledergerber B., Buzibye A., Henning L., et al. (2018). Delayed sputum culture conversion in tuberculosis-human immunodeficiency virus-coinfected patients with low isoniazid and rifampicin concentrations. Clin. Infect. Dis. An Official Publ. Infect. Dis. Soc. Am. 67 (5), 708–716. 10.1093/cid/ciy179  [DOI](https://doi.org/10.1093/cid/ciy179) | [PMC free article](/articles/PMC6094003/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29514175/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Infect.%20Dis.%20An%20Official%20Publ.%20Infect.%20Dis.%20Soc.%20Am.&title=Delayed%20sputum%20culture%20conversion%20in%20tuberculosis-human%20immunodeficiency%20virus-coinfected%20patients%20with%20low%20isoniazid%20and%20rifampicin%20concentrations&author=C.%20Sekaggya-Wiltshire&author=A.%20von%20Braun&author=M.%20Lamorde&author=B.%20Ledergerber&author=A.%20Buzibye&volume=67&issue=5&publication_year=2018&pages=708-716&pmid=29514175&doi=10.1093/cid/ciy179&)

46. Shen C., Meng Q., Zhang G., Hu W. (2008). Rifampicin exacerbates isoniazid-induced toxicity in human but not in rat hepatocytes in tissue-like cultures. Br. J. Pharmacol. 153 (4), 784–791. 10.1038/sj.bjp.0707611  [DOI](https://doi.org/10.1038/sj.bjp.0707611) | [PMC free article](/articles/PMC2259215/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18071298/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Pharmacol.&title=Rifampicin%20exacerbates%20isoniazid-induced%20toxicity%20in%20human%20but%20not%20in%20rat%20hepatocytes%20in%20tissue-like%20cultures&author=C.%20Shen&author=Q.%20Meng&author=G.%20Zhang&author=W.%20Hu&volume=153&issue=4&publication_year=2008&pages=784-791&pmid=18071298&doi=10.1038/sj.bjp.0707611&)

47. Silva D. R., Muñoz-Torrico M., Duarte R., Galvão T., Bonini E. H., Arbex F. F., et al. (2018). Risk factors for tuberculosis: diabetes, smoking, alcohol use, and the use of other drugs. J. Bras. Pneumol. 44 (2), 145–152. 10.1590/S1806-37562017000000443  [DOI](https://doi.org/10.1590/S1806-37562017000000443) | [PMC free article](/articles/PMC6044656/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29791552/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Bras.%20Pneumol.&title=Risk%20factors%20for%20tuberculosis:%20diabetes,%20smoking,%20alcohol%20use,%20and%20the%20use%20of%20other%20drugs&author=D.%20R.%20Silva&author=M.%20Mu%C3%B1oz-Torrico&author=R.%20Duarte&author=T.%20Galv%C3%A3o&author=E.%20H.%20Bonini&volume=44&issue=2&publication_year=2018&pages=145-152&pmid=29791552&doi=10.1590/S1806-37562017000000443&)

48. Sim E., Abuhammad A., Ryan A. (2014). Arylamine N-acetyltransferases: from drug metabolism and pharmacogenetics to drug discovery. Br. J. Pharmacol. 171 (11), 2705–2725. 10.1111/bph.12598  [DOI](https://doi.org/10.1111/bph.12598) | [PMC free article](/articles/PMC4158862/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24467436/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Pharmacol.&title=Arylamine%20N-acetyltransferases:%20from%20drug%20metabolism%20and%20pharmacogenetics%20to%20drug%20discovery&author=E.%20Sim&author=A.%20Abuhammad&author=A.%20Ryan&volume=171&issue=11&publication_year=2014&pages=2705-2725&pmid=24467436&doi=10.1111/bph.12598&)

49. Sistanizad M., Azizi E., Khalili H., Hajiabdolbaghi M., Gholami K., Mahjub R. (2011). Antituberculosis drug-induced hepatotoxicity in IranianTuberculosis patients: role of isoniazid metabolic polymorphism. Iran. J. Pharm. Res. IJPR 10 (3), 633–639.  [PMC free article](/articles/PMC3813031/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24250397/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Iran.%20J.%20Pharm.%20Res.%20IJPR&title=Antituberculosis%20drug-induced%20hepatotoxicity%20in%20IranianTuberculosis%20patients:%20role%20of%20isoniazid%20metabolic%20polymorphism&author=M.%20Sistanizad&author=E.%20Azizi&author=H.%20Khalili&author=M.%20Hajiabdolbaghi&author=K.%20Gholami&volume=10&issue=3&publication_year=2011&pages=633-639&pmid=24250397&)

50. Sodhi C. P., Rana S. V., Mehta S. K., Vaiphei K., Attri S., Thakur S., et al. (1996). Study of oxidative stress in isoniazid-induced hepatic injury in young rats with and without protein-energy malnutrition. J. Biochem. Toxicol. 11 (3), 139–146.  [DOI](https://doi.org/10.1002/(SICI)1522-7146(1996)11:3&#x0003c;139::AID-JBT6&#x0003e;3.0.CO;2-L) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9029273/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Biochem.%20Toxicol.&title=Study%20of%20oxidative%20stress%20in%20isoniazid-induced%20hepatic%20injury%20in%20young%20rats%20with%20and%20without%20protein-energy%20malnutrition&author=C.%20P.%20Sodhi&author=S.%20V.%20Rana&author=S.%20K.%20Mehta&author=K.%20Vaiphei&author=S.%20Attri&volume=11&issue=3&publication_year=1996&pages=139-146&pmid=9029273&doi=10.1002/(SICI)1522-7146(1996)11:3&#x0003c;139::AID-JBT6&#x0003e;3.0.CO;2-L&)

51. Spruance S. L., Reid J. E., Grace M., Samore M. (2004). Hazard ratio in clinical trials. Antimicrob. Agents Chemother. 48 (8), 2787–2792. 10.1128/AAC.48.8.2787-2792.2004  [DOI](https://doi.org/10.1128/AAC.48.8.2787-2792.2004) | [PMC free article](/articles/PMC478551/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15273082/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Antimicrob.%20Agents%20Chemother.&title=Hazard%20ratio%20in%20clinical%20trials&author=S.%20L.%20Spruance&author=J.%20E.%20Reid&author=M.%20Grace&author=M.%20Samore&volume=48&issue=8&publication_year=2004&pages=2787-2792&pmid=15273082&doi=10.1128/AAC.48.8.2787-2792.2004&)

52. Stanley L. A., Sim E. (2008). Update on the pharmacogenetics of NATs: structural considerations. Pharmacogenomics 9 (11), 1673–1693. 10.2217/14622416.9.11.1673  [DOI](https://doi.org/10.2217/14622416.9.11.1673) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19018723/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Update%20on%20the%20pharmacogenetics%20of%20NATs:%20structural%20considerations&author=L.%20A.%20Stanley&author=E.%20Sim&volume=9&issue=11&publication_year=2008&pages=1673-1693&pmid=19018723&doi=10.2217/14622416.9.11.1673&)

53. Sturkenboom M. G. G., Akkerman O. W., Bolhuis M. S., de Lange W. C. M., van der Werf T. S., Alffenaar J.-W. C. (2015). Adequate design of pharmacokinetic-pharmacodynamic studies will help optimize tuberculosis treatment for the future. Antimicrob. Agents Chemother. 59 (4), 2474. 10.1128/AAC.05173-14  [DOI](https://doi.org/10.1128/AAC.05173-14) | [PMC free article](/articles/PMC4356826/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25762792/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Antimicrob.%20Agents%20Chemother.&title=Adequate%20design%20of%20pharmacokinetic-pharmacodynamic%20studies%20will%20help%20optimize%20tuberculosis%20treatment%20for%20the%20future&author=M.%20G.%20G.%20Sturkenboom&author=O.%20W.%20Akkerman&author=M.%20S.%20Bolhuis&author=W.%20C.%20M.%20de%20Lange&author=T.%20S.%20van%20der%20Werf&volume=59&issue=4&publication_year=2015&pages=2474&pmid=25762792&doi=10.1128/AAC.05173-14&)

54. Taiwo I. A., Adeleye N., Anwoju F. O., Adeyinka A., Uzoma I. C., Bankole T. T. (2022). Sequence analysis for SNP detection and phylogenetic reconstruction of SARS-cov-2 isolated from Nigerian COVID-19 cases. New Microbes New Infect. 45, 100955. 10.1016/j.nmni.2022.100955  [DOI](https://doi.org/10.1016/j.nmni.2022.100955) | [PMC free article](/articles/PMC8763411/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35070322/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=New%20Microbes%20New%20Infect.&title=Sequence%20analysis%20for%20SNP%20detection%20and%20phylogenetic%20reconstruction%20of%20SARS-cov-2%20isolated%20from%20Nigerian%20COVID-19%20cases&author=I.%20A.%20Taiwo&author=N.%20Adeleye&author=F.%20O.%20Anwoju&author=A.%20Adeyinka&author=I.%20C.%20Uzoma&volume=45&publication_year=2022&pages=100955&pmid=35070322&doi=10.1016/j.nmni.2022.100955&)

55. Uematsu F., Ikawa S., Kikuchi H., Sagami I., Kanamaru R., Abe T., et al. (1994). Restriction fragment length polymorphism of the human CYP2E1 (cytochrome P450IIE1) gene and susceptibility to lung cancer: possible relevance to low smoking exposure. Pharmacogenetics 4 (2), 58–63. 10.1097/00008571-199404000-00002  [DOI](https://doi.org/10.1097/00008571-199404000-00002) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7915935/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Restriction%20fragment%20length%20polymorphism%20of%20the%20human%20CYP2E1%20(cytochrome%20P450IIE1)%20gene%20and%20susceptibility%20to%20lung%20cancer:%20possible%20relevance%20to%20low%20smoking%20exposure&author=F.%20Uematsu&author=S.%20Ikawa&author=H.%20Kikuchi&author=I.%20Sagami&author=R.%20Kanamaru&volume=4&issue=2&publication_year=1994&pages=58-63&pmid=7915935&doi=10.1097/00008571-199404000-00002&)

56. Wang K., Li M., Hakonarson H. (2010). ANNOVAR: functional annotation of genetic variants from high-throughput sequencing data. Nucleic Acids Res. 38 (16), e164. 10.1093/nar/gkq603  [DOI](https://doi.org/10.1093/nar/gkq603) | [PMC free article](/articles/PMC2938201/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20601685/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&title=ANNOVAR:%20functional%20annotation%20of%20genetic%20variants%20from%20high-throughput%20sequencing%20data&author=K.%20Wang&author=M.%20Li&author=H.%20Hakonarson&volume=38&issue=16&publication_year=2010&pages=e164&pmid=20601685&doi=10.1093/nar/gkq603&)

57. Wang P., Pradhan K., Zhong X.-B., Ma X. (2016). Isoniazid metabolism and hepatotoxicity. Acta Pharm. Sin. B 6 (5), 384–392. 10.1016/j.apsb.2016.07.014  [DOI](https://doi.org/10.1016/j.apsb.2016.07.014) | [PMC free article](/articles/PMC5045547/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27709007/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Acta%20Pharm.%20Sin.%20B&title=Isoniazid%20metabolism%20and%20hepatotoxicity&author=P.%20Wang&author=K.%20Pradhan&author=X.-B.%20Zhong&author=X.%20Ma&volume=6&issue=5&publication_year=2016&pages=384-392&pmid=27709007&doi=10.1016/j.apsb.2016.07.014&)

58. Whirl-Carrillo M., McDonagh E. M., Hebert J. M., Gong L., Sangkuhl K., Thorn C. F., et al. (2012). Pharmacogenomics knowledge for personalized medicine. Clin. Pharmacol. Ther. 92 (4), 414–417. 10.1038/clpt.2012.96  [DOI](https://doi.org/10.1038/clpt.2012.96) | [PMC free article](/articles/PMC3660037/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22992668/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Pharmacogenomics%20knowledge%20for%20personalized%20medicine&author=M.%20Whirl-Carrillo&author=E.%20M.%20McDonagh&author=J.%20M.%20Hebert&author=L.%20Gong&author=K.%20Sangkuhl&volume=92&issue=4&publication_year=2012&pages=414-417&pmid=22992668&doi=10.1038/clpt.2012.96&)

59. World Health Organization (2020). Global tuberculosis report 2020. Accessed https://www.who.int/publications/i/item/9789240013131.  [https://www.who.int/publications/i/item/9789240013131](https://www.who.int/publications/i/item/9789240013131)

60. World Health Organization (2022). Global tuberculosis report 2022. Accessed https://www.who.int/teams/global-tuberculosis-programme/tb-reports/global-tuberculosis-report-2022.  [https://www.who.int/teams/global-tuberculosis-programme/tb-reports/global-tuberculosis-report-2022](https://www.who.int/teams/global-tuberculosis-programme/tb-reports/global-tuberculosis-report-2022)

61. World Health Organization (2022). WHO operational handbook on tuberculosis Module 4: treatment – drug-susceptible tuberculosis treatment. Accessed https://www.who.int/publications/i/item/9789240050761.  [https://www.who.int/publications/i/item/9789240050761](https://www.who.int/publications/i/item/9789240050761) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35727905/)

62. Wu Z., Liu Q., Wang L., Zheng M., Guan M., Zhang M., et al. (2019). The essential role of CYP2E1 in metabolism and hepatotoxicity of N,N-dimethylformamide using a novel Cyp2e1 knockout mouse model and a population study. Archives Toxicol. 93 (11), 3169–3181. 10.1007/s00204-019-02567-7  [DOI](https://doi.org/10.1007/s00204-019-02567-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31501917/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Archives%20Toxicol.&title=The%20essential%20role%20of%20CYP2E1%20in%20metabolism%20and%20hepatotoxicity%20of%20N,N-dimethylformamide%20using%20a%20novel%20Cyp2e1%20knockout%20mouse%20model%20and%20a%20population%20study&author=Z.%20Wu&author=Q.%20Liu&author=L.%20Wang&author=M.%20Zheng&author=M.%20Guan&volume=93&issue=11&publication_year=2019&pages=3169-3181&pmid=31501917&doi=10.1007/s00204-019-02567-7&)

63. Yamada S., Tang M., Richardson K., Halaschek-Wiener J., Chan M., Cook V. J., et al. (2009). Genetic variations of NAT2 and CYP2E1 and isoniazid hepatotoxicity in a diverse population. Pharmacogenomics 10 (9), 1433–1445. 10.2217/pgs.09.66  [DOI](https://doi.org/10.2217/pgs.09.66) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19761367/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Genetic%20variations%20of%20NAT2%20and%20CYP2E1%20and%20isoniazid%20hepatotoxicity%20in%20a%20diverse%20population&author=S.%20Yamada&author=M.%20Tang&author=K.%20Richardson&author=J.%20Halaschek-Wiener&author=M.%20Chan&volume=10&issue=9&publication_year=2009&pages=1433-1445&pmid=19761367&doi=10.2217/pgs.09.66&)

64. Yang S., Hwang S. J., Park J. Y., Chung E. K., Lee J. I. (2019). Association of genetic polymorphisms of CYP2E1, NAT2, GST and SLCO1B1 with the risk of anti-tuberculosis drug-induced liver injury: a systematic review and meta-analysis. BMJ open 9 (8), e027940. 10.1136/bmjopen-2018-027940  [DOI](https://doi.org/10.1136/bmjopen-2018-027940) | [PMC free article](/articles/PMC6688699/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31375612/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMJ%20open&title=Association%20of%20genetic%20polymorphisms%20of%20CYP2E1,%20NAT2,%20GST%20and%20SLCO1B1%20with%20the%20risk%20of%20anti-tuberculosis%20drug-induced%20liver%20injury:%20a%20systematic%20review%20and%20meta-analysis&author=S.%20Yang&author=S.%20J.%20Hwang&author=J.%20Y.%20Park&author=E.%20K.%20Chung&author=J.%20I.%20Lee&volume=9&issue=8&publication_year=2019&pages=e027940&pmid=31375612&doi=10.1136/bmjopen-2018-027940&)

65. Yu Y.-Y., Tsao S.-M., Yang W.-T., Huang W.-C., Lin C.-H., Chen W.-W., et al. (2019). Association of drug metabolic enzyme genetic polymorphisms and adverse drug reactions in patients receiving rifapentine and isoniazid therapy for latent tuberculosis. Int. J. Environ. Res. Public Health 17 (1), 210. 10.3390/ijerph17010210  [DOI](https://doi.org/10.3390/ijerph17010210) | [PMC free article](/articles/PMC6981901/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31892222/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Environ.%20Res.%20Public%20Health&title=Association%20of%20drug%20metabolic%20enzyme%20genetic%20polymorphisms%20and%20adverse%20drug%20reactions%20in%20patients%20receiving%20rifapentine%20and%20isoniazid%20therapy%20for%20latent%20tuberculosis&author=Y.-Y.%20Yu&author=S.-M.%20Tsao&author=W.-T.%20Yang&author=W.-C.%20Huang&author=C.-H.%20Lin&volume=17&issue=1&publication_year=2019&pages=210&pmid=31892222&doi=10.3390/ijerph17010210&)
