# The interplay between SLC6A4 and HTR1A genetic variants that may lead to antidepressant failure

## Metadata
**Authors:** Sandra Hanna, Mark Faiz, Sanjida Ahmed, Cindy Hsieh, Sara Temkit, Cristina Nunez, Feng Zhou
**Journal:** The Pharmacogenomics Journal
**Date:** 2025 May 7
**DOI:** [10.1038/s41397-025-00370-5](https://doi.org/10.1038/s41397-025-00370-5)
**PMID:** 40335484
**PMCID:** PMC12058508
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC12058508/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC12058508/pdf/41397_2025_Article_370.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC12058508/pdf/41397_2025_Article_370.pdf)

## Abstract

The serotonin transporter (SLC6A4) and the serotonin autoreceptor (HTR1A) are two of the most extensively studied genes in the field of psychiatry, and their variants have been implicated in antidepressant response, specifically with selective serotonin reuptake inhibitors (SSRIs) which are widely regarded as the first-line medications for depression and anxiety. Variants of SLC6A4 and HTR1A have also been studied as risk factors for depression. In this retrospective study, we aim to investigate the relationship between all possible serotonin transporter (SLC6A4) and autoreceptor (HTR1A) variant expression combinations that may have contributed to the therapeutic failure of an SSRI and subsequent disability. In this study, we utilize data from a cohort of 302 European patients diagnosed with depression and/or anxiety who were referred to Personalized Prescribing Inc. (PPI) in 2022 as result of a mental health disability claim to determine whether statistical differences are present in this cohort as compared to general European population allele frequencies. Our data reveals the presence and relevance of significant differences in the presentation of SLC6A4 and HTR1A, specifically in a disability cohort, relative to the average European population. The SLC6A4 gene codes for the serotonin transporter; the SSRI drug target that aims to be blocked to prevent the recycling of serotonin, whereas the HTR1A plays an indirect role as an autoreceptor allowing serotonin levels to be maintained by the SSRI, as well as a direct role in modulating mood through post-synaptic serotonin interaction. This study has revealed statistically significant differences in the expression of these two genes together in increasing the likelihood of drug failure, specifically the presence of one or more G alleles at HTR1A rs6295 in combination with the SLC6A4 SS variant. The most significantly overrepresented combination in this cohort of patients suffering from depression and anxiety that have failed to achieve adequate symptom remission on previous SSRI trials is HTR1A rs6295 GG-SLC6A4 SS which is overrepresented in this study by over 74% at a p-value well below 0.01. Genotyping anti-depressant drug targets may play an important role in optimizing anti-depressant drug response and research developments for future therapies.

Subject terms: Gene expression, Depression

## Introduction

Selective serotonin reuptake inhibitors (SSRIs) are among the first-line antidepressant medications for the treatment of depression and/or anxiety [[1](#CR1)]. This class of medications work by binding to the serotonin transporter (*SLC6A4*) to block serotonin from being transported back to the pre-synaptic neuron for repacking [[1](#CR1)]. Theoretically, the less serotonin returning to the pre-synaptic neuron, the more serotonin available in the synaptic cleft to interact with post-synaptic serotonin receptors and modulate mood [[2](#CR2)].

The most widely studied genetic variation in the *SLC6A4* involves a variation in the length of the promoter region of the serotonin transporter protein (5-HTTLPR) [[2](#CR2)]. The short (S) variant is known to have 50% less protein expression of the serotonin transporter, compared to the long (L) allele [[2](#CR2)]. Studies indicate that patients with the long/long (LL) or long/short (LS) genotypes experience superior therapeutic response to SSRIs relative to short/short (SS) genotype carriers [[2](#CR2)]. However, studies related to this variant have revealed conflicting results. Some studies indicate that those with the SS or SL variants have improved therapeutic outcomes relative to individuals with the LL variant [[3](#CR3)], while other studies have reported no association between this gene’s variants and response to SSRIs [[4](#CR4)], despite this transporter being the main target of SSRI anti-depressant medications.

These conflicting results can partially be explained by another single nucleotide polymorphism (SNP) within the promoter region of *SLC6A4*; the rs25531A>G, which has been found to convert an L allele in the presence of its G variant to an S like expression, as stated in a meta- analysis by Kiera Stein et al. [[3](#CR3)].

It is postulated that the delay in SSRI therapeutic response (8–12 weeks) is related to the desensitization of the serotonin autoreceptor (*HTR1A*) [[5](#CR5)]. This pre-synaptic receptor is sensitive to changes in serotonin transmission and becomes initially sensitized in response to the increase in serotonin levels in the cleft due to serotonin transporter blockade, through a negative feedback mechanism, and thus allows less serotonin to leave the neuron [[6](#CR6)]. In a few weeks time (~3–6 weeks), the autoreceptor is eventually desensitized due to consistent pressure, allowing serotonin to leave the pre-synaptic neuron unchallenged [[6](#CR6)].

The *HTR1A* autoreceptor has also been the target of much debate within the psychopharmacology community, particularly the *HTR1A* rs6295 single nucleotide polymorphism (SNP) located in the 3-UTR part of the gene determining the transcription of repressing factors such as deaf-1 [[7](#CR7)]. The variant possibilities for this SNP are CC, CG, or GG [[7](#CR7)]. The G variant is missing the NUDR/deaf-1 repressor in the pre-synapse which results in heightened expression of the autoreceptor in the pre-synapse [[7](#CR7)]. This repressor functions as an enhancer in the post-synapse, and its absence leads to decreased expression of the post-synaptic 5-HT1A receptor [[7](#CR7)]. Thus, the 5-HT1A autoreceptor involving the G allele of the rs6295 SNP is hypothesized not to desensitize with ease in response to SSRI administration, limiting treatment success, especially with homozygous expression of the G allele [[7](#CR7)].

We therefore propose that the two genes (*SLC6A4* and *HTR1A*) ought to be studied together to determine SSRI efficacy. We hypothesize that the patients who have claimed disability because of failed treatment of depressive or anxious disorders and who were referred to PPI for a pharmacogenomic test have genetic profiles that predispose them to therapeutic failure of first-line antidepressant medications (SSRIs). In keeping with our observations and research, we expected that the statistical analysis reveal that the S allele of *SLC6A4* and G allele of *HTR1A* are overrepresented in this patient cohort, while accounting for pharmacokinetic variants of the *CYP2D6* and *CYP2C19* that may also lead to antidepressant medication failure.

## Materials and methods

### Study design and setting

The study began with a cohort of 492 patients that underwent pharmacogenomic testing (Rx Report-Psychiatry & Pain) in 2022 due to a mental health related condition. We excluded non-European patients, patients below 18 years of age or above 65 years of age, and patients suffering from bipolar disorder and/or ADHD. Only patients who were diagnosed by their physicians as meeting the criteria for a DSM-IV diagnosis of an anxious or depressive disorder not including bipolar disorder were considered in this study. Non-Europeans were excluded, as our cohort does not accurately reflect the ethnic diversity of all-ethnicity allele frequency statistics. To obtain accurate statistics pertaining to the difference in variation of antidepressant drug target (*SLC6A4* and *HTR1A* rs6295) genetic expression in a disabled population as against the general population, this study will compare the allele frequency of genetic drug target variants in European patients who have claimed disability as against the allele frequencies of these variants in the general European world population. A cohort of 492 European patients diagnosed with a depressive or anxious subtype were initially considered for this analysis.

Of these 492 individuals, 17 patients had both ultra-rapid function for the *CYP2C19* liver enzyme and ultra-rapid or poor function for the *CYP2D6* liver enzyme, respectively. These 17 patients were excluded as their medication failure could likely be explained by the genetic variations in their liver enzymes resulting in blunted response and/or an inability to tolerate the antidepressant due to adverse effects. The study sample was thus reduced to 475 patients.

Of the remaining 475 patients, 173 were excluded as they were not referred to us by an insurer or disability management company. The selection of the disability cohort (as referred by disability case managers from reputable Canadian insurance and disability management companies) limited eligibility to those who have confirmation of depressive or anxious subtype diagnosis and one or more drug failure based not only patient reports but on physician notes requested by insurance companies, furthering the veracity of diagnosis and history. The remaining cohort studied was 302 European disability patients referred by disability case managers. This study focuses on these 302 disability patients that failed to achieve therapeutic success during an 8–12-week trial of one or more standard maintenance doses of SSRIs as per these patients’ self-reported specifications on the signed consent form and medication questionnaire. This sample size includes an adequate number of samples to result in a minimal margin of error (~3.3%) at a confidence interval of 95% based on European world population prevalence. Therapeutic failure in this study was defined as failure to achieve a 50% reduction in depressive or anxious symptoms as per the PHQ-9 depression related questionnaire and the GAD-7 anxiety related questionnaire as compared to initial diagnostic scores, whereas success was defined as a 50% reduction in PHQ-9 depression or GAD-7 anxiety questionnaire scores as compared to previous scores. The cohort consists of 217 females and 85 males. It is important to note that upon referral, each patient is assigned a case pharmacist who determines the patient’s eligibility for testing based on medication history and adherence questionnaires. Thus, patients who likely failed medications due to complex medication regimens consisting of drug-drug interactions, drug-lifestyle interactions, duplicate therapies and phenoconversions related to liver enzyme inhibition or induction are largely eliminated prior to the testing as the pharmacist can identify another reason for drug failure. Thus, the tested cohort was limited to true mechanistic drug failure. All patients signed a consent form to allow their genetic data to be evaluated by PPI. As per the article 2.5 of the Tri-Council Policy Statement 2 of the Canadian human research ethics guidelines, the present analysis qualifies as a research study not requiring informed consent, due to the secondary use of non-identifiable human data otherwise collected for commercial purposes. The protocol was reviewed by the Research Ethics Board of Veritas IRB. Veritas IRB is a Canadian owned and accredited independent research ethics review board, located in Montreal, Québec. The REB determined that an Informed Consent Form (ICF) is not required for this study, although a consent form was previously signed by patients due to their participation in the commercial pharmacogenomic test, this consent form had specified that data may be used in an aggregated form for research purposes.

### Patient and public involvement statement

Due to the retrospective nature of the analysis of secondary information, patients and the public were not involved in the design, conduct, reporting, or dissemination plans of our research.

### Procedure

The patients were tested using the Rx Report-Psychiatry & Pain panel by PPI. The panel tests for 54 genes (104 SNPs) [[8](#CR8)] including liver enzymes (e.g., *CYP2C19* and *CYP2D6*), P-glycoprotein (P-gp) (e.g., *ABCB1*) present at the blood brain barrier, serotonin transporter (*SLC6A4*) L>S, rs25531A>G alleles, and serotonin autoreceptor *HTR1A* (rs6295 C>G).

All patients were contacted by Canadian Part A pharmacists registered and licensed by the Ontario College of Pharmacists, to complete a questionnaire on relevant diagnostic, medical, and lifestyle factors as well as depression and anxiety related health questionnaires including PHQ-9 and GAD-7. A saliva test kit was sent to patients for sample collection and patients then subsequently returned their saliva sample to the PPI laboratory. DNA was extracted from the saliva samples and amplified using polymerase chain reaction (PCR). The genetic variations (SNPs) were detected using MassArray technology by Agena Bioscience. All reagents and primers were provided by Agena Bioscience, USA.

The genetic variations (SNPs) identified were analyzed by PPI’s proprietary software and interpreted by registered pharmacists. A result summary was provided to patients as well as their treating physicians, privy to patient consent. Disability case managers received a copy of the medication compatibility summary from the pharmacist without any genetic information.

### Statistical analysis

We compared the difference in genetic variant frequency between the disability cohort of 302 European patients and the European world population prevalence of variant combinations of the *SLC6A4* and *HTR1A rs6295*, respectively. The statistical difference in allele frequencies between the European disability cohort and the European world population was calculated for all 3 variant possibilities for each gene, as well as for all 9 combination possibilities of the two genes in interplay with one another, using the following formula:

| Observeddisabilityalellefrequencyproportion%−ExpectedEuropeanallelefrequencyproportion(%)ExpectedEuropeanAlleleFrequencyProportion(%) |
| --- |
### Statistical significance

We calculated the degree of statistical significance of the difference in *SLC6A4-HTR1A* variant presentations between the disability cohort and the general European population using the one proportion z-test formula to determine standard deviation and subsequent *p*-value. We used a two-tailed hypothesis and the conventional *p*-value of 0.05 as the threshold for statistical significance. The one sample z-test formula where p is the expected proportion, po is the observed proportion and n is the 302 European disability cohort, is:

| z=p−popo(1−Po)/n |
| --- |
## Results

The genetic allele frequency of the *SLC6A4* gene for Europeans was obtained by genotyping serotonin transporter polymorphisms 5-HTTLPR [[9](#CR9)]. The L allele is 50% and the S allele is 50%. The calculated genotype percentage of LL is 25%, SS is 25%, and SL is 50% [[9](#CR9)].

The genetic allele frequency of the *HTR1A* rs6295 gene for Europeans was obtained from The National Library of Medicine, dbSNP [[10](#CR10)]. The C allele is 50% and the G allele is 49%. The CC allele is 26%, the GG allele is 24%, and the CG allele is 50% [[10](#CR10)].

The results confirm that the *SLC6A4* SS (Table [1](#Tab1)) and HTR1A GG (Table [2](#Tab2)) alleles are overrepresented in the disability population compared to the general European population.

### Table 1.

| Alleles | Number of Patients (n = 302) | Observed Disability Cohort (%) | Expected European population (%) | Difference in Disability Cohort vs. Europeans (%) |
| --- | --- | --- | --- | --- |
| LL | 78 | 25.83% | 25.00% | + 3.31% |
| LS | 119 | 39.40% | 50.00% | −21.19% |
| SS | 105 | 34.77% | 25.00% | +39.07% |

Table 1 Caption: Comparison of SLC6A4 genetic variation frequencies between disability cohort and European population.

### Table 2.

| Alleles | Number of Patients (n = 302) | Observed Disability Cohort (%) | Expected European Population (%) | Difference in Disability Cohort vs. Europeans (%) |
| --- | --- | --- | --- | --- |
| CC | 70 | 23.18% | 25.73% | −9.92% |
| CG | 138 | 45.70% | 49.90% | −8.42% |
| GG | 94 | 31.13% | 24.27% | + 28.23% |

Table 2 Caption: Comparison of HTR1A genetic variation frequencies between disability cohort and European population.

Our hypothesis, however, pertains to the cross talk between the serotonin transporter and autoreceptor. Thus, to assess the effect of these two genes on SSRI therapeutic efficacy, the data was filtered to determine the number of patients in each group of 9 possible combinations of *SLC6A4-HTR1A* variations. (Table [3](#Tab3)).

### Table 3.

|   | HTR1A | CC | CG | GG |
| --- | --- | --- | --- | --- |
| SLC6A4 | LL | LL-CC | LL-CG | LL-GG |
| LS | LS-CC | LS-CG | LS-GG |  |
| SS | SS-CC | SS-CG | SS-GG |  |

Table 3 Caption: 9 possible combinations of SLC6A4-HTR1A genetic variations.

To determine the percentage for each of the 9 *SLC6A4-HTR1A* combinations in an average European population, we multiplied individual allele frequencies of all *SLC6A4* (LL, LS, SS) and *HTR1A* (CC, CG, GG) genotypes by one another. For example, to calculate the percentage of the population in the combination group LL–CC for the general European population, we multiplied LL (0.25) with CC (0.26) to obtain 0.0643 (6.43%).

Likewise, we filtered our disability cohort for the above combinations to determine the number of individuals and percentages that fall within each *SLC6A4-HTR1A* combination. (Table [4](#Tab4)).

### Table 4.

|   |   | Number of Patients(n = 302) | Observed Disability Cohort (%) | Expected European Population (%) | Number of patients (n = 302) | Observed Disability Cohort (%) | Expected European Population (%) | Number of Patients (n = 302) | Observed Disability Cohort (%) | Expected European Population (%) |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| HTR1A |  | CC |  |  | CG |  |  | GG |  |  |
| SLC6A4 | LL | 16 | 5.30% | 6.43% | 38 | 12.58% | 12.48% | 24 | 7.95% | 6.07% |
| SL | 34 | 11.26% | 12.87% | 47 | 15.56% | 24.95% | 38 | 12.58% | 12.14% |  |
| SS | 20 | 6.62% | 6.43% | 53 | 17.55% | 12.48% | 32 | 10.60% | 6.07% |  |

Table 4 Caption: Percentage of population in each SLC6A4-HTR1A combination group.

The results affirm that the combination of *SLC6A4* SS and *HTR1A* GG is significantly overrepresented in the disability cohort compared to the general European population (Table [5](#Tab5)).

### Table 5.

| SCL6A4- HTR1A Allelic Possibilities | Statistical Difference in Allelic Combination - Disability cohort as against European Population | Z- score | p- value |
| --- | --- | --- | --- |
| LL-CC | −17.64% | −0.804 | 0.421 |
| LL-CG | +0.86% | 0.054 | 0.956 |
| LL-GG | +30.96% | 1.367 | 0.171 |
| SL-CC | −12.49% | −0.834 | 0.404 |
| SL-CG | −37.62% | −3.770 | 0.000 |
| SL-GG | +3.68% | 0.237 | 0.813 |
| SS-CC | +2.95% | 0.134 | 0.893 |
| SS-CG | +40.68% | 2.665 | 0.007 |
| SS-GG | +74.61% | 2.951 | 0.003 |

Table 5 Caption: Statistical difference in SCL6A4- HTR1A allelic combination (Disability vs European population) and subsequent z-scores and p-values from which statistical significance was determined.

The results reveal that the combination of *SLC6A4* SS - *HTR1A* rs6295 GG is highly overrepresented (+74.61%, *p* = 0.003) in the disability cohort compared to European allele frequency presentation of this combination. The combination of SS-CG (+40.68%, *p* = 0.007) is also significantly overrepresented.

While LL-GG (+30.96%, *p* = 0.171), SL-GG (+3.68%, *p* = 0.813) and LL-CG (0.86%, *p* = 0.956) are overrepresented compared to European world population presentations, these are statistically insignificant. SS-CC (+2.95, *p* = 0.893) is slightly and insignificantly overrepresented compared to expected European population statistics.

In contrast, LL-CC is underrepresented (−17.64%, *p* = 0.421), and SL-CG (−37.62%, *p* = 0.000) significantly underrepresented to a high degree in the disability cohort. SL-CC (−12.49%, *p* = 0.404) is also slightly underrepresented.

There was no significant difference in the frequency of these combinations between males and females (see Supplementary Table [1](#MOESM1)). The larger proportion of females in the disability cohort can be attributed to a higher incidence of anxious and depressive disorders in females and/or a higher number of disability claims among females.

## Discussion

The statistical analysis validates our hypothesis that patients with the *SLC6A4* SS and *HTR1A* rs6295 GG (SS-GG) are overrepresented (+74.61%) to a statistically significant degree (*p* = 0.003) in our disability cohort and may be more likely to fail treatment with a first-line medication for anxiety and depression (i.e., SSRIs). Results additionally revealed that those with allelic combinations of SS-CG, SS-CC, SL-GG, LL-CG, and LL-GG were also overrepresented (+40.68, +2.95 +3.68, +0.86, and +30.96%, respectively) in the studied disability cohort.

The presence of a homozygous C variant at *HTR1A* rs6295 is consistently underrepresented in the disability cohort except for in the presence of the homozygous S variant, while the same is not true of the L allele of *SLC6A4*. Additionally, the individual presence of one G allele at rs6295 is more overrepresented than the presence of just one S allele at *SLC6A4*, indicating that the G allele is likely to be more harmful than S allele in impeding SSRI therapeutic effect. This is further highlighted in the comparison of LL-GG being markedly more overrepresented than SS-CC indicating that the G allele is more impactful in negatively affecting therapeutic response to SSRIs, or that those with SS-CC variants are less likely to experience anxiety and/or depression.

LL-CC, SL-CC, and SL-CG are underrepresented. This is in line with previous studies indicating that the C allele of *HTR1A* rs6295 may improve response, whereas the G allele may harm response [[7](#CR7)].

The presence of one L variant of the *SLC6A4* in a patient with one C variant at *HTR1A* (SL-CG) is a significantly underrepresented (−37.62%, *p* = 0.000) combination in the disability cohort, despite being the most common and statistically likely combination within the European population. This indicates that the presence of just one L allele of *SLC6A4* with the presence of one C allele is sufficiently redeeming to potentiate drug response.

Where the results diverge is pertaining to LL-CG; this combination is overrepresented very minimally and insignificantly by 0.86% in the disability cohort. This may not be a failure of the homozygous L combination’s ability to compensate for the presence of a G allele at rs6295, but perhaps due the limitation of comparing results as against the general European population, which may encompass drug-resistant individuals, as opposed to a cohort of drug responders, which are difficult to access. Alternatively, it is possible that those with LL serotonin transporters in combination with one or more G alleles at *HTR1A* rs6295 may require higher doses for sufficient (80–90%) SLC6A4 occupancy due to the high *SLC6A4* expression of LL, such that serotonin levels are maximally increased and able to apply desensitization pressure to regulate the highly expressed autoreceptor. This would also justify LL-GG being more overrepresented in this study than LS-GG.

A meta-analytic study from Kiera Stein et. al. indicates that those with SS variant serotonin transporters experience a higher side-effect incidence and decreased response to SSRIs [[3](#CR3)]. This may be due to the lower availability of serotonin transporters for blockade. Some of these studies reveal that increased dosing results in the same therapeutic effect as those expressing the LL variant [[11](#CR11), [12](#CR12)]. At higher SSRI doses, the maximal number of serotonin transporters are occupied, and thus therapeutic success is achieved, albeit at the expense of increasing the likelihood of adverse events, which would account for the moderately weaker effect of one S variant at *SLC6A4*, relative to one G variant at rs6295.

Currently, CPIC (The Clinical Pharmacogenetics Implementation Consortium) does not provide any actionable recommendations pertaining to *SLC6A4* and SSRI therapeutics, though *SLC6A4* has shown positive association results in meta-analytic studies as well as in this study, warranting continued research to solidify findings and determine evidence-based implications for clinical use [[13](#CR13)].

Various studies have implicated the 5-HT1A autoreceptor in the response to SSRIs [[14](#CR14), [15](#CR15)]. As per Richardson- Jones et. Al 2010, mice modified to express lower levels of 5-HT1A autoreceptors show a significantly superior response to Fluoxetine (SSRI) treatment as well as individuals with a C allele at rs6295 [[16](#CR16)]. Extracellular 5HT levels were markedly increased in these mice compared with high expression 5-HT1A mice, as is observable in Fig. [1A](#Fig1) [[16](#CR16)].

### Fig. 1. Interactive effects of HTR1A rs6295 and SLC6A4 polymorphisms on SSRI treatment.

![Fig. 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/72c9/12058508/d5cd38e14c23/41397_2025_370_Fig1_HTML.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=12058508_41397_2025_370_Fig1_HTML.jpg)

A HTR1A rs6295 CC polymorphism and SLC6A4 SL or LL may result in significantly increased therapeutic response to SSRIs. The small blue circles represent serotonin. The green triangles represent an SSRI. Once an SSRI occupies and blocks the SLC6A4 serotonin re-uptake transporters represented by blue squares, there are increased levels of serotonin in the space between the pre and post synaptic neuron as well as heightened interaction with post-synaptic serotonin receptors. Initially, due increased serotonin transmission, and through a negative feedback mechanism, pre-synaptic HTR1A autoreceptors then limit the amount of serotonin leaving the neuron. Eventually, through continually increased cleft serotonin levels, the HTR1A autoreceptors are desensitized, allowing serotonin to flow freely out of the neuron, resulting in SSRI therapeutic success. B HTR1A rs6295 GG polymorphism and SLC6A4 SS polymorphism may result in significantly decreased therapeutic response to SSRIs. The small blue circles represent serotonin. The green triangles represent an SSRI. Once an SSRI occupies the few SLC6A4 serotonin re-uptake transporters of an individual with the SS low expression variant, the increase in levels of serotonin in the space between the pre and post synaptic neuron is minimal. Additionally, due to slight increased serotonin neurotransmission and through a negative feedback mechanism, HTR1A autoreceptors limit the amount of serotonin leaving the neuron. Due to an increased number HTR1A autoreceptors in rs6295 GG carriers, this minimal change towards slightly heightened serotonin levels, is insufficient to desensitize the function of HTR1A autoreceptors, thus, the change in neurotransmission is weak, and the SSRI fails to be therapeutically effective.

The heightened *HTR1A* autoreceptor expression of G allele carriers impede an SSRI’s ability to desensitize this receptor through consistent reuptake inhibition and increased cleft serotonin levels, as is observable in Fig. [1B](#Fig1) [[14](#CR14)].

SSRIs primarily modulate mood through 5-HT1A heteroreceptors [[17](#CR17)]. 5HT interaction with *HTR1A* heteroreceptors in the post-synapse modulates mood and has a negative control on NMDA activation in the dentate gyrus and on mature granule cells, also activating ERK, suppressing GSK3B, and ultimately playing an important role in neurogenesis [[18](#CR18)]. G allele carriers have fewer post-synaptic 5-HT1A receptors [[5](#CR5)]. This dual effect of 5-HT1A receptors contributes to the scope of its individual effect, which as per our data analysis is more significant than that of one S allele of *SLC6A4*.

Thus we postulate that the L allele of *SLC6A4* is possibly less protective than the C allele of 5-HT1A, or rather the S allele of *SLC6A4* is less harmful than the G allele of 5-HT1A. Patients with either a homozygous *SLC6A4* SS or *HTR1A* rs6295 GG and more specifically the combination of both these gene variants are less likely to respond to SSRI monotherapy as first-line treatment for depression and/or anxiety at standard doses.

## Limitations

There are many limitations worth noting when considering these results. Though the results are statistically significant, the cohort of patients who have claimed mental health related disability due to drug failure are best compared to a population of drug responders rather than general European Caucasian population allele frequencies, to determine a more accurate difference in allele frequency presentations. However, drug responders are difficult to incentivize, and this would requisite a future study using primary data, as first-time drug responders seldom seek pharmacogenomic testing. Additionally, the subjective nature of endpoints for depressive and anxious disorders complicates the measurement of failure and success. Symptoms for psychiatric disorders are largely subjective and answers given are dependent on many consistently fluctuating factors. The cohort was limited to those who have claimed mental health related short- or long-term disability with an insurance company in attempt to curb the inclusion of those who have not yet likely tried several medications. Furthermore, the secondary collection of medical history from patients may have led to the elimination of some relevant information.

## Clinical implications

Multiple strategies exist to improve SSRI response. Strategies include the addition of 5-HT1A antagonists such as Pindolol. However, there has been mixed success with Pindolol due to its lack of selectivity for the pre-synaptic or post-synaptic 5-HT1A receptor [[17](#CR17)]. Buspirone is a 5-HT1A partial agonist, though use of this medication is limited by its multiple daily dosing requirements [[17](#CR17)]. Perhaps the most superior adjunct in those with homozygous or heterozygous G allele at the presence of 5-HT1A rs6295 is low dose Aripiprazole, a common adjunct to SSRIs as it is a 5-HT1A partial agonist that is dosed once daily.

Other options include first time treatment with serotonin partial agonist reuptake inhibitors (SPARIs) such as Vilazodone or Vortioxetine. Much like SSRIs, SPARIs inhibit the serotonin transporter but also have 5-HT1A partial agonist properties. For those who have poor function of the *CYP2D6* liver enzyme, Vilazodone may be the better alternative as it is unaffected by *CYP2D6* activity, though it does need to be taken with food for optimal bioavailability [[17](#CR17)]. In contrast, Vortioxetine does not need to be taken with food and has some mild stimulant properties due to 5-HT3 blockade, additionally disinhibiting neurotransmitters such as adrenaline, glutamate, and acetylcholine in those who may require regulation of these systems in addition to serotonin [[19](#CR19)].

PPI reached out to the 32 patients to whom were suggested SPARIs (i.e., Vortioxetine, Vilazodone with/without Bupropion), based on the PPI test results that displayed a *SLC6A4* SS-*HTR1A* rs6295 GG combination, and who may have been subsequently prescribed one of these medications by their treating physicians. Only 8 patients were willing to provide input, while the others did not respond to follow-up emails.

Of these 8 disability patients, 7 self- reported positive effect and medication tolerance, as per the pharmacogenomic test results and pharmacist written report, though, feedback and statistics on physician adoption of test results in this specific study are lacking. Vortioxetine as well as Bupropion fall within first-line medications as per CANMAT guidelines (Canadian Network for Mood and Anxiety Treatments) and Vilazodone is classified as a second-line medication [[20](#CR20)].

Though the evidence is weak due to a very limited feedback cohort, the success rate of medication recommendations for this *HTR1A* rs6295 GG and *SLC6A4* SS cohort is ~87.5%.

## Conclusion

This data analysis supports findings related to the G allele of *HTR1A* rs6295 and SSRI failure. Specifically, homozygous GG carriers are consistently overrepresented in our disability cohort data due to treatment failure, as are heterozygous G carriers in the presence of an *SLC6A4* SS variant or an *SLC6A4* LL variant. A homozygous S at *SLC6A4* also can impede response, however, the individual presence of one S allele has a much weaker total overrepresentation and effect than an individual G allele at rs6295. Many studies have yielded conflicting results on the role of these two genes in SSRI medication response. This is the first data analysis of its kind to justify the long hypothesized cross talk between these two genes and antidepressant medication success or failure.

Future studies should include determining SSRI medication candidacy for those who are SS carriers, as they already have lower expression of the drug target aiming to be blocked. Perhaps MAO-A, MAO-B, and COMT activity in the pre-synapse and cleft should be considered to identify which SS carriers are more predisposed to a serotonin deficiency due to high MAO-A activity, or who may be a possible candidate for a stimulating medication such as Bupropion (Wellbutrin XL) based on COMT and MAO-B activity. Investigating such possibilities will be the direction of future PPI disability cohort studies, to better shed light on discrepant results.

This analysis provides evidence that pharmacodynamic genetic variations may play a significant role in SSRI anti-depressant failure, and possibly subsequent disability. Thus, individuals most likely to experience treatment failure on SSRIs can be identified prior to therapy and prescribed an optimal regimen. Pharmacogenomics, the study of how variations in genes affect an individual’s response to medication [[21](#CR21)], has the potential to revolutionize the treatment of mental illness by facilitating the selection of medications that are best suited to an individual’s unique genetic profile. Pharmacogenomics can also help to prevent drug failure that may lead to financial loss due to mental health disability.

## Supplementary information

## Acknowledgements

We thank Organizational Solutions Inc., The Health Association of Nova Scotia, and Manulife for referring their disability patients to us. None received compensation for their support.

## Author contributions

All authors had full access to all the data in the study and took responsibility for the integrity of the data and the accuracy of the data analysis. • Conceptualization: Sandra Hanna *, Mark Faiz. • Methodology: Feng Zhou. • Investigation: Feng Zhou. • Data curation: Sandra Hanna, Cindy Hsieh, Sara Temkit. • Formal analysis: Sandra Hanna, Mark Faiz, Cindy Hsieh. • Writing-original draft: Sandra Hanna, Mark Faiz. • Project administration: Sandra Hanna, Sanjida Ahmed. • Resources: Mark Faiz. • Supervision: Sanjida Ahmed. • Validation: Sanjida Ahmed. • Funding Acquisition: Mark Faiz. • Software: Mark Faiz. • Visualization: Cristina Nunez. • Writing-review & editing: Sandra Hanna, Mark Faiz, Cindy Hsieh, Sara Temkit, Feng Zhou, Cristina Nunez.

## Funding

The author(s) received no specific funding for this work. The study was funded by Personalized Prescribing Inc. 175 West Beaver Creek Rd Unit 24, Richmond Hill, ON L4B 3M1, T: 647-943-0210 E: info@personalizedprescribing.com, [www.personalizedprescribing.com](http://www.personalizedprescribing.com).

## Data availability

All data relevant to the study is included in the article.

## Competing interests

The authors listed below declare that they are employed by PPI, the company that conducted this study. However, this study was not funded by any external sources and included strictly the use of secondary data.

## Ethics approval and consent to participate

All methods followed relevant ethical guidelines and regulations. The study was approved by the Veritas IRB, Montreal, Quebec (reference number 2023-3293-14986-2), for the human studies involved. Informed consent was obtained from participants voluntarily seeking commercial pharmacogenomic testing, with consent to use their data in aggregated form for research. However, under Article 2.5 of the Tri-Council Policy Statement 2, this analysis qualifies as a research study not requiring additional informed consent, due to the secondary use of non-identifiable data collected for commercial purposes.

## Footnotes

## Supplementary information

The online version contains supplementary material available at 10.1038/s41397-025-00370-5.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

All data relevant to the study is included in the article.

### Supplementary Materials

### Data Availability Statement

All data relevant to the study is included in the article.

## References

1. Chu A, Wadhwa R. Selective serotonin reuptake inhibitors. Treasure Island (FL): StatPearls Publishing; 2022.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/32119293/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Chu%20A,%20Wadhwa%20R.%20Selective%20serotonin%20reuptake%20inhibitors.%20Treasure%20Island%20(FL):%20StatPearls%20Publishing;%202022.)

2. Luddington NS, Mandadapu A, Husk M, El-Mallakh RS. Clinical implications of genetic variation in the serotonin transporter promoter region: a review. Prim Care Companion J Clin Psychiatry. 2009;11:93–102.  [DOI](https://doi.org/10.4088/pcc.08r00656) | [PMC free article](/articles/PMC2708005/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19617941/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Luddington%20NS,%20Mandadapu%20A,%20Husk%20M,%20El-Mallakh%20RS.%20Clinical%20implications%20of%20genetic%20variation%20in%20the%20serotonin%20transporter%20promoter%20region:%20a%20review.%20Prim%20Care%20Companion%20J%20Clin%20Psychiatry.%202009;11:93%E2%80%93102.)

3. Stein K, Maruf AA, Müller DJ, Bishop JR, Bousman CA. Serotonin transporter genetic variation and antidepressant response and tolerability: a systematic review and meta-analysis. J Pers Med. 2021;11:1334.  [DOI](https://doi.org/10.3390/jpm11121334) | [PMC free article](/articles/PMC8707702/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34945806/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Stein%20K,%20Maruf%20AA,%20M%C3%BCller%20DJ,%20Bishop%20JR,%20Bousman%20CA.%20Serotonin%20transporter%20genetic%20variation%20and%20antidepressant%20response%20and%20tolerability:%20a%20systematic%20review%20and%20meta-analysis.%20J%20Pers%20Med.%202021;11:1334.)

4. Mrazek DA, Rush AJ, Biernacka JM, O’Kane DJ, Cunningham JM, Wieben ED, et al. SLC6A4 variation and citalopram response. Am J Med Genet B Neuropsychiatr Genet. 2009;150B:341–51.  [DOI](https://doi.org/10.1002/ajmg.b.30816) | [PMC free article](/articles/PMC2660379/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18618621/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Mrazek%20DA,%20Rush%20AJ,%20Biernacka%20JM,%20O%E2%80%99Kane%20DJ,%20Cunningham%20JM,%20Wieben%20ED,%20et%20al.%20SLC6A4%20variation%20and%20citalopram%20response.%20Am%20J%20Med%20Genet%20B%20Neuropsychiatr%20Genet.%202009;150B:341%E2%80%9351.)

5. Gray NA, Milak MS, DeLorenzo C, Ogden RT, Huang Y, Mann JJ, et al. Antidepressant treatment reduces serotonin-1A autoreceptor binding in major depressive disorder. Biol Psychiatry. 2013;74:26–31.  [DOI](https://doi.org/10.1016/j.biopsych.2012.11.012) | [PMC free article](/articles/PMC3690146/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23374637/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Gray%20NA,%20Milak%20MS,%20DeLorenzo%20C,%20Ogden%20RT,%20Huang%20Y,%20Mann%20JJ,%20et%20al.%20Antidepressant%20treatment%20reduces%20serotonin-1A%20autoreceptor%20binding%20in%20major%20depressive%20disorder.%20Biol%20Psychiatry.%202013;74:26%E2%80%9331.)

6. Mnie-Filali O, Abrial E, Lambás-Señas L, Haddjeri N. Long-term adaptive changes induced by antidepressants: from conventional to novel therapies. IntechOpen; 2013. Available from: 10.5772/54553.

7. Donaldson ZR, le Francois B, Santos TL, Almli LM, Boldrini M, Champagne FA, et al. The functional serotonin 1a receptor promoter polymorphism, rs6295, is associated with psychiatric illness and differences in transcription. Transl Psychiatry. 2016;6:e746–6.  [DOI](https://doi.org/10.1038/tp.2015.226) | [PMC free article](/articles/PMC4872437/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26926882/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Donaldson%20ZR,%20le%20Francois%20B,%20Santos%20TL,%20Almli%20LM,%20Boldrini%20M,%20Champagne%20FA,%20et%20al.%20The%20functional%20serotonin%201a%20receptor%20promoter%20polymorphism,%20rs6295,%20is%20associated%20with%20psychiatric%20illness%20and%20differences%20in%20transcription.%20Transl%20Psychiatry.%202016;6:e746%E2%80%936.)

8. Ahmed S, Tahir R, Akhtar U, Faiz M. Pharmacogenomics guided prescription changes improved medication effectiveness in patients with mental health-related disability: a retrospective cohort analyses. Front Genet. 2021;12:644694.  [DOI](https://doi.org/10.3389/fgene.2021.644694) | [PMC free article](/articles/PMC8370841/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34413872/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Ahmed%20S,%20Tahir%20R,%20Akhtar%20U,%20Faiz%20M.%20Pharmacogenomics%20guided%20prescription%20changes%20improved%20medication%20effectiveness%20in%20patients%20with%20mental%20health-related%20disability:%20a%20retrospective%20cohort%20analyses.%20Front%20Genet.%202021;12:644694.)

9. Odgerel Z, Talati A, Hamilton SP, Levinson DF, Weissman MM. Genotyping serotonin transporter polymorphisms 5-HTTLPR and rs25531 in European- and African-American subjects from the National Institute of Mental Health’s Collaborative Center for Genomic Studies. Transl Psychiatry. 2013;3:e307–7.  [DOI](https://doi.org/10.1038/tp.2013.80) | [PMC free article](/articles/PMC3784769/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24064711/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Odgerel%20Z,%20Talati%20A,%20Hamilton%20SP,%20Levinson%20DF,%20Weissman%20MM.%20Genotyping%20serotonin%20transporter%20polymorphisms%205-HTTLPR%20and%20rs25531%20in%20European-%20and%20African-American%20subjects%20from%20the%20National%20Institute%20of%20Mental%20Health%E2%80%99s%20Collaborative%20Center%20for%20Genomic%20Studies.%20Transl%20Psychiatry.%202013;3:e307%E2%80%937.)

10. rs6295 RefSNP Report - dbSNP - NCBI. Available from: https://www.ncbi.nlm.nih.gov/snp/rs6295.  [https://www.ncbi.nlm.nih.gov/snp/rs6295](https://www.ncbi.nlm.nih.gov/snp/rs6295)

11. Shen H, Hagino Y, Kobayashi H, Shinohara-Tanaka K, Ikeda K, Yamamoto H, et al. Regional differences in extracellular dopamine and serotonin assessed by in vivo microdialysis in mice lacking dopamine and/or serotonin transporters. Neuropsychopharmacology. 2004;29:1790–9.  [DOI](https://doi.org/10.1038/sj.npp.1300476) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15226739/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Shen%20H,%20Hagino%20Y,%20Kobayashi%20H,%20Shinohara-Tanaka%20K,%20Ikeda%20K,%20Yamamoto%20H,%20et%20al.%20Regional%20differences%20in%20extracellular%20dopamine%20and%20serotonin%20assessed%20by%20in%20vivo%20microdialysis%20in%20mice%20lacking%20dopamine%20and/or%20serotonin%20transporters.%20Neuropsychopharmacology.%202004;29:1790%E2%80%939.)

12. Rausch JL, Johnson ME, Fei YJ, Li JQ, Shendarkar N, Mac Hobby H, et al. Initial conditions of serotonin transporter kinetics and genotype: influence on ssri treatment trial outcome. Biol Psychiatry. 2002;51:723–32.  [DOI](https://doi.org/10.1016/s0006-3223(01)01283-5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11983186/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Rausch%20JL,%20Johnson%20ME,%20Fei%20YJ,%20Li%20JQ,%20Shendarkar%20N,%20Mac%20Hobby%20H,%20et%20al.%20Initial%20conditions%20of%20serotonin%20transporter%20kinetics%20and%20genotype:%20influence%20on%20ssri%20treatment%20trial%20outcome.%20Biol%20Psychiatry.%202002;51:723%E2%80%9332.)

13. CPIC® Guideline for serotonin reuptake inhibitor antidepressants and CYP2D6, CYP2C19, CYP2B6, SLC6A4, and HTR2A. cpicpgx.org. Available from: https://cpicpgx.org/guidelines/cpic-guideline-for-ssri-and-snri-antidepressants/.  [https://cpicpgx.org/guidelines/cpic-guideline-for-ssri-and-snri-antidepressants/](https://cpicpgx.org/guidelines/cpic-guideline-for-ssri-and-snri-antidepressants/) | [DOI](https://doi.org/10.1002/cpt.2903) | [PMC free article](/articles/PMC10564324/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/37032427/)

14. Albert P, Fiori L. Transcriptional dys-regulation in anxiety and major depression: 5-HT1A gene promoter architecture as a therapeutic opportunity. Curr Pharm Des. 2014;20:3738–50.  [DOI](https://doi.org/10.2174/13816128113196660740) | [PMC free article](/articles/PMC4514528/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24180393/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Albert%20P,%20Fiori%20L.%20Transcriptional%20dys-regulation%20in%20anxiety%20and%20major%20depression:%205-HT1A%20gene%20promoter%20architecture%20as%20a%20therapeutic%20opportunity.%20Curr%20Pharm%20Des.%202014;20:3738%E2%80%9350.)

15. Porcelli S, Drago A, Fabbri C, Gibiino S, Calati R, Serretti A. Pharmacogenetics of antidepressant response. J Psychiatry Neurosci. 2011;36:87–113.  [DOI](https://doi.org/10.1503/jpn.100059) | [PMC free article](/articles/PMC3044192/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21172166/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Porcelli%20S,%20Drago%20A,%20Fabbri%20C,%20Gibiino%20S,%20Calati%20R,%20Serretti%20A.%20Pharmacogenetics%20of%20antidepressant%20response.%20J%20Psychiatry%20Neurosci.%202011;36:87%E2%80%93113.)

16. Richardson-Jones JW, Craige CP, Guiard BP, Stephen A, Metzger KL, Kung HF, et al. 5-HT1A autoreceptor levels determine vulnerability to stress and response to antidepressants. Neuron. 2010;65:40–52.  [DOI](https://doi.org/10.1016/j.neuron.2009.12.003) | [PMC free article](/articles/PMC2941196/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20152112/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Richardson-Jones%20JW,%20Craige%20CP,%20Guiard%20BP,%20Stephen%20A,%20Metzger%20KL,%20Kung%20HF,%20et%20al.%205-HT1A%20autoreceptor%20levels%20determine%20vulnerability%20to%20stress%20and%20response%20to%20antidepressants.%20Neuron.%202010;65:40%E2%80%9352.)

17. Ślifirski G, Król M, Turło J. 5-HT receptors and the development of new antidepressants. Int J Mol Sci. 2021;22:9015.  [DOI](https://doi.org/10.3390/ijms22169015) | [PMC free article](/articles/PMC8396477/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34445721/) | [Google Scholar](https://scholar.google.com/scholar_lookup?%C5%9Alifirski%20G,%20Kr%C3%B3l%20M,%20Tur%C5%82o%20J.%205-HT%20receptors%20and%20the%20development%20of%20new%20antidepressants.%20Int%20J%20Mol%20Sci.%202021;22:9015.)

18. Celada P, Puig M, Amargós-Bosch M, Adell A, Artigas F. The therapeutic role of 5-HT1A and 5-HT2A receptors in depression. J Psychiatry Neurosci. 2004;29:252–65.  [PMC free article](/articles/PMC446220/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15309042/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Celada%20P,%20Puig%20M,%20Amarg%C3%B3s-Bosch%20M,%20Adell%20A,%20Artigas%20F.%20The%20therapeutic%20role%20of%205-HT1A%20and%205-HT2A%20receptors%20in%20depression.%20J%20Psychiatry%20Neurosci.%202004;29:252%E2%80%9365.)

19. McIntyre R. The role of new antidepressants in clinical practice in Canada: a brief review of vortioxetine, levominacipran ER, and vilazodone. National Library of Medicine. Dovepress; 2017;13:2913–9.  [DOI](https://doi.org/10.2147/NDT.S150589) | [PMC free article](/articles/PMC5716309/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29238196/)

20. Kennedy S. Canadian psychiatric association association des psychiatres du Canada. Can J Psychiatry. 2016;61:540–60.

21. Roden DM, Wilke RA, Kroemer HK, Stein CM. Pharmacogenomics. Circulation. 2011;123:1661–70.  [DOI](https://doi.org/10.1161/CIRCULATIONAHA.109.914820) | [PMC free article](/articles/PMC3093198/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21502584/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Roden%20DM,%20Wilke%20RA,%20Kroemer%20HK,%20Stein%20CM.%20Pharmacogenomics.%20Circulation.%202011;123:1661%E2%80%9370.)
