# Genetic Susceptibility of HLA Alleles to Non-Steroidal Anti-Inflammatory Drug Hypersensitivity in the Taiwanese Population

## Metadata
**Authors:** Szu-Ling Chang, Chih-Hung Lai, Guan-Cheng Lin, Yi-Ming Chen, Mei-Hsuan Lee, Han-Shui Hsu, I-Chieh Chen
**Journal:** Biomedicines
**Date:** 2023 Dec 11
**DOI:** [10.3390/biomedicines11123273](https://doi.org/10.3390/biomedicines11123273)
**PMID:** 38137494
**PMCID:** PMC10741656
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10741656/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC10741656/pdf/biomedicines-11-03273.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC10741656/pdf/biomedicines-11-03273.pdf)

## Abstract

Background: Human leukocyte antigen (HLA) genes are important in many immune processes and contribute to many adverse drug reactions. Whether genetic variations in the HLA region are associated with non-steroid anti-inflammatory drug (NSAID) hypersensitivity remains uncertain. Therefore, the aim of our study was to identify HLA genetic variations in patients with NSAID hypersensitivity in the Taiwanese population. Methods: This hospital-based, retrospective case-control study enrolled 37,156 participants with NSAID exposure from the Taiwan Precision Medicine Initiative (TPMI), who were all genotyped and imputed to fine map HLA typing. Our study assigned 1217 cases to the NSAID allergy group and 12,170 controls to a matched group. Logistic regression analyses were utilized to explore associations between HLA alleles and NSAID hypersensitivity. Results: Overall, 13,387 patients were genotyped for eight major HLA alleles. Allele frequencies were different between the two groups. In the NSAID allergy group, the genotype frequencies of HLA-A*02:01, HLA-A*34:01, and HLA-DQA1*06:01 were found to be markedly elevated compared to the control group, a significance that persisted even after applying the Bonferroni correction. Furthermore, the risk of NSAID allergy demonstrated a significant association with HLA-A*02:01 (OR = 1.29, p < 0.001) and HLA-A*34:01 (OR = 9.90, p = 0.001), in comparison to their respective counterparts. Notably, the genotype frequency of HLA-B*46:01 exhibited a significant increase in the severe allergy group when compared with the mild allergy group. Conclusions: We identified HLA genotypes linked to the onset and severity of NSAID hypersensitivity. Our findings establish a basis for precision prescription in future clinical applications.

Keywords: NSAID hypersensitivity, drug allergy, HLA alleles, imputation, genotypes

## 1. Introduction

Non-steroid anti-inflammatory drugs (NSAIDs) are the most commonly prescribed medicine in the current era. They are used to treat pain, fever, and many inflammatory processes [[1](#B1-biomedicines-11-03273)]. The main mechanism of action is the inhibition of the enzyme Cyclooxygenase (COX), which results in decreased synthesis of prostaglandin and other prostanoids [[2](#B2-biomedicines-11-03273)]. In addition to the therapeutic usage of NSAIDs, these drugs are well known to exert multiple adverse effects, including cardiovascular, gastrointestinal, nephrotoxic, hepatotoxic, and hypersensitive reactions [[3](#B3-biomedicines-11-03273)]. The clinical patterns of NSAID hypersensitivity have diverse symptoms, which can often present a diagnostic challenge [[4](#B4-biomedicines-11-03273)].

NSAID hypersensitivity, an adverse drug reaction (ADR), is a significant cause of drug-related allergies, posing a potential risk of lethal outcomes [[5](#B5-biomedicines-11-03273),[6](#B6-biomedicines-11-03273)]. Early detection of NSAID hypersensitivity is crucial, particularly in patients with no prior exposure to NSAIDs. Studies have shown that the prevalence of NSAID hypersensitivity in the general population ranges from 0.5 to 3.5% [[7](#B7-biomedicines-11-03273),[8](#B8-biomedicines-11-03273)]. However, this prevalence is substantially higher in certain high-risk groups, with 14.89% of severe asthma patients and 9.69% of nasal polyps patients exhibiting NSAID hypersensitivity [[9](#B9-biomedicines-11-03273)]. These observations suggest a potential link between NSAID hypersensitivity and human leukocyte antigen (*HLA*) gene expression. To delve deeper into the genetic mechanisms underlying NSAID hypersensitivity phenotypes, we employed high-throughput sequencing and characterized the genetic landscape associated with NSAID hypersensitivity phenotypes. Our results may be useful for developing new treatments and therapies for individuals with NSAID hypersensitivity.

The AC-cAMP (adenylate cyclase-cyclic adenosine monophosphate) and GC-cGMP (guanylate cyclase-cyclic guanosine monophosphate) pathways are pivotal signaling cascades within cells, intricately involved in mediating diverse physiological processes. These pathways are crucial for translating the effects of NSAIDs and are closely tied to the body’s inflammatory response. (1) The AC-cAMP pathway involves adenylate cyclase (AC), which catalyzes the conversion of ATP to cyclic AMP (cAMP). Elevated cAMP levels can modulate various cellular functions, influencing inflammation and immune responses. In the context of NSAID allergy, these pathways are disrupted, contributing to the adverse effects observed. Gaseous mediators such as nitric oxide (NO) and carbon monoxide (CO) further complicate this scenario. Some studies have implicated gaseous mediators, such as nitric oxide (NO) and carbon monoxide (CO), in the pathogenesis of NSAID allergy. NO is a signaling molecule that plays a role in various biological processes, including inflammation and vasodilation [[10](#B10-biomedicines-11-03273)]. (2) The dysregulation of these pathways and the involvement of gaseous mediators contribute to the manifestation of NSAID hypersensitivity [[11](#B11-biomedicines-11-03273)].

The *HLA* genes of the major histocompatibility complex (MHC) are important in many immune processes, and they are correlated with many autoimmune diseases and hypersensitivity reactions [[12](#B12-biomedicines-11-03273)]. Studies have shown that specific *HLA* genes are associated with enhanced risk of drug hypersensitivity reactions. For example, in Asian populations, *HLA-B*1502* was identified as a strong genetic risk factor for carbamazepine-induced Stevens–Johnson syndrome (SJS) and toxic epidermal necrolysis (TEN) [[13](#B13-biomedicines-11-03273),[14](#B14-biomedicines-11-03273)]. Similarly, *HLA-B*5801* was associated with an increased risk of allopurinol-induced SJS and TEN in Han Chinese populations [[15](#B15-biomedicines-11-03273)]. Numerous *HLA* genotypes have a significant association with adverse drug reactions, and several previous studies reported an association of NSAID hypersensitivity with MHC [[16](#B16-biomedicines-11-03273),[17](#B17-biomedicines-11-03273)]. In addition, *HLA-A*02:06, HLA-A*66:01, HLA-B*44:03,* and *HLA-C*12:03* were reported to be associated with NSAID hypersensitivity in the HLA –ADR web-based database [[18](#B18-biomedicines-11-03273)]. Furthermore, some *HLA* genes were more frequent in the asthma group [[19](#B19-biomedicines-11-03273)] and the chronic urticarial group [[20](#B20-biomedicines-11-03273)]. However, whether genetic variations in the *HLA* region are associated with NSAID hypersensitivity remains largely unknown. The purpose of this study was to utilize a hospital-based database to identify *HLA* genetic predisposition in patients who have experienced adverse drug reactions to NSAIDs in the Taiwanese population.

## 2. Materials and Methods

### 2.1. Study Design

This case-control study included 58,091 Taiwanese and was conducted using data from the Taiwan Precision Medicine Initiative (TPMI). From June 2019 to December 2021, all participants were recruited from Taichung Veterans General Hospital (TCVGH) and their electronic health records were collected. Our study cohort comprised 13,387 patients, whose genetic profiles were connected to medical claims data in TCVGH, including demographic characteristics, procedures, examinations, diagnoses, surgeries, medication prescriptions, inpatient services, and outpatient services. This study, which involved human participants, was approved by the Ethics Committee of Taichung Veterans General Hospital Institutional Review Board (IRB no. SF19153A). Written informed consent was obtained from all participants in accordance with the principles defined in the Declaration of Helsinki.

### 2.2. HLA Allele Typing

Our study cohort consisted of 13,387 patients whose DNA from venous blood (2 mL) was genotyped for 8 major MHC Class I (HLA-A, -B, -C) and Class II (-DPA1, -DPB1, -DQA1, -DQB1, -DRB1) loci using an NXType^™^ Class I NGS HLA typing kit and an AllType^™^ NGS 11-Loci Amplification kit (Thermo Fisher Scientific, Waltham, MA, USA). The sequences were analyzed using Axiom HLA Analysis 1.2 (Thermo Fisher Scientific, Waltham, MA, USA), which utilizes advanced imputation methods to enable accurate HLA typing from SNP genotype data over the extended MHC region.

### 2.3. HLA Imputation for Genotype Data

The R library *HLA* Genotype Imputation with the Attribute Bagging (HIBAG) package [[21](#B21-biomedicines-11-03273)] was utilized to perform HLA imputation for the *HLA* genes *HLA-DQA1*, *HLA-DQB1*, and *HLA-DRB1*. The imputation model used was specific to individuals of Asian ancestry. The resulting imputed data displayed two-field (4-digit) resolution for *HLA* alleles with allele frequencies (AFs) of at least 5.0%.

### 2.4. Participants

As illustrated in [Supplementary Figure S1](#app1-biomedicines-11-03273), the initial cohort included participants who had available genotyping information for *HLA* allele. We extracted participants over the age of 18 years who were exposed to NSAIDs. Data sources included 37,156 participants exposed to NSAIDs; 1217 participants who had an adverse drug reaction report (ADR) were defined as the NSAID allergy group, and 35,939 participants without ADR were defined as the NSAID non-allergy group. Both groups were matched by age and gender at a ratio of 1:10 ([Supplementary Figure S1](#app1-biomedicines-11-03273)).

### 2.5. Covariates

In our study, clinical diagnoses were made based on the International Classification of Diseases, Ninth Revision, Clinical Modification (ICD-9-CM) diagnosis codes, requiring a minimum of two outpatient diagnoses or one inpatient diagnosis between January 2009 and December 2021. We obtained comorbidity information from the electronic health records of TCVGH based on ICD-9 diagnostic codes for asthma (ICD-9-CM code 493), atopic dermatitis (ICD-9-CM code 691.8), allergic rhinitis (ICD-9-CM code 477), and urticaria (ICD-9-CM code 708). Then, we compared the allergy severity and *HLA* allele typing. According to the allergy severity level [[22](#B22-biomedicines-11-03273)], patients with severe systemic reaction (Grade III) were defined as having severe allergy, and those with local reaction/mild-to-moderate systemic reaction (Grade I/II) were defined as having mild allergy.

### 2.6. Statistical Analysis

SAS version 9.4 software (SAS Institute Inc., Cary, NC, USA) was utilized to perform the data analysis. Participants’ characteristics were presented as mean values with standard deviations and percentages across the groups. The chi-square test was utilized to determine the statistical significance between categorical variables, while univariable and multivariable logistic regression analyses were conducted to estimate the associations between *HLA* genotypes and the risks of NSAID allergy. Odds ratios (ORs) and 95% confidence intervals (95% CIs) of *HLA* alleles were estimated using logistic regression models. The final model included significant covariates; the level of significance was set at a *p* value of less than 0.05.

## 3. Results

### 3.1. Characteristics of Participants

This study enrolled a total of 13,387 patients, consisting of 4860 males and 8527 females, who were genotyped for eight major *HLA* alleles. [Table 1](#biomedicines-11-03273-t001) shows the basic characteristics of the participants. The mean age of this study group was 56 ± 16 years, and compared to men, women had a greater prevalence of developing NSAID allergy. A higher IgE value was observed in the NSAID allergy group, compared to the control group. Furthermore, the patients in the NSAID allergy group had a lower creatinine level compared to the controls (*p* < 0.001). Compared to the control group, the NSAID allergy group had significantly higher prevalence rates of comorbidities, such as allergic rhinitis (24.7% vs. 20.6%, *p* < 0.001), asthma (13.3% vs. 8.0%, *p* < 0.001), urticaria (9.5% vs. 7.4%, *p* = 0.01), and atopic dermatitis (5.9% vs. 4.4%, *p* = 0.02).

### Table 1.

| Variables | Total n = 13,387 |  | NSAID Allergy n = 1217 |  | Control n = 12,170 |  | p Value |
| --- | --- | --- | --- | --- | --- | --- | --- |
| n | (%) | n | (%) | n | (%) |  |  |
| Age, years a | 56 ± 16 |  | 55 ± 16 |  | 56 ± 16 |  | 0.46 |
| Sex (n, %) b |   |   |   |   |   |   |   |
| female | 8527 | (63.7) | 772 | (63.4) | 7755 | (63.7) |   |
| male | 4860 | (36.3) | 445 | (36.6) | 4415 | (36.3) | 0.87 |
| BMI a | 25.1 ± 4.7 |  | 24.9 ± 4.6 |  | 25.1 ± 4.7 |  | 0.34 |
| WBC a | 7317 ± 2278 |  | 7228 ± 2176 |  | 7326 ± 2288 |  | 0.16 |
| Hemoglobin (HgB) a | 12.9 ± 1.7 |  | 13.1 ± 1.7 |  | 12.9 ± 1.8 |  | 0.001 |
| PLT a | 248 ± 68 |  | 252 ± 703,580 |  | 248 ± 68 |  | 0.06 |
| Creatinine a | 1.22 ± 1.54 |  | 1.05 ± 1.06 |  | 1.23 ± 1.58 |  | <0.001 |
| GOT a | 28 ± 20 |  | 28 ± 20 |  | 28 ± 20 |  | 0.93 |
| GPT a | 28 ± 20 |  | 28 ± 19 |  | 28 ± 20 |  | 0.52 |
| IgE a | 223 ± 641 |  | 307 ± 611 |  | 212 ± 644 |  | 0.02 |
| Eosinophils a | 276 ± 3283 |  | 235 ± 267 |  | 281 ± 3488 |  | 0.87 |
| Tryptase a | 5.81 ± 5.84 |  | 2.27 ± 0.88 |  | 7.23 ± 6.41 |  | 0.16 |
| Comorbidity (n, %) b |   |   |   |   |   |   |   |
| Allergic rhinitis |   |   |   |   |   |   | <0.001 |
| No | 10,583 | (79.1) | 917 | (75.3) | 9666 | (79.4) |   |
| Yes | 2804 | (20.9) | 300 | (24.7) | 2504 | (20.6) |   |
| Asthma |   |   |   |   |   |   | <0.001 |
| No | 12,247 | (91.5) | 1055 | (86.7) | 11,192 | (92) |   |
| Yes | 1140 | (8.5) | 162 | (13.3) | 978 | (8) |   |
| Atopic dermatitis |   |   |   |   |   |   | 0.02 |
| No | 12,778 | (95.5) | 1145 | (94.1) | 11,633 | (95.6) |   |
| Yes | 609 | (4.5) | 72 | (5.9) | 537 | (4.4) |   |
| Urticaria |   |   |   |   |   |   | 0.01 |
| No | 12,375 | (92.4) | 1101 | (90.5) | 11,274 | (92.6) |   |
| Yes | 1012 | (7.6) | 116 | (9.5) | 896 | (7.4) |   |
| Autoimmune |   |   |   |   |   |   | 0.65 |
| No | 10,219 | (76.3) | 936 | (76.9) | 9283 | (76.3) |   |
| Yes | 3168 | (23.7) | 281 | (23.1) | 2887 | (23.7) |   |

Table 1 Caption: Baseline characteristics of the participants.

### 3.2. Allele Frequencies and Association between HLA Alleles and NSAID Allergy

The allele frequencies were calculated by means of direct counting. The genotype frequencies of the *HLA* alleles are shown in [Table 2](#biomedicines-11-03273-t002), with only those alleles exhibiting statistical significance presented in the table. [Supplementary tables](#app1-biomedicines-11-03273) have been meticulously prepared to encompass a comprehensive list of all identified alleles, offering a more thorough overview of the data. In the NSAID allergy group, the genotype frequencies of *HLA-A*02:01*, *HLA-A*34:01*, and *HLA-DQA1*06:01* were found to be significantly elevated compared to the control group, even after applying the Bonferroni correction. Furthermore, the risk of NSAID allergy was significantly associated with *HLA-A*02:01* (OR = 1.29, *p* < 0.001) and *HLA-A*34:01* (OR = 9.90, *p* = 0.001), compared to their counterparts ([Table 2](#biomedicines-11-03273-t002)).

### Table 2.

| Variables |  | NSAID Allergy |  | Control |  | p Value a,c | Risk of NSAID Allergy |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
| n = 2434 (Alleles) |  | n = 24,340 (Alleles) |  |  |  |  |  |  |
| n | (%) | n | (%) | OR (95% CI) | p Value b,c |  |  |  |
| A*02:01 | No | 2146 | (88.2) | 21,958 | (90.2) |   |   |   |
| Yes | 288 | (11.8) | 2382 | (9.8) | 0.002 * | 1.29 (1.12–1.48) | <0.001 * |  |
| A*34:01 | No | 2430 | (99.8) | 24,336 | (99.9) |   |   |   |
| Yes | 4 | (0.2) | 4 | (0.0) | <0.001 * | 9.90 (2.47–39.7) | 0.001 * |  |
| DQA1*03:01 | No | 2286 | (93.9) | 22,517 | (92.5) |   |   |   |
| Yes | 148 | (6.1) | 1823 | (7.5) | 0.013 | 0.79 (0.66–0.95) | 0.011 |  |
| DQA1*06:01 | No | 2174 | (89.3) | 22,165 | (91.1) |   |   |   |
| Yes |   | (10.7) | 2175 | (8.9) | 0.005 * | 1.19 (1.03–1.39) | 0.020 |  |
| DQB1*03:02 | No | 2268 | (93.2) | 22,367 | (91.9) |   |   |   |
| Yes | 166 | (6.8) | 1973 | (8.1) | 0.028 | 0.82 (0.69–0.98) | 0.027 |  |
| DRB1*12:02 | No | 2161 | (88.8) | 22,017 | (90.5) |   |   |   |
| Yes | 273 | (11.2) | 2323 | (9.5) | 0.009 * | 1.17 (1.01–1.36) | 0.032 |  |
| DRB1*16:02 | No | 2285 | (93.9) | 23,101 | (94.9) |   |   |   |
| Yes | 149 | (6.1) | 1239 | (5.1) | 0.032 | 1.21 (1.01–1.46) | 0.038 |  |

Table 2 Caption: The genotype frequencies of the HLA alleles in the participants.

### 3.3. Association of HLA Alleles and Comorbidities with the Risk of NSAID Allergy

In order to assess the association between *HLA* alleles and comorbidities, we utilized a multiple logistic regression model and evaluated its statistical significance. As shown in [Figure 1](#biomedicines-11-03273-f001), the risk of NSAID allergy was significantly associated with *HLA-A*02:01* (OR = 1.29, 95% CI: 1.12–1.48, *p* < 0.001) and *HLA-DRB1*16:02* (OR = 1.22, 95% CI: 1.01–1.47, *p* = 0.003), compared to their counterparts. Additionally, asthma (OR = 1.67, 95% CI: 1.38–2.02, *p* < 0.001) was an independent risk factor for the development of NSAID allergy.

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b6c4/10741656/9d20442e765e/biomedicines-11-03273-g001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=10741656_biomedicines-11-03273-g001.jpg)

Risk of NSAID allergy and HLA alleles in the participants. Error bars represent the 95% confidence intervals of the odds ratios. Multivariate logistic regression adjusted by age, gender, and potential confounders.

### 3.4. Distribution and Association of HLA Alleles with the Severity of Allergy

Depending on the severity of the NSAID-induced allergy, we stratified the patients into severe and mild allergy groups, respectively ([Table 3](#biomedicines-11-03273-t003)). In the severe allergy group, the mean age was 57 ± 17 years, which was significantly higher than in the mild allergy group (*p* = 0.01). However, there were no statistically significant differences observed between the blood tests. A higher prevalence of asthma (23.4% vs. 13.2%, *p* = 0.01) was also observed in the severe allergy group, compared to the mild allergy group. The genotype frequencies of *HLA* alleles and their association with the severity of allergy are shown in [Table 4](#biomedicines-11-03273-t004). Higher frequencies of *HLA-B*46:01*, *HLA-DPB1*02:02,* and *HLA-DRB*03:01* were observed in the severe allergy group compared to the mild allergy group. Specifically, the genotype frequency of *HLA-B*46:01* was significantly increased in the severe allergy group after Bonferroni correction when compared with the mild allergy group. Additionally, the risk of allergy severity was linked to specific HLA allele genotypes, including *HLA-B*46:01* (OR = 1.68, 95% CI: 1.04–2.71, *p* = 0.035), and *HLA-DPB1*02:02* (OR = 1.95, 95% CI: 1.11–3.42, *p* = 0.020), when compared to their counterparts. Conversely, *HLA-DPB1*05:01* (OR = 0.62, 95% CI: 0.39–0.99, *p* = 0.045) was associated with a reduced risk of severity in the severe allergy group, although these associations did not reach statistical significance after rigorous Bonferroni correction.

### Table 3.

| Variables | Total n = 746 |  | Severe Allergy n = 94 |  | Mild Allergy n = 652 |  | p Value |
| --- | --- | --- | --- | --- | --- | --- | --- |
| n | (%) | n | (%) | n | (%) |  |  |
| Age, years a | 54 ± 15 |  | 57 ± 17 |  | 53 ± 15 |  | 0.01 |
| Sex (n, %) b |   |   |   |   |   |   |   |
| female | 485 | (65.0) | 60 | (63.8) | 425 | (65.2) |   |
| male | 261 | (35.0) | 34 | (36.2) | 227 | (34.8) | 0.89 |
| BMI a | 25.0 ± 4.6 |  | 24.9 ± 3.8 |  | 25.0 ± 4.7 |  | 0.84 |
| Allergic rhinitis b |   |   |   |   |   |   |   |
| No | 542 | (72.7) | 66 | (70.2) | 476 | (73.0) |   |
| Yes | 204 | (27.3) | 28 | (29.8) | 176 | (27.0) | 0.66 |
| Asthma b |   |   |   |   |   |   |   |
| No | 638 | (85.5) | 72 | (76.6) | 566 | (86.8) |   |
| Yes | 108 | (14.5) | 22 | (23.4) | 86 | (13.2) | 0.01 |
| Atopic dermatitis b |   |   |   |   |   |   |   |
| No | 703 | (94.2) | 89 | (94.7) | 614 | (94.2) |   |
| Yes | 43 | (5.8) | 5 | (5.3) | 38 | (5.8) | 1.00 |
| Urticaria b |   |   |   |   |   |   |   |
| No | 668 | (89.5) | 85 | (90.4) | 583 | (89.4) |   |
| Yes | 78 | (10.5) | 9 | (9.6) | 69 | (10.6) | 0.91 |
| Autoimmune b |   |   |   |   |   |   |   |
| No | 562 | (75.3) | 75 | (79.8) | 487 | (74.7) |   |
| Yes | 184 | (24.7) | 19 | (20.2) | 165 | (25.3) | 0.35 |

Table 3 Caption: Characteristics of the study participants with allergy severity levels.

### Table 4.

| Variables |  | Severe Allergy |  | Mild Allergy |  | p Value a,c | Risk of Severe Allergy |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
| n = 188 (Alleles) |  | n = 1304 (Alleles) |  |  |  |  |  |  |
| n | (%) | n | (%) | OR (95% CI) | p Value b,c |  |  |  |
| B*46:01 | No | 154 | (81.9) | 1154 | (88.5) |   |   |   |
| Yes | 34 | (18.1) | 150 | (11.5) | 0.014 * | 1.68 (1.04–2.71) | 0.035 |  |
| DPB1*02:02 | No | 168 | (89.4) | 1225 | (93.9) |   |   |   |
| Yes | 20 | (10.6) | 79 | (6.1) | 0.0280 | 1.95 (1.11–3.42) | 0.020 |  |
| DPB1*05:01 | No | 112 | (59.6) | 671 | (51.5) |   |   |   |
| Yes | 76 | (40.4) | 633 | (48.5) | 0.0450 | 0.62 (0.39–0.99) | 0.045 |  |
| DRB1*03:01 | No | 168 | (89.4) | 1220 | (93.6) |   |   |   |
| Yes | 20 | (10.6) | 84 | (6.4) | 0.0500 | 1.73 (0.99–3.02) | 0.054 |  |

Table 4 Caption: Genotype frequencies of allergy severity in the participants.

### 3.5. Association of HLA Alleles and Comorbidities with Severity of Allergy

*HLA* alleles and comorbidities both had significant impacts on the risk of severity of NSAID-induced allergy. As shown in [Figure 2](#biomedicines-11-03273-f002), *HLA-B*46:01* (OR = 1.76, 95% CI: 1.08–2.89, *p* = 0.02) and *HLA-DPB1*02:02* (OR = 1.86, 95% CI: 1.03–3.34, *p* = 0.04) were significantly associated with an increased risk of allergy severity. Similarly, the risk of severity of allergy was strongly associated with asthma (OR = 2.01, 95% CI: 1.10–3.68, *p* = 0.02) using the multiple logistic regression model ([Figure 2](#biomedicines-11-03273-f002)).

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b6c4/10741656/b5f45346823e/biomedicines-11-03273-g002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=10741656_biomedicines-11-03273-g002.jpg)

Risk of NSAID allergy severity and HLA alleles in the participants. Error bars represent the 95% confidence intervals of the odds ratios. Multivariate logistic regression adjusted by age, gender, and potential confounders.

## 4. Discussion

NSAID hypersensitivity likely involves multiple mechanisms, including genetic predisposition, drug metabolism, pharmacokinetics, and environmental factors. Our novel finding implies that *HLA* allele frequency distribution differed between the NSAID allergic and control groups. Certain potential *HLA* alleles could potentially identify NSAID hypersensitivity in the Taiwanese population. In this study, we demonstrated an association between *HLA-A*02:01*, *HLA-A*34:01,* and NSAID-induced ADR. Patients with *HLA-A*02:01* or *HLA-A*34:01* had significantly increased NSAID allergic risk regardless of comorbidities of asthma. With respect to the severity of NSAID allergy, *HLA-B*46:01* and *HLA-DPB1*02:02* had a higher risk; in contrast, *HLA-DPB1*05:01* had a decreased risk of allergy severity. In terms of the comorbidities of asthma, *HLA-B*46:01* and *HLA-DPB1*02:02* had a higher risk, although these associations did not reach statistical significance after rigorous Bonferroni correction.

In our study, the reason we chose *HLA* genes as the target for NSAID hypersensitivity is that more allergic diseases, such as allergic rhinitis, atopic dermatitis, and asthma, are found in patients with NSAID allergy [[23](#B23-biomedicines-11-03273),[24](#B24-biomedicines-11-03273)], as shown in our data. In the current study, asthma had a stronger correlation (OR = 1.67, 95% CI: 1.38–2.02, *p* < 0.001) with the development of NSAID allergy. In previous studies, various *HLA* genes have been demonstrated to be associated with asthma, despite inconsistent findings [[25](#B25-biomedicines-11-03273),[26](#B26-biomedicines-11-03273),[27](#B27-biomedicines-11-03273)]. Therefore, the *HLA* gene could be a possible anchor for genetic targeting in this group.

Previous evidence showed correlations between *HLA* genes and drug hypersensitivity reactions [[28](#B28-biomedicines-11-03273),[29](#B29-biomedicines-11-03273),[30](#B30-biomedicines-11-03273)], and therefore, pharmacogenetic screening is a possible way to detect patients at high risk of severe drug reactions. As in our study cohort, we found that patients with certain *HLA* alleles tended to have a greater risk of NSAID allergy (*HLA-A*02:01*, *HLA-A *34:01*, *HLA-DQA1*06:01*, *HLA-DRB1*12:02*, *HLA-DRB1*16:02*). In the previous literature, *HLA-DRB* and *HLA-DQ* variability were shown to be associated with aspirin-related hypersensitivity, although the sample size was small [[31](#B31-biomedicines-11-03273),[32](#B32-biomedicines-11-03273),[33](#B33-biomedicines-11-03273),[34](#B34-biomedicines-11-03273)]. For example, *HLA-DRB* and *HLA-DPB1* showed a correlation with aspirin-induced asthma [[33](#B33-biomedicines-11-03273)]. *HLA-DRB1*1302*, *HLA-DQB1*0609*, and *HLA-DPB1*0201* were found to be related to aspirin-induced urticaria [[32](#B32-biomedicines-11-03273)]. In the HLA–ADR web-based database, *HLA-A*02:06* and HLA-B*44:03 had a higher NSAID ADR risk [[18](#B18-biomedicines-11-03273)]. Our study has a much larger sample size and demonstrated a different pattern of *HLA* allele frequency among NSAID allergy patients; however, the alleles identified in our study were not consistent with those found in the previous studies, potentially due to variations in ethnicity and epigenetic profiles.

In addition to *HLA* polymorphisms, many genetic studies have been shown to have a relationship with NSAID hypersensitivity, such as the *ALOX* 15 gene in the arachidonic acid pathway and DAO in histamine metabolism [[35](#B35-biomedicines-11-03273),[36](#B36-biomedicines-11-03273),[37](#B37-biomedicines-11-03273)]. The antigen-presenting cells are the key to activating an allergy reaction, which highlights the importance of *HLA* genes [[38](#B38-biomedicines-11-03273),[39](#B39-biomedicines-11-03273)]. Therefore, it is rational to use them as a genetic marker for hypersensitivity reactions. *HLA* has an important role in antigen-presenting cells to distinguish self from non-self peptides and trigger naïve T cell immunity, and therefore, it is the initial step for many drug hypersensitivity reactions [[29](#B29-biomedicines-11-03273)]. The different *HLA* allele variations could explain the diversity of ADR. Many studies have discussed the *HLA*–ADR relationship that has been identified in web-based databases. However, there are few data on NSAID hypersensitivity [[18](#B18-biomedicines-11-03273)].

Genotyping *HLAs* can be useful for identifying individuals who are at higher risk of developing severe adverse reactions to certain drugs, allowing healthcare providers to avoid prescribing these drugs to these individuals, thus reducing the risk of adverse reactions. In the current study, we had a large cohort which contained 37,156 patients with NSAID exposure. We found different *HLA* allele polymorphisms between the NSAID hypersensitivity and control groups.

Nevertheless, this study had certain limitations. First, it is worth noting that our study is retrospective in nature, and as such, the diagnosis of NSAID hypersensitivity was not confirmed through provocation tests. We did not differentiate between immediate hypersensitivity, delayed hypersensitivity, or pseudo-allergic reactions. Additionally, the potential existence of cross-reactivity among different classes of NSAIDs was not explored in this study. To gain a more comprehensive understanding, future research should consider prospective study designs with a focus on confirming our findings through detailed diagnostic assessments. Second, our study did not delve into the realm of epigenetic factors. The influence of epigenetic modifications on gene expression, mediated by various environmental factors and experiences, could contribute to phenotypic variations within the Taiwanese population. Further research is needed to unravel how these epigenetic changes may impact gene expression and its role in NSAID hypersensitivity. Third, it is important to recognize that our analysis was limited to genetic variation within the *HLA* region. While this is a significant factor in hypersensitivity reactions, there may be other genetic pathways outside the *HLA* region that play a role. Future prospective studies should aim to integrate multiple genetic pathways and investigate their interplay to provide a more comprehensive understanding of NSAID hypersensitivity.

## 5. Conclusions

In this retrospective case-control study, our findings underscore the clinical significance of certain *HLA* alleles in the context of NSAID hypersensitivity. Specifically, we identified a significant association between *HLA-A*02:01* and *HLA-A*34:01* with NSAID hypersensitivity, shedding light on the genetic factors involved in this condition. Furthermore, our study revealed that individuals carrying *HLA-B*46:01* and *HLA-DPB1*02:02* alleles faced a significantly elevated risk of severe NSAID allergy. These results not only enhance our understanding of the genetic basis of NSAID hypersensitivity but also have clinical implications for risk assessment and management. The recent identification of *HLA* alleles associated with NSAID allergy severity highlights personalized medicine potential. Tailoring treatments based on individual *HLA* status holds promise for safer interventions. Future studies should integrate clinical phenotypes to grasp genetics, drug metabolism, and environmental factors’ interplay in NSAID hypersensitivity, advancing precision medicine, and understanding drug reactions in practice.

## Acknowledgments

We thank all of the participants and investigators from the Taiwan Precision Medicine Initiative. The authors sincerely appreciate the assistance of the Center for Translational Medicine of Taichung Veterans General Hospital.

## Abbreviations

| BMI | body max index |
| --- | --- |
| WBC | white blood cell |
| HgB | hemoglobin |
| PLT | platelet |
| GOT | glutamic-oxalacetic transaminase. |
| GPT | glutamyl pyruvic transaminase |
| HLA | Human leukocyte antigen |
| NSAIDs | Non-steroid anti-inflammatory drugs |
| ADR | adverse drug reactions |
| MHC | major histocompatibility complex |
| SJS | Stevens–Johnson syndrome |
| TEN | toxic epidermal necrolysis |
| AFs | allele frequencies |
| ICD-9-CM | International Classification of Diseases, Ninth Revision, Clinical Modification |
## Supplementary Materials

The following supporting information can be downloaded at [https://www.mdpi.com/article/10.3390/biomedicines11123273/s1](https://www.mdpi.com/article/10.3390/biomedicines11123273/s1). Supplementary Figure S1: Illustrative flow chart of the study design. Table S1. The genotype frequencies of the *HLA-A* alleles in the participants. Table S2. The genotype frequencies of the *HLA-B* alleles in the participants. Table S3. The genotype frequencies of the *HLA-C* alleles in the participants. Table S4. The genotype frequencies of the *HLA-DPA1* alleles in the participants. Table S5. The genotype frequencies of the *HLA-DPB1* alleles in the participants. Table S6. The genotype frequencies of the *HLA-DQA1* alleles in the participants. Table S7. The genotype frequencies of the *HLA-DQB1* alleles in the participants. Table S8. The genotype frequencies of the *HLA-DRB1* alleles in the participants.

## Author Contributions

Research idea and study design: S.-L.C. and I.-C.C.; data acquisition: S.-L.C., G.-C.L., Y.-M.C. and I.-C.C.; data analysis/interpretation: S.-L.C., C.-H.L., Y.-M.C., M.-H.L., H.-S.H. and I.-C.C.; statistical analysis: G.-C.L. and I.-C.C.; supervision or mentorship: C.-H.L., Y.-M.C., M.-H.L., H.-S.H. and I.-C.C. All authors have read and agreed to the published version of the manuscript.

## Institutional Review Board Statement

This research project was approved by the ethics committee of the Taichung Veterans General Hospital Institutional Review Board (IRB no. SF19153A), and all of the participants gave written informed consent.

## Informed Consent Statement

Informed consent was obtained from all subjects involved in the study.

## Data Availability Statement

The original contributions presented in the study are included in the article and further inquiries can be directed to the corresponding author.

## Conflicts of Interest

The authors declare that the research was conducted in the absence of any commercial or financial relationship that could be construed as a potential conflict of interest. The authors have no relevant financial or non-financial interests to disclose. The funders had no role in the design of the study; in the collection, analyses, or interpretation of data; in the writing of the manuscript; or in the decision to publish the results.

## Funding Statement

This study was funded by the Ministry of Science and Technology, Taiwan [grant numbers, NSTC-110-2634-F-A49-005, NSTC-111-2218-E-039-001, and NSTC-111-2314-B-075A-003-MY3], Taichung Veterans General Hospital, Taiwan [grant numbers TCVGH-1127304B, TCVGH-1127301C, TCVGH-1127302D, and TCVGH-YM1120110], and Taichung Veterans General Hospital/National Health Research Institutes Joint Research Program [grant number TCVGH-YM1100103].

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

The original contributions presented in the study are included in the article and further inquiries can be directed to the corresponding author.

### Supplementary Materials

### Data Availability Statement

The original contributions presented in the study are included in the article and further inquiries can be directed to the corresponding author.

## References

1. Ong C.K., Lirk P., Tan C.H., Seymour R.A. An evidence-based update on nonsteroidal anti-inflammatory drugs. Clin. Med. Res. 2007;5:19–34. doi: 10.3121/cmr.2007.698.  [DOI](https://doi.org/10.3121/cmr.2007.698) | [PMC free article](/articles/PMC1855338/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17456832/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Med.%20Res.&title=An%20evidence-based%20update%20on%20nonsteroidal%20anti-inflammatory%20drugs&author=C.K.%20Ong&author=P.%20Lirk&author=C.H.%20Tan&author=R.A.%20Seymour&volume=5&publication_year=2007&pages=19-34&pmid=17456832&doi=10.3121/cmr.2007.698&)

2. Bindu S., Mazumder S., Bandyopadhyay U. Non-steroidal anti-inflammatory drugs (NSAIDs) and organ damage: A current perspective. Biochem. Pharmacol. 2020;180:114147. doi: 10.1016/j.bcp.2020.114147.  [DOI](https://doi.org/10.1016/j.bcp.2020.114147) | [PMC free article](/articles/PMC7347500/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32653589/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem.%20Pharmacol.&title=Non-steroidal%20anti-inflammatory%20drugs%20(NSAIDs)%20and%20organ%20damage:%20A%20current%20perspective&author=S.%20Bindu&author=S.%20Mazumder&author=U.%20Bandyopadhyay&volume=180&publication_year=2020&pages=114147&pmid=32653589&doi=10.1016/j.bcp.2020.114147&)

3. Harirforoosh S., Asghar W., Jamali F. Adverse effects of nonsteroidal antiinflammatory drugs: An update of gastrointestinal, cardiovascular and renal complications. J. Pharm. Pharm. Sci. 2013;16:821–847. doi: 10.18433/J3VW2F.  [DOI](https://doi.org/10.18433/J3VW2F) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24393558/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Pharm.%20Pharm.%20Sci.&title=Adverse%20effects%20of%20nonsteroidal%20antiinflammatory%20drugs:%20An%20update%20of%20gastrointestinal,%20cardiovascular%20and%20renal%20complications&author=S.%20Harirforoosh&author=W.%20Asghar&author=F.%20Jamali&volume=16&publication_year=2013&pages=821-847&pmid=24393558&doi=10.18433/J3VW2F&)

4. Yeung W.Y.W., Park H.S. Update on the Management of Nonsteroidal Anti-Inflammatory Drug Hypersensitivity. Yonsei Med. J. 2020;61:4–14. doi: 10.3349/ymj.2020.61.1.4.  [DOI](https://doi.org/10.3349/ymj.2020.61.1.4) | [PMC free article](/articles/PMC6938782/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31887794/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Yonsei%20Med.%20J.&title=Update%20on%20the%20Management%20of%20Nonsteroidal%20Anti-Inflammatory%20Drug%20Hypersensitivity&author=W.Y.W.%20Yeung&author=H.S.%20Park&volume=61&publication_year=2020&pages=4-14&pmid=31887794&doi=10.3349/ymj.2020.61.1.4&)

5. Hermans M.A.W., Otten R., Karim A.F., van Maaren M.S. Nonsteroidal anti-inflammatory drug hypersensitivity: Not always an allergy! Neth. J. Med. 2018;76:52–59.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/29515006/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neth.%20J.%20Med.&title=Nonsteroidal%20anti-inflammatory%20drug%20hypersensitivity:%20Not%20always%20an%20allergy!&author=M.A.W.%20Hermans&author=R.%20Otten&author=A.F.%20Karim&author=M.S.%20van%20Maaren&volume=76&publication_year=2018&pages=52-59&pmid=29515006&)

6. Gomes E.R., Demoly P. Epidemiology of hypersensitivity drug reactions. Curr. Opin. Allergy Clin. Immunol. 2005;5:309–316. doi: 10.1097/01.all.0000173785.81024.33.  [DOI](https://doi.org/10.1097/01.all.0000173785.81024.33) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15985812/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr.%20Opin.%20Allergy%20Clin.%20Immunol.&title=Epidemiology%20of%20hypersensitivity%20drug%20reactions&author=E.R.%20Gomes&author=P.%20Demoly&volume=5&publication_year=2005&pages=309-316&pmid=15985812&doi=10.1097/01.all.0000173785.81024.33&)

7. Thong B.Y. Nonsteroidal anti-inflammatory drug hypersensitivity in the Asia-Pacific. Asia Pac. Allergy. 2018;8:e38. doi: 10.5415/apallergy.2018.8.e38.  [DOI](https://doi.org/10.5415/apallergy.2018.8.e38) | [PMC free article](/articles/PMC6209595/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30402405/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Asia%20Pac.%20Allergy&title=Nonsteroidal%20anti-inflammatory%20drug%20hypersensitivity%20in%20the%20Asia-Pacific&author=B.Y.%20Thong&volume=8&publication_year=2018&pages=e38&pmid=30402405&doi=10.5415/apallergy.2018.8.e38&)

8. Zhou L., Dhopeshwarkar N., Blumenthal K.G., Goss F., Topaz M., Slight S.P., Bates D.W. Drug allergies documented in electronic health records of a large healthcare system. Allergy. 2016;71:1305–1313. doi: 10.1111/all.12881.  [DOI](https://doi.org/10.1111/all.12881) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26970431/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Allergy&title=Drug%20allergies%20documented%20in%20electronic%20health%20records%20of%20a%20large%20healthcare%20system&author=L.%20Zhou&author=N.%20Dhopeshwarkar&author=K.G.%20Blumenthal&author=F.%20Goss&author=M.%20Topaz&volume=71&publication_year=2016&pages=1305-1313&pmid=26970431&doi=10.1111/all.12881&)

9. Rajan J.P., Wineinger N.E., Stevenson D.D., White A.A. Prevalence of aspirin-exacerbated respiratory disease among asthmatic patients: A meta-analysis of the literature. J. Allergy Clin. Immunol. 2015;135:676–681.e1. doi: 10.1016/j.jaci.2014.08.020.  [DOI](https://doi.org/10.1016/j.jaci.2014.08.020) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25282015/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Allergy%20Clin.%20Immunol.&title=Prevalence%20of%20aspirin-exacerbated%20respiratory%20disease%20among%20asthmatic%20patients:%20A%20meta-analysis%20of%20the%20literature&author=J.P.%20Rajan&author=N.E.%20Wineinger&author=D.D.%20Stevenson&author=A.A.%20White&volume=135&publication_year=2015&pages=676-681.e1&pmid=25282015&doi=10.1016/j.jaci.2014.08.020&)

10. Denninger J.W., Marletta M.A. Guanylate cyclase and the ˙NO/cGMP signaling pathway. Biochim. Biophys. Acta. 1999;1411:334–350. doi: 10.1016/S0005-2728(99)00024-9.  [DOI](https://doi.org/10.1016/S0005-2728(99)00024-9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10320667/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochim.%20Biophys.%20Acta&title=Guanylate%20cyclase%20and%20the%20%CB%99NO/cGMP%20signaling%20pathway&author=J.W.%20Denninger&author=M.A.%20Marletta&volume=1411&publication_year=1999&pages=334-350&pmid=10320667&doi=10.1016/S0005-2728(99)00024-9&)

11. Laukeviciene A., Ugincius P., Korotkich I., Lazauskas R., Kevelaitis E. Anaphylaxis of small arteries: Putative role of nitric oxide and prostanoids. Medicina. 2010;46:38–44. doi: 10.3390/medicina46010007.  [DOI](https://doi.org/10.3390/medicina46010007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20234162/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Medicina&title=Anaphylaxis%20of%20small%20arteries:%20Putative%20role%20of%20nitric%20oxide%20and%20prostanoids&author=A.%20Laukeviciene&author=P.%20Ugincius&author=I.%20Korotkich&author=R.%20Lazauskas&author=E.%20Kevelaitis&volume=46&publication_year=2010&pages=38-44&pmid=20234162&doi=10.3390/medicina46010007&)

12. Choo S.Y. The HLA system: Genetics, immunology, clinical testing, and clinical implications. Yonsei Med. J. 2007;48:11–23. doi: 10.3349/ymj.2007.48.1.11.  [DOI](https://doi.org/10.3349/ymj.2007.48.1.11) | [PMC free article](/articles/PMC2628004/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17326240/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Yonsei%20Med.%20J.&title=The%20HLA%20system:%20Genetics,%20immunology,%20clinical%20testing,%20and%20clinical%20implications&author=S.Y.%20Choo&volume=48&publication_year=2007&pages=11-23&pmid=17326240&doi=10.3349/ymj.2007.48.1.11&)

13. Chang C.C., Too C.L., Murad S., Hussein S.H. Association of HLA-B*1502 allele with carbamazepine-induced toxic epidermal necrolysis and Stevens-Johnson syndrome in the multi-ethnic Malaysian population. Int. J. Dermatol. 2011;50:221–224. doi: 10.1111/j.1365-4632.2010.04745.x.  [DOI](https://doi.org/10.1111/j.1365-4632.2010.04745.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21244392/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Dermatol.&title=Association%20of%20HLA-B*1502%20allele%20with%20carbamazepine-induced%20toxic%20epidermal%20necrolysis%20and%20Stevens-Johnson%20syndrome%20in%20the%20multi-ethnic%20Malaysian%20population&author=C.C.%20Chang&author=C.L.%20Too&author=S.%20Murad&author=S.H.%20Hussein&volume=50&publication_year=2011&pages=221-224&pmid=21244392&doi=10.1111/j.1365-4632.2010.04745.x&)

14. Ferrell P.B., Jr., McLeod H.L. Carbamazepine, HLA-B*1502 and risk of Stevens-Johnson syndrome and toxic epidermal necrolysis: US FDA recommendations. Pharmacogenomics. 2008;9:1543–1546. doi: 10.2217/14622416.9.10.1543.  [DOI](https://doi.org/10.2217/14622416.9.10.1543) | [PMC free article](/articles/PMC2586963/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18855540/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Carbamazepine,%20HLA-B*1502%20and%20risk%20of%20Stevens-Johnson%20syndrome%20and%20toxic%20epidermal%20necrolysis:%20US%20FDA%20recommendations&author=P.B.%20Ferrell&author=H.L.%20McLeod&volume=9&publication_year=2008&pages=1543-1546&pmid=18855540&doi=10.2217/14622416.9.10.1543&)

15. Hung S.I., Chung W.H., Liou L.B., Chu C.C., Lin M., Huang H.P., Lin Y.L., Lan J.L., Yang L.C., Hong H.S., et al. HLA-B*5801 allele as a genetic marker for severe cutaneous adverse reactions caused by allopurinol. Proc. Natl. Acad. Sci. USA. 2005;102:4134–4139. doi: 10.1073/pnas.0409500102.  [DOI](https://doi.org/10.1073/pnas.0409500102) | [PMC free article](/articles/PMC554812/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15743917/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&title=HLA-B*5801%20allele%20as%20a%20genetic%20marker%20for%20severe%20cutaneous%20adverse%20reactions%20caused%20by%20allopurinol&author=S.I.%20Hung&author=W.H.%20Chung&author=L.B.%20Liou&author=C.C.%20Chu&author=M.%20Lin&volume=102&publication_year=2005&pages=4134-4139&pmid=15743917&doi=10.1073/pnas.0409500102&)

16. Khalili N. The Immunogenetics of Cutaneous Drug Reactions. Adv. Exp. Med. Biol. 2022;1367:411–431. doi: 10.1007/978-3-030-92616-8_17.  [DOI](https://doi.org/10.1007/978-3-030-92616-8_17) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35286706/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Adv.%20Exp.%20Med.%20Biol.&title=The%20Immunogenetics%20of%20Cutaneous%20Drug%20Reactions&author=N.%20Khalili&volume=1367&publication_year=2022&pages=411-431&pmid=35286706&doi=10.1007/978-3-030-92616-8_17&)

17. Quiralte J., Sánchez-García F., Torres M.J., Blanco C., Castillo R., Ortega N., de Castro F.R., Pérez-Aciego P., Carrillo T. Association of HLA-DR11 with the anaphylactoid reaction caused by nonsteroidal anti-inflammatory drugs. J. Allergy Clin. Immunol. 1999;103:685–689. doi: 10.1016/S0091-6749(99)70243-5.  [DOI](https://doi.org/10.1016/S0091-6749(99)70243-5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10200020/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Allergy%20Clin.%20Immunol.&title=Association%20of%20HLA-DR11%20with%20the%20anaphylactoid%20reaction%20caused%20by%20nonsteroidal%20anti-inflammatory%20drugs&author=J.%20Quiralte&author=F.%20S%C3%A1nchez-Garc%C3%ADa&author=M.J.%20Torres&author=C.%20Blanco&author=R.%20Castillo&volume=103&publication_year=1999&pages=685-689&pmid=10200020&doi=10.1016/S0091-6749(99)70243-5&)

18. Ghattaoraya G.S., Dundar Y., González-Galarza F.F., Maia M.H., Santos E.J., da Silva A.L., McCabe A., Middleton D., Alfirevic A., Dickson R., et al. A web resource for mining HLA associations with adverse drug reactions: HLA-ADR. Database. 2016;2016:baw069. doi: 10.1093/database/baw069.  [DOI](https://doi.org/10.1093/database/baw069) | [PMC free article](/articles/PMC5647400/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27189608/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Database&title=A%20web%20resource%20for%20mining%20HLA%20associations%20with%20adverse%20drug%20reactions:%20HLA-ADR&author=G.S.%20Ghattaoraya&author=Y.%20Dundar&author=F.F.%20Gonz%C3%A1lez-Galarza&author=M.H.%20Maia&author=E.J.%20Santos&volume=2016&publication_year=2016&pages=baw069&pmid=27189608&doi=10.1093/database/baw069&)

19. Dahmani D.I., Chila N., Abdelouahab F., Bouyoucef H., Bougrida M., Rouabah L., Nedjar F. Association between HLA-class II genes and asthma susceptibility in a selected Constantine population. Pan Afr. Med. J. 2020;35:48. doi: 10.11604/pamj.2020.35.48.13340.  [DOI](https://doi.org/10.11604/pamj.2020.35.48.13340) | [PMC free article](/articles/PMC7250225/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32537053/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pan%20Afr.%20Med.%20J.&title=Association%20between%20HLA-class%20II%20genes%20and%20asthma%20susceptibility%20in%20a%20selected%20Constantine%20population&author=D.I.%20Dahmani&author=N.%20Chila&author=F.%20Abdelouahab&author=H.%20Bouyoucef&author=M.%20Bougrida&volume=35&publication_year=2020&pages=48&pmid=32537053&doi=10.11604/pamj.2020.35.48.13340&)

20. Doğan N., Çildağ S., Yenisey Ç., Şentürk T. The association between chronic spontaneous urticaria and HLA class I and class II antigen. Turk. J. Med. Sci. 2020;50:1231–1235. doi: 10.3906/sag-1907-159.  [DOI](https://doi.org/10.3906/sag-1907-159) | [PMC free article](/articles/PMC7491287/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32394676/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Turk.%20J.%20Med.%20Sci.&title=The%20association%20between%20chronic%20spontaneous%20urticaria%20and%20HLA%20class%20I%20and%20class%20II%20antigen&author=N.%20Do%C4%9Fan&author=S.%20%C3%87ilda%C4%9F&author=%C3%87.%20Yenisey&author=T.%20%C5%9Eent%C3%BCrk&volume=50&publication_year=2020&pages=1231-1235&pmid=32394676&doi=10.3906/sag-1907-159&)

21. Zheng X., Shen J., Cox C., Wakefield J.C., Ehm M.G., Nelson M.R., Weir B.S. HIBAG--HLA genotype imputation with attribute bagging. Pharmacogenom. J. 2014;14:192–200. doi: 10.1038/tpj.2013.18.  [DOI](https://doi.org/10.1038/tpj.2013.18) | [PMC free article](/articles/PMC3772955/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23712092/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenom.%20J.&title=HIBAG--HLA%20genotype%20imputation%20with%20attribute%20bagging&author=X.%20Zheng&author=J.%20Shen&author=C.%20Cox&author=J.C.%20Wakefield&author=M.G.%20Ehm&volume=14&publication_year=2014&pages=192-200&pmid=23712092&doi=10.1038/tpj.2013.18&)

22. Niggemann B., Beyer K. Time for a new grading system for allergic reactions? Allergy. 2016;71:135–136. doi: 10.1111/all.12765.  [DOI](https://doi.org/10.1111/all.12765) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26381421/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Allergy&title=Time%20for%20a%20new%20grading%20system%20for%20allergic%20reactions?&author=B.%20Niggemann&author=K.%20Beyer&volume=71&publication_year=2016&pages=135-136&pmid=26381421&doi=10.1111/all.12765&)

23. Asero R. Single NSAID hypersensitivity is associated with atopic status. Eur. Ann. Allergy Clin. Immunol. 2015;47:48–53.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/25781194/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur.%20Ann.%20Allergy%20Clin.%20Immunol.&title=Single%20NSAID%20hypersensitivity%20is%20associated%20with%20atopic%20status&author=R.%20Asero&volume=47&publication_year=2015&pages=48-53&pmid=25781194&)

24. Asero R. Intolerance to nonsteroidal anti-inflammatory drugs might precede by years the onset of chronic urticaria. J. Allergy Clin. Immunol. 2003;111:1095–1098. doi: 10.1067/mai.2003.1444.  [DOI](https://doi.org/10.1067/mai.2003.1444) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12743575/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Allergy%20Clin.%20Immunol.&title=Intolerance%20to%20nonsteroidal%20anti-inflammatory%20drugs%20might%20precede%20by%20years%20the%20onset%20of%20chronic%20urticaria&author=R.%20Asero&volume=111&publication_year=2003&pages=1095-1098&pmid=12743575&doi=10.1067/mai.2003.1444&)

25. Li J., Hao Y., Li W., Lv X., Gao P. HLA-G in asthma and its potential as an effective therapeutic agent. Allergol. Immunopathol. 2023;51:22–29. doi: 10.15586/aei.v51i1.650.  [DOI](https://doi.org/10.15586/aei.v51i1.650) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36617818/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Allergol.%20Immunopathol.&title=HLA-G%20in%20asthma%20and%20its%20potential%20as%20an%20effective%20therapeutic%20agent&author=J.%20Li&author=Y.%20Hao&author=W.%20Li&author=X.%20Lv&author=P.%20Gao&volume=51&publication_year=2023&pages=22-29&pmid=36617818&doi=10.15586/aei.v51i1.650&)

26. Kontakioti E., Domvri K., Papakosta D., Daniilidis M. HLA and asthma phenotypes/endotypes: A review. Hum. Immunol. 2014;75:930–939. doi: 10.1016/j.humimm.2014.06.022.  [DOI](https://doi.org/10.1016/j.humimm.2014.06.022) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24994462/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum.%20Immunol.&title=HLA%20and%20asthma%20phenotypes/endotypes:%20A%20review&author=E.%20Kontakioti&author=K.%20Domvri&author=D.%20Papakosta&author=M.%20Daniilidis&volume=75&publication_year=2014&pages=930-939&pmid=24994462&doi=10.1016/j.humimm.2014.06.022&)

27. Sánchez-Borges M., Caballero-Fonseca F., Capriles-Hulett A. Cofactors and comorbidities in patients with aspirin/NSAID hypersensitivity. Allergol. Immunopathol. 2017;45:573–578. doi: 10.1016/j.aller.2016.08.010.  [DOI](https://doi.org/10.1016/j.aller.2016.08.010) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27865540/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Allergol.%20Immunopathol.&title=Cofactors%20and%20comorbidities%20in%20patients%20with%20aspirin/NSAID%20hypersensitivity&author=M.%20S%C3%A1nchez-Borges&author=F.%20Caballero-Fonseca&author=A.%20Capriles-Hulett&volume=45&publication_year=2017&pages=573-578&pmid=27865540&doi=10.1016/j.aller.2016.08.010&)

28. Saper V.E., Ombrello M.J., Tremoulet A.H., Montero-Martin G., Prahalad S., Canna S., Shimizu C., Deutsch G., Tan S.Y., Remmers E.F., et al. Severe delayed hypersensitivity reactions to IL-1 and IL-6 inhibitors link to common HLA-DRB1*15 alleles. Ann. Rheum. Dis. 2022;81:406–415. doi: 10.1136/annrheumdis-2021-220578.  [DOI](https://doi.org/10.1136/annrheumdis-2021-220578) | [PMC free article](/articles/PMC10564446/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34789453/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann.%20Rheum.%20Dis.&title=Severe%20delayed%20hypersensitivity%20reactions%20to%20IL-1%20and%20IL-6%20inhibitors%20link%20to%20common%20HLA-DRB1*15%20alleles&author=V.E.%20Saper&author=M.J.%20Ombrello&author=A.H.%20Tremoulet&author=G.%20Montero-Martin&author=S.%20Prahalad&volume=81&publication_year=2022&pages=406-415&pmid=34789453&doi=10.1136/annrheumdis-2021-220578&)

29. Redwood A.J., Pavlos R.K., White K.D., Phillips E.J. HLAs: Key regulators of T-cell-mediated drug hypersensitivity. HLA. 2018;91:3–16. doi: 10.1111/tan.13183.  [DOI](https://doi.org/10.1111/tan.13183) | [PMC free article](/articles/PMC5743596/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29171940/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=HLA&title=HLAs:%20Key%20regulators%20of%20T-cell-mediated%20drug%20hypersensitivity&author=A.J.%20Redwood&author=R.K.%20Pavlos&author=K.D.%20White&author=E.J.%20Phillips&volume=91&publication_year=2018&pages=3-16&pmid=29171940&doi=10.1111/tan.13183&)

30. Negrini S., Becquemont L. HLA-associated drug hypersensitivity and the prediction of adverse drug reactions. Pharmacogenomics. 2017;18:1441–1457. doi: 10.2217/pgs-2017-0090.  [DOI](https://doi.org/10.2217/pgs-2017-0090) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29017379/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=HLA-associated%20drug%20hypersensitivity%20and%20the%20prediction%20of%20adverse%20drug%20reactions&author=S.%20Negrini&author=L.%20Becquemont&volume=18&publication_year=2017&pages=1441-1457&pmid=29017379&doi=10.2217/pgs-2017-0090&)

31. Pacor M.L., Di Lorenzo G., Mansueto P., Martinelli N., Esposito-Pellitteri M., Pradella P., Uxa L., Di Fede G., Rini G., Corrocher R. Relationship between human leucocyte antigen class I and class II and chronic idiopathic urticaria associated with aspirin and/or NSAIDs hypersensitivity. Mediat. Inflamm. 2006;2006:62489. doi: 10.1155/MI/2006/62489.  [DOI](https://doi.org/10.1155/MI/2006/62489) | [PMC free article](/articles/PMC1657079/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17392574/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mediat.%20Inflamm.&title=Relationship%20between%20human%20leucocyte%20antigen%20class%20I%20and%20class%20II%20and%20chronic%20idiopathic%20urticaria%20associated%20with%20aspirin%20and/or%20NSAIDs%20hypersensitivity&author=M.L.%20Pacor&author=G.%20Di%20Lorenzo&author=P.%20Mansueto&author=N.%20Martinelli&author=M.%20Esposito-Pellitteri&volume=2006&publication_year=2006&pages=62489&pmid=17392574&doi=10.1155/MI/2006/62489&)

32. Kim S.H., Choi J.H., Lee K.W., Kim S.H., Shin E.S., Oh H.B., Suh C.H., Nahm D.H., Park H.S. The human leucocyte antigen-DRB1*1302-DQB1*0609-DPB1*0201 haplotype may be a strong genetic marker for aspirin-induced urticaria. Clin. Exp. Allergy. 2005;35:339–344. doi: 10.1111/j.1365-2222.2004.02197.x.  [DOI](https://doi.org/10.1111/j.1365-2222.2004.02197.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15784113/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Exp.%20Allergy&title=The%20human%20leucocyte%20antigen-DRB1*1302-DQB1*0609-DPB1*0201%20haplotype%20may%20be%20a%20strong%20genetic%20marker%20for%20aspirin-induced%20urticaria&author=S.H.%20Kim&author=J.H.%20Choi&author=K.W.%20Lee&author=S.H.%20Kim&author=E.S.%20Shin&volume=35&publication_year=2005&pages=339-344&pmid=15784113&doi=10.1111/j.1365-2222.2004.02197.x&)

33. Dekker J.W., Nizankowska E., Schmitz-Schumann M., Pile K., Bochenek G., Dyczek A., Cookson W.O., Szczeklik A. Aspirin-induced asthma and HLA-DRB1 and HLA-DPB1 genotypes. Clin. Exp. Allergy. 1997;27:574–577. doi: 10.1111/j.1365-2222.1997.tb00747.x.  [DOI](https://doi.org/10.1111/j.1365-2222.1997.tb00747.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9179433/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Exp.%20Allergy&title=Aspirin-induced%20asthma%20and%20HLA-DRB1%20and%20HLA-DPB1%20genotypes&author=J.W.%20Dekker&author=E.%20Nizankowska&author=M.%20Schmitz-Schumann&author=K.%20Pile&author=G.%20Bochenek&volume=27&publication_year=1997&pages=574-577&pmid=9179433&doi=10.1111/j.1365-2222.1997.tb00747.x&)

34. Esmaeilzadeh H., Nabavi M., Amirzargar A.A., Aryan Z., Arshi S., Bemanian M.H., Fallahpour M., Mortazavi N., Rezaei N. HLA-DRB and HLA-DQ genetic variability in patients with aspirin-exacerbated respiratory disease. Am. J. Rhinol. Allergy. 2015;29:e63–e69. doi: 10.2500/ajra.2015.29.4154.  [DOI](https://doi.org/10.2500/ajra.2015.29.4154) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25975240/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Rhinol.%20Allergy&title=HLA-DRB%20and%20HLA-DQ%20genetic%20variability%20in%20patients%20with%20aspirin-exacerbated%20respiratory%20disease&author=H.%20Esmaeilzadeh&author=M.%20Nabavi&author=A.A.%20Amirzargar&author=Z.%20Aryan&author=S.%20Arshi&volume=29&publication_year=2015&pages=e63-e69&pmid=25975240&doi=10.2500/ajra.2015.29.4154&)

35. Trinh H.K.T., Pham L.D., Le K.M., Park H.S. Pharmacogenomics of Hypersensitivity to Non-steroidal Anti-inflammatory Drugs. Front. Genet. 2021;12:647257. doi: 10.3389/fgene.2021.647257.  [DOI](https://doi.org/10.3389/fgene.2021.647257) | [PMC free article](/articles/PMC8269449/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34249079/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front.%20Genet.&title=Pharmacogenomics%20of%20Hypersensitivity%20to%20Non-steroidal%20Anti-inflammatory%20Drugs&author=H.K.T.%20Trinh&author=L.D.%20Pham&author=K.M.%20Le&author=H.S.%20Park&volume=12&publication_year=2021&pages=647257&pmid=34249079&doi=10.3389/fgene.2021.647257&)

36. Plaza-Serón M.D.C., García-Martín E., Agúndez J.A., Ayuso P. Hypersensitivity reactions to nonsteroidal anti-inflammatory drugs: An update on pharmacogenetics studies. Pharmacogenomics. 2018;19:1069–1086. doi: 10.2217/pgs-2018-0079.  [DOI](https://doi.org/10.2217/pgs-2018-0079) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30081739/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Hypersensitivity%20reactions%20to%20nonsteroidal%20anti-inflammatory%20drugs:%20An%20update%20on%20pharmacogenetics%20studies&author=M.D.C.%20Plaza-Ser%C3%B3n&author=E.%20Garc%C3%ADa-Mart%C3%ADn&author=J.A.%20Ag%C3%BAndez&author=P.%20Ayuso&volume=19&publication_year=2018&pages=1069-1086&pmid=30081739&doi=10.2217/pgs-2018-0079&)

37. Kim S.H., Park H.S. Genetic markers for differentiating aspirin-hypersensitivity. Yonsei Med. J. 2006;47:15–21. doi: 10.3349/ymj.2006.47.1.15.  [DOI](https://doi.org/10.3349/ymj.2006.47.1.15) | [PMC free article](/articles/PMC2687575/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16502481/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Yonsei%20Med.%20J.&title=Genetic%20markers%20for%20differentiating%20aspirin-hypersensitivity&author=S.H.%20Kim&author=H.S.%20Park&volume=47&publication_year=2006&pages=15-21&pmid=16502481&doi=10.3349/ymj.2006.47.1.15&)

38. Murdaca G., Contini P., Negrini S., Ciprandi G., Puppo F. Immunoregulatory Role of HLA-G in Allergic Diseases. J. Immunol. Res. 2016;2016:6865758. doi: 10.1155/2016/6865758.  [DOI](https://doi.org/10.1155/2016/6865758) | [PMC free article](/articles/PMC4931064/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27413762/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Immunol.%20Res.&title=Immunoregulatory%20Role%20of%20HLA-G%20in%20Allergic%20Diseases&author=G.%20Murdaca&author=P.%20Contini&author=S.%20Negrini&author=G.%20Ciprandi&author=F.%20Puppo&volume=2016&publication_year=2016&pages=6865758&pmid=27413762&doi=10.1155/2016/6865758&)

39. Mosaad Y.M. Clinical Role of Human Leukocyte Antigen in Health and Disease. Scand. J. Immunol. 2015;82:283–306. doi: 10.1111/sji.12329.  [DOI](https://doi.org/10.1111/sji.12329) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26099424/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Scand.%20J.%20Immunol.&title=Clinical%20Role%20of%20Human%20Leukocyte%20Antigen%20in%20Health%20and%20Disease&author=Y.M.%20Mosaad&volume=82&publication_year=2015&pages=283-306&pmid=26099424&doi=10.1111/sji.12329&)
