# CYP450 and drug efflux transporters polymorphism influence clinical outcomes of Thai osimertinib-treated non-small cell lung cancer patients

## Metadata
**Authors:** Teerapat Majam, Chonlaphat Sukasem, Thanyanan Reungwetwattana, Phichai Chansriwong, Chalirmporn Atasilp, Narumol Trachu, Thanaporn Thamrongjirapat, Rattanaporn Sukprasong, Jennis Meanwatthana
**Journal:** Frontiers in Pharmacology
**Date:** 2023 Nov 6
**DOI:** [10.3389/fphar.2023.1222435](https://doi.org/10.3389/fphar.2023.1222435)
**PMID:** 38026963
**PMCID:** PMC10657898
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10657898/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC10657898/pdf/fphar-14-1222435.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC10657898/pdf/fphar-14-1222435.pdf)

## Abstract

Background: Osimertinib has shown greater efficacy than standard epidermal growth factor receptor tyrosine kinase inhibitors (EGFR-TKIs) and fewer grade 3 or higher adverse drug reactions (ADRs) in patients with advanced non-small cell lung cancer (NSCLC) harboring epidermal growth factor receptor (EGFR) mutations. However, the clinical outcomes of osimertinib treatment vary depending on the patient’s ethnicity. Therefore, further research is necessary to evaluate the impact of single nucleotide polymorphisms (SNPs) in cytochrome P450 (CYP450) and drug transporters on the therapeutic outcomes and ADRs to osimertinib in Thai patients, to provide improved pharmacological treatments for cancer patients.

Methods: This retrospective and prospective cohort study enrolled 63 Thai patients with NSCLC treated with 80 mg of osimertinib once daily as monotherapy. Seventeen SNPs in candidate genes related to drug metabolism and transport pathways were analyzed in each patient. Chi-square or Fisher’s exact tests were used to evaluate the associations between SNPs and clinical outcomes, including ADR incidence and objective response rate (ORR). In addition, the correlation between the genotype and median time to treatment failure (TTF) or progression-free survival (PFS) was assessed using Kaplan-Meier analysis and a log-rank test.

Results: We identified six SNPs (rs2231142 and rs2622604 in ABCG2, rs762551 in CYP1A2, rs1057910 in CYP2C9, rs28371759 in CYP3A4, and CYP2A6 deletion polymorphism (CYP2A6*4)) that significantly increased the incidence of ADRs. In addition, we found two SNPs (rs2069514 in CYP1A2 and rs1057910 in CYP2C9) that significantly decreased the median TTF, and two SNPs (rs28399433 in CYP2A6 and rs1057910 in CYP2C9) that significantly decreased the median progression-free survival (PFS). Specifically, we found that one of these SNPs (rs1057910 in CYP2C9) influenced ADRs, TTF, and PFS. Additionally, SNPs in the CYP2A6 heterozygous variant (non4/*4) significantly increased ADR incidence, leading to a high frequency of dose reduction (27.0%).

Conclusion: Our study demonstrated significant SNPs associated with increased ADR incidence, decreased PFS, and decreased TTF in Thai patients with NSCLC treated with osimertinib. The CYP2C9 (*3) and CYP2A6 (*4) allele frequencies differed between ethnicities and were associated with an increased incidence of ADRs. These findings highlight the importance of considering genetic factors in NSCLC treatment and may facilitate personalized medicine approaches. Moreover, our study showed a higher incidence of ADRs than the previous trials, including FLAURA and AURA2, and a higher frequency of dose reduction than reported in the AURA 3 trial, possibly due to genetic differences among the study populations.

Keywords: non-small cell lung cancer, pharmacogenetics, single nucleotide polymorphisms (SNPs), osimertinib, drug-metabolizing enzymes, transporters

## 1 Introduction

Lung cancer is the leading cause of cancer-related deaths, accounting for approximately 18% ([Sung et al., 2021](#B31)). Approximately 80%–85% of these cases are classified as non-small cell lung cancer (NSCLC) ([Nicholson et al., 2022](#B23)). In Thai patients with NSCLC, *epidermal growth factor receptor* (*EGFR*) mutations were most commonly detected, accounting for 68% of cases ([Detarkom et al., 2018](#B9)). Activating *EGFR* mutations have been identified as predictive indicators of sensitivity to first- and second-generation EGFR tyrosine kinase inhibitors (TKIs). Acquired resistance develops 9–12 months after treatment initiation ([Westover et al., 2018](#B34)). One common mechanism underlying acquired resistance is the substitution of threonine with methionine at amino acid position 790 (T790M) in exon 20 of the *EGFR* gene, which accounts for 50%–60% of cases. This mutation impairs the binding of both first- and second-generation EGFR-TKIs by enhancing the ATP-binding affinity of the kinase domain of the EGFR mutant receptor, leading to treatment resistance ([Yu et al., 2013](#B36)).

Osimertinib is a third-generation EGFR-TKI that irreversibly binds to cysteine-797 at the ATP-binding site of the EGFR kinase domain ([Li et al., 2023](#B18)). It potently inhibits *EGFR* phosphorylation in the cases of exon 19 deletion and exon 21 L858R substitution, with IC50 values ranging 13–54 nmol/L. Additionally, in cell lines harboring *EGFR* T790M mutation, osimertinib demonstrates remarkable potency with an IC50 of less than 15 nmol/L. In addition, Osimertinib exhibits high selectivity for mutated EGFR receptors over wild-type EGFR (IC50: 480–1865 nmol/L) ([Remon et al., 2018](#B27)), resulting in less severe gastrointestinal and skin toxicities than those elicited by the first- or second-generation EGFR-TKIs. Additionally, osimertinib has improved overall survival in previously untreated advanced NSCLC patients with *EGFR* mutations compared with standard EGFR-TKIs ([Ramalingam et al., 2020](#B26)). Osimertinib is predominantly metabolized by CYP3A4, CYP2A6, CYP2C9, CYP3A5, and CYP2E1 enzymes, which account for 44.4%, 15.5%, 12.0%, 9.6%, and 3.0% of the metabolism, respectively ([Dickinson et al., 2016](#B10)). It produces at least two circulating metabolites, AZ5104 and AZ7550, accounting for 10% of the parent compound ([AstraZeneca, 2021](#B2)). AZ7550 has a potency and selectivity profile comparable to osimertinib, whereas AZ5104 has an 8-fold greater potency against EGFR mutations ([Han et al., 2021](#B14)).

In a previous study, variability in the steady-state area under the plasma drug concentration-time curve (AUCs) of AZ5104 between ethnic groups was observed, with a 10%–23% decrease in Asian *versus* Caucasian patients; however, the underlying reason remains unclear ([Brown et al., 2017](#B3)). CYP450 predominantly metabolizes Osimertinib and is a substrate for P-glycoprotein (P-gp), which is encoded by *ABCB1,* and breast cancer resistance protein (BCRP), which is encoded by *ABCG2* ([AstraZeneca, 2021](#B2)), which may lead to individual variations in plasma osimertinib concentrations due to genetic polymorphisms. A previous study also found a linear relationship between ADR development and osimertinib concentration, with an increased risk of rash, diarrhea, and QTc prolongation ([Brown et al., 2017](#B3)). Moreover, in Asian patients, the incidence of QTc prolongation was slightly higher than that reported in the FLAURA study ([Soria et al., 2018](#B30); [Cho et al., 2019](#B6)). Furthermore, a significant association was demonstrated between the AUC_0-24_ of osimertinib, SNPs rs1128503 in *ABCB1*, and SNPs rs2231137 in *ABCG2* with grade 2 or higher adverse events ([Ishikawa et al., 2023](#B17)). The previous studies on EGFR-TKIs found a correlation between SNPs rs762551 in *CYP1A2* and the severity of erlotinib-induced skin rash, along with the development of diarrhea and SNPs rs2470890 in *ABCB1* and rs776746 in *CYP3A5* ([Liao et al., 2020](#B19)). Additionally, SNPs rs2032582 in *ABCB1* were associated with afatinib-induced diarrhea ([Sogawa et al., 2020](#B29)), and reduced function of *CYP2D6* increased the risk of gefitinib-induced rash ([Suzumura et al., 2012](#B32)). However, the association between SNPs in *CYP450* and drug efflux transporters and efficacy outcomes remains unclear. We investigated seventeen SNPs in *CYP450* and drug efflux transporters that may alter the pharmacokinetic and pharmacodynamic profile of osimertinib, intending to advance personalized medicine approaches.

## 2 Materials and methods

### 2.1 Patients and study design

We recruited 63 NSCLC patients for this retrospective and prospective cohort study between June 2022 and January 2023 from the Division of Medical Oncology, Department of Medicine, Faculty of Medicine, Ramathibodi Hospital, Mahidol University, Bangkok, Thailand. The inclusion criteria were histologically confirmed *EGFR* mutation, monotherapy with 80 mg osimertinib once daily, age at diagnosis of >18 years, and normal baseline laboratory findings (complete blood count, renal function test, and liver function test). Patients were excluded if they received other medications that interfered with osimertinib drug levels or if toxicity was reported in the osimertinib-approved product monograph, such as itraconazole, rifampicin, simvastatin, and amiodarone ([AstraZeneca, 2021](#B2)). All patients provided written consent before enrolling in the study, and the study was approved by the Ethics Committee of Ramathibodi (ethics approval code: COA. MURA 2022/370).

### 2.2 Genotyping methods

Genomic DNA was isolated from an EDTA tube (6 mL) using an automatic DNA extraction system (MagNaPure; Roche, Mannheim, Germany). The concentration of DNA was approximately 5 ng/μL, and the A260/A280 ratio was in the range 1.70–2.10. A total of 17 SNPs were genotyped, namely, *ABCB1* rs1128503, *ABCG2* rs1871744, *ABCG2* rs2231142, *ABCG2* rs2231164, *ABCG2* rs2622604, *ABCG2* rs4148157, *CYP1A2* rs1871744, *CYP1A2* rs2069514, *CYP1A2* rs762551, *CYP2A6**4, *CYP2A6* rs28399433, *CYP2C9* rs1057910, *CYP2C9* rs1799853, *CYP3A4* rs28371759, *CYP3A5* rs10264272, *CYP3A5* rs776746, and *POR* rs1057868. Genotyping was conducted using real-time PCR ViiA7 (ABI, Foster City, CA, USA), following the manufacturer’s instructions. All samples were analyzed with positive and negative controls in 96-well plates to ensure the authenticity of the results. Genotyping of the candidate genes was performed using a TaqMan real-time PCR assay (ABI, Foster City, CA, USA).

### 2.3 Clinical endpoint assessment

We analyzed the association between the SNPs and clinical outcomes, including the incidence of adverse drug reactions (ADRs), median time to treatment failure (TTF), median progression-free survival (PFS), and objective response rate (ORR). ADRs were periodically assessed by each patient’s physician using the National Cancer Institute Common Terminology Criteria for Adverse Events version 5.0. The Naranjo algorithm was used to determine causality. TTF was the interval between initiating osimertinib treatment and a new locally directed or systemic treatment other than osimertinib monotherapy. PFS was defined as the time from osimertinib treatment initiation to disease progression or death from any cause. ORR was defined as the percentage of study patients who achieved a complete or partial response to treatment within a certain period, as assessed by each patient’s physician according to the Response Evaluation Criteria in Solid Tumors version 1.1.

### 2.4 Statistical analysis

Associations between SNPs and clinical outcomes, including ADRs and ORR, and those between patients’ baseline characteristics and clinical outcomes were evaluated using the appropriate chi-square test or Fisher’s exact test. Genetic polymorphisms were assessed for concordance with Hardy-Weinberg Equilibrium (HWE) using Fisher’s exact test. Linkage disequilibrium was explored using Haploview version 4.0. Univariate and multivariate logistic regression analyses were performed to identify the factors associated with clinical outcomes. The multivariate logistic regression analyses included all variables and all SNPs with a *p*-value of <0.1 from the univariate analysis, which is presented in [Supplementary Table S2](#s11). The correlation between TTF, PFS, and genotype was assessed using Kaplan-Meier analysis and the log-rank test. Statistical analysis was performed using SPSS version 23 for Windows, and significance was set at *p* < 0.05. The sample size calculation was based on a prospective longitudinal observational cohort study of 53 patients with advanced NSCLC receiving osimertinib therapy, which found a significant association between SNPs rs2231137 in *ABCG2* and grade ≥2 adverse events (*p* = 0.008). Of the *ABCG2* wild-type (G/G) patients, 22 (68.75%) had grade ≥2 adverse events, while all three (100%) of the *ABCG2* mutant-type (A/A) patients had grade ≥2 adverse events ([Ishikawa et al., 2023](#B17)). Therefore, based on the output of sample size calculation from n4Studies for a cohort study with binary outcomes ([Ngamjarus, 2016](#B22)), at least 58 patients were needed.

## 3 Results

### 3.1 Patient characteristics

The patient characteristics are presented in [Table 1](#T1). Sixty-three patients were included in the study: 20 (31.75%) men and 43 (68.25%) women, with a median age of 68 years (range 60–73). Forty-two patients (66.7%) had *EGFR* exon 19 deletion, nineteen patients (30.2%) had an L858R substitution, one (1.6%) had *EGFR* exon 20 insertion, and one (1.6%) had *EGFR* exon 18 G719X mutation. Forty-six patients (73%) harbored the *EGFR* T790M mutation. Patient characteristics were not significantly associated with the ADRs, TTF, PFS, or ORR. Ten patients (15.9%) received osimertinib as first-line therapy, whereas thirty-six patients (57.1%) and 17 patients (27.0%) received osimertinib as second- and later-line therapy, respectively. The previous systemic therapies included erlotinib (40.8%), gefitinib (35.5%), platinum-doublet chemotherapy (21.9%), and mobocertinib (1.8%). The line of treatment and previous systemic therapy were not significantly associated with the clinical outcomes (*p* > 0.05). All patients were administered 80 mg of osimertinib once daily. At the time of data cutoff, the mean follow-up duration was 18 (10–30) months, and 46 patients (73%) were still receiving osimertinib at the initial dosage. All patients had normal baseline laboratory test results, including those of complete blood count, liver function tests, and renal function tests.

### TABLE 1.

| Characteristics | Median [25th – 75th percentile] or number (%) |  |  |
| --- | --- | --- | --- |
| 1st line (n = 10) | 2nd line (n = 36) | ≥3rd line (n = 17) |  |
| Sex |   |   |   |
| Female | 8 (80.0) | 22 (61.1) | 13 (76.5) |
| Male | 2 (20.0) | 14 (38.9) | 4 (23.5) |
| Age (years) | 63.5 [56.0–71.5] | 70.5 [61.0–73.0] | 66.0 [60.0–72.0] |
| BMI (kg/m 2 ) | 22.5 [21.2–25.2] | 20.8 [19.3–23.8] | 21.73 [19.5–23.8] |
| ECOG PS |   |   |   |
| 0 | 6 (60.0) | 10 (27.8) | 6 (35.3) |
| 1 | 4 (40.0) | 16 (44.4) | 10 (58.8) |
| 2 | 0 (0) | 9 (25.0) | 1 (5.9) |
| 3 | 0 (0) | 1 (2.8) | 0 (0) |
| Smoking status |   |   |   |
| Never | 9 (90.0) | 28 (77.8) | 14 (82.3) |
| Former | 1 (10.0) | 6 (16.7) | 1 (5.9) |
| Current | 0 (0) | 0 (0) | 1 (5.9) |
| Passive | 0 (0) | 2 (5.5) | 1 (5.9) |
| Staging |   |   |   |
| IIIA | 0 (0) | 1 (2.8) | 0 (0) |
| IV | 10 (100.0) | 35 (97.2) | 17 (100) |
| Cerebral metastasis | 2 (20.0) | 4 (11.1) | 7 (41.2) |
| Type of EGFR mutation |   |   |   |
| Exon 18 G719x | 0 (0) | 1 (2.8) | 0 (0) |
| Exon 19 deletion | 6 (60.0) | 22 (61.1) | 14 (82.4) |
| Exon 20 insertion | 0 (0) | 1 (2.8) | 0 (0) |
| Exon 21 L858R | 4 (40.0) | 12 (33.3) | 3 (17.6) |
| Exon 20 T790M | 1 (10.0) a | 29 (80.6) b | 16 (94.1) b |
| Dose adjustment |   |   |   |
| Dose holding | 1 (10.0) | 3 (8.3) | 3 (17.6) |
| Dose reduction | 3 (30.0) | 9 (25.0) | 5 (29.4) |

Table 1 Caption: Baseline characteristics of the patients enrolled in the study.

### 3.2 Genotype frequencies

The genotype status of drug-metabolizing enzymes and transporters was determined for all 63 patients ([Table 2](#T2)). No genotype distribution deviated from the Hardy–Weinberg equation.

### TABLE 2.

| SNP-ID | Gene | Genotype | n | Identified frequency (%) | Allele | Allele frequency in the present study (%) | Allele frequency in the previous report (%) | HWE p-value |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
| rs1128503 | ABCB1 | G/G | 9 | 14.3 | G | 37.3 | 38.9 | 0.69 |
| G/A | 29 | 46.0 | A | 62.7 | 61.1 |  |  |  |
| A/A | 25 | 39.7 |   |   |   |  |  |  |
| rs2231142 | ABCG2 | C/C | 31 | 49.2 | C | 70.6 | 70.9 | 0.99 |
| C/A | 27 | 42.9 | A | 29.4 | 29.1 |  |  |  |
| A/A | 5 | 7.9 |   |   |   |  |  |  |
| rs2231164 | ABCG2 | T/T | 18 | 28.6 | T | 52.4 | 50.4 | 0.75 |
| T/C | 30 | 47.6 | C | 47.6 | 49.6 |  |  |  |
| C/C | 15 | 23.8 |   |   |   |  |  |  |
| rs2622604 | ABCG2 | C/C | 38 | 60.3 | C | 76.2 | 81.4 | 0.18 |
| C/T | 20 | 31.7 | T | 23.8 | 18.6 |  |  |  |
| T/T | 5 | 7.9 |   |   |   |  |  |  |
| rs4148157 | ABCG2 | G/G | 38 | 60.3 | G | 77.8 | 75.0 | 0.49 |
| G/A | 22 | 34.9 | A | 22.2 | 25.0 |  |  |  |
| A/A | 3 | 4.8 |   |   |   |  |  |  |
| rs1871744 | ABCG2 | T/T | 27 | 42.9 | T | 65.9 | 72.5 | 0.15 |
| T/C | 29 | 46.0 | C | 34.1 | 27.5 |  |  |  |
| C/C | 7 | 11.1 |   |   |   |  |  |  |
| rs2069514 | CYP1A2 | G/G | 31 | 49.2 | G | 70.6 | 73.0 | 0.65 |
| G/A | 27 | 42.9 | A | 29.4 | 27.0 |  |  |  |
| A/A | 5 | 7.9 |   |   |   |  |  |  |
| rs762551 | CYP1A2 | C/C | 3 | 4.8 | C | 31.0 | 35.4 | 0.36 |
| C/A | 33 | 52.4 | A | 69.0 | 64.6 |  |  |  |
| A/A | 27 | 42.9 |   |   |   |  |  |  |

Table 2 Caption: Genotype and allele frequencies of SNPs were compared between the present study and a previous report ( PharmGKB, 2021).

### 3.3 SNPs associated with osimertinib-induced ADRs

[Table 3](#T3) shows the association between genetic polymorphisms and the overall ADR incidence. We found that SNP rs1057910 in *CYP2C9* was significantly associated with an increased incidence of grade 3 ADRs (*p* = 0.003). Additionally, we identified several SNPs that were significantly associated with an increased incidence of specific ADRs, including rs2622604 in *ABCG2* mutant-type (T/T) and *CYP2A6* heterozygous variant (non*4/*4), which were significantly associated with diarrhea (*p* = 0.011 and *p* = 0.046, respectively). Furthermore, SNPs rs2231142 in *ABCG2* mutant-type (A/A), rs1057910 in *CYP2C9* mutant-type (C/C), rs28371759 in *CYP3A4* heterozygous variant (A/G), and rs762551 in *CYP1A2* wild-type (C/C) were identified. These SNPs were associated with myalgia, grade 3 acneiform rash, QTc prolongation, and bullous dermatitis, respectively (*p* = 0.007, *p* = 0.012, *p* = 0.001, and *p* = 0.006, respectively; [Table 4](#T4)).

### TABLE 3.

| SNP-ID | Gene | Genotype | n | Identified frequency (%) | Allele | Allele frequency in the present study (%) | Allele frequency in the previous report (%) | HWE p-value |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
| CYP2A6*4 | CYP2A6 | non*4/non*4 | 54 | 85.7 | non*4 | 92.9 | 95.3 | 0.28 |
| non*4/*4 | 9 | 14.3 | *4 | 7.1 | 4.7 |  |  |  |
| *4/*4 | 0 | 0 |   |   |   |  |  |  |
| rs28399433 | CYP2A6 | A/A | 43 | 68.3 | A | 81.7 | 86.4 | 0.19 |
| A/C | 17 | 27.0 | C | 18.3 | 13.6 |  |  |  |
| C/C | 3 | 4.8 |   |   |   |  |  |  |
| rs1799853 | CYP2C9 | C/C | 63 | 100 | C | 100 | 99.8 | 0.95 |
| C/T | 0 | 0 | T | 0 | 0.2 |  |  |  |
| T/T | 0 | 0 |   |   |   |  |  |  |
| rs1057910 | CYP2C9 | A/A | 57 | 90.5 | A | 94.4 | 95.6 | 0.44 |
| A/C | 5 | 7.9 | C | 5.6 | 4.4 |  |  |  |
| C/C | 1 | 1.6 |   |   |   |  |  |  |
| rs28371759 | CYP3A4 | A/A | 60 | 95.2 | A | 97.6 | 98.5 | 0.17 |
| A/G | 3 | 4.8 | G | 2.4 | 1.5 |  |  |  |
| G/G | 0 | 0 |   |   |   |  |  |  |
| rs776746 | CYP3A5 | A/A | 5 | 7.9 | A | 32.5 | 27.8 | 0.27 |
| A/G | 31 | 49.2 | G | 67.5 | 72.2 |  |  |  |
| G/G | 27 | 42.9 |   |   |   |  |  |  |
| rs10264272 | CYP3A5 | G/G | 63 | 100 | G | 100 | 100 | 1.00 |
| G/A | 0 | 0 | A | 0 | 0 |  |  |  |
| A/A | 0 | 0 |   |   |   |  |  |  |
| rs41303343 | CYP3A5 | −/− | 63 | 100 | no-insT | 100 | 100 | 1.00 |
| -/T | 0 | 0 | insT | 0 | 0 |  |  |  |
| T/T | 0 | 0 |   |   |   |  |  |  |
| rs1057868 | POR | C/C | 26 | 41.3 | C | 65.1 | 59.9 | 0.29 |
| C/T | 30 | 47.6 | T | 34.9 | 40.1 |  |  |  |
| T/T | 7 | 11.1 |   |   |   |  |  |  |

Table 3 Caption: Genotype and allele frequencies of SNPs were compared between the present study and a previous report ( PharmGKB, 2021) (cont.).

### TABLE 4.

| SNP-ID | Gene | Genotype | Incidence of specific ADRs [n/N (%)] | Specific ADRs | p-Value |
| --- | --- | --- | --- | --- | --- |
| rs2231142 | ABCG2 | C/C | 0/31 (0) | Myalgia | 0.007 a |
| C/A | 0/27 (0) |  |  |  |  |
| A/A | 1/5 (20.0) |  |  |  |  |
| rs2622604 | ABCG2 | C/C | 7/38 (18.4) | Diarrhea | 0.011 a |
| C/T | 6/20 (30.0) |  |  |  |  |
| T/T | 4/5 (80.0) |  |  |  |  |
| rs762551 | CYP1A2 | C/C | 1/3 (33.3) | Bullous dermatitis | 0.006 a |
| C/A | 0/33 (0) |  |  |  |  |
| A/A | 0/27 (0) |  |  |  |  |
| CYP2A6 a 4 | CYP2A6 | non a 4/non a 4 | 12/54 (22.2) | Diarrhea | 0.046 a |
| non a 4/ a 4 | 5/9 (55.6) |  |  |  |  |
| rs1057910 | CYP2C9 | A/A | 0/57 (0) | Acneiform rash grade 3 | 0.012 a |
| A/C | 0/5 (0) |  |  |  |  |
| C/C | 1/1 (100) |  |  |  |  |
| rs28371759 | CYP3A4 | A/A | 11/60 (18.3) | QTc prolongation (%) | 0.001 a |
| A/G | 3/3 (100.0) |  |  |  |  |

Table 4 Caption: Summary of significant association between SNPs and incidence of specific ADRs.

### 3.4 SNPs associated with osimertinib efficacy outcomes

After initiating osimertinib, 31 patients (49.2%) had an objective response, including one patient (1.6%) with a complete response and 30 patients (47.6%) with a partial response. In the non-response group, 32 patients (50.8%) did not respond, including 30 patients (47.6%) with stable disease and two patients (3.2%) with progressive disease.

The median TTF was 19 (10.3–29.0) months. In addition, SNPs rs2069514 in *CYP1A2* mutant-type (A/A) and rs1057910 in *CYP2C9* heterozygous variant (A/C) were significantly associated with decreased TTF with *p* < 0.001 and 0.041, respectively. These findings are presented in [Table 5](#T5) and [Figure 1](#F1).

### TABLE 5.

| SNP-ID | Gene | Genotype | All grade ADRs [n/N (%)] |  | p-Value | Severity of ADRs [n/N (%)] |  | p-Value | TTF (months (95% CI)) | p-Value | PFS (months (95% CI)) | p-Value |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| ADRs | Non-ADRs | Grade 1, 2 | Grade 3 |  |  |  |  |  |  |  |  |  |
| rs1128503 | ABCB1 | G/G | 9/9 (100) | 0/9 (0) | 0.150 | 7/9 (77.8) | 2/9 (22.2) | 0.682 | 34.0 (34.0–34.0) | 0.216 | 42.0 (42.0–42.0) | 0.283 |
| G/A | 28/29 (96.6)21/25 | 1/29 (3.4) | 25/28 (89.3) | 3/28 (10.7) | 21.9 (14.3–29.6) | 30.3 (24.7–35.8) |  |  |  |  |  |  |
| A/A | (84.0) | 4/25 (16.0) | 18/21 (85.7) | 3/21 (14.3) | 16.6 (11.1–22.1) | 30.9 (21.7–40.0) |  |  |  |  |  |  |
| rs2231142 | ABCG2 | C/C | 29/31 (93.5) | 2/31 (6.5) | 0.557 | 24/29 (82.8) | 5/29 (17.2) | 0.607 | 23.9 (17.4–30.3) | 0.420 | 39.0 (31.4–46.6) | 0.081 |
| C/A | 25/27 (92.6) | 2/27 (7.4) | 22/25 (88.0) | 3/25 (12.0) | 16.7 (9.0–24.4) | 26.7 (20.3–33.2) |  |  |  |  |  |  |
| A/A | 4/5 (80.0) | 1/5 (20.0) | 4/4 (100) | 0/4 (0) | 21.5 (0.0–46.0) | 42.0 (42.0–42.0) |  |  |  |  |  |  |
| rs2231164 | ABCG2 | T/T | 17/18 (94.4) | 1/18 (5.6) | 0.669 | 12/17 (70.6) | 5/17 (29.4) | 0.055 | 26.6 (18.3–34.9) | 0.128 | 37.5 (26.7–48.3) | 0.287 |
| T/C | 28/30 (93.3) | 2/30 (6.7) | 25/28 (89.3) | 3/28 (10.7) | 15.4 (9.2–21.6) | 30.0 (22.0–34.0) |  |  |  |  |  |  |
| C/C | 13/15 (86.7) | 2/15 (13.3) | 13/13 (100.0) | 0/13 (0) | 24.2 (12.8–35.6) | 39.0 (31.8–46.2) |  |  |  |  |  |  |
| rs2622604 | ABCG2 | C/C | 34/38 (89.5) | 4/38 (10.5) | 0.602 | 29/34 (85.3) | 5/34 (14.7) | 0.837 | 19.6 (13.3–26.0) | 0.825 | 30.0 (24.4–35.6) | 0.828 |
| C/T | 19/20 (95.0) | 1/20 (5.0) | 17/19 (89.5) | 2/19 (10.5) | 23.3 (10.6–35.9) | 42.0 (42.0–42.0) |  |  |  |  |  |  |
| T/T | 5/5 (100) | 0/5 (0) | 4/5 (80.0) | 1/5 (20.0) | 20.0 (14.1–25.9) | NE |  |  |  |  |  |  |
| rs4148157 | ABCG2 | G/G | 36/38 (94.7) | 2/38 (5.3) | 0.447 | 29/36 (80.6) | 7/36 (19.4) | 0.271 | 20.9 (15.4–25.4) | 0.930 | 35.7 (28.3–43.0) | 0.529 |
| G/A | 19/22 (86.4) | 3/22 (13.6) | 18/19 (94.7) | 1/19 (5.3) | 19.0 (8.3–29.7) | 30.5 (25.2–35.9) |  |  |  |  |  |  |
| A/A | 3/3 (100) | 0/3 (0) | 3/3 (100) | 0/3 (0) | 21.5 (0.0–46.0) | 42.0 (42.0–42.0) |  |  |  |  |  |  |
| rs1871744 | ABCG2 | T/T | 24/27 (88.9) | 3/27 (11.1) | 0.459 | 20/24 (83.3) | 4/24 (16.7) | 0.806 | 16.8 (9.2–24.4) | 0.734 | 35.7 (28.6–42.8) | 0.540 |
| T/C | 28/29 (96.6) | 1/29 (3.4) | 25/28 (89.3) | 3/28 (10.7) | 23.3 (15.9–30.6) | 29.1 (23.3–34.9) |  |  |  |  |  |  |
| C/C | 6/7 (85.7) | 1/7 (14.3) | 5/6 (83.3) | 1/6 (16.7) | 27.0 (19.2–34.8) | 42.0 (42.0–42.0) |  |  |  |  |  |  |
| rs2069514 | CYP1A2 | G/G | 28/31 (90.3) | 3/31 (9.7) | 0.752 | 23/28 (82.1) | 5/28 (17.9) | 0.534 | 24.0 (18.6–29.4) | <0.001 a | 38.4 (33.1–43.7) | 0.166 |
| G/A | 25/27 (92.6) | 2/27 (7.4) | 22/25 (88.0) | 3/25 (12.0) | 16.0 (11.8–20.2) | 26.3 (21.4–31.2) |  |  |  |  |  |  |
| A/A | 5/5 (100) | 0/5 (0) | 5/5 (100) | 0/5 (0) | 3.0 (3.0–3.0) | 22.0 (9.1–34.8) |  |  |  |  |  |  |

Table 5 Caption: Association between SNPs and clinical outcomes.

### FIGURE 1.

![FIGURE 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f216/10657898/dd1f16332704/fphar-14-1222435-g001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=10657898_fphar-14-1222435-g001.jpg)

Kaplan–Meier estimates and log-rank tests for median time to treatment failure (TTF) associated with SNPs. (A) Association between SNPs rs2069514 in CYP1A2 and TTF; (B) association between SNPs rs1057910 in CYP2C9 and TTF.

The median PFS was significantly decreased in patients with SNPs rs28399433 in *CYP2A6* mutant-type (C/C) and rs1057910 in *CYP2C9* heterozygous variant (A/C), with *p* = 0.023 and <0.001, respectively. In addition, among patients who received osimertinib as second-line therapy (*N* = 36), SNPs rs28399433 in *CYP2A6* mutant-type (C/C) and rs1057910 in *CYP2C9* heterozygous variant (A/C) were also significantly associated with decreased PFS, with *p* values of 0.001 and 0.010, respectively ([Table 6](#T6); [Figure 2](#F2)).

### TABLE 6.

| SNP-ID | Gene | Genotype | All grade ADRs [n/N (%)] |  | p-Value | Severity of ADRs [n/N (%)] |  | p-Value | TTF (months (95% CI)) | p-Value | PFS (months (95% CI)) | p-Value |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| ADRs | Non-ADRs | Grade 1, 2 | Grade 3 |  |  |  |  |  |  |  |  |  |
| rs762551 | CYP1A2 | C/C | 3/3 (100) | 0/3 (0) | 0.425 | 2/3 (66.7) | 1/3 (33.3) | 0.584 | NE | 0.987 | NE | 0.539 |
| C/A | 29/33 (87.9) | 4/33 (12.1) | 25/29 (86.2) | 4/29 (13.8) | 20.5 (15.2–25.7) | 30.0 (24.6–35.4) |  |  |  |  |  |  |
| A/A | 26/27 (96.3) | 1/27 (3.7) | 23/26 (88.5) | 3/26 (11.5) | 20.0 (2.2–37.8) | 42.0 (42.0–42.0) |  |  |  |  |  |  |
| CYP2A6*4 | CYP2A6 | non*4/non*4 | 49/54 (90.7) | 5/54 (9.3) | 0.450 | 42/49 (85.7) | 7/49 (14.3) | 0.639 | 21.2 (16.1–26.3) | 0.685 | 34.3 (16.9–52.7) | 0.847 |
| non*4/*4 | 9/9 (100) | 0/9 (0) | 8/9 (88.9) | 1/9 (11.1) | 15.7 (0.0–33.9) | 34.8 (28.5–40.1) |  |  |  |  |  |  |
| rs28399433 | CYP2A6 | A/A | 40/43 (93.0) | 3/43 (7.0) | 0.721 | 34/40 (85.0) | 6/40 (15.0) | 0.767 | 18.9 (13.1–26.7) | 0.181 | 33.3 (27.1–39.6) | <0.001* |
| A/C | 15/17 (88.2) | 2/17 (11.8) | 13/15 (86.7) | 2/15 (13.3) | 24.3 (15.1–33.6) | 42.0 (42.0–42.0) |  |  |  |  |  |  |
| C/C | 3/3 (100) | 0/3 (0) | 3/3 (100) | 0/3 (0) | 12.0 (0.0–27.7) | 6.0 (6.0–6.0) |  |  |  |  |  |  |
| rs1057910 | CYP2C9 | A/A | 53/57 (93.0) | 4/57 (7.0) | 0.563 | 48/53 (90.6) | 5/53 (9.4) | 0.003* | 23.0 (13.5–32.4) | 0.041* | 42.0 (42.0–42.0) | 0.010* |
| A/C | 4/5 (80.0) | 1/5 (20.0) | 2/4 (50.0) | 2/4 (50.0) | 14.0 (0.0–31.6) | 24.0 (9.3–38.7) |  |  |  |  |  |  |
| C/C | 1/1 (100) | 0/1 (0) | 0/1 (0) | 1/1 (100) | NE | NE |  |  |  |  |  |  |
| rs28371759 | CYP3A4 | A/A | 55/60 (91.7) | 5/60 (8.3) | 0.777 | 48/55 (87.3) | 7/55 (12.7) | 0.365 | 20.4 (15.4–25.4) | NE | 34.8 (29.8–39.9) | NE |
| A/G | 3/3 (100) | 0/0 (0) | 2/3 (66.7) | 1/3 (33.3) | NE | NE |  |  |  |  |  |  |
| rs776746 | CYP3A5 | A/A | 4/5 (80.0) | 1/5 (20.0) | 0.577 | 4/4 (100) | 0/4 (0) | 0.690 | 27.0 (23.1–30.9) | 0.835 | 30.0 (30.0–30.0) | 0.356 |
| A/G | 29/31 (93.5) | 2/31 (6.5) | 25/29 (86.2) | 4/29 (13.8) | 22.5 (14.3–30.7) | 37.7 (31.7–43.6) |  |  |  |  |  |  |
| G/G | 25/27 (92.6) | 2/27 (7.4) | 21/25 (84.0) | 4/25 (16.0) | 18.3 (11.1–25.4) | 28.6 (21.4–35.9) |  |  |  |  |  |  |
| rs1057868 | POR | C/C | 25/26 (96.2) | 1/26 (3.8) | 0.302 | 21/25 (84.0) | 4/25 (16.0) | 0.528 | 15.8 (6.4–25.2) | 0.259 | 30.0 (19.8–40.2) | 0.464 |
| C/T | 26/30 (86.7) | 4/30 (13.3) | 22/26 (84.6) | 4/26 (15.4) | 21.9 (16.1–27.7) | 42.0 (42.0–42.0) |  |  |  |  |  |  |
| T/T | 7/7 (100) | 0/7 (0) | 7/7 (100) | 0/7 (0) | NE | NE |  |  |  |  |  |  |

Table 6 Caption: Association between SNPs and clinical outcomes (cont.).

### FIGURE 2.

![FIGURE 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f216/10657898/5183311b8492/fphar-14-1222435-g002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=10657898_fphar-14-1222435-g002.jpg)

Kaplan–Meier estimates and log-rank tests for median progression-free survival (PFS) associated with SNPs among patients who received osimertinib as second-line therapy. (A) association between SNPs rs28399433 in CYP2A6 and PFS; and (B) association between SNPs rs1057910 in CYP2C9 and PFS.

### 3.5 Incidence of ADRs

The incidence of ADRs is shown in [Table 7](#T7). Six patients (9.5%) required a dose hold, and seventeen patients (27.0%) required a dose reduction from the standard prescription owing to ADRs, which included diarrhea in four patients (6.3%), acneiform rash in three patients (4.8%), neutropenia in three patients (4.8%), thrombocytopenia in one patient (1.6%), bullous dermatitis in one patient (1.6%), myositis in one patient (1.6%), transaminitis in one patient (1.6%), QTc prolongation in one patient (1.6%), mucositis in one patient (1.6%), and alopecia in one patient (1.6%). Dose reduction included reducing the dose to 80 mg every other day in 10 patients (15.9%), 80 mg three times a week in 5 patients (7.9%), 80 mg five times a week in 1 patient (1.6%), and 40 mg once daily in 1 patient (1.6%).

### TABLE 7.

| Adverse drug reactions | All grades (%) | Grade 1 (%) | Grade 2 (%) | Grade 3 (%) |
| --- | --- | --- | --- | --- |
| Acneiform rash | 29 (46.0) | 15 (23.8) | 13 (20.6) | 1 (1.6) |
| Dry skin | 23 (36.5) | 20 (31.7) | 3 (4.8) | 0 (0) |
| Diarrhea | 17 (27.0) | 11 (17.4) | 3 (4.8) | 3 (4.8) |
| QTc prolongation | 14 (22.2) | 7 (11.1) | 3 (4.8) | 4 (6.3) |
| Dry eye | 12 (19.0) | 11 (17.5) | 1 (1.6) | 0 (0) |
| Paronychia | 8 (12.7) | 6 (9.5) | 2 (3.2) | 0 (0) |
| Thrombocytopenia | 8 (12.7) | 7 (11.1) | 1 (1.6) | 0 (0) |
| Mucositis | 7 (11.1) | 5 (7.9) | 2 (3.2) | 0 (0) |
| Transaminitis | 5 (7.9) | 4 (6.3) | 1 (1.6) | 0 (0) |
| Anemia | 5 (7.9) | 4 (6.3) | 1 (1.6) | 0 (0) |
| Neutropenia | 4 (6.3) | 0 (0) | 4 (6.3) | 0 (0) |
| Urticaria | 2 (3.2) | 1 (1.6) | 1 (1.6) | 0 (0) |
| Alopecia | 1 (1.6) | 0 (0) | 1 (1.6) | 0 (0) |
| Bullous dermatitis | 1 (1.6) | 0 (0) | 1 (1.6) | 0 (0) |
| Papulopustular rash | 1 (1.6) | 0 (0) | 0 (0) | 1 (1.6) |
| Paroxysmal AF | 1 (1.6) | 0 (0) | 1 (1.6) | 0 (0) |
| Nausea/vomiting | 1 (1.6) | 1 (1.6) | 0 (0) | 0 (0) |
| Anorexia | 1 (1.6) | 1 (1.6) | 0 (0) | 0 (0) |
| Myalgia | 1 (1.6) | 1 (1.6) | 0 (0) | 0 (0) |
| Myositis | 1 (1.6) | 0 (0) | 0 (0) | 1 (1.6) |

Table 7 Caption: Incidence of adverse drug reactions.

## 4 Discussion

A recent population pharmacokinetic study showed a linear correlation between exposure to osimertinib, measured using the AUCs of the parent compound and two active metabolites (AZ5104 and AZ7550), and the incidence of ADRs ([Brown et al., 2017](#B3)). AZ5104, in particular, demonstrated an 8-fold greater potency against EGFR mutations ([Han et al., 2021](#B14)). While AZ7550 and AZ5104 are present in approximately 10% of the parent compound ([Brown et al., 2017](#B3)), a decrease in AZ5104 AUCs of 10%–23% in Asian patients compared with that of Caucasian patients may influence clinical outcomes. In addition, a study in Asian populations reported a significant association between SNPs rs1128503 in *ABCB1* and rs2231137 in *ABCG2* and osimertinib-induced grade ≥2 adverse events ([Ishikawa et al., 2023](#B17)). However, the association between SNPs and the efficacy of osimertinib remains unclear, and the specific SNPs that influence the pharmacokinetics and clinical effects of osimertinib in NSCLC remain unknown.

This is the first study to analyze a large number of genetic polymorphisms in candidate genes involved in the osimertinib pharmacokinetic pathway to enable the assessment of both efficacy and safety endpoints. All allele frequencies were consistent with the Hardy–Weinberg equilibrium ([PharmGKB, 2021](#B25)). In the present study, the frequency of dose reduction (27.0%) was higher than that reported in the AURA 3 trial (16.5%), and the incidence of ADRs was higher than that in the FLAURA, AURA2, and AURA3 trials ([Goss et al., 2016](#B13); [Mok et al., 2017](#B21); [Soria et al., 2018](#B30)). This may be because of genetic differences between patient populations; Asian patients who were administered osimertinib in these studies accounted for only 62%, 63%, and 65% of the patients, respectively. The frequency of the *ABCG2* rs2231164 (C) allele, known as a loss-of-function variant, was 23.72% in the South Asian population and only 12.12% in the European population ([Whirl-Carrillo et al., 2021](#B35)). Furthermore, the frequency of the *CYP2A6**4 allele, a slower metabolizer, also differed significantly between ethnic groups ([Pang et al., 2015](#B24)), potentially leading to higher plasma concentrations of osimertinib in the Thai population. These genetic differences may have contributed to the observed differences in dose reductions and ADRs between our study and previous clinical trials. Additionally, we found that the ORR in our study was 49.2%, which was lower than that reported in the AURA 3 trial (71%) ([Mok et al., 2017](#B21)). This difference in response rates may be because of the differences in patient populations between the two studies. In the AURA 3 trial, patients received osimertinib as second-line therapy and 96% of cases and later-line therapy in 4%. In contrast, in our study, 57.1% of patients received osimertinib as second-line therapy, and 27.0% received osimertinib as later-line therapy.

In our study, we identified six SNPs that were significantly associated with the incidence of ADRs, namely, rs2231142 in *ABCG2* mutant-type (A/A), rs2622604 in *ABCG2* mutant-type (T/T), *CYP2A6* heterozygous variant (non*4/*4), rs1057910 in *CYP2C9* mutant-type (C/C), rs28371759 in *CYP3A4* heterozygous variant (A/G) and rs762551 in *CYP1A2* wild-type (C/C). These findings are consistent with previous studies reporting significant associations between genetic polymorphisms and ADR risk. For example, rs2231142 in the *ABCG2* mutant-type (A/A) is significantly associated with sunitinib-induced severe thrombocytopenia ([Low et al., 2016](#B20)), whereas rs2622604 in the *ABCG2* mutant-type (T/T) is significantly associated with irinotecan-induced severe myelosuppression ([Cha et al., 2009](#B5)). Similarly, the *CYP2A6* (*4) allele is significantly associated with letrozole-induced ADRs ([Desta et al., 2011](#B8)), whereas the *CYP3A4* *18 (G) allele is significantly associated with tacrolimus-induced ADRs ([Bruckmueller et al., 2015](#B4)). Notably, in contrast to other genes, rs762551 in the *CYP1A2* wild-type (C/C) was significantly related to osimertinib-induced ADRs. This result may be attributable to the higher enzyme activity observed in the presence of an inducer, such as smoking or heavy coffee consumption, which leads to higher enzyme activity in the *CYP1A2* mutant-type (A/A) ([Djordjevic et al., 2010](#B11); [Wang et al., 2012](#B33)), and a lower incidence of ADRs was observed in this variant. With regard to efficacy outcomes, we identified two SNPs (rs2069514 in *CYP1A2* and rs1057910 in *CYP2C9*) that were significantly associated with the median TTF and two SNPs (rs28399433 in *CYP2A6* and rs1057910 in *CYP2C9*) that were significantly associated with the median PFS. Notably, one of these SNPs (rs1057910 in *CYP2C9*) was significantly associated with ADRs, TTF, and PFS.

The SNPs in *CYP450* and the drug efflux transporters discussed above were significantly associated with ADRs, TTF, and PFS. Because osimertinib is a substrate of CYP450, ABCB1, and ABCG2 ([AstraZeneca, 2021](#B2)), polymorphisms in these genes may affect the distribution and pharmacokinetics of this drug. Therefore, we hypothesized that mutations causing the decreased function of CYP450, ABCB1, and ABCG2 may affect the tissue distribution and accumulation of osimertinib. An *in vivo* study suggested that ABCB1 and ABCG2 are involved in the tissue accumulation of other TKIs ([Al-Shammari et al., 2019](#B1)). As the active metabolite of osimertinib (AZ5104) is more potent than the parent compound against EGFR mutations ([Remon et al., 2018](#B27)) and accounts for approximately 10% of the parent compound ([Brown et al., 2017](#B3)), the accumulation of osimertinib may lead to a significantly increased incidence of ADRs but significantly decreased TTF and PFS. For example, SNP rs1057910 (C), located in the *CYP2C9* gene, typically encodes the amino acid leucine at position 359, and the resulting allele is also known as *CYP2C9**3, which is a decreased function variant ([VCV000008408.12 - ClinVar - NCBI, 2013](#B7)). This variant may lead to poor metabolism of osimertinib in patients carrying the *CYP2C9**3 allele, which increases the risk of osimertinib-induced ADRs due to higher osimertinib exposure but also decreases survival outcomes due to lower exposure to its active metabolite (AZ5104).

These findings are similar to those of a previous study on the osimertinib exposure-response relationship, which found that the mortality rate was significantly higher in the high osimertinib drug level group ([Rodier et al., 2022](#B28)). Additionally, the probability of developing rashes, diarrhea, or QTc prolongation increased with exposure, and a linear relationship between adverse event development and osimertinib levels was identified ([Brown et al., 2017](#B3)). The mechanism that supports EGFR-TKI-induced ADRs is the inhibition of EGFR1 and EGFR2 (HER2) signaling, leading to a reduction in growth and impaired healing of the epithelium where EGFR is expressed. This subsequently causes alterations in keratinocyte proliferation and differentiation, reduced growth, impaired healing of the intestinal epithelium, and alterations in myocyte growth ([Giovannini et al., 2009](#B12); [Hirsh et al., 2014](#B15); [Ikebe et al., 2020](#B16)).

Our study has several limitations. First, the sample size was small, and some clinical data were retrospectively collected from medical records. Nevertheless, no genotype distribution deviated from the Hardy–Weinberg equation ([PharmGKB, 2021](#B25)), and the clinical data were confirmed by the patient’s physician. Second, our inability to control for other confounding factors, such as compliance with osimertinib treatment caused by home oral medication but we had oncology pharmacist pill count measures of compliance at every visit, and all patients maintained a 100% compliance rate. Additionally, there were no observed drug interactions that influenced osimertinib drug levels. However, we did not restrict patient coffee consumption, which may have an impact on higher enzyme activity in the SNP rs762551 in *CYP1A2* mutant-type (A/A). This enzyme was inducible by heavy coffee consumption and has been associated with a lower incidence of ADRs ([Djordjevic et al., 2010](#B11); [Wang et al., 2012](#B33)). Finally, a pharmacokinetic analysis was not included in this study. However, the association between SNPs and clinical outcomes was consistent with that in previous reports.

In conclusion, our study identified significant SNPs associated with increased ADRs incidence, decreased TTF, and decreased PFS in Thai patients with NSCLC treated with osimertinib. The findings can potentially guide treatment decisions and help optimize individualized therapy for patients with NSCLC harboring EGFR mutations. However, more extensive studies with analysis of osimertinib and its active metabolite drug levels must be conducted to confirm these findings.

## Acknowledgments

The success of this study required the assistance and contributions of many individuals. We would like to express our sincere gratitude to all the patients who agreed to participate in this study and the staff who provided invaluable support throughout the research process.

## Funding Statement

The Doctor Kasem Pangsrivongse Foundation (Thailand) provided a research grant (02-002-65) to fund this study, which was established to support professional development in public healthcare. The authors were solely responsible for the preparation of the study protocol, data analysis, and writing and submission of the manuscript, ensuring the independence and objectivity of the study’s results.

## Data availability statement

The original contributions presented in the study are included in the article/[Supplementary Materials](#s11), further inquiries can be directed to the corresponding author.

## Ethics statement

The studies involving humans were approved by The Ethics Committee of Ramathibodi (ethics approval code: COA. MURA 2022/370). The studies were conducted in accordance with the local legislation and institutional requirements. The participants provided their written informed consent to participate in this study.

## Author contributions

Conception and design: TM, JM, TR, CS and CA; Sample process: TR, CS, JM, NT, TM, TT and PC; Analysis: TR, CS, TM, NT, JM, TT RS and CA; Interpretation of data: TR, CS, JM, TM, TT, NT and PC; Conclusion and suggestion: TR, CS and JM; Drafting and Editing manuscript: TM, JM, TR and CS. Supervising project: TM, JM, TR and CS. TM was the first author, TR was the essentially intellectual contributor, and JM was the corresponding author. All authors contributed to the article and approved the submitted version.

## Conflict of interest

The authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.

## Publisher’s note

All claims expressed in this article are solely those of the authors and do not necessarily represent those of their affiliated organizations, or those of the publisher, the editors and the reviewers. Any product that may be evaluated in this article, or claim that may be made by its manufacturer, is not guaranteed or endorsed by the publisher.

## Supplementary material

The Supplementary Material for this article can be found online at: [https://www.frontiersin.org/articles/10.3389/fphar.2023.1222435/full#supplementary-material](https://www.frontiersin.org/articles/10.3389/fphar.2023.1222435/full#supplementary-material)

## Abbreviations

ADR, Adverse drug reactions; BCRP, Breast cancer resistance protein; EGFR, Epidermal growth factor receptor; NSCLC, Non-small cell lung cancer; ORR, Objective response rate; PFS, Progression-free survival; SNP, Single nucleotide polymorphisms; TKI, Tyrosine kinase inhibitors; TTF, Time to treatment failure.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

The original contributions presented in the study are included in the article/[Supplementary Materials](#s11), further inquiries can be directed to the corresponding author.

### Supplementary Materials

### Data Availability Statement

The original contributions presented in the study are included in the article/[Supplementary Materials](#s11), further inquiries can be directed to the corresponding author.

## References

1. Al-Shammari A. H., Masuo Y., Fujita K. I., Yoshikawa Y., Nakamichi N., Kubota Y., et al. (2019). Influx and efflux transporters contribute to the increased dermal exposure to active metabolite of regorafenib after repeated oral administration in mice. J. Pharm. Sci. 108 (6), 2173–2179. 10.1016/j.xphs.2019.01.018  [DOI](https://doi.org/10.1016/j.xphs.2019.01.018) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30685396/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Pharm.%20Sci.&title=Influx%20and%20efflux%20transporters%20contribute%20to%20the%20increased%20dermal%20exposure%20to%20active%20metabolite%20of%20regorafenib%20after%20repeated%20oral%20administration%20in%20mice&author=A.%20H.%20Al-Shammari&author=Y.%20Masuo&author=K.%20I.%20Fujita&author=Y.%20Yoshikawa&author=N.%20Nakamichi&volume=108&issue=6&publication_year=2019&pages=2173-2179&pmid=30685396&doi=10.1016/j.xphs.2019.01.018&)

2. AstraZeneca (2021). TAGRISSO (osimertinib) [Product monograph]. Mississauga, Ontario: AstraZeneca Canada Inc. Available from: https://www.astrazeneca.ca/content/dam/az-ca/downloads/productinformation/tagrisso-product-monograph-en.pdf.  [https://www.astrazeneca.ca/content/dam/az-ca/downloads/productinformation/tagrisso-product-monograph-en.pdf](https://www.astrazeneca.ca/content/dam/az-ca/downloads/productinformation/tagrisso-product-monograph-en.pdf) | [Google Scholar](https://scholar.google.com/scholar_lookup?title=TAGRISSO%20(osimertinib)%20%5BProduct%20monograph%5D&publication_year=2021&)

3. Brown K., Comisar C., Witjes H., Maringwa J., de Greef R., Vishwanathan K., et al. (2017). Population pharmacokinetics and exposure-response of osimertinib in patients with non-small cell lung cancer. Br. J. Clin. Pharmacol. 83 (6), 1216–1226. 10.1111/bcp.13223  [DOI](https://doi.org/10.1111/bcp.13223) | [PMC free article](/articles/PMC5427226/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28009438/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Clin.%20Pharmacol.&title=Population%20pharmacokinetics%20and%20exposure-response%20of%20osimertinib%20in%20patients%20with%20non-small%20cell%20lung%20cancer&author=K.%20Brown&author=C.%20Comisar&author=H.%20Witjes&author=J.%20Maringwa&author=R.%20de%20Greef&volume=83&issue=6&publication_year=2017&pages=1216-1226&pmid=28009438&doi=10.1111/bcp.13223&)

4. Bruckmueller H., Werk A. N., Renders L., Feldkamp T., Tepel M., Borst C., et al. (2015). Which genetic determinants should be considered for tacrolimus dose optimization in kidney transplantation? A combined analysis of genes affecting the CYP3A locus. Ther. Drug Monit. 37 (3), 288–295. 10.1097/ftd.0000000000000142  [DOI](https://doi.org/10.1097/ftd.0000000000000142) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25271728/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ther.%20Drug%20Monit.&title=Which%20genetic%20determinants%20should%20be%20considered%20for%20tacrolimus%20dose%20optimization%20in%20kidney%20transplantation?%20A%20combined%20analysis%20of%20genes%20affecting%20the%20CYP3A%20locus&author=H.%20Bruckmueller&author=A.%20N.%20Werk&author=L.%20Renders&author=T.%20Feldkamp&author=M.%20Tepel&volume=37&issue=3&publication_year=2015&pages=288-295&pmid=25271728&doi=10.1097/ftd.0000000000000142&)

5. Cha P. C., Mushiroda T., Zembutsu H., Harada H., Shinoda N., Kawamoto S., et al. (2009). Single nucleotide polymorphism in ABCG2 is associated with irinotecan-induced severe myelosuppression. J. Hum. Genet. 54 (10), 572–580. 10.1038/jhg.2009.80  [DOI](https://doi.org/10.1038/jhg.2009.80) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19696792/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Hum.%20Genet.&title=Single%20nucleotide%20polymorphism%20in%20ABCG2%20is%20associated%20with%20irinotecan-induced%20severe%20myelosuppression&author=P.%20C.%20Cha&author=T.%20Mushiroda&author=H.%20Zembutsu&author=H.%20Harada&author=N.%20Shinoda&volume=54&issue=10&publication_year=2009&pages=572-580&pmid=19696792&doi=10.1038/jhg.2009.80&)

6. Cho B. C., Chewaskulyong B., Lee K. H., Dechaphunkul A., Sriuranpong V., Imamura F., et al. (2019). Osimertinib versus standard of care EGFR TKI as first-line treatment in patients with EGFRm advanced NSCLC: FLAURA asian subset. J. Thorac. Oncol. 14 (1), 99–106. 10.1016/j.jtho.2018.09.004  [DOI](https://doi.org/10.1016/j.jtho.2018.09.004) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30240852/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Thorac.%20Oncol.&title=Osimertinib%20versus%20standard%20of%20care%20EGFR%20TKI%20as%20first-line%20treatment%20in%20patients%20with%20EGFRm%20advanced%20NSCLC:%20FLAURA%20asian%20subset&author=B.%20C.%20Cho&author=B.%20Chewaskulyong&author=K.%20H.%20Lee&author=A.%20Dechaphunkul&author=V.%20Sriuranpong&volume=14&issue=1&publication_year=2019&pages=99-106&pmid=30240852&doi=10.1016/j.jtho.2018.09.004&)

7. CV000008408.12 - ClinVar - NCBI (2013). VCV000008408.12 - ClinVar - NCBI. Available from: https://www.ncbi.nlm.nih.gov/clinvar/variation/8408/.  [https://www.ncbi.nlm.nih.gov/clinvar/variation/8408/](https://www.ncbi.nlm.nih.gov/clinvar/variation/8408/)

8. Desta Z., Kreutz Y., Nguyen A. T., Li L., Skaar T., Kamdem L. K., et al. (2011). Plasma letrozole concentrations in postmenopausal women with breast cancer are associated with CYP2A6 genetic variants, body mass index, and age. Clin. Pharmacol. Ther. 90 (5), 693–700. 10.1038/clpt.2011.174  [DOI](https://doi.org/10.1038/clpt.2011.174) | [PMC free article](/articles/PMC3667657/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21975350/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Plasma%20letrozole%20concentrations%20in%20postmenopausal%20women%20with%20breast%20cancer%20are%20associated%20with%20CYP2A6%20genetic%20variants,%20body%20mass%20index,%20and%20age&author=Z.%20Desta&author=Y.%20Kreutz&author=A.%20T.%20Nguyen&author=L.%20Li&author=T.%20Skaar&volume=90&issue=5&publication_year=2011&pages=693-700&pmid=21975350&doi=10.1038/clpt.2011.174&)

9. Detarkom S., Incharoen P., Jinawat A., Trachu N., Kamprerasart K., Prasongsook N., et al. (2018). P3.09-08 tumor heterogeneity and molecular profile of NSCLC in Thai population. J. Thorac. Oncol. 13 (10), S949–S950. 10.1016/j.jtho.2018.08.1777  [DOI](https://doi.org/10.1016/j.jtho.2018.08.1777) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Thorac.%20Oncol.&title=P3.09-08%20tumor%20heterogeneity%20and%20molecular%20profile%20of%20NSCLC%20in%20Thai%20population&author=S.%20Detarkom&author=P.%20Incharoen&author=A.%20Jinawat&author=N.%20Trachu&author=K.%20Kamprerasart&volume=13&issue=10&publication_year=2018&pages=S949-S950&doi=10.1016/j.jtho.2018.08.1777&)

10. Dickinson P. A., Cantarini M. V., Collier J., Frewer P., Martin S., Pickup K., et al. (2016). Metabolic disposition of osimertinib in rats, dogs, and humans: insights into a drug designed to bind covalently to a cysteine residue of epidermal growth factor receptor. Drug Metabolism Dispos. 44 (8), 1201–1212. 10.1124/dmd.115.069203  [DOI](https://doi.org/10.1124/dmd.115.069203) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27226351/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metabolism%20Dispos.&title=Metabolic%20disposition%20of%20osimertinib%20in%20rats,%20dogs,%20and%20humans:%20insights%20into%20a%20drug%20designed%20to%20bind%20covalently%20to%20a%20cysteine%20residue%20of%20epidermal%20growth%20factor%20receptor&author=P.%20A.%20Dickinson&author=M.%20V.%20Cantarini&author=J.%20Collier&author=P.%20Frewer&author=S.%20Martin&volume=44&issue=8&publication_year=2016&pages=1201-1212&pmid=27226351&doi=10.1124/dmd.115.069203&)

11. Djordjevic N., Ghotbi R., Jankovic S., Aklillu E. (2010). Induction of CYP1A2 by heavy coffee consumption is associated with the CYP1A2 −163C>A polymorphism. Eur. J. Clin. Pharmacol. 66 (7), 697–703. 10.1007/s00228-010-0823-4  [DOI](https://doi.org/10.1007/s00228-010-0823-4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20390257/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Clin.%20Pharmacol.&title=Induction%20of%20CYP1A2%20by%20heavy%20coffee%20consumption%20is%20associated%20with%20the%20CYP1A2%20%E2%88%92163C>A%20polymorphism&author=N.%20Djordjevic&author=R.%20Ghotbi&author=S.%20Jankovic&author=E.%20Aklillu&volume=66&issue=7&publication_year=2010&pages=697-703&pmid=20390257&doi=10.1007/s00228-010-0823-4&)

12. Giovannini M., Gregorc V., Belli C., Roca E., Lazzari C., Viganò M. G., et al. (2009). Clinical significance of skin toxicity due to EGFR-targeted therapies. J. Oncol. 2009, 849051–849058. 10.1155/2009/849051  [DOI](https://doi.org/10.1155/2009/849051) | [PMC free article](/articles/PMC2699661/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19584908/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Oncol.&title=Clinical%20significance%20of%20skin%20toxicity%20due%20to%20EGFR-targeted%20therapies&author=M.%20Giovannini&author=V.%20Gregorc&author=C.%20Belli&author=E.%20Roca&author=C.%20Lazzari&volume=2009&publication_year=2009&pages=849051-849058&pmid=19584908&doi=10.1155/2009/849051&)

13. Goss G., Tsai C. M., Shepherd F. A., Bazhenova L., Lee J. S., Chang G. C., et al. (2016). Osimertinib for pretreated EGFR Thr790Met-positive advanced non-small-cell lung cancer (AURA2): a multicentre, open-label, single-arm, phase 2 study. Lancet Oncol. 17 (12), 1643–1652. 10.1016/s1470-2045(16)30508-3  [DOI](https://doi.org/10.1016/s1470-2045(16)30508-3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27751847/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet%20Oncol.&title=Osimertinib%20for%20pretreated%20EGFR%20Thr790Met-positive%20advanced%20non-small-cell%20lung%20cancer%20(AURA2):%20a%20multicentre,%20open-label,%20single-arm,%20phase%202%20study&author=G.%20Goss&author=C.%20M.%20Tsai&author=F.%20A.%20Shepherd&author=L.%20Bazhenova&author=J.%20S.%20Lee&volume=17&issue=12&publication_year=2016&pages=1643-1652&pmid=27751847&doi=10.1016/s1470-2045(16)30508-3&)

14. Han L., Zhang X., Wang Z., Zhang X., Zhao L., Fu W., et al. (2021). SH-1028, an irreversible third-generation EGFR TKI, overcomes t790m-mediated resistance in non-small cell lung cancer. Front. Pharmacol. 12, 665253. 10.3389/fphar.2021.665253  [DOI](https://doi.org/10.3389/fphar.2021.665253) | [PMC free article](/articles/PMC8111447/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33986687/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front.%20Pharmacol.&title=SH-1028,%20an%20irreversible%20third-generation%20EGFR%20TKI,%20overcomes%20t790m-mediated%20resistance%20in%20non-small%20cell%20lung%20cancer&author=L.%20Han&author=X.%20Zhang&author=Z.%20Wang&author=X.%20Zhang&author=L.%20Zhao&volume=12&publication_year=2021&pages=665253&pmid=33986687&doi=10.3389/fphar.2021.665253&)

15. Hirsh V., Blais N., Burkes R., Verma S., Croitoru K. (2014). Management of diarrhea induced by epidermal growth factor receptor tyrosine kinase inhibitors. Curr. Oncol. 21 (6), 329–336. 10.3747/co.21.2241  [DOI](https://doi.org/10.3747/co.21.2241) | [PMC free article](/articles/PMC4257116/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25489260/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr.%20Oncol.&title=Management%20of%20diarrhea%20induced%20by%20epidermal%20growth%20factor%20receptor%20tyrosine%20kinase%20inhibitors&author=V.%20Hirsh&author=N.%20Blais&author=R.%20Burkes&author=S.%20Verma&author=K.%20Croitoru&volume=21&issue=6&publication_year=2014&pages=329-336&pmid=25489260&doi=10.3747/co.21.2241&)

16. Ikebe S., Amiya R., Minami S., Ihara S., Higuchi Y., Komuta K. (2020). Osimertinib-induced cardiac failure with QT prolongation and torsade de pointes in a patient with advanced pulmonary adenocarcinoma. Int. Cancer Conf. J. 10 (1), 68–71. 10.1007/s13691-020-00450-2  [DOI](https://doi.org/10.1007/s13691-020-00450-2) | [PMC free article](/articles/PMC7797397/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33489705/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int.%20Cancer%20Conf.%20J.&title=Osimertinib-induced%20cardiac%20failure%20with%20QT%20prolongation%20and%20torsade%20de%20pointes%20in%20a%20patient%20with%20advanced%20pulmonary%20adenocarcinoma&author=S.%20Ikebe&author=R.%20Amiya&author=S.%20Minami&author=S.%20Ihara&author=Y.%20Higuchi&volume=10&issue=1&publication_year=2020&pages=68-71&pmid=33489705&doi=10.1007/s13691-020-00450-2&)

17. Ishikawa E., Yokoyama Y., Chishima H., Kasai H., Kuniyoshi O., Kimura M., et al. (2023). Population pharmacokinetics, Pharmacogenomics, and adverse events of osimertinib and its two active metabolites, AZ5104 and AZ7550, in Japanese patients with advanced non-small cell lung cancer: a prospective observational study. Investig. New Drugs 41 (1), 122–133. 10.1007/s10637-023-01328-9  [DOI](https://doi.org/10.1007/s10637-023-01328-9) | [PMC free article](/articles/PMC10030409/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36637703/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Investig.%20New%20Drugs&title=Population%20pharmacokinetics,%20Pharmacogenomics,%20and%20adverse%20events%20of%20osimertinib%20and%20its%20two%20active%20metabolites,%20AZ5104%20and%20AZ7550,%20in%20Japanese%20patients%20with%20advanced%20non-small%20cell%20lung%20cancer:%20a%20prospective%20observational%20study&author=E.%20Ishikawa&author=Y.%20Yokoyama&author=H.%20Chishima&author=H.%20Kasai&author=O.%20Kuniyoshi&volume=41&issue=1&publication_year=2023&pages=122-133&pmid=36637703&doi=10.1007/s10637-023-01328-9&)

18. Li Y., Mao T., Wang J., Zheng H., Hu Z., Cao P., et al. (2023). Toward the next generation EGFR inhibitors: an overview of osimertinib resistance mediated by EGFR mutations in non-small cell lung cancer. Cell Commun. Signal. 21 (1), 71. 10.1186/s12964-023-01082-8  [DOI](https://doi.org/10.1186/s12964-023-01082-8) | [PMC free article](/articles/PMC10088170/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/37041601/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Commun.%20Signal.&title=Toward%20the%20next%20generation%20EGFR%20inhibitors:%20an%20overview%20of%20osimertinib%20resistance%20mediated%20by%20EGFR%20mutations%20in%20non-small%20cell%20lung%20cancer&author=Y.%20Li&author=T.%20Mao&author=J.%20Wang&author=H.%20Zheng&author=Z.%20Hu&volume=21&issue=1&publication_year=2023&pages=71&pmid=37041601&doi=10.1186/s12964-023-01082-8&)

19. Liao D., Liu Z., Zhang Y., Liu N., Yao D., Cao L., et al. (2020). Polymorphisms of drug-metabolizing enzymes and transporters contribute to the individual variations of erlotinib steady state trough concentration, treatment outcomes, and adverse reactions in epidermal growth factor receptor–mutated non-small cell lung cancer patients. Front. Pharmacol. 11, 664. 10.3389/fphar.2020.00664  [DOI](https://doi.org/10.3389/fphar.2020.00664) | [PMC free article](/articles/PMC7225310/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32457635/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front.%20Pharmacol.&title=Polymorphisms%20of%20drug-metabolizing%20enzymes%20and%20transporters%20contribute%20to%20the%20individual%20variations%20of%20erlotinib%20steady%20state%20trough%20concentration,%20treatment%20outcomes,%20and%20adverse%20reactions%20in%20epidermal%20growth%20factor%20receptor%E2%80%93mutated%20non-small%20cell%20lung%20cancer%20patients&author=D.%20Liao&author=Z.%20Liu&author=Y.%20Zhang&author=N.%20Liu&author=D.%20Yao&volume=11&publication_year=2020&pages=664&pmid=32457635&doi=10.3389/fphar.2020.00664&)

20. Low S. K., Fukunaga K., Takahashi A., Matsuda K., Hongo F., Nakanishi H., et al. (2016). Association study of a functional variant on ABCG2 gene with sunitinib-induced severe adverse drug reaction. PLOS ONE 11 (2), e0148177. 10.1371/journal.pone.0148177  [DOI](https://doi.org/10.1371/journal.pone.0148177) | [PMC free article](/articles/PMC4767438/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26914831/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLOS%20ONE&title=Association%20study%20of%20a%20functional%20variant%20on%20ABCG2%20gene%20with%20sunitinib-induced%20severe%20adverse%20drug%20reaction&author=S.%20K.%20Low&author=K.%20Fukunaga&author=A.%20Takahashi&author=K.%20Matsuda&author=F.%20Hongo&volume=11&issue=2&publication_year=2016&pages=e0148177&pmid=26914831&doi=10.1371/journal.pone.0148177&)

21. Mok T. S., Wu Y. L., Ahn M. J., Garassino M. C., Kim H. R., Ramalingam S. S., et al. (2017). Osimertinib or platinum–pemetrexed in EGFR t790m–positive lung cancer. N. Engl. J. Med. 376 (7), 629–640. 10.1056/nejmoa1612674  [DOI](https://doi.org/10.1056/nejmoa1612674) | [PMC free article](/articles/PMC6762027/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27959700/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%20Engl.%20J.%20Med.&title=Osimertinib%20or%20platinum%E2%80%93pemetrexed%20in%20EGFR%20t790m%E2%80%93positive%20lung%20cancer&author=T.%20S.%20Mok&author=Y.%20L.%20Wu&author=M.%20J.%20Ahn&author=M.%20C.%20Garassino&author=H.%20R.%20Kim&volume=376&issue=7&publication_year=2017&pages=629-640&pmid=27959700&doi=10.1056/nejmoa1612674&)

22. Ngamjarus C. (2016). n4Studies: sample size calculation for an epidemiological study on a smart device | siriraj medical journal. n4Studies: sample size calculation for an epidemiological study on a smart device | siriraj medical journal. Available from: https://he02.tci-thaijo.org/index.php/sirirajmedj/article/view/58342.  [https://he02.tci-thaijo.org/index.php/sirirajmedj/article/view/58342](https://he02.tci-thaijo.org/index.php/sirirajmedj/article/view/58342) | [Google Scholar](https://scholar.google.com/scholar_lookup?title=n4Studies:%20sample%20size%20calculation%20for%20an%20epidemiological%20study%20on%20a%20smart%20device%20%7C%20siriraj%20medical%20journal.%20n4Studies:%20sample%20size%20calculation%20for%20an%20epidemiological%20study%20on%20a%20smart%20device%20%7C%20siriraj%20medical%20journal&author=C.%20Ngamjarus&publication_year=2016&)

23. Nicholson A. G., Tsao M. S., Beasley M. B., Borczuk A. C., Brambilla E., Cooper W. A., et al. (2022). The 2021 WHO classification of lung tumors: impact of advances since 2015. J. Thorac. Oncol. 17 (3), 362–387. 10.1016/j.jtho.2021.11.003  [DOI](https://doi.org/10.1016/j.jtho.2021.11.003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34808341/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Thorac.%20Oncol.&title=The%202021%20WHO%20classification%20of%20lung%20tumors:%20impact%20of%20advances%20since%202015&author=A.%20G.%20Nicholson&author=M.%20S.%20Tsao&author=M.%20B.%20Beasley&author=A.%20C.%20Borczuk&author=E.%20Brambilla&volume=17&issue=3&publication_year=2022&pages=362-387&pmid=34808341&doi=10.1016/j.jtho.2021.11.003&)

24. Pang C., Liu J. H., Xu Y. S., Chen C., Dai P. G. (2015). The allele frequency of CYP2A6*4 in four ethnic groups of China. Exp. Mol. Pathology 98 (3), 546–548. 10.1016/j.yexmp.2015.03.040  [DOI](https://doi.org/10.1016/j.yexmp.2015.03.040) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25862079/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Exp.%20Mol.%20Pathology&title=The%20allele%20frequency%20of%20CYP2A6*4%20in%20four%20ethnic%20groups%20of%20China&author=C.%20Pang&author=J.%20H.%20Liu&author=Y.%20S.%20Xu&author=C.%20Chen&author=P.%20G.%20Dai&volume=98&issue=3&publication_year=2015&pages=546-548&pmid=25862079&doi=10.1016/j.yexmp.2015.03.040&)

25. PharmGKB (2021). PharmGKB. Available from: https://www.pharmgkb.org/.  [https://www.pharmgkb.org/](https://www.pharmgkb.org/) | [Google Scholar](https://scholar.google.com/scholar_lookup?title=PharmGKB&publication_year=2021&)

26. Ramalingam S. S., Vansteenkiste J., Planchard D., Cho B. C., Gray J. E., Ohe Y., et al. (2020). Overall survival with osimertinib in untreated, EGFR-mutated advanced NSCLC. N. Engl. J. Med. 382 (1), 41–50. 10.1056/nejmoa1913662  [DOI](https://doi.org/10.1056/nejmoa1913662) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31751012/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%20Engl.%20J.%20Med.&title=Overall%20survival%20with%20osimertinib%20in%20untreated,%20EGFR-mutated%20advanced%20NSCLC&author=S.%20S.%20Ramalingam&author=J.%20Vansteenkiste&author=D.%20Planchard&author=B.%20C.%20Cho&author=J.%20E.%20Gray&volume=382&issue=1&publication_year=2020&pages=41-50&pmid=31751012&doi=10.1056/nejmoa1913662&)

27. Remon J., Steuer C., Ramalingam S., Felip E. (2018). Osimertinib and other third-generation EGFR TKI in EGFR-mutant NSCLC patients. Ann. Oncol. 29, i20–i27. 10.1093/annonc/mdx704  [DOI](https://doi.org/10.1093/annonc/mdx704) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29462255/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann.%20Oncol.&title=Osimertinib%20and%20other%20third-generation%20EGFR%20TKI%20in%20EGFR-mutant%20NSCLC%20patients&author=J.%20Remon&author=C.%20Steuer&author=S.%20Ramalingam&author=E.%20Felip&volume=29&publication_year=2018&pages=i20-i27&pmid=29462255&doi=10.1093/annonc/mdx704&)

28. Rodier T., Puszkiel A., Cardoso E., Balakirouchenane D., Narjoz C., Arrondeau J., et al. (2022). Exposure–response analysis of osimertinib in patients with advanced non-small-cell lung cancer. Pharmaceutics 14 (9), 1844. 10.3390/pharmaceutics14091844  [DOI](https://doi.org/10.3390/pharmaceutics14091844) | [PMC free article](/articles/PMC9504753/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36145591/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmaceutics&title=Exposure%E2%80%93response%20analysis%20of%20osimertinib%20in%20patients%20with%20advanced%20non-small-cell%20lung%20cancer&author=T.%20Rodier&author=A.%20Puszkiel&author=E.%20Cardoso&author=D.%20Balakirouchenane&author=C.%20Narjoz&volume=14&issue=9&publication_year=2022&pages=1844&pmid=36145591&doi=10.3390/pharmaceutics14091844&)

29. Sogawa R., Nakashima C., Nakamura T., Takeuchi K., Kimura S., Komiya K., et al. (2020). Association of genetic polymorphisms with afatinib-induced diarrhoea. Vivo 34 (3), 1415–1419. 10.21873/invivo.11922  [DOI](https://doi.org/10.21873/invivo.11922) | [PMC free article](/articles/PMC7279839/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32354939/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Vivo&title=Association%20of%20genetic%20polymorphisms%20with%20afatinib-induced%20diarrhoea&author=R.%20Sogawa&author=C.%20Nakashima&author=T.%20Nakamura&author=K.%20Takeuchi&author=S.%20Kimura&volume=34&issue=3&publication_year=2020&pages=1415-1419&pmid=32354939&doi=10.21873/invivo.11922&)

30. Soria J. C., Ohe Y., Vansteenkiste J., Reungwetwattana T., Chewaskulyong B., Lee K. H., et al. (2018). Osimertinib in UntreatedEGFR-mutated advanced non–small-cell lung cancer. N. Engl. J. Med. 378 (2), 113–125. 10.1056/nejmoa1713137  [DOI](https://doi.org/10.1056/nejmoa1713137) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29151359/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%20Engl.%20J.%20Med.&title=Osimertinib%20in%20UntreatedEGFR-mutated%20advanced%20non%E2%80%93small-cell%20lung%20cancer&author=J.%20C.%20Soria&author=Y.%20Ohe&author=J.%20Vansteenkiste&author=T.%20Reungwetwattana&author=B.%20Chewaskulyong&volume=378&issue=2&publication_year=2018&pages=113-125&pmid=29151359&doi=10.1056/nejmoa1713137&)

31. Sung H., Ferlay J., Siegel R. L., Laversanne M., Soerjomataram I., Jemal A., et al. (2021). Global cancer statistics 2020: GLOBOCAN estimates of incidence and mortality worldwide for 36 cancers in 185 countries. CA A Cancer J. Clin. 71 (3), 209–249. 10.3322/caac.21660  [DOI](https://doi.org/10.3322/caac.21660) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33538338/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=CA%20A%20Cancer%20J.%20Clin.&title=Global%20cancer%20statistics%202020:%20GLOBOCAN%20estimates%20of%20incidence%20and%20mortality%20worldwide%20for%2036%20cancers%20in%20185%20countries&author=H.%20Sung&author=J.%20Ferlay&author=R.%20L.%20Siegel&author=M.%20Laversanne&author=I.%20Soerjomataram&volume=71&issue=3&publication_year=2021&pages=209-249&pmid=33538338&doi=10.3322/caac.21660&)

32. Suzumura T., Kimura T., Kudoh S., Umekawa K., Nagata M., Matsuura K., et al. (2012). Reduced CYP2D6 function is associated with gefitinib-induced rash in patients with non-small cell lung cancer. BMC Cancer 12 (1), 568. 10.1186/1471-2407-12-568  [DOI](https://doi.org/10.1186/1471-2407-12-568) | [PMC free article](/articles/PMC3536666/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23207012/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Cancer&title=Reduced%20CYP2D6%20function%20is%20associated%20with%20gefitinib-induced%20rash%20in%20patients%20with%20non-small%20cell%20lung%20cancer&author=T.%20Suzumura&author=T.%20Kimura&author=S.%20Kudoh&author=K.%20Umekawa&author=M.%20Nagata&volume=12&issue=1&publication_year=2012&pages=568&pmid=23207012&doi=10.1186/1471-2407-12-568&)

33. Wang L., Hu Z., Deng X., Wang Y., Zhang Z., Cheng Z. N. (2012). Association between CommonCYP1A2Polymorphisms and theophylline metabolism in non-smoking healthy volunteers. Basic & Clin. Pharmacol. Toxicol. 112 (4), 257–263. 10.1111/bcpt.12038  [DOI](https://doi.org/10.1111/bcpt.12038) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23167834/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Basic%20&%20Clin.%20Pharmacol.%20Toxicol.&title=Association%20between%20CommonCYP1A2Polymorphisms%20and%20theophylline%20metabolism%20in%20non-smoking%20healthy%20volunteers&author=L.%20Wang&author=Z.%20Hu&author=X.%20Deng&author=Y.%20Wang&author=Z.%20Zhang&volume=112&issue=4&publication_year=2012&pages=257-263&pmid=23167834&doi=10.1111/bcpt.12038&)

34. Westover D., Zugazagoitia J., Cho B., Lovly C., Paz-Ares L. (2018). Mechanisms of acquired resistance to first- and second-generation EGFR tyrosine kinase inhibitors. Ann. Oncol. 29, i10–i19. 10.1093/annonc/mdx703  [DOI](https://doi.org/10.1093/annonc/mdx703) | [PMC free article](/articles/PMC6454547/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29462254/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann.%20Oncol.&title=Mechanisms%20of%20acquired%20resistance%20to%20first-%20and%20second-generation%20EGFR%20tyrosine%20kinase%20inhibitors&author=D.%20Westover&author=J.%20Zugazagoitia&author=B.%20Cho&author=C.%20Lovly&author=L.%20Paz-Ares&volume=29&publication_year=2018&pages=i10-i19&pmid=29462254&doi=10.1093/annonc/mdx703&)

35. Whirl‐Carrillo M., Huddart R., Gong L., Sangkuhl K., Thorn C. F., Whaley R., et al. (2021). An evidence‐based framework for evaluating Pharmacogenomics knowledge for personalized medicine. Clin. Pharmacol. Ther. 110 (3), 563–572. 10.1002/cpt.2350  [DOI](https://doi.org/10.1002/cpt.2350) | [PMC free article](/articles/PMC8457105/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34216021/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=An%20evidence%E2%80%90based%20framework%20for%20evaluating%20Pharmacogenomics%20knowledge%20for%20personalized%20medicine&author=M.%20Whirl%E2%80%90Carrillo&author=R.%20Huddart&author=L.%20Gong&author=K.%20Sangkuhl&author=C.%20F.%20Thorn&volume=110&issue=3&publication_year=2021&pages=563-572&pmid=34216021&doi=10.1002/cpt.2350&)

36. Yu H. A., Arcila M. E., Rekhtman N., Sima C. S., Zakowski M. F., Pao W., et al. (2013). Analysis of tumor specimens at the time of acquired resistance to EGFR-TKI therapy in 155 patients with EGFR-mutant lung cancers. Clin. Cancer Res. 19 (8), 2240–2247. 10.1158/1078-0432.ccr-12-2246  [DOI](https://doi.org/10.1158/1078-0432.ccr-12-2246) | [PMC free article](/articles/PMC3630270/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23470965/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Cancer%20Res.&title=Analysis%20of%20tumor%20specimens%20at%20the%20time%20of%20acquired%20resistance%20to%20EGFR-TKI%20therapy%20in%20155%20patients%20with%20EGFR-mutant%20lung%20cancers&author=H.%20A.%20Yu&author=M.%20E.%20Arcila&author=N.%20Rekhtman&author=C.%20S.%20Sima&author=M.%20F.%20Zakowski&volume=19&issue=8&publication_year=2013&pages=2240-2247&pmid=23470965&doi=10.1158/1078-0432.ccr-12-2246&)
