# The important roles of ERAP1, ERAP2 genes polymorphisms and their DNA methylation levels in pulmonary tuberculosis

## Metadata
**Authors:** Shi-Hong Su, Xue-Qian Cai, Yong-Huai Li, Ai-Hui Xu, Qian Huang, Hua Niu, Qing-Hai You, Geng-Yun Sun
**Journal:** BMC Infectious Diseases
**Date:** 2025 Feb 5
**DOI:** [10.1186/s12879-025-10452-1](https://doi.org/10.1186/s12879-025-10452-1)
**PMID:** 39910453
**PMCID:** PMC11800520
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11800520/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC11800520/pdf/12879_2025_Article_10452.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC11800520/pdf/12879_2025_Article_10452.pdf)

## Abstract

**Background:** 
Genetic variations in endoplasmic reticulum aminopeptidase (ERAP1, ERAP2) genes are associated with the pathogenesis of multiple diseases, including infectious diseases. The objective of our study was to assess whether ERAP1, ERAP2 genes polymorphisms and methylation levels affect the risk of pulmonary tuberculosis (PTB).

**Methods:** 
We genotyped ten single nucleotide polymorphisms (SNPs) in ERAP1, ERAP2 genes among 497 PTB patients and 502 controls by SNPscan technique. Additionally, we detected the ERAP1, ERAP2 genes methylation levels in 98 PTB patients and 97 controls through Illumina Hiseq platform.

**Results:** 
Our results showed that the GG genotype, G allele frequencies of ERAP2 gene rs2549782 were significantly increased in PTB patients compared to controls, and rs2549782 polymorphism was related to the increased risk of PTB under recessive model. In addition, no significant relationship was found between ERAP1 gene rs13167972, rs17086651, rs469783, rs26618, rs3734016, ERAP2 gene rs17524572, rs1230358, rs2549794, rs117041256 and PTB susceptibility. Among PTB patients, the frequencies of rs17524572 TT genotype, T allele were significantly associated with drug-induced liver injury, and rs1230358, rs2549782, rs2549794 polymorphisms were linked to the occurrence of fever. Haplotype analysis of ERAP2 gene suggested that the frequency of CTGGT haplotype was higher, while the frequency of CTGTT haplotype was lower in PTB patients. When compared with controls, the ERAP1 methylation level was lower in PTB patients, while the ERAP2 methylation level was significantly increased. Moreover, rs1230358 polymorphism was found to affect the ERAP2 methylation level.

**Conclusion:** 
The ERAP2 rs2549782 variant and altered methylation levels of ERAP1, ERAP2 genes were related to susceptibility to PTB.

**Supplementary Information:** 
The online version contains supplementary material available at 10.1186/s12879-025-10452-1.

Keywords: ERAP1, ERAP2, Pulmonary tuberculosis, Single nucleotide polymorphisms, DNA methylation

### Background

Genetic variations in endoplasmic reticulum aminopeptidase (*ERAP1*, *ERAP2*) genes are associated with the pathogenesis of multiple diseases, including infectious diseases. The objective of our study was to assess whether *ERAP1*, *ERAP2* genes polymorphisms and methylation levels affect the risk of pulmonary tuberculosis (PTB).

### Methods

We genotyped ten single nucleotide polymorphisms (SNPs) in *ERAP1*, *ERAP2* genes among 497 PTB patients and 502 controls by SNPscan technique. Additionally, we detected the *ERAP1*, *ERAP2* genes methylation levels in 98 PTB patients and 97 controls through Illumina Hiseq platform.

### Results

Our results showed that the GG genotype, G allele frequencies of *ERAP2* gene rs2549782 were significantly increased in PTB patients compared to controls, and rs2549782 polymorphism was related to the increased risk of PTB under recessive model. In addition, no significant relationship was found between *ERAP1* gene rs13167972, rs17086651, rs469783, rs26618, rs3734016, *ERAP2* gene rs17524572, rs1230358, rs2549794, rs117041256 and PTB susceptibility. Among PTB patients, the frequencies of rs17524572 TT genotype, T allele were significantly associated with drug-induced liver injury, and rs1230358, rs2549782, rs2549794 polymorphisms were linked to the occurrence of fever. Haplotype analysis of *ERAP2* gene suggested that the frequency of CTGGT haplotype was higher, while the frequency of CTGTT haplotype was lower in PTB patients. When compared with controls, the *ERAP1* methylation level was lower in PTB patients, while the *ERAP2* methylation level was significantly increased. Moreover, rs1230358 polymorphism was found to affect the *ERAP2* methylation level.

### Conclusion

The *ERAP2* rs2549782 variant and altered methylation levels of *ERAP1*,* ERAP2* genes were related to susceptibility to PTB.

### Supplementary Information

The online version contains supplementary material available at 10.1186/s12879-025-10452-1.

## Introduction

Tuberculosis (TB) is a highly consequential communicable ailment caused by Mycobacterium tuberculosis (MTB), with pulmonary tuberculosis (PTB) being the most common type. It continues to pose a substantial global public health concern [[1](#CR1)]. Although the global incidence of TB is gradually declining, MTB infection is responsible for 9.9 million new cases and 1.3 million deaths, placing a significant disease burden on society [[2](#CR2)]. The pathogenesis of TB is intricate, as evidenced by research indicating that only 10% of individuals infected with MTB ultimately develop active TB, while the majority remain asymptomatic [[3](#CR3)]. This observation implies that genetic factors significantly contribute to the progression of this disease following MTB infection. Furthermore, twin studies and genome-wide association studies (GWAS) have further substantiated the impact of host genetic variation on susceptibility to TB [[4](#CR4)–[6](#CR6)].

At present, single nucleotide polymorphism (SNP) analysis was an important tool to find the key genes associated with disease susceptibility. Accumulating evidence had indicated that polymorphism in immune-related genes could contribute to the development of TB [[7](#CR7), [8](#CR8)]. Immediately following MTB infection, the host’s innate immune system established its first defense mechanism. Several genes involved in this first stage were associated with TB susceptibility, including mannose-binding lectin, surfactant protein, CD14 [[8](#CR8)–[10](#CR10)]. In addition, the gene variations in vitamin D metabolic pathways, lncRNA were also related to TB susceptibility and several clinical manifestations [[11](#CR11), [12](#CR12)]. On the other hand, the field of epigenetics had emerged as a significant avenue of inquiry for investigating the connection between genetic variation and the development of diseases. DNA methylation was an important epigenetic trait, which entailed the addition of a methyl group to cytosine in CpG dinucleotide. Moreover, DNA methylation played critical roles in many biological processes, such as genomic imprinting, x chromosome inactivation, and aging [[13](#CR13)]. Notably, recent research endeavors had examined the impact of epigenetics on the outcome and pathophysiology of TB [[14](#CR14)].

Some antigen presentation genes also played an important role in controlling MTB infection during the adaptive immune response activated after MTB infection. In the process of antigen presentation by the major histocompatibility complex (MHC) class I, the peptide was trimmed to a suitable size by endoplasmic reticulum aminopeptidase (ERAP) so that it could bind to MHC-I and be assembled into peptide-loading complexes, which had prominent roles in inducing a positive immune response against the pathogens [[15](#CR15), [16](#CR16)]. ERAPs (ERAP1 and ERAP2) belonged to the oxytocin subfamily of the M1 zinc metallopeptidase family and were originally considered to be a homology of insulin-regulated aminopeptidase or human placental leucine aminopeptidase [[17](#CR17)]. As a human leukocyte antigen (HLA) class I gene, *ERAPs* genes polymorphisms had also been shown to be associated with the pathogenesis of several autoimmune diseases and infectious diseases [[18](#CR18), [19](#CR19)].

Some previous studies had demonstrated that *HLA* alleles polymorphisms could increase susceptibility or provide resistance to PTB, and that *ERAP1*, *ERAP2* genes polymorphisms were associated with PTB susceptibility [[19](#CR19)–[21](#CR21)]. However, the study regarding to the roles of *ERAP1*, *ERAP2* genetic variants in PTB was still very limited. Therefore, we analyzed the influence of genetic variation and DNA methylation in the *ERAP1* and *ERAP2* genes on the risk of PTB in a Chinese Han population.

## Materials and methods

### Study participants

The analysis of the association between PTB susceptibility and *ERAP1*, *ERAP*2 genes polymorphisms involved 999 subjects (497 patients with PTB and 502 controls). Subsequently, we selected 98 patients with PTB and 97 controls from the aforementioned study subjects to detect promoter methylation levels of *ERAP1*, *ERAP2* genes. According to the Diagnosis for PTB (WS 288–2017) and the Classification of TB (WS 196–2017) from the Health Industry Standard of the People’s Republic of China, PTB patients were selected from the Anhui Chest Hospital with the support of experts. The diagnostic criteria of PTB mainly included: (1) MTB infection confirmed by culture sputum smear, interferon-γ release assay, and tuberculin skin test and other bacteriological assessment; (2) The clinical symptoms and chest radiograph consistent with TB. Patients with PTB who had HIV positive, cancer, immune deficiency, autoimmune disease, and other infectious diseases were excluded from the study. Several clinical manifestations, including included fever, drug resistance, drug-induced liver injury (DILI), pulmonary infection, leukopenia, sputum smears, etc., of PTB patients were also collected in this study. A group of controls was included from health examine center in the same area and had no history of TB, infectious disease, lung disease, bacterial or viral infection.

### SNP selection and DNA extraction

The SNPs were identified for genotyping through a combination of screening tagSNPs and reviewing current research in this study. We initially identified tagSNPs with a minor allele frequency (MAF) ≥ 0.05 in CHB, encompassing all common SNPs within the chromosomal regions of *ERAP1*, *ERAP2* and their flanking 2.0 kbp regions using genetic data from Ensembl genome browser 85 and CHBS_1000g. The selection was performed utilizing linkage disequilibrium (LD) analysis with an *r*^2^ equal to 0.8 and employing Haploview 4.0 software (Cambridge, MA, USA). Then, we also reviewed the studies regarding on the association between *ERAP1* and *ERAP2* genes polymorphisms and susceptibility to other diseases to identify SNPs that affect disease susceptibility. Finally, we selected five tagSNPs in *ERAP1* gene (rs13167972, rs17086651, rs469783, rs26618, rs3734016) and five tagSNPs in *ERAP2* gene (rs2549782, rs2549794, rs1230358, rs17524572, rs117041256) for genotyping.

About 3–5 mL of peripheral blood was extracted from each study subject, and genomic DNA of peripheral blood lymphocytes was extracted using the Flexi Gene-DNA Kit (Qiagen, Valencia, CA).

### Genotyping and methylation detecting

The genotyping of these ten SNPs was performed by the SNPscan technique with technical support from the Center for Genetic & Genomic Analysis, Genesky Biotechnologies (Inc., Shanghai). Furthermore, samples were included in the final analysis only if all ten SNPs had been successfully genotyped. The specific steps of SNPscan technique could be found in the previous studies [[22](#CR22), [23](#CR23)].

MethylTarget^®^ was used to detect the promoter DNA methylation levels of *ERAP1* and *ERAP2* genes with the support of the Center for Genetic & Genomic Analysis, Genesky Biotechnologies (Inc., Shanghai). Briefly, the CpG islands in the promoter region of *ERAP1* and *ERAP2* genes were sequenced with the Illumina Hiseq platform. As shown in Table [S1](#MOESM1), the primers were designed to amplify the specific sites of interest from the bisulfite-converted DNA, and the methylation levels at specific sites of each gene could be obtained through calculating the average methylation levels of all CpG sites on the fragment.

### Statistical analysis

The Chi-square (*χ*^2^) was applied for Hardy-Weinberg equilibrium test in normal controls. A logistic regression model was used to investigate the association between genotype frequencies, allele frequencies, three inheritance models (dominant model, recessive model, over-dominant model) and PTB risk. The *ERAP1*,* ERAP2* methylation levels were expressed as median value and interquartile range. A comparison of the methylation levels of these genes among two groups, three groups were respectively calculated using Mann-Whitney *U* test, Kruskal-Wallis *H* test. Haplotype analysis of each gene was performed using SHEsis software. Statistical analysis was conducted using SPSS 23.0 (Armonk, NY: IBM Corp, USA), and statistical significance was defined as a *P*-value less than 0.05.

## Results

### Association between ERAP1, ERAP2 genes allele, genotype frequencies and PTB susceptibility

The average age of 497 PTB patients (included 313 males and 184 females) and 502 normal controls (included 301 males and 201 females) were 44.72 ± 18.03 years and 44.42 ± 3.50 years respectively. There were no significant differences in gender and age distributions between the two groups. Table [1](#Tab1) showed the allele and genotype data of each SNP in *ERAP1*, *ERAP2* genes, and HWE results for controls showed that the genotype distribution of all SNPs matched the genetic balance (*P* > 0.05) (Table [S2](#MOESM1)).

### Table 1.

| SNP | Analyze model |  | PTB patients | Controls | P value |  | OR (95% CI) |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| ERAP1 |  |  |  |  |   |  |  |  |  |
| rs17086651 | Genotype | CC | 35(7.04) | 31(6.18) | 0.699 |  | 1.106(0.664,1.842) |  |  |
|   |   | TC | 170(34.21) | 185(36.85) | 0.435 |  | 0.900(0.691,1.173) |  |  |
|   |   | TT | 292(58.75) | 286(56.97) | Reference |  |  |  |  |
|   | Allele | C | 240(24.14) | 247(24.60) | 0.812 |  | 1.006(0.957,1.057) |  |  |
|   |   | T | 754(75.86) | 757(75.40) | Reference |  |  |  |  |
|   | Dominant model | TT | 292(58.75) | 286(56.97) | 0.569 |  | 0.959(0.829,1.109) |  |  |
|   |   | TC + CC | 205(41.25) | 216(43.03) | Reference |  |  |  |  |
|   | Recessive model | CC | 35(7.04) | 31(6.18) | 0.581 |  | 1.151(0.698,1.898) |  |  |
|   |   | TC + TT | 462(92.96) | 471(93.82) | Reference |  |  |  |  |
|   | Over-dominant model | TC | 170(34.21) | 185(36.85) | 0.382 |  | 1.123(0.866,1.455) |  |  |
|   |   | CC + TT | 327(65.79) | 317(63.15) | Reference |  |  |  |  |
| rs13167972 | Genotype | GG | 5(1.01) | 7(1.39) | 0.594 |  | 0.730(0.230,2.320) |  |  |
|   |   | AG | 89(17.91) | 83(16.53) | 0.584 |  | 1.096(0.789,1.523) |  |  |
|   |   | AA | 403(81.09) | 412(82.07) | Reference |  |  |  |  |
|   | Allele | G | 99(9.96) | 97(9.66) | 0.823 |  | 0.997(0.968,1.026) |  |  |
|   |   | A | 895(90.04) | 907(90.34) | Reference |  |  |  |  |
|   | Dominant model | AA | 467(93.96) | 478(95.22) | 0.679 |  | 1.057(0.812,1.378) |  |  |
|   |   | AG + GG | 94(18.91) | 90(17.93) | Reference |  |  |  |  |
|   | Recessive model | GG | 5(1.01) | 7(1.39) | 0.573 |  | 0.719(0.227,2.280) |  |  |
|   |   | AG + AA | 492(98.99) | 495(98.61) | Reference |  |  |  |  |
|   | Over-dominant model | AG | 89(17.91) | 83(16.53) | 0.565 |  |  |  | 0.908(0.654,1.261) |
|   |   | GG + AA | 408(82.09) | 419(83.47) | Reference |  |  |  |  |
| rs26618 | Genotype | CC | 34(6.84) | 22(4.38) | 0.088 |  | 1.629(0.930,2.853) |  |  |
|   |   | TC | 184(37.02) | 186(37.05) | 0.755 |  | 1.042(0.803,1.354) |  |  |
|   |   | TT | 279(56.14) | 294(58.57) | Reference |  |  |  |  |
|   | Allele | C | 252(25.35) | 230(22.91) | 0.202 |  | 1.143(0.931,1.403) |  |  |
|   |   | T | 742(74.65) | 774(77.09) | Reference |  |  |  |  |
|   | Dominant model | TT | 279(56.14) | 294(58.57) | 0.438 |  | 0.905(0.705,1.164) |  |  |
|   |   | TC + CC | 218(43.86) | 208(41.43) | Reference |  |  |  |  |
|   | Recessive model | CC | 34(6.84) | 22(4.38) | 0.094 |  | 1.602(0.923,2.781) |  |  |
|   |   | TC + TT | 463(93.16) | 480(95.62) | Reference |  |  |  |  |
|   | Over-dominant model | TC | 184(37.02) | 186(37.05) | 0.874 |  | 1.020(0.796,1.308) |  |  |
|   |   | CC + TT | 313(62.98) | 316(62.95) | Reference |  |  |  |  |
| rs3734016 | Genotype | TT | 17(3.42) | 16(3.19) | 0.815 |  | 1.087(0.541,2.185) |  |  |
|   |   | CT | 129(25.96) | 127(25.30) | 0.794 |  | 1.039(0.781,1.383) |  |  |
|   |   | CC | 351(70.62) | 359(71.51) | Reference |  |  |  |  |
|   | Allele | T | 163(16.40) | 159(15.84) | 0.733 |  | 1.042(0.821,1.323) |  |  |
|   |   | C | 831(83.60) | 845(84.16) | Reference |  |  |  |  |
|   | Dominant model | CC | 351(70.62) | 359(71.51) | 0.756 |  | 0.958(0.728,1.259) |  |  |
|   |   | CT + TT | 146(29.38) | 143(28.49) | Reference |  |  |  |  |
|   | Recessive model | TT | 17(3.42) | 16(3.19) | 0.837 |  | 1.076(0.537,2.154) |  |  |
|   |   | CT + CC | 480(96.58) | 486(96.81) | Reference |  |  |  |  |
|   | Over-dominant model | CC + TT | 368(74.04) | 375(74.7) | 0.812 | 0.966(0.727,1.284) |  |  |  |
|   |   | CT | 129(25.96) | 127(25.3) | Reference |  |  |  |  |
| rs469783 | Genotype | TT | 119(23.94) | 107(21.31) | 0.282 |  | 1.213(0.853,1.725) |  |  |
|   |   | CT | 246(49.50) | 251(50.00) | 0.656 |  | 1.069(0.796,1.435) |  |  |
|   |   | CC | 132(26.56) | 144(28.69) | Reference |  |  |  |  |
|   | Allele | T | 484(48.69) | 465(46.31) | 0.287 |  | 1.100(0.923,1.311) |  |  |
|   |   | C | 510(51.31) | 539(53.69) | Reference |  |  |  |  |
|   | Dominant model | CC | 132(26.56) | 144(28.69) | 0.453 |  | 0.899(0.681,1.187) |  |  |
|   |   | CT + TT | 365(73.44) | 358(71.31) | Reference |  |  |  |  |
|   | Recessive model | TT | 119(23.94) | 107(21.31) | 0.321 |  | 1.162(0.864,1.564) |  |  |
|   |   | CT + CC | 378(76.06) | 395(78.69) | Reference |  |  |  |  |
|   | Over-dominant model | CC + TT | 251(50.5) | 251(50) | 0.874 |  | 1.010(0.893,1.143) |  |  |
|   |   | CT | 246(49.5) | 251(50) | Reference |  |  |  |  |
| ERAP2 |  |  |  |  |  |  |  |  |  |
| rs17524572 | Genotype | TT | 14(2.82) | 11(2.19) | 0.478 |  | 1.337(0.599,2.982) |  |  |
|   |   | CT | 107(21.53) | 96(19.12) | 0.318 |  | 1.171(0.859,1.596) |  |  |
|   |   | CC | 376(75.65) | 395(78.69) | Reference |  |  |  |  |
|   | Allele | T | 135(13.58) | 118(11.75) | 0.219 |  | 1.180(0.906,1.537) |  |  |
|   |   | C | 859(86.42) | 886(88.25) | Reference |  |  |  |  |
|   | Dominant model | CC | 376(75.65) | 395(78.69) | 0.254 |  | 0.842(0.626,1.132) |  |  |
|   |   | CT + TT | 121(24.35) | 107(21.31) | Reference |  |  |  |  |
|   | Recessive model | TT | 14(2.82) | 11(2.19) | 0.528 |  | 1.294(0.582,2.878) |  |  |
|   |   | CT + CC | 483(97.18) | 491(97.81) | Reference |  |  |  |  |
|   | Over-dominant model | CC + TT | 390(78.47) | 406(80.88) | 0.345 |  |  | 0.862(0.633,1.173) |  |
|   |   | CT | 107(21.53) | 96(19.12) | Reference |  |  |  |  |
| rs117041256 | Genotype | GG | 5(1.01) | 3(0.60) | 0.441 |  | 1.760(0.418,7.411) |  |  |
|   |   | TG | 81(16.30) | 65(12.95) | 0.128 |  | 1.316(0.924,1.873) |  |  |
|   |   | TT | 411(82.70) | 434(86.45) | Reference |  |  |  |  |
|   | Allele | G | 91(9.15) | 71(7.07) | 0.089 |  | 1.324(0.958,1.830) |  |  |
|   |   | T | 903(90.85) | 933(92.93) | Reference |  |  |  |  |
|   | Dominant model | TT | 411(82.70) | 434(86.45) | 0.101 |  | 0.749(0.530,1.058) |  |  |
|   |   | TG + GG | 86(17.30) | 68(13.55) | Reference |  |  |  |  |
|   | Recessive model | GG | 5(1.01) | 3(0.60) | 0.474 |  | 1.690(0.402,7.112) |  |  |
|   |   | TG + TT | 492(98.99) | 499(99.40) | Reference |  |  |  |  |
|   | Over-dominant model | GG + TT | 416(83.7) | 437(87.05) | 0.135 |  |  |  | 0.764(0.537,1.087) |
|   |   | TG | 81(16.3) | 65(12.95) | Reference |  |  |  |  |
| rs1230358 | Genotype | TT | 54(10.87) | 40(7.97) | 0.143 |  | 1.395(0.893,2.179) |  |  |
|   |   | TG | 204(41.05) | 215(42.83) | 0.883 |  | 0.981(0.755,1.274) |  |  |
|   |   | GG | 239(48.09) | 247(49.20) | Reference |  |  |  |  |
|   | Allele | T | 312(31.39) | 295(29.38) | 0.330 |  | 1.099(0.909,1.331) |  |  |
|   |   | G | 682(68.61) | 709(70.62) | Reference |  |  |  |  |
|   | Dominant model | GG | 239(48.09) | 247(49.20) | 0.725 |  | 0.956(0.746,1.226) |  |  |
|   |   | TG + TT | 258(51.91) | 255(50.80) | Reference |  |  |  |  |
|   | Recessive model | TT | 54(10.87) | 40(7.97) | 0.118 |  | 1.408(0.917,2.162) |  |  |
|   |   | TG + GG | 443(89.13) | 462(92.03) | Reference |  |  |  |  |
|   | Over-dominant model | GG + TT | 293(58.95) | 287(57.17) | 0.568 |  |  | 1.076(0.836,1.383) |  |
|   |   | TG | 204(41.05) | 215(42.83) | Reference |  |  |  |  |
| rs2549782 | Genotype | GG | 96(19.32) | 65(12.95) | 0.011 |  | 1.643(1.123,2.404) |  |  |
|   |   | GT | 241(48.49) | 259(51.59) | 0.806 |  | 1.035(0.785,1.365) |  |  |
|   |   | TT | 160(32.19) | 178(35.46) | Reference |  |  |  |  |
|   | Allele | G | 433(43.56) | 389(38.75) | 0.029 |  | 1.220(1.021,1.459) |  |  |
|   |   | T | 561(56.44) | 615(61.25) | Reference |  |  |  |  |
|   | Dominant model | TT | 160(32.19) | 178(35.46) | 0.276 |  | 0.864(0.665,1.123) |  |  |
|   |   | GT + GG | 337(67.81) | 324(64.54) | Reference |  |  |  |  |
|   | Recessive model | GG | 96(19.32) | 65(12.95) | 0.006 |  | 1.610(1.142,2.268) |  |  |
|   |   | GT + TT | 401(80.68) | 437(87.05) | Reference |  |  |  |  |
|   | Over-dominant model | GG + TT | 256(51.51) | 243(48.41) | 0.327 |  | 1.132(0.883,1.451) |  |  |
|   |   | GT | 241(48.49) | 259(51.59) | Reference |  |  |  |  |
| rs2549794 | Genotype | CC | 45(9.05) | 34(6.77) | 0.283 |  | 1.299(0.806,2.092) |  |  |
|   |   | CT | 186(37.42) | 207(41.24) | 0.345 |  | 0.882(0.679,1.145) |  |  |
|   |   | TT | 266(53.52) | 261(51.99) | Reference |  |  |  |  |
|   | Allele | C | 276(27.77) | 275(27.39) | 0.851 |  | 1.019(0.837,1.240) |  |  |
|   |   | T | 718(72.23) | 729(72.61) | Reference |  |  |  |  |
|   | Dominant model | TT | 266(53.52) | 261(51.99) | 0.628 |  | 1.063(0.829,1.363) |  |  |
|   |   | CT + CC | 231(46.48) | 241(48.01) | Reference |  |  |  |  |
|   | Recessive model | CC | 45(9.05) | 34(6.77) | 0.183 |  | 1.370(0.862,2.179) |  |  |
|   |   | CT + TT | 452(90.95) | 468(93.23) | Reference |  |  |  |  |
|   | Over-dominant model | CT | 186(37.42) | 207(41.24) | 0218 |  | 1.173(0.910,1.513) |  |  |
|   |   | CC + TT | 311(62.58) | 295(58.76) | Reference |  |  |  |  |

Table 1 Caption: The frequency distribution of ERAP1, ERAP2 genes genotype and allele among PTB patients and controls

We found that the frequency distribution of *ERAP2* gene rs2549782 allele and genotype were different between PTB patients and controls, and the GG genotype, G allele frequencies were both increased in PTB patients when compared to controls (GG versus TT: *P* = 0.011; G versus T: *P* = 0.029). In addition, the GG genotype of rs2549782 polymorphism was a risk factor for PTB compared to the GT/TT genotype under recessive model (GG versus GT + TT: *P* = 0.006). Other SNPs in *ERAP2* gene, including rs17524572, rs1230358, rs2549794 and rs117041256, were not associated with the risk of PTB. Similarly, the rs13167972, rs17086651, rs26618, rs469783, rs3734016 polymorphism in *ERAP1* gene showed certain difference in allelic and genotypic frequencies among PTB patients and controls, while no statistically significant difference was observed.

### Association between ERAP1, ERAP2 allele, genotype frequency and clinical manifestations of PTB patients

To examine the potential association between these SNPs and several clinical manifestations in patients with PTB, stratified analyses were conducted (Table [2](#Tab2)). In *ERAP2* gene, rs17524572 TT genotype, T allele frequencies were significantly increased in PTB patients with DILI in comparison to those without DILI (*P* = 0.006, *P* = 0.044). This study also found a statistically significant association between rs1230358, rs2549782, rs2549794 polymorphisms and the occurrence of fever in PTB patients. The elevated frequencies of rs1230358 GG genotype, G allele, rs2549794 TT genotype, T allele were found in PTB patients with fever (*P* = 0.017, *P* = 0.040, *P* = 0.040, *P* = 0.022). The lower frequencies of GG genotype and G allele in rs2549782 in connection with fever among PTB patients (*P* = 0.025, *P* = 0.009). No significant correlation was observed between *ERAP1* gene polymorphism and the clinical manifestations of patients with PTB.

### Table 2.

| SNP | Allele | Clinical features |  | Group | Genotype |  |  | P value | Allele |  | P value |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| (Mm) | MM | Mm | mm | M | m |  |  |  |  |  |  |
| ERAP1 |  |  |  |  |  |  |  |  |  |  |  |
| rs17086651 | TC | fever |  | + | 52(62.65) | 23(27.71) | 8(9.64) | 0.292 | 127(76.51) | 39(23.49) | 0.818 |
|   |   |   |  | - | 239(57.87) | 147(35.59) | 27(6.54) |   | 625(75.67) | 201(24.33) |   |
|   |   | drug resistance |  | + | 41(55.41) | 26(35.14) | 7(9.46) | 0.641 | 108(72.97) | 40(27.03) | 0.383 |
|   |   |   |  | - | 250(59.24) | 144(34.12) | 28(6.64) |   | 644(76.3) | 200(23.7) |   |
|   |   | DILI |  | + | 38(55.88) | 25(36.76) | 5(7.35) | 0.880 | 101(74.26) | 35(25.74) | 0.651 |
|   |   |   |  | - | 253(59.11) | 145(33.88) | 30(7.01) |   | 651(76.05) | 205(23.95) |   |
|   |   | pulmonary infection |  | + | 43(70.49) | 15(24.59) | 3(4.92) | 0.135 | 101(82.79) | 21(17.21) | 0.055 |
|   |   |   |  | - | 248(57.01) | 155(35.63) | 32(7.36) |   | 651(74.83) | 219(25.17) |   |
|   |   | hypoproteinemia |  | + | 27(65.85) | 10(24.39) | 4(9.76) | 0.345 | 64(78.05) | 18(21.95) | 0.621 |
|   |   |   |  | - | 264(58.02) | 160(35.16) | 31(6.81) |   | 688(75.6) | 222(24.4) |   |
|   |   | sputum smear-positive |  | + | 88(62.86) | 42(30) | 10(7.14) | 0.403 | 218(77.86) | 62(22.14) | 0.315 |
|   |   |   |  | - | 177(56.55) | 114(36.42) | 22(7.03) |   | 468(74.76) | 158(25.24) |   |
| rs13167972 | TC | fever |  | + | 68(80) | 17(20) | 0(0) | 0.499 | 153(90) | 17(10) | 0.977 |
|   |   |   |  | - | 336(81.36) | 72(17.43) | 5(1.21) |   | 744(90.07) | 82(9.93) |   |
|   |   | drug resistance |  | + | 64(86.49) | 9(12.16) | 1(1.35) | 0.361 | 137(92.57) | 11(7.43) | 0.262 |
|   |   |   |  | - | 338(80.09) | 80(18.96) | 4(0.95) |   | 756(89.57) | 88(10.43) |   |
|   |   | DILI |  | + | 55(80.88) | 13(19.12) | 0(0) | 0.652 | 123(90.44) | 13(9.56) | 0.860 |
|   |   |   |  | - | 347(81.07) | 76(17.76) | 5(1.17) |   | 770(89.95) | 86(10.05) |   |
|   |   | pulmonary infection |  | + | 45(73.77) | 16(26.23) | 0(0) | 0.148 | 106(86.89) | 16(13.11) | 0.217 |
|   |   |   |  | - | 357(82.07) | 73(16.78) | 5(1.15) |   | 787(90.46) | 83(9.54) |   |
|   |   | hypoproteinemia |  | + | 34(82.93) | 7(17.07) | 0(0) | 0.783 | 75(91.46) | 7(8.54) | 0.649 |
|   |   |   |  | - | 368(80.88) | 82(18.02) | 5(1.1) |   | 818(89.89) | 92(10.11) |   |
|   |   | sputum smear-positive |  | + | 117(83.57) | 21(15) | 2(1.43) | 0.481 | 255(91.07) | 25(8.93) | 0.414 |
|   |   |   |  | - | 249(79.55) | 61(19.49) | 3(0.96) |   | 559(89.3) | 67(10.7) |   |
| rs26618 | TC | fever |  | + | 55(66.27) | 21(25.3) | 7(8.43) | 0.051 | 131(78.92) | 35(21.08) | 0.161 |
|   |   |   |  | - | 223(54) | 163(39.47) | 27(6.54) |   | 609(73.73) | 217(26.27) |   |
|   |   | drug resistance |  | + | 38(51.35) | 30(40.54) | 6(8.11) | 0.664 | 106(71.62) | 42(28.38) | 0.367 |
|   |   |   |  | - | 240(56.87) | 154(36.49) | 28(6.64) |   | 634(75.12) | 210(24.88) |   |
|   |   | DILI |  | + | 38(55.88) | 26(38.24) | 4(5.88) | 0.934 | 102(75) | 34(25) | 0.907 |
|   |   |   |  | - | 240(56.07) | 158(36.92) | 30(7.01) |   | 638(74.53) | 218(25.47) |   |
|   |   | pulmonary infection |  | + | 35(57.38) | 19(31.15) | 7(11.48) | 0.240 | 89(72.95) | 33(27.05) | 0.656 |
|   |   |   |  | - | 243(55.86) | 165(37.93) | 27(6.21) |   | 651(74.83) | 219(25.17) |   |
|   |   | hypoproteinemia |  | + | 25(60.98) | 13(31.71) | 3(7.32) | 0.757 | 63(76.83) | 19(23.17) | 0.628 |
|   |   |   |  | - | 253(55.6) | 171(37.58) | 31(6.81) |   | 677(74.4) | 233(25.6) |   |
|   |   | sputum smear-positive |  | + | 80(57.14) | 50(35.71) | 10(7.14) | 0.865 | 210(75) | 70(25) | 0.703 |
|   |   |   |  | - | 171(54.63) | 120(38.34) | 22(7.03) |   | 462(73.8) | 164(26.2) |   |
| rs3734016 | CT | fever |  | + | 64(77.11) | 17(20.48) | 2(2.41) | 0.354 | 145(87.35) | 21(12.65) | 0.150 |
|   |   |   |  | - | 286(69.25) | 112(27.12) | 15(3.63) |   | 684(82.81) | 142(17.19) |   |
|   |   | drug resistance |  | + | 50(67.57) | 21(28.38) | 3(4.05) | 0.819 | 121(81.76) | 27(18.24) | 0.519 |
|   |   |   |  | - | 300(71.09) | 108(25.59) | 14(3.32) |   | 708(83.89) | 136(16.11) |   |
|   |   | DILI |  | + | 44(64.71) | 19(27.94) | 5(7.35) | 0.133 | 107(78.68) | 29(21.32) | 0.097 |
|   |   |   |  | - | 306(71.5) | 110(25.7) | 12(2.8) |   | 722(84.35) | 134(15.65) |   |
|   |   | pulmonary infection |  | + | 45(73.77) | 13(21.31) | 3(4.92) | 0.565 | 103(84.43) | 19(15.57) | 0.785 |
|   |   |   |  | - | 305(70.11) | 116(26.67) | 14(3.22) |   | 726(83.45) | 144(16.55) |   |
|   |   | hypoproteinemia |  | + | 30(73.17) | 10(24.39) | 1(2.44) | 0.898 | 70(85.37) | 12(14.63) | 0.647 |
|   |   |   |  | - | 320(70.33) | 119(26.15) | 16(3.52) |   | 759(83.41) | 151(16.59) |   |
|   |   | sputum smear-positive |  | + | 103(73.57) | 32(22.86) | 5(3.57) | 0.583 | 238(85) | 42(15) | 0.399 |
|   |   |   |  | - | 216(69.01) | 86(27.48) | 11(3.51) |   | 518(82.75) | 108(17.25) |   |
| rs469783 | CT | fever |  | + | 28(33.73) | 36(43.37) | 19(22.89) | 0.240 | 92(55.42) | 74(44.58) | 0.234 |
|   |   |   |  | - | 103(24.94) | 210(50.85) | 100(24.21) |   | 416(50.36) | 410(49.64) |   |
|   |   | drug resistance |  | + | 17(22.97) | 39(52.7) | 18(24.32) | 0.755 | 73(49.32) | 75(50.68) | 0.619 |
|   |   |   |  | - | 114(27.01) | 207(49.05) | 101(23.93) |   | 435(51.54) | 409(48.46) |   |
|   |   | DILI |  | + | 15(22.06) | 37(54.41) | 16(23.53) | 0.625 | 67(49.26) | 69(50.74) | 0.625 |
|   |   |   |  | - | 116(27.1) | 209(48.83) | 103(24.07) |   | 441(51.52) | 415(48.48) |   |
|   |   | pulmonary infection |  | + | 14(22.95) | 28(45.9) | 19(31.15) | 0.369 | 56(45.9) | 66(54.1) | 0.210 |
|   |   |   |  | - | 117(26.9) | 218(50.11) | 100(22.99) |   | 452(51.95) | 418(48.05) |   |
|   |   | hypoproteinemia |  | + | 13(31.71) | 20(48.78) | 8(19.51) | 0.652 | 46(56.10) | 36(43.90) | 0.355 |
|   |   |   |  | - | 118(25.93) | 226(49.67) | 111(24.40) |   | 462(50.77) | 448(49.23) |   |
|   |   | sputum smear-positive |  | + | 42(30.00) | 63(45.00) | 35(25.00) | 0.404 | 147(52.50) | 133(47.50) | 0.407 |
|   |   |   |  | - | 76(24.28) | 158(50.48) | 79(25.24) |   | 310(49.52) | 316(50.48) |   |
| ERAP2 |  |  |  |  |  |  |  |  |  |  |  |
| rs17524572 | CT | fever |  | + | 66(79.52) | 16(19.28) | 1(1.2) | 0.504 | 148(89.16) | 18(10.84) | 0.255 |
|   |   |   |  | - | 309(74.82) | 91(22.03) | 13(3.15) |   | 709(85.84) | 117(14.16) |   |
|   |   | drug resistance |  | + | 49(66.22) | 23(31.08) | 2(2.7) | 0.097 | 121(81.76) | 27(18.24) | 0.075 |
|   |   |   |  | - | 326(77.25) | 84(19.91) | 12(2.84) |   | 736(87.2) | 108(12.8) |   |
|   |   | DILI |  | + | 48(70.59) | 14(20.59) | 6(8.82) | 0.006 | 110(80.88) | 26(19.12) | 0.044 |
|   |   |   |  | - | 327(76.4) | 93(21.73) | 8(1.87) |   | 747(87.27) | 109(12.73) |   |
|   |   | pulmonary infection |  | + | 50(81.97) | 9(14.75) | 2(3.28) | 0.382 | 109(89.34) | 13(10.66) | 0.310 |
|   |   |   |  | - | 325(74.71) | 98(22.53) | 12(2.76) |   | 748(85.98) | 122(14.02) |   |
|   |   | hypoproteinemia |  | + | 33(80.49) | 8(19.51) | 0(0) | 0.475 | 74(90.24) | 8(9.76) | 0.288 |
|   |   |   |  | - | 342(75.16) | 99(21.76) | 14(3.08) |   | 783(86.04) | 127(13.96) |   |
|   |   | sputum smear-positive |  | + | 109(77.86) | 28(20) | 3(2.14) | 0.717 | 246(87.86) | 34(12.14) | 0.400 |
|   |   |   |  | - | 233(74.44) | 71(22.68) | 9(2.88) |   | 537(85.78) | 89(14.22) |   |
| rs117041256 | TG | fever |  | + | 71(85.54) | 12(14.46) | 0(0) | 0.517 | 154(92.77) | 12(7.23) | 0.342 |
|   |   |   |  | - | 339(82.08) | 69(16.71) | 5(1.21) |   | 747(90.44) | 79(9.56) |   |
|   |   | drug resistance |  | + | 56(75.68) | 18(24.32) | 0(0) | 0.091 | 130(87.84) | 18(12.16) | 0.172 |
|   |   |   |  | - | 354(83.89) | 63(14.93) | 5(1.18) |   | 771(91.35) | 73(8.65) |   |
|   |   | DILI |  | + | 55(80.88) | 13(19.12) | 0(0) | 0.549 | 123(90.44) | 13(9.56) | 0.867 |
|   |   |   |  | - | 355(82.94) | 68(15.89) | 5(1.17) |   | 778(90.89) | 78(9.11) |   |
|   |   | pulmonary infection |  | + | 56(91.8) | 4(6.56) | 1(1.64) | 0.080 | 116(95.08) | 6(4.92) | 0.082 |
|   |   |   |  | - | 354(81.38) | 77(17.7) | 4(0.92) |   | 785(90.23) | 85(9.77) |   |
|   |   | hypoproteinemia |  | + | 35(85.37) | 6(14.63) | 0(0) | 0.752 | 76(92.68) | 6(7.32) | 0.543 |
|   |   |   |  | - | 375(82.42) | 75(16.48) | 5(1.1) |   | 825(90.66) | 85(9.34) |   |
|   |   | sputum smear-positive |  | + | 118(84.29) | 20(14.29) | 2(1.43) | 0.715 | 256(91.43) | 24(8.57) | 0.681 |
|   |   |   |  | - | 257(82.11) | 53(16.93) | 3(0.96) |   | 567(90.58) | 59(9.42) |   |
| rs1230358 | GT | fever |  | + | 51(61.45) | 23(27.71) | 9(10.84) | 0.017 | 125(75.3) | 41(24.7) | 0.040 |
|   |   |   |  | - | 187(45.28) | 181(43.83) | 45(10.9) |   | 555(67.19) | 271(32.81) |   |
|   |   | drug resistance |  | + | 36(48.65) | 32(43.24) | 6(8.11) | 0.698 | 104(70.27) | 44(29.73) | 0.625 |
|   |   |   |  | - | 202(47.87) | 172(40.76) | 48(11.37) |   | 576(68.25) | 268(31.75) |   |
|   |   | DILI |  | + | 33(48.53) | 28(41.18) | 7(10.29) | 0.985 | 94(69.12) | 42(30.88) | 0.878 |
|   |   |   |  | - | 205(47.9) | 176(41.12) | 47(10.98) |   | 586(68.46) | 270(31.54) |   |
|   |   | pulmonary infection |  | + | 26(42.62) | 27(44.26) | 8(13.11) | 0.638 | 79(64.75) | 43(35.25) | 0.335 |
|   |   |   |  | - | 212(48.74) | 177(40.69) | 46(10.57) |   | 601(69.08) | 269(30.92) |   |
|   |   | hypoproteinemia |  | + | 21(51.22) | 15(36.59) | 5(12.2) | 0.822 | 57(69.51) | 25(30.49) | 0.844 |
|   |   |   |  | - | 217(47.69) | 189(41.54) | 49(10.77) |   | 623(68.46) | 287(31.54) |   |
|   |   | sputum smear-positive |  | + | 70(50) | 51(36.43) | 19(13.57) | 0.374 | 191(68.21) | 89(31.79) | 0.961 |
|   |   |   |  | - | 146(46.65) | 134(42.81) | 33(10.54) |   | 426(68.05) | 200(31.95) |   |
| rs2549782 | TG | fever |  | + | 37(44.58) | 35(42.17) | 11(13.25) | 0.025 | 109(65.66) | 57(34.34) | 0.009 |
|   |   |   |  | - | 123(29.78) | 205(49.64) | 85(20.58) |   | 451(54.6) | 375(45.4) |   |
|   |   | drug resistance |  | + | 26(35.14) | 30(40.54) | 18(24.32) | 0.295 | 82(55.41) | 66(44.59) | 0.781 |
|   |   |   |  | - | 134(31.75) | 210(49.76) | 78(18.48) |   | 478(56.64) | 366(43.36) |   |
|   |   | DILI |  | + | 20(29.41) | 34(50) | 14(20.59) | 0.86 | 74(54.41) | 62(45.59) | 0.606 |
|   |   |   |  | - | 140(32.71) | 206(48.13) | 82(19.16) |   | 486(56.78) | 370(43.22) |   |
|   |   | pulmonary infection |  | + | 21(34.43) | 31(50.82) | 9(14.75) | 0.623 | 73(59.84) | 49(40.16) | 0.421 |
|   |   |   |  | - | 139(31.95) | 209(48.05) | 87(20) |   | 487(55.98) | 383(44.02) |   |
|   |   | hypoproteinemia |  | + | 15(36.59) | 18(43.9) | 8(19.51) | 0.800 | 48(58.54) | 34(41.46) | 0.691 |
|   |   |   |  | - | 145(31.87) | 222(48.79) | 88(19.34) |   | 512(56.26) | 398(43.74) |   |
|   |   | sputum smear-positive |  | + | 53(37.86) | 59(42.14) | 28(20) | 0.201 | 165(58.93) | 115(41.07) | 0.229 |
|   |   |   |  | - | 93(29.71) | 156(49.84) | 64(20.45) |   | 342(54.63) | 284(45.37) |   |
| rs2549794 | TC | fever |  | + | 55(66.27) | 22(26.51) | 6(7.23) | 0.040 | 132(79.52) | 34(20.48) | 0.022 |
|   |   |   |  | - | 211(51.09) | 163(39.47) | 39(9.44) |   | 585(70.82) | 241(29.18) |   |
|   |   | drug resistance |  | + | 42(56.76) | 23(31.08) | 9(12.16) | 0.372 | 107(72.30) | 41(27.70) | 0.996 |
|   |   |   |  | - | 224(53.08) | 162(38.39) | 36(8.53) |   | 610(72.27) | 234(27.73) |   |
|   |   | DILI |  | + | 35(51.47) | 28(41.18) | 5(7.35) | 0.725 | 98(72.06) | 38(27.94) | 0.951 |
|   |   |   |  | - | 231(53.97) | 157(36.68) | 40(9.35) |   | 619(72.31) | 237(27.69) |   |
|   |   | pulmonary infection |  | + | 29(47.54) | 26(42.62) | 6(9.84) | 0.590 | 84(68.85) | 38(31.15) | 0.367 |
|   |   |   |  | - | 237(54.48) | 159(36.55) | 39(8.97) |   | 633(72.76) | 237(27.24) |   |
|   |   | hypoproteinemia |  | + | 21(51.22) | 18(43.90) | 2(4.88) | 0.487 | 60(73.17) | 22(26.83) | 0.850 |
|   |   |   |  | - | 245(53.85) | 167(36.70) | 43(9.45) |   | 657(72.20) | 253(27.80) |   |
|   |   | sputum smear-positive |  | + | 77(55.00) | 49(35.00) | 14(10.00) | 0.794 | 203(72.50) | 77(27.50) | 0.699 |
|   |   |   |  | - | 163(52.08) | 120(38.34) | 30(9.58) |   | 446(71.25) | 180(28.75) |   |

Table 2 Caption: The relationship between ERAP1, ERAP2 genes polymorphisms and the clinical manifestations in patients with PTB

### Haplotype analysis

The haplotype of *ERAP1* gene rs17086651-s13167972-rs26618-rs3734016-rs469783 and *ERAP2* gene rs17524572-rs117041256-rs1230358-rs2549782-rs2549794 were constructed using SHEsis software. We included some main haplotype with a frequency of more than 3% for the analysis. In *ERAP1* gene, six main haplotypes (CACCT, CATCC, CATTT, TACCT, TATCC, TGTCT) were detected, and there was no statistically significant association between the frequency distribution of these haplotypes and susceptibility to PTB (Table [3](#Tab3)). In *ERAP2* gene, five main haplotypes (CTGGT, CTGTT, CTTGC, TGGGT, TTGTT) were detected, and the frequency of CTGGT haplotype was higher in PTB patients (*P* = 0.038), while the frequency of CTGTT haplotype was lower (*P* = 0.016) (Table [3](#Tab3)).

### Table 3.

| Haplotype | PTB patients | Controls | P value | OR (95% CI) |
| --- | --- | --- | --- | --- |
| ERAP1 rs17086651-rs13167972-rs26618-rs3734016-rs469783 |  |  |  |  |
| CACCT | 30.71(0.031) | 25.64(0.026) | 0.446 | 1.229(0.722,2.094) |
| CATCC | 57.33(0.058) | 73.19(0.073) | 0.187 | 0.786(0.549,1.124) |
| CATTT | 148.75(0.150) | 144.59(0.144) | 0.652 | 1.059(0.825,1.359) |
| TACCT | 208.46(0.210) | 194.84(0.194) | 0.321 | 1.118(0.897,1.393) |
| TATCC | 427.11(0.430) | 445.97(0.444) | 0.640 | 0.958(0.799,1.148) |
| TGTCT | 59.70(0.060) | 66.68(0.066) | 0.599 | 0.908(0.632,1.303) |
| ERAP2 rs17524572-rs117041256-rs1230358-rs2549782-rs2549794 |  |  |  |  |
| CTGGT | 66.09(0.066) | 45.85(0.046) | 0.038 | 1.506(1.022,2.222) |
| CTGTT | 484.51(0.487) | 545.45(0.543) | 0.016 | 0.796(0.661,0.958) |
| CTTGC | 252.42(0.254) | 245.17(0.244) | 0.526 | 1.069(0.870,1.314) |
| TGGGT | 59.70(0.060) | 46.86(0.047) | 0.166 | 1.320(0.890,1.958) |

Table 3 Caption: Haplotype analysis of ERAP1, ERAP2 genes in PTB patients and controls

Subsequently, we conducted an analysis to explore the potential correlations between the main haplotypes and several clinical manifestations (fever, drug resistance, DILI, pulmonary infection, hypoproteinemia, sputum smear-positive) in patients with PTB (Table [S3](#MOESM1)-[S8](#MOESM1)). In *ERAP1* gene, the findings revealed a significantly reduced frequency of the CATCC haplotype among PTB patients with pulmonary infection as compared to the patients without pulmonary infection (*P* = 0.024) (Table [S6](#MOESM1)). In *ERAP2* gene, our analysis indicated that the frequency of CTTGC haplotype was decreased in PTB patients with fever (*P* = 0.046), while the frequency of TTGTT haplotype was increased in PTB patients with DILI (*P* = 0.027) (Table [S3](#MOESM1), [S5](#MOESM1)).

### The methylation levels of ERAP1, ERAP2 genes in PTB patients and normal controls

We included 98 PTB patients (53 males and 45 females) and 97 controls (51 males and 46 females) to compare the differences in the methylation levels of *ERAP1*, *ERAP2* genes. The mean age of PTB patients and controls were 43.52 ± 17.59 years and 43.45 ± 4.49 years. The age and gender distributions were similar between these two groups (*P* > 0.05). This study detected the methylation levels of two specific target fragments in *ERAP1* gene (ERAP1_1, ERAP1_2), one specific target fragment in *ERAP2* gene (ERAP2_1) among PTB patients and controls (Table [4](#Tab4)). When compared to controls, the ERAP1_1, ERAP1_2 methylation levels were significantly decreased in PTB patients (*P*<0.001, *P* = 0.002), while the ERAP2_1 methylation level was significantly increased (*P*<0.001). Then, the methylation levels of *ERAP1*, *ERAP2* genes were obtained through measuring the average methylation level of all CpG sites on the fragments. We found that *ERAP1* methylation level was significantly lower in PTB patients that in controls (*P*<0.001), and *ERAP2* methylation level was significantly increased (*P*<0.001) (Fig. [1](#Fig1)).

### Table 4.

| Specific fragments | PTB patients (n = 98) | Controls (n = 97) | P value |
| --- | --- | --- | --- |
| ERAP1_1 | 0.0064(0.0061,0.0067) | 0.0067(0.0063,0.0071) | < 0.001 |
| ERAP1_2 | 0.0068(0.0065,0.0072) | 0.0071(0.0067,0.0076) | 0.002 |
| ERAP2_1 | 0.0128(0.0106,0.0156) | 0.0101(0.0087,0.0123) | < 0.001 |

Table 4 Caption: The methylation levels of specific target fragments in ERAP1, ERAP2 genes among PTB patients and controls

### Fig. 1.

![Fig. 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/93b1/11800520/0a1b40196c19/12879_2025_10452_Fig1_HTML.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=11800520_12879_2025_10452_Fig1_HTML.jpg)

The methylation levels of ERAP1, ERAP2 in PTB patients and normal controls

Moreover, we also included an analysis of DNA methylation haplotypes in relation to PTB. Our investigation revealed an elevation in the methylation levels of two distinct DNA methylation haplotypes (ERAP1_1 TTTTTTTTTTTTTTTT, ERAP1_2 TTTTTTTTTTTTTTT) among PTB patients (Table [S9](#MOESM1)). The possible association between *ERAP1*, *ERAP2* methylation levels and several clinical manifestations was assessed in PTB patients (Table [5](#Tab5)). The results showed that the increased *ERAP1* methylation level was significantly associated with DILI in PTB patients (*P* = 0.006), and the elevated *ERAP2* methylation level was respectively significantly associated with fever, hypoproteinemia, sputum smear-positive (*P* = 0.025, *P* = 0.028, *P* = 0.022).

### Table 5.

| Group | +- | N | ERAP1 methylation level | P value | ERAP2 methylation level | P value |
| --- | --- | --- | --- | --- | --- | --- |
| fever | + | 16 | 0.0065(0.0061,0.0066) | 0.115 | 0.0155(0.0113,0.0175) | 0.025 |
|   | - | 82 | 0.0066(0.0064,0.0070) |   | 0.0124(0.0104,0.0153) |   |
| drug resistance | + | 12 | 0.0065(0.0060,0.0068) | 0.190 | 0.0115(0.0097,0.0146) | 0.225 |
|   | - | 86 | 0.0066(0.0064,0.0069) |   | 0.0129(0.0106,0.0157) |   |
| DILI | + | 13 | 0.0069(0.0065,0.0071) | 0.006 | 0.0109(0.0082,0.0162) | 0.150 |
|   | - | 85 | 0.0065(0.0063,0.0068) |   | 0.0129(0.0107,0.0156) |   |
| pulmonary infection | + | 12 | 0.0065(0.0064,0.0070) | 0.641 | 0.0120(0.0108,0.0178) | 0.573 |
|   | - | 86 | 0.0066(0.0063,0.0069) |   | 0.0128(0.0105,0.0155) |   |
| hypoproteinemia | + | 4 | 0.0067(0.0065,0.0071) | 0.389 | 0.0175(0.0138,0.0229) | 0.028 |
|   | - | 94 | 0.0065(0.0063,0.0069) |   | 0.0126(0.0105,0.0154) |   |
| sputum smear-positive | + | 34 | 0.0065(0.0063,0.0068) | 0.488 | 0.0142(0.0118,0.0166) | 0.022 |
|   | - | 54 | 0.0066(0.0063,0.0069) |   | 0.0114(0.0099,0.0154) |   |

Table 5 Caption: Association between ERAP1, ERAP2 methylation levels and clinical characteristics in patients with PTB

### Associations between ERAP1, ERAP2 genes variation with their methylation levels

In this study, the effects of *ERAP1*, *ERAP2* genes polymorphisms on their methylation levels were also examined in PTB patients. Our results suggested that *ERAP2* gene rs1230358 showed a significant association of different genotypes with the methylation level of *ERAP2* gene (Table [6](#Tab6)).

### Table 6.

| ERAP1 SNP | Genotype | N | ERAP1 methylation level | P value |
| --- | --- | --- | --- | --- |
| rs17086651 | CC | 10 | 0.0066(0.0060,0.0068) | 0.144 |
|   | TC | 36 | 0.0065(0.0063,0.0068) |   |
|   | TT | 52 | 0.0066(0.0064,0.0070) |   |
| rs13167972 | GG | 2 | 0.0069(0.0066,0.0072) | 0.527 |
|   | AG | 18 | 0.0066(0.0063,0.0070) |   |
|   | AA | 78 | 0.0065(0.0063,0.0069) |   |
| rs26618 | CC | 4 | 0.0067(0.0065,0.0074) | 0.597 |
|   | TC | 38 | 0.0065(0.0063,0.0070) |   |
|   | TT | 56 | 0.0066(0.0063,0.0069) |   |
| rs3734016 | TT | 7 | 0.0067(0.0060,0.0069) | 0.963 |
|   | CT | 26 | 0.0066(0.0061,0.0069) |   |
|   | CC | 65 | 0.0065(0.0064,0.0069) |   |
| rs469783 | CC | 24 | 0.0064(0.0065,0.0069) | 0.999 |
|   | CT | 46 | 0.0066(0.0063,0.0070) |   |
|   | TT | 28 | 0.0066(0.0063,0.0068) |   |
| ERAP2 SNP | Genotype | N | ERAP2 methylation level | P value |
| rs17524572 | TT | 3 | 0.0109(0.0101,0.0110) | 0.327 |
|   | CT | 29 | 0.0127(0.0104,0.0154) |   |
|   | CC | 66 | 0.0129(0.0108,0.0159) |   |
| rs117041256 | GG | 1 | 0.0105 | 0.27 |
|   | TG | 24 | 0.0112(0.0102,0.0153) |   |
|   | TT | 73 | 0.0130(0.0108,0.0159) |   |
| rs1230358 | TT | 14 | 0.0147(0.0118,0.0180) | 0.040 |
|   | TG | 36 | 0.0117(0.0097,0.0150) |   |
|   | GG | 48 | 0.0135(0.0109,0.0154) |   |
| rs2549782 | GG | 26 | 0.0131(0.0114,0.0157) | 0.615 |
|   | GT | 46 | 0.0126(0.0103,0.0157) |   |
|   | TT | 26 | 0.0118(0.0104,0.0158) |   |
| rs2549794 | CC | 11 | 0.0145(0.0106,0.0182) | 0.231 |
|   | CT | 32 | 0.0117(0.0104,0.0153) |   |
|   | TT | 55 | 0.0130(0.0106,0.0155) |   |

Table 6 Caption: The influence of the ERAP1, ERAP2 genotypes on their methylation levels

## Discussion

Antigen presentation system played key roles in the interaction between host resistance and pathogen, and ERAPs (ERAP1 and ERAP2) were closely involved in the antigen presentation process, which were important components of the antigen delivery pathway [[24](#CR24)]. Therefore, ERAPs might be potential targets and regulators for studying the pathogenesis of infectious diseases. In addition, ERAPs exhibited a considerable number of genetic polymorphisms, and several variants had been shown to participate in the progression of the disease by affecting the enzyme activity and selectivity of these proteins [[25](#CR25)–[27](#CR27)]. For example, previous studies suggested that polymorphisms in *ERAPs* showed significant associations with HIV infection [[26](#CR26)], chronic hepatitis C [[27](#CR27)]. To further understand the important significance of *ERAPs* genetic variation in PTB, the present study not only analyzed the relationship between ten SNPs in *ERAP1* and *ERAP2* genes and PTB susceptibility, but also detected their DNA methylation levels in PTB patients for the first time.

ERAP1 was a member of M1 family of metalopeptidases, which had enzymatic function in antigen presentation and inflammatory responses [[28](#CR28)]. The human *ERAP1* gene was located on chromosome 5, with a total length of 47 379 bp and consists of 20 exons. ERAP1 molecule could crystallize into two conformations, including closed conformation and open conformation, and the major differences between the closed conformation and open conformation had a crucial impact on ERAP1 activity. Some SNPs might change the conformation of ERAP1, thus affecting the function of ERAP1 in regulating immune and inflammatory processes [[29](#CR29), [30](#CR30)]. Previous studies demonstrated that *ERAP1* was considered a potentially attractive ankylosing spondylitis (AS) candidate gene, and *ERAP1* rs27044, rs30187 polymorphisms were significantly correlated with AS susceptibility [[30](#CR30)], and several specific haplotypes in *ERAP1* could influence the risk of AS in Iranian population [[31](#CR31)]. Another study suggested that *ERAP1* rs26618 C allele was associated with an increased risk of chronic HCV infection [[27](#CR27)]. Zhang et al. investigated the relationship between *ERAP1* gene polymorphism and PTB, and found that rs26618 was associated with the risk of PTB, while rs2549782, rs1056893 were not related to PTB risk [[19](#CR19)]. In the present study, we found that the relationship between *ERAP1* gene rs13167972, rs17086651, rs469783, rs26618, rs3734016 variants and PTB susceptibility was no statistically significant. This was different from the results by zhang et al. [[19](#CR19)], which might be due to differences in sample size, sample source, experimental method and analysis method, and similar studies were needed to validate the results of this study. PTB patients usually had a variety of clinical manifestations, including fever, liver damage, drug resistance, etc., and the occurrence and development of these clinical manifestations might also be influenced by genetic variation [[32](#CR32)]. Here, we did not find an association between *ERAP1* gene polymorphism and clinical manifestations of PTB. Unfortunately, this study did not provide critical evidence for the involvement of *ERAP1* gene polymorphisms in PTB.

The human *ERAP2* gene was also located on chromosome 5, which was 41,438 bp in length and consists of 19 exons, and was involved in bacterial and viral infections. Regarding infectious diseases, one study found that *ERAP2* gene rs2248374 could arise the risk for chronic HCV [[27](#CR27)], and another study showed that the rs2549782 variant was significantly related to the risk of HIV infection in Italians, and the TT genotype frequency was elevated in the HIV-1 exposure seronegative popualtion [[26](#CR26)]. Analogously, the results by Biasin et al., suggested that rs2549782 variant might be an impotent factor for exposure to HIV-1 infection among intravenous drug users in the Spanish population [[33](#CR33)]. In this study, we found that rs2549782 G allele, GG genotype were both showed a significant association with the increased risk of PTB. In addition, the GT/TT genotype of rs2549782 exhibited a decreased frequency in PTB patients when compared to controls under recessive model. Consistent with our results, Zhang et al. also found that the G allele of rs2549782 was the risk factors for PTB [[19](#CR19)]. In addition, the frequencies of two haplotypes (CTGGT, CTGTT) in *ERAP2* gene were associated with PTB susceptibility. These results provided important evidence for the viewpoint that *ERAP2* gene polymorphism, specifically rs2549782, was a key factor affecting the genetic background of PTB susceptibility. In addition, *ERAP2* gene polymorphism could affect some clinical characteristics of PTB, such as drug resistance [[19](#CR19)]. Our results also revealed that *ERAP2* gene rs17524572, rs1230358, rs2549782, rs2549794 variants were respectively associated with DILI, fever of PTB. Therefore, *ERAP2* gene variation was shown to be involved in the occurrence and development of PTB.

The genetic information existed not only in DNA sequences, but also in epigenetic variation. As an important epigenetic trait, DNA methylation was related to multiple biological processes, and its roles in human diseases had attracted increasing attention [[34](#CR34)]. Wang et al. found that the abnormal methylation levels of several key genes in the vitamin D metabolic pathway were associated with the risk and prognosis of PTB [[35](#CR35)]. Therefore, DNA methylation had become an important direction for further studying the PTB prevention and control strategies. The roles of *ERAPs* genes methylation in human disease had been studied. For example, the *ERAP1* methylation level was highly expressed in patients with AS, and could affect the *ERAP1* mRNA expression level, which was considered to be a therapeutic target for AS [[36](#CR36)]. Here, we found that *ERAP1* methylation level in PTB patients was significantly decreased in comparison to controls, while *ERAP2* methylation level was increased. Moreover, the stratification analysis showed that *ERAP1* methylation level was related to DILI, and *ERAP2* methylation level was associated with fever, hypoproteinemia, sputum smear-positive in PTB patients. These results suggested that *ERAP1*, *ERAP2* genes were involved in the pathogenesis of PTB, and their methylation levels might be used to diagnose PTB and reflected the treatment of PTB patients. Previous study had also reported a certain relationship between DNA methylation and genetic variation [[37](#CR37)]. Here, we noticed an interesting finding that the ERAP2 methylation level was affected by rs1230358 variant in PTB patients. This provided an important basis for further exploring the role of *ERAP2* gene variation in the pathogenesis of PTB.

In summary, *ERAP2* rs2549782 variant was confirmed to be associated with PTB genetic susceptibility, while *ERAP1* genes polymorphisms were not related to the risk of PTB in Chinese population. Moreover, the decreased *ERAP1* methylation level and the increased *ERAP2* methylation level were found in PTB patients. However, several limitations should be recognized in our study. First, the sample size could not be considered large enough, which might affect the power of our results to some extent. Second, this study did not exclude the possible influence of environmental factors and ethnic background. Therefore, in order to verify and explore the precise mechanism of *ERAP1* and *ERAP2* gene in PTB, it was necessary to carry out replication studies with different ethnic populations, larger sample size.

## Electronic supplementary material

Below is the link to the electronic supplementary material.

## Author contributions

QH, HN, Q-HY and G-YS designed the study. QH and S-HS conducted the experiment and performed the statistical analyses. X-QC, Y-HL and A-HX participated in sample collection. QH, HN and S-HS drafted the manuscript. Q-HY and G-YS contributed to the manuscript revision. All authors approved the final submitted version.

## Funding

No.

## Data availability

The data presented in the study are deposited in the dbSNP repository, accession number 1063327. Data is provided within the manuscript or supplementary information files.

## Declarations

### Ethics approval and consent to participate

This study was approved by the Ethical Committee of Anhui Chest Hospital (K2023-011). All the study subjects provided informed consent to participate in this study.

### Consent for publication

Not applicable.

### Competing interests

The authors declare no competing interests.

### Clinical trial number

Not applicable.

## Footnotes

## Contributor Information

Qian Huang, Email: huangxiaoxi201602@163.com.

Hua Niu, Email: nhvssj@sina.com.

Qing-Hai You, Email: amormor@126.com.

Geng-Yun Sun, Email: sungengy@126.com.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

The data presented in the study are deposited in the dbSNP repository, accession number 1063327. Data is provided within the manuscript or supplementary information files.

### Supplementary Materials

### Data Availability Statement

The data presented in the study are deposited in the dbSNP repository, accession number 1063327. Data is provided within the manuscript or supplementary information files.

## References

1. Schrager LK, Vekemens J, Drager N, Lewinsohn DM, Olesen OF. The status of Tuberculosis vaccine development. Lancet Infect Dis. 2020;20(3):e28–37.  [DOI](https://doi.org/10.1016/S1473-3099(19)30625-5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32014117/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Schrager%20LK,%20Vekemens%20J,%20Drager%20N,%20Lewinsohn%20DM,%20Olesen%20OF.%20The%20status%20of%20Tuberculosis%20vaccine%20development.%20Lancet%20Infect%20Dis.%202020;20(3):e28%E2%80%9337.)

2. World Health Organization. Global tuberculosis report 2021. https://www.who.int/publications/digital/global-tuberculosis-report-2021  [https://www.who.int/publications/digital/global-tuberculosis-report-2021](https://www.who.int/publications/digital/global-tuberculosis-report-2021)

3. Lu T, Wang M, Liu N, Zhang S, Shi L, Bao L, Luo F, Shi L, Liu S, Yao Y. Transporter Associated with Antigen Processing 1 gene polymorphisms increase the susceptibility to tuberculosis. Pharmgenomics Pers Med. 2023;16:325–36.  [DOI](https://doi.org/10.2147/PGPM.S404339) | [PMC free article](/articles/PMC10108862/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/37077653/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Lu%20T,%20Wang%20M,%20Liu%20N,%20Zhang%20S,%20Shi%20L,%20Bao%20L,%20Luo%20F,%20Shi%20L,%20Liu%20S,%20Yao%20Y.%20Transporter%20Associated%20with%20Antigen%20Processing%201%20gene%20polymorphisms%20increase%20the%20susceptibility%20to%20tuberculosis.%20Pharmgenomics%20Pers%20Med.%202023;16:325%E2%80%9336.)

4. Möller M, Hoal EG. Current findings, challenges and novel approaches in human genetic susceptibility to tuberculosis. Tuberculosis. 2010;90(2):71–83.  [DOI](https://doi.org/10.1016/j.tube.2010.02.002) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20206579/) | [Google Scholar](https://scholar.google.com/scholar_lookup?M%C3%B6ller%20M,%20Hoal%20EG.%20Current%20findings,%20challenges%20and%20novel%20approaches%20in%20human%20genetic%20susceptibility%20to%20tuberculosis.%20Tuberculosis.%202010;90(2):71%E2%80%9383.)

5. Miyahara R, Smittipat N, Juthayothin T, et al. Risk factors associated with large clusters of tuberculosis patients determined by whole-genome sequencing in a high-tuberculosis-burden country. Tuberculosis. 2020;125:101991.  [DOI](https://doi.org/10.1016/j.tube.2020.101991) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32949880/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Miyahara%20R,%20Smittipat%20N,%20Juthayothin%20T,%20et%20al.%20Risk%20factors%20associated%20with%20large%20clusters%20of%20tuberculosis%20patients%20determined%20by%20whole-genome%20sequencing%20in%20a%20high-tuberculosis-burden%20country.%20Tuberculosis.%202020;125:101991.)

6. Zheng R, Li Z, He F, et al. Genome-wide association study identifies two risk loci for tuberculosis in Han Chinese. Nat Commun. 2018;9(1):4072.  [DOI](https://doi.org/10.1038/s41467-018-06539-w) | [PMC free article](/articles/PMC6172286/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30287856/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Zheng%20R,%20Li%20Z,%20He%20F,%20et%20al.%20Genome-wide%20association%20study%20identifies%20two%20risk%20loci%20for%20tuberculosis%20in%20Han%20Chinese.%20Nat%20Commun.%202018;9(1):4072.)

7. van Tong H, Velavan TP, Thye T, Meyer CG. Human genetic factors in tuberculosis: an update. Trop Med Int Health. 2017;22(9):1063–71.  [DOI](https://doi.org/10.1111/tmi.12923) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28685916/) | [Google Scholar](https://scholar.google.com/scholar_lookup?van%20Tong%20H,%20Velavan%20TP,%20Thye%20T,%20Meyer%20CG.%20Human%20genetic%20factors%20in%20tuberculosis:%20an%20update.%20Trop%20Med%20Int%20Health.%202017;22(9):1063%E2%80%9371.)

8. Fol M, Druszczynska M, Wlodarczyk M, Ograczyk E, Rudnicka W. Immune response gene polymorphisms in tuberculosis. Acta Biochim Pol. 2015;62(4):633–40.  [DOI](https://doi.org/10.18388/abp.2015_1130) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26634232/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Fol%20M,%20Druszczynska%20M,%20Wlodarczyk%20M,%20Ograczyk%20E,%20Rudnicka%20W.%20Immune%20response%20gene%20polymorphisms%20in%20tuberculosis.%20Acta%20Biochim%20Pol.%202015;62(4):633%E2%80%9340.)

9. Liu M, Wang Q, Liu H, Yin C, Mijiti X, Anwaierjiang A, Wan K, Xu M, Li M, Nong S, Li G, Xiao H. Association of Mannose-Binding Lectin 2 gene polymorphism with tuberculosis based on Mycobacterium tuberculosis lineages. Infect Drug Resist. 2022;15:1225–34.  [DOI](https://doi.org/10.2147/IDR.S344935) | [PMC free article](/articles/PMC8959721/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35355619/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Liu%20M,%20Wang%20Q,%20Liu%20H,%20Yin%20C,%20Mijiti%20X,%20Anwaierjiang%20A,%20Wan%20K,%20Xu%20M,%20Li%20M,%20Nong%20S,%20Li%20G,%20Xiao%20H.%20Association%20of%20Mannose-Binding%20Lectin%202%20gene%20polymorphism%20with%20tuberculosis%20based%20on%20Mycobacterium%20tuberculosis%20lineages.%20Infect%20Drug%20Resist.%202022;15:1225%E2%80%9334.)

10. Zhao JW, Jiao L, Guo MM, Zheng L, Wang XB, Gao SH, Ying BW, Ming L. SFTPC genetic polymorphisms are associated with tuberculosis susceptibility and clinical phenotype in a western Chinese Han population. Exp Ther Med. 2020;20(5):100.  [DOI](https://doi.org/10.3892/etm.2020.9230) | [PMC free article](/articles/PMC7507020/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32973949/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Zhao%20JW,%20Jiao%20L,%20Guo%20MM,%20Zheng%20L,%20Wang%20XB,%20Gao%20SH,%20Ying%20BW,%20Ming%20L.%20SFTPC%20genetic%20polymorphisms%20are%20associated%20with%20tuberculosis%20susceptibility%20and%20clinical%20phenotype%20in%20a%20western%20Chinese%20Han%20population.%20Exp%20Ther%20Med.%202020;20(5):100.)

11. Li HM, Wang LJ, Tang F, Pan HF, Zhang TP. Association between Genetic Polymorphisms of lncRNA NEAT1 and pulmonary tuberculosis risk, clinical manifestations in a Chinese Population. Infect Drug Resist. 2022;15:2481–9.  [DOI](https://doi.org/10.2147/IDR.S354863) | [PMC free article](/articles/PMC9109893/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35586561/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Li%20HM,%20Wang%20LJ,%20Tang%20F,%20Pan%20HF,%20Zhang%20TP.%20Association%20between%20Genetic%20Polymorphisms%20of%20lncRNA%20NEAT1%20and%20pulmonary%20tuberculosis%20risk,%20clinical%20manifestations%20in%20a%20Chinese%20Population.%20Infect%20Drug%20Resist.%202022;15:2481%E2%80%939.)

12. Zhang TP, Chen SS, Zhang GY, Shi SJ, Wei L, Li HM. Association of vitamin D pathway genes polymorphisms with pulmonary tuberculosis susceptibility in a Chinese population. Genes Nutr. 2021;16(1):6.  [DOI](https://doi.org/10.1186/s12263-021-00687-3) | [PMC free article](/articles/PMC8061222/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33882819/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Zhang%20TP,%20Chen%20SS,%20Zhang%20GY,%20Shi%20SJ,%20Wei%20L,%20Li%20HM.%20Association%20of%20vitamin%20D%20pathway%20genes%20polymorphisms%20with%20pulmonary%20tuberculosis%20susceptibility%20in%20a%20Chinese%20population.%20Genes%20Nutr.%202021;16(1):6.)

13. Guastafierro T, Bacalini MG, Marcoccia A, Gentilini D, Pisoni S, Di Blasio AM, et al. Genome-wide DNA methylation analysis in blood cells from patients with Werner syndrome. Clin Epigenetics. 2017;9:92.  [DOI](https://doi.org/10.1186/s13148-017-0389-4) | [PMC free article](/articles/PMC5577832/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28861129/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Guastafierro%20T,%20Bacalini%20MG,%20Marcoccia%20A,%20Gentilini%20D,%20Pisoni%20S,%20Di%20Blasio%20AM,%20et%20al.%20Genome-wide%20DNA%20methylation%20analysis%20in%20blood%20cells%20from%20patients%20with%20Werner%20syndrome.%20Clin%20Epigenetics.%202017;9:92.)

14. Sengupta S, Pattanaik KP, Mishra S, Sonawane A. Epigenetic orchestration of host immune defences by Mycobacterium tuberculosis. Microbiol Res. 2023;273:127400.  [DOI](https://doi.org/10.1016/j.micres.2023.127400) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/37196490/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Sengupta%20S,%20Pattanaik%20KP,%20Mishra%20S,%20Sonawane%20A.%20Epigenetic%20orchestration%20of%20host%20immune%20defences%20by%20Mycobacterium%20tuberculosis.%20Microbiol%20Res.%202023;273:127400.)

15. Chang SC, Momburg F, Bhutani N, Goldberg AL. The ER aminopeptidase, ERAP1, trims precursors to lengths of MHC class I peptides by a molecular ruler mechanism. Proc Natl Acad Sci U S A. 2005;102(47):17107–12.  [DOI](https://doi.org/10.1073/pnas.0500721102) | [PMC free article](/articles/PMC1287962/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16286653/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Chang%20SC,%20Momburg%20F,%20Bhutani%20N,%20Goldberg%20AL.%20The%20ER%20aminopeptidase,%20ERAP1,%20trims%20precursors%20to%20lengths%20of%20MHC%20class%20I%20peptides%20by%20a%20molecular%20ruler%20mechanism.%20Proc%20Natl%20Acad%20Sci%20U%20S%20A.%202005;102(47):17107%E2%80%9312.)

16. Chen H, Li L, Weimershaus M, Evnouchidou I, van Endert P, Bouvier M. ERAP1-ERAP2 dimers trim MHC I-bound precursor peptides; implications for understanding peptide editing. Sci Rep. 2016;6:28902.  [DOI](https://doi.org/10.1038/srep28902) | [PMC free article](/articles/PMC4981824/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27514473/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Chen%20H,%20Li%20L,%20Weimershaus%20M,%20Evnouchidou%20I,%20van%20Endert%20P,%20Bouvier%20M.%20ERAP1-ERAP2%20dimers%20trim%20MHC%20I-bound%20precursor%20peptides;%20implications%20for%20understanding%20peptide%20editing.%20Sci%20Rep.%202016;6:28902.)

17. Serwold T, Gonzalez F, Kim J, Jacob R, Shastri N. ERAAP customizes peptides for MHC class I molecules in the endoplasmic reticulum. Nature. 2002;419(6906):480–3.  [DOI](https://doi.org/10.1038/nature01074) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12368856/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Serwold%20T,%20Gonzalez%20F,%20Kim%20J,%20Jacob%20R,%20Shastri%20N.%20ERAAP%20customizes%20peptides%20for%20MHC%20class%20I%20molecules%20in%20the%20endoplasmic%20reticulum.%20Nature.%202002;419(6906):480%E2%80%933.)

18. Gelfman S, Monnet D, Ligocki AJ, Tabary T, Moscati A, Bai X, et al. Regeneron Genetics Center. ERAP1, ERAP2, and two copies of HLA-Aw19 alleles increase the risk for Birdshot Chorioretinopathy in HLA-A29 Carriers. Invest Ophthalmol Vis Sci. 2021;62(14):3.  [DOI](https://doi.org/10.1167/iovs.62.14.3) | [PMC free article](/articles/PMC8572510/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34727153/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Gelfman%20S,%20Monnet%20D,%20Ligocki%20AJ,%20Tabary%20T,%20Moscati%20A,%20Bai%20X,%20et%20al.%20Regeneron%20Genetics%20Center.%20ERAP1,%20ERAP2,%20and%20two%20copies%20of%20HLA-Aw19%20alleles%20increase%20the%20risk%20for%20Birdshot%20Chorioretinopathy%20in%20HLA-A29%20Carriers.%20Invest%20Ophthalmol%20Vis%20Sci.%202021;62(14):3.)

19. Zhang S, Liu S, Liu N, Li C, Wang H, Shi L, Zhang X, Bao L, Yao Y, Shi L. Polymorphisms in ERAP1 and ERAP2 genes are Associated with Tuberculosis in the Han Chinese. Front Genet. 2020;11:566190.  [DOI](https://doi.org/10.3389/fgene.2020.566190) | [PMC free article](/articles/PMC7676896/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33250919/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Zhang%20S,%20Liu%20S,%20Liu%20N,%20Li%20C,%20Wang%20H,%20Shi%20L,%20Zhang%20X,%20Bao%20L,%20Yao%20Y,%20Shi%20L.%20Polymorphisms%20in%20ERAP1%20and%20ERAP2%20genes%20are%20Associated%20with%20Tuberculosis%20in%20the%20Han%20Chinese.%20Front%20Genet.%202020;11:566190.)

20. Salie M, van der Merwe L, Möller M, et al. Associations between human leukocyte antigen class I variants and the Mycobacterium tuberculosis subtypes causing disease. J Infect Dis. 2014;209(2):216–23.  [DOI](https://doi.org/10.1093/infdis/jit443) | [PMC free article](/articles/PMC3873786/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23945374/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Salie%20M,%20van%20der%20Merwe%20L,%20M%C3%B6ller%20M,%20et%20al.%20Associations%20between%20human%20leukocyte%20antigen%20class%20I%20variants%20and%20the%20Mycobacterium%20tuberculosis%20subtypes%20causing%20disease.%20J%20Infect%20Dis.%202014;209(2):216%E2%80%9323.)

21. Yuliwulandari R, Sachrowardi Q, Nakajima H, et al. Association of HLA-A, -B, and -DRB1 with pulmonary tuberculosis in western Javanese Indonesia. Hum Immunol. 2010;71(7):697–701.  [DOI](https://doi.org/10.1016/j.humimm.2010.04.005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20438789/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Yuliwulandari%20R,%20Sachrowardi%20Q,%20Nakajima%20H,%20et%20al.%20Association%20of%20HLA-A,%20-B,%20and%20-DRB1%20with%20pulmonary%20tuberculosis%20in%20western%20Javanese%20Indonesia.%20Hum%20Immunol.%202010;71(7):697%E2%80%93701.)

22. Peng W, Chen H, Zhao Z, et al. TLR1 polymorphisms are significantly associated with the occurrence, presentation and drug-adverse reactions of tuberculosis in western Chinese adults. Oncotarget. 2017;9(2):1691–704.  [DOI](https://doi.org/10.18632/oncotarget.23067) | [PMC free article](/articles/PMC5788591/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29416723/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Peng%20W,%20Chen%20H,%20Zhao%20Z,%20et%20al.%20TLR1%20polymorphisms%20are%20significantly%20associated%20with%20the%20occurrence,%20presentation%20and%20drug-adverse%20reactions%20of%20tuberculosis%20in%20western%20Chinese%20adults.%20Oncotarget.%202017;9(2):1691%E2%80%93704.)

23. Zhang TP, Li R, Wang LJ et al. Impact of m6A demethylase (ALKBH5, FTO) genetic polymorphism and expression levels on the development of pulmonary tuberculosis. Front Cell Infect Microbiol 2022;12:1074380.  [DOI](https://doi.org/10.3389/fcimb.2022.1074380) | [PMC free article](/articles/PMC9817133/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36619747/)

24. Yao Y, Liu N, Zhou Z, Shi L. Influence of ERAP1 and ERAP2 gene polymorphisms on disease susceptibility in different populations. Hum Immunol. 2019;80(5):325–34.  [DOI](https://doi.org/10.1016/j.humimm.2019.02.011) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30797823/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Yao%20Y,%20Liu%20N,%20Zhou%20Z,%20Shi%20L.%20Influence%20of%20ERAP1%20and%20ERAP2%20gene%20polymorphisms%20on%20disease%20susceptibility%20in%20different%20populations.%20Hum%20Immunol.%202019;80(5):325%E2%80%9334.)

25. Evnouchidou I, Birtley J, Seregin S, Papakyriakou A, Zervoudi E, Samiotaki M, et al. A common single nucleotide polymorphism in endoplasmic reticulum aminopeptidase 2 induces a specificity switch that leads to altered antigen processing. J Immunol. 2012;189(5):2383–92.  [DOI](https://doi.org/10.4049/jimmunol.1200918) | [PMC free article](/articles/PMC3530405/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22837489/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Evnouchidou%20I,%20Birtley%20J,%20Seregin%20S,%20Papakyriakou%20A,%20Zervoudi%20E,%20Samiotaki%20M,%20et%20al.%20A%20common%20single%20nucleotide%20polymorphism%20in%20endoplasmic%20reticulum%20aminopeptidase%202%20induces%20a%20specificity%20switch%20that%20leads%20to%20altered%20antigen%20processing.%20J%20Immunol.%202012;189(5):2383%E2%80%9392.)

26. Cagliani R, Riva S, Biasin M, Fumagalli M, Pozzoli U, Lo Caputo S, et al. Genetic diversity at endoplasmic reticulum aminopeptidases is maintained by balancing selection and is associated with natural resistance to HIV-1 infection. Hum Mol Genet. 2010;19(23):4705–14.  [DOI](https://doi.org/10.1093/hmg/ddq401) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20843824/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Cagliani%20R,%20Riva%20S,%20Biasin%20M,%20Fumagalli%20M,%20Pozzoli%20U,%20Lo%20Caputo%20S,%20et%20al.%20Genetic%20diversity%20at%20endoplasmic%20reticulum%20aminopeptidases%20is%20maintained%20by%20balancing%20selection%20and%20is%20associated%20with%20natural%20resistance%20to%20HIV-1%20infection.%20Hum%20Mol%20Genet.%202010;19(23):4705%E2%80%9314.)

27. Liu S, Cao D, Shen Y, Li Y, Li Y, Shi L, Yu J, Li C, Zhang X, Sun M, Yao Y. The ERAP gene is associated with HCV chronic infection in a Chinese Han population. Hum Immunol. 2017;78(11–12):731–8.  [DOI](https://doi.org/10.1016/j.humimm.2017.10.005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29037997/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Liu%20S,%20Cao%20D,%20Shen%20Y,%20Li%20Y,%20Li%20Y,%20Shi%20L,%20Yu%20J,%20Li%20C,%20Zhang%20X,%20Sun%20M,%20Yao%20Y.%20The%20ERAP%20gene%20is%20associated%20with%20HCV%20chronic%20infection%20in%20a%20Chinese%20Han%20population.%20Hum%20Immunol.%202017;78(11%E2%80%9312):731%E2%80%938.)

28. Szczypiorska M, Sánchez A, Bartolomé N, Arteta D, Sanz J, Brito E, et al. ERAP1 polymorphisms and haplotypes are associated with ankylosing spondylitis susceptibility and functional severity in a Spanish population. Rheumatology (Oxford). 2011;50(11):1969–75.  [DOI](https://doi.org/10.1093/rheumatology/ker229) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21865284/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Szczypiorska%20M,%20S%C3%A1nchez%20A,%20Bartolom%C3%A9%20N,%20Arteta%20D,%20Sanz%20J,%20Brito%20E,%20et%20al.%20ERAP1%20polymorphisms%20and%20haplotypes%20are%20associated%20with%20ankylosing%20spondylitis%20susceptibility%20and%20functional%20severity%20in%20a%20Spanish%20population.%20Rheumatology%20(Oxford).%202011;50(11):1969%E2%80%9375.)

29. Nguyen TT, Chang SC, Evnouchidou I, York IA, Zikos C, Rock KL, Goldberg AL, Stratikos E, Stern LJ. Structural basis for antigenic peptide precursor processing by the endoplasmic reticulum aminopeptidase ERAP1. Nat Struct Mol Biol. 2011;18(5):604–13.  [DOI](https://doi.org/10.1038/nsmb.2021) | [PMC free article](/articles/PMC3087843/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21478864/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Nguyen%20TT,%20Chang%20SC,%20Evnouchidou%20I,%20York%20IA,%20Zikos%20C,%20Rock%20KL,%20Goldberg%20AL,%20Stratikos%20E,%20Stern%20LJ.%20Structural%20basis%20for%20antigenic%20peptide%20precursor%20processing%20by%20the%20endoplasmic%20reticulum%20aminopeptidase%20ERAP1.%20Nat%20Struct%20Mol%20Biol.%202011;18(5):604%E2%80%9313.)

30. Gao S, Xu T, Liang W, Xun C, Deng Q, Guo H, Sheng W. Association of rs27044 and rs30187 polymorphisms in endoplasmic reticulum aminopeptidase 1 gene and ankylosing spondylitis susceptibility: a meta-analysis. Int J Rheum Dis. 2020;23(4):499–510.  [DOI](https://doi.org/10.1111/1756-185X.13795) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31984677/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Gao%20S,%20Xu%20T,%20Liang%20W,%20Xun%20C,%20Deng%20Q,%20Guo%20H,%20Sheng%20W.%20Association%20of%20rs27044%20and%20rs30187%20polymorphisms%20in%20endoplasmic%20reticulum%20aminopeptidase%201%20gene%20and%20ankylosing%20spondylitis%20susceptibility:%20a%20meta-analysis.%20Int%20J%20Rheum%20Dis.%202020;23(4):499%E2%80%93510.)

31. Mahmoudi M, Jamshidi AR, Amirzargar AA, Farhadi E, Nourijelyani K, Fallahi S, Oraei M, Noori S, Nicknam MH. Association between endoplasmic reticulum aminopeptidase-1 (ERAP-1) and susceptibility to ankylosing spondylitis in Iran. Iran J Allergy Asthma Immunol. 2012;11(4):294–300.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/23264405/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Mahmoudi%20M,%20Jamshidi%20AR,%20Amirzargar%20AA,%20Farhadi%20E,%20Nourijelyani%20K,%20Fallahi%20S,%20Oraei%20M,%20Noori%20S,%20Nicknam%20MH.%20Association%20between%20endoplasmic%20reticulum%20aminopeptidase-1%20(ERAP-1)%20and%20susceptibility%20to%20ankylosing%20spondylitis%20in%20Iran.%20Iran%20J%20Allergy%20Asthma%20Immunol.%202012;11(4):294%E2%80%93300.)

32. Li HM, Tang F, Wang LJ, Huang Q, Pan HF, Zhang TP. Association of N6-methyladenosine readers’ genes variation and expression level with pulmonary tuberculosis. Front Public Health. 2022;10:925303.  [DOI](https://doi.org/10.3389/fpubh.2022.925303) | [PMC free article](/articles/PMC9441624/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36072379/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Li%20HM,%20Tang%20F,%20Wang%20LJ,%20Huang%20Q,%20Pan%20HF,%20Zhang%20TP.%20Association%20of%20N6-methyladenosine%20readers%E2%80%99%20genes%20variation%20and%20expression%20level%20with%20pulmonary%20tuberculosis.%20Front%20Public%20Health.%202022;10:925303.)

33. Biasin M, Sironi M, Saulle I, de Luca M, la Rosa F, Cagliani R, Forni D, Agliardi C, lo Caputo S, Mazzotta F, Trabattoni D, Macias J, Pineda JA, Caruz A, Clerici M. Endoplasmic reticulum aminopeptidase 2 haplotypes play a role in modulating susceptibility to HIV infection. AIDS. 2013;27(11):1697–706.  [DOI](https://doi.org/10.1097/QAD.0b013e3283601cee) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23435305/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Biasin%20M,%20Sironi%20M,%20Saulle%20I,%20de%20Luca%20M,%20la%20Rosa%20F,%20Cagliani%20R,%20Forni%20D,%20Agliardi%20C,%20lo%20Caputo%20S,%20Mazzotta%20F,%20Trabattoni%20D,%20Macias%20J,%20Pineda%20JA,%20Caruz%20A,%20Clerici%20M.%20Endoplasmic%20reticulum%20aminopeptidase%202%20haplotypes%20play%20a%20role%20in%20modulating%20susceptibility%20to%20HIV%20infection.%20AIDS.%202013;27(11):1697%E2%80%93706.)

34. Low WF, Ngeow YF, Chook JB, Tee KK, Ong SK, Peh SC, Bong JJ, Mohamed R. Hepatitis B virus DNA methylation and its potential role in chronic hepatitis B. Expert Rev Mol Med. 2022;25:e11.  [DOI](https://doi.org/10.1017/erm.2022.38) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36380484/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Low%20WF,%20Ngeow%20YF,%20Chook%20JB,%20Tee%20KK,%20Ong%20SK,%20Peh%20SC,%20Bong%20JJ,%20Mohamed%20R.%20Hepatitis%20B%20virus%20DNA%20methylation%20and%20its%20potential%20role%20in%20chronic%20hepatitis%20B.%20Expert%20Rev%20Mol%20Med.%202022;25:e11.)

35. Wang M, Kong W, He B, Li Z, Song H, Shi P, Wang J. Vitamin D and the promoter methylation of its metabolic pathway genes in association with the risk and prognosis of tuberculosis. Clin Epigenetics. 2018;10(1):118.  [DOI](https://doi.org/10.1186/s13148-018-0552-6) | [PMC free article](/articles/PMC6136159/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30208925/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Wang%20M,%20Kong%20W,%20He%20B,%20Li%20Z,%20Song%20H,%20Shi%20P,%20Wang%20J.%20Vitamin%20D%20and%20the%20promoter%20methylation%20of%20its%20metabolic%20pathway%20genes%20in%20association%20with%20the%20risk%20and%20prognosis%20of%20tuberculosis.%20Clin%20Epigenetics.%202018;10(1):118.)

36. Ma Y, Fan D, Xu S, Deng J, Gao X, Guan S, Zhang X, Pan F. Ankylosing Spondylitis patients display aberrant ERAP1 gene DNA methylation and expression. Immunol Invest. 2022;51(6):1548–60.  [DOI](https://doi.org/10.1080/08820139.2021.1982965) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34555981/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Ma%20Y,%20Fan%20D,%20Xu%20S,%20Deng%20J,%20Gao%20X,%20Guan%20S,%20Zhang%20X,%20Pan%20F.%20Ankylosing%20Spondylitis%20patients%20display%20aberrant%20ERAP1%20gene%20DNA%20methylation%20and%20expression.%20Immunol%20Invest.%202022;51(6):1548%E2%80%9360.)

37. Schultz MD, He Y, Whitaker JW, Hariharan M, Mukamel EA, Leung D, Rajagopal N, Nery JR, Urich MA, Chen H, et al. Human body epigenome maps reveal noncanonical DNA methylation variation. Nature. 2015;523:212–6.  [DOI](https://doi.org/10.1038/nature14465) | [PMC free article](/articles/PMC4499021/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26030523/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Schultz%20MD,%20He%20Y,%20Whitaker%20JW,%20Hariharan%20M,%20Mukamel%20EA,%20Leung%20D,%20Rajagopal%20N,%20Nery%20JR,%20Urich%20MA,%20Chen%20H,%20et%20al.%20Human%20body%20epigenome%20maps%20reveal%20noncanonical%20DNA%20methylation%20variation.%20Nature.%202015;523:212%E2%80%936.)
