# Pharmacogenetics of colorectal cancer in a third-level hospital in Valencia

## Metadata
**Authors:** Ana Comes-Raga, Luis Sendra Gisbert, Goitzane Marcaida-Benito, Salvador F Aliño Pellicer, María José Herrero Cervera
**Journal:** Advances in Laboratory Medicine
**Date:** 2024 Nov 7
**DOI:** [10.1515/almed-2024-0146](https://doi.org/10.1515/almed-2024-0146)
**PMID:** 39713544
**PMCID:** PMC11661530
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11661530/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC11661530/pdf/almed-5-4-almed-2024-0146.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC11661530/pdf/almed-5-4-almed-2024-0146.pdf)

## Abstract

**Objectives:** 
Genetic variants with associated pharmacokinetic and pharmacodynamic effects have an impact on the development of adverse drug reactions and survival of patients with colorectal cancer.

**Methods:** 
A selection of genetic variants was performed according to the established chemotherapy and the pharmacogenetic databases. Genotyping was performed using MassArray technology (Agena Bioscience). Variant-toxicity and survival-genotype correlations were assessed using logistic regression (SPSS v.28.0.1.1).

**Results:** 
Genotyping of 25 SNPs was performed in 96 patients. In relation to the DPYD gene, 3.5 % had the rs75017182 mutation; 4.7 % the rs1801158 mutation and 7.1 % the rs1801160 mutation. Genotypic frequencies in the UGT1A1 gene were 39.4 % (*1/*1); 37.9 % (*1/*28); 19.7 % (*28/*28); and 3 % (*1/*36). The genotypes CT of the rs1801160 variant, AT of the rs67376798 variant (DPYD) and *1/*36 (UGT1A1) were associated with low survival (p-value: 0.006, <0.001, and 0.052, respectively). The most frequent adverse reactions were gastrointestinal disorders, followed by neurotoxicity. The CC genotype (rs1801160, DPYD) was associated with a lower risk for developing severe gastrointestinal events, whereas CC (rs1801158, DPYD) was associated with a lower risk of developing severe general hematologic toxicity.

**Conclusions:** 
The population frequencies obtained in our study for rs1801160 and rs75017182 (DPYD); and for *1/*28, *28/*, and *1/*36 (UGT1A1) were inconsistent with the frequencies reported for the Spanish population in the literature. The genotypes CT of rs1801160, AT of rs67376798 (DPYD), and 1/*36 (UGT1A1) were associated with lower survival rates.

Keywords: pharmacogenetics, precision medicine, colorectal cancer, polymorphism, variant, genotyping

### Objectives

Genetic variants with associated pharmacokinetic and pharmacodynamic effects have an impact on the development of adverse drug reactions and survival of patients with colorectal cancer.

### Methods

A selection of genetic variants was performed according to the established chemotherapy and the pharmacogenetic databases. Genotyping was performed using MassArray technology (Agena Bioscience). Variant-toxicity and survival-genotype correlations were assessed using logistic regression (SPSS v.28.0.1.1).

### Results

Genotyping of 25 SNPs was performed in 96 patients. In relation to the *DPYD* gene, 3.5 % had the rs75017182 mutation; 4.7 % the rs1801158 mutation and 7.1 % the rs1801160 mutation. Genotypic frequencies in the *UGT1A1* gene were 39.4 % (*1/*1); 37.9 % (*1/*28); 19.7 % (*28/*28); and 3 % (*1/*36). The genotypes CT of the rs1801160 variant, AT of the rs67376798 variant (*DPYD*) and *1/*36 (*UGT1A1*) were associated with low survival (p-value: 0.006, <0.001, and 0.052, respectively). The most frequent adverse reactions were gastrointestinal disorders, followed by neurotoxicity. The CC genotype (rs1801160, *DPYD*) was associated with a lower risk for developing severe gastrointestinal events, whereas CC (rs1801158, *DPYD*) was associated with a lower risk of developing severe general hematologic toxicity.

### Conclusions

The population frequencies obtained in our study for rs1801160 and rs75017182 (*DPYD)*; and for *1/*28, *28/*, and *1/*36 (*UGT1A1)* were inconsistent with the frequencies reported for the Spanish population in the literature. The genotypes CT of rs1801160, AT of rs67376798 (*DPYD)*, and 1/*36 (*UGT1A1*) were associated with lower survival rates.

## Introduction

Pharmacogenetics is the study of the influence of individual genetic components on drug response and clinical outcomes. The effectiveness and toxicity of a particular drug are influenced by genetic variation among individuals [[1](#j_almed-2024-0146_ref_001)].

Chemotherapeutic agents generally have a narrow therapeutic margin, and their effectiveness and safety are determined by individual variations in their metabolism. The approach to colorectal cancer (CRC) treatment is based on a range of chemotherapy options involving multiple drug combinations, with fluoropyrimidines being the cornerstone for treatment. Fluoropyrimidines are usually administered in combination with platinum compounds or irinotecan [[2](#j_almed-2024-0146_ref_002)].

Fluoropyrimidines may cause severe adverse drug reactions, including mucositis, diarrhea and myelosuppression and, in 1 % of patients, they may cause death. This toxicity is frequently caused by the reduced activity of dihydropyrimidine dehydrogenase (DPD), the metabolic enzyme responsible for the breakdown of fluoropyrimidines that is encoded by the *DPYD* gene. Some DPYD mutations may reduce DPD activity, leading to previously described adverse events [[3](#j_almed-2024-0146_ref_003)].

Platinum compounds (oxaliplatinum and cisplatinum) are used in combination with fluoropyrimidines for the treatment of CRC, and their use is not exempt from toxicity [[4](#j_almed-2024-0146_ref_004)].

SN-38 is the active metabolite of irinotecan and a topoisomerase-I inhibitor, affecting DNA replication and transcription. The *UGT1A1* gene mediates the detoxification and conjugation of SN-38. Thus, mutations in this gene may cause toxicities including neutropenia, myelosuppression and diarrhea in patients receiving this treatment [[5](#j_almed-2024-0146_ref_005)].

Platforms such as PharmGKB, the Clinical Pharmacogenetics Implementation Consortium (CPIC) and the Dutch Pharmacogenetics Working Group regularly conduct literature reviews and publish guidelines on the applications of pharmacogenetics in clinical practice. These efforts have contributed to a better understanding of pharmacokinetics and the genes associated with pharmacodynamics [[6](#j_almed-2024-0146_ref_006), [7](#j_almed-2024-0146_ref_007)]. Moreover, regulatory agencies such as the Food and Drug Administration (FDA) and the European Medicines Agency (EMA), provide pharmacogenetic information in the data sheets of the agents used for the treatment of CRC [[8](#j_almed-2024-0146_ref_008), [9](#j_almed-2024-0146_ref_009)].

In Spain, the Society of Pharmacogenetics and Pharmacogenomics (SEFF) and the Spanish Society of Medical Oncology (SEOM) have published clinical guidelines for the genotyping of six mutations of the *DPYD* gene in candidates to fluoropyrimidine-based chemotherapy [[10](#j_almed-2024-0146_ref_010)].

In this study, genotyping was performed of genetic variants of the pharmacogenes involved in the metabolism of anticancer treatments in CRC. The study sample was composed of patients diagnosed with CRC in our Center. The objective of the study was to determine the frequency of these variants in our population and identify genetic biomarkers associated with the risk for developing adverse events and/or with reduced drug efficacy.

## Materials and methods

A retrospective observational study was carried out in which genetic variants involved in the metabolism of antineoplastic drugs for CRC were analyzed, in a sample of patients diagnosed with CRC in our Center (Consorcio Hospital General Universitario de Valencia, CHGUV). A literature search of publications and recommendations of the aforementioned organizations was performed to identify the variants involved in the metabolism of these agents. A total of 25 genetic variants were selected. The genes selected and their polymorphic variants are detailed in [Table 1](#j_almed-2024-0146_tab_001).

### Table 1:

| Genes | Mutations | Level of evidencea |
| --- | --- | --- |
| DPYD | rs3918290 | 1 A |
| rs67376798 | 1 A |  |
| rs55886062 | 1 A |  |
| rs75017182 | 1 A |  |
| rs115232898 | 1 A |  |
| rs1801158 | 1 A |  |
| rs1801160 | 1 A |  |
| rs1801266 | 1 A |  |
| rs1801268 | 1 A |  |
| rs59086055 | 1 A |  |
| rs777425216 | NA |  |
| rs78060119 | 1 A |  |
| rs80081766 | NA |  |
| GSTP | rs1695 | 3 |
| XPC | rs2228001 | 3 |
| ERCC1 | rs3212986 | 3 |
| rs11615 | 3 |  |
| XRCC1 | rs25487 | 2 B |
| UGT1A1 | *1/*1 | 1 |
| *28/*28 | 1 |  |
| *1/*28 | 1 |  |
| *1/*36 | 1 |  |
| rs4148323 | 1 B |  |
| SEMAC | rs7779029 | 3 |
| C8orf34 | rs1517114 | 3 |

Table 1 Caption: Genes and mutations related to CRC pharmacogenetics analyzed in this research study.

The sample was composed of patients >18 years diagnosed with CRC in our Center undergoing anticancer treatments, which completed a year of treatment and follow-up. Patients not undergoing anticancer treatments or refusing to take part in the study were excluded.

Study variables included demographics (age at diagnosis, sex, date of birth, and ethnicity); tumor-related data (date of diagnosis, tumor stage, presence/absence of metastases); treatment regimen received; adverse drug reactions (ADRs) and evaluation of response. All data was collected at the date of diagnosis, except for ADRs and the evaluation of response, which were collected one year after diagnosis.

ADRs were classified according to the Common Toxicity Criteria of Adverse Events (CTCAE v 4.03) [[11](#j_almed-2024-0146_ref_011)]. ADRs were categorized as neurotoxicity, gastrointestinal (GI), hematologic, kidney, skin, and hepatic toxicity. ADR severity was also collected and categorized (mild: grade 1 and 2; severe≥3).

The oncologists followed the Response Evaluation Criteria In Solid Tumors (RECIST v1.1) [[12](#j_almed-2024-0146_ref_012)].

The project was approved by the Ethics Committee of our Center. The samples collected and analyzed were composed of DNA extracted from whole blood drawn by venous puncture. SNPs were analyzed using the MassArray platform (Agena Bioscience).

The results obtained were analyzed using SPSS^®^ version 28.0.1.1. The correlation between SNPs and documented toxicities was assessed using logistic regression, statistical significance and odds ratios. The association between survival and the genotypes expressed in the population was examined using logistic regression for binary variables and Kaplan-Meier plotting. Survival was assessed one year after diagnosis of the neoplasm.

## Results

A total of 96 subjects were recruited, of whom 39 % were women and 56 % were men, with ages ranging from 28 to 83 years, and a median age of 67 years. All patients had Spanish nationality, except for one North American, three Latin Americans, one Russian, and one Bulgarian patient.

At diagnosis, 6 % of patients had stage-I CRC; 21 % had stage II CRC; 29 % had stage-III CRC, and 37 % had stage-IV CRC. As many as 20.4 % of patients had findings suggesting the presence of metastasis. The therapeutic regimens were based on fluoropyrimidines (5-Fluorouracil or Capecitabine) administered either, alone or in combination with radiotherapy, platinum compounds (Cisplatinum or Oxaliplatinum), and monoclonal antibodies (Bevacizumab or Panitumumab) and Irinotecan as add-on therapy. The different combinations are shown in [Figure 1](#j_almed-2024-0146_fig_001). In total, 34 % of patients received second-line treatment, whereas 6.6 % received three lines of treatment for their disease.

### Figure 1:

![Figure 1:](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0a29/11661530/6cf4b57491a7/j_almed-2024-0146_fig_001.jpg)

Graph of the therapeutic schemes used as first-line treatment in our study population. MAB (monoclonal antibodies): bevacizumab, and panitumumab; Pt (platinum compounds): cisplatin, oxaliplatin; FP (fluoropyrimidines): 5-fluorouracil, capecitabine; IRI (irinotecan); RT (radiotherapy); MT (monotherapy).

Screening for 25 SNPs was performed. The results are shown in [Table 2](#j_almed-2024-0146_tab_002), which includes all the genotypes frequency rates obtained, categorized by wild-type (WT), homozygous mutant (HomMut), and heterozygous mutant (HetMut).

### Table 2:

|   | Gen DPYD |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| % | rs3918290(*2) | rs67376798-DPYD*14 | Rs55886062(*13) | rs75017182G>C | rs56038477 | rs115232898 | rs1801158 | rs1801160 | rs1801266 | rs1801268 | rs59086055 | rs777425216 | rs78060119 | rs80081766 | rs9996584 |
| Wild type | 100 | 99 | 100 | 96.5 | 96.5 | 100 | 95.3 | 92.9 | 100 | 100 | 100 | 100 | 100 | 100 | 31.8 |
| HetMut | 0 | 1 | 0 | 3.5 | 3.5 | 0 | 4.7 | 7.1 | 0 | 0 | 0 | 0 | 0 | 0 | 51.8 |
| HomMut | 0 | 0 | 0 | 0 | 0 | 0 | 0 | 0 | 0 | 0 | 0 | 0 | 0 | 0 | 16.4 |
|   |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
|   | Gen GSTP | Gen XPC | Gen ERCC1 | Gen XRCC1 | Gen ERCC1 | Gen SEMAC | Gen C8orf34 | UGT1A1 |  |  |   |   |   |  |  |
|   | rs1695 | rs2228001 | rs3212986 | rs25487 | rs11615 | rs7779029 | rs1517114 | rs4148323 | *1/*1 | 39.4 |   |   |   |   |   |
|   |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
| Wild type | 8.2 | 30.6 | 2.4 | 41.2 | 9.4 | 1.2 | 14.1 | 100 | *28/*28 | 19.7 |   |   |   |   |   |
| HetMut | 34.1 | 54.1 | 37.6 | 44.7 | 45.9 | 14.1 | 41.2 | 0 | *1/*28 | 37.9 |   |   |   |   |   |
| HomMut | 56.5 | 15.3 | 60.0 | 41.2 | 44.7 | 84.7 | 44.7 | 0 | *1/*36 | 3.0 |   |   |   |   |   |

Table 2 Caption: Genotyping results for our population. Results are expressed as % of carriers of the wild type genotype, heterozigous mutation (HetMut) and homozygous mutation (HomMut). Results for the UGT1A1 gene are shown separately.

The *DPYD* gene was identified in 3.5 % of HetMut patients for the rs75017182 mutation, in 4.7 % of patients with the rs1801158 mutation, and in 7.1 % of patients with the rs1801160 mutation.

With respect to the *UGT1A1* gene, 77.3 % of patients had the *1/*1 and *1/*28 genotype; 19.7 % had the *28/*28 genotype, and 3 % had the *1/*36 genotype. All patients had the WT genotype of the rs4148323 mutation.

In relation to the rs7779029 mutation in the *SEMAC* gene, 84.7 % of patients had the HomMut genotype; 14.1 % the HetMut genotype; and 1.2 % had the WT genotype. With respect to the rs1517114 mutation in the *C8orf34* gene, 14.1 % had the WT genotype, 41.2 % had the HetMut and 44.7 % had the HomMut genotype.

In relation to platinum compounds, screening was performed for the rs1695 mutations of the *GSTP* gene, with 8.2 % being WT, 34.1 % HetMut, and 56.5 % being HomMut; the rs2228001 mutation in the *XPC* gene (30.6 % WT, 54.1 % HetMut, and 15.3 % HomMut); the rs3212986 mutation in the *ERCC1* gene (2.4 % WT, 37.6 % HetMut, and 60 % HomMut); the rs25487 mutation in the *XRCC1* gene (41.2 % WT, 44.7 % HetMut, and 41.2 % HomMut); and the rs11615 mutation in the *ERCC1* gene (9.4 % WT, 45.9 % HetMut, and 44.7 % HomMut). All data is shown in [Table 2](#j_almed-2024-0146_tab_002).

Regarding survival results for the SNPs studied, a statistically significant relationship was observed between genotypes CT of the rs1801160 mutation in the *DPYD* gene, AT (heterozygous mutation) of the rs67376798 mutation in the *DPYD* gene, and *1/*36 mutation in the *UGT1A1* gene (0.006, <0.001, and 0.052 respectively) with less survival. Survival curves are shown in [Figure 2](#j_almed-2024-0146_fig_002).

### Figure 2:

![Figure 2:](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0a29/11661530/85999e8d792b/j_almed-2024-0146_fig_002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=11661530_j_almed-2024-0146_fig_002.jpg)

Graph displaying results from Kaplan-Meier survival plots. The graph on the left. (A) Represents the rs1801160 mutation (CC, CT genotypes); the graph in the middle (B) represents the rs67376798 mutation and the AT (heterozygous mutation) and wild type (TT) genotypes; the graph on the right (C) represents the UGT1A 1 gene and the mutations studied.

The following results were obtained from the ADRs study:

The association between the type of anticancer therapy and the ADRs developed by the patient is displayed in [Table 3](#j_almed-2024-0146_tab_003). All patients developed more than a type of ADR during the course of treatment. Gastrointestinal events were most frequent in fluoropyrimidines + radiotherapy; fluoropyrimidines + platinum compounds; and fluoropyrimidines + platinum compound + irinotecan + associated monoclonal antibodies. Gastrointestinal toxicities were as frequent as hematologic toxicities in fluoropyrimidines + associated monoclonal antibodies. Finally, patients developed the same rate of gastrointestinal and neurological reactions when ironotecan was administered.

### Table 3:

| Therapeutic scheme | Hematologic toxicity, % |  | Gastrointestinal toxicity, % |  | Neurologic toxicity, % |  |
| --- | --- | --- | --- | --- | --- | --- |
| Mild | Severe | Mild | Severe | Mild | Severe |  |
| Fluoropyrimidines (alone) | 7.7 | NA | 38.5 | 7.7 | 46.2 | NA |
| Fluoropyrimidines+radiation therapy | 14.3 | NA | 28.6 | 42.9 | 14.3 | NA |
| Fluoropyrimidines+MAB | 50.0 | NA | NA | 50.0 | NA | NA |
| Fluoropyrimidines+Platin C. | 48.8 | 2.4 | 58.5 | 22.0 | 19.5 | 7.3 |
| Fluoropyrimidines+Platin C.+MAB | 46.2 | 3.8 | 46.2 | 19.2 | 21.1 | 7.7 |
| Fluoropyrimidines+MAB+Irinotecan | NA | NA | 100.0 | NA | NA | NA |
| Irinotecan | NA | NA | NA | 50.0 | NA | 50.0 |

Table 3 Caption: ADRs frequency rates by therapeutic scheme administered.

A relationship was observed between the CC genotype of the rs1801160 mutation in the *DPYD* gene and a lower risk for developing severe GI events (p-value<0.001; odds ratio 0.064). Likewise, the CC genotype of the rs1801158 mutation in the *DPYD* gene was also associated with a lower risk for developing all-type hematologic reactions (p-value<0.087; odds ratio 0.133) and severe hematologic reactions (p-value<0.001; odds ratio 0.058). This variant was classified as protective against severe hematologic adverse reactions.

No correlation was observed between the genotypes studied and neurological, kidney, skin and hepatic toxicities.

## Discussion

This study revealed that 15 % of CRC patients are carriers of a mutation in the *DPYD* gene. This mutation is associated with the loss of function of the DPD enzyme, resulting in an increased toxicity of fluoropyrimidine-based regimes. This rate exceeds the one reported in previous studies for the Spanish population, reporting a rate of 10 % [[13](#j_almed-2024-0146_ref_013)].

The most frequent mutation was rs1801160 (six patients), followed by rs75017182 (five patients) and rs1801158 (four patients).

According to the literature, the rs1801160 mutation has a frequency rate of 5.6 % for the CT genotype, and 1.9 % for the TT genotype in the Caucasian population. In our series, the CT genotype had a frequency rate of 7.1 % andno cases of homozygous mutation (TT) have been reported to date.

The rs75017182 mutation is in linkage disequilibrium with rs56038477, with the two being related to the HapB3 haplotype. According to the evidence available for the Spanish population, the rs75017182 mutation has a population frequency of 3.7 %. In our series, frequency was 3.5 %. Since these two mutations are not always in a perfect linkage disequilibrium, screening for both mutations were performed in the entire study population [[14](#j_almed-2024-0146_ref_014)].

In total, 4.7 % of our patients were carriers of the CT genotype of the rs1801158 mutation. None of our patients had the TT genotype. According to the literature, rates are 9.3 % and 0.9 % for the CT and TT genotypes, respectively.

According to the results obtained, the most frequent genotypes in our population were the CT genotype of the rs1801160 mutation and the GC genotype of the rs1801158, with a lower frequency of rs1801158, both in heterozygous and homozygous carriers.

For the rs1695 (*GSTP*), rs2228001 (*XPC*), rs3212986 (*ERCC1*), rs25487 (*XRCC1*), and rs11615 (*ERCC1*) mutations involved in the metabolism and action of platinum-based treatments, no statistically significant differences were observed between the genotypes of our series and the ones reported in the literature for the Spanish population [[15](#j_almed-2024-0146_ref_015)].

Regarding the genes associated with response to irinotecan and its associated ADRs, some inconsistencies were detected in the frequency of mutations in the *UGT1A1* gene in the Spanish population. The *1/*1 mutation in our series had a frequency of 39.4 %, which is consistent with the one reported for the European population (40 %). The *28/*28 mutation had a frequency of 19.7 %, which exceeds the one reported for the Spanish population, 9 %. The *1/*28 genotype had a frequency of 37.9 % in our series, whereas the frequency described for the Spanish population was 51 %. Whereas not cases of *1/*36 have been reported in the literature, a carrier of this genotype was identified in our series (3.0 %) [[16](#j_almed-2024-0146_ref_016)].

The rs1801158 and rs1801160 mutations of the *DPYD* gene were associated with a lower frequency of severe hematologic and GI events, respectively. Similar results have been reported for the rs1801158 mutation [[17](#j_almed-2024-0146_ref_017)]. However, there is evidence in the literature of the role of the rs1801160 mutation in the development of severe GI adverse events, which is inconsistent with our results [[18](#j_almed-2024-0146_ref_018)].

There is no evidence in the literature of an association between lower survival rate with the CT genotype of rs1801160 (p-value: 0.006) and the AT genotype of the rs67376798 (p-value<0.001) mutations in the DPYD gene, or 1/*36 in the UGT1A1 gene (p-value: 0.052).

Under the auspices of the Spanish Ministry of Health, all public hospitals in Spain offer pharmacogenetic services involving a panel of genetic tests. To date, the genes and mutations included for screening in the treatment of CRC are *DPYD**2A, *DPYD**13, *DPYD* c.2846A>T, *DPYD* c.1236G>A/HapB3; and *UGT1A1* [[19](#j_almed-2024-0146_ref_019)]. The aim of this study was to provide evidence supporting the association between genetic mutations and phenotypes. These results may help improve the quality of personalized treatments.

It is necessary to perform a population study of the genes involved in the main metabolic pathways of the agents administered in CRC to ensure their appropriate administration.

### Limitations of the study

The results of this study were obtained during the course of a doctoral researcher’s PhD project. For the purposes of this project, other variables were collected that may have influenced the results obtained and should be examined in future studies.

These preliminary results should be tested and confirmed in a larger cohort study using more powerful statistical analyses to validate their use in clinical practice.

## Acknowledgments

Fundación José Luís Castaño-SEQC.

## Footnotes

## References

1. Genetic Alliance, New York-Mid-Atlantic Consortium for Genetic and Newborn Screening Services . Understanding genetics: A New York, Mid-Atlantic guide for patients and health professionals. Morrisville, NC: Estados Unidos de América; 2009. Lulu.com.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/23304754/) | [Google Scholar](https://scholar.google.com/scholar_lookup?title=Understanding%20genetics:%20A%20New%20York,%20Mid-Atlantic%20guide%20for%20patients%20and%20health%20professionals&publication_year=2009&)

2. Cortejoso L, García Ma I, García-Alfonso P, Grávalos C, Robles L, González-Haba E, et al. Validación de polimorfismos genéticos asociados a toxicidad al tratamiento quimioterápico en pacientes de cancer colorrectal. Farm Hosp. 2014;38:283–90. doi: 10.7399/fh.2014.38.4.1121.  [DOI](https://doi.org/10.7399/fh.2014.38.4.1121) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25137161/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Farm%20Hosp&title=Validaci%C3%B3n%20de%20polimorfismos%20gen%C3%A9ticos%20asociados%20a%20toxicidad%20al%20tratamiento%20quimioter%C3%A1pico%20en%20pacientes%20de%20cancer%20colorrectal&author=L%20Cortejoso&author=I%20Garc%C3%ADa%20Ma&author=P%20Garc%C3%ADa-Alfonso&author=C%20Gr%C3%A1valos&author=L%20Robles&volume=38&publication_year=2014&pages=283-90&pmid=25137161&doi=10.7399/fh.2014.38.4.1121&)

3. Henricks LM, Lunenburg CATC, de Man FM, Meulendijks D, Frederix GWJ, Kienhuis E, et al. DPYD genotype-guided dose individualisation of fluoropyrimidine therapy in patients with cancer: a prospective safety analysis. Lancet Oncol. 2018;19:1459–67. doi: 10.1016/s1470-2045(18)30686-7.  [DOI](https://doi.org/10.1016/s1470-2045(18)30686-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30348537/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet%20Oncol&title=DPYD%20genotype-guided%20dose%20individualisation%20of%20fluoropyrimidine%20therapy%20in%20patients%20with%20cancer:%20a%20prospective%20safety%20analysis&author=LM%20Henricks&author=CATC%20Lunenburg&author=FM%20de%20Man&author=D%20Meulendijks&author=GWJ%20Frederix&volume=19&publication_year=2018&pages=1459-67&pmid=30348537&doi=10.1016/s1470-2045(18)30686-7&)

4. Perego P, Robert J. Oxaliplatinum in the era of personalized medicine: from mechanistic studies to clinical efficacy. Cancer Chemother Pharmacol. 2016;77:5–18. doi: 10.1007/s00280-015-2901-x.  [DOI](https://doi.org/10.1007/s00280-015-2901-x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26589793/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Chemother%20Pharmacol&title=Oxaliplatinum%20in%20the%20era%20of%20personalized%20medicine:%20from%20mechanistic%20studies%20to%20clinical%20efficacy&author=P%20Perego&author=J%20Robert&volume=77&publication_year=2016&pages=5-18&pmid=26589793&doi=10.1007/s00280-015-2901-x&)

5. Pratt VM, Scott SA, Pirmohamed M, Esquivel B, Kattman BL, Malheiro AJ. Medical genetics summaries. . 2012. [el 24 de abril de 2024]. https://pubmed.ncbi.nlm.nih.gov/28520340/ Disponible en. citado.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/28520340/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28520340/)

6. PharmGKB PharmGKB. . [el 24 de abril de 2024]. https://www.pharmgkb.org/gene/PA145 Disponible en. citado.  [https://www.pharmgkb.org/gene/PA145](https://www.pharmgkb.org/gene/PA145)

7. CPIC Cpicpgx.org. . [el 24 de abril de 2024]. https://cpicpgx.org/ Disponible en. citado.  [https://cpicpgx.org/](https://cpicpgx.org/)

8. FDA en español. U.S. Food and Drug Administration. FDA; 2024. [el 24 de abril de 2024]. https://www.fda.gov/about-fda/fda-en-espanol Disponible en. citado.  [https://www.fda.gov/about-fda/fda-en-espanol](https://www.fda.gov/about-fda/fda-en-espanol) | [Google Scholar](https://scholar.google.com/scholar_lookup?title=FDA%20en%20espa%C3%B1ol.%20U.S.%20Food%20and%20Drug%20Administration&publication_year=2024&)

9. Agencia Europea de Medicamentos (EMA) European Union; [el 24 de abril de 2024]. https://european-union.europa.eu/institutions-law-budget/institutions-and-bodies/search-all-eu-institutions-and-bodies/european-medicines-agency-ema_es Disponible en. citado.  [https://european-union.europa.eu/institutions-law-budget/institutions-and-bodies/search-all-eu-institutions-and-bodies/european-medicines-agency-ema_es](https://european-union.europa.eu/institutions-law-budget/institutions-and-bodies/search-all-eu-institutions-and-bodies/european-medicines-agency-ema_es) | [Google Scholar](https://scholar.google.com/scholar_lookup?title=Agencia%20Europea%20de%20Medicamentos%20(EMA)&)

10. García-Alfonso P, Saiz-Rodríguez M, Mondéjar R, Salazar J, Páez D, Borobia AM, et al. Consensus of experts from the Spanish Pharmacogenetics and Pharmacogenomics Society and the Spanish Society of Medical Oncology for the genotyping of DPYD in cancer patients who are candidates for treatment with fluoropyrimidines. Clin Transl Oncol. 2022;24:483–94. doi: 10.1007/s12094-021-02708-4.  [DOI](https://doi.org/10.1007/s12094-021-02708-4) | [PMC free article](/articles/PMC8885558/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34773566/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Transl%20Oncol&title=Consensus%20of%20experts%20from%20the%20Spanish%20Pharmacogenetics%20and%20Pharmacogenomics%20Society%20and%20the%20Spanish%20Society%20of%20Medical%20Oncology%20for%20the%20genotyping%20of%20DPYD%20in%20cancer%20patients%20who%20are%20candidates%20for%20treatment%20with%20fluoropyrimidines&author=P%20Garc%C3%ADa-Alfonso&author=M%20Saiz-Rodr%C3%ADguez&author=R%20Mond%C3%A9jar&author=J%20Salazar&author=D%20P%C3%A1ez&volume=24&publication_year=2022&pages=483-94&pmid=34773566&doi=10.1007/s12094-021-02708-4&)

11. Common Terminology criteria for adverse events (CTCAE) version 4.0 published. . 2009  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=.&title=Common%20Terminology%20criteria%20for%20adverse%20events%20(CTCAE)%20version%204.0%20published&publication_year=2009&)

12. Eisenhauer EA, Therasse P, Bogaerts J, Schwartz LH, Sargent D, Ford R, et al. New response evaluation criteria in solid tumours: revised RECIST guideline (version 1.1) Eur J Cancer. 2009;45:228–47. doi: 10.1016/j.ejca.2008.10.026.  [DOI](https://doi.org/10.1016/j.ejca.2008.10.026) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19097774/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Cancer&title=New%20response%20evaluation%20criteria%20in%20solid%20tumours:%20revised%20RECIST%20guideline%20(version%201.1)&author=EA%20Eisenhauer&author=P%20Therasse&author=J%20Bogaerts&author=LH%20Schwartz&author=D%20Sargent&volume=45&publication_year=2009&pages=228-47&pmid=19097774&doi=10.1016/j.ejca.2008.10.026&)

13. Riera P, Riba M, Bernal S, Virgili AC, Páez D, Moreno ME. Frequency and clinical relevance of DPYD genetic variants in gastrointestinal cancer patients. Farm Hosp. 2021;45:5–10.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/35379107/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Farm%20Hosp&title=Frequency%20and%20clinical%20relevance%20of%20DPYD%20genetic%20variants%20in%20gastrointestinal%20cancer%20patients&author=P%20Riera&author=M%20Riba&author=S%20Bernal&author=AC%20Virgili&author=D%20P%C3%A1ez&volume=45&publication_year=2021&pages=5-10&pmid=35379107&)

14. Turner AJ, Haidar CE, Yang W, Boone EC, Offer SM, Empey PE, et al. Updated DPYD HapB3 haplotype structure and implications for pharmacogenomic testing. Clin Transl Sci. 2024;17:e13699. doi: 10.1111/cts.13699.  [DOI](https://doi.org/10.1111/cts.13699) | [PMC free article](/articles/PMC10777430/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/38129972/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Transl%20Sci&title=Updated%20DPYD%20HapB3%20haplotype%20structure%20and%20implications%20for%20pharmacogenomic%20testing&author=AJ%20Turner&author=CE%20Haidar&author=W%20Yang&author=EC%20Boone&author=SM%20Offer&volume=17&publication_year=2024&pages=e13699&pmid=38129972&doi=10.1111/cts.13699&)

15. 1000 genomes [Internet] . [el 25 de abril de 2024]. https://www.internationalgenome.org/ Internationalgenome.org. Disponible en. citado.  [https://www.internationalgenome.org/](https://www.internationalgenome.org/)

16. Fernández Salazar JM, Remacha Sevilla A, del Río Conde E, Baiget Bastús M. Distribución del genotipo A(TA)7TAA asociado al síndrome de Gilbert en la población española [Distribution of the A(TA)7TAA genotype associated with Gilbert syndrome in the Spanish population] Med Clin. 2000;115:540–1. Spanish.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/11141380/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Med%20Clin&title=Distribuci%C3%B3n%20del%20genotipo%20A(TA)7TAA%20asociado%20al%20s%C3%ADndrome%20de%20Gilbert%20en%20la%20poblaci%C3%B3n%20espa%C3%B1ola%20%5BDistribution%20of%20the%20A(TA)7TAA%20genotype%20associated%20with%20Gilbert%20syndrome%20in%20the%20Spanish%20population%5D&author=JM%20Fern%C3%A1ndez%20Salazar&author=A%20Remacha%20Sevilla&author=E%20del%20R%C3%ADo%20Conde&author=M%20Baiget%20Bast%C3%BAs&volume=115&publication_year=2000&pages=540-1&pmid=11141380&)

17. Ruzzo A, Graziano F, Galli F, Galli F, Rulli E, Lonardi S, et al. Dihydropyrimidine dehydrogenase pharmacogenetics for predicting fluoropyrimidine-related toxicity in the randomised, phase III adjuvant TOSCA trial in high-risk colon cancer patients. Br J Cancer. 2017;117:1269–77. doi: 10.1038/bjc.2017.289.  [DOI](https://doi.org/10.1038/bjc.2017.289) | [PMC free article](/articles/PMC5709672/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29065426/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Cancer&title=Dihydropyrimidine%20dehydrogenase%20pharmacogenetics%20for%20predicting%20fluoropyrimidine-related%20toxicity%20in%20the%20randomised,%20phase%20III%20adjuvant%20TOSCA%20trial%20in%20high-risk%20colon%20cancer%20patients&author=A%20Ruzzo&author=F%20Graziano&author=F%20Galli&author=F%20Galli&author=E%20Rulli&volume=117&publication_year=2017&pages=1269-77&pmid=29065426&doi=10.1038/bjc.2017.289&)

18. Deenen MJ, Tol J, Burylo AM, Doodeman VD, de Boer A, Vincent A, et al. Relationship between single nucleotide polymorphisms and haplotypes in DPYD and toxicity and efficacy of capecitabine in advanced colorectal cancer. Clin Cancer Res. 2011;17:3455–68. doi: 10.1158/1078-0432.ccr-10-2209.  [DOI](https://doi.org/10.1158/1078-0432.ccr-10-2209) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21498394/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=Relationship%20between%20single%20nucleotide%20polymorphisms%20and%20haplotypes%20in%20DPYD%20and%20toxicity%20and%20efficacy%20of%20capecitabine%20in%20advanced%20colorectal%20cancer&author=MJ%20Deenen&author=J%20Tol&author=AM%20Burylo&author=VD%20Doodeman&author=A%20de%20Boer&volume=17&publication_year=2011&pages=3455-68&pmid=21498394&doi=10.1158/1078-0432.ccr-10-2209&)

19. Consejo interterritorial. Sistema nacional de salud ACUERDO SOBRE EL CATÁLOGO DE PRUEBAS GENÉTICAS DE LA CARTERA COMÚN DE SERVICIOS DEL SISTEMA NACIONAL DE SALUD. . [24 junio 2024]. https://www.sanidad.gob.es/organizacion/consejoInterterri/docs/1553.pdf [Internet]. Disponible. Consultado.  [https://www.sanidad.gob.es/organizacion/consejoInterterri/docs/1553.pdf](https://www.sanidad.gob.es/organizacion/consejoInterterri/docs/1553.pdf)
