# Prognostic Role of Human Leukocyte Antigen Alleles and Cytokine Single-Nucleotide Polymorphisms in Patients with Chronic Myeloid Leukemia Treated with Tyrosine Kinase Inhibitor Drugs

## Metadata
**Authors:** Samuel Kinde Birru, Ilias Doxiadis, Rawleigh Howe, Tsehayneh Kelemu, Saifu Hailu Chala, Abdulaziz Sherif, Fisihatsion Tadesse, Aster Tsegaye, Amha Gebremedhin, Claudia Lehmann
**Journal:** Genes
**Date:** 2024 Jun 2
**DOI:** [10.3390/genes15060732](https://doi.org/10.3390/genes15060732)
**PMID:** 38927668
**PMCID:** PMC11203291
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11203291/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC11203291/pdf/genes-15-00732.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC11203291/pdf/genes-15-00732.pdf)

## Abstract

Tyrosine kinase inhibitor (TKI) drugs have significantly improved chronic myeloid leukemia (CML) outcomes. Neopeptides from CML cells may induce specific immune responses, which are crucial for deep molecular (DMR) and treatment-free remission (TFR). In this study of Ethiopian patients with CML (n = 162), the HLA alleles and single-nucleotide polymorphisms of five cytokines revealed significant associations with clinical outcomes. Clinically unfavorable outcomes correlated with HLA alleles A*03:01/02, A*23:17:01, B*57:01/02/03, and HLA-DRB4*01:01 (p-value = 0.0347, p-value = 0.0285, p-value = 0.037, and p-value = 0.0127, respectively), while HLA-DRB4*01:03:01 was associated with favorable outcomes (p-value = 0.0058). After assigning values for the ‘low’, ‘intermediate’, and ‘high’ gene expression of the SNPs’ respective cytokine genes, Kaplan–Meier estimates for relapse-free survival, adjusted for age, treatment duration, and relapse risk among patients after the administration of TKIs, indicated that a gene expression ratio above the overall median of TNF-α, IL-6, and the combination of TGF-β1/IL-10, IFNγ, and IL-6/IL-10 TGF-β1 was correlated with a higher likelihood of treatment failure ((RR: 3.01; 95% CI: 1.1–8.3; p-value = 0.0261) and (RR: 2.4; 95% CI: 1.1–5.2; p-value = 0.022), respectively). Multi-SNPs, surpassing single-SNPs, and HLA allele polymorphisms showed promise in predicting outcomes of patients with CML during TKI treatment, prompting further exploration into their potential utility.

Keywords: cytokine polymorphism, CML, Ethiopian HLA frequencies, tyrosine kinase inhibitor, IL-6, IL-10, TNF-α, IFN-γ, TGF-β, Th17, Treg

## 1. Introduction

The distinguishing feature of chronic myeloid leukemia (CML) is the presence of the Philadelphia chromosome (Ph), detected in over 95% of patients. This genetic mutation involves the reciprocal exchange of the lengthy segments of chromosome 22 with the BCR gene and those of chromosome 9 with the ABL gene (9;22). This translocation is responsible for the persistent activation of tyrosine kinase activity and the suppression of apoptosis [[1](#B1-genes-15-00732),[2](#B2-genes-15-00732)]. Tyrosine kinase inhibitor (TKI) drugs have revolutionized CML treatment, extending the lives of patients in the chronic phase (CP) to near-normal expectancy, and for some individuals in deep molecular remission (DMR), they can be considered for a safe treatment-free remission (TFR) [[3](#B3-genes-15-00732),[4](#B4-genes-15-00732),[5](#B5-genes-15-00732)]. Nonetheless, up to 40–60% of patients can relapse following the discontinuation of TKI treatment [[6](#B6-genes-15-00732)] despite being in the CP during the course of TKI treatment. Both innate [[7](#B7-genes-15-00732)] and adaptive immunity, such as reduced immune suppressive cells [[8](#B8-genes-15-00732),[9](#B9-genes-15-00732)] and mesenchymal stromal cells with immunomodulatory roles [[10](#B10-genes-15-00732)], can impact DMR and thereby TFR.

Peptides from the BCR-ABL breakpoint can be processed within the cytosol, loaded onto major histocompatibility complex (MHC) molecules, and presented on the surface of CML cells, potentially allowing T cell recognition [[11](#B11-genes-15-00732)]. This suggests that ‘neoantigens’ derived from the BCR-ABL fusion gene may provoke specific immune responses, making CML cells a potential target for the immune system [[12](#B12-genes-15-00732)]. The MHC [[13](#B13-genes-15-00732)] comprises a series of genes and their products involved in protecting the respective individual against viral or bacterial infections and parasites. It is the most polymorphic system in humans. The products are named human leucocyte antigens (HLAs). The variability is specific and differs from landmark to landmark, reflecting the provenience of the individual. Functionally, HLA antigens are the counterparts of the T, B, and NK cell receptors and are involved in the self-non-self-recognition [[14](#B14-genes-15-00732)]. HLA is used diagnostically in stem cell and solid organ transplantation. An analysis of HLA ligands in samples of patients with primary CML has revealed previously unknown CML-associated antigens capable of triggering diverse T cell responses [[15](#B15-genes-15-00732)]. Additionally, the efficacy of TKIs is influenced by increased NK cell levels, which rely on interactions between NK cell receptors (KIRs) and HLA alleles [[16](#B16-genes-15-00732)]. A study of the European Group for Blood and Marrow Transplantation (EBMT) indicated that the presence of HLA-B8, particularly when co-expressed with HLA-A3, and the sole expression of HLA-DR4 were associated with a diminished incidence of CML [[17](#B17-genes-15-00732),[18](#B18-genes-15-00732)]. However, these findings necessitate further assessment using cutting-edge sequencing technologies across diverse patient groups and are especially confirmed in other populations.

In Iranian patients, haplotypes such as *HLA-DRB1*11/-DQB1*03:01/-DQA1*05:05* and *HLA-DRB1*04/-DQB1*03:02/-DQA1*03:01* seem to be predisposed to CML [[19](#B19-genes-15-00732),[20](#B20-genes-15-00732)]. In contrast, in Turkish patients, there was a protective effect of *DRB1*11* and *B*35*. Here, DRB4 homozygosity and *B*37* were regarded as risk factors [[21](#B21-genes-15-00732)]. The validation of genetic markers is a prerequisite for a reliable prognostic tool in managing CML and achieving TFR. Specific allele significance requires confirmation across diverse populations of patients with CML.

The level and activity of cytokines is attributed to single-nucleotide polymorphisms (SNPs) within the promoter region or other regulatory sequences of cytokine genes, which in turn impact the peripheral levels of cytokines and thus pathogenesis [[22](#B22-genes-15-00732),[23](#B23-genes-15-00732),[24](#B24-genes-15-00732)]. Population studies have indicated a strong association of SNPs with primary immune thrombocytopenia among Caucasian adults [[25](#B25-genes-15-00732)], the risk of leukemogenesis in a non-Chinese population [[26](#B26-genes-15-00732)], the severity of autoimmune hemolytic anemias in an Italian population [[27](#B27-genes-15-00732)], and the risk and overall survival of diffuse large B cell lymphoma in an Arab population [[28](#B28-genes-15-00732)]. These reports emphasize the importance of cytokine gene SNPs across diverse populations, shedding light on the intricacies of homeostasis and disease development as well as HLA allele polymorphism.

Functionally, TGF-β1, in conjunction with IL-6 or IL-21, supports Th17 cell differentiation. However, specific SNPs at positions +869 (T/C) and +915 (G/C) in the first exon of the TGF-β1 gene alter codon 10 (from leucine to proline) and codon 25 (from arginine to proline), resulting in varied expression levels [[29](#B29-genes-15-00732)]. IL-10, also known for its pivotal role in enhancing tumor-specific immune surveillance and curbing detrimental inflammation, is increasingly recognized in the host’s battle against malignant cells. Three crucial SNPs in its promoter region (−1082, −819, and −592) influence IL-10 production in cultured cells, particularly when an ‘A’ is present at positions −1082 and −592, resulting in decreased IL-10 levels [[30](#B30-genes-15-00732),[31](#B31-genes-15-00732)], which might curb deleterious inflammatory responses.

Similarly, functional studies have revealed that the G allele of the SNP variant −764G/C, situated nearby the IFN-γ promoter region, enhances promoter activity by a factor of two to three fold, correlating with both viral clearance and better treatment response in individuals with HCV infection [[32](#B32-genes-15-00732)]. In the context of malignancies, TNFα SNP (−863C/A, −857C/T, −308 G/A, and −238 G/A) was strongly associated with a higher risk of hepatocellular carcinoma (HCC) [[33](#B33-genes-15-00732)], and the different combinations of SNPs of TNFα (−238G/A or −308G/A) with IL-10 (−592C/A) were highly prevalent among patients with cervical cancer. PBMC (samples from patients with cervical cancer) activated with PHA also indicated that the TNFα (−308A/A) genotype demonstrated increased proliferation rates, elevated IL-4 and TGF-β, and decreased IL-2 levels [[34](#B34-genes-15-00732)], and it was postulated that SNPs of cytokine genes act as the potential predictors of cervical HPV infection progression to neoplasia.

In CML, cytokine gene polymorphism other than histocompatibility disparity had a vivid impact on the incidence of patients with CML compared to the healthy control group. The most frequent genotype/haplotype among patients with CML was TGF-β1TG/TG (at codon 10G/G and codon 25G/G), which correlates with high gene expression, followed by IFN-γA/T (at intron 1, +874A/T), which correlates with intermediate gene expression, and IL-10 ACC/ACC (−1082A/A, −819C/C, −592C/C) and ATA/ATA (−1082A/A, −819T/T, −592A/A), both of which correlate with low gene expression. In contrast, there is a low frequency among CML vs. control of the genotypes TGF-β1CG/CG (codon 10C/C and codon 25G/G) and IL-10 GCC/ACC (−1082G/A, −819C/C, −592C/C), both of which correlated with intermediate gene expression [[35](#B35-genes-15-00732)]. However, according to the study, the production of TGF-β1 in patients with CML is higher and IL-10 is lower than in normal subjects, suggesting the possibility of immune bias during the immunological surveillance of CML cells.

In addition to cytokine SNPs, another pivotal factor in immunogenetics is the polymorphism of human leukocyte antigens (HLAs).

Studies on HLA frequencies for the African population are scarce, with limited and outdated references, particularly for the Ethiopian population, often relying on now obsolete methods [[36](#B36-genes-15-00732),[37](#B37-genes-15-00732),[38](#B38-genes-15-00732)]. Only a few African studies, such as Abshir et al., reported high-resolution HLA data from Somalia [[39](#B39-genes-15-00732)]. Most genetic CML studies were attributed to the non-African population. Immune–genetic studies among patients from African populations have been crucial for understanding diverse genetic variations. These studies have provided insights into CML disease progression, TKI response, and possible genetic adaptations. The SNP cytokine analysis was conducted based on arbitrary values assigned for ‘low’, ‘intermediate’, and ‘high’ gene expression, respectively. This approach allows us to simulate and compute the ratios of the cumulative sum of five cytokines (IL-10, IL-6, IFN-γ, TGF-β1, and TNFα) and to calculate the Kaplan–Meier failure estimate of patients with CML with favorable and unfavorable treatment outcomes in the course of treatment for up to 12 years.

Histocompatibility disparity was observed among the patients regarding their treatment outcomes. In addition, a couple of new alleles were detected, which are still under investigation and require confirmation. One advantage of this study was that the HLA genotyping was performed using next-generation sequencing (NGS) with three-field resolution, which could be an asset for future population genetic studies.

## 2. Materials and Methods

### 2.1. Cohorts

A total of 162 and 63 cytogenetically confirmed Ph+ chronic myeloid leukemia (CML) patients were enrolled consecutively. Those patients were visiting the hematology clinic at Tikur Anbessa Specialized Hospital (TASH), Addis Ababa University (AAU), and Addis Ababa, Ethiopia. The study was approved by the Institutional Review Board (IRB), CHS-IRB 015/2023, of the college of health sciences of the AAU. Written informed consent was received from all participants. The patient cohort was divided into two subgroups, favorable and unfavorable, in terms of TKI response according to the updated NCCN guidelines for the treatment of CML [[40](#B40-genes-15-00732)].

In the overall cohort of patients with CML, 63/162 individuals (41.7%) underwent random selection for further assay—cytokine single-nucleotide polymorphism (SNP) analysis. Within this subset, the pre-treatment median age and duration were 37 years (range: 19–69) and 48 months (range: 1–144), respectively, with 69.8% exhibiting a high Sokal [[41](#B41-genes-15-00732)] risk score ([Table 1](#genes-15-00732-t001)).

### Table 1.

| Clinical Feature of Total CML Cohort | n | % |
| --- | --- | --- |
| Number of patients | 63 |   |
| Male-to-female ratio | 2 | 58.7:41.3 |
| Stage of disease at diagnosis |   |   |
| CP | 59 | 93.7 |
| AP | 2 | 3.2 |
| BP | 2 | 3.2 |
| Splenomegaly at diagnosis | 62 | 98 |
| Hemoglobin at diagnosis |   |   |
| </=8 g/dL * | 10 | 15.9 |
| <12 g/dL | 35 | 55.5 |
| >/=12 g/dL | 18 | 28.6 |
| Sokal risk score at diagnosis |   |   |
| Low | 3 | 4.8 |
| Intermediate | 16 | 25.4 |
| High | 44 | 69.8 |
| WBC < 100,000/µL at diagnosis | 15 | 23.8 |
| Platelet > 450,000/µL at diagnosis | 17 | 26.9 |
| Median age at diagnosis in years (range) | 37(19–69) |   |
| Median treatment duration, months (range) | 48 (1–144) |   |
| Current BCR-ABL 1% (IS); median (95% CI) | 33.8 (3.4–38.0) |   |

Table 1 Caption: Clinical features of patients with chronic myeloid leukemia.

During data analysis, subjects in accelerated or blast crises and those solely on cytoreductive therapy or those who were TKI naïve during treatment were excluded. Among 51 patients in the chronic phase of CML undergoing TKI treatment (81%), either as a first-line or subsequent therapy, 29 (46%) experienced unfavorable clinical outcomes, while 22 (35%) experienced favorable ones. The median (range) of BCR-ABL1 (IS) was 42.2 (2.0–78.2) for unfavorable outcomes and 0.02 (0.0001–0.25) for favorable ones ([Table 2](#genes-15-00732-t002)).

### Table 2.

| Post-Treatment Clinical Features (n = 51 *) | Clinical Outcome, Median (Range) ** |  |
| --- | --- | --- |
| Favorable (n = 22) | Unfavorable (n = 29) |  |
| Treatment duration, months | 60 (9–144) | 36 (1–144) |
| WBC × 103/µL | 6.05 (3.4–10.7) | 19.1 (2.8–180) |
| Platelets t × 103/µL | 225.5 (74–390) | 182 (24–1314) |
| Hemoglobin, g/dL | 14,2 (13–17) | 12 (7.6–16.9) |
| BCR::ABL1 (IS) | 0.02 (0.0001–0.25) | 42.2 (2.02–78.22) |

Table 2 Caption: Post-treatment clinical features of patients with CML.

### 2.2. DNA Preparation

DNA was isolated from EDTA blood samples according to the manufacturer’s recommendation (QIAamp DNA Blood Mini Kit, QIAGEN, and Hilden, Germany). In short, Qiagen columns (QIAamp DNA mini kit; Qiagen, Hilden, Germany) were used for the extraction of DNA from total white blood cells obtained from EDTA-treated peripheral venous blood collected with Vacutainer Tubes (Becton Dickinson). Briefly, samples were mixed with four-fold ice-cold NH_4_Cl lysing buffer (pH 7.4), kept for 20–25 min on ice until RBC lysis was apparently complete. After washing with ice-cold PBS, pelleted total white blood cells were lysed in guanidine thiocyanate buffer (GTC) containing 8 µL/mL β-mercaptoethanol. A QIAshredder spin columns (QIAGEN #79656) were used to extract DNA according to the manufacturer’s protocol. The purity and quantity of eluted total DNA was measured using a Thermo Scientific™ NanoDrop™ One Spectrophotometer (Waltham, MA, USA).

### 2.3. Cytokine Genotyping

Cytokine genotyping was performed by the polymerase chain reaction-sequence-specific primer (PCR-SSP) method, using the Cytokine Genotyping Tray kit according to the manufacturer’s instructions (Cytokine Genotyping Tray, One Lambda Canoga Park, CA, USA). Briefly, the amplified products of TNF-α, TGF-β, IL-10, IL-6, and IFN-γ alleles and internal control, the human β-globin gene, were analyzed via agarose gel electrophoresis. The interpretation of SNPs (see [Table 3](#genes-15-00732-t003)) for these five cytokines was based on the presence or the absence of specific amplified fragments.

### Table 3.

| Cytokine | Genotype | All Patients with CML (n = 63) |  | Only TKI Treated Patients with CML (n = 51) ** |  |  |  | p-Value * |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Favorable Group (n = 22) |  | Unfavorable Group (n = 29) |  |  |  |  |  |  |
| n | % | n | % | n | % |  |  |  |
| TNF-α (promoter −308G, −308A) | G/G low | 60 | 95.2 | 21 | 95.5 | 27 | 93.1 | 1.000 |
| G/A high | 2 | 3.2 | 1 | 4.5 | 1 | 3.4 | 1.000 |  |
| A/A high | 1 | 1.2 | 0 | 0.0 | 1 | 3.4 | 1.000 |  |
| IL-6 (promoter −174C, −174G) | C/C low | 3 | 4.8 | 0 | 0.0 | 1 | 3.4 | 1.000 |
| G/G high | 48 | 76.2 | 19 | 86.4 | 20 | 69.0 | 0.193 |  |
| G/C high | 12 | 19.0 | 3 | 13.6 | 8 | 27.6 | 0.312 |  |
| IFN-γ (intron +874T, +874A) | A/A low | 31 | 49.2 | 9 | 40.9 | 18 | 62.1 | 0.164 |
| T/A intermediate | 25 | 39.7 | 11 | 50.0 | 9 | 31.0 | 0.248 |  |
| T/T high | 7 | 11.1 | 2 | 9.1 | 2 | 6.9 | 1.000 |  |
| TGF-β (codon 10T, 10C, 25C, 25G) | C/C G/C low | 2 | 3.2 | 2 | 9.1 | 0 | 0.0 | 0.181 |
| C/C C/C low | 1 | 1.6 | 0 | 0.0 | 1 | 3.4 | 1.000 |  |
| T/T C/C low | 0 | 0.0 | 0 | 0.0 | 0 | 0.0 | na |  |
| T/C C/C low | 0 | 0.0 | 0 | 0.0 | 0 | 0.0 | na |  |
| T/C G/C intermediate | 6 | 9.5 | 1 | 4.5 | 4 | 13.8 | 0.375 |  |
| C/C G/G intermediate | 9 | 14.3 | 3 | 13.6 | 5 | 17.2 | 1.000 |  |
| T/T G/C intermediate | 0 | 0.0 | 0 | 0.0 | 0 | 0.0 | 1.000 |  |
| T/T G/G high | 22 | 34.9 | 7 | 31.8 | 11 | 37.9 | 0.770 |  |
| T/C G/G high | 23 | 36.5 | 9 | 40.9 | 8 | 27.6 | 0.377 |  |
| IL-10 (promoter −1082A, −1082G, −819T, −819C, −592A, −592C) | ACC/ACC low | 1 | 1.6 | 0 | 0.0 | 0 | 0.0 | na |
| ACC/ATA low | 11 | 17.5 | 5 | 22.7 | 6 | 20.7 | 1.000 |  |
| ATA/ATA low | 3 | 4.8 | 1 | 4.5 | 2 | 6.9 | 1.000 |  |
| GCC/ACC intermediate | 18 | 28.6 | 3 | 13.6 | 9 | 31.0 | 0.193 |  |
| GCC/ATA intermediate | 16 | 25.4 | 5 | 22.7 | 6 | 20.7 | 1.000 |  |
| GCC/GCC high | 14 | 22.2 | 8 | 36.4 | 6 | 20.7 | 0.342 |  |

Table 3 Caption: Cytokine polymorphism in CML favorable and unfavorable groups.

Using simple descriptive statistics, patients with and without a favorable clinical outcome were compared based on the respective SNP that were associated phenotypically with low-, intermediate-, or high-gene expression levels of each cytokine. Furthermore, the ratio of pro-inflammatory regulatory cytokine (IFN-γ, TNF-α, IL-6) to anti-inflammatory regulatory cytokine (IL-10, TGF-β1) gene expression level was computed among patients with CML having favorable and unfavorable clinical outcomes.

### 2.4. HLA High-Resolution Typing

HLA typing was performed using commercial test kits, the AllType NGS 11-Loci (One Lambda, West Hills, CA, USA) and AlloSeq^®^ Tx 17 (CareDx, San Francisco, CA, USA). For some samples, we additionally used the NANOTYPE^TM^ (Omixon Biocomputing Kft., Budapest, Hungary) system to clarify the results. The sequencing was carried out on a MiSeq Sequencing device (Illumina, San Diego, CA, USA), strictly following the manufacturer’s recommendations. Furthermore, samples were confirmed with the NANOTYPE^TM^ using the Nanopore technology (Oxford Nanopore, Oxford, UK), especially in the case of rare and new alleles. The laboratory in Leipzig (Germany) has European (EFI) and national (DAkkS) accreditation for NGS HLA typing.

### 2.5. Data Analysis and Interpretation

The cytokine data obtained through PCR-SSP was entered into an evaluation form provided by the company, and SNP-associated cytokine expression levels (‘low’, ‘intermediate’, or ‘high’) were assigned following the provided protocol. To facilitate statistical analysis, arbitrary values (1, 2, or 3, respectively) were assigned to each level of cytokine expression. To evaluate the immunological homeostasis of patients with CMOL based on SNP-related cytokine expression levels, the sum ratio of multiple cytokines—both pro-inflammatory and anti-inflammatory—in various combinations was computed. The overall median of all patients served as a cutoff for Kaplan–Meier analysis (Mantel–Haenszel model) to estimate the probability of relapse-free survival in the favorable versus unfavorable groups (refer to the analysis examples shown below). The analysis was adjusted for variables such as age and treatment duration (Stata Statistical Software, version 13, StataCorp LLC, College Station, TX, USA). The primary objective of this statistical approach was to evaluate how subject’s cytokine SNP affects the risk of relapse.

The calculation of the ratio of cytokine cocktail:

| {TNFα+IL6+TGFβ1}{IL10+ IFNγ}={(G/G‘low’=1)+(G/G‘high’=3)+(T/TG/G‘high’=3)}{(GCC/GCC‘high’=3)+(T/A‘Intermediate’=2)}=7/5=1.4 |
| --- |
If the overall (n = 51) median {(n + 1) ÷ 2}th ratio of the cytokine cocktail was 1.0, then, the cutoff for Kaplan–Meier would be set at a value of ‘1’, and the probability of treatment failures of the above subject (1.4) was calculated using K-M.

The HLA results were determined by analyzing the fastq files in the Type Stream Visual (TSV) software v2.0 (One Lambda) with the AlloSeq Assign v1.02 software (CareDx) or with NanoTYPER^TM^ v1.1.0 (Omixon). The current IMGT/HLA database at the time of the analysis was used for the HLA analysis [[42](#B42-genes-15-00732)]. For statistical analysis, we used SPSS v29.

In the present study, we directly compared the HLA typing results of the favorable and unfavorable cohorts making use of the one-tailed Fisher exact test, without making use of a control cohort, since the patients were recruited from all Ethiopian regions.

## 3. Results

### 3.1. Summary and Descriptive Statistics of the SNP vs. Clinical Outcomes

For the initial analysis of the cytokine polymorphism of all patients with CML on TKI treatment (n = 51), we observed that some cytokine polymorphisms do not occur in this Ethiopian patient cohort, such as TGF-β T/T C/C (low), TGF-β T/C C/C (low), and TGF-β T/T G/C (intermediate) ([Table 3](#genes-15-00732-t003)).

In the total cohort of patients with CML ([Table 3](#genes-15-00732-t003)), 60 out of the total 63 cases of patients with CML (95%) had TNF-α polymorphism associated with ‘low’ gene expression (G/G), and the proportion among subgroups was similar. Among the subset of patients with CML in the chronic phase undergoing TKI initiation (n = 51), ‘intermediate’ TGF-β1 gene expression was 4/22 (18%) for patients with favorable and 9/29 (31%) with unfavorable clinical outcomes. Within the favorable subgroup, G/G ‘high’ IL-6 gene expression was evident in 19/22 individuals (86%), whereas the unfavorable group exhibited a lower expression in 20/29 cases (69%). The IFN-γ A/A ‘low’ gene expression was 9/22 (41%) and 18/29 (62%) among favorable and unfavorable clinical outcomes, respectively. For IL-10, the GCC/GCC ‘high’ gene expression among favorable and unfavorable subgroups were 8/22 (36%) and 15/22 (21%), respectively. Conversely, in the unfavorable subgroup, 15/29 individuals (52%) demonstrated a slightly higher proportion of ‘intermediate’ gene expression levels for IL-10 with GCC/ACC and GCC/ATA compared to those in the favorable subgroup, i.e., 8/22 (36%).

### 3.2. Estimates of the Probability of Relapse-Free Survival

As per [Table 3](#genes-15-00732-t003), no statistically significant association was observed between clinical outcomes and the individual cytokine gene expression profiles. However, with the simultaneous computation of multiple cytokines, the following pattern was unveiled.

[Figure 1](#genes-15-00732-f001) depicts the median ratio of multiple cytokine gene expression profiles pertinent to designated SNPs, plotting the treatment duration and success and failure variables as remission and relapse, respectively. The relative risk (RR), 95% CI, and *p*-values were calculated using Kaplan–Meier based on the Mantel–Haenszel model ([Figure 1](#genes-15-00732-f001)a–d).

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e070/11203291/9c6834d5a0b1/genes-15-00732-g001a.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=11203291_genes-15-00732-g001a.jpg)

Age–adjusted survival function: (a) ratio of TNFα + IL-6 + TGFβ1/IL-10 + IFNγ; (b) ratio of IL-6/IL-10 + TGFβ1; (c) ratio of IL-6 + TNFα/IL-10; and (d) ratio of IFNγ + TNFα/IL-10.

The ratio of the combination of TNFα+IL6+TGFβ1/IL10+IFNγ gene polymorphism was computed, and a possible association with the probability of relapse-free survival was explored. Kaplan–Meier (Mantel–Haenszel model) estimates of the probability of relapse-free survival, for a gene expression ratio above the overall median, were relatively high (RR: 3.01; 95% CI: 1.1–8.3; *p*-value = 0.0261) ([Figure 1](#genes-15-00732-f001)a). The ratio of the results of IL6/IL10+TGFβ1 was computed. Similarly, subjects with a gene expression ratio above the overall median were 2.4 times more at risk for relapse during the period of TKI treatment (RR: 2.4; 95% CI: 1.1–5.2; *p*-value = 0.022) ([Figure 1](#genes-15-00732-f001)b). The effect of individual cytokine SNPs (ratio of IL6/IL10) was not statistically significant (RR: 0.445; 95% CI: 0.180–1.1; *p*-value = 0.072) nor was the ratio of IL6/TGFβ1 (RR: 0.013). Though patients with a higher than median gene expression (ratio of TNFα/IL10) had twice the risk for relapse, it was not statistically significant (RR: 1.92; 95% CI: 0.877–4.20; *p*-value: = 0.097). Regarding the ratio of IL6+TNFα/IL10, the relative risk of relapse is twice as high among patients with a higher than median gene expression profile (RR: 2.05; 95% CI: 0.94–4.46; *p*-value = 0.064), albeit not significant ([Figure 1](#genes-15-00732-f001)c). The ratio of IFNγ+TNFα/IL10, in patients with a gene expression profile above the median ratio, had twice the risk of relapse during the treatment period of 10 years or less (RR: 1.94; 95% CI: 0.89–4.23; *p*-value = 0.0876) ([Figure 1](#genes-15-00732-f001)d).

### 3.3. HLA Polymorphism in the Cohort of Patients Treated for CML

Among all patients with CML in this study, we observed a high frequency of *A*02:01:01, A*03:01:01,* and *A*68:02:01* for A locus alleles, ranging from 8.3 to 16.0%. In the B locus, the most frequently observed alleles were the following: *B*07:02:01, B*15:220:01, B*49:01:01*, and *B*53:01:01* in the range of 7.4% to 9.6%. For the C locus, the alleles *C*04:01:01, C*07:01:01*, and *C*06:02:01* were most prevalent (13.6–14.2%). Interestingly, the allele *B*15:220:01* was found with a frequency of 7.7% in the Ethiopian population, much higher than the observed 1.2% frequency in Gaza, 0.2% in Brazil, and <0.02% in Europe [[43](#B43-genes-15-00732)]. According to Schell et al. [[44](#B44-genes-15-00732)], this B allele is regarded as a conserved haplotype associated with *A*66:01*, *C*12:03*, and *DRB1*07:01*. Many of the alleles observed at a high prevalence in this study have also been reported in other populations. These include *A*02:01:01, A*03:01:01, B*07:02:01*, and *C*07:01:01*, as well as *HLA-A*30, B*53*, *and B*58*, which are common in African populations ([Table 4](#genes-15-00732-t004)).

### Table 4.

| Locus |   |   |   |   | Favorable Group |  | Unfavorable Group |  |   |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| HLA-A | Phenotype Count (n = 162) | Allele Count (2n = 324) | Phenotype Frequency (%) | Frequency (%) | Allele Count (2n = 96) | Allele Frequency (%) | Allele Count (2n = 206) | Allele Frequency (%) | Fisher Exact (p-Value) |
| A*01:01:01 | 22.0 | 23.0 | 0.2 | 7.1 | 10 | 10.4 | 13 | 6.3 | ns |
| A*01:02:01 | 1.0 | 1.0 | 0.0 | 0.3 | 1 | 1.0 | 0 | 0 | ns |
| A*01:03:01 | 21.0 | 21.0 | 0.2 | 6.5 | 7 | 7.3 | 12 | 5.8 | ns |
| A*02:01:01 | 50.0 | 52.0 | 0.4 | 16.0 | 13 | 13.5 | 37 | 18.0 | ns |
| A*02:02:01 | 26.0 | 26.0 | 0.2 | 8.0 | 13 | 13.5 | 12 | 5.8 | ns |
| A*02:05:01 | 12.0 | 12.0 | 0.1 | 3.7 | 5 | 5.2 | 7 | 3.4 | ns |
| A*02:05:02 | 1.0 | 1.0 | 0.0 | 0.3 | 0 | 0 | 1 | 0.5 | ns |
| A*02:06:01 | 1.0 | 1.0 | 0.0 | 0.3 | 0 | 0 | 1 | 0.5 | ns |
| A*02:104 | 1.0 | 1.0 | 0.0 | 0.3 | 1 | 1.0 | 0 | 0 | ns |
| A*02:14 | 1.0 | 1.0 | 0.0 | 0.3 | 0 | 0 | 1 | 0.5 | ns |
| A*02:22:01 | 4.0 | 4.0 | 0.0 | 1.2 | 1 | 1.0 | 2 | 1.0 | ns |
| A*03:01:01 | 28.0 | 29.0 | 0.2 | 9.0 | 5 | 5.2 | 23 | 11.2 | 0.0347 * |
| A*03:02:01 | 2.0 | 2.0 | 0.0 | 0.6 | 0 | 0 | 2 | 1.0 |  |
| A*11:01:01 | 3.0 | 3.0 | 0.0 | 0.9 | 2 | 2.1 | 1 | 0.5 | ns |
| A*23:01:01 | 5.0 | 5.0 | 0.0 | 1.5 | 1 | 1.0 | 4 | 1.9 | ns |
| A*23:17:01 | 10.0 | 10.0 | 0.1 | 3.1 | 0 | 0 | 9 | 4.4 | 0.0285 ** |
| A*24:02:01 | 5.0 | 5.0 | 0.0 | 1.5 | 1 | 1.0 | 4 | 1.9 | ns |
| A*26:01:01 | 1.0 | 1.0 | 0.0 | 0.3 | 0 | 0 | 0 | 0 | ns |
| A*29:02:01 | 1.0 | 1.0 | 0.0 | 0.3 | 0 | 0 | 1 | 0.5 | ns |
| A*30:01:01 | 19.0 | 19.0 | 0.2 | 5.9 | 2 | 2.1 | 14 | 6.8 | ns |
| A*30:02:01 | 19.0 | 19.0 | 0.2 | 5.9 | 5 | 5.2 | 11 | 5.3 | ns |
| A*30:04:01 | 7.0 | 7.0 | 0.1 | 2.2 | 2 | 2.1 | 5 | 2.4 | ns |
| A*30:10 | 5.0 | 6.0 | 0.0 | 1.9 | 2 | 2.1 | 3 | 1.5 | ns |
| A*31:04 | 1.0 | 1.0 | 0.0 | 0.3 | 0 | 0 | 1 | 0.5 | ns |
| A*32:01:01 | 10.0 | 10.0 | 0.1 | 3.1 | 3 | 3.1 | 7 | 3.4 | ns |
| A*33:03:01 | 4.0 | 4.0 | 0.0 | 1.2 | 0 | 0 | 4 | 1.9 | ns |
| A*33:123N | 2.0 | 2.0 | 0.0 | 0.6 | 0 | 0 | 2 | 1.0 | ns |
| A*66:01:01 | 10.0 | 10.0 | 0.1 | 3.1 | 4 | 4.2 | 5 | 2.4 | ns |
| A*66:03:01 | 3.0 | 3.0 | 0.0 | 0.9 | 0 | 0 | 3 | 1.5 | ns |
| A*68:01:01 | 10.0 | 10.0 | 0.1 | 3.1 | 4 | 4.2 | 6 | 2.9 | ns |
| A*68:02:01 | 26.0 | 27.0 | 0.2 | 8.3 | 8 | 8.3 | 14 | 6.8 | ns |
| A*74:01:01 | 3.0 | 3.0 | 0.0 | 0.9 | 3 | 3.1 | 0 | 0 | ns |
| A*74:03 | 4.0 | 4.0 | 0.0 | 1.2 | 3 | 3.1 | 1 | 0.5 | ns |
| Locus |   |   |   |   | Favorable group |  | Unfavorable group |  |   |
| HLA-B | phenotype count (n = 162) | allele count (2n = 324) | phenotype frequency (%) | frequency (%) | allele count (2n = 96) | allele frequency (%) | allele count (2n = 206) | allele frequency (%) | Fisher exact (p-value) |
| B*07:02:01 | 31.0 | 31.0 | 0.2 | 9.6 | 12 | 12.5 | 17 | 8.3 | ns |
| B*07:05:01 | 15.0 | 16.0 | 0.1 | 4.9 | 3 | 3.1 | 10 | 4.9 | ns |
| B*07:06:01 | 1.0 | 1.0 | 0.0 | 0.3 | 0 | 0 | 1 | 0.5 | ns |
| B*08:01:01 | 6.0 | 6.0 | 0.0 | 1.9 | 2 | 2.1 | 4 | 1.9 | ns |
| B*13:02:01 | 16.0 | 17.0 | 0.1 | 5.2 | 6 | 6.3 | 11 | 5.3 | ns |
| B*14:01:01 | 2.0 | 2.0 | 0.0 | 0.6 | 1 | 1 | 1 | 0.5 | ns |
| B*14:02:01 | 13.0 | 13.0 | 0.1 | 4.0 | 3 | 3.1 | 10 | 4.9 | ns |
| B*15:10:01 | 5.0 | 5.0 | 0.0 | 1.5 | 1 | 1 | 3 | 1.5 | ns |
| B*15:16:01 | 1.0 | 1.0 | 0.0 | 0.3 | 0 | 0 | 1 | 0.5 | ns |
| B*15:17:01 | 4.0 | 4.0 | 0.0 | 1.2 | 1 | 1 | 3 | 1.5 | ns |
| B*15:18:01 | 2.0 | 2.0 | 0.0 | 0.6 | 0 | 0 | 2 | 1 | ns |
| B*15:220:01 | 24.0 | 25.0 | 0.2 | 7.7 | 7 | 7.3 | 17 | 8.3 | ns |
| B*18:01:01 | 7.0 | 7.0 | 0.1 | 2.2 | 3 | 3.1 | 4 | 1.9 | ns |
| B*27:03 | 6.0 | 6.0 | 0.0 | 1.9 | 2 | 2.1 | 4 | 1.9 | ns |
| B*27:05:02 | 1.0 | 1.0 | 0.0 | 0.3 | 0 | 0 | 1 | 0.5 | ns |
| B*27:26 | 1.0 | 1.0 | 0.0 | 0.3 | 0 | 0 | 1 | 0.5 | ns |
| B*35:01:01 | 5.0 | 5.0 | 0.0 | 1.5 | 1 | 1 | 4 | 1.9 | ns |
| B*35:03:01 | 1.0 | 1.0 | 0.0 | 0.3 | 0 | 0 | 1 | 0.5 | ns |
| B*35:08:01 | 3.0 | 3.0 | 0.0 | 0.9 | 0 | 0 | 3 | 1.5 | ns |
| B*37:01:01 | 1.0 | 1.0 | 0.0 | 0.3 | 1 | 1 | 0 | 0 | ns |
| B*39:10:01 | 10.0 | 10.0 | 0.1 | 3.1 | 3 | 3.1 | 5 | 2.4 | ns |
| B*39:24:01 | 4.0 | 5.0 | 0.0 | 1.5 | 2 | 2.1 | 3 | 1.5 | ns |
| B*40:01:02 | 3.0 | 3.0 | 0.0 | 0.9 | 1 | 1 | 2 | 1 | ns |
| B*40:02:01 | 1.0 | 1.0 | 0.0 | 0.3 | 0 | 0 | 1 | 0.5 | ns |
| B*41:01:01 | 16.0 | 16.0 | 0.1 | 4.9 | 5 | 5.2 | 7 | 3.4 | ns |
| B*41:02:01 | 6.0 | 6.0 | 0.0 | 1.9 | 3 | 3.1 | 2 | 1 | ns |
| B*44:03:01 | 16.0 | 18.0 | 0.1 | 5.6 | 4 | 4.2 | 14 | 6.8 | ns |
| B*44:03:02 | 1.0 | 1.0 | 0.0 | 0.3 | 0 | 0 | 1 | 0.5 | ns |
| B*47:01:01 | 5.0 | 6.0 | 0.0 | 1.9 | 1 | 1 | 5 | 2.4 | ns |
| B*49:01:01 | 25.0 | 25.0 | 0.2 | 7.7 | 3 | 3.1 | 19 | 9.2 | ns |
| B*50:01:01 | 4.0 | 4.0 | 0.0 | 1.2 | 3 | 3.1 | 1 | 0.5 | ns |
| B*51:01:01 | 12.0 | 13.0 | 0.1 | 4.0 | 3 | 3.1 | 9 | 4.4 | ns |
| B*51:08:01 | 2.0 | 2.0 | 0.0 | 0.6 | 1 | 1 | 1 | 0.5 | ns |
| B*53:01:01 | 24.0 | 24.0 | 0.2 | 7.4 | 8 | 8.3 | 15 | 7.3 | ns |
| B*55:01:01 | 1.0 | 1.0 | 0.0 | 0.3 | 0 | 0 | 0 | 0 | ns |
| B*57:01:01 | 3.0 | 3.0 | 0.0 | 0.9 | 1 | 1 | 2 | 1 | 0.037 * |
| B*57:02:01 | 5.0 | 5.0 | 0.0 | 1.5 | 2 | 2.1 | 2 | 1 |  |
| B*57:03:01 | 12.0 | 12.0 | 0.1 | 3.7 | 7 | 7.3 | 5 | 2.4 | 0.0451 ** |
| B*58:01:01 | 18.0 | 18.0 | 0.1 | 5.6 | 5 | 5.2 | 12 | 5.8 | ns |
| B*73:01 | 2.0 | 2.0 | 0.0 | 0.6 | 1 | 1 | 1 | 0.5 | ns |
| B*81:02:02 | 1.0 | 1.0 | 0.0 | 0.3 | 0 | 0 | 1 | 0.5 | ns |
| Locus |   |   |   |   | Favorable group |  | Unfavorable group |  |   |
| HLA-C | phenotype count (n = 162) | allele count (2n = 324) | phenotype frequency (%) | frequency (%) | allele count (2n = 96) | allele frequency (%) | allele count (2n = 206) | allele frequency (%) | Fisher exact (p-value) |
| C*02:02:02 | 9.0 | 9.0 | 0.1 | 2.8 | 2 | 2.1 | 7 | 3.4 | ns |
| C*03:02:01 | 1.0 | 1.0 | 0.0 | 0.3 | 1 | 1 | 0 | 0 | ns |
| C*03:02:02 | 10.0 | 10.0 | 0.1 | 3.1 | 2 | 2.1 | 7 | 3.4 | ns |
| C*03:04:01 | 2.0 | 2.0 | 0.0 | 0.6 | 0 | 0 | 2 | 1 | ns |
| C*03:04:02 | 5.0 | 5.0 | 0.0 | 1.5 | 1 | 1 | 3 | 1.5 | ns |
| C*04:01:01 | 43.0 | 46.0 | 0.3 | 14.2 | 12 | 12.5 | 31 | 15 | ns |
| C*04:07:01 | 2.0 | 2.0 | 0.0 | 0.6 | 2 | 2.1 | 0 | 0 | ns |
| C*06:02:01 | 40.0 | 43.0 | 0.3 | 13.3 | 17 | 17.7 | 26 | 12.6 | ns |
| C*07:01:01 | 43.0 | 44.0 | 0.3 | 13.6 | 9 | 9.4 | 32 | 15.5 | ns |
| C*07:01:02 | 10.0 | 10.0 | 0.1 | 3.1 | 5 | 5.2 | 5 | 2.4 | ns |
| C*07:02:01 | 33.0 | 33.0 | 0.3 | 10.2 | 11 | 11.5 | 20 | 9.7 | ns |
| C*07:04:01 | 8.0 | 8.0 | 0.1 | 2.5 | 4 | 4.2 | 4 | 1.9 | ns |
| C*07:06:01 | 1.0 | 1.0 | 0.0 | 0.3 | 0 | 0 | 1 | 0.5 | ns |
| C*07:18:01 | 7.0 | 7.0 | 0.1 | 2.2 | 2 | 2.1 | 5 | 2.4 | ns |
| C*07:623 | 1.0 | 1.0 | 0.0 | 0.3 | 0 | 0 | 1 | 0.5 | ns |
| C*08:02:01 | 13.0 | 13.0 | 0.1 | 4.0 | 3 | 3.1 | 10 | 4.9 | ns |
| C*08:183 | 1.0 | 1.0 | 0.0 | 0.3 | 0 | 0 | 1 | 0.5 | ns |
| C*08:28 | 1.0 | 1.0 | 0.0 | 0.3 | 1 | 1 | 0 | 0 | ns |
| C*12:03:01 | 9.0 | 9.0 | 0.1 | 2.8 | 3 | 3.1 | 4 | 1.9 | ns |
| C*12:167 | 1.0 | 1.0 | 0.0 | 0.3 | 0 | 0 | 1 | 0.5 | ns |
| C*14:02:01 | 2.0 | 2.0 | 0.0 | 0.6 | 1 | 1 | 1 | 0.5 | ns |
| C*14:03:01 | 2.0 | 2.0 | 0.0 | 0.6 | 0 | 0 | 2 | 1 | ns |
| C*15:02:01 | 2.0 | 2.0 | 0.0 | 0.6 | 0 | 0 | 1 | 0.5 | ns |
| C*15:05:01 | 5.0 | 5.0 | 0.0 | 1.5 | 2 | 2.1 | 3 | 1.5 | ns |
| C*15:05:02 | 16.0 | 17.0 | 0.1 | 5.2 | 3 | 3.1 | 11 | 5.3 | ns |
| C*16:01:01 | 5.0 | 5.0 | 0.0 | 1.5 | 3 | 3.1 | 2 | 1 | ns |
| C*16:02:01 | 6.0 | 6.0 | 0.0 | 1.9 | 1 | 1 | 5 | 2.4 | ns |
| C*16:04:01 | 6.0 | 6.0 | 0.0 | 1.9 | 1 | 1 | 5 | 2.4 | ns |
| C*17:01:01 | 19.0 | 19.0 | 0.2 | 5.9 | 6 | 6.3 | 8 | 3.9 | ns |
| C*17:03:01 | 4.0 | 4.0 | 0.0 | 1.2 | 2 | 2.1 | 2 | 1 | ns |
| C*18:01:01 | 1.0 | 1.0 | 0.0 | 0.3 | 1 | 1 | 0 | 0 | ns |
| C*18:02:01 | 8.0 | 8.0 | 0.1 | 2.5 | 1 | 1 | 6 | 2.9 | ns |
| Locus |   |   |   |   | Favorable group |  | Unfavorable group |  |   |
| HLA-DRB1 | phenotype count (n = 162) | allele count (2n = 324) | phenotype frequency (%) | frequency (%) | allele count (2n = 96) | allele frequency (%) | allele count (2n = 206) | allele frequency (%) | Fisher exact (p-value) |
| DRB1*01:02:01 | 36 | 40 | 0.2 | 12.3 | 9 | 9.4 | 26 | 12.6 | ns |
| DRB1*03:01:01 | 30 | 30 | 0.2 | 9.3 | 11 | 11.5 | 17 | 8.3 | ns |
| DRB1*03:144 | 3 | 3 | 0.0 | 0.9 | 0 | 0.0 | 3 | 1.5 | ns |
| DRB1*04:01:01 | 7 | 7 | 0.0 | 2.2 | 3 | 3.1 | 4 | 1.9 | ns |
| DRB1*04:03:01 | 9 | 9 | 0.1 | 2.8 | 1 | 1.0 | 7 | 3.4 | ns |
| DRB1*04:04:01 | 10 | 10 | 0.1 | 3.1 | 5 | 5.2 | 4 | 1.9 | ns |
| DRB1*04:05:01 | 8 | 8 | 0.0 | 2.5 | 4 | 4.2 | 4 | 1.9 | ns |
| DRB1*04:06:01 | 1 | 1 | 0.0 | 0.3 | 1 | 1.0 | 0 | 0.0 | ns |
| DRB1*04:06:02 | 2 | 2 | 0.0 | 0.6 | 0 | 0.0 | 2 | 1.0 | ns |
| DRB1*04:08:01 | 1 | 1 | 0.0 | 0.3 | 0 | 0.0 | 1 | 0.5 | ns |
| DRB1*04:201 | 1 | 1 | 0.0 | 0.3 | 0 | 0.0 | 1 | 0.5 | ns |
| DRB1*07:01:01 | 57 | 64 | 0.4 | 19.8 | 17 | 17.7 | 40 | 19.4 | ns |
| DRB1*08:04:01 | 16 | 16 | 0.1 | 4.9 | 4 | 4.2 | 10 | 4.9 | ns |
| DRB1*08:08 | 4 | 4 | 0.0 | 1.2 | 1 | 1.0 | 3 | 1.5 | ns |
| DRB1*10:01:01 | 8 | 8 | 0.0 | 2.5 | 2 | 2.1 | 6 | 2.9 | ns |
| DRB1*11:01:01 | 5 | 5 | 0.0 | 1.5 | 0 | 0.0 | 4 | 1.9 | ns |
| DRB1*11:01:02 | 1 | 1 | 0.0 | 0.3 | 0 | 0.0 | 1 | 0.5 | ns |
| DRB1*13:01:01 | 6 | 6 | 0.0 | 1.9 | 2 | 2.1 | 4 | 1.9 | ns |
| DRB1*13:02:01 | 56 | 63 | 0.3 | 19.4 | 20 | 20.8 | 43 | 20.9 | ns |
| DRB1*13:03:01 | 11 | 12 | 0.1 | 3.7 | 5 | 5.2 | 7 | 3.4 | ns |
| DRB1*14:01:01 | 1 | 1 | 0.0 | 0.3 | 0 | 0.0 | 1 | 0.5 | ns |
| DRB1*14:54:01 | 5 | 5 | 0.0 | 1.5 | 3 | 3.1 | 1 | 0.5 | ns |
| DRB1*15:01:01 | 3 | 3 | 0.0 | 0.9 | 1 | 1.0 | 2 | 1.0 | ns |
| DRB1*15:03:01 | 24 | 24 | 0.1 | 7.4 | 7 | 7.3 | 15 | 7.3 | ns |
| HLA-DRB345 | phenotype count (n = 162) | allele count (2n = 324) | phenotype frequency (%) | frequency (%) | allele count (2n = 96) | allele frequency (%) | allele count (2n = 206) | allele frequency (%) | Fisher exact (p-value) |
| DRB3*01:01:02 | 18 | 21 | 0.1 | 6.5 | 5 | 5.2 | 15 | 7.3 | ns |
| DRB3*02:02:01 | 47 | 53 | 0.3 | 16.4 | 18 | 18.8 | 30 | 14.6 | ns |
| DRB3*03:01:01 | 54 | 70 | 0.3 | 21.6 | 19 | 19.8 | 51 | 24.8 | ns |
| DRB3*03:15 | 1 | 1 | 0.0 | 0.3 | 1 | 1.0 | 0 | 0.0 | ns |
| DRB4*01:01:01 | 13 | 21 | 0.1 | 6.5 | 2 | 2.1 | 19 | 9.2 | 0.0127 |
| DRB4*01:03:01 | 69 | 77 | 0.4 | 23.8 | 28 | 29.2 | 36 | 17.5 | 0.0058 |
| DRB4*01:03:01:02N | 15 | 16 | 0.1 | 4.9 | 4 | 4.2 | 12 | 5.8 | ns |
| DRB4*01:03:03 | 6 | 6 | 0.0 | 1.9 | 2 | 2.1 | 4 | 1.9 | ns |
| DRB5*01:01:01 | 30 | 31 | 0.2 | 9.6 | 9 | 9.4 | 20 | 9.7 | ns |
| non # | 28 | 28 | 0.2 | 8.6 | 8 | 8.3 | 19 | 9.2 | ns |
| Locus |   |   |   |   | Favorable group |  | Unfavorable group |  |   |
| HLA-DQA1 | phenotype count (n = 162) | allele count (2n = 324) | phenotype frequency (%) | frequency (%) | allele count (2n = 96) | allele frequency (%) | allele count (2n = 206) | allele frequency (%) | Fisher exact (p-value) |
| DQA1*01:01:02 | 36 | 40 | 0.2 | 12.3 | 9 | 9.4 | 26 | 12.6 | ns |
| DQA1*01:02:01 | 75 | 82 | 0.5 | 25.3 | 26 | 27.1 | 55 | 26.7 | ns |
| DQA1*01:02:02 | 8 | 8 | 0.0 | 2.5 | 4 | 4.2 | 3 | 1.5 | ns |
| DQA1*01:03:01 | 6 | 6 | 0.0 | 1.9 | 2 | 2.1 | 4 | 1.9 | ns |
| DQA1*01:04:01 | 3 | 3 | 0.0 | 0.9 | 2 | 2.1 | 0 | 0.0 | ns |
| DQA1*01:05:01 | 8 | 8 | 0.0 | 2.5 | 2 | 2.1 | 6 | 2.9 | ns |
| DQA1*01:05:02 | 3 | 3 | 0.0 | 0.9 | 1 | 1.0 | 2 | 1.0 | ns |
| DQA1*02:01:01 | 57 | 64 | 0.4 | 19.8 | 15 | 15.6 | 42 | 20.4 | ns |
| DQA1*03:01:01 | 14 | 14 | 0.1 | 4.3 | 3 | 3.1 | 10 | 4.9 | ns |
| DQA1*03:03:01 | 25 | 25 | 0.2 | 7.7 | 11 | 11.5 | 13 | 6.3 | ns |
| DQA1*04:01:01 | 4 | 4 | 0.0 | 1.2 | 1 | 1.0 | 3 | 1.5 | ns |
| DQA1*04:01:02 | 4 | 4 | 0.0 | 1.2 | 2 | 2.1 | 2 | 1.0 | ns |
| DQA1*05:01:01 | 32 | 33 | 0.2 | 10.2 | 11 | 11.5 | 20 | 9.7 | ns |
| DQA1*05:05:01 | 30 | 30 | 0.2 | 9.3 | 7 | 7.3 | 20 | 9.7 | ns |
| HLA-DQB1 | phenotype count (n = 162) | allele count (2n = 324) | phenotype frequency (%) | frequency (%) | allele count (2n = 96) | allele frequency (%) | allele count (2n = 206) | allele frequency (%) | Fisher exact (p-value) |
| DQB1*02:01:01 | 31 | 33 | 0.2 | 10.2 | 12 | 12.5 | 19 | 9.2 | ns |
| DQB1*02:02:01 | 56 | 64 | 0.3 | 19.8 | 16 | 16.7 | 41 | 19.9 | ns |
| DQB1*03:01:01 | 32 | 33 | 0.2 | 10.2 | 9 | 9.4 | 22 | 10.7 | ns |
| DQB1*03:01:04 | 3 | 3 | 0.0 | 0.9 | 0 | 0.0 | 2 | 1.0 | ns |
| DQB1*03:02:01 | 16 | 16 | 0.1 | 4.9 | 5 | 5.2 | 11 | 5.3 | ns |
| DQB1*03:03:02 | 7 | 7 | 0.0 | 2.2 | 3 | 3.1 | 4 | 1.9 | ns |
| DQB1*04:02:01 | 10 | 10 | 0.1 | 3.1 | 5 | 5.2 | 4 | 1.9 | ns |
| DQB1*04:02:13 | 2 | 2 | 0.0 | 0.6 | 1 | 1.0 | 1 | 0.5 | ns |
| DQB1*04:87 | 4 | 4 | 0.0 | 1.2 | 1 | 1.0 | 3 | 1.5 | ns |
| DQB1*05:01:01 | 44 | 49 | 0.3 | 15.1 | 11 | 11.5 | 32 | 15.5 | ns |
| DQB1*05:03:01 | 3 | 3 | 0.0 | 0.9 | 2 | 2.1 | 0 | 0.0 | ns |
| DQB1*06:02:01 | 23 | 23 | 0.1 | 7.1 | 5 | 5.2 | 17 | 8.3 | ns |
| DQB1*06:03:01 | 11 | 11 | 0.1 | 3.4 | 4 | 4.2 | 6 | 2.9 | ns |
| DQB1*06:04:01 | 41 | 45 | 0.3 | 13.9 | 16 | 16.7 | 29 | 14.1 | ns |
| DQB1*06:09:01 | 20 | 20 | 0.1 | 6.2 | 5 | 5.2 | 15 | 7.3 | ns |
| DQB1*06:27:01 | 1 | 1 | 0.0 | 0.3 | 1 | 1.0 | 0 | 0.0 | ns |
| Locus |   |   |   |   | Favorable group |  | Unfavorable group |  |   |
| HLA-DPA1 | phenotype count (n = 162) | allele count (2n = 324) | phenotype frequency (%) | frequency (%) | allele count (2n = 96) | allele frequency (%) | allele count (2n = 206) | allele frequency (%) | Fisher exact (p-value) |
| DPA1*01:03:01 | 163.0 | 184.0 | 1.0 | 56.8 | 55 | 57.3 | 118 | 57.3 | ns |
| DPA1*01:04 | 6.0 | 6.0 | 0.0 | 1.9 | 2 | 2.1 | 4 | 1.9 | ns |
| DPA1*01:04:01 | 18.0 | 20.0 | 0.1 | 6.2 | 4 | 4.2 | 15 | 7.3 | ns |
| DPA1*01:05 | 1.0 | 1.0 | 0.0 | 0.3 | 0 | 0.0 | 1 | 0.5 | ns |
| DPA1*01:30 | 5.0 | 5.0 | 0.0 | 1.5 | 1 | 1.0 | 4 | 1.9 | ns |
| DPA1*01:58 | 5.0 | 5.0 | 0.0 | 1.5 | 2 | 2.1 | 3 | 1.5 | ns |
| DPA1*01:58:01 | 1.0 | 1.0 | 0.0 | 0.3 | 0 | 0.0 | 1 | 0.5 | ns |
| DPA1*02:01:01 | 73.0 | 79.0 | 0.5 | 24.4 | 23 | 24.0 | 48 | 23.3 | ns |
| DPA1*02:01:07 | 4.0 | 4.0 | 0.0 | 1.2 | 1 | 1.0 | 2 | 1.0 | ns |
| DPA1*02:02:02 | 5.0 | 5.0 | 0.0 | 1.5 | 4 | 4.2 | 1 | 0.5 | ns |
| DPA1*02:09 | 4.0 | 4.0 | 0.0 | 1.2 | 1 | 1.0 | 2 | 1.0 | ns |
| DPA1*03:01:01 | 8.0 | 8.0 | 0.0 | 2.5 | 2 | 2.1 | 6 | 2.9 | ns |
| DPA1*03:01:02 | 1.0 | 1.0 | 0.0 | 0.3 | 0 | 0.0 | 1 | 0.5 | ns |
| DPA1*04:02 | 1.0 | 1.0 | 0.0 | 0.3 | 1 | 1.0 | 0 | 0.0 | ns |
| HLA-DPB1 | phenotype count (n = 162) | allele count (2n = 324) | phenotype frequency (%) | frequency (%) | allele count (2n = 96) | allele frequency (%) | allele count (2n = 206) | allele frequency (%) | Fisher exact (p-value) |
| DPB1*01:01:01 | 2.0 | 2.0 | 0.0 | 0.6 | 3 | 3.1 | 1 | 0.5 | ns |
| DPB1*01:01:02 | 5.0 | 5.0 | 0.0 | 1.5 | 0 | 0.0 | 2 | 1.0 | ns |
| DPB1*02:01:02 | 71.0 | 76.0 | 0.4 | 23.5 | 23 | 24.0 | 50 | 24.3 | ns |
| DPB1*02:01:18 | 1.0 | 1.0 | 0.0 | 0.3 | 0 | 0.0 | 1 | 0.5 | ns |
| DPB1*03:01:01 | 12.0 | 12.0 | 0.1 | 3.7 | 2 | 2.1 | 9 | 4.4 | ns |
| DPB1*04:01:01 | 63.0 | 65.0 | 0.4 | 20.1 | 19 | 19.8 | 43 | 20.9 | ns |
| DPB1*04:02:01 | 5.0 | 5.0 | 0.0 | 1.5 | 1 | 1.0 | 3 | 1.5 | ns |
| DPB1*05:01:01 | 1.0 | 1.0 | 0.0 | 0.3 | 1 | 1.0 | 0 | 0.0 | ns |
| DPB1*09:01:01 | 6.0 | 6.0 | 0.0 | 1.9 | 0 | 0.0 | 6 | 2.9 | ns |
| DPB1*104:01:01 | 15.0 | 16.0 | 0.1 | 4.9 | 7 | 7.3 | 7 | 3.4 | ns |
| DPB1*105:01:01 | 2.0 | 2.0 | 0.0 | 0.6 | 0 | 0.0 | 2 | 1.0 | ns |
| DPB1*105:02:01 | 1.0 | 1.0 | 0.0 | 0.3 | 0 | 0.0 | 1 | 0.5 | ns |
| DPB1*11:01:01 | 17.0 | 18.0 | 0.1 | 5.6 | 6 | 6.3 | 8 | 3.9 | ns |
| DPB1*124:01:01 | 3.0 | 3.0 | 0.0 | 0.9 | 1 | 1.0 | 2 | 1.0 | ns |
| DPB1*13:01:01 | 12.0 | 13.0 | 0.1 | 4.0 | 4 | 4.2 | 6 | 2.9 | ns |
| DPB1*133:01 | 1.0 | 1.0 | 0.0 | 0.3 | 0 | 0.0 | 1 | 0.5 | ns |
| DPB1*135:01:01 | 1.0 | 1.0 | 0.0 | 0.3 | 1 | 1.0 | 0 | 0.0 | ns |
| DPB1*14:01:01 | 6.0 | 6.0 | 0.0 | 1.9 | 3 | 3.1 | 3 | 1.5 | ns |
| DPB1*15:01:01 | 27.0 | 29.0 | 0.2 | 9.0 | 8 | 8.3 | 19 | 9.2 | ns |
| DPB1*17:01:01 | 33.0 | 34.0 | 0.2 | 10.5 | 8 | 8.3 | 24 | 11.7 | ns |
| DPB1*290:01 | 1.0 | 1.0 | 0.0 | 0.3 | 0 | 0.0 | 1 | 0.5 | ns |
| DPB1*30:01:01 | 7.0 | 7.0 | 0.0 | 2.2 | 2 | 2.1 | 5 | 2.4 | ns |
| DPB1*34:01:01 | 4.0 | 4.0 | 0.0 | 1.2 | 2 | 2.1 | 2 | 1.0 | ns |
| DPB1*39:01:01 | 4.0 | 4.0 | 0.0 | 1.2 | 0 | 0.0 | 4 | 1.9 | ns |
| DPB1*47:01:01 | 2.0 | 2.0 | 0.0 | 0.6 | 1 | 1.0 | 1 | 0.5 | ns |
| DPB1*49:01:01 | 1.0 | 1.0 | 0.0 | 0.3 | 0 | 0.0 | 1 | 0.5 | ns |
| DPB1*54:01 | 2.0 | 2.0 | 0.0 | 0.6 | 0 | 0.0 | 2 | 1.0 | ns |
| DPB1*55:01:01 | 4.0 | 4.0 | 0.0 | 1.2 | 2 | 2.1 | 2 | 1.0 | ns |
| DPB1*665:01:01 | 1.0 | 1.0 | 0.0 | 0.3 | 1 | 1.0 | 0 | 0.0 | ns |
| DPB1*702:01 | 1.0 | 1.0 | 0.0 | 0.3 | 1 | 1.0 | 0 | 0.0 | ns |

Table 4 Caption: HLA class I (A, B, C) and HLA class II (DRB1, DQB1, DQA1, DPA1, DPB1) polymorphisms in an Ethiopian population with CML.

For HLA class II (DRB1, DRB345, DQA1, DQB1, DPA1, DPB1), we observed the DRB1 alleles *DRB1*07:01:01*, *DRB1*13:02:01*, *DRB1*01:02:01*, *DRB4*01:03:01, DRB3*03:01:01,* and *DRB3*02:02:01* with frequencies greater than 10%. The alleles of the *DRB1*12* group are missing in the studied cohort. With respect to the DQA1 locus, the most common alleles in this Ethiopian CML cohort were *DQA1*01:02:01*, *DQA1*02:01:01*, *DQA1*01:01:02*, and *DQA1*05:01:01* (each >10%). For the DQB1 locus, the three most frequent alleles were *DQB1*02:02:01*, *DQB1*05:01:01*, and *DQB1*06:04:01*. For DPA1, we found *DPA1*01:03:01* (>56%) followed by *DPA1*02:01:01* (>24%). Finally, for the DPB1 locus, the most frequent alleles were *DPB1*02:01:02* (>23%), *DPB1*04:01:01* (>20%), and *DPB1*17:01:01* (>10%). Most of these HLA class II alleles were also found in other populations ([Table 5](#genes-15-00732-t005)), including European Caucasian, such as *DRB1*15:03:01*, *DQB1*06:09:01*, *DPB1*15:01:01*, and *DPB1*17:01:01* [[43](#B43-genes-15-00732)].

In the treated cohorts, we found less HLA-*A*23:17:01* (*p*-value = 0.0285) in the favorable group compared to the unfavorable group. The frequency of *A*03* is increased by a factor of 2.3 in the unfavorable group (*p*-value = 0.0347). *B*57:03:01* showed a higher frequency of 7.3% in the favorable group compared to the unfavorable group (2.4%) (*p* = 0.0451). A comparison of allele frequencies of *HLA-DRB4*01:01:01* and *HLA-DRB4*01:03:01* revealed a significant accumulation of the first in the unfavorable group (9.2%), whereas the second was found to be increased in the favorable group with a frequency of 29.2% (*p* = 0.0127 and 0.0058, respectively). For all other HLA loci (C, DRB1, DQA1, DQB1, DPA1 and DPB1), no significant differences between the two cohorts could be found ([Table 4](#genes-15-00732-t004)).

The HLA polymorphism in the Ethiopian cohort with CML is shown in [Table 4](#genes-15-00732-t004), reflecting the expected significant differences in the European and other cohorts ([Table 5](#genes-15-00732-t005)). Several possible new alleles were observed during this study and are under investigation [[45](#B45-genes-15-00732)].

### Table 5.

| Population | Region | Most Common HLA Alleles | Remarks | References |
| --- | --- | --- | --- | --- |
| Europe | Romania | A*02:01, A*03:01, A*01:01, A*24:02, A*26:01, B*51:01, B*08:01, B*35:01, B*14:02, B*07:02, DRB1*03:01, DRB1*16:01, DRB1*11:01, DRB1*07:01, DRB1*15:01 | donors | [46] |
| Germany | A*02:01, B*07:02, C*07:01, DRB1*15:01, DRB1*07:01, DQB1*03:01, DQB1*06:02 | healthy, ill people, COVID-19 infected | [47] |  |
| Europe | A*01:01, A*24:02, A*33:01, A*68:01, B*08:01, B*44:01, B*57:01, C*12:02, DRB1*01:01, DRB1*01:02, DRB1*03:01, DRB1*04:01, DRB1*04:03, DRB1*07:01, DRB1*11:01, DRB1*11:04, DRB1*13:01, DRB1*13:02, DRB1*15:01, DQB1*02:01, DQB1*02:02, DQB1*03:01, DQB1*03:02, DQB1*04:02, DQB1*05:01, DQB1*05:02, DQB1*05:03, DQB1*06:01, DQB1*06:03, DQB1*06:04, DPB1*01:01, DPB1*02:01, DPB1*04:01, DPB1*04:02, DPB1*06:01, DPB1*11:01 |   | [48] |  |
| European descent group | A*02 (22.4%), A*03 (15.7%), A*24 (9.4%), B*07 (14.6%), B*44 (10,6%), B*51 (7.3%), C*07 (24.6%), C*03 (12.6%), C*06 (11.5%), DRB1*15 (15.2%), DRB1*04 (14.3%), DRB1*03 (11.5%) | all | [49] |  |
| Africa | Sudan, Chad, Burkina-Faso | B*53 | no A*25, A*26,low A*11, no B*48, B*54, B*67 | [50] |
| Ethiopia | DR3, DR4 | low DR5, DR8, DR10 | [37] |  |
| Kenya, Tanzania, Uganda | A*02:01:01 (10.90%), B*58:02 (8.79%), C*06:02:01 (16.98%), DPA1*01:03:01 (40.60%), DPB1*01:01:01 (23.45%), DQA1*01:02:01 (31.03%), DQB1*03:01:01 (21.79%), DRB1*11:01:02 (11.65%), DRB3*02:02:01 (31.65%),DRB4*01:01:01 (10.50%), DRB5*01:01:01 (10.50%) | higher homozygosity: HLA-B, DQA1, DQB1, DRB1 high frequency: DRB1*11 (21.51%) | [51] |  |
| Gabonese, Liberians | DR15/16 DQB1*06:02 | Lebanese low DR11 | [38] |  |
| African descent group | A*30 (13.1%), A*68 (12.8%), A*02 (12.3%), B*15 (12.6%), B*58 (11.7%), B*44 (9.9), C*07 (16.8%), C*06 (16.4%), C*04 (12.8%), DRB1*03 (17.2%), DRB1*13 (16.1%), DRB1*11 (16%) | all | [49] |  |
| Asia | Republic of Korea | A*33:03:01-B*44:03:01-C*14:03-DRB1*13:02:01-DQB1*06:04:01-DPB1*04:01:01 (2.89%), A*33:03:01-B*44:03:01-C*14:03 (4.91%), DRB1*08:03:02-DQA1*01:03:01-DQB1*06:01:01-DPA1*02:02:02-DPB1*05:01:01 (5.41%), DRB1*04:05:01-DRB4*01:03:01 (12.72%), DQA1*01:03:01-DQB1*06:01:01 (13.01%), DPA1*02:02:02-DPB1*05:01:01 (30.83%) | adults from medical school | [52] |
| Arab Emirates | Saudi | A*02:01:01G (20.2%), A*24:02:01G (7.5%), B*51:01:01G (19.0%), B*50:01:01G (12.3%), C*06:02:01G (16.7%), C*07:02:01G (12.2%), DRB1*07:01:01 (15.7%), DRB1∗03:01:01G (13.3%); DQB1∗02:01:01G (29.9%), DQB1∗03:02:01G (13.2%); andDPB1∗04:01:01G (35.2%), DPB1∗02:01:02G (21.8%) | stem cell donors | [53] |

Table 5 Caption: HLA class I and HLA class II allele frequencies in various populations.

## 4. Discussion

### 4.1. Cytokine Polymorphism

Unfortunately, due to a lack of healthy cohorts as controls, we were limited in making strong inferences, being a general limitation of this study. Therefore, we made comparisons between the favorable and unfavorable groups of patients treated with TKIs. The immunomodulatory effect of TKIs and the achievement of durable DMR, which is a prognostic condition for TFR, and the presence of quiescent TKI-insensitive residual CML leukemic stem cells (LSCs) during sustained TFR draw significant scientific attention in comprehending the immune landscape of patients with CML, which could perhaps underscore the pivotal role of effector and suppressor arms of the immune landscape [[54](#B54-genes-15-00732),[55](#B55-genes-15-00732),[56](#B56-genes-15-00732),[57](#B57-genes-15-00732)].

Our study did not probe the immune cellular compartments, but we rather focused on the genetic landscape of cytokine gene SNPs and HLA allele polymorphisms, which could be a detrimental factor in the balance of effector and suppressor polarization of the immune system. Hence, we examined the concerted effect of multiple cytokine SNPs and HLAs and their correlation with the prognosis of patients with CML.

Clinical outcomes were categorized into two types, favorable and unfavorable, based on the European Leukemia Net (ELN) guidelines, in reference to the treatment milestone achieved during treatment.

Overall, patient cohorts with unfavorable clinical outcomes, in the absence of TK domain mutation, had poor hematological remission: WBC (median: 19.2 Range: 2.8–180) vs. patients with favorable treatment outcome (median: 6.05; range: 3.4–10.7) ([Table 2](#genes-15-00732-t002)). These are not particularly surprising clinical phenomena: poor hematological remission has previously been associated with an immunosuppressive trait due to the heightened expression of the PD-L1 gene in monocytes and basophils, as well as other genes linked with reactive oxygen species (ROS) in neutrophils [[54](#B54-genes-15-00732),[55](#B55-genes-15-00732)], thus supporting the evidence for the ELN advice of pre-treating high leukocyte counts with cytoreductive drugs prior to TKI initiation [[58](#B58-genes-15-00732)].

In view of individual polymorphisms and the cocktails of five cytokine SNPs and 11 loci HLA polymorphisms, an appraisal of the immunomodulatory activity of TKI on overall clinical outcomes is noteworthy.

In general, IL-6 has been associated with a wide variety of inflammatory states [[56](#B56-genes-15-00732)]. Thus, it became the cytokine grabbing perhaps the most research attention during the COVID-19 pandemic.

In our patients, G/G or G/C—both ‘high expressors’ for rs1474348, IL-6 (−174)—were predominant in all cohorts, which was in line with the high allele frequency in an African population [[57](#B57-genes-15-00732)]. Of the two, the frequency of the G/G genotype was 86% and 69% among the favorable and unfavorable subgroups, respectively, and the G/C genotype was 14.0% and 28.0%, respectively.

In Turkish patients with CML vs. healthy controls, there was a significant difference in the frequency of the genotype: G/G (88.3% vs. 50% *p*-value = 0.010) and G/C (12% vs. 43.2%, *p*-value = 0.002) [[22](#B22-genes-15-00732)]. Even though our study lacks healthy controls, the proportion of G/G was higher than G/C in patients with CML with both favorable and unfavorable outcomes, which is in line with the Turkish study. With regard to functional characteristic, G/C and G/G genotypes (both associated with ‘high’ gene expression of IL-6) were significantly associated with the pathogenesis of Parkinson’s disease [[59](#B59-genes-15-00732)] and reduced disease-free survival of breast carcinoma [[60](#B60-genes-15-00732)], respectively. The C/C genotype (associated with ‘low’ expression of IL-6) appears to benefit the immune landscape, possibly by avoiding a vigorous inflammatory response.

The benefit of low expression of IL-6 could be partially explained by the study finding that DC-mediated T cell activation was enhanced in IL-6 KO (knock out) mice [[61](#B61-genes-15-00732)], and the level of serum IL-6 levels in CML-blast crises was higher than that in the chronic phase, and the level in the chronic phase was higher than that in normal controls [[62](#B62-genes-15-00732)].

The polymorphism of the TNF-α (−857) C/T TT genotype ‘higher expressers’ was associated with the risk of AML leukemogenesis [[63](#B63-genes-15-00732)]. Other studies focusing on the TNF-α rs1800629 SNP (−308G/A) have revealed that the G/G ‘low expresser’ genotype was associated with a high risk of adult B-ALL in Sudanese patients, while the −308G/A polymorphism was linked to SLE [[64](#B64-genes-15-00732),[65](#B65-genes-15-00732)] and AML mortality [[66](#B66-genes-15-00732)]. In our study, 48 out of 51 patients (94.1%) exhibited the G/G genotype. Similar findings of a lack of a significant prognostic role among patients with CML were reported elsewhere [[35](#B35-genes-15-00732)].

IFN-γ crucially combats viruses and exhibits anti-tumor and anti-proliferative properties [[67](#B67-genes-15-00732)]. Nuclear factor kappa-B (NF-κB) preferentially binds to +874T allele, perhaps boosting IFN-γ expression and affecting the pathogenesis of tuberculosis [[68](#B68-genes-15-00732)], and T carriers had statistically increased CML risk in a dominant model (i.e., TT + TA vs. AA) [[69](#B69-genes-15-00732)]. With regard to its functional impact, IFN-γ stimulation resulted in the upregulation of the major histocompatibility complex (MHC)-II and its master regulator class II transactivator (CIITA) in CML stem/progenitor cells, and the enhanced proliferation of the responder CD4^+^CD69^+^ T cells [[70](#B70-genes-15-00732)]. The proportion of T/T genotypes (high expressers) was very low in all patients with CML, 4/51 (7.8%). Though difficult to make a direct inference, due to the lack of healthy cohorts as a control, low gene expression could have a possibly negative link with CML, which prompts the necessity for further study.

Regarding the SNP TGF-β1 (codons 10 and 25), there was no significant (*p*-value > 0.05) difference between patients with CML with favorable and unfavorable outcomes. From the overall cohorts, only 2/22 and 1/29 had ‘low expressers’ among favorable and unfavorable clinical outcomes. A clinically study [[71](#B71-genes-15-00732)] indicated that poor TKI responders had an enrichment of genes associated with the TGF-β and TNF-α pathways and a quiescence signature that maintains leukemia-initiating cells in chronic myeloid leukemia [[72](#B72-genes-15-00732),[73](#B73-genes-15-00732)].

IL-10 functions at the crossroads of immune stimulation and suppression in different malignancies, according to a meta-analysis [[31](#B31-genes-15-00732)]. Three crucial SNPs in its promoter region (−1082, −819, and −592) influence IL-10 production, and when an ‘A’ is present at positions −1082 and −592, this results in decreased IL-10 levels [[30](#B30-genes-15-00732)]. In our study, a high frequency, namely, 9/14 (64.3%), of ‘high-expression’ IL-10 genotypes (GCC/GCC; −1082G/G, −819 C/C, −592 C/C) was among the favorable clinical outcomes, while ‘intermediate-expression’ IL-10 (GCC/ACC, ATA; −1082G/G, −819 C/C, T, −592 C/C, A) was higher (15/22 (68.2%)) among unfavorable clinical outcomes (Pearson χ^2^ =3.6526; *p*-value = 0.056). The high expression of IL-10 appeared to contribute to a favorable clinical outcome. In line with this, findings from the ‘KIWI’ study suggested that the TKI withdrawal syndrome (TWS) could be linked to a decrease in IL-10 levels [[71](#B71-genes-15-00732)]. Nevertheless, an earlier work suggested that CML mesenchymal stem cell (MSC) overexpressing immunomodulatory factors TGFβ1, IL-6, and IL-10 might contribute to CML immune escape through the activation of myeloid-derived suppressor cells (G-MDSC) [[69](#B69-genes-15-00732),[74](#B74-genes-15-00732)]. Based on [Figure 1](#genes-15-00732-f001)a,b, the gene expression ratio of TNFα+IL-6+TGFβ1/IL-10+IFNγ and IL-6/IL-10+TGFβ1 above the overall median indicates a high probability of relapse, namely, a factor of 3 and 2.4 times, respectively. These intricate relationships may simply highlight the necessity for multi-cytokine computation rather than relying on a single level of cytokine as a potential prognostic tool.

The ratios of IL-6/IL-10 and TNFα/IL-10 showed no statistically significant differences among patient groups (RR: 0.445, 95% CI: 0.180–1.1, *p*-value = 0.072; and RR: 1.92, 95% CI: 0.877–4.20, *p*-value = 0.097, respectively). This clearly shows that a single or double cytokine would not provide a strong prognostic value.

In another combination ([Figure 1](#genes-15-00732-f001)c,d), the ratio of IL-6+TNFα/IL-10 and IFNγ + TNFα/IL-10, while showing an almost two-fold relative risk of relapse in both cases, was not statistically significant. The limited sample size, the lack of healthy controls, and the fact that 94.1% of our patients were low expressers of TNFα might have influenced the strength of the inference.

### 4.2. HLA Polymorphism

The HLA polymorphism within the studied population differs significantly from other populations, for example, the European population, with respect to the involvement of alleles not existing in the latter ([Table 5](#genes-15-00732-t005)).

The HLA polymorphism observed within the Ethiopian cohort with CML, when increased in number and compared to ‘healthy’ individuals, will form the basis for future analyses on the prevalence of the disease in Ethiopia. The latter is a limitation of the study presented here. We compared the results obtained from the favorable vs. the unfavorable group of patients as stated below. These results might offer the possibility to select patients for treatment on the basis of their immunogenetic background. It forms the basis for further testing. In this respect, extended typing via next- or third-generation sequencing seems to be the method of choice.

In general, in this study, we observed some rare and new alleles, which need confirmation and submission in the IGMT/HLA database. A comparison with a control cohort was not within the scope of the present study; instead, we concentrated on the possible involvement of HLA and cytokines in the results of TKI treatment. Nevertheless, the results of the CML group reveal the following: HLA-*A*03:01:01* was the second most frequent HLA-A allele in the population of Ethiopian patients with CML and third in the total MHC class I alleles. HLA-A*03 was reported to be associated with a lower risk in the European cohort, when co-expressed with HLA-B8, which in Europe represents the allele HLA-*B*08:01*. In the Ethiopian cohort analyzed here, HLA-*B*08:01* was observed in about 2% of the two CML subgroups. With respect to the co-expression of the two alleles, we found two cases of co-expression versus two cases on non-co-expression. The HLA-DRB1*04 allele group was found in 12% of the Ethiopian patients with CML, far more than the 2% reported for an East African population by Peterson et al. [[51](#B51-genes-15-00732)]. We were aware that due to the low number of patients in both groups, these data must be confirmed in a larger cohort. A differential association was observed for *DRB4*01:01:01*, which demonstrated significant accumulation in the unfavorable group (9% vs. 2%), while the HLA allele *DRB4*01:03:01* accumulated in the favorable group (29% vs. 18%). Interestingly, the alleles of the DRB1*11 allele group were almost completely missing (2/162 found), confirming the results of Fort et al. [[75](#B75-genes-15-00732)]. These alleles have been reported to be associated with CML. All other alleles and allele groups were similarly expressed in both cohorts of favorable and unfavorable treatment outcomes, with the exception of the non-availability of the HLA-DRB345 genes. This is the case for the haplotypes DRB1*01, DRB1*08, and DRB1*10 (*p* = 0.0183). This warrants further clarification in a more extended study.

These preliminary explorative studies revealed the possible associations of the CML clinical outcome with alleles of HLA and cytokine SNPs. Besides the need for a standardized mechanism of associating SNPs with their function, multi-SNPs, surpassing single SNPs, and HLA allele polymorphisms showed promise in predicting patient outcomes during TKI treatment, prompting further exploration for future TFR trials.

## Acknowledgments

We would like to thank all subjects who voluntarily took part in this study and all members of the staff in the administration, clinic, and laboratories for their immense support.

## Author Contributions

Conceptualization, C.L., S.K.B. and I.D.; methodology; S.K.B. and C.L.; software, S.K.B. and C.L.; formal analysis, S.K.B., C.L. and I.D.; data curation, S.K.B. and C.L.; writing—original draft preparation, C.L., S.K.B. and I.D.; writing—review and editing, C.L., S.K.B., R.H., T.K., S.H.C., A.S., F.T., A.T., A.G. and I.D.; supervision C.L.; project administration and funding acquisition, S.K.B. All authors have read and agreed to the published version of the manuscript.

## Institutional Review Board Statement

The study has been conducted according to the guidelines of the Declaration of Helsinki and approved by the College of Health Sciences’ Institutional Review Board (CHS-IRB 015/2023), Addis Ababa University (AAU), Addis Ababa, Ethiopia.

## Informed Consent Statement

Informed consent was obtained from all subjects involved in the study.

## Data Availability Statement

The data supporting this study’s findings are available from the corresponding author upon reasonable request.

## Conflicts of Interest

The authors declare no conflicts of interest.

## Funding Statement

S.K.B. received a short-term research fellowship from DAAD scholarship program, bi-nationally supervised doctoral degrees, 2018/19 (57381333).

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Data Availability Statement

The data supporting this study’s findings are available from the corresponding author upon reasonable request.

### Data Availability Statement

The data supporting this study’s findings are available from the corresponding author upon reasonable request.

## References

1. Deininger M.W., Goldman J.M., Melo J.V. The molecular biology of chronic myeloid leukemia. Blood. 2000;96:3343–3356. doi: 10.1182/blood.V96.10.3343.  [DOI](https://doi.org/10.1182/blood.V96.10.3343) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11071626/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=The%20molecular%20biology%20of%20chronic%20myeloid%20leukemia&author=M.W.%20Deininger&author=J.M.%20Goldman&author=J.V.%20Melo&volume=96&publication_year=2000&pages=3343-3356&pmid=11071626&doi=10.1182/blood.V96.10.3343&)

2. Bedi A., Zehnbauer B.A., Barber J.P., Sharkis S.J., Jones R.J. Inhibition of apoptosis by BCR-ABL in chronic myeloid leukemia. Blood. 1994;83:2038–2044. doi: 10.1182/blood.V83.8.2038.2038.  [DOI](https://doi.org/10.1182/blood.V83.8.2038.2038) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8161775/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Inhibition%20of%20apoptosis%20by%20BCR-ABL%20in%20chronic%20myeloid%20leukemia&author=A.%20Bedi&author=B.A.%20Zehnbauer&author=J.P.%20Barber&author=S.J.%20Sharkis&author=R.J.%20Jones&volume=83&publication_year=1994&pages=2038-2044&pmid=8161775&doi=10.1182/blood.V83.8.2038.2038&)

3. Bower H., Björkholm M., Dickman P.W., Höglund M., Lambert P.C., Andersson T.M.-L. Life Expectancy of Patients With Chronic Myeloid Leukemia Approaches the Life Expectancy of the General Population. JCO J. Clin. Oncol. 2016;34:2851–2857. doi: 10.1200/JCO.2015.66.2866.  [DOI](https://doi.org/10.1200/JCO.2015.66.2866) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27325849/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JCO%20J.%20Clin.%20Oncol.&title=Life%20Expectancy%20of%20Patients%20With%20Chronic%20Myeloid%20Leukemia%20Approaches%20the%20Life%20Expectancy%20of%20the%20General%20Population&author=H.%20Bower&author=M.%20Bj%C3%B6rkholm&author=P.W.%20Dickman&author=M.%20H%C3%B6glund&author=P.C.%20Lambert&volume=34&publication_year=2016&pages=2851-2857&pmid=27325849&doi=10.1200/JCO.2015.66.2866&)

4. Mahon F.-X., Réa D., Guilhot J., Guilhot F., Huguet F., Nicolini F., Legros L., Charbonnier A., Guerci A., Varet B., et al. Discontinuation of imatinib in patients with chronic myeloid leukaemia who have maintained complete molecular remission for at least 2 years: The prospective, multicentre Stop Imatinib (STIM) trial. Lancet Oncol. 2010;11:1029–1035. doi: 10.1016/S1470-2045(10)70233-3.  [DOI](https://doi.org/10.1016/S1470-2045(10)70233-3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20965785/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet%20Oncol.&title=Discontinuation%20of%20imatinib%20in%20patients%20with%20chronic%20myeloid%20leukaemia%20who%20have%20maintained%20complete%20molecular%20remission%20for%20at%20least%202%20years:%20The%20prospective,%20multicentre%20Stop%20Imatinib%20(STIM)%20trial&author=F.-X.%20Mahon&author=D.%20R%C3%A9a&author=J.%20Guilhot&author=F.%20Guilhot&author=F.%20Huguet&volume=11&publication_year=2010&pages=1029-1035&pmid=20965785&doi=10.1016/S1470-2045(10)70233-3&)

5. Hasford J., Baccarani M., Hoffmann V., Guilhot J., Saussele S., Rosti G., Guilhot F., Porkka K., Ossenkoppele G., Lindoerfer D., et al. Predicting complete cytogenetic response and subsequent progression-free survival in 2060 patients with CML on imatinib treatment: The EUTOS score. Blood. 2011;118:686–692. doi: 10.1182/blood-2010-12-319038.  [DOI](https://doi.org/10.1182/blood-2010-12-319038) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21536864/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Predicting%20complete%20cytogenetic%20response%20and%20subsequent%20progression-free%20survival%20in%202060%20patients%20with%20CML%20on%20imatinib%20treatment:%20The%20EUTOS%20score&author=J.%20Hasford&author=M.%20Baccarani&author=V.%20Hoffmann&author=J.%20Guilhot&author=S.%20Saussele&volume=118&publication_year=2011&pages=686-692&pmid=21536864&doi=10.1182/blood-2010-12-319038&)

6. Mahon F.-X. Treatment-free remission in CML: Who, how, and why? Hematol. Am. Soc. Hematol. Educ. Program. 2017;2017:102–109. doi: 10.1182/asheducation-2017.1.102.  [DOI](https://doi.org/10.1182/asheducation-2017.1.102) | [PMC free article](/articles/PMC6142562/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29222243/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hematol.%20Am.%20Soc.%20Hematol.%20Educ.%20Program&title=Treatment-free%20remission%20in%20CML:%20Who,%20how,%20and%20why?&author=F.-X.%20Mahon&volume=2017&publication_year=2017&pages=102-109&pmid=29222243&doi=10.1182/asheducation-2017.1.102&)

7. Schütz C., Inselmann S., Saussele S., Dietz C.T., Mu Ller M.C., Eigendorff E., Brendel C.A., Metzelder S.K., Bru Mmendorf T.H., Waller C., et al. Expression of the CTLA-4 ligand CD86 on plasmacytoid dendritic cells (pDC) predicts risk of disease recurrence after treatment discontinuation in CML. Leukemia. 2017;31:829–836. doi: 10.1038/leu.2017.9.  [DOI](https://doi.org/10.1038/leu.2017.9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28074067/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Leukemia&title=Expression%20of%20the%20CTLA-4%20ligand%20CD86%20on%20plasmacytoid%20dendritic%20cells%20(pDC)%20predicts%20risk%20of%20disease%20recurrence%20after%20treatment%20discontinuation%20in%20CML&author=C.%20Sch%C3%BCtz&author=S.%20Inselmann&author=S.%20Saussele&author=C.T.%20Dietz&author=M.C.%20Mu%20Ller&volume=31&publication_year=2017&pages=829-836&pmid=28074067&doi=10.1038/leu.2017.9&)

8. Hughes A., Clarson J., Tang C., Vidovic L., White D.L., Hughes T.P., Yong A.S.M. CML patients with deep molecular responses to TKI have restored immune effectors and decreased PD-1 and immune suppressors. Blood. 2017;129:1166–1176. doi: 10.1182/blood-2016-10-745992.  [DOI](https://doi.org/10.1182/blood-2016-10-745992) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28049640/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=CML%20patients%20with%20deep%20molecular%20responses%20to%20TKI%20have%20restored%20immune%20effectors%20and%20decreased%20PD-1%20and%20immune%20suppressors&author=A.%20Hughes&author=J.%20Clarson&author=C.%20Tang&author=L.%20Vidovic&author=D.L.%20White&volume=129&publication_year=2017&pages=1166-1176&pmid=28049640&doi=10.1182/blood-2016-10-745992&)

9. Ross D.M., Pagani I.S., Irani Y.D., Clarson J., Leclercq T., Dang P., McLean J., Saunders V.A., Carne L., Reynolds J., et al. Lenalidomide maintenance treatment after imatinib discontinuation: Results of a phase 1 clinical trial in chronic myeloid leukaemia. Br. J. Haematol. 2019;186:e56–e60. doi: 10.1111/bjh.15894.  [DOI](https://doi.org/10.1111/bjh.15894) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30916385/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Haematol.&title=Lenalidomide%20maintenance%20treatment%20after%20imatinib%20discontinuation:%20Results%20of%20a%20phase%201%20clinical%20trial%20in%20chronic%20myeloid%20leukaemia&author=D.M.%20Ross&author=I.S.%20Pagani&author=Y.D.%20Irani&author=J.%20Clarson&author=T.%20Leclercq&volume=186&publication_year=2019&pages=e56-e60&pmid=30916385&doi=10.1111/bjh.15894&)

10. Zhang X., Tu H., Yang Y., Jiang X., Hu X., Luo Q., Li J. Bone marrow-derived mesenchymal stromal cells promote resistance to tyrosine kinase inhibitors in chronic myeloid leukemia via the IL-7/JAK1/STAT5 pathway. J. Biol. Chem. 2019;294:12167–12179. doi: 10.1074/jbc.RA119.008037.  [DOI](https://doi.org/10.1074/jbc.RA119.008037) | [PMC free article](/articles/PMC6690703/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31235520/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem.&title=Bone%20marrow-derived%20mesenchymal%20stromal%20cells%20promote%20resistance%20to%20tyrosine%20kinase%20inhibitors%20in%20chronic%20myeloid%20leukemia%20via%20the%20IL-7/JAK1/STAT5%20pathway&author=X.%20Zhang&author=H.%20Tu&author=Y.%20Yang&author=X.%20Jiang&author=X.%20Hu&volume=294&publication_year=2019&pages=12167-12179&pmid=31235520&doi=10.1074/jbc.RA119.008037&)

11. Clark R.E., Dodi I.A., Hill S.C., Lill J.R., Aubert G., Macintyre A.R., Rojas J., Bourdon A., Bonner P.L., Wang L., et al. Direct evidence that leukemic cells present HLA-associated immunogenic peptides derived from the BCR-ABL b3a2 fusion protein. Blood. 2001;98:2887–2893. doi: 10.1182/blood.v98.10.2887.  [DOI](https://doi.org/10.1182/blood.v98.10.2887) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11698267/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Direct%20evidence%20that%20leukemic%20cells%20present%20HLA-associated%20immunogenic%20peptides%20derived%20from%20the%20BCR-ABL%20b3a2%20fusion%20protein&author=R.E.%20Clark&author=I.A.%20Dodi&author=S.C.%20Hill&author=J.R.%20Lill&author=G.%20Aubert&volume=98&publication_year=2001&pages=2887-2893&pmid=11698267&doi=10.1182/blood.v98.10.2887&)

12. Lim S.H., Coleman S. Chronic myeloid leukemia as an immunological target. Am. J. Hematol. 1997;54:61–67. doi: 10.1002/(SICI)1096-8652(199701)54:1<61::AID-AJH9>3.0.CO;2-2.  [DOI](https://doi.org/10.1002/(SICI)1096-8652(199701)54:1&#x0003c;61::AID-AJH9&#x0003e;3.0.CO;2-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8980262/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Hematol.&title=Chronic%20myeloid%20leukemia%20as%20an%20immunological%20target&author=S.H.%20Lim&author=S.%20Coleman&volume=54&publication_year=1997&pages=61-67&pmid=8980262&doi=10.1002/(SICI)1096-8652(199701)54:1&#x0003c;61::AID-AJH9&#x0003e;3.0.CO;2-2&)

13. Thorsby E. A short history of HLA. Tissue Antigens. 2009;74:101–116. doi: 10.1111/j.1399-0039.2009.01291.x.  [DOI](https://doi.org/10.1111/j.1399-0039.2009.01291.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19523022/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Tissue%20Antigens&title=A%20short%20history%20of%20HLA&author=E.%20Thorsby&volume=74&publication_year=2009&pages=101-116&pmid=19523022&doi=10.1111/j.1399-0039.2009.01291.x&)

14. Zinkernagel R.M., Doherty P.C. The discovery of MHC restriction. Immunol. Today. 1997;18:14–17. doi: 10.1016/s0167-5699(97)80008-4.  [DOI](https://doi.org/10.1016/s0167-5699(97)80008-4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9018968/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Immunol.%20Today&title=The%20discovery%20of%20MHC%20restriction&author=R.M.%20Zinkernagel&author=P.C.%20Doherty&volume=18&publication_year=1997&pages=14-17&pmid=9018968&doi=10.1016/s0167-5699(97)80008-4&)

15. Bilich T., Nelde A., Bichmann L., Roerden M., Salih H.R., Kowalewski D.J., Schuster H., Tsou C.-C., Marcu A., Neidert M.C., et al. The HLA ligandome landscape of chronic myeloid leukemia delineates novel T-cell epitopes for immunotherapy. Blood. 2019;133:550–565. doi: 10.1182/blood-2018-07-866830.  [DOI](https://doi.org/10.1182/blood-2018-07-866830) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30530751/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=The%20HLA%20ligandome%20landscape%20of%20chronic%20myeloid%20leukemia%20delineates%20novel%20T-cell%20epitopes%20for%20immunotherapy&author=T.%20Bilich&author=A.%20Nelde&author=L.%20Bichmann&author=M.%20Roerden&author=H.R.%20Salih&volume=133&publication_year=2019&pages=550-565&pmid=30530751&doi=10.1182/blood-2018-07-866830&)

16. Ureshino H., Shindo T., Kojima H., Kusunoki Y., Miyazaki Y., Tanaka H., Saji H., Kawaguchi A., Kimura S. Allelic Polymorphisms of KIRs and HLAs Predict Favorable Responses to Tyrosine Kinase Inhibitors in CML. Cancer Immunol. Res. 2018;6:745–754. doi: 10.1158/2326-6066.CIR-17-0462.  [DOI](https://doi.org/10.1158/2326-6066.CIR-17-0462) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29695383/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Immunol.%20Res.&title=Allelic%20Polymorphisms%20of%20KIRs%20and%20HLAs%20Predict%20Favorable%20Responses%20to%20Tyrosine%20Kinase%20Inhibitors%20in%20CML&author=H.%20Ureshino&author=T.%20Shindo&author=H.%20Kojima&author=Y.%20Kusunoki&author=Y.%20Miyazaki&volume=6&publication_year=2018&pages=745-754&pmid=29695383&doi=10.1158/2326-6066.CIR-17-0462&)

17. Posthuma E.F., Falkenburg J.H., Apperley J.F., Gratwohl A., Roosnek E., Hertenstein B., Schipper R.F., Schreuder G.M., D’Amaro J., Oudshoorn M., et al. HLA-B8 and HLA-A3 coexpressed with HLA-B8 are associated with a reduced risk of the development of chronic myeloid leukemia. The Chronic Leukemia Working Party of the EBMT. Blood. 1999;93:3863–3865.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/10339494/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=HLA-B8%20and%20HLA-A3%20coexpressed%20with%20HLA-B8%20are%20associated%20with%20a%20reduced%20risk%20of%20the%20development%20of%20chronic%20myeloid%20leukemia.%20The%20Chronic%20Leukemia%20Working%20Party%20of%20the%20EBMT&author=E.F.%20Posthuma&author=J.H.%20Falkenburg&author=J.F.%20Apperley&author=A.%20Gratwohl&author=E.%20Roosnek&volume=93&publication_year=1999&pages=3863-3865&pmid=10339494&)

18. Posthuma E.F., Falkenburg J.H., Apperley J.F., Gratwohl A., Hertenstein B., Schipper R.F., Oudshoorn M., Biezen J.H., Hermans J., Willemze R., et al. HLA-DR4 is associated with a diminished risk of the development of chronic myeloid leukemia (CML). Chronic Leukemia Working Party of the European Blood and Marrow Transplant Registry. Leukemia. 2000;14:859–862. doi: 10.1038/sj.leu.2401774.  [DOI](https://doi.org/10.1038/sj.leu.2401774) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10803518/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Leukemia&title=HLA-DR4%20is%20associated%20with%20a%20diminished%20risk%20of%20the%20development%20of%20chronic%20myeloid%20leukemia%20(CML).%20Chronic%20Leukemia%20Working%20Party%20of%20the%20European%20Blood%20and%20Marrow%20Transplant%20Registry&author=E.F.%20Posthuma&author=J.H.%20Falkenburg&author=J.F.%20Apperley&author=A.%20Gratwohl&author=B.%20Hertenstein&volume=14&publication_year=2000&pages=859-862&pmid=10803518&doi=10.1038/sj.leu.2401774&)

19. Amirzargar A.-A., Khosravi F., Dianat S.S., Alimoghadam K., Ghavamzadeh F., Ansaripour B., Moradi B., Nikbin B. Association of HLA class II allele and haplotype frequencies with chronic myelogenous leukemia and age-at-onset of the disease. Pathol. Oncol. Res. 2007;13:47–51. doi: 10.1007/BF02893440.  [DOI](https://doi.org/10.1007/BF02893440) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17387388/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pathol.%20Oncol.%20Res.&title=Association%20of%20HLA%20class%20II%20allele%20and%20haplotype%20frequencies%20with%20chronic%20myelogenous%20leukemia%20and%20age-at-onset%20of%20the%20disease&author=A.-A.%20Amirzargar&author=F.%20Khosravi&author=S.S.%20Dianat&author=K.%20Alimoghadam&author=F.%20Ghavamzadeh&volume=13&publication_year=2007&pages=47-51&pmid=17387388&doi=10.1007/BF02893440&)

20. Khosravi F., Amirzargar A., Sarafnejad A., Nicknam M.H., Alimoghadam K., Dianat S., Solgi G., Nikbin B. HLA class II allele and haplotype frequencies in Iranian patients with leukemia. Iran. J. Allergy Asthma Immunol. 2007;6:137–142.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/17893434/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Iran.%20J.%20Allergy%20Asthma%20Immunol.&title=HLA%20class%20II%20allele%20and%20haplotype%20frequencies%20in%20Iranian%20patients%20with%20leukemia&author=F.%20Khosravi&author=A.%20Amirzargar&author=A.%20Sarafnejad&author=M.H.%20Nicknam&author=K.%20Alimoghadam&volume=6&publication_year=2007&pages=137-142&pmid=17893434&)

21. Oguz F.S., Kalayoglu S., Diler A.S., Tozkir H., Sargin D., Carin M., Dorak M.T. HLA system affects the age-at-onset in chronic myeloid leukemia. Am. J. Hematol. 2003;73:256–262. doi: 10.1002/ajh.10365.  [DOI](https://doi.org/10.1002/ajh.10365) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12879429/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Hematol.&title=HLA%20system%20affects%20the%20age-at-onset%20in%20chronic%20myeloid%20leukemia&author=F.S.%20Oguz&author=S.%20Kalayoglu&author=A.S.%20Diler&author=H.%20Tozkir&author=D.%20Sargin&volume=73&publication_year=2003&pages=256-262&pmid=12879429&doi=10.1002/ajh.10365&)

22. Pehlivan M., Sahin H.H., Pehlivan S., Ozdilli K., Kaynar L., Oguz F.S., Sever T., Yilmaz M., Eser B., Ogret Y.D., et al. Prognostic importance of single-nucleotide polymorphisms in IL-6, IL-10, TGF-β1, IFN-γ, and TNF-α genes in chronic phase chronic myeloid leukemia. Genet. Test. Mol. Biomark. 2014;18:403–409. doi: 10.1089/gtmb.2014.0011.  [DOI](https://doi.org/10.1089/gtmb.2014.0011) | [PMC free article](/articles/PMC4043362/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24819026/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genet.%20Test.%20Mol.%20Biomark.&title=Prognostic%20importance%20of%20single-nucleotide%20polymorphisms%20in%20IL-6,%20IL-10,%20TGF-%CE%B21,%20IFN-%CE%B3,%20and%20TNF-%CE%B1%20genes%20in%20chronic%20phase%20chronic%20myeloid%20leukemia&author=M.%20Pehlivan&author=H.H.%20Sahin&author=S.%20Pehlivan&author=K.%20Ozdilli&author=L.%20Kaynar&volume=18&publication_year=2014&pages=403-409&pmid=24819026&doi=10.1089/gtmb.2014.0011&)

23. Holweg C.T., Baan C.C., Balk A.H., Niesters H.G., Maat A.P., Mulder P.M., Weimar W. The transforming growth factor-beta1 codon 10 gene polymorphism and accelerated graft vascular disease after clinical heart transplantation. Transplantation. 2001;71:1463–1467. doi: 10.1097/00007890-200105270-00018.  [DOI](https://doi.org/10.1097/00007890-200105270-00018) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11391236/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Transplantation&title=The%20transforming%20growth%20factor-beta1%20codon%2010%20gene%20polymorphism%20and%20accelerated%20graft%20vascular%20disease%20after%20clinical%20heart%20transplantation&author=C.T.%20Holweg&author=C.C.%20Baan&author=A.H.%20Balk&author=H.G.%20Niesters&author=A.P.%20Maat&volume=71&publication_year=2001&pages=1463-1467&pmid=11391236&doi=10.1097/00007890-200105270-00018&)

24. Ishimura T., Fujisawa M., Isotani S., Higuchi A., Iijima K., Arakawa S., Hohenfellner K., Flanders K.C., Yoshikawa N., Kamidono S. Transforming growth factor-beta1 expression in early biopsy specimen predicts long-term graft function following pediatric renal transplantation. Clin. Transplant. 2001;15:185–191. doi: 10.1034/j.1399-0012.2001.150307.x.  [DOI](https://doi.org/10.1034/j.1399-0012.2001.150307.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11389709/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Transplant.&title=Transforming%20growth%20factor-beta1%20expression%20in%20early%20biopsy%20specimen%20predicts%20long-term%20graft%20function%20following%20pediatric%20renal%20transplantation&author=T.%20Ishimura&author=M.%20Fujisawa&author=S.%20Isotani&author=A.%20Higuchi&author=K.%20Iijima&volume=15&publication_year=2001&pages=185-191&pmid=11389709&doi=10.1034/j.1399-0012.2001.150307.x&)

25. Sarpatwari A., Bussel J.B., Ahmed M., Erqou S., Semple J.W., Newland A.C., Bennett D., Pharoah P., Provan D. Single nucleotide polymorphism (SNP) analysis demonstrates a significant association of tumour necrosis factor-alpha (TNFA) with primary immune thrombocytopenia among Caucasian adults. Hematology. 2011;16:243–248. doi: 10.1179/102453311X13025568941808.  [DOI](https://doi.org/10.1179/102453311X13025568941808) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21756542/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hematology&title=Single%20nucleotide%20polymorphism%20(SNP)%20analysis%20demonstrates%20a%20significant%20association%20of%20tumour%20necrosis%20factor-alpha%20(TNFA)%20with%20primary%20immune%20thrombocytopenia%20among%20Caucasian%20adults&author=A.%20Sarpatwari&author=J.B.%20Bussel&author=M.%20Ahmed&author=S.%20Erqou&author=J.W.%20Semple&volume=16&publication_year=2011&pages=243-248&pmid=21756542&doi=10.1179/102453311X13025568941808&)

26. Gao S., Tang K., Chen J., Wang J. The single nucleotide polymorphisms of interleukin-10 are associated with the risk of leukaemia: Evidence from 18 case-control studies. Medicine. 2020;99:e23006. doi: 10.1097/MD.0000000000023006.  [DOI](https://doi.org/10.1097/MD.0000000000023006) | [PMC free article](/articles/PMC7598852/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33126384/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Medicine&title=The%20single%20nucleotide%20polymorphisms%20of%20interleukin-10%20are%20associated%20with%20the%20risk%20of%20leukaemia:%20Evidence%20from%2018%20case-control%20studies&author=S.%20Gao&author=K.%20Tang&author=J.%20Chen&author=J.%20Wang&volume=99&publication_year=2020&pages=e23006&pmid=33126384&doi=10.1097/MD.0000000000023006&)

27. Zaninoni A., Fattizzo B., Giannotta J.A., Fermo E., Cecchi N., Bianchi P., Barcellini W. Single Nucleotide Polymorphisms of Cytokine Genes in Warm Autoimmune Hemolytic Anemias: Relationship with Clinical and Hematological Parameters. Blood. 2021;138:4142. doi: 10.1182/blood-2021-146006.  [DOI](https://doi.org/10.1182/blood-2021-146006) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Single%20Nucleotide%20Polymorphisms%20of%20Cytokine%20Genes%20in%20Warm%20Autoimmune%20Hemolytic%20Anemias:%20Relationship%20with%20Clinical%20and%20Hematological%20Parameters&author=A.%20Zaninoni&author=B.%20Fattizzo&author=J.A.%20Giannotta&author=E.%20Fermo&author=N.%20Cecchi&volume=138&publication_year=2021&pages=4142&doi=10.1182/blood-2021-146006&)

28. Al-Khatib S.M., Abdo N., Al-Eitan L.N., Al-Mistarehi A.-H., Zahran D.J., Kewan T.Z. The Impact of IL-6 and IL-10 Gene Polymorphisms in Diffuse Large B-Cell Lymphoma Risk and Overall Survival in an Arab Population: A Case-Control Study. Cancers. 2020;12:382. doi: 10.3390/cancers12020382.  [DOI](https://doi.org/10.3390/cancers12020382) | [PMC free article](/articles/PMC7072608/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32046104/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancers&title=The%20Impact%20of%20IL-6%20and%20IL-10%20Gene%20Polymorphisms%20in%20Diffuse%20Large%20B-Cell%20Lymphoma%20Risk%20and%20Overall%20Survival%20in%20an%20Arab%20Population:%20A%20Case-Control%20Study&author=S.M.%20Al-Khatib&author=N.%20Abdo&author=L.N.%20Al-Eitan&author=A.-H.%20Al-Mistarehi&author=D.J.%20Zahran&volume=12&publication_year=2020&pages=382&pmid=32046104&doi=10.3390/cancers12020382&)

29. Awad M.R., El-Gamel A., Hasleton P., Turner D.M., Sinnott P.J., Hutchinson I.V. Genotypic variation in the transforming growth factor-beta1 gene: Association with transforming growth factor-beta1 production, fibrotic lung disease, and graft fibrosis after lung transplantation. Transplantation. 1998;66:1014–1020. doi: 10.1097/00007890-199810270-00009.  [DOI](https://doi.org/10.1097/00007890-199810270-00009) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9808485/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Transplantation&title=Genotypic%20variation%20in%20the%20transforming%20growth%20factor-beta1%20gene:%20Association%20with%20transforming%20growth%20factor-beta1%20production,%20fibrotic%20lung%20disease,%20and%20graft%20fibrosis%20after%20lung%20transplantation&author=M.R.%20Awad&author=A.%20El-Gamel&author=P.%20Hasleton&author=D.M.%20Turner&author=P.J.%20Sinnott&volume=66&publication_year=1998&pages=1014-1020&pmid=9808485&doi=10.1097/00007890-199810270-00009&)

30. Turner D.M., Williams D.M., Sankaran D., Lazarus M., Sinnott P.J., Hutchinson I.V. An investigation of polymorphism in the interleukin-10 gene promoter. Eur. J. Immunogenet. 1997;24:1–8. doi: 10.1111/j.1365-2370.1997.tb00001.x.  [DOI](https://doi.org/10.1111/j.1365-2370.1997.tb00001.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9043871/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Immunogenet.&title=An%20investigation%20of%20polymorphism%20in%20the%20interleukin-10%20gene%20promoter&author=D.M.%20Turner&author=D.M.%20Williams&author=D.%20Sankaran&author=M.%20Lazarus&author=P.J.%20Sinnott&volume=24&publication_year=1997&pages=1-8&pmid=9043871&doi=10.1111/j.1365-2370.1997.tb00001.x&)

31. Dennis K.L., Blatner N.R., Gounari F., Khazaie K. Current status of interleukin-10 and regulatory T-cells in cancer. Curr. Opin. Oncol. 2013;25:637–645. doi: 10.1097/CCO.0000000000000006.  [DOI](https://doi.org/10.1097/CCO.0000000000000006) | [PMC free article](/articles/PMC4322764/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24076584/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr.%20Opin.%20Oncol.&title=Current%20status%20of%20interleukin-10%20and%20regulatory%20T-cells%20in%20cancer&author=K.L.%20Dennis&author=N.R.%20Blatner&author=F.%20Gounari&author=K.%20Khazaie&volume=25&publication_year=2013&pages=637-645&pmid=24076584&doi=10.1097/CCO.0000000000000006&)

32. Huang Y., Yang H., Borg B.B., Su X., Rhodes S.L., Yang K., Tong X., Tang G., Howell C.D., Rosen H.R., et al. A functional SNP of interferon-gamma gene is important for interferon-alpha-induced and spontaneous recovery from hepatitis C virus infection. Proc. Natl. Acad. Sci. USA. 2007;104:985–990. doi: 10.1073/pnas.0609954104.  [DOI](https://doi.org/10.1073/pnas.0609954104) | [PMC free article](/articles/PMC1783426/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17215375/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&title=A%20functional%20SNP%20of%20interferon-gamma%20gene%20is%20important%20for%20interferon-alpha-induced%20and%20spontaneous%20recovery%20from%20hepatitis%20C%20virus%20infection&author=Y.%20Huang&author=H.%20Yang&author=B.B.%20Borg&author=X.%20Su&author=S.L.%20Rhodes&volume=104&publication_year=2007&pages=985-990&pmid=17215375&doi=10.1073/pnas.0609954104&)

33. Wungu C.D.K., Ariyanto F.C., Prabowo G.I., Soetjipto, Handajani R. Association between five types of Tumor Necrosis Factor-α gene polymorphism and hepatocellular carcinoma risk: A meta-analysis. BMC Cancer. 2020;20:1134. doi: 10.1186/s12885-020-07606-6.  [DOI](https://doi.org/10.1186/s12885-020-07606-6) | [PMC free article](/articles/PMC7686711/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33228594/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Cancer&title=Association%20between%20five%20types%20of%20Tumor%20Necrosis%20Factor-%CE%B1%20gene%20polymorphism%20and%20hepatocellular%20carcinoma%20risk:%20A%20meta-analysis&author=C.D.K.%20Wungu&author=F.C.%20Ariyanto&author=G.I.%20Prabowo&author=%20Soetjipto&author=R.%20Handajani&volume=20&publication_year=2020&pages=1134&pmid=33228594&doi=10.1186/s12885-020-07606-6&)

34. Du G.-H., Wang J.-K., Richards J.R., Wang J.-J. Genetic polymorphisms in tumor necrosis factor alpha and interleukin-10 are associated with an increased risk of cervical cancer. Int. Immunopharmacol. 2019;66:154–161. doi: 10.1016/j.intimp.2018.11.015.  [DOI](https://doi.org/10.1016/j.intimp.2018.11.015) | [PMC free article](/articles/PMC6348885/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30453149/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int.%20Immunopharmacol.&title=Genetic%20polymorphisms%20in%20tumor%20necrosis%20factor%20alpha%20and%20interleukin-10%20are%20associated%20with%20an%20increased%20risk%20of%20cervical%20cancer&author=G.-H.%20Du&author=J.-K.%20Wang&author=J.R.%20Richards&author=J.-J.%20Wang&volume=66&publication_year=2019&pages=154-161&pmid=30453149&doi=10.1016/j.intimp.2018.11.015&)

35. Amirzargar A.A., Bagheri M., Ghavamzadeh A., Alimoghadam K., Khosravi F., Rezaei N., Moheydin M., Ansaripour B., Moradi B., Nikbin B. Cytokine gene polymorphism in Iranian patients with chronic myelogenous leukaemia. Int. J. Immunogenet. 2005;32:167–171. doi: 10.1111/j.1744-313X.2005.00502.x.  [DOI](https://doi.org/10.1111/j.1744-313X.2005.00502.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15932621/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Immunogenet.&title=Cytokine%20gene%20polymorphism%20in%20Iranian%20patients%20with%20chronic%20myelogenous%20leukaemia&author=A.A.%20Amirzargar&author=M.%20Bagheri&author=A.%20Ghavamzadeh&author=K.%20Alimoghadam&author=F.%20Khosravi&volume=32&publication_year=2005&pages=167-171&pmid=15932621&doi=10.1111/j.1744-313X.2005.00502.x&)

36. Abdulkadir J., Worku Y., Schreuder G.M., D’Amaro J., de Vries R.R., Ottenhoff T.H. HLA-DR and -DQ antigens in malnutrition-related diabetes mellitus in Ethiopians: A clue to its etiology? Tissue Antigens. 1989;34:284–289. doi: 10.1111/j.1399-0039.1989.tb01744.x.  [DOI](https://doi.org/10.1111/j.1399-0039.1989.tb01744.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2629160/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Tissue%20Antigens&title=HLA-DR%20and%20-DQ%20antigens%20in%20malnutrition-related%20diabetes%20mellitus%20in%20Ethiopians:%20A%20clue%20to%20its%20etiology?&author=J.%20Abdulkadir&author=Y.%20Worku&author=G.M.%20Schreuder&author=J.%20D%E2%80%99Amaro&author=R.R.%20de%20Vries&volume=34&publication_year=1989&pages=284-289&pmid=2629160&doi=10.1111/j.1399-0039.1989.tb01744.x&)

37. Ottenhoff T.H., Mengistu M., Tadesse G., de Vries R.R., Converse P.J. HLA-DR and DQ antigens in insulin-dependent diabetics in Ethiopia. Tissue Antigens. 1987;30:193–197. doi: 10.1111/j.1399-0039.1987.tb01621.x.  [DOI](https://doi.org/10.1111/j.1399-0039.1987.tb01621.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/3125628/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Tissue%20Antigens&title=HLA-DR%20and%20DQ%20antigens%20in%20insulin-dependent%20diabetics%20in%20Ethiopia&author=T.H.%20Ottenhoff&author=M.%20Mengistu&author=G.%20Tadesse&author=R.R.%20de%20Vries&author=P.J.%20Converse&volume=30&publication_year=1987&pages=193-197&pmid=3125628&doi=10.1111/j.1399-0039.1987.tb01621.x&)

38. Schnittger L., May J., Loeliger C.C., Gallin M.Y., Erttmann K.D., Bienzle U., Kremsner P.G., Meyer C.G. HLA DRB1-DQA1-DQB1 haplotype diversity in two African populations. Tissue Antigens. 1997;50:546–551. doi: 10.1111/j.1399-0039.1997.tb02911.x.  [DOI](https://doi.org/10.1111/j.1399-0039.1997.tb02911.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9389330/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Tissue%20Antigens&title=HLA%20DRB1-DQA1-DQB1%20haplotype%20diversity%20in%20two%20African%20populations&author=L.%20Schnittger&author=J.%20May&author=C.C.%20Loeliger&author=M.Y.%20Gallin&author=K.D.%20Erttmann&volume=50&publication_year=1997&pages=546-551&pmid=9389330&doi=10.1111/j.1399-0039.1997.tb02911.x&)

39. Ali A.A., Aalto M., Jonasson J., Osman A. Genome-wide analyses disclose the distinctive HLA architecture and the pharmacogenetic landscape of the Somali population. Sci. Rep. 2020;10:5652. doi: 10.1038/s41598-020-62645-0.  [DOI](https://doi.org/10.1038/s41598-020-62645-0) | [PMC free article](/articles/PMC7101338/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32221414/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&title=Genome-wide%20analyses%20disclose%20the%20distinctive%20HLA%20architecture%20and%20the%20pharmacogenetic%20landscape%20of%20the%20Somali%20population&author=A.A.%20Ali&author=M.%20Aalto&author=J.%20Jonasson&author=A.%20Osman&volume=10&publication_year=2020&pages=5652&pmid=32221414&doi=10.1038/s41598-020-62645-0&)

40. NCCN Recently Updated Guidelines. [(accessed on 30 May 2023)]. Available online: https://www.nccn.org/guidelines/recently-published-guidelines.  [https://www.nccn.org/guidelines/recently-published-guidelines](https://www.nccn.org/guidelines/recently-published-guidelines)

41. Sokal J.E., Baccarani M., Russo D., Tura S. Staging and prognosis in chronic myelogenous leukemia. Semin. Hematol. 1988;25:49–61.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/3279515/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Semin.%20Hematol.&title=Staging%20and%20prognosis%20in%20chronic%20myelogenous%20leukemia&author=J.E.%20Sokal&author=M.%20Baccarani&author=D.%20Russo&author=S.%20Tura&volume=25&publication_year=1988&pages=49-61&pmid=3279515&)

42. HLA Nomenclature. [(accessed on 29 May 2023)]. Available online: https://hla.alleles.org/nomenclature/committee.html.  [https://hla.alleles.org/nomenclature/committee.html](https://hla.alleles.org/nomenclature/committee.html)

43. The Allele Frequency Net Database [Search HLA Allele Frequencies] [(accessed on 30 May 2023)]. Available online: http://www.allelefrequencies.net/hla6006a.asp.  [http://www.allelefrequencies.net/hla6006a.asp](http://www.allelefrequencies.net/hla6006a.asp)

44. Schell A., Leisenbach R., Coman C., Parissiadis A., Tourne S. A new HLA-B*15 allele, B*15:220, found in three individuals sharing the HLA-A*66:01, HLA-C*12:03 and HLADRB1*07:01 alleles. Tissue Antigens. 2011;78:287–288. doi: 10.1111/j.1399-0039.2011.01717.x.  [DOI](https://doi.org/10.1111/j.1399-0039.2011.01717.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21732916/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Tissue%20Antigens&title=A%20new%20HLA-B*15%20allele,%20B*15:220,%20found%20in%20three%20individuals%20sharing%20the%20HLA-A*66:01,%20HLA-C*12:03%20and%20HLADRB1*07:01%20alleles&author=A.%20Schell&author=R.%20Leisenbach&author=C.%20Coman&author=A.%20Parissiadis&author=S.%20Tourne&volume=78&publication_year=2011&pages=287-288&pmid=21732916&doi=10.1111/j.1399-0039.2011.01717.x&)

45. Lehmann C. (University Hospital Leipzig, Leipzig, Germany). Personal communication. 2021.

46. Constantinescu I., Boșcaiu V., Cianga P., Dinu A.-A., Gai E., Melinte M., Moise A. The frequency of HLA alleles in the Romanian population. Immunogenetics. 2016;68:167–178. doi: 10.1007/s00251-015-0891-9.  [DOI](https://doi.org/10.1007/s00251-015-0891-9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26711124/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Immunogenetics&title=The%20frequency%20of%20HLA%20alleles%20in%20the%20Romanian%20population&author=I.%20Constantinescu&author=V.%20Bo%C8%99caiu&author=P.%20Cianga&author=A.-A.%20Dinu&author=E.%20Gai&volume=68&publication_year=2016&pages=167-178&pmid=26711124&doi=10.1007/s00251-015-0891-9&)

47. Lehmann C., Loeffler-Wirth H., Balz V., Enczmann J., Landgraf R., Lakowa N., Gruenewald T., Fischer J.C., Doxiadis I. Immunogenetic Predisposition to SARS-CoV-2 Infection. Biology. 2022;12:37. doi: 10.3390/biology12010037.  [DOI](https://doi.org/10.3390/biology12010037) | [PMC free article](/articles/PMC9855425/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36671730/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biology&title=Immunogenetic%20Predisposition%20to%20SARS-CoV-2%20Infection&author=C.%20Lehmann&author=H.%20Loeffler-Wirth&author=V.%20Balz&author=J.%20Enczmann&author=R.%20Landgraf&volume=12&publication_year=2022&pages=37&pmid=36671730&doi=10.3390/biology12010037&)

48. Sanchez-Mazas A., Nunes J.M., Middleton D., Sauter J., Buhler S., McCabe A., Hofmann J., Baier D.M., Schmidt A.H., Nicoloso G., et al. Common and well-documented HLA alleles over all of Europe and within European sub-regions: A catalogue from the European Federation for Immunogenetics. HLA. 2017;89:104–113. doi: 10.1111/tan.12956.  [DOI](https://doi.org/10.1111/tan.12956) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28102034/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=HLA&title=Common%20and%20well-documented%20HLA%20alleles%20over%20all%20of%20Europe%20and%20within%20European%20sub-regions:%20A%20catalogue%20from%20the%20European%20Federation%20for%20Immunogenetics&author=A.%20Sanchez-Mazas&author=J.M.%20Nunes&author=D.%20Middleton&author=J.%20Sauter&author=S.%20Buhler&volume=89&publication_year=2017&pages=104-113&pmid=28102034&doi=10.1111/tan.12956&)

49. van Janse Rensburg W.J., de Kock A., Bester C., Kloppers J.F. HLA major allele group frequencies in a diverse population of the Free State Province, South Africa. Heliyon. 2021;7:e06850. doi: 10.1016/j.heliyon.2021.e06850.  [DOI](https://doi.org/10.1016/j.heliyon.2021.e06850) | [PMC free article](/articles/PMC8085688/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33981900/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Heliyon&title=HLA%20major%20allele%20group%20frequencies%20in%20a%20diverse%20population%20of%20the%20Free%20State%20Province,%20South%20Africa&author=W.J.%20van%20Janse%20Rensburg&author=A.%20de%20Kock&author=C.%20Bester&author=J.F.%20Kloppers&volume=7&publication_year=2021&pages=e06850&pmid=33981900&doi=10.1016/j.heliyon.2021.e06850&)

50. Sanchez-Mazas A., Černý V., Di D., Buhler S., Podgorná E., Chevallier E., Brunet L., Weber S., Kervaire B., Testi M., et al. The HLA-B landscape of Africa: Signatures of pathogen-driven selection and molecular identification of candidate alleles to malaria protection. Mol. Ecol. 2017;26:6238–6252. doi: 10.1111/mec.14366.  [DOI](https://doi.org/10.1111/mec.14366) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28950417/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Ecol.&title=The%20HLA-B%20landscape%20of%20Africa:%20Signatures%20of%20pathogen-driven%20selection%20and%20molecular%20identification%20of%20candidate%20alleles%20to%20malaria%20protection&author=A.%20Sanchez-Mazas&author=V.%20%C4%8Cern%C3%BD&author=D.%20Di&author=S.%20Buhler&author=E.%20Podgorn%C3%A1&volume=26&publication_year=2017&pages=6238-6252&pmid=28950417&doi=10.1111/mec.14366&)

51. Peterson T.A., Bielawny T., Lacap P., Hardie R.-A., Daniuk C., Mendoza L., Thavaneswaran S., Kariri T., Kimani J., Wachihi C., et al. Diversity and Frequencies of HLA Class I and Class II Genes of an East African Population. Open J. Genet. 2014;4:99–124. doi: 10.4236/ojgen.2014.42013.  [DOI](https://doi.org/10.4236/ojgen.2014.42013) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Open%20J.%20Genet.&title=Diversity%20and%20Frequencies%20of%20HLA%20Class%20I%20and%20Class%20II%20Genes%20of%20an%20East%20African%20Population&author=T.A.%20Peterson&author=T.%20Bielawny&author=P.%20Lacap&author=R.-A.%20Hardie&author=C.%20Daniuk&volume=4&publication_year=2014&pages=99-124&doi=10.4236/ojgen.2014.42013&)

52. Baek I.-C., Choi E.-J., Shin D.-H., Kim H.-J., Choi H., Kim T.-G. Allele and haplotype frequencies of human leukocyte antigen-A, -B, -C, -DRB1, -DRB3/4/5, -DQA1, -DQB1, -DPA1, and -DPB1 by next generation sequencing-based typing in Koreans in South Korea. PLoS ONE. 2021;16:e0253619. doi: 10.1371/journal.pone.0253619.  [DOI](https://doi.org/10.1371/journal.pone.0253619) | [PMC free article](/articles/PMC8216545/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34153078/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&title=Allele%20and%20haplotype%20frequencies%20of%20human%20leukocyte%20antigen-A,%20-B,%20-C,%20-DRB1,%20-DRB3/4/5,%20-DQA1,%20-DQB1,%20-DPA1,%20and%20-DPB1%20by%20next%20generation%20sequencing-based%20typing%20in%20Koreans%20in%20South%20Korea&author=I.-C.%20Baek&author=E.-J.%20Choi&author=D.-H.%20Shin&author=H.-J.%20Kim&author=H.%20Choi&volume=16&publication_year=2021&pages=e0253619&pmid=34153078&doi=10.1371/journal.pone.0253619&)

53. Jawdat D., Uyar F.A., Alaskar A., Müller C.R., Hajeer A. HLA-A, -B, -C, -DRB1, -DQB1, and -DPB1 Allele and Haplotype Frequencies of 28,927 Saudi Stem Cell Donors Typed by Next-Generation Sequencing. Front. Immunol. 2020;11:544768. doi: 10.3389/fimmu.2020.544768.  [DOI](https://doi.org/10.3389/fimmu.2020.544768) | [PMC free article](/articles/PMC7643328/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33193311/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front.%20Immunol.&title=HLA-A,%20-B,%20-C,%20-DRB1,%20-DQB1,%20and%20-DPB1%20Allele%20and%20Haplotype%20Frequencies%20of%2028,927%20Saudi%20Stem%20Cell%20Donors%20Typed%20by%20Next-Generation%20Sequencing&author=D.%20Jawdat&author=F.A.%20Uyar&author=A.%20Alaskar&author=C.R.%20M%C3%BCller&author=A.%20Hajeer&volume=11&publication_year=2020&pages=544768&pmid=33193311&doi=10.3389/fimmu.2020.544768&)

54. Climent N., Plana M. Immunomodulatory Activity of Tyrosine Kinase Inhibitors to Elicit Cytotoxicity Against Cancer and Viral Infection. Front. Pharmacol. 2019;10:1232. doi: 10.3389/fphar.2019.01232.  [DOI](https://doi.org/10.3389/fphar.2019.01232) | [PMC free article](/articles/PMC6813222/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31680987/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front.%20Pharmacol.&title=Immunomodulatory%20Activity%20of%20Tyrosine%20Kinase%20Inhibitors%20to%20Elicit%20Cytotoxicity%20Against%20Cancer%20and%20Viral%20Infection&author=N.%20Climent&author=M.%20Plana&volume=10&publication_year=2019&pages=1232&pmid=31680987&doi=10.3389/fphar.2019.01232&)

55. Harada I., Sasaki H., Murakami K., Nishiyama A., Nakabayashi J., Ichino M., Miyazaki T., Kumagai K., Matsumoto K., Hagihara M., et al. Compromised anti-tumor-immune features of myeloid cell components in chronic myeloid leukemia patients. Sci. Rep. 2021;11:18046. doi: 10.1038/s41598-021-97371-8.  [DOI](https://doi.org/10.1038/s41598-021-97371-8) | [PMC free article](/articles/PMC8433374/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34508131/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&title=Compromised%20anti-tumor-immune%20features%20of%20myeloid%20cell%20components%20in%20chronic%20myeloid%20leukemia%20patients&author=I.%20Harada&author=H.%20Sasaki&author=K.%20Murakami&author=A.%20Nishiyama&author=J.%20Nakabayashi&volume=11&publication_year=2021&pages=18046&pmid=34508131&doi=10.1038/s41598-021-97371-8&)

56. Hunter C.A., Jones S.A. IL-6 as a keystone cytokine in health and disease. Nat. Immunol. 2015;16:448–457. doi: 10.1038/ni1117-1271b. Erratum in Nat. Immunol. 2017, 18, 1271.  [DOI](https://doi.org/10.1038/ni1117-1271b) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25898198/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Immunol.&title=IL-6%20as%20a%20keystone%20cytokine%20in%20health%20and%20disease&author=C.A.%20Hunter&author=S.A.%20Jones&volume=16&publication_year=2015&pages=448-457&pmid=25898198&doi=10.1038/ni1117-1271b&)

57. National Institute of Environmental Health Sciences SNPinfo Web Server. [(accessed on 30 December 2023)]; Available online: https://snpinfo.niehs.nih.gov/cgi-bin/snpinfo/snpfunc.cgi.  [https://snpinfo.niehs.nih.gov/cgi-bin/snpinfo/snpfunc.cgi](https://snpinfo.niehs.nih.gov/cgi-bin/snpinfo/snpfunc.cgi)

58. Hochhaus A., Baccarani M., Silver R.T., Schiffer C., Apperley J.F., Cervantes F., Clark R.E., Cortes J.E., Deininger M.W., Guilhot F., et al. European LeukemiaNet 2020 recommendations for treating chronic myeloid leukemia. Leukemia. 2020;34:966–984. doi: 10.1038/s41375-020-0776-2.  [DOI](https://doi.org/10.1038/s41375-020-0776-2) | [PMC free article](/articles/PMC7214240/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32127639/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Leukemia&title=European%20LeukemiaNet%202020%20recommendations%20for%20treating%20chronic%20myeloid%20leukemia&author=A.%20Hochhaus&author=M.%20Baccarani&author=R.T.%20Silver&author=C.%20Schiffer&author=J.F.%20Apperley&volume=34&publication_year=2020&pages=966-984&pmid=32127639&doi=10.1038/s41375-020-0776-2&)

59. Li N., Wang J.-X., Huo T.-T., Zhao J.-R., Wang T.-J. Associations of IL-1β and IL-6 gene polymorphisms with Parkinson’s disease. Eur. Rev. Med. Pharmacol. Sci. 2021;25:890–897. doi: 10.26355/eurrev_202101_24657.  [DOI](https://doi.org/10.26355/eurrev_202101_24657) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33577043/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur.%20Rev.%20Med.%20Pharmacol.%20Sci.&title=Associations%20of%20IL-1%CE%B2%20and%20IL-6%20gene%20polymorphisms%20with%20Parkinson%E2%80%99s%20disease&author=N.%20Li&author=J.-X.%20Wang&author=T.-T.%20Huo&author=J.-R.%20Zhao&author=T.-J.%20Wang&volume=25&publication_year=2021&pages=890-897&pmid=33577043&doi=10.26355/eurrev_202101_24657&)

60. Snoussi K., Strosberg A.D., Bouaouina N., Ben Ahmed S., Chouchane L. Genetic variation in pro-inflammatory cytokines (interleukin-1beta, interleukin-1alpha and interleukin-6) associated with the aggressive forms, survival, and relapse prediction of breast carcinoma. Eur. Cytokine Netw. 2005;16:253–260.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/16464738/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur.%20Cytokine%20Netw.&title=Genetic%20variation%20in%20pro-inflammatory%20cytokines%20(interleukin-1beta,%20interleukin-1alpha%20and%20interleukin-6)%20associated%20with%20the%20aggressive%20forms,%20survival,%20and%20relapse%20prediction%20of%20breast%20carcinoma&author=K.%20Snoussi&author=A.D.%20Strosberg&author=N.%20Bouaouina&author=S.%20Ben%20Ahmed&author=L.%20Chouchane&volume=16&publication_year=2005&pages=253-260&pmid=16464738&)

61. Park S.-J., Nakagawa T., Kitamura H., Atsumi T., Kamon H., Sawa S.-I., Kamimura D., Ueda N., Iwakura Y., Ishihara K., et al. IL-6 regulates in vivo dendritic cell differentiation through STAT3 activation. J. Immunol. 2004;173:3844–3854. doi: 10.4049/jimmunol.173.6.3844.  [DOI](https://doi.org/10.4049/jimmunol.173.6.3844) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15356132/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Immunol.&title=IL-6%20regulates%20in%20vivo%20dendritic%20cell%20differentiation%20through%20STAT3%20activation&author=S.-J.%20Park&author=T.%20Nakagawa&author=H.%20Kitamura&author=T.%20Atsumi&author=H.%20Kamon&volume=173&publication_year=2004&pages=3844-3854&pmid=15356132&doi=10.4049/jimmunol.173.6.3844&)

62. Anand M., Chodda S.K., Parikh P.M., Nadkarni J.S. Abnormal levels of proinflammatory cytokines in serum and monocyte cultures from patients with chronic myeloid leukemia in different stages, and their role in prognosis. Hematol. Oncol. 1998;16:143–154. doi: 10.1002/(SICI)1099-1069(199812)16:4<143::AID-HON628>3.0.CO;2-U.  [DOI](https://doi.org/10.1002/(SICI)1099-1069(199812)16:4&#x0003c;143::AID-HON628&#x0003e;3.0.CO;2-U) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10414234/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hematol.%20Oncol.&title=Abnormal%20levels%20of%20proinflammatory%20cytokines%20in%20serum%20and%20monocyte%20cultures%20from%20patients%20with%20chronic%20myeloid%20leukemia%20in%20different%20stages,%20and%20their%20role%20in%20prognosis&author=M.%20Anand&author=S.K.%20Chodda&author=P.M.%20Parikh&author=J.S.%20Nadkarni&volume=16&publication_year=1998&pages=143-154&pmid=10414234&doi=10.1002/(SICI)1099-1069(199812)16:4&#x0003c;143::AID-HON628&#x0003e;3.0.CO;2-U&)

63. Soma K., Nanami G., Kasamatsu T., Murakami Y., Ishihara R., Awata M., Yamane E., Sunaga M., Asao Y., Hashimoto N., et al. Th1 Cytokine Polymorphism Gene: TNF-alpha -857C/T Affects the Pathogenesis and Progression of Acute Myeloid Leukemia. Blood. 2019;134:1431. doi: 10.1182/blood-2019-129409.  [DOI](https://doi.org/10.1182/blood-2019-129409) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Th1%20Cytokine%20Polymorphism%20Gene:%20TNF-alpha%20-857C/T%20Affects%20the%20Pathogenesis%20and%20Progression%20of%20Acute%20Myeloid%20Leukemia&author=K.%20Soma&author=G.%20Nanami&author=T.%20Kasamatsu&author=Y.%20Murakami&author=R.%20Ishihara&volume=134&publication_year=2019&pages=1431&doi=10.1182/blood-2019-129409&)

64. Abdalhabib E.K., Algarni A., Saboor M., Alanazi F., Ibrahim I.K., Alfeel A.H., Alanazi A.M., Alanazi A.M., Alruwaili A.M., Alanazi M.H., et al. Association of TNF-α rs1800629 with Adult Acute B-Cell Lymphoblastic Leukemia. Genes. 2022;13:1237. doi: 10.3390/genes13071237.  [DOI](https://doi.org/10.3390/genes13071237) | [PMC free article](/articles/PMC9320751/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35886021/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genes&title=Association%20of%20TNF-%CE%B1%20rs1800629%20with%20Adult%20Acute%20B-Cell%20Lymphoblastic%20Leukemia&author=E.K.%20Abdalhabib&author=A.%20Algarni&author=M.%20Saboor&author=F.%20Alanazi&author=I.K.%20Ibrahim&volume=13&publication_year=2022&pages=1237&pmid=35886021&doi=10.3390/genes13071237&)

65. Chen L., Huang Z., Liao Y., Yang B., Zhang J. Association between tumor necrosis factor polymorphisms and rheumatoid arthritis as well as systemic lupus erythematosus: A meta-analysis. Braz. J. Med. Biol. Res. 2019;52:e7927. doi: 10.1590/1414-431X20187927.  [DOI](https://doi.org/10.1590/1414-431X20187927) | [PMC free article](/articles/PMC6437938/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30916218/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Braz.%20J.%20Med.%20Biol.%20Res.&title=Association%20between%20tumor%20necrosis%20factor%20polymorphisms%20and%20rheumatoid%20arthritis%20as%20well%20as%20systemic%20lupus%20erythematosus:%20A%20meta-analysis&author=L.%20Chen&author=Z.%20Huang&author=Y.%20Liao&author=B.%20Yang&author=J.%20Zhang&volume=52&publication_year=2019&pages=e7927&pmid=30916218&doi=10.1590/1414-431X20187927&)

66. Bănescu C., Tripon F., Trifa A.P., Crauciuc A.G., Moldovan V.G., Bogliş A., Benedek I., Dima D., Cândea M., Duicu C., et al. Cytokine rs361525, rs1800750, rs1800629, rs1800896, rs1800872, rs1800795, rs1800470, and rs2430561 SNPs in relation with prognostic factors in acute myeloid leukemia. Cancer Med. 2019;8:5492–5506. doi: 10.1002/cam4.2424.  [DOI](https://doi.org/10.1002/cam4.2424) | [PMC free article](/articles/PMC6745852/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31373163/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Med.&title=Cytokine%20rs361525,%20rs1800750,%20rs1800629,%20rs1800896,%20rs1800872,%20rs1800795,%20rs1800470,%20and%20rs2430561%20SNPs%20in%20relation%20with%20prognostic%20factors%20in%20acute%20myeloid%20leukemia&author=C.%20B%C4%83nescu&author=F.%20Tripon&author=A.P.%20Trifa&author=A.G.%20Crauciuc&author=V.G.%20Moldovan&volume=8&publication_year=2019&pages=5492-5506&pmid=31373163&doi=10.1002/cam4.2424&)

67. Stark G.R., Kerr I.M., Williams B.R., Silverman R.H., Schreiber R.D. How cells respond to interferons. Annu. Rev. Biochem. 1998;67:227–264. doi: 10.1146/annurev.biochem.67.1.227.  [DOI](https://doi.org/10.1146/annurev.biochem.67.1.227) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9759489/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annu.%20Rev.%20Biochem.&title=How%20cells%20respond%20to%20interferons&author=G.R.%20Stark&author=I.M.%20Kerr&author=B.R.%20Williams&author=R.H.%20Silverman&author=R.D.%20Schreiber&volume=67&publication_year=1998&pages=227-264&pmid=9759489&doi=10.1146/annurev.biochem.67.1.227&)

68. Rossouw M., Nel H.J., Cooke G.S., van Helden P.D., Hoal E.G. Association between tuberculosis and a polymorphic NFkappaB binding site in the interferon gamma gene. Lancet. 2003;361:1871–1872. doi: 10.1016/S0140-6736(03)13491-5.  [DOI](https://doi.org/10.1016/S0140-6736(03)13491-5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12788577/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Association%20between%20tuberculosis%20and%20a%20polymorphic%20NFkappaB%20binding%20site%20in%20the%20interferon%20gamma%20gene&author=M.%20Rossouw&author=H.J.%20Nel&author=G.S.%20Cooke&author=P.D.%20van%20Helden&author=E.G.%20Hoal&volume=361&publication_year=2003&pages=1871-1872&pmid=12788577&doi=10.1016/S0140-6736(03)13491-5&)

69. Wu Z., Sun Y., Zhu S., Tang S., Liu C., Qin W. Association of Interferon gamma +874T/A Polymorphism and Leukemia Risk: A Meta-Analysis. Medicine. 2016;95:e3129. doi: 10.1097/MD.0000000000003129.  [DOI](https://doi.org/10.1097/MD.0000000000003129) | [PMC free article](/articles/PMC4998384/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27015189/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Medicine&title=Association%20of%20Interferon%20gamma%20+874T/A%20Polymorphism%20and%20Leukemia%20Risk:%20A%20Meta-Analysis&author=Z.%20Wu&author=Y.%20Sun&author=S.%20Zhu&author=S.%20Tang&author=C.%20Liu&volume=95&publication_year=2016&pages=e3129&pmid=27015189&doi=10.1097/MD.0000000000003129&)

70. Tarafdar A., Hopcroft L.E.M., Gallipoli P., Pellicano F., Cassels J., Hair A., Korfi K., Jørgensen H.G., Vetrie D., Holyoake T.L., et al. CML cells actively evade host immune surveillance through cytokine-mediated downregulation of MHC-II expression. Blood. 2017;129:199–208. doi: 10.1182/blood-2016-09-742049.  [DOI](https://doi.org/10.1182/blood-2016-09-742049) | [PMC free article](/articles/PMC5305055/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27793879/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=CML%20cells%20actively%20evade%20host%20immune%20surveillance%20through%20cytokine-mediated%20downregulation%20of%20MHC-II%20expression&author=A.%20Tarafdar&author=L.E.M.%20Hopcroft&author=P.%20Gallipoli&author=F.%20Pellicano&author=J.%20Cassels&volume=129&publication_year=2017&pages=199-208&pmid=27793879&doi=10.1182/blood-2016-09-742049&)

71. Johnson-Ansah H.A., Briant A., Charbonnier A., Cambier N., Damaj G.L., Lenain P., Henry A., Parienti J.-J., Guenet D., GAC Besnard A.-C., et al. Interleukin 10 (IL10) and IL15 Significantly Decreased in Chronic Myeloid Leukemia Patients Presenting with Tyrosine Kinase Inhibitor (TKI) Withdrawal Syndrome (TWS). Results from the Prospective, Explorative and Multicenter «Kiwis» Study Designed for TWS. Clinicaltrial.Gov ID: NCT03996096; Study Number: 18-206. Blood. 2023;142:6335. doi: 10.1182/blood-2023-177651.  [NCT03996096](https://clinicaltrials.gov/ct2/show/NCT03996096) | [DOI](https://doi.org/10.1182/blood-2023-177651) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Interleukin%2010%20(IL10)%20and%20IL15%20Significantly%20Decreased%20in%20Chronic%20Myeloid%20Leukemia%20Patients%20Presenting%20with%20Tyrosine%20Kinase%20Inhibitor%20(TKI)%20Withdrawal%20Syndrome%20(TWS).%20Results%20from%20the%20Prospective,%20Explorative%20and%20Multicenter%20%C2%ABKiwis%C2%BB%20Study%20Designed%20for%20TWS.%20Clinicaltrial.Gov%20ID:NCT03996096;%20Study%20Number:%2018-206&author=H.A.%20Johnson-Ansah&author=A.%20Briant&author=A.%20Charbonnier&author=N.%20Cambier&author=G.L.%20Damaj&volume=142&publication_year=2023&pages=6335&doi=10.1182/blood-2023-177651&)

72. Naka K., Hoshii T., Muraguchi T., Tadokoro Y., Ooshio T., Kondo Y., Nakao S., Motoyama N., Hirao A. TGF-beta-FOXO signalling maintains leukaemia-initiating cells in chronic myeloid leukaemia. Nature. 2010;463:676–680. doi: 10.1038/nature08734.  [DOI](https://doi.org/10.1038/nature08734) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20130650/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=TGF-beta-FOXO%20signalling%20maintains%20leukaemia-initiating%20cells%20in%20chronic%20myeloid%20leukaemia&author=K.%20Naka&author=T.%20Hoshii&author=T.%20Muraguchi&author=Y.%20Tadokoro&author=T.%20Ooshio&volume=463&publication_year=2010&pages=676-680&pmid=20130650&doi=10.1038/nature08734&)

73. Giustacchini A., Thongjuea S., Barkas N., Woll P.S., Povinelli B.J., Booth C.A.G., Sopp P., Norfo R., Rodriguez-Meira A., Ashley N., et al. Single-cell transcriptomics uncovers distinct molecular signatures of stem cells in chronic myeloid leukemia. Nat. Med. 2017;23:692–702. doi: 10.1038/nm.4336.  [DOI](https://doi.org/10.1038/nm.4336) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28504724/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Med.&title=Single-cell%20transcriptomics%20uncovers%20distinct%20molecular%20signatures%20of%20stem%20cells%20in%20chronic%20myeloid%20leukemia&author=A.%20Giustacchini&author=S.%20Thongjuea&author=N.%20Barkas&author=P.S.%20Woll&author=B.J.%20Povinelli&volume=23&publication_year=2017&pages=692-702&pmid=28504724&doi=10.1038/nm.4336&)

74. Kumar A., Anand T., Bhattacharyya J., Sharma A., Jaganathan B.G. K562 chronic myeloid leukemia cells modify osteogenic differentiation and gene expression of bone marrow stromal cells. J. Cell Commun. Signal. 2018;12:441–450. doi: 10.1007/s12079-017-0412-8.  [DOI](https://doi.org/10.1007/s12079-017-0412-8) | [PMC free article](/articles/PMC5910323/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28963654/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Cell%20Commun.%20Signal.&title=K562%20chronic%20myeloid%20leukemia%20cells%20modify%20osteogenic%20differentiation%20and%20gene%20expression%20of%20bone%20marrow%20stromal%20cells&author=A.%20Kumar&author=T.%20Anand&author=J.%20Bhattacharyya&author=A.%20Sharma&author=B.G.%20Jaganathan&volume=12&publication_year=2018&pages=441-450&pmid=28963654&doi=10.1007/s12079-017-0412-8&)

75. Fort M., de Stefano G.F., Cambon-Thomsen A., Giraldo-Alvarez P., Dugoujon J.M., Ohayon E., Scano G., Abbal M. HLA class II allele and haplotype frequencies in Ethiopian Amhara and Oromo populations. Tissue Antigens. 1998;51:327–336. doi: 10.1111/j.1399-0039.1998.tb02971.x.  [DOI](https://doi.org/10.1111/j.1399-0039.1998.tb02971.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9583804/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Tissue%20Antigens&title=HLA%20class%20II%20allele%20and%20haplotype%20frequencies%20in%20Ethiopian%20Amhara%20and%20Oromo%20populations&author=M.%20Fort&author=G.F.%20de%20Stefano&author=A.%20Cambon-Thomsen&author=P.%20Giraldo-Alvarez&author=J.M.%20Dugoujon&volume=51&publication_year=1998&pages=327-336&pmid=9583804&doi=10.1111/j.1399-0039.1998.tb02971.x&)
