# Synergistic toxicity with copper contributes to NAT2-associated isoniazid toxicity

## Metadata
**Authors:** Jihoon G Yoon, Dong Geon Jang, Sung-Gyu Cho, Chaeyoung Lee, Shin Hye Noh, Soo Kyung Seo, Jung Woo Yu, Hyeon Woo Chung, KyeoRe Han, Soon Sung Kwon, Dai Hoon Han, Jaeseong Oh, In-Jin Jang, Sang-Hoon Kim, Young-Koo Jee, Hyun Lee, Dong Won Park, Jang Won Sohn, Ho Joo Yoon, Chul Hoon Kim, Jae Myun Lee, Sang-Heon Kim, Min Goo Lee
**Journal:** Experimental & Molecular Medicine
**Date:** 2024 Mar 1
**DOI:** [10.1038/s12276-024-01172-8](https://doi.org/10.1038/s12276-024-01172-8)
**PMID:** 38424191
**PMCID:** PMC10984958
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10984958/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC10984958/pdf/12276_2024_Article_1172.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC10984958/pdf/12276_2024_Article_1172.pdf)

## Abstract

Anti-tuberculosis (AT) medications, including isoniazid (INH), can cause drug-induced liver injury (DILI), but the underlying mechanism remains unclear. In this study, we aimed to identify genetic factors that may increase the susceptibility of individuals to AT-DILI and to examine genetic interactions that may lead to isoniazid (INH)-induced hepatotoxicity. We performed a targeted sequencing analysis of 380 pharmacogenes in a discovery cohort of 112 patients (35 AT-DILI patients and 77 controls) receiving AT treatment for active tuberculosis. Pharmacogenome-wide association analysis was also conducted using 1048 population controls (Korea1K). NAT2 and ATP7B genotypes were analyzed in a replication cohort of 165 patients (37 AT-DILI patients and 128 controls) to validate the effects of both risk genotypes. NAT2 ultraslow acetylators (UAs) were found to have a greater risk of AT-DILI than other genotypes (odds ratio [OR] 5.6 [95% confidence interval; 2.5–13.2], P = 7.2 × 10−6). The presence of ATP7B gene 832R/R homozygosity (rs1061472) was found to co-occur with NAT2 UA in AT-DILI patients (P = 0.017) and to amplify the risk in NAT2 UA (OR 32.5 [4.5–1423], P = 7.5 × 10−6). In vitro experiments using human liver-derived cell lines (HepG2 and SNU387 cells) revealed toxic synergism between INH and Cu, which were strongly augmented in cells with defective NAT2 and ATP7B activity, leading to increased mitochondrial reactive oxygen species generation, mitochondrial dysfunction, DNA damage, and apoptosis. These findings link the co-occurrence of ATP7B and NAT2 genotypes to the risk of INH-induced hepatotoxicity, providing novel mechanistic insight into individual AT-DILI susceptibility.

Yoon et al. showed that individuals who carry NAT2 UAs and ATP7B 832R/R genotypes are at increased risk of developing isoniazid hepatotoxicity, primarily due to the increased synergistic toxicity between isoniazid and copper, which exacerbates mitochondrial dysfunction-related apoptosis.

Subject terms: Genetics research, Mechanisms of disease, Predictive markers, Tuberculosis

## Introduction

Tuberculosis (TB) caused by the infectious bacterium *Mycobacterium tuberculosis* remains a significant global health concern. Indeed, TB continues to be a leading cause of death worldwide, with 1.6 million individuals dying due to TB in 2021^1^. An obstacle to the successful treatment of TB is the occurrence of adverse drug reactions to anti-tuberculosis (AT) medications. Among these reactions, idiosyncratic drug-induced liver injury (DILI) poses a significant challenge, particularly when triggered by the key AT drug isoniazid (INH)^2,3^. INH has been identified as the second most common agent causing DILI, further emphasizing its clinical importance^4^. Notably, ethnicity plays a crucial role, as Asians, particularly people in India, have a greater incidence of DILI^3,5–7^.

The increased risk of developing AT-DILI has been associated with certain genetic factors, such as functionally null variants of the INH-metabolizing enzyme NAT2 (known as ultraslow acetylators, NAT2 UAs)^8^. Meta-analyses and genome-wide association studies (GWASs) have validated this increased risk across diverse ethnic backgrounds^9–11^. However, the *NAT2* genotyping approach lacks specificity, as only a small percentage of individuals with *NAT2* UA genotypes develop AT-DILI. Consequently, the underlying mechanism of AT-DILI remains poorly understood.

Given that AT-DILI can lead to drug discontinuation, acute liver failure, and subsequent high mortality and morbidity, accurately predicting and mitigating risk by considering individual genetic factors is of utmost importance^12,13^. Therefore, the present study aimed to investigate genetic variants and molecular mechanisms that may predispose individuals to a greater susceptibility to AT-DILI.

## Materials and methods

### Patient cohort

Between 2009 and 2015, 35 apparent AT-DILI patients and 77 treatment-tolerant controls were enrolled from multiple medical centers located in the Republic of Korea, including Eulji, Dankook, Hallym, and Hanyang University hospitals. This discovery cohort included 112 patients who were treated with the isoniazid (INH) – rifampin (RIF) – ethambutol (EMB) – pyrazinamide (PZA) regimen for active TB. Between 2016 and 2017, additional patients (*n* = 165) were recruited from the above hospitals; 37 AT-DILI patients and 128 treatment-tolerant controls were included. This replication cohort included 101 patients with active TB and 64 health care workers with latent TB infection; these patients were also included in a previous report (Table [1](#Tab1))^14^. Informed consent was obtained from all participants, and the study was conducted in accordance with relevant ethical guidelines. The study was reviewed by the Institutional Review Board of Hanyang University Hospital (IRB number 2012-01-009/2017-02-018). We excluded patients with other underlying diseases to minimize the effects of clinical variables.

### Table 1.

| Characteristics | Discovery cohort (n = 112) |  |  | Replication cohort (n = 165) |  |  |
| --- | --- | --- | --- | --- | --- | --- |
| Case | Control | P value | Case | Control | P value |  |
| Number of patients | 35 | 77 |   | 37 | 128 |   |
| Sex, Female (%) | 19 (54.3%) | 34 (44.2%) | 0.415 | 21 (56.8%) | 77 (60.2%) | 0.709 |
| Age, median (age range, years) | 44 (17–84) | 45 (18–84) | 0.638 | 46 (16–80) | 43 (19–84) | 0.277 |
| Diagnosis (Medication) |  |  |  |  |  |  |
| Active TB (HREZ, %) | 35 (100%) | 77 (100%) | >0.999 | 29 (78.4%) | 72 (56.3%) | 0.021 |
| Latent TB (INH, %) | 0 | 0 | 8 (21.6%) | 56 (43.7%) |  |  |
| DILI classification |  |  |  |  |  |  |
| Hepatocellular type | 33 (94.3%) | NA |   | 29 (78.4%) | NA | 0.09 |
| Mixed type | 2 (5.7%) | NA |   | 5 (13.5%) | NA | 0.435 |
| Cholestatic type | 0 | NA |   | 3 (8.1%) | NA | 0.24 |
| LFT profiles, peak values |  |  |  |  |  |  |
| ALT, median (IQR; IU/L) | 349 (257–662) | 17 (12–26) | <0.001 | 366 (205–685) | 25 (19–35) | <0.001 |
| AST, median (IQR; IU/L) | 279 (196–550) | 25 (20–29) | <0.001 | 295 (157–566) | 25 (18–34) | <0.001 |
| Total bilirubin, median (IQR; mg/dL) | 1.01 (0.59–3.2) | 0.5 (0.40–0.80) | <0.001 | 0.90 (0.25–2.64) | 0.64 (0.45–0.80) | <0.001 |
| Investigation method | Targeted sequencing for 380 pharmacogenes |  |  | Genotyping for NAT2, ATP7B |  |  |

Table 1 Caption: Clinical characteristics of the patients.

### Patient enrollment and evaluation process

A baseline evaluation of TB infection, including medical history, chest X-ray or computed tomography (CT) results, sputum culture, and acid-fast bacillus staining, was performed. The inclusion criteria were as follows: adult patients newly diagnosed with active TB confirmed by a lesion evident via an imaging study (chest X-ray or CT) or positive results from sputum culture or acid-fast bacillus staining. Patients who met any of the following criteria before initiation of AT medication were excluded: (1) abnormal liver function test (LFT) results; (2) chronic liver disease, including fatty liver disease, liver cirrhosis, and alcoholic liver disease; (3) Hepatitis B or C virus carriage; (4) chronic alcoholism; (5) renal dysfunction or chronic kidney disease; (7) other chronic disorders requiring systemic therapy; and (8) poor medication compliance. Latent TB infection was diagnosed via an interferon-gamma releasing assay (QuantiFERON-TB Gold In-Tube tests; Cellestis Ltd., Carnegie, VIC, Australia), as described previously^14^.

Patients with active TB received a first-line standard treatment regimen of the isoniazid (H) + rifampin (R) + ethambutol (E) + pyrazinamide (Z) combination for the initial 2 months, followed by an HR (±E) combination for 4 months or longer; latent TB infection was treated with INH monotherapy for 9 months following WHO guidelines^15^. The administration period and cessation point of AT medication were determined based on the therapeutic response or clinical manifestations assessed by physicians. We followed international and Korean guidelines for TB treatment and the hepatotoxicity of anti-TB medications^16,17^. In brief, we discontinued treatment until the patients had recovered from DILI and then rechallenged them with the medications one by one while closely monitoring their symptoms and liver function tests. When the causative drug was identified during rechallenge, we removed it from the regimen and treated the patients with the remaining anti-TB drugs. LFTs, including assessments of aspartate transaminase (AST), alanine aminotransferase (ALT), total bilirubin, and alkaline phosphatase (ALP) levels, were performed at each visit during outpatient follow-up.

### DILI classification

DILI cases (Supplementary Table [1](#MOESM1)) were determined based on the following criteria established by DILI Expert Working Group^18^: (1) alanine aminotransferase (ALT) ≥ fivefold the upper limit of normal (ULN), (2) alkaline phosphatase (ALP) ≥ twofold the ULN, and (3) alanine transaminase (ALT) ≥ threefold the ULN and total bilirubin ≥ twofold the ULN. Cases were further assessed using Roussel Uclaf Causality Assessment Method (RUCAM) by two or more independent experts (JGY, YKG, and SHK) to confirm AT-DILI, and only patients with an RUCAM scale of probable or highly probable (score ≥ 6) were included. Treatment-tolerant controls were selected based on liver enzyme levels within reference ranges. To reduce the potential for selection bias for treatment-tolerant controls, we also included a population control group (Korea1K; [http://1000genomes.kr/](http://1000genomes.kr/))^19^ in case-control analyses.

### Targeted pharmacogenomic (PGx) sequencing

Genomic DNA samples obtained from 112 participants in the discovery cohort were subjected to sequencing for 380 pharmacogenes (Supplementary Table [2](#MOESM1)). Genomic DNA was extracted from peripheral blood samples using QIAamp DNA Mini Kit (Qiagen, Hilden, Germany) according to the manufacturer’s protocol. Target panels were produced to cover all the transcript isoforms (Ensemble transcript) and ± 50 bp fragments of nearby junctions using an in-solution hybrid capture method, as described previously (Celemics, Inc., Seoul, Korea)^20^. Total read counts of 10.7 million per sample with a length of 150 bp paired-end reads were generated in 0.87 Mb target regions using the HiSeq 2500 system (Illumina, Inc., San Diego, USA) and mapped to the GrCh37/hg19 reference genome using Burrows–Wheeler Aligner (BWA v0.7.17). The average depth of coverage for target regions was 317 ×, and the fraction of targets above 100 × was 93.6%. The bioinformatics pipeline was constructed following Genome Analysis Tool Kit (GATK) best practices for germline small-nucleotide variant (SNV) discovery using GATK v4.1.4, as described previously^21,22^. Quality control for genotype calls was performed following the GATK genotype refinement workflow. Variants with a genotype quality <20, depth of coverage <20, missing genotypes >10%, or violating the Hardy‒Weinberg equilibrium (*P* < 10^−6^) were filtered out. Functional annotation of the detected variants (Supplementary Table [3](#MOESM1)) was performed using ANNOVAR with the RefGene, EnsGene, Cytoband, avSNP150, gnomAD v2.1.1, dbNSFP v4.1a, and PharmGKB databases^23,24^.

### Assessment of known candidates: NAT2, CYP2E1, and GSTM1

Candidate markers previously known to be associated with AT-DILI were evaluated for the *NAT2*, *CYP2E1*, and *GSTM1* genes (Supplementary Table [4](#MOESM1))^25^. NAT2 acetylator status was determined as previously described: (1) rapid acetylator (RA; *4/*4), (2) intermediate acetylator (IA; *4/*5, *4/*6, *4/*7, *4/*12, and *4/*19), (3) slow acetylator (SA; *5/*6 and *5/*7), and (4) UA (*6/*6, *6/*7, and *7/*7)^11^. The *CYP2E1* genotype was previously classified as c1/c1 (wild-type; WT), c1/c2, or c2/c2; these genotypes indicate *1A/*1A, *1A/*5 and *5/*5, respectively, according to the recently recommended nomenclature, and the *1A/*1A genotype is regarded as the risk genotype based on the marker rs3813867^25^. Given that the *GSTM1* null variant is a copy-number variant (CNV), a read-depth-based CNV detection method was used to distinguish the null genotype, as previously described^20^.

### Pharmacogenome-wide association study (PGxWAS)

Case‒control association studies were conducted on 1760 common PGx variants (minor allele frequency (MAF) ≥ 5%) detected in the discovery cohort (*n* = 112). Sex chromosomes were excluded from the analysis owing to the complexity of the statistical analysis. An ancestry-matched Korean population control (*n* = 1048) was used to increase the statistical power. Whole-genome sequencing data originating from Korean Genome Project (Korea1K) were obtained from the Korean Genomics Center at the Ulsan National Institute of Science and Technology (UNIST)^19^. Variants located in the target regions were extracted for 1048 individuals who participated in the Korea1K dataset after liftover (hg38 to hg19) and quality control processes. Consequently, association analyses were conducted using two different controls (Supplementary Table [5](#MOESM1)): a treatment-tolerant control (*n* = 77) and a population control (*n* = 1048) using PLINK software v1.9 ([https://www.cog-genomics.org/plink/](https://www.cog-genomics.org/plink/))^26^. Fisher’s exact test was used to calculate *P* values under the Cochran-Armitage trend model. Manhattan plots and QQ plots (Supplementary Fig. [1](#MOESM1)) were generated using the R package ‘qqman’^27^.

### Rare variant association analysis

Gene-based association analysis was carried out using less frequent functional variants (MAF < 5%, obtained from the gnomAD exome v2.1.1 EAS population frequency). The functional variants included missense, nonsense, frameshift, and splicing variants (within 2 bp at exon junctions), as defined by Ensemble annotation. Gene-based collapsing was used to construct gene sets from these variants. The sequence kernel association test (SKAT), optimized SKAT (SKAT-O), and burden test were applied to the DILI patients and treatment controls using the R package ‘SKAT’^28,29^. Genes with more than two functional variants and *P* < 0.05 are listed in Supplementary Table [6](#MOESM1).

### NAT2 and ATP7B genotyping

Sequencing of the *NAT2* gene in the discovery cohort indicated that the following five *NAT2* haplotypes could be used to distinguish among Koreans: rs1041983 (c.282C>T, p.Y94Y; C___8684085_20), rs1801280 (c.341T>C, p.I114T; C___1204093_20), rs1799930 (c.590G>A, p.R197Q; C___1204091_10), rs1208 (c.803A>G, p.K268R; C____572769_20), and rs1799931 (c.857G>A, p.G286E; C____572770_20). One patient harbored the rare variant rs1805158 (c.190C>T, p.R64W), which did not affect the acetylator phenotype. Therefore, *NAT2* and *ATP7B* genotyping was performed for the above five single-nucleotide polymorphisms (SNPs) and the *ATP7B* lead SNP rs1061472 (c.2495A>C, p.K832R) using genomic DNA samples acquired from 165 participants in the replication cohort. A TaqMan assay (Thermo Fisher Scientific, Massachusetts, USA) was used for genotyping using the ABI PRISM 7900HT real-time PCR system (Applied Biosystems, Foster City, CA, USA).

### Tissue-specific gene expression data

Tissue-specific gene expression data were analyzed via the Genotype-Tissue Expression (GTEx) portal ([https://www.gtexportal.org/](https://www.gtexportal.org/)). The 19 variants in the four candidate genes (*ATP7B*, *NAT1*, *PROM2*, and *SLCO2B1*; Supplementary Table [5](#MOESM1)) were investigated in GTEx to assess whether there is a variant associated with gene expression. Tissue and isoform expression data for the *NAT1* gene and the variant rs7845127 were obtained. The sQTL violin plot for the variant (rs7845127) was generated from the 208 genotyped liver tissue samples using GTEX Analysis Release V8 data (dbGaP Accession phs000424.v8.p2; Supplementary Fig. [6](#MOESM1)).

### Cell lines

Details on the cell and molecular experimental procedures used are described in the [Supplementary Materials and Methods](#MOESM1). Human hepatocellular carcinoma (HepG2) cells (HB-8065) and human embryonic kidney (HEK) 293 T cells (CRL-3216) were obtained from the American Type Culture Collection (ATCC, USA). SNU387 cells (CRL-2237) were obtained from the Korean Cell Line Bank (KCLB, Republic of Korea).

### Generation of gene knockout and knockdown cells

The CRISPR‒Cas9 system was used to generate *ATP7B* knockout (KO) HepG2 cells. The lentiCRISPRv2 vector (Addgene #52961) was digested with BsmBI and ligated with gRNAs targeting exons 9 and 17 of the *ATP7B* gene. To generate a stable *NAT2* knockdown construct in SNU387 cells using lentiviral vectors, HEK293T cells were plated at a density of approximately 7.0 × 10^5^ cells in 5 mL of medium in a 6 cm tissue culture plate one day before transfection. NAT2 short hairpin RNA (shRNA) (TRCN0000034910) and the empty pLKO.1 vector were obtained from the human RNAi Consortium (TRC) library. The sequence information of the gRNA, shRNA, and primers used in this study is listed in Supplementary Table [10](#MOESM1).

### NAT2 and ATP7B overexpression

We examined the effects of *NAT2* and *ATP7B* gene supplementation by introducing pCMV3 vectors containing cDNA purchased from Sino Biological (#HG11247-CF and #HG17426-UT, respectively). Mutant plasmids encoding ATP7B-R832 and NAT2*7 (G286E) were generated by PCR-based site-directed mutagenesis. Flag-tagged NAT2 plasmids were transfected into HepG2 cells, and the effects of NAT2 overexpression were compared with those of empty vector transfection. We also exogenously expressed the ATP7B WT (K832), ATP7B mutant (R832), NAT2 WT (*4) and NAT2 mutant (*7) alleles in *ATP7B* KO HepG2 cells to compare their activities.

### Fluorescence-activated cell sorting (FACS)

FACS analyses were used to study the effects of various concentrations of INH (Sigma‒Aldrich I3377, St. Louis, USA), copper (II) chloride (CuCl_2_, Sigma‒Aldrich 222011, St. Louis, USA), and carbonyl cyanide 4-(trifluoromethoxy)-phenylhydrazone (FCCP, Sigma‒Aldrich C2920, St. Louis, USA) on WT or *ATP7B* KO HepG2 cells.

### Western blotting

HepG2 cells, carrying WT or mutant plasmids, were cultured and treated with INH and CuCl_2_ before lysis and protein extraction. The proteins were then separated, transferred to membranes, and subjected to western blotting using primary and HRP-conjugated secondary antibodies, after which the bands were visualized through an enhanced chemiluminescence (ECL) detection reagent. For oligomerization studies, the HMGB1 protein was treated with CuCl_2_ and H_2_O_2_ and analyzed under nonreducing conditions; the details of the antibodies used are provided in Supplementary Table [11](#MOESM1).

### DNA fragmentation assay

DNA fragmentation was detected using an Apoptotic DNA Ladder Kit (Roche, 11835246001, Germany) according to the manufacturer’s instructions.

### Viability measurements

Cell viability (%) was calculated using absorbance data following the equation (A_Treated_ - A_Blank_)/(A_Control_ - A_Blank_) × 100. The lethal dose (LD50) was calculated from the best-fit dose-response curve using the least-squares method via the following equation: Y = Bottom + (Top-Bottom)/(1 + 10^((LogLD50 - X)*Hillslope).

### Statistical analysis

Comparisons of two groups were carried out using the Mann‒Whitney U test. Odds ratios (ORs) and 95% confidence intervals (CIs) were calculated from contingency tables based on Fisher’s exact test. The Benjamini–Hochberg method was used to determine the false discovery rate (FDR) for multiple corrections. All statistical analyses of genetic data were conducted using R v3.6 software (*R* Foundation for Statistical Computing, Vienna, Austria). Cell and molecular data were analyzed via Student’s *t*-test, multiple *t*-tests with FDR correction, or analysis of variance followed by Tukey’s multiple comparison test, as appropriate, using the GraphPad Prism software package (v8.4.2). A significance level of *P* < 0.05 was used throughout the analyses.

Please see the [Supplementary Information](#MOESM1) for detailed information on the materials and processes used in this study.

## Results

### Pharmacogenes and patient characteristics

In the present study, we aimed to investigate the relationship between pharmacogenetic variants and the development of DILI (Fig. [1](#Fig1)). For this purpose, a sequencing study was conducted on 380 pharmacogenes (PGxs) associated with drug absorption, distribution, metabolism, and excretion (ADME), as well as pharmacodynamics, in 112 participants enrolled in the discovery cohort. PGx genes were selected based on comprehensive literature reviews and information from the PharmGKB database ([https://www.pharmgkb.org/](https://www.pharmgkb.org/)) (Supplementary Table [2](#MOESM1)). The discovery cohort consisted of 35 cases of DILI, including 33 hepatocellular and 2 mixed-type DILI. Additionally, 77 treatment-tolerant controls diagnosed with active TB and treated with the standard regimen were included. To validate our findings, we established a replication cohort comprising patients with both active TB (*n* = 101) and latent TB (*n* = 64) who were treated with the standard TB regimen and INH monotherapy, respectively, and conducted *NAT2* and *ATP7B* genotyping. Within the replication cohort, we identified 37 patients with DILI, as categorized into 29 hepatocellular, 5 mixed-type, and 3 cholestatic-type cases. In total, the combined cohort consisted of 72 DILI patients of 277 patients, with hepatocellular types comprising the majority (86.1%) (Supplementary Table [1](#MOESM1)). In both the discovery and replication cohorts (Table [1](#Tab1)), no significant differences were observed between the case and control groups in terms of sex or age. However, it was notable that the frequency of DILI was greater in patients with active TB (28.7%) than in those with latent TB (12.5%) in the replication cohort (*P* = 0.021). This difference can be partially attributed to variations in drug regimens between the active TB group (HREZ combination therapy) and the latent TB group (INH monotherapy).

### Fig. 1. Study cohorts and workflow.

![Fig. 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c5c0/10984958/0e7ddf089b8a/12276_2024_1172_Fig1_HTML.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=10984958_12276_2024_1172_Fig1_HTML.jpg)

A schematic depiction illustrating the study cohorts and workflow. The study comprised two primary cohorts, discovery and replication cohorts, both of which consisted of patients who received anti-TB medications. In the discovery cohort, sequencing was conducted for a comprehensive set of 380 pharmacogenes. Subsequently, the replication cohort was genotyped specifically for the NAT2 and ATP7B genes. Furthermore, we included a Korean population control group, which was sourced from the Korean Genome Project dataset, encompassing genetic data for 1048 healthy individuals who had undergone genome sequencing.

### Pharmacogenomic variants associated with AT-DILI

In the discovery cohort, we conducted sequencing analysis and identified a total of 4127 variants in the target regions of 380 pharmacogenes. Notably, 373 of these variants are novel and not reported in the dbSNP 150 database. These novel variants hold potential significance in influencing individual drug responses, as indicated in Supplementary Table [3](#MOESM1). For our pharmacogenome-wide association study (PGxWAS), we analyzed 35 patients with DILI and two sets of controls: 77 treatment-tolerant individuals and 1048 population controls (Fig. [2](#Fig2)). This analysis revealed several candidate genes, namely, *ATP7B*, *NAT1*, *PROM2*, and *SLCO2B1*, with the strongest association signal observed for the *ATP7B* gene (Fig. [2a](#Fig2), Supplementary Table [5](#MOESM1)). Our findings were supported by the quantile-quantile (QQ) plot, which demonstrated significant inflation signals compared to the population control (Supplementary Fig. [1](#MOESM1)). Specifically, we observed a potential association between the *ATP7B* K832R (rs1061472) variant, which is known to alter ATP7B properties according to previous studies^30,31^. Additionally, we assessed genes with known associations^12,13^ and found a significant association between NAT2 UA status and AT-DILI (OR 11.0 [2.6–66.7], *P* = 1.4 × 10^−4^). However, no associations were found with nonfunctional alleles of the *CYP2E1* or *GSTM1* gene (Supplementary Table [4](#MOESM1)). Moreover, we performed rare variant analyses using SKAT, SKAT-O, and Burden tests, but no significant associations with AT-DILI in other pharmacogenes were observed after correcting for multiple comparisons (Supplementary Table [6](#MOESM1)).

### Fig. 2. Discovery of pharmacogenetic variants associated with AT-DILI.

![Fig. 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c5c0/10984958/7b39e322b52f/12276_2024_1172_Fig2_HTML.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=10984958_12276_2024_1172_Fig2_HTML.jpg)

a Volcano plots showing the log2-transformed odds ratio (OR) on the x-axis and -log P value on the y-axis of the pharmacogenome-wide association study (PGxWAS) conducted in two different controls. The left panel shows the results from the comparison of 35 AT-DILI patients vs. 77 treatment-tolerant controls, and the right panel shows the results from the comparison of 35 AT-DILI patients vs. 1048 population controls. The ATP7B gene consistently showed the strongest signal in these comparisons. The orange and red dashed lines represent significance thresholds of P < 0.01 and P < 0.001, respectively. b Higher frequencies of NAT2 UAs were observed in AT-DILI patients than in treatment-tolerant controls in both the discovery (n = 112) and replication (n = 165) cohorts (OR 5.6 [2.5–13.2], P = 7.2 × 10−6). Furthermore, the combination of NAT2 UAs with ATP7B 832R/R occurred more frequently in AT-DILI patients (OR 32.5 [4.5–1423], P = 7.5 × 10−6; *P < 0.05; **P < 0.01; ***P < 0.001).

Based on these results, we identified *NAT2* and *ATP7B* genotypes as potential contributors to the risk of AT-DILI. To further investigate these genotypes, we conducted comprehensive analysis of both the discovery cohort (*n* = 112) and the replication cohort (*n* = 165). The frequency of NAT2 genotypes in our cohort aligned with previous reports on 1,000 Koreans and the Korea1K group (Supplementary Table [7](#MOESM1))^19,32^. Consistent with prior findings, NAT2 UAs were associated with an increased risk of AT-DILI in the combined cohort (OR 5.6 [2.5–13.2], *P* = 7.2 × 10^−6^; Table [2](#Tab2))^9–11^. Interestingly, we observed a far greater incidence of co-occurring NAT2 UAs and ATP7B 832 R/R homozygosity in AT-DILI patients (13.9%, 10/72) than in treatment-tolerant controls (0.5%, 1/205; *P* = 0.017; Fig. [2b](#Fig2)). While *ATP7B* 832R/R homozygosity alone exhibited a modest effect, combination with *NAT2* UAs (resulting in co-occurrence of NAT2-ATP7B risk genotypes) significantly increased risk (OR 32.5 [4.5–1,423], *P* = 7.5 × 10^−6^). In the discovery cohort, six of 35 AT-DILI patients (17.1%) had these genotypes, but none of the 77 treatment-tolerant controls (0.0%) had these genotypes (OR > 66.4, *P* = 2.2 × 10^−16^). Similarly, in the replication cohort, four of 37 AT-DILI patients (10.8%) and one of the 128 treatment-tolerant controls (0.8%) carried these genotypes (OR 15.1 [1.43–760], *P* = 0.009). Overall, *NAT2*-*ATP7B* risk genotypes were present in 13.9% of all AT-DILI patients (*n* = 72), which was significantly greater than the corresponding proportions in the treatment-tolerant controls (*n* = 205; 0.5%) and population controls (*n* = 1048; 1.8%, Supplementary Table [8](#MOESM1)).

### Table 2.

| Risk Genotypes |   | NAT2 UA |  | ATP7B 832 R/R |  | NAT2 UA + ATP7B 832 R/R |  |
| --- | --- | --- | --- | --- | --- | --- | --- |
| n | % | n | % | n | % |  |  |
| Discovery cohort (n = 112) | Case | 11 | 31.4 | 11 | 31.4 | 6 | 17.1 |
| Control | 3 | 3.9 | 10 | 13 | 0 | 0 |  |
| OR [95% CI] | 11.02 [2.62–66.67] |  | 3.04 [1.03–9.13] |  | >66.4 |  |  |
| P value | 1.4 × 10−4 |  | 0.035 |  | 6.8 × 10−4 |  |  |
| Replication cohort (n = 165) | Case | 9 | 24.3 | 9 | 24.3 | 4 | 10.8 |
| Control | 10 | 7.8 | 22 | 17.2 | 1 | 0.78 |  |
| OR [95% CI] | 3.75 [1.23–11.41] |  | 1.54 [0.56–3.98] |  | 15.06 [1.43–760] |  |  |
| P value | 0.015 |  | 0.34 |  | 9.3 × 10−3 |  |  |
| Combined cohort (n = 277) | Case | 20 | 27.8 | 20 | 27.8 | 10 | 13.9 |
| Control | 13 | 6.3 | 32 | 15.6 | 1 | 0.48 |  |
| OR [95% CI] | 5.63 [2.48–13.22] |  | 2.07 [1.03–4.10] |  | 32.45 [4.46–1424] |  |  |
| P value | 7.2 × 10−6 |  | 0.034 |  | 7.5 × 10−6 |  |  |
| Population control (n = 1,048) |   | 129 | 12.3 | 147 | 14 | 19 | 1.8 |
| OR [95% CI]a | 2.73 [1.50–4.84] |  | 2.35 [1.29–4.15] |  | 8.70 [3.46–20.67] |  |  |
| P valuea | 8.8 × 10−4 |  | 0.003 |  | 4.8 × 10−6 |  |  |

Table 2 Caption: High frequencies of co-occurring NAT2 and ATP7B risk genotypes in patients with anti-tuberculosis drug-induced liver injury.

### Synergistic toxicity caused by INH and Cu

To identify the underlying molecular mechanisms of *NAT2* and *ATP7B-*associated AT-DILI, we first evaluated the synergistic cytotoxicity between AT medications (INH and RIF) and Cu by measuring the viability of hepatocyte-derived HepG2 cells. Notably, isobolographic analyses of CC_50_ (cytotoxic concentration 50) revealed that Cu has a toxic synergism with INH but not with RIF (Fig. [3a, b](#Fig3)).

### Fig. 3. Synergistic toxicity between isoniazid (INH) and copper (Cu) is regulated by ATP7B and NAT2 activities.

![Fig. 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c5c0/10984958/7f149de3ed79/12276_2024_1172_Fig3_HTML.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=10984958_12276_2024_1172_Fig3_HTML.jpg)

Assessment of drug synergism between copper (Cu) and antituberculosis drugs. In HepG2 cells, CC50 (the 50% cytotoxic concentration) values were measured for various combinations of a isoniazid (INH) or b rifampin (RIF) with Cu for 48 h. Isobolograms of CC50 values for INH-Cu and RIF-Cu cotreatments are shown. The dashed lines indicate the additive isobole (no interaction). A toxic synergistic effect between INH and Cu was observed but not between RIF and Cu. The toxic synergistic effect was more pronounced in ATP7B knockout (KO) cells. c Cell viability assay. The INH-induced cell death rate in the presence of Cu (for 24 h) was significantly greater in ATP7B KO cells than in wild-type (WT) cells. d Expression of NAT2 mRNA in eight human liver-derived cell lines was examined via quantitative reverse transcription PCR (n = 3). Among the cell lines tested, SNU387 cells exhibited the highest expression of NAT2, whereas HepG2 cells exhibited the lowest. e, f Exogenous expression of NAT2 attenuates the toxicity caused by treatment with INH and Cu (for 24 h) in HepG2 cells. The effect of exogenous NAT2 expression on cell survival was more pronounced in ATP7B KO cells (c, n = 5) than in WT cells (b, n = 5). Bar graph data are shown as the mean ± SEM. *P < 0.05 according to multiple t tests with FDR correction.

Next, the effects of NAT2 and ATP7B on INH-Cu cytotoxicity were evaluated via gene deletion and supplementation experiments. We generated *ATP7B* KO HepG2 cells using the CRISPR‒Cas9 system and then exogenously expressed WT (K832) and mutant (R832) ATP7B in the KO cells using plasmids containing the codon-optimized ATP7B sequence that prevents guide RNA-mediated degradation (Supplementary Fig. [2a, b](#MOESM1)). Importantly, *ATP7B* KO strongly augmented the synergistic effect of INH-Cu on toxicity according to isobolographic analysis (Fig. [3a](#Fig3)); thus, INH-induced cell death in the presence of Cu was significantly greater in *ATP7B* KO cells than in WT cells (Fig. [3c](#Fig3)).

Since HepG2 cells barely express detectable levels of NAT2 (Fig. [3d](#Fig3)), we analyzed the effects of NAT2 on INH-Cu toxicity via gene supplementation. Exogenous NAT2 expression partially mitigated INH-Cu-induced cell death in WT HepG2 cells (Fig. [3e](#Fig3)). Notably, the effect of exogenous NAT2 expression was more prominent in *ATP7B* KO HepG2 cells (Fig. [3f](#Fig3)), indicating that simultaneous reductions in NAT2 and ATP7B activities facilitate INH-Cu toxicity.

Because SNU-387 cells endogenously express ATP7B and exhibited the highest expression levels of NAT2 among the examined hepatocyte-derived cells (Figs. [3](#Fig3)d and [4a](#Fig4)), we evaluated the effects of ATP7B and NAT2 silencing in these cells. NAT2-downregulated cells were generated by stably expressing shRNAs targeting NAT2. Gene knockdown of *ATP7B* was achieved in these cells by transient treatment with siRNAs against *ATP7B* (Fig. [4a](#Fig4) and Supplementary Fig. [2c, d](#MOESM1)). Single knockdown of either *NAT2* or *ATP7B* partially reduced cell survival in the presence of INH and Cu (Fig. [4b](#Fig4)). Notably, double knockdown of *NAT2* and *ATP7B* further increased susceptibility to INH-Cu toxicity (Fig. [4b](#Fig4)), which provides a basis for the increased risk of AT-DILI in individuals with high-risk *NAT2* and *ATP7B* genotypes.

### Fig. 4. Reduced activity of NAT2 and ATP7B augments isoniazid (INH)-copper (Cu) toxicity in SNU387 and HepG2 liver-derived cells.

![Fig. 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c5c0/10984958/96d7621f6c23/12276_2024_1172_Fig4_HTML.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=10984958_12276_2024_1172_Fig4_HTML.jpg)

a, b Toxicity of INH-Cu (for 48 h) was assayed in SNU387 cells with knockdown of NAT2 and ATP7B. Gene knockdown of NAT2 was achieved via stable expression of shRNAs against NAT2, and that of ATP7B was achieved through transient transfection with siRNAs against ATP7B. Western blotting images verifying the effects of NAT2 and ATP7B knockdown (a). Single knockdown of either NAT2 or ATP7B partially reduced cell survival in the presence of INH and Cu (for 48 h), and double knockdown of NAT2 and ATP7B further augmented INH-Cu toxicity (b, n = 4). Bar graph data are shown as the mean ± SEM. *P < 0.05, **P < 0.01 by multiple t tests with FDR correction. Effects of exogenously supplementing ATP7B and NAT2 with the wild-type (WT) or mutant protein in ATP7B KO cells were examined. An example of immunoblotting for native and exogenous ATP7B expression is shown in c, and a summary of the densitometric analysis is shown in Supplementary Fig. 2b. Coexpression of WT and mutant ATP7B and NAT2 proteins in ATP7B KO cells via plasmid transfection was confirmed through immunoblotting (d). A cell viability assay was performed on cells treated with INH and Cu for 48 h e. Exogenous expression of WT ATP7B (K832) or NAT2 (*4) led to alleviation of INH-Cu toxicity. Cell survival effects were greatly diminished in cells expressing the mutant ATP7B-R832 and NAT2-UA (*7B) proteins. A (WT), ATP7B wild-type; A (R832), ATP7B-R832; N (WT), NAT2 wild-type; N (UA), NAT2*7B. Bar graph data are shown as the mean ± SEM (n = 5–8). *P < 0.05 by ANOVA followed by Tukey’s multiple comparison test. Concentrations of INH and Cu are in μM.

Then, we examined the effects of exogenously supplementing ATP7B and NAT2 with the WT or mutant protein in *ATP7B* KO HepG2 cells (Fig. [4c–e](#Fig4)). The toxicity of INH-Cu (INH 500 μM, Cu 500 μM) was substantially rescued by exogenous expression of the WT-ATP7B (K832) and WT-NAT2 (*4) proteins (Fig. [4e](#Fig4), black bar). This cell survival effect of ATP7B was partially reduced in cells expressing mutant ATP7B-R832 (Fig. [4e](#Fig4)), which is consistent with previous reports that ATP7B-R832 is a hypomorphic mutant^30,31^. Notably, cell survival was greatly diminished in cells expressing the mutant ATP7B-R832 and NAT2-UA (*7) proteins, indicating that hepatocytes with mutant *ATP7B* and *NAT2* alleles are highly vulnerable to INH-Cu toxicity.

### INH and Cu evoke mitochondrial injury

We next examined the cellular mechanisms responsible for INH-Cu toxicity. Initially, cell death patterns were analyzed via FACS analyses in HepG2 cells stained with annexin V and 7-AAD. Control experiments in HepG2 cells treated with cell injury-inducing agents, such as FCCP and H_2_O_2_, showed that *ATP7B* KO did not induce differences in intrinsic cell survival (Supplementary Fig. [3](#MOESM1)). In addition, treatment with INH alone at concentrations up to 1000 µM did not significantly induce cell death or mitochondrial injury (Supplementary Fig. [4](#MOESM1)). Subsequently, we investigated the toxicity mechanisms associated with Cu alone or INH-Cu treatment (Fig. [5](#Fig5)). Treatments with Cu alone up to 500 µM elicited no or mild cell toxicity. Notably, treatment with INH and Cu for 24 h strongly increased the number of annexin V-positive *ATP7B* KO cells (Fig. [5a, b](#Fig5)), and a longer period of INH and Cu incubation (48 h) eventually produced annexin V and 7-AAD double-positive cells (Fig. [5c, d](#Fig5)). Taken together, these findings suggest that apoptotic mechanisms are primarily involved in INH-Cu cytotoxicity. Consequently, western blot analyses of apoptosis markers showed that cotreatment with INH and Cu for 24 h triggered a series of apoptotic signals, such as increases in levels of the cleaved forms of PARP, caspase-3, and caspase-9, in *ATP7B* KO cells (Fig. [5e](#Fig5)). A DNA fragmentation assay also indicated that apoptosis-induced DNA fragmentation was increased by INH and Cu treatment, particularly in *ATP7B* KO cells (Fig. [5f](#Fig5)).

### Fig. 5. Isoniazid (INH) and copper (Cu) induce apoptosis in HepG2 cells.

![Fig. 5](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c5c0/10984958/67c9e30e9098/12276_2024_1172_Fig5_HTML.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=10984958_12276_2024_1172_Fig5_HTML.jpg)

a-d Examples of FACS dot plots constructed for HepG2 cells treated with INH (1000 µM) or Cu (500 µM) for 24 h or 48 h are shown in a and c, respectively. Summarized results of multiple experiments are presented in b and d (n = 3). Treatment with INH-Cu for 24 h strongly increased the percentage of annexin V-positive ATP7B KO cells, and a longer period of INH and Cu incubation (48 h) produced annexin V and 7-AAD double-positive cells. e Western blot analysis of apoptosis markers in HepG2 cells treated with INH or Cu for 24 h. Summarized results of multiple experiments are presented in the lower panels (n = 3). f Results of an apoptotic DNA fragmentation assay. Increased DNA fragmentation was noted in ATP7B KO cells treated with Cu alone or INH-Cu. Identical results were obtained in four independent experiments. Bar graph data are shown as the mean ± SEM. *P < 0.05, **P < 0.01, ***P < 0.001 by multiple t tests with FDR correction. FCCP (25 µM) and cisplatin (40 µM) were used as positive controls.

Mitochondria play a central role in apoptotic cell death^33^, and dysfunction of these organelles is known to be associated with diverse types of liver injury, including INH-induced DILI^34^. While healthy mitochondria spontaneously produce reactive oxygen species (ROS) as a byproduct via the electron transport chain reaction, damage to mitochondria might cause overproduction of ROS, which, in turn, may trigger cellular signaling that can induce cell death in various ways. Notably, FACS analyses using probes responsive to mitochondrial ROS (MitoSOX) and mitochondrial membrane potentials (MitoTracker CMXRos) revealed that INH and Cu treatment strongly increased both mitochondrial ROS generation (Fig. [6a, b](#Fig6)) and the number of dysfunctional mitochondria (Fig. [6c, d](#Fig6)) in *ATP7B* KO cells. Taken together, these results imply that accumulation of Cu in *ATP7B* KO HepG2 cells evokes synergistic toxicity with INH via mitochondrial injury, which leads to apoptotic cell death.

### Fig. 6. Isoniazid (INH) and copper (Cu) induce mitochondrial dysfunction in HepG2 cells.

![Fig. 6](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c5c0/10984958/37c88a6678a6/12276_2024_1172_Fig6_HTML.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=10984958_12276_2024_1172_Fig6_HTML.jpg)

Measurements of mitochondrial ROS generation using FACS analyses with MitoSOX. Examples of FACS dot plots are shown (a), and results of multiple experiments are presented in (b, n = 3). The percentage of MitoSOX-positive cells was greatly increased in ATP7B KO cells treated with INH (1000 µM) or Cu (500 µM) for 24 h. Measurement of functional mitochondria (%) using the MitoTracker system. MitoTracker CMXRos stains mitochondria in live cells, and its relative accumulation compared to MitoTracker FM depends on membrane potential. Examples of FACS dot plots are shown (c), and results of multiple experiments are presented in (d, n = 3). The percentage of dysfunctional mitochondria was greatly increased in ATP7B KO cells treated with INH or Cu for 24 h. Bar graph data are shown as the mean ± SEM. *P < 0.05, **P < 0.01, ***P < 0.001 by multiple t tests with FDR correction. FCCP (25 μM) was used as a positive control.

Nevertheless, other types of cell death pathways, such as necrosis and pyroptosis, did not appear to play major roles in INH-Cu toxicity. For example, treatment of ATP7B KO cells with INH and Cu resulted in a slight increase in the level of cellular lipid peroxidation, which seems to be linked to excessive production of mitochondrial ROS. However, the increase in lipid peroxidation induced by INH and Cu was only 13% of that induced by H_2_O_2,_ causing necrotic cell death (Supplementary Fig. [5a, b](#MOESM1)). In addition, INH-Cu treatment neither evoked activation of caspase-1 or Gasdermin D (Supplementary Fig. [5c](#MOESM1)) nor oligomerization of HMGB1 (Supplementary Fig. [5d](#MOESM1)), implying that pyroptotic mechanisms are not directly involved in INH- and Cu-induced cell death in HepG2 cells.

## Discussion

The current understanding of the mechanisms underlying AT-DILI remains incomplete. Previous studies, including meta-analyses and GWASs, have demonstrated an association between *NAT2* UA status and an elevated risk of AT-DILI^9–11^. However, the specific pathways influencing the risk of AT-DILI have yet to be fully elucidated. In this study, we investigated the genetic variants and molecular mechanisms that may predispose individuals to greater AT-DILI susceptibility. We found that individuals with genetic variations in the Cu transporter *ATP7B* gene and in the INH-metabolizing enzyme *NAT2* gene are at a greater risk of developing AT-DILI. Specifically, these two genes exhibited strong gene-gene interactions in AT-DILI. The presence of an *ATP7B* variant significantly amplifies the risk of developing AT-DILI when combined with the *NAT2* UA genotype, as more than 90% of individuals with these combined genotypes are likely to develop AT-DILI. By conducting additional studies, we further investigated the underlying mechanisms involved. Our results suggest that Cu exerts synergistic toxicity with INH, potentially through mitochondrial dysfunction. These findings shed light on the mechanism of INH-induced hepatotoxicity and may help to identify patients at high risk of developing AT-DILI to prevent adverse drug reactions during TB management.

In addition to the *NAT2* and *ATP7B* variants, our study identified several PGx variants that might contribute to variations in individual drug responses. Notably, among these are common variants of the *NAT1*, *PROM2*, and *SLCO2B1* genes (Supplementary Table [5](#MOESM1)), as well as rare variants of the *GSTM2*, *ADH1B*, *HMOX2*, *SCN1A*, and *TNFRSF4* genes (Supplementary Table [6](#MOESM1)). Although these differences did not reach statistical significance in our analysis, conducting further investigations with a larger cohort may provide statistical power to validate their involvement in AT-DILI. Of particular interest is the *NAT1* variant (rs7845127), which we observed to be a splicing quantitative trait locus (sQTL) potentially associated with increased expression of a liver-specific isoform (ENST00000517441.5) of *NAT1* (Supplementary Fig. [6](#MOESM1)). Considering the established role of N-acetyltransferases in INH metabolism, upregulation of NAT1 expression might reduce the risk of AT-DILI, which aligns with the findings observed in our study (odds ratio 0.42–0.51; Supplementary Table [5](#MOESM1)).

In this study, we employed strict criteria to include only patients with apparent DILI who met the designated threshold of RUCAM score of 6 or higher. Patients with minor elevations in liver function tests were excluded from the discovery cohort. By implementing these stringent criteria, we aimed to ensure the accuracy and reliability of our findings. To minimize potential selection bias, we further conducted case‒control analysis utilizing a population control group (Korea1K, *n* = 1048). Although the number of DILI patients (*n* = 72) and carriers of the NAT2-ATP7B high-risk genotype (*n* = 10, 13.9%) in our study cohort might appear to be relatively small, their significance becomes apparent when considering the low prevalence (1.8%) of high-risk genotype carriers within the Korean population. These results indicate a meaningful association between the high-risk genotype and AT-DILI, and emphasize the need for larger-scale studies to confirm our findings.

We observed a significantly greater risk of AT-DILI in individuals with *NAT2* UAs and *ATP7B* 832R/R homozygotes (OR 32.5 [4.5–1423], *P* = 7.5 × 10^−6^), while *ATP7B* 832R/R homozygotes exhibited a small independent effect size (vs. treatment-tolerant control, OR 2.07 [1.03–4.10], *P* = 0.034; vs. population control, OR 2.35 [1.29–4.15], *P* = 0.003; Table [2](#Tab2)). Interestingly, the combination of NAT2 UA genotypes with ATP7B 832R/R homozygosity strongly improved the PPV (60.0% vs. 18.7%, *P* < 0.001) for AT-DILI prediction without significantly compromising the NPV (96.1% vs. 95.6%, *P* > 0.05) compared to NAT2 UA genotypes alone, as shown in Supplementary Table [9](#MOESM1). These findings suggest that the inclusion of *ATP7B* genotypes in pharmacogenetic screening for *NAT2* UAs, in combination with use of a genotype-guided regimen^35^, may further improve the prediction of patient susceptibility to AT-DILI. Furthermore, our analysis revealed a noteworthy observation regarding the frequency of high-risk genotype carriers (those with *NAT2* UAs and *ATP7B* 832R/R homozygosity) in South Asian populations (Supplementary Fig. [7](#MOESM1)). Analysis of the 1000 Genome database indicated a substantially greater prevalence of these genotypes (7.1%) in the South Asian region, which aligns with the reported high incidence of AT-DILI in these populations^3,5–7^.

Many patients in our cohorts had taken AT medications other than INH. One such medication is RIF, which has been reported to cause mitochondrial injury in hepatocytes^36^. Therefore, it is important to consider the possibility of a more complex mechanism underlying the clinical situation. However, our findings revealed that Cu exhibited strong synergistic toxicity with INH (Fig. [3a](#Fig3)) but that only additive effects were observed with RIF (Fig. [3b](#Fig3)). This observation provides further evidence that INH is primarily responsible for NAT2- and ATP7B-associated AT-DILI. One potential mechanism that may explain this phenomenon is formation of an INH-Cu complex^37,38^. Previous studies have reported that Cu complexes formed with INH and its derivatives possess a strong affinity for DNA binding and display cytotoxic properties, which may contribute to INH-induced hepatotoxicity^39–41^.

In hepatocytes, ATP7B plays a pivotal role in regulating Cu homeostasis^42^. Biallelic mutations in the *ATP7B* gene are known to cause Wilson’s disease, which is characterized by accumulation of intracellular Cu and manifestation of liver symptoms such as acute hepatitis, hepatomegaly, jaundice, and cirrhosis^43^. While the *ATP7B* K832R variant (rs1061472) is generally considered benign in Wilson’s disease, the ATP7B K832R (rs1061472) polymorphism has been shown to alter ATP7B properties in vitro and in vivo, affecting its trafficking and transporting activities as well as its serum Cu levels in humans^30,31,44^. Our study suggests that individuals with Wilson’s disease or ATP7B-defective alleles who also harbor *NAT2* UAs may be at increased risk of INH-induced hepatotoxicity (Fig. [4c](#Fig4)). It is possible that the occurrence of INH-induced hepatotoxicity in patients with Wilson’s disease has been underestimated, as diagnosis of DILI often relies on exclusion criteria^45^. In fact, a patient in India diagnosed with Wilson’s disease was reported to develop INH-induced hepatotoxicity^46^. Therefore, caution should be exercised when using INH in patients with Wilson’s disease, particularly individuals with the *NAT2* UA genotype.

In conclusion, our study aimed to identify genetic factors contributing to AT-DILI through targeted PGx sequencing and genotyping. The results revealed that *ATP7B* 832R/R homozygosity increases the risk of AT-DILI in *NAT2* UAs. Experimental analyses demonstrated that ATP7B influences an individual’s susceptibility to INH-induced hepatotoxicity by affecting the synergistic toxicity between Cu and INH, which is mediated through mitochondrial apoptosis. These findings provide novel mechanistic insights into the underlying mechanisms of INH-induced hepatotoxicity and have potential implications for personalized prevention and treatment of AT-DILI, ultimately improving patient care and drug safety in TB management.

## Supplementary information

## Acknowledgements

We thank Sungwon Jeon and Jong Bhak from the Korean Genomics Center at UNIST for providing technical assistance during the Korea1K data-sharing process. We would also like to thank Dong Su Jang from the MID at Yonsei University College of Medicine for providing the medical illustrations and Benjamin J. Latimer for his editorial assistance. We would like to acknowledge Dr. Jeon-Soo Shin from the Department of Microbiology and Immunology at Yonsei University College of Medicine for kindly providing the recombinant human HMGB1 and dimeric HMGB1 proteins. This research was supported by grants (HI15C1543) from the Korea Health Industry Development Institute (KHIDI) funded by the Ministry of Health & Welfare and grants (2022R1A2C3002917 and 2022R1A4A1031336) from the National Research Foundation (NRF) of Korea funded by the Ministry of Science and ICT, Republic of Korea.

## Author contributions

J.G.Y.: designed the study, performed the genetic analysis and molecular experiments, analyzed the data, and wrote the manuscript; D.G.J.: performed the cell and molecular experiments and wrote the manuscript; S.G.C.: performed the cell and molecular experiments and wrote the manuscript; S.S.K: performed the cell and molecular experiments; C.L: performed the cell and molecular experiments; S.H.N: performed the cell and molecular experiments; S.K.S: performed the cell and molecular experiments; J.W.Y.: performed the genetic analysis; H.W.C: performed the cell and molecular experiments; K.R.H: performed the cell and molecular experiments; S.S.K: contributed to the cell and molecular experiments; D.H.H: contributed to the cell and molecular experiments; J.O.: contributed to the study materials and patient enrollment; .-J.J.: designed the study; Sang-Hoon.K.: contributed to the study materials and patient enrolment; Y.-K.J.: contributed to the study materials and patient enrollment; H.L. contributed to the study materials and patient enrollment; D.W.P. contributed to the study materials and patient enrollment; J.W.S. contributed to the study materials and patient enrollment; H.J.Y. contributed to the study materials and patient enrollment; C.H.K. contributed to the cell and molecular experiments; J.M.L.: designed and supervised the cell and molecular experiments; Sang-Heon.K: designed and supervised the study, contributed to the study materials and patient enrollment; M.G.L.: designed and supervised the study, and wrote and reviewed the manuscript.

## Data availability

Targeted sequencing of the PGx panel data can be accessed through the Clinical & Omics Data Archive (CODA; [https://coda.nih.go.kr/](https://coda.nih.go.kr/)) under accession number R001218. Any additional information required to reanalyze the data reported in this work is available from the corresponding author upon reasonable request.

## Competing interests

The authors declare no competing interests.

## Footnotes

## Contributor Information

Jae Myun Lee, Email: jaemyun@yuhs.ac.

Sang-Heon Kim, Email: sangheonkim@hanyang.ac.kr.

Min Goo Lee, Email: mlee@yuhs.ac.

## Supplementary information

The online version contains supplementary material available at 10.1038/s12276-024-01172-8.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

Targeted sequencing of the PGx panel data can be accessed through the Clinical & Omics Data Archive (CODA; [https://coda.nih.go.kr/](https://coda.nih.go.kr/)) under accession number R001218. Any additional information required to reanalyze the data reported in this work is available from the corresponding author upon reasonable request.

### Supplementary Materials

### Data Availability Statement

Targeted sequencing of the PGx panel data can be accessed through the Clinical & Omics Data Archive (CODA; [https://coda.nih.go.kr/](https://coda.nih.go.kr/)) under accession number R001218. Any additional information required to reanalyze the data reported in this work is available from the corresponding author upon reasonable request.

## References

1. Furin J, Cox H, Pai M. Tuberculosis. Lancet. 2019;393:1642–1656. doi: 10.1016/S0140-6736(19)30308-3.  [DOI](https://doi.org/10.1016/S0140-6736(19)30308-3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30904262/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Tuberculosis&author=J%20Furin&author=H%20Cox&author=M%20Pai&volume=393&publication_year=2019&pages=1642-1656&pmid=30904262&doi=10.1016/S0140-6736(19)30308-3&)

2. Andrade RJ, et al. Drug-induced liver injury. Nat Rev Dis Prim. 2019;5:58. doi: 10.1038/s41572-019-0105-0.  [DOI](https://doi.org/10.1038/s41572-019-0105-0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31439850/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Dis%20Prim&title=Drug-induced%20liver%20injury&author=RJ%20Andrade&volume=5&publication_year=2019&pages=58&pmid=31439850&doi=10.1038/s41572-019-0105-0&)

3. Tweed CD, et al. Liver toxicity associated with tuberculosis chemotherapy in the REMoxTB study. BMC Med. 2018;16:46. doi: 10.1186/s12916-018-1033-7.  [DOI](https://doi.org/10.1186/s12916-018-1033-7) | [PMC free article](/articles/PMC5875008/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29592805/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Med&title=Liver%20toxicity%20associated%20with%20tuberculosis%20chemotherapy%20in%20the%20REMoxTB%20study&author=CD%20Tweed&volume=16&publication_year=2018&pages=46&pmid=29592805&doi=10.1186/s12916-018-1033-7&)

4. Chalasani N, et al. Features and outcomes of 899 patients with drug-induced liver injury: The DILIN Prospective Study. Gastroenterology. 2015;148:1340–52.e7. doi: 10.1053/j.gastro.2015.03.006.  [DOI](https://doi.org/10.1053/j.gastro.2015.03.006) | [PMC free article](/articles/PMC4446235/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25754159/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastroenterology&title=Features%20and%20outcomes%20of%20899%20patients%20with%20drug-induced%20liver%20injury:%20The%20DILIN%20Prospective%20Study&author=N%20Chalasani&volume=148&publication_year=2015&pages=1340-52.e7&pmid=25754159&doi=10.1053/j.gastro.2015.03.006&)

5. Parthasarathy R, et al. Hepatic toxicity in South Indian patients during treatment of tuberculosis with short-course regimens containing isoniazid, rifampicin and pyrazinamide. Tubercle. 1986;67:99–108. doi: 10.1016/0041-3879(86)90003-6.  [DOI](https://doi.org/10.1016/0041-3879(86)90003-6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/3775870/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Tubercle&title=Hepatic%20toxicity%20in%20South%20Indian%20patients%20during%20treatment%20of%20tuberculosis%20with%20short-course%20regimens%20containing%20isoniazid,%20rifampicin%20and%20pyrazinamide&author=R%20Parthasarathy&volume=67&publication_year=1986&pages=99-108&pmid=3775870&doi=10.1016/0041-3879(86)90003-6&)

6. Saha A, et al. Prevalence of hepatotoxicity from antituberculosis therapy: a five-year experience from South India. J Prim Care Community Health. 2016;7:171–174. doi: 10.1177/2150131916642431.  [DOI](https://doi.org/10.1177/2150131916642431) | [PMC free article](/articles/PMC5932686/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27056794/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Prim%20Care%20Community%20Health&title=Prevalence%20of%20hepatotoxicity%20from%20antituberculosis%20therapy:%20a%20five-year%20experience%20from%20South%20India&author=A%20Saha&volume=7&publication_year=2016&pages=171-174&pmid=27056794&doi=10.1177/2150131916642431&)

7. Devarbhavi H, et al. The Indian network of drug-induced liver injury: etiology, clinical features, outcome and prognostic markers in 1288 patients. J Clin Exp Hepatol. 2021;11:288–298. doi: 10.1016/j.jceh.2020.11.002.  [DOI](https://doi.org/10.1016/j.jceh.2020.11.002) | [PMC free article](/articles/PMC8103312/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33994711/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Exp%20Hepatol&title=The%20Indian%20network%20of%20drug-induced%20liver%20injury:%20etiology,%20clinical%20features,%20outcome%20and%20prognostic%20markers%20in%201288%20patients&author=H%20Devarbhavi&volume=11&publication_year=2021&pages=288-298&pmid=33994711&doi=10.1016/j.jceh.2020.11.002&)

8. Huang YS, et al. Polymorphism of the N-acetyltransferase 2 gene as a susceptibility risk factor for antituberculosis drug-induced hepatitis. Hepatology. 2002;35:883–889. doi: 10.1053/jhep.2002.32102.  [DOI](https://doi.org/10.1053/jhep.2002.32102) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11915035/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hepatology&title=Polymorphism%20of%20the%20N-acetyltransferase%202%20gene%20as%20a%20susceptibility%20risk%20factor%20for%20antituberculosis%20drug-induced%20hepatitis&author=YS%20Huang&volume=35&publication_year=2002&pages=883-889&pmid=11915035&doi=10.1053/jhep.2002.32102&)

9. Suvichapanich S, et al. NAT2 ultra-slow acetylator and risk of anti-tuberculosis drug-induced liver injury: a genotype-based meta-analysis. Pharmacogenet Genomics. 2018;28:167–176. doi: 10.1097/FPC.0000000000000339.  [DOI](https://doi.org/10.1097/FPC.0000000000000339) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29781872/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=NAT2%20ultra-slow%20acetylator%20and%20risk%20of%20anti-tuberculosis%20drug-induced%20liver%20injury:%20a%20genotype-based%20meta-analysis&author=S%20Suvichapanich&volume=28&publication_year=2018&pages=167-176&pmid=29781872&doi=10.1097/FPC.0000000000000339&)

10. Suvichapanich S, et al. Genomewide Association Study confirming the association of nat2 with susceptibility to antituberculosis drug-induced liver injury in Thai patients. Antimicrob Agents Chemother. 2019;63:e02692–18. doi: 10.1128/AAC.02692-18.  [DOI](https://doi.org/10.1128/AAC.02692-18) | [PMC free article](/articles/PMC6658740/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31109976/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Antimicrob%20Agents%20Chemother&title=Genomewide%20Association%20Study%20confirming%20the%20association%20of%20nat2%20with%20susceptibility%20to%20antituberculosis%20drug-induced%20liver%20injury%20in%20Thai%20patients&author=S%20Suvichapanich&volume=63&publication_year=2019&pages=e02692-18&pmid=31109976&doi=10.1128/AAC.02692-18&)

11. Nicoletti P, et al. Genetic risk factors in drug-induced liver injury due to isoniazid-containing antituberculosis drug regimens. Clin Pharm Ther. 2021;109:1125–1135. doi: 10.1002/cpt.2100.  [DOI](https://doi.org/10.1002/cpt.2100) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33135175/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharm%20Ther&title=Genetic%20risk%20factors%20in%20drug-induced%20liver%20injury%20due%20to%20isoniazid-containing%20antituberculosis%20drug%20regimens&author=P%20Nicoletti&volume=109&publication_year=2021&pages=1125-1135&pmid=33135175&doi=10.1002/cpt.2100&)

12. Kim SH, et al. Genetic polymorphisms of drug-metabolizing enzymes and anti-TB drug-induced hepatitis. Pharmacogenomics. 2009;10:1767–1779. doi: 10.2217/pgs.09.100.  [DOI](https://doi.org/10.2217/pgs.09.100) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19891553/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Genetic%20polymorphisms%20of%20drug-metabolizing%20enzymes%20and%20anti-TB%20drug-induced%20hepatitis&author=SH%20Kim&volume=10&publication_year=2009&pages=1767-1779&pmid=19891553&doi=10.2217/pgs.09.100&)

13. Chen M, Suzuki A, Borlak J, Andrade RJ, Lucena MI. Drug-induced liver injury: Interactions between drug properties and host factors. J Hepatol. 2015;63:503–514. doi: 10.1016/j.jhep.2015.04.016.  [DOI](https://doi.org/10.1016/j.jhep.2015.04.016) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25912521/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Hepatol&title=Drug-induced%20liver%20injury:%20Interactions%20between%20drug%20properties%20and%20host%20factors&author=M%20Chen&author=A%20Suzuki&author=J%20Borlak&author=RJ%20Andrade&author=MI%20Lucena&volume=63&publication_year=2015&pages=503-514&pmid=25912521&doi=10.1016/j.jhep.2015.04.016&)

14. Chung SJ, et al. Adherence to nine-month isoniazid for latent tuberculosis infection in healthcare workers: a prospective study in a tertiary hospital. Sci Rep. 2020;10:6462. doi: 10.1038/s41598-020-63156-8.  [DOI](https://doi.org/10.1038/s41598-020-63156-8) | [PMC free article](/articles/PMC7160120/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32296096/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sci%20Rep&title=Adherence%20to%20nine-month%20isoniazid%20for%20latent%20tuberculosis%20infection%20in%20healthcare%20workers:%20a%20prospective%20study%20in%20a%20tertiary%20hospital&author=SJ%20Chung&volume=10&publication_year=2020&pages=6462&pmid=32296096&doi=10.1038/s41598-020-63156-8&)

15. WHO. Treatment of tuberculosis: guidelines. 4th edn (World Health Organization, 2010).  [PubMed](https://pubmed.ncbi.nlm.nih.gov/23741786/)

16. Blumberg HM, et al. American Thoracic Society/Centers for Disease Control and Prevention/Infectious Diseases Society of America: treatment of tuberculosis. Am J Respir Crit Care Med. 2003;167:603–662. doi: 10.1164/rccm.167.4.603.  [DOI](https://doi.org/10.1164/rccm.167.4.603) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12588714/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Respir%20Crit%20Care%20Med&title=American%20Thoracic%20Society/Centers%20for%20Disease%20Control%20and%20Prevention/Infectious%20Diseases%20Society%20of%20America:%20treatment%20of%20tuberculosis&author=HM%20Blumberg&volume=167&publication_year=2003&pages=603-662&pmid=12588714&doi=10.1164/rccm.167.4.603&)

17. Saukkonen JJ, et al. An official ATS statement: hepatotoxicity of antituberculosis therapy. Am J Respir Crit Care Med. 2006;174:935–952. doi: 10.1164/rccm.200510-1666ST.  [DOI](https://doi.org/10.1164/rccm.200510-1666ST) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17021358/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Respir%20Crit%20Care%20Med&title=An%20official%20ATS%20statement:%20hepatotoxicity%20of%20antituberculosis%20therapy&author=JJ%20Saukkonen&volume=174&publication_year=2006&pages=935-952&pmid=17021358&doi=10.1164/rccm.200510-1666ST&)

18. Aithal GP, et al. Case definition and phenotype standardization in drug-induced liver injury. Clin Pharm Ther. 2011;89:806–815. doi: 10.1038/clpt.2011.58.  [DOI](https://doi.org/10.1038/clpt.2011.58) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21544079/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharm%20Ther.&title=Case%20definition%20and%20phenotype%20standardization%20in%20drug-induced%20liver%20injury&author=GP%20Aithal&volume=89&publication_year=2011&pages=806-815&pmid=21544079&doi=10.1038/clpt.2011.58&)

19. Jeon S, et al. Korean Genome Project: 1094 Korean personal genomes with clinical information. Sci Adv. 2020;6:eaaz7835. doi: 10.1126/sciadv.aaz7835.  [DOI](https://doi.org/10.1126/sciadv.aaz7835) | [PMC free article](/articles/PMC7385432/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32766443/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sci%20Adv&title=Korean%20Genome%20Project:%201094%20Korean%20personal%20genomes%20with%20clinical%20information&author=S%20Jeon&volume=6&publication_year=2020&pages=eaaz7835&pmid=32766443&doi=10.1126/sciadv.aaz7835&)

20. Han SM, et al. Targeted next-generation sequencing for comprehensive genetic profiling of pharmacogenes. Clin Pharm Ther. 2017;101:396–405. doi: 10.1002/cpt.532.  [DOI](https://doi.org/10.1002/cpt.532) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27727443/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharm%20Ther&title=Targeted%20next-generation%20sequencing%20for%20comprehensive%20genetic%20profiling%20of%20pharmacogenes&author=SM%20Han&volume=101&publication_year=2017&pages=396-405&pmid=27727443&doi=10.1002/cpt.532&)

21. Yoon JG, et al. Unraveling the genomic architecture of the CYP3A Locus and ADME genes for personalized tacrolimus dosing. Transplantation. 2021;105:2213–2225. doi: 10.1097/TP.0000000000003660.  [DOI](https://doi.org/10.1097/TP.0000000000003660) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33654003/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Transplantation&title=Unraveling%20the%20genomic%20architecture%20of%20the%20CYP3A%20Locus%20and%20ADME%20genes%20for%20personalized%20tacrolimus%20dosing&author=JG%20Yoon&volume=105&publication_year=2021&pages=2213-2225&pmid=33654003&doi=10.1097/TP.0000000000003660&)

22. Yoon JG, et al. Molecular diagnosis of craniosynostosis using targeted next-generation sequencing. Neurosurgery. 2020;87:294–302. doi: 10.1093/neuros/nyz470.  [DOI](https://doi.org/10.1093/neuros/nyz470) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31754721/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neurosurgery&title=Molecular%20diagnosis%20of%20craniosynostosis%20using%20targeted%20next-generation%20sequencing&author=JG%20Yoon&volume=87&publication_year=2020&pages=294-302&pmid=31754721&doi=10.1093/neuros/nyz470&)

23. Wang K, Li M, Hakonarson H. ANNOVAR: functional annotation of genetic variants from high-throughput sequencing data. Nucleic Acids Res. 2010;38:e164. doi: 10.1093/nar/gkq603.  [DOI](https://doi.org/10.1093/nar/gkq603) | [PMC free article](/articles/PMC2938201/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20601685/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=ANNOVAR:%20functional%20annotation%20of%20genetic%20variants%20from%20high-throughput%20sequencing%20data&author=K%20Wang&author=M%20Li&author=H%20Hakonarson&volume=38&publication_year=2010&pages=e164&pmid=20601685&doi=10.1093/nar/gkq603&)

24. Barbarino JM, Whirl-Carrillo M, Altman RB, Klein TE. PharmGKB: A worldwide resource for pharmacogenomic information. Wiley Interdiscip Rev Syst Biol Med. 2018;10:e1417. doi: 10.1002/wsbm.1417.  [DOI](https://doi.org/10.1002/wsbm.1417) | [PMC free article](/articles/PMC6002921/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29474005/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Wiley%20Interdiscip%20Rev%20Syst%20Biol%20Med&title=PharmGKB:%20A%20worldwide%20resource%20for%20pharmacogenomic%20information&author=JM%20Barbarino&author=M%20Whirl-Carrillo&author=RB%20Altman&author=TE%20Klein&volume=10&publication_year=2018&pages=e1417&pmid=29474005&doi=10.1002/wsbm.1417&)

25. Huang YS. Genetic polymorphisms of drug-metabolizing enzymes and the susceptibility to antituberculosis drug-induced liver injury. Expert Opin Drug Metab Toxicol. 2007;3:1–8. doi: 10.1517/17425255.3.1.1.  [DOI](https://doi.org/10.1517/17425255.3.1.1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17269890/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Expert%20Opin%20Drug%20Metab%20Toxicol&title=Genetic%20polymorphisms%20of%20drug-metabolizing%20enzymes%20and%20the%20susceptibility%20to%20antituberculosis%20drug-induced%20liver%20injury&author=YS%20Huang&volume=3&publication_year=2007&pages=1-8&pmid=17269890&doi=10.1517/17425255.3.1.1&)

26. Purcell S, et al. PLINK: a tool set for whole-genome association and population-based linkage analyses. Am J Hum Genet. 2007;81:559–575. doi: 10.1086/519795.  [DOI](https://doi.org/10.1086/519795) | [PMC free article](/articles/PMC1950838/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17701901/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=PLINK:%20a%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses&author=S%20Purcell&volume=81&publication_year=2007&pages=559-575&pmid=17701901&doi=10.1086/519795&)

27. Turner SD. qqman: an R package for visualizing GWAS results using Q-Q and manhattan plots. J Open Source Softw. 2018;3:731. doi: 10.21105/joss.00731.  [DOI](https://doi.org/10.21105/joss.00731) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Open%20Source%20Softw&title=qqman:%20an%20R%20package%20for%20visualizing%20GWAS%20results%20using%20Q-Q%20and%20manhattan%20plots&author=SD%20Turner&volume=3&publication_year=2018&pages=731&doi=10.21105/joss.00731&)

28. Wu MC, et al. Rare-variant association testing for sequencing data with the sequence kernel association test. Am J Hum Genet. 2011;89:82–93. doi: 10.1016/j.ajhg.2011.05.029.  [DOI](https://doi.org/10.1016/j.ajhg.2011.05.029) | [PMC free article](/articles/PMC3135811/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21737059/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Rare-variant%20association%20testing%20for%20sequencing%20data%20with%20the%20sequence%20kernel%20association%20test&author=MC%20Wu&volume=89&publication_year=2011&pages=82-93&pmid=21737059&doi=10.1016/j.ajhg.2011.05.029&)

29. Lee S, et al. Optimal unified approach for rare-variant association testing with application to small-sample case-control whole-exome sequencing studies. Am J Hum Genet. 2012;91:224–237. doi: 10.1016/j.ajhg.2012.06.007.  [DOI](https://doi.org/10.1016/j.ajhg.2012.06.007) | [PMC free article](/articles/PMC3415556/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22863193/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Optimal%20unified%20approach%20for%20rare-variant%20association%20testing%20with%20application%20to%20small-sample%20case-control%20whole-exome%20sequencing%20studies&author=S%20Lee&volume=91&publication_year=2012&pages=224-237&pmid=22863193&doi=10.1016/j.ajhg.2012.06.007&)

30. McCann CJ, et al. Single nucleotide polymorphisms in the human ATP7B gene modify the properties of the ATP7B protein. Metallomics. 2019;11:1128–1139. doi: 10.1039/c9mt00057g.  [DOI](https://doi.org/10.1039/c9mt00057g) | [PMC free article](/articles/PMC6878656/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31070637/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Metallomics&title=Single%20nucleotide%20polymorphisms%20in%20the%20human%20ATP7B%20gene%20modify%20the%20properties%20of%20the%20ATP7B%20protein&author=CJ%20McCann&volume=11&publication_year=2019&pages=1128-1139&pmid=31070637&doi=10.1039/c9mt00057g&)

31. Huster D, et al. Diverse functional properties of Wilson disease ATP7B variants. Gastroenterology. 2012;142:947–956.e5. doi: 10.1053/j.gastro.2011.12.048.  [DOI](https://doi.org/10.1053/j.gastro.2011.12.048) | [PMC free article](/articles/PMC3461965/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22240481/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastroenterology&title=Diverse%20functional%20properties%20of%20Wilson%20disease%20ATP7B%20variants&author=D%20Huster&volume=142&publication_year=2012&pages=947-956.e5&pmid=22240481&doi=10.1053/j.gastro.2011.12.048&)

32. Lee SY, et al. Complete sequencing of a genetic polymorphism in NAT2 in the Korean population. Clin Chem. 2002;48:775–777. doi: 10.1093/clinchem/48.5.775.  [DOI](https://doi.org/10.1093/clinchem/48.5.775) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11978608/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Chem&title=Complete%20sequencing%20of%20a%20genetic%20polymorphism%20in%20NAT2%20in%20the%20Korean%20population&author=SY%20Lee&volume=48&publication_year=2002&pages=775-777&pmid=11978608&doi=10.1093/clinchem/48.5.775&)

33. Bock FJ, Tait SWG. Mitochondria as multifaceted regulators of cell death. Nat Rev Mol Cell Biol. 2020;21:85–100. doi: 10.1038/s41580-019-0173-8.  [DOI](https://doi.org/10.1038/s41580-019-0173-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31636403/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Mol%20Cell%20Biol&title=Mitochondria%20as%20multifaceted%20regulators%20of%20cell%20death&author=FJ%20Bock&author=SWG%20Tait&volume=21&publication_year=2020&pages=85-100&pmid=31636403&doi=10.1038/s41580-019-0173-8&)

34. Boelsterli UA, Lee KK. Mechanisms of isoniazid-induced idiosyncratic liver injury: emerging role of mitochondrial stress. J. Gastroenterol Hepatol. 2014;29:678–687. doi: 10.1111/jgh.12516.  [DOI](https://doi.org/10.1111/jgh.12516) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24783247/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Gastroenterol%20Hepatol&title=Mechanisms%20of%20isoniazid-induced%20idiosyncratic%20liver%20injury:%20emerging%20role%20of%20mitochondrial%20stress&author=UA%20Boelsterli&author=KK%20Lee&volume=29&publication_year=2014&pages=678-687&pmid=24783247&doi=10.1111/jgh.12516&)

35. Yoo H, et al. A pilot study to investigate the utility of NAT2 genotype-guided isoniazid monotherapy regimens in NAT2 slow acetylators. Pharmacogenetics Genomics. 2021;31:68–73. doi: 10.1097/FPC.0000000000000423.  [DOI](https://doi.org/10.1097/FPC.0000000000000423) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33165168/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics%20Genomics&title=A%20pilot%20study%20to%20investigate%20the%20utility%20of%20NAT2%20genotype-guided%20isoniazid%20monotherapy%20regimens%20in%20NAT2%20slow%20acetylators&author=H%20Yoo&volume=31&publication_year=2021&pages=68-73&pmid=33165168&doi=10.1097/FPC.0000000000000423&)

36. Chowdhury A, et al. Mitochondrial oxidative stress and permeability transition in isoniazid and rifampicin induced liver injury in mice. J Hepatol. 2006;45:117–126. doi: 10.1016/j.jhep.2006.01.027.  [DOI](https://doi.org/10.1016/j.jhep.2006.01.027) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16545483/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Hepatol&title=Mitochondrial%20oxidative%20stress%20and%20permeability%20transition%20in%20isoniazid%20and%20rifampicin%20induced%20liver%20injury%20in%20mice&author=A%20Chowdhury&volume=45&publication_year=2006&pages=117-126&pmid=16545483&doi=10.1016/j.jhep.2006.01.027&)

37. Cole A, May PM, Williams DR. Metal binding by pharmaceuticals. Part 3. Copper (II) and zinc (II) interactions with isoniazid. Agents Actions. 1983;13:91–97. doi: 10.1007/BF01994289.  [DOI](https://doi.org/10.1007/BF01994289) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/6858791/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Agents%20Actions&title=Metal%20binding%20by%20pharmaceuticals.%20Part%203.%20Copper%20(II)%20and%20zinc%20(II)%20interactions%20with%20isoniazid&author=A%20Cole&author=PM%20May&author=DR%20Williams&volume=13&publication_year=1983&pages=91-97&pmid=6858791&doi=10.1007/BF01994289&)

38. Krivis AF, Rabb JM. Cuprous complexes formed with isonicotinic hydrazide. Science. 1969;164:1064–1065. doi: 10.1126/science.164.3883.1064.  [DOI](https://doi.org/10.1126/science.164.3883.1064) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/4976919/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Cuprous%20complexes%20formed%20with%20isonicotinic%20hydrazide&author=AF%20Krivis&author=JM%20Rabb&volume=164&publication_year=1969&pages=1064-1065&pmid=4976919&doi=10.1126/science.164.3883.1064&)

39. Divakar S, Vasudevachari MB, Antony A, Easwaran KR. Studies on the interaction of cupric isonicotinohydrazide with DNA. Biochemistry. 1987;26:3781–3785. doi: 10.1021/bi00387a007.  [DOI](https://doi.org/10.1021/bi00387a007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/3651412/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochemistry&title=Studies%20on%20the%20interaction%20of%20cupric%20isonicotinohydrazide%20with%20DNA&author=S%20Divakar&author=MB%20Vasudevachari&author=A%20Antony&author=KR%20Easwaran&volume=26&publication_year=1987&pages=3781-3785&pmid=3651412&doi=10.1021/bi00387a007&)

40. Ramadevi P, Singh R, Prajapati A, Gupta S, Chakraborty D. Cu(II) Complexes of Isoniazid Schiff Bases: DNA/BSA Binding and Cytotoxicity Studies on A549 Cell Line. Adv Chem. 2014;2014:1–14. doi: 10.1155/2014/630575.  [DOI](https://doi.org/10.1155/2014/630575) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Adv%20Chem&title=Cu(II)%20Complexes%20of%20Isoniazid%20Schiff%20Bases:%20DNA/BSA%20Binding%20and%20Cytotoxicity%20Studies%20on%20A549%20Cell%20Line&author=P%20Ramadevi&author=R%20Singh&author=A%20Prajapati&author=S%20Gupta&author=D%20Chakraborty&volume=2014&publication_year=2014&pages=1-14&doi=10.1155/2014/630575&)

41. Silva PB, et al. In vitro activity of Copper(II) complexes, loaded or unloaded into a nanostructured lipid system, against mycobacterium tuberculosis. Int J Mol Sci. 2016;17:745. doi: 10.3390/ijms17050745.  [DOI](https://doi.org/10.3390/ijms17050745) | [PMC free article](/articles/PMC4881567/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27196901/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Mol%20Sci&title=In%20vitro%20activity%20of%20Copper(II)%20complexes,%20loaded%20or%20unloaded%20into%20a%20nanostructured%20lipid%20system,%20against%20mycobacterium%20tuberculosis&author=PB%20Silva&volume=17&publication_year=2016&pages=745&pmid=27196901&doi=10.3390/ijms17050745&)

42. Polishchuk EV, et al. Wilson disease protein ATP7B utilizes lysosomal exocytosis to maintain copper homeostasis. Dev Cell. 2014;29:686–700. doi: 10.1016/j.devcel.2014.04.033.  [DOI](https://doi.org/10.1016/j.devcel.2014.04.033) | [PMC free article](/articles/PMC4070386/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24909901/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Dev%20Cell&title=Wilson%20disease%20protein%20ATP7B%20utilizes%20lysosomal%20exocytosis%20to%20maintain%20copper%20homeostasis&author=EV%20Polishchuk&volume=29&publication_year=2014&pages=686-700&pmid=24909901&doi=10.1016/j.devcel.2014.04.033&)

43. Tao TY, Gitlin JD. Hepatic copper metabolism: insights from genetic disease. Hepatology. 2003;37:1241–1247. doi: 10.1053/jhep.2003.50281.  [DOI](https://doi.org/10.1053/jhep.2003.50281) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12773998/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hepatology&title=Hepatic%20copper%20metabolism:%20insights%20from%20genetic%20disease&author=TY%20Tao&author=JD%20Gitlin&volume=37&publication_year=2003&pages=1241-1247&pmid=12773998&doi=10.1053/jhep.2003.50281&)

44. Mercer SW, Wang JB, Burke R. Modeling of the pathogenic effect of copper transporter mutations that cause menkes and wilson diseases, motor neuropathy, and susceptibility to Alzheimer’s disease. J. Biol. Chem. 2017;292:4113–4122. doi: 10.1074/jbc.M116.756163.  [DOI](https://doi.org/10.1074/jbc.M116.756163) | [PMC free article](/articles/PMC5354492/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28119449/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem.&title=Modeling%20of%20the%20pathogenic%20effect%20of%20copper%20transporter%20mutations%20that%20cause%20menkes%20and%20wilson%20diseases,%20motor%20neuropathy,%20and%20susceptibility%20to%20Alzheimer%E2%80%99s%20disease&author=SW%20Mercer&author=JB%20Wang&author=R%20Burke&volume=292&publication_year=2017&pages=4113-4122&pmid=28119449&doi=10.1074/jbc.M116.756163&)

45. European Association for the Study of the Liver. Clinical Practice Guideline Panel, C., Panel, m. & representative, E.G.B. EASL Clinical Practice Guidelines: Drug-induced liver injury. J Hepatol. 2019;70:1222–1261. doi: 10.1016/j.jhep.2019.02.014.  [DOI](https://doi.org/10.1016/j.jhep.2019.02.014) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30926241/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Hepatol&title=Clinical%20Practice%20Guideline%20Panel,%20C.,%20Panel,%20m.%20&%20representative,%20E.G.B.%20EASL%20Clinical%20Practice%20Guidelines:%20Drug-induced%20liver%20injury&volume=70&publication_year=2019&pages=1222-1261&pmid=30926241&doi=10.1016/j.jhep.2019.02.014&)

46. Singal P, Punia VP, Lohakare AC, Bansal S. Wilson’s disease unmasked by antitubercular therapy induced liver injury. J Assoc Phys India. 2013;61:514–515.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/24772766/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Assoc%20Phys%20India&title=Wilson%E2%80%99s%20disease%20unmasked%20by%20antitubercular%20therapy%20induced%20liver%20injury&author=P%20Singal&author=VP%20Punia&author=AC%20Lohakare&author=S%20Bansal&volume=61&publication_year=2013&pages=514-515&pmid=24772766&)
