# Variation in the Gene Encoding the Serotonin 2A Receptor Is Associated with Outcome of Antidepressant Treatment

## Metadata
**Authors:** Francis J McMahon, Silvia Buervenich, Dennis Charney, Robert Lipsky, A John Rush, Alexander F Wilson, Alexa J M Sorant, George J Papanicolaou, Gonzalo Laje, Maurizio Fava, Madhukar H Trivedi, Stephen R Wisniewski, Husseini Manji
**Journal:** American Journal of Human Genetics
**Date:** 2006 Mar 20
**DOI:** [10.1086/503820](https://doi.org/10.1086/503820)
**PMID:** 16642436
**PMCID:** PMC1474035
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1474035/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC1474035/pdf/AJHGv78p804.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC1474035/pdf/AJHGv78p804.pdf)

## Abstract

Depressive disorders account for a large and increasing global burden of disease. Although the condition of many patients improves with medication, only a minority experience full remission, and patients whose condition responds to one medication may not have a response to others. Individual variation in antidepressant treatment outcome is, at present, unpredictable but may have a partial genetic basis. We searched for genetic predictors of treatment outcome in 1,953 patients with major depressive disorder who were treated with the antidepressant citalopram in the Sequenced Treatment Alternatives for Depression (STAR*D) study and were prospectively assessed. In a split-sample design, a selection of 68 candidate genes was genotyped, with 768 single-nucleotide–polymorphism markers chosen to detect common genetic variation. We detected significant and reproducible association between treatment outcome and a marker in HTR2A (P range 1×10-6 to 3.7×10-5 in the total sample). Other markers in HTR2A also showed evidence of association with treatment outcome in the total sample. HTR2A encodes the serotonin 2A receptor, which is downregulated by citalopram. Participants who were homozygous for the A allele had an 18% reduction in absolute risk of having no response to treatment, compared with those homozygous for the other allele. The A allele was over six times more frequent in white than in black participants, and treatment was less effective among black participants. The A allele may contribute to racial differences in outcomes of antidepressant treatment. Taken together with prior neurobiological findings, these new genetic data make a compelling case for a key role of HTR2A in the mechanism of antidepressant action.

## Methods

### Sample

The rationale, methods, and design of the STAR*D study have been detailed elsewhere.[13](#RF16) In brief, investigators at 14 regional centers across the United States implemented a standard study protocol at 41 clinical sites.

Subjects provided separate written informed consent for study participation and for the collection of blood samples for genetic studies. Outpatients aged 18–75 years with a baseline Hamilton Depression Rating Scale score[14](#RF17)^,^[15](#RF18) of ⩾14 who met DSM-IV[16](#RF19) criteria for nonpsychotic MDD were eligible. Patients with bipolar, psychotic, or obsessive-compulsive disorders were excluded, as were those with primary eating disorders, general medical conditions that contraindicated study medications, substance dependence requiring inpatient detoxification, and clear nonresponse or intolerance to any protocol antidepressant during current episode or those who were pregnant or breast-feeding.

The 16-item Quick Inventory of Depressive Symptomatology–Clinician-rated (QIDS-C_16_)[12](#RF15)^,^[13](#RF16)^,^[17](#RF20)^,^[18](#RF21) was obtained at baseline and at each treatment visit, to measure symptom severity. The intraclass correlation coefficient for the QIDS-C_16_, repeated across raters over 4 years, was 0.96 (A.J.R.'s unpublished data). Patients with a baseline QIDS-C_16_ >10 were eligible if the treating clinician determined that outpatient treatment with an antidepressant medication was indicated and safe. At level 1, the protocol required an adequate dose of citalopram for a sufficient time to maximize the likelihood of treatment success, to ensure that those who did not improve were most likely unresponsive to the medication, not just underdosed.[12](#RF15) No concomitant medications were allowed, aside from benzodiazepines and hypnotics if needed. A CONSORT (Consolidated Standards of Reporting Trials) diagram of the current study sample is shown in [figure 1](#FG1).

### Figure 1.

![Figure  1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3d6c/1474035/a14483473f4f/AJHGv78p804fg1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=1474035_AJHGv78p804fg1.jpg)

CONSORT chart of genotyping and analysis of STAR*D sample. Samples dropped from analysis comprised 34 subjects who were noncompliant with medications, 19 with initial QIDS-C16 <10 or missing, 5 with missing clinical data, 4 whose molecular and recorded sex did not match, and 3 duplicate samples.

### DNA Samples

DNA samples were collected from 1,953 participants. A sample of 20 ml of whole blood was collected in citrate-treated vacuum tubes and was shipped overnight to the Rutgers Cell Repository, where lymphocytes were extracted and cryopreserved using standard methods. DNA was extracted using GenePure chemistry (Qiagen) and was shipped on dry ice to the NIH laboratories. Samples were arrayed using a Tecan Genesis robot (Lipsky Lab), then sex-verified with a set of three X-linked and two Y-linked markers (McMahon Lab). Four sex discrepancies were identified and were excluded before samples were genotyped further.

A summary of the sample characteristics is shown in [table 1](#TB1). Those who consented to have blood drawn were similar to those in the full study sample but showed slight differences in several variables that reached statistical significance because of the large sample size. Subjects who consented to have blood drawn were older and better educated, with higher household income, and were more likely to be married, to be retired, and to describe themselves as white. These subjects were also more likely to come from a primary-care setting and to report more time elapsed since their first major depressive episode (MDE), more episodes, and greater comorbidity. These differences cannot affect the genetic association results, which derive from comparisons among the genotyped subjects. However, these differences may limit the generalizability of our findings, and clinical outcomes in the genotyped sample may differ somewhat from those in the full STAR*D sample.[12](#RF15)

### Table 1.

|   |   | Patients with Blood Drawn |  | Comparison |  |  |
| --- | --- | --- | --- | --- | --- | --- |
| Characteristic | Complete Sample(N=4,041) | Yes(n=1,953) | No(n=2,088) | Test Statistica | df | Pb |
| Sociodemographic: |   |   |   |   |   |   |
| Mean (±SD) age (years) | 40.5 ± 13.3 | 42.7 ± 13.4 | 38.4 ± 12.9 | t = 10.31 | 4,037 | <.0001 |
| Sex: |   |   |   | χ2 = 1.48 | 1 | NS |
| Male | 1,509 (37.3) | 748 (38.3) | 761 (36.4) |   |   |   |
| Female | 2,532 (62.7) | 1,205 (61.7) | 1,327 (63.6) |   |   |   |
| Race: |   |   |   | χ2 = 13.20 | 2 | .0014 |
| White | 3,055 (75.7) | 1,526 (78.2) | 1,529 (73.4) |   |   |   |
| Black | 709 (17.6) | 313 (16.0) | 396 (19.0) |   |   |   |
| Other/mixed | 272 (6.7) | 113 (5.8) | 159 (7.6) |   |   |   |
| Mean (±SD) no. of years of education | 13.4 ± 3.2 | 13.6 ± 3.2 | 13.3 ± 3.2 | χ2 = 13.01 | 1 | .0003 |
| Employment: |   |   |   | χ2 = 19.65 | 2 | <.0001 |
| Employed | 2,311 (57.3) | 1,092 (55.9) | 1,219 (58.6) |   |   |   |
| Unemployed | 1,489 (36.9) | 715 (36.6) | 774 (37.2) |   |   |   |
| Retired | 234 (5.8) | 146 (7.5) | 88 (4.2) |   |   |   |
| Mean (±SD) monthly household income (U.S. $) | 2,419 ± 3,143 | 2,521 ± 3,202 | 2,318 ± 3,082 | χ2 = 6.23 | 1 | .0125 |
| Medical insurance: |   |   |   | χ2 = .68 | 2 | NS |
| Private | 2,022 (51.8) | 998 (52.4) | 1,024 (51.2) |   |   |   |
| Public | 553 (14.2) | 270 (14.2) | 283 (14.1) |   |   |   |
| None | 1,332 (34.1) | 638 (33.5) | 694 (34.7) |   |   |   |
| Marital status: |   |   |   | χ2 = 15.12 | 3 | .0017 |
| Single | 1,207 (29.9) | 545 (27.9) | 662 (31.8) |   |   |   |
| Married/cohabiting | 1,663 (41.2) | 838 (42.9) | 825 (39.6) |   |   |   |
| Divorced/separated | 1,037 (25.7) | 493 (25.2) | 544 (26.1) |   |   |   |
| Widowed | 128 (3.2) | 77 (3.9) | 51 (2.4) |   |   |   |
| Clinical: |   |   |   |   |   |   |
| Mean (±SD) age at first MDE (years) | 25.5 ± 14.4 | 26.1 ± 14.9 | 24.9 ± 13.9 | χ2 = 3.40 | 1 | NS |
| Mean (±SD) time since first MDE (years) | 15.0 ± 13.1 | 16.6 ± 13.9 | 13.5 ± 12.1 | χ2 = 42.07 | 1 | <.0001 |
| Mean (±SD) no. of MDEs | 5.9 ± 11.4 | 6.4 ± 12.5 | 5.4 ± 10.2 | χ2 = 11.09 | 1 | .0009 |
| Suicide ever attempted: |   |   |   | χ2 = 6.26 | 1 | .0123 |
| Yes | 667 (16.5) | 293 (15.0) | 374 (17.9) |   |   |   |
| No | 3,370 (83.5) | 1,659 (85.0) | 1,711 (82.1) |   |   |   |
| No. of psychiatric comorbidities: |   |   |   | χ2 = 24.39 | 4 | <.0001 |
| 0 | 1,510 (38.2) | 781 (40.9) | 729 (35.7) |   |   |   |
| 1 | 1,028 (26.0) | 510 (26.7) | 518 (25.4) |   |   |   |
| 2 | 607 (15.4) | 282 (14.8) | 325 (15.9) |   |   |   |
| 3 | 342 (8.7) | 133 (7.0) | 209 (10.2) |   |   |   |
| ⩾4 | 465 (11.8) | 204 (10.7) | 261 (12.8) |   |   |   |
| Current episode: |   |   |   |   |   |   |
| Clinical setting: |   |   |   | χ2 = 29.97 | 1 | <.0001 |
| Primary | 1,575 (39) | 846 (43.3) | 729 (34.9) |   |   |   |
| Specialty | 2,466 (61) | 1,107 (56.7) | 1,359 (65.1) |   |   |   |
| Mean (±SD) duration of current episode (mo) | 24.5 ± 52.0 | 24.8 ± 53.1 | 24.3 ± 51.0 | χ2 = .78 | 1 | .3764 |
| HDRS-17c | 18.8 ± 6.5 | 18.4 ± 6.2 | 19.6 ± 6.9 | t = 1.65 | 330 | .099 |
| QID-S16 | 13.8 ± 4.2 | 13.4 ± 4.1 | 14.5 ± 4.5 | t = 2.20 | 373 | .0283 |

Table 1 Caption: Selected Demographic and Clinical Characteristics of the STAR*D Sample[Note]

### Phenotypes

All phenotype definitions and assignments were settled in advance and were assigned before genotyping. Patients were scored for treatment outcome in two ways: designated remission and response ([fig. 2](#FG2)). In the absence of external validators, our choice of categorical phenotypes was guided (1) by careful work with the STAR*D clinicians—in advance of the genotyping—to develop distinctions that had face validity and took advantage of the large body of data available from the STAR*D trial; (2) by ensuring maximal contrast between the outcome groups, to improve power, and creating “probable” groups that approximated the more narrowly defined categories, to test their robustness; (3) and by paying special attention to full remission of symptoms, since this was the primary target outcome of treatment.

### Figure 2.

![Figure  2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3d6c/1474035/e1bf2a81d3ce/AJHGv78p804fg2.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=1474035_AJHGv78p804fg2.jpg)

Treatment-outcome phenotypes. Subjects who completed at least 6 wk of treatment with citalopram were assigned a remission and response phenotype that was based on the QIDS-C16 score at the last treatment visit. Those who met score criteria for remission or response after 3–6 wk of treatment were grouped with probable remitters or probable responders, respectively.

*Remitters* achieved a QIDS-C_16_ score of ⩽5 at the last treatment visit; probable remitters achieved a score of 6 or 7. Nonremitters had a QIDS-C_16_ score of ⩾10 at the last visit. Those with a final QIDS-C_16_ score in the borderline range of 8 and 9 were excluded from analysis.

*Responders* achieved at least a 50% reduction in baseline QIDS-C_16_ at the last treatment visit; probable responders achieved a 45%–50% reduction. Nonresponders did not achieve even a 40% reduction in baseline QIDS-C_16_ score at the last treatment visit. Those with a reduction in QIDS-C_16_ in the borderline range of 40%–45% were excluded from analysis.

Only patients who completed at least 6 wk of treatment were included in the primary analysis. Patients who achieved the required QIDS-C_16_ scores after <6 wk of treatment but who received at least 3 wk of treatment were assigned to the appropriate outcome group but were classified as “probable.” Those who did not complete at least 3 wk of treatment were excluded from analysis. Similarly, subjects who were classified as “intolerant” or “probably intolerant” were removed from the nonremitter and nonresponder groups but were retained in the remitter and responder groups, since intolerant subjects were probably not able to take the full effective dose of citalopram but might have responded if they had. Assessment of tolerability is discussed below. Subjects who did not adhere to the treatment regimen were excluded from analysis.

As a secondary test, relative change in QIDS-C_16_ score at the last visit (expressed as percentage change from initial score) was tested as a quantitative trait, after removal of intolerant and nonadherent subjects.

### Tolerability

Medication tolerability comprises an individual’s objective and perceived side-effect burden and typically increases over time and with response to treatment.[19](#RF22) Since failure to consider tolerability could lead to misclassification of intolerant patients as nonresponders, we scored all subjects as tolerant, probably tolerant, intolerant, or probably intolerant on the basis of an algorithm that considered study exit data and the Global Rating of Side Effect Burden (GRSEB).[13](#RF16) In brief, all subjects who elected to continue citalopram at the end of the level 1 treatment period were considered tolerant, whereas subjects who refused to continue citalopram or who left the study because of side effects were considered intolerant. The remaining subjects were classified on the basis of GRSEB score into probably tolerant (no more than moderate side effects) or probably intolerant (more than moderate side effects). A small number of subjects with missing GRSEB scores were classified according to whether they took citalopram for <4 wk (probably intolerant) or ⩾4 wk (probably tolerant).

### Candidate Genes

Sixty-eight genes were chosen for study from among a larger list of plausible candidates. Genes primarily involved in drug metabolism were excluded, by prior agreement, since these will be studied by another group using the same set of DNA samples. Genes were scored by an expert panel (D.C., W. Drevets, H.M., and F.J.M.) on the basis of (1) prior evidence of association with antidepressant outcome (1–3 points), (2) prior evidence of association with major mood disorders (1–3 points), and (3) known functional variant(s) (0–1 points). Under this scoring system, candidate genes could receive 0–7 points; higher scores conferred higher priority for study. Genes with a score of ⩾4 were used to seed sets of related genes broadly encompassing five main pathways: serotonin related (*n*=20), glutamate related (*n*=16), dopamine related (*n*=3), adrenergic (*n*=4), and neurotrophic (*n*=4), along with selected genes in other pathways (*n*=21). The complete list of genes studied is shown in [table 2](#TB2).

### Table 2.

| Hypothesis and Gene Symbol | Gene Name |
| --- | --- |
| Dopamine hypothesis: |   |
| TH | Tyrosine hydroxylase |
| COMT | Catechol-O-methyltransferase |
| MAOA | Monoamine-oxidase A |
| Adrenergic hypothesis: |   |
| ADRA2A | alpha-2a adrenergic receptor |
| ADRA2C | alpha-2c adrenergic receptor |
| DBH | Dopamine beta-hydroxylase |
| SLC6A2 | Norepinephrin transporter |
| Serotonin hypothesis: |   |
| SLC6A4 | Serotonin transporter |
| TPH1 | Tryptophane hydroxylase-1 |
| TPH2 | Tryptophane hydroxylase-2 |
| HTR1A | Serotonin receptors |
| HTR1B | Serotonin receptors |
| HTR1D | Serotonin receptors |
| HTR1E | Serotonin receptors |
| HTR1F | Serotonin receptors |
| HTR2A | Serotonin receptors |
| HTR2B/PSMD1 | Serotonin receptors |
| HTR2C | Serotonin receptors |
| HTR3A | Serotonin receptors |
| HTR3B | Serotonin receptors |
| HTR3C | Serotonin receptors |
| HTR3D | Serotonin receptors |
| HTR3E | Serotonin receptors |
| HTR4 | Serotonin receptors |
| HTR5A | Serotonin receptors |
| HTR6 | Serotonin receptors |
| HTR7 | Serotonin receptors |
| Glutamate hypothesis: |   |
| GRIA1 | AMPA receptors |
| GRIA2 | AMPA receptors |
| GRIA3 | AMPA receptors |
| GRIA4 | AMPA receptors |
| GRIN1 | NMDA receptors |
| GRIN2A | NMDA receptors |
| GRIN2B | NMDA receptors |
| GRIN2C | NMDA receptors |
| GRIN2D | NMDA receptors |
| GRIN3A | NMDA receptors |
| GRIK1 | Kainate receptors |
| GRIK2 | Kainate receptors |
| GRIK3 | Kainate receptors |
| GRIK4 | Kainate receptors |
| GRIK5 | Kainate receptors |
| SLC1A1 | Glutamate/aspartate transporter |
| Neurotrophin hypothesis: |   |
| BDNF | BDNF |
| NTRK2 | Trk-B |
| BCL2 | B-cell CLL/lymphoma 2 |
| BAG1 | BCL2-associated athanogene |
| Other signaling pathways: |   |
| PPP1R1B | DARP-32 |
| NR3C2 | Mineralocorticoid receptor |
| CREB1 | CREB |
| MAPK1 | Mitogen-activated protein kinase 1 |
| GSK3B | Glycogen synthase kinase 3 beta |
| CAMK1 | Calcium/calmodulin-dependent protein kinase I |
| PPP3R2 | Calcineurin B (located within GRIN3A) |
| Other genes: |   |
| FKBP5 | FK506 binding protein 5 |
| LAMA4 | Laminin alpha-4 |
| GNB3 | Guanine nucleotide binding protein |
| OGG1 | 8-Oxoguanine DNA glycosylase |
| NET-5 | Tetraspan NET-5 |
| NBL1 | Neuroblastoma, suppression of tumorigenicity 1 |
| GRWD1 | Glutamate-rich WD repeat containing 1 |
| RPP30 | Ribonuclease P (30 kDa) |
| RNF20 | Hypothetical protein FLJ20690 |
| FBXO38 | F-box only protein 38 |
| ARHGAP10 | rho GTPase activating protein 21 |
| NR1I2 | Orphan nuclear receptor PAR2 |
| KDELR1 | KDELR1 protein |
| ATP1A3 | ATPase, Na+/K+ transporting, alpha-3 polypeptide |

Table 2 Caption: List of Genes Screened

### Selection of SNP Markers

For each candidate gene, genotype data spanning the coding region and up to 2 kb of flanking sequence were downloaded from the [International HapMap Project](#RF2), accessed November 2004.[20](#RF23) Since the STAR*D sample is mostly white, data from the CEPH sample (Utah residents with northern and western European ancestry) were used. The program LDSelect[21](#RF24) was used to select an optimal set of available SNPs to genotype, at an *r*^2^ threshold of ⩾0.8. From the remaining SNPs, we further excluded those with a minor-allele frequency <7.5%, since we expected the alleles that contribute to treatment outcome in this data set to be common. Six nonsynonymous SNPs and four SNPs reported elsewhere[22](#RF25)^,^[23](#RF26) to be associated with treatment outcome were added to the set, which brought the total to 768. Illumina then performed a bioinformatic screen that identified 12 markers that would likely fail in their BeadArray assay. Predicted failures were replaced by a nearby marker that was in strong linkage disequilibrium (LD) with the excluded SNP, if available. Absent this, a nearby marker with an allele frequency similar to that of the excluded marker was selected. The complete list of SNPs genotyped, along with flanking sequence and expected alleles, is available on request.

### Genotyping Methods

Since all of the collected samples were not available at the start of the experiment, only the first 1,380 samples were shipped to Illumina, where they were genotyped on the Illumina BeadArray platform, a highly accurate, high-throughput assay.[24](#RF27) At Illumina, 99.78% of samples were successfully genotyped, and 97.92% of SNPs produced usable data, so that a total of 1,034,602 of a possible 1,035,504 genotypes were returned, including 11,280 blind duplicate genotypes, all of which matched exactly.

On the basis of the results of the first 1,380 samples, five SNPs in the remaining samples were genotyped using Taqman chemistry, were scored on a fluorescent plate reader (Molecular Dynamics) without regard to phenotype. By design, 394 samples were genotyped at *rs7997012* and *rs1928040* both by Illumina and in-house (McMahon Lab). No discrepancies were detected.

### Analysis Plan

The primary experiment was based on comparison of allele and genotype frequencies between subjects who benefited or did not benefit from citalopram therapy. Because the number of tests in this experiment was large, a split sample design was employed. The 1,380 samples genotyped for all SNPs were divided, a priori, into a discovery sample and a replication sample. The discovery sample consisted of two-thirds of the total sample genotyped at Illumina; the replication sample consisted of the remaining one-third. The choice of asymmetric sample sizes for the discovery and test samples was based on the large overall sample size. Splitting the sample in this fashion allowed for the detection of genetic effects of equal size but with a more conservative nominal significance level in the discovery sample than in the test sample. The split samples were matched on sex and reported race (collapsed into “white,” “black,” and “other/mixed”). Any SNPs meeting the a priori significance levels in both the discovery and test samples (see below) were tested for robustness in the total data set.

### Power Analysis

The power to detect association was estimated in two ways. In the first, standard statistical methods[25](#RF28) were used to determine the effect size for approximate sample sizes for the discovery and replication samples and for a third sample that included the discovery, replication, and all remaining samples. The discovery sample was screened with a nominal significance level of .01, which had a power of 0.9 to detect an allelic effect size of 0.16. The power to detect a similar effect size in the smaller replication sample was ∼0.8 at the *P*=.05 level.

In addition, the power to detect allelic and genotypic (additive) effects was estimated for logistic regression analysis for a variety of genetic models (Quanto v. 1.0 beta [[Gene×Environment, Gene×Gene Interaction home page](#RF1)]).[26](#RF29) There are a large number of possible underlying genetic models. Estimates of power under a specified model are appropriate only when the specified model is, in fact, the true underlying model. For example, in an unmatched case-control design (case:control ratio of 1:0.5), for a log-additive model with allele frequency of .3, a significance level of .01, and a power of .90, the detectable genetic relative risks for sample sizes of 600 and 300 were ∼1.5 and ∼1.7, respectively. At a power of .80 and a significance level of .05, corresponding genetic relative risks were ∼1.4 and ∼1.6, respectively.

### Race and Ethnicity

Subjects were enrolled into the STAR*D study without regard to race or ethnicity. Detailed data on ethnicity were not collected, but most subjects identified themselves as “white,” “black,” or “other.” We attempted to verify this self-reported race using STRUCTURE,[27](#RF30) which assigns a probability of group membership to individuals on the basis of marker-allele frequencies. Since the number of subjects of “other” ethnicity in the sample was small, they were excluded from the analysis. We prepared a test data set of 1,284 individuals containing 57 unlinked loci selected from among those genotyped at Illumina, and we ran this in STRUCTURE under an admixture model with correlated allele frequencies, performing 20,000 burn-ins and 20,000 repetitions. This set of markers robustly distinguished between black and white subjects in this sample, with median posterior probabilities of group membership of 0.8 for the black subjects and 0.6 for the white subjects. We then divided the sample into self-identified whites and self-identified blacks and ran STRUCTURE again with the same markers. In each subject, STRUCTURE identified only one major cluster, and a one-population model gave the best fit to the data (data not shown).

### Statistical Methods

All pairs of individuals were compared using RELCHECK,[28](#RF31) which verified that individuals were unrelated. Because the sample consisted entirely of patients with major depression, a marker that increased risk for major depression in many of these individuals would not necessarily be in Hardy-Weinberg equilibrium.[29](#RF32) Thus, SNPs were not removed from analysis because of deviation from Hardy-Weinberg expectations.

Tests of association included the Pearson χ^2^ test (2×2 for allelic and 2×3 for genotypic associations), likelihood-ratio χ^2^ test (2×2 for allelic and 2×3 for genotypic associations), and Fisher’s exact test (2×2 for allelic associations only). Since each test has complementary advantages, all tests were used to make decisions about replication. This strategy might increase false-positive results due to multiple testing, but the high correlation between the tests makes this unlikely. Some of the SNPs tested had relatively rare alleles, which led to contingency tables with small cell sizes. The Pearson χ^2^ test is only an approximation for these SNPs but makes no assumptions about the underlying model. Fisher’s exact test is robust to small cells but assumes that the marginal distributions of the contingency table are fixed, which may not hold in this sample. When small cell size warnings were ignored, correlations among *P* values from the Pearson χ^2^, likelihood-ratio χ^2^, and Fisher’s exact test were high (> 0.9) for the main SNP of interest. Fisher’s exact test was the most conservative of the tests considered, so those are the findings presented in the “[Results](#SC2)” section. For the quantitative outcome phenotype, genotypic means were compared with the *F* test. All *P* values were from two-tailed tests. LD between adjacent SNPs was estimated using Haploview 3.2,[30](#RF33) which generates estimates of *D′* and *r*^2^ that are based on the input genotypes.

## Results

### Marker Coverage

The genotyped SNPs sampled the common variation within the genes selected for study at a median pairwise *D′* value of 0.81. Overall, 75% of adjacent marker pairs were in LD at a *D*^′^>0.73. Only 5% of SNP pairs were associated with each other at an *r*^2^ value of >0.8, demonstrating that our SNP-selection strategy effectively excluded redundant markers.

### Allelic Association Results

Each SNP was tested for association with treatment response and remission in the discovery sample (*n*=1,380). A total of 12 SNPs met or exceeded the nominal significance level of .01 for one or both phenotypes. Of these, only one SNP met or exceeded the nominal significance level of .05 in the replication sample for the same allele and phenotype ([fig. 3](#FG3)). This SNP, *rs7997012,* resides in the second intron of the gene *HTR2A,* which encodes the serotonin 2A receptor. None of the other 767 SNPs met these strict criteria for association and replication in this experiment.

### Figure 3.

![Figure  3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3d6c/1474035/2adc050a431b/AJHGv78p804fg3.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=1474035_AJHGv78p804fg3.jpg)

Allelic association between treatment outcome and each of 768 SNPs representing 68 candidate genes. Allelic association P values were ranked in the discovery and replication samples, then the ranks were summed across samples. For visual clarity, the inverse of the sum of ranks is shown on the Y-axis. The remission phenotype is indicated by the unblackened circles, the response phenotype by the blackened circles. Only rs7997012 met the a priori P value thresholds in both samples. Another SNP, rs2178865, ranked highly in both samples but fell short of the a priori significance level in the replication sample.

In *HTR2A,* significant association was detected, in both samples, between the same (A) allele of *rs7997012* and treatment response ([table 3](#TB3)). (Evidence of association was also detected between this allele and the remission phenotype, but this did not meet our a priori significance levels in both samples.) An additional SNP in *HTR2A* (*rs1928040*) showed evidence of association with response and remission in the discovery sample but not in the replication sample. LD analysis demonstrated that these two SNPs were not in strong LD with each other ([fig. 4](#FG4)).

### Table 3.

|   | All |  |  | White |  |  | Black |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
|   |   | P |  |   | P |  |   | P |  |
| Phenotype and SNP | N | Allelewise | Genotypewise | n | Allelewise | Genotypewise | n | Allelewise | Genotypewise |
| Remission: |   |   |   |   |   |   |   |   |   |
| rs7997012 | 1,149 | .000024 | .000035 | 911 | .00107 | .00183 | 170 | NS | NS |
| rs1928040 | 1,148 | .0446 | .0701 | 910 | .0626 | NS | 170 | NS | NS |
| rs6313 | 1,183 | NS | NS | 942 | NS | NS | 172 | NS | NS |
| rs6311 | 1,180 | NS | NS | 939 | NS | NS | 172 | .0431 | .0874 |
| Response: |   |   |   |   |   |   |   |   |   |
| rs7997012 | 1,329 | .000037 | .000002 | 1,049 | .00183 | .000157 | 199 | NS | NS |
| rs1928040 | 1,327 | .0709 | NS | 1,048 | NS | NS | 199 | NS | NS |
| rs6313 | 1,372 | NS | NS | 1,086 | NS | NS | 202 | NS | NS |
| rs6311 | 1,371 | NS | NS | 1,084 | NS | NS | 203 | .0918 | .0149 |
| Change in QIDS-C16: |   |   |   |   |   |   |   |   |   |
| rs7997012 | 1,749 | .000007 | .00000146 | 1,380 | .00123 | .000516 | 261 | NS | NS |
| rs1928040 | 1,747 | .0214 | .0072 | 1,378 | .0738 | .0887 | 261 | NS | NS |
| rs6313 | 1,802 | NS | .0878 | 1,425 | NS | NS | 264 | NS | .0353 |
| rs6311 | 1,804 | .0599 | .0494 | 1,426 | NS | NS | 265 | .0094 | .0261 |

Table 3 Caption: Results of Association Analysis of Genotyped HTR2A SNPs, Stratified by Race[Note]

### Figure 4.

![Figure  4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3d6c/1474035/bbb58893b584/AJHGv78p804fg4.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=1474035_AJHGv78p804fg4.jpg)

Physical positions and LD relationships among SNP markers genotyped in the STAR*D sample. This figure shows the Refseq gene model for HTR2A taken from the UCSC Genome Browser (build 43), juxtaposed on a graphical representation of intermarker r2 values produced by Haploview 3.2.30

On the basis of these results, we genotyped *rs7997012* and *rs1928040* in the remaining subjects and tested association with treatment outcome in the total sample. Significant evidence of association was again observed between *rs7997012* and treatment outcome ([table 3](#TB3)), with *P* values on the order of 10^-6^ for the treatment response phenotype. Evidence of association between *rs1928040* and treatment outcome was not substantially stronger in the total sample.

Since *rs7997012* and *rs1928040* are both intronic SNPs with no known function, we also genotyped two SNPs in *HTR2A* (*rs6313* and *rs6311*) that may have functional importance.[31](#RF34)^^^–^[33](#RF36) Both SNPs were in tight LD with each other and with *rs1928040,* but they were not in significant LD with *rs7997012.* Neither *rs6316* nor *rs6311* showed significant association with treatment response or remission in the total sample ([table 3](#TB3)).

Further support for association with *rs7997012* was obtained in the quantitative-trait analysis ([table 3](#TB3)). The marker *rs7997012* was significantly associated with relative change in initial QIDS-C_16_ score, with a *P* value of 7.0×10^-5^ for allelic and 1.0×10^-6^ for genotypic association. Less-significant evidence of association was also observed at other *HTR2A* SNPs ([table 3](#TB3)).

### Stratified Analyses

Having established an association between treatment outcome and *HTR2A* in this sample, we next sought to characterize the association with respect to medication tolerability and race, using data sets stratified by the variable of interest. We used *rs7997012* as the test SNP and remission as the outcome variable, since they had yielded the strongest evidence of association in the earlier analyses.

In the primary analysis, medication-intolerant subjects were dropped from the *nonremitter* and *nonresponder* groups. However, this clinically justified decision created an imbalance with the *remitter* and *responder* groups, which included medication-intolerant subjects. To rule out any role of this imbalance in the primary association finding, we dropped the medication-intolerant subjects from the *remitter* and *responder* groups, then reanalyzed the data. Significant association between *rs7997012* and treatment outcome remained in the sample of medication-tolerant subjects (*P*=2.3×10^-5^, *n*=1,140 for the remission; *P*=2.3×10^-5^, *n*=1,303 for the response phenotypes).

It has been suggested that black patients respond less well than white patients to antidepressant medications, particularly SSRIs, but firm data are lacking. When we divided this sample by race into “white” and “black” strata, two important differences emerged. First, we noted that the A allele of *rs7997012* that was associated with better treatment outcome in the mixed-race sample was more than six times more frequent in the white than in the black participants (allele frequency in *whites*=0.42, in blacks, 0.06). Second, it was apparent that the association between *HTR2A* and treatment outcome was largely confined to the white participants. No significant association between *rs7997012* and either treatment response or remission was detected in the black participants, although nominally significant results appeared for *rs6313* and *rs6311* in some analyses ([table 3](#TB3)).

### Potential Confounders

Carriers of the A allele of *rs7997012* were compared with the other cases by means of several variables that could themselves be associated with treatment outcome: maximum citalopram dose (in mg), tolerability, sex, and initial QIDS-C_16_ score. Carriers of the A allele did not differ from the other cases for any of these variables (data not shown).

### Strength of Association

For treatment-outcome phenotypes, the usual measures of strength of association, on the basis of odds ratios, are perhaps less informative than are measures that directly capture differences in clinical outcome.[34](#RF37) Thus, we compared the rates of treatment response among participants grouped by genotype at *rs7997012.* Overall, 79.9% of those homozygous for the A allele were classified as “responders,” compared with 62.4% of those homozygous for the G allele. Among white participants, the values were 79.7% and 63.5%, respectively. Thus, the AA genotype at *rs7997013* confers a 16%–18% reduction in absolute risk of being a nonresponder in this sample.

## Discussion

To our knowledge, this is the first demonstration of significant, reproducible association between genetic variation and outcome of antidepressant treatment. The association signal reproduces in both our discovery and our replication samples and increases in significance with the inclusion of additional subjects. We evaluated three related definitions of treatment effectiveness: two categorical outcomes (response and remission) and a quantitative outcome based on relative change in the final symptom score. The categorical outcomes were defined in a way that created gaps between responders/remitters and nonresponders/nonremitters, thus increasing the contrast between these groups. The quantitative-trait analysis (which included all patients) supported the results of the categorical phenotypes and demonstrated that the observed association was not a consequence of the arbitrary outcome categories. Furthermore, the *HTR2A* allele that was associated with better treatment outcome was more than six times more frequent in white than in black participants, who had an overall less favorable treatment outcome in this sample.[12](#RF15)

This study has several limitations and strengths. We did not perform a complete genomewide association study, so we cannot conclude that we have found all the genes that may be relevant to antidepressant-treatment outcome. Instead, we screened 68 genes selected by an expert panel as the best candidates. Given the available marker coverage, we did not sample all common variation in these genes, but pairwise LD analysis indicated that much of the common variation was sampled. The large sample size allowed detection of weak effects with high statistical confidence, along with replication testing. The association results, however, do not necessarily point to functional variants in *HTR2A.* Our strongest results are not attributable to either of the suspected functional variants in *HTR2A.* Instead, these results point to intronic variants whose functional relevance is unknown. (Indeed, given the local patterns of LD, we cannot fully exclude the possibility that the association signal actually reflects another, nearby gene.) However, our results set the stage for a focused search for functional variants in the 3′ end of *HTR2A* in future studies. We employed phenotypes on the basis of a uniform rating of clinical symptoms that allowed for comparatively simple scoring of treatment outcome. Alternative methods for scoring treatment outcome exist and could point to different genes. The STAR*D study employed a naturalistic ascertainment and treatment protocol. Thus, these results should have general relevance in the clinic population but may differ from those obtained in more narrowly ascertained study groups.

As expected for a single gene, the clinical impact of *HTR2A* on treatment outcome is modest. In the total sample, white subjects homozygous for the allele associated with better treatment outcome had an 18% decrease in absolute risk of nonresponse to citalopram, compared with subjects homozygous for the other allele. Additional alleles predictive of treatment outcome will need to be discovered before clinically more-relevant effect sizes are obtainable. In this regard, one pertinent negative finding should be highlighted. Outcome after treatment with SSRIs has been most often associated with variation in the gene encoding the serotonin transporter, *SLC6A4,* the primary target of SSRI action (reviewed by Anguelova et al.[35](#RF38)). We found no evidence of association among any of the four genotyped *SLC6A4* markers and treatment outcome in these data. Our split sample design, with a requirement that the same allele show association with the same phenotype by the same test in both samples, was implemented as a way to control for multiple testing but may have reduced the power to detect modest association signals.[36](#RF39)

We found an interesting two-way relationship between race, *HTR2A* variation, and treatment outcomes in this sample. First, significantly fewer black patients had favorable treatment outcomes in this study, partly because of more early dropouts.[12](#RF15) This is consistent with the few published reports that black patients respond less often than do whites to antidepressant medications,[37](#RF40) although the differences have generally been attributed to psychosocial factors. Second, the association found between *HTR2A* and antidepressant outcome in this study was largely confined to the white subjects. Although the sample size of the black population was smaller, no trend of association with *rs7997012* was detected for any treatment-outcome phenotype. Third, the allele that was associated with better treatment outcome was more than six times more frequent in whites than blacks in this sample. These results suggest that genetic variation in *HTR2A* should be considered along with psychosocial factors in attempts to explain racial differences in antidepressant-treatment outcomes.

Genetic variation in *HTR2A* has been widely implicated in a variety of neuropsychiatric disorders (reviewed by Norton and Owen[38](#RF41)), but it has not been convincingly demonstrated that it affects antidepressant-treatment outcome. One previous study found suggestive evidence that *HTR2A* was associated with a delayed and sustained pattern of treatment outcome in a small sample.[39](#RF42) Another small study found suggestive evidence that a different SNP in *HTR2A* was associated with short-term treatment outcome.[40](#RF43) One study of 443 depressed inpatients detected a marginally significant association between *HTR2A* variants and outcome.[41](#RF44) Finally, one small study observed differential treatment response in patients with one or two C alleles of the T102C polymorphism *rs6311*.[42](#RF45)

Although the precise molecular mechanisms by which antidepressants exert their beneficial effects remain to be fully elucidated, considerable data implicate the serotonergic system.[43](#RF46)^,^[44](#RF47) On the basis of radioligand binding, signal transduction, and amino acid sequences, 5-HT effectors currently comprise seven distinct receptors (5HT1–7). The 5-HT2A, B, and C subtypes are positively coupled with the enzyme phospholipase C (PLC).[45](#RF48) The 5-HT2A receptors are postsynaptic receptors that are highly enriched in neocortex and regulate the function of prefrontal-subcortical circuits implicated in the pathophysiology of depression.[46](#RF49)^,^[47](#RF50) The 5-HT2A receptors interact with Gq/G11 guanine nucleotide binding proteins (G proteins) and thereby stimulate PLC to produce the intracellular second messengers sn-1,2-DAG (an endogenous activator of protein kinase C) and inositol-1,4,5-trisphosphate (IP3), which stimulates the release of Ca^++^ from intracellular stores.

Considerable neurobiological data suggest that the 5-HT2A receptor plays an important role in antidepressant-drug action. Different classes of antidepressants, including citalopram, downregulate 5-HT2A receptors in rodent and primate forebrain within a time frame paralleling therapeutic effects in humans.[44](#RF47)^,^[46](#RF49)^,^[48](#RF51)^,^[49](#RF52) Selective 5-HT2A antagonists are effective in animal models of depression, and antisense oligonucleotides directed against 5-HT2A receptors regulate the development of depressivelike behavior in the learned-helplessness model.[50](#RF53) Although citalopram does not directly bind to 5-HT2A receptors, several antidepressants do bind 5-HT2A receptors as antagonists, which likely plays an important role in their therapeutic action.[43](#RF46) Finally, a growing body of work suggests that antidepressants bring about their delayed therapeutic effects by the induction of neuronal plasticity mediated by increased expression of brain-derived neurotrophic factor (BDNF).[47](#RF50)^,^[51](#RF54)^,^[52](#RF55) In this context, it is noteworthy that 5-HT2A antagonists block stress-induced downregulation of BDNF mRNA in rodents.[53](#RF56) Together, these results suggest that many of the neuroplastic events believed to underlie the efficacy of SSRIs are mediated, in part, via 5HT2A receptors.[53](#RF56) Although future studies are needed to delineate the precise cellular mechanisms by which the *HTR2A* SNP described herein affects response to the therapeutic effects of citalopram, the new genetic data presented here, taken together with the existing neurobiologic findings, make a compelling case for a key role of *HTR2A* in the mechanism of antidepressant action.

In conclusion, this study demonstrates that genetic variation in *HTR2A* is reproducibly associated with outcome of citalopram treatment in a large sample of outpatients with MDD. This same variation may contribute to racial differences in outcomes with SSRI treatment. Further studies are needed to define the functional changes in *HTR2A* that account for the association signal in this sample.

## Acknowledgments

This research was supported in part by the Intramural Research Programs of the NIMH, the National Institute on Alcohol Abuse and Alcoholism, and the National Human Genome Research Institute, NIH. The authors appreciate the efforts of the STAR*D research team in performing the clinical study and gathering the DNA samples. Data and sample collection was funded with federal funds from the NIMH, NIH, under contract N01MH90003 to University of Texas Southwestern Medical Center at Dallas (principal investigator, A. J. Rush). We thank Nirmala Akula and Jo Steele, for technical advice, and the Rutgers Cell and DNA Repository, for extracting DNA and providing samples to our laboratories. The content of this publication does not necessarily reflect the views or policies of the DHHS, nor does mention of trade names, commercial products, or organizations imply endorsement by the U.S. Government. We appreciate the support of Forest Laboratories for providing citalopram at no cost for the STAR*D study. Most important, we thank the study participants, without whom this study would not be possible.

## References

1. Gene×Environment, Gene×Gene Interaction home page, http://hydra.usc.edu/gxe/ (for Quanto, v. 1.0 beta)  [http://hydra.usc.edu/gxe/](http://hydra.usc.edu/gxe/)

2. International HapMap Project, http://www.hapmap.org/  [http://www.hapmap.org/](http://www.hapmap.org/)

3. UCSC Genome Browser, http://genome.ucsc.edu/  [http://genome.ucsc.edu/](http://genome.ucsc.edu/)
