# Synergistic Effect of Cytochrome P450 Family 3 Subfamily A Member 5 (CYP3A5) Genetic Variants in Tacrolimus Dose Determination in Indian Renal Transplant Patients

## Metadata
**Authors:** Rashmi J Hemani, Priyal M Chauhan, Ratika Srivastava, Nitiraj B Shete, Amit S Jojera, Shailesh M Soni, Sishir D Gang, Abhijit M Konnur, Umapati N Hegde, Hardik B Patel, Banibrata N Mukhopadhyay, Manan A Raval, Sachchida Nand Pandey
**Journal:** ACS Pharmacology & Translational Science
**Date:** 2024 Aug 13
**DOI:** [10.1021/acsptsci.4c00055](https://doi.org/10.1021/acsptsci.4c00055)
**PMID:** 39539273
**PMCID:** PMC11555502
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11555502/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC11555502/pdf/pt4c00055.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC11555502/pdf/pt4c00055.pdf)

## Abstract

Tacrolimus (TAC) has a narrow therapeutic index and shows interindividual variabilities in its blood concentration. Although guidelines recommend a genetic variant (rs776746) to determine the optimized TAC dose, discrepancies in accuracy have been noted. Therefore, studying other variants of CYP3A5 may improve the accuracy of the TAC dose. Clinical exome sequencing (CES) was performed in 219 renal transplant patients. The SNPs of CYP3A5 covered by CES were recorded. The TAC blood trough concentration/dose (C0/D) was calculated on day 7 and months 1, 3, 6, and 12 of post-transplantation, and association with CYP3A5 genotypes was studied. Further, biopsy-proven rejection and pathological events were analyzed for their association with CYP3A5 genotypes. Out of 35 variants of CYP3A5 covered in CES, rs776746, rs15524, rs4646449, and rs464645 were significantly associated with the TAC C0/D on day 7 and months 1, 3, and 6. Further analysis showed that the slow-metabolizing genotypes of all four SNPs synergistically associated with the TAC C0/D on day 7 and months 1, 3, 6, and 12. The “CC” genotype of rs776746 showed a significant association (RR = 1.613; p = 0.035) with allograft rejection. In addition, cox regression analysis showed that the presence of the “CA” genotype of rs4646453 increased (HR = 7.258; 95% CI = 1.354–38.904) the risk of development of pathological events, respectively. Four variants of CYP3A5 were synergistically associated with the TAC dose determination. In addition, rs776746 and rs4646453 may be associated with allograft rejection and pathological events, respectively.

Keywords: Tacrolimus, pharmacogenomics, CYP3A5, renal transplantation, single nucleotide polymorphism

## 1. Introduction

Tacrolimus (FK-506), a calcineurin antagonist, has enormously elevated the success of solid organ transplantations and graft survival rates compared with other immunosuppressives. It is accompanied by greater inter- and intravariabilities regarding the pharmacokinetics (PK) and has a tapered therapeutic window, which are the barriers to achieving effective and safe blood trough concentrations (*C*_0_).^1^ Procuring and maintaining the targeted concentration of antirejection agents by giving suitable doses are critical during the earliest post-transplantation. Because of its tapered therapeutic index, therapeutic drug monitoring (TDM) needs to be applied precisely to avoid over- or underconcentration, which might result in TAC-related toxicities or graft rejection, respectively.^2^ Attainment and maintenance of a favorable dose of TAC is still a major barrier setting back its wider use.^3^

TAC is biotransformed in the intestine and hepatic tissues through hepatic isoenzymes cytochrome P450, i.e., *CYP3A5* and *CYP3A4*. P-Glycoprotein (P-gp) pumps out TAC through an efflux mechanism for transportation in the gut.^4,5^ Any single nucleotide polymorphism (SNP) in those genes will be directly reflected in the drug dose and its systemic concentration.^6−8^ A polymorphism may result in 40–50% variations in the metabolism and clearance of TAC.^9^ The most extensively and commonly studied SNP for this enzyme is *CYP3A5*3* (rs776746), where the transition from A to G takes place at 6869 positions in 3 intronic regions. An abstruse splice site is formed, resulting in an abnormal termination codon and malfunctioning protein.^10^ One expressing this allele is classified as a *CYP3A5* nonexpresser, slow metabolizer, or homozygote.

Genome-wide screening and whole exome sequencing research represent a promising new strategy for finding novel SNPs of the *CYP3A5* gene that may be associated with Tacrolimus.^11,12^ Moreover, a patient’s clinical status, such as hyperimmune activity in the recipient, may also impact the TAC efficacy. This new approach is anticipated to identify novel genetic variations that can accurately predict TAC PK. Therefore, optimizing TAC dosing by designed dosing algorithms, including clinical factors and genetic variants affecting the PK of TAC, is necessary to forecast the TAC *C*_0_.

Although a large number of studies have been conducted to determine the association of the *CYP3A5* genotype with TAC dose response variabilities, so far, the determined pharmacogenetic variables have had minimal predictive status and limited clinical utility.^1,13,14^ Very few studies are being conducted on the influence of the *CYP3A5* genotype on TAC PK among the Indian population during and after renal transplantation. The primary purpose of this study was to determine the role of the *CYP3A5* genetic variants on the TAC dose and *C*_0_ among Indian renal transplant recipients. Pharmacogenomics-based Tacrolimus dosing would favor rapid attainment of the TAC therapeutic dose compared to weight-adjusted dosing for patients with solid organ transplants. This approach has proved to be beneficial in reducing drug-related side effects, overconcentration, and rejection issues and improving graft survival rates.

## 2. Materials and Methods

### 2.1. Subject Characteristics

This case-only study was performed on 219 renal transplant patients at Muljibhai Patel Urological Hospital (MPUH), Nadiad, Gujarat, India. All of the patients underwent renal transplant from December 2019 to August 2022 and were treated with Tacrolimus for the 1st-year postrenal transplant. Clinical exome sequencing (CES) data were analyzed for SNP coverage of the *CYP3A5* gene. CES is a high throughput sequencing of a panel of genes (exons and the flanking region). CES covers genes curated from databases such as OMIM, HGMD, and ClinVar, which are associated with the disease. Initially, we analyzed CES data of 232 patients, but 6 patients were excluded because TAC was not a part of their treatment, 2 patients were removed as they underwent retransplantation, and 5 more patients were excluded since the *CYP3A5* gene was not covered in CES. Therefore, 219 patients were included in the study for further analysis. Demographic details and follow-up data were retrieved from the case record forms of the patients. This study, EC/755/2021, was approved by the human ethics committee of Muljibhai Patel Society for Research in Nephro-Urology, Nadiad. Demographic details of patients, including their age, gender, and body weight at five different time points, day 7 and months 1, 3, 6, and 12 of post-transplantation, were recorded from the patients’ case record files.

These are all living donor transplants. TAC was started 3 days before transplantation. TAC daily doses (mg) and the associated *C*_0_ at day 7 and months 1, 3, 6, and 12 of post-transplantation, along with comorbidities and causes of native kidney disease (NKD), were retrieved from the medical report of the follow-up. The TAC *C*_0_ was obtained from a MAGLUMI FK-506 chemiluminescence immunoassay (CLIA). Routine TAC doses were recorded from prescriptions, and the weight-adjusted TAC dose was determined through routine TAC dose/weight (mg/kg/day). The ratio of the routine *C*_0_ in correspondence to the routine weight-adjusted TAC dose (ng/mL per mg/kg) was used to obtain the concentration dose ratio (*C*_0_/*D*). The starting TAC dosage administered to patients after thr seventh day of transplantation was approximately 0.12–0.15 mg/kg/day in two divided doses. Dose adjustments were done as per the desired TAC *C*_0_ of 5–15 ng/mL.

Biopsy-proven acute rejections and pathological event details were collected in the post-transplant biopsy. For biopsy-proven rejection, all biopsies were scored according to the Banff 2019 criteria for T-cell and antibody-mediated rejection. Only those biopsies that met the criteria for T-cell-mediated or antibody-mediated rejection were classified as biopsy-proven rejections.^15^ All morphological changes present in the biopsy apart from rejection were classified as other pathological events. These pathological events were thrombotic microangiopathy (TMA) and patchy cortical/tubular necrosis secondary to TMA, acute tubular injury (ATI) and mild to moderate focal ATI, mild to moderate capillaritis, mild to moderate focal capillaritis, suspicious tubulitis, mild tubulitis, mild glomerulitis, mild to moderate focal glomerulosclerosis, and membranous nephropathy. TAC-related nephrotoxicity was identified in those biopsies which displayed morphological features of TAC toxicity.^16^

### 2.2. DNA Isolation and Clinical Exome Sequencing

Genomic DNA was isolated from whole blood using a QIAamp DNA blood mini kit (Qiagen, Germany) and quantified using Qubit fluorometry (Thermo Fisher Scientific). For the library preparation, 200 ng of the Qubit-quantified DNA was fragmented into ∼350 bp inserts. The library was hybridized and enriched using a ∼16.5 Mb custom exome panel (Nimblegen, CA). The library was assessed for fragment size distribution using Tape Station (Agilent) and was quantified using Qubit (Thermo Fisher Scientific) and sequenced as 2 × 150 bp paired-end reads on an Illumina HiseqX (Illumina, CA) sequencer according to the manufacturer’s protocol. The library was sequenced to an average sequencing depth of ≥80–100×. Sequencing has been performed at MedGenome Laboratories Ltd., Bangalore, India.

### 2.3. Variant Calling and Annotation

Following quality check and adapter trimming using fastq-mcf (version 1.04.676), the sequencing reads obtained are aligned to the human reference genome (GRCh38.p13) using the BWA program. The aligned reads were sorted, duplicate reads were removed, and the variants were called using the GATK Best Practices pipeline using Sentieon (v201808.07). Gene annotation of the variants was performed using the VEP program against the Ensembl release 99 human gene model. The variants were annotated for the allele frequency using population databases GnomAD (v3.0) and 1000 Genomes and a MedGenome population-specific database. In addition to single nucleotide variants (SNVs) and small InDels, copy number variants (CNVs) are detected from targeted sequence data using the ExomeDepth (v1.1.10) method. The variants with a minor allele frequency of >5% in population databases were filtered out.

### 2.4. Statistical Analysis

SPSS (version 25.0) was used for all statistical analyses. R-Studio (version 4.2.0) with package dev tools, LD heatmap, was used to calculate the linkage disequilibrium (LD) plot. LD was not observed between studied genetic variants of the *CYP3A5* gene ([Figure S1](https://pubs.acs.org/doi/suppl/10.1021/acsptsci.4c00055/suppl_file/pt4c00055_si_001.pdf)). The mean and standard deviation were used as a summary measure for quantifiable parameters, while for qualitative variables, the frequency and percentage were also noted. Association of *CYP3A5* SNPs with the TAC *C*_0_/*D* [(ng/mL)/(mg/kg/day)] was performed using one-way ANOVA and posthoc analysis. Also, one-way ANOVA was applied to study the synergistic effect of rs15524, rs4646449, rs4646453, and rs776746 variants. The chi-square test was used to study the association between *CYP3A5* polymorphisms and biopsy-proven allograft rejection using MedCalc (version 20.007). Cox regression was performed to know the hazardous effect of SNPs on pathological events. The Hardy–Weinberg equilibrium (HWE) was calculated for each SNP and found to be 1. The global distribution variation of the *CYP3A5* polymorphisms was analyzed by comparing the allele frequencies of our population with the IndiGen database and the genome browser Phase 3 population allele frequency data.

## 3. Results

### 3.1. Characteristics of the Studied Renal Recipients

Of the 219 renal transplant recipients studied, 181 (82.6%) were male and 38 (17.4%) were female. The majority of native kidney disease (NKD) causes were undetermined with 129 (58.9%) patients, followed by diabetes kidney disease in 24 (10.96%) patients and chronic glomerulonephritis in 18 (8.22%) patients. Few patients had immunoglobulin A nephropathy 11 (5.02%) and autosomal dominant polycystic kidney disease in 3 (1.37%) patients ([Table 1](#tbl1)). Other patients included cases of postpartum cortical necrosis, congenital anomalies of the kidney and the urinary tract (CAKUT), hypertensive nephrosclerosis, chronic tubulointerstitial disease (CTID), granulomatous interstitial nephritis, focal segmental glomerulosclerosis (FSGS), membranous nephropathy, reflux nephropathy, and renal stone diseases ([Table 1](#tbl1)). The 191 (87.21%) patients had the comorbid condition of hypertension, followed by anemia in 109 (60.89%) patients, which was followed by metabolic bone disease in 48 (26.82%) patients and diabetes mellitus in 26 (14.53%) patients. Other patients had heart disease, mineral imbalance, infections, Alport’s syndrome, brain disorders, drug overdose, and surgeries ([Table 1](#tbl1)).

### Table 1. Demographic Characteristics of Renal Transplant Patientsa.

| study subject (n) | 219 |
| --- | --- |
| age in years (average ± SD) | 36.18 ± 12.31 |
| gender n (%) |   |
| male | 181 (82.6) |
| female | 38 (17.4) |
| native kidney disease n (%) |   |
| undetermined | 129 (58.9) |
| others | 34 (15.53) |
| diabetes kidney disease (DKD) | 24 (10.96) |
| chronic glomerulonephritis | 18 (8.22) |
| immunoglobulin A nephropathy | 11 (5.02) |
| autosomal dominant polycystic kidney disease | 3 (1.37) |
| comorbidities n (%) |   |
| hypertension | 191 (87.21) |
| anemia | 109 (60.89) |
| others | 90 (50.28) |
| metabolic bone disease | 48 (26.82) |
| diabetes mellitus | 26 (14.53) |
| no comorbidity | 9 (4.11) |

Table 1 Caption: an: Number of patients.

### 3.2. Dose Adjustment and the Level of Tacrolimus in Transplant Recipients

The details of the drug doses, concentrations, weight-adjusted doses, and concentration/weight-adjusted doses are given in [Table 2](#tbl2). The TAC dose and concentration were calculated on day 7 and months 1, 3, 6, and 12 of postrenal transplantation. The average dose of TAC during the 12-month follow-up was 4.77 ± 1.96 to 6.24 ± 1.53 mg/day. The average concentration and concentration/weight-adjusted dose (*C*_0_/*D*) of TAC in the 12-month follow-up were 7.76 ± 2.14 to 8.18 ± 2.18 ng/mL and 74.67 ± 23.84 to 130.11 ± 134.67 ng/mL per mg/kg/day, respectively.

### Table 2. Dose and Blood Concentration of Tacrolimus at Different Time Points in Renal Transplant Patientsb.

| follow-up | 7th day | 1st month | 3rd month | 6th month | 12th month |
| --- | --- | --- | --- | --- | --- |
| Tacrolimus dose (mg/day) | 6.24 ± 1.53 (219) | 6.17 ± 1.83 (208) | 5.8 ± 1.94 (189) | 5.35 ± 1.97 (148) | 4.77 ± 1.96 (114) |
| Tacrolimus concentration (ng/mL) | 7.88 ± 2.21 (219) | 7.99 ± 1.86 (207) | 8.16 ± 1.93 (188) | 8.18 ± 2.18 (145) | 7.76 ± 2.14 (109) |
| weight-adjusted Tacrolimus dose (mg/kg/day) | 0.11 ± 0.02 (219) | 0.11 ± 0.04 (208) | 0.1 ± 0.04 (189) | 0.09 ± 0.04 (148) | 0.08 ± 0.04 (114) |
| concentration/dosea ratio (C0/D) (ng/mL)/(mg/kg/day) | 74.67 ± 23.84 (219) | 82.78 ± 40.61 (207) | 94.81 ± 56.25 (188) | 107.09 ± 69.17 (145) | 130.11 ± 134.67 (109) |

Table 2 Caption: aWeight-adjusted dose. bAll results in the above table are represented as mean ± SD (n). SD: standard deviation, n: number of patients.

### 3.3. CYP3A5 Variants and Tacrolimus C0/D

Thirty-five *CYP3A5* variants were covered in the CES data of the studied patients. We have analyzed the correlation of SNPs with the TAC *C*_0_/*D* on day 7 and months 1, 3, 6, and 12 of post-transplantation ([Table 3](#tbl3)). The TAC *C*_0_/*D* was significantly higher at day 7 for *CYP3A5* SNP (rs776746) in recipients with the CC genotype (79.38 ± 21.35 ng/mL per mg/kg per day; *p* = 0.017) than that of recipients with the TC genotype (69.21 ± 22.55 ng/mL per mg/kg per day) and the TT genotype (73.61 ± 30.5 ng/mL per mg/kg per day). The TAC *C*_0_/*D* was also significantly higher at months 1, 3, 6, and 12 of post-transplantation for SNP (rs776746) in recipients with the CC genotype (*p* < 0.000) in comparison to recipients with the CT and TT genotypes ([Table 3](#tbl3)).

### Table 3. Association of CYP3A5 SNPs with the Tacrolimus C0/D at Five Time Points in Renal Transplant Patientsa.

|   |   | 7th day |  | 1st month |  | 3rd month |  | 6th month |  | 12th month |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Rs-ID | genotype | C0/D ± SD (n) | p-value | C0/D ± SD (n) | p-value | C0/D ± SD (n) | p-value | C0/D ± SD (n) | p-value | C0/D ± SD (n) | p-value |
| rs15524 | AA | 80.69 ± 24.57 (120) | >0.001 | 98.56 ± 42.53 (111) | >0.001 | 113.82 ± 54.92 (99) | >0.001 | 128.63 ± 69.07 (80) | >0.001 | 135.28 ± 61.87 (53) | >0.001 |
| AG | 69.87 ± 21.85 (74) |   | 68.83 ± 30.66 (72) |   | 71.76 ± 32.08 (67) |   | 81.06 ± 49.13 (51) |   | 84.89 ± 39.61 (43) |   |  |
| GG | 63.01 ± 18.01 (25) |   | 54.59 ± 24.75 (24) |   | 63.49 ± 26.34 (22) |   | 65.48 ± 33.47 (14) |   | 65.89 ± 26.57 (13) |   |  |
| rs4646449 | CC | 77.55 ± 24.27 (181) | 0.002 | 89 ± 41.20 (171) | >0.001 | 99.11 ± 51.88 (156) | 0.001 | 112.58 ± 66.83 (126) | 0.005 | 110.75 ± 58.90 (98) | 0.136 |
| CT | 62.95 ± 14.07 (23) |   | 55.80 ± 15.17 (22) |   | 63.46 ± 22.78 (20) |   | 61.95 ± 17.41 (12) |   | 81.82 ± 31.10 (05) |   |  |
| TT | 62.90 ± 22.07 (15) |   | 55.24 ± 31.90 (14) |   | 61.78 ± 27.43 (12) |   | 58.94 ± 17.87 (07) |   | 68.90 ± 31.91 (06) |   |  |
| rs4646453 | CC | 78.04 ± 24.53 (174) | 0.001 | 90.01 ± 41.50 (164) | >0.001 | 100.17 ± 52.30 (149) | >0.001 | 114.59 ± 67.65 (120) | 0.001 | 109.98 ± 58.45 (93) | 0.235 |
| CA | 63.40 ± 13.63 (30) |   | 57.71 ± 18.12 (29) |   | 66.62 ± 27.51 (27) |   | 66.35 ± 20.81 (17) |   | 103.45 ± 57.47 (10) |   |  |
| AA | 63.16 ± 21.85 (15) |   | 55.93 ± 31.60 (14) |   | 62.30 ± 27.19 (12) |   | 57.72 ± 16.90 (08) |   | 68.90 ± 31.91 (06) |   |  |
| rs776746 | TT | 73.61 ± 30.5 (37) | 0.017 | 69.61 ± 40.7 (35) | >0.001 | 78.09 ± 44.36 (33) | >0.001 | 77.91 ± 47.36 (21) | >0.001 | 84.33 ± 47.48 (20) | >0.001 |
| TC | 69.21 ± 22.55 (73) |   | 68.57 ± 30.81 (71) |   | 72.82 ± 32.59 (67) |   | 81.60 ± 49.47 (50) |   | 85.68 ± 39.75 (42) |   |  |
| CC | 79.38 ± 21.35 (109) |   | 98.15 ± 41.56 (101) |   | 113.81 ± 55.14 (88) |   | 130.06 ± 69.62 (74) |   | 135.98 ± 62.97 (47) |   |  |

Table 3 Caption: aC0/D: Concentration/weight-adjusted dose ratio (ng/mL)/(mg/kg/day), SD: standard deviation.

Another variant of *CYP3A5*, SNP (rs15524), was found to be associated with the TAC *C*_0_/*D*. The TAC *C*_0_/*D* was significantly higher at day 7 for SNP (rs15524) in recipients with the AA genotype (80.69 ± 24.57 ng/mL per mg/kg per day; *p* < 0.000) than that of recipients with the AG genotype (69.87 ± 21.85 ng/mL per mg/kg per day) and the GG genotype (63.01 ± 18.01 ng/mL per mg/kg per day). The TAC *C*_0_/*D* was also significantly higher at months 1, 3, 6, and 12 of post-transplantation for SNP (rs776746) in recipients with the AA genotype (*p* < 0.000) in comparison to recipients with the AG and GG genotypes ([Table 3](#tbl3)).

An additional variation of the *CYP3A5* gene, specifically SNP (rs4646449), was discovered to have a connection with the TAC *C*_0_/*D*. At day 7, recipients with the CC genotype displayed a significantly higher TAC *C*_0_/*D* level (77.55 ± 24.27 ng/mL per mg/kg per day; *p* = 0.002) compared to those with the CT genotype (62.95 ± 14.07 ng/mL per mg/kg per day) and the TT genotype (62.90 ± 22.07 ng/mL per mg/kg per day). Furthermore, recipients with the CC genotype for SNP (rs776746) demonstrated significantly elevated TAC *C*_0_/*D* levels at months 1, 3, and 6 following transplantation (*p* < 0.005) when compared to individuals with the CT and TT genotypes ([Table 3](#tbl3)). At month 12, recipients with the CC genotype have a higher TAC *C*_0_/*D* level (110.75 ± 58.90 ng/mL per mg/kg per day) compared to those with the CT genotype (81.82 ± 31.10 ng/mL per mg/kg per day) and the TT genotype (68.90 ± 31.91 ng/mL per mg/kg per day) but this difference was not significant ([Table 3](#tbl3)).

A different variant of the *CYP3A5* gene, known as SNP (rs4646453), was found to be associated with the TAC *C*_0_/*D*. On day 7, recipients with the CC genotype had notably higher TAC *C*_0_/*D* levels (78.04 ± 24.53 ng/mL per mg/kg per day; *p* = 0.001) compared to those with the CA genotype (63.40 ± 13.63 ng/mL per mg/kg per day) and the AA genotype (63.16 ± 21.85 ng/mL per mg/kg per day). Additionally, recipients with the CC genotype for SNP (rs4646453) showed significantly elevated TAC *C*_0_/*D* levels at months 1, 3, and 6 of post-transplantation (*p* < 0.001) when compared to individuals with the CA and AA genotypes. When we analyzed the *C*_0_/*D* at month 12, recipients with the CC genotype had a higher Tacrolimus *C*_0_/*D* level (109.98 ± 58.45 ng/mL per mg/kg per day) compared to those with the CA genotype (103.45 ± 57.47 ng/mL per mg/kg per day) and the AA genotype (68.90 ± 31.91 ng/mL per mg/kg per day) but this difference was not significant ([Table 3](#tbl3)).

We also analyzed the synergistic effect of four SNPs, rs15524, rs4646449, rs4646453, and rs776746 of *CYP3A5* on Tacrolimus *C*_0_/*D* at day 7 and months 1, 3, 6, and 12 of post-transplantation ([Table 4](#tbl4)). Combinations of all four SNPs were grouped as slow- and fast-metabolizing genotypes and analyzed for association with the corresponding mean of the TAC *C*_0_/*D*. The genotype combination of the slow metabolizer was found in 107 recipients, and the fast-metabolizing genotype was found in 38 recipients. On day 7, recipients with the slow metabolizer had higher TAC *C*_0_/*D* levels (79.70 ± 21.13 ng/mL per mg/kg per day; *p* < 0.000) compared to the fast metabolizer (62.93 ± 17.38 ng/mL per mg/kg per day). Slow-metabolizing recipients showed significantly elevated Tacrolimus *C*_0_/*D* levels at months 1, 3, 6, and 12 of post-transplantation (*p* < 0.003) when compared to fast metabolizers ([Table 4](#tbl4)). Multiple combinations of SNPs were also analyzed for their synergistic effect, and the data are summarized in [Table S1](https://pubs.acs.org/doi/suppl/10.1021/acsptsci.4c00055/suppl_file/pt4c00055_si_001.pdf).

### Table 4. Synergistic Effect of CYP3A5 SNPs on the Tacrolimus C0/D at Five Time Points in Renal Transplant Patientsa.

|   | 7th day |  | 1st month |  | 3rd month |  | 6th month |  | 12th month |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| metabolism | C0/D ± SD (n) | p-value | C0/D ± SD (n) | p-value | C0/D ± SD (n) | p-value | C0/D ± SD (n) | p-value | C0/D ± SD (n) | p-value |
| slowb | 79.70 ± 21.13 (107) | >0.001 | 98.27 ± 41.75 (100) | >0.001 | 113.81 ± 55.14 (88) | >0.001 | 130.06 ± 69.62 (74) | >0.001 | 135.98 ± 62.97 (47) | 0.003 |
| fastb | 62.93 ± 17.38 (38) | 55.58 ± 22.71 (36) | 62.83 ± 24.20 (32) | 60.84 ± 17.14 (19) | 74.78 ± 30.68 (11) |  |  |  |  |  |

Table 4 Caption: aC0/D: Concentration/Weight-adjusted dose ratio (ng/mL)/(mg/kg/day), SD: standard deviation. bSlow (AA, CC, CC, CC) and fast (AG, GG, CT, TT, CA, AA, TT, TC) genotypes of SNPs rs15524, rs4646449, rs4646453, and rs776746.

### 3.4. Association of CYP3A5 Polymorphisms with Biopsy-Proven Rejection and Pathological Events

Biopsy-proven acute rejections were observed in 9 (10.71%) individuals in 84 biopsies postrenal transplant ([Table S2](https://pubs.acs.org/doi/suppl/10.1021/acsptsci.4c00055/suppl_file/pt4c00055_si_001.pdf)). The CC genotype of rs776746 had significantly higher rejection rates [RR = 1.613; CI (0.94–2.75); *p* = 0.035] when compared with TT and TC genotypes ([Figure 1](#fig1)). We also analyzed the association of four SNPs with pathological events observed in the post-transplant biopsy. A total of 39 (46.42%) biopsies showed pathological events of 84 biopsies ([Table S2](https://pubs.acs.org/doi/suppl/10.1021/acsptsci.4c00055/suppl_file/pt4c00055_si_001.pdf)). Pathological events were recorded from biopsies. The fast metabolizer genotype (CA) of rs4646453 tends to develop histopathologically proven pathological events, which may initiate allograft rejection (HR = 7.258; 95% CI = 1.354–38.904) in comparison to the slow metabolizer genotype (CC), as illustrated in [Figure 2](#fig2). Other SNPs of *CYP3A5* and Cox regression analysis results are mentioned in [Table S3](https://pubs.acs.org/doi/suppl/10.1021/acsptsci.4c00055/suppl_file/pt4c00055_si_001.pdf).

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2665/11555502/e91a58edb564/pt4c00055_0001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=11555502_pt4c00055_0001.jpg)

Renal allograft rejection with the association of rs776746 in terms of relative risk (RR). The chi-square test was used to study the association between CYP3A5 polymorphisms and biopsy-proven allograft rejection. The CYP3A5*3/*3 “CC” genotype of the CYP3A5 variant rs776746 showed significant association with biopsy-proven allograft rejection with RR = 1.613 and CI (0.94–2.75) and p-value = 0.035. The CYP3A5*1/*1 + CYP3A5*1/*3 TT + TC genotype of the CYP3A5 variant rs776746 showed an insignificant association with biopsy-proven allograft rejection.

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2665/11555502/a451dac0b4d2/pt4c00055_0002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=11555502_pt4c00055_0002.jpg)

Impact of CYP3A5 SNPs on histopathologically proven pathological events in postrenal transplant patients. (A) Fast metabolizer genotype (CC) of rs4646453 tends to increase the risk of developing histopathologically proven pathological events by 7.258 times in comparison to the slow metabolizer (CC). (B) rs15524, slow metabolizer genotype (GG), and fast metabolizer genotypes (AG and AA), (C) rs4646449, slow metabolizer genotype (CC), and fast metabolizer genotypes (CT and TT) representing fast metabolizers, and (D) rs776746, slow metabolizer genotype (CC), and fast metabolizer genotypes (TT and CT) are not significantly associated with histopathologically proven pathological events.

### 3.5. Frequency Distribution of Genotypes and Alleles

All 35 SNPs of the *CYP3A5* genotype distribution in renal recipients are given in [Table S4](https://pubs.acs.org/doi/suppl/10.1021/acsptsci.4c00055/suppl_file/pt4c00055_si_001.pdf). On analyzing rs15524 polymorphisms, AA, AG, and GG genotypes were observed in 54.79, 33.79, and 11.42% of recipients, respectively, whereas A and G alleles were observed in 71.68 and 28.32% recipients, respectively. The analysis of rs4646449 polymorphisms revealed that CC, CT, and TT genotypes were observed in 82.65, 10.5, and 6.85% of recipients, respectively. The C and T alleles were observed in 87.89 and 12.11% of recipients, respectively. The rs4646453 polymorphism’s wild-type genotype CC, heterozygous GT and CA, and homozygous AA and alleles C and A frequencies were 79.45, 13.7, 6.85, 86.30, and 13.70% in recipients respectively. Moreover, in the rs776746 polymorphism, the frequencies of TT, TC, and CC genotypes were 16.89, 33.33, and 49.77%, and T and C alleles showed frequencies of 25.11 and 74.89%, respectively, in our cohort ([Table S4](https://pubs.acs.org/doi/suppl/10.1021/acsptsci.4c00055/suppl_file/pt4c00055_si_001.pdf)).

The allele frequency distribution of four associated *CYP3A5* polymorphisms in the current study was compared with the 1000 Genomes Project Phase 3 ([http://asia.ensembl.org/](http://asia.ensembl.org/)) and IndiGen databases ([https://indigen.igib.in/](https://indigen.igib.in/)) ([Figure 3](#fig3)). It was observed that the frequency of rs15524 in our study was similar to that of the Indigen and South Asian population and East African population of the 1000 Genomes Project. The frequency was lower compared to Europeans and Admixed Americans and higher compared to the African population of the 1000 Genomes Project, as shown in [Figure 3](#fig3)A. The frequency of rs46449 in our study was slightly higher compared to Indigen, South Asian, and East African populations of the 1000 Genomes Project. The frequency was lower compared to European, African, and Admixed American populations of 1000 Genomes Project, [Figure 3](#fig3)B. Further, the frequency of rs46453 in our study was similar to Indigen, South Asian, and East African populations of the 1000 Genomes Project. The frequency was lower compared to Europeans, Africans, and Admixed Americans, except for the PUR population of the 1000 Genomes Project, as shown in [Figure 3](#fig3)C. Moreover, the frequency of rs776746 in our study was similar to those of IndiGen, South Asian, and East African populations of the 1000 Genomes Project. The frequency was lower compared to Africans and higher compared to Europeans and the Admixed American population of the 1000 Genomes Project, as shown in [Figure 3](#fig3)D.

### Figure 3.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2665/11555502/5b5ff8d30b82/pt4c00055_0003.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=11555502_pt4c00055_0003.jpg)

Four SNPs of the CYP3A5 gene were significantly associated with the TAC dose response. The frequency distribution of these SNPs was compared with the population studied in the 1000 Genomes Project Phase 3 and Indigen. (A) Comparison of the allele frequency of rs15524, (B) comparison of the allele frequency of rs4646449, (C) comparison of the allele frequency of rs4646453, and (D) comparison of the allele frequency of rs776746.

## 4. Discussion

Therapeutic drug monitoring (TDM) has become a crucial and integral technique for Tacrolimus dosing to manage drug blood levels. Although doses of TAC prescribed as per TDM can only be applied after the patient’s exposure to TAC, the initial TAC dosing strategy needs to be defined based on the predictive biomarker. In the present study, we studied the association of *CYP3A5* variants with the TAC dose and concentration in renal transplant patients.^17^

We analyzed the influence of all 35 SNPs of *CYP3A5* involved in the metabolism of TAC to the concentration/weight-adjusted dose (*C*_0_/*D*) at day 7 and months 1, 3, 6, and 12 of post-transplantation. We further confirmed that *CYP3A5*3/*3* “CC” (rs776746) polymorphism was a statistically significant (*p* < 0.05) biomarker for the determination of the TAC *C*_0_/*D* from previously conducted studies.^18−21^ The diverse distribution of the *CYP3A5* gene was found to be an attributed factor for varied TAC bioavailability among patients. Individuals with functional or wild-type alleles (*1, ‘T’) are classified as *CYP3A5* expressers, showing increased Tacrolimus metabolism and clearance. Carriers with *3/*3 (CC) alleles (homozygous) are known as *CYP3A5* nonexpressers with the suppressed metabolic ability of the enzyme.^14^ An equation was developed using multilinear regression, including *CYP3A4* and ABCB1 variants, sex, and age, to help physicians decide initial TAC doses per body weight (kg). This genotype-guided Tacrolimus dosing equation may help calculate personalized doses for each patient, avoiding graft rejection due to underconcentration and toxicity due to overconcentration of Tacrolimus.^17^

Another SNP of *CYP3A5* (rs15524) was found to be associated with the TAC *C*_0_/*D* at day 7 and months 1, 3, 6, and 12 of post-transplantation. AA (wild-allele) genotype carriers of rs15524 showed an increased dose-adjusted *C*_0_/*D* ratio compared to those of AG and GG genotypes. AG and GG genotypes showed an increased metabolic activity, resulting in enhanced TAC metabolism and clearance. Higher doses of TAC were prescribed to AG and GG carriers due to the rapid elimination of the drug from the body.^8,22,23^ An association was found between *CYP3A5* (rs4646453) and the TAC *C*_0_/*D* at day 7 and months 1, 3, 6, and 12 of post-transplantation. CC (wild-type) genotype carriers were found to express comparatively elevated TAC *C*_0_/*D* values in comparison to CA and AA genotype carriers. CA and AA increased the enzyme expression, potentiating TAC metabolism and elimination.^23^ Both rs15524 and rs4646453 showed an association with an initial post-transplantation variation in TAC deposition and concentration.^24^ We also found that a novel variant of *CYP3A5* (rs4646449) associated with the TAC *C*_0_/*D* at day 7 and months 1, 3, and 6 of post-transplantation was also found, indicating that it affects the Tacrolimus *C*_0_/*D*. CC (wild-type) genotype carriers of rs4646449 showed an increased dose-adjusted *C*_0_/*D* ratio compared to CT and TT genotypes. CT and TT genotypes showed increased metabolic actions of the *CYP3A5* protein, resulting in enhanced TAC metabolism and clearance. Higher doses of TAC were prescribed to CT and TT carriers due to the rapid elimination of the drug from the body.

Many efforts have been made to optimize the TAC therapy as per the patient’s genotype. A substantial influence of the *CYP3A5* polymorphism early after the first Tacrolimus dosing among the patients from multiethnic populations was observed.^6^ When compared to the conventional method, body weight-based TAC dosing, the French Tactique trial demonstrated that *CYP3A5*-guided initial TAC dosing resulted in considerably more patients achieving the targeted concentration within 3 days after the start of treatment.^25^ As per the Clinical Pharmacogenetics Implementation Consortium (CPIC), patients with *CYP3A5* moderate (*1/*3 or CC genotype) or fast metabolizers (homozygous *1/*1 or TT genotype) are recommended to be prescribed a higher initial dosing by 1.5–2 times. However, the total initial dose should not exceed 0.3 mg/kg/day. They also suggested that TDM should be implemented for any dose modifications.^26^ A new classification and regression tree model was established in 2020 to determine the initial dose of TAC in Chinese kidney transplant patients as per the *CYP3A5* genotype and hemoglobin levels.^27^

The synergistic impact of genetic variations can potentially influence protein complexes by modifying the protein function.^28^ Moreover, the cumulative effect of genetic variants may lead to fluctuations in the levels of critical biomolecules, either surpassing or falling below a specific threshold.^29^ Consequently, we conducted an analysis to assess the combined impact of SNPs (rs15524, rs4646449, rs4646453, and rs776746) in the *CYP3A5* gene on TAC *C*_0_/*D* levels at various time points of post-transplantation, including day 7 and months 1, 3, 6, and 12. We compared combinations of all four SNPs, defined as slow- and fast-metabolizing genotype combinations with their respective TAC *C*_0_/*D*. The synergistic effect of slow genotypes in transplant patients demonstrated an association with reduced TAC pharmacokinetics at all-time intervals. This indicates that the slow metabolizers should be initialized with a lower dose of TAC. A previous study showed the combined effect of *CYP3A5* and *MDR*1 variants on the TAC daily dose requirement.^10^

We found an increased risk of biopsy-proven acute allograft rejections in renal recipients with a homozygous variant (CC) of rs776746. The CC genotype of rs776746 denotes a slow metabolizer, as it shows a higher TAC *C*_0_/*D*, which indicates that a lower dose is required to maintain the necessary TAC concentration in blood. Our study findings are contrary to previous studies. In previous studies, there was an increased risk of biopsy-proven acute allograft rejection during the first-month post-transplantation in patients with *CYP3A5*1/*1* (TT, fast metabolizers) in comparison to *CYP3A5*3/*3* (CC, slow metabolizers).^30,31^ However, in our study, we are reporting the *CYP3A5*3/*3* (CC) genotype as a risk for biopsy-proven allograft rejection. This contrary finding may be attributed a small sample size of biopsy-proven rejections. In addition, we have also observed that most of the recipients with the CC genotype had a lower *C*_0_/*D* during the rejection event than the average *C*_0_/*D* expected. Possibly, the clinician may have modified the TAC dose, which may have attained the lower concentration and caused the rejection. Further, in our studied recipients, the rejection rates were very low (<5%); hence, it is difficult to attribute it to the Tacrolimus *C*_0_/*D* alone for rejections. These findings remarkably highlight the significance of adjusting TAC daily doses during the initial post-transplantation period, when there is a higher chance of rejection episodes occurring. The study showed that there was no significant difference observed in glomerular filtration rates or graft functioning or patient survival over a median 37-month follow-up period.^32^ Cardiovascular risk due to new-onset diabetes mellitus after transplantation (NODAT) is also correlated with decreased patient and allograft survival rates.^33−35^ Many studies have compared the incidence of NODAT in Cyclosporine A to that in TAC-treated patients. It was found that TAC has more consistently been shown to have a more significant diabetogenic effect.^36,37^

Biopsy-proven pathological events and TAC-related nephrotoxicity events were recorded. In our analysis, we found that the presence of the fast metabolizer genotype “CA” of rs4646453 tends to be 7.258 times riskier in developing the histopathological proven pathological events, which may initiate allograft rejection in comparison to the slow metabolizer genotype “CC”. A study found that the presence of *CYP3A5*1* was linked to a higher incidence of early T-cell-mediated rejection due to lower TAC trough levels.^31^ In another study, it was found that acute rejection episodes were significantly higher in patients with the *CYP3A5*1/*1* (TT) genotype (40%) than in patients with *CYP3A5*1/*3* (TC) (20%) or *CYP3A5*3/*3* (CC) (13%) genotypes.^38^ Nevertheless, variations in the genetic factors associated with this discrepancy might be attributed to differences in the ethnic backgrounds of the populations under study. Employing genotype-dependent adjustments for TAC concentrations could potentially result in fewer adverse events, all while maintaining effective immunosuppression. It is important to acknowledge that the present study has limitations due to the relatively small number of biopsies conducted to substantiate this observation.

When comparing allele frequencies of *CYP3A5* genetic variants of our samples with the 1000 Genomes Project Phase 3 populations pertaining to distinct ethnicities, we found significant interpopulation variations, indicating that our sample characteristics are varied in terms of the allele frequency distribution with some populations while being similar to other populations. However, the frequency distribution of studied *CYP3A5* SNPs was similar to the frequency of *CYP3A5* SNPs in the IndiGen database. The IndiGen database has the SNP frequency data of the Indian population. The variation in the frequency of SNPs emphasizes the need to develop population-specific dose prediction algorithms based on these gene polymorphisms to draw clinically relevant results, which is highlighted by these interethnic disparities in allele frequencies of drug metabolizer and transporter genes. While conducting clinical trials, it is also important to consider the pharmacogenomic analysis of the people involved in trials.

In conclusion, two SNPs of *CYP3A5*, i.e., rs776746 and rs15524, are strongly associated with the variation in the TAC dose response. In addition, rs4646453 and rs4646449 were also found to affect initial TAC *C*_0_/*D* values. These four variants of *CYP3A5* were also synergistically associated with the TAC dose determination. In addition, rs776746 and rs4646453 were associated with allograft rejection and pathological events, respectively. Genotype-based starting TAC doses might help transplant patients attain targeted TAC concentrations, avoiding toxicities from overconcentration and rejection from underconcentration. Our outcomes will allow physicians to design a personalized immunosuppressive regimen based on the patient’s genotype.

## Acknowledgments

We would like to acknowledge Muljibhai Patel Urological Hospital for intramural funds. The study was partially supported by fund from the Indian Council of Medical Research (5/4/7-4/Nephro/2020-NCD-II). Ramanbhai Patel College of Pharmacy, CHARUSAT for PhD scholars’ fellowship (CPSF).

## Glossary

## Supporting Information Available

The Supporting Information is available free of charge at [https://pubs.acs.org/doi/10.1021/acsptsci.4c00055](https://pubs.acs.org/doi/10.1021/acsptsci.4c00055?goto=supporting-info).

## Author Contributions

^∇^ R.J.H. and P.M.C. contributed equally to this work and shared first authorship. R.J.H.: data collection, data analysis, paper writing, and editing. P.M.C.: data collection, data handling and originally writing and editing the paper. R.S.: conceptualizing the project, paper review, and editing. N.B.S.: statistical analysis, paper review, and editing. A.S.J.: histopathological analysis. S.M.S.: histopathological analysis. S.D.G.: patient recruitment, paper review, and editing. A.M.K.: patient recruitment, paper review, and editing. U.N.H.: patient recruitment, paper review, and editing. H.B.P.: patient recruitment, paper review, and editing B.N.M.: supervising the Tacrolimus pharmacokinetics. M.A.R.: paper review and editing. S.N.P.: conceptualizing and supervising the project, leading the investigation, paper writing, review, and editing.

The authors declare no competing financial interest.

## Supplementary Material

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Coto E.; Tavira B.; Suárez-Álvarez B.; López-Larrea C.; Díaz-Corte C.; Ortega F.; Álvarez V. Pharmacogenetics of tacrolimus: Ready for clinical translation. Kidney Int. Suppl. 2011, 1, 58–62. 10.1038/kisup.2011.14.  [DOI](https://doi.org/10.1038/kisup.2011.14) | [PMC free article](/articles/PMC4089617/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25028625/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Kidney%20Int.%20Suppl.&title=Pharmacogenetics%20of%20tacrolimus:%20Ready%20for%20clinical%20translation&author=E.%20Coto&author=B.%20Tavira&author=B.%20Su%C3%A1rez-%C3%81lvarez&author=C.%20L%C3%B3pez-Larrea&author=C.%20D%C3%ADaz-Corte&volume=1&publication_year=2011&pages=58-62&pmid=25028625&doi=10.1038/kisup.2011.14&)

2. Brunet M.; Van Gelder T.; Åsberg A.; Haufroid V.; Hesselink D. A.; Langman L.; Lemaitre F.; Marquet P.; Seger C.; Shipkova M.; Vinks A.; Wallemacq P.; Wieland E.; Woillard J. B.; Barten M. J.; Budde K.; Colom H.; Dieterlen M. T.; Elens L.; Johnson-Davis K. L.; Kunicki P. K.; Macphee I.; Masuda S.; Mathew B. S.; Millán O.; Mizuno T.; Moes D. J. A. R.; Monchaud C.; Noceti O.; Pawinski T.; Picard N.; Van Schaik R.; Sommerer C.; Vethe N. T.; De Winter B.; Christians U.; Bergan S. Therapeutic Drug Monitoring of Tacrolimus-Personalized Therapy: Second Consensus Report. Ther. Drug Monit. 2019, 41, 261–307. 10.1097/FTD.0000000000000640.  [DOI](https://doi.org/10.1097/FTD.0000000000000640) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31045868/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ther.%20Drug%20Monit.&title=Therapeutic%20Drug%20Monitoring%20of%20Tacrolimus-Personalized%20Therapy:%20Second%20Consensus%20Report&author=M.%20Brunet&author=T.%20Van%20Gelder&author=A.%20%C3%85sberg&author=V.%20Haufroid&author=D.%20A.%20Hesselink&volume=41&publication_year=2019&pages=261-307&pmid=31045868&doi=10.1097/FTD.0000000000000640&)

3. Ben-Fredj N.; Hannachi I.; Chadli Z.; Ben-Romdhane H.; A Boughattas N.; Ben-Fadhel N.; Aouam K. Dosing algorithm for Tacrolimus in Tunisian Kidney transplant patients: Effect of CYP 3A4*1B and CYP3A4*22 polymorphisms. Toxicol. Appl. Pharmacol. 2020, 407, 115245. 10.1016/j.taap.2020.115245.  [DOI](https://doi.org/10.1016/j.taap.2020.115245) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32949580/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Toxicol.%20Appl.%20Pharmacol.&title=Dosing%20algorithm%20for%20Tacrolimus%20in%20Tunisian%20Kidney%20transplant%20patients:%20Effect%20of%20CYP%203A4*1B%20and%20CYP3A4*22%20polymorphisms&author=N.%20Ben-Fredj&author=I.%20Hannachi&author=Z.%20Chadli&author=H.%20Ben-Romdhane&author=N.%20A%20Boughattas&volume=407&publication_year=2020&pages=115245&pmid=32949580&doi=10.1016/j.taap.2020.115245&)

4. Sakaeda T.; Nakamura T.; Okumura K. Pharmacogenetics of MDR1 and its impact on the pharmacokinetics and pharmacodynamics of drugs. Pharmacogenomics 2003, 4, 397–410. 10.1517/phgs.4.4.397.22747.  [DOI](https://doi.org/10.1517/phgs.4.4.397.22747) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12831320/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Pharmacogenetics%20of%20MDR1%20and%20its%20impact%20on%20the%20pharmacokinetics%20and%20pharmacodynamics%20of%20drugs&author=T.%20Sakaeda&author=T.%20Nakamura&author=K.%20Okumura&volume=4&publication_year=2003&pages=397-410&pmid=12831320&doi=10.1517/phgs.4.4.397.22747&)

5. De Jonge H.; De Loor H.; Verbeke K.; Vanrenterghem Y.; Kuypers D. R. In vivo CYP3A4 activity, CYP3A5 genotype, and hematocrit predict tacrolimus dose requirements and clearance in renal transplant patients. Clin. Pharmacol. Ther. 2012, 92, 366–375. 10.1038/clpt.2012.109.  [DOI](https://doi.org/10.1038/clpt.2012.109) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22871995/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=In%20vivo%20CYP3A4%20activity,%20CYP3A5%20genotype,%20and%20hematocrit%20predict%20tacrolimus%20dose%20requirements%20and%20clearance%20in%20renal%20transplant%20patients&author=H.%20De%20Jonge&author=H.%20De%20Loor&author=K.%20Verbeke&author=Y.%20Vanrenterghem&author=D.%20R.%20Kuypers&volume=92&publication_year=2012&pages=366-375&pmid=22871995&doi=10.1038/clpt.2012.109&)

6. Haufroid V.; Wallemacq P.; VanKerckhove V.; Elens L.; De Meyer M.; Eddour D. C.; Malaise J.; Lison D.; Mourad M. CYP3A5 and ABCB1 polymorphisms and tacrolimus pharmacokinetics in renal transplant candidates: Guidelines from an experimental study. Am. J. Transplant. 2006, 6, 2706–2713. 10.1111/j.1600-6143.2006.01518.x.  [DOI](https://doi.org/10.1111/j.1600-6143.2006.01518.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17049058/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Transplant.&title=CYP3A5%20and%20ABCB1%20polymorphisms%20and%20tacrolimus%20pharmacokinetics%20in%20renal%20transplant%20candidates:%20Guidelines%20from%20an%20experimental%20study&author=V.%20Haufroid&author=P.%20Wallemacq&author=V.%20VanKerckhove&author=L.%20Elens&author=M.%20De%20Meyer&volume=6&publication_year=2006&pages=2706-2713&pmid=17049058&doi=10.1111/j.1600-6143.2006.01518.x&)

7. Mourad M.; Wallemacq P.; De Meyer M.; Brandt D.; Van Kerkhove V.; Malaise J.; Eddour D. C.; Lison D.; Haufroid V. The influence of genetic polymorphisms of cytochrome P450 3A5 and ABCB1 on starting dose- and weight-standardized tacrolimus trough concentrations after kidney transplantation in relation to renal function. Clin. Chem. Lab. Med. 2006, 44, 1192–1198. 10.1515/CCLM.2006.229.  [DOI](https://doi.org/10.1515/CCLM.2006.229) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17032130/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Chem.%20Lab.%20Med.&title=The%20influence%20of%20genetic%20polymorphisms%20of%20cytochrome%20P450%203A5%20and%20ABCB1%20on%20starting%20dose-%20and%20weight-standardized%20tacrolimus%20trough%20concentrations%20after%20kidney%20transplantation%20in%20relation%20to%20renal%20function&author=M.%20Mourad&author=P.%20Wallemacq&author=M.%20De%20Meyer&author=D.%20Brandt&author=V.%20Van%20Kerkhove&volume=44&publication_year=2006&pages=1192-1198&pmid=17032130&doi=10.1515/CCLM.2006.229&)

8. Tamashiro E. Y.; Felipe C. R.; Genvigir F. D. V.; Rodrigues A. C.; Campos A. B.; Hirata R. D. C.; Jr H. T.; Medina-pestana J. O. Influence of CYP3A4 and CYP3A5 polymorphisms on tacrolimus and sirolimus exposure in stable kidney transplant recipients. Drug Metab. Pers. Ther. 2017, 32, 89–95. 10.1515/dmpt-2016-0036.  [DOI](https://doi.org/10.1515/dmpt-2016-0036) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28593920/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab.%20Pers.%20Ther.&title=Influence%20of%20CYP3A4%20and%20CYP3A5%20polymorphisms%20on%20tacrolimus%20and%20sirolimus%20exposure%20in%20stable%20kidney%20transplant%20recipients&author=E.%20Y.%20Tamashiro&author=C.%20R.%20Felipe&author=F.%20D.%20V.%20Genvigir&author=A.%20C.%20Rodrigues&author=A.%20B.%20Campos&volume=32&publication_year=2017&pages=89-95&pmid=28593920&doi=10.1515/dmpt-2016-0036&)

9. Woillard J.-B.; Mourad M.; Neely M.; Capron A.; van Schaik R. H.; Schaik R. H.; van Gelder T.; Van; Lloberas N.; Gelder T.; Hesselink D. A.; Van; Marquet P.; Lloberas N.; Haufroid V.; Hesselink D. A. Tacrolimus Updated Guidelines through popPK Modeling: How to Benefit More from CYP3A Pre-emptive Genotyping Prior to Kidney Transplantation. Front. Pharmacol. 2017, 8, 358. 10.3389/fphar.2017.00358.  [DOI](https://doi.org/10.3389/fphar.2017.00358) | [PMC free article](/articles/PMC5462973/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28642710/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front.%20Pharmacol.&title=Tacrolimus%20Updated%20Guidelines%20through%20popPK%20Modeling:%20How%20to%20Benefit%20More%20from%20CYP3A%20Pre-emptive%20Genotyping%20Prior%20to%20Kidney%20Transplantation&author=J.-B.%20Woillard&author=M.%20Mourad&author=M.%20Neely&author=A.%20Capron&author=R.%20H.%20van%20Schaik&volume=8&publication_year=2017&pages=358&pmid=28642710&doi=10.3389/fphar.2017.00358&)

10. Prasad N.; Jaiswal A.; Behera M. R.; Agarwal V.; Kushwaha R.; Bhadauria D.; Kaul A.; Gupta A. Melding Pharmacogenomic Effect of MDR-1 and CYP3A5 Gene Polymorphism on Tacrolimus Dosing in Renal Transplant Recipients in Northern India. Kidney Int. Rep. 2020, 5, 28–38. 10.1016/j.ekir.2019.09.013.  [DOI](https://doi.org/10.1016/j.ekir.2019.09.013) | [PMC free article](/articles/PMC6943758/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31922058/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Kidney%20Int.%20Rep.&title=Melding%20Pharmacogenomic%20Effect%20of%20MDR-1%20and%20CYP3A5%20Gene%20Polymorphism%20on%20Tacrolimus%20Dosing%20in%20Renal%20Transplant%20Recipients%20in%20Northern%20India&author=N.%20Prasad&author=A.%20Jaiswal&author=M.%20R.%20Behera&author=V.%20Agarwal&author=R.%20Kushwaha&volume=5&publication_year=2020&pages=28-38&pmid=31922058&doi=10.1016/j.ekir.2019.09.013&)

11. Oetting W. S.; Wu B.; Schladt D. P.; Guan W.; Remmel R. P.; Mannon R. B.; Matas A. J.; Israni A. K.; Jacobson P. A. Genome-wide association study identifies the common variants in CYP3A4 and CYP3A5 responsible for variation in tacrolimus trough concentration in Caucasian kidney transplant recipients. Pharmacogenomics J. 2018, 18, 501–505. 10.1038/tpj.2017.49.  [DOI](https://doi.org/10.1038/tpj.2017.49) | [PMC free article](/articles/PMC8513493/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29160300/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J.&title=Genome-wide%20association%20study%20identifies%20the%20common%20variants%20in%20CYP3A4%20and%20CYP3A5%20responsible%20for%20variation%20in%20tacrolimus%20trough%20concentration%20in%20Caucasian%20kidney%20transplant%20recipients&author=W.%20S.%20Oetting&author=B.%20Wu&author=D.%20P.%20Schladt&author=W.%20Guan&author=R.%20P.%20Remmel&volume=18&publication_year=2018&pages=501-505&pmid=29160300&doi=10.1038/tpj.2017.49&)

12. Sohn M.; Kim M. G.; Han N.; Kim I. W.; Gim J.; Min S. Il.; Song E. Y.; Kim Y. S.; Jung H. S.; Shin Y. K.; Ha J.; Oh J. M. Whole exome sequencing for the identification of CYP3A7 variants associated with tacrolimus concentrations in kidney transplant patients. Sci. Rep. 2018, 8, 18064 10.1038/s41598-018-36085-w.  [DOI](https://doi.org/10.1038/s41598-018-36085-w) | [PMC free article](/articles/PMC6305386/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30584253/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&title=Whole%20exome%20sequencing%20for%20the%20identification%20of%20CYP3A7%20variants%20associated%20with%20tacrolimus%20concentrations%20in%20kidney%20transplant%20patients&author=M.%20Sohn&author=M.%20G.%20Kim&author=N.%20Han&author=I.%20W.%20Kim&author=J.%20Gim&volume=8&publication_year=2018&pages=18064&pmid=30584253&doi=10.1038/s41598-018-36085-w&)

13. Boughton O.; Borgulya G.; Cecconi M.; Fredericks S.; Moreton-Clack M.; Macphee I. A. M. A published pharmacogenetic algorithm was poorly predictive of tacrolimus clearance in an independent cohort of renal transplant recipients. Br. J. Clin. Pharmacol. 2013, 76, 425–431. 10.1111/bcp.12076.  [DOI](https://doi.org/10.1111/bcp.12076) | [PMC free article](/articles/PMC3769669/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23305195/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Clin.%20Pharmacol.&title=A%20published%20pharmacogenetic%20algorithm%20was%20poorly%20predictive%20of%20tacrolimus%20clearance%20in%20an%20independent%20cohort%20of%20renal%20transplant%20recipients&author=O.%20Boughton&author=G.%20Borgulya&author=M.%20Cecconi&author=S.%20Fredericks&author=M.%20Moreton-Clack&volume=76&publication_year=2013&pages=425-431&pmid=23305195&doi=10.1111/bcp.12076&)

14. Chen L.; Prasad G. V. R. CYP3A5 polymorphisms in renal transplant recipients: Influence on tacrolimus treatment. Pharmgenomics. Pers. Med. 2018, 11, 23–33. 10.2147/PGPM.S107710.  [DOI](https://doi.org/10.2147/PGPM.S107710) | [PMC free article](/articles/PMC5846312/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29563827/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmgenomics.%20Pers.%20Med.&title=CYP3A5%20polymorphisms%20in%20renal%20transplant%20recipients:%20Influence%20on%20tacrolimus%20treatment&author=L.%20Chen&author=G.%20V.%20R.%20Prasad&volume=11&publication_year=2018&pages=23-33&pmid=29563827&doi=10.2147/PGPM.S107710&)

15. Loupy A.; Haas M.; Roufosse C.; Naesens M.; Adam B.; Afrouzian M.; Akalin E.; Alachkar N.; Bagnasco S.; Becker J. U.; Cornell L. D.; Clahsen-van Groningen M. C.; Demetris A. J.; Dragun D.; Duong van Huyen J. P.; Farris A. B.; Fogo A. B.; Gibson I. W.; Glotz D.; Gueguen J.; Kikic Z.; Kozakowski N.; Kraus E.; Lefaucheur C.; Liapis H.; Mannon R. B.; Montgomery R. A.; Nankivell B. J.; Nickeleit V.; Nickerson P.; Rabant M.; Racusen L.; Randhawa P.; Robin B.; Rosales I. A.; Sapir-Pichhadze R.; Schinstock C. A.; Seron D.; Singh H. K.; Smith R. N.; Stegall M. D.; Zeevi A.; Solez K.; Colvin R. B.; Mengel M. The Banff 2019 Kidney Meeting Report (I): Updates on and clarification of criteria for T cell– and antibody-mediated rejection. Am. J. Transplant. 2020, 20, 2318–2331. 10.1111/ajt.15898.  [DOI](https://doi.org/10.1111/ajt.15898) | [PMC free article](/articles/PMC7496245/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32463180/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Transplant.&title=The%20Banff%202019%20Kidney%20Meeting%20Report%20(I):%20Updates%20on%20and%20clarification%20of%20criteria%20for%20T%20cell%E2%80%93%20and%20antibody-mediated%20rejection&author=A.%20Loupy&author=M.%20Haas&author=C.%20Roufosse&author=M.%20Naesens&author=B.%20Adam&volume=20&publication_year=2020&pages=2318-2331&pmid=32463180&doi=10.1111/ajt.15898&)

16. Jennette J. C.; D’Agati V. D.. Heptinstall’s Pathology of the Kidney, 8th ed.; Lippincott Williams & Wilkins, 2023.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Heptinstall%E2%80%99s%20Pathology%20of%20the%20Kidney&publication_year=2023&)

17. Srinivas L.; Gracious N.; Nair R. R. Pharmacogenetics Based Dose Prediction Model for Initial Tacrolimus Dosing in Renal Transplant Recipients. Front. Pharmacol. 2021, 12, 726784. 10.3389/fphar.2021.726784.  [DOI](https://doi.org/10.3389/fphar.2021.726784) | [PMC free article](/articles/PMC8669916/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34916931/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front.%20Pharmacol.&title=Pharmacogenetics%20Based%20Dose%20Prediction%20Model%20for%20Initial%20Tacrolimus%20Dosing%20in%20Renal%20Transplant%20Recipients&author=L.%20Srinivas&author=N.%20Gracious&author=R.%20R.%20Nair&volume=12&publication_year=2021&pages=726784&pmid=34916931&doi=10.3389/fphar.2021.726784&)

18. Li C.-J.; Li L.; Lin L.; Jiang H.; Zhong Z.; Li W.; Zhang Y.; Zheng P.; Tan X.; Zhou L. Impact of the CYP3A5, CYP3A4, COMT, IL-10 and POR Genetic Polymorphisms on Tacrolimus Metabolism in Chinese Renal Transplant Recipients. PLoS One 2014, 9, e86206. 10.1371/journal.pone.0086206.  [DOI](https://doi.org/10.1371/journal.pone.0086206) | [PMC free article](/articles/PMC3897654/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24465960/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Impact%20of%20the%20CYP3A5,%20CYP3A4,%20COMT,%20IL-10%20and%20POR%20Genetic%20Polymorphisms%20on%20Tacrolimus%20Metabolism%20in%20Chinese%20Renal%20Transplant%20Recipients&author=C.-J.%20Li&author=L.%20Li&author=L.%20Lin&author=H.%20Jiang&author=Z.%20Zhong&volume=9&publication_year=2014&pages=e86206&pmid=24465960&doi=10.1371/journal.pone.0086206&)

19. Khan A. R.; Ali K.; et al. CYP3A5 gene polymorphisms and their impact on dosage and trough concentration of tacrolimus among kidney transplant patients: a systematic review and meta-analysis. Pharmacogenomics J. 2020, 20, 553–562. 10.1038/s41397-019-0144-7.  [DOI](https://doi.org/10.1038/s41397-019-0144-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31902947/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J.&title=CYP3A5%20gene%20polymorphisms%20and%20their%20impact%20on%20dosage%20and%20trough%20concentration%20of%20tacrolimus%20among%20kidney%20transplant%20patients:%20a%20systematic%20review%20and%20meta-analysis&author=A.%20R.%20Khan&author=K.%20Ali&volume=20&publication_year=2020&pages=553-562&pmid=31902947&doi=10.1038/s41397-019-0144-7&)

20. Mendrinou E.; Mashaly M. E.; Al Okily A. M.; Mohamed M. E.; Refaie A. F.; Elsawy E. M.; Saleh H. H.; Sheashaa H.; Patrinos G. P. CYP3A5 Gene-Guided Tacrolimus Treatment of Living-Donor Egyptian Kidney Transplanted Patients. Front. Pharmacol. 2020, 11, 1218. 10.3389/fphar.2020.01218.  [DOI](https://doi.org/10.3389/fphar.2020.01218) | [PMC free article](/articles/PMC7431691/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32848803/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front.%20Pharmacol.&title=CYP3A5%20Gene-Guided%20Tacrolimus%20Treatment%20of%20Living-Donor%20Egyptian%20Kidney%20Transplanted%20Patients&author=E.%20Mendrinou&author=M.%20E.%20Mashaly&author=A.%20M.%20Al%20Okily&author=M.%20E.%20Mohamed&author=A.%20F.%20Refaie&volume=11&publication_year=2020&pages=1218&pmid=32848803&doi=10.3389/fphar.2020.01218&)

21. Hannachi I.; Chadli Z.; Kerkeni E.; Kolsi A.; Hammouda M.; Chaabane A.; Ben Fredj N.; Touitou Y.; Boughattas N. A.; Aouam K. Influence of CYP3A polymorphisms on tacrolimus pharmacokinetics in kidney transplant recipients. Pharmacogenomics J. 2021, 21, 69–77. 10.1038/s41397-020-00179-4.  [DOI](https://doi.org/10.1038/s41397-020-00179-4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32843687/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J.&title=Influence%20of%20CYP3A%20polymorphisms%20on%20tacrolimus%20pharmacokinetics%20in%20kidney%20transplant%20recipients&author=I.%20Hannachi&author=Z.%20Chadli&author=E.%20Kerkeni&author=A.%20Kolsi&author=M.%20Hammouda&volume=21&publication_year=2021&pages=69-77&pmid=32843687&doi=10.1038/s41397-020-00179-4&)

22. Maseko N.; Yang S.; Li C.; Zhang S.; Wang R.; Zhang Y.; Li C.; Zhang C.; Li L. Impact of genetic polymorphisms on tacrolimus trough blood concentration in Chinese liver transplant recipients. Pharmacogenomics 2023, 24, 207–217. 10.2217/pgs-2022-0180.  [DOI](https://doi.org/10.2217/pgs-2022-0180) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36927114/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Impact%20of%20genetic%20polymorphisms%20on%20tacrolimus%20trough%20blood%20concentration%20in%20Chinese%20liver%20transplant%20recipients&author=N.%20Maseko&author=S.%20Yang&author=C.%20Li&author=S.%20Zhang&author=R.%20Wang&volume=24&publication_year=2023&pages=207-217&pmid=36927114&doi=10.2217/pgs-2022-0180&)

23. Liu J.; Feng D.; Kan X.; Zheng M.; Zhang X.; Wang Z.; Sun L.; Chen H.; Gao X.; Lu T.; Gu M.; Tan R.; Han Z. Polymorphisms in the CYP3A5 gene significantly affect the pharmacokinetics of sirolimus after kidney transplantation. Pharmacogenomics 2021, 22, 903–912. 10.2217/pgs-2021-0083.  [DOI](https://doi.org/10.2217/pgs-2021-0083) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34523354/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Polymorphisms%20in%20the%20CYP3A5%20gene%20significantly%20affect%20the%20pharmacokinetics%20of%20sirolimus%20after%20kidney%20transplantation&author=J.%20Liu&author=D.%20Feng&author=X.%20Kan&author=M.%20Zheng&author=X.%20Zhang&volume=22&publication_year=2021&pages=903-912&pmid=34523354&doi=10.2217/pgs-2021-0083&)

24. Wang Z.; Zheng M.; Yang H.; Han Z.; Tao J.; Chen H.; Sun L.; Guo M.; Wang L.; Tan R.; Wei J.-F.; Gu M. Association of Genetic Variants in CYP3A4, CYP3A5, CYP2C8, and CYP2C19 with Tacrolimus Pharmacokinetics in Renal Transplant Recipients. Curr. Drug Metab. 2019, 20, 609–618. 10.2174/1389200220666190627101927.  [DOI](https://doi.org/10.2174/1389200220666190627101927) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31244435/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr.%20Drug%20Metab.&title=Association%20of%20Genetic%20Variants%20in%20CYP3A4,%20CYP3A5,%20CYP2C8,%20and%20CYP2C19%20with%20Tacrolimus%20Pharmacokinetics%20in%20Renal%20Transplant%20Recipients&author=Z.%20Wang&author=M.%20Zheng&author=H.%20Yang&author=Z.%20Han&author=J.%20Tao&volume=20&publication_year=2019&pages=609-618&pmid=31244435&doi=10.2174/1389200220666190627101927&)

25. Thervet E.; Loriot M. A.; Barbier S.; Buchler M.; Ficheux M.; Choukroun G.; Toupance O. Optimization of Initial Tacrolimus Dose Using Pharmacogenetic Testing. Clin. Pharmacol. Ther. 2009, 87, 721–726. 10.1038/clpt.2010.17.  [DOI](https://doi.org/10.1038/clpt.2010.17) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20393454/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Optimization%20of%20Initial%20Tacrolimus%20Dose%20Using%20Pharmacogenetic%20Testing&author=E.%20Thervet&author=M.%20A.%20Loriot&author=S.%20Barbier&author=M.%20Buchler&author=M.%20Ficheux&volume=87&publication_year=2009&pages=721-726&pmid=20393454&doi=10.1038/clpt.2010.17&)

26. Birdwell K. A.; Decker B.; Barbarino J. M.; Peterson J. F.; Stein C. M.; Sadee W.; Wang D.; Vinks A. A.; He Y.; Swen J. J.; Leeder J. S.; Van Schaik R. H. N.; Thummel K. E.; Klein T. E.; Caudle K. E.; MacPhee I. A. M. Clinical Pharmacogenetics Implementation Consortium (CPIC) guidelines for CYP3A5 genotype and tacrolimus dosing. Clin. Pharmacol. Ther. 2015, 98, 19–24. 10.1002/cpt.113.  [DOI](https://doi.org/10.1002/cpt.113) | [PMC free article](/articles/PMC4481158/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25801146/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Clinical%20Pharmacogenetics%20Implementation%20Consortium%20(CPIC)%20guidelines%20for%20CYP3A5%20genotype%20and%20tacrolimus%20dosing&author=K.%20A.%20Birdwell&author=B.%20Decker&author=J.%20M.%20Barbarino&author=J.%20F.%20Peterson&author=C.%20M.%20Stein&volume=98&publication_year=2015&pages=19-24&pmid=25801146&doi=10.1002/cpt.113&)

27. Wang P.; Zhang Q.; Tian X.; Yang J.; Zhang X. Tacrolimus starting dose prediction based on genetic polymorphisms and clinical factors in chinese renal transplant recipients. Genet. Test. Mol. Biomarkers 2020, 24, 665–673. 10.1089/gtmb.2020.0077.  [DOI](https://doi.org/10.1089/gtmb.2020.0077) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32985896/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genet.%20Test.%20Mol.%20Biomarkers&title=Tacrolimus%20starting%20dose%20prediction%20based%20on%20genetic%20polymorphisms%20and%20clinical%20factors%20in%20chinese%20renal%20transplant%20recipients&author=P.%20Wang&author=Q.%20Zhang&author=X.%20Tian&author=J.%20Yang&author=X.%20Zhang&volume=24&publication_year=2020&pages=665-673&pmid=32985896&doi=10.1089/gtmb.2020.0077&)

28. Boone C.; Bussey H.; Andrews B. J. Exploring genetic interactions and networks with yeast. Nat. Rev. Genet. 2007, 8, 437–449. 10.1038/nrg2085.  [DOI](https://doi.org/10.1038/nrg2085) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17510664/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Genet.&title=Exploring%20genetic%20interactions%20and%20networks%20with%20yeast&author=C.%20Boone&author=H.%20Bussey&author=B.%20J.%20Andrews&volume=8&publication_year=2007&pages=437-449&pmid=17510664&doi=10.1038/nrg2085&)

29. Lehner B. Molecular mechanisms of epistasis within and between genes. Trends Genet. 2011, 27, 323–331. 10.1016/j.tig.2011.05.007.  [DOI](https://doi.org/10.1016/j.tig.2011.05.007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21684621/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Trends%20Genet.&title=Molecular%20mechanisms%20of%20epistasis%20within%20and%20between%20genes&author=B.%20Lehner&volume=27&publication_year=2011&pages=323-331&pmid=21684621&doi=10.1016/j.tig.2011.05.007&)

30. S S.; N G.; R Nair R. Pharmacogenomics of CYP3A5 Polymorphism: Predicting Dose-adjusted Trough Levels of Tacrolimus in South Indian Renal Transplant Patients. J. Pharmacogenomics Pharmacoproteomics 2016, 7, 10–14. 10.4172/2153-0645.1000161.  [DOI](https://doi.org/10.4172/2153-0645.1000161) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Pharmacogenomics%20Pharmacoproteomics&title=Pharmacogenomics%20of%20CYP3A5%20Polymorphism:%20Predicting%20Dose-adjusted%20Trough%20Levels%20of%20Tacrolimus%20in%20South%20Indian%20Renal%20Transplant%20Patients&author=S.%20S&author=G.%20N&author=R.%20R%20Nair&volume=7&publication_year=2016&pages=10-14&doi=10.4172/2153-0645.1000161&)

31. Min S. I.; Kim S. Y.; Ahn S. H.; Min S. K.; Kim S. H.; Kim Y. S.; Moon K. C.; Oh J. M.; Kim S. J.; Ha J. CYP3A5 *1 allele: Impacts on early acute rejection and graft function in tacrolimus-based renal transplant recipients. Transplantation 2010, 90, 1394–1400. 10.1097/TP.0b013e3181fa93a4.  [DOI](https://doi.org/10.1097/TP.0b013e3181fa93a4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21076384/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Transplantation&title=CYP3A5%20*1%20allele:%20Impacts%20on%20early%20acute%20rejection%20and%20graft%20function%20in%20tacrolimus-based%20renal%20transplant%20recipients&author=S.%20I.%20Min&author=S.%20Y.%20Kim&author=S.%20H.%20Ahn&author=S.%20K.%20Min&author=S.%20H.%20Kim&volume=90&publication_year=2010&pages=1394-1400&pmid=21076384&doi=10.1097/TP.0b013e3181fa93a4&)

32. Anutrakulchai S.; Pongskul C.; Kritmetapak K.; et al. Therapeutic concentration achievement and allograft survival comparing usage of conventional tacrolimus doses and CYP3A5 genotype - guided doses in renal transplantation patients. Br. J. Clin. Pharmacol. 2019, 85, 1964–1973. 10.1111/bcp.13980.  [DOI](https://doi.org/10.1111/bcp.13980) | [PMC free article](/articles/PMC6710516/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31077425/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Clin.%20Pharmacol.&title=Therapeutic%20concentration%20achievement%20and%20allograft%20survival%20comparing%20usage%20of%20conventional%20tacrolimus%20doses%20and%20CYP3A5%20genotype%20-%20guided%20doses%20in%20renal%20transplantation%20patients&author=S.%20Anutrakulchai&author=C.%20Pongskul&author=K.%20Kritmetapak&volume=85&publication_year=2019&pages=1964-1973&pmid=31077425&doi=10.1111/bcp.13980&)

33. Schnitzler M.; Kasiske B. L.; Snyder J. J.; Gilbertson D.; Matas A. J. Diabetes mellitus after kidney transplantation in the United States [1] (multiple letters). Am. J. Transplant. 2003, 3, 1318. 10.1046/j.1600-6143.2003.00228.x.  [DOI](https://doi.org/10.1046/j.1600-6143.2003.00228.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12603213/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Transplant.&title=Diabetes%20mellitus%20after%20kidney%20transplantation%20in%20the%20United%20States%20%5B1%5D%20(multiple%20letters)&author=M.%20Schnitzler&author=B.%20L.%20Kasiske&author=J.%20J.%20Snyder&author=D.%20Gilbertson&author=A.%20J.%20Matas&volume=3&publication_year=2003&pages=1318&pmid=12603213&doi=10.1046/j.1600-6143.2003.00228.x&)

34. Cosio F. G.; Kudva Y.; Van Der Velde M.; Larson T. S.; Textor S. C.; Griffin M. D.; Stegall M. D. New onset hyperglycemia and diabetes are associated with increased cardiovascular risk after kidney transplantation. Kidney Int. 2005, 67, 2415–2421. 10.1111/j.1523-1755.2005.00349.x.  [DOI](https://doi.org/10.1111/j.1523-1755.2005.00349.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15882287/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Kidney%20Int.&title=New%20onset%20hyperglycemia%20and%20diabetes%20are%20associated%20with%20increased%20cardiovascular%20risk%20after%20kidney%20transplantation&author=F.%20G.%20Cosio&author=Y.%20Kudva&author=M.%20Van%20Der%20Velde&author=T.%20S.%20Larson&author=S.%20C.%20Textor&volume=67&publication_year=2005&pages=2415-2421&pmid=15882287&doi=10.1111/j.1523-1755.2005.00349.x&)

35. Pham P.-T.; Pham P.-M.; Pham P.-C. New onset diabetes after transplantation (NODAT): an overview. Diabetes, Metab. Syndr. Obes. Targets Ther. 2011, 4, 175. 10.2147/DMSO.S19027.  [DOI](https://doi.org/10.2147/DMSO.S19027) | [PMC free article](/articles/PMC3131798/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21760734/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Diabetes,%20Metab.%20Syndr.%20Obes.%20Targets%20Ther.&title=New%20onset%20diabetes%20after%20transplantation%20(NODAT):%20an%20overview&author=P.-T.%20Pham&author=P.-M.%20Pham&author=P.-C.%20Pham&volume=4&publication_year=2011&pages=175&pmid=21760734&doi=10.2147/DMSO.S19027&)

36. Vincenti F.; Friman S.; Scheuermann E.; Rostaing L.; Jenssen T.; Campistol J. M.; Uchida K.; Pescovitz M. D.; Marchetti P.; Tuncer M.; Citterio F.; Wiecek A.; Chadban S.; El-Shahawy M.; Budde K.; Goto N. Results of an international, randomized trial comparing glucose metabolism disorders and outcome with cyclosporine versus tacrolimus. Am. J. Transplant. 2007, 7, 1506–1514. 10.1111/j.1600-6143.2007.01749.x.  [DOI](https://doi.org/10.1111/j.1600-6143.2007.01749.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17359512/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Transplant.&title=Results%20of%20an%20international,%20randomized%20trial%20comparing%20glucose%20metabolism%20disorders%20and%20outcome%20with%20cyclosporine%20versus%20tacrolimus&author=F.%20Vincenti&author=S.%20Friman&author=E.%20Scheuermann&author=L.%20Rostaing&author=T.%20Jenssen&volume=7&publication_year=2007&pages=1506-1514&pmid=17359512&doi=10.1111/j.1600-6143.2007.01749.x&)

37. Woodward R. S.; Schnitzler M. A.; Baty J.; Lowell J. A.; Lopez-Rocafort L.; Haider S.; Woodworth T. G.; Brenna D. C. Incidence and cost of new onset diabetes mellitus among U.S. wait-listed and transplanted renal allograft recipients. Am. J. Transplant. 2003, 3, 590–598. 10.1034/j.1600-6143.2003.00082.x.  [DOI](https://doi.org/10.1034/j.1600-6143.2003.00082.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12752315/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Transplant.&title=Incidence%20and%20cost%20of%20new%20onset%20diabetes%20mellitus%20among%20U.S.%20wait-listed%20and%20transplanted%20renal%20allograft%20recipients&author=R.%20S.%20Woodward&author=M.%20A.%20Schnitzler&author=J.%20Baty&author=J.%20A.%20Lowell&author=L.%20Lopez-Rocafort&volume=3&publication_year=2003&pages=590-598&pmid=12752315&doi=10.1034/j.1600-6143.2003.00082.x&)

38. Nair S. S.; Sarasamma S.; Gracious N.; George J.; Anish T. S.; Rao R. Polymorphism of the CYP3A5 gene and its effect on tacrolimus blood level. Exp. Clin. Transplant. 2015, 13, 197–200. 10.6002/ect.mesot2014.O167.  [DOI](https://doi.org/10.6002/ect.mesot2014.O167) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25894154/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Exp.%20Clin.%20Transplant.&title=Polymorphism%20of%20the%20CYP3A5%20gene%20and%20its%20effect%20on%20tacrolimus%20blood%20level&author=S.%20S.%20Nair&author=S.%20Sarasamma&author=N.%20Gracious&author=J.%20George&author=T.%20S.%20Anish&volume=13&publication_year=2015&pages=197-200&pmid=25894154&doi=10.6002/ect.mesot2014.O167&)
