# CYP2D6 and CYP3A4 involvement in the primary oxidative metabolism of hydrocodone by human liver microsomes

## Metadata
**Authors:** Mark R Hutchinson, Andrew Menelaou, David J R Foster, Janet K Coller, Andrew A Somogyi
**Journal:** British Journal of Clinical Pharmacology
**Date:** 2004 Mar
**DOI:** [10.1046/j.1365-2125.2003.02002.x](https://doi.org/10.1046/j.1365-2125.2003.02002.x)
**PMID:** 14998425
**PMCID:** PMC1884456
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1884456/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC1884456/pdf/bcp0057-0287.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC1884456/pdf/bcp0057-0287.pdf)

## Abstract

**Aim:** 
To determine the Michaelis-Menten kinetics of hydrocodone metabolism to its O- and N-demethylated products, hydromorphone and norhydrocodone, to determine the individual cytochrome P450 enzymes involved, and to predict the in vivo hepatic intrinsic clearance of hydrocodone via these pathways.

**Methods:** 
Liver microsomes from six CYP2D6 extensive metabolizers (EM) and one CYP2D6 poor metabolizer (PM) were used to determine the kinetics of hydromorphone and norhydrocodone formation. Chemical and antibody inhibitors were used to identify the cytochrome P450 isoforms catalyzing these pathways. Expressed recombinant cytochrome P450 enzymes were used to characterize further the metabolism of hydrocodone.

**Results:** 
Hydromorphone formation in liver microsomes from CYP2D6 EMs was dependent on a high affinity enzyme (Km = 26 µm) contributing 95%, and to a lesser degree a low affinity enzyme (Km = 3.4 mm). In contrast, only a low affinity enzyme (Km = 8.5 mm) formed this metabolite in the liver from the CYP2D6 PM, with significantly decreased hydromorphone formation compared with the livers from the EMs. Norhydrocodone was formed by a single low affinity enzyme (Km = 5.1 mm) in livers from both CYP2D6 EM and PM. Recombinant CYP2D6 and CYP3A4 formed only hydromorphone and only norhydrocodone, respectively. Hydromorphone formation was inhibited by quinidine (a selective inhibitor of CYP2D6 activity), and monoclonal antibodies specific to CYP2D6. Troleandomycin, ketoconazole (both CYP3A4 inhibitors) and monoclonal antibodies specific for CYP3A4 inhibited norhydrocodone formation. Extrapolation of in vitro to in vivo data resulted in a predicted total hepatic clearance of 227 ml h−1 kg−1 and 124 ml h−1 kg−1 for CYP2D6 EM and PM, respectively.

**Conclusions:** 
The O-demethylation of hydrocodone is predominantly catalyzed by CYP2D6 and to a lesser extent by an unknown low affinity cytochrome P450 enzyme. Norhydrocodone formation was attributed to CYP3A4. Comparison of recalculated published clearance data for hydrocodone, with those predicted in the present work, indicate that about 40% of the clearance of hydrocodone is via non-CYP pathways. Our data also suggest that the genetic polymorphisms of CYP2D6 may influence hydrocodone metabolism and its therapeutic efficacy.

Keywords: CYP2D6, CYP3A4, genotype, human, hydrocodone, hydromorphone, in vitro, liver microsomes, norhydrocodone

### Aim

To determine the Michaelis-Menten kinetics of hydrocodone metabolism to its O- and N-demethylated products, hydromorphone and norhydrocodone, to determine the individual cytochrome P450 enzymes involved, and to predict the *in vivo* hepatic intrinsic clearance of hydrocodone via these pathways.

### Methods

Liver microsomes from six *CYP2D6* extensive metabolizers (EM) and one *CYP2D6* poor metabolizer (PM) were used to determine the kinetics of hydromorphone and norhydrocodone formation. Chemical and antibody inhibitors were used to identify the cytochrome P450 isoforms catalyzing these pathways. Expressed recombinant cytochrome P450 enzymes were used to characterize further the metabolism of hydrocodone.

### Results

Hydromorphone formation in liver microsomes from *CYP2D6* EMs was dependent on a high affinity enzyme (*K*_m_ = 26 µm) contributing 95%, and to a lesser degree a low affinity enzyme (*K*_m_ = 3.4 mm). In contrast, only a low affinity enzyme (*K*_m_ = 8.5 mm) formed this metabolite in the liver from the *CYP2D6* PM, with significantly decreased hydromorphone formation compared with the livers from the EMs. Norhydrocodone was formed by a single low affinity enzyme (*K*_m_ = 5.1 mm) in livers from both *CYP2D6* EM and PM. Recombinant CYP2D6 and CYP3A4 formed only hydromorphone and only norhydrocodone, respectively. Hydromorphone formation was inhibited by quinidine (a selective inhibitor of CYP2D6 activity), and monoclonal antibodies specific to CYP2D6. Troleandomycin, ketoconazole (both CYP3A4 inhibitors) and monoclonal antibodies specific for CYP3A4 inhibited norhydrocodone formation. Extrapolation of *in vitro* to *in vivo* data resulted in a predicted total hepatic clearance of 227 ml h^−1^ kg^−1^ and 124 ml h^−1^ kg^−1^ for *CYP2D6* EM and PM, respectively.

### Conclusions

The O-demethylation of hydrocodone is predominantly catalyzed by CYP2D6 and to a lesser extent by an unknown low affinity cytochrome P450 enzyme. Norhydrocodone formation was attributed to CYP3A4. Comparison of recalculated published clearance data for hydrocodone, with those predicted in the present work, indicate that about 40% of the clearance of hydrocodone is via non-CYP pathways. Our data also suggest that the genetic polymorphisms of *CYP2D6* may influence hydrocodone metabolism and its therapeutic efficacy.

## Introduction

Hydrocodone is a semisynthetic opioid, structurally related to codeine ([Figure 1](#fig01)). It was first synthesized in the 1920s [[1](#b1)] and is used clinically as an analgesic and antitussive agent. Hydrocodone undergoes cytochrome P450 (CYP) dependent oxidative metabolism to the O- and N-demethylated products, hydromorphone and norhydrocodone, respectively [[2](#b2)] ([Figure 1](#fig01)). However, data on the overall metabolism and routes of clearance of hydrocodone are incomplete. Cone *et al.*[[2](#b2)] recovered only 26% of the dose as parent drug and O- and N-demethylated metabolites in urine, whereas Otton *et al.*[[3](#b3)] recovered only 16% of the dose as conjugated and unconjugated hydrocodone and hydromorphone in urine and did not measure norhydrocodone.

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/68a1/1884456/6753213bcea0/bcp0057-0287-f1.jpg)

Metabolic pathway showing the O-demethylation and N-demethylation of hydrocodone to hydromorphone and norhydrocodone, respectively. The structure of codeine is also depicted

The µ opioid receptor binding affinity of hydromorphone is 30 times greater than that of hydrocodone [[4](#b4)] and has therefore been suggested to contribute to the therapeutic efficacy and side-effects of the parent drug [[5](#b5)], although this remains controversial [[6](#b6)]. The polymorphic enzyme CYP2D6 catalyzes the O-demethylation pathway of many 4,5-epoxymorphinan opioids such as codeine [[7](#b7)], oxycodone [[8](#b8)] and dihydrocodeine [[9](#b9)]. The CYP isoforms mediating the oxidative metabolism of hydrocodone, especially the N-demethylated pathway, have not been well characterized *in vitro*. In contrast, there is considerable *in vivo* evidence that CYP2D6 is involved in the O-demethylation reaction [[3](#b3), [6](#b6)]. Approximately 7% of the Caucasian population have inactivating mutations in the gene encoding CYP2D6 or have complete deletion of the gene [[10](#b10)]. These subjects are classified poor metabolizers (PM) and are distinguishable from extensive metabolizers (EM) by their impaired ability to metabolize compounds that are substrates for this enzyme. The clearance of drugs metabolized by CYP2D6 can be 10–20 fold lower in PMs than EMs [[11](#b11)].

*In vitro*, CYP-mediated metabolism data are increasingly being used to predict *in vivo* intrinsic and hepatic clearance values. This tool becomes even more powerful when used in combination with *in vivo* human data. By comparing the *in vivo* clearance values predicted from *in vitro* data with those obtained *in vivo* from human subjects, differences can be highlighted and further investigated. If binding of the substrate to microsomal proteins is accounted for, the ability to accurately predict *in vivo* clearance value parameters is enhanced [[12](#b12)–[14](#b14)].

The aims of this study were to determine the kinetics of hydrocodone metabolism to its O- and N-demethylated primary metabolites, hydromorphone and norhydrocodone, to define the involvement of individual CYP enzymes and to predict the *in vivo* hepatic intrinsic clearance of hydrocodone through these pathways.

## Methods

### Chemicals

Hydrocodone base, norhydrocodone base and norhydromorphone HCl salt were obtained from Dr S. Hosztafi, ICN Alkaloida Company (Tiszavasvari, Hungary). Noroxycodone was obtained from Du Pont Pharmaceuticals (Wilmington, Delaware, USA). Furafylline and S-mephenytoin were purchased from Ultrafine Chemicals (Manchester, England). Bovine serum albumin (fraction V), diethyldithiocarbamate, ±-isocitric acid Na_3_, isocitrate dehydrogenase (NADP, type IV), sulphaphenazole, troleandomycin, coumarin, chlorzoxazone, hydrocodone bitartrate, hydromorphone HCl and sodium perchlorate (NaClO_4_) were purchased from Sigma Chemical Co (St Louis, MO, USA). Ketoconazole was obtained from Janssen Pharmaceuticals (Beerse, Belgium) and quinidine sulphate and nicotinamide adenine dinucleotide phosphate sodium salt were purchased from Merck (Darmstadt, Germany). Omeprazole was donated as a condition of trade by Dr M. Ching (Department of Medicine, Repatriation Campus, Austin and Repatriation Medical Centre, Melbourne, Australia). Monoclonal antibodies raised against human CYP2D6 (MAB-2D6), CYP3A4 (MAB-3A4), CYP1A2 (MAB-1A2) and CYP2E1 (MAB-2E1) and microsomes from human lymphoblastoid cells containing expressed CYP3A4, CYP2D6 and CYP2C19, and supersomes from baculovirus-insect cells containing expressed recombinant CYP2C19 were purchased from BD GENTEST™ (a BD Biosciences Company, Woburn, MA, USA). All other reagents and chemicals were obtained from commercial sources and were of analytical grade.

### Human liver microsomes

Ethics approval for the study was obtained from the Committee on the Ethics of Human Experimentation of the University of Adelaide and the Human Ethics Committee of the Royal Adelaide Hospital. Human liver samples (HLS) were obtained during hepatectomy from patients (coded HLS# 5, 11, 21, 23, 24, 31 and 35) who had given their written informed consent for their tissue to be used as reported previously [[15](#b15)–[17](#b17)]. Donors ranged in age from 41 to 73 years, three were female and all had normal clinical chemistry and haematology measurements prior to surgery. All tissue samples used were normal based on gross morphology. Microsomes were prepared by differential centrifugation of liver homogenates [[18](#b18)], and total protein and CYP content of the microsomes were determined using previously established methods [[19](#b19), [20](#b20)]. Liver samples and microsomes were stored at −80°C until used. Genomic DNA was isolated from human liver tissue using a QIAamp® DNA mini kit (QIAGEN, Clifton Hill, Australia) according to the manufacturer's protocol. Genotyping for *CYP2D6***3* and **4*, and *CYP2C19***2* and **3* alleles was performed by allele-specific polymerase chain reactions (PCR) as described previously [[21](#b21), [22](#b22)]. The *CYP2D6* genotype of HLS# 5, 11, 23, 31 and 35 was *CYP2D6***1/***1*, that of HLS# 21 was *CYP2D6***1/***4* and that of HLS# 24 was *CYP2D6***4/***4* (PM). The *CYP2C19* genotype of HLS# 21, 23, 24, 31 and 35 was *CYP2C19***1/***1*, and that of HLS# 5 and 11 was *CYP2C19***1/***2*. Protein concentration of the human liver microsome preparations ranged from 3.68 to 10.26 mg ml^−1^ and the total CYP concentration ranged from 154 to 598 pmol CYP mg^−1^ protein.

### Microsomal incubations

Incubations were performed in duplicate at 37 °C in a shaking water bath for 45 min in a final volume of 250 µl. Each sample contained 175 µl of substrate (over a concentration range of 25 µm to 15 mm) diluted in incubation buffer (0.1 m disodium hydrogen phosphate, pH 7.4), 25 µl of a NADPH regenerating system (1 mg ml^−1^ isocitrate dehydrogenase, 10 mm NADP-, 50 mm±-isocitric acid Na_3_ and 50 mm MgCl_2_ 6H_2_O) and 50 µl of microsomal protein (200 µg). The reaction was stopped by the addition of 20% sodium carbonate that contained 5.5 µm of the internal standard, noroxycodone. The formation rates of hydromorphone and norhydrocodone were linear up to 80 min and 1200 µg microsomal protein.

### Hydromorphone and norhydrocodone quantification

The measurement of hydromorphone and norhydrocodone in microsomal incubations has been described recently [[23](#b23)]. Briefly, the method involved a liquid-liquid extraction/re-extraction procedure, followed by reverse-phase HPLC with UV detection. Interassay performance was assessed from six analyses of quality control (QC) samples prepared in duplicate and containing a low (LQC, 4 µm), a medium (MQC, 40 µm), and a high (HQC, 160 µm) concentration of both hydromorphone and norhydrocodone. The lower limit of quantification (LLOQ) was 0.63 µm for hydromorphone and 1.25 µm for norhydrocodone. Inter-assay accuracy and precision (mean accuracy ± % CV) for hydromorphone (*n* = 6) was 114 ± 4%, 99 ± 4%, 109 ± 6% and 106 ± 4%, whilst for norhydrocodone (*n* = 6) it was 108 ± 7%, 98 ± 4%, 107 ± 6% and 109 ± 5% for the LQC, MQC, HQC and LLOQ, respectively. Intra-assay accuracy and precision for hydromorphone (*n* = 6) was 109 ± 4%, 100 ± 1%, 109 ± 5% and 100 ± 2%, whilst for norhydrocodone (*n* = 6) it was 110 ± 6%, 98 ± 2%, 108 ± 5% and 93 ± 6% for the LQC, MQC, HQC and LLOQ, respectively.

### Inhibition studies – chemical inhibitors

Liver microsomes from three *CYP2D6* EM (HLS# 5, 11, 21) and one *CYP2D6* PM (HLS# 24) were incubated with the NADPH regenerating system and a range of chemical inhibitors. Substrate concentrations used were similar to the *K*_m_ value estimated for each pathway in each liver sample. The CYP isoform specific chemical inhibitors [[24](#b24)–[26](#b26)] used (isoform and final incubation concentration) were quinidine (CYP2D6, 0.1 and 1 µm), coumarin (CYP2A6, 100 µm), diethyldithiocarbamate (CYP2E1, 10 µm) all dissolved in water, S-mephenytoin (CYP2C19, 100 µm), troleandomycin (CYP3A4, 1 and 10 µm), chlorzoxazone (CYP2E1, 10 µm), ketoconazole (CYP3A4, 1 and 10 µm) all dissolved in 0.5% methanol, furafylline (CYP1A2, 25 and 100 µm) dissolved in 0.1% acetonitrile, sulphaphenazole (CYP2C9, 100 µm) dissolved in 0.83% dimethylsulphoxide and omeprazole (CYP3A4 and CYP2C19, 100 µm) dissolved in 0.5% methanol at pH 10.5. Incubation conditions did not alter from the kinetic study, except that the mechanism-based inhibitors diethyldithiocarbamate, furafylline and troleandomycin were preincubated (20 min at 37 °C) with all incubation constituents prior to the addition of substrate. Previous studies have demonstrated that the solvents used to dissolve the inhibitors can influence the degree of formation of metabolites [[16](#b16), [27](#b27), [28](#b28)]. Therefore, controls with the equivalent solvent concentrations were used. Control incubations containing putative inhibitors alone produced no peaks that interfered chromatographically with those of hydromorphone or norhydrocodone.

### Inhibition studies – CYP isoform specific antibodies

Liver microsomes from three *CYP2D6* EM (HLS# 5, 11 and 21) were incubated in duplicate with MAB-3A4, MAB-2D6, MAB-1A2 and MAB-2E1. The substrate concentrations used were similar to the *K*_m_ value estimated in each microsome sample for each pathway. The concentration of MAB-3A4, MAB-2D6, MAB-1A2 and MAB-2E1 was 8 µl 200 µg^−1^ microsomal protein in accordance with the manufacturer's directions. Microsomes and antibodies were pre-incubated for 15 min on ice, prior to the addition of the remaining incubation constituents. Other incubation conditions were the same as in the kinetic study.

### Metabolite formation from expressed CYP isoforms

Expressed CYP isoforms were used in accordance with the manufacturer's directions to determine the extent of hydromorphone and norhydrocodone formation. Incubation conditions were the same as in the kinetic study, except that the CYP enzymes were stored on ice until they were added to the reaction mixture. The protein concentration used was the same (200 µg per sample) as in the kinetic studies. Single substrate concentrations (1.5 mm) were used initially to determine any metabolite formation. Subsequently, kinetic profiles of hydromorphone (CYP2D6) and norhydrocodone (CYP3A4) formation were obtained using substrate concentration ranges of 50 µm−15 mm and 100 µm−8 mm, respectively.

### Liver microsome binding

Microsomes from two human liver samples (HLS# 5 and 21) were assessed for the degree of hydrocodone binding using an MPS micropartition kit (Amicon, Beverly, MA, USA) with two types of YMT filtration membranes (10000 and 30000 kDa molecular weight cut off, Amicon). Binding was measured at three concentrations of hydrocodone (100 µm, 500 µm, 2 mm) in duplicate and at the same concentration of microsomal protein as used previously. The devices were used according to the manufacturer's directions using a Beckman J2-21 centrifuge with both the chamber and the fixed angle rotor (JA 20.1, Beckman) prewarmed to 37 °C. Hydrocodone, incubation buffer and microsomal protein were placed in the device (final volume 500 µl) and incubated for 10 min at 37 °C in a shaking water bath followed by a 2–5 min centrifuge at 1000 × *g*. The resultant filtrate was then injected directly onto the HPLC system. The data were expressed as percent of control incubations containing no microsomal protein.

### Data analysis

The rates of hydromorphone and norhydrocodone formation (*V*) from the kinetic studies were expressed as nmol mg^−1^protein h^−1^. Eadie-Hofstee (*V*/[S] *vs V*) plots were constructed, and a single enzyme and two enzyme Michaelis-Menten equations were fitted to the unweighted data using nonlinear least-squares regression analysis (Regression, Blackwell Scientific Publications, Oxford, UK) in order to provide estimates of *V*_max_ and *K*_m_. The decision to use a one or two enzyme model was based on visual inspection of the goodness of fit of the predicted line to the observed data, a smaller sum of squared deviations and visual inspection of the Eadie-Hofstee plot. Intrinsic clearance (CL_int_) was calculated as *V*_max_/*K*_m_. The percent involvement of each oxidative metabolic pathway was calculated by dividing the intrinsic clearance of each pathway by the total intrinsic clearance (sum of the intrinsic clearance of each pathway). Inhibition data were expressed as a percent of controls.

Extrapolation of *in vitro* CL_int_ to predict *in vivo* hepatic clearance of hydrocodone () to each of the quantified metabolites was performed using the well-stirred model [[12](#b12), [13](#b13)] described by the following equation:

| Category |
| --- |
|   |
where Q_H_ is hepatic blood flow, *f*_u plasma_ is the unbound fraction of hydrocodone in plasma (assumed to be 1), *f*_u micro_ is the unbound fraction of hydrocodone in microsomes and CL_int (whole organ)_ is the *in vitro* CL_int_ value scaled up to the whole liver for each metabolic pathway quantified. Predictions were made for a 75 kg subject, assuming Q_H_ = 21 ml min^−1^ kg^−1^, 20 g liver kg^−1^ body weight^−1^ and 45 mg microsomal protein g^−1^ liver [[12](#b12), [13](#b13)]. Apparent human oral hydrocodone clearance data from Otton *et al.*[[3](#b3)] were converted to apparent systemic clearance () using the well stirred model described by the equation:

| Category |
| --- |
|   |
where is the value of apparent oral clearance of hydrocodone reported by Otton *et al.*[[3](#b3)], Q_H_ is hepatic blood flow and *f*_u plasma_ is the unbound fraction of hydrocodone in plasma. Values used in the calculations were Q_H_ = 21 ml min^−1^ kg^−1^ and *f*_u plasma_ was assumed to equal 1 since no data are available for this parameter and the binding of similar 4,5-epoxymorphinans such as codeine is less than 10%[[29](#b29)].

Mean and standard deviation of inhibitor experiments were calculated using the following equations.

Mean

| Category |
| --- |
|   |
Standard deviation

| Category |
| --- |
|   |
Statistical significance of inhibition was assessed using a one-way anova with a Dunnett's multiple comparisons *posthoc test*. Significance was set at *P* < 0.05 and all data are expressed as mean ± standard deviation (SD).

## Results

The Eadie-Hofstee plots of hydromorphone formation from the EM livers were found to be curvilinear ([Figure 2a](#fig02)), in contrast to that from the PM liver, which was linear ([Figure 2a](#fig02)). Subsequently, the two-enzyme Michaelis-Menten kinetic model was fitted to all the EM liver data ([Figure 2b](#fig02)), with the single enzyme kinetic model fitting the kinetic data of the PM liver ([Figure 2b](#fig02)). [Table 1](#tbl1) contains the results of the kinetic analysis of hydromorphone formation. Liver microsomes from the *CYP2D6* PM liver formed substantially less hydromorphone than did the microsomes from the *CYP2D6* EM liver. *CYP2C19* genotype did not affect hydromorphone formation.

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/68a1/1884456/63ab26213dc5/bcp0057-0287-f2.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=1884456_bcp0057-0287-f2.jpg)

A) Eadie-Hofstee plot of hydromorphone formation in livers from CYP2D6 EM (▪) and PM (▴) (V/[S] is multiplied by a factor of 50 to allow it to fit on the same graph). B) Michaelis-Menten plot of (V vs[S] mm) hydromorphone formation in livers from CYP2D6 EM and PM. C) Eadie-Hofstee plot of norhydrocodone formation in livers from CYP2D6 EM. D) Michaelis-Menten kinetic profile of norhydrocodone formation in a liver from a CYP2D6 EM. HLS# 21 was the liver from the CYP2D6 EM. [S] represents the hydrocodone concentration. Error bars indicate SD of duplicate samples

### Table 1.

|   | CYP2D6 EM |  |  |  |  |  | EM | CYP2D6 PM |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Kinetic parameters | #5 | #11 | #21 | #23 | #31 | #3 | Mean ± SD | #24 |
| Vmax1 (nmol mg−1 protein h−1) | 1.6 | 2.1 | 1.0 | 1.6 | 0.8 | 1.2 | 1.4 ± 0.5 |   |
| Km1 (µm) | 56.9 | 31.6 | 3.0 | 49.2 | 5.0 | 10.0 | 26 ± 23 |   |
| CLint1 (µl h−1 mg−1 protein) | 27.6 | 66.5 | 343 | 33.0 | 150 | 123 | 124 ± 118 |   |
| Vmax2 (nmol mg−1 protein h−1) | 7.3 | 8.6 | 4.6 | 3.3 | 1.9 | 5.5 | 5.2 ± 2.5 | 6.4 |
| Km2 (mm) | 3.1 | 10.1 | 0.5 | 0.6 | 1.2 | 5.2 | 3.4 ± 3.7 | 8.5 |
| CLint 2 (µl h−1 mg−1 protein) | 2.4 | 0.9 | 9.4 | 5.5 | 1.6 | 1.1 | 3.5 ± 3.4 | 0.7 |

Table 1 Caption: Kinetic parameters for hydromorphone formation by liver microsomes from six CYP2D6 EM and one PM (#24). Vmax is the maximum rate of formation, Km is the concentration at which half the Vmax is obtained and CLint is the intrinsic clearance calculated as Vmax/Km

Eadie-Hofstee plots for norhydrocodone formation were linear in microsomes from both the *CYP2D6* EM and PM livers ([Figure 2c](#fig02)). The single enzyme Michaelis-Menten kinetics model best described data from all the livers ([Figure 2d](#fig02) and [Table 2](#tbl2)). *CYP2D6* or *CYP2C19* genotype did not affect norhydrocodone formation.

### Table 2.

|   | HLS |  |  |  |  |  |  |   |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Kinetic parameters | #5 | #11 | #21 | #23 | #31 | #35 | #24 | Mean ± SD |
| Vmax (nmol mg−1 protein h−1) | 1016 | 1001 | 175 | 98.6 | 233 | 406 | 527 | 494 ± 380 |
| Km (mm) | 3.9 | 9.1 | 1.1 | 1.8 | 7.8 | 7.4 | 4.3 | 5.1 ± 3.0 |
| CLint (µl h−1 mg−1 protein) | 261 | 110 | 153 | 55.5 | 29.9 | 54.5 | 123 | 112 ± 79 |

Table 2 Caption: Kinetic parameters for norhydrocodone formation by liver microsomes from six CYP2D6 EM and one PM (#24). Vmax is the maximum rate of formation, Km is the concentration at which half the Vmax is obtained and CLint is the intrinsic clearance calculated as Vmax/Km

Hydromorphone formation was mediated predominantly (95 ± 5%, range 86–99%) by a high affinity enzyme with little involvement of a low affinity enzyme (5 ± 5%, range 1–14%). [Figure 3](#fig03) shows the contribution of the O- and N-demethylation pathways to total oxidative metabolism for each liver sample. The mean contribution of the high affinity enzyme to the intrinsic clearance of hydrocodone to hydromorphone was 52% of the sum of the intrinsic clearances for O- and N-demethylation, but the variability between liver microsomes from *CYP2D6* EM livers was large (9–83%).

### Figure 3.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/68a1/1884456/1acfe7748537/bcp0057-0287-f3.jpg)

Percent contribution of the three oxidative metabolic pathways: hydromorphone high () and low affinity (▪) sites and norhydrocodone (□), in hydrocodone metabolism. Data were obtained from division of the intrinsic clearance of each pathway by the sum of the calculated intrinsic clearances

Hydromorphone formation was significantly inhibited by 1 µm quinidine (*P* < 0.01; decrease = −57%, 95% confidence intervals of the difference −96%, −17%) in the three *CYP2D6* EM ([Figure 4a](#fig04)). Quinidine did not affect this reaction in the liver from the *CYP2D6* PM (data not shown). Furafylline (25 µm, *P* < 0.05; −19%, −34%, −4% and 100 µm, *P* < 0.05; −18%, −33%, −2%) also inhibited hydromorphone formation in livers from both EM and PM. No inhibition was observed with sulphaphenazole (data not shown).

### Figure 4.

![Figure 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/68a1/1884456/08bc1001a19f/bcp0057-0287-f4.jpg)

Inhibition of hydromorphone (A) and norhydrocodone (B) formation by a range of chemical inhibitors in liver microsomes from CYP2D6 EM subjects. Error bars indicate SD of duplicate samples. *P < 0.05

In the livers from the three *CYP2D6* EM, norhydrocodone formation was inhibited in a concentration-dependent manner by troleandomycin at 10 µm (*P* < 0.01; −70%; −92%, −48%) and 1 µm (*P* < 0.01; −39%; −60%, −17%) and by ketoconazole at 1 µm (*P* < 0.01; −62%; −84%, −40%) ([Figure 4b](#fig04)).

Only the MAB-2D6 antibody caused significant (*P* < 0.01; −74%; −103%, −45%) inhibition of hydromorphone formation ([Figure 5](#fig05)), MAB-3A4 antibodies inhibited norhydrocodone formation significantly (*P* < 0.01; −76%; −95%, −58%) whereas the other antibodies had no significant effect (*P* > 0.05) ([Figure 5](#fig05))*.*

### Figure 5.

![Figure 5](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/68a1/1884456/9b6146fcec1d/bcp0057-0287-f5.jpg)

Inhibition of hydromorphone (▪) and norhydrocodone (□) formation by monoclonal antibodies directed towards human CYPs. Error bars indicate SD of duplicate samples. *P < 0.05

Recombinant CYP2C19 (from human lymphoblastoid cell microsomes) did not form quantifiable amounts of the two hydrocodone metabolites, whereas CYP2C19 supersomes (baculovirus-insect cell microsomes) formed hydromorphone and norhydrocodone. Kinetic parameters for hydromorphone formation by CYP2D6 (*K*_m_= 54 µm) and norhydrocodone formation by CYP3A4 (*K*_m_ = 2.6 mm) were obtained by fitting the single enzyme Michaelis-Menten kinetic model to the data.

Analysis of the microsomal protein binding data (based on a 30 000 kDa molecular weight cut-off) revealed that 18.7 ± 0.8% of hydrocodone was bound in a concentration-independent manner. There was no increase in the binding of hydrocodone using 10 000 kDa membranes, suggesting that no protein bound hydrocodone passed through the larger pore membranes. Therefore, the data from the 30 000 kDa membranes were used for the *in vitro* to *in vivo* scaling calculations. Consequently, *K*_m_ values for hydrocodone corrected for microsomal binding were 21 µm, 2.8 mm and 6.9 mm for the high and low affinity sites of hydromorphone formation and the site of norhydrocodone formation, respectively.

The *in vitro* to *in vivo* scaling calculations (taking into account microsomal binding) gave values for of 116 ± 98 ml h^−1^ kg^−1^, 4 ± 4 ml h^−1^ kg^−1^ and 107 ± 76 ml h^−1^ kg^−1^ for the high and low affinity sites of hydromorphone formation and for norhydrocodone formation, respectively, in the liver microsomes from *CYP2D6* EM. The sum of these clearances was 227 ± 122 ml h^−1^ kg^−1^. The scaled for hydromorphone formation with the liver from the *CYP2D6* PM was 1 ml h^−1^ kg^−1^, whereas that for norhydrocodone was 123 ml h^−1^ kg^−1^. The apparent mean systemic hepatic clearance values for hydrocodone calculated from the data of Otton *et al.*[[3](#b3)] were 367 ml h^−1^ kg^−1^ and 230 ml h^−1^ kg^−1^ for the *CYP2D6* EM and PM subjects, respectively, and the clearance of hydrocodone to hydromorphone reported by Otton *et al.*[[3](#b3)] was calculated to be 27.4 ± 9.9 ml h^−1^ kg^−1^ in *CYP2D6* EM subjects.

## Discussion

This study was undertaken to characterize the CYP isoforms involved in hydrocodone metabolism to its O- and N-demethylated primary oxidative metabolites hydromorphone and norhydrocodone, respectively, and to calculate the predicted *in vivo* hepatic clearance of hydrocodone via these two pathways.

Hydromorphone formation in livers from *CYP2D6* EM subjects was mediated by a high and a low affinity enzyme, with the former accounting for 97% of the *in vitro* intrinsic clearance to this metabolite. The considerable (12-fold) variability in intrinsic clearance to hydromorphone between the livers from the six EM subjects studied may indicate the presence of undetected *CYP2D6* variant alleles, which may have resulted in decreased or enhanced enzyme activity. In contrast, hydromorphone formation in the liver from a *CYP2D6* PM subject was carried out by a single enzyme with a *K*_m_ value within the range found for the low affinity enzyme in the livers of the *CYP2D6* EM subjects. A previous study has reported single enzyme kinetics for hydromorphone formation (mean *K*_m_ = 300 µm) in microsomes from three human liver samples [[3](#b3)]. The reason for the discrepancy between these and our findings is likely to be due to the limited hydrocodone concentration range (10–375 µm) used by Otton *et al.*[[3](#b3)] compared with the present study.

Quinidine and antibodies directed towards CYP2D6 all substantially inhibited hydromorphone formation, indicating almost exclusive involvement of CYP2D6 in hydromorphone formation [[24](#b23)]. The inhibition of hydromorphone formation by furafylline at 25 µm was not increased when increasing the inhibitor concentration to 100 µm (data not shown). Since antibodies against CYP1A2 had no effect on hydromorphone formation, it appears that inhibition by furafylline may be nonspecific. Hydromorphone was formed by CYP2D6 expressed enzymes (*K*_m_ = 54 µm) and CYP2C19 supersomes (from baculovirus-insect cells), but not by CYP2C19 microsomes (from lymphoblastoid cells). The different expression systems and the difference between their catalytic activities as reported by the manufacturer (S-mephenytoin 4′-hydroxylase activity: CYP2C19 supersomes 6.6 pM min^−1^ pM P450 vs CYP2C19 microsomes 1.9 pM min^−1^ pM P450) may have contributed to the differences observed.

Taking into consideration all these findings, it was concluded that the O-demethylation of hydrocodone to hydromorphone is catalyzed predominantly by CYP2D6 at therapeutic concentrations. The 3-O-demethylation of structurally related 4,5-epoxymorphinans such as codeine [[30](#b30)], oxycodone [[8](#b8), [9](#b9), [30](#b30), [31](#b31)] and dihydrocodeine [[9](#b9)] has also been shown to be dependent on CYP2D6. The low rate of formation of hydromorphone observed in the livers of *CYP2D6* PM subjects by Otton *et al.*[[3](#b3)] is likely to be catalyzed by the low affinity enzyme site observed in the present study.

Norhydrocodone formation by the livers from *CYP2D6* EM and PM subjects was dependent on a single low affinity enzyme and accounted for a mean of 47% of the combined *in vitro* intrinsic clearances of hydrocodone to hydromorphone and norhydrocodone. Troleandomycin, ketoconazole and antibodies directed towards CYP3A4 significantly inhibited norhydrocodone formation. The *K*_m_ for the formation of this metabolite by CYP3A4 expressed enzymes (2.58 mm) was within the range observed in the human liver microsomes (1.1–9.1 mm). These cumulative data indicate that CYP3A4 is mainly responsible for norhydrocodone formation [[24](#b24), [26](#b26)]. The N-demethylation of the structurally related 4,5-epoxymorphinans, codeine [[31](#b31)], dihydrocodeine [[9](#b9)] and oxycodone [[8](#b8)] is also mediated principally by CYP3A4.

The mean combined N- and O-demethylation *in vivo* clearance of hydrocodone calculated using the *in vitro* data corrected for microsomal binding was less in both EM (38%) and PM (46%) subjects than those reported by Otton *et al.*[[3](#b3)] (recalculated from their data). This difference is likely to be due to the presence of other routes of hydrocodone elimination, such as reduction at the 6-keto position [[2](#b2)] as well as faecal elimination of hydrocodone [[32](#b32)] through biliary and/or intestinal secretion. The magnitude of the difference between the observed hydrocodone clearance data of Otton *et al.*[[3](#b3)] and those predicted by us is similar for both CYP2D6 EM and PM subjects, suggesting that the non-CYP clearance of hydrocodone is the same for both genotypic groups.

Otton *et al.*[[3](#b3)] also calculated the partial clearance of hydrocodone to hydromorphone as the amount of hydromorphone (conjugated and nonconjugated) in urine divided by the AUC of hydrocodone. These values (28 ± 10 and 3.4 ± 2.4 ml h^−1^ kg^−1^ in the five EM and six PM subjects, respectively) are inconsistent with the difference in hydrocodone oral clearance between *CYP2D6* EM and PM subjects (664 and 387 ml h^−1^ kg^−1^) but are consistent with the almost equal contribution of O- and N-demethylation to the overall clearance of hydrocodone. Other pathways of hydromorphone metabolism (reduction at the 6-position, N-demethylation and conjugation [[33](#b33)]) and elimination described above may explain the discrepancy in the differences in the partial and oral clearances.

The elimination of hydrocodone by *CYP2D6* PM subjects is altered with substantially less formation of hydromorphone, such that the relative contribution of norhydrocodone is increased with an increase in the amount of hydrocodone recovered unchanged in urine, as found by Otton *et al*. [[3](#b3)] (18% PM vs 10% EM). The effect of lower hydromorphone concentrations on the analgesic effects of hydrocodone in PM subjects is unknown. Lelas *et al.*[[34](#b34)] found that in primates, inhibition of hydromorphone formation by prior administration of quinidine resulted in plasma hydromorphone concentrations approximately 10% of those of controls, but this had negligible effects on the antinociceptive actions of hydrocodone. Lelas *et al.*[[34](#b34)] found that hydrocodone was equipotent with hydromorphone at reversing the naltrexone discriminating stimulus in morphine dependent monkeys, despite a 30-fold difference in µ opioid receptor binding affinity [[4](#b4)]. This finding is in stark contrast to data on other structurally related 4,5-epoxymorphinans. For example codeine seems to require the O-demethylation of the parent compound to produce the active metabolite morphine for analgesia [[35](#b35)]. Whether hydrocodone can produce significant analgesia in its own right even though it has comparatively low µ opioid receptor binding affinity is not known. Further research is required to elucidate the effect of *CYP2D6* PM genotype on hydrocodone analgesia and cough suppression.

In conclusion, our findings from human liver microsomes, expressed CYP enzymes and CYP isoform specific chemical and antibody inhibitors indicate that hydrocodone is extensively metabolized via CYP2D6 and CYP3A4 to hydromorphone and norhydrocodone, respectively, similar to other 4,5-epoxymorphinans. Extrapolation of *in vitro* data to *in vivo* intrinsic clearance values indicates that in *CYP2D6* PM subjects hydromorphone formation is only 1% that of that in *CYP2D6* EM subjects. Further pharmacokinetic and pharmacodynamic studies are required to clarify the relative importance of the parent drug and hydromorphone to the therapeutic efficacy and toxicity of hydrocodone, in relation to CYP2D6 phenotype.

## Acknowledgments

Mark R. Hutchinson is the recipient of an Australian Postgraduate Award. Dr J. K. Coller is the recipient of a C. J. Martin Post-Doctoral Training Fellowship from the National Health and Medical Research Council of Australia. We thank Dr S. Hosztafi (ICN Alkaloida Company, Tiszavasvari, Hungary) for the supply of hydrocodone, norhydrocodone and norhydromorphone.

## References

1. Forbes JA, Bowser MW, Calderazzo JP, Foor VM. An evaluation of the analgesic efficacy of three opioid-analgesic combinations in postoperative oral surgery pain. J Oral Surg. 1981;39:108–112.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/6936524/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Oral%20Surg&title=An%20evaluation%20of%20the%20analgesic%20efficacy%20of%20three%20opioid-analgesic%20combinations%20in%20postoperative%20oral%20surgery%20pain&author=JA%20Forbes&author=MW%20Bowser&author=JP%20Calderazzo&author=VM%20Foor&volume=39&publication_year=1981&pages=108-112&pmid=6936524&)

2. Cone E, Darwin W, Gorodetzky C, Tan T. Comparative metabolism of hydrocodone in man, rat, guinea pig, rabbit, and dog. Drug Metab Dispos. 1978;6:488–493.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/28931/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Comparative%20metabolism%20of%20hydrocodone%20in%20man,%20rat,%20guinea%20pig,%20rabbit,%20and%20dog&author=E%20Cone&author=W%20Darwin&author=C%20Gorodetzky&author=T%20Tan&volume=6&publication_year=1978&pages=488-493&pmid=28931&)

3. Otton SV, Schadel M, Cheung SW, Kaplan HL, Busto UE, Sellers EM. CYP2D6 phenotype determines the metabolic conversion of hydrocodone to hydromorphone. Clin Pharmacol Ther. 1993;54:463–472. doi: 10.1038/clpt.1993.177.  [DOI](https://doi.org/10.1038/clpt.1993.177) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7693389/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=CYP2D6%20phenotype%20determines%20the%20metabolic%20conversion%20of%20hydrocodone%20to%20hydromorphone&author=SV%20Otton&author=M%20Schadel&author=SW%20Cheung&author=HL%20Kaplan&author=UE%20Busto&volume=54&publication_year=1993&pages=463-472&pmid=7693389&doi=10.1038/clpt.1993.177&)

4. Chen ZR, Irvine RJ, Somogyi AA, Bochner F. Mu receptor binding of some commonly used opioids and their metabolites. Life Sci. 1991;48:2165–2171. doi: 10.1016/0024-3205(91)90150-a.  [DOI](https://doi.org/10.1016/0024-3205(91)90150-a) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1851921/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Life%20Sci&title=Mu%20receptor%20binding%20of%20some%20commonly%20used%20opioids%20and%20their%20metabolites&author=ZR%20Chen&author=RJ%20Irvine&author=AA%20Somogyi&author=F%20Bochner&volume=48&publication_year=1991&pages=2165-2171&pmid=1851921&doi=10.1016/0024-3205(91)90150-a&)

5. Hennies HH, Friderichs E, Schneider J. Receptor binding, analgesic and antitussive potency of tramadol and other selected opioids. Arzneimittelforschung. 1988;38:877–880.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/2849950/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arzneimittelforschung&title=Receptor%20binding,%20analgesic%20and%20antitussive%20potency%20of%20tramadol%20and%20other%20selected%20opioids&author=HH%20Hennies&author=E%20Friderichs&author=J%20Schneider&volume=38&publication_year=1988&pages=877-880&pmid=2849950&)

6. Kaplan HL, Busto UE, Baylon GJ, Cheung SW, Otton SV, Somer G, et al. Inhibition of cytochrome P450 2D6 metabolism of hydrocodone to hydromorphone does not importantly affect abuse liability. J Pharmacol Exp Ther. 1997;281:103–108.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/9103485/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacol%20Exp%20Ther&title=Inhibition%20of%20cytochrome%20P450%202D6%20metabolism%20of%20hydrocodone%20to%20hydromorphone%20does%20not%20importantly%20affect%20abuse%20liability&author=HL%20Kaplan&author=UE%20Busto&author=GJ%20Baylon&author=SW%20Cheung&author=SV%20Otton&volume=281&publication_year=1997&pages=103-108&pmid=9103485&)

7. Yue Q, Hasselstrom J, Svensson J, Sawe J. Pharmacokinetics of codeine and its metabolites in Caucasian healthy volunteers. comparison between extensive and poor hydroxylators of debrisoquine. Br J Clin Pharmacol. 1991;31:635–642. doi: 10.1111/j.1365-2125.1991.tb05585.x.  [DOI](https://doi.org/10.1111/j.1365-2125.1991.tb05585.x) | [PMC free article](/articles/PMC1368572/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1867957/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=Pharmacokinetics%20of%20codeine%20and%20its%20metabolites%20in%20Caucasian%20healthy%20volunteers.%20comparison%20between%20extensive%20and%20poor%20hydroxylators%20of%20debrisoquine&author=Q%20Yue&author=J%20Hasselstrom&author=J%20Svensson&author=J%20Sawe&volume=31&publication_year=1991&pages=635-642&pmid=1867957&doi=10.1111/j.1365-2125.1991.tb05585.x&)

8. Quinn I. Adelaide: University of Adelaide; 2000. Identification of CYP450 isoforms involved in the metabolism of oxycodone [Masters of Medical Science Thesis]  [Google Scholar](https://scholar.google.com/scholar_lookup?Quinn%20I.%20Adelaide:%20University%20of%20Adelaide;%202000.%20Identification%20of%20CYP450%20isoforms%20involved%20in%20the%20metabolism%20of%20oxycodone%20%5BMasters%20of%20Medical%20Science%20Thesis%5D)

9. Kirkwood L, Nation R, Somogyi A. Characterisation of the human cytochrome P450 enzymes involved in the metabolism of dihydrocodeine. Br J Clin Pharmacol. 1997;44:549–555. doi: 10.1046/j.1365-2125.1997.t01-1-00626.x.  [DOI](https://doi.org/10.1046/j.1365-2125.1997.t01-1-00626.x) | [PMC free article](/articles/PMC2042888/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9431830/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=Characterisation%20of%20the%20human%20cytochrome%20P450%20enzymes%20involved%20in%20the%20metabolism%20of%20dihydrocodeine&author=L%20Kirkwood&author=R%20Nation&author=A%20Somogyi&volume=44&publication_year=1997&pages=549-555&pmid=9431830&doi=10.1046/j.1365-2125.1997.t01-1-00626.x&)

10. Daly A, Brockmoller J, Broly F, Eichelbaum M, Evans W, Gonzalez F, et al. Nomenclature for human CYP2D6 alleles. Pharmacogenetics. 1996;6:193–201. doi: 10.1097/00008571-199606000-00001.  [DOI](https://doi.org/10.1097/00008571-199606000-00001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8807658/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Nomenclature%20for%20human%20CYP2D6%20alleles&author=A%20Daly&author=J%20Brockmoller&author=F%20Broly&author=M%20Eichelbaum&author=W%20Evans&volume=6&publication_year=1996&pages=193-201&pmid=8807658&doi=10.1097/00008571-199606000-00001&)

11. Kroemer HK, Eichelbaum M. ‘It's the genes, stupid’. Molecular bases and clinical consequences of genetic cytochrome P450 2D6 polymorphism. Life Sci. 1995;56:2285–2298. doi: 10.1016/0024-3205(95)00223-s.  [DOI](https://doi.org/10.1016/0024-3205(95)00223-s) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7791516/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Life%20Sci&title=%E2%80%98It's%20the%20genes,%20stupid%E2%80%99.%20Molecular%20bases%20and%20clinical%20consequences%20of%20genetic%20cytochrome%20P450%202D6%20polymorphism&author=HK%20Kroemer&author=M%20Eichelbaum&volume=56&publication_year=1995&pages=2285-2298&pmid=7791516&doi=10.1016/0024-3205(95)00223-s&)

12. Obach RS. Nonspecific binding to microsomes – impact on scale-up of in vitro intrinsic clearance to hepatic clearance as assessed through examination of warfarin, imipramine, and propranolol. Drug Metab Dispos. 1997;24:1047–1049.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/9394025/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Nonspecific%20binding%20to%20microsomes%20%E2%80%93%20impact%20on%20scale-up%20of%20in%20vitro%20intrinsic%20clearance%20to%20hepatic%20clearance%20as%20assessed%20through%20examination%20of%20warfarin,%20imipramine,%20and%20propranolol&author=RS%20Obach&volume=24&publication_year=1997&pages=1047-1049&pmid=9394025&)

13. Obach RS, Baxter JG, Liston TE, Silber BM, Jones BC, Macintyre F. The prediction of human pharmacokinetic parameters from preclinical and in vitro metabolism data. J Pharmacol Exp Ther. 1997;283:46–58.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/9336307/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacol%20Exp%20Ther&title=The%20prediction%20of%20human%20pharmacokinetic%20parameters%20from%20preclinical%20and%20in%20vitro%20metabolism%20data&author=RS%20Obach&author=JG%20Baxter&author=TE%20Liston&author=BM%20Silber&author=BC%20Jones&volume=283&publication_year=1997&pages=46-58&pmid=9336307&)

14. McLure JA, Miners JO, Birkett DJ. Nonspecific binding of drugs to human liver microsomes. Br J Clin Pharmacol. 2000;49:453–461. doi: 10.1046/j.1365-2125.2000.00193.x.  [DOI](https://doi.org/10.1046/j.1365-2125.2000.00193.x) | [PMC free article](/articles/PMC2014956/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10792203/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=Nonspecific%20binding%20of%20drugs%20to%20human%20liver%20microsomes&author=JA%20McLure&author=JO%20Miners&author=DJ%20Birkett&volume=49&publication_year=2000&pages=453-461&pmid=10792203&doi=10.1046/j.1365-2125.2000.00193.x&)

15. Foster DJ, Somogyi AA, Bochner F. Methadone N-demethylation in human liver microsomes: lack of stereoselectivity and involvement of CYP3A4. Br J Clin Pharmacol. 1999;47:403–412. doi: 10.1046/j.1365-2125.1999.00921.x.  [DOI](https://doi.org/10.1046/j.1365-2125.1999.00921.x) | [PMC free article](/articles/PMC2014231/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10233205/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=Methadone%20N-demethylation%20in%20human%20liver%20microsomes:%20lack%20of%20stereoselectivity%20and%20involvement%20of%20CYP3A4&author=DJ%20Foster&author=AA%20Somogyi&author=F%20Bochner&volume=47&publication_year=1999&pages=403-412&pmid=10233205&doi=10.1046/j.1365-2125.1999.00921.x&)

16. Coller J, Somogyi A, Bochner F. Comparison of (S)-mephenytoin and proguanil oxidation in vitro: contribution of several CYP isoforms. Br J Clin Pharmacol. 1999;48:158–167. doi: 10.1046/j.1365-2125.1999.00005.x.  [DOI](https://doi.org/10.1046/j.1365-2125.1999.00005.x) | [PMC free article](/articles/PMC2014289/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10417492/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=Comparison%20of%20(S)-mephenytoin%20and%20proguanil%20oxidation%20in%20vitro:%20contribution%20of%20several%20CYP%20isoforms&author=J%20Coller&author=A%20Somogyi&author=F%20Bochner&volume=48&publication_year=1999&pages=158-167&pmid=10417492&doi=10.1046/j.1365-2125.1999.00005.x&)

17. Coller JK, Somogyi AA, Bochner F. Flunitrazepam oxidative metabolism in human liver microsomes: involvement of CYP2C19 and CYP3A4. Xenobiotica. 1999;29:973–986. doi: 10.1080/004982599238056.  [DOI](https://doi.org/10.1080/004982599238056) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10574680/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Xenobiotica&title=Flunitrazepam%20oxidative%20metabolism%20in%20human%20liver%20microsomes:%20involvement%20of%20CYP2C19%20and%20CYP3A4&author=JK%20Coller&author=AA%20Somogyi&author=F%20Bochner&volume=29&publication_year=1999&pages=973-986&pmid=10574680&doi=10.1080/004982599238056&)

18. Zanger U, Vilbois F, Hardwick J, Meyer U. Absence of hepatic cytochrome P450bufI causes genetically deficient debrisoquine in man. Biochemistry. 1988;27:5447–5454. doi: 10.1021/bi00415a010.  [DOI](https://doi.org/10.1021/bi00415a010) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/3052575/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochemistry&title=Absence%20of%20hepatic%20cytochrome%20P450bufI%20causes%20genetically%20deficient%20debrisoquine%20in%20man&author=U%20Zanger&author=F%20Vilbois&author=J%20Hardwick&author=U%20Meyer&volume=27&publication_year=1988&pages=5447-5454&pmid=3052575&doi=10.1021/bi00415a010&)

19. Lowry O, Rosebrough N, Farr A, Randall R. Protein measurement with folin phenol reagent. J Biol Chem. 1951;193:265–275.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/14907713/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Protein%20measurement%20with%20folin%20phenol%20reagent&author=O%20Lowry&author=N%20Rosebrough&author=A%20Farr&author=R%20Randall&volume=193&publication_year=1951&pages=265-275&pmid=14907713&)

20. Omura T, Sato R. The carbon monoxide-binding pigment of liver microsomes, II Solubilisation, purification and properties. J Biol Chem. 1964;234:2374–2385.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/14209972/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=The%20carbon%20monoxide-binding%20pigment%20of%20liver%20microsomes,%20II%20Solubilisation,%20purification%20and%20properties&author=T%20Omura&author=R%20Sato&volume=234&publication_year=1964&pages=2374-2385&pmid=14209972&)

21. Heim M, Meyer UA. Genotyping of poor metabolisers of debrisoquine by allele-specific PCR amplification. Lancet. 1990;336:529–532. doi: 10.1016/0140-6736(90)92086-w.  [DOI](https://doi.org/10.1016/0140-6736(90)92086-w) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1975039/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Genotyping%20of%20poor%20metabolisers%20of%20debrisoquine%20by%20allele-specific%20PCR%20amplification&author=M%20Heim&author=UA%20Meyer&volume=336&publication_year=1990&pages=529-532&pmid=1975039&doi=10.1016/0140-6736(90)92086-w&)

22. Coller JK, Somogyi AA, Bochner F. Association between CYP2C19 genotype and proguanil oxidative polymorphism. Br J Clin Pharmacol. 1997;43:659–660. doi: 10.1046/j.1365-2125.1997.00596.x.  [DOI](https://doi.org/10.1046/j.1365-2125.1997.00596.x) | [PMC free article](/articles/PMC2042790/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9205829/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=Association%20between%20CYP2C19%20genotype%20and%20proguanil%20oxidative%20polymorphism&author=JK%20Coller&author=AA%20Somogyi&author=F%20Bochner&volume=43&publication_year=1997&pages=659-660&pmid=9205829&doi=10.1046/j.1365-2125.1997.00596.x&)

23. Menelaou A, Hutchinson MR, Quinn I, Christensen A, Somogyi AA. Quantification of the O- and N-demethylated metabolites of hydrocodone and oxycodone in human liver microsomes using liquid chromatography with ultraviolet absorbance detection. J Chromatogr B. 2002;785:81–88. doi: 10.1016/s1570-0232(02)00856-5.  [DOI](https://doi.org/10.1016/s1570-0232(02)00856-5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12535841/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Chromatogr%20B&title=Quantification%20of%20the%20O-%20and%20N-demethylated%20metabolites%20of%20hydrocodone%20and%20oxycodone%20in%20human%20liver%20microsomes%20using%20liquid%20chromatography%20with%20ultraviolet%20absorbance%20detection&author=A%20Menelaou&author=MR%20Hutchinson&author=I%20Quinn&author=A%20Christensen&author=AA%20Somogyi&volume=785&publication_year=2002&pages=81-88&pmid=12535841&doi=10.1016/s1570-0232(02)00856-5&)

24. Newton D, Wang R, Lu A. Cytochrome P450 inhibitors: Evaluation of specificities in the in vitro metabolism of therapeutic agents by human liver microsomes. Drug Metab Dispos. 1995;23:154–158.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/7720520/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Cytochrome%20P450%20inhibitors:%20Evaluation%20of%20specificities%20in%20the%20in%20vitro%20metabolism%20of%20therapeutic%20agents%20by%20human%20liver%20microsomes&author=D%20Newton&author=R%20Wang&author=A%20Lu&volume=23&publication_year=1995&pages=154-158&pmid=7720520&)

25. Tucker GT, Houston JB, Huang SM. Optimizing drug development: strategies to assess drug metabolism/transporter interaction potential – towards a consensus. Br J Clin Pharmacol. 2001;52:107–117. doi: 10.1046/j.0306-5251.2001.temp.1441.x.  [DOI](https://doi.org/10.1046/j.0306-5251.2001.temp.1441.x) | [PMC free article](/articles/PMC2014497/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11453898/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=Optimizing%20drug%20development:%20strategies%20to%20assess%20drug%20metabolism/transporter%20interaction%20potential%20%E2%80%93%20towards%20a%20consensus&author=GT%20Tucker&author=JB%20Houston&author=SM%20Huang&volume=52&publication_year=2001&pages=107-117&pmid=11453898&doi=10.1046/j.0306-5251.2001.temp.1441.x&)

26. Baldwin S, Bloomer J, Smith G, Ayrton A, Clarke S, Chenery R. Ketoconazole and sulphaphenazole as the respective selective inhibitors of P4503A and 2C9. Xenobiotica. 1995;25:261–270. doi: 10.3109/00498259509061850.  [DOI](https://doi.org/10.3109/00498259509061850) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7618352/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Xenobiotica&title=Ketoconazole%20and%20sulphaphenazole%20as%20the%20respective%20selective%20inhibitors%20of%20P4503A%20and%202C9&author=S%20Baldwin&author=J%20Bloomer&author=G%20Smith&author=A%20Ayrton&author=S%20Clarke&volume=25&publication_year=1995&pages=261-270&pmid=7618352&doi=10.3109/00498259509061850&)

27. Hickman D, Wang J-P, Unadkat JD. Evaluation of the selectivity of in vitro probes and suitability of organic solvents for the measurement of human cytochrome P450 monooxygenase activities. Drug Metab Dispos. 1998;26:207–215.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/9492382/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Evaluation%20of%20the%20selectivity%20of%20in%20vitro%20probes%20and%20suitability%20of%20organic%20solvents%20for%20the%20measurement%20of%20human%20cytochrome%20P450%20monooxygenase%20activities&author=D%20Hickman&author=J-P%20Wang&author=JD%20Unadkat&volume=26&publication_year=1998&pages=207-215&pmid=9492382&)

28. Chauret N, Gauthier A, Nicoll-Griffith D. Effects of common organic solvents on in vitro cytochrome P450 – mediated metabolic activities in human liver microsomes. Drug Metab Dispos. 1998;26:1–4.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/9443844/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Effects%20of%20common%20organic%20solvents%20on%20in%20vitro%20cytochrome%20P450%20%E2%80%93%20mediated%20metabolic%20activities%20in%20human%20liver%20microsomes&author=N%20Chauret&author=A%20Gauthier&author=D%20Nicoll-Griffith&volume=26&publication_year=1998&pages=1-4&pmid=9443844&)

29. Goodman LS, Gilman A. The Pharmacological Basis of Therapeutics. 9. New York: McGraw-Hill; 1996.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=The%20Pharmacological%20Basis%20of%20Therapeutics&author=LS%20Goodman&author=A%20Gilman&publication_year=1996&)

30. Chen Z, Somogyi A, Bochner F. Polymorphic O-demethylation of codeine. Lancet. 1988;2:914–915. doi: 10.1016/s0140-6736(88)92529-9.  [DOI](https://doi.org/10.1016/s0140-6736(88)92529-9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2902373/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Polymorphic%20O-demethylation%20of%20codeine&author=Z%20Chen&author=A%20Somogyi&author=F%20Bochner&volume=2&publication_year=1988&pages=914-915&pmid=2902373&doi=10.1016/s0140-6736(88)92529-9&)

31. Caraco Y, Sheller J, Wood AJ. Pharmacogenetic determination of the effects of codeine and prediction of drug interactions. J Pharmacol Exp Ther. 1996;278:1165–1174.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/8819499/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacol%20Exp%20Ther&title=Pharmacogenetic%20determination%20of%20the%20effects%20of%20codeine%20and%20prediction%20of%20drug%20interactions&author=Y%20Caraco&author=J%20Sheller&author=AJ%20Wood&volume=278&publication_year=1996&pages=1165-1174&pmid=8819499&)

32. Park JI, Nakamura GR, Griesemer EC, Noguchi TT. Hydromorphone detected in bile following hydrocodone ingestion. J Forensic Sci. 1982;27:223–224.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/6178789/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Forensic%20Sci&title=Hydromorphone%20detected%20in%20bile%20following%20hydrocodone%20ingestion&author=JI%20Park&author=GR%20Nakamura&author=EC%20Griesemer&author=TT%20Noguchi&volume=27&publication_year=1982&pages=223-224&pmid=6178789&)

33. Zheng M, McErlane KM, Ong MC. Hydromorphone metabolites: isolation and identification from pooled urine samples of a cancer patient. Xenobiotica. 2002;32:427–439. doi: 10.1080/00498250110119090.  [DOI](https://doi.org/10.1080/00498250110119090) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12065064/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Xenobiotica&title=Hydromorphone%20metabolites:%20isolation%20and%20identification%20from%20pooled%20urine%20samples%20of%20a%20cancer%20patient&author=M%20Zheng&author=KM%20McErlane&author=MC%20Ong&volume=32&publication_year=2002&pages=427-439&pmid=12065064&doi=10.1080/00498250110119090&)

34. Lelas S, Wegert S, Otton SV, Sellers EM, France CP. Inhibitors of cytochrome P450 differentially modify discriminative- stimulus and antinociceptive effects of hydrocodone and hydromorphone in rhesus monkeys. Drug Alcohol Depend. 1999;54:239–249. doi: 10.1016/s0376-8716(98)00169-0.  [DOI](https://doi.org/10.1016/s0376-8716(98)00169-0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10372797/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Alcohol%20Depend&title=Inhibitors%20of%20cytochrome%20P450%20differentially%20modify%20discriminative-%20stimulus%20and%20antinociceptive%20effects%20of%20hydrocodone%20and%20hydromorphone%20in%20rhesus%20monkeys&author=S%20Lelas&author=S%20Wegert&author=SV%20Otton&author=EM%20Sellers&author=CP%20France&volume=54&publication_year=1999&pages=239-249&pmid=10372797&doi=10.1016/s0376-8716(98)00169-0&)

35. Sindrup SH, Brøsen K. The pharmacogenetics of codeine hypoalgesia. Pharmacogenetics. 1995;5:335–346. doi: 10.1097/00008571-199512000-00001.  [DOI](https://doi.org/10.1097/00008571-199512000-00001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8845855/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=The%20pharmacogenetics%20of%20codeine%20hypoalgesia&author=SH%20Sindrup&author=K%20Br%C3%B8sen&volume=5&publication_year=1995&pages=335-346&pmid=8845855&doi=10.1097/00008571-199512000-00001&)
