# CYP2D6 Phenotype and Breast Cancer Outcomes: A Bias Analysis and Meta-Analysis

## Metadata
**Authors:** Richard F MacLehose, Thomas P Ahern, Lindsay J Collin, Aixin Li, Timothy L Lash
**Journal:** Cancer epidemiology, biomarkers & prevention : a publication of the American Association for Cancer Research, cosponsored by the American Society of Preventive Oncology
**Date:** 2025 Feb 6
**DOI:** [10.1158/1055-9965.EPI-24-0755](https://doi.org/10.1158/1055-9965.EPI-24-0755)
**PMID:** 39540781
**PMCID:** PMC11802319
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11802319/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC11802319/pdf/nihms-2033140.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC11802319/pdf/nihms-2033140.pdf)

## Abstract

**Background:** 
We evaluated the impact of systematic bias due to loss of heterozygosity and incomplete phenotyping in studies examining the relationship between CYP2D6 variants and breast cancer recurrence among women treated with tamoxifen.

**Methods:** 
We performed a systematic review of the literature on tamoxifen, CYP2D6 variants, and breast cancer recurrence. A quantitative bias analysis was performed to adjust for loss of heterozygosity and incomplete phenotyping. Bias-adjusted results were then combined in a meta-analysis.

**Results:** 
Thirty-three studies informed the bias analysis and meta-analysis on CYP2D6 variants and breast cancer recurrence and/or mortality. An unadjusted meta-analysis suggested increased risk of recurrence and/or mortality for poor relative to normal metabolizers [relative risk (RR)=1.28; 95% simulation interval (SI) : 1.04 – 1.58] with substantial heterogeneity (I2=27%; p for heterogeneity=0.07). Adjusting for loss of heterozygosity and incomplete genotyping resulted in a slight change in the effect estimate and a decrease in heterogeneity (RR=1.34; 95% SI: 1.10 – 1.63; I2=0%; p for heterogeneity =0.17). Intermediate metabolizers had a slightly increased risk of recurrence and/or mortality relative to normal metabolizers (RR=1.15; 95%SI: 1.00 – 1.34; I2=0%; p for heterogeneity=0.89).

**Conclusions:** 
Adjusting for biases such as loss of heterozygosity and incomplete genotyping reduced observed heterogeneity between studies. Individuals with poor CYP2D6 phenotypes were at increased risk for breast cancer outcomes compared to those with normal phenotypes.

**Impact:** 
Reduction in CYP2D6 activity was associated with an increased risk of breast cancer recurrence and/or mortality, and results underscore the importance of quantitatively adjusting for biases when aggregating study results.

### Background:

We evaluated the impact of systematic bias due to loss of heterozygosity and incomplete phenotyping in studies examining the relationship between CYP2D6 variants and breast cancer recurrence among women treated with tamoxifen.

### Methods:

We performed a systematic review of the literature on tamoxifen, CYP2D6 variants, and breast cancer recurrence. A quantitative bias analysis was performed to adjust for loss of heterozygosity and incomplete phenotyping. Bias-adjusted results were then combined in a meta-analysis.

### Results:

Thirty-three studies informed the bias analysis and meta-analysis on CYP2D6 variants and breast cancer recurrence and/or mortality. An unadjusted meta-analysis suggested increased risk of recurrence and/or mortality for poor relative to normal metabolizers [relative risk (RR)=1.28; 95% simulation interval (SI) : 1.04 – 1.58] with substantial heterogeneity (I^2^=27%; p for heterogeneity=0.07). Adjusting for loss of heterozygosity and incomplete genotyping resulted in a slight change in the effect estimate and a decrease in heterogeneity (RR=1.34; 95% SI: 1.10 – 1.63; I^2^=0%; p for heterogeneity =0.17). Intermediate metabolizers had a slightly increased risk of recurrence and/or mortality relative to normal metabolizers (RR=1.15; 95%SI: 1.00 – 1.34; I^2^=0%; p for heterogeneity=0.89).

### Conclusions:

Adjusting for biases such as loss of heterozygosity and incomplete genotyping reduced observed heterogeneity between studies. Individuals with poor CYP2D6 phenotypes were at increased risk for breast cancer outcomes compared to those with normal phenotypes.

### Impact:

Reduction in CYP2D6 activity was associated with an increased risk of breast cancer recurrence and/or mortality, and results underscore the importance of quantitatively adjusting for biases when aggregating study results.

## Introduction

Tamoxifen has been used in the treatment of breast cancer in the United States since 1977 ([1](#R1)). Tamoxifen functions by binding to estrogen receptors, thereby preventing estrogen from binding to those same receptors ([2](#R2)). Deprived of estrogen, estrogen receptor (ER)-positive breast cancer cells cease to grow or grow more slowly. Tamoxifen treatment has been associated with a dramatic reduction in breast cancer recurrence among women with ER-positive, non-metastatic breast cancers ([3](#R3),[4](#R4)). As a result, tamoxifen therapy has become widely used in both pre- and postmenopausal women, although it is currently indicated primarily for premenopausal women ([5](#R5)).

The pharmacologic activity of tamoxifen is partially controlled by the cytochrome P-450 enzyme family ([4](#R4)). While tamoxifen itself can bind estrogen receptors, cytochrome P-450 2D6–encoded by the *CYP2D6* gene–metabolizes tamoxifen into endoxifen, which binds to estrogen receptors with 100-fold greater affinity than the parent molecule ([6](#R6)-[8](#R8)). The gene encoding the CYP2D6 enzyme has substantial variability, with over 100 allelic variants documented ([9](#R9),[10](#R10)). Variant *CYP2D6* alleles influence the enzyme’s ability to convert the less metabolically active tamoxifen into the more active endoxifen ([8](#R8)). Some alleles, such as *4 and *5, can completely eliminate metabolic activity of the CYP2D6 enzyme ([8](#R8)). Other alleles, such as *41, reduce the activity of the CYP2D6 enzyme ([8](#R8)), and copy number variants may increase its activity.

The presence of genetic variants that reduce phenotypic CYP2D6 activity has led to concern about the effectiveness of tamoxifen as a treatment for breast cancer patients who carry reduced or nonfunctional variant alleles ([11](#R11)). Approximately 40 studies have examined the association between *CYP2D6* variants and risk of recurrent breast cancer among women receiving tamoxifen therapy. Recent meta-analyses found remarkable heterogeneity in results, ranging from a strong protective effect of variant alleles (RR=0.28) to strong harmful effects (RR=13.1) ([12](#R12),[13](#R13)). Due in part to this heterogeneity, there is substantial controversy about whether *CYP2D6* variants increase the risk of breast cancer recurrence, as well as the clinical implications for patients who have *CYP2D6* variants of concern ([14](#R14)).

Two primary sources of bias have been proposed to explain these heterogenous results. The first is potential bias due to loss of heterozygosity, a not uncommon event in the course of carcinogenesis for some tumors ([15](#R15)). Many studies in the extant literature on breast cancer recurrence genotyped *CYP2D6* variants in DNA extracted from tumor tissue rather than from non-neoplastic tissue. Loss of heterozygosity in the tumor tissue could result in misclassification of the true (germline) *CYP2D6* genotype and result in a greater proportion of homozygotes in these studies. A recent meta-analysis by Ahern *et al* evaluated this hypothesis ([12](#R12)). A second source of bias is incomplete genotyping leading to phenotypic misclassification. With over 100 variant alleles, few studies have probed the genome deeply enough to completely characterize *CYP2D6* genotype (and inferred phenotype) of study participants. Many studies in persons of European origin look only at the most common knockout allele, *4. Absence of the *4 allele, however, does not necessarily mean an individual has wildtype *CYP2D6* activity since another knockout or reduced function allele could still be present. Similarly, many studies in persons of Asian origin only assayed the reduced function *10 allele. Algorithms exist to map the genotypic variants to phenotypic *CYP2D6* activity. However, individual studies measure varying sets of functional *CYP2D6* alleles, and most studies use different phenotype mapping algorithms, making results difficult to compare across studies ([16](#R16),[17](#R17)).

The aim of the current study is to estimate the effect of *CYP2D6* metabolic phenotype on breast cancer recurrence and/or mortality among women with ER-positive breast cancer who were treated with tamoxifen, accounting for these two potential sources of bias. We conducted an in-depth systematic review of the existing literature on *CYP2D6* and breast cancer recurrence and a quantitative bias analysis to adjust individual study results simultaneously for loss of heterozygosity and phenotypic misclassification. After adjusting for these biases, we conducted a meta-analysis to aggregate study results.

## Materials and Methods

We conducted a systematic literature review to identify published articles on the effect of *CYP2D6* variant alleles on breast cancer recurrence and/or mortality among tamoxifen-treated women. Studies were identified by a PubMed search. Exact search terms are reported in the online appendix and details have been previously described ([18](#R18)). Authors RM, TL, TA, and LC reviewed results of the search to determine whether the article contained information on *CYP2D6* genotype or phenotype and risk of recurrent breast cancer among tamoxifen-treated breast cancer patients.

Authors RM, TL, TA, and LC abstracted data from each research article; all abstracted data were reviewed a second time by RM. Study-specific information was abstracted for year of data collection, study design, length of follow-up, duration of tamoxifen therapy, and potential for immortal person-time bias. Outcome definition (recurrence, death, or composite recurrence/death) was recorded for each study. Primary analyses included studies with any of these three outcomes; however, in sensitivity analyses, we stratified by outcome definition. Studies were classified as having participants largely of European (North America or Europe, as well as 1 study in Brazil and 1 study in Syria) or Asian origin. We recorded summary information on study participant demographics (average age at diagnosis, percent postmenopausal) and clinical characteristics (adjuvant chemotherapy, tumor stage, and mastectomy) when reported. We also recorded the source of DNA (tumor vs non-neoplastic tissue), the set of *CYP2D6* variants assayed, and the phenotype mapping algorithm used, if any.

There were substantial differences in the effects estimated between studies. Many studies reported the effect of specific genotypes (e.g., *4/*4 vs wt/wt) on breast cancer recurrence, while others reported the effect of the imputed CYP2D6 phenotype on recurrence. This information was abstracted and recorded for the quantitative bias analysis and meta-analysis. When possible, we abstracted cell counts for genotype or phenotype by outcome status. Phenotypes were typically categorized as normal (NM), extensive (EM), intermediate (IM), or poor (PM) metabolizers. We treated NM and EM as equivalent. Phenotypic categories were defined differently by different authors across studies, and we recorded or derived the phenotypic algorithm used in each study when possible. For one study, we could not completely determine the phenotypic algorithm that was used and assumed it was the gold standard algorithm (defined below). We performed a sensitivity analysis excluding the study as well. Finally, some studies reported different collapsed categorizations of genotype or phenotype (e.g., “IM or NM” as the referent). The studies were used when it was possible to abstract cell counts, but studies that only reported summary effect estimates with different referent categories were excluded. Effect estimates and 95% confidence intervals were abstracted from studies, with preference given to unadjusted estimates when provided. When crude cell counts were available, we computed and analyzed risk ratios for consistency between studies. In studies where we could not abstract crude cell counts, we treated hazard ratios as equivalent to risk ratios. For one study with small cell sizes, we added 1 to each cell frequency to stabilize estimation ([19](#R19)).

We used probabilistic bias analysis methods to adjust individual studies for the two potential sources of bias due to misclassification (i.e., loss of heterozygosity and incomplete genotyping leading to phenotypic misclassification) ([20](#R20)). Adjusting for misclassification in a probabilistic bias analysis requires both study-specific data and values for bias parameters. Here, study-specific data are the genotype (or phenotype)-by-outcome contingency tables abstracted from each study. As there are two potential sources of bias in these studies, we require two sets of values for bias parameters: (1) sensitivity and specificity estimates that inform the amount of misclassification from loss of heterozygosity, and (2) sensitivity and specificity estimates for phenotypic misclassification. Because the sensitivities and specificities are not known with certainty, each is assigned a probability distribution, and the uncertainty in those parameters is incorporated into our final estimates. Probabilistic bias analysis repeatedly samples from distributions for the sensitivity and specificity parameters for each source of misclassification ([Figure 1](#F1)). The sampled sensitivity and specificity parameters are used to adjust the observed study-specific data for misclassification in sequence: first, adjusting for bias due to loss of heterozygosity (if necessary) and then adjusting for bias due to phenotype misclassification (if necessary). After adjusting the observed data for both sources of misclassification, we incorporate study-specific random error by random sampling, which we describe below. Repeating this procedure many times (10^5^ iterations) results in a distribution for the misclassification-adjusted effect of *CYP2D6* phenotype on breast cancer recurrence. Point estimates and lower and upper 95% simulation intervals are computed using the median, 2.5^th^, and 97.5th percentiles, respectively, of the distribution of bias-adjusted associations.

### Figure 1:

![Figure 1:](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b2ff/11802319/01e00809ee4f/nihms-2033140-f0001.jpg)

Flowchart describing the probabilistic bias analysis to adjust study-specific estimates for misclassification due to loss of heterozygosity and incomplete genotyping leading to phenotypic misclassification.

### Loss of Heterozygosity adjustment

Loss of heterozygosity can lead to genotypic (and, subsequently, phenotypic) misclassification of CYP2D6 activity when genotypes are assayed from tumor samples rather than in non-neoplastic tissue. For all studies that genotyped *CYP2D6* in tumor samples and provided genotype (or phenotype) by outcome contingency tables (N=3), we adjusted for misclassification due to loss of heterozygosity. Given the underlying mechanism of large-scale chromosomal rearrangement, we assumed phenotypes and the *10 allele data were misclassified due to loss of heterozygosity at the same rate as the *4 allele. Information on sensitivity and specificity due to loss of heterozygosity was available from six studies that evaluated the concordance of genotypes in paired tumor and non-neoplastic tissues from the same patients ([12](#R12)). The number of samples assayed in each concordance study ranged from 10 to 190 ([12](#R12)). Only 3 of the published studies provided contingency tables necessary to estimate the bias parameters ([21](#R21)-[23](#R23)). Data from these three studies were aggregated into a summary 3x3 contingency table from which we estimated the probability of a “misclassified” tumor-based genotype (variant/variant, variant/wild type, wild type/wild type) given a true non-neoplastic tissue-based genotype. In total, there are nine such probabilities, which are direct extensions of sensitivity and specificity to a three-category exposure.

The bias adjustment for loss of heterozygosity followed the steps laid out in Fox et al.([20](#R20))

#### Step 1) Specify bias parameters and distributions

Bias parameters necessary to adjust for loss of heterozygosity are categorical analogs of sensitivity and specificity. The probability of a tumor-derived *4 genotype given a benign tissue-derived *4 genotype is expressed as Pr(T=t∣B=b), where T is the tumor tissue-derived genotype, and B is the benign tissue-derived genotype. Both t and b can take on the values of 1=vv (*4 variant/ *4 variant), 2=wv (wildtype/*4 variant), or 3=ww (wild type/wild type). These 9 bias parameters were informed from data described in the previous paragraph on concordance between *4 allele status in tumor and non-tumor derived samples. Because these bias parameters were not known with certainty, we specified bias parameter distributions for them. A common approach is to use beta distributions to express uncertainty in sensitivity or specificity parameters. However, this specification did not suffice since our genotype exposure had >2 levels. Instead, we specified Dirichlet distributions for the bias parameters. The Dirichlet distribution is an extension of a beta distribution to more than 2 levels.

| [Pr(T=1∣B=b),Pr(T=2∣B=b),Pr(T=3∣B=b)]∼Dirichlet(αbvv,αbvw,αbww) |
| --- |
The parameters of the Dirichlet distribution, αbvv, αbvw, αbww, were the number of people in the validation studies who had benign genotype b and were found to have tumor genotype vv, vw, or ww. We added 1 to each cell of the validation data to stabilize estimates. Each draw from the Dirichlet distribution ensured that ∑tPr(T=t∣B=b)=1.

#### Step 2) Sample bias parameters

We drew random samples of the classification probabilities from the Dirichlet distribution above. This involved three sets of random samples, one for each non-neoplastic tissue genotype.

#### Step 3) Estimate the expected genotype frequency in benign tissue

The nine randomly sampled classification probabilities in step 2 were combined in a 3x3 matrix P, with individual elements Pr(T=t∣B=b). The expected non-neoplastic tissue cell counts were computed using the matrix method, P−1Oc=Ec. Here, Oc is a vector of length three that contains the observed number of vv, vw, and ww genotypes from neoplastic tissue in a specific study for cases (c=1) or controls (c=0). Cases refer to those with the outcome (recurrence and /or death), while controls refer to those without the outcome. The vector Ec=(vvce,vwce,wwce) contains the expected frequencies that would have been observed in non-neoplastic tissue given the classification probabilities in P. This process was repeated separately for cases and controls.

#### Step 4) Sample the expected prevalence of each benign tissue type

The expected genotype frequencies in Step (3) were used as parameters in a Dirichlet distribution to sample the prevalence of benign tissue genotype:

| [Pr(B=1∣C=c),Pr(B=2∣C=c),Pr(B=3∣C=c)]∼Dirichlet(vvce,vwce,wwce) |
| --- |
#### Step 5) Compute positive and negative predictive values for reclassification

The sampled prevalences in Step (4) were combined, using Bayes theorem, with the non-differential classification probabilities sampled in Step (1) to compute the predictive values: Pr(B=b∣T=t,C=c).

#### Step 6) Sample the benign genotypes

The predictive values from Step (5) were used to sample the number of people (conditional on those with a given observed neoplastic tissue genotype) who would have had each of the three genotypes if a non-neoplastic tissue sample had been sampled instead:

| [vv1,cexp,vw1,cexp,ww1,cexp]∼Multinomial(pcvv,vvcobs)[vv2,cexp,vw2,cexp,ww2,cexp]∼Multinomial(pcvw,vwcobs)[vv3,cexp,vw3,cexp,ww3,cexp]∼Multinomial(pcww,wwcobs) |
| --- |
Here, pct=[Pr(B=vv∣T=t,C=c),Pr(B=vw∣T=t,C=c),Pr(B=ww∣T=t,C=c)]. The bias-adjusted cell counts are then:

| vvcexp=vv1,cexp+vv2,cexp+vv3,cexpvwcexp=vw1,cexp+vw2,cexp+vw3,cexpwwcexp=ww1,cexp+ww2,cexp+ww3,cexp |
| --- |
These resulting cell counts are the number of genotypes that would have been observed among cases and controls without loss of heterozygosity, assuming a valid bias model.

### Incomplete Genotype Adjustment

The second source of bias was misclassification of CYP2D6 phenotype due to incomplete genotyping. CYP2D6 phenotype algorithms map *CYP2D6* genotypes to an activity score (AS), which is then categorized as normal, intermediate, or poor metabolizer. We used the consensus algorithm developed by Caudle *et al*, combining the Clinical Pharmacogenetics Implementation Consortium and Dutch Pharmacogenetics Working Group, as our gold standard ([17](#R17)). This algorithm reduces the metabolic activity of the *10 allele relative to previous algorithms. Most published studies on CYP2D6 and breast cancer recurrence did not use this algorithm; many relied on *ad hoc* phenotype algorithms or an older version of the gold standard algorithm in which the *10 allele was considered more metabolically active ([16](#R16),[24](#R24)). Adjusting for misclassification due to incomplete genotyping and differences in phenotype algorithm required sensitivity and specificity estimates that relate the genotype to phenotype [e.g., Pr(genotype = *4/*4 ∣ gold standard phenotype= PM)], or that relate different phenotype algorithms to the gold standard algorithm [e.g. Pr(old phenotype algorithm = PM ∣ gold standard phenotype algorithm = PM)]. These probabilities required a contingency table of the gold standard phenotype by genotype or imperfect phenotype.

We used data from the 1000 Genomes Project to calculate gold standard by imperfect phenotype contingency tables (RRID: SCR_008801; ftp.1000genomes.ebi.ac.uk/vol1/ftp/data_collections/1000G_2504_high_coverage/working/20201028_3202_phased/). We calculated the gold standard phenotype for everyone in the 1000 Genomes data, based on common star alleles (*2a, *2b, *3, *4, *6, *8, *9, *10, *17, *29a, *29b, *41). Of note, some alleles were gene deletions, and the location of these variants was unavailable in 1000 Genomes; we used the adjacent genomic location instead. On each chromosome, the activity score is calculated as the minimum of the allele-specific activity scores. For instance, if a chromosome is *4 (activity score = 0) and *10 (activity score =0.25), they receive an activity score of 0. A final activity score was derived by adding the two chromosome-specific activity scores for everyone in the 1000 Genome Project, and activity scores were classified as Poor (AS=0), Intermediate (0<AS<1.25), or normal (AS≥1.25) metabolizers. For each study that provided phenotype by outcome contingency data, we replicated that study’s phenotypic algorithm in the 1000 Genomes data. This procedure allowed us to generate a unique “gold-standard by misclassified phenotype” contingency table for each study. To impute the missing activity score phenotype for studies that only had *4 genotypes, we treated the *4 allele as the phenotype. We calculated the conditional probability of having the misclassified phenotype (or genotype) conditional on the gold standard phenotype: Pr(PTm=pm∣PTg=pg), where PTm is the benign tissue misclassified phenotype or genotype (either directly as reported from the study, or as imputed in the previous step) and pm is one of three phenotypes (1=PM, 2=IM, 3=NM) or genotypes (1=v/v, 2=w/v, 3=w/w). PTg is the gold standard phenotype with pg one of three activity levels (1=PM, 2=IM, 3=NM). We used data from 1000 Genomes Project to estimate the required sensitivities and specificities and to conduct a probabilistic bias analysis for misclassification due to incomplete genotyping ([Figure 1](#F1), Step 2). Given substantial differences in allele frequencies, we conducted these analyses separately for Asian and European populations.

The steps for incomplete genotype bias adjustment follow the steps outlined for loss of heterozygosity:

#### Step 1) Specify bias parameters and distributions

We used data from the 1000 Genome Project, as described above to estimate the 9 analogs of sensitivity and specificity: Pr(PTm=pm∣PTg=pg). We assumed that these probabilities followed Dirichlet distributions:

| [Pr(PTm=1∣PTg=pg),Pr(PTm=2∣PTg=pg),Pr(PTm=3∣PTg=pg)]∼Dirichlet(αpg1,αpg2,αpg3) |
| --- |
The parameters of the Dirichlet distribution (αpgpm) are the number of people with gold standard phenotype pg and misclassified phenotype (or genotype) equal to pm calculated from the 1000 Genome Project data.

#### Step 2) Sample bias parameters

We sampled the sensitivity and specificity analogs from these Dirichlet distributions. Three sets of probability distributions were sampled from three Dirichlet distributions, one for each gold standard phenotype. This process resulted in nine classification probabilities, which were used to estimate the bias-adjusted phenotypes for cases and controls.

#### Steps 3 and 4) Estimate the expected phenotype frequency and prevalence

When implementing this procedure to estimate the expected phenotype in studies conducted in Asian populations, the 1000 Genome Project data categorized no one as PM. Instead of nine non-zero cells, there were six because the probability of truly being PM was 0 in these data. This resulted in a misclassification matrix that was not of full rank and an over-identified solution when imputing the expected number of gold standard phenotypes. We used a least-squares solution to arrive at a best (in the sense of minimizing squared error) solution.

The expected prevalences, conditional on outcome, Pr(PTg=pg∣C=c), were sampled from Dirichlet distributions based on the bias-adjusted phenotypes in a similar process to the one described above for loss of heterozygosity.

#### Step 5) Compute positive and negative predictive values

The sampled activity score prevalences were combined with sample classification probabilities, using Bayes theorem, to compute predictive values of the gold standard phenotype given the observed, misclassified, genotype or phenotype, Pr(PTg=pg∣PTm=pm).

#### Step 6) Sample the bias-adjusted phenotypes

The positive and negative predictive values from step (5) were used to sample the number of people with each gold standard phenotype. These counts were the final estimated number of people with each phenotype (by outcome category), adjusted for loss of heterogeneity and incomplete genotyping. These adjusted counts were used to calculate bias-adjusted risk ratios.

#### Step 7) Incorporate random error

To incorporate conventional standard error, new log risk ratios were sampled from normal distributions centered on the bias-adjusted log risk ratio from step (6) with a standard error calculated from the bias-adjusted cell counts in step (6). This entire process was repeated 100,000 iterations. Summary point estimates were obtained as the median risk ratio of all 100,000 iterations. Simulation intervals were estimated as the 2.5^th^ and 97.5^th^ quantiles.

Twelve studies identified in the systematic review did not provide phenotype or genotype cell counts by outcome. For these studies, we could only abstract effect estimates and confidence intervals. Among those studies that were conducted using tumor data, we assumed the amount of bias due to loss of heterozygosity was the same as among studies for which we could abstract data on cell counts. For studies with cell count data, we calculated the difference in point estimates and variances between the crude and bias-adjusted effect estimates on the log scale. The crude estimate in these studies was calculated from crude cell count data, and for studies in Asian populations, we categorized data such that individuals with a *10/*1 genotype were in the referent (NM) category, consistent with Caudle et al ([17](#R17)). Inverse variance weighted averages of the bias and variance were calculated separately for studies conducted primarily in Asian and European populations. These estimates were used to adjust for misclassification in the probabilistic bias analysis by adding the average bias and increase in variance to the observed log effect measure and variance for those studies that provided only effect estimates.

Bias adjusted point estimates and simulation intervals were then combined in a random effects meta-analysis, using restricted maximum likelihood. We calculated p-values for tests of heterogeneity and estimated I^2^, a measure of heterogeneity between studies. Meta analyses were conducted separately by geographic region. We estimated the effect of PM phenotype vs NM phenotype among European populations, IM phenotype vs NM phenotype among European populations, IM phenotype vs NM phenotype among Asian populations, and combined IM or PM phenotype vs NM phenotype among European populations. We excluded studies among Asian populations in analyses involving PM because of the extreme rarity of that phenotype in abstracted study data and in the 1000 Genomes data. For studies in primarily Asian populations, all but 1 study used non-neoplastic DNA sources, so we do not present meta-analysis results for LOH adjusted for studies in primarily Asian populations.

We examined whether abstracted factors impacted observed heterogeneity or phenotype effects by fitting meta regression models adjusting for study characteristics. Given inconsistent reporting, we limited meta regression analyses to the following study characteristics: age at diagnosis, year data collection began, percent postmenopausal, median years of follow-up, study design (cohort vs case-control), source of DNA data (tumor or non-neoplastic), and possible immortal person time bias. As a sensitivity analysis, we repeated analyses separately based on outcome definition (recurrence, death, and recurrence/death). We conducted both Begg and Egger tests for evidence of publication bias. Code to replicate analyses is available at [https://github.com/macl0029/cyp2d6bias](https://github.com/macl0029/cyp2d6bias). All analyses were conducted in R v4.4.0 and meta- analyses were conducted in Stata v18.

### Data availability:

Data were generated by the authors and included in the article. Data spreadsheets and code are available at [https://github.com/macl0029/cyp2d6bias](https://github.com/macl0029/cyp2d6bias).

## Results

We identified 39 studies that provided information on *CYP2D6* genotype or phenotype and risk of breast cancer recurrence or death. Two of these studies contained information on the same participants ([25](#R25),[26](#R26)). We retained the study that contained direct information on genotype counts by outcome ([26](#R26)). Five studies only reported effect estimates that used a combination of IM and NM (or *4/wt and wt/wt) as the referent category and were therefore excluded. This resulted in a total of 33 studies contributing to our quantitative bias analysis. When implementing the adjustment for misclassification, four studies with contingency table data (Wegman et al, Thompson et al, Chamnanphon et al, and Teh et al) were incompatible with the sensitivity and specificity estimates, resulting in a substantial proportion (>20%) of bias analysis iterations with impossible results. For these studies, we used their genotype/phenotype by outcome data to compute crude risk ratios and 95% confidence intervals. Bias adjustments were then computed using the summary estimates, as described above, ignoring the individual cell counts. [Supplementary Table S1](#SD1) presents results stratified by outcome type (recurrence alone, death, recurrence or death) and stratified by source of DNA (tumor vs non-neoplastic). Tests of heterogeneity by these factors were not statistically significant, so we did not stratify our main results.

Of the 33 total studies, 12 studies contributed individual genotype (or phenotype)-by-outcome cell counts, and 6 contributed summary effect estimates to the analysis of PM vs NM phenotype on risk of breast cancer outcomes ([Table 1](#T1); studies with “NA” in the cell counts provided only summary effects). All these studies were based in countries where the population was classified as largely of European origin. Few of the studies conducted in Asian populations reported any PM individuals and the 1000 Genomes database contained no PM individuals. As a result, studies from Asian populations were excluded from the PM vs NM analysis. A random effects meta-analysis of the misclassified study results produced a summary estimate of RR=1.28, 95% Simulation Intervals (SI): 1.04–1.58 ([Table 2](#T2); Forest plot and inverse normal plot [Supplementary Figures S1](#SD2) & [S2](#SD4)). A test of heterogeneity resulted in a p-value of 0.07, and the I^2^ statistic was 27%, indicating presence of substantial heterogeneity. After adjusting for loss of heterozygosity and incomplete genotyping, the RR was slightly further from the null, and simulation intervals were wider (RR=1.34, 95% SI: 1.10–1.63). However, the heterogeneity p-value had increased and the I^2^ statistic had fallen to 0%. In meta-regression analyses, no abstracted clinical, demographic, or study variables were meaningfully associated with a change in the effect estimate. Sensitivity analyses that excluded the studies with data incompatible with sensitivity and specificity parameters had only minor impacts on summary estimates ([Supplementary Figure S3](#SD5)). Sensitivity analyses limiting to only studies with cell count data also resulted in little change ([Supplementary Figure S4](#SD6))

### Table 1:

|   |   |   |   |   |   | Casesa |  | Non Casesa |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Author Name | Year | Regionb | TissueSamplec | Genotype orphenotype | Crude RR orHR (95% CI) | PM | NM | PM | NM |
| Abraham(31) | 2010 | Europe | Non | Genotype | 1.1 (0.8, 1.7) | 23 | 302 | 107 | 1648 |
| Argalacsova(32) | 2017 | Europe | Non | Phenotype | 1 (0.4, 2.3) | 5 | 31 | 21 | 130 |
| Bijl(33) | 2009 | Europe | Non | Genotype | 2.3 (1.2, 4.6) | 3 | 17 | 1 | 35 |
| Dezentje(34) | 2013 | Europe | Tumor | Genotype | 1.3 (0.5, 3.2) | 5 | 36 | 37 | 363 |
| He(35) | 2020 | Europe | Non | Phenotype | 1.9 (1.1, 3.3) | 15 | 48 | 98 | 645 |
| Lash(26) | 2011 | Europe | Tumor | Genotype | 1.4 (0.9, 2.3) | 41 | 299 | 30 | 308 |
| Markkula(36) | 2014 | Europe | Non | Phenotype | 0.5 (0.1, 3.8) | NA | NA | NA | NA |
| Mwinyi(37) | 2014 | Europe | Non | Phenotype | 1 (0.1, 7.1) | 1 | 5 | 5 | 25 |
| Regan(38) | 2012 | Europe | Tumor | Phenotype | 0.8 (0.4, 1.5) | 8 | 75 | 78 | 535 |
| Schroth(39) | 2007 | Europe | Non | Phenotype | 2.5 (1.1, 5.7) | 5 | 17 | 9 | 101 |
| Thompson(40) | 2011 | Europe | Tumor | Phenotype | 0.6 (0.2, 1.4) | NA | NA | NA | NA |
| Wegman(41) | 2007 | Europe | Tumor | Genotype | 0.3 (0.1, 1) | NA | NA | NA | NA |
| Goetz(42) | 2013 | Europe | Tumor | Phenotype | 2.5 (1, 5.7) | NA | NA | NA | NA |
| Gor(43) | 2010 | Europe | Non | Genotype | 0.9 (0.5, 1.6) | NA | NA | NA | NA |
| Rae(44) | 2012 | Europe | Tumor | Phenotype | 1 (0.5, 2.1) | NA | NA | NA | NA |
| De Ameida Melo(45) | 2016 | Europe | Non | Phenotype | 1.2 (0.1, 20) | 1 | 26 | 1 | 31 |
| Ismail Al-Khalil(19) | 2022 | Europe | Non | Phenotype | 1.4 (1, 2.2) | 4 | 45 | 0 | 33 |
| Rangel-Mendez(46) | 2020 | Europe | Non | Phenotype | 4.7 (1, 22.6) | 1 | 6 | 1 | 50 |

Table 1 Caption: Summary of studies examining poor vs normal CYP2D6 metabolizers included in the meta-analysis of CYP2D6 and breast cancer recurrence.

### Table 2:

|   | RR | 95% SimulationInterval | P-value forheterogeneity | I2 |
| --- | --- | --- | --- | --- |
| PMa vs NMa, European (N = 18) |  |  |  |  |
| Non-bias adjusted | 1.28 | (1.04 – 1.58) | 0.07 | 27% |
| Adjusted only for loss of heterozygosity | 1.37 | (1.11 – 1.68) | 0.27 | 9% |
| Adjusted for loss of heterozygosity and incomplete genotyping | 1.34 | (1.10 – 1.63) | 0.17 | 0% |
| IMa vs NMa, European (N = 16) |  |  |  |  |
| Non-bias adjusted | 1.12 | (1.01 – 1.24) | 0.76 | 0% |
| Adjusted only for loss of heterozygosity | 1.13 | (1.01 – 1.25) | 0.79 | 0% |
| Adjusted for loss of heterozygosity and incomplete genotyping | 1.15 | (1.00 – 1.34) | 0.89 | 0% |
| IMa vs NMa, Asian (N = 10)b |  |  |  |  |
| Non-bias adjusted | 2.29 | (1.40 – 3.74) | <0.01 | 78% |
| Adjusted only for loss of heterozygosity | NA | NA | NA | NA |
| Adjusted for loss of heterozygosity and incomplete genotyping | 2.21 | (1.30 – 3.75) | <0.01 | 73% |
| IM/PMa vs NMa, European (N=18) |  |  |  |  |
| Non-bias adjusted | 1.12 | (1.02 – 1.23) | 0.36 | 0% |
| Adjusted only for loss of heterozygosity | 1.13 | (1.03 – 1.25) | 0.39 | 0% |
| Adjusted for loss of heterozygosity and incomplete genotyping | 1.15 | (1.01 – 1.32) | 0.69 | 0% |

Table 2 Caption: Meta-analysis results of the association between CYP2D6 phenotype and breast cancer recurrence, adjusting for sources of misclassification.

Analyses of IM vs NM phenotype on breast cancer recurrence risk were initially conducted combining studies with European and Asian populations. However, we observed substantial heterogeneity when all studies were combined, so we subsequently stratified the analysis of IM vs NM by study region. Among studies in European populations ([Table 3](#T3); N=16 total and 12 with cell count data on phenotype/genotype by outcome and 4 with only summary effect estimates), the crude effect (RR=1.12; 95%SI: 1.01–1.24) was largely unchanged after adjusting for loss of heterozygosity and incomplete genotyping (RR=1.15; 95%SI: 1.00–1.34; [Supplementary Figure S5](#SD7) and [S6](#SD8)). Uncertainty in the point estimate increased after adjustment, due to incorporating uncertainty in the bias parameters. There was little evidence of heterogeneity before or after the bias adjustments. Among studies in Asian populations (N=10; four with data on genotype/phenotype-by-outcome data and six with summary effect estimates only), the crude effect was elevated before bias adjustment (RR=2.29; 95% SI: 1.40–3.74; [Table 2](#T2) and [Supplementary Figure S7](#SD9)) with an effect modestly closer to the null after bias adjustments (RR=2.21; 95% SI: 1.30–3.75). Heterogeneity of study-specific estimates remained substantial with I^2^=73%. In meta-regression analyses, no abstracted clinical, demographic, or study variables were meaningfully associated with a change in the effect estimate, except for publication year with some evidence of decreased magnitude of effects with increasing year. There was evidence of publication bias using the Begg test but not the Egger test.

### Table 3:

|   |   |   |   |   |   | Casesa |  | Non Casesa |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Author Name | Year | Regionb | TissueSamplec | Genotype orphenotype | Crude RR orHR (95% CI) | IM | NM | IM | NM |
| Abraham(31) | 2010 | Europe | Non | Genotype | 1 (0.9, 1.3) | 159 | 302 | 819 | 1648 |
| Argalacsova(32) | 2017 | Europe | Non | Phenotype | 1 (0.7, 1.6) | 28 | 31 | 112 | 130 |
| Bijl(33) | 2009 | Europe | Non | Genotype | 1.6 (0.9, 2.7) | 15 | 17 | 14 | 35 |
| Dezentje(34) | 2013 | Europe | Tumor | Genotype | 1 (0.6, 1.7) | 16 | 36 | 164 | 363 |
| He(35) | 2020 | Europe | Non | Phenotype | 1.2 (0.8, 1.8) | 42 | 48 | 461 | 645 |
| Lash(26) | 2011 | Europe | Tumor | Genotype | 1 (0.8, 1.3) | 154 | 299 | 159 | 308 |
| Markkula(36) | 2014 | Europe | Non | Phenotype | 1 (0.5, 2.1) | NA | NA | NA | NA |
| Mwinyi(37) | 2014 | Europe | Non | Phenotype | 0.9 (0.3, 2.4) | 9 | 5 | 53 | 25 |
| Regan(38) | 2012 | Europe | Tumor | Phenotype | 1.2 (0.8, 1.7) | 40 | 75 | 237 | 535 |
| Schroth(39) | 2007 | Europe | Non | Phenotype | 2 (1.1, 3.6) | 19 | 17 | 46 | 101 |
| Wegman(41) | 2007 | Europe | Tumor | Genotype | 1 (0.8, 1.4) | NA | NA | NA | NA |
| Goetz(42) | 2013 | Europe | Tumor | Phenotype | 1.5 (1, 2.3) | NA | NA | NA | NA |
| Gor(43) | 2010 | Europe | Non | Genotype | 1.1 (0.6, 2) | NA | NA | NA | NA |
| De Ameida Melo(45) | 2016 | Europe | Non | Phenotype | 1.9 (0.7, 5.4) | 13 | 26 | 8 | 31 |
| Ismail Al-Khalil(19) | 2022 | Europe | Non | Phenotype | 1 (0.7, 1.6) | 8 | 45 | 5 | 33 |
| Rangel-Mendez(46) | 2020 | Europe | Non | Phenotype | 1.4 (0.3, 6.3) | 2 | 6 | 11 | 50 |
| Chamnanphon(47) | 2013 | Asia | Non | Phenotype | 1.3 (0.8, 2) | NA | NA | NA | NA |
| Kiyotani(48) | 2010 | Asia | Non | Phenotype | 2.7 (1.6, 4.7) | 18 | 23 | 45 | 196 |
| Park(49) | 2011 | Asia | Non | Phenotype | 4.8 (1.6, 14.6) | 3 | 8 | 5 | 94 |
| Sirachainan(50) | 2012 | Asia | Non | Genotype | 1.7 (0.5, 6) | 11 | 9 | 8 | 11 |
| Teh(51) | 2012 | Asia | Non | Phenotype | 4.5 (1.7, 11.7) | NA | NA | NA | NA |
| Damodaran(52) | 2012 | Asia | Non | Phenotype | 7.2 (1.8, 28.9) | NA | NA | NA | NA |
| Lan(53) | 2018 | Asia | Non | Genotype | 0.8 (0.6, 1) | 39 | 180 | 32 | 74 |
| Lei(54) | 2016 | Asia | Non | Genotype | 1.9 (0.4, 9.5) | NA | NA | NA | NA |
| Sukasem(55) | 2012 | Asia | Non | Genotype | 6.8 (1.5, 31.7) | NA | NA | NA | NA |
| Zhang(56) | 2015 | Asia | Tumor | Genotype | 1.9 (0.9, 4.2) | NA | NA | NA | NA |

Table 3 Caption: Summary of studies examining intermediate vs normal CYP2D6 metabolizers included in the meta analysis of CYP2D6 and breast cancer recurrence.

Eighteen studies ([Table 4](#T4)) contributed to estimates of IM or PM phenotype vs NM phenotype on risk of breast cancer recurrence. Only studies conducted in European populations were included to reduce effect estimate heterogeneity, since studies in Asian populations had few or no PM individuals. Of the included studies, 12 included phenotype/genotype by outcome cell counts, while 6 contained only summary study effect estimates. A random effects meta-analysis of crude estimates ([Table 2](#T2)) yielded evidence of modest increased recurrence risk with IM/PM phenotype, relative to NM phenotype (RR=1.12; 95% SI: 1.02-1.23). After bias adjustment, the meta-estimate was largely unchanged (RR=1.15; 95%CI: 1.01–1.32; [Supplementary Figure S8](#SD10)) with little evidence of heterogeneity. In meta-regression analyses, no abstracted clinical, demographic, or study variables were meaningfully associated with a change in the effect estimate. In tests for publication bias, the Begg test was statistically significant, but the Egger test was not.

### Table 4:

|   |   |   |   |   |   | Casesa |  | Non Casesa |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Author Name | Year | Regionb | TissueSamplec | Genotype orphenotype | Crude RR orHR (95% CI) | PM/IM | NM | PM/IM | NM |
| Abraham(31) | 2010 | Europe | Non | Genotype | 1.1 (0.9, 1.3) | 182 | 302 | 926 | 1648 |
| Argalacsova(32) | 2017 | Europe | Non | Phenotype | 1 (0.7, 1.6) | 33 | 31 | 133 | 130 |
| Bijl(33) | 2009 | Europe | Non | Genotype | 1.7 (1, 2.7) | 18 | 17 | 15 | 35 |
| Dezentje(34) | 2013 | Europe | Tumor | Genotype | 1 (0.6, 1.8) | 21 | 36 | 201 | 363 |
| He(46) | 2020 | Europe | Non | Phenotype | 1.3 (0.9, 1.9) | 57 | 48 | 559 | 645 |
| Lash(26) | 2011 | Europe | Tumor | Genotype | 1.1 (0.8, 1.4) | 195 | 299 | 189 | 308 |
| Mwinyi(37) | 2014 | Europe | Non | Phenotype | 0.9 (0.3, 2.4) | 10 | 5 | 58 | 25 |
| Regan(38) | 2012 | Europe | Tumor | Phenotype | 1.1 (0.8, 1.5) | 48 | 75 | 315 | 535 |
| Schroth(39) | 2007 | Europe | Non | Phenotype | 2.1 (1.2, 3.7) | 24 | 17 | 55 | 101 |
| Thompson | 2011 | Europe | Tumor | Phenotype | 1.0 (0.7, 1.3) | NA | NA | NA | NA |
| Johansson(57) | 2016 | Europe | Non | Phenotype | 2.1 (1, 4.5) | NA | NA | NA | NA |
| Mayer(58) | 2019 | Europe | Non | Phenotype | 1.4 (0.7, 2.8) | NA | NA | NA | NA |
| Morrow(59) | 2012 | Europe | Non | Phenotype | 1 (0.3, 3.3) | NA | NA | NA | NA |
| Nowell(60) | 2005 | Europe | Non | Genotype | 0.6 (0.3, 1.1) | NA | NA | NA | NA |
| Sanchez-Spitman(61) | 2019 | Europe | Non | Phenotype | 1 (0.6, 1.8) | NA | NA | NA | NA |
| De Ameida Melo(45) | 2016 | Europe | Non | Phenotype | 1.9 (0.7, 5) | 14 | 26 | 9 | 31 |
| Ismail Al-Khalil(19) | 2022 | Europe | Non | Phenotype | 1.2 (0.8, 1.7) | 12 | 45 | 5 | 33 |
| Rangel-Mendez(46) | 2020 | Europe | Non | Phenotype | 1.9 (0.5, 6.6) | 3 | 6 | 12 | 50 |

Table 4 Caption: Summary of studies examining poor or intermediate vs normal CYP2D6 metabolizers included in the meta-analysis of CYP2D6 and breast cancer recurrence.

## Discussion

This study is the first to estimate the effect of *CYP2D6* variants on breast cancer recurrence and/or mortality, adjusting for both misclassification due to loss of heterozygosity and misclassification of phenotype due to incomplete genotyping. After conducting a probabilistic bias analysis on the results from 33 studies, we found evidence of an increased risk of recurrence and/or mortality for the PM (vs NM) CYP2D6 phenotype. The effect of IM (vs NM) CYP2D6 phenotype was modest among studies conducted in primarily European populations, but substantially elevated among studies conducted in primarily Asian populations. The literature on *CYP2D6* variants is notable for the substantial heterogeneity between studies. Studies measured CYP2D6 phenotype using different star alleles and assayed either tumor or non-neoplastic cells. In a literature with such diverse methods for measuring the exposure, heterogeneous results should be expected. Our most notable finding is that, after adjusting for biases due to loss of heterozygosity and incomplete genotyping, we observed substantial reductions in between-study heterogeneity.

Our results were consistent with a previous meta-analysis by Ahern et al that focused on loss of heterozygosity due to measuring genotypes in tumor-derived DNA ([12](#R12)). That study was focused on the effect of the *4 allele on breast cancer recurrence. Since *4 is the most common knockout allele in European populations and leads to classification as a poor metabolizer, it is not surprising that our results examining adjustment for loss of heterozygosity are in alignment with Ahern et al’s results on *4 allele alone. Studies that assessed tumor vs non-neoplastic cells for loss of heterozygosity exhibited some heterogeneity, with one study showing far lower concordance than the other two (see Table 2 in [Ahern et al 2017](#R12)). Sensitivity analyses that implemented the bias adjustment separately for these two sets of validation data did not produce meaningfully different results.

The bias adjusted meta-analytic results from studies conducted in Asian populations are notable for their large effect sizes, reflecting the much larger crude effects on average in those studies. Individual study effects ranged from risk ratios of 1.4 to nearly 14, and all studies were notable for extreme levels of imprecision. Multiple articles have pointed out that effects of this magnitude are implausible ([12](#R12),[27](#R27)). Meta-analyses of trials of tamoxifen have shown that five years of tamoxifen approximately halves the risk of breast cancer recurrence among women whose tumors are estrogen receptor positive ([3](#R3)). If *CYP2D6* variants entirely eliminated the therapeutic benefit of tamoxifen, at most, individuals with those variants would have about double the risk of breast cancer recurrence ([27](#R27)). However, even this conservative assumption is untenable since, at most, complete knockout of *CYP2D6* would reduce formation of endoxifen, but would not eliminate its formation via other metabolic paths nor would it inhibit tamoxifen itself or its other metabolites from binding to estrogen receptors and exerting antagonism. Further, studies conducted in Asian populations generally had few or no individuals categorized as poor metabolizers, a consequence of the fact that the *10 allele prevalent in persons of Asian origin is a reduced function variant, not a knockout variant, such as the *4 allele prevalent in populations of European origin. Nonetheless, the meta-analysis effect estimate from Asian population studies is far larger than the meta-analysis effect estimate comparing poor metabolizers with normal metabolizers in European populations. For these reasons, we suggest many of the studies conducted in Asian populations may be subject to substantial bias other than those we have addressed quantitatively. Tests of publication bias were not consistent, with the Begg test finding a statistically significant result and the Egger test finding a non-significant result, suggesting that publication selection forces may not explain all of this pattern.

Our study has several limitations. First, we have only adjusted for exposure misclassification due to loss of heterozygosity and incomplete genotyping. Other biases exist in this literature for which we have not accounted. Of note, immortal person-time bias has been raised as a possibility in some of these studies ([28](#R28)). However, when we excluded studies with some evidence of immortal person time bias by design, results were largely unchanged. Second, our bias adjustment for misclassification in studies for which we only had summary effect estimates and not cell counts relied on an approximation based on the assumption that misclassification bias is linear on the log-scale. Third, our gold standard measure for phenotype relied on 12 of the many identified star alleles for *CYP2D6*; however, the remaining star alleles were far less common. When replicating phenotype categorizations utilized by individual studies, we restricted our analysis to only those 12 star alleles listed above. Fourth, our bias adjustments produced bias-adjusted risk ratios. Studies that did not have individual cell counts generally provided hazard ratios. We combined these risk ratios and hazard ratios in our meta-analyses, which could introduce bias. Fifth, studies that provided only summary effect estimates sometimes used more exclusive reference conditions than our gold standard. However, sensitivity analyses that restricted to only studies where it was possible to abstract contingency table data and estimate risk ratio were very similar to overall results ([Supplementary Figure S4](#SD6)).

Guidelines for the treatment of breast cancer currently do not recommend testing for *CYP2D6* genotype when determining the best course of treatment ([29](#R29),[30](#R30)). In part, the basis for this recommendation is due to the observed heterogeneity of study results. In this study, we have adjusted for two likely sources of that heterogeneity: misclassification of exposure status due to loss of heterozygosity and incomplete genotyping. The quantitative bias analysis conducted in this study reduced overall heterogeneity and yielded results consistent with the hypothesis that poor metabolizers are at slightly increased risk of recurrence. However, residual biases may still be impacting results in populations with implausibly high effect estimates.

## Supplementary Material

## Acknowledgements:

This work was supported by the National Library of Medicine (grant R01LM013049, awarded to T.L.L.). Lindsay J. Collin was supported by K99CA277580 from the National Cancer Institute of the National Institutes of Health.

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

Data were generated by the authors and included in the article. Data spreadsheets and code are available at [https://github.com/macl0029/cyp2d6bias](https://github.com/macl0029/cyp2d6bias).

### Supplementary Materials

### Data Availability Statement

Data were generated by the authors and included in the article. Data spreadsheets and code are available at [https://github.com/macl0029/cyp2d6bias](https://github.com/macl0029/cyp2d6bias).

## References

1. Osborne CK. Tamoxifen in the Treatment of Breast Cancer. Wood AJJ, editor. N Engl J Med. 1998;339:1609–18.  [DOI](https://doi.org/10.1056/NEJM199811263392207) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9828250/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&author=CK%20Osborne&author=AJJ%20Wood&volume=339&publication_year=1998&pages=1609-18&pmid=9828250&doi=10.1056/NEJM199811263392207&)

2. Jordan VC, Chem. C. Metabolites of tamoxifen in animals and man: Identification, pharmacology, and significance. Breast Cancer Res Tr. 1982;2:123–38.  [DOI](https://doi.org/10.1007/BF01806449) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/6184101/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Breast%20Cancer%20Res%20Tr&title=Metabolites%20of%20tamoxifen%20in%20animals%20and%20man:%20Identification,%20pharmacology,%20and%20significance&author=VC%20Jordan&author=C%20Chem.&volume=2&publication_year=1982&pages=123-38&pmid=6184101&doi=10.1007/BF01806449&)

3. Early Breat Cancer Trialists Collaborative Group. Effects of chemotherapy and hormonal therapy for early breast cancer on recurrence and 15-year survival: an overview of the randomised trials. Lancet. 2005;365:1687–717.  [DOI](https://doi.org/10.1016/S0140-6736(05)66544-0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15894097/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Effects%20of%20chemotherapy%20and%20hormonal%20therapy%20for%20early%20breast%20cancer%20on%20recurrence%20and%2015-year%20survival:%20an%20overview%20of%20the%20randomised%20trials&volume=365&publication_year=2005&pages=1687-717&pmid=15894097&doi=10.1016/S0140-6736(05)66544-0&)

4. Osborne CK, Elledge R, Fuqua SA. Estrogen Receptors in Breast Cancer Therapy: Blocking the action of estrogen receptors controls the growth of breast cancer. Scientific American Science and Medicine. 1996;3:32–41.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Scientific%20American%20Science%20and%20Medicine&title=Estrogen%20Receptors%20in%20Breast%20Cancer%20Therapy:%20Blocking%20the%20action%20of%20estrogen%20receptors%20controls%20the%20growth%20of%20breast%20cancer&author=CK%20Osborne&author=R%20Elledge&author=SA%20Fuqua&volume=3&publication_year=1996&pages=32-41&)

5. Burstein HJ, Griggs JJ, Prestrud AA, Temin S. American Society of Clinical Oncology Clinical Practice Guideline Update on Adjuvant Endocrine Therapy for Women With Hormone Receptor–Positive Breast Cancer. JOP. Wolters Kluwer; 2010;6:243–6.  [DOI](https://doi.org/10.1200/JOP.000082) | [PMC free article](/articles/PMC2936467/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21197188/) | [Google Scholar](https://scholar.google.com/scholar_lookup?title=JOP&author=HJ%20Burstein&author=JJ%20Griggs&author=AA%20Prestrud&author=S%20Temin&publication_year=2010&)

6. Johnson MD, Zuo H, Lee K-H, Trebley JP, Rae JM, Weatherman RV, et al. Pharmacological characterization of 4-hydroxy-N-desmethyl tamoxifen, a novel active metabolite of tamoxifen. Breast Cancer Res Treat. 2004;85:151–9.  [DOI](https://doi.org/10.1023/B:BREA.0000025406.31193.e8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15111773/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Breast%20Cancer%20Res%20Treat&title=Pharmacological%20characterization%20of%204-hydroxy-N-desmethyl%20tamoxifen,%20a%20novel%20active%20metabolite%20of%20tamoxifen&author=MD%20Johnson&author=H%20Zuo&author=K-H%20Lee&author=JP%20Trebley&author=JM%20Rae&volume=85&publication_year=2004&pages=151-9&pmid=15111773&doi=10.1023/B:BREA.0000025406.31193.e8&)

7. Lim YC, Li L, Desta Z, Zhao Q, Rae JM, Flockhart DA, et al. Endoxifen, a secondary metabolite of tamoxifen, and 4-OH-tamoxifen induce similar changes in global gene expression patterns in MCF-7 breast cancer cells. J Pharmacol Exp Ther. 2006;318:503–12.  [DOI](https://doi.org/10.1124/jpet.105.100511) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16690721/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacol%20Exp%20Ther&title=Endoxifen,%20a%20secondary%20metabolite%20of%20tamoxifen,%20and%204-OH-tamoxifen%20induce%20similar%20changes%20in%20global%20gene%20expression%20patterns%20in%20MCF-7%20breast%20cancer%20cells&author=YC%20Lim&author=L%20Li&author=Z%20Desta&author=Q%20Zhao&author=JM%20Rae&volume=318&publication_year=2006&pages=503-12&pmid=16690721&doi=10.1124/jpet.105.100511&)

8. Goetz MP, Kamal A, Ames MM. Tamoxifen pharmacogenomics: the role of CYP2D6 as a predictor of drug response. Clin Pharmacol Ther. 2008;83:160–6.  [DOI](https://doi.org/10.1038/sj.clpt.6100367) | [PMC free article](/articles/PMC2752373/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17882159/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Tamoxifen%20pharmacogenomics:%20the%20role%20of%20CYP2D6%20as%20a%20predictor%20of%20drug%20response&author=MP%20Goetz&author=A%20Kamal&author=MM%20Ames&volume=83&publication_year=2008&pages=160-6&pmid=17882159&doi=10.1038/sj.clpt.6100367&)

9. Taylor C, Crosby I, Yip V, Maguire P, Pirmohamed M, Turner RM. A Review of the Important Role of CYP2D6 in Pharmacogenomics. Genes. Multidisciplinary Digital Publishing Institute; 2020;11:1295.  [DOI](https://doi.org/10.3390/genes11111295) | [PMC free article](/articles/PMC7692531/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33143137/) | [Google Scholar](https://scholar.google.com/scholar_lookup?title=Genes&author=C%20Taylor&author=I%20Crosby&author=V%20Yip&author=P%20Maguire&author=M%20Pirmohamed&publication_year=2020&)

10. Nofziger C, Turner AJ, Sangkuhl K, Whirl-Carrillo M, Agúndez JAG, Black JL, et al. PharmVar GeneFocus: CYP2D6. Clinical Pharmacology & Therapeutics. 2020;107:154–70.  [DOI](https://doi.org/10.1002/cpt.1643) | [PMC free article](/articles/PMC6925641/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31544239/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clinical%20Pharmacology%20&%20Therapeutics&title=PharmVar%20GeneFocus:%20CYP2D6&author=C%20Nofziger&author=AJ%20Turner&author=K%20Sangkuhl&author=M%20Whirl-Carrillo&author=JAG%20Ag%C3%BAndez&volume=107&publication_year=2020&pages=154-70&pmid=31544239&doi=10.1002/cpt.1643&)

11. Desta Z, Flockhart DA. Germline pharmacogenetics of tamoxifen response: have we learned enough? J Clin Oncol. 2007;25:5147–9.  [DOI](https://doi.org/10.1200/JCO.2007.13.4957) | [PMC free article](/articles/PMC2764520/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18024859/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Germline%20pharmacogenetics%20of%20tamoxifen%20response:%20have%20we%20learned%20enough?&author=Z%20Desta&author=DA%20Flockhart&volume=25&publication_year=2007&pages=5147-9&pmid=18024859&doi=10.1200/JCO.2007.13.4957&)

12. Ahern TP, Hertz DL, Damkier P, Ejlertsen B, Hamilton-Dutoit SJ, Rae JM, et al. Cytochrome P-450 2D6 (CYP2D6) Genotype and Breast Cancer Recurrence in Tamoxifen-Treated Patients: Evaluating the Importance of Loss of Heterozygosity. Am J Epidemiol. 2017;185:75–85.  [DOI](https://doi.org/10.1093/aje/kww178) | [PMC free article](/articles/PMC5253974/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27988492/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Epidemiol&title=Cytochrome%20P-450%202D6%20(CYP2D6)%20Genotype%20and%20Breast%20Cancer%20Recurrence%20in%20Tamoxifen-Treated%20Patients:%20Evaluating%20the%20Importance%20of%20Loss%20of%20Heterozygosity&author=TP%20Ahern&author=DL%20Hertz&author=P%20Damkier&author=B%20Ejlertsen&author=SJ%20Hamilton-Dutoit&volume=185&publication_year=2017&pages=75-85&pmid=27988492&doi=10.1093/aje/kww178&)

13. Province MA, Goetz MP, Brauch H, Flockhart DA, Hebert JM, Whaley R, et al. CYP2D6 genotype and adjuvant tamoxifen: meta-analysis of heterogeneous study populations. Clin Pharmacol Ther. 2014;95:216–27.  [DOI](https://doi.org/10.1038/clpt.2013.186) | [PMC free article](/articles/PMC3904554/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24060820/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=CYP2D6%20genotype%20and%20adjuvant%20tamoxifen:%20meta-analysis%20of%20heterogeneous%20study%20populations&author=MA%20Province&author=MP%20Goetz&author=H%20Brauch&author=DA%20Flockhart&author=JM%20Hebert&volume=95&publication_year=2014&pages=216-27&pmid=24060820&doi=10.1038/clpt.2013.186&)

14. Lash TL, Ahern TP, Cronin-Fenton D, Garne JP, Hamilton-Dutoit S, Kvistgaard ME, et al. Modification of tamoxifen response: what have we learned? J Clin Oncol. 2008;26:1764–5; author reply 1765–1766.  [DOI](https://doi.org/10.1200/JCO.2007.15.5432) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18375910/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Modification%20of%20tamoxifen%20response:%20what%20have%20we%20learned?&author=TL%20Lash&author=TP%20Ahern&author=D%20Cronin-Fenton&author=JP%20Garne&author=S%20Hamilton-Dutoit&volume=26&publication_year=2008&pages=1764-5&pmid=18375910&doi=10.1200/JCO.2007.15.5432&)

15. Zhang X, Sjöblom T. Targeting Loss of Heterozygosity: A Novel Paradigm for Cancer Therapy. Pharmaceuticals (Basel). 2021;14:57.  [DOI](https://doi.org/10.3390/ph14010057) | [PMC free article](/articles/PMC7828287/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33450833/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmaceuticals%20(Basel)&title=Targeting%20Loss%20of%20Heterozygosity:%20A%20Novel%20Paradigm%20for%20Cancer%20Therapy&author=X%20Zhang&author=T%20Sj%C3%B6blom&volume=14&publication_year=2021&pages=57&pmid=33450833&doi=10.3390/ph14010057&)

16. Goetz MP, Sangkuhl K, Guchelaar H-J, Schwab M, Province M, Whirl-Carrillo M, et al. Clinical Pharmacogenetics Implementation Consortium (CPIC) Guideline for CYP2D6 and Tamoxifen Therapy. Clin Pharmacol Ther. 2018;103:770–7.  [DOI](https://doi.org/10.1002/cpt.1007) | [PMC free article](/articles/PMC5931215/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29385237/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Clinical%20Pharmacogenetics%20Implementation%20Consortium%20(CPIC)%20Guideline%20for%20CYP2D6%20and%20Tamoxifen%20Therapy&author=MP%20Goetz&author=K%20Sangkuhl&author=H-J%20Guchelaar&author=M%20Schwab&author=M%20Province&volume=103&publication_year=2018&pages=770-7&pmid=29385237&doi=10.1002/cpt.1007&)

17. Caudle KE, Sangkuhl K, Whirl-Carrillo M, Swen JJ, Haidar CE, Klein TE, et al. Standardizing CYP2D6 Genotype to Phenotype Translation: Consensus Recommendations from the Clinical Pharmacogenetics Implementation Consortium and Dutch Pharmacogenetics Working Group. Clin Transl Sci. 2020;13:116–24.  [DOI](https://doi.org/10.1111/cts.12692) | [PMC free article](/articles/PMC6951851/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31647186/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Transl%20Sci&title=Standardizing%20CYP2D6%20Genotype%20to%20Phenotype%20Translation:%20Consensus%20Recommendations%20from%20the%20Clinical%20Pharmacogenetics%20Implementation%20Consortium%20and%20Dutch%20Pharmacogenetics%20Working%20Group&author=KE%20Caudle&author=K%20Sangkuhl&author=M%20Whirl-Carrillo&author=JJ%20Swen&author=CE%20Haidar&volume=13&publication_year=2020&pages=116-24&pmid=31647186&doi=10.1111/cts.12692&)

18. Ahern TP, MacLehose RF, Haines L, Cronin-Fenton DP, Damkier P, Collin LJ, et al. Improving the transparency of meta-analyses with interactive web applications. BMJ Evidence-Based Medicine. Royal Society of Medicine; 2021;26:327–32.  [DOI](https://doi.org/10.1136/bmjebm-2019-111308) | [PMC free article](/articles/PMC7530078/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32220861/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMJ%20Evidence-Based%20Medicine.%20Royal%20Society%20of%20Medicine&title=Improving%20the%20transparency%20of%20meta-analyses%20with%20interactive%20web%20applications&author=TP%20Ahern&author=RF%20MacLehose&author=L%20Haines&author=DP%20Cronin-Fenton&author=P%20Damkier&volume=26&publication_year=2021&pages=327-32&pmid=32220861&doi=10.1136/bmjebm-2019-111308&)

19. Ismail Al-khalil W, Al-Salhi L, Rijjal S, Aljamali M, Youssef LA. The frequencies of CYP2D6 alleles and their impact on clinical outcomes of adjuvant tamoxifen therapy in Syrian breast cancer patients. BMC Cancer. 2022;22:1067.  [DOI](https://doi.org/10.1186/s12885-022-10148-8) | [PMC free article](/articles/PMC9571463/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36243690/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Cancer&title=The%20frequencies%20of%20CYP2D6%20alleles%20and%20their%20impact%20on%20clinical%20outcomes%20of%20adjuvant%20tamoxifen%20therapy%20in%20Syrian%20breast%20cancer%20patients&author=W%20Ismail%20Al-khalil&author=L%20Al-Salhi&author=S%20Rijjal&author=M%20Aljamali&author=LA%20Youssef&volume=22&publication_year=2022&pages=1067&pmid=36243690&doi=10.1186/s12885-022-10148-8&)

20. Fox MP, MacLehose RF, Lash TL. Applying Quantitative Bias Analysis to Epidemiologic Data. Cham: Springer International Publishing; 2021.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Applying%20Quantitative%20Bias%20Analysis%20to%20Epidemiologic%20Data&author=MP%20Fox&author=RF%20MacLehose&author=TL%20Lash&publication_year=2021&)

21. Ahern TP, Christensen M, Cronin-Fenton DP, Lunetta KL, Rosenberg CL, Sørensen HT, et al. Concordance of metabolic enzyme genotypes assayed from paraffin-embedded, formalin-fixed breast tumors and normal lymphatic tissue. Clin Epidemiol. 2010;2:241–6.  [DOI](https://doi.org/10.2147/CLEP.S13811) | [PMC free article](/articles/PMC2998813/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21152250/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Epidemiol&title=Concordance%20of%20metabolic%20enzyme%20genotypes%20assayed%20from%20paraffin-embedded,%20formalin-fixed%20breast%20tumors%20and%20normal%20lymphatic%20tissue&author=TP%20Ahern&author=M%20Christensen&author=DP%20Cronin-Fenton&author=KL%20Lunetta&author=CL%20Rosenberg&volume=2&publication_year=2010&pages=241-6&pmid=21152250&doi=10.2147/CLEP.S13811&)

22. Goetz MP, Sun JX, Suman VJ, Silva GO, Perou CM, Nakamura Y, et al. Loss of heterozygosity at the CYP2D6 locus in breast cancer: implications for germline pharmacogenetic studies. Journal of the National Cancer Institute. 2015;107.  [DOI](https://doi.org/10.1093/jnci/dju401) | [PMC free article](/articles/PMC4565524/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25490892/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20the%20National%20Cancer%20Institute&title=Loss%20of%20heterozygosity%20at%20the%20CYP2D6%20locus%20in%20breast%20cancer:%20implications%20for%20germline%20pharmacogenetic%20studies&author=MP%20Goetz&author=JX%20Sun&author=VJ%20Suman&author=GO%20Silva&author=CM%20Perou&volume=107&publication_year=2015&pmid=25490892&doi=10.1093/jnci/dju401&)

23. Rae JM, Cordero KE, Scheys JO, Lippman ME, Flockhart DA, Johnson MD. Genotyping for polymorphic drug metabolizing enzymes from paraffin-embedded and immunohistochemically stained tumor samples: Pharmacogenetics. 2003;13:501–7.  [DOI](https://doi.org/10.1097/00008571-200308000-00008) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12893988/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Genotyping%20for%20polymorphic%20drug%20metabolizing%20enzymes%20from%20paraffin-embedded%20and%20immunohistochemically%20stained%20tumor%20samples&author=JM%20Rae&author=KE%20Cordero&author=JO%20Scheys&author=ME%20Lippman&author=DA%20Flockhart&volume=13&publication_year=2003&pages=501-7&pmid=12893988&doi=10.1097/00008571-200308000-00008&)

24. Bank PCD, Caudle KE, Swen JJ, Gammal RS, Whirl-Carrillo M, Klein TE, et al. Comparison of the Guidelines of the Clinical Pharmacogenetics Implementation Consortium and the Dutch Pharmacogenetics Working Group. Clin Pharmacol Ther. 2018;103:599–618.  [DOI](https://doi.org/10.1002/cpt.762) | [PMC free article](/articles/PMC5723247/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28994452/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Comparison%20of%20the%20Guidelines%20of%20the%20Clinical%20Pharmacogenetics%20Implementation%20Consortium%20and%20the%20Dutch%20Pharmacogenetics%20Working%20Group&author=PCD%20Bank&author=KE%20Caudle&author=JJ%20Swen&author=RS%20Gammal&author=M%20Whirl-Carrillo&volume=103&publication_year=2018&pages=599-618&pmid=28994452&doi=10.1002/cpt.762&)

25. Ahern TP, Collin LJ, Baurley JW, Kjærsgaard A, Nash R, Maliniak ML, et al. Metabolic Pathway Analysis and Effectiveness of Tamoxifen in Danish Breast Cancer Patients. Cancer Epidemiol Biomarkers Prev. 2020;29:582–90.  [DOI](https://doi.org/10.1158/1055-9965.EPI-19-0833) | [PMC free article](/articles/PMC7060091/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31932415/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Epidemiol%20Biomarkers%20Prev&title=Metabolic%20Pathway%20Analysis%20and%20Effectiveness%20of%20Tamoxifen%20in%20Danish%20Breast%20Cancer%20Patients&author=TP%20Ahern&author=LJ%20Collin&author=JW%20Baurley&author=A%20Kj%C3%A6rsgaard&author=R%20Nash&volume=29&publication_year=2020&pages=582-90&pmid=31932415&doi=10.1158/1055-9965.EPI-19-0833&)

26. Lash TL, Cronin-Fenton D, Ahern TP, Rosenberg CL, Lunetta KL, Silliman RA, et al. CYP2D6 Inhibition and Breast Cancer Recurrence in a Population-Based Study in Denmark. J Natl Cancer Inst. 2011;103:489–500.  [DOI](https://doi.org/10.1093/jnci/djr010) | [PMC free article](/articles/PMC3057982/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21325141/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Natl%20Cancer%20Inst&title=CYP2D6%20Inhibition%20and%20Breast%20Cancer%20Recurrence%20in%20a%20Population-Based%20Study%20in%20Denmark&author=TL%20Lash&author=D%20Cronin-Fenton&author=TP%20Ahern&author=CL%20Rosenberg&author=KL%20Lunetta&volume=103&publication_year=2011&pages=489-500&pmid=21325141&doi=10.1093/jnci/djr010&)

27. Lash TL, Lien EA, Sørensen HT, Hamilton-Dutoit S. Genotype-guided tamoxifen therapy: time to pause for reflection? Lancet Oncol. 2009;10:825–33.  [DOI](https://doi.org/10.1016/S1470-2045(09)70030-0) | [PMC free article](/articles/PMC2895727/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19647203/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet%20Oncol&title=Genotype-guided%20tamoxifen%20therapy:%20time%20to%20pause%20for%20reflection?&author=TL%20Lash&author=EA%20Lien&author=HT%20S%C3%B8rensen&author=S%20Hamilton-Dutoit&volume=10&publication_year=2009&pages=825-33&pmid=19647203&doi=10.1016/S1470-2045(09)70030-0&)

28. Lash TL, Cole SR. Immortal Person-Time in Studies of Cancer Outcomes. JCO. 2009;27:e55–6.  [DOI](https://doi.org/10.1200/JCO.2009.24.1877) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19597013/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JCO&title=Immortal%20Person-Time%20in%20Studies%20of%20Cancer%20Outcomes&author=TL%20Lash&author=SR%20Cole&volume=27&publication_year=2009&pages=e55-6&pmid=19597013&doi=10.1200/JCO.2009.24.1877&)

29. Gradishar WJ, Anderson BO, Blair SL, Burstein HJ, Cyr A, Elias AD, et al. Breast Cancer Version 3.2014. J Natl Compr Canc Netw. 2014;12:542–90.  [DOI](https://doi.org/10.6004/jnccn.2014.0058) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24717572/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Natl%20Compr%20Canc%20Netw&title=Breast%20Cancer%20Version%203.2014&author=WJ%20Gradishar&author=BO%20Anderson&author=SL%20Blair&author=HJ%20Burstein&author=A%20Cyr&volume=12&publication_year=2014&pages=542-90&pmid=24717572&doi=10.6004/jnccn.2014.0058&)

30. Visvanathan K, Chlebowski RT, Hurley P, Col NF, Ropka M, Collyar D, et al. American Society of Clinical Oncology Clinical Practice Guideline Update on the Use of Pharmacologic Interventions Including Tamoxifen, Raloxifene, and Aromatase Inhibition for Breast Cancer Risk Reduction. J Clin Oncol. 2009;27:3235–58.  [DOI](https://doi.org/10.1200/JCO.2008.20.5179) | [PMC free article](/articles/PMC2716943/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19470930/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=American%20Society%20of%20Clinical%20Oncology%20Clinical%20Practice%20Guideline%20Update%20on%20the%20Use%20of%20Pharmacologic%20Interventions%20Including%20Tamoxifen,%20Raloxifene,%20and%20Aromatase%20Inhibition%20for%20Breast%20Cancer%20Risk%20Reduction&author=K%20Visvanathan&author=RT%20Chlebowski&author=P%20Hurley&author=NF%20Col&author=M%20Ropka&volume=27&publication_year=2009&pages=3235-58&pmid=19470930&doi=10.1200/JCO.2008.20.5179&)

31. Abraham JE, Maranian MJ, Driver KE, Platte R, Kalmyrzaev B, Baynes C, et al. CYP2D6 gene variants: association with breast cancer specific survival in a cohort of breast cancer patients from the United Kingdom treated with adjuvant tamoxifen. Breast Cancer Res. 2010;12:R64.  [DOI](https://doi.org/10.1186/bcr2629) | [PMC free article](/articles/PMC2949659/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20731819/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Breast%20Cancer%20Res&title=CYP2D6%20gene%20variants:%20association%20with%20breast%20cancer%20specific%20survival%20in%20a%20cohort%20of%20breast%20cancer%20patients%20from%20the%20United%20Kingdom%20treated%20with%20adjuvant%20tamoxifen&author=JE%20Abraham&author=MJ%20Maranian&author=KE%20Driver&author=R%20Platte&author=B%20Kalmyrzaev&volume=12&publication_year=2010&pages=R64&pmid=20731819&doi=10.1186/bcr2629&)

32. Argalacsova S, Slanar O, Bakhouche H, Pertuzelka L. Impact of ABCB1 and CYP2D6 polymorphisms on tamoxifen treatment outcomes and adverse events in breast cancer patients. J BUON. 2017;22:1217–26.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/29135105/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20BUON&title=Impact%20of%20ABCB1%20and%20CYP2D6%20polymorphisms%20on%20tamoxifen%20treatment%20outcomes%20and%20adverse%20events%20in%20breast%20cancer%20patients&author=S%20Argalacsova&author=O%20Slanar&author=H%20Bakhouche&author=L%20Pertuzelka&volume=22&publication_year=2017&pages=1217-26&pmid=29135105&)

33. Bijl MJ, van Schaik RHN, Lammers LA, Hofman A, Vulto AG, van Gelder T, et al. The CYP2D6*4 polymorphism affects breast cancer survival in tamoxifen users. Breast Cancer Res Treat. 2009;118:125–30.  [DOI](https://doi.org/10.1007/s10549-008-0272-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19189212/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Breast%20Cancer%20Res%20Treat&title=The%20CYP2D6*4%20polymorphism%20affects%20breast%20cancer%20survival%20in%20tamoxifen%20users&author=MJ%20Bijl&author=RHN%20van%20Schaik&author=LA%20Lammers&author=A%20Hofman&author=AG%20Vulto&volume=118&publication_year=2009&pages=125-30&pmid=19189212&doi=10.1007/s10549-008-0272-2&)

34. Dezentjé VO, van Schaik RHN, Vletter-Bogaartz JM, van der Straaten T, Wessels J a. M, Kranenbarg EM-K, et al. CYP2D6 genotype in relation to tamoxifen efficacy in a Dutch cohort of the tamoxifen exemestane adjuvant multinational (TEAM) trial. Breast Cancer Res Treat. 2013;140:363–73.  [DOI](https://doi.org/10.1007/s10549-013-2619-6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23842856/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Breast%20Cancer%20Res%20Treat&title=CYP2D6%20genotype%20in%20relation%20to%20tamoxifen%20efficacy%20in%20a%20Dutch%20cohort%20of%20the%20tamoxifen%20exemestane%20adjuvant%20multinational%20(TEAM)%20trial&author=VO%20Dezentj%C3%A9&author=RHN%20van%20Schaik&author=JM%20Vletter-Bogaartz&author=T%20van%20der%20Straaten&author=J%20a.%20M%20Wessels&volume=140&publication_year=2013&pages=363-73&pmid=23842856&doi=10.1007/s10549-013-2619-6&)

35. He W, Grassmann F, Eriksson M, Eliasson E, Margolin S, Thoren L, et al. CYP2D6 Genotype Predicts Tamoxifen Discontinuation and Prognosis in Patients With Breast Cancer. J Clin Oncol. United States; 2020;38:548–57.  [DOI](https://doi.org/10.1200/JCO.19.01535) | [PMC free article](/articles/PMC7030887/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31800347/) | [Google Scholar](https://scholar.google.com/scholar_lookup?title=J%20Clin%20Oncol&author=W%20He&author=F%20Grassmann&author=M%20Eriksson&author=E%20Eliasson&author=S%20Margolin&publication_year=2020&)

36. Markkula A, Hjertberg M, Rose C, Ingvar C, Jernström H. No association found between CYP2D6 genotype and early breast cancer events in tamoxifen-treated patients. Acta Oncol. 2014;53:195–200.  [DOI](https://doi.org/10.3109/0284186X.2013.840739) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24125101/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Acta%20Oncol&title=No%20association%20found%20between%20CYP2D6%20genotype%20and%20early%20breast%20cancer%20events%20in%20tamoxifen-treated%20patients&author=A%20Markkula&author=M%20Hjertberg&author=C%20Rose&author=C%20Ingvar&author=H%20Jernstr%C3%B6m&volume=53&publication_year=2014&pages=195-200&pmid=24125101&doi=10.3109/0284186X.2013.840739&)

37. Mwinyi J, Vokinger K, Jetter A, Breitenstein U, Hiller C, Kullak-Ublick GA, et al. Impact of variable CYP genotypes on breast cancer relapse in patients undergoing adjuvant tamoxifen therapy. Cancer Chemother Pharmacol. 2014;73:1181–8.  [DOI](https://doi.org/10.1007/s00280-014-2453-5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24682508/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Chemother%20Pharmacol&title=Impact%20of%20variable%20CYP%20genotypes%20on%20breast%20cancer%20relapse%20in%20patients%20undergoing%20adjuvant%20tamoxifen%20therapy&author=J%20Mwinyi&author=K%20Vokinger&author=A%20Jetter&author=U%20Breitenstein&author=C%20Hiller&volume=73&publication_year=2014&pages=1181-8&pmid=24682508&doi=10.1007/s00280-014-2453-5&)

38. Regan MM, Leyland-Jones B, Bouzyk M, Pagani O, Tang W, Kammler R, et al. CYP2D6 genotype and tamoxifen response in postmenopausal women with endocrine-responsive breast cancer: the breast international group 1-98 trial. J Natl Cancer Inst. 2012;104:441–51.  [DOI](https://doi.org/10.1093/jnci/djs125) | [PMC free article](/articles/PMC3309132/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22395644/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Natl%20Cancer%20Inst&title=CYP2D6%20genotype%20and%20tamoxifen%20response%20in%20postmenopausal%20women%20with%20endocrine-responsive%20breast%20cancer:%20the%20breast%20international%20group%201-98%20trial&author=MM%20Regan&author=B%20Leyland-Jones&author=M%20Bouzyk&author=O%20Pagani&author=W%20Tang&volume=104&publication_year=2012&pages=441-51&pmid=22395644&doi=10.1093/jnci/djs125&)

39. Schroth W, Antoniadou L, Fritz P, Schwab M, Muerdter T, Zanger UM, et al. Breast cancer treatment outcome with adjuvant tamoxifen relative to patient CYP2D6 and CYP2C19 genotypes. J Clin Oncol. 2007;25:5187–93.  [DOI](https://doi.org/10.1200/JCO.2007.12.2705) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18024866/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Breast%20cancer%20treatment%20outcome%20with%20adjuvant%20tamoxifen%20relative%20to%20patient%20CYP2D6%20and%20CYP2C19%20genotypes&author=W%20Schroth&author=L%20Antoniadou&author=P%20Fritz&author=M%20Schwab&author=T%20Muerdter&volume=25&publication_year=2007&pages=5187-93&pmid=18024866&doi=10.1200/JCO.2007.12.2705&)

40. Thompson AM, Johnson A, Quinlan P, Hillman G, Fontecha M, Bray SE, et al. Comprehensive CYP2D6 genotype and adherence affect outcome in breast cancer patients treated with tamoxifen monotherapy. Breast Cancer Res Treat. 2011;125:279–87.  [DOI](https://doi.org/10.1007/s10549-010-1139-x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20809362/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Breast%20Cancer%20Res%20Treat&title=Comprehensive%20CYP2D6%20genotype%20and%20adherence%20affect%20outcome%20in%20breast%20cancer%20patients%20treated%20with%20tamoxifen%20monotherapy&author=AM%20Thompson&author=A%20Johnson&author=P%20Quinlan&author=G%20Hillman&author=M%20Fontecha&volume=125&publication_year=2011&pages=279-87&pmid=20809362&doi=10.1007/s10549-010-1139-x&)

41. Wegman P, Elingarami S, Carstensen J, Stål O, Nordenskjöld B, Wingren S. Genetic variants of CYP3A5, CYP2D6, SULT1A1, UGT2B15 and tamoxifen response in postmenopausal patients with breast cancer. Breast Cancer Res. 2007;9:R7.  [DOI](https://doi.org/10.1186/bcr1640) | [PMC free article](/articles/PMC1851378/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17244352/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Breast%20Cancer%20Res&title=Genetic%20variants%20of%20CYP3A5,%20CYP2D6,%20SULT1A1,%20UGT2B15%20and%20tamoxifen%20response%20in%20postmenopausal%20patients%20with%20breast%20cancer&author=P%20Wegman&author=S%20Elingarami&author=J%20Carstensen&author=O%20St%C3%A5l&author=B%20Nordenskj%C3%B6ld&volume=9&publication_year=2007&pages=R7&pmid=17244352&doi=10.1186/bcr1640&)

42. Goetz MP, Suman VJ, Hoskin TL, Gnant M, Filipits M, Safgren SL, et al. CYP2D6 metabolism and patient outcome in the Austrian Breast and Colorectal Cancer Study Group trial (ABCSG) 8. Clin Cancer Res. 2013;19:500–7.  [DOI](https://doi.org/10.1158/1078-0432.CCR-12-2153) | [PMC free article](/articles/PMC3548984/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23213055/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=CYP2D6%20metabolism%20and%20patient%20outcome%20in%20the%20Austrian%20Breast%20and%20Colorectal%20Cancer%20Study%20Group%20trial%20(ABCSG)%208&author=MP%20Goetz&author=VJ%20Suman&author=TL%20Hoskin&author=M%20Gnant&author=M%20Filipits&volume=19&publication_year=2013&pages=500-7&pmid=23213055&doi=10.1158/1078-0432.CCR-12-2153&)

43. Gor PP, Su HI, Gray RJ, Gimotty PA, Horn M, Aplenc R, et al. Cyclophosphamide-metabolizing enzyme polymorphisms and survival outcomes after adjuvant chemotherapy for node-positive breast cancer: a retrospective cohort study. Breast Cancer Res. 2010;12:R26.  [DOI](https://doi.org/10.1186/bcr2570) | [PMC free article](/articles/PMC2917014/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20459744/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Breast%20Cancer%20Res&title=Cyclophosphamide-metabolizing%20enzyme%20polymorphisms%20and%20survival%20outcomes%20after%20adjuvant%20chemotherapy%20for%20node-positive%20breast%20cancer:%20a%20retrospective%20cohort%20study&author=PP%20Gor&author=HI%20Su&author=RJ%20Gray&author=PA%20Gimotty&author=M%20Horn&volume=12&publication_year=2010&pages=R26&pmid=20459744&doi=10.1186/bcr2570&)

44. Rae JM, Drury S, Hayes DF, Stearns V, Thibert JN, Haynes BP, et al. CYP2D6 and UGT2B7 genotype and risk of recurrence in tamoxifen-treated breast cancer patients. J Natl Cancer Inst. 2012;104:452–60.  [DOI](https://doi.org/10.1093/jnci/djs126) | [PMC free article](/articles/PMC3611934/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22395643/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Natl%20Cancer%20Inst&title=CYP2D6%20and%20UGT2B7%20genotype%20and%20risk%20of%20recurrence%20in%20tamoxifen-treated%20breast%20cancer%20patients&author=JM%20Rae&author=S%20Drury&author=DF%20Hayes&author=V%20Stearns&author=JN%20Thibert&volume=104&publication_year=2012&pages=452-60&pmid=22395643&doi=10.1093/jnci/djs126&)

45. De Ameida Melo M, De Vasconcelos-Valença RJ, Neto FM, Borges RS, Costa-Silva DR, Da Conceição Barros-Oliveira M, et al. CYP2D6 gene polymorphisms in Brazilian patients with breast cancer treated with adjuvant tamoxifen and its association with disease recurrence. Biomed Rep. 2016;5:574–8.  [DOI](https://doi.org/10.3892/br.2016.771) | [PMC free article](/articles/PMC5103674/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27882219/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biomed%20Rep&title=CYP2D6%20gene%20polymorphisms%20in%20Brazilian%20patients%20with%20breast%20cancer%20treated%20with%20adjuvant%20tamoxifen%20and%20its%20association%20with%20disease%20recurrence&author=M%20De%20Ameida%20Melo&author=RJ%20De%20Vasconcelos-Valen%C3%A7a&author=FM%20Neto&author=RS%20Borges&author=DR%20Costa-Silva&volume=5&publication_year=2016&pages=574-8&pmid=27882219&doi=10.3892/br.2016.771&)

46. Jorge-Aaron R-M, Rodrigo R-C, de Jesus A-MI, Esther M-PR. CYP2D6 does not impact on breast cancer-free survival in Southeast Mexican patients under tamoxifen treatment. Per Med. England; 2020;17:261–70.  [DOI](https://doi.org/10.2217/pme-2019-0135) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32589097/) | [Google Scholar](https://scholar.google.com/scholar_lookup?title=Per%20Med&author=R-M%20Jorge-Aaron&author=R-C%20Rodrigo&author=A-MI%20de%20Jesus&author=M-PR%20Esther&publication_year=2020&)

47. Chamnanphon M, Pechatanan K, Sirachainan E, Trachu N, Chantratita W, Pasomsub E, et al. Association of CYP2D6 and CYP2C19 polymorphisms and disease-free survival of Thai post-menopausal breast cancer patients who received adjuvant tamoxifen. Pharmgenomics Pers Med. 2013;6:37–48.  [DOI](https://doi.org/10.2147/PGPM.S42330) | [PMC free article](/articles/PMC3681433/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23776391/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmgenomics%20Pers%20Med&title=Association%20of%20CYP2D6%20and%20CYP2C19%20polymorphisms%20and%20disease-free%20survival%20of%20Thai%20post-menopausal%20breast%20cancer%20patients%20who%20received%20adjuvant%20tamoxifen&author=M%20Chamnanphon&author=K%20Pechatanan&author=E%20Sirachainan&author=N%20Trachu&author=W%20Chantratita&volume=6&publication_year=2013&pages=37-48&pmid=23776391&doi=10.2147/PGPM.S42330&)

48. Kiyotani K, Mushiroda T, Hosono N, Tsunoda T, Kubo M, Aki F, et al. Lessons for pharmacogenomics studies: association study between CYP2D6 genotype and tamoxifen response. Pharmacogenet Genomics. 2010;20:565–8.  [DOI](https://doi.org/10.1097/FPC.0b013e32833af231) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20574415/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Lessons%20for%20pharmacogenomics%20studies:%20association%20study%20between%20CYP2D6%20genotype%20and%20tamoxifen%20response&author=K%20Kiyotani&author=T%20Mushiroda&author=N%20Hosono&author=T%20Tsunoda&author=M%20Kubo&volume=20&publication_year=2010&pages=565-8&pmid=20574415&doi=10.1097/FPC.0b013e32833af231&)

49. Park HS, Choi J-Y, Lee M-J, Park S, Yeo C-W, Lee SS, et al. Association between genetic polymorphisms of CYP2D6 and outcomes in breast cancer patients with tamoxifen treatment. J Korean Med Sci. 2011;26:1007–13.  [DOI](https://doi.org/10.3346/jkms.2011.26.8.1007) | [PMC free article](/articles/PMC3154335/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21860550/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Korean%20Med%20Sci&title=Association%20between%20genetic%20polymorphisms%20of%20CYP2D6%20and%20outcomes%20in%20breast%20cancer%20patients%20with%20tamoxifen%20treatment&author=HS%20Park&author=J-Y%20Choi&author=M-J%20Lee&author=S%20Park&author=C-W%20Yeo&volume=26&publication_year=2011&pages=1007-13&pmid=21860550&doi=10.3346/jkms.2011.26.8.1007&)

50. Sirachainan E, Jaruhathai S, Trachu N, Panvichian R, Sirisinha T, Ativitavas T, et al. CYP2D6 polymorphisms influence the efficacy of adjuvant tamoxifen in Thai breast cancer patients. Pharmgenomics Pers Med. 2012;5:149–53.  [DOI](https://doi.org/10.2147/PGPM.S32160) | [PMC free article](/articles/PMC3513236/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23226070/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmgenomics%20Pers%20Med&title=CYP2D6%20polymorphisms%20influence%20the%20efficacy%20of%20adjuvant%20tamoxifen%20in%20Thai%20breast%20cancer%20patients&author=E%20Sirachainan&author=S%20Jaruhathai&author=N%20Trachu&author=R%20Panvichian&author=T%20Sirisinha&volume=5&publication_year=2012&pages=149-53&pmid=23226070&doi=10.2147/PGPM.S32160&)

51. Teh LK, Mohamed NI, Salleh MZ, Rohaizak M, Shahrun NS, Saladina JJ, et al. The risk of recurrence in breast cancer patients treated with tamoxifen: polymorphisms of CYP2D6 and ABCB1. AAPS J. 2012;14:52–9.  [DOI](https://doi.org/10.1208/s12248-011-9313-6) | [PMC free article](/articles/PMC3291182/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22183189/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=AAPS%20J&title=The%20risk%20of%20recurrence%20in%20breast%20cancer%20patients%20treated%20with%20tamoxifen:%20polymorphisms%20of%20CYP2D6%20and%20ABCB1&author=LK%20Teh&author=NI%20Mohamed&author=MZ%20Salleh&author=M%20Rohaizak&author=NS%20Shahrun&volume=14&publication_year=2012&pages=52-9&pmid=22183189&doi=10.1208/s12248-011-9313-6&)

52. Damodaran SE, Pradhan SC, Umamaheswaran G, Kadambari D, Reddy KS, Adithan C. Genetic polymorphisms of CYP2D6 increase the risk for recurrence of breast cancer in patients receiving tamoxifen as an adjuvant therapy. Cancer Chemother Pharmacol. 2012;70:75–81.  [DOI](https://doi.org/10.1007/s00280-012-1891-1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22623212/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Chemother%20Pharmacol&title=Genetic%20polymorphisms%20of%20CYP2D6%20increase%20the%20risk%20for%20recurrence%20of%20breast%20cancer%20in%20patients%20receiving%20tamoxifen%20as%20an%20adjuvant%20therapy&author=SE%20Damodaran&author=SC%20Pradhan&author=G%20Umamaheswaran&author=D%20Kadambari&author=KS%20Reddy&volume=70&publication_year=2012&pages=75-81&pmid=22623212&doi=10.1007/s00280-012-1891-1&)

53. Lan B, Ma F, Zhai X, Li Q, Chen S, Wang J, et al. The relationship between the CYP2D6 polymorphisms and tamoxifen efficacy in adjuvant endocrine therapy of breast cancer patients in Chinese Han population. Int J Cancer. 2018;143:184–9.  [DOI](https://doi.org/10.1002/ijc.31291) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29396856/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Cancer&title=The%20relationship%20between%20the%20CYP2D6%20polymorphisms%20and%20tamoxifen%20efficacy%20in%20adjuvant%20endocrine%20therapy%20of%20breast%20cancer%20patients%20in%20Chinese%20Han%20population&author=B%20Lan&author=F%20Ma&author=X%20Zhai&author=Q%20Li&author=S%20Chen&volume=143&publication_year=2018&pages=184-9&pmid=29396856&doi=10.1002/ijc.31291&)

54. Lei L, Wang X, Wu X-D, Wang Z, Chen Z-H, Zheng Y-B, et al. Association of CYP2D6*10 (c.100C>T) polymorphisms with clinical outcome of breast cancer after tamoxifen adjuvant endocrine therapy in Chinese population. Am J Transl Res. 2016;8:3585–92.  [PMC free article](/articles/PMC5009411/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27648149/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Transl%20Res&title=Association%20of%20CYP2D6*10%20(c.100C>T)%20polymorphisms%20with%20clinical%20outcome%20of%20breast%20cancer%20after%20tamoxifen%20adjuvant%20endocrine%20therapy%20in%20Chinese%20population&author=L%20Lei&author=X%20Wang&author=X-D%20Wu&author=Z%20Wang&author=Z-H%20Chen&volume=8&publication_year=2016&pages=3585-92&pmid=27648149&)

55. Sukasem C, Sirachainan E, Chamnanphon M, Pechatanan K, Sirisinha T, Ativitavas T, et al. Impact of CYP2D6 polymorphisms on tamoxifen responses of women with breast cancer: a microarray-based study in Thailand. Asian Pac J Cancer Prev. 2012;13:4549–53.  [DOI](https://doi.org/10.7314/apjcp.2012.13.9.4549) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23167378/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Asian%20Pac%20J%20Cancer%20Prev&title=Impact%20of%20CYP2D6%20polymorphisms%20on%20tamoxifen%20responses%20of%20women%20with%20breast%20cancer:%20a%20microarray-based%20study%20in%20Thailand&author=C%20Sukasem&author=E%20Sirachainan&author=M%20Chamnanphon&author=K%20Pechatanan&author=T%20Sirisinha&volume=13&publication_year=2012&pages=4549-53&pmid=23167378&doi=10.7314/apjcp.2012.13.9.4549&)

56. Zhang X, Pu Z, Ge J, Shen J, Yuan X, Xie H. Association of CYP2D6*10, OATP1B1 A388G, and OATP1B1 T521C polymorphisms and overall survival of breast cancer patients after tamoxifen therapy. Med Sci Monit. 2015;21:563–9.  [DOI](https://doi.org/10.12659/MSM.893473) | [PMC free article](/articles/PMC4345853/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25701109/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Med%20Sci%20Monit&title=Association%20of%20CYP2D6*10,%20OATP1B1%20A388G,%20and%20OATP1B1%20T521C%20polymorphisms%20and%20overall%20survival%20of%20breast%20cancer%20patients%20after%20tamoxifen%20therapy&author=X%20Zhang&author=Z%20Pu&author=J%20Ge&author=J%20Shen&author=X%20Yuan&volume=21&publication_year=2015&pages=563-9&pmid=25701109&doi=10.12659/MSM.893473&)

57. Johansson H, Gandini S, Serrano D, Gjerde J, Lattanzi M, Macis D, et al. A pooled analysis of CYP2D6 genotype in breast cancer prevention trials of low-dose tamoxifen. Breast Cancer Res Treat. 2016;159:97–108.  [DOI](https://doi.org/10.1007/s10549-016-3932-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27484880/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Breast%20Cancer%20Res%20Treat&title=A%20pooled%20analysis%20of%20CYP2D6%20genotype%20in%20breast%20cancer%20prevention%20trials%20of%20low-dose%20tamoxifen&author=H%20Johansson&author=S%20Gandini&author=D%20Serrano&author=J%20Gjerde&author=M%20Lattanzi&volume=159&publication_year=2016&pages=97-108&pmid=27484880&doi=10.1007/s10549-016-3932-7&)

58. Mayer SE, Weiss NS, Chubak J, Doody DR, Carlson CS, Makar KW, et al. CYP2D6-inhibiting medication use and inherited CYP2D6 variation in relation to adverse breast cancer outcomes after tamoxifen therapy. Cancer Causes Control. 2019;30:103–12.  [DOI](https://doi.org/10.1007/s10552-018-1117-x) | [PMC free article](/articles/PMC6353670/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30542984/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Causes%20Control&title=CYP2D6-inhibiting%20medication%20use%20and%20inherited%20CYP2D6%20variation%20in%20relation%20to%20adverse%20breast%20cancer%20outcomes%20after%20tamoxifen%20therapy&author=SE%20Mayer&author=NS%20Weiss&author=J%20Chubak&author=DR%20Doody&author=CS%20Carlson&volume=30&publication_year=2019&pages=103-12&pmid=30542984&doi=10.1007/s10552-018-1117-x&)

59. Morrow PK, Serna R, Broglio K, Pusztai L, Nikoloff DM, Hillman GR, et al. Effect of CYP2D6 polymorphisms on breast cancer recurrence. Cancer. 2012;118:1221–7.  [DOI](https://doi.org/10.1002/cncr.26407) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21823108/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer&title=Effect%20of%20CYP2D6%20polymorphisms%20on%20breast%20cancer%20recurrence&author=PK%20Morrow&author=R%20Serna&author=K%20Broglio&author=L%20Pusztai&author=DM%20Nikoloff&volume=118&publication_year=2012&pages=1221-7&pmid=21823108&doi=10.1002/cncr.26407&)

60. Nowell SA, Ahn J, Rae JM, Scheys JO, Trovato A, Sweeney C, et al. Association of genetic variation in tamoxifen-metabolizing enzymes with overall survival and recurrence of disease in breast cancer patients. Breast Cancer Res Treat. 2005;91:249–58.  [DOI](https://doi.org/10.1007/s10549-004-7751-x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15952058/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Breast%20Cancer%20Res%20Treat&title=Association%20of%20genetic%20variation%20in%20tamoxifen-metabolizing%20enzymes%20with%20overall%20survival%20and%20recurrence%20of%20disease%20in%20breast%20cancer%20patients&author=SA%20Nowell&author=J%20Ahn&author=JM%20Rae&author=JO%20Scheys&author=A%20Trovato&volume=91&publication_year=2005&pages=249-58&pmid=15952058&doi=10.1007/s10549-004-7751-x&)

61. Sanchez-Spitman A, Dezentjé V, Swen J, Moes DJAR, Böhringer S, Batman E, et al. Tamoxifen Pharmacogenetics and Metabolism: Results From the Prospective CYPTAM Study. J Clin Oncol. 2019;JCO1800307.  [DOI](https://doi.org/10.1200/JCO.18.00307) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30676859/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Tamoxifen%20Pharmacogenetics%20and%20Metabolism:%20Results%20From%20the%20Prospective%20CYPTAM%20Study&author=A%20Sanchez-Spitman&author=V%20Dezentj%C3%A9&author=J%20Swen&author=DJAR%20Moes&author=S%20B%C3%B6hringer&publication_year=2019&pages=JCO1800307&pmid=30676859&doi=10.1200/JCO.18.00307&)
