# Pharmacogenetics of outcome in children with acute lymphoblastic leukemia

## Metadata
**Authors:** Jose Claudio C Rocha, Cheng Cheng, Wei Liu, Shinji Kishi, Soma Das, Edwin H Cook, John T Sandlund, Jeffrey Rubnitz, Raul Ribeiro, Dario Campana, Ching-Hon Pui, William E Evans, Mary V Relling
**Journal:** Blood
**Date:** 2005 Feb 15
**DOI:** [10.1182/blood-2004-11-4544](https://doi.org/10.1182/blood-2004-11-4544)
**PMID:** 15713801
**PMCID:** PMC1895006
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1895006/

## Abstract

Acquired genetic characteristics of acute lymphoblastic leukemia (ALL) cells are used to individualize therapy, whereas germ line genetic characteristics generally are not. We determined whether ALL outcome was related to 16 genetic polymorphisms affecting the pharmacodynamics of antileukemic agents. Of 246 children, 116 were treated on the lower-risk (LR) and 130 on the higher-risk (HR) arms of a St Jude protocol. Patients in the HR group with the glutathione S-transferase (GSTM1) nonnull genotype had greater risk of hematologic relapse (P = .03), which was further increased by the thymidylate synthetase (TYMS) 3/3 genotype (P = .03). These genotypes remained predictive in multivariate analyses (P < .001 and .003, respectively). No genotypes were predictive in the LR arm. Expression of these 2 genes in ALL blasts was lower in those with low-activity genotypes. For central nervous system relapse, among the HR group, the vitamin D receptor start site (P = .02) and intron 8 genotypes (P = .04) predisposed, whereas for LR patients the TYMS 3/3 genotype predisposed (P = .04). The GSTM1 non-null and TYMS 3/3 genotypes are plausibly linked to drug resistance. Polymorphisms interact to influence antileukemic outcome and represent determinants of response that can be used to optimize therapy. (Blood. 2005;105:4752-4758)

## Introduction

Childhood acute lymphoblastic leukemia (ALL) is cured in approximately 80% of patients.[1](#ref1)^-^[4](#ref4) One treatment strategy for improving cure has been to modify the intensity of therapy based on the acquired genetic characteristics of the leukemia, using more intensive therapy for ALL with molecular markers (eg, the t(9;22) or *MLL* translocations), indicating a resistant leukemia.[5](#ref5) Other than age, host characteristics are usually not used for assigning patients with ALL to risk groups or to determine the intensity of therapy.

ALL regimens consist of combination chemotherapy (eg, vincristine, glucocorticoid, methotrexate, a thiopurine, and asparaginase with or without anthracyclines, topoisomerase II inhibitors, cytarabine, and cyclophosphamide) administered continuously over 2 to 3 years.[1](#ref1)^-^[4](#ref4)^,^[6](#ref6) Outcome of ALL may be influenced by modest changes in drug dose or exposure.[7](#ref7)^-^[10](#ref10) Thus, if the determinants of interpatient variability in drug pharmacodynamics were better defined, tailoring drug therapy based on these factors might further improve outcome.

Germ line polymorphisms in genes that code for the proteins involved in the pharmacodynamics of antileukemic agents are common, with the frequency of the “variant” allele ranging from 5% to 50%.[11](#ref11)^,^[12](#ref12) Although we and others have explored whether germ line polymorphisms may relate to ALL outcomes,[13](#ref13)^-^[19](#ref19) such studies have examined only a few polymorphisms at a time, have failed to adjust for other risk factors, have not distinguished among relapse risk versus other types of adverse outcome, or have potential for selection bias by including only a small fraction of patients enrolled on the treatment protocols. Most prior studies have not assessed whether gene-gene interactions could influence the relationships between germ line polymorphisms and outcomes.

Herein, we have used a candidate gene approach to assess whether ALL outcome is related to 16 common polymorphisms in genes plausibly linked to the pharmacodynamics of the drug therapy, accounting for known prognostic factors, types of ALL outcome, competing risks, and gene-gene interactions.

## Patients, materials, and methods

### Study population

Of the 247 children with newly diagnosed ALL treated on the St Jude Children's Research Hospital Total XIIIB study for ALL,[20](#ref20) 246 were included; one patient died within a few weeks after diagnosis and a germ line DNA sample could not be obtained. The research was approved by our Institutional Review Board, and informed consent was obtained from parents, guardians, or patients (as appropriate). Patients were prospectively assigned to risk groups that determined the intensity of therapy; those with at least one of the following were assigned to the higher-risk group: age younger than 1 year or older than 10 years (except cases with DNA index ≥ 1.16 and ≤ 1.60), initial leukocyte count greater than 50 × 10^9^/L (except cases with DNA index ≥ 1.16 and ≤ 1.60), central nervous system (CNS) involvement (defined as ≥ 5 white cells/mm^3^ in cerebrospinal fluid and the presence of leukemic blasts on cytocentrifuge preparation from a nontraumatic spinal tap, or cranial palsies), testicular involvement, greater than 5% leukemic blasts in bone marrow on day 19, T-cell ALL, presence of *BCR-ABL* [t(9;22)], *MLL-AF4* [t(4;11)], *E2A-PBX1* [t(1;19)] in pre-B ALL, or *MLL* rearrangement by karyotype and/or molecular probes. All others were enrolled on the lower-risk arm.[21](#ref21) Minimal residual disease on day 43 was assessed by flow cytometric assays as previously described.[21](#ref21)^,^[22](#ref22)

Race/ethnicity was classified by a hospital staff member at the time of diagnosis (using 1994-1998 policies). Parents (or patients if older than 18 years of age) were asked which of the race categories (American Indian, black, Chinese, Filipino, Hispanic, Japanese, white, or unknown) that they considered applied to their child. Race and/or ethnicity was assessed in this study, because of the prior association of race with ALL outcome in some studies,[23](#ref23)^,^[24](#ref24) and because of the considerable racial differences in frequencies of genotypes.[11](#ref11)^,^[12](#ref12)^,^[25](#ref25)

In summary, treatment consisted of high-dose methotrexate, mercaptopurine, or both, followed by therapy with prednisone, vincristine, daunorubicin, asparaginase, cytarabine, and etoposide.[20](#ref20) At complete remission, patients received 2 weeks of high-dose methotrexate and mercaptopurine, followed by 120 weeks of risk-directed postremission rotating pairs of chemotherapy. For lower-risk cases, postremission therapy consisted of daily mercaptopurine and weekly methotrexate, high-dose methotrexate and mercaptopurine every 8 weeks for the first year, and dexamethasone plus vincristine every 4 weeks. For higher-risk cases, postremission therapy consisted of drug pairs (etoposide, cyclophosphamide, mercaptopurine, methotrexate, cytarabine, dexamethasone, vincristine, and mercaptopurine) administered in weekly rotation, interrupted by reinduction therapy from weeks 16 to 21.[21](#ref21) Events were defined as any relapse, death, or secondary malignancies, using criteria previously published.[26](#ref26)

### Genotyping

DNA was extracted from normal blood cells. Candidate genes exhibited polymorphisms and encoded proteins that were involved in the pharmacokinetics or pharmacodynamics of antileukemic agents ([Figure 1](#fig1)). Priority was given to polymorphisms that were demonstrated to be associated with phenotypes in both clinical and preclinical studies.[9](#ref9)^,^[11](#ref11)^-^[14](#ref14)^,^[27](#ref27)^-^[33](#ref33) Vincristine, prednisone, dexamethasone, etoposide, and daunorubicin are substrates for cytochrome P4503A4 *(CYP3A4)*, cytochrome P4503A5 *(CYP3A5)*, and/or p-glycoprotein *(MDR1)*, which are partially regulated by the vitamin D receptor *(VDR)*.[27](#ref27)^,^[34](#ref34)^,^[35](#ref35) The parent drug or metabolites of etoposide, daunorubicin, and cyclophosphamide are substrates for glutathione *S*-transferases (*GSTM1, GSTP1*, and *GSTT1*).[28](#ref28) Etoposide is glucuronidated by UDP-glucuronosyltransferase 1A1 (*UGT1A1*).[36](#ref36) Mercaptopurine is a substrate for thiopurine *S*-methyltransferase (*TPMT*),[9](#ref9) and methotrexate interacts with methylenetetrahydrofolate reductase (*MTHFR*), thymidylate synthetase (*TYMS*), and the reduced folate carrier (*RFC*, or *SLC19A1*).[11](#ref11) Prednisone and dexamethasone bind to the glucocorticoid receptor (*NR3C1*).[29](#ref29)

### Figure 1.

![Figure 1.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5ac4/1895006/1a50c78670b7/zh80120579930001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=1895006_zh80120579930001.jpg)

Interaction of primary antileukemic agents (in rectangles) with products of polymorphic genes (in circles) that were subject of study. GMP indicates mercaptopurine; MTX, methotrexate.

Genotyping was performed for the 16 polymorphic loci using methods as previously described: *CYP3A4*1B* (A>G at position -392) and *CYP3A5*3* (G>A at position 22893)[37](#ref37); *GSTP1* 313A>G,[37](#ref37) *GSTM1* deletion and *GSTT1* deletion[16](#ref16); *MDR1* exon 21 (2677G>T/A) and *MDR1* exon 26 (3435C>T)[37](#ref37); *MTHFR* 677C>T[38](#ref38) and *MTHFR* 1298A>C[30](#ref30); *NR3C1* 1088A>G[31](#ref31); *RFC* 80G>A[38](#ref38); *TPMT* 238G>C, *TPMT* 460G>A, and *TPMT* 719A>G[32](#ref32); *TYMS* enhancer repeat[33](#ref33); *UGT1A1* promoter repeat polymorphism[37](#ref37); *VDR* intron 8 G>A, and *VDR* FokI (start-site) T>C.[37](#ref37)

### Gene expression

In all children with available RNA in their diagnostic ALL blasts, gene expression levels were analyzed using the Affymetrix GeneChip array HG_U95Av2, as described.[39](#ref39) Expression signals were scaled to a target average intensity and analyzed using Affymetrix Microarray Suite (MAS) version 5.0 (Affymetrix, Santa Clara, CA). The expression levels of *GSTM1* and *TYMS* were based on probe sets 39054_at and 37899_at, respectively. Patients whose blasts had numeric or structural abnormalities of chromosomes 1 and 18 were excluded from the analysis of blast expression of the *GSTM1* and *TYMS* genes, located on 1p13.3 and 18p11.32, respectively.

### Statistical analysis

The distribution of genotypes between racial groups or between risk groups was compared using chi-square test with Yates correction. The cumulative incidences of hematologic and CNS relapse by risk group were compared using the Gray estimator with incorporation of competing events.[40](#ref40) Death in remission and second malignancy were treated as competing risks. Survival time was the time between diagnosis and the date of last follow-up.

Genotypes for each of the 16 polymorphisms were pooled into binary groups for the statistical analysis. For *GSTM1* deletion, *GSTT1* deletion, and *NR3C1* 1088A>G, the genotypic groups were originally measured as binary ([Table 1](#tbl1)). For *CYP3A4*1B, CYP3A5*3, GSTP1* 313A>G, *MDR1* exon 21 (2677G>T/A), *MDR1* exon 26 (3435C>T), *MTHFR* 677C>T, *MTHFR* 1298A>C, *RFC* 80G>A, *VDR* intron 8 G>A, and *VDR* FokI T>C loci, it was not clear a priori how to pool genotypes, and so the analysis was conducted using both possible major genotypic groups (eg, *RFC* 80 GG versus GA or AA, and *RFC* 80 GG or GA versus AA). For each locus, the groupings that yielded the most significant predictors of outcome are indicated in [Table 1](#tbl1). For *TYMS* enhancer and *UGT1A1* promoter repeats, genotypes were merged according to prior functional data into 2 distinct groups (*TYMS* 3/3 versus others, and *UGT1A1* 7/7 versus others); rare genotypes (ie, *TYMS* 2/9 [n = 1] and *UGT1A1* 5/7 [n = 2]) were excluded from the analysis. For *TPMT*, common haplotypes that combined *TPMT* 460G and *TPMT* 719A, which constitute the ^*^3A allele, were assumed, so that none of the patients were homozygous deficient for *TPMT* ([Table 1](#tbl1)).[32](#ref32)

### Table 1.

|   | No. patients, higher-risk arm (%) |  |  |  | No. patients, lower-risk arm (%) |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Loci and genotypes | Whites | Blacks | Others | Total | Whites | Blacks | Others | Total |
| CYP3A4*1B |  |  |  |  |  |  |  |  |
| AA | 88 (96.7) | 4 (13.8) | 8 (80) | 100 (76.9) | 75 (93.7) | 3 (18.7) | 19 (95) | 97 (83.6) |
| AG and GG | 3 (3.3) | 25 (86.2) | 2 (20) | 30 (23.1) | 5 (6.3) | 13 (81.3) | 1 (5) | 19 (16.4) |
| CYP3A5*3 |  |  |  |  |  |  |  |  |
| GG | 75 (82.4) | 4 (13.8) | 7 (70) | 86 (66.1) | 67 (83.7) | 1 (6.2) | 15 (75) | 83 (71.5) |
| AG and AA | 16 (17.6) | 25 (86.2) | 3 (30) | 44 (33.9) | 13 (16.3) | 15 (93.8) | 5 (25) | 33 (28.5) |
| GSTM1 deletion |  |  |  |  |  |  |  |  |
| Null | 42 (46.1) | 12 (41.4) | 5 (50) | 59 (45.4) | 37 (46.2) | 4 (25) | 5 (25) | 46 (39.7) |
| Nonnull | 49 (53.9) | 17 (58.6) | 5 (50) | 71 (54.6) | 43 (53.8) | 12 (75) | 15 (75) | 70 (60.3) |
| GSTP1 313A>G |  |  |  |  |  |  |  |  |
| GG | 9 (9.9) | 9 (31) | 3 (30) | 21 (16.2) | 15 (18.7) | 4 (25) | 1 (5) | 20 (17.2) |
| AG and AA | 82 (90.1) | 20 (69) | 7 (70) | 109 (83.8) | 65 (81.3) | 12 (75) | 19 (95) | 96 (82.8) |
| GSTT1 deletion |  |  |  |  |  |  |  |  |
| Null | 16 (17.6) | 3 (10.3) | 1 (10) | 20 (15.4) | 10 (12.5) | 5 (31.2) | 5 (25) | 20 (17.2) |
| Nonnull | 75 (82.4) | 26 (89.7) | 9 (90) | 110 (84.6) | 70 (87.5) | 11 (68.8) | 15 (75) | 96 (82.8) |
| MDR1 exon 21 G>T/A |  |  |  |  |  |  |  |  |
| GG | 19 (20.9) | 25 (86.2) | 5 (50) | 49 (37.7) | 24 (30) | 12 (75) | 6 (30) | 42 (36.2) |
| GT and others | 72 (79.1) | 4 (13.8) | 5 (50) | 81 (62.3) | 56 (70) | 4 (25) | 14 (70) | 74 (63.8) |
| MDR1 exon 26 C>T |  |  |  |  |  |  |  |  |
| CC | 14 (15.4) | 22 (75.9) | 2 (20) | 38 (29.2) | 20 (25) | 7 (43.7) | 5 (25) | 32 (27.6) |
| CT and TT | 77 (84.6) | 7 (24.1) | 8 (80) | 92 (70.8) | 60 (75) | 9 (56.3) | 15 (75) | 84 (72.4) |
| MTHFR 677C>T |  |  |  |  |  |  |  |  |
| CC | 40 (44) | 22 (75.9) | 6 (60) | 68 (52.3) | 36 (45) | 9 (56.3) | 9 (45) | 54 (46.5) |
| CT and TT | 51 (56) | 7 (24.1) | 4 (40) | 62 (47.7) | 44 (55) | 7 (43.7) | 11 (55) | 62 (53.5) |
| MTHFR 1298A>C |  |  |  |  |  |  |  |  |
| AA | 49 (53.9) | 19 (65.6) | 2 (20) | 70 (53.9) | 39 (48.7) | 11 (68.8) | 10 (50) | 60 (51.7) |
| AC and CC | 42 (46.1) | 10 (34.4) | 8 (80) | 60 (46.1) | 41 (51.3) | 5 (31.2) | 10 (50) | 56 (48.3) |
| NR3C1 1088A>G |  |  |  |  |  |  |  |  |
| AA | 86 (94.5) | 29 (100) | 9 (90) | 124 (95.4) | 78 (97.5) | 16 (100) | 20 (100) | 114 (98.3) |
| AG | 5 (5.5) | 0 (0) | 1 (10) | 6 (4.6) | 2 (2.5) | 0 (0) | 0 (0) | 2 (1.7) |
| RFC 80A>G |  |  |  |  |  |  |  |  |
| GG | 33 (36.3) | 8 (27.6) | 3 (30) | 44 (33.8) | 25 (31.2) | 4 (25) | 4 (20) | 33 (28.4) |
| AG and AA | 58 (63.7) | 21 (72.4) | 7 (70) | 86 (66.2) | 55 (68.8) | 12 (75) | 16 (80) | 83 (71.6) |
| TPMT combined genotypes |  |  |  |  |  |  |  |  |
| 238GG, 460GG and 719AA | 84 (92.3) | 28 (96.5) | 9 (90) | 121 (93.1) | 77 (96.3) | 15 (93.7) | 18 (90) | 110 (94.8) |
| Others | 7 (7.7) | 1 (3.5) | 1 (10) | 9 (6.9) | 3 (3.7) | 1 (6.3) | 2 (10) | 6 (5.2) |
| TYMS enhancer repeat |  |  |  |  |  |  |  |  |
| 3/3 | 23 (25.3) | 15 (51.7) | 5 (50) | 43 (33.1) | 19 (23.7) | 4 (25) | 12 (60) | 35 (30.2) |
| 2/2, 2/3 and others* | 68 (74.7) | 14 (48.3) | 5 (50) | 87 (66.9) | 61 (76.3) | 12 (75) | 8 (40) | 81 (69.8) |
| UGT1A1 promoter repeat |  |  |  |  |  |  |  |  |
| 7/7 | 8 (8.8) | 2 (6.9) | 1 (10) | 11 (8.5) | 7 (8.7) | 3 (18.7) | 4 (20) | 14 (12.1) |
| 6/6, 6/7 and others† | 83 (91.2) | 27 (93.1) | 9 (90) | 119 (91.5) | 73 (91.3) | 13 (81.3) | 16 (80) | 102 (87.9) |
| VDR intron 8 G>A |  |  |  |  |  |  |  |  |
| GG | 39 (42.9) | 18 (62.1) | 4 (40) | 61 (46.9) | 30 (37.5) | 9 (56.3) | 12 (60) | 51 (44) |
| GA and AA | 52 (57.1) | 11 (37.9) | 6 (60) | 69 (53.1) | 50 (62.5) | 7 (43.7) | 8 (40) | 65 (56) |
| VDR Fokl |  |  |  |  |  |  |  |  |
| CC | 37 (40.7) | 17 (58.6) | 5 (50) | 59 (45.4) | 30 (37.5) | 10 (62.5) | 13 (65) | 53 (45.7) |
| TC and T T | 54 (59.3) | 12 (41.4) | 5 (50) | 71 (54.6) | 50 (62.5) | 6 (37.5) | 7 (35) | 63 (54.3) |

Table 1 Caption: Genotype frequencies by race and risk group

To assess possible pharmacogenetic determinants of relapse, a classification and regression tree (CART) approach was used.[41](#ref41) Statistical analysis was performed on each polymorphism individually to make a decision tree for prediction of relapses. For each outcome, the most significant polymorphism determined the second node of the CART (after treatment arm), and polymorphisms for the remaining 15 loci were then analyzed, and the process was repeated. At each node, all the loci that had not yet been assessed for association between genotypes with outcome were analyzed, and the polymorphism with the most significant association (by the smallest *P* value using Gray test) was used to define that node and the downward branches. The CARTs were stopped either when no further significant genotypes were observed or when the number of patients in the terminal nodes was equal to 10. In these univariate analyses, Gray test was used to compare the cumulative incidences of relapse by genotypic groups.[40](#ref40)

To assess whether genetic variants had independent prognostic significance for outcome, we performed multiple regression analyses adjusting for the established risk factors in this protocol[21](#ref21): initial leukocyte count (< 100 × 10^9^/L versus ≥ 100 × 10^9^/L), DNA index (1.16-1.6 versus others), presence versus absence of *BCR-ABL* [t(9;22)] or of *MLL-AF4* [t(4;11)], percentage of blasts in the bone marrow on day 19 (< 5% blasts versus ≥ 5% blasts), and minimal residual disease on remission date (< 0.01% versus ≥ 0.01%). For the analysis of CNS relapse, CNS status at diagnosis was also included as a possible prognostic factor (no blasts in cerebrospinal fluid in an atraumatic or traumatic lumbar puncture versus other CNS status). Fine and Gray estimator with incorporation of competing events was used to compare the cumulative incidence of relapse by genotypic groups.[42](#ref42)

To assess the possible correlation between genotypes at the 16 polymorphic loci, pair-wise associations were performed using Fisher exact or chi-square tests. We used Wilcoxon rank-sum test to compare the expression levels of *GSTM1* and *TYMS* among genotypic groups.

## Results

After a median follow-up of 5.3 years, there have been a total of 47 adverse events (relapses, second malignancies, and deaths in remission), including 9 relapses in the lower-risk arm and 23 in the higher-risk arm. Because the type of treatment, as well as relapse incidence and event-free survival, differed by risk group (*P* = .01 and *P* = .002, respectively), we performed the analysis separately in the lower- and higher-risk groups. Although many allele frequencies differed significantly between blacks and whites ([Table 1](#tbl1)), and race has been associated with ALL cure rates in other studies,[23](#ref23)^,^[24](#ref24) race was not a prognostic factor in this single-institution protocol.[21](#ref21)^,^[43](#ref43) Thus, the genotype/phenotype association analysis was not stratified by race. Genotypes of the 16 loci in 246 patients are summarized in [Figure 2](#fig2).

### Figure 2.

![Figure 2.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5ac4/1895006/21cf1ce2135f/zh80120579930002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=1895006_zh80120579930002.jpg)

Visual genotype graph for 16 polymorphic loci (rows) among the 246 patients (columns) with ALL treated in the Total XIIIB Protocol. Those homozygous for the major allele are depicted in blue, homozygous for the minor allele in yellow, and heterozygous are in green. For GSTM1 and GSTT1 deletions, the nonnull and the null genotypes are represented in blue and in yellow, respectively. The rare genotypes for TYMS and UGT1A1 were excluded from the plot (in white).

Among those on the higher-risk arm, the nonnull *GSTM1* genotype ([Figure 3A](#fig3)) was associated with an increased risk of hematologic relapse (isolated or combined) (5-year cumulative incidence, 17.1% ± 4.5% versus 5.1% ± 2.9% for *GSTM1* null genotype, *P* = .03). Among those with the *GSTM1* nonnull genotype, the presence of the *TYMS* 3/3 genotype was associated with a higher-risk of hematologic relapse (5-year cumulative incidence, 29.2% ± 9.5% versus 10.9% ± 4.7% for *TYMS* 2/3 or 2/2 genotypes, *P* = .02) ([Figure 3B](#fig3)), whereas among those with *GSTM1* null genotype, the *NR3C1* AG genotype was associated with an increased risk of hematologic relapse (5-year cumulative incidence, 33.3% ± 33.3% versus 3.6% ± 2.5% for *NR3C1* AA genotype, *P* = .02). In univariate analyses, the only other significant predictors of hematologic relapse were the presence of *BCR-ABL* [t(9;22)] at diagnosis (*P* < .001), the percentage of blasts in the day 19 bone marrow (*P* < .001), and minimal residual disease status on remission date (*P* < .001). In multivariate analyses, including other features predictive of hematologic relapse, either the *GSTM1* nonnull genotype alone (*P* < .001) or the combined *GSTM1* nonnull and *TYMS* 3/3 genotypes (*P* = .003), were independent prognostic factors for hematologic relapse (Tables [2](#tbl2) and [3](#tbl3)). No genotype was associated with hematologic relapse among those patients on the lower-risk arm.

### Figure 3.

![Figure 3.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5ac4/1895006/e542b743ded8/zh80120579930003.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=1895006_zh80120579930003.jpg)

Cumulative incidence curves for the risk of hematologic relapse for ALL patients assigned in the higher-risk arm of the Total XIIIB Protocol. Children with the GSTM1 nonnull genotype (n = 71) experienced a higher risk of relapse compared with children with the null genotype (n = 59) (P = .03) (A). Among patients with the GSTM1 nonnull genotype, those with the TYMS 3/3 genotype (n = 24) were at even higher risk of relapse compared with those with the TYMS 2/2 or 2/3 genotypes (n = 47) (P = .03) (B).

### Table 2.

| Feature* | Coefficient | SE | Hazard ratio (95% CI) | P† |
| --- | --- | --- | --- | --- |
| GSTM1 (nonnull vs null) | 2.90 | 0.784 | 18.1 (3.9-84.3) | .0002 |
| Leukocyte count (fewer than 100 × 109/L vs at least 100 × 109/L) | -1.74 | 0.844 | 0.18 (0.03-0.92) | .04 |
| t(9;22)/BCR-ABL (present vs absent) | 0.83 | 0.949 | 2.29 (0.36-14.7) | .38 |
| t(4;11)/MLL-AF4 (present vs absent) | 2.94 | 0.972 | 19.0 (2.82-128) | .0025 |
| Day 19 marrow (at least 5% blasts vs less than 5% blasts) | -0.46 | 0.888 | 0.63 (0.11-3.58) | .60 |
| Minimal residual disease on remission date (at least 0.01% vs less than 0.01%) | 2.69 | 0.972 | 14.7 (2.18-98.5) | .0057 |

Table 2 Caption: Risk of hematologic relapse among patients on the higher-risk arm, based on GSTM1 genotype, by multivariate analysis

### Table 3.

| Feature* | Coefficient | SE | Hazard ratio (95% CI) | P† |
| --- | --- | --- | --- | --- |
| Combined GSTM1 and TYMS (GSTM1 nonnull + TYMS 3/3 vs others) | 2.67 | 0.905 | 14.5 (2.46-85.2) | .0031 |
| Leukocyte count (fewer than 100 × 109/L vs at least 100 × 109/L) | -1.95 | 0.964 | 0.14 (0.02-0.94) | .043 |
| t(9;22)/BCR-ABL (present vs absent) | 0.38 | 0.906 | 1.47 (0.25-8.66) | .67 |
| t(4;11)/MLL-AF4 (present vs absent) | 3.13 | 0.985 | 22.8 (1.20-157) | .0015 |
| Day 19 marrow (at least 5% blasts vs less than 5% blasts) | 1.07 | 0.551 | 2.92 (0.99-8.61) | .052 |
| Minimal residual disease on remission date (at least 0.01% vs less than 0.01%) | 2.75 | 1.00 | 15.7 (2.20-112) | .0060 |

Table 3 Caption: Risk of hematologic relapse among patients on the higher-risk arm, based on combined GSTM1 nonnull and TYMS genotypes, by multivariate analysis

To explore whether genotypes were related to expression of those genes in the target tissue, we compared gene expression levels in the same set of patients in whom *GSTM1* and *TYMS* genotypes were prognostic. In diagnostic ALL blasts from children treated with the higher-risk therapy, those with the *GSTM1* nonnull genotype had higher *GSTM1* gene expression than those with the null genotype (*P* < .001), and among those with the *GSTM1* nonnull genotype, there was a trend for higher *TYMS* expression in those with the *TYMS* 3/3 compared with 2/3 or 2/2 genotypes (*P* = .09) ([Figure 4](#fig4)).

### Figure 4.

![Figure 4.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5ac4/1895006/533a1eed1a9f/zh80120579930004.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=1895006_zh80120579930004.jpg)

Expression levels (natural log-scaled signal intensities) of GSTM1, based on GSTM1 genotype among patients in the higher-risk arm, and of TYMS, based on TYMS genotype, among those with GSTM1 nonnull genotype. The boxes and central horizontal define the quartiles and median; the whiskers represent the 10th and 90th percentiles. The number of patients (n) included in each group is indicated below the plots.

Because different polymorphisms might be important for drug penetration into the CNS, we performed a separate analysis of genetic determinants of CNS relapse (isolated or combined), although only 6 and 2 CNS relapses occurred on the higher- and lower-risk arms, respectively. On the higher-risk arm, the *VDR* FokI T allele was associated with an increased risk of CNS relapse (5-year cumulative incidence, 8.6% ± 3.4% versus 0% for *VDR* FokI CC genotype, *P* = .02). All CNS relapses occurred among those with at least 1 *VDR* FokI T allele, but the frequency of *VDR* FokI genotypes did not differ (*P* > .14 for all) between patients with high- versus low-risk features (by initial leukocyte count, CNS status at diagnosis, DNA index, *BCR-ABL* [t(9;22)], *MLL-AF4* [t(4;11)], day 19 bone marrow blasts, and minimal residual disease). Among those with at least 1 *VDR* FokI T allele, the *VDR* intron 8 GG genotype was associated with a greater risk of CNS relapse (5-year cumulative incidence, 15.6% ± 6.6% versus 2.7% ± 2.7% for *VDR* intron 8 AA genotype, *P* = .04), and the combined *VDR* FokI and *VDR* intron 8 genotypes were significantly associated with CNS relapse (*P* = .01) in a multivariate analysis including other prognostic features. On the lower-risk arm, the *TYMS* 3/3 genotype was associated with an increased risk of CNS relapse (5-year cumulative incidence, 5.7% ± 4% versus 0% for *TYMS* 2/3 or 2/2 genotypes, *P* = .04).

We compared the frequencies of significantly predictive genotypes in the higher- and lower-risk arms: no predictive polymorphisms differed in frequency between risk groups. Within risk groups, the only association of predictive genotypes with other patient features was among the higher-risk group: *GSTM1* null genotype was more common (*P* = .03) among those with initial leukocyte count at least 100 × 10^9^/L (57.9%) than those with less than 100 × 10^9^/L (36.8%), but *GSTM1* remained a significant predictor of hematologic relapse even after adjustment for presenting leukocyte counts and other prognostic features ([Table 2](#tbl2)). To assess the possibility that the association of genotypes with outcome was confounded by associations among genotypes, we assessed the correlations among genotypes at the 16 polymorphic loci. Although there was the expected linkage disequilibrium within some loci (eg, *CYP3A4* with *CYP3A5, MDR1* exon 21 with *MDR1* exon 26 polymorphisms), there was no significant association between any of the predictive genotypes (eg, *GSTM1* was not associated with *TYMS* or *NR3C1* genotypes) (data not shown).

## Discussion

The most common reason for failure in childhood ALL is hematologic relapse. We previously found that the only prognostic features in this protocol were risk group (treatment arm), initial leukocyte count, DNA index, *BCR-ABL* [t(9;22)], *MLL-AF4* [t(4;11)], and early treatment response.[21](#ref21) Herein, after accounting for the difference in relapse hazard because of treatment arms, risk group, and other prognostic features (Tables [2](#tbl2) and [3](#tbl3)), the incidence of hematologic relapse was significantly associated with common germ line genetic polymorphisms. In the higher-risk arm, the *GSTM1* deletion was a determinant for the risk of hematologic relapse: 13 of the 16 children who experienced a hematologic relapse had the nonnull genotype. Our analysis allowed us to identify gene-gene interactions that modified the relapse risk. For patients with the *GSTM1* nonnull genotype, the presence of the higher activity *TYMS* 3/3 genotype was associated with greater risk of hematologic relapse (*P* = .03). That *GSTM1* and *TYMS* genotypes remained prognostic even after adjustment for factors reflective of early treatment response suggests that these genotypes continue to have an impact on prognosis for treatment administered beyond the remission induction period. This finding is plausible, in that drugs that are substrates for glutathione *S*-transferases (GSTs; eg, cyclophosphamide, etoposide), or that target thymidylate synthetase (TYMS; eg, methotrexate) were important components of the therapy given after induction. Although confidence intervals are large, the hazard ratios associated with unfavorable *GSTM1* or combined *GSTM1/TYMS* genotypes are on par with those associated with features such as *MLL* translocations or the presence of minimal residual disease (Tables [2](#tbl2) and [3](#tbl3)). Among those with the *GSTM1* null genotype, the *NR3C1* G allele was associated with an increased relapse risk, although a very small number of patients carried the G allele.

No polymorphisms were associated with hematologic relapse among patients treated on the lower-risk arm. That the *GSTM1* germ line polymorphism was important for relapse among patients in the higher-risk arm, but not in the lower-risk arm, is consistent with more intensive therapy given to the former patients.[20](#ref20) GSTs catalyze the inactivation of many antileukemic agents and their metabolites (eg, cyclophosphamide, anthracyclines, etoposide, and steroids)[28](#ref28) and protect against oxidative stress. Null *GST* genotypes have been linked with improved outcome in some ALL trials[14](#ref14)^,^[16](#ref16) but not in others.[17](#ref17) Our study is distinguished by the fact that virtually all patients on the trial were included in this analysis. Because cyclophosphamide and etoposide constituted a major proportion of therapy in the higher-risk patients, it is plausible that *GST* genotype was of greater importance in the higher-risk than in the lower-risk arm. Using pretreatment ALL blasts, we confirmed that the germ line nonnull *GSTM1* genotype is associated with higher *GSTM1* expression, in support of the hypothesis that the inferior outcome associated with the nonnull genotype is related to the increased detoxification conferred by *GSTM1* expression.

The *TYMS* enhancer repeat 3/3 genotype causes higher expression and activity of one of the major targets of methotrexate, thymidylate synthetase.[11](#ref11)^,^[33](#ref33) Higher thymidylate synthetase requires higher concentrations of methotrexate and its metabolites for inhibition[33](#ref33) and cytotoxicity.[44](#ref44)^,^[45](#ref45) Although one single-gene study reported shorter event-free survival in children with ALL with the 3/3 genotype,[18](#ref18) another case-control study of 80 patients failed to find such an association.[15](#ref15) Such disparate findings may be due to differences in intensity of methotrexate therapy between studies. In our study, among patients on the higher-risk arm with an unfavorable *GSTM1* status, the *TYMS* expression tended to be higher ([Figure 4](#fig4)), and the *TYMS* 3/3 genotype was associated with a higher risk of hematologic relapse. Our findings illustrate the potential for gene-gene interactions: in the patients with low glutathione transferase activity, it is possible that methotrexate was a less critical element of therapy than among those with high glutathione transferase; thus, a polymorphism in the target of methotrexate (thymidylate synthetase) was of less importance in those with a favorable (low activity) than an unfavorable (high activity) *GSTM1* genotype.

Only 2 CNS relapses occurred in the lower-risk group, and both patients were homozygous for the high-activity *TYMS* 3/3 genotype. For those on the lower-risk arm, methotrexate (given as systemic high doses and as intrathecal injections) was the primary form of CNS prophylaxis; thus, it is plausible that this unfavorable genotype might constitute a therapeutic disadvantage for this subset of patients.

In the higher-risk arm, 2 polymorphisms in the *VDR* locus were associated with risk of CNS relapse, which remained significant predictors after adjusting for other prognostic features. The VDR regulates the expression of CYP3A4 and possibly p-glycoprotein,[34](#ref34)^,^[35](#ref35) which are important in the disposition of vincristine, etoposide, daunorubicin, prednisone, and dexamethasone. Both *VDR* polymorphisms have been associated with many clinical phenotypes.[46](#ref46)^-^[48](#ref48) If the *VDR* FokI T allele and intron 8 GG genotype were associated with higher expression of p-glycoprotein, their association with an increased risk of relapse might be consistent with worse penetration of active drugs into the CNS (due to higher p-glycoprotein in the blood-brain barrier) and with higher risk of relapse because of greater p-glycoprotein-mediated biliary excretion of antileukemic drugs. Whether polymorphisms in *VDR* directly affect drug sensitivity or expression of downstream targets (CYP3A4 and/or p-glycoprotein) is not clear.

Relationships between genotypes and outcome could be influenced by a causative relationship between the polymorphism and response to therapy, or by an a priori relationship between the polymorphism and the development of specific subtypes of ALL. Our data favor the former rather than the latter. First, no polymorphisms differed in allele frequency between risk groups. Second, among the higher-risk group, although the *GSTM1* genotype was associated with a high presenting leukocyte count, *GSTM1* remained a significant predictor of relapse after adjusting for this feature.

This single-institution protocol was an ideal platform in which to assess how genotypes predicted outcomes in a racially diverse patient group, because, although genotypes differed by race, this is one of the first studies in which ALL cure rates did not differ by race.[21](#ref21)^;^[43](#ref43) Thus, the genotype/outcome relationships we identified maintained prognostic significance in a therapeutic setting in which race itself was unimportant. Moreover, the same genotypes (*GSTM1* nonnull and *TYMS* 3/3) were predictive for hematologic relapse within the major subgroups of whites and blacks (data not shown). Dosing based on pharmacogenetics holds the promise for delivery of “color-blind” therapy: genotyping allows for individualized dosing according to genetic rather than racial characteristics.[49](#ref49)

Improved outcomes in patients with ALL depend on interactions among drugs, ALL blast sensitivity, and host factors. Currently, patients whose ALL blasts have unfavorable acquired genetic characteristics receive more intensive therapy. In our study of 16 target polymorphic loci, several common polymorphisms were found to be prognostic, most of which have pharmacologic plausibility. However, we highlight that prognostic genotypes are highly treatment dependent; hence, caution should be used in attempting to generalize these results to other protocols. Both of the primary polymorphisms associated with relapse, the high activity *TYMS* 3/3 and the *GSTM1* nonnull genotypes, would be expected to cause drug resistance and were linked with higher gene expression in the target tissue of ALL blasts. We acknowledge that it is likely that additional pharmacogenetic prognostic factors will be identified in other protocols and that genotypes that were not prognostic in this study might be important in other protocols: as with all prognostic features, they depend on treatment. In the future, patients may be classified not only on the genetic characteristics of their blasts but also on host genetic characteristics, so that therapy could be intensified according to a pharmacogenetic index of drug resistance. We conclude that pharmacogenetics influences the outcome of antileukemic therapy and may provide a tool to improve individualization of therapy.

## Acknowledgments

We thank our protocol coinvestigators, clinical staff, research nurses (Sheri Ring, Lisa Walters, Margaret Edwards, Terri Kuehner, and Paula Condy), the patients and their parents for their participation. We also thank Jean Cai, Pam McGill, and Nancy Duran for laboratory assistance and Wenjian Yang, Nancy Kornegay, Carl Panetta, and Mark Wilkinson for computing assistance.

All the authors contributed substantially to the manuscript. M.V.R. and W.E.E. designed the research; J.C.C.R., S.K., S.D., E.H.C., J.T.S., J.R., R.R., D.C., C.-H.P., W.E.E., and M.V.R. performed the research; J.C.R., C.C., and W.L. analyzed the data; and J.C.R. and M.V.R. were primary authors of the paper.

## References

1. Pui CH, Relling MV, Downing JR. Acute lymphoblastic leukemia. N Engl J Med. 2004;350: 1535-1548.  [DOI](https://doi.org/10.1056/NEJMra023001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15071128/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med.&volume=350&publication_year=2004&pages=1535&pmid=15071128&doi=10.1056/NEJMra023001&)

2. Nachman JB, Sather HN, Sensel MG, et al. Augmented post-induction therapy for children with high-risk acute lymphoblastic leukemia and a slow response to initial therapy. N Engl J Med. 1998;338: 1663-1671.  [DOI](https://doi.org/10.1056/NEJM199806043382304) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9614257/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med.&volume=338&publication_year=1998&pages=1663&pmid=9614257&doi=10.1056/NEJM199806043382304&)

3. Schrappe M, Reiter A, Ludwig WD, et al. Improved outcome in childhood acute lymphoblastic leukemia despite reduced use of anthracyclines and cranial radiotherapy: results of trial ALL-BFM 90 German-Austrian-Swiss ALL-BFM Study Group. Blood. 2000;95: 3310-3322.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/10828010/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood.&volume=95&publication_year=2000&pages=3310&pmid=10828010&)

4. Silverman LB, Gelber RD, Dalton VK, et al. Improved outcome for children with acute lymphoblastic leukemia: results of Dana-Farber Consortium Protocol 91-01. Blood. 2001;97: 1211-1218.  [DOI](https://doi.org/10.1182/blood.v97.5.1211) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11222362/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood.&volume=97&publication_year=2001&pages=1211&pmid=11222362&doi=10.1182/blood.v97.5.1211&)

5. Pui CH, Gaynon PS, Boyett JM, et al. Outcome of treatment in childhood acute lymphoblastic leukaemia with rearrangements of the 11q23 chromosomal region. Lancet. 2002;359: 1909-1915.  [DOI](https://doi.org/10.1016/S0140-6736(02)08782-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12057554/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet.&volume=359&publication_year=2002&pages=1909&pmid=12057554&doi=10.1016/S0140-6736(02)08782-2&)

6. Pui CH, Evans WE. Acute lymphoblastic leukemia. N Engl J Med. 1998;339: 605-615.  [DOI](https://doi.org/10.1056/NEJM199808273390907) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9718381/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med.&volume=339&publication_year=1998&pages=605&pmid=9718381&doi=10.1056/NEJM199808273390907&)

7. Evans WE, Relling MV, Rodman JH, et al. Conventional compared with individualized chemotherapy for childhood acute lymphoblastic leukemia. N Engl J Med. 1998;338: 499-505.  [DOI](https://doi.org/10.1056/NEJM199802193380803) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9468466/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med.&volume=338&publication_year=1998&pages=499&pmid=9468466&doi=10.1056/NEJM199802193380803&)

8. Evans WE, Crom WR, Abromowitch M, et al. Clinical pharmacodynamics of high-dose methotrexate in acute lymphocytic leukemia. N Engl J Med. 1986;314: 471-477.  [DOI](https://doi.org/10.1056/NEJM198602203140803) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/3456079/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med.&volume=314&publication_year=1986&pages=471&pmid=3456079&doi=10.1056/NEJM198602203140803&)

9. Relling MV, Hancock ML, Boyett JM, Pui C-H, Evans WE. Prognostic importance of 6-mercaptopurine dose intensity in acute lymphoblastic leukemia. Blood. 1999;93: 2817-2823.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/10216075/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood.&volume=93&publication_year=1999&pages=2817&pmid=10216075&)

10. Camitta B, Mahoney D, Leventhal B, et al. Intensive intravenous methotrexate and mercaptopurine treatment of higher-risk non-T, non-B acute lymphocytic leukemia: a Pediatric Oncology Group study. J Clin Oncol. 1994;12: 1383-1389.  [DOI](https://doi.org/10.1200/JCO.1994.12.7.1383) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8021728/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol.&volume=12&publication_year=1994&pages=1383&pmid=8021728&doi=10.1200/JCO.1994.12.7.1383&)

11. Evans WE, McLeod HL. Pharmacogenomics—drug disposition, drug targets, and side effects. N Engl J Med. 2003;348: 538-549.  [DOI](https://doi.org/10.1056/NEJMra020526) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12571262/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med.&volume=348&publication_year=2003&pages=538&pmid=12571262&doi=10.1056/NEJMra020526&)

12. Weinshilboum R. Inheritance and drug response. N Engl J Med. 2003;348: 529-537.  [DOI](https://doi.org/10.1056/NEJMra020021) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12571261/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med.&volume=348&publication_year=2003&pages=529&pmid=12571261&doi=10.1056/NEJMra020021&)

13. Lennard L, Lilleyman JS, Van Loon J, Weinshilboum RM. Genetic variation in response to 6-mercaptopurine for childhood acute lymphoblastic leukaemia. Lancet. 1990;336: 225-229.  [DOI](https://doi.org/10.1016/0140-6736(90)91745-v) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1973780/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet.&volume=336&publication_year=1990&pages=225&pmid=1973780&doi=10.1016/0140-6736(90)91745-v&)

14. Stanulla M, Schrappe M, Brechlin AM, Zimmermann M, Welte K. Polymorphisms within glutathione S-transferase genes (GSTM1, GSTT1, GSTP1) and risk of relapse in childhood B-cell precursor acute lymphoblastic leukemia: a case-control study. Blood. 2000;95: 1222-1228.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/10666194/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood.&volume=95&publication_year=2000&pages=1222&pmid=10666194&)

15. Lauten M, Asgedom G, Welte K, Schrappe M, Stanulla M. Thymidylate synthase gene polymorphism and its association with relapse in childhood B-cell precursor acute lymphoblastic leukemia. Haematologica. 2003;88: 353-354.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/12651279/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Haematologica.&volume=88&publication_year=2003&pages=353&pmid=12651279&)

16. Chen C-L, Liu Q, Pui C-H, et al. Higher frequency of glutathione S-transferase deletions in black children with acute lymphoblastic leukemia. Blood. 1997;89: 1701-1707.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/9057653/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood.&volume=89&publication_year=1997&pages=1701&pmid=9057653&)

17. Davies SM, Bhatia S, Ross JA. et al. Glutathione S-transferase genotypes, genetic susceptibility, and outcome of therapy in childhood acute lymphoblastic leukemia. Blood. 2002;100: 67-71.  [DOI](https://doi.org/10.1182/blood.v100.1.67) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12070010/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood.&volume=100&publication_year=2002&pages=67&pmid=12070010&doi=10.1182/blood.v100.1.67&)

18. Krajinovic M, Costea I, Chiasson S. Polymorphism of the thymidylate synthase gene and outcome of acute lymphoblastic leukaemia. Lancet. 2002;359: 1033-1034.  [DOI](https://doi.org/10.1016/S0140-6736(02)08065-0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11937185/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet.&volume=359&publication_year=2002&pages=1033&pmid=11937185&doi=10.1016/S0140-6736(02)08065-0&)

19. Krajinovic M, Labuda D, Mathonnet G, et al. Polymorphisms in genes encoding drugs and xenobiotic metabolizing enzymes, DNA repair enzymes, and response to treatment of childhood acute lymphoblastic leukemia. Clin Cancer Res. 2002;8: 802-810.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/11895912/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res.&volume=8&publication_year=2002&pages=802&pmid=11895912&)

20. Relling MV, Boyett JM, Blanco JG, et al. Granulocyte-colony stimulating factor and the risk of secondary myeloid malignancy after etoposide treatment. Blood. 2003;101: 3862-3867.  [DOI](https://doi.org/10.1182/blood-2002-08-2405) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12531808/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood.&volume=101&publication_year=2003&pages=3862&pmid=12531808&doi=10.1182/blood-2002-08-2405&)

21. Pui CH, Sandlund JT, Pei D, et al. Improved outcome for children with acute lymphoblastic leukemia: results of Total Therapy Study XIIIB at St Jude Children's Research Hospital. Blood. 2004;104: 2690-2696.  [DOI](https://doi.org/10.1182/blood-2004-04-1616) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15251979/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood.&volume=104&publication_year=2004&pages=2690&pmid=15251979&doi=10.1182/blood-2004-04-1616&)

22. Coustan-Smith E, Sancho J, Hancock ML, et al. Clinical importance of minimal residual disease in childhood acute lymphoblastic leukemia. Blood. 2000;96: 2691-2696.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/11023499/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood.&volume=96&publication_year=2000&pages=2691&pmid=11023499&)

23. Kadan-Lottick NS, Ness KK, Bhatia S, Gurney JG. Survival variability by race and ethnicity in childhood acute lymphoblastic leukemia. JAMA. 2003;290: 2008-2014.  [DOI](https://doi.org/10.1001/jama.290.15.2008) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14559954/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA.&volume=290&publication_year=2003&pages=2008&pmid=14559954&doi=10.1001/jama.290.15.2008&)

24. Pollock BH, DeBaun MR, Camitta BM, et al. Racial differences in the survival of childhood B-precursor acute lymphoblastic leukemia: a Pediatric Oncology Group Study. J Clin Oncol. 2000;18: 813-823.  [DOI](https://doi.org/10.1200/JCO.2000.18.4.813) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10673523/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol.&volume=18&publication_year=2000&pages=813&pmid=10673523&doi=10.1200/JCO.2000.18.4.813&)

25. Blanco JG, Edick MJ, Hancock ML, et al. Genetic polymorphisms in CYP3A5, CYP3A4 and NQO1 in children who developed therapy-related myeloid malignancies. Pharmacogenetics. 2002;12: 605-611.  [DOI](https://doi.org/10.1097/00008571-200211000-00004) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12439220/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics.&volume=12&publication_year=2002&pages=605&pmid=12439220&doi=10.1097/00008571-200211000-00004&)

26. Pui CH, Cheng C, Leung W, et al. Extended follow-up of long-term survivors of childhood acute lymphoblastic leukemia. N Engl J Med. 2003;349: 640-649.  [DOI](https://doi.org/10.1056/NEJMoa035091) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12917300/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med.&volume=349&publication_year=2003&pages=640&pmid=12917300&doi=10.1056/NEJMoa035091&)

27. Pastan I, Gottesman MM. Multidrug resistance. Annu Rev Med. 1991;42: 277-286.  [DOI](https://doi.org/10.1146/annurev.me.42.020191.001425) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2035973/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annu%20Rev%20Med.&volume=42&publication_year=1991&pages=277&pmid=2035973&doi=10.1146/annurev.me.42.020191.001425&)

28. Townsend DM, Tew KD. The role of glutathione-S-transferase in anti-cancer drug resistance. Oncogene. 2003;22: 7369-7375.  [DOI](https://doi.org/10.1038/sj.onc.1206940) | [PMC free article](/articles/PMC6361125/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14576844/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncogene.&volume=22&publication_year=2003&pages=7369&pmid=14576844&doi=10.1038/sj.onc.1206940&)

29. Tissing WJ, Meijerink JP, den Boer ML, Pieters R. Molecular determinants of glucocorticoid sensitivity and resistance in acute lymphoblastic leukemia. Leukemia. 2003;17: 17-25.  [DOI](https://doi.org/10.1038/sj.leu.2402733) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12529655/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Leukemia.&volume=17&publication_year=2003&pages=17&pmid=12529655&doi=10.1038/sj.leu.2402733&)

30. Weisberg IS, Jacques PF, Selhub J, et al. The 1298A→C polymorphism in methylenetetrahydrofolate reductase (MTHFR): in vitro expression and association with homocysteine. Atherosclerosis. 2001;156: 409-415.  [DOI](https://doi.org/10.1016/s0021-9150(00)00671-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11395038/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Atherosclerosis.&volume=156&publication_year=2001&pages=409&pmid=11395038&doi=10.1016/s0021-9150(00)00671-7&)

31. Huizenga NA, Koper JW, De Lange P, et al. A polymorphism in the glucocorticoid receptor gene may be associated with and increased sensitivity to glucocorticoids in vivo. J Clin Endocrinol Metab. 1998;83: 144-151.  [DOI](https://doi.org/10.1210/jcem.83.1.4490) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9435432/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Endocrinol%20Metab.&volume=83&publication_year=1998&pages=144&pmid=9435432&doi=10.1210/jcem.83.1.4490&)

32. Yates CR, Krynetski EY, Loennechen T, et al. Molecular diagnosis of thiopurine S-methyltransferase deficiency: genetic basis for azathioprine and mercaptopurine intolerance. Ann Intern Med. 1997;126: 608-614.  [DOI](https://doi.org/10.7326/0003-4819-126-8-199704150-00003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9103127/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Intern%20Med.&volume=126&publication_year=1997&pages=608&pmid=9103127&doi=10.7326/0003-4819-126-8-199704150-00003&)

33. Wang W, Marsh S, Cassidy J, McLeod HL. Pharmacogenomic dissection of resistance to thymidylate synthase inhibitors. Cancer Res. 2001;61: 5505-5510.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/11454699/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res.&volume=61&publication_year=2001&pages=5505&pmid=11454699&)

34. Schmiedlin-Ren P, Thummel KE, Fisher JM, et al. Expression of enzymatically active CYP3A4 by Caco-2 cells grown on extracellular matrix-coated permeable supports in the presence of 1alpha,25-dihydroxyvitamin D3. Mol Pharmacol. 1997;51: 741-754.  [DOI](https://doi.org/10.1124/mol.51.5.741) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9145912/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Pharmacol.&volume=51&publication_year=1997&pages=741&pmid=9145912&doi=10.1124/mol.51.5.741&)

35. Thummel KE, Brimer C, Yasuda K, et al. Transcriptional control of intestinal cytochrome P-4503A by 1alpha,25-dihydroxy vitamin D3. Mol Pharmacol. 2001;60: 1399-1406.  [DOI](https://doi.org/10.1124/mol.60.6.1399) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11723248/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Pharmacol.&volume=60&publication_year=2001&pages=1399&pmid=11723248&doi=10.1124/mol.60.6.1399&)

36. Watanabe Y, Nakajima M, Ohashi N, Kume T, Yokoi T. Glucuronidation of etoposide in human liver microsomes is specifically catalyzed by UDP-glucuronosyltransferase 1A1. Drug Metab Dispos. 2003;31: 589-595.  [DOI](https://doi.org/10.1124/dmd.31.5.589) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12695347/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos.&volume=31&publication_year=2003&pages=589&pmid=12695347&doi=10.1124/dmd.31.5.589&)

37. Kishi S, Yang W, Boureau B, et al. Effects of prednisone and genetic polymorphisms on etoposide disposition in children with acute lymphoblastic leukemia. Blood. 2004;103: 67-72.  [DOI](https://doi.org/10.1182/blood-2003-06-2105) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12969965/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood.&volume=103&publication_year=2004&pages=67&pmid=12969965&doi=10.1182/blood-2003-06-2105&)

38. Kishi S, Griener JC, Cheng C, et al. Homocysteine, pharmacogenetics, and neurotoxicity in children with leukemia. J Clin Oncol. 2003;21: 3084-3091.  [DOI](https://doi.org/10.1200/JCO.2003.07.056) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12915598/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol.&volume=21&publication_year=2003&pages=3084&pmid=12915598&doi=10.1200/JCO.2003.07.056&)

39. Yeoh EJ, Ross ME, Shurtleff SA, et al. Classification, subtype discovery, and prediction of outcome in pediatric acute lymphoblastic leukemia by gene expression profiling. Cancer Cell. 2002;1: 133-143.  [DOI](https://doi.org/10.1016/s1535-6108(02)00032-6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12086872/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Cell.&volume=1&publication_year=2002&pages=133&pmid=12086872&doi=10.1016/s1535-6108(02)00032-6&)

40. Gray RJ. A class of K-sample tests for comparing the cumulative incidence of a competing risk. Ann Statistics. 1988;16: 1141-1154.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Statistics.&volume=16&publication_year=1988&pages=1141&)

41. Breiman L, Friedman JH, Olshen RA, Stone CJ. Classification and Regression Trees. Wadsworth and Brooks, Pacific Grove, CA 1984.

42. Fine JP, Gray RJ. A proportional hazards model for the subdistribution of a competing risk. J A Stat Assoc. 1999;94: 496-509.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20A%20Stat%20Assoc.&volume=94&publication_year=1999&pages=496&)

43. Pui CH, Sandlund JT, Pei D, et al. Results of therapy for acute lymphoblastic leukemia in black and white children. JAMA. 2003;290: 2001-2007.  [DOI](https://doi.org/10.1001/jama.290.15.2001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14559953/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA.&volume=290&publication_year=2003&pages=2001&pmid=14559953&doi=10.1001/jama.290.15.2001&)

44. Johnston PG, Mick R, Recant W, et al. Thymidylate synthase expression and response to neoadjuvant chemotherapy in patients with advanced head and neck cancer. J Natl Cancer Inst. 1997;89: 308-313.  [DOI](https://doi.org/10.1093/jnci/89.4.308) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9048835/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Natl%20Cancer%20Inst.&volume=89&publication_year=1997&pages=308&pmid=9048835&doi=10.1093/jnci/89.4.308&)

45. Villafranca E, Okruzhnov Y, Dominguez MA, et al. Polymorphisms of the repeated sequences in the enhancer region of the thymidylate synthase gene promoter may predict downstaging after preoperative chemoradiation in rectal cancer. J Clin Oncol. 2001;19: 1779-1786.  [DOI](https://doi.org/10.1200/JCO.2001.19.6.1779) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11251009/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol.&volume=19&publication_year=2001&pages=1779&pmid=11251009&doi=10.1200/JCO.2001.19.6.1779&)

46. Morrison NA, Qi JC, Tokita A, et al. Prediction of bone density from vitamin D receptor alleles. Nature. 1994;367: 284-287.  [DOI](https://doi.org/10.1038/367284a0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8161378/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature.&volume=367&publication_year=1994&pages=284&pmid=8161378&doi=10.1038/367284a0&)

47. Eisman JA. Pharmacogenetics of the vitamin D receptor and osteoporosis. Drug Metab Dispos. 2001;29: 505-512.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/11259341/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos.&volume=29&publication_year=2001&pages=505&pmid=11259341&)

48. Sainz J, Van Tornout JM, Loro ML, et al. Vitamin D-receptor gene polymorphisms and bone density in prepubertal American girls of Mexican descent. N Engl J Med. 1997;337: 77-82.  [DOI](https://doi.org/10.1056/NEJM199707103370202) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9211676/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med.&volume=337&publication_year=1997&pages=77&pmid=9211676&doi=10.1056/NEJM199707103370202&)

49. Phillips KA, Veenstra DL, Oren E, Lee JK, Sadee W. Potential role of pharmacogenomics in reducing adverse drug reactions: a systematic review. JAMA. 2001;286: 2270-2279.  [DOI](https://doi.org/10.1001/jama.286.18.2270) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11710893/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA.&volume=286&publication_year=2001&pages=2270&pmid=11710893&doi=10.1001/jama.286.18.2270&)
