# The interaction between a leflunomide-response methylation site (cg17330251) and variant (rs705379) on response to leflunomide in patients with rheumatoid arthritis

## Metadata
**Authors:** Feng Zhao, Yulan Chen, Haina Liu, Lei Jin, Xin Feng, Bingbing Dai, Meng Chen, Qiao Wang, Yuxin Yao, Ruobing Liao, Junyi Zhao, Bingjia Qu, Ying Song, Lingyu Fu
**Journal:** Frontiers in Pharmacology
**Date:** 2025 Mar 20
**DOI:** [10.3389/fphar.2025.1499723](https://doi.org/10.3389/fphar.2025.1499723)
**PMID:** 40183079
**PMCID:** PMC11965123
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11965123/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC11965123/pdf/fphar-16-1499723.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC11965123/pdf/fphar-16-1499723.pdf)

## Abstract

**Objectives:** 
This research aims to reveal the mechanisms of the effect of the Paraoxonase 1 (PON1) gene on response to leflunomide (LEF) in rheumatoid arthritis (RA) patients, in terms of single nucleotide polymorphism (SNP), DNA methylation levels.

**Methods:** 
A total of 240 RA patients enrolled were categorized into the good response group and the non-response group according to the difference in DAS28 scores between baseline and 6 months after LEF administration. The identified LEF-response cytosine-phosphate-guanines (CpGs) island (cg17330251) and its internal SNPs (rs705379, etc.) located at the PON1 promoter were detected by Sanger sequencing and methyl target sequencing.

**Results:** 
A total of 12 CpG sites at cg17330251 could be identified in our RA patients. There were significant difference between the responders and non-responders in nine CpG sites: cg17330251_2, cg17330251_3, cg17330251_4, cg17330251_6, cg17330251_7, cg17330251_8, cg17330251_9, cg17330251_10, cg17330251_12, [OR (95CI%) = 0.492 (0.250, 0.969), 0.478 (0.243, 0.940), 0.492 (0.250, 0.969), 0.461 (0.234, 0.907), 0.492 (0.250, 0.969), 0.437 (0.225, 0.849), 0.478 (0.243, 0.941), 0.421 (0.212, 0.836), 0.424 (0.213, 0.843), P < 0.05, respectively]. At all these nine CpG sites, the proportions of low methylation levels in the responders were higher than those in the non-responders (P < 0.05). In a dominant model, there was a significant difference in rs705379 wildtype CC and mutant genotypes (CT + TT) between the responders and non-responders (P < 0.05). The average methylation level of 12 CpG sites was lowest in rs705379-CC (median 0.229, IQR 0.195–0.287), then rs705379-CT (median 0.363, IQR 0.332–0.395), and rs705379-TT (median:0.531, IQR:0.496–0.557). The average methylation levels of 12 CpG sites were significantly negative correlated with ΔDAS28 (r = −0.13, P < 0.05). The Logistic regression indicated that combined effect of rs705379, DNA methylation of the PON1 gene [OR (95CI%) = 1.277 [1.003, 1.626)], systemic inflammation index (SIRI) [OR (95CI%) = 1.079 (1.018, 1.143)] served as protective factors on response to LEF in RA patients.

**Conclusion:** 
The RA patients with SNP-rs705379-CC, the low methylation level of PON1-cg17330251 and more SIRI would be susceptible of response to LEF and more suitable to choose LEF treatment.

Keywords: rheumatoid arthritis, leflunomide, PON1, DNA methylation4, single nucleotide polymorphism

### Objectives

This research aims to reveal the mechanisms of the effect of the Paraoxonase 1 (*PON1*) gene on response to leflunomide (LEF) in rheumatoid arthritis (RA) patients, in terms of single nucleotide polymorphism (SNP), DNA methylation levels.

### Methods

A total of 240 RA patients enrolled were categorized into the good response group and the non-response group according to the difference in DAS28 scores between baseline and 6 months after LEF administration. The identified LEF-response cytosine-phosphate-guanines (CpGs) island (cg17330251) and its internal SNPs (rs705379, etc.) located at the *PON1* promoter were detected by Sanger sequencing and methyl target sequencing.

### Results

A total of 12 CpG sites at cg17330251 could be identified in our RA patients. There were significant difference between the responders and non-responders in nine CpG sites: cg17330251_2, cg17330251_3, cg17330251_4, cg17330251_6, cg17330251_7, cg17330251_8, cg17330251_9, cg17330251_10, cg17330251_12, [OR (95CI%) = 0.492 (0.250, 0.969), 0.478 (0.243, 0.940), 0.492 (0.250, 0.969), 0.461 (0.234, 0.907), 0.492 (0.250, 0.969), 0.437 (0.225, 0.849), 0.478 (0.243, 0.941), 0.421 (0.212, 0.836), 0.424 (0.213, 0.843), *P* < 0.05, respectively]. At all these nine CpG sites, the proportions of low methylation levels in the responders were higher than those in the non-responders (*P* < 0.05). In a dominant model, there was a significant difference in rs705379 wildtype CC and mutant genotypes (CT + TT) between the responders and non-responders (*P* < 0.05). The average methylation level of 12 CpG sites was lowest in rs705379-CC (median 0.229, IQR 0.195–0.287), then rs705379-CT (median 0.363, IQR 0.332–0.395), and rs705379-TT (median:0.531, IQR:0.496–0.557). The average methylation levels of 12 CpG sites were significantly negative correlated with ΔDAS28 (*r* = −0.13, *P* < 0.05). The Logistic regression indicated that combined effect of rs705379, DNA methylation of the *PON1* gene [OR (95CI%) = 1.277 [1.003, 1.626)], systemic inflammation index (SIRI) [OR (95CI%) = 1.079 (1.018, 1.143)] served as protective factors on response to LEF in RA patients.

### Conclusion

The RA patients with SNP-rs705379-CC, the low methylation level of *PON1*-cg17330251 and more SIRI would be susceptible of response to LEF and more suitable to choose LEF treatment.

## 1 Introduction

Rheumatoid arthritis (RA) is a persistent inflammatory autoimmune disease influencing joint synovial tissue, and the tendon sheath; it produces severe joint deformities as well as disability. RA patients also exhibit extra-articular manifestations and comorbidities, such as cardiovascular disease, respiratory disease, and so on ([Figus et al., 2021](#B18)). Approximately 1% of the world population has been diagnosed with RA, and this affliction is more prevalent among women than men ([Smolen et al., 2016](#B45)). It is currently recognized that RA patients have about a 50% increased risk of premature mortality in comparison with the general population, leading to an expected reduction in life expectancy of 3–10 years ([Myasoedova et al., 2010](#B33)). Sufferers of this disease often experience low quality of life and reduced life expectancy, with a simultaneous increase in usage of the healthcare system. They have a higher likelihood of experiencing unemployment, and pose a huge financial burden on individuals and society ([Eriksson et al., 2015](#B17); [Roodenrijs et al., 2021](#B40)).

Currently, RA cannot be completely cured and research indicates that RA patients who receive timeous and effective treatment, leading to significant symptom relief, experience a reduced risk of disability and premature death ([Scirè et al., 2014](#B43); [Ohta and Sano, 2024](#B36)). The mainstay treatment for RA involves the use of disease-modifying antirheumatic drugs, with methotrexate (MTX) and leflunomide (LEF) being first-line treatment medications among them ([Abbasi et al., 2019](#B1)) and LEF has similar efficacy to MTX ([Qi et al., 2021](#B38)), Although research indicated that LEF had a significantly lower incidence of adverse reactions in the treatment of RA compared to MTX ([Wang et al., 2021](#B50)), some RA patients might experience adverse reactions such as abdominal pain, diarrhea, nausea, vomiting, itching, and alopecia when taking LEF ([Molina Molina et al., 2015](#B32); [Koller et al., 2019](#B29)). Hence, it is necessary to identify biomarkers predictive of LEF responses and ensure that patients can be administered customized therapeutic regimens that guarantee safety and efficacy.

In our previous research, LEF-related response signatures were identified by a whole-genome DNA methylation profiling using Illumina 850k methylation arrays and a targeted bisulfite sequencing assay. Following the prognostic models developed by machine learning algorithms, a seven-differentially methylated position (DMP)-based prognostic signatu7gere consisting of cg17330251, cg19814518, cg20124410, cg21109666, cg22572476, cg23403192, and cg24432675 were identified and incorporated to predict RA patients’ response to LEF following a 6-month treatment period; these were located at paraoxonase 1(*PON1*), adenosine deaminase RNA-specific B2 (*ADARB2*), ubiquitin-specific peptidase 16(*USP16*), U2AF homology motif kinase 1(*UHMK1*), and disrupted-in-schizophrenia 1 (*DISC1*), respectively ([Chen et al., 2023](#B11)).

Among these genes, *PON1* is an esterase enzyme that participates in maintaining the body’s oxidative balance. It is involved in the metabolism of oxidized lipids and contributes to the removal of free radicals. In 2020, J. Parada-Turska, G et al. found that the PON1 concentration was reduced in the group with high disease activity compared to the group with low disease activity ([Parada-Turska et al., 2020](#B37)), based on the Disease Activity Score of 28 joints (DAS28) and erythrocyte sedimentation rate (ESR) (DAS28-ESR), suggesting a potential association between *PON1* and the prognosis of RA. DNA methylation, a primary regulatory mechanism in epigenetics, is reversible through a process known as demethylation. This process can be influenced by various environmental factors including diet and medication in a time and tissue-dependent manner, so it can be used as a good therapeutic candidate. Some studies found that the activity and concentration of PON1 may be affected by both single nucleotide polymorphisms (SNPs) and DNA methylation ([Zaragoza-García et al., 2021](#B54); [Huen et al., 2015](#B26)). K. Huen et al. showed that DNA methylation level might be negatively correlated with PON1 enzyme activity ([Huen et al., 2015](#B26)). In chlorpyrifos-resistant individuals with abnormal lipid profiles, the DNA methylation level of the *PON1* promoter was found to be associated with reduced expression of *PON1* mRNA ([Su et al., 2019](#B47)). In renal cell carcinoma, it has also been found that high methylation of *PON1* was associated with downregulation of mRNA and protein levels ([Li and Yu, 2019](#B30)). At 2021, research suggested that, the carriers with TT genotype at the rs705379 (−108C>T) site of *PON1* showed lower PON1 levels and enzyme activity in RA patients ([Zaragoza-García et al., 2021](#B54)).

Both SNP and DNA methylation not only serve as important genetic biomarkers but also play important roles in gene regulation. Thus, combining genetic and methylation data could enhance our comprehension of disease causation and prognosis. Otherwise, some studies have shown that changes in SNP sites affect the modification of DNA methylation, thereby playing a role in gene transcriptional regulation ([Izzi et al., 2016](#B27); [Chen et al., 2016](#B10); [Hannon et al., 2018](#B24); [Ardicli et al., 2024](#B4)). The relationship between SNP site specific CpG and the pathogenesis of RA has been confirmed in multiple cell lines ([Frank-Bertoncelj et al., 2017](#B19); [Julià et al., 2017](#B28); [Ai et al., 2018](#B2); [Clark et al., 2020](#B13)). In 2020, the research results of Alexander D et al. ([Clark et al., 2020](#B13)) confirmed that 37% of DNA sites associated with RA are involved in regulating changes in cis CpG methylation. Our previous identified hypomethylated cg17330251 was located at the *PON1* promoter, and interestingly, there were several functional SNPs such as the −108C/T (rs705379) locus located inside cg17330251 island, which may affect the binding of the transcription factor Sp1 (specificity protein 1, Sp1) ([Deakin et al., 2003](#B14)) It is worth noting that, according to the literature, there are currently no studies on the association between the *PON1* gene DNA methylation, SNP rs705379, and the response to LEF in RA patients.

To evaluate the interaction between the genetic variants and the epigenetic aberration (cg17330251) of *PON1* gene on predicting RA patient’s LEF response status. We hypothesized that allele-specific DNA methylation might affect the promoter activity of *PON1*, thereby resulting in gene expression decreasing and subsequently affecting the response of RA patients to LEF. Our study may prove helpful to the individual treatment of the RA patients.

## 2 Materials and methods

### 2.1 Study design and patient population

This study received approval from the medical ethics committee of the First Affiliated Hospital of China Medical University [approval number: (2021)89]. Patients with RA (age >18 years and of Han ethnicity) who fulfilled the 1987 revised criteria of the American College of Rheumatology (ACR) ([Arnett et al., 1988](#B5)) or the 2010 ACR/European League Against Rheumatism (EULAR) ([Aletaha et al., 2010](#B3)) criteria for the classification of RA were recruited from four Class A tertiary hospital hospitals in Liaoning Province (the First Hospital of China Medical University, Shengjing Hospital of China Medical University, the First Affiliated Hospital of Jinzhou Medical University, Dalian Municipal Central Hospital) between June 2018 to June 2020. The exclusion criteria were other autoimmune diseases (e. g. systemic lupus erythematosus, scleroderma, dermatomyositis, etc.); RA patients whose main treatment drug is not LEF but MTX or other DMARDs; pregnant or lactating patients, and patients with malignant tumors who cannot cooperate with follow-up.

All the patients were mainly treated with LEF, and could also receive combination therapy with non-steroidal anti-inflammatory drugs and small doses of corticosteroids. All participants in our study completed 6 months of follow-up. In this study, clinical data on RA patients were obtained at baseline and month six follow-up visits, as previously described, including 28-joint tender and swollen joint counts (TJC28 and SJC28), pain visual analogue scale (pain VAS), erythrocyte sedimentation rate (ESR). According to the above indicators, we were able to calculate Disease Activity Score 28-joint (DAS28) ( DAS28=0.56×√TJC+0.28×√SJC+0.70×ln⁡⁡ESR+0.014×VAS ), which assessed disease activity of RA patients. Response status for LEF treatment was determined based on changes in DAS28 (ΔDAS28 = DAS28 at baseline-DAS28 at 6 months). A good response was defined by a DAS28 improvement of >1.2 with a final DAS28 score ≤3.2, while nonresponse was defined by an improvement in DAS28 ≤ 0.6 or 0.6 ≤ DAS28 ≤ 1.2 with a DAS28 > 5.1 end-point after LEF treatment for 6 months. Patients meeting criteria falling between these categories were categorized as moderate responders ([Atzeni et al., 2009](#B6); [Wells et al., 2009](#B53); [van Riel and Renskers, 2016](#B49)). Patients exhibiting a good or moderate response were consolidated into a category termed “responder”. For the 240 patients that underwent LEF treatment for 6 months, 93 and 147 were non-responders and responders, respectively, as per EULAR criteria.

After providing standardized training to the investigators, on-site surveys were conducted in questionnaire form, covering the following: (1): Patient demographics: age, gender, etc. (2) Behavioral factors: smoking, alcohol consumption, etc. (3) Disease-related factors: number of swollen joints, number of painful joints, VAS scores, etc. Clinical indicator data, including biochemical and immunological indicators, glucose and lipid metabolism indicators, and complete blood count indicators, were collected by retrieving medical records. The data were then grouped according to the criteria established by each hospital center. We computed the neutrophil-to-lymphocyte ratio (NLR), platelet-to-lymphocyte ratio (PLR), systemic immune-inflammation index (SII), and systemic inflammation response index (SIRI) using the following formulas ([Wang et al., 2023](#B51)): NLR = neutrophil count/lymphocyte count; PLR = platelet count/lymphocyte count; SII = (neutrophil count × platelet count)/lymphocyte count; and SIRI = (neutrophil count × monocyte count)/lymphocyte count.

### 2.2 DNA extraction and methylation analysis

Detection of methylation levels in the promoter region of the *PON1* gene using the Targeted Bisulfite Sequencing (MethylTarget™) was conducted by the Genesky Biotechnologies Inc. (Shanghai, China). Each RA patient provided a 5 mL venous blood sample in the morning following an overnight fast. DNA was extracted from peripheral blood using the QIAamp DNA Blood Mini Kit (QIAGEN) within 72 h. Subsequently, bisulfite transformation was performed using the EZ DNA Methylation-Gold Kit (ZYMO, CA, United States). The samples were then subjected to PCR amplification with HotStart Taq polymerase (TaKaRa, Dalian, China), followed by library construction. The final step involved high-throughput sequencing on the Illumina Hiseq platform (Illumina, CA, United States) in 2 × 150 bp paired-end sequencing mode.

### 2.3 SNP selection and genotyping

Inside the cg17330251 island located at *PON1* gene promoter, there were four polymorphisms, rs705379, rs705380, rs705381, and rs553285883 in our population. SNP selection criteria include: (1): minor allele frequency (MAF) ≥10% within Asian data; (2); Hardy–Weinberg equilibrium (HWE) tests were performed on the above four SNPs: the rs705379 and rs553285883 were in HWE (*P* > 0.05), whereas the others deviated from it (*P* < 0.05) ([Supplementary Table 1](#s12)); (3) the transcription factor binding site prediction: on-line tools such as PROMO ([https://alggen.lsi.upc.es/cgi-bin/promo_v3/promo/promoinit.cgi?dirDB=TF_8.3](https://alggen.lsi.upc.es/cgi-bin/promo_v3/promo/promoinit.cgi?dirDB=TF_8.3)) and JASPAR ([https://jaspar.genereg.net/](https://jaspar.genereg.net/)) were used to predict the transcription factors potentially binding. Both PROMO website and JASPAR website suggesting that Pax-5, P53, Sp1, and ETF might be the transcription factors binding to the rs705379 site (shown in [Supplementary Table 2](#s12)). Transcription factor Sp1 might bind to the rs705379 locus with a score value of 16.18, and the predicted binding sequence was GGGGCGGGGG.

Genotyping analysis of SNP was performed using Sanger sequencing and then PCR primer were designed using Primer3 software, and synthesized by the Genesky Biotechnologies Inc. (Shanghai, China). Primer sequences for rs705379 were: F: GGG​TGA​GCG​CAA​TCA​GCT​TC; and R: TGG​ACT​AGG​CAC​CTA​TTC​TCT​GTC​TTC.

### 2.4 PON1 network analysis

To elucidate the role of *PON1*, GeneMANIA ([http://www.genemania.org/](http://www.genemania.org/)) was employed to discover genes exhibiting analogous functions to *PON1* and construct an interactive functional-association network ([Warde-Farley et al., 2010](#B52)). The network for *PON1* was established including co-expression, co-localization, physical interactions, genetic interaction and predictions to analyze function.

### 2.5 Statistical analyses

HWE was assessed using Chi-square tests. All continuous variables in this study exhibited non-normal distributions, so they were presented as medians [interquartile range (IQR)]. Categorical variables were described using frequency and percentages. Subsequent analyses included non-parametric tests and chi-squared tests as appropriate. Significant differences between groups were analyzed using the Kruskal–Wallis test. A correlation analysis was conducted by calculating the Spearman correlation coefficient (Spearman *r*). For variables with less than 10% missing data, multiple imputation methods were applied to handle missing values.

The relationship between methylation, SNP, and other variables with prognosis of RA with LEF was determined by logistic regression adjusting for age and sex. The dependent variable was RA prognosis after LEF-administering (non-responder was assigned as 0). *PON1* methylation levels were divided into three groups (low, medium, high) according to quartiles, the cut-offs were 25th percentile and 75th percentile. Methylation levels at cg17330251 (significant sites) were assigned values of 0 for hypermethylation, 1 for intermediate methylation, and 2 for hypomethylation. The TT, CT, and CC of the *PON1* rs705379 locus were assigned scores of 0, 1, or 2. The assigned values were then summed up to derive the composite genetic protective score (Score), which was further categorized into quartiles for analyzing its association with the prognosis of RA response to LEF. The score was incorporated into the logistic regression analysis to estimate the impact of multiple omics loci in the *PON1* gene on predicting the response of RA patients to LEF.

Statistical analyses were performed using IBM SPSS Statistics for Windows, Version 25.0 (Released 2017; IBM Corp., Armonk, New York, United States) and R software (Version 4.3.0, Vienna, Austria) along with the RStudio interface (Version 2023.03.0, Boston, MA, United States). GraphPad Prism 9.0.0 was used to plot the results. The significance level was set at *P* < 0.05.

## 3 Results

### 3.1 Demographic characteristics of study participants

A total of 240 patients with RA were enrolled in this study who could be divided into the response group (*n* = 147) and the non-response group (*n* = 93). Descriptive analysis was performed on the basic information of the patients, and the results are summarized in [Supplementary Table 3](#s12). Monocyte (MONO) (*P* = 0.020) showed a statistically significant difference between two groups. After univariate Logistic analysis, the plateletcrit (PCT) (OR = 0.608 [0.309, 1.197)], MONO [OR = 1.900 (1.101, 3.278)], platelet count (PLT) [OR = 0.642 (0.328, 1.256)], and systemic inflammation response index (SIRI) [OR = 1.084 (1.022, 1.149)] were potential risk factors for RA prognosis according to the criterion of *P* < 0.2.

### 3.2 The relationship between the DNA methylation level of PON1-cg17330251 and prognosis of RA response to LEF

After detection of methylation levels, there were a total of 12 CpG sites at *PON1-*cg17330251 in our patients. There were significant difference between the responders and non-responders in 9 CpG sites, cg17330251_2, cg17330251_3, cg17330251_4, cg17330251_6, cg17330251_7, cg17330251_8, cg17330251_9, cg17330251_10, cg17330251_12, (OR (95CI%) = 0.492 (0.250, 0.969), 0.478 (0.243, 0.940), 0.492 (0.250, 0.969), 0.461 (0.234, 0.907), 0.492 (0.250, 0.969), 0.437 (0.225, 0.849), 0.478 (0.243, 0.941), 0.421 (0.212, 0.836), and 0.424 (0.213, 0.843), respectively). At all these nine CpG sites, the proportions of low methylation levels in the responders were higher than those in the non-responders (*P* < 0.05). RA patients with low methylation levels would be a significantly better prognosis than those with moderate methylation levels. Specifically, at the cg17330251_6, RA patients with low methylation levels would be also have a better prognosis than those with high methylation levels [OR (95CI%) = 0.478 (0.243, 0.940)] ([Supplementary Table 4](#s12)).

### 3.3 The association between PON1-rs705379 and prognosis of RA response to LEF

In dominant model, there was significant difference in rs705379 wildtype CC and mutant genotypes (CT + TT) between the responders and non-responders (*P* < 0.05). More CC-carriers were found in the response group (34.01%) than in the non-response group (21.50%) [OR (95%CI) = 0.532 (0.291, 0.973)]. In codominant and recessive models, there were no significant differences in the distribution of genotypes at the rs705379 locus of the *PON1* gene in responders and non-responders (*P* > 0.05). The results are shown in [Table 1](#T1).

### TABLE 1.

| Model type | Genotype | Responders (n, %) | Non-responders (n, %) | χ2 | P a | OR (95%CI) | P b |
| --- | --- | --- | --- | --- | --- | --- | --- |
| Codominant | CC | 50 (34.01) | 20 (21.50) | 4.404 | 0.111 | Ref |   |
| CT | 63 (42.86) | 49 (52.69) | 0.513 (0.270,0.973) | 0.041 |  |  |  |
| TT | 34 (23.13) | 24 (25.81) | 0.572 (0.272,1.202) | 0.141 |  |  |  |
| Dominant | CC | 50 (34.01) | 20 (21.50) | 4.314 | 0.038 | Ref |   |
| CT + TT | 97 (65.99) | 73 (78.50) | 0.532 (0.291,0.973) | 0.041 |  |  |  |
| Recessive | CC + CT | 113 (76.87) | 69 (74.19) | 0.223 | 0.637 | Ref |   |
| TT | 34 (23.13) | 24 (25.81) | 0.879 (0.479,1.611) | 0.676 |  |  |  |
| Allele | C | 163 (55.4) | 89 (47.8) | 2.643 | 0.105 | Ref |   |
| T | 131 (44.6) | 97 (52.2) | 0.737 (0.510,1.066) | 0.105 |  |  |  |

Table 1 Caption: Genotype frequencies and risk estimates of PON1 rs705379 in the RA responders and non-responders.

### 3.4 The association among rs705379, average methylation levels of cg17330251 of PON1 and ΔDAS28

There were significant statistical differences observed in ΔDAS28 among different genotypes at the rs705379 ([Figure 1A](#F1)). Notably, there were significant differences in average methylation levels of 12 CpG sites between the genotype CC, genotype CT and genotype TT at the rs705379 (*P* < 0.001, respectively) ([Figure 1B](#F1)). The average methylation level was lowest in rs705379-CC (median 0.229, IQR 0.195–0.287), then rs705379-CT (median 0.363, IQR 0.332–0.395), and rs705379-TT (median:0.531, IQR:0.496–0.557). The results of the correlation analysis indicated that the average methylation levels of 12 CpG sites were significantly negatively correlated with ΔDAS28 (*r* = −0.13, *P* < 0.05), as shown in [Figure 1C](#F1).

### FIGURE 1.

![FIGURE 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0603/11965123/4317ff10304e/fphar-16-1499723-g001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=11965123_fphar-16-1499723-g001.jpg)

Correlations among rs705379, average methylation levels of cg17330251 of PON1 and ΔDAS28. ***P < 0.001. (A) ΔDAS28 in each genotype at rs705379; (B) Average methylation level of cg17330251 in each genotype at rs705379; (C) Correlation between average methylation level of cg17330251 and ΔDAS28.

### 3.5 Multifactor analysis for the prognosis of LEF-taking RA patients

Two models were constructed by the Logistic regression ([Table 2](#T2)). In Model 1, the genetic protective score was analyzed by univariate logistic analysis and there was a significant association with the prognosis of RA after LEF administration. In Model 2, the genetic protective score and four potential risk factors for RA prognosis (PCT, MONO, PLT and SIRI) were analyzed by logistic regression, and the result showed that the genetic protective score and SIRI were significantly associated with the RA prognosis. In both the models, the increased protective score was associated with better response to LEF in RA patients (OR = 1.313, 95%CI = 1.035–1.665, *P* = 0.025; OR = 1.277, 95%CI = 1.003–1.626, *P* = 0.048). More SIRI was also related to better response to LEF in RA patients (OR = 1.079, 95%CI = 1.018–1.143, *P* = 0.011). In Model 2, there was no issue of collinearity among the variables (*P* > 0.05, [Supplementary Table 5](#s12)).

### TABLE 2.

| Model | Indicators | β | S.E. | Wald | df | OR (95%CI) | P |
| --- | --- | --- | --- | --- | --- | --- | --- |
| Model 1 |  |  |  |  |  |  |  |
|   | Genetic score | 0.272 | 0.121 | 5.046 | 1 | 1.313 (1.035,1.665) | 0.025 |
| Constant | −0.184 | 0.311 | 0.351 | 1 | 0.554 | 0.832 |  |
| Model 2 |  |  |  |  |  |  |  |
|   | Genetic score | 0.244 | 0.123 | 3.927 | 1 | 1.277 (1.003,1.626) | 0.048 |
| SIRI | 0.076 | 0.030 | 6.501 | 1 | 1.079 (1.018,1.143) | 0.011 |  |
| Constant | −0.380 | 0.323 | 1.378 | 1 | 0.684 | 0.240 |  |

Table 2 Caption: The results of multifactor Logistic regression analysis combining patient genetic factors with traditional prognostic factors.

### 3.6 PON1 network analysis

A gene–gene interaction network for *PON1* was constructed, and its function was analyzed using the GeneMANIA database ([Figure 2](#F2)). Functional analysis indicated that 20 proteins were correlated with PON1, including PON3, PON2, NFIC, CLU, and APOA1. These proteins showed the greatest correlation in terms of the icosanoid metabolic process (False discovery rate = 1.986 × 10^−5^), complement activation, lipoprotein particle, plasma lipoprotein particle, protein-lipid complex (False Discovery Rate = 3.937 × 10^−5^). Additionally, these proteins were correlated in terms of fatty acid derivative metabolic process and fatty acid metabolic process. Importantly, *PON1* gene exhibits co-expression and co-localization relationships with members of the cytochrome P450 family, such as *CYP2C8* and *CYP2E1*, as well as *GPLD1* gene, which were associated with the drug metabolic processes and drug responses ([Supplementary Table 6](#s12)).

### FIGURE 2.

![FIGURE 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0603/11965123/d2a834f80f1c/fphar-16-1499723-g002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=11965123_fphar-16-1499723-g002.jpg)

Biological interaction network of PON1. Different colors represent diverse bioinformatics methods.

## 4 Discussion

In this study, the interactions between genetics, DNA methylation at *PON1* susceptibility cg17330251 and functional SNP rs705379 were analyzed. We initially discovered that RA patients with low methylation levels would be more responsive to LEF than those with medium methylation levels. After that, RA patients with the *PON1*-rs705379-CC genotype would be susceptible to response to LEF and better suited to adoption of LEF treatment. We then found that a reduction of methylation levels led to a significant increase in ΔDAS28. More importantly, in the carriers of rs705379-CC, the methylation levels of the cg17330251 significantly decreased, and then ΔDAS28 would significantly increase. All these suggested that functional rs705379 site can alter CpG site modification and thus influence epigenetic regulation as associated with prognostic response to LEF in RA patients. Our results may represent a functional link between genetic variation and an epigenetic modification for *PON1* expression, and further effects on patient susceptibility to LEF. To the best of our knowledge, this is the first study to explore the mechanism of the *PON1* gene on RA patients’ response to LEF based on the integration of genetic and epigenetic factors.

Our previous research revealed that *PON1*, *ADARB2*, *USP16*, *UHMK1*, and *DISC1* might serve as biomarkers for predicting the response to LEF in RA patients ([Chen et al., 2023](#B11)). Through Gene Ontology (GO) bioinformatics enrichment analysis, we found that the *PON1* gene was primarily involved in the ester oxygenase pathway ([Figure 3](#F3)). Its expressed product, PON1, is an esterase that metabolizes oxidized lipids and organophosphates. It participates in eliminating free radicals to maintain oxidative balance, protecting both high-density lipoprotein (HDL) and low-density lipoprotein (LDL) from oxidation, and playing a crucial anti-inflammatory role ([Aviram et al., 1998](#B7)). Research on the *PON1* gene has mainly focused on cardiovascular diseases, with relatively limited studies in RA at present. A study from 2020 proposed that the overexpression of the human *PON1* transgene was linked to a decrease in inflammatory arthritis. This effect may be closely related to increased circulating PON1 activity, the upregulation of the hepatic glutathione pathway, and a decrease in circulating biologically active mediators. These findings suggested that targeting *PON1* can potentially be a therapeutic strategy for joint diseases, including RA ([Charles-Schoeman et al., 2020](#B9)). The findings of this study align with the aforementioned perspectives.

### FIGURE 3.

![FIGURE 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0603/11965123/bf5eaf806663/fphar-16-1499723-g003.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=11965123_fphar-16-1499723-g003.jpg)

GO enrichment analysis of PON1 gene.

The current study implied that the hypermethylation pattern within the promoter region of the *PON1* gene might be a contributing factor to the insufficient response to LEF in RA. Methylation of CpG islands situated in the gene’s promoter region is a consistently acknowledged mechanism for expression silencing ([Vanderkraats et al., 2013](#B48)). The inadequate response to LEF in RA might be attributed to elevated methylation levels influencing aromatase activity (a metric indicative of the amount of PON1 protein in the bloodstream) ([Huen et al., 2015](#B26); [Diels et al., 2021](#B16); [de la Iglesia et al., 2014](#B15)). SNP can affect the methylation of nearby CpG sites, leading to differential methylation levels between different genotypes, a phenomenon known as allele-specific methylation (ASM) ([Schalkwyk et al., 2010](#B42); [Shoemaker et al., 2010](#B44)). The methylation levels of various CpG sites within the cg17330251 methylation island differed significantly among genotypes at the rs705379, providing robust evidence for the occurrence of ASM, which is consistent with the findings reported by [Huen et al. (2015)](#B26). More particularly, [Diels et al. (2021)](#B16) found that TT-genotype carriers of the regulatory variant rs705379 had higher DNA-methylation values at the *PON1* promoter region and such a relationship was found in our study. 22% of ASM genes also exhibited allele-specific gene expression, and this expression was not tissue-specific ([Gertz et al., 2011](#B20)). N. Gupta et al. demonstrated that individuals with genotypes CT and TT exhibited a lower PON1 activity compared to those with genotype CC ([Gupta et al., 2011](#B23)). Although we had performed mRNA expression analysis of the PON1 gene in our blood samples, unfortunately, due to the low expression of the *PON1* gene in these samples, the mRNA expression was only detected in a few samples and could not be compared between the responders and non-responders. M. J. Bonder et al. found a negative correlation between DNA methylation levels in the *PON1* promoter region and its mRNA expression levels in liver samples ([Bonder et al., 2014](#B8)). When verifying the mRNA expression level of this gene, liver samples need to be collected, which is very difficult in RA patients.

Recently, a number of studies proved that genetic and epigenetic factors play a crucial role as prognostic biomarkers in RA ([Sánchez-Maldonado et al., 2021](#B41); [Manuel Sánchez-Maldonado et al., 2020](#B31); [Gravand et al., 2023](#B22); [Ravaei et al., 2022](#B39)). DNA methylation, as the most studied epigenetic modification, is thought to play an important role in RA disease pathogenesis and in mediating the relationship between genetic variants and patient outcomes ([Nair et al., 2017](#B35); [Srivastava and Rasool, 2024](#B46); [Nair et al., 2020](#B34)). The regulatory mechanisms of gene expression are intricate and remain not completely elucidated. we hypothesized that the presence of the C nucleotide at the upstream C/T polymorphic site could lead to low methylation of the adjacent C. Its mechanism may be that rs705379-C had a stronger binding affinity with the transcription factor Sp1, which initiated transcription, forming a DNA-RNA hybrid. This can prevent DNA methyltransferase from binding to the promoter region, resulting in hypomethylation ([Ginno et al., 2012](#B21); [Höller et al., 1988](#B25); [Chen and Zhang, 2020](#B12)). Delightfully, Deakin and others have already verified, through a dual-luciferase assay, rs705379 was located in the binding sequence of the transcription factor Sp1 and the C allele exhibited a stronger binding ability to the transcription factor Sp1 than the T allele ([Deakin et al., 2003](#B14)). The results of the data analysis in this study could confirm this mechanism, and we did not consider it necessary to perform dual luciferase experiments to verify it again.

A comprehensive analysis of genetic factors and traditional clinical prognostic indicators prompted that higher genetic protective score (SNP and DNA methylation) and SIRI were related to the better response to LEF in RA patients. This indicates that there is significant potential for clinical application in assisting physicians with the development of treatment plans for patients with RA. By utilizing these insights, healthcare providers can create more tailored and effective strategies to manage the disease. Ultimately, this could enhance patient outcomes and improve the overall management of RA. Considering that the study population was limited to four hospitals in Liaoning Province, China, the generalizability of the results obtained in this study may be limited. Further validation in a larger and more diverse population is necessary to enhance the robustness and applicability of the findings. Besides, in this study, multiple testing corrections for CpG site analysis (FDR or Bonferroni adjustment) was not applied, due to the subtlety of the CpG site effects and the aim is to broadly screen for methylation sites that may be associated with prognosis in the initial exploratory stage. This could introduce false-positive outcomes. Therefore, we intend to validate the results in an expanded population sample.

## 5 Conclusion

In conclusion, to the best of our knowledge, this study represents the first investigation into the prognostic implications of the rs705379 of *PON1*, DNA methylation levels in patients with RA undergoing LEF treatment. The conserved genotypes of polymorphisms in *PON1*, hypomethylation of the promoter region collectively exert a synergistic impact on the prognosis of RA. The present study offers a novel perspective on the role of *PON1* in the prognosis of RA.

## Acknowledgments

We express our gratitude to all the participants in this research.

## Funding Statement

The author(s) declare that financial support was received for the research, authorship, and/or publication of this article. This work was supported by the Program of the National Natural Science Foundation of China (82173604).

## Data availability statement

The raw data supporting the conclusions of this article will be made available by the authors, without undue reservation.

## Ethics statement

The studies involving humans were approved by the medical ethics committee of the First Affiliated Hospital of China Medical University. The studies were conducted in accordance with the local legislation and institutional requirements. The participants provided their written informed consent to participate in this study. Written informed consent was obtained from the individual(s) for the publication of any potentially identifiable images or data included in this article.

## Author contributions

FZ: Writing–original draft. YC: Writing–original draft. HL: Writing–review and editing. LJ: Writing–review and editing. XF: Writing–review and editing. BD: Writing–review and editing. MC: Writing–original draft. QW: Writing–original draft. YY: Writing–original draft. RL: Writing–original draft. JZ: Writing–original draft. BQ: Writing–original draft. YS: Writing–original draft. LF: Writing–review and editing.

## Conflict of interest

The authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.

## Publisher’s note

All claims expressed in this article are solely those of the authors and do not necessarily represent those of their affiliated organizations, or those of the publisher, the editors and the reviewers. Any product that may be evaluated in this article, or claim that may be made by its manufacturer, is not guaranteed or endorsed by the publisher.

## Supplementary material

The Supplementary Material for this article can be found online at: [https://www.frontiersin.org/articles/10.3389/fphar.2025.1499723/full#supplementary-material](https://www.frontiersin.org/articles/10.3389/fphar.2025.1499723/full#supplementary-material)

## Glossary

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

The raw data supporting the conclusions of this article will be made available by the authors, without undue reservation.

### Supplementary Materials

### Data Availability Statement

The raw data supporting the conclusions of this article will be made available by the authors, without undue reservation.

## References

1. Abbasi M., Mousavi M. J., Jamalzehi S., Alimohammadi R., Bezvan M. H., Mohammadi H., et al. (2019). Strategies toward rheumatoid arthritis therapy; the old and the new. J. Cell Physiol. 234 (7), 10018–10031. 10.1002/jcp.27860  [DOI](https://doi.org/10.1002/jcp.27860) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30536757/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Cell%20Physiol.&title=Strategies%20toward%20rheumatoid%20arthritis%20therapy;%20the%20old%20and%20the%20new&author=M.%20Abbasi&author=M.%20J.%20Mousavi&author=S.%20Jamalzehi&author=R.%20Alimohammadi&author=M.%20H.%20Bezvan&volume=234&issue=7&publication_year=2019&pages=10018-10031&pmid=30536757&doi=10.1002/jcp.27860&)

2. Ai R., Laragione T., Hammaker D., Boyle D. L., Wildberg A., Maeshima K., et al. (2018). Comprehensive epigenetic landscape of rheumatoid arthritis fibroblast-like synoviocytes. Nat. Commun. 9 (1), 1921. 10.1038/s41467-018-04310-9  [DOI](https://doi.org/10.1038/s41467-018-04310-9) | [PMC free article](/articles/PMC5953939/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29765031/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&title=Comprehensive%20epigenetic%20landscape%20of%20rheumatoid%20arthritis%20fibroblast-like%20synoviocytes&author=R.%20Ai&author=T.%20Laragione&author=D.%20Hammaker&author=D.%20L.%20Boyle&author=A.%20Wildberg&volume=9&issue=1&publication_year=2018&pages=1921&pmid=29765031&doi=10.1038/s41467-018-04310-9&)

3. Aletaha D., Neogi T., Silman A. J., Funovits J., Felson D. T., Bingham C. O., 3rd, et al. (2010). 2010 Rheumatoid arthritis classification criteria: an American College of Rheumatology/European League against Rheumatism collaborative initiative. Arthritis Rheum. 62(9):2569–2581. 10.1002/art.27584  [DOI](https://doi.org/10.1002/art.27584) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20872595/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arthritis%20Rheum.&title=2010%20Rheumatoid%20arthritis%20classification%20criteria:%20an%20American%20College%20of%20Rheumatology/European%20League%20against%20Rheumatism%20collaborative%20initiative&author=D.%20Aletaha&author=T.%20Neogi&author=A.%20J.%20Silman&author=J.%20Funovits&author=D.%20T.%20Felson&volume=62&issue=9&publication_year=2010&pages=2569-2581&pmid=20872595&doi=10.1002/art.27584&)

4. Ardicli S., Yigitgor P., Babayev H., Ozen D., Bozkurt B., Senturk N., et al. (2024). The markers of the predictive DNA test for canine hip dysplasia may have a stronger relationship with elbow dysplasia. Heliyon 10 (18), e37716. 10.1016/j.heliyon.2024.e37716  [DOI](https://doi.org/10.1016/j.heliyon.2024.e37716) | [PMC free article](/articles/PMC11417230/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/39315210/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Heliyon&title=The%20markers%20of%20the%20predictive%20DNA%20test%20for%20canine%20hip%20dysplasia%20may%20have%20a%20stronger%20relationship%20with%20elbow%20dysplasia&author=S.%20Ardicli&author=P.%20Yigitgor&author=H.%20Babayev&author=D.%20Ozen&author=B.%20Bozkurt&volume=10&issue=18&publication_year=2024&pages=e37716&pmid=39315210&doi=10.1016/j.heliyon.2024.e37716&)

5. Arnett F. C., Edworthy S. M., Bloch D. A., McShane D. J., Fries J. F., Cooper N. S., et al. (1988). The American Rheumatism Association 1987 revised criteria for the classification of rheumatoid arthritis. Arthritis Rheum. 31 (3), 315–324. 10.1002/art.1780310302  [DOI](https://doi.org/10.1002/art.1780310302) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/3358796/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arthritis%20Rheum.&title=The%20American%20Rheumatism%20Association%201987%20revised%20criteria%20for%20the%20classification%20of%20rheumatoid%20arthritis&author=F.%20C.%20Arnett&author=S.%20M.%20Edworthy&author=D.%20A.%20Bloch&author=D.%20J.%20McShane&author=J.%20F.%20Fries&volume=31&issue=3&publication_year=1988&pages=315-324&pmid=3358796&doi=10.1002/art.1780310302&)

6. Atzeni F., Antivalle M., Pallavicini F. B., Caporali R., Bazzani C., Gorla R., et al. (2009). Predicting response to anti-TNF treatment in rheumatoid arthritis patients. Autoimmun. Rev. 8 (5), 431–437. 10.1016/j.autrev.2009.01.005  [DOI](https://doi.org/10.1016/j.autrev.2009.01.005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19211043/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Autoimmun.%20Rev.&title=Predicting%20response%20to%20anti-TNF%20treatment%20in%20rheumatoid%20arthritis%20patients&author=F.%20Atzeni&author=M.%20Antivalle&author=F.%20B.%20Pallavicini&author=R.%20Caporali&author=C.%20Bazzani&volume=8&issue=5&publication_year=2009&pages=431-437&pmid=19211043&doi=10.1016/j.autrev.2009.01.005&)

7. Aviram M., Rosenblat M., Bisgaier C. L., Newton R. S., Primo-Parmo S. L., La Du B. N. (1998). Paraoxonase inhibits high-density lipoprotein oxidation and preserves its functions. A possible peroxidative role for paraoxonase. J. Clin. Invest 101 (8), 1581–1590. 10.1172/JCI1649  [DOI](https://doi.org/10.1172/JCI1649) | [PMC free article](/articles/PMC508738/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9541487/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Invest&title=Paraoxonase%20inhibits%20high-density%20lipoprotein%20oxidation%20and%20preserves%20its%20functions.%20A%20possible%20peroxidative%20role%20for%20paraoxonase&author=M.%20Aviram&author=M.%20Rosenblat&author=C.%20L.%20Bisgaier&author=R.%20S.%20Newton&author=S.%20L.%20Primo-Parmo&volume=101&issue=8&publication_year=1998&pages=1581-1590&pmid=9541487&doi=10.1172/JCI1649&)

8. Bonder M. J., Kasela S., Kals M., Tamm R., Lokk K., Barragan I., et al. (2014). Genetic and epigenetic regulation of gene expression in fetal and adult human livers. BMC Genomics 15 (1), 860. 10.1186/1471-2164-15-860  [DOI](https://doi.org/10.1186/1471-2164-15-860) | [PMC free article](/articles/PMC4287518/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25282492/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Genomics&title=Genetic%20and%20epigenetic%20regulation%20of%20gene%20expression%20in%20fetal%20and%20adult%20human%20livers&author=M.%20J.%20Bonder&author=S.%20Kasela&author=M.%20Kals&author=R.%20Tamm&author=K.%20Lokk&volume=15&issue=1&publication_year=2014&pages=860&pmid=25282492&doi=10.1186/1471-2164-15-860&)

9. Charles-Schoeman C., Wang J., Shahbazian A., Lee Y. Y., Wang X., Grijalva V., et al. (2020). Suppression of inflammatory arthritis in human serum paraoxonase 1 transgenic mice. Sci. Rep. 10 (1), 16848. 10.1038/s41598-020-74016-w  [DOI](https://doi.org/10.1038/s41598-020-74016-w) | [PMC free article](/articles/PMC7546628/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33033318/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&title=Suppression%20of%20inflammatory%20arthritis%20in%20human%20serum%20paraoxonase%201%20transgenic%20mice&author=C.%20Charles-Schoeman&author=J.%20Wang&author=A.%20Shahbazian&author=Y.%20Y.%20Lee&author=X.%20Wang&volume=10&issue=1&publication_year=2020&pages=16848&pmid=33033318&doi=10.1038/s41598-020-74016-w&)

10. Chen L., Ge B., Casale F. P., Vasquez L., Kwan T., Garrido-Martín D., et al. (2016). Genetic drivers of epigenetic and transcriptional variation in human immune cells. Cell 167 (5), 1398–1414. 10.1016/j.cell.2016.10.026  [DOI](https://doi.org/10.1016/j.cell.2016.10.026) | [PMC free article](/articles/PMC5119954/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27863251/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell&title=Genetic%20drivers%20of%20epigenetic%20and%20transcriptional%20variation%20in%20human%20immune%20cells&author=L.%20Chen&author=B.%20Ge&author=F.%20P.%20Casale&author=L.%20Vasquez&author=T.%20Kwan&volume=167&issue=5&publication_year=2016&pages=1398-1414&pmid=27863251&doi=10.1016/j.cell.2016.10.026&)

11. Chen Y., Wang Q., Liu H., Jin L., Feng X., Dai B., et al. (2023). The prognostic value of whole-genome DNA methylation in response to Leflunomide in patients with Rheumatoid Arthritis. Front. Immunol. 14, 1173187. 10.3389/fimmu.2023.1173187  [DOI](https://doi.org/10.3389/fimmu.2023.1173187) | [PMC free article](/articles/PMC10513488/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/37744384/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front.%20Immunol.&title=The%20prognostic%20value%20of%20whole-genome%20DNA%20methylation%20in%20response%20to%20Leflunomide%20in%20patients%20with%20Rheumatoid%20Arthritis&author=Y.%20Chen&author=Q.%20Wang&author=H.%20Liu&author=L.%20Jin&author=X.%20Feng&volume=14&publication_year=2023&pages=1173187&pmid=37744384&doi=10.3389/fimmu.2023.1173187&)

12. Chen Z., Zhang Y. (2020). Role of mammalian DNA methyltransferases in development. Annu. Rev. Biochem. 89, 135–158. 10.1146/annurev-biochem-103019-102815  [DOI](https://doi.org/10.1146/annurev-biochem-103019-102815) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31815535/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annu.%20Rev.%20Biochem.&title=Role%20of%20mammalian%20DNA%20methyltransferases%20in%20development&author=Z.%20Chen&author=Y.%20Zhang&volume=89&publication_year=2020&pages=135-158&pmid=31815535&doi=10.1146/annurev-biochem-103019-102815&)

13. Clark A. D., Nair N., Anderson A. E., Thalayasingam N., Naamane N., Skelton A. J., et al. (2020). Lymphocyte DNA methylation mediates genetic risk at shared immune-mediated disease loci. J. Allergy Clin. Immunol. 145 (5), 1438–1451. 10.1016/j.jaci.2019.12.910  [DOI](https://doi.org/10.1016/j.jaci.2019.12.910) | [PMC free article](/articles/PMC7201180/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31945409/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Allergy%20Clin.%20Immunol.&title=Lymphocyte%20DNA%20methylation%20mediates%20genetic%20risk%20at%20shared%20immune-mediated%20disease%20loci&author=A.%20D.%20Clark&author=N.%20Nair&author=A.%20E.%20Anderson&author=N.%20Thalayasingam&author=N.%20Naamane&volume=145&issue=5&publication_year=2020&pages=1438-1451&pmid=31945409&doi=10.1016/j.jaci.2019.12.910&)

14. Deakin S., Leviev I., Brulhart-Meynet M. C., James R. W. (2003). Paraoxonase-1 promoter haplotypes and serum paraoxonase: a predominant role for polymorphic position - 107, implicating the Sp1 transcription factor. Biochem. J. 372 (Pt 2), 643–649. 10.1042/BJ20021670  [DOI](https://doi.org/10.1042/BJ20021670) | [PMC free article](/articles/PMC1223427/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12639220/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem.%20J.&title=Paraoxonase-1%20promoter%20haplotypes%20and%20serum%20paraoxonase:%20a%20predominant%20role%20for%20polymorphic%20position%20-%20107,%20implicating%20the%20Sp1%20transcription%20factor&author=S.%20Deakin&author=I.%20Leviev&author=M.%20C.%20Brulhart-Meynet&author=R.%20W.%20James&volume=372&issue=Pt%202&publication_year=2003&pages=643-649&pmid=12639220&doi=10.1042/BJ20021670&)

15. de la Iglesia R., Mansego M. L., Sánchez-Muniz F. J., Zulet M. A., Martinez J. A. (2014). Arylesterase activity is associated with antioxidant intake and paraoxonase-1 (PON1) gene methylation in metabolic syndrome patients following an energy restricted diet. Excli J. 13, 416–426.  [PMC free article](/articles/PMC4464483/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26417268/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Excli%20J.&title=Arylesterase%20activity%20is%20associated%20with%20antioxidant%20intake%20and%20paraoxonase-1%20(PON1)%20gene%20methylation%20in%20metabolic%20syndrome%20patients%20following%20an%20energy%20restricted%20diet&author=R.%20de%20la%20Iglesia&author=M.%20L.%20Mansego&author=F.%20J.%20S%C3%A1nchez-Muniz&author=M.%20A.%20Zulet&author=J.%20A.%20Martinez&volume=13&publication_year=2014&pages=416-426&pmid=26417268&)

16. Diels S., Cuypers B., Tvarijonaviciute A., Derudas B., Van Dijck E., Verrijken A., et al. (2021). A targeted multi-omics approach reveals paraoxonase-1 as a determinant of obesity-associated fatty liver disease. Clin. Epigenetics 13 (1), 158. 10.1186/s13148-021-01142-1  [DOI](https://doi.org/10.1186/s13148-021-01142-1) | [PMC free article](/articles/PMC8360816/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34389043/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Epigenetics&title=A%20targeted%20multi-omics%20approach%20reveals%20paraoxonase-1%20as%20a%20determinant%20of%20obesity-associated%20fatty%20liver%20disease&author=S.%20Diels&author=B.%20Cuypers&author=A.%20Tvarijonaviciute&author=B.%20Derudas&author=E.%20Van%20Dijck&volume=13&issue=1&publication_year=2021&pages=158&pmid=34389043&doi=10.1186/s13148-021-01142-1&)

17. Eriksson J. K., Johansson K., Askling J., Neovius M. (2015). Costs for hospital care, drugs and lost work days in incident and prevalent rheumatoid arthritis: how large, and how are they distributed? Ann. Rheum. Dis. 74 (4), 648–654. 10.1136/annrheumdis-2013-204080  [DOI](https://doi.org/10.1136/annrheumdis-2013-204080) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24323395/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann.%20Rheum.%20Dis.&title=Costs%20for%20hospital%20care,%20drugs%20and%20lost%20work%20days%20in%20incident%20and%20prevalent%20rheumatoid%20arthritis:%20how%20large,%20and%20how%20are%20they%20distributed?&author=J.%20K.%20Eriksson&author=K.%20Johansson&author=J.%20Askling&author=M.%20Neovius&volume=74&issue=4&publication_year=2015&pages=648-654&pmid=24323395&doi=10.1136/annrheumdis-2013-204080&)

18. Figus F. A., Piga M., Azzolin I., McConnell R., Iagnocco A. (2021). Rheumatoid arthritis: extra-articular manifestations and comorbidities. Autoimmun. Rev. 20 (4), 102776. 10.1016/j.autrev.2021.102776  [DOI](https://doi.org/10.1016/j.autrev.2021.102776) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33609792/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Autoimmun.%20Rev.&title=Rheumatoid%20arthritis:%20extra-articular%20manifestations%20and%20comorbidities&author=F.%20A.%20Figus&author=M.%20Piga&author=I.%20Azzolin&author=R.%20McConnell&author=A.%20Iagnocco&volume=20&issue=4&publication_year=2021&pages=102776&pmid=33609792&doi=10.1016/j.autrev.2021.102776&)

19. Frank-Bertoncelj M., Trenkmann M., Klein K., Karouzakis E., Rehrauer H., Bratus A., et al. (2017). Epigenetically-driven anatomical diversity of synovial fibroblasts guides joint-specific fibroblast functions. Nat. Commun. 8, 14852. 10.1038/ncomms14852  [DOI](https://doi.org/10.1038/ncomms14852) | [PMC free article](/articles/PMC5376654/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28332497/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&title=Epigenetically-driven%20anatomical%20diversity%20of%20synovial%20fibroblasts%20guides%20joint-specific%20fibroblast%20functions&author=M.%20Frank-Bertoncelj&author=M.%20Trenkmann&author=K.%20Klein&author=E.%20Karouzakis&author=H.%20Rehrauer&volume=8&publication_year=2017&pages=14852&pmid=28332497&doi=10.1038/ncomms14852&)

20. Gertz J., Varley K. E., Reddy T. E., Bowling K. M., Pauli F., Parker S. L., et al. (2011). Analysis of DNA methylation in a three-generation family reveals widespread genetic influence on epigenetic regulation. PLoS Genet. 7 (8), e1002228. 10.1371/journal.pgen.1002228  [DOI](https://doi.org/10.1371/journal.pgen.1002228) | [PMC free article](/articles/PMC3154961/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21852959/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet.&title=Analysis%20of%20DNA%20methylation%20in%20a%20three-generation%20family%20reveals%20widespread%20genetic%20influence%20on%20epigenetic%20regulation&author=J.%20Gertz&author=K.%20E.%20Varley&author=T.%20E.%20Reddy&author=K.%20M.%20Bowling&author=F.%20Pauli&volume=7&issue=8&publication_year=2011&pages=e1002228&pmid=21852959&doi=10.1371/journal.pgen.1002228&)

21. Ginno P. A., Lott P. L., Christensen H. C., Korf I., Chédin F. (2012). R-loop formation is a distinctive characteristic of unmethylated human CpG island promoters. Mol. Cell 45 (6), 814–825. 10.1016/j.molcel.2012.01.017  [DOI](https://doi.org/10.1016/j.molcel.2012.01.017) | [PMC free article](/articles/PMC3319272/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22387027/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Cell&title=R-loop%20formation%20is%20a%20distinctive%20characteristic%20of%20unmethylated%20human%20CpG%20island%20promoters&author=P.%20A.%20Ginno&author=P.%20L.%20Lott&author=H.%20C.%20Christensen&author=I.%20Korf&author=F.%20Ch%C3%A9din&volume=45&issue=6&publication_year=2012&pages=814-825&pmid=22387027&doi=10.1016/j.molcel.2012.01.017&)

22. Gravand A., Alesaeidi S., Khoshbakht S., Saghaei M., Kenarangi T., Mosallaei M., et al. (2023). Demethylation of CDKN2A in systemic lupus erythematosus and rheumatoid arthritis: a blood biomarker for diagnosis and assessment of disease activity. Clin. Rheumatol. 42, 3387–3395. 10.1007/s10067-023-06736-z  [DOI](https://doi.org/10.1007/s10067-023-06736-z) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/37597101/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Rheumatol.&title=Demethylation%20of%20CDKN2A%20in%20systemic%20lupus%20erythematosus%20and%20rheumatoid%20arthritis:%20a%20blood%20biomarker%20for%20diagnosis%20and%20assessment%20of%20disease%20activity&author=A.%20Gravand&author=S.%20Alesaeidi&author=S.%20Khoshbakht&author=M.%20Saghaei&author=T.%20Kenarangi&volume=42&publication_year=2023&pages=3387-3395&pmid=37597101&doi=10.1007/s10067-023-06736-z&)

23. Gupta N., Singh S., Maturu V. N., Sharma Y. P., Gill K. D. (2011). Paraoxonase 1 (PON1) polymorphisms, haplotypes and activity in predicting cad risk in North-West Indian Punjabis. PLoS One 6 (5), e17805. 10.1371/journal.pone.0017805  [DOI](https://doi.org/10.1371/journal.pone.0017805) | [PMC free article](/articles/PMC3101202/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21629682/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Paraoxonase%201%20(PON1)%20polymorphisms,%20haplotypes%20and%20activity%20in%20predicting%20cad%20risk%20in%20North-West%20Indian%20Punjabis&author=N.%20Gupta&author=S.%20Singh&author=V.%20N.%20Maturu&author=Y.%20P.%20Sharma&author=K.%20D.%20Gill&volume=6&issue=5&publication_year=2011&pages=e17805&pmid=21629682&doi=10.1371/journal.pone.0017805&)

24. Hannon E., Gorrie-Stone T. J., Smart M. C., Burrage J., Hughes A., Bao Y., et al. (2018). Leveraging DNA-methylation quantitative-trait loci to characterize the relationship between methylomic variation, gene expression, and complex traits. Am. J. Hum. Genet. 103 (5), 654–665. 10.1016/j.ajhg.2018.09.007  [DOI](https://doi.org/10.1016/j.ajhg.2018.09.007) | [PMC free article](/articles/PMC6217758/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30401456/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Hum.%20Genet.&title=Leveraging%20DNA-methylation%20quantitative-trait%20loci%20to%20characterize%20the%20relationship%20between%20methylomic%20variation,%20gene%20expression,%20and%20complex%20traits&author=E.%20Hannon&author=T.%20J.%20Gorrie-Stone&author=M.%20C.%20Smart&author=J.%20Burrage&author=A.%20Hughes&volume=103&issue=5&publication_year=2018&pages=654-665&pmid=30401456&doi=10.1016/j.ajhg.2018.09.007&)

25. Höller M., Westin G., Jiricny J., Schaffner W. (1988). Sp1 transcription factor binds DNA and activates transcription even when the binding site is CpG methylated. Genes Dev. 2 (9), 1127–1135. 10.1101/gad.2.9.1127  [DOI](https://doi.org/10.1101/gad.2.9.1127) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/3056778/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genes%20Dev.&title=Sp1%20transcription%20factor%20binds%20DNA%20and%20activates%20transcription%20even%20when%20the%20binding%20site%20is%20CpG%20methylated&author=M.%20H%C3%B6ller&author=G.%20Westin&author=J.%20Jiricny&author=W.%20Schaffner&volume=2&issue=9&publication_year=1988&pages=1127-1135&pmid=3056778&doi=10.1101/gad.2.9.1127&)

26. Huen K., Yousefi P., Street K., Eskenazi B., Holland N. (2015). PON1 as a model for integration of genetic, epigenetic, and expression data on candidate susceptibility genes. Environ. Epigenet 1 (1), dvv003. 10.1093/eep/dvv003  [DOI](https://doi.org/10.1093/eep/dvv003) | [PMC free article](/articles/PMC4762373/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26913202/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Environ.%20Epigenet&title=PON1%20as%20a%20model%20for%20integration%20of%20genetic,%20epigenetic,%20and%20expression%20data%20on%20candidate%20susceptibility%20genes&author=K.%20Huen&author=P.%20Yousefi&author=K.%20Street&author=B.%20Eskenazi&author=N.%20Holland&volume=1&issue=1&publication_year=2015&pages=dvv003&pmid=26913202&doi=10.1093/eep/dvv003&)

27. Izzi B., Pistoni M., Cludts K., Akkor P., Lambrechts D., Verfaillie C., et al. (2016). Allele-specific DNA methylation reinforces PEAR1 enhancer activity. Blood 128 (7), 1003–1012. 10.1182/blood-2015-11-682153  [DOI](https://doi.org/10.1182/blood-2015-11-682153) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27313330/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Allele-specific%20DNA%20methylation%20reinforces%20PEAR1%20enhancer%20activity&author=B.%20Izzi&author=M.%20Pistoni&author=K.%20Cludts&author=P.%20Akkor&author=D.%20Lambrechts&volume=128&issue=7&publication_year=2016&pages=1003-1012&pmid=27313330&doi=10.1182/blood-2015-11-682153&)

28. Julià A., Absher D., López-Lasanta M., Palau N., Pluma A., Waite Jones L., et al. (2017). Epigenome-wide association study of rheumatoid arthritis identifies differentially methylated loci in B cells. Hum. Mol. Genet. 26 (14), 2803–2811. 10.1093/hmg/ddx177  [DOI](https://doi.org/10.1093/hmg/ddx177) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28475762/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum.%20Mol.%20Genet.&title=Epigenome-wide%20association%20study%20of%20rheumatoid%20arthritis%20identifies%20differentially%20methylated%20loci%20in%20B%20cells&author=A.%20Juli%C3%A0&author=D.%20Absher&author=M.%20L%C3%B3pez-Lasanta&author=N.%20Palau&author=A.%20Pluma&volume=26&issue=14&publication_year=2017&pages=2803-2811&pmid=28475762&doi=10.1093/hmg/ddx177&)

29. Koller G., Cusnir I., Hall J., Ye C. (2019). Reversible alopecia areata: a little known side effect of leflunomide. Clin. Rheumatol. 38 (7), 2015–2016. 10.1007/s10067-019-04577-3  [DOI](https://doi.org/10.1007/s10067-019-04577-3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31044385/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Rheumatol.&title=Reversible%20alopecia%20areata:%20a%20little%20known%20side%20effect%20of%20leflunomide&author=G.%20Koller&author=I.%20Cusnir&author=J.%20Hall&author=C.%20Ye&volume=38&issue=7&publication_year=2019&pages=2015-2016&pmid=31044385&doi=10.1007/s10067-019-04577-3&)

30. Li X., Yu Q. (2019). PON1 hypermethylation is associated with progression of renal cell carcinoma. J. Cell Mol. Med. 23 (10), 6646–6657. 10.1111/jcmm.14537  [DOI](https://doi.org/10.1111/jcmm.14537) | [PMC free article](/articles/PMC6787518/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31400051/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Cell%20Mol.%20Med.&title=PON1%20hypermethylation%20is%20associated%20with%20progression%20of%20renal%20cell%20carcinoma&author=X.%20Li&author=Q.%20Yu&volume=23&issue=10&publication_year=2019&pages=6646-6657&pmid=31400051&doi=10.1111/jcmm.14537&)

31. Manuel Sánchez-Maldonado J., Martínez-Bueno M., Canhão H., Ter Horst R., Muñoz-Peña S., Moñiz-Díez A., et al. (2020). NFKB2 polymorphisms associate with the risk of developing rheumatoid arthritis and response to TNF inhibitors: results from the REPAIR consortium. Sci. Rep. 10 (1), 4316. 10.1038/s41598-020-61331-5  [DOI](https://doi.org/10.1038/s41598-020-61331-5) | [PMC free article](/articles/PMC7062729/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32152480/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&title=NFKB2%20polymorphisms%20associate%20with%20the%20risk%20of%20developing%20rheumatoid%20arthritis%20and%20response%20to%20TNF%20inhibitors:%20results%20from%20the%20REPAIR%20consortium&author=J.%20Manuel%20S%C3%A1nchez-Maldonado&author=M.%20Mart%C3%ADnez-Bueno&author=H.%20Canh%C3%A3o&author=R.%20Ter%20Horst&author=S.%20Mu%C3%B1oz-Pe%C3%B1a&volume=10&issue=1&publication_year=2020&pages=4316&pmid=32152480&doi=10.1038/s41598-020-61331-5&)

32. Molina Molina M. I., Pinochet Paiva C. M., Quezada Morales J. I. (2015). Alopecia areata secondary to the use of leflunomide in patients with rheumatoid arthritis: a case report and literature review. Medwave 15 (11), e6350. 10.5867/medwave.2015.11.6350  [DOI](https://doi.org/10.5867/medwave.2015.11.6350) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26819061/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Medwave&title=Alopecia%20areata%20secondary%20to%20the%20use%20of%20leflunomide%20in%20patients%20with%20rheumatoid%20arthritis:%20a%20case%20report%20and%20literature%20review&author=M.%20I.%20Molina%20Molina&author=C.%20M.%20Pinochet%20Paiva&author=J.%20I.%20Quezada%20Morales&volume=15&issue=11&publication_year=2015&pages=e6350&pmid=26819061&doi=10.5867/medwave.2015.11.6350&)

33. Myasoedova E., Davis J. M., 3rd, Crowson C. S., Gabriel S. E. (2010). Epidemiology of rheumatoid arthritis: rheumatoid arthritis and mortality. Curr. Rheumatol. Rep. 12 (5), 379–385. 10.1007/s11926-010-0117-y  [DOI](https://doi.org/10.1007/s11926-010-0117-y) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20645137/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr.%20Rheumatol.%20Rep.&title=Epidemiology%20of%20rheumatoid%20arthritis:%20rheumatoid%20arthritis%20and%20mortality&author=E.%20Myasoedova&author=J.%20M.%20Davis&author=C.%20S.%20Crowson&author=S.%20E.%20Gabriel&volume=12&issue=5&publication_year=2010&pages=379-385&pmid=20645137&doi=10.1007/s11926-010-0117-y&)

34. Nair N., Plant D., Verstappen S. M., Isaacs J. D., Morgan A. W., Hyrich K. L., et al. (2020). Differential DNA methylation correlates with response to methotrexate in rheumatoid arthritis. Rheumatol. Oxf. 59 (6), 1364–1371. 10.1093/rheumatology/kez411  [DOI](https://doi.org/10.1093/rheumatology/kez411) | [PMC free article](/articles/PMC7244777/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31598719/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Rheumatol.%20Oxf.&title=Differential%20DNA%20methylation%20correlates%20with%20response%20to%20methotrexate%20in%20rheumatoid%20arthritis&author=N.%20Nair&author=D.%20Plant&author=S.%20M.%20Verstappen&author=J.%20D.%20Isaacs&author=A.%20W.%20Morgan&volume=59&issue=6&publication_year=2020&pages=1364-1371&pmid=31598719&doi=10.1093/rheumatology/kez411&)

35. Nair N., Wilson A. G., Barton A. (2017). DNA methylation as a marker of response in rheumatoid arthritis. Pharmacogenomics 18 (14), 1323–1332. 10.2217/pgs-2016-0195  [DOI](https://doi.org/10.2217/pgs-2016-0195) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28836487/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=DNA%20methylation%20as%20a%20marker%20of%20response%20in%20rheumatoid%20arthritis&author=N.%20Nair&author=A.%20G.%20Wilson&author=A.%20Barton&volume=18&issue=14&publication_year=2017&pages=1323-1332&pmid=28836487&doi=10.2217/pgs-2016-0195&)

36. Ohta R., Sano C. (2024). Effectiveness of general medicine in the management of rheumatoid arthritis in rural settings: a systematic review and meta-ethnography. Cureus 16 (11), e73726. 10.7759/cureus.73726  [DOI](https://doi.org/10.7759/cureus.73726) | [PMC free article](/articles/PMC11646328/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/39677127/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cureus&title=Effectiveness%20of%20general%20medicine%20in%20the%20management%20of%20rheumatoid%20arthritis%20in%20rural%20settings:%20a%20systematic%20review%20and%20meta-ethnography&author=R.%20Ohta&author=C.%20Sano&volume=16&issue=11&publication_year=2024&pages=e73726&pmid=39677127&doi=10.7759/cureus.73726&)

37. Parada-Turska J., Wójcicka G., Beltowski J. (2020). Paraoxonase 1 phenotype and protein N-homocysteinylation in patients with rheumatoid arthritis: implications for cardiovascular disease. Antioxidants (Basel) 9 (9), 899. 10.3390/antiox9090899  [DOI](https://doi.org/10.3390/antiox9090899) | [PMC free article](/articles/PMC7555791/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32967340/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Antioxidants%20(Basel)&title=Paraoxonase%201%20phenotype%20and%20protein%20N-homocysteinylation%20in%20patients%20with%20rheumatoid%20arthritis:%20implications%20for%20cardiovascular%20disease&author=J.%20Parada-Turska&author=G.%20W%C3%B3jcicka&author=J.%20Beltowski&volume=9&issue=9&publication_year=2020&pages=899&pmid=32967340&doi=10.3390/antiox9090899&)

38. Qi W., Xia Y., Li X., Cao J. (2021). Clinical efficacy and safety of methotrexate compared with leflunomide in the treatment of rheumatoid arthritis: a protocol for systematic review and meta-analysis. Med. Baltim. 100 (51), e28285. 10.1097/MD.0000000000028285  [DOI](https://doi.org/10.1097/MD.0000000000028285) | [PMC free article](/articles/PMC8701934/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34941113/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Med.%20Baltim.&title=Clinical%20efficacy%20and%20safety%20of%20methotrexate%20compared%20with%20leflunomide%20in%20the%20treatment%20of%20rheumatoid%20arthritis:%20a%20protocol%20for%20systematic%20review%20and%20meta-analysis&author=W.%20Qi&author=Y.%20Xia&author=X.%20Li&author=J.%20Cao&volume=100&issue=51&publication_year=2021&pages=e28285&pmid=34941113&doi=10.1097/MD.0000000000028285&)

39. Ravaei A., Pulsatelli L., Assirelli E., Meliconi R., Ciaffi J., Gremese E., et al. (2022). Seropositivity-dependent association between LINE-1 methylation and response to methotrexate therapy in early rheumatoid arthritis patients. Genes (Basel) 13 (11), 2012. 10.3390/genes13112012  [DOI](https://doi.org/10.3390/genes13112012) | [PMC free article](/articles/PMC9690891/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36360249/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genes%20(Basel)&title=Seropositivity-dependent%20association%20between%20LINE-1%20methylation%20and%20response%20to%20methotrexate%20therapy%20in%20early%20rheumatoid%20arthritis%20patients&author=A.%20Ravaei&author=L.%20Pulsatelli&author=E.%20Assirelli&author=R.%20Meliconi&author=J.%20Ciaffi&volume=13&issue=11&publication_year=2022&pages=2012&pmid=36360249&doi=10.3390/genes13112012&)

40. Roodenrijs N. M. T., Welsing P. M. J., van der Goes M. C., Tekstra J., Lafeber F., Jacobs J. W. G., et al. (2021). Healthcare utilization and economic burden of difficult-to-treat rheumatoid arthritis: a cost-of-illness study. Rheumatol. Oxf. 60 (10), 4681–4690. 10.1093/rheumatology/keab078  [DOI](https://doi.org/10.1093/rheumatology/keab078) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33502493/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Rheumatol.%20Oxf.&title=Healthcare%20utilization%20and%20economic%20burden%20of%20difficult-to-treat%20rheumatoid%20arthritis:%20a%20cost-of-illness%20study&author=N.%20M.%20T.%20Roodenrijs&author=P.%20M.%20J.%20Welsing&author=M.%20C.%20van%20der%20Goes&author=J.%20Tekstra&author=F.%20Lafeber&volume=60&issue=10&publication_year=2021&pages=4681-4690&pmid=33502493&doi=10.1093/rheumatology/keab078&)

41. Sánchez-Maldonado J. M., Cáliz R., López-Nevot M., Cabrera-Serrano A. J., Moñiz-Díez A., Canhão H., et al. (2021). Validation of GWAS-identified variants for anti-TNF drug response in rheumatoid arthritis: a meta-analysis of two large cohorts. Front. Immunol. 12, 672255. 10.3389/fimmu.2021.672255  [DOI](https://doi.org/10.3389/fimmu.2021.672255) | [PMC free article](/articles/PMC8579100/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34777329/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front.%20Immunol.&title=Validation%20of%20GWAS-identified%20variants%20for%20anti-TNF%20drug%20response%20in%20rheumatoid%20arthritis:%20a%20meta-analysis%20of%20two%20large%20cohorts&author=J.%20M.%20S%C3%A1nchez-Maldonado&author=R.%20C%C3%A1liz&author=M.%20L%C3%B3pez-Nevot&author=A.%20J.%20Cabrera-Serrano&author=A.%20Mo%C3%B1iz-D%C3%ADez&volume=12&publication_year=2021&pages=672255&pmid=34777329&doi=10.3389/fimmu.2021.672255&)

42. Schalkwyk L. C., Meaburn E. L., Smith R., Dempster E. L., Jeffries A. R., Davies M. N., et al. (2010). Allelic skewing of DNA methylation is widespread across the genome. Am. J. Hum. Genet. 86 (2), 196–212. 10.1016/j.ajhg.2010.01.014  [DOI](https://doi.org/10.1016/j.ajhg.2010.01.014) | [PMC free article](/articles/PMC2820163/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20159110/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Hum.%20Genet.&title=Allelic%20skewing%20of%20DNA%20methylation%20is%20widespread%20across%20the%20genome&author=L.%20C.%20Schalkwyk&author=E.%20L.%20Meaburn&author=R.%20Smith&author=E.%20L.%20Dempster&author=A.%20R.%20Jeffries&volume=86&issue=2&publication_year=2010&pages=196-212&pmid=20159110&doi=10.1016/j.ajhg.2010.01.014&)

43. Scirè C. A., Lunt M., Marshall T., Symmons D. P., Verstappen S. M. (2014). Early remission is associated with improved survival in patients with inflammatory polyarthritis: results from the Norfolk Arthritis Register. Ann. Rheum. Dis. 73 (9), 1677–1682. 10.1136/annrheumdis-2013-203339  [DOI](https://doi.org/10.1136/annrheumdis-2013-203339) | [PMC free article](/articles/PMC4145442/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23749581/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann.%20Rheum.%20Dis.&title=Early%20remission%20is%20associated%20with%20improved%20survival%20in%20patients%20with%20inflammatory%20polyarthritis:%20results%20from%20the%20Norfolk%20Arthritis%20Register&author=C.%20A.%20Scir%C3%A8&author=M.%20Lunt&author=T.%20Marshall&author=D.%20P.%20Symmons&author=S.%20M.%20Verstappen&volume=73&issue=9&publication_year=2014&pages=1677-1682&pmid=23749581&doi=10.1136/annrheumdis-2013-203339&)

44. Shoemaker R., Deng J., Wang W., Zhang K. (2010). Allele-specific methylation is prevalent and is contributed by CpG-SNPs in the human genome. Genome Res. 20 (7), 883–889. 10.1101/gr.104695.109  [DOI](https://doi.org/10.1101/gr.104695.109) | [PMC free article](/articles/PMC2892089/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20418490/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res.&title=Allele-specific%20methylation%20is%20prevalent%20and%20is%20contributed%20by%20CpG-SNPs%20in%20the%20human%20genome&author=R.%20Shoemaker&author=J.%20Deng&author=W.%20Wang&author=K.%20Zhang&volume=20&issue=7&publication_year=2010&pages=883-889&pmid=20418490&doi=10.1101/gr.104695.109&)

45. Smolen J. S., Aletaha D., McInnes I. B. (2016). Rheumatoid arthritis. Lancet 388 (10055), 2023–2038. 10.1016/S0140-6736(16)30173-8  [DOI](https://doi.org/10.1016/S0140-6736(16)30173-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27156434/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Rheumatoid%20arthritis&author=J.%20S.%20Smolen&author=D.%20Aletaha&author=I.%20B.%20McInnes&volume=388&issue=10055&publication_year=2016&pages=2023-2038&pmid=27156434&doi=10.1016/S0140-6736(16)30173-8&)

46. Srivastava S., Rasool M. (2024). Genetics, epigenetics and autoimmunity constitute a Bermuda triangle for the pathogenesis of rheumatoid arthritis. Life Sci. 357, 123075. 10.1016/j.lfs.2024.123075  [DOI](https://doi.org/10.1016/j.lfs.2024.123075) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/39341491/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Life%20Sci.&title=Genetics,%20epigenetics%20and%20autoimmunity%20constitute%20a%20Bermuda%20triangle%20for%20the%20pathogenesis%20of%20rheumatoid%20arthritis&author=S.%20Srivastava&author=M.%20Rasool&volume=357&publication_year=2024&pages=123075&pmid=39341491&doi=10.1016/j.lfs.2024.123075&)

47. Su J., Li J., Yu Q., Xu X., Wang J., Yang J., et al. (2019). Association of PON1 gene promoter DNA methylation with the risk of Clopidogrel resistance in patients with coronary artery disease. J. Clin. Lab. Anal. 33 (5), e22867. 10.1002/jcla.22867  [DOI](https://doi.org/10.1002/jcla.22867) | [PMC free article](/articles/PMC6595294/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30891852/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Lab.%20Anal.&title=Association%20of%20PON1%20gene%20promoter%20DNA%20methylation%20with%20the%20risk%20of%20Clopidogrel%20resistance%20in%20patients%20with%20coronary%20artery%20disease&author=J.%20Su&author=J.%20Li&author=Q.%20Yu&author=X.%20Xu&author=J.%20Wang&volume=33&issue=5&publication_year=2019&pages=e22867&pmid=30891852&doi=10.1002/jcla.22867&)

48. Vanderkraats N. D., Hiken J. F., Decker K. F., Edwards J. R. (2013). Discovering high-resolution patterns of differential DNA methylation that correlate with gene expression changes. Nucleic Acids Res. 41 (14), 6816–6827. 10.1093/nar/gkt482  [DOI](https://doi.org/10.1093/nar/gkt482) | [PMC free article](/articles/PMC3737560/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23748561/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&title=Discovering%20high-resolution%20patterns%20of%20differential%20DNA%20methylation%20that%20correlate%20with%20gene%20expression%20changes&author=N.%20D.%20Vanderkraats&author=J.%20F.%20Hiken&author=K.%20F.%20Decker&author=J.%20R.%20Edwards&volume=41&issue=14&publication_year=2013&pages=6816-6827&pmid=23748561&doi=10.1093/nar/gkt482&)

49. van Riel P. L., Renskers L. (2016). The Disease Activity Score (DAS) and the Disease Activity Score using 28 joint counts (DAS28) in the management of rheumatoid arthritis. Clin. Exp. Rheumatol. 34 (5 Suppl. 101), S40-S44–s4.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/27762189/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Exp.%20Rheumatol.&title=The%20Disease%20Activity%20Score%20(DAS)%20and%20the%20Disease%20Activity%20Score%20using%2028%20joint%20counts%20(DAS28)%20in%20the%20management%20of%20rheumatoid%20arthritis&author=P.%20L.%20van%20Riel&author=L.%20Renskers&volume=34&issue=5%20Suppl.%20101&publication_year=2016&pages=S40-S44-s4&pmid=27762189&)

50. Wang L., Zhang Y., Zhu L., Li Y. (2021). Efficacy and safety of methotrexate in the treatment of rheumatoid arthritis: a retrospective study. Ann. Palliat. Med. 10 (12), 12095–12100. 10.21037/apm-21-2331  [DOI](https://doi.org/10.21037/apm-21-2331) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35016466/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann.%20Palliat.%20Med.&title=Efficacy%20and%20safety%20of%20methotrexate%20in%20the%20treatment%20of%20rheumatoid%20arthritis:%20a%20retrospective%20study&author=L.%20Wang&author=Y.%20Zhang&author=L.%20Zhu&author=Y.%20Li&volume=10&issue=12&publication_year=2021&pages=12095-12100&pmid=35016466&doi=10.21037/apm-21-2331&)

51. Wang R. H., Wen W. X., Jiang Z. P., Du Z. P., Ma Z. H., Lu A. L., et al. (2023). The clinical value of neutrophil-to-lymphocyte ratio (NLR), systemic immune-inflammation index (SII), platelet-to-lymphocyte ratio (PLR) and systemic inflammation response index (SIRI) for predicting the occurrence and severity of pneumonia in patients with intracerebral hemorrhage. Front. Immunol. 14, 1115031. 10.3389/fimmu.2023.1115031  [DOI](https://doi.org/10.3389/fimmu.2023.1115031) | [PMC free article](/articles/PMC9969881/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36860868/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front.%20Immunol.&title=The%20clinical%20value%20of%20neutrophil-to-lymphocyte%20ratio%20(NLR),%20systemic%20immune-inflammation%20index%20(SII),%20platelet-to-lymphocyte%20ratio%20(PLR)%20and%20systemic%20inflammation%20response%20index%20(SIRI)%20for%20predicting%20the%20occurrence%20and%20severity%20of%20pneumonia%20in%20patients%20with%20intracerebral%20hemorrhage&author=R.%20H.%20Wang&author=W.%20X.%20Wen&author=Z.%20P.%20Jiang&author=Z.%20P.%20Du&author=Z.%20H.%20Ma&volume=14&publication_year=2023&pages=1115031&pmid=36860868&doi=10.3389/fimmu.2023.1115031&)

52. Warde-Farley D., Donaldson S. L., Comes O., Zuberi K., Badrawi R., Chao P., et al. (2010). The GeneMANIA prediction server: biological network integration for gene prioritization and predicting gene function. Nucleic Acids Res. 38 (Web Server issue), W214–W220. 10.1093/nar/gkq537  [DOI](https://doi.org/10.1093/nar/gkq537) | [PMC free article](/articles/PMC2896186/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20576703/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&title=The%20GeneMANIA%20prediction%20server:%20biological%20network%20integration%20for%20gene%20prioritization%20and%20predicting%20gene%20function&author=D.%20Warde-Farley&author=S.%20L.%20Donaldson&author=O.%20Comes&author=K.%20Zuberi&author=R.%20Badrawi&volume=38&issue=Web%20Server%20issue&publication_year=2010&pages=W214-W220&pmid=20576703&doi=10.1093/nar/gkq537&)

53. Wells G., Becker J. C., Teng J., Dougados M., Schiff M., Smolen J., et al. (2009). Validation of the 28-joint Disease Activity Score (DAS28) and European League against Rheumatism response criteria based on C-reactive protein against disease progression in patients with rheumatoid arthritis, and comparison with the DAS28 based on erythrocyte sedimentation rate. Ann. Rheum. Dis. 68 (6), 954–960. 10.1136/ard.2007.084459  [DOI](https://doi.org/10.1136/ard.2007.084459) | [PMC free article](/articles/PMC2674547/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18490431/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann.%20Rheum.%20Dis.&title=Validation%20of%20the%2028-joint%20Disease%20Activity%20Score%20(DAS28)%20and%20European%20League%20against%20Rheumatism%20response%20criteria%20based%20on%20C-reactive%20protein%20against%20disease%20progression%20in%20patients%20with%20rheumatoid%20arthritis,%20and%20comparison%20with%20the%20DAS28%20based%20on%20erythrocyte%20sedimentation%20rate&author=G.%20Wells&author=J.%20C.%20Becker&author=J.%20Teng&author=M.%20Dougados&author=M.%20Schiff&volume=68&issue=6&publication_year=2009&pages=954-960&pmid=18490431&doi=10.1136/ard.2007.084459&)

54. Zaragoza-García O., Guzmán-Guzmán I. P., Moreno-Godínez M. E., Navarro-Zarza J. E., Antonio-Vejar V., Ramírez M., et al. (2021). PON-1 haplotype (-108C>T, L55M, and Q192R) modulates the serum levels and activity PONase promoting an atherogenic lipid profile in rheumatoid arthritis patients. Clin. Rheumatol. 40 (2), 741–752. 10.1007/s10067-020-05218-w  [DOI](https://doi.org/10.1007/s10067-020-05218-w) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32556934/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Rheumatol.&title=PON-1%20haplotype%20(-108C>T,%20L55M,%20and%20Q192R)%20modulates%20the%20serum%20levels%20and%20activity%20PONase%20promoting%20an%20atherogenic%20lipid%20profile%20in%20rheumatoid%20arthritis%20patients&author=O.%20Zaragoza-Garc%C3%ADa&author=I.%20P.%20Guzm%C3%A1n-Guzm%C3%A1n&author=M.%20E.%20Moreno-God%C3%ADnez&author=J.%20E.%20Navarro-Zarza&author=V.%20Antonio-Vejar&volume=40&issue=2&publication_year=2021&pages=741-752&pmid=32556934&doi=10.1007/s10067-020-05218-w&)
