# The Impact of the Rs1044457 Polymorphism in the CMPK1 Gene on the Response Rate to Gemcitabine‐Based Chemotherapy in Metastatic NSCLC Patients

## Metadata
**Authors:** Ghassan Saod Saleh, Fouad Kadhim Gatea, Qasim Sharhan Al‐Mayah, Hayder Lazim
**Journal:** Advanced Genetics
**Date:** 2025 Apr 4
**DOI:** [10.1002/ggn2.202400058](https://doi.org/10.1002/ggn2.202400058)
**PMID:** 40657553
**PMCID:** PMC12245479
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC12245479/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC12245479/pdf/GGN2-6-2400058.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC12245479/pdf/GGN2-6-2400058.pdf)

## Abstract

This study aims to evaluate the role of a specific gene polymorphism, Cytidine/Uridine Monophosphate Kinase 1 (CMPK1) rs1044457, in predicting the response to gemcitabine‐based chemotherapy in patients with NSCLC. A total of 98 NSCLC patients are enrolled in the study. Based on their response, patients are classified as either responders or non‐responders. Specific primers are designed to amplify the rs1044457 variant and performed genotyping using restriction fragment length polymorphism (RFLP). The rs1044457 variant showed a statistically significant difference in frequency between responder and non‐responder patients. The mutant genotype (TT) is more frequent in non‐responder patients (18.75%) compared to responder patients (4%), with an odds ratio [OR] of 5.93 (95% confidence interval [CI] = 1.16–30.25, p = 0.032). Additionally, at the allelic level, the mutant allele (T) is more common in non‐responder patients (36.46%) compared to responder patients (23%), with a statistically significant odds ratio of 1.92 (95% CI = 1.03–3.58, p = 0.040). The findings of this study suggest that the mutant allele (allele T) of the rs1044457 variant may serve as a risk factor for resistance to gemcitabine‐based chemotherapy in patients with NSCLC.

Keywords: CMPK1 gene polymorphism, gemcitabine response rate, non‐small cell lung cancer, rs1044457 polymorphism

## 1. Introduction

Lung cancer is the leading cause of mortality globally.[](#ggn210106-supinfo-0001)In particular, non‐small cell lung cancer (NSCLC) accounts for the majority of these cases. While novel drugs targeting specific molecular pathways have become available, cytotoxic chemotherapy remains the primary treatment modality for advanced NSCLC. Historically, gemcitabine and taxane/cisplatin have been conventionally employed in the management of NSCLC, demonstrating comparable treatment efficacy in terms of response rates, patient survival, and time to disease progression, as demonstrated in phase III clinical trials.^[^ [1](#ggn210106-bib-0001), [2](#ggn210106-bib-0002) ^]^ Gemcitabine, a pyrimidine analogue (2,2‐difluorodeoxycytidine, dFdC), exerts its cytotoxic effects as an antimetabolite and exhibits activity against a broad spectrum of solid tumours, including pancreatic, oesophageal, breast, and NSCLC.^[^ [3](#ggn210106-bib-0003) ^]^

Gemcitabine necessitates cellular uptake followed by phosphorylation for its activation. Upon entry into the cell, gemcitabine is phosphorylated by deoxycytidine kinase (dCK) to yield a monophosphate derivative. This monophosphate is subsequently converted to the diphosphate form (dFdCDP) through the action of cytidine monophosphate kinase (CMPK). These phosphorylation steps are critical for the cytotoxic efficacy of gemcitabine. Polymorphisms in the CMPK1 gene, such as rs1044457, can modulate both the activity and expression levels of CMPK1, thereby affecting the therapeutic efficiency of gemcitabine in targeting neoplastic cells.^[^ [4](#ggn210106-bib-0004), [5](#ggn210106-bib-0005) ^]^

Based on the information available, the polymorphism of rs1044457 may play a role in the treatment of non‐small cell lung cancer (NSCLC) patients with gemcitabine. Recent findings suggest that dCK polymorphisms, such as the CC genotype of rs1044457, might influence dCK expression and, ultimately, gemcitabine activity in various cancers, including pancreatic cancer.^[^ [6](#ggn210106-bib-0006) ^]^

In their study, Mirta et al.^[^ [7](#ggn210106-bib-0007) ^]^ identified an association between the CC genotype of the rs1044457CT 3′UTR SNP and reduced formation and clearance of gemcitabine triphosphate in individuals with various solid tumours. This effect was observed in comparison to individuals possessing the TT genotype who also have cancer.

The presence of the SNP rs1044457 (C>T) in the 3′untranslated region (UTR) suggests that it likely plays a regulatory role in the expression of the CMPK1 gene. The 3′UTR is known to regulate mRNA stability, export to the cytoplasm, sub‐cellular localization, and translation efficiency, thereby influencing the overall synthesis of proteins.^[^ [8](#ggn210106-bib-0008) ^]^ Consequently, polymorphisms and/or mutations in these regions can have a strong impact on gene expression and the resulting cellular viability, growth, and development

While the specific impact of this polymorphism on gemcitabine treatment in NSCLC patients is not explicitly outlined in the provided search results, the observed associations in other cancer types suggest the potential for similar implications in NSCLC. However, it's important to note that more robust data are needed to confirm the influence of dCK polymorphisms, including rs1044457, on gemcitabine activity in NSCLC patients.^[^ [9](#ggn210106-bib-0009) ^]^

Genetic polymorphisms play an important role in modulating the activity of transporters and enzymes involved in gemcitabine metabolism, explaining the reported interindividual variability in gemcitabine efficacy. Extensive research has been conducted to investigate the correlation between genetic polymorphisms and the clinical effectiveness of gemcitabine across diverse patient cohorts, including those with lung, pancreatic, and breast cancer.^[^ [10](#ggn210106-bib-0010), [11](#ggn210106-bib-0011), [12](#ggn210106-bib-0012), [13](#ggn210106-bib-0013), [14](#ggn210106-bib-0014) ^]^ Notably, studies in pancreatic cancer patients have revealed a link between the CMPK1 gene polymorphism and the survival rate after gemcitabine therapy.^[^ [15](#ggn210106-bib-0015) ^]^

Therefore, it is reasonable to evaluate that genetic variations in the genes associated with gemcitabine pharmacology may serve as useful predictors for anticipating favourable treatment outcomes with this specific chemotherapeutic agent. The primary objective of the current investigation was to elucidate the impact of the rs1044457 variation on the response to gemcitabine‐based chemotherapy in individuals diagnosed with non‐small cell lung cancer (NSCLC).

## 2. Patients and Methods

### 2.1. Study Design

The present cross‐sectional study was conducted over a period spanning from August 2022 to January 2023, encompassing a cohort of 98 patients of Arabic ethnicity diagnosed with non‐small cell lung cancer (NSCLC) who had undergone four to six cycles of gemcitabine chemotherapy. The study was carried out at multiple clinical sites, including the Oncology Department at Baghdad Medical City, the Oncology Teaching Hospital, the Oncology Department at Al Imamain Al Kadhumain Medical City, and Al Amal Hospital. Furthermore, the molecular assays were conducted at the medical research unit of the College of Medicine, Al‐Nahrain University. The data were then collected in February 2023 and made accessible for research purposes in March 2023.

The inclusion criteria for the study comprised patients with NSCLC who had received four to six cycles of gemcitabine chemotherapy, while those undergoing palliative therapy for advanced NSCLC stages, newly diagnosed patients, and individuals who received gemcitabine therapy for less than four cycles were excluded. The treatment regimen for gemcitabine consisted of an administration of 1000 mg m^−2^ on days 1 and 8, in conjunction with docetaxel at a dosage of 85 mg m^−2^ on day 8, with a three‐week interval between cycles.

Data pertaining to demographic variables such as age, gender, smoking status, comorbidities, family history of malignancy, and occupation, as well as clinical information encompassing treatment protocol, tumour characteristics (including size, site, primary tumour classification, lymph node involvement, and tumour grade and stage), were collected from patient's medical records for analysis.

Nevertheless, substantial inter‐individual variability in patient responses to gemcitabine‐based chemotherapy necessitates the identification of determinants influencing patient outcomes with this agent.

### 2.2. Ethical Considerations and Consent

The samples used in this investigation were acquired specifically for research purposes, and their use was approved by the Ethical Review Board of the College of Medicine at Al‐Nahrain University (IRB/101, No: UNCOMIRB01022024/2nd of August 2022). Before their participation, all patients involved completed informed consent documentation. Furthermore, the authors of this study had no access to any information that could potentially disclose the identities of the participants, neither during nor after the data gathering process.

### 2.3. Patients Categorization

The study participants were classified into two groups based on their treatment response, namely responders and non‐responders. Their treatment responses were assessed in accordance with the response evaluation criteria in solid tumours (RECIST) guidelines by the treating physician. RECIST criteria classified the outcome into Complete Response (CR), Partial Response (PR), Progressive Disease (PD) and Stable Disease (SD).^[^ [16](#ggn210106-bib-0016) ^]^ However, in our study, responders included SD and PR (there was no CR), while non‐responders included PR, based on the reduction in the primary tumour size.

Tumour response was evaluated through clinical examination and the use of computed tomography (CT) scans at intervals of every two treatment cycles. Additionally, the efficacy and toxicity of the treatment were evaluated, and any chemotherapy‐induced adverse effects were documented.

### 2.4. Blood Samples, Genomic DNA Extraction and Gene Amplification

≈3 mL of venous blood was aseptically collected from each participant and transferred into EDTA tubes to prevent coagulation. Genomic DNA extraction was performed using a commercially available kit (Promega, USA) in accordance with the manufacturer's prescribed protocol. The quantification of the extracted DNA samples was conducted utilizing the QuantiFluor dsDNA System (Promega, USA) and measured on the Quantus Fluorometer (Promega, USA) following the manufacturer's protocols.

The current study used two primers for gene amplification and genotyping of the rs1044457 variant, which was obtained from Macrogen, Korea. The forward primer sequence was (5′‐TGCTCACCAAACGAAGGGTA‐3′) and the reverse primer sequence was (5′‐AGTTCCTTTGAAATTTGGTCCACA‐3). The PCR conditions employed in this study were as follows: an initial denaturation step at 95 °C for 5 min, followed by denaturation at 94 °C for 30 s, annealing at 58 °C for 30 s, extension at 72 °C for 40 s, with the three‐step cycle repeated 30 times, and finally a final extension step at 72 °C for 7 min. Subsequently, the PCR products underwent 2% gel electrophoresis, and the banding pattern was visualized using UV light.

The PCR product was then subjected to digestion by the restriction enzyme. Specifically, 5 µL of the PCR products were combined with 0.1µL of the (*Aci*l, 10U µL^−1^) restriction enzyme obtained from Sibenzyme, Russia. Additionally, 1.5 µL of the 10X restriction buffer and 0.1 µL of bovine serum albumin (BSA) were added to the mixture, which was then brought to a total volume of 15 µL using molecular grade water. Mineral oil (20 µl) was added to all tubes to prevent evaporation. The reaction mixture was incubated in a 60 °C water bath for 3 h. The resulting restriction products were subsequently resolved on a 2% agarose gel.

### 2.5. Statistical Analysis

The statistical analysis was performed using SPSS software version 25.0 (SPSS, Chicago). Continuous data were subjected to the calculation of mean and standard deviation, followed by analysis using the Student's t‐test. The deviation of genotype, namely CC, CT, and TT, in both the responsive and unresponsive groups, from Hardy‐Weinberg Equilibrium (HWE) was assessed using Chi‐square. This test also served for the evaluation of binomial variables expressed as both numerical values and percentages. To explore the association between the CMPK1 rs1044457 and responsiveness to gemcitabine, binary logistic regression was employed to compute the odds ratio (OR) and the corresponding 95% confidence intervals (CI). A p‐value of 0.05 or less was considered indicative of statistical significance.

Continuous data were initially subjected to a normality test using the Shapiro‐Wilk test. Most variables were found to be normally distributed, with the exception of tumour size reduction. Consequently, the comparison between responsive and non‐responsive groups in terms of tumour size reduction will be analysed using the non‐parametric Mann–Whitney U test.

## 3. Results

### 3.1. Demographic Characteristics of the Patients

This study enrolled a cohort comprising 98 individuals diagnosed with NSCLC who underwent treatment involving gemcitabine. The average age of the cohorts of patients exhibiting responsive and non‐responsive reactions to the treatment was found to be 56.04 ± 10.12 years and 57.79 ± 9.02 years, respectively, with no significant difference observed. Furthermore, the distribution of male and female participants was nearly uniform in both groups, displaying no statistically significant divergence. Similarly, the two cohorts demonstrated comparable characteristics in terms of weight, height, and body mass index (BMI), with no statistically significant difference. Additionally, the prevalence of smoking was observed to be common within both cohorts, reported at 72% and 77.07% among responsive and non‐responsive patients, respectively, without significant differences. Moreover, approximately two‐thirds of patients in each group exhibited an Eastern Cooperative Oncology Group (ECOG) score of 2, with no significant differences (**Table** [1](#ggn210106-tbl-0001)).

### Table 1.

| Variables |  | Responsive [n = 50] | Non‐responsive [n = 48] | P‐value |
| --- | --- | --- | --- | --- |
| Age (years) | Mean ± SD | 56.04 ± 10.12 | 57.79 ± 9.02 | 0.369 * |
| Range | 38–70 | 34–70 |   |  |
| Gender | Male | 25(50%) | 25(52.08%) | 0.837 ** |
| Female | 25(50%) | 23(47.92%) |   |  |
| Weight (kg) | Mean ± SD | 66.34 ± 6.49 | 65.35 ± 7.13 | 0.476 * |
| Range | 54–83 | 54–82 |   |  |
| Height (cm) | Mean ± SD | 162.2 ± 6.03 | 162.83 ± 5.34 | 0.584 * |
| Range | 150–175 | 152–180 |   |  |
| BMI (kg/m2) | Mean ± SD | 25.17 ± 1.44 | 24.61 ± 1.98 | 0.110 * |
| Range | 21.45–28.65 | 20.09–29.38 |   |  |
| Smoking | Never | 14(28%) | 11(22.92%) | 0.718 ** |
| Ex/current smokers | 36(72%) | 37(77.07%) |   |  |
| ECOG | One | 9(18%) | 4(8.33%) | 0.160 ** |
| Two | 33(66%) | 30(62.5%) |   |  |
| Three | 8(16%) | 14(29.17%) |   |  |

Table 1 Caption: Demographic characteristics of the patients.

### 3.2. Therapeutic and Clinical Characteristics of the Patients

The prevalence of hypertension and ischemic heart diseases (IHD) exhibited a slightly higher incidence among non‐responsive patients (37.5% and 14.58%, respectively) compared to responsive patients (24% and 10%, respectively); however, these disparities did not reach statistical significance. Conversely, other comorbidities were non‐significantly more prevalent among responsive patients than non‐responsive patients (8% vs 6.35%). The initial tumour size in the responsive group was (45.15 ± 36.67 cm), comparable to that of non‐responsive patients at (42.26 ± 33.18 cm), with no statistically significant variance. Following 4–6 cycles of treatment, the mean final tumour size in responsive patients reduced to (34.6 ± 30.94 cm), significantly lower than that of non‐responsive patients (57.86 ± 43.8 cm) with a highly significant difference. Approximately three‐fourths of the patients in either group received 6 treatment cycles, with no significant discrepancy between the two groups (**Table** [2](#ggn210106-tbl-0002)).

### Table 2.

| Variables |  | Responsive [n = 50] | Non‐responsive [n = 48] | P‐value |
| --- | --- | --- | --- | --- |
| Comorbidities | No comorbidity | 12(24%) | 10(20.83%) | 0.821 ** |
| Diabetes mellitus | 12(24%) | 18(37.5%) | 0.147 |  |
| Hypertension | 15(30%) | 15(31.25%) | 0.893 |  |
| Ischemic heart disease | 5(10%) | 7(14.58%) | 0.489 |  |
| Others | 4(8%) | 3(6.35%) | 0.737 |  |
| Initial tumor size, cm | Mean ± SD | 45.15 ± 36.67 | 42.26 ± 33.18 | 0.151 * |
| Range | 3.48–120.96 | 14.94–127.6 |   |  |
| Final tumor size, cm | Mean ± SD | 34.6 ± 30.94 | 57.86 ± 43.8 | 0.003 * |
| Range | 0.5–101.92 | 16.4–142.74 |   |  |
| Treatment cycles | 4 | 13(26.67%) | 9(13.33%) | 0.390 ** |
| 6 | 37(73.33%) | 39(86.67%) |   |  |

Table 2 Caption: Clinical characteristics of the patients.

### 3.3. Reduction in Tumour Size

The study observed a discernible reduction in tumour size as determined through CT scan analysis, with the responsive group exhibiting an average reduction of 11.82 ± 13.98 cm (ranging from 2.27 to 57.51 cm), as opposed to the non‐responsive group, which demonstrated a reduction of −15.31 ± 6.27 cm (ranging from −2.7 to −24.18 cm). The application of the Mann–Whitney U test yielded a notably significant difference in tumour size reduction between the two groups (**Figure** [1](#ggn210106-fig-0001)).

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/abe7/12245479/2516509d91a8/GGN2-6-2400058-g005.jpg)

Reduction in tumour sizes in responsive and non‐responsive patients.

### 3.4. Correlation of Tumour Reduction with Other Variables

Pearson's correlation test was used to investigate the possible correlation between tumour reduction and various continuous variables. Among responsive patients, the analysis revealed a negative correlation between tumour size reduction and age (r = −0.315, p = 0.026), as well as a significant positive correlation with treatment cycles (r = 0.305, p = 0.031). Conversely, in non‐responsive patients, tumour size reduction exhibited no significant correlation with any of the included variables as shown in **Table** [3](#ggn210106-tbl-0003), **Figure** [2](#ggn210106-fig-0002), and [3](#ggn210106-fig-0003).

### Table 3.

| Variable | Responsive |  | Non‐Responsive |  |
| --- | --- | --- | --- | --- |
| Coefficient | p‐value | Coefficient | p‐value |  |
| Age | −0.315 | 0.026 | −0.159 | 0.279 |
| Weight | −0.031 | 0.830 | 0.181 | 0.218 |
| Height | −0.177 | 0.218 | −0.021 | 0.889 |
| BMI | 0.181 | 0.209 | 0.219 | 0.135 |
| No. of cycles | 0.305 | 0.031 | 0.227 | 0.126 |
| ECOG score | −0.221 | 0.118 | −0.256 | 0.080 |

Table 3 Caption: Association between tumour size reduction and other continuous variables in responsive and non‐responsive patients.

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/abe7/12245479/abd9f1e0aac0/GGN2-6-2400058-g003.jpg)

Scatter plot and regression line between age and tumour size reduction in responsive patients.

### Figure 3.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/abe7/12245479/2b092c64d2f9/GGN2-6-2400058-g004.jpg)

Scatter plot and regression line between number of cycles and tumour size reduction in responsive patients.

### 3.5. Association of Tumour Reduction with Gender, Smoking and Comorbidity

In the context of responsive and non‐responsive groups, the findings revealed notable distinctions in tumour size reduction based on gender, smoking history, and comorbidities among the patients. Specifically, females exhibited a greater reduction in tumour size compared to males in the responsive group, displaying mean values of 15.53 ± 16.26 cm and 8.22 ± 8.97 cm, respectively, which was statistically significant. Furthermore, non‐smoker patients demonstrated a significantly higher level of tumour size reduction in comparison to ex/current smokers in both responsive and non‐responsive patients. Additionally, patients without comorbidities exhibited a higher mean reduction in tumour size (12.4 ± 13.01 cm) than those with comorbidities (8.04 ± 4.55 cm), although the difference was not significant (**Table** [4](#ggn210106-tbl-0004)).

### Table 4.

| Variables |  | Responsive | Non‐responsive |
| --- | --- | --- | --- |
| Gender | Males Females | 8.22 ± 8.97 15.53 ± 16.26 | −15.65 ± 5.54 −14.95 ± 7.09 |
| p‐value | 0.055 | 0.705 |  |
| Smoking | Yes No | 7.67 ± 4.1 13.51 ± 15.47 | −17.34 ± 35.79 −13.11 ± 6.14 |
| p‐ value | 0.026 | 0.018 |  |
| Comorbidity | Yes No | 8.04 ± 4.55 12.4 ± 13.01 | −14.93 ± 6.71 −15.96 ± 5.6 |
| p‐value | 0.157 | 0.586 |  |

Table 4 Caption: Association of tumour reduction with gender, smoking and comorbidity.

### 3.6. CMPK1 Gene and rs1044457 Polymorphism

Two primers were employed for the amplification of this polymorphism through conventional PCR. Subsequent gel electrophoresis of the PCR product demonstrated a fragment length of 219 base pairs (bp), as shown in **Figure** [4](#ggn210106-fig-0004). Genotyping was executed utilizing the Restriction Fragment Length Polymorphism (RFLP) technique. Subsequent digestion with the *Aci*l restriction enzyme revealed the presence of three genotypes: CC, CT, and TT, as illustrated in **Figure** [5](#ggn210106-fig-0005).

### Figure 4.

![Figure 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/abe7/12245479/eaf6df35bc98/GGN2-6-2400058-g002.jpg)

Gel electrophoresis of rs1044457 gene plymorphism amplified with a specific pair of primers using conventional PCR. The PCR product was stained with ethidium bromide. The fragment length was 219 bp. M: 100 bp DNA ladder. Samples were randomly selected for responsive (lanes 1–5) and non‐responsive patients (lanes 6–10).

### Figure 5.

![Figure 5](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/abe7/12245479/30301655250e/GGN2-6-2400058-g001.jpg)

Genotyping of rs1044457 gene polymorphism after digestion with AciI restriction enzyme and stained with ethidium bromide. Lanes 1, 6,7 and 12: TT genotype; lanes 2,3,4,9and 10: CC genotypes, lanes 5,8 and 11: CT genotype; M: 50 bp DNA ladder. The fragment length was 219 bp. M: 100 bp DNA ladder. Samples were randomly selected for responsive (lanes 1–6) and non‐responsive patients (lanes 7‐12).

The prevalence of the wild type of CC genotype was observed to be higher in responsive patients compared to non‐responsive patients, accounting for 58% and 45.83% respectively; however, this disparity did not yield statistical significance. Conversely, the occurrence of the mutant genotype (TT) was notably greater in non‐responsive patients (18.75%) in comparison to responsive patients (4%), exhibiting a statistically significant difference (OR = 5.93, 95% CI = 1.16–30.25, p = 0.032). This polymorphism appears to demonstrate dominant inheritance, as evidenced by the higher prevalence of the CC+CT genotype in responsive patients (96%) versus non‐responsive patients (84.62%), with a significant distinction (OR = 5.53, 95%CI = 1.13–27.14, p = 0.035). Furthermore, at the allelic level, the mutant allele (T) was notably more prevalent in non‐responsive patients (36.46%) than in responsive patients (23%), with a statistically significant contrast (OR = 1.92, 95%CI = 1.03–3.58, p = 0.040), as shown in **Table** [5](#ggn210106-tbl-0005).

### Table 5.

| rs1044457 |  | Responsive [50] | Non‐responsive [48] | P‐value | OR [95%CI] |
| --- | --- | --- | --- | --- | --- |
| Genotypes | CC CT TT | 29(58%) 19 (38%) 2(4%) | 22(45.83%) 17(35.42%) 9(18.75%) | 0.100 0.706 0.032 | 1.0 1.18 (0.50–2.78) 5.93 (1.16–30.25) |
| Dominant model | CC+CT TT | 48(96%) 2(4%) | 39(84.62%) 9(18.75%) | 0.035 | 1.0 5.53 (1.13–27.14) |
| Recessive model | CC CT+TT | 29(58%) 21(42%) | 22(45.83%) 26(54.17%) | 0.229 | 1.0 1.63 (0.73–3.63) |
| Alleles | C T | 77(77%) 23(23%) | 61(63.54%) 35(36.46%) | 0.040 | 1.0 1.92 (1.03–3.58) |

Table 5 Caption: The frequency of different genotypes and alleles of the polymorphism rs1044457 in responsive and non‐responsive patients.

All responders exhibited some degree of tumour size reduction, whereas all non‐responders experienced tumour enlargement. Based on these observations, we categorized the patients according to their tumour size outcomes and analysed the correlation between these outcomes and the different genotypes and alleles of the rs1044457 polymorphism.

The distribution of genotypes was assessed in both responsive and non‐responsive groups. To calculate the Hardy–Weinberg equilibrium (HWE) the chi‐square test statistic was calculated to be 5.488 with 2 degrees of freedom (d.f.), resulting in a *p*‐value of 0.064. This *p*‐value indicates that the observed differences are not statistically significant at the conventional significance level of 0.05. Therefore, it can be concluded that there are no significant differences between the Hardy–Weinberg equilibrium (HWE) and the dataset under consideration. Consequently, our dataset complies with the Hardy–Weinberg equilibrium (HWE) based on the results of the statistical analysis.

## 4. Discussion

Gemcitabine is an important component in the treatment of advanced NSCLC. A meta‐analysis showed that gemcitabine‐based chemotherapy resulted in a median survival of 9 months, compared to 8.2 months with non‐gemcitabine combinations in advanced NSCLC patients.^[^ [17](#ggn210106-bib-0017) ^]^ This highlights the potential benefits of gemcitabine‐based regimens in NSCLC treatment.

The impact of genotypes present at the rs1044457 locus on drug pharmacokinetics has been emphasized in various studies.^[^ [7](#ggn210106-bib-0007), [8](#ggn210106-bib-0008) ^]^ Although there is evidence suggesting that CMPK1 polymorphisms affect chemotherapy outcomes, specific research focusing on the rs1044457 polymorphism in the context of metastatic non‐small cell lung cancer (NSCLC) is not directly available in the current literature.

This study identified three distinct polymorphisms CC, CT, and TT through genotype analysis. Here, CC represents the wild‐type genotype, while TT denotes the mutant form. These polymorphisms are observed in both the responsive and unresponsive groups. Furthermore, the distribution of these genotypes within each group adheres to the principles of Hardy–Weinberg equilibrium.

The most interesting finding in the current study was that the mutant genotype (TT) of CMPK1 rs1044457 gene polymorphism had a higher prevalence in non‐responsive patients (18.75%) compared to responsive patients (4%). This difference was found to be statistically significant (odds ratio = 5.93, 95% confidence interval = 1.16–30.25, p‐value = 0.032). However, there was no direct comparison between TT‐bearing non‐responsive and TT‐bearing responsive, but the categorization of groups into responsive and non‐responsive was almost entirely based on the tumour size reduction. Thus, there was an indirect comparison.

A study focusing on advanced NSCLC patients treated with gemcitabine or taxane/cisplatinum found that polymorphisms in CMPK1 were associated with differential survival outcomes. Specifically, the wild‐type genotypes of CMPK1 IVS1+1057 and IVS1‐928 were linked to shorter overall survival in patients treated with gemcitabine/cisplatinum, but not in those treated with taxane/cisplatinum. This indicates that CMPK1 polymorphisms may have a specific impact on gemcitabine‐based treatments in NSCLC patients.^[^ [18](#ggn210106-bib-0018) ^]^

A recent study investigated the role of CMPK1 polymorphisms in gemcitabine‐based chemotherapy for HER2‐negative metastatic breast cancer patients. The research found that the rs1044457 polymorphism in CMPK1 was strongly associated with progression‐free survival (PFS). Patients with the CC genotype showed a higher 6‐month PFS rate (78.9%) compared to those with CT and TT genotypes (55.9%). This suggests that the rs1044457 polymorphism may influence the response to gemcitabine‐based treatments, although in breast cancer rather than NSCLC.^[^ [4](#ggn210106-bib-0004) ^]^

CMPK1 plays a key role in the phosphorylation of nucleoside analogues like gemcitabine. Specifically, it converts gemcitabine monophosphate (dFdCMP) into gemcitabine diphosphate (dFdCDP), a critical step in the drug's activation. Polymorphisms in the CMPK1 gene, such as rs1044457, may potentially alter the enzyme's activity, expression, or stability. In patients carrying the T allele of this polymorphism, the conversion of gemcitabine monophosphate (inactive) to gemcitabine diphosphate (active) might be less efficient. This could result in lower intracellular levels of the active drug, potentially reducing its cytotoxic effects and leading to poorer therapeutic outcomes. Furthermore, this polymorphism may also affect the transcriptional regulation of CMPK1, resulting in lower expression levels of the enzyme. This, in turn, could similarly reduce gemcitabine activation and compromise its efficacy.

According to the results of the present study, age was negatively associated with a reduction in tumour size while there was a positive correlation between the reduction in tumour size and the number of treatment cycles. Our findings agreed with other studies performed on elderly patients with various malignancies showing that elderly patients benefit from chemotherapy to a similar extent as younger patients, with manageable side‐ effects.^[^ [19](#ggn210106-bib-0019), [20](#ggn210106-bib-0020) ^]^

The vast majority of these subgroup analyses demonstrated that the efficacy results are similar in patients aged <70 years old and in those aged ≥70 years.^[^ [21](#ggn210106-bib-0021), [22](#ggn210106-bib-0022) ^]^ In these trials, no difference in survival was observed according to age. The only study showing a shorter survival in elderly patients is the combined analysis of two Southwest Oncology Group trials. One trial compared cisplatin alone to cisplatin‐vinorelbine, and the other compared cisplatin‐vinorelbine to carboplatin‐paclitaxel.^[^ [23](#ggn210106-bib-0023) ^]^ A study conducted in China showed that age (<70 vs ≥70 years) did not affect treatment outcomes, but older adults were more likely to experience adverse events, highlighting the importance of patient selection to inform treatment decisions.^[^ [24](#ggn210106-bib-0024) ^]^

In the present study, a notable difference in tumour reduction was observed between females and males, favouring males, although the *p*‐value exceeded the threshold for statistical significance. Previous studies have shown that females tend to exhibit slower drug clearance for gemcitabine, resulting in higher systemic drug exposure and increased toxicity. In contrast, males typically metabolize and eliminate gemcitabine more efficiently, which may enhance its therapeutic effects while reducing side effects.^[^ [25](#ggn210106-bib-0025) ^]^ Additionally, females with NSCLC are more likely to have EGFR mutations, which respond better to targeted EGFR inhibitors rather than chemotherapy. On the other hand, males with NSCLC more frequently exhibit KRAS mutations or smoking‐related carcinogenesis, making them more reliant on chemotherapy than on targeted treatments.^[^ [26](#ggn210106-bib-0026) ^]^

The recommended number of treatment cycles for gemcitabine therapy in NSCLC ranges from four to six. The longer the duration of treatment, the greater the response. Chemotherapy selectively targets cells undergoing mitosis, the process of cell division. Resting cells are not susceptible to destruction. Therefore, with each successive cycle, a greater number of tumour cells are eradicated. However, with increased doses of gemcitabine, pulmonary toxicities have been reported such as bronchospasm, capillary leak syndrome, and non‐cardiogenic pulmonary edema.^[^ [27](#ggn210106-bib-0027) ^]^

Alternatively, a study conducted by Woo et al.^[^ [14](#ggn210106-bib-0014) ^]^ found that individuals with the CC genotype, diagnosed with pancreatic cancer and undergoing treatment with a combination of gemcitabine and erlotinib or fluoropyrimidines, experience improved overall survival rates and a longer time to tumour and disease progression compared to individuals with the TT genotype.

Taken together, these findings highlight the significance of the CMPK1 rs1044457 gene polymorphism in response to gemcitabine‐based chemotherapy. It is advisable to consider alternative chemotherapy options for NSCLC patients who carry the T allele of this polymorphism. The role of the rs1044457 polymorphism in gemcitabine treatment of NSCLC patients is an area of ongoing research, and while initial findings are promising, further studies are required to establish its specific impact in this context.

### 4.1. Limitations of the Study

The study's time constraints prevented an analysis of the relationship between the rs1044457 polymorphism in the CMPK1 gene and overall survival. Additionally, the relatively small cohort of 98 patients may have limited the study's statistical power and its generalizability to the broader NSCLC population. This small sample size also restricted the ability to perform multivariate analyses and adjust for potential confounding factors, such as comorbidities, lifestyle factors (e.g., smoking), and other genetic variations.

When considering genetic polymorphisms, such as rs1044457, in treatment responses, it's essential to account for the variability among different ethnic groups and subpopulations. This variability underscores the importance of personalized medicine tailored to genetic diversity.

## Conflict of Interest

The authors declare no conflict of interest.

## Author Contributions

G.S.S was responsible for the conceptualization, data curation, formal analysis, investigation, methodology, validation, visualization, and writing of the original draft, as well as reviewing and editing the manuscript. F.K.G contributed to the conceptualization, data curation, formal analysis, resources, supervision, validation, visualization, and writing of the original draft, along with reviewing and editing the manuscript. Q.S.A was involved in the conceptualization, data curation, formal analysis, methodology, resources, supervision, validation, visualization, and writing of the original draft, and also participated in the review and editing process. H.L took part in the conceptualization, data curation, formal analysis, methodology, project administration, software development, validation, visualization, and writing of the original draft, while also reviewing and editing the manuscript.

## Supporting information

Saleh G. S., Gatea F. K., Al‐Mayah Q. S., Lazim H., The Impact of the Rs1044457 Polymorphism in the CMPK1 Gene on the Response Rate to Gemcitabine‐Based Chemotherapy in Metastatic NSCLC Patients. Advanced Genetics 2025, 6, 2400058. 10.1002/ggn2.202400058

## Data Availability Statement

The data that support the findings of this study are available in the supplementary material of this article.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

The data that support the findings of this study are available in the supplementary material of this article.

### Supplementary Materials

### Data Availability Statement

The data that support the findings of this study are available in the supplementary material of this article.

## References

1. Smit E. F., van Meerbeeck J. P., Lianes P., Debruyne C., Legrand C., Schramel F., H. Smit, Gaafar R., Biesma B., Manegold C., Neymark N., J. Clin. Oncol. 2003, 21, 3909.  [DOI](https://doi.org/10.1200/JCO.2003.03.195) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14581415/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Oncol.&author=E.%20F.%20Smit&author=J.%20P.%20van%20Meerbeeck&author=P.%20Lianes&author=C.%20Debruyne&author=C.%20Legrand&volume=21&publication_year=2003&pages=3909&pmid=14581415&doi=10.1200/JCO.2003.03.195&)

2. Tsironis G., Ziogas D. C., Kyriazoglou A., Lykka M., Koutsoukos K., Bamias A., Dimopoulos M. A., Ann. Transl. Med. 2018, 6, 143.  [DOI](https://doi.org/10.21037/atm.2018.02.18) | [PMC free article](/articles/PMC5952019/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29862232/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann.%20Transl.%20Med.&author=G.%20Tsironis&author=D.%20C.%20Ziogas&author=A.%20Kyriazoglou&author=M.%20Lykka&author=K.%20Koutsoukos&volume=6&publication_year=2018&pages=143&pmid=29862232&doi=10.21037/atm.2018.02.18&)

3. Toschi L., Finocchiaro G., Bartolini S., Gioia V., Cappuzzo F., Future Oncol. 2005, 1, 7.  [DOI](https://doi.org/10.1517/14796694.1.1.7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16555971/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Future%20Oncol.&author=L.%20Toschi&author=G.%20Finocchiaro&author=S.%20Bartolini&author=V.%20Gioia&author=F.%20Cappuzzo&volume=1&publication_year=2005&pages=7&pmid=16555971&doi=10.1517/14796694.1.1.7&)

4. Cho E. H., Kim J. Y., Im S. A., Jung K. H., Sohn J., Lee K. S., Chae Y. S., Lee K. H., Kim J. H., Jang J. H., Ahn J. H., ESMO Open 2021, 6, 100236.  [DOI](https://doi.org/10.1016/j.esmoop.2021.100236) | [PMC free article](/articles/PMC8390551/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34438242/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=ESMO%20Open&author=E.%20H.%20Cho&author=J.%20Y.%20Kim&author=S.%20A.%20Im&author=K.%20H.%20Jung&author=J.%20Sohn&volume=6&publication_year=2021&pages=100236&pmid=34438242&doi=10.1016/j.esmoop.2021.100236&)

5. Ciccolini J., Serdjebi C., Peters G. J., Giovannetti E., Cancer Chemother. Pharmacol. 2016, 78, 1.  [DOI](https://doi.org/10.1007/s00280-016-3003-0) | [PMC free article](/articles/PMC4921117/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27007129/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Chemother.%20Pharmacol.&author=J.%20Ciccolini&author=C.%20Serdjebi&author=G.%20J.%20Peters&author=E.%20Giovannetti&volume=78&publication_year=2016&pages=1&pmid=27007129&doi=10.1007/s00280-016-3003-0&)

6. Alvarellos M. L., Lamba J., Sangkuhl K., Thorn C. F., Wang L., Klein D. J., Altman R. B., Klein T. E., Pharmacogenet. Genomics 2014, 24, 564.  [DOI](https://doi.org/10.1097/FPC.0000000000000086) | [PMC free article](/articles/PMC4189987/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25162786/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet.%20Genomics&author=M.%20L.%20Alvarellos&author=J.%20Lamba&author=K.%20Sangkuhl&author=C.%20F.%20Thorn&author=L.%20Wang&volume=24&publication_year=2014&pages=564&pmid=25162786&doi=10.1097/FPC.0000000000000086&)

7. Mitra A. K., Kirstein M. N., Khatri A., Skubitz K. M., Dudek A. Z., Greeno E. W., Kratzke R. A., Lamba J. K., Pharmacogenomics 2012, 13, 1009.  [DOI](https://doi.org/10.2217/pgs.12.81) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22838949/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&author=A.%20K.%20Mitra&author=M.%20N.%20Kirstein&author=A.%20Khatri&author=K.%20M.%20Skubitz&author=A.%20Z.%20Dudek&volume=13&publication_year=2012&pages=1009&pmid=22838949&doi=10.2217/pgs.12.81&)

8. Moore M. J., Science 2005, 309, 1514.  [DOI](https://doi.org/10.1126/science.1111443) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16141059/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&author=M.%20J.%20Moore&volume=309&publication_year=2005&pages=1514&pmid=16141059&doi=10.1126/science.1111443&)

9. Toschi L., Cappuzzo F., Onco Targets Ther 2009, 2, 209.  [DOI](https://doi.org/10.2147/ott.s4645) | [PMC free article](/articles/PMC2886326/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20616908/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Onco%20Targets%20Ther&author=L.%20Toschi&author=F.%20Cappuzzo&volume=2&publication_year=2009&pages=209&pmid=20616908&doi=10.2147/ott.s4645&)

10. Li L., Schaid D. J., Fridley B. L., Kalari K. R., Jenkins G. D., Abo R. P., Batzler A., Moon I., Pelleymounter L., Eckloff B. W., Wieben E. D., Sun Z., Yang P., Wang L., Pharmacogenet. Genomics 2012, 22, 105.  [DOI](https://doi.org/10.1097/FPC.0b013e32834dd7e2) | [PMC free article](/articles/PMC3259218/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22173087/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet.%20Genomics&author=L.%20Li&author=D.%20J.%20Schaid&author=B.%20L.%20Fridley&author=K.%20R.%20Kalari&author=G.%20D.%20Jenkins&volume=22&publication_year=2012&pages=105&pmid=22173087&doi=10.1097/FPC.0b013e32834dd7e2&)

11. Okazaki T., Javle M., Tanaka M., Abbruzzese J. L., Li D., Clin. Cancer Res. 2010, 16, 320.  [DOI](https://doi.org/10.1158/1078-0432.CCR-09-1555) | [PMC free article](/articles/PMC2802655/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20028759/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Cancer%20Res.&author=T.%20Okazaki&author=M.%20Javle&author=M.%20Tanaka&author=J.%20L.%20Abbruzzese&author=D.%20Li&volume=16&publication_year=2010&pages=320&pmid=20028759&doi=10.1158/1078-0432.CCR-09-1555&)

12. Soo R. A., Wang L. Z., Ng S. S., Chong P. Y., Yong W. P., Lee S. C., Liu J. J., Choo T. B., Tham L. S., Lee H. S., Goh B. C., Soong R., Lung Cancer 2009, 63, 121.  [DOI](https://doi.org/10.1016/j.lungcan.2008.04.010) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18538445/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lung%20Cancer&author=R.%20A.%20Soo&author=L.%20Z.%20Wang&author=S.%20S.%20Ng&author=P.%20Y.%20Chong&author=W.%20P.%20Yong&volume=63&publication_year=2009&pages=121&pmid=18538445&doi=10.1016/j.lungcan.2008.04.010&)

13. Tanaka M., Javle M., Dong X., Eng C., Abbruzzese J. L., Li D., Cancer 2010, 116, 5325.  [DOI](https://doi.org/10.1002/cncr.25282) | [PMC free article](/articles/PMC2966859/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20665488/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer&author=M.%20Tanaka&author=M.%20Javle&author=X.%20Dong&author=C.%20Eng&author=J.%20L.%20Abbruzzese&volume=116&publication_year=2010&pages=5325&pmid=20665488&doi=10.1002/cncr.25282&)

14. Woo H. I., Kim K.‐K., Choi H., Kim S., Jang K.‐T., Yi J. H., Park Y. S., Park J. O., Lee S.‐Y., Pharmacogenomics 2012, 13, 1023.  [DOI](https://doi.org/10.2217/pgs.12.82) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22838950/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&author=H.%20I.%20Woo&author=K.%E2%80%90K.%20Kim&author=H.%20Choi&author=S.%20Kim&author=K.%E2%80%90T.%20Jang&volume=13&publication_year=2012&pages=1023&pmid=22838950&doi=10.2217/pgs.12.82&)

15. Lee S.‐Y., Im S.‐A., Park Y. H., Woo S. Y., Kim S., Choi M. K., Chang W., Ahn J. S., Im Y.‐H., Eur J Cancer 2014, 50, 698.  [DOI](https://doi.org/10.1016/j.ejca.2013.11.028) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24361227/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Cancer&author=S.%E2%80%90Y.%20Lee&author=S.%E2%80%90A.%20Im&author=Y.%20H.%20Park&author=S.%20Y.%20Woo&author=S.%20Kim&volume=50&publication_year=2014&pages=698&pmid=24361227&doi=10.1016/j.ejca.2013.11.028&)

16. Eisenhauer E. A., Therasse P., Bogaerts J., Schwartz L. H., Sargent D., Ford R., Dancey J., Arbuck S., Gwyther S., Mooney M., Rubinstein L., Shankar L., Dodd L., Kaplan R., Lacombe D., Verweij J., Eur J Cancer 2009, 45, 228.  [DOI](https://doi.org/10.1016/j.ejca.2008.10.026) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19097774/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Cancer&author=E.%20A.%20Eisenhauer&author=P.%20Therasse&author=J.%20Bogaerts&author=L.%20H.%20Schwartz&author=D.%20Sargent&volume=45&publication_year=2009&pages=228&pmid=19097774&doi=10.1016/j.ejca.2008.10.026&)

17. Dong S., Guo A.‐L., Chen Z.‐H., Wang Z., Zhang X.‐C., Huang Y., Xie Z., Yan H.‐H., Cheng H., Wu Y.‐L., J. Hematol. Oncol. 2010, 3, 10.  [DOI](https://doi.org/10.1186/1756-8722-3-10) | [PMC free article](/articles/PMC2855513/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20226083/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Hematol.%20Oncol.&author=S.%20Dong&author=A.%E2%80%90L.%20Guo&author=Z.%E2%80%90H.%20Chen&author=Z.%20Wang&author=X.%E2%80%90C.%20Zhang&volume=3&publication_year=2010&pages=10&pmid=20226083&doi=10.1186/1756-8722-3-10&)

18. Ryu J.‐S., Shin E.‐S., Nam H.‐S., Yi H.‐G., Cho J.‐H., Kim C.‐S., Kim H.‐J., Lee J.‐E., J Thorac Oncol 2011, 6, 1320.  [DOI](https://doi.org/10.1097/JTO.0b013e3182208e26) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21642870/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Thorac%20Oncol&author=J.%E2%80%90S.%20Ryu&author=E.%E2%80%90S.%20Shin&author=H.%E2%80%90S.%20Nam&author=H.%E2%80%90G.%20Yi&author=J.%E2%80%90H.%20Cho&volume=6&publication_year=2011&pages=1320&pmid=21642870&doi=10.1097/JTO.0b013e3182208e26&)

19. Langer C. J., J Natl Cancer Inst 2002, 94, 173.  [DOI](https://doi.org/10.1093/jnci/94.3.173) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11830607/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Natl%20Cancer%20Inst&author=C.%20J.%20Langer&volume=94&publication_year=2002&pages=173&pmid=11830607&doi=10.1093/jnci/94.3.173&)

20. Langer C. J., Vangel M., Schiller J., Harrington D. R., Sandier A., Belani C., Johnson D., Lung Cancer 2003, 41, S17.14565510  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lung%20Cancer&author=C.%20J.%20Langer&author=M.%20Vangel&author=J.%20Schiller&author=D.%20R.%20Harrington&author=A.%20Sandier&volume=41&publication_year=2003&pages=S17&)

21. Ansari R. H., Socinski M. A., Edelman M. J., Belani C. P., Gonin R., Catalano R. B., Marinucci D. M., Comis R. L., Obasaju C. K., Chen R., Monberg M. J., Treat J., Crit. Rev. Oncol. Hematol. 2011, 78, 162.  [DOI](https://doi.org/10.1016/j.critrevonc.2010.03.003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20413322/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Crit.%20Rev.%20Oncol.%20Hematol.&author=R.%20H.%20Ansari&author=M.%20A.%20Socinski&author=M.%20J.%20Edelman&author=C.%20P.%20Belani&author=R.%20Gonin&volume=78&publication_year=2011&pages=162&pmid=20413322&doi=10.1016/j.critrevonc.2010.03.003&)

22. Lilenbaum R. C., Herndon J. E., List M. A., Desch C., Watson D. M., Miller A. A., Graziano S. L., Perry M. C., Saville W., Chahinian P., Weeks J. C., Holland J. C., Green M. R., J. Clin. Oncol. 2005, 23, 190.  [DOI](https://doi.org/10.1200/JCO.2005.07.172) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15625373/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Oncol.&author=R.%20C.%20Lilenbaum&author=J.%20E.%20Herndon&author=M.%20A.%20List&author=C.%20Desch&author=D.%20M.%20Watson&volume=23&publication_year=2005&pages=190&pmid=15625373&doi=10.1200/JCO.2005.07.172&)

23. Kelly K., Crowley J., Bunn P. A., Presant C. A., Grevstad P. K., Moinpour C. M., Ramsey S. D., Wozniak A. J., Weiss G. R., Moore D. F., Israel V. K., Livingston R. B., Gandara D. R., J. Clin. Oncol. 2001, 19, 3210.  [DOI](https://doi.org/10.1200/JCO.2001.19.13.3210) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11432888/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Oncol.&author=K.%20Kelly&author=J.%20Crowley&author=P.%20A.%20Bunn&author=C.%20A.%20Presant&author=P.%20K.%20Grevstad&volume=19&publication_year=2001&pages=3210&pmid=11432888&doi=10.1200/JCO.2001.19.13.3210&)

24. Li X., Huang D.‐B., Zhang Q., Guo C.‐X., Fu Q.‐H., Zhang X.‐C., Tang T.‐Y., Su W., Chen Y.‐W., Chen W., Ma T., Gao S.‐L., Que R.‐S., Bai X.‐L., Liang T.‐B., Pancreatology 2020, 20, 95.  [DOI](https://doi.org/10.1016/j.pan.2019.11.012) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31786057/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pancreatology&author=X.%20Li&author=D.%E2%80%90B.%20Huang&author=Q.%20Zhang&author=C.%E2%80%90X.%20Guo&author=Q.%E2%80%90H.%20Fu&volume=20&publication_year=2020&pages=95&pmid=31786057&doi=10.1016/j.pan.2019.11.012&)

25. Soldin O. P., Mattison D. R., Clin. Pharmacokinet. 2009, 48, 143.  [DOI](https://doi.org/10.2165/00003088-200948030-00001) | [PMC free article](/articles/PMC3644551/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19385708/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacokinet.&author=O.%20P.%20Soldin&author=D.%20R.%20Mattison&volume=48&publication_year=2009&pages=143&pmid=19385708&doi=10.2165/00003088-200948030-00001&)

26. Bell D. W., Brannigan B. W., Matsuo K., Finkelstein D. M., Sordella R., Settleman J., Mitsudomi T., Haber D. A., Clin. Cancer Res. 2008, 14, 4079.  [DOI](https://doi.org/10.1158/1078-0432.CCR-07-5030) | [PMC free article](/articles/PMC3391698/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18593984/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Cancer%20Res.&author=D.%20W.%20Bell&author=B.%20W.%20Brannigan&author=K.%20Matsuo&author=D.%20M.%20Finkelstein&author=R.%20Sordella&volume=14&publication_year=2008&pages=4079&pmid=18593984&doi=10.1158/1078-0432.CCR-07-5030&)

27. Turco C., Jary M., Kim S., Moltenis M., Degano B., Manzoni P., Nguyen T., Genet B., Rabier M.‐B. V., Heyd B., Borg C., Clin Med Insights Oncol 2015, 9, 75.  [DOI](https://doi.org/10.4137/CMO.S26537) | [PMC free article](/articles/PMC4559186/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26380562/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Med%20Insights%20Oncol&author=C.%20Turco&author=M.%20Jary&author=S.%20Kim&author=M.%20Moltenis&author=B.%20Degano&volume=9&publication_year=2015&pages=75&pmid=26380562&doi=10.4137/CMO.S26537&)
