# Genome-Wide Association Study Identifies Novel Candidate Variants Associated with Postoperative Nausea and Vomiting

## Metadata
**Authors:** Daisuke Nishizawa, Ryozo Morino, Rie Inoue, Seii Ohka, Shinya Kasai, Junko Hasegawa, Yuko Ebata, Kyoko Nakayama, Hiroyuki Sumikura, Masakazu Hayashida, Miyuki Yokota, Kazutaka Ikeda
**Journal:** Cancers
**Date:** 2023 Sep 26
**DOI:** [10.3390/cancers15194729](https://doi.org/10.3390/cancers15194729)
**PMID:** 37835423
**PMCID:** PMC10571790
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10571790/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC10571790/pdf/cancers-15-04729.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC10571790/pdf/cancers-15-04729.pdf)

## Abstract

**Simple Summary:** 
Postoperative nausea and vomiting (PONV) is experienced by approximately 30% of patients who undergo general anesthesia. However, many genetic factors involved in the vulnerability to PONV remain unidentified. The aim of our genome-wide association study (GWAS) was to comprehensively explore genetic variations associated with PONV. We identified several single-nucleotide polymorphisms (SNPs) that may possibly be associated with the frequency of nausea and vomiting, of which the most potent were the rs2776262, rs140703637, rs7212072, rs12444143, rs45574836, and rs1752136 SNPs. These results indicate that these SNPs in the LOC100506403, CNTN5, SHISA6, RBFOX1, ATP8B3, and LOC105370198 gene regions could serve as markers that predict the vulnerability to PONV.

**Abstract:** 
Considerable individual differences are widely observed in the incidence of postoperative nausea and vomiting (PONV). We conducted a genome-wide association study (GWAS) to identify potential candidate single-nucleotide polymorphisms (SNPs) that contribute to PONV by utilizing whole-genome genotyping arrays with more than 950,000 markers. The subjects were 806 patients who provided written informed consent and underwent elective surgery under general anesthesia with propofol or desflurane. The GWAS showed that two SNPs, rs2776262 and rs140703637, in the LOC100506403 and CNTN5 gene regions, respectively, were significantly associated with the frequency of nausea. In another GWAS conducted only on patients who received propofol, rs7212072 and rs12444143 SNPs in the SHISA6 and RBFOX1 gene regions, respectively, were significantly associated with the frequency of nausea as well as the rs2776262 SNP, and the rs45574836 and rs1752136 SNPs in the ATP8B3 and LOC105370198 gene regions, respectively, were significantly associated with vomiting. Among these SNPs, clinical and SNP data were available for the rs45574836 SNP in independent subjects who underwent laparoscopic gynecological surgery, and the association was replicated in these subjects. These results indicate that these SNPs could serve as markers that predict the vulnerability to PONV. Our findings may provide valuable information for achieving satisfactory prophylactic treatment for PONV.

Keywords: postoperative nausea and vomiting, anesthesia, propofol, single-nucleotide polymorphisms, genome-wide association study

### Simple Summary

Postoperative nausea and vomiting (PONV) is experienced by approximately 30% of patients who undergo general anesthesia. However, many genetic factors involved in the vulnerability to PONV remain unidentified. The aim of our genome-wide association study (GWAS) was to comprehensively explore genetic variations associated with PONV. We identified several single-nucleotide polymorphisms (SNPs) that may possibly be associated with the frequency of nausea and vomiting, of which the most potent were the rs2776262, rs140703637, rs7212072, rs12444143, rs45574836, and rs1752136 SNPs. These results indicate that these SNPs in the *LOC100506403*, *CNTN5*, *SHISA6*, *RBFOX1*, *ATP8B3*, and *LOC105370198* gene regions could serve as markers that predict the vulnerability to PONV.

### Abstract

Considerable individual differences are widely observed in the incidence of postoperative nausea and vomiting (PONV). We conducted a genome-wide association study (GWAS) to identify potential candidate single-nucleotide polymorphisms (SNPs) that contribute to PONV by utilizing whole-genome genotyping arrays with more than 950,000 markers. The subjects were 806 patients who provided written informed consent and underwent elective surgery under general anesthesia with propofol or desflurane. The GWAS showed that two SNPs, rs2776262 and rs140703637, in the *LOC100506403* and *CNTN5* gene regions, respectively, were significantly associated with the frequency of nausea. In another GWAS conducted only on patients who received propofol, rs7212072 and rs12444143 SNPs in the *SHISA6* and *RBFOX1* gene regions, respectively, were significantly associated with the frequency of nausea as well as the rs2776262 SNP, and the rs45574836 and rs1752136 SNPs in the *ATP8B3* and *LOC105370198* gene regions, respectively, were significantly associated with vomiting. Among these SNPs, clinical and SNP data were available for the rs45574836 SNP in independent subjects who underwent laparoscopic gynecological surgery, and the association was replicated in these subjects. These results indicate that these SNPs could serve as markers that predict the vulnerability to PONV. Our findings may provide valuable information for achieving satisfactory prophylactic treatment for PONV.

## 1. Introduction

General anesthesia is commonly utilized in surgeries for the treatment of cancer and other diseases. Postoperative nausea and vomiting (PONV) is the most common adverse event following general anesthesia [[1](#B1-cancers-15-04729)], with an estimated incidence of 30% in the general surgical population and as high as 80% in high-risk cohorts [[1](#B1-cancers-15-04729),[2](#B2-cancers-15-04729),[3](#B3-cancers-15-04729)]. The development of PONV is associated with significantly lower postoperative quality of life (QOL) [[1](#B1-cancers-15-04729),[4](#B4-cancers-15-04729)]. Unresolved PONV may result in prolonged post-anesthesia stays in the care unit or hospital that can significantly increase overall healthcare costs [[3](#B3-cancers-15-04729)]. PONV is thought to be multifactorial, involving anesthetic, surgical, and individual risk factors [[2](#B2-cancers-15-04729),[5](#B5-cancers-15-04729),[6](#B6-cancers-15-04729),[7](#B7-cancers-15-04729)]. Female gender, a history of PONV, non-smoking status, a history of motion sickness, and younger age are patient-specific predictors, and the use of volatile anesthetics, the duration of anesthesia, postoperative opioid use, and nitrous oxide have been reported to be anesthesia-related predictors. Cholecystectomy, gynecological surgery, and laparoscopic procedures have been shown to be surgical risk factors for PONV [[8](#B8-cancers-15-04729),[9](#B9-cancers-15-04729),[10](#B10-cancers-15-04729)]. Among these, four major factors—female gender, a history of PONV and/or motion sickness, non-smoking status, and the use of postoperative opioids—were incorporated into a simplified risk score to predict PONV that was developed by Apfel et al. (1999) [[2](#B2-cancers-15-04729)]. However, even patients at low PONV risk according to their assigned Apfel score may experience PONV, suggesting a genetic predisposition [[11](#B11-cancers-15-04729)].

Previous genetic studies of candidate molecules that are related to nausea/vomiting mechanisms identified human genetic variants associated with PONV-related phenotypes in genes that encode serotonin receptor type 3, dopamine receptor type 2, μ-opioid receptor, neurokinin 1 receptor, serotonin transporter, adenosine triphosphate (ATP)-binding cassette subfamily B member 1 transporter, organic cation transporter (OCT), and cytochrome P450 2D6 isoform, among others [[11](#B11-cancers-15-04729),[12](#B12-cancers-15-04729),[13](#B13-cancers-15-04729),[14](#B14-cancers-15-04729)]. To date, genetic polymorphisms within and around several genes, including *HTR3A* [[15](#B15-cancers-15-04729),[16](#B16-cancers-15-04729)], *HTR3B* [[16](#B16-cancers-15-04729),[17](#B17-cancers-15-04729),[18](#B18-cancers-15-04729)], *DRD2*/*ANKK1* [[19](#B19-cancers-15-04729),[20](#B20-cancers-15-04729)], *OPRM1* [[21](#B21-cancers-15-04729),[22](#B22-cancers-15-04729)], *TACR1* [[23](#B23-cancers-15-04729)], *SLC6A4* [[24](#B24-cancers-15-04729)], *ABCB1* [[25](#B25-cancers-15-04729),[26](#B26-cancers-15-04729),[27](#B27-cancers-15-04729)], *OCT1* (*SLC22A1*) [[28](#B28-cancers-15-04729)], and *CYP2D6* [[29](#B29-cancers-15-04729),[30](#B30-cancers-15-04729),[31](#B31-cancers-15-04729)], have been found to be associated with PONV-related phenotypes. PONV-related single-nucleotide polymorphisms (SNPs) have also been comprehensively explored based on recent advances in high-density SNP arrays that can screen hundreds of thousands or millions of genetic markers throughout the human genome. For example, Janicki et al. (2011) found that one SNP in the *CHRM3* gene, rs2165870, which encodes muscarinic acetylcholine receptor 3, was associated with PONV in a genome-wide association study (GWAS) using pooled deoxyribonucleic acid (DNA) and a separate verification study [[32](#B32-cancers-15-04729)]. Klenke et al. (2018) confirmed the association between the rs2165870 SNP and PONV [[33](#B33-cancers-15-04729)] and identified another SNP, rs349358, in the gene that encodes potassium voltage-gated channel subfamily B member 2 (*KCNB2*), which was significantly associated with PONV [[14](#B14-cancers-15-04729)]. Another GWAS was reported in female subjects who had a higher risk of PONV and underwent breast cancer surgery with standardized propofol anesthesia and antiemetics, in which six variants with a suggestive association with PONV (*p* < 1 × 10^−5^) were identified, and the association with the *DRD*2 variant rs1800497 (TaqIA) in previous studies [[19](#B19-cancers-15-04729),[20](#B20-cancers-15-04729)] was replicated [[34](#B34-cancers-15-04729)]. Although the aforementioned GWASs were all conducted in subjects of European origin, a GWAS in subjects of Asian origin was also recently reported [[35](#B35-cancers-15-04729)]. Sugino et al. (2020) performed a GWAS using a DNA microarray that was optimized for genotyping in Japanese populations and identified 78 SNPs that were associated with the incidence of PONV in a limited sample of 24 female patients (*p* < 1 × 10^−4^). Among these, associations of the two candidate SNPs, the rs1333114 SNP of the *PTPRD* gene and the rs11232965 SNP of the *MIR4300HG* gene, were verified in independent samples [[35](#B35-cancers-15-04729)]. However, GWASs with relatively large sample sizes have not been conducted in Asian populations to date, and many genetic factors that contribute to PONV remain unknown.

In the present study, we conducted a GWAS on subjects who were scheduled to undergo general anesthesia by total intravenous anesthesia (TIVA) with propofol or inhalational anesthesia with desflurane to identify potential genetic variants contributing to the vulnerability to PONV. Considering that the entire population was a mixture of subjects who underwent general anesthesia with propofol and desflurane, another GWAS was also performed only in subjects who underwent general anesthesia with propofol to explore the genetic factors associated with PONV.

## 2. Materials and Methods

### 2.1. Patients

#### 2.1.1. Patients Who Underwent Elective Surgery under General Anesthesia with Propofol or Desflurane

Enrolled in the study were 806 adult patients (20–93 years old, 432 males and 374 females) who were scheduled to undergo elective surgery for cancer under general anesthesia by TIVA with propofol or inhalational anesthesia with desflurane at The Cancer Institute Hospital of the Japanese Foundation for Cancer Research (JFCR; CIH samples). The exclusion criteria were the following: (1) patients to whom mild or more emetogenic antitumor agents were administered or were scheduled to be administered from 6 days before the start of the study to 48 h after surgery; (2) patients with symptomatic brain metastases; (3) patients who used the following antiemetic drugs within 48 h before and during surgery: 5-hydroxytryptamine 3 (5-HT_3_) receptor antagonists (granisetron, ondansetron, azasetron, etc.), phenothiazines (chlorpromazine, prochlorperazine, perphenazine, etc.), butyrophenone-based preparations (haloperidol, droperidol, etc.), benzamide preparations (sulpiride, tiapride, sultopride, etc.), dopamine receptor antagonists (metoclopramide, itopride, domperidone, etc.), antihistamines (hydroxyzine, dimenhydrinate, diphenhydramine), or NK1 receptor antagonists (apireptant); (4) patients who were mentally unable to communicate; (5) patients who were pregnant; (6) patients who were judged to be inappropriate for inclusion in the study by the investigator; and (7) patients of the Head and Neck Department and Gastroenterology Department who needed advanced management in the postoperative intensive care unit. The major reasons for applying these exclusion criteria were the possible influence of these factors on the incidence and severity of PONV and the collection of accurate data. The cancellation criteria were the following: (1) patients for whom blood collection was not possible and (2) patients whose informed consent was withdrawn. All of the individuals who were included in the study were of Japanese origin. Peripheral blood samples were collected from these subjects for gene analysis. Detailed demographic and clinical data of the subjects are provided in [Table 1](#cancers-15-04729-t001).

| Demographic Data: | n | Minimum | Maximum | Mean | SD | Median |
| --- | --- | --- | --- | --- | --- | --- |
| Gender |   |   |   |   |   |   |
| male | 432 |   |   |   |   |   |
| female | 374 |   |   |   |   |   |
| Age [years] | 806 | 23 | 93 | 58.80 | 13.25 | 60.00 |
| Height [cm] | 806 | 140.8 | 184.9 | 162.39 | 8.16 | 161.90 |
| Weight [kg] | 806 | 31.5 | 109.1 | 59.01 | 11.15 | 57.65 |
| Body mass index (BMI) [kg/m2] | 806 | 14.46 | 39.05 | 22.29 | 3.39 | 22.02 |
| History of smoking | 806 |   |   |   |   |   |
| absence | 453 |   |   |   |   |   |
| presence | 353 |   |   |   |   |   |
| Frequency of alcohol drinking per week | 806 | 0 | 7 | 2.65 | 7.00 | 1.00 |
| History of motion sickness | 806 |   |   |   |   |   |
| absence | 495 |   |   |   |   |   |
| presence | 311 |   |   |   |   |   |
| History of PONV | 806 |   |   |   |   |   |
| absence | 722 |   |   |   |   |   |
| presence | 84 |   |   |   |   |   |
| Surgery and clinical data for postoperative period: | n | Minimum | Maximum | Mean | SD | Median |
| Duration of anaesthesia [min] | 806 | 30 | 1050 | 258.62 | 1050.00 | 214.50 |
| Duration of surgery [min] | 806 | 4 | 977 | 203.34 | 977.00 | 163.00 |
| Type of anesthesia | 806 |   |   |   |   |   |
| TIVA (propofol) | 442 |   |   |   |   |   |
| inhalational anesthesia (desflurane) | 364 |   |   |   |   |   |
| Total dose of remifentanil [μg] | 806 | 0 | 24,300 | 3135.61 | 24,300.00 | 2500.00 |
| Total dose of fentanyl [μg] | 806 | 0 | 800 | 198.26 | 800.00 | 200.00 |
| Postoperative administration of narcotic drugs | 806 |   |   |   |   |   |
| absence | 427 |   |   |   |   |   |
| presence | 379 |   |   |   |   |   |
| Postoperative administration of opioids including pentazocine | 806 |   |   |   |   |   |
| absence | 340 |   |   |   |   |   |
| presence | 466 |   |   |   |   |   |
| Experience of pain | 806 |   |   |   |   |   |
| absence | 250 |   |   |   |   |   |
| presence | 556 |   |   |   |   |   |
| Frequency of pain | 806 | 0 | 15 | 2.90 | 15.00 | 2.00 |
| PONV: | n | Minimum | Maximum | Mean | SD | Median |
| Nausea | 806 |   |   |   |   |   |
| absence | 541 |   |   |   |   |   |
| presence | 265 |   |   |   |   |   |
| Frequency of nausea | 806 | 0 | 10 | 0.83 | 10.00 | 0.00 |
| Vomiting | 806 |   |   |   |   |   |
| absence | 657 |   |   |   |   |   |
| presence | 149 |   |   |   |   |   |
| PONV | 806 |   |   |   |   |   |
| absence | 526 |   |   |   |   |   |
| presence | 280 |   |   |   |   |   |

Table Caption: Demographic and clinical data of patient subjects for the GWAS.

The study was conducted according to the guidelines of the Declaration of Helsinki and approved by the Institutional Review Board or Ethics Committee of The Cancer Institute Hospital and Tokyo Metropolitan Institute of Medical Science (Tokyo, Japan). Written informed consent was obtained from all of the patients.

#### 2.1.2. Patients Who Underwent Laparoscopic Gynecological Surgery

Enrolled in the study were 350 females (20–70 years old) who were classified as American Society of Anesthesiologists (ASA) Physical Status (ASA-PS) Class I or II and were scheduled to undergo laparoscopic gynecological surgery (LGS) under general anesthesia for benign gynecological diseases (e.g., uterine myoma and ovarian cysts) at Juntendo University Hospital between June 2017 and May 2019 (JUH samples). Excluded were patients who chronically received antipsychotic drugs, antiepileptic drugs, or opioid analgesics; patients with obstructive sleep apnea syndrome; and patients whose body mass index (BMI) was >30 kg/m^2^. Additionally, patients for whom surgery was converted from LGS to open abdominal surgery and patients who underwent re-operation for hemorrhage were excluded. The major reasons for applying these exclusion criteria were the possible influence of these factors on the incidence and severity of PONV and the collection of accurate data.

The study was conducted according to the guidelines of the Declaration of Helsinki and was approved by the Institutional Review Board or Ethics Committee of Juntendo University School of Medicine and Tokyo Metropolitan Institute of Medical Science (Tokyo, Japan). Written informed consent was obtained from all of the patients.

### 2.2. Patient Characteristics and Clinical Data

#### 2.2.1. Patient Characteristics and Clinical Data in Patients Who Underwent Elective Surgery under General Anesthesia with Propofol or Desflurane

In the CIH samples, we obtained data on patient characteristics (gender, age, height, weight, and BMI), history of smoking, frequency of alcohol drinking per week, history of motion sickness, history of PONV, surgery data, clinical data for the postoperative period (duration of anesthesia, duration of surgery, type of anesthesia, total dose of remifentanil, total dose of fentanyl, postoperative administration of narcotic drugs, and postoperative administration of opioids (including pentazocine)), the experience and frequency of postoperative pain, and PONV data ([Table 1](#cancers-15-04729-t001)).

For PONV evaluation, the evaluator recorded the number of nausea and vomiting episodes that occurred in the postoperative period. Vomiting was defined as episodes of vomiting and/or retching that occurred once or more (the act of excreting the contents of the stomach) or dry vomiting (the act of trying to vomit without excreting the contents of the stomach). Vomiting that occurred as separate events was defined as the absence of vomiting for at least 1 min between two events. The investigator thoroughly explained the definition of vomiting to the subject.

The presence or absence of nausea, frequency of nausea, presence or absence of vomiting, and presence or absence of PONV (the presence or absence of the incidence of nausea and/or vomiting) were used as endpoints for the genetic association analysis in the present study. Despite possible correlations among the four major endpoint variables, GWASs were performed for all four of these phenotypes in case even slight differences in these endpoint values could be caused by some slightly or moderately different genetic variants. The frequency of vomiting was not analyzed because vomiting occurred only once in most of the subjects. The clinical data on the subjects are detailed in [Table 1](#cancers-15-04729-t001). Although several kinds of opioids were administered during surgery and the postoperative period, opioid narcotic administration was not standardized in the present study, because opioid administration during surgery and the postoperative period were treated as different variates in the clinical data analyses.

#### 2.2.2. Patient Characteristics and Clinical Data in Patients Who Underwent Laparoscopic Gynecological Surgery

For the JUH samples, patient characteristics, surgical protocols, and postoperative pain management are detailed in another report by Inoue et al. (in preparation). Briefly, anesthesia was induced with remifentanil at a rate of 0.5 μg/kg/min and the target-controlled infusion (TCI) of propofol at a target concentration of 3–5 μg/mL using a TCI pump (TE-371, Terumo, Tokyo, Japan). Dexamethasone (6.6 mg) and droperidol (1.25 mg) were administered intravenously (i.v.) to prevent PONV. Around the end of surgery, infusions of propofol and remifentanil were discontinued. Fentanyl (approximately 4 μg/kg) and acetaminophen (20 mg/kg, up to 1000 mg) were administered i.v. to achieve immediate postoperative pain relief. When patients complained of significant pain, fentanyl (50–100 μg) was given in increments. After adequate immediate postoperative pain relief was achieved, postoperative pain was managed with i.v. fentanyl patient-controlled analgesia (PCA) combined with droperidol (fentanyl (1000 μg in 20 mL) and droperidol (2.5 mg in 1 mL) diluted with normal saline (80 mL) to a total volume of 101 mL) that commenced using a CADD-Legacy PCA pump (Smiths Medical Japan, Tokyo, Japan). Additionally, acetaminophen (20 mg/kg, up to 1000 mg) was administered i.v. every 6 h until it became unnecessary during the first 24 h postoperative period. When the analgesia that was achieved with i.v. fentanyl PCA combined with repeated doses of acetaminophen was inadequate, i.v. flurbiprofen axetil (50 mg) or i.v. pentazocine (30 mg) were given as rescue analgesics. The presence or absence of PONV was assessed at the same time as pain intensities, every 3 h postoperatively or whenever patients complained of PONV. When required, i.v. metoclopramide (10 mg) or rectal domperidone (60 mg) were used to treat PONV. The number of patients who experienced PONV or other adverse effects of fentanyl within the 24 h postoperative period was noted. The characteristics of these clinical data are summarized in [Supplementary Table S1.](#app1-cancers-15-04729)

### 2.3. Whole-Genome Genotyping and Quality Control

For the CIH samples, 10 mL of venous blood was sampled during anesthesia for the subsequent preparation of genomic DNA specimens. The DNA concentration was adjusted to 100 ng/L for whole-genome genotyping using a NanoDrop ND-1000 Spectrophotometer (NanoDrop Technologies, Wilmington, DE, USA). Whole-genome genotyping was performed using Infinium Assay II with an iScan system (Illumina, San Diego, CA, USA), according to the manufacturer’s instructions. Three kinds of BeadChips were used to genotype 806 samples: HumanOmniExpressExome-8 v. 1.2 (total markers: 964,193), HumanOmniExpressExome-8 v. 1.3 (total markers: 958,497), and HumanOmniExpressExome-8 v. 1.4 (total markers: 960,919). The BeadChips included a number of probes that were specific to copy number variation markers, but most of the BeadChips were for SNP markers on the human autosomes or sex chromosomes. Approximately 946,000 SNP markers were commonly included in all of the BeadChips.

For the JUH samples, 10 mL of venous blood was also sampled to prepare DNA specimens. According to the manufacturer’s recommendations, whole-genome genotyping was performed using Infinium Assay II with an iScan system (Illumina). Infinium Asian Screening Array-24 v. 1.0 BeadChips (one kind) were utilized to genotype 333 patient samples (total markers: 659,184). Numerous copy number variation markers were included in the BeadChips, but the majority of the BeadChips were for SNP markers on the human autosomes or sex chromosomes.

GenomeStudio v. 2.0.4 with the Genotyping v. 2.0.4 module (Illumina) was used to examine the data for samples with their entire genomes genotyped to assess the quality of the findings. Following data cleaning, quality control was performed as for the CIH samples. The patient samples retained a total of 651,086 SNP markers after this screening step. Among these SNPs, the genotype data for a potent SNP found in the GWAS, rs45574836 (exm1401859), were extracted and used for a further replication study.

In the CIH samples, for phenotypes of the frequency of nausea in all patients, the frequency of nausea in patients who received propofol, and the presence/absence of vomiting in patients who received propofol, log QQ *p*-value plots were subsequently drawn as a result of the GWAS for the 806 samples to check the pattern of the generated *p*-value distribution, in which the observed *p*-values against the values that were expected from the null hypothesis of a uniform distribution, calculated as −log10 (*p* value), were plotted for each model. All of the plots were mostly concordant with the expected line (y = x), especially over the range of 0 < −log10 (*p* value) < 2.5 for each model in the frequency of nausea in patients who received propofol and vomiting in patients who received propofol, and over the range of 0 < −log10 (*p* value) < 4.5 for each model in vomiting in patients who received propofol, indicating no apparent population stratification of the samples that were used in the study ([Supplementary Figures S1–S3](#app1-cancers-15-04729)). The Functional Mapping and Annotation of Genome-Wide Association Studies (FUMA GWAS) v. 1.3.9 platform was used to visualize the QQ plots [[36](#B36-cancers-15-04729)].

### 2.4. Statistical Analysis

In the GWAS for CIH samples, the presence/absence of nausea, frequency of nausea, presence/absence of vomiting, and presence/absence of PONV (the presence/absence of nausea or vomiting) were used as indices of the vulnerability or severity of PONV during the 48 h postoperative period. Before the analyses, quantitative values of the frequency of nausea, represented as integer numbers of the incidence of nausea during the 48 h postoperative period, were modified based on the procedure of variable transformation that was developed by Yeo and Johnson (2000), with the lambda value set as 0.5 for approximation to the normal distribution [[37](#B37-cancers-15-04729)]. To explore the associations between SNPs and the incidence of PONV, Fisher’s exact test or the Cochran–Armitage trend test were conducted in analyses using both the group of all patients and that of patients who received propofol. This was carried out to compare genotype data between subjects with the presence of incidence and subjects with the absence of incidence. To explore associations between the SNPs and the frequency of nausea in analyses using both the group of all patients and that of patients who received propofol, linear regression analyses were conducted in which the variable-transformed frequency of nausea and genotype data for each SNP were incorporated as dependent and independent variables, respectively, with proper covariates. Trend or additive, dominant, and recessive genetic models were used for the analyses due to our previously insufficient knowledge about genetic factors that are associated with PONV. Male genotypes were not included in the analysis of X chromosome markers, whereas both male and female individuals were included in the association study for autosomal markers. PLINK v. 1.07 ([https://zzz.bwh.harvard.edu/plink/index.shtml](https://zzz.bwh.harvard.edu/plink/index.shtml); accessed on 25 June 2023) [[38](#B38-cancers-15-04729)], gPLINK v. 2.050 [[38](#B38-cancers-15-04729)], and Haploview v. 4.1 [[39](#B39-cancers-15-04729)] were used to perform the statistical analyses and visualize the results. Bonferroni’s correction of multiple comparisons was performed to determine the significance of the results. The criterion for significance in the GWAS was set to *p* < 7.812 × 10^−8^ (~0.05/640,000), considering that valid statistical data were obtained for approximately 570,000–640,000 SNPs. Additionally, Hardy–Weinberg equilibrium was tested using the *χ*^2^ test (*df* = 1) for genotypic distributions of SNPs that were significantly associated with the phenotypes, with values of significant deviation set to *p* = 0.05.

In the replication study for JUH samples, clinical data on the presence/absence of nausea, presence/absence of vomiting, and presence/absence of PONV (the presence/absence of nausea or vomiting) during the 24 h postoperative period were made available and used for an additional association study. Fisher’s exact test or the Cochran–Armitage trend test were conducted in analyses using all patient samples, as in the GWAS. Trend, dominant, and recessive genetic models were, again, used for the analyses. PLINK v. 1.07 [[38](#B38-cancers-15-04729)] was used to perform the statistical analyses. The criterion for significance in the analysis was set to *p* < 0.05. Additionally, Hardy–Weinberg equilibrium was tested using the *χ*^2^ test (*df* = 1) for genotypic distributions of the candidate SNPs, with values of significant deviation set to *p* = 0.05.

### 2.5. Additional In Silico Analysis

#### 2.5.1. Power Analysis

Statistical power analyses were preliminarily performed using G*Power 3.1.3 software [[40](#B40-cancers-15-04729)]. Power analyses for Fisher’s exact tests, with degrees of freedom set to 2, indicated that the expected power (1 minus type II error probability) was 80.0% for the type I error probability, which was set to 1.000 × 10^−7^ (close to 7.812 × 10^−8^) when risk allele frequencies for patients with nausea and/or vomiting and patients without nausea and/or vomiting were 0.2756 and 0.1000, 0.3138 and 0.1000, and 0.2729 and 0.1000, and when the sample sizes for patients with nausea and/or vomiting and patients without nausea and/or vomiting were 265 and 541, 149 and 657, and 280 and 526, respectively, in the present study. However, for the same type I error probability and sample sizes of 265 and 541, 149 and 657, and 280 and 526, the expected power decreased to 50.0% when the risk allele frequencies for patients with nausea and/or vomiting and patients without nausea and/or vomiting were 0.2478 and 0.1000, 0.2788 and 0.1000, and 0.2455 and 0.1000, respectively. Conversely, the estimated risk allele frequencies for patients with nausea and/or vomiting and patients without nausea and/or vomiting were 0.2905 and 0.1000, 0.3327 and 0.1000, and 0.2876 and 0.1000 for the same type I error probability, and sample sizes of 265 and 541, 149 and 657, and 280 and 526, respectively, were required in order to achieve 90% power. Therefore, a single analysis in the present study was expected to detect true associations with the phenotypes, with 80% statistical power for effect sizes from large to moderately medium but not small, although the exact effect size is poorly understood in cases of SNPs that significantly contribute to PONV.

#### 2.5.2. Reference of Databases

Several databases and bioinformatic tools were referenced to more thoroughly examine the candidate SNPs that may be related to human vulnerability to PONV, including the National Center for Biotechnology (NCBI) database ([http://www.ncbi.nlm.nih.gov](http://www.ncbi.nlm.nih.gov); accessed on 19 January 2023), HaploReg v. 4.1 ([https://pubs.broadinstitute.org/mammals/haploreg/haploreg.php](https://pubs.broadinstitute.org/mammals/haploreg/haploreg.php); accessed on 25 June 2023) [[41](#B41-cancers-15-04729)], SNPinfo Web Server ([https://snpinfo.niehs.nih.gov](https://snpinfo.niehs.nih.gov); accessed on 25 June 2023) [[42](#B42-cancers-15-04729)], the Genotype-Tissue Expression (GTEx) portal ([https://gtexportal.org/home/](https://gtexportal.org/home/); accessed on 25 June 2023) [[43](#B43-cancers-15-04729)], the PheWeb database ([https://pheweb.jp/](https://pheweb.jp/); accessed on 18 July 2023) [[44](#B44-cancers-15-04729)], and the SIFT tool ([https://sift.bii.a-star.edu.sg/](https://sift.bii.a-star.edu.sg/); accessed on 18 July 2023). HaploReg is a tool for investigating non-coding genomic annotations at variations in haplotype blocks, such as potential regulatory SNPs at disease-associated sites [[41](#B41-cancers-15-04729)]. The SNPinfo Web Server is a set of web-based SNP selection tools (freely available at [https://snpinfo.niehs.nih.gov](https://snpinfo.niehs.nih.gov); accessed on 19 January 2023) where investigators can specify genes or linkage regions and select SNPs based on GWAS results, linkage disequilibrium (LD), and predicted functional characteristics of both coding and non-coding SNPs [[42](#B42-cancers-15-04729)]. The GTEx project, an ongoing effort to create a comprehensive public resource to study tissue-specific gene expression and regulation [[43](#B43-cancers-15-04729)], is the basis of the GTEx portal, which offers open access to data such as gene expression, quantitative trait loci, and histology images. The PheWeb database is a platform that releases GWAS summary statistics of the BioBank Japan Project (BBJ) [[44](#B44-cancers-15-04729)]. The SIFT tool was utilized to estimate whether amino acid substitution would affect protein function based on sequence homology and the physical properties of amino acids. Furthermore, the protein structures of contactin 5 and ATPase phospholipid transporting 8B3 were predicted from the amino acid sequence (NCBI accession no. [NP_001230199.1](https://www.ncbi.nlm.nih.gov/protein/NP_001230199.1) and [NP_001171473.1](https://www.ncbi.nlm.nih.gov/protein/NP_001171473.1), respectively) by the SWISS-MODEL server ([http://swissmodel.expasy.org/](http://swissmodel.expasy.org/); accessed on 18 July 2023).

## 3. Results

### 3.1. Impact of Clinical Variables on the Incidence of PONV in Subjects Who Underwent Elective Surgery under General Anesthesia with Propofol or Desflurane

In the CIH samples, nausea, vomiting, and PONV (nausea and/or vomiting) occurred in 32.88%, 18.49%, and 34.74% of the subjects, respectively ([Table 1](#cancers-15-04729-t001)). Prior to the GWAS, multivariate regression analysis was conducted to explore possible factors influencing the quantitative trait of the frequency of nausea among subjects characterized by various demographic and clinical data, described in [Table 1](#cancers-15-04729-t001). Significant associations were revealed between the variable-transformed frequency of nausea and gender (*β* = 0.1743, *p* = 0.0343), history of smoking (*β* = −0.2480, *p* = 0.0001), history of motion sickness (*β* = 0.1782, *p* = 0.0093), history of PONV (*β* = 0.3034, *p* = 0.0049), postoperative administration of narcotic drugs (*β* = 0.3383, *p* < 0.0001), frequency of pain (*β* = 0.0240, *p* = 0.0267), and duration of surgery (min) (*β* = 0.0007 *p* = 0.0152). Therefore, these variables were incorporated as covariates in the linear regression analyses for the frequency of nausea.

### 3.2. Identification of Genetic Polymorphisms Associated with PONV in All Patients Who Underwent Elective Surgery under General Anesthesia with Propofol or Desflurane

We comprehensively explored genetic variants that were associated with the presence or absence of nausea, frequency of nausea, presence or absence of vomiting, and presence or absence of PONV in a total of 806 patient subjects of CIH samples. We investigated common genetic factors for PONV by TIVA with propofol or inhalational anesthesia with desflurane. A total of 943,259 SNPs that met the quality control standards in the GWAS of all patients were examined for their relationships with the phenotypes in the trend, additive, dominant, and recessive models. No SNPs showed significant associations with the presence or absence of nausea, presence or absence of vomiting, or presence or absence of PONV ([Supplementary Tables S2–S4](#app1-cancers-15-04729)), but significant associations were found with the frequency of nausea. Significant associations were found for the rs140703637 (exm2274524) SNP on chromosome 11 in the dominant model (*p* = 5.555 × 10^−8^; [Table 2](#cancers-15-04729-t002), [Figure 1](#cancers-15-04729-f001)b) and the rs2776262 SNP on chromosome 21 in the recessive model (*p* = 7.573 × 10^−8^; [Table 2](#cancers-15-04729-t002), [Figure 1](#cancers-15-04729-f001)c). Many of the examined SNPs’ computed -log10 *p*-values (observed *p*-values), which were based on the null hypothesis of a uniform distribution in the QQ plot, differed from the predicted values ([Supplementary Figure S1](#app1-cancers-15-04729)). The values for SNPs with significant associations in [Table 2](#cancers-15-04729-t002) (rs140703637 and rs2776262) and other SNPs were obviously higher than the predicted values ([Supplementary Figure S1b,c](#app1-cancers-15-04729)). However, no significant associations were found in the additive model for this phenotype ([Table 2](#cancers-15-04729-t002), [Figure 1](#cancers-15-04729-f001)a). The rs140703637 SNP is located in the exon region of the contactin 5 (*CNTN5*) gene, which leads to missense mutation of the gene, and the rs2776262 SNP is located in the intron region of the *LOC100506403* gene, which is a non-coding gene that is not characterized well according to the annotation file that was supplied by manufacturer of the BeadChips or NCBI database ([Table 3](#cancers-15-04729-t003)). As shown in [Table 2](#cancers-15-04729-t002), when the heterozygous and homozygous minor alleles of the rs140703637 and rs2776262 SNPs were carried, respectively, it was associated with a greater frequency of nausea during the 48 h postoperative period. None of the genotype distributions for the SNPs that were significantly associated with the phenotype significantly deviated from theoretical Hardy–Weinberg equilibrium (*χ2* = 0.0050, *p* = 0.9975 for rs140703637; χ^2^ = 0.2384, *p* = 0.8876 for rs2776262).

### Table 2.

| Model | Rank | CHR | SNP | Position | p | Related Gene | Genotype (Patients) |  |  | Genotype (Mean) |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| A/A | A/B | B/B | A/A | A/B | B/B |  |  |  |  |  |  |  |
| Additive | 1 | 21 | rs2776262 | 36940158 | 0.00000008533 | (LOC100506403) | 2 | 95 | 709 | 4.145 | 0.313 | 0.556 |
| Additive | 2 | 19 | rs12609817 | 53859182 | 0.0000001055 |   | 4 | 109 | 693 | 2.882 | 0.486 | 0.531 |
| Additive | 3 | 7 | rs921634 | 47872845 | 0.0000001393 | PKD1L1 | 9 | 145 | 652 | 2.293 | 0.479 | 0.525 |
| Additive | 4 | 5 | rs1587176 | 174851190 | 0.0000002228 |   | 5 | 172 | 628 | 2.761 | 0.552 | 0.515 |
| Additive | 5 | 5 | rs10061408 | 117292776 | 0.0000005159 |   | 6 | 109 | 691 | 2.446 | 0.61 | 0.508 |
| Additive | 6 | 5 | rs10071777 | 117286831 | 0.000000524 |   | 6 | 109 | 690 | 2.446 | 0.61 | 0.509 |
| Additive | 7 | 8 | rs6558049 | 28093041 | 0.0000006488 |   | 5 | 109 | 692 | 2.747 | 0.508 | 0.525 |
| Additive | 8 | 16 | rs7192373 | 90032149 | 0.0000006522 | DEF8 | 2 | 58 | 746 | 3.612 | 0.588 | 0.524 |
| Additive | 9 | 7 | rs17173793 | 153523345 | 0.0000008597 |   | 2 | 111 | 691 | 3.766 | 0.707 | 0.499 |
| Additive | 10 | 5 | rs13159091 | 148067337 | 0.0000008984 |   | 2 | 100 | 704 | 3.808 | 0.638 | 0.513 |
| Additive | 11 | 5 | rs17414326 | 117297258 | 0.000001209 |   | 2 | 68 | 736 | 3.646 | 0.708 | 0.512 |
| Additive | 12 | 5 | rs1157652 | 8565638 | 0.000001661 |   | 21 | 193 | 592 | 1.572 | 0.543 | 0.497 |
| Additive | 13 | 2 | rs3112976 | 180440437 | 0.000001666 | ZNF385B | 2 | 118 | 686 | 3.449 | 0.595 | 0.518 |
| Additive | 14 | 4 | exm2265817 | 5023112 | 0.000001687 |   | 2 | 64 | 740 | 3.808 | 0.57 | 0.525 |
| Additive | 14 | 4 | rs10937615 | 5023112 | 0.000001687 |   | 2 | 64 | 740 | 3.808 | 0.57 | 0.525 |
| Additive | 16 | 12 | exm-rs11057830 | 125307053 | 0.000001762 | SCARB1 | 3 | 120 | 683 | 2.986 | 0.523 | 0.528 |
| Additive | 17 | 9 | exm-rs755109 | 100696203 | 0.000001821 | HEMGN | 13 | 176 | 617 | 1.713 | 0.542 | 0.51 |
| Additive | 18 | 9 | rs1475696 | 100691397 | 0.000001857 | HEMGN | 13 | 175 | 618 | 1.713 | 0.54 | 0.511 |
| Additive | 19 | 19 | rs11084950 | 2657397 | 0.000002102 | GNG7 | 2 | 104 | 700 | 3.766 | 0.503 | 0.532 |
| Additive | 20 | 12 | rs2137547 | 26261873 | 0.000002264 |   | 3 | 91 | 712 | 3.018 | 0.603 | 0.517 |
| Dominant | 1 | 11 | exm2274524 | 100221580 | 0.00000005555 * | CNTN5 | 0 | 4 | 801 | NA | 2.913 | 0.52 |
| Dominant | 2 | 1 | exm1762808 | 220823972 | 0.0000005544 | MARK1 | 0 | 2 | 802 | NA | 3.766 | 0.53 |
| Dominant | 3 | 8 | rs10087234 | 68735313 | 0.000001101 |   | 0 | 7 | 799 | NA | 2.205 | 0.522 |
| Dominant | 4 | 19 | exm1473939 | 42937953 | 0.000001221 | CXCL17 | 1 | 7 | 798 | 0 | 2.64 | 0.519 |
| Dominant | 5 | 3 | rs164464 | 8749012 | 0.000001845 |   | 0 | 2 | 803 | NA | 3.553 | 0.53 |
| Dominant | 6 | 3 | rs241055 | 8738790 | 0.000001855 |   | 0 | 2 | 804 | NA | 3.553 | 0.529 |
| Dominant | 6 | 3 | rs12497498 | 8751490 | 0.000001855 |   | 0 | 2 | 804 | NA | 3.553 | 0.529 |
| Dominant | 6 | 3 | rs17049459 | 8751984 | 0.000001855 |   | 0 | 2 | 804 | NA | 3.553 | 0.529 |
| Dominant | 9 | 15 | rs16948440 | 65255168 | 0.000002007 |   | 0 | 2 | 790 | NA | 3.612 | 0.524 |
| Dominant | 9 | 17 | rs222843 | 7145981 | 0.000002007 |   | 121 | 373 | 311 | 0.438 | 0.384 | 0.759 |
| Dominant | 11 | 2 | rs2075225 | 71063066 | 0.00000221 |   | 106 | 373 | 326 | 0.451 | 0.392 | 0.727 |
| Dominant | 12 | 17 | exm1286317 | 7163739 | 0.000002647 | CLDN7 | 121 | 374 | 311 | 0.438 | 0.385 | 0.756 |
| Dominant | 13 | 15 | rs34636936 | 65297261 | 0.000002663 | MTFMT | 0 | 2 | 804 | NA | 3.612 | 0.529 |
| Dominant | 14 | 22 | rs1210829 | 20308800 | 0.000003367 |   | 21 | 211 | 569 | 0.523 | 0.818 | 0.437 |
| Dominant | 15 | 4 | rs2204206 | 149713872 | 0.000003862 |   | 207 | 374 | 225 | 0.41 | 0.471 | 0.761 |
| Dominant | 16 | 17 | rs222835 | 7134129 | 0.000004137 | DVL2 | 120 | 362 | 324 | 0.481 | 0.365 | 0.749 |
| Dominant | 17 | 17 | rs222837 | 7132556 | 0.000004902 | DVL2 | 118 | 363 | 325 | 0.477 | 0.368 | 0.746 |
| Dominant | 18 | 17 | rs739669 | 7122377 | 0.000005338 | DLG4 | 118 | 362 | 326 | 0.477 | 0.369 | 0.744 |
| Dominant | 19 | 22 | exm2010161 | 38474406 | 0.000006687 | SLC16A8 | 0 | 2 | 804 | NA | 3.278 | 0.53 |
| Dominant | 20 | 4 | rs11734518 | 152213113 | 0.000008271 |   | 107 | 377 | 322 | 0.705 | 0.642 | 0.356 |
| Recessive | 1 | 21 | rs2776262 | 36940158 | 0.00000007573 * | (LOC100506403) | 2 | 95 | 709 | 4.145 | 0.313 | 0.556 |
| Recessive | 2 | 19 | rs12609817 | 53859182 | 0.00000009569 |   | 4 | 109 | 693 | 2.882 | 0.486 | 0.531 |
| Recessive | 3 | 7 | rs921634 | 47872845 | 0.0000001274 | PKD1L1 | 9 | 145 | 652 | 2.293 | 0.479 | 0.525 |
| Recessive | 4 | 5 | rs1587176 | 174851190 | 0.0000002107 |   | 5 | 172 | 628 | 2.761 | 0.552 | 0.515 |
| Recessive | 5 | 8 | rs6558049 | 28093041 | 0.0000006248 |   | 5 | 109 | 692 | 2.747 | 0.508 | 0.525 |
| Recessive | 6 | 5 | rs10061408 | 117292776 | 0.0000006556 |   | 6 | 109 | 691 | 2.446 | 0.61 | 0.508 |
| Recessive | 7 | 16 | rs7192373 | 90032149 | 0.0000006607 | DEF8 | 2 | 58 | 746 | 3.612 | 0.588 | 0.524 |
| Recessive | 8 | 5 | rs10071777 | 117286831 | 0.0000006649 |   | 6 | 109 | 690 | 2.446 | 0.61 | 0.509 |
| Recessive | 9 | 5 | rs13159091 | 148067337 | 0.000001017 |   | 2 | 100 | 704 | 3.808 | 0.638 | 0.513 |
| Recessive | 10 | 7 | rs17173793 | 153523345 | 0.00000114 |   | 2 | 111 | 691 | 3.766 | 0.707 | 0.499 |
| Recessive | 11 | 5 | rs17414326 | 117297258 | 0.000001587 |   | 2 | 68 | 736 | 3.646 | 0.708 | 0.512 |
| Recessive | 12 | 4 | exm2265817 | 5023112 | 0.000001705 |   | 2 | 64 | 740 | 3.808 | 0.57 | 0.525 |
| Recessive | 12 | 4 | rs10937615 | 5023112 | 0.000001705 |   | 2 | 64 | 740 | 3.808 | 0.57 | 0.525 |
| Recessive | 14 | 2 | rs3112976 | 180440437 | 0.00000171 | ZNF385B | 2 | 118 | 686 | 3.449 | 0.595 | 0.518 |
| Recessive | 15 | 12 | exm-rs11057830 | 125307053 | 0.00000185 | SCARB1 | 3 | 120 | 683 | 2.986 | 0.523 | 0.528 |
| Recessive | 16 | 10 | rs1342273 | 111973339 | 0.00000187 | MXI1 | 12 | 159 | 635 | 1.815 | 0.41 | 0.544 |
| Recessive | 17 | 5 | rs1157652 | 8565638 | 0.000001997 |   | 21 | 193 | 592 | 1.572 | 0.543 | 0.497 |
| Recessive | 18 | 19 | rs11084950 | 2657397 | 0.00000201 | GNG7 | 2 | 104 | 700 | 3.766 | 0.503 | 0.532 |
| Recessive | 19 | 5 | rs2591580 | 165406607 | 0.000002074 |   | 12 | 165 | 629 | 1.801 | 0.438 | 0.538 |
| Recessive | 20 | 9 | rs10115047 | 96631287 | 0.000002086 |   | 25 | 238 | 543 | 1.457 | 0.47 | 0.523 |

Table 2 Caption: Top 20 candidate SNPs selected from the GWAS for the frequency of nausea in all patients.

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2875/10571790/d320dd62a47a/cancers-15-04729-g001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=10571790_cancers-15-04729-g001.jpg)

Manhattan plot as a result of the GWAS for the frequency of nausea in all patients. (a) Plot of the results from the additive model. (b) Plot of the results from the dominant model. (c) Plot of the results from the recessive model. The highest part of each dot represents the calculated value. The red line indicates the threshold for a significant association.

### Table 3.

| Model | Rank | CHR | SNP | Position | p | Related Gene | Genotype (Patients) |  |  | Genotype (Mean) |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| A/A | A/B | B/B | A/A | A/B | B/B |  |  |  |  |  |  |  |
| Additive | 1 | 17 | rs7212072 | 11310500 | 0.00000003919 * | SHISA6 | 13 | 158 | 271 | 2.02 | 0.523 | 0.504 |
| Additive | 2 | 21 | rs2776262 | 36940158 | 0.00000005028 * | (LOC100506403) | 2 | 45 | 395 | 4.145 | 0.342 | 0.562 |
| Additive | 3 | 16 | rs12444143 | 6071910 | 0.0000000777 * | RBFOX1 | 84 | 197 | 161 | 0.981 | 0.572 | 0.312 |
| Additive | 4 | 10 | rs2499891 | 66680444 | 0.000000254 |   | 21 | 173 | 248 | 1.499 | 0.586 | 0.454 |
| Additive | 5 | 11 | rs7110244 | 95910507 | 0.0000003371 | MAML2 | 3 | 61 | 378 | 3.262 | 0.71 | 0.509 |
| Additive | 6 | 17 | rs1513743 | 11306268 | 0.000000375 | SHISA6 | 14 | 159 | 269 | 1.875 | 0.52 | 0.508 |
| Additive | 7 | 9 | rs1937955 | 25721500 | 0.0000004509 |   | 2 | 60 | 380 | 3.808 | 0.559 | 0.538 |
| Additive | 8 | 17 | rs4293419 | 11306043 | 0.0000005146 | SHISA6 | 3 | 96 | 343 | 3.413 | 0.646 | 0.505 |
| Additive | 9 | 17 | rs1034899 | 11306660 | 0.000000536 | SHISA6 | 3 | 96 | 342 | 3.413 | 0.646 | 0.507 |
| Additive | 10 | 10 | rs7911209 | 66726865 | 0.0000006068 |   | 20 | 176 | 245 | 1.501 | 0.573 | 0.467 |
| Additive | 11 | 10 | rs10733810 | 66723680 | 0.0000006447 |   | 20 | 168 | 254 | 1.501 | 0.595 | 0.455 |
| Additive | 12 | 8 | rs1735176 | 146074281 | 0.0000006883 |   | 18 | 164 | 259 | 1.733 | 0.443 | 0.547 |
| Additive | 13 | 16 | rs11649132 | 17032787 | 0.0000006937 |   | 15 | 138 | 289 | 1.587 | 0.601 | 0.48 |
| Additive | 14 | 14 | rs1256520 | 65737193 | 0.0000007361 |   | 2 | 81 | 359 | 4.162 | 0.606 | 0.524 |
| Additive | 14 | 14 | rs1256526 | 65739905 | 0.0000007361 |   | 2 | 81 | 359 | 4.162 | 0.606 | 0.524 |
| Additive | 16 | 16 | rs7192373 | 90032149 | 0.0000008101 | DEF8 | 2 | 29 | 411 | 3.612 | 0.677 | 0.532 |
| Additive | 17 | 8 | rs2322976 | 28106141 | 0.0000009019 |   | 2 | 57 | 383 | 3.808 | 0.601 | 0.532 |
| Additive | 18 | 8 | rs6558049 | 28093041 | 0.0000009166 |   | 2 | 59 | 381 | 3.808 | 0.58 | 0.534 |
| Additive | 19 | 13 | rs816958 | 108522895 | 0.0000009365 |   | 14 | 96 | 331 | 1.878 | 0.466 | 0.527 |
| Additive | 20 | 8 | exm734131 | 146076708 | 0.0000009716 | COMMD5 | 18 | 164 | 260 | 1.687 | 0.448 | 0.545 |
| Additive | 20 | 8 | rs1209879 | 146076708 | 0.0000009716 | COMMD5 | 18 | 164 | 260 | 1.687 | 0.448 | 0.545 |
| Dominant | 1 | 3 | rs13100791 | 49641049 | 0.0000009768 | BSN | 0 | 4 | 438 | NA | 2.629 | 0.536 |
| Dominant | 1 | 3 | exm315855 | 49737954 | 0.0000009768 | RNF123 | 0 | 4 | 438 | NA | 2.629 | 0.536 |
| Dominant | 1 | 3 | exm316496 | 49869455 | 0.0000009768 | TRAIP | 0 | 4 | 438 | NA | 2.629 | 0.536 |
| Dominant | 4 | 12 | exm1046926 | 123340542 | 0.000001253 | HIP1R | 0 | 2 | 440 | NA | 3.808 | 0.541 |
| Dominant | 5 | 11 | rs7933966 | 32875597 | 0.000001333 | PRRG4 | 94 | 224 | 124 | 0.501 | 0.379 | 0.915 |
| Dominant | 5 | 11 | exm-rs10767971 | 32895664 | 0.000001333 |   | 94 | 224 | 124 | 0.501 | 0.379 | 0.915 |
| Dominant | 5 | 11 | rs10767971 | 32895664 | 0.000001333 |   | 94 | 224 | 124 | 0.501 | 0.379 | 0.915 |
| Dominant | 8 | 5 | rs252110 | 141339522 | 0.000001858 |   | 0 | 8 | 433 | NA | 2.209 | 0.523 |
| Dominant | 9 | 5 | rs34221525 | 68798118 | 0.000001959 | OCLN | 0 | 7 | 435 | NA | 2.165 | 0.529 |
| Dominant | 10 | 11 | rs4755454 | 32903263 | 0.000002224 |   | 96 | 225 | 121 | 0.49 | 0.392 | 0.911 |
| Dominant | 11 | 9 | rs17725257 | 16216102 | 0.000002292 |   | 0 | 6 | 436 | NA | 2.427 | 0.53 |
| Dominant | 12 | 3 | rs241055 | 8738790 | 0.000002929 |   | 0 | 2 | 440 | NA | 3.553 | 0.542 |
| Dominant | 12 | 3 | rs164464 | 8749012 | 0.000002929 |   | 0 | 2 | 440 | NA | 3.553 | 0.542 |
| Dominant | 12 | 3 | rs12497498 | 8751490 | 0.000002929 |   | 0 | 2 | 440 | NA | 3.553 | 0.542 |
| Dominant | 12 | 3 | rs17049459 | 8751984 | 0.000002929 |   | 0 | 2 | 440 | NA | 3.553 | 0.542 |
| Dominant | 16 | 1 | rs198412 | 11900437 | 0.000003433 | CLCN6,NPPA-AS1 | 0 | 4 | 438 | NA | 2.964 | 0.533 |
| Dominant | 17 | 22 | exm2010161 | 38474406 | 0.000003506 | SLC16A8 | 0 | 2 | 440 | NA | 3.278 | 0.543 |
| Dominant | 18 | 17 | rs17780388 | 30606301 | 0.000003661 | RHBDL3 | 0 | 5 | 437 | NA | 2.671 | 0.531 |
| Dominant | 19 | 11 | rs197697 | 32834416 | 0.000003879 |   | 92 | 224 | 126 | 0.503 | 0.383 | 0.901 |
| Dominant | 20 | 7 | rs1982436 | 134470632 | 0.000003965 | CALD1 | 10 | 128 | 304 | 0.414 | 0.918 | 0.407 |
| Recessive | 1 | 17 | rs7212072 | 11310500 | 0.00000003412 * | SHISA6 | 13 | 158 | 271 | 2.02 | 0.523 | 0.504 |
| Recessive | 2 | 21 | rs2776262 | 36940158 | 0.00000004121 * | (LOC100506403) | 2 | 45 | 395 | 4.145 | 0.342 | 0.562 |
| Recessive | 3 | 8 | rs1735176 | 146074281 | 0.0000001881 |   | 18 | 164 | 259 | 1.733 | 0.443 | 0.547 |
| Recessive | 4 | 10 | rs2499891 | 66680444 | 0.0000002799 |   | 21 | 173 | 248 | 1.499 | 0.586 | 0.454 |
| Recessive | 5 | 8 | exm734131 | 146076708 | 0.0000002805 | COMMD5 | 18 | 164 | 260 | 1.687 | 0.448 | 0.545 |
| Recessive | 5 | 8 | rs1209879 | 146076708 | 0.0000002805 | COMMD5 | 18 | 164 | 260 | 1.687 | 0.448 | 0.545 |
| Recessive | 7 | 17 | rs1513743 | 11306268 | 0.0000003181 | SHISA6 | 14 | 159 | 269 | 1.875 | 0.52 | 0.508 |
| Recessive | 8 | 9 | rs1937955 | 25721500 | 0.0000004615 |   | 2 | 60 | 380 | 3.808 | 0.559 | 0.538 |
| Recessive | 9 | 11 | rs7110244 | 95910507 | 0.0000004699 | MAML2 | 3 | 61 | 378 | 3.262 | 0.71 | 0.509 |
| Recessive | 10 | 13 | rs816958 | 108522895 | 0.000000523 |   | 14 | 96 | 331 | 1.878 | 0.466 | 0.527 |
| Recessive | 11 | 10 | rs7911209 | 66726865 | 0.0000005299 |   | 20 | 176 | 245 | 1.501 | 0.573 | 0.467 |
| Recessive | 12 | 17 | rs4293419 | 11306043 | 0.0000007621 | SHISA6 | 3 | 96 | 343 | 3.413 | 0.646 | 0.505 |
| Recessive | 13 | 22 | rs8142156 | 19692751 | 0.0000007685 |   | 4 | 64 | 374 | 2.599 | 0.367 | 0.566 |
| Recessive | 13 | 22 | rs9618670 | 19694333 | 0.0000007685 |   | 4 | 63 | 375 | 2.599 | 0.373 | 0.564 |
| Recessive | 15 | 14 | rs1256520 | 65737193 | 0.0000007818 |   | 2 | 81 | 359 | 4.162 | 0.606 | 0.524 |
| Recessive | 15 | 14 | rs1256526 | 65739905 | 0.0000007818 |   | 2 | 81 | 359 | 4.162 | 0.606 | 0.524 |
| Recessive | 17 | 10 | rs10733810 | 66723680 | 0.0000007878 |   | 20 | 168 | 254 | 1.501 | 0.595 | 0.455 |
| Recessive | 18 | 17 | rs1034899 | 11306660 | 0.0000007904 | SHISA6 | 3 | 96 | 342 | 3.413 | 0.646 | 0.507 |
| Recessive | 19 | 16 | rs7192373 | 90032149 | 0.0000008464 | DEF8 | 2 | 29 | 411 | 3.612 | 0.677 | 0.532 |
| Recessive | 20 | 8 | rs6558049 | 28093041 | 0.0000009534 |   | 2 | 59 | 381 | 3.808 | 0.58 | 0.534 |
| Recessive | 20 | 8 | rs2322976 | 28106141 | 0.0000009534 |   | 2 | 57 | 383 | 3.808 | 0.601 | 0.532 |

Table 3 Caption: Top 20 candidate SNPs selected from the GWAS for the frequency of nausea in patients who received propofol.

### 3.3. Identification of Genetic Polymorphisms Associated with PONV in Patients Who Underwent Elective Surgery under General Anesthesia with Propofol

Although two kinds of anesthesia were used in patients who underwent general anesthesia at The Cancer Institute Hospital, the major type of anesthesia was TIVA by propofol ([Table 1](#cancers-15-04729-t001)). Considering the possibility that genetic factors would more greatly contribute to the incidence of PONV without the use of volatile anesthetics compared with the use of volatile anesthetics, which are known to cause PONV more often than i.v. anesthetics, we then conducted another GWAS of the same SNPs by including only a subgroup of 442 patients who underwent general anesthesia by TIVA with propofol in CIH samples ([Table 1](#cancers-15-04729-t001)). No SNPs showed significant associations with the presence or absence of nausea or PONV ([Supplementary Tables S5 and S6](#app1-cancers-15-04729)), but significant associations were found with the frequency of nausea. Significant associations were found for the rs7212072 SNP on chromosome 17 (*p* = 3.919 × 10^−8^), the rs2776262 SNP on chromosome 21 (*p* = 5.028 × 10^−8^), and the rs12444143 SNP on chromosome 16 (*p* = 7.770 × 10^−8^) in the additive model ([Table 3](#cancers-15-04729-t003), [Figure 2](#cancers-15-04729-f002)a). Associations were also significant for the rs7212072 SNP (*p* = 3.412 × 10^−8^) and rs2776262 SNP (*p* = 4.121 × 10^−8^) in the recessive model ([Table 3](#cancers-15-04729-t003), [Figure 2](#cancers-15-04729-f002)c). However, no significant associations were found in the dominant model for this phenotype ([Table 3](#cancers-15-04729-t003), [Figure 2](#cancers-15-04729-f002)b). Many of the examined SNPs’ computed -log10 *p* values (observed *p* values) differed from the predicted values ([Supplementary Figure S2](#app1-cancers-15-04729)), and the values for SNPs with significant associations in [Table 3](#cancers-15-04729-t003) (rs7212072, rs2776262, and rs12444143) and other SNPs were obviously higher than the predicted values ([Supplementary Figure S2a,c](#app1-cancers-15-04729)). Additionally, significant associations were found between the presence or absence of vomiting and the rs45574836 (exm1401859) SNP on chromosome 19 (*p* = 2.972 × 10^−8^) and rs1752136 SNP on chromosome 13 (*p* = 6.384 × 10^−8^) in the trend model ([Table 4](#cancers-15-04729-t004), [Figure 3](#cancers-15-04729-f003)a). However, significant associations were not found in the dominant or recessive models in this phenotype ([Table 4](#cancers-15-04729-t004), [Figure 3](#cancers-15-04729-f003)b,c). Many of the examined SNPs’ computed −log10 *p* values (observed *p* values) differed from the predicted values ([Supplementary Figure S3](#app1-cancers-15-04729)), and the values for SNPs with significant associations in [Table 4](#cancers-15-04729-t004) (rs45574836 and rs1752136) and other SNPs were obviously higher than the predicted values ([Supplementary Figure S3a](#app1-cancers-15-04729)). The rs7212072 and rs12444143 SNPs are located in intron regions of genes that encode shisa family member 6 (*SHISA6*) and RNA binding fox-1 homolog 1 (*RBFOX1*), respectively ([Table 3](#cancers-15-04729-t003)), and the rs45574836 SNP is located in exon regions of the gene that encodes ATPase phospholipid transporting 8B3 (*ATP8B3*), which leads to missense mutation of the gene ([Table 4](#cancers-15-04729-t004)). The rs1752136 SNP is located in intron regions of the *LOC105370198* gene, which is a non-coding gene that is not characterized well according to the annotation file that was supplied by the manufacturer of the BeadChips or NCBI database ([Table 4](#cancers-15-04729-t004)). As shown in [Table 3](#cancers-15-04729-t003) and [Table 4](#cancers-15-04729-t004), the presence of minor alleles of the rs7212072, rs2776262, rs12444143, rs45574836, and rs1752136 SNPs was additively or homozygously associated with a greater frequency of nausea or greater incidence of vomiting during the 48 h postoperative period. None of the genotype distributions for the SNPs that were significantly associated with the phenotypes significantly deviated from the theoretical Hardy–Weinberg equilibrium (*χ*^2^ = 3.1493, *p* = 0.0760 for rs7212072; *χ*^2^ = 0.3401, *p* = 0.5598 for rs2776262; *χ*^2^ = 2.8784, *p* = 0.0898 for rs12444143; *χ*^2^ = 0.2499, *p* = 0.6171 for rs45574836; *χ*^2^ = 0.7072, *p* = 0.4004 for rs1752136).

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2875/10571790/d386b09e78ef/cancers-15-04729-g002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=10571790_cancers-15-04729-g002.jpg)

Manhattan plot as a result of the GWAS for the frequency of nausea in patients who received propofol. (a) Plot of the results from the additive model. (b) Plot of the results from the dominant model. (c) Plot of the results from the recessive model. The highest part of each dot represents the calculated value. The red line indicates the threshold for a significant association.

### Table 4.

| Model | Rank | CHR | SNP | Position | p | Related Gene | Genotype (Vomiting +) |  |  | Genotype (Vomiting −) |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| A/A | A/B | B/B | A/A | A/B | B/B |  |  |  |  |  |  |  |
| Trend | 1 | 19 | exm1401859 | 1806667 | 0.00000002972 * | ATP8B3 | 4 | 31 | 62 | 1 | 44 | 300 |
| Trend | 2 | 13 | rs1752136 | 48726219 | 0.00000006384 * | (LOC105370198) | 0 | 20 | 77 | 0 | 14 | 331 |
| Trend | 3 | 1 | rs6675501 | 24672641 | 0.000003994 | GRHL3 | 42 | 41 | 14 | 73 | 165 | 107 |
| Trend | 4 | 16 | rs16954219 | 80925781 | 0.000006147 |   | 4 | 43 | 50 | 4 | 83 | 258 |
| Trend | 5 | 17 | rs16944600 | 11245099 | 0.000006816 | SHISA6 | 4 | 23 | 70 | 27 | 167 | 151 |
| Trend | 6 | 14 | rs10438059 | 33026271 | 0.00001081 | AKAP6 | 13 | 48 | 36 | 16 | 123 | 205 |
| Trend | 7 | 16 | rs12928123 | 61178279 | 0.00001632 |   | 5 | 33 | 59 | 5 | 60 | 280 |
| Trend | 7 | 16 | rs9929283 | 61182342 | 0.00001632 |   | 5 | 33 | 59 | 5 | 60 | 280 |
| Trend | 9 | 9 | rs10867734 | 83933198 | 0.00001865 |   | 2 | 14 | 81 | 0 | 16 | 329 |
| Trend | 10 | 2 | exm-rs6726639 | 112753097 | 0.00002221 | MERTK | 31 | 46 | 15 | 65 | 132 | 131 |
| Trend | 11 | 15 | rs765 | 48035685 | 0.00002252 | SEMA6D | 7 | 49 | 41 | 11 | 106 | 228 |
| Trend | 12 | 9 | rs7850697 | 131095112 | 0.00002275 | COQ4 | 2 | 41 | 54 | 38 | 189 | 118 |
| Trend | 12 | 9 | rs7030121 | 131101919 | 0.00002275 |   | 2 | 41 | 54 | 38 | 189 | 118 |
| Trend | 14 | 9 | rs12684445 | 98805260 | 0.00002621 |   | 11 | 47 | 39 | 14 | 117 | 214 |
| Trend | 15 | 7 | rs12704714 | 93930713 | 0.00002643 |   | 32 | 39 | 26 | 42 | 164 | 138 |
| Trend | 16 | 16 | rs12445491 | 61145947 | 0.00002755 |   | 8 | 30 | 59 | 6 | 66 | 273 |
| Trend | 17 | 9 | rs306772 | 124092355 | 0.00002782 | GSN | 1 | 32 | 64 | 1 | 51 | 293 |
| Trend | 18 | 9 | rs2240960 | 131039250 | 0.00002911 | SWI5 | 4 | 37 | 56 | 39 | 186 | 118 |
| Trend | 19 | 4 | rs2086431 | 9509791 | 0.00003047 |   | 4 | 28 | 65 | 3 | 49 | 293 |
| Trend | 20 | 14 | rs17098983 | 32868873 | 0.00003088 | AKAP6 | 0 | 20 | 77 | 19 | 128 | 198 |
| Dominant | 1 | 19 | exm1401859 | 1806667 | 0.0000009938 | ATP8B3 | 4 | 31 | 62 | 1 | 44 | 300 |
| Dominant | 2 | 17 | rs16944600 | 11245099 | 0.000001015 | SHISA6 | 4 | 23 | 70 | 27 | 167 | 151 |
| Dominant | 3 | 13 | rs1752136 | 48726219 | 0.000001177 |   | 0 | 20 | 77 | 0 | 14 | 331 |
| Dominant | 4 | 5 | rs7715247 | 50318550 | 0.000001902 |   | 7 | 22 | 68 | 31 | 167 | 147 |
| Dominant | 5 | 4 | rs11946898 | 181926518 | 0.000005267 |   | 23 | 62 | 12 | 65 | 157 | 123 |
| Dominant | 6 | 3 | rs1524511 | 179642067 | 0.000005693 | PEX5L | 10 | 53 | 34 | 23 | 111 | 211 |
| Dominant | 7 | 5 | rs622304 | 50279857 | 0.000005922 |   | 7 | 20 | 70 | 27 | 159 | 159 |
| Dominant | 8 | 4 | rs12642493 | 181933056 | 0.00001346 |   | 23 | 60 | 14 | 64 | 153 | 128 |
| Dominant | 9 | 9 | rs3808657 | 19127877 | 0.00001358 |   | 11 | 24 | 61 | 47 | 167 | 131 |
| Dominant | 10 | 18 | rs8093227 | 2078441 | 0.00001524 |   | 11 | 62 | 24 | 34 | 140 | 171 |
| Dominant | 11 | 16 | rs16954219 | 80925781 | 0.00001919 |   | 4 | 43 | 50 | 4 | 83 | 258 |
| Dominant | 12 | 2 | exm-rs6726639 | 112753097 | 0.00002021 | MERTK | 31 | 46 | 15 | 65 | 132 | 131 |
| Dominant | 13 | 5 | rs250216 | 50281358 | 0.00002777 |   | 4 | 21 | 72 | 19 | 152 | 174 |
| Dominant | 14 | 5 | rs13175573 | 50297517 | 0.00002949 |   | 5 | 23 | 69 | 22 | 161 | 162 |
| Dominant | 15 | 11 | rs624584 | 107344724 | 0.00003038 |   | 4 | 12 | 81 | 12 | 122 | 211 |
| Dominant | 16 | 15 | rs765 | 48035685 | 0.00003277 | SEMA6D | 7 | 49 | 41 | 11 | 106 | 228 |
| Dominant | 17 | 23 | rs12688309 | 113796171 | 0.00003574 |   | 0 | 8 | 81 | 5 | 79 | 199 |
| Dominant | 18 | 1 | rs12027987 | 85206774 | 0.00004063 |   | 8 | 31 | 58 | 53 | 167 | 125 |
| Dominant | 19 | 5 | rs12659587 | 50269410 | 0.00004483 |   | 4 | 21 | 72 | 19 | 150 | 176 |
| Dominant | 20 | 9 | rs9792672 | 23087926 | 0.00004684 |   | 1 | 4 | 92 | 4 | 73 | 268 |
| Recessive | 1 | 1 | rs1195866 | 81678254 | 0.00000133 |   | 14 | 28 | 55 | 5 | 117 | 219 |
| Recessive | 2 | 7 | rs12704714 | 93930713 | 0.000005402 |   | 32 | 39 | 26 | 42 | 164 | 138 |
| Recessive | 3 | 1 | rs734591 | 203449615 | 0.00001109 | PRELP | 43 | 29 | 25 | 72 | 174 | 99 |
| Recessive | 4 | 6 | rs4897427 | 130810954 | 0.00001688 |   | 29 | 34 | 33 | 38 | 177 | 130 |
| Recessive | 5 | 6 | rs9492605 | 130796396 | 0.00002139 |   | 24 | 31 | 42 | 27 | 154 | 162 |
| Recessive | 6 | 7 | rs2904188 | 68504050 | 0.00002176 |   | 11 | 21 | 65 | 4 | 96 | 245 |
| Recessive | 6 | 7 | rs2869745 | 68539977 | 0.00002176 |   | 11 | 22 | 64 | 4 | 98 | 243 |
| Recessive | 8 | 1 | rs6675501 | 24672641 | 0.00002366 | GRHL3 | 42 | 41 | 14 | 73 | 165 | 107 |
| Recessive | 9 | 1 | rs880878 | 203435718 | 0.00002728 |   | 43 | 30 | 24 | 76 | 172 | 97 |
| Recessive | 10 | 4 | rs16994732 | 38717953 | 0.00003138 |   | 8 | 27 | 62 | 1 | 89 | 255 |
| Recessive | 11 | 7 | rs12698219 | 158145606 | 0.0000321 | PTPRN2 | 9 | 18 | 70 | 2 | 65 | 278 |
| Recessive | 12 | 10 | rs7895191 | 70221267 | 0.00003529 | DNA2 | 0 | 46 | 51 | 41 | 143 | 161 |
| Recessive | 12 | 10 | rs12220316 | 70230840 | 0.00003529 | DNA2 | 0 | 46 | 51 | 41 | 143 | 161 |
| Recessive | 14 | 1 | rs3766902 | 203478370 | 0.00003617 |   | 39 | 36 | 22 | 65 | 158 | 122 |
| Recessive | 15 | 6 | rs4395717 | 20826281 | 0.00003964 | CDKAL1 | 7 | 59 | 30 | 89 | 174 | 82 |
| Recessive | 16 | 6 | rs4077405 | 20876683 | 0.00004018 | CDKAL1 | 7 | 60 | 30 | 88 | 174 | 83 |
| Recessive | 17 | 8 | exm719103 | 124440174 | 0.00004181 | WDYHV1 | 28 | 44 | 25 | 38 | 181 | 126 |
| Recessive | 17 | 8 | rs6999234 | 124440174 | 0.00004181 | WDYHV1 | 28 | 44 | 25 | 38 | 181 | 126 |
| Recessive | 17 | 8 | exm719120 | 124448736 | 0.00004181 | WDYHV1 | 28 | 44 | 25 | 38 | 181 | 126 |
| Recessive | 17 | 8 | rs7014678 | 124448736 | 0.00004181 | WDYHV1 | 28 | 44 | 25 | 38 | 181 | 126 |
| Recessive | 17 | 8 | exm719122 | 124448804 | 0.00004181 | WDYHV1 | 28 | 44 | 25 | 38 | 181 | 126 |
| Recessive | 17 | 8 | exm719125 | 124449466 | 0.00004181 | WDYHV1 | 28 | 44 | 25 | 38 | 181 | 126 |
| Recessive | 17 | 8 | rs3824250 | 124449466 | 0.00004181 | WDYHV1 | 28 | 44 | 25 | 38 | 181 | 126 |
| Recessive | 17 | 8 | rs13269287 | 124453662 | 0.00004181 | WDYHV1 | 28 | 44 | 25 | 38 | 181 | 126 |
| Recessive | 17 | 8 | rs7822061 | 124456727 | 0.00004181 |   | 28 | 44 | 25 | 38 | 181 | 126 |

Table 4 Caption: Top 20 candidate SNPs selected from the GWAS for vomiting in patients who received propofol.

### Figure 3.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2875/10571790/0f08a6a6ab90/cancers-15-04729-g003.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=10571790_cancers-15-04729-g003.jpg)

Manhattan plot as a result of the GWAS for vomiting in patients who received propofol. (a) Plot of the results from the trend model. (b) Plot of the results from the dominant model. (c) Plot of the results from the recessive model. The highest point of each dot represents the calculated value. The red line indicates the threshold for a significant association.

### 3.4. Replication of Possible Association between the rs45574836 (exm1401859) SNP and Vomiting in Patients Who Underwent Gynecological Laparoscopic Surgery

The observed associations between the rs140703637, rs2776262, rs7212072, and rs12444143 SNPs and the frequency of nausea, as well as between the rs45574836 and rs1752136 SNPs and the presence of vomiting, that were identified in the GWAS of CIH samples suggest that the rs140703637, rs2776262, rs7212072, and rs12444143 SNPs were related to vulnerability to nausea, whereas the rs45574836 and rs1752136 SNPs were related to the vulnerability to vomiting in this cohort. Carriers of risk alleles of these SNPs may have presented higher vulnerability than non-carriers. To examine whether the possible difference between genotypes in the vulnerability to nausea/vomiting could be corroborated in another cohort of patients, we compared phenotypic traits that were related to PONV between genotypes of these SNPs in patients who underwent LGS (JUH samples). Clinical data on the frequency of nausea during the postoperative period were unavailable, but clinical data on the presence/absence of nausea, vomiting, and PONV were available in the JUH samples. Although the probe for the rs1752136 SNP was not included in the SNP array for genotyping in JUH samples, the probe for the rs45574836 SNP was included in the SNP array, and genotype data were available. Therefore, an additional association study for replication of the GWAS results was conducted for this SNP. The association analyses of the presence/absence of nausea, vomiting, and PONV revealed significant associations with vomiting in the trend and dominant models (*p* = 0.0239 in the trend model; *p* = 0.0313 in the dominant model; [Table 5](#cancers-15-04729-t005)). Significant associations were also found between this SNP and PONV in the trend and dominant models (*p* = 0.0464 in trend model; *p* = 0.0289 in dominant model; [Table 5](#cancers-15-04729-t005)). The association between this SNP and nausea was marginally significant in the dominant model (*p* = 0.0645), but no significant associations were found in the other analyses ([Table 5](#cancers-15-04729-t005)). As observed in the GWAS of CIH samples, strong associations between this SNP and vomiting in the trend and dominant models were observed in the replication study of JUH samples ([Table 5](#cancers-15-04729-t005)), suggesting that the presence of the minor A allele of this SNP is associated with PONV, especially vomiting. The genotype distributions for this SNP in JUH samples were 263, 67, and 3 for the G/G, G/A, and A/A genotypes, respectively, which did not significantly deviate from the theoretical Hardy–Weinberg equilibrium (*χ2* = 0.3158, *p* = 0.5742).

### Table 5.

| Phenotype(+/−) | GWAS |  |  |  |  |  | Replication Study |  |  |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Genotype |  |  | p | p | p | Genotype |  |  | p | p | p |  |
| G/G | G/A | A/A | (Trend) | (Dominant) | (Recessive) | G/G | G/A | A/A | (Trend) | (Dominant) | (Recessive) |  |
| Nausea (−) | 252 | 39 | 2 | 0.001456 | 0.002497 | 0.3409 | 236 | 54 | 3 | 0.1010 | 0.06452 | 1 |
| Nausea (+) | 110 | 36 | 3 | 27 | 13 | 0 |  |  |  |  |  |  |
| Vomiting (−) | 300 | 44 | 1 | 2.972 × 10−8 † | 9.938 × 10−7 | 0.009167 | 253 | 59 | 3 | 0.02389 * | 0.03131 * | 1 |
| Vomiting (+) | 62 | 31 | 4 | 10 | 8 | 0 |  |  |  |  |  |  |
| PONV (−) | 244 | 37 | 1 | 0.0003522 | 0.001215 | 0.05993 | 234 | 52 | 3 | 0.04636 * | 0.02885 * | 1 |
| PONV (+) | 118 | 38 | 4 | 29 | 15 | 0 |  |  |  |  |  |  |

Table 5 Caption: Results of the replication study for the rs45574836 (exm1401859) SNP, selected in the GWAS for vomiting in patients who received propofol.

## 4. Discussion

To identify potential genetic variants that contribute to the vulnerability to PONV, we conducted a GWAS on patient subjects who underwent general anesthesia by TIVA with propofol or inhalational anesthesia with desflurane ([Table 1](#cancers-15-04729-t001)). Our GWAS identified several potent SNPs that may possibly be associated with the frequency of nausea, including rs2776262, rs7212072, and rs12444143 in the additive and/or recessive models ([Table 2](#cancers-15-04729-t002) and [Table 3](#cancers-15-04729-t003); [Figure 1](#cancers-15-04729-f001) and [Figure 2](#cancers-15-04729-f002)) and rs140703637 (exm2274524) in the dominant model ([Table 2](#cancers-15-04729-t002); [Figure 1](#cancers-15-04729-f001)). Our GWAS also identified potent SNPs that may possibly be associated with the presence/absence of vomiting, including rs45574836 (exm1401859) and rs1752136 in the trend model ([Table 4](#cancers-15-04729-t004); [Figure 3](#cancers-15-04729-f003)). Among them, the association between the rs45574836 (exm1401859) SNP and vomiting that was found in CIH samples was replicated in JUH samples ([Table 5](#cancers-15-04729-t005)). The results suggested that carriers of the minor A allele of the rs45574836 SNP in the *ATP8B3* gene region that leads to a missense mutation were more vulnerable to vomiting, and, thus, tended to experience vomiting more often than noncarriers during the postoperative period ([Table 4](#cancers-15-04729-t004) and [Table 5](#cancers-15-04729-t005)). Future studies with larger sample sizes are required in order to corroborate the significant results that were identified in our GWAS.

Previous studies have identified several genetic variants that are associated with phenotypes that are related to PONV [[11](#B11-cancers-15-04729),[12](#B12-cancers-15-04729),[13](#B13-cancers-15-04729),[14](#B14-cancers-15-04729)]. Included in the variants that have been identified in previous candidate gene association studies are the rs33940208, rs1985242, rs10160548, and rs1176713 SNPs of the *HTR3A* gene [[15](#B15-cancers-15-04729),[16](#B16-cancers-15-04729)]; the rs34236293, rs45519137, rs3758987, and -100_-102AAG deletion polymorphisms of the *HTR3B* gene [[16](#B16-cancers-15-04729),[17](#B17-cancers-15-04729),[18](#B18-cancers-15-04729)]; the rs1800497 SNP of the *DRD2*/*ANKK1* gene [[19](#B19-cancers-15-04729),[20](#B20-cancers-15-04729)]; the rs1799971 and rs9397685 SNPs of the *OPRM1* gene [[21](#B21-cancers-15-04729),[22](#B22-cancers-15-04729)]; the rs3755468 SNP of the *TACR1* gene [[23](#B23-cancers-15-04729)]; the 5-HTTLPR and rs25531 polymorphisms of the *SLC6A4* gene [[24](#B24-cancers-15-04729)]; the rs2032582 and rs1045642 SNPs of the *ABCB1* gene [[25](#B25-cancers-15-04729),[26](#B26-cancers-15-04729),[27](#B27-cancers-15-04729)]; the rs12208357 SNP of the *OCT1* (*SLC22A1*) gene [[28](#B28-cancers-15-04729)]; and the rs16947, rs35742686, rs1135824, rs3892097, rs5030655, rs5030867, rs5030865, rs5030656, rs1065852, and rs5030863 SNPs of the *CYP2D6* gene [[12](#B12-cancers-15-04729),[29](#B29-cancers-15-04729),[30](#B30-cancers-15-04729),[31](#B31-cancers-15-04729)]. Additionally, genetic variants identified in previous GWASs were the rs2165870 SNP of the *CHRM3* gene, the rs349358 SNP of the *KCNB2* gene, the rs1800497 SNP of the *DRD2*/*ANKK1* gene, the rs1333114 SNP of the *PTPRD* gene, and the rs11232965 SNP of the *MIR4300HG* gene [[32](#B32-cancers-15-04729),[33](#B33-cancers-15-04729),[34](#B34-cancers-15-04729),[35](#B35-cancers-15-04729)]. Among these SNPs, the rs10160548 SNP of the *HTR3A* gene, the rs3758987 SNP of the *HTR3B* gene, the rs1800497 SNP of the *DRD2*/*ANKK1* gene, the rs1799971 SNP of the *OPRM1* gene, the rs3755468 SNP of the *TACR1* gene, and the rs2032582 and rs1045642 SNPs of the *ABCB1* gene were included in the SNP array that was used in the present study. However, almost none of these SNPs were even nominally significantly associated with PONV-related phenotypes (*p* > 0.05) according to our association analysis (details not shown), except for the rs10160548 SNP of the *HTR3A* gene and the rs1799971 SNP of the *OPRM1* gene, which showed some associations in patients who received propofol anesthesia (*p* = 0.03864, nausea in additive model; *p* = 0.03512, nausea in recessive model; *p* = 0.02408, PONV in additive model; and *p* = 0.004705, PONV in the recessive model for the rs10160548 SNP; *p* = 0.04958, frequency of nausea in the dominant model for the rs1799971 SNP). For the rs10160548 SNP, the results of our association analysis indicated that the minor allele was associated with a lower incidence of nausea/PONV compared with noncarriers in patients who received propofol anesthesia, which appears to be in accordance with the trend that was observed for nausea in a previous report by Lin et al. (2014) [[15](#B15-cancers-15-04729)]. For the rs1799971 SNP, the results of our association analysis indicated that carriers of the G/G genotype were associated with more frequent nausea compared with noncarriers, which appears to be the opposite trend to the results that were reported by Lee et al. (2015), in which patients with the G/G genotype were lower on the PONV scale upon arrival to the post-anesthesia care unit [[21](#B21-cancers-15-04729)]. These different results between the present study and previous studies might indicate the general difficulty of replicating the results of human genetic association studies, likely because of the heterogeneity of the study designs, length of the postoperative period during which PONV was recorded, the severity of PONV, the types of surgery, the types of anesthetics, and the genetic backgrounds of the subjects, among other factors.

Propofol is one of the most commonly used intravenous anesthetics, and inhalational volatile anesthetics, such as sevoflurane and desflurane, are also often used in general anesthesia. Indeed, most previous genetic association studies of PONV-related phenotypes were conducted in patients who underwent general anesthesia with these anesthetics [[14](#B14-cancers-15-04729),[15](#B15-cancers-15-04729),[18](#B18-cancers-15-04729),[19](#B19-cancers-15-04729),[20](#B20-cancers-15-04729),[21](#B21-cancers-15-04729),[22](#B22-cancers-15-04729),[23](#B23-cancers-15-04729),[25](#B25-cancers-15-04729),[26](#B26-cancers-15-04729),[28](#B28-cancers-15-04729),[30](#B30-cancers-15-04729),[33](#B33-cancers-15-04729),[34](#B34-cancers-15-04729),[35](#B35-cancers-15-04729)]. In the present study, the subjects who were recruited for the GWAS were patients who underwent general anesthesia with either propofol or desflurane. Given that the use of volatile anesthetics, per se, is known as a strong anesthesia-related risk factor for PONV [[8](#B8-cancers-15-04729),[9](#B9-cancers-15-04729),[10](#B10-cancers-15-04729)], one could assume that genetic factors would more greatly contribute to the incidence of PONV without the use of volatile anesthetics compared with that with the use of volatile anesthetics. From this viewpoint, we conducted two kinds of GWASs. One was performed in all subjects to investigate common genetic factors for PONV by TIVA with propofol or inhalational anesthesia with desflurane. The other was performed only in subjects who underwent general anesthesia by TIVA with propofol. As a result, two SNPs were identified to be significantly associated with the frequency of nausea in all subjects ([Table 2](#cancers-15-04729-t002); [Figure 1](#cancers-15-04729-f001)), and three SNPs were identified to be significantly associated with the same phenotype in the subgroup of subjects who underwent general anesthesia by TIVA with propofol ([Table 3](#cancers-15-04729-t003); [Figure 2](#cancers-15-04729-f002)), among which one SNP was common in both subject groups ([Table 2](#cancers-15-04729-t002) and [Table 3](#cancers-15-04729-t003); [Figure 1](#cancers-15-04729-f001) and [Figure 2](#cancers-15-04729-f002)). Moreover, the analysis of the subgroup of subjects also identified two SNPs that were significantly associated with the incidence of vomiting ([Table 4](#cancers-15-04729-t004); [Figure 3](#cancers-15-04729-f003)), which were not identified to be significantly associated with the same phenotype in the analysis of all subjects ([Supplementary Table S3](#app1-cancers-15-04729)), although the sample size in the analysis for the subgroup with propofol was much smaller than that for all subjects ([Table 1](#cancers-15-04729-t001)). These outcomes might suggest the possibility that different genetic variants are involved in the etiology of PONV in the case of propofol and desflurane, as well as suggesting the importance of the homogeneity of samples used in human genetic association studies.

The best candidate SNPs in the present study were rs2776262, rs140703637, rs7212072, and rs12444143 for the frequency of nausea ([Table 2](#cancers-15-04729-t002) and [Table 3](#cancers-15-04729-t003); [Figure 1](#cancers-15-04729-f001) and [Figure 2](#cancers-15-04729-f002)) and rs45574836 and rs1752136 for the presence/absence of vomiting ([Table 4](#cancers-15-04729-t004); [Figure 3](#cancers-15-04729-f003)). The rs2776262, rs140703637, rs7212072, rs12444143, rs45574836, and rs1752136 SNPs are located in the *LOC100506403*, *CNTN5*, *SHISA6*, *RBFOX1*, *ATP8B3*, and *LOC105370198* genes, respectively ([Table 2](#cancers-15-04729-t002), [Table 3](#cancers-15-04729-t003) and [Table 4](#cancers-15-04729-t004)). Among them, *LOC100506403* and *LOC105370198* are non-coding genes that do not appear to have been characterized well to date. The *CNTN5*, *SHISA6*, *RBFOX1*, and *ATP8B3* genes encode contactin 5, shisa family member 6, RNA binding fox-1 homolog 1, and ATPase phospholipid transporting 8B3, respectively. According to the NCBI database, contactin 5 is a member of the immunoglobulin superfamily and contactin family, which mediate cell surface interactions during nervous system development, and biased expression in the placenta and thyroid, among others, has been observed. Likewise, shisa family member 6 is predicted to enable ionotropic glutamate receptor binding activity and to be involved in several processes, including excitatory chemical synaptic transmission, the regulation of short-term neuronal synaptic plasticity, and the regulation of signal transduction. Biased expression in the brain and endometrium, among others, has been observed. The RNA-binding fox-1 homolog 1 is known as a Fox-1 family of RNA-binding proteins that regulates tissue-specific alternative splicing in metazoa, and biased expression in the brain and heart has been observed. The ATPase phospholipid transporting 8B3 belongs to the family of P-type cation transport ATPases and the subfamily of aminophospholipid-transporting ATPases that transport phosphatidylserine and phosphatidylethanolamine from one side of a bilayer to the other. Biased expression in the testis and endometrium, among others, has been observed.

To date, none of the *LOC100506403*, *CNTN5*, *SHISA6*, *RBFOX1*, *ATP8B3*, and *LOC105370198* genes have been reported to be involved in mechanisms of nausea or vomiting. Although *Cntn5* knockout animals exhibited no behavioral abnormalities, the mice were leaner, with less body mass and lower fat percentages than wildtype animals. Their cardiovascular parameters (heart rate, blood pressure, and blood flow speed) were elevated compared with the controls [[45](#B45-cancers-15-04729)]. According to the PheWeb database [[44](#B44-cancers-15-04729)], the rs140703637 SNP of the *CNTN5* gene is significantly associated with Parkinson’s disease, in which the dopamine system is known to be involved in its etiology [[46](#B46-cancers-15-04729)]; it is also known to be involved in the etiology of PONV [[47](#B47-cancers-15-04729)]. To our knowledge, no previous reports have suggested that functional changes are caused by this nonsynonymous polymorphism. Functional changes may not be easily evaluated by structural changes in the protein, which could be predicted from each amino acid sequence by the SWISS-MODEL server ([Supplementary Figure S4](#app1-cancers-15-04729)). However, our database search estimated that the possible impact of the amino-acid substitution from isoleucine to leucine, through a base substitution from A to C in the rs140703637 SNP, on the structure and function of the human protein is predicted to be “deleterious” according to the SIFT tool, suggesting that a dramatic change may be expected to result from this SNP. For *SHISA6* and *RBFOX1*, although no relationships with PONV have been reported, two SNPs in these genes, rs2908972 and rs10500355, respectively, were interestingly shown to be strongly associated with myopia (*p* = 5.000 × 10^−24^ for rs2908972; *p* = 2.000 × 10^−63^ for rs10500355) [[48](#B48-cancers-15-04729)] according to the Phenotype-Genotype Integrator (PheGenI), which is available in the NCBI database. This might imply that these two genes are commonly involved in mechanisms of myopia and nausea. Although our database search in the GTEx portal [[43](#B43-cancers-15-04729)] did not find any significant associations between the best candidate SNPs in the present study, which are mentioned above, and gene expression, the rs2256472 and rs2406801 SNPs, which were strongly linked to rs1752136 (*r2* ≥ 0.8) based on HaploReg v. 4.1 [[41](#B41-cancers-15-04729)] and the SNPinfo Web Server [[42](#B42-cancers-15-04729)], were shown to be significantly associated with the expression of two nearby genes, succinate-CoA ligase ADP-forming subunit β (*SUCLA2*) and long intergenic non-protein coding RNA 562 (*LINC00562*). This suggests that this SNP is involved in mechanisms of vomiting through the alteration of expression of these genes, although further details are unknown. A role of Atp8b3 in mouse sperm cell capacitation has been suggested [[49](#B49-cancers-15-04729)], but the possible relationship between ATP8B3 and PONV remains unknown. Moreover, the amino acid substitution from alanine to threonine, through a base substitution from G to A in the rs45574836 SNP, seemingly caused no fundamental changes in our predicted protein structures according to the SWISS-MODEL server ([Supplementary Figure S5](#app1-cancers-15-04729)). However, according to the PheWeb database, the rs45574836 SNP is moderately associated with constipation (*p* = 2.9 × 10^−3^), which is known as a major side effect of opioids, as well as nausea and vomiting [[50](#B50-cancers-15-04729)]. Furthermore, the association between the rs45574836 SNP and vomiting in the CIH samples was replicated in the JUH samples in the same genetic model in the present study ([Table 5](#cancers-15-04729-t005)), suggesting that this is a plausible candidate SNP that contributes to individual differences in the vulnerability to vomiting.

One limitation of the present study was the lack of detailed surgical records in the CIH samples; thus, no details regarding surgical procedures are described in the CIH cohort ([Table 1](#cancers-15-04729-t001)). Given that cholecystectomy, gynecological surgery, and laparoscopic procedures have been observed as surgical risk factors for PONV in addition to other anesthesia-related predictors [[8](#B8-cancers-15-04729),[9](#B9-cancers-15-04729),[10](#B10-cancers-15-04729)], the incidence of PONV may be influenced by specific types of surgeries in the subjects, suggesting that the strength of impact of certain types of surgeries as an environmental factor that influences PONV can be different among various surgeries. The lack of surgical information for each patient in the present study hampered our ability to conduct a GWAS in subjects with more homogeneity. In the present study, patients who underwent numerous types of surgeries were recruited, and the GWAS was conducted regardless of type of surgery in the CIH samples. With additional information regarding types of surgeries, further GWASs could be performed in particular subsets of patients, such as only in patients undergoing laparoscopic procedures, which was difficult in the present study because of the lack of such information. These limitations notwithstanding, the present study identified several SNPs that were significantly associated with nausea and vomiting. Regardless of the type of surgery, these SNPs might commonly impact PONV, and they were associated with vulnerability to PONV in our analyses of whole samples.

## 5. Conclusions

In conclusion, our GWASs revealed that the rs2776262, rs140703637, rs7212072, and rs12444143 SNPs, as well as the rs45574836 and rs1752136 SNPs, were significantly associated with the frequency of nausea and presence/absence of vomiting, respectively, during the postoperative period in patients who underwent elective surgery under general anesthesia with propofol or desflurane. The association between the rs45574836 SNP and vomiting was replicated in patients who underwent laparoscopic gynecological surgery. Although the present results need to be corroborated by more research with larger sample sizes and in other populations, these findings indicate that these SNPs in the *LOC100506403*, *CNTN5*, *SHISA6*, *RBFOX1*, *ATP8B3*, and *LOC105370198* gene regions could serve as markers that predict the vulnerability to PONV.

## Acknowledgments

We thank Michael Arends for editing the manuscript. We are grateful to the volunteers for their participation in the study and to the anesthesiologists and surgeons for collecting the clinical data.

## Supplementary Materials

The following supporting information can be downloaded at: [https://www.mdpi.com/article/10.3390/cancers15194729/s1](https://www.mdpi.com/article/10.3390/cancers15194729/s1). Table S1. Demographic and clinical data of patient subjects for the replication study. Table S2. Top 20 candidate SNPs selected from the GWAS for nausea in all patients. Table S3. Top 20 candidate SNPs selected from the GWAS for vomiting in all patients. Table S4. Top 20 candidate SNPs selected from the GWAS for PONV in all patients. Table S5. Top 20 candidate SNPs selected from the GWAS for nausea in patients who received propofol. Table S6. Top 20 candidate SNPs selected from the GWAS for PONV in patients who received propofol. Figure S1. Log quantile-quantile (QQ) *p*-value plot as a result of the GWAS for the frequency of nausea in all patients. Figure S2. Log quantile-quantile (QQ) *p*-value plot as a result of the GWAS for the frequency of nausea in patients with propofol. Figure S3. Log quantile-quantile (QQ) *p*-value plot as a result of the GWAS for vomiting in patients who received propofol. Figure S4. Protein structures of contactin 5 predicted from amino acid sequence (NCBI accession no. [NP_001230199.1](https://www.ncbi.nlm.nih.gov/protein/NP_001230199.1)). Figure S5. Protein structures of ATPase phospholipid transporting 8B3 predicted from amino acid sequence (NCBI accession no. [NP_001171473.1](https://www.ncbi.nlm.nih.gov/protein/NP_001171473.1)).

## Author Contributions

Conceptualization, R.M., M.Y. and K.I.; methodology, D.N., J.H., K.N. and Y.E.; software, D.N.; validation, D.N., J.H., K.N. and Y.E.; formal analysis, D.N.; investigation, D.N., S.O. and S.K.; resources, R.M., M.Y., R.I., M.H. and H.S.; data curation, D.N., J.H., K.N. and Y.E.; writing—original draft preparation, D.N.; writing—review and editing, D.N., R.M., S.O. and K.I.; visualization, D.N.; supervision, R.M., M.Y. and K.I.; project administration, K.I.; funding acquisition, D.N. and K.I. All authors have read and agreed to the published version of the manuscript.

## Institutional Review Board Statement

The study was conducted in accordance with the Declaration of Helsinki and approved by the Institutional Review Board (or Ethics Committee) of The Cancer Institute Hospital (protocol code: 2015-1063, date of approval: 25 November 2015), Juntendo University School of Medicine (protocol code: 2015053, date of approval: 18 September 2015), and Tokyo Metropolitan Institute of Medical Science (protocol code: 21–27, date of approval: 31 March 2021).

## Informed Consent Statement

Informed consent was obtained from all subjects involved in the study.

## Data Availability Statement

The data that are presented in this study are available upon request from the corresponding author.

## Conflicts of Interest

Kazutaka Ikeda has received support from Asahi Kasei Pharma Corporation and SBI Pharmaceuticals Co. Ltd., and speaker’s and consultant’s fees from MSD K.K., VistaGen Therapeutics, Inc., Atheneum Partners Otsuka Pharmaceutical Co. Ltd., Taisho Pharmaceutical Co. Ltd., Eisai, Daiichi-Sankyo, Inc., Sumitomo Pharma, Japan Tobacco, Inc., EA Pharma Co. Ltd., and Nippon Chemiphar. The authors declare no other conflict of interest. The funders had no role in the design of the study; in the collection, analyses, or interpretation of data; in writing the manuscript; or in the decision to publish the results.

## Funding Statement

This research was funded by grants from the Japan Society for the Promotion of Science (JSPS) KAKENHI (Nos. 22790518, 23390377, 24790544, 26293347, JP22H04922 [AdAMS], 17H04324, 17K08970, 18K08829, 20K09259, and 21H03028), Ministry of Health, Labour, and Welfare (MHLW) of Japan (No. H26-Kakushintekigan-ippan-060), Japan Agency for Medical Research and Development (AMED; Nos. JP19ek0610011 and JP19dk0307071), Smoking Research Foundation (Tokyo, Japan), Japan Research Foundation for Clinical Pharmacology (JRFCP), and Asahi Kasei Pharma Open Innovation. The article processing charge was funded by the Japan Society for the Promotion of Science (JSPS) KAKENHI.

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

The data that are presented in this study are available upon request from the corresponding author.

### Supplementary Materials

### Data Availability Statement

The data that are presented in this study are available upon request from the corresponding author.

## References

1. Gan T.J., Belani K.G., Bergese S., Chung F., Diemunsch P., Habib A.S., Jin Z., Kovac A.L., Meyer T.A., Urman R.D., et al. Fourth consensus guidelines for the management of postoperative nausea and vomiting. Anesth. Analg. 2020;131:411–448. doi: 10.1213/ANE.0000000000004833.  [DOI](https://doi.org/10.1213/ANE.0000000000004833) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32467512/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anesth.%20Analg.&title=Fourth%20consensus%20guidelines%20for%20the%20management%20of%20postoperative%20nausea%20and%20vomiting&author=T.J.%20Gan&author=K.G.%20Belani&author=S.%20Bergese&author=F.%20Chung&author=P.%20Diemunsch&volume=131&publication_year=2020&pages=411-448&pmid=32467512&doi=10.1213/ANE.0000000000004833&)

2. Apfel C.C., Laara E., Koivuranta M., Greim C.A., Roewer N. A simplified risk score for predicting postoperative nausea and vomiting: Conclusions from cross-validations between two centers. Anesthesiology. 1999;91:693–700. doi: 10.1097/00000542-199909000-00022.  [DOI](https://doi.org/10.1097/00000542-199909000-00022) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10485781/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anesthesiology&title=A%20simplified%20risk%20score%20for%20predicting%20postoperative%20nausea%20and%20vomiting:%20Conclusions%20from%20cross-validations%20between%20two%20centers&author=C.C.%20Apfel&author=E.%20Laara&author=M.%20Koivuranta&author=C.A.%20Greim&author=N.%20Roewer&volume=91&publication_year=1999&pages=693-700&pmid=10485781&doi=10.1097/00000542-199909000-00022&)

3. Gan T.J., Diemunsch P., Habib A.S., Kovac A., Kranke P., Meyer T.A., Watcha M., Chung F., Angus S., Apfel C.C., et al. Consensus guidelines for the management of postoperative nausea and vomiting. Anesth. Analg. 2014;118:85–113. doi: 10.1213/ANE.0000000000000002.  [DOI](https://doi.org/10.1213/ANE.0000000000000002) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24356162/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anesth.%20Analg.&title=Consensus%20guidelines%20for%20the%20management%20of%20postoperative%20nausea%20and%20vomiting&author=T.J.%20Gan&author=P.%20Diemunsch&author=A.S.%20Habib&author=A.%20Kovac&author=P.%20Kranke&volume=118&publication_year=2014&pages=85-113&pmid=24356162&doi=10.1213/ANE.0000000000000002&)

4. Parra-Sanchez I., Abdallah R., You J., Fu A.Z., Grady M., Cummings K., 3rd, Apfel C., Sessler D.I. A time-motion economic analysis of postoperative nausea and vomiting in ambulatory surgery. Can. J. Anaesth. 2012;59:366–375. doi: 10.1007/s12630-011-9660-x.  [DOI](https://doi.org/10.1007/s12630-011-9660-x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22223185/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Can.%20J.%20Anaesth.&title=A%20time-motion%20economic%20analysis%20of%20postoperative%20nausea%20and%20vomiting%20in%20ambulatory%20surgery&author=I.%20Parra-Sanchez&author=R.%20Abdallah&author=J.%20You&author=A.Z.%20Fu&author=M.%20Grady&volume=59&publication_year=2012&pages=366-375&pmid=22223185&doi=10.1007/s12630-011-9660-x&)

5. Watcha M.F., White P.F. Postoperative nausea and vomiting: Its etiology, treatment, and prevention. Anesthesiology. 1992;77:162–184. doi: 10.1097/00000542-199207000-00023.  [DOI](https://doi.org/10.1097/00000542-199207000-00023) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1609990/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anesthesiology&title=Postoperative%20nausea%20and%20vomiting:%20Its%20etiology,%20treatment,%20and%20prevention&author=M.F.%20Watcha&author=P.F.%20White&volume=77&publication_year=1992&pages=162-184&pmid=1609990&doi=10.1097/00000542-199207000-00023&)

6. Palazzo M.G., Strunin L. Anaesthesia and emesis: I. Etiology. Can. Anaesth. Soc. J. 1984;31:178–187. doi: 10.1007/BF03015257.  [DOI](https://doi.org/10.1007/BF03015257) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/6367902/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Can.%20Anaesth.%20Soc.%20J.&title=Anaesthesia%20and%20emesis:%20I.%20Etiology&author=M.G.%20Palazzo&author=L.%20Strunin&volume=31&publication_year=1984&pages=178-187&pmid=6367902&doi=10.1007/BF03015257&)

7. Camu F., Lauwers M.H., Verbessem D. Incidence and aetiology of postoperative nausea and vomiting. Eur. J. Anaesthesiol. Suppl. 1992;6:25–31.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/1425622/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Anaesthesiol.%20Suppl.&title=Incidence%20and%20aetiology%20of%20postoperative%20nausea%20and%20vomiting&author=F.%20Camu&author=M.H.%20Lauwers&author=D.%20Verbessem&volume=6&publication_year=1992&pages=25-31&pmid=1425622&)

8. Apfel C.C., Heidrich F.M., Jukar-Rao S., Jalota L., Hornuss C., Whelan R.P., Zhang K., Cakmakkaya O.S. Evidence-based analysis of risk factors for postoperative nausea and vomiting. Br. J. Anaesth. 2012;109:742–753. doi: 10.1093/bja/aes276.  [DOI](https://doi.org/10.1093/bja/aes276) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23035051/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Anaesth.&title=Evidence-based%20analysis%20of%20risk%20factors%20for%20postoperative%20nausea%20and%20vomiting&author=C.C.%20Apfel&author=F.M.%20Heidrich&author=S.%20Jukar-Rao&author=L.%20Jalota&author=C.%20Hornuss&volume=109&publication_year=2012&pages=742-753&pmid=23035051&doi=10.1093/bja/aes276&)

9. Gan T.J., Meyer T., Apfel C.C., Chung F., Davis P.J., Eubanks S., Kovac A., Philip B.K., Sessler D.I., Temo J., et al. Consensus guidelines for managing postoperative nausea and vomiting. Anesth. Analg. 2003;97:62–71. doi: 10.1213/01.ANE.0000068580.00245.95.  [DOI](https://doi.org/10.1213/01.ANE.0000068580.00245.95) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12818945/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anesth.%20Analg.&title=Consensus%20guidelines%20for%20managing%20postoperative%20nausea%20and%20vomiting&author=T.J.%20Gan&author=T.%20Meyer&author=C.C.%20Apfel&author=F.%20Chung&author=P.J.%20Davis&volume=97&publication_year=2003&pages=62-71&pmid=12818945&doi=10.1213/01.ANE.0000068580.00245.95&)

10. Morino R., Ozaki M., Nagata O., Yokota M. Incidence of and risk factors for postoperative nausea and vomiting at a Japanese Cancer Center: First large-scale study in Japan. J. Anesth. 2013;27:18–24. doi: 10.1007/s00540-012-1468-5.  [DOI](https://doi.org/10.1007/s00540-012-1468-5) | [PMC free article](/articles/PMC3574566/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22923285/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Anesth.&title=Incidence%20of%20and%20risk%20factors%20for%20postoperative%20nausea%20and%20vomiting%20at%20a%20Japanese%20Cancer%20Center:%20First%20large-scale%20study%20in%20Japan&author=R.%20Morino&author=M.%20Ozaki&author=O.%20Nagata&author=M.%20Yokota&volume=27&publication_year=2013&pages=18-24&pmid=22923285&doi=10.1007/s00540-012-1468-5&)

11. Klenke S., Frey U.H. Genetic variability in postoperative nausea and vomiting: A systematic review. Eur. J. Anaesthesiol. 2020;37:959–968. doi: 10.1097/EJA.0000000000001224.  [DOI](https://doi.org/10.1097/EJA.0000000000001224) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32371828/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Anaesthesiol.&title=Genetic%20variability%20in%20postoperative%20nausea%20and%20vomiting:%20A%20systematic%20review&author=S.%20Klenke&author=U.H.%20Frey&volume=37&publication_year=2020&pages=959-968&pmid=32371828&doi=10.1097/EJA.0000000000001224&)

12. Janicki P.K., Sugino S. Genetic factors associated with pharmacotherapy and background sensitivity to postoperative and chemotherapy-induced nausea and vomiting. Exp. Brain Res. 2014;232:2613–2625. doi: 10.1007/s00221-014-3968-z.  [DOI](https://doi.org/10.1007/s00221-014-3968-z) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24792505/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Exp.%20Brain%20Res.&title=Genetic%20factors%20associated%20with%20pharmacotherapy%20and%20background%20sensitivity%20to%20postoperative%20and%20chemotherapy-induced%20nausea%20and%20vomiting&author=P.K.%20Janicki&author=S.%20Sugino&volume=232&publication_year=2014&pages=2613-2625&pmid=24792505&doi=10.1007/s00221-014-3968-z&)

13. Lopez-Morales P., Flores-Funes D., Sanchez-Migallon E.G., Liron-Ruiz R.J., Aguayo-Albasini J.L. Genetic factors associated with postoperative nausea and vomiting: A systematic review. J. Gastrointest. Surg. 2018;22:1645–1651. doi: 10.1007/s11605-018-3788-8.  [DOI](https://doi.org/10.1007/s11605-018-3788-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29725907/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Gastrointest.%20Surg.&title=Genetic%20factors%20associated%20with%20postoperative%20nausea%20and%20vomiting:%20A%20systematic%20review&author=P.%20Lopez-Morales&author=D.%20Flores-Funes&author=E.G.%20Sanchez-Migallon&author=R.J.%20Liron-Ruiz&author=J.L.%20Aguayo-Albasini&volume=22&publication_year=2018&pages=1645-1651&pmid=29725907&doi=10.1007/s11605-018-3788-8&)

14. Klenke S., de Vries G.J., Schiefer L., Seyffert N., Bachmann H.S., Peters J., Frey U.H. Genetic contribution to PONV risk. Anaesth. Crit. Care Pain Med. 2020;39:45–51. doi: 10.1016/j.accpm.2019.04.012.  [DOI](https://doi.org/10.1016/j.accpm.2019.04.012) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31077873/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anaesth.%20Crit.%20Care%20Pain%20Med.&title=Genetic%20contribution%20to%20PONV%20risk&author=S.%20Klenke&author=G.J.%20de%20Vries&author=L.%20Schiefer&author=N.%20Seyffert&author=H.S.%20Bachmann&volume=39&publication_year=2020&pages=45-51&pmid=31077873&doi=10.1016/j.accpm.2019.04.012&)

15. Joy Lin Y.M., Hsu C.D., Hsieh H.Y., Tseng C.C., Sun H.S. Sequence variants of the HTR3A gene contribute to the genetic prediction of postoperative nausea in Taiwan. J. Hum. Genet. 2014;59:655–660. doi: 10.1038/jhg.2014.89.  [DOI](https://doi.org/10.1038/jhg.2014.89) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25339231/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Hum.%20Genet.&title=Sequence%20variants%20of%20the%20HTR3A%20gene%20contribute%20to%20the%20genetic%20prediction%20of%20postoperative%20nausea%20in%20Taiwan&author=Y.M.%20Joy%20Lin&author=C.D.%20Hsu&author=H.Y.%20Hsieh&author=C.C.%20Tseng&author=H.S.%20Sun&volume=59&publication_year=2014&pages=655-660&pmid=25339231&doi=10.1038/jhg.2014.89&)

16. Rueffert H., Thieme V., Wallenborn J., Lemnitz N., Bergmann A., Rudlof K., Wehner M., Olthoff D., Kaisers U.X. Do variations in the 5-HT3A and 5-HT3B serotonin receptor genes (HTR3A and HTR3B) influence the occurrence of postoperative vomiting? Anesth. Analg. 2009;109:1442–1447. doi: 10.1213/ane.0b013e3181b2359b.  [DOI](https://doi.org/10.1213/ane.0b013e3181b2359b) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19713259/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anesth.%20Analg.&title=Do%20variations%20in%20the%205-HT3A%20and%205-HT3B%20serotonin%20receptor%20genes%20(HTR3A%20and%20HTR3B)%20influence%20the%20occurrence%20of%20postoperative%20vomiting?&author=H.%20Rueffert&author=V.%20Thieme&author=J.%20Wallenborn&author=N.%20Lemnitz&author=A.%20Bergmann&volume=109&publication_year=2009&pages=1442-1447&pmid=19713259&doi=10.1213/ane.0b013e3181b2359b&)

17. Ma X.X., Chen Q.X., Wu S.J., Hu Y., Fang X.M. Polymorphisms of the HTR3B gene are associated with post-surgery emesis in a Chinese Han population. J. Clin. Pharm. Ther. 2013;38:150–155. doi: 10.1111/jcpt.12033.  [DOI](https://doi.org/10.1111/jcpt.12033) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23464988/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Pharm.%20Ther.&title=Polymorphisms%20of%20the%20HTR3B%20gene%20are%20associated%20with%20post-surgery%20emesis%20in%20a%20Chinese%20Han%20population&author=X.X.%20Ma&author=Q.X.%20Chen&author=S.J.%20Wu&author=Y.%20Hu&author=X.M.%20Fang&volume=38&publication_year=2013&pages=150-155&pmid=23464988&doi=10.1111/jcpt.12033&)

18. Kim M.S., Lee J.R., Choi E.M., Kim E.H., Choi S.H. Association of 5-HT3B receptor gene polymorphisms with the efficacy of ondansetron for postoperative nausea and vomiting. Yonsei Med. J. 2015;56:1415–1420. doi: 10.3349/ymj.2015.56.5.1415.  [DOI](https://doi.org/10.3349/ymj.2015.56.5.1415) | [PMC free article](/articles/PMC4541676/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26256989/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Yonsei%20Med.%20J.&title=Association%20of%205-HT3B%20receptor%20gene%20polymorphisms%20with%20the%20efficacy%20of%20ondansetron%20for%20postoperative%20nausea%20and%20vomiting&author=M.S.%20Kim&author=J.R.%20Lee&author=E.M.%20Choi&author=E.H.%20Kim&author=S.H.%20Choi&volume=56&publication_year=2015&pages=1415-1420&pmid=26256989&doi=10.3349/ymj.2015.56.5.1415&)

19. Frey U.H., Schnee C., Achilles M., Silvanus M.T., Esser J., Peters J. Postoperative nausea and vomiting: The role of the dopamine D2 receptor TaqIA polymorphism. Eur. J. Anaesthesiol. 2016;33:84–89. doi: 10.1097/EJA.0000000000000320.  [DOI](https://doi.org/10.1097/EJA.0000000000000320) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26225495/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Anaesthesiol.&title=Postoperative%20nausea%20and%20vomiting:%20The%20role%20of%20the%20dopamine%20D2%20receptor%20TaqIA%20polymorphism&author=U.H.%20Frey&author=C.%20Schnee&author=M.%20Achilles&author=M.T.%20Silvanus&author=J.%20Esser&volume=33&publication_year=2016&pages=84-89&pmid=26225495&doi=10.1097/EJA.0000000000000320&)

20. Nakagawa M., Kuri M., Kambara N., Tanigami H., Tanaka H., Kishi Y., Hamajima N. Dopamine D2 receptor Taq IA polymorphism is associated with postoperative nausea and vomiting. J. Anesth. 2008;22:397–403. doi: 10.1007/s00540-008-0661-z.  [DOI](https://doi.org/10.1007/s00540-008-0661-z) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19011779/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Anesth.&title=Dopamine%20D2%20receptor%20Taq%20IA%20polymorphism%20is%20associated%20with%20postoperative%20nausea%20and%20vomiting&author=M.%20Nakagawa&author=M.%20Kuri&author=N.%20Kambara&author=H.%20Tanigami&author=H.%20Tanaka&volume=22&publication_year=2008&pages=397-403&pmid=19011779&doi=10.1007/s00540-008-0661-z&)

21. Lee S.H., Kim J.D., Park S.A., Oh C.S., Kim S.H. Effects of μ-opioid receptor gene polymorphism on postoperative nausea and vomiting in patients undergoing general anesthesia with remifentanil: Double blinded randomized trial. J. Korean Med. Sci. 2015;30:651–657. doi: 10.3346/jkms.2015.30.5.651.  [DOI](https://doi.org/10.3346/jkms.2015.30.5.651) | [PMC free article](/articles/PMC4414652/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25931799/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Korean%20Med.%20Sci.&title=Effects%20of%20%CE%BC-opioid%20receptor%20gene%20polymorphism%20on%20postoperative%20nausea%20and%20vomiting%20in%20patients%20undergoing%20general%20anesthesia%20with%20remifentanil:%20Double%20blinded%20randomized%20trial&author=S.H.%20Lee&author=J.D.%20Kim&author=S.A.%20Park&author=C.S.%20Oh&author=S.H.%20Kim&volume=30&publication_year=2015&pages=651-657&pmid=25931799&doi=10.3346/jkms.2015.30.5.651&)

22. Sugino S., Hayase T., Higuchi M., Saito K., Moriya H., Kumeta Y., Kurosawa N., Namiki A., Janicki P.K. Association of μ-opioid receptor gene (OPRM1) haplotypes with postoperative nausea and vomiting. Exp. Brain Res. 2014;232:2627–2635. doi: 10.1007/s00221-014-3987-9.  [DOI](https://doi.org/10.1007/s00221-014-3987-9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24858579/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Exp.%20Brain%20Res.&title=Association%20of%20%CE%BC-opioid%20receptor%20gene%20(OPRM1)%20haplotypes%20with%20postoperative%20nausea%20and%20vomiting&author=S.%20Sugino&author=T.%20Hayase&author=M.%20Higuchi&author=K.%20Saito&author=H.%20Moriya&volume=232&publication_year=2014&pages=2627-2635&pmid=24858579&doi=10.1007/s00221-014-3987-9&)

23. Hayase T., Sugino S., Moriya H., Yamakage M. TACR1 gene polymorphism and sex differences in postoperative nausea and vomiting. Anaesthesia. 2015;70:1148–1159. doi: 10.1111/anae.13082.  [DOI](https://doi.org/10.1111/anae.13082) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26012530/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anaesthesia&title=TACR1%20gene%20polymorphism%20and%20sex%20differences%20in%20postoperative%20nausea%20and%20vomiting&author=T.%20Hayase&author=S.%20Sugino&author=H.%20Moriya&author=M.%20Yamakage&volume=70&publication_year=2015&pages=1148-1159&pmid=26012530&doi=10.1111/anae.13082&)

24. Wesmiller S.W., Bender C.M., Sereika S.M., Ahrendt G., Bonaventura M., Bovbjerg D.H., Conley Y. Association between serotonin transport polymorphisms and postdischarge nausea and vomiting in women following breast cancer surgery. Oncol. Nurs. Forum. 2014;41:195–202. doi: 10.1188/14.ONF.195-202.  [DOI](https://doi.org/10.1188/14.ONF.195-202) | [PMC free article](/articles/PMC4023462/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24578078/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncol.%20Nurs.%20Forum&title=Association%20between%20serotonin%20transport%20polymorphisms%20and%20postdischarge%20nausea%20and%20vomiting%20in%20women%20following%20breast%20cancer%20surgery&author=S.W.%20Wesmiller&author=C.M.%20Bender&author=S.M.%20Sereika&author=G.%20Ahrendt&author=M.%20Bonaventura&volume=41&publication_year=2014&pages=195-202&pmid=24578078&doi=10.1188/14.ONF.195-202&)

25. Choi E.M., Lee M.G., Lee S.H., Choi K.W., Choi S.H. Association of ABCB1 polymorphisms with the efficacy of ondansetron for postoperative nausea and vomiting. Anaesthesia. 2010;65:996–1000. doi: 10.1111/j.1365-2044.2010.06476.x.  [DOI](https://doi.org/10.1111/j.1365-2044.2010.06476.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20707787/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anaesthesia&title=Association%20of%20ABCB1%20polymorphisms%20with%20the%20efficacy%20of%20ondansetron%20for%20postoperative%20nausea%20and%20vomiting&author=E.M.%20Choi&author=M.G.%20Lee&author=S.H.%20Lee&author=K.W.%20Choi&author=S.H.%20Choi&volume=65&publication_year=2010&pages=996-1000&pmid=20707787&doi=10.1111/j.1365-2044.2010.06476.x&)

26. Farhat K., Iqbal J., Waheed A., Mansoor Q., Ismail M., Pasha A.K. Association of anti-emetic efficacy of ondansetron with G2677T polymorphism in a drug transporter gene ABCB1 in Pakistani population. J. Coll. Physicians Surg. Pak. 2015;25:486–490.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/26208549/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Coll.%20Physicians%20Surg.%20Pak.&title=Association%20of%20anti-emetic%20efficacy%20of%20ondansetron%20with%20G2677T%20polymorphism%20in%20a%20drug%20transporter%20gene%20ABCB1%20in%20Pakistani%20population&author=K.%20Farhat&author=J.%20Iqbal&author=A.%20Waheed&author=Q.%20Mansoor&author=M.%20Ismail&volume=25&publication_year=2015&pages=486-490&pmid=26208549&)

27. Dzambazovska-Trajkovska V., Nojkov J., Kartalov A., Kuzmanovska B., Spiroska T., Seljmani R., Trajkovski G., Matevska-Geshkovska N., Dimovski A. Association of single-nucleotide polymorhism C3435T in the ABCB1 gene with opioid sensitivity in treatment of postoperative pain. Pril. 2016;37:73–80. doi: 10.1515/prilozi-2016-0019.  [DOI](https://doi.org/10.1515/prilozi-2016-0019) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27883323/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pril&title=Association%20of%20single-nucleotide%20polymorhism%20C3435T%20in%20the%20ABCB1%20gene%20with%20opioid%20sensitivity%20in%20treatment%20of%20postoperative%20pain&author=V.%20Dzambazovska-Trajkovska&author=J.%20Nojkov&author=A.%20Kartalov&author=B.%20Kuzmanovska&author=T.%20Spiroska&volume=37&publication_year=2016&pages=73-80&pmid=27883323&doi=10.1515/prilozi-2016-0019&)

28. Balyan R., Zhang X., Chidambaran V., Martin L.J., Mizuno T., Fukuda T., Vinks A.A., Sadhasivam S. OCT1 genetic variants are associated with postoperative morphine-related adverse effects in children. Pharmacogenomics. 2017;18:621–629. doi: 10.2217/pgs-2017-0002.  [DOI](https://doi.org/10.2217/pgs-2017-0002) | [PMC free article](/articles/PMC5591462/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28470102/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=OCT1%20genetic%20variants%20are%20associated%20with%20postoperative%20morphine-related%20adverse%20effects%20in%20children&author=R.%20Balyan&author=X.%20Zhang&author=V.%20Chidambaran&author=L.J.%20Martin&author=T.%20Mizuno&volume=18&publication_year=2017&pages=621-629&pmid=28470102&doi=10.2217/pgs-2017-0002&)

29. Candiotti K.A., Birnbach D.J., Lubarsky D.A., Nhuch F., Kamat A., Koch W.H., Nikoloff M., Wu L., Andrews D. The impact of pharmacogenomics on postoperative nausea and vomiting: Do CYP2D6 allele copy number and polymorphisms affect the success or failure of ondansetron prophylaxis? Anesthesiology. 2005;102:543–549. doi: 10.1097/00000542-200503000-00011.  [DOI](https://doi.org/10.1097/00000542-200503000-00011) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15731591/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anesthesiology&title=The%20impact%20of%20pharmacogenomics%20on%20postoperative%20nausea%20and%20vomiting:%20Do%20CYP2D6%20allele%20copy%20number%20and%20polymorphisms%20affect%20the%20success%20or%20failure%20of%20ondansetron%20prophylaxis?&author=K.A.%20Candiotti&author=D.J.%20Birnbach&author=D.A.%20Lubarsky&author=F.%20Nhuch&author=A.%20Kamat&volume=102&publication_year=2005&pages=543-549&pmid=15731591&doi=10.1097/00000542-200503000-00011&)

30. Janicki P.K., Schuler H.G., Jarzembowski T.M., Rossi M., 2nd Prevention of postoperative nausea and vomiting with granisetron and dolasetron in relation to CYP2D6 genotype. Anesth. Analg. 2006;102:1127–1133. doi: 10.1213/01.ane.0000200364.55798.3f.  [DOI](https://doi.org/10.1213/01.ane.0000200364.55798.3f) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16551910/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anesth.%20Analg.&title=Prevention%20of%20postoperative%20nausea%20and%20vomiting%20with%20granisetron%20and%20dolasetron%20in%20relation%20to%20CYP2D6%20genotype&author=P.K.%20Janicki&author=H.G.%20Schuler&author=T.M.%20Jarzembowski&author=M.%20Rossi&volume=102&publication_year=2006&pages=1127-1133&pmid=16551910&doi=10.1213/01.ane.0000200364.55798.3f&)

31. Wesmiller S.W., Henker R.A., Sereika S.M., Donovan H.S., Meng L., Gruen G.S., Tarkin I.S., Conley Y.P. The association of CYP2D6 genotype and postoperative nausea and vomiting in orthopedic trauma patients. Biol. Res. Nurs. 2013;15:382–389. doi: 10.1177/1099800412449181.  [DOI](https://doi.org/10.1177/1099800412449181) | [PMC free article](/articles/PMC4028967/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22718526/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biol.%20Res.%20Nurs.&title=The%20association%20of%20CYP2D6%20genotype%20and%20postoperative%20nausea%20and%20vomiting%20in%20orthopedic%20trauma%20patients&author=S.W.%20Wesmiller&author=R.A.%20Henker&author=S.M.%20Sereika&author=H.S.%20Donovan&author=L.%20Meng&volume=15&publication_year=2013&pages=382-389&pmid=22718526&doi=10.1177/1099800412449181&)

32. Janicki P.K., Vealey R., Liu J., Escajeda J., Postula M., Welker K. Genome-wide association study using pooled DNA to identify candidate markers mediating susceptibility to postoperative nausea and vomiting. Anesthesiology. 2011;115:54–64. doi: 10.1097/ALN.0b013e31821810c7.  [DOI](https://doi.org/10.1097/ALN.0b013e31821810c7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21694509/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anesthesiology&title=Genome-wide%20association%20study%20using%20pooled%20DNA%20to%20identify%20candidate%20markers%20mediating%20susceptibility%20to%20postoperative%20nausea%20and%20vomiting&author=P.K.%20Janicki&author=R.%20Vealey&author=J.%20Liu&author=J.%20Escajeda&author=M.%20Postula&volume=115&publication_year=2011&pages=54-64&pmid=21694509&doi=10.1097/ALN.0b013e31821810c7&)

33. Klenke S., de Vries G.J., Schiefer L., Seyffert N., Bachmann H.S., Peters J., Frey U.H. CHRM3 rs2165870 polymorphism is independently associated with postoperative nausea and vomiting, but combined prophylaxis is effective. Br. J. Anaesth. 2018;121:58–65. doi: 10.1016/j.bja.2018.02.025.  [DOI](https://doi.org/10.1016/j.bja.2018.02.025) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29935595/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Anaesth.&title=CHRM3%20rs2165870%20polymorphism%20is%20independently%20associated%20with%20postoperative%20nausea%20and%20vomiting,%20but%20combined%20prophylaxis%20is%20effective&author=S.%20Klenke&author=G.J.%20de%20Vries&author=L.%20Schiefer&author=N.%20Seyffert&author=H.S.%20Bachmann&volume=121&publication_year=2018&pages=58-65&pmid=29935595&doi=10.1016/j.bja.2018.02.025&)

34. Ahlstrom S.E., Bergman P.H., Jokela R.M., Olkkola K.T., Kaunisto M.A., Kalso E.A. Clinical and genetic factors associated with post-operative nausea and vomiting after propofol anaesthesia. Acta Anaesthesiol. Scand. 2023;67:1018–1027. doi: 10.1111/aas.14261.  [DOI](https://doi.org/10.1111/aas.14261) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/37156489/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Acta%20Anaesthesiol.%20Scand.&title=Clinical%20and%20genetic%20factors%20associated%20with%20post-operative%20nausea%20and%20vomiting%20after%20propofol%20anaesthesia&author=S.E.%20Ahlstrom&author=P.H.%20Bergman&author=R.M.%20Jokela&author=K.T.%20Olkkola&author=M.A.%20Kaunisto&volume=67&publication_year=2023&pages=1018-1027&pmid=37156489&doi=10.1111/aas.14261&)

35. Sugino S., Konno D., Kawai Y., Nagasaki M., Endo Y., Hayase T., Yamazaki-Higuchi M., Kumeta Y., Tachibana S., Saito K., et al. Long non-coding RNA MIR4300HG polymorphisms are associated with postoperative nausea and vomiting: A genome-wide association study. Hum. Genom. 2020;14:31. doi: 10.1186/s40246-020-00282-4.  [DOI](https://doi.org/10.1186/s40246-020-00282-4) | [PMC free article](/articles/PMC7491086/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32928300/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum.%20Genom.&title=Long%20non-coding%20RNA%20MIR4300HG%20polymorphisms%20are%20associated%20with%20postoperative%20nausea%20and%20vomiting:%20A%20genome-wide%20association%20study&author=S.%20Sugino&author=D.%20Konno&author=Y.%20Kawai&author=M.%20Nagasaki&author=Y.%20Endo&volume=14&publication_year=2020&pages=31&pmid=32928300&doi=10.1186/s40246-020-00282-4&)

36. Watanabe K., Taskesen E., van Bochoven A., Posthuma D. Functional mapping and annotation of genetic associations with FUMA. Nat. Commun. 2017;8:1826. doi: 10.1038/s41467-017-01261-5.  [DOI](https://doi.org/10.1038/s41467-017-01261-5) | [PMC free article](/articles/PMC5705698/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29184056/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&title=Functional%20mapping%20and%20annotation%20of%20genetic%20associations%20with%20FUMA&author=K.%20Watanabe&author=E.%20Taskesen&author=A.%20van%20Bochoven&author=D.%20Posthuma&volume=8&publication_year=2017&pages=1826&pmid=29184056&doi=10.1038/s41467-017-01261-5&)

37. Yeo I., Johnson R. A new family of power transformations to improve normality or symmetry. Biometrika. 2000;87:954–959. doi: 10.1093/biomet/87.4.954.  [DOI](https://doi.org/10.1093/biomet/87.4.954) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biometrika&title=A%20new%20family%20of%20power%20transformations%20to%20improve%20normality%20or%20symmetry&author=I.%20Yeo&author=R.%20Johnson&volume=87&publication_year=2000&pages=954-959&doi=10.1093/biomet/87.4.954&)

38. Purcell S., Neale B., Todd-Brown K., Thomas L., Ferreira M.A., Bender D., Maller J., Sklar P., de Bakker P.I., Daly M.J., et al. PLINK: A tool set for whole-genome association and population-based linkage analyses. Am. J. Hum. Genet. 2007;81:559–575. doi: 10.1086/519795.  [DOI](https://doi.org/10.1086/519795) | [PMC free article](/articles/PMC1950838/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17701901/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Hum.%20Genet.&title=PLINK:%20A%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses&author=S.%20Purcell&author=B.%20Neale&author=K.%20Todd-Brown&author=L.%20Thomas&author=M.A.%20Ferreira&volume=81&publication_year=2007&pages=559-575&pmid=17701901&doi=10.1086/519795&)

39. Barrett J.C., Fry B., Maller J., Daly M.J. Haploview: Analysis and visualization of LD and haplotype maps. Bioinformatics. 2005;21:263–265. doi: 10.1093/bioinformatics/bth457.  [DOI](https://doi.org/10.1093/bioinformatics/bth457) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15297300/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=Haploview:%20Analysis%20and%20visualization%20of%20LD%20and%20haplotype%20maps&author=J.C.%20Barrett&author=B.%20Fry&author=J.%20Maller&author=M.J.%20Daly&volume=21&publication_year=2005&pages=263-265&pmid=15297300&doi=10.1093/bioinformatics/bth457&)

40. Faul F., Erdfelder E., Lang A.G., Buchner A. G*Power 3: A flexible statistical power analysis program for the social, behavioral, and biomedical sciences. Behav. Res. Methods. 2007;39:175–191. doi: 10.3758/bf03193146.  [DOI](https://doi.org/10.3758/bf03193146) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17695343/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Behav.%20Res.%20Methods&title=G*Power%203:%20A%20flexible%20statistical%20power%20analysis%20program%20for%20the%20social,%20behavioral,%20and%20biomedical%20sciences&author=F.%20Faul&author=E.%20Erdfelder&author=A.G.%20Lang&author=A.%20Buchner&volume=39&publication_year=2007&pages=175-191&pmid=17695343&doi=10.3758/bf03193146&)

41. Ward L.D., Kellis M. HaploReg: A resource for exploring chromatin states, conservation, and regulatory motif alterations within sets of genetically linked variants. Nucleic Acids Res. 2012;40:D930–D934. doi: 10.1093/nar/gkr917.  [DOI](https://doi.org/10.1093/nar/gkr917) | [PMC free article](/articles/PMC3245002/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22064851/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&title=HaploReg:%20A%20resource%20for%20exploring%20chromatin%20states,%20conservation,%20and%20regulatory%20motif%20alterations%20within%20sets%20of%20genetically%20linked%20variants&author=L.D.%20Ward&author=M.%20Kellis&volume=40&publication_year=2012&pages=D930-D934&pmid=22064851&doi=10.1093/nar/gkr917&)

42. Xu Z., Taylor J.A. SNPinfo: Integrating GWAS and candidate gene information into functional SNP selection for genetic association studies. Nucleic Acids Res. 2009;37:W600–W605. doi: 10.1093/nar/gkp290.  [DOI](https://doi.org/10.1093/nar/gkp290) | [PMC free article](/articles/PMC2703930/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19417063/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&title=SNPinfo:%20Integrating%20GWAS%20and%20candidate%20gene%20information%20into%20functional%20SNP%20selection%20for%20genetic%20association%20studies&author=Z.%20Xu&author=J.A.%20Taylor&volume=37&publication_year=2009&pages=W600-W605&pmid=19417063&doi=10.1093/nar/gkp290&)

43. GTEx Consortium The Genotype-Tissue Expression (GTEx) project. Nat. Genet. 2013;45:580–585. doi: 10.1038/ng.2653.  [DOI](https://doi.org/10.1038/ng.2653) | [PMC free article](/articles/PMC4010069/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23715323/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet.&title=The%20Genotype-Tissue%20Expression%20(GTEx)%20project&volume=45&publication_year=2013&pages=580-585&pmid=23715323&doi=10.1038/ng.2653&)

44. Sakaue S., Kanai M., Tanigawa Y., Karjalainen J., Kurki M., Koshiba S., Narita A., Konuma T., Yamamoto K., Akiyama M., et al. A cross-population atlas of genetic associations for 220 human phenotypes. Nat. Genet. 2021;53:1415–1424. doi: 10.1038/s41588-021-00931-x.  [DOI](https://doi.org/10.1038/s41588-021-00931-x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34594039/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet.&title=A%20cross-population%20atlas%20of%20genetic%20associations%20for%20220%20human%20phenotypes&author=S.%20Sakaue&author=M.%20Kanai&author=Y.%20Tanigawa&author=J.%20Karjalainen&author=M.%20Kurki&volume=53&publication_year=2021&pages=1415-1424&pmid=34594039&doi=10.1038/s41588-021-00931-x&)

45. Smirnov A.V., Kontsevaya G.V., Feofanova N.A., Anisimova M.V., Serova I.A., Gerlinskaya L.A., Battulin N.R., Moshkin M.P., Serov O.L. Unexpected phenotypic effects of a transgene integration causing a knockout of the endogenous Contactin-5 gene in mice. Transgenic Res. 2018;27:1–13. doi: 10.1007/s11248-017-0053-y.  [DOI](https://doi.org/10.1007/s11248-017-0053-y) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29264679/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Transgenic%20Res.&title=Unexpected%20phenotypic%20effects%20of%20a%20transgene%20integration%20causing%20a%20knockout%20of%20the%20endogenous%20Contactin-5%20gene%20in%20mice&author=A.V.%20Smirnov&author=G.V.%20Kontsevaya&author=N.A.%20Feofanova&author=M.V.%20Anisimova&author=I.A.%20Serova&volume=27&publication_year=2018&pages=1-13&pmid=29264679&doi=10.1007/s11248-017-0053-y&)

46. Kalia L.V., Lang A.E. Parkinson’s disease. Lancet. 2015;386:896–912. doi: 10.1016/S0140-6736(14)61393-3.  [DOI](https://doi.org/10.1016/S0140-6736(14)61393-3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25904081/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Parkinson%E2%80%99s%20disease&author=L.V.%20Kalia&author=A.E.%20Lang&volume=386&publication_year=2015&pages=896-912&pmid=25904081&doi=10.1016/S0140-6736(14)61393-3&)

47. Moon Y.E. Postoperative nausea and vomiting. Korean J. Anesthesiol. 2014;67:164–170. doi: 10.4097/kjae.2014.67.3.164.  [DOI](https://doi.org/10.4097/kjae.2014.67.3.164) | [PMC free article](/articles/PMC4188760/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25302092/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Korean%20J.%20Anesthesiol.&title=Postoperative%20nausea%20and%20vomiting&author=Y.E.%20Moon&volume=67&publication_year=2014&pages=164-170&pmid=25302092&doi=10.4097/kjae.2014.67.3.164&)

48. Pickrell J.K., Berisa T., Liu J.Z., Segurel L., Tung J.Y., Hinds D.A. Detection and interpretation of shared genetic influences on 42 human traits. Nat. Genet. 2016;48:709–717. doi: 10.1038/ng.3570.  [DOI](https://doi.org/10.1038/ng.3570) | [PMC free article](/articles/PMC5207801/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27182965/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet.&title=Detection%20and%20interpretation%20of%20shared%20genetic%20influences%20on%2042%20human%20traits&author=J.K.%20Pickrell&author=T.%20Berisa&author=J.Z.%20Liu&author=L.%20Segurel&author=J.Y.%20Tung&volume=48&publication_year=2016&pages=709-717&pmid=27182965&doi=10.1038/ng.3570&)

49. Folmer D.E., Oude Elferink R.P.J., Paulusma C.C. P4 ATPases: Lipid flippases and their role in disease. Biochim. Biophys. Acta. 2009;1791:628–635. doi: 10.1016/j.bbalip.2009.02.008.  [DOI](https://doi.org/10.1016/j.bbalip.2009.02.008) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19254779/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochim.%20Biophys.%20Acta&title=P4%20ATPases:%20Lipid%20flippases%20and%20their%20role%20in%20disease&author=D.E.%20Folmer&author=R.P.J.%20Oude%20Elferink&author=C.C.%20Paulusma&volume=1791&publication_year=2009&pages=628-635&pmid=19254779&doi=10.1016/j.bbalip.2009.02.008&)

50. Benyamin R., Trescot A.M., Datta S., Buenaventura R., Adlaka R., Sehgal N., Glaser S.E., Vallejo R. Opioid complications and side effects. Pain Physician. 2008;11((Suppl. 2)):S105–S120. doi: 10.36076/ppj.2008/11/S105.  [DOI](https://doi.org/10.36076/ppj.2008/11/S105) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18443635/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pain%20Physician&title=Opioid%20complications%20and%20side%20effects&author=R.%20Benyamin&author=A.M.%20Trescot&author=S.%20Datta&author=R.%20Buenaventura&author=R.%20Adlaka&volume=11&issue=(Suppl.%202)&publication_year=2008&pages=S105-S120&pmid=18443635&doi=10.36076/ppj.2008/11/S105&)
