# Improving single nucleotide polymorphisms genotyping accuracy for dihydropyrimidine dehydrogenase testing in pharmacogenetics

## Metadata
**Authors:** Annalaura Montella, Sueva Cantalupo, Giuseppe D’Alterio, Vincenzo Damiano, Achille Iolascon, Mario Capasso
**Journal:** Exploration of Targeted Anti-tumor Therapy
**Date:** 2024 Apr 24
**DOI:** [10.37349/etat.2024.00223](https://doi.org/10.37349/etat.2024.00223)
**PMID:** 38745766
**PMCID:** PMC11090686
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11090686/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC11090686/pdf/etat-05-1002223.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC11090686/pdf/etat-05-1002223.pdf)

## Abstract

Fluoropyrimidines, crucial in cancer treatment, often cause toxicity concerns even at standard doses. Toxic accumulation of fluoropyrimidine metabolites, culminating in adverse effects, can stem from impaired dihydropyrimidine dehydrogenase (DPYD) enzymatic function. Emerging evidence underscores the role of single nucleotide polymorphisms (SNPs) in DPYD gene, capable of inducing DPYD activity deficiency. Consequently, DPYD genotyping’s importance is on the rise in clinical practice before initiating fluoropyrimidine treatment. Although polymerase chain reaction (PCR) followed by Sanger sequencing (SS; PCR-SS) is a prevalent method for DPYD genotyping, it may encounter limitations. In this context, there is reported a case in which a routine PCR-SS approach for genotyping DPYD SNP rs55886062 failed in a proband of African descent. The Clinical Pharmacogenetics Implementation Consortium (CPIC) categorizes the guanine (G) allele of this SNP as non-functional. The enforcement of whole genome sequencing (WGS) approach led to the identification of two adenine (A) insertions near the PCR primers annealing regions in the proband, responsible for a sequence frameshift and a genotyping error for rs55886062. These SNPs (rs145228578, 1-97981199-T-TA and rs141050810, 1-97981622-G-GA) were extremely rare in non-Finnish Europeans (0.05%) but prevalent in African populations (16%). Although limited evidence was available for these SNPs, they were catalogued as benign variants in public databases. Notably, these two SNPs exhibited a high linkage disequilibrium [LD; squared correlation coefficient (R2) = 0.98]. These findings highlighted the importance to consider the prevalence of genetic variants within diverse ethnic populations when designing primers and probes for SNP genotyping in pharmacogenetic testing. This preventive measure is essential to avoid sequence frameshifts or primer misalignments arising from SNP occurrences in the genome, which can compromise PCR-SS and lead to genotyping failures. Furthermore, this case highlights the significance of exploring alternative genotyping approaches, like WGS, when confronted with challenges associated with conventional techniques.

Keywords: Pharmacogenetics, dihydropyrimidine dehydrogenase, genotyping

## Introduction

Fluoropyrimidines, including 5-fluorouracil and its pre-prodrug capecitabine, are frequently used for the treatment of pancreatic, colorectal, breast, gastric, head and neck cancers [[1](#B1)]. Although these drugs represent safe and effective chemotherapeutics, they frequently lead to the development of toxicity even at standard doses, recording up to 30% risk of severe adverse events up to date [[2](#B2)].

One of the causes of this toxicity is the deficiency of the enzyme dihydropyrimidine dehydrogenase (DPYD), encoded by *DPYD* gene and involved in fluoropyrimidines catabolism. An impairment of DPYD enzymatic function causes a toxic accumulation of fluoropyrimidine metabolites with consequent implication in terms of adverse events [[3](#B3)–[6](#B6)]. Recent evidence ascribes to germline pathogenic variants in the *DPYD* gene a crucial role in the development of DPYD deficiency, leading to a reduction in DPYD activity [[7](#B7)–[9](#B9)]. As a result, it has become increasingly common to perform *DPYD* genotyping prior to initiating treatment with fluoropyrimidines. This practice is aimed to the essential purpose of mitigating adverse events and minimizing treatment-related toxicity. By identifying individuals with these genetic variants in advance, healthcare providers can tailor treatment plans and dosages to enhance both the safety and efficacy of fluoropyrimidine-based therapies.

In light of these advancements, numerous medical agencies worldwide have issued guidelines recommending upfront *DPYD* testing for patients scheduled to receive fluoropyrimidine-based drugs [[10](#B10)–[13](#B13)]. The Clinical Pharmacogenetics Implementation Consortium (CPIC) has played a prominent role in this regard, setting guidelines for *DPYD* genotyping when determining fluoropyrimidine dosing. These guidelines emphasize four specific single nucleotide polymorphisms (SNPs) within the *DPYD* gene: c.1905 + 1guanine (G) > adenine (A; rs3918290), c.1679thymidine (T) > G (rs55886062), c.2846A > T (rs67376798), and c.1236G > A (rs56038477), with the latter serving as the tag SNP for haplotype B3 (HapB3), in association with rs75017182 c.1129 – 5923cytosine (C) > G [[11](#B11)]. These SNPs have been singled out due to their significant prevalence in the population and their established impact on enzyme function and the risk of treatment-related toxicity.

In particular, c.1905 + 1G > A (rs3918290) represents the first *DPYD* variant to be associated with a 50% reduced function (for heterozygous patients) or non-activity (for homozygous ones) of the enzyme, increasing the risk of toxicity after fluoropyrimidines treatment [[14](#B14)]. Later, c.2846A > T (rs67376798), c.1679T > G (rs55886062) and c.1236G > A (rs56038477) were identified in correlation with a 3.02, 4.4 and 1.59-fold relative risk of severe fluoropyrimidines-induced toxicity, respectively [[15](#B15)]. Another SNP associated with the decrease of DPYD enzymatic activity is the missense variant c.2194G > A (rs1801160), which is also correlated to an approximately 2.4- and 1.9-fold increased risk of haematological toxicity and neutropenia [[16](#B16)]. Considering all these evidences, *DPYD* genotyping is becoming more and more popular in clinical practice before chemotherapy treatment.

The gold standard method to perform pharmacogenetic tests, including genotyping, is polymerase chain reaction (PCR) followed by Sanger sequencing (SS). While this approach may be more expensive than other genotyping technologies, it is the method of choice for genotyping cancer patients for *DPYD* SNPs due to its reliability and precision. This approach employs primer pairs designed to target specific regions of the *DPYD* gene, encompassing exon 11, 13, 14, 18, 22, and intron 10, to investigate variants like rs56038477, rs55886062, rs1801160, rs67376798, rs3918290, and rs75017182.

However, in a recent case involving a proband of African ethnic origin, genotyping of *DPYD* rs55886062 using PCR-SS encountered difficulties, resulting in an electropherogram sequence frameshift. To overcome this issue, whole genome sequencing (WGS) was employed. The implementation of WGS led to successfully genotyped rs55886062 and also revealed the presence of two additional SNPs, 1-97981199-T-TA and 1-97981622-G-GA, in the patient. These new identified SNPs were responsible for the genotyping failure encountered with the traditional method. This case highlights the importance of considering alternative genotyping approaches, such as WGS, when facing challenges with standard techniques. Moreover, it points out also the importance to consider the prevalence of genetic variants within diverse ethnic populations when designing primers and probes for SNP genotyping in pharmacogenetic testing, such as PCR followed by the SS method.

## Case report

A 48-year-old male of Nigerian origin has been diagnosed with adenocarcinoma (grade 2) in the right colon. The tumor has spread to one lymph node (pT3N1 M1c), and genetic analysis reveals the presence of the p.Gly12Asp (c.35G > A) mutation in codon 12 of exon 2 of the *KRAS* gene within the tumor tissue.

*DPYD* pharmacogenetic test was required to treat the patient with advanced first-line chemotherapy according to the FOLFOX scheme (oxaliplatin in combination with fluorouracil and folic acid) with the addition of Bevacizumab of which he had already completed six cycles.

### Results

To predict potential toxicity in an adenocarcinoma patient prior to fluorouracil treatment, the genotyping of specific *DPYD* SNPs, including rs56038477, rs55886062, rs1801160, rs67376798, rs3918290, and rs75017182 was performed. Using already reported primers pair [[17](#B17)], PCR-SS revealed that the patient carried the most common genotypes for rs56038477, rs1801160, rs67376798, rs3918290, and rs75017182 ([Figure 1](#fig1)). However, the patient exhibited heterozygosity for rs61622928 (c.12184G > A) and rs60511679 (c.2195T > G), which are not associated with a reduction in DPYD enzymatic activity, as documented in the PharmGKB database [[18](#B18), [19](#B19)].

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/02c4/11090686/5c1ee88ff284/etat-05-1002223-g001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=11090686_etat-05-1002223-g001.jpg)

Genotyping of DPYD SNPs associated with toxicity and enzymatic activity reduction in the patient diagnosed with adenocarcinoma. The figure schematically presents the outcomes of SS for exon 11, 13, 14 (including a portion of intron 14–15), 18, 22, and intron 10. The yellow arrows indicate the forward primers used for PCR and sequencing, while the green arrows represent the reverse primers. Below the schematic representation, are reported the GRCh37/hg19 reference genome, the patient’s sequence, and the electropherograms obtained through SS using the forward primer. The investigated SNPs are highlighted in red

Concerning the SNP rs55886062, a reading frameshift was observed in the electropherogram using both the forward and reverse primers ([Figure 1](#fig1)). In order to indirectly determine the patient’s genotype for rs55886062, the LDlink program was used to assess the linkage among rs55886062, the examined *DPYD* SNPs and the two additional SNPs identified in the patient. Since no evidence of LD between rs55886062 and these SNPs (*R*^2^ = 0.00) were found, the employment of WGS was necessary to genotype rs55886062.

Therefore, WGS results confirmed that the patient carried the homozygous T allele for rs55886062 ([Figure 2](#fig2), in green) and highlighted presence of one-base insertion A located downstream the forward primer ([Figure 2](#fig2), in violet) and another insertion upstream the annealing region of the reverse primer ([Figure 2](#fig2), in orange). Both insertions were found in close proximity of A repeats.

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/02c4/11090686/00bee5d400b1/etat-05-1002223-g002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=11090686_etat-05-1002223-g002.jpg)

WGS data visualization. The upper part of image shows the schematic representation of WGS results about the genomic region (chr1: 97981161–97981656) containing rs55886062 (chr1: 97981343). Chromosome short and long arms are reported as “p” and “q” letter in the scheme, respectively. WGS identified the presence of T/T genotype for rs55886062 (depicted in green) and an A insertion downstream the primer of the forward primer (rs145228578 or 1-97981199-T-TA, shown in purple) and upstream of the annealing region of reverse primer (rs141050810 or 1-97981622-G-GA, shown in orange). The primers used for PCR are represented in blue. In the lower part of figure, binary alignment and map (BAM) files containing WGS data from the proband are loaded into the Integrative Genomics Viewer (IGV). Noteworthy findings are highlighted in red boxes, including rs55886062 (in a green box), rs145228578 or 1-97981199-T-TA (in a purple box) and rs141050810 or 1-97981622-G-GA (in an orange box)

In the gnomAD database, the A insertion near the alignment site of the forward primer corresponded to the variant 1-97981199-T-TA (rs145228578), while the other A insertion upstream the annealing region of the reverse primer corresponded to the variant 1-97981622-G-GA (rs141050810). It is worth noting that these genetic variants were found to be quite rare among non-Finnish Europeans (0.05%) but more prevalent in Africans (16%; [Figure 3](#fig3)).

### Figure 3.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/02c4/11090686/1e8ca0ce1ccd/etat-05-1002223-g003.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=11090686_etat-05-1002223-g003.jpg)

Frequencies of 1-97981199-T-TA and 1-97981622-G-GA across various ethnic populations. (A) Histogram displays the population frequencies of 1-97981199-T-TA obtained from the gnomAD v2.1.1 database; (B) the population frequencies of 1-97981622-G-GA taken from the same database are reported

This frequency is also observed within certain Nigerian ethnicities: for the Esan and Yoruba in Ibadan, Nigeria, the frequencies of the variant 1-97981199-T-TA are 16.2% and 17.1% respectively, while for the variant 1-97981622-G-GA are 16.2% and 16.7% respectively ([Table 1](#t1)).

### Table 1.

| Population | 1-97981199-T-TA* | 1-97981622-G-GA* |
| --- | --- | --- |
| African Caribbean in Barbados | 18.2% | 18.2% |
| African Ancestry in Southwest US | 18.0% | 18.0% |
| Esan in Nigeria | 16.2% | 16.2% |
| Gambian in Western Division, The Gambia | 26.5% | 26.1% |
| Luhya in Webuye, Kenya | 15.7% | 15.7% |
| Mende in Sierra Leone | 17.6% | 17.6% |
| Yoruba in Ibadan, Nigeria | 17.1% | 16.7% |

Table 1 Caption: Allele frequencies of 1-97981199-T-TA and 1-97981622-G-GA across African ancestry populations

In both ClinVar and Franklin databases, 1-97981199-T-TA and 1-97981622-G-GA were reported as benign variants. Moreover, the enquiry of LDlink tool highlighted that these two SNPs were in LD (*R*^2^ = 0.99).

Altogether these findings suggested that the presence of 1-97981199-T-TA and 1-97981622-G-GA near the PCR primer annealing regions led to the electropherogram frameshift and the SS failure. In order to address this issue, an alternative primer pair within the region flanked by the primers previously used for amplifying *DPYD* exon 13 was developed. Subsequent sequencing of this amplified region successfully verified the patient’s genotype for rs55886062, as shown in [Figure 4](#fig4).

### Figure 4.

![Figure 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/02c4/11090686/062521c095fb/etat-05-1002223-g004.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=11090686_etat-05-1002223-g004.jpg)

Genotyping of rs55886062 using primer pairs designed in close proximity to the SNP site. The schematic representation displays the results of SS, employing a primer pair positioned within the region flanked by the primers previously used to amplify exon 13 of the DPYD gene. In the figure, the yellow arrow represents the forward primer used for PCR and sequencing, while the green arrow represents the reverse primer. Below the representation, the GRCh37/hg19 reference genome, the patient’s sequence, and the electropherogram obtained via SS with the forward primer are reported. The investigated SNPs are highlighted in red

### Material and methods

#### DNA extraction

Patient venous blood was collected into dipotassium-ethylenediaminetetraacetic acid (K2-EDTA) tube (Becton, Dickinson Vacutainer) and processed by using Wizard^®^ Genomic DNA Purification Kit (Promega), according to manufacturer instruction. DNA purity was checked using the NanoPhotometer^®^ spectrophotometer (IMPLEN) and its quality was assessed on 0.8% agarose gel.

#### PCR and SS

To routinely perform genotyping for *DPYD* SNPs strongly linked to toxicity following fluoropyrimidines treatment (specifically, rs56038477, rs55886062, rs1801160, rs67376798, rs3918290, and rs75017182), a PCR on exon 11, 13, 14 (including a segment of intron 14), 18, 22, and intron 10 was carried out using primer pairs previously documented in the literature [[17](#B17)]. All PCRs were performed using KAPA2G Robust HotStart ReadyMix PCR Kit (Kapa Biosystems) at the cycle conditions of 95°C for 3 min, followed by 35 cycles of 95°C for 30 s, 60°C for 45 s, and 72°C for 45 s with a final elongation step of 72°C for 1 min. The amplification of exon 13 inside the region flanked by 1-97981199-T-TA and 1-97981622-G-GA was performed using the following primers: 5’-tgtaaaacgacggccagtAGAAATGGCCGGATTGAAGT-3’ and 5’-caggaaacagctatgaccAAGTTTTGGTGAGGGCAAAACC-3’ (M13 sequence is highlighted in lower case). After performing PCR with KAPA2G Robust HotStart ReadyMix PCR Kit at the cycle conditions of 95°C for 3 min, followed by 35 cycles of 95°C for 15 s, 65°C for 15 s, and 72°C for 15 s with a final elongation step of 72°C for 1 min, PCR purification and SS were performed by DNA lab facility at the institute “CEINGE Biotecnologie Avanzate Franco Salvatore”.

#### WGS

WGS of the patient was performed on an Illumina HiSeq1500 platform at 35× depth. The paired-end sequencing produced 150 bp long reads which were aligned to the GRCh37/hg19 reference genome using Burrows-Wheeler Alignment with maximal exact matches (BWA-MEM2) tool (doi:10.1109/IPDPS.2019.00041). PCR duplicates were marked and removed using the MarkDuplicates tool of the genome analysis toolkit (GATK) suite [[20](#B20)]. SNPs and insertions and deletions (INDELs) were detected using the HaplotypeCaller program of GATK suite [[20](#B20)]. The resulting variant call format (VCF) files were annotated with ANNOtate VARiation (ANNOVAR) [[21](#B21)].

#### Evaluation of SNPs frequencies and pathogenicity

GnomAD v2.1.1 ([https://gnomad.broadinstitute.org/](https://gnomad.broadinstitute.org/)) and 1000 Genomes Project Phase 3 (Ensembl, [https://grch37.ensembl.org/](https://grch37.ensembl.org/)) databases were used to evaluate SNP frequencies in different populations. To evaluate SNPs pathogenicity, ClinVar ([https://www.ncbi.nlm.nih.gov/clinvar/](https://www.ncbi.nlm.nih.gov/clinvar/)) and Franklin ([https://franklin.genoox.com/](https://franklin.genoox.com/)) were consulted. SNP linkage was analysed using the LDlink tools ([https://ldlink.nih.gov/](https://ldlink.nih.gov/)).

## Discussion

Fluoropyrimidines constitute common medications for treating cancers with 2 million of treated patients annually [[22](#B22)–[24](#B24)]. Indeed, they remain the most effective drugs often used alone or in combination with other medications for treatment of colorectal (1.8 million), gastric (1 million), and pancreatic (460 thousand) cancers [[25](#B25)]. However, fluoropyrimidines may cause significant toxicities and adverse side effects such as nausea/vomiting, diarrhea, mucositis, alopecia, myelosuppression, heart toxicity, hand-foot syndrome (HFS), leukopenia and neutropenia [[26](#B26)]. Nowadays, the treatment-related mortality rate is 0.2–1.0% [[27](#B27)].

Underlying the severe adverse effects from treatment with fluoropyrimidines there are four *DPYD* SNPs (rs3918290, rs67376798, rs55886062 and rs56038477) considered the most clinically relevant and with statistically significant association with severe toxicity [[2](#B2)]. For this reason, several international agencies and consortia released recommendations on *DPYD* testing prior to treatment with fluoropyrimidines [[10](#B10)–[13](#B13)]. The current *DPYD* guideline released by CPIC recommends to reduce the dose of fluoropyrimidines by 25–50%, from the full standard dose, in individuals with a DPYD activity score of 1.5 [[11](#B11)]. Moreover, a recent prospective study provides evidence for genotype-guided dosing of decreased function alleles/variants, supporting a recommendation for a 50% dose reduction in heterozygous carriers of the decreased function variants rs67376798 or rs75017182 [[2](#B2)].

In the current scenario, the *DPYD* genotyping test is becoming increasingly essential before initiating fluoropyrimidines treatment. In fact, oncologists are now more frequently requesting *DPYD* genotyping for cancer patients, leading genetic laboratories to dedicate greater efforts towards enhancing the efficiency and accuracy of these genetic tests to deliver rapid and reliable results. Among the several technologies employed for *DPYD* genotyping, PCR-SS stands out as the most reliable method, despite its relatively high cost. However, it is worth noting that a significant portion of the primers used in genetic screening targets intronic regions, which are under less selective pressure compared to coding ones. Therefore, these intron-based primers tend to accumulate a greater number of genetic alterations.

Using primer pairs previously documented in the literature [[17](#B17)], *DPYD* genotyping was performed in an adenocarcinoma patient of African descent through PCR-SS. During this process, a challenge in rs55886062 genotyping was encountered, which was ultimately resolved using a WGS approach. WGS revealed the presence of two nucleotide insertions, 1-97981199-T-TA and 1-97981622-G-GA, near the annealing regions of the primers employed for amplifying *DPYD* exon 13. Specifically, 1-97981199-T-TA was situated in intron 12–13, while 1-97981622-G-GA resided in intron 13–14 of the *DPYD* gene. Both of these variants were identified as benign intronic variants in ClinVar and Franklin. Notably, these two variants were found to be more prevalent in the African population, which matches the patient’s ethnic background. Additionally, it is worth mentioning that 1-97981199-T-TA and 1-97981622-G-GA were observed to be in LD.

The presence of these two genetic variants led to a frameshift sequence in the electropherogram, making the detection of rs55886062 unfeasible. To address this limitation, the usage of a secondary primer pair within exon 13, designed to exclude the regions where the patient’s SNPs (1-97981199-T-TA and 1-97981622-G-GA) are located, was needed.

This case study underscores the critical importance of considering alternative genotyping approaches, such as WGS, when conventional techniques represent challenges. It highlights the need for flexibility in genotyping methods to accommodate unexpected complexities in an individual’s genetic makeup, especially in the context of pharmacogenetic testing. Additionally, this case serves as a valuable reminder of the significance of taking into account the diversity of genetic variants within different ethnic populations when designing primers and probes for SNP genotyping in pharmacogenetic testing. Genetic diversity among various ethnic groups can significantly impact the accuracy and reliability of genotyping methods. Hence, when developing genotyping assays for pharmacogenetics, it is crucial to consider the genetic variability and adapt the approach accordingly to ensure comprehensive and accurate results, particularly in a multicultural and diverse patient population.

## Abbreviations

## Declarations

### Author contributions

AM: Conceptualization, Data curation, Methodology, Visualization, Writing—original draft, Writing—review & editing. SC: Investigation, Methodology. GDA: Data curation, Formal analysis, Investigation, Methodology. VD: Resources. AI: Supervision, Writing—review & editing. MC: Conceptualization, Project administration, Supervision, Writing—review & editing. All authors read and approved the submitted version.

### Conflicts of interest

The authors declare that they have no conflicts of interest.

### Ethical approval

The review of this case report by the Ethics Committee was waived as it aligns with the Italian Association of Medical Oncology (AIOM) guidelines, contributes new or unusual findings to the medical community, enhancing awareness and encouraging advancements in medical knowledge and practice. Moreover, the study complies with the Declaration of Helsinki’s ethical principles.

### Consent to participate

Informed consent to participate in the study was obtained from all participants.

### Consent to publication

Not applicable.

### Availability of data and materials

Not applicable.

### Funding

Not applicable.

### Copyright

© The Author(s) 2024.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Data Availability Statement

Not applicable.

### Data Availability Statement

Not applicable.

## References

1. 5-Fluorouracil (i.v.), capecitabine and tegafur containing products: pre-treatment testing to identify DPD-deficient patients at increased risk of severe toxicity [Internet] [Cited 2023 Nov 6]. Available from: https://www.ema.europa.eu/en/documents/dhpc/direct-healthcare-professional-communication-dhpc-5-fluorouracil-iv-capecitabine-and-tegafur-containing-products-pre-treatment-testing-identify-dpd-deficient-patients-increased-risk-severe-toxicity_en.pdf .  [https://www.ema.europa.eu/en/documents/dhpc/direct-healthcare-professional-communication-dhpc-5-fluorouracil-iv-capecitabine-and-tegafur-containing-products-pre-treatment-testing-identify-dpd-deficient-patients-increased-risk-severe-toxicity_en.pdf](https://www.ema.europa.eu/en/documents/dhpc/direct-healthcare-professional-communication-dhpc-5-fluorouracil-iv-capecitabine-and-tegafur-containing-products-pre-treatment-testing-identify-dpd-deficient-patients-increased-risk-severe-toxicity_en.pdf)

2. Henricks LM, Lunenburg CATC, de Man FM, Meulendijks D, Frederix GWJ, Kienhuis E, et al. DPYD genotype-guided dose individualisation of fluoropyrimidine therapy in patients with cancer: a prospective safety analysis. Lancet Oncol. 2018;19:1459–67. doi: 10.1016/S1470-2045(18)30686-7.  [DOI](https://doi.org/10.1016/S1470-2045(18)30686-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30348537/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet%20Oncol&title=DPYD%20genotype-guided%20dose%20individualisation%20of%20fluoropyrimidine%20therapy%20in%20patients%20with%20cancer:%20a%20prospective%20safety%20analysis&author=LM%20Henricks&author=CATC%20Lunenburg&author=FM%20de%20Man&author=D%20Meulendijks&author=GWJ%20Frederix&volume=19&publication_year=2018&pages=1459-67&pmid=30348537&doi=10.1016/S1470-2045(18)30686-7&)

3. Raida M, Schwabe W, Häusler P, Van Kuilenburg AB, Van Gennip AH, Behnke D, et al. Prevalence of a common point mutation in the dihydropyrimidine dehydrogenase (DPD) gene within the 5’-splice donor site of intron 14 in patients with severe 5-fluorouracil (5-FU)- related toxicity compared with controls. Clin Cancer Res. 2001;7:2832–9.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/11555601/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=Prevalence%20of%20a%20common%20point%20mutation%20in%20the%20dihydropyrimidine%20dehydrogenase%20(DPD)%20gene%20within%20the%205%E2%80%99-splice%20donor%20site%20of%20intron%2014%20in%20patients%20with%20severe%205-fluorouracil%20(5-FU)-%20related%20toxicity%20compared%20with%20controls&author=M%20Raida&author=W%20Schwabe&author=P%20H%C3%A4usler&author=Kuilenburg%20AB%20Van&author=Gennip%20AH%20Van&volume=7&publication_year=2001&pages=2832-9&pmid=11555601&)

4. Gentile G, Botticelli A, Lionetto L, Mazzuca F, Simmaco M, Marchetti P, et al. Genotype-phenotype correlations in 5-fluorouracil metabolism: a candidate DPYD haplotype to improve toxicity prediction. Pharmacogenomics J. 2016;16:320–5. doi: 10.1038/tpj.2015.56.  [DOI](https://doi.org/10.1038/tpj.2015.56) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26216193/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Genotype-phenotype%20correlations%20in%205-fluorouracil%20metabolism:%20a%20candidate%20DPYD%20haplotype%20to%20improve%20toxicity%20prediction&author=G%20Gentile&author=A%20Botticelli&author=L%20Lionetto&author=F%20Mazzuca&author=M%20Simmaco&volume=16&publication_year=2016&pages=320-5&pmid=26216193&doi=10.1038/tpj.2015.56&)

5. Henricks LM, Lunenburg CA, Meulendijks D, Gelderblom H, Cats A, Swen JJ, et al. Translating DPYD genotype into DPD phenotype: using the DPYD gene activity score. Pharmacogenomics. 2015;16:1277–86. doi: 10.2217/pgs.15.70.  [DOI](https://doi.org/10.2217/pgs.15.70) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26265346/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Translating%20DPYD%20genotype%20into%20DPD%20phenotype:%20using%20the%20DPYD%20gene%20activity%20score&author=LM%20Henricks&author=CA%20Lunenburg&author=D%20Meulendijks&author=H%20Gelderblom&author=A%20Cats&volume=16&publication_year=2015&pages=1277-86&pmid=26265346&doi=10.2217/pgs.15.70&)

6. Toffoli G, Giodini L, Buonadonna A, Berretta M, De Paoli A, Scalone S, et al. Clinical validity of a DPYD-based pharmacogenetic test to predict severe toxicity to fluoropyrimidines. Int J Cancer. 2015;137:2971–80. doi: 10.1002/ijc.29654.  [DOI](https://doi.org/10.1002/ijc.29654) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26099996/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Cancer&title=Clinical%20validity%20of%20a%20DPYD-based%20pharmacogenetic%20test%20to%20predict%20severe%20toxicity%20to%20fluoropyrimidines&author=G%20Toffoli&author=L%20Giodini&author=A%20Buonadonna&author=M%20Berretta&author=Paoli%20A%20De&volume=137&publication_year=2015&pages=2971-80&pmid=26099996&doi=10.1002/ijc.29654&)

7. Terrazzino S, Cargnin S, Del Re M, Danesi R, Canonico PL, Genazzani AA. DPYD IVS14+1G>A and 2846A>T genotyping for the prediction of severe fluoropyrimidine-related toxicity: a meta-analysis. Pharmacogenomics. 2013;14:1255–72. doi: 10.2217/pgs.13.116.  [DOI](https://doi.org/10.2217/pgs.13.116) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23930673/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=DPYD%20IVS14+1G>A%20and%202846A>T%20genotyping%20for%20the%20prediction%20of%20severe%20fluoropyrimidine-related%20toxicity:%20a%20meta-analysis&author=S%20Terrazzino&author=S%20Cargnin&author=Re%20M%20Del&author=R%20Danesi&author=PL%20Canonico&volume=14&publication_year=2013&pages=1255-72&pmid=23930673&doi=10.2217/pgs.13.116&)

8. Rosmarin D, Palles C, Church D, Domingo E, Jones A, Johnstone E, et al. Genetic markers of toxicity from capecitabine and other fluorouracil-based regimens: investigation in the QUASAR2 study, systematic review, and meta-analysis. J Clin Oncol. 2014;32:1031–9. doi: 10.1200/JCO.2013.51.1857.  [DOI](https://doi.org/10.1200/JCO.2013.51.1857) | [PMC free article](/articles/PMC4879695/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24590654/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Genetic%20markers%20of%20toxicity%20from%20capecitabine%20and%20other%20fluorouracil-based%20regimens:%20investigation%20in%20the%20QUASAR2%20study,%20systematic%20review,%20and%20meta-analysis&author=D%20Rosmarin&author=C%20Palles&author=D%20Church&author=E%20Domingo&author=A%20Jones&volume=32&publication_year=2014&pages=1031-9&pmid=24590654&doi=10.1200/JCO.2013.51.1857&)

9. Sharma BB, Rai K, Blunt H, Zhao W, Tosteson TD, Brooks GA. Pathogenic DPYD variants and treatment-related mortality in patients receiving fluoropyrimidine chemotherapy: a systematic review and meta-analysis. Oncologist. 2021;26:1008–16. doi: 10.1002/onco.13967.  [DOI](https://doi.org/10.1002/onco.13967) | [PMC free article](/articles/PMC8649021/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34506675/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncologist&title=Pathogenic%20DPYD%20variants%20and%20treatment-related%20mortality%20in%20patients%20receiving%20fluoropyrimidine%20chemotherapy:%20a%20systematic%20review%20and%20meta-analysis&author=BB%20Sharma&author=K%20Rai&author=H%20Blunt&author=W%20Zhao&author=TD%20Tosteson&volume=26&publication_year=2021&pages=1008-16&pmid=34506675&doi=10.1002/onco.13967&)

10. EMA recommendations on DPD testing prior to treatment with fluorouracil, capecitabine, tegafur and flucytosine [Internet] European Medicines Agency; c2020 [cited 2023 Nov 6]. Available from: https://www.ema.europa.eu/en/documents/referral/fluorouracil-and-fluorouracil-related-substances-article-31-referral-ema-recommendations-dpd-testing-prior-treatment-fluorouracil-capecitabine-tegafur-and-flucytosine_en.pdf .  [https://www.ema.europa.eu/en/documents/referral/fluorouracil-and-fluorouracil-related-substances-article-31-referral-ema-recommendations-dpd-testing-prior-treatment-fluorouracil-capecitabine-tegafur-and-flucytosine_en.pdf](https://www.ema.europa.eu/en/documents/referral/fluorouracil-and-fluorouracil-related-substances-article-31-referral-ema-recommendations-dpd-testing-prior-treatment-fluorouracil-capecitabine-tegafur-and-flucytosine_en.pdf)

11. Amstutz U, Henricks LM, Offer SM, Barbarino J, Schellens JHM, Swen JJ, et al. Clinical Pharmacogenetics Implementation Consortium (CPIC) guideline for dihydropyrimidine dehydrogenase genotype and fluoropyrimidine dosing: 2017 update. Clin Pharmacol Ther. 2018;103:210–6. doi: 10.1002/cpt.911.  [DOI](https://doi.org/10.1002/cpt.911) | [PMC free article](/articles/PMC5760397/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29152729/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Clinical%20Pharmacogenetics%20Implementation%20Consortium%20(CPIC)%20guideline%20for%20dihydropyrimidine%20dehydrogenase%20genotype%20and%20fluoropyrimidine%20dosing:%202017%20update&author=U%20Amstutz&author=LM%20Henricks&author=SM%20Offer&author=J%20Barbarino&author=JHM%20Schellens&volume=103&publication_year=2018&pages=210-6&pmid=29152729&doi=10.1002/cpt.911&)

12. Lunenburg CATC, van der Wouden CH, Nijenhuis M, Crommentuijn-van Rhenen MH, de Boer-Veger NJ, Buunk AM, et al. Dutch Pharmacogenetics Working Group (DPWG) guideline for the gene-drug interaction of DPYD and fluoropyrimidines. Eur J Hum Genet. 2020;28:508–17. doi: 10.1038/s41431-019-0540-0.  [DOI](https://doi.org/10.1038/s41431-019-0540-0) | [PMC free article](/articles/PMC7080718/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31745289/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Hum%20Genet&title=Dutch%20Pharmacogenetics%20Working%20Group%20(DPWG)%20guideline%20for%20the%20gene-drug%20interaction%20of%20DPYD%20and%20fluoropyrimidines&author=CATC%20Lunenburg&author=CH%20van%20der%20Wouden&author=M%20Nijenhuis&author=Rhenen%20MH%20Crommentuijn-van&author=NJ%20de%20Boer-Veger&volume=28&publication_year=2020&pages=508-17&pmid=31745289&doi=10.1038/s41431-019-0540-0&)

13. Hamzic S, Aebi S, Joerger M, Montemurro M, Ansari M, Amstutz U, et al. Fluoropyrimidine chemotherapy: recommendations for DPYD genotyping and therapeutic drug monitoring of the Swiss Group of Pharmacogenomics and personalised therapy. Swiss Med Wkly. 2020;150:w20375. doi: 10.4414/smw.2020.20375.  [DOI](https://doi.org/10.4414/smw.2020.20375) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33232506/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Swiss%20Med%20Wkly&title=Fluoropyrimidine%20chemotherapy:%20recommendations%20for%20DPYD%20genotyping%20and%20therapeutic%20drug%20monitoring%20of%20the%20Swiss%20Group%20of%20Pharmacogenomics%20and%20personalised%20therapy&author=S%20Hamzic&author=S%20Aebi&author=M%20Joerger&author=M%20Montemurro&author=M%20Ansari&volume=150&publication_year=2020&pages=w20375&pmid=33232506&doi=10.4414/smw.2020.20375&)

14. Vreken P, Van Kuilenburg AB, Meinsma R, Smit GP, Bakker HD, De Abreu RA, et al. A point mutation in an invariant splice donor site leads to exon skipping in two unrelated Dutch patients with dihydropyrimidine dehydrogenase deficiency. J Inherit Metab Dis. 1996;19:645–54. doi: 10.1007/BF01799841.  [DOI](https://doi.org/10.1007/BF01799841) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8892022/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Inherit%20Metab%20Dis&title=A%20point%20mutation%20in%20an%20invariant%20splice%20donor%20site%20leads%20to%20exon%20skipping%20in%20two%20unrelated%20Dutch%20patients%20with%20dihydropyrimidine%20dehydrogenase%20deficiency&author=P%20Vreken&author=Kuilenburg%20AB%20Van&author=R%20Meinsma&author=GP%20Smit&author=HD%20Bakker&volume=19&publication_year=1996&pages=645-54&pmid=8892022&doi=10.1007/BF01799841&)

15. Loganayagam A, Arenas Hernandez M, Corrigan A, Fairbanks L, Lewis CM, Harper P, et al. Pharmacogenetic variants in the DPYD, TYMS, CDA and MTHFR genes are clinically significant predictors of fluoropyrimidine toxicity. Br J Cancer. 2013;108:2505–15. doi: 10.1038/bjc.2013.262.  [DOI](https://doi.org/10.1038/bjc.2013.262) | [PMC free article](/articles/PMC3694243/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23736036/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Cancer&title=Pharmacogenetic%20variants%20in%20the%20DPYD,%20TYMS,%20CDA%20and%20MTHFR%20genes%20are%20clinically%20significant%20predictors%20of%20fluoropyrimidine%20toxicity&author=A%20Loganayagam&author=Hernandez%20M%20Arenas&author=A%20Corrigan&author=L%20Fairbanks&author=CM%20Lewis&volume=108&publication_year=2013&pages=2505-15&pmid=23736036&doi=10.1038/bjc.2013.262&)

16. Kim W, Cho YA, Kim DC, Lee KE. Elevated risk of fluoropyrimidine-associated toxicity in European patients with DPYD genetic polymorphism: a systematic review and meta-analysis. J Pers Med. 2022;12:225. doi: 10.3390/jpm12020225.  [DOI](https://doi.org/10.3390/jpm12020225) | [PMC free article](/articles/PMC8875904/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35207713/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pers%20Med&title=Elevated%20risk%20of%20fluoropyrimidine-associated%20toxicity%20in%20European%20patients%20with%20DPYD%20genetic%20polymorphism:%20a%20systematic%20review%20and%20meta-analysis&author=W%20Kim&author=YA%20Cho&author=DC%20Kim&author=KE%20Lee&volume=12&publication_year=2022&pages=225&pmid=35207713&doi=10.3390/jpm12020225&)

17. Coenen MJH, Paulussen ADC, Breuer M, Lindhout M, Tserpelis DCJ, Steyls A, et al. Evolution of dihydropyrimidine dehydrogenase diagnostic testing in a single center during an 8-year period of time. Curr Ther Res Clin Exp. 2018;90:1–7. doi: 10.1016/j.curtheres.2018.10.001.  [DOI](https://doi.org/10.1016/j.curtheres.2018.10.001) | [PMC free article](/articles/PMC6258870/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30510603/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Ther%20Res%20Clin%20Exp&title=Evolution%20of%20dihydropyrimidine%20dehydrogenase%20diagnostic%20testing%20in%20a%20single%20center%20during%20an%208-year%20period%20of%20time&author=MJH%20Coenen&author=ADC%20Paulussen&author=M%20Breuer&author=M%20Lindhout&author=DCJ%20Tserpelis&volume=90&publication_year=2018&pages=1-7&pmid=30510603&doi=10.1016/j.curtheres.2018.10.001&)

18. Whirl-Carrillo M, McDonagh EM, Hebert JM, Gong L, Sangkuhl K, Thorn CF, et al. Pharmacogenomics knowledge for personalized medicine. Clin Pharmacol Ther. 2012;92:414–7. doi: 10.1038/clpt.2012.96.  [DOI](https://doi.org/10.1038/clpt.2012.96) | [PMC free article](/articles/PMC3660037/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22992668/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Pharmacogenomics%20knowledge%20for%20personalized%20medicine&author=M%20Whirl-Carrillo&author=EM%20McDonagh&author=JM%20Hebert&author=L%20Gong&author=K%20Sangkuhl&volume=92&publication_year=2012&pages=414-7&pmid=22992668&doi=10.1038/clpt.2012.96&)

19. Whirl-Carrillo M, Huddart R, Gong L, Sangkuhl K, Thorn CF, Whaley R, et al. An evidence-based framework for evaluating pharmacogenomics knowledge for personalized medicine. Clin Pharmacol Ther. 2021;110:563–72. doi: 10.1002/cpt.2350.  [DOI](https://doi.org/10.1002/cpt.2350) | [PMC free article](/articles/PMC8457105/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34216021/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=An%20evidence-based%20framework%20for%20evaluating%20pharmacogenomics%20knowledge%20for%20personalized%20medicine&author=M%20Whirl-Carrillo&author=R%20Huddart&author=L%20Gong&author=K%20Sangkuhl&author=CF%20Thorn&volume=110&publication_year=2021&pages=563-72&pmid=34216021&doi=10.1002/cpt.2350&)

20. McKenna A, Hanna M, Banks E, Sivachenko A, Cibulskis K, Kernytsky A, et al. The genome analysis toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data. Genome Res. 2010;20:1297–303. doi: 10.1101/gr.107524.110.  [DOI](https://doi.org/10.1101/gr.107524.110) | [PMC free article](/articles/PMC2928508/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20644199/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res&title=The%20genome%20analysis%20toolkit:%20a%20MapReduce%20framework%20for%20analyzing%20next-generation%20DNA%20sequencing%20data&author=A%20McKenna&author=M%20Hanna&author=E%20Banks&author=A%20Sivachenko&author=K%20Cibulskis&volume=20&publication_year=2010&pages=1297-303&pmid=20644199&doi=10.1101/gr.107524.110&)

21. Wang K, Li M, Hakonarson H. ANNOVAR: functional annotation of genetic variants from high-throughput sequencing data. Nucleic Acids Res. 2010;38:e164. doi: 10.1093/nar/gkq603.  [DOI](https://doi.org/10.1093/nar/gkq603) | [PMC free article](/articles/PMC2938201/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20601685/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=ANNOVAR:%20functional%20annotation%20of%20genetic%20variants%20from%20high-throughput%20sequencing%20data&author=K%20Wang&author=M%20Li&author=H%20Hakonarson&volume=38&publication_year=2010&pages=e164&pmid=20601685&doi=10.1093/nar/gkq603&)

22. Caudle KE, Thorn CF, Klein TE, Swen JJ, McLeod HL, Diasio RB, et al. Clinical Pharmacogenetics Implementation Consortium guidelines for dihydropyrimidine dehydrogenase genotype and fluoropyrimidine dosing. Clin Pharmacol Ther. 2013;94:640–5. doi: 10.1038/clpt.2013.172.  [DOI](https://doi.org/10.1038/clpt.2013.172) | [PMC free article](/articles/PMC3831181/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23988873/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Clinical%20Pharmacogenetics%20Implementation%20Consortium%20guidelines%20for%20dihydropyrimidine%20dehydrogenase%20genotype%20and%20fluoropyrimidine%20dosing&author=KE%20Caudle&author=CF%20Thorn&author=TE%20Klein&author=JJ%20Swen&author=HL%20McLeod&volume=94&publication_year=2013&pages=640-5&pmid=23988873&doi=10.1038/clpt.2013.172&)

23. Diasio RB, Beavers TL, Carpenter JT. Familial deficiency of dihydropyrimidine dehydrogenase. Biochemical basis for familial pyrimidinemia and severe 5-fluorouracil-induced toxicity. J Clin Invest. 1988;81:47–51. doi: 10.1172/JCI113308.  [DOI](https://doi.org/10.1172/JCI113308) | [PMC free article](/articles/PMC442471/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/3335642/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Invest&title=Familial%20deficiency%20of%20dihydropyrimidine%20dehydrogenase.%20Biochemical%20basis%20for%20familial%20pyrimidinemia%20and%20severe%205-fluorouracil-induced%20toxicity&author=RB%20Diasio&author=TL%20Beavers&author=JT%20Carpenter&volume=81&publication_year=1988&pages=47-51&pmid=3335642&doi=10.1172/JCI113308&)

24. Boisdron-Celle M, Remaud G, Traore S, Poirier AL, Gamelin L, Morel A, et al. 5-Fluorouracil-related severe toxicity: a comparison of different methods for the pretherapeutic detection of dihydropyrimidine dehydrogenase deficiency. Cancer Lett. 2007;249:271–82. doi: 10.1016/j.canlet.2006.09.006.  [DOI](https://doi.org/10.1016/j.canlet.2006.09.006) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17064846/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Lett&title=5-Fluorouracil-related%20severe%20toxicity:%20a%20comparison%20of%20different%20methods%20for%20the%20pretherapeutic%20detection%20of%20dihydropyrimidine%20dehydrogenase%20deficiency&author=M%20Boisdron-Celle&author=G%20Remaud&author=S%20Traore&author=AL%20Poirier&author=L%20Gamelin&volume=249&publication_year=2007&pages=271-82&pmid=17064846&doi=10.1016/j.canlet.2006.09.006&)

25. Worldwide cancer data [Internet] WCRF International; [cited 2023 Nov 6]. Available from: https://www.wcrf.org/cancer-trends/worldwide-cancer-data/  [https://www.wcrf.org/cancer-trends/worldwide-cancer-data/](https://www.wcrf.org/cancer-trends/worldwide-cancer-data/)

26. Gmeiner WH. A narrative review of genetic factors affecting fluoropyrimidine toxicity. Precis Cancer Med. 2021;4:38. doi: 10.21037/pcm-21-17.  [DOI](https://doi.org/10.21037/pcm-21-17) | [PMC free article](/articles/PMC8664072/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34901834/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Precis%20Cancer%20Med&title=A%20narrative%20review%20of%20genetic%20factors%20affecting%20fluoropyrimidine%20toxicity&author=WH%20Gmeiner&volume=4&publication_year=2021&pages=38&pmid=34901834&doi=10.21037/pcm-21-17&)

27. Wörmann B, Bokemeyer C, Burmeister T, Köhne CH, Schwab M, Arnold D, et al. Dihydropyrimidine dehydrogenase testing prior to treatment with 5-fluorouracil, capecitabine, and tegafur: a consensus paper. Oncol Res Treat. 2020;43:628–36. doi: 10.1159/000510258.  [DOI](https://doi.org/10.1159/000510258) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33099551/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncol%20Res%20Treat&title=Dihydropyrimidine%20dehydrogenase%20testing%20prior%20to%20treatment%20with%205-fluorouracil,%20capecitabine,%20and%20tegafur:%20a%20consensus%20paper&author=B%20W%C3%B6rmann&author=C%20Bokemeyer&author=T%20Burmeister&author=CH%20K%C3%B6hne&author=M%20Schwab&volume=43&publication_year=2020&pages=628-36&pmid=33099551&doi=10.1159/000510258&)
