# Investigation of inherited noncoding genetic variation impacting the pharmacogenomics of childhood acute lymphoblastic leukemia treatment

## Metadata
**Authors:** Kashi Raj Bhattarai, Robert J Mobley, Kelly R Barnett, Daniel C Ferguson, Baranda S Hansen, Jonathan D Diedrich, Brennan P Bergeron, Satoshi Yoshimura, Wenjian Yang, Kristine R Crews, Christopher S Manring, Elias Jabbour, Elisabeth Paietta, Mark R Litzow, Steven M Kornblau, Wendy Stock, Hiroto Inaba, Sima Jeha, Ching-Hon Pui, Cheng Cheng, Shondra M Pruett-Miller, Mary V Relling, Jun J Yang, William E Evans, Daniel Savic
**Journal:** Nature Communications
**Date:** 2024 May 1
**DOI:** [10.1038/s41467-024-48124-4](https://doi.org/10.1038/s41467-024-48124-4)
**PMID:** 38693155
**PMCID:** PMC11063049
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11063049/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC11063049/pdf/41467_2024_Article_48124.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC11063049/pdf/41467_2024_Article_48124.pdf)

## Abstract

Defining genetic factors impacting chemotherapy failure can help to better predict response and identify drug resistance mechanisms. However, there is limited understanding of the contribution of inherited noncoding genetic variation on inter-individual differences in chemotherapy response in childhood acute lymphoblastic leukemia (ALL). Here we map inherited noncoding variants associated with treatment outcome and/or chemotherapeutic drug resistance to ALL cis-regulatory elements and investigate their gene regulatory potential and target gene connectivity using massively parallel reporter assays and three-dimensional chromatin looping assays, respectively. We identify 54 variants with transcriptional effects and high-confidence gene connectivity. Additionally, functional interrogation of the top variant, rs1247117, reveals changes in chromatin accessibility, PU.1 binding affinity and gene expression, and deletion of the genomic interval containing rs1247117 sensitizes cells to vincristine. Together, these data demonstrate that noncoding regulatory variants associated with diverse pharmacological traits harbor significant effects on allele-specific transcriptional activity and impact sensitivity to antileukemic agents.

Subject terms: Cancer genomics, Cancer therapeutic resistance, Gene regulation, Genetic markers, Acute lymphocytic leukaemia

## Introduction

Due to continual advances in treatment protocol over the last 60 years, the survival rate of the most common malignancy in children, acute lymphoblastic leukemia (ALL), has dramatically improved to over 90% in high-income countries^1^. Despite these advances, survival rates of pediatric patients experiencing refractory or relapsed ALL were only 30–50%, and those of adults were especially low (~10%)^2^. Thus, improving the understanding of the underlying genetic risk factors impacting response to ALL chemotherapy is a major step in improving outcomes for patients with refractory or relapsed ALL.

Genome-wide association studies (GWAS) have identified numerous inherited DNA sequence variants associated with treatment outcome in childhood ALL from clinical trials carried out by St. Jude Children’s Research Hospital and the Children’s Oncology group^3–5^. This includes GWAS analyses that identified inherited genetic contributors associated with patient relapse^4,5^ and persistence of minimal residual disease (MRD) after induction chemotherapy^3^, which is an early indicator of treatment failure^6–9^. In addition, ex vivo chemotherapeutic drug sensitivity testing using primary ALL cells from patients serves as an informative pharmacological phenotype^10^. When integrated with genotype profiling for GWAS, these analyses identify variants contributing to antileukemic drug resistance that reflects in vivo and ex vivo resistance and is, therefore, predictive of treatment outcomes in patients^10–22^.

Because most GWAS variants, including pharmacogenomic variants^23,24^, lie in noncoding sequences in the human genome, their connection to gene regulation and cellular biology has yet to be established. Moreover, given that dozens of variants are typically in strong linkage disequilibrium (LD) with an associated sentinel variant, pinpointing causal variants at GWAS loci has been challenging. Noncoding GWAS variants have been consistently linked to disruption of *cis*-regulatory element (CRE) activity and gene regulation^25^. As a result, the functional evaluation of these regulatory variants involves an examination of their allele-specific activities on transcriptional output, which has traditionally been a low-throughput endeavor. Therefore, the functional investigation of all associated regulatory variations at GWAS loci (sentinel and LD) proved to be an intractable hurdle to investigators. Recent technological advances, however, have ameliorated these challenges through the advent of massively parallel reporter assays (MPRAs) where the reporter aspect is often a self-transcribed barcode in the 3′-UTR of a reporter gene that is detected using next-generation sequencing. MPRAs allow the simultaneous, rapid, and robust detection of differences in transcriptional output from a library of *cis*-regulatory sequences of interest^26–28^. MPRAs have since been applied to the study of regulatory variation at GWAS loci through an examination of allele-specific effects on reporter gene expression^29–35^.

Another challenge is connecting regulatory variation at promoter-distal CREs to a target gene, as the closest gene may not be the target gene^36^. To circumvent these challenges, regulatory variation can be coupled to transcriptomics to identify variants impacting the expression of a candidate target gene through expression quantitative trait locus (eQTL) mapping^36,37^. Functional genomics offers additional solutions through the mapping of three-dimensional (3D) genomic interactions^38^. Because promoter-distal CREs (e.g., enhancers) regulate gene expression through long-range 3D looping to the promoters of target genes^25^, an attractive assay to identify gene targets of promoter-distal regulatory variants is promoter capture Hi-C (promoter CHiC)^39^. Promoter capture Hi-C, and related 3D chromatin interaction assays have been implemented at multiple GWAS loci to identify gene targets of promoter-distal regulatory variation^29,36,40–43^.

In this work, to better understand the underlying genetic and gene regulatory factors that impact diverse pharmacological traits in ALL, we performed a comprehensive functional interrogation of GWAS regulatory variants that map to ALL accessible chromatin sites and that are associated with ALL treatment outcome (i.e., relapse and persistence of MRD after induction chemotherapy) in patients and/or ex vivo chemotherapeutic drug resistance in primary ALL cells from patients using MPRA. We coupled these results with promoter CHiC to identify candidate target genes of functional regulatory variants with significant allele-specific effects on reporter gene expression. Finally, we functionally investigated the impact of the top regulatory variants chemotherapeutic drug resistance in ALL cell lines. Collectively, this study implements a comprehensive functional investigation of the allele-specific, gene-regulatory effects of noncoding variants associated with the pharmacogenomics of ALL treatment, and, therefore, fills an unmet need for large-scale functional examinations of regulatory GWAS variants associated with ALL pharmacological traits.

## Results

### Identification of noncoding regulatory variants impacting the pharmacogenomics of ALL treatment

Single-nucleotide variants (SNVs) impacting diverse pharmacological traits in ALL were identified for functional interrogation. We chose SNVs associated with relapse or persistence of MRD after induction chemotherapy in childhood ALL patients to investigate the role of inherited noncoding regulatory variants impacting clinical phenotypes (i.e., treatment outcome). These SNVs were identified from published GWAS of ALL patients enrolled in St. Jude Children’s Research Hospital and the Children’s Oncology Group clinical protocols^3–5^ (see Methods for variant selection criteria). Variant selection also included prioritization for treatment outcome SNVs associated with drug resistance phenotypes in primary ALL cells to enrich for variation impacting ALL cell biology (see Methods for variant selection criteria). These treatment outcome-associated variants, as well as all variants in high LD (*r*^2^ > 0.8) with the sentinel GWAS variants, were further evaluated (Fig. [1a, b](#Fig1)).

### Fig. 1. Identification and mapping of regulatory variants impacting the pharmacogenomics of ALL treatment.

![Fig. 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7c1c/11063049/cef98de85d48/41467_2024_48124_Fig1_HTML.jpg)

a SNVs of interest from GWAS were pursued based on association with ex vivo chemotherapeutic drug resistance in primary ALL cells from patients and/or treatment outcome. Dex dexamethasone, Pred prednisolone, VCR vincristine, 6MP 6-mercaptopurine, 6TG 6-thioguanine, LASP L-asparaginase. b GWAS SNVs were combined with ALL disease susceptibly control GWAS SNVs and SNVs in high LD (R2 > 0.8) and c mapped to accessible chromatin sites in ALL cell lines, ALL PDXs, and primary ALL cells from patients. Of the 1696 SNVs mapped to accessible chromatin sites, 35 are control SNVs. Source data are provided in the Source Data file.

We also identified variants directly associated with ex vivo chemotherapeutic drug resistance in primary ALL cells from patients by performing GWAS analyses using SNV genotype information and ex vivo drug resistance assay results for six antileukemic agents (prednisolone, dexamethasone, vincristine, L-asparaginase, 6-mercaptopurine [6MP] and 6-thioguanine [6TG]) in primary ALL cells from 312–344 patients (not all patients were tested for all drugs) enrolled in the Total Therapy XVI clinical protocol at St. Jude Children’s Research Hospital (see Methods). We further prioritized functional ex vivo drug resistance SNVs by determining if they were eQTLs in primary ALL cells or related cell types (i.e., whole blood and EBV-transformed lymphocytes) from the Genotype-Tissue Expression (GTEx) consortium^37^ (see Methods for variant selection criteria). Ex vivo drug resistance variants that were also identified as eQTLs, as well as variants in high LD (*r*^2^ > 0.8) with these sentinel GWAS variants, were further evaluated (Fig. [1a, b](#Fig1)).

GWAS have also been performed for childhood ALL disease susceptibility and identified several GWAS loci harboring variants with genome-wide significance^44–50^. Several follow-up studies of these GWAS loci have identified candidate causal noncoding variants and mechanisms involving gene regulatory disruptions^51–53^. As a result, we used ALL disease susceptibility variants (*n* = 11), as well as variants in high LD (*r*^2^ > 0.8) with them, for further analysis as positive controls in our study (Fig. [1a, b](#Fig1)).

Because most of these variants map to noncoding portions of the human genome, these data point to disruptions in gene regulation as the underlying mechanism of how these variants impact ALL cell biology. We therefore utilized assay for transposase-accessible chromatin with high-throughput sequencing (ATAC-seq)^54^ chromatin accessibility data in 161 ALL cell models, comprised of primary ALL cells (cryopreserved, *n* = 24^55^; fresh, *n* = 120^56^), ALL cell lines (*n* = 14) and ALL patient-derived xenografts (PDXs, *n* = 3), to uncover which variants map to putative CREs in ALL cells^57^ (i.e., regulatory variants; Fig. [1c](#Fig1)). Although we detected variation in ATAC-seq TSS enrichment scores and peak counts that is to be expected from such a large, mixed cohort of ALL cell models, the peaks called were largely reproducible (found in >3 samples) within each group (Supplementary Fig. [1a–c](#MOESM2)). ATAC-seq data from primary ALL cells, ALL cell lines, and PDXs were combined and identified 1696 regulatory variants at accessible chromatin sites in ALL cells for functional investigation (Fig. [1c](#Fig1) and Supplementary Data [1](#MOESM4)).

### Assessing the impact of regulatory variation on transcriptional output using MPRA

To examine the functional effects of these 1696 regulatory variants on transcriptional output in a high-throughput manner we utilized a barcode-based MPRA platform^29,32^ to measure differences in allele-specific transcriptional output (Fig. [2a](#Fig2)). Oligonucleotides containing 175-bp of genomic sequence centered on each reference (ref) or alternative (alt) variant allele, a restriction site, and a unique 10-bp barcode sequence were cloned into plasmids. An open reading frame containing a minimal promoter driving GFP was then inserted at the restriction site between the alleles of interest and their unique barcodes (Fig. [2a](#Fig2)). We utilized 28 unique 3′UTR DNA barcodes per variant allele (56 barcodes per regulatory variant), and variants near bidirectional promoters (47 total variants) were tested using both sequence orientations. In total, 97,608 variant-harboring oligonucleotides were evaluated for allele-specific differences in gene regulatory activity (Fig. [2a](#Fig2)).

### Fig. 2. MPRA identifies regulatory variants with allele-specific effects on gene expression.

![Fig. 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7c1c/11063049/e9f67bfc96ca/41467_2024_48124_Fig2_HTML.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=11063049_41467_2024_48124_Fig2_HTML.jpg)

a Diagram describing design of MPRA (also see Methods). b–d Significant MPRA hits were identified by Benjamini–Hochberg FDR corrected two-tailed Students T tests. b Distribution of significant changes in allele-specific transcriptional activity across all SNVs. c Number of MPRA SNVs showing significant (Adj. p < 0.05) changes in allele-specific transcriptional activity in each ALL cell line. d Pairwise linear correlation between changes in allele-specific transcriptional activity for all significant (Adj. p < 0.05) changes across all cell lines. R2 correlation and p value are provided. All source data and statistical parameters are provided in the Source Data file.

Following transfection into 7 different B-cell precursor ALL (B-ALL; 697, BALL1, Nalm6, REH, RS411, SEM, SUPB15) and 3 T-cell ALL (T-ALL; CEM, Jurkat, P12-Ichikawa) human cell lines (*n* = 4 transfections per cell line; 40 total), the transcriptional activity of each allele variant was measured by high-throughput sequencing to determine the barcode representation in reporter mRNA and compared to DNA counts obtained from high-throughput sequencing of the MPRA plasmid pool (Fig. [2a](#Fig2)). In the 10 cell lines MPRA detected 4633 instances of significant differential activity between alleles across 91% (1538/1696) of regulatory variants tested (Fig. [2b, c](#Fig2), Supplementary Data [2](#MOESM5)). The 10 ALL cell lines showed substantial differences in the total number of regulatory variants harboring significant allele-specific activity, which we suspect largely stems from differences in transfection efficiency (Fig. [2c](#Fig2)). Importantly, when comparing changes in allele-specific MPRA activity for each regulatory variant we found that significant changes in activity (adj. *p* < 0.05) were highly correlated between ALL cell lines, with 87% concordance in allelic-specific activity, suggesting that significant MPRA hits were likely to be robust and reproducible between cell lines (Fig. [2d](#Fig2)). Allele-specific MPRA activities were also correlated using all pairwise cell line comparisons for each regulatory variant, irrespective of significance (Supplementary Fig. [2a](#MOESM2)). Importantly, 31 of the 35 positive control variants (i.e., ALL disease susceptibility-associated variants and variants in high LD) showed significant allelic effects in at least 1 cell line, and 10 showed significant and concordant allelic effects in at least three ALL cell lines, including two variants (*rs3824662* at *GATA3* locus and *rs75777619* at *8q24.21*) directly associated with ALL susceptibility^44,49,52^ (Supplementary Data [2](#MOESM5)). The risk A allele at *rs3824662* was associated with higher *GATA3* expression and chromatin accessibility and demonstrated significantly higher allele-specific activity in our MPRA^44,52^, thereby demonstrating that the MPRA could detect allelic effects previously identified by others.

To further validate MPRA hits in an ex vivo model, we performed MPRA using two B-ALL PDX samples that were freshly harvested from mice. These samples detected 26 and 67 significant gene regulatory variants, respectively, and showed significant correlation with the cell line MPRA data (Supplementary Fig. [2b, c](#MOESM2), Supplementary Data [3](#MOESM6)). We attribute the detection of relatively lower numbers of variants in PDXs to technical effects stemming from poor transfection efficiency and limited cell survival ex vivo. Overall, our data suggest that the cohort of SNVs tested contained functional regulatory variants with the potential to impact gene regulation.

### Identification of functional regulatory variants showing reproducible and concordant changes in allele-specific gene expression

To further focus on regulatory variants most likely to broadly impact gene regulation in ALL cells, we prioritized 556 variants with significant (adj. *p* < 0.05) and concordant allele-specific activities in at least three ALL cell lines (i.e., functional regulatory variants; Fig. [3a–d](#Fig3), Supplementary Data [4](#MOESM7)). Most of these functional regulatory variants (318/556) mapped to accessible chromatin found only in primary ALL cell samples, underscoring the importance of incorporating chromatin architecture from primary ALL cells, and 54 functional regulatory variants mapped to transcription factor footprints in primary ALL cells (Supplementary Fig. [3](#MOESM2)). Additionally, we used Genomic Regions Enrichment of Annotations Tool (GREAT) to associate these SNVs with their nearby genes and search for enrichment in gene ontology biological processes pathways^58^. Although GREAT identified gene associations for nearly all SNVs, we found no significant pathway associations (Supplementary Data [4](#MOESM7) and [5](#MOESM8)). Because further functional investigation of variants in primary ALL cells or PDXs ex vivo is largely intractable, we focused on 210 functional regulatory variants that were detected in open chromatin in one of the 14 ALL cell lines that we had generated ATAC-seq data (Fig. [3d](#Fig3)). Most of these variants (159/210; 76%) were also found in accessible chromatin in PDXs and/or in primary ALL cells from patients (Fig. [3d](#Fig3)).

### Fig. 3. Identification of functional regulatory variants with reproducible and concordant effects in allele-specific stimulation of transcriptional activity.

![Fig. 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7c1c/11063049/c2c591625af7/41467_2024_48124_Fig3_HTML.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=11063049_41467_2024_48124_Fig3_HTML.jpg)

a 556 of the 1696 SNVs assayed are functional regulatory variants with reproducible (FDR < 0.05 in >2 cell lines) and concordant (same directionality in >2 cell lines) changes in allele-specific activity. b Frequency distribution plot showing the number of cell samples showing concordant and significant MPRA activity of variants. c Plot showing the distribution of log2-adjusted activity between alternative (Alt) and reference (Ref) alleles across 556 functional regulatory variants. 210 SNVs (in blue) mapped to accessible chromatin sites in ALL cell lines and 346 SNVs (in black) mapped only to accessible chromatin sites identified in primary ALL cells and/or PDXs. d Upset plot shows how many functional regulatory variants map to open chromatin in diverse ALL cell models. 210 of the 556 functional regulatory variants are found in accessible chromatin sites that were identified in an ALL cell line. Source data are provided in the Source Data file.

For additional validation using traditional luciferase reporter assays, we prioritized these 210 functional regulatory variants based on allele-specific effect size and selected high-ranking SNVs. Dual-luciferase reporter assays showed similar allele-specific changes in activity to that which was detected by MPRA for 7 SNVs tested (Supplementary Fig. [4a–k](#MOESM2)). In fact, a significant positive correlation (*p* = 0.0017) was observed between the allelic effects detected by MPRA and luciferase reporter assays (Supplementary Fig. [4l](#MOESM2)). Together, these analyses assessed the robustness of our MPRA screen of functional regulatory variants and identified 556 SNVs with reproducible and concordant allele-specific effects on gene regulation. Importantly, 210 of the 556 significant hits that were concordant in at least three cell lines were found in open chromatin sites in ALL cell lines and, therefore, warranted further exploration.

### Association of functional regulatory variants with putative gene targets

To better understand how these variants impact cellular phenotypes, we first determined if the 210 functional regulatory variants found in accessible chromatin sites in ALL cell lines could be directly associated with a target gene. While 35 functional regulatory variants were localized close (±2.5 kb) to nearby promoters (Fig. [4a](#Fig4), Supplementary Data [4](#MOESM7) and [6](#MOESM9)), 175 variants were promoter-distal (>2.5 kb), and therefore likely to map to CREs with unclear gene targets (Fig. [4a](#Fig4)). While CREs are often associated with the nearest genes, 3D chromatin looping methods are a more reliable method to associate a CRE with its target gene promoter. In pursuit of evidence-based association of promoters and specific CREs, we performed two related chromatin looping methods, H3K27Ac HiChIP^59^ and promoter capture HiC (CHiC)^39^, in 8 of 10 ALL cell lines used in MPRA and determined that 19 of the 175 non-promoter functional regulatory variants showed connectivity to distal promoters in the same cell line where allele-specific MPRA activity and chromatin accessibility were detected (Fig. [4a](#Fig4), Supplementary Data [6](#MOESM9)). Interestingly, H3K27Ac HiChIP and promoter CHiC called similar numbers of loops across all 8 cell lines (690,579 versus 660,313, respectively), but promoter CHiC loop calling was more consistent per cell line (Supplementary Fig. [5](#MOESM2), Supplementary Data [7](#MOESM10)). HiChIP detected no looping at any of the 556 reproducible and concordant SNVs from the MPRA, and the 19 SNVs showing connectivity to a promoter were solely detected by promoter CHiC, further highlighting the utility of this method in GWAS-oriented studies^41,60–63^.

### Fig. 4. Promoter CHiC identifies target genes of functional regulatory variants.

![Fig. 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7c1c/11063049/f2dc95ddb84a/41467_2024_48124_Fig4_HTML.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=11063049_41467_2024_48124_Fig4_HTML.jpg)

a Data show the number of functional regulatory variants mapping to open chromatin in cell lines that associate directly with promoters (within 2.5 kb) or that are distally promoter-connected via promoter CHiC. b MPRA data show distal regulatory variants in accessible chromatin (some promoter-connected by promoter CHiC data) exhibit stronger effects on allele-specific activity than promoter-associated functional regulatory variants. ANOVA with Kruskal–Wallis test was performed with Dunn’s correction for multiple comparisons. c Amongst distally promoter-connected functional regulatory, variants that map to intronic and distal intergenic sequences showed greater activity than those in UTRs. ANOVA with Kruskal–Wallis test was performed with Dunn’s correction for multiple comparisons. d, e Data show the ranked allele-specific activity distribution of MPRA data for d promoter-associated functional regulatory variants and e distally promoter-connected functional regulatory variants. All source data and statistical parameters are provided in the Source Data file.

In prioritizing functional regulatory variants, we were interested in the gene regulatory impact of variants at TSS-proximal promoter-associated versus TSS-distal promoter-connected CREs as measured by MPRA. Interestingly, we found that SNVs found at TSS-distal open chromatin sites, promoter-associated or not, showed higher allele-specific changes in MPRA activity than those at promoters (Fig. [4b](#Fig4)). While we acknowledge that many of the 156 variants for which we did not detect a relationship with a promoter are likely to have meaningful gene targets, we focused on CREs containing variants with known gene targets in ALL cells for functional validation. Amongst the TSS-distal promoter-connected functional regulatory variants, we found that distal intergenic and intronic SNVs showed significantly higher allele-specific activity than those in UTRs (Fig. [4c](#Fig4)). These data suggest that the most robust allelic effects attributable to these regulatory variants are likely to occur at distal intergenic and intronic sites >2.5 kb from the TSS of the target gene.

Next, we ranked TSS-proximal promoter-associated and TSS-distal promoter-connected functional regulatory variants by the geometric mean of their significant MPRA data to account for the magnitude of allele-specific activity and the reproducibility of a significant change across ALL cell lines (Fig. [4d, e](#Fig4)). This analysis identified *rs1247117* as the most robust functional regulatory variants, which we then pursued for mechanistic understanding (Fig. [4e](#Fig4)).

### rs1247117 determines genomic accessibility, PU.1 binding, and EIF3A expression

We pursued functional validation of *rs1247117* based on its highest-ranking geometric mean of MPRA allelic effect. *rs1247117* is in high LD with two GWAS sentinel variants (rs1312895, *r*^2^ = 0.99; *rs1247118*, *r*^2^ = 1) that are associated with persistence of MRD after induction chemotherapy^3^. This functional regulatory variant maps to a distal intergenic region harboring chromatin accessibility downstream of the *CACUL1* gene, for which it is an eQTL in EBV-transformed lymphocytes^37^. However, we found that *rs1247117* loops to the *EIF3A* promoter in Nalm6 B-ALL cells (Fig. [5a](#Fig5)). We, therefore, explored how this accessible chromatin site might recruit transcriptional regulators that would depend on the allele present at *rs1247117*. For this, we first performed ChIP-seq for RNA pol II and H3K27Ac, which confirmed RNA Pol II occupancy and H3K27Ac enrichment in Nalm6 cells, indicating that *rs1247117* is associated with an active CRE (Fig. [5a](#Fig5)). Through an examination of the underlying DNA sequence spanning *rs1247117*, we found that the reference guanine (G) risk allele at rs1247117 resides in a PU.1 transcription factor binding motif that is disrupted by the alternative adenine (A) allele (Fig. [5b](#Fig5)). Although the risk G allele is the reference allele, the alternative A allele is more common in human populations. Supporting PU.1 binding at this location, accessible chromatin profiling in primary ALL cells identified an accessible chromatin site and PU.1 footprint spanning *rs1247117* in diverse ALL samples (Supplementary Fig. [6a, b](#MOESM2)). Significantly greater chromatin accessibility at *rs1247117* was also observed in heterozygous (GA) patient samples compared to patient samples homozygous for the alternative A allele (Supplementary Fig. [6c](#MOESM2)), and the G allele at *rs1247117* harbored significantly greater ATAC-seq read count compared to the A allele (Supplementary Fig. [6d](#MOESM2)). Importantly, we determined that PU.1 was bound at this site in Nalm6 cells using CUT and RUN^64^ (Fig. [5a](#Fig5)).

### Fig. 5. rs1247117 impacts PU.1 occupancy and EIF3A expression.

![Fig. 5](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7c1c/11063049/3b2242adbbfb/41467_2024_48124_Fig5_HTML.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=11063049_41467_2024_48124_Fig5_HTML.jpg)

a IGV genome browser image in Nalm6 cells showing the genomic context, chromatin accessibility, and EIF3A promoter connectivity using promoter CHiC of the top functional regulatory variant, rs1247117, with the highest allele-specific MPRA activity. Genomic binding profiles are also shown for RNA polymerase II (RNA Pol2), histone H3 lysine 27 acetylation (H3K27Ac), and PU.1. b rs1247117 lies in a PU.1 binding motif. The human genome reference sequence, Nalm6 genome sequence, location of rs1247117, and PU.1 position weight matrix are shown. c Design of biotinylated DNA probes for in vitro rs1247117 pulldown. d Biotinylated DNA pulldown shows rs1247117 allele-dependent enrichment of PU.1 binding. Blot shown is representative of two independent experiments. Densitometric quantification of two blots is shown. e CRISPR/Cas9 was used to change the allele at rs1247117 from A > G in Nalm6 cells. Data show the location of gRNA and ssODN, as well as NGS reads obtained from clone 1 and 2 at rs1247117. f PU.1 ChIP-PCR shows increased PU.1 binding at the rs1247117 locus using two A > G modified clones and 3 primer sets. Data shown are mean ± SD of three independent experiments for each primer set. Two-way ANOVA with Dunnett’s multiple comparisons correction, n = 3. g ATAC-seq data normalized for frequency of reads in peaks (FRIP) show a significantly higher count of G nucleotides in two clones of A > G modified Nalm6 cells compared to the count of A nucleotides detected in parental Nalm6 cells. Data shown are the mean ± SD. Bonferroni corrected, two-tailed Student’s T tests, n = 3. h Western blots and quantification showing decreased EIF3A expression in A > G modified Nalm6 cells. Blots shown are representative of three independent experiments. Quantification data shown are the mean ± SD. Two-tailed Student’s T tests compare parental Nalm6 to combined data from A > G clones, n = 3. All source data and statistical parameters are provided in the Source Data file.

Nalm6 cells contain the alternative A allele that disrupts the PU.1 motif at *rs1247117*, yet our data suggests that this site still binds PU.1 (Fig. [5a, b](#Fig5)). This led us to hypothesize that PU.1 binding affinity for the PU.1 motif surrounding *rs1247117* would be strengthened by the risk G allele. Therefore, we designed biotinylated DNA probes containing two tandem 25-bp regions centered on reference G or alternative A allele-containing *rs1247117* to test this hypothesis (Fig. [5c](#Fig5)). Using biotinylated probes, we performed an in vitro DNA-affinity pulldown from Nalm6 nuclear lysate and found that while PU.1 was indeed bound to the alternative A allele, PU.1 was more robustly bound to the reference G allele at *rs1247117* (Fig. [5d](#Fig5)). To further assess the impact of the *rs1247117* allele on PU.1 binding, we changed the Nalm6 allele from A to G using CRISPR/Cas9 (Fig. [5e](#Fig5); AA = parental genotype, GG = mutated genotype). We used ChIP-PCR to determine that PU.1 binding was increased with the G allele relative to the A allele at the CRE containing *rs1247117* in two A > G Nalm6 clones across 3 unique primer sets within the PU.1 peak at rs1247117 that was detected in Nalm6 cells (Fig. [5f](#Fig5)). We then asked if transposase accessibility was also increased at the CRE containing *rs1247117* when the G allele was present. Using ATAC-seq, we found that accessibility was indeed increased at *rs1247117* in mutated Nalm6 cells with the G allele when compared to the parental Nalm6 cells containing the A allele (Fig. [5g](#Fig5)). These data suggest that the risk G allele increases genomic accessibility and the affinity of PU.1 binding at *rs1247117* relative to the alternative A allele.

We were next interested in how allele-specific PU.1 binding at *rs1247117* was related to the expression of the protein encoded by the connected gene, EIF3A. We found that the G allele, which increased recruitment of PU.1, resulted in decreased expression of EIF3A when compared to Nalm6 cells with the A allele (Fig. [5h](#Fig5)). These data suggest that PU.1 recruitment to the CRE containing *rs1247117* results in a net-repressive effect on EIF3A protein levels, and that less PU.1 recruitment with the A allele results in greater EIF3A expression.

### Deletion of CREs containing top MPRA SNVs demonstrates their impact on drug sensitivity

Clonal selection can lead to the accumulation of random SNVs and even larger structural variations^65^ that can confound functional interpretation of more complex *trans* phenotypic effects. Therefore, to examine the connection between *rs1247117* and the persistence of MRD after induction chemotherapy, we decided to use CRISPR/Cas9 to delete the CRE containing *rs1247117* in heterogeneous cell pools of Nalm6 and SUPB15 cells (rs1247117 del) to avoid clonal selection (Fig. [6a, b](#Fig6), Supplementary Fig. [7a](#MOESM2)). Given that loss of the CRE containing *rs1247117* would abolish PU.1 recruitment at this region, we hypothesized that *rs1247117* del would result in increased EIF3A expression. Accordingly, we found that EIF3A expression was elevated in *rs1247117* del cells relative to parental Nalm6 and SUPB15 cells, respectively (Fig. [6c, d](#Fig6), Supplementary Fig. [7b](#MOESM2)), further supporting an inverse relationship between PU.1 binding at *rs1247117* and EIF3A expression.

### Fig. 6. Deletion of the CRE containing rs1247117 leads to increased EIF3A expression and sensitizes cells to vincristine.

![Fig. 6](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7c1c/11063049/818828fc7445/41467_2024_48124_Fig6_HTML.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=11063049_41467_2024_48124_Fig6_HTML.jpg)

a Diagram on the left showing the genomic context of the rs1247117 CRE deletion in Nalm6 cells in relation to chromatin accessibility, PU.1 binding and rs1247117. Black bar represents ATAC-seq peak, green par represents PU.1 peak, and red bar represents region deleted using CRISPR/Cas9 genome editing. b Gel shows validation of deletion using primers flanking deleted region. Arrow points to PCR fragment with deletion in heterogeneous Nalm6 cell pools harboring deletion compared to wild-type parental Nalm6 cells. c EIF3A gene expression is upregulated upon deletion of the CRE containing rs1247117. RT-qPCR data show the mean ± SD of three independent experiments. Two-tailed Student’s T test. d Western blots and quantification showing increased EIF3A expression in rs1247117 del Nalm6 cells. Blots shown are representative of four independent experiments. Quantification data show the mean ± SD. Two-tailed Student’s T tests, n = 4. e–g Drug sensitivity data comparing viability relative to vehicle treatment of wild-type parental Nalm6 cells and Nalm6 cells with rs1247117 CRE deletion after vincristine (VCR) treatment for 24 (n = 3), 48 (n = 3) and 72 (n = 3) hours at the indicated concentrations. Non-linear regression and F test analysis indicate that these dose-response curves are significantly different. h Caspase 3/7 activity assays comparing Caspase activity relative to vehicle treatment of wild-type parental Nalm6 cells and Nalm6 cells with rs1247117 CRE deletion after vincristine (VCR) treatment for 72 hours at the indicated concentrations (n = 3). Dose-response curves of non-linear regression indicate that these curves are significantly different. Non-linear regression and F test analysis indicate that these dose-response curves are significantly different. All source data are provided in the Source Data file.

Because the risk G allele at *rs1247117* was also associated with vincristine resistance in primary ALL cells from patients, we additionally sought to determine the impact of the CRE deletion containing *rs1247117* on cellular response to vincristine treatment. We hypothesized that because the risk G allele is associated with enhanced PU.1 binding and resistance to vincristine, complete disruption of PU.1 binding in Nalm6 cells harboring the CRE deletion would show increased sensitivity to vincristine relative to parental Nalm6 cells. As predicted, Nalm6 cells with the CRE deletion exhibited significantly increased sensitivity to vincristine across a range of concentrations after 24, 48, and 72 hours of treatment (Fig. [6e–g](#Fig6)), and we found consistent effects on cell viability in SUPB15 cells (Supplementary Fig. [7c](#MOESM2)). Consistent with enhanced sensitivity to vincristine, we also found increased caspase 3/7 activity in *rs1247117* del Nalm6 cells relative to parental Nalm6 cells after 72hrs and across a range of vincristine concentrations (Fig. [6h](#Fig6)). These data suggest that a functional regulatory variant alters the binding affinity of a key transcription factor, PU.1, and disruption of this locus impacts EIF3A expression and vincristine sensitivity in ALL cells. To further validate our methodology utilizing CRISPR/Cas9 to delete CREs, we deleted CREs spanning two additional top variants, *rs7426865* and *rs12660691* (see Fig. [4e](#Fig4)), that was associated with the ex vivo resistance to 6-mercaptopurine and dexamethasone, respectively, in primary ALL cells. Deletion of these CREs also impacted protein expression and sensitivity to the associated chemotherapeutic agent, thereby supporting our functional approach (Supplementary Figs. [8](#MOESM2) and [9](#MOESM2)).

We next wanted to connect EIF3A directly to vincristine resistance. Given that EIF3A is an essential gene per the Broad Institute’s DepMap, we opted to test the hypothesis EIF3A overexpression alone was sufficient to impact the Nalm6 cell response to vincristine. We, therefore, used lentiviral transduction to overexpress EIF3A in Nalm6 cells and compared EIF3A overexpression (EIF3A OE) cells to control infected cells (Nalm6 WT, Supplementary Fig. [10a](#MOESM2)). Using two independent infections of EIF3A OE, we found that at 48 hr and 72 hr, EIF3A OE cells were more sensitive to vincristine than Nalm6 WT cells (Supplementary Fig. [10b](#MOESM2)). These data suggest that EIF3A expression impacts the ALL cell response to vincristine, with higher expression sensitizing cells to the drug, and further establishes this gene as the likely target of the association.

## Discussion

Using MPRA, we systematically interrogated the functional effects of inherited noncoding variation associated with relapse, persistence of MRD after induction chemotherapy, and/or ex vivo chemotherapeutic drug resistance in childhood ALL. We refined our search to regulatory variants that were found in accessible chromatin sites in 161 ALL cell models, including primary ALL cells from patients, PDXs, and ALL cell lines, as those noncoding regions were likely to be participating in transcriptional regulation. Using MPRA we identified 556 functional regulatory variants showing reproducible and concordant changes in an allele-specific gene regulatory activity. To further explore the impact of these variants on gene regulation in ALL cell lines, we selected a subset of functional regulatory variants from MPRA that were within an accessible chromatin site in an ALL cell line. We overcame difficulties in associating promoter-distal functional regulatory variants with gene targets using promoter CHiC and found 19 variants with robust looping to a distal promoter, as well as 35 promoter-associated functional regulatory variants.

We identified *rs1247117* as the top functional regulatory variant, showing the highest geometric mean of differential transcription activity, which was identified in 9 of 10 ALL cell lines assayed by MPRA. We found that the allele present at *rs1247117* was a determinant of PU.1 transcription factor binding, with the risk G allele leading to greater chromatin accessibility and PU.1 binding affinity. Interestingly, the allele-specific activities as measured by MPRA and traditional dual-luciferase reporter assays suggest that the reference G allele at *rs1247117* stimulates transcription more than the alternative A allele, and we suspect this is driven by greater PU.1 binding affinity in these episomal assays. However, our endogenous genetic manipulation that disrupted PU.1 binding at this locus in Nalm6 cells suggests that *EIF3A* expression is driven inversely to PU.1 binding. Moreover, complete disruption of PU.1 binding resulted in greater sensitivity to vincristine, which is consistent with the risk G allele contributing to both greater PU.1 binding affinity and vincristine resistance in primary ALL cells from patients. This discrepancy may be due to the ability of PU.1 to act in an activating or repressing manner on gene expression dependent on its genomic context and other transcriptional regulators present^66,67^. While not addressed within the scope of this work, our hypothesis is that MPRA and luciferase reporter assays, which are episomal and utilize a non-native minimal promoter, detected transcription-activating PU.1 activities rather than the PU.1 repressive activities we detect within the endogenous locus. Collectively, these observations stress the importance of performing subsequent functional follow-up experimentation within an endogenous sequence context.

The *rs1247117* risk G allele frequency is ~20-30% in African (33%), American (29%), and East Asian (23%) populations, and only ~10% in Europeans (10%) and South Asians (11%). Notably, vincristine-associated neurotoxicity is more common in Europeans compared to African Americans^68^. However, this effect has been attributed to greater cytochrome P450 3A5 (*CYP3A5*) allele expression in African Americans compared to Europeans. Importantly, this variant was originally associated with the persistence of MRD after induction chemotherapy^3^, and numerous reports demonstrate poorer outcomes in patients of African and American ancestries compared to White populations^69–73^. Overall, our results are consistent with these studies and suggest that greater resistance to vincristine, which is typically given during induction chemotherapy, is a potential link between the risk G allele at *rs1247117* and the persistence of MRD in patients after induction chemotherapy.

Although the risk G allele at *rs1247117* is associated with decreased *EIF3A* expression and increased risk of MRD after induction chemotherapy and vincristine resistance, it remains unclear why *EIF3A* expression might impact vincristine efficacy. *EIF3A* expression has been correlated with cell cycle progression, and others have shown that *EIF3A* knockdown leads to decreased cell proliferation^74^. However, vincristine can act on microtubules to rapidly kill cells during G1 and later the mitotic spindle to arrest cells during metaphase, so increased *EIF3A* expression may facilitate cell cycle progression, thus increasing the rate at which metaphase mitotic spindles are disrupted by vincristine^75^. *EIF3A* expression has also been previously linked to chemotherapeutic sensitivity in both melanoma and lung cancer^76,77^. *EIF3A* expression led to decreased phosphorylation of ERK, supporting the effect of vemurafenib-induced MAP kinase signaling blockade, while *EIF3A* loss led to sustained activation of ERK and therapeutic resistance^76^. Interestingly, ERK activation is important in G1/S progression, and therefore, it follows that EIF3A-dependent inhibition of ERK may support the rapid killing of cells in the G1 phase shortly after initial vincristine treatment^75,78^. This notion is supported by significantly greater sensitivity to vincristine of ALL cells harboring greater *EIF3A* expression through disruption of a distal CRE after just 24 hours of treatment.

The regulatory variants assayed in this study were originally discovered from GWAS in patient samples, and most of our functional regulatory variant hits from MPRA were present in accessible chromatin sites found only in primary ALL cells from patients. Consequently, these data highlight both substantial differences in the chromatin landscape between immortalized cell lines and primary cells and a limitation of our study that relied on the functional exploration of top regulatory variants in ALL cell line models. An optimal approach would be to validate top functional regulatory variants in patient samples; however, this is not currently feasible due to the limited duration of patient sample viability in culture for genetic manipulation. Nonetheless, future implementation of promoter CHiC in patient samples can be used to map gene connectivity of promoter-distal functional regulatory variants found only in primary cells, and these gene targets can then be genetically disrupted in ALL cell line models for functional validation.

We acknowledge that our MPRA analysis only provides information on the allele-specific gene regulatory effects of these variants, and any connection with pharmacological effects would require additional validation, as we have provided and demonstrated with experimentation. Additionally, because genotyping was performed in primary ALL cells for variants associated with ex vivo drug resistance, we cannot exclude the possibility that some of our tested variants are somatic and not germline. Nonetheless, because we were comprehensive and provided extensive functional data across diverse ALL cell samples for this large set of variants, we believe that our rich dataset can additionally be utilized as an important resource for future GWAS studies on ALL pharmacogenomics. Namely, as future clinical trials are performed and additional GWAS variants impacting clinical phenotypes are discovered, our dataset can be queried to determine if newly discovered variants harbor functional effects on gene expression.

This translational work represents the largest functional investigation of inherited noncoding variation that is associated with diverse pharmacological traits in ALL to date. Our study identified hundreds of functional regulatory variants with significant, reproducible, and concordant allele-specific effects on gene expression, and further connected gene regulatory disruptions to differences in chemotherapy response through alterations in antileukemic drug sensitivity in ALL cells. Collectively, these data support the importance of noncoding, gene regulatory disruptions in the pharmacogenomics of ALL treatments. The further functional investigation of these regulatory variants and the discovery of additional inherited variants impacting therapeutic outcomes can be used by clinicians to tailor therapies based on a patient’s unique genetic makeup through precision or personalized medicine.

## Methods

### Ethics

This study complies with all relevant ethical regulations, and the relevant study protocols were approved by the institutional review boards at St. Jude Children’s Research Hospital and Children’s Hospital of Philadelphia, as well as the Institutional Animal Care and Use Committee of St. Jude.

### Patient samples and consent

All patients or their legal guardians provided written informed consent. The use of these samples was approved by the institutional review board at St. Jude Children’s Research Hospital. Ex vivo drug sensitivity assay and genotyping datasets were obtained from primary ALL cells of patients enrolled in St. Jude Children’s Research Hospital (Memphis, Tennessee) Total Therapy XV (TOTXV, [NCT00137111](https://clinicaltrials.gov/ct2/show/NCT00137111)) and Total Therapy XVI (TOTXVI, [NCT00549848](https://clinicaltrials.gov/ct2/show/NCT00549848)) protocols^10,21^. RNA-seq data was obtained from primary ALL cells of patients enrolled on TOTXVI. Patient sample ATAC-seq data was obtained from published studies^55,56^. No associations between survival or treatment response and next-generation sequencing data were performed on samples from patients who are enrolled in ongoing clinical trials. No considerations for sex and/or gender were made for this study, and the pharmacogenomic traits were identified regardless of sex and or gender.

### Selection of SNVs impacting treatment outcome in patients from published GWAS

We chose 13 relapse SNVs with *p* < 1 × 10^−5^ from Yang et al.^4^, 19 ancestry-specific SNVs relapse SNPs associated with relapse in both discovery and replication ALL patient cohorts (*p* < 0.05) from Karol et al.^5^ and 3 SNVs associated with persistence of MRD with *p* < 1 × 10^−6^ from Yang et al.^3^ (*n* = 35). In addition, we chose all SNVs (*n* = 126; Karol et al.^5^ *n* = 104, Yang et al.^4^ *n* = 10 and Yang et al.^3^ *n* = 12) from these GWAS with nominal genome-wide association (*p* < 0.05) but that were additionally associated with ex vivo drug resistance (*p* < 0.05) in primary ALL cells from patients. Relapse SNVs additionally associated with ex vivo drug resistance were already identified by Karol et al.^5^. GWAS for ex vivo drug resistance (*p* < 0.05) using primary ALL cells from patients enrolled on TOTXV and TOTXVI protocols were performed on SNVs identified in Yang et al.^4^ and Yang et al.^3^ (see Methods below).

### Ex vivo drug sensitivity assays in primary ALL cells

Ex vivo drug sensitivity assay data was previously published^10,21^. The ex vivo drug sensitivity assays are previously described^21^. Briefly, primary leukemia cells were isolated from the bone marrow or peripheral blood of newly diagnosed ALL patients from St. Jude Total Therapy XVI protocol (TOTXVI, [NCT00549848](https://clinicaltrials.gov/ct2/show/NCT00549848)) and tested for antileukemic drug sensitivity by a 96-hour MTT assay using a range of drug concentrations. Primary ALL cells were treated with prednisolone (*n* = 320), dexamethasone (*n* = 312), bacterially derived L-asparaginase (*n* = 335), vincristine (*n* = 323), 6-mercaptopurine (*n* = 344) and 6-thioguanine (*n* = 325). Following drug treatment, the lethal concentration resulting in 50% viability (LC_50_) was calculated for each patient sample.

### Genotyping in primary ALL cells

Genotyping data was previously published^21^. Briefly, genomic DNA was collected from primary ALL cells of patients enrolled on TOTXV and TOTXVI protocols using Blood and Cell Culture DNA kit (Qiagen), and SNP genotyping was performed using the Affymetrix Gene ChIP Human Mapping 500 K array or Affymetrix Genome-Wide Human SNP 6.0 array. Genotypes were called BRLMM algorithm in the Affymetrix GTYPE software ([http://www.affymetrix.com/products/software/specific/gtype.affx](http://www.affymetrix.com/products/software/specific/gtype.affx)) as previously described^79^. We excluded SNVs for call rates <95% among patients or minor allele frequencies <1%. SNPs were additionally filtered based on call rate and Hardy Weinberg equilibrium (*p* > 1e-4). Genome-wide genotypes were imputed using Topmed imputation server.

### Ex vivo drug resistance GWAS

Imputed genotypes with *r*^2^ ≥ 0.6 were tested for their association with LC50. To test the association between LC50 and genotype, LC50 were first log10 transformed and treated as a continuous dependent variable and genotype as independent variable. Genotypes were coded as 0, 1, and 2. The association between LC50 and genotype was tested using a general linear regression model for additive genotypic effect using PLINK. Two-tailed p-values were generated using a Wald test. Top 5 principal components were included as covariates in the regression model to control for population stratification. Ex vivo drug resistance GWAS was performed using primary ALL cells from patients enrolled on TOTXVI protocol. All statistical analysis was performed in R v4.0.2. SNVs that were nominally associated with ex vivo drug resistance (*p* < 0.05) and that were also identified as eQTLs in the same patient cohort (*p* < 0.05; see Methods below) were included in the MPRA analysis. For ex vivo drug resistance associations with treatment outcome SNVs (see Methods above), GWAS was performed from patients enrolled on TOTXV and TOTXVI protocols separately, and the GWAS results were then combined through meta-analysis using METAL.

### Cell lines, cell culture, and authentication

Leukemia cell lines (Nalm6 DSMZ ACC128, Jurkat DSMZ ACC282, B-ALL-1 DSMZ ACC742, 697 DSMZ ACC42, CEM DSMZ ACC240, REH DSMZ ACC22, P12-Ichikawa DSMZ ACC34, SEM DSMZ ACC546, Loucy DSMZ ACC394, DND41 DSMZ ACC525, HSB2 DSMZ ACC435, MOLT16 DSMZ ACC29, RS411 DSMZ ACC508, SUPB15 DSMZ ACC389) were cultured in RPMI 1640 + 10% fetal bovine serum. Cells were not cultured beyond 25 passages after we received them for experiments in this work. PCR-based mycoplasma testing determined that the cell lines were negative for mycoplasma contamination. STR profiling was confirmed before freezing down cell aliquots before passage 5 after we received them.

### eQTL mapping in primary ALL cells

RNA-seq data was obtained from a previous publication^21^. Briefly, total RNA from primary ALL cells was isolated using RNAeasy Mini kit (Qiagen), and mRNA sequencing using an Illumina HiSeq platform was performed by the Hartwell Center for Bioinformatics and Biotechnology at St. Jude Children’s Research Hospital. Genotype information and RNA expression data from patients enrolled on TOTXVI protocol were correlated to identify eQTLs. QTL mapping was performed using multiple linear regression and log-transformed FPKM gene expression as the dependent variable and genotype as the independent variable. Genotypes were coded as 0, 1, and 2. Patient genetic ancestry was included in the linear model as covariates. Two-tailed p-values were generated using a Wald test. All statistical analysis was performed in R v4.0.2.

### Quantitative real-time PCR (qPCR)

See [Supplemental Methods](#MOESM2) for more details.

### Dual-luciferase reporter assays

See [Supplemental Methods](#MOESM2) for more details.

### MPRAs

MPRA oligo design, plasmid library construction, plasmid transfection, sequencing, and analysis are provided in [Supplemental Methods](#MOESM2).

### Epigenomic profiling in ALL cells

See [Supplemental Methods](#MOESM2) for more details.

### PU.1 in vitro binding affinity assay

DNA pulldown assay was adopted from the previous article and performed using a modified protocol^80^. See [Supplemental Methods](#MOESM2) for more details.

### CRISPR/Cas9 deletion

*rs1247117* deletions in Nalm6 were generated using CRISPR-Cas9 technology. See [Supplemental Methods](#MOESM2) for more details.

### Nalm6 vincristine sensitivity assays

Drug viability assays were performed as previously with slight modification^22^. Additional information is provided in [Supplemental Methods](#MOESM2).

### Statistical analysis

See [Supplemental Methods](#MOESM2) for more details on statistical analyses performed.

### Reporting summary

Further information on research design is available in the [Nature Portfolio Reporting Summary](#MOESM18) linked to this article.

### Supplementary information

### Supplementary information

### Source data

## Acknowledgements

We would like to thank the Hartwell Center at St. Jude for ATAC-seq, ChIP-seq, Cut & Run, promoter CHiC, HiChIP, and MPRA next-generation sequencing. We would also like to thank Jeremy Hunt and Brandon Smart for their technical support. This work was performed at St. Jude Children’s Research Hospital and was also conducted in part by the ECOG-ACRIN Cancer Research Group. This work is supported by the National Cancer Institute (R01CA234490—Daniel Savic, P30CA021765—St. Jude Children’s Research Hospital, U10CA180820—ECOG-ACRIN Medical Research Foundation, UG1CA232760—MAYO Clinic, Rochester and UG1CA189859—Montefiore Medical Center), the National Institute of General Medical Studies (P50GM115279—St. Jude Children’s Research Hospital) and the American Lebanese Syrian Associated Charities (ALSAC). The content is solely the responsibility of the authors and does not necessarily represent the official views of the National Institutes of Health.

## Author contributions

Kashi R.B., R.J.M.: investigation, writing—original draft, review, and editing, visualization, supervision, methodology, conceptualization, Kelly R.B.: formal analysis, data curation, D.C.F., B.S.H., J.D.D., B.P.B., S.Y.: investigation, W.Y., C.C.: formal analysis, K.R.C., C.S.M., E.J., S.M.K., W.S., H.I., S.J., C.P., J.J.Y.: resources, E.P., M.R.L., M.V.R., W.E.E.: resources, funding acquisition, S.P.-M.: methodology, D.S.: conceptualization, methodology, supervision, project administration, funding acquisition, writing—original draft, review, and editing.

## Peer review

### Peer review information

*Nature Communications* thanks Martin Walsh, Andrew DeWan and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. A peer review file is available.

## Data availability

The ATAC-seq, promoter CHiC, HiChIP, RNA Pol II, and PU.1 genomic binding data generated in this study are publicly available in the GEO database under accession code [GSE224204](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE224204). The fresh patient sample ATAC-seq data used in this study are publicly available in the GEO database under accession code [GSE226400](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE226400). The frozen patient sample ATAC-seq data used in this study are publicly available in the GEO database under accession code [GSE161501](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE161501). H3K27Ac ChIP-seq data, “[GSE175482](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE175482) Nalm6 H3K27ac 0 hr merged.bw”, used in this study are publicly available in the GEO database under accession code [GSE175482](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE175482). All other data generated in this study are provided in [Supplementary Information](#MOESM2) and Source Data file. [Source data](#Sec29) are provided with this paper.

## Competing interests

The authors declare no competing interests.

## Footnotes

## Supplementary information

The online version contains supplementary material available at 10.1038/s41467-024-48124-4.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

The ATAC-seq, promoter CHiC, HiChIP, RNA Pol II, and PU.1 genomic binding data generated in this study are publicly available in the GEO database under accession code [GSE224204](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE224204). The fresh patient sample ATAC-seq data used in this study are publicly available in the GEO database under accession code [GSE226400](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE226400). The frozen patient sample ATAC-seq data used in this study are publicly available in the GEO database under accession code [GSE161501](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE161501). H3K27Ac ChIP-seq data, “[GSE175482](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE175482) Nalm6 H3K27ac 0 hr merged.bw”, used in this study are publicly available in the GEO database under accession code [GSE175482](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE175482). All other data generated in this study are provided in [Supplementary Information](#MOESM2) and Source Data file. [Source data](#Sec29) are provided with this paper.

### Supplementary Materials

### Data Availability Statement

The ATAC-seq, promoter CHiC, HiChIP, RNA Pol II, and PU.1 genomic binding data generated in this study are publicly available in the GEO database under accession code [GSE224204](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE224204). The fresh patient sample ATAC-seq data used in this study are publicly available in the GEO database under accession code [GSE226400](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE226400). The frozen patient sample ATAC-seq data used in this study are publicly available in the GEO database under accession code [GSE161501](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE161501). H3K27Ac ChIP-seq data, “[GSE175482](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE175482) Nalm6 H3K27ac 0 hr merged.bw”, used in this study are publicly available in the GEO database under accession code [GSE175482](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE175482). All other data generated in this study are provided in [Supplementary Information](#MOESM2) and Source Data file. [Source data](#Sec29) are provided with this paper.

## References

1. Hunger SP, Mullighan CG. Acute lymphoblastic leukemia in children. N. Engl. J. Med. 2015;373:1541–1552. doi: 10.1056/NEJMra1400972.  [DOI](https://doi.org/10.1056/NEJMra1400972) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26465987/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%20Engl.%20J.%20Med.&title=Acute%20lymphoblastic%20leukemia%20in%20children&author=SP%20Hunger&author=CG%20Mullighan&volume=373&publication_year=2015&pages=1541-1552&pmid=26465987&doi=10.1056/NEJMra1400972&)

2. Oriol A, et al. Outcome after relapse of acute lymphoblastic leukemia in adult patients included in four consecutive risk-adapted trials by the PETHEMA Study Group. Haematologica. 2010;95:589–596. doi: 10.3324/haematol.2009.014274.  [DOI](https://doi.org/10.3324/haematol.2009.014274) | [PMC free article](/articles/PMC2857188/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20145276/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Haematologica&title=Outcome%20after%20relapse%20of%20acute%20lymphoblastic%20leukemia%20in%20adult%20patients%20included%20in%20four%20consecutive%20risk-adapted%20trials%20by%20the%20PETHEMA%20Study%20Group&author=A%20Oriol&volume=95&publication_year=2010&pages=589-596&pmid=20145276&doi=10.3324/haematol.2009.014274&)

3. Yang JJ, et al. Genome-wide interrogation of germline genetic variation associated with treatment response in childhood acute lymphoblastic leukemia. JAMA. 2009;301:393–403. doi: 10.1001/jama.2009.7.  [DOI](https://doi.org/10.1001/jama.2009.7) | [PMC free article](/articles/PMC2664534/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19176441/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA&title=Genome-wide%20interrogation%20of%20germline%20genetic%20variation%20associated%20with%20treatment%20response%20in%20childhood%20acute%20lymphoblastic%20leukemia&author=JJ%20Yang&volume=301&publication_year=2009&pages=393-403&pmid=19176441&doi=10.1001/jama.2009.7&)

4. Yang JJ, et al. Genome-wide association study identifies germline polymorphisms associated with relapse of childhood acute lymphoblastic leukemia. Blood. 2012;120:4197–4204. doi: 10.1182/blood-2012-07-440107.  [DOI](https://doi.org/10.1182/blood-2012-07-440107) | [PMC free article](/articles/PMC3501717/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23007406/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Genome-wide%20association%20study%20identifies%20germline%20polymorphisms%20associated%20with%20relapse%20of%20childhood%20acute%20lymphoblastic%20leukemia&author=JJ%20Yang&volume=120&publication_year=2012&pages=4197-4204&pmid=23007406&doi=10.1182/blood-2012-07-440107&)

5. Karol SE, et al. Genetics of ancestry-specific risk for relapse in acute lymphoblastic leukemia. Leukemia. 2017;31:1325–1332. doi: 10.1038/leu.2017.24.  [DOI](https://doi.org/10.1038/leu.2017.24) | [PMC free article](/articles/PMC5462853/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28096535/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Leukemia&title=Genetics%20of%20ancestry-specific%20risk%20for%20relapse%20in%20acute%20lymphoblastic%20leukemia&author=SE%20Karol&volume=31&publication_year=2017&pages=1325-1332&pmid=28096535&doi=10.1038/leu.2017.24&)

6. Cave H, et al. Clinical significance of minimal residual disease in childhood acute lymphoblastic leukemia. European Organization for Research and Treatment of Cancer–Childhood Leukemia Cooperative Group. N. Engl. J. Med. 1998;339:591–598. doi: 10.1056/NEJM199808273390904.  [DOI](https://doi.org/10.1056/NEJM199808273390904) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9718378/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%20Engl.%20J.%20Med.&title=Clinical%20significance%20of%20minimal%20residual%20disease%20in%20childhood%20acute%20lymphoblastic%20leukemia.%20European%20Organization%20for%20Research%20and%20Treatment%20of%20Cancer%E2%80%93Childhood%20Leukemia%20Cooperative%20Group&author=H%20Cave&volume=339&publication_year=1998&pages=591-598&pmid=9718378&doi=10.1056/NEJM199808273390904&)

7. Borowitz MJ, et al. Minimal residual disease detection in childhood precursor-B-cell acute lymphoblastic leukemia: relation to other risk factors. A Children’s Oncology Group study. Leukemia. 2003;17:1566–1572. doi: 10.1038/sj.leu.2403001.  [DOI](https://doi.org/10.1038/sj.leu.2403001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12886244/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Leukemia&title=Minimal%20residual%20disease%20detection%20in%20childhood%20precursor-B-cell%20acute%20lymphoblastic%20leukemia:%20relation%20to%20other%20risk%20factors.%20A%20Children%E2%80%99s%20Oncology%20Group%20study&author=MJ%20Borowitz&volume=17&publication_year=2003&pages=1566-1572&pmid=12886244&doi=10.1038/sj.leu.2403001&)

8. Coustan-Smith E, et al. Clinical importance of minimal residual disease in childhood acute lymphoblastic leukemia. Blood. 2000;96:2691–2696. doi: 10.1182/blood.V96.8.2691.  [DOI](https://doi.org/10.1182/blood.V96.8.2691) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11023499/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Clinical%20importance%20of%20minimal%20residual%20disease%20in%20childhood%20acute%20lymphoblastic%20leukemia&author=E%20Coustan-Smith&volume=96&publication_year=2000&pages=2691-2696&pmid=11023499&doi=10.1182/blood.V96.8.2691&)

9. Zhou J, et al. Quantitative analysis of minimal residual disease predicts relapse in children with B-lineage acute lymphoblastic leukemia in DFCI ALL Consortium Protocol 95-01. Blood. 2007;110:1607–1611. doi: 10.1182/blood-2006-09-045369.  [DOI](https://doi.org/10.1182/blood-2006-09-045369) | [PMC free article](/articles/PMC1975844/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17485550/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Quantitative%20analysis%20of%20minimal%20residual%20disease%20predicts%20relapse%20in%20children%20with%20B-lineage%20acute%20lymphoblastic%20leukemia%20in%20DFCI%20ALL%20Consortium%20Protocol%2095-01&author=J%20Zhou&volume=110&publication_year=2007&pages=1607-1611&pmid=17485550&doi=10.1182/blood-2006-09-045369&)

10. Lee SHR, et al. Pharmacotypes across the genomic landscape of pediatric acute lymphoblastic leukemia and impact on treatment response. Nat. Med. 2023;29:170–179. doi: 10.1038/s41591-022-02112-7.  [DOI](https://doi.org/10.1038/s41591-022-02112-7) | [PMC free article](/articles/PMC9873558/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36604538/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Med.&title=Pharmacotypes%20across%20the%20genomic%20landscape%20of%20pediatric%20acute%20lymphoblastic%20leukemia%20and%20impact%20on%20treatment%20response&author=SHR%20Lee&volume=29&publication_year=2023&pages=170-179&pmid=36604538&doi=10.1038/s41591-022-02112-7&)

11. Pieters R, et al. Relation of cellular drug resistance to long-term clinical outcome in childhood acute lymphoblastic leukaemia. Lancet. 1991;338:399–403. doi: 10.1016/0140-6736(91)91029-T.  [DOI](https://doi.org/10.1016/0140-6736(91)91029-T) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1678081/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Relation%20of%20cellular%20drug%20resistance%20to%20long-term%20clinical%20outcome%20in%20childhood%20acute%20lymphoblastic%20leukaemia&author=R%20Pieters&volume=338&publication_year=1991&pages=399-403&pmid=1678081&doi=10.1016/0140-6736(91)91029-T&)

12. Bosanquet AG. Correlations between therapeutic response of leukaemias and in-vitro drug-sensitivity assay. Lancet. 1991;337:711–714. doi: 10.1016/0140-6736(91)90287-Y.  [DOI](https://doi.org/10.1016/0140-6736(91)90287-Y) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1672185/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Correlations%20between%20therapeutic%20response%20of%20leukaemias%20and%20in-vitro%20drug-sensitivity%20assay&author=AG%20Bosanquet&volume=337&publication_year=1991&pages=711-714&pmid=1672185&doi=10.1016/0140-6736(91)90287-Y&)

13. Hwang WS, Chen LM, Huang SH, Wang CC, Tseng MT. Prediction of chemotherapy response in human leukemia using in vitro chemosensitivity test. Leuk. Res. 1993;17:685–688. doi: 10.1016/0145-2126(93)90074-U.  [DOI](https://doi.org/10.1016/0145-2126(93)90074-U) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8355513/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Leuk.%20Res.&title=Prediction%20of%20chemotherapy%20response%20in%20human%20leukemia%20using%20in%20vitro%20chemosensitivity%20test&author=WS%20Hwang&author=LM%20Chen&author=SH%20Huang&author=CC%20Wang&author=MT%20Tseng&volume=17&publication_year=1993&pages=685-688&pmid=8355513&doi=10.1016/0145-2126(93)90074-U&)

14. Pieters R, Kaspers GJ, Klumper E, Veerman AJ. Clinical relevance of in vitro drug resistance testing in childhood acute lymphoblastic leukemia: the state of the art. Med Pediatr. Oncol. 1994;22:299–308. doi: 10.1002/mpo.2950220502.  [DOI](https://doi.org/10.1002/mpo.2950220502) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8127253/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Med%20Pediatr.%20Oncol.&title=Clinical%20relevance%20of%20in%20vitro%20drug%20resistance%20testing%20in%20childhood%20acute%20lymphoblastic%20leukemia:%20the%20state%20of%20the%20art&author=R%20Pieters&author=GJ%20Kaspers&author=E%20Klumper&author=AJ%20Veerman&volume=22&publication_year=1994&pages=299-308&pmid=8127253&doi=10.1002/mpo.2950220502&)

15. Klumper E, et al. In vitro cellular drug resistance in children with relapsed/refractory acute lymphoblastic leukemia. Blood. 1995;86:3861–3868. doi: 10.1182/blood.V86.10.3861.bloodjournal86103861.  [DOI](https://doi.org/10.1182/blood.V86.10.3861.bloodjournal86103861) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7579354/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=In%20vitro%20cellular%20drug%20resistance%20in%20children%20with%20relapsed/refractory%20acute%20lymphoblastic%20leukemia&author=E%20Klumper&volume=86&publication_year=1995&pages=3861-3868&pmid=7579354&doi=10.1182/blood.V86.10.3861.bloodjournal86103861&)

16. Hongo T, Yajima S, Sakurai M, Horikoshi Y, Hanada R. In vitro drug sensitivity testing can predict induction failure and early relapse of childhood acute lymphoblastic leukemia. Blood. 1997;89:2959–2965. doi: 10.1182/blood.V89.8.2959.  [DOI](https://doi.org/10.1182/blood.V89.8.2959) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9108416/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=In%20vitro%20drug%20sensitivity%20testing%20can%20predict%20induction%20failure%20and%20early%20relapse%20of%20childhood%20acute%20lymphoblastic%20leukemia&author=T%20Hongo&author=S%20Yajima&author=M%20Sakurai&author=Y%20Horikoshi&author=R%20Hanada&volume=89&publication_year=1997&pages=2959-2965&pmid=9108416&doi=10.1182/blood.V89.8.2959&)

17. Kaspers GJ, et al. In vitro cellular drug resistance and prognosis in newly diagnosed childhood acute lymphoblastic leukemia. Blood. 1997;90:2723–2729. doi: 10.1182/blood.V90.7.2723.  [DOI](https://doi.org/10.1182/blood.V90.7.2723) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9326239/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=In%20vitro%20cellular%20drug%20resistance%20and%20prognosis%20in%20newly%20diagnosed%20childhood%20acute%20lymphoblastic%20leukemia&author=GJ%20Kaspers&volume=90&publication_year=1997&pages=2723-2729&pmid=9326239&doi=10.1182/blood.V90.7.2723&)

18. Den Boer ML, et al. Patient stratification based on prednisolone-vincristine-asparaginase resistance profiles in children with acute lymphoblastic leukemia. J. Clin. Oncol. 2003;21:3262–3268. doi: 10.1200/JCO.2003.11.031.  [DOI](https://doi.org/10.1200/JCO.2003.11.031) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12947061/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Oncol.&title=Patient%20stratification%20based%20on%20prednisolone-vincristine-asparaginase%20resistance%20profiles%20in%20children%20with%20acute%20lymphoblastic%20leukemia&author=ML%20Den%20Boer&volume=21&publication_year=2003&pages=3262-3268&pmid=12947061&doi=10.1200/JCO.2003.11.031&)

19. Frost BM, et al. Increased in vitro cellular drug resistance is related to poor outcome in high-risk childhood acute lymphoblastic leukaemia. Br. J. Haematol. 2003;122:376–385. doi: 10.1046/j.1365-2141.2003.04442.x.  [DOI](https://doi.org/10.1046/j.1365-2141.2003.04442.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12877664/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Haematol.&title=Increased%20in%20vitro%20cellular%20drug%20resistance%20is%20related%20to%20poor%20outcome%20in%20high-risk%20childhood%20acute%20lymphoblastic%20leukaemia&author=BM%20Frost&volume=122&publication_year=2003&pages=376-385&pmid=12877664&doi=10.1046/j.1365-2141.2003.04442.x&)

20. Holleman A, et al. Gene-expression patterns in drug-resistant acute lymphoblastic leukemia cells and response to treatment. N. Engl. J. Med. 2004;351:533–542. doi: 10.1056/NEJMoa033513.  [DOI](https://doi.org/10.1056/NEJMoa033513) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15295046/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%20Engl.%20J.%20Med&title=Gene-expression%20patterns%20in%20drug-resistant%20acute%20lymphoblastic%20leukemia%20cells%20and%20response%20to%20treatment&author=A%20Holleman&volume=351&publication_year=2004&pages=533-542&pmid=15295046&doi=10.1056/NEJMoa033513&)

21. Autry RJ, et al. Integrative genomic analyses reveal mechanisms of glucocorticoid resistance in acute lymphoblastic leukemia. Nat. Cancer. 2020;1:329–344. doi: 10.1038/s43018-020-0037-3.  [DOI](https://doi.org/10.1038/s43018-020-0037-3) | [PMC free article](/articles/PMC7467080/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32885175/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Cancer&title=Integrative%20genomic%20analyses%20reveal%20mechanisms%20of%20glucocorticoid%20resistance%20in%20acute%20lymphoblastic%20leukemia&author=RJ%20Autry&volume=1&publication_year=2020&pages=329-344&pmid=32885175&doi=10.1038/s43018-020-0037-3&)

22. Bergeron BP, et al. Epigenomic profiling of glucocorticoid responses identifies cis-regulatory disruptions impacting steroid resistance in childhood acute lymphoblastic leukemia. Leukemia. 2022;36:2374–2383. doi: 10.1038/s41375-022-01685-z.  [DOI](https://doi.org/10.1038/s41375-022-01685-z) | [PMC free article](/articles/PMC9522591/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36028659/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Leukemia&title=Epigenomic%20profiling%20of%20glucocorticoid%20responses%20identifies%20cis-regulatory%20disruptions%20impacting%20steroid%20resistance%20in%20childhood%20acute%20lymphoblastic%20leukemia&author=BP%20Bergeron&volume=36&publication_year=2022&pages=2374-2383&pmid=36028659&doi=10.1038/s41375-022-01685-z&)

23. Smith RP, Lam ET, Markova S, Yee SW, Ahituv N. Pharmacogene regulatory elements: from discovery to applications. Genome Med. 2012;4:45. doi: 10.1186/gm344.  [DOI](https://doi.org/10.1186/gm344) | [PMC free article](/articles/PMC3506911/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22630332/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Med.&title=Pharmacogene%20regulatory%20elements:%20from%20discovery%20to%20applications&author=RP%20Smith&author=ET%20Lam&author=S%20Markova&author=SW%20Yee&author=N%20Ahituv&volume=4&publication_year=2012&pages=45&pmid=22630332&doi=10.1186/gm344&)

24. Luizon MR, Ahituv N. Uncovering drug-responsive regulatory elements. Pharmacogenomics. 2015;16:1829–1841. doi: 10.2217/pgs.15.121.  [DOI](https://doi.org/10.2217/pgs.15.121) | [PMC free article](/articles/PMC4716675/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26555224/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Uncovering%20drug-responsive%20regulatory%20elements&author=MR%20Luizon&author=N%20Ahituv&volume=16&publication_year=2015&pages=1829-1841&pmid=26555224&doi=10.2217/pgs.15.121&)

25. Sakabe NJ, Savic D, Nobrega MA. Transcriptional enhancers in development and disease. Genome Biol. 2012;13:238. doi: 10.1186/gb-2012-13-1-238.  [DOI](https://doi.org/10.1186/gb-2012-13-1-238) | [PMC free article](/articles/PMC3334578/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22269347/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Biol.&title=Transcriptional%20enhancers%20in%20development%20and%20disease&author=NJ%20Sakabe&author=D%20Savic&author=MA%20Nobrega&volume=13&publication_year=2012&pages=238&pmid=22269347&doi=10.1186/gb-2012-13-1-238&)

26. White MA, Myers CA, Corbo JC, Cohen BA. Massively parallel in vivo enhancer assay reveals that highly local features determine the cis-regulatory function of ChIP-seq peaks. Proc. Natl Acad. Sci. USA. 2013;110:11952–11957. doi: 10.1073/pnas.1307449110.  [DOI](https://doi.org/10.1073/pnas.1307449110) | [PMC free article](/articles/PMC3718143/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23818646/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl%20Acad.%20Sci.%20USA&title=Massively%20parallel%20in%20vivo%20enhancer%20assay%20reveals%20that%20highly%20local%20features%20determine%20the%20cis-regulatory%20function%20of%20ChIP-seq%20peaks&author=MA%20White&author=CA%20Myers&author=JC%20Corbo&author=BA%20Cohen&volume=110&publication_year=2013&pages=11952-11957&pmid=23818646&doi=10.1073/pnas.1307449110&)

27. Kheradpour P, et al. Systematic dissection of regulatory motifs in 2000 predicted human enhancers using a massively parallel reporter assay. Genome Res. 2013;23:800–811. doi: 10.1101/gr.144899.112.  [DOI](https://doi.org/10.1101/gr.144899.112) | [PMC free article](/articles/PMC3638136/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23512712/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res.&title=Systematic%20dissection%20of%20regulatory%20motifs%20in%202000%20predicted%20human%20enhancers%20using%20a%20massively%20parallel%20reporter%20assay&author=P%20Kheradpour&volume=23&publication_year=2013&pages=800-811&pmid=23512712&doi=10.1101/gr.144899.112&)

28. Savic D, et al. Promoter-distal RNA polymerase II binding discriminates active from inactive CCAAT/ enhancer-binding protein beta binding sites. Genome Res. 2015;25:1791–1800. doi: 10.1101/gr.191593.115.  [DOI](https://doi.org/10.1101/gr.191593.115) | [PMC free article](/articles/PMC4665001/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26486725/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res.&title=Promoter-distal%20RNA%20polymerase%20II%20binding%20discriminates%20active%20from%20inactive%20CCAAT/%20enhancer-binding%20protein%20beta%20binding%20sites&author=D%20Savic&volume=25&publication_year=2015&pages=1791-1800&pmid=26486725&doi=10.1101/gr.191593.115&)

29. Ulirsch JC, et al. Systematic functional dissection of common genetic variation affecting red blood cell traits. Cell. 2016;165:1530–1545. doi: 10.1016/j.cell.2016.04.048.  [DOI](https://doi.org/10.1016/j.cell.2016.04.048) | [PMC free article](/articles/PMC4893171/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27259154/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell&title=Systematic%20functional%20dissection%20of%20common%20genetic%20variation%20affecting%20red%20blood%20cell%20traits&author=JC%20Ulirsch&volume=165&publication_year=2016&pages=1530-1545&pmid=27259154&doi=10.1016/j.cell.2016.04.048&)

30. Kircher M, et al. Saturation mutagenesis of twenty disease-associated regulatory elements at single base-pair resolution. Nat. Commun. 2019;10:3583. doi: 10.1038/s41467-019-11526-w.  [DOI](https://doi.org/10.1038/s41467-019-11526-w) | [PMC free article](/articles/PMC6687891/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31395865/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&title=Saturation%20mutagenesis%20of%20twenty%20disease-associated%20regulatory%20elements%20at%20single%20base-pair%20resolution&author=M%20Kircher&volume=10&publication_year=2019&pages=3583&pmid=31395865&doi=10.1038/s41467-019-11526-w&)

31. Choi J, et al. Massively parallel reporter assays of melanoma risk variants identify MX2 as a gene promoting melanoma. Nat. Commun. 2020;11:2718. doi: 10.1038/s41467-020-16590-1.  [DOI](https://doi.org/10.1038/s41467-020-16590-1) | [PMC free article](/articles/PMC7264232/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32483191/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&title=Massively%20parallel%20reporter%20assays%20of%20melanoma%20risk%20variants%20identify%20MX2%20as%20a%20gene%20promoting%20melanoma&author=J%20Choi&volume=11&publication_year=2020&pages=2718&pmid=32483191&doi=10.1038/s41467-020-16590-1&)

32. Joslin AC, et al. A functional genomics pipeline identifies pleiotropy and cross-tissue effects within obesity-associated GWAS loci. Nat. Commun. 2021;12:5253. doi: 10.1038/s41467-021-25614-3.  [DOI](https://doi.org/10.1038/s41467-021-25614-3) | [PMC free article](/articles/PMC8421397/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34489471/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&title=A%20functional%20genomics%20pipeline%20identifies%20pleiotropy%20and%20cross-tissue%20effects%20within%20obesity-associated%20GWAS%20loci&author=AC%20Joslin&volume=12&publication_year=2021&pages=5253&pmid=34489471&doi=10.1038/s41467-021-25614-3&)

33. Ajore R, et al. Functional dissection of inherited non-coding variation influencing multiple myeloma risk. Nat. Commun. 2022;13:151. doi: 10.1038/s41467-021-27666-x.  [DOI](https://doi.org/10.1038/s41467-021-27666-x) | [PMC free article](/articles/PMC8748989/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35013207/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&title=Functional%20dissection%20of%20inherited%20non-coding%20variation%20influencing%20multiple%20myeloma%20risk&author=R%20Ajore&volume=13&publication_year=2022&pages=151&pmid=35013207&doi=10.1038/s41467-021-27666-x&)

34. Long E, et al. Massively parallel reporter assays and variant scoring identified functional variants and target genes for melanoma loci and highlighted cell-type specificity. Am. J. Hum. Genet. 2022;109:2210–2229. doi: 10.1016/j.ajhg.2022.11.006.  [DOI](https://doi.org/10.1016/j.ajhg.2022.11.006) | [PMC free article](/articles/PMC9748337/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36423637/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Hum.%20Genet.&title=Massively%20parallel%20reporter%20assays%20and%20variant%20scoring%20identified%20functional%20variants%20and%20target%20genes%20for%20melanoma%20loci%20and%20highlighted%20cell-type%20specificity&author=E%20Long&volume=109&publication_year=2022&pages=2210-2229&pmid=36423637&doi=10.1016/j.ajhg.2022.11.006&)

35. Klein JC, et al. Functional testing of thousands of osteoarthritis-associated variants for regulatory activity. Nat. Commun. 2019;10:2434. doi: 10.1038/s41467-019-10439-y.  [DOI](https://doi.org/10.1038/s41467-019-10439-y) | [PMC free article](/articles/PMC6547687/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31164647/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&title=Functional%20testing%20of%20thousands%20of%20osteoarthritis-associated%20variants%20for%20regulatory%20activity&author=JC%20Klein&volume=10&publication_year=2019&pages=2434&pmid=31164647&doi=10.1038/s41467-019-10439-y&)

36. Smemo S, et al. Obesity-associated variants within FTO form long-range functional connections with IRX3. Nature. 2014;507:371–375. doi: 10.1038/nature13138.  [DOI](https://doi.org/10.1038/nature13138) | [PMC free article](/articles/PMC4113484/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24646999/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Obesity-associated%20variants%20within%20FTO%20form%20long-range%20functional%20connections%20with%20IRX3&author=S%20Smemo&volume=507&publication_year=2014&pages=371-375&pmid=24646999&doi=10.1038/nature13138&)

37. Consortium GT, et al. Genetic effects on gene expression across human tissues. Nature. 2017;550:204–213. doi: 10.1038/nature24277.  [DOI](https://doi.org/10.1038/nature24277) | [PMC free article](/articles/PMC5776756/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29022597/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Genetic%20effects%20on%20gene%20expression%20across%20human%20tissues&author=GT%20Consortium&volume=550&publication_year=2017&pages=204-213&pmid=29022597&doi=10.1038/nature24277&)

38. van Steensel B, Dekker J. Genomics tools for unraveling chromosome architecture. Nat. Biotechnol. 2010;28:1089–1095. doi: 10.1038/nbt.1680.  [DOI](https://doi.org/10.1038/nbt.1680) | [PMC free article](/articles/PMC3023824/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20944601/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Biotechnol.&title=Genomics%20tools%20for%20unraveling%20chromosome%20architecture&author=B%20van%20Steensel&author=J%20Dekker&volume=28&publication_year=2010&pages=1089-1095&pmid=20944601&doi=10.1038/nbt.1680&)

39. Mifsud B, et al. Mapping long-range promoter contacts in human cells with high-resolution capture Hi-C. Nat. Genet. 2015;47:598–606. doi: 10.1038/ng.3286.  [DOI](https://doi.org/10.1038/ng.3286) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25938943/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet.&title=Mapping%20long-range%20promoter%20contacts%20in%20human%20cells%20with%20high-resolution%20capture%20Hi-C&author=B%20Mifsud&volume=47&publication_year=2015&pages=598-606&pmid=25938943&doi=10.1038/ng.3286&)

40. Montefiori, L. E. et al. A promoter interaction map for cardiovascular disease genetics. Elife710.7554/eLife.35788 (2018).  [DOI](https://doi.org/10.7554/eLife.35788) | [PMC free article](/articles/PMC6053306/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29988018/)

41. Javierre BM, et al. Lineage-specific genome architecture links enhancers and non-coding disease variants to target gene promoters. Cell. 2016;167:1369–1384.e1319. doi: 10.1016/j.cell.2016.09.037.  [DOI](https://doi.org/10.1016/j.cell.2016.09.037) | [PMC free article](/articles/PMC5123897/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27863249/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell&title=Lineage-specific%20genome%20architecture%20links%20enhancers%20and%20non-coding%20disease%20variants%20to%20target%20gene%20promoters&author=BM%20Javierre&volume=167&publication_year=2016&pages=1369-1384.e1319&pmid=27863249&doi=10.1016/j.cell.2016.09.037&)

42. Thulson, E. et al. 3D chromatin structure in chondrocytes identifies putative osteoarthritis risk genes. Genetics22210.1093/genetics/iyac141 (2022).  [DOI](https://doi.org/10.1093/genetics/iyac141) | [PMC free article](/articles/PMC9713432/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36099032/)

43. Shi C, et al. Chromatin looping links target genes with genetic risk loci for dermatological traits. J. Invest. Dermatol. 2021;141:1975–1984. doi: 10.1016/j.jid.2021.01.015.  [DOI](https://doi.org/10.1016/j.jid.2021.01.015) | [PMC free article](/articles/PMC8315765/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33607115/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Invest.%20Dermatol.&title=Chromatin%20looping%20links%20target%20genes%20with%20genetic%20risk%20loci%20for%20dermatological%20traits&author=C%20Shi&volume=141&publication_year=2021&pages=1975-1984&pmid=33607115&doi=10.1016/j.jid.2021.01.015&)

44. Perez-Andreu V, et al. Inherited GATA3 variants are associated with Ph-like childhood acute lymphoblastic leukemia and risk of relapse. Nat. Genet. 2013;45:1494–1498. doi: 10.1038/ng.2803.  [DOI](https://doi.org/10.1038/ng.2803) | [PMC free article](/articles/PMC4039076/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24141364/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet.&title=Inherited%20GATA3%20variants%20are%20associated%20with%20Ph-like%20childhood%20acute%20lymphoblastic%20leukemia%20and%20risk%20of%20relapse&author=V%20Perez-Andreu&volume=45&publication_year=2013&pages=1494-1498&pmid=24141364&doi=10.1038/ng.2803&)

45. Perez-Andreu V, et al. A genome-wide association study of susceptibility to acute lymphoblastic leukemia in adolescents and young adults. Blood. 2015;125:680–686. doi: 10.1182/blood-2014-09-595744.  [DOI](https://doi.org/10.1182/blood-2014-09-595744) | [PMC free article](/articles/PMC4304112/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25468567/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=A%20genome-wide%20association%20study%20of%20susceptibility%20to%20acute%20lymphoblastic%20leukemia%20in%20adolescents%20and%20young%20adults&author=V%20Perez-Andreu&volume=125&publication_year=2015&pages=680-686&pmid=25468567&doi=10.1182/blood-2014-09-595744&)

46. Xu H, et al. Inherited coding variants at the CDKN2A locus influence susceptibility to acute lymphoblastic leukaemia in children. Nat. Commun. 2015;6:7553. doi: 10.1038/ncomms8553.  [DOI](https://doi.org/10.1038/ncomms8553) | [PMC free article](/articles/PMC4544058/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26104880/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&title=Inherited%20coding%20variants%20at%20the%20CDKN2A%20locus%20influence%20susceptibility%20to%20acute%20lymphoblastic%20leukaemia%20in%20children&author=H%20Xu&volume=6&publication_year=2015&pages=7553&pmid=26104880&doi=10.1038/ncomms8553&)

47. Hungate EA, et al. A variant at 9p21.3 functionally implicates CDKN2B in paediatric B-cell precursor acute lymphoblastic leukaemia aetiology. Nat. Commun. 2016;7:10635. doi: 10.1038/ncomms10635.  [DOI](https://doi.org/10.1038/ncomms10635) | [PMC free article](/articles/PMC4754340/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26868379/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&title=A%20variant%20at%209p21.3%20functionally%20implicates%20CDKN2B%20in%20paediatric%20B-cell%20precursor%20acute%20lymphoblastic%20leukaemia%20aetiology&author=EA%20Hungate&volume=7&publication_year=2016&pages=10635&pmid=26868379&doi=10.1038/ncomms10635&)

48. Vijayakrishnan J, et al. Genome-wide association study identifies susceptibility loci for B-cell childhood acute lymphoblastic leukemia. Nat. Commun. 2018;9:1340. doi: 10.1038/s41467-018-03178-z.  [DOI](https://doi.org/10.1038/s41467-018-03178-z) | [PMC free article](/articles/PMC5890276/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29632299/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&title=Genome-wide%20association%20study%20identifies%20susceptibility%20loci%20for%20B-cell%20childhood%20acute%20lymphoblastic%20leukemia&author=J%20Vijayakrishnan&volume=9&publication_year=2018&pages=1340&pmid=29632299&doi=10.1038/s41467-018-03178-z&)

49. Vijayakrishnan J, et al. Identification of four novel associations for B-cell acute lymphoblastic leukaemia risk. Nat. Commun. 2019;10:5348. doi: 10.1038/s41467-019-13069-6.  [DOI](https://doi.org/10.1038/s41467-019-13069-6) | [PMC free article](/articles/PMC6877561/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31767839/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&title=Identification%20of%20four%20novel%20associations%20for%20B-cell%20acute%20lymphoblastic%20leukaemia%20risk&author=J%20Vijayakrishnan&volume=10&publication_year=2019&pages=5348&pmid=31767839&doi=10.1038/s41467-019-13069-6&)

50. Vijayakrishnan J, et al. The 9p21.3 risk of childhood acute lymphoblastic leukaemia is explained by a rare high-impact variant in CDKN2A. Sci. Rep. 2015;5:15065. doi: 10.1038/srep15065.  [DOI](https://doi.org/10.1038/srep15065) | [PMC free article](/articles/PMC4604478/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26463672/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&title=The%209p21.3%20risk%20of%20childhood%20acute%20lymphoblastic%20leukaemia%20is%20explained%20by%20a%20rare%20high-impact%20variant%20in%20CDKN2A&author=J%20Vijayakrishnan&volume=5&publication_year=2015&pages=15065&pmid=26463672&doi=10.1038/srep15065&)

51. Studd JB, et al. Genetic and regulatory mechanism of susceptibility to high-hyperdiploid acute lymphoblastic leukaemia at 10p21.2. Nat. Commun. 2017;8:14616. doi: 10.1038/ncomms14616.  [DOI](https://doi.org/10.1038/ncomms14616) | [PMC free article](/articles/PMC5337971/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28256501/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&title=Genetic%20and%20regulatory%20mechanism%20of%20susceptibility%20to%20high-hyperdiploid%20acute%20lymphoblastic%20leukaemia%20at%2010p21.2&author=JB%20Studd&volume=8&publication_year=2017&pages=14616&pmid=28256501&doi=10.1038/ncomms14616&)

52. Yang H, et al. Noncoding genetic variation in GATA3 increases acute lymphoblastic leukemia risk through local and global changes in chromatin conformation. Nat. Genet. 2022;54:170–179. doi: 10.1038/s41588-021-00993-x.  [DOI](https://doi.org/10.1038/s41588-021-00993-x) | [PMC free article](/articles/PMC9794680/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35115686/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet.&title=Noncoding%20genetic%20variation%20in%20GATA3%20increases%20acute%20lymphoblastic%20leukemia%20risk%20through%20local%20and%20global%20changes%20in%20chromatin%20conformation&author=H%20Yang&volume=54&publication_year=2022&pages=170-179&pmid=35115686&doi=10.1038/s41588-021-00993-x&)

53. Zhao X, et al. Molecular mechanisms of ARID5B-mediated genetic susceptibility to acute lymphoblastic leukemia. J. Natl. Cancer Inst. 2022;114:1287–1295. doi: 10.1093/jnci/djac101.  [DOI](https://doi.org/10.1093/jnci/djac101) | [PMC free article](/articles/PMC9468286/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35575404/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Natl.%20Cancer%20Inst.&title=Molecular%20mechanisms%20of%20ARID5B-mediated%20genetic%20susceptibility%20to%20acute%20lymphoblastic%20leukemia&author=X%20Zhao&volume=114&publication_year=2022&pages=1287-1295&pmid=35575404&doi=10.1093/jnci/djac101&)

54. Buenrostro JD, Giresi PG, Zaba LC, Chang HY, Greenleaf WJ. Transposition of native chromatin for fast and sensitive epigenomic profiling of open chromatin, DNA-binding proteins and nucleosome position. Nat. Methods. 2013;10:1213–1218. doi: 10.1038/nmeth.2688.  [DOI](https://doi.org/10.1038/nmeth.2688) | [PMC free article](/articles/PMC3959825/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24097267/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Methods&title=Transposition%20of%20native%20chromatin%20for%20fast%20and%20sensitive%20epigenomic%20profiling%20of%20open%20chromatin,%20DNA-binding%20proteins%20and%20nucleosome%20position&author=JD%20Buenrostro&author=PG%20Giresi&author=LC%20Zaba&author=HY%20Chang&author=WJ%20Greenleaf&volume=10&publication_year=2013&pages=1213-1218&pmid=24097267&doi=10.1038/nmeth.2688&)

55. Diedrich JD, et al. Profiling chromatin accessibility in pediatric acute lymphoblastic leukemia identifies subtype-specific chromatin landscapes and gene regulatory networks. Leukemia. 2021;35:3078–3091. doi: 10.1038/s41375-021-01209-1.  [DOI](https://doi.org/10.1038/s41375-021-01209-1) | [PMC free article](/articles/PMC8435544/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33714976/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Leukemia&title=Profiling%20chromatin%20accessibility%20in%20pediatric%20acute%20lymphoblastic%20leukemia%20identifies%20subtype-specific%20chromatin%20landscapes%20and%20gene%20regulatory%20networks&author=JD%20Diedrich&volume=35&publication_year=2021&pages=3078-3091&pmid=33714976&doi=10.1038/s41375-021-01209-1&)

56. Barnett KR, et al. Epigenomic mapping reveals distinct B cell acute lymphoblastic leukemia chromatin architectures and regulators. Cell Genom. 2023;3:100442. doi: 10.1016/j.xgen.2023.100442.  [DOI](https://doi.org/10.1016/j.xgen.2023.100442) | [PMC free article](/articles/PMC10726428/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/38116118/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Genom.&title=Epigenomic%20mapping%20reveals%20distinct%20B%20cell%20acute%20lymphoblastic%20leukemia%20chromatin%20architectures%20and%20regulators&author=KR%20Barnett&volume=3&publication_year=2023&pages=100442&pmid=38116118&doi=10.1016/j.xgen.2023.100442&)

57. Klemm SL, Shipony Z, Greenleaf WJ. Chromatin accessibility and the regulatory epigenome. Nat. Rev. Genet. 2019;20:207–220. doi: 10.1038/s41576-018-0089-8.  [DOI](https://doi.org/10.1038/s41576-018-0089-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30675018/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Genet.&title=Chromatin%20accessibility%20and%20the%20regulatory%20epigenome&author=SL%20Klemm&author=Z%20Shipony&author=WJ%20Greenleaf&volume=20&publication_year=2019&pages=207-220&pmid=30675018&doi=10.1038/s41576-018-0089-8&)

58. McLean CY, et al. GREAT improves functional interpretation of cis-regulatory regions. Nat. Biotechnol. 2010;28:495–501. doi: 10.1038/nbt.1630.  [DOI](https://doi.org/10.1038/nbt.1630) | [PMC free article](/articles/PMC4840234/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20436461/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Biotechnol.&title=GREAT%20improves%20functional%20interpretation%20of%20cis-regulatory%20regions&author=CY%20McLean&volume=28&publication_year=2010&pages=495-501&pmid=20436461&doi=10.1038/nbt.1630&)

59. Mumbach MR, et al. HiChIP: efficient and sensitive analysis of protein-directed genome architecture. Nat. Methods. 2016;13:919–922. doi: 10.1038/nmeth.3999.  [DOI](https://doi.org/10.1038/nmeth.3999) | [PMC free article](/articles/PMC5501173/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27643841/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Methods&title=HiChIP:%20efficient%20and%20sensitive%20analysis%20of%20protein-directed%20genome%20architecture&author=MR%20Mumbach&volume=13&publication_year=2016&pages=919-922&pmid=27643841&doi=10.1038/nmeth.3999&)

60. Dryden NH, et al. Unbiased analysis of potential targets of breast cancer susceptibility loci by Capture Hi-C. Genome Res. 2014;24:1854–1868. doi: 10.1101/gr.175034.114.  [DOI](https://doi.org/10.1101/gr.175034.114) | [PMC free article](/articles/PMC4216926/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25122612/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res.&title=Unbiased%20analysis%20of%20potential%20targets%20of%20breast%20cancer%20susceptibility%20loci%20by%20Capture%20Hi-C&author=NH%20Dryden&volume=24&publication_year=2014&pages=1854-1868&pmid=25122612&doi=10.1101/gr.175034.114&)

61. Jager R, et al. Capture Hi-C identifies the chromatin interactome of colorectal cancer risk loci. Nat. Commun. 2015;6:6178. doi: 10.1038/ncomms7178.  [DOI](https://doi.org/10.1038/ncomms7178) | [PMC free article](/articles/PMC4346635/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25695508/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&title=Capture%20Hi-C%20identifies%20the%20chromatin%20interactome%20of%20colorectal%20cancer%20risk%20loci&author=R%20Jager&volume=6&publication_year=2015&pages=6178&pmid=25695508&doi=10.1038/ncomms7178&)

62. Martin P, et al. Capture Hi-C reveals novel candidate genes and complex long-range interactions with related autoimmune risk loci. Nat. Commun. 2015;6:10069. doi: 10.1038/ncomms10069.  [DOI](https://doi.org/10.1038/ncomms10069) | [PMC free article](/articles/PMC4674669/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26616563/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&title=Capture%20Hi-C%20reveals%20novel%20candidate%20genes%20and%20complex%20long-range%20interactions%20with%20related%20autoimmune%20risk%20loci&author=P%20Martin&volume=6&publication_year=2015&pages=10069&pmid=26616563&doi=10.1038/ncomms10069&)

63. McGovern A, et al. Capture Hi-C identifies a novel causal gene, IL20RA, in the pan-autoimmune genetic susceptibility region 6q23. Genome Biol. 2016;17:212. doi: 10.1186/s13059-016-1078-x.  [DOI](https://doi.org/10.1186/s13059-016-1078-x) | [PMC free article](/articles/PMC5088679/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27799070/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Biol.&title=Capture%20Hi-C%20identifies%20a%20novel%20causal%20gene,%20IL20RA,%20in%20the%20pan-autoimmune%20genetic%20susceptibility%20region%206q23&author=A%20McGovern&volume=17&publication_year=2016&pages=212&pmid=27799070&doi=10.1186/s13059-016-1078-x&)

64. Skene, P. J. & Henikoff, S. An efficient targeted nuclease strategy for high-resolution mapping of DNA binding sites. Elife610.7554/eLife.21856 (2017).  [DOI](https://doi.org/10.7554/eLife.21856) | [PMC free article](/articles/PMC5310842/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28079019/)

65. Panda A, et al. Clonally selected lines after CRISPR-Cas editing are not isogenic. CRISPR J. 2023;6:176–182. doi: 10.1089/crispr.2022.0050.  [DOI](https://doi.org/10.1089/crispr.2022.0050) | [PMC free article](/articles/PMC10123805/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/37071670/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=CRISPR%20J.&title=Clonally%20selected%20lines%20after%20CRISPR-Cas%20editing%20are%20not%20isogenic&author=A%20Panda&volume=6&publication_year=2023&pages=176-182&pmid=37071670&doi=10.1089/crispr.2022.0050&)

66. Hosokawa H, et al. Transcription factor PU.1 represses and activates gene expression in early T cells by redirecting partner transcription factor binding. Immunity. 2018;49:782. doi: 10.1016/j.immuni.2018.09.019.  [DOI](https://doi.org/10.1016/j.immuni.2018.09.019) | [PMC free article](/articles/PMC6207499/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30332634/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Immunity&title=Transcription%20factor%20PU.1%20represses%20and%20activates%20gene%20expression%20in%20early%20T%20cells%20by%20redirecting%20partner%20transcription%20factor%20binding&author=H%20Hosokawa&volume=49&publication_year=2018&pages=782&pmid=30332634&doi=10.1016/j.immuni.2018.09.019&)

67. Carey HA, et al. Enhancer variants reveal a conserved transcription factor network governed by PU.1 during osteoclast differentiation. Bone Res. 2018;6:8. doi: 10.1038/s41413-018-0011-1.  [DOI](https://doi.org/10.1038/s41413-018-0011-1) | [PMC free article](/articles/PMC5874256/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29619268/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bone%20Res.&title=Enhancer%20variants%20reveal%20a%20conserved%20transcription%20factor%20network%20governed%20by%20PU.1%20during%20osteoclast%20differentiation&author=HA%20Carey&volume=6&publication_year=2018&pages=8&pmid=29619268&doi=10.1038/s41413-018-0011-1&)

68. Renbarger JL, McCammack KC, Rouse CE, Hall SD. Effect of race on vincristine-associated neurotoxicity in pediatric acute lymphoblastic leukemia patients. Pediatr. Blood Cancer. 2008;50:769–771. doi: 10.1002/pbc.21435.  [DOI](https://doi.org/10.1002/pbc.21435) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18085684/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pediatr.%20Blood%20Cancer&title=Effect%20of%20race%20on%20vincristine-associated%20neurotoxicity%20in%20pediatric%20acute%20lymphoblastic%20leukemia%20patients&author=JL%20Renbarger&author=KC%20McCammack&author=CE%20Rouse&author=SD%20Hall&volume=50&publication_year=2008&pages=769-771&pmid=18085684&doi=10.1002/pbc.21435&)

69. Yang JJ, et al. Ancestry and pharmacogenomics of relapse in acute lymphoblastic leukemia. Nat. Genet. 2011;43:237–241. doi: 10.1038/ng.763.  [DOI](https://doi.org/10.1038/ng.763) | [PMC free article](/articles/PMC3104508/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21297632/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet&title=Ancestry%20and%20pharmacogenomics%20of%20relapse%20in%20acute%20lymphoblastic%20leukemia&author=JJ%20Yang&volume=43&publication_year=2011&pages=237-241&pmid=21297632&doi=10.1038/ng.763&)

70. Kadan-Lottick NS, Ness KK, Bhatia S, Gurney JG. Survival variability by race and ethnicity in childhood acute lymphoblastic leukemia. JAMA. 2003;290:2008–2014. doi: 10.1001/jama.290.15.2008.  [DOI](https://doi.org/10.1001/jama.290.15.2008) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14559954/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA&title=Survival%20variability%20by%20race%20and%20ethnicity%20in%20childhood%20acute%20lymphoblastic%20leukemia&author=NS%20Kadan-Lottick&author=KK%20Ness&author=S%20Bhatia&author=JG%20Gurney&volume=290&publication_year=2003&pages=2008-2014&pmid=14559954&doi=10.1001/jama.290.15.2008&)

71. Bhatia S, et al. Racial and ethnic differences in survival of children with acute lymphoblastic leukemia. Blood. 2002;100:1957–1964. doi: 10.1182/blood-2002-02-0395.  [DOI](https://doi.org/10.1182/blood-2002-02-0395) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12200352/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Racial%20and%20ethnic%20differences%20in%20survival%20of%20children%20with%20acute%20lymphoblastic%20leukemia&author=S%20Bhatia&volume=100&publication_year=2002&pages=1957-1964&pmid=12200352&doi=10.1182/blood-2002-02-0395&)

72. Pollock BH, et al. Racial differences in the survival of childhood B-precursor acute lymphoblastic leukemia: a Pediatric Oncology Group Study. J. Clin. Oncol. 2000;18:813–823. doi: 10.1200/JCO.2000.18.4.813.  [DOI](https://doi.org/10.1200/JCO.2000.18.4.813) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10673523/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Oncol.&title=Racial%20differences%20in%20the%20survival%20of%20childhood%20B-precursor%20acute%20lymphoblastic%20leukemia:%20a%20Pediatric%20Oncology%20Group%20Study&author=BH%20Pollock&volume=18&publication_year=2000&pages=813-823&pmid=10673523&doi=10.1200/JCO.2000.18.4.813&)

73. Pui CH, et al. Results of therapy for acute lymphoblastic leukemia in black and white children. JAMA. 2003;290:2001–2007. doi: 10.1001/jama.290.15.2001.  [DOI](https://doi.org/10.1001/jama.290.15.2001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14559953/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA&title=Results%20of%20therapy%20for%20acute%20lymphoblastic%20leukemia%20in%20black%20and%20white%20children&author=CH%20Pui&volume=290&publication_year=2003&pages=2001-2007&pmid=14559953&doi=10.1001/jama.290.15.2001&)

74. Dong Z, et al. Role of eIF3a in regulating cell cycle progression. Exp. Cell Res. 2009;315:1889–1894. doi: 10.1016/j.yexcr.2009.03.009.  [DOI](https://doi.org/10.1016/j.yexcr.2009.03.009) | [PMC free article](/articles/PMC11645673/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19327350/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Exp.%20Cell%20Res.&title=Role%20of%20eIF3a%20in%20regulating%20cell%20cycle%20progression&author=Z%20Dong&volume=315&publication_year=2009&pages=1889-1894&pmid=19327350&doi=10.1016/j.yexcr.2009.03.009&)

75. Kothari A, Hittelman WN, Chambers TC. Cell cycle–dependent mechanisms underlie vincristine-induced death of primary acute lymphoblastic leukemia cells. Cancer Res. 2016;76:3553–3561. doi: 10.1158/0008-5472.CAN-15-2104.  [DOI](https://doi.org/10.1158/0008-5472.CAN-15-2104) | [PMC free article](/articles/PMC4911277/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27197148/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res.&title=Cell%20cycle%E2%80%93dependent%20mechanisms%20underlie%20vincristine-induced%20death%20of%20primary%20acute%20lymphoblastic%20leukemia%20cells&author=A%20Kothari&author=WN%20Hittelman&author=TC%20Chambers&volume=76&publication_year=2016&pages=3553-3561&pmid=27197148&doi=10.1158/0008-5472.CAN-15-2104&)

76. Jiang, S.-L. et al. The downregulation of eIF3a contributes to vemurafenib resistance in melanoma by activating ERK via PPP2R1B. Front. Pharmacol.1210.3389/fphar.2021.720619 (2021).  [DOI](https://doi.org/10.3389/fphar.2021.720619) | [PMC free article](/articles/PMC8430041/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34512348/)

77. Yin J-Y, et al. Effect of eIF3a on response of lung cancer patients to platinum-based chemotherapy by regulating DNA Repair. Clin. Cancer Res. 2011;17:4600–4609. doi: 10.1158/1078-0432.CCR-10-2591.  [DOI](https://doi.org/10.1158/1078-0432.CCR-10-2591) | [PMC free article](/articles/PMC11531068/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21610145/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Cancer%20Res.&title=Effect%20of%20eIF3a%20on%20response%20of%20lung%20cancer%20patients%20to%20platinum-based%20chemotherapy%20by%20regulating%20DNA%20Repair&author=J-Y%20Yin&volume=17&publication_year=2011&pages=4600-4609&pmid=21610145&doi=10.1158/1078-0432.CCR-10-2591&)

78. Torii S, Yamamoto T, Tsuchiya Y, Nishida E. ERK MAP kinase in G1 cell cycle progression and cancer. Cancer Sci. 2006;97:697–702. doi: 10.1111/j.1349-7006.2006.00244.x.  [DOI](https://doi.org/10.1111/j.1349-7006.2006.00244.x) | [PMC free article](/articles/PMC11158792/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16800820/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Sci.&title=ERK%20MAP%20kinase%20in%20G1%20cell%20cycle%20progression%20and%20cancer&author=S%20Torii&author=T%20Yamamoto&author=Y%20Tsuchiya&author=E%20Nishida&volume=97&publication_year=2006&pages=697-702&pmid=16800820&doi=10.1111/j.1349-7006.2006.00244.x&)

79. French D, et al. Acquired variation outweighs inherited variation in whole genome analysis of methotrexate polyglutamate accumulation in leukemia. Blood. 2009;113:4512–4520. doi: 10.1182/blood-2008-07-172106.  [DOI](https://doi.org/10.1182/blood-2008-07-172106) | [PMC free article](/articles/PMC2680361/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19066393/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Acquired%20variation%20outweighs%20inherited%20variation%20in%20whole%20genome%20analysis%20of%20methotrexate%20polyglutamate%20accumulation%20in%20leukemia&author=D%20French&volume=113&publication_year=2009&pages=4512-4520&pmid=19066393&doi=10.1182/blood-2008-07-172106&)

80. Makowski MM, et al. An interaction proteomics survey of transcription factor binding at recurrent TERT promoter mutations. Proteomics. 2016;16:417–426. doi: 10.1002/pmic.201500327.  [DOI](https://doi.org/10.1002/pmic.201500327) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26553150/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proteomics&title=An%20interaction%20proteomics%20survey%20of%20transcription%20factor%20binding%20at%20recurrent%20TERT%20promoter%20mutations&author=MM%20Makowski&volume=16&publication_year=2016&pages=417-426&pmid=26553150&doi=10.1002/pmic.201500327&)
