# FOXO3 polymorphisms influence the risk and prognosis of rhabdomyosarcoma in children

## Metadata
**Authors:** Xiaohong Zhang, Yaping Sun, Huilin Niu, Ping Tan, Shanshan Liu, Xiaoping Liu, Xiaodan Liu, Ailing Luo, Mansi Cai, Yaping Yan, Ling Xu, Xu Yang
**Journal:** Frontiers in Oncology
**Date:** 2024 Apr 24
**DOI:** [10.3389/fonc.2024.1387735](https://doi.org/10.3389/fonc.2024.1387735)
**PMID:** 38720807
**PMCID:** PMC11076676
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11076676/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC11076676/pdf/fonc-14-1387735.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC11076676/pdf/fonc-14-1387735.pdf)

## Abstract

**Background:** 
Rhabdomyosarcoma(RMS) is the most common soft tissue sarcoma in children and single nucleotide polymorphisms(SNPs) in certain genes influence risk of RMS. Although FOXO3 had been reported in multiple cancers including RMS, the role of FOXO3 polymorphisms in RMS remains unclear. In this case-control study, we evaluated the association of FOXO3 SNPs with RMS risk and prognosis in children.

**Methods:** 
Four FOXO3 SNPs(rs17069665 A>G, rs4946936 T>C, rs4945816 C>T and rs9400241 C>A) were genotyped in 110 RMS cases and 359 controls. The associations between FOXO3 polymorphisms and RMS risk were determined by odds ratios(ORs) with 95% confidence intervals(CIs). The associations of rs17069665 and rs4946936 with overall survival in RMS children were estimated using the Kaplan-Meier method and log-rank test. Functional analysis in silico was performed to estimate the probability that rs17069665 and rs4946936 might influence the regulation of FOXO3.

**Results:** 
We found that rs17069665 (GG vs. AA+AG, adjusted OR=2.96; 95%CI [1.10-3.32]; P=0.010) and rs4946936 (TC+CC vs. TT, adjusted OR=0.48; 95%CI [0.25-0.90]; P=0.023) were related to the increased and decreased RMS risk, respectively. Besides, rs17069665(P<0.001) and rs4946936(P<0.001) were associated with decreased and increased overall survival in RMS patients, respectively. Functional analysis showed that rs17069665 and rs4946936 might influence the transcription and expression of FOXO3 via altering the bindings to MYC, CTCF, and/or RELA.

**Conclusions:** 
This study revealed that FOXO3 polymorphisms influence the RMS susceptibility and prognosis in children, and might altered the expression of FOXO3. FOXO3 polymorphism was suggested as a biomarker for RMS susceptibility and prognosis.

Keywords: FOXO3, single nucleotide polymorphisms (SNPs), rhabdomyosarcoma (RMS), susceptibility, prognosis

### Background

Rhabdomyosarcoma(RMS) is the most common soft tissue sarcoma in children and single nucleotide polymorphisms(SNPs) in certain genes influence risk of RMS. Although *FOXO3* had been reported in multiple cancers including RMS, the role of *FOXO3* polymorphisms in RMS remains unclear. In this case-control study, we evaluated the association of *FOXO3* SNPs with RMS risk and prognosis in children.

### Methods

Four *FOXO3* SNPs(rs17069665 A>G, rs4946936 T>C, rs4945816 C>T and rs9400241 C>A) were genotyped in 110 RMS cases and 359 controls. The associations between *FOXO3* polymorphisms and RMS risk were determined by odds ratios(ORs) with 95% confidence intervals(CIs). The associations of rs17069665 and rs4946936 with overall survival in RMS children were estimated using the Kaplan-Meier method and log-rank test. Functional analysis in silico was performed to estimate the probability that rs17069665 and rs4946936 might influence the regulation of *FOXO3*.

### Results

We found that rs17069665 (GG vs. AA+AG, adjusted OR=2.96; 95%CI [1.10-3.32]; *P*=0.010) and rs4946936 (TC+CC vs. TT, adjusted OR=0.48; 95%CI [0.25-0.90]; *P*=0.023) were related to the increased and decreased RMS risk, respectively. Besides, rs17069665(*P*<0.001) and rs4946936(*P*<0.001) were associated with decreased and increased overall survival in RMS patients, respectively. Functional analysis showed that rs17069665 and rs4946936 might influence the transcription and expression of *FOXO3* via altering the bindings to MYC, CTCF, and/or RELA.

### Conclusions

This study revealed that *FOXO3* polymorphisms influence the RMS susceptibility and prognosis in children, and might altered the expression of *FOXO3*. *FOXO3* polymorphism was suggested as a biomarker for RMS susceptibility and prognosis.

## Introduction

Rhabdomyosarcoma (RMS) is a myogenic tumor that arises from cells committed to skeletal muscle differentiation ([1](#B1), [2](#B2)). Although relatively rare, RMS is the most common soft tissue sarcoma and the third most prevalent extracranial solid tumor in children ([3](#B3), [4](#B4)). Despite of developments in the therapy and management of RMS patients in the past decades, the etiology of RMS remains unclear and the prognosis of RMS is also still poor ([3](#B3), [5](#B5)). Cancer occurs due to environmental exposure and/or genetic factors. In RMS initiation and development, genetic change has been suggested to play a vital role ([3](#B3)). One prevalent type of genetic variation in human disease is single nucleotide polymorphisms (SNPs). SNPs of certain genes have been validated to influence the incidence and outcome of multiple cancers including RMS ([6](#B6), [7](#B7)). However, many SNPs in genes associated with RMS are still not discovered and need to be explored.

Forkhead Box O3(*FOXO3*), also called Forkhead in Rhabdomyosarcoma-Like 1 (FKHRL1), belongs to the forkhead box O (FoxO) family of transcription factors which regulate the transcription of genes involved in multiple cellular processes (proliferation, apoptosis, autophagy, *etc.*) *(* [8](#B8)–[10](#B10)). Evidence supported that the aberrant expression or dysfunction of FOXO3 was associated with various types of cancers including RMS ([11](#B11)–[14](#B14)). In addition, gene polymorphism of *FOXO3* has been validated to influence various human disorders including cancer, such as pancreatic cancer, colorectal cancer, active tuberculosis, polycystic ovary syndrome, and acute lymphoblastic leukemia (ALL) ([15](#B15)–[20](#B20)). However, the association of FOXO3 polymorphisms with childhood RMS risk and outcome has never been reported.

In this case-control study, we conducted genotyping assay for FOXO3 SNPs (rs17069665, rs4946936, rs4945816, and rs9400241) and estimated the association of FOXO3 polymorphisms with RMS risk and prognosis in children. Besides, the underlying mechanisms involved in FOXO3 regulation by these polymorphisms were explored via functional analysis in silico.

## Materials and methods

### Study subjects

In this current study, a total of 110 RMS cases and 359 non-cancer healthy controls were recruited from Guangzhou Women and Children’s Medical Center (GWCMC). Briefly, patients under 18 years old with RMS were identified with histological confirmation and enrolled in this study by pediatric clinicians. Each case was newly diagnosed and provided a detailed medical record. Patients with other malignancies, secondary disorders, or chemotherapy/radiotherapy records were excluded. Thus, 110 children with RMS were recruited during June 2012 to June 2019, and 359 age- and sex-matched healthy volunteers were collected as controls. The controls were randomly selected from children receiving routine physical examinations. All individuals included in this study were ethnic Han Chinese. Before participation, a written informed consent for each case was obtained in advance. This study was conducted under the guideline of the Institutional Review Board of GWCMC.

### SNP Selection and genotyping

In this study, four SNPs were selected using data from SNPinfo and NCBI dbSNP databases as described previously ([21](#B21)–[23](#B23)). Briefly, the selection was based on three criteria as the following: (1) SNPs with the minor allele frequency (MAF) >5% in Chinese population; (2) located in or near the FOXO3 gene (i.e., 3’-UTR or 5’-UTR); (3) affecting splicing activity, transcription factor binding sites (TFBS) or miRNA-binding sites activity. At last, rs17069665 A>G, rs4946936 T>C, rs4945816 C>T, and rs9400241 C>A were retrieved for further analyses.

For genotyping, DNA was extracted from peripheral blood sample of each participant using the TIANamp DNAKit (TianGen, Beijing, China). In the genotyping assays, the corresponding SNP probes and primers (ABI, Massachusetts, USA) were used to genotype these four SNPs as previously described ([21](#B21)). For quality control, 10% samples were selected randomly for direct sequencing ([24](#B24), [25](#B25)), and a concordance rate of 100% was obtained.

### Functional analysis in silico

The associations of rs17069665 A>G and rs4946936 T>C with FOXO3 expression were determined using the expression quantitative trait loci (eQTL) analysis in GTEx portal ([https://www.gtexportal.org/home/](https://www.gtexportal.org/home/)) ([26](#B26)). The probability that these two polymorphisms might influence the regulation of FOXO3 was estimated by using the Roadmap Epigenome Browser ([27](#B27), [28](#B28)), the ENCODE Project ([29](#B29), [30](#B30)), and TFBIND software ([31](#B31)). In brief, promoter and enhancer were predicted via DNase hypersensitivity (DHS) and histone modification of muscle cells in Roadmap Epigenomics data. TFBIND was used to evaluate whether FOXO3 polymorphism rs17069665 A>G or rs4946936 T>C altered any TFBS, and then ENCODE ChIP-seq experiments of MYC, CTCF, and RELA (Experiment Series: ENCSR000DLR, ENCSR000ANS, ENCSR000EBD) were used to assess the binding signals and motifs overlapping rs17069665 or rs4946936.

### Statistical analyses

The demographic variables and genotype distribution of each SNP between the RMS case and control groups were compared using a 2-sided χ 2 test. For each SNP in the control group, Hardy-Weinberg equilibrium (HWE) was assessed using goodness-of-fit chi-square test. The strength of the association between FOXO3 polymorphism and RMS susceptibility was estimated using logistic regression analysis with odds ratios (ORs) and 95% confidence intervals (CIs), adjusting for age and sex. False-positive report probability (FPRP) was conducted for significant findings, as described previously ([32](#B32)). We adopted OR of 2.00 and 0.50 for risk and protective effects, respectively. FPRP values < 0.2 were considered noteworthy. Overall survival was estimated with Kaplan-Meier method, and then differences between subgroups were compared using the log-rank test. All statistical analyses were performed with SAS software and GraphPad Prism software; a two-sided *P* < 0.05 was considered to be statistically significant.

## Results

### Characteristics of subject

Totally, 110 RMS cases and 359 healthy controls were finally enrolled in this study. The detailed demographic characteristics are all summarized in [Table 1](#T1). Briefly, no significant difference in the age (*P* = 0.947) or sex (*P* = 0.613) distribution was found between cases and controls. According to histological classification, 74 (67.27%) cases were diagnosed as embryonal RMS (ERMS), 27 (24.55%) cases as alveolar RMS (ARMS), and 9 (8.18%) cases as other subtypes including anaplastic and mixed RMS. In addition, other information including risk level, clinical stage, and site of origin were also included in [Table 1](#T1).

### Table 1.

| Variables | RMS Cases (n=110) |  |  | Controls (n=359) |  | P a |
| --- | --- | --- | --- | --- | --- | --- |
| No. | % |  | No. | % |   |  |
| Age range, years | 0.10-13.9 |  |  | 0.50-13.0 |  | 0.947 |
| Mean ± SD | 3.21 ± 3.10 |  |  | 4.85 ± 2.38 |  |   |
| <10 | 104 | 94.55 |  | 340 | 94.71 |   |
| ≥10 | 6 | 5.45 |  | 19 | 5.29 |   |
| Sex |   |   |  |   |   | 0.613 |
| Female | 44 | 40.00 |  | 134 | 37.33 |   |
| Male | 66 | 60.00 |  | 225 | 62.67 |   |
| Histological subtype |  |  |  |  |  |  |
| Embryonal | 74 | 67.27 |  |   |   |   |
| Alveolar | 27 | 24.55 |  |   |   |   |
| Others | 9 | 8.18 |  |   |   |   |
| Risk level |  |  |  |  |  |  |
| Low | 14 | 12.73 |  |   |   |   |
| Medium | 44 | 40.00 |  |   |   |   |
| High | 52 | 47.27 |  |   |   |   |
| Site |  |  |  |  |  |  |
| Head and neck | 11 | 10.00 |  |   |   |   |
| Trunk and limbs | 33 | 30.00 |  |   |   |   |
| Genitourinary system | 19 | 17.27 |  |   |   |   |
| Thoracic, abdominal and pelvic cavities | 42 | 38.18 |  |   |   |   |
| Others | 5 | 4.55 |  |   |   |   |
| Clinical stage |  |  |  |  |  |  |
| I | 7 | 6.36 |  |   |   |   |
| II | 14 | 12.73 |  |   |   |   |
| III | 63 | 57.27 |  |   |   |   |
| IV | 21 | 19.09 |  |   |   |   |
| NA | 5 | 4.55 |  |   |   |   |

Table 1 Caption: Frequency distribution of selected characteristics in RMS cases and cancer-free controls.

### Associations between FOXO3 gene polymorphisms and RMS susceptibility

According to the SNP selection strategy, four SNPs (rs17069665 A>G, rs4946936 T>C, rs4945816 C>T, and rs9400241 C>A) that overlap with TFBS or miRNA-binding sites were selected ([Supplementary Table S1](#SM1)). The genotype frequencies of FOXO3 SNPs in all 110 cases and 359 controls and their association with RMS risk were displayed in [Table 2](#T2). All these four SNPs were in HWE (*P* _HWE_ > 0.05) among the control. Significant differences between RMS cases and controls were observed for rs17069665 A>G (*P* = 0.011) and rs4946936 T>C polymorphism (*P* = 0.025) in a recessive (GG vs. AA+AG) and a dominant (TC + CC vs. TT) model, respectively. After adjustment with age and sex, all these 2 polymorphisms, namely rs17069665 G allele (GG vs. AA+AG: adjusted OR = 2.96; 95% CI [1.10-3.32]; *P* = 0.010) and rs4946936 C allele (TC+CC vs. TT: adjusted OR=0.48; 95%CI [0.25-0.90]; *P*=0.023), were significantly related to the increased and decreased RMS risk, respectively. The remaining 2 polymorphisms (rs4945816 C>T and rs9400241 C>A), however, were not found to be in association with RMS risk.

### Table 2.

| Genotype | Cases(N=110) | Controls(N=359) | P a | Crude OR(95% CI) | P | Adjusted OR(95% CI) b | P b |
| --- | --- | --- | --- | --- | --- | --- | --- |
| rs17069665 (HWE=0.7104, A/G: wt/mut) |  |  |  |  |  |  |  |
| AA | 71 (64.55) | 241 (67.13) |   | 1.00 |   | 1.00 |   |
| AG | 28 (25.45) | 105 (29.25) |   | 0.91 (0.55-1.48) | 0.693 | 0.90 (0.55-1.47) | 0.668 |
| GG | 11 (10.00) | 13 (3.62) |   | 2.87 (1.23-6.69) | 0.014 | 2.89 (1.24-6.73) | 0.014 |
| Additive(GG vs. AG vs. AA) |  |   | 0.158 | 1.29 (0.91-1.83) | 0.159 | 1.31 (0.92-1.87) | 0.165 |
| Dominant(AG+GG vs. AA) | 39 (35.45) | 118 (32.87) | 0.615 | 1.12 (0.72-1.76) | 0.615 | 1.12 (0.71-1.75) | 0.633 |
| Recessive(GG vs. AA+AG) | 99 (90.00) | 346 (96.38) | 0.008 | 2.96 (1.29-6.80) | 0.011 | 2.98 (1.29-6.86) | 0.010 |
| rs4946936 (HWE=0.3811, T/C: wt/mut) |  |  |  |  |  |  |  |
| TT | 17 (15.45) | 29 (8.08) |   | 1.00 |   | 1.00 |   |
| TC | 37 (33.64) | 158 (44.01) |   | 0.40 (0.20-0.80) | 0.010 | 0.39 (0.19-0.78) | 0.010 |
| CC | 56 (50.91) | 172(47.91) |   | 0.55 (0.28-1.09) | 0.086 | 0.55 (0.28-1.08) | 0.083 |
| Additive (CC vs. TC vs. TT) |   |   | 0.542 | 0.91 (0.66-1.25) | 0.542 | 0.91 (0.66-1.25) | 0.560 |
| Dominant (TC + CC vs. TT) | 93 (84.55) | 330 (91.92) | 0.023 | 0.48 (0.25-0.91) | 0.025 | 0.48 (0.25-0.90) | 0.023 |
| Recessive (CC vs. TT + TC) | 54 (49.09) | 187 (52.09) | 0.582 | 1.13 (0.74-1.73) | 0.582 | 1.14 (0.74-1.76) | 0.542 |
| rs4945816 (HWE=0.4398, C/T: wt/mut) |  |  |  |  |  |  |  |
| CC | 15 (13.64) | 30 (8.36) |   | 1.00 |   | 1.00 |   |
| CT | 38 (34.55) | 158 (44.01) |   | 0.49 (0.24-1.15) | 0.145 | 0.49 (0.24-1.13) | 0.139 |
| TT | 57 (51.82) | 171(47.63) |   | 0.67 (0.34-1.33) | 0.248 | 0.67 (0.34-1.33) | 0.248 |
| Additive (TT vs. CT vs. CC) |   |   | 0.878 | 0.98 (0.70-1.35) | 0.878 | 0.98 (0.71-1.36) | 0.902 |
| Dominant (CT + TT vs. CC) | 95 (86.36) | 329 (91.64) | 0.442 | 1.18 (0.77-1.81) | 0.442 | 1.19 (0.78-1.84) | 0.410 |
| Recessive (TT vs. CC + CT) | 53 (48.18) | 188 (52.37) | 0.100 | 0.58 (0.30-1.12) | 0.103 | 0.57 (0.30-1.11) | 0.099 |
| rs9400241 (HWE=0.7986, C/A: wt/mut) |  |  |  |  |  |  |  |
| CC | 16 (14.55) | 30 (8.36) |   | 1.00 |   | 1.00 |   |
| CA | 38 (34.54) | 151 (42.06) |   | 0.47 (0.26-1.27) | 0.347 | 0.46 (0.23-1.27) | 0.324 |
| AA | 56 (50.91) | 178 (49.58) |   | 0.59 (0.30-1.16) | 0.127 | 1.70 (0.86-3.36) | 0.124 |
| Additive (AA vs. CA vs. CC) |  |   | 0.500 | 0.89 (0.65-1.23) | 0.500 | 0.89 (0.65-1.24) | 0.513 |
| Dominant (CA +AA vs. CC) | 94 (85.45) | 329 (91.64) | 0.056 | 0.54 (0.28-1.03) | 0.059 | 0.53 (0.28-1.02) | 0.056 |
| Recessive (AA vs. CC+ CA) | 54 (49.09) | 181 (50.42) | 0.808 | 1.06 (0.69-1.61) | 0.808 | 1.07 (0.69-1.63) | 0.774 |

Table 2 Caption: Logistic regression analysis of associations between FOXO3 polymorphisms and RMS susceptibility.

### Subgroup and stratification analyses

To further explore the association of the FOXO3 gene rs17069665 A>G and rs4946936 T>C polymorphisms with RMS susceptibility, we performed subgroup and stratification analyses in terms of age, sex, histological classification, risk level, clinical stage, and site of origin. As shown in [Table 3](#T3), rs17069665 (GG vs. AA+AG) and rs4946936 (TC+CC vs. TT) were both associated with RMS in male (adjusted OR=3.33, 95% CI [1.16-9. 57], *P* =0.026; adjusted OR=0.36; 95% CI [0.17-0.78], *P*=0.009; respectively), in children aged < 10 years (adjusted OR=2.98, 95% CI [1.29-6.87], *P* =0.011; adjusted OR=0.45; 95% CI [0.24-0.86], *P*=0.016; respectively), in children with tumor of stage II (adjusted OR=9.95, 95% CI [2.73-36.23], *P* =0.001; adjusted OR=0.16; 95% CI [0.05-0.50], *P*=0.002; respectively), and in children with tumor in genitourinary system (adjusted OR=5.03, 95% CI [1.29-19.57], *P* =0.020; adjusted OR=0.25; 95% CI [0.08-0.73], *P*=0.012; respectively). In addition, rs17069665 (GG vs. AA+AG) was related to RMS in children with ERMS (adjusted OR=3.59, 95% CI [1.47-8.76], *P* =0.005), in children with tumor in high risk level (adjusted OR=3.57, 95% CI [1.28-9.90], *P* =0.015), in children with tumor of stage IV (adjusted OR=4.65, 95% CI [1.19-18.22], *P* =0.028), and in children with tumor in trunk and limbs (adjusted OR=4.00, 95% CI [1.20-13.34], *P* =0.024). Rs4946936 (TC+CC vs. TT) was related to RMS in children with tumor in medium risk level (adjusted OR=0.33, 95% CI [0.14-0.76], *P* =0.009) and in children with tumor in head and neck (adjusted OR=0.24, 95% CI [0.06-0.95], *P* =0.043). In the FPRP analysis ([Supplementary Table S2](#SM1)), most of the above significant findings remained noteworthy at the prior probability level of 0.25(FPRP<0.200), which further strengthen the significant associations of rs17069665 and rs4946936 with RMS.

### Table 3.

| Variables | rs17069665(cases/controls) |  | Adjusted OR a | P a | rs4946936(cases/controls) |  |  | Adjusted OR a | P a |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
|   | AA+AG | GG | (95% CI) |   | TT | TC+CC |  | (95% CI) |   |
| Age, years |  |  |  |  |  |  |  |  |  |
| <10 | 93/327 | 11/13 | 2.98 (1.29-6.87) | 0.011 | 17/28 |  | 87/312 | 0.45 (0.24-0.86) | 0.016 |
| ≥10 | 6/19 | 0/0 | 0.36 (0.05-2.50) | – | 0/1 |  | 6/18 | 999 (0.01-999) | 0.984 |
| Gender |  |  |  |  |  |  |  |  |  |
| Female | 40/129 | 4/5 | 2.48 (0.64-9.72) | 0.191 | 4/10 |  | 40/124 | 0.81 (0.24-2.72) | 0.729 |
| Male | 59/217 | 7/8 | 3.33 (1.16-9. 57) | 0.026 | 13/19 |  | 53/206 | 0.36 (0.17-0.78) | 0.009 |
| Histological subtype |  |  |  |  |  |  |  |  |  |
| Embryonal | 65/346 | 9/13 | 3.59 (1.47-8.76) | 0.005 | 11/29 |  | 63/330 | 0.51 (0.24-1.07) | 0.075 |
| Alveolar | 26/346 | 1/13 | 1.11 (0.14-8.87) | 0.925 | 5/29 |  | 22/330 | 0.36 (0.12-1.03) | 0.056 |
| Others | 8/346 | 1/13 | 3.61(0.41-31.76) | 0.247 | 1/29 |  | 8/330 | 0.69 (0.08-5.75) | 0.730 |
| Risk level |  |  |  |  |  |  |  |  |  |
| Low | 12/346 | 2/13 | 4.59(0.92-22.90) | 0.063 | 2/29 |  | 12/330 | 0.52 (0.11-2.45) | 0.410 |
| Medium | 41/346 | 3/13 | 1.83 (0.50-6.77) | 0.365 | 9/29 |  | 35/330 | 0.33 (0.14-0.76) | 0.009 |
| High | 46/346 | 6/13 | 3.57 (1.28-9.90) | 0.015 | 6/29 |  | 46/330 | 0.68 (0.27-1.73) | 0.415 |
| Site |  |  |  |  |  |  |  |  |  |
| Head and neck | 10/346 | 1/13 | 2.52(0.30-21.15) | 0.396 | 3/29 |  | 8/330 | 0.24(0.06-0.95) | 0.043 |
| Trunk and limbs | 29/346 | 4/13 | 4.00(1.20-13.34) | 0.024 | 3/29 |  | 30/330 | 0.83 (0.24-2.91) | 0.767 |
| Genitourinary system | 16/346 | 3/13 | 5.03(1.29-19.57) | 0.020 | 5/29 |  | 14/330 | 0.25(0.08-0.73) | 0.012 |
| Thoracic, abdominal and pelvic cavities | 39/346 | 3/13 | 2.00 (0.54-7.34) | 0.297 | 6/29 |  | 36/330 | 0.54 (0.21-1.39) | 0.203 |
| Others | 5/346 | 0/13 | 0.01 (0.01-999) | 0.985 | 0/29 |  | 5/330 | 999 (0.01-999) | 0.977 |
| Clinical stage |  |  |  |  |  |  |  |  |  |
| I | 7/346 | 0/13 | 0.01 (0.01-999) | 0.981 | 2/29 |  | 5/330 | 0.20(0.04-1.14) | 0.069 |
| II | 10/346 | 4/13 | 9.95(2.73-36.23) | 0.001 | 5/29 |  | 9/330 | 0.16(0.05-0.50) | 0.002 |
| III | 59/346 | 4/13 | 1.78 (0.56-5.66) | 0.331 | 5/29 |  | 58/330 | 1.00 (0.37-2.69) | 0.996 |
| IV | 18/346 | 3/13 | 4.65(1.19-18.22) | 0.028 | 4/29 |  | 17/330 | 0.37 (0.12-1.20) | 0.097 |

Table 3 Caption: Subgroup and stratification analysis of FOXO3 polymorphisms with RMS susceptibility.

### Overall survival analyses

The above results indicated that FOXO3 polymorphisms, rs17069665 and rs4946936, influenced RMS risk in children. However, it’s not clear whether they influence the prognosis of RMS. Therefore, we evaluated the associations of rs17069665 and rs4946936 with overall survival in RMS patients. The results were described in [Figure 1](#f1). For rs17069665, RMS patients with GG allele had a poor overall survival than those with AA/AG ([Figure 1A](#f1), *P* < 0.001). Compared with rs4946936 TT, TC/CC allele significantly contributed to the increased overall survival in RMS patients ([Figure 1B](#f1), *P* < 0.001). The results suggested that FOXO3 polymorphisms were associated with the prognosis in RMS children.

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9f69/11076676/1930b44bac5d/fonc-14-1387735-g001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=11076676_fonc-14-1387735-g001.jpg)

Overall survival of patients with RMS. (A) Survival curve of RMS patients with rs17069665 allele AA/AG or GG; (B) Survival curve of RMS patients with rs4946936 allele TC/CC or TT.

### Functional analysis

To explore the possible mechanisms by which rs17069665 and rs4946936 influence the risk and prognosis of RMS, we evaluated the probability of rs17069665 and rs4946936 polymorphism altering expression regulation of FOXO3. In the eQTL analysis, rs17069665 G allele was associated with lower expression of FOXO3 gene (*P* = 8.52×10^-4^) ([Supplementary Figure S1A](#SF1)). Rs4946936 C allele displayed a trend of high expression of FOXO3, although no significance was achieved (*P* = 3.32×10^-1^) ([Supplementary Figure S1B](#SF1)). The Roadmap Epigenomics data showed that rs17069665 ([Supplementary Figure S2A](#SF2)) and rs4946936 ([Supplementary Figure S2B](#SF2)) both overlapped DHS marks and histone modifications related to the enhancer and promoter in multiple tissue types. These were further supported by H3K9ac, H3K27ac, H3K4me1, H3K4me3, and DHS ChIP data in muscle cells ([Figures 2A, B](#f2)). TFBIND analysis showed that rs17069665 altered the binding affinity to transcription factors including ARNT, BHLHE40, MYC, MYOD, NKX2-5 and USF. Rs4946936 altered the binding affinity to CAP1, CTCF, MYB, RELA and SETDB1. However, ENCODE ChIP-seq analysis showed that only MYC binds to DNA motif overlapping rs17069665 ([Figure 3A](#f3)). MYC has a higher preference for the non-risk allele A than risk allele G ([Figure 3B](#f3)). As for rs4946936, only CTCF and RELA bind to DNA motif overlapping it ([Figure 3C](#f3)). CTCF and RELA have higher preferences for the non-risk allele C than risk allele T ([Figure 3D](#f3)). The results revealed that rs17069665 and rs4946936 might influence the transcription of FOXO3 via altering the bindings to MYC, CTCF and/or RELA.

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9f69/11076676/2385cb723c2c/fonc-14-1387735-g002.jpg)

Rs17069665 and rs4946936 both overlap promoter and enhancer of FOXO3 gene. H3K4me1, H3K4me3, H3K9ac, H3K27ac, and DHS ChIP-seq signals at rs17069665 (A) and rs4946936 (B) loci.

### Figure 3.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9f69/11076676/5f8a184decfe/fonc-14-1387735-g003.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=11076676_fonc-14-1387735-g003.jpg)

Rs17069665 and rs4946936 modulated the binding to MYC, RELA and/or CTCF. (A) MYC ChIP-seq signal at the rs17069665 locus; (B) Predicted preferential binding of MYC to the non-risk allele A of rs17069665; (C) RELA and CTCF ChIP-seq signals at the rs4946936 locus; (D) Predicted preferential bindings of RELA and CTCF to the non-risk allele C of rs4946936.

## Discussion

In this case-control study with 110 RMS cases and 359 healthy controls, we evaluated the potential association of FOXO3 gene polymorphisms with RMS risk and prognosis in Chinese children. Among these four SNPs of FOXO3 in this study, we found that rs17069665 (GG vs. AA+AG) was significantly associated with increased RMS risk and poor prognosis, rs4946936 (TC+CC vs. TT) was significantly associated with decreased RMS risk and good prognosis. To our knowledge, the current study is the first one exploring the association between FOXO3 polymorphisms and RMS.

As a member of FOXO transcription factor family, FOXO3, as well as FOXO1, -4 and -6, controls transcription of target genes by binding to the Forkhead Response Element ([33](#B33)–[35](#B35)). Compared with the other three FOXO members, FOXO3 seemed to be more predominant in controlling cancer progression, although they potentially regulate the same target genes ([36](#B36), [37](#B37)). FOXO3 has been demonstrated to regulate various genes playing key roles in multiple cellular process including proliferation, apoptosis, drug resistance and stem cell properties ([38](#B38)–[42](#B42)). For instance, FOXO3 impaired the cancer stem cell phenotype of squamous cell carcinoma by controlling the transcriptional activity of SOX2 ([11](#B11)). The activation of FOXO3 sensitized tumors to anti-PD-1 therapy by inhibiting c-Myc and STAT3 ([43](#B43)). FOXO3 inhibited cell proliferation and induced apoptosis in colorectal cancer by regulating BIM expression ([44](#B44)). Certainly, FOXO3 is also regulated at multiple levels including genetic regulation and epigenetic modification (transcription, post-transcription, translation and post-translation) ([40](#B40)). FOXO3 protein was phosphorylated by proteins including ERK and AKT, which resulted in its inactivation and degradation ([37](#B37), [45](#B45)). FOXO3 gene polymorphisms were also investigated in multiple cancers, such as leukemia, pancreatic cancer, and hepatocellular carcinoma ([16](#B16), [17](#B17), [46](#B46)). However, FOXO3 polymorphisms have never been reported in RMS. It is urgent to conduct relevant investigations.

In the present study, we explored the association of four FOXO3 SNP sites (rs17069665, rs4946936, rs4945816 and rs9400241) with RMS. Rs17069665 was previously reported to increase ALL susceptibility ([17](#B17)), but had not been reported in other cancers. Rs4946936 was found to be associated with ALL ([47](#B47)), thyroid cancer ([48](#B48)), and head and neck cancer ([49](#B49)), but not with hepatocellular carcinoma ([46](#B46)). In this study, we found that rs17069665 and rs4946936 polymorphisms were associated with the increased and decreased RMS risk, respectively, and influence prognosis, for the first time. Rs17069665 is located in the intron one of FOXO3 gene, which overlaps with the FOXO3 promoter and enhancer. In the analyses of transcription factor binding, rs17069665 was found to disrupt the binding to MYC showing preferential binding of the non-risk allele A. Rs4946936, which is located in the 3’-UTR of FOXO3, was also found to overlap with the promoter and enhancer regions and to alter the binding to CTCF and RELA showing preferential binding of the non-risk allele C. SNP-gene expression analysis indicated that these two polymorphisms influenced the expression of FOXO3. Considering the tumor suppressing function of FOXO3, rs17069665 and rs4946936 might influence RMS risk and prognosis via regulating the expression of FOXO3 by altering the bindings to MYC, CTCF and/or RELA. The above potential mechanisms need to be validated in future studies.

As for the rest two SNPs (rs4945816 and rs9400241), both of which are located in the 3’UTR of FOXO3, no association between them and RMS risk was found. Several previous studies investigated the relationships between them and cancer risks. Rs4945816 was reported in studies on ALL and thyroid cancer, but no association was found ([17](#B17), [48](#B48)). Rs9400241 was associated with ALL and melanoma risks ([17](#B17), [50](#B50)). Along with the previous studies, our study indicates that the genetic variation of FOXO3 is complex, depending on cancer types. Besides, the variety in ethnicity and sample composition need to be taken into consideration.

Although this study is the first to investigate the association of FOXO3 polymorphisms with RMS risk and prognosis, several limitations should be considered. First, all participates were recruited from one hospital in south China, which may cause selection bias. Second, only 4 SNPs were genotyped in the present study and more potentially functional SNPs should be done in the future. Third, the sample size in this study was still not large enough because of the low incidence of RMS in China. Therefore, larger multicenter studies are warranted to further confirm the roles of FOXO3 in RMS. Finally, other factors including environment exposure and dietary intake were not available in this study. The functions of FOXO3 polymorphisms in the progression of RMS also need to be further explored.

## Conclusion

In conclusion, the current study explored the association of FOXO3 polymorphisms (rs17069665, rs4946936, rs4945816 and rs9400241) with RMS in Chinese children and firstly demonstrated that rs17069665 was associated with increased RMS susceptibility and poor prognosis, while rs4946936 was associated with decreased RMS susceptibility and good prognosis, and that the two polymorphisms might influence the transcription of FOXO3 via altering the binding to MYC, CTCF and/or RELA. This study indicated that FOXO3 polymorphism might serve as a biomarker for RMS susceptibility and prognosis. Certainly, larger multicenter studies, as well as functional experiments, are encouraged to further elucidate the role of FOXO3 polymorphism and the underlying mechanisms in RMS.

## Data availability statement

The original contributions presented in the study are included in the article/[Supplementary Material](#SM1). Further inquiries can be directed to the corresponding authors.

## Ethics statement

The studies involving humans were approved by Institutional Review Boards of Guangzhou Women and Children’s Medical Center. The studies were conducted in accordance with the local legislation and institutional requirements. Written informed consent for participation in this study was provided by the participants’ legal guardians/next of kin.

## Author contributions

XZ: Conceptualization, Writing – original draft, Writing – review & editing, Methodology, Data curation, Formal analysis, Validation. YS: Writing – original draft, Writing – review & editing, Conceptualization, Data curation, Formal analysis, Methodology, Funding acquisition. HN: Writing – original draft, Writing – review & editing, Investigation. PT: Writing – original draft, Writing – review & editing, Investigation. SL: Investigation, Writing – original draft, Writing – review & editing. XPL: Investigation, Writing – original draft, Writing – review & editing. XDL: Investigation, Writing – original draft, Writing – review & editing. AL: Investigation, Writing – original draft, Writing – review & editing. MC: Investigation, Writing – original draft, Writing – review & editing. YY: Investigation, Writing – original draft, Writing – review & editing. LX: Writing – original draft, Writing – review & editing, Conceptualization, Funding acquisition, Supervision. XY: Conceptualization, Funding acquisition, Supervision, Writing – original draft, Writing – review & editing, Methodology, Project administration.

## Acknowledgments

The authors thank all the patients and parents for their contribution and acknowledge the ENCODE Consortium and the ENCODE production laboratories generating the particular datasets.

## Funding Statement

The author(s) declare financial support was received for the research, authorship, and/or publication of this article. This study was supported by grants from National Natural Science Foundation of China (82101959, 81870115, and 32001043), Guangdong Basic and Applied Basic Research Foundation (2020A1515110886), and Guangzhou Municipal Science and Technology Project (202102020344).

## Conflict of interest

The authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.

## Publisher’s note

All claims expressed in this article are solely those of the authors and do not necessarily represent those of their affiliated organizations, or those of the publisher, the editors and the reviewers. Any product that may be evaluated in this article, or claim that may be made by its manufacturer, is not guaranteed or endorsed by the publisher.

## Supplementary material

The Supplementary Material for this article can be found online at: [https://www.frontiersin.org/articles/10.3389/fonc.2024.1387735/full#supplementary-material](https://www.frontiersin.org/articles/10.3389/fonc.2024.1387735/full#supplementary-material)

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

The original contributions presented in the study are included in the article/[Supplementary Material](#SM1). Further inquiries can be directed to the corresponding authors.

### Supplementary Materials

### Data Availability Statement

The original contributions presented in the study are included in the article/[Supplementary Material](#SM1). Further inquiries can be directed to the corresponding authors.

## References

1. Gasparini P, Ferrari A, Casanova M, Limido F, Massimino M, Sozzi G, et al. MiRNAs as players in rhabdomyosarcoma development. Int J Mol Sci. (2019) 20:5818. doi: 10.3390/ijms20225818  [DOI](https://doi.org/10.3390/ijms20225818) | [PMC free article](/articles/PMC6888285/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31752446/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Mol%20Sci&title=MiRNAs%20as%20players%20in%20rhabdomyosarcoma%20development&author=P%20Gasparini&author=A%20Ferrari&author=M%20Casanova&author=F%20Limido&author=M%20Massimino&volume=20&publication_year=2019&pages=5818&pmid=31752446&doi=10.3390/ijms20225818&)

2. Kashi VP, Hatley ME, Galindo RL. Probing for a deeper understanding of rhabdomyosarcoma: insights from complementary model systems. Nat Rev Cancer. (2015) 15:426–39. doi: 10.1038/nrc3961  [DOI](https://doi.org/10.1038/nrc3961) | [PMC free article](/articles/PMC4599785/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26105539/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Cancer&title=Probing%20for%20a%20deeper%20understanding%20of%20rhabdomyosarcoma:%20insights%20from%20complementary%20model%20systems&author=VP%20Kashi&author=ME%20Hatley&author=RL%20Galindo&volume=15&publication_year=2015&pmid=26105539&doi=10.1038/nrc3961&)

3. Ognjanovic S, Linabery AM, Charbonneau B, Ross JA. Trends in childhood rhabdomyosarcoma incidence and survival in the United States, 1975-2005. Cancer-Am Cancer Soc. (2009) 115:4218–26. doi: 10.1002/cncr.24465  [DOI](https://doi.org/10.1002/cncr.24465) | [PMC free article](/articles/PMC2953716/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19536876/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer-Am%20Cancer%20Soc&title=Trends%20in%20childhood%20rhabdomyosarcoma%20incidence%20and%20survival%20in%20the%20United%20States,%201975-2005&author=S%20Ognjanovic&author=AM%20Linabery&author=B%20Charbonneau&author=JA%20Ross&volume=115&publication_year=2009&pmid=19536876&doi=10.1002/cncr.24465&)

4. Xu N, Hua Z, Ba G, Zhang S, Liu Z, Thiele CJ, et al. The anti-tumor growth effect of a novel agent DMAMCL in rhabdomyosarcoma in vitro and in vivo . J Exp Clin Canc Res. (2019) 38:118. doi: 10.1186/s13046-019-1107-1  [DOI](https://doi.org/10.1186/s13046-019-1107-1) | [PMC free article](/articles/PMC6408795/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30850026/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Exp%20Clin%20Canc%20Res&title=The%20anti-tumor%20growth%20effect%20of%20a%20novel%20agent%20DMAMCL%20in%20rhabdomyosarcoma%20in%20vitro%20and%20in%20vivo&author=N%20Xu&author=Z%20Hua&author=G%20Ba&author=S%20Zhang&author=Z%20Liu&volume=38&publication_year=2019&pmid=30850026&doi=10.1186/s13046-019-1107-1&)

5. McCarthy N, Dolgikh N, Logue S, Patterson JB, Zeng Q, Gorman AM, et al. The IRE1 and PERK arms of the unfolded protein response promote survival of rhabdomyosarcoma cells. Cancer Lett. (2020) 490:76–88. doi: 10.1016/j.canlet.2020.07.009  [DOI](https://doi.org/10.1016/j.canlet.2020.07.009) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32679165/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Lett&title=The%20IRE1%20and%20PERK%20arms%20of%20the%20unfolded%20protein%20response%20promote%20survival%20of%20rhabdomyosarcoma%20cells&author=N%20McCarthy&author=N%20Dolgikh&author=S%20Logue&author=JB%20Patterson&author=Q%20Zeng&volume=490&publication_year=2020&pages=76-88&pmid=32679165&doi=10.1016/j.canlet.2020.07.009&)

6. Labib RM A, Abdelrahim ME, Elnadi E, RM H, Yassin D. CYP2B6rs2279343 is associated with improved survival of pediatric rhabdomyosarcoma treated with cyclophosphamide. PloS One. (2016) 11:e158890. doi: 10.1371/journal.pone.0158890  [DOI](https://doi.org/10.1371/journal.pone.0158890) | [PMC free article](/articles/PMC4936837/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27388155/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PloS%20One&title=CYP2B6rs2279343%20is%20associated%20with%20improved%20survival%20of%20pediatric%20rhabdomyosarcoma%20treated%20with%20cyclophosphamide&author=A%20Labib%20RM&author=ME%20Abdelrahim&author=E%20Elnadi&author=H%20RM&author=D%20Yassin&volume=11&publication_year=2016&pages=e158890&pmid=27388155&doi=10.1371/journal.pone.0158890&)

7. Nishimura R, Takita J, Sato Otsubo A, Kato M, Koh K, Hanada R, et al. Characterization of genetic lesions in rhabdomyosarcoma using a high-density single nucleotide polymorphism array. Cancer Sci. (2013) 104:856–64. doi: 10.1111/cas.12173  [DOI](https://doi.org/10.1111/cas.12173) | [PMC free article](/articles/PMC7657110/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23578105/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Sci&title=Characterization%20of%20genetic%20lesions%20in%20rhabdomyosarcoma%20using%20a%20high-density%20single%20nucleotide%20polymorphism%20array&author=R%20Nishimura&author=J%20Takita&author=A%20Sato%20Otsubo&author=M%20Kato&author=K%20Koh&volume=104&publication_year=2013&pmid=23578105&doi=10.1111/cas.12173&)

8. Dong Z, Zhong X, Lei Q, Chen F, Cui H. Transcriptional activation of SIRT6 via FKHRL1/FOXO3a inhibits the Warburg effect in glioblastoma cells. Cell Signal. (2019) 60:100–13. doi: 10.1016/j.cellsig.2019.04.009  [DOI](https://doi.org/10.1016/j.cellsig.2019.04.009) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31004738/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Signal&title=Transcriptional%20activation%20of%20SIRT6%20via%20FKHRL1/FOXO3a%20inhibits%20the%20Warburg%20effect%20in%20glioblastoma%20cells&author=Z%20Dong&author=X%20Zhong&author=Q%20Lei&author=F%20Chen&author=H%20Cui&volume=60&publication_year=2019&pmid=31004738&doi=10.1016/j.cellsig.2019.04.009&)

9. Guo X, Li Z, Zhu X, Zhan M, Wu C, Ding X, et al. A coherent FOXO3-SNAI2 feed-forward loop in autophagy. Proc Natl Acad Sci. (2022) 119:e2118285119. doi: 10.1073/pnas.2118285119  [DOI](https://doi.org/10.1073/pnas.2118285119) | [PMC free article](/articles/PMC8931368/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35271390/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci&title=A%20coherent%20FOXO3-SNAI2%20feed-forward%20loop%20in%20autophagy&author=X%20Guo&author=Z%20Li&author=X%20Zhu&author=M%20Zhan&author=C%20Wu&volume=119&publication_year=2022&pages=e2118285119&pmid=35271390&doi=10.1073/pnas.2118285119&)

10. Kim CG, Lee H, Gupta N, Ramachandran S, Kaushik I, Srivastava S, et al. Role of Forkhead Box Class O proteins in cancer progression and metastasis. Semin Cancer Biol. (2018) 50:142–51. doi: 10.1016/j.semcancer.2017.07.007  [DOI](https://doi.org/10.1016/j.semcancer.2017.07.007) | [PMC free article](/articles/PMC5794649/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28774834/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Semin%20Cancer%20Biol&title=Role%20of%20Forkhead%20Box%20Class%20O%20proteins%20in%20cancer%20progression%20and%20metastasis&author=CG%20Kim&author=H%20Lee&author=N%20Gupta&author=S%20Ramachandran&author=I%20Kaushik&volume=50&publication_year=2018&pmid=28774834&doi=10.1016/j.semcancer.2017.07.007&)

11. Chen Y, Zhao H, Liang W, Jiang E, Zhou X, Shao Z, et al. Autophagy regulates the cancer stem cell phenotype of head and neck squamous cell carcinoma through the noncanonical FOXO3/SOX2 axis. Oncogene. (2022) 41:634–46. doi: 10.1038/s41388-021-02115-7  [DOI](https://doi.org/10.1038/s41388-021-02115-7) | [PMC free article](/articles/PMC8799462/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34795388/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncogene&title=Autophagy%20regulates%20the%20cancer%20stem%20cell%20phenotype%20of%20head%20and%20neck%20squamous%20cell%20carcinoma%20through%20the%20noncanonical%20FOXO3/SOX2%20axis&author=Y%20Chen&author=H%20Zhao&author=W%20Liang&author=E%20Jiang&author=X%20Zhou&volume=41&publication_year=2022&pmid=34795388&doi=10.1038/s41388-021-02115-7&)

12. Coomans De Brachène A, Demoulin J. FOXO transcription factors in cancer development and therapy. Cell Mol Life Sci. (2016) 73:1159–72. doi: 10.1007/s00018-015-2112-y  [DOI](https://doi.org/10.1007/s00018-015-2112-y) | [PMC free article](/articles/PMC11108379/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26686861/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Mol%20Life%20Sci&title=FOXO%20transcription%20factors%20in%20cancer%20development%20and%20therapy&author=A%20Coomans%20De%20Brach%C3%A8ne&author=J%20Demoulin&volume=73&publication_year=2016&pmid=26686861&doi=10.1007/s00018-015-2112-y&)

13. Panagopoulos I, Gorunova L, Andersen K, Lund-Iversen M, Tafjord S, Micci F, et al. Fusion of the paired box 3 (PAX3) and myocardin (MYOCD) genes in pediatric rhabdomyosarcoma. Cancer Genomics - Proteomics. (2021) 18:723–34. doi: 10.21873/cgp.20293  [DOI](https://doi.org/10.21873/cgp.20293) | [PMC free article](/articles/PMC8569814/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34697065/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Genomics%20-%20Proteomics&title=Fusion%20of%20the%20paired%20box%203%20(PAX3)%20and%20myocardin%20(MYOCD)%20genes%20in%20pediatric%20rhabdomyosarcoma&author=I%20Panagopoulos&author=L%20Gorunova&author=K%20Andersen&author=M%20Lund-Iversen&author=S%20Tafjord&volume=18&publication_year=2021&pmid=34697065&doi=10.21873/cgp.20293&)

14. Yao J, Wang J, Xu Y, Guo Q, Sun Y, Liu J, et al. CDK9 inhibition blocks the initiation of PINK1-PRKN-mediated mitophagy by regulating the SIRT1-FOXO3-BNIP3 axis and enhances the therapeutic effects involving mitochondrial dysfunction in hepatocellular carcinoma. Autophagy. (2021) 18:1879–97. doi: 10.1080/15548627.2021.2007027  [DOI](https://doi.org/10.1080/15548627.2021.2007027) | [PMC free article](/articles/PMC9450969/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34890308/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Autophagy&title=CDK9%20inhibition%20blocks%20the%20initiation%20of%20PINK1-PRKN-mediated%20mitophagy%20by%20regulating%20the%20SIRT1-FOXO3-BNIP3%20axis%20and%20enhances%20the%20therapeutic%20effects%20involving%20mitochondrial%20dysfunction%20in%20hepatocellular%20carcinoma&author=J%20Yao&author=J%20Wang&author=Y%20Xu&author=Q%20Guo&author=Y%20Sun&volume=18&publication_year=2021&pages=1879-97&pmid=34890308&doi=10.1080/15548627.2021.2007027&)

15. Miyamoto Y, Schirripa M, Suenaga M, Cao S, Zhang W, Okazaki S, et al. A polymorphism in the cachexia-associated gene INHBA predicts efficacy of regorafenib in patients with refractory metastatic colorectal cancer. PloS One. (2020) 15:e239439. doi: 10.1371/journal.pone.0239439  [DOI](https://doi.org/10.1371/journal.pone.0239439) | [PMC free article](/articles/PMC7514061/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32970737/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PloS%20One&title=A%20polymorphism%20in%20the%20cachexia-associated%20gene%20INHBA%20predicts%20efficacy%20of%20regorafenib%20in%20patients%20with%20refractory%20metastatic%20colorectal%20cancer&author=Y%20Miyamoto&author=M%20Schirripa&author=M%20Suenaga&author=S%20Cao&author=W%20Zhang&volume=15&publication_year=2020&pages=e239439&pmid=32970737&doi=10.1371/journal.pone.0239439&)

16. Yan H, Li Q, Wu J, Hu W, Jiang J, Shi L, et al. MiR-629 promotes human pancreatic cancer progression by targeting FOXO3. Cell Death Dis. (2017) 8:e3154. doi: 10.1038/cddis.2017.525  [DOI](https://doi.org/10.1038/cddis.2017.525) | [PMC free article](/articles/PMC5682687/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29072689/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Death%20Dis&title=MiR-629%20promotes%20human%20pancreatic%20cancer%20progression%20by%20targeting%20FOXO3&author=H%20Yan&author=Q%20Li&author=J%20Wu&author=W%20Hu&author=J%20Jiang&volume=8&publication_year=2017&pages=e3154&pmid=29072689&doi=10.1038/cddis.2017.525&)

17. Yang X, Wu X, Fang N, Liu X, Liu X, Yang L, et al. FOXO3 gene polymorphisms influence the risk of acute lymphoblastic leukemia in Chinese children. J Cell Biochem. (2020) 121:2019–26. doi: 10.1002/jcb.29436  [DOI](https://doi.org/10.1002/jcb.29436) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31691337/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Cell%20Biochem&title=FOXO3%20gene%20polymorphisms%20influence%20the%20risk%20of%20acute%20lymphoblastic%20leukemia%20in%20Chinese%20children&author=X%20Yang&author=X%20Wu&author=N%20Fang&author=X%20Liu&author=X%20Liu&volume=121&publication_year=2020&pmid=31691337&doi=10.1002/jcb.29436&)

18. Uroog L, Rahmani AH, Alsahli MA, Almatroodi SA, Wani RA, Rizvi M. Genetic profile of FOXO3 Single Nucleotide Polymorphism in colorectal cancer patients. Oncology. (2023) 102:299–309. doi: 10.1159/000533729  [DOI](https://doi.org/10.1159/000533729) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/37857267/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncology&title=Genetic%20profile%20of%20FOXO3%20Single%20Nucleotide%20Polymorphism%20in%20colorectal%20cancer%20patients&author=L%20Uroog&author=AH%20Rahmani&author=MA%20Alsahli&author=SA%20Almatroodi&author=RA%20Wani&volume=102&publication_year=2023&pages=299-309&pmid=37857267&doi=10.1159/000533729&)

19. Lu Y, Zhu Y, Wang X, Wang F, Peng J, Hou H, et al. FOXO3 rs12212067: T > G association with active tuberculosis in han chinese population. Inflammation. (2016) 39:10–5. doi: 10.1007/s10753-015-0217-y  [DOI](https://doi.org/10.1007/s10753-015-0217-y) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26223437/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Inflammation&title=FOXO3%20rs12212067:%20T%20>%20G%20association%20with%20active%20tuberculosis%20in%20han%20chinese%20population&author=Y%20Lu&author=Y%20Zhu&author=X%20Wang&author=F%20Wang&author=J%20Peng&volume=39&publication_year=2016&pmid=26223437&doi=10.1007/s10753-015-0217-y&)

20. Rakhshani Nejad A, Sargazi S, Ghasemi M, Samareh Moosavi S, Heidari Nia M, Saravani R. Association study to evaluate Foxo1 and Foxo3 gene polymorphisms in polycystic ovary syndrome: a preliminary case-control study and in silico analysis. Mol Biol Rep. (2023) 50:3569–80. doi: 10.1007/s11033-023-08292-w  [DOI](https://doi.org/10.1007/s11033-023-08292-w) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36790598/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Biol%20Rep&title=Association%20study%20to%20evaluate%20Foxo1%20and%20Foxo3%20gene%20polymorphisms%20in%20polycystic%20ovary%20syndrome:%20a%20preliminary%20case-control%20study%20and%20in%20silico%20analysis&author=A%20Rakhshani%20Nejad&author=S%20Sargazi&author=M%20Ghasemi&author=S%20Samareh%20Moosavi&author=M%20Heidari%20Nia&volume=50&publication_year=2023&pmid=36790598&doi=10.1007/s11033-023-08292-w&)

21. Guo H, Li N, Sun Y, Wu C, Deng H, Xu L, et al. MYBL2 gene polymorphism is associated with acute lymphoblastic leukemia susceptibility in children. Front Oncol. (2021) 11:734588. doi: 10.3389/fonc.2021.734588  [DOI](https://doi.org/10.3389/fonc.2021.734588) | [PMC free article](/articles/PMC8456030/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34568071/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front%20Oncol&title=MYBL2%20gene%20polymorphism%20is%20associated%20with%20acute%20lymphoblastic%20leukemia%20susceptibility%20in%20children&author=H%20Guo&author=N%20Li&author=Y%20Sun&author=C%20Wu&author=H%20Deng&volume=11&publication_year=2021&pages=734588&pmid=34568071&doi=10.3389/fonc.2021.734588&)

22. He J, Qiu L, Wang M, Hua R, Zhang R, Yu H, et al. Polymorphisms in the XPG gene and risk of gastric cancer in Chinese populations. Hum Genet. (2012) 131:1235–44. doi: 10.1007/s00439-012-1152-8  [DOI](https://doi.org/10.1007/s00439-012-1152-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22371296/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Genet&title=Polymorphisms%20in%20the%20XPG%20gene%20and%20risk%20of%20gastric%20cancer%20in%20Chinese%20populations&author=J%20He&author=L%20Qiu&author=M%20Wang&author=R%20Hua&author=R%20Zhang&volume=131&publication_year=2012&pmid=22371296&doi=10.1007/s00439-012-1152-8&)

23. Xu Z, Taylor JA. SNPinfo: integrating GWAS and candidate gene information into functional SNP selection for genetic association studies. Nucleic Acids Res. (2009) 37:W600–5. doi: 10.1093/nar/gkp290  [DOI](https://doi.org/10.1093/nar/gkp290) | [PMC free article](/articles/PMC2703930/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19417063/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=SNPinfo:%20integrating%20GWAS%20and%20candidate%20gene%20information%20into%20functional%20SNP%20selection%20for%20genetic%20association%20studies&author=Z%20Xu&author=JA%20Taylor&volume=37&publication_year=2009&pmid=19417063&doi=10.1093/nar/gkp290&)

24. Lou J, Gong J, Ke J, Tian J, Zhang Y, Li J, et al. A functional polymorphism located at transcription factor binding sites, rs6695837 near LAMC1 gene, confers risk of colorectal cancer in Chinese populations. Carcinogenesis. (2017) 38:w204. doi: 10.1093/carcin/bgw204  [DOI](https://doi.org/10.1093/carcin/bgw204) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28039327/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Carcinogenesis&title=A%20functional%20polymorphism%20located%20at%20transcription%20factor%20binding%20sites,%20rs6695837%20near%20LAMC1%20gene,%20confers%20risk%20of%20colorectal%20cancer%20in%20Chinese%20populations&author=J%20Lou&author=J%20Gong&author=J%20Ke&author=J%20Tian&author=Y%20Zhang&volume=38&publication_year=2017&pages=w204&pmid=28039327&doi=10.1093/carcin/bgw204&)

25. Yang X, He J, Chang Y, Luo A, Luo A, Zhang J, et al. HOTAIR gene polymorphisms contribute to increased neuroblastoma susceptibility in Chinese children. Cancer-Am Cancer Soc. (2018) 124:2599–606. doi: 10.1002/cncr.31353  [DOI](https://doi.org/10.1002/cncr.31353) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29603181/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer-Am%20Cancer%20Soc&title=HOTAIR%20gene%20polymorphisms%20contribute%20to%20increased%20neuroblastoma%20susceptibility%20in%20Chinese%20children&author=X%20Yang&author=J%20He&author=Y%20Chang&author=A%20Luo&author=A%20Luo&volume=124&publication_year=2018&pmid=29603181&doi=10.1002/cncr.31353&)

26. Consortium TG. The genotype-tissue expression (GTEx) project. Nat Genet. (2013) 45:580–5. doi: 10.1038/ng.2653  [DOI](https://doi.org/10.1038/ng.2653) | [PMC free article](/articles/PMC4010069/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23715323/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=The%20genotype-tissue%20expression%20(GTEx)%20project&author=TG%20Consortium&volume=45&publication_year=2013&pmid=23715323&doi=10.1038/ng.2653&)

27. Kundaje A, Meuleman W, Ernst J, Bilenky M, Yen A, Heravi-Moussavi A, et al. Integrative analysis of 111 reference human epigenomes. Nature. (2015) 518:317–30. doi: 10.1038/nature14248  [DOI](https://doi.org/10.1038/nature14248) | [PMC free article](/articles/PMC4530010/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25693563/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Integrative%20analysis%20of%20111%20reference%20human%20epigenomes&author=A%20Kundaje&author=W%20Meuleman&author=J%20Ernst&author=M%20Bilenky&author=A%20Yen&volume=518&publication_year=2015&pmid=25693563&doi=10.1038/nature14248&)

28. Zhou X, Li D, Zhang B, Lowdon RF, Rockweiler NB, Sears RL, et al. Epigenomic annotation of genetic variants using the Roadmap Epigenome Browser. Nat Biotechnol. (2015) 33:345–6. doi: 10.1038/nbt.3158  [DOI](https://doi.org/10.1038/nbt.3158) | [PMC free article](/articles/PMC4467764/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25690851/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Biotechnol&title=Epigenomic%20annotation%20of%20genetic%20variants%20using%20the%20Roadmap%20Epigenome%20Browser&author=X%20Zhou&author=D%20Li&author=B%20Zhang&author=RF%20Lowdon&author=NB%20Rockweiler&volume=33&publication_year=2015&pmid=25690851&doi=10.1038/nbt.3158&)

29. Consortium TEP. An integrated encyclopedia of DNA elements in the human genome. Nature. (2012) 489:57–74. doi: 10.1038/nature11247  [DOI](https://doi.org/10.1038/nature11247) | [PMC free article](/articles/PMC3439153/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22955616/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=An%20integrated%20encyclopedia%20of%20DNA%20elements%20in%20the%20human%20genome&author=TEP%20Consortium&volume=489&publication_year=2012&pages=57-74&pmid=22955616&doi=10.1038/nature11247&)

30. Davis CA, Hitz BC, Sloan CA, Chan ET, Davidson JM, Gabdank I, et al. The Encyclopedia of DNA elements (ENCODE): data portal update. Nucleic Acids Res. (2018) 46:D794–801. doi: 10.1093/nar/gkx1081  [DOI](https://doi.org/10.1093/nar/gkx1081) | [PMC free article](/articles/PMC5753278/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29126249/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=The%20Encyclopedia%20of%20DNA%20elements%20(ENCODE):%20data%20portal%20update&author=CA%20Davis&author=BC%20Hitz&author=CA%20Sloan&author=ET%20Chan&author=JM%20Davidson&volume=46&publication_year=2018&pmid=29126249&doi=10.1093/nar/gkx1081&)

31. Tsunoda T, Takagi T. Estimating transcription factor bindability on DNA. Bioinformatics. (1999) 15:622–30. doi: 10.1093/bioinformatics/15.7.622  [DOI](https://doi.org/10.1093/bioinformatics/15.7.622) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10487870/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=Estimating%20transcription%20factor%20bindability%20on%20DNA&author=T%20Tsunoda&author=T%20Takagi&volume=15&publication_year=1999&pmid=10487870&doi=10.1093/bioinformatics/15.7.622&)

32. Wacholder S, Chanock S, Garcia-Closas M, El GL, Rothman N. Assessing the probability that a positive report is false: an approach for molecular epidemiology studies. J Natl Cancer Inst. (2004) 96:434–42. doi: 10.1093/jnci/djh075  [DOI](https://doi.org/10.1093/jnci/djh075) | [PMC free article](/articles/PMC7713993/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15026468/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Natl%20Cancer%20Inst&title=Assessing%20the%20probability%20that%20a%20positive%20report%20is%20false:%20an%20approach%20for%20molecular%20epidemiology%20studies&author=S%20Wacholder&author=S%20Chanock&author=M%20Garcia-Closas&author=GL%20El&author=N%20Rothman&volume=96&publication_year=2004&pmid=15026468&doi=10.1093/jnci/djh075&)

33. Chiacchiera F, Simone C. The AMPK-FoxO3A axis as a target for cancer treatment. Cell Cycle. (2010) 9:1091–6. doi: 10.4161/cc.9.6.11035  [DOI](https://doi.org/10.4161/cc.9.6.11035) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20190568/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Cycle&title=The%20AMPK-FoxO3A%20axis%20as%20a%20target%20for%20cancer%20treatment&author=F%20Chiacchiera&author=C%20Simone&volume=9&publication_year=2010&pmid=20190568&doi=10.4161/cc.9.6.11035&)

34. Jiramongkol Y, Lam EW. FOXO transcription factor family in cancer and metastasis. Cancer Metastasis Rev. (2020) 39:681–709. doi: 10.1007/s10555-020-09883-w  [DOI](https://doi.org/10.1007/s10555-020-09883-w) | [PMC free article](/articles/PMC7497309/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32372224/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Metastasis%20Rev&title=FOXO%20transcription%20factor%20family%20in%20cancer%20and%20metastasis&author=Y%20Jiramongkol&author=EW%20Lam&volume=39&publication_year=2020&pages=681-709&pmid=32372224&doi=10.1007/s10555-020-09883-w&)

35. Yusuf D, Butland SL, Swanson MI, Bolotin E, Ticoll A, Cheung WA, et al. The transcription factor encyclopedia. Genome Biol. (2012) 13:R24. doi: 10.1186/gb-2012-13-3-r24  [DOI](https://doi.org/10.1186/gb-2012-13-3-r24) | [PMC free article](/articles/PMC3439975/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22458515/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Biol&title=The%20transcription%20factor%20encyclopedia&author=D%20Yusuf&author=SL%20Butland&author=MI%20Swanson&author=E%20Bolotin&author=A%20Ticoll&volume=13&publication_year=2012&pages=R24&pmid=22458515&doi=10.1186/gb-2012-13-3-r24&)

36. Lam EW, Brosens JJ, Gomes AR, Koo CY. Forkhead box proteins: tuning forks for transcriptional harmony. Nat Rev Cancer. (2013) 13:482–95. doi: 10.1038/nrc3539  [DOI](https://doi.org/10.1038/nrc3539) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23792361/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Cancer&title=Forkhead%20box%20proteins:%20tuning%20forks%20for%20transcriptional%20harmony&author=EW%20Lam&author=JJ%20Brosens&author=AR%20Gomes&author=CY%20Koo&volume=13&publication_year=2013&pmid=23792361&doi=10.1038/nrc3539&)

37. Wu W, Lin J, Pan C, Chan T, Liu C, Wu W, et al. Amplification-driven BCL6-suppressed cytostasis is mediated by transrepression of FOXO3 and post-translational modifications of FOXO3 in urinary bladder urothelial carcinoma. Theranostics. (2020) 10:707–24. doi: 10.7150/thno.39018  [DOI](https://doi.org/10.7150/thno.39018) | [PMC free article](/articles/PMC6929993/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31903146/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Theranostics&title=Amplification-driven%20BCL6-suppressed%20cytostasis%20is%20mediated%20by%20transrepression%20of%20FOXO3%20and%20post-translational%20modifications%20of%20FOXO3%20in%20urinary%20bladder%20urothelial%20carcinoma&author=W%20Wu&author=J%20Lin&author=C%20Pan&author=T%20Chan&author=C%20Liu&volume=10&publication_year=2020&pmid=31903146&doi=10.7150/thno.39018&)

38. Lee HK, Cha HS, Nam MJ, Park K, Yang Y, Lee J, et al. Broussochalcone A induces apoptosis in human renal cancer cells via ROS level elevation and activation of FOXO3 signaling pathway. Oxid Med Cell Longev. (2021) 2021:1–17. doi: 10.1155/2021/2800706  [DOI](https://doi.org/10.1155/2021/2800706) | [PMC free article](/articles/PMC8566040/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34745413/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oxid%20Med%20Cell%20Longev&title=Broussochalcone%20A%20induces%20apoptosis%20in%20human%20renal%20cancer%20cells%20via%20ROS%20level%20elevation%20and%20activation%20of%20FOXO3%20signaling%20pathway&author=HK%20Lee&author=HS%20Cha&author=MJ%20Nam&author=K%20Park&author=Y%20Yang&volume=2021&publication_year=2021&pages=1-17&pmid=34745413&doi=10.1155/2021/2800706&)

39. Liu H, Song Y, Qiu H, Liu Y, Luo K, Yi Y, et al. Downregulation of FOXO3a by DNMT1 promotes breast cancer stem cell properties and tumorigenesis. Cell Death Differentiat. (2020) 27:966–83. doi: 10.1038/s41418-019-0389-3  [DOI](https://doi.org/10.1038/s41418-019-0389-3) | [PMC free article](/articles/PMC7206060/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31296961/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Death%20Differentiat&title=Downregulation%20of%20FOXO3a%20by%20DNMT1%20promotes%20breast%20cancer%20stem%20cell%20properties%20and%20tumorigenesis&author=H%20Liu&author=Y%20Song&author=H%20Qiu&author=Y%20Liu&author=K%20Luo&volume=27&publication_year=2020&pmid=31296961&doi=10.1038/s41418-019-0389-3&)

40. Liu Y, Ao X, Ding W, Ponnusamy M, Wu W, Hao X, et al. Critical role of FOXO3a in carcinogenesis. Mol Cancer. (2018) 17:104. doi: 10.1186/s12943-018-0856-3  [DOI](https://doi.org/10.1186/s12943-018-0856-3) | [PMC free article](/articles/PMC6060507/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30045773/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Cancer&title=Critical%20role%20of%20FOXO3a%20in%20carcinogenesis&author=Y%20Liu&author=X%20Ao&author=W%20Ding&author=M%20Ponnusamy&author=W%20Wu&volume=17&publication_year=2018&pages=104&pmid=30045773&doi=10.1186/s12943-018-0856-3&)

41. Luo L, Zhang Z, Qiu N, Ling L, Jia X, Song Y, et al. Disruption of FOXO3a-miRNA feedback inhibition of IGF2/IGF-1R/IRS1 signaling confers Herceptin resistance in HER2-positive breast cancer. Nat Commun. (2021) 12:2699. doi: 10.1038/s41467-021-23052-9  [DOI](https://doi.org/10.1038/s41467-021-23052-9) | [PMC free article](/articles/PMC8113606/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33976188/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&title=Disruption%20of%20FOXO3a-miRNA%20feedback%20inhibition%20of%20IGF2/IGF-1R/IRS1%20signaling%20confers%20Herceptin%20resistance%20in%20HER2-positive%20breast%20cancer&author=L%20Luo&author=Z%20Zhang&author=N%20Qiu&author=L%20Ling&author=X%20Jia&volume=12&publication_year=2021&pages=2699&pmid=33976188&doi=10.1038/s41467-021-23052-9&)

42. Qin Y, Sun W, Wang Z, Dong W, He L, Zhang T, et al. RBM47/SNHG5/FOXO3 axis activates autophagy and inhibits cell proliferation in papillary thyroid carcinoma. Cell Death Dis. (2022) 13:270. doi: 10.1038/s41419-022-04728-6  [DOI](https://doi.org/10.1038/s41419-022-04728-6) | [PMC free article](/articles/PMC8956740/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35338124/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Death%20Dis&title=RBM47/SNHG5/FOXO3%20axis%20activates%20autophagy%20and%20inhibits%20cell%20proliferation%20in%20papillary%20thyroid%20carcinoma&author=Y%20Qin&author=W%20Sun&author=Z%20Wang&author=W%20Dong&author=L%20He&volume=13&publication_year=2022&pages=270&pmid=35338124&doi=10.1038/s41419-022-04728-6&)

43. Chung YM, Khan PP, Wang H, Tsai W, Qiao Y, Yu B, et al. Sensitizing tumors to anti-PD-1 therapy by promoting NK and CD8+ T cells via pharmacological activation of FOXO3. J Immunother Cancer. (2021) 9:e2772. doi: 10.1136/jitc-2021-002772  [DOI](https://doi.org/10.1136/jitc-2021-002772) | [PMC free article](/articles/PMC8663085/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34887262/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Immunother%20Cancer&title=Sensitizing%20tumors%20to%20anti-PD-1%20therapy%20by%20promoting%20NK%20and%20CD8+%20T%20cells%20via%20pharmacological%20activation%20of%20FOXO3&author=YM%20Chung&author=PP%20Khan&author=H%20Wang&author=W%20Tsai&author=Y%20Qiao&volume=9&publication_year=2021&pages=e2772&pmid=34887262&doi=10.1136/jitc-2021-002772&)

44. Ju H, Tan JY, Cao B, Song MQ, Tian ZB. Effects of miR-223 on colorectal cancer cell proliferation and apoptosis through regulating FoxO3a/BIM. Eur Rev Med Pharmacol Sci. (2018) 22:3771–8. doi: 10.26355/eurrev_201806_15259  [DOI](https://doi.org/10.26355/eurrev_201806_15259) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29949152/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20Rev%20Med%20Pharmacol%20Sci&title=Effects%20of%20miR-223%20on%20colorectal%20cancer%20cell%20proliferation%20and%20apoptosis%20through%20regulating%20FoxO3a/BIM&author=H%20Ju&author=JY%20Tan&author=B%20Cao&author=MQ%20Song&author=ZB%20Tian&volume=22&publication_year=2018&pmid=29949152&doi=10.26355/eurrev_201806_15259&)

45. Yang J, Zong CS, Xia W, Yamaguchi H, Ding Q, Xie X, et al. ERK promotes tumorigenesis by inhibiting FOXO3a via MDM2-mediated degradation. Nat Cell Biol. (2008) 10:138–48. doi: 10.1038/ncb1676  [DOI](https://doi.org/10.1038/ncb1676) | [PMC free article](/articles/PMC2376808/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18204439/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Cell%20Biol&title=ERK%20promotes%20tumorigenesis%20by%20inhibiting%20FOXO3a%20via%20MDM2-mediated%20degradation&author=J%20Yang&author=CS%20Zong&author=W%20Xia&author=H%20Yamaguchi&author=Q%20Ding&volume=10&publication_year=2008&pmid=18204439&doi=10.1038/ncb1676&)

46. Tan C, Liu S, Tan S, Zeng X, Yu H, Li A, et al. Polymorphisms in microRNA target sites of forkhead box O genes are associated with hepatocellular carcinoma. PloS One. (2015) 10:e119210. doi: 10.1371/journal.pone.0119210  [DOI](https://doi.org/10.1371/journal.pone.0119210) | [PMC free article](/articles/PMC4357486/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25739100/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PloS%20One&title=Polymorphisms%20in%20microRNA%20target%20sites%20of%20forkhead%20box%20O%20genes%20are%20associated%20with%20hepatocellular%20carcinoma&author=C%20Tan&author=S%20Liu&author=S%20Tan&author=X%20Zeng&author=H%20Yu&volume=10&publication_year=2015&pages=e119210&pmid=25739100&doi=10.1371/journal.pone.0119210&)

47. Wang Y, Zhou L, Chen J, Li J, He L, Wu P, et al. Association of the 3’UTR FOXO3a polymorphism rs4946936 with an increased risk of childhood acute lymphoblastic leukemia in a Chinese population. Cell Physiol Biochem. (2014) 34:325–32. doi: 10.1159/000363002  [DOI](https://doi.org/10.1159/000363002) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25060657/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Physiol%20Biochem&title=Association%20of%20the%203%E2%80%99UTR%20FOXO3a%20polymorphism%20rs4946936%20with%20an%20increased%20risk%20of%20childhood%20acute%20lymphoblastic%20leukemia%20in%20a%20Chinese%20population&author=Y%20Wang&author=L%20Zhou&author=J%20Chen&author=J%20Li&author=L%20He&volume=34&publication_year=2014&pmid=25060657&doi=10.1159/000363002&)

48. Röhlen N, Döring C, Hansmann ML, Grünwald F, Vorländer C, Bechstein WO, et al. FOXO3a Polymorphism rs4946936 and its interaction with vitamin D in differentiated thyroid cancer. Exp Clin Endocrinol Diabetes. (2014) 122:P84. doi: 10.1055/s-0034-1372101  [DOI](https://doi.org/10.1055/s-0034-1372101) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Exp%20Clin%20Endocrinol%20Diabetes&title=FOXO3a%20Polymorphism%20rs4946936%20and%20its%20interaction%20with%20vitamin%20D%20in%20differentiated%20thyroid%20cancer&author=N%20R%C3%B6hlen&author=C%20D%C3%B6ring&author=ML%20Hansmann&author=F%20Gr%C3%BCnwald&author=C%20Vorl%C3%A4nder&volume=122&publication_year=2014&pages=P84&doi=10.1055/s-0034-1372101&)

49. Ahmed MW, Mahjabeen I, Gul S, Khursheed A, Mehmood A, Kayani MA. Relationship of single nucleotide polymorphisms and haplotype interaction of mitochondrial unfolded protein response pathway genes with head and neck cancer. Future Oncol. (2019) 15:3819–29. doi: 10.2217/fon-2019-0365  [DOI](https://doi.org/10.2217/fon-2019-0365) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31651195/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Future%20Oncol&title=Relationship%20of%20single%20nucleotide%20polymorphisms%20and%20haplotype%20interaction%20of%20mitochondrial%20unfolded%20protein%20response%20pathway%20genes%20with%20head%20and%20neck%20cancer&author=MW%20Ahmed&author=I%20Mahjabeen&author=S%20Gul&author=A%20Khursheed&author=A%20Mehmood&volume=15&publication_year=2019&pmid=31651195&doi=10.2217/fon-2019-0365&)

50. Hernando B, Pena-Chilet M, Ibarrola-Villava M, Martin-Gonzalez M, Gomez-Fernandez C, Ribas G, et al. Genetic 3’UTR variation is associated with human pigmentation characteristics and sensitivity to sunlight. Exp Dermatol. (2017) 26:896–903. doi: 10.1111/exd.13333  [DOI](https://doi.org/10.1111/exd.13333) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28266728/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Exp%20Dermatol&title=Genetic%203%E2%80%99UTR%20variation%20is%20associated%20with%20human%20pigmentation%20characteristics%20and%20sensitivity%20to%20sunlight&author=B%20Hernando&author=M%20Pena-Chilet&author=M%20Ibarrola-Villava&author=M%20Martin-Gonzalez&author=C%20Gomez-Fernandez&volume=26&publication_year=2017&pages=896-903&pmid=28266728&doi=10.1111/exd.13333&)
