# The role of genetic polymorphisms in the sulfation of pregnenolone by human cytosolic sulfotransferase SULT2B1a

## Metadata
**Authors:** Eid Alatwi, Ahsan F Bairam
**Journal:** Scientific Reports
**Date:** 2024 Apr 5
**DOI:** [10.1038/s41598-024-56303-y](https://doi.org/10.1038/s41598-024-56303-y)
**PMID:** 38580665
**PMCID:** PMC10997614
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10997614/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC10997614/pdf/41598_2024_Article_56303.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC10997614/pdf/41598_2024_Article_56303.pdf)

## Abstract

Pregnenolone is a key intermediate in the biosynthesis of many steroid hormones and neuroprotective steroids. Sulfotransferase family cytosolic 2B member 1 (SULT2B1a) has been reported to be highly selective to sulfate pregnenolone. This study aimed to clarify the effect of missense single nucleotide polymorphisms (SNPs) of the human SULT2B1 gene on the sulfating activity of coded SULT2B1a allozymes toward Pregnenolone. To investigate the effects of single nucleotide polymorphisms of the SULT2B1 gene on the sulfation of pregnenolone by SULT2B1a allozymes, 13 recombinant SULT2B1a allozymes were generated, expressed, and purified using established procedures. Human SULT2B1a SNPs were identified by a comprehensive database search. 13 SULT2B1a nonsynonymous missense coding SNPs (cSNPs) were selected, and site-directed mutagenesis was used to generate the corresponding cDNAs, packaged in pGEX-2TK expression vector, encoding these 13 SULT2B1a allozymes, which were bacterially expressed in BL21 E. coli cells and purified by glutathione-Sepharose affinity chromatography. Purified SULT2B1a allozymes were analyzed for sulfating activities towards pregnenolone. In comparison with the wild-type SULT2B1a, of the 13 allozymes, 11 showed reduced activity toward pregnenolone at 0.1 µM. Specifically, P134L and R259Q allozymes, reported to be involved in autosomal-recessive congenital ichthyosis, displayed low activity (1–10%) toward pregnenolone. The findings of this study may demonstrate the impact of genetic polymorphism on the sulfation of pregnenolone in individuals with different SULT2B1 genotypes.

Keywords: Pregnenolone, Sulfation, SULT2B1a, SNPs

Subject terms: Biochemistry, Genetics, Physiology, Diseases, Endocrinology, Gastroenterology, Medical research

## Introduction

Pregnenolone, the precursor of steroid hormones, can be synthesized from cholesterol in various organs such as the adrenal cortex, other peripheral steroidogenic tissues, and the brain under the action of cytochrome P450, CYP11A1^1^. Pregnenolone serves as an intermediate in the biosynthesis of many steroid hormones, including progestogens, androgens, estrogens, glucocorticoids, mineralocorticoids, and dehydroepiandrosterone (DHEA)^2^. Pregnenolone and DHEA function as sex hormone precursors and neuroprotective steroids^3^, and their sulfate derivatives display excitatory cellular actions that are important in learning and memory during cognitive aging^4^. In particular, the sulfoconjugated form of pregnenolone, pregnenolone sulfate, is involved in enhancing cognitive abilities and memory through gamma-aminobutyric acid A receptor and N-methyl-D-aspartate receptor^5^. This neuroactive metabolite has been reported to be highly detected in the human brain^6^. Besides, steroid hormone sulfates can also be deconjugated and exert their steroid function in target tissues such as the breast, prostate, and testis^7^.

Steroid hormone sulfates are synthesized by the hydroxysteroid sulfotransferases (SULT2A1 and SULT2B1), which catalyze the transfer of a sulfonate group from 3’-phosphoadenosine-5’-phosphosulfate (PAPS) to the 3-hydroxy group of steroids^8^. Sulfation reactions play essential roles in humans and other mammals, and constitute an important conjugation pathway for metabolizing xenobiotic compounds^9^. In cytosol, inorganic sulfate is conjugated with adenosine monophosphate by adenosine triphosphate (ATP) sulfurylase to form adenosine 5’-phosphosulfate (APS), which is subsequently phosphorylated under the action of APS kinase to form PAPS^10^. There are 13 known cytosolic sulfotransferase genes (*SULTs*) in humans, of which *SULT2* family members (SULT2A1 and SULT2B1) have been shown to be responsible for the sulfation of steroids^11^. The SULT2B1 subfamily consists of two members, SULT2B1a and SULT2B1b, both of which are encoded by the same gene^12^. SULT2B1a and SULT2B1b differ only in their amino terminus. Both SULT2B1a and SULT2B1b are capable of sulfating pregnenolone; SULT2B1a favors more strongly pregnenolone as a substrate^13^. In addition, SULT2B1a has been reported to be able to sulfate DHEA^14^ and the 3α- and 3β-hydroxyl groups present in sterols and bile acids^15^.

Genetic polymorphisms of the *SULT2B1* gene have been reported to be associated with several diseases, and it has recently been reported that SULT2B1 gene polymorphisms affect the sulfation of pregnenolone by the SULT2B1b variant^16^. A total of 10 SULT2B1b allozymes showed decreased sulfating activity toward pregnenolone, implicating that inter-individual variations in the specific activity of SULT2B1b may result in different steroid hormone homeostasis. A noteworthy question, however, is whether the sulfating activity of another SULT2B1 variant, SULT2B1a, can be influenced by nonsynonymous coding SNPs (cSNPs) in *SULT2B1*, thus leading to distinct physiological and pathological effects related to steroid metabolism in different individuals.

Since SULT2B1a plays a significant role in the homeostasis and metabolism of endogenous compounds, this study hypothesized that *SULT2B1* coding SNPs may lead to SULT2B1a allozymes having different sulfating activities toward pregnenolone. In order to verify the hypothesis, a systematic search was performed on *SULT2B1* SNPs deposited in several SNP databases. In total, 13 nonsynonymous cSNPs in *SULT2B1a* were selected and investigated based on the potential importance of their amino acid substitutions. The corresponding SULT2B1a allozymes were generated, expressed, and purified, and their sulfating activities toward pregnenolone were examined with three different substrate concentrations.

## Materials and methods

### Materials

Pregnenolone, adenosine 5′-triphosphate (ATP), 4-(2-hydroxyethyl)-1-piperazineethanesulfonic acid (HEPES), Trizma base, and dithiothreitol (DTT) were from Sigma-Aldrich (St. Louis, MO, USA). PrimeSTAR^®^ Max DNA polymerase was a product of Takara Bio (Mountain View, CA, USA). Oligonucleotide primers were synthesized by Eurofins Genomics (Louisville, KY, USA). *Dpn I* and the DNA ladder were products of New England Biolabs (Ipswich, MA, USA). QIAprep^®^ Spin Miniprep Kit was from QIAGEN (Germantown, MD, USA). Glutathione Sepharose was a product of GE Healthcare Bio-Sciences (Pittsburgh, PA, USA). Cellulose TLC plates and Ultrafree-MC 5000 NMWL filter units were from EMD Millipore (Billerica, MA, USA). X-ray films were sourced from Products International Corporation (Mt Prospect, IL, USA). Ecolume scintillation cocktail was purchased from MP Biomedicals, LLC. (Solon, OH, USA). Carrier-free sodium [^35^S] sulfate was from American Radiolabeled Chemicals (St. Louis, MO, USA). [^35^S]PAPS was synthesized from adenosine triphosphate (ATP) and carrier-free [^35^S]sulfate using recombinant human bifunctional ATP sulfurylase/ adenosine 5’-phosphosulfate (APS) kinase. The [^35^S]PAPS synthesized was adjusted to the required concentration and specific activity of 15 Ci/mmol at 1.4 mM by the addition of unlabeled PAPS^17^. All other chemicals were of the highest grade commercially available.

## Methods

### Identification and analysis of human SULT2B1 SNPs

Five online databases, U.S. National Center for Biotechnology Information (NCBI) (RRID: SCR_006472), Ensembl Variation (RRID: SCR_001630) database, the Research Collaboratory for Structural Bioinformatics Protein Data Bank (RCSB PDB) (RRID: SCR_012820), Online Mendelian Inheritance in Man (OMIM) (RRID: SCR_006437), and The Universal Protein Resource (UniProt) (RRID: SCR_002380), were systematically searched to identify the sequence variations of the human *SULT2B1* gene. Human *SULT2B1* SNPs were identified and categorized based on their location in the gene into coding SNPs (synonymous, nonsense, and non-synonymous (missense)), non-coding SNPs (introns, 5’-untranslated region (5’UTR), and 3’-untranslated region (3’UTR)).

### Preparation of cDNAs encoding different human SULT2B1a allozymes, expression, and purification of recombinant SULT2B1a allozymes

Polymerase chain reaction (PCR) was performed to generate cDNAs encoding SULT2B1a allozymes using specific sense and antisense mutagenic primers and PrimeSTAR^®^ Max DNA polymerase. The PCR-amplified wt-SULT2B1a cDNA packaged in pGEX-4 T-2 prokaryotic expression vector was used as a template. PCR amplification conditions consisted of 12 cycles of 30 s at 94 °C, 1 min at 55 °C, and 7 min at 72 °C. The final PCR reaction mixtures were incubated for one h at 37 °C with *Dpn* I to digest the wild-type template. Upon digestion of the wild-type template, pGEX-4 T-2 plasmids containing mutated SULT2B1a cDNAs were transformed into NEB 5-alpha *E. coli* competent cells and then extracted from the cells. The individual mutated SULT2B1a cDNA, verified the nucleotide sequences^18^, was then transformed into competent BL21 *E. coli* cells. The transformed cells were grown in LB medium containing 100 μg/mL ampicillin, and 0.1 mM Isopropyl-β-D-thiogalactopyranoside (IPTG) was supplied at OD600 nm ∼0.5. After overnight incubation at 25 °C, cells were collected and resuspended in 20 mL ice-cold lysis buffer (10 mM Tris–HCl, pH 8.0, 150 mm NaCl, and 1 mM EDTA) and homogenized using an Aminco French press. The crude homogenates were centrifuged at 10,000 × *g* for 20 min at 4 °C, and the supernatants were individually fractionated with glutathione sepharose resin. The resin, washed with lysis buffer, was treated with thrombin digestion buffer (50 mM Tris–HCl, pH 8.0, 150 mM NaCl, and 2.5 mM CaCl_2_) containing 3.5 unit/ml bovine thrombin for 15 min at 25 °C. Upon centrifugation, the supernatant containing the purified recombinant SULT2B1a allozyme was collected and analyzed by 12% sodium dodecyl sulfate (SDS)–polyacrylamide gel electrophoresis (PAGE) in order to assess the purity of purified recombinant SULT2B1a allozyme. Protein concentrations of purified enzymes were determined by the Bradford protein assay method with bovine serum albumin as a standard^19^.

### Enzymatic assay for the sulfating activity of the purified recombinant SULT2B1a allozymes

The sulfating activity of the purified recombinant SULT2B1a allozymes towards pregnenolone was performed with [^35^S]-labeled PAPS. The standard assay mixture, with a final volume of 20 µL involving 50 mM HEPES buffer at pH 7.4, 1 mM DTT, the sulfonate donor (14 µM [^35^S]PAPS (15 Ci/mmol)), and the substrate (pregnenolone) or DMSO for the control. The reaction was initiated by adding 0.8 µg of the enzyme, allowed to proceed for 10 min at 37 °C, and terminated by heating at 98 °C for 3 min. Precipitates, formed in the heated reaction mixture, were cleared by centrifugation at 13,000 × g for 3 min. Then, 1 μl of the cleared reaction mixture was spotted on a thin-layer chromatography (TLC) plate, and the plate was subjected to TLC analysis using a solvent system containing *n*-butanol/isopropanol/formic acid/water in a ratio of 3:1:1:1 (by volume). Subsequently, the TLC plate was air-dried and autoradiographed using an X-ray film. The autoradiograph taken from the TLC plate was used to locate the radioactive spot corresponding to the sulfated product, and the located spot was cut from the TLC plate, eluted in 0.5 mL water, and mixed thoroughly with 2 ml of Ecolume scintillation liquid. The radioactivity was measured using a liquid scintillation counter. The results obtained were used to calculate the specific activity in the unit of sulfated product formed/min/mg of enzyme.

### Kinetic studies and statistical analysis

The SULT assay was performed using different concentrations of pregnenolone (0.05, 0.1, 0,15, 0.2, 0.3, 0.6, 1, 5, 10 µM) at pH 7.4 to determine the kinetic parameters for the wild-type SULT2B1a according to the procedure described above. Data obtained were analyzed based on Michaelis–Menten kinetics using GraphPad Prism^®^ (RRID: SCR_002798) v 6.0 software to calculate the *K*_m_ and *V*_max_ using non-linear regression curve.

## Results

### Identification and categorization of SNPs for the human SULT2B1 gene

Nucleotide sequence variations of human *SULT2B1* were analyzed by several online databases, including the NCBI, Ensembl Variation, and UniProt. Those sequence analyses detected a total of 20,459 *SULT2B1* SNPs, which were then categorized based on their location within the gene as being either coding (synonymous, nonsense, and non-synonymous [missense]) or non-coding (intronic or located in the 5’-UTR or 3’-UTR). In addition to the aforementioned database search, 16 *SULT2B1* SNPs were reported to be related to diseases by PubMed search (cf. Table [1](#Tab1)). Of the identified SNPs, 326 were found to be missense cSNPs that result in amino acid changes in the protein product. These amino acid changes were further evaluated in terms of their locations relative to major enzyme features: the substrate binding site, the phosphosulfate binding (PSB) region, and PAPS-binding regions of SULT2B1, and the dimerization motif observed in other general SULT enzymes (Fig. [1](#Fig1))^14,20,21^. In this study, considering the chemical change of the mutated residues, 13 of the 326 missense SNPs were selected for further enzymatic investigation. Table [2](#Tab2) summarizes the selected amino acid changes, their locations, and the sense and antisense mutagenic primers designed for PCR amplification of corresponding *SULT2B1a* cDNAs.

### Table 1.

| SNP ID | Location | Effect |
| --- | --- | --- |
| rs4149455 | Non coding region (intron variant) | Associated with reduced risks of esophageal squamous22 |
| rs1052131 | Coding region (synonymous codon) |  |
| rs12460535 | Non coding region (intron variant) | Associated with a reduced risk of prostate cancer progression23 |
| rs2665582 | Non coding region (intron variant) |  |
| rs10426628 | Non coding region (5’-UTR) |  |
| rs3786749 | Intron variant | Associated with colorectal cancer progression24 |
| rs16982149 | Coding region (missense mutation) | Cause changes in sulfating activity25 |
| rs16982158 |  |  |
| rs16982169 |  |  |
| rs17842463 |  |  |
| rs3760806 | Non coding region (5’-UTR) | Associated with an increased risk of colorectal cancer26 |
| rs3760808 | Non coding region (intron variant) | Associated with an increased risk of prostate cancer27 |
| rs10424237 |  |  |
| rs762765702 | Coding region (missense mutation) | Cause autosomal-recessive congenital ichthyosis in humans28 |
| rs1114167424 |  |  |
| rs1114167426 | Non coding region (splice donor) |  |

Table 1 Caption: SULT2B1 SNPs that have been reported in scientific literature.

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/341f/10997614/8d23cab22e51/41598_2024_56303_Fig1_HTML.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=10997614_41598_2024_56303_Fig1_HTML.jpg)

The amino acid sequence of the human SULT2B1a marked with residues/sequences reported to be involved in PAPS-binding, substrate-binding, dimerization-motif, and/or catalysis. The residues in yellow are those that are involved in PAPS-binding. Residues in green refer to those involved in the dimerization-motif. Residues in red are those involved in substrate-binding. Residues in purple refer to the substituting amino acids coded by the selected missense cSNPs. The figure was generated using protter tool29, a web tool for interactive protein feature visualization. SULT2B1 Sulfotransferase family cytosolic 2B member 1, PAPS 3’-phosphoadenosine-5’-phosphosulfate, cSNP coding single nucleotide polymorphism.

### Table 2.

| SNP code no | hSULT2B1a allozymes | Mutagenic primer nucleotides sequence |
| --- | --- | --- |
| – | Asp46Asn | 5’-ACCCAAGATGTGCGGGACAACGACATCTTTATCATC-3’ 5’-GATGATAAAGATGTCGTTGTCCCGCACATCTTGGGT-3’ |
| rs777924668 | Pro54Ala | 5’-ATCTTTATCATCACCTACGCCAAGTCAGGCACGACC-3’ 5’-GGTCGTGCCTGACTTGGCGTAGGTGATGATAAAGAT-3’ |
| rs746398875 | Gly57Val | 5’-ATCACCTACCCCAAGTCAGTCACGACCTGGATGATC-3’ 5’-GATCATCCAGGTCGTGACTGACTTGGGGTAGGTGAT-3’ |
| rs527454384 | Thr58Met | 5’-ACCTACCCCAAGTCAGGCATGACCTGGATGATCGAG-3’ 5’-CTCGATCATCCAGGTCATGCCTGACTTGGGGTAGGT-3’ |
| rs777140014 | Arg132His | 5’-AAGGTGATCTACATGGGCCACAACCCCCGGGACGTT-3’ 5’-AACGTCCCGGGGGTTGTGGCCCATGTAGATCACCTT-3’ |
| rs1114167424 | Pro134Leu | 5’-TCTACATGGGCCGCAACCTCCGGGACGTTGTGGTCT-3’ 5’-AGACCACAACGTCCCGGAGGTTGCGGCCCATGTAGA-3’ |
| rs16982158 | Asp176Asn | 5’-CAGTTTGGCTCCTGGTTCAACCACATTAAGGGCTGG-3’ 5’-CCAGCCCTTAATGTGGTTGAACCAGGAGCCAAACTG-3’ |
| rs16982169 | Arg215His | 5’-ATCTGTGGGTTCCTGGGCCATCCGCTGGGCAAGGAG-3’ 5’-CTCCTTGCCCAGCGGATGGCCCAGGAACCCACAGAT-3’ |
| rs765224593 | Ser229Thr | 5’-GGCTCCGTCGTGGCACACACAACCTTCAGCGCCATG-3’ 5’-CATGGCGCTGAAGGTTGTGTGTGCCACGACGGAGCC-3’ |
| rs762765702 | Arg259Gln | 5’-CGTCGCGGGGCCTTCCTCCAGAAAGGGGTCTGCGGC-3’ 5’-GCCGCAGACCCCTTTCTGGAGGAAGGCCCCGCGACG-3’ |
| rs774212320 | Gly261Val | 5’-GGGGCCTTCCTCCGGAAAGTGGTCTGCGGCGACTGG-3’ 5’-CCAGTCGCCGCAGACCACTTTCCGGAGGAAGGCCCC-3’ |
| – | Gly261Trp | 5’-GGGGCCTTCCTCCGGAAATGGGTCTGCGGCGACTGG-3’ 5’-CCAGTCGCCGCAGACCCATTTCCGGAGGAAGGCCCC-3’ |
| rs17842463 | Pro330Leu | 5’-CTGGAGCGTGAGCCCAGACTCAACTCCAGCCCCAGC-3’ 5’-GCTGGGGCTGGAGTTGAGTCTGGGCTCACGCTCCAG-3’ |

Table 2 Caption: List of the selected SULT2B1a cSNPs, sense and antisense mutagenic primers sets designed for the site-directed mutagenesis of the corresponding cDNAs encoding human SULT2B1a allozymes.

### Expression and purification of recombinant human SULT2B1a allozymes

cDNAs encoding the 13 selected SULT2B1a allozymes were packaged in the pGEX-4 T-2 as described in the “[Methods](#Sec4)” section. The recombinant allozymes prepared were analyzed by SDS-PAGE and confirmed to be homogeneous and the high purity (cf. Fig. [2](#Fig2)). For all 13 allozymes, the apparent molecular weights were comparable to the predicted molecular weight (~ 39.6 kD) of wild-type SULT2B1a (cf. lane 1 in Fig. [2](#Fig2)).

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/341f/10997614/2da31e2fc74f/41598_2024_56303_Fig2_HTML.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=10997614_41598_2024_56303_Fig2_HTML.jpg)

SDS gel electrophoretic pattern of purified wt-SULT2B1a and SULT2B1a allozymes. SDS-PAGE was performed on a 12% gel, followed by Coomassie blue staining. Lane * indicates the migrating positions of protein molecular weight markers co-electrophoresed. Samples analyzed in lanes 1 through 14 correspond to SULT2B1a-wt, P45A, T58M, P134L, R259Q, R132H, D176N, R215H, P330L, S229T, G57V, G261V, G261W, and D46N allozymes. wt wild-type, SULT2B1 Sulfotransferase family cytosolic 2B member 1.

### Kinetic study of pregnenolone sulfation by wild-type SULT2B1a

The sulfating activity of wild-type SULT2B1a was evaluated using different concentrations of the substrate pregnenolone**,** as mentioned in the “[Methods](#Sec4)” section. Sulfation of pregnenolone by SULT2B1a appeared to follow the Michaelis–Menten kinetics up to a substrate concentration of 5 µM. *K*_m_ and *V*_max_ values determined were 0.50 ± 0.03 µM and 0.36 ± 0.01 nmol/min/mg, respectively (cf. Fig. [3](#Fig3)).

### Figure 3.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/341f/10997614/ff575b6fd344/41598_2024_56303_Fig3_HTML.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=10997614_41598_2024_56303_Fig3_HTML.jpg)

Concentration-dependent sulfation of pregnenolone by wild-type SULT2B1a. Data shown represent calculated mean ± standard deviation derived from three experiments. wt wild-type, SULT2B1a Sulfotransferase family cytosolic 2B member 1a, PREG pregnenolone.

### Sulfating activities of SULT2B1a allozymes with pregnenolone

The sulfating activities of purified SULT2B1a allozymes toward pregnenolone were experimentally determined and compared to the wild-type enzyme. These assays utilized three different pregnenolone concentrations based on the results from the kinetic study of wild-type SULT2B1a: one concentration below the *K*_m_*,* one close to the *K*_m_, and one above the *K*_m_. Figure [4](#Fig4) depicts the results obtained from these comparative studies. Across all three substrate concentrations (0.1, 0.5, and 2.5 µM), the relative pregnenolone-sulfating activities of the 13 allozymes tested were generally consistent. Seven allozymes exhibited barely detectable sulfating activity at across all three concentrations: P54A, G57V, T58M, R132H, R259Q, G261V, and G261W. Of the remaining five allozymes, three (P134L, D176N, R215H) showed lower activity at 0.1 µM and similar activity at 2.5 µM, implicating that those three substitutions may weaken the affinity with pregnenolone. S229T and P330L showed slightly higher activity than the wild-type, implicating that two substitutions may increase the affinity with pregnenolone. The last allozyme, D46N, exhibited the sulfating activity comparable to the wild-type at 0.5 µM, but the activity at 2.5 µM was reduced to one-tenth of its activity at 0.5 µM. D46N substitution may alter the Michaelis–Menten kinetics of wild-type to the substrate inhibition kinetics.

### Figure 4.

![Figure 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/341f/10997614/4f457cd15edd/41598_2024_56303_Fig4_HTML.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=10997614_41598_2024_56303_Fig4_HTML.jpg)

Specific activities of human SULT2B1a allozymes for the sulfation of pregnenolone at 0.1 µM (A), 0.5 µM (B), and 2.5 µM (C) concentrations. Data shown represent means ± standard deviations derived from three determinations. One-way ANOVA was performed in combination with Dunnett’s multiple comparison test. SULT2B1 Sulfotransferase family cytosolic 2B member 1, PREG pregnenolone.

## Discussion

The endogenous steroid pregnenolone is the precursor intermediate from which most other steroid hormones are derived. Sulfation of hydroxysteroids is predominantly carried out by SULT2 members, and SULT2B1a has been shown to favor pregnenolone as a substrate compared to other members of the *SULT2* family^30–32^. The current study represents the first systematic investigation of the pregnenolone sulfating activity of SULT2B1a allozymes, including their substrate affinity compared to the wild-type enzyme. A systematic database search identified missense cSNPs that occur in the human *SULT2B1a* gene. A total of 13 missense cSNPs were selected to evaluate their sulfating activities toward pregnenolone. The assays with three concentrations of pregnenolone, selected with reference to the *K*_m_ of the wild-type enzyme, clearly indicated the differential sulfating activities and affinities of allozymes with pregnenolone.

Several studies in the literature provide information regarding what parts of SULT2B1a are essential for its sulfating activity^32,33^. Mutational analysis of human SULT2B1a has revealed that removing either 53 amino acids from the long carboxyl-terminal end or eight from the amino-terminal end does not affect its pregnenolone sulfotransferase activity^32^. While no crystal structure is available for SULT2B1a specifically, one has been reported for the alternative splice variant, SULT2B1b^20^. Regarding the enzyme’s internal structure, the PSB loop is composed of the conserved amino acid residues ^52^TYPKSGT^58^, of which the residues Lys^55^, Ser^56^, Gly^57^, and Thr^58^ are involved in binding the 5’-phosphate of PAP/PAPS. Meanwhile, the residues Arg^132^, Ser^140^, Arg^259^, Lys^260^, and Gly^261^ are involved in binding its 3’-phosphate. The adenine group of PAP also interacts with the amino acid residues Trp^75^, Tyr^195^, Ser^229^, and Phe^246^, while the catalytic activity of the enzyme is due to His^110^^20^. Notably, consistent with expectations regarding impacts on enzyme activity, the majority of the SULT2B1a allozymes examined in this study involve amino acid changes either within or close to these important structural elements.

In the cases of allozymes with little or no detectable sulfating activity, six directly involve the abovementioned structural residues (G57V, T58M, R132H, R259Q, G261V, and G261W), while the other two involve nearby residues (D46N and P54A). Substitution of these critical amino acids may have weakened the interaction between SULT2B1a and its essential cofactor PAPS, and consequently abolished or dramatically decreased allozyme activity toward pregnenolone. Meanwhile, the other five tested allozymes (P134L, D176N, R215H, S229T, and P330L) exhibited sulfating activities comparable to the wild-type at 2.5 µM. These included two substitutions impacting key structural residues or nearby residues (P134L and S229T), and three involving residues relatively distant from known essential elements (D176N, R215H, and P330L). For P134L, it is probable that the substitution of this proline (a turn-inducing residue) with leucine (a non-turn-inducing residue) affected the conformation of the protein backbone, thus impairing (but not preventing) hydrogen bonding between the NH_2_ of nearby Arg^132^ and the oxygen atom of the 3’-phosphate in PAPS^20,28^. By contrast, despite directly impacting an essential residue, the substitution S229T resulted in an allozyme with 96% of the activity of wild-type. This outcome is consistent with the chemical similarity of serine and the substituted threonine (both being polar and hydrophilic amino acids).

Meanwhile, the allozyme with substitution D176N did not replace a known key residue, yet exhibited reduced sulfating activity. The original aspartic acid (D) and the substituted asparagine (N) have different chemical properties, which likely relates to the reduced activity and affinity of this allozyme. Distinct from any other tested allozyme, the R215H substitution yielded sulfating activity slightly lower than the wild-type at 0.1 µM and no effect at higher two concentrations. In this case, as the original and substituted amino acids featured similar chemical properties (both being positively charged), the substitution may weakly affect the affinity but not the maximum velocity. Finally, the P330L substitution resulted in an allozyme with activity 30% lower than that of the wild-type at 2.5 µM. This allozyme replaced a turn-inducing proline with non-turn-inducing leucine; as proline residues are important for the conformation of the protein backbone, it would be reasonable to expect this polymorphism to greatly impact activity. However, previous mutational analysis of SULT2B1a showed the removal of 53 amino acids from the long carboxyl-terminal end, including P330, to have no effect on its pregnenolone sulfotransferase activity^32^, implying relatively less structural import for this residue. These observations are consistent with the activity data of SULT2B1b allozymes toward pregnenolone previously reported^16^, suggesting that single nucleotide polymorphisms on the common region of the *SULT2B1* gene may have a similar effect on the sulfating activity of the two splice variants of SULT2B1 toward pregnenolone.

It should be noted that two non-synonymous coding cSNPs of the SULT2B1b variant in the *SULT2B1* gene, c.446C > T (P149L) and c.821G > A (R274Q), have been previously reported to be involved in autosomal recessive ichthyosis (ARCI)^28^. Impairment of cholesterol metabolism, including accumulation of cholesterol or cholesterol sulfate, has been observed in patients with ARCI^28,34^. The SULT2B1b variant, responsible for cholesterol sulfation, has been recognized to be involved in ARCI. The two cSNPs mentioned above may also generate two SULT2B1a allozymes, P134L and R259Q. In our study, SULT2B1a wild-type and all selected allozymes have been analyzed for sulfating activity toward cholesterol. As expected, all have exhibited no detectable sulfating activity toward cholesterol while both P134L and R259Q allozyme showed reduced pregnenolone-sulfating activity. Within the differentiating sequence, residues I20 and I23 are primarily responsible for SULT2B1b’s high cholesterol sulfating activity^32^. In particular, residue I20 forms positive Van der Waal’s interactions with the long chain C17 cholesterol atom; such interactions are negligible in the binding of pregnenolone^20^. Therefore, secondary structure in the amino terminus is responsible for determining the substrate selectivity of SULT2B1 isoforms (*a* and *b*). Therefore, ARCI patients with the aforementioned two cSNPs may result in the loss of SULT2B1a function, as well as SULT2B1b, implying that less sulfation of pregnenolone and DHEA possibly impair the neurosteroid physiology.

## Conclusions

This study is the first to provide information concerning the effects of *SULT2B1* cSNPs on the pregnenolone-sulfating activity of corresponding SULT2B1a allozymes. Of the 13 tested allozymes, eight exhibited minimal to no detectable sulfating activity (D46N, P54A, G57V, T58M, R132H, R259Q, G261V, and G261W), while the other five exhibited differential sulfating activity in comparison with the wild-type (P134L, D176N, R215H, S229T, and P330L). Thus, this research clearly demonstrates a significant effect of some *SULT2B1* genetic polymorphisms on corresponding allozyme activity, and supports the possibility that individuals harboring different *SULT2B1a* genotypes may have differential capacity in sulfating pregnenolone. Additional research may help in predicting the risk of some *SULT2B1a*-associated diseases, as well as assist the design of personalized regimens of relevant drugs for individuals with different *SULT2B1a* genotypes.

## Author contributions

E.A wrote the main manuscript text and prepared figures and tables. All authors reviewed the manuscript

## Funding

This work was supported in part by a grant from the National Institutes of Health [Grant #R03HD071146].

## Data availability

The datasets generated during and/or analyzed during the current study are available in the Figshare repository, DOI: [https://figshare.com/s/44e0183a2db3dcff627f](https://figshare.com/s/44e0183a2db3dcff627f). Data are available under the terms of the Creative Commons Zero “No rights reserved” data waiver (CC0 1.0 Public domain dedication).

## Competing interests

The authors declare no competing interests.

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Data Availability Statement

The datasets generated during and/or analyzed during the current study are available in the Figshare repository, DOI: [https://figshare.com/s/44e0183a2db3dcff627f](https://figshare.com/s/44e0183a2db3dcff627f). Data are available under the terms of the Creative Commons Zero “No rights reserved” data waiver (CC0 1.0 Public domain dedication).

### Data Availability Statement

The datasets generated during and/or analyzed during the current study are available in the Figshare repository, DOI: [https://figshare.com/s/44e0183a2db3dcff627f](https://figshare.com/s/44e0183a2db3dcff627f). Data are available under the terms of the Creative Commons Zero “No rights reserved” data waiver (CC0 1.0 Public domain dedication).

## References

1. Thomas, L. L. et al.Foye’s Principles of Medicinal Chemistry (Lippincott Williams & Wilkins P. S., Wolters Kluwer, 2008).  [Google Scholar](https://scholar.google.com/scholar_lookup?Thomas,%20L.%20L.%20et%20al.Foye%E2%80%99s%20Principles%20of%20Medicinal%20Chemistry%20(Lippincott%20Williams%20&%20Wilkins%20P.%20S.,%20Wolters%20Kluwer,%202008).)

2. Henderson, E., Weinberg, M. & Wright, W. A. Pregnenolone. J. Clin. Endocrinol.10(4), 455–474 (1950).  [DOI](https://doi.org/10.1210/jcem-10-4-455) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15415436/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Henderson,%20E.,%20Weinberg,%20M.%20&%20Wright,%20W.%20A.%20Pregnenolone.%20J.%20Clin.%20Endocrinol.10(4),%20455%E2%80%93474%20(1950).)

3. Veiga, S., Garcia-Segura, L. M. & Azcoitia, I. Neuroprotection by the steroids pregnenolone and dehydroepiandrosterone is mediated by the enzyme aromatase. J. Neurobiol.56(4), 398–406 (2003).  [DOI](https://doi.org/10.1002/neu.10249) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12918023/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Veiga,%20S.,%20Garcia-Segura,%20L.%20M.%20&%20Azcoitia,%20I.%20Neuroprotection%20by%20the%20steroids%20pregnenolone%20and%20dehydroepiandrosterone%20is%20mediated%20by%20the%20enzyme%20aromatase.%20J.%20Neurobiol.56(4),%20398%E2%80%93406%20(2003).)

4. Vallée, M., Mayo, W. & Le Moal, M. Role of pregnenolone, dehydroepiandrosterone and their sulfate esters on learning and memory in cognitive aging. Brain Res. Rev.37(1–3), 301–312 (2001).  [DOI](https://doi.org/10.1016/s0165-0173(01)00135-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11744095/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Vall%C3%A9e,%20M.,%20Mayo,%20W.%20&%20Le%20Moal,%20M.%20Role%20of%20pregnenolone,%20dehydroepiandrosterone%20and%20their%20sulfate%20esters%20on%20learning%20and%20memory%20in%20cognitive%20aging.%20Brain%20Res.%20Rev.37(1%E2%80%933),%20301%E2%80%93312%20(2001).)

5. Reddy, D. S. Neurosteroids: Endogenous role in the human brain and therapeutic potentials. In Progress in Brain Research 113–137 (Elsevier, 2010).  [DOI](https://doi.org/10.1016/B978-0-444-53630-3.00008-7) | [PMC free article](/articles/PMC3139029/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21094889/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Reddy,%20D.%20S.%20Neurosteroids:%20Endogenous%20role%20in%20the%20human%20brain%20and%20therapeutic%20potentials.%20In%20Progress%20in%20Brain%20Research%20113%E2%80%93137%20(Elsevier,%202010).)

6. Nishikawa, T. & Strott, C. A. Unconjugated and sulfoconjugated steroids in plasma and zones of the adrenal cortex of the guinea pig. Steroids41(1), 105–119 (1983).  [DOI](https://doi.org/10.1016/0039-128x(83)90021-1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/6581618/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Nishikawa,%20T.%20&%20Strott,%20C.%20A.%20Unconjugated%20and%20sulfoconjugated%20steroids%20in%20plasma%20and%20zones%20of%20the%20adrenal%20cortex%20of%20the%20guinea%20pig.%20Steroids41(1),%20105%E2%80%93119%20(1983).)

7. Mueller, J. W. et al. The regulation of steroid action by sulfation and desulfation. Endocr. Rev.36(5), 526–563 (2015).  [DOI](https://doi.org/10.1210/er.2015-1036) | [PMC free article](/articles/PMC4591525/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26213785/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Mueller,%20J.%20W.%20et%20al.%20The%20regulation%20of%20steroid%20action%20by%20sulfation%20and%20desulfation.%20Endocr.%20Rev.36(5),%20526%E2%80%93563%20(2015).)

8. Tsutsui, K. & Haraguchi, S. Pregnenolone sulfate. In Handbook of Hormones (eds Tsutsui, K. & Haraguchi, S.) 540-e96A−2 (Elsevier, 2016).  [Google Scholar](https://scholar.google.com/scholar_lookup?Tsutsui,%20K.%20&%20Haraguchi,%20S.%20Pregnenolone%20sulfate.%20In%20Handbook%20of%20Hormones%20(eds%20Tsutsui,%20K.%20&%20Haraguchi,%20S.)%20540-e96A%E2%88%922%20(Elsevier,%202016).)

9. Coughtrie, M. W. et al. Sulfation of endogenous compounds and xenobiotics—interactions and function in health and disease. Chemico-Biol. Interact.92(1–3), 247–256 (1994).  [DOI](https://doi.org/10.1016/0009-2797(94)90067-1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8033257/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Coughtrie,%20M.%20W.%20et%20al.%20Sulfation%20of%20endogenous%20compounds%20and%20xenobiotics%E2%80%94interactions%20and%20function%20in%20health%20and%20disease.%20Chemico-Biol.%20Interact.92(1%E2%80%933),%20247%E2%80%93256%20(1994).)

10. Li, H. et al. The isolation and characterization of cDNA encoding the mouse bifunctional ATP sulfurylase-adenosine 5′-phosphosulfate kinase. J. Biol. Chem.270(49), 29453–29459 (1995).  [DOI](https://doi.org/10.1074/jbc.270.49.29453) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7493984/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Li,%20H.%20et%20al.%20The%20isolation%20and%20characterization%20of%20cDNA%20encoding%20the%20mouse%20bifunctional%20ATP%20sulfurylase-adenosine%205%E2%80%B2-phosphosulfate%20kinase.%20J.%20Biol.%20Chem.270(49),%2029453%E2%80%9329459%20(1995).)

11. Alherz, F. A. et al. On the role of genetic polymorphisms in the sulfation of cholesterol by human cytosolic sulphotransferase SULT2B1b. J. Biochem.164(3), 215–221 (2018).  [DOI](https://doi.org/10.1093/jb/mvy042) | [PMC free article](/articles/PMC6094445/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29701841/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Alherz,%20F.%20A.%20et%20al.%20On%20the%20role%20of%20genetic%20polymorphisms%20in%20the%20sulfation%20of%20cholesterol%20by%20human%20cytosolic%20sulphotransferase%20SULT2B1b.%20J.%20Biochem.164(3),%20215%E2%80%93221%20(2018).)

12. Her, C. et al. Human hydroxysteroid sulfotransferase SULT2B1: Two enzymes encoded by a single chromosome 19 gene. Genomics53(3), 284–295 (1998).  [DOI](https://doi.org/10.1006/geno.1998.5518) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9799594/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Her,%20C.%20et%20al.%20Human%20hydroxysteroid%20sulfotransferase%20SULT2B1:%20Two%20enzymes%20encoded%20by%20a%20single%20chromosome%2019%20gene.%20Genomics53(3),%20284%E2%80%93295%20(1998).)

13. Lee, K. A. et al. Crystal Structure of Human Cholesterol Sulfotransferase (SULT2B1b) in the Presence of Pregnenolone and 3′-Phosphoadenosine 5′-Phosphate RATIONALE FOR SPECIFICITY DIFFERENCES BETWEEN PROTOTYPICAL SULT2A1 AND THE SULT2B1 ISOFORMS. J. Biol. Chem.278(45), 44593–44599 (2003).  [DOI](https://doi.org/10.1074/jbc.M308312200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12923182/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Lee,%20K.%20A.%20et%20al.%20Crystal%20Structure%20of%20Human%20Cholesterol%20Sulfotransferase%20(SULT2B1b)%20in%20the%20Presence%20of%20Pregnenolone%20and%203%E2%80%B2-Phosphoadenosine%205%E2%80%B2-Phosphate%20RATIONALE%20FOR%20SPECIFICITY%20DIFFERENCES%20BETWEEN%20PROTOTYPICAL%20SULT2A1%20AND%20THE%20SULT2B1%20ISOFORMS.%20J.%20Biol.%20Chem.278(45),%2044593%E2%80%9344599%20(2003).)

14. Falany, C. N. & Rohn-Glowacki, K. J. SULT2B1: unique properties and characteristics of a hydroxysteroid sulfotransferase family. Drug Metabol. Rev.45(4), 388–400 (2013).  [DOI](https://doi.org/10.3109/03602532.2013.835609) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24020383/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Falany,%20C.%20N.%20&%20Rohn-Glowacki,%20K.%20J.%20SULT2B1:%20unique%20properties%20and%20characteristics%20of%20a%20hydroxysteroid%20sulfotransferase%20family.%20Drug%20Metabol.%20Rev.45(4),%20388%E2%80%93400%20(2013).)

15. Huang, J. et al. Kinetic analysis of bile acid sulfation by stably expressed human sulfotransferase 2A1 (SULT2A1). Xenobiotica40(3), 184–194 (2010).  [DOI](https://doi.org/10.3109/00498250903514607) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20102295/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Huang,%20J.%20et%20al.%20Kinetic%20analysis%20of%20bile%20acid%20sulfation%20by%20stably%20expressed%20human%20sulfotransferase%202A1%20(SULT2A1).%20Xenobiotica40(3),%20184%E2%80%93194%20(2010).)

16. Alherz, F. A. et al. Effect of SULT2B1 genetic polymorphisms on the sulfation of dehydroepiandrosterone and pregnenolone by SULT2B1b allozymes. Mol. Cell. Endocrinol.496, 110535 (2019).  [DOI](https://doi.org/10.1016/j.mce.2019.110535) | [PMC free article](/articles/PMC6733586/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31400397/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Alherz,%20F.%20A.%20et%20al.%20Effect%20of%20SULT2B1%20genetic%20polymorphisms%20on%20the%20sulfation%20of%20dehydroepiandrosterone%20and%20pregnenolone%20by%20SULT2B1b%20allozymes.%20Mol.%20Cell.%20Endocrinol.496,%20110535%20(2019).)

17. Yanagisawa, K. et al. cDNA cloning, expression, and characterization of the human bifunctional ATP sulfurylase/adenosine 5′-phosphosulfate kinase enzyme. Biosci. Biotechnol. Biochem.62(5), 1037–1040 (1998).  [DOI](https://doi.org/10.1271/bbb.62.1037) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9648242/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Yanagisawa,%20K.%20et%20al.%20cDNA%20cloning,%20expression,%20and%20characterization%20of%20the%20human%20bifunctional%20ATP%20sulfurylase/adenosine%205%E2%80%B2-phosphosulfate%20kinase%20enzyme.%20Biosci.%20Biotechnol.%20Biochem.62(5),%201037%E2%80%931040%20(1998).)

18. Sanger, F., Nicklen, S. & Coulson, A. R. DNA sequencing with chain-terminating inhibitors. Proc. Natl. Acad. Sci.74(12), 5463–5467 (1977).  [DOI](https://doi.org/10.1073/pnas.74.12.5463) | [PMC free article](/articles/PMC431765/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/271968/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Sanger,%20F.,%20Nicklen,%20S.%20&%20Coulson,%20A.%20R.%20DNA%20sequencing%20with%20chain-terminating%20inhibitors.%20Proc.%20Natl.%20Acad.%20Sci.74(12),%205463%E2%80%935467%20(1977).)

19. Bradford, M. M. A rapid and sensitive method for the quantitation of microgram quantities of protein utilizing the principle of protein-dye binding. Anal. Biochem.72(1–2), 248–254 (1976).  [DOI](https://doi.org/10.1016/0003-2697(76)90527-3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/942051/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Bradford,%20M.%20M.%20A%20rapid%20and%20sensitive%20method%20for%20the%20quantitation%20of%20microgram%20quantities%20of%20protein%20utilizing%20the%20principle%20of%20protein-dye%20binding.%20Anal.%20Biochem.72(1%E2%80%932),%20248%E2%80%93254%20(1976).)

20. Lee, K. A. et al. Crystal structure of human cholesterol sulfotransferase (SULT2B1b) in the presence of pregnenolone and 3’-phosphoadenosine 5’-phosphate. Rationale for specificity differences between prototypical SULT2A1 and the SULT2BG1 isoforms. J. Biol. Chem.278(45), 44593–9 (2003).  [DOI](https://doi.org/10.1074/jbc.M308312200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12923182/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Lee,%20K.%20A.%20et%20al.%20Crystal%20structure%20of%20human%20cholesterol%20sulfotransferase%20(SULT2B1b)%20in%20the%20presence%20of%20pregnenolone%20and%203%E2%80%99-phosphoadenosine%205%E2%80%99-phosphate.%20Rationale%20for%20specificity%20differences%20between%20prototypical%20SULT2A1%20and%20the%20SULT2BG1%20isoforms.%20J.%20Biol.%20Chem.278(45),%2044593%E2%80%939%20(2003).)

21. Petrotchenko, E. V. et al. The dimerization motif of cytosolic sulfotransferases. FEBS Lett.490(1–2), 39–43 (2001).  [DOI](https://doi.org/10.1016/s0014-5793(01)02129-9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11172807/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Petrotchenko,%20E.%20V.%20et%20al.%20The%20dimerization%20motif%20of%20cytosolic%20sulfotransferases.%20FEBS%20Lett.490(1%E2%80%932),%2039%E2%80%9343%20(2001).)

22. Hyland, P. L. et al. Genetic variants in sex hormone metabolic pathway genes and risk of esophageal squamous cell carcinoma. Carcinogenesis34(5), 1062–1068 (2013).  [DOI](https://doi.org/10.1093/carcin/bgt030) | [PMC free article](/articles/PMC3643422/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23358850/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Hyland,%20P.%20L.%20et%20al.%20Genetic%20variants%20in%20sex%20hormone%20metabolic%20pathway%20genes%20and%20risk%20of%20esophageal%20squamous%20cell%20carcinoma.%20Carcinogenesis34(5),%201062%E2%80%931068%20(2013).)

23. Levesque, E. et al. Steroidogenic germline polymorphism predictors of prostate cancer progression in the estradiol pathway. Clin. Cancer Res.20(11), 2971–2983 (2014).  [DOI](https://doi.org/10.1158/1078-0432.CCR-13-2567) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24682418/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Levesque,%20E.%20et%20al.%20Steroidogenic%20germline%20polymorphism%20predictors%20of%20prostate%20cancer%20progression%20in%20the%20estradiol%20pathway.%20Clin.%20Cancer%20Res.20(11),%202971%E2%80%932983%20(2014).)

24. Liang, Y. et al. Genetic variations affecting serum carcinoembryonic antigen levels and status of regional lymph nodes in patients with sporadic colorectal cancer from Southern China. PLoS One9(6), e97923 (2014).  [DOI](https://doi.org/10.1371/journal.pone.0097923) | [PMC free article](/articles/PMC4062418/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24941225/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Liang,%20Y.%20et%20al.%20Genetic%20variations%20affecting%20serum%20carcinoembryonic%20antigen%20levels%20and%20status%20of%20regional%20lymph%20nodes%20in%20patients%20with%20sporadic%20colorectal%20cancer%20from%20Southern%20China.%20PLoS%20One9(6),%20e97923%20(2014).)

25. Ji, Y. et al. Human hydroxysteroid sulfotransferase SULT2B1 pharmacogenomics: Gene sequence variation and functional genomics. J. Pharmacol. Exp. Ther.322(2), 529–540 (2007).  [DOI](https://doi.org/10.1124/jpet.107.122895) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17496163/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Ji,%20Y.%20et%20al.%20Human%20hydroxysteroid%20sulfotransferase%20SULT2B1%20pharmacogenomics:%20Gene%20sequence%20variation%20and%20functional%20genomics.%20J.%20Pharmacol.%20Exp.%20Ther.322(2),%20529%E2%80%93540%20(2007).)

26. Li, S. et al. Association study of genetic variants in estrogen metabolic pathway genes and colorectal cancer risk and survival. Arch. Toxicol.92(6), 1991–1999 (2018).  [DOI](https://doi.org/10.1007/s00204-018-2195-y) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29766219/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Li,%20S.%20et%20al.%20Association%20study%20of%20genetic%20variants%20in%20estrogen%20metabolic%20pathway%20genes%20and%20colorectal%20cancer%20risk%20and%20survival.%20Arch.%20Toxicol.92(6),%201991%E2%80%931999%20(2018).)

27. Koutros, S. et al. Xenobiotic metabolizing gene variants, pesticide use, and risk of prostate cancer. Pharmacogenet. Genom.21(10), 615 (2011).  [DOI](https://doi.org/10.1097/FPC.0b013e3283493a57) | [PMC free article](/articles/PMC3172373/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21716162/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Koutros,%20S.%20et%20al.%20Xenobiotic%20metabolizing%20gene%20variants,%20pesticide%20use,%20and%20risk%20of%20prostate%20cancer.%20Pharmacogenet.%20Genom.21(10),%20615%20(2011).)

28. Heinz, L. et al. Mutations in SULT2B1 cause autosomal-recessive congenital ichthyosis in humans. Am. J. Hum. Genet.100(6), 926–939 (2017).  [DOI](https://doi.org/10.1016/j.ajhg.2017.05.007) | [PMC free article](/articles/PMC5473727/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28575648/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Heinz,%20L.%20et%20al.%20Mutations%20in%20SULT2B1%20cause%20autosomal-recessive%20congenital%20ichthyosis%20in%20humans.%20Am.%20J.%20Hum.%20Genet.100(6),%20926%E2%80%93939%20(2017).)

29. Omasits, U. et al. Protter: Interactive protein feature visualization and integration with experimental proteomic data. Bioinformatics30(6), 884–886 (2014).  [DOI](https://doi.org/10.1093/bioinformatics/btt607) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24162465/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Omasits,%20U.%20et%20al.%20Protter:%20Interactive%20protein%20feature%20visualization%20and%20integration%20with%20experimental%20proteomic%20data.%20Bioinformatics30(6),%20884%E2%80%93886%20(2014).)

30. Falany, C. N., Vazquez, M. E. & Kalb, J. Purification and characterization of human liver dehydroepiandrosterone sulphotransferase. Biochem. J.260(3), 641–646 (1989).  [DOI](https://doi.org/10.1042/bj2600641) | [PMC free article](/articles/PMC1138726/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2764897/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Falany,%20C.%20N.,%20Vazquez,%20M.%20E.%20&%20Kalb,%20J.%20Purification%20and%20characterization%20of%20human%20liver%20dehydroepiandrosterone%20sulphotransferase.%20Biochem.%20J.260(3),%20641%E2%80%93646%20(1989).)

31. Meloche, C. A. & Falany, C. N. Expression and characterization of the human 3β-hydroxysteroid sulfotransferases (SULT2B1a and SULT2B1b). J. Steroid Biochem. Mol. Biol.77(4–5), 261–269 (2001).  [DOI](https://doi.org/10.1016/s0960-0760(01)00064-4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11457664/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Meloche,%20C.%20A.%20&%20Falany,%20C.%20N.%20Expression%20and%20characterization%20of%20the%20human%203%CE%B2-hydroxysteroid%20sulfotransferases%20(SULT2B1a%20and%20SULT2B1b).%20J.%20Steroid%20Biochem.%20Mol.%20Biol.77(4%E2%80%935),%20261%E2%80%93269%20(2001).)

32. Fuda, H. et al. Mutational analysis of human hydroxysteroid sulfotransferase SULT2B1 isoforms reveals that exon 1B of the SULT2B1 gene produces cholesterol sulfotransferase, whereas exon 1A yields pregnenolone sulfotransferase. J. Biol. Chem.277(39), 36161–36166 (2002).  [DOI](https://doi.org/10.1074/jbc.M207165200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12145317/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Fuda,%20H.%20et%20al.%20Mutational%20analysis%20of%20human%20hydroxysteroid%20sulfotransferase%20SULT2B1%20isoforms%20reveals%20that%20exon%201B%20of%20the%20SULT2B1%20gene%20produces%20cholesterol%20sulfotransferase,%20whereas%20exon%201A%20yields%20pregnenolone%20sulfotransferase.%20J.%20Biol.%20Chem.277(39),%2036161%E2%80%9336166%20(2002).)

33. Lee, K. A. et al. Crystal structure of human cholesterol sulfotransferase (SULT2B1b) in the presence of pregnenolone and 3′-phosphoadenosine 5′-phosphate: Rationale for specificity differences between prototypical SULT2A1 and the SULT2B1 isoforms. J. Biol. Chem.278(45), 44593–44599 (2003).  [DOI](https://doi.org/10.1074/jbc.M308312200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12923182/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Lee,%20K.%20A.%20et%20al.%20Crystal%20structure%20of%20human%20cholesterol%20sulfotransferase%20(SULT2B1b)%20in%20the%20presence%20of%20pregnenolone%20and%203%E2%80%B2-phosphoadenosine%205%E2%80%B2-phosphate:%20Rationale%20for%20specificity%20differences%20between%20prototypical%20SULT2A1%20and%20the%20SULT2B1%20isoforms.%20J.%20Biol.%20Chem.278(45),%2044593%E2%80%9344599%20(2003).)

34. Williams, M. & Elias, P. Stratum corneum lipids in disorders of cornification: increased cholesterol sulfate content of stratum corneum in recessive x-linked ichthyosis. J. Clin. Investing.68(6), 1404–1410 (1981).  [DOI](https://doi.org/10.1172/JCI110391) | [PMC free article](/articles/PMC370941/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/6947980/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Williams,%20M.%20&%20Elias,%20P.%20Stratum%20corneum%20lipids%20in%20disorders%20of%20cornification:%20increased%20cholesterol%20sulfate%20content%20of%20stratum%20corneum%20in%20recessive%20x-linked%20ichthyosis.%20J.%20Clin.%20Investing.68(6),%201404%E2%80%931410%20(1981).)
