# Genetic risk factors for drug-induced long QT syndrome: findings from a large real-world case–control study

## Metadata
**Authors:** Ana I Lopez-Medina, Alessandra M Campos-Staffico, Choudhary Anwar A Chahal, Isabella Volkers, Juliet P Jacoby, Omer Berenfeld, Jasmine A Luzum
**Journal:** Pharmacogenomics
**Date:** 2024 Mar 20
**DOI:** [10.2217/pgs-2023-0229](https://doi.org/10.2217/pgs-2023-0229)
**PMID:** 38506312
**PMCID:** PMC10964839
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10964839/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC10964839/pdf/pgs-25-117.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC10964839/pdf/pgs-25-117.pdf)

## Abstract

**Aim:** 
Drug-induced long QT syndrome (diLQTS), an adverse effect of many drugs, can lead to sudden cardiac death. Candidate genetic variants in cardiac ion channels have been associated with diLQTS, but several limitations of previous studies hamper clinical utility.

**Materials & methods:** 
Thus, the purpose of this study was to assess the associations of KCNE1-D85N, KCNE2-I57T and SCN5A-G615E with diLQTS in a large observational case–control study (6,083 self-reported white patients treated with 27 different high-risk QT-prolonging medications; 12.0% with diLQTS).

**Results:** 
KCNE1-D85N significantly associated with diLQTS (adjusted odds ratio: 2.24 [95% CI: 1.35–3.58]; p = 0.001). Given low minor allele frequencies, the study had insufficient power to analyze KCNE2-I57T and SCN5A-G615E.

**Conclusion:** 
KCNE1-D85N is a risk factor for diLQTS that should be considered in future clinical practice guidelines.

Keywords: arrhythmia, drug-induced long QT syndrome, KCNE1-D85N, pharmacogenomics, torsade de pointes

### Aim:

Drug-induced long QT syndrome (diLQTS), an adverse effect of many drugs, can lead to sudden cardiac death. Candidate genetic variants in cardiac ion channels have been associated with diLQTS, but several limitations of previous studies hamper clinical utility.

### Materials & methods:

Thus, the purpose of this study was to assess the associations of *KCNE1*-D85N, *KCNE2*-I57T and *SCN5A*-G615E with diLQTS in a large observational case–control study (6,083 self-reported white patients treated with 27 different high-risk QT-prolonging medications; 12.0% with diLQTS).

### Results:

*KCNE1*-D85N significantly associated with diLQTS (adjusted odds ratio: 2.24 [95% CI: 1.35–3.58]; p = 0.001). Given low minor allele frequencies, the study had insufficient power to analyze *KCNE2*-I57T and *SCN5A*-G615E.

### Conclusion:

*KCNE1*-D85N is a risk factor for diLQTS that should be considered in future clinical practice guidelines.

## Materials & methods

### Study design

This is a single-center, retrospective case–control study using clinical and genomic data from the Michigan Genomics Initiative (MGI) [[21](#B21)]. MGI is the genomic biobank that integrates whole genome array and imputed genotype data with electronic health records (EHR) at the University of Michigan Health System (also called ‘Michigan Medicine’). MGI participants are primarily recruited while awaiting a diagnostic or interventional procedure either at a preoperative appointment or on the day of their operative procedure at Michigan Medicine. Eligible patients aged 18 years or older who were prescribed at least one dose of a high-risk QT-prolonging drug (see ‘Drug Exposure’ section below for more details) between 1 March 2001 and 30 September 2022 were selected from MGI for this study. QTc data was available starting in 2012. Patients were excluded from the study if they: were diagnosed with congenital long QT syndrome (cLQTS); had any QTc >500 ms before any treatment with a high-risk QT-prolonging drug; were diagnosed with left bundle branch block; and/or used a pacemaker. The study was carried out in accordance with the Declaration of Helsinki and was approved by the local Institutional Review Board with a waiver of informed consent.

### Subject selection

Patients included in the study were selected from Michigan Medicine's EHR with MGI genetic data ([Figure 1](#F1)). The initial eligibility criteria involved individuals who had been prescribed at least one dose of a high-risk QT-prolonging drug ([Table 1](#T1)) during the study period. Subsequently, we refined the patient sample by excluding those without any ECG measurements. A single ECG during the prescription for a high-risk drug was sufficient for inclusion in the study, since QTc >500 ms is an established risk factor for severe arrythmias. Moreover, most patients did not have an ECG measured prior to a prescription for a high-risk drug. Pediatric patients were excluded from the study. Given our focus on diLQTS, we also excluded patients with a QTc >500 ms before any treatment with a high-risk QT-prolonging drug, with a diagnosis of congenital long QT syndrome, or patients who were not prescribed a QTc-prolonging drug at the time of their maximum QTc values (sometimes the maximum QTc was measured during gaps in time between prescriptions for QT-prolonging drugs). Additionally, considering the impact of pacemakers and left bundle branch block on QT interval measurement accuracy, we also excluded patients with these conditions from this study. In order to ensure independence among the participants, pairwise genetic relatedness was also considered during patient selection. Patients with close relatives within the sample (kinship coefficients greater than 0.125, in other words, first or second-degree relatives) were randomly selected from each related pair and excluded. Additionally, 59 patients were also excluded because, even though they were genotyped by MGI, their genotype data did not meet quality control standards. The genotype data for the three candidate variants analyzed: *KCNE1*-D85N, *KCNE2*-I57T and *SCN5A*-G615E (see the ‘Candidate variant selection’ section for detail) were checked across all racial groups. However, these variants were not found in any non-white race groups. Consequently, these three variants were exclusively analyzed within the white patient subgroup in our sample, comprising a total of 6083 patients.

### Figure 1. . Flow chart with patient selection.

![Figure 1. ](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d90b/10964839/e0622b5af874/pgs-25-117-g1.jpg)

cLQTS: Congenital long QT syndrome; ECG: Electrocardiogram; EHR: Electronic health record; LBBB: Left bundle branch block; MGI: Michigan genomics initiative.

### Table 1. . List of high-risk QT-prolonging drugs with a known risk of TdP on the US market from CredibleMeds.

| AntiarrhythmicsQuinidineSotalolDofetilideDisopyramideFlecainideAmiodaroneIbutilideDronedaroneProcainamide | AntibioticsClarithromycinAzithromycinLevofloxacinErythromycinMoxifloxacinCiprofloxacin | AntipsychoticsHaloperidolDroperidolChlorpromazinePimozineThioridazineAntidepressantsCitalopramEscitalopram | AntifungalsFluconazolePentamidineCholinesterase inhibitorDonepezilOpioid agonistMethadone |
| --- | --- | --- | --- |
| Phosphodiesterase inhibitorsCilostazolAnagrelide | AntimalarialsChloroquineHydroxychloroquineAnestheticsPropofolSevofluraneCocaine | Anti-cancerOxaliplatinVandetanibArsenic trioxideCesium ChlorideMobocertinib | VasodilatorPapaverineAntiemeticsOndansetron |

Table 1 Caption: Data taken from [8].

### Data collection

Patients were initially identified using the DataDirect system [[22](#B22)], a self-serve tool that enables access to discrete clinical data, such as demographics, anthropometry, vital signs, diagnoses, procedures, exams, medications (ordered and administered) and labs (ordered and results). To ensure compliance with exclusion criteria, the Electronic Medical Record Search Engine (EMERSE) was utilized to search through clinical notes (dictated or typed) for terms [[23](#B23)], from cardiology to pathology. Data collection focused around two points in time: the start date for the first prescription for a QT-prolonging drug ([Table 1](#T1)), and the index date was defined as the date of the highest QTc value (Bazett) measured during any high-risk QT-prolonging drug prescription. Baseline refers to the time period prior to the start of any prescription for a QT-prolonging drug. Other medications were collected within a 1-year window around the index date. Age was determined at the index date. The identification of comorbidities relied on ICD-9 and -10 diagnosis codes and lab values, subsequently calculated using Elixhauser and Charlson scoring systems, as previously described [[24](#B24)].

### Drug exposure

The list of high-risk QT-prolonging drugs was sourced from the evidence-based list compilation in CredibleMeds [[25](#B25)], a federally-funded, expert-curated, nonprofit website dedicated to foster safe medication practices. CredibleMeds classifies medications into conditional, possible, or known risk for TdP. To prioritize drugs with the highest level of evidence of association with TdP risk, we selected only those categorized as known risk for TdP. The high-risk QT-prolonging drugs list, detailed in [Table 1](#T1), guided patient selection and subsequent analysis. Medication records were collected regardless of route of administation, formulation, dose or frequency. Therefore, exposure to high-risk QT-prolonging drugs was defined as the presence or absence of any listed medication in [Table 1](#T1) within the patient's medication list. Additionally, drug–drug interactions with these QT-prolonging drugs were defined as concurrent and systemic use of either cytochrome P450/p-glycoprotein (CYP/p-gp) inhibitors or inducers with known major interaction potential, as outlined by Micromedex^®^ [[26](#B26)].

### Outcomes

The primary outcome was the presence of QTc prolongation, defined as a change of >60 ms from the baseline QTc and/or an absolute QTc value ≥500 ms while patients were prescribed any high-risk QT-prolonging drugs specified in [Table 1](#T1). Controls had maximum QTc values <500 ms during their prescription(s) for QT-prolonging drug(s). The QTc interval measurements were automatically recorded using a computer-based, FDA-approved electrocardiogram (ECG) system (General Electric [GE] MUSE™ Cardiology Information System) [[27](#B27)], which is the standard system utilized in routine clinical practice at Michigan Medicine. The QT interval measurements were obtained from an average across the 12 leads of the ECG [[27](#B27)], with the primary outcome corrected for heart rate using the Bazett formula (QTc = QT/√ RR). However, recognizing that Bazett's formula tends to overestimate QT duration at extreme heart rates [[28](#B28)], sensitivity analyses were also conducted using two additional QT correction methods: Fridericia (QTc = QT/RR^1/3^) [[29](#B29)] and Framingham (QTc = QT + 0.154 × (1 – RR)) [[30](#B30)]. Additionally, if the QRS complex exceeded 120 ms, further QTc correction was performed using QTc = QTc-(QRS-100), in line with established literature [[31](#B31)].

### Candidate variant selection

A semi-quantitative scoring system to evaluate the strength of evidence for pharmacodynamic genetic variants as risk factors for diLQTS (i.e., characterized as limited, moderate, strong or definitive evidence) was established [[17](#B17)]. Among the evaluated variants, one displayed definitive evidence (*KCNE1*-D85N), one exhibited strong evidence (*KCNE2*-T8A), while four showcased moderate evidence (*SCN5A*-L1825P, *SCN5A*-G615E, *KCNE2*-I57T, *KCNE1*-D76N). The rest of the 107 genetic variants had limited evidence. For this study, we specifically selected the genetic variants with at least moderate strength of evidence for association with diLQTS, covered by MGI genotyping or imputation, totaling three: *KCNE1*-D85N, *KCNE2*-I57T and *SCN5A*-G615E.

### Genomic data

Genomic data were made available by the MGI, as previously published [[21](#B21)]. Briefly, all genotyping was performed at the UM Advanced Genomics Core lab with standard quality checks [[32](#B32)], and using Illumina Infinium CoreExome v12.1 bead arrays^®^ (Illumina, CA, USA). Quality checks were routinely conducted in batches of genotyped samples, leading to samples excluded from the study based on the following criteria: participant withdrawal, genotype-inferred sex mismatch or missing self-reported sex, atypical sex chromosomal aberration, high kinship coefficient (≥0.45) with a different sample, sample-level call-rate below 99%, technical duplicates or twin sample with higher call rate, contamination level exceeding 2.5%, call rate on any individual chromosome ≤95%, or sample processed in a flagged DNA extraction batch due to technical issue. Imputation was performed using the world-famous Michigan Imputation Server [[33](#B33)] with the Haplotype Reference Consortium r1.1 (HRC) [[34](#B34)] as the reference panel. Standard post-imputation filters were applied to remove poorly imputed variants (r^2^ <0.3 and MAF <0.01%) to ensure a high-quality dataset. The genotypes for all three candidate genetic variants analyzed were imputed. Pairwise genetic relatedness among all patients in the sample was analyzed using Kinship-based INference for GWAS (KING) v2.1.3 [[35](#B35)]. Patients with close relatives within the sample, with kinship coefficients greater than 0.125 (i.e., first or second degree relatives), were randomly selected from each related pair and excluded from the sample.

### Statistical analysis

Patients were stratified into two groups based on the primary outcome: without prolonged QTc and with prolonged QTc. Categorical variables were represented as counts and percentages and compared between subgroups using the χ^2^ test (or Fisher's exact test, when necessary). The distribution of the continuous variables was assessed using the Kolmogorov-Smirnov test and visual inspection of distribution plots. Normally distributed continuous variables were represented as mean ± standard deviation (SD) and compared between groups using the Student's *t*-test. Non-normally distributed continuous variables were represented as median ± interquartile range (IQR) and compared between groups using the Mann-Whitney U Test.

Univariable logistic regression models were used to assess the independent association of clinical variables with prolonged QTc presence. Variables showing significance (p < 0.05) between the groups were considered as clinical predictors, and were included in logistic regression models as covariates for the genetic variants. Given that several clinical variables significantly differed between groups, a propensity score was calculated for each patient using all variables with p < 0.05 [[36](#B36)]. Matching the two groups (with and without prolonged QTc) 1:1 by propensity score formed a new propensity-matched sample. The MatchIt package for RStudio was used to calculate propensity scores and match patients. Subsequently, multivariable logistic regression models were used to assess the association of each of the three genetic variants with diLQTS risk. Odds ratios (OR) and respective 95% confidence intervals (95% CI) were calculated for the unmatched sample in unadjusted and propensity score adjusted models (Models 1 and 3), and for the propensity-matched sample in unadjusted models (Model 2). Model 3 was considered the primary results due to its superior statistical power compared with Model 2. Unlike Model 1, Model 3 incorporated adjustments with clinical covariates. In consideration of the low MAF for each variant, the three candidate genetic variants was determined *a priori* to be tested using the dominant genetic model (i.e., major allele homozygotes vs heterozygotes + minor allele homozygotes). To minimize potential population stratification, self-identified race groups were analyzed separately. A Bonferroni-corrected p-value of 0.0167 (0.05÷3) for each candidate genetic variant was *a priori* set as the threshold for statistical significance. Given this Bonferroni-corrected alpha, total sample size available for analysis (n = 6,083), event rate defined by the Bazett QT correction method, and MAF observed for each variant ([Table 2](#T2)), this analysis had 80% power to detect odds ratios of 2.21, 9.03 and 44.85 for *KCNE1*-D85N, *KCNE2*-I57T and *SCN5A*-G615E, respectively. Furthermore, an exploratory analysis was conducted based on specific drug class and individual drugs, if at least 500 patients were treated with any of the high-risk QT-prolonging drugs. All statistical analyses were performed using R version 4.2.2.

### Table 2. . Allele frequency, genotype frequency and Hardy-Weinberg equilibrium assessment of the candidate genetic variants for diLQTS among self-reported white patients.

| SNPs | Allele frequency in non-Finnish European population in gnomAD | Allele frequency observed in sample | Genotype frequencies, n (%) | HWE (p-value) |
| --- | --- | --- | --- | --- |
| SCN5A-G615Ers12720452 | 0.0005178 | 0.000329 | CC: 6079 (99.9)CT: 4 (0.1)TT: 0 (0.00) | 0.980 |
| KCNE2-I57Trs74315448 | 0.001045 | 0.000986 | TT: 6071 (99.8)TC: 12 (0.2)CC: 0 (0.00) | 0.938 |
| KCNE1-D85Nrs1805128 | 0.01223 | 0.009206 | CC: 5971 (98.1)CT: 112 (1.9)TT: 0 (0.00) | 0.469 |

Table 2 Caption: HWE: Hardy-Weinberg equilibrium.

## Results

### Clinical characteristics

A total of 6989 eligible patients met the study inclusion/exclusion criteria ([Figure 1](#F1)): 6083 (87.0%) self-reported to be White, 565 (8.1%) African–American, 111 (1.6%) Asian, 46 (0.7%) American Indian or Alaska Native, 2 (0.03%) Native Hawaiian or Other Pacific Islander, and 182 (2.6%) unknown race. Genotype data for all three candidate variants, *KCNE1*-D85N, *KCNE2*-I57T and *SCN5A*-G615E, were checked across all racial groups, but none of these variants were found in any of the non-white race groups. This finding is consistent with the reported MAF in gnomAD, where these three variants are most frequent among European populations and rare in non-European populations. Consequently, these three variants were exclusively analyzed within the white patient subgroup in our study. The genotype and allele frequencies for these three genetic variants among white patients are shown in [Table 2](#T2). All genotype frequencies were in Hardy-Weinberg equilibrium with p-values >0.05, and all of the allele frequencies were similar to those previously reported for Europeans in gnomAD.

The primary outcome of QTc prolongation occurred in 12.0% of all patients, with a prevalence of 11.4% in women and 12.8% in men (p = 0.103 for sex-based difference). The clinical characteristics of both groups are shown in [Table 3](#T3). Overall, before propensity score matching, patients with prolonged QTc exhibited a significantly higher prevalence of electrolyte disturbances (hypokalemia, defined as potassium <3.5 mEq/l and hypocalcemia, defined as calcium <8.5 mg/dl), renal (chronic kidney disease), liver and cardiovascular conditions (congestive heart failure, coronary artery syndrome, hypertension, peripheral vascular disease, arrhythmias and history of myocardial infarction and stroke), chronic obstructive pulmonary disease, and diabetes mellitus compared with patients without prolonged QTc. Moreover, patients with prolonged QTc had a significantly higher prevalence of history of alcohol consumption, use of loop diuretics, digoxin and beta-blockers, than those without prolonged QTc. Elixhauser and Charlson comorbidities scores were also significantly higher in patients with prolonged QTc than those without prolonged QTc. In contrast, patients without prolonged QTc had a higher prevalence of concomitant usage of over two QTc-prolonging drugs, compared with those with prolonged QTc. There were no significant differences observed in any of the other variables assessed such as age, sex, body mass index, hypothyroidism or cancer. For the subset of patients that had baseline ECG data available (N = 433, 7% of total sample size), the median number of days [interquarile range] between the baseline QTc and the index date was 610 [1011] and 844 [1458] for those without QTc prolongation and with prolonged QTc, respectively (p = 0.519). The time between the start date of the QT-prolonging drug and the index date was also similar between the two groups (Median [IQR]: 1622 [1395] for those without QTc prolongation and 1607 [1457] for those with QTc prolongation; p = 0.964). For the purpose of matching and adjustment, age was dichotomized at 68 years old because age at 68 years old has been previously associated as an independent risk factor for diLQTS in hospitalized patients [[37](#B37)]. After applying a 1:1 propensity score matching, 733 patients were included in each group, effectively eliminating all of the significant differences in the clinical characteristics. In addition, no significant differences were found in clinical characteristics between carriers and non-carriers of the three candidate genetic variants (except for age and use of digoxin, which were significantly higher in the four carriers of *SCN5A*-G165E; [Supplementary Table 1](https://www.futuremedicine.com/doi/suppl/10.2217/pgs-2023-0229/suppl_file/pgs-25-117-s1.docx)).

### Table 3. . Clinical characteristics compared between patients without and with prolonged QTc in the unmatched and propensity-matched samples.

| Characteristics | Unmatched sample |  | p-value | Propensity-matched sample |  | p-value |
| --- | --- | --- | --- | --- | --- | --- |
|   | Without prolonged QTc | With prolonged QTc |   | Without prolonged QTc | With prolonged QTc |   |
| Patients, n (%) | 5,350 (88.0) | 733 (12.0) | __ | 733 (50.0) | 733 (50.0) | __ |
| Aged 68 years or older, n (%) | 2,967 (55.5) | 403 (55.0) | 0.838 | 297 (40.5) | 263 (35.9) | 0.076 |
| Female, n (%) | 2,985 (55.8) | 385 (52.5) | 0.103 | 373 (50.9) | 385 (52.5) | 0.565 |
| Hypokalemia, n (%) | 325 (6.1) | 103 (14.1) | <0.001 | 86 (11.7) | 103 (14.1) | 0.212 |
| Hypocalcemia, n (%) | 1,005 (18.8) | 298 (40.7) | <0.001 | 317 (43.2) | 298 (40.7) | 0.341 |
| Hypomagnesemia, n (%) | 1,059 (19.8) | 128 (17.5) | 0.149 | 155 (21.1) | 128 (17.5) | 0.085 |
| BMI (kg/m2), mean (SD) | 30.3 (7.5) | 29.9 (7.9) | 0.343 | 30.7 (7.5) | 29.9 (7.9) | 0.070 |
| CHF, n (%) | 369 (6.9) | 148 (20.2) | <0.001 | 137 (18.7) | 148 (20.2) | 0.509 |
| HTN, n (%) | 3,010 (56.3) | 495 (67.5) | <0.001 | 509 (69.4) | 495 (67.5) | 0.465 |
| CAD, n (%) | 2,279 (42.6) | 387 (52.8) | <0.001 | 393 (53.6) | 387 (52.8) | 0.794 |
| Hypothyroidism, n (%) | 754 (14.1) | 105 (14.3) | 0.911 | 119 (16.2) | 105 (14.3) | 0.345 |
| History of MI, n (%) | 374 (7.0) | 96 (13.1) | <0.001 | 91 (12.4) | 96 (13.1) | 0.754 |
| DM, n (%) | 1,097 (20.5) | 222 (30.3) | <0.001 | 234 (31.9) | 222 (30.3) | 0.535 |
| DM complicated, n (%) | 353 (6.6) | 87 (11.9) | <0.001 | 91 (12.4) | 87 (11.9) | 0.810 |
| History of Stroke, n (%) | 529 (9.9) | 121 (16.5) | <0.001 | 121 (16.5) | 121 (16.5) | 1.000 |
| CKD, n (%) | 707 (13.2) | 184 (25.1) | <0.001 | 202 (27.6) | 184 (25.1) | 0.313 |
| Severe liver disease, n (%) | 82 (1.5) | 30 (4.1) | <0.001 | 35 (4.8) | 30 (4.1) | 0.612 |
| Liver disease, n (%) | 598 (11.2) | 115 (15.7) | 0.001 | 134 (18.3) | 115 (15.7) | 0.211 |
| COPD, n (%) | 429 (8.0) | 96 (13.1) | <0.001 | 97 (13.2) | 96 (13.1) | 1.000 |
| Cancer, n (%) | 385 (7.2) | 51 (7.0) | 0.887 | 53 (7.3) | 51 (7.0) | 0.925 |
| PVD, n (%) | 458 (8.6) | 113 (15.4) | <0.001 | 118 (16.1) | 113 (15.4) | 0.774 |
| Arrhythmia, n (%) | 974 (18.2) | 223 (30.4) | <0.001 | 209 (28.5) | 223 (30.4) | 0.456 |
| History of alcohol consumption, n (%) | 189 (3.5) | 47 (6.4) | <0.001 | 44 (6.0) | 47 (6.4) | 0.829 |
| Loop diuretic, n (%) | 659 (12.3) | 261 (35.6) | <0.001 | 254 (34.7) | 261 (35.6) | 0.743 |
| Digoxin, n (%) | 21 (0.4) | 16 (2.2) | <0.001 | 15 (2.0) | 16 (2.2) | 1.000 |
| Beta-blockers, n (%) | 1,453 (27.2) | 341 (46.5) | <0.001 | 350 (47.7) | 341 (46.5) | 0.676 |
| >2 QT-prolonging drugs, n (%) | 2,771 (51.8) | 272 (37.1) | <0.001 | 274 (37.4) | 272 (37.1) | 0.957 |
| Elixhauser score (IQR) | 7.0 (13.0) | 11.0 (14.0) | <0.001 | 12 (14.0) | 11 (14.0) | 0.547 |
| Charlson score (IQR) | 3 (5.0) | 4 (5.0) | <0.001 | 5 (5) | 4 (5) | 0.727 |
| Days between start date of QT-prolonging drug and maximum QTc value (IQR) | 1622 (1395) | 1607 (1457) | 0.964 | 1739 (1368) | 1607 (1457) | 0.108 |

Table 3 Caption: Bolded p-values <0.05. BMI: Body mass index; CAD: Coronary artery disease; CHF: Congestive heart failure; CKD: Chronic kidney disease; COPD: Chronic obstructive pulmonary disease; CrCl: Creatinine clearance; DM: Diabetes mellitus; HTN: Hypertension; IQR: Interquartile range; MI: Myocardial infarction; PVD: Peripheral vascular disease.

### Drug exposure

Out of the 40 drugs marketed in the US that are classified as known risk for TdP by CredibleMeds ([Table 1](#T1)), 27 (67.5%) were prescribed in our sample ([Table 4](#T4)). Dofetilide, dronedarone, sotalol and papaverine were prescribed significantly more frequently for those who had prolonged QTc than those without prolonged QTc. In contrast, azithromycin, ciprofloxacin, propofol and ondansetron were prescribed significantly more frequently for those without prolonged QTc than those with prolonged QTc.

### Table 4. . Comparison of QTc-prolonging drugs prescribed to patients without and with prolonged QTc.

| Drug class | Drugs, n (%) | Without prolonged QTc (n = 5,350) (88.0%) | With prolonged QTc (n = 733) (12.0%) | p-value |
| --- | --- | --- | --- | --- |
| Antiarrhythmics | Disopyramide | 7 (0.1) | 0 (0.0) | 0.604 |
| Dofetilide | 14 (0.2) | 24 (2.6) | <0.001 |  |
| Dronedarone | 2 (0.0) | 3 (0.3) | 0.021 |  |
| Flecainide | 40 (0.7) | 10 (1.1) | 0.212 |  |
| Quinidine | 0 (0.0) | 1 (0.1) | 0.137 |  |
| Sotalol | 39 (0.7) | 21(2.3) | <0.001 |  |
| Procainamide | 2 (0.0) | 1 (0.1) | 0.357 |  |
| Antibiotics | Azithromycin | 556 (9.5) | 26 (2.8) | <0.001 |
| Ciprofloxacin | 607 (10.4) | 54 (5.8) | <0.001 |  |
| Clarithromycin | 4 (0.1) | 1 (0.1) | 0.521 |  |
| Levofloxacin | 290 (5.0) | 34 (3.7) | 0.102 |  |
| Moxifloxacin | 11 (0.2) | 1 (0.1) | 1.000 |  |
| Erythromycin | 11 (0.2) | 3 (0.3) | 0.426 |  |
| Antidepressants | Citalopram | 414 (7.1) | 43 (4.6) | 0.007 |
| Escitalopram | 382 (6.5) | 49 (5.3) | 0.167 |  |
| Cholinesterase inhibitor | Donepezil | 46 (0.8) | 10 (1.1) | 0.332 |
| Opioid agonist | Methadone | 54 (0.9) | 10 (1.1) | 0.587 |
| Antipsychotics | Haloperidol | 66 (1.1) | 12 (1.3) | 0.787 |
| Chlorpromazine | 9 (0.2) | 2 (0.2) | 0.655 |  |
| Antifungals | Fluconazole | 388 (6.7) | 58(6.3) | 0.712 |
| Pentamidine | 2 (0.0) | 2 (0.2) | 0.093 |  |
| Antimalarial | Hydroxychloroquine | 173 (3) | 33 (3.6) | 0.378 |
| Anesthetic | Propofol | 2,263 (38.8) | 275 (29.7) | <0.001 |
| Antiemetic | Ondansetron | 4315 (74.0) | 537 (58.0) | <0.001 |
| Oncology | Oxaliplatin | 16 (0.3) | 2 (0.2) | 1.000 |
| Vasodilator | Papaverine | 38 (0.7) | 21 (2.3) | <0.001 |
| Phosphodiesterase Inhibitor | Cilostazol | 32 (0.5) | 2 (0.2) | 0.312 |

Table 4 Caption: Bolded p-values indicate p less than 0.05.

### Associations of candidate genetic variants with the risk of drug-induced long QT syndrome

The association of *KCNE1*-D85N (rs1805128) and *KCNE2*-I57T (rs7415448) with the risk of diLQTS, as defined by Bazett's QT correction method is displayed in [Figure 2](#F2). *KCNE1*-D85N met the Bonferroni-corrected level of statistical significance (p < 0.0167) in all models: Model 1 (p = 0.006), Model 2 (p = 0.009) and Model 3 (p = 0.001). The odds ratios for *KCNE1*-D85N were 1.9, 2.9 and 2.2 in Models 1, 2, and 3 respectively. *KCNE2*-I57T did not meet the Bonferroni-corrected level of statistical significance in any of the models using the Bazett QTc correction method (p > 0.1 in all 3 models). The odds ratios for *KCNE2*-I57T were 2.4, 1.5 and 1.5 in Models 1, 2 and 3, respectively. Only four total patients carried the *SCN5A*-G615E (rs12720452) variant, and none of the carriers had prolonged QTc. Thus, *SCN5A*-G615E was not included in [Figure 2](#F2). However, complete regression results for all three variants are displayed in [Supplementary Table 2](https://www.futuremedicine.com/doi/suppl/10.2217/pgs-2023-0229/suppl_file/pgs-25-117-s1.docx).

### Figure 2. . Forest plot of the logistic regression models assessing the association of candidate genetic variants with diLQTS in white patients (dominant genetic model).

![Figure 2. ](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d90b/10964839/d0e95576f560/pgs-25-117-g2.jpg)

Model 1: Unmatched sample with unadjusted model (total n = 6,083). Model 2: Propensity-matched sample with unadjusted model (total n = 1466). Model 3: Unmatched sample with model adjusted for propensity score (total n = 6083). *p < 0.0167 (Bonferroni correction for multiple comparisons) The squares represent the odds ratio of the individual candidate gene variant in the three models and the horizontal lines indicate the 95% confidence interval. OR: Odds ratio.

Since Bazett's formula for correcting QT interval tends to overestimate the QT duration at extreme heart rates, sensitivity analyses were also performed using the Fridericia and Framingham QT correction methods. The findings are shown in [Supplementary Table 3](https://www.futuremedicine.com/doi/suppl/10.2217/pgs-2023-0229/suppl_file/pgs-25-117-s1.docx). Regardless of the QT correction method, none of the four *SCN5A*-G615E carriers were in the prolonged QTc group. Therefore, the results for *SCN5A*-G615E were similar across the three QTc correction methods. However, the results for *KCNE1*-D85N and *KCNE2*-I57T slightly differed among the three QTc correction methods. *KCNE1*-D85N did not meet the Bonferroni-corrected level of statistical significance in any of the three models with Fridericia QT correction method or Model 1 or Model 2 with the Framingham correction method (p ≥ 0.04). However, *KCNE1*-D85N was still statistically significant in Model 3 with the Framingham QT correction method (OR: 2.25; 95% CI: 1.11–4.16; p = 0.015). In contrast, even though *KCNE2*-I57T was not statistically significant in any of the models using the Bazett correction method, *KCNE2*-I57T was statistically significant in Model 1 with the Fridericia QT correction method (OR: 6.53; 95% CI: 1.74–20.80; p = 0.002) and Models 1 and 3 with the Framingham QT correction method (*KCNE2*-I57T in Model 3 with Framingham: OR: 6.70; 95% CI: 1.51–24.85; p = 0.006).

Given that *KCNE1*-D85N was the most frequent of the three variants, we also conducted exploratory analyses for *KCNE1*-D85N by specific drug class and individual drugs if at least 500 patients were treated with any of the individual drugs. The findings are shown detailed in [Supplementary Figure 1](https://www.futuremedicine.com/doi/suppl/10.2217/pgs-2023-0229/suppl_file/pgs-25-117-s1.docx). Overall, all p-values exceeded 0.3 across various individuals drugs (ondasentron, azithromycin, propofol, ciprofloxacin) and drug classes (antiarrhythmics, antibiotics, antidepressants/antipsychotics, antifungals). Although these subgroup analyses are underpowered, these results do not suggest that *KCNE1*-D85N affects risk differently among different drugs/drug classes.

## Discussion

To the best of our knowledge, this is the largest study that investigated the association of candidate genetic variants with the risk of diLQTS. The largest previous candidate gene studies with a similar endpoint all had <100 cases [[38–40](#B38)]. We analyzed three candidate genetic variants with moderate-to-high level of prior evidence for their association with diLQTS in a previous literature review and had genotype data available in our study: *KCNE1*-D85N, *KCNE2*-I57T and *SCN5A*-G615E [[17](#B17)]. Our results further support *KCNE1*-D85N as significant risk variant for diLQTS. The study had insufficient power to provide precise estimates for *KCNE2*-I57T and *SCN5A*-G615E.

### KCNE1-D85N & drug-induced long QT syndrome

*KCNE1* encodes the beta subunit of voltage-gated potassium channels, a pivotal player in modulating the alpha subunit of the cardiac channel encoded by KCNQ1. Together, KCNQ1 and KCNE1 form a complex that generates the slowly activating potassium current, also known as I_Ks_. This current, in conjunction with the rapidly activating potassium current (I_Kr_), results in cardiac repolarization. D85N leads to a substitution of asparagine with aspartic acid in the C-terminal of the beta subunit. When examined in heterologous expression models, *KCNE1*-D85N abolished up to 50% of *KCNQ1*-encoded currents in the absence of drugs [[41](#B41)]. Additionally, *in vitro* studies have found that the cell membrane expression of *KCNE1*-D85N is 20% lower than that of the wild-type [[42](#B42)]. While there is no direct effect between D85N and drugs, the susceptibility to diLQTS in D85N carriers may be attributed to a diminished ‘repolarization reserve’. Repolarization reserve refers to redundancy in mechanisms compensating for the inhibition of Kv11.1 [[43](#B43)]. These *in vitro* findings have been translated in both clinical candidate gene association studies [[44–46](#B44)] and a whole-exome sequencing study [[47](#B47)], thereby confirming the association between *KCNE1*-D85N and diLQTS. On the other hand, several other candidate gene studies did not identify the association between the D85N variant and diLQTS [[40](#B40),[48–56](#B48)]. This lack of association may be attributed to the small sample size of cases in the negative studies (largest sample size consisting of only 77 cases) and the low MAF of this variant (only ~1% in Europeans). Consequently, it is plausible that these studies might be underpowered and have introduced type 2 errors. By analyzing this variant in a much larger sample size herein, we heighten the confidence that the association between *KCNE1*-D85N and diLQTS is not either spurious or a false-positive, especially after meeting the Bonferroni corrected level of significance. Moreover, an added strength of our investigation lies in utilizing authentic patient data from real-world clinical settings, diverging from the highly controlled clinical trial in a previous study [[54](#B54)]. This improvement markedly reinforces the relevance and applicability of our findings in real-world clinical settings.

### KCNE2-I57T & drug-induced long QT syndrome

The *KCNE2* gene, very similar to *KCNE1*, encodes the beta subunit of voltage-gated potassium channels including the hERG channel, which generate I_Kr_ [[57](#B57)]. The I57T variant occurs in the transmembrane segment of the accessory β-subunit, and can decrease I_Kr_ by 34% [[38](#B38)]. Previous candidate gene [[38](#B38),[58](#B58)] and *in vitro* studies supporting functional effects [[38](#B38),[59–61](#B59)] have found *KCNE2*-I57T to be significantly associated with diLQTS. However, there are other studies that have yielded contradictory findings regarding this variant [[40](#B40),[44](#B44),[45](#B45),[47](#B47),[52](#B52),[53](#B53),[62](#B62)]. In our study, *KCNE2*-I57T did not meet the Bonferroni-corrected level of statistical significance in any of the models using the Bazett QTc correction method. However, some of the sensitivity analyses using the Fridericia and Framingham QTc correction methods were statistically significant. These discrepancies arising from our findings for the I57T variant could be explained by the decrease in statistical power due to its lower allele frequency (approximately 0.1% MAF in Europeans). In contrast to the D85N variant, the occurrence of the I57T variant was notably rarer, approximately 9.5-times less frequent within our study. Therefore, whether or not the I57T variant is a definitive risk factor for diLQTS remains unclear. To determine the ultimate implication of this variant in diLQTS, it is imperative to perform further investigations encompassing sample sizes that exceed that of our current study.

### SCN5A-G615E & drug-induced long QT syndrome

*SCN5A* encodes the α-subunit of the cardiac sodium channel, crucial for the rapid depolarization upstroke of the cardiac action potential [[63](#B63)]. The G615E occurs in the intracellular domains of the sodium channel α-subunits. However, there have been conflicting findings regarding its functional impact in experimental studies [[39](#B39),[64](#B64)]. While one study in *Xenopus* oocytes indicated that G615E reduced recovery times from fast inactivation, affecting cardiac sodium channel activity [[64](#B64)], a contradictory study by Yang *et al.* revealed that this variant did not alter the amplitude or voltage dependence of sodium channel activation/inactivation in mammalian expression vectors, suggesting it played no role in the diLQTS phenotype [[39](#B39)]. Several earlier candidate gene association studies reported an association of G615E with diLQTS [[39](#B39),[62](#B62),[65](#B65)]. However, in various other candidate gene association studies, G615E was not found in any diLQTS cases [[40](#B40),[44](#B44),[45](#B45),[49](#B49),[53](#B53),[55](#B55),[62](#B62),[66–68](#B66)], possibly due to its rarity (approximately 0.05% MAF in Europeans). Previous pharmacogenomic analyses indicated that rare genetic variants might have much larger effect sizes than common ones [[69](#B69)]. Therefore, while a larger expected effect size for G615E might compensate for lower statistical power due to its very low MAF, our study's power estimate demonstrated that even a large sample size like ours was severely underpowered to detect a significant association for G615E, even with a potentially substantial effect size. Thus, more studies are needed to confirm the association of *SCN5A*-G615E with diLQTS risk. Despite the very small number of carriers, it is worth noting that none of the four G615E carriers in our study exhibited prolonged QT.

### Clinical implication

The ultimate goal concerning *KCNE1*-D85N involves the clinical implementation of pharmacogenetic testing to prevent diLQTS. Currently, there is a lack of clinical practice guidelines or recommendations from authorities like the US FDA and Clinical Pharmacogenetics Implementation Consortium regarding the management of carriers of this variant. A recent position statement by the European Society of Cardiology emphasized the relevance of personalized treatment and risk stratification for diLQTS [[20](#B20)], but it also stressed the need for comprehensive validation studies to effectively translate *KCNE1*-D85N into clinical practice. Conducting randomized clinical trials to showcase the clinical utility of pharmacogenetic testing for these variants is probably not feasible due to their low MAF [[70](#B70)]. Despite this, the wealth of evidence from comprehensive studies aligned with our large-scale investigation suggest that *KCNE1*-D85N warrants formal evaluation for publication in a clinical practice guideline, and possibly for clinical implementation as well. The recent, landmark clinical trial published by the Ubiquitous Pharmacogenomics Consortium demonstrated the feasibility and efficacy of a pharmacogenetic testing panel in reducing the risk of severe adverse drug reactions [[71](#B71)], suggesting that a pharmacogenomic panel incorporating D85N would be feasible with minimal upfront costs and lifelong applicability for the patient. This approach is suggested to minimize prescription complications and to improve healthcare with substantial cost-benefit perspective.

### Limitations

This study has several limitations that require careful consideration. Primarily, it was a retrospective and observational study conducted within a single health system, possibly restricting the generalizability of our findings. A baseline ECG, in other words, prior to the prescription of any high-risk drug, was not available for most patients. Thus, even though we excluded patients with a known diagnosis for long QT syndrome, we assumed that most patients did not have prolonged QTc prior to the prescription. Our reliance on comorbidity data extracted from the EHR using ICD-9 and ICD-10 codes is known to be not entirely accurate. Patients were categorized based on self-identified race groups instead of ancestry, although there is a very high agreement between self-identified white patients in MGI and European ancestry [[21](#B21)]. MGI participants are predominantly recruited while awaiting a diagnostic or interventional procedure; as a result, our sample is not a random representation of patients at Michigan Medicine, potentially introducing selection biases. Additionally, our study focused solely on drugs posing a high risk of TdP, whereas there are over 100 drugs with a conditional risk for TdP [[8](#B8)], which were not included in our analysis. Furthermore, the data collected regarding patients' medication lists encompassed only prescribed medications, and assessing adherence to these prescriptions was not feasible. Our analysis only considered pharmacodynamic variants with at least moderate prior evidence of association with diLQTS; thus excluding the potential contribution of other pharmacodynamic variants with lower levels of prior evidence as well as pharmacokinetic variants, which might also influence diLQTS risk [[17](#B17)].

While Bazett's formula is the most widely adopted in clinical practice and is preferred for long QT syndrome types 1 and 2 [[72](#B72)], our study recognizes its tendency to overestimate the number of patients with potentially dangerous QTc prolongation. Notably, the Fridericia and Framingham correction formulae have demonstrated superior rate correction and a better estimate of 30-day and 1-year mortality [[73](#B73)]. The robustness of our findings is evident in the consistency of associations for *KCNE1*-D85N in Model 3 across the three QT correction methods, as shown in [Supplementary Table 3](https://www.futuremedicine.com/doi/suppl/10.2217/pgs-2023-0229/suppl_file/pgs-25-117-s1.docx). The results in Model 3 of the Framingham study with Bazett further reinforces the reliability of our conclusions. However, it is worth noting that although the p-value for D85N with Fridericia in Model 3 did not achieve statistical significance, the odds ratio was nearly 2. While the result did not reach conventional significance levels, the proximity to significance suggests that *KCNE1*-D85N, when using Fridericia, could potentially be a risk factor for diLQTS, and our study was underpowered to detect it. Finally, QT prolongation, the focus of our study, serves as a surrogate measure for more severe clinical outcomes, such as TdP and sudden cardiac death.

## Conclusion

This is the largest study of candidate genetic variants in cardiac ion channels associated with the risk of diLQTS to date. *KCNE1*-D85N significantly associated with diLQTS risk, while the study was underpowered to determine associations for *KCNE2*-I57T and *SCN5A*-G615E with diLQTS. Incorporating these findings into a clinical pharmacogenomic program may translate to preventing adverse events.

## Supplementary Material

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Roden DM. Drug-induced prolongation of the QT interval. N. Engl. J. Med. 350(10), 1013–1022 (2004).  [DOI](https://doi.org/10.1056/NEJMra032426) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14999113/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%20Engl.%20J.%20Med.&title=Drug-induced%20prolongation%20of%20the%20QT%20interval&volume=350&issue=10&publication_year=2004&pages=1013-1022&pmid=14999113&doi=10.1056/NEJMra032426&)

2. Drew BJ, Ackerman MJ, Funk M et al. Prevention of torsade de pointes in hospital settings: a scientific statement from the American Heart Association and the American College of Cardiology Foundation. J. Am. Coll. Cardiol. 55(9), 934–947 (2010).  [DOI](https://doi.org/10.1016/j.jacc.2010.01.001) | [PMC free article](/articles/PMC3057430/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20185054/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Am.%20Coll.%20Cardiol.&title=Prevention%20of%20torsade%20de%20pointes%20in%20hospital%20settings:%20a%20scientific%20statement%20from%20the%20American%20Heart%20Association%20and%20the%20American%20College%20of%20Cardiology%20Foundation&volume=55&issue=9&publication_year=2010&pages=934-947&pmid=20185054&doi=10.1016/j.jacc.2010.01.001&)

3. Desai M, Li L, Desta Z, Malik M, Flockhart D. Variability of heart rate correction methods for the QT interval. Br. J. Clin. Pharmacol. 55(6), 511–517 (2003).  [DOI](https://doi.org/10.1046/j.1365-2125.2003.01791.x) | [PMC free article](/articles/PMC1884246/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12814443/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Clin.%20Pharmacol.&title=Variability%20of%20heart%20rate%20correction%20methods%20for%20the%20QT%20interval&volume=55&issue=6&publication_year=2003&pages=511-517&pmid=12814443&doi=10.1046/j.1365-2125.2003.01791.x&)

4. Luo S, Michler K, Johnston P, Macfarlane PW. A comparison of commonly used QT correction formulae: the effect of heart rate on the QTc of normal ECGs. J. Electrocardiol. (37 Suppl.), 81–90 (2004).  [DOI](https://doi.org/10.1016/j.jelectrocard.2004.08.030) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15534815/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Electrocardiol.&title=A%20comparison%20of%20commonly%20used%20QT%20correction%20formulae:%20the%20effect%20of%20heart%20rate%20on%20the%20QTc%20of%20normal%20ECGs&issue=Suppl.&publication_year=2004&pages=81-90&pmid=15534815&doi=10.1016/j.jelectrocard.2004.08.030&)

5. Patel C, Yan GX, Antzelevitch C. Short QT syndrome: from bench to bedside. Circ. Arrhythm Electrophysiol. 3(4), 401–408 (2010).  [DOI](https://doi.org/10.1161/CIRCEP.109.921056) | [PMC free article](/articles/PMC2933105/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20716721/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circ.%20Arrhythm%20Electrophysiol.&title=Short%20QT%20syndrome:%20from%20bench%20to%20bedside&volume=3&issue=4&publication_year=2010&pages=401-408&pmid=20716721&doi=10.1161/CIRCEP.109.921056&)

6. Goldenberg I, Horr S, Moss AJ et al. Risk for life-threatening cardiac events in patients with genotype-confirmed long-QT syndrome and normal-range corrected QT intervals. J. Am. Coll. Cardiol. 57(1), 51–59 (2011).  [DOI](https://doi.org/10.1016/j.jacc.2010.07.038) | [PMC free article](/articles/PMC3332533/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21185501/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Am.%20Coll.%20Cardiol.&title=Risk%20for%20life-threatening%20cardiac%20events%20in%20patients%20with%20genotype-confirmed%20long-QT%20syndrome%20and%20normal-range%20corrected%20QT%20intervals&volume=57&issue=1&publication_year=2011&pages=51-59&pmid=21185501&doi=10.1016/j.jacc.2010.07.038&)

7. Adler A, Novelli V, Amin AS et al. An international, multicentered, evidence-based reappraisal of genes reported to cause congenital long QT syndrome. Circulation 141(6), 418–428 (2020).  [DOI](https://doi.org/10.1161/CIRCULATIONAHA.119.043132) | [PMC free article](/articles/PMC7017940/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31983240/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation&title=An%20international,%20multicentered,%20evidence-based%20reappraisal%20of%20genes%20reported%20to%20cause%20congenital%20long%20QT%20syndrome&volume=141&issue=6&publication_year=2020&pages=418-428&pmid=31983240&doi=10.1161/CIRCULATIONAHA.119.043132&)

8. Woosley RL, Heise CW, Gallo T, Woosley D, Romero KA. QTdrugs List,AZCERT, Inc. 1457 E. Desert Garden Dr., Tucson, AZ 85718 (2022). www.CredibleMeds.org  [www.CredibleMeds.org](http://www.crediblemeds.org/)

9. Roden DM. A current understanding of drug-induced QT prolongation and its implications for anticancer therapy. Cardiovasc. Res. 115(5), 895–903 (2019).  [DOI](https://doi.org/10.1093/cvr/cvz013) | [PMC free article](/articles/PMC7967705/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30689740/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cardiovasc.%20Res.&title=A%20current%20understanding%20of%20drug-induced%20QT%20prolongation%20and%20its%20implications%20for%20anticancer%20therapy&volume=115&issue=5&publication_year=2019&pages=895-903&pmid=30689740&doi=10.1093/cvr/cvz013&)

10. Del Camino D, Holmgren M, Liu Y, Yellen G. Blocker protection in the pore of a voltage-gated K+ channel and its structural implications. Nature 403(6767), 321–325 (2000).  [DOI](https://doi.org/10.1038/35002099) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10659852/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Blocker%20protection%20in%20the%20pore%20of%20a%20voltage-gated%20K+%20channel%20and%20its%20structural%20implications&volume=403&issue=6767&publication_year=2000&pages=321-325&pmid=10659852&doi=10.1038/35002099&)

11. Mitcheson JS, Chen J, Lin M, Culberson C, Sanguinetti MC. A structural basis for drug-induced long QT syndrome. Proc. Natl Acad. Sci. USA 97(22), 12329–12333 (2000).  [DOI](https://doi.org/10.1073/pnas.210244497) | [PMC free article](/articles/PMC17341/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11005845/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl%20Acad.%20Sci.%20USA&title=A%20structural%20basis%20for%20drug-induced%20long%20QT%20syndrome&volume=97&issue=22&publication_year=2000&pages=12329-12333&pmid=11005845&doi=10.1073/pnas.210244497&)

12. Wang W, Mackinnon R. Cryo-EM structure of the open human ether-à-go-go-related K(+) channel hERG. Cell 169(3), 422–430; e410 (2017).  [DOI](https://doi.org/10.1016/j.cell.2017.03.048) | [PMC free article](/articles/PMC5484391/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28431243/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell&title=Cryo-EM%20structure%20of%20the%20open%20human%20ether-%C3%A0-go-go-related%20K(+)%20channel%20hERG&volume=169&issue=3&publication_year=2017&pages=422-430&pmid=28431243&doi=10.1016/j.cell.2017.03.048&)

13. Pickham D, Helfenbein E, Shinn JA et al. High prevalence of corrected QT interval prolongation in acutely ill patients is associated with mortality: results of the QT in Practice (QTIP) Study. Crit. Care Med. 40(2), 394–399 (2012).  [DOI](https://doi.org/10.1097/CCM.0b013e318232db4a) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22001585/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Crit.%20Care%20Med.&title=High%20prevalence%20of%20corrected%20QT%20interval%20prolongation%20in%20acutely%20ill%20patients%20is%20associated%20with%20mortality:%20results%20of%20the%20QT%20in%20Practice%20(QTIP)%20Study&volume=40&issue=2&publication_year=2012&pages=394-399&pmid=22001585&doi=10.1097/CCM.0b013e318232db4a&)

14. Khatib R, Sabir FRN, Omari C, Pepper C, Tayebjee MH. Managing drug-induced QT prolongation in clinical practice. Postgrad Med. J. 97(1149), 452–458 (2021).  [DOI](https://doi.org/10.1136/postgradmedj-2020-138661) | [PMC free article](/articles/PMC8237186/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33122341/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Postgrad%20Med.%20J.&title=Managing%20drug-induced%20QT%20prolongation%20in%20clinical%20practice&volume=97&issue=1149&publication_year=2021&pages=452-458&pmid=33122341&doi=10.1136/postgradmedj-2020-138661&)

15. Strauss DG, Vicente J, Johannesen L et al. Common genetic variant risk score is associated with drug-induced QT prolongation and torsade de pointes risk: a pilot study. Circulation 135(14), 1300–1310 (2017).  [DOI](https://doi.org/10.1161/CIRCULATIONAHA.116.023980) | [PMC free article](/articles/PMC5380476/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28213480/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation&title=Common%20genetic%20variant%20risk%20score%20is%20associated%20with%20drug-induced%20QT%20prolongation%20and%20torsade%20de%20pointes%20risk:%20a%20pilot%20study&volume=135&issue=14&publication_year=2017&pages=1300-1310&pmid=28213480&doi=10.1161/CIRCULATIONAHA.116.023980&)

16. Niemeijer MN, Van Den Berg ME, Eijgelsheim M, Rijnbeek PR, Stricker BH. Pharmacogenetics of drug-induced QT interval prolongation: an update. Drug Saf. 38(10), 855–867 (2015).  [DOI](https://doi.org/10.1007/s40264-015-0316-6) | [PMC free article](/articles/PMC4579250/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26108299/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Saf.&title=Pharmacogenetics%20of%20drug-induced%20QT%20interval%20prolongation:%20an%20update&volume=38&issue=10&publication_year=2015&pages=855-867&pmid=26108299&doi=10.1007/s40264-015-0316-6&)

17. Lopez-Medina AI, Chahal CaA, Luzum JA. The genetics of drug-induced QT prolongation: evaluating the evidence for pharmacodynamic variants. Pharmacogenomics 23(9), 543–557 (2022).  [DOI](https://doi.org/10.2217/pgs-2022-0027) | [PMC free article](/articles/PMC9245591/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35698903/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=The%20genetics%20of%20drug-induced%20QT%20prolongation:%20evaluating%20the%20evidence%20for%20pharmacodynamic%20variants&volume=23&issue=9&publication_year=2022&pages=543-557&pmid=35698903&doi=10.2217/pgs-2022-0027&)

18. Kannankeril PJ, Roden DM, Norris KJ, Whalen SP, George AL Jr, Murray KT. Genetic susceptibility to acquired long QT syndrome: pharmacologic challenge in first-degree relatives. Heart Rhythm 2(2), 134–140 (2005).  [DOI](https://doi.org/10.1016/j.hrthm.2004.10.039) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15851285/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Heart%20Rhythm&title=Genetic%20susceptibility%20to%20acquired%20long%20QT%20syndrome:%20pharmacologic%20challenge%20in%20first-degree%20relatives&volume=2&issue=2&publication_year=2005&pages=134-140&pmid=15851285&doi=10.1016/j.hrthm.2004.10.039&)

19. Itoh H, Crotti L, Aiba T et al. The genetics underlying acquired long QT syndrome: impact for genetic screening. Eur. Heart J. 37(18), 1456–1464 (2016).  [DOI](https://doi.org/10.1093/eurheartj/ehv695) | [PMC free article](/articles/PMC4914885/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26715165/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur.%20Heart%20J.&title=The%20genetics%20underlying%20acquired%20long%20QT%20syndrome:%20impact%20for%20genetic%20screening&volume=37&issue=18&publication_year=2016&pages=1456-1464&pmid=26715165&doi=10.1093/eurheartj/ehv695&)

20. Magavern EF, Kaski JC, Turner RM et al. The role of pharmacogenomics in contemporary cardiovascular therapy: a position statement from the European Society of Cardiology Working Group on Cardiovascular Pharmacotherapy. Eur. Heart J. Cardiovasc. Pharmacother. 8(1), 85–99 (2022).  [DOI](https://doi.org/10.1093/ehjcvp/pvab018) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33638977/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur.%20Heart%20J.%20Cardiovasc.%20Pharmacother.&title=The%20role%20of%20pharmacogenomics%20in%20contemporary%20cardiovascular%20therapy:%20a%20position%20statement%20from%20the%20European%20Society%20of%20Cardiology%20Working%20Group%20on%20Cardiovascular%20Pharmacotherapy&volume=8&issue=1&publication_year=2022&pages=85-99&pmid=33638977&doi=10.1093/ehjcvp/pvab018&)

21. Zawistowski M, Fritsche LG, Pandit A et al. The Michigan Genomics Initiative: a biobank linking genotypes and electronic clinical records in Michigan Medicine patients. Cell Genom. 3(2), 100257 (2023).  [DOI](https://doi.org/10.1016/j.xgen.2023.100257) | [PMC free article](/articles/PMC9932985/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36819667/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Genom.&title=The%20Michigan%20Genomics%20Initiative:%20a%20biobank%20linking%20genotypes%20and%20electronic%20clinical%20records%20in%20Michigan%20Medicine%20patients&volume=3&issue=2&publication_year=2023&pages=100257&pmid=36819667&doi=10.1016/j.xgen.2023.100257&)

22. Office of Research. University of Michigan. https://research.medicine.umich.edu/our-units/data-office-clinical-translational-research/self-serve-data-tools (Accessed 28 Jun 2021).  [https://research.medicine.umich.edu/our-units/data-office-clinical-translational-research/self-serve-data-tools](https://research.medicine.umich.edu/our-units/data-office-clinical-translational-research/self-serve-data-tools)

23. Hanauer DA, Mei Q, Law J, Khanna R, Zheng K. Supporting information retrieval from electronic health records: a report of University of Michigan's nine-year experience in developing and using the Electronic Medical Record Search Engine (EMERSE). J. Biomed. Inform 55, 290–300 (2015).  [DOI](https://doi.org/10.1016/j.jbi.2015.05.003) | [PMC free article](/articles/PMC4527540/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25979153/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Biomed.%20Inform&title=Supporting%20information%20retrieval%20from%20electronic%20health%20records:%20a%20report%20of%20University%20of%20Michigan's%20nine-year%20experience%20in%20developing%20and%20using%20the%20Electronic%20Medical%20Record%20Search%20Engine%20(EMERSE)&volume=55&publication_year=2015&pages=290-300&pmid=25979153&doi=10.1016/j.jbi.2015.05.003&)

24. Wasey JO. Package “icd” version 3.3. Comorbidities from ICD-9 and ICD-10 codes, manipulation and validation (2018). www.rdocumentation.org/packages/icd/versions/3.3 (Accessed 8 February 2023).  [www.rdocumentation.org/packages/icd/versions/3.3](https://www.rdocumentation.org/packages/icd/versions/3.3)

25. Woosley RL, Black K, Heise CW, Romero K. CredibleMeds.org: what does it offer? Trends Cardiovasc. Med. 28(2), 94–99 (2018).  [DOI](https://doi.org/10.1016/j.tcm.2017.07.010) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28801207/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Trends%20Cardiovasc.%20Med.&title=CredibleMeds.org:%20what%20does%20it%20offer?&volume=28&issue=2&publication_year=2018&pages=94-99&pmid=28801207&doi=10.1016/j.tcm.2017.07.010&)

26. Micromedex® (electronic version). Merative, Ann Arbor, Michigan, USA. www.micromedexsolutions.com/ (Accessed 22 August 2022).  [www.micromedexsolutions.com/](https://www.micromedexsolutions.com/)

27. Sorajja D, Bhakta MD, Scott LR, Altemose GT, Srivathsan K. Utilization of electrocardiographic P-wave duration for AV interval optimization in dual-chamber pacemakers. Indian Pacing Electrophysiol. J. 10(9), 383–392 (2010).  [PMC free article](/articles/PMC2933366/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20930956/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Indian%20Pacing%20Electrophysiol.%20J.&title=Utilization%20of%20electrocardiographic%20P-wave%20duration%20for%20AV%20interval%20optimization%20in%20dual-chamber%20pacemakers&volume=10&issue=9&publication_year=2010&pages=383-392&pmid=20930956&)

28. Chiladakis J, Kalogeropoulos A, Zagkli F, Koutsogiannis N, Chouchoulis K, Alexopoulos D. Predicting torsade de pointes in acquired long QT syndrome: optimal identification of critical QT interval prolongation. Cardiology 122(1), 3–11 (2012).  [DOI](https://doi.org/10.1159/000338345) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22626988/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cardiology&title=Predicting%20torsade%20de%20pointes%20in%20acquired%20long%20QT%20syndrome:%20optimal%20identification%20of%20critical%20QT%20interval%20prolongation&volume=122&issue=1&publication_year=2012&pages=3-11&pmid=22626988&doi=10.1159/000338345&)

29. Fridericia LS. Die Systolendauer im Elektrokardiogramm bei normalen Menschen und bei Herzkranken. Acta Med. Scandin. 53(1), 469–486 (1920).  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Acta%20Med.%20Scandin.&title=Die%20Systolendauer%20im%20Elektrokardiogramm%20bei%20normalen%20Menschen%20und%20bei%20Herzkranken&volume=53&issue=1&publication_year=1920&pages=469-486&)

30. Sagie A, Larson MG, Goldberg RJ, Bengtson JR, Levy D. An improved method for adjusting the QT interval for heart rate (the Framingham Heart Study). Am. J. Cardiol. 70(7), 797–801 (1992).  [DOI](https://doi.org/10.1016/0002-9149(92)90562-d) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1519533/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Cardiol.&title=An%20improved%20method%20for%20adjusting%20the%20QT%20interval%20for%20heart%20rate%20(the%20Framingham%20Heart%20Study)&volume=70&issue=7&publication_year=1992&pages=797-801&pmid=1519533&doi=10.1016/0002-9149(92)90562-d&)

31. Lester RM, Paglialunga S, Johnson IA. QT assessment in early drug development: the long and the short of it. Int. J. Mol. Sci. 20(6), 1324 (2019).  [DOI](https://doi.org/10.3390/ijms20061324) | [PMC free article](/articles/PMC6471571/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30884748/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Mol.%20Sci.&title=QT%20assessment%20in%20early%20drug%20development:%20the%20long%20and%20the%20short%20of%20it&volume=20&issue=6&publication_year=2019&pages=1324&pmid=30884748&doi=10.3390/ijms20061324&)

32. Zajac GJM, Fritsche LG, Weinstock JS et al. Estimation of DNA contamination and its sources in genotyped samples. Genet. Epidemiol. 43(8), 980–995 (2019).  [DOI](https://doi.org/10.1002/gepi.22257) | [PMC free article](/articles/PMC6829038/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31452258/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genet.%20Epidemiol.&title=Estimation%20of%20DNA%20contamination%20and%20its%20sources%20in%20genotyped%20samples&volume=43&issue=8&publication_year=2019&pages=980-995&pmid=31452258&doi=10.1002/gepi.22257&)

33. Michigan Imputation Services. https://imputationserver.sph.umich.edu/index.html#! (Accessed 28 June 2022).  [https://imputationserver.sph.umich.edu/index.html#!](https://imputationserver.sph.umich.edu/index.html#!)

34. Mccarthy S, Das S, Kretzschmar W et al. A reference panel of 64,976 haplotypes for genotype imputation. Nat. Genet. 48(10), 1279–1283 (2016).  [DOI](https://doi.org/10.1038/ng.3643) | [PMC free article](/articles/PMC5388176/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27548312/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet.&title=A%20reference%20panel%20of%2064,976%20haplotypes%20for%20genotype%20imputation&volume=48&issue=10&publication_year=2016&pages=1279-1283&pmid=27548312&doi=10.1038/ng.3643&)

35. Manichaikul A, Mychaleckyj JC, Rich SS, Daly K, Sale M, Chen WM. Robust relationship inference in genome-wide association studies. Bioinformatics 26(22), 2867–2873 (2010).  [DOI](https://doi.org/10.1093/bioinformatics/btq559) | [PMC free article](/articles/PMC3025716/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20926424/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=Robust%20relationship%20inference%20in%20genome-wide%20association%20studies&volume=26&issue=22&publication_year=2010&pages=2867-2873&pmid=20926424&doi=10.1093/bioinformatics/btq559&)

36. D'agostino RB Jr. Propensity score methods for bias reduction in the comparison of a treatment to a non-randomized control group. Stat. Med. 17(19), 2265–2281 (1998).  [DOI](https://doi.org/10.1002/(sici)1097-0258(19981015)17:19<2265::aid-sim918>3.0.co;2-b) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9802183/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Stat.%20Med.&title=Propensity%20score%20methods%20for%20bias%20reduction%20in%20the%20comparison%20of%20a%20treatment%20to%20a%20non-randomized%20control%20group&volume=17&issue=19&publication_year=1998&pages=2265-2281&pmid=9802183&doi=10.1002/(sici)1097-0258(19981015)17:19<2265::aid-sim918>3.0.co;2-b&)

37. Tisdale JE, Jaynes HA, Kingery JR et al. Development and validation of a risk score to predict QT interval prolongation in hospitalized patients. Circ. Cardiovasc Qual Outcomes 6(4), 479–487 (2013).  [DOI](https://doi.org/10.1161/CIRCOUTCOMES.113.000152) | [PMC free article](/articles/PMC3788679/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23716032/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circ.%20Cardiovasc%20Qual%20Outcomes&title=Development%20and%20validation%20of%20a%20risk%20score%20to%20predict%20QT%20interval%20prolongation%20in%20hospitalized%20patients&volume=6&issue=4&publication_year=2013&pages=479-487&pmid=23716032&doi=10.1161/CIRCOUTCOMES.113.000152&)

38. Sesti F, Abbott GW, Wei J et al. A common polymorphism associated with antibiotic-induced cardiac arrhythmia. Proc. Natl Acad. Sci. USA 97(19), 10613–10618 (2000).  [DOI](https://doi.org/10.1073/pnas.180223197) | [PMC free article](/articles/PMC27073/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10984545/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl%20Acad.%20Sci.%20USA&title=A%20common%20polymorphism%20associated%20with%20antibiotic-induced%20cardiac%20arrhythmia&volume=97&issue=19&publication_year=2000&pages=10613-10618&pmid=10984545&doi=10.1073/pnas.180223197&)

39. Yang P, Kanki H, Drolet B et al. Allelic variants in long-QT disease genes in patients with drug-associated torsades de pointes. Circulation 105(16), 1943–1948 (2002).  [DOI](https://doi.org/10.1161/01.cir.0000014448.19052.4c) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11997281/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation&title=Allelic%20variants%20in%20long-QT%20disease%20genes%20in%20patients%20with%20drug-associated%20torsades%20de%20pointes&volume=105&issue=16&publication_year=2002&pages=1943-1948&pmid=11997281&doi=10.1161/01.cir.0000014448.19052.4c&)

40. Itoh H, Sakaguchi T, Ding WG et al. Latent genetic backgrounds and molecular pathogenesis in drug-induced long-QT syndrome. Circ. Arrhythm Electrophysiol. 2(5), 511–523 (2009).  [DOI](https://doi.org/10.1161/CIRCEP.109.862649) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19843919/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circ.%20Arrhythm%20Electrophysiol.&title=Latent%20genetic%20backgrounds%20and%20molecular%20pathogenesis%20in%20drug-induced%20long-QT%20syndrome&volume=2&issue=5&publication_year=2009&pages=511-523&pmid=19843919&doi=10.1161/CIRCEP.109.862649&)

41. Westenskow P, Splawski I, Timothy KW, Keating MT, Sanguinetti MC. Compound mutations: a common cause of severe long-QT syndrome. Circulation 109(15), 1834–1841 (2004).  [DOI](https://doi.org/10.1161/01.CIR.0000125524.34234.13) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15051636/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation&title=Compound%20mutations:%20a%20common%20cause%20of%20severe%20long-QT%20syndrome&volume=109&issue=15&publication_year=2004&pages=1834-1841&pmid=15051636&doi=10.1161/01.CIR.0000125524.34234.13&)

42. Sakata S, Kurata Y, Li P et al. Instability of KCNE1-D85N that causes long QT syndrome: stabilization by verapamil. Pacing Clin. Electrophysiol. 37(7), 853–863 (2014).  [DOI](https://doi.org/10.1111/pace.12360) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24499369/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pacing%20Clin.%20Electrophysiol.&title=Instability%20of%20KCNE1-D85N%20that%20causes%20long%20QT%20syndrome:%20stabilization%20by%20verapamil&volume=37&issue=7&publication_year=2014&pages=853-863&pmid=24499369&doi=10.1111/pace.12360&)

43. Roden DM. Taking the “idio” out of “idiosyncratic”: predicting torsades de pointes. Pacing Clin. Electrophysiol. 21(5), 1029–1034 (1998).  [DOI](https://doi.org/10.1111/j.1540-8159.1998.tb00148.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9604234/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pacing%20Clin.%20Electrophysiol.&title=Taking%20the%20%E2%80%9Cidio%E2%80%9D%20out%20of%20%E2%80%9Cidiosyncratic%E2%80%9D:%20predicting%20torsades%20de%20pointes&volume=21&issue=5&publication_year=1998&pages=1029-1034&pmid=9604234&doi=10.1111/j.1540-8159.1998.tb00148.x&)

44. Paulussen AD, Gilissen RA, Armstrong M et al. Genetic variations of KCNQ1, KCNH2, SCN5A, KCNE1, and KCNE2 in drug-induced long QT syndrome patients. J. Mol. Med. (Berl) 82(3), 182–188 (2004).  [DOI](https://doi.org/10.1007/s00109-003-0522-z) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14760488/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Mol.%20Med.%20(Berl)&title=Genetic%20variations%20of%20KCNQ1,%20KCNH2,%20SCN5A,%20KCNE1,%20and%20KCNE2%20in%20drug-induced%20long%20QT%20syndrome%20patients&volume=82&issue=3&publication_year=2004&pages=182-188&pmid=14760488&doi=10.1007/s00109-003-0522-z&)

45. Kaab S, Crawford DC, Sinner MF et al. A large candidate gene survey identifies the KCNE1-D85N polymorphism as a possible modulator of drug-induced torsades de pointes. Circ. Cardiovasc Genet 5(1), 91–99 (2012).  [DOI](https://doi.org/10.1161/CIRCGENETICS.111.960930) | [PMC free article](/articles/PMC3288202/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22100668/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circ.%20Cardiovasc%20Genet&title=A%20large%20candidate%20gene%20survey%20identifies%20the%20KCNE1-D85N%20polymorphism%20as%20a%20possible%20modulator%20of%20drug-induced%20torsades%20de%20pointes&volume=5&issue=1&publication_year=2012&pages=91-99&pmid=22100668&doi=10.1161/CIRCGENETICS.111.960930&)

46. Martinez-Matilla M, Blanco-Verea A, Santori M et al. Genetic susceptibility in pharmacodynamic and pharmacokinetic pathways underlying drug-induced arrhythmia and sudden unexplained deaths. Forensic Sci. Int. Genet 42, 203–212 (2019).  [DOI](https://doi.org/10.1016/j.fsigen.2019.07.010) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31376648/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Forensic%20Sci.%20Int.%20Genet&title=Genetic%20susceptibility%20in%20pharmacodynamic%20and%20pharmacokinetic%20pathways%20underlying%20drug-induced%20arrhythmia%20and%20sudden%20unexplained%20deaths&volume=42&publication_year=2019&pages=203-212&pmid=31376648&doi=10.1016/j.fsigen.2019.07.010&)

47. Weeke P, Mosley JD, Hanna D et al. Exome sequencing implicates an increased burden of rare potassium channel variants in the risk of drug-induced long QT interval syndrome. J. Am. Coll. Cardiol. 63(14), 1430–1437 (2014).  [DOI](https://doi.org/10.1016/j.jacc.2014.01.031) | [PMC free article](/articles/PMC4018823/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24561134/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Am.%20Coll.%20Cardiol.&title=Exome%20sequencing%20implicates%20an%20increased%20burden%20of%20rare%20potassium%20channel%20variants%20in%20the%20risk%20of%20drug-induced%20long%20QT%20interval%20syndrome&volume=63&issue=14&publication_year=2014&pages=1430-1437&pmid=24561134&doi=10.1016/j.jacc.2014.01.031&)

48. Corponi F, Fabbri C, Boriani G et al. Corrected QT Interval Prolongation in Psychopharmacological Treatment and Its Modulation by Genetic Variation. Neuropsychobiology 77(2), 67–72 (2019).  [DOI](https://doi.org/10.1159/000493400) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30544110/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neuropsychobiology&title=Corrected%20QT%20Interval%20Prolongation%20in%20Psychopharmacological%20Treatment%20and%20Its%20Modulation%20by%20Genetic%20Variation&volume=77&issue=2&publication_year=2019&pages=67-72&pmid=30544110&doi=10.1159/000493400&)

49. Spellmann I, Reinhard MA, Veverka D et al. QTc prolongation in short-term treatment of schizophrenia patients: effects of different antipsychotics and genetic factors. Eur. Arch. Psychiatry Clin. Neurosci. 268(4), 383–390 (2018).  [DOI](https://doi.org/10.1007/s00406-018-0880-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29429138/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur.%20Arch.%20Psychiatry%20Clin.%20Neurosci.&title=QTc%20prolongation%20in%20short-term%20treatment%20of%20schizophrenia%20patients:%20effects%20of%20different%20antipsychotics%20and%20genetic%20factors&volume=268&issue=4&publication_year=2018&pages=383-390&pmid=29429138&doi=10.1007/s00406-018-0880-8&)

50. Behr ER, Ritchie MD, Tanaka T et al. Genome wide analysis of drug-induced torsades de pointes: lack of common variants with large effect sizes. PLOS ONE 8(11), e78511 (2013).  [DOI](https://doi.org/10.1371/journal.pone.0078511) | [PMC free article](/articles/PMC3819377/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24223155/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLOS%20ONE&title=Genome%20wide%20analysis%20of%20drug-induced%20torsades%20de%20pointes:%20lack%20of%20common%20variants%20with%20large%20effect%20sizes&volume=8&issue=11&publication_year=2013&pages=e78511&pmid=24223155&doi=10.1371/journal.pone.0078511&)

51. Avery CL, Sitlani CM, Arking DE et al. Drug-gene interactions and the search for missing heritability: a cross-sectional pharmacogenomics study of the QT interval. Pharmacogenomics J. 14(1), 6–13 (2014).  [DOI](https://doi.org/10.1038/tpj.2013.4) | [PMC free article](/articles/PMC3766418/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23459443/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J.&title=Drug-gene%20interactions%20and%20the%20search%20for%20missing%20heritability:%20a%20cross-sectional%20pharmacogenomics%20study%20of%20the%20QT%20interval&volume=14&issue=1&publication_year=2014&pages=6-13&pmid=23459443&doi=10.1038/tpj.2013.4&)

52. Chevalier P, Rodriguez C, Bontemps L et al. Non-invasive testing of acquired long QT syndrome: evidence for multiple arrhythmogenic substrates. Cardiovasc. Res. 50(2), 386–398 (2001).  [DOI](https://doi.org/10.1016/s0008-6363(01)00263-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11334843/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cardiovasc.%20Res.&title=Non-invasive%20testing%20of%20acquired%20long%20QT%20syndrome:%20evidence%20for%20multiple%20arrhythmogenic%20substrates&volume=50&issue=2&publication_year=2001&pages=386-398&pmid=11334843&doi=10.1016/s0008-6363(01)00263-2&)

53. Makita N, Horie M, Nakamura T et al. Drug-induced long-QT syndrome associated with a subclinical SCN5A mutation. Circulation 106(10), 1269–1274 (2002).  [DOI](https://doi.org/10.1161/01.cir.0000027139.42087.b6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12208804/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation&title=Drug-induced%20long-QT%20syndrome%20associated%20with%20a%20subclinical%20SCN5A%20mutation&volume=106&issue=10&publication_year=2002&pages=1269-1274&pmid=12208804&doi=10.1161/01.cir.0000027139.42087.b6&)

54. Aberg K, Adkins DE, Liu Y et al. Genome-wide association study of antipsychotic-induced QTc interval prolongation. Pharmacogenomics J. 12(2), 165–172 (2012).  [DOI](https://doi.org/10.1038/tpj.2010.76) | [PMC free article](/articles/PMC3388904/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20921969/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J.&title=Genome-wide%20association%20study%20of%20antipsychotic-induced%20QTc%20interval%20prolongation&volume=12&issue=2&publication_year=2012&pages=165-172&pmid=20921969&doi=10.1038/tpj.2010.76&)

55. Zerdazi EH, Vorspan F, Marees AT et al. QT length during methadone maintenance treatment: gene x dose interaction. Fundam. Clin. Pharmacol. 33(1), 96–106 (2019).  [DOI](https://doi.org/10.1111/fcp.12405) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30086202/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Fundam.%20Clin.%20Pharmacol.&title=QT%20length%20during%20methadone%20maintenance%20treatment:%20gene%20x%20dose%20interaction&volume=33&issue=1&publication_year=2019&pages=96-106&pmid=30086202&doi=10.1111/fcp.12405&)

56. Roberts JD, Krahn AD, Ackerman MJ et al. Loss-of-Function KCNE2 Variants: True Monogenic Culprits of Long-QT Syndrome or Proarrhythmic Variants Requiring Secondary Provocation? Circ. Arrhythm Electrophysiol. 10(8), e005282 (2017).  [DOI](https://doi.org/10.1161/CIRCEP.117.005282) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28794082/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circ.%20Arrhythm%20Electrophysiol.&title=Loss-of-Function%20KCNE2%20Variants:%20True%20Monogenic%20Culprits%20of%20Long-QT%20Syndrome%20or%20Proarrhythmic%20Variants%20Requiring%20Secondary%20Provocation?&volume=10&issue=8&publication_year=2017&pages=e005282&pmid=28794082&doi=10.1161/CIRCEP.117.005282&)

57. Schmitt N, Grunnet M, Olesen SP. Cardiac potassium channel subtypes: new roles in repolarization and arrhythmia. Physiol. Rev. 94(2), 609–653 (2014).  [DOI](https://doi.org/10.1152/physrev.00022.2013) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24692356/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Physiol.%20Rev.&title=Cardiac%20potassium%20channel%20subtypes:%20new%20roles%20in%20repolarization%20and%20arrhythmia&volume=94&issue=2&publication_year=2014&pages=609-653&pmid=24692356&doi=10.1152/physrev.00022.2013&)

58. Itoh H, Crotti L, Aiba T et al. The genetics underlying acquired long QT syndrome: impact for genetic screening. Eur. Heart J. 37(18), 1456–1464 (2016).  [DOI](https://doi.org/10.1093/eurheartj/ehv695) | [PMC free article](/articles/PMC4914885/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26715165/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur.%20Heart%20J.&title=The%20genetics%20underlying%20acquired%20long%20QT%20syndrome:%20impact%20for%20genetic%20screening&volume=37&issue=18&publication_year=2016&pages=1456-1464&pmid=26715165&doi=10.1093/eurheartj/ehv695&)

59. Mccrossan ZA, Roepke TK, Lewis A, Panaghie G, Abbott GW. Regulation of the Kv2.1 potassium channel by MinK and MiRP1. J. Membr. Biol. 228(1), 1–14 (2009).  [DOI](https://doi.org/10.1007/s00232-009-9154-8) | [PMC free article](/articles/PMC2849987/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19219384/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Membr.%20Biol.&title=Regulation%20of%20the%20Kv2.1%20potassium%20channel%20by%20MinK%20and%20MiRP1&volume=228&issue=1&publication_year=2009&pages=1-14&pmid=19219384&doi=10.1007/s00232-009-9154-8&)

60. Abbott GW, Sesti F, Splawski I et al. MiRP1 forms IKr potassium channels with HERG and is associated with cardiac arrhythmia. Cell 97(2), 175–187 (1999).  [DOI](https://doi.org/10.1016/s0092-8674(00)80728-x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10219239/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell&title=MiRP1%20forms%20IKr%20potassium%20channels%20with%20HERG%20and%20is%20associated%20with%20cardiac%20arrhythmia&volume=97&issue=2&publication_year=1999&pages=175-187&pmid=10219239&doi=10.1016/s0092-8674(00)80728-x&)

61. Tinel N, Diochot S, Borsotto M, Lazdunski M, Barhanin J. KCNE2 confers background current characteristics to the cardiac KCNQ1 potassium channel. EMBO J. 19(23), 6326–6330 (2000).  [DOI](https://doi.org/10.1093/emboj/19.23.6326) | [PMC free article](/articles/PMC305874/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11101505/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=EMBO%20J.&title=KCNE2%20confers%20background%20current%20characteristics%20to%20the%20cardiac%20KCNQ1%20potassium%20channel&volume=19&issue=23&publication_year=2000&pages=6326-6330&pmid=11101505&doi=10.1093/emboj/19.23.6326&)

62. Sanchez O, Campuzano O, Fernández-Falgueras A et al. Natural and undetermined sudden death: value of post-mortem genetic investigation. PLOS ONE 11(12), e0167358 (2016).  [DOI](https://doi.org/10.1371/journal.pone.0167358) | [PMC free article](/articles/PMC5145162/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27930701/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLOS%20ONE&title=Natural%20and%20undetermined%20sudden%20death:%20value%20of%20post-mortem%20genetic%20investigation&volume=11&issue=12&publication_year=2016&pages=e0167358&pmid=27930701&doi=10.1371/journal.pone.0167358&)

63. Aronsen JM, Swift F, Sejersted OM. Cardiac sodium transport and excitation-contraction coupling. J. Mol. Cell. Cardiol. 61, 11–19 (2013).  [DOI](https://doi.org/10.1016/j.yjmcc.2013.06.003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23774049/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Mol.%20Cell.%20Cardiol.&title=Cardiac%20sodium%20transport%20and%20excitation-contraction%20coupling&volume=61&publication_year=2013&pages=11-19&pmid=23774049&doi=10.1016/j.yjmcc.2013.06.003&)

64. Albert CM, Nam EG, Rimm EB et al. Cardiac sodium channel gene variants and sudden cardiac death in women. Circulation 117(1), 16–23 (2008).  [DOI](https://doi.org/10.1161/CIRCULATIONAHA.107.736330) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18071069/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation&title=Cardiac%20sodium%20channel%20gene%20variants%20and%20sudden%20cardiac%20death%20in%20women&volume=117&issue=1&publication_year=2008&pages=16-23&pmid=18071069&doi=10.1161/CIRCULATIONAHA.107.736330&)

65. Ramirez AH, Shaffer CM, Delaney JT et al. Novel rare variants in congenital cardiac arrhythmia genes are frequent in drug-induced torsades de pointes. Pharmacogenomics J. 13(4), 325–329 (2013).  [DOI](https://doi.org/10.1038/tpj.2012.14) | [PMC free article](/articles/PMC3422407/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22584458/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J.&title=Novel%20rare%20variants%20in%20congenital%20cardiac%20arrhythmia%20genes%20are%20frequent%20in%20drug-induced%20torsades%20de%20pointes&volume=13&issue=4&publication_year=2013&pages=325-329&pmid=22584458&doi=10.1038/tpj.2012.14&)

66. Napolitano C, Schwartz PJ, Brown AM et al. Evidence for a cardiac ion channel mutation underlying drug-induced QT prolongation and life-threatening arrhythmias. J. Cardiovasc Electrophysiol. 11(6), 691–696 (2000).  [DOI](https://doi.org/10.1111/j.1540-8167.2000.tb00033.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10868744/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Cardiovasc%20Electrophysiol.&title=Evidence%20for%20a%20cardiac%20ion%20channel%20mutation%20underlying%20drug-induced%20QT%20prolongation%20and%20life-threatening%20arrhythmias&volume=11&issue=6&publication_year=2000&pages=691-696&pmid=10868744&doi=10.1111/j.1540-8167.2000.tb00033.x&)

67. Splawski I, Timothy KW, Tateyama M et al. Variant of SCN5A sodium channel implicated in risk of cardiac arrhythmia. Science 297(5585), 1333–1336 (2002).  [DOI](https://doi.org/10.1126/science.1073569) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12193783/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Variant%20of%20SCN5A%20sodium%20channel%20implicated%20in%20risk%20of%20cardiac%20arrhythmia&volume=297&issue=5585&publication_year=2002&pages=1333-1336&pmid=12193783&doi=10.1126/science.1073569&)

68. Watanabe J, Fukui N, Suzuki Y et al. Effect of GWAS-identified genetic variants on maximum QT interval in patients with schizophrenia receiving antipsychotic agents: a 24-hour Holter ECG study. J. Clin. Psychopharmacol. 37(4), 452–455 (2017).  [DOI](https://doi.org/10.1097/JCP.0000000000000724) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28590366/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Psychopharmacol.&title=Effect%20of%20GWAS-identified%20genetic%20variants%20on%20maximum%20QT%20interval%20in%20patients%20with%20schizophrenia%20receiving%20antipsychotic%20agents:%20a%2024-hour%20Holter%20ECG%20study&volume=37&issue=4&publication_year=2017&pages=452-455&pmid=28590366&doi=10.1097/JCP.0000000000000724&)

69. Ramsey LB, Bruun GH, Yang W et al. Rare versus common variants in pharmacogenetics: SLCO1B1 variation and methotrexate disposition. Genome Res. 22(1), 1–8 (2012).  [DOI](https://doi.org/10.1101/gr.129668.111) | [PMC free article](/articles/PMC3246196/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22147369/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res.&title=Rare%20versus%20common%20variants%20in%20pharmacogenetics:%20SLCO1B1%20variation%20and%20methotrexate%20disposition&volume=22&issue=1&publication_year=2012&pages=1-8&pmid=22147369&doi=10.1101/gr.129668.111&)

70. Luzum JA, Petry N, Taylor AK, Van Driest SL, Dunnenberger HM, Cavallari LH. Moving pharmacogenetics into practice: it's all about the evidence! Clin. Pharmacol. Ther. 110(3), 649–661 (2021).  [DOI](https://doi.org/10.1002/cpt.2327) | [PMC free article](/articles/PMC8376790/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34101169/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Moving%20pharmacogenetics%20into%20practice:%20it's%20all%20about%20the%20evidence!&volume=110&issue=3&publication_year=2021&pages=649-661&pmid=34101169&doi=10.1002/cpt.2327&)

71. Swen JJ, Van Der Wouden CH, Manson LE et al. A 12-gene pharmacogenetic panel to prevent adverse drug reactions: an open-label, multicentre, controlled, cluster-randomised crossover implementation study. Lancet 401(10374), 347–356 (2023).  [DOI](https://doi.org/10.1016/S0140-6736(22)01841-4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36739136/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=A%2012-gene%20pharmacogenetic%20panel%20to%20prevent%20adverse%20drug%20reactions:%20an%20open-label,%20multicentre,%20controlled,%20cluster-randomised%20crossover%20implementation%20study&volume=401&issue=10374&publication_year=2023&pages=347-356&pmid=36739136&doi=10.1016/S0140-6736(22)01841-4&)

72. Dahlberg P, Diamant UB, Gilljam T, Rydberg A, Bergfeldt L. QT correction using Bazett's formula remains preferable in long QT syndrome type 1 and 2. Ann. Noninv. Electrocardiol. 26(1), e12804 (2021).  [DOI](https://doi.org/10.1111/anec.12804) | [PMC free article](/articles/PMC7816807/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33070409/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann.%20Noninv.%20Electrocardiol.&title=QT%20correction%20using%20Bazett's%20formula%20remains%20preferable%20in%20long%20QT%20syndrome%20type%201%20and%202&volume=26&issue=1&publication_year=2021&pages=e12804&pmid=33070409&doi=10.1111/anec.12804&)

73. Vandenberk B, Vandael E, Robyns T et al. Which QT correction formulae to use for QT monitoring? J. Am. Heart Assoc. 5(6), e004252 (2016).  [DOI](https://doi.org/10.1161/JAHA.116.003264) | [PMC free article](/articles/PMC4937268/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27317349/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Am.%20Heart%20Assoc.&title=Which%20QT%20correction%20formulae%20to%20use%20for%20QT%20monitoring?&volume=5&issue=6&publication_year=2016&pages=e004252&pmid=27317349&doi=10.1161/JAHA.116.003264&)
