# Variant‐based heritability assessment of dexmedetomidine and fentanyl clearance in pediatric patients

## Metadata
**Authors:** Morgan L Shannon, Ayesha Muhammad, Nathan T James, Michael L Williams, Joseph Breeyear, Todd Edwards, Jonathan D Mosley, Leena Choi, Prince Kannankeril, Sara Van Driest
**Journal:** Clinical and Translational Science
**Date:** 2023 Jun 26
**DOI:** [10.1111/cts.13574](https://doi.org/10.1111/cts.13574)
**PMID:** 37353859
**PMCID:** PMC10499425
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10499425/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC10499425/pdf/CTS-16-1628.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC10499425/pdf/CTS-16-1628.pdf)

## Abstract

Despite complex pathways of drug disposition, clinical pharmacogenetic predictors currently rely on only a few high effect variants. Quantification of the polygenic contribution to variability in drug disposition is necessary to prioritize target drugs for pharmacogenomic approaches and guide analytic methods. Dexmedetomidine and fentanyl, often used in postoperative care of pediatric patients, have high rates of inter‐individual variability in dosing requirements. Analyzing previously generated population pharmacokinetic parameters, we used Bayesian hierarchical mixed modeling to measure narrow‐sense (additive) heritability (hSNP2) of dexmedetomidine and fentanyl clearance in children and identify relative contributions of small, moderate, and large effect‐size variants to hSNP2. We used genome‐wide association studies (GWAS) to identify variants contributing to variation in dexmedetomidine and fentanyl clearance, followed by functional analyses to identify associated pathways. For dexmedetomidine, median clearance was 33.0 L/h (interquartile range [IQR] 23.8–47.9 L/h) and hSNP2 was estimated to be 0.35 (90% credible interval 0.00–0.90), with 45% of hSNP2 attributed to large‐, 32% to moderate‐, and 23% to small‐effect variants. The fentanyl cohort had median clearance of 8.2 L/h (IQR 4.7–16.7 L/h), with estimated hSNP2 of 0.30 (90% credible interval 0.00–0.84). Large‐effect variants accounted for 30% of hSNP2, whereas moderate‐ and small‐effect variants accounted for 37% and 33%, respectively. As expected, given small sample sizes, no individual variants or pathways were significantly associated with dexmedetomidine or fentanyl clearance by GWAS. We conclude that clearance of both drugs is highly polygenic, motivating the future use of polygenic risk scores to guide appropriate dosing of dexmedetomidine and fentanyl.

There is wide inter‐individual variation in pharmacokinetics of dexmedetomidine and fentanyl. Genomic contribution to this inter‐individual pharmacokinetic variation for both drugs remains poorly understood, as candidate gene analyses have failed to identify specific genes explaining a significant portion of the variability.

How much of the inter‐individual variability in dexmedetomidine and fentanyl clearance can be attributed to common single nucleotide variation in the genome? In addition, within this heritable component, what proportion can be attributed to variants with individually small, medium, or large effects?

Dexmedetomidine and fentanyl clearance are moderately heritable and highly polygenic traits, with small‐ and moderate‐effect variants accounting for the majority of estimated heritability.

Our findings of polygenic contribution to dexmedetomidine and fentanyl clearance highlight the importance of using genome‐wide approaches for the investigation of pharmacologic phenotypes. Our study motivates the application of heritability analyses to additional pharmacokinetic datasets. Our findings demonstrate the need to assemble much larger cohorts that will enable the development, validation, and eventual implementation of polygenic risk scores for genetically‐informed drug dosing in clinical practice.

## INTRODUCTION

Precision medicine, and specifically the individualization of treatment decisions based on genetic variation (pharmacogenomics), is a rapidly evolving area of clinical care. Clinical pharmacogenomics is generally limited to the use of well‐studied genetic variants with high impact on drug outcomes. For example, the enzyme thiopurine S‐methyltransferase plays a vital role in metabolism of thiopurine drugs, and up to 10% of individuals in different populations are characterized as poor or intermediate metabolizers. In many clinical settings, genetic testing is performed to identify those with decreased enzyme activity prior to initiating therapy with a thiopurine agent, as alternate drugs or greatly reduced doses may be warranted to avoid severe toxicity.[1](#cts13574-bib-0001), [2](#cts13574-bib-0002), [3](#cts13574-bib-0003) More recently, a genome‐wide association study (GWAS) identified variants in a second gene, *NUDT15*, contributing to thiopurine toxicity, leading to the addition of *NUDT15* variants to pharmacogenomic tests.[1](#cts13574-bib-0001), [4](#cts13574-bib-0004) However, variants in these two genes are not fully predictive of thiopurine toxicity.[3](#cts13574-bib-0003) To more fully predict outcomes for drugs with residual variability or where no single pharmacogene has been identified, there is increasing focus on developing polygenic predictors of drug disposition and response.[5](#cts13574-bib-0005), [6](#cts13574-bib-0006)

Dexmedetomidine, an α_2_‐agonist, and fentanyl, an opioid, are frequently used post‐operatively in pediatric intensive care units (ICUs) to achieve sedation and analgesia.[7](#cts13574-bib-0007), [8](#cts13574-bib-0008), [9](#cts13574-bib-0009), [10](#cts13574-bib-0010) Both are commonly dosed via continuous intravenous infusions using fixed weight‐based rates, which is suitable for some patients but might require titration due to over‐ or undersedation for others. Importantly, both over‐ and undersedation can be harmful to pediatric ICU patients. Oversedation has been associated with bradycardia, respiratory depression, increased need for mechanical ventilation, and prolonged ICU stays, whereas undersedation increases the risk for agitation, patient self‐injury, and anxiety. Both over‐ and undersedation have been reported to lead to development of delirium, withdrawal syndromes, neuromuscular weakness and atrophy, and post‐traumatic stress disorder.[11](#cts13574-bib-0011)

The need for dosing adjustments of dexmedetomidine and fentanyl to maintain appropriate sedation is due in part to well‐described inter‐individual variation in pharmacokinetics. For both drugs, previous studies have shown wide variation among individuals in drug absorption, bioavailability, and clearance.[8](#cts13574-bib-0008), [12](#cts13574-bib-0012), [13](#cts13574-bib-0013), [14](#cts13574-bib-0014), [15](#cts13574-bib-0015) Specifically, inter‐individual variations in drug clearance rates in pediatric ICU patients have been reported as high as three‐fold for dexmedetomidine[16](#cts13574-bib-0016) and 10‐fold for fentanyl.[17](#cts13574-bib-0017) Pharmacogenomics research on dexmedetomidine and fentanyl pharmacokinetics has focused primarily on genes encoding enzymes known to play important roles in drug metabolism; these include cytochrome P450 (CYP) 2A6 and glucuronidation enzymes UGT1A4 and UGT2B10 for dexmedetomidine and CYP3A4 and CYP3A5 for fentanyl. To date, genes that have been identified explain little of the inter‐individual variability observed for clearance of both drugs.[18](#cts13574-bib-0018), [19](#cts13574-bib-0019), [20](#cts13574-bib-0020) The broader polygenic contribution to dexmedetomidine and fentanyl clearance has not been explored.

Genome‐wide approaches, including GWAS and Bayesian nonlinear modeling, take a broader approach by looking at the contribution of variants across the genome to a phenotype of interest. GWAS analyses individually examine millions of common genetic variants to identify those associated with a given phenotype.[21](#cts13574-bib-0021) Bayesian nonlinear models can quantify the collective contributions of large numbers of single nucleotide polymorphisms (SNPs) and estimate narrow‐sense heritability (hSNP2), the proportion of phenotypic variation that can be attributed to additive influences of SNPs.[22](#cts13574-bib-0022) In contrast to GWAS‐based estimations of hSNP2, Bayesian modeling can account for linkage disequilibrium by assuming that not all variants will have a non‐zero effect on the phenotype. The estimated hSNP2 can also be subdivided into relative contributions from genes with individually small, moderate, or large effects on the phenotype of interest. These genome‐wide approaches are complementary and frequently used in assessment of nonpharmacologic phenotypes, but their application within pharmacogenomics has been limited to date. We previously demonstrated validity of using one Bayesian method to explore pharmacologic outcomes, even when applied to small sample sizes,[23](#cts13574-bib-0023) and GWAS approaches have successfully detected pharmacogenomic signals despite small cohorts in a few examples where effect size of the variant(s) is very large,[24](#cts13574-bib-0024), [25](#cts13574-bib-0025) including the *NUDT15* association to thiopurine toxicity, as described above.[1](#cts13574-bib-0001), [4](#cts13574-bib-0004)

In this study, we used the results of previously performed population pharmacokinetic analyses[16](#cts13574-bib-0016), [17](#cts13574-bib-0017) and deployed polygenic approaches to assess genomic contribution to dexmedetomidine and fentanyl clearance in children after cardiac surgery. We used BayesR,[26](#cts13574-bib-0026), [27](#cts13574-bib-0027), [28](#cts13574-bib-0028) an established Bayesian hierarchical analysis method, to estimate hSNP2 of dexmedetomidine and fentanyl clearance and assess the relative contribution of small‐, moderate‐, and large‐effect variants. We hypothesized that common genomic variation contributes substantially to inter‐individual variation in these phenotypes and that many variants with small effects on the phenotypes would cumulatively account for a greater proportion of heritability than few variants with large effects. We additionally assessed for specific variants that may be associated with dexmedetomidine or fentanyl clearance in our datasets using GWAS.

## METHODS

### Study design and data collection

The parent study supporting participant recruitment, specimen collection, and data collection was approved by the Vanderbilt University Medical Center Institutional Review Board and has been previously described in detail.[29](#cts13574-bib-0029) Pediatric patients undergoing surgery for congenital heart disease were voluntarily enrolled in this study via written parental informed consent and, when appropriate, informed assent from the patient. Enrollment of participants into the parent study began in 2007 and is ongoing; the genomic analyses presented here used data collected from April 2013 to October 2017. All participants were admitted to the pediatric cardiac ICU following surgery and underwent routine clinical care as determined by the primary clinical team, including selection and dosing of all medications. Study participants provided a blood or saliva sample for DNA extraction, and drug concentrations were measured from remnant blood samples collected during the course of clinical care. Participants were excluded from analysis if their surgery was canceled, if they had no genotype data available, if they did not survive to hospital discharge, or if they required extracorporeal membrane oxygenation postoperatively.

Study data were collected and stored using REDCap, a secure web application hosted at Vanderbilt University.[30](#cts13574-bib-0030) Medical history and demographic data were documented by the study team upon study enrollment. Surgical and clinical data were extracted from the electronic health record (EHR) by the study team during the hospital stay. Drug data, including dosing and times of administration, were extracted from the EHR and the Vanderbilt Enterprise Data Warehouse and standardized using the EHR2PKPD system.[31](#cts13574-bib-0031)

This study was reviewed by the Institutional Review Board at Vanderbilt University Medical Center and determined to constitute non‐human subject research.

### Drug concentration measurement and clearance calculations

The primary outcomes used for analysis were the clearances estimated per individual for each of the two target drugs, as previously described.[16](#cts13574-bib-0016), [17](#cts13574-bib-0017) Briefly, dexmedetomidine and fentanyl dosing data were extracted from the EHR and drug concentrations were measured in remnant plasma specimens collected for research purposes using high‐throughput tandem mass spectrometry. Measured drug concentrations, drug dosing data, and covariate data collected for the parent study were used to generate a population pharmacokinetic model to estimate clearance of each drug, using Monolix 2021R. A two‐compartment model was selected as the base model for both drugs, and covariate modeling was performed using prespecified covariates.

For dexmedetomidine, the following covariates were considered in the population pharmacokinetic modeling based on previous research and biologic plausibility: *UGT1A4*, *UGT2B10*, and *CYP2A6* variants, body weight, postnatal age, postmenstrual age, sex, Society of Thoracic Surgeons‐European Association for Cardio‐Thoracic Surgery (STAT) Congenital Heart Surgery Mortality score, cardiac bypass time, length of ICU stay, and serum creatinine. Of these, only body weight and postmenstrual age significantly improved model fit and remained in the final covariate model.[16](#cts13574-bib-0016)

For fentanyl, CYP3A metabolizer status (based on *CYP3A45* genotype), body weight, postnatal age, postmenstrual age, sex, patient‐reported race, serum creatinine, STAT score, cardiac bypass time, concomitant CYP3A inducers and inhibitors, and the first five principal components of ancestry were considered in the pharmacokinetic model. Body weight, postnatal age, STAT score, and CYP3A metabolizer status were included in the final covariate model.[17](#cts13574-bib-0017)

For the final model for each drug, patient covariates, estimated model parameters, and the conditional mode of the individual random effect on clearance were used to calculate the estimated clearances used in this analysis.

### Genotyping, quality control, and processing

Genome‐wide SNPs were derived from either the Affymetrix Axiom Precision Medicine Research Array or Precision Medicine Diversity Array (Thermo Fisher Scientific) using Genome Reference Consortium Human Build 37 (GRCh37/hg19). The BayesR method is sensitive to population substructure, meaning that inclusion of individuals across different ancestries could confound analyses and result in spurious findings. Therefore, analyses were restricted to individuals of European ancestry as this population was the most prominent in our cohorts and the only population for which sample size was sufficient for BayesR. To do this, within each cohort (dexmedetomidine and fentanyl), genotype data from participants was combined with that of Hapmap European (Utah residents with Northern and Western European ancestry from the CEPH collection), Asian (Han Chinese in Beijing, China and Japanese in Tokyo, Japan), and African (African ancestry in Southwest USA; Luhya in Webuye, Kenya; Maasai in Kinyawa, Kenya; and Yoruba in Ibadan, Nigeria) populations to assess ancestry. Principal component (PC) analyses were performed, and, because individuals with self‐reported white race overlapped heavily with the HapMap European population and segregated from Asian and African HapMap populations, final analyses were restricted to participants within three standard deviations of the mean of PC1 and PC2 of self‐reported white participants (Figure [S1](#cts13574-supitem-0001)). This approach effectively restricted our analyses to participants of European ancestry.

Genotype data were imputed to the TOPMed reference panel R2 using the TOPMed Imputation Server. Imputed gene dosages were converted to hard calls using PLINK2 and filtered by info scores greater than 0.8. SNPs were included if they met the following quality control (QC) parameters: minor allele frequency greater than 1% within the cohort, SNP genotyping rate greater than 98%, and Hardy–Weinberg equilibrium greater than 10^−6^. Sex chromosomes were excluded from analysis. Individuals were removed if they had a genotype call rate less than 98%, discrepancy between the genetically‐estimated sex and the sex assigned in the database, or a high degree of genetic relatedness to another sample using identity‐by‐descent analysis (Figure [S2](#cts13574-supitem-0002)).

For all individuals who passed QC, any clearance values greater than the median + 3 × interquartile range [IQR] or less than the median − 3 × IQR were removed as outliers from the data set for analysis.

### BayesR analysis

Following QC of data sets as described above, SNPs in high linkage disequilibrium (*r* ^2^ > 0.9) were removed. Residuals were then calculated from clearance values incorporating the first 20 PCs of ancestry. Residuals were used in the final Bayesian analysis because the BayesR software is not able to include covariates (Figure [S2](#cts13574-supitem-0002)).

For each cohort, BayesR (version 1), a hierarchical Bayesian mixture model using Markov Chain Monte Carlo estimation, was used to fit all SNPs simultaneously to a model predicting drug clearance in order to give unbiased estimates of SNP effect sizes.[26](#cts13574-bib-0026), [27](#cts13574-bib-0027) BayesR uses *k* different normal distributions to model the prior distribution of SNP effects, where the sum of the mixture proportions is constrained to unity. As in previous analyses, we set *k* = 4,[27](#cts13574-bib-0027) where each component was modeled as a normal distribution with a mean of 0 and a variance of 0, 0.01%, 0.1%, and 1% of the additive genetic variance, respectively, as shown: pβ|π,σg2=π1N0,0×σg2+π2N0,0.0001×σg2+π3N 0,0.001×σg2+π4N0,0.01×σg2. In this model, β signifies a mixture of four zero‐mean normal distributions of SNP effects with a fixed relative variance for each mixture component, *π* represents mixture proportions constrained to sum to unity, and σg2 is additive genetic variance explained by SNPs. The mean and variance of the first component is set to zero to account for sparseness in the model. The components *k1*, *k2*, *k3*, and *k4* are referred to as no‐effect, small‐effect (each SNP explaining 0.01% of the variance), moderate‐effect (each SNP explaining 0.1% of the variance), and large‐effect SNPs (each SNP explaining 1% of the variance), respectively. To estimate σg2, the algorithm uses a Gibbs scheme to sample values from each unknown parameter's posterior distribution.[26](#cts13574-bib-0026), [27](#cts13574-bib-0027)

Narrow‐sense heritability (hSNP2) was calculated as hSNP2=σg2σg2+σe2 where σg2 and σe2 (residual variance) were estimated by BayesR. The number of iterations was increased from the default 20,000 to 60,000 in our analyses to allow for convergence given the small sample sizes in our data sets. Default settings were used for all other prior distribution parameters. The 90% highest density credible intervals were calculated. Results were processed using custom R scripts.

### Genomewide association studies

Dexmedetomidine and fentanyl cohorts used for GWAS were identical to those input into BayesR, and the same quality control steps were performed (Figure [S2](#cts13574-supitem-0002)). SNPs in high linkage disequilibrium (*r* ^2^ > 0.7) were removed to reduce computational time, leaving 986,408 and 1,080,019 SNPs for final analysis for dexmedetomidine and fentanyl, respectively. PLINK software (versions 1.9 and 2.0) using linear regression was utilized to test each SNP for association with the continuous drug clearance phenotype. PLINK software was also used to extract the first 10 PCs, which were used as covariates.[32](#cts13574-bib-0032) Results were processed using custom R scripts, and Manhattan and quantile‐quantile (Q‐Q) plots were generated using the qqman package (version 0.1.8).

We performed post hoc functional analyses of variants with association *p* values less than or equal to 1 × 10^−4^ by GWAS using the Database for Annotation, Visualization, and Integrated Discovery (DAVID) functional annotation tool.[33](#cts13574-bib-0033) Gene lists for each data set (dexmedetomidine and fentanyl) included all genes within 100,000 base pairs of those GWAS SNPs with association *p* values less than or equal to 1 × 10^−4^. Functional analyses using DAVID were restricted to gene ontology (molecular function, biological process, and cellular component), pathways (KEGG pathway, BioCarta, and biological biochemical image database), and diseases (OMIM and UniProt).

## RESULTS

### Study population characteristics

Characteristics of the dexmedetomidine and fentanyl cohorts are shown in Table [1](#cts13574-tbl-0001) and summaries of surgical procedures incurred by participants in each cohort are shown in Table [S1](#cts13574-supitem-0005). The dexmedetomidine cohort included 354 participants, as described previously.[16](#cts13574-bib-0016) Due to sensitivity of the BayesR method to population substructure, 70 individuals were removed due to genotypes consistent with non‐European ancestry, classified as individuals with PCs outside three standard deviations of the mean PC1 or PC2 of self‐reported white participants (Figure [S1](#cts13574-supitem-0001)) and one individual was removed due to high relatedness. Three additional individuals were excluded from analysis due to identification of log‐transformed clearance values as outliers; therefore, a total of 280 subjects were included for final analysis of dexmedetomidine clearance. The median postnatal age at the time of surgery was 20.7 months (IQR 5.2–72.8) and the median body weight was 9.6 kg (IQR 6.1–19.6). Median length of ICU stay was 3 days (IQR 2–7), and median cardiac bypass time was 1.6 h (1.1–2.4). Median individual dexmedetomidine clearance, as estimated by population pharmacokinetic modeling,[16](#cts13574-bib-0016) was 33.4 L/h (IQR 23.5–47.8).

### TABLE 1.

|   | Dexmedetomidine (n = 280) | Fentanyl (n = 346) |
| --- | --- | --- |
| Age (months) | 20.7 (5.2–72.8) | 8.5 (3.7–60.1) |
| Weight (kg) | 9.6 (6.1–19.6) | 7.7 (5.2–17.1) |
| Male sex | 145 (51.7%) | 184 (53.2%) |
| Self‐reported white race | 277 (98.9%) | 340 (98.3%) |
| Length of ICU stay (days) | 3 (2–7) | 4 (2–7) |
| Cardiac bypass time (h) | 1.6 (1.1–2.4) | 1.7 (1.1–2.5) |
| STAT score | 2 (1–3) | 2 (1–3) |
| Clearance (L/h) | 33.4 (23.5–47.8) | 8.1 (4.6–16.6) |

Table 1 Caption: Dexmedetomidine and fentanyl cohort characteristics.

The previously described fentanyl cohort included 434 participants[17](#cts13574-bib-0017); 87 subjects were removed due to being outside of three standard deviations of the first two PCs of self‐reported white participants (Figure [S1](#cts13574-supitem-0001)) and one individual was removed due to high relatedness, leaving a total of 346 individuals for final analysis. The median postnatal age at the time of surgery was 8.5 months (IQR 3.7–60.1) and the median body weight was 7.7 kg (IQR 5.2–17.1). Median length of ICU stay was 4 days (IQR 2–7), and median cardiac bypass time was 1.7 h (1.1–2.5). Median fentanyl clearance, as estimated by population pharmacokinetic modeling,[17](#cts13574-bib-0017) was 8.1 L/h (IQR 4.6–16.6).

### Heritability estimates of dexmedetomidine and fentanyl clearance

Dexmedetomidine clearance values from 280 individuals were log‐transformed and adjusted for the first 20 PCs (Figure [S3](#cts13574-supitem-0003)). Covariates that were considered for inclusion in the population pharmacokinetic models used to calculate drug clearance values were not considered for inclusion in the BayesR model to estimate heritability. Residual values and 1,121,432 genetic variants were input into the BayesR model. The estimated additive genetic variance (σg2) was 0.13 with a residual variance (σe2) of 0.24, resulting in a calculated hSNP2 of 0.35 with a 90% highest density credible interval of 0.00–0.90 (Figure [1](#cts13574-fig-0001)). As described above, the estimated hSNP2 was further subdivided into relative contributions from small‐, moderate‐, and large‐effect variants, which account for 0.01%, 0.1%, and 1% of σg2, respectively. For dexmedetomidine, 36% of the estimated hSNP2 was accounted for by 2997 small‐effect variants, 26% was accounted for by 264 moderate‐effect variants, and 38% was accounted for by 57 large‐effect variants. Therefore, over half of the estimated hSNP2 of dexmedetomidine clearance was attributed to small and moderate effect‐size variants (Figure [1](#cts13574-fig-0001)).

### FIGURE 1.

![FIGURE 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fdd5/10499425/1eca170eaef8/CTS-16-1628-g002.jpg)

Narrow‐sense heritability (hSNP2) estimates of dexmedetomidine and fentanyl clearance. Total hSNP2 is further subdivided into relative contributions from large‐effect (dark blue), moderate‐effect (medium blue), and small‐effect (light blue) variants. Numerical values within each effect size box represent the number of variants that comprise each effect size category. Error bars represent 90% high‐density credible intervals for hSNP2.

For fentanyl, clearance values from 346 individuals were adjusted using the first 20 PCs (Figure [S3](#cts13574-supitem-0003)) and analyzed with 1,222,265 genetic variants using BayesR. The estimated σg2 was 0.25, σe2 was 0.61, and calculated hSNP2 was 0.29 with a 90% highest density credible interval of 0.00–0.84 (Figure [1](#cts13574-fig-0001)). Relative contributions from small‐, moderate‐, and large‐effect variants were as follows: 29% of estimated hSNP2 was accounted for by 2420 small‐effect variants, 42% was accounted for by 365 moderate‐effect variants, and 38% was accounted for by 55 large‐effect variants. As with dexmedetomidine, more than half of the estimated hSNP2 of fentanyl clearance was attributed to small and moderate effect‐size variants (Figure [1](#cts13574-fig-0001)).

### Genome‐wide association analysis of dexmedetomidine and fentanyl clearance

Given the substantial heritability of clearance for both drugs and small number of large‐effect variants for each drug, we performed GWAS to identify any variants significantly associated with the outcomes. To our knowledge, these are the first GWAS of dexmedetomidine and fentanyl clearance performed to date. Q‐Q plots demonstrated mild inflation of smaller *p* values observed for dexmedetomidine and a relatively uniform distribution for fentanyl (Figure [S4](#cts13574-supitem-0004)). For both dexmedetomidine and fentanyl clearance, GWAS failed to identify any variants meeting the genome‐wide statistical significance threshold of 5 × 10^−8^ (Figure [2](#cts13574-fig-0002)).

### FIGURE 2.

![FIGURE 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fdd5/10499425/05bb19a081ed/CTS-16-1628-g001.jpg)

Manhattan plots showing genome‐wide association study results for dexmedetomidine (a) and fentanyl (b) clearance. For both plots, all genetic variants are shown as individual data points ordered by chromosome across the x‐axis. The y‐axis represents the negative log of the associated p value for each variant. The red line represents the standard genome‐wide significance threshold of 5 × 10−8. The blue line at 1 × 10−5 is suggestive of association. No variants in either the dexmedetomidine or fentanyl clearance analysis crossed this threshold to reach statistical significance.

Results of the GWAS analysis are reported in Tables [S2](#cts13574-supitem-0006) and [S3](#cts13574-supitem-0007). For dexmedetomidine clearance, the variants within 100,000 base pairs of coding regions with the strongest statistical associations (smallest *p* values) were: *CACNB2*, a voltage‐gated calcium channel; *LRRC8C*, which facilitates cAMP‐cGMP import across plasma membranes; and *RAP1A*, a member of the RAS oncogene family (Table [S2](#cts13574-supitem-0006)). For fentanyl clearance, the variants within coding regions with the smallest *p* values were: *CD58*, involved in adhesion and activation of T cells; *DYSF*, which contributes to calcium‐mediated muscle contraction; and *ORAI2*, which plays a role in store‐operated calcium channel activity (Table [S3](#cts13574-supitem-0007)).

We subsequently used the DAVID functional annotation tool to perform post hoc functional analyses on the variants with GWAS *p* values less than or equal to 1 × 10^−4^ within 100,000 bp of known genes (Tables [S4](#cts13574-supitem-0008) and [S5](#cts13574-supitem-0009)). These analyses did not identify any statistically significant pathways or functions enriched for clearance of either drug. For dexmedetomidine, the top three gene ontology categories were anchoring junction, specific granule membrane, and integral component of plasma membrane (Table [S6](#cts13574-supitem-0010)). Exogenous drug catabolism was among the top 10 pathways identified, which was associated with a total of four genes: *CYP2A13*, *CYP2A7*, *CYP2F1*, and *NOS1*. *CYP2A13*, *CYP2A7*, and *CYP2F1* were all located within 100,000 base pairs of a single SNP on chromosome 19. For the fentanyl clearance data set, membrane, protein destabilization, and positive regulation of protein serine/threonine kinase activity were the top three gene ontology categories that emerged. No pathways related to drug metabolism were enriched for the fentanyl data set (Table [S7](#cts13574-supitem-0011)).

## DISCUSSION

In this study, we sought to explore the genomic contribution to variation among pediatric patients in clearance of two drugs commonly used after cardiac surgery: dexmedetomidine and fentanyl. Our data verify the clinical observation of large inter‐individual variability in drug clearance, as our pharmacokinetic analyses demonstrated an approximately two‐fold difference between the first and third quartile of estimated dexmedetomidine clearance and an almost four‐fold difference for fentanyl clearance, consistent with previous estimates.[16](#cts13574-bib-0016), [17](#cts13574-bib-0017) We used the clearance data, coupled with genome‐wide genotype data, to employ a Bayesian hierarchical modeling methodology. Given that both hSNP2 estimations were greater than 0.25, inter‐individual variation in clearance of dexmedetomidine and fentanyl in this population is moderately heritable. Further, the observation that less than 40% of hSNP2 is attributable to large‐effect variants suggests that these traits are highly polygenic. The credible intervals for our observations are wide, which may be improved with larger sample sizes. These results suggest that, for both drugs, many genes with individually smaller effect sizes combine to comprise a substantial genomic contribution to drug clearance. Therefore, polygenic approaches combined with large sample sizes are necessary to gain a comprehensive understanding of genetic contribution to the individual pharmacokinetics of these drugs, which may ultimately be utilized, for example in the form of polygenic risk scores, in clinical practice to individualize therapy.

Although less than 40% of the estimated heritability for both dexmedetomidine and fentanyl clearance was attributed to variants with large effect sizes, their estimated contribution to heritability of both drugs was substantial. We therefore utilized GWAS in an attempt to identify any SNPs that were significantly associated with dexmedetomidine or fentanyl clearance. These analyses failed to identify any Bonferroni‐corrected statistically significant SNPs for clearance of either drug. Secondary functional analyses of genes with *p* values smaller than 1 × 10^−4^ using the DAVID functional annotation tool also did not pinpoint specific pathways or processes that were enriched within our data sets. For dexmedetomidine clearance, exogenous drug catabolism and the epoxygenase P450 pathway were among the top functional hits. However, neither of these pathways reached statistical significance and each involved only a few genes, all of which were located near a single variant identified by GWAS. Furthermore, these enzymes have not been previously reported to play a role in dexmedetomidine metabolism. For fentanyl clearance, no pathways or processes related to drug metabolism were enriched. In agreement with the results of the Bayesian analysis, these findings also suggest a highly polygenic contribution of the genome to dexmedetomidine and fentanyl clearance and support continued efforts toward utilization of polygenic approaches to investigate the pharmacokinetics of both drugs. It is also important to consider that drug metabolism and distribution are dependent on pathways across many tissue types[34](#cts13574-bib-0034), [35](#cts13574-bib-0035) and are highly variable across the lifespan.[36](#cts13574-bib-0036), [37](#cts13574-bib-0037) Therefore, future efforts aimed at defining pathways and predictive models across different tissue types and different ages may aid in the identification of novel genetic associations with drug outcomes.

Taken together, our data motivate collection of larger and more ancestrally diverse cohorts of dexmedetomidine‐ and fentanyl‐exposed individuals to develop and validate polygenic predictive models for these drugs, incorporating effects of many genes with a spectrum of effect size. Such predictors are now available for nonpharmacologic phenotypes such as type 2 diabetes,[38](#cts13574-bib-0038) coronary artery disease,[39](#cts13574-bib-0039), [40](#cts13574-bib-0040) neurodegenerative disease,[41](#cts13574-bib-0041), [42](#cts13574-bib-0042) and some cancers.[43](#cts13574-bib-0043), [44](#cts13574-bib-0044), [45](#cts13574-bib-0045) Polygenic risk predictors have been demonstrated to be especially useful for those individuals at the tail ends of the distribution for a given phenotype.[46](#cts13574-bib-0046) Applied to dexmedetomidine and fentanyl clearance, polygenic risk scores may be most useful for identifying individuals at the highest risk for undersedation or oversedation with standard drug dosing regimens secondary to differences in drug clearance. These approaches may be useful across a wide range of drugs with complex pharmacokinetic and/or pharmacodynamic pathways. Estimations of hSNP2 for drug traits may help identify the best target drugs for genomic approaches (i.e., those with high heritability).

Our study was limited by the application of modeling methodologies traditionally used with large sample sizes, such as Bayesian modeling and GWAS, to our small cohorts. This limitation likely resulted in decreased precision and wide credible intervals in our Bayesian analyses; however, Bayesian modeling has been previously used for pharmacogenomic analyses with similarly sized data sets.[23](#cts13574-bib-0023) Additionally, it has been demonstrated that in polygenic phenotypes accounted for by primarily small‐effect variants, as is true for both of our phenotypes, large sample sizes are required to detect significant variants with GWAS.[47](#cts13574-bib-0047) Therefore, given the small sample sizes, our GWAS analyses may be underpowered to detect the statistical association of some genes with our phenotypes and may have missed potentially significantly associated variants. Furthermore, development of clinically useful polygenic risk scores requires data sets much larger than those commonly available for pharmacologic outcomes. To combat this persistent limitation of pharmacogenomic research, we advocate for continued efforts toward curation of large data sets. To do so would require building scalable, sharable data structures and collaborative efforts both within and across international borders. In today's era, with the ubiquity of EHRs and advent of high throughput analytic techniques, such collaboration is both achievable and worthwhile.

Another limitation inherent in heritability studies is the need to restrict analyses to individuals of a single ancestry. Studies conducted in the United States and Europe often restrict analyses to European ancestry, and, as a result, the majority of publications in genomics include only individuals of European descent.[48](#cts13574-bib-0048), [49](#cts13574-bib-0049) Our study was limited in a similar manner as our cohort was relatively small and currently lacks sufficient numbers of individuals of non‐European ancestries to perform analyses in other ancestries. It is paramount that we prioritize diversity within pharmacogenomics research; doing so will ensure applicability of findings across populations and foster new discoveries. We are continuously enrolling subjects in this cohort and hope to soon build large enough sample sizes to repeat these analyses for individuals of other ancestries. Additionally, methodological developments in datasets of diverse and complex ancestries would enable more ancestrally inclusive studies that may reveal important differences in inter‐individual drug variability.

Finally, both a limitation of our current study and an opportunity for future investigation is the application of these same methodologies to diverse pharmacodynamic outcomes. For dexmedetomidine and fentanyl, such outcomes may include standardized sedation scores. These analyses may identify a unique set of genes that influence the response to dexmedetomidine and fentanyl in this population. In order to be successful, these strategies must account for differences in pharmacokinetics, either through the use of robust modeling or through the measurement of drug concentrations.

In summary, our study demonstrates that the genome contributes substantially to inter‐individual variation in dexmedetomidine and fentanyl clearance in the pediatric cardiac surgery population. Clearance of both drugs was polygenic, with small and moderate effect‐size variants comprising the majority of estimated heritability, whereas no individual variant had a significant association with clearance of either drug. Taken together these findings motivate the development of polygenic predictors of drug responses with the ultimate goal of implementing genomically informed drug dosing into clinical practice.

## AUTHOR CONTRIBUTIONS

M.L.S., A.M., N.T.J., M.L.W., L.C., and S.L.V.D. wrote the manuscript. M.L.S., A.M., N.T.J., M.L.W., T.E., J.D.M., L.C., P.K., and S.L.V.D. designed the research. M.L.S., A.M., N.T.J., M.L.W., J.B., and S.L.V.D. performed the research. M.L.S. analyzed the data.

## FUNDING INFORMATION

This work is supported by the National Institutes of Health (NIH) National Institute of General Medical Sciences (NIGMS) R01 GM132204 to S.L.V. This work is also supported in part by NIH/NIGMS R01 GM124109 to LC and R01 HD084461 from the NIH Eunice Kennedy Shriver National Institute of Child Health & Human Development to P.J.K. A.M. is supported by a grant from the American Heart Association (20PRE35180088) and from the Vanderbilt Medical Scientist Training Program (T32GM007347). S.L.V. and P.J.K. are supported by P50HD106446. This study used REDCap, supported by UL1 TR000445 from NIH National Center for Advancing Translational Sciences.

## CONFLICT OF INTEREST STATEMENT

The authors declared no competing interests for this work.

## Supporting information

## ACKNOWLEDGMENTS

The authors would like to thank the Research Immersion program at Vanderbilt University School of Medicine and Dr. Lea Davis for her advice and mentorship. The authors would also like to thank Dr. Sabrina Holley for her feedback and editing. This work was conducted in part using the resources of the Advanced Computing Center for Research and Education at Vanderbilt University, Nashville, Tennessee.

Shannon ML, Muhammad A, James NT, et al. Variant‐based heritability assessment of dexmedetomidine and fentanyl clearance in pediatric patients. Clin Transl Sci. 2023;16:1628‐1638. doi: 10.1111/cts.13574

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Relling MV, Schwab M, Whirl‐Carrillo M, et al. Clinical pharmacogenetics implementation consortium guideline for thiopurine dosing based on tpmt and nudt 15 genotypes: 2018 update. Clin Pharmacol Ther. 2019;105(5):1095‐1105. doi: 10.1002/cpt.1304  [DOI](https://doi.org/10.1002/cpt.1304) | [PMC free article](/articles/PMC6576267/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30447069/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Clinical%20pharmacogenetics%20implementation%20consortium%20guideline%20for%20thiopurine%20dosing%20based%20on%20tpmt%20and%20nudt%2015%20genotypes:%202018%20update&author=MV%20Relling&author=M%20Schwab&author=M%20Whirl%E2%80%90Carrillo&volume=105&issue=5&publication_year=2019&pages=1095-1105&pmid=30447069&doi=10.1002/cpt.1304&)

2. Asadov C, Aliyeva G, Mustafayeva K. Thiopurine S‐methyltransferase as a pharmacogenetic biomarker: significance of testing and review of major methods. Cardiovasc Hematol Agents Med Chem. 2017;15(1):23‐30. doi: 10.2174/1871525715666170529091921  [DOI](https://doi.org/10.2174/1871525715666170529091921) | [PMC free article](/articles/PMC5740490/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28552060/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cardiovasc%20Hematol%20Agents%20Med%20Chem&title=Thiopurine%20S%E2%80%90methyltransferase%20as%20a%20pharmacogenetic%20biomarker:%20significance%20of%20testing%20and%20review%20of%20major%20methods&author=C%20Asadov&author=G%20Aliyeva&author=K%20Mustafayeva&volume=15&issue=1&publication_year=2017&pages=23-30&pmid=28552060&doi=10.2174/1871525715666170529091921&)

3. Lennard L. Implementation of TPMT testing: TPMT testing. Br J Clin Pharmacol. 2014;77(4):704‐714. doi: 10.1111/bcp.12226  [DOI](https://doi.org/10.1111/bcp.12226) | [PMC free article](/articles/PMC3971986/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23962279/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=Implementation%20of%20TPMT%20testing:%20TPMT%20testing&author=L%20Lennard&volume=77&issue=4&publication_year=2014&pages=704-714&pmid=23962279&doi=10.1111/bcp.12226&)

4. Yang JJ, Landier W, Yang W, et al. Inherited NUDT15 variant is a genetic determinant of mercaptopurine intolerance in children with acute lymphoblastic leukemia. J Clin Oncol. 2015;33(11):1235‐1242. doi: 10.1200/JCO.2014.59.4671  [DOI](https://doi.org/10.1200/JCO.2014.59.4671) | [PMC free article](/articles/PMC4375304/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25624441/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Inherited%20NUDT15%20variant%20is%20a%20genetic%20determinant%20of%20mercaptopurine%20intolerance%20in%20children%20with%20acute%20lymphoblastic%20leukemia&author=JJ%20Yang&author=W%20Landier&author=W%20Yang&volume=33&issue=11&publication_year=2015&pages=1235-1242&pmid=25624441&doi=10.1200/JCO.2014.59.4671&)

5. Johnson D, Wilke MAP, Lyle SM, et al. A systematic review and analysis of the use of polygenic scores in pharmacogenomics. Clin Pharmacol Ther. 2022;111(4):919‐930. doi: 10.1002/cpt.2520  [DOI](https://doi.org/10.1002/cpt.2520) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34953075/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=A%20systematic%20review%20and%20analysis%20of%20the%20use%20of%20polygenic%20scores%20in%20pharmacogenomics&author=D%20Johnson&author=MAP%20Wilke&author=SM%20Lyle&volume=111&issue=4&publication_year=2022&pages=919-930&pmid=34953075&doi=10.1002/cpt.2520&)

6. Lewis CM, Vassos E. Polygenic risk scores: from research tools to clinical instruments. Genome Med. 2020;12(1):44. doi: 10.1186/s13073-020-00742-5  [DOI](https://doi.org/10.1186/s13073-020-00742-5) | [PMC free article](/articles/PMC7236300/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32423490/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Med&title=Polygenic%20risk%20scores:%20from%20research%20tools%20to%20clinical%20instruments&author=CM%20Lewis&author=E%20Vassos&volume=12&issue=1&publication_year=2020&pages=44&pmid=32423490&doi=10.1186/s13073-020-00742-5&)

7. Plambech MZ, Afshari A. Dexmedetomidine in the pediatric population: a review. Minerva Anestesiol. 2015;81(3):320‐332.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/24824958/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Minerva%20Anestesiol&title=Dexmedetomidine%20in%20the%20pediatric%20population:%20a%20review&author=MZ%20Plambech&author=A%20Afshari&volume=81&issue=3&publication_year=2015&pages=320-332&pmid=24824958&)

8. Weerink MAS, Struys MMRF, Hannivoort LN, Barends CRM, Absalom AR, Colin P. Clinical pharmacokinetics and pharmacodynamics of dexmedetomidine. Clin Pharmacokinet. 2017;56(8):893‐913. doi: 10.1007/s40262-017-0507-7  [DOI](https://doi.org/10.1007/s40262-017-0507-7) | [PMC free article](/articles/PMC5511603/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28105598/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacokinet&title=Clinical%20pharmacokinetics%20and%20pharmacodynamics%20of%20dexmedetomidine&author=MAS%20Weerink&author=MMRF%20Struys&author=LN%20Hannivoort&author=CRM%20Barends&author=AR%20Absalom&volume=56&issue=8&publication_year=2017&pages=893-913&pmid=28105598&doi=10.1007/s40262-017-0507-7&)

9. Lee S. Dexmedetomidine: present and future directions. Korean J Anesthesiol. 2019;72(4):323‐330. doi: 10.4097/kja.19259  [DOI](https://doi.org/10.4097/kja.19259) | [PMC free article](/articles/PMC6676029/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31220910/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Korean%20J%20Anesthesiol&title=Dexmedetomidine:%20present%20and%20future%20directions&author=S%20Lee&volume=72&issue=4&publication_year=2019&pages=323-330&pmid=31220910&doi=10.4097/kja.19259&)

10. Ziesenitz VC, Vaughns JD, Koch G, Mikus G, van den Anker JN. Pharmacokinetics of fentanyl and its derivatives in children: a comprehensive review. Clin Pharmacokinet. 2018;57(2):125‐149. doi: 10.1007/s40262-017-0569-6  [DOI](https://doi.org/10.1007/s40262-017-0569-6) | [PMC free article](/articles/PMC5756700/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28688027/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacokinet&title=Pharmacokinetics%20of%20fentanyl%20and%20its%20derivatives%20in%20children:%20a%20comprehensive%20review&author=VC%20Ziesenitz&author=JD%20Vaughns&author=G%20Koch&author=G%20Mikus&author=JN%20van%20den%20Anker&volume=57&issue=2&publication_year=2018&pages=125-149&pmid=28688027&doi=10.1007/s40262-017-0569-6&)

11. Egbuta C, Mason KP. Current state of analgesia and sedation in the pediatric intensive care unit. J Clin Med. 2021;10(9):1847. doi: 10.3390/jcm10091847  [DOI](https://doi.org/10.3390/jcm10091847) | [PMC free article](/articles/PMC8122992/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33922824/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Med&title=Current%20state%20of%20analgesia%20and%20sedation%20in%20the%20pediatric%20intensive%20care%20unit&author=C%20Egbuta&author=KP%20Mason&volume=10&issue=9&publication_year=2021&pages=1847&pmid=33922824&doi=10.3390/jcm10091847&)

12. Oosten AW, Abrantes JA, Jönsson S, et al. Treatment with subcutaneous and transdermal fentanyl: results from a population pharmacokinetic study in cancer patients. Eur J Clin Pharmacol. 2016;72(4):459‐467. doi: 10.1007/s00228-015-2005-x  [DOI](https://doi.org/10.1007/s00228-015-2005-x) | [PMC free article](/articles/PMC4792338/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26762381/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Clin%20Pharmacol&title=Treatment%20with%20subcutaneous%20and%20transdermal%20fentanyl:%20results%20from%20a%20population%20pharmacokinetic%20study%20in%20cancer%20patients&author=AW%20Oosten&author=JA%20Abrantes&author=S%20J%C3%B6nsson&volume=72&issue=4&publication_year=2016&pages=459-467&pmid=26762381&doi=10.1007/s00228-015-2005-x&)

13. Iirola T, Aantaa R, Laitio R, et al. Pharmacokinetics of prolonged infusion of high‐dose dexmedetomidine in critically ill patients. Crit Care. 2011;15(5):R257. doi: 10.1186/cc10518  [DOI](https://doi.org/10.1186/cc10518) | [PMC free article](/articles/PMC3334808/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22030215/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Crit%20Care&title=Pharmacokinetics%20of%20prolonged%20infusion%20of%20high%E2%80%90dose%20dexmedetomidine%20in%20critically%20ill%20patients&author=T%20Iirola&author=R%20Aantaa&author=R%20Laitio&volume=15&issue=5&publication_year=2011&pages=R257&pmid=22030215&doi=10.1186/cc10518&)

14. Välitalo PA, Ahtola‐Sätilä T, Wighton A, Sarapohja T, Pohjanjousi P, Garratt C. Population pharmacokinetics of dexmedetomidine in critically ill patients. Clin Drug Investig. 2013;33(8):579‐587. doi: 10.1007/s40261-013-0101-1  [DOI](https://doi.org/10.1007/s40261-013-0101-1) | [PMC free article](/articles/PMC3717151/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23839483/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Drug%20Investig&title=Population%20pharmacokinetics%20of%20dexmedetomidine%20in%20critically%20ill%20patients&author=PA%20V%C3%A4litalo&author=T%20Ahtola%E2%80%90S%C3%A4til%C3%A4&author=A%20Wighton&author=T%20Sarapohja&author=P%20Pohjanjousi&volume=33&issue=8&publication_year=2013&pages=579-587&pmid=23839483&doi=10.1007/s40261-013-0101-1&)

15. Kuip EJM, Zandvliet ML, Koolen SLW, Mathijssen RHJ, van der Rijt CCD. A review of factors explaining variability in fentanyl pharmacokinetics; focus on implications for cancer patients: variability in fentanyl pharmacokinetics. Br J Clin Pharmacol. 2017;83(2):294‐313. doi: 10.1111/bcp.13129  [DOI](https://doi.org/10.1111/bcp.13129) | [PMC free article](/articles/PMC5237702/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27619152/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=A%20review%20of%20factors%20explaining%20variability%20in%20fentanyl%20pharmacokinetics;%20focus%20on%20implications%20for%20cancer%20patients:%20variability%20in%20fentanyl%20pharmacokinetics&author=EJM%20Kuip&author=ML%20Zandvliet&author=SLW%20Koolen&author=RHJ%20Mathijssen&author=CCD%20van%20der%20Rijt&volume=83&issue=2&publication_year=2017&pages=294-313&pmid=27619152&doi=10.1111/bcp.13129&)

16. James NT, Breeyear JH, Caprioli R, et al. Population pharmacokinetic analysis of dexmedetomidine in children using real‐world data from electronic health records and remnant specimens. Br J Clin Pharmacol. 2022;88:2885‐2898. doi: 10.1111/bcp.15194  [DOI](https://doi.org/10.1111/bcp.15194) | [PMC free article](/articles/PMC9106818/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34957589/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=Population%20pharmacokinetic%20analysis%20of%20dexmedetomidine%20in%20children%20using%20real%E2%80%90world%20data%20from%20electronic%20health%20records%20and%20remnant%20specimens&author=NT%20James&author=JH%20Breeyear&author=R%20Caprioli&volume=88&publication_year=2022&pages=2885-2898&pmid=34957589&doi=10.1111/bcp.15194&)

17. Williams ML, Kannankeril PJ, Breeyear JH, Edwards TL, Van Driest SL, Choi L. Effect of CYP3A5 and CYP3A4 genetic variants on fentanyl pharmacokinetics in a pediatric population. Clin Pharmacol Ther. 2022;111(4):896‐908. doi: 10.1002/cpt.2506  [DOI](https://doi.org/10.1002/cpt.2506) | [PMC free article](/articles/PMC8940650/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34877660/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Effect%20of%20CYP3A5%20and%20CYP3A4%20genetic%20variants%20on%20fentanyl%20pharmacokinetics%20in%20a%20pediatric%20population&author=ML%20Williams&author=PJ%20Kannankeril&author=JH%20Breeyear&author=TL%20Edwards&author=SL%20Van%20Driest&volume=111&issue=4&publication_year=2022&pages=896-908&pmid=34877660&doi=10.1002/cpt.2506&)

18. Zhou S, Skaar DJ, Jacobson PA, Huang RS. Pharmacogenomics of medications commonly used in the intensive care unit. Front Pharmacol. 2018;9:1436. doi: 10.3389/fphar.2018.01436  [DOI](https://doi.org/10.3389/fphar.2018.01436) | [PMC free article](/articles/PMC6289166/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30564130/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front%20Pharmacol&title=Pharmacogenomics%20of%20medications%20commonly%20used%20in%20the%20intensive%20care%20unit&author=S%20Zhou&author=DJ%20Skaar&author=PA%20Jacobson&author=RS%20Huang&volume=9&publication_year=2018&pages=1436&pmid=30564130&doi=10.3389/fphar.2018.01436&)

19. Magarbeh L, Gorbovskaya I, Le Foll B, Jhirad R, Müller DJ. Reviewing pharmacogenetics to advance precision medicine for opioids. Biomed Pharmacother. 2021;142:112060. doi: 10.1016/j.biopha.2021.112060  [DOI](https://doi.org/10.1016/j.biopha.2021.112060) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34523422/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biomed%20Pharmacother&title=Reviewing%20pharmacogenetics%20to%20advance%20precision%20medicine%20for%20opioids&author=L%20Magarbeh&author=I%20Gorbovskaya&author=B%20Le%20Foll&author=R%20Jhirad&author=DJ%20M%C3%BCller&volume=142&publication_year=2021&pages=112060&pmid=34523422&doi=10.1016/j.biopha.2021.112060&)

20. Kohli U, Pandharipande P, Muszkat M, et al. CYP2A6 genetic variation and dexmedetomidine disposition. Eur J Clin Pharmacol. 2012;68(6):937‐942. doi: 10.1007/s00228-011-1208-z  [DOI](https://doi.org/10.1007/s00228-011-1208-z) | [PMC free article](/articles/PMC3352974/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22271297/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Clin%20Pharmacol&title=CYP2A6%20genetic%20variation%20and%20dexmedetomidine%20disposition&author=U%20Kohli&author=P%20Pandharipande&author=M%20Muszkat&volume=68&issue=6&publication_year=2012&pages=937-942&pmid=22271297&doi=10.1007/s00228-011-1208-z&)

21. Uffelmann E, Huang QQ, Munung NS, et al. Genome‐wide association studies. Nat Rev Methods Primer. 2021;1(1):59. doi: 10.1038/s43586-021-00056-9  [DOI](https://doi.org/10.1038/s43586-021-00056-9) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Methods%20Primer&title=Genome%E2%80%90wide%20association%20studies&author=E%20Uffelmann&author=QQ%20Huang&author=NS%20Munung&volume=1&issue=1&publication_year=2021&pages=59&doi=10.1038/s43586-021-00056-9&)

22. Mayhew AJ, Meyre D. Assessing the heritability of complex traits in humans: methodological challenges and opportunities. Curr Genomics. 2017;18(4):332‐340. doi: 10.2174/1389202918666170307161450  [DOI](https://doi.org/10.2174/1389202918666170307161450) | [PMC free article](/articles/PMC5635617/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29081689/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Genomics&title=Assessing%20the%20heritability%20of%20complex%20traits%20in%20humans:%20methodological%20challenges%20and%20opportunities&author=AJ%20Mayhew&author=D%20Meyre&volume=18&issue=4&publication_year=2017&pages=332-340&pmid=29081689&doi=10.2174/1389202918666170307161450&)

23. Muhammad A, Aka IT, Birdwell KA, et al. Genome‐wide approach to measure variant‐based heritability of drug outcome phenotypes. Clin Pharmacol Ther. 2021;110(3):714‐722. doi: 10.1002/cpt.2323  [DOI](https://doi.org/10.1002/cpt.2323) | [PMC free article](/articles/PMC8376753/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34151428/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Genome%E2%80%90wide%20approach%20to%20measure%20variant%E2%80%90based%20heritability%20of%20drug%20outcome%20phenotypes&author=A%20Muhammad&author=IT%20Aka&author=KA%20Birdwell&volume=110&issue=3&publication_year=2021&pages=714-722&pmid=34151428&doi=10.1002/cpt.2323&)

24. Ozeki T, Mushiroda T, Yowang A, et al. Genome‐wide association study identifies HLA‐A*3101 allele as a genetic risk factor for carbamazepine‐induced cutaneous adverse drug reactions in Japanese population. Hum Mol Genet. 2011;20(5):1034‐1041. doi: 10.1093/hmg/ddq537  [DOI](https://doi.org/10.1093/hmg/ddq537) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21149285/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=Genome%E2%80%90wide%20association%20study%20identifies%20HLA%E2%80%90A*3101%20allele%20as%20a%20genetic%20risk%20factor%20for%20carbamazepine%E2%80%90induced%20cutaneous%20adverse%20drug%20reactions%20in%20Japanese%20population&author=T%20Ozeki&author=T%20Mushiroda&author=A%20Yowang&volume=20&issue=5&publication_year=2011&pages=1034-1041&pmid=21149285&doi=10.1093/hmg/ddq537&)

25. Ahlström S, Bergman P, Jokela R, et al. First genome‐wide association study on rocuronium dose requirements shows association with SLCO1A2. Br J Anaesth. 2021;126(5):949‐957. doi: 10.1016/j.bja.2021.01.029  [DOI](https://doi.org/10.1016/j.bja.2021.01.029) | [PMC free article](/articles/PMC8132880/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33676726/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Anaesth&title=First%20genome%E2%80%90wide%20association%20study%20on%20rocuronium%20dose%20requirements%20shows%20association%20with%20SLCO1A2&author=S%20Ahlstr%C3%B6m&author=P%20Bergman&author=R%20Jokela&volume=126&issue=5&publication_year=2021&pages=949-957&pmid=33676726&doi=10.1016/j.bja.2021.01.029&)

26. Erbe M, Hayes BJ, Matukumalli LK, et al. Improving accuracy of genomic predictions within and between dairy cattle breeds with imputed high‐density single nucleotide polymorphism panels. J Dairy Sci. 2012;95(7):4114‐4129. doi: 10.3168/jds.2011-5019  [DOI](https://doi.org/10.3168/jds.2011-5019) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22720968/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Dairy%20Sci&title=Improving%20accuracy%20of%20genomic%20predictions%20within%20and%20between%20dairy%20cattle%20breeds%20with%20imputed%20high%E2%80%90density%20single%20nucleotide%20polymorphism%20panels&author=M%20Erbe&author=BJ%20Hayes&author=LK%20Matukumalli&volume=95&issue=7&publication_year=2012&pages=4114-4129&pmid=22720968&doi=10.3168/jds.2011-5019&)

27. Moser G, Lee SH, Hayes BJ, Goddard ME, Wray NR, Visscher PM. Simultaneous discovery, estimation and prediction analysis of complex traits using a Bayesian mixture model. Haley C, ed. PLOS Genet. 2015;11(4):e1004969. doi: 10.1371/journal.pgen.1004969  [DOI](https://doi.org/10.1371/journal.pgen.1004969) | [PMC free article](/articles/PMC4388571/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25849665/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLOS%20Genet&title=Simultaneous%20discovery,%20estimation%20and%20prediction%20analysis%20of%20complex%20traits%20using%20a%20Bayesian%20mixture%20model.%20Haley%20C,%20ed&author=G%20Moser&author=SH%20Lee&author=BJ%20Hayes&author=ME%20Goddard&author=NR%20Wray&volume=11&issue=4&publication_year=2015&pages=e1004969&pmid=25849665&doi=10.1371/journal.pgen.1004969&)

28. Mollandin F, Rau A, Croiseau P. An evaluation of the predictive performance and mapping power of the BayesR model for genomic prediction. G3 (Bethesda). 2021;11(11):jkab225. doi: 10.1093/g3journal/jkab225  [DOI](https://doi.org/10.1093/g3journal/jkab225) | [PMC free article](/articles/PMC8527474/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34849780/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=G3%20(Bethesda)&title=An%20evaluation%20of%20the%20predictive%20performance%20and%20mapping%20power%20of%20the%20BayesR%20model%20for%20genomic%20prediction&author=F%20Mollandin&author=A%20Rau&author=P%20Croiseau&volume=11&issue=11&publication_year=2021&pages=jkab225&pmid=34849780&doi=10.1093/g3journal/jkab225&)

29. Van Driest SL, Marshall MD, Hachey B, et al. Pragmatic pharmacology: population pharmacokinetic analysis of fentanyl using remnant samples from children after cardiac surgery: population PK of fentanyl using remnant samples from children. Br J Clin Pharmacol. 2016;81(6):1165‐1174. doi: 10.1111/bcp.12903  [DOI](https://doi.org/10.1111/bcp.12903) | [PMC free article](/articles/PMC4876191/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26861166/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=Pragmatic%20pharmacology:%20population%20pharmacokinetic%20analysis%20of%20fentanyl%20using%20remnant%20samples%20from%20children%20after%20cardiac%20surgery:%20population%20PK%20of%20fentanyl%20using%20remnant%20samples%20from%20children&author=SL%20Van%20Driest&author=MD%20Marshall&author=B%20Hachey&volume=81&issue=6&publication_year=2016&pages=1165-1174&pmid=26861166&doi=10.1111/bcp.12903&)

30. Harris PA, Taylor R, Thielke R, Payne J, Gonzalez N, Conde JG. Research electronic data capture (REDCap)—a metadata‐driven methodology and workflow process for providing translational research informatics support. J Biomed Inform. 2009;42(2):377‐381. doi: 10.1016/j.jbi.2008.08.010  [DOI](https://doi.org/10.1016/j.jbi.2008.08.010) | [PMC free article](/articles/PMC2700030/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18929686/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biomed%20Inform&title=Research%20electronic%20data%20capture%20(REDCap)%E2%80%94a%20metadata%E2%80%90driven%20methodology%20and%20workflow%20process%20for%20providing%20translational%20research%20informatics%20support&author=PA%20Harris&author=R%20Taylor&author=R%20Thielke&author=J%20Payne&author=N%20Gonzalez&volume=42&issue=2&publication_year=2009&pages=377-381&pmid=18929686&doi=10.1016/j.jbi.2008.08.010&)

31. Choi L, Beck C, McNeer E, et al. Development of a system for Postmarketing population pharmacokinetic and pharmacodynamic studies using real‐world data from electronic health records. Clin Pharmacol Ther. 2020;107(4):934‐943. doi: 10.1002/cpt.1787  [DOI](https://doi.org/10.1002/cpt.1787) | [PMC free article](/articles/PMC7093250/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31957870/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Development%20of%20a%20system%20for%20Postmarketing%20population%20pharmacokinetic%20and%20pharmacodynamic%20studies%20using%20real%E2%80%90world%20data%20from%20electronic%20health%20records&author=L%20Choi&author=C%20Beck&author=E%20McNeer&volume=107&issue=4&publication_year=2020&pages=934-943&pmid=31957870&doi=10.1002/cpt.1787&)

32. Price AL, Patterson NJ, Plenge RM, Weinblatt ME, Shadick NA, Reich D. Principal components analysis corrects for stratification in genome‐wide association studies. Nat Genet. 2006;38(8):904‐909. doi: 10.1038/ng1847  [DOI](https://doi.org/10.1038/ng1847) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16862161/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Principal%20components%20analysis%20corrects%20for%20stratification%20in%20genome%E2%80%90wide%20association%20studies&author=AL%20Price&author=NJ%20Patterson&author=RM%20Plenge&author=ME%20Weinblatt&author=NA%20Shadick&volume=38&issue=8&publication_year=2006&pages=904-909&pmid=16862161&doi=10.1038/ng1847&)

33. Sherman BT, Hao M, Qiu J, et al. DAVID: a web server for functional enrichment analysis and functional annotation of gene lists (2021 update). Nucleic Acids Res. 2022;50(W1):W216‐W221. doi: 10.1093/nar/gkac194  [DOI](https://doi.org/10.1093/nar/gkac194) | [PMC free article](/articles/PMC9252805/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35325185/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=DAVID:%20a%20web%20server%20for%20functional%20enrichment%20analysis%20and%20functional%20annotation%20of%20gene%20lists%20(2021%20update)&author=BT%20Sherman&author=M%20Hao&author=J%20Qiu&volume=50&issue=W1&publication_year=2022&pages=W216-W221&pmid=35325185&doi=10.1093/nar/gkac194&)

34. Piñero J, Gonzalez‐Perez A, Guney E, et al. Network, transcriptomic and genomic features differentiate genes relevant for drug response. Front Genet. 2018;9:412. doi: 10.3389/fgene.2018.00412  [DOI](https://doi.org/10.3389/fgene.2018.00412) | [PMC free article](/articles/PMC6168038/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30319692/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front%20Genet&title=Network,%20transcriptomic%20and%20genomic%20features%20differentiate%20genes%20relevant%20for%20drug%20response&author=J%20Pi%C3%B1ero&author=A%20Gonzalez%E2%80%90Perez&author=E%20Guney&volume=9&publication_year=2018&pages=412&pmid=30319692&doi=10.3389/fgene.2018.00412&)

35. Drozdzik M, Busch D, Lapczuk J, et al. Protein abundance of clinically relevant drug‐metabolizing enzymes in the human liver and intestine: a comparative analysis in paired tissue specimens. Clin Pharmacol Ther. 2018;104(3):515‐524. doi: 10.1002/cpt.967  [DOI](https://doi.org/10.1002/cpt.967) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29205295/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Protein%20abundance%20of%20clinically%20relevant%20drug%E2%80%90metabolizing%20enzymes%20in%20the%20human%20liver%20and%20intestine:%20a%20comparative%20analysis%20in%20paired%20tissue%20specimens&author=M%20Drozdzik&author=D%20Busch&author=J%20Lapczuk&volume=104&issue=3&publication_year=2018&pages=515-524&pmid=29205295&doi=10.1002/cpt.967&)

36. Mangoni AA, Jackson SHD. Age‐related changes in pharmacokinetics and pharmacodynamics: basic principles and practical applications. Br J Clin Pharmacol. 2004;57(1):6‐14. doi: 10.1046/j.1365-2125.2003.02007.x  [DOI](https://doi.org/10.1046/j.1365-2125.2003.02007.x) | [PMC free article](/articles/PMC1884408/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14678335/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=Age%E2%80%90related%20changes%20in%20pharmacokinetics%20and%20pharmacodynamics:%20basic%20principles%20and%20practical%20applications&author=AA%20Mangoni&author=SHD%20Jackson&volume=57&issue=1&publication_year=2004&pages=6-14&pmid=14678335&doi=10.1046/j.1365-2125.2003.02007.x&)

37. Batchelor HK, Marriott JF. Paediatric pharmacokinetics: key considerations. Br J Clin Pharmacol. 2015;79(3):395‐404. doi: 10.1111/bcp.12267  [DOI](https://doi.org/10.1111/bcp.12267) | [PMC free article](/articles/PMC4345950/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25855821/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=Paediatric%20pharmacokinetics:%20key%20considerations&author=HK%20Batchelor&author=JF%20Marriott&volume=79&issue=3&publication_year=2015&pages=395-404&pmid=25855821&doi=10.1111/bcp.12267&)

38. Läll K, Mägi R, Morris A, Metspalu A, Fischer K. Personalized risk prediction for type 2 diabetes: the potential of genetic risk scores. Genet Med. 2017;19(3):322‐329. doi: 10.1038/gim.2016.103  [DOI](https://doi.org/10.1038/gim.2016.103) | [PMC free article](/articles/PMC5506454/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27513194/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genet%20Med&title=Personalized%20risk%20prediction%20for%20type%202%20diabetes:%20the%20potential%20of%20genetic%20risk%20scores&author=K%20L%C3%A4ll&author=R%20M%C3%A4gi&author=A%20Morris&author=A%20Metspalu&author=K%20Fischer&volume=19&issue=3&publication_year=2017&pages=322-329&pmid=27513194&doi=10.1038/gim.2016.103&)

39. Mega JL, Stitziel NO, Smith JG, et al. Genetic risk, coronary heart disease events, and the clinical benefit of statin therapy: an analysis of primary and secondary prevention trials. Lancet. 2015;385(9984):2264‐2271. doi: 10.1016/S0140-6736(14)61730-X  [DOI](https://doi.org/10.1016/S0140-6736(14)61730-X) | [PMC free article](/articles/PMC4608367/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25748612/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Genetic%20risk,%20coronary%20heart%20disease%20events,%20and%20the%20clinical%20benefit%20of%20statin%20therapy:%20an%20analysis%20of%20primary%20and%20secondary%20prevention%20trials&author=JL%20Mega&author=NO%20Stitziel&author=JG%20Smith&volume=385&issue=9984&publication_year=2015&pages=2264-2271&pmid=25748612&doi=10.1016/S0140-6736(14)61730-X&)

40. Inouye M, Abraham G, Nelson CP, et al. Genomic risk prediction of coronary artery disease in 480,000 adults. J Am Coll Cardiol. 2018;72(16):1883‐1893. doi: 10.1016/j.jacc.2018.07.079  [DOI](https://doi.org/10.1016/j.jacc.2018.07.079) | [PMC free article](/articles/PMC6176870/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30309464/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Am%20Coll%20Cardiol&title=Genomic%20risk%20prediction%20of%20coronary%20artery%20disease%20in%20480,000%20adults&author=M%20Inouye&author=G%20Abraham&author=CP%20Nelson&volume=72&issue=16&publication_year=2018&pages=1883-1893&pmid=30309464&doi=10.1016/j.jacc.2018.07.079&)

41. Escott‐Price V, Sims R, Bannister C, et al. Common polygenic variation enhances risk prediction for Alzheimer's disease. Brain. 2015;138(12):3673‐3684. doi: 10.1093/brain/awv268  [DOI](https://doi.org/10.1093/brain/awv268) | [PMC free article](/articles/PMC5006219/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26490334/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Brain&title=Common%20polygenic%20variation%20enhances%20risk%20prediction%20for%20Alzheimer's%20disease&author=V%20Escott%E2%80%90Price&author=R%20Sims&author=C%20Bannister&volume=138&issue=12&publication_year=2015&pages=3673-3684&pmid=26490334&doi=10.1093/brain/awv268&)

42. Escott‐Price V, Nalls MA, Morris HR, et al. Polygenic risk of p arkinson disease is correlated with disease age at onset. Ann Neurol. 2015;77(4):582‐591. doi: 10.1002/ana.24335  [DOI](https://doi.org/10.1002/ana.24335) | [PMC free article](/articles/PMC4737223/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25773351/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Neurol&title=Polygenic%20risk%20of%20p%20arkinson%20disease%20is%20correlated%20with%20disease%20age%20at%20onset&author=V%20Escott%E2%80%90Price&author=MA%20Nalls&author=HR%20Morris&volume=77&issue=4&publication_year=2015&pages=582-591&pmid=25773351&doi=10.1002/ana.24335&)

43. The Breast and Prostate Cancer Cohort Consortium (BPC3), The PRACTICAL (Prostate Cancer Association Group to Investigate Cancer‐Associated Alterations in the Genome) Consortium, The COGS (Collaborative Oncological Gene‐environment Study) Consortium et al. A meta‐analysis of 87,040 individuals identifies 23 new susceptibility loci for prostate cancer. Nat Genet. 2014;46(10):1103‐1109. doi: 10.1038/ng.3094  [DOI](https://doi.org/10.1038/ng.3094) | [PMC free article](/articles/PMC4383163/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25217961/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=A%20meta%E2%80%90analysis%20of%2087,040%20individuals%20identifies%2023%20new%20susceptibility%20loci%20for%20prostate%20cancer&volume=46&issue=10&publication_year=2014&pages=1103-1109&pmid=25217961&doi=10.1038/ng.3094&)

44. Mavaddat N, Pharoah PDP, Michailidou K, et al. Prediction of breast cancer risk based on profiling with common genetic variants. J Natl Cancer Inst. 2015;107(5):djv036. doi: 10.1093/jnci/djv036  [DOI](https://doi.org/10.1093/jnci/djv036) | [PMC free article](/articles/PMC4754625/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25855707/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Natl%20Cancer%20Inst&title=Prediction%20of%20breast%20cancer%20risk%20based%20on%20profiling%20with%20common%20genetic%20variants&author=N%20Mavaddat&author=PDP%20Pharoah&author=K%20Michailidou&volume=107&issue=5&publication_year=2015&pages=djv036&pmid=25855707&doi=10.1093/jnci/djv036&)

45. Thomas M, Sakoda LC, Hoffmeister M, et al. Genome‐wide modeling of polygenic risk score in colorectal cancer risk. Am J Hum Genet. 2020;107(3):432‐444. doi: 10.1016/j.ajhg.2020.07.006  [DOI](https://doi.org/10.1016/j.ajhg.2020.07.006) | [PMC free article](/articles/PMC7477007/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32758450/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Genome%E2%80%90wide%20modeling%20of%20polygenic%20risk%20score%20in%20colorectal%20cancer%20risk&author=M%20Thomas&author=LC%20Sakoda&author=M%20Hoffmeister&volume=107&issue=3&publication_year=2020&pages=432-444&pmid=32758450&doi=10.1016/j.ajhg.2020.07.006&)

46. Khera AV, Chaffin M, Aragam KG, et al. Genome‐wide polygenic scores for common diseases identify individuals with risk equivalent to monogenic mutations. Nat Genet. 2018;50(9):1219‐1224. doi: 10.1038/s41588-018-0183-z  [DOI](https://doi.org/10.1038/s41588-018-0183-z) | [PMC free article](/articles/PMC6128408/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30104762/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Genome%E2%80%90wide%20polygenic%20scores%20for%20common%20diseases%20identify%20individuals%20with%20risk%20equivalent%20to%20monogenic%20mutations&author=AV%20Khera&author=M%20Chaffin&author=KG%20Aragam&volume=50&issue=9&publication_year=2018&pages=1219-1224&pmid=30104762&doi=10.1038/s41588-018-0183-z&)

47. Nishino J, Ochi H, Kochi Y, Tsunoda T, Matsui S. Sample size for successful genome‐wide association study of major depressive disorder. Front Genet. 2018;9:227. doi: 10.3389/fgene.2018.00227  [DOI](https://doi.org/10.3389/fgene.2018.00227) | [PMC free article](/articles/PMC6032046/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30002671/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front%20Genet&title=Sample%20size%20for%20successful%20genome%E2%80%90wide%20association%20study%20of%20major%20depressive%20disorder&author=J%20Nishino&author=H%20Ochi&author=Y%20Kochi&author=T%20Tsunoda&author=S%20Matsui&volume=9&publication_year=2018&pages=227&pmid=30002671&doi=10.3389/fgene.2018.00227&)

48. Sirugo G, Williams SM, Tishkoff SA. The missing diversity in human genetic studies. Cell. 2019;177(1):26‐31. doi: 10.1016/j.cell.2019.02.048  [DOI](https://doi.org/10.1016/j.cell.2019.02.048) | [PMC free article](/articles/PMC7380073/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30901543/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell&title=The%20missing%20diversity%20in%20human%20genetic%20studies&author=G%20Sirugo&author=SM%20Williams&author=SA%20Tishkoff&volume=177&issue=1&publication_year=2019&pages=26-31&pmid=30901543&doi=10.1016/j.cell.2019.02.048&)

49. Popejoy AB, Fullerton SM. Genomics is failing on diversity. Nature. 2016;538(7624):161‐164. doi: 10.1038/538161a  [DOI](https://doi.org/10.1038/538161a) | [PMC free article](/articles/PMC5089703/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27734877/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Genomics%20is%20failing%20on%20diversity&author=AB%20Popejoy&author=SM%20Fullerton&volume=538&issue=7624&publication_year=2016&pages=161-164&pmid=27734877&doi=10.1038/538161a&)
