# HLA-DRB1 is associated with cefaclor-induced immediate hypersensitivity

## Metadata
**Authors:** So-Young Park, So Young Park, Sujin Seo, Hyouk-Soo Kwon, Seung-Hyun Kim, Sae-Hoon Kim, Hye-Kyung Park, Yoon-Seok Chang, Cheol-Woo Kim, Byung Jae Lee, Hae-Sim Park, You Sook Cho, Heung-Bum Oh, David A Ostrov, Sungho Won, Tae Bum Kim
**Journal:** The World Allergy Organization Journal
**Date:** 2024 Apr 13
**DOI:** [10.1016/j.waojou.2024.100901](https://doi.org/10.1016/j.waojou.2024.100901)
**PMID:** 38638799
**PMCID:** PMC11021981
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11021981/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC11021981/pdf/main.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC11021981/pdf/main.pdf)

## Abstract

**Background:** 
Drug-induced hypersensitivity such as anaphylaxis is an important cause of drug-related morbidity and mortality. Cefaclor is a leading cause of drug induced type I hypersensitivity in Korea, but little is yet known about genetic biomarkers to predict this hypersensitivity reaction. We aimed to evaluate the possible involvement of genes in cefaclor induced type I hypersensitivity.

**Methods:** 
Whole exome sequencing (WES) and HLA genotyping were performed in 43 patients with cefaclor induced type I hypersensitivity. In addition, homology modeling was performed to identify the binding forms of cefaclor to HLA site.

**Results:** 
Anaphylaxis was the most common phenotype of cefaclor hypersensitivity (90.69%). WES results show that rs62242177 and rs62242178 located in LIMD1 region were genome-wide significant at the 5 × 10−8 significance level. Cefaclor induced type I hypersensitivity was significantly associated with HLA-DRB1∗04:03 (OR 4.61 [95% CI 1.51–14.09], P < 0.002) and HLA-DRB1∗14:54 (OR 3.86 [95% CI 1.09–13.67], P < 0.002).

**Conclusion:** 
LIMD1, HLA-DRB1∗04:03 and HLA-DRB1∗14:54 may affect susceptibility to cefaclor induced type I hypersensitivity. Further confirmative studies with a larger patient population should be performed to ascertain the role of HLA-DRB1 and LIMD1 in the development of cefaclor induced hypersensitivity.

Keywords: Drug hypersensitivity, Whole exome sequencing, Immediate hypersensitivity, Cephalosporin, Cefaclor

### Background

Drug-induced hypersensitivity such as anaphylaxis is an important cause of drug-related morbidity and mortality. Cefaclor is a leading cause of drug induced type I hypersensitivity in Korea, but little is yet known about genetic biomarkers to predict this hypersensitivity reaction. We aimed to evaluate the possible involvement of genes in cefaclor induced type I hypersensitivity.

### Methods

Whole exome sequencing (WES) and HLA genotyping were performed in 43 patients with cefaclor induced type I hypersensitivity. In addition, homology modeling was performed to identify the binding forms of cefaclor to HLA site.

### Results

Anaphylaxis was the most common phenotype of cefaclor hypersensitivity (90.69%). WES results show that rs62242177 and rs62242178 located in LIMD1 region were genome-wide significant at the 5 × 10^−8^ significance level. Cefaclor induced type I hypersensitivity was significantly associated with HLA-DRB1∗04:03 (OR 4.61 [95% CI 1.51–14.09], P < 0.002) and HLA-DRB1∗14:54 (OR 3.86 [95% CI 1.09–13.67], P < 0.002).

### Conclusion

LIMD1, HLA-DRB1∗04:03 and HLA-DRB1∗14:54 may affect susceptibility to cefaclor induced type I hypersensitivity. Further confirmative studies with a larger patient population should be performed to ascertain the role of HLA-DRB1 and LIMD1 in the development of cefaclor induced hypersensitivity.

## Introduction

Drug hypersensitivity reactions (DHRs) occur by an immunological mechanism, these reactions are heterogeneous and unpredictable.[1](#bib1) DHRs can be classified as immediate and delayed/non-immediate reactions, according to the time from the administration of the drug to the onset of symptoms.[1](#bib1)^,^[2](#bib2) Immediate hypersensitivity usually occurs within 1–6 h of drug administration and is manifested by urticaria, angioedema, bronchospasm, and/or anaphylaxis.[3](#bib3) Early diagnosis and prediction are crucial because DHRs such as anaphylaxis can be life-threatening, unfortunately, there are no screening tools to predict DHRs yet.

Cephalosporins are one of the most widely prescribed antibiotics and with the increase in cephalosporin use in clinics, they are frequently involved in drug hypersensitivity reactions (DHRs).[4](#bib4) Cefaclor is commonly administered in Korea for infection such as soft tissue infections, upper and lower respiratory tract infection, otitis media, urinary tract infections, etc.[5](#bib5) Cefaclor was identified as one of the most common causes of drug-induced anaphylaxis in large-scale data analysis in Korea.[6](#bib6) There have been efforts to find patients at risk of developing hypersensitivity reactions to cephalosporins, but most were limited with respect to retrospectively identifying risk factors such as past history of penicillin or cephalosporin allergy.[7](#bib7) In a prospective study, intradermal skin tests with cephalosporin in subjects without a history of beta lactam allergy were not useful for predicting immediate hypersensitivity.[8](#bib8) Also, a large multicenter cohort study revealed no clinical efficacy of screening intradermal tests at population level.[9](#bib9) In conclusion, there is no accurate tool to predict immediate hypersensitivity to cephalosporins, hence efforts to find susceptible individuals are continuing.

The immune stimulations of DHRs are caused by small molecules such as drugs themselves or their reactive metabolites interacting with proteins.[10](#bib10) In addition, the development of a biomarker that can predict drug hypersensitivity is crucial and genetic biomarkers that predict DHRs are the most studied to date.[11](#bib11)^,^[12](#bib12) In this study, whole exome sequencing (WES) was performed on patients to find proteins expected to be involved in cefaclor hypersensitivity. In addition, HLA typing was performed to find HLA genetic markers expected to be involved in the underlying immune response.

## Methods

### Subjects

We recruited 43 patients with cefaclor hypersensitivity from 6 hospitals in Korea. We reviewed clinical history of all participants by allergy specialists at each center with either positive response to oral provocation test, specific IgE test to cefaclor (ImmunoCAP, PhadiatopTM) or intradermal skin test, and patients who had Type I hypersensitivity reaction to cefaclor were selected as cases. Controls were randomly selected from participants without any history of adverse drug reaction. Written informed consent was obtained from all patients. The study was performed in accordance with the Declaration of Helsinki and the protocols were approved by the institutional review board (IRB number: AMC 2011-0939).

### Whole exome sequencing and discovery analyses

Whole exome sequencing (WES) was generated for 14 cases and 125 controls, 139 subjects in total, to identify single nucleotide variants (SNVs) associated with the immediate hypersensitivity to cefaclor. Hiseq2500 sequencing system (Illumina) was performed for sequencing with 2x101bp read length. Following that, sequencing run was aligned to NCBI b37 using BWA algorithms. The alignment calibration and variant calling were implemented by using GATK, and GRCh37.75 was used for variant annotation. SNVs were filtered out if missing rate >3%, HWE <10^−5^ or MAF <0.1. After the quality control process, 2554 variants for 14 cases and 125 controls remained and they were used for the single variant association analyses.

Association of each SNV with the immediate sensitivity was tested with Fisher's exact test. Population stratification was adjusted by BACON with default options for conjugacy and hypermeters.[13](#bib13) BACON is an extended version of genomic control. Gene-based analyses were conducted by using FUMA.[14](#bib14) Genes with less than or equal to 2 SNVs were excluded and default options for lead variants and candidate variants identification were set except for MHC region. MHC region was included to replicate the association of the HLA genes in gene-based tests.

### HLA genotyping

HLA were genotyped for 43 cases and 159 controls. DNA samples were HLA-typed at the loci of HLA-DP, HLA-DQ and HLA-DR with AVITATM HLA SBT and were analyzed by BIOWITHUSTM SBT Analyzer. Association of HLA genotyped data was analyzed using SPSS (version 18.0, SPSS Inc., Chicago, IL, USA). Fisher's exact test was used to compare the carrier frequencies of HLA alleles between groups and p-values were adjusted using Bonferroni method. In addition to the HLA genotype data, 59 individuals exhibiting immediate hypersensitivity to cefaclor were genotyped using the KoreanChip v1.1. These individuals were subsequently matched with 590 controls.[15](#bib15) Specifically, HLA gene alleles within the MHC locus were imputed using the Michigan Imputation Server, with Eagle v2.4 chosen for phasing.[16](#bib16) Alleles with low imputation quality (Rsq≤0.7) were excluded from further analysis. For each allele, logistic regression was employed utilizing imputed probabilistic dosage genotypes. This approach accounted for imputation uncertainty while controlling for individual age, sex and top 10 principal component scores. A one-tailed test was informed by the directional hypothesis from the analyses with HLA genotyped data.

To synthesize results from the HLA genotyped analysis and HLA imputed analysis, Fisher's method was applied.[17](#bib17) Given the intricate correlation among HLA alleles, the significance level was determined using the Bonferroni adjustment for a 0.05 significance level, as per the Nyholt method.[18](#bib18)

### Homology modeling

The amino acid sequence of HLA-DRB1∗04:03 was used as the basis for homology modeling using the Swiss Modeler. The structure of HLA-DRB1∗04:03 was superimposed on the crystal structure of HLA-DR1. The resulting structure file containing the alpha and beta chains of HLA-DR were used for molecular docking using Autodock Vina.

## Results

### Clinical characteristics of patients with cefaclor induced immediate hypersensitivity

Descriptive statistics for 43 patients are provided in [Supplementary Table 1](#appsec1). The mean age was 45.97 years and 65% of those were females. Most (90.69%) were manifested as anaphylaxis. They presented symptoms of immediate hypersensitivity to cefaclor, and 4 patients among them presented with urticaria and angioedema. The mean of total IgE level and specific IgE level to cefaclor were 2.99 ± 1.21 KU/L, and 242.64 ± 228.27 KU/L, respectively. Oral provocation test was performed for 14 patients and all displayed positive responses. Intradermal skin test was performed for 21 patients and 14 of them displayed positive responses. Detailed results for individual patients are provided in [Supplementary Table 2](#appsec1).

### Discovery analyses with whole exome sequencing

A total of 2554 variants for 14 cases and 125 controls were used for the single variant association analyses. Their quantile-quantile (QQ) and Manhattan plots with the corrected p-values are provided in [Fig. 1](#fig1). Results show that rs62242177 and rs62242178 located in the LIMD1 region were genome-wide significant at the 5 × 10^−8^ significance level ([Table 1](#tbl1)). For gene-based analyses, LIM Domain Containing 1 (LIMD1) gene was significant with the Benjamini-Hochberg adjusted p-values (P = 7.45 × 10^−5^ for LIMD1) ([Fig. 1](#fig1), [Table 2](#tbl2)).[19](#bib19) All the SNV p-values were used for gene-set analysis and one of the biological processes, cytoplasmic mRNA processing body assembly, was found as significant ([Table 3](#tbl3)).

### Fig. 1.

![Fig. 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e9ce/11021981/8049d787ae57/gr1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=11021981_gr1.jpg)

Whole exome sequencing (WES) and gene based analysis with cefaclor hypersensitivity. (a) Q-Q plots and (b) Manhattan plot of WES; (c) Q-Q plots and (d) Manhattan plot of gene-based analysis. The variants in HLA region are highlighted in red.

### Table 1.

| rs ID | Chr | BP | Reference/Alternative allele | MAF | MAF in ALFA | Number of case/controls | Alternative allele frequency in case/control | Fisher's exact test P-value | Adjusted p-value | Gene symbol |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| rs62242177 | 3 | 45677637 | G/T | 0.9496 | 1 | 14/125 | 0.5/1 | 2.96E-16 | 7.35E-11 | LIMD1 |
| rs62242178 | 3 | 45677638 | C/G | 0.9496 | 1 | 14/125 | 0.5/1 | 2.96E-16 | 7.35E-11 | LIMD1 |

Table 1 Caption: Genome-wide significant SNVs with whole exome sequencing (WES)

### Table 2.

| Symbol | Gene | Chr | Start Position | End Position | Number of variants | p-value | Adjusted p-value |
| --- | --- | --- | --- | --- | --- | --- | --- |
| LIMD1 | ENSG00000144791 | 3 | 45596886 | 45727830 | 3 | 4.85E-08 | 7.45E-05 |

Table 2 Caption: Gene-based analysis of LIMD1 with FUMA. P-value was adjusted using Benjamini-Hochberg

### Table 3.

| GO term | Number of SNVs | Beta | Beta STD | SE | P | Pbonferroni |
| --- | --- | --- | --- | --- | --- | --- |
| GO_bp:go_cytoplasmic_mrna_processing_body_assembly | 2 | 4.7696 | 0.172 | 0.9561 | 3.43E-07 | 0.0031 |

Table 3 Caption: Significant gene-set analyzed with FUMA

### HLA association analyses

HLA has been reported to be genome-wide significant and it was further analyzed with WES discovery data and HLA genotyped and HLA imputed data. The gene-based test of HLA class II genes with WES discovery data yielded HLA-DQA1 and HLA-DRB1 significant values with a p-value of 0.0237 and 0.0592, respectively ([Table 4](#tbl4)). HLA-DRB1∗04:03 and HLA-DRB1∗14:54 were more frequently present in patients with cefaclor induced immediate hypersensitivity compared with control subjects (OR 4.61, 95% CI 1.51–14.09, p-value 0.009, meta p-value 0.002 and OR 3.86, 95% CI 1.09–13.67, p-value 0.002, respectively, [Table 5](#tbl5)).

### Table 4.

| Symbol | Gene | Start Position | End Position | Number of variants Included | Z statistics | p-value |
| --- | --- | --- | --- | --- | --- | --- |
| HLA-DQA1 | ENSG00000196735 | 32595956 | 32614839 | 4 | 1.98 | 0.0237 |
| HLA-DRB1 | ENSG00000196126 | 32546546 | 32557625 | 3 | 1.56 | 0.0592 |
| HLA-DQB1 | ENSG00000179344 | 32627244 | 32636160 | 13 | 0.91 | 0.1825 |
| HLA-DRA | ENSG0000020428 | 32407619 | 32412823 | 5 | 0.62 | 0.2682 |

Table 4 Caption: Gene-based analysis of HLA class II genes with WES

### Table 5.

|   | HLA genotype data |  |  |  |  | HLA imputed data |  |  |  |  | Meta p-valuea |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
|   | N | Allele frequency | OR | 95% CI | p-value | N | Allele frequency | Beta | se | p-value |  |
| HLA-DRB1 04:03 |  |  |  |  |  |  |  |  |  |  |  |
| All |  |  |  |  |  |  |  |  |  |  |  |
| Case | 43 | 0.0814 | 4.61 | 1.51–14.09 | 0.009 | 59 | 0.0375 | 1.1586 | 0.6987 | 0.097 | 0.0021b |
| Control | 159 | 0.0189 | 590 | 0.0183 |  |  |  |  |  |  |  |
| Anaphylaxis |  |  |  |  |  |  |  |  |  |  |  |
| Case | 39 | 0.0897 | 5.13 | 1.67–15.72 | 0.006 | 59 | 0.0375 | 1.1586 | 0.6987 | 0.097 | 0.0014b |
| Control | 159 | 0.0189 | 590 | 0.0183 |  |  |  |  |  |  |  |
| Positive specific IgE to cefaclor |  |  |  |  |  |   |  |  |  |  |   |
| Case | 29 | 0.0862 | 4.91 | 1.45–16.65 | 0.016 | 59 | 0.0375 | 1.1586 | 0.6987 | 0.097 | 0.0034b |
| Control | 159 | 0.0189 | 590 | 0.0183 |  |  |  |  |  |  |  |
| HLA-DRB1 14:54 |  |  |  |  |  |  |  |  |  |  |  |
| All |  |  |  |  |  |  |  |  |  |  |  |
| Case | 43 | 0.0581 | 3.86 | 1.09–13.67 | 0.040 | Not available due to the low imputation quality |  |  |  |  |  |
| Control | 159 | 0.0157 |  |  |  |  |  |  |  |  |  |
| Positive specific IgE to cefaclor |  |  |  |  |  |  |  |  |  |  |  |
| Case | 29 | 0.0862 | 5.91 | 1.65–21.10 | 0.010 |  |  |  |  |  |  |
| Control | 159 | 0.0157 |  |  |  |  |  |  |  |  |  |

Table 5 Caption: The result for HLA typed and imputed data.

Subgroup analyses were performed for the HLA allele frequency according to the symptom of hypersensitivity and positivity of specific IgE to cefaclor ([Table 5](#tbl5)). HLA-DRB1∗04:03 was significantly different for the anaphylaxis subgroup (OR 5.13, 95% CI 1.67–15.72, p-value 0.006, meta p-value 0.001). HLA-DRB1∗04:03 and HLA-DRB1∗14:54 were also significantly associated with cefaclor-induced immediate hypersensitivity for the subgroup of patients with positive specific IgE to cefaclor (OR 4.91, 95% CI 1.45–16.65, p-value 0.016, meta p-value 0.003 and OR 5.91, 95% CI 1.65–21.10, p-value 0.010). Haplotype frequencies were also compared and the most significant association was found for HLA-DRB1∗04:03 and HLA-DRB1∗14:54 (p-value <0.001, [Table 6](#tbl6)). We used molecular docking to gain insight in potential intermolecular contacts between cefaclor and HLA molecules. We generated a homology model of HLA-DRB1∗04:03 (SWISS_MODELER) and used molecular docking to predict the top scoring binding orientations in the antigen binding cleft. Cefaclor was predicted to bind the central portion of the HLA-DR molecule, with potential contacts with both the α-chain and β-chains ([Fig. 2](#fig2)). These data are consistent with a model in which the cognate drug interaction with the antigen binding cleft of associated HLA molecules permits drug presentation to responding T cells.

### Table 6.

| Haplotype | Allele | Disease | Control | Combined | p-value∗ |
| --- | --- | --- | --- | --- | --- |
| DRB1-DQB1 | 04:03–03:02 | 0.081 | 0.019 | 0.047 | <0.001 |
| DRB1-DPB1 | 14:54–02:01 | 0.035 | 0 | 0.007 | <0.001 |
| DRB1-DPB1 | 04:03–04:01 | 0.023 | 0 | 0.005 | 0.04 |
| DRB1-DPB1 | 04:03–05:01 | 0.047 | 0.009 | 0.017 | 0.05 |
| DRB1-DQB1-DPB1 | 04:03–03:02–04:01 | 0.023 | 0 | 0.005 | 0.04 |
| DRB1-DQB1-DPB1 | 04:03–03:02–05:01 | 0.047 | 0.009 | 0.017 | 0.04 |
| DRB1-DQB1-DPB1 | 14:54–05:02–02:02 | 0.023 | 0 | 0.005 | <0.0001 |
| DRB1-DQB1-DPB1 | 14:54–05:03–02:01 | 0.023 | 0 | 0.006 | 0.01 |

Table 6 Caption: Analysis of haplotype frequency

### Fig. 2.

![Fig. 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e9ce/11021981/6bebd02897c1/gr2.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=11021981_gr2.jpg)

Cefaclor is predicted to interact with HLA-DRB1∗04:03. A homology model of HLA-DRB1∗04:03 is shown where the molecular surface is colored teal for carbon, blue for nitrogen, red for oxygen and yellow for sulfur.

## Discussion

Anaphylaxis is the most dangerous form of immediate hypersensitivity reaction to drugs and what is more threatening is its unpredictability. To date, no validated screening tools are present that are capable of predicting an individual's susceptibility to such serious adverse reactions to cefaclor. Genetic variation is a major cause of individual differences in the susceptibility to several disorders. In this regard, several genetic association studies related to DHRs have been conducted. Among them, the relationship between the polymorphism of HLA alleles and severe cutaneous adverse reactions (SCAR) has been the most studied; unfortunately, this association is often drug, ethnic, and phenotype-specific.[12](#bib12)^,^[20](#bib20)

From this study, we found that HLA-DRB1∗0403 was more frequently present in patients with cefaclor induced anaphylaxis, suggesting the possibility of genetic predisposition to having severe hypersensitivity reactions to cefaclor. The HLA-DRB1 gene encodes the beta chain of major histocompatibility complex (MHC) class II, and MHC class II molecules are expressed on antigen presenting cells (B cells, macrophages, and dendritic cells) and activate CD4^+^ helper T cells.[21](#bib21) In many studies so far, HLA-DRB1 has been reported in relation to drug hypersensitivity, especially our previous study revealed genetic variation in HLA-DRB1 associated with drug hypersensitivity in Korea.[15](#bib15) In another study, HLA-DRB1∗1302 was related to aspirin-induced urticaria/angioedema in Korea.[22](#bib22)

There are few studies that have conducted WES in patients with DHRs. WES provides more than 95% of the exons, which contains 85% of disease-causing mutations in many diseases.[23](#bib23) In our WES analysis, LIMD1 was significantly associated with DHRs. Further research on the expression and function of this protein in DHRs should be conducted, however, LIMD1 is known as a member of the Zyxin family proteins and functions as a tumor suppressor.[24](#bib24)^,^[25](#bib25)

The development of a biomarker that can predict drug hypersensitivity is very important and genetic biomarkers that predict DHRs are the most studied to date.[11](#bib11)^,^[12](#bib12) Since HLA interacts with T cell receptors, HLA genetic studies have been conducted focusing on SCAR, a delayed type hypersensitivity reaction caused by the underlying T cell immune reactions. However, recent studies have provided evidence of HLA alleles as predisposing factors for immediate reactions. It has been reported that HLA-DRB1∗10:01 is related to beta-lactam-induced immediate hypersensitivity reactions.[26](#bib26) In addition, HLA-DRB1∗07:01 was associated with a asparaginase hypersensitivity.[27](#bib27) Although the exact immunological mechanisms have not been elucidated, we speculate that interaction between B cells and T cells via HLA class II trigger the immunoglobulin switching that leads to the generation of specific Ig E antibodies.[26](#bib26)

This study has limitations. First, the sample size is extremely small. A large number of subjects may be useful in increasing the accuracy of WES. However, it is difficult to obtain samples from patients presenting typical clinical manifestations associated with a certain causative drug. Second, The expression of phenotypes among the subjects may vary. This fact could potentially weaken the observed allele frequency and associations with actual allergies. Additionally, the absence of an optimal testing strategy to definitively ascertain aminocephalosporin allergies is recognized as a broader limitation in the allergy field when conducting such studies. To mitigate these limitations, we performed oral provocation tests, skin test, and serum specific IgE test to establish the most accurate diagnosed and phenotypes possible. Third, we could not perform replication analysis or functional analysis. We should conduct additional research that supports concrete results.[28](#bib28) Finally, while our study identified association between specific alleles and the drug hypersensitivity, the rarity of the alleles may limit our explanatory power. The low frequency of these alleles within the study population may imply that their contribution to the observed effects may be modest. This brings forth the need for caution in interpreting to these rare alleles at this juncture, and additional research is needed to explore a broader spectrum of alleles and their potential associations.

Nevertheless, this study is significant in that it is the first attempt to find a genetic biomarker associated with cefaclor immediate-type hypersensitivity in Korea. We have for the first time reported an association of HLA-DRB1∗04:03 with cefaclor Type I hypersensitivity reactions. In particular, this genetic marker could be developed as a clinically useful tool for predicting the occurrence of cefaclor hypersensitivity reactions. Unlike previous studies, we report the associated genes that may be involved in cefaclor hypersensitivity in Korea, and this may have great significance in the clinical field for the efficient diagnosis and prediction of cefaclor hypersensitivity.

## Abbreviations

WES, whole exome sequencing; HLA, human leukocyte antigen; DHRs, drug hypersensitivity reactions; SNVs, single nucleotide variants; LIMD1, LIM Domain Containing 1; SCAR, severe cutaneous adverse reactions; MHC, major histocompatibility complex

## Funding information

This research was supported by the Bio & Medical Technology Development Program of the National Research Foundation (NRF) funded by the Korean government (MSIT) (2019M3E5D3073365).

## Availability of data and materials

The data used or analyzed during the study is available upon reasonable request from the corresponding author.

## Author contributions

All authors have contributed to the content of the manuscript, and the respective roles of each author are as follows: Study conception, design, image assessment, and supervision: Tae-Bum Kim and Sungho Won. Study design, review of experiments, data analysis and interpretation, image assessment, and writing of the draft: So-Young Park, So Young Park, and Tae-Bum Kim. Study design, mainly conducted experiments, data analysis and interpretation: So-Young Park, So Young Park, Sujin Seo, Sungho Won, and Tae-Bum Kim.

Review of image assessment, and data analysis and interpretation: So-Young Park, Hyouk-Soo Kwon, Seung-Hyun Kim, Sae-Hoon Kim, Heung-Bum Oh, David A. Ostrov, and Tae-Bum Kim. Review of study conception and supervision: Hye-Kyung Park, Yoon-Seok Chang, Cheol-Woo Kim, Byung Jae Lee, Hae-Sim Park, and You Sook Cho.

## Ethics approval

All patients provided written informed consent at the time of cohort enrollment, and this study was approved by the institutional review boards of Asan Medical Center (AMC 2011-0939).

## Consent for publication

The manuscript's publishing is approved by all of the authors.

## Declaration of competing interest

The authors have no conflict of interest relevant to this article to disclose.

## Footnotes

## Contributor Information

Sungho Won, Email: sunghow@gmail.com.

Tae Bum Kim, Email: tbkim@amc.seoul.kr.

## Appendix A. Supplementary data

The following is the Supplementary data to this article.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

The data used or analyzed during the study is available upon reasonable request from the corresponding author.

### Supplementary Materials

### Data Availability Statement

The data used or analyzed during the study is available upon reasonable request from the corresponding author.

## References

1. Wilkerson R.G. Drug hypersensitivity reactions. Emerg. Med. Clin. North Am. 2022;40:39–55. doi: 10.1016/j.emc.2021.09.001.  [DOI](https://doi.org/10.1016/j.emc.2021.09.001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34782090/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Emerg.%20Med.%20Clin.%20North%20Am.&title=Drug%20hypersensitivity%20reactions&author=R.G.%20Wilkerson&volume=40&publication_year=2022&pages=39-55&pmid=34782090&doi=10.1016/j.emc.2021.09.001&)

2. Gruchalla R.S. Clinical assessment of drug-induced disease. Lancet. 2000;356:1505–1511. doi: 10.1016/S0140-6736(00)02885-3.  [DOI](https://doi.org/10.1016/S0140-6736(00)02885-3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11081549/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Clinical%20assessment%20of%20drug-induced%20disease&author=R.S.%20Gruchalla&volume=356&publication_year=2000&pages=1505-1511&pmid=11081549&doi=10.1016/S0140-6736(00)02885-3&)

3. Romano A., Guéant-Rodriguez R.M., Viola M., et al. Diagnosing immediate reactions to cephalosporins. Clin Exp Allergy. 2005;35:1234–1242. doi: 10.1111/j.1365-2222.2005.02317.x.  [DOI](https://doi.org/10.1111/j.1365-2222.2005.02317.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16164453/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Exp%20Allergy&title=Diagnosing%20immediate%20reactions%20to%20cephalosporins&author=A.%20Romano&author=R.M.%20Gu%C3%A9ant-Rodriguez&author=M.%20Viola&volume=35&publication_year=2005&pages=1234-1242&pmid=16164453&doi=10.1111/j.1365-2222.2005.02317.x&)

4. Blanca M., Torres M.J., Mayorga C., et al. Immediate allergic reactions to betalactams: facts and controversies. Curr Opin Allergy Clin Immunol. 2004;4:261–266. doi: 10.1097/01.all.0000136764.74065.15.  [DOI](https://doi.org/10.1097/01.all.0000136764.74065.15) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15238790/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Opin%20Allergy%20Clin%20Immunol&title=Immediate%20allergic%20reactions%20to%20betalactams:%20facts%20and%20controversies&author=M.%20Blanca&author=M.J.%20Torres&author=C.%20Mayorga&volume=4&publication_year=2004&pages=261-266&pmid=15238790&doi=10.1097/01.all.0000136764.74065.15&)

5. Kim Y.A., Park Y.S., Youk T., Lee H., Lee K. Changes in antimicrobial usage patterns in Korea: 12-year analysis based on database of the national health insurance service-national sample cohort. Sci Rep. 2018;8 doi: 10.1038/s41598-018-30673-6.  [DOI](https://doi.org/10.1038/s41598-018-30673-6) | [PMC free article](/articles/PMC6093866/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30111796/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sci%20Rep&title=Changes%20in%20antimicrobial%20usage%20patterns%20in%20Korea:%2012-year%20analysis%20based%20on%20database%20of%20the%20national%20health%20insurance%20service-national%20sample%20cohort&author=Y.A.%20Kim&author=Y.S.%20Park&author=T.%20Youk&author=H.%20Lee&author=K.%20Lee&volume=8&publication_year=2018&pmid=30111796&doi=10.1038/s41598-018-30673-6&)

6. Ahn K.M., Kim B.K., Yang M.S. Risk factors of anaphylaxis in Korea: identifying drug-induced anaphylaxis culprits using big data. Medicine (Baltim) 2022;101 doi: 10.1097/MD.0000000000030224.  [DOI](https://doi.org/10.1097/MD.0000000000030224) | [PMC free article](/articles/PMC9439811/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36107532/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Medicine%20(Baltim)&title=Risk%20factors%20of%20anaphylaxis%20in%20Korea:%20identifying%20drug-induced%20anaphylaxis%20culprits%20using%20big%20data&author=K.M.%20Ahn&author=B.K.%20Kim&author=M.S.%20Yang&volume=101&publication_year=2022&pmid=36107532&doi=10.1097/MD.0000000000030224&)

7. Madaan A., Li J.T. Cephalosporin allergy. Immunol. Allergy Clin. North Am. 2004;24:463–476. doi: 10.1016/j.iac.2004.03.009. (vi-vii)  [DOI](https://doi.org/10.1016/j.iac.2004.03.009) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15242721/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Immunol.%20Allergy%20Clin.%20North%20Am.&title=Cephalosporin%20allergy&author=A.%20Madaan&author=J.T.%20Li&volume=24&publication_year=2004&pages=463-476&pmid=15242721&doi=10.1016/j.iac.2004.03.009&)

8. Yoon S.Y., Park S.Y., Kim S., et al. Validation of the cephalosporin intradermal skin test for predicting immediate hypersensitivity: a prospective study with drug challenge. Allergy. 2013;68:938–944. doi: 10.1111/all.12182.  [DOI](https://doi.org/10.1111/all.12182) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23751142/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Allergy&title=Validation%20of%20the%20cephalosporin%20intradermal%20skin%20test%20for%20predicting%20immediate%20hypersensitivity:%20a%20prospective%20study%20with%20drug%20challenge&author=S.Y.%20Yoon&author=S.Y.%20Park&author=S.%20Kim&volume=68&publication_year=2013&pages=938-944&pmid=23751142&doi=10.1111/all.12182&)

9. Yang M.S., Kang D.Y., Seo B., et al. Incidence of cephalosporin-induced anaphylaxis and clinical efficacy of screening intradermal tests with cephalosporins: a large multicenter retrospective cohort study. Allergy. 2018;73:1833–1841. doi: 10.1111/all.13435.  [DOI](https://doi.org/10.1111/all.13435) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29517808/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Allergy&title=Incidence%20of%20cephalosporin-induced%20anaphylaxis%20and%20clinical%20efficacy%20of%20screening%20intradermal%20tests%20with%20cephalosporins:%20a%20large%20multicenter%20retrospective%20cohort%20study&author=M.S.%20Yang&author=D.Y.%20Kang&author=B.%20Seo&volume=73&publication_year=2018&pages=1833-1841&pmid=29517808&doi=10.1111/all.13435&)

10. Pichler W.J. The important role of non-covalent drug-protein interactions in drug hypersensitivity reactions. Allergy. 2022;77:404–415. doi: 10.1111/all.14962.  [DOI](https://doi.org/10.1111/all.14962) | [PMC free article](/articles/PMC9291849/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34037262/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Allergy&title=The%20important%20role%20of%20non-covalent%20drug-protein%20interactions%20in%20drug%20hypersensitivity%20reactions&author=W.J.%20Pichler&volume=77&publication_year=2022&pages=404-415&pmid=34037262&doi=10.1111/all.14962&)

11. Klaewsongkram J., Sukasem C., Thantiworasit P., et al. Analysis of HLA-B allelic variation and IFN-γ ELISpot responses in patients with severe cutaneous adverse reactions associated with drugs. J Allergy Clin Immunol Pract. 2019;7:219–227.e4. doi: 10.1016/j.jaip.2018.05.004.  [DOI](https://doi.org/10.1016/j.jaip.2018.05.004) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29800753/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%C2%A0Allergy%20Clin%20Immunol%20Pract&title=Analysis%20of%20HLA-B%20allelic%20variation%20and%20IFN-%CE%B3%20ELISpot%20responses%20in%20patients%20with%20severe%20cutaneous%20adverse%20reactions%20associated%20with%20drugs&author=J.%20Klaewsongkram&author=C.%20Sukasem&author=P.%20Thantiworasit&volume=7&publication_year=2019&pages=219-227.e4&pmid=29800753&doi=10.1016/j.jaip.2018.05.004&)

12. Pirmohamed M., Ostrov D.A., Park B.K. New genetic findings lead the way to a better understanding of fundamental mechanisms of drug hypersensitivity. J Allergy Clin Immunol. 2015;136:236–244. doi: 10.1016/j.jaci.2015.06.022.  [DOI](https://doi.org/10.1016/j.jaci.2015.06.022) | [PMC free article](/articles/PMC4534769/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26254050/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%C2%A0Allergy%20Clin%20Immunol&title=New%20genetic%20findings%20lead%20the%20way%20to%20a%20better%20understanding%20of%20fundamental%20mechanisms%20of%20drug%20hypersensitivity&author=M.%20Pirmohamed&author=D.A.%20Ostrov&author=B.K.%20Park&volume=136&publication_year=2015&pages=236-244&pmid=26254050&doi=10.1016/j.jaci.2015.06.022&)

13. van Iterson M., van Zwet E.W., Heijmans B.T. Controlling bias and inflation in epigenome- and transcriptome-wide association studies using the empirical null distribution. Genome Biol. 2017;18:19. doi: 10.1186/s13059-016-1131-9.  [DOI](https://doi.org/10.1186/s13059-016-1131-9) | [PMC free article](/articles/PMC5273857/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28129774/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Biol&title=Controlling%20bias%20and%20inflation%20in%20epigenome-%20and%20transcriptome-wide%20association%20studies%20using%20the%20empirical%20null%20distribution&author=M.%20van%20Iterson&author=E.W.%20van%20Zwet&author=B.T.%20Heijmans&volume=18&publication_year=2017&pages=19&pmid=28129774&doi=10.1186/s13059-016-1131-9&)

14. Watanabe K., Taskesen E., van Bochoven A., Posthuma D. Functional mapping and annotation of genetic associations with FUMA. Nat Commun. 2017;8:1826. doi: 10.1038/s41467-017-01261-5.  [DOI](https://doi.org/10.1038/s41467-017-01261-5) | [PMC free article](/articles/PMC5705698/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29184056/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&title=Functional%20mapping%20and%20annotation%20of%20genetic%20associations%20with%20FUMA&author=K.%20Watanabe&author=E.%20Taskesen&author=A.%20van%20Bochoven&author=D.%20Posthuma&volume=8&publication_year=2017&pages=1826&pmid=29184056&doi=10.1038/s41467-017-01261-5&)

15. Park S.Y., Do A.R., Park T., Won S., Kim T.B. Genome-wide association study identified a novel genetic variation in HLA-DRB1 associated with drug hypersensitivity. Ann Allergy Asthma Immunol. 2022;128:335–337. doi: 10.1016/j.anai.2021.11.022.  [DOI](https://doi.org/10.1016/j.anai.2021.11.022) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34864189/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Allergy%20Asthma%20Immunol&title=Genome-wide%20association%20study%20identified%20a%20novel%20genetic%20variation%20in%20HLA-DRB1%20associated%20with%20drug%20hypersensitivity&author=S.Y.%20Park&author=A.R.%20Do&author=T.%20Park&author=S.%20Won&author=T.B.%20Kim&volume=128&publication_year=2022&pages=335-337&pmid=34864189&doi=10.1016/j.anai.2021.11.022&)

16. Das S., Forer L., Schönherr S., et al. Next-generation genotype imputation service and methods. Nat Genet. 2016;48:1284–1287. doi: 10.1038/ng.3656.  [DOI](https://doi.org/10.1038/ng.3656) | [PMC free article](/articles/PMC5157836/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27571263/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Next-generation%20genotype%20imputation%20service%20and%20methods&author=S.%20Das&author=L.%20Forer&author=S.%20Sch%C3%B6nherr&volume=48&publication_year=2016&pages=1284-1287&pmid=27571263&doi=10.1038/ng.3656&)

17. Fisher R.A. Springer; 1970. Statistical Methods for Research Workers. Breakthroughs in Statistics: Methodology and Distribution; pp. 66–70.  [Google Scholar](https://scholar.google.com/scholar_lookup?Fisher%20R.A.%20Springer;%201970.%20Statistical%20Methods%20for%20Research%20Workers.%20Breakthroughs%20in%20Statistics:%20Methodology%20and%20Distribution;%20pp.%2066%E2%80%9370.)

18. Nyholt D.R. A simple correction for multiple testing for single-nucleotide polymorphisms in linkage disequilibrium with each other. Am J Hum Genet. 2004;74:765–769. doi: 10.1086/383251.  [DOI](https://doi.org/10.1086/383251) | [PMC free article](/articles/PMC1181954/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14997420/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=A%C2%A0simple%20correction%20for%20multiple%20testing%20for%20single-nucleotide%20polymorphisms%20in%20linkage%20disequilibrium%20with%20each%20other&author=D.R.%20Nyholt&volume=74&publication_year=2004&pages=765-769&pmid=14997420&doi=10.1086/383251&)

19. Sun W., Reich B.J., Cai T.T., Guindani M., Schwartzman A. False discovery control in large-scale spatial multiple testing. J R Stat. Soc. Series B Stat. Methodol. 2015;77:59–83. doi: 10.1111/rssb.12064.  [DOI](https://doi.org/10.1111/rssb.12064) | [PMC free article](/articles/PMC4310249/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25642138/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%C2%A0R%20Stat.%20Soc.%20Series%20B%20Stat.%20Methodol.&title=False%20discovery%20control%20in%20large-scale%20spatial%20multiple%20testing&author=W.%20Sun&author=B.J.%20Reich&author=T.T.%20Cai&author=M.%20Guindani&author=A.%20Schwartzman&volume=77&publication_year=2015&pages=59-83&pmid=25642138&doi=10.1111/rssb.12064&)

20. Fan W.L., Shiao M.S., Hui R.C., et al. HLA association with drug-induced adverse reactions. J. Immunol. Res. 2017;2017 doi: 10.1155/2017/3186328.  [DOI](https://doi.org/10.1155/2017/3186328) | [PMC free article](/articles/PMC5733150/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29333460/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%C2%A0Immunol.%20Res.&title=HLA%20association%20with%20drug-induced%20adverse%20reactions&author=W.L.%20Fan&author=M.S.%20Shiao&author=R.C.%20Hui&volume=2017&publication_year=2017&pmid=29333460&doi=10.1155/2017/3186328&)

21. Chung W.H., Hung S.I., Chen Y.T. Human leukocyte antigens and drug hypersensitivity. Curr Opin Allergy Clin Immunol. 2007;7:317–323. doi: 10.1097/ACI.0b013e3282370c5f.  [DOI](https://doi.org/10.1097/ACI.0b013e3282370c5f) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17620823/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Opin%20Allergy%20Clin%20Immunol&title=Human%20leukocyte%20antigens%20and%20drug%20hypersensitivity&author=W.H.%20Chung&author=S.I.%20Hung&author=Y.T.%20Chen&volume=7&publication_year=2007&pages=317-323&pmid=17620823&doi=10.1097/ACI.0b013e3282370c5f&)

22. Kim S.H., Choi J.H., Lee K.W., et al. The human leucocyte antigen-DRB1∗1302-DQB1∗0609-DPB1∗0201 haplotype may be a strong genetic marker for aspirin-induced urticaria. Clin Exp Allergy. 2005;35:339–344. doi: 10.1111/j.1365-2222.2004.02197.x.  [DOI](https://doi.org/10.1111/j.1365-2222.2004.02197.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15784113/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Exp%20Allergy&title=The%20human%20leucocyte%20antigen-DRB1%E2%88%971302-DQB1%E2%88%970609-DPB1%E2%88%970201%20haplotype%20may%20be%20a%20strong%20genetic%20marker%20for%20aspirin-induced%20urticaria&author=S.H.%20Kim&author=J.H.%20Choi&author=K.W.%20Lee&volume=35&publication_year=2005&pages=339-344&pmid=15784113&doi=10.1111/j.1365-2222.2004.02197.x&)

23. Rabbani B., Tekin M., Mahdieh N. The promise of whole-exome sequencing in medical genetics. J Hum Genet. 2014;59:5–15. doi: 10.1038/jhg.2013.114.  [DOI](https://doi.org/10.1038/jhg.2013.114) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24196381/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%C2%A0Hum%20Genet&title=The%20promise%20of%20whole-exome%20sequencing%20in%20medical%20genetics&author=B.%20Rabbani&author=M.%20Tekin&author=N.%20Mahdieh&volume=59&publication_year=2014&pages=5-15&pmid=24196381&doi=10.1038/jhg.2013.114&)

24. Zhou J., Zhang L., Zhou W., Chen Y., Cheng Y., Dong J. LIMD1 phosphorylation in mitosis is required for mitotic progression and its tumor-suppressing activity. FEBS J. 2019;286:963–974. doi: 10.1111/febs.14743.  [DOI](https://doi.org/10.1111/febs.14743) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30600590/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=FEBS%20J&title=LIMD1%20phosphorylation%20in%20mitosis%20is%20required%20for%20mitotic%20progression%20and%20its%20tumor-suppressing%20activity&author=J.%20Zhou&author=L.%20Zhang&author=W.%20Zhou&author=Y.%20Chen&author=Y.%20Cheng&volume=286&publication_year=2019&pages=963-974&pmid=30600590&doi=10.1111/febs.14743&)

25. Wang L., Sparks-Wallace A., Casteel J.L., Howell M.E.A., Ning S. Algorithm-based meta-analysis reveals the mechanistic interaction of the tumor suppressor LIMD1 with non-small-cell lung carcinoma. Front Oncol. 2021;11 doi: 10.3389/fonc.2021.632638.  [DOI](https://doi.org/10.3389/fonc.2021.632638) | [PMC free article](/articles/PMC8044451/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33869018/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front%20Oncol&title=Algorithm-based%20meta-analysis%20reveals%20the%20mechanistic%20interaction%20of%20the%20tumor%20suppressor%20LIMD1%20with%20non-small-cell%20lung%20carcinoma&author=L.%20Wang&author=A.%20Sparks-Wallace&author=J.L.%20Casteel&author=M.E.A.%20Howell&author=S.%20Ning&volume=11&publication_year=2021&pmid=33869018&doi=10.3389/fonc.2021.632638&)

26. Nicoletti P., Carr D.F., Barrett S., et al. Beta-lactam-induced immediate hypersensitivity reactions: a genome-wide association study of a deeply phenotyped cohort. J Allergy Clin Immunol. 2021;147:1830–1837.e15. doi: 10.1016/j.jaci.2020.10.004.  [DOI](https://doi.org/10.1016/j.jaci.2020.10.004) | [PMC free article](/articles/PMC8100096/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33058932/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%C2%A0Allergy%20Clin%20Immunol&title=Beta-lactam-induced%20immediate%20hypersensitivity%20reactions:%20a%20genome-wide%20association%20study%20of%20a%20deeply%20phenotyped%20cohort&author=P.%20Nicoletti&author=D.F.%20Carr&author=S.%20Barrett&volume=147&publication_year=2021&pages=1830-1837.e15&pmid=33058932&doi=10.1016/j.jaci.2020.10.004&)

27. Fernandez C.A., Smith C., Yang W., et al. HLA-DRB1∗07:01 is associated with a higher risk of asparaginase allergies. Blood. 2014;124:1266–1276. doi: 10.1182/blood-2014-03-563742.  [DOI](https://doi.org/10.1182/blood-2014-03-563742) | [PMC free article](/articles/PMC4141516/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24970932/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=HLA-DRB1%E2%88%9707:01%20is%20associated%20with%20a%20higher%20risk%20of%20asparaginase%20allergies&author=C.A.%20Fernandez&author=C.%20Smith&author=W.%20Yang&volume=124&publication_year=2014&pages=1266-1276&pmid=24970932&doi=10.1182/blood-2014-03-563742&)

28. Gueant J.L., Romano A., Cornejo-Garcia J.A., et al. HLA-DRA variants predict penicillin allergy in genome-wide fine-mapping genotyping. J Allergy Clin Immunol. 2015;135:253–259. doi: 10.1016/j.jaci.2014.07.047.  [DOI](https://doi.org/10.1016/j.jaci.2014.07.047) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25224099/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%C2%A0Allergy%20Clin%20Immunol&title=HLA-DRA%20variants%20predict%20penicillin%20allergy%20in%20genome-wide%20fine-mapping%20genotyping&author=J.L.%20Gueant&author=A.%20Romano&author=J.A.%20Cornejo-Garcia&volume=135&publication_year=2015&pages=253-259&pmid=25224099&doi=10.1016/j.jaci.2014.07.047&)
