# Dual single‑nucleotide polymorphism biomarker combination for opioid selection to treat cancer pain

## Metadata
**Authors:** Yoshihiko Fujita, Hiromichi Matsuoka, Yasutaka Chiba, Junji Tsurutani, Takeshi Yoshida, Kiyohiro Sakai, Miki Nakura, Ryo Sakamoto, Chihiro Makimura, Yoichi Ohtake, Kaoru Tanaka, Hidetoshi Hayashi, Takayuki Takahama, Junko Tanizaki, Atsuko Koyama, Kazuto Nishio, Kazuhiko Nakagawa
**Journal:** Molecular and Clinical Oncology
**Date:** 2024 Nov 29
**DOI:** [10.3892/mco.2024.2809](https://doi.org/10.3892/mco.2024.2809)
**PMID:** 39720462
**PMCID:** PMC11667419
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11667419/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC11667419/pdf/mco-22-02-02809.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC11667419/pdf/mco-22-02-02809.pdf)

## Abstract

We have been exploring biomarkers that could help physicians select the appropriate opioid for individualized treatment of cancer pain. Recently, we identified a single nucleotide polymorphism (SNP) of CCL11 (rs17809012) as one such biomarker that was significantly associated with the analgesic effect of morphine. The current study measured the plasma concentrations of chemokines/cytokines in pre-treatment plasma samples of a total of 138 patients who were randomized to receive morphine (n=70) or oxycodone (n=68). Based on the results, one cytokine, IL-16, was identified whose plasma concentrations showed a clear bias between patients with cancer pain who responded well and responded poorly to oxycodone. A genotypic analysis also identified a SNP of the IL-16 gene (rs4778889) as being significantly associated with the analgesic effect of oxycodone. Whether both of the SNPs identified as being significant (CCL11 rs17809012 and IL-16 rs4778889) could be used in combination to accurately predict which opioid might be the most suitable to provide pain relief in patients with cancer was assessed. Morphine tended to provide superior analgesic effect over oxycodone in patients with the IL-16 rs4778889 TT genotype and the CCL11 rs17809012 AG/GG genotype (n=45), while a trend towards a greater analgesic effect of oxycodone was observed in patients with other genotype combinations of these SNPs (n=93) (P=0.0012 for the interaction), suggesting that the IL-16 rs4778889 and CCL11 rs17809012 SNPs could serve as a potential dual-biomarker combination for personalized analgesic therapy in patients with cancer pain.

Keywords: plasma concentration, single nucleotide polymorphism, genotype, morphine, oxycodone, CCL11, IL-16, cancer pain

## Introduction

Recently, single nucleotide polymorphisms (SNPs) have begun to be considered as potential biomarkers to select the appropriate opioid for use in patients with cancer pain. A *CYP2D6* SNP (rs1065852) has been shown to affect the metabolism of several opioids (e.g., codeine and tramadol) and been proposed as a predictive biomarker for selection of the most suitable opioid for treating cancer pain ([1-3](#b1-MCO-22-2-02809)). The µ-opioid receptor gene (*OPRM1*) has a SNP that could allow individualization of pain treatment based on the predicted response. For G-allele carriers of this SNP, tapentadol and methadone may be more suitable than hydromorphone, oxycodone or fentanyl ([4](#b4-MCO-22-2-02809)). In Japan, morphine and oxycodone are the most frequently used opioids, although there is still a lack of consensus on which of the two would be the better choice in individual patients ([5](#b5-MCO-22-2-02809)). Since the sensitivity to and side effects of opioids vary widely among patients, we also attempted to identify SNPs of genes that could potentially predict differences in the responses of patients to morphine and oxycodone. To identify the most appropriate biomarker SNP(s) for predicting the efficacy of each opioid, we conducted a randomized controlled trial, the RELIEF study (Trial registration number: UMIN000015579; date of registration: November 4, 2014, patients recruited: November 2014 to February 2020 in Kindai University Hospital, Osaka, Japan.), in which we randomized a total of 138 patients (1:1) to receive either morphine (Group M) or oxycodone (Group O), based on the *COMT* rs4680 SNP as a biomarker; we identified several candidate SNPs in this trial from among the SNPs that have previously been suggested as possibly being linked to pain sensitivity and/or opioid efficacy ([6-8](#b6-MCO-22-2-02809)). Based on further screening, we identified a SNP, *CCL11* rs17809012, as having the potential to predict the response to morphine ([6](#b6-MCO-22-2-02809)). We assessed the analgesic response in each patient on a numerical rating scale (NRS) for pain. The ∆NRS, defined as the difference between the NRS score recorded before the start of opioid treatment and that recorded after opioid dose titration, was smaller (namely, the degree of pain relief was smaller) in patients with the *CCL11* rs17809012 AA genotype treated with morphine [least square mean (LSM) for ∆NRS, 2.33] as compared with that in the AA+oxycodone (LSM 3.48), AG/GG+morphine (LSM 3.58), and AG/GG+oxycodone (LSM 3.16) groups ([6](#b6-MCO-22-2-02809)). These results suggest that the *CCL11* rs17809012 SNP could be a predictive biomarker for the effect of morphine.

In regard to the key mechanisms underlying chronic pain, it has come to be increasingly accepted that chemokines (such as CCL11) and cytokines serve as major mediators that activate glial cell interactions with neurons ([9](#b9-MCO-22-2-02809),[10](#b10-MCO-22-2-02809)). Therefore, we sought to explore additional biomarkers, besides the *CCL11* rs17809012 SNP, from among the 39 chemokines/cytokines included as analytes in the Bio-Plex Pro Human Chemokine assay kit used by us. In this study, we measured the plasma concentrations of these 39 chemokines/cytokines in pre-treatment plasma samples collected from a total of 138 patients enrolled in the RELIEF study, and found that one cytokine, IL-16, showed a bias in plasma concentrations between patients who responded well and responded poorly to oxycodone. Moreover, genetic analysis also showed that rs4778889 SNP genotype residing in the *IL-16* gene may allow discrimination between patients who responded well and responded poorly to oxycodone. Based on these findings of our current and previous studies, we propose that the dual-biomarker combination of *CCL11* rs17809012 and *IL-16* rs4778889 SNPs could be useful to accurately guide selection of the more appropriate opioid between morphine and oxycodone in individual patients with cancer pain.

## Patients and methods

### Patients and samples

We enrolled a total of 138 patients with advanced malignancies based on our eligibility criteria in the RELIEF study, a randomized controlled trial recently conducted by us (Trial registration number: UMIN000015579) ([8](#b8-MCO-22-2-02809)). Our cohort did not include any subjects whose families could influence the genotype independency. In the present study, we measured the plasma concentrations of chemokines/cytokines in pre-treatment plasma samples of the study subjects and performed genotyping of the *IL-16* SNPs in their DNA samples. The 138 patients who fulfilled the eligibility criteria of suffering from cancer pain that necessitated daily treatment with opioids were randomized to either morphine (Group M; n=70) or oxycodone (Group O; n=68) ([Fig. 1](#f1-MCO-22-2-02809)).

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/576e/11667419/dda861e4c63c/mco-22-02-02809-g00.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=11667419_mco-22-02-02809-g00.jpg)

Schema of the study design. IL-16, interleukin 16; CCL11, C-C motif chemokine ligand 11; NRS, numerical rating scale; SNP, single-nucleotide polymorphism.

Calculation of the optimal study sample size and the inclusion and exclusion criteria have been described in our previous trial report ([8](#b8-MCO-22-2-02809)). The baseline characteristics of the 138 patients are presented in [Table I](#tI-MCO-22-2-02809).

### Table I.

|   |  | Morphine group (n=70) |  |  | Oxycodone group (n=68) |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- |
| Required dose | Number of patients | Low (n=41) | High (n=29) | P-value | Low (n=53) | High (n=15) | P-value |
| Age (years), n (%)a |   |   |   | 0.052 |   |   | 1 |
| <70 | 62 | 23 (56.1) | 9 (31.0) |   | 23 (43.4) | 7 (46.7) |   |
| ≥70 | 76 | 18 (43.9) | 20 (69.0) |   | 30 (56.6) | 8 (53.3) |   |
| Sex, n (%)a |   |   |   | 0.225 |   |   | 0.382 |
| Males | 79 | 22 (53.7) | 20 (69.0) |   | 27 (50.9) | 10 (66.7) |   |
| Females | 59 | 19 (46.3) | 9 (31.0) |   | 26 (49.1) | 5 (33.3) |   |
| Performance status, n (%)a,c |   |   |   | 1 |   |   | 1 |
| 0 | 10 | 3 (7.3) | 0 (0) |   | 3 (5.7) | 4 (26.7) |   |
| 1 | 80 | 23 (56.1) | 18 (62.1) |   | 31 (58.5) | 6 (40.0) |   |
| 2 | 35 | 11 (26.8) | 7 (24.1) |   | 15 (28.3) | 3 (20.0) |   |
| 3 | 10 | 3 (7.3) | 3 (10.3) |   | 2 (3.8) | 2 (13.3) |   |
| 4 | 4 | 1 (2.4) | 1 (3.5) |   | 2 (3.8) | 0 (0) |   |
| Pre-NRS score, mean (SD)b |   | 4.9 (2.0) | 5.9 (2.1) | 0.031 | 5.0 (1.9) | 7.0 (1.6) | 0.0006 |
| HADS score, mean (SD)b |   | 15.3 (7.5) | 16.6 (7.9) | 0.5 | 14.5 (7.3) | 15.5 (6.9) | 0.641 |
| SF-MPQ-2 score, mean (SD)b |   | 38.9 (28.5) | 44.2 (31.2) | 0.459 | 52.1 (40.2) | 75.1 (36.4) | 0.055 |

Table Caption: Baseline characteristics of the patients.

### Opioid administration and dose titration

The opioid-naïve patients were started on treatment with an intermediate-release (IR) opioid, according to the guidelines for opioid use and titration (NCCN Guidelines™, Adult Cancer Pain) ([11](#b11-MCO-22-2-02809),[12](#b12-MCO-22-2-02809)), by specialized palliative care physicians. Opioid titration on day 1 following onset of cancer pain has been described in detail in a previous report ([6](#b6-MCO-22-2-02809)). In brief, the minimum standard starting dose of an IR opioid, that is, 5 mg for morphine and 2.5 mg for oxycodone, is administered repeatedly to the patients until a decrease of the score on an NRS for pain (0=no pain to 10=maximal pain) by ≥33% or by ≤3 is recorded post-titration (day 1) as compared with the score recorded prior to the start of opioid treatment ([6](#b6-MCO-22-2-02809)). Classification of the patients according to the required opioid dose (high-dose group/low-dose group) for each opioid was as defined previously ([6](#b6-MCO-22-2-02809),[8](#b8-MCO-22-2-02809)); namely, patients requiring ≥10 mg of IR morphine, or ≥7.5 mg of IR oxycodone were classified into the high-dose group, while those requiring 5 mg of IR morphine or 5 mg or less of IR oxycodone were classified into the low-dose group.

### Measurement of the plasma chemokine/cytokine concentrations

Plasma samples of the patients were collected on day 1 prior to the start of treatment (pre-treatment samples) using a Venoject II vacuum blood-collecting tube (Terumo). The blood samples were centrifuged for 10 min at 1,200 g, and the supernatants (pre-treatment plasma samples) were frozen immediately and stored at -80˚C until use. The concentrations of the 39 chemokine/cytokines listed in [Table II](#tII-MCO-22-2-02809) were measured in the pre-treatment plasma samples of the patients using a BioPlex 200 System (Bio-Rad Laboratories), in accordance with the manufacturer's protocols. Levels of one of the cytokines (GM-CSF) included as an analyte in the kit were omitted from the analysis, because only 29 out of the 138 patients had detectable amounts of this cytokine in the plasma.

### Table II.

| Items | Non-abbreviated form | Median concentrationa |
| --- | --- | --- |
| CCL21 | C-C Motif Chemokine Ligand 21 | 3426.4 |
| CXCL13 | C-X-C Motif Chemokine Ligand 13 | 38.2 |
| CCL27 | C-C Motif Chemokine Ligand 27 | 950.2 |
| CXCL5 | C-X-C Motif Chemokine Ligand 5 | 1048.1 |
| CCL11 | C-C Motif Chemokine Ligand 11 | 95.1 |
| CCL24 | C-C Motif Chemokine Ligand 24 | 68.6 |
| CCL26 | C-C Motif Chemokine Ligand 26 | 15.0 |
| CX3CL1 | C-X3-C Motif Chemokine Ligand 1 | 154.6 |
| CXCL6 | C-X-C Motif Chemokine Ligand 6 | 55.8 |
| CXCL1 | C-X-C Motif Chemokine Ligand 1 | 186.7 |
| CXCL2 | C-X-C Motif Chemokine Ligand 2 | 291.4 |
| CCL1 | C-C Motif Chemokine Ligand 1 | 58.3 |
| IFN-γ | Interferon-gamma | 2.4 |
| IL-1β | Interleukin-1 beta | 16.3 |
| IL-2 | Interleukin-2 | 10.1 |
| IL-4 | Interleukin-4 | 15.9 |
| IL-6 | Interleukin-6 | 21.8 |
| IL-8 | Interleukin-8 | 29.7 |
| IL-10 | Interleukin-10 | 25.7 |
| IL-16 | Interleukin-16 | 323.2 |
| CXCL10 | C-X-C Motif Chemokine Ligand 10 | 72.0 |
| CXCL11 | C-X-C Motif Chemokine Ligand 11 | 343.9 |
| CCL2 | C-C Motif Chemokine Ligand 2 | 24.3 |
| CCL8 | C-C Motif Chemokine Ligand 8 | 22.1 |
| CCL7 | C-C Motif Chemokine Ligand 7 | 71.9 |
| CCL13 | C-C Motif Chemokine Ligand 13 | 27.0 |
| CCL22 | C-C Motif Chemokine Ligand 22 | 441.0 |
| MIF | Macrophage Migration Inhibitory Factor | 689.9 |
| CXCL9 | C-X-C Motif Chemokine Ligand 9 | 410.0 |
| CCL3 | C-C Motif Chemokine Ligand 3 | 7.3 |
| CCL15 | C-C Motif Chemokine Ligand 15 | 3458.4 |
| CCL20 | C-C Motif Chemokine Ligand 20 | 17.4 |
| CCL19 | C-C Motif Chemokine Ligand 19 | 109.8 |
| CCL23 | C-C Motif Chemokine Ligand 23 | 285.4 |
| CXCL16 | C-X-C Motif Chemokine Ligand 16 | 424.2 |
| CXCL12 | C-X-C Motif Chemokine Ligand 12 | 607.7 |
| CCL17 | C-C Motif Chemokine Ligand 17 | 88.7 |
| CCL25 | C-C Motif Chemokine Ligand 25 | 435.1 |
| TNF-α | Tumor Necrosis Factor-alpha | 23.2 |

Table Caption: Chemokines/cytokines measured for plasma concentrations.

### Genotyping

Genomic DNA was isolated from the blood samples, as described previously ([13](#b13-MCO-22-2-02809)). Genotyping was performed for 2 SNPs (rs4778889 and rs11556218) of the *IL-16* gene (Interleukin 16, Gene ID: 3603) and a SNP (rs17809012) of the *CCL11* gene (C-C Motif Chemokine 11, Gene ID: 6356) using a PCR-based Taqman SNP Genotyping Assay, in accordance with the manufacturer's instructions (Thermo Fisher Scientific, Inc.).

### Statistical analysis

The differences in the required dose (high or low) were estimated for each opioid using Fisher's exact test for categorical variables or Mann-Whitney U test for ordinal data ([Table I](#tI-MCO-22-2-02809)). To screen for chemokines/cytokines relevant to the effects of the opioids, patients were divided into high- and low-concentration groups for each analyte according to its plasma concentration using the cutoff value that had been defined as the median concentrations for all patients ([Table II](#tII-MCO-22-2-02809)). The ∆NRS, defined as the difference in the score on an NRS for pain (hereinafter, NRS score) before the start of opioid treatment on day 1 and after opioid titration (day 1), was used as the dependent variable for comparing between the high- and low-concentration groups for each analyte using simple regression analyses. In addition to the analytes, the independent variables considered were the age (<70/≥70 years), sex, performance status score (1/≥2), pre-NRS score ([1-10](#b1-MCO-22-2-02809)), total score on the HADS (Hospital Anxiety and Depression Scale) ([14](#b14-MCO-22-2-02809)), total score on the SF-MPQ-2 (Short-Form McGill Pain Questionnaire-2) ([7](#b7-MCO-22-2-02809)), and the required drug dose (high or low), among which the pre-NRS, HADS, and SF-MPQ-2 scores were ordinal variables.

For the analysis in the genotypic study of the SNPs, we characterized the SNPs of *CCL11* rs17809012, *IL-16* rs4778889 and *IL-16* rs11556218 by performing simple regression analyses separately for Group M and Group O. As the *CCL11* rs17809012 SNPs had already been characterized for 135 patients in our previous study ([6](#b6-MCO-22-2-02809)), we additionally analyzed this SNP for the 3 remaining patients in this study. ∆NRS was set as the dependent variable.

We also analyzed the three SNPs (*CCL11* rs17809012, *IL-16* rs4778889 and *IL-16* rs11556218) in the entire subject population (n=138), adding ‘treatment (morphine or oxycodone)’ as the independent variable in place of ‘dose’, which was omitted due to the incompatible dosage forms between the two opioids ([8](#b8-MCO-22-2-02809)). We analyzed data from the entire subject population by a simple regression analysis and a multiple regression analysis with adjustments for confounding variables. The variance inflation factor (VIF) was used to diagnose problems of multicollinearity. P<0.05 was set as denoting statistical significance. The analyses were performed using the JMP statistical software (v14.2; SAS Institute).

## Results

### Screening for chemokines/cytokines with predictive potential for opioid effect

Out of the 39 cytokines/chemokines, our simple regression analyses identified one candidate predictive factor, IL-16, as a cytokine whose plasma concentrations were significantly correlated with the effect of oxycodone ([Table SI](#SD1-MCO-22-2-02809)).

We also examined the concentration-treatment interactions for the ∆NRS. A forest plot was constructed based on the estimate (relative risk) with its 95% CI in the 2 concentration groups (low and high) for each analyte ([Fig. 2](#f2-MCO-22-2-02809)). Better efficacy with oxycodone was observed in patients with plasma IL-16 concentrations in the lower half of the concentration range, while morphine was more effective in patients with IL-16 concentrations in the upper half of the concentration range (P value for interaction=0.02).

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/576e/11667419/0ca5d7e03080/mco-22-02-02809-g01.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=11667419_mco-22-02-02809-g01.jpg)

Forest plots comparing the ∆NRS for the two treatments according to the plasma concentrations (low and high) of 39 chemokines/cytokines. The dotted line represents the regression coefficient (estimate) for treatment in the overall subject population. ∆NRS, difference in the score on the numerical rating scale for pain recorded prior to the start of opioid treatment and after opioid titration. CI, confidence interval; L, concentrations (pg/ml) lower than the median values for each cytokine; H, concentrations (pg/ml) higher than the median values for each cytokine; NRS, numerical rating scale.

### Genotyping study

Next, we focused on the SNPs residing in the *IL-16* gene. We selected the rs4778889 and rs11556218 SNPs, which have been identified previously as being functional ([15-18](#b15-MCO-22-2-02809)). As suggested before ([19](#b19-MCO-22-2-02809)), these two SNPs were found to be closely related. Patients with the major genotype of rs4777889 (TT) exclusively showed the major genotype of rs11556218 (TT). Meanwhile, patients with the minor allele (C) of rs4777889 had either the major allele (T) or minor allele (G) of rs11556218 ([Table SII](#SD2-MCO-22-2-02809)). These results suggest that the minor allele in rs11556218 emerged in an *IL-16* gene with the minor nucleotide (C) in rs4777889 that is more ancestral, and linkage disequilibrium was evident between the two SNPs loci (~9.3 kbp) in our cohort (d'=0.999).

We first confirmed if these SNPs were linked to the analgesic effect of oxycodone, as the analgesic effect of oxycodone differed between patients with higher and lower plasma concentrations of IL-16 ([Table SI](#SD1-MCO-22-2-02809)), and the expression levels of the *IL-16* gene could be modulated by these *IL-16* SNPs. A simple regression analysis for Group O showed that the *IL-16* rs4777889 and *IL-16* rs11556218 genotypes were correlated with the ∆NRS. The ∆NRS values in the patients who were homozygous for the major allele (*IL-16* rs4777889 TT and *IL-16* rs11556218 TT) were significantly lower by 0.45 and 0.44, respectively, on average, as compared with the ∆NRS values in patients who were carrying the minor alleles (*IL-16* rs4777889 TC/CC and *IL-16* rs11556218 TG/GG; P=0.027 and 0.034, respectively) ([Table III](#tIII-MCO-22-2-02809)). In contrast, for patients of Group M, the ∆NRS value was lower by 0.56 in the patients who were homozygous for the major allele of *CCL11* (rs17809012 AA) as compared with that in patients who were carrying the minor allele [rs17809012 (AG/GG)] (P=0.019) ([Table III](#tIII-MCO-22-2-02809)) ([6](#b6-MCO-22-2-02809)). These results confirmed that the rs4777889 (or rs11556218) and rs17809012 SNPs could be specific biomarkers to predict the analgesic effects of oxycodone and morphine, respectively.

### Table III.

|   | Morphine (n=70) |  |  |  | Oxycodone (n=68) |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Variable | β | t-value | Partial regression coefficient (95% CI) | P-value | β | t-value | Partial regression coefficient (95% CI) | P-value |
| Age | -0.02 | -0.19 | 0.85 (-0.53 to 0.43) | 0.851 | 0.04 | 0.30 | 0.76 (-0.33 to 0.44) | 0.764 |
| Sex | -0.03 | -0.22 | -0.05 (-0.54 to 0.44) | 0.828 | -0.12 | -0.95 | -0.18 (-0.57 to 0.21) | 0.345 |
| Performance status | -0.07 | -0.60 | -0.15 (5-0.64 to 0.3) | 0.549 | 0.22 | 1.84 | 0.36 (-0.03 to 0.75) | 0.070 |
| Pre-NRS score | 0.58 | 5.81 | 0.55 (0.36 to 0.74) | <0.0001 | 0.60 | 6.11 | 0.47 (0.32 to 0.63) | <0.0001 |
| HADS score | 0.07 | 0.59 | 0.02 (-0.04 to 0.08) | 0.555 | 0.15 | 1.19 | 0.03 (-0.02 to 0.09) | 0.237 |
| SF-MPQ-2 Total score | 0.26 | 2.26 | 0.02 (-0.00 to 0.03) | 0.027 | 0.29 | 2.42 | 0.01 (0.002 to 0.02) | 0.018 |
| Dose | -0.16 | -1.38 | -0.33 (-0.81 to 0.15) | 0.174 | -0.15 | -1.22 | -0.28 (-0.74 to 0.18) | 0.228 |
| Genotype |   |   |   |   |   |   |   |   |
| IL-16 (rs4778889) | -0.01 | -0.10 | -0.03 (-0.52 to 0.47) | 0.918 | -0.27 | -2.27 | -0.45 (-0.84 to -0.05) | 0.027 |
| IL-16 (rs11556218) | -0.17 | -1.42 | -0.42 (-1.01 to 0.17) | 0.161 | -0.26 | -2.17 | -0.44 (-0.84 to -0.03) | 0.034 |
| CCL11 (rs17809012) | -0.28 | -2.41 | -0.56 (-1.01 to -0.10) | 0.019 | 0.11 | 0.86 | 0.16 (-0.22 to 0.55) | 0.394 |

Table Caption: Simple regression analyses to identify the SNP determinants of the ΔNRS on day 1 in the morphine and oxycodone groups.

Regardless of the opioids that were used in the overall subject population, these SNPs appeared to affect the ∆NRS, although analysis using a simple regression model revealed that the differences between the *IL-16* rs4777889 genotype groups were statistically insignificant ([Table SIII](#SD3-MCO-22-2-02809)). We also performed a multiple linear regression analysis with adjustments for the age, sex, ps, pre-NRS score, treatment used, genotype, and total scores on the HADS and SF-MPQ-2, which still revealed significant differences of the ∆NRS between the *CCL11* rs17809012 genotype groups (difference in ∆NRS between the genotype groups=0.25 with P=0.049), but not between the *IL-16* rs4777889 genotype groups (difference in ∆NRS between the genotype groups=0.14, with P=0.286) ([Table SIV](#SD4-MCO-22-2-02809), multiple regression model 1). We did not select *IL-16* rs4777889 and *IL-16* rs11556218 SNPs as covariables at the same time, because these SNPs with linkage disequilibrium highly confounded each other (with VIFs of 2.85 and 2.98, respectively; data not shown). However, the analysis using *CCL11* rs17809012 and *IL-16* rs11556218 SNPs as covariables showed significant differences of the ∆NRS between both the *CCL11* rs17809012 genotype groups and *IL-16* rs11556218 genotype groups (differences in ∆NRS between these genotype groups=0.28 and 0.31, with P=0.028 and 0.040, respectively; [Table SIV](#SD4-MCO-22-2-02809), multiple regression model 2), and there seemed to be no strong confounding variables, with uniformly low VIF values for all SNPs (<1.5) for both multiple regression models 1 and 2 shown in [Table SIV](#SD4-MCO-22-2-02809).

### Predictive factors for opioid selection

Next, we examined the genotype-treatment interactions influencing the ∆NRS. The LSM for ∆NRS for each genotype-treatment interaction was calculated based on the results of the multiple regression analysis ([Table SV](#SD5-MCO-22-2-02809)). A significant interaction (P=0.018) was observed between the *CCL11* genotype and treatment ([Table SV](#SD5-MCO-22-2-02809) and [Fig. 3](#f3-MCO-22-2-02809)) ([6](#b6-MCO-22-2-02809)), while no such interaction was observed between the *IL-16* rs4777889 or *IL-16* rs11556218 genotype and treatment ([Table SV](#SD5-MCO-22-2-02809) and [Fig. 3](#f3-MCO-22-2-02809)).

### Figure 3.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/576e/11667419/7fb9c238a218/mco-22-02-02809-g02.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=11667419_mco-22-02-02809-g02.jpg)

Forest plots comparing the ∆NRS for the two treatments according to the genotypes of three SNPs (IL-16_rs4778889, IL-16_rs11556218 and CCL11_rs17809012) and the genotype combinations (I-IV for each of the IL-16/CCL11 SNP combinations). The dotted line represents the regression coefficient (estimate) for treatment in the overall subject population. ∆NRS, difference in the score on the numerical rating scale for pain recorded prior to the start of opioid treatment and after opioid titration. NRS, numerical rating scale; SNP, single-nucleotide polymorphism; CI, confidence interval; Others, patients with the other genotype combinations; IL-16, interleukin 16; CCL11, C-C motif chemokine ligand 11.

Four combinations of genotypes can be generated from the two SNPs, *IL-16*_rs4777889/*CCL11*_rs17809012, i.e. i) TT/AA, ii) TT/(AG+GG), iii) (TC+CC)/AA, iv) (TC+CC)/(AG+GG). We also analyzed the interactions influencing the ∆NRS between each of the 4 genotype combinations and treatment ([Table SV](#SD5-MCO-22-2-02809) and [Fig. 3](#f3-MCO-22-2-02809)). Significant interaction (P=0.001) was detected when comparison was conducted between patients with ‘TT/AG+GG’ (n=45) and others (n=93) (rs4778889/rs17809012, combination II; [Table SV](#SD5-MCO-22-2-02809) and [Fig. 3](#f3-MCO-22-2-02809)). As shown in [Table IV](#tIV-MCO-22-2-02809), the LSM of the ∆NRS was 4.00 for the Group M patients with the TT/AG+GG genotype, which was higher by 1.4 than the LSM in the Group M patients with the remaining genotypes. In contrast, the LSM of the ∆NRS was 2.85 for the Group O patients with the TT/AG+GG genotype, which was lower by 0.68 than the LSM for the Group O patients with the remaining genotypes. No significant genotype-treatment interactions were observed for any of the other genotype combinations ([Fig. 3](#f3-MCO-22-2-02809) and [Table IV](#tIV-MCO-22-2-02809)). A similar analysis was performed for another *IL-16* SNP (rs11556218), and we detected a weaker significance level of the genotype-treatment interaction for the ∆NRS (P=0.022; [Fig. 3](#f3-MCO-22-2-02809)).

### Table IV.

| Variablea | Groupb | n | LSM | Standard error | 95% CI |
| --- | --- | --- | --- | --- | --- |
| IL-16 (rs4778889)*treatment | TT + morphine | 43 | 3.02 | 0.27 | 2.48-3.56 |
|   | TC/CC + morphine | 27 | 3.07 | 0.35 | 2.39-3.76 |
|   | TT + oxycodone | 45 | 3.03 | 0.27 | 2.50-3.56 |
|   | TC/CC + oxycodone | 23 | 3.83 | 0.37 | 3.09-4.57 |
| CCL11 (rs17809012)*treatment | AA + morphine | 34 | 2.47 | 0.30 | 1.87-3.07 |
|   | AG/GG + morphine | 36 | 3.58 | 0.29 | 3.00-4.17 |
|   | AA + oxycodone | 38 | 3.45 | 0.29 | 2.88-4.01 |
|   | AG/GG + oxycodone | 30 | 3.12 | 0.32 | 2.48-3.75 |
| (rs4778889/rs17809012)*treatment | (TT/AA) + morphine | 21 | 2.29 | 0.39 | 1.52-3.05 |
|   | (Others) + morphine | 49 | 3.37 | 0.25 | 2.87-3.87 |
|   | (TT/AA) + oxycodone | 22 | 3.23 | 0.38 | 2.48-3.98 |
|   | (Others) + oxycodone | 46 | 3.34 | 0.26 | 2.82-3.86 |
| (rs4778889/rs17809012)*treatment | [TT/(AG/GG)] + morphine | 22 | 4.00 | 0.37 | 3.27-4.73 |
|   | (Others) + morphine | 48 | 2.60 | 0.25 | 2.11-3.10 |
|   | [TT/(AG/GG)] + oxycodone | 23 | 2.85 | 0.36 | 2.13-3.56 |
|   | (Others) + oxycodone | 45 | 3.53 | 0.26 | 3.02-4.05 |
| (rs4778889/rs17809012)*treatment | [(TC/CC)/AA] + morphine | 13 | 2.77 | 0.50 | 1.78-3.76 |
|   | (Others) + morphine | 57 | 3.11 | 0.24 | 2.63-3.58 |
|   | [(TC/CC)/AA] + oxycodone | 16 | 3.75 | 0.45 | 2.86-4.64 |
|   | (Others) + oxycodone | 52 | 3.16 | 0.25 | 2.67-3.66 |
| (rs4778889/rs17809012)*treatment | [(TC/CC)/(AG/GG)] + morphine | 14 | 2.93 | 0.48 | 1.97-3.88 |
|   | (Others) + morphine | 56 | 3.07 | 0.24 | 2.59-3.55 |
|   | [(TC/CC)/(AG/GG)] + oxycodone | 7 | 4.00 | 0.68 | 2.65-5.35 |
|   | (Others) + oxycodone | 61 | 3.22 | 0.23 | 2.76-3.68 |

Table Caption: LSMs of ∆NRS for patients (n=138) in terms of interaction between their genotype (combination) and treatment.

## Discussion

In the previous study, we confirmed three SNPs (*TRPV1* rs222749, *CCL11* rs17809012, *HNMT* rs1050891) as being involved in the analgesic effect of morphine. Out of the three, we found that the *CCL11* rs17809012 SNPs could be a potential biomarker to guide selection of the more suitable opioid between morphine and oxycodone for treating cancer pain ([6](#b6-MCO-22-2-02809)). Patients of Group M with the major *CCL11* rs17809012 genotype (AA) showed a significantly reduced ∆NRS (P=0.006), suggesting that oxycodone should be preferred for patients with this genotype of *CCL11* to obtain better pain relief. However, for the patients with the minor allele of the rs17809012 (AG/GG), morphine appeared to be a better choice, but this interaction was not statistically significant (P=0.358) ([6](#b6-MCO-22-2-02809)).

In the current study, we used pre-treatment plasma samples of patients to screen for chemokines/cytokines with potential value as biomarker(s) to guide opioid selection. From among the 39 chemokines/cytokines measured, we identified only the plasma concentrations of IL-16 as possibly having the potential to predict the analgesic effect of oxycodone. Analysis of the interaction between the plasma concentration of IL-16 and treatment for the ∆NRS showed that patients with plasma IL-16 concentrations in the lower half of the range of concentrations responded significantly better to oxycodone treatment.

We next focused on several SNPs residing in the *IL-16* gene. We selected two (rs4778889 and rs11556218 SNPs), which have been identified previously as being functional ([15-18](#b15-MCO-22-2-02809)). For both SNPs, in the Group O patients, homozygosity for the major allele (TT for both SNPs) was associated with a reduced ∆NRS, implying a lower analgesic effect, as compared with the genotypes including at least one minor allele (C for rs4778889 or G for rs11556218). These minor alleles may be linked to low plasma concentrations (expression levels) of IL-16, because in the current study, we showed that the effect of oxycodone in the Group O patients was significantly better in those with lower concentrations of IL-16. This result, however, appeared to be inconsistent with the finding of Burkart *et al* ([20](#b20-MCO-22-2-02809)), who reported a several-fold increased expression of *IL-16* associated with the minor allele (C) as compared with the major allele (T) of rs4778889, which is putatively located in the promoter region of the human *IL-16* gene ([21](#b21-MCO-22-2-02809)). They used a luciferase reporter assay in an *in vitro* experiment, which may not have accurately reproduced the status *in vivo* ([22](#b22-MCO-22-2-02809),[23](#b23-MCO-22-2-02809)). Indeed, a positive association has been reported between the rs4778889 TT genotype and IL-16 expression levels in Crohn's disease ([24](#b24-MCO-22-2-02809)) and Grave's diseases ([25](#b25-MCO-22-2-02809)), which are consistent with our results. Further analyses may be required to clarify this issue.

The rs11556218 SNP, another *IL-16* SNP located on exon 6 of the gene, can result in an asparagine to lysine substitution in the IL-16 protein. This substitution may alter the protein structure and function ([26](#b26-MCO-22-2-02809)), but whether it affects the pain perception or sensitivity remains largely unknown, although it has been reported to be associated with an elevated risk of development of gastric cancer, colorectal cancer and osteosarcoma ([16](#b16-MCO-22-2-02809),[17](#b17-MCO-22-2-02809)). We found that this SNP was in linkage disequilibrium with rs4778889. Thus, a trend towards a reduced ∆NRS value associated with homozygosity for the major allele (TT for both SNPs) as compared with heterozygosity or homozygosity for the minor allele was observed for both the SNPs in the Group O patients ([Table III](#tIII-MCO-22-2-02809)). When the interaction between treatment and the genotype for the ∆NRS was analyzed, the trends associated with the *IL-16* rs4778889 genotypes were found to be stronger because the combination of the rs4778889 TT genotype with the *CCL11* rs17809012 AG/GG genotype was associated with a higher effect of morphine (P=0.034), while no such association was observed for combination of the rs11556218 TT genotype with the *CCL11* rs17809012 AG/GG genotype (P=0.22) ([Fig. 3](#f3-MCO-22-2-02809)). Thus, the Group M patients with the rs4778889 TT genotype and rs17809012 AG/GG genotype showed an LSM for the ∆NRS of 4.00, which was more than 1.00 higher than LSMs in the Group M patients with the other rs4778889/rs17809012 genotype combinations and the Group O patients with the same genotype combination ([Fig. 4](#f4-MCO-22-2-02809)). In contrast, the Group O patients with the genotype combinations other than rs4778889 TT/rs17809012 (AG/GG) showed an LSM for the ∆NRS that was about 1.00 higher than LSMs in the Group M patients with the respective genotype combinations ([Fig. 4](#f4-MCO-22-2-02809)).

### Figure 4.

![Figure 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/576e/11667419/6b8097f998b7/mco-22-02-02809-g03.jpg)

Significance of the ∆NRS in patients treated with morphine (blue) or oxycodone (orange) is expressed by the LSMs ± standard error separately for the genotype combinations (rs4778889/rs17809012) of: i) TT/AA; ii) TT/(AG/GG); iii) (TC/CC)/AA; and iv) (TC/CC)/(AG/GG). ∆NRS, difference in the score on the numerical rating scale for pain recorded prior to the start of opioid treatment and after opioid titration; Numbers in parentheses indicate the numbers of patients. LSMs, least square means; NRS, numerical rating scale.

SNPs of drug transporters, metabolizing enzymes, and opioid receptors known to modulate the pharmacokinetic and pharmacodynamic effects of opioids have been suggested as potentially useful biomarkers for aiding in opioid selection for patients with cancer pain ([1-3](#b1-MCO-22-2-02809)). To the best of our knowledge, this study, as an extension of our previous study ([6](#b6-MCO-22-2-02809)), is the first to show that cytokine or chemokine SNPs could also be used to choose between opioids (morphine or oxycodone). Our previous study suggested that the CCL11 rs17809012 SNP could be a biomarker that could predict the effect of morphine ([6](#b6-MCO-22-2-02809)). However, our current study demonstrated that two SNPs (i.e. CCL11 rs17809012 and IL-16 rs4778889) in combination could significantly predict the effect of both opioids and, therefore, enhance the validity of the choice (morphine or oxycodone) further than the use of CCL11 rs17809012 alone.

IL-16 is considered as being a proinflammatory cytokine, and by binding to its receptor (CD4), it promotes the secretion of inflammatory cytokines, such as TNF-α, IL-1β and IL-6([11](#b11-MCO-22-2-02809)). IL-6 is also known as a pronociceptive cytokine, like CCL11([27](#b27-MCO-22-2-02809)). These cytokines/chemokines form networks that induce nociceptive and neuropathic pains. How these networks contribute to the pathogenesis of cancer pain, characterized by a mixed-mechanism pain state, is still unknown. Our findings regarding the interactions between the cytokines (or chemokines) and opioids may be expected to pave the way towards elucidation of the mechanisms of cancer pain and its treatment.

Our study had some limitations. Cancer pain is widely known to be an inflammatory response mediated by complex interactions among many cytokines and chemokines. These humoral factors are induced in different ways depending on the cancer type, grade and stage. We enrolled subjects with a variety of cancer types in the study ([Table SVI](#SD6-MCO-22-2-02809)), however, classification of patients into these categories could not be performed due to our small sample size and lack of data, which could have introduced some biases. Second, we detected a positive relationship between the genotype and plasma levels for IL-16 in this study, but unexpectedly, this was not the case for CCL11. While we measured the plasma concentrations of CCL11, we observed no relationship of the plasma concentrations of CCL11 with the treatment effect in this study, unlike the case for the *CCL11* genotype, which showed a significant correlation with the treatment effect ([6](#b6-MCO-22-2-02809)). This divergence could weaken the reliability of our findings; however, a positive relationship between the plasma concentrations and the genotype may not necessarily be observed if the genotype is not linked to regulation of the gene expression but to other biological function(s) of the encoded protein.

In conclusion, we found two biomarker SNPs that can be used in combination to guide treatment selection between morphine and oxycodone for the treatment of cancer pain. Patients with the *IL-16* rs4778889 TT genotype and *CCL11* rs17809012 AG/GG genotype may be expected to benefit from treatment with morphine, while patients with the remaining genotype combinations could be expected to benefit from treatment with oxycodone, both of which are significant. Nucleotide sequencing of these two SNP regions can be readily performed in patients with cancer pain, so that physicians can have the option of selecting the more effective opioid for individual patients with cancer pain, a new therapeutic concept that warrants further clinical evaluation.

## Supplementary Material

## Acknowledgements

The authors would like to thank Mrs. Mami Kitano, Mrs. Haruka Sakamoto, Mrs. Yume Shinkai (all from the Department of Medical Oncology, Kindai University Faculty of Medicine, Osaka, Japan) and Dr Masato Terashima (Department of Genome Biology, Kindai University Faculty of Medicine, Osaka, Japan) for genomic DNA isolation.

## Funding Statement

**Funding:** This study was financially supported by the Health Labor Sciences Research Grant (Grant for Innovative Clinical Cancer Research: H26-Innovative Cancer-General-056; grant no. 16ck0106059h0003) and the Japan Agency for Medical Research and Development (Innovative Clinical Cancer Research; grant no. 17ck0106328h0001).

## Availability of data and materials

The data generated in the present study may be requested from the corresponding author.

## Authors' contributions

YF, HM, YC, JTs, AK, KNi and KNa designed the study. YF performed the experiments and collected the data. HM, JTs, TY, KS, MN, RS, CM, YO, KT, HH, TT and JTa collected the clinical data. YF, HM, YC, JTs and TY analyzed and interpreted the data. YF and HM drafted the manuscript. YF, HM, YC, JTs, AK, KNi and KNa revised the manuscript critically. YF, HM and JTs confirm the authenticity of all the raw data. All the authors have read and approved the final manuscript.

## Ethics approval and consent for participation

The study was conducted according to the guidelines of the Declaration of Helsinki and the Japanese ethical guidelines for clinical research with the approval of the Ethical Committee of Kindai University Faculty of Medicine (approval no. 26-130). Written informed consent was obtained from all participants involved in the study.

## Patient consent for publication

The publication of data was approved by all patients.

## Competing interests

The authors declare that they have no competing interests.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

The data generated in the present study may be requested from the corresponding author.

### Supplementary Materials

### Data Availability Statement

The data generated in the present study may be requested from the corresponding author.

## References

1. Owusu Obeng A, Hamadeh I, Smith M. Review of opioid pharmacogenetics and considerations for pain management. Pharmacotherapy. 2017;37:1105–1121. doi: 10.1002/phar.1986.  [DOI](https://doi.org/10.1002/phar.1986) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28699646/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacotherapy&title=Review%20of%20opioid%20pharmacogenetics%20and%20considerations%20for%20pain%20management&author=A%20Owusu%20Obeng&author=I%20Hamadeh&author=M%20Smith&volume=37&publication_year=2017&pages=1105-1121&pmid=28699646&doi=10.1002/phar.1986&)

2. Crews KR, Monte AA, Huddart R, Caudle KE, Kharasch ED, Gaedigk A, Dunnenberger HM, Leeder JS, Callaghan JT, Samer CF, et al. Clinical pharmacogenetics implementation consortium guideline for CYP2D6, OPRM1, and COMT genotypes and select opioid therapy. Clin Pharmacol Ther. 2021;110:888–896. doi: 10.1002/cpt.2149.  [DOI](https://doi.org/10.1002/cpt.2149) | [PMC free article](/articles/PMC8249478/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33387367/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Clinical%20pharmacogenetics%20implementation%20consortium%20guideline%20for%20CYP2D6,%20OPRM1,%20and%20COMT%20genotypes%20and%20select%20opioid%20therapy&author=KR%20Crews&author=AA%20Monte&author=R%20Huddart&author=KE%20Caudle&author=ED%20Kharasch&volume=110&publication_year=2021&pages=888-896&pmid=33387367&doi=10.1002/cpt.2149&)

3. Vieira CMP, Fragoso RM, Pereira D, Medeiros R. Pain polymorphisms and opioids: An evidence-based review. Mol Med Rep. 2019;19:1423–1434. doi: 10.3892/mmr.2018.9792.  [DOI](https://doi.org/10.3892/mmr.2018.9792) | [PMC free article](/articles/PMC6390004/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30592275/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Med%20Rep&title=Pain%20polymorphisms%20and%20opioids:%20An%20evidence-based%20review&author=CMP%20Vieira&author=RM%20Fragoso&author=D%20Pereira&author=R%20Medeiros&volume=19&publication_year=2019&pages=1423-1434&pmid=30592275&doi=10.3892/mmr.2018.9792&)

4. Takemura M, Niki K, Okamoto Y, Kawamura T, Kohno M, Matsuda Y, Ikeda K. Comparison of the effects of OPRM1 A118G polymorphism using different opioids: A prospective study. J Pain Symptom Manage. 2024;67:39–49.e5. doi: 10.1016/j.jpainsymman.2023.09.017.  [DOI](https://doi.org/10.1016/j.jpainsymman.2023.09.017) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/37757956/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pain%20Symptom%20Manage&title=Comparison%20of%20the%20effects%20of%20OPRM1%20A118G%20polymorphism%20using%20different%20opioids:%20A%20prospective%20study&author=M%20Takemura&author=K%20Niki&author=Y%20Okamoto&author=T%20Kawamura&author=M%20Kohno&volume=67&publication_year=2024&pages=39-49.e5&pmid=37757956&doi=10.1016/j.jpainsymman.2023.09.017&)

5. Caraceni A, Hanks G, Kaasa S, Bennett MI, Brunelli C, Cherny N, Dale O, De Conno F, Fallon M, Hanna M, et al. Use of opioid analgesics in the treatment of cancer pain: Evidence-based recommendations from the EAPC. Lancet Oncol. 2012;13:e58–e68. doi: 10.1016/S1470-2045(12)70040-2.  [DOI](https://doi.org/10.1016/S1470-2045(12)70040-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22300860/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet%20Oncol&title=Use%20of%20opioid%20analgesics%20in%20the%20treatment%20of%20cancer%20pain:%20Evidence-based%20recommendations%20from%20the%20EAPC&author=A%20Caraceni&author=G%20Hanks&author=S%20Kaasa&author=MI%20Bennett&author=C%20Brunelli&volume=13&publication_year=2012&pages=e58-e68&pmid=22300860&doi=10.1016/S1470-2045(12)70040-2&)

6. Fujita Y, Matsuoka H, Chiba Y, Tsurutani J, Yoshida T, Sakai K, Nakura M, Sakamoto R, Makimura C, Ohtake Y, et al. Novel single nucleotide polymorphism biomarkers to predict opioid effects for cancer pain. Oncol Lett. 2023;26(355) doi: 10.3892/ol.2023.13941.  [DOI](https://doi.org/10.3892/ol.2023.13941) | [PMC free article](/articles/PMC10398630/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/37545623/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncol%20Lett&title=Novel%20single%20nucleotide%20polymorphism%20biomarkers%20to%20predict%20opioid%20effects%20for%20cancer%20pain&author=Y%20Fujita&author=H%20Matsuoka&author=Y%20Chiba&author=J%20Tsurutani&author=T%20Yoshida&volume=26&issue=355&publication_year=2023&pmid=37545623&doi=10.3892/ol.2023.13941&)

7. Matsuoka H, Tsurutani J, Chiba Y, Fujita Y, Terashima M, Yoshida T, Sakai K, Otake Y, Koyama A, Nishio K, Nakagawa K. Selection of opioids for cancer-related pain using a biomarker: A randomized, multi-institutional, open-label trial (RELIEF study) BMC Cancer. 2017;17(674) doi: 10.1186/s12885-017-3664-z.  [DOI](https://doi.org/10.1186/s12885-017-3664-z) | [PMC free article](/articles/PMC5639590/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28985716/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Cancer&title=Selection%20of%20opioids%20for%20cancer-related%20pain%20using%20a%20biomarker:%20A%20randomized,%20multi-institutional,%20open-label%20trial%20(RELIEF%20study)&author=H%20Matsuoka&author=J%20Tsurutani&author=Y%20Chiba&author=Y%20Fujita&author=M%20Terashima&volume=17&issue=674&publication_year=2017&pmid=28985716&doi=10.1186/s12885-017-3664-z&)

8. Matsuoka H, Tsurutani J, Chiba Y, Fujita Y, Sakai K, Yoshida T, Nakura M, Sakamoto R, Makimura C, Ohtake Y, et al. Morphine versus oxycodone for cancer pain using a catechol-O-methyltransferase genotype biomarker: A multicenter, randomized, open-label, phase III clinical trial (RELIEF study) Oncologist. 2023;28:278–e166. doi: 10.1093/oncolo/oyac233.  [DOI](https://doi.org/10.1093/oncolo/oyac233) | [PMC free article](/articles/PMC10020805/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36426809/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncologist&title=Morphine%20versus%20oxycodone%20for%20cancer%20pain%20using%20a%20catechol-O-methyltransferase%20genotype%20biomarker:%20A%20multicenter,%20randomized,%20open-label,%20phase%20III%20clinical%20trial%20(RELIEF%20study)&author=H%20Matsuoka&author=J%20Tsurutani&author=Y%20Chiba&author=Y%20Fujita&author=K%20Sakai&volume=28&publication_year=2023&pages=278-e166&pmid=36426809&doi=10.1093/oncolo/oyac233&)

9. Ji RR, Berta T, Nedergaard M. Glia and pain: Is chronic pain a gliopathy? Pain. 2013;154 (Suppl 1):S10–S28. doi: 10.1016/j.pain.2013.06.022.  [DOI](https://doi.org/10.1016/j.pain.2013.06.022) | [PMC free article](/articles/PMC3858488/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23792284/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pain&title=Glia%20and%20pain:%20Is%20chronic%20pain%20a%20gliopathy?&author=RR%20Ji&author=T%20Berta&author=M%20Nedergaard&volume=154%20(Suppl%201)&publication_year=2013&pages=S10-S28&pmid=23792284&doi=10.1016/j.pain.2013.06.022&)

10. Ramesh G, MacLean AG, Philipp MT. Cytokines and chemokines at the crossroads of neuroinflammation, neurodegeneration, and neuropathic pain. Mediators Inflamm. 2013;2013(480739) doi: 10.1155/2013/480739.  [DOI](https://doi.org/10.1155/2013/480739) | [PMC free article](/articles/PMC3753746/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23997430/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mediators%20Inflamm&title=Cytokines%20and%20chemokines%20at%20the%20crossroads%20of%20neuroinflammation,%20neurodegeneration,%20and%20neuropathic%20pain&author=G%20Ramesh&author=AG%20MacLean&author=MT%20Philipp&volume=2013&issue=480739&publication_year=2013&pmid=23997430&doi=10.1155/2013/480739&)

11. Swarm R, Abernethy AP, Anghelescu DL, Benedetti C, Blinderman CD, Boston B, Cleeland C, Coyle N, Deleon-Casasola OA, Eilers JG, et al. Adult cancer pain. J Natl Compr Canc Netw. 2010;8:1046–1086. doi: 10.6004/jnccn.2010.0076.  [DOI](https://doi.org/10.6004/jnccn.2010.0076) | [PMC free article](/articles/PMC6506268/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20876544/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Natl%20Compr%20Canc%20Netw&title=Adult%20cancer%20pain&author=R%20Swarm&author=AP%20Abernethy&author=DL%20Anghelescu&author=C%20Benedetti&author=CD%20Blinderman&volume=8&publication_year=2010&pages=1046-1086&pmid=20876544&doi=10.6004/jnccn.2010.0076&)

12. Swarm RA, Dans M. NCCN frameworks for resource stratification of NCCN guidelines: Adult cancer pain and palliative care. J Natl Compr Canc Netw. 2018;16:628–631. doi: 10.6004/jnccn.2018.0044.  [DOI](https://doi.org/10.6004/jnccn.2018.0044) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29784742/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Natl%20Compr%20Canc%20Netw&title=NCCN%20frameworks%20for%20resource%20stratification%20of%20NCCN%20guidelines:%20Adult%20cancer%20pain%20and%20palliative%20care&author=RA%20Swarm&author=M%20Dans&volume=16&publication_year=2018&pages=628-631&pmid=29784742&doi=10.6004/jnccn.2018.0044&)

13. Matsuoka H, Makimura C, Koyama A, Fujita Y, Tsurutani J, Sakai K, Sakamoto R, Nishio K, Nakagawa K. Prospective replication study implicates the catechol-O-methyltransferase Val158Met polymorphism as a biomarker for the response to morphine in patients with cancer. Biomed Rep. 2017;7:380–384. doi: 10.3892/br.2017.963.  [DOI](https://doi.org/10.3892/br.2017.963) | [PMC free article](/articles/PMC5590034/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28928973/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biomed%20Rep&title=Prospective%20replication%20study%20implicates%20the%20catechol-O-methyltransferase%20Val158Met%20polymorphism%20as%20a%20biomarker%20for%20the%20response%20to%20morphine%20in%20patients%20with%20cancer&author=H%20Matsuoka&author=C%20Makimura&author=A%20Koyama&author=Y%20Fujita&author=J%20Tsurutani&volume=7&publication_year=2017&pages=380-384&pmid=28928973&doi=10.3892/br.2017.963&)

14. Matsuoka H, Yoshiuchi K, Koyama A, Makimura C, Fujita Y, Tsurutani J, Sakai K, Sakamoto R, Nishio K, Nakagawa K. Expectation of a decrease in pain affects the prognosis of pain in cancer patients: A prospective cohort study of response to morphine. Int J Behav Med. 2017;24:535–541. doi: 10.1007/s12529-017-9644-5.  [DOI](https://doi.org/10.1007/s12529-017-9644-5) | [PMC free article](/articles/PMC5509817/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28265809/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Behav%20Med&title=Expectation%20of%20a%20decrease%20in%20pain%20affects%20the%20prognosis%20of%20pain%20in%20cancer%20patients:%20A%20prospective%20cohort%20study%20of%20response%20to%20morphine&author=H%20Matsuoka&author=K%20Yoshiuchi&author=A%20Koyama&author=C%20Makimura&author=Y%20Fujita&volume=24&publication_year=2017&pages=535-541&pmid=28265809&doi=10.1007/s12529-017-9644-5&)

15. De Souza VH, De Alencar JB, Tiyo BT, Alves HV, Vendramini ECL, Sell AM, Visentainer JEL. Association of functional IL16 polymorphisms with cancer and cardiovascular disease: A meta-analysis. Oncotarget. 2020;11:3405–3417. doi: 10.18632/oncotarget.27715.  [DOI](https://doi.org/10.18632/oncotarget.27715) | [PMC free article](/articles/PMC7486693/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32934782/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncotarget&title=Association%20of%20functional%20IL16%20polymorphisms%20with%20cancer%20and%20cardiovascular%20disease:%20A%20meta-analysis&author=VH%20De%20Souza&author=JB%20De%20Alencar&author=BT%20Tiyo&author=HV%20Alves&author=ECL%20Vendramini&volume=11&publication_year=2020&pages=3405-3417&pmid=32934782&doi=10.18632/oncotarget.27715&)

16. Gao LB, Rao L, Wang YY, Liang WB, Li C, Xue H, Zhou B, Sun H, Li Y, Lv ML, et al. The association of interleukin-16 polymorphisms with IL-16 serum levels and risk of colorectal and gastric cancer. Carcinogenesis. 2009;30:295–299. doi: 10.1093/carcin/bgn281.  [DOI](https://doi.org/10.1093/carcin/bgn281) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19073878/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Carcinogenesis&title=The%20association%20of%20interleukin-16%20polymorphisms%20with%20IL-16%20serum%20levels%20and%20risk%20of%20colorectal%20and%20gastric%20cancer&author=LB%20Gao&author=L%20Rao&author=YY%20Wang&author=WB%20Liang&author=C%20Li&volume=30&publication_year=2009&pages=295-299&pmid=19073878&doi=10.1093/carcin/bgn281&)

17. Tang YJ, Wang JL, Xie KG, Lan CG. Association of interleukin 16 gene polymorphisms and plasma IL16 level with osteosarcoma risk. Sci Rep. 2016;6(34607) doi: 10.1038/srep34607.  [DOI](https://doi.org/10.1038/srep34607) | [PMC free article](/articles/PMC5050415/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27703190/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sci%20Rep&title=Association%20of%20interleukin%2016%20gene%20polymorphisms%20and%20plasma%20IL16%20level%20with%20osteosarcoma%20risk&author=YJ%20Tang&author=JL%20Wang&author=KG%20Xie&author=CG%20Lan&volume=6&issue=34607&publication_year=2016&pmid=27703190&doi=10.1038/srep34607&)

18. Zhu J, Qin C, Yan F, Wang M, Ding Q, Zhang Z, Yin C. IL-16 polymorphism and risk of renal carcinoma: Association in a Chinese population. Int J Urol. 2010;17:700–707. doi: 10.1111/j.1442-2042.2010.02559.x.  [DOI](https://doi.org/10.1111/j.1442-2042.2010.02559.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20529140/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Urol&title=IL-16%20polymorphism%20and%20risk%20of%20renal%20carcinoma:%20Association%20in%20a%20Chinese%20population&author=J%20Zhu&author=C%20Qin&author=F%20Yan&author=M%20Wang&author=Q%20Ding&volume=17&publication_year=2010&pages=700-707&pmid=20529140&doi=10.1111/j.1442-2042.2010.02559.x&)

19. Liu Z, Ma L, Qiu S, Jia T. Genetic polymorphisms of interleukin-16 are associated with susceptibility to primary knee osteoarthritis. Int J Clin Exp Med. 2015;8:1401–1405.  [PMC free article](/articles/PMC4358600/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25785145/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Clin%20Exp%20Med&title=Genetic%20polymorphisms%20of%20interleukin-16%20are%20associated%20with%20susceptibility%20to%20primary%20knee%20osteoarthritis&author=Z%20Liu&author=L%20Ma&author=S%20Qiu&author=T%20Jia&volume=8&publication_year=2015&pages=1401-1405&pmid=25785145&)

20. Burkart KM, Barton SJ, Holloway JW, Yang IA, Cakebread JA, Cruikshank W, Little F, Jin X, Farrer LA, Clough JB, et al. Association of asthma with a functional promoter polymorphism in the IL16 gene. J Allergy Clin Immunol. 2006;117:86–91. doi: 10.1016/j.jaci.2005.10.011.  [DOI](https://doi.org/10.1016/j.jaci.2005.10.011) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16387589/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Allergy%20Clin%20Immunol&title=Association%20of%20asthma%20with%20a%20functional%20promoter%20polymorphism%20in%20the%20IL16%20gene&author=KM%20Burkart&author=SJ%20Barton&author=JW%20Holloway&author=IA%20Yang&author=JA%20Cakebread&volume=117&publication_year=2006&pages=86-91&pmid=16387589&doi=10.1016/j.jaci.2005.10.011&)

21. Nakayama EE, Wasi C, Ajisawa A, Iwamoto A, Shioda T. A new polymorphism in the promoter region of the human interleukin-16 (IL-16) gene. Genes Immun. 2000;1:293–294. doi: 10.1038/sj.gene.6363672.  [DOI](https://doi.org/10.1038/sj.gene.6363672) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11196708/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genes%20Immun&title=A%20new%20polymorphism%20in%20the%20promoter%20region%20of%20the%20human%20interleukin-16%20(IL-16)%20gene&author=EE%20Nakayama&author=C%20Wasi&author=A%20Ajisawa&author=A%20Iwamoto&author=T%20Shioda&volume=1&publication_year=2000&pages=293-294&pmid=11196708&doi=10.1038/sj.gene.6363672&)

22. Elvert G, Kappel A, Heidenreich R, Englmeier U, Lanz S, Acke T, Rauter M, Plate K, Sieweke M, Breier G, Flamme I. Cooperative interaction of hypoxia-inducible factor-2alpha (HIF-2alpha) and Ets-1 in the transcriptional activation of vascular endothelial growth factor receptor-2 (Flk-1) J Biol Chem. 2003;278:7520–7530. doi: 10.1074/jbc.M211298200.  [DOI](https://doi.org/10.1074/jbc.M211298200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12464608/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Cooperative%20interaction%20of%20hypoxia-inducible%20factor-2alpha%20(HIF-2alpha)%20and%20Ets-1%20in%20the%20transcriptional%20activation%20of%20vascular%20endothelial%20growth%20factor%20receptor-2%20(Flk-1)&author=G%20Elvert&author=A%20Kappel&author=R%20Heidenreich&author=U%20Englmeier&author=S%20Lanz&volume=278&publication_year=2003&pages=7520-7530&pmid=12464608&doi=10.1074/jbc.M211298200&)

23. Kappel A, Schlaeger TM, Flamme I, Orkin SH, Risau W, Breier G. Role of SCL/Tal-1, GATA, and ets transcription factor binding sites for the regulation of flk-1 expression during murine vascular development. Blood. 2000;96:3078–3085.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/11049987/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Role%20of%20SCL/Tal-1,%20GATA,%20and%20ets%20transcription%20factor%20binding%20sites%20for%20the%20regulation%20of%20flk-1%20expression%20during%20murine%20vascular%20development&author=A%20Kappel&author=TM%20Schlaeger&author=I%20Flamme&author=SH%20Orkin&author=W%20Risau&volume=96&publication_year=2000&pages=3078-3085&pmid=11049987&)

24. Glas J, Török HP, Unterhuber H, Radlmayr M, Folwaczny C. The-295T-to-C promoter polymorphism of the IL-16 gene is associated with Crohn's disease. Clin Immunol. 2003;106:197–200. doi: 10.1016/s1521-6616(03)00021-4.  [DOI](https://doi.org/10.1016/s1521-6616(03)00021-4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12706406/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Immunol&title=The-295T-to-C%20promoter%20polymorphism%20of%20the%20IL-16%20gene%20is%20associated%20with%20Crohn's%20disease&author=J%20Glas&author=HP%20T%C3%B6r%C3%B6k&author=H%20Unterhuber&author=M%20Radlmayr&author=C%20Folwaczny&volume=106&publication_year=2003&pages=197-200&pmid=12706406&doi=10.1016/s1521-6616(03)00021-4&)

25. Gu XJ, Cui B, Zhao ZF, Chen HY, Li XY, Wang S, Ning G, Zhao YJ. Association of the interleukin (IL)-16 gene polymorphisms with Graves' disease. Clin Immunol. 2008;127:298–302. doi: 10.1016/j.clim.2008.01.017.  [DOI](https://doi.org/10.1016/j.clim.2008.01.017) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18394967/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Immunol&title=Association%20of%20the%20interleukin%20(IL)-16%20gene%20polymorphisms%20with%20Graves'%20disease&author=XJ%20Gu&author=B%20Cui&author=ZF%20Zhao&author=HY%20Chen&author=XY%20Li&volume=127&publication_year=2008&pages=298-302&pmid=18394967&doi=10.1016/j.clim.2008.01.017&)

26. Matalliotakis M, Zervou MI, Eliopoulos E, Matalliotaki C, Rahmioglu N, Kalogiannidis I, Zondervan K, Spandidos DA, Matalliotakis I, Goulielmos GN. The role of IL-16 gene polymorphisms in endometriosis. Int J Mol Med. 2018;41:1469–1476. doi: 10.3892/ijmm.2018.3368.  [DOI](https://doi.org/10.3892/ijmm.2018.3368) | [PMC free article](/articles/PMC5819913/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29328375/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Mol%20Med&title=The%20role%20of%20IL-16%20gene%20polymorphisms%20in%20endometriosis&author=M%20Matalliotakis&author=MI%20Zervou&author=E%20Eliopoulos&author=C%20Matalliotaki&author=N%20Rahmioglu&volume=41&publication_year=2018&pages=1469-1476&pmid=29328375&doi=10.3892/ijmm.2018.3368&)

27. Pawlik K, Ciechanowska A, Ciapała K, Rojewska E, Makuch W, Mika J. Blockade of CC chemokine receptor type 3 diminishes pain and enhances opioid analgesic potency in a model of neuropathic pain. Front Immunol. 2021;12(781310) doi: 10.3389/fimmu.2021.781310.  [DOI](https://doi.org/10.3389/fimmu.2021.781310) | [PMC free article](/articles/PMC8593225/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34795678/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front%20Immunol&title=Blockade%20of%20CC%20chemokine%20receptor%20type%203%20diminishes%20pain%20and%20enhances%20opioid%20analgesic%20potency%20in%20a%20model%20of%20neuropathic%20pain&author=K%20Pawlik&author=A%20Ciechanowska&author=K%20Ciapa%C5%82a&author=E%20Rojewska&author=W%20Makuch&volume=12&issue=781310&publication_year=2021&pmid=34795678&doi=10.3389/fimmu.2021.781310&)
