# A Semi-Mechanistic Population Pharmacokinetic Model of Noscapine in Healthy Subjects Considering Hepatic First-Pass Extraction and CYP2C9 Genotypes

## Metadata
**Authors:** Zhendong Chen, Max Taubert, Chunli Chen, Jana Boland, Qian Dong, Muhammad Bilal, Charalambos Dokos, Bertil Wachall, Manfred Wargenau, Bernhard Scheidel, Martin H J Wiesen, Elke Schaeffeler, Roman Tremmel, Matthias Schwab, Uwe Fuhr
**Journal:** Drugs in R&D
**Date:** 2024 May 29
**DOI:** [10.1007/s40268-024-00466-6](https://doi.org/10.1007/s40268-024-00466-6)
**PMID:** 38809387
**PMCID:** PMC11315837
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11315837/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC11315837/pdf/40268_2024_Article_466.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC11315837/pdf/40268_2024_Article_466.pdf)

## Abstract

**Introduction:** 
Noscapine is a commonly used cough suppressant, with ongoing research on its anti-inflammatory and anti-tumor properties. The drug has a pronounced pharmacokinetic variability.

**Objective:** 
This evaluation aims to describe the pharmacokinetics of noscapine using a semi-mechanistic population pharmacokinetic model and to identify covariates that could explain inter-individual pharmacokinetic variability.

**Methods:** 
Forty-eight healthy volunteers (30 men and 18 women, mean age 33 years) were enrolled in a randomized, two-period, two-stage, crossover bioequivalence study of noscapine in two different liquid formulations. Noscapine plasma concentrations following oral administration of noscapine 50 mg were evaluated by a non-compartmental analysis and by a population pharmacokinetic model separately.

**Results:** 
Compared to the reference formulation, the test formulation exhibited ratios (with 94.12% confidence intervals) of 0.784 (0.662–0.929) and 0.827 (0.762–0.925) for peak plasma concentrations and area under the plasma concentration–time curve, respectively. Significant differences in p values (< 0.01) were both observed when comparing peak plasma concentrations and area under the plasma concentration–time curve between CYP2C9 genotype-predicted phenotypes. A three-compartmental model with zero-order absorption and first-order elimination process best described the plasma data. The introduction of a liver compartment was able to describe the profound first-pass effect of noscapine. Total body weight and the CYP2C9 genotype-predicted phenotype were both identified as significant covariates on apparent clearance, which was estimated as 958 ± 548 L/h for extensive metabolizers (CYP2C9*1/*1 and *1/*9), 531 ± 304 L/h for intermediate metabolizers with an activity score of 1.5 (CYP2C9*1/*2), and 343 ± 197 L/h for poor metabolizers and intermediate metabolizers with an activity score of 1.0 (CYP2C9*1/*3, *2/*3, and*3/*3).

**Conclusion:** 
The current work is expected to facilitate the future pharmacokinetic/pharmacodynamic development of noscapine. This study was registered prior to starting at “Deutsches Register Klinischer Studien” under registration no. DRKS00017760.

**Supplementary Information:** 
The online version contains supplementary material available at 10.1007/s40268-024-00466-6.

### Introduction

Noscapine is a commonly used cough suppressant, with ongoing research on its anti-inflammatory and anti-tumor properties. The drug has a pronounced pharmacokinetic variability.

### Objective

This evaluation aims to describe the pharmacokinetics of noscapine using a semi-mechanistic population pharmacokinetic model and to identify covariates that could explain inter-individual pharmacokinetic variability.

### Methods

Forty-eight healthy volunteers (30 men and 18 women, mean age 33 years) were enrolled in a randomized, two-period, two-stage, crossover bioequivalence study of noscapine in two different liquid formulations. Noscapine plasma concentrations following oral administration of noscapine 50 mg were evaluated by a non-compartmental analysis and by a population pharmacokinetic model separately.

### Results

Compared to the reference formulation, the test formulation exhibited ratios (with 94.12% confidence intervals) of 0.784 (0.662–0.929) and 0.827 (0.762–0.925) for peak plasma concentrations and area under the plasma concentration–time curve, respectively. Significant differences in *p* values (< 0.01) were both observed when comparing peak plasma concentrations and area under the plasma concentration–time curve between *CYP2C9* genotype-predicted phenotypes. A three-compartmental model with zero-order absorption and first-order elimination process best described the plasma data. The introduction of a liver compartment was able to describe the profound first-pass effect of noscapine. Total body weight and the *CYP2C9* genotype-predicted phenotype were both identified as significant covariates on apparent clearance, which was estimated as 958 ± 548 L/h for extensive metabolizers (*CYP2C9*1/*1* and **1/*9*), 531 ± 304 L/h for intermediate metabolizers with an activity score of 1.5 (*CYP2C9*1/*2*), and 343 ± 197 L/h for poor metabolizers and intermediate metabolizers with an activity score of 1.0 (*CYP2C9*1/*3*, **2/*3*, and**3/*3*).

### Conclusion

The current work is expected to facilitate the future pharmacokinetic/pharmacodynamic development of noscapine. This study was registered prior to starting at “Deutsches Register Klinischer Studien” under registration no. DRKS00017760.

### Supplementary Information

The online version contains supplementary material available at 10.1007/s40268-024-00466-6.

## Key Points

| A semi-physiological model of noscapine was established, effectively describing the first-pass hepatic metabolism through the inclusion of a liver compartment. |
| --- |
| The CYP2C9 genotype had a major significant impact on noscapine clearance, and differences in noscapine clearance between CYP2C9 genotypes were quantified. |
| Total body weight and age were identified as significant covariates on noscapine clearance and inter-compartmental clearance, respectively. |
## Introduction

Noscapine is a naturally occurring alkaloid derived from the opium poppy. It has a long history of medicinal use and is mainly used as an antitussive agent while showing minimal toxic effects [[1](#CR1)]. The main mechanism of action of noscapine appears to be σ-receptor agonism [[2](#CR2)]. Suppression of bradykinin production has also been suggested as a mode of action in successfully suppressing cough induced by angiotensin-converting enzyme inhibitors [[3](#CR3)]. Furthermore, recent studies indicated its potential use in further therapeutic areas [[4](#CR4)]. In addition to antitussive activity, noscapine showed anti-inflammatory effects and various anti-tumor properties, including inhibiting cell proliferation, inducing apoptosis, and disrupting microtubule dynamics in cancer cells [[5](#CR5)]. These effects make noscapine a candidate for further investigation in cancer therapy, although more research is needed to fully understand its mechanisms and potential clinical applications.

The pharmacokinetic (PK) properties of noscapine have been investigated by previous studies. Following oral administration, noscapine is rapidly absorbed from the gastrointestinal tract with a peak plasma concentration (*C*_max_) reached within 1–2 h. However, noscapine undergoes an extensive first-pass metabolism in the liver, resulting in relatively low oral bioavailability of approximately 30% [[6](#CR6), [7](#CR7)]. In addition, a disproportionate increase in the area under the plasma concentration–time curve (AUC) of noscapine was observed in the oral dose range of 100–300 mg, with a three-fold increase in dose resulting in a nine-fold increase in AUC due to saturable first-pass metabolism [[7](#CR7)]. Previous clinical findings and in vitro studies demonstrated the involvement of cytochrome P450 (*CYP*) *2C9*, *2C19* and *3A4/5* in the noscapine metabolism, with *CYP2C9* being identified as the most important enzyme [[8](#CR8)–[11](#CR11)]. After entering the circulatory system, noscapine exhibits wide distribution throughout the body with a large volume of distribution of 4.7 L/kg, indicating that it distributes well into tissues [[6](#CR6)]. Noscapine exposure following oral administration showed significant variability among individuals, with a coefficient of variation (CV) of 73% observed for the AUC following administration of a 200-mg tablet [[7](#CR7)]. The plasma elimination half-life of noscapine also varies between individuals but is generally around 4–5 h [[7](#CR7)].

Genetic variations observed in CYP genes, which code for enzymes involved in drug metabolism, are known to result in altered enzyme activity. This alteration mainly leads to differences in the pharmacokinetics of drugs, which may also result in differences in pharmacodynamics [[12](#CR12)]. Previous studies provided evidence that two most prevalent allelic variants in European and Asian populations, *CYP2C9*2* and *CYP2C9*3*, exhibit enzyme activity decreased by approximately 30% and 80%, respectively, when compared with the normal-function allele, *CYP2C9*1* [[13](#CR13)]. The most common allele in Africans, *CYP2C9*9*, had no relevant effect on the pharmacokinetics of substrates including phenytoin and warfarin [[14](#CR14), [15](#CR15)]. The *CYP2C19* alleles *CYP2C19*2, CYP2C19*3, CYP3A5*3,* and *CYP3A5*6* are related to null activity, while *CYP2C19*17* is related to increased activity as reflected by respective effects on the pharmacokinetics of substrates mainly eliminated by *CYP2C19* [[16](#CR16), [17](#CR17)]. In addition, *CYP3A4*22* appears to be the most significant relevant genetic variants, which is associated with reduced enzyme activity compared with the wild-type allele [[18](#CR18)]. To date, there has been no investigation into the correlation between genetic variations in these *CYP*s and the variability observed in noscapine exposure after oral administration.

We conducted a bioequivalence study to primarily compare *C*_max_ and AUC from timepoint zero up to the last timepoint (AUC_0–t_) between a reformulated noscapine suspension formulation relative to a respective reference formulation (Nipaxon^®^ 5-mg/mL oral suspension). The purpose of the present secondary evaluation is to develop a semi-physiological PK model that characterizes hepatic first-pass extraction of noscapine and identifies covariates that could explain inter-individual PK variability of noscapine. This model is expected to help future investigations of noscapine in different therapeutic indications by a PK/pharmacodynamic analysis.

## Material and Methods

### Study Design

Pharmacokinetic data were obtained from a randomized, two-period, two-stage, cross-over bioequivalence study with two different formulations of noscapine conducted in 30 male and 18 female healthy subjects. The participants were recruited from the volunteer panel of the Clinical Pharmacology Unit of the Institute for Pharmacology at the University Hospital of Cologne. Subjects aged ≥ 18 years with a body mass index between 18.5 and 30 kg/m^2^ were enrolled for the study when all results of the screening examination were available and the subject was considered eligible. Key exclusion criteria were suspicion of hypersensitivity to noscapine or other excipients of the formulations, history of any severe disease, any relevant clinical or laboratory abnormality, any concomitant medication, smoking, drug addiction, and pregnancy or breast feeding. Because of the pandemic situation caused by severe acute respiratory syndrome coronavirus 2 during the study, a negative test result within 48 h prior to hospitalization on the ward was required, only one subject per room was allowed, and the subjects had to comply with certain infection control rules. The study was designed to include up to two stages, with 48 subjects in each stage receiving a single dose of a Nipaxon^®^ 5-mg/mL oral suspension (McNeil, Solna, Sweden) [as the reference] or a reformulated oral noscapine suspension (InfectoPharm, Heppenheim, Germany) [as the test]. In each stage, the subjects were randomly assigned to two treatment sequences (reference-test, or test-reference), with a wash-out phase of 6–14 days between each period. The two-stage design was used to account for the uncertainty of the true intra-individual variability for both *C*_max_ and AUC_0–t_ [[19](#CR19)]. The second stage was to be conducted if bioequivalence was not shown in the first stage and if achieving bioequivalence was not futile. The sample size for the standard average bioavailability testing considered to allow an appropriate assessment of relative bioavailability of noscapine from the two formulations would be *n* = 48 for alpha = 0.0294 (allowing for the two-stage design) and a power of 80% if the true *μ*_test_/*μ*_reference_ ratio was in the 0.95–1.05 range and intra-individual variability for both *C*_max_ and AUC_0–t_ did not exceed 30%. For the PK analysis, blood samples were collected using lithium heparin tubes at baseline, 0.17, 0.33, 0.5, 0.67, 0.83, 1, 1.25, 1.5, 1.75, 2, 2.5, 3, 3.5, 4, 6, 9, 12, 16, and 24 h after a single oral dose of noscapine 50 mg. Samples for laboratory tests were taken at the screening visit 2–21 days prior to the first administration of the study drug. Samples for genotyping were taken only from randomized subjects and just prior to the first administration of study drugs. A principal component analysis based on the genome-wide genetic data indicated that most individuals of the cohort were of European ancestry, while two subjects clustered to South Asian and American populations and two other individuals clustered to the African population (Fig. [S1](#MOESM1) of the Electronic Supplementary Material [ESM]).

### Analytical Method

The quantification of noscapine in plasma employed a validated liquid chromatography–mass spectrometry method [[20](#CR20)]. Noscapine was extracted from plasma by a liquid-liquid extraction method using diethyl ether. An LC-20AD liquid chromatography system (Shimadzu, Duisburg, Germany) with PAL HTC-xt autosampler (CTC Analytics AG, Zwingen, Switzerland) was used for chromatographic separation. The separation process was conducted using a Luna C_18_ column (3 μm; 100 × 2.0 mm; Phenomenex, Aschaffenburg, Germany) coupled with a pre-column Security Guard C_18_ (4 × 2.0 mm; Phenomenex) at a column temperature of 40 °C. The mobile phase consisted of acetonitrile/water (80/20, v/v) containing 0.4 μM of ammonium acetate and 0.1% ammonium hydroxide at a flow rate of 0.35 mL/min. For mass spectrometry analysis, a 5500 triple quadrupole mass spectrometer (AB Sciex, Concord, Canada) equipped with an electrospray ionization source (TurboIonSpray^®^) was utilized. The settings for IonSpray voltage and temperature were 5500 V and 500 °C, respectively. The ion-pair transitions of 414.0→220.1 and 322.1→304.2 were used to monitor noscapine and oxycodone-d_6_ (internal standard), respectively. A data analysis was carried out using Analyst software version 1.6.2 (AB Sciex). Linearity was demonstrated over a standard curve that ranges from 0.100 to 100 ng/mL. Accuracy and precision were evaluated across four concentrations: 0.250, 8.00, 40.0, and 80.0 ng/mL. The relative deviations for accuracy fell within the range of − 7.9 to 0.9%, while the coefficients of variations for precision fell within the range of 2.8–12.7%. An incurred sample reanalysis was performed to requirements of the European Medicines Agency guideline on bioanalytical method validation, and 100% of the repeated samples aligned with the acceptance criteria of ± 20% [[21](#CR21)].

### Non-compartmental Pharmacokinetic Analysis

A non-compartmental analysis was conducted using SAS version 9.4 (SAS Institute, Cary, NC, USA). Plasma PK parameters were calculated including apparent elimination rate constants (*λ*_z_), terminal plasma elimination half-life (*t*_1/2_), *C*_max_, time to reach maximum plasma concentration (*t*_max_), AUC_0–t_, and AUC from time 0 extrapolated to infinity (AUC_0–∞_). The *λ*_z_ was calculated by a linear regression analysis of the terminal log-linear portion of the individual plasma concentration versus time curves and *t*_1/2_ was then calculated as ln(2)/*λ*_z_. Peak plasma concentration and *t*_max_ were obtained directly from the actual sampling times and concentrations, respectively. Area under the plasma concentration–time curve from timepoint zero up to the last timepoint was determined according to the linear trapezoidal rule concerning the interval of t = 0 to *t*_max_. From there up to the last measured value above the lower limit of quantification (LLOQ), it was determined using the logarithmic trapezoidal rule. Area under the plasma concentration–time curve from time 0 extrapolated to infinity was calculated as AUC_0–t_ + *C*_last_(fit)/*λ*_z_, where the *C*_last_(fit) is the last fitted concentration according to regression analysis. Peak plasma concentration and AUC_0–t_ were compared separately using the *t*-test (*p* < 0.01) between different genotype-predicted phenotype groups. To assess relative bioavailability, log-transformed *C*_max_, AUC_0–t_, and AUC_0–∞_ were submitted to a linear analysis of variance model with effects for sequence. The mean square error of the analysis of variance was used as the variance estimate to calculate 94.12% confidence intervals (CIs). Bioequivalence of two different formulations could be concluded if the respective 94.12% CIs for *μ*_test_/*μ*_reference_ estimates were completely within the respective boundaries of 0.80–1.25 for *C*_max_ and AUC_0–t_.

### Population Pharmacokinetic Analysis

The population PK analysis was performed using a non-linear mixed-effects modeling software NONMEM version 7.4.0 (ICON Development Solutions, Omaha, NE, USA) [[22](#CR22)]. Model execution and diagnostic procedures were performed using Perl-speaks-NONMEM (PsN) version 5.3.0 (Uppsala University, Uppsala, Sweden) [[23](#CR23)]. The first-order conditional estimation with interaction method was used throughout model development. A statistical criterion based on a difference of > 3.84 in the objective function value (OFV) between two nested models (*p* < 0.05) that differed by one parameter was employed for model selection [[24](#CR24)]. Post-processing and plotting of NONMEM data were completed using R version 4.3.0 ([https://www.R-project.org/](https://www.R-project.org/)).

#### Structural Pharmacokinetic Model

The structural model was developed in a stepwise manner, initially starting with the simplest model, which was a one-compartment model with first-order absorption and linear elimination kinetics. Subsequently, following approaches were explored to improve the model, including: (i) evaluating the number of apparent distribution compartments; (ii) comparing first-order, zero-order, and combined (first-order plus zero-order) absorption kinetics; (iii) incorporating a liver compartment to describe first-pass metabolism; (iv) introducing a lag time or varying numbers of transit compartment between depot and central compartments; (v) separating the parameter estimation for absorption processes for the different formulations; (vi) comparing linear and non-linear elimination processes; and (vii) assessing the random effects including inter-individual variability (IIV) and inter-occasion variability (IOV).

The IIV for PK parameters was modeled exponentially as the following equation: θi=θ×eηi, where θi represents the value of the individual parameter, θ represents the population point estimate of the parameter, and ηi represents a normally distributed random variable with a mean of 0 and variance of ω2. An additive residual error model was applied to the log-transformed observations versus time data (equivalent to an exponential error model with untransformed data). The schematic of the structural model is shown in Fig. [1](#Fig1).

![Fig. 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a770/11315837/78fea5c6bdeb/40268_2024_466_Fig1_HTML.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=11315837_40268_2024_466_Fig1_HTML.jpg)

Schematic of the structural model of noscapine. CL/F apparent clearance, F1 relative bioavailability for test formulation of noscapine, Qh liver plasma flow (fixed at 55 L/h), Qp1/F intercompartment clearance between central and first peripheral compartment, Qp2/F intercompartment clearance between central and second peripheral compartment, Rate apparent absorption constant rate (equivalent to dose/absorption duration), TR average transit rate, Vc/F central compartment volume, Vh liver volume (fixed at 1.5 L), Vp1/F first peripheral compartment volume, Vp2/F second peripheral compartment volume

#### Covariate Model

A covariate analysis was performed on the structural model using forward addition and backward elimination methods, applying significance levels of 0.05 (ΔOFV ≤ − 3.84) and 0.01 (ΔOFV ≤ − 6.63), respectively. The effects of age, total body weight (TBW), height, body mass index, alanine aminotransferase, aspartate aminotransferase, alkaline phosphatase, estimated glomerular filtration rate (CKD-EPI, 2009) [[25](#CR25)], and plasma urea concentration from laboratory test results as well as genotype-predicted phenotypes of *CYP2C9*, *CYP2C19*, *CYP3A4*, *CYP3A5*, and *CYP2E1* on appropriate PK parameters were investigated. For covariate relationships, continuous covariates were modeled with mean-centered power models and categorical covariates were modeled with conditional effects.

#### Model Evaluation

During development of the structural model, sensitivity analyses were performed to verify the impact of predefined values and equations for parameters related to the liver compartment on parameter estimates, especially on values of apparent clearance (CL/*F*), relative bioavailability of test formulation (F1), IIV, and residual unexplained variability. Therefore, various values of liver volume and liver plasma flow (*Q*_h_) as well as different approaches to encoding the relationship between the liver and central compartment were evaluated separately using reported values and equations [[26](#CR26)–[30](#CR30)].

Following the covariate analysis, the final model was evaluated by individual fit plots, goodness-of-fit plots, and visual predictive checks (*n* = 1000) stratified by the covariates of interest. Model stability and parameter uncertainty were assessed by sampling importance resampling (SIR, *M*/*m* ratio = 5000/1000) [[31](#CR31)].

## Results

### Demographics and Genetic Polymorphism Analysis

A total of 1920 plasma concentrations of noscapine were collected from 48 healthy subjects, all of whom completed the two-period treatment. Observations with concentrations below the LLOQ were treated as zero before *C*_max_ (96 concentrations) and were discarded after *C*_max_ (17 concentrations) for the analysis. The study population included 30 men and 18 women with a mean age of 33 years (range 19–65 years). Detailed demographic information and baseline covariate values of enrolled subjects are listed in Table [1](#Tab1).

### Table 1.

| Characteristic | N | Mean | SD | Median | Range |
| --- | --- | --- | --- | --- | --- |
| Demographics |  |  |  |  |  |
| Male | 30 | – | – | – |   |
| Female | 18 | – | – | – |   |
| Age (years) | 48 | 33 | 11 | 29 | 19.0–65.0 |
| Height (cm) | 48 | 177 | 9 | 178 | 159–197 |
| Total body weight (kg) | 48 | 77.3 | 12.7 | 77.0 | 54.3–107 |
| BMI (kg/m2) | 48 | 24.6 | 2.9 | 24.3 | 18.0–29.5 |
| Covariates |  |  |  |  |  |
| ALT (U/L) | 48 | 24.1 | 14.0 | 22.0 | 7.0–96.0 |
| AST (U/L) | 48 | 23.5 | 5.7 | 22.0 | 14.0–39.0 |
| ALP (U/L) | 48 | 64.8 | 13.5 | 63.0 | 36.0–103 |
| eGFR (mL/min) | 48 | 107 | 14 | 76.0 | 76.0–132 |
| Urea (mg/dL) | 48 | 26.7 | 6.6 | 13.0 | 13.0–40.0 |

Table 1 Caption: Demographics and covariates for enrolled subjects

Genotyping of *CYP2C9*, *CYP2C19*, *CYP3A4*, *CYP3A5*, and *CYP2E1* was performed for all 48 subjects in this study. Genotype frequencies and phenotype classifications are shown in Table [2](#Tab2). According to the reported relative enzyme functions of different genotypes, *CYP2C9*, *CYP2C19*, *CYP3A4,* and *CYP3A5* were classified into three, four, two, and three phenotypes, respectively. Because of limited predictive data support, the phenotype of *CYP2E1* was not applicable in this study [[12](#CR12)]. Extensive metabolizers (EMs) or normal metabolizers were defined as possessing two alleles with normal enzyme function (i.e., *CYP2C9*1* and *CYP2C9*9*, *CYP2C19*1*, *CYP3A4*1*, and *CYP3A5*1*). Intermediate metabolizers (IMs) were individuals with one allele with relative dysfunction (i.e., *CYP2C9*2*, *CYP2C19*2*, and *CYP3A4*22*) or severe dysfunction (i.e., *CYP2C9*3, CYP3A5*3,* and *CYP3A5*6*) and no severe dysfunction in the other allele [[32](#CR32)–[34](#CR34)]. Poor metabolizers (PMs) were individuals with two alleles displaying severe dysfunction (i.e., *CYP2C9*3, CYP3A5*3,* and *CYP3A5*6*) [[32](#CR32)]. Of note, *CYP2C9*1/*3* and *CYP2C9*2/*2* as IMs with an activity score (AS) of 1, were classified together with the PMs into one group in this study because of the limited sample size of homozygous carriers [[35](#CR35)]. In contrast, rapid metabolizers (RMs) and ultrarapid metabolizers (UMs) were characterized by having one and two alleles of increased enzyme function (i.e., *CYP2C19*17*), respectively [[36](#CR36)].

### Table 2.

| Predicted phenotype | Genotype | No. of subjects | Frequency (%) | Reported frequency in European population (PharmGKB, dbSNP) |
| --- | --- | --- | --- | --- |
| CYP2C9 |  |  |  |  |
| EM | *1/*1 | 31 | 64.6 | 62.8 |
|   | *1/*9 | 1 | 2.1 |   |
| IM with AS of 1.5 | *1/*2 | 10 | 20.8 | 20.2 |
| PM & IM with AS of 1 | *1/*3 | 3 | 6.3 | 12 |
|   | *2/*3 | 2 | 4.2 | 1.9 |
|   | *3/*3 | 1 | 2.1 | 0.5 |
| CYP2C19 |  |  |  |  |
| UM | *17/*17 | 2 | 4.2 | 4.6 |
| RM | *1/*17 | 9 | 18.8 | 26 |
| EM | *1/*1 | 24 | 50.0 | 39 |
| IM | *1/*2 | 10 | 20.8 | 18 |
|   | *2/*17 | 3 | 6.3 | 6 |
| CYP3A4 |  |  |  |  |
| EM | *1/*1 | 39 | 81.3 |   |
| IM | *1/*22 | 9 | 18.8 | MAF ~ 5, here 9% |
| CYP3A5 |  |  |  |  |
| EM | *1/*1 | 0 | 0 |   |
| IM | *1/*3 | 12 | 25.0 | Frequency of *1 is 5.3%a |
| PM | *3/*3 | 35 | 72.9 | Frequency of *3 is 94.3%a |
|   | *3/*6 | 1 | 2.1 | Frequency of *6 is 0.3%a |
| CYP2E1 |  |  |  |  |
| NAb | *1/*1 | 34 | 70.8 |   |
|   | *1/*5 | 6 | 12.5 |   |
|   | *1/*7 | 8 | 16.7 |   |

Table 2 Caption: Summary of genotype and classification of CYP2C9, CYP2C19, CYP3A4, CYP3A5, and CYP2E1 in enrolled subjects (n = 48)

### Non-compartmental Pharmacokinetic Analysis

The mean plasma concentration–time profiles of noscapine stratified by formulation and *CYP2C9* genotype-predicted phenotype following a single oral dose of 50 mg are shown in Fig. S2 of the ESM. There were minor differences in *C*_max_ and AUC_0–t_ between the two formulations. However, it was evident that subjects with a dysfunctional *CYP2C9* allele had a noticeable higher level in both *C*_max_ and AUC_0–t_. The corresponding PK parameters of noscapine in both test and reference formulations are presented in Table [3](#Tab3). Noscapine was rapidly absorbed, with a median *t*_max_ of 0.5 h in both liquid formulations. In both formulations, noscapine showed a similar *t*_1/2_ with 8.42 and 8.29 h. The inter-individual CV (coefficient of variation) of AUC_0–t_ for test and reference formulations were 79.0 and 78.9%, respectively, which is comparable to the reported value of 73% [[7](#CR7)]. In terms of *C*_max_, AUC_0–t_, and AUC_0–∞_ of noscapine, the test formulation exhibited ratios (with 94.12% CIs of 0.784 (0.662–0.929), 0.827 (0.748–0.915), and 0.840 (0.762–0.925) compared to the reference formulation, respectively. These values did not completely fall within the bioequivalence acceptability range of 0.8–1.25, indicating that the two formulations investigated in this study are not bioequivalent. Boxplots of the *C*_max_ and AUC_0–t_ stratified by different genotype-predicted phenotypes of *CYP2C9*, *CYP2C19*, *CYP3A4*, *CYP3A5*, and *CYP2E1* are displayed in Fig. [2](#Fig2). Significant differences were observed only between genotype-predicted phenotype groups of *CYP2C9* when comparing *C*_max_ (*p* values*,* 0.00091 and 0.00013) and AUC_0–t_ (*p* values, 0.0047 and 0.00096) for EMs with IMs (AS of 1.5) and EMs with PMs & IMs (AS of 1.0), respectively.

### Table 3.

| Parametersa | Test |  | Reference |  | T/R ratio | 94.12% CIs | CVintra (%)c |
| --- | --- | --- | --- | --- | --- | --- | --- |
| Geometric mean (SD) | CV (%) | Geometric mean (SD) | CV (%) |  |  |  |  |
| t1/2 (h) | 8.42 (4.00) | 47.5 | 8.29 (3.54) | 42.7 | – | – | – |
| tmax (h)b | 0.5 (0.17–2.0) | 40.5 | 0.5 (0.33–1.25) | 37.0 | – | – | – |
| Cmax (μg/L) | 36.8 (31.2) | 84.9 | 46.9 (41.4) | 88.2 | 0.784 (1.821) | 0.662–0.929 | 44.9 |
| AUC0–t (μg/L·h) | 53.1 (41.9) | 79.0 | 64.1 (50.6) | 78.9 | 0.827 (1.428) | 0.748–0.915 | 25.9 |
| AUC0–∞ (μg/L·h) | 57.0 (44.5) | 78.1 | 67.8 (53.1) | 78.3 | 0.840 (1.409) | 0.762–0.925 | 24.9 |

Table 3 Caption: Pharmacokinetic parameters of noscapine 50 mg in test and reference formulations by a non-compartmental analysis (n = 48)

### Fig. 2.

![Fig. 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a770/11315837/745ce436e1a2/40268_2024_466_Fig2_HTML.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=11315837_40268_2024_466_Fig2_HTML.jpg)

Peak plasma concentration (Cmax) and area under the plasma concentration–time curve from timepoint zero up to the last timepoint (AUC0–t) stratified by genotype-predicted phenotype groups of CYP2C9, CYP2C19, CYP3A4, CYP3A5, and CYP2E1. *Genotype-predicted phenotype of CYP2E1 is not available. AS activity score, EM extensive metabolizer, IM intermediate metabolizer, PM poor metabolizer, RM rapid metabolizer, UM ultrarapid metabolizer

### Population Pharmacokinetic Analysis

#### Structural Population Pharmacokinetic Model

A three-compartment model provided the most reliable fit for noscapine PK profiles, surpassing both one-compartment and two-compartment models. Further improvements and attempts to refine the three-compartment model were listed in Table [S1](#MOESM1) of the ESM. Zero-order absorption described the absorption process best, while using transit compartments instead of simply introducing a lag time better described the delayed gastric emptying. Manually including four sequential transit compartments proved to be the most robust approach compared with estimating the number of transit compartments and transit time [[37](#CR37)]. The introduction of an F1 for the test formulation and a liver compartment consistently improved the model fit, regardless of the absorption models used, and therefore, they were both retained in the structural model.

A non-linear elimination model was slightly better compared with the linear elimination model (ΔOFV = − 6.79), but the estimation precision (relative standard error) for the maximum rate of reaction (*V*_max_) and the concentration of substrate that allows the enzyme to achieve half *V*_max_ (*K*_m_) were 37.0% and 45.0%, respectively. These values both further increased during subsequent model development, indicating that the data did not support precise estimation of *V*_max_ and *K*_m_ for the non-linear elimination of noscapine. Therefore, the linear elimination model was applied in the final model.

The IIVs of absorption duration, CL/*F*, central compartment volume (*V*_c_/*F*), and F1 were estimated at 209, 76.4, 29.3, and 34.1% in the base model, respectively. Incorporating IOVs of the transit rate and inter-compartment clearance (CL) between the central and first peripheral compartment (*Q*_p1_/*F*) resulted in significant decreases of − 1360 and − 208 in ΔOFV, respectively. Consequently, the IOVs of the transit rate and *Q*_p1_/*F* were estimated as 50.6 and 32.4%, respectively. The results of other attempts, such as separating the estimation of lag time or absorption duration for different formulations and incorporating a non-linear elimination process, are not presented because of their tendency to yield unstable models and parameter estimations lacking pharmacological significance.

#### Covariate Analysis

The impact of the type of preparation, subject covariates, and genotype-predicted phenotypes on post-hoc parameter estimates were explored based on the full structural PK model. Among the parameters describing drug liberation/absorption processes, only apparent bioavailability differed between test and reference preparations. In line with the findings from a non-compartmental analysis, the *CYP2C9* genotype-predicted phenotype had a significant effect on CL/*F* (ΔOFV = − 16.4 when incorporating two additional parameters). In addition, age (ΔOFV = − 10.1) and TBW (ΔOFV = − 7.63) were found to be significant covariates on *Q*_p1_/*F* and CL/*F*, respectively. Because of a lack of significant effects, other clinical characteristics and genotype-predicted phenotypes of *CYP* enzymes were not incorporated into the analysis. Therefore, the final model equations for *Q*_p1_/*F* and CL/*F* are presented below:

| Qp1F=Age29.00.348×TVQp1F×eη1, |
| --- |
| CLF=TBW77.31.34×TVCLphenotypeF×eη2, |
| --- |
where TVQ_p1_/*F* is the typical population value for *Q*_p1_/*F* and TVCL/*F*_phenotype is the typical population value for CL/*F* classified according to different *CYP2C9* genotype-predicted phenotypes. Point estimates for CL/*F* were 958, 531, and 343 L/h for EMs, IMs (AS of 1.5), and PMs & IMs (AS of 1.0), respectively.

#### Model Evaluation

The comparison of parameter estimates in sensitivity analyses is displayed in Table S2 of the ESM. Schemes of different settings for predefined parameters and equations related to the liver compartment are shown in Fig. S3 of the ESM. Employing covariate-related values for liver volume and *Q*_h_ as well as incorporating the hepatic extraction rate using CL/*F* and *Q*_h_ did not demonstrate any advantage in parameter estimates over settings in the final model. In contrast, estimates of essential parameters i.e., CL/*F*, F1, IIVs, and residual unexplained variability, remained nearly unchanged in models with different settings of the liver compartment. Therefore, the simpler relationship between liver and central compartments, and predefined values of 1.5 L and 55 L/h for liver volume and *Q*_h_, respectively, were applied in the structural model.

An overview of the goodness-of-fit plots for the final model is displayed in Fig. S4 of the ESM. As shown, a satisfactory fit was achieved between the observed and predicted values, without trends of conditional weighted residuals over time. The adequacy of the predictive properties of the final model was demonstrated in Fig. [3](#Fig3), which displays the visual predictive check plots with stratification based on the *CYP2C9* genotype-predicted phenotype.

![Fig. 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a770/11315837/c66e9c3575cb/40268_2024_466_Fig3_HTML.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=11315837_40268_2024_466_Fig3_HTML.jpg)

Visual predictive check (n = 1000) stratified by predicted CYP2C9 genotype-predicted phenotype for the final model. Dots represent observed concentrations. Black solid lines represent the median values, while dashed lines show the 5th and 95th percentiles of observed concentrations. Shaded areas are the model-predicted 95% confidence intervals for the 5th (green), 50th (yellow), and 95th (green) percentiles from 1000 simulated data sets. AS activity score, EM extensive metabolizer, IM intermediate metabolizer, PM poor metabolizer

The final model parameter estimates and the 95% CI of the SIR result with an *M*/*m* ratio of 5000/1000 are displayed in Table [4](#Tab4). The model was demonstrated stable by the consistency observed between parameter estimates and the medians from the SIR method. The diagnostic plots of SIR including distributions of the differences in OFV, spatial trend plots, and temporal trend plots (shown in Figs. S5–S7, respectively) collectively suggested that a sufficient *M*/*m* ratio was used and no discernible trends of the resampling proportion were observed for all PK parameters.

| Parameter | Final model |  | SIR with M/m ratio of 5000/1000 |  |
| --- | --- | --- | --- | --- |
| Estimate | RSE (%) | Median | 95% confidence interval |  |
| D1 (h) | 0.0624 | 28.6 | 0.0663 | 0.0353–0.105 |
| CL/F_EM (L/h) | 958 | 10.3 | 939 | 747–1144 |
| CL/F_IM_1.5 (L/h) | 531 | 23.0 | 549 | 347–809 |
| CL/F_PM_IM_1.0 (L/h) | 343 | 24.8 | 364 | 224–549 |
| Vc/F (L) | 49.7 | 5.9 | 49.6 | 43.7–55.2 |
| Qp1/F (L/h) | 18.9 | 4.0 | 18.8 | 17.3–20.3 |
| Vp1/F (L) | 243 | 3.6 | 243 | 227–261 |
| Qp2/F (L/h) | 25.8 | 4.3 | 25.8 | 23.6–27.8 |
| Vp2/F (L) | 31.2 | 4.5 | 31.2 | 28.4–33.9 |
| F1 (%) | 82.8 | 5.4 | 83.1 | 75.2–92.2 |
| TR (1/h) | 13.0 | 6.4 | 13.0 | 11.4–14.7 |
| Covariates |  |  |  |  |
| TBW on CL/F | 1.34 | 35.1 | 1.35 | 0.485–2.33 |
| Age on Qp1/F | 0.348 | 32.3 | 0.343 | 0.125–0.568 |
| Inter-individual variability (IIV, %)a |  |  |  |  |
| D1 | 226 (19.1%)c | 40.6 | 247 | 120–639 |
| CL/F | 57.3 (0.1%) | 18.8 | 58.3 | 45.5–75.6 |
| Vc/F | 28.5 (5.2%) | 15.5 | 29.2 | 24.0–36.2 |
| F1 | 34.1 (1.2%) | 16.4 | 34.5 | 27.9–40.6 |
| Correlation for IIVsb |  |  |  |  |
| D1 and CL/F | 0.109 | 225 | – | – |
| D1 and V/F | − 0.175 | 92.6 | – | – |
| D1 and F1 | 0.162 | 131 | – | – |
| Inter-occasion variability (%) |  |  |  |  |
| TR | 50.7 (2.7%)c | 16.5 | 51.0 | 41.3–62.4 |
| Qp1/F | 30.4 (14.5%) | 13.2 | 31.0 | 25.4–41.3 |
| Residual variability |  |  |  |  |
| Additive error | 0.0420d (9.5%)c | 2.4 | 0.0418 | 0.0397–0.438 |

Table Caption: Parameter estimates and SIR results from the final model

## Discussion

A bioequivalence study was conducted on 48 healthy subjects to compare two different formulations of noscapine. Plasma data were collected after oral administration of noscapine 50 mg, which were then utilized to develop a population PK model. The established model facilitated the evaluation of how polymorphisms in *CYP2C9*, *CYP2C19*, *CYP3A4*, *CYP3A5*, and *CYP2E1* affected noscapine CL. Additionally, the model also enabled the investigation into the influence of various covariates such as demographics (age, weight, height, and body mass index), liver function markers (aspartate aminotransferase, alanine aminotransferase, alkaline phosphatase), and kidney function indexes (estimated glomerular filtration rate, urea plasma concentration) on noscapine PK parameters.

Despite the complexity and influence of numerous physiological factors on the drug absorption process, the population PK models commonly used tend to be relatively simple and empirical. One of the challenges of this study was to find a stable model capable of simultaneously describing the absorption process of noscapine well in both formulations. In addition to the attempts listed in Table [S1](#MOESM1) of the ESM, alternative approaches were explored, such as separate estimation of absorption duration, absorption rate constant, lag time, and the number of transit compartments. The results showed that none of above approaches can generate a stable model with pharmacologically reasonable estimates of PK parameters. Subsequently, introduction of four sequential transit compartments with IOVs of the transit rate following zero-order absorption was proven to be the best approach to describe the complicated absorption process of noscapine in this case for both preparations. As no difference was found between the reference and test formulations in noscapine PK parameters describing the rate of liberation/absorption processes, it is tempting to speculate that the lower bioavailability of the test formulation could be due to a lower amount of drug released from the suspension.

A three-compartmental model, consisting of one central compartment and two apparent distribution compartments, fits the noscapine data best and was consequently selected as the structural model. The volumes of distribution in central (*V*_c_), first peripheral (*V*_p1_), and second peripheral (*V*_p2_) compartments were estimated at 49.7, 243, and 31.2 L, respectively, which indicated that noscapine has a wide distribution in the human body. Two previous studies on noscapine distribution demonstrated that high concentrations of noscapine were observed in various organs, including the liver, spleen, kidney, lung, and brain, after intravenous administration in rats [[38](#CR38), [39](#CR39)]. Therefore, considering its characteristics of lipophilicity (Log*P* = 2.6), it is not surprising that a large volume of distribution was observed in previous studies and this study [[40](#CR40), [41](#CR41)].

Saturable first-pass metabolism of noscapine was confirmed through the observation of non-linear pharmacokinetics following oral administration within the dose range of 100–300 mg [[7](#CR7)]. In the present study, introduction of a liver compartment between depot and central compartments led to a significant decrease of 562 in ΔOFV, which well explained the first-pass metabolism effect. Although a non-linear elimination process was discovered during model development, the estimates of *V*_max_ and *K*_m_ lacked accuracy with a relative standard error above 30%. Therefore, an oral dose of noscapine 50 mg may not result in relevant saturation of the corresponding metabolism enzymes in humans. The apparent *t*_1/2_ observed in this study is approximately twice the previously reported value of 4.5 h [[6](#CR6)]. The observed difference could be attributed to the utilization of a more sensitive liquid chromatography-mass spectrometry method with a LLOQ of 0.100 μg/L in this study, compared with the high-performance liquid chromatography-ultraviolet method with a LLOQ of 5.0 μg/L in the previous study.

A previous in vitro study demonstrated that that primary metabolism of noscapine is mainly mediated by *CYP2C9*, with some contribution of *CP2C19* and *CYP3A4/5*, while there is no in vivo information on the relative contribution of individual enzymes to overall noscapine CL [[11](#CR11)]. As an over-the-counter drug, noscapine has been reported to interact with vitamin K antagonists such as acenocoumarol, phenprocoumon, and warfarin, all of which are known *CYP2C9* substrates, thus *CYP2C9* genotypes may also have an impact on such interactions [[42](#CR42)]. In this study, genotypes of *CYP2C9*, *CYP2C19*, and *CYP3A4* were investigated among all enrolled subjects. As shown in Table [2](#Tab2), genotype frequencies of *CYP2C9* were close to reported genotype frequencies in Caucasian individuals [[43](#CR43)]. The allele frequencies of *CYP2C19*2*_,_* CYP2C19*17, CYP3A5*3,* and *CYP2C19*6* were found to be 13.5, 16.7, 86.5, and 1.0%, respectively, which closely align with the reported values [[44](#CR44)]. However, the allele frequencies of *CYP3A4*22* and *CYP3A5*1* were 9.4 and 12.5%, respectively, which are both two-fold higher than reported values of 3–5%, and 5.3% [[34](#CR34), [45](#CR45)]. Deviations from published data could be explained by the small sample size of 48 (and the genetically determined non-European ancestry of four samples [Fig. [S1](#MOESM1) of the ESM]).

The impact of the *CYP2C9* genotype on the pharmacokinetics of drugs metabolized by this enzyme has been described for a number of drugs. For instance, S-warfarin metabolism was demonstrated through in vitro experiments, which revealed that the *CYP2C9*2* and **3* variants displayed 70% and 5% metabolic efficiency, in comparison to the wild-type enzyme, respectively [[46](#CR46)]. Among 156 patients, oral plasma CL of warfarin was reported as 2.4 ± 1.2, 2.2 ± 0.9, and 1.5 ± 1.0 mL/min for *CYP2C9*1/*1*, **1/*2*, and **1/*3* carriers, respectively [[47](#CR47)]. Additionally, the genotype effect on tolbutamide elimination was evident with plasma CLs of 0.85, 0.77, 0.60, and 0.57 L/h for *CYP2C9*1/*1*, **1/*2*, **1/*3*, and **2/*2* carriers, respectively [[48](#CR48)]. However, a more pronounced effect of the *CYP2C9* genotype was observed on noscapine elimination in this study, with apparent CL of 958, 531, and 343 L/h for EMs (**1/*1* and **1/*9*), IMs with an AS of 1.5 (**1/*2*), and PMs & IMs with an AS of 1.0 (**1/*3*, **2/*3*, **3/*3*), respectively. Compared with the nearly 100% oral bioavailability of warfarin, the pronounced first-pass metabolism of noscapine by *CYP2C9* results in a low oral bioavailability (~ 30%) [[49](#CR49)]. Thus, the *CYP2C9* genotype effects may influence oral bioavailability and thus contribute to large differences in apparent CL between the genotype groups.

Compared with the published PK studies of noscapine [[6](#CR6), [7](#CR7)], we observed similar characteristics, including a fast absorption process, profound first-pass metabolism, a large volume of distribution, and high IIV in plasma exposure. However, no obvious saturable elimination process or enzyme inhibition was observed in noscapine elimination. This discrepancy may be attributed to the lower oral dose of 50 mg investigated in our study compared with the minimum oral dose of 100 mg investigated in previous PK studies. Furthermore, our findings indicate that genetic variations in *CYP2C9* significantly influence noscapine metabolism following a single dose of 50 mg, while variations in *CYP3A4* and *CYP2C19* do not lead to PK differences between groups [[9](#CR9)–[11](#CR11)]. This observation could be also associated with the comparatively lower plasma concentrations of noscapine after a single dose of 50 mg.

Several limitations remained in this study. Only one dose group of noscapine 50 mg was investigated for pharmacokinetics, which failed to support the investigation of its non-linear metabolic behavior. The primary objective of this bioequivalence study was not focused on a genotype assessment, and thus the sample size was not specifically optimized for this purpose. The small sample size of the *CYP2C9*3* carrier, with particularly only one *CYP2C9*3/*3* carrier, resulted in relatively high relative standard error for CL estimates of the *CYP2C9*3* allele when attempting to separately estimate CL for each *CYP2C9* allele. Consequently, to address this limitation, all six individuals with the *CYP2C9*3* variant were grouped together as PMs & IMs with an AS of 1.0 and CL for each *CYP2C9* genotype-predicted phenotype was estimated instead of the genotype. The high IIV of CL/*F*, which remained at 57.3% even after inclusion of related covariates, may be linked to the unknown variability in absolute bioavailability. Unfortunately, there is a lack of intravenous administration data to enhance our understanding of first-pass metabolism, which would help determine the absolute CL of noscapine.

## Conclusions

Overall, a semi-physiological model of noscapine was successfully established, effectively describing the first-pass hepatic metabolism, and providing an explanation for the impact of genetic variations on noscapine metabolism. The *CYP2C9* enzyme was confirmed to play the most important role at noscapine metabolism at oral dose of 50 mg. Additionally, total body weight and age were identified as significant covariates on CL and *Q*_p1_/*F*, respectively. The current work has provided a stable PK model for noscapine, which is expected to facilitate its future PK/pharmacodynamic development.

## Supplementary Information

Below is the link to the electronic supplementary material.

## Acknowledgements

Zhendong Chen received a scholarship from the China Scholarship Council for support of his PhD studies. Roman Tremmel, Elke Schaeffeler, and Matthias Schwab were supported by the Robert Bosch Stiftung, Stuttgart, Germany. Chunli Chen is supported by the 2022 ESI International High Impact Research Article Cooperation Program (No. 212-54900112), the National Natural Science Foundation of Heilongjiang Province (No. YQ2022C017), and the International Postdoctoral Exchange Fellowship Program from the Office of China Postdoctoral Council (Nos. 2020106 and PC2020013). The excellent technical assistance concerning the genotyping of Thomas Hees is gratefully acknowledged. We thank Prof. Dr. Markus M. Nöthen and Dr. Per Hoffmann for providing the SNP microarray data used in this study.

## Declarations

### Funding

The clinical study was funded by InfectoPharm Arzneimittel und Consilium GmbH, 64646 Heppenheim, Germany.

### Conflict of interest

Beyond funding of the study, Zhendong Chen, Max Taubert, Chunli Chen, Jana Boland, Qian Dong, Muhammad Bilal, Charalambos Dokos, Bertil Wachall, Manfred Wargenau, Bernhard Scheidel, Martin H. J. Wiesen, Elke Schaeffeler, Roman Tremmel, Matthias Schwab, and Uwe Fuhr have no conflicts of interest that are directly relevant to the content of this article.

### Ethics approval

The study protocol was approved by the Ethics Committee (No. 21-1404-AMG-ff) of the Faculty of Medicine of the University of Cologne, Germany (EUDRA-CT No. 2019-002012-12), and by the pertinent authorities and was conducted in accordance with Good Clinical Practice guidelines and the Declaration of Helsinki (64th WMA General Assembly, Brazil, October 2013). This study was registered prior to starting at “Deutsches Register Klinischer Studien” under registration no. DRKS00017760.

### Consent to participate

All subjects provided informed written consent to participate in genotyping and pharmacokinetic studies.

### Consent for publication

Not applicable.

### Availability of data and material

The data that support the findings of this study are available on request from the corresponding author.

### Code availability

Not applicable.

### Author contributions

ZC and UF wrote the manuscript; UF and BW designed the research; ZC, QD, MB, CD, BW, BS, MHJW, ES, RT, MS, and UF performed the research; ZC, MT, CC, MW, and JB analyzed the data.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Empey DW, Laitinen LA, Young GA, Bye CE, Hughes DT. Comparison of the antitussive effects of codeine phosphate 20 mg, dextromethorphan 30 mg and noscapine 30 mg using citric acid-induced cough in normal subjects. Eur J Clin Pharmacol. 1979;16(6):393–7. 10.1007/BF00568199  [DOI](https://doi.org/10.1007/BF00568199) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/527635/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Empey%20DW,%20Laitinen%20LA,%20Young%20GA,%20Bye%20CE,%20Hughes%20DT.%20Comparison%20of%20the%20antitussive%20effects%20of%20codeine%20phosphate%2020%20mg,%20dextromethorphan%2030%20mg%20and%20noscapine%2030%20mg%20using%20citric%20acid-induced%20cough%20in%20normal%20subjects.%20Eur%20J%20Clin%20Pharmacol.%201979;16(6):393%E2%80%937.%2010.1007/BF00568199)

2. Kamei J. Role of opioidergic and serotonergic mechanisms in cough and antitussives. Pulm Pharmacol. 1996;9(5–6):349–56. 10.1006/pulp.1996.0046  [DOI](https://doi.org/10.1006/pulp.1996.0046) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9232674/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Kamei%20J.%20Role%20of%20opioidergic%20and%20serotonergic%20mechanisms%20in%20cough%20and%20antitussives.%20Pulm%20Pharmacol.%201996;9(5%E2%80%936):349%E2%80%9356.%2010.1006/pulp.1996.0046)

3. Mooraki A, Jenabi A, Jabbari M, Zolfaghari MI, Javanmardi SZ, Mahmoudian M, et al. Noscapine suppresses angiotensin converting enzyme inhibitors-induced cough. Nephrology (Carlton). 2005;10(4):348–50. 10.1111/j.1440-1797.2005.00429.x  [DOI](https://doi.org/10.1111/j.1440-1797.2005.00429.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16109080/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Mooraki%20A,%20Jenabi%20A,%20Jabbari%20M,%20Zolfaghari%20MI,%20Javanmardi%20SZ,%20Mahmoudian%20M,%20et%20al.%20Noscapine%20suppresses%20angiotensin%20converting%20enzyme%20inhibitors-induced%20cough.%20Nephrology%20(Carlton).%202005;10(4):348%E2%80%9350.%2010.1111/j.1440-1797.2005.00429.x)

4. Karlsson MO, Dahlström B, Neil A. Characterization of high-affinity binding sites for the antitussive [3H]noscapine in guinea pig brain tissue. Eur J Pharmacol. 1988;145(2):195–203. 10.1016/0014-2999(88)90230-0  [DOI](https://doi.org/10.1016/0014-2999(88)90230-0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/3350041/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Karlsson%20MO,%20Dahlstr%C3%B6m%20B,%20Neil%20A.%20Characterization%20of%20high-affinity%20binding%20sites%20for%20the%20antitussive%20%5B3H%5Dnoscapine%20in%20guinea%20pig%20brain%20tissue.%20Eur%20J%20Pharmacol.%201988;145(2):195%E2%80%93203.%2010.1016/0014-2999(88)90230-0)

5. Rahmanian-Devin P, Baradaran Rahimi V, Jaafari MR, Golmohammadzadeh S, Sanei-Far Z, Askari VR. Noscapine, an emerging medication for different diseases: a mechanistic review. Evid Based Complement Altern Med. 2021;2021:8402517. 10.1155/2021/8402517  [DOI](https://doi.org/10.1155/2021/8402517) | [PMC free article](/articles/PMC8648453/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34880922/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Rahmanian-Devin%20P,%20Baradaran%20Rahimi%20V,%20Jaafari%20MR,%20Golmohammadzadeh%20S,%20Sanei-Far%20Z,%20Askari%20VR.%20Noscapine,%20an%20emerging%20medication%20for%20different%20diseases:%20a%20mechanistic%20review.%20Evid%20Based%20Complement%20Altern%20Med.%202021;2021:8402517.%2010.1155/2021/8402517) | [Retracted](/articles/PMC10732963/)

6. Dahlström B, Mellstrand T, Löfdahl CG, Johansson M. Pharmacokinetic properties of noscapine. Eur J Clin Pharmacol. 1982;22(6):535–9. 10.1007/BF00609627  [DOI](https://doi.org/10.1007/BF00609627) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7128665/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Dahlstr%C3%B6m%20B,%20Mellstrand%20T,%20L%C3%B6fdahl%20CG,%20Johansson%20M.%20Pharmacokinetic%20properties%20of%20noscapine.%20Eur%20J%20Clin%20Pharmacol.%201982;22(6):535%E2%80%939.%2010.1007/BF00609627)

7. Karlsson MO, Dahlström B, Eckernäs SA, Johansson M, Alm AT. Pharmacokinetics of oral noscapine. Eur J Clin Pharmacol. 1990;39(3):275–9. 10.1007/BF00315110  [DOI](https://doi.org/10.1007/BF00315110) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2257866/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Karlsson%20MO,%20Dahlstr%C3%B6m%20B,%20Eckern%C3%A4s%20SA,%20Johansson%20M,%20Alm%20AT.%20Pharmacokinetics%20of%20oral%20noscapine.%20Eur%20J%20Clin%20Pharmacol.%201990;39(3):275%E2%80%939.%2010.1007/BF00315110)

8. Scordo MG, Melhus H, Stjernberg E, Edvardsson AM, Wadelius M. Warfarin-noscapine interaction: a series of four case reports. Ann Pharmacother. 2008;42(3):448–50. 10.1345/aph.1K544  [DOI](https://doi.org/10.1345/aph.1K544) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18303135/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Scordo%20MG,%20Melhus%20H,%20Stjernberg%20E,%20Edvardsson%20AM,%20Wadelius%20M.%20Warfarin-noscapine%20interaction:%20a%20series%20of%20four%20case%20reports.%20Ann%20Pharmacother.%202008;42(3):448%E2%80%9350.%2010.1345/aph.1K544)

9. Fang ZZ, Zhang YY, Ge GB, Huo H, Liang SC, Yang L. Time-dependent inhibition (TDI) of CYP3A4 and CYP2C9 by noscapine potentially explains clinical noscapine-warfarin interaction. Br J Clin Pharmacol. 2010;69(2):193–9. 10.1111/j.1365-2125.2009.03572.x  [DOI](https://doi.org/10.1111/j.1365-2125.2009.03572.x) | [PMC free article](/articles/PMC2824481/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20233183/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Fang%20ZZ,%20Zhang%20YY,%20Ge%20GB,%20Huo%20H,%20Liang%20SC,%20Yang%20L.%20Time-dependent%20inhibition%20(TDI)%20of%20CYP3A4%20and%20CYP2C9%20by%20noscapine%20potentially%20explains%20clinical%20noscapine-warfarin%20interaction.%20Br%20J%20Clin%20Pharmacol.%202010;69(2):193%E2%80%939.%2010.1111/j.1365-2125.2009.03572.x)

10. Rosenborg S, Stenberg M, Otto S, Ostervall J, Masquelier M, Yue QY, et al. Clinically significant CYP2C inhibition by noscapine but not by glucosamine. Clin Pharmacol Ther. 2010;88(3):343–6. 10.1038/clpt.2010.107  [DOI](https://doi.org/10.1038/clpt.2010.107) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20668444/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Rosenborg%20S,%20Stenberg%20M,%20Otto%20S,%20Ostervall%20J,%20Masquelier%20M,%20Yue%20QY,%20et%20al.%20Clinically%20significant%20CYP2C%20inhibition%20by%20noscapine%20but%20not%20by%20glucosamine.%20Clin%20Pharmacol%20Ther.%202010;88(3):343%E2%80%936.%2010.1038/clpt.2010.107)

11. Fang ZZ, Krausz KW, Li F, Cheng J, Tanaka N, Gonzalez FJ. Metabolic map and bioactivation of the anti-tumour drug noscapine. Br J Pharmacol. 2012;167(6):1271–86. 10.1111/j.1476-5381.2012.02067.x  [DOI](https://doi.org/10.1111/j.1476-5381.2012.02067.x) | [PMC free article](/articles/PMC3504993/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22671862/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Fang%20ZZ,%20Krausz%20KW,%20Li%20F,%20Cheng%20J,%20Tanaka%20N,%20Gonzalez%20FJ.%20Metabolic%20map%20and%20bioactivation%20of%20the%20anti-tumour%20drug%20noscapine.%20Br%20J%20Pharmacol.%202012;167(6):1271%E2%80%9386.%2010.1111/j.1476-5381.2012.02067.x)

12. Zanger UM, Schwab M. Cytochrome P450 enzymes in drug metabolism: regulation of gene expression, enzyme activities, and impact of genetic variation. Pharmacol Ther. 2013;138(1):103–41. 10.1016/j.pharmthera.2012.12.007  [DOI](https://doi.org/10.1016/j.pharmthera.2012.12.007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23333322/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Zanger%20UM,%20Schwab%20M.%20Cytochrome%20P450%20enzymes%20in%20drug%20metabolism:%20regulation%20of%20gene%20expression,%20enzyme%20activities,%20and%20impact%20of%20genetic%20variation.%20Pharmacol%20Ther.%202013;138(1):103%E2%80%9341.%2010.1016/j.pharmthera.2012.12.007)

13. Crespi CL, Miller VP. The R144C change in the CYP2C9*2 allele alters interaction of the cytochrome P450 with NADPH:cytochrome P450 oxidoreductase. Pharmacogenetics. 1997;7(3):203–10. 10.1097/00008571-199706000-00005  [DOI](https://doi.org/10.1097/00008571-199706000-00005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9241660/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Crespi%20CL,%20Miller%20VP.%20The%20R144C%20change%20in%20the%20CYP2C9*2%20allele%20alters%20interaction%20of%20the%20cytochrome%20P450%20with%20NADPH:cytochrome%20P450%20oxidoreductase.%20Pharmacogenetics.%201997;7(3):203%E2%80%9310.%2010.1097/00008571-199706000-00005)

14. Blaisdell J, Jorge-Nebert LF, Coulter S, Ferguson SS, Lee SJ, Chanas B, et al. Discovery of new potentially defective alleles of human CYP2C9. Pharmacogenetics. 2004;14(8):527–37. 10.1097/01.fpc.0000114759.08559.51  [DOI](https://doi.org/10.1097/01.fpc.0000114759.08559.51) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15284535/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Blaisdell%20J,%20Jorge-Nebert%20LF,%20Coulter%20S,%20Ferguson%20SS,%20Lee%20SJ,%20Chanas%20B,%20et%20al.%20Discovery%20of%20new%20potentially%20defective%20alleles%20of%20human%20CYP2C9.%20Pharmacogenetics.%202004;14(8):527%E2%80%9337.%2010.1097/01.fpc.0000114759.08559.51)

15. Niinuma Y, Saito T, Takahashi M, Tsukada C, Ito M, Hirasawa N, et al. Functional characterization of 32 CYP2C9 allelic variants. Pharmacogenomics J. 2014;14(2):107–14. 10.1038/tpj.2013.22  [DOI](https://doi.org/10.1038/tpj.2013.22) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23752738/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Niinuma%20Y,%20Saito%20T,%20Takahashi%20M,%20Tsukada%20C,%20Ito%20M,%20Hirasawa%20N,%20et%20al.%20Functional%20characterization%20of%2032%20CYP2C9%20allelic%20variants.%20Pharmacogenomics%20J.%202014;14(2):107%E2%80%9314.%2010.1038/tpj.2013.22)

16. Lee SJ. Clinical application of CYP2C19 pharmacogenetics toward more Pprsonalized Medicine. Front Genet. 2013;3:318. 10.3389/fgene.2012.00318  [DOI](https://doi.org/10.3389/fgene.2012.00318) | [PMC free article](/articles/PMC3561709/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23378847/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Lee%20SJ.%20Clinical%20application%20of%20CYP2C19%20pharmacogenetics%20toward%20more%20Pprsonalized%20Medicine.%20Front%20Genet.%202013;3:318.%2010.3389/fgene.2012.00318)

17. Kuehl P, Zhang J, Lin Y, Lamba J, Assem M, Schuetz J, et al. Sequence diversity in CYP3A promoters and characterization of the genetic basis of polymorphic CYP3A5 expression. Nat Genet. 2001;27(4):383–91. 10.1038/86882  [DOI](https://doi.org/10.1038/86882) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11279519/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Kuehl%20P,%20Zhang%20J,%20Lin%20Y,%20Lamba%20J,%20Assem%20M,%20Schuetz%20J,%20et%20al.%20Sequence%20diversity%20in%20CYP3A%20promoters%20and%20characterization%20of%20the%20genetic%20basis%20of%20polymorphic%20CYP3A5%20expression.%20Nat%20Genet.%202001;27(4):383%E2%80%9391.%2010.1038/86882)

18. Werk AN, Cascorbi I. Functional gene variants of CYP3A4. Clin Pharmacol Ther. 2014;96(3):340–8. 10.1038/clpt.2014.129  [DOI](https://doi.org/10.1038/clpt.2014.129) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24926778/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Werk%20AN,%20Cascorbi%20I.%20Functional%20gene%20variants%20of%20CYP3A4.%20Clin%20Pharmacol%20Ther.%202014;96(3):340%E2%80%938.%2010.1038/clpt.2014.129)

19. European Medicine Agency. Committee for Medicinal Products for Human Use (CHMP) guideline on the investigation of bioequivalence. CPMP/EWP/QWP/1401/98 Rev 1/Corr **. 2010. Available from: https://www.ema.europa.eu/en/documents/scientific-guideline/guideline-investigation-bioequivalence-rev1_en.pdf. Accessed 16 May 2024.  [https://www.ema.europa.eu/en/documents/scientific-guideline/guideline-investigation-bioequivalence-rev1_en.pdf](https://www.ema.europa.eu/en/documents/scientific-guideline/guideline-investigation-bioequivalence-rev1_en.pdf)

20. Zhu L, Chen X, Zhang Y, Yu H, Zhong D. Simultaneous determination of methylephedrine and noscapine in human plasma by liquid chromatography-tandem mass spectrometry. J Chromatogr B Analyt Technol Biomed Life Sci. 2005;820(2):175–82. 10.1016/j.jchromb.2005.03.010  [DOI](https://doi.org/10.1016/j.jchromb.2005.03.010) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15899371/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Zhu%20L,%20Chen%20X,%20Zhang%20Y,%20Yu%20H,%20Zhong%20D.%20Simultaneous%20determination%20of%20methylephedrine%20and%20noscapine%20in%20human%20plasma%20by%20liquid%20chromatography-tandem%20mass%20spectrometry.%20J%20Chromatogr%20B%20Analyt%20Technol%20Biomed%20Life%20Sci.%202005;820(2):175%E2%80%9382.%2010.1016/j.jchromb.2005.03.010)

21. Guideline on bioanalytical method validation (EMEA/CHMP/EWP/192217/2009 Rev.1 Corr. 2**); 2011. Available from: https://www.ema.europa.eu/en/documents/scientific-guideline/guideline-bioanalytical-method-validation_en.pdf. Accessed 16 May 2024.  [https://www.ema.europa.eu/en/documents/scientific-guideline/guideline-bioanalytical-method-validation_en.pdf](https://www.ema.europa.eu/en/documents/scientific-guideline/guideline-bioanalytical-method-validation_en.pdf)

22. Boeckmann A, Sheiner L, Beal S. NONMEM users guide: part V. San Francisco: University of California; 2001.  [Google Scholar](https://scholar.google.com/scholar_lookup?Boeckmann%20A,%20Sheiner%20L,%20Beal%20S.%20NONMEM%20users%20guide:%20part%20V.%20San%20Francisco:%20University%20of%20California;%202001.)

23. Lindbom L, Pihlgren P, Jonsson EN. PsN-Toolkit: a collection of computer intensive statistical methods for non-linear mixed effect modeling using NONMEM. Comput Methods Progr Biomed. 2005;79(3):241–57. 10.1016/j.cmpb.2005.04.005  [DOI](https://doi.org/10.1016/j.cmpb.2005.04.005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16023764/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Lindbom%20L,%20Pihlgren%20P,%20Jonsson%20EN.%20PsN-Toolkit:%20a%20collection%20of%20computer%20intensive%20statistical%20methods%20for%20non-linear%20mixed%20effect%20modeling%20using%20NONMEM.%20Comput%20Methods%20Progr%20Biomed.%202005;79(3):241%E2%80%9357.%2010.1016/j.cmpb.2005.04.005)

24. National Institute of Standards and Technology and Semiconductor Manufacturing Technology (NIST/SEMATECH). e-Handbook of statistical methods. Section 1.3.6.7.4. Critical values of the chi-square distribution. Available from: https://www.itl.nist.gov/div898/handbook/eda/section3/eda3674.htm. Accessed 14 June 2023.  [https://www.itl.nist.gov/div898/handbook/eda/section3/eda3674.htm](https://www.itl.nist.gov/div898/handbook/eda/section3/eda3674.htm)

25. Levey AS, Stevens LA, Schmid CH, Zhang YL, Castro AF 3rd, Feldman HI, CKD-EPI (Chronic Kidney Disease Epidemiology Collaboration), et al. A new equation to estimate glomerular filtration rate. Ann Intern Med. 2009;150(9):604–12. 10.7326/0003-4819-150-9-200905050-00006  [DOI](https://doi.org/10.7326/0003-4819-150-9-200905050-00006) | [PMC free article](/articles/PMC2763564/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19414839/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Levey%20AS,%20Stevens%20LA,%20Schmid%20CH,%20Zhang%20YL,%20Castro%20AF%203rd,%20Feldman%20HI,%20CKD-EPI%20(Chronic%20Kidney%20Disease%20Epidemiology%20Collaboration),%20et%20al.%20A%20new%20equation%20to%20estimate%20glomerular%20filtration%20rate.%20Ann%20Intern%20Med.%202009;150(9):604%E2%80%9312.%2010.7326/0003-4819-150-9-200905050-00006)

26. Brown RP, Delp MD, Lindstedt SL, Rhomberg LR, Beliles RP. Physiological parameter values for physiologically based pharmacokinetic models. Toxicol Ind Health. 1997;13(4):407–84. 10.1177/074823379701300401  [DOI](https://doi.org/10.1177/074823379701300401) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9249929/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Brown%20RP,%20Delp%20MD,%20Lindstedt%20SL,%20Rhomberg%20LR,%20Beliles%20RP.%20Physiological%20parameter%20values%20for%20physiologically%20based%20pharmacokinetic%20models.%20Toxicol%20Ind%20Health.%201997;13(4):407%E2%80%9384.%2010.1177/074823379701300401)

27. Johnson TN, Tucker GT, Tanner MS, Rostami-Hodjegan A. Changes in liver volume from birth to adulthood: a meta-analysis. Liver Transpl. 2005;11(12):1481–93. 10.1002/lt.20519  [DOI](https://doi.org/10.1002/lt.20519) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16315293/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Johnson%20TN,%20Tucker%20GT,%20Tanner%20MS,%20Rostami-Hodjegan%20A.%20Changes%20in%20liver%20volume%20from%20birth%20to%20adulthood:%20a%20meta-analysis.%20Liver%20Transpl.%202005;11(12):1481%E2%80%9393.%2010.1002/lt.20519)

28. Gordi T, Xie R, Huong NV, Huong DX, Karlsson MO, Ashton M. A semiphysiological pharmacokinetic model for artemisinin in healthy subjects incorporating autoinduction of metabolism and saturable first-pass hepatic extraction. Br J Clin Pharmacol. 2005;59(2):189–98. 10.1111/j.1365-2125.2004.02321.x  [DOI](https://doi.org/10.1111/j.1365-2125.2004.02321.x) | [PMC free article](/articles/PMC1884742/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15676041/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Gordi%20T,%20Xie%20R,%20Huong%20NV,%20Huong%20DX,%20Karlsson%20MO,%20Ashton%20M.%20A%20semiphysiological%20pharmacokinetic%20model%20for%20artemisinin%20in%20healthy%20subjects%20incorporating%20autoinduction%20of%20metabolism%20and%20saturable%20first-pass%20hepatic%20extraction.%20Br%20J%20Clin%20Pharmacol.%202005;59(2):189%E2%80%9398.%2010.1111/j.1365-2125.2004.02321.x)

29. Quinney SK, Mohamed AN, Hebert MF, Haas DM, Clark S, Umans JG, et al. A semi-mechanistic metabolism model of CYP3A substrates in pregnancy: predicting changes in midazolam and nifedipine pharmacokinetics. CPT Pharmacometr Syst Pharmacol. 2012;1(9): e2. 10.1038/psp.2012.5  [DOI](https://doi.org/10.1038/psp.2012.5) | [PMC free article](/articles/PMC3603475/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23835882/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Quinney%20SK,%20Mohamed%20AN,%20Hebert%20MF,%20Haas%20DM,%20Clark%20S,%20Umans%20JG,%20et%20al.%20A%20semi-mechanistic%20metabolism%20model%20of%20CYP3A%20substrates%20in%20pregnancy:%20predicting%20changes%20in%20midazolam%20and%20nifedipine%20pharmacokinetics.%20CPT%20Pharmacometr%20Syst%20Pharmacol.%202012;1(9):%20e2.%2010.1038/psp.2012.5)

30. Krishnatry AS, Voelkner A, Dhar A, Prohn M, Ferron-Brady G. Population pharmacokinetic modeling of molibresib and its active metabolites in patients with solid tumors: a semimechanistic autoinduction model. CPT Pharmacometr Syst Pharmacol. 2021;10(7):709–22. 10.1002/psp4.12639  [DOI](https://doi.org/10.1002/psp4.12639) | [PMC free article](/articles/PMC8302244/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33955700/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Krishnatry%20AS,%20Voelkner%20A,%20Dhar%20A,%20Prohn%20M,%20Ferron-Brady%20G.%20Population%20pharmacokinetic%20modeling%20of%20molibresib%20and%20its%20active%20metabolites%20in%20patients%20with%20solid%20tumors:%20a%20semimechanistic%20autoinduction%20model.%20CPT%20Pharmacometr%20Syst%20Pharmacol.%202021;10(7):709%E2%80%9322.%2010.1002/psp4.12639)

31. Dosne AG, Bergstrand M, Harling K, Karlsson MO. Improving the estimation of parameter uncertainty distributions in nonlinear mixed effects models using sampling importance resampling. J Pharmacokinet Pharmacodyn. 2016;43(6):583–96. 10.1007/s10928-016-9487-8  [DOI](https://doi.org/10.1007/s10928-016-9487-8) | [PMC free article](/articles/PMC5110709/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27730482/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Dosne%20AG,%20Bergstrand%20M,%20Harling%20K,%20Karlsson%20MO.%20Improving%20the%20estimation%20of%20parameter%20uncertainty%20distributions%20in%20nonlinear%20mixed%20effects%20models%20using%20sampling%20importance%20resampling.%20J%20Pharmacokinet%20Pharmacodyn.%202016;43(6):583%E2%80%9396.%2010.1007/s10928-016-9487-8)

32. Kirchheiner J, Tsahuridu M, Jabrane W, Roots I, Brockmöller J. The CYP2C9 polymorphism: from enzyme kinetics to clinical dose recommendations. Per Med. 2004;1(1):63–84. 10.1517/17410541.1.1.63  [DOI](https://doi.org/10.1517/17410541.1.1.63) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29793229/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Kirchheiner%20J,%20Tsahuridu%20M,%20Jabrane%20W,%20Roots%20I,%20Brockm%C3%B6ller%20J.%20The%20CYP2C9%20polymorphism:%20from%20enzyme%20kinetics%20to%20clinical%20dose%20recommendations.%20Per%20Med.%202004;1(1):63%E2%80%9384.%2010.1517/17410541.1.1.63)

33. Hicks JK, Bishop JR, Sangkuhl K, Müller DJ, Ji Y, Leckband SG, Clinical Pharmacogenetics Implementation Consortium, et al. Clinical Pharmacogenetics Implementation Consortium (CPIC) guideline for CYP2D6 and CYP2C19 genotypes and dosing of selective serotonin reuptake inhibitors. Clin Pharmacol Ther. 2015;98(2):127–34. 10.1002/cpt.147  [DOI](https://doi.org/10.1002/cpt.147) | [PMC free article](/articles/PMC4512908/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25974703/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Hicks%20JK,%20Bishop%20JR,%20Sangkuhl%20K,%20M%C3%BCller%20DJ,%20Ji%20Y,%20Leckband%20SG,%20Clinical%20Pharmacogenetics%20Implementation%20Consortium,%20et%20al.%20Clinical%20Pharmacogenetics%20Implementation%20Consortium%20(CPIC)%20guideline%20for%20CYP2D6%20and%20CYP2C19%20genotypes%20and%20dosing%20of%20selective%20serotonin%20reuptake%20inhibitors.%20Clin%20Pharmacol%20Ther.%202015;98(2):127%E2%80%9334.%2010.1002/cpt.147)

34. Mulder TAM, van Eerden RAG, de With M, Elens L, Hesselink DA, Matic M, et al. CYP3A4∗22 genotyping in clinical practice: ready for implementation? Front Genet. 2021;12: 711943. 10.3389/fgene.2021.711943  [DOI](https://doi.org/10.3389/fgene.2021.711943) | [PMC free article](/articles/PMC8296839/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34306041/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Mulder%20TAM,%20van%20Eerden%20RAG,%20de%20With%20M,%20Elens%20L,%20Hesselink%20DA,%20Matic%20M,%20et%20al.%20CYP3A4%E2%88%9722%20genotyping%20in%20clinical%20practice:%20ready%20for%20implementation?%20Front%20Genet.%202021;12:%20711943.%2010.3389/fgene.2021.711943)

35. Theken KN, Lee CR, Gong L, Caudle KE, Formea CM, Gaedigk A, et al. Clinical Pharmacogenetics Implementation Consortium guideline (CPIC) for CYP2C9 and nonsteroidal anti-inflammatory drugs. Clin Pharmacol Ther. 2020;108(2):191–200. 10.1002/cpt.1830  [DOI](https://doi.org/10.1002/cpt.1830) | [PMC free article](/articles/PMC8080882/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32189324/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Theken%20KN,%20Lee%20CR,%20Gong%20L,%20Caudle%20KE,%20Formea%20CM,%20Gaedigk%20A,%20et%20al.%20Clinical%20Pharmacogenetics%20Implementation%20Consortium%20guideline%20(CPIC)%20for%20CYP2C9%20and%20nonsteroidal%20anti-inflammatory%20drugs.%20Clin%20Pharmacol%20Ther.%202020;108(2):191%E2%80%93200.%2010.1002/cpt.1830)

36. PharmGKB (2023) Gene-specific information tables for CYP2C19. Available from: https://www.pharmgkb.org/page/cyp2c19RefMaterials.  [https://www.pharmgkb.org/page/cyp2c19RefMaterials](https://www.pharmgkb.org/page/cyp2c19RefMaterials)

37. Savic RM, Jonker DM, Kerbusch T, Karlsson MO. Implementation of a transit compartment model for describing drug absorption in pharmacokinetic studies. J Pharmacokinet Pharmacodyn. 2007;34(5):711–26. 10.1007/s10928-007-9066-0  [DOI](https://doi.org/10.1007/s10928-007-9066-0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17653836/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Savic%20RM,%20Jonker%20DM,%20Kerbusch%20T,%20Karlsson%20MO.%20Implementation%20of%20a%20transit%20compartment%20model%20for%20describing%20drug%20absorption%20in%20pharmacokinetic%20studies.%20J%20Pharmacokinet%20Pharmacodyn.%202007;34(5):711%E2%80%9326.%2010.1007/s10928-007-9066-0)

38. Nayak KP, Brochmann-hanssen E, Way EL. Biological disposition of noscapine. I: kinetics of metabolism, urinary excretion, and organ distribution. J Pharm Sci. 1965;54:191–4. 10.1002/jps.2600540206  [DOI](https://doi.org/10.1002/jps.2600540206) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14293802/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Nayak%20KP,%20Brochmann-hanssen%20E,%20Way%20EL.%20Biological%20disposition%20of%20noscapine.%20I:%20kinetics%20of%20metabolism,%20urinary%20excretion,%20and%20organ%20distribution.%20J%20Pharm%20Sci.%201965;54:191%E2%80%934.%2010.1002/jps.2600540206)

39. Priyadarshani A, Chuttani K, Mittal G, Bhatnagar A. Radiolabeling, biodistribution and gamma scintigraphy of noscapine hydrochloride in normal and polycystic ovary induced rats. J Ovarian Res. 2010;3:10. 10.1186/1757-2215-3-10  [DOI](https://doi.org/10.1186/1757-2215-3-10) | [PMC free article](/articles/PMC2877043/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20420718/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Priyadarshani%20A,%20Chuttani%20K,%20Mittal%20G,%20Bhatnagar%20A.%20Radiolabeling,%20biodistribution%20and%20gamma%20scintigraphy%20of%20noscapine%20hydrochloride%20in%20normal%20and%20polycystic%20ovary%20induced%20rats.%20J%20Ovarian%20Res.%202010;3:10.%2010.1186/1757-2215-3-10)

40. Nourbakhsh F, Askari VR. Biological and pharmacological activities of noscapine: focusing on its receptors and mechanisms. BioFactors. 2021;47(6):975–91. 10.1002/biof.1781  [DOI](https://doi.org/10.1002/biof.1781) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34534373/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Nourbakhsh%20F,%20Askari%20VR.%20Biological%20and%20pharmacological%20activities%20of%20noscapine:%20focusing%20on%20its%20receptors%20and%20mechanisms.%20BioFactors.%202021;47(6):975%E2%80%9391.%2010.1002/biof.1781)

41. Mansoor A, Mahabadi N. Volume of distribution [updated 2022 Jul 25]. In: StatPearls Treasure Island: StatPearls Publishing; 2023. Available from: https://www.ncbi.nlm.nih.gov/books/NBK545280/. Accessed 16 May 2024.  [https://www.ncbi.nlm.nih.gov/books/NBK545280/](https://www.ncbi.nlm.nih.gov/books/NBK545280/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31424864/)

42. Lokhorst B, Rolfes L, Jessurun NT. Interaction of OTC drug noscapine and acenocoumarol and phenprocoumon. Br J Clin Pharmacol. 2019;85(5):1041–3. 10.1111/bcp.13887  [DOI](https://doi.org/10.1111/bcp.13887) | [PMC free article](/articles/PMC6475677/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30809820/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Lokhorst%20B,%20Rolfes%20L,%20Jessurun%20NT.%20Interaction%20of%20OTC%20drug%20noscapine%20and%20acenocoumarol%20and%20phenprocoumon.%20Br%20J%20Clin%20Pharmacol.%202019;85(5):1041%E2%80%933.%2010.1111/bcp.13887)

43. Siddiqi A, Khan DA, Khan FA, Naveed AK. Impact of CYP2C9 genetic polymorphism on warfarin dose requirements in Pakistani population. Pak J Pharm Sci. 2010;23(4):417–22.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/20884456/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Siddiqi%20A,%20Khan%20DA,%20Khan%20FA,%20Naveed%20AK.%20Impact%20of%20CYP2C9%20genetic%20polymorphism%20on%20warfarin%20dose%20requirements%20in%20Pakistani%20population.%20Pak%20J%20Pharm%20Sci.%202010;23(4):417%E2%80%9322.)

44. Petrović J, Pešić V, Lauschke VM. Frequencies of clinically important CYP2C19 and CYP2D6 alleles are graded across Europe. Eur J Hum Genet. 2020;28(1):88–94. 10.1038/s41431-019-0480-8. 10.1038/s41431-019-0480-8  [DOI](https://doi.org/10.1038/s41431-019-0480-8) | [PMC free article](/articles/PMC6906321/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31358955/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Petrovi%C4%87%20J,%20Pe%C5%A1i%C4%87%20V,%20Lauschke%20VM.%20Frequencies%20of%20clinically%20important%20CYP2C19%20and%20CYP2D6%20alleles%20are%20graded%20across%20Europe.%20Eur%20J%20Hum%20Genet.%202020;28(1):88%E2%80%9394.%2010.1038/s41431-019-0480-8.%2010.1038/s41431-019-0480-8)

45. Zhou Y, Ingelman-Sundberg M, Lauschke VM. Worldwide distribution of cytochrome P450 alleles: a meta-analysis of population-scale sequencing projects. Clin Pharmacol Ther. 2017;102(4):688–700. 10.1002/cpt.690  [DOI](https://doi.org/10.1002/cpt.690) | [PMC free article](/articles/PMC5600063/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28378927/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Zhou%20Y,%20Ingelman-Sundberg%20M,%20Lauschke%20VM.%20Worldwide%20distribution%20of%20cytochrome%20P450%20alleles:%20a%20meta-analysis%20of%20population-scale%20sequencing%20projects.%20Clin%20Pharmacol%20Ther.%202017;102(4):688%E2%80%93700.%2010.1002/cpt.690)

46. Rettie AE, Korzekwa KR, Kunze KL, Lawrence RF, Eddy AC, Aoyama T, et al. Hydroxylation of warfarin by human cDNA-expressed cytochrome P-450: a role for P-4502C9 in the etiology of (S)-warfarin-drug interactions. Chem Res Toxicol. 1992;5(1):54–9. 10.1021/tx00025a009  [DOI](https://doi.org/10.1021/tx00025a009) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1581537/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Rettie%20AE,%20Korzekwa%20KR,%20Kunze%20KL,%20Lawrence%20RF,%20Eddy%20AC,%20Aoyama%20T,%20et%20al.%20Hydroxylation%20of%20warfarin%20by%20human%20cDNA-expressed%20cytochrome%20P-450:%20a%20role%20for%20P-4502C9%20in%20the%20etiology%20of%20(S)-warfarin-drug%20interactions.%20Chem%20Res%20Toxicol.%201992;5(1):54%E2%80%939.%2010.1021/tx00025a009)

47. Loebstein R, Yonath H, Peleg D, Almog S, Rotenberg M, Lubetsky A, et al. Interindividual variability in sensitivity to warfarin: nature or nurture? Clin Pharmacol Ther. 2001;70(2):159–64. 10.1067/mcp.2001.117444  [DOI](https://doi.org/10.1067/mcp.2001.117444) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11503010/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Loebstein%20R,%20Yonath%20H,%20Peleg%20D,%20Almog%20S,%20Rotenberg%20M,%20Lubetsky%20A,%20et%20al.%20Interindividual%20variability%20in%20sensitivity%20to%20warfarin:%20nature%20or%20nurture?%20Clin%20Pharmacol%20Ther.%202001;70(2):159%E2%80%9364.%2010.1067/mcp.2001.117444)

48. Jetter A, Kinzig-Schippers M, Skott A, Lazar A, Tomalik-Scharte D, Kirchheiner J, et al. Cytochrome P450 2C9 phenotyping using low-dose tolbutamide. Eur J Clin Pharmacol. 2004;60(3):165–71. 10.1007/s00228-004-0754-z  [DOI](https://doi.org/10.1007/s00228-004-0754-z) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15045499/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Jetter%20A,%20Kinzig-Schippers%20M,%20Skott%20A,%20Lazar%20A,%20Tomalik-Scharte%20D,%20Kirchheiner%20J,%20et%20al.%20Cytochrome%20P450%202C9%20phenotyping%20using%20low-dose%20tolbutamide.%20Eur%20J%20Clin%20Pharmacol.%202004;60(3):165%E2%80%9371.%2010.1007/s00228-004-0754-z)

49. Lv C, Liu C, Yao Z, Gao X, Sun L, Liu J, et al. The clinical pharmacokinetics and pharmacodynamics of warfarin when combined with compound Danshen: a case study for combined treatment of coronary heart diseases with atrial fibrillation. Front Pharmacol. 2017;8:826. 10.3389/fphar.2017.00826  [DOI](https://doi.org/10.3389/fphar.2017.00826) | [PMC free article](/articles/PMC5702344/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29209208/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Lv%20C,%20Liu%20C,%20Yao%20Z,%20Gao%20X,%20Sun%20L,%20Liu%20J,%20et%20al.%20The%20clinical%20pharmacokinetics%20and%20pharmacodynamics%20of%20warfarin%20when%20combined%20with%20compound%20Danshen:%20a%20case%20study%20for%20combined%20treatment%20of%20coronary%20heart%20diseases%20with%20atrial%20fibrillation.%20Front%20Pharmacol.%202017;8:826.%2010.3389/fphar.2017.00826)
