# Functional Variant in a Bitter-Taste Receptor (hTAS2R16) Influences Risk of Alcohol Dependence

## Metadata
**Authors:** Anthony L Hinrichs, Jen C Wang, Bernd Bufe, Jennifer M Kwon, John Budde, Rebecca Allen, Sarah Bertelsen, Whitney Evans, Danielle Dick, John Rice, Tatiana Foroud, John Nurnberger, Jay A Tischfield, Samuel Kuperman, Raymond Crowe, Victor Hesselbrock, Marc Schuckit, Laura Almasy, Bernice Porjesz, Howard J Edenberg, Henri Begleiter, Wolfgang Meyerhof, Laura J Bierut, Alison M Goate
**Journal:** American Journal of Human Genetics
**Date:** 2005 Nov 21
**DOI:** [10.1086/499253](https://doi.org/10.1086/499253)
**PMID:** 16385453
**PMCID:** PMC1380207
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1380207/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC1380207/pdf/AJHGv78p103.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC1380207/pdf/AJHGv78p103.pdf)

## Abstract

A coding single-nucleotide polymorphism (cSNP), K172N, in hTAS2R16, a gene encoding a taste receptor for bitter β-glucopyranosides, shows significant association with alcohol dependence (P=.00018). This gene is located on chromosome 7q in a region reported elsewhere to exhibit linkage with alcohol dependence. The SNP is located in the putative ligand-binding domain and is associated with an increased sensitivity to many bitter β-glucopyranosides in the presence of the N172 allele. Individuals with the ancestral allele K172 are at increased risk of alcohol dependence, regardless of ethnicity. However, this risk allele is uncommon in European Americans (minor-allele frequency [MAF] 0.6%), whereas 45% of African Americans carry the allele (MAF 26%), which makes it a much more significant risk factor in the African American population.

## Material and Methods

### Study Subjects and Assessment

*Linkage sample.—*Alcohol-dependent probands, defined by DSM-IIIR alcohol dependence (American Psychiatric Association [1987](#RF7)) and Feighner-criteria for definite alcoholism (Feighner et al. [1972](#RF20)), were systematically recruited from alcohol-treatment units, and their biological relatives were invited to participate in the study. All subjects were assessed using the Semi-Structured Assessment for the Genetics of Alcoholism (Bucholz et al. [1994](#RF10); Hesselbrock et al. [1999](#RF23)), a semi-structured interview designed as a polydiagnostic instrument that generates Feighner, DSM-IIIR, DSM-IV (American Psychiatric Association [1994](#RF8)), and ICD-10 (World Health Organization [1993](#RF42)) diagnoses of alcohol dependence. These diagnoses are essentially nested, with DSM-IIIR and Feighner definite alcoholism defining the broadest diagnosis, and ICD-10, the narrowest definition of dependence (Culverhouse et al. [2005](#RF16)). Informed consent was obtained from all subjects. A total of 262 families—including 2,310 individuals, with an average of 4.6 alcohol-dependent individuals per pedigree—were selected for genetic-linkage studies. Among these pedigrees, 298 individuals from 35 pedigrees are African American, and 8 pedigrees are of mixed ancestry (by self-report).

*Additional trios.—*The COGA sample contains additional pedigrees with cell lines that were not informative for linkage and had therefore not been selected for the linkage sample. From these, we identified 85 trios consisting of a DSM-IV–defined alcohol-dependent individual and two parents. This sample of “additional trios,” including five African American trios, was typed for SNP *rs846664.*

### Identity-by-Descent (IBD) Sharing

Nonparametric multipoint linkage analysis of independent (*n*-1) affected sibling pairs was conducted using [ASPEX](#RF1), which allows large sibships to be included in analyses. Linkage analyses were performed using the SIBPHASE option, which infers allele sharing if there is ambiguity between identity by state and IBD, by use of marker frequencies in the sample. To avoid biases due to ethnic stratification, maximum-likelihood allele-frequency estimates were obtained, from the USERM13 subroutine of MENDEL (Boehnke [1991](#RF9)), separately for African American and European American pedigrees. Maximum-likelihood estimates of sharing are displayed in [figure 2](#FG2).

### Figure 2.

![Figure  2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b8e6/1380207/67902d1a1ecc/AJHGv78p103fg2.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=1380207_AJHGv78p103fg2.jpg)

Affected-sibling-pair sharing on chromosome 7. Sharing computed, by ASPEX sib_phase, using all parents, with large pedigrees down-weighted to n-1. The solid line represents all pedigrees; the dotted line represents pedigrees in which no individual has a copy of the rare polymorphism; the dashed line represents pedigrees in which at least one individual has a copy of the rare polymorphism.

### Association Analysis

Transmit (Clayton [1999](#RF15)), an extension of the transmission/disequilibrium test (Spielman and Ewens [1996](#RF37)) used to test for association in extended pedigrees to allow for missing parental genotypes, was used to test each SNP individually for evidence of linkage and association. The three closely correlated alcohol-dependence phenotypes—DSM-IIIR and Feighner definite alcoholism, DSM-IV alcohol dependence, and ICD-10 alcohol dependence—were tested to examine the consistency of results. For the additional trios, association was first tested, using Transmit, in this sample alone and was then computed again when combined with the linkage sample.

### SNP Assays

The [dbSNP](#RF3) database was used to identify SNPs within and flanking the *hTAS2R16* gene. Both pyrosequencing ([Biotage Pyrosequencing](#RF2)) and mass spectrometry (Sequenom) methods were used for SNP genotyping. For pyrosequencing, PCR primers were selected using the MacVector 6.5.3 program (Accelrys) to yield 200–500-bp genomic fragments containing the SNP. Standard procedures were followed to generate PCR products. Sequencing primers were designed using the Pyrosequencing Primer Design program. For mass spectrometry, PCR primers, termination mixes, and multiplexing capabilities were determined with Sequenom Spectro Designer software v2.00.17. Standard PCR procedures were used to amplify PCR products. All unincorporated nucleotides were deactivated with shrimp alkaline phosphatase. A primer-extension reaction was then performed with the mass-extension primer and the appropriate termination mix. The primer-extension products were then cleaned with resin and were spotted onto a silicon SpectroChip. The chip was scanned by mass spectrometry (Bruker), and the resulting genotype spectra were analyzed with the Sequenom SpectroTYPER software.

### Sequence Analysis to Identify Additional Variants

The entire coding region of *hTAS2R16* was sequenced in both directions in DNA from 14 people, including one European American and six African Americans homozygous for the minor allele of *rs846664* and seven African Americans heterozygous for the same SNP. Publicly available sequence databases were used to select PCR primers, to amplify the coding exon plus at least 60 bp of flanking intronic sequence. The PCR product was purified using a QIAquick PCR purification kit (Qiagen) to remove excess primers. Purified PCR product was sequenced using the BigDye Terminator Cycle Sequencing method and then was electrophoresed on an ABI3100 automated DNA sequencer (Applied Biosystems [ABI]). Electropherograms were analyzed using ABI DNA Sequencing Analysis Software, version 3.4.

### Heterologous Expression

Generation of the *hTAS2R16* haplotypes and functional analysis in HEK293 cells were performed as described elsewhere (Bufe et al. [2002](#RF12); Soranzo et al. [2005](#RF36)).

## Results

The entire COGA linkage sample was genotyped with four SNPs, including two nonsynonymous cSNPs, *K172N* (*rs846664*) and *R222H* (*rs860170*). Since the three cSNPs showed dramatic differences in allele frequency between African Americans and European Americans, we stratified samples by race for all analyses ([table 1A](#TB1)). Three SNPs were in Hardy-Weinberg equilibrium (HWE) in the founders of the stratified subsets. SNP *rs846664* has a very low minor-allele frequency (MAF) in European Americans and is not in HWE in this sample, but it is in HWE in the African American samples, in which the MAF is much higher. We used Transmit to determine the pairwise disequilibrium between the SNPs and observed high levels of linkage disequilibrium (LD) (*D*^′^⩾0.89). Transmit was also used to test each SNP individually for evidence of association between the SNPs and the alcohol-dependence phenotypes. The nonsynonymous cSNP *rs846664* showed significant association with all three correlated alcohol-dependence diagnoses in the COGA linkage sample ([table 1B](#TB1)). This association appears to be driven by the African American subset (*P*=.004 for DSM-IV dependence); however, the non–African Americans clearly contribute to the significance of the *P* value when the polymorphism is present, because the overall significance is substantially greater in the combined sample (*P*=.0008 for DSM-IV dependence), indicating that the *rs846664* polymorphism is also overtransmitted, when it occurs, in non–African American populations ([table 1B](#TB1)). In the linkage sample, a trend of association was also observed between the synonymous cSNP *rs1204014* and DSM-IV alcohol dependence. The significance of this association increased (*P*=.003) when the test for association was restricted to the subset of African American families. Neither the noncoding SNP *rs978739* nor the nonsynonymous cSNP *rs860170* showed any association with alcohol dependence in our sample. Haplotype analyses using the two nonsynonymous cSNPs (*rs846664* and *rs860170*) and the two significantly associated SNPs (*rs846664* and *rs1204014*) were less significant than the single SNP association results for *rs846664* (data not shown).

### Table 1.

| A. MAFs of Each SNP in Different Sample Sets |  |  |  |  |
| --- | --- | --- | --- | --- |
|   | MAFa for SNP |  |  |  |
| Sample (No. of Families) | rs978739(noncoding) | rs846664(K172N) | rs860170(R222H) | rs1204014(T282T) |
| All families (262) | .35 | .04 | .30 | .08 |
| African American families (35) | .34 | .26 | .10 | .27 |
| European American families (219) | .35 | .006 | .32 | .05 |
| B. Association of Alcohol-Dependence Diagnoses with SNPs in hTAS2R16 in the COGA Linkage Sample |  |  |  |  |
|   | P Valuec for SNP |  |  |  |
| Sample and Alcohol-Dependence Diagnosisb | rs978739(noncoding) | rs846664(K172N) | rs860170(R222H) | rs1204014(T282T) |
| All families: |   |   |   |   |
| COGA (N=1,065) | .277 | .008 | .512 | .256 |
| DSM-IV (N=909) | .114 | .0008 | .307 | .051 |
| ICD10 (N=683) | .369 | .006 | .336 | .186 |
| African American: |   |   |   |   |
| COGA (N=128) | .695 | .024 | .547 | .028 |
| DSM-IV (N=112) | .859 | .004 | .913 | .003 |
| ICD10 (N=87) | .945 | .066 | .843 | .116 |
| European American: |   |   |   |   |
| COGA (N=907) | .193 | .593 | .378 | .608 |
| DSM-IV (N=768) | .061 | .626 | .252 | .826 |
| ICD10 (N=574) | .297 | .890 | .260 | .840 |

Table 1 Caption: Association of Alcohol-Dependence Diagnoses with SNPs in hTAS2R16 in the COGA Linkage Sample

To further explore the role of SNP *rs846664,* we stratified the linkage sample into those families containing the minor allele K172 and those without and performed affected-sibling-pair linkage analysis with [ASPEX](#RF1). The families, including 62 nuclear pedigrees, with the minor allele exhibited IBD sharing of 61.0% at the linkage peak (*D7S1799*) on chromosome 7 for DSM-IIIR and Feighner definite alcoholism, whereas families, including 344 nuclear pedigrees, without the K172 allele exhibited IBD sharing of 55.7% (overall sharing is 56.5%) ([fig. 2](#FG2)). Thus, it appears that, although only 15% of the nuclear families have one or more individuals carrying the minor allele, those families contribute disproportionately to the linkage signal on chromosome 7. The procedure of comparing IBD sharing in pedigrees with and without a putative risk allele is conservative and may, in fact, underestimate the effect of the allele (Li et al. [2004](#RF28)).

To extend our results with *rs846664,* we genotyped this SNP in an independent sample: 85 trios (consisting of a DSM-IV alcohol-dependent individual and two parents) ([table 2](#TB2)). With use of Transmit, the independent trios showed an overtransmission of the K172 allele, with a trend of association with DSM-IV alcohol dependence. When the data from the 85 trios were combined with that of the linkage sample, we observed a *P* value of .00018 for DSM-IV dependence and substantial overtransmission of the K172 allele (79 observed/62 expected transmission). Strong association was also detected with the correlated alcohol-dependence diagnoses—DSM-IIIR and Feighner definite alcoholism (*P*=.002) and ICD-10 dependence (*P*=.002). Sequencing of the coding region of the *hTAS2R16* gene in individuals homozygous and heterozygous for the minor allele (K172) confirmed that there were two nonsynonymous coding changes in the gene—the lysine→asparagine mutation at codon 172 (*rs846664*) and the arginine→histidine mutation at codon 222 (*rs860170*)—and a synonymous cSNP at codon 282 (*rs1204014*) ([fig. 3](#FG3)). No additional SNPs were observed. Given the LD pattern in both European Americans and Africans derived from the [International HapMap Project](#RF4) and the fact that the neighboring genes (*CADPS2* and *SLC13A1*) are each >100 kb from the *hTAS2R16* gene (figs. [4](#FG4) and [5](#FG5)), it is very likely that the alcoholism-susceptibility locus detected in the present study is within *hTAS2R16.*

### Table 2.

| Sample | No. of NuclearPedigrees/Affected Offspring | No. of Pedigrees with Heterozygous Parents | No. of Observed/Expected Transmissions | P Valuea |
| --- | --- | --- | --- | --- |
| COGA linkage sample | 383/758 | 23 | 76/61 | .0008 |
| Additional trios | 85/85 | 3 | 3/1.5 | .083 |
| Combined sample: | 468/843 | 26 | 79/62 | .00018 |
| African American subset | 53/96 | 17 | 59/47.5 | .0011 |
| European American subset | 398/720 | 5 | 6/6.8 | .649 |

Table 2 Caption: Association of DSM-IV Alcohol Dependence with SNP rs846664

### Figure 3.

![Figure  3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b8e6/1380207/c672dedcf9e1/AJHGv78p103fg3.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=1380207_AJHGv78p103fg3.jpg)

Predicted topology of hTAS2R16. E = extracellular domain; TM = transmembrane domain; I = intracellular domain.

### Figure 4.

![Figure  4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b8e6/1380207/f318b3781227/AJHGv78p103fg4.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=1380207_AJHGv78p103fg4.jpg)

Pairwise LD between markers flanking three SNPs in TAS2R16 (500 kb) from Yoruba in population from Ibadan, Nigeria.

### Figure 5.

![Figure  5](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b8e6/1380207/0f70a2422726/AJHGv78p103fg5.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=1380207_AJHGv78p103fg5.jpg)

Pairwise LD between markers flanking two SNPs in TAS2R16 (500 kb) from CEPH (Utah residents with ancestry from northern and western Europe) population.

The *K172N* substitution *rs846664* is located in the extracellular loop 2 between transmembrane domains 4 and 5 ([fig. 3](#FG3)). In G protein–coupled receptors, including the TAS2Rs, this domain has been associated with ligand binding (Adler et al. [2000](#RF6); Pronin et al. [2004](#RF34)). Moreover, experimental evidence has demonstrated that the extracellular loop 2 is involved in the activation of the bitter-taste receptor *hTAS2R43* by its agonist 6–nitrosaccharin (Pronin et al. [2004](#RF34)), which suggests that the *K172N* substitution in the extracellular loop 2 may alter receptor signaling/taste perception. To directly test whether *K172N* and/or *R222H* influence *hTAS2R16* function, we expressed cDNAs coding for the three *hTAS2R16* haplotypes N172 + R222, N172 + H222, and K172 + H222 in HEK293T cells, and we performed functional assays with four bitter-taste agonists as described elsewhere (Bufe et al. [2002](#RF12); Soranzo et al. [2005](#RF36)). We did not analyze the fourth predicted haplotype (K172 + R222) because it was not observed in the entire COGA data set. The concentrations of half-maximal responses (EC_50_) for the N172 + R222 and N172 + H222 constructs were not significantly different for any of the four structurally divergent β-glucopyranosides. In contrast, the K172 + H222 construct exhibited a higher EC_50_ for three of the four compounds tested ([table 3](#TB3)). These results show that the *K172N* polymorphism results in a functional change to the receptor, whereas no functional effect of the *R222H* polymorphism was observed.

### Table 3.

|   | EC50 Values (mM) for Haplotype |  |  | P Value for t Test |  |  |
| --- | --- | --- | --- | --- | --- | --- |
| Agonist | N172 + R222 | N172 + H222 | K172 + H222 | NR/KH | NH/KH | NR/NH |
| 8-Hydroxyquinoline-β-d-glucoside | .6±.3 | .7±.4 | 1.6±.8 | .002 | .012 | .302 |
| Helicin | 1.7±.9 | 1.8±.8 | 2.9±.9 | .017 | .025 | .796 |
| Phenyl-β-d-glucoside | 1.0±.3 | .9±.2 | 1.5±.6 | .024 | .003 | .249 |
| n-Hexyl-β-d-glucoside | 1.6±.9 | 1.6±.8 | 2.0±1.0 | .227 | .294 | .884 |

Table 3 Caption: Average EC50 Values of hTAS2R16 Variants for Four Bitter-Taste Agonists[Note]

## Discussion

Elsewhere, we examined a candidate gene, *CHRM2,* near the linkage peak for alcohol dependence on chromosome 7 and reported evidence of association between genetic variation in this gene and alcohol dependence and related phenotypes (Jones et al. [2004](#RF24); Wang et al. [2004](#RF40)). However, this gene is at the edge of our linkage peak, and several analyses suggest that this association cannot account totally for the observed linkage. Since there are probably many genetic factors influencing the development of alcoholism, we pursued other candidate genes in this region, specifically the *hTAS2r16* bitter-taste–receptor gene.

Our SNP analysis with Transmit detected a significant association between the nonsynonymous cSNP *K172N* (*rs846664*) in *hTAS2R16* and all three nested alcohol-dependence diagnoses in the COGA linkage sample. To extend this result, we genotyped this variant in an independent sample of 85 trios. The association in the trios showed a trend only in the same direction as the original sample, most likely because of the small sample size. When the trios were combined with the linkage sample, the evidence of association increased. After a Bonferroni correction for 12 tests in the overall linkage sample (four SNPs and three alcoholism diagnoses), the DSM-IV diagnosis remained significant. Although the four SNPs are essentially uncorrelated because of differences in allele frequencies, the overall correction is overly conservative, since the alcoholism diagnoses are highly correlated. The analyses of allele-frequency differences in subpopulations were exploratory and would not be significant after a Bonferroni correction.

The LD structure in this region in both European Americans and Africans indicates that the *TAS2R16* gene is the only gene in the LD block containing the SNP *rs846664.* Furthermore, the neighboring genes are >100 kb away from *hTAS2R16.* These observations strongly support the hypothesis that the alcoholism-susceptibility locus detected in this study is within the *hTAS2R16* gene. Since no novel cSNPs were identified from our sequencing analysis and no association was observed between cSNP *R222H* and alcohol dependence, both our genetic analysis and the heterologous expression studies suggests that the K172 allele is the functional variant.

The K172 allele is uncommon in the European Americans in our sample (with an MAF of 0.6%), but 45% of African Americans in our sample carry this allele (MAF 26%). To assess the distribution of the K172 allele across multiple populations, we typed this SNP in the Human Genome Diversity Project–CEPH Human Genome Diversity Cell Line Panel, which includes 1,057 individuals and represents 52 different populations (Cann et al. [2002](#RF13)). The MAF for *rs846664* had a range of 10%–44% in African populations, but it was not detected or was present at very low frequency in non-African populations ([table 4](#TB4)). These frequency data and our own findings that the K172 allele is associated with alcoholism in African American pedigrees (tables[1](#TB1) and [2](#TB2)) suggest that this SNP is likely to be a much more significant risk factor for alcoholism among populations of African origin.

### Table 4.

| Population (No. of Subjects) | Allele K172 | Allele N172 |
| --- | --- | --- |
| Yoruba (24) | 44 | 56 |
| Bantu S.E. (8) | 31 | 69 |
| Biaka Pygmy (36) | 21 | 79 |
| Mandenka (24) | 21 | 79 |
| Mbuti Pygmy (15) | 20 | 80 |
| Bantu N.E. (12) | 17 | 83 |
| San (7) | 14 | 86 |
| Mozabite (30) | 10 | 90 |
| Palestinian (50) | 6 | 94 |
| Maya (25) | 4 | 96 |
| Bedouin (47) | 2 | 98 |
| Brahui (24) | 2 | 98 |
| Adygei (17) | 0 | 100 |
| Balochi (26), Makrani (26), Sindhi (24), Burusho (25), Hazara (22), Kalash (25), Pathan (25) | 0 | 100 |
| Basque (30), French (24) | 0 | 100 |
| Bergamo (27), Sardinian (14), Tuscan (8) | 0 | 100 |
| Cambodian (11) | 0 | 100 |
| Colombia (13) | 0 | 100 |
| Dai (10), Lahu (10), Naxi (10) | 0 | 100 |
| Daur (10) | 0 | 100 |
| Druze (47) | 0 | 100 |
| Han (45) | 0 | 100 |
| Hezhen (10), Oroqen (10), Tu (10) | 0 | 100 |
| Japanese (31) | 0 | 100 |
| Karitiana (24) | 0 | 100 |
| Miaozu (10), Uygur (10), Xibo (10) | 0 | 100 |
| Mongola (10) | 0 | 100 |
| Melanesian (22) | 0 | 100 |
| Orcadian (16) | 0 | 100 |
| Papuan (17) | 0 | 100 |
| Pima (25) | 0 | 100 |
| Russian (25) | 0 | 100 |
| She (10), Tujia (10), Yizu (10) | 0 | 100 |
| Surui (21) | 0 | 100 |
| Yakut (25) | 0 | 100 |

Table 4 Caption: Allele Frequency Distribution (%) of cSNP rs846664 in Different Ethnic Populations

Comparative sequence analysis of several primate species indicates that the ancestral allele at residue 172 of the *hTAS2R16* gene is, in fact, the K172 allele, which is the minor allele in all human populations examined to date ([table 4](#TB4)) (Soranzo et al. [2005](#RF36)). The presence of the minor human allele (K172) in all primate species that have been sequenced suggests that the N172 allele must have undergone massive positive selection during human evolution. The N172 allele is associated with an increased sensitivity to bitter β-glucopyranosides such as salicin, arbutin, and cyanogenic glycosides (Soranzo et al. [2005](#RF36)). Our functional studies confirm the association between the N172 allele and increased sensitivity to bitter β-glucopyranosides ([table 3](#TB3)). The increased sensitivity to bitter compounds in the diet may have driven the positive selection of this allele (Soranzo et al. [2005](#RF36)). These results suggest that the bitter taste of some alcoholic drinks may influence drinking habits and that lack of this bitter-taste variant is a susceptibility factor for alcoholism. However, direct measurements of the perception of bitter taste in subjects homozygous for the K172 or N172 alleles with different bitter-taste agonists and a detailed analysis of their drinking habits are needed to further examine the influence of these alleles on taste perception.

In summary, our genetic and functional data demonstrate that the K172 allele of the polymorphism *rs846664* within the putative ligand-binding domain of the *hTAS2R16* receptor reduces sensitivity of the receptor to bitter-taste stimuli and may thereby influence susceptibility to alcohol dependence.

## Acknowledgments

The COGA (principal investigator: H. Begleiter; coprincipal investigators: L. Bierut, H. Edenberg, V. Hesselbrock, and B. Porjesz) includes nine different centers where data collection, analysis, and storage take place. The nine sites and principal investigators and coinvestigators are: University of Connecticut (V. Hesselbrock); Indiana University (H. Edenberg, J. Nurnberger Jr., P. M. Conneally, and T. Foroud); University of Iowa (S. Kuperman and R. Crowe); State University of New York Health Science Center at Brooklyn (B. Porjesz and H. Begleiter); Washington University (L. Bierut, A. Goate, and J. Rice); University of California at San Diego (M. Schuckit); Howard University (R. Taylor); Rutgers University (J. Tischfield); and Southwest Foundation (L. Almasy). Zhaoxia Ren serves as the National Institute on Alcohol Abuse and Alcoholism (NIAAA) staff collaborator. This national collaborative study is supported by National Institutes of Health grant U10AA08401 from the NIAAA. W. Meyerhof is the recipient of a grant from the German Science Foundation. In memory of Theodore Reich, coprincipal investigator of COGA since its inception and one of the founders of modern psychiatric genetics, we acknowledge his immeasurable and fundamental scientific contributions to COGA and the field.

## References

1. ASPEX, http://aspex.sourceforge.net/ (for affected sib-pair exclusion mapping)  [http://aspex.sourceforge.net/](http://aspex.sourceforge.net/)

2. Biotage Pyrosequencing, http://www.pyrosequencing.com/  [http://www.pyrosequencing.com/](http://www.pyrosequencing.com/)

3. dbSNP, http://www.ncbi.nlm.nih.gov/SNP/  [http://www.ncbi.nlm.nih.gov/SNP/](http://www.ncbi.nlm.nih.gov/SNP/)

4. International HapMap Project, http://www.hapmap.org/  [http://www.hapmap.org/](http://www.hapmap.org/)

5. Online Mendelian Inheritance in Man (OMIM), http://www.ncbi.nlm.nih.gov/Omim/ (for alcoholism, depressive disorder, CHRM2, hTAS2R38, and hTAS2R16)  [http://www.ncbi.nlm.nih.gov/Omim/](http://www.ncbi.nlm.nih.gov/Omim/)
