# Multi-ancestry genome-wide association meta-analysis of buprenorphine treatment response

## Metadata
**Authors:** Christal N Davis, Yousef Khan, Richard C Crist, Rachel Vickers-Smith, Emily E Hartwell, Joel Gelernter, Kyle Kampman, Rachel L Kember, Anne Le Moigne, Celine M Laffont, Henry R Kranzler
**Journal:** Neuropsychopharmacology
**Date:** 2025 May 6
**DOI:** [10.1038/s41386-025-02117-z](https://doi.org/10.1038/s41386-025-02117-z)
**PMID:** 40328918
**PMCID:** PMC12260092
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC12260092/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC12260092/pdf/41386_2025_Article_2117.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC12260092/pdf/41386_2025_Article_2117.pdf)

## Abstract

Although the mu-opioid partial agonist buprenorphine is increasingly being prescribed to treat opioid use disorder, patients’ responses to the drug vary and few clinical and no genetic predictors of treatment response have been identified. We conducted a genome-wide association study (GWAS) meta-analysis of buprenorphine treatment response (defined using urine drug screen results) in 4394 Veterans with opioid use disorder from the VA Million Veteran Program (751 of African-like ancestry [AFR] and 3643 of European-like ancestry [EUR]) and 296 participants from a clinical trial of extended-release buprenorphine (nAFR = 104, nEUR = 192). We conducted within-ancestry GWAS in both cohorts, followed by cross-ancestry, fixed-effects GWAS meta-analyses within and across cohorts. We also examined associations between demographic and clinical characteristics and buprenorphine treatment response. The cross-ancestry meta-analysis of both cohorts identified one genome-wide significant locus (rs149319538) that maps to SLC39A10, a gene that encodes a zinc transporter. Phenome-wide association analyses of the lead variant implicated connectivity of the uncinate fasciculus, a limbic white matter fiber tract. Of the clinical characteristics, only the presence of chronic pain and a lower maximum buprenorphine dosage were related to higher odds of treatment response in adjusted models. We report here the first genome-wide significant variant associated with buprenorphine treatment response. Larger samples are needed to replicate these findings and identify additional clinical and genetic factors that predict buprenorphine treatment efficacy to enable the use of a precision approach to OUD treatment.

Subject terms: Predictive markers, Psychiatric disorders

## Introduction

In 2022, more than six million U.S. individuals aged 12 or older were diagnosed with an opioid use disorder (OUD) [[1](#CR1)]. Individuals who misuse opioids are at high risk of an opioid overdose, which in 2023 resulted in about 81,083 U.S. deaths [[2](#CR2)]. Three medications are currently approved by the Food and Drug Administration (FDA) to treat OUD: the opioid agonists methadone and buprenorphine and the opioid antagonist naltrexone [[3](#CR3)]. Both methadone, a full opioid agonist, and buprenorphine, a partial opioid agonist, alleviate withdrawal symptoms and craving by binding to opioid receptors in the brain without producing the intense high resulting from other opioids. Despite their pharmacological differences, both drugs reduce opioid use and overdose risk [[4](#CR4), [5](#CR5)]. While methadone can be prescribed for treating OUD only in federally sanctioned opioid treatment programs, buprenorphine can be prescribed by any credentialed clinician, leading to a more rapid increase in the prescribing rate of buprenorphine than of methadone [[6](#CR6)].

The factors contributing to variation in buprenorphine treatment outcomes are complex, encompassing demographic, clinical, and genetic factors. Poor adherence to treatment with buprenorphine is a key factor that limits its efficacy, reducing the prevalence of long-term abstinence in buprenorphine-treated patients [[7](#CR7), [8](#CR8)]. Extended-release buprenorphine (BUP-XR) was developed to enhance adherence by providing sustained therapeutic concentrations of the drug over a one-month period. Clinical and demographic factors, such as a history of hepatitis C (HCV) infection, underdosing, and younger age, also predict a poorer response [[9](#CR9), [10](#CR10)]. Understanding the impact of these and other factors is critical for enhancing the effectiveness of buprenorphine therapy and tailoring treatment to the needs of individuals.

Most genetic research on the response to buprenorphine treatment has employed candidate-gene approaches, often in small samples. Several studies of variants in the delta-opioid receptor gene (*OPRD1*) showed associations with treatment response [[11](#CR11)–[14](#CR14)]. The largest of these included 566 individuals of European-like (EUR) ancestry and 77 individuals of African-like (AFR) ancestry [[12](#CR12)]. The first study to adopt a hypothesis-free approach with a genome-wide association study (GWAS) of buprenorphine response [[9](#CR9)], conducted in a sample of 1616 Veterans of EUR ancestry with OUD, identified no genome-wide significant (GWS) loci.

Using a larger sample to provide greater statistical power, we conducted a cross-ancestry GWAS meta-analysis combining data from a cohort of buprenorphine-treated Veterans with OUD (n_AFR_ = 751 and n_EUR_ = 3643) within the Million Veteran Program (MVP) and patients from a clinical trial of extended-release buprenorphine (n_AFR_ = 104 and n_EUR_ = 192) [[15](#CR15)]. This meta-analysis represents the largest, and first multi-ancestry, GWAS of buprenorphine treatment response. These initial findings provide proof of concept of a more robust and comprehensive understanding of the genetic factors influencing therapeutic outcomes for OUD and pave the way for advancing a precision medicine approach to buprenorphine treatment.

## Materials and methods

This study complies with all relevant ethical regulations. It was approved by both the VA Central Institutional Review Board (IRB) and IRBs at all MVP sites. For the Indivior cohort, a centralized IRB reviewed and approved the study protocol. All patients provided written informed consent to participate.

### MVP cohort

The MVP is a large biobank created and maintained by the U.S. Department of Veterans Affairs (VA). The latest release of MVP data includes electronic health record (EHR) information on 963,753 Veterans, about two-thirds of whom also have available genotype data. Using cohort definitions from a previous GWAS in a smaller portion of the MVP sample [[9](#CR9)], EHR data were queried to identify Veterans who had received at least one inpatient or two outpatient International Classification of Diseases (ICD)-9/10 diagnosis codes for OUD and for whom there was genotype data and evidence of at least 28 consecutive days of buprenorphine exposure. Exposure was defined using evidence of filled prescriptions for buprenorphine. For individuals with more than one buprenorphine exposure period, we used data from only the first period. The exposure window for each patient started the day the buprenorphine prescription was filled and ended either when more than two weeks had passed with no new buprenorphine prescription, or a maximum of 180 days had passed. The buprenorphine treatment windows for this study occurred between August 4, 2003 and October 28, 2023.

Urine drug screen (UDS) results included both qualitative and quantitative values that required reconciliation with cutoff thresholds. The median number of UDS completed during treatment was 16 (M = 23.85, SD = 24.40). Treatment responders were individuals with consistently negative UDS results for opioids (including methadone) throughout treatment. Despite variation in UDS reporting, outcome results for nearly all UDS records (~98%) were categorized and included in the analyses. We excluded one patient for whom there was no UDS obtained during treatment, which yielded a total of 4393 individuals (Fig. [1a](#Fig1)).

### Fig. 1. Flow diagram.

![Fig. 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3e85/12260092/998863b5f536/41386_2025_2117_Fig1_HTML.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=12260092_41386_2025_2117_Fig1_HTML.jpg)

Flowchart of inclusion/exclusion criteria for (a) the Million Veteran Program (MVP) and (b) Indivior cohorts. OUD opioid use disorder, UDS urine drug screen, CHOP Children’s Hospital ofPhiladelphia, EUR-like European-like ancestry, AFR-like African-like ancestry.

### Indivior cohort

The Indivior cohort comprised participants in a randomized, double-blind placebo-controlled study of BUP-XR conducted in 36 U.S. treatment centers [[15](#CR15)]. Participants were included if they were age 18–65 and had a Diagnostic and Statistical Manual of Mental Disorders, Fifth Edition (DSM-5) diagnosis [[16](#CR16)] of moderate or severe OUD in the three months prior to study enrollment. Complete eligibility criteria have been described previously [[15](#CR15), [17](#CR17)].

Following a 3-day induction period, participants received at least 7 days of 8–24 mg daily of sublingual buprenorphine. If, during that time, they reported no significant opioid craving or withdrawal symptoms, they were randomly assigned to receive 6 monthly injections of 300 mg, 2 monthly injections of 300 mg plus 4 monthly injections of 100 mg, or volume-matched placebo doses for 6 months. Participants were seen weekly for 24 weeks. At each visit, a UDS, self-reported illicit drug use via the timeline follow-back method, measures of opioid withdrawal and craving, and safety assessments were obtained.

The primary efficacy endpoint in the clinical trial was the percentage of visits (through weeks 5–24) at which participants were opioid abstinent, defined as both UDS and self-reports of illicit opioid use being negative. The secondary endpoint was treatment success, a binary measure of the number of participants who achieved at least 80% opioid abstinence during weeks 5–24. For both endpoints, missing values during the period of weeks 5–24 were imputed as positive. Consistent with the clinical trial outcome definition, we defined treatment responders in the Indivior cohort as individuals who were abstinent for at least 80% of the time from weeks 5–24, with missing UDS values treated as positive. We excluded 27 individuals for whom DNA was not available and 54 individuals who were not of EUR or AFR ancestry, yielding a total of 296 participants for GWAS (Fig. [1b](#Fig1)).

### Demographic and clinical association analyses

In the MVP cohort, we queried EHR data for comorbid disorders and correlates of treatment response using the same approach as in our previous GWAS [[9](#CR9)]. We identified ICD-9/10 codes (Supplementary Methods) for the following comorbid disorders: human immunodeficiency virus (HIV), hepatitis C (HCV) infection, anxiety, depression, posttraumatic stress disorder (PTSD), and chronic pain. We included only diagnoses received prior to or during the buprenorphine treatment window. We also extracted demographic information, including age, sex, body mass index (BMI), and self-reported race and ethnicity, and clinical information, including the median and maximum daily buprenorphine dosage calculated from the dosage of medication, the number of doses, and the number of days’ supply prescribed.

In the Indivior cohort, we used several measures as described in the appendix to the phase 3 BUP-XR trial [[15](#CR15)]. Specific quantitative variables such as the Visual Analog Scale (VAS) craving score and Clinical Opiate Withdrawal Scale (COWS) score were measured at baseline, defined as the last non-missing value prior to the first BUP-XR injection. Demographics and BMI were obtained at screening, defined as 15–30 days prior to the first injection. Presence of comorbidities (HCV, depression, anxiety, bipolar, and any psychiatric disorder) and tobacco, alcohol, and cocaine use were identified from the medical history obtained at screening.

To examine associations of the clinical and demographic factors with treatment response, we conducted logistic regressions in RStudio [[18](#CR18)]. Initial bivariate unadjusted models were run first, and phenotypes that were associated with treatment response at *p* < 0.1 were included together in an adjusted model. Analyses were run separately within each cohort.

### Genotyping, imputation, and GWAS

Procedures for genotyping and imputation in MVP participants have been described previously [[19](#CR19)]. Briefly, genotype data were generated using a custom ThermoFisher Axiom MVP 1.0 platform. Imputation was performed using a hybrid reference panel of the African Genome Resources [[20](#CR20)] and the 1000 Genomes Project [[21](#CR21)]. Genetically inferred ancestry (GIA) was estimated using a random forest classifier trained on 1000 Genomes data and applied to the MVP principal component analysis (PCA) data [[22](#CR22)].

Genotyping of Indivior participants was performed using the Illumina Infinium Global Screening Array-24 kit at the Center for Applied Genomics, Children’s Hospital of Philadelphia. The quality of genetic data was assessed by comparing genetically determined sex with self-reported sex, and identity-by-descent was analyzed to ensure unrelatedness of individuals (Supplementary Fig. [1](#MOESM1)). Imputation was performed on the TOPMed Imputation Server [[23](#CR23)] using the 1000 Genomes Phase 3 reference panel [[21](#CR21)]. PCA was performed on a merged dataset of Indivior and 1000 Genomes individuals, and GIA was assigned to Indivior participants based on the shortest Euclidean distance to the nearest center of each of five superpopulation groups (Supplementary Fig. [2](#MOESM1)). Only individuals of AFR or EUR ancestry were retained due to small numbers of individuals in other GIA groups.

GWAS were conducted within GIA groups in PLINK v2.0 [[24](#CR24)] using logistic regression models. Genetic variants with a minor allele frequency >1%, missing call rates <5%, Hardy-Weinberg equilibrium >1 × 10^–6^, and imputation quality >0.7 were included. We used the first 10 within-ancestry PCs, age, and sex as covariates. In MVP, a sensitivity analysis included maximum daily buprenorphine dosage as an additional covariate due to the large variability in dosage among participants. In the Indivior sample, we performed a sensitivity analysis with treatment responders defined as those with consistently negative UDS throughout the treatment period to mirror the phenotyping procedures of the MVP sample (Supplementary Methods).

### GWAS meta-analysis

Cross-ancestry (AFR + EUR) meta-analyses were performed in each cohort separately using METAL [[25](#CR25)]. For the cross-cohort meta-analysis, the within-ancestry GWAS results from the MVP and Indivior samples were first meta-analyzed together using fixed effects inverse-variance weighted meta-analysis. Next, cross-ancestry meta-analyses were performed on these results across the two samples. As a sensitivity analysis, we meta-analyzed results from the GWAS in the MVP cohort with the sensitivity analysis GWAS conducted in the Indivior sample (i.e., using a 100% abstinence threshold to define treatment response; Supplementary Methods). Genomic control corrections were applied when there was evidence of inflation (λ > 1). We performed linkage disequilibrium clumping of the cross-ancestry meta-analysis results using the 1000 Genomes phase 3 ALL reference panel with a window distance of 3000 kb and r^2^ threshold of 0.10.

### Post-GWAS analyses

Finemapping of the cross-ancestry meta-analysis was performed using the Bayesian approach standard in LocusZoom [[26](#CR26)]. This method identifies the most likely causal variant using Bayes factors and assumes a single causal variant per locus. Using Functional Mapping and Annotation of GWAS (FUMA) [[27](#CR27)], we performed gene mapping based on: (1) position (10 kb), (2) eQTLs (BrainSeq, PsychENCODE, BRAINEAC, and GTEx v8 tissue types), and (3) 3D chromatin interactions (HiC data from the adult cortex, dorsolateral prefrontal cortex, hippocampus, ventricles, and neural progenitor cells). We also calculated the observed scale single-nucleotide polymorphism (SNP)-based heritability using linkage disequilibrium score regression (LDSC). For lead SNPs identified in the cross-ancestry meta-analysis of MVP and Indivior, a SNP-based phenome-wide association study (PheWAS) was conducted using the *ieugwasr* package ([https://github.com/MRCIEU/ieugwasr](https://github.com/MRCIEU/ieugwasr)). To capture phenotypic associations of genetic liability more broadly, we also calculated polygenic scores (PGS) for buprenorphine treatment response in the Yale-Penn cohort using PRS-CS [[28](#CR28)]. Given the GWAS sample’s limited size, we set the global shrinkage parameter to 0.01 as recommended by the software’s creators [[28](#CR28)]. For other options, we used the default settings. A PheWAS was conducted separately for AFR and EUR individuals in the Yale-Penn sample using the within-ancestry meta-analysis results and procedures described previously [[29](#CR29)]. A false discovery rate (FDR) correction was applied to the PheWAS results to account for multiple testing.

## Results

### Demographic and clinical factors

For both the MVP and Indivior samples, the only demographic factors associated with buprenorphine treatment response were EUR ancestry and self-reported White race, the effects of which were nonsignificant after adjusting for other characteristics (Tables [1](#Tab1), [2](#Tab2)). In the MVP sample, a higher daily maximum buprenorphine dosage was associated with reduced odds of responding to treatment, which persisted in models adjusted for genetic ancestry, self-reported race, HCV, anxiety, depression, PTSD, and chronic pain (aOR [95% confidence interval (CI)] = 0.99 [0.98–0.995], *p* = 0.003). In unadjusted models, the presence of several comorbidities was related to treatment response, but only chronic pain remained significant following adjustment (aOR [95% CI] = 1.16 [1.01–1.33], *p* = 0.03), reflecting increased odds of buprenorphine treatment response among individuals with chronic pain. In the adjusted model for the Indivior sample, there were no significant associations with treatment response. Comorbid substance use and substance use disorders were not related to treatment response in the unadjusted or adjusted models in either cohort.

### Table 1.

| Variable | % or Mean | SD or N | Unadj. OR (95% CI) | P value | Adj. OR (95% CI) | P value |
| --- | --- | --- | --- | --- | --- | --- |
| Age | 49.62 | 13.43 | 0.99 (0.99–1.00) | 0.44 | – | – |
| Sex (ref = Male) |   |   | 0.84 (0.68–1.05) | 0.12 | – | – |
| Female | 8.81% | 387 |   |   |   |   |
| Male | 91.19% | 4006 |   |   |   |   |
| Ancestry (ref = AFR) |   | 1.81 (1.54 - 2.12) | 4.95 × 10–13 | 1.13 (0.65–1.96) | 0.67 |   |
| EUR | 82.91% | 3643 |   |   |   |   |
| AFR | 17.09% | 751 |   |   |   |   |
| Hispanic or Latine | 1.08% | 47 | 0.85 (0.47–1.57) | 0.59 | – | – |
| Self-reported race (ref = non-White) |   | 1.83 (1.56 - 2.14) | 1.02 × 10–13 | 1.64 (0.95–2.80) | 0.07 |   |
| Black/African | 16.75% | 723 |   |   |   |   |
| American |   |   |   |   |   |   |
| White | 82.09% | 3544 |   |   |   |   |
| Other | 1.16% | 50 |   |   |   |   |
| Treatment response | 67.65% | 2972 | – | – | – | – |
| Median daily dosage (mg) | 12.63 | 6.86 | 0.99 (0.99–1.00) | 0.23 | – | – |
| Max daily dosage (mg) | 15.19 | 7.48 | 0.99 (0.98–1.00) | 0.01 | 0.99 (0.98–1.00) | <0.01 |
| Body mass index | 28.66 | 5.53 | 1.01 (1.00–1.02) | 0.19 |   |   |
| HIV | 2.25% | 99 | 1.05 (0.69–1.63) | 0.82 | – | – |
| Hepatitis C | 31.13% | 1368 | 0.85 (0.74–0.97) | 0.02 | 0.94 (0.81–1.08) | 0.38 |
| Anxiety | 38.39% | 1709 | 1.23 (1.08–1.40) | <0.01 | 1.06 (0.92–1.23) | 0.40 |
| Depression | 58.92% | 2589 | 1.24 (1.09–1.41) | <0.01 | 1.15 (1.00–1.33) | 0.06 |
| Posttraumatic stress disorder | 50.09% | 2201 | 1.18 (1.04–1.33) | 0.01 | 1.11 (0.97–1.27) | 0.13 |
| Other substance use disorder | 63.25% | 2779 | 0.91 (0.80–1.04) | 0.18 |   |   |
| Chronic pain | 49.54% | 2177 | 1.26 (1.11–1.43) | 3.17 × 10–4 | 1.16 (1.01–1.33) | 0.03 |

Table 1 Caption: Million Veteran Program sample characteristics and their associations with treatment response (N = 4394).

### Table 2.

| Variable | % or Mean | SD or N | Unadj. OR (95% CI) | P value | Adj. OR (95%CI) | P value |
| --- | --- | --- | --- | --- | --- | --- |
| Age | 40.4 | 11.4 | 0.99 (0.97–1.01) | 0.41 | – | – |
| Sex (ref = Male) |   |   | 1.10 (0.67–1.79) | 0.72 | – | – |
| Female | 34.8% | 103 |   |   |   |   |
| Male | 65.2% | 193 |   |   |   |   |
| Ancestry (ref = AFR) |   |   | 2.24 (1.33–3.84) | 2.8 × 10–3 | 0.90 (0.04–10.6) | 0.93 |
| EUR | 64.9% | 192 |   |   |   |   |
| AFR | 35.1% | 104 |   |   |   |   |
| Hispanic or Latine | 2.02% | 6 | 0.87 (0.12–4.52) | 0.87 | – | – |
| Self-reported race (ref = non-White) |   |   | 2.29 (1.36–3.92) | 2.1 × 10–3 | 2.54 (0.22–59.8) | 0.47 |
| Black/African | 34.5% | 102 |   |   |   |   |
| American |   |   |   |   |   |   |
| White | 64.5% | 191 |   |   |   |   |
| Other | 1.01% | 3 |   |   |   |   |
| Treatment response | 36.5% | 108 | – | – | – | – |
| Dose (ref = 100 mg) |   |   | 1.0 (0.62–1.61) | 1.00 | – | – |
| 100 mg | 50.0% | 148 |   |   |   |   |
| 300 mg | 50.0% | 148 |   |   |   |   |
| Body mass index | 25.8 | 4.45 | 1.04 (0.99–1.10) | 0.15 | – | – |
| Hepatitis C | 14.2% | 42 | 0.96 (0.48–1.88) | 0.91 | – | – |
| Depression | 12.8% | 38 | 1.02 (0.49–2.04) | 0.96 | – | – |
| Anxiety | 10.8% | 32 | 0.55 (0.22–1.22) | 0.16 | – | – |
| Bipolar disorder | 2.03% | 6 | 1.76 (0.32–9.67) | 0.49 | – | – |
| Any psychiatric disorder | 19.9% | 59 | 0.72 (0.38–1.31) | 0.29 | – | – |
| Current tobacco use | 85.8% | 254 | 0.58 (0.30–1.13) | 0.11 | – | – |
| Current alcohol use | 54.7% | 162 | 1.05 (0.66–1.70) | 0.83 | – | – |
| Current cocaine use | 37.5% | 111 | 0.91 (0.56–1.48) | 0.71 | – | – |
| Baseline VAS | 6.35 | 12.6 | 0.99 (0.96–1.01) | 0.27 | – | – |
| Baseline COWS | 2.33 | 2.90 | 0.94 (0.86–1.02) | 0.18 | – | – |

Table 2 Caption: Indivior sample characteristics and their associations with treatment response (N = 296).

### GWAS

The primary GWAS, a cross-ancestry meta-analysis of both the MVP and Indivior cohorts, yielded two GWS SNPs—rs149319538 (lead SNP) and rs113613122—within a locus on chromosome 2 (Supplementary Fig. [3](#MOESM1)). Fine mapping indicated that both SNPs were within the credible set of causal variants for the locus (posterior probabilities of 0.491 and 0.329, respectively). Using FUMA, the GWS lead SNP was mapped to *SLC39A10*, a zinc transporter gene, based on 3D chromatin interaction data from neural progenitor cells (FDR *p* value = 4.64 × 10^–7^) and left cerebral ventricular cells (FDR *p* value = 1.04 × 10^–8^). Additionally, there were suggestive associations for 223 SNPs (*p* < 1 × 10^–5^), corresponding to 15 independent loci (Table [3](#Tab3) and Fig. [2](#Fig2)). The meta-analysis had an observed scale SNP-based heritability of 0.18 (SE = 0.09, 95% CI = 0.02–0.35).

### Table 3.

| Chromosome | Position | SNP | P value | Nearest Gene |
| --- | --- | --- | --- | --- |
| 2 | 196279963 | rs149319538 | 2.66 × 10–08 | AC064834.1 |
| 11 | 114180814 | rs2852445 | 6.60 × 10–07 | NNMT |
| 6 | 122683440 | rs9401519 | 7.36 × 10–07 | RNU1-18P |
| 6 | 54294688 | rs669917 | 1.55 × 10–06 | CLNS1AP1 |
| 8 | 130043829 | rs6470688 | 1.70 × 10–06 | LINC00977 |
| 6 | 54695917 | rs12215138 | 2.22 × 10–06 | RP11-524H19.2 |
| 9 | 71905345 | rs147697343 | 2.37 × 10–06 | FAM189A2 |
| 6 | 54365478 | rs573501000 | 4.16 × 10–06 | CLNS1AP1 |
| 13 | 111459167 | rs72663484 | 4.23 × 10–06 | LINC00567 |
| 13 | 102667059 | rs60776232 | 5.00 × 10–06 | FGF14 |
| 2 | 153461929 | rs66706786 | 5.21 × 10–06 | FMNL2 |
| 1 | 2563901 | rs147506475 | 6.47 × 10–06 | MMEL1 |
| 10 | 14767985 | rs113072550 | 6.50 × 10–06 | FAM107B |
| 17 | 29691368 | rs2057769 | 7.72 × 10–06 | NF1 |
| 16 | 83173960 | rs9932942 | 9.50 × 10–06 | CDH13 |
| 4 | 179359799 | rs72706653 | 9.81 × 10–06 | RNA5SP173 |

Table 3 Caption: Significant and suggestive lead SNPs from the cross-ancestry GWAS meta-analysis of the Million Veteran Program and Indivior cohorts.

### Fig. 2. Manhattan plot of the cross-ancestry GWAS meta-analysis of the Million Veteran Program and Indivior samples.

![Fig. 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3e85/12260092/53b4cd747aab/41386_2025_2117_Fig2_HTML.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=12260092_41386_2025_2117_Fig2_HTML.jpg)

The blue dotted line indicates p < 1 × 10–5, and the solid red line represents the genome-wide significance threshold of p < 5 × 10–8. The lead variants for nominally and genome-wide significant loci are annotated, with the significant lead variant annotated in bold.

The MVP GWAS in AFR participants yielded no GWS loci (Supplementary Table [1](#MOESM1) and Supplementary Fig. [4](#MOESM1)). The MVP GWAS in EUR participants yielded one GWS locus on chromosome 2, with the lead SNP identified as rs149319538 (*p* = 2.19 × 10^–8^). In a sensitivity analysis that included maximum daily buprenorphine dosage as a covariate, the lead SNP in EUR individuals remained significant, with no significant findings in the AFR GWAS (Supplementary Fig. [5](#MOESM1)). The cross-ancestry meta-analysis in the MVP cohort identified the same lead SNP—rs149319538 (*p* = 2.74 × 10^–8^)—as in the EUR GWAS (Supplementary Fig. [6](#MOESM1)). Within the Indivior sample, there were no GWS loci in either GIA group (Supplementary Fig. [7](#MOESM1)) or in the cross-ancestry GWAS meta-analysis (Supplementary Table [1](#MOESM1) and Supplementary Fig. [8](#MOESM1)).

Findings for the sensitivity analysis that defined treatment response using a 100% abstinent threshold in the Indivior cohort were consistent with those of the primary GWAS. Within the Indivior sample, using the 100% abstinence treatment response threshold, GWAS identified no significant loci in either GIA group (Supplementary Fig. [9](#MOESM1)) or in the cross-ancestry meta-analysis (Supplementary Fig. [10](#MOESM1)). The cross-ancestry meta-analysis of the Indivior sensitivity analysis and MVP cohort yielded the same GWS locus as the primary meta-analysis (Supplementary Fig. [11](#MOESM1)).

### PheWAS

A PheWAS of rs149319538 identified 22 significant associations after Bonferroni correction (Supplementary Fig. [12](#MOESM1)). These included cardiovascular outcomes (i.e., ischemic stroke and hypertensive heart disease), pain-related conditions (i.e., arthritis and chondropathies), and an inflammatory marker (tumor necrosis factor ligand superfamily). Notably, multiple brain image-derived phenotypes (IDPs) were associated with the SNP. Greater white matter integrity in the uncinate fasciculus, a tract connecting the ventral prefrontal cortex (vPFC) with the amygdala, emerged as a consistent finding. Although prior to FDR correction for the PGS-based PheWAS there were 26 significant associations in EUR and 39 in AFR individuals, none remained significant after FDR correction (Supplementary Figs. [13](#MOESM1), [14](#MOESM1)).

## Discussion

Combining data from AFR and EUR ancestry individuals from the MVP and a large clinical trial of BUP-XR, we identified the first GWS SNP associated with buprenorphine treatment response. Although located in an intergenic region, the lead SNP mapped to *SLC39A10* based on chromatin interaction data from neural progenitor and left cerebral ventricular cells. *SLC39A10* encodes the ZIP10 zinc transporter, which transports zinc into the cytoplasm from the extracellular space in brain, liver, erythroid and kidney tissues [[30](#CR30)]. Emerging evidence from patient populations suggests a role for zinc in the response to opioids. For example, studies have shown that zinc deficiencies are prevalent in patients treated with opioids and that zinc-deficient patients often require a higher dosage of opioids to manage post-surgical pain [[31](#CR31)]. In a randomized controlled trial of zinc supplementation in 68 patients receiving methadone treatment for OUD, supplemental zinc reduced depression, anxiety, and stress, while also decreasing the likelihood of substance use and craving [[32](#CR32)]. These findings, combined with our genetic evidence, suggest that variation linked to *SLC39A10* may influence buprenorphine efficacy by affecting zinc transport and availability.

The lead SNP showed several phenotypic associations using publicly available GWAS data. Specifically, a greater number of effect alleles, indicating an increased propensity for treatment response, was associated in the UKBiobank ([https://www.nealelab.is/uk-biobank](https://www.nealelab.is/uk-biobank)) and FinnGen samples [[33](#CR33)] with pain-related conditions like chrondropathies (e.g., osteoarthritis or spinal disc herniation). This PheWAS finding aligns with the phenotypic results from the MVP sample, where individuals with chronic pain had significantly higher odds of responding to buprenorphine treatment. Additionally, several brain IDPs were related to the lead SNP, including greater white matter integrity in the uncinate fasciculus, a tract that connects the vPFC and amygdala. Despite mixed findings [[34](#CR34)], reduced white matter integrity in the uncinate fasciculus has been associated with adolescent binge alcohol and cannabis use [[35](#CR35)], cognitive deficits in individuals with temporal lobe epilepsy [[36](#CR36)], and social-emotional abnormalities in frontotemporal dementia [[37](#CR37)]. Greater white matter integrity in the fiber bundle, as indicated by greater complexity in fiber orientation, may support emotion regulation and decision-making, potentially contributing to a better response to buprenorphine treatment in individuals with OUD. Further research is needed to understand how variation in white matter microstructure may influence treatment response.

The PGS-based PheWAS in the Yale-Penn sample identified possible associations with various non-opioid substance use, psychiatric, and sociodemographic phenotypes, although none were significant after multiple testing correction. A better understanding of the mechanisms underlying these associations may require larger samples but could help to differentiate patient subgroups that differ in their likelihood of benefitting from buprenorphine treatment. Such risk stratification could enable more targeted and effective treatment strategies for OUD.

In addition to genetic factors, two clinical characteristics—maximum daily dosage and comorbid chronic pain—were associated with treatment response in the MVP cohort. Flexible prescribing in clinical settings may explain why in MVP a higher maximum buprenorphine dosage was associated with a poorer treatment response, which contrasts with other findings that a lower dosage of buprenorphine is associated with poorer treatment outcomes [[10](#CR10)]. That is, prescribers may start buprenorphine at a low dosage with a plan to increase it over time only in individuals whose OUD requires it. This approach may initially underdose some patients, resulting in a greater likelihood of relapse. Because we used a binary measure of abstinence as our treatment outcome, any relapse would result in the individual being classified as not responding to treatment. Consistent with the PheWAS finding of an association with pain-related conditions, individuals with chronic pain diagnoses were more likely to remain abstinent during buprenorphine treatment. Although in a meta-analysis buprenorphine was more effective at treating chronic pain in patients without OUD than those with OUD [[38](#CR38)], even a modest pain reduction could increase treatment adherence and efficacy in OUD patients, potentially explaining the higher rate of treatment response observed in individuals with chronic pain. Future research should explore whether optimized early buprenorphine dosing and integrated pain management can enhance outcomes for individuals with OUD being treated with buprenorphine.

### Limitations

Despite the strengths of this study, including a comparatively large, multi-ancestry sample, several limitations should be noted. The frequency and timing of UDS varied across participants and cohorts, potentially leading to some misclassification of treatment response. Additionally, the use of EHR data in the MVP cohort prevented us from assessing treatment retention as an outcome, and we were unable to evaluate reliably treatment adherence among participants. In addition to variation in the reporting of buprenorphine treatment retention, the lack of systematic methods for collecting and testing UDS in the MVP cohort necessitated that we utilize a binary outcome (response vs. non-response) rather than a continuous measure of treatment response. The MVP data spanned two decades (2003–2023), during which time both the availability of prescribed opioids and the nature of the illicit opioid supply have changed substantially. Because of the limited sample size, we could not evaluate cohort effects, which could influence the findings. Whereas the sample is predominantly male, we could not address possible sex differences in genetic predictors of treatment response, which will be an important area for future research. Finally, although our GWAS meta-analysis increased statistical power relative to our previous GWAS in veterans [[9](#CR9)], the complex genetic architecture of buprenorphine response likely involves many loci of small effect, necessitating considerably larger samples to uncover them.

## Conclusions

In the largest, and first multi-ancestry, GWAS of buprenorphine treatment response, we identified a novel genomic locus that may influence therapeutic outcomes in individuals with OUD. The lead SNP at this locus on chromosome 2 maps to *SLC39A10*, which encodes a zinc transporter, and was associated with greater white matter integrity in the uncinate fasciculus, a fiber tract connecting brain regions involved in emotional regulation and decision-making. In addition to the genetic factors that influence buprenorphine treatment response, we found that OUD patients with chronic pain had greater odds of responding to buprenorphine, likely due to its dual role as both an analgesic and maintenance treatment for OUD. These findings underscore the importance of considering both genetic and clinical factors that shape individual responses to buprenorphine treatment. Further research is needed to elucidate the underlying mechanisms of these associations and inform the development of personalized treatment strategies for OUD.

## Supplementary information

## Acknowledgements

The views expressed in this article are those of the authors and do not necessarily represent the position or policy of the Department of Veterans Affairs or the US Government. The MVP Core Acknowledgment is in Supplementary Material. This work was supported by Merit Review Awards from the US Department of Veterans Affairs (I01 CX001734 and I01 BX004820 and IK2 CX002336); the VISN 4 Mental Illness Research, Education and Clinical Center; NIDA grant P30 DA046345 and NIAAA grant K01 AA028292 (to RLK). Indivior provided DNA samples from a buprenorphine study funded by the company and paid for the samples to be genotyped. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.

## Author contributions

CND and YK conducted the formal analysis and data visualization. CND, YK, and HRK wrote the original draft, with all authors contributing to the review and editing of the manuscript. The study was conceptualized by HRK, CML, ALM, RV-S, RCC, RLK, YK, and CND. HRK and RLK provided supervision. Funding for the study was acquired by KK, HRK, CML, and ALM.

## Data availability

The full summary-level association data for the MVP sample are available through dbGaP: [https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs001672.v1.p1](https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs001672.v1.p1) (accession number phs001672.v1.p1). Data from the meta-analysis of the MVP and Indivior samples are available from [https://zenodo.org/records/15103475](https://zenodo.org/records/15103475).

## Competing interests

ALM and CML are employees of Indivior. HRK is a member of advisory boards for Altimmune and Clearmind Medicine; a consultant to Sobrera Pharmaceuticals and Altimmune; the recipient of research funding and medication supplies for an investigator-initiated study from Alkermes; and a member of the American Society of Clinical Psychopharmacology’s Alcohol Clinical Trials Initiative, which was supported in the last three years by Alkermes, Dicerna, Ethypharm, Imbrium, Indivior, Kinnov, Lilly, Otsuka, and Pear. HRK and ALM are inventors on U.S. provisional patent “Multi-ancestry Genome-wide Association Meta-analysis of Buprenorphine Treatment Response.” The remaining authors have nothing to disclose.

## Footnotes

## Supplementary information

The online version contains supplementary material available at 10.1038/s41386-025-02117-z.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

The full summary-level association data for the MVP sample are available through dbGaP: [https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs001672.v1.p1](https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs001672.v1.p1) (accession number phs001672.v1.p1). Data from the meta-analysis of the MVP and Indivior samples are available from [https://zenodo.org/records/15103475](https://zenodo.org/records/15103475).

### Supplementary Materials

### Data Availability Statement

The full summary-level association data for the MVP sample are available through dbGaP: [https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs001672.v1.p1](https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs001672.v1.p1) (accession number phs001672.v1.p1). Data from the meta-analysis of the MVP and Indivior samples are available from [https://zenodo.org/records/15103475](https://zenodo.org/records/15103475).

## References

1. Substance Abuse and Mental Health Services Administration. (Center for Behavioral Health Statistics and Quality, Substance Abuse and Mental Health Services Administration, 2023) Key substance use and mental health indicators in the United States: Results from the 2022 National Survey on Drug Use and Health. https://www.samhsa.gov/data/report/2022-nsduh-annual-national-report.  [https://www.samhsa.gov/data/report/2022-nsduh-annual-national-report](https://www.samhsa.gov/data/report/2022-nsduh-annual-national-report)

2. Centers for Disease Control and Prevention National Center for Health Statistics. (2024) National Vital Statistics System, Mortality 2018–2022 on CDC WONDER Online Database.  [DOI](https://doi.org/10.15620/cdc/158788) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/39412872/)

3. Bell J, Strang J. Medication treatment of opioid use disorder. Biol Psychiatry. 2020;87:82–8.  [DOI](https://doi.org/10.1016/j.biopsych.2019.06.020) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31420089/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Bell%20J,%20Strang%20J.%20Medication%20treatment%20of%20opioid%20use%20disorder.%20Biol%20Psychiatry.%202020;87:82%E2%80%938.)

4. Larochelle MR, Bernson D, Land T, Stopka TJ, Wang N, Xuan Z, et al. Medication for opioid use disorder after nonfatal opioid overdose and association with mortality: a cohort study. Ann Intern Med. 2018;169:137–45.  [DOI](https://doi.org/10.7326/M17-3107) | [PMC free article](/articles/PMC6387681/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29913516/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Larochelle%20MR,%20Bernson%20D,%20Land%20T,%20Stopka%20TJ,%20Wang%20N,%20Xuan%20Z,%20et%20al.%20Medication%20for%20opioid%20use%20disorder%20after%20nonfatal%20opioid%20overdose%20and%20association%20with%20mortality:%20a%20cohort%20study.%20Ann%20Intern%20Med.%202018;169:137%E2%80%9345.)

5. Sun T, Katenka N, Kogut S, Bratberg J, Rich J, Buchanan A. Evaluation of the effectiveness of buprenorphine-naloxone on opioid overdose and death among insured patients with opioid use disorder in the United States. Pharmacoepidemiology. 10.3390/pharma1030010.  [DOI](https://doi.org/10.3390/pharma1030010) | [PMC free article](/articles/PMC9896393/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36743423/)

6. Olfson M, Zhang V, Schoenbaum M, King M. Trends in buprenorphine treatment in the United States, 2009–2018. JAMA. 2020;323:276–77.  [DOI](https://doi.org/10.1001/jama.2019.18913) | [PMC free article](/articles/PMC6990679/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31961408/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Olfson%20M,%20Zhang%20V,%20Schoenbaum%20M,%20King%20M.%20Trends%20in%20buprenorphine%20treatment%20in%20the%20United%20States,%202009%E2%80%932018.%20JAMA.%202020;323:276%E2%80%9377.)

7. Mattick RP, Breen C, Kimber J, Davoli M. Buprenorphine maintenance versus placebo or methadone maintenance for opioid dependence. Cochrane Database Syst Rev. 2014:CD002207.  [DOI](https://doi.org/10.1002/14651858.CD002207.pub2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15266465/)

8. Kinsky S, Houck PR, Mayes K, Loveland D, Daley D, Schuster JM. A comparison of adherence, outcomes, and costs among opioid use disorder Medicaid patients treated with buprenorphine and methadone: A view from the payer perspective. J Subst Abus Treat. 2019;104:15–21.  [DOI](https://doi.org/10.1016/j.jsat.2019.05.015) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31370980/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Kinsky%20S,%20Houck%20PR,%20Mayes%20K,%20Loveland%20D,%20Daley%20D,%20Schuster%20JM.%20A%20comparison%20of%20adherence,%20outcomes,%20and%20costs%20among%20opioid%20use%20disorder%20Medicaid%20patients%20treated%20with%20buprenorphine%20and%20methadone:%20A%20view%20from%20the%20payer%20perspective.%20J%20Subst%20Abus%20Treat.%202019;104:15%E2%80%9321.)

9. Crist RC, Vickers-Smith R, Kember RL, Rentsch CT, Xu H, Edelman EJ, et al. Analysis of genetic and clinical factors associated with buprenorphine response. Drug Alcohol Depend. 2021;227:109013.  [DOI](https://doi.org/10.1016/j.drugalcdep.2021.109013) | [PMC free article](/articles/PMC9328121/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34488071/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Crist%20RC,%20Vickers-Smith%20R,%20Kember%20RL,%20Rentsch%20CT,%20Xu%20H,%20Edelman%20EJ,%20et%20al.%20Analysis%20of%20genetic%20and%20clinical%20factors%20associated%20with%20buprenorphine%20response.%20Drug%20Alcohol%20Depend.%202021;227:109013.)

10. Grande LA, Cundiff D, Greenwald MK, Murray M, Wright TE, Martin SA. Evidence on buprenorphine dose limits: a review. J Addict Med. 2023;17:509–16.  [DOI](https://doi.org/10.1097/ADM.0000000000001189) | [PMC free article](/articles/PMC10547105/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/37788601/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Grande%20LA,%20Cundiff%20D,%20Greenwald%20MK,%20Murray%20M,%20Wright%20TE,%20Martin%20SA.%20Evidence%20on%20buprenorphine%20dose%20limits:%20a%20review.%20J%20Addict%20Med.%202023;17:509%E2%80%9316.)

11. Clarke TK, Crist RC, Ang A, Ambrose-Lanci LM, Lohoff FW, Saxon AJ, et al. Genetic variation in OPRD1 and the response to treatment for opioid dependence with buprenorphine in European-American females. Pharmacogenomics J. 2014;14:303–08.  [DOI](https://doi.org/10.1038/tpj.2013.30) | [PMC free article](/articles/PMC3988270/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24126707/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Clarke%20TK,%20Crist%20RC,%20Ang%20A,%20Ambrose-Lanci%20LM,%20Lohoff%20FW,%20Saxon%20AJ,%20et%20al.%20Genetic%20variation%20in%20OPRD1%20and%20the%20response%20to%20treatment%20for%20opioid%20dependence%20with%20buprenorphine%20in%20European-American%20females.%20Pharmacogenomics%20J.%202014;14:303%E2%80%9308.)

12. Crist RC, Clarke T-K, Ang A, Ambrose-Lanci LM, Lohoff FW, Saxon AJ, et al. An intronic variant in OPRD1 predicts treatment outcome for opioid dependence in African-Americans. Neuropsychopharmacology. 2013;38:2003–10.  [DOI](https://doi.org/10.1038/npp.2013.99) | [PMC free article](/articles/PMC3746708/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23612435/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Crist%20RC,%20Clarke%20T-K,%20Ang%20A,%20Ambrose-Lanci%20LM,%20Lohoff%20FW,%20Saxon%20AJ,%20et%20al.%20An%20intronic%20variant%20in%20OPRD1%20predicts%20treatment%20outcome%20for%20opioid%20dependence%20in%20African-Americans.%20Neuropsychopharmacology.%202013;38:2003%E2%80%9310.)

13. Crist RC, Phillips KA, Furnari MA, Moran LM, Doyle GA, McNicholas LF, et al. Replication of the pharmacogenetic effect of rs678849 on buprenorphine efficacy in African–Americans with opioid use disorder. Pharmacogenomics J. 2019;19:260–68.  [DOI](https://doi.org/10.1038/s41397-018-0065-x) | [PMC free article](/articles/PMC6486881/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30368523/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Crist%20RC,%20Phillips%20KA,%20Furnari%20MA,%20Moran%20LM,%20Doyle%20GA,%20McNicholas%20LF,%20et%20al.%20Replication%20of%20the%20pharmacogenetic%20effect%20of%20rs678849%20on%20buprenorphine%20efficacy%20in%20African%E2%80%93Americans%20with%20opioid%20use%20disorder.%20Pharmacogenomics%20J.%202019;19:260%E2%80%9368.)

14. Kranzler HR, Lynch KG, Crist RC, Hartwell E, Le Moigne A, Laffont CM, et al. A delta-opioid receptor gene polymorphism moderates the therapeutic response to extended-release buprenorphine in opioid use disorder. Int J Neuropsychopharmacol. 2021;24:89–96.  [DOI](https://doi.org/10.1093/ijnp/pyaa069) | [PMC free article](/articles/PMC7883889/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32920647/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Kranzler%20HR,%20Lynch%20KG,%20Crist%20RC,%20Hartwell%20E,%20Le%20Moigne%20A,%20Laffont%20CM,%20et%20al.%20A%20delta-opioid%20receptor%20gene%20polymorphism%20moderates%20the%20therapeutic%20response%20to%20extended-release%20buprenorphine%20in%20opioid%20use%20disorder.%20Int%20J%20Neuropsychopharmacol.%202021;24:89%E2%80%9396.)

15. Haight BR, Learned SM, Laffont CM, Fudala PJ, Zhao Y, Garofalo AS, et al. Efficacy and safety of a monthly buprenorphine depot injection for opioid use disorder: a multicentre, randomised, double-blind, placebo-controlled, phase 3 trial. Lancet. 2019;393:778–90.  [DOI](https://doi.org/10.1016/S0140-6736(18)32259-1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30792007/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Haight%20BR,%20Learned%20SM,%20Laffont%20CM,%20Fudala%20PJ,%20Zhao%20Y,%20Garofalo%20AS,%20et%20al.%20Efficacy%20and%20safety%20of%20a%20monthly%20buprenorphine%20depot%20injection%20for%20opioid%20use%20disorder:%20a%20multicentre,%20randomised,%20double-blind,%20placebo-controlled,%20phase%203%20trial.%20Lancet.%202019;393:778%E2%80%9390.)

16. American Psychiatric Association. Diagnostic and statistical manual of mental disorders: DSM-5. Fifth edition ed. American Psychiatric Association: Arlington, VA; 2013.

17. FDA Briefing Document. RBP-6000 (extended-release buprenorphine). Accessed March 22, 2020 ed. Joint Meeting of Psychopharmacologic Drugs Advisory Committee and Drug Safety and Risk Management Advisory Committee: October 31; 2017.

18. RStudio Team. Integrated development environment for R. RStudio, PBC: Boston, MA, USA. 2020.

19. Hunter-Zinck H, Shi Y, Li M, Gorman BR, Ji S-G, Sun N, et al. Genotyping array design and data quality control in the million veteran program. Am J Hum Genet. 2020;106:535–48.  [DOI](https://doi.org/10.1016/j.ajhg.2020.03.004) | [PMC free article](/articles/PMC7118558/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32243820/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Hunter-Zinck%20H,%20Shi%20Y,%20Li%20M,%20Gorman%20BR,%20Ji%20S-G,%20Sun%20N,%20et%20al.%20Genotyping%20array%20design%20and%20data%20quality%20control%20in%20the%20million%20veteran%20program.%20Am%20J%20Hum%20Genet.%202020;106:535%E2%80%9348.)

20. Gurdasani D, Carstensen T, Tekola-Ayele F, Pagani L, Tachmazidou I, Hatzikotoulas K, et al. The African Genome Variation Project shapes medical genetics in Africa. Nature. 2015;517:327–32.  [DOI](https://doi.org/10.1038/nature13997) | [PMC free article](/articles/PMC4297536/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25470054/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Gurdasani%20D,%20Carstensen%20T,%20Tekola-Ayele%20F,%20Pagani%20L,%20Tachmazidou%20I,%20Hatzikotoulas%20K,%20et%20al.%20The%20African%20Genome%20Variation%20Project%20shapes%20medical%20genetics%20in%20Africa.%20Nature.%202015;517:327%E2%80%9332.)

21. Auton A, Abecasis GR, Altshuler DM, Durbin RM, Abecasis GR, Bentley DR, et al. A global reference for human genetic variation. Nature. 2015;526:68–74.  [DOI](https://doi.org/10.1038/nature15393) | [PMC free article](/articles/PMC4750478/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26432245/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Auton%20A,%20Abecasis%20GR,%20Altshuler%20DM,%20Durbin%20RM,%20Abecasis%20GR,%20Bentley%20DR,%20et%20al.%20A%20global%20reference%20for%20human%20genetic%20variation.%20Nature.%202015;526:68%E2%80%9374.)

22. Verma A, Huffman JE, Rodriguez A, Conery M, Liu M, Ho YL, et al. Diversity and scale: genetic architecture of 2,068 traits in the VA Million Veteran Program. Science. 2024:385:eadj1182.  [DOI](https://doi.org/10.1126/science.adj1182) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/39024449/)

23. Das S, Forer L, Schönherr S, Sidore C, Locke AE, Kwong A, et al. Next-generation genotype imputation service and methods. Nat Genet. 2016;48:1284–87.  [DOI](https://doi.org/10.1038/ng.3656) | [PMC free article](/articles/PMC5157836/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27571263/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Das%20S,%20Forer%20L,%20Sch%C3%B6nherr%20S,%20Sidore%20C,%20Locke%20AE,%20Kwong%20A,%20et%20al.%20Next-generation%20genotype%20imputation%20service%20and%20methods.%20Nat%20Genet.%202016;48:1284%E2%80%9387.)

24. Chang CC, Chow CC, Tellier LC, Vattikuti S, Purcell SM, Lee JJ. Second-generation PLINK: rising to the challenge of larger and richer datasets. Gigascience. 2015;4:7.  [DOI](https://doi.org/10.1186/s13742-015-0047-8) | [PMC free article](/articles/PMC4342193/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25722852/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Chang%20CC,%20Chow%20CC,%20Tellier%20LC,%20Vattikuti%20S,%20Purcell%20SM,%20Lee%20JJ.%20Second-generation%20PLINK:%20rising%20to%20the%20challenge%20of%20larger%20and%20richer%20datasets.%20Gigascience.%202015;4:7.)

25. Willer CJ, Li Y, Abecasis GR. METAL: fast and efficient meta-analysis of genomewide association scans. Bioinformatics. 2010;26:2190–91.  [DOI](https://doi.org/10.1093/bioinformatics/btq340) | [PMC free article](/articles/PMC2922887/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20616382/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Willer%20CJ,%20Li%20Y,%20Abecasis%20GR.%20METAL:%20fast%20and%20efficient%20meta-analysis%20of%20genomewide%20association%20scans.%20Bioinformatics.%202010;26:2190%E2%80%9391.)

26. Maller JB, McVean G, Byrnes J, Vukcevic D, Palin K, Su Z, et al. Bayesian refinement of association signals for 14 loci in 3 common diseases. Nat Genet. 2012;44:1294–301.  [DOI](https://doi.org/10.1038/ng.2435) | [PMC free article](/articles/PMC3791416/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23104008/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Maller%20JB,%20McVean%20G,%20Byrnes%20J,%20Vukcevic%20D,%20Palin%20K,%20Su%20Z,%20et%20al.%20Bayesian%20refinement%20of%20association%20signals%20for%2014%20loci%20in%203%20common%20diseases.%20Nat%20Genet.%202012;44:1294%E2%80%93301.)

27. Watanabe K, Taskesen E, van Bochoven A, Posthuma D. Functional mapping and annotation of genetic associations with FUMA. Nat Commun. 2017;8:1826.  [DOI](https://doi.org/10.1038/s41467-017-01261-5) | [PMC free article](/articles/PMC5705698/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29184056/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Watanabe%20K,%20Taskesen%20E,%20van%20Bochoven%20A,%20Posthuma%20D.%20Functional%20mapping%20and%20annotation%20of%20genetic%20associations%20with%20FUMA.%20Nat%20Commun.%202017;8:1826.)

28. Ge T, Chen C-Y, Ni Y, Feng Y-CA, Smoller JW. Polygenic prediction via Bayesian regression and continuous shrinkage priors. Nat Commun. 2019;10:1776.  [DOI](https://doi.org/10.1038/s41467-019-09718-5) | [PMC free article](/articles/PMC6467998/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30992449/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Ge%20T,%20Chen%20C-Y,%20Ni%20Y,%20Feng%20Y-CA,%20Smoller%20JW.%20Polygenic%20prediction%20via%20Bayesian%20regression%20and%20continuous%20shrinkage%20priors.%20Nat%20Commun.%202019;10:1776.)

29. Kember RL, Vickers-Smith R, Xu H, Toikumo S, Niarchou M, Zhou H, et al. Cross-ancestry meta-analysis of opioid use disorder uncovers novel loci with predominant effects in brain regions associated with addiction. Nat Neurosci. 2022;25:1279–87.  [DOI](https://doi.org/10.1038/s41593-022-01160-z) | [PMC free article](/articles/PMC9682545/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36171425/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Kember%20RL,%20Vickers-Smith%20R,%20Xu%20H,%20Toikumo%20S,%20Niarchou%20M,%20Zhou%20H,%20et%20al.%20Cross-ancestry%20meta-analysis%20of%20opioid%20use%20disorder%20uncovers%20novel%20loci%20with%20predominant%20effects%20in%20brain%20regions%20associated%20with%20addiction.%20Nat%20Neurosci.%202022;25:1279%E2%80%9387.)

30. Jeong J, Eide DJ. The SLC39 family of zinc transporters. Mol Asp Med. 2013;34:612–9.  [DOI](https://doi.org/10.1016/j.mam.2012.05.011) | [PMC free article](/articles/PMC3602797/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23506894/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Jeong%20J,%20Eide%20DJ.%20The%20SLC39%20family%20of%20zinc%20transporters.%20Mol%20Asp%20Med.%202013;34:612%E2%80%939.)

31. Tantillo TJ, Jagtiani MR, Silverman ER, Bitterman AD, Scuderi GR. Zinc deficiency: a cause of opioid-induced physical dependence and addiction in post-operative total hip arthroplasty patients. J Opioid Manag. 2021;17:145–54.  [DOI](https://doi.org/10.5055/jom.2021.0607) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33890278/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Tantillo%20TJ,%20Jagtiani%20MR,%20Silverman%20ER,%20Bitterman%20AD,%20Scuderi%20GR.%20Zinc%20deficiency:%20a%20cause%20of%20opioid-induced%20physical%20dependence%20and%20addiction%20in%20post-operative%20total%20hip%20arthroplasty%20patients.%20J%20Opioid%20Manag.%202021;17:145%E2%80%9354.)

32. Amini Z, HeidariFarsani E. Investigating the effect of zinc supplementation on probability of relapse and mental health in patients with opioid use disorder undergoing methadone maintenance treatment. Subst Abus Treat Prev Policy. 2023;18:1.  [DOI](https://doi.org/10.1186/s13011-023-00514-5) | [PMC free article](/articles/PMC9817328/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36609286/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Amini%20Z,%20HeidariFarsani%20E.%20Investigating%20the%20effect%20of%20zinc%20supplementation%20on%20probability%20of%20relapse%20and%20mental%20health%20in%20patients%20with%20opioid%20use%20disorder%20undergoing%20methadone%20maintenance%20treatment.%20Subst%20Abus%20Treat%20Prev%20Policy.%202023;18:1.)

33. Kurki MI, Karjalainen J, Palta P, Sipilä TP, Kristiansson K, Donner KM, et al. FinnGen provides genetic insights from a well-phenotyped isolated population. Nature. 2023;613:508–18.  [DOI](https://doi.org/10.1038/s41586-022-05473-8) | [PMC free article](/articles/PMC9849126/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36653562/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Kurki%20MI,%20Karjalainen%20J,%20Palta%20P,%20Sipil%C3%A4%20TP,%20Kristiansson%20K,%20Donner%20KM,%20et%20al.%20FinnGen%20provides%20genetic%20insights%20from%20a%20well-phenotyped%20isolated%20population.%20Nature.%202023;613:508%E2%80%9318.)

34. Von Der Heide RJ, Skipper LM, Klobusicky E, Olson IR. Dissecting the uncinate fasciculus: disorders, controversies and a hypothesis. Brain. 2013;136:1692–707.  [DOI](https://doi.org/10.1093/brain/awt094) | [PMC free article](/articles/PMC3673595/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23649697/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Von%20Der%20Heide%20RJ,%20Skipper%20LM,%20Klobusicky%20E,%20Olson%20IR.%20Dissecting%20the%20uncinate%20fasciculus:%20disorders,%20controversies%20and%20a%20hypothesis.%20Brain.%202013;136:1692%E2%80%93707.)

35. Jacobus J, Squeglia LM, Bava S, Tapert SF. White matter characterization of adolescent binge drinking with and without co-occurring marijuana use: A 3-year investigation. Psychiatry Res: Neuroimaging. 2013;214:374–81.  [DOI](https://doi.org/10.1016/j.pscychresns.2013.07.014) | [PMC free article](/articles/PMC3900025/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24139957/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Jacobus%20J,%20Squeglia%20LM,%20Bava%20S,%20Tapert%20SF.%20White%20matter%20characterization%20of%20adolescent%20binge%20drinking%20with%20and%20without%20co-occurring%20marijuana%20use:%20A%203-year%20investigation.%20Psychiatry%20Res:%20Neuroimaging.%202013;214:374%E2%80%9381.)

36. Diao L, Yu H, Zheng J, Chen Z, Huang D, Yu L. Abnormalities of the uncinate fasciculus correlate with executive dysfunction in patients with left temporal lobe epilepsy. Magn Reson Imaging. 2015;33:544–50.  [DOI](https://doi.org/10.1016/j.mri.2015.02.011) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25705021/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Diao%20L,%20Yu%20H,%20Zheng%20J,%20Chen%20Z,%20Huang%20D,%20Yu%20L.%20Abnormalities%20of%20the%20uncinate%20fasciculus%20correlate%20with%20executive%20dysfunction%20in%20patients%20with%20left%20temporal%20lobe%20epilepsy.%20Magn%20Reson%20Imaging.%202015;33:544%E2%80%9350.)

37. Agosta F, Scola E, Canu E, Marcone A, Magnani G, Sarro L, et al. White matter damage in frontotemporal lobar degeneration spectrum. Cereb Cortex. 2012;22:2705–14.  [DOI](https://doi.org/10.1093/cercor/bhr288) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21988828/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Agosta%20F,%20Scola%20E,%20Canu%20E,%20Marcone%20A,%20Magnani%20G,%20Sarro%20L,%20et%20al.%20White%20matter%20damage%20in%20frontotemporal%20lobar%20degeneration%20spectrum.%20Cereb%20Cortex.%202012;22:2705%E2%80%9314.)

38. Lazaridou A, Paschali M, Edwards RR, Gilligan C. Is buprenorphine effective for chronic pain? a systematic review and meta-analysis. Pain Med. 2020;21:3691–99.  [DOI](https://doi.org/10.1093/pm/pnaa089) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32330264/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Lazaridou%20A,%20Paschali%20M,%20Edwards%20RR,%20Gilligan%20C.%20Is%20buprenorphine%20effective%20for%20chronic%20pain?%20a%20systematic%20review%20and%20meta-analysis.%20Pain%20Med.%202020;21:3691%E2%80%9399.)
