# CD274 (PD-L1) Polymorphisms as Predictors of Efficacy in First-Line Platinum-Based Chemotherapy for Extensive-Stage Small Cell Lung Cancer

## Metadata
**Authors:** Andrés Barba, Laura López-Vilaró, Malena Ferre, Sergio Martinez-Recio, Margarita Majem, Ivana Sullivan, Juliana Salazar
**Journal:** International Journal of Molecular Sciences
**Date:** 2025 Apr 29
**DOI:** [10.3390/ijms26094245](https://doi.org/10.3390/ijms26094245)
**PMID:** 40362483
**PMCID:** PMC12072405
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC12072405/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC12072405/pdf/ijms-26-04245.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC12072405/pdf/ijms-26-04245.pdf)

## Abstract

The cornerstone of first-line treatment in extensive-stage small cell lung cancer (ES-SCLC) is platinum- and etoposide-based chemotherapy. Platinum compounds could immunomodulate the tumor microenvironment in addition to their cytotoxic effect. Genetic variation in immune checkpoint (IC) pathways may predict chemotherapy efficacy. Polymorphisms in the IC genes were determined, and their association with survival was analyzed in 78 patients with ES-SCLC treated with chemotherapy. PD-L1 protein expression in tumor tissue was determined. Three variants in CD274 were associated with better median progression-free survival (mPFS): rs2297136 (hazard ratio [HR] 0.52, 95% CI 0.29–0.93; p = 0.03), rs2282055 (HR 0.23, 95% CI 0.09–0.64; p = 0.005), and rs822336 (HR 0.41, 95% CI 0.23–0.73; p = 0.002). CTLA4 rs231775 was also associated with mPFS (HR 0.30, 95% CI 0.14–0.63; p = 0.002). The variants CD274 rs2297136 and CD274 rs822336 were associated with platinum sensitivity (odds ratio [OR] 0.13, 95% CI 0.02–0.70; p = 0.02, and OR 0.08, 95% CI 0.01–0.46; p = 0.005, respectively). CD274 rs2297136 was also associated with better overall survival (p = 0.02), but not after adjustment for covariates. No association was found between CD274 germline variants and PD-L1 tumor expression. Our results suggest that CD274 and CTLA4 variants may be predictive biomarkers for platinum plus etoposide treatment in ES-SCLC.

Keywords: extensive-stage small cell lung cancer, platinum-based chemotherapy, pharmacogenetics, biomarker, CD274 gene, PD-L1

## 1. Introduction

Small cell lung cancer (SCLC) accounts for 15% of all lung cancer cases [[1](#B1-ijms-26-04245)] and is strongly related to tobacco exposure [[2](#B2-ijms-26-04245)]. Platinum-based chemotherapy plus etoposide is the cornerstone of treatment for patients with untreated SCLC [[3](#B3-ijms-26-04245),[4](#B4-ijms-26-04245)]. The addition of immune checkpoint (IC) inhibitors to chemotherapy has shown moderate improvement in progression-free survival (PFS) and overall survival (OS) in patients with extensive-stage (ES) disease [[3](#B3-ijms-26-04245),[4](#B4-ijms-26-04245),[5](#B5-ijms-26-04245)]. These outcomes could be improved by introducing new drugs [[6](#B6-ijms-26-04245),[7](#B7-ijms-26-04245)] or molecular biomarkers for accurate patient selection into current chemoimmunotherapy. Validation of new molecular classifications of SCLC tumors, such as those based on differential expression of transcriptional regulators [[8](#B8-ijms-26-04245),[9](#B9-ijms-26-04245)], could also contribute to improved outcomes. Unfortunately, there are at present no predictors of efficacy for platinum-based treatments with or without immunotherapy in ES-SCLC. Programmed cell death ligand 1 (PD-L1) expression and tumor mutation burden status have been extensively investigated as immunotherapy biomarkers in some tumors [[10](#B10-ijms-26-04245),[11](#B11-ijms-26-04245)]. Although PD-L1 expression has been validated in non-small cell lung cancer (NSCLC), neither of these two biomarkers has been confirmed in ES-SCLC and are therefore not recommended in therapeutic decision-making [[12](#B12-ijms-26-04245)]. Other molecular markers have been postulated for platinum-based chemotherapy in NSCLC, such as excision repair cross-complementation group 1 (ERCC1) protein expression in tumor tissue [[13](#B13-ijms-26-04245)] and *ERCC1* germline polymorphisms [[14](#B14-ijms-26-04245)]. However, these markers have also not been validated in ES-SCLC [[15](#B15-ijms-26-04245)].

In ES-SCLC, identifying and validating predictors for platinum-based chemotherapy is of great interest, as their implementation would enhance the synergistic clinical effect of chemotherapy and immunotherapy. This proposal is based on the knowledge that the mechanism of action of platinum compounds entails not only the formation of DNA adducts that induce cell cycle arrest and cell apoptosis in proliferating tumor cells [[16](#B16-ijms-26-04245)], but also the modulation of the antitumor immune response [[17](#B17-ijms-26-04245),[18](#B18-ijms-26-04245),[19](#B19-ijms-26-04245)]. The host immune response is tightly controlled by IC molecules both to prevent autoimmunity and to protect against infections and cancer. The programmed cell death protein-1 (PD-1) receptor, expressed on T cells, natural killer cells, monocytes, B cells, and dendritic cells, and its ligand PD-L1, expressed by antigen-presenting cells, are crucial in the negative regulation of the immune response [[20](#B20-ijms-26-04245)]. However, in the tumor microenvironment, tumor cells also express these IC molecules as a mechanism to evade immune surveillance by inactivating cytotoxic T cells [[21](#B21-ijms-26-04245)]. In this context, the efficacy of a platinum-based regimen may be conditioned by functional alterations of IC molecules [[22](#B22-ijms-26-04245)].

Single-nucleotide polymorphisms (SNPs) in genes encoding ICs have been proposed to influence the efficacy of chemotherapy [[23](#B23-ijms-26-04245)] and survival [[24](#B24-ijms-26-04245),[25](#B25-ijms-26-04245)] in Asian patients with NSCLC and SCLC [[26](#B26-ijms-26-04245)]. To the best of our knowledge, however, only the *CD155* rs1058402 variant has been associated with survival in ES-SCLC [[26](#B26-ijms-26-04245)], although this variant has a minor allele frequency of less than 0.05 in European populations. Therefore, the rationale for this study is to explore new treatment-selection biomarkers in the IC pathways that may contribute to identifying a subset of ES-SCLC patients who would benefit most from platinum-containing regimens. Here, we examined a comprehensive set of SNPs in genes coding for receptors and ligands that mediate IC signaling in a retrospective cohort of ES-SCLC patients treated with first-line platinum-etoposide chemotherapy. In addition, we investigated whether there was a correlation between *CD274* (*PD-L1*) genetic variants and PD-L1 protein expression.

## 2. Results

### 2.1. Patients’ Characteristics and Clinical Variants Analyses

[Table 1](#ijms-26-04245-t001) shows the main characteristics of the ES-SCLC patients.

### Table 1.

| Characteristic | n = 78 |
| --- | --- |
| Male (%) | 56 (71.8) |
| Age, median (range), years | 65 (38–86) |
| ECOG performance status (%) |   |
| 0 | 5 (6.4) |
| 1 | 41 (56.2) |
| 2 | 22 (28.2) |
| 3–4 | 8 (10.3) |
| Unknown | 2 (2.6) |
| Smoking (%) |   |
| Current | 52 (66.7) |
| Former | 25 (32.1) |
| Unknown | 1 (1.3) |
| Brain metastases (%) | 16 (21.3) |
| Liver metastases (%) | 35 (44.9) |
| Chemotherapy regimen selected (%) |   |
| Cisplatin | 26 (33.3) |
| Carboplatin | 52 (66.7) |
| Median number of cycles (range) | 6 (2–6) |
| Consolidative thoracic radiation therapy (%) | 19 (24.4) |

Table 1 Caption: Baseline patient characteristics and treatment details.

The median follow-up was 10.0 (interquartile range 6.9–15.8) months. The median OS was 9.7 months (95% confidence interval [CI] 7.8–11.5), and the median PFS was 6.1 months (95% CI 5.3–6.8). Patients with ECOG PS 0-1 had a better median OS than those with ECOG ≥ 2 (11.5 months [95% CI 6.9–16.2] versus 8.2 months [95% CI 7.6–8.7]; *p* = 0.001). No differences were observed in median PFS according to ECOG (*p* = 0.3). Patients treated with consolidative thoracic radiation therapy (TRT) had a better median OS (17.1 months [95% CI 7.0–27.1] versus 8.4 months [95% CI 6.6–10.3]; *p* = 0.002) and median PFS (7.3 months [95% CI 6.2–8.4] versus 5.2 months [95% CI 4.6–5.9]; *p* = 0.002) than patients who did not. No differences were found in median PFS or median OS according to sex (*p* = 0.7 and *p* = 0.4, respectively) or type of platinum (*p* = 0.08 and *p* = 0.07, respectively).

The analyses showed that 36.5% of patients were platinum-based chemotherapy sensitive. There was a significant positive association between platinum sensitivity and consolidative TRT (odds ratio [OR] 0.16, 95% CI 0.05–0.49; *p* < 0.001). Numerical differences were found between platinum sensitivity and cisplatin treatment (OR 0.37, 95% CI 0.14–1.00; *p* = 0.05).

### 2.2. Polymorphisms of Immune Checkpoint Genes and Progression-Free Survival Analyses

The full results of the univariate analyses between the genetic variants of IC signaling molecules and PFS are shown in [Table S1](#app1-ijms-26-04245). *CD274* rs2282055 was the only variant that reached the Bonferroni-adjusted significance level for multiple comparisons (*p* < 0.001). Multivariate analyses for PFS showed significant associations for the rs2297136, rs2282055, and rs822336 variants in the *CD274* gene and the *CTLA4* rs231775 variant ([Table 2](#ijms-26-04245-t002)).

### Table 2.

|   |   |   | Progression-Free Survival |  |   |   |
| --- | --- | --- | --- | --- | --- | --- |
|   |   |   | Univariate Analysis |  | Multivariate Analysis |  |
| SNP | n = 75 | mPFS (95% CI), Months | HR (95% CI) | pa Value | HR (95% CI) | pb Value |
| CD274 rs2297136 |   |   |   |   |   |   |
| AA | 18 | 4.6 (3.8–5.3) | Reference (1) | 0.007 |   |   |
| AG | 43 | 6.1 (5.2–6.9) | 0.58 (0.33–1.02) |   |   |   |
| GG | 14 | 6.8 (6.2–7.4) | 0.30 (0.14–0.66) |   |   |   |
|   |   |   |   |   |   |   |
| AA | 18 | 4.6 (3.8–5.3) | Reference (1) |   |   |   |
| AG-GG | 57 | 6.3 (5.7–6.8) | 0.50 (0.29–0.87) | 0.01 * | 0.52 (0.29–0.93) | 0.03 |
| CD274 rs2282055 |   |   |   |   |   |   |
| TT | 42 | 6.1 (5.1–7.0) | Reference (1) | 0.001 |   |   |
| TG | 28 | 6.1 (4.4–7.8) | 0.79 (0.48–1.28) |   |   |   |
| GG | 5 | 4.6 (0.0–9.7) | 4.48 (1.64–12.23) |   |   |   |
|   |   |   |   |   |   |   |
| GG | 5 | 4.6 (0.0–9.7) | Reference (1) |   |   |   |
| TT-TG | 70 | 6.1 (5.6–6.6) | 0.20 (0.08–0.55) | <0.001 ** | 0.23 (0.09–0.64) | 0.005 |
| CD274 rs822336 |   |   |   |   |   |   |
| GG | 21 | 4.8 (3.9–5.8) | Reference (1) | 0.02 |   |   |
| GC | 35 | 6.5 (5.9–7.0) | 0.48 (0.27–0.84) |   |   |   |
| CC | 19 | 6.6 (5.9–7.3) | 0.50 (0.26–0.96) |   |   |   |
|   |   |   |   |   |   |   |
| GG | 21 | 4.8 (3.9–5.8) | Reference (1) |   |   |   |
| GC-CC | 54 | 6.5 (6.0–7.0) | 0.48 (0.29–0.82) | 0.006 * | 0.41 (0.23–0.73) | 0.002 |
| CTLA4 rs231775 |   |   |   |   |   |   |
| AA | 34 | 6.1 (5.9–6.3) | Reference (1) | 0.005 |   |   |
| AG | 32 | 6.3 (4.6–7.9) | 0.94 (0.58–1.55) |   |   |   |
| GG | 9 | 4.6 (3.9–5.3) | 3.10 (1.43–6.72) |   |   |   |
|   |   |   |   |   |   |   |
| GG | 9 | 4.6 (3.9–5.3) | Reference (1) |   |   |   |
| AA-AG | 66 | 6.1 (5.7–6.5) | 0.31 (0.15–0.07) | 0.001 ** | 0.30 (0.14–0.63) | 0.002 |

Table 2 Caption: Analyses between the significant immune checkpoint genetic variants and progression-free survival of extensive-stage small cell lung cancer patients.

For *CD274* rs2297136, patients with the AG and GG genotypes had a longer median PFS (6.3 months) than patients with the AA genotype (4.6 months) (hazard ratio [HR] 0.52, 95% CI 0.29–0.93; *p* = 0.03 in a dominant model) ([Figure 1](#ijms-26-04245-f001)A). For *CD274* rs2282055, patients with TT and TG genotypes had a longer median PFS (6.1 months) than patients with the GG genotype (4.6 months) (HR 0.23, 95% CI 0.09–0.64; *p* = 0.005 in a recessive model) ([Figure 1](#ijms-26-04245-f001)B). For *CD274* rs822336, patients with the GC and CC genotypes had a longer median PFS (6.5 months) than patients with the GG genotype (4.8 months) (HR 0.41, 95% CI 0.23–0.73; *p* = 0.002 in a dominant model) ([Figure 1](#ijms-26-04245-f001)C). For *CTLA4* rs231775, patients with the AA and AG genotypes had a longer median PFS (6.1 months) than patients with the GG genotype (4.6 months) (HR 0.30, 95% CI 0.14–0.63; *p* = 0.002 in a recessive model) ([Figure 1](#ijms-26-04245-f001)D).

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/da67/12072405/50113bec9f6b/ijms-26-04245-g001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=12072405_ijms-26-04245-g001.jpg)

Progression-free survival curves of extensive stage-small cell lung cancer patients (n = 75) according to the (A) CD274 rs2297136 variant in a dominant model of inheritance, (B) CD274 rs2282055 variant in a recessive model of inheritance, (C) CD274 rs822336 variant in a dominant model of inheritance, and (D) CTLA4 rs231775 variant in a recessive model of inheritance.

### 2.3. Polymorphisms of Immune Checkpoint Genes and Overall Survival Analyses

Regarding OS, the full results of the univariate analyses are shown in [Table S2](#app1-ijms-26-04245). Multivariate analyses showed marginal differences for *CD274* rs2297136: patients with the AG and GG genotypes had longer median OS (10.4 months) than patients with the AA genotype (7.5 months) (HR 0.62, 95% CI 0.36–1.09; *p* = 0.1 in a dominant model) ([Figure S1](#app1-ijms-26-04245)).

### 2.4. Haplotype Analyses for CD274 Variants and Survival

Haplotype analyses were performed to determine the effect of *CD274* variants (rs2297136|rs2282055|rs822336) on PFS and OS. The GGG haplotype (frequency = 6%) was associated with median PFS (*p* = 0.009 univariate; *p* = 0.01 multivariate) and OS (*p* < 0.001 univariate; *p* < 0.001 multivariate) ([Table S3](#app1-ijms-26-04245)).

### 2.5. Polymorphisms of Immune Checkpoint Genes and Platinum Sensitivity Analyses

Analyses between platinum sensitivity and the four genetic variants associated with PFS showed significant associations for rs2297136 and rs822336 variants in the *CD274* gene when the covariates were included ([Table S4](#app1-ijms-26-04245)). Patients carrying the *CD274* rs2297136-G allele (OR 0.13, 95% CI 0.02–0.70; *p* = 0.02 in a dominant model) and those carrying the *CD274* rs822336-C allele (OR 0.08, 95% CI 0.01–0.46; *p* = 0.005 in a dominant model) were more likely to be platinum-sensitive.

### 2.6. Analyses of PD-L1 Protein Expression and CD274 Polymorphisms

Fifty-two tumor tissue samples (66.7%) were evaluable for PD-L1 expression analysis. The positivity of PD-L1 expression by the Tumor Proportion Score (TPS) was 3.8%, and no patients presented PD-L1 ≥ 50%. The positivity of PD-L1 expression by the Combined Positive Score (CPS) was 23.1%.

There was no correlation between *CD274* germline variants and PD-L1 tumor expression.

## 3. Discussion

In this retrospective cohort of ES-SCLC patients treated with first-line platinum-etoposide chemotherapy, some variants in the *CD274* (also known as *PD-L1*) and *CTLA4* genes were significantly associated with PFS.

ES-SCLC is an aggressive disease with a poor prognosis [[27](#B27-ijms-26-04245)]. As the addition of IC inhibitors to platinum-based chemotherapy has provided a modest benefit, several treatment options are currently being investigated to improve survival in these patients [[6](#B6-ijms-26-04245),[7](#B7-ijms-26-04245)]. In this scenario, platinum doublet remains the cornerstone of first-line treatment and a second-line choice for those patients with relapsed disease who are sensitive to platinum [[28](#B28-ijms-26-04245)]. However, to date, no biomarkers have been identified to explain the interindividual variability in response to platinum compounds in ES-SCLC.

In lung cancer, few studies [[23](#B23-ijms-26-04245),[24](#B24-ijms-26-04245),[25](#B25-ijms-26-04245),[26](#B26-ijms-26-04245)] have analyzed genetic variants in genes of the immune checkpoint signaling pathway based on the immunomodulatory effects of chemotherapy on the tumor microenvironment [[17](#B17-ijms-26-04245),[18](#B18-ijms-26-04245),[19](#B19-ijms-26-04245)]. These studies have identified associations between chemotherapy response or survival and the *CD155* rs1058402 and *CD226* rs763361 variants in SCLC, although only the *CD155* rs1058402 variant was associated with OS in extensive-stage disease [[26](#B26-ijms-26-04245)], and variants in the *CD274* gene in NSCLC [[23](#B23-ijms-26-04245),[24](#B24-ijms-26-04245),[25](#B25-ijms-26-04245)]. In patients with advanced-stage NSCLC, the *CD274* rs2297136-G allele was found to be associated with chemotherapy response and OS [[23](#B23-ijms-26-04245)]. In addition, in postoperative NSCLC patients treated with adjuvant platinum-based chemotherapy, patients carrying GC and CC genotypes for *CD274* rs822336 showed better survival [[25](#B25-ijms-26-04245)].

Our results are consistent with those reported in NSCLC [[23](#B23-ijms-26-04245),[25](#B25-ijms-26-04245)] and show that variants in the *CD274* gene may be feasible biomarkers of platinum-based chemotherapy efficacy in ES-SCLC. Patients carrying the rs2297136-G allele and patients carrying the rs822336-C allele had a longer median PFS, and they were more likely to respond to platinum-based chemotherapy. In addition, carriers of the rs2297136-G allele had a longer median OS, although this was not statistically significant, a result in line with the significant association observed with OS of the low-frequency haplotype GGG that includes the protective G allele for this variant. These findings suggest that the *CD274* variants might help identify patients with a higher probability of longer PFS and platinum-sensitive disease, thus lending sensitivity to second-line treatments, especially in those patients who received platinum rechallenge [[29](#B29-ijms-26-04245)].

The *CD274* gene encodes the immune inhibitory ligand PD-L1, which binds to the PD-1 receptor, blocking T-cell activation to prevent autoimmunity, a mechanism also used by the tumor to escape immune surveillance [[30](#B30-ijms-26-04245)]. *CD274* rs2282055 is an intron variant [[31](#B31-ijms-26-04245)]. The prediction scores (rank 1f and score 0.55) of the variant from the Regulome database [[32](#B32-ijms-26-04245)] show that there is evidence of its putative functionality. The *CD274* rs2297136 variant located in the 3′-UTR region may alter the microRNA binding sites for miR-324-5p and miR-296-5p and thus affect mRNA translational repression or destabilization [[33](#B33-ijms-26-04245)]. In silico studies predicted a higher minimum free energy of hybridization of miR-324-5p and mRNA for the G allele [[23](#B23-ijms-26-04245)], and in vitro studies showed that miR-296-5p suppressed the expression of the construct containing the G allele [[34](#B34-ijms-26-04245)]. *CD274* rs822336 is an upstream variant, and the C allele may confer reduced PD-L1 mRNA expression [[25](#B25-ijms-26-04245)] and promoter activity [[24](#B24-ijms-26-04245)]. Thus, these germline *CD274* variants may be functional and could lead to decreased PD-L1 protein levels.

Based on these findings, PD-L1 in immune cells may therefore have an effect on the response to platinum compounds. Notably, in SCLC, PD-L1 expression occurs predominantly in immune cells rather than in tumor cells [[35](#B35-ijms-26-04245)], so down-regulation of the PD-1/PD-L1 pathway may induce host immune responses and contribute to the effect of the antitumor activity of chemotherapy observed in our study. However, in the present study, no correlation was found between *CD274* germline variants and PD-L1 expression in a subgroup of patients. It should be noted that PD-L1 expression may be modified by therapeutic exposure to chemotherapy with or without immunotherapy or by disease progression. In the tumor microenvironment, transcription, post-transcription, and post-translation processes determine PD-L1 expression [[36](#B36-ijms-26-04245)]. Thus, the low levels of PD-L1 protein expression and its possible temporal variation throughout disease progression may explain the lack of correlation observed in our study between germline *CD274* SNPs and protein expression at diagnosis.

Another result in this study was that patients with the AA and AG genotypes for the *CTLA4* rs231775 variant had longer median PFS than homozygous patients for the G allele. The *CTLA4* gene encodes for a protein that is expressed in activated T cells and is involved in attenuating the immune response. The *CTLA4* rs231775 A > G is a missense polymorphism (p.Thr17Ala) located in exon 1 [[37](#B37-ijms-26-04245)]. However, in vitro experiments have shown that the protein expression is reduced in GG homozygotes compared to AA homozygotes [[38](#B38-ijms-26-04245)], and further investigation is required before considering the variant a biomarker of platinum efficacy.

Platinum compounds have a cytotoxic effect on cancer cells but also act as host immune modulators. Cisplatin has been shown to increase major histocompatibility complex class I, enhance T-cell infiltration, proliferation, and cytotoxic activity, and decrease the expression of immunosuppressive cells, including myeloid-derived suppressor cells and Treg cells [[17](#B17-ijms-26-04245)].

The findings of the present study are consistent with this knowledge and support the suggestion that the common genetic variation in the IC pathway would help identify those ES-SCLC patients treated with platinum compounds who have a favorable response. The study cohort was collected prior to the approval of combination chemotherapy with immunotherapy in ES-SCLC. This design allowed us to investigate predictive biomarkers for platinum-based chemotherapy without the bias that adding immunotherapy might have introduced, especially in a study focusing on the evaluation of genes in IC signaling pathways.

This pharmacogenetic study has some limitations. First, the sample size was modest, although the statistical power was sufficient for the objectives of the study. In addition, it was a cohort of patients from a single center and homogeneously treated. Second, the analyses did not show a correlation between germline *CD274* variants and PD-L1 expression in the tumor at diagnosis, probably due to low PD-L1 protein expression in the tumor, and further experiments would be necessary to elucidate the underlying mechanisms.

## 4. Materials and Methods

### 4.1. Study Population

This was a retrospective association study involving 78 patients with ES-SCLC treated between September 2010 and May 2021 at Hospital de la Santa Creu i Sant Pau (Barcelona, Spain). The study included patients treated with standard carboplatin/cisplatin-based chemotherapy plus etoposide for first-line treatment who had received at least two cycles of platinum-based chemotherapy and excluded those who received IC treatment. The inclusion of patients treated with consolidative TRT after first-line platinum-based chemotherapy was permitted. Patients previously treated with chemotherapy for limited-stage SCLC were excluded from the study.

### 4.2. Definitions of Clinical Endpoints

PFS was defined as the time from the date of initiation of chemotherapy to the date of disease progression, as defined by RECIST 1.1 [[39](#B39-ijms-26-04245)], or death from any cause, whichever occurred first. OS was defined as the time from the date of initiation of chemotherapy to the date of death from any cause. Platinum sensitivity status was dichotomized into platinum-sensitive and platinum-resistant. Platinum-sensitive was defined as progression ≥ 90 days from the last dose of platinum-based chemotherapy, and platinum-resistant was defined as progression < 90 days from the last dose of platinum-based chemotherapy [[40](#B40-ijms-26-04245)]. Patients who progressed during platinum-based chemotherapy (platinum-refractory) were classified as platinum-resistant patients.

### 4.3. Selection of Polymorphisms

Fifteen SNPs were selected in the *CD274* (*PD-L1*), *PDCD1*, *CTLA4*, *CD226*, and *LAG3* genes encoding IC signaling molecules. These are SNPs with functional evidence [[41](#B41-ijms-26-04245)] or previously associated with susceptibility, prognosis, or treatment response in autoimmune diseases and some cancers (see [Table 3](#ijms-26-04245-t003)).

### Table 3.

| Gene Symbol | Reference SNP | Variant Description | References for Rationale |
| --- | --- | --- | --- |
| CD274 (PD-L1) | rs4143815 | 3′-UTR | [23,24,34,42] |
| rs2297136 | 3′-UTR | [23,34,43] |  |
| rs2282055 | Intron | [42,44] |  |
| rs822336 | 2 KB Upstream | [24,25,43] |  |
| PDCD1 | rs2227981 | Synonymous | [45,46] |
| rs10204525 | 3′-UTR | [47,48] |  |
| rs11568821 | Intron | [45,49] |  |
| rs7421861 | Intron | [45,48] |  |
| CTLA4 | rs4553808 | 2 KB Upstream | [50] |
| rs231775 | Missense | [50,51] |  |
| rs3087243 | 500B Downstream | [52] |  |
| CD226 | rs763361 | Missense | [26,53] |
| LAG3 | rs2365095 | Intron | [54] |
| rs870849 | Missense | [55] |  |
| rs3782735 | Intron | [56] |  |

Table 3 Caption: Main characteristics of the immune checkpoint genetic variants analyzed.

### 4.4. Genotyping

DNA from whole blood samples was extracted automatically (Autopure LS system, Qiagen, Hilden, Germany). The genotyping of the samples was carried out by real-time PCR using TaqMan^®^ SNP genotyping assays on a 7900 HT Real Time PCR System (Applied Biosystems, Foster City, CA, USA) according to the manufacturers’ instructions. All the samples were successfully genotyped.

The allele frequencies of the SNPs were similar to those observed in European populations [[57](#B57-ijms-26-04245)]. The genotype frequencies were in Hardy–Weinberg equilibrium, except for *CD226* rs763361 (*p* < 0.05), which was removed from the analyses. The 1000 Genomes Project Phase 3 linkage disequilibrium data for *CD274* gene variants in European populations are shown in [Table S5](#app1-ijms-26-04245).

### 4.5. Immunohistochemistry Analysis

PD-L1 protein expression was assessed through immunohistochemistry on formalin-fixed paraffin-embedded (FFPE) tumor tissue sections using the PD-L1 IHC 22C3 pharmDx kit (Dako Ltd., Cheshire, UK). FFPE tissue sections, 4 µm thick, were deparaffinized and rehydrated. Antigen retrieval was performed by heating the slides for 20 min at 97 °C in a low-pH solution using a PT Link system (Dako Ltd., Cheshire, UK). The sections were then stained automatically with the Dako Autostainer Link 48 (Agilent Technologies, Santa Clara, CA, USA) using the ready-to-use anti-human PD-L1 mouse monoclonal antibody, clone 22C3 (Dako Ltd., Cheshire, UK). This was followed by sequential incubation with a mouse anti-rabbit IgG linker and a secondary antibody and horseradish peroxidase conjugated to a dextran polymer backbone. The slides were then revealed using 3,3′-diaminobenzidine tetrahydrochloride chromogen, counterstained with hematoxylin, and coverslipped. Tonsil tissue served as the positive external control.

Two independent investigators (LL and MF) evaluated the PD-L1 staining, and discordant cases were reviewed for consensus. PD-L1 expression was interpreted using two scoring systems: the TPS and the CPS. The TPS was defined as the percentage of viable tumor cells showing partial or complete membrane staining of any intensity. A specimen was considered PD-L1 positive if the TPS was ≥1%, and high PD-L1 expression was defined as TPS ≥ 50%. The CPS was calculated by dividing the total number of PD-L1-positive cells (membrane-staining tumor cells, cytoplasm-staining lymphocytes, and macrophages) by the total number of viable tumor cells, multiplied by 100. A specimen was considered PD-L1 positive if the CPS was ≥1. Considering the expected low proportion of TPS positivity [[35](#B35-ijms-26-04245),[58](#B58-ijms-26-04245)], CPS was performed to evaluate PD-L1 expression in tumor-inflammatory immune cells.

### 4.6. Statistical Analysis

Deviations from Hardy–Weinberg equilibrium were assessed for each SNP using the χ^2^ test. Codominant, dominant, or recessive genetic models of inheritance were used for each SNP as appropriate. Survival curves and median survival were calculated using the Kaplan–Meier method. Differences in survival were tested using the log-rank test. Cox proportional hazards regression models were adjusted for multivariate analysis with Eastern Cooperative Oncology Group (ECOG) performance status (PS) (2–4 versus 0–1) and consolidative TRT as covariates. The results are expressed as HRsand 95% CIs. The rate of sensitivity to platinum was analyzed for those SNPs for which we observed significant differences in median PFS in our analyses. Differences in platinum sensitivity were calculated using the χ^2^ test and logistic regression analyses, considering platinum compound (cisplatin or carboplatin) and consolidative TRT as covariates. The results are expressed as ORs with 95% CIs. Progression data for three patients and platinum sensitivity status for four patients were not available and were therefore excluded from the analysis. The size of our sample had a statistical power of 75–87% to detect the effect of polymorphisms with a w = 0.3–0.35. (α = 0.05) (G*power version 3.1.9.7, Düsseldorf, Germany) [[59](#B59-ijms-26-04245)]. Statistical significance was set at *p* ≤ 0.05. Correction for multiple comparisons was set at *p* < 0.001 using the Bonferroni method. Statistical analyses were performed using IBM SPSS Statistics (version 29.0) and the statistical package PLINK (v1.07.2) [[60](#B60-ijms-26-04245)].

## 5. Conclusions

Our results suggest that genetic variants in *CD274* may be predictor biomarkers of survival in ES-SCLC patients treated with platinum- and etoposide-chemotherapy. These results support existing evidence that platinum compounds may be involved in the immunomodulation of the tumor microenvironment. Validation in a larger cohort of patients treated with the current standard treatment of chemotherapy plus IC inhibitors is required.

## Acknowledgments

We thank Carolyn Newey for language editing and Olga Bell for technical support.

## Supplementary Materials

The supporting information can be downloaded at: [https://www.mdpi.com/article/10.3390/ijms26094245/s1](https://www.mdpi.com/article/10.3390/ijms26094245/s1).

## Author Contributions

Conceptualization, A.B. and J.S.; methodology, A.B., L.L.-V., M.F., I.S. and J.S.; formal analysis, A.B. and J.S.; investigation, A.B., L.L.-V., M.F. and J.S.; resources, A.B., L.L.-V., M.F., S.M.-R., M.M., I.S. and J.S.; data curation, A.B., L.L.-V., M.F., S.M.-R., M.M., I.S. and J.S.; writing—original draft preparation, A.B. and J.S.; writing—review and editing, A.B., L.L.-V., M.F., S.M.-R., M.M., I.S. and J.S.; supervision, I.S. and J.S.; project administration, J.S. All authors have read and agreed to the published version of the manuscript.

## Institutional Review Board Statement

The study was performed in accordance with the Declaration of Helsinki and was approved by the Ethics committee of the Institut de Recerca Sant Pau (IIBSP-CCP-2022-56).

## Informed Consent Statement

Informed consent was obtained from all subjects involved in the study.

## Data Availability Statement

The data presented in this study include sensitive or confidential information and are not publicly available due to ethical committee regulations but are available upon request from the corresponding authors.

## Conflicts of Interest

The authors declare no conflicts of interest.

## Funding Statement

This research received no external funding.

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

The data presented in this study include sensitive or confidential information and are not publicly available due to ethical committee regulations but are available upon request from the corresponding authors.

### Supplementary Materials

### Data Availability Statement

The data presented in this study include sensitive or confidential information and are not publicly available due to ethical committee regulations but are available upon request from the corresponding authors.

## References

1. Lung Cancer Statistics|How Common Is Lung Cancer? [(accessed on 12 February 2025)]. Available online: https://www.cancer.org/cancer/types/lung-cancer/about/key-statistics.html.  [https://www.cancer.org/cancer/types/lung-cancer/about/key-statistics.html](https://www.cancer.org/cancer/types/lung-cancer/about/key-statistics.html)

2. Ou S.H.I., Ziogas A., Zell J.A. Prognostic Factors for Survival in Extensive Stage Small Cell Lung Cancer (ED-SCLC): The Importance of Smoking History, Socioeconomic and Marital Statuses, and Ethnicity. J. Thorac. Oncol. 2009;4:37–43. doi: 10.1097/JTO.0b013e31819140fb.  [DOI](https://doi.org/10.1097/JTO.0b013e31819140fb) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19096304/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Thorac.%20Oncol.&title=Prognostic%20Factors%20for%20Survival%20in%20Extensive%20Stage%20Small%20Cell%20Lung%20Cancer%20(ED-SCLC):%20The%20Importance%20of%20Smoking%20History,%20Socioeconomic%20and%20Marital%20Statuses,%20and%20Ethnicity&author=S.H.I.%20Ou&author=A.%20Ziogas&author=J.A.%20Zell&volume=4&publication_year=2009&pages=37-43&pmid=19096304&doi=10.1097/JTO.0b013e31819140fb&)

3. Horn L., Mansfield A.S., Szczęsna A., Havel L., Krzakowski M., Hochmair M.J., Huemer F., Losonczy G., Johnson M.L., Nishio M., et al. First-Line Atezolizumab plus Chemotherapy in Extensive-Stage Small-Cell Lung Cancer. N. Engl. J. Med. 2018;379:2220–2229. doi: 10.1056/NEJMoa1809064.  [DOI](https://doi.org/10.1056/NEJMoa1809064) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30280641/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%20Engl.%20J.%20Med.&title=First-Line%20Atezolizumab%20plus%20Chemotherapy%20in%20Extensive-Stage%20Small-Cell%20Lung%20Cancer&author=L.%20Horn&author=A.S.%20Mansfield&author=A.%20Szcz%C4%99sna&author=L.%20Havel&author=M.%20Krzakowski&volume=379&publication_year=2018&pages=2220-2229&pmid=30280641&doi=10.1056/NEJMoa1809064&)

4. Paz-Ares L., Dvorkin M., Chen Y., Reinmuth N., Hotta K., Trukhin D., Statsenko G., Hochmair M.J., Özgüroğlu M., Ji J.H., et al. Durvalumab plus Platinum-Etoposide versus Platinum-Etoposide in First-Line Treatment of Extensive-Stage Small-Cell Lung Cancer (CASPIAN): A Randomised, Controlled, Open-Label, Phase 3 Trial. Lancet. 2019;394:1929–1939. doi: 10.1016/S0140-6736(19)32222-6.  [DOI](https://doi.org/10.1016/S0140-6736(19)32222-6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31590988/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Durvalumab%20plus%20Platinum-Etoposide%20versus%20Platinum-Etoposide%20in%20First-Line%20Treatment%20of%20Extensive-Stage%20Small-Cell%20Lung%20Cancer%20(CASPIAN):%20A%20Randomised,%20Controlled,%20Open-Label,%20Phase%203%20Trial&author=L.%20Paz-Ares&author=M.%20Dvorkin&author=Y.%20Chen&author=N.%20Reinmuth&author=K.%20Hotta&volume=394&publication_year=2019&pages=1929-1939&pmid=31590988&doi=10.1016/S0140-6736(19)32222-6&)

5. Cheng Y., Han L., Wu L., Chen J., Sun H., Wen G., Ji Y., Dvorkin M., Shi J., Pan Z., et al. Effect of First-Line Serplulimab vs Placebo Added to Chemotherapy on Survival in Patients with Extensive-Stage Small Cell Lung Cancer: The ASTRUM-005 Randomized Clinical Trial. JAMA. 2022;328:1223–1232. doi: 10.1001/jama.2022.16464.  [DOI](https://doi.org/10.1001/jama.2022.16464) | [PMC free article](/articles/PMC9516323/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36166026/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA&title=Effect%20of%20First-Line%20Serplulimab%20vs%20Placebo%20Added%20to%20Chemotherapy%20on%20Survival%20in%20Patients%20with%20Extensive-Stage%20Small%20Cell%20Lung%20Cancer:%20The%20ASTRUM-005%20Randomized%20Clinical%20Trial&author=Y.%20Cheng&author=L.%20Han&author=L.%20Wu&author=J.%20Chen&author=H.%20Sun&volume=328&publication_year=2022&pages=1223-1232&pmid=36166026&doi=10.1001/jama.2022.16464&)

6. Paz-Ares L., Reck M., Peters S., Borghaei H., Herbst R., Siddiqui M., Cuchelkar V., Bhatt K., Chakrabarti D., Wang L., et al. IMporte: A Phase III Study of Lurbinectedin and Atezolizumab Versus Atezolizumab as Maintenance Therapy in ES-SCLC; Proceedings of the Presented at World Conference on Lung Cancer (WCLC) 2022 (Abstract 2350); Vienna, Austria. 6–9 August 2022.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20Presented%20at%20World%20Conference%20on%20Lung%20Cancer%20(WCLC)%202022%20(Abstract%202350)&title=IMporte:%20A%20Phase%20III%20Study%20of%20Lurbinectedin%20and%20Atezolizumab%20Versus%20Atezolizumab%20as%20Maintenance%20Therapy%20in%20ES-SCLC&author=L.%20Paz-Ares&author=M.%20Reck&author=S.%20Peters&author=H.%20Borghaei&author=R.%20Herbst&)

7. Perol M., Ahn M.-J., Cheng Y., Clarke J., Dingemans A.-M., Gay C., Navarro A., Schuler M., Yoshida T., Martinez P., et al. Tarlatamab Plus Durvalumab as First-Line Maintenance in Extensive-Stage Small-Cell Lung Cancer: DeLLphi-305 Phase 3 Trial; Proceedings of the Presented at World Conference on Lung Cancer (WCLC) 2024 (P1.13A.02); San Diego, CA, USA. 7–10 September 2024.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20Presented%20at%20World%20Conference%20on%20Lung%20Cancer%20(WCLC)%202024%20(P1.13A.02)&title=Tarlatamab%20Plus%20Durvalumab%20as%20First-Line%20Maintenance%20in%20Extensive-Stage%20Small-Cell%20Lung%20Cancer:%20DeLLphi-305%20Phase%203%20Trial&author=M.%20Perol&author=M.-J.%20Ahn&author=Y.%20Cheng&author=J.%20Clarke&author=A.-M.%20Dingemans&)

8. Rudin C.M., Brambilla E., Faivre-Finn C., Sage J. Small-Cell Lung Cancer. Nat. Rev. Dis. Prim. 2021;7:3. doi: 10.1038/s41572-020-00235-0.  [DOI](https://doi.org/10.1038/s41572-020-00235-0) | [PMC free article](/articles/PMC8177722/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33446664/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Dis.%20Prim.&title=Small-Cell%20Lung%20Cancer&author=C.M.%20Rudin&author=E.%20Brambilla&author=C.%20Faivre-Finn&author=J.%20Sage&volume=7&publication_year=2021&pages=3&pmid=33446664&doi=10.1038/s41572-020-00235-0&)

9. Rudin C.M., Poirier J.T., Byers L.A., Dive C., Dowlati A., George J., Heymach J.V., Johnson J.E., Lehman J.M., MacPherson D., et al. Molecular Subtypes of Small Cell Lung Cancer: A Synthesis of Human and Mouse Model Data. Nat. Rev. Cancer. 2019;19:289–297. doi: 10.1038/s41568-019-0133-9.  [DOI](https://doi.org/10.1038/s41568-019-0133-9) | [PMC free article](/articles/PMC6538259/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30926931/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Cancer&title=Molecular%20Subtypes%20of%20Small%20Cell%20Lung%20Cancer:%20A%20Synthesis%20of%20Human%20and%20Mouse%20Model%20Data&author=C.M.%20Rudin&author=J.T.%20Poirier&author=L.A.%20Byers&author=C.%20Dive&author=A.%20Dowlati&volume=19&publication_year=2019&pages=289-297&pmid=30926931&doi=10.1038/s41568-019-0133-9&)

10. Keppens C., Dequeker E.M., Pauwels P., Ryska A., ‘t Hart N., von der Thüsen J.H. PD-L1 Immunohistochemistry in Non-Small-Cell Lung Cancer: Unraveling Differences in Staining Concordance and Interpretation. Virchows Arch. 2021;478:827–839. doi: 10.1007/s00428-020-02976-5.  [DOI](https://doi.org/10.1007/s00428-020-02976-5) | [PMC free article](/articles/PMC8099807/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33275169/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Virchows%20Arch.&title=PD-L1%20Immunohistochemistry%20in%20Non-Small-Cell%20Lung%20Cancer:%20Unraveling%20Differences%20in%20Staining%20Concordance%20and%20Interpretation&author=C.%20Keppens&author=E.M.%20Dequeker&author=P.%20Pauwels&author=A.%20Ryska&author=N.%20%E2%80%98t%20Hart&volume=478&publication_year=2021&pages=827-839&pmid=33275169&doi=10.1007/s00428-020-02976-5&)

11. Hellmann M.D., Ciuleanu T.-E., Pluzanski A., Lee J.S., Otterson G.A., Audigier-Valette C., Minenza E., Linardou H., Burgers S., Salman P., et al. Nivolumab plus Ipilimumab in Lung Cancer with a High Tumor Mutational Burden. N. Engl. J. Med. 2018;378:2093–2104. doi: 10.1056/NEJMoa1801946.  [DOI](https://doi.org/10.1056/NEJMoa1801946) | [PMC free article](/articles/PMC7193684/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29658845/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%20Engl.%20J.%20Med.&title=Nivolumab%20plus%20Ipilimumab%20in%20Lung%20Cancer%20with%20a%20High%20Tumor%20Mutational%20Burden&author=M.D.%20Hellmann&author=T.-E.%20Ciuleanu&author=A.%20Pluzanski&author=J.S.%20Lee&author=G.A.%20Otterson&volume=378&publication_year=2018&pages=2093-2104&pmid=29658845&doi=10.1056/NEJMoa1801946&)

12. Liu S.V., Reck M., Mansfield A.S., Mok T., Scherpereel A., Reinmuth N., Garassino M.C., De Castro Carpeno J., Califano R., Nishio M., et al. Updated Overall Survival and PD-L1 Subgroup Analysis of Patients with Extensive-Stage Small-Cell Lung Cancer Treated with Atezolizumab, Carboplatin, and Etoposide (IMpower133) J. Clin. Oncol. 2021;39:619–630. doi: 10.1200/JCO.20.01055.  [DOI](https://doi.org/10.1200/JCO.20.01055) | [PMC free article](/articles/PMC8078320/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33439693/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Oncol.&title=Updated%20Overall%20Survival%20and%20PD-L1%20Subgroup%20Analysis%20of%20Patients%20with%20Extensive-Stage%20Small-Cell%20Lung%20Cancer%20Treated%20with%20Atezolizumab,%20Carboplatin,%20and%20Etoposide%20(IMpower133)&author=S.V.%20Liu&author=M.%20Reck&author=A.S.%20Mansfield&author=T.%20Mok&author=A.%20Scherpereel&volume=39&publication_year=2021&pages=619-630&pmid=33439693&doi=10.1200/JCO.20.01055&)

13. Olaussen K.A., Dunant A., Fouret P., Brambilla E., André F., Haddad V., Taranchon E., Filipits M., Pirker R., Popper H.H., et al. DNA Repair by ERCC1 in Non-Small-Cell Lung Cancer and Cisplatin-Based Adjuvant Chemotherapy. N. Engl. J. Med. 2006;355:983–991. doi: 10.1056/NEJMoa060570.  [DOI](https://doi.org/10.1056/NEJMoa060570) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16957145/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%20Engl.%20J.%20Med.&title=DNA%20Repair%20by%20ERCC1%20in%20Non-Small-Cell%20Lung%20Cancer%20and%20Cisplatin-Based%20Adjuvant%20Chemotherapy&author=K.A.%20Olaussen&author=A.%20Dunant&author=P.%20Fouret&author=E.%20Brambilla&author=F.%20Andr%C3%A9&volume=355&publication_year=2006&pages=983-991&pmid=16957145&doi=10.1056/NEJMoa060570&)

14. Hamilton G., Rath B. Pharmacogenetics of Platinum-Based Chemotherapy in Non-Small Cell Lung Cancer: Predictive Validity of Polymorphisms of ERCC1. Expert Opin. Drug Metab. Toxicol. 2018;14:17–24. doi: 10.1080/17425255.2018.1416095.  [DOI](https://doi.org/10.1080/17425255.2018.1416095) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29226731/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Expert%20Opin.%20Drug%20Metab.%20Toxicol.&title=Pharmacogenetics%20of%20Platinum-Based%20Chemotherapy%20in%20Non-Small%20Cell%20Lung%20Cancer:%20Predictive%20Validity%20of%20Polymorphisms%20of%20ERCC1&author=G.%20Hamilton&author=B.%20Rath&volume=14&publication_year=2018&pages=17-24&pmid=29226731&doi=10.1080/17425255.2018.1416095&)

15. Barba A., López-Vilaró L., Ferre M., Majem M., Martinez-Recio S., Bell O., Arranz M.J., Salazar J., Sullivan I. ERCC1 and ERCC2 Polymorphisms Predict the Efficacy and Toxicity of Platinum-Based Chemotherapy in Small Cell Lung Cancer. Pharmaceutics. 2024;16:1121. doi: 10.3390/pharmaceutics16091121.  [DOI](https://doi.org/10.3390/pharmaceutics16091121) | [PMC free article](/articles/PMC11434779/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/39339159/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmaceutics&title=ERCC1%20and%20ERCC2%20Polymorphisms%20Predict%20the%20Efficacy%20and%20Toxicity%20of%20Platinum-Based%20Chemotherapy%20in%20Small%20Cell%20Lung%20Cancer&author=A.%20Barba&author=L.%20L%C3%B3pez-Vilar%C3%B3&author=M.%20Ferre&author=M.%20Majem&author=S.%20Martinez-Recio&volume=16&publication_year=2024&pages=1121&pmid=39339159&doi=10.3390/pharmaceutics16091121&)

16. Todd R.C., Lippard S.J. Inhibition of Transcription by Platinum Antitumor Compounds. Metallomics. 2009;1:280–291. doi: 10.1039/b907567d.  [DOI](https://doi.org/10.1039/b907567d) | [PMC free article](/articles/PMC2752884/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20046924/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Metallomics&title=Inhibition%20of%20Transcription%20by%20Platinum%20Antitumor%20Compounds&author=R.C.%20Todd&author=S.J.%20Lippard&volume=1&publication_year=2009&pages=280-291&pmid=20046924&doi=10.1039/b907567d&)

17. De Biasi A.R., Villena-Vargas J., Adusumilli P.S. Cisplatin-Induced Antitumor Immunomodulation: A Review of Preclinical and Clinical Evidence. Clin. Cancer Res. 2014;20:5384–5391. doi: 10.1158/1078-0432.CCR-14-1298.  [DOI](https://doi.org/10.1158/1078-0432.CCR-14-1298) | [PMC free article](/articles/PMC4216745/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25204552/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Cancer%20Res.&title=Cisplatin-Induced%20Antitumor%20Immunomodulation:%20A%20Review%20of%20Preclinical%20and%20Clinical%20Evidence&author=A.R.%20De%20Biasi&author=J.%20Villena-Vargas&author=P.S.%20Adusumilli&volume=20&publication_year=2014&pages=5384-5391&pmid=25204552&doi=10.1158/1078-0432.CCR-14-1298&)

18. Nejad E.B., Van Der Sluis T.C., Van Duikeren S., Yagita H., Janssen G.M., Van Veelen P.A., Melief C.J.M., Van Der Burg S.H., Arens R. Tumor Eradication by Cisplatin Is Sustained by CD80/86-Mediated Costimulation of CD8+ T Cells. Cancer Res. 2016;76:6017–6029. doi: 10.1158/0008-5472.CAN-16-0881.  [DOI](https://doi.org/10.1158/0008-5472.CAN-16-0881) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27569212/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res.&title=Tumor%20Eradication%20by%20Cisplatin%20Is%20Sustained%20by%20CD80/86-Mediated%20Costimulation%20of%20CD8+%20T%20Cells&author=E.B.%20Nejad&author=T.C.%20Van%20Der%20Sluis&author=S.%20Van%20Duikeren&author=H.%20Yagita&author=G.M.%20Janssen&volume=76&publication_year=2016&pages=6017-6029&pmid=27569212&doi=10.1158/0008-5472.CAN-16-0881&)

19. Van Wigcheren G.F., De Haas N., Mulder T.A., Horrevorts S.K., Bloemendal M., Hins-Debree S., Mao Y., Kiessling R., van Herpen C.M.L., Flórez-Grau G., et al. Cisplatin Inhibits Frequency and Suppressive Activity of Monocytic Myeloid-Derived Suppressor Cells in Cancer Patients. Oncoimmunology. 2021;10:1935557. doi: 10.1080/2162402X.2021.1935557.  [DOI](https://doi.org/10.1080/2162402X.2021.1935557) | [PMC free article](/articles/PMC8237969/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34239773/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncoimmunology&title=Cisplatin%20Inhibits%20Frequency%20and%20Suppressive%20Activity%20of%20Monocytic%20Myeloid-Derived%20Suppressor%20Cells%20in%20Cancer%20Patients&author=G.F.%20Van%20Wigcheren&author=N.%20De%20Haas&author=T.A.%20Mulder&author=S.K.%20Horrevorts&author=M.%20Bloemendal&volume=10&publication_year=2021&pages=1935557&pmid=34239773&doi=10.1080/2162402X.2021.1935557&)

20. Sun C., Mezzadra R., Schumacher T.N. Regulation and Function of the PD-L1 Checkpoint. Immunity. 2018;48:434–452. doi: 10.1016/j.immuni.2018.03.014.  [DOI](https://doi.org/10.1016/j.immuni.2018.03.014) | [PMC free article](/articles/PMC7116507/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29562194/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Immunity&title=Regulation%20and%20Function%20of%20the%20PD-L1%20Checkpoint&author=C.%20Sun&author=R.%20Mezzadra&author=T.N.%20Schumacher&volume=48&publication_year=2018&pages=434-452&pmid=29562194&doi=10.1016/j.immuni.2018.03.014&)

21. Jiang X., Wang J., Deng X., Xiong F., Ge J., Xiang B., Wu X., Ma J., Zhou M., Li X., et al. Role of the Tumor Microenvironment in PD-L1/PD-1-Mediated Tumor Immune Escape. Mol. Cancer. 2019;18:10. doi: 10.1186/s12943-018-0928-4.  [DOI](https://doi.org/10.1186/s12943-018-0928-4) | [PMC free article](/articles/PMC6332843/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30646912/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Cancer&title=Role%20of%20the%20Tumor%20Microenvironment%20in%20PD-L1/PD-1-Mediated%20Tumor%20Immune%20Escape&author=X.%20Jiang&author=J.%20Wang&author=X.%20Deng&author=F.%20Xiong&author=J.%20Ge&volume=18&publication_year=2019&pages=10&pmid=30646912&doi=10.1186/s12943-018-0928-4&)

22. Kula A., Dawidowicz M., Kiczmer P., Prawdzic Seńkowska A., Świętochowska E. The Role of Genetic Polymorphism within PD-L1 Gene in Cancer. Review. Exp. Mol. Pathol. 2020;116:104494. doi: 10.1016/j.yexmp.2020.104494.  [DOI](https://doi.org/10.1016/j.yexmp.2020.104494) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32679050/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Exp.%20Mol.%20Pathol.&title=The%20Role%20of%20Genetic%20Polymorphism%20within%20PD-L1%20Gene%20in%20Cancer.%20Review&author=A.%20Kula&author=M.%20Dawidowicz&author=P.%20Kiczmer&author=A.%20Prawdzic%20Se%C5%84kowska&author=E.%20%C5%9Awi%C4%99tochowska&volume=116&publication_year=2020&pages=104494&pmid=32679050&doi=10.1016/j.yexmp.2020.104494&)

23. Lee S.Y., Jung D.K., Choi J.E., Jin C.C., Hong M.J., Do S.K., Kang H.G., Lee W.K., Seok Y., Lee E.B., et al. PD-L1 Polymorphism Can Predict Clinical Outcomes of Non-Small Cell Lung Cancer Patients Treated with First-Line Paclitaxel-Cisplatin Chemotherapy. Sci. Rep. 2016;6:25952. doi: 10.1038/srep25952.  [DOI](https://doi.org/10.1038/srep25952) | [PMC free article](/articles/PMC4867646/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27181838/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&title=PD-L1%20Polymorphism%20Can%20Predict%20Clinical%20Outcomes%20of%20Non-Small%20Cell%20Lung%20Cancer%20Patients%20Treated%20with%20First-Line%20Paclitaxel-Cisplatin%20Chemotherapy&author=S.Y.%20Lee&author=D.K.%20Jung&author=J.E.%20Choi&author=C.C.%20Jin&author=M.J.%20Hong&volume=6&publication_year=2016&pages=25952&pmid=27181838&doi=10.1038/srep25952&)

24. Lee S.Y., Jung D.K., Choi J.E., Jin C.C., Hong M.J., Do S.K., Kang H.G., Lee W.K., Seok Y., Lee E.B., et al. Functional Polymorphisms in PD-L1 Gene Are Associated with the Prognosis of Patients with Early Stage Non-Small Cell Lung Cancer. Gene. 2017;599:28–35. doi: 10.1016/j.gene.2016.11.007.  [DOI](https://doi.org/10.1016/j.gene.2016.11.007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27838455/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gene&title=Functional%20Polymorphisms%20in%20PD-L1%20Gene%20Are%20Associated%20with%20the%20Prognosis%20of%20Patients%20with%20Early%20Stage%20Non-Small%20Cell%20Lung%20Cancer&author=S.Y.%20Lee&author=D.K.%20Jung&author=J.E.%20Choi&author=C.C.%20Jin&author=M.J.%20Hong&volume=599&publication_year=2017&pages=28-35&pmid=27838455&doi=10.1016/j.gene.2016.11.007&)

25. Zhao M., Zhang J., Chen S., Wang Y., Tian Q. Influence of Programmed Death Ligand-1-Gene Polymorphism Rs822336 on the Prognosis and Safety of Postoperative Patients with NSCLC Who Received Platinum-Based Adjuvant Chemotherapy. Cancer Manag. Res. 2020;12:6755–6766. doi: 10.2147/CMAR.S255072.  [DOI](https://doi.org/10.2147/CMAR.S255072) | [PMC free article](/articles/PMC7414937/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32801904/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Manag.%20Res.&title=Influence%20of%20Programmed%20Death%20Ligand-1-Gene%20Polymorphism%20Rs822336%20on%20the%20Prognosis%20and%20Safety%20of%20Postoperative%20Patients%20with%20NSCLC%20Who%20Received%20Platinum-Based%20Adjuvant%20Chemotherapy&author=M.%20Zhao&author=J.%20Zhang&author=S.%20Chen&author=Y.%20Wang&author=Q.%20Tian&volume=12&publication_year=2020&pages=6755-6766&pmid=32801904&doi=10.2147/CMAR.S255072&)

26. Lee J.H., Yoo S.S., Hong M.J., Choi J.E., Kim S., Kang H.G., Do S.K., Kim J.H., Baek S.A., Lee W.K., et al. Impact of Immune Checkpoint Gene CD155 Ala67Thr and CD226 Gly307Ser Polymorphisms on Small Cell Lung Cancer Clinical Outcome. Sci. Rep. 2021;11:1794. doi: 10.1038/s41598-021-81260-1.  [DOI](https://doi.org/10.1038/s41598-021-81260-1) | [PMC free article](/articles/PMC7815735/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33469055/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&title=Impact%20of%20Immune%20Checkpoint%20Gene%20CD155%20Ala67Thr%20and%20CD226%20Gly307Ser%20Polymorphisms%20on%20Small%20Cell%20Lung%20Cancer%20Clinical%20Outcome&author=J.H.%20Lee&author=S.S.%20Yoo&author=M.J.%20Hong&author=J.E.%20Choi&author=S.%20Kim&volume=11&publication_year=2021&pages=1794&pmid=33469055&doi=10.1038/s41598-021-81260-1&)

27. Tsao M.S., Rosenthal A., Nicholson A.G., Detterbeck F., Eberhardt W.E.E., Lievens Y., Lim E., Matilla J.-M., Yatabe Y., Filosso P.L., et al. The International Association for the Study of Lung Cancer Staging Project: The Database and Proposal for the Revision of the Staging of Pulmonary Neuroendocrine Carcinoma in the Forthcoming Ninth Edition of the TNM Classification for Lung Cancer. J. Thorac. Oncol. 2025 doi: 10.1016/j.jtho.2025.01.013.  [DOI](https://doi.org/10.1016/j.jtho.2025.01.013) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/39864546/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Thorac.%20Oncol.&title=The%20International%20Association%20for%20the%20Study%20of%20Lung%20Cancer%20Staging%20Project:%20The%20Database%20and%20Proposal%20for%20the%20Revision%20of%20the%20Staging%20of%20Pulmonary%20Neuroendocrine%20Carcinoma%20in%20the%20Forthcoming%20Ninth%20Edition%20of%20the%20TNM%20Classification%20for%20Lung%20Cancer&author=M.S.%20Tsao&author=A.%20Rosenthal&author=A.G.%20Nicholson&author=F.%20Detterbeck&author=W.E.E.%20Eberhardt&publication_year=2025&pmid=39864546&doi=10.1016/j.jtho.2025.01.013&)

28. Baize N., Monnet I., Greillier L., Geier M., Lena H., Janicot H., Vergnenegre A., Crequit J., Lamy R., Auliac J.B., et al. Carboplatin plus Etoposide versus Topotecan as Second-Line Treatment for Patients with Sensitive Relapsed Small-Cell Lung Cancer: An Open-Label, Multicentre, Randomised, Phase 3 Trial. Lancet Oncol. 2020;21:1224–1233. doi: 10.1016/S1470-2045(20)30461-7.  [DOI](https://doi.org/10.1016/S1470-2045(20)30461-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32888454/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet%20Oncol.&title=Carboplatin%20plus%20Etoposide%20versus%20Topotecan%20as%20Second-Line%20Treatment%20for%20Patients%20with%20Sensitive%20Relapsed%20Small-Cell%20Lung%20Cancer:%20An%20Open-Label,%20Multicentre,%20Randomised,%20Phase%203%20Trial&author=N.%20Baize&author=I.%20Monnet&author=L.%20Greillier&author=M.%20Geier&author=H.%20Lena&volume=21&publication_year=2020&pages=1224-1233&pmid=32888454&doi=10.1016/S1470-2045(20)30461-7&)

29. Owonikoko T.K., Behera M., Chen Z., Bhimani C., Curran W.J., Khuri F.R., Ramalingam S.S. A Systematic Analysis of Efficacy of Second-Line Chemotherapy in Sensitive and Refractory Small-Cell Lung Cancer. J. Thorac. Oncol. 2012;7:866–872. doi: 10.1097/JTO.0b013e31824c7f4b.  [DOI](https://doi.org/10.1097/JTO.0b013e31824c7f4b) | [PMC free article](/articles/PMC3381878/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22722788/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Thorac.%20Oncol.&title=A%20Systematic%20Analysis%20of%20Efficacy%20of%20Second-Line%20Chemotherapy%20in%20Sensitive%20and%20Refractory%20Small-Cell%20Lung%20Cancer&author=T.K.%20Owonikoko&author=M.%20Behera&author=Z.%20Chen&author=C.%20Bhimani&author=W.J.%20Curran&volume=7&publication_year=2012&pages=866-872&pmid=22722788&doi=10.1097/JTO.0b013e31824c7f4b&)

30. Fabrizio F.P., Trombetta D., Rossi A., Sparaneo A., Castellana S., Muscarella L.A. Gene Code CD274/PD-L1: From Molecular Basis toward Cancer Immunotherapy. Ther. Adv. Med. Oncol. 2018;10:1–18. doi: 10.1177/1758835918815598.  [DOI](https://doi.org/10.1177/1758835918815598) | [PMC free article](/articles/PMC6299305/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30574211/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ther.%20Adv.%20Med.%20Oncol.&title=Gene%20Code%20CD274/PD-L1:%20From%20Molecular%20Basis%20toward%20Cancer%20Immunotherapy&author=F.P.%20Fabrizio&author=D.%20Trombetta&author=A.%20Rossi&author=A.%20Sparaneo&author=S.%20Castellana&volume=10&publication_year=2018&pages=1-18&pmid=30574211&doi=10.1177/1758835918815598&)

31. National Center for Biotechnology Information Rs2282055. [(accessed on 10 May 2024)];2025 Available online: https://www.ncbi.nlm.nih.gov/snp/rs2282055.  [https://www.ncbi.nlm.nih.gov/snp/rs2282055](https://www.ncbi.nlm.nih.gov/snp/rs2282055)

32. Boyle A.P., Hong E.L., Hariharan M., Cheng Y., Schaub M.A., Kasowski M., Karczewski K.J., Park J., Hitz B.C., Weng S., et al. Annotation of Functional Variation in Personal Genomes Using RegulomeDB. Genome Res. 2012;22:1790–1797. doi: 10.1101/gr.137323.112.  [DOI](https://doi.org/10.1101/gr.137323.112) | [PMC free article](/articles/PMC3431494/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22955989/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res.&title=Annotation%20of%20Functional%20Variation%20in%20Personal%20Genomes%20Using%20RegulomeDB&author=A.P.%20Boyle&author=E.L.%20Hong&author=M.%20Hariharan&author=Y.%20Cheng&author=M.A.%20Schaub&volume=22&publication_year=2012&pages=1790-1797&pmid=22955989&doi=10.1101/gr.137323.112&)

33. Hausser J., Zavolan M. Identification and Consequences of MiRNA-Target Interactions—Beyond Repression of Gene Expression. Nat. Rev. Genet. 2014;15:599–612. doi: 10.1038/nrg3765.  [DOI](https://doi.org/10.1038/nrg3765) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25022902/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Genet.&title=Identification%20and%20Consequences%20of%20MiRNA-Target%20Interactions%E2%80%94Beyond%20Repression%20of%20Gene%20Expression&author=J.%20Hausser&author=M.%20Zavolan&volume=15&publication_year=2014&pages=599-612&pmid=25022902&doi=10.1038/nrg3765&)

34. Du W., Zhu J., Chen Y., Zeng Y., Shen D., Zhang N., Ning W., Liu Z., Huang J.A. Variant SNPs at the MicroRNA Complementary Site in the B7-H1 3′-untranslated Region Increase the Risk of Non-small Cell Lung Cancer. Mol. Med. Rep. 2017;16:2682–2690. doi: 10.3892/mmr.2017.6902.  [DOI](https://doi.org/10.3892/mmr.2017.6902) | [PMC free article](/articles/PMC5547930/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28677815/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Med.%20Rep.&title=Variant%20SNPs%20at%20the%20MicroRNA%20Complementary%20Site%20in%20the%20B7-H1%203%E2%80%B2-untranslated%20Region%20Increase%20the%20Risk%20of%20Non-small%20Cell%20Lung%20Cancer&author=W.%20Du&author=J.%20Zhu&author=Y.%20Chen&author=Y.%20Zeng&author=D.%20Shen&volume=16&publication_year=2017&pages=2682-2690&pmid=28677815&doi=10.3892/mmr.2017.6902&)

35. Schultheis A.M., Scheel A.H., Ozretić L., George J., Thomas R.K., Hagemann T., Zander T., Wolf J., Buettner R. PD-L1 Expression in Small Cell Neuroendocrine Carcinomas. Eur. J. Cancer. 2015;51:421–426. doi: 10.1016/j.ejca.2014.12.006.  [DOI](https://doi.org/10.1016/j.ejca.2014.12.006) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25582496/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Cancer&title=PD-L1%20Expression%20in%20Small%20Cell%20Neuroendocrine%20Carcinomas&author=A.M.%20Schultheis&author=A.H.%20Scheel&author=L.%20Ozreti%C4%87&author=J.%20George&author=R.K.%20Thomas&volume=51&publication_year=2015&pages=421-426&pmid=25582496&doi=10.1016/j.ejca.2014.12.006&)

36. Yi M., Niu M., Xu L., Luo S., Wu K. Regulation of PD-L1 Expression in the Tumor Microenvironment. J. Hematol. Oncol. 2021;14:10. doi: 10.1186/s13045-020-01027-5.  [DOI](https://doi.org/10.1186/s13045-020-01027-5) | [PMC free article](/articles/PMC7792099/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33413496/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Hematol.%20Oncol.&title=Regulation%20of%20PD-L1%20Expression%20in%20the%20Tumor%20Microenvironment&author=M.%20Yi&author=M.%20Niu&author=L.%20Xu&author=S.%20Luo&author=K.%20Wu&volume=14&publication_year=2021&pages=10&pmid=33413496&doi=10.1186/s13045-020-01027-5&)

37. National Center for Biotechnology Information Rs231775. [(accessed on 10 May 2024)];2025 Available online: https://www.ncbi.nlm.nih.gov/snp/rs231775.  [https://www.ncbi.nlm.nih.gov/snp/rs231775](https://www.ncbi.nlm.nih.gov/snp/rs231775)

38. Anjos S., Nguyen A., Ounissi-Benkalha H., Tessier M.C., Polychronakos C. A Common Autoimmunity Predisposing Signal Peptide Variant of the Cytotoxic T-Lymphocyte Antigen 4 Results in Inefficient Glycosylation of the Susceptibility Allele. J. Biol. Chem. 2002;277:46478–46486. doi: 10.1074/jbc.M206894200.  [DOI](https://doi.org/10.1074/jbc.M206894200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12244107/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem.&title=A%20Common%20Autoimmunity%20Predisposing%20Signal%20Peptide%20Variant%20of%20the%20Cytotoxic%20T-Lymphocyte%20Antigen%204%20Results%20in%20Inefficient%20Glycosylation%20of%20the%20Susceptibility%20Allele&author=S.%20Anjos&author=A.%20Nguyen&author=H.%20Ounissi-Benkalha&author=M.C.%20Tessier&author=C.%20Polychronakos&volume=277&publication_year=2002&pages=46478-46486&pmid=12244107&doi=10.1074/jbc.M206894200&)

39. Eisenhauer E.A., Therasse P., Bogaerts J., Schwartz L.H., Sargent D., Ford R., Dancey J., Arbuck S., Gwyther S., Mooney M., et al. New Response Evaluation Criteria in Solid Tumours: Revised RECIST Guideline (Version 1.1) Eur. J. Cancer. 2009;45:228–247. doi: 10.1016/j.ejca.2008.10.026.  [DOI](https://doi.org/10.1016/j.ejca.2008.10.026) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19097774/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Cancer&title=New%20Response%20Evaluation%20Criteria%20in%20Solid%20Tumours:%20Revised%20RECIST%20Guideline%20(Version%201.1)&author=E.A.%20Eisenhauer&author=P.%20Therasse&author=J.%20Bogaerts&author=L.H.%20Schwartz&author=D.%20Sargent&volume=45&publication_year=2009&pages=228-247&pmid=19097774&doi=10.1016/j.ejca.2008.10.026&)

40. Dingemans A.M.C., Früh M., Ardizzoni A., Besse B., Faivre-Finn C., Hendriks L.E., Lantuejoul S., Peters S., Reguart N., Rudin C.M., et al. Small-Cell Lung Cancer: ESMO Clinical Practice Guidelines for Diagnosis, Treatment and Follow-Up. Ann. Oncol. Off. J. Eur. Soc. Med. Oncol. 2021;32:839–853. doi: 10.1016/j.annonc.2021.03.207.  [DOI](https://doi.org/10.1016/j.annonc.2021.03.207) | [PMC free article](/articles/PMC9464246/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33864941/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann.%20Oncol.%20Off.%20J.%20Eur.%20Soc.%20Med.%20Oncol.&title=Small-Cell%20Lung%20Cancer:%20ESMO%20Clinical%20Practice%20Guidelines%20for%20Diagnosis,%20Treatment%20and%20Follow-Up&author=A.M.C.%20Dingemans&author=M.%20Fr%C3%BCh&author=A.%20Ardizzoni&author=B.%20Besse&author=C.%20Faivre-Finn&volume=32&publication_year=2021&pages=839-853&pmid=33864941&doi=10.1016/j.annonc.2021.03.207&)

41. Wagner M., Jasek M., Karabon L. Immune Checkpoint Molecules-Inherited Variations as Markers for Cancer Risk. Front. Immunol. 2021;11:606721. doi: 10.3389/fimmu.2020.606721.  [DOI](https://doi.org/10.3389/fimmu.2020.606721) | [PMC free article](/articles/PMC7840570/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33519815/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front.%20Immunol.&title=Immune%20Checkpoint%20Molecules-Inherited%20Variations%20as%20Markers%20for%20Cancer%20Risk&author=M.%20Wagner&author=M.%20Jasek&author=L.%20Karabon&volume=11&publication_year=2021&pages=606721&pmid=33519815&doi=10.3389/fimmu.2020.606721&)

42. Nomizo T., Ozasa H., Tsuji T., Funazo T., Yasuda Y., Yoshida H., Yagi Y., Sakamori Y., Nagai H., Hirai T., et al. Clinical Impact of Single Nucleotide Polymorphism in PD-L1 on Response to Nivolumab for Advanced Non-Small-Cell Lung Cancer Patients. Sci. Rep. 2017;7:45124. doi: 10.1038/srep45124.  [DOI](https://doi.org/10.1038/srep45124) | [PMC free article](/articles/PMC5362817/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28332580/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&title=Clinical%20Impact%20of%20Single%20Nucleotide%20Polymorphism%20in%20PD-L1%20on%20Response%20to%20Nivolumab%20for%20Advanced%20Non-Small-Cell%20Lung%20Cancer%20Patients&author=T.%20Nomizo&author=H.%20Ozasa&author=T.%20Tsuji&author=T.%20Funazo&author=Y.%20Yasuda&volume=7&publication_year=2017&pages=45124&pmid=28332580&doi=10.1038/srep45124&)

43. Wu Y., Zhao T., Jia Z., Cao D., Cao X., Pan Y., Zhao D., Zhang B., Jiang J. Polymorphism of the Programmed Death-Ligand 1 Gene Is Associated with Its Protein Expression and Prognosis in Gastric Cancer. J. Gastroenterol. Hepatol. 2019;34:1201–1207. doi: 10.1111/jgh.14520.  [DOI](https://doi.org/10.1111/jgh.14520) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30353572/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Gastroenterol.%20Hepatol.&title=Polymorphism%20of%20the%20Programmed%20Death-Ligand%201%20Gene%20Is%20Associated%20with%20Its%20Protein%20Expression%20and%20Prognosis%20in%20Gastric%20Cancer&author=Y.%20Wu&author=T.%20Zhao&author=Z.%20Jia&author=D.%20Cao&author=X.%20Cao&volume=34&publication_year=2019&pages=1201-1207&pmid=30353572&doi=10.1111/jgh.14520&)

44. Funazo T.Y., Nomizo T., Ozasa H., Tsuji T., Yasuda Y., Yoshida H., Sakamori Y., Nagai H., Hirai T., Kim Y.H. Clinical Impact of Low Serum Free T4 in Patients with Non-Small Cell Lung Cancer Treated with Nivolumab. Sci. Rep. 2019;9:17085. doi: 10.1038/s41598-019-53327-7.  [DOI](https://doi.org/10.1038/s41598-019-53327-7) | [PMC free article](/articles/PMC6864095/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31745135/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&title=Clinical%20Impact%20of%20Low%20Serum%20Free%20T4%20in%20Patients%20with%20Non-Small%20Cell%20Lung%20Cancer%20Treated%20with%20Nivolumab&author=T.Y.%20Funazo&author=T.%20Nomizo&author=H.%20Ozasa&author=T.%20Tsuji&author=Y.%20Yasuda&volume=9&publication_year=2019&pages=17085&pmid=31745135&doi=10.1038/s41598-019-53327-7&)

45. Hashemi M., Karami S., Sarabandi S., Moazeni-Roodi A., Małecki A., Ghavami S., Wiechec E. Association between PD-1 and PD-L1 Polymorphisms and the Risk of Cancer: A Meta-Analysis of Case-Control Studies. Cancers. 2019;11:1150. doi: 10.3390/cancers11081150.  [DOI](https://doi.org/10.3390/cancers11081150) | [PMC free article](/articles/PMC6721817/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31405171/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancers&title=Association%20between%20PD-1%20and%20PD-L1%20Polymorphisms%20and%20the%20Risk%20of%20Cancer:%20A%20Meta-Analysis%20of%20Case-Control%20Studies&author=M.%20Hashemi&author=S.%20Karami&author=S.%20Sarabandi&author=A.%20Moazeni-Roodi&author=A.%20Ma%C5%82ecki&volume=11&publication_year=2019&pages=1150&pmid=31405171&doi=10.3390/cancers11081150&)

46. de With M., Hurkmans D.P., De Hoop E.O., Lalouti A., Bins S., El Bouazzaoui S., van Brakel M., Debets R., Aerts J.G.J.V., van Schaik R.H.N., et al. Germline Variation in PDCD1 Is Associated with Overall Survival in Patients with Metastatic Melanoma Treated with Anti-PD-1 Monotherapy. Cancers. 2021;13:1370. doi: 10.3390/cancers13061370.  [DOI](https://doi.org/10.3390/cancers13061370) | [PMC free article](/articles/PMC8002987/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33803602/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancers&title=Germline%20Variation%20in%20PDCD1%20Is%20Associated%20with%20Overall%20Survival%20in%20Patients%20with%20Metastatic%20Melanoma%20Treated%20with%20Anti-PD-1%20Monotherapy&author=M.%20de%20With&author=D.P.%20Hurkmans&author=E.O.%20De%20Hoop&author=A.%20Lalouti&author=S.%20Bins&volume=13&publication_year=2021&pages=1370&pmid=33803602&doi=10.3390/cancers13061370&)

47. Sunakawa Y., Cao S., Volz N.B., Berger M.D., Yang D., Parekh A., Zhang W., Matsusaka S., Ning Y., Stremitzer S., et al. Genetic Variations in Immunomodulatory Pathways to Predict Survival in Patients with Locoregional Gastric Cancer. Pharmacogenom. J. 2017;17:528–534. doi: 10.1038/tpj.2016.46.  [DOI](https://doi.org/10.1038/tpj.2016.46) | [PMC free article](/articles/PMC7507965/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27241062/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenom.%20J.&title=Genetic%20Variations%20in%20Immunomodulatory%20Pathways%20to%20Predict%20Survival%20in%20Patients%20with%20Locoregional%20Gastric%20Cancer&author=Y.%20Sunakawa&author=S.%20Cao&author=N.B.%20Volz&author=M.D.%20Berger&author=D.%20Yang&volume=17&publication_year=2017&pages=528-534&pmid=27241062&doi=10.1038/tpj.2016.46&)

48. Wagner M., Tupikowski K., Jasek M., Tomkiewicz A., Witkowicz A., Ptaszkowski K., Karpinski P., Zdrojowy R., Halon A., Karabon L. SNP-SNP Interaction in Genes Encoding PD-1/PD-L1 Axis as a Potential Risk Factor for Clear Cell Renal Cell Carcinoma. Cancers. 2020;12:3521. doi: 10.3390/cancers12123521.  [DOI](https://doi.org/10.3390/cancers12123521) | [PMC free article](/articles/PMC7760680/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33255938/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancers&title=SNP-SNP%20Interaction%20in%20Genes%20Encoding%20PD-1/PD-L1%20Axis%20as%20a%20Potential%20Risk%20Factor%20for%20Clear%20Cell%20Renal%20Cell%20Carcinoma&author=M.%20Wagner&author=K.%20Tupikowski&author=M.%20Jasek&author=A.%20Tomkiewicz&author=A.%20Witkowicz&volume=12&publication_year=2020&pages=3521&pmid=33255938&doi=10.3390/cancers12123521&)

49. Parakh S., Musafer A., Paessler S., Witkowski T., Suen C.S.N.L.W., Tutuka C.S.A., Carlino M.S., Menzies A.M., Scolyer R.A., Cebon J., et al. PDCD1 Polymorphisms May Predict Response to Anti-PD-1 Blockade in Patients With Metastatic Melanoma. Front. Immunol. 2021;12:672521. doi: 10.3389/fimmu.2021.672521.  [DOI](https://doi.org/10.3389/fimmu.2021.672521) | [PMC free article](/articles/PMC8220213/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34177913/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front.%20Immunol.&title=PDCD1%20Polymorphisms%20May%20Predict%20Response%20to%20Anti-PD-1%20Blockade%20in%20Patients%20With%20Metastatic%20Melanoma&author=S.%20Parakh&author=A.%20Musafer&author=S.%20Paessler&author=T.%20Witkowski&author=C.S.N.L.W.%20Suen&volume=12&publication_year=2021&pages=672521&pmid=34177913&doi=10.3389/fimmu.2021.672521&)

50. Breunis W.B., Tarazona-Santos E., Chen R., Kiley M., Rosenberg S.A., Chanock S.J. Influence of Cytotoxic T Lymphocyte-Associated Antigen 4 (CTLA4) Common Polymorphisms on Outcome in Treatment of Melanoma Patients with CTLA-4 Blockade. J. Immunother. 2008;31:586–590. doi: 10.1097/CJI.0b013e31817fd8f3.  [DOI](https://doi.org/10.1097/CJI.0b013e31817fd8f3) | [PMC free article](/articles/PMC3276400/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18528295/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Immunother.&title=Influence%20of%20Cytotoxic%20T%20Lymphocyte-Associated%20Antigen%204%20(CTLA4)%20Common%20Polymorphisms%20on%20Outcome%20in%20Treatment%20of%20Melanoma%20Patients%20with%20CTLA-4%20Blockade&author=W.B.%20Breunis&author=E.%20Tarazona-Santos&author=R.%20Chen&author=M.%20Kiley&author=S.A.%20Rosenberg&volume=31&publication_year=2008&pages=586-590&pmid=18528295&doi=10.1097/CJI.0b013e31817fd8f3&)

51. Parra E.R., Jiang M., Machado-Rugolo J., Yaegashi L.B., Prieto T., Farhat C., de Sá V.K., Nagai M.A., de Lima V.C.C., Takagaki T., et al. Variants in Epithelial-Mesenchymal Transition and Immune Checkpoint Genes Are Associated With Immune Cell Profiles and Predict Survival in Non-Small Cell Lung Cancer. Arch. Pathol. Lab. Med. 2020;144:1234–1244. doi: 10.5858/arpa.2019-0419-OA.  [DOI](https://doi.org/10.5858/arpa.2019-0419-OA) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32150457/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arch.%20Pathol.%20Lab.%20Med.&title=Variants%20in%20Epithelial-Mesenchymal%20Transition%20and%20Immune%20Checkpoint%20Genes%20Are%20Associated%20With%20Immune%20Cell%20Profiles%20and%20Predict%20Survival%20in%20Non-Small%20Cell%20Lung%20Cancer&author=E.R.%20Parra&author=M.%20Jiang&author=J.%20Machado-Rugolo&author=L.B.%20Yaegashi&author=T.%20Prieto&volume=144&publication_year=2020&pages=1234-1244&pmid=32150457&doi=10.5858/arpa.2019-0419-OA&)

52. Houcken J., Degenhart C., Bender K., König J., Frommer L., Kahaly G.J. PTPN22 and CTLA-4 Polymorphisms Are Associated With Polyglandular Autoimmunity. J. Clin. Endocrinol. Metab. 2018;103:1977–1984. doi: 10.1210/jc.2017-02577.  [DOI](https://doi.org/10.1210/jc.2017-02577) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29409002/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Endocrinol.%20Metab.&title=PTPN22%20and%20CTLA-4%20Polymorphisms%20Are%20Associated%20With%20Polyglandular%20Autoimmunity&author=J.%20Houcken&author=C.%20Degenhart&author=K.%20Bender&author=J.%20K%C3%B6nig&author=L.%20Frommer&volume=103&publication_year=2018&pages=1977-1984&pmid=29409002&doi=10.1210/jc.2017-02577&)

53. Bai L., Jiang J., Li H., Zhang R. Role of CD226 Rs763361 Polymorphism in Susceptibility to Multiple Autoimmune Diseases. Immunol. Investig. 2020;49:926–942. doi: 10.1080/08820139.2019.1703737.  [DOI](https://doi.org/10.1080/08820139.2019.1703737) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31854233/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Immunol.%20Investig.&title=Role%20of%20CD226%20Rs763361%20Polymorphism%20in%20Susceptibility%20to%20Multiple%20Autoimmune%20Diseases&author=L.%20Bai&author=J.%20Jiang&author=H.%20Li&author=R.%20Zhang&volume=49&publication_year=2020&pages=926-942&pmid=31854233&doi=10.1080/08820139.2019.1703737&)

54. AL-Eitan L., Al Qudah M., Al Qawasmeh M. Candidate Gene Association Analysis of Multiple Sclerosis in the Jordanian Arab Population: A Case-Control Study. Gene. 2020;758:144959. doi: 10.1016/j.gene.2020.144959.  [DOI](https://doi.org/10.1016/j.gene.2020.144959) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32683075/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gene&title=Candidate%20Gene%20Association%20Analysis%20of%20Multiple%20Sclerosis%20in%20the%20Jordanian%20Arab%20Population:%20A%20Case-Control%20Study&author=L.%20AL-Eitan&author=M.%20Al%20Qudah&author=M.%20Al%20Qawasmeh&volume=758&publication_year=2020&pages=144959&pmid=32683075&doi=10.1016/j.gene.2020.144959&)

55. Wang S., Zhang X., Leng S., Xu Q., Sheng Z., Zhang Y., Yu J., Feng Q., Hou M., Peng J., et al. Immune Checkpoint-Related Gene Polymorphisms Are Associated With Primary Immune Thrombocytopenia. Front. Immunol. 2021;11:615941. doi: 10.3389/fimmu.2020.615941.  [DOI](https://doi.org/10.3389/fimmu.2020.615941) | [PMC free article](/articles/PMC7874092/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33584705/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front.%20Immunol.&title=Immune%20Checkpoint-Related%20Gene%20Polymorphisms%20Are%20Associated%20With%20Primary%20Immune%20Thrombocytopenia&author=S.%20Wang&author=X.%20Zhang&author=S.%20Leng&author=Q.%20Xu&author=Z.%20Sheng&volume=11&publication_year=2021&pages=615941&pmid=33584705&doi=10.3389/fimmu.2020.615941&)

56. Lee K.M., Baris D., Zhang Y., Hosgood H.D., Menashe I., Yeager M., Zahm S.H., Wang S.S., Purdue M.P., Chanock S., et al. Common Single Nucleotide Polymorphisms in Immunoregulatory Genes and Multiple Myeloma Risk among Women in Connecticut. Am. J. Hematol. 2010;85:560–563. doi: 10.1002/ajh.21760.  [DOI](https://doi.org/10.1002/ajh.21760) | [PMC free article](/articles/PMC2910184/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20568250/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Hematol.&title=Common%20Single%20Nucleotide%20Polymorphisms%20in%20Immunoregulatory%20Genes%20and%20Multiple%20Myeloma%20Risk%20among%20Women%20in%20Connecticut&author=K.M.%20Lee&author=D.%20Baris&author=Y.%20Zhang&author=H.D.%20Hosgood&author=I.%20Menashe&volume=85&publication_year=2010&pages=560-563&pmid=20568250&doi=10.1002/ajh.21760&)

57. Auton A., Abecasis G.R., Altshuler D.M., Durbin R.M., Bentley D.R., Chakravarti A., Clark A.G., Donnelly P., Eichler E.E., Flicek P., et al. A Global Reference for Human Genetic Variation. Nature. 2015;526:68–74. doi: 10.1038/NATURE15393.  [DOI](https://doi.org/10.1038/NATURE15393) | [PMC free article](/articles/PMC4750478/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26432245/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=A%20Global%20Reference%20for%20Human%20Genetic%20Variation&author=A.%20Auton&author=G.R.%20Abecasis&author=D.M.%20Altshuler&author=R.M.%20Durbin&author=D.R.%20Bentley&volume=526&publication_year=2015&pages=68-74&pmid=26432245&doi=10.1038/NATURE15393&)

58. Paz-Ares L., Garassino M.C., Chen Y., Reinmuth N., Hotta K., Poltoratskiy A., Trukhin D., Hochmair M.J., Özgüroǧlu M., Ji J.H., et al. Durvalumab ± Tremelimumab + Platinum-Etoposide in Extensive-Stage Small Cell Lung Cancer (CASPIAN): Outcomes by PD-L1 Expression and Tissue Tumor Mutational Burden. Clin. Cancer Res. 2024;30:824–835. doi: 10.1158/1078-0432.CCR-23-1689.  [DOI](https://doi.org/10.1158/1078-0432.CCR-23-1689) | [PMC free article](/articles/PMC10870117/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/37801329/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Cancer%20Res.&title=Durvalumab%20%C2%B1%20Tremelimumab%20+%20Platinum-Etoposide%20in%20Extensive-Stage%20Small%20Cell%20Lung%20Cancer%20(CASPIAN):%20Outcomes%20by%20PD-L1%20Expression%20and%20Tissue%20Tumor%20Mutational%20Burden&author=L.%20Paz-Ares&author=M.C.%20Garassino&author=Y.%20Chen&author=N.%20Reinmuth&author=K.%20Hotta&volume=30&publication_year=2024&pages=824-835&pmid=37801329&doi=10.1158/1078-0432.CCR-23-1689&)

59. Erdfelder E., FAul F., Buchner A., Lang A.G. Statistical Power Analyses Using G*Power 3.1: Tests for Correlation and Regression Analyses. Behav. Res. Methods. 2009;41:1149–1160. doi: 10.3758/BRM.41.4.1149.  [DOI](https://doi.org/10.3758/BRM.41.4.1149) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19897823/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Behav.%20Res.%20Methods&title=Statistical%20Power%20Analyses%20Using%20G*Power%203.1:%20Tests%20for%20Correlation%20and%20Regression%20Analyses&author=E.%20Erdfelder&author=F.%20FAul&author=A.%20Buchner&author=A.G.%20Lang&volume=41&publication_year=2009&pages=1149-1160&pmid=19897823&doi=10.3758/BRM.41.4.1149&)

60. Purcell S., Neale B., Todd-Brown K., Thomas L., Ferreira M.A.R., Bender D., Maller J., Sklar P., De Bakker P.I.W., Daly M.J., et al. PLINK: A Tool Set for Whole-Genome Association and Population-Based Linkage Analyses. Am. J. Hum. Genet. 2007;81:559–575. doi: 10.1086/519795.  [DOI](https://doi.org/10.1086/519795) | [PMC free article](/articles/PMC1950838/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17701901/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Hum.%20Genet.&title=PLINK:%20A%20Tool%20Set%20for%20Whole-Genome%20Association%20and%20Population-Based%20Linkage%20Analyses&author=S.%20Purcell&author=B.%20Neale&author=K.%20Todd-Brown&author=L.%20Thomas&author=M.A.R.%20Ferreira&volume=81&publication_year=2007&pages=559-575&pmid=17701901&doi=10.1086/519795&)
