# CBR1 and CBR3 pharmacogenetics and their influence on doxorubicin disposition in Asian breast cancer patients

## Metadata
**Authors:** Suman Lal, Edwin Sandanaraj, Zee Wan Wong, Peter CS Ang, Nan Soon Wong, Edmund JD Lee, Balram Chowbay
**Journal:** Cancer Science
**Date:** 2008 Oct 13
**DOI:** [10.1111/j.1349-7006.2008.00903.x](https://doi.org/10.1111/j.1349-7006.2008.00903.x)
**PMID:** 19016765
**PMCID:** PMC11160041
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11160041/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC11160041/pdf/CAS-99-2045.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC11160041/pdf/CAS-99-2045.pdf)

## Abstract

The present study aimed to identify polymorphic genes encoding carbonyl reductases (CBR1, CBR3) and investigate their influence on doxorubicin disposition in Asian breast cancer patients (n = 62). Doxorubicin (60 mg/m2) was administered every 3 weeks for four to six cycles and the pharmacokinetic parameters were estimated using non‐compartmental analysis (WinNonlin). The Mann–Whitney U‐test was used to assess genotypic–phenotypic correlations. Five CBR1 (–48G>A, c.219G>C, c.627C>T, c.693G>A, +967G>A) and CBR3 (c.11G>A, c.255C>T, c.279C>T, c.606G>A, c.730G>A) polymorphisms were identified. The CBR1 D2 diplotypes were characterized by the presence of at least one variant allele at the c.627C>T and +967G>A loci. Patients in the CBR1 D1 diplotype group had significantly higher clearance (CL) normalized to body surface area (BSA) (CL/BSA[L/h/m2]: median 25.09; range 16.44–55.66) and significantly lower exposure levels; area under curve (AUC0–∞/dose/BSA [h/m5]; median 15.08; range 6.18–38.03) of doxorubicin compared with patients belonging to the CBR1 D2 diplotype group (CL/BSA[L/h/m2]; median 20.88; range 8.68–31.79, P = 0.014; and AUC0–∞/dose/BSA[h/m5]; median 21.35; range 9.82–67.17, P = 0.007 respectively). No significant influence of CBR3 polymorphisms on the pharmacokinetics of doxorubicin were observed in Asian cancer patients. The present exploratory study shows that CBR1 D2 diplotypes correlate with significantly higher exposure levels of doxorubicin, suggesting the possibility of lowered intracellular conversion to doxorubicinol in these patients. Further evaluation of carbonyl reductase polymorphisms in influencing the treatment efficacy of doxorubicin‐based chemotherapy in Asian cancer patients are warranted. (Cancer Sci 2008; 99: 2045–2054)

## Materials and Methods

**Healthy subjects. ** The healthy subjects comprised three ethnic groups predominant in the Asian population: Chinese, *n* = 100; Malay, *n* = 100; and Indian, *n* = 100. The ethnicity of the study subjects were confirmed by careful screening and verified against their National Registry Identification Cards. All participants provided approved informed consent for the study. The study was approved by the ethics review committee of the National Cancer Center, Singapore.

**Breast cancer patients. ** Patients who had histologically confirmed invasive breast cancer (*n* = 62) and received adjuvant chemotherapy with doxorubicin were recruited for the present study. Informed consent was obtained from all patients and the protocol was approved by the institutional ethics committee at the National Cancer Center, Singapore. Inclusion criteria required patients to have adequate bone marrow (absolute neutrophil count >1500/µL, platelet count >100 000/µL), hepatic function (aspartate amino transferase [AST] and alanine amino transferase [ALT] levels ≤2.5 and total bilirubin <2.0 times the upper limit of normal), and renal function (serum creatinine <140 mmol/L) documented at the time of enrollment. Patients with serious comorbidities, including poorly controlled diabetes mellitus, ischemic heart disease, uncontrolled hypertension, active infection or performance status score ≥2 on the Eastern Cooperative Oncology Group scale were excluded from the study. Pregnancy and use of growth factors during the cycle of chemotherapy were also considered as criteria for exclusion. No forms of endocrine therapy, immunotherapy, or biological response modifiers were allowed during the period of chemotherapy.

**Doxorubicin administration and pharmacokinetic analysis. ** Doxorubicin was administered at a dose of 60 mg/m^2^ intravenously over 20 min on a cycle of once every 3 weeks after standard premedications, with intravenous 10 mg dexamethasone, 50 mg diphenhydramine, 300 mg cimetidine, or 50 mg ranitidine. The adjuvant regimen also included intravenous 600 mg/m^2^ cyclophosphamide administered over 30 min once every 3 weeks. Hematological parameters, including hemoglobin, total leukocyte count, platelet count, and the absolute neutrophil count after the start of treatment (days 7 and 14) and prior to the start of each cycle, were measured. New treatment cycles were started only if the absolute neutrophil count was ≥1500/µL and the platelet count >100 000/µL. Blood samples for pharmacokinetic analysis were drawn at the following time points after the start of infusion on the first day of the first cycle of chemotherapy: at predose, 5, 15, and 30 min and at 1, 4, 8, and 24 h. Blood samples were collected in plain ethylene diamine tetracetic acid‐containing vacutainer glass tubes and centrifuged immediately at 1000* g * for 10 min. The plasma fraction was collected and stored at –20°C until analysis.

Plasma concentrations of doxorubicin and its major metabolite doxorubicinol were estimated by reversed‐phase high‐performance liquid chromatography with fluorescence detection as described previously.^(^ [24](#b24) ^)^ Briefly, following a single protein precipitation step, chromatographic separation was accomplished using a C‐18 column with a mobile phase consisting of 50 mmol/L sodium phosphate buffer:acetonitrile:1‐propanol (65:25:2, v/v), pH 2.0. The analytes were measured by fluorescence detection with an excitation wavelength of 480 nm and emission wavelength of 560 nm. The lower limits of quantitation were 10 ng/mL for doxorubicin and 5 ng/mL for doxorubicinol. The calibration curves were linear over a concentration range of 10–2500 ng/mL for doxorubicin and 5–1250 ng/mL for doxorubicinol. The within‐day and between‐day coefficients of variation were less than 13%.

The pharmacokinetic parameters were determined using a non‐linear regression program (WinNonLin version 2.1; Pharsight, Mountain View, CA, USA). The area under the plasma concentration–time curve (AUC) was calculated from time zero to the time (*t*) of the last detectable concentration (AUC_0→t_) using the trapezoidal rule. The area was extrapolated to infinity (AUC_0–∞_) by adding *C* _t_/*λz* to AUC_0→t_, where *C* _t_ is the last detectable plasma concentration and *λz* is the elimination rate constant. Peak plasma concentrations (*C* _max_) were identified directly from individual subject concentration–time curves.

**Identification of CBR1 and CBR3 variants by sequence analysis. ** Purified genomic DNA was isolated from peripheral blood samples (5 mL) using red cell lysis buffer and proteinase K digestion. The following primers were used for amplifying *CBR1*: exon 1, F‐5′‐gaggggtagggatggttcag‐3′, R‐5′‐cctagggaggcgttatggac‐3′; exon 2, F‐5′‐ggcagagggcactaagttttt‐3′, R‐5′‐ggacaaagtcctgaggcaaa‐3′; and exon 3, F‐5′‐tcacct ctctacgggattgtt‐3′, R‐5′‐tcccttgaccttttaggttga‐3′. The primers used for amplifying *CBR3* were: exon 1, F‐5′‐atttggc ttcggacacctc‐3′, R‐5′‐accctcctcggtcactcaag‐3′; exon 2, F‐5′‐aggatccactttgtcatgtgag‐3′, R‐5′‐gcacccctgagtgatttctg‐3′; exon 3, F‐5′‐agggcgagactctgtctcaa‐3′, R‐5′‐agcgag actccgtctcaaaa‐3′. Amplification of the exonic regions was done using polymerase chain reaction and the products electrophoresed on 2% agarose gels. Unincorporated nucleotides and primers were removed by incubating the amplified polymerase chain reaction fragments with Shrimp Alkaline Phosphatase (Promega, Madison, WI, USA) and exonuclease 1 (New England Biolabs, MA, USA) for 45 min at 37°C followed by 15 min at 72°C.^(^ [25](#b25) ^)^ The sequencing reactions were done using the 3730 DNA Analyzer (Applied Biosystems, Foster City, CA, USA) and the resultant trace files were base‐called using Sequencing Analysis v.5.2 (Applied Biosystems) and assembled with SeqScape v.2.5 (Applied Biosystems).

**Statistical analysis. ** Fisher's exact test was used to assess the differences in genotype and allele distributions among the different groups in healthy subjects and cancer patients. The possible contribution of subject characteristics (ethnicity, age, height, and body surface area [BSA]) on the pharmacokinetics of doxorubicin and doxorubicinol were determined using univariate analysis. Other independent variables investigated to evaluate their relative contribution to the variability in pharmacokinetic parameters of doxorubicin and doxorubicinol included patient biochemical parameters such as albumin, total protein and markers of hepatic function (AST, ALT, alkaline phosphatase, and bilirubin) and renal function (serum creatinine). The non‐parametric Mann–Whitney *U*‐test was used for pairwise comparisons between genotype groups. Pharmacokinetic parameters are reported as median values unless otherwise indicated. Univariate and multivariate analysis was carried out to identify the significant association of patient covariates and genotypes with pharmacokinetic parameters. Backward exclusion regression analysis was carried out to identify the independent influence of genetic covariates in the present study compared to previously reported classes in our patient cohort. The minimum level of statistical significance was set at *P* = 0.05 for all statistical comparisons. Linkage analysis of the *CBR1* and *CBR3* polymorphisms was carried out using Helixtree software (HelixTree Genetics Analysis Software Golden Helix, MT, USA). All statistical analysis was carried out using Stata Statistical Software release 7.0 (Stata Corporation, TX, USA).

## Results

**Patient demographics. ** The majority of the breast cancer patients belonged to the Chinese ethnic group (74%), followed by Malays (18%) and Indians (8%). The median age, height, and BSA of the patients were 51 years (range 29–73 years), 154 cm (range 144–168 cm), and 1.52 m^2^ (range 1.23–1.95 m^2^) respectively.

**Doxorubicin pharmacokinetics. ** Plasma samples for pharmacokinetic analysis were available in 52 of the 62 patients and a high degree of interpatient variability was observed with regard to the pharmacokinetic parameters of doxorubicin and doxorubicinol ([Table 1](#t1)). Approximately six‐fold variation in clearance (median 22.9 [L/h/m^2^]; range 8.7–55.7 [L/h/m^2^]) and 13‐fold variation in the volume of distribution at steady state (*V* _ss_) of doxorubicin (median 237.5 [L/m^2^]; range 53.9–703.7 [L/m^2^]) were observed among the cancer patients. Interpatient variability was also high with regard to doxorubicinol pharmacokinetic parameters with more than six‐fold variation observed in its AUC_0–∞_ (median 9.6 [h/m^5^]; range 3.7–24.3 [h/m^5^]). Univariate analysis revealed the patients’ heights and weights to be significant covariates influencing the AUC_0–∞_ (*P <* 0.015 and *P* < 0.0001, respectively) and *C* _max_ of doxorubicin (*P <* 0.028 and *P* < 0.0001 respectively). Age and ethnicity were identified as significant covariates in relation to the half life (*t* _1/2_; h) of doxorubicin (*P <* 0.018). The relationship between the half life of doxorubicin and age was further analyzed following stratification according to age (below and above 60 years). The *t* _1/2_ of doxorubicin was significantly higher in patients above 60 years of age (median 15.73 years; range 4.7–24.6 years) compared with patients below 60 years of age (median 12.54 years; range 8.1–17.8 years; *P* < 0.02). No significant influences of other covariates were observed on the pharmacokinetics of doxorubicin and doxorubicinol.

### Table 1.

| Pharmacokinetic parameter | Median (range) |
| --- | --- |
| Doxorubicin |  |
| Cmax/dose/BSA (/m5) | 34.01 (4.5–97.7) |
| AUC0–∞/dose/BSA (h/m5) | 17.5 (6.2–67.2) |
| t1/2 (h) | 15.4 (4.7–24.6) |
| CL/BSA (L/h/m2) | 22.9 (8.7–55.7) |
| Vss/BSA (L/m2) | 237.5 (53.9–703.7) |
| Doxorubicinol |  |
| Cmax/dose/BSA (10−2/m5) | 0.33 (0.15–2.0) |
| AUC0–∞/dose/BSA (h/m5) | 9.6 (3.7–24.3) |
| t1/2 (h) | 27.5 (6.3–112.4) |

Table 1 Caption: Summary of doxorubicin pharmacokinetic parameters in Asian breast cancer patients

** CBR1 polymorphisms in healthy subjects and breast cancer patients. ** Five polymorphisms were identified by direct sequencing of the exonic regions of the *CBR1* gene ([Table 2](#t2)). Two identified polymorphisms were in the 5′‐UTR (–*48G>A*) and 3′‐UTR (+*967G>A*) regions. Among the coding region polymorphisms, one synonymous transversion (c.*219G>C*) in exon 1, and two synonymous transitions (c.*627C>T* and c.*693G>A*) in exon 3 were identified.

### Table 2.

| Polymorphisms | Location | n | Genotype frequencies |  |  | Allele frequency |  |
| --- | --- | --- | --- | --- | --- | --- | --- |
| –48G>A [rs11542168] | 5′ UTR |   | GG | GA | AA | G | A |
| Healthy subjects |  |  |  |  |  |  |  |
| Chinese |   | 100 | 95 (95) | 5 (5) | 0 | 0.98 | 0.02 |
| Malays |   | 100 | 99 (99) | 1 (1) | 0 | 0.99 | 0.01 |
| Indians |   | 100 | 100 (100) | 0 | 0 | 1 | 0 |
| Pooled Asian population |   | 300 | 294 (98) | 6 (2) | 0 | 0.99 | 0.01 |
| Asian cancer patients |   | 62 | 58 (93.5) | 4 (6.5) | 0 | 0.97 | 0.03 |
| c.219G>C [rs25678] | Exon 1 |   | GG | GC | CC | G | C |
| Healthy subjects |  |  |  |  |  |  |  |
| Chinese |   | 100 | 70 (70) | 27 (27) | 3 (3) | 0.84 | 0.16 |
| Malays |   | 100 | 77 (77) | 16 (16) | 7 (7) | 0.85 | 0.15 |
| Indians |   | 99 | 69 (69.7) | 21 (21.2) | 9 (9.1) | 0.80 | 0.20 |
| Pooled Asian population |   | 299 | 216 (72.2) | 64 (21.4) | 19 (6.4) | 0.83 | 0.17 |
| Asian cancer patients |   | 60 | 36 (60) | 18 (30) | 6 (10) | 0.75 | 0.25 |
| c.627C>T [rs20572] | Exon 3 |   | CC | CT | TT | C | T |
| Healthy subjects |  |  |  |  |  |  |  |
| Chinese |   | 100 | 62 (62) | 25 (25) | 13 (13) | 0.74 | 0.26 |
| Malays |   | 100 | 66 (66) | 17 (17) | 17 (17) | 0.75 | 0.25 |
| Indians |   | 100 | 68 (68) | 27 (27) | 5 (5) | 0.82 | 0.18 |
| Pooled Asian population |   | 300 | 196 (65.3) | 69 (23) | 35 (11.7) | 0.77 | 0.23 |
| Asian cancer patients |   | 60 | 38 (63.4) | 10 (16.6) | 12 (20) | 0.72 | 0.28 |
| c.693G>A [rs2230192] | Exon 3 |   | GG | GA | AA | G | A |
| Healthy subjects |  |  |  |  |  |  |  |
| Chinese |   | 100 | 100 (100) | 0 | 0 | 1 | 0 |
| Malays |   | 100 | 99 (99) | 1 (1) | 0 | 0.99 | 0.01 |
| Indians |   | 100 | 99 (99) | 1 (1) | 0 | 0.99 | 0.01 |
| Pooled Asian population |   | 300 | 298 (99.3) | 2 (0.7) | 0 | 0.997 | 0.003 |
| Asian cancer patients |   | 62 | 62 (100) | 0 | 0 | 1 | 0 |
| +967G>A [rs9024] | 3′ UTR |   | GG | GA | AA | G | A |
| Healthy subjects |  |  |  |  |  |  |  |
| Chinese |   | 100 | 63 (63) | 28 (28) | 9 (9) | 0.77 | 0.23 |
| Malays |   | 100 | 66 (66) | 21 (21) | 13 (13) | 0.77 | 0.23 |
| Indians |   | 100 | 72 (72) | 23 (23) | 5 (5) | 0.84 | 0.17 |
| Pooled Asian population |   | 300 | 201 (67) | 72 (24) | 27 (9) | 0.79 | 0.21 |
| Asian cancer patients |   | 60 | 37 (61.7) | 14 (23.3) | 9 (15) | 0.73 | 0.27 |

Table 2 Caption: Genotype and allele frequency of CBR1 polymorphisms in Asian ethnic groups and breast cancer patients

The genotype and allele frequencies of the five *CBR1* polymorphisms identified in the Asian ethnic groups and breast cancer patients are listed in [Table 2](#t2). The –*48G>A *polymorphism was monomorphic in the Indian population but was present at low frequencies among the Chinese (0.02) and Malays (0.01). No significant differences in the allele and genotype frequencies of the c.*219G>C* polymorphism were observed between different Asian ethnic groups. The allele frequency of the c.*627C>T* polymorphism was significantly higher among the Chinese (0.26) when compared with the Indians (0.18, *P* = 0.05). The c.*693G>A* polymorphism was monomorphic in the Chinese population but was present at low frequencies among Malays and Indians (0.01 in each case). The *+967G>A* polymorphism was present at a significantly higher frequency among Chinese and Malays (0.23 in each case) when compared with the Indians (0.17, *P* = 0.01 in each case). The allele frequencies of the *CBR1* polymorphisms among the breast cancer patients conformed to the frequencies found among the local population.

** CBR1 linkage disequilibrium analysis. ** Pairwise analysis showed moderate linkage of c.*219G>C* with the c.*627C>T* and +*967G>A* polymorphisms among Chinese and Malays ([measure of disequilibrium]|D′| > 0.7 in all groups), whereas the degree of linkage was weak among Indians (c.*219G>C* and c.*627C>T*; [|D′|~0.57] and c.*219G>C* and +*967G>A *[|D′|~0.41]). The degree of linkage between the c.*627C>T* and +*967G>A *polymorphisms was high in all ethnic groups (|D′| > 0.8 in all cases).

[Table 3](#t3) shows the *CBR1* haplotypes and diplotypes identified in the healthy Asian ethnic groups and breast cancer patients. A total of eight *CBR1* haplotypes were determined in the Chinese population and four high‐frequency haplotypes (H1, H2, H3, and H4) accounted for a cumulative frequency of 95%. Among the seven haplotypes inferred in the Malay population, three were identified as high‐frequency haplotypes (H1, H2, and H3) and had a cumulative frequency of 90%. There were seven haplotypes identified in the Indian population and four high‐frequency haplotypes (H1, H2, H3, and H5) constituted 93% of the total haplotype frequencies. The major diplotypes (>5%) among the Chinese (H1‐H1, H1‐H2), Malays (H1‐H1, H1‐H2, H1‐H3, H1‐H6, and H3‐H3), and Indians (H1‐H1, H1‐H2, and H1‐H4) comprised 71, 74, and 77% of the total diplotypes, respectively, in each ethnic group.

### Table 3.

| Chinese (n = 100) |  |  |  |  | Malays (n = 100) |  |  |  |  | Indians (n = 100) |  |  |  |  | Asian breast cancer Patients (n = 62) |  |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Haplotypes † |   |  |  | Frequency | Haplotypes |   |  |  | Frequency | Haplotypes |   |  |  | Frequency | Haplotypes |   |  |  | Frequency |
| H1 | G | C | G | 0.70 | H1 | G | C | G | 0.69 | H1 | G | C | G | 0.71 | H1b | G | C | G | 0.65 |
| H2 | C | T | A | 0.13 | H2 | C | T | A | 0.11 | H2 | C | T | A | 0.09 | H2b | C | T | A | 0.13 |
| H3 | G | T | A | 0.07 | H3 | G | T | A | 0.10 | H3 | C | C | G | 0.08 | H3b | G | T | A | 0.10 |
| H4 | G | T | G | 0.05 | H4 | G | T | G | 0.03 | H4 | G | T | A | 0.05 | H4b | C | T | G | 0.05 |
| H5 | C | C | G | 0.02 | H5 | C | C | G | 0.03 | H5 | C | T | G | 0.03 | H5b | C | C | G | 0.04 |
| H6 | G | C | A | 0.02 | H6 | G | C | A | 0.03 | H6 | G | C | A | 0.03 | H6b | C | C | A | 0.03 |
| H7 | C | C | A | 0.01 | H7 | C | T | G | 0.01 | H7 | G | T | G | 0.01 | H7b | G | T | G | <0.01 |
| H8 | C | T | G | <0.01 |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
| Diplotypes |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
| H1‐H1 | GG | CC | GG | 0.56 | H1‐H1 | GG | CC | GG | 0.57 | H1‐H1 | GG | CC | GG | 0.59 | H1b‐H1b | GG | CC | GG | 0.56 |
| H1‐H2 | GC | CT | GA | 0.15 | H1‐H2 | GC | CT | GA | 0.07 | H1‐H2 | GC | CT | GA | 0.13 | H1b‐H2b | GC | CT | GA | 0.08 |
|   |   |   |   |   | H2‐H2 | CC | TT | AA | 0.06 | H1‐H4 | GG | CT | GA | 0.05 | H2b‐H3b | GC | TT | AA | 0.07 |
|   |   |   |   |   | H1‐H3 | GG | CT | GA | 0.05 |   |   |   |   |   | H2b‐H2b | CC | TT | AA | 0.05 |
|   |   |   |   |   | H1‐H6 | GG | CC | GA | 0.05 |   |   |   |   |   | H3b‐H4b | GG | TT | GA | 0.05 |
|   |   |   |   |   | H3‐H3 | GG | TT | AA | 0.05 |   |   |   |   |   |   |   |   |   |   |

Table 3 Caption: CBR1 haplotypes and diplotypes in Asian ethnic groups and breast cancer patients

The linkage pattern between the c.*219G>C*, c.*627C>T*, and +*967G>A* polymorphisms (|D′| ~0.7, in all groups) in the breast cancer patients was similar to those of the healthy populations from which they were recruited ([Fig. 1a](#f1)). A total of seven haplotype groups were identified and the major haplotypes (>5%) were H1_b_ (*G‐C‐G*), H2_b_ (*C‐T‐A*), and H4_b_ (*C‐T‐G*) with a cumulative frequency of 83% ([Table 3](#t3)). Among the 12 diplotypes inferred, the major diplotypes were H1_b_‐H1_b_ (56%), H1_b_‐H2_b_ (8%), H2_b_‐H3_b_ (7%), H2_b_‐H2_b_ (5%), and H3_b_‐H4_b_ (5%), which represented a cumulative frequency of 81%.

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7e51/11160041/43543b11f587/CAS-99-2045-g001.jpg)

Pairwise linkage disequilibrium among (a) CBR1 and (b) CBR3 polymorphisms in Asian breast cancer patients.

** CBR1 polymorphisms and doxorubicin pharmacokinetics. ** The influence of the major *CBR1* polymorphisms (c.*219G>C*, c.*627C>T*, and +*967G>A*) on the pharmacokinetics of doxorubicin in Asian breast cancer patients is summarized in [Table 4](#t4). There was no significant influence of the c.*219G>C* polymorphism on doxorubicin and doxorubicinol pharmacokinetics. The genotypic effect of the c.*627C>T* and +*967G>A* polymorphisms on the pharmacokinetics of doxorubicin and doxorubicinol were similar ([Table 4](#t4)). Patients carrying the reference c.*627CC* genotype had significantly higher clearance (median 24.07 CL/BSA[L/h/m^2^]; range 16.44–49.40 CL/BSA[L/h/m^2^]) of doxorubicin when compared to patients who were heterozygous (median 20.13 CL/BSA[L/h/m^2^]; range 8.68–31.79 CL/BSA[L/h/m^2^], *P* = 0.028). Breast cancer patients carrying the reference c.*627CC* genotype showed a trend toward lower exposure levels (median 15.44 AUC_0–∞_/dose/BSA[h/m^5^]; range 6.18–38.03 AUC_0–∞_/dose/BSA[h/m^5^]) of doxorubicin compared with patients who were heterozygous (median 20.36 AUC_0–∞_/dose/BSA[h/m^5^]; range 9.82–67.17 AUC_0–∞_/dose/BSA[h/m^5^], *P* = 0.066) or homozygous for the variant allele (median 20.74 AUC_0–∞_/dose/BSA[h/m^5^]; range 15.15–30.16 AUC_0–∞_/dose/BSA[h/m^5^], *P* = 0.60). Significantly lower peak plasma concentrations of doxorubicinol (median 0.32 *C* _max_/dose/BSA[h/m^5^]; range 0.16–2.0 *C* _max_/dose/BSA[h/m^5^]) were observed in patients with the reference c.*627CC* genotypes compared to heterozygotes (median 0.49 *C* _max_/dose/BSA[h/m^5^]; range 0.18–1.49 *C* _max_/dose/BSA[h/m^5^], *P* = 0.030).

### Table 4.

| Pharmacokinetic parameters | CBR1 genotypes |  |  | Pairwise P‐values* |  |  |
| --- | --- | --- | --- | --- | --- | --- |
| c.219G>C | GG (n = 30) | GC (n = 15) | CC (n = 5) | GG versus GC | GC versus CC | GG versus CC |
| Doxorubicin |  |  |  |  |  |  |
| AUC0−∞/dose/BSA (h/m5) | 15.85 (6.18–67.17) | 18.76 (9.82–38.03) | 15.15 (12.96–20.74) | 0.139 | 0.191 | 0.944 |
| Cmax/dose/BSA (/m5) | 33.12 (4.55–97.70) | 41.59 (12.80–81.22) | 29.92 (17.59–42.38) | 0.126 | 0.191 | 0.689 |
| CL/BSA (L/h/m2) | 23.30 (8.68–55.66) | 21.57 (13.80–31.79) | 26.09 (19.57–28.53) | 0.094 | 0.163 | 0.906 |
| Doxorubicinol |  |  |  |  |  |  |
| AUC0−∞/dose/BSA (h/m5) | 9.473 (3.679–21.71) | 11.64 (6.48–24.34) | 7.77 (6.3–16.8) | 0.064 | 0.278 | 0.923 |
| Cmax/dose/BSA (/m5) | 0.31 (0.1483–2.004) | 0.39 (0.18–1.49) | 0.38 (0.28–0.49) | 0.152 | 0.541 | 0.556 |
| c.627C>T | CC (n = 31) | CT (n = 10) | TT (n = 9) | CC versus CT | CT versus TT | CC versus TT |
| Doxorubicin |  |  |  |  |  |  |
| AUC0−∞/dose/BSA (h/m5) | 15.44 (6.18–38.03) | 20.36 (9.82–67.17) | 20.74 (15.15–30.16) | 0.066 | 0.968 | 0.060 |
| Cmax/dose/BSA (/m5) | 31.08 (4.55–97.70) | 41.81 (12.80–81.22) | 37.98 (27.76–80.72) | 0.187 | 1.000 | 0.164 |
| CL/BSA (L/h/m2) | 24.07 (16.44–49.40) | 20.13 (8.68–31.79) | 23.66 (13.80–28.53) | 0.028 | 0.315 | 0.348 |
| Doxorubicinol |  |  |  |  |  |  |
| AUC0−∞/dose/BSA (h/m5) | 9.47 (3.68–21.71) | 12.87 (6.59–24.34) | 10.52 (6.3–16.8) | 0.054 | 0.328 | 0.357 |
| Cmax/dose/BSA (/m5) | 0.32 (0.16–2.0) | 0.49 (0.18–1.49) | 0.38 (0.18–0.57) | 0.030 | 0.182 | 0.650 |
| +967G>A | GG (n = 30) | GA (n = 13) | AA (n = 7) | GG versus GA | GA versus AA | GG versus AA |
| Doxorubicin |  |  |  |  |  |  |
| AUC0−∞/dose/BSA (h/m5) | 15.62 (6.18–29.33) | 22.43 (9.816–67.17) | 18.76 (15.15–31.63) | 0.024 | 0.526 | 0.126 |
| Cmax/dose/BSA (/m5) | 32.03 (4.55–97.70) | 40.96 (12.80–81.22) | 42.04 (29.92–73.53) | 0.239 | 0.751 | 0.126 |
| CL/BSA (L/h/m2) | 24.42 (16.44–49.40) | 20.43 (8.675–31.79) | 24.93 (15.04–28.53) | 0.009 | 0.178 | 0.473 |
| Doxorubicinol |  |  |  |  |  |  |
| AUC0−∞/dose/BSA (h/m5) | 9.5 (3.68–24.34) | 10.40 (6.48–16.79) | 10.87 (6.25–18.71) | 0.402 | 0.880 | 0.484 |
| Cmax/dose/BSA (/m5) | 0.32 (0.16–2.0) | 0.48 (0.22–1.49) | 0.39 (0.18–0.57) | 0.026 | 0.178 | 0.862 |

Table 4 Caption: CBR1 polymorphisms and pharmacokinetics of doxorubicin in Asian breast cancer patients

With regard to the +*967G>A* polymorphism, breast cancer patients carrying the reference +*967GG* genotype also had significantly higher clearance (median 24.42 CL/BSA[L/h/m^2^]; range 16.44–49.40 CL/BSA[L/h/m^2^]) of doxorubicin when compared with patients who were heterozygous (median 20.43 CL/BSA[L/h/m^2^]; range 8.68–31.79 CL/BSA[L/h/m^2^]) for the polymorphism (*P =* 0.009). Patients carrying the reference +*967GG* genotype had significantly lower exposure levels (median 15.62 AUC_0–∞_/dose/BSA[h/m^5^]; range 6.18–29.33 AUC_0–∞_/dose/BSA[h/m^5^]) of doxorubicin when compared with patients who were heterozygous (median 22.43 AUC_0–∞_/dose/BSA[h/m^5^]; range 9.82–67.17 AUC_0–∞_/dose/BSA[h/m^5^]) (*P =* 0.024). The +*967GG* genotypes were also associated with a significantly lower peak plasma concentration of doxorubicinol (median 0.32 *C* _max_/dose/BSA[h/m^5^]; range 0.16–2.0 *C* _max_/dose/BSA[h/m^5^]) when compared with the patients harboring the heterozygous genotypes (median 0.48 *C* _max_/dose/BSA[h/m^5^]; range 0.22–1.49 *C* _max_/dose/BSA[h/m^5^]) (*P =* 0.026).

No major influences of individual *CBR1* haplotypes on doxorubicin pharmacokinetics were observed in the present study. We examined the phenotypic associations of linked high‐frequency *CBR1 *polymorphisms with the pharmacokinetics of doxorubicin by considering diplotypes that included composite genetic information from all of the loci of interest. Sequence clustering using the complete linkage algorithm identified two major groups (D1 and D2) among the 12 *CBR1* diplotypes in Asian breast cancer patients ([Fig. 2](#f2)). The D2 diplotype group was characterized by the presence of at least one variant allele at both the c.*627 *and +*967* loci. Significant differences were observed in the doxorubicin pharmacokinetic parameters between the two diplotype groups. Patients belonging to the D1 diplotype group had significantly higher clearance of doxorubicin (median 25.09 CL/BSA[L/h/m^2^]; range 16.44–55.66 CL/BSA[L/h/m^2^]) compared with patients belonging to the D2 diplotype group (median 20.88 CL/BSA[L/h/m^2^]; range 8.68–31.79 CL/BSA[L/h/m^2^]) (*P =* 0.014) ([Fig. 3a](#f3)). Patients belonging to the D1 diplotype group also had significantly lower exposure levels of doxorubicin (median 15.08 AUC_0–∞_/dose/BSA[h/m^5^]; range 6.18–38.03 AUC_0–∞_/dose/BSA[h/m^5^]) compared to patients belonging to the D2 diplotype group (median 21.35 AUC_0–∞_/dose/BSA[h/m^5^]; range 9.82–67.17 AUC_0–∞_/dose/BSA[h/m^5^]) (*P =* 0.007) ([Fig. 3b](#f3)). The peak plasma concentrations of doxorubicin in patients belonging to the D1 diplotype group were also significantly lower (median 25.49 *C* _max_/dose/BSA[/m^5^]; range 4.55–77.67* C* _max_/dose/BSA[/m^5^]) compared with patients belonging to the D2 diplotype group (median 41.50 *C* _max_/dose/BSA[/m^5^]; range 12.80–81.22 *C* _max_/dose/BSA[/m^5^]) (*P =* 0.038). With regard to doxorubicinol, patients harboring the D1 diplotype family had lower exposure levels of doxorubicinol (median 8.99 AUC_0–∞_/dose/BSA[h/m^5^]; range 3.68–20.63 AUC_0–∞_/dose/BSA[h/m^5^]) when compared with patients belonging to the D2 diplotype family (median 11.28 AUC_0–∞_/dose/BSA[h/m^5^]; range 6.25–18.71 AUC_0–∞_/dose/BSA[h/m^5^]) (*P* = 0.057).

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7e51/11160041/0aa5dcac8940/CAS-99-2045-g002.jpg)

CBR1 diplotype groups observed in Asian breast cancer patients.

### Figure 3.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7e51/11160041/5d6ae18c55bb/CAS-99-2045-g003.jpg)

Influence of CBR1 diplotypes on (a) clearance and (b) exposure levels of doxorubicin in Asian breast cancer patients. N, number of subjects.

** CBR3 polymorphisms in healthy subjects and breast cancer patients. ** Direct sequencing of the 5′‐UTR, 3′‐UTR, and coding regions of the *CBR3* gene identified a total of five polymorphisms. One non‐synonymous transition (c.*11G>A*; C4Y) and two synonymous transitions (c.*255T>C* and c.*279C>T*) were identified in exon 1. A synonymous (c.*606G>A*) and a non‐synonymous (c.*730G>A*; V244M) polymorphism were identified in exon 3 of the *CBR3* gene.

The genotype and allele frequencies of the five *CBR3* polymorphisms identified in the Asian healthy ethnic groups and the breast cancer patients are listed in [Table 5](#t5). The allele frequency of the c.*11G>A* polymorphism was significantly higher among the Chinese (0.38) and Malays (0.41) when compared to Indians (0.28) (Chinese vs Indians; *P* = 0.03, Malays vs Indians; *P* = 0.01). The c.*255T>C* polymorphism was observed to be monomorphic in the Chinese healthy subjects. The allele frequency of the c.*279C>T* polymorphism was significantly higher among the Chinese (0.39) when compared to Indians (0.29, *P* = 0.03). The c.*606G>A* polymorphism was monomorphic in the Chinese and Malay healthy subjects but was present at an allele frequency of 0.03 among the Indians. The allele frequency of the c.*730G>A *polymorphism was also significantly higher among the Indians when compared to the Chinese and Malays (Indians vs Chinese, *P* < 0.0001; Indians vs Malays, *P* = 0.003). The genotype and allele frequencies of *CBR3* polymorphisms among cancer patients conformed to the frequency in healthy study subjects.

### Table 5.

| Polymorphisms | Location |  | Genotype frequencies |  |  | Allele frequency |  |
| --- | --- | --- | --- | --- | --- | --- | --- |
| c.11G>A [rs8133052] | Exon 1 | n | GG | GA | AA | G | A |
| Healthy subjects |  |  |  |  |  |  |  |
| Chinese |   | 100 | 40 (40) | 44 (44) | 16 (16) | 0.62 | 0.38 |
| Malays |   | 100 | 33 (33) | 52 (52) | 15 (15) | 0.59 | 0.41 |
| Indians |   | 100 | 51 (51) | 42 (42) | 7 (7) | 0.72 | 0.28 |
| Pooled Asian population |   | 300 | 124 (41.3) | 138 (46) | 38 (12.7) | 0.64 | 0.36 |
| Asian Cancer patients |   | 62 | 32 (51.6) | 18 (29) | 12 (19.4) | 0.66 | 0.33 |
| c.255T>C [rs881711] | Exon 1 |   | TT | TC | CC | T | C |
| Healthy subjects |  |  |  |  |  |  |  |
| Chinese |   | 97 | 63 (64.9) | 34 (35.1) | 0 | 0.82 | 0.18 |
| Malays |   | 100 | 72 (72) | 25 (25) | 3 (3) | 0.84 | 0.16 |
| Indians |   | 100 | 75 (75) | 24 (24) | 1 (1) | 0.87 | 0.13 |
| Pooled Asian population |   | 297 | 210 (70.7) | 83 (27.9) | 4 (1.3) | 0.85 | 0.15 |
| Asian cancer patients |   | 61 | 41 (67.2) | 17 (27.9) | 3 (4.9) | 0.81 | 0.19 |
| c.279C>T [rs881712] | Exon 1 |   | CC | CT | TT | C | T |
| Healthy subjects |  |  |  |  |  |  |  |
| Chinese |   | 98 | 38 (38.8) | 44 (44.9) | 16 (16.3) | 0.61 | 0.39 |
| Malays |   | 100 | 39 (39) | 46 (46) | 15 (15) | 0.62 | 0.38 |
| Indians |   | 100 | 50 (50) | 42 (42) | 8 (8) | 0.71 | 0.29 |
| Pooled Asian population |   | 298 | 127 (42.6) | 132 (44.3) | 39 (13.1) | 0.65 | 0.35 |
| Asian cancer patients |   | 61 | 33 (54.1) | 16 (26.2) | 12 (19.7) | 0.67 | 0.33 |
| c.606G>A [rs17849671] | Exon 3 |   | GG | GA | AA | G | A |
| Healthy subjects |  |  |  |  |  |  |  |
| Chinese |   | 100 | 100 (100) | 0 | 0 | 1 | 0 |
| Malays |   | 99 | 99 (100) | 0 | 0 | 1 | 0 |
| Indians |   | 97 | 92 (94.8) | 5 (5.2) | 0 | 0.97 | 0.03 |
| Pooled Asian population |   | 296 | 291 (98.3) | 5 (1.7) | 0 | 0.99 | 0.01 |
| Asian cancer patients |   | 61 | 59 (96.7) | 2 (3.3) | 0 | 0.98 | 0.02 |
| c.730G>A [rs1056892] | Exon 3 |   | GG | GA | AA | G | A |
| Healthy subjects |  |  |  |  |  |  |  |
| Chinese |   | 99 | 37 (37.4) | 47 (47.5) | 15 (15.1) | 0.61 | 0.38 |
| Malays |   | 98 | 31 (31.6) | 51 (52.0) | 16 (16.4) | 0.58 | 0.42 |
| Indians |   | 98 | 22 (22.4) | 40 (40.8) | 36 (36.8) | 0.43 | 0.57 |
| Pooled Asian population |   | 295 | 90 (30.5) | 138 (46.8) | 67 (22.7) | 0.54 | 0.46 |
| Asian cancer patients |   | 62 | 24 (38.7) | 23 (37.1) | 15 (24.2) | 0.57 | 0.43 |

Table 5 Caption: Genotype and allele frequency of CBR3 polymorphisms in Asian ethnic groups and breast cancer patients

** CBR3 linkage disequilibrium analysis. ** Strong linkage of the c.*11G>A *polymorphism with the c.*255T>C*, c.*279C>T*, and c.*730G>A* polymorphisms was observed in all of the ethnic groups (|D′| > 0.9 in all cases). The c.*11G>A *and c.*606G>A *polymorphisms were strongly linked in the Indian healthy population (|D′| > 0.9). The linkage between the c.*255T>C *and c.*730G>A* polymorphisms were strong among Chinese and Malays (|D′| > 0.9) but weak among Indians (|D′|~0.3). Strong linkage was also observed between the c.*279C>T* and c.*730G>A* polymorphisms among all of the ethnic groups (|D′| > 0.9 in all groups). A total of six *CBR3* haplotypes were determined in the Chinese population and three high‐frequency haplotypes (H1, H2, and H3) accounted for a cumulative frequency of 92% ([Table 6](#t6)). Among the nine haplotypes inferred in the Malay population, four were identified as high‐frequency haplotypes (H1, H2, H3, and H4) and had a cumulative frequency of 94%. There were nine haplotypes identified in the Indian population, of which the H1, H2, and H3 haplotypes constituted 90% of the total haplotype frequencies. The H1 haplotype was tagged by the c.*730G>A* polymorphism, H2 haplotype by the linked c.*11G>A *and c.*279C>T* polymorphisms, and the H3 haplotype by the c.*255T>C* polymorphism in all ethnic groups studied.

### Table 6.

| Haplotypes | Chinese (n = 100) |  |  |  |  | Frequency | Haplotypes | Malays (n = 100) |  |  |  |  | Frequency | Haplotypes | Indians (n = 100) |  |  |  |  | Frequency | Haplotypes | Asian breast cancer patients (n = 62) |  |  |  |  | Frequency |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| H1 | G | T | C | G | A | 0.39 | H1 | G | T | C | G | A | 0.40 | H1 | G | T | C | G | A | 0.53 | H1b | G | T | C | G | A | 0.41 |
| H2 | A | T | T | G | G | 0.36 | H2 | A | T | T | G | G | 0.34 | H2 | A | T | T | G | G | 0.26 | H2b | A | T | T | G | G | 0.31 |
| H3 | G | C | C | G | G | 0.17 | H3 | G | C | C | G | G | 0.15 | H3 | G | C | C | G | G | 0.11 | H3b | G | C | C | G | G | 0.19 |
| H4 | G | T | C | G | G | 0.04 | H4 | A | T | C | G | G | 0.05 | H4 | G | T | C | A | G | 0.03 | H4b | G | T | C | G | G | 0.03 |
| H5 | G | T | T | G | G | 0.02 | H5 | G | T | T | G | G | 0.02 | H5 | G | C | C | G | A | 0.02 | H5b | A | T | C | G | G | 0.03 |
| H6 | A | T | C | G | G | 0.01 | H6 | A | T | T | G | A | 0.01 | H6 | G | T | T | G | G | 0.02 | H6b | G | T | T | G | A | 0.01 |
|   |   |   |   |   |   |   | H7 | G | T | C | G | G | 0.01 | H7 | G | T | C | G | G | 0.01 | H7b | G | T | C | A | G | 0.01 |
|   |   |   |   |   |   |   | H8 | G | C | T | G | A | <0.01 | H8 | G | T | T | G | A | 0.01 | H8b | G | T | C | A | A | 0.01 |
|   |   |   |   |   |   |   | H9 | A | C | T | G | G | <0.01 | H9 | A | T | C | G | G | 0.01 | H9b | G | T | T | G | G | <0.01 |
| Diplotypes |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
| H1‐H2 | GA | TT | CT | GG | GA | 0.24 | H1‐H2 | GA | TT | CT | GG | GA | 0.29 | H1‐H1 | GG | TT | CC | GG | AA | 0.31 | H1b‐H1b | GG | TT | CC | GG | AA | 0.23 |
| H1‐H3 | GG | TC | CC | GG | GA | 0.18 | H1‐H1 | GG | TT | CC | GG | AA | 0.16 | H1‐H2 | GA | TT | CT | GG | GA | 0.28 | H2b‐H2b | AA | TT | TT | GG | GG | 0.18 |
| H1‐H1 | GG | TT | CC | GG | AA | 0.15 | H1‐H3 | GG | TC | CC | GG | GA | 0.11 | H2‐H3 | GA | TC | CT | GG | GG | 0.10 | H1b‐H3b | GG | TC | CC | GG | GA | 0.16 |
| H2‐H2 | AA | TT | TT | GG | GG | 0.15 | H2‐H2 | AA | TT | TT | GG | GG | 0.11 | H1‐H3 | GG | TC | CC | GG | GA | 0.09 | H2b‐H3b | GA | TC | CT | GG | GG | 0.11 |
| H2‐H3 | GA | TC | CT | GG | GG | 0.15 | H2‐H3 | GA | TT | CT | GG | GG | 0.11 | H2‐H2 | AA | TT | TT | GG | GG | 0.06 | H1b‐H2b | GA | TT | CT | GG | GA | 0.11 |
|   |   |   |   |   |   |   | H1‐H4 | GA | TT | CT | GG | GA | 0.07 |   |   |   |   |   |   |   | H3b‐H3b | GG | CC | CC | GG | GG | 0.05 |

Table 6 Caption: CBR3 haplotypes and diplotypes in Asian ethnic groups and breast cancer patients

Pairwise analysis of *CBR3* polymorphisms in cancer patients showed a high degree of linkage between the c.*11G>A*, c.*255T>*C, c.*279C>T*, c.*606G>A*, and c.*730G>A* polymorphisms (|D′| > 0.9 in each case) ([Fig. 1b](#f1)). Haplotype analysis revealed a total of nine haplotype groups, of which the H1_b_ (*G‐T‐C‐G‐A*), H2_b_ (*A‐T‐T‐G‐G*), and H3_b_ (*G‐C‐C‐G‐G*) haplotypes had a cumulative frequency of 91% ([Table 6](#t6)). Similar to the healthy subjects, the H1_b_, H2_b_, and H3_b_ haplotypes in breast cancer patients were tagged by the c.*730G>A*, c.*11G>A*, and c.*279C>T*, and c.*255T>C* polymorphisms, respectively. A total of 14 diplotypes were inferred by the expectation–maximization algorithm and the major diplotypes (H1_b_‐H1_b_[23%], H2_b_‐H2_b_[18%], H1_b_‐H3_b_[16%], H2_b_‐H3_b_[11%], H1_b_‐H2_b_[11%], and H3_b_‐H3_b_[5%]) accounted for a cumulative frequency of 84%.

No significant influences of the *CBR3* polymorphisms (c.*11G>A,* c.*255T>C*, c.*279C>T*, c.*606G>A*, and c.*730G>A*) on doxorubicin and doxorubicinol pharmacokinetics were observed in the present study (data not shown). Examination of the influence of the major *CBR3* haplotype and diplotype structures also failed to reveal any significant association with doxorubicin disposition in Asian breast cancer patients.

## Discussion

Polymorphisms in the coding or regulatory regions of genes encoding CBR drug‐metabolizing enzymes may have significant phenotypic influences due to their critical roles in the phase I metabolism of both endogenous and xenobiotic compounds.^(^ [26](#b26) ^)^ The alterations in tissue‐specific expression and activity of CBR could also contribute to interindividual variations in the pharmacokinetics and pharmacodynamics of several putative drug substrates. The present study is the first report on the pharmacogenetic profiling of the human CBR genes (*CBR1* and *CBR3*) in three distinct Asian ethnic groups (Chinese, Malays, and Indians) and their impact on the pharmacokinetics of doxorubicin and doxorubicinol in Asian breast cancer patients.

Screening the exon and exon–intron boundaries of the *CBR1* gene in 300 healthy Asian subjects identified five polymorphisms (–*48G>A*, c.*219G>C*, c.*627C>T*, c.*693G>A*, and +*967G>A*) that exhibited significant interethnic variations in genotype and allele frequencies. The allele frequency of the c.*219G>C* polymorphism among the Chinese (0.16), Malay (0.15), and Indian (0.20) ethnic groups were similar to the frequency reported in the Caucasian population (0.15).^(^ [17](#b17) ^)^ The non‐synonymous c.*262G>A* (V88I) polymorphism was absent in the Asian health subjects and patients. This polymorphism is present at a low frequency among African‐Americans (<0.2%) and has been postulated to be associated with higher intracellular prostaglandin E2 levels.^(^ [17](#b17) ^)^ The allele frequency of the c.*627C>T* polymorphism in the Indian population (0.18) was similar to the Caucasians (0.15) but lower compared to the Chinese subjects (0.26, *P* = 0.05). The c.*693G>A* polymorphism reported in Caucasians (0.02)^(^ [17](#b17) ^)^ was monomorphic among the Chinese and occurred at a low frequency in both Malays and Indians (0.01 in each ethnic group).

Earlier reports by Lakhman *et al*. showed wide interethnic variation in the allele frequency of the* CBR3* c.*730G>A* (V244M) polymorphism (0.07 among Pacific origin and 0.67 among Indo‐Pakistanis) that was proposed to cause structural variation and alterations in dinucleotide binding interactions, leading to higher catalytic efficiency of the enzyme.^(^ [18](#b18) ^)^ In the present study, the c.*730G>A *polymorphism occurred at a significantly higher allele frequency among the Indians (0.57) compared to the Chinese (0.38) and Malays (0.42) (*P <* 0.0001 and *P* = 0.003, respectively). The significantly lower allele frequency of the c.*11G>A* and c.*279C>T* polymorphism among Indians when compared to the rest of the ethnic groups and the absence of c.*255T>C* and c.*606G>A *polymorphisms in the Chinese subjects suggests the existence of wide interethnic differences in the frequency of *CBR3* polymorphisms.

Association studies with polymorphic data from linked loci suggest that diplotype constitution, which characterizes the subset of every single‐locus genotype in a gene and is representative of the type of chromosome pairs in an individual,^(^ [27](#b27), [28](#b28) ^)^ could be major determinants of phenotypes.^(^ [29](#b29) ^)^ In the present study, homozygosity for the reference allele at the *CBR1c.627C>T* and +*967G>A* loci was observed to be associated with increased clearance and reduced exposure levels of doxorubicin in Asian breast cancer patients. The reasons for the lack of influence of the homozygous variant alleles at the *CBR1* c.*627C>T *and +*967G>A *loci on doxorubicin pharmacokinetic parameters are unknown and could possibly be due to a smaller cohort of breast cancer patients studied. Because the *CBR1* c.*219G>C*, c.*627C>T*, and +*967G>A* polymorphisms were observed to be in linkage disequilibrium, further analysis was done to delineate the influences of the inferred *CBR1* diplotypes on doxorubicin pharmacokinetics. Pairwise comparisons revealed that patients belonging to the *CBR1* D1 diplotype group had 20% higher clearance (*P =* 0.007) and 29% lower exposure (AUC_0–∞_) of doxorubicin (*P =* 0.014) compared to patients belonging to the *CBR1* D2 diplotype group. The peak plasma concentration of doxorubicin was also 38% lower among patients belonging to the *CBR1* D1 diplotype group (*P =* 0.038). This suggests that the increased clearance of doxorubicin in patients harboring the *CBR1 *D1 diplotype cohort may probably be due to the increased CBR activity, resulting in increased metabolic conversion of doxorubicin to doxorubicinol among these patients. The *CBR1* D2 diplotypes were characterized by the presence of at least one variant allele at the c.*627* and 3′‐UTR +*967* loci. There have been no studies to date that examined the functional roles of the synonymous c.*627C>T* polymorphism. Recent reports investigating polymorphic variations in drug transporters have indicated that synonymous polymorphisms can alter mRNA stability and translation efficiency, contributing to alterations in drug disposition.^(^ [30](#b30) ^)^ The 3′‐UTR regions have several *cis*‐acting binding sites as several transcription factors and mutations in these regions have been associated with alterations in protein expression and disease phenotypes.^(^ [31](#b31), [32](#b32) ^)^ Future studies are required to delineate the functional roles of the synonymous c.*627C>T *polymorphism and the 3′‐UTR +*967G>A* polymorphism in affecting the pharmacokinetics of doxorubicin and doxorubicinol.

The increased clearance of doxorubicin in breast cancer patients belonging to the *CBR1* D1 diplotype group was, however, not associated with increased exposure levels of doxorubicinol. Possible reasons that may have accounted for this observation may include further metabolism of doxorubicinol to its aglycone, doxorubicinolaglycone,^(^ [32](#b32) ^)^ thereby rendering it more favorable for excretion. In addition, the activities of other carbonyl‐group reducing enzymes such as aldehyde reductases (*AKR1A1*) and dihydrodiol dehydrogenase 2 (*AKR1C2*) have been identified to be important in contributing to the intracellular formation of doxorubicinol,^(^ [10](#b10), [33](#b33) ^)^ and may also influence its clearance. Doxorubicinol is also a substrate for the efflux transporter ABCB1 substrate and its clearance may hence be affected by its affinity toward ABCB1 and the variable ABCB1 expression levels in the biliary canalicular membrane of hepatocytes.^(^ [34](#b34) ^)^ We have previously shown that homozygosity for the reference allele at the *ABCB1* c.*1236C>T*, c.*2677G>T/A*, and c.*3435C>T* loci was associated with significantly lower peak plasma concentrations of doxorubicin and doxorubicinol.^(^ [35](#b35) ^)^ To examine whether the observed influence of *CBR1* diplotypes is independent of the influence of *ABCB1* homozygosity at the respective loci, we carried out a backward exclusion regression incorporating the published *ABCB1 *classes and *CBR1* diplotypes. It was observed that the influence of *CBR1* diplotypes was retained as significant despite the exclusion of *ABCB1* classes in the model. Regression analysis showed that the observed contribution of *CBR1* diplotypes to the doxorubicin clearance was highly significant and independent of *ABCB1* classes (*R* ^2^ = 40.4%; *P* = 0.008). We also carried out a multivariate analysis to examine the *CBR1* diplotypic effect on the increased exposure of doxorubicin by incorporating the significant patient covariates (age, ethnicity, height, and weight), which were significantly associated with *t* _1/2_ and AUC_0–∞_ of doxorubicin in the univariate analysis. Multivariate analysis revealed that doxorubicin exposure was significantly associated with patients’ weights and *CBR1* diplotype groups (*P <* 0.0001 and *P* = 0.022 respectively). Age and ethnicity had no significant influence on doxorubicin exposure levels. With regard to *CBR3* pharmacogenetics, the *CBR3* c.*730G>A* polymorphism was previously reported to alter the catalytic efficiency of the enzyme.^(^ [18](#b18) ^)^ However, no influence of the c.*730G>A* polymorphism on doxorubicin pharmacokinetics was observed in the present study. The lack of association of C*BR3 *polymorphic variants with doxorubicin disposition could be due to the relatively low level of hepatic *CBR3* expression.^(^ [15](#b15) ^)^

Similar to the roles played by ATP‐binding cassette (ABC) transporters, CBR polymorphisms could also contribute to the mechanisms of drug resistance, treatment efficacy, and toxicity during cancer chemotherapy. CBR expression in K562 cells resulted in protection against cytotoxicity when treated with doxorubicin,^(^ [36](#b36) ^)^ whereas cells selected for doxorubicin resistance showed a 60% increase in CBR mRNA levels.^(^ [37](#b37) ^)^ Taxanes, such as paclitaxel, have been shown to increase the intracellular conversion of doxorubicin to doxorubicinol by binding to the regulatory allosteric and catalytic sites of CBR enzymes, thereby influencing the efficacy and toxicity of combination chemotherapy regimens.^(^ [38](#b38) ^)^ CBR‐mediated intracellular conversion of doxorubicin to doxorubicinol has also been proposed to be important in the development of anthracycline cardiotoxicity.^(^ [39](#b39) ^)^ *In vitro* studies have shown a higher potency of doxorubicinol than doxorubicin in depressing myocardial contractility and inhibiting ion channel pumps,^(^ [40](#b40) ^)^ whereas studies on transgenic mice overexpressing *CBR1* in the heart showed higher intracardiac levels of doxorubicinol and increased signs of myocardial damage after doxorubicin administration.^(^ [39](#b39) ^)^ Although the precise molecular mechanisms that mediate anthracycline cardiotoxicity remain to be identified, alterations in intracellular concentrations of doxorubicin and doxorubicinol by polymorphic CBR could be assumed to play significant roles in the incidence of cardiotoxicity observed in a subset of patients on chemotherapy.

In conclusion, the present study identified novel *CBR1* and *CBR3* polymorphisms and characterized their haplotype and diplotype structures in three distinct Asian ethnic groups and breast cancer patients. The study is the first report on the contribution of CBR polymorphisms on doxorubicin pharmacokinetics in Asian breast cancer patients. The present exploratory study shows that *CBR1* D2 diplotypes tagged by at least one variant allele at the *CBR1* c.*627* and *+967* loci correlate with significantly higher exposure levels of doxorubicin, suggesting the possibility of lowered intracellular conversion to doxorubicinol in these patients. Given the modest sample size of the patients analyzed, the findings should be considered exploratory in nature, and larger studies with a higher sample size must be carried out to arrive at definitive conclusions and to further evaluate the role of CBR polymorphisms in influencing treatment efficacy in Asian cancer patients on doxorubicin‐based adjuvant chemotherapy.

## Acknowledgments

The authors would like to thank the Singhealth Research Fund (SRF‐SU110/2004) and Singapore Cancer Syndicate (SCS‐PS0023) for funding this project.

## References

1. Guengerich FP. Cytochrome P450. Structure, Mechanism, and Biochemistry, 2nd edn. New York: Plenum Press, 1995.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Structure,%20Mechanism,%20and%20Biochemistry&author=FP%20Guengerich&publication_year=1995&)

2. Felsted RL, Richter DR, Bachur NR. Rat liver aldehyde reductase. Biochem Pharmacol 1977; 26: 1117–24.  [DOI](https://doi.org/10.1016/0006-2952(77)90054-5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19021/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20Pharmacol&title=Rat%20liver%20aldehyde%20reductase&author=RL%20Felsted&author=DR%20Richter&author=NR%20Bachur&volume=26&publication_year=1977&pages=1117-24&pmid=19021&doi=10.1016/0006-2952(77)90054-5&)

3. Felsted RL, Bachur NR. Mammalian carbonyl reductases. Drug Metab Rev 1980; 11: 1–60.  [DOI](https://doi.org/10.3109/03602538008994021) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7000481/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Rev&title=Mammalian%20carbonyl%20reductases&author=RL%20Felsted&author=NR%20Bachur&volume=11&publication_year=1980&pages=1-60&pmid=7000481&doi=10.3109/03602538008994021&)

4. Forrest GL, Gonzalez B. Carbonyl reductase. Chem Biol Interact 2000; 129: 21–40.  [DOI](https://doi.org/10.1016/s0009-2797(00)00196-4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11154733/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chem%20Biol%20Interact&title=Carbonyl%20reductase&author=GL%20Forrest&author=B.%20Gonzalez&volume=129&publication_year=2000&pages=21-40&pmid=11154733&doi=10.1016/s0009-2797(00)00196-4&)

5. Tanaka M, Bateman R, Rauh D et al . An unbiased cell morphology‐based screen for new, biologically active small molecules. PLoS Biol 2005; 3: e128.  [DOI](https://doi.org/10.1371/journal.pbio.0030128) | [PMC free article](/articles/PMC1073692/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15799708/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Biol&title=An%20unbiased%20cell%20morphology%E2%80%90based%20screen%20for%20new,%20biologically%20active%20small%20molecules&author=M%20Tanaka&author=R%20Bateman&author=D%20Rauh&volume=3&publication_year=2005&pages=e128&pmid=15799708&doi=10.1371/journal.pbio.0030128&)

6. Jarabak J. Polycyclic aromatic hydrocarbon quinone‐mediated oxidation reduction cycling catalyzed by a human placental NADPH‐linked carbonyl reductase. Arch Biochem Biophys 1991; 291: 334–8.  [DOI](https://doi.org/10.1016/0003-9861(91)90143-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1659323/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arch%20Biochem%20Biophys&title=Polycyclic%20aromatic%20hydrocarbon%20quinone%E2%80%90mediated%20oxidation%20reduction%20cycling%20catalyzed%20by%20a%20human%20placental%20NADPH%E2%80%90linked%20carbonyl%20reductase&author=J.%20Jarabak&volume=291&publication_year=1991&pages=334-8&pmid=1659323&doi=10.1016/0003-9861(91)90143-7&)

7. Maser E, Richter E, Friebertshauser J. The identification of 11β‐hydroxysteroid dehydrogenase as carbonyl reductase of the tobacco‐specific nitrosamine 4‐(methylnitrosamino)‐1‐(3‐pyridyl)‐1‐butanone. Eur J Biochem 1996; 238: 484–9.  [DOI](https://doi.org/10.1111/j.1432-1033.1996.0484z.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8681962/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Biochem&title=The%20identification%20of%2011%CE%B2%E2%80%90hydroxysteroid%20dehydrogenase%20as%20carbonyl%20reductase%20of%20the%20tobacco%E2%80%90specific%20nitrosamine%204%E2%80%90(methylnitrosamino)%E2%80%901%E2%80%90(3%E2%80%90pyridyl)%E2%80%901%E2%80%90butanone&author=E%20Maser&author=E%20Richter&author=J.%20Friebertshauser&volume=238&publication_year=1996&pages=484-9&pmid=8681962&doi=10.1111/j.1432-1033.1996.0484z.x&)

8. Gonzalez B, Akman S, Doroshow J, Rivera H, Kaplan WD, Forrest GL. Protection against daunorubicin cytotoxicity by expression of a cloned human carbonyl reductase cDNA in K562 leukemia cells. Cancer Res 1995; 15: 4646–50.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/7553643/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=Protection%20against%20daunorubicin%20cytotoxicity%20by%20expression%20of%20a%20cloned%20human%20carbonyl%20reductase%20cDNA%20in%20K562%20leukemia%20cells&author=B%20Gonzalez&author=S%20Akman&author=J%20Doroshow&author=H%20Rivera&author=WD%20Kaplan&volume=15&publication_year=1995&pages=4646-50&pmid=7553643&)

9. Oppermann U, Filling C, Hult M et al . Short‐chain dehydrogenases/reductases (SDR): the 2002 update. Chem Biol Interact 2003; 143–144: 247–53.  [DOI](https://doi.org/10.1016/s0009-2797(02)00164-3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12604210/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chem%20Biol%20Interact&title=Short%E2%80%90chain%20dehydrogenases/reductases%20(SDR):%20the%202002%20update&author=U%20Oppermann&author=C%20Filling&author=M%20Hult&volume=143%E2%80%93144&publication_year=2003&pages=247-53&pmid=12604210&doi=10.1016/s0009-2797(02)00164-3&)

10. Ohara H, Miyabe Y, Deyashiki Y, Matsuura K, Hara A. Reduction of drug ketones by dihydrodiol dehydrogenases, carbonyl reductase and aldehyde reductase of human liver. Biochem Pharmacol 1995; 50: 221–7.  [DOI](https://doi.org/10.1016/0006-2952(95)00124-i) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7632166/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20Pharmacol&title=Reduction%20of%20drug%20ketones%20by%20dihydrodiol%20dehydrogenases,%20carbonyl%20reductase%20and%20aldehyde%20reductase%20of%20human%20liver&author=H%20Ohara&author=Y%20Miyabe&author=Y%20Deyashiki&author=K%20Matsuura&author=A.%20Hara&volume=50&publication_year=1995&pages=221-7&pmid=7632166&doi=10.1016/0006-2952(95)00124-i&)

11. Breyer‐Pfaff U, Nill K. High‐affinity stereoselective reduction of the enantiomers of ketotifen and of ketonic nortriptyline metabolites by aldo‐keto reductases from human liver. Biochem Pharmacol 2000; 59: 249–60.  [DOI](https://doi.org/10.1016/s0006-2952(99)00319-6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10609553/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20Pharmacol&title=High%E2%80%90affinity%20stereoselective%20reduction%20of%20the%20enantiomers%20of%20ketotifen%20and%20of%20ketonic%20nortriptyline%20metabolites%20by%20aldo%E2%80%90keto%20reductases%20from%20human%20liver&author=U%20Breyer%E2%80%90Pfaff&author=K.%20Nill&volume=59&publication_year=2000&pages=249-60&pmid=10609553&doi=10.1016/s0006-2952(99)00319-6&)

12. Someya T, Muratake T, Hirokane G, Shibasaki M, Shimoda K, Takahashi S. Interindividual variation in bromperidol metabolism and relationship to therapeutic effects. J Clin Psychopharmacol 2000; 20: 175–80.  [DOI](https://doi.org/10.1097/00004714-200004000-00009) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10770455/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Psychopharmacol&title=Interindividual%20variation%20in%20bromperidol%20metabolism%20and%20relationship%20to%20therapeutic%20effects&author=T%20Someya&author=T%20Muratake&author=G%20Hirokane&author=M%20Shibasaki&author=K%20Shimoda&volume=20&publication_year=2000&pages=175-80&pmid=10770455&doi=10.1097/00004714-200004000-00009&)

13. Shimoda K, Shibasaki M, Inaba T, Cheung SW, Someya T, Takahashi S. Carbonyl reduction of timiperone in human liver cytosol. Pharmacol Toxicol 1998; 83: 164–8.  [DOI](https://doi.org/10.1111/j.1600-0773.1998.tb01463.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9820877/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacol%20Toxicol&title=Carbonyl%20reduction%20of%20timiperone%20in%20human%20liver%20cytosol&author=K%20Shimoda&author=M%20Shibasaki&author=T%20Inaba&author=SW%20Cheung&author=T%20Someya&volume=83&publication_year=1998&pages=164-8&pmid=9820877&doi=10.1111/j.1600-0773.1998.tb01463.x&)

14. Ahmed NK, Felsted RL, Bachur NR. Heterogeneity of anthracycline antibiotic carbonyl reductases in mammalian livers. Biochem Pharmacol 1978; 27: 2713–19.  [DOI](https://doi.org/10.1016/0006-2952(78)90047-3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31888/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20Pharmacol&title=Heterogeneity%20of%20anthracycline%20antibiotic%20carbonyl%20reductases%20in%20mammalian%20livers&author=NK%20Ahmed&author=RL%20Felsted&author=NR%20Bachur&volume=27&publication_year=1978&pages=2713-19&pmid=31888&doi=10.1016/0006-2952(78)90047-3&)

15. Matsunaga T, Shintani S, Hara A. Multiplicity of mammalian reductases for xenobiotic carbonyl compounds. Drug Metab Pharmacokinet 2006; 21: 1–18.  [DOI](https://doi.org/10.2133/dmpk.21.1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16547389/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Pharmacokinet&title=Multiplicity%20of%20mammalian%20reductases%20for%20xenobiotic%20carbonyl%20compounds&author=T%20Matsunaga&author=S%20Shintani&author=A.%20Hara&volume=21&publication_year=2006&pages=1-18&pmid=16547389&doi=10.2133/dmpk.21.1&)

16. Avramopoulos D, Cox T, Forrest GL, Chakravarti A, Antonarakis SE. Linkage mapping of the carbonyl reductase (CBR) gene on human chromosome 21 using a DNA polymorphism in the 3′ untranslated region. Genomics 1992; 13: 447–8.  [DOI](https://doi.org/10.1016/0888-7543(92)90268-w) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1612603/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genomics&title=Linkage%20mapping%20of%20the%20carbonyl%20reductase%20(CBR)%20gene%20on%20human%20chromosome%2021%20using%20a%20DNA%20polymorphism%20in%20the%203%E2%80%B2%20untranslated%20region&author=D%20Avramopoulos&author=T%20Cox&author=GL%20Forrest&author=A%20Chakravarti&author=SE%20Antonarakis&volume=13&publication_year=1992&pages=447-8&pmid=1612603&doi=10.1016/0888-7543(92)90268-w&)

17. Gonzalez‐Covarrubias V, Ghosh D, Lakhman SS et al . A functional genetic polymorphism on human carbonyl reductase 1 (CBR1 V88I) impacts on catalytic activity and NADPH binding affinity. Drug Metab Dispos 2007; 35: 973–80.  [DOI](https://doi.org/10.1124/dmd.107.014779) | [PMC free article](/articles/PMC2442771/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17344335/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=A%20functional%20genetic%20polymorphism%20on%20human%20carbonyl%20reductase%201%20(CBR1%20V88I)%20impacts%20on%20catalytic%20activity%20and%20NADPH%20binding%20affinity&author=V%20Gonzalez%E2%80%90Covarrubias&author=D%20Ghosh&author=SS%20Lakhman&volume=35&publication_year=2007&pages=973-80&pmid=17344335&doi=10.1124/dmd.107.014779&)

18. Lakhman SS, Ghosh D, Blanco JG. Functional significance of a natural allelic variant of human carbonyl reductase 3 (CBR3). Drug Metab Dispos 2005; 33: 254–7.  [DOI](https://doi.org/10.1124/dmd.104.002006) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15537833/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Functional%20significance%20of%20a%20natural%20allelic%20variant%20of%20human%20carbonyl%20reductase%203%20(CBR3)&author=SS%20Lakhman&author=D%20Ghosh&author=JG%20Blanco&volume=33&publication_year=2005&pages=254-7&pmid=15537833&doi=10.1124/dmd.104.002006&)

19. Sciotti M, Wermuth B. Coenzyme specificity of human monomeric carbonyl reductase. contribution of Lys‐15, Ala‐37 and Arg‐38. Chem Biol Interact 2001; 130–132: 871–8.  [DOI](https://doi.org/10.1016/s0009-2797(00)00242-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11306102/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chem%20Biol%20Interact&title=Coenzyme%20specificity%20of%20human%20monomeric%20carbonyl%20reductase.%20contribution%20of%20Lys%E2%80%9015,%20Ala%E2%80%9037%20and%20Arg%E2%80%9038&author=M%20Sciotti&author=B.%20Wermuth&volume=130%E2%80%93132&publication_year=2001&pages=871-8&pmid=11306102&doi=10.1016/s0009-2797(00)00242-8&)

20. Takanashi S, Bachur NR. Adriamycin metabolism in man: Evidence from urinary metabolites. Drug Metab Dispos 1976; 4: 79–87.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/3405/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Adriamycin%20metabolism%20in%20man:%20Evidence%20from%20urinary%20metabolites&author=S%20Takanashi&author=NR%20Bachur&volume=4&publication_year=1976&pages=79-87&pmid=3405&)

21. Speth PA, Van Hoesel QG, Haanen C. Clinical pharmacokinetics of doxorubicin. Clin Pharmacokinet 1988; 15: 15–31.  [DOI](https://doi.org/10.2165/00003088-198815010-00002) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/3042244/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacokinet&title=Clinical%20pharmacokinetics%20of%20doxorubicin&author=PA%20Speth&author=QG%20Van%20Hoesel&author=C.%20Haanen&volume=15&publication_year=1988&pages=15-31&pmid=3042244&doi=10.2165/00003088-198815010-00002&)

22. Huang Y. Pharmacogenetics/genomics of membrane transporters in cancer chemotherapy. Cancer Metastasis Rev 2007; 26: 183–201.  [DOI](https://doi.org/10.1007/s10555-007-9050-6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17323126/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Metastasis%20Rev&title=Pharmacogenetics/genomics%20of%20membrane%20transporters%20in%20cancer%20chemotherapy&author=Y.%20Huang&volume=26&publication_year=2007&pages=183-201&pmid=17323126&doi=10.1007/s10555-007-9050-6&)

23. Gustafson DL, Long ME. Alterations in P‐glycoprotein expression in mouse tissues by doxorubicin: implications for pharmacokinetics in multiple dosing regimens. Chemico-Biol Interacts 2001; 138: 43–57.  [DOI](https://doi.org/10.1016/s0009-2797(01)00257-5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11640914/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chemico-Biol%20Interacts&title=Alterations%20in%20P%E2%80%90glycoprotein%20expression%20in%20mouse%20tissues%20by%20doxorubicin:%20implications%20for%20pharmacokinetics%20in%20multiple%20dosing%20regimens&author=DL%20Gustafson&author=ME%20Long&volume=138&publication_year=2001&pages=43-57&pmid=11640914&doi=10.1016/s0009-2797(01)00257-5&)

24. Zhou Q, Chowbay B. Determination of doxorubicin and its metabolites in rat serum and bile by LC: application to preclinical pharmacokinetic studies. J Pharm Biomed Anal 2002; 30: 1063–74.  [DOI](https://doi.org/10.1016/s0731-7085(02)00442-9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12408897/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharm%20Biomed%20Anal&title=Determination%20of%20doxorubicin%20and%20its%20metabolites%20in%20rat%20serum%20and%20bile%20by%20LC:%20application%20to%20preclinical%20pharmacokinetic%20studies&author=Q%20Zhou&author=B.%20Chowbay&volume=30&publication_year=2002&pages=1063-74&pmid=12408897&doi=10.1016/s0731-7085(02)00442-9&)

25. Nickerson DA, Tobe VO, Taylor SL. PolyPhred: automating the detection and genotyping of single nucleotide substitutions using fluorescence‐based resequencing. Nucleic Acids Res 1997; 25: 2745–51.  [DOI](https://doi.org/10.1093/nar/25.14.2745) | [PMC free article](/articles/PMC146817/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9207020/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=PolyPhred:%20automating%20the%20detection%20and%20genotyping%20of%20single%20nucleotide%20substitutions%20using%20fluorescence%E2%80%90based%20resequencing&author=DA%20Nickerson&author=VO%20Tobe&author=SL%20Taylor&volume=25&publication_year=1997&pages=2745-51&pmid=9207020&doi=10.1093/nar/25.14.2745&)

26. Oppermann UC, Maser E. Molecular and structural aspects of xenobiotic carbonyl metabolizing enzymes. Role of reductases and dehydrogenases in xenobiotic phase I reactions. Toxicology 2000; 144: 71–81.  [DOI](https://doi.org/10.1016/s0300-483x(99)00192-4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10781873/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Toxicology&title=Molecular%20and%20structural%20aspects%20of%20xenobiotic%20carbonyl%20metabolizing%20enzymes.%20Role%20of%20reductases%20and%20dehydrogenases%20in%20xenobiotic%20phase%20I%20reactions&author=UC%20Oppermann&author=E.%20Maser&volume=144&publication_year=2000&pages=71-81&pmid=10781873&doi=10.1016/s0300-483x(99)00192-4&)

27. Lima JJ, Matsushima N, Kissoon N, Wang J, Sylvester JE, Jusko WJ. Modeling the metabolic effects of terbutaline in β2‐adrenergic receptor diplotypes. Clin Pharmacol Ther 2004; 76: 27–37.  [DOI](https://doi.org/10.1016/j.clpt.2004.03.006) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15229461/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Modeling%20the%20metabolic%20effects%20of%20terbutaline%20in%20%CE%B22%E2%80%90adrenergic%20receptor%20diplotypes&author=JJ%20Lima&author=N%20Matsushima&author=N%20Kissoon&author=J%20Wang&author=JE%20Sylvester&volume=76&publication_year=2004&pages=27-37&pmid=15229461&doi=10.1016/j.clpt.2004.03.006&)

28. Ho RH, Choi L, Lee W et al . Effect of drug transporter genotypes on pravastatin disposition in European‐ and African‐American participants. Pharmacogenet Genomics 2007; 17: 647–56.  [DOI](https://doi.org/10.1097/FPC.0b013e3280ef698f) | [PMC free article](/articles/PMC4063287/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17622941/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Effect%20of%20drug%20transporter%20genotypes%20on%20pravastatin%20disposition%20in%20European%E2%80%90%20and%20African%E2%80%90American%20participants&author=RH%20Ho&author=L%20Choi&author=W%20Lee&volume=17&publication_year=2007&pages=647-56&pmid=17622941&doi=10.1097/FPC.0b013e3280ef698f&)

29. Kitamura Y, Moriguchi M, Kaneko H et al . Determination of probability distribution of diplotype configuration (diplotype distribution) for each subject from genotypic data using the EM algorithm. Ann Hum Genet 2002; 66: 183–93.  [DOI](https://doi.org/10.1017/S0003480002001124) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12174210/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Hum%20Genet&title=Determination%20of%20probability%20distribution%20of%20diplotype%20configuration%20(diplotype%20distribution)%20for%20each%20subject%20from%20genotypic%20data%20using%20the%20EM%20algorithm&author=Y%20Kitamura&author=M%20Moriguchi&author=H%20Kaneko&volume=66&publication_year=2002&pages=183-93&pmid=12174210&doi=10.1017/S0003480002001124&)

30. Sauna ZE, Kimchi‐Sarfaty C, Ambudkar SV, Gottesman MM. Silent polymorphisms speak: how they affect pharmacogenomics and the treatment of cancer. Cancer Res 2007; 67: 9609–12.  [DOI](https://doi.org/10.1158/0008-5472.CAN-07-2377) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17942888/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=Silent%20polymorphisms%20speak:%20how%20they%20affect%20pharmacogenomics%20and%20the%20treatment%20of%20cancer&author=ZE%20Sauna&author=C%20Kimchi%E2%80%90Sarfaty&author=SV%20Ambudkar&author=MM%20Gottesman&volume=67&publication_year=2007&pages=9609-12&pmid=17942888&doi=10.1158/0008-5472.CAN-07-2377&)

31. Conne B, Stutz A, Vassalli JD. The 3′‐untranslated region of messenger RNA: a molecular ‘hotspot’ for pathology? Nat Med 2000; 6: 637–41.  [DOI](https://doi.org/10.1038/76211) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10835679/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Med&title=The%203%E2%80%B2%E2%80%90untranslated%20region%20of%20messenger%20RNA:%20a%20molecular%20%E2%80%98hotspot%E2%80%99%20for%20pathology?&author=B%20Conne&author=A%20Stutz&author=JD%20Vassalli&volume=6&publication_year=2000&pages=637-41&pmid=10835679&doi=10.1038/76211&)

32. Citores MJ, Rua‐Figueroa I, Rodriguez‐Gallego C et al . The dinucleotide repeat polymorphism in the 3′UTR of the CD154 gene has a functional role on protein expression and is associated with systemic lupus erythematosus. Ann Rheum Dis 2004; 63: 310–17.  [DOI](https://doi.org/10.1136/ard.2003.006148) | [PMC free article](/articles/PMC1754911/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14962968/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Rheum%20Dis&title=The%20dinucleotide%20repeat%20polymorphism%20in%20the%203%E2%80%B2UTR%20of%20the%20CD154%20gene%20has%20a%20functional%20role%20on%20protein%20expression%20and%20is%20associated%20with%20systemic%20lupus%20erythematosus&author=MJ%20Citores&author=I%20Rua%E2%80%90Figueroa&author=C%20Rodriguez%E2%80%90Gallego&volume=63&publication_year=2004&pages=310-17&pmid=14962968&doi=10.1136/ard.2003.006148&)

33. Ax W, Soldan M, Koch L, Maser E. Development of daunorubicin resistance in tumour cells by induction of carbonyl reduction. Biochem Pharmacol 2000; 59: 293–300.  [DOI](https://doi.org/10.1016/s0006-2952(99)00322-6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10609558/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20Pharmacol&title=Development%20of%20daunorubicin%20resistance%20in%20tumour%20cells%20by%20induction%20of%20carbonyl%20reduction&author=W%20Ax&author=M%20Soldan&author=L%20Koch&author=E.%20Maser&volume=59&publication_year=2000&pages=293-300&pmid=10609558&doi=10.1016/s0006-2952(99)00322-6&)

34. Nadali F, Pourfathollah AA, Alimoghaddam K et al . Multidrug resistance inhibition by antisense oligonucleotide against MDR1/mRNA in P‐glycoprotein expressing leukemic cells. Hematology 2007; 12: 393–401.  [DOI](https://doi.org/10.1080/10245330701283991) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17852455/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hematology&title=Multidrug%20resistance%20inhibition%20by%20antisense%20oligonucleotide%20against%20MDR1/mRNA%20in%20P%E2%80%90glycoprotein%20expressing%20leukemic%20cells&author=F%20Nadali&author=AA%20Pourfathollah&author=K%20Alimoghaddam&volume=12&publication_year=2007&pages=393-401&pmid=17852455&doi=10.1080/10245330701283991&)

35. Lal S, Wong ZW, Sandanaraj E et al . The influence of ABCB1 and ABCG2 polymorphisms on doxorubicin disposition in Asian breast cancer patients. Cancer Sci 2008; 99: 816–23.  [DOI](https://doi.org/10.1111/j.1349-7006.2008.00744.x) | [PMC free article](/articles/PMC11158672/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18377430/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Sci&title=The%20influence%20of%20ABCB1%20and%20ABCG2%20polymorphisms%20on%20doxorubicin%20disposition%20in%20Asian%20breast%20cancer%20patients&author=S%20Lal&author=ZW%20Wong&author=E%20Sandanaraj&volume=99&publication_year=2008&pages=816-23&pmid=18377430&doi=10.1111/j.1349-7006.2008.00744.x&)

36. Gonzalez B, Akman S, Doroshow J, Rivera H, Kaplan WD, Forrest GL. Protection against daunorubicin cytotoxicity by expression of a cloned human carbonyl reductase cDNA in K562 leukemia cells. Cancer Res 1995; 55: 4646–50.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/7553643/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=Protection%20against%20daunorubicin%20cytotoxicity%20by%20expression%20of%20a%20cloned%20human%20carbonyl%20reductase%20cDNA%20in%20K562%20leukemia%20cells&author=B%20Gonzalez&author=S%20Akman&author=J%20Doroshow&author=H%20Rivera&author=WD%20Kaplan&volume=55&publication_year=1995&pages=4646-50&pmid=7553643&)

37. Soldan M, Ax W, Plebuch M, Koch L, Maser E. Cytostatic drug resistance. Role of phase‐I daunorubicin metabolism in cancer cells. Adv Exp Med Biol 1999; 463: 529–38.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/10352729/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Adv%20Exp%20Med%20Biol&title=Cytostatic%20drug%20resistance.%20Role%20of%20phase%E2%80%90I%20daunorubicin%20metabolism%20in%20cancer%20cells&author=M%20Soldan&author=W%20Ax&author=M%20Plebuch&author=L%20Koch&author=E.%20Maser&volume=463&publication_year=1999&pages=529-38&pmid=10352729&)

38. Salvatorelli E, Menna P, Cascegna S et al . Paclitaxel and docetaxel stimulation of doxorubicinol formation in the human heart: implications for cardiotoxicity of doxorubicin‐taxane chemotherapies. J Pharmacol Exp Ther 2006; 318: 424–33.  [DOI](https://doi.org/10.1124/jpet.106.103846) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16614166/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacol%20Exp%20Ther&title=Paclitaxel%20and%20docetaxel%20stimulation%20of%20doxorubicinol%20formation%20in%20the%20human%20heart:%20implications%20for%20cardiotoxicity%20of%20doxorubicin%E2%80%90taxane%20chemotherapies&author=E%20Salvatorelli&author=P%20Menna&author=S%20Cascegna&volume=318&publication_year=2006&pages=424-33&pmid=16614166&doi=10.1124/jpet.106.103846&)

39. Forrest GL, Gonzalez B, Tseng W, Li X, Mann J. Human carbonyl reductase overexpression in the heart advances the development of doxorubicin‐induced cardiotoxicity in transgenic mice. Cancer Res 2000; 60: 5158–64.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/11016643/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=Human%20carbonyl%20reductase%20overexpression%20in%20the%20heart%20advances%20the%20development%20of%20doxorubicin%E2%80%90induced%20cardiotoxicity%20in%20transgenic%20mice&author=GL%20Forrest&author=B%20Gonzalez&author=W%20Tseng&author=X%20Li&author=J.%20Mann&volume=60&publication_year=2000&pages=5158-64&pmid=11016643&)

40. Olson RD, Mushlin PS, Brenner DE et al . Doxorubicin cardiotoxicity may be caused by its metabolite, doxorubicinol. Proc Natl Acad Sci 1988; 85: 3585–9.  [DOI](https://doi.org/10.1073/pnas.85.10.3585) | [PMC free article](/articles/PMC280258/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2897122/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci&title=Doxorubicin%20cardiotoxicity%20may%20be%20caused%20by%20its%20metabolite,%20doxorubicinol&author=RD%20Olson&author=PS%20Mushlin&author=DE%20Brenner&volume=85&publication_year=1988&pages=3585-9&pmid=2897122&doi=10.1073/pnas.85.10.3585&)
