# PCK1 and SLC22A2 gene variants associated with response to metformin treatment in type 2 diabetes

## Metadata
**Authors:** Sophie St-Amour, Laurence Tessier, Janie Harnois, Catherine Allard, Alexandre Lavoie, Philippe Caron, Luigi Bouchard, Patrice Perron, Karine Tremblay
**Journal:** PLOS ONE
**Date:** 2025 Feb 10
**DOI:** [10.1371/journal.pone.0305511](https://doi.org/10.1371/journal.pone.0305511)
**PMID:** 39928707
**PMCID:** PMC11809887
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11809887/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC11809887/pdf/pone.0305511.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC11809887/pdf/pone.0305511.pdf)

## Abstract

Type 2 diabetes (T2D) is a chronic disorder affecting 462 million worldwide, often managed with metformin as first-line treatment. However, metformin’s response varies among individuals, including up to 30% experiencing serious adverse drug reactions (ADRs) and 20-50% inefficacy. These differences may be due to various factors, including pharmacogenetic (PGx) variants. The PGx variants documented so far could affect both the safety and efficacy of metformin, but due to a lack of replication studies, none reached the clinical evidence-level needed to be used as a predictive marker for treatment response. Therefore, this study aims to evaluate the association between the presence of candidate PGx variants and metformin response in T2D subjects. We conducted an association study involving 108 T2D participants currently or previously treated with metformin. A characterization of their therapeutic response was carried out through questionnaires and pharmacological profile reviews. DNA samples were collected during their single visit to perform genotyping of 24 selected candidate PGx variants. Association analyses between candidate PGx variants and metformin response were performed. Among the subjects included in the analyses (n = 84), 25% were non-responders, and 58% experienced ADRs. At the time of study enrollment, 93.9% of non-responders continued to use metformin. The odds of being a non-responder to metformin are 5.6 times higher for homozygous carriers of the alternative allele of a variant within the PCK1 gene (rs4810083) compared to the other genotypes (95% interval confidence [1.9–16.6]). Two variants in perfect linkage disequilibrium within the SLC22A2 gene (rs316019 and rs316009) were associated with increase odds of having ADRs, where homozygous genotype carriers are 7.3 times more likely to have ADRs presentation (95% interval confidence [1.85–29.01]). This study identified associations between PCK1 and SLC22A2 candidate PGx variants and metformin response in T2D treatment. Additional genetic and functional studies are necessary to elucidate the variants’ impact in metformin’s pharmacological mechanisms.

## Introduction

Diabetes, the most common endocrine disorder, affected over 5.7 million individuals in Canada in 2022, including type 1 and type 2 [[1](#pone.0305511.ref001)], and is projected to affect 693 million people worldwide by 2045 [[2](#pone.0305511.ref002)]. This is a significant rise compared to earlier data, and diabetes continues to represent a major public health challenge in Canada. Diabetes costs approximately $30 billion to Canada’s healthcare system annually [[1](#pone.0305511.ref001)]. This chronic disease predisposes affected people to many complications including vascular and kidney diseases, retinopathy and neuropathy [[3](#pone.0305511.ref003)].

Type 2 diabetes (T2D) is the most common form of diabetes accounting for more than 90% of cases [[4](#pone.0305511.ref004)]. T2D is primarily caused by an increased peripheral insulin resistance accompanied by the pancreatic beta cells being unable to produce and secrete sufficient amounts of insulin to maintain blood glucose homeostasis [[3](#pone.0305511.ref003)]. Insulin production and secretion defect as well as resistance hinder glucose uptake and utilization by the effector cells, leading to hyperglycemia [[3](#pone.0305511.ref003)], a condition that can lead to the complications associated with diabetes [[3](#pone.0305511.ref003),[4](#pone.0305511.ref004)]. Environmental and genetic factors appear to modulate the development of T2D [[3](#pone.0305511.ref003),[5](#pone.0305511.ref005)]. In Canada, if HbA_1C_ exceeds the targeted threshold by more than 1.5% after three months of lifestyle modification, metformin is recommended immediately as first-line medication [[6](#pone.0305511.ref006)]. Metformin primarily reduces hepatic gluconeogenesis and, to a lesser extent, hepatic glycogenolysis, while increasing insulin-mediated glucose uptake in skeletal muscles and fat cells [[7](#pone.0305511.ref007)].

The global burden of diabetes continues to rise, affecting millions of individuals worldwide, placing immense strain on healthcare systems, and necessitating effective, individualized treatments [[1](#pone.0305511.ref001),[2](#pone.0305511.ref002)]. A significant challenge in managing diabetes is the variability in patients’ responses to medications like metformin. It is well recognized that interindividual response to medication may vary, sometimes requiring weeks to optimize treatment and control symptoms sometimes leading to adverse drug reactions (ADRs). For instance, metformin users may experience ADRs such as nausea and diarrhea, which affect about 30% of users, with 5% discontinuing treatment due to severe reactions [[7](#pone.0305511.ref007)–[10](#pone.0305511.ref010)]. Metformin users also face a higher risk of vitamin B12 deficiency [[8](#pone.0305511.ref008)] and in rare cases, lactic acidosis occurs, representing the most severe and potentially fatal ADR [[7](#pone.0305511.ref007)]. On the other hand, some individuals may never achieve the expected therapeutic effect, the so-called non-responders (NR). Regarding metformin’s efficacy, 20% to 50% of users are reported as NR [[11](#pone.0305511.ref011),[12](#pone.0305511.ref012)]. NRs are difficult to identify. Current clinical guidelines recommend the addition of a second-line drug without further investigation if glycemic targets are not achieved after three to six months [[12](#pone.0305511.ref012)–[16](#pone.0305511.ref016)]. In this context, identifying metformin NRs may take a few years [[17](#pone.0305511.ref017)]. This approach can lead to prolonged periods of ineffective treatment for NRs, while others may experience ADRs without the expected benefits. Identifying metformin NR or those at risk of ADRs prior to initiation could significantly impact treatment delay, glycemic control, costs and quality of life of people living with diabetes.

Drug response variability, which complicates diabetes management, arises from a complex combination of genetic and lifestyle factors [[18](#pone.0305511.ref018)–[20](#pone.0305511.ref020)], including age, concomitant therapies, drug interactions and genetic variants [[18](#pone.0305511.ref018)]. When assessing metformin response, factors such as body mass index (BMI), medication compliance and dosage account for only part of the response variability [[21](#pone.0305511.ref021)]. Another fraction of this variability may be due to the presence of pharmacogenetic (PGx) variants [[21](#pone.0305511.ref021)]. Although PGx application is in its infancy in T2D, at least 74 PGx variants (distributed in 40 known genes) have been documented to influence metformin efficacy or safety profile [[22](#pone.0305511.ref022),[23](#pone.0305511.ref023)]. The hepatocytes transporters (*SLC22A1*, *SLC22A2*, *SLC47A1* or *SLC2A2*) and kinase proteins involved in gluconeogenesis (*ATM*) [[22](#pone.0305511.ref022),[23](#pone.0305511.ref023)] were the only genes replicated in at least two independent studies. Other associated genes are involved in intracellular signaling (*PRKAA1/2, STK11*) or function as transcription factors (*SP1, PPARGC1A/B, HNF4A, TCF7L2* and *HNF1B*) [[22](#pone.0305511.ref022)–[24](#pone.0305511.ref024)]. Additionally, there are genes associated with energy homeostasis and metabolism (*PEPCK*, *GCK* and *CPA6*), as well as those involved in insulin signaling and sensitivity (*IRS1*, *KCNJ11*, *CSMD1*, *CDKN2A/B*, *CAPN10* and *NBEA*) [[22](#pone.0305511.ref022),[24](#pone.0305511.ref024)–[28](#pone.0305511.ref028)].

In the present study, we aimed to assess the association between candidate PGx variants and metformin response phenotypes in T2D treatment. Our results support that candidate PGx variants should become clinical guideline annotations.

## Results

### Characteristics of the participants

[S1 Table](#pone.0305511.s001) presents the participants’ characteristics related to health, lifestyle, phenotype, and ADRs. Briefly, 84 participants were included in the cohort of which, 25% (n = 21) were NRs and 59.0% (n = 49) have reported experiencing at least one ADR. All variables presented in [S1 Table](#pone.0305511.s001) were tested for potential association with metformin efficacy and safety. Among all tested variables, seven were significantly associated with metformin’s efficacy: musculoskeletal diseases (p-value = 0.004), neurological diseases (p-value = 0.032), gastrointestinal diseases (p-value = 0.021), levels of HbA_1C_ at metformin initiation (p-value = 0.041), levels of HbA_1C_ and fasting glucose one year after metformin initiation (p-value = < 0.0001) and taking another antidiabetic drug at study inclusion (p-value = 0.013). Additionally, four variables were significantly associated with metformin safety: diet score at T2D diagnosis (p-value = 0.033), diet score at study inclusion (p-value = 0.004), physical activity at study inclusion (p-value = 0.041) and mental health issues (p-value = 0.011). Detailed of the entire cohort’s characterisation is also available in Supporting information.

### Genotype analysis and distribution in the cohort

A total of 34 variants were genotyped for all 105 participants who provided DNA samples ([S2 Table](#pone.0305511.s002)). The Agena iPLEX technology was used at the Genome Québec “Centre d’expertises et de services” (CES) (Montréal, Canada) (platform [genomequebec.com](https://genomequebec.com)). Following genotyping, 7 single nucleotide variations (SNVs) were excluded due to their low call rates (<90%) and 3 SNVs due to the presence of only one genotype (no variation). Hardy-Weinberg Equilibrium (HWE) was tested for all variants, and all were found to be at equilibrium after Holm-Bonferroni correction. Two DNA samples were excluded from genetic analyses due to genotyping failures. Therefore, genetic statistical analyses were performed on a total of 82 participants for 24 SNVs (and only 80 participants with safety available and genotyping available) ([S2 Table](#pone.0305511.s002)).

### Pharmacogenetic association with response to metformin

A comparison between responders and NRs was conducted for the selected variants selected for their reported effect on metformin efficacy (n = 14, [S3 Table](#pone.0305511.s003)). Only *PCK1-*rs4810083 was significantly associated with metformin response in a recessive model (adjusted p-value = 0.0346; [Table 1](#pone.0305511.t001)), but not in the dominant model (results presented in the [S4 Table](#pone.0305511.s004)). NRs were predominantly homozygous for the alternative allele (55.0%). In comparison, in responders, only 18.0% were homozygous for the alternative allele. Thus, individuals that are homozygous carriers of the *PCK1-*rs4810083 alternative allele have 5.6 times more risk of being NRs to metformin than the 2 other genotype’s carriers (recessive model: OR = 5.56, IC 95% [1.86–16.63]).

### Table 1. Genotype distribution between metformin’s responders and non-responders.

| Genotypes n (%) | R(n = 61) | NR(n = 21) | p-valuea | Adjusted p-value |
| --- | --- | --- | --- | --- |
| ABBC8 - rs4148609 |  |  |  |  |
| CC | 28 (46.7) | 8 (38.1) | 0.678 | 1.000 |
| CT | 28 (46.7) | 11 (52.4) |   |   |
| TT | 4 (6.7) | 2 (9.5) |   |   |
| Recessive model b |  |  |  |  |
| CC + CT | 56 (93.3) | 19 (90.5) | 0.647 | 1.000 |
| TT | 4 (6.7) | 2 (9.5) |   |   |
| Capn10 - rs3792269 |  |  |  |  |
| AA | 43 (74.1) | 15 (71.4) | 0.834 | 1.000 |
| AG | 14 (24.1) | 6 (28.6) |   |   |
| GG | 1 (1.7) | 0 (0) |   |   |
| Recessive model |   |   |   |   |
| AA + AG | 57 (98.3) | 21 (100.0) | 1.000 | 1.000 |
| GG | 1 (1.7) | 0 (0) |   |   |
| CPA6 - rs2162145 |  |  |  |  |
| TT | 3 (4.9) | 0 (0) | 0.819 | 1.000 |
| TC | 19 (31.1) | 7 (33.3) |   |   |
| CC | 39 (63.9) | 14 (66.7) |   |   |
| Recessive model |   |   |   |   |
| TT + TC | 22 (36.1) | 7 (33.3) | 1.000 | 1.000 |
| CC | 39 (63.9) | 14 (66.7) |   |   |
| CSMD1 - rs2954625 |  |  |  |  |
| CC | 44 (74.6) | 12 (60.0) | 0.361 | 1.000 |
| CT | 13 (22.0) | 7 (35.0) |   |   |
| TT | 2 (3.4) | 1 (5.0) |   |   |
| Recessive model |   |   |   |   |
| CC + CT | 57 (96.6) | 19 (95.0) | 1.000 | 1.000 |
| TT | 2 (3.4) | 1 (5.0) |   |   |
| GCK - rs2908289 |  |  |  |  |
| GG | 43 (70.5) | 17 (81.0) | 0.761 | 1.000 |
| GA | 16 (26.2) | 4 (19.0) |   |   |
| AA | 2 (3.3) | 0 (0) |   |   |
| Recessive model |   |   |   |   |
| GG + GA | 59 (96.7) | 21 (100.0) | 1.000 | 1.000 |
| AA | 2 (3.3) | 0 (0) |   |   |
| HNF1B - rs11868513 |  |  |  |  |
| GG | 43 (70.5) | 16 (76.2) | 0.506 | 1.000 |
| GA | 15 (24.6) | 3 (14.3) |   |   |
| AA | 3 (4.9) | 2 (9.5) |   |   |
| Recessive model |  |  |  |  |
| GG + GA | 58 (95.1) | 19 (90.5) | 0.598 | 1.000 |
| AA | 3 (4.9) | 2 (9.5) |   |   |
| IRS1 - rs1801278 |  |  |  |  |
| CC | 48 (78.7) | 17 (81.0) | 1.000 | 1.000 |
| CT | 12 (19.7) | 4 (19.0) |   |   |
| TT | 1 (1.6) | 0 (0) |   |   |
| Recessive model |  |  |  |  |
| CC + CT | 60 (98.4) | 21 (100.0) | 1.000 | 1.000 |
| TT | 1 (1.6) | 0 (0) |   |   |
| KCNJ11 - rs5219 |   |   |   |   |
| TT | 14 (23.3) | 5 (23.8) | 0.690 | 1.000 |
| CT | 26 (43.3) | 7 (33.3) |   |   |
| CC | 20 (33.3) | 9 (42.9) |   |   |
| Recessive model |   |   |   |   |
| TT + CT | 40 (66.7) | 12 (57.1) | 0.442 | 1.000 |
| CC | 20 (33.3) | 9 (42.9) |   |   |
| KCNJ11 - rs7124355 |  |  |  |  |
| AA | 13 (21.7) | 5 (23.8) | 0.685 | 1.000 |
| AG | 24 (40.0) | 6 (28.6) |   |   |
| GG | 23 (38.3) | 10 (47.6) |   |   |
| Recessive model |  |  |  |  |
| AA + AG | 37 (61.7) | 11 (52.4) | 0.607 | 1.000 |
| GG | 23 (38.3) | 10 (47.6) |   |   |
| NBEA - rs57081354 |  |  |  |  |
| TT | 53 (86.9) | 20 (95.2) | 0.435 | 1.000 |
| TC | 8 (13.1) | 1 (4.8) |   |   |
| CC | 0 (0) | 0 (0) |   |   |
| Recessive model |  |  |  |  |
| TT + TC | 31 (100.0) | 21 (100.0) | 1.000 | 1.000 |
| CC | 0 (0) | 0 (0) |   |   |
| PCK1 - rs4810083 |  |  |  |  |
| TT | 15 (24.6) | 2 (10.0) *1 | 0.008 | 0.104 |
| TC | 35 (57.4) | 7 (35.0) *2 |   |   |
| CC | 11 (18.0) | 11 (55.0) *1,2 |   |   |
| Recessive model |  |  |  |  |
| TT + TC | 50 (82.0) | 9 (45.0) | 0.003 | 0.039 |
| CC | 11 (18.0) | 11 (55.0) |   |   |
| PPARGC1A - rs10213440 |  |  |  |  |
| TT | 39 (65.0) | 13 (65.0) | 1.000 | 1.000 |
| TC | 19 (31.7) | 7 (35.0) |   |   |
| CC | 2 (3.3) | 0 (0) |   |   |
| Recessive model |  |  |  |  |
| TT + TC | 58 (96.7) | 20 (100.0) | 1.000 | 1.000 |
| CC | 2 (3.3) | 0 (0) |   |   |
| STK11 - rs741765 |  |  |  |  |
| CC | 35 (58.3) | 17 (81.0) | 0.193 | 1.000 |
| CT | 23 (38.3) | 4 (19.0) |   |   |
| TT | 2 (3.3) | 0 (0) |   |   |
| Recessive model |  |  |  |  |
| CC + CT | 58 (96.7) | 21 (100.0) | 1.000 | 1.000 |
| TT | 2 (3.3) | 0 (0) |   |   |
| TCF7L2 - rs2908289 |  |  |  |  |
| CC | 24 (40.0) | 7 (35.0) | 0.940 | 1.000 |
| CT | 26 (43.3) | 10 (50.0) |   |   |
| TT | 10 (16.7) | 3 (15.0) |   |   |
| Recessive model |  |  |  |  |
| CC + CT | 50 (83.3) | 17 (85.0) | 1.000 | 1.000 |
| TT | 10 (16.7) | 3 (15.0) |   |   |

Table 1 Caption: Abbreviations: R, responders; NR, non-responders; ABCC8, ATP-binding cassette transporter sub-family C member 8; CAPN10, Calpain 10; CPA6, Carboxypeptidase A6; CSMD1, CUB and sushi multiple domains 1; GCK, Glucokinase; HNF1B, Hepatocyte nuclear factor 1; IRS1, Insulin receptor substrate 1; KCNJ11, Potassium inwardly rectifying channel subfamily J member 11; NBEA, Neurobeachin; PCK1, Phosphoenolpyruvate carboxykinase 1; PPARGC1A, Peroxisome proliferator-activated receptor gamma coactivator 1-alpha; STK11, Serine/threonine kinase 11; TCF7L2, Transcription factor 7-like 2. aP-value of Fisher exact test comparing Rs to NRs. Bold numbers indicate significance (p < 0.05) and asterisks (*) indicate significance for post-hoc using Holm–Bonferroni adjustment. bDominant model results presented in supplement.

### Variants associated with safety to metformin

All PGx variants involved in metformin’s pharmacokinetic mechanisms (n = 9, [S3 Table](#pone.0305511.s003)) were analyzed for association with ADRs occurrence using the three genetic models. *SLC22A2-*rs316019 was associated with presence of ADRs (codominant and recessive adjusted p-values = 0.024 and 0.024, respectively, [Table 2](#pone.0305511.t002)). Dominant model couldn’t be tested because of the absence of homozygous for the reference allele in both safety groups ([S5 Table](#pone.0305511.s005)). Among individuals without ADR, there were significantly more heterozygous (33.3%) than among those presenting with ADRs (6.4% of heterozygous) (p-value = 0.024). An odd ratio of 7.3 was found for homozygous carriers of *SLC22A2-*rs316019 alternative allele, which suggests a seven fold increase in odds of having ADRs (95% interval confidence [1.85–29.01]) when compared to individuals carrying a reference allele.

### Table 2. Genotype distribution between metformin’s safety phenotypes in the studied cohort.

| Genotypes n (%) | ADRs(n = 47) | No-ADR(n = 33) | p-valuea | Adjusted p-value |
| --- | --- | --- | --- | --- |
| SLC22A1 - rs594709 |  |  |  |  |
| GG | 6 (13.0) | 2 (6.1) | 0.637 | 1.000 |
| GA | 21 (45.7) | 18 (54.5) |   |   |
| AA | 19 (41.3) | 13 (39.4) |   |   |
| Recessive model b |  |  |  |  |
| GG + GA | 27 (58.7) | 20 (30.6) | 1.000 | 1.000 |
| AA | 19 (41.3) | 13 (39.4) |   |   |
| SLC22A1 - rs1867351 |  |  |  |  |
| TT | 25 (53.2) | 18 (54.5) | 1.000 | 1.000 |
| TC | 20 (42.6) | 14 (42.4) |   |   |
| CC | 2 (4.3) | 1 (3.0) |   |   |
| Recessive model |  |  |  |  |
| TT + TC | 27 (58.7) | 20 (30.6) | 1.000 | 1.000 |
| CC | 19 (41.3) | 13 (39.4) |   |   |
| SLC22A1 - rs12208357 |  |  |  |  |
| CC | 44 (93.6) | 30 (90.9) | 0.687 | 1.000 |
| CT | 3 (6.4) | 3 (9.1) |   |   |
| TT | 0 (0) | 0 (0) |   |   |
| Recessive model |  |  |  |  |
| CC + CT | 47 (100.0) | 33 (100.0) | 1.000 | 1.000 |
| TT | 0 (0.0) | 0 (0.0) |   |   |
| SLC22A2 - rs316019/rs316009 |  |  |  |  |
| AA/TT | 0 (0) | 0 (0) | 0.003 | 0.024 |
| AC/TC | 3 (6.4) | 11 (33.3) |   |   |
| CC/CC | 44 (93.6) | 22 (66.7) |   |   |
| Recessive model |  |  |  |  |
| AA/TT + AC/TC | 3 (6.4) | 11 (33.3) | 0.003 | 0.024 |
| CC/CC | 44 (93.6) | 22 (66.7) |   |   |
| SLC22A3 - rs2076828 |  |  |  |  |
| CC | 10 (21.3) | 16 (48.5) * | 0.033 | 0.264 |
| CG | 30 (63.8) | 13 (39.4) * |   |   |
| GG | 7 (14.9) | 4 (12.1) |   |   |
| Recessive model |  |  |  |  |
| CC + CG | 40 (85.1) | 29 (87.9) | 1.000 | 1.000 |
| GG | 7 (14.9) | 4 (12.1) |   |   |
| SLC47A1 - rs8065082 |  |  |  |  |
| CC | 21 (44.7) | 12 (36.4) | 0.485 | 1.000 |
| CT | 19 (40.4) | 18 (54.5) |   |   |
| TT | 7 (14.9) | 3 (9.1) |   |   |
| Recessive model |  |  |  |  |
| CC + CT | 40 (85.1) | 30 (90.9) | 0.512 | 1.000 |
| TT | 7 (14.9) | 3 (9.1) |   |   |
| SLC47A1 - rs2289669 |  |  |  |  |
| GG | 24 (51.1) | 13 (39.4) | 0.389 | 1.000 |
| GA | 18 (38.3) | 18 (54.5) |   |   |
| AA | 5 (10.6) | 2 (6.1) |   |   |
| Recessive model |   |   |   |   |
| GG + GA | 42 (90.9) | 31 (93.9) | 0.694 | 1.000 |
| AA | 5 (10.6) | 2 (6.1) |   |   |
| SLC47A2 - rs12943590 |  |  |  |  |
| GG | 23 (52.3) | 15 (45.5) | 0.682 | 1.000 |
| GA | 17 (38.6) | 16 (48.5) |   |   |
| AA | 4 (9.1) | 2 (6.1) |   |   |
| Recessive model |  |  |  |  |
| GG + GA | 40 (90.9) | 31 (93.9) | 0.694 | 1.000 |
| AA | 4 (9.1) | 2 (6.1) |   |   |
| SLC47A2 - rs34834489 |  |  |  |  |
| GG | 19 (43.2) | 9 (29.0) | 0.180 | 1.000 |
| GA | 16 (36.4) | 18 (58.1) |   |   |
| AA | 9 (20.5) | 4 (12.9) |   |   |
| Recessive model |  |  |  |  |
| GG + GA | 35 (79.5) | 27 (87.1) | 0.539 | 1.000 |
| AA | 9 (20.5) | 4 (12.9) |   |   |

Table 2 Caption: Abbreviations: ADR, adverse drug reaction; SLC22A1, Solute carrier family 22 member 1; SLC22A2, Solute carrier family 22 member 2; SLC22A3, Solute carrier family 22 member 3; SLC47A1, Solute carrier family 47 member 1; SLC47A2, Solute carrier family 47 member 2; STK11, Serine/threonine kinase 11; TCF7L2, Transcription factor 7-like 2. aP-value of Fisher exact test comparing ADRs to No-ADR. Bold numbers indicate significance (p < 0.05) and asterisks (*) indicate significance for post-hoc using Holm–Bonferroni adjustment. bDominant model results presented in supplement.

## Discussion

Our study aimed to assess the association between previously associated PGx variants and the efficacy and safety of metformin as part of the treatment of T2D. A cohort of 84 participants was recruited and deeply phenotyped allowing us to find positive associations with *PCK1*-rs4810083 and *SLC22A2*-rs316019 and metformin response phenotypes.

Out of the 14 variants that were analyzed for metformin efficacy, *PCK1*-rs4810083 was associated with nonresponse in a recessive model. NRs were predominantly homozygous for the alternative allele (C), making them 5.6 times more likely to be an NR compared to those homozygous for the reference allele (T) or heterozygous. This variant was previously linked to metformin efficacy. Tkáč et al. did not find a significant association between the two alleles and metformin efficacy (HbA_1C_ < 7%) at 6 months [[29](#pone.0305511.ref029)]. This variant was also associated with metformin efficacy in the Diabetes Prevention Program where the allele (T) was associated with a lower diabetes incidence under metformin treatment which is in line with our findings [[30](#pone.0305511.ref030)].

The *PCK1* gene encodes the phosphoenolpyruvate carboxykinae 1 (PEPCK) enzyme, and the rs4810083 variant is located 2 kb upstream of this gene [[31](#pone.0305511.ref031)]. PEPCK is a crucial enzyme for gluconeogenesis, primarily occurring in the liver. It catalyzes the conversion of oxaloacetate into phosphoenolpyruvate while consuming a guanosine triphosphate [[32](#pone.0305511.ref032)]. Literature reports that *PCK1* can be underexpressed and overexpressed which results in different clinical conditions. For example, *PCK1* deficiency, although rare, manifests as hypoglycemia, lactic acidosis, liver failure, and excessive excretion of tricarboxylic acid cycle metabolites in the urine [[33](#pone.0305511.ref033)]. In contrast, overexpression of PEPCK could lead to phenotypes such as obesity, lipodystrophy, fatty liver, and T2D [[32](#pone.0305511.ref032)]. The mechanisms underlying these phenotypes are not yet fully understood, and the same applies to the rs4810083 variant. Depending on the variant’s effect on PEPCK expression, it is possible that rs4810083 (T/T) may reduce PEPCK expression, leading to decreased gluconeogenesis. In such cases, individuals would already have a dysregulated metabolic pathway that metformin could not further mitigate, resulting in nonresponse to the medication. Furthermore, if rs4810083 (T/T) activates PEPCK expression and the gluconeogenesis pathway, it may lead to persistent hyperglycemia despite metformin treatment, as gluconeogenesis remains active. Functional studies are necessary to elucidate the cellular role of *PCK1-*rs4810083 in gluconeogenesis, T2D, and metformin response.

Out of the 9 variants evaluated for metformin safety, two of them, rs316019 and 316009, in the same gene *SLC22A2* were associated with presence of ADRs related to metformin, and no homozygotes for the reference allele (A/A) was observed. These two variants are in perfect linkage disequilibrium, which was observed in this cohort with a genotype pattern identical between both variants. Participants who were homozygous for the alternative allele for rs316019 (C/C) or rs316009 (C/C) reported more ADRs, indicating that being homozygous for rs316019 (C/C) or rs316009 (C/C) increases the odds of having ADRs by 7.33 times. Mixed results were reported regarding the impact of these genotypes on response phenotypes and safety profile related to metformin use. Some findings and associations have reported the impact of the variants on metformin efficacy and not ADRs. Abrahams-October et al. found that the genotype for rs316009 (C/T) may significantly improve metformin response compared to (C/C). The same trend was observed for the rs316019 with the genotype (C/A) but was not significant after correction [[34](#pone.0305511.ref034)]. On the other hand, regarding pharmacokinetics, Chen and colleagues found in healthy individuals that rs316019 heterozygous (C/A) have a greater renal clearance of metformin than homozygous for the alternative allele (C) [[35](#pone.0305511.ref035)]. Some research groups have supported this statement, while others concluded the opposite [[36](#pone.0305511.ref036),[37](#pone.0305511.ref037)].

Both variants are part of the *SLC22A*2 gene, which encodes for the organic cation transporter 2 (OCT2), responsible for transporting organic cations, such as metformin, from the bloodstream into the renal tubular cells, primarily in the proximal tubules of the nephron [[38](#pone.0305511.ref038)]. The most well-known variant in *SLC22A2* is the associated rs316019, a missense mutation that results in the change of an alanine for a serine at position 270 of the protein [[39](#pone.0305511.ref039)]. This modification appears to impact the transporter’s affinity for metformin and its capacity to clear metformin from the bloodstream into the renal tubule, resulting in an elevated metformin plasma concentration [[10](#pone.0305511.ref010)]. The other *SLC22A2-*associated variant found, rs316009, is located in a transcription factor binding site [[40](#pone.0305511.ref040)]. No functional study has been conducted to understand the effects of this variant on OCT2 expression and function. However, due to its location, it is possible that this variant lowers the expression of the transporter by limiting access to the transcription factor binding site, resulting in lower excretion of metformin and plasma accumulation, which could explain the increase in ADRs for the individuals homozygous for the alternative allele (C) [[34](#pone.0305511.ref034),[41](#pone.0305511.ref041)]. An increase in the serum concentration of metformin resulting from a decrease in renal excretion could prolong metformin’s effect, such as inhibiting neoglucogenesis and increasing lactate production, which could increase the possibility of ADRs such as lactic acidosis [[37](#pone.0305511.ref037)]. In summary, regarding safety, *SLC22A2-*rs316019 and *SLC22A2-*316009 may exert a protective effect against ADRs when individuals are heterozygote. Both *SLC22A2*-associated variants may influence the expression and conformation of the transporters, resulting in lower expression of the transporter and a decrease in its affinity to metformin, therefore limiting its elimination. Such impacts may be responsible for the higher proportion of ADRs observed in individuals homozygous for the alternative allele of both variants, but the exact mechanisms of the functional consequences remain to be further investigated. OCTs play a critical role in metformin’s pharmacokinetics, and accurately characterizing variants in these transporters could help predict ADRs in individuals using metformin for T2D.

To better understand the factors involved in metformin response variability and to characterize the participants, we also evaluated co-variables that could influence metformin efficacy and safety [[42](#pone.0305511.ref042)] . Associations were found, such as the impact of musculoskeletal, neurological, and gastrointestinal disease on metformin efficacy, and diet, physical activity and mental health issues on its safety profile, but further research is needed. These co-variables, which may modify metformin response, were not included in the genetic analyses as it was not the aim of the study.

A notable finding and strength of this study is the association found between variants and two phenotypes of metformin response evaluated on each cohort’s participant, literature reporting either one or the other. Interviews with participants during visits also provided valuable insights into the complexity of the disease and co-variables influencing metformin response, rarely available in reported studies. Nonetheless, reliance on self-reported dietary habits and lifestyle factors may introduce biases and inaccuracies, which present a limitation. Additionally, certain variables required probing participants’ memory particularly concerning information regarding metformin’s initiation. To control this recall bias, participants were asked to assess their memory reliability when providing information, which helped in evaluating the reliability of the reported information. Also, data collected in medical and pharmacological records served to corroborate the information given by participants and to collect exact lab results to define efficacy phenotypes. Another study limitation is the relatively small size of the cohort for a pharmacogenetics study investigating multiple genetic variants. This cohort size limits the statistical power of the analyses, thereby constraining the ability to detect smaller effect sizes and reducing the generalizability of the findings to a broader population. In addition, the homogeneity of the study population, which is predominantly of European descent, further limits the generalizability of the finding to more diverse populations. To address the small sample size, we controlled by a stringent and targeted candidate variants selection and deep and well performed phenotyping allowing us to find significant associations for two important genes. These findings replicate some associations previously observed in literature, but not all. Further genetic and functional studies with larger and more genetically diverse cohorts are warranted to elucidate the PGx markers of metformin response that would could contribute to optimizing T2D treatment.

## Conclusion

This study reports significant associations between *PCK1* and *SLC22A2* candidate PGx variants and metformin response phenotypes in the treatment of T2D. Additional genetic and functional studies are necessary to elucidate the impact of the associated variants in metformin’s pharmacological and biological mechanisms. Understanding the PGx determinants of metformin response is crucial not only for elucidating its mechanism but also for providing physicians with tools to treat patients more effectively while reducing healthcare costs associated with prescribing inefficient medication. In fact, PGx holds the promise to help optimize patients’ glycemic control and avoid ADRs, thereby reducing their risk of disease progression and occurrence of long-term complications limiting their quality of life.

## Methods

### Study population

This observational retrospective, multicentric, descriptive and analytical case-control pilot study was conducted at “Centre intégré universitaire de santé et de services sociaux du Saguenay–Lac-Saint-Jean” (CIUSSS-SLSJ) research center and at “Centre de recherche du centre hospitalier universitaire de Sherbrooke” (CRCHUS). Adult volunteers included in this study were adults and volunteers. They had a diagnosis of T2D confirmed by a HbA_1C_ between 6.5 and 8.5% at diagnosis. The participants were metformin users for at least one year or used metformin in the last 10 years. All participants allowed access to medical and pharmacological records. Ethical approval was obtained from the Research ethics Committee of the CIUSSS-SLSJ (#2019-045, MP-25-2020-229) and from the Research Ethics Committee of CRCHUS (#MEO-25-2023-4759). The consent obtain was written.

### Recruitment process

Participants were referred by treating physicians or recruited through advertising leaflets distributed in medical clinics, pharmacies, or on social media. Volunteers were contacted to explain the research project and confirm their willingness to participate by delegated team members (SSA, LT, JH). When interested, they were invited to come to one of the recruiting sites for a one-hour visit. Those for whom moving was a limiting factor, a phone appointment was conducted (n = 98). Upon obtaining free and signed informed consent, data collection and sampling of biospecimens (either 10 mL of blood (n = 101) or 4 ml of saliva (n = 4)) were performed and integrated to the principal investigator’s (KT) biobank for future research purposes.

A total of 155 men and women were contacted by the research team, and all were provided with project details and criteria validation. Among them, 108 were found eligible and 105 provided DNA samples. Out of these 105 participants, 21 were excluded from genetic analyses for not meeting the inclusion criteria (having HbA_1C_ under 8,5% at metformin initiation or starting therapy with only metformin) after medical records verification. [Fig 1](#pone.0305511.g001) summarises the study participants selection flowchart.

### Fig 1. Trajectory and criteria of the participants’ selection.

![Fig 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fe65/11809887/bcf687da1b25/pone.0305511.g001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=11809887_pone.0305511.g001.jpg)

155 patients were contacted by the research team and 108 patients were recruited. Our of the 108 participants, 3 did not give DNA samples du to lost to follow up. Following medical chart review, it was found that 21 of the recruited patients had either missing data to properly phenotype or didn’t meet the criteria of glycemic value at metformin initiation and they were therefore excluded from analyses. Out of the 84 patients of gave DNA samples, two participants were excluded from genotype-phenotype analyses du to sequencing difficulties. Therefore, 82 participants were included for the genotype-phenotype analyses.

### Data collection

During their visit, participant self-reported information about demographics, metformin, lifestyle habits, other comorbidities, compliance with dosage instruction, or the presence of ADRs to metformin. The diet was assessed through a self-reported diet score ranging from 1 to 10 established by our research team. Participants were asked to assign themselves a score based on their perception of their diet management at their T2D diagnosis and at study inclusion. Physical activity was also reported by participants and recorded at the same two timepoints. Participants were considered active if they engaged in at least 150 minutes of moderate activity or 75 minutes of intense activity per week, following the World Health Organization (WHO) criteria [[22](#pone.0305511.ref022)]. All self-reported data was assessed for participant’s memory and data accuracy. To ensure data quality, participants were asked to evaluate their memory, ranging from “I perfectly remember” to “I don’t remember at all”. If the participant didn’t remember the information perfectly, the data was excluded from the analysis. Laboratory data (HbA_1C_, fasting glucose, insulin, creatinine, and the estimated glomerular filtration rate (eGFR, calculated using the CKD-EPI equation)) was collected by reviewing participants’ medical records a posteriori [[10](#pone.0305511.ref010)]. Ten percent of the laboratory data entry were validated by independent team members (JH, LT) to ensure data quality (discrepancies found were lower than 4%). eGFR after the initiation of metformin was collected to assess dosage changes related to renal impairment. ADRs were mostly self-reported but some were verified in the medical chart if available. Participants also granted access to their archived pharmacological records through a request sent to their pharmacy to collect the concomitant medications at the beginning of metformin treatment. Study data were collected and managed using REDCap electronic data capture tools hosted at Université de Sherbrooke [[24](#pone.0305511.ref024),[25](#pone.0305511.ref025)].

### Phenotyping

The data collected was used to define each participant’s response phenotypes related to efficacy (responder (R) or NR) and safety (ADRs or absence of ADR). The characterization of metformin efficacy involved assessing glycemic control based on serum blood glucose markers that either did or did not meet the targets set by the Canadian consensus on the treatment of T2D in the latest revised version (2018) [[26](#pone.0305511.ref026)]. To ensure accuracy, the endocrinologist (PP) validated the criteria used to characterize the phenotypes. ADRs were self-reported by participants and, when available, by their treating physician or nurse. The main biomarker targeted was HbA_1C_, along with fasting glucose (8 hours) when available. In instances of discrepancies between these two biomarkers, HbA_1C_ was prioritized.

In this study, response to metformin was defined as achieving adequate blood glucose control after 1 year of treatment with metformin. A responder was defined as having an HbA_1C_ equal to or less than 7% (6.5% being rarely achieved, which would be too restrictive as a criterion) or fasting glucose equal to or below 7 mmol/L after 1 year of treatment with metformin only. On the other hand, a participant was classified as a NR if HbA_1C_ was higher than 7% or fasting glucose levels exceeded 7 mmol/L. Among other parameters used to define a NR, metformin discontinuation during the first year of therapy due to inefficacy or the presence of serious ADRs served as a discriminating factor. Additionally, receiving a second or a third hypoglycemic medication within the first year was considered indicative of non-response.

### Selection of PGx variants

A literature review was conducted on PubMed^®^ and PharmGKB [[43](#pone.0305511.ref043)] to produce a list of PGx variants reported to be associated with metformin efficacy and ADRs (n = 73, [S3 Table](#pone.0305511.s003)). Among them, 34 were selected for genotyping based on their reported minor allelic frequency (greater than 5% on NCBI - presented in [Table 3](#pone.0305511.t003)). Variants were excluded if they lacked a locus or if they were triallelic. Some variants were also excluded from genotyping due to their localization in repeated regions or due to the presence of many neighboring variants, which posed a high risk of unsuccessful genotyping.

### Table 3. Candidate variants for metformin response in T2D treatment and in T2D disease.

| GeneSymbol[GeneID]Locus | Variants -SNVa | Alleles(REF>ALT)b | Allele frequency [allele]c | SNV type [amino acids] | Phenotype of response | Ethnicity (Number) |
| --- | --- | --- | --- | --- | --- | --- |
| ABCC8 [6833] 11p15.1 | rs4148609 | C > T | 0.34 [T] | Intron | Efficacy | N/A |
| CAPN10 [11132] 2q37.3 | rs3792269 | A > G | 0.14 [G] | Synonymous [Pre200Pre] | Efficacy | N/A (144) |
| CDKN2 [1029] 9p21.3 | rs10811661 | T > C | 0.16 [C] | N/A | Efficacy | N/A |
| CPA6 [57094] 8q13.2 | rs2162145 | T > C | 0.41 [C] | 2KB Upstream | Efficacy | CAUC (845) |
| CSMD1 [64478] 8p23.2 | rs2954625 | C > T | 0.27 [T] | Intron | Efficacy | MG (1056) |
| GCK [2645] 7p13 | rs2908289 | G > A | 0.21 [A] | Intron | Efficacy | N/A |
| HFN1B [6928] 17q12 | rs11868513 | G > A | 0.16 [A] | Intron | Efficacy | N/A (927) |
| HNFF4A [3172] 20q13.12 | rs11086926 | T > G | 0.13 [G] | 3’UTR | Efficacy | MG (927) |
| IRS1 [3667] 2q36.3 | rs1801278 | C > T | 0.05 [T] | Missense [Gly972Arg] | Efficacy | N/A |
| KCNJ11 [3767] 11p15.1 | rs5219 | T > C | 0.36 [C] | Missense | Efficacy | CAUC (317) |
|   | rs7124355 | A > G | 0.26 [G] | N/A | Efficacy | N/A |
| MEF2A [4205] 15q26.3 | rs424892 | C > T | 0.25 [T] | Intron | Efficacy | N/A |
| NBEA [26960] 13q13.3 | rs57081354 | T > C | 0.08 [C] | Intron | Efficacy | MG (1312) |
| PCK1 [5105] 20q13.31 | rs4810083 | T > C | 0.45 [C] | 2KB Upstream | Efficacy | N/A (148) |
| PPARGC1A [10891] 4p15.2 | rs10213440 | T > C | 0.21 [C] | Intron | Efficacy | N/A (148) |
| PPARGC1B [133522] 5q32 | rs741579 | A > G | 0.25 [G] | N/A | Efficacy | N/A |
| PRKAA1 [5562] 5p13.1 | rs249429 | C > T | 0.29 [T] | Intron | Efficacy | N/A (144) |
| PRKAA2 [108079] 1p32.2 | r9803799 | T > G | 0.11 [G] | Non-coding | Efficacy | N/A |
| SLC22A1 [6580] 6q25.3 | rs594709 | G > A | 0.35 [A] | Intron | Safety | Asian (53) |
|   | rs1867351 | T > C | 0.25 [C] | Synonymous [Ser52Ser] | Safety | MG (106) |
|   | rs12208357 | C > T | 0.05 [T] | Missense [Arg61Cys] | Safety | MG (106); unknown (12); CAUC (208) (4557) |
| SLC22A2 [6582] 6q25.3 | rs316019 | A > C | 0.10 [C] | Missense [Ala270Ser] | Safety | MG (106) (23) European (50)(5224)(148); Asian (96); N/A (34) |
|   | rs3119309 | C > T | 0.09 [T] | Intron | Safety | N/A |
|   | rs316009 | T > C | 0.09 [C] | Intron | Safety | MG (1056) |
| SLC22A3 [6581] 6q25.3 | rs2076828 | C > G | 0.43 [G] | Non-coding | Safety | N/A (57) |
| SLC2A2 [20526] 3q26.2 | rs8192675 | T > C | 0.42 [C] | Intron | Safety | CAUC (10577) |
| SLC47A1 [55244] 17p11.2 | rs8065082 | C > T | 0.43 [T] | Intron | Safety | MG (990) |
|   | rs2252281 | T > C | 0.37 [C] | 5’ UTR | Safety | CAUC (2651) (50); MG (57) (145) (106); N/A (34) |
|   | rs2289669 | G > A | 0.32 [A] | Intron | Safety | Asian (220) (53); CAUC (148) (331) (5205) (50); N/A (34); MG (23) (106) |
| SLC47A2 [146802] 17p11.2 | rs12943590 | G > A | 0.28 [A] | Intron | Safety | Asian (98) (12)(12)(96); CAUC (189) (40); MG (32) (57) (106); N/A (34) |
|   | rs34834489 | G > A | 0.27 [A] | Upstream gene | Safety | Asian (12) |
| SP1 [6667] 12q13.13 | rs784888 | G > C | 0.11 [C] | Intron | Safety | MG (440) (57) (106) |
| STK11 [6794] 19p13.3 | rs741765 | C > T | 0.24 [T] | Intron | Efficacy | N/A |
| TCF7L2 [6934] 10q25.2-q25.3 | rs7903146 | C > T | 0.27 [T] | Intron | Efficacy | MG (608) |

Table 3 Caption: Abbreviations: CAUC, Caucasian; HbA1C, glycated hemoglobin; MG, multiple group; N/A, not available or unknown. aData from GRCh38/hg38 version; http://ncbi.nlm.nih.gov/snp/. bThe reference allele and the alternative allele are based on the National Center of Biotechnology Information (NCBI) classification. cFrom ALFA project.

### Genotyping

Among the 108 participants, 105 provided biospecimens for DNA genotyping. Blood aliquots were prepared for DNA isolation using Qiagen Puregene kit (Hilden, Germany) following standardized laboratory procedures for genetic analysis. Saliva sampling was conducted using the GenoTech^®^ saliva sample collection kit OG-500 (Ontario, Canada) and sent through postal mail collection according to the manufacturer’ instructions. DNA was extracted from saliva using the reagent prepIT.L2P from DNA GenoTech^®^ (Ontario, Canada). The genotyping technology utilized was the Mass Array System iPlex Assay multiplex genotyping, conducted on the Agena Bioscience platform (San Diego, USA). The Genome Québec “Centre d’expertises et de services” (CES) (Montréal, Canada) is equipped with this platform and operates following standardized laboratory procedures. DNA samples were shipped after isolation according to CES quality standards.

### Statistical analyses

Descriptive analyses were conducted to characterize the cohort. For dichotomous and categorical variables, Fisher’s exact tests were used to compared responders to NRs and presence or absence of ADRs. For continuous variables, the Wilcoxon rank-sum test was used for comparison between groups (responders vs NR and presence vs absence of ADRs). Potential relationships (p < 0.05) were evaluated with *post hoc* pairwise 2x2 Fisher tests, followed by Holm-Bonferroni correction. A Hardy-Weinberg equilibrium exact test was performed on each genetic variant. Variants were classified for their role in metformin mechanism. All variants regarding metformin pharmacodynamics and T2D development were put together and analyzed for the efficacy phenotypes of metformin. On the other hand, if a gene had a variant that was responsible for metformin pharmacokinetics, such as transporters, this gene was automatically analyzed for the safety profile of metformin. If a variant was previously associated with efficacy and safety to metformin, safety was prioritized. This resulted in 14 variants analysed for phenotype response and 9 variants analyzed for safety. Three genetics models were tested for each variant: codominant (AA vs AB vs BB), dominant (AA vs. AB/BB), and recessive (AA/AB vs. BB). Multiple testing corrections were performed according to the method proposed by Li, J. & Ji, L. (2005) for adjusting multilocus analyses by calculating the effective number of variants analyzed [[28](#pone.0305511.ref028)]. Specifically, Bonferroni correction for genetic analyses of metformin’s efficacy was conducted for an effective number of 13 variants, and Bonferroni correction for metformin’s safety was performed for an effective number of 8 variants. If the p-value after correction was < 0.05, the result was considered significant. For significant associations (adjusted p-value < 0.05), a logistic regression model was conducted to quantify the effect by an odds ratio along with a Wald 95% confidence interval.

## Supporting information

## Acknowledgments

We would like to thank all participants, physicians and pharmacists dedicated to this study, as well as the research team members for their involvement. The research team would like to thank the nurses from both research centers for their involvement in collecting blood samples. Specifically, the research team would like to thank Ann-Lorie Gagnon, Mohamad Issa, Elliot Fortin, Philippe Mercier, Mathilde Côté, Raudel Simon Ladron de Guevera, Ling Jing Li, Xuanning Yang and Émy Labonté for their participation in data entry, collection, and validation.

## Abbreviations

## Data Availability

The dataset used to produce the results of this study is under the principal investigator’s biobank framework access policies, supervised by the local ethic committee and compliant with the Quebec Government law on access to health data. As per consent form, no participant has signed to make their data publicly available. Any access to data or samples biobanked must be requested using the Nagano project ID [#2019-045, DT2 Metormine] by emailing to the CIUSSS-SLSJ ethic committee at: guichetunique.slsj@ssss.gouv.qc.ca. If any problems are encountered with access requests, please contact the corresponding author who can provide assistance.

## Funding Statement

The authors also acknowledge this study was financially supported by “Diabète Québec” ([www.diabete.qc.ca](https://www.diabete.qc.ca)) (PI : KT, CO-PI: LB, PP, AL), the “Fonds de recherche du Québec en Santé” ([https://frq.gouv.qc.ca/sante/](https://frq.gouv.qc.ca/sante/)) (KT), the Canadian Institutes of Health Research ([https://cihr-irsc.gc.ca/e/193.html](https://cihr-irsc.gc.ca/e/193.html)) (SSA), and the “Faculté de médecine et de sciences de la santé de l’Université de Sherbrooke” ([https://www.usherbrooke.ca/medecine/](https://www.usherbrooke.ca/medecine/)) (KT, SSA). No sponsors or funders play any role in the study design, data collection and analysis, decision to publish, or preparation of the manuscript.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

The dataset used to produce the results of this study is under the principal investigator’s biobank framework access policies, supervised by the local ethic committee and compliant with the Quebec Government law on access to health data. As per consent form, no participant has signed to make their data publicly available. Any access to data or samples biobanked must be requested using the Nagano project ID [#2019-045, DT2 Metormine] by emailing to the CIUSSS-SLSJ ethic committee at: guichetunique.slsj@ssss.gouv.qc.ca. If any problems are encountered with access requests, please contact the corresponding author who can provide assistance.

### Supplementary Materials

### Data Availability Statement

The dataset used to produce the results of this study is under the principal investigator’s biobank framework access policies, supervised by the local ethic committee and compliant with the Quebec Government law on access to health data. As per consent form, no participant has signed to make their data publicly available. Any access to data or samples biobanked must be requested using the Nagano project ID [#2019-045, DT2 Metormine] by emailing to the CIUSSS-SLSJ ethic committee at: guichetunique.slsj@ssss.gouv.qc.ca. If any problems are encountered with access requests, please contact the corresponding author who can provide assistance.

## References

1. Diabetes in Canada: Backgrounder. Ottawa: Diabetes Canada; 2023.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Diabetes%20Canada&title=Backgrounder.%20Ottawa:&publication_year=2023&)

2. Cho NH, Shaw JE, Karuranga S, Huang Y, da Rocha Fernandes JD, Ohlrogge AW, et al. IDF diabetes atlas: global estimates of diabetes prevalence for 2017 and projections for 2045. Diabetes Res Clin Pract. 2018;138:271–81. doi: 10.1016/j.diabres.2018.02.023  [DOI](https://doi.org/10.1016/j.diabres.2018.02.023) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29496507/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Diabetes%20Res%20Clin%20Pract&title=IDF%20diabetes%20atlas:%20global%20estimates%20of%20diabetes%20prevalence%20for%202017%20and%20projections%20for%202045&author=NH%20Cho&author=JE%20Shaw&author=S%20Karuranga&author=Y%20Huang&author=JD%20da%20Rocha%20Fernandes&volume=138&publication_year=2018&pages=271-81&pmid=29496507&doi=10.1016/j.diabres.2018.02.023&)

3. Klandorf HP, Stark SWRNAD. Diabetes mellitus. Salem Press; 2019.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Diabetes%20mellitus&author=HP%20Klandorf&author=SWRNAD%20Stark&publication_year=2019&)

4. Chatterjee S, Khunti K, Davies MJ. Type 2 diabetes. Lancet. 2017;389(10085):2239–51. doi: 10.1016/S0140-6736(17)30058-2  [DOI](https://doi.org/10.1016/S0140-6736(17)30058-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28190580/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Type%202%20diabetes&author=S%20Chatterjee&author=K%20Khunti&author=MJ%20Davies&volume=389&issue=10085&publication_year=2017&pages=2239-51&pmid=28190580&doi=10.1016/S0140-6736(17)30058-2&)

5. Reitman ML, Schadt EE. Pharmacogenetics of metformin response: a step in the path toward personalized medicine. J Clin Invest. 2007;117(5):1226–9. doi: 10.1172/JCI32133  [DOI](https://doi.org/10.1172/JCI32133) | [PMC free article](/articles/PMC1857273/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17476355/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Invest&title=Pharmacogenetics%20of%20metformin%20response:%20a%20step%20in%20the%20path%20toward%20personalized%20medicine&author=ML%20Reitman&author=EE%20Schadt&volume=117&issue=5&publication_year=2007&pages=1226-9&pmid=17476355&doi=10.1172/JCI32133&)

6. Lipscombe L, Booth G, Butalia S, Dasgupta K, Eurich DT, et al. Pharmacologic glycemic management of type 2 diabetes in adults. Can J Diabetes. 2018;42(Suppl 1):S88–103. doi: 10.1016/j.jcjd.2017.10.034  [DOI](https://doi.org/10.1016/j.jcjd.2017.10.034) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29650116/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Can%20J%20Diabetes&title=Pharmacologic%20glycemic%20management%20of%20type%202%20diabetes%20in%20adults&author=L%20Lipscombe&author=G%20Booth&author=S%20Butalia&author=K%20Dasgupta&author=DT%20Eurich&volume=42&issue=Suppl%201&publication_year=2018&pages=S88-103&pmid=29650116&doi=10.1016/j.jcjd.2017.10.034&)

7. Kirpichnikov D, McFarlane SI, Sowers JR. Metformin: an update. Ann Intern Med. 2002;137(1):25–33. doi: 10.7326/0003-4819-137-1-200207020-00009  [DOI](https://doi.org/10.7326/0003-4819-137-1-200207020-00009) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12093242/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Intern%20Med&title=Metformin:%20an%20update&author=D%20Kirpichnikov&author=SI%20McFarlane&author=JR%20Sowers&volume=137&issue=1&publication_year=2002&pages=25-33&pmid=12093242&doi=10.7326/0003-4819-137-1-200207020-00009&)

8. Imran SA, Agarwal G, Bajaj HS, Ross S. Targets for glycemic control. Can J Diabetes. 2018;42(Suppl 1):S42–6. doi: 10.1016/j.jcjd.2017.10.030  [DOI](https://doi.org/10.1016/j.jcjd.2017.10.030) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29650110/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Can%20J%20Diabetes&title=Targets%20for%20glycemic%20control&author=SA%20Imran&author=G%20Agarwal&author=HS%20Bajaj&author=S%20Ross&volume=42&issue=Suppl%201&publication_year=2018&pages=S42-6&pmid=29650110&doi=10.1016/j.jcjd.2017.10.030&)

9. Bailey CJ, Turner RC. Metformin. N Engl J Med. 1996;334(9):574–9. doi: 10.1056/NEJM199602293340906  [DOI](https://doi.org/10.1056/NEJM199602293340906) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8569826/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Metformin&author=CJ%20Bailey&author=RC%20Turner&volume=334&issue=9&publication_year=1996&pages=574-9&pmid=8569826&doi=10.1056/NEJM199602293340906&)

10. Evans WE, McLeod HL. Pharmacogenomics–drug disposition, drug targets, and side effects. N Engl J Med. 2003;348(6):538–49. doi: 10.1056/NEJMra020526  [DOI](https://doi.org/10.1056/NEJMra020526) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12571262/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Pharmacogenomics%E2%80%93drug%20disposition,%20drug%20targets,%20and%20side%20effects&author=WE%20Evans&author=HL%20McLeod&volume=348&issue=6&publication_year=2003&pages=538-49&pmid=12571262&doi=10.1056/NEJMra020526&)

11. Sherifali D. The effect of oral antidiabetic agents on A1C levels: a systematic review and meta-analysis. Diabetes Care. 2010;33(8):1859–64. doi: 10.2337/dc09-1727  [DOI](https://doi.org/10.2337/dc09-1727) | [PMC free article](/articles/PMC2909079/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20484130/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Diabetes%20Care&title=The%20effect%20of%20oral%20antidiabetic%20agents%20on%20A1C%20levels:%20a%20systematic%20review%20and%20meta-analysis&author=D%20Sherifali&volume=33&issue=8&publication_year=2010&pages=1859-64&pmid=20484130&doi=10.2337/dc09-1727&)

12. Staiger H. Pharmacogenetics: implications for modern type 2 diabetes therapy. Rev Diabet Stud. 2015;12(3–4):363–76. doi: 10.1900/RDS.2015.12.363  [DOI](https://doi.org/10.1900/RDS.2015.12.363) | [PMC free article](/articles/PMC5275760/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27111121/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Rev%20Diabet%20Stud&title=Pharmacogenetics:%20implications%20for%20modern%20type%202%20diabetes%20therapy&author=H%20Staiger&volume=12&issue=3%E2%80%934&publication_year=2015&pages=363-76&pmid=27111121&doi=10.1900/RDS.2015.12.363&)

13. Kahn SE, Haffner SM, Heise MA, Herman WH, Holman RR, Jones NP, et al. Glycemic durability of rosiglitazone, metformin, or glyburide monotherapy. N Engl J Med. 2006;355(23):2427–43. doi: 10.1056/NEJMoa066224  [DOI](https://doi.org/10.1056/NEJMoa066224) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17145742/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Glycemic%20durability%20of%20rosiglitazone,%20metformin,%20or%20glyburide%20monotherapy&author=SE%20Kahn&author=SM%20Haffner&author=MA%20Heise&author=WH%20Herman&author=RR%20Holman&volume=355&issue=23&publication_year=2006&pages=2427-43&pmid=17145742&doi=10.1056/NEJMoa066224&)

14. Ordelheide A-M, Hrabě de Angelis M, Häring H-U, Staiger H. Pharmacogenetics of oral antidiabetic therapy. Pharmacogenomics. 2018;19(6):577–87. doi: 10.2217/pgs-2017-0195  [DOI](https://doi.org/10.2217/pgs-2017-0195) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29580198/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Pharmacogenetics%20of%20oral%20antidiabetic%20therapy&author=A-M%20Ordelheide&author=M%20Hrab%C4%9B%20de%20Angelis&author=H-U%20H%C3%A4ring&author=H%20Staiger&volume=19&issue=6&publication_year=2018&pages=577-87&pmid=29580198&doi=10.2217/pgs-2017-0195&)

15. Bloomgarden ZT, Dodis R, Viscoli CM, Holmboe ES, Inzucchi SE. Lower baseline glycemia reduces apparent oral agent glucose-lowering efficacy: a meta-regression analysis. Diabetes Care. 2006;29(9):2137–9. doi: 10.2337/dc06-1120  [DOI](https://doi.org/10.2337/dc06-1120) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16936168/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Diabetes%20Care&title=Lower%20baseline%20glycemia%20reduces%20apparent%20oral%20agent%20glucose-lowering%20efficacy:%20a%20meta-regression%20analysis&author=ZT%20Bloomgarden&author=R%20Dodis&author=CM%20Viscoli&author=ES%20Holmboe&author=SE%20Inzucchi&volume=29&issue=9&publication_year=2006&pages=2137-9&pmid=16936168&doi=10.2337/dc06-1120&)

16. Cook MN, Girman CJ, Stein PP, Alexander CM, Holman RR. Glycemic control continues to deteriorate after sulfonylureas are added to metformin among patients with type 2 diabetes. Diabetes Care. 2005;28(5):995–1000. doi: 10.2337/diacare.28.5.995  [DOI](https://doi.org/10.2337/diacare.28.5.995) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15855556/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Diabetes%20Care&title=Glycemic%20control%20continues%20to%20deteriorate%20after%20sulfonylureas%20are%20added%20to%20metformin%20among%20patients%20with%20type%202%20diabetes&author=MN%20Cook&author=CJ%20Girman&author=PP%20Stein&author=CM%20Alexander&author=RR%20Holman&volume=28&issue=5&publication_year=2005&pages=995-1000&pmid=15855556&doi=10.2337/diacare.28.5.995&)

17. Pasquel FJ, Klein R, Adigweme A, Hinedi Z, Coralli R, Pimentel JL, et al. Metformin-associated lactic acidosis. Am J Med Sci. 2015;349(3):263–7. doi: 10.1097/MAJ.0b013e3182a562b7  [DOI](https://doi.org/10.1097/MAJ.0b013e3182a562b7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24326619/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Med%20Sci&title=Metformin-associated%20lactic%20acidosis&author=FJ%20Pasquel&author=R%20Klein&author=A%20Adigweme&author=Z%20Hinedi&author=R%20Coralli&volume=349&issue=3&publication_year=2015&pages=263-7&pmid=24326619&doi=10.1097/MAJ.0b013e3182a562b7&)

18. Kalow W, Tang BK, Endrenyi L. Hypothesis: comparisons of inter- and intra-individual variations can substitute for twin studies in drug research. Pharmacogenetics. 1998;8(4):283–9. doi: 10.1097/00008571-199808000-00001  [DOI](https://doi.org/10.1097/00008571-199808000-00001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9731714/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Hypothesis:%20comparisons%20of%20inter-%20and%20intra-individual%20variations%20can%20substitute%20for%20twin%20studies%20in%20drug%20research&author=W%20Kalow&author=BK%20Tang&author=L%20Endrenyi&volume=8&issue=4&publication_year=1998&pages=283-9&pmid=9731714&doi=10.1097/00008571-199808000-00001&)

19. Vesell ES. Pharmacogenetic perspectives gained from twin and family studies. Pharmacol Ther. 1989;41(3):535–52. doi: 10.1016/0163-7258(89)90130-7  [DOI](https://doi.org/10.1016/0163-7258(89)90130-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2654965/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacol%20Ther&title=Pharmacogenetic%20perspectives%20gained%20from%20twin%20and%20family%20studies&author=ES%20Vesell&volume=41&issue=3&publication_year=1989&pages=535-52&pmid=2654965&doi=10.1016/0163-7258(89)90130-7&)

20. Zhou K, Donnelly L, Yang J, Li M, Deshmukh H, Van Zuydam N, et al. Heritability of variation in glycaemic response to metformin: a genome-wide complex trait analysis. Lancet Diabetes Endocrinol. 2014;2(6):481–7. doi: 10.1016/S2213-8587(14)70050-6  [DOI](https://doi.org/10.1016/S2213-8587(14)70050-6) | [PMC free article](/articles/PMC4038749/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24731673/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet%20Diabetes%20Endocrinol&title=Heritability%20of%20variation%20in%20glycaemic%20response%20to%20metformin:%20a%20genome-wide%20complex%20trait%20analysis&author=K%20Zhou&author=L%20Donnelly&author=J%20Yang&author=M%20Li&author=H%20Deshmukh&volume=2&issue=6&publication_year=2014&pages=481-7&pmid=24731673&doi=10.1016/S2213-8587(14)70050-6&)

21. Gloyn AL, Drucker DJ. Precision medicine in the management of type 2 diabetes. Lancet Diabetes Endocrinol. 2018;6(11):891–900. doi: 10.1016/S2213-8587(18)30052-4  [DOI](https://doi.org/10.1016/S2213-8587(18)30052-4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29699867/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet%20Diabetes%20Endocrinol&title=Precision%20medicine%20in%20the%20management%20of%20type%202%20diabetes&author=AL%20Gloyn&author=DJ%20Drucker&volume=6&issue=11&publication_year=2018&pages=891-900&pmid=29699867&doi=10.1016/S2213-8587(18)30052-4&)

22. Maruthur NM, Gribble MO, Bennett WL, Bolen S, Wilson LM, Balakrishnan P, et al. The pharmacogenetics of type 2 diabetes: a systematic review. Diabetes Care. 2014;37(3):876–86. doi: 10.2337/dc13-1276  [DOI](https://doi.org/10.2337/dc13-1276) | [PMC free article](/articles/PMC3931386/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24558078/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Diabetes%20Care&title=The%20pharmacogenetics%20of%20type%202%20diabetes:%20a%20systematic%20review&author=NM%20Maruthur&author=MO%20Gribble&author=WL%20Bennett&author=S%20Bolen&author=LM%20Wilson&volume=37&issue=3&publication_year=2014&pages=876-86&pmid=24558078&doi=10.2337/dc13-1276&)

23. Srinivasan S, Kaur V, Chamarthi B, Littleton KR, Chen L, Manning AK, et al. TCF7L2 genetic variation augments incretin resistance and influences response to a sulfonylurea and metformin: the study to understand the genetics of the acute response to metformin and glipizide in humans (SUGAR-MGH). Diabetes Care. 2018. Mar;41(3):554–61. doi: 10.2337/dc17-1386  [DOI](https://doi.org/10.2337/dc17-1386) | [PMC free article](/articles/PMC5829963/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29326107/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Diabetes%20Care&title=TCF7L2%20genetic%20variation%20augments%20incretin%20resistance%20and%20influences%20response%20to%20a%20sulfonylurea%20and%20metformin:%20the%20study%20to%20understand%20the%20genetics%20of%20the%20acute%20response%20to%20metformin%20and%20glipizide%20in%20humans%20(SUGAR-MGH)&author=S%20Srinivasan&author=V%20Kaur&author=B%20Chamarthi&author=KR%20Littleton&author=L%20Chen&volume=41&issue=3&publication_year=2018&pages=554-61&pmid=29326107&doi=10.2337/dc17-1386&)

24. Goswami S, Yee SW, Xu F, Sridhar SB, Mosley JD, Takahashi A, et al. A Longitudinal HbA1c Model Elucidates Genes Linked to Disease Progression on Metformin. Clin Pharmacol Ther. 2016. Nov;100(5):537–47. doi: 10.1002/cpt.428  [DOI](https://doi.org/10.1002/cpt.428) | [PMC free article](/articles/PMC5534241/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27415606/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=A%20Longitudinal%20HbA1c%20Model%20Elucidates%20Genes%20Linked%20to%20Disease%20Progression%20on%20Metformin&author=S%20Goswami&author=SW%20Yee&author=F%20Xu&author=SB%20Sridhar&author=JD%20Mosley&volume=100&issue=5&publication_year=2016&pages=537-47&pmid=27415606&doi=10.1002/cpt.428&)

25. Rotroff DM, Yee SW, Zhou K, Marvel SW, Shah HS, Jack JR, et al. Genetic Variants in CPA6 and PRPF31 Are Associated With Variation in Response to Metformin in Individuals With Type 2 Diabetes. Diabetes. 2018. Jul;67(7):1428–40. doi: 10.2337/db17-1164  [DOI](https://doi.org/10.2337/db17-1164) | [PMC free article](/articles/PMC6014560/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29650774/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Diabetes&title=Genetic%20Variants%20in%20CPA6%20and%20PRPF31%20Are%20Associated%20With%20Variation%20in%20Response%20to%20Metformin%20in%20Individuals%20With%20Type%202%20Diabetes&author=DM%20Rotroff&author=SW%20Yee&author=K%20Zhou&author=SW%20Marvel&author=HS%20Shah&volume=67&issue=7&publication_year=2018&pages=1428-40&pmid=29650774&doi=10.2337/db17-1164&)

26. Florez JC, Jablonski KA, McAteer JB, Franks PW, Mason CC, Mather K, et al. Effects of genetic variants previously associated with fasting glucose and insulin in the Diabetes Prevention Program. PLoS One. 2012;7(9):e44424. doi: 10.1371/journal.pone.0044424  [DOI](https://doi.org/10.1371/journal.pone.0044424) | [PMC free article](/articles/PMC3439414/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22984506/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Effects%20of%20genetic%20variants%20previously%20associated%20with%20fasting%20glucose%20and%20insulin%20in%20the%20Diabetes%20Prevention%20Program&author=JC%20Florez&author=KA%20Jablonski&author=JB%20McAteer&author=PW%20Franks&author=CC%20Mason&volume=7&issue=9&publication_year=2012&pages=e44424&pmid=22984506&doi=10.1371/journal.pone.0044424&)

27. Masilela C, Pearce B, Ongole JJ, Adeniyi OV, Benjeddou M. Single nucleotide polymorphisms associated with metformin and sulphonylureas’ glycaemic response among South African adults with type 2 diabetes mellitus. J Pers Med. 2021. Feb 6;11(2):104. doi: 10.3390/jpm11020104  [DOI](https://doi.org/10.3390/jpm11020104) | [PMC free article](/articles/PMC7914534/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33561991/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pers%20Med&title=Single%20nucleotide%20polymorphisms%20associated%20with%20metformin%20and%20sulphonylureas%E2%80%99%20glycaemic%20response%20among%20South%20African%20adults%20with%20type%202%20diabetes%20mellitus&author=C%20Masilela&author=B%20Pearce&author=JJ%20Ongole&author=OV%20Adeniyi&author=M%20Benjeddou&volume=11&issue=2&publication_year=2021&pages=104&pmid=33561991&doi=10.3390/jpm11020104&)

28. Li Q, Qiao Y, Wang C, Zhang G, Zhang X, Xu L. Associations between two single-nucleotide polymorphisms (rs1801278 and rs2943641) of insulin receptor substrate 1 gene and type 2 diabetes susceptibility: a meta-analysis. Endocrine. 2016. Jan 1;51(1):52–62. doi: 10.1007/s12020-015-0770-z  [DOI](https://doi.org/10.1007/s12020-015-0770-z) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26582067/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Endocrine&title=Associations%20between%20two%20single-nucleotide%20polymorphisms%20(rs1801278%20and%20rs2943641)%20of%20insulin%20receptor%20substrate%201%20gene%20and%20type%202%20diabetes%20susceptibility:%20a%20meta-analysis&author=Q%20Li&author=Y%20Qiao&author=C%20Wang&author=G%20Zhang&author=X%20Zhang&volume=51&issue=1&publication_year=2016&pages=52-62&pmid=26582067&doi=10.1007/s12020-015-0770-z&)

29. rs4810083 RefSNP Report - dbSNP - NCBI [Internet]. [cited 2024 Mar 28]. Available from: https://www-ncbi-nlm-nih-gov.ezproxy.usherbrooke.ca/snp/rs4810083  [https://www-ncbi-nlm-nih-gov.ezproxy.usherbrooke.ca/snp/rs4810083](https://www-ncbi-nlm-nih-gov.ezproxy.usherbrooke.ca/snp/rs4810083) | [Google Scholar](https://scholar.google.com/scholar_lookup?rs4810083%20RefSNP%20Report%20-%20dbSNP%20-%20NCBI%20%5BInternet%5D.%20%5Bcited%202024%20Mar%2028%5D.%20Available%20from:%20https://www-ncbi-nlm-nih-gov.ezproxy.usherbrooke.ca/snp/rs4810083)

30. Beale EG, Harvey BJ, Forest C. PCK1 and PCK2 as candidate diabetes and obesity genes. Cell Biochem Biophys. 2007. Jul 1;48(2):89–95. doi: 10.1007/s12013-007-0025-6  [DOI](https://doi.org/10.1007/s12013-007-0025-6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17709878/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Biochem%20Biophys&title=PCK1%20and%20PCK2%20as%20candidate%20diabetes%20and%20obesity%20genes&author=EG%20Beale&author=BJ%20Harvey&author=C%20Forest&volume=48&issue=2&publication_year=2007&pages=89-95&pmid=17709878&doi=10.1007/s12013-007-0025-6&)

31. Oishi K, Siegel C, Cork EE, Chen H, Imagawa E. Novel missense variants in PCK1 gene cause cytosolic PEPCK deficiency with growth failure from inadequate caloric intake. J Hum Genet. 2021. Mar;66(3):321–5. doi: 10.1038/s10038-020-00823-8  [DOI](https://doi.org/10.1038/s10038-020-00823-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32908218/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Hum%20Genet&title=Novel%20missense%20variants%20in%20PCK1%20gene%20cause%20cytosolic%20PEPCK%20deficiency%20with%20growth%20failure%20from%20inadequate%20caloric%20intake&author=K%20Oishi&author=C%20Siegel&author=EE%20Cork&author=H%20Chen&author=E%20Imagawa&volume=66&issue=3&publication_year=2021&pages=321-5&pmid=32908218&doi=10.1038/s10038-020-00823-8&)

32. Abrahams-October Z, Xhakaza L, Pearce B, Mandisa Masilela C, Benjeddou M, Vincent Adeniyi O, et al. Genetic association of solute carrier transporter gene variants with metformin response. Balkan J Med Genet. 2021. Jul 27;24(1):47–56. doi: 10.2478/bjmg-2021-0004  [DOI](https://doi.org/10.2478/bjmg-2021-0004) | [PMC free article](/articles/PMC8366475/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34447659/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Balkan%20J%20Med%20Genet&title=Genetic%20association%20of%20solute%20carrier%20transporter%20gene%20variants%20with%20metformin%20response&author=Z%20Abrahams-October&author=L%20Xhakaza&author=B%20Pearce&author=C%20Mandisa%20Masilela&author=M%20Benjeddou&volume=24&issue=1&publication_year=2021&pages=47-56&pmid=34447659&doi=10.2478/bjmg-2021-0004&)

33. Chen Y, Li S, Brown C, Cheatham S, Castro RA, Leabman MK, et al. Effect of genetic variation in the organic cation transporter 2 on the renal elimination of metformin. Pharmacogenet Genomics. 2009. Jul;19(7):497–504. doi: 10.1097/FPC.0b013e32832cc7e9  [DOI](https://doi.org/10.1097/FPC.0b013e32832cc7e9) | [PMC free article](/articles/PMC3104496/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19483665/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Effect%20of%20genetic%20variation%20in%20the%20organic%20cation%20transporter%202%20on%20the%20renal%20elimination%20of%20metformin&author=Y%20Chen&author=S%20Li&author=C%20Brown&author=S%20Cheatham&author=RA%20Castro&volume=19&issue=7&publication_year=2009&pages=497-504&pmid=19483665&doi=10.1097/FPC.0b013e32832cc7e9&)

34. Genetic Variants of the Organic Cation Transporter 2 Influence the Disposition of Metformin [Internet]. [cited 2024 Mar 28]. Available from: https://ascpt.onlinelibrary.wiley.com/doi/epdf/10.1038/clpt.2008.61  [https://ascpt.onlinelibrary.wiley.com/doi/epdf/10.1038/clpt.2008.61](https://ascpt.onlinelibrary.wiley.com/doi/epdf/10.1038/clpt.2008.61) | [Google Scholar](https://scholar.google.com/scholar_lookup?Genetic%20Variants%20of%20the%20Organic%20Cation%20Transporter%202%20Influence%20the%20Disposition%20of%20Metformin%20%5BInternet%5D.%20%5Bcited%202024%20Mar%2028%5D.%20Available%20from:%20https://ascpt.onlinelibrary.wiley.com/doi/epdf/10.1038/clpt.2008.61)

35. Hou W, Zhang D, Lu W, Zheng T, Wan L, Li Q. Polymorphism of organic cation transporter 2 improves glucose-lowering effect of metformin via influencing its pharmacokinetics in Chinese type 2 diabetic patients. Mol Diagn Ther. 2015. Feb 1;19(1):25–33. doi: 10.1007/s40291-014-0126-z  [DOI](https://doi.org/10.1007/s40291-014-0126-z) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25573751/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Diagn%20Ther&title=Polymorphism%20of%20organic%20cation%20transporter%202%20improves%20glucose-lowering%20effect%20of%20metformin%20via%20influencing%20its%20pharmacokinetics%20in%20Chinese%20type%202%20diabetic%20patients&author=W%20Hou&author=D%20Zhang&author=W%20Lu&author=T%20Zheng&author=L%20Wan&volume=19&issue=1&publication_year=2015&pages=25-33&pmid=25573751&doi=10.1007/s40291-014-0126-z&)

36. Mannino GC, Andreozzi F, Sesti G. Pharmacogenetics of type 2 diabetes mellitus, the route toward tailored medicine. Diabetes Metab Res Rev. 2019. Mar;35(3):e3109. doi: 10.1002/dmrr.3109  [DOI](https://doi.org/10.1002/dmrr.3109) | [PMC free article](/articles/PMC6590177/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30515958/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Diabetes%20Metab%20Res%20Rev&title=Pharmacogenetics%20of%20type%202%20diabetes%20mellitus,%20the%20route%20toward%20tailored%20medicine&author=GC%20Mannino&author=F%20Andreozzi&author=G%20Sesti&volume=35&issue=3&publication_year=2019&pages=e3109&pmid=30515958&doi=10.1002/dmrr.3109&)

37. rs316019 RefSNP Report - dbSNP - NCBI [Internet]. [cited 2024 Mar 29]. Available from: https://www-ncbi-nlm-nih-gov.ezproxy.usherbrooke.ca/snp/rs316019  [https://www-ncbi-nlm-nih-gov.ezproxy.usherbrooke.ca/snp/rs316019](https://www-ncbi-nlm-nih-gov.ezproxy.usherbrooke.ca/snp/rs316019) | [Google Scholar](https://scholar.google.com/scholar_lookup?rs316019%20RefSNP%20Report%20-%20dbSNP%20-%20NCBI%20%5BInternet%5D.%20%5Bcited%202024%20Mar%2029%5D.%20Available%20from:%20https://www-ncbi-nlm-nih-gov.ezproxy.usherbrooke.ca/snp/rs316019)

38. Sajib AA, Islam T, Paul N, Yeasmin S. Interaction of rs316019 variants of SLC22A2 with metformin and other drugs- an in silico analysis. J Genet Eng Biotechnol. 2018. Dec;16(2):769–75. doi: 10.1016/j.jgeb.2018.01.003  [DOI](https://doi.org/10.1016/j.jgeb.2018.01.003) | [PMC free article](/articles/PMC6353654/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30733798/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Genet%20Eng%20Biotechnol&title=Interaction%20of%20rs316019%20variants%20of%20SLC22A2%20with%20metformin%20and%20other%20drugs-%20an%20in%20silico%20analysis&author=AA%20Sajib&author=T%20Islam&author=N%20Paul&author=S%20Yeasmin&volume=16&issue=2&publication_year=2018&pages=769-75&pmid=30733798&doi=10.1016/j.jgeb.2018.01.003&)

39. rs316009 RefSNP Report - dbSNP - NCBI [Internet]. [cited 2024 Mar 29]. Available from: https://www-ncbi-nlm-nih-gov.ezproxy.usherbrooke.ca/snp/rs316009  [https://www-ncbi-nlm-nih-gov.ezproxy.usherbrooke.ca/snp/rs316009](https://www-ncbi-nlm-nih-gov.ezproxy.usherbrooke.ca/snp/rs316009)

40. Physical activity [Internet]. [cited 2024 Mar 14]. Available from: https://www.who.int/news-room/fact-sheets/detail/physical-activity  [https://www.who.int/news-room/fact-sheets/detail/physical-activity](https://www.who.int/news-room/fact-sheets/detail/physical-activity) | [Google Scholar](https://scholar.google.com/scholar_lookup?Physical%20activity%20%5BInternet%5D.%20%5Bcited%202024%20Mar%2014%5D.%20Available%20from:%20https://www.who.int/news-room/fact-sheets/detail/physical-activity)

41. Levey AS, Stevens LA, Schmid CH, Zhang YL, Castro AF , Feldman HI, et al. A new equation to estimate glomerular filtration rate. Ann Intern Med. 2009. May 5;150(9):604–12. doi: 10.7326/0003-4819-150-9-200905050-00006  [DOI](https://doi.org/10.7326/0003-4819-150-9-200905050-00006) | [PMC free article](/articles/PMC2763564/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19414839/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Intern%20Med&title=A%20new%20equation%20to%20estimate%20glomerular%20filtration%20rate&author=AS%20Levey&author=LA%20Stevens&author=CH%20Schmid&author=YL%20Zhang&author=AF%20Castro&volume=150&issue=9&publication_year=2009&pages=604-12&pmid=19414839&doi=10.7326/0003-4819-150-9-200905050-00006&)

42. Harris PA, Taylor R, Minor BL, Elliott V, Fernandez M, O’Neal L, et al. The REDCap consortium: building an international community of software platform partners. J Biomed Inform. 2019;95:103208. doi: 10.1016/j.jbi.2019.103208  [DOI](https://doi.org/10.1016/j.jbi.2019.103208) | [PMC free article](/articles/PMC7254481/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31078660/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biomed%20Inform&title=The%20REDCap%20consortium:%20building%20an%20international%20community%20of%20software%20platform%20partners&author=PA%20Harris&author=R%20Taylor&author=BL%20Minor&author=V%20Elliott&author=M%20Fernandez&volume=95&publication_year=2019&pages=103208&pmid=31078660&doi=10.1016/j.jbi.2019.103208&)

43. Whirl-Carrillo M, Huddart R, Gong L, Sangkuhl K, Thorn CF, Whaley R, et al. An evidence-based framework for evaluating pharmacogenomics knowledge for personalized medicine. Clin Pharmacol Ther. 2021. Sep;110(3):563–72. doi: 10.1002/cpt.2350  [DOI](https://doi.org/10.1002/cpt.2350) | [PMC free article](/articles/PMC8457105/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34216021/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=An%20evidence-based%20framework%20for%20evaluating%20pharmacogenomics%20knowledge%20for%20personalized%20medicine&author=M%20Whirl-Carrillo&author=R%20Huddart&author=L%20Gong&author=K%20Sangkuhl&author=CF%20Thorn&volume=110&issue=3&publication_year=2021&pages=563-72&pmid=34216021&doi=10.1002/cpt.2350&)
