# Genetic Variants Associated With Response to Platinum-Based Chemotherapy in Non-Small Cell Lung Cancer Patients: A Field Synopsis and Meta‐Analysis

## Metadata
**Authors:** Hilary Sito, Mohamad Ayub Khan Sharzehan, Md Asiful Islam, Shing Cheng Tan
**Journal:** British Journal of Biomedical Science
**Date:** 2024 Feb 21
**DOI:** [10.3389/bjbs.2024.11835](https://doi.org/10.3389/bjbs.2024.11835)
**PMID:** 38450253
**PMCID:** PMC10914946
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10914946/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC10914946/pdf/bjbs-81-11835.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC10914946/pdf/bjbs-81-11835.pdf)

## Abstract

Background: Publications on the associations of genetic variants with the response to platinum-based chemotherapy (PBC) in NSCLC patients have surged over the years, but the results have been inconsistent. Here, a comprehensive meta-analysis was conducted to combine eligible studies for a more accurate assessment of the pharmacogenetics of PBC in NSCLC patients.

Methods: Relevant publications were searched in PubMed, Scopus, and Web of Science databases through 15 May 2021. Inclusion criteria for eligible publications include studies that reported genotype and allele frequencies of NSCLC patients treated with PBC, delineated by their treatment response (sensitive vs. resistant). Publications on cell lines or animal models, duplicate reports, and non-primary research were excluded. Epidemiological credibility of cumulative evidence was assessed using the Newcastle-Ottawa Scale (NOS) and Venice criteria. Begg’s and Egger’s tests were used to assess publication bias. Cochran’s Q-test and I2 test were used to calculate the odds ratio and heterogeneity value to proceed with the random effects or fixed-effects method. Venice criteria were used to assess the strength of evidence, replication methods and protection against bias in the studies.

Results: A total of 121 publications comprising 29,478 subjects were included in this study, and meta-analyses were performed on 184 genetic variants. Twelve genetic variants from 10 candidate genes showed significant associations with PBC response in NSCLC patients with strong or moderate cumulative epidemiological evidence (increased risk: ERCC1 rs3212986, ERCC2 rs1799793, ERCC2 rs1052555, and CYP1A1 rs1048943; decreased risk: GSTM1 rs36631, XRCC1 rs1799782 and rs25487, XRCC3 rs861539, XPC rs77907221, ABCC2 rs717620, ABCG2 rs2231142, and CDA rs1048977). Bioinformatics analysis predicted possible damaging or deleterious effects for XRCC1 rs1799782 and possible low or medium functional impact for CYP1A1 rs1048943.

Conclusion: Our results provide an up-to-date summary of the association between genetic variants and response to PBC in NSCLC patients.

Keywords: platinum-based chemotherapy, non-small cell lung cancer, polymorphism, systematic review, meta-analysis

## Introduction

Lung cancer is the leading cause of cancer-related deaths worldwide, accounting for 18.0% of all cancer deaths in 2020 [[1](#B1)]. Non-small cell lung cancer (NSCLC), which constitutes 85% of all lung cancer cases, is the major subtype of the cancer [[2](#B2)]. Platinum-based chemotherapy (PBC) is commonly used as first-line treatment and also as adjuvants to radiotherapy or surgery in patients with late-stage NSCLC [[3](#B3)–[5](#B5)]. However, only 30%–40% of patients show a good response to PBC, and tests for extreme drug resistance have shown high variability in chemoresistance (24%–88%) [[6](#B6), [7](#B7)].

The high variability in patient response to PBC has been attributed to individual genetic variants. In recent years, there has been a surge of publications on the pharmacogenetics of PBC in NSCLC patients. Many studies have reported that the genetic variants associated with platinum chemoresistance are involved in the DNA repair pathway, cellular trafficking and drug transport, and metabolic pathways [[8](#B8)–[10](#B10)]. However, the results of published studies are mostly inconsistent and inconclusive. The main reasons for the inconsistency of results are small sample sizes leading to low statistical power, heterogeneity of ethnicities in the different studies due to population stratification, or differences between histological subtypes [[11](#B11)]. A comprehensive meta-analysis is therefore necessary to yield a more precise estimate of the association between genetic variants and PBC responses and also to provide a field synopsis of research in this area. To our knowledge, the most recent comprehensive meta-analysis reporting the association of genetic variants with response to PBC was published in 2017, and only included studies up to 31 January 2016 [[12](#B12)]. As there have been numerous new studies published after January 2016, the inclusion of these studies may yield different results than the previous meta-analysis [[13](#B13)].

Here, we aim to identify, strengthen and interpret the associations of genetic variants with response to PBC in patients with NSCLC patients using a comprehensive systematic review and meta-analysis. We used the standardized guidelines including the Venice criteria to systematically assess the credibility of all relevant studies from publications [[14](#B14)–[17](#B17)]. In this work, a total of 184 genetic variants were examined based on the reported genotype frequencies for each variant corresponding to either chemoresistance or good response to PBC in NSCLC patients [[18](#B18)]. Results were also stratified by ethnicity to provide greater insight into the underlying factors that influence response to PBC.

## Materials and Methods

### Search Strategy, Eligibility Criteria and Data Extraction

The meta-analysis was reported in accordance with the PRISMA guidelines [[16](#B16)]. A comprehensive literature search was performed on PubMed, Scopus, and Web of Science (WoS) databases using combinations of three groups of keywords: platinum OR cisplatin OR carboplatin OR oxaliplatin OR nedaplatin; polymorphism OR SNP OR variant OR mutation; NSCLC OR non-small cell lung cancer up to 15 May 2021. We then used alternative wording for the above terms for a supplemental search. No language restrictions were applied to the literature search, and languages other than English, Malay and Chinese were translated using a professional translation service. Authors were contacted by email to obtain the missing full text publications. Two investigators (HS and SCT) independently selected eligible studies based on the predefined inclusion and exclusion criteria. Disagreements between the two investigators were resolved through a process of discussion and mutual consensus. The inclusion criteria were: 1) included patients should be confirmed as having NSCLC, 2) PBC was administered for treatment, 3) contained data on genotype and allele frequencies (or sufficient data to derive this information) and on the treatment response. The following studies were excluded: 1) they were performed in cell lines or animals, 2) duplicate reports, 3) non-original research (e.g., reviews, case reports, and meta-analyses). If it was ambiguous whether two or more studies contained overlapping data, we contacted the study author(s) by email to verify. The study protocol was prospectively registered at PROSPERO (registration number: CRD42021254570).

Relevant data were extracted independently by two investigators (HS and SCT), and discrepancies were resolved by discussion and mutual consensus. The following information was extracted from each eligible study: first author, publication year, ethnicity, genotyping methods, gene and variant information, chemotherapeutic agents, SNPs, and disease stage. The ethnicity of study participants was broadly divided into European and Asian. The quality of eligible studies was independently assessed by two investigators (HS and SCT) using the Modified Newcastle-Ottawa Scale for Case-Control Studies of Genetic Association [[19](#B19)], and discrepancies between the investigators were resolved by discussion and mutual consensus. A study was considered to be of good quality if it had 6 or more stars [[14](#B14)].

### Data Management and Abstraction

Two studies by Mlak et al. [[20](#B20), [21](#B21)] had overlapping datasets, so only the more recent dataset with the larger population data was used, as recommended by Little et al. [[22](#B22)]. To ensure that the nomenclature of genetic variants was consistent, the “rs” number identifiers from the public single nucleotide polymorphism (SNP) database (dbSNP) [[23](#B23)] were used to designate shortlisted genetic variants. For the remaining variants without “rs” numbers, the nomenclature described in the original articles was used.

### Statistical Analysis

We used genotype frequencies of NSCLC patients classified into the non-responding and responding groups as indicators of PBC response. In the included studies which adhered to RECIST criteria, patients were divided as follows: the non-responding group comprised patients with stable or progressive diseases (SD and PD), while the responding group comprised complete and partial responders (CR and PR) [[18](#B18)]. Meta-analyses were performed under five genetic models: 1) homozygous model (homozygous variant genotype versus wild type genotype), 2) heterozygous model (heterozygous genotype versus wild type genotype), 3) dominant model (heterozygous and homozygous variant genotypes versus wild type genotype), 4) recessive model (homozygous variant versus wild type and heterozygous genotypes), and 5) allele model (variant allele versus wild type allele). If a study reported the frequency of the homozygous and heterozygous genotypes but not that of the allele, we derived the allele model frequency by calculating the sum of the frequencies of the individual genotypes. In contrast, if a study reported the allele frequency but did not distinguish between homozygous and heterozygous genotypes, the data analysis was performed according to the allele model but not other models. This led to variations in the number of studies included in our meta-analysis for the different genetic models. This methodology is consistent with standard practices in the field of meta-analysis of genetic association studies [[24](#B24)]. Conventional comparisons from publications were used to evaluate the effects of genetic variants that were not single nucleotide polymorphisms (e.g., *GSTM1* [null vs. present]).

Heterogeneity among studies was assessed by using Cochran’s Q test and the I^2^ test. Studies with a Q test *p*-value of <0.10 and and I^2^ heterogeneity value of >50% were considered highly heterogeneous. The random-effects method was used to calculate the pooled odds ratio (OR) and the corresponding 95% confidence interval (CI) in studies with high heterogeneity to estimate the association between genetic variants and response to platinum-based therapy; otherwise, a fixed-effects method was used. Statistical assessment of publication bias was performed using Begg’s rank correlation and Egger’s linear regression tests, followed by visual inspection of the funnel plot for asymmetry. The significance level was set at 0.05 unless otherwise stated. The strength of epidemiological evidence was assessed using the Venice criteria [[15](#B15)]. Subgroup analysis was performed based on the ethnicity of the patients and methodological quality of the studies. Sensitivity analyses were performed by iteratively omitting one study at a time to determine the stability and robustness of the results. All statistical analyses were performed using STATA/S.E 14.0 (StataCorp, College Station, TX).

### Functional Annotations

We performed further genomic annotations for genetic variants that showed significant associations with PBC response with moderate or strong epidemiological evidence. We used Ensembl Variation Pathogenicity Predictions [[25](#B25)] which includes a wide range of algorithms such as SIFT and PolyPhen-2 for variations leading to amino acid substitutions, CADD to measure variant deleteriousness, and REVEL, MutationAssessor and MetaLR for human missense variant pathogenicity scores.

## Results

### Study Selection and Characteristics

The flowchart of the study selection is shown in [Figure 1](#F1). The initial comprehensive literature search for relevant publications in the PubMed, Scopus, and Web of Science databases yielded 9,144 results, which were subjected to deduplication and screening to exclude publications that did not contain genotype information or were irrelevant. The selection process identified 121 eligible publications ([Supplementary Table S1](#s11)). The eligible studies involved 29,478 NSCLC patients and reported a total of 184 genetic variants from 95 genes. The vast majority (∼85%) of the studies were conducted in Asians. More than 80% of the publications focused only on advanced NSCLC (stages III–IV), while the remainder included early stages too.

### FIGURE 1.

![FIGURE 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/bd07/10914946/9a7c46d316b0/bjbs-81-11835-g001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=10914946_bjbs-81-11835-g001.jpg)

Flowchart of literature search and selection.

### Overall Meta-Analysis Findings

Among the 184 genetic variants analyzed, a total of 67 genetic variants in 49 candidate genes showed statistically significant associations with response to platinum chemotherapy in at least one genetic model (*p* < 0.05) ([Supplementary Table S2](#s11)). With the exception of *BAX* rs4645878 (G/A) and *XRCC1* rs25487 (G/A), all other genetic variants were consistent in the magnitude of association across different genetic models (either OR>1 or OR<1). Of the remaining 66 genetic variants, 32 were putative “risk” variants that are significantly associated with increased risk of chemoresistance, while 33 were “protective” variants associated with increased sensitivity to PBC.

Credibility assessment using the Venice criteria revealed that the association of one genetic variant (*ERCC2* rs1799793) had strong cumulative epidemiological evidence, whereas 12 variants (*ABCC2* rs717620, *ABCG2* rs2231142, *CDA* rs1048977, *CYP1A1* rs1048943, *ERCC1* rs3212986, *ERCC2* rs1799793 and rs1052555, *GSTM1* rs36631, *XPC* rs77907221, *XRCC1* rs1799782 and rs25487, and *XRCC3* rs861539) showed moderate evidence in at least one genetic model (it should be noted that the cumulative epidemiological evidence of *ERCC2* rs1799793 was strong under the dominant genetic model and moderate under the allele model) ([Supplementary Table S2](#s11); [Supplementary Figures S1–S12](#s11)). Under the dominant model, *ERCC2* rs1799793 was associated with an increased risk of PBC chemoresistance (OR = 1.186, 95% CI = 1.000–1.407), although the association was at borderline significance (*p* = 0.049). Similarly, an increased risk of chemoresistance was observed for under the allele model (OR = 1.311, 95% CI = 1.082–1.590, *p* < 0.01), as well as for *CYP1A1* rs1048943 (dominant model, OR = 2.593, 95% CI = 1.535–4.381, *p* < 0.01; heterozygous model, OR = 2.512, 95% CI = 1.437–4.392, *p* < 0.01; allele model, OR = 1.851, 95% CI = 1.303–2.631, *p* < 0.01), *ERCC1* rs3212986 (recessive model, OR = 1.685, 95% CI = 1.167–2.433, *p* < 0.01; homozygous model, OR = 1.980, 95% CI = 1.346–2.913, *p* < 0.01; allele model, OR = 1.417, 95% CI = 1.052–1.910; *p* = 0.02), *ERCC2* rs1052555 (dominant model, OR = 1.473, 95% CI = 1.063–2.042, *p* = 0.02), and *XRCC1* rs25487 (recessive model, OR = 1.526, 95% CI = 1.105–2.107, *p* = 0.01). In contrast, significantly decreased risk for chemoresistance was observed for *ABCC2* rs717620 (allele model, OR = 0.044, 95% CI = 0.008–0.241, *p* < 0.01), *ABCG2* rs2231142 (recessive model, OR = 0.480, 95% CI = 0.316–0.727, *p* < 0.01; homozygous model, OR = 0.477, 95% CI = 0.306–0.741, *p* < 0.01; allele model, OR = 0.754, 95% CI = 0.612–0.929, *p* < 0.01), *CDA* rs1048977 (allele model, OR = 0.516, 95% CI = 0.342–0.778, *p* = 0.02), *GSTM1* rs36631 (allele model, OR = 0.531, 95% CI = 0.411–0.687, *p* < 0.01), *XPC* rs77907221 (heterozygous model, OR = 0.369, 95% CI = 0.193–0.703, *p* < 0.01; allele model, OR = 0.639, 95% CI = 0.465–0.877, *p* < 0.01), *XRCC1* rs1799782 (recessive model, OR = 0.611, 95% CI = 0.453–0.825, *p* < 0.01; homozygous 0.494, 95% CI = 0.316–0.773, *p* < 0.01; allele model, OR = 0.625, 95% CI = 0.473–0.825, *p* < 0.01), *XRCC1* rs25487 (homozygous model, OR = 0.647, 95% CI = 0.455–0.920, *p* = 0.02), and *XRCC3* rs861539 (allele model, OR = 0.734, 95% CI = 0.586–0.918, *p* < 0.01; dominant model, OR = 0.686, 95% CI = 0.508–0.925, *p* = 0.02).

### Sensitivity Analysis

Sensitivity analysis of the genetic variants associated with response to PBC that had moderate or high epidemiological evidence showed that the results were not significantly altered when any of the individual studies was omitted from the analysis, suggesting that the associations were robust and not driven by any single study. The sensitivity analysis plots for these variants are provided in [Supplementary Figures](#s11).

### Subgroup Analysis

Subgroup analysis sorted by ethnicity was performed for *ERCC1* rs11615, *ERCC1* rs3212986 and *ERCC2* rs13181 as only these variants were reported in at least three studies in the Asian and European populations ([Supplementary Table S3](#s11); [Supplementary Figures S13–S15](#s11)). For *ERCC1* rs11615, an increased risk of PBC chemoresistance was observed in Asians under all genetic models examined (homozygous model, OR = 1.287, 95% CI = 0.729–2.272; heterozygous model, OR = 1.165, 95% CI = 0.854–1.589; dominant model, OR = 1.189, 95% CI = 0.843–1.676; recessive model, OR = 1.287, 95% CI = 0.729–2.272; allele model, OR = 1.112, 95% CI = 0.835–1.481), whereas a decreased risk was observed in Europeans (homozygous model, OR = 0.851, 95% CI = 0.510–1.418; heterozygous model, OR = 0.771, 95% CI = 0.496–1.197; dominant model, OR = 0.789, 95% CI = 0.520–1.197; recessive model, OR = 0.964, 95% CI = 0.659–1.410; allele model, OR = 0.923, 95% CI = 0.729–1.169), although the associations were not statistically significant (*p* > 0.05).

Similarly, *ERCC1* rs3212986 was statistically significantly associated with an increased risk of PBC chemoresistance in Asians (dominant model, OR = 1.556, 95% CI = 1.078–2.244, *p* = 0.018), but a decreased risk was observed in Europeans with borderline lack of significance (OR = 0.718, 95% CI = 0.510–1.010, *p* = 0.057).

*ERCC2* rs13181 also showed a similar result, with OR was <1 for Europeans in the allele, dominant and heterozygous models, but OR > 1 for Asians in the allele and dominant models. Nevertheless, these associations were not statistically significant (*p* > 0.05; [Supplementary Table S3](#s11)).

Subgroup analysis by methodological quality of the studies was not performed because the vast majority of the included studies were of good quality.

### Publication Bias

As shown in [Supplementary Table S2](#s11), publication bias was observed in several genetic variants (*p* < 0.05 in both Begg’s and Egger’s tests). However, the presence of publication bias would result in the genetic variants being classified as having weak epidemiological evidence according to the Venice criteria. Therefore, none of the shortlisted variants (those significantly associated with PBC response in NSCLC patients with strong or moderate cumulative epidemiological evidence, i.e., *ABCC2* rs717620, *ABCG2* rs2231142, *CDA* rs1048977, *CYP1A1* rs1048943, *ERCC1* rs3212986, *ERCC2* rs1799793 and rs1052555, *GSTM1* rs36631, *XPC* rs77907221, *XRCC1* rs1799782 and rs25487, and *XRCC3* rs861539) showed the presence of publication bias. Nevertheless, it should be noted that the results of *CDA* rs1048977 (allele model), *CYP1A1* rs1048943 (heterozygous, dominant, and allele models), *ERCC2* rs1052555 (dominant model), *GSTM1* rs36631, *XPC* rs77907221 (heterozygous and allele models) and *XRCC1* rs25487 (recessive model) should be interpreted with caution as the publication bias of these genetic variants was either at the borderline of statistical significance in Begg’s or Egger’s test, or could not be analyzed with Egger’s test because only two studies were available for the meta-analysis. Despite this, obvious asymmetry was not observed for any of these variants ([Supplementary Figures S16–S27](#s11)).

### Functional Annotation

Bioinformatics analysis was performed for the shortlisted genetic variants to clarify the functional effects of the variants. The missense variant *XRCC1* rs1799782 could have a deleterious effect or probably damaging effect because it has low SIFT scores of 0.01–0.04 and high PolyPhen-2 scores of 0.393–0.999 for most of the *XRCC1* transcripts. Another missense variant *CYP1A1* rs1048943 has a medium Mutation Assessor score of 0.587 for the prediction of the functional impact of amino acid substitutions. The remaining missense variants for *ERCC2* rs1799793, *XRCC1* rs25487, *XRCC1* rs861539 and *ABCG2* rs2231142 showed scores for benign and tolerant variants. Pathogenicity prediction algorithms were not available for *ERCC1* rs11615, *ERCC1* rs3212986, *XPC* rs77907221, *ERCC2* rs1052555, *CDA* rs1048977 and *ABCC* rs717620 as the transcripts were either synonymous variants, downstream gene variants, 5′-UTR or 3′-UTR variants, nonsense-mediated decay variants, non-coding exon variants, or intron variants.

### Non-Significant Associations

A non-significant association (*p* > 0.05) in at least one genetic model examined was found for 171 genetic variants in 95 genes, many of which overlapped with those showing significant associations. The list of the non-significant associations is shown in [Supplementary Table S4](#s11).

## Discussion

To our knowledge, the most recent comprehensive meta-analysis examining associations between genetic variants and response to PBC in NSCLC patients was published in 2017, with the literature search last conducted on 31 January 2016 [[12](#B12)]. Since then, a large number of studies reporting the associations of genetic variants to response to PBC have been published. Here, we performed an updated meta-analysis that identified 12 genetic variants in 10 candidate genes that were statistically significantly associated with response to PBC in NSCLC patients with strong or moderate cumulative epidemiological evidence.

A large proportion of the genetic variants are involved in DNA synthesis and repair, consistent with the previous meta-analysis [[12](#B12)]. The mechanism of platinum drugs involves the formation of DNA lesions that lead to apoptosis of cancer cells. Therefore, genetic variants affecting DNA synthesis and repair would influence efficacy of PBC [[26](#B26), [27](#B27)]. *ERCC1* is one of the genes involved in nucleotide excision repair (NER). High expression of ERCC1 was found in cisplatin-resistant cancer cells, and can be used to predict low clinical efficacy of cisplatin [[28](#B28)]. However, we demonstrated that the *ERCC1* rs11615 was not significantly associated with PBC response under all genetic models. This is in contrast to with previous meta-analyses and gene association studies which showed significant associations with response to PBC [[12](#B12), [20](#B20), [29](#B29)–[38](#B38)]. The results may be due to a bias in favor skewed results towards of selected ethnicities, as new studies have been added since the previous meta-analysis. This postulation is supported by the results of our subgroup analysis, which showed that the Asian population has a higher OR compared to the European population. The *ERCC1* rs11615 variant allele has been suggested by several studies to affect NER by reducing *ERCC1* transcripts, which impairs DNA repair and leads to DNA damage accumulation [[39](#B39), [40](#B40)], but the functional studies revealed similar results for both the variant and wild type alleles in other studies [[41](#B41)]. The inconsistencies in the functional studies could be due to indirect mechanisms such as linkage disequilibrium or regulation by other factors. The variant allele of *ERCC1* rs3212986 was also found to be associated with PBC response in the current meta-analysis, which is consistent with previous studies [[12](#B12), [30](#B30), [34](#B34), [42](#B42)–[46](#B46)]. Several studies have suggested the *ERCC1* rs3212986 variant may modulate mRNA expression, because it is located in the 3′-UTR, is in linkage disequilibrium with the neighboring *XRCC1* and *XPD* genes, which may affect mRNA stability, and the bioinformatics analysis of *ERCC1* rs32123986 revealed the possible alteration of secondary structure in the 3′-UTR and post-transcriptional regulation via binding miRNAs [[40](#B40), [47](#B47), [48](#B48)]. The *ERCC1* rs3212986 showed population stratification in the subgroup analysis, with the Asian population having a higher OR (1.556, 95% CI = 1.078–2.244) compared to the European population under the dominant genetic model (OR = 0.718, 95% CI = 0.510–1.010). The underlying mechanisms for the racial differences in PBC response are not clearly defined, but could be due to gene-gene or gene-environment interactions, which include differences in ethnicity or lifestyle [[49](#B49)–[51](#B51)].

*ERCC2 (XPD)* is another gene involved in NER, with XPD serving the DNA helicase subunit in TFIIH [[52](#B52)]. Genetic variants of *ERCC2* have been shown to be associated with impaired DNA repair capacity, leading to accumulation of DNA adducts [[53](#B53)]. The variant allele of *ERCC2* rs1052555 was also identified in this meta-analysis to be associated with response to PBC, consistent with Li et al. and Tan et al. [[12](#B12), [54](#B54)]. The *ERCC2* rs1052555 is a silent substitution predicted to form a splicing abolish domain or exon splicing enhancer that affects post-transcriptional mRNA splicing [[54](#B54)]. The altered splicing could result in different expression levels of ERCC2 that contribute to PBC response. On the other hand, the variant allele of *XRCC1* rs1799782 was found to be associated with a protective effect against PBC chemoresistance, which was in agreement with several studies, including a recent meta-analysis by Zhang et al. [[12](#B12), [55](#B55)–[65](#B65)]. One possible mechanism by which the variant allele of *XRCC1* rs1799782 protects against PBC chemoresistance is by interfering with the repair of DNA breaks caused by platinum drugs, as the *XRCC1* rs1799782 was predicted to be deleterious or damaging by bioinformatics tools. However, other functional studies suggest that allelic substitution of the variant may indirectly affect the stability of *XRCC1* via interactions with miRNA and polymerases involved in DNA repair [[66](#B66), [67](#B67)]. Similarly, the variant allele of *XRCC3* rs861539 was also demonstrated to be associated with a protective effect against chemoresistance, consistent with the meta-analysis by Tan et al. [[12](#B12)]. The protective effect of *XRCC3* rs861539 against chemoresistance is supported by the bioinformatics prediction of a resulting benign and tolerated XRCC3 that contributes to the proper function of DNA repair.

Interestingly, after incorporating data from recent publications [[21](#B21), [45](#B45), [68](#B68)], we identified several new significant associations that were not found to be statistically significant in the previous meta-analysis by Tan et al. These include variants in genes involved in DNA repair (*ERCC2* rs1799793, *XRCC1* rs25487, and *XPC* rs77907221). The variant allele of *ERCC2* rs1799793 showed associations with PBC chemoresistance, supporting the findings of a previous meta-analysis by Qiu et al. [[69](#B69)]*.* It is unlikely that *ERCC2* rs1799793 causes PBC chemoresistance by affecting DNA repair, as our bioinformatics analysis predicted that the genetic variant is benign and tolerated. Instead, the *ERCC2* rs1799793 may indirectly contribute to platinum chemoresistance by dysregulating apoptosis, as the resulting Asp-to-Asn substitution led to a 2.5-fold increased apoptotic response in a lymphoblastoid cell line [[70](#B70)]. Similarly, the variant allele of *XRCC1* rs25487 also showed associations with PBC chemoresistance, consistent with a recent meta-analysis by Zhang et al. [[61](#B61)]. Bioinformatics analyses predicted that the *XRCC1* rs25487 variants are benign and tolerated, suggesting that the genetic variant also does not directly affect DNA repair to result in PBC chemoresistance. On the other hand, *XPC* rs77907221 insertions were found to be associated with better PBC response here consistent with a few older studies [[71](#B71), [72](#B72)]. Since there are only a few studies on the functional effects of *XPC* SNPs*,* the causal mechanism of *XPC* rs77907221 is unclear. However, the strong linkage disequilibrium between *XPC* rs7790722, *XPC* rs2228001 and *XPC* intron 11 may make a synergistic contribution to the PBC response [[73](#B73), [74](#B74)].

Besides, whereas Tan et al. previously reported significant associations of *XPA* rs1800975, *ERCC2* rs13181 and *ERCC5* rs2296147 with PBC response, this result was not replicated in our work. For *XPA* rs1800975 and *ERCC5* rs2296147, we could find only two eligible studies and therefore, classified the variants as having poor cumulative epidemiologic evidence. For *ERCC2* rs13181, no statistically significantly association with PBC response was found in our meta-analysis, possibly due to the high heterogeneity score in the homozygous, recessive, dominant and allele models after the addition of recent publications. Subgroup analysis showed that Asians had a higher OR than Europeans, suggesting that population stratification contributes to the high heterogeneity score. The confounding effect of ethnicity is evident because *ERCC2* rs13181 showed significant associations in Europeans but not in Asians in the dominant model.

Metabolic and detoxification regulators also account for a large proportion of the genetic variants identified as having significant associations with PBC responses in this meta-analysis. The cytidine deaminase (CDA) enzyme plays a critical role in the metabolism and inactivation of gemcitabine, which is commonly used in combination with platinum drugs to treat NSCLC [[75](#B75), [76](#B76)]. *CDA* rs1048977 is a newly identified genetic variant that shows a significant association with PBC response, which was not reported in the previous meta-analysis by Tan et al. [[12](#B12)] The variant allele of *CDA* rs1048977 showed a significant association with better response to PBC in the allele and recessive models, which is consistent with Hu et al. and Ludovini et al., although in the recessive model it was considered as having weak cumulative epidemiologic evidence [[77](#B77), [78](#B78)]. Ludovini et al. suggested that the *CDA* rs1048977 is associated with better response due to lower enzyme activity which results in high drug availability. However, a more recent study by Ciccolini et al. showed that the polymorphism did not alter CDA activity, suggesting indirect regulatory mechanisms [[78](#B78), [79](#B79)].

*CYP1A1* is another gene involved in the metabolism of antineoplastic drugs and has been shown to influence responses to PBC [[80](#B80)]. *CYP1A1* rs1048943 is another newly identified genetic variant in this current meta-analysis that shows a significant association with response to PBC. The variant allele of *CYP1A1* rs1048943 was also newly identified in this meta-analysis to be significantly associated with PBC chemoresistance and classified as having moderate cumulative epidemiological evidence in the heterozygous, dominant, and allele models. Interestingly, *CYP1A1* rs1048943 had a slight tendency to deleterious functional effects based on the Mutation Assessor bioinformatics score, which assesses the evolutionary conservation of affected amino acids in protein homologs. The *CYP1A1* rs1048943 is another newly identified genetic variant that has not been reported to show associations with PBC response in any previous meta-analysis, but was previously shown to be significantly associated with lung cancer susceptibility [[81](#B81)]*.* Furthermore, the *CYP1A1* rs1048943 variant allele has previously been shown to have higher enzyme activity that promotes DNA adduct accumulation [[82](#B82)–[84](#B84)].

*ABCC2* and *ABCG2* are drug transporters that play critical roles for the influx and efflux of platinum drugs [[85](#B85)]. Overexpression of ABCC2 during cisplatin treatment has been shown to contribute to cisplatin chemoresistance due to less DNA-cisplatin adduct formation, and lower intracellular accumulation of cisplatin [[85](#B85)–[90](#B90)]. In this meta-analysis, we identified two significant variants in these genes that had not been reported to be associated with response to PBC in previous meta-analyses, i.e., *ABCC2* rs717620 and *ABCG2* rs2231142. The drug transporters *ABCC2* rs717620 variant allele and *ABCG2* rs2231142 variant allele were both found to be significantly associated with protective effects against PBC chemoresistance. The better response of NSCLC patients to the *ABCC2* rs717620 variant allele is consistent with the reports by Han et al. and Qiao et al [[91](#B91), [92](#B92)]*.* The *ABCC2* rs717620 may have indirect effects on PBC response, as Zhang et al. reported no effects on mRNA expression or downstream open reading frame translation, whereas Nguyen et al. reported that the *ABCC2* rs717620, together with *ABCC2* rs18885301 and *ABCC2* rs2804402, increased promoter activity by 35% [[93](#B93), [94](#B94)]. Similarly, the variant allele of *ABCG2* rs2231142 was also reported to be associated with better PBC response by Qiao et al. [[95](#B95)]. The improved PBC response by *ABCG2* rs2231142 may be due to reduced efflux of platinum drugs from tumor cells using ABCG2. Since *ABCG2* rs2231142 was predicted to be benign or tolerated in our bioinformatics analysis, the decreased ABCG2 activity may occur indirectly via regulation of protein levels. Previous studies have suggested that *ABCG2* rs2231142 elicits its good PBC response by having lower protein levels in the lungs and increased platinum bioavailability in cell lines [[96](#B96)–[101](#B101)]. Previous meta-analyses by Tan et al. and Wei et al. also identified *MTHFR* rs18001133 and *MDR* rs1045642 as genetic variants that showed significant associations with PBC response [[12](#B12), [33](#B33)], but no statistically significant association was observed in this current meta-analysis. The lack of significant associations may be due to the significant heterogeneity reported for most genetic models after the addition of new publications.

The current meta-analysis provides the most up-to-date field synopsis and assessment of the associations between genetic variations and response to PBC. In addition, there are sufficient data from recent publications to allow the meta-analyses to be performed for all genetic models (whereas the previous meta-analysis by Tan et al. was limited to the homozygous, heterozygous, and dominant models). The allele model here combined both the homozygous and heterozygous into a single allele model. One advantage provided by the allele model is to distinguish the biological effects when heterozygous and homozygous genotypes are similar. Furthermore, by focusing on allele frequencies rather than genotypes in our meta-analysis, we take advantage of the increased number of observations, essentially doubling our sample size since each individual contributes two alleles. This approach not only increases the statistical power of our study and allows for more precise detection of genetic associations, but also shifts our focus more directly to the individual effects of the alleles, allowing for a clearer understanding of their role in the disease under study. Our study findings provide the impetus for clinical applications in precision medicine, where complementary data on gene-gene, gene-environment, functional studies, and validation in local population of patients will drive the development of potential prognostic biomarkers in the clinical setting. Identification of variants that showed a robust association with response to PBC could contribute to a more accurate assessment of the potential target population for optimizing the efficacy of PBC [[102](#B102)]. These findings could be helpful in the development of genetic tests that can predict the response of patients to PBC and thus enable a more targeted selection of therapy [[103](#B103)]. For example, patients with genotypes associated with improved respond to PBC could be prioritized for the treatment, which would improve treatment efficacy and patient outcomes. In addition, understanding the genetic basis of PBC response may lead to optimization of drug dosing. Personalized dosing based on individual genetic profiles may maximize therapeutic efficacy while minimizing adverse effects, which could significantly improve the quality of life of NSCLC patients undergoing treatment. The identified genetic variants also provide insights into the molecular mechanisms underlying PBC resistance [[104](#B104)]. This knowledge is invaluable for drug development as it can guide the development of new agents that can overcome resistance or improve the efficacy of existing treatments. This genetic knowledge could also facilitate the development of combination therapies. By understanding the genetic profiles that drive resistance to PBC, clinicians can better select complementary treatments that may overcome resistance and improve overall treatment efficacy. In addition, these findings also have implications for health economics [[105](#B105)]. By identifying patients who are more likely to benefit from PBC, healthcare resources can be allocated more efficiently, reducing the cost of ineffective treatments and the management of side effects. The discovery of these genetic variants also opens up opportunities for further research. For example, investigating the interaction between these genetic factors and environmental or lifestyle factors could lead to a more comprehensive understanding of the response to PBC in NSCLC, ultimately leading to improved patient care and better outcomes.

However, there are several limitations to this meta-analysis. First, more than 80% of the genetic polymorphisms identified in the eligible publications were reported in only one or two studies, resulting in small sample sizes and led to having poor cumulative epidemiologic evidence for a number of genetic variations. The limited number of studies on individual variants also prevented us from performing sensitivity analyses, subgroup analyses or tests for publication bias, which may be a source of heterogeneity. We also did not examine the gene-gene or gene-environment interactions that might influence the efficacy of PBC. It is also possible that individual genetic variants identified to be associated with response to PBC here are not the causal variants. The genetic variants could be in linkage disequilibrium with other SNPs or alter the stability of the resulting mRNA. The gene-gene and gene-environment interactions, along with the possible linkage disequilibrium with causal variants, need to be further investigated before the significant variants are used as clinical prognostic biomarkers. For instance, the combined effects of *ERCC1* rs11615, *ERCC1* rs3212986 and *ERCC2* rs1799793 were shown to decrease the overall survival in non-squamous NSCLC patients undergoing pemetrexed/platinum-based chemotherapy but the individual effects of SNPS towards OS were not significant [[106](#B106)]. The OS of the patient decreased with the presence of every unfavorable allele from approximately 30 months with two to three unfavorable alleles to 11.8 months (*p* = 0.01) with four unfavourable alleles [[106](#B106)]. In addition, genome-wide scans should also be conducted in more populations to identify novel genetic variants associated with response to PBC in an unbiased manner. It would also be interesting to investigate whether these genetic variants are associated with lncRNA dysregulations and mitochondrial DNA alterations, which have recently been shown to influence cisplatin response [[107](#B107), [108](#B108)]. Finally, functional predictions of significant genetic variants were only made using *in silico* methods, which use different algorithms that assess the degree of conservation of amino acids across different species, physiochemical properties, or the combination of functional data and variant annotations, leading to discrepancies in the predictions [[109](#B109)]. Thus, the results of functional predictions should be interpreted with caution. Despite possible discrepancies in functional prediction, the use of multiple algorithms is advantageous as it allows for a comprehensive analysis, leading to more robust and reliable conclusions [[110](#B110)]. This approach allows for a balanced interpretation of the results and ensures that conclusions are not overly reliant on a single predictive model, but are instead informed by a spectrum of computational insights, which increases confidence in the pathogenicity prediction of each genetic variant. In the future, additional *in vitro* or *in vivo* work is needed to confirm and validate the function of these genetic variants [[111](#B111)].

In conclusion, our meta-analyses identified 12 genetic variants in 10 candidate genes (*ABCC2* rs717620, *ABCG2* rs2231142, *CDA* rs1048977, *CYP1A1* rs1048943, *ERCC1* rs3212986, *ERCC2* rs1799793 and rs1052555, *GSTM1* rs36631, *XPC* rs77907221, *XRCC1* rs1799782 and rs25487, and *XRCC3* rs861539) that showed statistically significant associations with response to PBC in NSCLC patients with strong or moderate cumulative epidemiological evidence. We also identified 172 genetic variants that were not associated with PBC response in at least one genetic model. Our results provide the most up-to-date summary and field synopsis of the genetic variants associated with response to PBC in NSCLC patients.

## Summary Table

### What Is Known About This Subject

### What This Paper Adds

## Concluding Statement

This work represents an advance in biomedical science because it offers the most comprehensive, up-to-date synthesis of genetic variants influencing platinum-based chemotherapy response in NSCLC patients.

## Funding Statement

This study was supported by the Research University Grant of Universiti Kebangsaan Malaysia (GUP-2020-076). The funder had no role in the design of the study and collection, analysis, and interpretation of data and in writing the manuscript.

## Data Availability Statement

The original contributions presented in the study are included in the article/[Supplementary Material](#s11), further inquiries can be directed to the corresponding author.

## Author Contributions

HS: Data curation, formal analysis, investigation, methodology, validation, visualization, writing—original draft. MS: Investigation, validation. MI: Funding acquisition, resources, writing—review and editing. ST: Conceptualization, formal analysis, investigation, funding acquisition, methodology, project administration, supervision, writing—review and editing. All authors contributed to the article and approved the submitted version.

## Conflict of Interest

The authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.

## Supplementary Material

The Supplementary Material for this article can be found online at: [https://www.frontierspartnerships.org/articles/10.3389/bjbs.2024.11835/full#supplementary-material](https://www.frontierspartnerships.org/articles/10.3389/bjbs.2024.11835/full#supplementary-material)

## Abbreviations

PBC, Platinum-based chemotherapy; SNP, Single nucleotide polymorphism; NSCLC, Non-small cell lung cancer; NOS, Newcastle-Ottawa Scale; RECIST, Response Evaluation Criteria in Solid Tumours; CADD, Combined Annotation Dependent Depletion; SIFTL, Sorting Intolerant From Tolerant; PolyPhen, Polymorphism Phenotyping; REVEL, Rare exome variant ensemble learner; ABCC2, ATP binding cassette Subfamily C Member 2; ABCG2, ATP binding cassette transporter G2; BAX, Bcl-2 associated protein; CDA, cytidine deaminase; CYP1A1, Cytochrome P450 1A1; ERCC1, Excision repair cross complementation group 1; ERCC2, Excision repair cross complementation group 2; ERCC2, Excision repair cross complementation group 5; GSTM1, Glutathione S-transferase 1; MTHFR, Methylenetetrahydrofolate reductase; MDR, Multidrug resistance mutation; XPC, Xeroderma pigmentosum complementation group C; XRCC1, X-ray cross-complementing protein 1; XRCC3, X-ray cross-complementing protein 3; UTR, untranslated region.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

The original contributions presented in the study are included in the article/[Supplementary Material](#s11), further inquiries can be directed to the corresponding author.

### Supplementary Materials

### Data Availability Statement

The original contributions presented in the study are included in the article/[Supplementary Material](#s11), further inquiries can be directed to the corresponding author.

## References

1. Sung H, Ferlay J, Siegel RL, Laversanne M, Soerjomataram I, Jemal A, et al. Global Cancer Statistics 2020: GLOBOCAN Estimates of Incidence and Mortality Worldwide for 36 Cancers in 185 Countries. CA Cancer J Clin (2021) 71(3):209–49. 10.3322/caac.21660  [DOI](https://doi.org/10.3322/caac.21660) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33538338/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=CA%20Cancer%20J%20Clin&title=Global%20Cancer%20Statistics%202020:%20GLOBOCAN%20Estimates%20of%20Incidence%20and%20Mortality%20Worldwide%20for%2036%20Cancers%20in%20185%20Countries&author=H%20Sung&author=J%20Ferlay&author=RL%20Siegel&author=M%20Laversanne&author=I%20Soerjomataram&volume=71&issue=3&publication_year=2021&pages=209-49&pmid=33538338&doi=10.3322/caac.21660&)

2. Molina JR, Yang P, Cassivi SD, Schild SE, Adjei AA. Non–Small Cell Lung Cancer: Epidemiology, Risk Factors, Treatment, and Survivorship. Mayo Clinic Proc Mayo Clinic (2008) 83(5):584–94. 10.4065/83.5.584  [DOI](https://doi.org/10.4065/83.5.584) | [PMC free article](/articles/PMC2718421/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18452692/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mayo%20Clinic%20Proc%20Mayo%20Clinic&title=Non%E2%80%93Small%20Cell%20Lung%20Cancer:%20Epidemiology,%20Risk%20Factors,%20Treatment,%20and%20Survivorship&author=JR%20Molina&author=P%20Yang&author=SD%20Cassivi&author=SE%20Schild&author=AA%20Adjei&volume=83&issue=5&publication_year=2008&pages=584-94&pmid=18452692&doi=10.4065/83.5.584&)

3. Chen VW, Ruiz BA, Hsieh MC, Wu XC, Ries LAG, Lewis DR. Analysis of Stage and Clinical/Prognostic Factors for Lung Cancer From SEER Registries: AJCC Staging and Collaborative Stage Data Collection System. Cancer (2014) 120(S23):3781–92. 10.1002/cncr.29045  [DOI](https://doi.org/10.1002/cncr.29045) | [PMC free article](/articles/PMC4239667/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25412390/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer&title=Analysis%20of%20Stage%20and%20Clinical/Prognostic%20Factors%20for%20Lung%20Cancer%20From%20SEER%20Registries:%20AJCC%20Staging%20and%20Collaborative%20Stage%20Data%20Collection%20System&author=VW%20Chen&author=BA%20Ruiz&author=MC%20Hsieh&author=XC%20Wu&author=LAG%20Ries&volume=120&issue=S23&publication_year=2014&pages=3781-92&pmid=25412390&doi=10.1002/cncr.29045&)

4. Goldstraw P, Chansky K, Crowley J, Rami-Porta R, Asamura H, Eberhardt WEE, et al. The IASLC Lung Cancer Staging Project: Proposals for Revision of the TNM Stage Groupings in the Forthcoming (Eighth) Edition of the TNM Classification for Lung Cancer. J Thorac Oncol (2016) 11(1):39–51. 10.1016/j.jtho.2015.09.009  [DOI](https://doi.org/10.1016/j.jtho.2015.09.009) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26762738/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Thorac%20Oncol&title=The%20IASLC%20Lung%20Cancer%20Staging%20Project:%20Proposals%20for%20Revision%20of%20the%20TNM%20Stage%20Groupings%20in%20the%20Forthcoming%20(Eighth)%20Edition%20of%20the%20TNM%20Classification%20for%20Lung%20Cancer&author=P%20Goldstraw&author=K%20Chansky&author=J%20Crowley&author=R%20Rami-Porta&author=H%20Asamura&volume=11&issue=1&publication_year=2016&pages=39-51&pmid=26762738&doi=10.1016/j.jtho.2015.09.009&)

5. Machado-Rugolo J, Prieto TG, Fabro AT, Cuentas ERP, Sá VK, Baldavira CM, et al. Relevance of PD-L1 Non-Coding Polymorphisms on the Prognosis of a Genetically Admixed NSCLC Cohort. Pharmgenomics Pers Med (2021) 14:239–52. 10.2147/PGPM.S286717  [DOI](https://doi.org/10.2147/PGPM.S286717) | [PMC free article](/articles/PMC7894801/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33623414/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmgenomics%20Pers%20Med&title=Relevance%20of%20PD-L1%20Non-Coding%20Polymorphisms%20on%20the%20Prognosis%20of%20a%20Genetically%20Admixed%20NSCLC%20Cohort&author=J%20Machado-Rugolo&author=TG%20Prieto&author=AT%20Fabro&author=ERP%20Cuentas&author=VK%20S%C3%A1&volume=14&publication_year=2021&pages=239-52&pmid=33623414&doi=10.2147/PGPM.S286717&)

6. Sirohi B, Ashley S, Norton A, Popat S, Hughes S, Papadopoulos P, et al. Early Response to Platinum-Based First-Line Chemotherapy in Non-Small Cell Lung Cancer May Predict Survival. J Thorac Oncol (2007) 2(8):735–40. 10.1097/JTO.0b013e31811f3a7d  [DOI](https://doi.org/10.1097/JTO.0b013e31811f3a7d) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17762340/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Thorac%20Oncol&title=Early%20Response%20to%20Platinum-Based%20First-Line%20Chemotherapy%20in%20Non-Small%20Cell%20Lung%20Cancer%20May%20Predict%20Survival&author=B%20Sirohi&author=S%20Ashley&author=A%20Norton&author=S%20Popat&author=S%20Hughes&volume=2&issue=8&publication_year=2007&pages=735-40&pmid=17762340&doi=10.1097/JTO.0b013e31811f3a7d&)

7. d’Amato TA, Landreneau RJ, Ricketts W, Huang W, Parker R, Mechetner E, et al. Chemotherapy Resistance and Oncogene Expression in Non-Small Cell Lung Cancer. J Thorac Cardiovasc Surg (2007) 133(2):352–63. 10.1016/j.jtcvs.2006.10.019  [DOI](https://doi.org/10.1016/j.jtcvs.2006.10.019) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17258563/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Thorac%20Cardiovasc%20Surg&title=Chemotherapy%20Resistance%20and%20Oncogene%20Expression%20in%20Non-Small%20Cell%20Lung%20Cancer&author=TA%20d%E2%80%99Amato&author=RJ%20Landreneau&author=W%20Ricketts&author=W%20Huang&author=R%20Parker&volume=133&issue=2&publication_year=2007&pages=352-63&pmid=17258563&doi=10.1016/j.jtcvs.2006.10.019&)

8. Yin JY, Li X, Zhou HH, Liu ZQ. Pharmacogenomics of Platinum-Based Chemotherapy Sensitivity in NSCLC: Toward Precision Medicine. Pharmacogenomics (2016) 17(12):1365–78. 10.2217/pgs-2016-0074  [DOI](https://doi.org/10.2217/pgs-2016-0074) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27462924/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Pharmacogenomics%20of%20Platinum-Based%20Chemotherapy%20Sensitivity%20in%20NSCLC:%20Toward%20Precision%20Medicine&author=JY%20Yin&author=X%20Li&author=HH%20Zhou&author=ZQ%20Liu&volume=17&issue=12&publication_year=2016&pages=1365-78&pmid=27462924&doi=10.2217/pgs-2016-0074&)

9. Zheng Y, Ma L, Sun Q. Clinically-Relevant ABC Transporter for Anti-Cancer Drug Resistance. Front Pharmacol (2021) 0:705. 10.3389/fphar.2021.648407  [DOI](https://doi.org/10.3389/fphar.2021.648407) | [PMC free article](/articles/PMC8089384/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33953682/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front%20Pharmacol&title=Clinically-Relevant%20ABC%20Transporter%20for%20Anti-Cancer%20Drug%20Resistance&author=Y%20Zheng&author=L%20Ma&author=Q%20Sun&volume=0&publication_year=2021&pages=705&pmid=33953682&doi=10.3389/fphar.2021.648407&)

10. Hamilton G, Rath B. Pharmacogenetics of Platinum-Based Chemotherapy in Non-Small Cell Lung Cancer: Predictive Validity of Polymorphisms of ERCC1. Expert Opin Drug Metab Toxicol (2018) 14(1):17–24. 10.1080/17425255.2018.1416095  [DOI](https://doi.org/10.1080/17425255.2018.1416095) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29226731/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Expert%20Opin%20Drug%20Metab%20Toxicol&title=Pharmacogenetics%20of%20Platinum-Based%20Chemotherapy%20in%20Non-Small%20Cell%20Lung%20Cancer:%20Predictive%20Validity%20of%20Polymorphisms%20of%20ERCC1&author=G%20Hamilton&author=B%20Rath&volume=14&issue=1&publication_year=2018&pages=17-24&pmid=29226731&doi=10.1080/17425255.2018.1416095&)

11. Tan SC. Low Penetrance Genetic Polymorphisms as Potential Biomarkers for Colorectal Cancer Predisposition. J Gene Med (2018) 20(4):e3010. 10.1002/jgm.3010  [DOI](https://doi.org/10.1002/jgm.3010) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29424105/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Gene%20Med&title=Low%20Penetrance%20Genetic%20Polymorphisms%20as%20Potential%20Biomarkers%20for%20Colorectal%20Cancer%20Predisposition&author=SC%20Tan&volume=20&issue=4&publication_year=2018&pages=e3010&pmid=29424105&doi=10.1002/jgm.3010&)

12. Tan LMM, Qiu CFF, Zhu T, Jin YXX, Li X, Yin JYY, et al. Genetic Polymorphisms and Platinum-Based Chemotherapy Treatment Outcomes in Patients With Non-Small Cell Lung Cancer: A Genetic Epidemiology Study Based Meta-Analysis. Sci Rep (2017) 7(1):5593. 10.1038/s41598-017-05642-0  [DOI](https://doi.org/10.1038/s41598-017-05642-0) | [PMC free article](/articles/PMC5514117/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28717179/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sci%20Rep&title=Genetic%20Polymorphisms%20and%20Platinum-Based%20Chemotherapy%20Treatment%20Outcomes%20in%20Patients%20With%20Non-Small%20Cell%20Lung%20Cancer:%20A%20Genetic%20Epidemiology%20Study%20Based%20Meta-Analysis&author=LMM%20Tan&author=CFF%20Qiu&author=T%20Zhu&author=YXX%20Jin&author=X%20Li&volume=7&issue=1&publication_year=2017&pages=5593&pmid=28717179&doi=10.1038/s41598-017-05642-0&)

13. Tan SC, Low TY, Mohamad Hanif EA, Sharzehan MAK, Kord-Varkaneh H, Islam MA. The Rs9340799 Polymorphism of the Estrogen Receptor Alpha (ESR1) Gene and Its Association With Breast Cancer Susceptibility. Sci Rep (2021) 11(1):18619. 10.1038/s41598-021-97935-8  [DOI](https://doi.org/10.1038/s41598-021-97935-8) | [PMC free article](/articles/PMC8452701/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34545128/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sci%20Rep&title=The%20Rs9340799%20Polymorphism%20of%20the%20Estrogen%20Receptor%20Alpha%20(ESR1)%20Gene%20and%20Its%20Association%20With%20Breast%20Cancer%20Susceptibility&author=SC%20Tan&author=TY%20Low&author=EA%20Mohamad%20Hanif&author=MAK%20Sharzehan&author=H%20Kord-Varkaneh&volume=11&issue=1&publication_year=2021&pages=18619&pmid=34545128&doi=10.1038/s41598-021-97935-8&)

14. Huang R, Chen X. Increased Spot Urine Albumin-To-Creatinine Ratio and Stroke Incidence: A Systematic Review and Meta-Analysis. J Stroke Cerebrovasc Dis (2019) 28(10):104260–7. 10.1016/j.jstrokecerebrovasdis.2019.06.018  [DOI](https://doi.org/10.1016/j.jstrokecerebrovasdis.2019.06.018) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31350166/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Stroke%20Cerebrovasc%20Dis&title=Increased%20Spot%20Urine%20Albumin-To-Creatinine%20Ratio%20and%20Stroke%20Incidence:%20A%20Systematic%20Review%20and%20Meta-Analysis&author=R%20Huang&author=X%20Chen&volume=28&issue=10&publication_year=2019&pages=104260-7&pmid=31350166&doi=10.1016/j.jstrokecerebrovasdis.2019.06.018&)

15. Ioannidis JP, Boffetta P, Little J, O’Brien TR, Uitterlinden AG, Vineis P, et al. Assessment of Cumulative Evidence on Genetic Associations: Interim Guidelines. Int J Epidemiol (2008) 37(1):120–32. 10.1093/ije/dym159  [DOI](https://doi.org/10.1093/ije/dym159) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17898028/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Epidemiol&title=Assessment%20of%20Cumulative%20Evidence%20on%20Genetic%20Associations:%20Interim%20Guidelines&author=JP%20Ioannidis&author=P%20Boffetta&author=J%20Little&author=TR%20O%E2%80%99Brien&author=AG%20Uitterlinden&volume=37&issue=1&publication_year=2008&pages=120-32&pmid=17898028&doi=10.1093/ije/dym159&)

16. Moher D, Liberati A, Tetzlaff J, Altman D, G. PRISMA Group. Preferred Reporting Items for Systematic Reviews and Meta-Analyses: The PRISMA Statement. J Clin Epidemiol (2009) 62(10):1006–12. 10.1016/j.jclinepi.2009.06.005  [DOI](https://doi.org/10.1016/j.jclinepi.2009.06.005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19631508/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Epidemiol&title=Preferred%20Reporting%20Items%20for%20Systematic%20Reviews%20and%20Meta-Analyses:%20The%20PRISMA%20Statement&author=D%20Moher&author=A%20Liberati&author=J%20Tetzlaff&author=D,%20G%20Altman&volume=62&issue=10&publication_year=2009&pages=1006-12&pmid=19631508&doi=10.1016/j.jclinepi.2009.06.005&)

17. Little J, Higgins J. The HuGENet HuGE Review Handbook, Version 1.0 (2006).  [Google Scholar](https://scholar.google.com/scholar_lookup?title=The%20HuGENet%20HuGE%20Review%20Handbook,%20Version%201.0&author=J%20Little&author=J%20Higgins&publication_year=2006&)

18. Ko CC, Yeh LR, Kuo YT, Chen JH. Imaging Biomarkers for Evaluating Tumor Response: RECIST and Beyond. Biomark Res (2021) 9(1):52–20. 10.1186/s40364-021-00306-8  [DOI](https://doi.org/10.1186/s40364-021-00306-8) | [PMC free article](/articles/PMC8252278/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34215324/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biomark%20Res&title=Imaging%20Biomarkers%20for%20Evaluating%20Tumor%20Response:%20RECIST%20and%20Beyond&author=CC%20Ko&author=LR%20Yeh&author=YT%20Kuo&author=JH%20Chen&volume=9&issue=1&publication_year=2021&pages=52-20&pmid=34215324&doi=10.1186/s40364-021-00306-8&)

19. Tan SC, Lim PY, Fang J, Mokhtar MFM, Hanif EAM, Jamal R. Association Between MIR499A Rs3746444 Polymorphism and Breast Cancer Susceptibility: A Meta-Analysis. Sci Rep (2020) 10(1):3508–10. 10.1038/s41598-020-60442-3  [DOI](https://doi.org/10.1038/s41598-020-60442-3) | [PMC free article](/articles/PMC7044335/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32103099/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sci%20Rep&title=Association%20Between%20MIR499A%20Rs3746444%20Polymorphism%20and%20Breast%20Cancer%20Susceptibility:%20A%20Meta-Analysis&author=SC%20Tan&author=PY%20Lim&author=J%20Fang&author=MFM%20Mokhtar&author=EAM%20Hanif&volume=10&issue=1&publication_year=2020&pages=3508-10&pmid=32103099&doi=10.1038/s41598-020-60442-3&)

20. Mlak R, Krawczyk P, Ramlau R, Kalinka-Warzocha E, Wasylecka-Morawiec M, Wojas-Krawczyk K, et al. Predictive Value of ERCC1 and RRM1 Gene Single-Nucleotide Polymorphisms for First-Line Platinum- And Gemcitabine-Based Chemotherapy in Non-Small Cell Lung Cancer Patients. Oncol Rep (2013) 30(5):2385–98. 10.3892/or.2013.2696  [DOI](https://doi.org/10.3892/or.2013.2696) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23982437/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncol%20Rep&title=Predictive%20Value%20of%20ERCC1%20and%20RRM1%20Gene%20Single-Nucleotide%20Polymorphisms%20for%20First-Line%20Platinum-%20And%20Gemcitabine-Based%20Chemotherapy%20in%20Non-Small%20Cell%20Lung%20Cancer%20Patients&author=R%20Mlak&author=P%20Krawczyk&author=R%20Ramlau&author=E%20Kalinka-Warzocha&author=M%20Wasylecka-Morawiec&volume=30&issue=5&publication_year=2013&pages=2385-98&pmid=23982437&doi=10.3892/or.2013.2696&)

21. Mlak R, Krawczyk P, Homa-Mlak I, Powrózek T, Ciesielka M, Kozioł P, et al. Predictive Value of Single Nucleotide Polymorphisms of ERCC1, XPA, XPC, XPD and XPG Genes, Involved in NER Mechanism in Patients With Advanced NSCLC Treated With Cisplatin and Gemcitabine. Pathol Oncol Res (2019) 25(3):1035–45. 10.1007/s12253-018-0459-8  [DOI](https://doi.org/10.1007/s12253-018-0459-8) | [PMC free article](/articles/PMC6614130/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30066234/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pathol%20Oncol%20Res&title=Predictive%20Value%20of%20Single%20Nucleotide%20Polymorphisms%20of%20ERCC1,%20XPA,%20XPC,%20XPD%20and%20XPG%20Genes,%20Involved%20in%20NER%20Mechanism%20in%20Patients%20With%20Advanced%20NSCLC%20Treated%20With%20Cisplatin%20and%20Gemcitabine&author=R%20Mlak&author=P%20Krawczyk&author=I%20Homa-Mlak&author=T%20Powr%C3%B3zek&author=M%20Ciesielka&volume=25&issue=3&publication_year=2019&pages=1035-45&pmid=30066234&doi=10.1007/s12253-018-0459-8&)

22. Little J, Bradley L, Bray MS, Clyne M, Dorman J, Ellsworth DL, et al. Reporting, Appraising, and Integrating Data on Genotype Prevalence and Gene-Disease Associations. Am J Epidemiol (2002) 156(4):300–10. 10.1093/oxfordjournals.aje.a000179  [DOI](https://doi.org/10.1093/oxfordjournals.aje.a000179) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12181099/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Epidemiol&title=Reporting,%20Appraising,%20and%20Integrating%20Data%20on%20Genotype%20Prevalence%20and%20Gene-Disease%20Associations&author=J%20Little&author=L%20Bradley&author=MS%20Bray&author=M%20Clyne&author=J%20Dorman&volume=156&issue=4&publication_year=2002&pages=300-10&pmid=12181099&doi=10.1093/oxfordjournals.aje.a000179&)

23. National Center for Biotechnology Information (NCBI) Single Nucleotide Polymorphism Database (dbSNP) (2023). Available at: https://www.ncbi.nlm.nih.gov/snp/ .  [https://www.ncbi.nlm.nih.gov/snp/](https://www.ncbi.nlm.nih.gov/snp/) | [Google Scholar](https://scholar.google.com/scholar_lookup?National%20Center%20for%20Biotechnology%20Information%20(NCBI)%20Single%20Nucleotide%20Polymorphism%20Database%20(dbSNP)%20(2023).%20Available%20at:%20https://www.ncbi.nlm.nih.gov/snp/%20.)

24. Shim S, Kim J, Jung W, Shin IS, Bae JM. Meta-Analysis for Genome-Wide Association Studies Using Case-Control Design: Application and Practice. Epidemiol Health (2016) 38:e2016058. 10.4178/epih.e2016058  [DOI](https://doi.org/10.4178/epih.e2016058) | [PMC free article](/articles/PMC5309730/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28092928/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Epidemiol%20Health&title=Meta-Analysis%20for%20Genome-Wide%20Association%20Studies%20Using%20Case-Control%20Design:%20Application%20and%20Practice&author=S%20Shim&author=J%20Kim&author=W%20Jung&author=IS%20Shin&author=JM%20Bae&volume=38&publication_year=2016&pages=e2016058&pmid=28092928&doi=10.4178/epih.e2016058&)

25. Ensembl Variation Pathogenicity Predictions (2024). Available at: http://asia.ensembl.org/index.html .  [http://asia.ensembl.org/index.html](http://asia.ensembl.org/index.html) | [Google Scholar](https://scholar.google.com/scholar_lookup?Ensembl%20Variation%20Pathogenicity%20Predictions%20(2024).%20Available%20at:%20http://asia.ensembl.org/index.html%20.)

26. Grant CH, Gourley C. Relevant Cancer Diagnoses, Commonly Used Chemotherapy Agents and Their Biochemical Mechanisms of Action. In: Anderson RA, Spears N, editors. Cancer Treatment and the Ovary: Clinical and Laboratory Analysis of Ovarian Toxicity. Elsevier Inc; (2015). p. 21–33.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Cancer%20Treatment%20and%20the%20Ovary:%20Clinical%20and%20Laboratory%20Analysis%20of%20Ovarian%20Toxicity&author=CH%20Grant&author=C%20Gourley&publication_year=2015&)

27. Zhang S, Zhong X, Yuan H, Guo Y, Song D, Qi F, et al. Interfering in Apoptosis and DNA Repair of Cancer Cells to Conquer Cisplatin Resistance by Platinum(iv) Prodrugs. Chem Sci (2020) 11(15):3829–35. 10.1039/d0sc00197j  [DOI](https://doi.org/10.1039/d0sc00197j) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chem%20Sci&title=Interfering%20in%20Apoptosis%20and%20DNA%20Repair%20of%20Cancer%20Cells%20to%20Conquer%20Cisplatin%20Resistance%20by%20Platinum(iv)%20Prodrugs&author=S%20Zhang&author=X%20Zhong&author=H%20Yuan&author=Y%20Guo&author=D%20Song&volume=11&issue=15&publication_year=2020&pages=3829-35&doi=10.1039/d0sc00197j&)

28. Chen SH, Chang JY. New Insights Into Mechanisms of Cisplatin Resistance: From Tumor Cell to Microenvironment. Int J Mol Sci (2019) 20(17):4136. 10.3390/ijms20174136  [DOI](https://doi.org/10.3390/ijms20174136) | [PMC free article](/articles/PMC6747329/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31450627/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Mol%20Sci&title=New%20Insights%20Into%20Mechanisms%20of%20Cisplatin%20Resistance:%20From%20Tumor%20Cell%20to%20Microenvironment&author=SH%20Chen&author=JY%20Chang&volume=20&issue=17&publication_year=2019&pages=4136&pmid=31450627&doi=10.3390/ijms20174136&)

29. Yang Y, Xian L. The Association Between the GSTP1 A313G and GSTM1 Null/Present Polymorphisms and the Treatment Response of the Platinum-Based Chemotherapy in Non-Small Cell Lung Cancer (NSCLC) Patients: A Meta-Analysis. Tumour Biol (2014) 35(7):6791–9. 10.1007/s13277-014-1866-4  [DOI](https://doi.org/10.1007/s13277-014-1866-4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24729086/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Tumour%20Biol&title=The%20Association%20Between%20the%20GSTP1%20A313G%20and%20GSTM1%20Null/Present%20Polymorphisms%20and%20the%20Treatment%20Response%20of%20the%20Platinum-Based%20Chemotherapy%20in%20Non-Small%20Cell%20Lung%20Cancer%20(NSCLC)%20Patients:%20A%20Meta-Analysis&author=Y%20Yang&author=L%20Xian&volume=35&issue=7&publication_year=2014&pages=6791-9&pmid=24729086&doi=10.1007/s13277-014-1866-4&)

30. Xu TP, Shen H, Liu LX, Shu YQ. Association of ERCC1-C118T and -C8092A Polymorphisms With Lung Cancer Risk and Survival of Advanced-Stage Non-Small Cell Lung Cancer Patients Receiving Platinum-Based Chemotherapy: A Pooled Analysis Based on 39 Reports. Gene (2013) 526(2):265–74. 10.1016/j.gene.2013.05.021  [DOI](https://doi.org/10.1016/j.gene.2013.05.021) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23727606/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gene&title=Association%20of%20ERCC1-C118T%20and%20-C8092A%20Polymorphisms%20With%20Lung%20Cancer%20Risk%20and%20Survival%20of%20Advanced-Stage%20Non-Small%20Cell%20Lung%20Cancer%20Patients%20Receiving%20Platinum-Based%20Chemotherapy:%20A%20Pooled%20Analysis%20Based%20on%2039%20Reports&author=TP%20Xu&author=H%20Shen&author=LX%20Liu&author=YQ%20Shu&volume=526&issue=2&publication_year=2013&pages=265-74&pmid=23727606&doi=10.1016/j.gene.2013.05.021&)

31. Joerger M, Burgers SA, Baas P, Smit EF, Haitjema TJ, Bard MPL, et al. Germline Polymorphisms in Patients With Advanced Nonsmall Cell Lung Cancer Receiving First-Line Platinum-Gemcitabine Chemotherapy: A Prospective Clinical Study. Cancer (2012) 118(9):2466–75. 10.1002/cncr.26562  [DOI](https://doi.org/10.1002/cncr.26562) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22031394/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer&title=Germline%20Polymorphisms%20in%20Patients%20With%20Advanced%20Nonsmall%20Cell%20Lung%20Cancer%20Receiving%20First-Line%20Platinum-Gemcitabine%20Chemotherapy:%20A%20Prospective%20Clinical%20Study&author=M%20Joerger&author=SA%20Burgers&author=P%20Baas&author=EF%20Smit&author=TJ%20Haitjema&volume=118&issue=9&publication_year=2012&pages=2466-75&pmid=22031394&doi=10.1002/cncr.26562&)

32. Isla D, Sarries C, Rosell R, Alonso G, Domine M, Taron M, et al. Single Nucleotide Polymorphisms and Outcome in Docetaxel-Cisplatin-Treated Advanced Non-Small-Cell Lung Cancer. Ann Oncol (2004) 15(8):1194–203. 10.1093/annonc/mdh319  [DOI](https://doi.org/10.1093/annonc/mdh319) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15277258/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Oncol&title=Single%20Nucleotide%20Polymorphisms%20and%20Outcome%20in%20Docetaxel-Cisplatin-Treated%20Advanced%20Non-Small-Cell%20Lung%20Cancer&author=D%20Isla&author=C%20Sarries&author=R%20Rosell&author=G%20Alonso&author=M%20Domine&volume=15&issue=8&publication_year=2004&pages=1194-203&pmid=15277258&doi=10.1093/annonc/mdh319&)

33. Hai-bo W, Jing H, Li-hua S, Yun-yan Z, Fei-fei L, Min W, et al. A Meta-Analytic Review of ERCC1/MDR1 Polymorphism and Chemosensitivity to Platinum in Patients With Advanced Non-Small Cell Lung Cancer. Chin Med J (Engl) (2012) 125(16):2902–7. 10.3760/cma.j.issn.0366-6999.2012.16.019  [DOI](https://doi.org/10.3760/cma.j.issn.0366-6999.2012.16.019) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22932088/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chin%20Med%20J%20(Engl)&title=A%20Meta-Analytic%20Review%20of%20ERCC1/MDR1%20Polymorphism%20and%20Chemosensitivity%20to%20Platinum%20in%20Patients%20With%20Advanced%20Non-Small%20Cell%20Lung%20Cancer&author=W%20Hai-bo&author=H%20Jing&author=S%20Li-hua&author=Z%20Yun-yan&author=L%20Fei-fei&volume=125&issue=16&publication_year=2012&pages=2902-7&pmid=22932088&doi=10.3760/cma.j.issn.0366-6999.2012.16.019&)

34. Gao H, Ge RC, Liu HY, Wang Y, Yan S. Effect of ERCC1 Polymorphism on the Response to Chemotherapy and Clinical Outcome of Non-Small Cell Lung Cancer. Genet Mol Res (2014) 13(4):8997–9004. 10.4238/2014.October.31.14  [DOI](https://doi.org/10.4238/2014.October.31.14) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25366790/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genet%20Mol%20Res&title=Effect%20of%20ERCC1%20Polymorphism%20on%20the%20Response%20to%20Chemotherapy%20and%20Clinical%20Outcome%20of%20Non-Small%20Cell%20Lung%20Cancer&author=H%20Gao&author=RC%20Ge&author=HY%20Liu&author=Y%20Wang&author=S%20Yan&volume=13&issue=4&publication_year=2014&pages=8997-9004&pmid=25366790&doi=10.4238/2014.October.31.14&)

35. Sullivan I, Salazar J, Majem M, Pallarés C, Del Río E, Páez D, et al. Pharmacogenetics of the DNA Repair Pathways in Advanced Non-Small Cell Lung Cancer Patients Treated With Platinum-Based Chemotherapy. Cancer Lett (2014) 353(2):160–6. 10.1016/j.canlet.2014.07.023  [DOI](https://doi.org/10.1016/j.canlet.2014.07.023) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25069034/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Lett&title=Pharmacogenetics%20of%20the%20DNA%20Repair%20Pathways%20in%20Advanced%20Non-Small%20Cell%20Lung%20Cancer%20Patients%20Treated%20With%20Platinum-Based%20Chemotherapy&author=I%20Sullivan&author=J%20Salazar&author=M%20Majem&author=C%20Pallar%C3%A9s&author=E%20Del%20R%C3%ADo&volume=353&issue=2&publication_year=2014&pages=160-6&pmid=25069034&doi=10.1016/j.canlet.2014.07.023&)

36. Krawczyk P, Wojas-Krawczyk K, Mlak R, Kucharczyk T, Biernacka B, Milanowski J. Predictive Value of ERCC1 Single-Nucleotide Polymorphism in Patients Receiving Platinum-Based Chemotherapy for Locally-Advanced and Advanced Non-Small Cell Lung Cancer--A Pilot Study. Folia Histochem Cytobiol (2012) 50(1):80–6. 10.2478/18700  [DOI](https://doi.org/10.2478/18700) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22532140/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Folia%20Histochem%20Cytobiol&title=Predictive%20Value%20of%20ERCC1%20Single-Nucleotide%20Polymorphism%20in%20Patients%20Receiving%20Platinum-Based%20Chemotherapy%20for%20Locally-Advanced%20and%20Advanced%20Non-Small%20Cell%20Lung%20Cancer--A%20Pilot%20Study&author=P%20Krawczyk&author=K%20Wojas-Krawczyk&author=R%20Mlak&author=T%20Kucharczyk&author=B%20Biernacka&volume=50&issue=1&publication_year=2012&pages=80-6&pmid=22532140&doi=10.2478/18700&)

37. Cheng J, Ha M, Wang Y, Sun J, Chen J, Wang Y, et al. A C118T Polymorphism of ERCC1 and Response to Cisplatin Chemotherapy in Patients With Late-Stage Non-Small Cell Lung Cancer. J Cancer Res Clin Oncol (2012) 138(2):231–8. 10.1007/s00432-011-1090-1  [DOI](https://doi.org/10.1007/s00432-011-1090-1) | [PMC free article](/articles/PMC11824594/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22102173/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Cancer%20Res%20Clin%20Oncol&title=A%20C118T%20Polymorphism%20of%20ERCC1%20and%20Response%20to%20Cisplatin%20Chemotherapy%20in%20Patients%20With%20Late-Stage%20Non-Small%20Cell%20Lung%20Cancer&author=J%20Cheng&author=M%20Ha&author=Y%20Wang&author=J%20Sun&author=J%20Chen&volume=138&issue=2&publication_year=2012&pages=231-8&pmid=22102173&doi=10.1007/s00432-011-1090-1&)

38. Su D, Ma S, Liu P, Jiang Z, Lv W, Zhang Y, et al. Genetic Polymorphisms and Treatment Response in Advanced Non-Small Cell Lung Cancer. Lung Cancer (2007) 56(2):281–8. 10.1016/j.lungcan.2006.12.002  [DOI](https://doi.org/10.1016/j.lungcan.2006.12.002) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17222938/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lung%20Cancer&title=Genetic%20Polymorphisms%20and%20Treatment%20Response%20in%20Advanced%20Non-Small%20Cell%20Lung%20Cancer&author=D%20Su&author=S%20Ma&author=P%20Liu&author=Z%20Jiang&author=W%20Lv&volume=56&issue=2&publication_year=2007&pages=281-8&pmid=17222938&doi=10.1016/j.lungcan.2006.12.002&)

39. Das S, Naher L, Das Aka T, Aziz A, Shabnaz S, Shahriar M, et al. The ECCR1 Rs11615, ERCC4 Rs2276466, XPC Rs2228000 and XPC Rs2228001 Polymorphisms Increase the Cervical Cancer Risk and Aggressiveness in the Bangladeshi Population . Heliyon (2017) 7(1):e05919. 10.1016/j.heliyon.2021.e05919  [DOI](https://doi.org/10.1016/j.heliyon.2021.e05919) | [PMC free article](/articles/PMC7809183/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33490679/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Heliyon&title=The%20ECCR1%20Rs11615,%20ERCC4%20Rs2276466,%20XPC%20Rs2228000%20and%20XPC%20Rs2228001%20Polymorphisms%20Increase%20the%20Cervical%20Cancer%20Risk%20and%20Aggressiveness%20in%20the%20Bangladeshi%20Population&author=S%20Das&author=L%20Naher&author=T%20Das%20Aka&author=A%20Aziz&author=S%20Shabnaz&volume=7&issue=1&publication_year=2017&pages=e05919&pmid=33490679&doi=10.1016/j.heliyon.2021.e05919&)

40. Woelfelschneider A, Popanda O, Lilla C, Linseisen J, Mayer C, Celebi O, et al. A Distinct ERCC1 Haplotype Is Associated With mRNA Expression Levels in Prostate Cancer Patients. Carcinogenesis (2008) 29(9):1758–64. 10.1093/carcin/bgn067  [DOI](https://doi.org/10.1093/carcin/bgn067) | [PMC free article](/articles/PMC7110227/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18332046/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Carcinogenesis&title=A%20Distinct%20ERCC1%20Haplotype%20Is%20Associated%20With%20mRNA%20Expression%20Levels%20in%20Prostate%20Cancer%20Patients&author=A%20Woelfelschneider&author=O%20Popanda&author=C%20Lilla&author=J%20Linseisen&author=C%20Mayer&volume=29&issue=9&publication_year=2008&pages=1758-64&pmid=18332046&doi=10.1093/carcin/bgn067&)

41. Gao R, Reece K, Sissung T, Reed E, Price D, Figg W. The ERCC1 N118N Polymorphism Does Not Change Cellular ERCC1 Protein Expression or Platinum Sensitivity. Mutat Res (2011) 708(1–2):21–7. 10.1016/j.mrfmmm.2011.01.002  [DOI](https://doi.org/10.1016/j.mrfmmm.2011.01.002) | [PMC free article](/articles/PMC3057957/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21315089/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mutat%20Res&title=The%20ERCC1%20N118N%20Polymorphism%20Does%20Not%20Change%20Cellular%20ERCC1%20Protein%20Expression%20or%20Platinum%20Sensitivity&author=R%20Gao&author=K%20Reece&author=T%20Sissung&author=E%20Reed&author=D%20Price&volume=708&issue=1%E2%80%932&publication_year=2011&pages=21-7&pmid=21315089&doi=10.1016/j.mrfmmm.2011.01.002&)

42. Yu D, Shi J, Sun T, Du X, Liu L, Zhang X, et al. Pharmacogenetic Role of ERCC1 Genetic Variants in Treatment Response of Platinum-Based Chemotherapy Among Advanced Non-Small Cell Lung Cancer Patients. Tumour Biol (2012) 33(3):877–84. 10.1007/s13277-011-0314-y  [DOI](https://doi.org/10.1007/s13277-011-0314-y) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22249976/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Tumour%20Biol&title=Pharmacogenetic%20Role%20of%20ERCC1%20Genetic%20Variants%20in%20Treatment%20Response%20of%20Platinum-Based%20Chemotherapy%20Among%20Advanced%20Non-Small%20Cell%20Lung%20Cancer%20Patients&author=D%20Yu&author=J%20Shi&author=T%20Sun&author=X%20Du&author=L%20Liu&volume=33&issue=3&publication_year=2012&pages=877-84&pmid=22249976&doi=10.1007/s13277-011-0314-y&)

43. Yang Y, Xian L. The Association Between the ERCC1/2 Polymorphisms and the Clinical Outcomes of the Platinum-Based Chemotherapy in Non-Small Cell Lung Cancer (NSCLC): A Systematic Review and Meta-Analysis. Tumour Biol (2014) 35(4):2905–21. 10.1007/s13277-013-1493-5  [DOI](https://doi.org/10.1007/s13277-013-1493-5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24338713/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Tumour%20Biol&title=The%20Association%20Between%20the%20ERCC1/2%20Polymorphisms%20and%20the%20Clinical%20Outcomes%20of%20the%20Platinum-Based%20Chemotherapy%20in%20Non-Small%20Cell%20Lung%20Cancer%20(NSCLC):%20A%20Systematic%20Review%20and%20Meta-Analysis&author=Y%20Yang&author=L%20Xian&volume=35&issue=4&publication_year=2014&pages=2905-21&pmid=24338713&doi=10.1007/s13277-013-1493-5&)

44. Huang D, Zhou Y. Nucleotide Excision Repair Gene Polymorphisms and Prognosis of Non-Small Cell Lung Cancer Patients Receiving Platinum-Based Chemotherapy: A Meta-Analysis Based on 44 Studies. Biomed Rep (2014) 2(4):452–62. 10.3892/br.2014.282  [DOI](https://doi.org/10.3892/br.2014.282) | [PMC free article](/articles/PMC4051473/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24944789/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biomed%20Rep&title=Nucleotide%20Excision%20Repair%20Gene%20Polymorphisms%20and%20Prognosis%20of%20Non-Small%20Cell%20Lung%20Cancer%20Patients%20Receiving%20Platinum-Based%20Chemotherapy:%20A%20Meta-Analysis%20Based%20on%2044%20Studies&author=D%20Huang&author=Y%20Zhou&volume=2&issue=4&publication_year=2014&pages=452-62&pmid=24944789&doi=10.3892/br.2014.282&)

45. Liao WYY, Ho CCC, Tsai THH, Chen KYY, Shih JYY, Yu CJJ. Combined Effect of ERCC1 and ERCC2 Polymorphisms on Overall Survival in Non-Squamous Non-Small-Cell Lung Cancer Patients Treated With First-Line Pemetrexed/Platinum. Lung Cancer (2018) 118:90–6. 10.1016/j.lungcan.2018.01.011  [DOI](https://doi.org/10.1016/j.lungcan.2018.01.011) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29572009/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lung%20Cancer&title=Combined%20Effect%20of%20ERCC1%20and%20ERCC2%20Polymorphisms%20on%20Overall%20Survival%20in%20Non-Squamous%20Non-Small-Cell%20Lung%20Cancer%20Patients%20Treated%20With%20First-Line%20Pemetrexed/Platinum&author=WYY%20Liao&author=CCC%20Ho&author=THH%20Tsai&author=KYY%20Chen&author=JYY%20Shih&volume=118&publication_year=2018&pages=90-6&pmid=29572009&doi=10.1016/j.lungcan.2018.01.011&)

46. Huang SJ, Wang YF, Jin ZY, Sun JY, Guo ZL. Role of ERCC1 Variants in Response to Chemotherapy and Clinical Outcome of Advanced Non-Small Cell Lung Cancer. Tumour Biol (2014) 35(5):4023–9. 10.1007/s13277-013-1526-0  [DOI](https://doi.org/10.1007/s13277-013-1526-0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24370899/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Tumour%20Biol&title=Role%20of%20ERCC1%20Variants%20in%20Response%20to%20Chemotherapy%20and%20Clinical%20Outcome%20of%20Advanced%20Non-Small%20Cell%20Lung%20Cancer&author=SJ%20Huang&author=YF%20Wang&author=ZY%20Jin&author=JY%20Sun&author=ZL%20Guo&volume=35&issue=5&publication_year=2014&pages=4023-9&pmid=24370899&doi=10.1007/s13277-013-1526-0&)

47. Chen P, Wiencke J, Aldape K, Kesler-Diaz A, Miike R, Kelsey K, et al. Association of an ERCC1 Polymorphism With Adult-Onset Glioma. Cancer Epidemiol Biomarkers Prev (2000) 9(8):843–7.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/10952103/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Epidemiol%20Biomarkers%20Prev&title=Association%20of%20an%20ERCC1%20Polymorphism%20With%20Adult-Onset%20Glioma&author=P%20Chen&author=J%20Wiencke&author=K%20Aldape&author=A%20Kesler-Diaz&author=R%20Miike&volume=9&issue=8&publication_year=2000&pages=843-7&pmid=10952103&)

48. Yu T, Xue P, Cui S, Zhang L, Zhang G, Xiao M, et al. Rs3212986 Polymorphism, a Possible Biomarker to Predict Smoking-Related Lung Cancer, Alters DNA Repair Capacity via Regulating ERCC1 Expression. Cancer Med (2018) 7(12):6317–30. 10.1002/cam4.1842  [DOI](https://doi.org/10.1002/cam4.1842) | [PMC free article](/articles/PMC6308093/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30453383/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Med&title=Rs3212986%20Polymorphism,%20a%20Possible%20Biomarker%20to%20Predict%20Smoking-Related%20Lung%20Cancer,%20Alters%20DNA%20Repair%20Capacity%20via%20Regulating%20ERCC1%20Expression&author=T%20Yu&author=P%20Xue&author=S%20Cui&author=L%20Zhang&author=G%20Zhang&volume=7&issue=12&publication_year=2018&pages=6317-30&pmid=30453383&doi=10.1002/cam4.1842&)

49. Wang LY, Cui JJ, Liu JY, Guo AX, Zhao ZY, Liu YZ, et al. Gene-Gene and Gene-Environment Interaction Data for Platinum-Based Chemotherapy in Non-Small Cell Lung Cancer. Scientific data (2018) 5:180284. 10.1038/sdata.2018.284  [DOI](https://doi.org/10.1038/sdata.2018.284) | [PMC free article](/articles/PMC6289114/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30531820/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Scientific%20data&title=Gene-Gene%20and%20Gene-Environment%20Interaction%20Data%20for%20Platinum-Based%20Chemotherapy%20in%20Non-Small%20Cell%20Lung%20Cancer&author=LY%20Wang&author=JJ%20Cui&author=JY%20Liu&author=AX%20Guo&author=ZY%20Zhao&volume=5&publication_year=2018&pages=180284&pmid=30531820&doi=10.1038/sdata.2018.284&)

50. Ezzeldin N, El-Lebedy D, Mohammed A. Gene-Environment and Gene-Gene Interactions Between CHRNA3 Rs1051730, XRCC1 Rs25487, and ERCC1 Rs735482 Variants Highly Elevate the Risk of Lung Cancer. Egypt J Med Hum Genet (2019) 20(1):23–7. 10.1186/s43042-019-0034-1  [DOI](https://doi.org/10.1186/s43042-019-0034-1) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Egypt%20J%20Med%20Hum%20Genet&title=Gene-Environment%20and%20Gene-Gene%20Interactions%20Between%20CHRNA3%20Rs1051730,%20XRCC1%20Rs25487,%20and%20ERCC1%20Rs735482%20Variants%20Highly%20Elevate%20the%20Risk%20of%20Lung%20Cancer&author=N%20Ezzeldin&author=D%20El-Lebedy&author=A%20Mohammed&volume=20&issue=1&publication_year=2019&pages=23-7&doi=10.1186/s43042-019-0034-1&)

51. Han Y, Liu J, Sun M, Zhang Z, Liu C, Sun Y. A Significant Statistical Advancement on the Predictive Values of ERCC1 Polymorphisms for Clinical Outcomes of Platinum-Based Chemotherapy in Non-Small Cell Lung Cancer: An Updated Meta-Analysis. Dis Markers (2016) 2016:7643981. 10.1155/2016/7643981  [DOI](https://doi.org/10.1155/2016/7643981) | [PMC free article](/articles/PMC4745870/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27057082/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Dis%20Markers&title=A%20Significant%20Statistical%20Advancement%20on%20the%20Predictive%20Values%20of%20ERCC1%20Polymorphisms%20for%20Clinical%20Outcomes%20of%20Platinum-Based%20Chemotherapy%20in%20Non-Small%20Cell%20Lung%20Cancer:%20An%20Updated%20Meta-Analysis&author=Y%20Han&author=J%20Liu&author=M%20Sun&author=Z%20Zhang&author=C%20Liu&volume=2016&publication_year=2016&pages=7643981&pmid=27057082&doi=10.1155/2016/7643981&)

52. Zhou M, Ding YJ, Feng Y, Zhang QR, Xiang Y, Wan HY. Association of Xeroderma Pigmentosum Group D (Asp312Asn, Lys751Gln) and Cytidine Deaminase (Lys27Gln, Ala70Thr) Polymorphisms With Outcome in Chinese Non-Small Cell Lung Cancer Patients Treated With Cisplatin-Gemcitabine. Genet Mol Res (2014) 13(2):3310–8. 10.4238/2014.April.29.9  [DOI](https://doi.org/10.4238/2014.April.29.9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24841663/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genet%20Mol%20Res&title=Association%20of%20Xeroderma%20Pigmentosum%20Group%20D%20(Asp312Asn,%20Lys751Gln)%20and%20Cytidine%20Deaminase%20(Lys27Gln,%20Ala70Thr)%20Polymorphisms%20With%20Outcome%20in%20Chinese%20Non-Small%20Cell%20Lung%20Cancer%20Patients%20Treated%20With%20Cisplatin-Gemcitabine&author=M%20Zhou&author=YJ%20Ding&author=Y%20Feng&author=QR%20Zhang&author=Y%20Xiang&volume=13&issue=2&publication_year=2014&pages=3310-8&pmid=24841663&doi=10.4238/2014.April.29.9&)

53. Spitz MR, Wu X, Wang Y, Wang LE, Shete S, Amos CI, et al. Modulation of Nucleotide Excision Repair Capacity by XPD Polymorphisms in Lung Cancer Patients. Cancer Res (2001) 61(4):1354–7.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/11245433/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=Modulation%20of%20Nucleotide%20Excision%20Repair%20Capacity%20by%20XPD%20Polymorphisms%20in%20Lung%20Cancer%20Patients&author=MR%20Spitz&author=X%20Wu&author=Y%20Wang&author=LE%20Wang&author=S%20Shete&volume=61&issue=4&publication_year=2001&pages=1354-7&pmid=11245433&)

54. Li YK, Xu Q, Sun LP, Gong YH, Jing JJ, Xing CZ, et al. Nucleotide Excision Repair Pathway Gene Polymorphisms Are Associated With Risk and Prognosis of Colorectal Cancer. World J Gastroenterol (2020) 26(3):307–23. 10.3748/wjg.v26.i3.307  [DOI](https://doi.org/10.3748/wjg.v26.i3.307) | [PMC free article](/articles/PMC6969885/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31988591/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=World%20J%20Gastroenterol&title=Nucleotide%20Excision%20Repair%20Pathway%20Gene%20Polymorphisms%20Are%20Associated%20With%20Risk%20and%20Prognosis%20of%20Colorectal%20Cancer&author=YK%20Li&author=Q%20Xu&author=LP%20Sun&author=YH%20Gong&author=JJ%20Jing&volume=26&issue=3&publication_year=2020&pages=307-23&pmid=31988591&doi=10.3748/wjg.v26.i3.307&)

55. Gu AQQ, Wang WMM, Chen WYY, Shi CLL, Lu JHH, Han JQQ. XRCC1 Genetic Polymorphisms and Sensitivity to Platinum-Based Drugs in Non-Small Cell Lung Cancer: An Update Meta-Analysis Based on 4708 Subjects. Int J Clin Exp Med (2015) 8(1):145–54.  [PMC free article](/articles/PMC4358438/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25784983/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Clin%20Exp%20Med&title=XRCC1%20Genetic%20Polymorphisms%20and%20Sensitivity%20to%20Platinum-Based%20Drugs%20in%20Non-Small%20Cell%20Lung%20Cancer:%20An%20Update%20Meta-Analysis%20Based%20on%204708%20Subjects&author=AQQ%20Gu&author=WMM%20Wang&author=WYY%20Chen&author=CLL%20Shi&author=JHH%20Lu&volume=8&issue=1&publication_year=2015&pages=145-54&pmid=25784983&)

56. Li L, Wan C, Wen FQ. Polymorphisms in the XRCC1 Gene Are Associated With Treatment Response to Platinum Chemotherapy in Advanced Non-Small Cell Lung Cancer Patients Based on Meta-Analysis. Genet Mol Res (2014) 13(2):3772–86. 10.4238/2014.May.16.1  [DOI](https://doi.org/10.4238/2014.May.16.1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24938464/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genet%20Mol%20Res&title=Polymorphisms%20in%20the%20XRCC1%20Gene%20Are%20Associated%20With%20Treatment%20Response%20to%20Platinum%20Chemotherapy%20in%20Advanced%20Non-Small%20Cell%20Lung%20Cancer%20Patients%20Based%20on%20Meta-Analysis&author=L%20Li&author=C%20Wan&author=FQ%20Wen&volume=13&issue=2&publication_year=2014&pages=3772-86&pmid=24938464&doi=10.4238/2014.May.16.1&)

57. Wu J, Liu J, Zhou Y, Ying J, Zou H, Guo S, et al. Predictive Value of XRCC1 Gene Polymorphisms on Platinum-Based Chemotherapy in Advanced Non-Small Cell Lung Cancer Patients: A Systematic Review and Meta-Analysis. Clin Cancer Res (2012) 18(14):3972–81. 10.1158/1078-0432.CCR-11-1531  [DOI](https://doi.org/10.1158/1078-0432.CCR-11-1531) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22705987/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=Predictive%20Value%20of%20XRCC1%20Gene%20Polymorphisms%20on%20Platinum-Based%20Chemotherapy%20in%20Advanced%20Non-Small%20Cell%20Lung%20Cancer%20Patients:%20A%20Systematic%20Review%20and%20Meta-Analysis&author=J%20Wu&author=J%20Liu&author=Y%20Zhou&author=J%20Ying&author=H%20Zou&volume=18&issue=14&publication_year=2012&pages=3972-81&pmid=22705987&doi=10.1158/1078-0432.CCR-11-1531&)

58. Yuan Z, Li J, Hu R, Jiao Y, Han Y, Weng Q. Predictive Assessment in Pharmacogenetics of XRCC1 Gene on Clinical Outcomes of Advanced Lung Cancer Patients Treated With Platinum-Based Chemotherapy. Scientific Rep (2015) 5(1):16482–21. 10.1038/srep16482  [DOI](https://doi.org/10.1038/srep16482) | [PMC free article](/articles/PMC4653744/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26585370/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Scientific%20Rep&title=Predictive%20Assessment%20in%20Pharmacogenetics%20of%20XRCC1%20Gene%20on%20Clinical%20Outcomes%20of%20Advanced%20Lung%20Cancer%20Patients%20Treated%20With%20Platinum-Based%20Chemotherapy&author=Z%20Yuan&author=J%20Li&author=R%20Hu&author=Y%20Jiao&author=Y%20Han&volume=5&issue=1&publication_year=2015&pages=16482-21&pmid=26585370&doi=10.1038/srep16482&)

59. Ke HG, Li J, Shen Y, You QS, Yan Y, Dong HX, et al. Prognostic Significance of GSTP1, XRCC1 and XRCC3 Polymorphisms in Non-Small Cell Lung Cancer Patients. Asian Pac J Cancer Prev (2012) 13(9):4413–6. 10.7314/apjcp.2012.13.9.4413  [DOI](https://doi.org/10.7314/apjcp.2012.13.9.4413) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23167352/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Asian%20Pac%20J%20Cancer%20Prev&title=Prognostic%20Significance%20of%20GSTP1,%20XRCC1%20and%20XRCC3%20Polymorphisms%20in%20Non-Small%20Cell%20Lung%20Cancer%20Patients&author=HG%20Ke&author=J%20Li&author=Y%20Shen&author=QS%20You&author=Y%20Yan&volume=13&issue=9&publication_year=2012&pages=4413-6&pmid=23167352&doi=10.7314/apjcp.2012.13.9.4413&)

60. Yuan P, Miao XP, Zhang XM, Wang ZH, Tan W, Sun Y, et al. XRCC1 and XPD Genetic Polymorphisms Predict Clinical Responses to Platinum-Based Chemotherapy in Advanced Non-Small Cell Lung Cancer. Zhonghua Zhong Liu Za Zhi (2006) 28(3):196–9.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/16875604/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Zhonghua%20Zhong%20Liu%20Za%20Zhi&title=XRCC1%20and%20XPD%20Genetic%20Polymorphisms%20Predict%20Clinical%20Responses%20to%20Platinum-Based%20Chemotherapy%20in%20Advanced%20Non-Small%20Cell%20Lung%20Cancer&author=P%20Yuan&author=XP%20Miao&author=XM%20Zhang&author=ZH%20Wang&author=W%20Tan&volume=28&issue=3&publication_year=2006&pages=196-9&pmid=16875604&)

61. Zhang N, Ouyang Y, Chang J, Liu P, Tian X, Yu J. Pharmacogenetic Association Between XRCC1 Polymorphisms and Response to Platinum-Based Chemotherapy in Asian Patients With NSCLC: A Meta-Analysis. Biomed Res Int (2020) 2020:3520764. 10.1155/2020/3520764  [DOI](https://doi.org/10.1155/2020/3520764) | [PMC free article](/articles/PMC7603545/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33150172/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biomed%20Res%20Int&title=Pharmacogenetic%20Association%20Between%20XRCC1%20Polymorphisms%20and%20Response%20to%20Platinum-Based%20Chemotherapy%20in%20Asian%20Patients%20With%20NSCLC:%20A%20Meta-Analysis&author=N%20Zhang&author=Y%20Ouyang&author=J%20Chang&author=P%20Liu&author=X%20Tian&volume=2020&publication_year=2020&pages=3520764&pmid=33150172&doi=10.1155/2020/3520764&)

62. Hong CY, Xu Q, Yue Z, Zhang Y, Yuan Y. Correlation of the Sensitivity to Vinorelbine Plus Cisplatin (NP) Chemotherapy With Polymorphism in the DNA Repair Gene XRCC1 in Non-Small Lung Cancer. Chin J Cancer (2009) 28(12):53–9. 10.5732/cjc.009.10139  [DOI](https://doi.org/10.5732/cjc.009.10139) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19958624/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chin%20J%20Cancer&title=Correlation%20of%20the%20Sensitivity%20to%20Vinorelbine%20Plus%20Cisplatin%20(NP)%20Chemotherapy%20With%20Polymorphism%20in%20the%20DNA%20Repair%20Gene%20XRCC1%20in%20Non-Small%20Lung%20Cancer&author=CY%20Hong&author=Q%20Xu&author=Z%20Yue&author=Y%20Zhang&author=Y%20Yuan&volume=28&issue=12&publication_year=2009&pages=53-9&pmid=19958624&doi=10.5732/cjc.009.10139&)

63. Sun X, Li F, Sun N, Shukui Q, Baoan C, Jifeng F, et al. Polymorphisms in XRCC1 and XPG and Response to Platinum-Based Chemotherapy in Advanced Non-Small Cell Lung Cancer Patients. Lung Cancer (2009) 65(2):230–6. 10.1016/j.lungcan.2008.11.014  [DOI](https://doi.org/10.1016/j.lungcan.2008.11.014) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19157633/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lung%20Cancer&title=Polymorphisms%20in%20XRCC1%20and%20XPG%20and%20Response%20to%20Platinum-Based%20Chemotherapy%20in%20Advanced%20Non-Small%20Cell%20Lung%20Cancer%20Patients&author=X%20Sun&author=F%20Li&author=N%20Sun&author=Q%20Shukui&author=C%20Baoan&volume=65&issue=2&publication_year=2009&pages=230-6&pmid=19157633&doi=10.1016/j.lungcan.2008.11.014&)

64. Zhao R, Chen G. Role of GSTP1 Ile105Val and XRCC1 Arg194Trp, Arg280His and Arg399Gln Gene Polymorphisms in the Clinical Outcome of Advanced Non-Small Cell Lung Cancer. Int J Clin Exp Pathol (2015) 8(11):14909–16.  [PMC free article](/articles/PMC4713607/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26823821/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Clin%20Exp%20Pathol&title=Role%20of%20GSTP1%20Ile105Val%20and%20XRCC1%20Arg194Trp,%20Arg280His%20and%20Arg399Gln%20Gene%20Polymorphisms%20in%20the%20Clinical%20Outcome%20of%20Advanced%20Non-Small%20Cell%20Lung%20Cancer&author=R%20Zhao&author=G%20Chen&volume=8&issue=11&publication_year=2015&pages=14909-16&pmid=26823821&)

65. Wang ZH, Miao XP, Tan W, Zhang XR, Xu BH, Lin DX. Single Nucleotide Polymorphisms in XRCC1 and Clinical Response to Platin-Based Chemotherapy in Advanced Non-Small Cell Lung Cancer. Chin J Cancer (2004) 23(8):865–8.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/15301704/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chin%20J%20Cancer&title=Single%20Nucleotide%20Polymorphisms%20in%20XRCC1%20and%20Clinical%20Response%20to%20Platin-Based%20Chemotherapy%20in%20Advanced%20Non-Small%20Cell%20Lung%20Cancer&author=ZH%20Wang&author=XP%20Miao&author=W%20Tan&author=XR%20Zhang&author=BH%20Xu&volume=23&issue=8&publication_year=2004&pages=865-8&pmid=15301704&)

66. Nicoloso MS, Sun H, Spizzo R, Kim H, Wickramasinghe P, Shimizu M, et al. Single Nucleotide Polymorphisms Inside Microrna Target Sites Influence Tumor Susceptibility. Cancer Res (2010) 70(7):2789–98. 10.1158/0008-5472.CAN-09-3541  [DOI](https://doi.org/10.1158/0008-5472.CAN-09-3541) | [PMC free article](/articles/PMC2853025/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20332227/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=Single%20Nucleotide%20Polymorphisms%20Inside%20Microrna%20Target%20Sites%20Influence%20Tumor%20Susceptibility&author=MS%20Nicoloso&author=H%20Sun&author=R%20Spizzo&author=H%20Kim&author=P%20Wickramasinghe&volume=70&issue=7&publication_year=2010&pages=2789-98&pmid=20332227&doi=10.1158/0008-5472.CAN-09-3541&)

67. Francis AM, Ramya R, Ganesan N, Paul SF, Munirajan A, Kumarasamy P, et al. Breast Cancer Susceptibility Gene in Base Excision Repair Pathway in a Southern Indian Population. J Clin Diagn Res (2018) 12(09). 10.7860/jcdr/2018/36975.12000  [DOI](https://doi.org/10.7860/jcdr/2018/36975.12000) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Diagn%20Res&title=Breast%20Cancer%20Susceptibility%20Gene%20in%20Base%20Excision%20Repair%20Pathway%20in%20a%20Southern%20Indian%20Population&author=AM%20Francis&author=R%20Ramya&author=N%20Ganesan&author=SF%20Paul&author=A%20Munirajan&volume=12&issue=09&publication_year=2018&doi=10.7860/jcdr/2018/36975.12000&)

68. Santarpia M, Ramirez JL, de Aguirre I, Garrido P, Pérez Cano M, Queralt C, et al. Correlation of DNA Repair Gene Polymorphisms With Clinical Outcome in Patients With Locally Advanced Non-Small-Cell Lung Cancer Receiving Induction Chemotherapy Followed by Surgery. Clin Lung Cancer (2017) 18(2):178–88.e4. 10.1016/j.cllc.2016.08.007  [DOI](https://doi.org/10.1016/j.cllc.2016.08.007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27908619/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Lung%20Cancer&title=Correlation%20of%20DNA%20Repair%20Gene%20Polymorphisms%20With%20Clinical%20Outcome%20in%20Patients%20With%20Locally%20Advanced%20Non-Small-Cell%20Lung%20Cancer%20Receiving%20Induction%20Chemotherapy%20Followed%20by%20Surgery&author=M%20Santarpia&author=JL%20Ramirez&author=I%20de%20Aguirre&author=P%20Garrido&author=M%20P%C3%A9rez%20Cano&volume=18&issue=2&publication_year=2017&pages=178-88.e4&pmid=27908619&doi=10.1016/j.cllc.2016.08.007&)

69. Qiu M, Yang X, Hu J, Ding X, Jiang F, Yin R, et al. Predictive Value of XPD Polymorphisms on Platinum-Based Chemotherapy in Non-Small Cell Lung Cancer: A Systematic Review and Meta-Analysis. PLoS One (2013) 8(8):e72251. 10.1371/journal.pone.0072251  [DOI](https://doi.org/10.1371/journal.pone.0072251) | [PMC free article](/articles/PMC3747109/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23977265/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Predictive%20Value%20of%20XPD%20Polymorphisms%20on%20Platinum-Based%20Chemotherapy%20in%20Non-Small%20Cell%20Lung%20Cancer:%20A%20Systematic%20Review%20and%20Meta-Analysis&author=M%20Qiu&author=X%20Yang&author=J%20Hu&author=X%20Ding&author=F%20Jiang&volume=8&issue=8&publication_year=2013&pages=e72251&pmid=23977265&doi=10.1371/journal.pone.0072251&)

70. Seker H, Butkiewicz D, Bowman E, Rusin M, Hedayati M, Grossman L, et al. Functional Significance of XPD Polymorphic Variants: Attenuated Apoptosis in Human Lymphoblastoid Cells With the XPD 312 Asp/Asp Genotype. Cancer Res (2001) 61(20):7430–4.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/11606376/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=Functional%20Significance%20of%20XPD%20Polymorphic%20Variants:%20Attenuated%20Apoptosis%20in%20Human%20Lymphoblastoid%20Cells%20With%20the%20XPD%20312%20Asp/Asp%20Genotype&author=H%20Seker&author=D%20Butkiewicz&author=E%20Bowman&author=M%20Rusin&author=M%20Hedayati&volume=61&issue=20&publication_year=2001&pages=7430-4&pmid=11606376&)

71. Yuan P, Miao XP, Zhang XM, Wang ZH, Tan W, Sun Y, et al. Polymorphisms in Nucleotide Excision Repair Genes XPC and XPD and Clinical Responses to Platinum-Based Chemotherapy in Advanced Non-Small Cell Lung Cancer. Zhonghua Yi Xue Za Zhi (2005) 85(14):972–5. 10.3760/j:issn:0376-2491.2005.14.011  [DOI](https://doi.org/10.3760/j:issn:0376-2491.2005.14.011) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16061005/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Zhonghua%20Yi%20Xue%20Za%20Zhi&title=Polymorphisms%20in%20Nucleotide%20Excision%20Repair%20Genes%20XPC%20and%20XPD%20and%20Clinical%20Responses%20to%20Platinum-Based%20Chemotherapy%20in%20Advanced%20Non-Small%20Cell%20Lung%20Cancer&author=P%20Yuan&author=XP%20Miao&author=XM%20Zhang&author=ZH%20Wang&author=W%20Tan&volume=85&issue=14&publication_year=2005&pages=972-5&pmid=16061005&doi=10.3760/j:issn:0376-2491.2005.14.011&)

72. Yuan P, Miao XP, Zhang XM, Wang ZH, Tan W, Zhang XR, et al. Correlation of Genetic Polymorphisms in Nucleotide Excision Repair System to Sensitivity of Advanced Non-Small Cell Lung Cancer Patients to Platinum-Based Chemotherapy. Chin J Cancer (2005) 24(12):1510–3.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/16351803/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chin%20J%20Cancer&title=Correlation%20of%20Genetic%20Polymorphisms%20in%20Nucleotide%20Excision%20Repair%20System%20to%20Sensitivity%20of%20Advanced%20Non-Small%20Cell%20Lung%20Cancer%20Patients%20to%20Platinum-Based%20Chemotherapy&author=P%20Yuan&author=XP%20Miao&author=XM%20Zhang&author=ZH%20Wang&author=W%20Tan&volume=24&issue=12&publication_year=2005&pages=1510-3&pmid=16351803&)

73. Lee GY, Jang JS, Lee SY, Jeon HS, Kim KM, Choi JE, et al. XPC Polymorphisms and Lung Cancer Risk. Int J Cancer (2005) 115(5):807–13. 10.1002/ijc.20900  [DOI](https://doi.org/10.1002/ijc.20900) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15729698/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Cancer&title=XPC%20Polymorphisms%20and%20Lung%20Cancer%20Risk&author=GY%20Lee&author=JS%20Jang&author=SY%20Lee&author=HS%20Jeon&author=KM%20Kim&volume=115&issue=5&publication_year=2005&pages=807-13&pmid=15729698&doi=10.1002/ijc.20900&)

74. Khan SSG, Metter EJE, Tarone RRE, Bohr VAV, Grossman L, Hedayati M, et al. A New Xeroderma Pigmentosum Group C Poly(AT) Insertion/Deletion Polymorphism. Carcinogenesis (2000) 21(10):1821–5. 10.1093/carcin/21.10.1821  [DOI](https://doi.org/10.1093/carcin/21.10.1821) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11023539/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Carcinogenesis&title=A%20New%20Xeroderma%20Pigmentosum%20Group%20C%20Poly(AT)%20Insertion/Deletion%20Polymorphism&author=SSG%20Khan&author=EJE%20Metter&author=RRE%20Tarone&author=VAV%20Bohr&author=L%20Grossman&volume=21&issue=10&publication_year=2000&pages=1821-5&pmid=11023539&doi=10.1093/carcin/21.10.1821&)

75. Tibaldi C, Camerini A, Tiseo M, Mazzoni F, Barbieri F, Vittimberga I, et al. Cytidine Deaminase Enzymatic Activity Is a Prognostic Biomarker in Gemcitabine/Platinum-Treated Advanced Non-Small-Cell Lung Cancer: A Prospective Validation Study. Br J Cancer (2018) 119(11):1326–31. 10.1038/s41416-018-0307-3  [DOI](https://doi.org/10.1038/s41416-018-0307-3) | [PMC free article](/articles/PMC6265283/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30405211/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Cancer&title=Cytidine%20Deaminase%20Enzymatic%20Activity%20Is%20a%20Prognostic%20Biomarker%20in%20Gemcitabine/Platinum-Treated%20Advanced%20Non-Small-Cell%20Lung%20Cancer:%20A%20Prospective%20Validation%20Study&author=C%20Tibaldi&author=A%20Camerini&author=M%20Tiseo&author=F%20Mazzoni&author=F%20Barbieri&volume=119&issue=11&publication_year=2018&pages=1326-31&pmid=30405211&doi=10.1038/s41416-018-0307-3&)

76. Masters GA, Temin S, Azzoli CG, Giaccone G, Baker SJ, Brahmer JR, et al. Systemic Therapy for Stage IV Non-Small-Cell Lung Cancer: American Society of Clinical Oncology Clinical Practice Guideline Update. J Clin Oncol (2015) 33(30):3488–515. 10.1200/JCO.2015.62.1342  [DOI](https://doi.org/10.1200/JCO.2015.62.1342) | [PMC free article](/articles/PMC5019421/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26324367/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Systemic%20Therapy%20for%20Stage%20IV%20Non-Small-Cell%20Lung%20Cancer:%20American%20Society%20of%20Clinical%20Oncology%20Clinical%20Practice%20Guideline%20Update&author=GA%20Masters&author=S%20Temin&author=CG%20Azzoli&author=G%20Giaccone&author=SJ%20Baker&volume=33&issue=30&publication_year=2015&pages=3488-515&pmid=26324367&doi=10.1200/JCO.2015.62.1342&)

77. Hu L, Mao X, Gao C, Xu Y, Li C, Wang T, et al. Cytidine Deaminase 435C>T Polymorphism Relates to Gemcitabine-Platinum Efficacy and Hematological Toxicity in Chinese Non-Small-Cell Lung Cancer Patients. Neoplasma (2021) 68:638–44. 10.4149/neo_2021_201116N1229  [DOI](https://doi.org/10.4149/neo_2021_201116N1229) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33724861/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neoplasma&title=Cytidine%20Deaminase%20435C>T%20Polymorphism%20Relates%20to%20Gemcitabine-Platinum%20Efficacy%20and%20Hematological%20Toxicity%20in%20Chinese%20Non-Small-Cell%20Lung%20Cancer%20Patients&author=L%20Hu&author=X%20Mao&author=C%20Gao&author=Y%20Xu&author=C%20Li&volume=68&publication_year=2021&pages=638-44&pmid=33724861&doi=10.4149/neo_2021_201116N1229&)

78. Ludovini V, Floriani I, Pistola L, Minotti V, Meacci M, Chiari R, et al. Association of Cytidine Deaminase and Xeroderma Pigmentosum Group D Polymorphisms With Response, Toxicity, and Survival in Cisplatin/Gemcitabine-Treated Advanced Non-Small Cell Lung Cancer Patients. J Thorac Oncol (2011) 6(12):2018–26. 10.1097/JTO.0b013e3182307e1f  [DOI](https://doi.org/10.1097/JTO.0b013e3182307e1f) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22052224/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Thorac%20Oncol&title=Association%20of%20Cytidine%20Deaminase%20and%20Xeroderma%20Pigmentosum%20Group%20D%20Polymorphisms%20With%20Response,%20Toxicity,%20and%20Survival%20in%20Cisplatin/Gemcitabine-Treated%20Advanced%20Non-Small%20Cell%20Lung%20Cancer%20Patients&author=V%20Ludovini&author=I%20Floriani&author=L%20Pistola&author=V%20Minotti&author=M%20Meacci&volume=6&issue=12&publication_year=2011&pages=2018-26&pmid=22052224&doi=10.1097/JTO.0b013e3182307e1f&)

79. Ciccolini J, Serdjebi C, Peters GJ, Giovannetti E. Pharmacokinetics and Pharmacogenetics of Gemcitabine as a Mainstay in Adult and Pediatric Oncology: An EORTC-PAMM Perspective. Cancer Chemother Pharmacol (2016) 78(1):1–12. 10.1007/s00280-016-3003-0  [DOI](https://doi.org/10.1007/s00280-016-3003-0) | [PMC free article](/articles/PMC4921117/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27007129/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Chemother%20Pharmacol&title=Pharmacokinetics%20and%20Pharmacogenetics%20of%20Gemcitabine%20as%20a%20Mainstay%20in%20Adult%20and%20Pediatric%20Oncology:%20An%20EORTC-PAMM%20Perspective&author=J%20Ciccolini&author=C%20Serdjebi&author=GJ%20Peters&author=E%20Giovannetti&volume=78&issue=1&publication_year=2016&pages=1-12&pmid=27007129&doi=10.1007/s00280-016-3003-0&)

80. İşcan M, Ada AO. Cytochrome P-450 Polymorphisms and Clinical Outcome in Patients With Non-Small Cell Lung Cancer. Turk J Pharm Sci (2017) 14(3):319–23. 10.4274/tjps.28291  [DOI](https://doi.org/10.4274/tjps.28291) | [PMC free article](/articles/PMC7227939/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32454631/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Turk%20J%20Pharm%20Sci&title=Cytochrome%20P-450%20Polymorphisms%20and%20Clinical%20Outcome%20in%20Patients%20With%20Non-Small%20Cell%20Lung%20Cancer&author=M%20%C4%B0%C5%9Fcan&author=AO%20Ada&volume=14&issue=3&publication_year=2017&pages=319-23&pmid=32454631&doi=10.4274/tjps.28291&)

81. Ji YN, Wang Q, Suo LJ. CYP1A1 Ile462Val Polymorphism Contributes to Lung Cancer Susceptibility Among Lung Squamous Carcinoma and Smokers: A Meta-Analysis. PLoS One (2012) 7(8):43397. 10.1371/journal.pone.0043397  [DOI](https://doi.org/10.1371/journal.pone.0043397) | [PMC free article](/articles/PMC3429471/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22952673/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=CYP1A1%20Ile462Val%20Polymorphism%20Contributes%20to%20Lung%20Cancer%20Susceptibility%20Among%20Lung%20Squamous%20Carcinoma%20and%20Smokers:%20A%20Meta-Analysis&author=YN%20Ji&author=Q%20Wang&author=LJ%20Suo&volume=7&issue=8&publication_year=2012&pages=43397&pmid=22952673&doi=10.1371/journal.pone.0043397&)

82. Crofts F, Taioll E, Trachman J, Cosma GN, Currie D, Toniolo P, et al. Functional Significance of Different Human CYP1A1 Genotypes. Carcinogenesis (1994) 15(12):2961–3. 10.1093/carcin/15.12.2961  [DOI](https://doi.org/10.1093/carcin/15.12.2961) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8001264/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Carcinogenesis&title=Functional%20Significance%20of%20Different%20Human%20CYP1A1%20Genotypes&author=F%20Crofts&author=E%20Taioll&author=J%20Trachman&author=GN%20Cosma&author=D%20Currie&volume=15&issue=12&publication_year=1994&pages=2961-3&pmid=8001264&doi=10.1093/carcin/15.12.2961&)

83. Hildebrandt AG, Schwarz D, Krusekopf S, Kleeberg U, Roots I. Recalling P446. P4501A1 (CYP1A1) Opting for Clinical Application. Drug Metab Rev (2007) 39(2–3):323–41. 10.1080/03602530701498026  [DOI](https://doi.org/10.1080/03602530701498026) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17786624/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Rev&title=Recalling%20P446.%20P4501A1%20(CYP1A1)%20Opting%20for%20Clinical%20Application&author=AG%20Hildebrandt&author=D%20Schwarz&author=S%20Krusekopf&author=U%20Kleeberg&author=I%20Roots&volume=39&issue=2%E2%80%933&publication_year=2007&pages=323-41&pmid=17786624&doi=10.1080/03602530701498026&)

84. Schoket B, Papp G, Lévay K, Mracková G, Kadlubar FF, Vincze I. Impact of Metabolic Genotypes on Levels of Biomarkers of Genotoxic Exposure. Mutat Res (2001) 482(1–2):57–69. 10.1016/s0027-5107(01)00210-x  [DOI](https://doi.org/10.1016/s0027-5107(01)00210-x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11535249/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mutat%20Res&title=Impact%20of%20Metabolic%20Genotypes%20on%20Levels%20of%20Biomarkers%20of%20Genotoxic%20Exposure&author=B%20Schoket&author=G%20Papp&author=K%20L%C3%A9vay&author=G%20Mrackov%C3%A1&author=FF%20Kadlubar&volume=482&issue=1%E2%80%932&publication_year=2001&pages=57-69&pmid=11535249&doi=10.1016/s0027-5107(01)00210-x&)

85. Chen Y, Zhou H, Yang S, Su D. Increased ABCC2 Expression Predicts Cisplatin Resistance in Non-Small Cell Lung Cancer. Cell Biochem Funct (2021) 39(2):277–86. 10.1002/cbf.3577  [DOI](https://doi.org/10.1002/cbf.3577) | [PMC free article](/articles/PMC7983913/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32815556/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Biochem%20Funct&title=Increased%20ABCC2%20Expression%20Predicts%20Cisplatin%20Resistance%20in%20Non-Small%20Cell%20Lung%20Cancer&author=Y%20Chen&author=H%20Zhou&author=S%20Yang&author=D%20Su&volume=39&issue=2&publication_year=2021&pages=277-86&pmid=32815556&doi=10.1002/cbf.3577&)

86. Zhou J, Kang Y, Chen L, Wang H, Liu J, Zeng S, et al. The Drug-Resistance Mechanisms of Five Platinum-Based Antitumor Agents. Front Pharmacol (2020) 0:343. 10.3389/fphar.2020.00343  [DOI](https://doi.org/10.3389/fphar.2020.00343) | [PMC free article](/articles/PMC7100275/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32265714/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front%20Pharmacol&title=The%20Drug-Resistance%20Mechanisms%20of%20Five%20Platinum-Based%20Antitumor%20Agents&author=J%20Zhou&author=Y%20Kang&author=L%20Chen&author=H%20Wang&author=J%20Liu&volume=0&publication_year=2020&pages=343&pmid=32265714&doi=10.3389/fphar.2020.00343&)

87. Korita PV, Wakai T, Shirai Y, Matsuda Y, Sakata J, Takamura M, et al. Multidrug Resistance-Associated Protein 2 Determines the Efficacy of Cisplatin in Patients With Hepatocellular Carcinoma. Oncol Rep (2010) 23(4):965–72. 10.3892/or_00000721  [DOI](https://doi.org/10.3892/or_00000721) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20204280/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncol%20Rep&title=Multidrug%20Resistance-Associated%20Protein%202%20Determines%20the%20Efficacy%20of%20Cisplatin%20in%20Patients%20With%20Hepatocellular%20Carcinoma&author=PV%20Korita&author=T%20Wakai&author=Y%20Shirai&author=Y%20Matsuda&author=J%20Sakata&volume=23&issue=4&publication_year=2010&pages=965-72&pmid=20204280&doi=10.3892/or_00000721&)

88. Yamasaki M, Makino T, Masuzawa T, Kurokawa Y, Miyata H, Takiguchi S, et al. Role of Multidrug Resistance Protein 2 (MRP2) in Chemoresistance and Clinical Outcome in Oesophageal Squamous Cell Carcinoma. Br J Cancer (2011) 104(4):707–13. 10.1038/sj.bjc.6606071  [DOI](https://doi.org/10.1038/sj.bjc.6606071) | [PMC free article](/articles/PMC3049584/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21206495/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Cancer&title=Role%20of%20Multidrug%20Resistance%20Protein%202%20(MRP2)%20in%20Chemoresistance%20and%20Clinical%20Outcome%20in%20Oesophageal%20Squamous%20Cell%20Carcinoma&author=M%20Yamasaki&author=T%20Makino&author=T%20Masuzawa&author=Y%20Kurokawa&author=H%20Miyata&volume=104&issue=4&publication_year=2011&pages=707-13&pmid=21206495&doi=10.1038/sj.bjc.6606071&)

89. Kool M, de Haas M, Scheffer G, Scheper R, van Eijk M, Juijn J, et al. Analysis of Expression of cMOAT (MRP2), MRP3, MRP4, and MRP5, Homologues of the Multidrug Resistance-Associated Protein Gene (MRP1), in Human Cancer Cell Lines. Cancer Res (1997) 57(16):3537–47.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/9270026/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=Analysis%20of%20Expression%20of%20cMOAT%20(MRP2),%20MRP3,%20MRP4,%20and%20MRP5,%20Homologues%20of%20the%20Multidrug%20Resistance-Associated%20Protein%20Gene%20(MRP1),%20in%20Human%20Cancer%20Cell%20Lines&author=M%20Kool&author=M%20de%20Haas&author=G%20Scheffer&author=R%20Scheper&author=M%20van%20Eijk&volume=57&issue=16&publication_year=1997&pages=3537-47&pmid=9270026&)

90. Liedert B, Materna V, Schadendorf D, Thomale J, Lage H. Overexpression of cMOAT (MRP2/ABCC2) Is Associated With Decreased Formation of Platinum-DNA Adducts and Decreased G2-Arrest in Melanoma Cells Resistant to Cisplatin. J Invest Dermatol (2003) 121(1):172–6. 10.1046/j.1523-1747.2003.12313.x  [DOI](https://doi.org/10.1046/j.1523-1747.2003.12313.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12839578/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Invest%20Dermatol&title=Overexpression%20of%20cMOAT%20(MRP2/ABCC2)%20Is%20Associated%20With%20Decreased%20Formation%20of%20Platinum-DNA%20Adducts%20and%20Decreased%20G2-Arrest%20in%20Melanoma%20Cells%20Resistant%20to%20Cisplatin&author=B%20Liedert&author=V%20Materna&author=D%20Schadendorf&author=J%20Thomale&author=H%20Lage&volume=121&issue=1&publication_year=2003&pages=172-6&pmid=12839578&doi=10.1046/j.1523-1747.2003.12313.x&)

91. Qiao R, Han B, Zhang W, Gu A, Shen J. The Correlation Between Single Nucleotide Polymorphism of ABC Transporter and Response Rate and Severe Toxicity in Lung Cancer Patients Treated With Platinum-Based Chemotherapy. Respirology (2016) 16(2):131.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Respirology&title=The%20Correlation%20Between%20Single%20Nucleotide%20Polymorphism%20of%20ABC%20Transporter%20and%20Response%20Rate%20and%20Severe%20Toxicity%20in%20Lung%20Cancer%20Patients%20Treated%20With%20Platinum-Based%20Chemotherapy&author=R%20Qiao&author=B%20Han&author=W%20Zhang&author=A%20Gu&author=J%20Shen&volume=16&issue=2&publication_year=2016&pages=131&)

92. Han JY, Lim HS, Yoo YK, Shin ES, Park YH, Lee SY, et al. Associations of ABCB1, ABCC2, and ABCG2 Polymorphisms With Irinotecan-Pharmacokinetics and Clinical Outcome in Patients With Advanced Non-Small Cell Lung Cancer. Cancer (2007) 110(1):138–47. 10.1002/cncr.22760  [DOI](https://doi.org/10.1002/cncr.22760) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17534875/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer&title=Associations%20of%20ABCB1,%20ABCC2,%20and%20ABCG2%20Polymorphisms%20With%20Irinotecan-Pharmacokinetics%20and%20Clinical%20Outcome%20in%20Patients%20With%20Advanced%20Non-Small%20Cell%20Lung%20Cancer&author=JY%20Han&author=HS%20Lim&author=YK%20Yoo&author=ES%20Shin&author=YH%20Park&volume=110&issue=1&publication_year=2007&pages=138-47&pmid=17534875&doi=10.1002/cncr.22760&)

93. Nguyen TD, Markova S, Liu W, Gow JM, Baldwin RM, Habashian M, et al. Functional Characterization of ABCC2 Promoter Polymorphisms and Allele-Specific Expression. Pharmacogenomics J (2013) 13:396–402. 10.1038/tpj.2012.20  [DOI](https://doi.org/10.1038/tpj.2012.20) | [PMC free article](/articles/PMC3435480/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22664480/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Functional%20Characterization%20of%20ABCC2%20Promoter%20Polymorphisms%20and%20Allele-Specific%20Expression&author=TD%20Nguyen&author=S%20Markova&author=W%20Liu&author=JM%20Gow&author=RM%20Baldwin&volume=13&publication_year=2013&pages=396-402&pmid=22664480&doi=10.1038/tpj.2012.20&)

94. Zhang Y, Zhao T, Li W, Vore M. The 5′-Untranslated Region of Multidrug Resistance Associated Protein 2 (MRP2; ABCC2) Regulates Downstream Open Reading Frame Expression Through Translational Regulation. Mol Pharmacol (2010) 77(2):237–46. 10.1124/mol.109.058982  [DOI](https://doi.org/10.1124/mol.109.058982) | [PMC free article](/articles/PMC2812073/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19890061/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Pharmacol&title=The%205%E2%80%B2-Untranslated%20Region%20of%20Multidrug%20Resistance%20Associated%20Protein%202%20(MRP2;%20ABCC2)%20Regulates%20Downstream%20Open%20Reading%20Frame%20Expression%20Through%20Translational%20Regulation&author=Y%20Zhang&author=T%20Zhao&author=W%20Li&author=M%20Vore&volume=77&issue=2&publication_year=2010&pages=237-46&pmid=19890061&doi=10.1124/mol.109.058982&)

95. Qiao R, Wu W, Lu D, Han B. Influence of Single Nucleotide Polymorphisms in ABCB1, ABCG2 and ABCC2 on Clinical Outcomes to Paclitaxel-Platinum Chemotherapy in Patients With Non-Small-Cell Lung Cancer. Int J Clin Exp Med (2016) 9(1):298–307.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Clin%20Exp%20Med&title=Influence%20of%20Single%20Nucleotide%20Polymorphisms%20in%20ABCB1,%20ABCG2%20and%20ABCC2%20on%20Clinical%20Outcomes%20to%20Paclitaxel-Platinum%20Chemotherapy%20in%20Patients%20With%20Non-Small-Cell%20Lung%20Cancer&author=R%20Qiao&author=W%20Wu&author=D%20Lu&author=B%20Han&volume=9&issue=1&publication_year=2016&pages=298-307&)

96. Tamura A, Wakabayashi K, Onishi Y, Takeda M, Ikegami Y, Sawada S, et al. Re-Evaluation and Functional Classification of Non-Synonymous Single Nucleotide Polymorphisms of the Human ATP-Binding Cassette Transporter ABCG2. Cancer Sci (2007) 98(2):231–9. 10.1111/j.1349-7006.2006.00371.x  [DOI](https://doi.org/10.1111/j.1349-7006.2006.00371.x) | [PMC free article](/articles/PMC11158858/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17297656/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Sci&title=Re-Evaluation%20and%20Functional%20Classification%20of%20Non-Synonymous%20Single%20Nucleotide%20Polymorphisms%20of%20the%20Human%20ATP-Binding%20Cassette%20Transporter%20ABCG2&author=A%20Tamura&author=K%20Wakabayashi&author=Y%20Onishi&author=M%20Takeda&author=Y%20Ikegami&volume=98&issue=2&publication_year=2007&pages=231-9&pmid=17297656&doi=10.1111/j.1349-7006.2006.00371.x&)

97. Furukawa T, Wakabayashi K, Tamura A, Nakagawa H, Morishima Y, Osawa Y, et al. Major SNP (Q141K) Variant of Human ABC Transporter ABCG2 Undergoes Lysosomal and Proteasomal Degradations. Pharm Res (2009) 26(2):469–79. 10.1007/s11095-008-9752-7  [DOI](https://doi.org/10.1007/s11095-008-9752-7) | [PMC free article](/articles/PMC2628956/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18958403/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharm%20Res&title=Major%20SNP%20(Q141K)%20Variant%20of%20Human%20ABC%20Transporter%20ABCG2%20Undergoes%20Lysosomal%20and%20Proteasomal%20Degradations&author=T%20Furukawa&author=K%20Wakabayashi&author=A%20Tamura&author=H%20Nakagawa&author=Y%20Morishima&volume=26&issue=2&publication_year=2009&pages=469-79&pmid=18958403&doi=10.1007/s11095-008-9752-7&)

98. Ripperger A, Benndorf R. The C421A (Q141K) Polymorphism Enhances the 3’-Untranslated Region (3’-UTR)-Dependent Regulation of ATP-Binding Cassette Transporter ABCG2. Biochem Pharmacol (2016) 104:139–47. 10.1016/j.bcp.2016.02.011  [DOI](https://doi.org/10.1016/j.bcp.2016.02.011) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26903388/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20Pharmacol&title=The%20C421A%20(Q141K)%20Polymorphism%20Enhances%20the%203%E2%80%99-Untranslated%20Region%20(3%E2%80%99-UTR)-Dependent%20Regulation%20of%20ATP-Binding%20Cassette%20Transporter%20ABCG2&author=A%20Ripperger&author=R%20Benndorf&volume=104&publication_year=2016&pages=139-47&pmid=26903388&doi=10.1016/j.bcp.2016.02.011&)

99. Mizuarai S, Aozasa N, Kotani H. Single Nucleotide Polymorphisms Result in Impaired Membrane Localization and Reduced Atpase Activity in Multidrug Transporter ABCG2. Int J Cancer (2004) 109(2):238–46. 10.1002/ijc.11669  [DOI](https://doi.org/10.1002/ijc.11669) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14750175/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Cancer&title=Single%20Nucleotide%20Polymorphisms%20Result%20in%20Impaired%20Membrane%20Localization%20and%20Reduced%20Atpase%20Activity%20in%20Multidrug%20Transporter%20ABCG2&author=S%20Mizuarai&author=N%20Aozasa&author=H%20Kotani&volume=109&issue=2&publication_year=2004&pages=238-46&pmid=14750175&doi=10.1002/ijc.11669&)

100. Kondo C, Suzuki H, Itoda M, Ozawa S, Sawada JI, Kobayashi D, et al. Functional Analysis of SNPs Variants of BCRP/ABCG2. Pharm Res (2004) 21(10):1895–903. 10.1023/b:pham.0000045245.21637.d4  [DOI](https://doi.org/10.1023/b:pham.0000045245.21637.d4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15553238/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharm%20Res&title=Functional%20Analysis%20of%20SNPs%20Variants%20of%20BCRP/ABCG2&author=C%20Kondo&author=H%20Suzuki&author=M%20Itoda&author=S%20Ozawa&author=JI%20Sawada&volume=21&issue=10&publication_year=2004&pages=1895-903&pmid=15553238&doi=10.1023/b:pham.0000045245.21637.d4&)

101. Ishikawa T, Wakabayashi-Nakao K, Nakagawa H. Methods to Examine the Impact of Nonsynonymous SNPs on Protein Degradation and Function of Human ABC Transporter. Methods Mol Biol (2013) 1015:225–50. 10.1007/978-1-62703-435-7_15  [DOI](https://doi.org/10.1007/978-1-62703-435-7_15) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23824860/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Methods%20Mol%20Biol&title=Methods%20to%20Examine%20the%20Impact%20of%20Nonsynonymous%20SNPs%20on%20Protein%20Degradation%20and%20Function%20of%20Human%20ABC%20Transporter&author=T%20Ishikawa&author=K%20Wakabayashi-Nakao&author=H%20Nakagawa&volume=1015&publication_year=2013&pages=225-50&pmid=23824860&doi=10.1007/978-1-62703-435-7_15&)

102. Mucaki EJ, Zhao JZL, Lizotte DJ, Rogan PK. Predicting Responses to Platin Chemotherapy Agents With Biochemically-Inspired Machine Learning. Signal Transduction Targeted Ther (2019) 4:1. 10.1038/s41392-018-0034-5  [DOI](https://doi.org/10.1038/s41392-018-0034-5) | [PMC free article](/articles/PMC6329797/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30652029/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Signal%20Transduction%20Targeted%20Ther&title=Predicting%20Responses%20to%20Platin%20Chemotherapy%20Agents%20With%20Biochemically-Inspired%20Machine%20Learning&author=EJ%20Mucaki&author=JZL%20Zhao&author=DJ%20Lizotte&author=PK%20Rogan&volume=4&publication_year=2019&pages=1&pmid=30652029&doi=10.1038/s41392-018-0034-5&)

103. Weiss ST, McLeod HL, Flockhart DA, Dolan ME, Benowitz NL, Johnson JA, et al. Creating and Evaluating Genetic Tests Predictive of Drug Response. Nat Rev Drug Discov (2008) 7:568–74. 10.1038/nrd2520  [DOI](https://doi.org/10.1038/nrd2520) | [PMC free article](/articles/PMC2682785/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18587383/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Drug%20Discov&title=Creating%20and%20Evaluating%20Genetic%20Tests%20Predictive%20of%20Drug%20Response&author=ST%20Weiss&author=HL%20McLeod&author=DA%20Flockhart&author=ME%20Dolan&author=NL%20Benowitz&volume=7&publication_year=2008&pages=568-74&pmid=18587383&doi=10.1038/nrd2520&)

104. Sui Q, Hu Z, Jin X, Bian Y, Liang J, Zhang H, et al. The Genomic Signature of Resistance to Platinum-Containing Neoadjuvant Therapy Based on Single-Cell Data. Cell Biosci (2023) 13(1):103. 10.1186/s13578-023-01061-z  [DOI](https://doi.org/10.1186/s13578-023-01061-z) | [PMC free article](/articles/PMC10249226/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/37291676/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Biosci&title=The%20Genomic%20Signature%20of%20Resistance%20to%20Platinum-Containing%20Neoadjuvant%20Therapy%20Based%20on%20Single-Cell%20Data&author=Q%20Sui&author=Z%20Hu&author=X%20Jin&author=Y%20Bian&author=J%20Liang&volume=13&issue=1&publication_year=2023&pages=103&pmid=37291676&doi=10.1186/s13578-023-01061-z&)

105. Nosrati M, Nikfar S, Hasanzad M, Hasanzad M. Economic Aspects in Precision Medicine and Pharmacogenomics. In: Hasanzad M, editor. Precision Medicine in Clinical Practice. Singapore: Springer; (2022). p. 171–8.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Precision%20Medicine%20in%20Clinical%20Practice&author=M%20Nosrati&author=S%20Nikfar&author=M%20Hasanzad&author=M%20Hasanzad&publication_year=2022&)

106. Liao WY, Ho CC, Tsai TH, Chen KY, Shih JY, Yu CJ. Combined Effect of ERCC1 and ERCC2 Polymorphisms on Overall Survival in Non-Squamous Non-Small-Cell Lung Cancer Patients Treated With First-Line Pemetrexed/Platinum. Lung Cancer (2018) 118:90–6. 10.1016/j.lungcan.2018.01.011  [DOI](https://doi.org/10.1016/j.lungcan.2018.01.011) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29572009/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lung%20Cancer&title=Combined%20Effect%20of%20ERCC1%20and%20ERCC2%20Polymorphisms%20on%20Overall%20Survival%20in%20Non-Squamous%20Non-Small-Cell%20Lung%20Cancer%20Patients%20Treated%20With%20First-Line%20Pemetrexed/Platinum&author=WY%20Liao&author=CC%20Ho&author=TH%20Tsai&author=KY%20Chen&author=JY%20Shih&volume=118&publication_year=2018&pages=90-6&pmid=29572009&doi=10.1016/j.lungcan.2018.01.011&)

107. Aminuddin A, Ng PY, Leong CO, Chua EW. Mitochondrial DNA Alterations May Influence the Cisplatin Responsiveness of Oral Squamous Cell Carcinoma. Scientific Rep (2020) 10(1):7885–17. 10.1038/s41598-020-64664-3  [DOI](https://doi.org/10.1038/s41598-020-64664-3) | [PMC free article](/articles/PMC7217862/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32398775/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Scientific%20Rep&title=Mitochondrial%20DNA%20Alterations%20May%20Influence%20the%20Cisplatin%20Responsiveness%20of%20Oral%20Squamous%20Cell%20Carcinoma&author=A%20Aminuddin&author=PY%20Ng&author=CO%20Leong&author=EW%20Chua&volume=10&issue=1&publication_year=2020&pages=7885-17&pmid=32398775&doi=10.1038/s41598-020-64664-3&)

108. Abu N, Hon KW, Jeyaraman S, Jamal R. Long Noncoding RNAs as Biotargets in Cisplatin-Based Drug Resistance. Future Oncol (2018) 14(29):3085–95. 10.2217/fon-2018-0303  [DOI](https://doi.org/10.2217/fon-2018-0303) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30468082/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Future%20Oncol&title=Long%20Noncoding%20RNAs%20as%20Biotargets%20in%20Cisplatin-Based%20Drug%20Resistance&author=N%20Abu&author=KW%20Hon&author=S%20Jeyaraman&author=R%20Jamal&volume=14&issue=29&publication_year=2018&pages=3085-95&pmid=30468082&doi=10.2217/fon-2018-0303&)

109. Garcia FADO, Andrade ESD, Palmero EI. Insights on Variant Analysis in Silico Tools for Pathogenicity Prediction. Front Genet (2022) 13:1010327. 10.3389/fgene.2022.1010327  [DOI](https://doi.org/10.3389/fgene.2022.1010327) | [PMC free article](/articles/PMC9774026/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36568376/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front%20Genet&title=Insights%20on%20Variant%20Analysis%20in%20Silico%20Tools%20for%20Pathogenicity%20Prediction&author=FADO%20Garcia&author=ESD%20Andrade&author=EI%20Palmero&volume=13&publication_year=2022&pages=1010327&pmid=36568376&doi=10.3389/fgene.2022.1010327&)

110. Chen H, Li J, Wang Y, Ng PKS, Tsang YH, Shaw KR, et al. Comprehensive Assessment of Computational Algorithms in Predicting Cancer Driver Mutations. Genome Biol (2020) 21(1):43. 10.1186/s13059-020-01954-z  [DOI](https://doi.org/10.1186/s13059-020-01954-z) | [PMC free article](/articles/PMC7033911/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32079540/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Biol&title=Comprehensive%20Assessment%20of%20Computational%20Algorithms%20in%20Predicting%20Cancer%20Driver%20Mutations&author=H%20Chen&author=J%20Li&author=Y%20Wang&author=PKS%20Ng&author=YH%20Tsang&volume=21&issue=1&publication_year=2020&pages=43&pmid=32079540&doi=10.1186/s13059-020-01954-z&)

111. Domené S, Scaglia PA, Gutiérrez ML, Domené HM. Applying Bioinformatic Platforms, In Vitro, and In Vivo Functional Assays in the Characterization of Genetic Variants in the GH/IGF Pathway Affecting Growth and Development. Cells (2021) 10(8):2063. 10.3390/cells10082063  [DOI](https://doi.org/10.3390/cells10082063) | [PMC free article](/articles/PMC8392544/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34440832/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cells&title=Applying%20Bioinformatic%20Platforms,%20In%20Vitro,%20and%20In%20Vivo%20Functional%20Assays%20in%20the%20Characterization%20of%20Genetic%20Variants%20in%20the%20GH/IGF%20Pathway%20Affecting%20Growth%20and%20Development&author=S%20Domen%C3%A9&author=PA%20Scaglia&author=ML%20Guti%C3%A9rrez&author=HM%20Domen%C3%A9&volume=10&issue=8&publication_year=2021&pages=2063&pmid=34440832&doi=10.3390/cells10082063&)
