# Novel Gene Polymorphisms for Stable Warfarin Dose in a Korean Population: Genome-Wide Association Study

## Metadata
**Authors:** Jung Sun Kim, Sak Lee, Jeong Yee, Kyemyung Park, Eun Jeong Jang, Byung Chul Chang, Hye Sun Gwak
**Journal:** Biomedicines
**Date:** 2023 Aug 19
**DOI:** [10.3390/biomedicines11082308](https://doi.org/10.3390/biomedicines11082308)
**PMID:** 37626805
**PMCID:** PMC10452379
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10452379/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC10452379/pdf/biomedicines-11-02308.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC10452379/pdf/biomedicines-11-02308.pdf)

## Abstract

Warfarin has a narrow therapeutic window and high intra- and inter-individual variability. Considering that many published papers on genotype-guided dosing are derived from European populations, the aim of this study was to investigate novel genetic variants associated with the variability of stable warfarin dose in the Korean population with cardiac valve replacement, using the GWAS approach. This retrospective cohort study was performed from January 1982 to December 2020 at the Severance Cardiovascular Hospital of Yonsei University College of Medicine. GWAS was performed to identify associations between genotypes and the warfarin maintenance dose, by comparing the allele frequency of genetic variants between individuals. Then, the extent of genetic and non-genetic factors on the dose variability was determined by multivariable regression analysis. The study enrolled 214 participants, and the most robust signal cluster was detected on chromosome 16 around VKORC1. Followed by VKORC1, three novel variants (NKX2-6 rs310279, FRAS1 rs4386623, and FAM201A rs1890109) showed an association with stable warfarin dose requirement in univariate analysis. The algorithm was constructed by using multivariable analysis that includes genetic and non-genetic factors, and it could explain 58.5% of the variations in stable warfarin doses. In this variability, VKORC1 rs9934438 and FRAS1 rs4386623 accounted for 33.0% and 9.9%, respectively. This GWAS analysis identified the fact that three novel variants (NKX2-6 rs310279, FRAS1 rs4386623, and FAM201A rs1890109) were associated with stable warfarin doses. Additional research is necessary to validate the results and establish personalized treatment strategies for the Korean population.

Keywords: warfarin, FRAS1, FAM201A, NKX2-6, polymorphism, GWAS

## 1. Introduction

Despite being approved over 60 years ago, warfarin remains a fundamental anticoagulation therapy for preventing and treating a broad range of thromboembolic disorders [[1](#B1-biomedicines-11-02308),[2](#B2-biomedicines-11-02308)]. Although newer agents such as direct oral anticoagulants are available, warfarin remains the drug of choice for patients who have undergone mechanical heart valve replacement [[3](#B3-biomedicines-11-02308)]. However, warfarin therapy requires frequent monitoring of the international normalized ratio (INR) to ensure its pharmacological effectiveness and to avoid complications arising from its narrow therapeutic index.

The intra- and inter-individual variability in warfarin dose requirements poses a significant challenge to maintaining optimal INR. Factors contributing to this variability include age, vitamin K intake, concomitant medications, acute and chronic disease status, ethnicity, and genetics [[4](#B4-biomedicines-11-02308),[5](#B5-biomedicines-11-02308),[6](#B6-biomedicines-11-02308)]. Warfarin exerts its anticoagulant effects by interfering with the cyclic inter-conversion of vitamin K and vitamin K epoxide, resulting in the inactivation of various coagulation factors (Factor II, VII, IX, and X), as well as the anticoagulant proteins C, S, and Z [[7](#B7-biomedicines-11-02308)]. It is a racemic mixture of R and S enantiomers. The S enantiomer, which is 3–5 times more potent than the R enantiomer, is metabolized primarily by the cytochrome P450 (CYP) 2C9, while the R form undergoes metabolism through CYP1A2 and CYP3A4 [[8](#B8-biomedicines-11-02308)]. Consequently, genetic variations in the genes encoding vitamin K epoxide reductase complex subunit 1 (*VKORC1*) and *CYP2C9* have been widely acknowledged as major contributors to inter-individual differences in stable warfarin doses [[9](#B9-biomedicines-11-02308)].

Distinct variations in minor allele frequency (MAF) among ethnic groups result in significant disparities in warfarin dosage requirements [[10](#B10-biomedicines-11-02308)]. For example, the variant allele frequency of *VKORC1* rs9923231 C>T is 88% in Asians, compared to only 5% in individuals of African ancestry [[11](#B11-biomedicines-11-02308),[12](#B12-biomedicines-11-02308)]. The *CYP2C9*2* (rs1799853) and *CYP2C9*3* (rs1057910) variants are more prevalent in those with European ancestry (MAF 10% and 7%, respectively) than in Asians (MAF < 1% and 3%, respectively). Given that much of the existing literature on genotype-guided dosing is derived from European populations, it is crucial to develop Asian-specific algorithms for individualized therapy [[10](#B10-biomedicines-11-02308)].

Genome-wide association studies (GWASs) provide a useful approach for identifying potential candidate genes related to population-specific variants. This method involves a comprehensive analysis of genetic variations across the entire genome to establish potential connections between genotype and phenotype [[13](#B13-biomedicines-11-02308)]. Several studies have employed GWAS to propose novel genetic polymorphisms that may be associated with variations in warfarin maintenance doses among different populations [[14](#B14-biomedicines-11-02308),[15](#B15-biomedicines-11-02308),[16](#B16-biomedicines-11-02308)]. However, there is a scarcity of research focusing on Asian-specific variants in the context of warfarin dose requirements, and to date, no GWAS has been conducted specifically within the Korean population. Consequently, this study aims to investigate novel genetic variants associated with the variability of stable warfarin doses in Korean individuals who have undergone cardiac valve replacement using the GWAS approach.

## 2. Materials and Methods

### 2.1. Study Participants and Data Collection

We conducted a retrospective cohort study using prospectively collected data from January 1982 to December 2020. The study population consisted of individuals aged 18 years or older who underwent mechanical heart valve replacement and subsequently received warfarin treatment at the Severance Cardiovascular Hospital of Yonsei University College of Medicine. The primary outcome was defined as the mean stable dose (mg/day), representing the average warfarin dosage necessary to achieve an INR of 2.0 to 3.0 for a minimum of three consecutive measurements.

Clinical data were obtained by reviewing electronic or scanned medical records, which included demographic information (age, sex, body weight, height, and body mass index), valve position, comorbidities, co-medications, INR values, and stable warfarin doses. All participants were required to provide written informed consent before participating in the study. The study protocol was reviewed and approved by the Institutional Review Board (IRB) of Yonsei University Medical Center (IRB numbers: 4-2009-0283 and 4-2020-0855). This study adhered to the ethical principles outlined in the 1694 Declaration of Helsinki and its subsequent amendments. It obtained ethics committee approval to ensure compliance with the necessary ethical standards for human research.

### 2.2. Genotyping and Sample Quality Control

Genomic DNA was extracted from ethylenediaminetetraacetic acid samples using the QIAamp DNA Blood Mini Kit (QIAGEN GmbH, Hilden, Germany). The AxiomTM Asia Precision Medicine Research Array (Thermo Fisher Scientific, Waltham, MA, USA) was employed for genotyping, and the genetic testing was entrusted to DNALink^®^. Out of more than 750,000 markers included in this genotyping array, approximately 540,000 markers were from South and East Asian populations. In the assay, 200 ng of genomic DNA was employed. After amplification, the arrays were stained and visualized on a GeneTitan MC Instrument (Affymetrix, Santa Clara, CA. USA). The Affymetrix GeneChip Command Console facilitated raw data generation, while genotype calling was performed using the AxiomGT1/BRLMM-P algorithm through Affymetrix Power Tools software version 2.11.3 (Thermo Fisher Scientific, Waltham, MA, USA).

Quality control (QC) procedures encompassed assessments of sex inconsistencies, sample relatedness, call rate, and hetero rate. After dish quality check (call rate and plate quality control) had been carried out, a relationship test was performed. The GWAS analyzed single-nucleotide polymorphisms (SNPs) with a MAF above 1%, call rate > 95%, and Hardy–Weinberg equilibrium at a threshold of *p* > 10^−4^.

### 2.3. Genome-Wide Association Studies (GWAS)

GWAS was conducted using PLINK software version 1.07 to evaluate the association between genetic variant allele frequency and stable warfarin doses (mg/day). Post-QC filtering, 525,953 SNPs were included in the analysis. We performed a linear regression analysis in PLINK software for these SNPs, using an additive genetic model. The association results were adjusted using covariates, such as age and sex. Then, GWAS results were visualized using the Manhattan plots and quantile-quantile (Q-Q) plots.

Functional annotation and identification of likely causal variants were performed using FUMA’s SNP2GENE in conjunction with fine-mapping analysis [[17](#B17-biomedicines-11-02308)]. Significant independent SNPs were identified from GWAS results and lead SNPs were selected if pairwise SNPs demonstrated R^2^ less than 0.1. To recruit enough variables, we used a relaxed threshold of 5 × 10^−7^ for SNP selection. Linkage disequilibrium (LD) blocks were merged into a genomic locus if the maximum distance between them was 250 kb. Reference data for LD analyses consisted of genetic data from East Asian populations in 1000 G phase 3.

We utilized Haploreg v4.2 to obtain information on MAF in Asian populations for the significant independent SNPs discovered [[11](#B11-biomedicines-11-02308)]. Additionally, we collected details of the SNPs such as chromosomal position, reference/alternate alleles, and functional information from the SNP database of the National Center for Biotechnology Information [[12](#B12-biomedicines-11-02308)].

### 2.4. Statistical Analysis

Categorical variables were analyzed with Fisher’s exact test or the chi-square test. The independent samples *t*-test was used to compare continuous variables between two groups while one-way ANOVA followed by post hoc Tukey’s b test were used for multiple comparisons among more than two different genotype groups. Continuous variables were presented as mean ± standard deviation (SD), with a *p*-value < 0.05 considered statistically significant. For the analysis, both dominant and recessive models were applied, and the most suitable model was selected by considering both effect size and statistical significance.

To analyze the contribution of associated SNPs to warfarin dose variability, univariate and multivariable analyses were conducted. For multivariable linear regression analysis, variables with *p* value less than 0.05 in the univariate analysis, in addition to age at operation and sex, were included. The stepwise selection was employed to select the best model, entering variables with *p* < 0.05 and removing them when *p* > 0.10. All statistical analyses were performed using IBM SPSS Statistics, version 20 software (International Business Machines Corp., New York, NY, USA).

## 3. Results

A total of 229 patients who underwent heart valve replacement therapy during the study period were assessed, and 214 patients with stable warfarin doses were included in the analysis. The average age of participants was 58.3 ± 10.1 years, with 143 females (66.8%) represented in the cohort ([Table 1](#biomedicines-11-02308-t001)). The median follow-up duration was 14.2 years (range: 1.0–29.7 years), and each patient had an average of 26.2 INR measurements. The mean stable warfarin dose required to maintain the target INR for three consecutive measurements was 5.5 ± 1.9 mg/day, with substantial inter-individual variability ranging from 2.0 to 14.1 mg/day. Patients receiving angiotensin receptor blockers (ARBs) and diuretics required significantly lower doses compared to those not on these co-medications (*p* = 0.009 and 0.001, respectively).

### Table 1.

| Characteristics | Number (%) (n = 214) | Stable Dose (mg/day) (mean ± SD) | p-Value |
| --- | --- | --- | --- |
| Age (year) | 58.3 ± 10.1 g |   |   |
| Age at operation (year) | 43.5 ± 11.2 g |   |   |
| Sex |   |   | 0.217 |
| Male | 71 (33.2) | 5.72 ± 1.95 |   |
| Female | 143 (66.8) | 5.38 ± 1.92 |   |
| Body mass index (kg/m2) |   |   | 0.480 |
| <25 | 160 (74.8) | 5.44 ± 1.83 |   |
| ≥25 | 54 (25.2) | 5.65 ± 2.23 |   |
| Valve position |   |   | 0.876 |
| Aortic | 47 (22.0) | 5.68 ± 1.60 |   |
| Mitral | 106 (49.5) | 5.46 ± 2.03 |   |
| Double a | 41 (19.2) | 5.38 ± 2.17 |   |
| Tricuspid b | 16 (7.5) | 5.34 ± 1.77 |   |
| Comorbidity |   |   |   |
| Hypertension |   |   | 0.174 |
| Yes | 18 (8.4) | 5.18 ± 1.70 |   |
| No | 196 (91.6) | 5.59 ± 2.00 |   |
| Diabetes mellitus |   |   | 0.940 |
| Yes | 18 (8.4) | 5.49 ± 1.94 |   |
| No | 196 (91.6) | 5.49 ± 1.94 |   |
| Congestive heart failure |   |   | 0.717 |
| Yes | 47 (22.0) | 5.40 ± 2.16 |   |
| No | 167 (78.0) | 5.52 ± 1.87 |   |
| Atrial fibrillation |   |   | 0.095 |
| Yes | 124 (57.9) | 5.30 ± 1.84 |   |
| No | 90 (42.1) | 5.75 ± 2.03 |   |
| Rheumatic disease |   |   | 0.650 |
| Yes | 122 (57.0) | 5.44 ± 1.90 |   |
| No | 92 (43.0) | 5.56 ± 1.99 |   |
| Myocardial infarction |   |   | 0.825 |
| Yes | 4 (1.90) | 5.70 ± 2.01 |   |
| No | 210 (98.1) | 5.49 ± 1.94 |   |
| Comedication |   |   |   |
| Antiplatelet drugs c |   |   | 0.582 |
| Yes | 8 (4.1) | 5.15 ± 1.10 |   |
| No | 188 (95.9) | 5.53 ± 1.95 |   |
| INR-increasing drugs d |   |   | 0.359 |
| Yes | 1 (0.5) | 3.75 |   |
| No | 198 (99.5) | 5.52 ± 1.92 |   |
| INR-decreasing drugs e |   |   | 0.353 |
| Yes | 2 (1.0) | 6.77 ± 2.50 |   |
| No | 197 (99.0) | 5.50 ± 1.91 |   |
| ARBs |   |   | 0.009 |
| Yes | 46 (21.9) | 4.81 ± 1.51 |   |
| No | 164 (78.1) | 5.66 ± 2.00 |   |
| ACEIs |   |   | 0.405 |
| Yes | 32 (15.2) | 5.21 ± 1.88 |   |
| No | 178 (84.8) | 5.52 ± 1.94 |   |
| Diuretics f |   |   | 0.001 |
| Yes | 90 (42.9) | 4.98 ± 1.89 |   |
| No | 120 (57.1) | 5.84 ± 1.89 |   |
| Calcium channel blockers |   |   | 0.389 |
| Yes | 31 (15.5) | 5.78 ± 2.01 |   |
| No | 169 (84.5) | 5.46 ± 1.90 |   |
| Statins |   |   | 0.455 |
| Yes | 8 (4.0) | 5.02 ± 1.43 |   |
| No | 192 (96.0) | 5.53 ± 1.93 |   |

Table 1 Caption: Demographic characteristics and warfarin stable dose (mg/day).

The association analysis between mean stable warfarin doses and SNPs is illustrated in [Figure 1](#biomedicines-11-02308-f001). With a relaxed genome-wide significance threshold of 5 × 10^−7^, fine-mapping analysis to pinpoint causal variants revealed 13 independent SNPs ([Supplementary Table S1](#app1-biomedicines-11-02308)) and five lead SNPs: *VKORC1* rs9934438, Fraser extracellular matrix complex subunit 1 (*FRAS1*) rs4386623, family with sequence similarity 201 member A (*FAM201A*) rs1890109, NK2 Homeobox-6 (*NKX2-6*) rs310279, and gamma-aminobutyric acid type A receptor subunit beta1 (*GABRB1*) rs117496075 ([Figure 2](#biomedicines-11-02308-f002)). The most pronounced signal cluster was identified on chromosome 16 surrounding *VKORC1*, with *VKORC1* rs9934438 exhibiting the lowest univariate *p*-value of 8.41 × 10^−20^. Apart from this SNP, none of the SNPs achieved a genome-wide significance of 5 × 10^−8^.

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4ac7/10452379/93db96b97493/biomedicines-11-02308-g001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=10452379_biomedicines-11-02308-g001.jpg)

Manhattan plot of a genome-wide association for mean stable dose. (a) Manhattan plot (b) quantile-quantile (Q-Q) plot. (a) The vertical axis indicates the value of –log(p-value) for genome-wide association analysis, and the horizontal axis indicates chromosome number. The red line indicates the genome-wide significance level (p < 5 × 10−8).

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4ac7/10452379/8f08f4381c2b/biomedicines-11-02308-g002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=10452379_biomedicines-11-02308-g002.jpg)

Locus-specific plots of lead single-nucleotide polymorphisms (SNPs): (a) VKORC1 rs9934438, (b) NKX2-6 rs310279, (c) FRAS1 rs4386623, (d) GABRB1 rs117496075, and (e) FAM201A rs1890109. The purple circle represents top lead SNPs, while red circles indicate independent significant SNPs.

The relationship between stable warfarin dose and genotypes was examined using univariate analysis ([Table 2](#biomedicines-11-02308-t002)). All lead SNPs, except for *GABRB1* rs17496075, demonstrated a significant difference in mean stable warfarin doses. Patients possessing the *VKORC1* rs9934438 GG genotype required 9.74 ± 2.18 mg/day, whereas those with AA and AG genotypes needed 5.41 ± 1.84 mg/day (*p* = 6.23 × 10^−6^). Notably, *FRAS1* rs4386623 A allele carriers necessitated significantly higher warfarin doses compared to non-carriers (9.70 ± 2.79 vs. 5.39 ± 1.80 mg/day, *p* = 4.45 × 10^−7^).

### Table 2.

| CHR | Gene Polymorphism | Position | Allele Frequency | Grouped Genotypes | Number (%) | Stable Dose (mg/day) (mean ± SD) | p-Value |
| --- | --- | --- | --- | --- | --- | --- | --- |
| 16 | VKORC1 rs9934438 | 31104878 | G:A = 0.11:0.89 | GG | 4 (3.3) | 9.74 ± 2.18 | 6.23 × 10−6 |
|   | AG, AA | 209 (96.7) | 5.41 ± 1.84 |   |  |  |  |
| 4 | FRAS1rs4386623 | 78991190 | G:A = 0.99:0.01 | GG | 209 (95.9) | 5.39 ± 1.80 | 4.45 × 10−7 |
|   | GA, AA | 5 (4.1) | 9.70 ± 2.79 |   |  |  |  |
| 9 | FAM201A rs1890109 | 38643996 | A:G = 0.98:0.02 | AA | 206 (94.6) | 5.39 ± 1.75 | 0.029 |
|   | AG, GG | 7 (5.4) | 9.04 ± 3.39 |   |  |  |  |
| 8 | NKX2-6rs310279 | 23618463 | A:G = 0.98:0.02 | AA | 205 (94.0) | 5.36 ± 1.78 | 9.30 × 10−3 |
|   | AG, GG | 8 (6.0) | 8.84 ± 2.78 |   |  |  |  |
| 4 | GABRB1 rs117496075 | 47013091 | G:A = 0.98:0.02 | GG | 208 (95.4) | 5.39 ± 1.78 | 0.055 |
|   | GA, AA | 6 (4.6) | 8.99 ± 3.54 |   |  |  |  |

Table 2 Caption: Effects of lead single-nucleotide polymorphisms on warfarin stable dose.

A stepwise linear regression analysis was conducted to ascertain the relative contributions of independent variables to the variability of stable warfarin doses ([Table 3](#biomedicines-11-02308-t003)). Model I, which incorporated age at operation, sex, and factors with *p* < 0.05 from the univariate analysis, accounted for 53.1% of the variability in stable warfarin dose. Genetic and non-genetic factors (age at operation, diuretics, and ARB) explained 48.2% and 6.4% of warfarin dose variability, respectively. The *VKORC1* rs9934438 allelic variation was the most influential, explaining 33.0% of dose variability, followed by *FRAS1* rs4386623 at 9.9% and *FAM201A* rs1890109 at 4.0%.

### Table 3.

| Predictors | Model I |  |  | Model II |  |  |
| --- | --- | --- | --- | --- | --- | --- |
| β (95% CI) | R2 | p-Value | β (95% CI) | R2 | p-Value |  |
| Age at operation (year) | −0.02 (−0.04, −0.01) | 0.011 | 0.027 | −0.02 (−0.04, −0.01) | 0.030 | 0.013 |
| ARBs | −0.54 (−1.00, −0.09) | 0.021 | 0.019 | −0.56 (−0.99, −0.14) | 0.015 | 0.010 |
| Diuretics | −0.46 (−0.85, −0.06) | 0.032 | 0.024 | −0.39 (−0.76, −0.02) | 0.009 | 0.040 |
| VKORC1 rs9934438 per A allele | −2.04 (−2.44, −1.64) | 0.330 | <2.0 × 10−16 | −1.98 (−2.44, −1.64) | 0.330 | <2.0 × 10−16 |
| FRAS1 rs4386623 (0 = GG, 1 = GA,AA) | 3.25 (2.04, 4.45) | 0.099 | 3.07 × 10−7 | 3.10 (2.04, 4.45) | 0.099 | 1.91 × 10−7 |
| FAM201A rs1890109 (0 = AA, 1 = AG,GG) | 1.71 (0.65, 2.76) | 0.040 | 0.002 | 1.91 (0.91, 2.90) | 0.047 | 2.00 × 10−4 |
| NKX2-6 rs310279 (0 = AA, 1 = AG,GG) | 1.43 (0.41, 2.44) | 0.013 | 0.006 | 1.32 (0.36, 2.28) | 0.020 | 0.007 |
| CYP2C9 rs1057910 per C allele |   |   |   | −1.75 (−2.41, −1.08) | 0.052 | 6.87 × 10−7 |
| Adjusted R2 |   | 0.531 |   |   | 0.585 |   |

Table 3 Caption: Multivariable linear regression analysis to identify single-nucleotide polymorphisms with warfarin stable dose.

Since *CYP2C9*3* is a well-established genetic factor in warfarin treatment, we developed Model II, which included this SNP, and performed additional analysis. Model II, incorporating *CYP2C9*3*, predicted further dose variability (R^2^ 0.585), with this SNP contributing to 5.2% of the variability.

## 4. Discussion

In this GWAS analysis, we identified three novel variants (*FRAS1* rs4386623, *FAM201A* rs1890109, and *NKX2-6* rs310279) that were significantly associated with stable warfarin dose requirements in patients who underwent heart valve replacements. Notably, patients with polymorphisms in these SNPs necessitated higher stable doses of warfarin. Additionally, the concurrent use of diuretics and ARBs emerged as significant factors contributing to an increase in stable warfarin doses. Our multivariable analysis, encompassing both genetic and non-genetic factors, accounted for over 50% of the variation in stable warfarin dosing.

Among the identified variants, *FRAS1* rs4386623 explained 9.9% of the variability, making it the second-strongest variable after *VKORC1* in our study. Mutations in *FRAS1* are known to be associated with Fraser syndrome, a disorder characterized by multisystem abnormalities, including malformations in the craniofacial, urogenital, and respiratory systems [[18](#B18-biomedicines-11-02308)]. While numerous studies have examined the involvement of *FRAS1* mutation in the pathogenesis and prognosis of diseases, there is limited research exploring its association with drug-related variations. A GWAS study conducted on sub-Saharan black African patients revealed that *FRAS1* rs7676083 was associated with a significant difference in the ratio of warfarin enantiomers and metabolites (RS-10hydroxywarfarin/RS-warfarin) [[19](#B19-biomedicines-11-02308)]. From a GWAS study of 74 invasive epithelial ovarian cancer patients observed at the Mayo Clinic, it was found that some genes including *FRAS1* may explain inter-patient variation in clinical response to platinum–taxane therapies [[20](#B20-biomedicines-11-02308)]. Additionally, the result of whole-exome sequencing and the GWAS study demonstrated that *FRAS1* polymorphism had significant influences on the pharmacokinetic parameters of dabigatran, a direct oral anticoagulant in a healthy Chinese population [[21](#B21-biomedicines-11-02308)].

The rs4386623 of *FRAS1* is an intron, but intron regions may impact mRNA splicing, thereby modifying protein expression or activity [[22](#B22-biomedicines-11-02308),[23](#B23-biomedicines-11-02308)]. This SNP is located at the 4q21.21 locus, and this region has been associated with the 4q deletion syndrome, which is a genetic disorder characterized by interstitial or terminal deletions on the long arm of chromosome 4 [[24](#B24-biomedicines-11-02308)]. Affected individuals typically exhibit craniofacial abnormalities and delayed growth, and may also present congenital heart defects such as atrial septal defect (ASD), ventricular septal defect (VSD), and patent ductus arteriosus [[24](#B24-biomedicines-11-02308),[25](#B25-biomedicines-11-02308)]. In a case report by Xu et al., the critical region for cardiovascular involvement in 4q deletion syndrome was narrowed down to the chromosomal region between 4q32.2 and q34.3. However, another study conducted in China reported that chromosome 4q21.1q21.21 (position: 78,328,101–79,689,207) was identified as having an uncertain but significant association with ASD and VSD in chromosomal microarray analysis [[26](#B26-biomedicines-11-02308),[27](#B27-biomedicines-11-02308)]. These congenital heart diseases can disrupt blood flow to the lungs, resulting in an increased risk of serious complications such as arrhythmia, high blood pressure, and stroke [[28](#B28-biomedicines-11-02308),[29](#B29-biomedicines-11-02308)]. However, the biological mechanism underlying the observed correlation between warfarin doses and *FRAS1* rs4386623 needs to be elucidated in further research.

*FAM201A* rs1890109, another novel variant, accounted for 4.7% of warfarin dose variability in Model II. FAM201A, a family of long non-coding RNA, was previously demonstrated to be implicated in various diseases, particularly cancers [[30](#B30-biomedicines-11-02308),[31](#B31-biomedicines-11-02308)]. FAM201A participated in cell proliferation, migration, and invasion by regulating ATP-binding cassette transporter E1. Consequently, the upregulation influenced the survival of the patients with lung squamous cell cancer [[30](#B30-biomedicines-11-02308),[31](#B31-biomedicines-11-02308)]. Aside from its role in cancers, a decrease in FAM201A expression has been linked to osteonecrosis of the femoral head [[32](#B32-biomedicines-11-02308)]. Recently, Chen et al. revealed that FAM201A was associated with an increased susceptibility to atrial fibrillation through ceRNA network analysis [[33](#B33-biomedicines-11-02308)]. This study indicated that the down-regulation of FAM201A could potentially be used as a predictive marker for atrial fibrillation susceptibility. According to the GTEx portal, eQTL analysis revealed that this SNP was recorded as a significant expression quantitative trait locus for the FAM201A (*p* = 2.3 × 10^−6^) in the left ventricle [[34](#B34-biomedicines-11-02308)]. The variant allele of rs1890109 is related to a decreased expression of FAM201A. Considering that *FAM201A* polymorphism is associated with atrial fibrillation, the effects of rs1890109 variant on gene expression may increase the risk of thrombosis and interfere with the coagulation cascade, consequently altering stable warfarin dose requirements. While this may provide preliminary insight into how FAM201A impacts warfarin metabolism, the detailed mechanism must be further explored.

The association between *NKX2-6* and stable warfarin doses was observed in our study, accounting for 2.0% of the variability. NKX2-6, a critical transcription factor, is involved in the development of the heart’s dorsal vessel. Mutations in this gene and its counterpart, NKX*2-5*, have been linked to various congenital heart defects [[35](#B35-biomedicines-11-02308)]. During embryogenesis in mice, the expression of NKX*2-6* occurs at the opposite poles of the developing heart [[36](#B36-biomedicines-11-02308)]. While the disruption of NKX*2-6* did not result in apparent heart malformations, the expansion of NKX*2-5* mRNA expression into these regions, where NKX*2-6* was typically present, was observed. This indicates that functional compensation occurred for the loss of NKX*2-6*. Furthermore, double knockout mouse embryos have demonstrated overlapping functions of NKX*2-6* and NKX*2-5*, as the development of the atria was less advanced in these embryos. This provides further evidence of the essential role of these two genes in cardiac development [[37](#B37-biomedicines-11-02308)]. Some research has proposed that genetic variations in these transcription factors were related to the higher risk of atrial fibrillation, suggesting that *NKX2-6* mutations might also increase thromboembolism risk [[38](#B38-biomedicines-11-02308),[39](#B39-biomedicines-11-02308)]. Hence, the observed influence of *NKX2-6* genetic variations on stable warfarin dose requirements in our study could be attributed to alterations in the coagulation system, derived from abnormal cardiovascular function.

Our findings indicated that co-administration of diuretics significantly affected stable warfarin doses. Diuretics are frequently prescribed to relieve pulmonary congestion [[40](#B40-biomedicines-11-02308)]. In our previous research, we demonstrated that the concurrent use of warfarin and diuretics may enhance warfarin’s anticoagulant effect in Korean patients with atrial fibrillation [[41](#B41-biomedicines-11-02308)]. There was a notable difference in stable warfarin between individuals who used hydrochlorothiazide and those who did not (2.97 ± 1.10 mg/day vs. 3.58 ± 1.14 mg/day, *p* = 0.009). However, when the specific subtype of diuretics was not considered, the significance became marginal (*p* = 0.069). On the contrary, a retrospective study conducted in two hospitals in the United States suggested that diuretics (namely furosemide and hydrochlorothiazide) did not significantly alter the INR in patients on stable warfarin therapy [[42](#B42-biomedicines-11-02308)]. It is important to note that the majority of participants in this study were Caucasians (88.6%), without any Asians included. Therefore, this difference in ethnic composition could potentially explain the inconsistent findings, as there may be variations due to ethnic diversity or possible interaction between genetic and environmental factors.

In our study, the concurrent use of ARBs decreased the required warfarin dose compared to that for non-users. An in vitro study investigating the effects of ARBs on the CYP2C9 activity in human liver microsomes has been conducted [[43](#B43-biomedicines-11-02308)]. The finding of this study indicated that specific chemical structures of ARBs significantly affected their affinity to the CYP2C9 enzyme, exerting inhibitory effects on the enzyme. However, a 30-day open-label, single-period study reported that telmisartan treatment reduced mean warfarin trough plasma concentration but did not change the INR [[44](#B44-biomedicines-11-02308)]. Therefore, the impact of ARBs on the requirements of stable warfarin doses remains unclear, and requires further investigation.

Previous GWAS findings revealed that *VKORC1* and *CYP2C9* significantly influenced the inter-variability in warfarin dose requirement [[16](#B16-biomedicines-11-02308),[45](#B45-biomedicines-11-02308),[46](#B46-biomedicines-11-02308),[47](#B47-biomedicines-11-02308)]. In populations of European descent, *VKORC1* and *CYP2C9* accounted for 25% and 9% of the dose variance, respectively [[46](#B46-biomedicines-11-02308)]. Our study results indicated that *VKORC1* and *CYP2C9* contributed 33.0% and 5.2% of the variance, respectively. However, *CYP2C9* did not achieve a genome-wide significant *p*-value in our GWAS, which could be due to allele frequency variations across different populations [[10](#B10-biomedicines-11-02308)]. This result is in accordance with the findings in a GWAS conducted with Japanese patients [[48](#B48-biomedicines-11-02308)]. In that study, only SNPs located near the *VKORC1* gene demonstrated a significant association with therapeutic warfarin dose at a genome-wide level. Neither SNPs in the *CYP2C9* gene nor the *CYP4F2* gene exhibited a genome-wide significant association. The genetic variation observed across different populations highlights the importance of employing an algorithm specifically designed for the Asian population when predicting INR levels.

Despite its retrospective design, our study represents the first GWAS to explore novel candidate genes related to stable warfarin dose requirement in a Korean population. We identified several novel variants significantly impacting warfarin dose requirement. Furthermore, our study exclusively included patients with heart valve replacement, ensuring homogeneity in patient characteristics and target INR. However, as this investigation was conducted in a single center with a limited sample size, the results might not be generalizable to populations with diverse genetic backgrounds. Also, a less-strict genome-wide threshold of 5 × 10^−7^ was used to recruit more variables in the study. As determining the correct *p*-value for statistical significance is crucial for controlling the number of false-positive associations, additional research is required to validate our algorithm in a replication cohort to confirm its efficacy.

## 5. Conclusions

Our GWAS has uncovered three previously unreported genetic variants (*FRAS1* rs4386623, *FAM201A* rs1890109, and *NKX2-6* rs310279) that exhibit significant associations with stable warfarin dose requirements in heart valve replacement patients. Further investigation is necessary to validate these findings and develop individualized treatment approaches tailored specifically to the Korean population.

## Acknowledgments

Not applicable.

## Supplementary Materials

The following supporting information can be downloaded at: [https://www.mdpi.com/article/10.3390/biomedicines11082308/s1](https://www.mdpi.com/article/10.3390/biomedicines11082308/s1), Table S1: Single-nucleotide polymorphisms reached genome significance at *p*-value < 5 × 10^−7^.

## Author Contributions

All the authors have made substantial contributions to the conception of the study. J.S.K., S.L., B.C.C. and H.S.G. contributed to designing the study. J.S.K., J.Y., and E.J.J. contributed to material preparation and data collection. J.S.K., J.Y., K.P., B.C.C. and H.S.G. performed data analysis and interpretation. J.S.K. and S.L. contributed to the drafting of the manuscript. B.C.C. and H.S.G. contributed to critical revision of the manuscript and gave final approval of the version to be published. All authors have read and agreed to the published version of the manuscript.

## Institutional Review Board Statement

The study was conducted in accordance with the Declaration of Helsinki and approved by the Institutional Review Board of Yonsei University Medical Center (IRB numbers: 4-2009-0283 and 4-2020-0855).

## Informed Consent Statement

Informed consent was obtained from all patients involved in the study.

## Data Availability Statement

Data is contained within the article. The data presented in this study is available upon reasonable request from the corresponding author.

## Conflicts of Interest

The authors declare that they have no conflict of interest. The funders had no role in the design of the study; in the collection, analyses, or interpretation of data; in the writing of the manuscript; or in the decision to publish the results.

## Funding Statement

This work was supported by the National Research Foundation (NRF) of Korea grant funded by the Ministry of Education (grant number 2017R1D1A1B03034033).

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

Data is contained within the article. The data presented in this study is available upon reasonable request from the corresponding author.

### Supplementary Materials

### Data Availability Statement

Data is contained within the article. The data presented in this study is available upon reasonable request from the corresponding author.

## References

1. Roden D.M., Johnson J.A., Kimmel S.E., Krauss R.M., Medina M.W., Shuldiner A., Wilke R.A. Cardiovascular pharmacogenomics. Circ. Res. 2011;109:807–820. doi: 10.1161/CIRCRESAHA.110.230995.  [DOI](https://doi.org/10.1161/CIRCRESAHA.110.230995) | [PMC free article](/articles/PMC3201825/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21921273/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circ.%20Res.&title=Cardiovascular%20pharmacogenomics&author=D.M.%20Roden&author=J.A.%20Johnson&author=S.E.%20Kimmel&author=R.M.%20Krauss&author=M.W.%20Medina&volume=109&publication_year=2011&pages=807-820&pmid=21921273&doi=10.1161/CIRCRESAHA.110.230995&)

2. Johnson J.A. Warfarin: An old drug but still interesting. Pharmacotherapy. 2008;28:1081–1083. doi: 10.1592/phco.28.9.1081.  [DOI](https://doi.org/10.1592/phco.28.9.1081) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18752378/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacotherapy&title=Warfarin:%20An%20old%20drug%20but%20still%20interesting&author=J.A.%20Johnson&volume=28&publication_year=2008&pages=1081-1083&pmid=18752378&doi=10.1592/phco.28.9.1081&)

3. Otto C.M., Nishimura R.A., Bonow R.O., Carabello B.A., Erwin J.P., Gentile F., Jneid H., Krieger E.V., Mack M., McLeod C., et al. 2020 ACC/AHA Guideline for the Management of Patients with Valvular Heart Disease: Executive Summary: A Report of the American College of Cardiology/American Heart Association Joint Committee on Clinical Practice Guidelines. Circulation. 2021;143:e35–e71. doi: 10.1161/cir.0000000000000932.  [DOI](https://doi.org/10.1161/cir.0000000000000932) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33332149/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation&title=2020%20ACC/AHA%20Guideline%20for%20the%20Management%20of%20Patients%20with%20Valvular%20Heart%20Disease:%20Executive%20Summary:%20A%20Report%20of%20the%20American%20College%20of%20Cardiology/American%20Heart%20Association%20Joint%20Committee%20on%20Clinical%20Practice%20Guidelines&author=C.M.%20Otto&author=R.A.%20Nishimura&author=R.O.%20Bonow&author=B.A.%20Carabello&author=J.P.%20Erwin&volume=143&publication_year=2021&pages=e35-e71&pmid=33332149&doi=10.1161/cir.0000000000000932&)

4. Johnson J.A. Ethnic differences in cardiovascular drug response: Potential contribution of pharmacogenetics. Circulation. 2008;118:1383–1393. doi: 10.1161/CIRCULATIONAHA.107.704023.  [DOI](https://doi.org/10.1161/CIRCULATIONAHA.107.704023) | [PMC free article](/articles/PMC2730023/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18809808/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation&title=Ethnic%20differences%20in%20cardiovascular%20drug%20response:%20Potential%20contribution%20of%20pharmacogenetics&author=J.A.%20Johnson&volume=118&publication_year=2008&pages=1383-1393&pmid=18809808&doi=10.1161/CIRCULATIONAHA.107.704023&)

5. Holbrook A.M., Pereira J.A., Labiris R., McDonald H., Douketis J.D., Crowther M., Wells P.S. Systematic overview of warfarin and its drug and food interactions. Arch. Intern. Med. 2005;165:1095–1106. doi: 10.1001/archinte.165.10.1095.  [DOI](https://doi.org/10.1001/archinte.165.10.1095) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15911722/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arch.%20Intern.%20Med.&title=Systematic%20overview%20of%20warfarin%20and%20its%20drug%20and%20food%20interactions&author=A.M.%20Holbrook&author=J.A.%20Pereira&author=R.%20Labiris&author=H.%20McDonald&author=J.D.%20Douketis&volume=165&publication_year=2005&pages=1095-1106&pmid=15911722&doi=10.1001/archinte.165.10.1095&)

6. White P.J. Patient factors that influence warfarin dose response. J. Pharm. Pract. 2010;23:194–204. doi: 10.1177/0897190010362177.  [DOI](https://doi.org/10.1177/0897190010362177) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21507814/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Pharm.%20Pract.&title=Patient%20factors%20that%20influence%20warfarin%20dose%20response&author=P.J.%20White&volume=23&publication_year=2010&pages=194-204&pmid=21507814&doi=10.1177/0897190010362177&)

7. Franco V., Polanczyk C.A., Clausell N., Rohde L.E. Role of dietary vitamin K intake in chronic oral anticoagulation: Prospective evidence from observational and randomized protocols. Am. J. Med. 2004;116:651–656. doi: 10.1016/j.amjmed.2003.12.036.  [DOI](https://doi.org/10.1016/j.amjmed.2003.12.036) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15121490/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Med.&title=Role%20of%20dietary%20vitamin%20K%20intake%20in%20chronic%20oral%20anticoagulation:%20Prospective%20evidence%20from%20observational%20and%20randomized%20protocols&author=V.%20Franco&author=C.A.%20Polanczyk&author=N.%20Clausell&author=L.E.%20Rohde&volume=116&publication_year=2004&pages=651-656&pmid=15121490&doi=10.1016/j.amjmed.2003.12.036&)

8. Miners J.O., Birkett D.J. Cytochrome P4502C9: An enzyme of major importance in human drug metabolism. Br. J. Clin. Pharmacol. 1998;45:525–538. doi: 10.1046/j.1365-2125.1998.00721.x.  [DOI](https://doi.org/10.1046/j.1365-2125.1998.00721.x) | [PMC free article](/articles/PMC1873650/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9663807/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Clin.%20Pharmacol.&title=Cytochrome%20P4502C9:%20An%20enzyme%20of%20major%20importance%20in%20human%20drug%20metabolism&author=J.O.%20Miners&author=D.J.%20Birkett&volume=45&publication_year=1998&pages=525-538&pmid=9663807&doi=10.1046/j.1365-2125.1998.00721.x&)

9. Limdi N.A. Warfarin pharmacogenetics: Challenges and opportunities for clinical translation. Front. Pharmacol. 2012;3:183. doi: 10.3389/fphar.2012.00183.  [DOI](https://doi.org/10.3389/fphar.2012.00183) | [PMC free article](/articles/PMC3490409/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23133417/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front.%20Pharmacol.&title=Warfarin%20pharmacogenetics:%20Challenges%20and%20opportunities%20for%20clinical%20translation&author=N.A.%20Limdi&volume=3&publication_year=2012&pages=183&pmid=23133417&doi=10.3389/fphar.2012.00183&)

10. Kaye J.B., Schultz L.E., Steiner H.E., Kittles R.A., Cavallari L.H., Karnes J.H. Warfarin Pharmacogenomics in Diverse Populations. Pharmacotherapy. 2017;37:1150–1163. doi: 10.1002/phar.1982.  [DOI](https://doi.org/10.1002/phar.1982) | [PMC free article](/articles/PMC6913521/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28672100/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacotherapy&title=Warfarin%20Pharmacogenomics%20in%20Diverse%20Populations&author=J.B.%20Kaye&author=L.E.%20Schultz&author=H.E.%20Steiner&author=R.A.%20Kittles&author=L.H.%20Cavallari&volume=37&publication_year=2017&pages=1150-1163&pmid=28672100&doi=10.1002/phar.1982&)

11. Ward L.D., Kellis M. HaploReg v4: Systematic mining of putative causal variants, cell types, regulators and target genes for human complex traits and disease. Nucleic Acids Res. 2016;44:D877–D881. doi: 10.1093/nar/gkv1340.  [DOI](https://doi.org/10.1093/nar/gkv1340) | [PMC free article](/articles/PMC4702929/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26657631/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&title=HaploReg%20v4:%20Systematic%20mining%20of%20putative%20causal%20variants,%20cell%20types,%20regulators%20and%20target%20genes%20for%20human%20complex%20traits%20and%20disease&author=L.D.%20Ward&author=M.%20Kellis&volume=44&publication_year=2016&pages=D877-D881&pmid=26657631&doi=10.1093/nar/gkv1340&)

12. Sayers E.W., Beck J., Bolton E.E., Bourexis D., Brister J.R., Canese K., Comeau D.C., Funk K., Kim S., Klimke W., et al. Database resources of the national center for biotechnology information. Nucleic Acids Res. 2021;49:D10. doi: 10.1093/nar/gkaa892.  [DOI](https://doi.org/10.1093/nar/gkaa892) | [PMC free article](/articles/PMC7778943/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33095870/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&title=Database%20resources%20of%20the%20national%20center%20for%20biotechnology%20information&author=E.W.%20Sayers&author=J.%20Beck&author=E.E.%20Bolton&author=D.%20Bourexis&author=J.R.%20Brister&volume=49&publication_year=2021&pages=D10&pmid=33095870&doi=10.1093/nar/gkaa892&)

13. Tam V., Patel N., Turcotte M., Bossé Y., Paré G., Meyre D. Benefits and limitations of genome-wide association studies. Nat. Rev. Genet. 2019;20:467–484. doi: 10.1038/s41576-019-0127-1.  [DOI](https://doi.org/10.1038/s41576-019-0127-1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31068683/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Genet.&title=Benefits%20and%20limitations%20of%20genome-wide%20association%20studies&author=V.%20Tam&author=N.%20Patel&author=M.%20Turcotte&author=Y.%20Boss%C3%A9&author=G.%20Par%C3%A9&volume=20&publication_year=2019&pages=467-484&pmid=31068683&doi=10.1038/s41576-019-0127-1&)

14. El Rouby N., Shahin M.H., Bader L., Khalifa S.I., Elewa H. Genomewide association analysis of warfarin dose requirements in Middle Eastern and North African populations. Clin. Transl. Sci. 2022;15:558–566. doi: 10.1111/cts.13176.  [DOI](https://doi.org/10.1111/cts.13176) | [PMC free article](/articles/PMC8841446/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34729928/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Transl.%20Sci.&title=Genomewide%20association%20analysis%20of%20warfarin%20dose%20requirements%20in%20Middle%20Eastern%20and%20North%20African%20populations&author=N.%20El%20Rouby&author=M.H.%20Shahin&author=L.%20Bader&author=S.I.%20Khalifa&author=H.%20Elewa&volume=15&publication_year=2022&pages=558-566&pmid=34729928&doi=10.1111/cts.13176&)

15. Parra E.J., Botton M.R., A Perini J., Krithika S., Bourgeois S., A Johnson T., Tsunoda T., Pirmohamed M., Wadelius M., A Limdi N., et al. Genome-wide association study of warfarin maintenance dose in a Brazilian sample. Pharmacogenomics. 2015;16:1253–1263. doi: 10.2217/pgs.15.73.  [DOI](https://doi.org/10.2217/pgs.15.73) | [PMC free article](/articles/PMC4573240/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26265036/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Genome-wide%20association%20study%20of%20warfarin%20maintenance%20dose%20in%20a%20Brazilian%20sample&author=E.J.%20Parra&author=M.R.%20Botton&author=J.%20A%20Perini&author=S.%20Krithika&author=S.%20Bourgeois&volume=16&publication_year=2015&pages=1253-1263&pmid=26265036&doi=10.2217/pgs.15.73&)

16. Perera M.A., Cavallari L.H., Limdi N.A., Gamazon E.R., Konkashbaev A., Daneshjou R., Pluzhnikov A., Crawford D.C., Wang J., Liu N., et al. Genetic variants associated with warfarin dose in African-American individuals: A genome-wide association study. Lancet. 2013;382:790–796. doi: 10.1016/S0140-6736(13)60681-9.  [DOI](https://doi.org/10.1016/S0140-6736(13)60681-9) | [PMC free article](/articles/PMC3759580/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23755828/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Genetic%20variants%20associated%20with%20warfarin%20dose%20in%20African-American%20individuals:%20A%20genome-wide%20association%20study&author=M.A.%20Perera&author=L.H.%20Cavallari&author=N.A.%20Limdi&author=E.R.%20Gamazon&author=A.%20Konkashbaev&volume=382&publication_year=2013&pages=790-796&pmid=23755828&doi=10.1016/S0140-6736(13)60681-9&)

17. Uffelmann E., Huang Q.Q., Munung N.S., De Vries J., Okada Y., Martin A.R., Martin H.C., Lappalainen T., Posthuma D. Genome-wide association studies. Nat. Rev. Methods Primers. 2021;1:59.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Methods%20Primers&title=Genome-wide%20association%20studies&author=E.%20Uffelmann&author=Q.Q.%20Huang&author=N.S.%20Munung&author=J.%20De%20Vries&author=Y.%20Okada&volume=1&publication_year=2021&pages=59&)

18. Hoefele J., Wilhelm C., Schiesser M., Mack R., Heinrich U., Weber L.T., Biskup S., Daumer-Haas C., Klein H.-G., Rost I. Expanding the mutation spectrum for Fraser syndrome: Identification of a novel heterozygous deletion in FRAS1. Gene. 2013;520:194–197. doi: 10.1016/j.gene.2013.02.031.  [DOI](https://doi.org/10.1016/j.gene.2013.02.031) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23473829/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gene&title=Expanding%20the%20mutation%20spectrum%20for%20Fraser%20syndrome:%20Identification%20of%20a%20novel%20heterozygous%20deletion%20in%20FRAS1&author=J.%20Hoefele&author=C.%20Wilhelm&author=M.%20Schiesser&author=R.%20Mack&author=U.%20Heinrich&volume=520&publication_year=2013&pages=194-197&pmid=23473829&doi=10.1016/j.gene.2013.02.031&)

19. Asiimwe I.G., Blockman M., Cohen K., Cupido C., Hutchinson C., Jacobson B., Lamorde M., Morgan J., Mouton J.P., Nakagaayi D., et al. A genome-wide association study of plasma concentrations of warfarin enantiomers and metabolites in sub-Saharan black-African patients. Front. Pharmacol. 2022;13:967082. doi: 10.3389/fphar.2022.967082.  [DOI](https://doi.org/10.3389/fphar.2022.967082) | [PMC free article](/articles/PMC9537548/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36210801/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front.%20Pharmacol.&title=A%20genome-wide%20association%20study%20of%20plasma%20concentrations%20of%20warfarin%20enantiomers%20and%20metabolites%20in%20sub-Saharan%20black-African%20patients&author=I.G.%20Asiimwe&author=M.%20Blockman&author=K.%20Cohen&author=C.%20Cupido&author=C.%20Hutchinson&volume=13&publication_year=2022&pages=967082&pmid=36210801&doi=10.3389/fphar.2022.967082&)

20. Fridley B.L., Ghosh T.M., Wang A., Raghavan R., Dai J., Goode E.L., Lamba J.K. Genome-wide study of response to platinum, taxane, and combination therapy in ovarian cancer: In vitro phenotypes, inherited variation, and disease recurrence. Front. Genet. 2016;7:37. doi: 10.3389/fgene.2016.00037.  [DOI](https://doi.org/10.3389/fgene.2016.00037) | [PMC free article](/articles/PMC4801852/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27047539/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front.%20Genet.&title=Genome-wide%20study%20of%20response%20to%20platinum,%20taxane,%20and%20combination%20therapy%20in%20ovarian%20cancer:%20In%20vitro%20phenotypes,%20inherited%20variation,%20and%20disease%20recurrence&author=B.L.%20Fridley&author=T.M.%20Ghosh&author=A.%20Wang&author=R.%20Raghavan&author=J.%20Dai&volume=7&publication_year=2016&pages=37&pmid=27047539&doi=10.3389/fgene.2016.00037&)

21. Xie Q., Li Y., Liu Z., Mu G., Zhang H., Zhou S., Wang Z., Wang Z., Jiang J., Li X., et al. SLC4A4, FRAS1, and SULT1A1 genetic variations associated with dabigatran metabolism in a healthy Chinese population. Front. Genet. 2022;13:873031. doi: 10.3389/fgene.2022.873031.  [DOI](https://doi.org/10.3389/fgene.2022.873031) | [PMC free article](/articles/PMC9136018/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35646073/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front.%20Genet.&title=SLC4A4,%20FRAS1,%20and%20SULT1A1%20genetic%20variations%20associated%20with%20dabigatran%20metabolism%20in%20a%20healthy%20Chinese%20population&author=Q.%20Xie&author=Y.%20Li&author=Z.%20Liu&author=G.%20Mu&author=H.%20Zhang&volume=13&publication_year=2022&pages=873031&pmid=35646073&doi=10.3389/fgene.2022.873031&)

22. Pagani F., Baralle F.E. Genomic variants in exons and introns: Identifying the splicing spoilers. Nat. Rev. Genet. 2004;5:389–396. doi: 10.1038/nrg1327.  [DOI](https://doi.org/10.1038/nrg1327) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15168696/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Genet.&title=Genomic%20variants%20in%20exons%20and%20introns:%20Identifying%20the%20splicing%20spoilers&author=F.%20Pagani&author=F.E.%20Baralle&volume=5&publication_year=2004&pages=389-396&pmid=15168696&doi=10.1038/nrg1327&)

23. Raponi M., Baralle D. Alternative splicing: Good and bad effects of translationally silent substitutions. FEBS J. 2010;277:836–840. doi: 10.1111/j.1742-4658.2009.07519.x.  [DOI](https://doi.org/10.1111/j.1742-4658.2009.07519.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20082637/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=FEBS%20J.&title=Alternative%20splicing:%20Good%20and%20bad%20effects%20of%20translationally%20silent%20substitutions&author=M.%20Raponi&author=D.%20Baralle&volume=277&publication_year=2010&pages=836-840&pmid=20082637&doi=10.1111/j.1742-4658.2009.07519.x&)

24. Strehle E.-M., Yu L., Rosenfeld J.A., Donkervoort S., Zhou Y., Chen T.-J., Martinez J.E., Fan Y.-S., Barbouth D., Zhu H., et al. Genotype-phenotype analysis of 4q deletion syndrome: Proposal of a critical region. Am. J. Med. Genet. A. 2012;158A:2139–2151. doi: 10.1002/ajmg.a.35502.  [DOI](https://doi.org/10.1002/ajmg.a.35502) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22847869/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Med.%20Genet.%20A&title=Genotype-phenotype%20analysis%20of%204q%20deletion%20syndrome:%20Proposal%20of%20a%20critical%20region&author=E.-M.%20Strehle&author=L.%20Yu&author=J.A.%20Rosenfeld&author=S.%20Donkervoort&author=Y.%20Zhou&volume=158A&publication_year=2012&pages=2139-2151&pmid=22847869&doi=10.1002/ajmg.a.35502&)

25. Strehle E.M., Bantock H.M. The phenotype of patients with 4q-syndrome. Genet. Couns. 2003;14:195–205.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/12872814/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genet.%20Couns.&title=The%20phenotype%20of%20patients%20with%204q-syndrome&author=E.M.%20Strehle&author=H.M.%20Bantock&volume=14&publication_year=2003&pages=195-205&pmid=12872814&)

26. Wang Y., Cao L., Liang D., Meng L., Wu Y., Qiao F., Ji X., Luo C., Zhang J., Xu T., et al. Prenatal chromosomal microarray analysis in fetuses with congenital heart disease: A prospective cohort study. Am. J. Obstet. Gynecol. 2018;218:244.E1–244.E17. doi: 10.1016/j.ajog.2017.10.225.  [DOI](https://doi.org/10.1016/j.ajog.2017.10.225) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29128521/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Obstet.%20Gynecol.&title=Prenatal%20chromosomal%20microarray%20analysis%20in%20fetuses%20with%20congenital%20heart%20disease:%20A%20prospective%20cohort%20study&author=Y.%20Wang&author=L.%20Cao&author=D.%20Liang&author=L.%20Meng&author=Y.%20Wu&volume=218&publication_year=2018&pages=244.E1-244.E17&pmid=29128521&doi=10.1016/j.ajog.2017.10.225&)

27. Xu W., Ahmad A., Dagenais S., Iyer R.K., Innis J.W. Chromosome 4q deletion syndrome: Narrowing the cardiovascular critical region to 4q32.2-q34.3. Am. J. Med. Genet. A. 2012;158A:635–640. doi: 10.1002/ajmg.a.34425.  [DOI](https://doi.org/10.1002/ajmg.a.34425) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22302627/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Med.%20Genet.%20A&title=Chromosome%204q%20deletion%20syndrome:%20Narrowing%20the%20cardiovascular%20critical%20region%20to%204q32.2-q34.3&author=W.%20Xu&author=A.%20Ahmad&author=S.%20Dagenais&author=R.K.%20Iyer&author=J.W.%20Innis&volume=158A&publication_year=2012&pages=635-640&pmid=22302627&doi=10.1002/ajmg.a.34425&)

28. Leppert M., Poisson S.N., Carroll J.D. Atrial Septal Defects and Cardioembolic Strokes. Cardiol. Clin. 2016;34:225–230. doi: 10.1016/j.ccl.2015.12.004.  [DOI](https://doi.org/10.1016/j.ccl.2015.12.004) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27150170/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cardiol.%20Clin.&title=Atrial%20Septal%20Defects%20and%20Cardioembolic%20Strokes&author=M.%20Leppert&author=S.N.%20Poisson&author=J.D.%20Carroll&volume=34&publication_year=2016&pages=225-230&pmid=27150170&doi=10.1016/j.ccl.2015.12.004&)

29. Mandalenakis Z., Rosengren A., Lappas G., Eriksson P., Hansson P.O., Dellborg M. Ischemic Stroke in Children and Young Adults with Congenital Heart Disease. J. Am. Heart Assoc. 2016;5:e003071. doi: 10.1161/JAHA.115.003071.  [DOI](https://doi.org/10.1161/JAHA.115.003071) | [PMC free article](/articles/PMC4802444/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26908411/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Am.%20Heart%20Assoc.&title=Ischemic%20Stroke%20in%20Children%20and%20Young%20Adults%20with%20Congenital%20Heart%20Disease&author=Z.%20Mandalenakis&author=A.%20Rosengren&author=G.%20Lappas&author=P.%20Eriksson&author=P.O.%20Hansson&volume=5&publication_year=2016&pages=e003071&pmid=26908411&doi=10.1161/JAHA.115.003071&)

30. Huang J., Yu Q., Zhou Y., Chu Y., Jiang F., Wang Q. FAM201A knockdown inhibits proliferation and invasion of lung adenocarcinoma cells by regulating miR-7515/GLO1 axis. J. Cell. Physiol. 2021;236:5620–5632. doi: 10.1002/jcp.30250.  [DOI](https://doi.org/10.1002/jcp.30250) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33687075/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Cell.%20Physiol.&title=FAM201A%20knockdown%20inhibits%20proliferation%20and%20invasion%20of%20lung%20adenocarcinoma%20cells%20by%20regulating%20miR-7515/GLO1%20axis&author=J.%20Huang&author=Q.%20Yu&author=Y.%20Zhou&author=Y.%20Chu&author=F.%20Jiang&volume=236&publication_year=2021&pages=5620-5632&pmid=33687075&doi=10.1002/jcp.30250&)

31. He W., Qiao Z.-X., Ma B. Long noncoding RNA FAM201A mediates the metastasis of lung squamous cell cancer via regulating ABCE1 expression. Eur. Rev. Med. Pharmacol. Sci. 2019;23:10343–10353. doi: 10.26355/eurrev_201912_19672.  [DOI](https://doi.org/10.26355/eurrev_201912_19672) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31841188/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur.%20Rev.%20Med.%20Pharmacol.%20Sci.&title=Long%20noncoding%20RNA%20FAM201A%20mediates%20the%20metastasis%20of%20lung%20squamous%20cell%20cancer%20via%20regulating%20ABCE1%20expression&author=W.%20He&author=Z.-X.%20Qiao&author=B.%20Ma&volume=23&publication_year=2019&pages=10343-10353&pmid=31841188&doi=10.26355/eurrev_201912_19672&)

32. Huang G., Zhao G., Xia J., Wei Y., Chen F., Chen J., Shi J. FGF2 and FAM201A affect the development of osteonecrosis of the femoral head after femoral neck fracture. Gene. 2018;652:39–47. doi: 10.1016/j.gene.2018.01.090.  [DOI](https://doi.org/10.1016/j.gene.2018.01.090) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29382571/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gene&title=FGF2%20and%20FAM201A%20affect%20the%20development%20of%20osteonecrosis%20of%20the%20femoral%20head%20after%20femoral%20neck%20fracture&author=G.%20Huang&author=G.%20Zhao&author=J.%20Xia&author=Y.%20Wei&author=F.%20Chen&volume=652&publication_year=2018&pages=39-47&pmid=29382571&doi=10.1016/j.gene.2018.01.090&)

33. Chen X., He X.Y., Dan Q., Li Y. FAM201A, a long noncoding RNA potentially associated with atrial fibrillation identified by ceRNA network analyses and WGCNA. BMC Med. Genom. 2022;15:80. doi: 10.1186/s12920-022-01232-w.  [DOI](https://doi.org/10.1186/s12920-022-01232-w) | [PMC free article](/articles/PMC8996407/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35410298/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Med.%20Genom.&title=FAM201A,%20a%20long%20noncoding%20RNA%20potentially%20associated%20with%20atrial%20fibrillation%20identified%20by%20ceRNA%20network%20analyses%20and%20WGCNA&author=X.%20Chen&author=X.Y.%20He&author=Q.%20Dan&author=Y.%20Li&volume=15&publication_year=2022&pages=80&pmid=35410298&doi=10.1186/s12920-022-01232-w&)

34. Carithers L.J., Moore H.M. The Genotype-Tissue Expression (GTEx) Project. Biopreserv. Biobank. 2015;13:307–308. doi: 10.1089/bio.2015.29031.hmm.  [DOI](https://doi.org/10.1089/bio.2015.29031.hmm) | [PMC free article](/articles/PMC4692118/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26484569/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biopreserv.%20Biobank.&title=The%20Genotype-Tissue%20Expression%20(GTEx)%20Project&author=L.J.%20Carithers&author=H.M.%20Moore&volume=13&publication_year=2015&pages=307-308&pmid=26484569&doi=10.1089/bio.2015.29031.hmm&)

35. Zaidi S., Brueckner M. Genetics and Genomics of Congenital Heart Disease. Circ. Res. 2017;120:923–940. doi: 10.1161/CIRCRESAHA.116.309140.  [DOI](https://doi.org/10.1161/CIRCRESAHA.116.309140) | [PMC free article](/articles/PMC5557504/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28302740/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circ.%20Res.&title=Genetics%20and%20Genomics%20of%20Congenital%20Heart%20Disease&author=S.%20Zaidi&author=M.%20Brueckner&volume=120&publication_year=2017&pages=923-940&pmid=28302740&doi=10.1161/CIRCRESAHA.116.309140&)

36. Biben C., Hatzistavrou T., Harvey R. Expression of NK-2 class homeobox gene Nkx2–6 in foregut endoderm and heart. Mech. Dev. 1998;73:125–127. doi: 10.1016/S0925-4773(98)00037-9.  [DOI](https://doi.org/10.1016/S0925-4773(98)00037-9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9545560/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mech.%20Dev.&title=Expression%20of%20NK-2%20class%20homeobox%20gene%20Nkx2%E2%80%936%20in%20foregut%20endoderm%20and%20heart&author=C.%20Biben&author=T.%20Hatzistavrou&author=R.%20Harvey&volume=73&publication_year=1998&pages=125-127&pmid=9545560&doi=10.1016/S0925-4773(98)00037-9&)

37. Tanaka M., Schinke M., Liao H.-S., Yamasaki N., Izumo S. Nkx2. 5 and Nkx2. 6, homologs of Drosophila tinman, are required for development of the pharynx. Mol. Cell. Biol. 2001;21:4391–4398. doi: 10.1128/MCB.21.13.4391-4398.2001.  [DOI](https://doi.org/10.1128/MCB.21.13.4391-4398.2001) | [PMC free article](/articles/PMC87098/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11390666/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Cell.%20Biol.&title=Nkx2.%205%20and%20Nkx2.%206,%20homologs%20of%20Drosophila%20tinman,%20are%20required%20for%20development%20of%20the%20pharynx&author=M.%20Tanaka&author=M.%20Schinke&author=H.-S.%20Liao&author=N.%20Yamasaki&author=S.%20Izumo&volume=21&publication_year=2001&pages=4391-4398&pmid=11390666&doi=10.1128/MCB.21.13.4391-4398.2001&)

38. Fatkin D., Santiago C.F., Huttner I.G., Lubitz S.A., Ellinor P.T. Genetics of Atrial Fibrillation: State of the Art in 2017. Heart Lung Circ. 2017;26:894–901. doi: 10.1016/j.hlc.2017.04.008.  [DOI](https://doi.org/10.1016/j.hlc.2017.04.008) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28601532/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Heart%20Lung%20Circ.&title=Genetics%20of%20Atrial%20Fibrillation:%20State%20of%20the%20Art%20in%202017&author=D.%20Fatkin&author=C.F.%20Santiago&author=I.G.%20Huttner&author=S.A.%20Lubitz&author=P.T.%20Ellinor&volume=26&publication_year=2017&pages=894-901&pmid=28601532&doi=10.1016/j.hlc.2017.04.008&)

39. Wang J., Zhang D.-F., Sun Y.-M., Li R.-G., Qiu X.-B., Qu X.-K., Liu X., Fang W.-Y., Yang Y.-Q. NKX2-6 mutation predisposes to familial atrial fibrillation. Int. J. Mol. Med. 2014;34:1581–1590. doi: 10.3892/ijmm.2014.1971.  [DOI](https://doi.org/10.3892/ijmm.2014.1971) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25319568/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Mol.%20Med.&title=NKX2-6%20mutation%20predisposes%20to%20familial%20atrial%20fibrillation&author=J.%20Wang&author=D.-F.%20Zhang&author=Y.-M.%20Sun&author=R.-G.%20Li&author=X.-B.%20Qiu&volume=34&publication_year=2014&pages=1581-1590&pmid=25319568&doi=10.3892/ijmm.2014.1971&)

40. Boon N.A., Bloomfield P. The medical management of valvar heart disease. Heart. 2002;87:395–400. doi: 10.1136/heart.87.4.395.  [DOI](https://doi.org/10.1136/heart.87.4.395) | [PMC free article](/articles/PMC1767079/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11907022/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Heart&title=The%20medical%20management%20of%20valvar%20heart%20disease&author=N.A.%20Boon&author=P.%20Bloomfield&volume=87&publication_year=2002&pages=395-400&pmid=11907022&doi=10.1136/heart.87.4.395&)

41. Park H.J., La H.-O., Gwak H.S. Effects of Diuretics on Warfarin Responses in Patients with Atrial Fibrillation. Korean J. Clin. Pharm. 2013;23:151–157.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Korean%20J.%20Clin.%20Pharm.&title=Effects%20of%20Diuretics%20on%20Warfarin%20Responses%20in%20Patients%20with%20Atrial%20Fibrillation&author=H.J.%20Park&author=H.-O.%20La&author=H.S.%20Gwak&volume=23&publication_year=2013&pages=151-157&)

42. Edwards H.D., Webb R.D., Conway S.E. Effect of oral diuretics on chronic warfarin therapy: A retrospective study. Expert. Opin. Drug Saf. 2012;11:375–380. doi: 10.1517/14740338.2012.624091.  [DOI](https://doi.org/10.1517/14740338.2012.624091) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21957964/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Expert.%20Opin.%20Drug%20Saf.&title=Effect%20of%20oral%20diuretics%20on%20chronic%20warfarin%20therapy:%20A%20retrospective%20study&author=H.D.%20Edwards&author=R.D.%20Webb&author=S.E.%20Conway&volume=11&publication_year=2012&pages=375-380&pmid=21957964&doi=10.1517/14740338.2012.624091&)

43. Kamiyama E., Yoshigae Y., Kasuya A., Takei M., Kurihara A., Ikeda T. Inhibitory effects of angiotensin receptor blockers on CYP2C9 activity in human liver microsomes. Drug Metab. Pharmacokinet. 2007;22:267–275. doi: 10.2133/dmpk.22.267.  [DOI](https://doi.org/10.2133/dmpk.22.267) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17827781/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab.%20Pharmacokinet.&title=Inhibitory%20effects%20of%20angiotensin%20receptor%20blockers%20on%20CYP2C9%20activity%20in%20human%20liver%20microsomes&author=E.%20Kamiyama&author=Y.%20Yoshigae&author=A.%20Kasuya&author=M.%20Takei&author=A.%20Kurihara&volume=22&publication_year=2007&pages=267-275&pmid=17827781&doi=10.2133/dmpk.22.267&)

44. Stangier J., Su CA P., Hendriks M.G., van Lier J.J., Sollie F.A., Oosterhuis B., Jonkman J.H. Steady-state pharmacodynamics and pharmacokinetics of warfarin in the presence and absence of telmisartan in healthy male volunteers. Pt 1J. Clin. Pharmacol. 2000;40:1331–1337. doi: 10.1177/009127000004001204.  [DOI](https://doi.org/10.1177/009127000004001204) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11185631/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Pharmacol.&title=Steady-state%20pharmacodynamics%20and%20pharmacokinetics%20of%20warfarin%20in%20the%20presence%20and%20absence%20of%20telmisartan%20in%20healthy%20male%20volunteers&author=J.%20Stangier&author=P.%20Su%20CA&author=M.G.%20Hendriks&author=J.J.%20van%20Lier&author=F.A.%20Sollie&volume=40&publication_year=2000&pages=1331-1337&pmid=11185631&doi=10.1177/009127000004001204&)

45. Takeuchi F., McGinnis R., Bourgeois S., Barnes C., Eriksson N., Soranzo N., Whittaker P., Ranganath V., Kumanduri V., McLaren W., et al. A genome-wide association study confirms VKORC1, CYP2C9, and CYP4F2 as principal genetic determinants of warfarin dose. PLoS Genet. 2009;5:e1000433. doi: 10.1371/journal.pgen.1000433.  [DOI](https://doi.org/10.1371/journal.pgen.1000433) | [PMC free article](/articles/PMC2652833/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19300499/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet.&title=A%20genome-wide%20association%20study%20confirms%20VKORC1,%20CYP2C9,%20and%20CYP4F2%20as%20principal%20genetic%20determinants%20of%20warfarin%20dose&author=F.%20Takeuchi&author=R.%20McGinnis&author=S.%20Bourgeois&author=C.%20Barnes&author=N.%20Eriksson&volume=5&publication_year=2009&pages=e1000433&pmid=19300499&doi=10.1371/journal.pgen.1000433&)

46. Cooper G.M., Johnson J.A., Langaee T.Y., Feng H., Stanaway I.B., Schwarz U.I., Ritchie M.D., Stein C.M., Roden D.M., Smith J.D., et al. A genome-wide scan for common genetic variants with a large influence on warfarin maintenance dose. Blood. 2008;112:1022–1027. doi: 10.1182/blood-2008-01-134247.  [DOI](https://doi.org/10.1182/blood-2008-01-134247) | [PMC free article](/articles/PMC2515139/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18535201/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=A%20genome-wide%20scan%20for%20common%20genetic%20variants%20with%20a%20large%20influence%20on%20warfarin%20maintenance%20dose&author=G.M.%20Cooper&author=J.A.%20Johnson&author=T.Y.%20Langaee&author=H.%20Feng&author=I.B.%20Stanaway&volume=112&publication_year=2008&pages=1022-1027&pmid=18535201&doi=10.1182/blood-2008-01-134247&)

47. Limdi N.A., Beasley T.M., Crowley M.R., A Goldstein J., Rieder M.J., A Flockhart D., Arnett D.K., Acton R.T., Liu N., Shendre A., et al. VKORC1 polymorphisms, haplotypes and haplotype groups on warfarin dose among African-Americans and European-Americans. Pharmacogenomics. 2008;9:1445–1458. doi: 10.2217/14622416.9.10.1445.  [DOI](https://doi.org/10.2217/14622416.9.10.1445) | [PMC free article](/articles/PMC2586955/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18855533/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=VKORC1%20polymorphisms,%20haplotypes%20and%20haplotype%20groups%20on%20warfarin%20dose%20among%20African-Americans%20and%20European-Americans&author=N.A.%20Limdi&author=T.M.%20Beasley&author=M.R.%20Crowley&author=J.%20A%20Goldstein&author=M.J.%20Rieder&volume=9&publication_year=2008&pages=1445-1458&pmid=18855533&doi=10.2217/14622416.9.10.1445&)

48. Cha P.-C., Mushiroda T., Takahashi A., Kubo M., Minami S., Kamatani N., Nakamura Y. Genome-wide association study identifies genetic determinants of warfarin responsiveness for Japanese. Hum. Mol. Genet. 2010;19:4735–4744. doi: 10.1093/hmg/ddq389.  [DOI](https://doi.org/10.1093/hmg/ddq389) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20833655/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum.%20Mol.%20Genet.&title=Genome-wide%20association%20study%20identifies%20genetic%20determinants%20of%20warfarin%20responsiveness%20for%20Japanese&author=P.-C.%20Cha&author=T.%20Mushiroda&author=A.%20Takahashi&author=M.%20Kubo&author=S.%20Minami&volume=19&publication_year=2010&pages=4735-4744&pmid=20833655&doi=10.1093/hmg/ddq389&)
