# Pharmacogenomic heterogeneity of N-acetyltransferase 2: a comprehensive analysis of real world data in Indian tuberculosis patients and from literature and database review

## Metadata
**Authors:** Levin Thomas, Chaithra, Yashi Batra, Mitali Mathur, Shrivathsa Kulavalli, Chidananda Sanju SV, Naveen Dutt, Pankaj Bhardwaj, Muralidhar Varma, Kavitha Saravu, Mithu Banerjee, Mahadev Rao
**Journal:** Annals of Medicine
**Date:** 2025 Mar 26
**DOI:** [10.1080/07853890.2025.2478316](https://doi.org/10.1080/07853890.2025.2478316)
**PMID:** 40138446
**PMCID:** PMC11948353
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11948353/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC11948353/pdf/IANN_57_2478316.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC11948353/pdf/IANN_57_2478316.pdf)

## Abstract

**Background:** 
Isoniazid is primarily metabolized by the arylamine N-acetyltransferase 2 (NAT2) enzyme. Single nucleotide polymorphisms (SNPs) in the NAT2 gene could classify an individual into three distinct phenotypes: rapid, intermediate and slow acetylators. NAT2 SNPs and the slow acetylator phenotype have been implicated as risk factors for the development of antitubercular drug-induced liver injury (AT-DILI) in several tuberculosis (TB) populations.

**Patients and methods:** 
We conducted a prospective observational study to characterize and compare the NAT2 SNPs, genotypes and phenotypes among patients with TB and AT-DILI from the Southern and Western regions of India. The NAT2 pharmacogenomic profile of patients from these regions was compared with the reports from several geographically diverse TB populations and participants of different genetic ancestries extracted from literature reviews and the ‘All of Us’ Research Program database, respectively.

**Results:** 
The TB patients of Southern and Western regions of India and several other geographically closer regions exhibited near similar NAT2 MAF characteristics. However significant heterogeneity in NAT2 SNPs was observed within and between countries among AT-DILI populations and the participants of different genetic ancestry from the ‘All of Us’ Research Program database. The MAF of the NAT2 SNPs rs1041983, rs1801280, rs1799929, rs1799930 and rs1208 of the TB patients from Southern and Western Indian Sites were in near range to that of the South Asian genetic ancestry of ‘All of Us’ Research Program database. About one-third of the total TB patients from the Southern and Western regions of India were NAT2 slow acetylators, among whom a relatively higher proportion experienced AT-DILI.

**Conclusion:** 
Further studies exploring the risk of NAT2 SNPs in different AT-DILI patients with larger sample sizes and a population-specific approach are required to establish a policy for NAT2 genotyping as a pre-emptive biomarker for AT-DILI monitoring for personalized isoniazid therapy in clinics.

Keywords: Isoniazid, N-acetyltransferase 2, pharmacogenomics, single nucleotide polymorphism, AT-DILI

### Background

Isoniazid is primarily metabolized by the arylamine *N*-acetyltransferase 2 (NAT2) enzyme. Single nucleotide polymorphisms (SNPs) in the *NAT2* gene could classify an individual into three distinct phenotypes: rapid, intermediate and slow acetylators. *NAT2* SNPs and the slow acetylator phenotype have been implicated as risk factors for the development of antitubercular drug-induced liver injury (AT-DILI) in several tuberculosis (TB) populations.

### Patients and methods

We conducted a prospective observational study to characterize and compare the *NAT2* SNPs, genotypes and phenotypes among patients with TB and AT-DILI from the Southern and Western regions of India. The *NAT2* pharmacogenomic profile of patients from these regions was compared with the reports from several geographically diverse TB populations and participants of different genetic ancestries extracted from literature reviews and the ‘All of Us’ Research Program database, respectively.

### Results

The TB patients of Southern and Western regions of India and several other geographically closer regions exhibited near similar *NAT2* MAF characteristics. However significant heterogeneity in *NAT2* SNPs was observed within and between countries among AT-DILI populations and the participants of different genetic ancestry from the ‘All of Us’ Research Program database. The MAF of the *NAT2* SNPs rs1041983, rs1801280, rs1799929, rs1799930 and rs1208 of the TB patients from Southern and Western Indian Sites were in near range to that of the South Asian genetic ancestry of ‘All of Us’ Research Program database. About one-third of the total TB patients from the Southern and Western regions of India were *NAT2* slow acetylators, among whom a relatively higher proportion experienced AT-DILI.

### Conclusion

Further studies exploring the risk of *NAT2* SNPs in different AT-DILI patients with larger sample sizes and a population-specific approach are required to establish a policy for *NAT2* genotyping as a pre-emptive biomarker for AT-DILI monitoring for personalized isoniazid therapy in clinics.

## 1. Background

*N*-acetyltransferase 2 (NAT2) enzyme is involved in the biotransformation (phase II) of several xenobiotics such as aromatic amine, heterocyclic amine, or hydrazine compounds as well as several carcinogens present in the diet, cigarette smoke and in the environment, by catalysing the transfer of an acetyl group (−COCH_3_) from acetyl-coenzyme A to the target molecules [[1–6](#CIT0001)]. Isoniazid, a first-line antitubercular drug (also known as isonicotinylhydrazine or INH), is predominantly metabolized (50%- 90%) in the liver and intestines by the NAT2 enzyme [[7](#CIT0007),[8](#CIT0008)]. Suboptimal isoniazid plasma concentrations have been reported in several tuberculosis (TB) populations, warranting the need for higher isoniazid doses to improve the treatment outcomes [[9–11](#CIT0009)]. Whereas TB patients experiencing antitubercular therapy (ATT) induced adverse drug reactions (ADRs) had higher isoniazid plasma concentrations [[12](#CIT0012)]. The variability in isoniazid pharmacokinetics in both TB patients and healthy volunteers has been majorly attributed to the *NAT2* single nucleotide polymorphisms (SNPs)/genotype [[13](#CIT0013),[14](#CIT0014)]. Hence, assessing the *NAT2* SNPs and genotype frequency is a potential host-based omics approach for addressing the inter- and intra-individual isoniazid pharmacokinetic variabilities, isoniazid dose optimization and preventing/reducing the risk of isoniazid-induced ADRs in TB patients for precision therapy of isoniazid [[15](#CIT0015)]. TB patients were also reported to have relatively higher NAT2 expression levels in natural killer (NK) cells and monocytes as compared to healthy volunteers, suggesting its potential role as an early biomarker in the innate immune response to *Mycobacterium tuberculosis* [[16–18](#CIT0016)].

Pharmacogenomics aids in developing strategies for individualizing patient treatment through knowledge of human genome variations and their influence on drug responses. SNPs represent the most frequent genome variations [[19](#CIT0019),[20](#CIT0020)]. Several SNPs have been identified in the *NAT2* gene and the most studied *NAT2* polymorphisms are rs1801279 (191 G > A), rs1041983 (282 C > T), rs1801280 (341 T > C), rs1799929 (481 C > T), rs1799930 (590 G > A), rs1208 (803 A > G) and rs1799931 (857 G > A). However, among these, the *NAT2* rs1801279 was reported to be absent in several Asian and Indian populations [[21–23](#CIT0021)]. These SNPs in the *NAT2* gene may change the structural and functional effects of the NAT2 enzymes by causing substrate affinity/conformational changes, reducing catalytic activity and protein stability and enhancing protein degradation [[24–28](#CIT0024)]. The *NAT2* phenotypes classified as rapid, intermediate and slow acetylators are derived from *NAT2* haplotypes/SNP combinations [[29](#CIT0029)]. These *NAT2* acetylator phenotypes account for the trimodal isoniazid elimination pattern, with *NAT2* rapid acetylators having a twofold or threefold increase in isoniazid clearance, compared to *NAT2* slow acetylators [[30](#CIT0030)]. Antitubercular drug-induced liver injury (AT-DILI) is a common ADR associated with antitubercular drugs such as isoniazid in TB patients. A meta-analysis reported the pooled incidence of AT-DILI of 11.50% (95%CI: 10.10%–12.97%) from 1990 to 2022 and showed an upward trend over time (*p* < .001) from 1999 to 2020 [[31](#CIT0031)]. Assessment of risk factors associated with AT-DILI is imperative for reducing hospitalization, morbidities and mortality during the ATT in TB patients [[32](#CIT0032),[33](#CIT0033)]. *NAT2* SNPs and slow acetylator phenotype have been associated with a higher risk of AT-DILI [[34](#CIT0034)]. *NAT2* genotype-guided isoniazid dosing may result in lower incidences of AT-DILI or early treatment failure compared to traditional dosing regimens [[35](#CIT0035)].

Significant *NAT2* genetic diversity exists among populations across diverse geographical regions of a country, ethnic backgrounds and populations of different countries, including South Asian populations [[21](#CIT0021),[36](#CIT0036),[37](#CIT0037)]. India has a genetically heterogeneous population, encompassing over 4500 anthropologically distinct populations of various caste, tribe and religious groups that exhibit diverse cultural, social and biological behaviours, making it a valuable resource for exploring clinically actionable pharmacogenomic variants [[38–40](#CIT0038)]. Significant differences in the allelic frequencies of several clinically actionable pharmacogenomic variants have been reported between the populations of geographically distinct regions of India and between the Indian and global populations [[40–42](#CIT0040)]. Studies focusing on *NAT2* pharmacogenomic heterogeneity of geographically diverse TB populations of a high TB burden country like India are sparse, leaving gaps in understanding region-specific allelic variant frequency and variations and their potential implications such as in the occurrence of AT-DILI. We hypothesize significant *NAT2* SNP heterogeneity among TB and AT-DILI populations within diverse geographical regions of India and between Indian and other global TB populations. Further, we hypothesize that *NAT2* slow acetylators exhibit a higher risk of developing AT-DILI, irrespective of geographical diversity within India.

Hence, we aimed to investigate the *NAT2* SNP frequency in the TB populations of two geographically diverse regions (Southern and Western) of India. Such a multicentric geographically diverse exploration of the *NAT2* SNP is imperative for deriving a strong conclusive statement for the potential use of *NAT2* genotyping as a preemptive biomarker for AT-DILI monitoring for national-level policy implementation, consideration of geographic-specific *NAT2* SNPs for AT-DILI monitoring, as well as futuristic exploration of using *NAT2* genotype as a potential covariate for isoniazid dose optimization in TB patients. Further, we aimed to compare and characterize the *NAT2* pharmacogenomic diversity from our studied population with several populations of different countries through a literature review of the studies reporting on the *NAT2* SNP frequency in different TB and AT-DILI populations as well as from the participants of the ‘All of Us’ Research Program database from the Public Tier dataset. The ‘All of Us’ Research Program is a longitudinal cohort study with a clinical-grade genomic data release of about ∼245,000 individuals of diverse groups [[43](#CIT0043)]. The ‘All of Us’ Research Hub has three data access tiers: a public tier dataset, a registered tier curated dataset and a controlled tier dataset. The public tier dataset contains only aggregate data with identifiers removed and is available to the public through Data Snapshots ([https://www.researchallofus.org/data-tools/data-snapshots/](https://www.researchallofus.org/data-tools/data-snapshots/)) and the public Data Browser ([https://databrowser.researchallofus.org/](https://databrowser.researchallofus.org/)) [[43](#CIT0043)].

## 2. Patients and methods

### 2.1. Pharmacogenomic profiling of NAT2 in TB patients from Southern and Western regions of India

#### 2.1.1. Study population

A prospective observational multicentric study was conducted, including patients diagnosed with TB and initiated on isoniazid-containing ATT from December 2021 to March 2024. TB patients visiting Kasturba Medical College, Manipal (Southern Indian Site) and All India Institute of Medical Sciences (AIIMS), Jodhpur (Western Indian Site) were included using a convenient sampling method. The study adhered to the principles of the Declaration of Helsinki. The study received institutional ethics committee (IEC) approval from each Study Site (Kasturba Hospital and Kasturba Medical College IEC No. 243/2019 and AIIMS Jodhpur IEC No. 2019/3522). Written informed consent was obtained for all patients. The demographic and clinical data were extracted from medical records and patient interviews. Later in this study, TB patients who developed AT-DILI and met either of the DILI criteria laid by the American Thoracic Society or NIH Drug-Induced Liver Injury Network (DILIN) were also identified [[44–46](#CIT0044)]. The serum samples were collected from these TB patients for *NAT2* genotyping and were stored at −80 °C until deoxyribonucleic acid (DNA) extraction.

#### 2.1.2. NAT2 genotyping

The QIAamp DNA blood mini kit (Qiagen, Hilden, Germany) was used to extract genomic DNA from the serum sample of the patients with TB according to the kit instruction manual. The extracted genomic DNA was quantitated on a NanoDrop 2000 spectrophotometer (ThermoFisher Scientific, U.S.A.). The genomic DNA samples were stored at −80 °C until analysis. Genotyping of the six SNPs in the *NAT2* exon 2 coding region, rs1041983, rs1801280, rs1799929, rs1799930, rs1208 and rs1799931 was performed using the TaqMan^®^ 5′-nuclease assay chemistry on QuantStudio^™^ 5 Real-Time PCR System (Applied Biosystems, U.S.A.). The predesigned TaqMan^™^ assay IDs of the six *NAT2* SNPs used for polymerase chain reaction (PCR) were C___8684085_20 (rs1041983), C___1204093_20 (rs1801280), C___1204092_20 (rs1799929), C___1204091_10 (rs1799930), C____572769_20 (rs1208) and C____572770_50 (rs1799931). The TaqMan probes, labelled with either VIC or FAM as a 5′ reporter dye and equipped with a nonfluorescent 3′ quencher with MGB, specifically target one of the two possible bases at each *NAT2* SNP position. The thermal conditions of the experiment consisted of a pre-reading stage at 60 °C for 30 s, a hold stage at 95 °C for 10 min; and a polymerase chain reaction (PCR) stage comprising of alternating 45 cycles of step 1 at 95 °C for 15 s and step 2 at 60 °C for 1 min and post-reading stage at 60 °C for 30 s. The PCR reaction mixture consisting of 20 ng of genomic DNA was used for each assay/per well on MicroAmp^™^ Optical 96-well reaction plate (Applied Biosystems, U.S.A.). In each experiment, control samples with no DNA template (negative control) were run to ensure that there was no amplification of any contaminating DNA. The genotype of each sample was determined by measuring the allele-specific fluorescence using QuantStudio^™^ Design and Analysis Software v1.5.2 (Applied Biosystems, U.S.A.) for allelic discrimination. The corresponding *NAT2* haplotypes and phenotypes were classified according to the consensus nomenclature of the human *NAT2* alleles haplotypes database. Assessment of the six SNPs in the *NAT2* coding region SNPs described above allows for identifying the reported human *NAT2* alleles. TB patients possessing two of the *NAT2* alleles associated with rapid acetylation activity (*NAT2**4, *NAT2**11, *NAT2**12 and *NAT2**13 families) are classified as *NAT2* rapid acetylators. TB patients with two alleles associated with slow acetylation activity (*NAT2**5, *NAT2**6, *NAT2**7 and *NAT2**14) are classified as *NAT2* slow acetylators, and TB patients with one rapid allele and one slow allele were classified as *NAT2* intermediate acetylators, as described in the consensus nomenclature [[16](#CIT0016),[47](#CIT0047),[48](#CIT0048)]. The minor allelic frequency (MAF) was determined for each *NAT2* SNP by the formula, MAF= (2⋅n_BB_+n_AB_)/2N, where n_BB_, n_AB_ and N are the number of individuals homozygous for the minor allele, number of individuals heterozygous (counted as contributing half the value of the allele) and total number of individuals, respectively.

### 2.2. Review of literature

We conducted a comprehensive literature search to identify the MAF of the six *NAT2* SNPs rs1041983, rs1801280, rs1799929, rs1799930, rs1208 and rs1799931 among different TB populations in PubMed and Google Scholar. We also extracted data from the literature describing the comparative MAF of the six *NAT2* SNP between TB patients who developed AT-DILI (cases) and those who did not develop AT-DILI (controls) of different TB populations.

### 2.3. ‘All of Us’ Research Program database

The MAF of the six *NAT2* SNPs analysed from the TB patients of the current study were compared with those of the participants (n=∼245,400) of the ‘All of Us’ Research Program from the data browser of the single nucleotide variant (SNV)/Indel Variants (public tier) [[49](#CIT0049)]. The MAF for the six *NAT2* SNPs [variant IDs 8-18400285-C-T (rs1041983), 8-18400344-T-C (rs1801280), 8-18400484-C-T (rs1799929), 8-18400593-G-A (rs1799930), 8-18400806-G-A (rs1208) and 8-18400860-G-A (rs1799931)] of the participants categorized into seven categories of genetic ancestry populations of African, East Asian, European, Latin American, Middle Eastern, South Asian and Remaining (individuals did not neatly fit the patterns of any of the genetic ancestry groups) were extracted. The MAF of the six *NAT2* SNPs of the combined Southern and Western Indian TB patients of the original research investigations were compared to the MAF observed among the different genetic ancestries in the ‘All of Us’ Research Program.

### 2.4. Statistical analysis

Descriptive statistics were used to summarize the demographic features of the TB patients. Frequency analysis summarized the *NAT2* SNPs, genotype and phenotype characteristics. The differences between the genotype frequencies of the TB patients with AT-DILI (TB + AT-DILI) and without AT-DILI (TB - AT-DILI) from the Southern and Western Indian Sites were determined using the Chi-square test or Fisher’s exact test ([Table 1](#t0001)). Hardy-Weinberg equilibrium (HWE) states that genetic variation within a large, randomly mating population remains constant across generations in the absence of any disruptive evolutionary influences. HWE was used to evaluate the expected and observed distributions of genotypes in the TB population [[50](#CIT0050)]. According to the HWE, the frequency of the three genotypes is determined by the binomial relationship (p + q)^2^ = 1, where p and q represent the frequency of two alternative alleles. The chi-square test was used to compare the observed genotype frequencies to those expected under HWE, with the test statistic calculated as χ2= ∑(0‐E)2E, where O and E represent the observed and expected genotype counts, respectively. SNPs with a Chi-square value of less than 3.84 at 1 degree of freedom (corresponding to a *p*-value > .05) were considered to be in equilibrium. The pairwise LD for the non-random association of the *NAT2* alleles at different loci was measured using the correlation coefficient (r^2^) using SNPstats, whose values range from 0 to 1, with 0 implying that the alleles are in complete linkage equilibrium and 1 implying that alleles are in completed LD [[51](#CIT0051),[52](#CIT0052)]. The LD plot of the *NAT2* SNPs was generated using SRplot [[53](#CIT0053)]. In the provided LD plots for 6 *NAT2* SNPs ([Figure 1](#F0001)), each diamond-shaped box represents the pairwise LD value (measured as r^2^) between two SNPs. The colour of each box indicates the strength of LD between the two SNPs, as shown by the r^2^ colour key, with the dark blue (low r^2^) indicating weak or no LD and orange to red (high r^2^) indicating strong LD, suggesting the SNPs are likely inherited together. The plots generated for the MAF of the six *NAT2* SNPs among the different TB populations ([Supplementary Figure 1](https://doi.org/10.1080/07853890.2025.2478316)), MAF of the six *NAT2* SNPs between TB patients who developed AT-DILI (cases) and did not develop AT-DILI (controls) ([Supplemental Figure 2](https://doi.org/10.1080/07853890.2025.2478316)) and comparison of the MAF among the different populations from the ‘All of Us’ Research Program database with that of the TB patients of the Southern and Western Indian Sites ([Figure 2](#F0002)) was generated using the GraphPad Prism Version 10.1.2 (GraphPad Software, USA).

### Table 1.

| NAT2 SNP rsID (nucleotide position) | Genotype | Southern Indian Site |  |  |  |  |  |  | Western Indian Site |  |  |  |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Total (N) | TB + AT-DILI (N) | TB-AT-DILI (N) | p Value | MAF (total) | MAF (DILI) | HWE (total) | Total (N) | TB + AT-DILI (N) | TB-AT-DILI (N) | p Value | MAF (total) | MAF (DILI) | HWE (total) |  |  |
| rs1041983 (282C > T) | CC | 84 | 10 | 74 | .327 | 0.43 | 0.50 | 0.51 | 69 | 0 | 69 | .002* | 0.41 | 0.68 | 0.24 |
| CT | 117 | 22 | 95 | 101 | 9 | 92 |  |  |  |  |  |  |  |  |  |
| TT | 49 | 10 | 39 | 32 | 5 | 27 |  |  |  |  |  |  |  |  |  |
| rs1801280 (341T > C) | TT | 111 | 14 | 97 | .194 | 0.34 | 0.39 | 0.91 | 83 | 7 | 76 | .398 | 0.36 | 0.25 | 0 |
| TC | 106 | 23 | 83 | 93 | 7 | 86 |  |  |  |  |  |  |  |  |  |
| CC | 33 | 5 | 28 | 26 | 0 | 26 |  |  |  |  |  |  |  |  |  |
| rs1799929 (481C > T) | CC | 116 | 15 | 101 | .265 | 0.33 | 0.38 | 0.79 | 97 | 8 | 89 | .610 | 0.31 | 0.21 | 0.04 |
| CT | 104 | 22 | 82 | 85 | 6 | 79 |  |  |  |  |  |  |  |  |  |
| TT | 30 | 5 | 25 | 20 | 0 | 20 |  |  |  |  |  |  |  |  |  |
| rs1799930 (590G > A) | GG | 101 | 15 | 86 | .679 | 0.37 | 0.42 | 0.67 | 90 | 3 | 87 | .168 | 0.33 | 0.46 | 0.01 |
| GA | 111 | 19 | 92 | 89 | 9 | 80 |  |  |  |  |  |  |  |  |  |
| AA | 38 | 8 | 30 | 23 | 2 | 21 |  |  |  |  |  |  |  |  |  |
| rs1208 (803A > G) | AA | 105 | 14 | 91 | .277 | 0.36 | 0.39 | 0.78 | 80 | 7 | 73 | .364 | 0.36 | 0.25 | 0.28 |
| AG | 109 | 23 | 86 | 97 | 7 | 90 |  |  |  |  |  |  |  |  |  |
| GG | 36 | 5 | 31 | 25 | 0 | 25 |  |  |  |  |  |  |  |  |  |
| rs1799931 (857G > A) | GG | 222 | 35 | 187 | .279 | 0.11 | 0.17 | 250 | 174 | 9 | 165 | .009* | 0.14 | 0.32 | 193.58 |
| GA | 0 | 0 | 0 | 1 | 1 | 0 |  |  |  |  |  |  |  |  |  |
| AA | 28 | 7 | 21 | 27 | 4 | 23 |  |  |  |  |  |  |  |  |  |

Table 1 Caption: Frequency of NAT2 SNPs in patients with TB and who developed AT-DILI from each study site.

### Figure 1.

![Figure 1.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2e0b/11948353/c4592377ea6c/IANN_A_2478316_F0001_C.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=11948353_IANN_A_2478316_F0001_C.jpg)

LD plot of TB patients from each study site. Abbreviations: LD: linkage disequilibrium. Note: The values inside the diamond-shaped boxes represent the r2 value between each pair of NAT2 SNP.

### Figure 2.

![Figure 2.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2e0b/11948353/1f6463a02cae/IANN_A_2478316_F0002_C.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=11948353_IANN_A_2478316_F0002_C.jpg)

MAF of different NAT2 SNPs across different participants of the ‘All of Us’ Research Program and current study (Southern and Western Indian Sites, n = 452). Abbreviations: MAF: minor allele frequency; NAT2: N-acetyltransferase 2; rsID: reference SNP cluster ID; SNP: single nucleotide polymorphism.

## 3. Results

### 3.1. Pharmacogenomic profiling of NAT2 of TB patients from Southern and Western regions of India

The median [Q1, Q3] age in years of the TB patients from the Southern Indian Site (*n* = 250) and Western Indian Site (*n* = 202) was 47 [34.7, 57] and 33.50 [24.7, 50] years, respectively. There were 161 (64.4%) male and 89 (35.6%) female TB patients from the Southern Indian Site and 124 (61.4%) male and 78 (38.6%) female TB patients from the Western Indian Site. The median [Q1, Q3] BMI (kg/m^2^) was 19.7 [16.4, 22.4] (*n* = 216) and 19.5 [16.6, 22.5] (*n* = 193) of the Southern Indian and Western Indian Site respectively. There were 164 pulmonary TB (PTB), 61 extrapulmonary TB (EPTB), 6 PTB+EPTB, 11 disseminated TB and 8 miliary TB cases from the Southern Indian Site, whereas 94 PTB, 106 EPTB, 1 disseminated TB and 1 miliary TB cases from the Western Indian Site. The MAF of the six *NAT2* SNPs rs1041983, rs1801280, rs1799929, rs1799930, rs1208 and rs1799931 of the combined TB patients of Southern and Western India (*n* = 452) was 0.42, 0.35, 0.32, 0.36, 0.36 and 0.12, respectively. The MAF of these *NAT2* SNPs classified into respective Study Sites is shown in [Table 1](#t0001). Higher MAF was observed for all the six *NAT2* SNPs among TB patients who experienced AT-DILI (*n* = 42) relative to all the TB patients of the Southern Indian Site, whereas higher MAF was observed for rs1041983, rs1799930 and rs1799931 among the AT-DILI group (*n* = 14) compared to the all-TB patient’s group of the Western Indian Site as shown in [Table 1](#t0001). A statistically significant difference between the genotypes of the TB patients with and without AT-DILI groups was observed in the Western Indian Site for the *NAT2* SNPs rs1041983 and rs1799931. All the *NAT2* SNPs were in HWE except for rs1799931 in both centres.

The frequency of *NAT2* rapid, intermediate and slow acetylators was 5.6%, 62.4% and 32% in Southern Indian Site and 6.4%, 59.4% and 34.2% in Western Indian Site, respectively. *NAT2* slow acetylators had a relatively higher proportion of TB patients who developed AT-DILI in both Southern and Western Indian Sites as shown in [Table 2](#t0002).

### Table 2.

|   |   | Southern Indian site |  |  |  |  | Western Indian site |  |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Sl. no | Genotypes | Total |  | AT-DILI |  | AT-DILI%: total% | Total |  | AT-DILI |  | AT-DILI%: total% |
| N | % | N | % | Ratio | N | % | N | % | Ratio |  |  |
| NAT2 rapid acetylators |  |  |  |  |  |  |  |  |  |  |  |
| 1 | NAT2*4/*4 | 10 | 4 | 0 | 0 | – | 11 | 5.4 | 0 | 0 | – |
| 2 | NAT2*4/*12A | 4 | 1.6 | 0 | 0 | – | 2 | 1.0 | 0 | 0 | – |
| Total |  | 14 | 5.6 | 0 | 0 | – | 13 | 6.4 | 0 | 0 | – |
| NAT2 intermediate acetylators |  |  |  |  |  |  |  |  |  |  |  |
| 3 | NAT2*4/*5A | 0 | 0 | 0 | 0 | – | 2 | 1.0 | 0 | 0 | – |
| 4 | NAT2*4/*5B | 67 | 26.8 | 9 | 21.4 | 0.8 | 48 | 23.8 | 0 | 0 | – |
| 5 | NAT2*4/*5C | 3 | 1.2 | 1 | 2.4 | 2.0 | 6 | 3.0 | 0 | 0 | – |
| 6 | NAT2*4/*6A | 76 | 30.4 | 12 | 28.6 | 0.9 | 49 | 24.3 | 3 | 21.4 | 0.9 |
| 7 | NAT2*4/*7B | 6 | 2.4 | 0 | 0 | – | 10 | 5.0 | 2 | 14.2 | 2.8 |
| 8 | NAT2*6A/*11A | 1 | 0.4 | 0 | 0 | – | 0 | 0 | 0 | 0 |   |
| 9 | NAT2*6A/*12A | 3 | 1.2 | 0 | 0 | – | 5 | 2.5 | 0 | 0 |   |
| Total |  | 156 | 62.4 | 22 | 52.4 | 0.8 | 120 | 59.4 | 5 | 35.7 | 0.6 |
| NAT2 slow acetylators |  |  |  |  |  |  |  |  |  |  |  |
| 10 | NAT2*5A/*6A | 0 | 0 | 0 | 0 | – | 2 | 0.9 | 0 | 0 | – |
| 11 | NAT2*5A/*7B | 1 | 0.4 | 0 | 0 | – | 0 | 0 | 0 | 0 | – |
| 12 | NAT2*5B/*6A | 56 | 22.4 | 13 | 31.0 | 1.4 | 42 | 20.7 | 5 | 35.7 | 1.7 |
| 13 | NAT2*5B/*7B | 9 | 3.6 | 5 | 11.9 | 3.3 | 10 | 4.9 | 1 | 7.1 | 1.4 |
| 14 | NAT2*5C/*6A | 2 | 0.8 | 0 | 0 | – | 8 | 3.9 | 1 | 7.1 | 1.8 |
| 15 | NAT2*5C/*7B | 1 | 0.4 | 0 | 0 | – | 0 | 0 | 0 | 0 | – |
| 16 | NAT2*6A/*7B | 11 | 4.4 | 2 | 4.8 | 1.1 | 7 | 3.4 | 2 | 14.3 | 4.2 |
| Total |   | 80 | 32 | 20 | 47.6 | 1.5 | 69 | 34.2 | 9 | 64.3 | 1.9 |

Table 2 Caption: Frequency of NAT2 phenotype and genotype among patients with TB and who developed AT-DILI from each study site.

Strong LD was observed between rs1801280 and rs1799929 (r^2^= 0.89 [Southern Indian Site] and 0.80 [Western Indian Site], rs1801280 and rs1208 (r^2^= 0.90 [Southern Indian Site] and 0.81 [Western Indian Site] among the TB population of both the centres as shown in [Figure 1](#F0001). Good correlations were also observed between rs1041983 and rs1799930 (r^2^= 0.79 [Southern Indian Site] and 0.73 [Western Indian Site] and between rs1799929 and rs1208 (r^2^= 0.80 [Southern Indian Site] and 0.62 [Western Indian Site].

### 3.2. Review of literature on MAF of NAT2 SNP of different TB populations

A relatively higher MAF (≥0.44) for rs1041983 was observed among TB patients of Southeast Asian countries such as Myanmar patients of Thailand [[54](#CIT0054)], Vietnam [[55](#CIT0055)] and Taiwan [[23](#CIT0023)], as well as in the Southern part of India [[56](#CIT0056)], whereas a lower MAF (≤0.03) was observed among TB patients of Latvia [[57](#CIT0057)] and Ethiopia [[58](#CIT0058)] as shown in [Supplementary Figure 1A](https://doi.org/10.1080/07853890.2025.2478316). The MAF for rs1801280 was relatively very high (≥0.35) among the TB patients of Brazil [[59–62](#CIT0059)] and Ethiopia [[58](#CIT0058)], whereas it was relatively very low (≤0.06) among TB patients from Southeast Asian countries such as Taiwan [[23](#CIT0023)] and Vietnam [[55](#CIT0055)] and low (0.18) among Russian TB patients [[63](#CIT0063)] ([Supplementary Figure 1B](https://doi.org/10.1080/07853890.2025.2478316)). The MAF reported for rs1801280 was 0.32 and 0.33 in Southern Indian [[56](#CIT0056)] and Northern Indian [[22](#CIT0022)] TB patients, respectively. Higher relative MAF (≥0.30) for rs1799929 was reported among several South American/Latin American TB patients such as Brazil [[59](#CIT0059),[60](#CIT0060),[62](#CIT0062)], Peru [[64](#CIT0064)], Venezuela [[65](#CIT0065)] and Mexico [[16](#CIT0016),[66](#CIT0066)] and lower MAF (≤0.06) among TB patients of Southeast and East Asian countries such as China [[67](#CIT0067)], Mongolia [[68](#CIT0068)], Taiwan [[23](#CIT0023)] and Vietnam [[55](#CIT0055)] and of Latvia [[57](#CIT0057)] ([Supplementary Figure 1C](https://doi.org/10.1080/07853890.2025.2478316)). The MAF of rs1799929 was reported to be 0.29 and 0.28 in Southern and Northern TB patients, respectively [[22](#CIT0022),[56](#CIT0056)]. Higher MAF for rs1799930 (≥0.30) was observed in Southern [[56](#CIT0056)], Northern [[22](#CIT0022)] and Western Indian [[69](#CIT0069)] TB patients. An MAF ≥0.26 was reported for TB patients of African countries such as Uganda [[70](#CIT0070)] and Ethiopia [[58](#CIT0058)], whereas South American/Latin American TB populations such as from Peru [[64](#CIT0064)], Venezuela [[65](#CIT0065)], Brazil [[59](#CIT0059),[61](#CIT0061),[62](#CIT0062)] and Mexico [[16](#CIT0016),[66](#CIT0066)] had a relatively lower MAF (≤0.17) for rs1799930 ([Supplementary Figure 1D](https://doi.org/10.1080/07853890.2025.2478316)). South American/Latin American TB patients such as from Brazil [[59–62](#CIT0059)], Venezuela [[65](#CIT0065)] and Mexico [[16](#CIT0016),[66](#CIT0066)] had relatively higher MAF (≥0.33) for rs1208, whereas a very relatively low MAF (≤0.06) was reported in TB patients from Southeast and East Asian countries such as Taiwan [[23](#CIT0023)] and Vietnam [[55](#CIT0055)], as well as from Latvia [[57](#CIT0057)] ([Supplementary Figure 1E](https://doi.org/10.1080/07853890.2025.2478316)). TB patients from Peru [[64](#CIT0064)] and East and Southeast Asian countries such as Mongolia [[68](#CIT0068)], Myanmar TB patients of Thailand [[54](#CIT0054)], Taiwan [[23](#CIT0023)] and Vietnam [[55](#CIT0055)] were reported to have a relatively higher MAF (≥0.16) for rs1799931, whereas a relatively lower MAF (0.04) was reported for TB patients in Ethiopia [[58](#CIT0058)] ([Supplementary Figure 1F](https://doi.org/10.1080/07853890.2025.2478316)). The MAF of rs1799931 in the Southern Indian [[56](#CIT0056)] and Northern Indian [[22](#CIT0022)] TB patients was 0.07 and 0.11, respectively.

A relatively higher MAF for rs1041983 was observed among the AT-DILI group compared to those who did not develop AT-DILI in the TB patients of Brazil [[61](#CIT0061),[62](#CIT0062)], Singapore [[71](#CIT0071)], Indonesia [[72](#CIT0072),[73](#CIT0073)] and China [[74](#CIT0074)] as shown in [Supplementary Figure 2A](https://doi.org/10.1080/07853890.2025.2478316). TB patients with AT-DILI in Brazil [[60–62](#CIT0060)] also had relatively higher MAF for rs1801280 compared to the non-AT-DILI group ([Supplementary Figure 2B](https://doi.org/10.1080/07853890.2025.2478316)). TB patients who developed AT-DILI in Brazil [[60](#CIT0060),[61](#CIT0061)], India [[69](#CIT0069)], Tunisia [[75](#CIT0075)], Mongolia [[68](#CIT0068)] and Indonesia [[72](#CIT0072)] had a higher MAF of rs1799929 ([Supplementary Figure 2C](https://doi.org/10.1080/07853890.2025.2478316)). Higher MAF for rs1799930 was also observed among the AT-DILI group of TB patients from Southeast and East Asian countries [[68](#CIT0068),[71–74](#CIT0071),[76](#CIT0076)], Brazil [[61](#CIT0061)] and African countries compared to the non-AT-DILI group (controls) [[58](#CIT0058),[75](#CIT0075)] ([Supplementary Figure 2D](https://doi.org/10.1080/07853890.2025.2478316)). Two reports from TB patients in Brazil [[60](#CIT0060),[62](#CIT0062)] also reported relatively higher MAF for rs1208 among the AT-DILI group ([Supplementary Figure 2E](https://doi.org/10.1080/07853890.2025.2478316)). A relatively higher MAF for rs1799931 was also reported among the TB patients who developed AT-DILI from Singapore [[71](#CIT0071)], Indonesia [[72](#CIT0072)], China [[74](#CIT0074)] as well as Brazil [[62](#CIT0062)] ([Supplementary Figure 2F](https://doi.org/10.1080/07853890.2025.2478316)) compared to the non-AT-DILI group.

[Supplementary Table 1](https://doi.org/10.1080/07853890.2025.2478316) describes the studies that have classified the *NAT2* acetylator into three categories, i.e. slow, intermediate and rapid by genotyping at least the six *NAT2* SNPs rs1041983, rs1801280, rs1799929, rs1799930, rs1208 and rs1799931 in the TB patients. Several studies in TB patients have shown that the *NAT2* slow acetylator varied from 22.7% to 58%, as shown in [Supplementary Table 1](https://doi.org/10.1080/07853890.2025.2478316).

### 3.3. ‘All of Us’ Research Program database

The MAF of the *NAT2* SNPs rs1041983, rs1801280, rs1799929, rs1799930 and rs1208 in the TB patients of the current study were in near ranges, whereas a slightly higher MAF was observed for rs1799931 in our study as compared to participants of South Asian genetic ancestry in the ‘All of Us’ Research Program database as shown in [Figure 2](#F0002). A relatively lower MAF for rs1801280, rs1799929 and rs1208 and a higher MAF for rs1799931 were observed for the East Asian population compared to other populations. The European and Middle Eastern participants had a near comparable and relatively higher MAF for rs1801280, rs1799929 and rs1208. The Latin American group had a relatively lower MAF for rs1799930 than the others. All the groups had a comparatively lower MAF of ≤0.16 for the rs1799931 compared to other *NAT2* SNPs.

## 4. Discussion

The current study explored the *NAT2* SNP diversity among TB populations of two different geographical regions of India and compared it to several other TB and AT-DILI populations and with a large-scale population of diverse genetic ancestries from the ‘All of Us’ Research Program. The MAF was near similar ranges for the six *NAT2* SNPs between the Southern and Western Indian TB patients. All six *NAT2* SNPs had a higher MAF among the TB patients who had AT-DILI as compared to the total TB population in the Southern Indian Site. However, a higher MAF only for the *NAT2* SNPs rs1041983, rs1799930 and rs1799931 were observed among the AT-DILI patients as compared to the total TB patients from the Western Indian Site. *NAT2* slow acetylators from the Western Indian Site had a relatively higher frequency of developing AT-DILI than the Southern Indian Site. We identified a strong LD between the *NAT2* SNPs rs1801280 and rs1799929 and between rs1801280 and rs1208 among both Southern and Western Indian TB patients.

South, Southeast and East Asian populations tend to have relatively higher MAF for rs1041983. Our study showed a relatively higher MAF of 0.43 and 0.41 for the rs1041983 from Southern and Western Indian TB patients which were in near comparable range to the previous report from South Indian TB patients [[56](#CIT0056)] as well as South Asian participants from the ‘All of Us’ Research Program database. A relatively higher MAF (≥0.44) for rs1041983 was observed among the Southeast and East Asian TB patients [[23](#CIT0023),[54](#CIT0054),[55](#CIT0055)]. The MAF for rs1041983 ranged from 0.22 to 0.39 for the South American/Latin American TB population ([Supplementary Figure 1A](https://doi.org/10.1080/07853890.2025.2478316)). Two distinct clusters of participants, one in the MAF range of 0.32–0.33 (European, Latin American and Middle Eastern participants) and another in the MAF range of 0.42–0.45 (Our study results, South Asian, African and East Asian participants) were observed ([Figure 2](#F0002)) for the rs1041983. A higher MAF for rs1041983 in the AT-DILI group was observed in both the Southern and Western Indian TB patients of our study relative to the MAF of total TB patients. rs1041983 was reported to affect the occurrence of AT-DILI significantly (OR: 2.39, 95%CI: 1.15–4.96, *p* = 0.019) among Indonesian paediatric TB patients [[72](#CIT0072)]. Higher MAF among the AT-DILI group was reported among Southeast and East Asian [[71–74](#CIT0071)] and South American TB populations [[61](#CIT0061),[62](#CIT0062)] ([Supplementary Figure 2A](https://doi.org/10.1080/07853890.2025.2478316)).

The MAF for the missense SNP rs1801280 for TB patients from the Southern and Western Indian Sites were in near comparable MAF range to the previous report from Southern and Northern Indian TB patients ([Supplementary Figure 1B](https://doi.org/10.1080/07853890.2025.2478316)) [[22](#CIT0022),[56](#CIT0056)] and South Asian and Latin American participants ([Figure 2](#F0002)). The MAF for rs1801280 was relatively higher among Brazilian [[59–62](#CIT0059)] and lower among Southeast and East Asian [[23](#CIT0023),[55](#CIT0055)] and Russian [[63](#CIT0063)] TB populations. A differential clustering of MAF for rs1801280 was observed, with a higher MAF observed for European and Middle Eastern participants, followed by a clustering of South Asian, current study and Latin American participants and a relatively lower and very low MAF observed for African and East Asian participants respectively. A higher MAF frequency for the AT-DILI group relative to the total TB population was observed in the Southern Indian TB patients in the current study, similar to reports from Brazilian TB patients [[60](#CIT0060),[61](#CIT0061)]. The MAF for rs1799929 for the TB population of the Southern and Western Indian Sites were in near ranges to the previous reports among Southern [[56](#CIT0056)] and Western Indian TB populations [[69](#CIT0069)] respectively ([Supplementary Figure 1C](https://doi.org/10.1080/07853890.2025.2478316)) and to the South Asian and Latin American participants ([Figure 2](#F0002)). Higher MAF for rs1799929 was observed among South American/Latin American populations [[16](#CIT0016),[59](#CIT0059),[60](#CIT0060),[62](#CIT0062),[64](#CIT0064),[66](#CIT0066)] and lower MAF among TB patients of Southeast and East Asian countries [[23](#CIT0023),[54](#CIT0054),[55](#CIT0055),[67](#CIT0067),[68](#CIT0068)] and Latvia [[57](#CIT0057)]. Similar to the rs1801280, a similar pattern of clustering of MAF of rs1799929 was observed, with a higher MAF observed for European and Middle Eastern participants, followed by a clustering of South Asian, current study, and Latin American participants and a relatively lower and very low MAF observed for African and East Asian population respectively. In our study, a relatively higher MAF for rs1799929 in the AT-DILI patients of the Southern Indian Site was observed, similar to a previous report from Indian TB patients [[69](#CIT0069)]. Higher MAF for rs1799929 in the AT-DILI patients were also reported among South American [[60](#CIT0060),[61](#CIT0061)], Southeast and East Asian [[68](#CIT0068),[72](#CIT0072)] and African population [[75](#CIT0075)] ([Supplementary Figure 2C](https://doi.org/10.1080/07853890.2025.2478316)).

Indian and South American/Latin American populations tend to have relatively higher and lower MAF for rs1799930 respectively. The MAF for rs1799930 for TB patients from Southern and Western Sites were in near comparable ranges to the relatively high MAF reported from Southern Indian [[56](#CIT0056)] and Northern Indian [[22](#CIT0022)] TB patients ([Supplementary Figure 1D](https://doi.org/10.1080/07853890.2025.2478316)) and to the South Asian participants ([Figure 2](#F0002)). A relatively lower MAF for rs1799930 was observed among several South American/Latin American TB populations [[16](#CIT0016),[59](#CIT0059),[61](#CIT0061),[62](#CIT0062),[64–66](#CIT0064)]. A relatively lower MAF was also observed among the Latin American participants of the ‘All of Us’ Research Program. A relatively higher MAF for rs1799930 among the patients who developed AT-DILI was observed in the AT-DILI group from Southern and Western Indian Sites, consistent with reports from Southeast and East Asian countries [[68](#CIT0068),[71](#CIT0071),[74](#CIT0074),[76](#CIT0076)]. The MAF for rs1208 from TB patients of Southern and Western Sites were in near comparable ranges with few reports of South American/Latin American TB patients ([Supplementary Figure 1E](https://doi.org/10.1080/07853890.2025.2478316)) and South Asian and Latin American participants ([Figure 2](#F0002)). A relatively higher MAF for rs1208 in South American/Latin American populations (16, 59–62, 66) and lower MAF among Southeast and East Asian [[23](#CIT0023),[55](#CIT0055)] and Latvia [[57](#CIT0057)] TB patients was observed. As with rs1801280 and 1799929, a relatively very low frequency for rs1208 was observed for the East Asian participants and a relatively higher MAF for Middle Eastern and European participants of the ‘All of Us’ Research Program. A higher MAF was reported for rs1208 in the AT-DILI group TB patients from the Southern Indian Site in our study, as observed in reports from the Brazilian TB population [[60](#CIT0060),[62](#CIT0062)] ([Supplementary Figure 2E](https://doi.org/10.1080/07853890.2025.2478316)) in our study. Compared to the other *NAT2* SNPs, a relatively lower MAF was observed for rs1799931 in our Study Sites and other TB populations and participants of the ‘All of Us’ Research Program. A relatively higher MAF was observed among the Southeast and East Asian TB population [[23](#CIT0023),[54](#CIT0054),[55](#CIT0055),[68](#CIT0068)] and Peruvian [[64](#CIT0064)] and lower in the Ethiopian population [[58](#CIT0058)] for rs1799931. The East Asian participants had a relatively higher MAF for rs1799931 in the ‘All of Us’ Research Program. A higher MAF was observed for rs1799931 in the AT-DILI patients from both the South and Western Indian Sites, similar to other reports from South American, Southeast and East Asian TB populations [[62](#CIT0062),[71](#CIT0071),[74](#CIT0074)].

A minor allele having a frequency of ≥0.05 (5%) and <0.01 (1%) is termed a common variant and a rare variant respectively [[77](#CIT0077),[78](#CIT0078)]. All six *NAT2* SNPs were identified to be common variants among the TB patients of both Southern and Western Indian Sites, as well as in the literature data on the Indian TB population, suggesting the need for a six SNP *NAT2* genotyping panel in the Indian TB population. Our study provides strong evidence that geographically close countries and populations with similar genetic ancestry generally shared common MAF characteristics for the six *NAT2* SNPs. Similar MAF characteristics were also observed for TB patients from the literature reviews and participants who had similar genetic ancestry from the ‘All of Us’ Research Program database. For instance, Southeast East and East Asian TB populations and participants of the East Asian genetic ancestry generally tend to have relatively higher MAF for rs1041983 and rs1799931 and a very low MAF for rs1801280, rs1799929 and rs1208. Though SNPs with MAF ≥0.05 are preferentially used for most of the case-control studies of human diseases, rare functional SNPs have been potentially implicated in several disease causation [[79](#CIT0079)]. Hence future studies focussing on the trade-off between cost and benefits should be assessed in larger sample-sized populations where these SNPs occur in low frequency to decide on their exclusion from the genotyping panels in settings having cost constraints. Literature review and database comparison showed that the Indian TB population from Southern and Western India shared near common MAF characteristics with several South American/Latin American TB populations and Latin American genetic ancestry respectively concerning the *NAT2* SNPs rs1801280, rs1799929, rs1208 and rs1799931. Also, TB patients from Southeast/East Asian Countries and participants of East Asian ancestry have a higher MAF for rs1041983 and rs1799931 compared to TB patients from Southern and Western Indian Sites. Inter and intra-country discrepancies have been observed in the distribution of MAF of the six *NAT2* among a few AT-DILI and non-DILI TB populations, warranting a region/population-specific large sample size approach for interrogating *NAT2* SNP as potential biomarkers for AT-DILI. All the detected SNPs except for rs1799931 were found in HWE, which was similarly reported in an Indian study on patients with TB and normal healthy volunteers [[80](#CIT0080)]. Several studies have shown that there exists an LD between rs1801280 and rs1799929 [[64](#CIT0064),[80–83](#CIT0080)], rs1801280 and rs1208 [[80](#CIT0080),[82](#CIT0082),[83](#CIT0083)], rs1799929 and rs1208 [80, 82, 83] and rs1041983 and rs1799930 [[80](#CIT0080)]. This could probably explain the trend of relatively higher MAF for European and Middle Eastern participants and lower MAF for East Asian participants for rs1801280, rs1799929 and rs1208 in the ‘All of Us’ Research Program.

About one-third of the TB populations of both the study centres were *NAT2* slow acetylator phenotype, and *NAT2* slow acetylator had a higher frequency of developing AT-DILI in our study population. Several other *NAT2* genotyping studies using the minimum of six SNP panels assessed in the current study reported the percentage of *NAT2* slow acetylators as 31.2% in the Vietnamese TB population [[55](#CIT0055)], 38.3% in the Northern Indian TB population [[22](#CIT0022)], 27.2% and 31.9% in Brazilian TB population [[61](#CIT0061),[62](#CIT0062)]. Reports from other population studies have shown the proportion of *NAT2* slow acetylators to be 39.6% among Mexican healthy volunteers [[16](#CIT0016)] and among Central Asian populations such as 26% in Kirghizs, 32% in Uzbeks and 35% in Kazakhs [[84](#CIT0084)]. Higher frequencies of *NAT2* slow acetylators among TB patients who developed AT-DILI have been reported in other populations [[61](#CIT0061)]. *NAT2* slow acetylators are at higher risk for developing several antitubercular drug-induced ADRs [[85](#CIT0085),[86](#CIT0086)]. TB patients with *NAT2* slow acetylator status have been reported to display a trend of association with a mild increase in liver enzymes [[87](#CIT0087)]. TB patients with the *NAT2* SNPs rs1801280, rs1799929 and rs1208 had higher liver function test (LFT) values such as ALP, ALT, AST, total and direct bilirubin and higher odds of developing AT-DILI [[88](#CIT0088)]. *NAT2* genotyping may also reduce TB patients’ risk of early treatment failure [[35](#CIT0035)]. The *NAT2* SNPs and genotype implicated as risk factors in a particular TB population may be utilized as a preemptive biomarker for stratifying the patients for more frequent serial monitoring of the LFT, particularly during the first two months of ATT, to reduce the incidence of AT-DILI.

Our results reveal that the diversity of the *NAT2* gene concerning SNPs, alleles and genotypes could occur within a country’s population and across countries within a continent. Considerable diversity was also observed in MAF of the *NAT2* SNPs in the TB patients experiencing AT-DILI in the TB patients of our two geographically diverse Study Sites as well as among different TB populations from the literature review. This *NAT2* genetic diversity across geographical regions may be attributed to the shift from a hunter-gatherer lifestyle to the adaptation of agricultural practices, gene-environment interactions and gene flow [[84](#CIT0084),[89](#CIT0089),[90](#CIT0090)]. The *NAT2* phenotypes derived from these combinations of these *NAT2* SNPs could significantly influence the isoniazid pharmacokinetics and the development of AT-DILI [[91](#CIT0091)]. The higher MAF of the *NAT2* SNPs and the potential association of the SNPs and phenotype with AT-DILI reinforces the need for consideration of *NAT2* pharmacogenomics as a potential preemptive biomarker for monitoring LFT and AT-DILI in TB treatment. Most of the research investigations including ours suggest that more than a quarter of the TB population are *NAT2* slow acetylators, warranting the need for introspection into the incorporation of *NAT2* genotyping as a potential diagnostic tool in the clinical management of TB. By identifying patients with the *NAT2* slow acetylator genotype, healthcare providers can predict an increased risk of AT-DILI. This allows for personalized TB treatment strategies, such as adjusting drug dosages or developing a closer LFT monitoring strategy to minimize adverse effects and optimize efficacy. The lack of ethnic-specific information and sampling from other geographical areas such as Northern, Eastern and Central parts of India is another limitation in providing a pan-Indian perspective on *NAT2* heterogeneity in TB and AT-DILI patients. Other study shortcomings include that not all studies reported/genotyped all six *NAT2* SNPs and categorized the *NAT2* acetylator into three phenotypes. The majority of the literature on *NAT2* SNP frequency in TB patients was reported from Southeast and East Asian, Indian and South American countries. There is a paucity of literature on *NAT2* genotyping from several other high-burden TB countries, which limited us to presenting a true global picture of *NAT2* heterogeneity. Nevertheless, the vast amount of data collected enabled us to provide a comprehensive picture of the *NAT2* genomic landscape across different parts of the world, thus building up a useful resource of frequency data for further studies focusing on *NAT2* molecular epidemiology and developing *NAT2* genotype as a potential biomarker for LFT monitoring during isoniazid-based ATT.

Further well-designed multicentric studies with *NAT2* genotyping among large and geographically and ethnically diverse TB and AT-DILI populations with a dynamic follow-up could overcome the limitations of our research and several other literatures on TB patients that were included in the current study of having smaller AT-DILI populations. Randomized controlled trials assessing the benefits of *NAT2* pharmacogenomic guided LFT monitoring and isoniazid dosing regimen are imperative for deriving conclusive evidence on the benefits of incorporating *NAT2* genotyping in the clinical management of TB patients.

## 5. Conclusion

The current study provides an overview of the pharmacogenomic landscape of *NAT2* SNPs among TB and AT-DILI patients across two different geographical regions of India, comparing it to TB populations from other parts of the world and populations with different genetic ancestries from the ‘All of Us’ Research Program database. Several geographically close regions and populations with similar genetic ancestries shared near similar *NAT2* SNP MAF characteristics, though diversities were also observed within a country population. All the six *NAT2* SNPs rs1041983, rs1801280, rs1799929, rs1799930, rs1208 and rs1799931 were identified as common variants in most of the population. Though ample evidence is available for using *NAT2* phenotype as a potential biomarker for AT-DILI, further validation of the different *NAT2* SNPs as risk factors for AT-DILI are required due to the significant MAF diversity observed among AT-DILI and controls within a region and across various regions. *NAT2* preemptive pharmacogenomic guided therapy could be a potential tool for monitoring and risk stratification of ADRs such as AT-DILI in TB patients, exemplified by the relatively higher proportion of AT-DILI among the *NAT2* slow acetylators in our study.

## Supplementary Material

## Acknowledgements

L.T. thanks the Indian Council of Medical Research for providing Senior Research Fellowship (No.45/25/2020/PHA/BMS). C. is thankful to the Junior Research Fellowship from ICMR (F.No.5/8/5/45/multicentric study/2019/ECD-1). The authors also acknowledge Mr. Francis Massey’s contributions for assistance in collection of samples. M.R. is also thankful to the Department of Science & Technology- Fund for Improvement of S&T Infrastructure in Universities and Higher Educational Institutions (DST-FIST) (TPN: 32196) for providing Quant 5 Studio real time PCR (Applied Biosystems, USA). The authors thank the ‘All of Us’ Research Program and its participants.

## Funding Statement

This work was supported by funding from the Indian Council of Medical Research [grant F.No.5/8/5/45/multicentric study/2019/ECD-1] to M.R. Open access funding is provided by the Manipal Academy of Higher Education (MAHE).

## Ethical approval

The study received institutional ethics committee approval from the two Study Sites (Kasturba Hospital and Kasturba Medical College IEC No. 243/2019 and AIIMS Jodhpur IEC No. 2019/3522).

## Author contributions

M.R. was involved in funding acquisition for the project. M.R. and L.T. were involved in the conceptualization of the study. C., L.T., M.B., Y.B. and M.M. were involved in patient recruitment, data collection and sample processing. L.T. was involved in data curation. L.T., S.K. and M.R. were involved in laboratory experiments. Analysis and interpretation of data were made by L.T. and M.R. L.T. wrote the first draft of the paper. C.S.S.V., N.D., P.B., M.V., K.S., M.B. and M.R. critically revised the manuscript. All authors have read and approved the final manuscript.

## Disclosure statement

No potential conflict of interest was reported by the author(s).

## Data availability statement

The data that support the findings of this study are available from the corresponding author upon reasonable request.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

The data that support the findings of this study are available from the corresponding author upon reasonable request.

### Supplementary Materials

### Data Availability Statement

The data that support the findings of this study are available from the corresponding author upon reasonable request.

## References

1. Hein DW, Doll MA, Rustan TD, et al. Metabolic activation and deactivation of arylamine carcinogens by ­recombinant human NAT1 and polymorphic NAT2 acetyltransferases. Carcinogenesis. 1993;14(8):1633–1638. doi: 10.1093/carcin/14.8.1633.  [DOI](https://doi.org/10.1093/carcin/14.8.1633) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8353847/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Carcinogenesis&title=Metabolic%20activation%20and%20deactivation%20of%20arylamine%20carcinogens%20by%20%C2%ADrecombinant%20human%20NAT1%20and%20polymorphic%20NAT2%20acetyltransferases&volume=14&issue=8&publication_year=1993&pages=1633-1638&pmid=8353847&doi=10.1093/carcin/14.8.1633&)

2. Kato R. Metabolic activation of mutagenic heterocyclic aromatic amines from protein pyrolysates. Crit Rev Toxicol. 1986;16(4):307–348. doi: 10.3109/10408448609037466.  [DOI](https://doi.org/10.3109/10408448609037466) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/3519087/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Crit%20Rev%20Toxicol&title=Metabolic%20activation%20of%20mutagenic%20heterocyclic%20aromatic%20amines%20from%20protein%20pyrolysates&volume=16&issue=4&publication_year=1986&pages=307-348&pmid=3519087&doi=10.3109/10408448609037466&)

3. Minchin RF, Reeves PT, Teitel CH, et al. N-and O-acetylation of aromatic and heterocyclic amine carcinogens by human monomorphic and polymorphic acetyltransferases expressed in COS-1 cells. Biochem Biophys Res Commun. 1992;185(3):839–844. doi: 10.1016/0006-291x(92)91703-s.  [DOI](https://doi.org/10.1016/0006-291x(92)91703-s) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1627140/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20Biophys%20Res%20Commun&title=N-and%20O-acetylation%20of%20aromatic%20and%20heterocyclic%20amine%20carcinogens%20by%20human%20monomorphic%20and%20polymorphic%20acetyltransferases%20expressed%20in%20COS-1%20cells&volume=185&issue=3&publication_year=1992&pages=839-844&pmid=1627140&doi=10.1016/0006-291x(92)91703-s&)

4. Zenser TV, Lakshmi VM, Rustan TD, et al. Human N-acetylation of benzidine: role of NAT1 and NAT2. Cancer Res. 1996;56(17):3941–3947.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/8752161/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=Human%20N-acetylation%20of%20benzidine:%20role%20of%20NAT1%20and%20NAT2&volume=56&issue=17&publication_year=1996&pages=3941-3947&pmid=8752161&)

5. Butcher NJ, Boukouvala S, Sim E, et al. Pharmacogenetics of the arylamine N-acetyltransferases. Pharmacogenomics J. 2002;2(1):30–42. doi: 10.1038/sj.tpj.6500053.  [DOI](https://doi.org/10.1038/sj.tpj.6500053) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11990379/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Pharmacogenetics%20of%20the%20arylamine%20N-acetyltransferases&volume=2&issue=1&publication_year=2002&pages=30-42&pmid=11990379&doi=10.1038/sj.tpj.6500053&)

6. Jančová P, Šiller M.. Phase II drug metabolism. In: James P editors. Topics on drug metabolism. London, United Kingdom: InTech; 2012. p. 35–60. Available from: https://www.intechopen.com/chapters/29241  [https://www.intechopen.com/chapters/29241](https://www.intechopen.com/chapters/29241) | [Google Scholar](https://scholar.google.com/scholar_lookup?title=Topics%20on%20drug%20metabolism&publication_year=2012&)

7. Klein DJ, Boukouvala S, McDonagh EM, et al. PharmGKB summary: isoniazid pathway, pharmacokinetics. Pharmacogenet Genomics. 2016;26(9):436–444. doi: 10.1097/FPC.0000000000000232.  [DOI](https://doi.org/10.1097/FPC.0000000000000232) | [PMC free article](/articles/PMC4970941/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27232112/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=PharmGKB%20summary:%20isoniazid%20pathway,%20pharmacokinetics&volume=26&issue=9&publication_year=2016&pages=436-444&pmid=27232112&doi=10.1097/FPC.0000000000000232&)

8. Unissa AN, Subbian S, Hanna LE, et al. Overview on mechanisms of isoniazid action and resistance in Mycobacterium tuberculosis. Infect Genet Evol. 2016;45:474–492. doi: 10.1016/j.meegid.2016.09.004.  [DOI](https://doi.org/10.1016/j.meegid.2016.09.004) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27612406/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Infect%20Genet%20Evol&title=Overview%20on%20mechanisms%20of%20isoniazid%20action%20and%20resistance%20in%20Mycobacterium%20tuberculosis&volume=45&publication_year=2016&pages=474-492&pmid=27612406&doi=10.1016/j.meegid.2016.09.004&)

9. Hong BL, D’Cunha R, Li P, et al. A systematic review and meta-analysis of isoniazid pharmacokinetics in healthy volunteers and patients with tuberculosis. Clin Ther. 2020;42(11):e220–e241. doi: 10.1016/j.clinthera.2020.09.009.  [DOI](https://doi.org/10.1016/j.clinthera.2020.09.009) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33032843/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Ther&title=A%20systematic%20review%20and%20meta-analysis%20of%20isoniazid%20pharmacokinetics%20in%20healthy%20volunteers%20and%20patients%20with%20tuberculosis&volume=42&issue=11&publication_year=2020&pages=e220-e241&pmid=33032843&doi=10.1016/j.clinthera.2020.09.009&)

10. Trentalange A, Borgogno E, Motta I, et al. Rifampicin and isoniazid maximal concentrations are below efficacy-associated thresholds in the majority of patients: time to increase the doses? Int J Antimicrob Agents. 2021;57(3):106297. doi: 10.1016/j.ijantimicag.2021.106297.  [DOI](https://doi.org/10.1016/j.ijantimicag.2021.106297) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33539932/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Antimicrob%20Agents&title=Rifampicin%20and%20isoniazid%20maximal%20concentrations%20are%20below%20efficacy-associated%20thresholds%20in%20the%20majority%20of%20patients:%20time%20to%20increase%20the%20doses?&volume=57&issue=3&publication_year=2021&pages=106297&pmid=33539932&doi=10.1016/j.ijantimicag.2021.106297&)

11. Sekaggya-Wiltshire C, von Braun A, Lamorde M, et al. Delayed Sputum Culture Conversion in Tuberculosis-Human Immunodeficiency Virus-Coinfected Patients With Low Isoniazid and Rifampicin Concentrations. Clin Infect Dis. 2018;67(5):708–716. doi: 10.1093/cid/ciy179.  [DOI](https://doi.org/10.1093/cid/ciy179) | [PMC free article](/articles/PMC6094003/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29514175/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Infect%20Dis&title=Delayed%20Sputum%20Culture%20Conversion%20in%20Tuberculosis-Human%20Immunodeficiency%20Virus-Coinfected%20Patients%20With%20Low%20Isoniazid%20and%20Rifampicin%20Concentrations&volume=67&issue=5&publication_year=2018&pages=708-716&pmid=29514175&doi=10.1093/cid/ciy179&)

12. Aït Moussa L, El Bouazzi O, Serragui S, et al. Rifampicin and isoniazid plasma concentrations in relation to ­adverse reactions in tuberculosis patients: a retrospective analysis. Ther Adv Drug Saf. 2016;7(6):239–247. doi: 10.1177/2042098616667704.  [DOI](https://doi.org/10.1177/2042098616667704) | [PMC free article](/articles/PMC5117155/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27904742/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ther%20Adv%20Drug%20Saf&title=Rifampicin%20and%20isoniazid%20plasma%20concentrations%20in%20relation%20to%20%C2%ADadverse%20reactions%20in%20tuberculosis%20patients:%20a%20retrospective%20analysis&volume=7&issue=6&publication_year=2016&pages=239-247&pmid=27904742&doi=10.1177/2042098616667704&)

13. Seng KY, Hee KH, Soon GH, et al. Population pharmacokinetic analysis of isoniazid, acetylisoniazid, and isonicotinic acid in healthy volunteers. Antimicrob Agents Chemother. 2015;59(11):6791–6799. doi: 10.1128/AAC.01244-15.  [DOI](https://doi.org/10.1128/AAC.01244-15) | [PMC free article](/articles/PMC4604378/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26282412/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Antimicrob%20Agents%20Chemother&title=Population%20pharmacokinetic%20analysis%20of%20isoniazid,%20acetylisoniazid,%20and%20isonicotinic%20acid%20in%20healthy%20volunteers&volume=59&issue=11&publication_year=2015&pages=6791-6799&pmid=26282412&doi=10.1128/AAC.01244-15&)

14. Cho YS, Jang TW, Kim HJ, et al. Isoniazid population pharmacokinetics and dose recommendation for Korean patients with tuberculosis based on target attainment analysis. J Clin Pharmacol. 2021;61(12):1567–1578. doi: 10.1002/jcph.1931.  [DOI](https://doi.org/10.1002/jcph.1931) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34157153/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Pharmacol&title=Isoniazid%20population%20pharmacokinetics%20and%20dose%20recommendation%20for%20Korean%20patients%20with%20tuberculosis%20based%20on%20target%20attainment%20analysis&volume=61&issue=12&publication_year=2021&pages=1567-1578&pmid=34157153&doi=10.1002/jcph.1931&)

15. Thomas L, Raju AP, Mallayasamy S, et al. Precision medicine strategies to improve isoniazid therapy in patients with tuberculosis. Eur J Drug Metab Pharmacokinet. 2024;49(5):541–557. doi: 10.1007/s13318-024-00910-7.  [DOI](https://doi.org/10.1007/s13318-024-00910-7) | [PMC free article](/articles/PMC11365851/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/39153028/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Drug%20Metab%20Pharmacokinet&title=Precision%20medicine%20strategies%20to%20improve%20isoniazid%20therapy%20in%20patients%20with%20tuberculosis&volume=49&issue=5&publication_year=2024&pages=541-557&pmid=39153028&doi=10.1007/s13318-024-00910-7&)

16. Salazar-González R, Gómez R, Romano-Moreno S, et al. Expression of NAT2 in immune system cells and the relation of NAT2 gene polymorphisms in the anti-tuberculosis therapy in Mexican mestizo population. Mol Biol Rep. 2014;41(12):7833–7843. doi: 10.1007/s11033-014-3677-5.  [DOI](https://doi.org/10.1007/s11033-014-3677-5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25163630/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Biol%20Rep&title=Expression%20of%20NAT2%20in%20immune%20system%20cells%20and%20the%20relation%20of%20NAT2%20gene%20polymorphisms%20in%20the%20anti-tuberculosis%20therapy%20in%20Mexican%20mestizo%20population&volume=41&issue=12&publication_year=2014&pages=7833-7843&pmid=25163630&doi=10.1007/s11033-014-3677-5&)

17. Portevin D, Via LE, Eum S, et al. Natural killer cells are recruited during pulmonary tuberculosis and their ex vivo responses to mycobacteria vary between healthy human donors in association with KIR haplotype. Cell Microbiol. 2012;14(11):1734–1744. doi: 10.1111/j.1462-5822.2012.01834.x.  [DOI](https://doi.org/10.1111/j.1462-5822.2012.01834.x) | [PMC free article](/articles/PMC3503254/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22788220/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Microbiol&title=Natural%20killer%20cells%20are%20recruited%20during%20pulmonary%20tuberculosis%20and%20their%20ex%20vivo%20responses%20to%20mycobacteria%20vary%20between%20healthy%20human%20donors%20in%20association%20with%20KIR%20haplotype&volume=14&issue=11&publication_year=2012&pages=1734-1744&pmid=22788220&doi=10.1111/j.1462-5822.2012.01834.x&)

18. Guerra C, Johal K, Morris D, et al. Control of Mycobacterium tuberculosis growth by activated natural killer cells. Clin Exp Immunol. 2012;168(1):142–152. doi: 10.1111/j.1365-2249.2011.04552.x.  [DOI](https://doi.org/10.1111/j.1365-2249.2011.04552.x) | [PMC free article](/articles/PMC3390505/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22385249/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Exp%20Immunol&title=Control%20of%20Mycobacterium%20tuberculosis%20growth%20by%20activated%20natural%20killer%20cells&volume=168&issue=1&publication_year=2012&pages=142-152&pmid=22385249&doi=10.1111/j.1365-2249.2011.04552.x&)

19. Crews KR, Hicks JK, Pui CH, et al. Pharmacogenomics and individualized medicine: translating science into practice. Clin Pharmacol Ther. 2012;92(4):467–475. doi: 10.1038/clpt.2012.120.  [DOI](https://doi.org/10.1038/clpt.2012.120) | [PMC free article](/articles/PMC3589526/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22948889/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Pharmacogenomics%20and%20individualized%20medicine:%20translating%20science%20into%20practice&volume=92&issue=4&publication_year=2012&pages=467-475&pmid=22948889&doi=10.1038/clpt.2012.120&)

20. Eichler EE, Nickerson DA, Altshuler D, et al. Completing the map of human genetic variation. Nature. 2007;447(7141):161–165. doi: 10.1038/447161a.  [DOI](https://doi.org/10.1038/447161a) | [PMC free article](/articles/PMC2685471/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17495918/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Completing%20the%20map%20of%20human%20genetic%20variation&volume=447&issue=7141&publication_year=2007&pages=161-165&pmid=17495918&doi=10.1038/447161a&)

21. Gutiérrez-Virgen JE, Piña-Pozas M, Hernández-Tobías EA, et al. NAT2 global landscape: genetic diversity and acetylation statuses from a systematic review. PLoS One. 2023;18(4):e0283726. doi: 10.1371/journal.pone.0283726.  [DOI](https://doi.org/10.1371/journal.pone.0283726) | [PMC free article](/articles/PMC10079069/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/37023111/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=NAT2%20global%20landscape:%20genetic%20diversity%20and%20acetylation%20statuses%20from%20a%20systematic%20review&volume=18&issue=4&publication_year=2023&pages=e0283726&pmid=37023111&doi=10.1371/journal.pone.0283726&)

22. Adole PS, Kharbanda PS, Sharma S.. N-acetyltransferase 2 (NAT2) gene polymorphism as a predisposing factor for phenytoin intoxication in tuberculous meningitis or tuberculoma patients having seizures - a pilot study. Indian J Med Res. 2016;143(5):581–590. doi: 10.4103/0971-5916.187106.  [DOI](https://doi.org/10.4103/0971-5916.187106) | [PMC free article](/articles/PMC4989831/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27488001/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Indian%20J%20Med%20Res&title=N-acetyltransferase%202%20(NAT2)%20gene%20polymorphism%20as%20a%20predisposing%20factor%20for%20phenytoin%20intoxication%20in%20tuberculous%20meningitis%20or%20tuberculoma%20patients%20having%20seizures%20-%20a%20pilot%20study&volume=143&issue=5&publication_year=2016&pages=581-590&pmid=27488001&doi=10.4103/0971-5916.187106&)

23. Ho HT, Wang TH, Hsiong CH, et al. The NAT2 tag SNP rs1495741 correlates with the susceptibility of antituberculosis drug-induced hepatotoxicity. Pharmacogenet Genomics. 2013;23(4):200–207. doi: 10.1097/FPC.0b013e32835e95e1.  [DOI](https://doi.org/10.1097/FPC.0b013e32835e95e1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23407048/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=The%20NAT2%20tag%20SNP%20rs1495741%20correlates%20with%20the%20susceptibility%20of%20antituberculosis%20drug-induced%20hepatotoxicity&volume=23&issue=4&publication_year=2013&pages=200-207&pmid=23407048&doi=10.1097/FPC.0b013e32835e95e1&)

24. Hein DW. N-acetyltransferase SNPs: emerging concepts serve as a paradigm for understanding complexities of personalized medicine. Expert Opin Drug Metab Toxicol. 2009;5(4):353–366. doi: 10.1517/17425250902877698.  [DOI](https://doi.org/10.1517/17425250902877698) | [PMC free article](/articles/PMC2762189/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19379125/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Expert%20Opin%20Drug%20Metab%20Toxicol&title=N-acetyltransferase%20SNPs:%20emerging%20concepts%20serve%20as%20a%20paradigm%20for%20understanding%20complexities%20of%20personalized%20medicine&volume=5&issue=4&publication_year=2009&pages=353-366&pmid=19379125&doi=10.1517/17425250902877698&)

25. Rajasekaran M, Abirami S, Chen C.. Effects of single ­nucleotide polymorphisms on human N-acetyltransferase 2 structure and dynamics by molecular dynamics simulation. PLoS One. 2011;6(9):e25801. doi: 10.1371/journal.pone.0025801.  [DOI](https://doi.org/10.1371/journal.pone.0025801) | [PMC free article](/articles/PMC3183086/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21980537/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Effects%20of%20single%20%C2%ADnucleotide%20polymorphisms%20on%20human%20N-acetyltransferase%202%20structure%20and%20dynamics%20by%20molecular%20dynamics%20simulation&volume=6&issue=9&publication_year=2011&pages=e25801&pmid=21980537&doi=10.1371/journal.pone.0025801&)

26. Zang Y, Zhao S, Doll MA, et al. The T341C (Ile114Thr) polymorphism of N-acetyltransferase 2 yields slow acetylator phenotype by enhanced protein degradation. Pharmacogenetics. 2004;14(11):717–723. doi: 10.1097/00008571-200411000-00002.  [DOI](https://doi.org/10.1097/00008571-200411000-00002) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15564878/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=The%20T341C%20(Ile114Thr)%20polymorphism%20of%20N-acetyltransferase%202%20yields%20slow%20acetylator%20phenotype%20by%20enhanced%20protein%20degradation&volume=14&issue=11&publication_year=2004&pages=717-723&pmid=15564878&doi=10.1097/00008571-200411000-00002&)

27. Fretland AJ, Leff MA, Doll MA, et al. Functional characterization of human N-acetyltransferase 2 (NAT2) single ­nucleotide polymorphisms. Pharmacogenetics. 2001;11(3):207–215. doi: 10.1097/00008571-200104000-00004.  [DOI](https://doi.org/10.1097/00008571-200104000-00004) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11337936/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Functional%20characterization%20of%20human%20N-acetyltransferase%202%20(NAT2)%20single%20%C2%ADnucleotide%20polymorphisms&volume=11&issue=3&publication_year=2001&pages=207-215&pmid=11337936&doi=10.1097/00008571-200104000-00004&)

28. Zang Y, Doll MA, Zhao S, et al. Functional characterization of single-nucleotide polymorphisms and haplotypes of human N-acetyltransferase 2. Carcinogenesis. 2007;28(8):1665–1671. doi: 10.1093/carcin/bgm085.  [DOI](https://doi.org/10.1093/carcin/bgm085) | [PMC free article](/articles/PMC2085371/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17434923/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Carcinogenesis&title=Functional%20characterization%20of%20single-nucleotide%20polymorphisms%20and%20haplotypes%20of%20human%20N-acetyltransferase%202&volume=28&issue=8&publication_year=2007&pages=1665-1671&pmid=17434923&doi=10.1093/carcin/bgm085&)

29. McDonagh EM, Boukouvala S, Aklillu E, et al. PharmGKB summary: very important pharmacogene information for N-acetyltransferase 2. Pharmacogenet Genomics. 2014;24(8):409–425. doi: 10.1097/FPC.0000000000000062.  [DOI](https://doi.org/10.1097/FPC.0000000000000062) | [PMC free article](/articles/PMC4109976/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24892773/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=PharmGKB%20summary:%20very%20important%20pharmacogene%20information%20for%20N-acetyltransferase%202&volume=24&issue=8&publication_year=2014&pages=409-425&pmid=24892773&doi=10.1097/FPC.0000000000000062&)

30. Thomas L, Raju AP, Chaitra, et al. Influence of N-acetyltransferase 2 (NAT2) genotype/single nucleotide polymorphisms on clearance of isoniazid in tuberculosis patients: a systematic review of population pharmacokinetic models. Eur J Clin Pharmacol 2022;78(10):1535–1553. doi: 10.1007/s00228-022-03362-722.  [DOI](https://doi.org/10.1007/s00228-022-03362-722) | [PMC free article](/articles/PMC9482569/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35852584/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Clin%20Pharmacol&title=Influence%20of%20N-acetyltransferase%202%20(NAT2)%20genotype/single%20nucleotide%20polymorphisms%20on%20clearance%20of%20isoniazid%20in%20tuberculosis%20patients:%20a%20systematic%20review%20of%20population%20pharmacokinetic%20models&volume=78&issue=10&publication_year=2022&pages=1535-1553&pmid=35852584&doi=10.1007/s00228-022-03362-722&)

31. Wang N, Chen X, Hao Z, et al. Incidence and temporal trend of antituberculosis drug-induced liver injury: a systematic review and meta-analysis. J Trop Med. 2022;2022:8266878–8266810. doi: 10.1155/2022/8266878.  [DOI](https://doi.org/10.1155/2022/8266878) | [PMC free article](/articles/PMC9553535/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36249736/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Trop%20Med&title=Incidence%20and%20temporal%20trend%20of%20antituberculosis%20drug-induced%20liver%20injury:%20a%20systematic%20review%20and%20meta-analysis&volume=2022&publication_year=2022&pages=8266878-8266810&pmid=36249736&doi=10.1155/2022/8266878&)

32. Devarbhavi H, Singh R, Patil M, et al. Outcome and ­determinants of mortality in 269 patients with combination anti-tuberculosis drug-induced liver injury. J Gastroenterol Hepatol. 2013;28(1):161–167. doi: 10.1111/j.1440-1746.2012.07279.x.  [DOI](https://doi.org/10.1111/j.1440-1746.2012.07279.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23020522/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Gastroenterol%20Hepatol&title=Outcome%20and%20%C2%ADdeterminants%20of%20mortality%20in%20269%20patients%20with%20combination%20anti-tuberculosis%20drug-induced%20liver%20injury&volume=28&issue=1&publication_year=2013&pages=161-167&pmid=23020522&doi=10.1111/j.1440-1746.2012.07279.x&)

33. Abbara A, Chitty S, Roe JK, et al. Drug-induced liver ­injury from antituberculous treatment: a retrospective study from a large TB centre in the UK. BMC Infect Dis. 2017;17(1):231. doi: 10.1186/s12879-017-2330-z.  [DOI](https://doi.org/10.1186/s12879-017-2330-z) | [PMC free article](/articles/PMC5366108/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28340562/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Infect%20Dis&title=Drug-induced%20liver%20%C2%ADinjury%20from%20antituberculous%20treatment:%20a%20retrospective%20study%20from%20a%20large%20TB%20centre%20in%20the%20UK&volume=17&issue=1&publication_year=2017&pages=231&pmid=28340562&doi=10.1186/s12879-017-2330-z&)

34. Zhang M, Wang S, Wilffert B, et al. The association between the NAT2 genetic polymorphisms and risk of DILI during anti-TB treatment: a systematic review and meta-analysis. Br J Clin Pharmacol. 2018;84(12):2747–2760. doi: 10.1111/bcp.13722.  [DOI](https://doi.org/10.1111/bcp.13722) | [PMC free article](/articles/PMC6256008/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30047605/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=The%20association%20between%20the%20NAT2%20genetic%20polymorphisms%20and%20risk%20of%20DILI%20during%20anti-TB%20treatment:%20a%20systematic%20review%20and%20meta-analysis&volume=84&issue=12&publication_year=2018&pages=2747-2760&pmid=30047605&doi=10.1111/bcp.13722&)

35. Azuma J, Ohno M, Kubota R, et al. NAT2 genotype guided regimen reduces isoniazid-induced liver injury and early treatment failure in the 6-month four-drug standard treatment of tuberculosis: a randomized controlled trial for pharmacogenetics-based therapy. Eur J Clin Pharmacol. 2013;69(5):1091–1101. doi: 10.1007/s00228-012-1429-9.  [DOI](https://doi.org/10.1007/s00228-012-1429-9) | [PMC free article](/articles/PMC3641305/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23150149/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Clin%20Pharmacol&title=NAT2%20genotype%20guided%20regimen%20reduces%20isoniazid-induced%20liver%20injury%20and%20early%20treatment%20failure%20in%20the%206-month%20four-drug%20standard%20treatment%20of%20tuberculosis:%20a%20randomized%20controlled%20trial%20for%20pharmacogenetics-based%20therapy&volume=69&issue=5&publication_year=2013&pages=1091-1101&pmid=23150149&doi=10.1007/s00228-012-1429-9&)

36. Lopes MQP, Teixeira RLF, Cabello PH, et al. Human N-acetyltransferase 2 (NAT2) gene variability in Brazilian populations from different geographical areas. Front Pharmacol. 2023;14:1278720. doi: 10.3389/fphar.2023.1278720.  [DOI](https://doi.org/10.3389/fphar.2023.1278720) | [PMC free article](/articles/PMC10684696/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/38035025/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front%20Pharmacol&title=Human%20N-acetyltransferase%202%20(NAT2)%20gene%20variability%20in%20Brazilian%20populations%20from%20different%20geographical%20areas&volume=14&publication_year=2023&pages=1278720&pmid=38035025&doi=10.3389/fphar.2023.1278720&)

37. Sabbagh A, Darlu P, Crouau-Roy B, et al. Arylamine N-acetyltransferase 2 (NAT2) genetic diversity and traditional subsistence: a worldwide population survey. PLoS One. 2011;6(4):e18507. doi: 10.1371/journal.pone.0018507.  [DOI](https://doi.org/10.1371/journal.pone.0018507) | [PMC free article](/articles/PMC3071824/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21494681/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Arylamine%20N-acetyltransferase%202%20(NAT2)%20genetic%20diversity%20and%20traditional%20subsistence:%20a%20worldwide%20population%20survey&volume=6&issue=4&publication_year=2011&pages=e18507&pmid=21494681&doi=10.1371/journal.pone.0018507&)

38. Dyson T. A population history of India: from the first modern people to the present day. Popul Dev Rev. 2018;45(1):249–251.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Popul%20Dev%20Rev&title=A%20population%20history%20of%20India:%20from%20the%20first%20modern%20people%20to%20the%20present%20day&volume=45&issue=1&publication_year=2018&pages=249-251&)

39. Mastana SS. Unity in diversity: an overview of the ­genomic anthropology of India. Ann Hum Biol. 2014;41(4):287–299. doi: 10.3109/03014460.2014.922615.  [DOI](https://doi.org/10.3109/03014460.2014.922615) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24932744/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Hum%20Biol&title=Unity%20in%20diversity:%20an%20overview%20of%20the%20%C2%ADgenomic%20anthropology%20of%20India&volume=41&issue=4&publication_year=2014&pages=287-299&pmid=24932744&doi=10.3109/03014460.2014.922615&)

40. Sahana S, Bhoyar RC, Sivadas A, et al. Pharmacogenomic landscape of Indian population using whole genomes. Clin Transl Sci. 2022;15(4):866–877. doi: 10.1111/cts.13153.  [DOI](https://doi.org/10.1111/cts.13153) | [PMC free article](/articles/PMC9010271/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35338580/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Transl%20Sci&title=Pharmacogenomic%20landscape%20of%20Indian%20population%20using%20whole%20genomes&volume=15&issue=4&publication_year=2022&pages=866-877&pmid=35338580&doi=10.1111/cts.13153&)

41. Nahar R, Deb R, Saxena R, et al. Variability in CYP2C9 allele frequency: a pilot study of its predicted impact on warfarin response among healthy South and North Indians. Pharmacol Rep. 2013;65(1):187–194. doi: 10.1016/s1734-1140(13)70977-0.  [DOI](https://doi.org/10.1016/s1734-1140(13)70977-0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23563037/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacol%20Rep&title=Variability%20in%20CYP2C9%20allele%20frequency:%20a%20pilot%20study%20of%20its%20predicted%20impact%20on%20warfarin%20response%20among%20healthy%20South%20and%20North%20Indians&volume=65&issue=1&publication_year=2013&pages=187-194&pmid=23563037&doi=10.1016/s1734-1140(13)70977-0&)

42. Giri AK, Khan NM, Basu A, et al. Pharmacogenetic landscape of clopidogrel in north Indians suggest distinct interpopulation differences in allele frequencies. Pharmacogenomics. 2014;15(5):643–653. doi: 10.2217/pgs.13.241.  [DOI](https://doi.org/10.2217/pgs.13.241) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24798721/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Pharmacogenetic%20landscape%20of%20clopidogrel%20in%20north%20Indians%20suggest%20distinct%20interpopulation%20differences%20in%20allele%20frequencies&volume=15&issue=5&publication_year=2014&pages=643-653&pmid=24798721&doi=10.2217/pgs.13.241&)

43. All of Us Research Program Genomics Investigators . Genomic data in the All of Us Research Program. Nature. 2024;627(8003):340–346. doi: 10.1038/s41586-023-06957-x.  [DOI](https://doi.org/10.1038/s41586-023-06957-x) | [PMC free article](/articles/PMC10937371/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/38374255/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Genomic%20data%20in%20the%20All%20of%20Us%20Research%20Program&volume=627&issue=8003&publication_year=2024&pages=340-346&pmid=38374255&doi=10.1038/s41586-023-06957-x&)

44. Saukkonen JJ, Cohn DL, Jasmer RM, et al. An official ATS statement: hepatotoxicity of antituberculosis therapy. Am J Respir Crit Care Med. 2006;174(8):935–952. doi: 10.1164/rccm.200510-1666ST.  [DOI](https://doi.org/10.1164/rccm.200510-1666ST) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17021358/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Respir%20Crit%20Care%20Med&title=An%20official%20ATS%20statement:%20hepatotoxicity%20of%20antituberculosis%20therapy&volume=174&issue=8&publication_year=2006&pages=935-952&pmid=17021358&doi=10.1164/rccm.200510-1666ST&)

45. Nahid P, Dorman SE, Alipanah N, et al. Official American Thoracic Society/Centers for Disease Control and Prevention/Infectious Diseases Society of America Clinical Practice Guidelines: treatment of drug-susceptible tuberculosis. Clin Infect Dis. 2016;63(7):e147–e195. doi: 10.1093/cid/ciw376.  [DOI](https://doi.org/10.1093/cid/ciw376) | [PMC free article](/articles/PMC6590850/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27516382/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Infect%20Dis&title=Official%20American%20Thoracic%20Society/Centers%20for%20Disease%20Control%20and%20Prevention/Infectious%20Diseases%20Society%20of%20America%20Clinical%20Practice%20Guidelines:%20treatment%20of%20drug-susceptible%20tuberculosis&volume=63&issue=7&publication_year=2016&pages=e147-e195&pmid=27516382&doi=10.1093/cid/ciw376&)

46. Chalasani N, Bonkovsky HL, Fontana R, et al. Features and outcomes of 899 patients with drug-induced liver injury: the DILIN prospective study. Gastroenterology. 2015;148(7):1340–1352.e7. doi: 10.1053/j.gastro.2015.03.006.  [DOI](https://doi.org/10.1053/j.gastro.2015.03.006) | [PMC free article](/articles/PMC4446235/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25754159/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastroenterology&title=Features%20and%20outcomes%20of%20899%20patients%20with%20drug-induced%20liver%20injury:%20the%20DILIN%20prospective%20study&volume=148&issue=7&publication_year=2015&pages=1340-1352.e7&pmid=25754159&doi=10.1053/j.gastro.2015.03.006&)

47. Conway LP, Rendo V, Correia MSP, et al. Unexpected acetylation of endogenous aliphatic amines by arylamine N-acetyltransferase NAT2. Angew Chem Int Ed Engl. 2020;59(34):14342–14346. doi: 10.1002/anie.202005915.  [DOI](https://doi.org/10.1002/anie.202005915) | [PMC free article](/articles/PMC7497018/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32497306/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Angew%20Chem%20Int%20Ed%20Engl&title=Unexpected%20acetylation%20of%20endogenous%20aliphatic%20amines%20by%20arylamine%20N-acetyltransferase%20NAT2&volume=59&issue=34&publication_year=2020&pages=14342-14346&pmid=32497306&doi=10.1002/anie.202005915&)

48. Human NAT2 Alleles (Haplotypes) [Internet] . Alexandroupolis: Department of Molecular Biology and Genetics, Democritus University of Thrace; 2024. [accessed 2024 Aug 1]. Available from: https://nat.mbg.duth.gr/Homepage.htm  [https://nat.mbg.duth.gr/Homepage.htm](https://nat.mbg.duth.gr/Homepage.htm)

49. All of Us Research Hub . SNV/indel variants; 2024. [accessed 2024 July 20]. Available from: https://databrowser.researchallofus.org/variants  [https://databrowser.researchallofus.org/variants](https://databrowser.researchallofus.org/variants)

50. Kockum I, Huang J, Stridh P.. Overview of genotyping technologies and methods. Curr Protoc. 2023;3(4):e727. doi: 10.1002/cpz1.727.  [DOI](https://doi.org/10.1002/cpz1.727) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/37026777/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Protoc&title=Overview%20of%20genotyping%20technologies%20and%20methods&volume=3&issue=4&publication_year=2023&pages=e727&pmid=37026777&doi=10.1002/cpz1.727&)

51. Institut Català d’Oncologia . SNPStats: your web tool for SNP analysis; 2024. [accessed 2024 Jul 22]. Available from: https://www.snpstats.net/start.htm  [https://www.snpstats.net/start.htm](https://www.snpstats.net/start.htm)

52. VanLiere JM, Rosenberg NA.. Mathematical properties of the r2 measure of linkage disequilibrium. Theor Popul Biol. 2008;74(1):130–137. doi: 10.1016/j.tpb.2008.05.006.  [DOI](https://doi.org/10.1016/j.tpb.2008.05.006) | [PMC free article](/articles/PMC2580747/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18572214/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Theor%20Popul%20Biol&title=Mathematical%20properties%20of%20the%20r2%20measure%20of%20linkage%20disequilibrium&volume=74&issue=1&publication_year=2008&pages=130-137&pmid=18572214&doi=10.1016/j.tpb.2008.05.006&)

53. Tang D, Chen M, Huang X, et al. SRplot: a free online platform for data visualization and graphing. PLoS One. 2023;18(11):e0294236. doi: 10.1371/journal.pone.0294236.  [DOI](https://doi.org/10.1371/journal.pone.0294236) | [PMC free article](/articles/PMC10635526/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/37943830/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=SRplot:%20a%20free%20online%20platform%20for%20data%20visualization%20and%20graphing&volume=18&issue=11&publication_year=2023&pages=e0294236&pmid=37943830&doi=10.1371/journal.pone.0294236&)

54. Thaw KS, Choppradit C, Kijsanayotin P, et al. Association of N-acetyltransferase 2 (NAT2) polymorphisms and elevated liver enzymes of Myanmar TB patients in Thailand. Thai J Pharm Sci. 2020;44(4):200–215. doi: 10.56808/3027-7922.2455.  [DOI](https://doi.org/10.56808/3027-7922.2455) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Thai%20J%20Pharm%20Sci&title=Association%20of%20N-acetyltransferase%202%20(NAT2)%20polymorphisms%20and%20elevated%20liver%20enzymes%20of%20Myanmar%20TB%20patients%20in%20Thailand&volume=44&issue=4&publication_year=2020&pages=200-215&doi=10.56808/3027-7922.2455&)

55. Le TA, Pham NHT, Dinh LD, et al. NAT2 gene polymorphisms and plasma isoniazid concentration in Vietnamese tuberculosis patients. Walailak J Sci & Tech. 2021;18(2):6517. doi: 10.48048/wjst.2021.6517.  [DOI](https://doi.org/10.48048/wjst.2021.6517) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Walailak%20J%20Sci%20&%20Tech&title=NAT2%20gene%20polymorphisms%20and%20plasma%20isoniazid%20concentration%20in%20Vietnamese%20tuberculosis%20patients&volume=18&issue=2&publication_year=2021&pages=6517&doi=10.48048/wjst.2021.6517&)

56. Hemanth Kumar AK, Ramesh K, Kannan T, et al. N-acetyltransferase gene polymorphisms & plasma isoniazid concentrations in patients with tuberculosis. Indian J Med Res. 2017;145(1):118–123. doi: 10.4103/ijmr.IJMR_2013_15.  [DOI](https://doi.org/10.4103/ijmr.IJMR_2013_15) | [PMC free article](/articles/PMC5460557/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28574024/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Indian%20J%20Med%20Res&title=N-acetyltransferase%20gene%20polymorphisms%20&%20plasma%20isoniazid%20concentrations%20in%20patients%20with%20tuberculosis&volume=145&issue=1&publication_year=2017&pages=118-123&pmid=28574024&doi=10.4103/ijmr.IJMR_2013_15&)

57. Igumnova V, Capligina V, Krams A, et al. Genotype and allele frequencies of isoniazid-metabolizing enzymes NAT2 and GSTM1 in Latvian tuberculosis patients. J Infect Chemother. 2016;22(7):472–477. doi: 10.1016/j.jiac.2016.04.003.  [DOI](https://doi.org/10.1016/j.jiac.2016.04.003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27236516/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Infect%20Chemother&title=Genotype%20and%20allele%20frequencies%20of%20isoniazid-metabolizing%20enzymes%20NAT2%20and%20GSTM1%20in%20Latvian%20tuberculosis%20patients&volume=22&issue=7&publication_year=2016&pages=472-477&pmid=27236516&doi=10.1016/j.jiac.2016.04.003&)

58. Yimer G, Ueda N, Habtewold A, et al. Pharmacogenetic & pharmacokinetic biomarker for efavirenz based ARV and rifampicin based anti-TB drug induced liver injury in TB-HIV infected patients. PLoS One. 2011;6(12):e27810. doi: 10.1371/journal.pone.0027810.  [DOI](https://doi.org/10.1371/journal.pone.0027810) | [PMC free article](/articles/PMC3232196/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22162992/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Pharmacogenetic%20&%20pharmacokinetic%20biomarker%20for%20efavirenz%20based%20ARV%20and%20rifampicin%20based%20anti-TB%20drug%20induced%20liver%20injury%20in%20TB-HIV%20infected%20patients&volume=6&issue=12&publication_year=2011&pages=e27810&pmid=22162992&doi=10.1371/journal.pone.0027810&)

59. Heinrich MM, Zembrzuski VM, Ota MM, et al. Factors associated with anti-TB drug-induced hepatotoxicity and ­genetic polymorphisms in indigenous and non-indigenous populations in Brazil. Tuberculosis. 2016;101:15–24. doi: 10.1016/j.tube.2016.07.006.  [DOI](https://doi.org/10.1016/j.tube.2016.07.006) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27865386/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Tuberculosis&title=Factors%20associated%20with%20anti-TB%20drug-induced%20hepatotoxicity%20and%20%C2%ADgenetic%20polymorphisms%20in%20indigenous%20and%20non-indigenous%20populations%20in%20Brazil&volume=101&publication_year=2016&pages=15-24&pmid=27865386&doi=10.1016/j.tube.2016.07.006&)

60. El-Jaick KB, Ribeiro-Alves M, Soares MVG, et al. Homozygotes NAT2*5B slow acetylators are highly associated with hepatotoxicity induced by anti-tuberculosis drugs. Mem Inst Oswaldo Cruz. 2022;117:e210328. doi: 10.1590/0074-02760210328.  [DOI](https://doi.org/10.1590/0074-02760210328) | [PMC free article](/articles/PMC9049236/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35588539/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mem%20Inst%20Oswaldo%20Cruz&title=Homozygotes%20NAT2*5B%20slow%20acetylators%20are%20highly%20associated%20with%20hepatotoxicity%20induced%20by%20anti-tuberculosis%20drugs&volume=117&publication_year=2022&pages=e210328&pmid=35588539&doi=10.1590/0074-02760210328&)

61. Possuelo LG, Castelan JA, de Brito TC, et al. Association of slow N-acetyltransferase 2 profile and anti-TB drug-induced hepatotoxicity in patients from Southern Brazil. Eur J Clin Pharmacol. 2008;64(7):673–681. doi: 10.1007/s00228-008-0484-8.  [DOI](https://doi.org/10.1007/s00228-008-0484-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18421452/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Clin%20Pharmacol&title=Association%20of%20slow%20N-acetyltransferase%202%20profile%20and%20anti-TB%20drug-induced%20hepatotoxicity%20in%20patients%20from%20Southern%20Brazil&volume=64&issue=7&publication_year=2008&pages=673-681&pmid=18421452&doi=10.1007/s00228-008-0484-8&)

62. Santos NP, Callegari-Jacques SM, Ribeiro Dos Santos AK, et al. N-acetyl transferase 2 and cytochrome P450 2E1 genes and isoniazid-induced hepatotoxicity in Brazilian patients. Int J Tuberc Lung Dis. 2013;17(4):499–504. doi: 10.5588/ijtld.12.0645.  [DOI](https://doi.org/10.5588/ijtld.12.0645) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23394127/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Tuberc%20Lung%20Dis&title=N-acetyl%20transferase%202%20and%20cytochrome%20P450%202E1%20genes%20and%20isoniazid-induced%20hepatotoxicity%20in%20Brazilian%20patients&volume=17&issue=4&publication_year=2013&pages=499-504&pmid=23394127&doi=10.5588/ijtld.12.0645&)

63. Krasnova NM, Alekseeva EA, Rudykh ZA, et al. Prevalence of polymorphisms in N-acetyltransferase 2 gene among patients of Yakut ethnicity newly diagnosed with pulmonary tuberculosis. Annals RAMS. 2020;75(2):154–161. doi: 10.15690/vramn1217.  [DOI](https://doi.org/10.15690/vramn1217) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annals%20RAMS&title=Prevalence%20of%20polymorphisms%20in%20N-acetyltransferase%202%20gene%20among%20patients%20of%20Yakut%20ethnicity%20newly%20diagnosed%20with%20pulmonary%20tuberculosis&volume=75&issue=2&publication_year=2020&pages=154-161&doi=10.15690/vramn1217&)

64. Levano KS, Jaramillo-Valverde L, Tarazona DD, et al. Allelic and genotypic frequencies of NAT2, CYP2E1, and AADAC genes in a cohort of Peruvian tuberculosis ­patients. Mol Genet Genomic Med. 2021;9(10):e1764. doi: 10.1002/mgg3.1764.  [DOI](https://doi.org/10.1002/mgg3.1764) | [PMC free article](/articles/PMC8580101/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34510815/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Genet%20Genomic%20Med&title=Allelic%20and%20genotypic%20frequencies%20of%20NAT2,%20CYP2E1,%20and%20AADAC%20genes%20in%20a%20cohort%20of%20Peruvian%20tuberculosis%20%C2%ADpatients&volume=9&issue=10&publication_year=2021&pages=e1764&pmid=34510815&doi=10.1002/mgg3.1764&)

65. Verhagen LM, Coenen MJ, López D, et al. Full-gene ­sequencing analysis of NAT2 and its relationship with isoniazid pharmacokinetics in Venezuelan children with tuberculosis. Pharmacogenomics. 2014;15(3):285–296. doi: 10.2217/pgs.13.230.  [DOI](https://doi.org/10.2217/pgs.13.230) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24533708/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Full-gene%20%C2%ADsequencing%20analysis%20of%20NAT2%20and%20its%20relationship%20with%20isoniazid%20pharmacokinetics%20in%20Venezuelan%20children%20with%20tuberculosis&volume=15&issue=3&publication_year=2014&pages=285-296&pmid=24533708&doi=10.2217/pgs.13.230&)

66. Herrera-Rodulfo A, Carrillo-Tripp M, Laura Yeverino-Gutierrez M, et al. NAT2 polymorphisms associated with the development of hepatotoxicity after first-line tuberculosis treatment in Mexican patients: from genotype to molecular structure characterization. Clin Chim Acta. 2021;519:153–162. doi: 10.1016/j.cca.2021.04.017.  [DOI](https://doi.org/10.1016/j.cca.2021.04.017) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33932406/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Chim%20Acta&title=NAT2%20polymorphisms%20associated%20with%20the%20development%20of%20hepatotoxicity%20after%20first-line%20tuberculosis%20treatment%20in%20Mexican%20patients:%20from%20genotype%20to%20molecular%20structure%20characterization&volume=519&publication_year=2021&pages=153-162&pmid=33932406&doi=10.1016/j.cca.2021.04.017&)

67. Lv X, Tang S, Xia Y, et al. NAT2 genetic polymorphisms and anti-tuberculosis drug-induced hepatotoxicity in Chinese community population. Ann Hepatol. 2012;11(5):700–707. doi: 10.1016/S1665-2681(19)31446-2.  [DOI](https://doi.org/10.1016/S1665-2681(19)31446-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22947533/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Hepatol&title=NAT2%20genetic%20polymorphisms%20and%20anti-tuberculosis%20drug-induced%20hepatotoxicity%20in%20Chinese%20community%20population&volume=11&issue=5&publication_year=2012&pages=700-707&pmid=22947533&doi=10.1016/S1665-2681(19)31446-2&)

68. Zhang D, Hao J, Hou R, et al. The role of NAT2 polymorphism and methylation in anti-tuberculosis drug-induced liver injury in Mongolian tuberculosis patients. J Clin Pharm Ther. 2020;45(3):561–569. doi: 10.1111/jcpt.13097.  [DOI](https://doi.org/10.1111/jcpt.13097) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32364660/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Pharm%20Ther&title=The%20role%20of%20NAT2%20polymorphism%20and%20methylation%20in%20anti-tuberculosis%20drug-induced%20liver%20injury%20in%20Mongolian%20tuberculosis%20patients&volume=45&issue=3&publication_year=2020&pages=561-569&pmid=32364660&doi=10.1111/jcpt.13097&)

69. Gupta VH, Amarapurkar DN, Singh M, et al. Association of N-acetyltransferase 2 and cytochrome P450 2E1 gene polymorphisms with antituberculosis drug-induced hepatotoxicity in Western India. J Gastroenterol Hepatol. 2013;28(8):1368–1374. doi: 10.1111/jgh.12194.  [DOI](https://doi.org/10.1111/jgh.12194) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23875638/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Gastroenterol%20Hepatol&title=Association%20of%20N-acetyltransferase%202%20and%20cytochrome%20P450%202E1%20gene%20polymorphisms%20with%20antituberculosis%20drug-induced%20hepatotoxicity%20in%20Western%20India&volume=28&issue=8&publication_year=2013&pages=1368-1374&pmid=23875638&doi=10.1111/jgh.12194&)

70. Calcagno A, Cusato J, Sekaggya-Wiltshire C, et al. The influence of pharmacogenetic variants in HIV/tuberculosis coinfected patients in Uganda in the SOUTH study. Clin Pharmacol Ther. 2019;106(2):450–457. doi: 10.1002/cpt.1403.  [DOI](https://doi.org/10.1002/cpt.1403) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30779340/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=The%20influence%20of%20pharmacogenetic%20variants%20in%20HIV/tuberculosis%20coinfected%20patients%20in%20Uganda%20in%20the%20SOUTH%20study&volume=106&issue=2&publication_year=2019&pages=450-457&pmid=30779340&doi=10.1002/cpt.1403&)

71. Chan SL, Chua APG, Aminkeng F, et al. Association and clinical utility of NAT2 in the prediction of isoniazid-induced liver injury in Singaporean patients. PLoS One. 2017;12(10):e0186200. doi: 10.1371/journal.pone.0186200.  [DOI](https://doi.org/10.1371/journal.pone.0186200) | [PMC free article](/articles/PMC5642896/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29036176/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Association%20and%20clinical%20utility%20of%20NAT2%20in%20the%20prediction%20of%20isoniazid-induced%20liver%20injury%20in%20Singaporean%20patients&volume=12&issue=10&publication_year=2017&pages=e0186200&pmid=29036176&doi=10.1371/journal.pone.0186200&)

72. Headriawan A, Pramono AA, Sukadi A, et al. NAT2 gene rs1041983 is associated with anti-tuberculosis drug ­induced hepatotoxicity among pediatric tuberculosis in Bandung, Indonesia. Appl Clin Genet. 2021;14:297–303. doi: 10.2147/TACG.S303668.  [DOI](https://doi.org/10.2147/TACG.S303668) | [PMC free article](/articles/PMC8184287/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34113149/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Appl%20Clin%20Genet&title=NAT2%20gene%20rs1041983%20is%20associated%20with%20anti-tuberculosis%20drug%20%C2%ADinduced%20hepatotoxicity%20among%20pediatric%20tuberculosis%20in%20Bandung,%20Indonesia&volume=14&publication_year=2021&pages=297-303&pmid=34113149&doi=10.2147/TACG.S303668&)

73. Yuliwulandari R, Susilowati RW, Wicaksono BD, et al. NAT2 variants are associated with drug-induced liver injury caused by anti-tuberculosis drugs in Indonesian patients with tuberculosis. J Hum Genet. 2016;61(6):533–537. doi: 10.1038/jhg.2016.10.  [DOI](https://doi.org/10.1038/jhg.2016.10) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26911349/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Hum%20Genet&title=NAT2%20variants%20are%20associated%20with%20drug-induced%20liver%20injury%20caused%20by%20anti-tuberculosis%20drugs%20in%20Indonesian%20patients%20with%20tuberculosis&volume=61&issue=6&publication_year=2016&pages=533-537&pmid=26911349&doi=10.1038/jhg.2016.10&)

74. An HR, Wu XQ, Wang ZY, et al. NAT2 and CYP2E1 polymorphisms associated with antituberculosis drug-induced hepatotoxicity in Chinese patients. Clin Exp Pharmacol Physiol. 2012;39(6):535–543. doi: 10.1111/j.1440-1681.2012.05713.x.  [DOI](https://doi.org/10.1111/j.1440-1681.2012.05713.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22506592/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Exp%20Pharmacol%20Physiol&title=NAT2%20and%20CYP2E1%20polymorphisms%20associated%20with%20antituberculosis%20drug-induced%20hepatotoxicity%20in%20Chinese%20patients&volume=39&issue=6&publication_year=2012&pages=535-543&pmid=22506592&doi=10.1111/j.1440-1681.2012.05713.x&)

75. Ben Mahmoud L, Ghozzi H, Kamoun A, et al. Polymorphism of the N-acetyltransferase 2 gene as a susceptibility risk factor for antituberculosis drug-induced hepatotoxicity in Tunisian patients with tuberculosis. Pathol Biol. 2012;60(5):324–330. doi: 10.1016/j.patbio.2011.07.001.  [DOI](https://doi.org/10.1016/j.patbio.2011.07.001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21856096/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pathol%20Biol&title=Polymorphism%20of%20the%20N-acetyltransferase%202%20gene%20as%20a%20susceptibility%20risk%20factor%20for%20antituberculosis%20drug-induced%20hepatotoxicity%20in%20Tunisian%20patients%20with%20tuberculosis&volume=60&issue=5&publication_year=2012&pages=324-330&pmid=21856096&doi=10.1016/j.patbio.2011.07.001&)

76. Suhuyanly N, Bakri S, Yusuf I, et al. N-Acetyl transferase (NAT) 2 polymorphisms and susceptibility to antituberculosis drug-induced hepatotoxicity in Indonesia. Int J Sci Basic Appl Res. 2017;33(1):313–322.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Sci%20Basic%20Appl%20Res&title=N-Acetyl%20transferase%20(NAT)%202%20polymorphisms%20and%20susceptibility%20to%20antituberculosis%20drug-induced%20hepatotoxicity%20in%20Indonesia&volume=33&issue=1&publication_year=2017&pages=313-322&)

77. Goswami C, Chattopadhyay A, Chuang EY.. Rare variants: data types and analysis strategies. Ann Transl Med. 2021;9(12):961–961. doi: 10.21037/atm-21-1635.  [DOI](https://doi.org/10.21037/atm-21-1635) | [PMC free article](/articles/PMC8267277/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34277761/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Transl%20Med&title=Rare%20variants:%20data%20types%20and%20analysis%20strategies&volume=9&issue=12&publication_year=2021&pages=961-961&pmid=34277761&doi=10.21037/atm-21-1635&)

78. Kent JW Jr. Rare variants, common markers: synthetic association and beyond. Genet Epidemiol. 2011;35(Suppl 1):S80–S84. doi: 10.1002/gepi.20655.  [DOI](https://doi.org/10.1002/gepi.20655) | [PMC free article](/articles/PMC3260649/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22128064/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genet%20Epidemiol&title=Rare%20variants,%20common%20markers:%20synthetic%20association%20and%20beyond&volume=35&issue=Suppl%201&publication_year=2011&pages=S80-S84&pmid=22128064&doi=10.1002/gepi.20655&)

79. Gorlov IP, Gorlova OY, Sunyaev SR, et al. Shifting paradigm of association studies: value of rare single-nucleotide polymorphisms. Am J Hum Genet. 2008;82(1):100–112. doi: 10.1016/j.ajhg.2007.09.006.  [DOI](https://doi.org/10.1016/j.ajhg.2007.09.006) | [PMC free article](/articles/PMC2253956/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18179889/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Shifting%20paradigm%20of%20association%20studies:%20value%20of%20rare%20single-nucleotide%20polymorphisms&volume=82&issue=1&publication_year=2008&pages=100-112&pmid=18179889&doi=10.1016/j.ajhg.2007.09.006&)

80. Singh N, Dubey S, Chinnaraj S, et al. Study of NAT2 gene polymorphisms in an Indian population: association with plasma isoniazid concentration in a cohort of tuberculosis patients. Mol Diagn Ther. 2009;13(1):49–58. doi: 10.1007/BF03256314.  [DOI](https://doi.org/10.1007/BF03256314) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19351215/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Diagn%20Ther&title=Study%20of%20NAT2%20gene%20polymorphisms%20in%20an%20Indian%20population:%20association%20with%20plasma%20isoniazid%20concentration%20in%20a%20cohort%20of%20tuberculosis%20patients&volume=13&issue=1&publication_year=2009&pages=49-58&pmid=19351215&doi=10.1007/BF03256314&)

81. Jarrar YB, Balasmeh AA, Jarrar W.. Sequence analysis of the N-acetyltransferase 2 gene (NAT2) among Jordanian volunteers. Libyan J Med. 2018;13(1):1408381. doi: 10.1080/19932820.2017.1408381.  [DOI](https://doi.org/10.1080/19932820.2017.1408381) | [PMC free article](/articles/PMC5717714/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29173142/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Libyan%20J%20Med&title=Sequence%20analysis%20of%20the%20N-acetyltransferase%202%20gene%20(NAT2)%20among%20Jordanian%20volunteers&volume=13&issue=1&publication_year=2018&pages=1408381&pmid=29173142&doi=10.1080/19932820.2017.1408381&)

82. Taja-Chayeb L, Agúndez JA, Miguez-Muñoz C, et al. Arylamine N-acetyltransferase 2 genotypes in a Mexican population. Genet Mol Res. 2012;11(2):1082–1092. doi: 10.4238/2012.April.27.7.  [DOI](https://doi.org/10.4238/2012.April.27.7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22614277/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genet%20Mol%20Res&title=Arylamine%20N-acetyltransferase%202%20genotypes%20in%20a%20Mexican%20population&volume=11&issue=2&publication_year=2012&pages=1082-1092&pmid=22614277&doi=10.4238/2012.April.27.7&)

83. Quan L, Chattopadhyay K, Nelson HH, et al. Differential association for N-acetyltransferase 2 genotype and phenotype with bladder cancer risk in Chinese population. Oncotarget. 2016;7(26):40012–40024. doi: 10.18632/oncotarget.9475.  [DOI](https://doi.org/10.18632/oncotarget.9475) | [PMC free article](/articles/PMC5129988/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27223070/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncotarget&title=Differential%20association%20for%20N-acetyltransferase%202%20genotype%20and%20phenotype%20with%20bladder%20cancer%20risk%20in%20Chinese%20population&volume=7&issue=26&publication_year=2016&pages=40012-40024&pmid=27223070&doi=10.18632/oncotarget.9475&)

84. Magalon H, Patin E, Austerlitz F, et al. Population genetic diversity of the NAT2 gene supports a role of acetylation in human adaptation to farming in Central Asia. Eur J Hum Genet. 2008;16(2):243–251. doi: 10.1038/sj.ejhg.5201963.  [DOI](https://doi.org/10.1038/sj.ejhg.5201963) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18043717/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Hum%20Genet&title=Population%20genetic%20diversity%20of%20the%20NAT2%20gene%20supports%20a%20role%20of%20acetylation%20in%20human%20adaptation%20to%20farming%20in%20Central%20Asia&volume=16&issue=2&publication_year=2008&pages=243-251&pmid=18043717&doi=10.1038/sj.ejhg.5201963&)

85. Costa GN, Magno LA, Santana CV, et al. Genetic ­interaction between NAT2, GSTM1, GSTT1, CYP2E1, and environmental factors is associated with adverse reactions to anti-tuberculosis drugs. Mol Diagn Ther. 2012;16(4):241–250. doi: 10.1007/BF03262213.  [DOI](https://doi.org/10.1007/BF03262213) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22788240/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Diagn%20Ther&title=Genetic%20%C2%ADinteraction%20between%20NAT2,%20GSTM1,%20GSTT1,%20CYP2E1,%20and%20environmental%20factors%20is%20associated%20with%20adverse%20reactions%20to%20anti-tuberculosis%20drugs&volume=16&issue=4&publication_year=2012&pages=241-250&pmid=22788240&doi=10.1007/BF03262213&)

86. Hiratsuka M, Kishikawa Y, Takekuma Y, et al. Genotyping of the N-acetyltransferase2 polymorphism in the prediction of adverse drug reactions to isoniazid in Japanese patients. Drug Metab Pharmacokinet. 2002;17(4):357–362. doi: 10.2133/dmpk.17.357.  [DOI](https://doi.org/10.2133/dmpk.17.357) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15618686/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Pharmacokinet&title=Genotyping%20of%20the%20N-acetyltransferase2%20polymorphism%20in%20the%20prediction%20of%20adverse%20drug%20reactions%20to%20isoniazid%20in%20Japanese%20patients&volume=17&issue=4&publication_year=2002&pages=357-362&pmid=15618686&doi=10.2133/dmpk.17.357&)

87. Forestiero FJ, Cecon L, Hirata MH, et al. Relationship of NAT2, CYP2E1 and GSTM1/GSTT1 polymorphisms with mild elevation of liver enzymes in Brazilian individuals under anti-tuberculosis drug therapy. Clin Chim Acta. 2013;415:215–219. doi: 10.1016/j.cca.2012.10.030.  [DOI](https://doi.org/10.1016/j.cca.2012.10.030) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23099118/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Chim%20Acta&title=Relationship%20of%20NAT2,%20CYP2E1%20and%20GSTM1/GSTT1%20polymorphisms%20with%20mild%20elevation%20of%20liver%20enzymes%20in%20Brazilian%20individuals%20under%20anti-tuberculosis%20drug%20therapy&volume=415&publication_year=2013&pages=215-219&pmid=23099118&doi=10.1016/j.cca.2012.10.030&)

88. Thomas L, Raju AP, Chaithra S, et al. Influence of N-acetyltransferase 2 polymorphisms and clinical variables on liver function profile of tuberculosis patients. Expert Rev Clin Pharmacol. 2024;17(3):263–274. doi: 10.1080/17512433.2024.2311314.  [DOI](https://doi.org/10.1080/17512433.2024.2311314) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/38287694/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Expert%20Rev%20Clin%20Pharmacol&title=Influence%20of%20N-acetyltransferase%202%20polymorphisms%20and%20clinical%20variables%20on%20liver%20function%20profile%20of%20tuberculosis%20patients&volume=17&issue=3&publication_year=2024&pages=263-274&pmid=38287694&doi=10.1080/17512433.2024.2311314&)

89. Aklillu E, Carrillo JA, Makonnen E, et al. N-acetyltransferase-2 (NAT2) phenotype is influenced by genotype-environment interaction in Ethiopians. Eur J Clin Pharmacol. 2018;74(7):903–911. doi: 10.1007/s00228-018-2448-y.  [DOI](https://doi.org/10.1007/s00228-018-2448-y) | [PMC free article](/articles/PMC5999125/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29589062/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Clin%20Pharmacol&title=N-acetyltransferase-2%20(NAT2)%20phenotype%20is%20influenced%20by%20genotype-environment%20interaction%20in%20Ethiopians&volume=74&issue=7&publication_year=2018&pages=903-911&pmid=29589062&doi=10.1007/s00228-018-2448-y&)

90. Martínez C, Agúndez JA, Olivera M, et al. Influence of genetic admixture on polymorphisms of drug-metabolizing enzymes: analyses of mutations on NAT2 and C gamma P2E1 genes in a mixed Hispanic population. Clin Pharmacol Ther. 1998;63(6):623–628. doi: 10.1016/S0009-9236(98)90085-6.  [DOI](https://doi.org/10.1016/S0009-9236(98)90085-6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9663176/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Influence%20of%20genetic%20admixture%20on%20polymorphisms%20of%20drug-metabolizing%20enzymes:%20analyses%20of%20mutations%20on%20NAT2%20and%20C%20gamma%20P2E1%20genes%20in%20a%20mixed%20Hispanic%20population&volume=63&issue=6&publication_year=1998&pages=623-628&pmid=9663176&doi=10.1016/S0009-9236(98)90085-6&)

91. Surarak T, Chumnumwat S, Nosoongnoen W, et al. Efficacy, safety, and pharmacokinetics of isoniazid ­affected by NAT2 polymorphisms in patients with tuberculosis: a systematic review. Clin Transl Sci. 2024;17(4):e13795. doi: 10.1111/cts.13795.  [DOI](https://doi.org/10.1111/cts.13795) | [PMC free article](/articles/PMC11022300/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/38629592/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Transl%20Sci&title=Efficacy,%20safety,%20and%20pharmacokinetics%20of%20isoniazid%20%C2%ADaffected%20by%20NAT2%20polymorphisms%20in%20patients%20with%20tuberculosis:%20a%20systematic%20review&volume=17&issue=4&publication_year=2024&pages=e13795&pmid=38629592&doi=10.1111/cts.13795&)
