# Association between SLCO1B1 genetic polymorphisms and bleeding risk in patients treated with edoxaban

## Metadata
**Authors:** Ji Min Han, Eun Jeong Jang, Jeong Yee, Tae-Jin Song, Dong-Hyeok Kim, Junbeom Park, Hye Sun Gwak
**Journal:** Scientific Reports
**Date:** 2023 Sep 25
**DOI:** [10.1038/s41598-023-43179-7](https://doi.org/10.1038/s41598-023-43179-7)
**PMID:** 37749323
**PMCID:** PMC10520058
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10520058/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC10520058/pdf/41598_2023_Article_43179.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC10520058/pdf/41598_2023_Article_43179.pdf)

## Abstract

Since SLCO1B1 encodes the uptake transporter OATP1B1, which can influence the pharmacokinetic and pharmacodynamic profiles of edoxaban, polymorphisms in SLCO1B1 may affect the edoxaban response. This study aimed to investigate the association between SLCO1B1 gene polymorphisms and the bleeding risk in patients receiving edoxaban. We genotyped 10 single-nucleotide polymorphisms (SNPs) from the SLCO1B1 gene in patients receiving edoxaban. We also analyzed rs3842 of ABCB1 as a confounder. The odds ratio (OR) and adjusted OR (AOR) were calculated from univariate and multivariable analysis, respectively. The area under the receiver operating characteristic curve (AUROC) was constructed for the discrimination of the model. A total of 159 patients receiving edoxaban were analyzed. Overdose and rs4149056 showed significant association with bleeding complications by around 11- and 5.5-fold, respectively. Additionally, patients with the rs4149057 variant allele (C) had a 3.9-fold increased bleeding risk compared with wild-type homozygote carriers (TT), whereas rs2306283 variant homozygote (GG) carriers had a 0.27-fold reduced bleeding risk compared with wild-type allele (A) carriers. Patients with the variant-type homozygote (CC) of ABCB1 rs3842 had a higher bleeding risk than T allele carriers (AOR = 5.3 and 5.9). The final models for multivariable analyses were acceptable based on the AUROC values (> 0.70). These findings may help predict bleeding risk in patients taking edoxaban and help personalize treatment.

Subject terms: Cardiology, Health care, Medical research, Risk factors

## Introduction

Direct oral anticoagulants (DOACs) are gradually increasing in use due to their rapid anticoagulant effect and similar efficacy to vitamin K antagonists^1^. Despite the advantage of not requiring a routine international normalized ratio test, bleeding including gastrointestinal bleeding, hemorrhage, hematochezia, hematuria, and epistaxis is a major complication of DOAC use^2^. Especially, Asian individuals with DOACs showed a higher risk of bleeding, including intracranial hemorrhage than non-Asians and may require individualized DOAC treatment^3^.

Several studies in pharmacogenomics have focused on individualizing drug treatment with DOACs, primarily investigating genes associated with drug metabolism and transport. For example, in the case of dabigatran, which is the only DOAC that is not metabolized by CYP P450, it has been reported that the *CES1* gene polymorphisms were associated with a lower dabigatran concentration due to the significant role of CES1 activity in drug metabolism^4,5^. For rivaroxaban, the impact of *ABCB1* gene polymorphism on its clinical significance remains inconclusive. While rs2032582 homozygous mutated genotype was associated with rivaroxaban-induced hemorrhage^6^, rs2032582 was not associated with rivaroxaban pharmacokinetic parameters in healthy volunteers^7^. In the case of apixaban, rs2231142 of *ABCG2* gene^8^, and rs776746 of *CYP3A5* were associated with increased plasma concentrations^9^. Conversely, there is limited research on pharmacogenomics studies focusing on edoxaban. Candidate genes involved in edoxaban metabolisms, such as *CES1, CYP3A4/5, ABCB1,* and *SLCO1B1*, have been considered, but only the relationship of rs1045642 of *ABCB1* gene and rs4149056 and rs2306283 of *SLCO1B1* gene with the pharmacokinetic parameters of edoxaban was studied^10,11^.

*SLCO1B1* encodes the uptake transporter OATP1B1, which can influence the pharmacokinetic and pharmacodynamic profiles of its substrates^12–14^. Several studies have investigated that various SNPs in *SLCO1B1* affect the pharmacokinetic profiles of medications in various diseases. Statins are among the most studied drugs for their association with *SLCO1B1* polymorphism. In a simvastatin PK study conducted on healthy volunteers, participants with the *SLCO1B1* rs4149056 CC genotype had a 221% higher AUC_0–∞_ of simvastatin acid than those with rs4149056 TT genotype^15^. In a lovastatin study, *SLCO1B1**5/15 or *15/*15 genotype group showed about threefold increased AUC_0-24_ compared to the *SLCO1B1**1A/*1A genotype group^16^. In addition, it was known that the *SLCO1B1* c.463CA genotype was associated with about 40% lower rifampin concentration in Tuberculosis patients^17^. The relationship between the increased AUC of lopinavir and rs4149056 in HIV-infected children was also reported^18^.

Currently, the impact of *SLCO1B1* polymorphism on OATP1B1 substrates is mainly studied in relation to edoxaban, specifically focused solely on the variants rs4149056 and rs2306283^10,11^. Moreover, these studies have primarily involved pharmacokinetic analyses, with no exploration of the pharmacodynamic effects of *SLCO1B1* polymorphism on edoxaban.

There are many ways to select SNPs in genetic studies. Linkage disequilibrium (LD) plays a critical role in genome-wide association studies for identifying genetic variation^19^. Genotyping of tag SNPs representing LD blocks is known to be sufficient to capture most haplotype structures of the human genome^20,21^. Therefore, the SNP selection process using tag SNPs can contribute to finding important SNPs more accurately. Meanwhile, *ABCB1* rs3842, a 3’-UTR SNP, can affect protein elimination by destroying or creating miRNA binding sites^22^. In addition, *ABCB1* rs3842 was found to affect the incidence of bleeding events in patients with DOACs in our previous study^23^. Adjusting it as a confounder is necessary to reveal more accurate effects of *SL*C*O1B1* SNPs.

Therefore, the objective of this study was to investigate the association between various *SLCO1B1* gene polymorphisms and the risk of bleeding complications in patients on edoxaban.

## Materials and methods

### Study patients and data collection

This study was a retrospective analysis of prospectively collected samples from June 2018 to December 2021. We recruited patients who had been using edoxaban and retrospectively collected the patients’ previous records. We collected samples with patient consent on the day of the patient’s first outpatient visit after the start of the study. It was conducted at Ewha Womans University Mokdong Hospital and Ewha Womans University Seoul Hospital.

The study subjects were individuals aged ≥ 20 years old who received edoxaban. Patients were excluded if they met the following criteria: (1) had thromboembolic or infarction-related events during the follow-up period, (2) experienced bleeding that was minor or unverified by health professionals while on treatment, (3) experienced bleeding after one year of edoxaban therapy, or (4) treated with edoxaban for less than three months (in the control group). We followed up for bleeding for one year after edoxaban initiation. The primary endpoint was any one-year major bleeding event and clinically relevant non-major bleeding (CRNMB) according to the International Society on Thrombosis and Haemostasis (ISTH) criteria^24,25^.

Among the study population, patients who experienced any one-year major bleeding event or CRNMB were classified as a case group and other patients were classified as a control group.

We obtained the data from electronic medical records. We collected patient demographic data, including sex, age, body mass index, creatinine clearance, prescription dose, concurrent medication, any history of myocardial infarction, stroke, transient ischemic attack, thromboembolism, or bleeding, comorbidities, smoking status, and alcohol status. The CHA_2_DS_2_-VASc stroke assessment score (congestive heart failure, hypertension, age ≥ 75 years, diabetes mellitus, stroke, vascular disease, age 65–74 years, and sex category; range 0–9) was calculated from its component variables^26^. The modified HAS-BLED (hypertension, abnormal renal or liver function, stroke, bleeding history or predisposition, elderly (age ≥ 65 years), concomitant drug and alcohol use; range 0–8) score, which is a specific risk score designed for bleeding risk assessment, were calculated from its component variables^27^.

The studies involving human participants were reviewed and approved by the Institutional Review Board of Ewha Womans University Mokdong Hospital and Ewha Womans University Seoul Hospital in accordance with the 1975 Helsinki Declaration and its later amendments (IRB numbers 2018-04-006 and 2019-05-038, respectively). Written informed consents from all participants were obtained before enrollment.

### Selection of SNPs and genotyping

We selected SNPs through the following process. First, the Haploreg program was used to identify minor allele frequencies and linkage disequilibrium (LD) data of each SNP in the Asian population^28^. The tagger function within the Haploview of v4.2 was used to assign tag SNPs. Tag SNPs of the *SLCO1B1* gene were assigned with a condition MAF ≥ 20% and an r^2^ threshold of 0.8 in Asian populations. Second, among them, nine SNPs, including one synonymous SNP (rs4149057^29,30^) and eight intronic SNPs (rs11045879^31,32^, rs12317268^33,34^, rs4149081^31,35^, rs999278^36^, rs2306283^13,37^, rs10841753^14,38^, rs2417957^39,40^, and rs4149042^41^) of *SLCO1B1* were selected based on previous studies and PharmGKB^42^, which is a pharmacogenomics Knowledge Base that offers information about how human genetic variation impacts drug response. Although rs4149056 (missense SNP) had a MAF of 0.13, it was included because it has been investigated for various drug-induced toxicity and studied with edoxaban^10,11,13,14,32,38^. Finally, a total of 10 SNPs were included in this study. We included *ABCB1* rs3842, a 3’-untranslated region (UTR) SNP, as a known confounder based on previous studies^23,43,44^.

The genomic deoxyribonucleic acid (DNA) of the patients was extracted from blood or saliva. Genomic DNA was extracted from EDTA blood samples using the QIAamp DNA Blood Mini Kit (QIAGEN GmbH, Hilden, Germany). Using OraGene-600 kits (DNA Genotek, Ottawa, Canada), saliva samples were collected and subjected to genomic DNA extraction with PrepIT reagents (DNA Genotek, Ottawa, Canada). The genotypes of 10 SNPs in *SLCO1B1* and *ABCB1* rs3842 were analyzed by TaqMan assay.

The TaqMan^®^ allele discrimination technique was used to perform RT-PCR on ABI 7300 instrument (Applied Biosystems, Carlsbad, CA, USA). The PCR was performed in a 25 μL optical 8-cap strip containing 11.25 μL of DNA samples and 13.75 μL of PCR mix. The PCR reagent mixture included 12.5 μL of the TaqMan Genotyping Master Mix and 1.25 μL of the 20X TaqMan SNP Genotyping Assay Mix (Applied Biosystems in Foster City, California, USA). The catalog numbers of used assays were 4,351,379 and 4,362,691. Ten minutes after denaturing at 95 °C, the PCR was run for 15 s at 92 °C for 40 cycles and 60 s at 60 °C for 40 cycles.

### Sample size

According to the RE-LY study^45^, the ROCKET-AF study^46^, and the ARISTOLE study^47^, the incidence of bleeding complications in patients taking DOAC was 14.6–18.1 per 100 person-years. Therefore, the bleeding incidence was assumed to be approximately 15.0%. It was assumed that the case group: control group = 1:5.7, a significant odds ratio of 3.5, a minor allele frequency of 0.2, and a dominant genetic model. When estimated using Quanto v.2.4 (sample size calculating program for genotyping studies) with a significance level of 5% and a power of 80%, the number of case groups required was 24 patients (number of control groups: 24 × 5.7 = 137), approximately 160 patients required. Therefore, the final target number was set at about 160 patients.

### Statistical analysis

We compared continuous variables between patients who experienced bleeding and those who did not, using unpaired t-tests. The Kolmogorov–Smirnov method was used to test for the normality of the continuous variables. We analyzed categorical variables with the chi-squared test and Fisher’s exact test. For genetic association analysis, we included both dominant and recessive models, and we selected the most appropriate model based on effect size and statistical significance. We identified independent risk factors for bleeding after adjusting for variables with P < 0.1 in the univariate analysis in addition to age, sex, and *ABCB1* rs3842 via a multivariable logistic regression model. The unadjusted odds ratio (OR) and adjusted OR (AOR) with the 95% confidence interval (CI) were calculated from univariate and multivariable analyses, respectively. For the selection of the best model, the method of backward hierarchical elimination was used. To test the fit of the prediction model, we performed the Hosmer–Lemeshow goodness-of-fit test. We further evaluated the model discrimination by calculating the area under the receiver operating characteristic curve (AUROC). The predictive power of the logistic regression model was calculated.

Time to a bleeding event was analyzed using the Kaplan–Meier survival curves and the log-rank test. The Cox proportional-hazards model was sued for the multivariable analysis. Factors having P < 0.1 from the univariate analysis along with strong confounders of sex and age were included in the multivariable analysis. Hazard ratio (HR) and adjusted HR were calculated from the univariate and multivariable analyses, respectively.

All analyses were based on two-tail statistics and were performed using the Statistical Package for Social Sciences version 20.0 (IBM Corp., Armonk, NY, USA). P < 0.05 was considered statistically significant.

## Results

We selected a total of 212 patients for the study, excluding 12 patients treated with edoxaban for < three months in the control group, 13 patients who experienced thromboembolic or infarction-related events during the follow-up period, 11 patients who reported minor bleeding during edoxaban treatment, 15 patients who had any bleeding at least one year after edoxaban therapy, one patient with a sample insufficient for DNA analysis, and one patient who withdrew informed consent (Fig. [1](#Fig1)). Finally, 159 patients were included in the analysis. A total of 18 patients (11.3%) experienced bleeding complications, 7 of which were major and 11 of which were CRNMB events. The time (mean ± standard deviation) to a bleeding event was 111.72 ± 118.44 days. The indication for edoxaban treatment in the study patients was atrial fibrillation or secondary stroke prevention. Of the 18 patients, half the patients visited the emergency room or were hospitalized for bleeding. Two of them received red blood cell transfusions. All hospitalized patients recovered and were discharged.

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/732c/10520058/f593032bc8ad/41598_2023_43179_Fig1_HTML.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=10520058_41598_2023_43179_Fig1_HTML.jpg)

Patient flowchart.

Table [1](#Tab1) shows the demographic and clinical characteristics of the study population taking edoxaban. The mean age of the included patients was 71 years, and 92 patients (57.9%) were male. Approximately one-third of the patients received an underdose of edoxaban. The most common co-medications were statins, followed by beta-blockers. Approximately 97% of the patients had atrial fibrillation, and approximately 64% had hypertension. There was no significant factor for the incidence of bleeding complications.

### Table 1.

| Characteristic | No. (%) (n = 159) | Bleeding complication, No. (%) or mean ± SD |  |  |
| --- | --- | --- | --- | --- |
| Bleeding | No bleeding | P |  |  |
| (n = 18) | (n = 141) |  |  |  |
| Sex |  |  |  |  |
| Female | 67 (42.1) | 5 (27.8) | 62 (44.0) | 0.190 |
| Male | 92 (57.9) | 13 (72.2) | 79 (56.0) |  |
| Age (years) |   | 71.00 ± 9.89 | 71.05 ± 10.55 | 0.990 |
| < 65 | 39 (24.5) | 3 (16.7) | 36 (25.5) | 0.570 |
| ≥ 65 | 120 (75.5) | 15 (83.3) | 105 (74.5) |   |
| BMI (kg/m2) |   | 25.13 ± 3.42 | 24.72 ± 3.56 | 0.670 |
| < 25 | 82 (56.6) | 10 (62.5) | 72 (55.8) | 0.610 |
| ≥ 25 | 63 (43.4) | 6 (27.5) | 57 (44.2) |   |
| Creatinine clearance (mL/min) |   | 65.19 ± 16.53 | 70.21 ± 25.83 | 0.450 |
| < 30 | 5 (3.4) | 0 (0.0) | 5 (3.8) | 1.000 |
| ≥ 30 | 143 (96.6) | 16 (100.0) | 127 (96.2) |   |
| Prescription dosea |  |  |  |  |
| Underdose | 49 (30.8) | 6 (33.3) | 43 (30.5) | 0.080 |
| Standard dose | 106 (66.7) | 10 (55.6) | 96 (68.1) |  |
| Overdose | 4 (2.5) | 2 (11.1) | 2 (1.4) |  |
| Co-medications |  |  |  |  |
| Antiplatelets | 22 (13.8) | 1 (5.6) | 21 (14.9) | 0.470 |
| ACEI or ARBs | 80 (50.3) | 6 (22.2) | 74 (52.5) | 0.130 |
| Beta-blockers | 97 (61.0) | 13 (72.2) | 84 (59.6) | 0.300 |
| Calcium channel blockers | 49 (30.8) | 3 (16.7) | 46 (32.6) | 0.170 |
| Diuretics | 36 (22.6) | 4 (22.2) | 32 (22.7) | 1.000 |
| Statins | 100 (62.9) | 9 (50.0) | 91 (64.5) | 0.230 |
| CYP inducers | 0 (0) | 0 (0.0) | 0 (0.0) | NA |
| CYP inhibitors | 14 (8.9) | 2 (11.1) | 12 (8.6) | 0.660 |
| Previous myocardial infarction | 14 (8.8) | 1 (5.6) | 13 (9.2) | 0.605 |
| Previous stroke/TIA/thromboembolism | 83 (52.2) | 10 (55.6) | 76 (51.8) | 0.762 |
| Previous bleeding events | 7 (4.4) | 0 (0) | 7 (5.0) | 0.334 |
| Comorbidities |  |  |  |  |
| Atrial fibrillation | 147 (97.4) | 17 (94.4) | 130 (97.7) | 0.400 |
| Hypertension | 102 (64.2) | 9 (50.0) | 93 (66.0) | 0.180 |
| Diabetes mellitus | 48 (30.2) | 3 (16.7) | 45 (31.9) | 0.180 |
| Heart failure | 20 (12.6) | 2 (11.1) | 18 (12.8) | 1.000 |
| Anemia | 49 (30.8) | 8 (44.4) | 41 (29.1) | 0.180 |
| Smoking | 20 (12.6) | 4 (22.2) | 16 (11.3) | 0.250 |
| Alcohol | 52 (37.4) | 7 (43.8) | 45 (36.6) | 0.580 |
| CHA2DS2-VASc risk of stroke |   | 3.44 ± 1.72 | 3.81 ± 1.54 | 0.351 |
| Modified HAS-BLED |   | 2.00 ± 0.84 | 2.06 ± 1.00 | 0.796 |

Table 1 Caption: Baseline characteristics of patients who administered edoxaban.

In the genotype analysis, rs4149057, rs999278, rs2306283, and rs4149056 of *SLCO1B1* and rs3842 of *ABCB1* were significantly associated with bleeding risk (Table [2](#Tab2)).

### Table 2.

| dbSNP rsID | Grouped genotype | Minor allele frequency | Amino acid change | Edoxaban |  |  |
| --- | --- | --- | --- | --- | --- | --- |
| Bleeding | No bleeding | P |  |  |  |  |
| (n = 18) | (n = 141) |  |  |  |  |  |
| rs11045879 (T > C) | TT, CT | 0.4 | – | 17 (94.4) | 115 (81.6) | 0.170 |
| CC | 1 (5.6) | 26 (18.4) |  |  |  |  |
| rs12317268 (A > G) | AA, AG | 0.38 | – | 17 (94.4) | 116 (82.3) | 0.188 |
| GG | 1 (5.6) | 25 (17.7) |  |  |  |  |
| rs4149057 (T > C) | TT | 0.27 | Leu191 = | 5 (27.8) | 77 (54.6) | 0.032 |
| CT, CC | 13 (72.2) | 64 (45.4) |  |  |  |  |
| rs4149081 (G > A) | GG, AG | 0.39 | – | 16 (94.1) | 115 (81.6) | 0.194 |
| AA | 1 (5.9) | 26 (18.4) |  |  |  |  |
| rs999278 (C > A) | CC | 0.27 | – | 5 (27.8) | 76 (53.9) | 0.037 |
| AC, AA | 13 (72.2) | 65 (46.1) |  |  |  |  |
| rs2306283 (A > G) | AA, AG | 0.28 | Asn130Asp | 13 (72.2) | 67 (47.5) | 0.048 |
| GG | 5 (27.8) | 74 (52.5) |  |  |  |  |
| rs10841753 (T > C) | TT | 0.31 | – | 12 (66.7) | 68 (48.2) | 0.141 |
| CT, CC | 6 (33.3) | 73 (51.8) |  |  |  |  |
| rs2417957 (C > T) | CC | 0.31 | – | 13 (72.2) | 68 (48.6) | 0.059 |
| CT, TT | 5 (27.8) | 72 (51.4) |  |  |  |  |
| rs4149042 (T > C) | TT, CT | 0.42 | – | 17 (94.4) | 110 (78.0) | 0.102 |
| CC | 1 (5.6) | 31 (22.0) |  |  |  |  |
| rs4149056 (T > C) | TT | 0.14 | Val174Ala | 8 (44.4) | 109 (77.3) | 0.003 |
| CT, CC | 10 (55.6) | 32 (22.7) |  |  |  |  |
| SLCO1B1*1A carrier | Yes |   |   | 13 (72.2) | 64 (46.4) | 0.039 |
| No | 5 (27.8) | 74 (53.6) |  |  |  |  |
| SLCO1B1*1B carrier | Yes |   |   | 16 (88.9) | 128 (92.8) | 0.563 |
| No | 2 (11.1) | 10 (7.2) |  |  |  |  |
| SLCO1B1*15 carrier | Yes |   |   | 10 (55.6) | 31 (22.5) | 0.003 |
| No | 8 (44.4) | 107 (77.5) |  |  |  |  |
| ABCB1 rs3842 (T > C) | TT, TC | 0.32 | – | 12 (66.7) | 127 (90.7) | 0.003 |
| CC | 6 (33.3) | 13 (9.3) |  |  |  |  |

Table 2 Caption: Effects of gene polymorphisms on bleeding complications in patients who administered edoxaban.

We performed a multivariable logistic regression analysis using variables with P < 0.1, age, and sex. We constructed two models for the multivariable analyses of the edoxaban subgroup, as one pair of SNPs (rs4149057 and rs999278) was in LD (r^2^ = 0.97) in study population (Table [3](#Tab3)). Model I included age, sex, overdose, rs999278, rs2306283, rs4149056, rs2417957, and rs3842. Model II included rs4149057 instead of rs999278 in Model I. Patients overdosed and variant-type allele (C) carriers of rs4149056 had an 11.0–11.4 and 5.5–5.7 fold increased bleeding risk, respectively, after adjusting for confounders. Additionally, patients with rs4149057 variant allele (C) had a 3.9-fold increased bleeding risk compared with wild-type homozygote (TT) carriers, whereas rs2306283 variant homozygote (GG) carriers had a 0.27-fold reduced bleeding risk compared with wild-type allele (A) carriers. *ABCB1* rs3842 variant homozygote carriers (CC) also showed a 5.3–5.9 fold increased bleeding risk.

### Table 3.

| Predictors |   | Unadjusted OR | Model I | Model II | Model III | Model IV | Model V & VI |
| --- | --- | --- | --- | --- | --- | --- | --- |
| (95% CIs) | Adjusted OR | Adjusted OR | Adjusted OR | Adjusted OR | Adjusted OR |  |  |
|   | (95% CI) | (95% CI) | (95% CI) | (95% CI) | (95% CI) |  |  |
| Age | ≥ 65 years | 1.71 (0.47–6.27) |   |   |   |   |   |
| Female |   | 0.49 (0.17–1.45) |   |   |   |   |   |
| Prescription dose | Overdose | 8.69 (1.14–65.96)* | 11.36 (1.23–114.36)* | 11.01 (1.08–111.97)* | 10.86 (1.07–110.35)* | 10.82 (1.07–109.91)* | 13.60 (1.59–116.64)* |
| rs4149057 (T > C) | CT, CC | 3.13(1.06–9.24)* |   | 3.92 (1.16–13.29)* |   | 3.82 (1.13–12.94)* |   |
| rs999278 (C > A) | AC, AA | 3.04 (1.03–8.98)* |   |   | 3.78 (1.11–12.80)* |   |   |
| rs2306283 (A > G) | GG | 0.35 (0.12–1.03) | 0.27 (0.08–0.90)* |   |   |   |   |
| rs4149056 (T > C) | TC, CC | 4.26 (1.55–11.69)** | 5.48 (1.75–17.23)** | 5.71(1.80–18.13)** |   |   |   |
| rs2417957 (C > T) | CT, TT | 0.36 (0.12–1.07) |   |   |   |   |   |
| SLCO1B1*15 carrier | Yes | 4.32 (1.57–11.87)** |   |   | 5.78 (1.82–18.36)** | 5.75 (1.81–18.24)** |   |
| SLCO1B1*1A carrier | Yes | 3.006 (1.017–8.891)* |   |   |   |   | 3.00 (0.96–9.40) |
| ABCB1 rs3842 (T > C) | CC | 4.89 (1.57–15.18)** | 5.89 (1.61–21.56)** | 5.33 (1.45–19.55)* | 5.32(1.45–19.50)* | 5.28 (1.44–19.35)* | 5.44 (1.66–17.86)** |

Table 3 Caption: Univariate and multivariable regression analyses to identify predictors for bleeding complications in edoxaban users.

Haplotype analyses including *SLCO1B1**1A, *SLCO1B1**1B, and *SLCO1B1**15) were also conducted (Table [3](#Tab3)). Among them, *SLCO1B1**1A and *SLCO1B1**15 were significant for bleeding events. In the multivariable analysis, patients with *SLCO1B1**15 showed an increased risk of bleeding events by 5.7–5.8 folds, whereas *SLCO1B1**1A failed to demonstrate statistical significance. RS999278 was a new SNP that increased the risk of bleeding complications by 3.7 times.

The Hosmer–Lemeshow test for bleeding revealed a good fit for the final model (Model I: χ^2^ = 0.11, P = 0.991, Model II: χ^2^ = 2.34, P = 0.674, Model III: χ^2^ = 2.43, P = 0.657, Model IV: χ^2^ = 2.32, P = 0.677). The AUROC value (Model I: 0.79, 95% CI 0.67–0.91, Model II: 0.78, 95% CI 0.65–0.91, Model III: 0.78, 95% CI 0.64–0.91, Model IV: 0.78, 95% CI 0.65–0.91) from multivariable logistic regression indicated the acceptable performance of all models (Fig. [2](#Fig2)). The predictive power of the bleeding prediction models was 91.1% in model I and II, 91.0% in model III and IV.

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/732c/10520058/90db08be0b1a/41598_2023_43179_Fig2_HTML.jpg)

The area under the receiver operating characteristics (AUROC) curves for bleeding complications for edoxaban users.

After Bonferroni correction of multiple comparisons for ten SNPs analyses, *SLCO1B1* rs4149056 and *ABCB1* rs3842 were the sole SNPs that reached statistical significance in the Table [2](#Tab2). When the simplest genetic model was constructed using these two SNPs, *SLCO1B1* rs4149056 and *ABCB1* rs3842 were associated with 4.7- and 5.7-fold increased bleeding risk, respectively. This genetic model still held a high predictive value (89.2%), similar to the predictive value (91.1%) of the prediction model shown initially in our study.

Since overdose can drive associations, further analyses were performed excluding overdose cases. Regardless of the model, two *SLCO1B1* SNPs (rs2306283, rs4149056) and one *ABCB1* SNP (rs3842) still showed statistical significance. Patients with the rs4149056 variant allele (C) had a 4.5-fold increased bleeding risk compared with wild-type homozygote carriers (TT), whereas rs2306283 variant homozygote (GG) carriers had a 0.25-fold reduced bleeding risk compared with wild-type allele (A) carriers. Patients with the variant-type homozygote (CC) of ABCB1 rs3842 had a higher bleeding risk than T allele carriers (AOR = 5.7).

Time to a bleeding event was analyzed. Among the clinical characteristics, there were no factors affecting the time to bleeding events (Supplementary Table [1](#MOESM1)). For genetic factors, *SLCO1B1* rs4149056, rs4149057, rs999278 and *ABCB1* rs3842 were significantly related with time to bleeding events in both univariate and multivariable analyses (Supplementary Tables [2](#MOESM1) and [3](#MOESM1)).

Two models were constructed for the multivariable analyses, as one pair of SNPs (rs4149057 and rs999278) was in LD (r^2^ = 0.97) in the study population (Supplementary Table [3](#MOESM1)). Rs4149056 and rs4149057 were associated with an increased hazard of time to bleeding by around 3.9 times and 3.0 times, respectively. Patients with the rs999278 variant allele (A) had a 2.9-fold increased hazard of time to bleeding compared with wild-type homozygote carriers (CC). Patients with the variant-type homozygote (CC) of *ABCB1* rs3842 had a 3.7–3.8- times the hazard of time to bleeding compared to T allele carriers.

## Discussion

This study revealed the clinical and genetic risk factors associated with bleeding complications during edoxaban therapy. Edoxaban overdose, *SLCO1B1* rs4149057, rs2306283, and rs4149056, and *ABCB1* rs3842 were significant factors for bleeding risk.

OATP1B1, a drug transporter expressed in the liver, plays an important role in transporting drugs and endogenous substrates from the blood into the hepatocytes^48^. The *SLCO1B1* gene polymorphism, which encodes OATP1B1, may affect transporter activity. Previous studies on the association between *SLCO1B1* and DOACs have been mainly limited to edoxaban^10,11^.

Among the *SLCO1B1* SNP rs4149057, rs2306283, and rs4149056 that were significant in our edoxaban study, rs4149056 is the only SNP whose association with edoxaban has been studied. In the case of rs4149056, the relationship with edoxaban pharmacokinetics was investigated, but the relationship with edoxaban pharmacodynamics including bleeding has not yet been studied.

Rs4149056 (*SLCO1B1**5) is a functional polymorphism in exon 5 and a well-studied SNP associated with drug toxicities such as stain-induced myopathy and methotrexate toxicity^13,14^. Rs4149056 was typically associated with reduced transporter activity, resulting in increased systemic drug exposure and increased toxicity risk^13,38^. In our study, rs4149056 significantly increased the bleeding risk. A pharmacogenomic analysis that combined 14 phase 1 studies investigated the relationship between *SLCO1B1* gene polymorphism (rs4149056) and edoxaban pharmacokinetics^10^. Although the effect of rs4149056 on overall edoxaban exposure was insignificant, carriers with variant allele (C) showed an increased exposure to M4, the most-abundant metabolite of edoxaban. M4 was considered an insignificant metabolite for the overall anticoagulant effect because it accounted for less than 10% of the total anticoagulant exposure^49^. In another edoxaban study on *SLCO1B1**15 haplotype, the correlation between prothrombin time and M4 concentration was significant in non-valvular atrial fibrillation patients with repeated administration, and the clinical contribution of M4 was confirmed^11^.

Unlike rs4149056, studies on effects of rs4149057 and rs2306283 in patients taking edoxaban are limited. In the case of rs2306283, the relationship between rs2306283 and statin-induced myopathy has been primarily investigated. However, the role of rs2306283 polymorphism in transport remains controversial. The variant allele (G) of rs2306283 was not associated with statin-induced myopathy^50^, but in another study, *SLCO1B1**15 carriers (rs4149056 allele C and rs2306283 allele G) showed higher rosuvastatin plasma concentration than *SLCO1B1* haplotype (rs4149056 allele T and rs2306283 allele A), which may increase the risk of statin-induced myopathy^51^. Although rs4149057 is a synonymous SNP located in exon 5 of *SLCO1B1*, it is associated with the clearance change of mitotane and irinotecan, substrates of OATP1B1^29,30^. This study suggests that they play a critical role in edoxaban therapy. Future studies should elucidate the mechanisms of these SNPs.

The *ABCB1* gene, which encodes the P-gp efflux pump, has been one of the frequently studied genes^52^. In our previous pharmacogenomics study, we found *ABCB1* rs3842 was a significant factor associated with increased bleeding risk in patients with DOACs^23^. *ABCB1* rs3842 is a 3’-UTP SNP that might influence protein expression by disrupting or creating microRNA binding sites^22^. Previous research also have shown that carriers of the *ABCB1* rs3842 variant allele (C) had high bioavailability or lower disease activity scores^43,44^. Accordingly, we analyzed including *ABCB1* rs3842 as a confounder, and rs3842 was also found to be a factor affecting bleeding complications in this study.

In this study, we observed an elevated bleeding risk among patients who experienced an overdose of edoxaban. To determine edoxaban overdose, we considered dose adjustment criteria based on the patient's renal function. DOAC overdose increases the risk of bleeding events^53^. In a systematic review regarding off-label dosages of DOACs, the incidence of adverse reactions was higher in patients administered an overdose than in those administered the recommended dose^54^. DOAC overdosing including edoxaban was associated with not only bleeding events but also stroke/systematic embolism, all-hospitalization, and all-cause mortality.

Several limitations were observed in this study, including relatively small sample size and exclusive focus on Asian individuals residing in Korea, which could potentially restrict the generalizability of the findings. Although we collected samples prospectively, there is a risk of bias associated with a retrospective design because patient data were collected retrospectively. Furthermore, the follow-up period was limited to one year to eliminate the likelihood of non-drug-related bleeding. Since this study was conducted on edoxaban, a substrate of *SLCO1B1*, the results of this study cannot be applied to other drugs or healthy populations. However, to our knowledge, this is the first study to investigate the association between *SLCO1B1* polymorphisms and edoxaban-related bleeding complications. The results must be validated in different populations to generalize and apply them in clinical practice.

## Supplementary Information

## Author contributions

All the authors have made substantial contributions to the conception of the study. J.M.H., E.J.J., J.P. and H.S.G. contributed to designing the study. J.M.H., J.Y., and T.S. contributed to material preparation and data collection. E.J.J., J.P., H.S.G. and D.K. performed data analysis and interpretation. J.M.H. and E.J.J. contributed to drafting of the manuscript. J.P. and H.S.G. contributed to critical revision of the manuscript. All authors approved the final manuscript.

## Funding

This work was supported by the National Research Foundation of Korea (NRF) grant funded by the Korea government (MSIT) (grant number: NRF-2020R1A2C1008120).

## Data availability

The datasets used and/or analysed during the current study available from the corresponding author on reasonable request.

## Competing interests

The authors declare no competing interests.

## Footnotes

## Contributor Information

Junbeom Park, Email: parkjb@ewha.ac.kr.

Hye Sun Gwak, Email: hsgwak@ewha.ac.kr.

## Supplementary Information

The online version contains supplementary material available at 10.1038/s41598-023-43179-7.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

The datasets used and/or analysed during the current study available from the corresponding author on reasonable request.

### Supplementary Materials

### Data Availability Statement

The datasets used and/or analysed during the current study available from the corresponding author on reasonable request.

## References

1. Schwarb H, Tsakiris DA. New direct oral anticoagulants (DOAC) and their use today. Dent. J. (Basel) 2016;4(1):5. doi: 10.3390/dj4010005.  [DOI](https://doi.org/10.3390/dj4010005) | [PMC free article](/articles/PMC5851208/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29563447/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Dent.%20J.%20(Basel)&title=New%20direct%20oral%20anticoagulants%20(DOAC)%20and%20their%20use%20today&author=H%20Schwarb&author=DA%20Tsakiris&volume=4&issue=1&publication_year=2016&pages=5&pmid=29563447&doi=10.3390/dj4010005&)

2. Seiffge DJ, Traenka C, Polymeris A, et al. Early start of DOAC after ischemic stroke: Risk of intracranial hemorrhage and recurrent events. Neurology. 2016;87(18):1856–1862. doi: 10.1212/WNL.0000000000003283.  [DOI](https://doi.org/10.1212/WNL.0000000000003283) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27694266/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neurology&title=Early%20start%20of%20DOAC%20after%20ischemic%20stroke:%20Risk%20of%20intracranial%20hemorrhage%20and%20recurrent%20events&author=DJ%20Seiffge&author=C%20Traenka&author=A%20Polymeris&volume=87&issue=18&publication_year=2016&pages=1856-1862&pmid=27694266&doi=10.1212/WNL.0000000000003283&)

3. Hori M, Connolly SJ, Zhu J, et al. Dabigatran versus warfarin: Effects on ischemic and hemorrhagic strokes and bleeding in Asian and non-Asians with atrial fibrillation. Stroke. 2013;44:1891–1896. doi: 10.1161/STROKEAHA.113.000990.  [DOI](https://doi.org/10.1161/STROKEAHA.113.000990) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23743976/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Stroke&title=Dabigatran%20versus%20warfarin:%20Effects%20on%20ischemic%20and%20hemorrhagic%20strokes%20and%20bleeding%20in%20Asian%20and%20non-Asians%20with%20atrial%20fibrillation&author=M%20Hori&author=SJ%20Connolly&author=J%20Zhu&volume=44&publication_year=2013&pages=1891-1896&pmid=23743976&doi=10.1161/STROKEAHA.113.000990&)

4. Paré G, Eriksson N, Lehr T, et al. Genetic determinants of dabigatran plasma levels and their relation to bleeding. Circulation. 2013;127(13):1404–1412. doi: 10.1161/CIRCULATIONAHA.112.001233.  [DOI](https://doi.org/10.1161/CIRCULATIONAHA.112.001233) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23467860/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation&title=Genetic%20determinants%20of%20dabigatran%20plasma%20levels%20and%20their%20relation%20to%20bleeding&author=G%20Par%C3%A9&author=N%20Eriksson&author=T%20Lehr&volume=127&issue=13&publication_year=2013&pages=1404-1412&pmid=23467860&doi=10.1161/CIRCULATIONAHA.112.001233&)

5. Dimatteo C, D’Andrea G, Vecchione G, et al. Pharmacogenetics of dabigatran etexilate interindividual variability. Thromb. Res. 2016;144:1–5. doi: 10.1016/j.thromres.2016.05.025.  [DOI](https://doi.org/10.1016/j.thromres.2016.05.025) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27261537/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Thromb.%20Res.&title=Pharmacogenetics%20of%20dabigatran%20etexilate%20interindividual%20variability&author=C%20Dimatteo&author=G%20D%E2%80%99Andrea&author=G%20Vecchione&volume=144&publication_year=2016&pages=1-5&pmid=27261537&doi=10.1016/j.thromres.2016.05.025&)

6. Lorenzini K, Daali Y, Fontana P, et al. Rivaroxaban-induced hemorrhage associated with ABCB1 genetic defect. Front. Pharmacol. 2016;7:494. doi: 10.3389/fphar.2016.00494.  [DOI](https://doi.org/10.3389/fphar.2016.00494) | [PMC free article](/articles/PMC5165251/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28066243/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front.%20Pharmacol.&title=Rivaroxaban-induced%20hemorrhage%20associated%20with%20ABCB1%20genetic%20defect&author=K%20Lorenzini&author=Y%20Daali&author=P%20Fontana&volume=7&publication_year=2016&pages=494&pmid=28066243&doi=10.3389/fphar.2016.00494&)

7. Gouin-Thibault I, Delavenne X, Blanchard A, et al. Interindividual variability in dabigatran and rivaroxaban exposure: Contribution of ABCB1 genetic polymorphisms and interaction with clarithromycin. J. Thromb. Haemost. 2017;15(2):273–283. doi: 10.1111/jth.13577.  [DOI](https://doi.org/10.1111/jth.13577) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27893182/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Thromb.%20Haemost.&title=Interindividual%20variability%20in%20dabigatran%20and%20rivaroxaban%20exposure:%20Contribution%20of%20ABCB1%20genetic%20polymorphisms%20and%20interaction%20with%20clarithromycin&author=I%20Gouin-Thibault&author=X%20Delavenne&author=A%20Blanchard&volume=15&issue=2&publication_year=2017&pages=273-283&pmid=27893182&doi=10.1111/jth.13577&)

8. Gulilat M, Keller D, Linton B, et al. Drug interactions and pharmacogenetic factors contribute to variation in apixaban concentration in atrial fibrillation patients in routine care. J. Thromb. Thrombolysis. 2020;49(2):294–303. doi: 10.1007/s11239-019-01962-2.  [DOI](https://doi.org/10.1007/s11239-019-01962-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31564018/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Thromb.%20Thrombolysis&title=Drug%20interactions%20and%20pharmacogenetic%20factors%20contribute%20to%20variation%20in%20apixaban%20concentration%20in%20atrial%20fibrillation%20patients%20in%20routine%20care&author=M%20Gulilat&author=D%20Keller&author=B%20Linton&volume=49&issue=2&publication_year=2020&pages=294-303&pmid=31564018&doi=10.1007/s11239-019-01962-2&)

9. Ueshima S, Hira D, Fujii R, et al. Impact of ABCB1, ABCG2, and CYP3A5 polymorphisms on plasma trough concentrations of apixaban in Japanese patients with atrial fibrillation. Pharm. Genom. 2017;27:329–336. doi: 10.1097/FPC.0000000000000294.  [DOI](https://doi.org/10.1097/FPC.0000000000000294) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28678049/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharm.%20Genom.&title=Impact%20of%20ABCB1,%20ABCG2,%20and%20CYP3A5%20polymorphisms%20on%20plasma%20trough%20concentrations%20of%20apixaban%20in%20Japanese%20patients%20with%20atrial%20fibrillation&author=S%20Ueshima&author=D%20Hira&author=R%20Fujii&volume=27&publication_year=2017&pages=329-336&pmid=28678049&doi=10.1097/FPC.0000000000000294&)

10. Vandell AG, Lee J, Shi M, et al. An integrated pharmacokinetic/pharmacogenomic analysis of ABCB1 and SLCO1B1 polymorphisms on edoxaban exposure. Pharmacogenom. J. 2018;18(1):153–159. doi: 10.1038/tpj.2016.82.  [DOI](https://doi.org/10.1038/tpj.2016.82) | [PMC free article](/articles/PMC5817390/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27897269/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenom.%20J.&title=An%20integrated%20pharmacokinetic/pharmacogenomic%20analysis%20of%20ABCB1%20and%20SLCO1B1%20polymorphisms%20on%20edoxaban%20exposure&author=AG%20Vandell&author=J%20Lee&author=M%20Shi&volume=18&issue=1&publication_year=2018&pages=153-159&pmid=27897269&doi=10.1038/tpj.2016.82&)

11. Nakagawa J, Kinjo T, Aiuchi N, et al. Associations among plasma concentrations of edoxaban and M-4, prothrombin time, and the SLCO1B1*15 haplotype in patients with non-valvular atrial fibrillation. Ther. Drug Monit. 2022;45(3):409–416. doi: 10.1097/FTD.0000000000001042.  [DOI](https://doi.org/10.1097/FTD.0000000000001042) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36150716/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ther.%20Drug%20Monit.&title=Associations%20among%20plasma%20concentrations%20of%20edoxaban%20and%20M-4,%20prothrombin%20time,%20and%20the%20SLCO1B1*15%20haplotype%20in%20patients%20with%20non-valvular%20atrial%20fibrillation&author=J%20Nakagawa&author=T%20Kinjo&author=N%20Aiuchi&volume=45&issue=3&publication_year=2022&pages=409-416&pmid=36150716&doi=10.1097/FTD.0000000000001042&)

12. Nishizoto Y, Ieiri I, Suzuki H, et al. Polymorphisms of OATP-C (SLC21A6) and OAT3 (SLC22A8) genes: Consequences for pravastatin pharmacokinetics. Clin. Pharmacol. Ther. 2003;73:554–565. doi: 10.1016/S0009-9236(03)00060-2.  [DOI](https://doi.org/10.1016/S0009-9236(03)00060-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12811365/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Polymorphisms%20of%20OATP-C%20(SLC21A6)%20and%20OAT3%20(SLC22A8)%20genes:%20Consequences%20for%20pravastatin%20pharmacokinetics&author=Y%20Nishizoto&author=I%20Ieiri&author=H%20Suzuki&volume=73&publication_year=2003&pages=554-565&pmid=12811365&doi=10.1016/S0009-9236(03)00060-2&)

13. Shahrure ZM, Irshaid YM, Mustafa KN, et al. SLCO1B1 gene polymorphisms (rs2306283 and rs4149056) and statin-induced myopathy in Jordanian diabetics. Curr. Rev. Clin. Exp. Pharmacol. 2021;16(3):281–288. doi: 10.2174/1574884715666200827105612.  [DOI](https://doi.org/10.2174/1574884715666200827105612) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32860365/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr.%20Rev.%20Clin.%20Exp.%20Pharmacol.&title=SLCO1B1%20gene%20polymorphisms%20(rs2306283%20and%20rs4149056)%20and%20statin-induced%20myopathy%20in%20Jordanian%20diabetics&author=ZM%20Shahrure&author=YM%20Irshaid&author=KN%20Mustafa&volume=16&issue=3&publication_year=2021&pages=281-288&pmid=32860365&doi=10.2174/1574884715666200827105612&)

14. Liu SG, Gao C, Zhang RD, et al. Polymorphisms in methotrexate transporters and their relationship to plasma methotrexate levels, toxicity of high-dose methotrexate, and outcome of pediatric acute lymphoblastic leukemia. Oncotarget. 2017;8(23):37761–37772. doi: 10.18632/oncotarget.17781.  [DOI](https://doi.org/10.18632/oncotarget.17781) | [PMC free article](/articles/PMC5514947/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28525903/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncotarget&title=Polymorphisms%20in%20methotrexate%20transporters%20and%20their%20relationship%20to%20plasma%20methotrexate%20levels,%20toxicity%20of%20high-dose%20methotrexate,%20and%20outcome%20of%20pediatric%20acute%20lymphoblastic%20leukemia&author=SG%20Liu&author=C%20Gao&author=RD%20Zhang&volume=8&issue=23&publication_year=2017&pages=37761-37772&pmid=28525903&doi=10.18632/oncotarget.17781&)

15. Pasanen MK, Neuvonen M, Neuvonen PJ, et al. SLCO1B1 polymorphism markedly affects the pharmacokinetics of simvastatin acid. Pharmacogenet. Genom. 2006;16(12):873–879. doi: 10.1097/01.fpc.0000230416.82349.90.  [DOI](https://doi.org/10.1097/01.fpc.0000230416.82349.90) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17108811/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet.%20Genom.&title=SLCO1B1%20polymorphism%20markedly%20affects%20the%20pharmacokinetics%20of%20simvastatin%20acid&author=MK%20Pasanen&author=M%20Neuvonen&author=PJ%20Neuvonen&volume=16&issue=12&publication_year=2006&pages=873-879&pmid=17108811&doi=10.1097/01.fpc.0000230416.82349.90&)

16. Tornio A, Vakkilainen J, Neuvonen M, et al. SLCO1B1 polymorphism markedly affects the pharmacokinetics of lovastatin acid. Pharmacogenet. Genom. 2015;25(8):382–387. doi: 10.1097/FPC.0000000000000148.  [DOI](https://doi.org/10.1097/FPC.0000000000000148) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26020121/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet.%20Genom.&title=SLCO1B1%20polymorphism%20markedly%20affects%20the%20pharmacokinetics%20of%20lovastatin%20acid&author=A%20Tornio&author=J%20Vakkilainen&author=M%20Neuvonen&volume=25&issue=8&publication_year=2015&pages=382-387&pmid=26020121&doi=10.1097/FPC.0000000000000148&)

17. Weiner M, Peloquin C, Burman W, et al. Effects of tuberculosis, race, and human gene SLCO1B1 polymorphisms on rifampin concentrations. Antimicrob. Agents Chemother. 2010;54(10):4192–4200. doi: 10.1128/AAC.00353-10.  [DOI](https://doi.org/10.1128/AAC.00353-10) | [PMC free article](/articles/PMC2944564/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20660695/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Antimicrob.%20Agents%20Chemother.&title=Effects%20of%20tuberculosis,%20race,%20and%20human%20gene%20SLCO1B1%20polymorphisms%20on%20rifampin%20concentrations&author=M%20Weiner&author=C%20Peloquin&author=W%20Burman&volume=54&issue=10&publication_year=2010&pages=4192-4200&pmid=20660695&doi=10.1128/AAC.00353-10&)

18. Rakhmanina N, Neely MN, Schaik R, et al. CYP3A5, ABCB1, and SLCO1B1 polymorphisms and pharmacokinetics and virologic outcome of lopinavir/ritonavir in HIV-infected children. Ther. Drug Monit. 2011;33(4):417–424. doi: 10.1097/FTD.0b013e318225384f.  [DOI](https://doi.org/10.1097/FTD.0b013e318225384f) | [PMC free article](/articles/PMC3139013/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21743379/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ther.%20Drug%20Monit.&title=CYP3A5,%20ABCB1,%20and%20SLCO1B1%20polymorphisms%20and%20pharmacokinetics%20and%20virologic%20outcome%20of%20lopinavir/ritonavir%20in%20HIV-infected%20children&author=N%20Rakhmanina&author=MN%20Neely&author=R%20Schaik&volume=33&issue=4&publication_year=2011&pages=417-424&pmid=21743379&doi=10.1097/FTD.0b013e318225384f&)

19. Weiss KM, Clark AG. Linkage disequilibrium and the mapping of complex human traits. Trends Genet. 2022;18:19–24. doi: 10.1016/s0168-9525(01)02550-1.  [DOI](https://doi.org/10.1016/s0168-9525(01)02550-1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11750696/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Trends%20Genet.&title=Linkage%20disequilibrium%20and%20the%20mapping%20of%20complex%20human%20traits&author=KM%20Weiss&author=AG%20Clark&volume=18&publication_year=2022&pages=19-24&pmid=11750696&doi=10.1016/s0168-9525(01)02550-1&)

20. Johnson GCL, Esposito L, Barratt BJ, et al. Haplotype tagging for the identification of common disease genes. Nat. Genet. 2001;29:233–237. doi: 10.1038/ng1001-233.  [DOI](https://doi.org/10.1038/ng1001-233) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11586306/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet.&title=Haplotype%20tagging%20for%20the%20identification%20of%20common%20disease%20genes&author=GCL%20Johnson&author=L%20Esposito&author=BJ%20Barratt&volume=29&publication_year=2001&pages=233-237&pmid=11586306&doi=10.1038/ng1001-233&)

21. Patil N, Berno AJ, Hinds DA, et al. Blocks of limited haplotype diversity revealed by high-resolution scanning of human chromosome 21. Science. 2001;294:1719–1723. doi: 10.1126/science.1065573.  [DOI](https://doi.org/10.1126/science.1065573) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11721056/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Blocks%20of%20limited%20haplotype%20diversity%20revealed%20by%20high-resolution%20scanning%20of%20human%20chromosome%2021&author=N%20Patil&author=AJ%20Berno&author=DA%20Hinds&volume=294&publication_year=2001&pages=1719-1723&pmid=11721056&doi=10.1126/science.1065573&)

22. Gow JM, Hodges LM, Chinn LW, et al. Substrate-dependent effects of human ABCB1 coding polymorphisms. J. Pharmacol. Exp. Ther. 2008;325:435–442. doi: 10.1124/jpet.107.135194.  [DOI](https://doi.org/10.1124/jpet.107.135194) | [PMC free article](/articles/PMC2612726/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18287207/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Pharmacol.%20Exp.%20Ther.&title=Substrate-dependent%20effects%20of%20human%20ABCB1%20coding%20polymorphisms&author=JM%20Gow&author=LM%20Hodges&author=LW%20Chinn&volume=325&publication_year=2008&pages=435-442&pmid=18287207&doi=10.1124/jpet.107.135194&)

23. Yoon HY, Song TJ, Yee J, et al. Association between genetic polymorphisms and bleeding in patients on direct oral anticoagulants. Pharmaceutics. 2022;14(9):1889. doi: 10.3390/pharmaceutics14091889.  [DOI](https://doi.org/10.3390/pharmaceutics14091889) | [PMC free article](/articles/PMC9501033/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36145636/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmaceutics&title=Association%20between%20genetic%20polymorphisms%20and%20bleeding%20in%20patients%20on%20direct%20oral%20anticoagulants&author=HY%20Yoon&author=TJ%20Song&author=J%20Yee&volume=14&issue=9&publication_year=2022&pages=1889&pmid=36145636&doi=10.3390/pharmaceutics14091889&)

24. Kaatz S, Ahmad D, Spyropoulos AC, et al. Definition of clinically relevant non-major bleeding in studies of anticoagulants in atrial fibrillation and venous thromboembolic disease in non-surgical patients: Communication from the SSC of the ISTH. J. Thromb. Haemost. 2015;13(11):2119–2126. doi: 10.1111/jth.13140.  [DOI](https://doi.org/10.1111/jth.13140) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26764429/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Thromb.%20Haemost.&title=Definition%20of%20clinically%20relevant%20non-major%20bleeding%20in%20studies%20of%20anticoagulants%20in%20atrial%20fibrillation%20and%20venous%20thromboembolic%20disease%20in%20non-surgical%20patients:%20Communication%20from%20the%20SSC%20of%20the%20ISTH&author=S%20Kaatz&author=D%20Ahmad&author=AC%20Spyropoulos&volume=13&issue=11&publication_year=2015&pages=2119-2126&pmid=26764429&doi=10.1111/jth.13140&)

25. Schulman S, Kearon C. Definition of major bleeding in clinical investigations of antihemostatic medicinal products in non-surgical patients. J. Thromb. Haemost. 2005;3(4):692–694. doi: 10.1111/j.1538-7836.2005.01204.x.  [DOI](https://doi.org/10.1111/j.1538-7836.2005.01204.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15842354/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Thromb.%20Haemost.&title=Definition%20of%20major%20bleeding%20in%20clinical%20investigations%20of%20antihemostatic%20medicinal%20products%20in%20non-surgical%20patients&author=S%20Schulman&author=C%20Kearon&volume=3&issue=4&publication_year=2005&pages=692-694&pmid=15842354&doi=10.1111/j.1538-7836.2005.01204.x&)

26. Lip GY, Nieuwlaat R, Pisters R, et al. Refining clinical risk stratification for predicting stroke and thromboembolism in atrial fibrillation using a novel risk factor-based approach: the euro heart survey on atrial fibrillation. Chest. 2010;137(2):263–272. doi: 10.1378/chest.09-1584.  [DOI](https://doi.org/10.1378/chest.09-1584) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19762550/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chest&title=Refining%20clinical%20risk%20stratification%20for%20predicting%20stroke%20and%20thromboembolism%20in%20atrial%20fibrillation%20using%20a%20novel%20risk%20factor-based%20approach:%20the%20euro%20heart%20survey%20on%20atrial%20fibrillation&author=GY%20Lip&author=R%20Nieuwlaat&author=R%20Pisters&volume=137&issue=2&publication_year=2010&pages=263-272&pmid=19762550&doi=10.1378/chest.09-1584&)

27. Pisters R, Lane DA, Nieuwlaat R, et al. A novel user-friendly score (HAS-BLED) to assess 1-year risk of major bleeding in patients with atrial fibrillation: The Euro Heart Survey. Chest. 2010;138(5):1093–1100. doi: 10.1378/chest.10-0134.  [DOI](https://doi.org/10.1378/chest.10-0134) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20299623/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chest&title=A%20novel%20user-friendly%20score%20(HAS-BLED)%20to%20assess%201-year%20risk%20of%20major%20bleeding%20in%20patients%20with%20atrial%20fibrillation:%20The%20Euro%20Heart%20Survey&author=R%20Pisters&author=DA%20Lane&author=R%20Nieuwlaat&volume=138&issue=5&publication_year=2010&pages=1093-1100&pmid=20299623&doi=10.1378/chest.10-0134&)

28. Ward LD, Kellis M. HaploReg v4: Systematic mining of putative causal variants, cell types, regulators and target genes for human complex traits and disease. Nucleic Acids Res. 2016;44(D1):D877–881. doi: 10.1093/nar/gkv1340.  [DOI](https://doi.org/10.1093/nar/gkv1340) | [PMC free article](/articles/PMC4702929/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26657631/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&title=HaploReg%20v4:%20Systematic%20mining%20of%20putative%20causal%20variants,%20cell%20types,%20regulators%20and%20target%20genes%20for%20human%20complex%20traits%20and%20disease&author=LD%20Ward&author=M%20Kellis&volume=44&issue=D1&publication_year=2016&pages=D877-881&pmid=26657631&doi=10.1093/nar/gkv1340&)

29. Karas S, Etheridge A, Nickerson D, et al. Integration of DNA sequencing with population pharmacokinetics to improve the prediction of irinotecan exposure in cancer patients. Br. J. Cancer. 2022;126(4):640–651. doi: 10.1038/s41416-021-01589-2.  [DOI](https://doi.org/10.1038/s41416-021-01589-2) | [PMC free article](/articles/PMC8854585/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34703007/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Cancer&title=Integration%20of%20DNA%20sequencing%20with%20population%20pharmacokinetics%20to%20improve%20the%20prediction%20of%20irinotecan%20exposure%20in%20cancer%20patients&author=S%20Karas&author=A%20Etheridge&author=D%20Nickerson&volume=126&issue=4&publication_year=2022&pages=640-651&pmid=34703007&doi=10.1038/s41416-021-01589-2&)

30. Yin A, Ettaieb MH, Swen J, et al. Population pharmacokinetic and pharmacogenetic analysis of mitotane in patients with adrenocortical carcinoma: Towards individualized dosing. Clin. Pharmacokinet. 2021;60(1):89–102. doi: 10.1007/s40262-020-00913-y.  [DOI](https://doi.org/10.1007/s40262-020-00913-y) | [PMC free article](/articles/PMC7809008/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32607875/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacokinet.&title=Population%20pharmacokinetic%20and%20pharmacogenetic%20analysis%20of%20mitotane%20in%20patients%20with%20adrenocortical%20carcinoma:%20Towards%20individualized%20dosing&author=A%20Yin&author=MH%20Ettaieb&author=J%20Swen&volume=60&issue=1&publication_year=2021&pages=89-102&pmid=32607875&doi=10.1007/s40262-020-00913-y&)

31. Lopez-Lopez E, Martin-Guerrero I, Ballesteros J, et al. Polymorphisms of the SLCO1B1 gene predict methotrexate-related toxicity in childhood acute lymphoblastic leukemia. Pediatr. Blood Cancer. 2011;57(4):612–619. doi: 10.1002/pbc.23074.  [DOI](https://doi.org/10.1002/pbc.23074) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21387541/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pediatr.%20Blood%20Cancer.&title=Polymorphisms%20of%20the%20SLCO1B1%20gene%20predict%20methotrexate-related%20toxicity%20in%20childhood%20acute%20lymphoblastic%20leukemia&author=E%20Lopez-Lopez&author=I%20Martin-Guerrero&author=J%20Ballesteros&volume=57&issue=4&publication_year=2011&pages=612-619&pmid=21387541&doi=10.1002/pbc.23074&)

32. Eldeem I, Yavuz D, Cumaogullari O, et al. SLCO1B1 polymorphisms are associated with drug intolerance in childhood leukemia maintenance therapy. J. Pediatr. Hematol. Oncol. 2018;40(5):e289–e294. doi: 10.1097/MPH.0000000000001153.  [DOI](https://doi.org/10.1097/MPH.0000000000001153) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29683944/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Pediatr.%20Hematol.%20Oncol.&title=SLCO1B1%20polymorphisms%20are%20associated%20with%20drug%20intolerance%20in%20childhood%20leukemia%20maintenance%20therapy&author=I%20Eldeem&author=D%20Yavuz&author=O%20Cumaogullari&volume=40&issue=5&publication_year=2018&pages=e289-e294&pmid=29683944&doi=10.1097/MPH.0000000000001153&)

33. Chasman DI, Giulianini F, MacFadyen J, et al. Genetic determinants of statin-induced low-density lipoprotein cholesterol reduction: The justification for the use of statins in prevention: An Intervention Trial Evaluating Rosuvastatin (JUPITER) trial. Circ. Cardiovasc. Genet. 2012;5(2):257–264. doi: 10.1161/CIRCGENETICS.111.961144.  [DOI](https://doi.org/10.1161/CIRCGENETICS.111.961144) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22331829/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circ.%20Cardiovasc.%20Genet.&title=Genetic%20determinants%20of%20statin-induced%20low-density%20lipoprotein%20cholesterol%20reduction:%20The%20justification%20for%20the%20use%20of%20statins%20in%20prevention:%20An%20Intervention%20Trial%20Evaluating%20Rosuvastatin%20(JUPITER)%20trial&author=DI%20Chasman&author=F%20Giulianini&author=J%20MacFadyen&volume=5&issue=2&publication_year=2012&pages=257-264&pmid=22331829&doi=10.1161/CIRCGENETICS.111.961144&)

34. Chu A, Giulianini F, Barratt D, et al. Differential genetic effects on statin-induced changes across low-density lipoprotein-related measures. Circ. Cardiovasc. Genet. 2015;8(5):688–695. doi: 10.1161/CIRCGENETICS.114.000962.  [DOI](https://doi.org/10.1161/CIRCGENETICS.114.000962) | [PMC free article](/articles/PMC4618030/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26273092/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circ.%20Cardiovasc.%20Genet.&title=Differential%20genetic%20effects%20on%20statin-induced%20changes%20across%20low-density%20lipoprotein-related%20measures&author=A%20Chu&author=F%20Giulianini&author=D%20Barratt&volume=8&issue=5&publication_year=2015&pages=688-695&pmid=26273092&doi=10.1161/CIRCGENETICS.114.000962&)

35. Hu M, Mak VW, Tomlinson B. Intronic variants in SLCO1B1 related to statin-induced myopathy are associated with the low-density lipoprotein cholesterol response to statins in Chinese patients with hyperlipidaemia. Pharmacogenet. Genomics. 2012;22(11):803–806. doi: 10.1097/FPC.0b013e3283557c98.  [DOI](https://doi.org/10.1097/FPC.0b013e3283557c98) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22668755/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet.%20Genomics.&title=Intronic%20variants%20in%20SLCO1B1%20related%20to%20statin-induced%20myopathy%20are%20associated%20with%20the%20low-density%20lipoprotein%20cholesterol%20response%20to%20statins%20in%20Chinese%20patients%20with%20hyperlipidaemia&author=M%20Hu&author=VW%20Mak&author=B%20Tomlinson&volume=22&issue=11&publication_year=2012&pages=803-806&pmid=22668755&doi=10.1097/FPC.0b013e3283557c98&)

36. Ramsey L, Bruun G, Yang W, et al. Rare versus common variants in pharmacogenetics: SLCO1B1 variation and methotrexate disposition. Genome Res. 2012;22(1):1–8. doi: 10.1101/gr.129668.111.  [DOI](https://doi.org/10.1101/gr.129668.111) | [PMC free article](/articles/PMC3246196/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22147369/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res.&title=Rare%20versus%20common%20variants%20in%20pharmacogenetics:%20SLCO1B1%20variation%20and%20methotrexate%20disposition&author=L%20Ramsey&author=G%20Bruun&author=W%20Yang&volume=22&issue=1&publication_year=2012&pages=1-8&pmid=22147369&doi=10.1101/gr.129668.111&)

37. Prado Y, Saavedra N, Zambrano T, et al. SLCO1B1 c.388A>G polymorphism in associated with HDL-C levels in response to atorvastatin in Chilean individuals. Int. J. Mol. Sci. 2015;16(9):20609–20619. doi: 10.3390/ijms160920609.  [DOI](https://doi.org/10.3390/ijms160920609) | [PMC free article](/articles/PMC4613221/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26334272/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Mol.%20Sci.&title=SLCO1B1%20c.388A>G%20polymorphism%20in%20associated%20with%20HDL-C%20levels%20in%20response%20to%20atorvastatin%20in%20Chilean%20individuals&author=Y%20Prado&author=N%20Saavedra&author=T%20Zambrano&volume=16&issue=9&publication_year=2015&pages=20609-20619&pmid=26334272&doi=10.3390/ijms160920609&)

38. Dempsey J, Kidwell K, Gersch C, et al. Effects of SLCO1B1 polymorphisms on plasma estrogen concentrations in women with breast cancer receiving aromatase inhibitors exemestane and letrozole. Pharmacogenomics. 2019;20(8):571–580. doi: 10.2217/pgs-2019-0020.  [DOI](https://doi.org/10.2217/pgs-2019-0020) | [PMC free article](/articles/PMC6891932/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31190621/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Effects%20of%20SLCO1B1%20polymorphisms%20on%20plasma%20estrogen%20concentrations%20in%20women%20with%20breast%20cancer%20receiving%20aromatase%20inhibitors%20exemestane%20and%20letrozole&author=J%20Dempsey&author=K%20Kidwell&author=C%20Gersch&volume=20&issue=8&publication_year=2019&pages=571-580&pmid=31190621&doi=10.2217/pgs-2019-0020&)

39. Sun Q, Liu HP, Zheng RJ, et al. Genetic polymorphisms of SLCO1B1, CYP2E1 and UGT1A1 and susceptibility to anti-tuberculosis drug-induced hepatotoxicity: A Chinese population-based prospective case-control study. Clin. Drug Investig. 2017;37(12):1125–1136. doi: 10.1007/s40261-017-0572-6.  [DOI](https://doi.org/10.1007/s40261-017-0572-6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28900877/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Drug%20Investig.&title=Genetic%20polymorphisms%20of%20SLCO1B1,%20CYP2E1%20and%20UGT1A1%20and%20susceptibility%20to%20anti-tuberculosis%20drug-induced%20hepatotoxicity:%20A%20Chinese%20population-based%20prospective%20case-control%20study&author=Q%20Sun&author=HP%20Liu&author=RJ%20Zheng&volume=37&issue=12&publication_year=2017&pages=1125-1136&pmid=28900877&doi=10.1007/s40261-017-0572-6&)

40. Polonikov A, Bocharova L, Azarova L, et al. The impact of genetic polymorphisms in glutamate-cysteine ligase, a key enzyme of glutathione biosynthesis, on ischemic stroke risk and brain infarct size. Life (Basel) 2022;12(4):602. doi: 10.3390/life12040602.  [DOI](https://doi.org/10.3390/life12040602) | [PMC free article](/articles/PMC9032935/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35455093/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Life%20(Basel)&title=The%20impact%20of%20genetic%20polymorphisms%20in%20glutamate-cysteine%20ligase,%20a%20key%20enzyme%20of%20glutathione%20biosynthesis,%20on%20ischemic%20stroke%20risk%20and%20brain%20infarct%20size&author=A%20Polonikov&author=L%20Bocharova&author=L%20Azarova&volume=12&issue=4&publication_year=2022&pages=602&pmid=35455093&doi=10.3390/life12040602&)

41. Song G, Chung JE, Yee J, et al. Effects of SLCO1B1 and SLCO1B3 genetic polymorphisms on valsartan pharmacokinetics in healthy Korean volunteers. J. Pers. Med. 2021;11(9):862. doi: 10.3390/jpm11090862.  [DOI](https://doi.org/10.3390/jpm11090862) | [PMC free article](/articles/PMC8467379/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34575639/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Pers.%20Med.&title=Effects%20of%20SLCO1B1%20and%20SLCO1B3%20genetic%20polymorphisms%20on%20valsartan%20pharmacokinetics%20in%20healthy%20Korean%20volunteers&author=G%20Song&author=JE%20Chung&author=J%20Yee&volume=11&issue=9&publication_year=2021&pages=862&pmid=34575639&doi=10.3390/jpm11090862&)

42. Whirl-Carrillo M, McDonagh EM, Hebert JM, et al. Pharmacogenomics knowledge for personalized medicine. Clin. Pharmacol. Ther. 2012;92(4):414–417. doi: 10.1038/clpt.2012.96.  [DOI](https://doi.org/10.1038/clpt.2012.96) | [PMC free article](/articles/PMC3660037/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22992668/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Pharmacogenomics%20knowledge%20for%20personalized%20medicine&author=M%20Whirl-Carrillo&author=EM%20McDonagh&author=JM%20Hebert&volume=92&issue=4&publication_year=2012&pages=414-417&pmid=22992668&doi=10.1038/clpt.2012.96&)

43. Cen H, Wen QW, Zhang HQ, et al. Associations between genetic polymorphisms within transporter genes and clinical response to methotrexate in Chinese rheumatoid arthritis patients: A pilot study. Pharm. Pers. Med. 2022;15:327–339. doi: 10.2147/PGPM.S350417.  [DOI](https://doi.org/10.2147/PGPM.S350417) | [PMC free article](/articles/PMC9013256/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35437350/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharm.%20Pers.%20Med.&title=Associations%20between%20genetic%20polymorphisms%20within%20transporter%20genes%20and%20clinical%20response%20to%20methotrexate%20in%20Chinese%20rheumatoid%20arthritis%20patients:%20A%20pilot%20study&author=H%20Cen&author=QW%20Wen&author=HQ%20Zhang&volume=15&publication_year=2022&pages=327-339&pmid=35437350&doi=10.2147/PGPM.S350417&)

44. Mukonzo JK, Röshammar D, Waako P, et al. A novel polymorphism in ABCB1 gene, CYP2B6*6 and sex predict single-dose efavirenz population pharmacokinetics in Ugandans. Br. J. Clin. Pharmacol. 2009;68:690–699. doi: 10.1111/j.1365-2125.2009.03516.x.  [DOI](https://doi.org/10.1111/j.1365-2125.2009.03516.x) | [PMC free article](/articles/PMC2791975/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19916993/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Clin.%20Pharmacol.&title=A%20novel%20polymorphism%20in%20ABCB1%20gene,%20CYP2B6*6%20and%20sex%20predict%20single-dose%20efavirenz%20population%20pharmacokinetics%20in%20Ugandans&author=JK%20Mukonzo&author=D%20R%C3%B6shammar&author=P%20Waako&volume=68&publication_year=2009&pages=690-699&pmid=19916993&doi=10.1111/j.1365-2125.2009.03516.x&)

45. Connolly SJ, Ezekowitz MD, Yusuf S, et al. Dabigatran versus warfarin in patients with atrial fibrillation. N. Engl. J. Med. 2009;361(12):1139–1151. doi: 10.1056/NEJMoa0905561.  [DOI](https://doi.org/10.1056/NEJMoa0905561) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19717844/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%20Engl.%20J.%20Med.&title=Dabigatran%20versus%20warfarin%20in%20patients%20with%20atrial%20fibrillation&author=SJ%20Connolly&author=MD%20Ezekowitz&author=S%20Yusuf&volume=361&issue=12&publication_year=2009&pages=1139-1151&pmid=19717844&doi=10.1056/NEJMoa0905561&)

46. Patel MR, Mahaffey KW, Garg J, et al. Rivaroxaban versus warfarin in nonvalvular atrial fibrillation. N. Engl. J. Med. 2011;365(10):883–891. doi: 10.1056/NEJMoa1009638.  [DOI](https://doi.org/10.1056/NEJMoa1009638) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21830957/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%20Engl.%20J.%20Med.&title=Rivaroxaban%20versus%20warfarin%20in%20nonvalvular%20atrial%20fibrillation&author=MR%20Patel&author=KW%20Mahaffey&author=J%20Garg&volume=365&issue=10&publication_year=2011&pages=883-891&pmid=21830957&doi=10.1056/NEJMoa1009638&)

47. Granger CB, Alexander JH, McMurray JJ, et al. Apixaban versus warfarin in patients with atrial fibrillation. N. Engl. J. Med. 2011;365(11):981–992. doi: 10.1056/NEJMoa1107039.  [DOI](https://doi.org/10.1056/NEJMoa1107039) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21870978/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%20Engl.%20J.%20Med.&title=Apixaban%20versus%20warfarin%20in%20patients%20with%20atrial%20fibrillation&author=CB%20Granger&author=JH%20Alexander&author=JJ%20McMurray&volume=365&issue=11&publication_year=2011&pages=981-992&pmid=21870978&doi=10.1056/NEJMoa1107039&)

48. Kalliokoski A, Niemi M. Impact of OATP transporters on pharmacokinetics. Br. J. Pharmacol. 2009;158(3):693–705. doi: 10.1111/j.1476-5381.2009.00430.x.  [DOI](https://doi.org/10.1111/j.1476-5381.2009.00430.x) | [PMC free article](/articles/PMC2765590/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19785645/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Pharmacol.&title=Impact%20of%20OATP%20transporters%20on%20pharmacokinetics&author=A%20Kalliokoski&author=M%20Niemi&volume=158&issue=3&publication_year=2009&pages=693-705&pmid=19785645&doi=10.1111/j.1476-5381.2009.00430.x&)

49. Bathala MS, Masumoto H, Oguma T, et al. Pharmacokinetics, biotransformation, and mass balance of edoxaban, a selective, direct factor Xa inhibitor, in humans. Drug Metab. Drug Metab. Dispos. 2012;40:2250–2255. doi: 10.1124/dmd.112.046888.  [DOI](https://doi.org/10.1124/dmd.112.046888) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22936313/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab.%20Drug%20Metab.%20Dispos.&title=Pharmacokinetics,%20biotransformation,%20and%20mass%20balance%20of%20edoxaban,%20a%20selective,%20direct%20factor%20Xa%20inhibitor,%20in%20humans&author=MS%20Bathala&author=H%20Masumoto&author=T%20Oguma&volume=40&publication_year=2012&pages=2250-2255&pmid=22936313&doi=10.1124/dmd.112.046888&)

50. Liu J, Liu XY, Chen S, et al. SLCO1B1 521T>C polymorphism associated with rosuvastatin-induced myotoxiciy in Chinese coronary artery disease patients: A nested case-control study. Eur. J. Clin. Pharm. 2017;73:1409–1416. doi: 10.1007/s00228-017-2318-z.  [DOI](https://doi.org/10.1007/s00228-017-2318-z) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28812116/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Clin.%20Pharm.&title=SLCO1B1%20521T>C%20polymorphism%20associated%20with%20rosuvastatin-induced%20myotoxiciy%20in%20Chinese%20coronary%20artery%20disease%20patients:%20A%20nested%20case-control%20study&author=J%20Liu&author=XY%20Liu&author=S%20Chen&volume=73&publication_year=2017&pages=1409-1416&pmid=28812116&doi=10.1007/s00228-017-2318-z&)

51. DeGorter MK, Tirona RG, Schwarz UI, et al. Clinical and pharmacogenetic predictors of circulating atorvastatin and rosuvastatin concentrations in routine clinical care. Circ. Cardiovasc. Genet. 2013;6(4):400–408. doi: 10.1161/CIRCGENETICS.113.000099.  [DOI](https://doi.org/10.1161/CIRCGENETICS.113.000099) | [PMC free article](/articles/PMC3922121/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23876492/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circ.%20Cardiovasc.%20Genet.&title=Clinical%20and%20pharmacogenetic%20predictors%20of%20circulating%20atorvastatin%20and%20rosuvastatin%20concentrations%20in%20routine%20clinical%20care&author=MK%20DeGorter&author=RG%20Tirona&author=UI%20Schwarz&volume=6&issue=4&publication_year=2013&pages=400-408&pmid=23876492&doi=10.1161/CIRCGENETICS.113.000099&)

52. Thompson LE, Davis BH, Narayan R, et al. Personalizing direct oral anticoagulant therapy for a diverse population: Role of race, kidney function, drug interactions, and pharmacogenetics. Clin. Pharmacol. Ther. 2023;113(3):585–599. doi: 10.1002/cpt.2714.  [DOI](https://doi.org/10.1002/cpt.2714) | [PMC free article](/articles/PMC9852362/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35857814/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Personalizing%20direct%20oral%20anticoagulant%20therapy%20for%20a%20diverse%20population:%20Role%20of%20race,%20kidney%20function,%20drug%20interactions,%20and%20pharmacogenetics&author=LE%20Thompson&author=BH%20Davis&author=R%20Narayan&volume=113&issue=3&publication_year=2023&pages=585-599&pmid=35857814&doi=10.1002/cpt.2714&)

53. Yao X, Shah ND, Sangaralingham LR, et al. Non-Vitamin K Antagonist oral anticoagulant dosing in patients with atrial fibrillation and renal dysfunction. J. Am. Coll. Cardiol. 2017;69(23):2779–2790. doi: 10.1016/j.jacc.2017.03.600.  [DOI](https://doi.org/10.1016/j.jacc.2017.03.600) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28595692/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Am.%20Coll.%20Cardiol.&title=Non-Vitamin%20K%20Antagonist%20oral%20anticoagulant%20dosing%20in%20patients%20with%20atrial%20fibrillation%20and%20renal%20dysfunction&author=X%20Yao&author=ND%20Shah&author=LR%20Sangaralingham&volume=69&issue=23&publication_year=2017&pages=2779-2790&pmid=28595692&doi=10.1016/j.jacc.2017.03.600&)

54. Santos J, António N, Rocha M, et al. Impact of direct oral anticoagulant off-label doses on clinical outcomes of atrial fibrillation patients: A systematic review. Br. J. Clin. Pharmacol. 2020;86(3):533–547. doi: 10.1111/bcp.14127.  [DOI](https://doi.org/10.1111/bcp.14127) | [PMC free article](/articles/PMC7080635/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31631392/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Clin.%20Pharmacol.&title=Impact%20of%20direct%20oral%20anticoagulant%20off-label%20doses%20on%20clinical%20outcomes%20of%20atrial%20fibrillation%20patients:%20A%20systematic%20review&author=J%20Santos&author=N%20Ant%C3%B3nio&author=M%20Rocha&volume=86&issue=3&publication_year=2020&pages=533-547&pmid=31631392&doi=10.1111/bcp.14127&)
