# CD80 DNA methylation and single-nucleotide polymorphism associated with clopidogrel response: a whole-genome DNA methylation analysis in acute coronary syndrome

## Metadata
**Authors:** Pei-Yuan Song, Mu-Peng Li, Li-Ming Peng, Xiao-Ping Chen
**Journal:** Research and Practice in Thrombosis and Haemostasis
**Date:** 2023 Feb 24
**DOI:** [10.1016/j.rpth.2023.100093](https://doi.org/10.1016/j.rpth.2023.100093)
**PMID:** 36970128
**PMCID:** PMC10031538
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10031538/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC10031538/pdf/main.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC10031538/pdf/main.pdf)

## Abstract

**Background:** 
Dual antiplatelet therapy with clopidogrel and aspirin is the primary treatment for patients who undergo percutaneous coronary intervention. However, the interindividual difference in clopidogrel response is remarkable, and high on-treatment platelet reactivity (HTPR) can increase the risk of thrombotic events after percutaneous coronary intervention.

**Objective:** 
We studied novel accessible factors that possibly affect clopidogrel response in DNA methylation.

**Methods:** 
Methylation 850K bead chips were used to detect DNA methylation levels. The platelet reactivity index (PRI) was determined in 330 subjects with acute coronary syndrome (ACS) after administration of clopidogrel 300 mg loading dose or at least 5 days of 75 mg daily maintenance dose.

**Results:** 
Overall, 32 discovery samples showed extreme clopidogrel response: 16 with HTPR (PRI > 75%) and 16 with non-HTPR (PRI < 26%). Overall, 61 differential methylation loci (DMLs) were observed between the 2 groups. Most were in the open sea and intergenic regions in the genome. In the validation stage, HTPR showed a lower level of CD80_cg06300880 methylation. Carriers of rs34394661 AA genotype, a CpG-single-nucleotide polymorphism at the CD80_cg06300880 locus, showed an increased odds for HTPR (overall odds ratio of patients with ACS = 7.31, 95% CI: 1.69-31.59, P = .008; non-ST elevation myocardial infarction-ACS: odds ratio = 12.69, 95% CI: 1.68-96.08, P = .01) and decreased CD80_cg06300880 methylation (P < .0001). Multivariate regression analysis showed that both CYP2C19 poor metabolizers and CD80_rs34394661 AA (P = .009) genotype were associated with higher odds for HTPR in the overall samples. In contrast, CD80_cg06300880 methylation (P = .002) caused lower odds for HTPR in patients with non-ST elevation myocardial infarction-ACS.

**Conclusion:** 
CD80_cg06300880 and CpG-single-nucleotide polymorphism rs34394661 could be independent predictors of HTPR with clopidogrel therapy.

Keywords: acute coronary syndrome (ACS), clopidogrel, DNA methylation, platelet, single-nucleotide polymorphism (SNP)

### Background

Dual antiplatelet therapy with clopidogrel and aspirin is the primary treatment for patients who undergo percutaneous coronary intervention. However, the interindividual difference in clopidogrel response is remarkable, and high on-treatment platelet reactivity (HTPR) can increase the risk of thrombotic events after percutaneous coronary intervention.

### Objective

We studied novel accessible factors that possibly affect clopidogrel response in DNA methylation.

### Methods

Methylation 850K bead chips were used to detect DNA methylation levels. The platelet reactivity index (PRI) was determined in 330 subjects with acute coronary syndrome (ACS) after administration of clopidogrel 300 mg loading dose or at least 5 days of 75 mg daily maintenance dose.

### Results

Overall, 32 discovery samples showed extreme clopidogrel response: 16 with HTPR (PRI > 75%) and 16 with non-HTPR (PRI < 26%). Overall, 61 differential methylation loci (DMLs) were observed between the 2 groups. Most were in the open sea and intergenic regions in the genome. In the validation stage, HTPR showed a lower level of *CD80_*cg06300880 methylation. Carriers of rs34394661 AA genotype, a CpG-single-nucleotide polymorphism at the *CD80_*cg06300880 locus, showed an increased odds for HTPR (overall odds ratio of patients with ACS = 7.31, 95% CI: 1.69-31.59, *P* = .008; non-ST elevation myocardial infarction-ACS: odds ratio = 12.69, 95% CI: 1.68-96.08, *P* = .01) and decreased *CD80_*cg06300880 methylation (*P* < .0001). Multivariate regression analysis showed that both *CYP2C19* poor metabolizers and *CD80_*rs34394661 AA (*P* = .009) genotype were associated with higher odds for HTPR in the overall samples. In contrast, *CD80_*cg06300880 methylation (*P* = .002) caused lower odds for HTPR in patients with non-ST elevation myocardial infarction-ACS.

### Conclusion

*CD80_*cg06300880 and CpG-single-nucleotide polymorphism rs34394661 could be independent predictors of HTPR with clopidogrel therapy.

## 1. Introduction

Percutaneous coronary intervention (PCI) is a routine treatment for patients with acute coronary syndrome (ACS). [[1](#bib1)] However, PCI and implanted stents can cause platelet activation and aggregation [[2](#bib2)], coronary plaque instability [[3](#bib3)], and endothelial dysfunction [[4](#bib4)], increasing the risk of severe ischemic events such as in-stent thrombosis and myocardial infarction. [[5](#bib5)] Dual antiplatelet therapy of aspirin and clopidogrel (a P2Y12 receptor inhibitor) is a standard treatment for preventing ischemic events such as in-stent thrombosis after PCI. [[6](#bib6),[7](#bib7)] Although dual antiplatelet therapy can significantly reduce the incidence of recurrent ischemic events, cardiovascular events still occur during clopidogrel therapy. [[8](#bib8)] As indicated by high on-treatment platelet reactivity (HTPR), clopidogrel resistance is one of the reasons for dual antiplatelet therapy treatment failures. [[9](#bib9),[10](#bib10)]

A remarkable interindividual difference in the response and clinical efficacy of clopidogrel is observed, and genetic factors are one of the determinants of these differences. Genetic variants affecting clopidogrel pharmacodynamics and pharmacokinetics pathways have been reported, such as cytochrome P450 CYP2C19, COX-1, PON1, and P2Y12 receptors [[11](#bib11)], especially CYP2C19. [[12](#bib12),[13](#bib13)] These variants impact clopidogrel efficacy. We have also validated or reported several polymorphisms that affect clopidogrel response in previous studies. [[[14]](#bib14), [[15]](#bib15), [[16]](#bib16), [[17]](#bib17)]

Patients are defined as normal metabolizers (*CYP2C19*∗1/∗1), intermediate metabolizers (*CYP2C19*∗1/∗2 or ∗1/∗3), and poor metabolizers (*CYP2C19*∗2/∗2 or ∗2/∗3 or ∗3/∗3), according to *CYP2C19* genotypes. [[18](#bib18)] When pharmacodynamics and pharmacokinetic parameters such as maximum platelet aggregation rate and area under the curve of active clopidogrel metabolite are evaluated, *CYP2C19* poor metabolizers require 4 times the routine dose of clopidogrel to achieve antiplatelet effects compared with those of normal metabolizers. [[19](#bib19)] Of note, a proportion (58.4%) of responders in *CYP2C19* are poor metabolizers, whereas a certain proportion (31.6%) of *CYP2C19* normal metabolizers are resistant to clopidogrel. [[20](#bib20)] However, the loss-of-function alleles of *CYP2C19* only explain 12% of individual differences in response to clopidogrel, and nongenetic factors such as old age, hyperlipidemia, and obesity explain ∼10% of clopidogrel resistance. [[21](#bib21)] Some other unknown genetic factors may also contribute to interindividual variations in clopidogrel response.

As a common and conservative epigenetic modification of genomic DNA, DNA methylation is involved in many biological processes. Abnormal DNA methylation can lead to development of various diseases, including coronary heart disease. [[22](#bib22)] Compared with genome sequence variations, epigenetic regulation such as DNA methylation is more dynamic. [[23](#bib23)]

The influence of DNA methylation on drug responses such as chemosensitivity and disease prognosis has been extensively studied in cancer. [[[24]](#bib24), [[25]](#bib25), [[26]](#bib26)] Current studies on the association between DNA methylation and clopidogrel resistance primarily focus on genes related to clopidogrel pharmacokinetics, such as ABCB1 that affects clopidogrel absorption in the small intestine and bile duct [[27](#bib27)], the rate-limiting enzymes CYP2C19 and PON1, which metabolize the activation of clopidogrel in the liver [[28](#bib28),[29](#bib29)], and the drug target P2Y12. [[30](#bib30)] Genome-wide methylation analysis identifies genes involved in the clopidogrel response, especially those in pharmacodynamic pathways.

This study analyzed genome-wide DNA methylation using an EPIC (850K) methylation chip to detect DMLs that could affect clopidogrel antiplatelet activity. Further analysis was performed to reveal the role of DNA methylation in clopidogrel response with the joining genomic variations. Finally, a dynamic prediction model for clopidogrel response was constructed, incorporating the genetic variations of *CYP2C19* loss-of-function mutants and *CD80* CpG-single-nucleotide polymorphism (SNP). These findings provide a theoretical basis for individualized clopidogrel therapy.

## 2. Methods

### 2.1. Study subjects

From November 2015 to March 2017, a total of 330 Chinese patients aged 18 to 80 years were recruited at Xiangya Hospital, Central South University. All patients were newly diagnosed with ACS and categorized into 3 subtypes: ST elevation myocardial infarction (STEMI), non-STEMI (NSTEMI), and unstable angina. The former is usually referred to as STE-ACS and the latter 2 as NSTE-ACS. [[31](#bib31)]

All patients received PCI and dual antiplatelet therapy using clopidogrel and aspirin. Exclusion criteria are shown in [Supplementary Figure 1](#appsec1). The clinical information collected included baseline characteristics (gender, age, medical history, smoking and alcohol history, and complications), medications, and others.

### 2.2. Blood sample preparation and platelet function determination

Peripheral venous blood samples (3 mL) were collected for the platelet VASP/P2Y12 assay and genomic DNA extraction. Samples were placed into anticoagulated tubes with 3.8% sodium citrate (1:9 citrate/blood, v/v, BD Biosciences). Samples were taken from patients 12 to 24 hours after clopidogrel LD or at least 5 days of MD therapy. VASP phosphorylation in platelets was measured by flow cytometry (Beckman Kurt company). The platelet reactivity index (PRI) reflecting the clopidogrel response was calculated according to the manufacturer’s instructions (BIOCYTEX). HTPR was defined as PRI > 50% as described elsewhere. [[32](#bib32)] Patients with PRI ≤ 50% were assigned to the non-HTPR group. To select DMLs that could affect the clopidogrel response, 16 subjects (PRI > 75%) in the HTPR group and 16 age-, gender-, and *CYP2C19* genotype-matched subjects (PRI < 26%) in the non-HTPR group were selected as the discovery samples. The rest samples were used for validation.

Genomic DNA samples were extracted from whole blood samples using a whole blood DNA extraction kit (Promega). DNA concentration was determined on a spectrophotometer (Thermo Fisher Scientific). DNA samples were stored at −80 °C until use. Polymorphisms of *CYP2C19∗2* (rs4244285) and *∗3* (rs4986893) were genotyped by polymerase chain reaction (PCR)-based sequencing. PCR primers for *CYP2C19∗2* genotyping were 5′-tcagaggctgcttgatagaaatc-3′ and 5′-ccttgacctgttaaacatccgta-3′. PCR primers for *CYP2C19∗3* genotyping were 5′-cttcaccctgtgatcccact-3′ and 5′-aaacatgccaattcagcaca-3′.

### 2.3. Identification of DMLs between HTPR and non-HTPR patients using a genome-wide methylation chip kit

Human Methylation 850K Bead Chip Kits (Illumina, Inc.) were used to complete bead chip processing. All bead chips were scanned according to the ISCAN system user guide to paving whole-genome fluorescence signal data. The original data were imported and assessed using Genome Studio (Illumina, Inc). The original signal intensities of each locus were computed according to methylation analysis algorithms. After loci filtering, fluorescence offset correction, background correction, quantile normalization, and probe type bias correction, each site's methylation level (beta value) was obtained. These procedures were supported by Genesky Biotechnologies Inc.

### 2.4. Validation of DMLs by DNA methylation-targeted sequencing

The validation stage included 298 samples. DNA methylation levels were detected by methylation-targeted next-generation sequencing. [[33](#bib33),[34](#bib34)] First multiple PCR amplification was performed on bisulfite-transformed DNA samples, and specific tags were added to the products. The amplification signal was amplified by mixing the PCR library and then sequenced.

According to the annotation of Methylation EPIC v1.0B3 Manifest File of the Illumina Methylation EPIC (850K) chip, DML *CD80_*cg06300880 contains the SNP rs34394661. Genotypes of CpG-SNP could be identified by next-generation methylation sequencing simultaneously. *CD80_*rs34394661 genotypes were also validated by direct sequencing. The forward primer 5′-tgtttctgtgctggtctcaagt-3′ and reverse primer 5′-gaggtaaggcaggaattgtcac-3′ were used for PCR amplification and sequencing.

### 2.5. Statistical analysis

Continuous variables are expressed as mean ± SD, and categorical variables are expressed as frequency (percentage). R language and LIMMA package were used to analyze differences in methylation levels of genome-wide methylation loci between HTPR and non-HTPR groups. A significant difference was judged with |Δβ| > 0.2. [[35](#bib35)] Becasue of small sample size and large variability of clinical samples in the discovery cohort, no CpG sites passed the threshold required to correct for the false discovery rate. Therefore, an uncorrected *P* value < .05 was used as the filtration criteria. [[36](#bib36)]

Statistical analyses were performed using SPSS25.0 (SPSS Inc). An independent sample *t*-test was used to compare continuous variables (age, biochemical indices, methylation level, PRI) between groups, and 1-way analysis of variance was used for comparisons among 3 or more groups. The paired *t*-test was conducted to compare PRI values before and after clopidogrel treatment. The chi-squared test was used to analyze genotype distribution between the groups. Binary logistic regression was analyzed to screen for factors associated with clopidogrel HTPR. *P* < .05 was considered statistically significant.

## 3. Results

### 3.1. Characteristics of patients

Overall, 456 participants were approached, and 361 patients with ACS signed the informed consent (79.17% consent rate). Of these, 25 did not meet the inclusion criteria and 6 later declined to provide blood samples, and 330 patients were finally included ([Supplementary Fig. 1](#appsec1)). PRI was obtained from all patients after LD or MD of clopidogrel. No significant differences in mean PRI values were observed between LD and MD groups (LD: 58.14% ± 23.18%, n = 178; MD: 54.28% ± 19.32%, n = 152, *P* = .104, respectively, [Supplementary Fig. 2A](#appsec1)). No differences in PRI were observed between clopidogrel LD and MD for 6 cases with PRI values determined after both LD and MD clopidogrel administration (*P* = .753; [Supplementary Fig. 2B](#appsec1)). Therefore, patients receiving LD and MD were considered as 1 group for the subsequent analysis.

Overall, 32 patients with extreme PRI values (22 LD and 10 MD samples, respectively) were selected for the discovery stage. The remaining 298 patients (156 LD and 142 MD samples) were used in the validation stage. The baseline characteristics of the patients are shown in [Table 1](#tbl1). In the discovery cohort, the mean PRI value was significantly higher in the HTPR group than in the non-HTPR group (84.1% ± 4.7% vs. 16.3% ± 7.6%, *P* = 3.332E-21, [Table 1](#tbl1)). The distribution of *CYP2C19* genotypes was also consistent (*P* = 1.000, [Table 1](#tbl1)).

### Table 1.

| Characteristics | Discovery cohort |  |  | Validation cohort |  |  |
| --- | --- | --- | --- | --- | --- | --- |
| Non-HTPR (n = 16) | HTPR (n = 16) | P value | Non-HTPR (n = 105) | HTPR (n = 193) | P value |  |
| Ethnicity (East Asian), n (%) | 16 (100) | 16 (100) | NA | 105 (100) | 193 (100) | NA |
| Age (years) | 59.8 ± 11.2 | 58.9 ± 9.8 | 0.829 | 62.5 ± 10.6 | 60.7 ± 9.7 | .153 |
| Male, n (%) | 11 (68.8) | 12 (75.0) | 0.694 | 71 (67.6) | 123 (63.7) | .501 |
| Hemaglobin (g/L) | 136.6 ± 18.1 | 137.7 ± 16.3 | 0.866 | 135.8 ± 19.2 | 136.3 ± 16 | .817 |
| Platelet count (10E9/L) | 212.7 ± 80.3 | 220.9 ± 61.8 | 0.749 | 205.3 ± 78.4 | 203.7 ± 65.7 | .877 |
| Mean platelet volume (fl) | 10.8 ± 2.1 | 10.5 ± 2.4 | 0.732 | 10.5 ± 1.9 | 10.8 ± 2.1 | .287 |
| Glucose (mmol/L) | 4.9 ± 0.4 | 5.5 ± 1.5 | 0.117 | 5.9 ± 1.9 | 5.8 ± 1.8 | .769 |
| Glycated serum proteins (mmol/L) | 2.0 ± 0.2 | 2.1 ± 0.4 | 0.180 | 2.1 ± 0.3 | 2.1 ± 0.3 | .845 |
| Total cholesterol (mmol/L) | 4.2 ± 2 | 4.3 ± 0.8 | 0.784 | 4.2 ± 1.3 | 4.5 ± 1.2 | .116 |
| Triglyceride (mmol/L) | 1.6 ± 0.6 | 1.9 ± 0.6 | 0.152 | 1.9 ± 1.3 | 2.2 ± 1.5 | .114 |
| High-density lipoprotein cholesterol (mmol/L) | 1.1 ± 0.2 | 1.0 ± 0.2 | 0.512 | 1.1 ± 0.3 | 1.0 ± 0.3 | .687 |
| Low-density lipoprotein cholesterol (mmol/L) | 2.6 ± 1.8 | 2.7 ± 0.6 | 0.855 | 2.6 ± 1.0 | 3.0 ± 1.7 | .075 |
| Aspartate transaminase (U/L) | 45.5 ± 62.7 | 43.8 ± 58.3 | 0.937 | 49.6 ± 102.9 | 41.3 ± 62.6 | .424 |
| Alanine transferase (U/L) | 32.0 ± 21.2 | 21.7 ± 13.9 | 0.124 | 37.6 ± 45.8 | 35.8 ± 39.3 | .733 |
| PRI (%) | 16.3 ± 7.6 | 84.1 ± 4.7 | 3.332E-21a | 35.2 ± 11.3 | 68.9 ± 10.9 | 1.167E-75a |
| Current smoking, n (%)b | 9 (56.3) | 7 (43.8) | 0.480 | 34 (35.8) | 70 (38.5) | .663 |
| Alcohol use, n (%)b | 5 (31.3) | 7 (43.8) | 0.465 | 24 (25.3) | 54 (29.7) | .439 |
| Dyslipidemia, n (%)b | 12 (75.0) | 10 (62.5) | 0.446 | 66 (69.5) | 104 (57.1) | .045a |
| Diabetes mellitus, n (%)b | 1 (6.3) | 1 (6.3) | 1 | 24 (25.3) | 39 (21.4) | .470 |
| Hypertension, n (%)b | 2 (12.5) | 2 (12.5) | 1 | 19 (20.0) | 48 (26.4) | .240 |
| CP2C19 genotypes |   |   | 1 |   |   | 4.352E-7a |
| Normal metabolizer, n (%) | 11 (68.8) | 11 (68.8) |   | 67 (63.8) | 64 (33.2) |   |
| Intermediate metabolizer, n (%) | 5 (31.3) | 5 (31.3) |   | 35 (33.3) | 99 (51.3) |   |
| Poor metabolizer, n (%) | 0 (0) | 0 (0) |   | 3 (2.9) | 30 (15.5) |   |

Table 1 Caption: Baseline demographics and clinical characteristics of the patients.

The validation population was divided into 2 groups according to the PRI value: 193 patients with HTPR (PRI ≥ 50%) and 105 patients with non-HTPR (PRI < 50%). *CYP2C19* poor metabolizers were significantly overrepresented in the HTPR group (15.5% in HTPR vs*.* 2.9% non-HTPR, *P* = 4.352E-7, [Table 1](#tbl1)), ([Supplementary Fig. 3](#appsec1)). The 2 groups had no significant difference in almost all patient demographics, clinical history, and laboratory findings in both stages (all *P* > .05, [Table 1](#tbl1)). Hypertension was more frequent in patients with HTPR only in the validation samples (*P* = 0.045, [Table 1](#tbl1)). Subgroup analyses did not show a difference in PRI values between the STE-ACS and NSTE-ACS subgroups (STE-ACS: 54.97% ± 21.11%, n = 53; NSTE-ACS:57.47% ± 19.11%, n = 245, *P* = .398, [Supplementary Fig. 4](#appsec1)).

### 3.2. Identification of DMLs between patients with HTPR and patients with non-HTPR

For the 32 discovery samples, 61 sites showed differences in DNA methylation levels between HTPR and non-HTPR groups ([Figure 1](#fig1)A, [Supplementary Table 1](#appsec1)). Compared with the non-HTPR group, DNA methylation levels of 27 DMLs were increased, whereas 34 DMLs were decreased in the HTPR group ([Figure 1](#fig1)A). The cluster map based on 61 DMLs distinguished patients with HTPR from patients with non-HTPR ([Figure 1](#fig1)B). DMLs on chromosomes 1-12 accounted for 81.97% and 72.17% of increased and decreased methylation loci ([Supplementary Table 3](#appsec1)), respectively, which could be explained by the concentrated number of methylation sites on these chromosomes selected on 850K chips ([Supplementary Fig. 5](#appsec1)). Few DMLs were located on CpG islands, shore, and shelf regions. In contrast, most DMLs were found in open sea regions (4 kb away from the island) and intergenic regions ([Supplementary Fig. 7](#appsec1)), regardless of directions of changes in patients with HTPR. A similar pattern of genome distribution for DMLs was observed when 22 patients were analyzed (11 HTPR and 11 non-HTPR) limited to LD clopidogrel administration ([Supplementary Figs. 6](#appsec1) and [8](#appsec1); [Supplementary Tables 2](#appsec1) and [3](#appsec1)).

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3295/10031538/c139bdc0c4db/gr1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=10031538_gr1.jpg)

850K chip analysis of DMLs between patients with different clopidogrel responses in the discovery stage. (A) Volcano plot distinguished DMLs between patients with non-HTPR and HTPR (n = 16 for each) in the discovery samples; (B) Clustering analysis of 61 DMLs between non-HTPR and HTPR patients with heat map in the discovery samples

### 3.3. Verification of DMLs

Because of high conservation of intragenic regions [[[37]](#bib37), [[38]](#bib38), [[39]](#bib39)], the top 10 DMLs ranked by *P* values within intragenic regions instead of intergenic areas were validated by targeted DNA methylation sequencing in 298 validation patients ([Table 2](#tbl2)). Except for *OR52M1_*cg17040924, the remaining 9 DMLs were successfully sequenced. The recently reported *GCK_*cg03812172 site [[40](#bib40)] was also validated ([Table 3](#tbl3)). Only the *CD80_*cg06300880 site showed a significant difference between non-HTPR and HTPR groups with the same gain and loss of methylation level as the discovery samples. In the entire population, patients with non-HTPR showed a significantly higher methylation level of *CD80*_cg06300880 than patients with HTPR (non-HTPR: 0.70 ± 0.24, n = 105; HTPR: 0.61 ± 0.31, n = 193, *P* = .005, [Table 3](#tbl3)). In the validation samples, a similar difference in *CD80_*cg06300880 methylation was observed between patients with non-HTPR and HTPR with LD clopidogrel administration (non-HTPR: 0.72 ± 0.23, n = 44; HTPR: 0.61 ± 0.31, n = 112, *P* = .017, [Supplementary Table 4](#appsec1)). To determine whether *CD80_*cg06300880 methylation was similar in ACS subgroups, further analysis was performed in the NSTE and STE subgroups separately. The decreased methylation of *CD80*_cg06300880 was observed only in patients with HTPR in the NSTE subgroup (non-HTPR: 0.71 ± 0.23, n = 85; HTPR: 0.60 ± 0.31, n = 160, *P* = .002, [Supplementary Table 5](#appsec1)).

### Table 2.

| IlmnID | CHR | CpG island | Gene name | Gene group | Δβ value | P value |
| --- | --- | --- | --- | --- | --- | --- |
| cg02573091 | 5 | Shore | ANKDD1B | Body | −0.220 | .002 |
| cg26038514 | 8 | Sea | ASAP1 | Body | 0.338 | .002 |
| cg06300880 | 3 | Sea | CD80 | TSS1500 | −0.269 | .002 |
| cg14007688 | 9 | Sea | DBH-AS1 | TSS1500 | −0.319 | .004 |
| cg13564529 | 9 | Sea | VAV2 | Body | −0.309 | .005 |
| cg11251367 | 1 | Sea | FMN2 | Body | −0.344 | .005 |
| cg10122899 | 12 | Sea | P2RX7 | Body | −0.291 | .005 |
| cg17040924 | 11 | Sea | OR52M1 | TSS1500 | 0.309 | .006 |
| cg13739190 | 5 | Sea | MAN2A1 | Body | −0.200 | .007 |
| cg00258945 | 5 | Sea | MAN2A1 | Body | −0.214 | .007 |

Table 2 Caption: The top 10 DMLs between HTPR and non-HTPR patients with clopidogrel therapy.

### Table 3.

| CpG ID | Non-HTPR (n = 105) | HTPR (n = 193) | P value |
| --- | --- | --- | --- |
| ANKDD1B_cg02573091 | 0.61 ± 0.22 | 0.57 ± 0.22 | .102 |
| ASAP1_cg26038514 | 0.53 ± 0.36 | 0.49 ± 0.36 | .386 |
| CD80_cg06300880 | 0.70 ± 0.24 | 0.61 ± 0.31 | .005a |
| DBH-AS1_cg14007688 | 0.53 ± 0.28 | 0.51 ± 0.3 | .564 |
| VAV2_cg13564529 | 0.53 ± 0.33 | 0.52 ± 0.32 | .780 |
| FMN2_cg11251367 | 0.37 ± 0.31 | 0.31 ± 0.32 | .195 |
| GCK_cg03812172 | 0.72 ± 0.27 | 0.76 ± 0.23 | .197 |
| MAN2A1_cg00258945 | 0.09 ± 0.18 | 0.07 ± 0.18 | .435 |
| MAN2A1_cg13739190 | 0.08 ± 0.17 | 0.08 ± 0.18 | .842 |
| P2RX7_cg10122899 | 0.52 ± 0.32 | 0.47 ± 0.33 | .183 |

Table 3 Caption: Comparison DNA methylation levels of 10 DMLs between patients with non-HTPR and HTPR in the validation stage.

### 3.4. CpG-SNP association analysis in the validation population

The *CD80_*cg06300880 site contains an SNP rs34394661 (G>A), the so-called CpG-SNP ([Figure 2](#fig2)A). When the SNP was genotyped by targeted methylation sequencing, a significant decrease in *CD80_*cg06300880 methylation level was observed with an increase in the number of rs34394661 A allele (*P* < .0001). The results followed a pattern of gene-dose effect ([Figures 2](#fig2)B, 2C; [Supplementary Fig. 9](#appsec1)) in both patients with overall ACS and ACS subgroups. We further analyzed the influence of the rs34394661 (G>A) polymorphism on the clopidogrel response. The genotype distribution of rs34394661 polymorphism was in agreement with Hardy–Weinberg equilibrium (*P* > 0.05). When unconditional logistic regression was performed, the rs34394661 AA genotype was associated with significantly higher odds of HTPR as compared with carriers of rs34394661 G allele during clopidogrel therapy in 298 validation samples (recessive model, c2 = 9.47 *P* = .002, [Supplementary Table 6](#appsec1)), and 245 validation population with NSTE-ACS (c2 = 9.70 *P* = .002, [Supplementary Table 7](#appsec1)). GTEX is a public database available for analyzing the effect of SNPs on the mRNA expression of surrounding genes and identifying expression quantitative trait loci (e-QTL) in tissues. When the GTEX database was analyzed, CD80 mRNA expression was increased with the number of rs34394661 A alleles in multiple tissues (*P* < .001) ([Supplementary Fig. 10](#appsec1)).

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3295/10031538/f1a7d5c879c9/gr2.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=10031538_gr2.jpg)

Comparison of CD80_cg06300880 methylation levels between genotypes of the rs34394661 G>A polymorphism. (A) Position of CpG-SNP rs34394661 in the CD80 promoter; (B) rs34394661 A allele decreased CD80_cg06300880 methylation level in all samples; (C) rs34394661 A allele decreased CD80_cg06300880 methylation level in the LD samples. The scatter dot plot is presented as mean ± SD. Significant differences are determined by P < .05, 1-way analysis of variance

### 3.5. Construction of the clopidogrel response prediction model

As *CD80_*cg06300880 methylation level, *CD80_*rs34394661 G>A polymorphism, and *CYP2C19* genotypes were associated with the odds of HTPR in our study, we performed univariate regression analysis. Results showed that an elevated *CD80_*cg06300880 methylation level was associated with a lower odds of HTPR in all samples (odds ratio, OR = 0.31, 95% CI: 0.13-0.75, *P* = .009, [Table 4](#tbl4)) and the NSTE-ACS subgroup of patients (OR = 0.25, 95% CI: 0.092-0.67, *P* = .006, [Supplementary Table 8](#appsec1)). In contrast, the *CD80_*rs34394661 AA genotype (all samples: OR = 7.31, 95% CI: 1.69-31.59, *P* = .008; NSTE-ACS subgroup: OR = 12.69, 95% CI: 1.68-96.08, *P* = .014) and *CYP2C19* genotypes (all samples: intermediate metabolizers, OR = 2.96, 95% CI: 1.77-4.96, *P* = 3.690E-5; poor metabolizers, OR = 10.47, 95% CI: 3.04-36.01, *P* = 1.944E-4; NSTE-ACS subgroup: intermediate metabolizers, OR = 3.66, 95% CI: 2.05-6.52, *P* = 1.095E-5; poor metabolizers, OR = 8.60, 95% CI: 2.43-30.44, *P* = 8.391E-4) were associated with higher odds of HTPR ([Table 4](#tbl4), [Supplementary Table 8](#appsec1)). To avoid interactions between rs34394661 polymorphism and *CD80_*cg06300880, forward LR for regression analysis was adopted. After adjustment for *CYP2C19* genotypes (all samples: intermediate metabolizers, OR = 2.95, 95% CI: 1.75-4.98, *P* = 5.202E-5; poor metabolizers, OR = 10.37, 95% CI: 2.99-35.92, *P* = 2.263E-4, [Table 4](#tbl4); NSTE-ACS subgroup: intermediate metabolizers, OR = 3.92, 95% CI: 2.16-7.12, *P* = 7.325E-6; poor metabolizers, OR = 10.30, 95% CI: 2.85-37.22, *P* = 3.773E-4, [Supplementary Table 8](#appsec1)), *CD80_*rs34394661 polymorphism was still a significant predictor of clopidogrel resistance (OR = 7.18, 95% CI = 1.62-31.85, *P* = .009, [Table 4](#tbl4)). The relationship between *CD80_*cg06300880 methylation level and HTPR disappeared when adjusted by the rs34394661 genotype. However, this genotype–phenotype association disappeared in the NSTE subgroup. The *CD80_*cg06300880 methylation remained significant in this subgroup of patients (OR = 0.18, 95% CI = 0.061-0.53, *P* = .002, [Supplementary Table 8](#appsec1)). The results indicated that *CD80_*rs34394661 may be an independent predictor of HTPR during clopidogrel therapy in ACS, regardless of the STE and NSTE subtypes. However, *CD80_*cg06300880 methylation was only predictive in patients with NSTE.

### Table 4.

| Variables |   | Crude OR (95% CI) | P value | Adjusted OR (95% CI) | P valuea |
| --- | --- | --- | --- | --- | --- |
| CYP2C19 genotypes | Normal metabolizer | 1.00 (ref) |   | 1.00 (ref) |   |
|   | Intermediate metabolizer | 2.96 (1.77-4.96) | 3.690E-5b | 2.95 (1.75-4.98) | 5.202E-5b |
|   | Poor metabolizer | 10.47 (3.04-36.01) | 1.944E-4b | 10.37 (2.99-35.92) | 2.263E-4b |
| CD80_cg06300880 | Methylation | 0.31 (0.13-0.75) | .009b | NA | NA |
| Rs34394661 genotypes | GG+GA | 1.00 (ref) |   | 1.00 (ref) |   |
|   | AA | 7.31 (1.69-31.59) | .008b | 7.18 (1.62-31.85) | .009∗ |

Table 4 Caption: Regression analysis for HTPR after clopidogrel in the validation cohort.

## 4. Discussion

In our study, using Illumina EPIC (850k) arrays, 61 DMLs were identified between HTPR and non-HTPR. Among these loci, lower methylation of *CD80_*cg06300880 in HTPR was observed both in discovery and validation stages. For rs34394661, a CpG-SNP of the *CD80_*cg06300880 locus, the greater the number of A alleles a patient carried, the higher odds of HTPR. Binary logistic regression identified rs34394661 polymorphism and *CD80_*cg06300880 methylation as independent predictors for clopidogrel response in all patients and patients with NSTE-ACS, respectively.

Illumina methylation BeadChip arrays cover CpG islands, shore (within 2 kb from *CpG* islands), shelf (within 4 kb from *CpG* islands), and open sea (4 kb away from island) regions. [[41](#bib41),[42](#bib42)] Most DMLs identified in our study were located in chromosome open sea or intergenic regions. Although the density of methylation sites is lower in open sea, a negative correlation between methylation levels of CpG sites in open sea and shore regions and gene expression is reported such as in the islet of patients with type 2 diabetes. [[41](#bib41)] When comparing DNA methylation patterns between children aged 0 to 5 and 5 to 10 years, methylation loss with age was mainly observed in non-CpG islands such as the open sea region (mainly located in introns and intergenic regions) In contrast, a gain of methylation with age mainly occurred in the CpG islands and gene promoters, areas with a high density of CpG loci. [[43](#bib43),[44](#bib44)] This suggests that CpG sites in the open sea region tend to lose methylation with an increase in age. DNA methylation and its aberrance have been extensively studied in many diseases, especially cancers, with hypermethylation predominantly occurring in CpG islands. In the meantime, hypomethylation in offshore regions is more common, especially for the open sea region. [[[45]](#bib45), [[46]](#bib46), [[47]](#bib47)] Intergenic regions can also contribute to the regulation of gene expression. DNA methylation of intergenic sequences may affect gene expression and/or chromosome compaction during cell division and meiosis. [[48](#bib48)] When comparing genomic methylation data between ependymoma and normal tissues, CpG sites in intergenic regions are more prone to demethylatation. [[49](#bib49)] In addition, there are also overlaps in gene regulatory elements that are usually located far from the genes they regulate, such as enhancers in intergenic regions. [[50](#bib50)] Overlapping regions can undergo tissue-specific differential methylation in evolutionally conserved nonprotein coding sequences ∼100 KB away from the gene body. These are involved in potential long-range regulatory elements such as superenhancers instead of relying on transcriptional regulation by promoter methylation alone. [[51](#bib51),[52](#bib52)] Positive correlations between DNA methylation in intergenic regions and gene expression have also been reported. [[53](#bib53)]

The top 10 DMLs were located in 9 genes, including *ANKDD1B, ASAP1, CD80, DBH-AS1, VAV2, FMN2, P2RX7, OR52M1,* and *MAN2A1*. Yang et al. [[54](#bib54)] screened differentially methylated genes in 6 patients with clopidogrel resistance and 6 patients with nonresistance. No shared genes were observed among the top genes from these studies. In our study, 16 patients with HTPR and 16 patients with non-HTPR were enrolled for 850K chip screening. All the discovery samples were well matched for *CYP2C19* genotypes, one of the critical and definite determinants of clopidogrel resistance in Chinese. [[55](#bib55)] We assume that the difference in sample size and the matching of *CYP2C19* genotypes may provide a possible explanation for the different findings between studies. The method used for determination of platelet function is also important. VASP-PRI involved in our research is a relatively unique indicator of P2Y12 receptor blockade and a preferred determination of clopidogrel response. [[56](#bib56),[57](#bib57)] At the same time, other platelet function tests, such as VerifyNow detected P2Y12 reaction unit (PRU), are based on platelet aggregation. Yet, VerifyNow-PRU is influenced by factors such as GPIIb/IIIa receptor antagonists [[58](#bib58)], platelet count [[59](#bib59)], and hematocrit level. [[60](#bib60)] Another genome-wide DNA methylation study related to clopidogrel efficacy was carried out in patients with ischemic stroke. [[61](#bib61)] By analyzing 48K DNA methylation sites in 21 pairs of patients with and without a recurrent vascular event during the 1-year follow-up with clopidogrel therapy, the lower *TRAF3_*cg03548645 methylation level was correlated with increased platelet aggregation. [[61](#bib61)] However, similar findings were not observed in our 850K chip study. This may be explained by different endpoints or phenotypes selected in our studies, and the recurrent vascular event is more complicated than clopidogrel resistance.

Only the *CD80_*cg06300880 site was replicated in the validation samples. Patients with HTPR showing significantly lower *CD80_*cg06300880 methylation levels in promoter methylation are observed to inhibit transcription and mRNA expression of the host genes by preventing binding of transcription factors. [[62](#bib62),[63](#bib63)] *CD80_*cg06300880 is located within the TSS1500 region in CD80 promoter. Although its role in regulating CD80 expression remains unclear, evidence shows that promoter’s hypermethylation may result in reduced CD80 expression. [[64](#bib64)]

Interestingly, CpG-SNP rs34394661 polymorphism occurs at the *CD80_*cg06300880 site (methylation on the complementary chain, changes from CG to CA), and the methylation site disappeared (changed from CG to CA) by the rs34394661 A allele. We observed that the rs34394661 AA genotype was associated with increased odds of HTPR. CpG-SNPs play an important role in linking genetic variations with epigenome modification. [[65](#bib65)] For example, *KCNQ1* DNA methylation has been reported to correlate with insulin sensitivity, serum adiponectin level, and CpG-SNP rs231840 (G>T) functions by ablating a methylation site and thus reduces the promoter methylation level. [[65](#bib65)] Besides, multiple shared CpG-SNPs of type 2 diabetes risk and birth weight are methylation QTL (me-QTL), metabolic QTL (meta-QTL), or e-QTL. [[66](#bib66)] Our GTEX analysis with the public database showed that *CD80* mRNA expression was increased with the number of rs34394661A allele in 9 tissues, which supports the possible tissue-wide influence of the SNP on both *CD80_*cg06300880 methylation and CD80 transcriptional activity.

Although e-QTL analysis indicated increased expression of *CD80* mRNA with the rs34394661 A allele, it is not known how the decrease in *CD80* DNA methylation/rs34394661 AA genotype decreases the clopidogrel response. CD80 is a membrane protein that plays a critical role in T-cell activation. [[67](#bib67)] Upregulation of CD80 expression on the macrophage surface is accompanied by activation of PI3K/AKT-MTORC1 signaling cascade. [[68](#bib68)] Activation of PI3K-AKT cascade after stimulation of P2Y12 receptor in platelet activation is also evident. [[[69]](#bib69), [[70]](#bib70)] This indicates that CD80 may act by regulating PI3K/AKT signaling in platelet activation and aggregation. More studies are required to confirm this assumption.

*CYP2C19* loss-of-function allele is a known variant that partially explains HTPR during clopidogrel therapy. The *CD80_*rs34394661 AA genotype was also associated with clopidogrel response in multivariate analysis. *CYP2C19* and *CD80_*rs34394661 are shown to predict HTPR in our overall samples regardless of ACS subtypes. In addition, our studies suggest that *CD80_*cg06300880 methylation status could predict HTPR only in the NSTE-ACS subgroup of patients. This may provide a higher prediction accuracy for HTPR with clopidogrel treatment. Although the alternatively used P2Y12 inhibitor ticagrelor can effectively avoid antiplatelet inhibition by drug genetic polymorphisms [[71](#bib71)], it results in higher overall incidence of bleeding events. Other adverse drug reactions such as dyspnea and arrhythmia are common for ticagrelor in East Asians. [[72](#bib72),[73](#bib73)] Therefore, clopidogrel remains the primary antiplatelet reagent currently used in clinics. New potential biomarker factors (cg06300880 methylation or rs34394661 polymorphism) for clopidogrel resistance may provide an additional basis for the individualized use of clopidogrel to improve efficacy. Of course, our findings need further validation in the future.

Our study has the following limitations. First, DMLs failed to pass Bonferroni correction in the discovery stage, which might lead to a high false positive rate. Second, not all DMLs were included in validation analysis, suggesting that the potential effects of some DMLs might be missing. Third, our prediction model could apply to endpoints (ischemic events or bleeding events), if the guidelines’ cutoff values of HTPR and LTPR were strictly followed. [[74](#bib74)] Of course, this remains speculative, and long-term follow-up is necessary. Fourth, the sample size, particularly the discovery cohort, was relatively small. The single-centered effects could not be ignored. Fifth, except for *CYP2C19*∗2 and *CYP2C19*∗3, other reported SNPs significantly associated with clopidogrel responsiveness were not included in the study. Finally, high cost and complex operation of methylation detection remain limitations to generalizing our findings.

## 5. Conclusions

New DMLs that may lead to clopidogrel resistance were investigated. A lower mean *CD80_*cg06300880 methylation level was observed in patients with HTPR. The AA genotype of CpG-SNP rs34394661 at *CD80_*cg06300880 site was associated with increased odds of HTPR, possibly by decreasing the *CD80_*cg06300880 methylation level. Models incorporating *CYP2C19* and *CD80_*rs34394661 genotypes, or *CD80_*cg06300880 methylation are supposed to optimize decision-making in clopidogrel use.

## Acknowlegments

The authors thank every contributor for their help in this research work**.**

### Funding

The work was supported by the National Key R&D program of China (No. 2017YFC0909302), the National Natural Science Foundation of China of China (No. 81874328), China Postdoctoral Science Foundation (2020M670106ZX), the Hunan Provincial Innovation Foundation for Postgraduate (CX20190154), and the Fundamental Research Funds for the Central Universities of Central South University (2019zzts098).

### Ethics statement

The study protocol was approved by the Ethics Committee of the Institute of Clinical Pharmacology, Central South University (approval number CTXY-140002-13). The ethical procedures followed the Declaration of Helsinki. The study was registered with the Chinese Clinical Trial Registry ([www.chictr.org.cn](http://www.chictr.org.cn)), number. ChiCTR-OPN-15006260.

### Author contributions

Conceptualization, P.S., M.L., and X.C.; Methodology, P.S.; Investigation, P.S. and M.L.; Writing-Original Draft, P.S.; Writing-Review & Editing, L.P. and X.C.; Funding Acquisition, X.C.; Resources, P.S., M.L., and L.P.; Supervision, X.C. All authors read and approved the final version of the paper.

### Relationship Disclosure

There are no competing interests to disclose.

### Informed patient consent

All patients signed informed consent.

### Data availability

All data relevant to this study are included in the article or provided as [supplemental materials](#appsec1) online. The data that support the findings of this study are available from the corresponding author upon reasonable request.

## Footnotes

## Contributor Information

Li-Ming Peng, Email: limingpeng@csu.edu.cn.

Xiao-Ping Chen, Email: chenxiaoping@csu.edu.cn.

## Supporting Information

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

All data relevant to this study are included in the article or provided as [supplemental materials](#appsec1) online. The data that support the findings of this study are available from the corresponding author upon reasonable request.

### Supplementary Materials

### Data Availability Statement

All data relevant to this study are included in the article or provided as [supplemental materials](#appsec1) online. The data that support the findings of this study are available from the corresponding author upon reasonable request.

## References

1. Bhatt D.L. Percutaneous coronary intervention in 2018. JAMA. 2018;319:2127–2128. doi: 10.1001/jama.2018.5281.  [DOI](https://doi.org/10.1001/jama.2018.5281) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29800163/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA&title=Percutaneous%20coronary%20intervention%20in%202018&author=D.L.%20Bhatt&volume=319&publication_year=2018&pages=2127-2128&pmid=29800163&doi=10.1001/jama.2018.5281&)

2. Kereiakes D.J., Gurbel P.A. Peri-procedural platelet function and platelet inhibition in percutaneous coronary intervention. JACC Cardiovasc Interv. 2008;1:111–121. doi: 10.1016/j.jcin.2008.01.005.  [DOI](https://doi.org/10.1016/j.jcin.2008.01.005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19463287/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JACC%20Cardiovasc%20Interv&title=Peri-procedural%20platelet%20function%20and%20platelet%20inhibition%20in%20percutaneous%20coronary%20intervention&author=D.J.%20Kereiakes&author=P.A.%20Gurbel&volume=1&publication_year=2008&pages=111-121&pmid=19463287&doi=10.1016/j.jcin.2008.01.005&)

3. Glaser R., Selzer F., Faxon D.P., Warren K.L., Howard A.C., James S., et al. Clinical progression of incidental, asymptomatic lesions discovered during culprit vessel coronary intervention. Circulation. 2005;111:143–149. doi: 10.1161/01.CIR.0000150335.01285.12.  [DOI](https://doi.org/10.1161/01.CIR.0000150335.01285.12) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15623544/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation&title=Clinical%20progression%20of%20incidental,%20asymptomatic%20lesions%20discovered%20during%20culprit%20vessel%20coronary%20intervention&author=R.%20Glaser&author=F.%20Selzer&author=D.P.%20Faxon&author=K.L.%20Warren&author=A.C.%20Howard&volume=111&publication_year=2005&pages=143-149&pmid=15623544&doi=10.1161/01.CIR.0000150335.01285.12&)

4. Mangiacapra F., Del B.M., Abbate A., Tommaso G., Emanuele B., Rocco A.M., et al. Role of endothelial dysfunction in determining angina after percutaneous coronary intervention: learning from pathophysiology to optimize treatment. Prog Cardiovasc Dis. 2020;63:233–242. doi: 10.1016/j.pcad.2020.02.009.  [DOI](https://doi.org/10.1016/j.pcad.2020.02.009) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32061633/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Prog%20Cardiovasc%20Dis&title=Role%20of%20endothelial%20dysfunction%20in%20determining%20angina%20after%20percutaneous%20coronary%20intervention:%20learning%20from%20pathophysiology%20to%20optimize%20treatment&author=F.%20Mangiacapra&author=B.M.%20Del&author=A.%20Abbate&author=G.%20Tommaso&author=B.%20Emanuele&volume=63&publication_year=2020&pages=233-242&pmid=32061633&doi=10.1016/j.pcad.2020.02.009&)

5. Shlofmitz E., Iantorno M., Waksman R. Restenosis of drug-eluting stents: a new classification system based on disease mechanism to guide treatment and state-of-the-art review. Circ Cardiovasc Interv. 2019;12 doi: 10.1161/CIRCINTERVENTIONS.118.007023.  [DOI](https://doi.org/10.1161/CIRCINTERVENTIONS.118.007023) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31345066/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circ%20Cardiovasc%20Interv&title=Restenosis%20of%20drug-eluting%20stents:%20a%20new%20classification%20system%20based%20on%20disease%20mechanism%20to%20guide%20treatment%20and%20state-of-the-art%20review&author=E.%20Shlofmitz&author=M.%20Iantorno&author=R.%20Waksman&volume=12&publication_year=2019&pmid=31345066&doi=10.1161/CIRCINTERVENTIONS.118.007023&)

6. Ibanez B., James S., Agewall S., Antunes M.J., Bucciarelli-Ducci C., Bueno H., et al. 2017 ESC Guidelines for the management of acute myocardial infarction in patients presenting with ST-segment elevation: the Task Force for the management of acute myocardial infarction in patients presenting with ST-segment elevation of the European Society of Cardiology (ESC) Eur Heart J. 2018;39:119–177. doi: 10.1093/eurheartj/ehx393.  [DOI](https://doi.org/10.1093/eurheartj/ehx393) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28886621/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20Heart%20J&title=2017%20ESC%20Guidelines%20for%20the%20management%20of%20acute%20myocardial%20infarction%20in%20patients%20presenting%20with%20ST-segment%20elevation:%20the%20Task%20Force%20for%20the%20management%20of%20acute%20myocardial%20infarction%20in%20patients%20presenting%20with%20ST-segment%20elevation%20of%20the%20European%20Society%20of%20Cardiology%20(ESC)&author=B.%20Ibanez&author=S.%20James&author=S.%20Agewall&author=M.J.%20Antunes&author=C.%20Bucciarelli-Ducci&volume=39&publication_year=2018&pages=119-177&pmid=28886621&doi=10.1093/eurheartj/ehx393&)

7. Kamran H., Jneid H., Kayani W.T., Virani S.S., Levine G.N., Nambi V., Khalid U. Oral antiplatelet therapy after acute coronary syndrome: a review. JAMA. 2021;325:1545–1555. doi: 10.1001/jama.2021.0716.  [DOI](https://doi.org/10.1001/jama.2021.0716) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33877270/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA&title=Oral%20antiplatelet%20therapy%20after%20acute%20coronary%20syndrome:%20a%20review&author=H.%20Kamran&author=H.%20Jneid&author=W.T.%20Kayani&author=S.S.%20Virani&author=G.N.%20Levine&volume=325&publication_year=2021&pages=1545-1555&pmid=33877270&doi=10.1001/jama.2021.0716&)

8. Matetzky S., Shenkman B., Guetta V., Shechter M., Beinart R., Goldenberg I., et al. Clopidogrel resistance is associated with increased risk of recurrent atherothrombotic events in patients with acute myocardial infarction. Circulation. 2004;109:3171–3175. doi: 10.1161/01.CIR.0000130846.46168.03.  [DOI](https://doi.org/10.1161/01.CIR.0000130846.46168.03) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15184279/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation&title=Clopidogrel%20resistance%20is%20associated%20with%20increased%20risk%20of%20recurrent%20atherothrombotic%20events%20in%20patients%20with%20acute%20myocardial%20infarction&author=S.%20Matetzky&author=B.%20Shenkman&author=V.%20Guetta&author=M.%20Shechter&author=R.%20Beinart&volume=109&publication_year=2004&pages=3171-3175&pmid=15184279&doi=10.1161/01.CIR.0000130846.46168.03&)

9. Bliden K.P., DiChiara J., Tantry U.S., Bassi A.K., Chaganti S.K., Gurbel P.A. Increased risk in patients with high platelet aggregation receiving chronic clopidogrel therapy undergoing percutaneous coronary intervention: is the current antiplatelet therapy adequate? J Am Coll Cardiol. 2007;49:657–666. doi: 10.1016/j.jacc.2006.10.050.  [DOI](https://doi.org/10.1016/j.jacc.2006.10.050) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17291930/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%C2%A0Am%20Coll%20Cardiol&title=Increased%20risk%20in%20patients%20with%20high%20platelet%20aggregation%20receiving%20chronic%20clopidogrel%20therapy%20undergoing%20percutaneous%20coronary%20intervention:%20is%20the%20current%20antiplatelet%20therapy%20adequate?&author=K.P.%20Bliden&author=J.%20DiChiara&author=U.S.%20Tantry&author=A.K.%20Bassi&author=S.K.%20Chaganti&volume=49&publication_year=2007&pages=657-666&pmid=17291930&doi=10.1016/j.jacc.2006.10.050&)

10. Wisniewski A., Filipska K. The phenomenon of clopidogrel high on-treatment platelet reactivity in ischemic stroke subjects: a comprehensive review. Int J Mol Sci. 2020;21:6408. doi: 10.3390/ijms21176408.  [DOI](https://doi.org/10.3390/ijms21176408) | [PMC free article](/articles/PMC7503235/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32899176/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Mol%20Sci&title=The%20phenomenon%20of%20clopidogrel%20high%20on-treatment%20platelet%20reactivity%20in%20ischemic%20stroke%20subjects:%20a%20comprehensive%20review&author=A.%20Wisniewski&author=K.%20Filipska&volume=21&publication_year=2020&pages=6408&pmid=32899176&doi=10.3390/ijms21176408&)

11. Pan Y., Chen W., Xu Y., Yi X., Han Y., Yang Q., et al. Genetic polymorphisms and clopidogrel efficacy for acute ischemic stroke or transient ischemic attack: a systematic review and meta-analysis. Circulation. 2017;135:21–33. doi: 10.1161/CIRCULATIONAHA.116.024913.  [DOI](https://doi.org/10.1161/CIRCULATIONAHA.116.024913) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27806998/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation&title=Genetic%20polymorphisms%20and%20clopidogrel%20efficacy%20for%20acute%20ischemic%20stroke%20or%20transient%20ischemic%20attack:%20a%20systematic%20review%20and%20meta-analysis&author=Y.%20Pan&author=W.%20Chen&author=Y.%20Xu&author=X.%20Yi&author=Y.%20Han&volume=135&publication_year=2017&pages=21-33&pmid=27806998&doi=10.1161/CIRCULATIONAHA.116.024913&)

12. Scott S.A., Sangkuhl K., Gardner E.E., Stein C.M., Hulot J.S., Johnson J.A., et al. Clinical Pharmacogenetics Implementation Consortium guidelines for cytochrome P450-2C19 (CYP2C19) genotype and clopidogrel therapy. Clin Pharmacol Ther. 2011;90:328–332. doi: 10.1038/clpt.2011.132.  [DOI](https://doi.org/10.1038/clpt.2011.132) | [PMC free article](/articles/PMC3234301/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21716271/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Clinical%20Pharmacogenetics%20Implementation%20Consortium%20guidelines%20for%20cytochrome%20P450-2C19%20(CYP2C19)%20genotype%20and%20clopidogrel%20therapy&author=S.A.%20Scott&author=K.%20Sangkuhl&author=E.E.%20Gardner&author=C.M.%20Stein&author=J.S.%20Hulot&volume=90&publication_year=2011&pages=328-332&pmid=21716271&doi=10.1038/clpt.2011.132&)

13. Scott S.A., Sangkuhl K., Stein C.M., Hulot J.S., Mega J.L., Roden D.M., et al. Clinical Pharmacogenetics Implementation Consortium guidelines for CYP2C19 genotype and clopidogrel therapy: 2013 update. Clin Pharmacol Ther. 2013;94:317–323. doi: 10.1038/clpt.2013.105.  [DOI](https://doi.org/10.1038/clpt.2013.105) | [PMC free article](/articles/PMC3748366/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23698643/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Clinical%20Pharmacogenetics%20Implementation%20Consortium%20guidelines%20for%20CYP2C19%20genotype%20and%20clopidogrel%20therapy:%202013%20update&author=S.A.%20Scott&author=K.%20Sangkuhl&author=C.M.%20Stein&author=J.S.%20Hulot&author=J.L.%20Mega&volume=94&publication_year=2013&pages=317-323&pmid=23698643&doi=10.1038/clpt.2013.105&)

14. Liu Y.L., Hu X.L., Song P.Y., Li H., Li M.P., Du Y.X., et al. Influence of GAS5/MicroRNA-223-3p/P2Y12 axis on clopidogrel response in coronary artery disease. J Am Heart Assoc. 2021;10 doi: 10.1161/JAHA.121.021129.  [DOI](https://doi.org/10.1161/JAHA.121.021129) | [PMC free article](/articles/PMC8751826/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34713722/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%C2%A0Am%20Heart%20Assoc&title=Influence%20of%20GAS5/MicroRNA-223-3p/P2Y12%20axis%20on%20clopidogrel%20response%20in%20coronary%20artery%20disease&author=Y.L.%20Liu&author=X.L.%20Hu&author=P.Y.%20Song&author=H.%20Li&author=M.P.%20Li&volume=10&publication_year=2021&pmid=34713722&doi=10.1161/JAHA.121.021129&)

15. Zhu K.X., Song P.Y., He-Li, Du Y.X., Ma Q.L., Peng L.M., et al. Association of FMO3 rs1736557 polymorphism with clopidogrel response in Chinese patients with coronary artery disease. Eur J Clin Pharmacol. 2021;77:359–368. doi: 10.1007/s00228-020-03024-6.  [DOI](https://doi.org/10.1007/s00228-020-03024-6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33089397/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Clin%20Pharmacol&title=Association%20of%20FMO3%20rs1736557%20polymorphism%20with%20clopidogrel%20response%20in%20Chinese%20patients%20with%20coronary%20artery%20disease&author=K.X.%20Zhu&author=P.Y.%20Song&author=%20He-Li&author=Y.X.%20Du&author=Q.L.%20Ma&volume=77&publication_year=2021&pages=359-368&pmid=33089397&doi=10.1007/s00228-020-03024-6&)

16. Li H., Zhang Y.J., Li M.P., Hu X.L., Song P.Y., Peng L.M., et al. Association of N6AMT1 rs2254638 polymorphism with clopidogrel response in Chinese patients with coronary artery disease. Front Pharmacol. 2018;9:1039. doi: 10.3389/fphar.2018.01039.  [DOI](https://doi.org/10.3389/fphar.2018.01039) | [PMC free article](/articles/PMC6156268/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30283338/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front%20Pharmacol&title=Association%20of%20N6AMT1%20rs2254638%20polymorphism%20with%20clopidogrel%20response%20in%20Chinese%20patients%20with%20coronary%20artery%20disease&author=H.%20Li&author=Y.J.%20Zhang&author=M.P.%20Li&author=X.L.%20Hu&author=P.Y.%20Song&volume=9&publication_year=2018&pages=1039&pmid=30283338&doi=10.3389/fphar.2018.01039&)

17. Wang J.Y., Zhang Y.J., Li H., Hu X.L., Li M.P., Song P.Y., et al. CRISPLD1 rs12115090 polymorphisms alters antiplatelet potency of clopidogrel in coronary artery disease patients in Chinese Han. Gene. 2018;678:226–232. doi: 10.1016/j.gene.2018.08.027.  [DOI](https://doi.org/10.1016/j.gene.2018.08.027) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30096456/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gene&title=CRISPLD1%20rs12115090%20polymorphisms%20alters%20antiplatelet%20potency%20of%20clopidogrel%20in%20coronary%20artery%20disease%20patients%20in%20Chinese%20Han&author=J.Y.%20Wang&author=Y.J.%20Zhang&author=H.%20Li&author=X.L.%20Hu&author=M.P.%20Li&volume=678&publication_year=2018&pages=226-232&pmid=30096456&doi=10.1016/j.gene.2018.08.027&)

18. Kim H.S., Chang K., Koh Y.S., Park M.W., Choi Y.S., Park C.S., et al. CYP2C19 poor metabolizer is associated with clinical outcome of clopidogrel therapy in acute myocardial infarction but not stable angina. Circ Cardiovasc Genet. 2013;6:514–521. doi: 10.1161/CIRCGENETICS.113.000109.  [DOI](https://doi.org/10.1161/CIRCGENETICS.113.000109) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24019397/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circ%20Cardiovasc%20Genet&title=CYP2C19%20poor%20metabolizer%20is%20associated%20with%20clinical%20outcome%20of%20clopidogrel%20therapy%20in%20acute%20myocardial%20infarction%20but%20not%20stable%20angina&author=H.S.%20Kim&author=K.%20Chang&author=Y.S.%20Koh&author=M.W.%20Park&author=Y.S.%20Choi&volume=6&publication_year=2013&pages=514-521&pmid=24019397&doi=10.1161/CIRCGENETICS.113.000109&)

19. Horenstein R.B., Madabushi R., Zineh I., Yerges-Armstrong L.M., Peer C.J., Schuck R.N., et al. Effectiveness of clopidogrel dose escalation to normalize active metabolite exposure and antiplatelet effects in CYP2C19 poor metabolizers. J Clin Pharmacol. 2014;54:865–873. doi: 10.1002/jcph.293.  [DOI](https://doi.org/10.1002/jcph.293) | [PMC free article](/articles/PMC4113831/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24710841/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%C2%A0Clin%20Pharmacol&title=Effectiveness%20of%20clopidogrel%20dose%20escalation%20to%20normalize%20active%20metabolite%20exposure%20and%20antiplatelet%20effects%20in%20CYP2C19%20poor%20metabolizers&author=R.B.%20Horenstein&author=R.%20Madabushi&author=I.%20Zineh&author=L.M.%20Yerges-Armstrong&author=C.J.%20Peer&volume=54&publication_year=2014&pages=865-873&pmid=24710841&doi=10.1002/jcph.293&)

20. Collet J.P., Hulot J.S., Cuisset T., Rangé G., Cayla G., Van Belle E., et al. Genetic and platelet function testing of antiplatelet therapy for percutaneous coronary intervention: the ARCTIC-GENE study. Eur J Clin Pharmacol. 2015;71:1315–1324. doi: 10.1007/s00228-015-1917-9.  [DOI](https://doi.org/10.1007/s00228-015-1917-9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26265231/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Clin%20Pharmacol&title=Genetic%20and%20platelet%20function%20testing%20of%20antiplatelet%20therapy%20for%20percutaneous%20coronary%20intervention:%20the%20ARCTIC-GENE%20study&author=J.P.%20Collet&author=J.S.%20Hulot&author=T.%20Cuisset&author=G.%20Rang%C3%A9&author=G.%20Cayla&volume=71&publication_year=2015&pages=1315-1324&pmid=26265231&doi=10.1007/s00228-015-1917-9&)

21. Shuldiner A.R., O'Connell J.R., Bliden K.P., Gandhi A., Ryan K., Horenstein R.B., et al. Association of cytochrome P450 2C19 genotype with the antiplatelet effect and clinical efficacy of clopidogrel therapy. JAMA. 2009;302:849–857. doi: 10.1001/jama.2009.1232.  [DOI](https://doi.org/10.1001/jama.2009.1232) | [PMC free article](/articles/PMC3641569/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19706858/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA&title=Association%20of%20cytochrome%20P450%202C19%20genotype%20with%20the%20antiplatelet%20effect%20and%20clinical%20efficacy%20of%20clopidogrel%20therapy&author=A.R.%20Shuldiner&author=J.R.%20O'Connell&author=K.P.%20Bliden&author=A.%20Gandhi&author=K.%20Ryan&volume=302&publication_year=2009&pages=849-857&pmid=19706858&doi=10.1001/jama.2009.1232&)

22. Rizzacasa B., Amati F., Romeo F., Novelli G., Mehta J.L. Epigenetic modification in coronary atherosclerosis: JACC review topic of the week. J Am Coll Cardiol. 2019;74:1352–1365. doi: 10.1016/j.jacc.2019.07.043.  [DOI](https://doi.org/10.1016/j.jacc.2019.07.043) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31488273/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%C2%A0Am%20Coll%20Cardiol&title=Epigenetic%20modification%20in%20coronary%20atherosclerosis:%20JACC%20review%20topic%20of%20the%20week&author=B.%20Rizzacasa&author=F.%20Amati&author=F.%20Romeo&author=G.%20Novelli&author=J.L.%20Mehta&volume=74&publication_year=2019&pages=1352-1365&pmid=31488273&doi=10.1016/j.jacc.2019.07.043&)

23. Evans L.W., Stratton M.S., Ferguson B.S. Dietary natural products as epigenetic modifiers in aging-associated inflammation and disease. Nat Prod Rep. 2020;37:653–676. doi: 10.1039/c9np00057g.  [DOI](https://doi.org/10.1039/c9np00057g) | [PMC free article](/articles/PMC7577396/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31993614/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Prod%20Rep&title=Dietary%20natural%20products%20as%20epigenetic%20modifiers%20in%20aging-associated%20inflammation%20and%20disease&author=L.W.%20Evans&author=M.S.%20Stratton&author=B.S.%20Ferguson&volume=37&publication_year=2020&pages=653-676&pmid=31993614&doi=10.1039/c9np00057g&)

24. Kalachand R.D., Stordal B., Madden S., Chandler B., Cunningham J., Goode E.L., et al. BRCA1 promoter methylation and clinical outcomes in ovarian cancer: an individual patient data meta-analysis. J Natl Cancer Inst. 2020;112:1190–1203. doi: 10.1093/jnci/djaa070.  [DOI](https://doi.org/10.1093/jnci/djaa070) | [PMC free article](/articles/PMC7735770/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32413141/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%C2%A0Natl%20Cancer%20Inst&title=BRCA1%20promoter%20methylation%20and%20clinical%20outcomes%20in%20ovarian%20cancer:%20an%20individual%20patient%20data%20meta-analysis&author=R.D.%20Kalachand&author=B.%20Stordal&author=S.%20Madden&author=B.%20Chandler&author=J.%20Cunningham&volume=112&publication_year=2020&pages=1190-1203&pmid=32413141&doi=10.1093/jnci/djaa070&)

25. Sledzinska P., Bebyn M.G., Furtak J., Kowalewski J., Lewandowska M.A. Prognostic and predictive biomarkers in gliomas. Int J Mol Sci. 2021;22 doi: 10.3390/ijms221910373.  [DOI](https://doi.org/10.3390/ijms221910373) | [PMC free article](/articles/PMC8508830/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34638714/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Mol%20Sci&title=Prognostic%20and%20predictive%20biomarkers%20in%20gliomas&author=P.%20Sledzinska&author=M.G.%20Bebyn&author=J.%20Furtak&author=J.%20Kowalewski&author=M.A.%20Lewandowska&volume=22&publication_year=2021&pmid=34638714&doi=10.3390/ijms221910373&)

26. Strathdee G., MacKean M.J., Illand M., Brown R. A role for methylation of the hMLH1 promoter in loss of hMLH1 expression and drug resistance in ovarian cancer. Oncogene. 1999;18:2335–2341. doi: 10.1038/sj.onc.1202540.  [DOI](https://doi.org/10.1038/sj.onc.1202540) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10327053/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncogene&title=A%C2%A0role%20for%20methylation%20of%20the%20hMLH1%20promoter%20in%20loss%20of%20hMLH1%20expression%20and%20drug%20resistance%20in%20ovarian%20cancer&author=G.%20Strathdee&author=M.J.%20MacKean&author=M.%20Illand&author=R.%20Brown&volume=18&publication_year=1999&pages=2335-2341&pmid=10327053&doi=10.1038/sj.onc.1202540&)

27. Su J., Yu Q., Zhu H., Li X., Cui H., Du W., et al. The risk of clopidogrel resistance is associated with ABCB1 polymorphisms but not promoter methylation in a Chinese Han population. Plos One. 2017;12 doi: 10.1371/journal.pone.0174511.  [DOI](https://doi.org/10.1371/journal.pone.0174511) | [PMC free article](/articles/PMC5373545/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28358842/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Plos%20One&title=The%20risk%20of%20clopidogrel%20resistance%20is%20associated%20with%20ABCB1%20polymorphisms%20but%20not%20promoter%20methylation%20in%20a%20Chinese%20Han%20population&author=J.%20Su&author=Q.%20Yu&author=H.%20Zhu&author=X.%20Li&author=H.%20Cui&volume=12&publication_year=2017&pmid=28358842&doi=10.1371/journal.pone.0174511&)

28. Sukmawan R., Hoetama E., Suridanda D.S., Giantini A., Listiyaningsih E., Gilang Rejeki V., et al. Increase in the risk of clopidogrel resistance and consequent TIMI flow impairment by DNA hypomethylation of CYP2C19 gene in STEMI patients undergoing primary percutaneous coronary intervention (PPCI) Pharmacol Res Perspect. 2021;9 doi: 10.1002/prp2.738.  [DOI](https://doi.org/10.1002/prp2.738) | [PMC free article](/articles/PMC7915409/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33641235/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacol%20Res%20Perspect&title=Increase%20in%20the%20risk%20of%20clopidogrel%20resistance%20and%20consequent%20TIMI%20flow%20impairment%20by%20DNA%20hypomethylation%20of%20CYP2C19%20gene%20in%20STEMI%20patients%20undergoing%20primary%20percutaneous%20coronary%20intervention%20(PPCI)&author=R.%20Sukmawan&author=E.%20Hoetama&author=D.S.%20Suridanda&author=A.%20Giantini&author=E.%20Listiyaningsih&volume=9&publication_year=2021&pmid=33641235&doi=10.1002/prp2.738&)

29. Su J., Li J., Yu Q., Xu X., Wang J., Yang J., et al. Association of PON1 gene promoter DNA methylation with the risk of Clopidogrel resistance in patients with coronary artery disease. J Clin Lab Anal. 2019;33 doi: 10.1002/jcla.22867.  [DOI](https://doi.org/10.1002/jcla.22867) | [PMC free article](/articles/PMC6595294/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30891852/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%C2%A0Clin%20Lab%20Anal&title=Association%20of%20PON1%20gene%20promoter%20DNA%20methylation%20with%20the%20risk%20of%20Clopidogrel%20resistance%20in%20patients%20with%20coronary%20artery%20disease&author=J.%20Su&author=J.%20Li&author=Q.%20Yu&author=X.%20Xu&author=J.%20Wang&volume=33&publication_year=2019&pmid=30891852&doi=10.1002/jcla.22867&)

30. Su J., Li X., Yu Q., Liu Y., Wang Y., Song H., et al. Association of P2Y12 gene promoter DNA methylation with the risk of clopidogrel resistance in coronary artery disease patients. Biomed Res Int. 2014;2014 doi: 10.1155/2014/450814.  [DOI](https://doi.org/10.1155/2014/450814) | [PMC free article](/articles/PMC3976931/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24745016/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biomed%20Res%20Int&title=Association%20of%20P2Y12%20gene%20promoter%20DNA%20methylation%20with%20the%20risk%20of%20clopidogrel%20resistance%20in%20coronary%20artery%20disease%20patients&author=J.%20Su&author=X.%20Li&author=Q.%20Yu&author=Y.%20Liu&author=Y.%20Wang&volume=2014&publication_year=2014&pmid=24745016&doi=10.1155/2014/450814&)

31. Valina C., Neumann F.J., Menichelli M., Mayer K., Woehrle J., Bernlochner I., et al. Ticagrelor or prasugrel in patients with non-ST-segment elevation acute coronary syndromes. J Am Coll Cardiol. 2020;76:2436–2446. doi: 10.1016/j.jacc.2020.09.584.  [DOI](https://doi.org/10.1016/j.jacc.2020.09.584) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33213722/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%C2%A0Am%20Coll%20Cardiol&title=Ticagrelor%20or%20prasugrel%20in%20patients%20with%20non-ST-segment%20elevation%20acute%20coronary%20syndromes&author=C.%20Valina&author=F.J.%20Neumann&author=M.%20Menichelli&author=K.%20Mayer&author=J.%20Woehrle&volume=76&publication_year=2020&pages=2436-2446&pmid=33213722&doi=10.1016/j.jacc.2020.09.584&)

32. Bonello L., Camoin-Jau L., Arques S., Boyer C., Panagides D., Wittenberg O., et al. Adjusted clopidogrel loading doses according to vasodilator-stimulated phosphoprotein phosphorylation index decrease rate of major adverse cardiovascular events in patients with clopidogrel resistance: a multicenter randomized prospective study. J Am Coll Cardiol. 2008;51:1404–1411. doi: 10.1016/j.jacc.2007.12.044.  [DOI](https://doi.org/10.1016/j.jacc.2007.12.044) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18387444/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%C2%A0Am%20Coll%20Cardiol&title=Adjusted%20clopidogrel%20loading%20doses%20according%20to%20vasodilator-stimulated%20phosphoprotein%20phosphorylation%20index%20decrease%20rate%20of%20major%20adverse%20cardiovascular%20events%20in%20patients%20with%20clopidogrel%20resistance:%20a%20multicenter%20randomized%20prospective%20study&author=L.%20Bonello&author=L.%20Camoin-Jau&author=S.%20Arques&author=C.%20Boyer&author=D.%20Panagides&volume=51&publication_year=2008&pages=1404-1411&pmid=18387444&doi=10.1016/j.jacc.2007.12.044&)

33. Feng H., Conneely K.N., Wu H. A Bayesian hierarchical model to detect differentially methylated loci from single-nucleotide resolution sequencing data. Nucleic Acids Res. 2014;42:e69. doi: 10.1093/nar/gku154.  [DOI](https://doi.org/10.1093/nar/gku154) | [PMC free article](/articles/PMC4005660/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24561809/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=A%C2%A0Bayesian%20hierarchical%20model%20to%20detect%20differentially%20methylated%20loci%20from%20single-nucleotide%20resolution%20sequencing%20data&author=H.%20Feng&author=K.N.%20Conneely&author=H.%20Wu&volume=42&publication_year=2014&pages=e69&pmid=24561809&doi=10.1093/nar/gku154&)

34. Paul D.S., Guilhamon P., Karpathakis A., Butcher L.M., Thirlwell C., Feber A., et al. Assessment of RainDrop BS-seq as a method for large-scale, targeted bisulfite sequencing. Epigenetics-Us. 2014;9:678–684. doi: 10.4161/epi.28041.  [DOI](https://doi.org/10.4161/epi.28041) | [PMC free article](/articles/PMC4063826/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24518816/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Epigenetics-Us&title=Assessment%20of%20RainDrop%20BS-seq%20as%20a%20method%20for%20large-scale,%20targeted%20bisulfite%20sequencing&author=D.S.%20Paul&author=P.%20Guilhamon&author=A.%20Karpathakis&author=L.M.%20Butcher&author=C.%20Thirlwell&volume=9&publication_year=2014&pages=678-684&pmid=24518816&doi=10.4161/epi.28041&)

35. Williams L.A., Mills L., Hooten A.J., Langer E., Roesler M., Frazier A.L., et al. Differences in DNA methylation profiles by histologic subtype of paediatric germ cell tumours: A report from the Children's Oncology Group. Br J Cancer. 2018;119:864–872. doi: 10.1038/s41416-018-0277-5.  [DOI](https://doi.org/10.1038/s41416-018-0277-5) | [PMC free article](/articles/PMC6189207/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30287918/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Cancer&title=Differences%20in%20DNA%20methylation%20profiles%20by%20histologic%20subtype%20of%20paediatric%20germ%20cell%20tumours:%20A%20report%20from%20the%20Children's%20Oncology%20Group&author=L.A.%20Williams&author=L.%20Mills&author=A.J.%20Hooten&author=E.%20Langer&author=M.%20Roesler&volume=119&publication_year=2018&pages=864-872&pmid=30287918&doi=10.1038/s41416-018-0277-5&)

36. Kim K., Jackson B.C., Zhang H., Toews D.P., Taylor S.A., Greig E.I., et al. Genetics and evidence for balancing selection of a sex-linked colour polymorphism in a songbird. Nat Commun. 2019;10:1852. doi: 10.1038/s41467-019-09806-6.  [DOI](https://doi.org/10.1038/s41467-019-09806-6) | [PMC free article](/articles/PMC6478913/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31015412/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&title=Genetics%20and%20evidence%20for%20balancing%20selection%20of%20a%20sex-linked%20colour%20polymorphism%20in%20a%20songbird&author=K.%20Kim&author=B.C.%20Jackson&author=H.%20Zhang&author=D.P.%20Toews&author=S.A.%20Taylor&volume=10&publication_year=2019&pages=1852&pmid=31015412&doi=10.1038/s41467-019-09806-6&)

37. Amante S.M., Montibus B., Cowley M., Barkas N., Setiadi J., Saadeh H., et al. Transcription of intragenic CpG islands influences spatiotemporal host gene pre-mRNA processing. Nucleic Acids Res. 2020;48:8349–8359. doi: 10.1093/nar/gkaa556.  [DOI](https://doi.org/10.1093/nar/gkaa556) | [PMC free article](/articles/PMC7470969/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32621610/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=Transcription%20of%20intragenic%20CpG%20islands%20influences%20spatiotemporal%20host%20gene%20pre-mRNA%20processing&author=S.M.%20Amante&author=B.%20Montibus&author=M.%20Cowley&author=N.%20Barkas&author=J.%20Setiadi&volume=48&publication_year=2020&pages=8349-8359&pmid=32621610&doi=10.1093/nar/gkaa556&)

38. Feng S., Cokus S.J., Zhang X., Chen P.Y., Bostick M., Goll M.G., et al. Conservation and divergence of methylation patterning in plants and animals. Proc Natl Acad Sci USA. 2010;107:8689–8694. doi: 10.1073/pnas.1002720107.  [DOI](https://doi.org/10.1073/pnas.1002720107) | [PMC free article](/articles/PMC2889301/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20395551/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20USA&title=Conservation%20and%20divergence%20of%20methylation%20patterning%20in%20plants%20and%20animals&author=S.%20Feng&author=S.J.%20Cokus&author=X.%20Zhang&author=P.Y.%20Chen&author=M.%20Bostick&volume=107&publication_year=2010&pages=8689-8694&pmid=20395551&doi=10.1073/pnas.1002720107&)

39. Maunakea A.K., Nagarajan R.P., Bilenky M., Ballinger T.J., D’Souza C., Fouse S.D., et al. Conserved role of intragenic DNA methylation in regulating alternative promoters. Nature. 2010;466:253–257. doi: 10.1038/nature09165.  [DOI](https://doi.org/10.1038/nature09165) | [PMC free article](/articles/PMC3998662/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20613842/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Conserved%20role%20of%20intragenic%20DNA%20methylation%20in%20regulating%20alternative%20promoters&author=A.K.%20Maunakea&author=R.P.%20Nagarajan&author=M.%20Bilenky&author=T.J.%20Ballinger&author=C.%20D%E2%80%99Souza&volume=466&publication_year=2010&pages=253-257&pmid=20613842&doi=10.1038/nature09165&)

40. Su J., Zheng N., Li Z., Huangfu N., Mei L., Xu X., et al. Association of GCK gene DNA methylation with the risk of clopidogrel resistance in acute coronary syndrome patients. J Clin Lab Anal. 2020;34 doi: 10.1002/jcla.23040.  [DOI](https://doi.org/10.1002/jcla.23040) | [PMC free article](/articles/PMC7031555/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31605429/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%C2%A0Clin%20Lab%20Anal&title=Association%20of%20GCK%20gene%20DNA%20methylation%20with%20the%20risk%20of%20clopidogrel%20resistance%20in%20acute%20coronary%20syndrome%20patients&author=J.%20Su&author=N.%20Zheng&author=Z.%20Li&author=N.%20Huangfu&author=L.%20Mei&volume=34&publication_year=2020&pmid=31605429&doi=10.1002/jcla.23040&)

41. Dayeh T., Volkov P., Salo S., Hall E., Nilsson E., Olsson A.H., et al. Genome-wide DNA methylation analysis of human pancreatic islets from type 2 diabetic and non-diabetic donors identifies candidate genes that influence insulin secretion. Plos Genet. 2014;10 doi: 10.1371/journal.pgen.1004160.  [DOI](https://doi.org/10.1371/journal.pgen.1004160) | [PMC free article](/articles/PMC3945174/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24603685/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Plos%20Genet&title=Genome-wide%20DNA%20methylation%20analysis%20of%20human%20pancreatic%20islets%20from%20type%202%20diabetic%20and%20non-diabetic%20donors%20identifies%20candidate%20genes%20that%20influence%20insulin%20secretion&author=T.%20Dayeh&author=P.%20Volkov&author=S.%20Salo&author=E.%20Hall&author=E.%20Nilsson&volume=10&publication_year=2014&pmid=24603685&doi=10.1371/journal.pgen.1004160&)

42. Bibikova M., Barnes B., Tsan C., Ho V., Klotzle B., Le J.M., et al. High density DNA methylation array with single CpG site resolution. Genomics. 2011;98:288–295. doi: 10.1016/j.ygeno.2011.07.007.  [DOI](https://doi.org/10.1016/j.ygeno.2011.07.007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21839163/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genomics&title=High%20density%20DNA%20methylation%20array%20with%20single%20CpG%20site%20resolution&author=M.%20Bibikova&author=B.%20Barnes&author=C.%20Tsan&author=V.%20Ho&author=B.%20Klotzle&volume=98&publication_year=2011&pages=288-295&pmid=21839163&doi=10.1016/j.ygeno.2011.07.007&)

43. Zampieri M., Ciccarone F., Calabrese R., Franceschi C., Burkle A., Caiafa P. Reconfiguration of DNA methylation in aging. Mech Ageing Dev. 2015;151:60–70. doi: 10.1016/j.mad.2015.02.002.  [DOI](https://doi.org/10.1016/j.mad.2015.02.002) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25708826/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mech%20Ageing%20Dev&title=Reconfiguration%20of%20DNA%20methylation%20in%20aging&author=M.%20Zampieri&author=F.%20Ciccarone&author=R.%20Calabrese&author=C.%20Franceschi&author=A.%20Burkle&volume=151&publication_year=2015&pages=60-70&pmid=25708826&doi=10.1016/j.mad.2015.02.002&)

44. Perez R.F., Santamarina P., Tejedor J.R., Urdinguio R.G., Álvarez-Pitti J., Redon P., et al. Longitudinal genome-wide DNA methylation analysis uncovers persistent early-life DNA methylation changes. J Transl Med. 2019;17:15. doi: 10.1186/s12967-018-1751-9.  [DOI](https://doi.org/10.1186/s12967-018-1751-9) | [PMC free article](/articles/PMC6327427/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30626398/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%C2%A0Transl%20Med&title=Longitudinal%20genome-wide%20DNA%20methylation%20analysis%20uncovers%20persistent%20early-life%20DNA%20methylation%20changes&author=R.F.%20Perez&author=P.%20Santamarina&author=J.R.%20Tejedor&author=R.G.%20Urdinguio&author=J.%20%C3%81lvarez-Pitti&volume=17&publication_year=2019&pages=15&pmid=30626398&doi=10.1186/s12967-018-1751-9&)

45. Fennell L., Dumenil T., Wockner L., Hartel G., Nones K., Bond C., et al. Integrative Genome-Scale DNA methylation analysis of a large and unselected cohort reveals 5 distinct subtypes of colorectal adenocarcinomas. Cell Mol Gastroenterol Hepatol. 2019;8:269–290. doi: 10.1016/j.jcmgh.2019.04.002.  [DOI](https://doi.org/10.1016/j.jcmgh.2019.04.002) | [PMC free article](/articles/PMC6699251/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30954552/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Mol%20Gastroenterol%20Hepatol&title=Integrative%20Genome-Scale%20DNA%20methylation%20analysis%20of%20a%20large%20and%20unselected%20cohort%20reveals%205%20distinct%20subtypes%20of%20colorectal%20adenocarcinomas&author=L.%20Fennell&author=T.%20Dumenil&author=L.%20Wockner&author=G.%20Hartel&author=K.%20Nones&volume=8&publication_year=2019&pages=269-290&pmid=30954552&doi=10.1016/j.jcmgh.2019.04.002&)

46. Sandoval J., Heyn H., Moran S., Serra-Musach J., Pujana M.A., Bibikova M., et al. Validation of a DNA methylation microarray for 450,000 CpG sites in the human genome. Epigenetics-Us. 2011;6:692–702. doi: 10.4161/epi.6.6.16196.  [DOI](https://doi.org/10.4161/epi.6.6.16196) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21593595/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Epigenetics-Us&title=Validation%20of%20a%20DNA%20methylation%20microarray%20for%20450,000%20CpG%20sites%20in%20the%20human%20genome&author=J.%20Sandoval&author=H.%20Heyn&author=S.%20Moran&author=J.%20Serra-Musach&author=M.A.%20Pujana&volume=6&publication_year=2011&pages=692-702&pmid=21593595&doi=10.4161/epi.6.6.16196&)

47. Timp W., Bravo H.C., McDonald O.G., Goggins M., Umbricht C., Zeiger M., et al. Large hypomethylated blocks as a universal defining epigenetic alteration in human solid tumors. Genome Med. 2014;6:61. doi: 10.1186/s13073-014-0061-y.  [DOI](https://doi.org/10.1186/s13073-014-0061-y) | [PMC free article](/articles/PMC4154522/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25191524/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Med&title=Large%20hypomethylated%20blocks%20as%20a%20universal%20defining%20epigenetic%20alteration%20in%20human%20solid%20tumors&author=W.%20Timp&author=H.C.%20Bravo&author=O.G.%20McDonald&author=M.%20Goggins&author=C.%20Umbricht&volume=6&publication_year=2014&pages=61&pmid=25191524&doi=10.1186/s13073-014-0061-y&)

48. Niles K.M., Chan D., La Salle S., Oakes C.C., Trasler J.M. Critical period of nonpromoter DNA methylation acquisition during prenatal male germ cell development. Plos One. 2011;6 doi: 10.1371/journal.pone.0024156.  [DOI](https://doi.org/10.1371/journal.pone.0024156) | [PMC free article](/articles/PMC3176233/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21949694/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Plos%20One&title=Critical%20period%20of%20nonpromoter%20DNA%20methylation%20acquisition%20during%20prenatal%20male%20germ%20cell%20development&author=K.M.%20Niles&author=D.%20Chan&author=S.%20La%20Salle&author=C.C.%20Oakes&author=J.M.%20Trasler&volume=6&publication_year=2011&pmid=21949694&doi=10.1371/journal.pone.0024156&)

49. Xie H., Wang M., Bonaldo M.F., Rajaram V., Stellpflug W., Smith C., et al. Epigenomic analysis of Alu repeats in human ependymomas. Proc Natl Acad Sci USA. 2010;107:6952–6957. doi: 10.1073/pnas.0913836107.  [DOI](https://doi.org/10.1073/pnas.0913836107) | [PMC free article](/articles/PMC2872440/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20351280/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20USA&title=Epigenomic%20analysis%20of%20Alu%20repeats%20in%20human%20ependymomas&author=H.%20Xie&author=M.%20Wang&author=M.F.%20Bonaldo&author=V.%20Rajaram&author=W.%20Stellpflug&volume=107&publication_year=2010&pages=6952-6957&pmid=20351280&doi=10.1073/pnas.0913836107&)

50. Yoder J.A., Walsh C.P., Bestor T.H. Cytosine methylation and the ecology of intragenomic parasites. Trends Genet. 1997;13:335–340. doi: 10.1016/s0168-9525(97)01181-5.  [DOI](https://doi.org/10.1016/s0168-9525(97)01181-5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9260521/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Trends%20Genet&title=Cytosine%20methylation%20and%20the%20ecology%20of%20intragenomic%20parasites&author=J.A.%20Yoder&author=C.P.%20Walsh&author=T.H.%20Bestor&volume=13&publication_year=1997&pages=335-340&pmid=9260521&doi=10.1016/s0168-9525(97)01181-5&)

51. Lister R., Pelizzola M., Dowen R.H., Hawkins R.D., Hon G., Tonti-Filippini J., et al. Human DNA methylomes at base resolution show widespread epigenomic differences. Nature. 2009;462:315–322. doi: 10.1038/nature08514.  [DOI](https://doi.org/10.1038/nature08514) | [PMC free article](/articles/PMC2857523/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19829295/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Human%20DNA%20methylomes%20at%20base%20resolution%20show%20widespread%20epigenomic%20differences&author=R.%20Lister&author=M.%20Pelizzola&author=R.H.%20Dowen&author=R.D.%20Hawkins&author=G.%20Hon&volume=462&publication_year=2009&pages=315-322&pmid=19829295&doi=10.1038/nature08514&)

52. Jones L.R., Young W., Divine G., Datta I., Chen K.M., Ozog D., et al. Genome-wide scan for methylation profiles in keloids. Dis Markers. 2015;2015 doi: 10.1155/2015/943176.  [DOI](https://doi.org/10.1155/2015/943176) | [PMC free article](/articles/PMC4446486/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26074660/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Dis%20Markers&title=Genome-wide%20scan%20for%20methylation%20profiles%20in%20keloids&author=L.R.%20Jones&author=W.%20Young&author=G.%20Divine&author=I.%20Datta&author=K.M.%20Chen&volume=2015&publication_year=2015&pmid=26074660&doi=10.1155/2015/943176&)

53. Hogart A., Nagarajan R.P., Patzel K.A., Yasui D.H., Lasalle J.M. 15Q11-13 GABAA receptor genes are normally biallelically expressed in brain yet are subject to epigenetic dysregulation in autism-spectrum disorders. Hum Mol Genet. 2007;16:691–703. doi: 10.1093/hmg/ddm014.  [DOI](https://doi.org/10.1093/hmg/ddm014) | [PMC free article](/articles/PMC1934608/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17339270/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=15Q11-13%20GABAA%20receptor%20genes%20are%20normally%20biallelically%20expressed%20in%20brain%20yet%20are%20subject%20to%20epigenetic%20dysregulation%20in%20autism-spectrum%20disorders&author=A.%20Hogart&author=R.P.%20Nagarajan&author=K.A.%20Patzel&author=D.H.%20Yasui&author=J.M.%20Lasalle&volume=16&publication_year=2007&pages=691-703&pmid=17339270&doi=10.1093/hmg/ddm014&)

54. Yang J., Yu Q., Xu Z., Zheng N., Zhong J., Li J., et al. Clopidogrel resistance is associated with DNA methylation of genes from whole blood of humans. Front Genet. 2020;11 doi: 10.3389/fgene.2020.583215.  [DOI](https://doi.org/10.3389/fgene.2020.583215) | [PMC free article](/articles/PMC7844369/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33519892/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front%20Genet&title=Clopidogrel%20resistance%20is%20associated%20with%20DNA%20methylation%20of%20genes%20from%20whole%20blood%20of%20humans&author=J.%20Yang&author=Q.%20Yu&author=Z.%20Xu&author=N.%20Zheng&author=J.%20Zhong&volume=11&publication_year=2020&pmid=33519892&doi=10.3389/fgene.2020.583215&)

55. Zhong W.P., Wu H., Chen J.Y., Li X.X., Lin H.M., Zhang B., et al. Genomewide association study identifies novel genetic loci that modify antiplatelet effects and pharmacokinetics of clopidogrel. Clin Pharmacol Ther. 2017;101:791–802. doi: 10.1002/cpt.589.  [DOI](https://doi.org/10.1002/cpt.589) | [PMC free article](/articles/PMC5485718/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27981573/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Genomewide%20association%20study%20identifies%20novel%20genetic%20loci%20that%20modify%20antiplatelet%20effects%20and%20pharmacokinetics%20of%20clopidogrel&author=W.P.%20Zhong&author=H.%20Wu&author=J.Y.%20Chen&author=X.X.%20Li&author=H.M.%20Lin&volume=101&publication_year=2017&pages=791-802&pmid=27981573&doi=10.1002/cpt.589&)

56. Bonello L., Paganelli F., Arpin-Bornet M., Auquier P., Sampol J., Dignat-George F., et al. Vasodilator-stimulated phosphoprotein phosphorylation analysis prior to percutaneous coronary intervention for exclusion of postprocedural major adverse cardiovascular events. J Thromb Haemost. 2007;5:1630–1636. doi: 10.1111/j.1538-7836.2007.02609.x.  [DOI](https://doi.org/10.1111/j.1538-7836.2007.02609.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17488353/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%C2%A0Thromb%20Haemost&title=Vasodilator-stimulated%20phosphoprotein%20phosphorylation%20analysis%20prior%20to%20percutaneous%20coronary%20intervention%20for%20exclusion%20of%20postprocedural%20major%20adverse%20cardiovascular%20events&author=L.%20Bonello&author=F.%20Paganelli&author=M.%20Arpin-Bornet&author=P.%20Auquier&author=J.%20Sampol&volume=5&publication_year=2007&pages=1630-1636&pmid=17488353&doi=10.1111/j.1538-7836.2007.02609.x&)

57. Siller-Matula J.M., Trenk D., Schror K., Gawaz M., Kristensen S.D., Storey R.F., et al. Response variability to P2Y12 receptor inhibitors: expectations and reality. JACC Cardiovasc Interv. 2013;6:1111–1128. doi: 10.1016/j.jcin.2013.06.011.  [DOI](https://doi.org/10.1016/j.jcin.2013.06.011) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24262612/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JACC%20Cardiovasc%20Interv&title=Response%20variability%20to%20P2Y12%20receptor%20inhibitors:%C2%A0expectations%20and%20reality&author=J.M.%20Siller-Matula&author=D.%20Trenk&author=K.%20Schror&author=M.%20Gawaz&author=S.D.%20Kristensen&volume=6&publication_year=2013&pages=1111-1128&pmid=24262612&doi=10.1016/j.jcin.2013.06.011&)

58. Dobrovolsky A.B., Laguta P.S., Guskova E.V., Yarovaya E.B., Titaeva E.V., Storozhilova A.N., et al. Effect of fibrinogen on platelet reactivity measured by the VerifyNow P2Y12 assay. Biochemistry (Moscow) 2016;81:439–444. doi: 10.1134/S0006297916050011.  [DOI](https://doi.org/10.1134/S0006297916050011) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27297894/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochemistry%20(Moscow)&title=Effect%20of%20fibrinogen%20on%20platelet%20reactivity%20measured%20by%20the%20VerifyNow%20P2Y12%20assay&author=A.B.%20Dobrovolsky&author=P.S.%20Laguta&author=E.V.%20Guskova&author=E.B.%20Yarovaya&author=E.V.%20Titaeva&volume=81&publication_year=2016&pages=439-444&pmid=27297894&doi=10.1134/S0006297916050011&)

59. Larsen S.B., Grove E.L., Neergaard-Petersen S., Wurtz M., Hvas A.M., Kristensen S.D. Determinants of reduced antiplatelet effect of aspirin in patients with stable coronary artery disease. Plos One. 2015;10 doi: 10.1371/journal.pone.0126767.  [DOI](https://doi.org/10.1371/journal.pone.0126767) | [PMC free article](/articles/PMC4436265/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25993271/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Plos%20One&title=Determinants%20of%20reduced%20antiplatelet%20effect%20of%20aspirin%20in%20patients%20with%20stable%20coronary%20artery%20disease&author=S.B.%20Larsen&author=E.L.%20Grove&author=S.%20Neergaard-Petersen&author=M.%20Wurtz&author=A.M.%20Hvas&volume=10&publication_year=2015&pmid=25993271&doi=10.1371/journal.pone.0126767&)

60. Kim Y.G., Such J.W., Park J.J., Oh I.Y., Yoon C.H., Cho Y.S., et al. Different influences of hematocrit on the results of two Point-Of-Care platelet function tests, the VerifyNow assay and multiple electrode platelet aggregometry. Plos One. 2014;9 doi: 10.1371/journal.pone.0114053.  [DOI](https://doi.org/10.1371/journal.pone.0114053) | [PMC free article](/articles/PMC4245259/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25427105/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Plos%20One&title=Different%20influences%20of%20hematocrit%20on%20the%20results%20of%20two%20Point-Of-Care%20platelet%20function%20tests,%20the%20VerifyNow%20assay%20and%20multiple%20electrode%20platelet%20aggregometry&author=Y.G.%20Kim&author=J.W.%20Such&author=J.J.%20Park&author=I.Y.%20Oh&author=C.H.%20Yoon&volume=9&publication_year=2014&pmid=25427105&doi=10.1371/journal.pone.0114053&)

61. Gallego-Fabrega C., Carrera C., Reny J.L., Fontana P., Slowik A., Pera J., et al. TRAF3 epigenetic regulation is associated with vascular recurrence in patients with ischemic stroke. Stroke. 2016;47:1180–1186. doi: 10.1161/STROKEAHA.115.012237.  [DOI](https://doi.org/10.1161/STROKEAHA.115.012237) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27026631/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Stroke&title=TRAF3%20epigenetic%20regulation%20is%20associated%20with%20vascular%C2%A0recurrence%20in%20patients%20with%20ischemic%20stroke&author=C.%20Gallego-Fabrega&author=C.%20Carrera&author=J.L.%20Reny&author=P.%20Fontana&author=A.%20Slowik&volume=47&publication_year=2016&pages=1180-1186&pmid=27026631&doi=10.1161/STROKEAHA.115.012237&)

62. Chen X., Hu H., Guan X., Xiong G., Wang Y., Wang K., et al. CpG island methylation status of miRNAs in esophageal squamous cell carcinoma. Int J Cancer. 2012;130:1607–1613. doi: 10.1002/ijc.26171.  [DOI](https://doi.org/10.1002/ijc.26171) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21547903/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Cancer&title=CpG%20island%20methylation%20status%20of%20miRNAs%20in%20esophageal%20squamous%20cell%20carcinoma&author=X.%20Chen&author=H.%20Hu&author=X.%20Guan&author=G.%20Xiong&author=Y.%20Wang&volume=130&publication_year=2012&pages=1607-1613&pmid=21547903&doi=10.1002/ijc.26171&)

63. Chuang S.M., Lu J.H., Lin K.L., Long C.Y., Lee Y.C., Hsiao H.P., et al. Epigenetic regulation of COX2 expression by DNA hypomethylation via NFkappaB activation in ketamine-induced ulcerative cystitis. Int J Mol Med. 2019;44:797–812. doi: 10.3892/ijmm.2019.4252.  [DOI](https://doi.org/10.3892/ijmm.2019.4252) | [PMC free article](/articles/PMC6657979/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31257475/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Mol%20Med&title=Epigenetic%20regulation%20of%20COX2%20expression%20by%20DNA%20hypomethylation%20via%20NFkappaB%20activation%20in%20ketamine-induced%20ulcerative%20cystitis&author=S.M.%20Chuang&author=J.H.%20Lu&author=K.L.%20Lin&author=C.Y.%20Long&author=Y.C.%20Lee&volume=44&publication_year=2019&pages=797-812&pmid=31257475&doi=10.3892/ijmm.2019.4252&)

64. Ball M.P., Li J.B., Gao Y., Lee J.H., LeProust E.M., Park I.H., et al. Targeted and genome-scale strategies reveal gene-body methylation signatures in human cells. Nat Biotechnol. 2009;27:361–368. doi: 10.1038/nbt.1533.  [DOI](https://doi.org/10.1038/nbt.1533) | [PMC free article](/articles/PMC3566772/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19329998/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Biotechnol&title=Targeted%20and%20genome-scale%20strategies%20reveal%20gene-body%20methylation%20signatures%20in%20human%20cells&author=M.P.%20Ball&author=J.B.%20Li&author=Y.%20Gao&author=J.H.%20Lee&author=E.M.%20LeProust&volume=27&publication_year=2009&pages=361-368&pmid=19329998&doi=10.1038/nbt.1533&)

65. Shah U.J., Xie W., Flyvbjerg A., Nolan J.J., Højlund K., Walker M., et al. Differential methylation of the type 2 diabetes susceptibility locus KCNQ1 is associated with insulin sensitivity and is predicted by CpG site specific genetic variation. Diabetes Res Clin Pract. 2019;148:189–199. doi: 10.1016/j.diabres.2019.01.008.  [DOI](https://doi.org/10.1016/j.diabres.2019.01.008) | [PMC free article](/articles/PMC6395844/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30641161/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Diabetes%20Res%20Clin%20Pract&title=Differential%20methylation%20of%20the%20type%202%20diabetes%20susceptibility%20locus%20KCNQ1%20is%20associated%20with%20insulin%20sensitivity%20and%20is%20predicted%20by%20CpG%20site%20specific%20genetic%20variation&author=U.J.%20Shah&author=W.%20Xie&author=A.%20Flyvbjerg&author=J.J.%20Nolan&author=K.%20H%C3%B8jlund&volume=148&publication_year=2019&pages=189-199&pmid=30641161&doi=10.1016/j.diabres.2019.01.008&)

66. Liu R.K., Lin X., Wang Z., Greenbaum J., Qiu C., Zeng C.P., et al. Identification of novel functional CpG-SNPs associated with Type 2 diabetes and birth weight. Aging (Albany NY) 2021;13:10619–10658. doi: 10.18632/aging.202828.  [DOI](https://doi.org/10.18632/aging.202828) | [PMC free article](/articles/PMC8064204/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33835050/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Aging%20(Albany%20NY)&title=Identification%20of%20novel%20functional%20CpG-SNPs%20associated%20with%20Type%202%20diabetes%20and%20birth%20weight&author=R.K.%20Liu&author=X.%20Lin&author=Z.%20Wang&author=J.%20Greenbaum&author=C.%20Qiu&volume=13&publication_year=2021&pages=10619-10658&pmid=33835050&doi=10.18632/aging.202828&)

67. Trzupek D., Dunstan M., Cutler A.J., Lee M., Godfrey L., Jarvis L., et al. Discovery of CD80 and CD86 as recent activation markers on regulatory T cells by protein-RNA single-cell analysis. Genome Med. 2020;12:55. doi: 10.1186/s13073-020-00756-z.  [DOI](https://doi.org/10.1186/s13073-020-00756-z) | [PMC free article](/articles/PMC7315544/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32580776/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Med&title=Discovery%20of%20CD80%20and%20CD86%20as%20recent%20activation%20markers%20on%20regulatory%20T%20cells%20by%20protein-RNA%20single-cell%20analysis&author=D.%20Trzupek&author=M.%20Dunstan&author=A.J.%20Cutler&author=M.%20Lee&author=L.%20Godfrey&volume=12&publication_year=2020&pages=55&pmid=32580776&doi=10.1186/s13073-020-00756-z&)

68. Shariq M., Quadir N., Sharma N., Singh J., Sheikh J.A., Khubaib M., et al. Mycobacterium tuberculosis RipA dampens TLR4-mediated host protective response using a multi-pronged approach involving autophagy, apoptosis, metabolic repurposing, and immune modulation. Front Immunol. 2021;12 doi: 10.3389/fimmu.2021.636644.  [DOI](https://doi.org/10.3389/fimmu.2021.636644) | [PMC free article](/articles/PMC7969667/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33746976/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front%20Immunol&title=Mycobacterium%20tuberculosis%20RipA%20dampens%20TLR4-mediated%20host%20protective%20response%20using%20a%20multi-pronged%20approach%20involving%20autophagy,%20apoptosis,%20metabolic%20repurposing,%20and%20immune%20modulation&author=M.%20Shariq&author=N.%20Quadir&author=N.%20Sharma&author=J.%20Singh&author=J.A.%20Sheikh&volume=12&publication_year=2021&pmid=33746976&doi=10.3389/fimmu.2021.636644&)

69. Nguyen T.A., Diodati J.G., Pharand C. Resistance to clopidogrel: a review of the evidence. J Am Coll Cardiol. 2005;45:1157–1164. doi: 10.1016/j.jacc.2005.01.034.  [DOI](https://doi.org/10.1016/j.jacc.2005.01.034) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15837243/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%C2%A0Am%20Coll%20Cardiol&title=Resistance%20to%20clopidogrel:%20a%20review%20of%20the%20evidence&author=T.A.%20Nguyen&author=J.G.%20Diodati&author=C.%20Pharand&volume=45&publication_year=2005&pages=1157-1164&pmid=15837243&doi=10.1016/j.jacc.2005.01.034&)

70. Angiolillo D.J., Fernandez-Ortiz A., Bernardo E., Alfonso F., Macaya C., Bass T.A., et al. Variability in individual responsiveness to clopidogrel: Clinical implications, management, and future perspectives. J Am Coll Cardiol. 2007;49:1505–1516. doi: 10.1016/j.jacc.2006.11.044.  [DOI](https://doi.org/10.1016/j.jacc.2006.11.044) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17418288/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%C2%A0Am%20Coll%20Cardiol&title=Variability%20in%20individual%20responsiveness%20to%20clopidogrel:%20Clinical%20implications,%20management,%20and%20future%20perspectives&author=D.J.%20Angiolillo&author=A.%20Fernandez-Ortiz&author=E.%20Bernardo&author=F.%20Alfonso&author=C.%20Macaya&volume=49&publication_year=2007&pages=1505-1516&pmid=17418288&doi=10.1016/j.jacc.2006.11.044&)

71. Valgimigli M., Bueno H., Byrne R.A., Collet J.P., Costa F., Jeppsson A., et al. 2017 ESC focused update on dual antiplatelet therapy in coronary artery disease developed in collaboration with EACTS: the Task Force for dual antiplatelet therapy in coronary artery disease of the European Society of Cardiology (ESC) and of the European Association for Cardio-Thoracic Surgery (EACTS) Eur Heart J. 2018;39:213–260. doi: 10.1093/eurheartj/ehx419.  [DOI](https://doi.org/10.1093/eurheartj/ehx419) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28886622/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20Heart%20J&title=2017%20ESC%20focused%20update%20on%20dual%20antiplatelet%20therapy%20in%20coronary%20artery%20disease%20developed%20in%20collaboration%20with%20EACTS:%20the%20Task%20Force%20for%20dual%20antiplatelet%20therapy%20in%20coronary%20artery%20disease%20of%20the%20European%20Society%20of%20Cardiology%20(ESC)%20and%20of%20the%20European%20Association%20for%20Cardio-Thoracic%20Surgery%20(EACTS)&author=M.%20Valgimigli&author=H.%20Bueno&author=R.A.%20Byrne&author=J.P.%20Collet&author=F.%20Costa&volume=39&publication_year=2018&pages=213-260&pmid=28886622&doi=10.1093/eurheartj/ehx419&)

72. Wang Y., Meng X., Wang A., Xie X., Pan Y., Johnston S.C., et al. Ticagrelor versus clopidogrel in CYP2C19 Loss-of-Function Carriers with Stroke or TIA. N Engl J Med. 2021;385:2520–2530. doi: 10.1056/NEJMoa2111749.  [DOI](https://doi.org/10.1056/NEJMoa2111749) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34708996/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%C2%A0Engl%20J%20Med&title=Ticagrelor%20versus%20clopidogrel%20in%20CYP2C19%20Loss-of-Function%20Carriers%20with%20Stroke%20or%20TIA&author=Y.%20Wang&author=X.%20Meng&author=A.%20Wang&author=X.%20Xie&author=Y.%20Pan&volume=385&publication_year=2021&pages=2520-2530&pmid=34708996&doi=10.1056/NEJMoa2111749&)

73. Kang J., Han J.K., Ahn Y., Chae S.C., Kim Y.J., Chae I.H., et al. Third-Generation P2Y12 inhibitors in east Asian acute myocardial infarction patients: A nationwide prospective multicentre study. Thromb Haemost. 2018;118:591–600. doi: 10.1055/s-0038-1626697.  [DOI](https://doi.org/10.1055/s-0038-1626697) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29534250/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Thromb%20Haemost&title=Third-Generation%20P2Y12%20inhibitors%20in%20east%20Asian%20acute%20myocardial%20infarction%20patients:%20A%20nationwide%20prospective%20multicentre%20study&author=J.%20Kang&author=J.K.%20Han&author=Y.%20Ahn&author=S.C.%20Chae&author=Y.J.%20Kim&volume=118&publication_year=2018&pages=591-600&pmid=29534250&doi=10.1055/s-0038-1626697&)

74. Sibbing D., Aradi D., Alexopoulos D., Ten Berg J., Bhatt D.L., Bonello L., et al. Updated expert consensus statement on platelet function and genetic testing for guiding P2Y(12) receptor inhibitor treatment in percutaneous coronary intervention. JACC Cardiovasc Interv. 2019;12:1521–1537. doi: 10.1016/j.jcin.2019.03.034.  [DOI](https://doi.org/10.1016/j.jcin.2019.03.034) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31202949/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JACC%20Cardiovasc%20Interv&title=Updated%20expert%20consensus%20statement%20on%20platelet%20function%20and%20genetic%20testing%20for%20guiding%20P2Y(12)%20receptor%20inhibitor%20treatment%20in%20percutaneous%20coronary%20intervention&author=D.%20Sibbing&author=D.%20Aradi&author=D.%20Alexopoulos&author=J.%20Ten%20Berg&author=D.L.%20Bhatt&volume=12&publication_year=2019&pages=1521-1537&pmid=31202949&doi=10.1016/j.jcin.2019.03.034&)
