# N-acetyltransferase Gene Variants Involved in Pediatric Idiosyncratic Drug-Induced Liver Injury

## Metadata
**Authors:** María Luisa Alés-Palmer, Francisco Andújar-Vera, Iván Iglesias-Baena, Paloma Muñoz-de-Rueda, Esther Ocete-Hita
**Journal:** Biomedicines
**Date:** 2024 Jun 11
**DOI:** [10.3390/biomedicines12061288](https://doi.org/10.3390/biomedicines12061288)
**PMID:** 38927494
**PMCID:** PMC11201799
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11201799/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC11201799/pdf/biomedicines-12-01288.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC11201799/pdf/biomedicines-12-01288.pdf)

## Abstract

Idiosyncratic drug-induced liver injury (DILI) is a complex multifactorial disease in which the toxic potential of the drug, together with genetic and acquired factors and deficiencies in adaptive processes, which limit the extent of damage, may determine susceptibility and make individuals unique in their development of hepatotoxicity. In our study, we sequenced the exomes of 43 pediatric patients diagnosed with DILI to identify important gene variations associated with this pathology. The result showed the presence of two variations in the NAT2 gene: c.590G>A (p.Arg197Gln) and c.341T>C (p.Ile114Thr). These variations could be found separately or together in 41 of the 43 patients studied. The presence of these variations as a risk factor for DILI could confirm the importance of the acetylation pathway in drug metabolism.

Keywords: DILI, whole-exome sequencing, risk factors, hepatotoxicity, liver injury, children, NAT-2 polymorphism

**Keywords:** DILI, whole-exome sequencing, risk factors, hepatotoxicity, liver injury, children, NAT-2 polymorphism

## 1. Introduction

Drug-induced liver injury (DILI) is a rare pathology (14 to 19 cases per 100,000 people) causing 40 to 50% of acute liver failure in adults [[1](#B1-biomedicines-12-01288)]. This problem has been less studied in the pediatric population despite the fact that in more than 40% of liver transplants, the reason for the fulminant liver failure that could be related to this health problem is unknown [[1](#B1-biomedicines-12-01288),[2](#B2-biomedicines-12-01288)]. It is usually caused by a variety of drugs, supplements and herbal products. DILI is a growing problem that even affects the manufacture of drugs, including others that can be found on the market. In children, this problem is increased by the fact that most drugs are used without an indication in the technical data sheet or even with an indication different from the one included in the technical data sheet due to the difficulty of conducting clinical trials in the child population [[2](#B2-biomedicines-12-01288),[3](#B3-biomedicines-12-01288)]. DILI can be divided into intrinsic reactions (which are dose-dependent and therefore predictable once a specific threshold concentration has been consumed) and idiosyncratic reactions, which are not dose-dependent and therefore cannot be explained by the known pharmacological properties of the drug [[3](#B3-biomedicines-12-01288),[4](#B4-biomedicines-12-01288)]. Since the end of the Human Genome Project, genetic analysis techniques have grown at an exponential rate. Whole-exome sequencing (WES) is one of the most efficient techniques for high-throughput genomic analysis, sequencing more than 90% of an individual’s coding DNA. Compared with whole-genome sequencing, the exome represents approximately 1% of the total genome, allowing an improvement in both effort and cost-effectiveness in mutation identification. This is because many disease-causing mutations are located in protein-coding regions [[5](#B5-biomedicines-12-01288),[6](#B6-biomedicines-12-01288)]. On the other hand, pathological genetic aberrations in regulatory regions or poorly sequenced exome regions will remain unidentified. Despite these limitations, WES has proven successful in the identification of both new genes and disease pathways, appearing as the main technique in many scientific articles [[7](#B7-biomedicines-12-01288)].

The relationships between human variations and phenotypes were queried using the ClinVar database ([https://www.ncbi.nlm.nih.gov/clinvar/](https://www.ncbi.nlm.nih.gov/clinvar/) (accessed on 2 May 2024)), which provides public and open access to its reports.

We hypothesize an improved understanding of the molecular mechanisms underlying the development and progression of DILI with the support of sequenced exome analysis.

Current high-throughput sequencing technologies are capable of uncovering genomic sequences whose analysis can be used to characterize genomic alterations (common or rare) in the development of DILI. The exome study approach may eventually provide important biological data supported by the genetic alterations suffered by DILI patients. The molecular target discovery of novel proteins could be used for targeted therapies or the identification of networks of genetic alterations relevant to the clinical context of the pathology of this study [[8](#B8-biomedicines-12-01288)].

N-acetyltransferase (NAT2) is a cytosolic phase II conjugation enzyme that is encoded by the NAT2 gene present on chromosome 8. It is a key human enzyme that plays a role in the detoxification and removal of many drugs, including carcinogenic arylamine compounds [[9](#B9-biomedicines-12-01288)].

NAT2 is one of only 2 N-acetyltransferase genes in humans; the other, NAT1, shows little variation between individuals, whereas NAT2 is known to have over 23 variants. N-acetyltransferases are enzymes acting primarily in the liver to detoxify a large number of chemicals and drugs. The NAT2 acetylation polymorphism is important because of its primary role in the activation and/or deactivation of many chemicals in the body’s environment. In turn, this can affect an individual’s cancer risk [[10](#B10-biomedicines-12-01288)].

Individuals can be classified as either rapid or slow metabolizers (i.e., detoxifiers). In general, slow metabolizers have higher rates of certain types of cancer and are more susceptible to adverse effects from chemicals metabolized by NAT2 [[11](#B11-biomedicines-12-01288),[12](#B12-biomedicines-12-01288),[13](#B13-biomedicines-12-01288)].

It takes two slow-metabolizer alleles to give rise to a slow-metabolizer phenotype, or to put it another way, the rapid-metabolizer allele is dominant over the slow metabolizer; therefore, only one is needed to be a rapid metabolizer.

The NAT2 gene is highly polymorphic, with more than a hundred variant alleles. Genetic variation in the NAT2 gene plays an important role in isoniazid-induced hepatotoxicity [[14](#B14-biomedicines-12-01288)]. Most of the variants (SNPs) are located within the 873 bp intron-free coding region of the NAT2 gene. Among the different alleles, four variant alleles—341 T>C (rs1801280; NAT2*5), 590 G>A (rs1799930; NAT2*6), 857 G>A (rs1799931; NAT2*7) and 191 G>A (rs1801279; NAT2*14)—are studied mainly because of their influence on the acetylation activity of the NAT2 enzyme [[15](#B15-biomedicines-12-01288)].

To enhance our understanding of the relationship between genetic polymorphisms and susceptibility to drug-induced liver injuries in the pediatric population, this study focuses on specific polymorphisms of the NAT2 gene. Previous studies have shown the importance of genetic variations in pharmacokinetics and drug response, but there is a significant gap in the literature regarding how these variations affect children, who may have distinct metabolic profiles compared to adults.

Therefore, the aim of this study was to improve our understanding of the development of pediatric DILI by determining drug-responsiveness, pathogenicity or alterations of uncertain significance in order to identify new strategies for therapeutic approaches and personalized therapy.

## 2. Materials and Methods

### 2.1. Population Study

The population of this study included 43 pediatric patients between the ages of 0 and 15 years who were selected by physicians from participating hospitals and had liver disease associated with the ingestion of medications or herbal products. The operational structure of the registry, the data recording process and the characterization of cases are described in detail in Ocete Hita E (2013) [[16](#B16-biomedicines-12-01288)]. Diagnoses of causality were made using the CIOMS/RUCAM evaluation method [[17](#B17-biomedicines-12-01288)]. The characteristics of the pediatric patients included in the study are summarized in [Table S1 of the supplementary File S1](#app1-biomedicines-12-01288).

All these patients were enrolled in the Spanish registry of hepatotoxic reactions in the pediatric population. The samples were collected and managed by the Biobank of the Public Health System of Andalusia in accordance with its internal procedures.

Due to the difficulty in obtaining healthy pediatric samples and to ensure that they were healthy participants with no history of liver disease, we obtained samples from 28 adults as controls to be compared by WES with our pediatric samples.

### 2.2. Ethics Approval and Consent to Participate

This study was conducted in accordance with the Declaration of Helsinki (as revised in 2013). Ethical approval for the study was obtained from the Granada Provincial Research Ethics Committee (ethics permission number: 0057-M1-20). Written informed consent to participate was obtained from the parents or legal guardians of all participants (under the age of 18) or from participants (for control individuals).

### 2.3. Sample Processing

DNA was isolated from peripheral blood samples obtained from all patients. The commercial HigherPurity™ Blood Genomic DNA Extraction Kit (Canvax, Biotech, Córdoba, Spain) was used for DNA purification according to the manufacturer’s instructions and was used to prepare fragment libraries suitable for massively parallel paired-end sequencing.

The quality and quantity of the gDNA were tested using 3 µL per sample using a 2200 TapeStation system. (A) The amount of DNA was measured by the picogreen method (Invitrogen^TM^, cat. #P7589, Waltham, MA, USA) using Victor 3 fluorometry. (B) DNA status: this was performed using the gel electrophoresis method because this method is a powerful means of revealing the condition (including the presence or absence) of DNA in a sample.

### 2.4. Whole-Exome Sequencing (WES)

A portion of the DNA obtained (50 ng) was fragmented, processed and amplified to generate a sequencing library using the Human Core Exome design and protocol from Twist Bioscience. Twist NGS Target Enrichment is a system based on 130-mer biotinylated ultralarge biotinylated cDNA bait to capture regions of interest from the NGS genomic fragment library and enrich them.

Quality control of the libraries was then performed; first, the size of the PCR-enriched fragments that make up the libraries was checked using an Agilent Technologies Bioanalyzer 2100 with a DNA 1000 chip. Secondly, the quantity of the libraries was checked by qPCR according to the Illumina qPCR quantification protocol guide (KAPA Library Quantification kits for Illumina Sequencing platforms).

Sequencing of the libraries was carried out on an Illumina NovaSeq 6000 Sequencing System. The primary calculations of the data obtained were performed in the integrated RTA (Real-Time Analysis) software v3.4.4. The processed files were transformed to FASTQ using the Illumina bcl2fastq v2.20.0 package, generating a set of paired reads with a size of about 150 bases in length for all samples.

### 2.5. WES Data Analysis

The FASTQ files obtained in the previous section were quality checked by FASTQC and, subsequently, the paired reads were mapped against the human revertant genome (version GRCh38.p13) using the BWA-MEM alignment program (bwa-0.7.17), thus generating .bam files of aligned reads without disordered sequences and alternate haplotypes. Possible duplicate (identical) reads due to PCR amplification were removed using the MarkDuplicates program of the Picard-tools analysis package (v2.18.2). Reads with identical starting positions were considered duplicates and were reduced to a single read.

The .bam files were recalibrated with the Base Quality Score Recalibration (BQSR) program, which uses machine learning algorithms to model sequencing errors empirically and adjust quality scores.

The recalibrated .bam files were used to identify changes (variants) with respect to the reference sequence using GATK’s Haplotype Caller algorithm (v4.0.5.1), thus generating .vcf files.

The identified variants were filtered with GATK’s VariantFiltration tool, followed by functional annotation of the variants using SnpEff (v5.0e 2021-03-09) and filtered with dbSNP (v154) and SNPs from the 1000 genomes project (Phase3). The final file format was vcf. Then, an in-house program and SnpEff was used to annotate variants with additional databases, including ESP6500 (ESP6500SI_V2), ClinVar, dbNSFP (v4.2c) and ACMG.

### 2.6. Variant Selection

The set of variants identified by whole-exome sequencing was narrowed down according to the phenotype described by ClinVar. For this purpose, those variations that were pathogenic or probably pathogenic, responded to drugs or were variables of uncertain significance were selected. In addition, gene variants with a synonymous effect were discarded.

Enrichment analysis of the proteins involved was performed using SRING v12.0.

### 2.7. Validation of DILI-Related Variants and Frequency in Our Samples

The variants found to be relevant would be validated by linkage of DILI-associated genes. Two databases were used to extract information about these genes involved in DILI. DisGeNET ([https://www.disgenet.org/](https://www.disgenet.org/) (accessed on 12 May 2023)) (GDA_score > 0.3) was used on the one hand, and Phenopedia ([https://phgkb.cdc.gov/](https://phgkb.cdc.gov/) (accessed on 12 May 2023)) on the other. The genes obtained in both databases were compared with the genes of the variants resulting from the previous section.

To determine the significance of these mutations in our samples, we discarded those that did not occur in at least 80% of our samples, i.e., each gene should contain at least one variation in 35 of the 43 samples that were sequenced.

### 2.8. Predicting the Functional Effect of Amino Acid Substitutions

Once variants with a high frequency in our samples that were related to DILI were obtained, computational tools such as SIFT ([https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3394338/](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3394338/) (accessed on 12 May 2023)), PROVEAN ([https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4528627/](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4528627/) (accessed on 12 May 2023)), Aling GVGD ([https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2563222/](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2563222/) (accessed on 25 April 2024)), Panther ([https://pubmed.ncbi.nlm.nih.gov/27193693/](https://pubmed.ncbi.nlm.nih.gov/27193693/) (accessed on 25 April 2024)) and PolyPhen 2 ([https://pubmed.ncbi.nlm.nih.gov/23315928/](https://pubmed.ncbi.nlm.nih.gov/23315928/) (accessed on 25 April 2024)) were used to determine which of these variants might be causative of the pathology or characteristic phenotype in DILI. Default thresholds were used to determine whether it was a deleterious/pathogenic mutation based on the prediction of each tool.

### 2.9. Statistical Analysis

Categorical variables were summarized using frequencies (%), and group comparisons were conducted using chi-square tests or Fisher’s exact test, when appropriate. A *p*-value < 0.05 was considered significant. The analysis was conducted using R-studio statistical software version 4.2.3.

### 2.10. Genotyping of SNPs in NAT2 by Real-Time qPCR

All genotyping experiments were conducted at the Integrated DNA Technologies (IDT) laboratory (Redwood City, CA, USA). The controls and patient samples were genotyped by rhAmp technology with designed primers for rs1801280 and rs1799930 following the manufacture protocol, and measurements were carried out with a BioRad real time thermocycler. The oligonucleotides were as follows: for rs1801280, Allele Primer 1/rhAmp-F/AATGTAATTCCTGCCGTCAArUGGTC/GT4/, Allele Primer 2/rhAmp-Y/AATGTAATTCCTGCCGTCAGrUGGTC/GT4/ and Locus Primer GCTACATCCCTCCAGTTAACAAATACrAGCAC/GT1/; for rs1799930, Allele Primer 1/rhAmp-F/TACTTATTTACGCTTGAACCTCGrAACAA/GT1/, Allele Primer 2/rhAmp-Y/ATACTTATTTACGCTTGAACCTCArAACAA/GT1/ and Locus Primer GCTCTGCAGGTATGTATTCATAGACTrCAAAA/GT2/.

## 3. Results

### 3.1. Sequence Coverage and Mutation Analysis

A total of 43 samples were analyzed after passing the quality controls for processing. This dataset provided a vast new reservoir of data. The exome sequencing specifications of the 43 DILI patient samples are shown in [Table 1](#biomedicines-12-01288-t001).

### Table 1.

| Data Specification | Mean ± SD |
| --- | --- |
| Total reads | 6.48 × 107 ± 8.96 × 106 |
| Total yield (bp) | 8.98 × 109 ± 1.25 × 109 |
| Average read length (bp) | 1.39 × 102 ± 4.33 × 100 |
| Average throughput depth of target regions (X) | 2.71 × 102 ± 3.78 × 101 |
| Initial mappable reads (mapped to human genome) | 6.48 × 107 ± 8.96 × 106 |
| Non-redundant reads | 5.31 × 107 ± 7.94 × 106 |
| On-target reads | 4.29 × 107 ± 8.87 × 106 |
| Number of on-target genotypes (more than 1× | 3.29 × 107 ± 5.58 × 104 |
| Number of on-target genotypes (more than 10×) | 3.26 × 107 ± 3.26 × 105 |
| Number of on-target genotypes (more than 20×) | 3.23 × 107 ± 1.84 × 105 |
| Mean depth of target regions (X) | 1.30 × 102 ± 2.69 × 101 |
| Number of SNPs | 6.00 × 104 ± 1.11 × 103 |
| Number of synonymous SNPs | 1.16 × 104 ± 1.47 × 102 |
| Number of Missense Variants | 1.09 × 104 ± 1.84 × 102 |
| Number of Stop Losses | 1.47 × 101 ± 2.65 × 100 |
| Number of indels | 8.78 × 103 ± 5.20 × 102 |
| Number of Frameshift Variants | 2.29 × 102 ± 1.98 × 101 |
| Number of Inframe Insertions | 1.86 × 102 ± 8.32 × 100 |
| Number of Inframe Deletions | 2.63 × 102 ± 1.18 × 101 |
| % Found in dbSNP138 | 9.32 × 101 ± 3.07 × 10−1 |
| % Found in dbSNP154 | 9.31 × 101 ± 4.06 × 10−1 |
| Het/Hom ratio | 1.69 × 100 ± 1.19 × 10−1 |
| Ts/Tv ratio | 2.40 × 100 ± 1.79 × 10−15 |

Table 1 Caption: Mean values in the exome sequencing data specifications of the 43 patients diagnosed with DILI.

WES identified a total of 2,578,853 variants in the 43 cases of pediatric patient samples diagnosed with DILI. From these, those variants whose ClinVar classification term was pathogenic, probably pathogenic, drug responsive, or of uncertain significance were selected. In addition, those variants that had a synonymous effect were discarded. The total number was reduced to 4871, and from this set of variants, a protein–protein interaction and biological enrichment network was established, the result of which is shown in the [Supplementary Materials (File S2)](#app1-biomedicines-12-01288).

The analysis focused on the pathology of this study by excluding genes that were not associated with DILI according to the DISGENT and Phenopedia databases. In addition, so that the variants identified would have certain relevance in this study, only those with a frequency of more than 80% in our samples were selected.

After applying the depicted workflow ([Figure 1](#biomedicines-12-01288-f001)), our analysis identified 361 variants (corresponding to the ABCB1, CYP2B6 and NAT2 genes) that could be interesting for this study. Each step is explained in detail in the following sections.

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f751/11201799/618a22fc3874/biomedicines-12-01288-g001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=11201799_biomedicines-12-01288-g001.jpg)

Variant selection workflow. Whole-exome sequencing data from 43 unrelated children diagnosed with drug-induced liver injury. Variants classified as “Pathogenic”, “Likely pathogenic”, “Drug Response” or “Uncertain Significance” by ClinVar were prioritized. The variations with synonymous effects were discarded. Next, those gene variants whose genes were not associated with DILI based on the DisGeNET (GDA-score > 0.3) and Phenopedia databases were excluded. Finally, only gene variants whose frequency in our samples was greater than 80% were used.

### 3.2. Validation of DILI-Related Variants

DILI-associated genes were obtained from the DisGeNET and Phenopedia databases. These genes were matched against the variants we found. In order to obtain a more robust comparison, those variants that appeared in more than 80% of our samples were selected. As a result, the ABCB1, NAT2 and CYP2B6 genes were obtained ([Figure 2](#biomedicines-12-01288-f002)).

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f751/11201799/12dca959d76f/biomedicines-12-01288-g002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=11201799_biomedicines-12-01288-g002.jpg)

Venn diagram showing the genes associated with DILI from the DisGeNET and Phenopedia databases, and the genes with variations found in 80% of our samples.

### 3.3. Functional Role of NAT2 Variants in DILI

#### 3.3.1. Prediction of Function

The computational tools described in the previous sections were used to determine whether these variants had a significant effect on the protein. For this purpose, the prediction scores were calculated. The thresholds used were those recommended by the literature for each tool. Those genes whose prediction was deleterious or probably damaging in at least one of the tools were kept ([Table 2](#biomedicines-12-01288-t002)).

| Feature | Variant 1 | Variant 2 |
| --- | --- | --- |
| Gene symbol | NAT2 |  |
| Description | N-Acetyltransferase 2 |  |
| rsID | rs1799930 | rs1801280 |
| Variation name | NM_000015.3 | NM_000015.2 |
| HGVS.c | c.590G>A | c.341T>C |
| HGVS.p | p.Arg197Gln | p.Ile114Thr |
| SIFT score/pred | 0.054/T 1 | 0.048/D 2 |
| PROVEAN score/pred | −2.84/D 2 | −4.18/D 2 |
| Align GVGD | 42.81 (Class C35) 3 | 89.28 (Class C65) 3 |
| Panther | Probably damaging | Possibly damaging |
| Polyphen 2 | Probably damaging | Possibly damaging |

Table Caption: Prediction data for NAT2 variants.

#### 3.3.2. Frequency of SNPs Linked to DILI and Statistical Analysis

Once the SNP analysis results were obtained from the 43 pediatric patient samples, they were matched against the 28 controls that were previously processed. For this purpose, the NAT2 gene was searched for in the sequencing of the controls, and the possible existence of the two variants selected for this study (rs1799930 and rs1801280) was verified.

The result in the case of the controls was that of the 28 samples sequenced by WES, only one of them had the rs1799930 variant and two of them had the rs1801280 variant ([Figure 3](#biomedicines-12-01288-f003)).

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f751/11201799/f15be083a8f3/biomedicines-12-01288-g003.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=11201799_biomedicines-12-01288-g003.jpg)

Frequency data of SNPs in NAT2 in control versus DILI patients according to WES.

Considering that the results indicated that 41 of the 43 samples from pediatric patients with DILI had at least one of the identified variants and that in the case of the controls, only 3 in 25 had at least one of these variants, the result of the statistical analysis of the observed cases was significant (*p* < 0.001; OR = 0.007) for the association of the pathology with having at least one of the two NAT2 variants.

#### 3.3.3. Frequency of Variants

The most frequent variants described within the NAT2 gene were analyzed in the cases and controls, obtaining the differential frequency between patients with DILI and healthy patients. Of the 13 SNPs compared, two variants (rs1799930 and rs1801280) stand out as being 12.7 and 6.9 times more frequent in patients with DILI, respectively ([Table 3](#biomedicines-12-01288-t003)).

|   | Controls |  | Patients |  |
| --- | --- | --- | --- | --- |
| NAT2 SNP | Proportions | % | Proportions | % |
| rs1801280 | 2 | 7.1 | 21 | 48.8 |
| rs1799930 | 1 | 3.6 | 19 | 44.2 |
| rs1801279 | 6 | 21.4 | 8 | 18.6 |
| rs4986997 | 5 | 17.9 | 8 | 18.6 |
| rs1041983 | 3 | 10.7 | 6 | 14.0 |
| rs1565684 | 3 | 10.7 | 5 | 11.6 |
| rs1799929 | 2 | 7.1 | 5 | 11.6 |
| rs1208 | 1 | 3.6 | 2 | 4.7 |
| rs1799931 | 1 | 3.6 | 2 | 4.7 |
| rs4271002 | 2 | 7.1 | 2 | 4.7 |
| rs4345600 | 1 | 3.6 | 2 | 4.7 |
| rs9987109 | 1 | 3.6 | 2 | 4.7 |
| rs4986996 | 1 | 3.6 | 1 | 2.3 |

Table Caption: NAT2 gene variant describes by WES as highly frequent located in samples from controls and DILI patients.

The deleterious variants of the NAT2 gene appeared in 41 of 43 samples: in 14 samples, we identified the c.341T>C mutation; in 12 samples, we identified the c.590G>A mutation; and in 15 samples, both appeared simultaneously.

The mean population frequency of all of the SNPs analyzed in the NAT2 gene and described as variants of interest in its activity were found to have mean levels of 7.9% in the controls and 14.8% in the cases, with significant differences noted for rs1799930 and rs1801280, where their frequency associated with DILI increased by 12.3 and 6.9 times, respectively.

#### 3.3.4. Real-Time Quantitative PCR Validation

Real-time PCR of the variants of interest was used to validate the results obtained by WES.

The results shown in [Figure 4](#biomedicines-12-01288-f004) and [Table 4](#biomedicines-12-01288-t004) confirm the relationship of the genotypes for both SNPs in NAT2 with an increased risk of or predisposition to triggering a DILI process, regardless of whether these variants are found in homozygosis or heterozygosis.

![Figure 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f751/11201799/f4a7ec784b28/biomedicines-12-01288-g004.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=11201799_biomedicines-12-01288-g004.jpg)

Frequency data of SNPs in NAT2 in control versus DILI patients according to real-time qPCR.

|   | n | SNP1 Count | SNP1 % | SNP2 Count | SNP2 % | SNP1-2 Count | SNP1-2 % |
| --- | --- | --- | --- | --- | --- | --- | --- |
| Controls | 16 | 1 | 6.25 | 2 | 12.50 | 0 | 0 |
| Patients | 8 | 3 | 37.50 | 5 | 62.50 | 2 | 25.00 |

Table Caption: Genotiping frecuency of NAT2 variants in controls and DILI patients.

Genotyping confirmed the results of significantly elevated frequency in the cases versus the controls, with 8 out of 10 cases being heterozygous forms versus homozygous forms, so that the presence of one or both alleles with one or both variants present (in 25% of the cases) are equally determinant as risk factors for DILI.

## 4. Discussion

The presented work reports for the first time the possible relationship of NAT2 polymorphisms with global susceptibility to DILI in the pediatric population.

The advent of digital, electronic and molecular technologies has enabled the study of whole genomes. The integration of this information into drug development has opened the door to pharmacogenomic interventions in direct patient care. In the future, pharmacogenomics will make it possible to better identify the drug of choice and to optimize dosing regimens based on an individual’s genetic characteristics.

The rs1799930 and rs1801280 polymorphisms, present in 41 of the 43 patients studied, constitute haplotypes associated with decreased enzyme activity (slow acetylation) due to Arg197Gln and Ile114Thr protein substitutions (NAT2*6 and NAT2*5, respectively) [[13](#B13-biomedicines-12-01288),[18](#B18-biomedicines-12-01288)].

The two variants reported in this work for their implication in the possible development or triggering of DILI are known, previously described SNPs that involve an amino acid change and lead to a reduction in the acetylating activity of the enzyme. The aim of our work is to determine the causal relationship from a clinical point of view, which may open the door to deepening our understanding of the mechanism and function involved in these amino acids.

A prediction of the functional effect of amino acid substitution was carried out by the SIFT, PROVEAN, AlingGVGD, Panther and PolyPhen tools.

The resolved structure of wild-type human NAT2 (PDB 2PFR, [https://doi.org/10.2210/pdb2PFR/pdb](https://doi.org/10.2210/pdb2PFR/pdb) (accessed on 25 April 2024)) reveals the localization of both amino acids (Arg197 and Ile114) in locations that may be determinant in the functional structure and activity of the enzyme.

The amino acid changes are located at external sites of the protein structure, with possible implications in the half-life of the enzyme by favoring protein ubiquitination [[19](#B19-biomedicines-12-01288)]. Reduced NAT2 activity has been associated with increased susceptibility to certain diseases, such as systemic lupus erythematosus, some types of cancer (hepatocellular carcinoma, bladder cancer, etc.) and increased toxicity to some drugs, such as isoniazid and hydralazine [[20](#B20-biomedicines-12-01288)].

Prior to the present work, multiple studies have highlighted the importance of different NAT2 polymorphisms, especially the slow acetylator phenotype, in susceptibility to drug-induced hepatotoxicity, particularly in the context of anti-tuberculosis treatment. Specifically, two relevant systematic reviews and meta-analyses explored the relationship between NAT2 polymorphisms and drug-induced liver injury (DILI) in the context of tuberculosis treatment. The first study evaluated the potential association between NAT2 polymorphisms and DILI during anti-tuberculosis treatment. It included 37 studies with 1527 cases and 7184 controls, and found that the slow NAT2 acetylator phenotype was associated with an increased risk of DILI during anti-tuberculosis treatment. The analysis showed variability in risk among different ethnic populations, highlighting the importance of considering genetic diversity in DILI risk assessment [[20](#B20-biomedicines-12-01288),[21](#B21-biomedicines-12-01288)].

The events triggered in the pediatric cases under study were idiosyncratic, and could not be related to any specific drug treatment, especially to those that have been described as particularly relevant for metabolization by NAT2, such as isoniazid, sulfamethoxazole, sulfasalazine, hydralazine, sulphadoxine, procainamide and dapsone [[22](#B22-biomedicines-12-01288)]. The added [supplementary table](#app1-biomedicines-12-01288) provides justification for this fact.

The second analysis included 14 studies with 474 cases and 1446 controls, and also confirmed a significant association between NAT2 slow acetylators and the risk of anti-tuberculosis drug-induced liver injury. This study suggests that patients with tuberculosis who are slow acetylators have an increased risk of DILI compared with other acetylation phenotypes. This finding underscores the potential utility of assessing NAT2 polymorphisms for the clinical prediction and prevention of DILI in patients undergoing anti-tuberculosis treatment [[23](#B23-biomedicines-12-01288),[24](#B24-biomedicines-12-01288)].

The present study was performed with a broad group of drugs and was conducted in the pediatric population.

The presence of NAT2 polymorphisms, considered to be slow acetylators, has been associated not only with increased risk of DILI by anti-tuberculosis drugs, but also with increased severity of liver damage [[25](#B25-biomedicines-12-01288),[26](#B26-biomedicines-12-01288)]. Likewise, the slow-acetylator phenotype has been associated with an increased risk of DILI by other drugs, such as trimethropyrim-sulfamethoxazole, in children and adults [[27](#B27-biomedicines-12-01288),[28](#B28-biomedicines-12-01288)].

The analyses presented were performed with a broad group of drugs and were conducted in a pediatric population (see [supplementary data](#app1-biomedicines-12-01288)).

As in adults, polymorphisms in the NAT2 gene can affect the metabolism of drugs and chemicals in children. However, the effects of NAT2 polymorphisms in children may be different from those in adults due to differences in physiology and the development of the enzyme system.

NAT2 activity in children may be different from that in adults, and may vary according to age and developmental stage. While the ontogeny of phase I enzymes is well established, age-dependent maturation changes in phase II enzymes such as NAT2 are not yet well understood. The maturation of NAT2 has been proposed to be achieved at around 5.3 years of age [[29](#B29-biomedicines-12-01288)].

Human growth and physiological maturation are nonlinear processes, with discordant changes that have a direct effect on drug pharmacokinetics [[28](#B28-biomedicines-12-01288),[29](#B29-biomedicines-12-01288)]. In fact, it has become evident that even in adults, the relationships between different pharmacokinetic parameters and covariates such as age and weight are not only nonlinear, but have high-order interactions, with data areas of discontinuity in the relationships [[30](#B30-biomedicines-12-01288),[31](#B31-biomedicines-12-01288),[32](#B32-biomedicines-12-01288),[33](#B33-biomedicines-12-01288)].

In this regard, the importance of considering pharmacogenetics and enzymatic maturation in the design of therapeutic regimens for neonates, especially in low-birth-weight or preterm infants, to optimize the efficacy and safety of isoniazid treatment has been previously noted. Beranger et al. investigated the influence of the NAT2 genotype and maturation on isoniazid exposure in low-birth-weight and preterm infants with or without HIV exposure. This pharmacokinetic population analysis included 57 newborns with a mean gestational age of 34 weeks, and followed their progress for 6 months. It was observed that 90% of NAT2 maturation is reached at 4.4 months postnatally, and that the NAT2 genotype, along with weight and postmenstrual age, significantly influences isoniazid exposure. These results suggest that isoniazid dosing in low-birth-weight and preterm infants needs frequent adjustments to account for growth and maturation, and that pharmacogenetics could be key to establishing safe and effective dosing regimens [[34](#B34-biomedicines-12-01288)].

Zhu et al. examined the role of the NAT2 genotype and enzyme maturation in isoniazid pharmacokinetics in perinatally HIV-exposed South African infants participating in a randomized controlled trial. Isoniazid plasma concentration measurements of 151 infants, starting at 3–4 months of age and receiving oral doses of 10 to 20 mg/kg/day for 24 months, were analyzed. The findings showed different enzyme maturation profiles for each of the three acetylation groups, with a significant increase in apparent isoniazid clearance in the fast and intermediate accelerators from 3 to 24 months of age, but no significant change in the slow-accelerator group. This suggests that the enzymatic maturation process of NAT2 is genotype-dependent [[35](#B35-biomedicines-12-01288)].

From the above, the data presented suggest that a specific challenge in the implementation of pharmacogenomics in pediatrics is the result of maturation in pharmacogenomic phenotypes, such as variability in the maturation of drug-metabolizing enzymes and transporters, which may affect drug response. Ontogeny, or the impact of maturation on these enzymes and transporters, is an intrinsic factor in drug response that can complicate the implementation of pharmacogenomics in pediatric populations.

Our findings underscore the complexity and importance of NAT2 genetic polymorphisms in individual drug response and the risk of hepatotoxicity. Consideration of these genetic factors could be crucial for improving treatment outcomes in patients, particularly in those undergoing prolonged treatment with potentially hepatotoxic drugs.

## 5. Conclusions

In general, evaluating NAT2 polymorphisms in children through genetic testing could be used to personalize pharmacological treatment and assess the risk of medication toxicity. However, it is important to consider that interpreting genetic test results in children may differ from those in adults and may require a more careful and age-appropriate approach. As a result of this study, it can be concluded that the identified variants can be described as a risk factor for causing DILI in pediatrics.

## Supplementary Materials

The following supporting information can be downloaded at: [https://www.mdpi.com/article/10.3390/biomedicines12061288/s1](https://www.mdpi.com/article/10.3390/biomedicines12061288/s1), File S1: Table S1: Characteristics of 43 children with DILI: sex, age, race, diagnosis, drug treatment and RUMCAM score; File S2: Protein–protein interaction and biological enrichment network.

## Author Contributions

Conceptualization, E.O.-H. and M.L.A.-P.; methodology, E.O.-H. and M.L.A.-P.; software, P.M.-d.-R., I.I.-B. and F.A.-V.; validation, F.A.-V., I.I.-B. and E.O.-H.; formal analysis, E.O.-H., F.A.-V., I.I.-B. and M.L.A.-P.; investigation, E.O.-H., F.A.-V. and M.L.A.-P.; resources, E.O.-H., F.A.-V. and M.L.A.-P.; data curation, E.O.-H., F.A.-V. and I.I.-B.; writing—original draft preparation, all authors; writing—review and editing, all authors; visualization, E.O.-H., F.A.-V. and M.L.A.-P.; supervision, E.O.-H.; project administration, E.O.-H. and P.M.-d.-R.; funding acquisition, E.O.-H. and P.M.-d.-R. All authors have read and agreed to the published version of the manuscript.

## Institutional Review Board Statement

The authors are accountable for all aspects of this work and ensuring that questions related to the accuracy or integrity of any part of the work are appropriately investigated and resolved. All procedures performed in this study were in accordance with the ethical standards of the institutional and/or national research committee(s) and with the Helsinki Declaration (as revised in 2013). Ethical approval for this study was obtained from the Research Ethics Committee of Granada, Spain (Ethics permission number: 0057-M1-20).

## Informed Consent Statement

Informed consent was obtained from all subjects involved in this study.

## Data Availability Statement

The analyzed data of the present study are available to any researcher upon request to any of the authors.

## Conflicts of Interest

The authors have no conflicts of interest to declare. The authors declare that this research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest. The funders had no role in the design of the study; in the collection, analyses, or interpretation of the data; in the writing of the manuscript; or in the decision to publish the results.

## Funding Statement

This research was funded by grants from the Institute of Health Carlos III (PI17/01989) and Servicio Andaluz de Salud (F2-0071-2015 and F1-0008-2022).

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

The analyzed data of the present study are available to any researcher upon request to any of the authors.

### Supplementary Materials

### Data Availability Statement

The analyzed data of the present study are available to any researcher upon request to any of the authors.

## References

1. Andrade R.J., Lucena M.I., Fernández M.C., Pelaez G., Pachkoria K., García-Ruiz E., García-Muñoz B., González-Grande R., Pizarro A., Durán J.A., et al. Drug-induced liver injury: An analysis of 461 incidences submitted to the Spanish registry over a 10-year period. Gastroenterology. 2005;129:512–521. doi: 10.1016/j.gastro.2005.05.006.  [DOI](https://doi.org/10.1016/j.gastro.2005.05.006) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16083708/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastroenterology&title=Drug-induced%20liver%20injury:%20An%20analysis%20of%20461%20incidences%20submitted%20to%20the%20Spanish%20registry%20over%20a%2010-year%20period&author=R.J.%20Andrade&author=M.I.%20Lucena&author=M.C.%20Fern%C3%A1ndez&author=G.%20Pelaez&author=K.%20Pachkoria&volume=129&publication_year=2005&pages=512-521&pmid=16083708&doi=10.1016/j.gastro.2005.05.006&)

2. Squires R.H., Shneider B.L., Bucuvalas J., Alonso E., Sokol R.J., Narkewicz M.R., Dhawan A., Rosenthal P., Rodriguez-Baez N., Murray K.F., et al. Acute liver failure in children: The first 348 patients in the pediatric acute liver failure study group. J. Pediatr. 2006;148:652–665. doi: 10.1016/j.jpeds.2005.12.051.  [DOI](https://doi.org/10.1016/j.jpeds.2005.12.051) | [PMC free article](/articles/PMC2662127/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16737880/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Pediatr.&title=Acute%20liver%20failure%20in%20children:%20The%20first%20348%20patients%20in%20the%20pediatric%20acute%20liver%20failure%20study%20group&author=R.H.%20Squires&author=B.L.%20Shneider&author=J.%20Bucuvalas&author=E.%20Alonso&author=R.J.%20Sokol&volume=148&publication_year=2006&pages=652-665&pmid=16737880&doi=10.1016/j.jpeds.2005.12.051&)

3. Kearns G.L., Abdel-Rahman S.M., Alander S.W., Blowey D.L., Leeder J.S., Kauffman R.E. Developmental Pharmacology—Drug Disposition, Action, and Therapy in Infants and Children. N. Engl. J. Med. 2003;349:1157–1167. doi: 10.1056/NEJMra035092.  [DOI](https://doi.org/10.1056/NEJMra035092) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/13679531/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%20Engl.%20J.%20Med.&title=Developmental%20Pharmacology%E2%80%94Drug%20Disposition,%20Action,%20and%20Therapy%20in%20Infants%20and%20Children&author=G.L.%20Kearns&author=S.M.%20Abdel-Rahman&author=S.W.%20Alander&author=D.L.%20Blowey&author=J.S.%20Leeder&volume=349&publication_year=2003&pages=1157-1167&pmid=13679531&doi=10.1056/NEJMra035092&)

4. Winnike J.H., Li Z., Wright F.A., MacDonald J.M., O’Connell T.M., Watkins P.B. Use of pharmaco-metabonomics for early prediction of acetaminophen-induced hepatotoxicity in humans. Clin. Pharmacol. Ther. 2010;88:45–51. doi: 10.1038/clpt.2009.240.  [DOI](https://doi.org/10.1038/clpt.2009.240) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20182423/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Use%20of%20pharmaco-metabonomics%20for%20early%20prediction%20of%20acetaminophen-induced%20hepatotoxicity%20in%20humans&author=J.H.%20Winnike&author=Z.%20Li&author=F.A.%20Wright&author=J.M.%20MacDonald&author=T.M.%20O%E2%80%99Connell&volume=88&publication_year=2010&pages=45-51&pmid=20182423&doi=10.1038/clpt.2009.240&)

5. Bamshad M.J., Ng S.B., Bigham A.W., Tabor H.K., Emond M.J., Nickerson D.A., Shendure J. Exome sequencing as a tool for Mendelian disease gene discovery. Nat. Rev. Genet. 2011;12:745–755. doi: 10.1038/nrg3031.  [DOI](https://doi.org/10.1038/nrg3031) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21946919/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Genet.&title=Exome%20sequencing%20as%20a%20tool%20for%20Mendelian%20disease%20gene%20discovery&author=M.J.%20Bamshad&author=S.B.%20Ng&author=A.W.%20Bigham&author=H.K.%20Tabor&author=M.J.%20Emond&volume=12&publication_year=2011&pages=745-755&pmid=21946919&doi=10.1038/nrg3031&)

6. Xavier A., Scott R.J., Talseth-Palmer B.A. TAPES: A tool for assessment and prioritisation in exome studies. PLoS Comput. Biol. 2019;15:e1007453. doi: 10.1371/journal.pcbi.1007453.  [DOI](https://doi.org/10.1371/journal.pcbi.1007453) | [PMC free article](/articles/PMC6814239/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31613886/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Comput.%20Biol.&title=TAPES:%20A%20tool%20for%20assessment%20and%20prioritisation%20in%20exome%20studies&author=A.%20Xavier&author=R.J.%20Scott&author=B.A.%20Talseth-Palmer&volume=15&publication_year=2019&pages=e1007453&pmid=31613886&doi=10.1371/journal.pcbi.1007453&)

7. Zhou J., Zhang M., Li X., Wang Z., Pan D., Shi Y. Correction to: Performance comparison of four types of target enrichment baits for exome DNA sequencing. Hereditas. 2023;160:35. doi: 10.1186/s41065-023-00296-7.  [DOI](https://doi.org/10.1186/s41065-023-00296-7) | [PMC free article](/articles/PMC10481460/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/37670385/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hereditas&title=Correction%20to:%20Performance%20comparison%20of%20four%20types%20of%20target%20enrichment%20baits%20for%20exome%20DNA%20sequencing&author=J.%20Zhou&author=M.%20Zhang&author=X.%20Li&author=Z.%20Wang&author=D.%20Pan&volume=160&publication_year=2023&pages=35&pmid=37670385&doi=10.1186/s41065-023-00296-7&)

8. Hoofnagle J.H., Björnsson E.S. Drug-Induced Liver Injury—Types and Phenotypes. N. Engl. J. Med. 2019;381:264–273. doi: 10.1056/NEJMra1816149.  [DOI](https://doi.org/10.1056/NEJMra1816149) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31314970/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%20Engl.%20J.%20Med.&title=Drug-Induced%20Liver%20Injury%E2%80%94Types%20and%20Phenotypes&author=J.H.%20Hoofnagle&author=E.S.%20Bj%C3%B6rnsson&volume=381&publication_year=2019&pages=264-273&pmid=31314970&doi=10.1056/NEJMra1816149&)

9. Gutiérrez-Virgen J.E., Piña-Poza M., Hernández-Tobías E.A., Taja-Chaye L., De Lourdes López-González M., Meraz-Ríos M.A., Gomez R. NAT2 global landscape: Genetic diversity and acetylation statuses from a systematic review. PLoS ONE. 2023;18:e0283726. doi: 10.1371/journal.pone.0283726.  [DOI](https://doi.org/10.1371/journal.pone.0283726) | [PMC free article](/articles/PMC10079069/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/37023111/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&title=NAT2%20global%20landscape:%20Genetic%20diversity%20and%20acetylation%20statuses%20from%20a%20systematic%20review&author=J.E.%20Guti%C3%A9rrez-Virgen&author=M.%20Pi%C3%B1a-Poza&author=E.A.%20Hern%C3%A1ndez-Tob%C3%ADas&author=L.%20Taja-Chaye&author=M.%20De%20Lourdes%20L%C3%B3pez-Gonz%C3%A1lez&volume=18&publication_year=2023&pages=e0283726&pmid=37023111&doi=10.1371/journal.pone.0283726&)

10. Zhu K., Xu A., Xia W., Li P., Zhang B., Jiang H., Zhou S., Wang R. Association Between NAT2 Polymorphism and Lung Cancer Risk: A Systematic Review and Meta-Analysis. Front. Oncol. 2021;11:567762. doi: 10.3389/fonc.2021.567762.  [DOI](https://doi.org/10.3389/fonc.2021.567762) | [PMC free article](/articles/PMC7991837/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33777732/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front.%20Oncol.&title=Association%20Between%20NAT2%20Polymorphism%20and%20Lung%20Cancer%20Risk:%20A%20Systematic%20Review%20and%20Meta-Analysis&author=K.%20Zhu&author=A.%20Xu&author=W.%20Xia&author=P.%20Li&author=B.%20Zhang&volume=11&publication_year=2021&pages=567762&pmid=33777732&doi=10.3389/fonc.2021.567762&)

11. Nakai T., Sakai D., Nakamura Y., Horikita N., Matsushita E., Naiki M., Watanabe M. Association of NAT2 genetic polymorphism with the efficacy of Neurotropin® for the enhancement of aggrecan gene expression in nucleus pulposus cells: A pilot study. BMC Med. Genom. 2021;14:79. doi: 10.1186/s12920-021-00926-x.  [DOI](https://doi.org/10.1186/s12920-021-00926-x) | [PMC free article](/articles/PMC7948325/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33706752/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Med.%20Genom.&title=Association%20of%20NAT2%20genetic%20polymorphism%20with%20the%20efficacy%20of%20Neurotropin%C2%AE%20for%20the%20enhancement%20of%20aggrecan%20gene%20expression%20in%20nucleus%20pulposus%20cells:%20A%20pilot%20study&author=T.%20Nakai&author=D.%20Sakai&author=Y.%20Nakamura&author=N.%20Horikita&author=E.%20Matsushita&volume=14&publication_year=2021&pages=79&pmid=33706752&doi=10.1186/s12920-021-00926-x&)

12. Matejcic M., Vogelsang M., Wang Y., Parker I.M. NAT1 and NAT2 genetic polymorphisms and environmental exposure as risk factors for oesophageal squamous cell carcinoma: A case-control study. BMC Cancer. 2015;15:150. doi: 10.1186/s12885-015-1105-4.  [DOI](https://doi.org/10.1186/s12885-015-1105-4) | [PMC free article](/articles/PMC4379954/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25886288/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Cancer&title=NAT1%20and%20NAT2%20genetic%20polymorphisms%20and%20environmental%20exposure%20as%20risk%20factors%20for%20oesophageal%20squamous%20cell%20carcinoma:%20A%20case-control%20study&author=M.%20Matejcic&author=M.%20Vogelsang&author=Y.%20Wang&author=I.M.%20Parker&volume=15&publication_year=2015&pages=150&pmid=25886288&doi=10.1186/s12885-015-1105-4&)

13. Tiis R.P., Osipova L.P., Lichman D.V., Voronina E.N., Filipenko M.L. Studying polymorphic variants of the NAT2 gene (NAT2*5 and NAT2*7) in Nenets populations of Northern Siberia. BMC Genet. 2020;21:115. doi: 10.1186/s12863-020-00909-4.  [DOI](https://doi.org/10.1186/s12863-020-00909-4) | [PMC free article](/articles/PMC7583309/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33092525/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Genet.&title=Studying%20polymorphic%20variants%20of%20the%20NAT2%20gene%20(NAT2*5%20and%20NAT2*7)%20in%20Nenets%20populations%20of%20Northern%20Siberia&author=R.P.%20Tiis&author=L.P.%20Osipova&author=D.V.%20Lichman&author=E.N.%20Voronina&author=M.L.%20Filipenko&volume=21&publication_year=2020&pages=115&pmid=33092525&doi=10.1186/s12863-020-00909-4&)

14. Khan S., Mandal R.K., Elasbali A.M., Dar S.A., Jawed A., Wahid M., Mahto H., Lohani M., Mishra B.N., Akhter N., et al. Pharmacogenetic association between NAT2 gene polymorphisms and isoniazid induced hepatotoxicity: Trial sequence meta-analysis as evidence. Biosci. Rep. 2019;39:BSR20180845. doi: 10.1042/BSR20180845.  [DOI](https://doi.org/10.1042/BSR20180845) | [PMC free article](/articles/PMC6331676/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30509962/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biosci.%20Rep.&title=Pharmacogenetic%20association%20between%20NAT2%20gene%20polymorphisms%20and%20isoniazid%20induced%20hepatotoxicity:%20Trial%20sequence%20meta-analysis%20as%20evidence&author=S.%20Khan&author=R.K.%20Mandal&author=A.M.%20Elasbali&author=S.A.%20Dar&author=A.%20Jawed&volume=39&publication_year=2019&pages=BSR20180845&pmid=30509962&doi=10.1042/BSR20180845&)

15. Megaraj V., Zhao T., Paumi C.M., Gerk P.M., Kim R.B., Vore M. Functional analysis of nonsynonymous single nucleotide polymorphisms of multidrug resistance-associated protein 2 (ABCC2) Pharmacogenet. Genom. 2011;21:506–515. doi: 10.1097/FPC.0b013e328348c786.  [DOI](https://doi.org/10.1097/FPC.0b013e328348c786) | [PMC free article](/articles/PMC3640355/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21691255/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet.%20Genom.&title=Functional%20analysis%20of%20nonsynonymous%20single%20nucleotide%20polymorphisms%20of%20multidrug%20resistance-associated%20protein%202%20(ABCC2)&author=V.%20Megaraj&author=T.%20Zhao&author=C.M.%20Paumi&author=P.M.%20Gerk&author=R.B.%20Kim&volume=21&publication_year=2011&pages=506-515&pmid=21691255&doi=10.1097/FPC.0b013e328348c786&)

16. Ocete Hita E., Martín García J.A., Giménez Sánchez F., Flores González J.C., Abril Molina A., Salmerón Escobar J., Ruiz Extremera A. Hepatotoxicity due to drugs or natural products in children. An. Pediatría. 2013;78:248–259. doi: 10.1016/j.anpedi.2012.06.012.  [DOI](https://doi.org/10.1016/j.anpedi.2012.06.012) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23017742/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=An.%20Pediatr%C3%ADa&title=Hepatotoxicity%20due%20to%20drugs%20or%20natural%20products%20in%20children&author=E.%20Ocete%20Hita&author=J.A.%20Mart%C3%ADn%20Garc%C3%ADa&author=F.%20Gim%C3%A9nez%20S%C3%A1nchez&author=J.C.%20Flores%20Gonz%C3%A1lez&author=A.%20Abril%20Molina&volume=78&publication_year=2013&pages=248-259&pmid=23017742&doi=10.1016/j.anpedi.2012.06.012&)

17. Danan G., Teschke R. RUCAM in Drug and Herb Induced Liver Injury: The Update. Int. J. Mol. Sci. 2015;17:14. doi: 10.3390/ijms17010014.  [DOI](https://doi.org/10.3390/ijms17010014) | [PMC free article](/articles/PMC4730261/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26712744/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Mol.%20Sci.&title=RUCAM%20in%20Drug%20and%20Herb%20Induced%20Liver%20Injury:%20The%20Update&author=G.%20Danan&author=R.%20Teschke&volume=17&publication_year=2015&pages=14&pmid=26712744&doi=10.3390/ijms17010014&)

18. Stanley L.A., Sim E. Update on the pharmacogenetics of NATs: Structural considerations. Pharmacogenomics. 2008;9:1673–1693. doi: 10.2217/14622416.9.11.1673.  [DOI](https://doi.org/10.2217/14622416.9.11.1673) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19018723/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Update%20on%20the%20pharmacogenetics%20of%20NATs:%20Structural%20considerations&author=L.A.%20Stanley&author=E.%20Sim&volume=9&publication_year=2008&pages=1673-1693&pmid=19018723&doi=10.2217/14622416.9.11.1673&)

19. Wu H., Dombrovsky L., Tempel W., Martin F., Loppnau P., Goodfellow G.H., Grant D.M., Plotnikov A.N. Structural basis of substrate-binding specificity of human arylamine N-acetyltransferases. J. Biol. Chem. 2007;282:30189–30197. doi: 10.1074/jbc.M704138200.  [DOI](https://doi.org/10.1074/jbc.M704138200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17656365/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem.&title=Structural%20basis%20of%20substrate-binding%20specificity%20of%20human%20arylamine%20N-acetyltransferases&author=H.%20Wu&author=L.%20Dombrovsky&author=W.%20Tempel&author=F.%20Martin&author=P.%20Loppnau&volume=282&publication_year=2007&pages=30189-30197&pmid=17656365&doi=10.1074/jbc.M704138200&)

20. Sim E., Fakis G., Laurieri N., Boukouvala S. Arylamine N-Acetyltransferases—From Drug Metabolism and Pharmacogenetics to Identification of Novel Targets for Pharmacological Intervention. Adv. Pharmacol. 2012;63:169–205. doi: 10.1016/B978-0-12-398339-8.00005-7.  [DOI](https://doi.org/10.1016/B978-0-12-398339-8.00005-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22776642/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Adv.%20Pharmacol.&title=Arylamine%20N-Acetyltransferases%E2%80%94From%20Drug%20Metabolism%20and%20Pharmacogenetics%20to%20Identification%20of%20Novel%20Targets%20for%20Pharmacological%20Intervention&author=E.%20Sim&author=G.%20Fakis&author=N.%20Laurieri&author=S.%20Boukouvala&volume=63&publication_year=2012&pages=169-205&pmid=22776642&doi=10.1016/B978-0-12-398339-8.00005-7&)

21. Chorfi L., Fercha A., Derouiche F., Sebihi F.Z., Houhou D., Chorfi K., Bendjemana K. N-Acetyltransferase 2, glutathione S-transferase gene polymorphisms and susceptibility to hepatocellular carcinoma in an Algerian population. Xenobiotica. 2022;52:99–104. doi: 10.1080/00498254.2022.2040642.  [DOI](https://doi.org/10.1080/00498254.2022.2040642) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35138223/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Xenobiotica&title=N-Acetyltransferase%202,%20glutathione%20S-transferase%20gene%20polymorphisms%20and%20susceptibility%20to%20hepatocellular%20carcinoma%20in%20an%20Algerian%20population&author=L.%20Chorfi&author=A.%20Fercha&author=F.%20Derouiche&author=F.Z.%20Sebihi&author=D.%20Houhou&volume=52&publication_year=2022&pages=99-104&pmid=35138223&doi=10.1080/00498254.2022.2040642&)

22. McDonagh E.M., Boukouvala S., Aklillu E., Hein D.W., Altman R.B., Klein T.E. PharmGKB summary: Very important pharmacogene information for N-acetyltransferase 2. Pharmacogenet. Genom. 2014;24:409–425. doi: 10.1097/FPC.0000000000000062.  [DOI](https://doi.org/10.1097/FPC.0000000000000062) | [PMC free article](/articles/PMC4109976/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24892773/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet.%20Genom.&title=PharmGKB%20summary:%20Very%20important%20pharmacogene%20information%20for%20N-acetyltransferase%202&author=E.M.%20McDonagh&author=S.%20Boukouvala&author=E.%20Aklillu&author=D.W.%20Hein&author=R.B.%20Altman&volume=24&publication_year=2014&pages=409-425&pmid=24892773&doi=10.1097/FPC.0000000000000062&)

23. Shi J., Xie M., Wang J., Xu Y., Liu X. Susceptibility of N-acetyltransferase 2 slow acetylators to antituberculosis drug-induced liver injury: A meta-analysis. Pharmacogenomics. 2015;16:2083–2097. doi: 10.2217/pgs.15.144.  [DOI](https://doi.org/10.2217/pgs.15.144) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26616266/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Susceptibility%20of%20N-acetyltransferase%202%20slow%20acetylators%20to%20antituberculosis%20drug-induced%20liver%20injury:%20A%20meta-analysis&author=J.%20Shi&author=M.%20Xie&author=J.%20Wang&author=Y.%20Xu&author=X.%20Liu&volume=16&publication_year=2015&pages=2083-2097&pmid=26616266&doi=10.2217/pgs.15.144&)

24. Wang P.Y., Xie S.Y., Hao Q., Zhang C., Jiang B.F. NAT2 polymorphisms and susceptibility to anti-tuberculosis drug-induced liver injury: A meta-analysis. Int. J. Tuberc. Lung Dis. 2012;16:589–595. doi: 10.5588/ijtld.11.0377.  [DOI](https://doi.org/10.5588/ijtld.11.0377) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22409928/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Tuberc.%20Lung%20Dis.&title=NAT2%20polymorphisms%20and%20susceptibility%20to%20anti-tuberculosis%20drug-induced%20liver%20injury:%20A%20meta-analysis&author=P.Y.%20Wang&author=S.Y.%20Xie&author=Q.%20Hao&author=C.%20Zhang&author=B.F.%20Jiang&volume=16&publication_year=2012&pages=589-595&pmid=22409928&doi=10.5588/ijtld.11.0377&)

25. Yuliwulandari R., Prayuni K., Susilowati R.W., Subagyo S., Soedarsono S., M Sofro A.S., Tokunaga K., Shin J.-G. NAT2 slow acetylator is associated with anti-tuberculosis drug-induced liver injury severity in indonesian population. Pharmacogenomics. 2019;20:1303–1310. doi: 10.2217/pgs-2019-0131.  [DOI](https://doi.org/10.2217/pgs-2019-0131) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31699005/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=NAT2%20slow%20acetylator%20is%20associated%20with%20anti-tuberculosis%20drug-induced%20liver%20injury%20severity%20in%20indonesian%20population&author=R.%20Yuliwulandari&author=K.%20Prayuni&author=R.W.%20Susilowati&author=S.%20Subagyo&author=S.%20Soedarsono&volume=20&publication_year=2019&pages=1303-1310&pmid=31699005&doi=10.2217/pgs-2019-0131&)

26. Cheng F., Jiang X.G., Zheng S.L., Wu T., Zhang Q., Ye X.C., Liu S., Shi J.C. N-acetyltransferase 2 genetic polymorphisms and anti-tuberculosis-drug-induced liver injury: A correlation study. Front. Pharmacol. 2023;14:1171353. doi: 10.3389/fphar.2023.1171353.  [DOI](https://doi.org/10.3389/fphar.2023.1171353) | [PMC free article](/articles/PMC10501134/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/37719844/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front.%20Pharmacol.&title=N-acetyltransferase%202%20genetic%20polymorphisms%20and%20anti-tuberculosis-drug-induced%20liver%20injury:%20A%20correlation%20study&author=F.%20Cheng&author=X.G.%20Jiang&author=S.L.%20Zheng&author=T.%20Wu&author=Q.%20Zhang&volume=14&publication_year=2023&pages=1171353&pmid=37719844&doi=10.3389/fphar.2023.1171353&)

27. Huang Y.S., Tseng S.Y., Chang T.E., Perng C.L., Huang Y.H. Sulfamethoxazole-trimethoprim-induced liver injury and genetic polymorphisms of NAT2 and CYP2C9 in Taiwan. Pharmacogenet. Genom. 2021;31:200–206. doi: 10.1097/FPC.0000000000000441.  [DOI](https://doi.org/10.1097/FPC.0000000000000441) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34149005/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet.%20Genom.&title=Sulfamethoxazole-trimethoprim-induced%20liver%20injury%20and%20genetic%20polymorphisms%20of%20NAT2%20and%20CYP2C9%20in%20Taiwan&author=Y.S.%20Huang&author=S.Y.%20Tseng&author=T.E.%20Chang&author=C.L.%20Perng&author=Y.H.%20Huang&volume=31&publication_year=2021&pages=200-206&pmid=34149005&doi=10.1097/FPC.0000000000000441&)

28. Zielińska E., Niewiarowski W., Bodalski J., Rȩbowski G., Skrȩtkowicz J., Mianowska K., Sekulska M. Genotyping of the arylamine N-acetyltransferase polymorphism in the prediction of idiosyncratic reactions to trimethoprim-sulfamethoxazole in infants. Pharm. World Sci. 1998;20:123–130. doi: 10.1023/A:1008664707825.  [DOI](https://doi.org/10.1023/A:1008664707825) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9618736/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharm.%20World%20Sci.&title=Genotyping%20of%20the%20arylamine%20N-acetyltransferase%20polymorphism%20in%20the%20prediction%20of%20idiosyncratic%20reactions%20to%20trimethoprim-sulfamethoxazole%20in%20infants&author=E.%20Zieli%C5%84ska&author=W.%20Niewiarowski&author=J.%20Bodalski&author=G.%20R%C8%A9bowski&author=J.%20Skr%C8%A9tkowicz&volume=20&publication_year=1998&pages=123-130&pmid=9618736&doi=10.1023/A:1008664707825&)

29. Rogers Z., Hiruy H., Pasipanodya J.G., Mbowane C., Adamson J., Ngotho L., Karim F., Jeena P., Bishai W., Gumbo T. The Non-Linear Child: Ontogeny, Isoniazid Concentration, and NAT2 Genotype Modulate Enzyme Reaction Kinetics and Metabolism. EBioMedicine. 2016;11:118. doi: 10.1016/j.ebiom.2016.07.031.  [DOI](https://doi.org/10.1016/j.ebiom.2016.07.031) | [PMC free article](/articles/PMC5049930/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27528266/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=EBioMedicine&title=The%20Non-Linear%20Child:%20Ontogeny,%20Isoniazid%20Concentration,%20and%20NAT2%20Genotype%20Modulate%20Enzyme%20Reaction%20Kinetics%20and%20Metabolism&author=Z.%20Rogers&author=H.%20Hiruy&author=J.G.%20Pasipanodya&author=C.%20Mbowane&author=J.%20Adamson&volume=11&publication_year=2016&pages=118&pmid=27528266&doi=10.1016/j.ebiom.2016.07.031&)

30. Chung H., Oh J., Yoon S.H., Yu K.S., Cho J.Y., Chung J.Y. A non-linear pharmacokinetic-pharmacodynamic relationship of metformin in healthy volunteers: An open-label, parallel group, randomized clinical study. PLoS ONE. 2018;13:e0191258. doi: 10.1371/journal.pone.0191258.  [DOI](https://doi.org/10.1371/journal.pone.0191258) | [PMC free article](/articles/PMC5771593/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29342199/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&title=A%20non-linear%20pharmacokinetic-pharmacodynamic%20relationship%20of%20metformin%20in%20healthy%20volunteers:%20An%20open-label,%20parallel%20group,%20randomized%20clinical%20study&author=H.%20Chung&author=J.%20Oh&author=S.H.%20Yoon&author=K.S.%20Yu&author=J.Y.%20Cho&volume=13&publication_year=2018&pages=e0191258&pmid=29342199&doi=10.1371/journal.pone.0191258&)

31. Hall R.G., Swancutt M.A., Gumbo T. Fractal geometry and the pharmacometrics of micafungin in overweight, obese, and extremely obese people. Antimicrob. Agents Chemother. 2011;55:5107–5112. doi: 10.1128/AAC.05193-11.  [DOI](https://doi.org/10.1128/AAC.05193-11) | [PMC free article](/articles/PMC3195005/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21876061/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Antimicrob.%20Agents%20Chemother.&title=Fractal%20geometry%20and%20the%20pharmacometrics%20of%20micafungin%20in%20overweight,%20obese,%20and%20extremely%20obese%20people&author=R.G.%20Hall&author=M.A.%20Swancutt&author=T.%20Gumbo&volume=55&publication_year=2011&pages=5107-5112&pmid=21876061&doi=10.1128/AAC.05193-11&)

32. Hall R.G., Swancutt M.A., Meek C., Leff R., Gumbo T. Weight drives caspofungin pharmacokinetic variability in overweight and obese people: Fractal power signatures beyond two-thirds or three-fourths. Antimicrob. Agents Chemother. 2013;57:2259–2264. doi: 10.1128/AAC.01490-12.  [DOI](https://doi.org/10.1128/AAC.01490-12) | [PMC free article](/articles/PMC3632956/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23459494/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Antimicrob.%20Agents%20Chemother.&title=Weight%20drives%20caspofungin%20pharmacokinetic%20variability%20in%20overweight%20and%20obese%20people:%20Fractal%20power%20signatures%20beyond%20two-thirds%20or%20three-fourths&author=R.G.%20Hall&author=M.A.%20Swancutt&author=C.%20Meek&author=R.%20Leff&author=T.%20Gumbo&volume=57&publication_year=2013&pages=2259-2264&pmid=23459494&doi=10.1128/AAC.01490-12&)

33. Jain M.K., Pasipanodya J.G., Alder L., Lee W.M., Gumbo T. Pegylated interferon fractal pharmacokinetics: Individualized dosing for hepatitis C virus infection. Antimicrob. Agents Chemother. 2013;57:1115–1120. doi: 10.1128/AAC.02208-12.  [DOI](https://doi.org/10.1128/AAC.02208-12) | [PMC free article](/articles/PMC3591896/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23183434/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Antimicrob.%20Agents%20Chemother.&title=Pegylated%20interferon%20fractal%20pharmacokinetics:%20Individualized%20dosing%20for%20hepatitis%20C%20virus%20infection&author=M.K.%20Jain&author=J.G.%20Pasipanodya&author=L.%20Alder&author=W.M.%20Lee&author=T.%20Gumbo&volume=57&publication_year=2013&pages=1115-1120&pmid=23183434&doi=10.1128/AAC.02208-12&)

34. Beranger A., Bekker A., Solans B.P., Cotton M.F., Mirochnick M., Violari A., Wang J., Cababasay M., Wiesner L., Browning R., et al. Influence of NAT2 Genotype and Maturation on Isoniazid Exposure in Low-Birth-Weight and Preterm Infants with or Without Human Immunodeficiency Virus (HIV) Exposure. Clin. Infect. Dis. 2022;75:1037–1045. doi: 10.1093/cid/ciac001.  [DOI](https://doi.org/10.1093/cid/ciac001) | [PMC free article](/articles/PMC9522418/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35134861/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Infect.%20Dis.&title=Influence%20of%20NAT2%20Genotype%20and%20Maturation%20on%20Isoniazid%20Exposure%20in%20Low-Birth-Weight%20and%20Preterm%20Infants%20with%20or%20Without%20Human%20Immunodeficiency%20Virus%20(HIV)%20Exposure&author=A.%20Beranger&author=A.%20Bekker&author=B.P.%20Solans&author=M.F.%20Cotton&author=M.%20Mirochnick&volume=75&publication_year=2022&pages=1037-1045&pmid=35134861&doi=10.1093/cid/ciac001&)

35. Zhu R., Kiser J.J., Seifart H.I., Werely C.J., Mitchell C.D., D’Argenio D.Z., Fletcher C.V. The pharmacogenetics of NAT2 enzyme maturation in perinatally HIV exposed infants receiving isoniazid. J. Clin. Pharmacol. 2012;52:511–519. doi: 10.1177/0091270011402826.  [DOI](https://doi.org/10.1177/0091270011402826) | [PMC free article](/articles/PMC3182303/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21558457/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Pharmacol.&title=The%20pharmacogenetics%20of%20NAT2%20enzyme%20maturation%20in%20perinatally%20HIV%20exposed%20infants%20receiving%20isoniazid&author=R.%20Zhu&author=J.J.%20Kiser&author=H.I.%20Seifart&author=C.J.%20Werely&author=C.D.%20Mitchell&volume=52&publication_year=2012&pages=511-519&pmid=21558457&doi=10.1177/0091270011402826&)
