# Impact of CYP2D6*2, CYP2D6*35, rs5758550, and related haplotypes on risperidone clearance in vivo

## Metadata
**Authors:** Elisabet Størset, Line Skute Bråten, Magnus Ingelman-Sundberg, Inger Johansson, Espen Molden, Marianne Kristiansen Kringen
**Journal:** European Journal of Clinical Pharmacology
**Date:** 2024 Jul 4
**DOI:** [10.1007/s00228-024-03721-6](https://doi.org/10.1007/s00228-024-03721-6)
**PMID:** 38963454
**PMCID:** PMC11393095
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11393095/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC11393095/pdf/228_2024_Article_3721.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC11393095/pdf/228_2024_Article_3721.pdf)

## Abstract

**Purpose:** 
The CYP2D6 gene exhibits significant polymorphism, contributing to variability in responses to drugs metabolized by CYP2D6. While CYP2D6*2 and CYP2D6*35 are presently designated as alleles encoding normal metabolism, this classification is based on moderate level evidence. Additionally, the role of the formerly called “enhancer” single nucleotide polymorphism (SNP) rs5758550 is unclear. In this study, the impacts of CYP2D6*2, CYP2D6*35 and rs5758550 on CYP2D6 activity were investigated using risperidone clearance as CYP2D6 activity marker.

**Methods:** 
A joint parent-metabolite population pharmacokinetic model was used to describe 1,565 serum concentration measurements of risperidone and 9-hydroxyrisperidone in 512 subjects. Risperidone population clearance was modeled as the sum of a CYP2D6-independent clearance term and the partial clearances contributed from each individually expressed CYP2D6 allele or haplotype. In addition to the well-characterized CYP2D6 alleles (*3-*6, *9, *10 and *41), *2, *35 and two haplotypes assigned as CYP2D6*2-rs5758550G and CYP2D6*2-rs5758550A were evaluated.

**Results:** 
Each evaluated CYP2D6 allele was associated with significantly lower risperidone clearance than the reference normal function allele CYP2D6*1 (p < 0.001). Further, rs5758550 differentiated the effect of CYP2D6*2 (p = 0.005). The haplotype-specific clearances for CYP2D6*2-rs5758550A, CYP2D6*2-rs5758550G and CYP2D6*35 were estimated to 30%, 66% and 57%, respectively, relative to the clearance for CYP2D6*1. Notably, rs5758550 is in high linkage disequilibrium (R2 > 0.85) with at least 24 other SNPs and cannot be assigned as a functional SNP.

**Conclusion:** 
CYP2D6*2 and CYP2D6*35 encode reduced risperidone clearance, and the extent of reduction for CYP2D6*2 is differentiated by rs5758550. Genotyping of these haplotypes might improve the precision of genotype-guided prediction of CYP2D6-mediated clearance.

**Supplementary Information:** 
The online version contains supplementary material available at 10.1007/s00228-024-03721-6.

Keywords: Pharmacogenomics, CYP haplotypes, Pharmacokinetic modelling, Precision dosing

### Purpose

The *CYP2D6* gene exhibits significant polymorphism, contributing to variability in responses to drugs metabolized by CYP2D6. While *CYP2D6*2* and *CYP2D6*35* are presently designated as alleles encoding normal metabolism, this classification is based on moderate level evidence. Additionally, the role of the formerly called “enhancer” single nucleotide polymorphism (SNP) rs5758550 is unclear. In this study, the impacts of *CYP2D6*2, CYP2D6*35* and rs5758550 on CYP2D6 activity were investigated using risperidone clearance as CYP2D6 activity marker.

### Methods

A joint parent-metabolite population pharmacokinetic model was used to describe 1,565 serum concentration measurements of risperidone and 9-hydroxyrisperidone in 512 subjects. Risperidone population clearance was modeled as the sum of a CYP2D6-independent clearance term and the partial clearances contributed from each individually expressed *CYP2D6* allele or haplotype. In addition to the well-characterized *CYP2D6* alleles (**3-*6*, **9*, **10* and **41*), **2*, **35* and two haplotypes assigned as *CYP2D6*2-rs5758550G* and *CYP2D6*2-rs5758550A* were evaluated.

### Results

Each evaluated *CYP2D6* allele was associated with significantly lower risperidone clearance than the reference normal function allele *CYP2D6*1* (p < 0.001). Further, rs5758550 differentiated the effect of *CYP2D6*2* (p = 0.005). The haplotype-specific clearances for *CYP2D6*2-rs5758550A*, *CYP2D6*2-rs5758550G* and *CYP2D6*35* were estimated to 30%, 66% and 57%, respectively, relative to the clearance for *CYP2D6*1*. Notably, rs5758550 is in high linkage disequilibrium (R^2^ > 0.85) with at least 24 other SNPs and cannot be assigned as a functional SNP.

### Conclusion

*CYP2D6*2* and *CYP2D6*35* encode reduced risperidone clearance, and the extent of reduction for *CYP2D6*2* is differentiated by rs5758550*.* Genotyping of these haplotypes might improve the precision of genotype-guided prediction of CYP2D6-mediated clearance.

### Supplementary Information

The online version contains supplementary material available at 10.1007/s00228-024-03721-6.

## Introduction

Cytochrome P450 2D6 (CYP2D6) is involved in the biotransformation of about 25% of clinically used drugs [[1](#CR1)]. CYP2D6 activity varies considerably between subjects, leading to substantial differences in systemic exposure and clinical response to standard doses of drugs metabolized by CYP2D6 [[2](#CR2)]. Genetic polymorphism is one of the main sources of interindividual differences in CYP2D6 activity. Many of the genetic variant alleles have been found to encode increased, reduced, or deficient CYP2D6 activity and are included in genotyping panels to predict individual dose requirements [[3](#CR3)]. However, the functional impact of many of these variants or haplotypes on CYP2D6 activity remains unclear. A better understanding of both known and novel variants or haplotypes affecting CYP2D6 activity would improve genotype-guided dosing and potentially clinical outcomes in patients treated with CYP2D6-metabolized drugs.

*CYP2D6*2* and *CYP2D6*35* are currently classified as normal functional alleles with similar effects on enzyme activity as *CYP2D6*1* [[3](#CR3), [4](#CR4)]. However, there are several studies on various CYP2D6 substrates (including tamoxifen, dextromethorphan, carvedilol, and brexpiprazole) suggesting that *CYP2D6*2* is associated with reduced CYP2D6 activity [[5](#CR5)–[9](#CR9)], which calls into question the currently proposed functional assignment of *CYP2D6*2*. However, no such effect was observed with other CYP2D6 substrates such as vortioxetine and tedatioxetine [[10](#CR10), [11](#CR11)], which may indicate that *CYP2D6*2* has a substrate-dependent effect on enzyme activity as also indicated from previous in vitro studies [[12](#CR12)]. The fact that *CYP2D6*2* is a Tier 1 allele recommended for routine clinical testing [[13](#CR13)] emphasizes the need for further studies to ensure correct functional annotation.

Previously, much attention has been taken to the role of the so-called “enhancer” single nucleotide polymorphism (SNP) rs5758550 G>A ~ 115 kb downstream of the *CYP2D6* gene in regulating *CYP2D6* gene expression [[14](#CR14)–[17](#CR17)], particularly for *CYP2D6*2* alleles. The* G* variant of this SNP has been hypothesized to increase *CYP2D6* transcription and contribute to interindividual variability in CYP2D6 activity, but other research groups have not succeeded in reproducing the “enhancer” effect [[18](#CR18), [19](#CR19)]. In a recent review of in vivo studies on the rs5758550 variant, it was concluded that there is currently insufficient evidence to support the use of rs5758550 in the clinic due to the methodological caveats of the studies and limited understanding of underlying mechanism [[17](#CR17)].

Regarding *CYP2D6*35*, the assigned classification of normal function is based on moderate level evidence from a limited number of studies [[4](#CR4)]. It was originally hypothesized that this variant encodes for increased enzyme activity [[20](#CR20)], but these results could not be reproduced in vitro and in vivo. More recent results have been inconsistent as to whether *CYP2D6*35* encodes for normal [[6](#CR6), [21](#CR21)] or even reduced enzyme activity [[8](#CR8), [12](#CR12), [22](#CR22)].

To obtain a measure of CYP2D6-mediated clearance in the present study, we used the population pharmacokinetic modelling approach on a large sample of parent-metabolite serum concentration data for risperidone, which is metabolized primarily by CYP2D6 with a rate known to be dependent on *CYP2D6* genotype [[23](#CR23), [24](#CR24)]. The aim of the study was to quantify the influence of several genetic variants or haplotypes on CYP2D6-mediated clearance, with a specific focus on *CYP2D6*2*, *CYP2D6*35*, and rs5758550.

## Methods

### Subjects and data

The included subjects were above the age of 18 years with known *CYP2D6* genotype and a history of therapeutic drug monitoring (TDM) of risperidone and 9-hydroxyrisperidone serum concentrations following oral administration of risperidone. Data were collected from routine clinical analyses of patient samples performed at the Center for Psychopharmacology, Diakonhjemmet Hospital (Oslo, Norway) between 2005 and 2022.

### Dosing regimen and pharmacokinetic sampling

The TDM requisition forms filled out by the treating physician specified information on dosing history and time between last dose and blood sampling. Samples were considered only if this information was clearly provided. The requisition form suggests blood sampling 12 to 24 h after the last dose as recommended for antipsychotics [[25](#CR25)]. To increase the sample size, samples taken between 9 and 30 h after the last dose were eligible for inclusion. Samples were included regardless of dose and dosing frequency (i.e., once- or twice-daily risperidone) as the individual dosing history was specified in the dataset for pharmacokinetic analysis. The TDM samples were assumed to be taken at steady state unless otherwise recorded on the requisition forms.

Samples were excluded if the requisition form indicated concomitant use of the CYP2D6-inhibiting drugs paroxetine, fluoxetine, or bupropion or the CYP3A-inducing drugs phenobarbital, phenytoin, or carbamazepine, or if long-acting formulations of risperidone or 9-hydroxyrisperidone (paliperidone) had been injected during the last 9 months.

### Serum concentration analysis

Serum concentration analysis of risperidone and 9-hydroxyrisperidone was carried out as part of the routine therapeutic drug monitoring service at the Center for Psychopharmacology. The method was based on ultrahigh-performance liquid chromatography high-resolution mass spectrometry (UHPLC-HRMS). Briefly, serum samples were purified by protein precipitation mixing 200 μL of serum aliquot with 400 μL of acetonitrile-methanol (90/10 vol/vol), which included the isotope-labelled internal standards. Following centrifugation, the supernatants were diluted 1:1 with ultrapure water, and 4 μL of purified sample was then injected into a Vanquish Binary UHPLC system coupled to a Q Exactive Orbitrap HRAM MS with electrospray ionization operated in positive ionization mode (Thermo Scientific, Waltham, MS, USA). Chromatographic separation was performed on a XBridge BEH C18 column (2.5 μm, 2.1 × 75 mm; Waters). The mobile phase gradient comprised a mixture of acetonitrile and ammonium acetate buffer (pH 4.8).

All the calibration curves were linear (*R*^2^ > 0.99) in validated ranges: risperidone, 1–200 nmol/L; 9OH-risperidone, 2.5–300 nmol/L. Imprecision and inaccuracy parameters of the assays were lower than 15%. The lower limit of detection was 0.4 nM for risperidone and 1 nM for 9-hydroxyrisperidone, and concentrations above this limit were reported and available for pharmacokinetic modelling analysis. During population modelling, values below the detection limit were handled by imputing the half of the lower limit of detection due to a proportion of < 10% [[26](#CR26), [27](#CR27)].

### Pharmacogenetic analyses

Genotyping of *CYP2D6* had previously been performed by TaqMan-based real-time PCR assays implemented for routine pharmacogenetic analysis at the Center for Psychopharmacology. The routine panel for *CYP2D6* genotyping included the null alleles *CYP2D6*3* (rs35742686), *CYP2D6*4* (rs3892097), *CYP2D6*5* (whole gene deletion), and *CYP2D6*6* (rs5030655) and the reduced function alleles *CYP2D6*9* (rs5030656), *CYP2D6*10* (rs1065852), and *CYP2D6*41* (rs28371725), as well as copy number analysis to identify multiplication of the *CYP2D6* gene giving rise to ultrarapid metabolism.

In addition to *CYP2D6* variant alleles, other gene variants or haplotypes may also affect risperidone clearance, including a variant in the gene encoding the nuclear factor I B (NFIB), which has been shown to regulate the expression of the *CYP2D6* gene [[28](#CR28), [29](#CR29)], and *CYP3A4*22,* which may alter CYP3A4-mediated risperidone clearance [[30](#CR30)].

To get a better coverage of variant alleles that may affect risperidone clearance, the DNA samples were reanalyzed with predesigned TaqMan-based real-time PCR assays (Thermo Fisher Scientific, Waltham, MA, USA) to detect *CYP2D6*2* (rs16947; C__27102425_10), *CYP2D6*35* (rs769258; C__7102444_F0), rs5758550 (C_29692254_10), *NFIB* (rs28379954; C_59359617_10), and *CYP3A4*22* (rs35599367; C__59013445_10). Subjects carrying *CYP2D6*2* and rs5758550 *G* and *A* were defined as carriers of haplotypes *CYP2D6*2-rs5758550G* and *CYP2D6*2-rs5758550A*, respectively.

### Exploratory data analysis

Initially, the metabolic ratios (MR) between the measured 9-hydroxyrisperidone and risperidone concentrations were calculated at each observation and considered a raw data-based approximation of enzyme activity. The relationship between the various CYP2D6 diplotypes and the median MR for each subject was visually explored using box plots and by pairwise comparisons using the Mann–Whitney *U* test. Any apparent relationship between CYP2D6 allele or haplotype and risperidone clearance was then quantified using population pharmacokinetic modelling of all data observations.

### Population pharmacokinetic modelling

The time courses of risperidone and 9-hydroxyrisperidone serum concentrations were analyzed using population pharmacokinetic modelling (i.e., non-linear mixed effects modelling). The model was developed to simultaneously describe the pharmacokinetics of the parent and the metabolite and to quantify parameter variability and covariate effects.

#### Structural and stochastic model

The structural model consisted of one compartment for risperidone (parent) and one compartment for 9-hydroxyrisperidone (metabolite), both with linear elimination. Due to the sparsely sampled data, additional compartments were not considered. The absorption of risperidone into the first compartment was described using a first-order absorption rate constant fixed to a previously reported value of 2.01 h^−1^ [[31](#CR31)]. As only data after oral administration were available, the oral bioavailability (F) of risperidone and the fraction of risperidone converted into 9-hydroxyrisperidone (*f*_met_) were not identifiable, and the reported disposition parameters of the parent and metabolite are therefore reported as apparent values (e.g. CL/F, CL_met_/[F × *f*_met_]) [[32](#CR32)].

Between-subject variability (BSV) was estimated for the clearance parameter of both the parent and metabolite using exponential models:

| CLi=TVCL×eηiCL |
| --- |
where CL_i_ is the risperidone or 9-hydroxyrisperidone clearance for the *i*th individual, and η_iCL_ denotes the difference between individual and population typical value (TVCL), which was assumed normally distributed with mean zero and variance ω_CL_^2^. The correlation between the individual clearances of risperidone and 9-hydroxyrisperidone was also estimated, while no BSV was estimated for the volume of distribution parameters due to the sparse sampling design. BSVs are reported as coefficients of variations, calculated as the square root of e^(ω2−1)^. The residual variability in concentration measurements (one submodel for each analyte) was initially modeled using combined additive and proportional error structures:

| Obsij=Predij1+ε1ij+ε2ij |
| --- |
where Obs_ij_ is the *j*th observed concentration in the *i*th individual, Pred_ij_ is the corresponding model prediction, and ε1 and ε2 are random error terms with means of zero and variances of σ1^2^ and σ2^2^, respectively. The residual error was considered simplified into proportional or additive submodels based on parameter estimate values and parameter significance levels.

#### Covariate model

To evaluate the effect of the various *CYP2D6* genotypes on risperidone clearance (CL), the clearance of risperidone was modeled as the sum of a CYP2D6-independent clearance term (base clearance: CL_base_) and the estimated clearances attributable to each of the *CYP2D6* alleles determined for the subject:

| CLrisperidone=CLbase+CLCYP2D6allele1+CLCYP2D6allele2 |
| --- |
As an example, the total risperidone clearance for a subject with *CYP2D6* diplotype of **1/*35* was predicted by the sum of the estimated parameters CL_base_, CL_CYP2D6*1_, and CL_CYP2D6*35_. Allele-specific clearances for *CYP2D6*2* (subsequently subdivided into new haplotypes based on rs5758550*A/G*), **9*, **10*, **35*, and **41* relative to *CYP2D6*1* were estimated, while the contribution to risperidone clearance from the deficient CYP2D6 alleles **3*, **4*, **5*, and **6* was a priori fixed to zero. Further, *NFIB* genotype was tested as a covariate on total clearance or CYP2D6-mediated clearance, and gene variation in *CYP3A4* was tested as a covariate on the base clearance term (which is expected to be primarily mediated by CYP3A4). Heterozygous and homozygous carriers of *CYP3A4*22* and the *NFIB-C* variant allele were merged due to minor proportions of homozygous carriers. In cases of unknown *CYP3A4* or *NFIB* genotype, the unknown genotype was tested either as a distinct covariate category or grouped together with the wildtype genotype.

In addition to gene variation, sex and age were evaluated as covariates on the clearance of risperidone. Sex and age were also evaluated on volumes of distributions and clearance of 9-hydroxyrisperidone, which has primarily renal elimination and is not a substrate for the enzymes under evaluation. Categorical covariates were evaluated by estimating the relative change in the pharmacokinetic parameter compared with the reference group. The impact of continuous covariates (i.e., patient age) was initially visually explored by estimating the relative change in the pharmacokinetic parameter of interest across age bins defined as 18–29 years, 30–39 years, 40–49 years, and so on. The relationship was then described in the final model using a mathematical function mimicking the observed relationship with age as a continuous covariate, such as linear, exponential, power, or piecewise linear functions.

The assessment of covariate relationships was undertaken using the forward inclusion and backward elimination procedure, which involves stepwise including the covariate with largest improvement in model fit for each round of covariate search until no more covariates significantly improve the model fit [[33](#CR33)]. During the inclusion of covariates, *p* < 0.05 was used as the significance level, while during the backward elimination of covariates from the full covariate model, a more stringent significance level of 0.01 was required for the covariate to be retained in the final model.

#### Model evaluation

Model selection was primarily based on the differences in objective function value (ΔOFV), where ΔOFV of ≥ 3.84 when adding one parameter corresponds to a *p*-value of < 0.05 (χ^2^ distribution, 1 degree of freedom). Models were also evaluated by inspecting standard goodness-of-fit plots (observed vs. predicted concentrations and conditional weighted residuals vs. predicted concentrations and time after dose), prediction-corrected visual predictive checks (pcVPCs, generated from 1000 simulations) [[34](#CR34)], biological plausibility of the parameter estimates and parameter uncertainty (95% confidence intervals derived from 5000 non-parametric bootstrap replicates) [[35](#CR35)]. Finally, the observed metabolic ratios that were used for the initial exploratory analysis were overlaid with the model-predicted typical metabolic ratio in each *CYP2D6* diplotype group.

#### Software and estimation method

Population pharmacokinetic modelling was performed using the non-linear mixed effects modelling software NONMEM (v. 7.5.1, ICON Development Solutions, Hanover, MD, USA) with the first-order conditional estimation method with interaction (FOCE-I). Piraña [[36](#CR36)] was used as graphical user interface. Data management, model evaluation, and graphical assessments were assisted by the R software, v. 4.2.1 [[37](#CR37)], and the Perl-Speaks-NONMEM (PsN) toolkit [[38](#CR38)].

### Haplotype analysis

Variants in high linkage disequilibrium (LD) with rs5758550 were assessed using LD-link [[39](#CR39)]. LD-link is based on whole-genome sequence (WGS) data from 2504 individuals in the 1000 Genomes Project. The European cohort (EUR *n* = 503, i.e., 1,006 alleles) was used as a reference population for the haplotype assessment in this study. Variants with high LD (*R*^2^ values > 0.85) together with the SNPs identifying *CYP2D6*2*, *CYP2D6*35*, and *CYP2D6*41* (rs16947, rs1135840, rs769258 (**35*), rs28371725 (**41*)) were used for haplotype analysis.

## Results

### Subjects and data

A total of 525 subjects were initially eligible for inclusion. Among these, 13 subjects were excluded due to inconclusive *CYP2D6* haplotypes (seven carriers of *CYP2D6*1/*2-rs5758550G x N* with unknown duplicated allele, and six carriers of *CYP2D6*2/*5* and rs5758550 *A/G*). This left 512 subjects and 1026 alleles for analysis (Table [1](#Tab1) and Supplementary Table [S1](#MOESM1)). The allele frequencies of *CYP2D6*2-rs5758550A*, *CYP2D6*2-rs5758550G*, *and CYP2D6*35* were 3% (*n* = 27), 16% (*n* = 166), and 5% (*n* = 54), respectively. Only three subjects carried *CYP2D6*1-rs5758550G*. These alleles were therefore merged with *CYP2D6*1-rs5758550A* (*n* = 398). The included subjects contributed a total of 1565 serum measurement pairs of risperidone and 9-hydroxyrisperidone.

### Table 1.

| Characteristic | Median (range) or number |
| --- | --- |
| Sex |  |
| Male, n | 275 |
| Female, n | 237 |
| Age, years; median (range) | 37 (18–89) |
| Risperidone daily dose (mg); median (range) | 3 (0.5–12) |
| Risperidone and 9-hydroxyrisperidone measurements |   |
| In total, n | 1565 |
| Per subject; median (range) | 2 (1–41) |
| Risperidone dosing frequency prior to sampling, n |   |
| One dose per day | 789 |
| Two doses per day | 758 |
| Three doses per day | 18 |
| Time between dose and sampling, hours; median (range) | 13 (9–30) |
| CYP2D6 allele, n |  |
| *1 | 401* |
| *35 | 54 |
| *2-rs5758550A | 27 |
| *2-rs5758550G | 166* |
| *9 | 20 |
| *10 | 23 |
| *41 | 66 |
| *3, 4, 5, or 6 (deficient alleles) | 269 |
| CYP3A4 allele, n |  |
| *1 | 939 |
| *22 | 27 |
| Not determined | 58 |
| NFIB allele, n |  |
| T | 965 |
| C | 53 |
| Not determined | 6 |

Table 1 Caption: Subject and sample characteristics

### Exploratory analysis

The initial exploratory analysis of the relationship between *CYP2D6* diplotypes and risperidone metabolic ratios (MR) indicated that each of the *CYP2D6* alleles under investigation was associated with reduced MR compared with the *CYP2D6*1* reference allele (Supplementary Table [S1](#MOESM1) and Supplementary Fig. [S1](#MOESM2)). Of particular interest, the observed MRs were generally lower in subjects carrying the *CYP2D6*2-rs5758550G*, *CYP2D6*2-rs5758550A*, or *CYP2D6*35* haplotypes/alleles compared with *CYP2D6*1*. This encouraged further modelling of the data to quantify and statistically compare each allele’s impact on risperidone clearance.

### Population pharmacokinetic model

#### Structural and stochastic model

The pharmacokinetics of risperidone and its main metabolite 9-hydroxyrisperidone were adequately described using the one-compartment disposition models. Between-subject variabilities in their clearances were estimated to be 125% and 53%, respectively, prior to the inclusion of covariates. The residual error was described by a combined proportional and additive model for risperidone, while a proportional error model was sufficient for 9-hydroxyrisperidone (additive component estimated close to zero with no significant increase in OFV upon removal).

#### Covariate model

The inclusion of *CYP2D6* genotype to the base model led to a marked model fit improvement (ΔOFV − 287 points, 6 *d.f., p* < 0.001). The *CYP2D6*2* allele parameter was further distinguished into significantly different values based on the rs5758550 variant, further improving the model fit (ΔOFV − 7.8, *p* = 0.005). Moreover, the *NFIB*-C variant allele was significantly associated with increased CYP2D6-mediated clearance (ΔOFV − 6.8, *p* = 0.009), while *CYP3A4**22 was not significant as a covariate in the model (*p* = 0.31).

For the impact of aging on the pharmacokinetics of risperidone and 9-hydroxyrisperidone, the exploratory analysis indicated stable clearances of both risperidone and 9-hydroxyrisperidone up to approximately 40 years followed by a linear decline at higher ages. The impacts of age on the clearances were therefore incorporated as piecewise linear functions with estimation of the age breaking points and the extents of linear decline above the age breaking point (Table [2](#Tab2)). These functions were supported for 9-hydroxyrisperidone clearance (ΔOFV − 62, *p* < 0.001) and for risperidone clearance (ΔOFV − 25, *p* < 0.001) and were superior to alternative candidate functions. No significant relationship was identified related to patient sex (*p* ≥ 0.23), and no covariates influenced the volume of distributions. After completing the forward inclusion of covariates, each covariate (including each *CYP2D6* variant allele) was removed from the final model sequentially and all were found to contribute significantly to the final model (*p* < 0.01).

| Model parameter | Parameter estimate | Bootstrap median | Bootstrap 95% CI |
| --- | --- | --- | --- |
| Risperidone (parent) |  |  |  |
| Fixed effects |  |  |  |
| Absorption rate constant (ka) (h−1) | 2.01 (fix)a |   |   |
| Volume of distribution (V/F) (L) | 333 | 317 | 177, 586 |
| Clearance (CL/F) (L/h) |  |  |  |
| Base | 4.2 | 4.2 | 3.6, 4.9 |
| CYP2D6*1 | 23.2 | 22.5 | 15.4, 31.6 |
| CYP2D6 allele (fraction of CYP2D6*1) |  |  |  |
| *2-rs5758550G | 0.66 | 0.66 | 0.50, 0.85 |
| *35 | 0.57 | 0.57 | 0.28, 0.90 |
| *2-rs5758550A | 0.30 | 0.31 | 0.17, 0.49 |
| *9 | 0.39 | 0.40 | 0.20, 0.68 |
| *10 | 0.32 | 0.33 | 0.15, 0.56 |
| *41 | 0.15 | 0.16 | 0.08, 0.26 |
| *3-*6 | 0 (fix) | - | - |
| NFIB allele (fraction of NFIB-T) |  |  |  |
| NFIB-C | 1.41 | 1.41 | 1.15, 1.73 |
| Age breaking point (years) | 34 | 34 | 28, 38 |
| Reduction per year (≥ breaking point) | − 0.9% | − 0.9% | − 1.2%, − 0.5% |
| Random effects |  |  |  |
| BSV CL/F (% CV) | 86% | 84% | 69%, 103% |
| Residual error, proportional (%) | 58% | 58% | 53%, 62% |
| Residual error, additive (ng/mL) | 0.06 | 0.06 | 0.04, 0.15 |
| 9-hydroxyrisperidone (metabolite) |  |  |  |
| Fixed effects |  |  |  |
| Volume of distribution (Vmet/F × fmet) | 96 | 97 | 77, 127 |
| Clearance (CLmet/F × fmet) | 8.0 | 8.0 | 7.5, 8.6 |
| Age breaking point (years) | 39 | 39 | 30, 48 |
| Reduction per year (≥ breaking point) | − 1.3% | − 1.3% | − 1.6%, − 1.0% |
| Random effects |  |  |  |
| BSV CLmet/fmet (% CV) | 49% | 49% | 43%, 55% |
| Residual error, proportional | 38% | 38% | 35%, 41% |
| Correlation with BSV CL/F | 0.43 | 0.41 | 0.25, 0.52 |

Table Caption: Final model parameter estimates

#### Final model evaluation and interpretation

The final model parameters are shown in Table [2](#Tab2). The estimated values were in close agreement with the bootstrap-derived median value for each parameter, and the 95% confidence intervals for each covariate excluded the null values (i.e., values representing no effect).

The clearances attributable to the alleles or haplotypes of primary interest were estimated at 30% for *CYP2D6*2-rs5758550A*, 66% for *CYP2D6*2-rs5758550G*, and 57% for *CYP2D6*35*, relative to the clearance for **1* (Fig. [1](#Fig1)). For the remaining decreased function alleles **9*, **10*, and **41*, the allele-specific clearances were estimated to be 39%, 32%, and 15%, respectively, relative to the clearance for **1*. Homo- or heterozygous carriers of the *NFIB-C* variant were estimated to have 41% higher CYP2D6-mediated clearance compared with subjects with the same *CYP2D6* genotype and *NFIB*-*T*. Finally, aging was associated with a reduction in clearance of 0.9% and 1.3% for each year above the ages of 34 and 39 years for risperidone and 9-hydroxyrisperidone, respectively.

![Fig. 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8ee1/11393095/cc51627f0ea2/228_2024_3721_Fig1_HTML.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=11393095_228_2024_3721_Fig1_HTML.jpg)

Population model-estimated risperidone clearance for each allele or haplotype, relative to CYP2D6*1. Points represent model estimates, and horizontal lines represent 95% confidence intervals. Each estimate is based on the following number of alleles: CYP2D6*1: 401, CYP2D6*2-rs5758550G: 166, CYP2D6*35: 54, CYP2D6*9: 20, CYP2D6*10: 23, CYP2D6*2-rs5758550A: 27, CYP2D6*41: 66, and deficient (CYP2D6*3, *4, *5, *6): 269

The prediction-corrected visual predictive check (Supplementary Fig. [S2](#MOESM3)) and diagnostic plots (Supplementary Fig. [S3](#MOESM4)) indicated that the model described the mean tendencies and variabilities in the time courses of drug concentrations well for both risperidone and 9-hydroxyrisperidone. The model-predicted metabolic ratio within each *CYP2D6* diplotype was also in overall agreement with the observed metabolic ratios (Fig. [2](#Fig2)), supporting that the model captures the trends in the underlying raw data.

![Fig. 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8ee1/11393095/9c5e7a80961e/228_2024_3721_Fig2_HTML.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=11393095_228_2024_3721_Fig2_HTML.jpg)

Model-predicted metabolic ratio (9-hydroxyrisperidone/risperidone concentration) together with observed metabolic ratio within each CYP2D6 diplotype. Horizontal lines represent the model-predicted metabolic ratio in a typical subject (with characteristic values similar to the median in the dataset: age 37 years, NFIB genotype TT, sampling 13 h after dose). Each dot represents one subject. The median individual values are depicted in the case of multiple observations per subject. Only diplotypes with ≥ 2 subjects are displayed (n = 504). Def = deficient allele (*3, *4, *5, or *6). *2-G = CYP2D6*2-rs5758550G. *2-A = CYP2D6*2-rs5758550A

### Haplotypes associated with rs5758550 G>A

LD analysis identified 24 SNPs in high LD to rs5758550 (Supplementary Fig. [S4](#MOESM5) and [S5](#MOESM6) [[39](#CR39)]). These SNPs, together with the SNPs identifying *CYP2D6*2*, *CYP2D6*35*, and *CYP2D6*41*, revealed a total of 34 haplotypes, and among these, 22 haplotypes were rare. The frequencies of *CYP2D6*2-rs5758550G* and *CYP2D6*2-rs5758550A* in the European reference cohort were 18.4% and 1.2%, respectively [[39](#CR39)]. Within the *CYP2D6*2-rs5758550G* haplotype, five different sub-haplotypes were found, and the most abundant among these had a frequency of 15.8%. This common haplotype contained all the 24 SNPs that were in high LD with rs5758550 (G>A). Among the *CYP2D6*2-rs5758550A* haplotypes, the most common sub-haplotype carried only 3 of the SNPs that were in high LD with rs5758550, indicating that 21 of the SNPs are specific for the *CYP2D6*2-rs5758550G* haplotype. All these SNPs are located downstream of the *CYP2D6* gene (chr22 42022906–chr22 41990945 (136 kb distance)). There is a high degree of interethnic differences for the haplotypes. In the East Asian reference cohort of the 1000 Genomes Project (EAS, *n* = 504), the allele frequency of the *CYP2D6*2-rs5758550G* and *CYP2D6*2-rs5758550A* haplotypes was 7.1% and 3%, respectively [[39](#CR39)].

## Discussion

Understanding the relationships between *CYP2D6* variants or haplotypes and CYP2D6 activity is important to predict the individual dose requirement for a vast number of drugs. Our study indicates a significant inhibitory effect of the *CYP2D6*2* and *CYP2D6*35* alleles on risperidone CYP2D6-mediated clearance. Risperidone clearance attributable to the rare *CYP2D6*2-rs5758550A* haplotype was only 30% relative to the clearance attributable to *CYP2D6*1*, which is different as compared to the common *CYP2D6*2-rs5758550G* haplotype where 66% of the clearance was estimated. *CYP2D6*2* is recommended for inclusion in *CYP2D6* genotyping panels [[13](#CR13)] but is currently classified with similar effects on enzyme activity as the wild type (**1*) [[3](#CR3)]. If our results also apply to other CYP2D6 substrates, interpreting *CYP2D6*2* as fully functional will overpredict drug clearance and dose requirements, especially for *CYP2D6*2-rs5758550A* carriers.

The association between rs5758550 and increased *CYP2D6* expression was first described by Wang et al., and they hypothesized that this effect and the reduced CYP2D6 activity function encoded by *CYP2D6*2* might have opposing effects on phenotype [[14](#CR14), [15](#CR15)]. Subsequent in vivo studies have been conflicting as to whether rs5758550 assessment can improve the prediction of CYP2D6 activity [[17](#CR17)–[19](#CR19), [40](#CR40), [41](#CR41)]. In all these studies, linkage disequilibrium with other SNPs in the locus was not considered, so we investigated the haplotypes carrying the rs5758550 mutation in more detail. We found 24 SNPs that showed high LD with rs5758550. The *CYP2D6*2-rs5758550G* haplotype included all 24 SNPs in high LD with rs5758550 (G>A), while the *CYP2D6*2-rs5758550A* haplotype carried only 3 of the SNPs in high LD. *CYP2D6*2-rs5758550G* was common (frequency 18.4%), while *CYP2D6*2-rs5758550A* was rare in Europeans (1.2%). However, in East Asians, *CYP2D6*2-rs5758550A* frequency was 3% [[39](#CR39)]. Previously, GWAS analyses of patients taking tamoxifen revealed a high degree of influence of the SNPs on Chr 22 in a region of 400 kb flanking the *CYP2D6 *gene on endoxifen levels [[41](#CR41)]. The data suggest that the *CYP2D6* locus on 22q13 within this 400 kb region is the most important genetic determinant for the regulation of *CYP2D6* expression. Since the specific SNPs in the non-CYP2D6 coding regions in this locus have not yet been identified, it is not possible to clarify which of the SNPs in high LD to *CYP2D6*2-rs5758550* might be important for the regulation of *CYP2D6* gene expression. This complexity may explain the contradictory results of previous studies, which are due to incomplete characterization of the alleles and also impair the current potential clinical use of this finding.

In our analysis, we also estimated risperidone CYP2D6-mediated clearance attributable to the *CYP2D6*35* allele to be 57% compared with *CYP2D6*1*, based on a total sample of 54 *CYP2D6*35* alleles. This estimate is similar to the previously reported value of 46% enzyme activity for the *CYP2D6*35* allele in a population pharmacokinetic study on carvedilol [[8](#CR8)]. For tamoxifen, reduced activity for the **35* allele has been reported in vitro [[12](#CR12)], but not in vivo [[6](#CR6)], suggesting potential substrate dependency.

The impacts of the well-known reduced function alleles *CYP2D6*9*, **10*, and **41* on risperidone CYP2D6-mediated clearance were also estimated in our analysis. The largest activity reduction was estimated for *CYP2D6*41* (15% residual activity), while 39% and 32% residual activities were estimated for *CYP2D6*9* and **10*, respectively. These impacts are as expected and in line with previous reports [[7](#CR7), [42](#CR42)], which also add confidence to the estimated effects associated with less studied variant alleles *CYP2D6*35* and *CYP2D6*2* as well as the rs5758550 SNP. Furthermore, we estimated that the impact of the mutated *NFIB-C* variant causes 41% increased CYP2D6-mediated clearance compared with the same *CYP2D6* allele and *NFIB-T*. This is in line with our previous findings in a partly overlapping patient sample [[29](#CR29)].

The clearance of risperidone in each subject was modeled as the sum of the partial clearances attributable to each individual *CYP2D6* allele, added to the CYP2D6-independent clearance term. In a typical normal metabolizer (e.g., *CYP2D6 *1/*1*), risperidone clearance is predicted to be 50.6 L/h. In a typical intermediate metabolizer (e.g., *CYP2D6 *1*/deficient allele), risperidone CL/F is predicted to be 27.4 L/h, whereas in a poor metabolizer (two *CYP2D6* deficient alleles), risperidone CL/F is predicted to be 4.2 L/h. These values are similar to the reported values in previously performed population pharmacokinetic analyses of risperidone in adult subjects with known *CYP2D6* genotypes [[43](#CR43)] as well as when *CYP2D6* genetic subpopulations were estimated using mixture modelling [[44](#CR44), [45](#CR45)].

In addition to the genetic covariates, aging was estimated to be associated with a decline in both risperidone and 9-hydroxyrisperidone clearance by 0.9 and 1.3% per year, respectively, after 34 and 39 years of age. The effect of aging on 9-hydroxyrisperidone has been reported previously [[31](#CR31), [43](#CR43), [44](#CR44)] and is expected since 9-hydroxyrisperidone is renally eliminated and age-related changes in renal function occur between 30 and 40 years of age [[46](#CR46)]. However, the precise mechanism underlying the age-related decline in risperidone clearance remains somewhat elusive as CYP2D6 activity remains relatively stable in the elderly. Age-related alterations in hepatic blood flow could play a role as risperidone exhibits intermediate hepatic extraction ratio [[47](#CR47)]. The model predicts that the total active moiety of risperidone and 9-hydroxyrisperidone would be more than doubled in subjects 75 years and older compared with a 30-year-old subject and supports the current recommendation of lower risperidone starting doses in the elderly.

Risperidone was selected as a CYP2D6 activity marker in this study to evaluate the impacts of allele variants on CYP2D6-mediated clearance. Notably, the major metabolite 9-hydroxyrisperidone is pharmacologically active, and this limits the clinical relevance of altered risperidone clearance [[43](#CR43)]. The Dutch Pharmacogenetic Working Group recommends that CYP2D6 poor metabolizers (i.e., patients carrying two deficient *CYP2D6* alleles) should start on 67% of the standard risperidone dose, while no action is recommended for intermediate metabolizers [[48](#CR48)]. Implementation of the reduced function haplotypes studied here would therefore not lead to altered risperidone starting dose recommendations. Thus, our results may be more relevant to optimize treatment with CYP2D6 substrates exhibiting narrower therapeutic indexes, including tricyclic antidepressants, and drugs dependent on CYP2D6 for bioactivation, such as tamoxifen, codeine, and tramadol. Considering the potential substrate-dependent effects of the investigated haplotypes, specific studies for such substances are needed prior to clinical implementation.

A limitation of our study is that the concentration measurements are derived from a TDM setting where measurements typically are only available from one sampling time during the dose interval. Further, information about patient characteristics was limited to age, sex, and genetics, with no information on body size or organ function parameters, which may also impact on risperidone pharmacokinetics. Still, as these missing potential covariates are not likely to be correlated with the genetic variants under investigation, the estimated genetic covariate effects are not expected to be biased. Another limitation is that it was not possible to determine at which allele the rs5758550 variant was located in heterozygous samples (e.g., rs5758550 *A/G* and *CYP2D6 *1/*2*). In such samples, it was assumed that the diplotype was **1-rs5758550A/*2-rs5758550G*, but it cannot be excluded that some of these subjects were truly **1-rs5758550G/*2-rs5758550A* carriers. As we only identified three subjects (< 1%) with rs5758550 *A/G* and *CYP2D6 *1/*1*, we do not believe that merging these subjects with the cohort of subject with rs5758550 *A/A* and *CYP2D6 *1/*1* genotype has influenced the results.

To summarize, we used the population pharmacokinetic approach to investigate the effects of *CYP2D6* variant alleles with unclear effects on enzyme activity using risperidone clearance as a CYP2D6 activity marker. Our results demonstrate that *CYP2D6*2* and *CYP2D6*35* encode reduced risperidone clearance and that there are two different haplotypes of *CYP2D6*2* in linkage to rs5758550 associated with different extent of clearance reduction*.* Thus, genotyping of these haplotypes might improve precision of genotype-guided prediction of CYP2D6-mediated clearance.

## Supplementary Information

Below is the link to the electronic supplementary material.

## Acknowledgements

The authors would like to thank all employees at the Center for Psychopharmacology for their daily routine work to generate the raw data used in this analysis.

## Author contribution

All authors contributed to the study conception and design. Material preparation, data collection, and analysis were performed by Elisabet Størset and Line Skute Bråten. All authors contributed to result interpretation. The first draft of the manuscript was written by Elisabet Størset, Line Skute Bråten, and Magnus Ingelman-Sundberg, and all authors critically reviewed the manuscript. All authors read and approved the final manuscript.

## Funding

Open access funding provided by Diakonhjemmet Hospital. This study was funded by The Swedish Research Council (grant number 2021-02801).

## Data availability

No datasets were generated or analysed during the current study.

## Declarations

### Ethics approval

The study was approved by the Regional Committee for Medical and Health Research Ethics (#2020/127234), and the Hospital Investigational Review Board approved the study. The included subjects had been given written information about the study with the possibility to reserve from study participation.

### Competing interests

Magnus Ingelman-Sundberg is co-founder and co-owner of HepaPredict AB. The remaining authors have no competing interests to declare that are relevant to the content of this article.

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

No datasets were generated or analysed during the current study.

### Supplementary Materials

### Data Availability Statement

No datasets were generated or analysed during the current study.

## References

1. Zhou SF (2009) Polymorphism of human cytochrome P450 2D6 and its clinical significance: part I. Clin Pharmacokinet 48(11):689–723 10.2165/11318030-000000000-00000  [DOI](https://doi.org/10.2165/11318030-000000000-00000) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19817501/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Zhou%20SF%20(2009)%20Polymorphism%20of%20human%20cytochrome%20P450%202D6%20and%20its%20clinical%20significance:%20part%20I.%20Clin%20Pharmacokinet%2048(11):689%E2%80%93723%2010.2165/11318030-000000000-00000)

2. Taylor C et al (2020) A review of the important role of CYP2D6 in pharmacogenomics. Genes (Basel) 11(11):1295  [DOI](https://doi.org/10.3390/genes11111295) | [PMC free article](/articles/PMC7692531/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33143137/)

3. Caudle KE et al (2020) Standardizing CYP2D6 genotype to phenotype translation: consensus recommendations from the Clinical Pharmacogenetics Implementation Consortium and Dutch Pharmacogenetics Working Group. Clin Transl Sci 13(1):116–124 10.1111/cts.12692  [DOI](https://doi.org/10.1111/cts.12692) | [PMC free article](/articles/PMC6951851/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31647186/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Caudle%20KE%20et%20al%20(2020)%20Standardizing%20CYP2D6%20genotype%20to%20phenotype%20translation:%20consensus%20recommendations%20from%20the%20Clinical%20Pharmacogenetics%20Implementation%20Consortium%20and%20Dutch%20Pharmacogenetics%20Working%20Group.%20Clin%20Transl%20Sci%2013(1):116%E2%80%93124%2010.1111/cts.12692)

4. Clinical Pharmacogenetics Implementation Consortium (CPIC®) CYP2D6 allele functionality reference. Available at: https://files.cpicpgx.org/data/report/current/allele_function_reference/CYP2D6_allele_functionality_reference.xlsx. Accessed 22 Mar 2024  [https://files.cpicpgx.org/data/report/current/allele_function_reference/CYP2D6_allele_functionality_reference.xlsx](https://files.cpicpgx.org/data/report/current/allele_function_reference/CYP2D6_allele_functionality_reference.xlsx)

5. van der Lee M, Guchelaar HJ, Swen JJ (2021) Substrate specificity of CYP2D6 genetic variants. Pharmacogenomics 22(16):1081–1089 10.2217/pgs-2021-0093  [DOI](https://doi.org/10.2217/pgs-2021-0093) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34569808/) | [Google Scholar](https://scholar.google.com/scholar_lookup?van%20der%20Lee%20M,%20Guchelaar%20HJ,%20Swen%20JJ%20(2021)%20Substrate%20specificity%20of%20CYP2D6%20genetic%20variants.%20Pharmacogenomics%2022(16):1081%E2%80%931089%2010.2217/pgs-2021-0093)

6. Hertz DL et al (2015) In vivo assessment of the metabolic activity of CYP2D6 diplotypes and alleles. Br J Clin Pharmacol 80(5):1122–1130 10.1111/bcp.12665  [DOI](https://doi.org/10.1111/bcp.12665) | [PMC free article](/articles/PMC4631184/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25907378/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Hertz%20DL%20et%20al%20(2015)%20In%20vivo%20assessment%20of%20the%20metabolic%20activity%20of%20CYP2D6%20diplotypes%20and%20alleles.%20Br%20J%20Clin%20Pharmacol%2080(5):1122%E2%80%931130%2010.1111/bcp.12665)

7. Abduljalil K et al (2010) Assessment of activity levels for CYP2D6*1, CYP2D6*2, and CYP2D6*41 genes by population pharmacokinetics of dextromethorphan. Clin Pharmacol Ther 88(5):643–651 10.1038/clpt.2010.137  [DOI](https://doi.org/10.1038/clpt.2010.137) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20881950/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Abduljalil%20K%20et%20al%20(2010)%20Assessment%20of%20activity%20levels%20for%20CYP2D6*1,%20CYP2D6*2,%20and%20CYP2D6*41%20genes%20by%20population%20pharmacokinetics%20of%20dextromethorphan.%20Clin%20Pharmacol%20Ther%2088(5):643%E2%80%93651%2010.1038/clpt.2010.137)

8. Sehrt D et al (2011) Carvedilol pharmacokinetics and pharmacodynamics in relation to CYP2D6 and ADRB pharmacogenetics. Pharmacogenomics 12(6):783–795 10.2217/pgs.11.20  [DOI](https://doi.org/10.2217/pgs.11.20) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21599570/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Sehrt%20D%20et%20al%20(2011)%20Carvedilol%20pharmacokinetics%20and%20pharmacodynamics%20in%20relation%20to%20CYP2D6%20and%20ADRB%20pharmacogenetics.%20Pharmacogenomics%2012(6):783%E2%80%93795%2010.2217/pgs.11.20)

9. Frederiksen T et al (2023) Estimating the in vivo function of CYP2D6 alleles through population pharmacokinetic modeling of brexpiprazole. Clin Pharmacol Ther 113(2):360–369 10.1002/cpt.2791  [DOI](https://doi.org/10.1002/cpt.2791) | [PMC free article](/articles/PMC10099095/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36350097/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Frederiksen%20T%20et%20al%20(2023)%20Estimating%20the%20in%20vivo%20function%20of%20CYP2D6%20alleles%20through%20population%20pharmacokinetic%20modeling%20of%20brexpiprazole.%20Clin%20Pharmacol%20Ther%20113(2):360%E2%80%93369%2010.1002/cpt.2791)

10. Frederiksen T et al (2021) Quantification of in vivo metabolic activity of CYP2D6 genotypes and alleles through population pharmacokinetic analysis of vortioxetine. Clin Pharmacol Ther 109(1):150–159 10.1002/cpt.1972  [DOI](https://doi.org/10.1002/cpt.1972) | [PMC free article](/articles/PMC7818194/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32599653/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Frederiksen%20T%20et%20al%20(2021)%20Quantification%20of%20in%20vivo%20metabolic%20activity%20of%20CYP2D6%20genotypes%20and%20alleles%20through%20population%20pharmacokinetic%20analysis%20of%20vortioxetine.%20Clin%20Pharmacol%20Ther%20109(1):150%E2%80%93159%2010.1002/cpt.1972)

11. Frederiksen T et al (2021) Cytochrome P450 2D6 genotype-phenotype characterization through population pharmacokinetic modeling of tedatioxetine. CPT Pharmacometrics Syst Pharmacol 10(9):983–993 10.1002/psp4.12635  [DOI](https://doi.org/10.1002/psp4.12635) | [PMC free article](/articles/PMC8452298/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33932135/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Frederiksen%20T%20et%20al%20(2021)%20Cytochrome%20P450%202D6%20genotype-phenotype%20characterization%20through%20population%20pharmacokinetic%20modeling%20of%20tedatioxetine.%20CPT%20Pharmacometrics%20Syst%20Pharmacol%2010(9):983%E2%80%93993%2010.1002/psp4.12635)

12. Muroi Y et al (2014) Functional characterization of wild-type and 49 CYP2D6 allelic variants for N-desmethyltamoxifen 4-hydroxylation activity. Drug Metab Pharmacokinet 29(5):360–366 10.2133/dmpk.DMPK-14-RG-014  [DOI](https://doi.org/10.2133/dmpk.DMPK-14-RG-014) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24647041/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Muroi%20Y%20et%20al%20(2014)%20Functional%20characterization%20of%20wild-type%20and%2049%20CYP2D6%20allelic%20variants%20for%20N-desmethyltamoxifen%204-hydroxylation%20activity.%20Drug%20Metab%20Pharmacokinet%2029(5):360%E2%80%93366%2010.2133/dmpk.DMPK-14-RG-014)

13. Pratt VM et al (2021) Recommendations for clinical CYP2D6 genotyping allele selection: a joint consensus recommendation of the Association for Molecular Pathology, College of American Pathologists, Dutch Pharmacogenetics Working Group of the Royal Dutch Pharmacists Association, and the European Society for Pharmacogenomics and Personalized Therapy. J Mol Diagn 23(9):1047–1064 10.1016/j.jmoldx.2021.05.013  [DOI](https://doi.org/10.1016/j.jmoldx.2021.05.013) | [PMC free article](/articles/PMC8579245/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34118403/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Pratt%20VM%20et%20al%20(2021)%20Recommendations%20for%20clinical%20CYP2D6%20genotyping%20allele%20selection:%20a%20joint%20consensus%20recommendation%20of%20the%20Association%20for%20Molecular%20Pathology,%20College%20of%20American%20Pathologists,%20Dutch%20Pharmacogenetics%20Working%20Group%20of%20the%20Royal%20Dutch%20Pharmacists%20Association,%20and%20the%20European%20Society%20for%20Pharmacogenomics%20and%20Personalized%20Therapy.%20J%20Mol%20Diagn%2023(9):1047%E2%80%931064%2010.1016/j.jmoldx.2021.05.013)

14. Wang D et al (2014) Common CYP2D6 polymorphisms affecting alternative splicing and transcription: long-range haplotypes with two regulatory variants modulate CYP2D6 activity. Hum Mol Genet 23(1):268–278 10.1093/hmg/ddt417  [DOI](https://doi.org/10.1093/hmg/ddt417) | [PMC free article](/articles/PMC3857955/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23985325/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Wang%20D%20et%20al%20(2014)%20Common%20CYP2D6%20polymorphisms%20affecting%20alternative%20splicing%20and%20transcription:%20long-range%20haplotypes%20with%20two%20regulatory%20variants%20modulate%20CYP2D6%20activity.%20Hum%20Mol%20Genet%2023(1):268%E2%80%93278%2010.1093/hmg/ddt417)

15. Wang D, Papp AC, Sun X (2015) Functional characterization of CYP2D6 enhancer polymorphisms. Hum Mol Genet 24(6):1556–1562 10.1093/hmg/ddu566  [DOI](https://doi.org/10.1093/hmg/ddu566) | [PMC free article](/articles/PMC4381757/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25381333/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Wang%20D,%20Papp%20AC,%20Sun%20X%20(2015)%20Functional%20characterization%20of%20CYP2D6%20enhancer%20polymorphisms.%20Hum%20Mol%20Genet%2024(6):1556%E2%80%931562%2010.1093/hmg/ddu566)

16. Ray B et al (2019) CYP2D6 haplotypes with enhancer single-nucleotide polymorphism rs5758550 and rs16947 (*2 allele): implications for CYP2D6 genotyping panels. Pharmacogenet Genomics 29(2):39–47 10.1097/FPC.0000000000000363  [DOI](https://doi.org/10.1097/FPC.0000000000000363) | [PMC free article](/articles/PMC6441960/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30520769/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Ray%20B%20et%20al%20(2019)%20CYP2D6%20haplotypes%20with%20enhancer%20single-nucleotide%20polymorphism%20rs5758550%20and%20rs16947%20(*2%20allele):%20implications%20for%20CYP2D6%20genotyping%20panels.%20Pharmacogenet%20Genomics%2029(2):39%E2%80%9347%2010.1097/FPC.0000000000000363)

17. Dinh JC et al (2022) The impact of the CYP2D6 “enhancer” single nucleotide polymorphism on CYP2D6 activity. Clin Pharmacol Ther 111(3):646–654 10.1002/cpt.2469  [DOI](https://doi.org/10.1002/cpt.2469) | [PMC free article](/articles/PMC8825689/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34716917/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Dinh%20JC%20et%20al%20(2022)%20The%20impact%20of%20the%20CYP2D6%20%E2%80%9Cenhancer%E2%80%9D%20single%20nucleotide%20polymorphism%20on%20CYP2D6%20activity.%20Clin%20Pharmacol%20Ther%20111(3):646%E2%80%93654%2010.1002/cpt.2469)

18. Sanchez-Spitman AB et al (2017) The effect of rs5758550 on CYP2D6*2 phenotype and formation of endoxifen in breast cancer patients using tamoxifen. Pharmacogenomics 18(12):1125–1132 10.2217/pgs-2017-0080  [DOI](https://doi.org/10.2217/pgs-2017-0080) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28745555/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Sanchez-Spitman%20AB%20et%20al%20(2017)%20The%20effect%20of%20rs5758550%20on%20CYP2D6*2%20phenotype%20and%20formation%20of%20endoxifen%20in%20breast%20cancer%20patients%20using%20tamoxifen.%20Pharmacogenomics%2018(12):1125%E2%80%931132%2010.2217/pgs-2017-0080)

19. Zanger UM et al (2021) Tri-allelic haplotypes determine and differentiate functionally normal allele CYP2D6*2 and impaired allele CYP2D6*41. Clin Pharmacol Ther 109(5):1256–1264 10.1002/cpt.2078  [DOI](https://doi.org/10.1002/cpt.2078) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33043448/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Zanger%20UM%20et%20al%20(2021)%20Tri-allelic%20haplotypes%20determine%20and%20differentiate%20functionally%20normal%20allele%20CYP2D6*2%20and%20impaired%20allele%20CYP2D6*41.%20Clin%20Pharmacol%20Ther%20109(5):1256%E2%80%931264%2010.1002/cpt.2078)

20. Løvlie R et al (2001) Polymorphisms in CYP2D6 duplication-negative individuals with the ultrarapid metabolizer phenotype: a role for the CYP2D6*35 allele in ultrarapid metabolism? Pharmacogenetics 11(1):45–55 10.1097/00008571-200102000-00006  [DOI](https://doi.org/10.1097/00008571-200102000-00006) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11207030/) | [Google Scholar](https://scholar.google.com/scholar_lookup?L%C3%B8vlie%20R%20et%20al%20(2001)%20Polymorphisms%20in%20CYP2D6%20duplication-negative%20individuals%20with%20the%20ultrarapid%20metabolizer%20phenotype:%20a%20role%20for%20the%20CYP2D6*35%20allele%20in%20ultrarapid%20metabolism?%20Pharmacogenetics%2011(1):45%E2%80%9355%2010.1097/00008571-200102000-00006)

21. Allorge D et al (2001) In-vitro analysis of the contribution of CYP2D6.35 to ultra-rapid metabolism. Pharmacogenetics 11(8):739–41 10.1097/00008571-200111000-00014  [DOI](https://doi.org/10.1097/00008571-200111000-00014) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11692084/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Allorge%20D%20et%20al%20(2001)%20In-vitro%20analysis%20of%20the%20contribution%20of%20CYP2D6.35%20to%20ultra-rapid%20metabolism.%20Pharmacogenetics%2011(8):739%E2%80%9341%2010.1097/00008571-200111000-00014)

22. Saito T et al (2018) Functional characterization of 50 CYP2D6 allelic variants by assessing primaquine 5-hydroxylation. Drug Metab Pharmacokinet 33(6):250–257 10.1016/j.dmpk.2018.08.004  [DOI](https://doi.org/10.1016/j.dmpk.2018.08.004) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30366777/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Saito%20T%20et%20al%20(2018)%20Functional%20characterization%20of%2050%20CYP2D6%20allelic%20variants%20by%20assessing%20primaquine%205-hydroxylation.%20Drug%20Metab%20Pharmacokinet%2033(6):250%E2%80%93257%2010.1016/j.dmpk.2018.08.004)

23. Jukic MM et al (2019) Effect of CYP2D6 genotype on exposure and efficacy of risperidone and aripiprazole: a retrospective, cohort study. Lancet Psychiatry 6(5):418–426 10.1016/S2215-0366(19)30088-4  [DOI](https://doi.org/10.1016/S2215-0366(19)30088-4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31000417/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Jukic%20MM%20et%20al%20(2019)%20Effect%20of%20CYP2D6%20genotype%20on%20exposure%20and%20efficacy%20of%20risperidone%20and%20aripiprazole:%20a%20retrospective,%20cohort%20study.%20Lancet%20Psychiatry%206(5):418%E2%80%93426%2010.1016/S2215-0366(19)30088-4)

24. Zhang L et al (2020) CYP2D6 genetic polymorphisms and risperidone pharmacokinetics: a systematic review and meta-analysis. Pharmacotherapy 40(7):632–647 10.1002/phar.2434  [DOI](https://doi.org/10.1002/phar.2434) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32519344/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Zhang%20L%20et%20al%20(2020)%20CYP2D6%20genetic%20polymorphisms%20and%20risperidone%20pharmacokinetics:%20a%20systematic%20review%20and%20meta-analysis.%20Pharmacotherapy%2040(7):632%E2%80%93647%2010.1002/phar.2434)

25. Hiemke C et al (2018) Consensus guidelines for therapeutic drug monitoring in neuropsychopharmacology: update 2017. Pharmacopsychiatry 51(1–02):9–62  [DOI](https://doi.org/10.1055/s-0043-116492) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28910830/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Hiemke%20C%20et%20al%20(2018)%20Consensus%20guidelines%20for%20therapeutic%20drug%20monitoring%20in%20neuropsychopharmacology:%20update%202017.%20Pharmacopsychiatry%2051(1%E2%80%9302):9%E2%80%9362)

26. Irby DJ et al (2021) Approaches to handling missing or “problematic” pharmacology data: pharmacokinetics. CPT Pharmacometrics Syst Pharmacol 10(4):291–308 10.1002/psp4.12611  [DOI](https://doi.org/10.1002/psp4.12611) | [PMC free article](/articles/PMC8099444/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33715307/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Irby%20DJ%20et%20al%20(2021)%20Approaches%20to%20handling%20missing%20or%20%E2%80%9Cproblematic%E2%80%9D%20pharmacology%20data:%20pharmacokinetics.%20CPT%20Pharmacometrics%20Syst%20Pharmacol%2010(4):291%E2%80%93308%2010.1002/psp4.12611)

27. Keizer RJ et al (2015) Incorporation of concentration data below the limit of quantification in population pharmacokinetic analyses. Pharmacol Res Perspect 3(2):e00131 10.1002/prp2.131  [DOI](https://doi.org/10.1002/prp2.131) | [PMC free article](/articles/PMC4448983/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26038706/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Keizer%20RJ%20et%20al%20(2015)%20Incorporation%20of%20concentration%20data%20below%20the%20limit%20of%20quantification%20in%20population%20pharmacokinetic%20analyses.%20Pharmacol%20Res%20Perspect%203(2):e00131%2010.1002/prp2.131)

28. Lenk H et al (2023) Impact of NFIB and CYP1A variants on clozapine serum concentration-a retrospective naturalistic cohort study on 526 patients with known smoking habits. Clin Transl Sci 16(1):62–72 10.1111/cts.13422  [DOI](https://doi.org/10.1111/cts.13422) | [PMC free article](/articles/PMC9841299/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36152308/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Lenk%20H%20et%20al%20(2023)%20Impact%20of%20NFIB%20and%20CYP1A%20variants%20on%20clozapine%20serum%20concentration-a%20retrospective%20naturalistic%20cohort%20study%20on%20526%20patients%20with%20known%20smoking%20habits.%20Clin%20Transl%20Sci%2016(1):62%E2%80%9372%2010.1111/cts.13422)

29. Lenk HC et al (2022) The polymorphic nuclear factor NFIB regulates hepatic CYP2D6 expression and influences risperidone metabolism in psychiatric patients. Clin Pharmacol Ther 111(5):1165–1174 10.1002/cpt.2571  [DOI](https://doi.org/10.1002/cpt.2571) | [PMC free article](/articles/PMC9314634/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35253216/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Lenk%20HC%20et%20al%20(2022)%20The%20polymorphic%20nuclear%20factor%20NFIB%20regulates%20hepatic%20CYP2D6%20expression%20and%20influences%20risperidone%20metabolism%20in%20psychiatric%20patients.%20Clin%20Pharmacol%20Ther%20111(5):1165%E2%80%931174%2010.1002/cpt.2571)

30. Mulder TAM et al (2021) CYP3A4(∗)22 Genotyping in clinical practice: ready for implementation? Front Genet 12:711943 10.3389/fgene.2021.711943  [DOI](https://doi.org/10.3389/fgene.2021.711943) | [PMC free article](/articles/PMC8296839/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34306041/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Mulder%20TAM%20et%20al%20(2021)%20CYP3A4(%E2%88%97)22%20Genotyping%20in%20clinical%20practice:%20ready%20for%20implementation?%20Front%20Genet%2012:711943%2010.3389/fgene.2021.711943)

31. Korell J et al (2017) Determination of plasma concentration reference ranges for risperidone and paliperidone. CPT Pharmacometrics Syst Pharmacol 6(9):589–595 10.1002/psp4.12217  [DOI](https://doi.org/10.1002/psp4.12217) | [PMC free article](/articles/PMC5613178/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28614899/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Korell%20J%20et%20al%20(2017)%20Determination%20of%20plasma%20concentration%20reference%20ranges%20for%20risperidone%20and%20paliperidone.%20CPT%20Pharmacometrics%20Syst%20Pharmacol%206(9):589%E2%80%93595%2010.1002/psp4.12217)

32. Ahlers SJ et al (2015) Morphine glucuronidation and elimination in intensive care patients: a comparison with healthy volunteers. Anesth Analg 121(5):1261–1273 10.1213/ANE.0000000000000936  [DOI](https://doi.org/10.1213/ANE.0000000000000936) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26332855/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Ahlers%20SJ%20et%20al%20(2015)%20Morphine%20glucuronidation%20and%20elimination%20in%20intensive%20care%20patients:%20a%20comparison%20with%20healthy%20volunteers.%20Anesth%20Analg%20121(5):1261%E2%80%931273%2010.1213/ANE.0000000000000936)

33. Jonsson EN, Karlsson MO (1998) Automated covariate model building within NONMEM. Pharm Res 15(9):1463–1468 10.1023/A:1011970125687  [DOI](https://doi.org/10.1023/A:1011970125687) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9755901/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Jonsson%20EN,%20Karlsson%20MO%20(1998)%20Automated%20covariate%20model%20building%20within%20NONMEM.%20Pharm%20Res%2015(9):1463%E2%80%931468%2010.1023/A:1011970125687)

34. Bergstrand M et al (2011) Prediction-corrected visual predictive checks for diagnosing nonlinear mixed-effects models. AAPS J 13(2):143–151 10.1208/s12248-011-9255-z  [DOI](https://doi.org/10.1208/s12248-011-9255-z) | [PMC free article](/articles/PMC3085712/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21302010/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Bergstrand%20M%20et%20al%20(2011)%20Prediction-corrected%20visual%20predictive%20checks%20for%20diagnosing%20nonlinear%20mixed-effects%20models.%20AAPS%20J%2013(2):143%E2%80%93151%2010.1208/s12248-011-9255-z)

35. Efron B (1979) Bootstrap methods: another look at the jackknife. Ann Stat 7:1–26 10.1214/aos/1176344552  [DOI](https://doi.org/10.1214/aos/1176344552) | [Google Scholar](https://scholar.google.com/scholar_lookup?Efron%20B%20(1979)%20Bootstrap%20methods:%20another%20look%20at%20the%20jackknife.%20Ann%20Stat%207:1%E2%80%9326%2010.1214/aos/1176344552)

36. Pirana LP. 100 Overlook Center, Suite 101. Princeton, NJ, 08540 USA

37. R Core Team. R: a language and environment for statistical computing. R Foundation for Statistical Computing, Vienna, Austria. https://www.R-project.org/  [https://www.R-project.org/](https://www.R-project.org/)

38. Lindbom L, Ribbing J, Jonsson EN (2004) Perl-speaks-NONMEM (PsN)–a Perl module for NONMEM related programming. Comput Methods Programs Biomed 75(2):85–94 10.1016/j.cmpb.2003.11.003  [DOI](https://doi.org/10.1016/j.cmpb.2003.11.003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15212851/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Lindbom%20L,%20Ribbing%20J,%20Jonsson%20EN%20(2004)%20Perl-speaks-NONMEM%20(PsN)%E2%80%93a%20Perl%20module%20for%20NONMEM%20related%20programming.%20Comput%20Methods%20Programs%20Biomed%2075(2):85%E2%80%9394%2010.1016/j.cmpb.2003.11.003)

39. Machiela MJ, Chanock SJ (2015) LDlink: a web-based application for exploring population-specific haplotype structure and linking correlated alleles of possible functional variants. Bioinformatics 31(21):3555–3557 10.1093/bioinformatics/btv402  [DOI](https://doi.org/10.1093/bioinformatics/btv402) | [PMC free article](/articles/PMC4626747/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26139635/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Machiela%20MJ,%20Chanock%20SJ%20(2015)%20LDlink:%20a%20web-based%20application%20for%20exploring%20population-specific%20haplotype%20structure%20and%20linking%20correlated%20alleles%20of%20possible%20functional%20variants.%20Bioinformatics%2031(21):3555%E2%80%933557%2010.1093/bioinformatics/btv402)

40. Thomas CD et al (2020) Examination of metoprolol pharmacokinetics and pharmacodynamics across CYP2D6 genotype-derived activity scores. CPT Pharmacometrics Syst Pharmacol 9(12):678–685 10.1002/psp4.12563  [DOI](https://doi.org/10.1002/psp4.12563) | [PMC free article](/articles/PMC7762806/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33067866/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Thomas%20CD%20et%20al%20(2020)%20Examination%20of%20metoprolol%20pharmacokinetics%20and%20pharmacodynamics%20across%20CYP2D6%20genotype-derived%20activity%20scores.%20CPT%20Pharmacometrics%20Syst%20Pharmacol%209(12):678%E2%80%93685%2010.1002/psp4.12563)

41. Khor CC et al (2023) Cross-ancestry genome-wide association study defines the extended CYP2D6 locus as the principal genetic determinant of endoxifen plasma concentrations. Clin Pharmacol Ther 113(3):712–723 10.1002/cpt.2846  [DOI](https://doi.org/10.1002/cpt.2846) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36629403/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Khor%20CC%20et%20al%20(2023)%20Cross-ancestry%20genome-wide%20association%20study%20defines%20the%20extended%20CYP2D6%20locus%20as%20the%20principal%20genetic%20determinant%20of%20endoxifen%20plasma%20concentrations.%20Clin%20Pharmacol%20Ther%20113(3):712%E2%80%93723%2010.1002/cpt.2846)

42. Jukic MM et al (2021) Evaluation of the CYP2D6 haplotype activity scores based on metabolic ratios of 4,700 patients treated with three different CYP2D6 substrates. Clin Pharmacol Ther 110(3):750–758 10.1002/cpt.2246  [DOI](https://doi.org/10.1002/cpt.2246) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33792048/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Jukic%20MM%20et%20al%20(2021)%20Evaluation%20of%20the%20CYP2D6%20haplotype%20activity%20scores%20based%20on%20metabolic%20ratios%20of%204,700%20patients%20treated%20with%20three%20different%20CYP2D6%20substrates.%20Clin%20Pharmacol%20Ther%20110(3):750%E2%80%93758%2010.1002/cpt.2246)

43. Vandenberghe F et al (2015) Genetics-based population pharmacokinetics and pharmacodynamics of risperidone in a psychiatric cohort. Clin Pharmacokinet 54(12):1259–1272 10.1007/s40262-015-0289-8  [DOI](https://doi.org/10.1007/s40262-015-0289-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26129906/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Vandenberghe%20F%20et%20al%20(2015)%20Genetics-based%20population%20pharmacokinetics%20and%20pharmacodynamics%20of%20risperidone%20in%20a%20psychiatric%20cohort.%20Clin%20Pharmacokinet%2054(12):1259%E2%80%931272%2010.1007/s40262-015-0289-8)

44. Feng Y et al (2008) Population pharmacokinetic analysis for risperidone using highly sparse sampling measurements from the CATIE study. Br J Clin Pharmacol 66(5):629–639 10.1111/j.1365-2125.2008.03276.x  [DOI](https://doi.org/10.1111/j.1365-2125.2008.03276.x) | [PMC free article](/articles/PMC2661978/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18771484/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Feng%20Y%20et%20al%20(2008)%20Population%20pharmacokinetic%20analysis%20for%20risperidone%20using%20highly%20sparse%20sampling%20measurements%20from%20the%20CATIE%20study.%20Br%20J%20Clin%20Pharmacol%2066(5):629%E2%80%93639%2010.1111/j.1365-2125.2008.03276.x)

45. Thyssen A et al (2010) Population pharmacokinetics of oral risperidone in children, adolescents and adults with psychiatric disorders. Clin Pharmacokinet 49(7):465–478 10.2165/11531730-000000000-00000  [DOI](https://doi.org/10.2165/11531730-000000000-00000) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20528007/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Thyssen%20A%20et%20al%20(2010)%20Population%20pharmacokinetics%20of%20oral%20risperidone%20in%20children,%20adolescents%20and%20adults%20with%20psychiatric%20disorders.%20Clin%20Pharmacokinet%2049(7):465%E2%80%93478%2010.2165/11531730-000000000-00000)

46. van der Burgh AC et al (2021) Determinants of the evolution of kidney function with age. Kidney Int Rep 6(12):3054–3063 10.1016/j.ekir.2021.10.006  [DOI](https://doi.org/10.1016/j.ekir.2021.10.006) | [PMC free article](/articles/PMC8640542/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34901574/) | [Google Scholar](https://scholar.google.com/scholar_lookup?van%20der%20Burgh%20AC%20et%20al%20(2021)%20Determinants%20of%20the%20evolution%20of%20kidney%20function%20with%20age.%20Kidney%20Int%20Rep%206(12):3054%E2%80%933063%2010.1016/j.ekir.2021.10.006)

47. Schlatter C et al (2009) Pharmacokinetic changes of psychotropic drugs in patients with liver disease: implications for dose adaptation. Drug Saf 32(7):561–578 10.2165/00002018-200932070-00003  [DOI](https://doi.org/10.2165/00002018-200932070-00003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19530743/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Schlatter%20C%20et%20al%20(2009)%20Pharmacokinetic%20changes%20of%20psychotropic%20drugs%20in%20patients%20with%20liver%20disease:%20implications%20for%20dose%20adaptation.%20Drug%20Saf%2032(7):561%E2%80%93578%2010.2165/00002018-200932070-00003)

48. Beunk L et al (2023) Dutch Pharmacogenetics Working Group (DPWG) guideline for the gene-drug interaction between CYP2D6, CYP3A4 and CYP1A2 and antipsychotics. Eur J Hum Genet 32(3):278–285  [DOI](https://doi.org/10.1038/s41431-023-01347-3) | [PMC free article](/articles/PMC10923774/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/37002327/)
