# Genome-Wide Association Study Identifies Genetic Polymorphisms Associated with Estimated Minimum Effective Concentration of Fentanyl in Patients Undergoing Laparoscopic-Assisted Colectomy

## Metadata
**Authors:** Daisuke Nishizawa, Tsutomu Mieda, Miki Tsujita, Hideyuki Nakagawa, Shigeki Yamaguchi, Shinya Kasai, Junko Hasegawa, Kyoko Nakayama, Yuko Ebata, Akira Kitamura, Hirotomo Shimizu, Tadayuki Takashima, Masakazu Hayashida, Kazutaka Ikeda
**Journal:** International Journal of Molecular Sciences
**Date:** 2023 May 8
**DOI:** [10.3390/ijms24098421](https://doi.org/10.3390/ijms24098421)
**PMID:** 37176129
**PMCID:** PMC10179231
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10179231/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC10179231/pdf/ijms-24-08421.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC10179231/pdf/ijms-24-08421.pdf)

## Abstract

Sensitivity to opioids varies widely among individuals. To identify potential candidate single-nucleotide polymorphisms (SNPs) that may significantly contribute to individual differences in the minimum effective concentration (MEC) of an opioid, fentanyl, we conducted a three-stage genome-wide association study (GWAS) using whole-genome genotyping arrays in 350 patients who underwent laparoscopic-assisted colectomy. To estimate the MEC of fentanyl, plasma and effect-site concentrations of fentanyl over the 24 h postoperative period were estimated with a pharmacokinetic simulation model based on initial bolus doses and subsequent patient-controlled analgesia doses of fentanyl. Plasma and effect-site MECs of fentanyl were indicated by fentanyl concentrations, estimated immediately before each patient-controlled analgesia dose. The GWAS revealed that an intergenic SNP, rs966775, that mapped to 5p13 had significant associations with the plasma MEC averaged over the 6 h postoperative period and the effect-site MEC averaged over the 12 h postoperative period. The minor G allele of rs966775 was associated with increases in these MECs of fentanyl. The nearest protein-coding gene around this SNP was DRD1, encoding the dopamine D1 receptor. In the gene-based analysis, the association was significant for the SERP2 gene in the dominant model. Our findings provide valuable information for personalized pain treatment after laparoscopic-assisted colectomy.

Keywords: opioids, analgesics, single-nucleotide polymorphism, genome-wide association study, minimum effective concentration, laparoscopic-assisted colectomy

## 1. Introduction

Opioids, such as morphine, fentanyl, oxycodone, and hydromorphone, are widely used as effective analgesics for the treatment of acute and chronic pain because of their robust antinociceptive effects. However, effects of opioids are not uniform across all patients, and considerable differences in the responsiveness or sensitivity to opioids are widely known [[1](#B1-ijms-24-08421),[2](#B2-ijms-24-08421)]. This can influence the total amount of analgesics that are required for adequate pain relief, which can hamper the effective treatment of pain in clinical practice. For example, the minimum effective concentrations (MECs) of fentanyl at which patients demand additional fentanyl doses to relieve recurring postoperative pain are reported to vary widely among patients from 0.23 to 0.99 ng/mL after orthopedic surgery [[3](#B3-ijms-24-08421)], from 0.23 to 1.18 ng/mL after open abdominal surgery [[4](#B4-ijms-24-08421)], from 0.30 to 1.45 (5–95 percentiles) ng/mL after open abdominal surgery [[5](#B5-ijms-24-08421)], and from 0.2 to 8.0 ng/mL after various surgical procedures [[6](#B6-ijms-24-08421)], indicating that MECs of fentanyl after certain surgical procedures have a more than four- to fivefold difference among individuals [[7](#B7-ijms-24-08421)]. Because of such significant differences in opioid sensitivity, empirical methods of administration that have been utilized by trial and error are an imperfect practice that can result in delayed or inadequate analgesia and possibly overdose [[7](#B7-ijms-24-08421)].

The required amounts of opioid analgesics may also vary among patients with pain depending on age, sex, weight, basal pain sensitivity, the type of surgery, perceived pain during the perioperative period [[2](#B2-ijms-24-08421)], and genetic factors. Previous twin studies of experimental heat and cold pressor pain reported that genetic effects were estimated to account for 12%, 60%, and 30% of the observed response variance (i.e., pain threshold) after administration of the opioid analgesic alfentanil for heat pain, cold-pressor pain, and cold-pressor pain, respectively [[8](#B8-ijms-24-08421),[9](#B9-ijms-24-08421)]. Although the variance of responses to opioids appears to be moderately influenced by genetic factors, potential genes and genetic variants that are involved in response variance have not yet been fully elucidated. Further studies are needed to delineate such genetic factors.

To date, many candidate gene association studies have been conducted [[10](#B10-ijms-24-08421),[11](#B11-ijms-24-08421),[12](#B12-ijms-24-08421)]. These studies have targeted various genes that are involved in pharmacokinetic and pharmacodynamic opioidergic pathways and pain-related genes of various modalities, such as the μ-opioid receptor (*OPRM1*) gene; cytochrome P450, family 2, subfamily D, polypeptide 6 (*CYP2D6*) gene; adenosine triphosphate-binding cassette (ABC), subfamily B (MDR/TAP), member 1 (*ABCB1*) gene; catechol-*O*-methyltransferase (*COMT*) gene; and genes that are related to cytokines (e.g., interleukin-1β, interleukin-6, and tumor necrosis factor-α) [[2](#B2-ijms-24-08421)]. Additional candidate genes are detailed in previous review articles [[10](#B10-ijms-24-08421),[11](#B11-ijms-24-08421),[12](#B12-ijms-24-08421)]. Genetic factors that are related to opioid sensitivity and responsiveness can also be explored using a genome-wide approach in genome-wide association studies (GWASs), although only a few studies have conducted GWASs of such phenotypes. One example is a prospective cross-sectional multinational multicenter study of patients with cancer from 11 European countries [[13](#B13-ijms-24-08421)] who were treated with opioids for moderate or severe pain. The strongest association with responsiveness to opioids was found for the rs12948783 single-nucleotide polymorphism (SNP), which is located upstream of the *RHBDF2* gene [[14](#B14-ijms-24-08421)].

We also conducted GWASs of phenotypes that are related to opioid sensitivity and candidate gene studies [[15](#B15-ijms-24-08421),[16](#B16-ijms-24-08421),[17](#B17-ijms-24-08421),[18](#B18-ijms-24-08421),[19](#B19-ijms-24-08421),[20](#B20-ijms-24-08421),[21](#B21-ijms-24-08421),[22](#B22-ijms-24-08421),[23](#B23-ijms-24-08421),[24](#B24-ijms-24-08421),[25](#B25-ijms-24-08421),[26](#B26-ijms-24-08421),[27](#B27-ijms-24-08421),[28](#B28-ijms-24-08421),[29](#B29-ijms-24-08421),[30](#B30-ijms-24-08421),[31](#B31-ijms-24-08421),[32](#B32-ijms-24-08421),[33](#B33-ijms-24-08421)]. In our GWASs, although genetic variants that were significantly associated with opioid responsiveness for the treatment were not found in patients with chronic pain [[30](#B30-ijms-24-08421)], we identified several SNPs, including the rs2952768 SNP (located near the *METTL21A* [*FAM119A*] and *CREB1* gene regions), that were significantly associated with postoperative opioid analgesic requirements in subjects who underwent cosmetic orthognathic surgery for mandibular prognathism [[18](#B18-ijms-24-08421)]. Furthermore, a GWAS of patients who were treated with opioid analgesics for the treatment of cancer pain identified several SNPs that were significantly associated with average daily opioid requirements for the treatment of pain, the best candidates of which were the rs1283671 and rs1283720 SNPs in the *ANGPT1* gene region. We also conducted GWASs of subjects who underwent laparoscopic-assisted colectomy (LAC), a surgery that is often categorized as minimally invasive because of much smaller skin incisions and less postoperative pain compared with traditional open abdominal surgery, although postoperative pain is not “minimal” after surgery [[22](#B22-ijms-24-08421)]. Our GWASs of subjects who underwent LAC identified several potent SNPs, including the nonsynonymous rs2076222 SNP in the *LAMB3* gene region, the rs199670311 nonsynonymous SNP in the *TMEM8A* gene region, and intronic SNPs, including rs4839603, in the *SLC9A9* gene region [[22](#B22-ijms-24-08421),[25](#B25-ijms-24-08421)].

Likely because of relative facileness, most previous human genetic studies have focused on opioid analgesic requirements for the treatment of disease-related pain, chronic pain, and perioperative/postoperative pain as the main endpoint to investigate genetic variants that are associated with human responsiveness and sensitivity to opioids [[10](#B10-ijms-24-08421),[11](#B11-ijms-24-08421),[12](#B12-ijms-24-08421),[13](#B13-ijms-24-08421),[14](#B14-ijms-24-08421),[15](#B15-ijms-24-08421),[16](#B16-ijms-24-08421),[17](#B17-ijms-24-08421),[18](#B18-ijms-24-08421),[19](#B19-ijms-24-08421),[20](#B20-ijms-24-08421),[21](#B21-ijms-24-08421),[22](#B22-ijms-24-08421),[23](#B23-ijms-24-08421),[24](#B24-ijms-24-08421),[25](#B25-ijms-24-08421),[26](#B26-ijms-24-08421),[27](#B27-ijms-24-08421),[28](#B28-ijms-24-08421),[29](#B29-ijms-24-08421),[30](#B30-ijms-24-08421),[31](#B31-ijms-24-08421),[32](#B32-ijms-24-08421),[33](#B33-ijms-24-08421)]. However, MECs at the plasma and effect site are not generally measured directly. Thus, these parameters have not been used to date in human genetic association studies. Nevertheless, with the development of pharmacokinetic/pharmacodynamic knowledge and the advancement of computer technology, it has become easier to simulate the process of plasma or effect-site concentrations of anesthetics and analgesics by leveraging related simulation software programs, such as STANPUMP ([http://opentci.org/code/stanpump](http://opentci.org/code/stanpump); accessed on 25 January 2023) and tivatrainer ([https://www.tivatrainer.com](https://www.tivatrainer.com); accessed on 25 January 2023). Such simulation software has been used in many studies to estimate plasma and effect-site concentrations of anesthetics and analgesics [[34](#B34-ijms-24-08421),[35](#B35-ijms-24-08421),[36](#B36-ijms-24-08421),[37](#B37-ijms-24-08421),[38](#B38-ijms-24-08421),[39](#B39-ijms-24-08421)].

In the present study, we conducted a GWAS of patients who underwent LAC to identify potential genetic variants that contribute to the efficacy of opioid analgesics based on information about estimated plasma or effect-site concentrations of fentanyl, which were calculated by utilizing one of the programs, the BeConSim Monitoring simulation software program ([http://www.masuinet.com](http://www.masuinet.com); accessed on 1 January 2020) [[40](#B40-ijms-24-08421),[41](#B41-ijms-24-08421),[42](#B42-ijms-24-08421)].

## 2. Results

### 2.1. Impact of Clinical Variables on Estimated MEC of Fentanyl in Subjects Who Underwent LAC

All 351 subjects completed the study. However, data were incomplete for one subject, particularly postoperative data. Therefore, postoperative data from the remaining 350 subjects were analyzed. Demographic, anesthetic, and surgical data for all 351 subjects are detailed in [Supplementary Table S1](#app1-ijms-24-08421) and our previous reports [[22](#B22-ijms-24-08421),[25](#B25-ijms-24-08421)].

Spearman’s rank correlation analysis indicated significant correlations among the 0–6 h plasma MEC, 0–12 h plasma MEC, 0–6 h effect-site MEC, and 0–12 h effect-site MEC (*ρ* = 0.976, *p* = 1.042 × 10^−231^, between 0–6 h plasma MEC and 0–12 h plasma MEC; *ρ* = 0.994, *p* < 1 × 10^−307^, between 0–6 h plasma MEC and 0–6 h effect-site MEC; *ρ* = 0.975, *p* = 7.383 × 10^−228^, between 0–6 h plasma MEC and 0–12 h effect-site MEC; *ρ* = 0.968, *p* = 1.043 × 10^−211^, between 0–12 h plasma MEC and 0–6 h effect-site MEC; *ρ* = 0.993, *p* < 1 × 10^−307^, between 0–12 h plasma MEC and 0–12 h effect-site MEC; *ρ* = 0.979, *p* = 3.049 × 10^−242^, between 0–6 h effect-site MEC and 0–12 h effect-site MEC). The Mann–Whitney test revealed no significant difference in the 0–6 h plasma MEC between sites of resection (*p* = 0.780), between anatomical extents of lymph node dissection (*p* = 0.740), or between genders (*p* = 0.177). The 0–12 h plasma MEC, 0–6 h effect-site MEC, and 0–12 h effect-site MEC were not significantly different between these parameters (details not shown). Multiple linear regression analyses revealed that the log-transformed 0–6 h plasma MEC was significantly associated with several clinical parameters, such as age (*β* = 0.004, *p* = 1.905 × 10^−3^), the average remifentanil infusion rate (*β* = 0.409, *p* = 1.454 × 10^−2^), the dose of fentanyl given around the end of surgery (*β* = 0.001, *p* = 4.400 × 10^−20^), and the 2 h postoperative pain score (*β* = 0.019, *p* = 1.415 × 10^−3^), and the trend was similar for 0–12 h plasma, 0–6 h effect-site, and 0–12 h effect-site MECs (details not shown). Therefore, these clinical variables were used as covariates in the subsequent analyses in the association study. Despite the strong correlations among the major endpoint variables, the 0–6 h plasma MEC, 0–12 h plasma MEC, 0–6 h effect-site MEC, and 0–12 h effect-site MEC, GWASs were performed for all of these four phenotypes in case even slight differences in these endpoint values could be caused by some slightly or moderately different genetic variants.

### 2.2. Identification of Genetic Polymorphisms Associated with Estimated MEC of Fentanyl in Patients Who Underwent LAC by GWAS

We then explored the association between genetic variations and opioid sensitivity, which was evaluated as the estimated plasma and effect-site MECs after surgery in a total of 350 subjects who underwent LAC that involved the administration of opioid analgesics [[22](#B22-ijms-24-08421),[25](#B25-ijms-24-08421)]. The surgical procedure was relatively uniform; thus, invasiveness and the resultant pain were regarded as relatively homogeneous among subjects. GWASs were conducted as a consecutive three-stage analysis to identify potent SNPs that were associated with the estimated 0–6 h plasma MEC, 0–12 h plasma MEC, 0–6 h effect-site MEC, and 0–12 h effect-site MEC. Consequently, 14, 26, and 10 SNPs were selected as the top candidates in the additive, dominant, and recessive models, respectively, after the final stage for the 0–6 h plasma MEC ([Supplementary Figure S1A](#app1-ijms-24-08421)). For the 0–12 h effect-site MEC, 14, 28, and 8 SNPs were selected as the top candidates in the additive, dominant, and recessive models, respectively, after the final stage ([Supplementary Figure S1B](#app1-ijms-24-08421)). Similarly, 21, 24, and 19 SNPs were initially selected in the additive, dominant, and recessive models, respectively, for the 0–12 h plasma MEC. Likewise, 17, 25, and 10 SNPs were initially selected in the additive, dominant, and recessive models, respectively, for the 0–6 h effect-site MEC (details not shown). The potent SNP lists are presented in [Table 1](#ijms-24-08421-t001) and [Table 2](#ijms-24-08421-t002) and [Supplementary Tables S2 and S3](#app1-ijms-24-08421). Among these, one SNP, rs966775, that mapped to 5p13 (GRCh37) showed significant associations with the 0–6 h plasma MEC and 0–12 h effect-site MEC after the final stage in the additive model (combined *β* = 0.0916, nominal *p* = 1.027 × 10^−7^, for the 0–6 h plasma MEC; combined *β* = 0.1071, nominal *p* = 1.299 × 10^−7^, for the 0–12 h effect-site MEC; [Table 1](#ijms-24-08421-t001) and [Table 2](#ijms-24-08421-t002)). The observed *p* values for this SNP, calculated as −log10 (*p* value), deviated from the expected values from the null hypothesis of uniform distribution in the quantile–quantile (QQ) *p*-value plots for the entire sample ([Supplementary Figure S1](#app1-ijms-24-08421) for the 0–6 h plasma MEC, [Supplementary Figure S2](#app1-ijms-24-08421) for the 0–12 h effect-site MEC). Similar strong associations with this SNP were observed for the 0–12 h plasma MEC and 0–6 h effect-site MEC after the final stage in the additive model (combined *β* = 0.0908, nominal *p* = 1.206 × 10^−7^, for the 0–12 h plasma MEC; combined *β* = 0.1095, nominal *p* = 1.942 × 10^−7^, for the 0–6 h effect-site MEC; [Supplementary Tables S2 and S3](#app1-ijms-24-08421)), although the associations were not significant. The rs966775 SNP is located in the intergenic region, and the nearest protein-coding gene from this SNP position was *DRD1*, which encodes the dopamine D_1_ receptor ([Supplementary Figure S3](#app1-ijms-24-08421)). A linkage disequilibrium (LD) block that includes the rs966775 SNP was assumed to span the approximately 1 kbp chromosomal region, and no SNPs showed high LD with this SNP in the neighboring region, including the *DRD1* gene region (pairwise calculated *r*^2^ = 0.93; [Supplementary Figure S3](#app1-ijms-24-08421)). When MECs (in ng/mL) were log-transformed and shown as mean ± standard error of the mean (SEM), 0–6 h plasma MECs were 0.4960 ± 0.0171, 0.5289 ± 0.0192, and 0.6839 ± 0.0393, and 0–12 h effect-site MECs were 0.5393 ± 0.0188, 0.5839 ± 0.0220, and 0.7589 ± 0.0433, in subjects with the A/A (*n* = 171), A/G (*n* = 137), and G/G (*n* = 41) genotypes of this SNP, respectively. The copy number of the minor G allele was associated with higher 0–6 h plasma and 0–12 h effect-site MECs. A similar trend was observed for 0–12 h plasma and 0–6 h effect-site MECs, and the copy number of the minor G allele was associated with greater MEC values for these phenotypes. The genotype distribution of this SNP met the criteria of the Hardy–Weinberg equilibrium tests (*χ2* = 2.7283, *p* = 0.0986).

### Table 1.

| Model | Rank | SNP | CHR | Position |   | 1st Stage |  |   | 2nd Stage |  |   | Final Stage |  |  |   | Combined |  |   | Related Gene |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
|   | β | p |   | β | p |   | β | p | q |   | β | p |   |  |  |  |  |  |  |
| Additive | 1 | rs966775 | 5 | 174,763,322 |   | 0.09569 | 0.004531 |   | 0.07693 | 0.03027 |   | 0.09776 | 0.0001217 | 0.0487 * |   | 0.09157 | 0.0000001027 |   | (DRD1) |
| Additive | 2 | rs6041532 | 20 | 12,652,435 |   | 0.2314 | 0.00159 |   | 0.1561 | 0.0334 |   | 0.2568 | 0.01035 | 0.4232 |   | 0.2003 | 0.000009788 |   | - |
| Additive | 3 | rs9354118 | 6 | 95,147,902 |   | 0.05395 | 0.02689 |   | 0.06611 | 0.01174 |   | 0.06085 | 0.02056 | 0.5427 |   | 0.06091 | 0.00002527 |   | - |
| Additive | 4 | rs9342409 | 6 | 95,098,682 |   | 0.05264 | 0.02903 |   | 0.06602 | 0.01574 |   | 0.05802 | 0.02816 | - |   | 0.06005 | 0.00004037 |   | - |
| Additive | 5 | rs4806716 | 19 | 54,639,868 |   | −0.05613 | 0.02562 |   | −0.06506 | 0.02383 |   | −0.06482 | 0.01373 | 0.4577 |   | −0.06019 | 0.00006081 |   | - |
| Additive | 6 | rs43211 | 19 | 54,652,203 |   | −0.05488 | 0.02417 |   | −0.08097 | 0.005927 |   | −0.05655 | 0.03242 | 0.6175 |   | −0.05957 | 0.00006086 |   | CNOT3 |
| Additive | 7 | rs9363197 | 6 | 95,104,559 |   | 0.05395 | 0.02689 |   | 0.05782 | 0.03145 |   | 0.05802 | 0.02816 | - |   | 0.05795 | 0.00006976 |   | - |
| Additive | 8 | rs4764074 | 12 | 14,428,118 |   | −0.08212 | 0.01157 |   | −0.05914 | 0.04189 |   | −0.05778 | 0.04175 | 0.7207 |   | −0.06609 | 0.0000889 |   | - |
| Additive | 9 | rs2676289 | 17 | 62,705,738 |   | −0.1448 | 0.04901 |   | −0.08609 | 0.03379 |   | −0.1407 | 0.005126 | 0.4232 |   | −0.1114 | 0.00008892 |   | - |
| Additive | 10 | rs2759632 | 10 | 10,218,843 |   | −0.09777 | 0.02416 |   | −0.09776 | 0.04 |   | −0.09442 | 0.01637 | 0.4926 |   | −0.09548 | 0.00009947 |   | - |
| Additive | 11 | rs1016214 | 20 | 16,992,615 |   | 0.1305 | 0.02942 |   | 0.2103 | 0.04618 |   | 0.1138 | 0.006869 | 0.4232 |   | 0.124 | 0.0001183 |   | - |
| Additive | 12 | rs452325 | 8 | 88,505,366 |   | 0.06272 | 0.01896 |   | 0.06914 | 0.02194 |   | 0.05307 | 0.04622 | - |   | 0.05799 | 0.0001731 |   | - |
| Additive | 13 | rs936229 | 15 | 75,132,319 |   | 0.07303 | 0.02838 |   | 0.06454 | 0.02857 |   | 0.06169 | 0.04701 | 0.7207 |   | 0.06311 | 0.0003549 |   | ULK3 |
| Additive | 14 | rs652930 | 1 | 201,628,577 |   | 0.091 | 0.03362 |   | 0.1072 | 0.02579 |   | 0.183 | 0.009896 | 0.4232 |   | 0.1015 | 0.0003897 |   | NAV1 |
| Dominant | 1 | rs6502266 | 17 | 13,395,720 |   | 0.1232 | 0.001448 |   | 0.1076 | 0.005443 |   | 0.08215 | 0.02588 | 0.4925 |   | 0.1004 | 0.000002452 |   | - |
| Dominant | 2 | rs9889837 | 17 | 13,392,473 |   | 0.1232 | 0.001448 |   | 0.1076 | 0.005443 |   | 0.07843 | 0.03363 | - |   | 0.09947 | 0.000003073 |   | - |
| Dominant | 3 | rs6481157 | 10 | 57,099,471 |   | −0.1346 | 0.0009998 |   | −0.0775 | 0.04471 |   | −0.08874 | 0.01887 | 0.4925 |   | −0.0947 | 0.00001424 |   | - |
| Dominant | 4 | rs17738087 | 15 | 26,905,021 |   | −0.1148 | 0.009143 |   | −0.1091 | 0.01244 |   | −0.09608 | 0.01112 | 0.4925 |   | −0.1004 | 0.00002 |   | GABRB3 |
| Dominant | 5 | rs17081058 | 13 | 25,267,734 |   | −0.09808 | 0.01557 |   | −0.09338 | 0.02287 |   | −0.08478 | 0.03985 | 0.4925 |   | −0.0972 | 0.0000271 |   | ATP12A |
| Dominant | 6 | rs1195916 | 12 | 131,503,109 |   | 0.08309 | 0.04349 |   | 0.07868 | 0.04677 |   | 0.1225 | 0.003758 | 0.4925 |   | 0.09485 | 0.00003543 |   | GPR133 |
| Dominant | 7 | rs172399 | 7 | 9,154,302 |   | 0.1075 | 0.007509 |   | 0.08153 | 0.0404 |   | 0.08112 | 0.0266 | 0.4925 |   | 0.08836 | 0.00005485 |   | - |
| Dominant | 8 | rs13278423 | 8 | 87,720,419 |   | −0.1184 | 0.00618 |   | −0.09612 | 0.02496 |   | −0.08617 | 0.03348 | 0.4925 |   | −0.09586 | 0.00006912 |   | CNGB3 |
| Dominant | 9 | rs1160226 | 13 | 25,271,434 |   | −0.08072 | 0.04288 |   | −0.09013 | 0.02275 |   | −0.07819 | 0.03853 | 0.4925 |   | −0.08602 | 0.00009199 |   | ATP12A |
| Dominant | 10 | rs3133206 | 18 | 57,237,478 |   | 0.08225 | 0.04547 |   | 0.09069 | 0.02442 |   | 0.09395 | 0.01793 | 0.4925 |   | 0.08851 | 0.00009607 |   | CCBE1 |
| Dominant | 11 | rs4963573 | 12 | 24,662,116 |   | −0.09225 | 0.01659 |   | −0.07917 | 0.03853 |   | −0.0841 | 0.02294 | 0.4925 |   | −0.0828 | 0.0001084 |   | SOX5 |
| Dominant | 12 | rs28350 | 3 | 42,418,446 |   | 0.1126 | 0.04845 |   | 0.129 | 0.02853 |   | 0.1208 | 0.01785 | 0.4925 |   | 0.1212 | 0.0001187 |   | - |
| Dominant | 13 | rs10956972 | 8 | 87,768,331 |   | 0.1033 | 0.01486 |   | 0.08581 | 0.03306 |   | 0.1052 | 0.01004 | - |   | 0.08923 | 0.0001351 |   | - |
| Dominant | 14 | rs1982563 | 8 | 87,776,019 |   | 0.1033 | 0.01486 |   | 0.08581 | 0.03306 |   | 0.1052 | 0.01004 | 0.4925 |   | 0.08923 | 0.0001351 |   | - |
| Dominant | 15 | rs4940475 | 18 | 57,311,314 |   | 0.08404 | 0.04074 |   | 0.09424 | 0.02218 |   | 0.08181 | 0.03923 | - |   | 0.08696 | 0.0001466 |   | CCBE1 |
| Dominant | 16 | rs5766289 | 22 | 45,408,177 |   | −0.08004 | 0.03874 |   | −0.08407 | 0.02975 |   | −0.0916 | 0.01366 | 0.4925 |   | −0.08221 | 0.0001498 |   | - |
| Dominant | 17 | rs1864309 | 18 | 57,309,059 |   | 0.08404 | 0.04074 |   | 0.09367 | 0.02112 |   | 0.08181 | 0.03923 | - |   | 0.08604 | 0.0001602 |   | CCBE1 |
| Dominant | 18 | rs1027804 | 8 | 18,919,857 |   | −0.08584 | 0.03366 |   | −0.07961 | 0.04826 |   | −0.1072 | 0.005536 | 0.4925 |   | −0.0851 | 0.0001687 |   | - |
| Dominant | 19 | rs7592517 | 2 | 76,777,279 |   | 0.1278 | 0.002637 |   | 0.08475 | 0.03565 |   | 0.07594 | 0.04886 | - |   | 0.08143 | 0.0002555 |   | - |
| Dominant | 20 | rs2139502 | 2 | 76,786,845 |   | 0.1278 | 0.002637 |   | 0.08475 | 0.03565 |   | 0.07594 | 0.04886 | 0.4925 |   | 0.08143 | 0.0002555 |   | - |
| Dominant | 21 | exm−rs10873636 | 15 | 26,888,978 |   | −0.09977 | 0.02877 |   | −0.09658 | 0.03058 |   | −0.07793 | 0.04325 | - |   | −0.08463 | 0.0003904 |   | GABRB3 |
| Dominant | 22 | rs10873636 | 15 | 26,888,978 |   | −0.09977 | 0.02877 |   | −0.09658 | 0.03058 |   | −0.07793 | 0.04325 | - |   | −0.08463 | 0.0003904 |   | GABRB3 |
| Dominant | 23 | rs1863459 | 15 | 26,892,676 |   | −0.09977 | 0.02877 |   | −0.09658 | 0.03058 |   | −0.07793 | 0.04325 | 0.4925 |   | −0.08463 | 0.0003904 |   | GABRB3 |
| Dominant | 24 | rs6667463 | 1 | 175,518,442 |   | −0.08513 | 0.03978 |   | −0.115 | 0.01136 |   | −0.08208 | 0.04367 | 0.4925 |   | −0.0833 | 0.0004116 |   | TNR |
| Dominant | 25 | rs12580224 | 12 | 71,086,426 |   | 0.1011 | 0.01189 |   | 0.08249 | 0.04176 |   | 0.07857 | 0.04728 | 0.4925 |   | 0.07887 | 0.0004391 |   | PTPRR |
| Dominant | 26 | rs11945758 | 4 | 118,667,234 |   | 0.09923 | 0.0248 |   | 0.09422 | 0.02897 |   | 0.088 | 0.02815 | 0.4925 |   | 0.08344 | 0.0004876 |   | - |
| Recessive | 1 | rs966775 | 5 | 174,763,322 |   | 0.1704 | 0.008618 |   | 0.1598 | 0.01971 |   | 0.1655 | 0.0005068 | 0.2027 |   | 0.1657 | 0.0000004313 |   | (DRD1) |
| Recessive | 2 | rs6041532 | 20 | 12,652,435 |   | 0.4645 | 0.001396 |   | 0.3096 | 0.03374 |   | 0.5211 | 0.008986 | 0.3114 |   | 0.4019 | 0.000008521 |   | - |
| Recessive | 3 | rs9354118 | 6 | 95,147,902 |   | 0.09027 | 0.03077 |   | 0.09682 | 0.03613 |   | 0.1358 | 0.001557 | 0.3116 |   | 0.1064 | 0.0000166 |   | - |
| Recessive | 4 | rs9342409 | 6 | 95,098,682 |   | 0.08627 | 0.03752 |   | 0.09933 | 0.04201 |   | 0.1358 | 0.001557 | 0.3865 |   | 0.1067 | 0.00002053 |   | - |
| Recessive | 5 | rs43211 | 19 | 54,652,203 |   | −0.08789 | 0.04048 |   | −0.1426 | 0.009216 |   | −0.1305 | 0.005266 | 0.3116 |   | −0.1106 | 0.00003108 |   | CNOT3 |
| Recessive | 6 | rs4764074 | 12 | 14,428,118 |   | −0.149 | 0.01646 |   | −0.1074 | 0.04912 |   | −0.1277 | 0.01546 | 0.3116 |   | −0.1258 | 0.00007724 |   | - |
| Recessive | 7 | rs2759632 | 10 | 10,218,843 |   | −0.1855 | 0.03017 |   | −0.1941 | 0.03807 |   | −0.2034 | 0.009063 | 0.693 |   | −0.1905 | 0.00008301 |   | - |
| Recessive | 8 | rs2146423 | 9 | 4,657,040 |   | 0.1379 | 0.01801 |   | 0.1222 | 0.03914 |   | 0.1779 | 0.004626 | 0.3116 |   | 0.1307 | 0.0001066 |   | C9orf68 |
| Recessive | 9 | rs12714409 | 2 | 596,532 |   | 0.09831 | 0.03269 |   | 0.1058 | 0.03244 |   | 0.1182 | 0.01311 | 0.3746 |   | 0.1038 | 0.0001265 |   | - |
| Recessive | 10 | rs2642589 | 10 | 71,513,647 |   | 0.2137 | 0.0422 |   | 0.1943 | 0.03691 |   | 0.2436 | 0.03985 | - |   | 0.2132 | 0.000292 |   | - |

Table 1 Caption: Top candidate SNPs selected from three-stage GWAS for the 0–6 h plasma MEC.

### Table 2.

| Model | Rank | SNP | CHR | Position |   | 1st Stage |  |   | 2nd Stage |  |   | Final Stage |  |  |   | Combined |  |   | Related Gene |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
|   | β | p |   | β | p |   | β | p | q |   | β | p |   |  |  |  |  |  |  |
| Additive | 1 | rs966775 | 5 | 174,763,322 |   | 0.1089 | 0.004956 |   | 0.09541 | 0.02628 |   | 0.1153 | 0.0001176 | 0.0487 * |   | 0.1071 | 0.0000001299 |   | (DRD1) |
| Additive | 2 | rs6041532 | 20 | 12,652,435 |   | 0.2568 | 0.002112 |   | 0.1767 | 0.04689 |   | 0.2887 | 0.01417 | 0.5852 |   | 0.2257 | 0.00002233 |   | - |
| Additive | 3 | rs9354118 | 6 | 95,147,902 |   | 0.05787 | 0.03746 |   | 0.07979 | 0.01181 |   | 0.07682 | 0.01282 | 0.5852 |   | 0.07173 | 0.00002448 |   | - |
| Additive | 4 | rs9342409 | 6 | 95,098,682 |   | 0.05643 | 0.04045 |   | 0.08091 | 0.01476 |   | 0.07346 | 0.01801 | - |   | 0.0711 | 0.00003707 |   | - |
| Additive | 5 | rs452325 | 8 | 88,505,366 |   | 0.08469 | 0.005179 |   | 0.07778 | 0.03335 |   | 0.06598 | 0.03512 | - |   | 0.07321 | 0.00005313 |   | - |
| Additive | 6 | rs391916 | 8 | 88,512,286 |   | 0.08987 | 0.003892 |   | 0.07335 | 0.04768 |   | 0.07024 | 0.03155 | - |   | 0.07531 | 0.00005547 |   | - |
| Additive | 7 | rs9363197 | 6 | 95,104,559 |   | 0.05787 | 0.03746 |   | 0.07095 | 0.02872 |   | 0.07346 | 0.01801 | - |   | 0.06866 | 0.00006093 |   | - |
| Additive | 8 | rs4764074 | 12 | 14,428,118 |   | −0.08937 | 0.01592 |   | −0.07677 | 0.02874 |   | −0.06911 | 0.0377 | 0.62 |   | −0.07886 | 0.00006821 |   | - |
| Additive | 9 | rs4806716 | 19 | 54,639,868 |   | −0.06474 | 0.02339 |   | −0.07928 | 0.02267 |   | −0.07113 | 0.02095 | 0.5852 |   | −0.06999 | 0.00007154 |   | - |
| Additive | 10 | rs375481 | 8 | 88,490,938 |   | 0.0844 | 0.005855 |   | 0.08185 | 0.02178 |   | 0.06467 | 0.03894 | 0.62 |   | 0.07169 | 0.00007405 |   | - |
| Additive | 11 | rs463809 | 8 | 88,513,842 |   | 0.0884 | 0.004368 |   | 0.07265 | 0.04837 |   | 0.06353 | 0.04896 | - |   | 0.07236 | 0.00009161 |   | - |
| Additive | 12 | rs12035559 | 1 | 34,499,921 |   | −0.09324 | 0.04609 |   | −0.08629 | 0.01417 |   | −0.0688 | 0.03792 | 0.62 |   | −0.08029 | 0.0001101 |   | CSMD2 |
| Additive | 13 | rs2759632 | 10 | 10,218,843 |   | −0.1166 | 0.01793 |   | −0.1207 | 0.03614 |   | −0.09306 | 0.04611 | 0.6725 |   | −0.1076 | 0.0001932 |   | - |
| Additive | 14 | rs4759709 | 12 | 131,001,468 |   | 0.07097 | 0.04568 |   | 0.08022 | 0.02878 |   | 0.08548 | 0.0188 | 0.5852 |   | 0.07568 | 0.000232 |   | RIMBP2 |
| Additive | 15 | rs2013536 | 8 | 87,669,792 |   | 0.06629 | 0.03569 |   | 0.07622 | 0.02542 |   | 0.05973 | 0.04567 | 0.6725 |   | 0.06185 | 0.0005526 |   | CNGB3 |
| Additive | 16 | rs2160974 | 12 | 108,883,621 |   | 0.06169 | 0.04197 |   | 0.07592 | 0.02893 |   | 0.06844 | 0.03283 | 0.62 |   | 0.06312 | 0.0005903 |   | - |
| Additive | 17 | rs2368473 | 17 | 32,534,215 |   | 0.172 | 0.04925 |   | 0.09229 | 0.04477 |   | 0.1336 | 0.04801 | 0.6725 |   | 0.09914 | 0.003209 |   | - |
| Dominant | 1 | rs6502266 | 17 | 13,395,720 |   | 0.1431 | 0.001155 |   | 0.1187 | 0.01152 |   | 0.08856 | 0.04179 | - |   | 0.1128 | 0.000006864 |   | - |
| Dominant | 2 | rs17081058 | 13 | 25,267,734 |   | −0.1161 | 0.01188 |   | −0.1097 | 0.02728 |   | −0.1085 | 0.02525 | 0.7497 |   | −0.1163 | 0.00001918 |   | ATP12A |
| Dominant | 3 | rs10836454 | 11 | 4,696,875 |   | −0.1235 | 0.03386 |   | −0.1135 | 0.03507 |   | −0.1552 | 0.002679 | 0.7497 |   | −0.1302 | 0.00002082 |   | - |
| Dominant | 4 | rs17738087 | 15 | 26,905,021 |   | −0.1233 | 0.01412 |   | −0.1238 | 0.01927 |   | −0.1158 | 0.009327 | 0.7497 |   | −0.1162 | 0.00002685 |   | GABRB3 |
| Dominant | 5 | rs751687 | 8 | 15,608,896 |   | 0.1227 | 0.004706 |   | 0.1077 | 0.02049 |   | 0.09564 | 0.02716 | 0.7497 |   | 0.1054 | 0.00002937 |   | TUSC3 |
| Dominant | 6 | rs9968875 | 6 | 151,313,367 |   | 0.2127 | 0.03445 |   | 0.2375 | 0.002204 |   | 0.2158 | 0.04357 | 0.7497 |   | 0.2128 | 0.00003786 |   | MTHFD1L |
| Dominant | 7 | rs4131101 | 5 | 119,195,837 |   | 0.1201 | 0.01361 |   | 0.1293 | 0.01648 |   | 0.1006 | 0.03878 | 0.7497 |   | 0.1165 | 0.00004616 |   | - |
| Dominant | 8 | rs746427 | 20 | 48,939,076 |   | −0.1004 | 0.02234 |   | −0.106 | 0.04067 |   | −0.1111 | 0.01493 | - |   | −0.1057 | 0.00004884 |   | - |
| Dominant | 9 | rs4580854 | 6 | 15,025,298 |   | −0.09948 | 0.02485 |   | −0.1086 | 0.0188 |   | −0.1015 | 0.0201 | 0.7497 |   | −0.1021 | 0.00005343 |   | - |
| Dominant | 10 | rs1431210 | 6 | 103,229,346 |   | 0.1369 | 0.01037 |   | 0.1107 | 0.02506 |   | 0.103 | 0.02836 | - |   | 0.1129 | 0.00005784 |   | - |
| Dominant | 11 | rs2143500 | 20 | 45,253,237 |   | 0.08748 | 0.04958 |   | 0.1251 | 0.009653 |   | 0.103 | 0.02006 | 0.7497 |   | 0.1019 | 0.00008058 |   | SLC13A3 |
| Dominant | 12 | exm2270377 | 6 | 103,225,137 |   | 0.1369 | 0.01037 |   | 0.103 | 0.03793 |   | 0.103 | 0.02836 | - |   | 0.1101 | 0.00008819 |   | - |
| Dominant | 13 | rs6020445 | 20 | 48,939,863 |   | −0.1004 | 0.02234 |   | −0.1111 | 0.03323 |   | −0.09611 | 0.03785 | 0.7497 |   | −0.102 | 0.00009374 |   | - |
| Dominant | 14 | rs3935993 | 5 | 119,196,820 |   | 0.1066 | 0.02517 |   | 0.1293 | 0.01648 |   | 0.1006 | 0.03878 | - |   | 0.1107 | 0.00009776 |   | - |
| Dominant | 15 | rs13195313 | 6 | 103,175,290 |   | 0.1276 | 0.0143 |   | 0.103 | 0.03793 |   | 0.1063 | 0.02584 | - |   | 0.1094 | 0.00009796 |   | - |
| Dominant | 16 | rs12817917 | 12 | 5,321,651 |   | 0.1188 | 0.02369 |   | 0.1311 | 0.02116 |   | 0.116 | 0.04445 | 0.7497 |   | 0.1212 | 0.0001064 |   | - |
| Dominant | 17 | rs1160226 | 13 | 25,271,434 |   | −0.09663 | 0.03322 |   | −0.1051 | 0.02826 |   | −0.09048 | 0.04211 | 0.7497 |   | −0.09989 | 0.0001114 |   | ATP12A |
| Dominant | 18 | rs13278423 | 8 | 87,720,419 |   | −0.1293 | 0.008762 |   | −0.1109 | 0.03276 |   | −0.1019 | 0.03276 | 0.7497 |   | −0.1093 | 0.0001131 |   | CNGB3 |
| Dominant | 19 | rs7592517 | 2 | 76,777,279 |   | 0.1386 | 0.004283 |   | 0.1108 | 0.02292 |   | 0.1029 | 0.02308 | - |   | 0.1004 | 0.0001222 |   | - |
| Dominant | 19 | rs2139502 | 2 | 76,786,845 |   | 0.1386 | 0.004283 |   | 0.1108 | 0.02292 |   | 0.1029 | 0.02308 | 0.7497 |   | 0.1004 | 0.0001222 |   | - |
| Dominant | 21 | rs10956972 | 8 | 87,768,331 |   | 0.1137 | 0.01871 |   | 0.1022 | 0.03598 |   | 0.1251 | 0.009372 | - |   | 0.1041 | 0.0001519 |   | - |
| Dominant | 21 | rs1982563 | 8 | 87,776,019 |   | 0.1137 | 0.01871 |   | 0.1022 | 0.03598 |   | 0.1251 | 0.009372 | 0.7497 |   | 0.1041 | 0.0001519 |   | - |
| Dominant | 23 | rs4963573 | 12 | 24,662,116 |   | −0.0974 | 0.02673 |   | −0.09413 | 0.04208 |   | −0.09436 | 0.03043 | 0.7497 |   | −0.09361 | 0.0001979 |   | SOX5 |
| Dominant | 24 | rs12580224 | 12 | 71,086,426 |   | 0.1173 | 0.01037 |   | 0.1015 | 0.0382 |   | 0.0941 | 0.0436 | 0.7497 |   | 0.09522 | 0.0003004 |   | PTPRR |
| Dominant | 25 | rs6792514 | 3 | 42,429,817 |   | 0.1291 | 0.04677 |   | 0.156 | 0.03299 |   | 0.1051 | 0.04156 | 0.7497 |   | 0.1183 | 0.0006295 |   | - |
| Recessive | 1 | rs966775 | 5 | 174,763,322 |   | 0.1908 | 0.01057 |   | 0.1977 | 0.01704 |   | 0.1931 | 0.0005773 | 0.1068 |   | 0.1919 | 0.0000006958 |   | (DRD1) |
| Recessive | 2 | rs6041532 | 20 | 12,652,435 |   | 0.5168 | 0.001824 |   | 0.3513 | 0.04671 |   | 0.5921 | 0.0118 | 0.3928 |   | 0.455 | 0.00001802 |   | - |
| Recessive | 3 | rs43211 | 19 | 54,652,203 |   | −0.1046 | 0.03218 |   | −0.1699 | 0.01039 |   | −0.1444 | 0.008904 | 0.3928 |   | −0.1278 | 0.00004229 |   | CNOT3 |
| Recessive | 4 | rs4764074 | 12 | 14,428,118 |   | −0.154 | 0.02998 |   | −0.1442 | 0.02867 |   | −0.1595 | 0.01012 | 0.3928 |   | −0.1515 | 0.00005068 |   | - |
| Recessive | 5 | rs2146423 | 9 | 4,657,040 |   | 0.1363 | 0.04086 |   | 0.1478 | 0.0392 |   | 0.2007 | 0.006745 | 0.3928 |   | 0.1479 | 0.0001913 |   | C9orf68 |
| Recessive | 6 | rs12714409 | 2 | 596,532 |   | 0.1057 | 0.04401 |   | 0.1324 | 0.02693 |   | 0.1247 | 0.02672 | 0.5038 |   | 0.1185 | 0.0001978 |   | - |
| Recessive | 7 | rs2759632 | 10 | 10,218,843 |   | −0.2173 | 0.02572 |   | −0.237 | 0.03636 |   | −0.2005 | 0.02973 | 0.5089 |   | −0.2112 | 0.0002074 |   | - |
| Recessive | 8 | rs2199503 | 3 | 119,778,489 |   | 0.1374 | 0.04339 |   | 0.1528 | 0.02812 |   | 0.1276 | 0.04407 | 0.5889 |   | 0.1352 | 0.0003717 |   | GSK3B |
| Recessive | 9 | rs10486791 | 7 | 16,284,326 |   | 0.1441 | 0.04552 |   | 0.1725 | 0.04761 |   | 0.2051 | 0.02432 | 0.5038 |   | 0.1639 | 0.0003742 |   | ISPD, LOC100506025 |
| Recessive | 10 | rs2368473 | 17 | 32,534,215 |   | 0.3506 | 0.04403 |   | 0.1788 | 0.04874 |   | 0.2906 | 0.03274 | 0.5267 |   | 0.2039 | 0.002313 |   | - |

Table 2 Caption: Top candidate SNPs selected from three-stage GWAS for the 0–12 h effect-site MEC.

### 2.3. Identification of Genes and Gene Sets Associated with Estimated MEC of Fentanyl in Patients Who Underwent LAC by Gene-Based and Gene-Set Analyses

Considering that effects of individual markers tend to be too weak to be detected by comprehensive analyses, such as GWASs, which target only single polymorphisms, we conducted gene-based and gene-set analyses, which are statistical methods that are used to analyze multiple genetic markers simultaneously to determine their joint effect. In both analyses using MAGMA software [[43](#B43-ijms-24-08421)], which was made accessible in the FUMA GWAS platform [[44](#B44-ijms-24-08421)], we investigated genes and gene sets that were related to the estimated MEC of fentanyl in a total of 350 patients who underwent LAC. As a result, 921,239 SNPs from the selected candidate genes and gene sets in the additive, dominant, and recessive models were included in the analyses of all patients. The top 20 candidate genes that were found in each genetic model by the gene-based analysis are listed in [Table 3](#ijms-24-08421-t003). In the dominant model, *SERP2*, the top candidate gene, was significantly associated with the 0–6 h plasma MEC (adjusted *p* = 0.02425; [Table 3](#ijms-24-08421-t003), [Figure 1](#ijms-24-08421-f001)B), 0–12 h plasma MEC (adjusted *p* = 0.02438; [Supplementary Table S4](#app1-ijms-24-08421)), and 0–12 h effect-site MEC (adjusted *p* = 0.03635; [Table 4](#ijms-24-08421-t004), [Figure 2](#ijms-24-08421-f002)B). The association between the *SERP2* gene and the 0–6 h effect-site MEC was marginally significant (adjusted *p* = 0.05245; [Supplementary Table S5](#app1-ijms-24-08421)). However, in both the additive and recessive models, none of the genes were significantly associated with the phenotypes ([Table 3](#ijms-24-08421-t003) and [Table 4](#ijms-24-08421-t004); [Supplementary Tables S4 and S5](#app1-ijms-24-08421); and [Figure 1](#ijms-24-08421-f001)A,C and [Figure 2](#ijms-24-08421-f002)A,C). The top 20 candidate gene sets for each phenotype that were found in each genetic model by the gene-set analysis are listed in [Supplementary Tables S6–S9](#app1-ijms-24-08421). As a result, the 0–6 h plasma MEC was significantly associated with the “go_paracrine_signaling” (adjusted *p* = 0.01093), “reactome_free_fatty_acid_receptors” (adjusted *p* = 0.01430), and “go_taste_receptor_activity” (adjusted *p* = 0.03004) gene sets in the additive model ([Supplementary Table S6](#app1-ijms-24-08421)) and the “go_negative_regulation_of_epidermal_cell_differentiation” (adjusted *p* = 0.01728), “go_paracrine_signaling” (adjusted *p* = 0.02210), “go_negative_regulation_of_epidermis_development” (adjusted *p* = 0.03244), and “go_negative_regulation_of_keratinocyte_differentiation” (adjusted *p* = 0.04440) gene sets in the recessive model ([Supplementary Table S6](#app1-ijms-24-08421)). The 0–12 h plasma MEC was significantly associated with the “sotiriou_breast_cancer_grade_1_vs_3_dn” gene set in the additive model (adjusted *p* = 0.04805; [Supplementary Table S7](#app1-ijms-24-08421)). The 0–6 h effect-site MEC was significantly associated with the “go_ccr2_chemokine_receptor_binding” and “go_paracrine_signaling” gene sets in the additive model (adjusted *p* = 0.01342 and 0.03030, respectively; [Supplementary Table S8](#app1-ijms-24-08421)) and significantly associated with the “go_paracrine_signaling” and “go_ccr2_chemokine_receptor_binding” gene sets in the recessive model (adjusted *p* = 0.01030 and 0.02499, respectively; [Supplementary Table S8](#app1-ijms-24-08421)). The 0–12 h effect-site MEC was significantly associated with the “pid_shp2_pathway” gene set in the recessive model (adjusted *p* = 0.02854; [Supplementary Table S9](#app1-ijms-24-08421)). The genes that were included in these gene sets are listed in [Supplementary Table S10](#app1-ijms-24-08421). The *SERP2* gene, which was significantly associated with the phenotypes in the gene-based analysis, was not included in any of the gene sets ([Supplementary Table S10](#app1-ijms-24-08421)). Among these genes, several genes were commonly included in two or three kinds of gene sets ([Supplementary Table S10](#app1-ijms-24-08421)). Eight genes (*EZH2*, *GRHL2*, *HOXA7*, *MSX2*, *REG3A*, *REG3G*, *SRSF6*, and *TP63*) were commonly included in three kinds of gene sets ([Supplementary Table S10](#app1-ijms-24-08421)).

### Table 3.

| Model | Rank | CHR | Gene Start Position | Gene Stop Position | Gene | nSNPs | Z Statistic | p | p a |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Additive | 1 | 2 | 220,378,892 | 220,403,494 | ASIC4 | 6 | 3.8796 | 0.00005232 | 0.90440352 |
| Additive | 2 | 9 | 132,500,610 | 132,515,326 | PTGES | 6 | 3.7421 | 0.000091253 | 1 |
| Additive | 3 | 2 | 220,299,568 | 220,363,009 | SPEG | 15 | 3.7182 | 0.00010034 | 1 |
| Additive | 4 | 2 | 20,448,452 | 20,551,995 | PUM2 | 6 | 3.678 | 0.00011753 | 1 |
| Additive | 5 | X | 135,295,381 | 135,338,641 | MAP7D3 | 6 | 3.6653 | 0.00012353 | 1 |
| Additive | 6 | 11 | 67,195,931 | 67,202,872 | RPS6KB2 | 2 | 3.5064 | 0.00022708 | 1 |
| Additive | 7 | 11 | 51,515,282 | 51,516,211 | OR4C46 | 1 | 3.3757 | 0.0003682 | 1 |
| Additive | 8 | 12 | 122,089,024 | 122,110,537 | MORN3 | 3 | 3.3375 | 0.00042262 | 1 |
| Additive | 9 | 11 | 51,411,378 | 51,412,448 | OR4A5 | 2 | 3.296 | 0.00049041 | 1 |
| Additive | 10 | 17 | 56,597,611 | 56,618,179 | 45173 | 5 | 3.2112 | 0.00066098 | 1 |
| Additive | 11 | 2 | 28,680,012 | 28,866,654 | PLB1 | 66 | 3.2041 | 0.00067735 | 1 |
| Additive | 12 | 12 | 12,813,825 | 12,849,141 | GPR19 | 11 | 3.1993 | 0.00068891 | 1 |
| Additive | 13 | 4 | 169,418,217 | 169,849,608 | PALLD | 119 | 3.1987 | 0.00069036 | 1 |
| Additive | 14 | 15 | 66,679,155 | 66,784,650 | MAP2K1 | 6 | 3.1084 | 0.00094037 | 1 |
| Additive | 15 | 1 | 235,490,665 | 235,507,847 | GGPS1 | 4 | 3.0385 | 0.0011887 | 1 |
| Additive | 16 | 1 | 41,157,320 | 41,237,275 | NFYC | 5 | 3.0258 | 0.0012398 | 1 |
| Additive | 17 | 18 | 24,432,002 | 24,445,782 | AQP4 | 2 | 3.0094 | 0.001309 | 1 |
| Additive | 18 | 17 | 42,325,753 | 42,345,509 | SLC4A1 | 10 | 2.9943 | 0.0013754 | 1 |
| Additive | 19 | 18 | 43,405,477 | 43,424,045 | SIGLEC15 | 6 | 2.9848 | 0.0014187 | 1 |
| Additive | 20 | 11 | 67,202,981 | 67,205,538 | PTPRCAP | 1 | 2.9592 | 0.001542 | 1 |
| Dominant | 1 | 13 | 44,947,801 | 44,971,850 | SERP2 | 5 | 4.6871 | 0.0000013857 | 0.02424975 * |
| Dominant | 2 | 4 | 169,418,217 | 169,849,608 | PALLD | 134 | 3.7987 | 0.000072725 | 1 |
| Dominant | 3 | 20 | 45,186,463 | 45,304,714 | SLC13A3 | 79 | 3.7129 | 0.00010247 | 1 |
| Dominant | 4 | 13 | 25,254,549 | 25,285,921 | ATP12A | 14 | 3.6666 | 0.00012288 | 1 |
| Dominant | 5 | 17 | 4,574,679 | 4,607,632 | PELP1 | 4 | 3.5425 | 0.0001982 | 1 |
| Dominant | 6 | 14 | 57,936,019 | 57,960,585 | C14orf105 | 7 | 3.5263 | 0.00021073 | 1 |
| Dominant | 7 | 17 | 5,402,747 | 5,522,744 | NLRP1 | 37 | 3.5107 | 0.00022346 | 1 |
| Dominant | 8 | 1 | 41,157,320 | 41,237,275 | NFYC | 5 | 3.3447 | 0.00041192 | 1 |
| Dominant | 9 | 10 | 5,435,061 | 5,446,793 | TUBAL3 | 7 | 3.3324 | 0.00043043 | 1 |
| Dominant | 10 | 2 | 218,148,742 | 218,621,316 | DIRC3 | 104 | 3.3216 | 0.00044747 | 1 |
| Dominant | 11 | 13 | 45,007,655 | 45,151,283 | TSC22D1 | 13 | 3.2597 | 0.00055764 | 1 |
| Dominant | 12 | 14 | 104,552,016 | 104,579,098 | ASPG | 6 | 3.254 | 0.00056887 | 1 |
| Dominant | 13 | 7 | 1,509,913 | 1,545,489 | INTS1 | 5 | 3.2398 | 0.00059799 | 1 |
| Dominant | 14 | 10 | 22,823,778 | 23,003,484 | PIP4K2A | 47 | 3.1764 | 0.00074551 | 1 |
| Dominant | 15 | 1 | 153,389,000 | 153,395,701 | S100A7A | 1 | 3.1454 | 0.0008292 | 1 |
| Dominant | 16 | 9 | 22,002,902 | 22,009,362 | CDKN2B | 3 | 3.1361 | 0.00085605 | 1 |
| Dominant | 17 | 11 | 63,580,860 | 63,595,190 | C11orf84 | 5 | 3.1307 | 0.0008719 | 1 |
| Dominant | 18 | 13 | 41,129,804 | 41,240,734 | FOXO1 | 16 | 3.0931 | 0.00099042 | 1 |
| Dominant | 19 | 11 | 123,676,043 | 123,677,095 | OR6M1 | 3 | 3.0782 | 0.0010414 | 1 |
| Dominant | 20 | 17 | 4,613,784 | 4,624,794 | ARRB2 | 1 | 3.0692 | 0.001073 | 1 |
| Recessive | 1 | 2 | 220,378,892 | 220,403,494 | ASIC4 | 6 | 3.8643 | 0.000055694 | 0.930702434 |
| Recessive | 2 | 9 | 132,500,610 | 132,515,326 | PTGES | 6 | 3.8064 | 0.000070511 | 1 |
| Recessive | 3 | 11 | 51,515,282 | 51,516,211 | OR4C46 | 1 | 3.6687 | 0.0001219 | 1 |
| Recessive | 4 | 11 | 67,195,931 | 67,202,872 | RPS6KB2 | 2 | 3.4441 | 0.00028648 | 1 |
| Recessive | 5 | 11 | 51,411,378 | 51,412,448 | OR4A5 | 2 | 3.4435 | 0.00028713 | 1 |
| Recessive | 6 | 12 | 122,089,024 | 122,110,537 | MORN3 | 3 | 3.3926 | 0.00034621 | 1 |
| Recessive | 7 | 2 | 220,299,568 | 220,363,009 | SPEG | 15 | 3.3645 | 0.0003834 | 1 |
| Recessive | 8 | 2 | 20,448,452 | 20,551,995 | PUM2 | 6 | 3.3366 | 0.00042402 | 1 |
| Recessive | 9 | 18 | 43,405,477 | 43,424,045 | SIGLEC15 | 6 | 3.202 | 0.00068241 | 1 |
| Recessive | 10 | 11 | 55,563,032 | 55,563,976 | OR5D14 | 2 | 3.177 | 0.00074398 | 1 |
| Recessive | 11 | 17 | 56,597,611 | 56,618,179 | 44808 | 5 | 3.1273 | 0.00088212 | 1 |
| Recessive | 12 | 15 | 66,679,155 | 66,784,650 | MAP2K1 | 6 | 3.1118 | 0.00092985 | 1 |
| Recessive | 13 | 1 | 45,240,923 | 45,244,451 | RPS8 | 1 | 3.1037 | 0.0009556 | 1 |
| Recessive | 14 | 11 | 60,197,062 | 60,222,687 | MS4A5 | 8 | 3.004 | 0.0013323 | 1 |
| Recessive | 15 | 17 | 41,717,756 | 41,739,322 | MEOX1 | 5 | 2.9757 | 0.0014618 | 1 |
| Recessive | 16 | 1 | 44,398,992 | 44,402,913 | ARTN | 2 | 2.9727 | 0.0014762 | 1 |
| Recessive | 17 | 9 | 19,408,925 | 19,452,018 | ACER2 | 9 | 2.9601 | 0.0015377 | 1 |
| Recessive | 18 | 12 | 12,813,825 | 12,849,141 | GPR19 | 11 | 2.9597 | 0.0015395 | 1 |
| Recessive | 19 | 12 | 54,104,903 | 54,121,529 | CALCOCO1 | 9 | 2.9198 | 0.0017513 | 1 |
| Recessive | 20 | 11 | 27,676,440 | 27,743,605 | BDNF | 10 | 2.9176 | 0.0017634 | 1 |

Table 3 Caption: Top 20 candidate genes selected from gene-based analysis for the 0–6 h plasma MEC.

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1c4c/10179231/93d66926f3a2/ijms-24-08421-g001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=10179231_ijms-24-08421-g001.jpg)

Manhattan plot of results of gene-based analysis for the 0–6 h plasma MEC. (A) Analytical plot in the additive model. (B) Analytical plot in the dominant model. (C) Analytical plot in the recessive model. The dashed red line shows the significant association threshold.

### Table 4.

| Model | Rank | CHR | Gene Start Position | Gene Stop Position | Gene | nSNPs | Z Statistic | p | p a |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Additive | 1 | 2 | 220,378,892 | 220,403,494 | ASIC4 | 6 | 3.9085 | 0.000046429 | 0.802571694 |
| Additive | 2 | 9 | 132,500,610 | 132,515,326 | PTGES | 6 | 3.7514 | 0.000087928 | 1 |
| Additive | 3 | 2 | 220,299,568 | 220,363,009 | SPEG | 15 | 3.6267 | 0.00014354 | 1 |
| Additive | 4 | 2 | 20,448,452 | 20,551,995 | PUM2 | 6 | 3.608 | 0.00015431 | 1 |
| Additive | 5 | X | 135,295,381 | 135,338,641 | MAP7D3 | 6 | 3.5274 | 0.00020985 | 1 |
| Additive | 6 | 4 | 169,418,217 | 169,849,608 | PALLD | 119 | 3.4411 | 0.00028971 | 1 |
| Additive | 7 | 11 | 67,195,931 | 67,202,872 | RPS6KB2 | 2 | 3.3222 | 0.0004466 | 1 |
| Additive | 8 | 10 | 18,240,768 | 18,332,221 | SLC39A12 | 37 | 3.2132 | 0.00065641 | 1 |
| Additive | 9 | 2 | 28,680,012 | 28,866,654 | PLB1 | 66 | 3.1911 | 0.00070872 | 1 |
| Additive | 10 | 1 | 235,490,665 | 235,507,847 | GGPS1 | 4 | 3.1896 | 0.00071235 | 1 |
| Additive | 11 | 12 | 122,089,024 | 122,110,537 | MORN3 | 3 | 3.162 | 0.00078353 | 1 |
| Additive | 12 | 2 | 178,477,720 | 178,483,694 | TTC30A | 6 | 3.1536 | 0.00080626 | 1 |
| Additive | 13 | 11 | 51,515,282 | 51,516,211 | OR4C46 | 1 | 3.1356 | 0.0008575 | 1 |
| Additive | 14 | 11 | 51,411,378 | 51,412,448 | OR4A5 | 2 | 3.118 | 0.00091033 | 1 |
| Additive | 15 | 1 | 41,157,320 | 41,237,275 | NFYC | 5 | 3.0802 | 0.0010343 | 1 |
| Additive | 16 | 17 | 42,325,753 | 42,345,509 | SLC4A1 | 10 | 2.9505 | 0.0015861 | 1 |
| Additive | 17 | 13 | 44,947,801 | 44,971,850 | SERP2 | 3 | 2.942 | 0.0016303 | 1 |
| Additive | 18 | 12 | 12,813,825 | 12,849,141 | GPR19 | 11 | 2.9204 | 0.0017477 | 1 |
| Additive | 19 | 4 | 178,163,693 | 178,169,927 | RP11-487E13.1 | 3 | 2.9023 | 0.0018523 | 1 |
| Additive | 20 | 2 | 242,673,994 | 242,708,231 | D2HGDH | 2 | 2.8772 | 0.0020061 | 1 |
| Dominant | 1 | 13 | 44,947,801 | 44,971,850 | SERP2 | 5 | 4.6035 | 0.0000020769 | 0.03634575 * |
| Dominant | 2 | 4 | 169,418,217 | 169,849,608 | PALLD | 134 | 3.9649 | 0.000036709 | 0.6424075 |
| Dominant | 3 | 20 | 45,186,463 | 45,304,714 | SLC13A3 | 79 | 3.7005 | 0.00010758 | 1 |
| Dominant | 4 | 17 | 4,574,679 | 4,607,632 | PELP1 | 4 | 3.6432 | 0.00013461 | 1 |
| Dominant | 5 | 13 | 25,254,549 | 25,285,921 | ATP12A | 14 | 3.6374 | 0.0001377 | 1 |
| Dominant | 6 | 14 | 57,936,019 | 57,960,585 | C14orf105 | 7 | 3.4622 | 0.00026792 | 1 |
| Dominant | 7 | 1 | 41,157,320 | 41,237,275 | NFYC | 5 | 3.4055 | 0.00033018 | 1 |
| Dominant | 8 | 17 | 5,402,747 | 5,522,744 | NLRP1 | 37 | 3.3951 | 0.00034301 | 1 |
| Dominant | 9 | 10 | 22,823,778 | 23,003,484 | PIP4K2A | 47 | 3.3317 | 0.00043156 | 1 |
| Dominant | 10 | 7 | 1,509,913 | 1,545,489 | INTS1 | 5 | 3.2775 | 0.00052371 | 1 |
| Dominant | 11 | 10 | 5,435,061 | 5,446,793 | TUBAL3 | 7 | 3.2772 | 0.00052424 | 1 |
| Dominant | 12 | 8 | 87,878,670 | 88,627,447 | CNBD1 | 106 | 3.2034 | 0.00067909 | 1 |
| Dominant | 13 | 14 | 104,552,016 | 104,579,098 | ASPG | 6 | 3.1844 | 0.00072522 | 1 |
| Dominant | 14 | 1 | 33,979,609 | 34,631,443 | CSMD2 | 246 | 3.1422 | 0.00083837 | 1 |
| Dominant | 15 | 2 | 218,148,742 | 218,621,316 | DIRC3 | 104 | 3.1384 | 0.00084928 | 1 |
| Dominant | 16 | 13 | 112,240,548 | 112,324,955 | RP11-65D24.2 | 22 | 3.1144 | 0.00092144 | 1 |
| Dominant | 17 | 13 | 45,007,655 | 45,151,283 | TSC22D1 | 13 | 3.096 | 0.00098065 | 1 |
| Dominant | 18 | 3 | 38,029,550 | 38,048,679 | VILL | 6 | 3.0576 | 0.0011155 | 1 |
| Dominant | 19 | 1 | 153,389,000 | 153,395,701 | S100A7A | 1 | 3.0495 | 0.001146 | 1 |
| Dominant | 20 | 4 | 169,277,886 | 169,458,937 | DDX60L | 75 | 3.0157 | 0.0012818 | 1 |
| Recessive | 1 | 2 | 220,378,892 | 220,403,494 | ASIC4 | 6 | 3.9596 | 0.000037539 | 0.627314229 |
| Recessive | 2 | 9 | 132,500,610 | 132,515,326 | PTGES | 6 | 3.8807 | 0.000052076 | 0.870242036 |
| Recessive | 3 | 11 | 51,515,282 | 51,516,211 | OR4C46 | 1 | 3.4626 | 0.0002675 | 1 |
| Recessive | 4 | 2 | 20,448,452 | 20,551,995 | PUM2 | 6 | 3.3671 | 0.00037976 | 1 |
| Recessive | 5 | 11 | 51,411,378 | 51,412,448 | OR4A5 | 2 | 3.3374 | 0.00042284 | 1 |
| Recessive | 6 | 11 | 67,195,931 | 67,202,872 | RPS6KB2 | 2 | 3.2775 | 0.00052365 | 1 |
| Recessive | 7 | 2 | 220,299,568 | 220,363,009 | SPEG | 15 | 3.2573 | 0.00056237 | 1 |
| Recessive | 8 | 2 | 178,477,720 | 178,483,694 | TTC30A | 6 | 3.235 | 0.00060813 | 1 |
| Recessive | 9 | 12 | 122,089,024 | 122,110,537 | MORN3 | 3 | 3.2166 | 0.00064853 | 1 |
| Recessive | 10 | 11 | 55,563,032 | 55,563,976 | OR5D14 | 2 | 3.0495 | 0.0011462 | 1 |
| Recessive | 11 | 7 | 44,836,279 | 44,864,163 | PPIA | 3 | 3.0277 | 0.0012321 | 1 |
| Recessive | 12 | 1 | 44,398,992 | 44,402,913 | ARTN | 2 | 2.9601 | 0.0015375 | 1 |
| Recessive | 13 | 16 | 88,519,725 | 88,603,424 | ZFPM1 | 10 | 2.9384 | 0.0016497 | 1 |
| Recessive | 14 | 2 | 204,259,068 | 204,400,133 | RAPH1 | 9 | 2.894 | 0.001902 | 1 |
| Recessive | 15 | 4 | 156,129,781 | 156,138,230 | NPY2R | 2 | 2.8848 | 0.0019584 | 1 |
| Recessive | 16 | 2 | 242,673,994 | 242,708,231 | D2HGDH | 2 | 2.874 | 0.0020268 | 1 |
| Recessive | 17 | 17 | 26,975,374 | 26,989,207 | SDF2 | 1 | 2.8623 | 0.002103 | 1 |
| Recessive | 18 | 1 | 235,490,665 | 235,507,847 | GGPS1 | 4 | 2.8614 | 0.0021089 | 1 |
| Recessive | 19 | 1 | 44,440,159 | 44,443,967 | ATP6V0B | 3 | 2.8318 | 0.002314 | 1 |
| Recessive | 20 | 2 | 204,192,942 | 204,312,446 | ABI2 | 6 | 2.8279 | 0.0023427 | 1 |

Table 4 Caption: Top 20 candidate genes selected from gene-based analysis for the 0–12 h effect-site MEC.

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1c4c/10179231/917b3f93b6a7/ijms-24-08421-g002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=10179231_ijms-24-08421-g002.jpg)

Manhattan plot of results of gene-based analysis for the 0–12 h effect-site MEC. (A) Analytical plot in the additive model. (B) Analytical plot in the dominant model. (C) Analytical plot in the recessive model. The dashed red line shows the significant association threshold.

## 3. Discussion

Although human genetic variants that are associated with human responsiveness and sensitivity to opioids have been explored by adopting opioid analgesics that are required for the treatment of disease-related pain, chronic pain, and perioperative/postoperative pain as the main endpoint [[10](#B10-ijms-24-08421),[11](#B11-ijms-24-08421),[12](#B12-ijms-24-08421),[13](#B13-ijms-24-08421),[14](#B14-ijms-24-08421),[15](#B15-ijms-24-08421),[16](#B16-ijms-24-08421),[17](#B17-ijms-24-08421),[18](#B18-ijms-24-08421),[19](#B19-ijms-24-08421),[20](#B20-ijms-24-08421),[21](#B21-ijms-24-08421),[22](#B22-ijms-24-08421),[23](#B23-ijms-24-08421),[24](#B24-ijms-24-08421),[25](#B25-ijms-24-08421),[26](#B26-ijms-24-08421),[27](#B27-ijms-24-08421),[28](#B28-ijms-24-08421),[29](#B29-ijms-24-08421),[30](#B30-ijms-24-08421),[31](#B31-ijms-24-08421),[32](#B32-ijms-24-08421),[33](#B33-ijms-24-08421)], plasma and effect-site MECs have not been investigated in genetic studies, likely because of difficulties in measuring actual values of plasma and effect-site MECs. To comprehensively explore genetic factors that underlie large individual differences in fentanyl responsiveness and sensitivity after LAC, we first conducted a GWAS in this cohort of patients, focusing on plasma and effect-site MECs of fentanyl that were estimated with a pharmacokinetic simulation model [[40](#B40-ijms-24-08421),[41](#B41-ijms-24-08421),[42](#B42-ijms-24-08421)]. As a result of the GWAS in surgical patients, 8–28 SNPs were selected as the top candidate SNPs that were significantly associated with a plasma or effect-site MEC that was averaged over the 0–6 h or 0–12 h postoperative period after LAC in all of the additive, dominant, and recessive models ([Table 1](#ijms-24-08421-t001) and [Table 2](#ijms-24-08421-t002); [Supplementary Tables S2 and S3](#app1-ijms-24-08421)). Among these, the rs966775 SNP that mapped to 5p13 had highly significant associations with 0–6 h plasma and 0–12 h effect-site MECs ([Table 1](#ijms-24-08421-t001) and [Table 2](#ijms-24-08421-t002)). A gene that is located near the region of this SNP was *DRD1*, which encodes the dopamine D_1_ receptor. Altogether, our data indicated that the rs966775 SNP near the *DRD1* gene significantly affected fentanyl sensitivity. Compared with non-carriers, G-allele carriers of this SNP were associated with higher plasma and/or effect-site MECs of fentanyl, suggesting that G allele carriers would feel pain at a higher plasma/effect-site fentanyl concentration and thus would require more frequent self-dosing of fentanyl for adequate pain control. Although we acknowledge that the sample size of 350 patients may not be sufficiently large to draw definitive conclusions about genetic markers that contribute to individual differences in the MEC of fentanyl and that further research is needed with larger sample sizes and greater statistical power to validate our findings, the present results suggest that the rs966775 SNP could serve as a marker that predicts the efficacy of opioid analgesics for the treatment of postoperative pain.

In clinical postoperative pain management using patient-controlled analgesia (PCA), continuous pain relief should be achieved if the plasma opioid concentration is maintained in excess of the MEC, whereas pain will return if it decreases to the MEC. Thus, the MEC is indicated by the need for an additional intravenous (i.v.) opioid because of recurring pain [[4](#B4-ijms-24-08421),[5](#B5-ijms-24-08421)]. The MEC of opioids varies depending on the type of surgery and intensity of postoperative pain. It gradually decreases with a decreasing intensity of postoperative pain [[3](#B3-ijms-24-08421),[4](#B4-ijms-24-08421),[5](#B5-ijms-24-08421),[6](#B6-ijms-24-08421)]. Nevertheless, the MEC remains relatively constant within each patient over the postoperative period but varies widely among patients even after the same type of surgery [[3](#B3-ijms-24-08421),[4](#B4-ijms-24-08421),[5](#B5-ijms-24-08421)]. Associations between genetic variants and MECs of opioids have not been investigated in genetic studies, likely because of difficulties in repeatedly measuring actual plasma opioid concentrations. However, opioids act on the effect site and not on plasma, and pharmacokinetic simulation models can predict plasma concentrations with acceptable accuracy [[35](#B35-ijms-24-08421),[45](#B45-ijms-24-08421),[46](#B46-ijms-24-08421)] and estimate effect-site concentrations that are not measurable in humans [[34](#B34-ijms-24-08421),[36](#B36-ijms-24-08421),[37](#B37-ijms-24-08421)]. Therefore, simulation models have been widely used in clinical studies [[6](#B6-ijms-24-08421),[34](#B34-ijms-24-08421),[35](#B35-ijms-24-08421),[36](#B36-ijms-24-08421),[37](#B37-ijms-24-08421),[38](#B38-ijms-24-08421),[39](#B39-ijms-24-08421)], including one that evaluated the plasma MEC of fentanyl [[6](#B6-ijms-24-08421)]. Using MECs of fentanyl that were determined with a simulation model, we conducted a GWAS and found that the rs966775 SNP significantly affected fentanyl sensitivity.

As mentioned above, we conducted GWASs of phenotypes that are related to opioid sensitivity and candidate gene studies [[15](#B15-ijms-24-08421),[16](#B16-ijms-24-08421),[17](#B17-ijms-24-08421),[18](#B18-ijms-24-08421),[19](#B19-ijms-24-08421),[20](#B20-ijms-24-08421),[21](#B21-ijms-24-08421),[22](#B22-ijms-24-08421),[23](#B23-ijms-24-08421),[24](#B24-ijms-24-08421),[25](#B25-ijms-24-08421),[26](#B26-ijms-24-08421),[27](#B27-ijms-24-08421),[28](#B28-ijms-24-08421),[29](#B29-ijms-24-08421),[30](#B30-ijms-24-08421),[31](#B31-ijms-24-08421),[32](#B32-ijms-24-08421),[33](#B33-ijms-24-08421)]. Several candidate SNPs were found to be associated with phenotypes that are related to opioid sensitivity/pain. Among these are the rs2076222 SNP in the *LAMB3* gene region, which was associated with postoperative 24 h fentanyl requirements in subjects who underwent LAC. Although this SNP was also found to be nominally significantly associated with the estimated plasma and effect-site MECs in the present study in the additive model (combined *β* = −0.07784, nominal *p* = 0.00232, for the 0–6 h plasma MEC; combined *β* = −0.07517, nominal *p* = 0.00310, for the 0–12 h plasma MEC; combined *β* = −0.09432, nominal *p* = 0.00250, for the 0–6 h effect-site MEC; combined *β* = −0.08721, nominal *p* = 0.00369, for the 0–12 h effect-site MEC) and in the recessive model (combined *β* = −0.14420, nominal *p* = 0.00409, for the 0–6 h plasma MEC; combined *β* = −0.13780, nominal *p* = 0.00581, for the 0–12 h plasma MEC; combined *β* = −0.17340, nominal *p* = 0.00469, for the 0–6 h effect-site MEC; combined *β* = −0.15910, nominal *p* = 0.00706, for the 0–12 h effect-site MEC), the nominally significant associations would likely be attributable to the strong correlation among the values for postoperative 24 h fentanyl requirements and estimated plasma and effect-site MECs in the present study. The associations between other candidate SNPs that we identified in previous studies as candidates for opioid sensitivity and the estimated plasma and effect-site MECs in the present study were not even nominally significant (details not shown). These results might indicate the general difficulty in replicating results of human genetic association studies or reflect phenotypical differences between postoperative analgesic requirements per se and the plasma and effect-site MECs that were estimated with a pharmacokinetic simulation model in the present study.

The best candidate SNP in the present study was rs966775, which was found in the intergenic region. The protein-coding gene on chromosome 5 that was nearest to this SNP site was the *DRD1* gene. This gene encodes the dopamine D_1_ receptor, which is the most abundant dopamine receptor in the central nervous system. Although the D_1_ receptor has been shown to be involved in mechanisms of opioid analgesia in animal studies [[47](#B47-ijms-24-08421),[48](#B48-ijms-24-08421),[49](#B49-ijms-24-08421),[50](#B50-ijms-24-08421)], the impact of this SNP on the expression and function of the *DRD1* gene product is not known but presumably may not be profound because this SNP is located more than 100 kbp from the gene region ([Supplementary Figure S3](#app1-ijms-24-08421)). The rs966775 SNP has not been previously reported to be associated with any phenotypes to date. Although this SNP was not in strong LD (*r*^2^ ≥ 0.80) with any other neighboring SNPs in our data ([Supplementary Figure S3](#app1-ijms-24-08421)), when these SNPs were referenced in HaploReg v. 4.1 and SNPinfo Web Server (accessed on 30 January 2023) [[51](#B51-ijms-24-08421),[52](#B52-ijms-24-08421)], they were in strong LD with the rs7725278, rs897747, rs2382021, rs2890873, rs3955076, rs76895738, and rs10060502 SNPs (*r*^2^ ≥ 0.80) and were moderately linked to the rs12652255 SNP (*r*^2^ = 0.68) in Asian populations, including Japan. HaploReg v. 4.1 also showed that the rs966775 SNP could change six motifs for DNA-binding proteins and overlaps with an enhancer in the fat and skin. Nevertheless, none of these SNPs were significantly associated with mRNA expression levels of any genes in any tissues according to the GTEx portal (accessed on 30 January 2023) [[53](#B53-ijms-24-08421)], suggesting that it is unlikely that these SNPs influence variations in opioid sensitivity among individuals by influencing the mRNA expression of some genes. The rs12652255 SNP was reported to be associated with the efficacy of Drotrecogin alfa, a drug with antithrombotic, profibrinolytic, anti-inflammatory, and cytoprotective properties in patients with severe sepsis [[54](#B54-ijms-24-08421)], but its contribution to the efficacy of opioid analgesics remains unknown.

Among the candidate SNPs that were selected in our GWAS for the 0–12 h plasma MEC in the dominant model, the rs9533839 SNP was included, which was annotated as the *SERP2* gene ([Supplementary Table S2](#app1-ijms-24-08421)). This gene was also significantly associated with the same trait ([Supplementary Table S4](#app1-ijms-24-08421)) and the 0–6 h plasma and 0–12 h effect-site MECs ([Table 3](#ijms-24-08421-t003) and [Table 4](#ijms-24-08421-t004)) in the gene-based analysis. The *SERP2* gene encodes stress-associated endoplasmic reticulum protein family member 2 (SERP2), which is predicted to be involved in the endoplasmic reticulum unfolded protein response and protein glycosylation. Although *SERP2* mRNA is known to be highly expressed in the brain, followed by the testis, according to the National Center for Biotechnology Information (NCBI) database, the functional relationship between this protein and the opioid system is unknown. In human cytogenetic studies, microdeletions of the *SERP2* gene were reported to be associated with acute lymphoblastic leukemias in children with Down syndrome [[55](#B55-ijms-24-08421)], and focal deletions of this gene were also identified in 2–6% of adult cases of acute lymphoblastic leukemia [[56](#B56-ijms-24-08421)]. However, no genetic variants, such as SNPs, in this gene region have been reported to be associated with diseases or other phenotypes. HaploReg v. 4.1 showed that the rs9533839 SNP could change six motifs for DNA-binding proteins and overlaps with an enhancer in nine tissues, and this SNP was found to be significantly associated with mRNA expression levels of the *TUSC8* gene in the prostate, breast (mammary tissue), and minor salivary gland according to the GTEx portal (accessed on 30 January 2023). The *TUSC8* gene encodes a non-coding RNA (ncRNA), TUSC8, and this ncRNA reportedly enhances the cisplatin sensitivity of non-small-cell lung cancer cells by regulating vascular endothelial growth factor A (VEGFA) [[57](#B57-ijms-24-08421)], although the involvement of this ncRNA in opioid sensitivity remains unknown.

In the gene-set analysis, several significant associations were also found ([Supplementary Tables S6–S9](#app1-ijms-24-08421)). Some of the genes that were included in the gene sets were included in two or three kinds of gene sets ([Supplementary Table S10](#app1-ijms-24-08421)). Among the gene sets that were included in two kinds of gene sets, the *VEGFA* gene ([Supplementary Table S10](#app1-ijms-24-08421)) is notable because VEGF-A protein, which is encoded by the *VEGFA* gene, is known to be involved in angiogenesis through activation of the opioid system [[58](#B58-ijms-24-08421)], although opioids could also exert a proangiogenic effect at low doses but an antiangiogenic (toxic) effect at high doses [[59](#B59-ijms-24-08421)]. Intriguingly, the *ANGPT1* gene was included in the “pid_shp2_pathway” gene set, which was significantly associated with the 0–12 h effect-site MEC in the recessive model ([Supplementary Table S9](#app1-ijms-24-08421)). The *ANGPT1* gene encodes angiopoietin-1, a secreted glycoprotein that is a member of the angiopoietin family. Angiopoietin-1 is also known to be involved in angiogenesis. Mice that were engineered to lack angiopoietin-1 exhibited angiogenic deficits [[60](#B60-ijms-24-08421)]. Although more studies are required, angiogenesis, with the involvement of angiopoietin-1, could also be modulated by actions of opioids, and the rs1283671 and rs1283720 SNPs within this gene region were found to be significantly associated with average daily opioid requirements for the treatment of cancer pain in our previous GWAS [[33](#B33-ijms-24-08421)].

## 4. Materials and Methods

### 4.1. Patients

#### 4.1.1. Patients Who Underwent LAC

Enrolled in the study were 351 adult patients (20–85 years old, 218 males and 133 females) without severe coexisting systemic disease (American Society of Anesthesiologists Physical Status [ASA-PS] I or II) who were scheduled to undergo LAC for colon or rectal cancer at Saitama Medical University International Medical Center. Excluded were patients with severe coexisting disease (ASA-PS ≥ III), those taking pain medication for chronic pain, and those who were unlikely to be able to use a PCA pump (e.g., because of dementia). All of the individuals who were included in the study were of Japanese origin. Peripheral blood samples were collected from these subjects for gene analysis. Detailed demographic and clinical data of the subjects are provided in [Supplementary Table S1](#app1-ijms-24-08421) and our previous reports [[22](#B22-ijms-24-08421),[25](#B25-ijms-24-08421)].

The study was conducted according to guidelines of the Declaration of Helsinki and approved by the Institutional Review Board or Ethics Committee of Saitama Medical University International Medical Center and Tokyo Metropolitan Institute of Medical Science (Tokyo, Japan). Written informed consent was obtained from all of the patients.

#### 4.1.2. Surgical Protocol and Clinical Data

The protocols for anesthesia, surgery, and postoperative pain management and clinical data are detailed in our previous reports [[22](#B22-ijms-24-08421),[25](#B25-ijms-24-08421)]. Briefly, general anesthesia was induced with fentanyl (0.1 mg), propofol (1–2 mg/kg), and rocuronium (0.8 mg/kg). After tracheal intubation, the inhalation of sevoflurane (1.5% in inspired concentration) and continuous infusion of remifentanil (0.25 µg/kg/min) were started. General anesthesia was thus maintained with sevoflurane, remifentanil, and rocuronium. At the end of surgery, remifentanil and sevoflurane were discontinued, and fentanyl (usually ≥ 0.1 mg) was given for immediate postoperative pain relief. The average remifentanil infusion rate (in µg/kg/min) during surgery was calculated by dividing the total dose of remifentanil that was required during surgery by the duration of surgery and body weight. When patients complained of even mild abdominal pain, fentanyl was given in increments of 0.05 mg until sufficient pain relief was achieved.

Postoperative pain was then managed with i.v. fentanyl PCA using a PCA pump (CADD-Legacy Model 6300, Smiths Medical Japan, Tokyo, Japan) that was filled with 1000 μg fentanyl diluted with normal saline to a total volume of 100 mL. The demand dose, dose lockout time, maximum allowable demand dose per hour, and continuous rate were set at 20 μg (2 mL), 5 min, 12 times (240 μg), and zero, respectively. Patient-controlled analgesia was principally continued for 24 h postoperatively. In cases of inadequate analgesia, i.v. flurbiprofen axetil (50 mg) or pentazocine (30 mg) was administered as a rescue analgesic. Severe postoperative nausea and vomiting were treated with i.v. droperidol (2.5 mg) or metoclopramide (10 mg).

Postoperative pain at rest was assessed on an 11-point numerical rating scale (0, no pain; 10, the worst pain imaginable). Sedation was assessed on a 4-point scale (0, awake and alert; 1, drowsy; 2, mostly asleep but easy to rouse; 3, asleep and difficult to rouse). Postoperative nausea and vomiting were assessed on a 4-point scale (0, no nausea or vomiting; 1, mild nausea; 2, severe nausea; 3, retching or vomiting). Postoperative pain scores, sedation scores, postoperative nausea and vomiting scores, respiratory rates, the cumulative number of PCA doses that were actually given to the patient, and the cumulative number of PCA doses that were attempted were recorded on a data collection sheet 2, 4, 6, 12, 18, and 24 h after surgery.

PCA fentanyl consumptions over 6 h, 12 h, and 24 h periods were calculated as cumulative doses of fentanyl that were actually given to patients via the PCA pump during the first 6 h, 12 h, and 24 h postoperative periods, respectively. The 6 h, 12 h, and 24 h total postoperative fentanyl requirements were calculated as sums of the i.v. fentanyl dose that was given around the end of surgery and 6 h, 12 h, and 24 h PCA fentanyl consumptions, respectively. Patient-controlled analgesia fentanyl consumptions and total postoperative fentanyl requirements were normalized to body weight. The 6 h, 12 h, and 24 h numbers of locked out doses were the differences between the cumulative number of doses attempted and doses that were given at 6, 12, and 24 h after surgery, respectively.

Plasma and effect-site concentrations of fentanyl over the 24 h postoperative period were estimated in each patient using BeConSim Monitoring ([http://www.masuinet.com](http://www.masuinet.com); accessed on 1 January 2020; [Supplementary Figure S1](#app1-ijms-24-08421))—a pharmacokinetic simulation program that was developed by Masui (2010) [[40](#B40-ijms-24-08421)] ([Supplementary Figure S4](#app1-ijms-24-08421)) based on Shafer’s three compartment model [[61](#B61-ijms-24-08421)]—by inputting relevant clinical data, including age, sex, height, weight, the fentanyl dose that was given around the end of surgery, and subsequent PCA fentanyl consumption profiles over the 24 h postoperative period. A pair of plasma and effect-site MECs of fentanyl were indicated by plasma and effect-site fentanyl concentrations that were estimated immediately before each self-dosing of fentanyl. All pairs of plasma and effect-site MECs in each patient were averaged over the 6 h, 12 h, and 24 h postoperative periods to determine pairs of average plasma and effect-site MECs over these periods, which were expressed as the 0–6 h, 0–12 h, and 0–24 h plasma and effect-site MECs, respectively. Because many patients had completely consumed PCA fentanyl (1000 μg) by 24 h postoperatively and not by 12 h postoperatively, 0–6 h and 0–12 h plasma and effect-site MECs and not 0–24 h plasma and effect-site MECs were used for the main study endpoints. Detailed clinical data of the subjects are detailed in [Supplementary Table S1](#app1-ijms-24-08421).

### 4.2. Whole-Genome Genotyping, Quality Control, and Gene-Based and Gene-Set Analyses

#### 4.2.1. Whole-Genome Genotyping and Quality Control

For patients who underwent LAC, 10 mL of venous blood was sampled during anesthesia for the later preparation of genomic DNA specimens. After total genomic DNA was extracted from whole-blood samples using standard procedures and the concentration was adjusted to 100 ng/μL, whole-genome genotyping was performed using the Infinium Assay II with an iScan system (Illumina, San Diego, CA, USA) according to the manufacturer’s instructions. A total of 921,239 SNP markers survived the entire quality control filtration process and were used for the GWAS (detailed in our previous report) [[22](#B22-ijms-24-08421)]. Two kinds of BeadChips were used for genotyping 256 and 95 samples, respectively: HumanOmniExpressExome-8 v. 1.0 (total markers: 951,117) and HumanOmniExpressExome-8 v. 1.1 (total markers: 958,178). Approximately 926,000 SNP markers were commonly included in all of the BeadChips. Quality control was properly performed the same way as in our previous report [[22](#B22-ijms-24-08421),[25](#B25-ijms-24-08421)].

For phenotypes of the estimated 0–6 h plasma MEC and 0–12 h effect-site MEC, log QQ *p*-value plots as a result of the GWAS for the combined 351 samples were subsequently drawn to check the pattern of the generated *p*-value distribution, in which the observed *p* values against the values that were expected from the null hypothesis of a uniform distribution, calculated as −log10 (*p* value), were plotted for each model. All of the plots were mostly concordant with the expected line (y = x), especially over the range of 0 < −log10 (*p* value) < 4 for each model, indicating no apparent population stratification of the samples that were used in the study ([Supplementary Figures S1 and S2](#app1-ijms-24-08421)).

#### 4.2.2. Gene-Based and Gene-Set Analyses

Gene-based and gene-set approaches were adopted with Multi-marker Analysis of GenoMic Annotation (MAGMA) v. 1.06 [[43](#B43-ijms-24-08421)], which is also available on the Functional Mapping and Annotation of Genome-Wide Association Studies (FUMA GWAS) v. 1.3.9 platform [[44](#B44-ijms-24-08421)], to better understand genetic backgrounds and molecular mechanisms that underlie complex traits, such as opioid sensitivity in patients who underwent LAC. To examine the combined relationship between all genetic markers in the gene and the phenotype, genetic marker data were aggregated to the level of full genes in the gene-based analysis. Similarly, individual genes were compiled into groupings of genes with similar biological, functional, or other properties for the gene-set analysis. Gene-set analyses can thus shed light on the role that particular biological pathways or cellular processes may play in the genetic basis of a trait [[43](#B43-ijms-24-08421)]. In these analyses, associations were explored for genes on autosomes 1–22 and the X chromosome, and the window of the genes to assign SNPs was set to 20 kb, thereby assigning SNPs within the 20 kb window of the gene (both sides) to that gene. For the reference panel, the 1000 Genome Phase3 EAS population was selected ([http://ftp.1000genomes.ebi.ac.uk/vol1/ftp/release/20130502](http://ftp.1000genomes.ebi.ac.uk/vol1/ftp/release/20130502); accessed on 18 January 2023). In the gene-set analysis, gene sets were defined using the Molecular Signatures Database (MSigDB) v. 7.5.1 ([https://www.gsea-msigdb.org/gsea/msigdb](https://www.gsea-msigdb.org/gsea/msigdb); accessed on 18 January 2023) [[62](#B62-ijms-24-08421)]. A total of 10,678 gene sets (curated gene sets: 4761, Gene Ontology [GO] terms: 5917) from MsigDB were tested. In both analyses, Bonferroni correction for multiple testing was performed for all tested genes and gene sets. Adjusted values of *p* < 0.05 in the results were considered significant. The FUMA GWAS platform was also used for the visualization of QQ plots for the GWAS results and Manhattan plots for the gene-based analysis results.

### 4.3. Statistical Analysis

A three-stage GWAS was conducted for patients who underwent LAC to investigate the association between opioid sensitivity after surgery and the 921,239 SNPs that met the quality control criteria in a total of 351 subjects (117, 117, and 117 subjects for the first-, second-, and final-stage analyses, respectively) for whom postoperative clinical data were available, as described in our previous report [[22](#B22-ijms-24-08421)]. As an index of opioid sensitivity after surgery, the estimated 0–6 h plasma MEC, 0–12 h plasma MEC, 0–6 h effect-site MEC, and 0–12 h effect-site MEC were used because these calculated values were expected to reflect the efficacy of fentanyl in each individual. Prior to the analyses, the quantitative values (ng/mL) were natural-log-transformed for approximation to the normal distribution according to the following formula: Value for analyses = Ln (1 + MEC value [ng/mL]). To explore the association between the SNPs and phenotypes, linear regression analyses were conducted in each stage of the analysis, in which the MEC value (ng/mL; log-transformed) and the genotype data for each SNP were incorporated as dependent and independent variables, respectively, with covariates that were found to be strongly associated with the dependent variable in a preliminary study. Male genotypes were not included in the analysis of X chromosome markers, whereas both male and female individuals were included in the association study for autosomal markers. Additive, dominant, and recessive genetic models for each minor allele were used for the analyses because of the previously insufficient knowledge about genetic factors that are associated with opioid sensitivity. The GWAS procedure is summarized in [Supplementary Figure S5](#app1-ijms-24-08421) for the 0–6 h plasma MEC and 0–12 h effect-site MEC, and the procedure was similar for the 0–12 h plasma MEC and 0–6 h effect-site MEC (details not shown). In the first-stage analysis of 117 subjects, the SNPs that showed statistical *p* < 0.05 were selected as candidate SNPs for the second-stage analysis among the 921,239 SNPs. For these SNPs, the second-stage analysis was conducted, and SNPs that showed *p* < 0.05 for the single analysis of this stage and combined analysis of the first and second stages were considered possible candidates. Similarly, the final-stage analysis was conducted by setting the threshold *p* values at 0.05, in which the SNPs that showed *p* < 0.05 for the single analysis of this stage and combined analysis of the first, second, and final stages were considered possible candidates. The potent SNPs were selected from these SNPs after LD-based SNP pruning to remove redundant SNPs due to strong LD (threshold *r*^2^ = 0.8) with each other, as conducted in a previous report [[63](#B63-ijms-24-08421)]. In the final stage, *q* values of the false discovery rate (FDR) were also calculated to correct for multiple testing for the SNPs that were selected after the second-stage analysis and LD-based SNP pruning, based on previous reports [[64](#B64-ijms-24-08421),[65](#B65-ijms-24-08421)]. The SNPs that showed *q* < 0.05 in the analysis among the SNPs that were selected after the final stage were considered to be genome-wide significant. Hardy–Weinberg equilibrium was additionally tested using Exact Tests for genotypic distributions of SNPs that were significantly associated with the phenotype. To calculate *q* values, Stratified False Discovery Rate (SFDR) software ([http://www.utstat.toronto.edu/sun/Software/SFDR/index.html](http://www.utstat.toronto.edu/sun/Software/SFDR/index.html); accessed on 18 January 2023) was used [[64](#B64-ijms-24-08421),[65](#B65-ijms-24-08421),[66](#B66-ijms-24-08421)]. All of the statistical analyses for genetics were performed using gPLINK v. 2.050, PLINK v. 1.07 ([http://zzz.bwh.harvard.edu/plink/index.shtml](http://zzz.bwh.harvard.edu/plink/index.shtml); accessed on 18 January 2023) [[67](#B67-ijms-24-08421)], and Haploview v. 4.2 ([https://www.broadinstitute.org/haploview/haploview](https://www.broadinstitute.org/haploview/haploview); accessed on 18 January 2023) [[68](#B68-ijms-24-08421)]. Additionally, correlation analysis, the Mann–Whitney test, and linear regression analysis were performed for statistical analyses of clinical variables using SPSS Statistics v. 25 software (IBM, Armonk, NY, USA). For the statistical analyses of clinical variables, the criterion for significance was set at *p* < 0.05.

### 4.4. Additional in Silico Analysis

#### 4.4.1. Power Analysis

Statistical power analyses were preliminarily performed using G*Power v. 3.0.5 [[69](#B69-ijms-24-08421)] as previously described [[22](#B22-ijms-24-08421),[25](#B25-ijms-24-08421)]. Power analyses for the linear regression analyses revealed that the expected power (1 minus type II error probability) was 98.6% for Cohen’s conventional “medium” effect size of 0.15 [[70](#B70-ijms-24-08421)] when the type I error probability was set at 0.05 and sample sizes were 117, corresponding to the sample size of each stage analysis in the present study. However, for the same type I error probability and sample sizes of 117, the expected power decreased to 32.9% when Cohen’s conventional “small” effect size was 0.02. Conversely, the estimated effect sizes were 0.0682 for the same type I error probability and sample sizes of 117 to achieve 80% power. Therefore, a single analysis in the present study was expected to detect true associations with the phenotype with 80% statistical power for effect sizes from large to moderately small, but not too small, although the exact effect size has been poorly understood in cases of SNPs that significantly contribute to opioid sensitivity.

#### 4.4.2. Linkage Disequilibrium Analysis

The LD analysis was performed using Haploview v. 4.2 [[68](#B68-ijms-24-08421)] for a total of 351 samples from patients who underwent LAC for the genomic position from ~174,760,000 to ~174,900,000 on chromosome 5 (GRCh37) that includes both the rs966775 SNP and *DRD1* gene and its flanking region to identify relationships between SNPs in this region. The commonly used *D*′ and *r*^2^ values were pairwise calculated using the genotype dataset for each SNP to estimate the strength of LD between SNPs. Linkage disequilibrium blocks were defined as in a previous study [[71](#B71-ijms-24-08421)]. For the visualization of LD plots with information about genomic position and related gene transcripts, the LDmatrix tool was also used ([https://ldlink.nci.nih.gov/?tab=ldmatrix](https://ldlink.nci.nih.gov/?tab=ldmatrix); accessed on 22 January 2023).

#### 4.4.3. Reference of Databases

Several databases and bioinformatic tools were referenced to more thoroughly examine the candidate SNP that may be related to human opioid analgesic sensitivity, including the NCBI database ([http://www.ncbi.nlm.nih.gov](http://www.ncbi.nlm.nih.gov); accessed on 19 January 2023), HaploReg v. 4.1 ([https://pubs.broadinstitute.org/mammals/haploreg/haploreg.php](https://pubs.broadinstitute.org/mammals/haploreg/haploreg.php); accessed on 19 January 2023) [[51](#B51-ijms-24-08421)], SNPinfo Web Server ([https://snpinfo.niehs.nih.gov](https://snpinfo.niehs.nih.gov); accessed on 19 January 2023) [[52](#B52-ijms-24-08421)], and Genotype-Tissue Expression (GTEx) portal ([https://gtexportal.org/home/](https://gtexportal.org/home/); accessed on 19 January 2023) [[53](#B53-ijms-24-08421)]. HaploReg is a tool for investigating non-coding genomic annotations at variations in haplotype blocks, such as potential regulatory SNPs at disease-associated sites [[51](#B51-ijms-24-08421)]. The SNPinfo Web Server is a set of web-based SNP selection tools (freely available at [https://snpinfo.niehs.nih.gov](https://snpinfo.niehs.nih.gov); accessed on 19 January 2023) where investigators can specify genes or linkage regions and select SNPs based on GWAS results, LD, and predicted functional characteristics of both coding and non-coding SNPs [[52](#B52-ijms-24-08421)]. The GTEx project, an ongoing effort to create a comprehensive public resource to research tissue-specific gene expression and regulation [[53](#B53-ijms-24-08421)], is the basis for the GTEx portal, which offers open access to such data as gene expression, quantitative trait loci, and histology images.

## 5. Conclusions

In conclusion, our GWASs revealed that the rs966775 SNP and *SERP2* gene were significantly associated with estimated plasma MECs over the 0–6 h and 0–12 h postoperative periods of fentanyl that was administered for the treatment of postoperative pain in LAC patients. Although the present results need to be corroborated by more research with larger sample sizes and greater statistical power, these findings indicate that the rs966775 SNP near the *DRD1* and *SERP2* genes could serve as markers that predict the efficacy of opioid analgesics for the treatment of postoperative pain.

## Acknowledgments

We thank Michael Arends for editing the manuscript. We are grateful to the volunteers for their participation in the study and anesthesiologists and surgeons for collecting the clinical data.

## Supplementary Materials

The supporting information can be downloaded at [https://www.mdpi.com/article/10.3390/ijms24098421/s1](https://www.mdpi.com/article/10.3390/ijms24098421/s1).

## Author Contributions

Conceptualization, H.S., T.T., M.H. and K.I.; methodology, D.N., J.H., K.N. and Y.E.; software, D.N.; validation, D.N., J.H., K.N. and Y.E.; formal analysis, D.N.; investigation, D.N. and S.K.; resources, T.M., M.T., H.N., S.Y. and A.K.; data curation, D.N., J.H., K.N. and Y.E.; writing— original draft preparation, D.N.; writing—review and editing, H.S., T.T., D.N., M.H. and K.I.; visualization, D.N.; supervision, A.K., M.H. and K.I.; project administration, M.H. and K.I.; funding acquisition, H.S., T.T., D.N., M.H. and K.I. All authors have read and agreed to the published version of the manuscript.

## Institutional Review Board Statement

The study was conducted in accordance with the Declaration of Helsinki and approved by the Institutional Review Board (or Ethics Committee) of Saitama Medical University International Medical Center (protocol code: 09-089, date of approval: 10 February 2010) and Tokyo Metropolitan Institute of Medical Science (protocol code: 22-11, date of approval: 31 March 2022).

## Informed Consent Statement

Informed consent was obtained from all subjects involved in the study.

## Data Availability Statement

Data that are presented in this study are available upon request from the corresponding author.

## Conflicts of Interest

Kazutaka Ikeda has received supports from Asahi Kasei Pharma Corporation and SBI Pharmaceuticals Co., Ltd., and speaker’s and consultant’s fees from MSD K.K., VistaGen Therapeutics, Inc., Atheneum Partners Otsuka Pharmaceutical Co. Ltd., Taisho Pharmaceutical Co. Ltd., Eisai, Daiichi-Sankyo, Inc., Sumitomo Pharma, Japan Tobacco, Inc., EA Pharma Co. Ltd., and Nippon Chemiphar. The authors declare no other conflicts of interest. The funders had no role in the design of the study; in the collection, analyses, or interpretation of data; in writing the manuscript; or in the decision to publish the results.

## Funding Statement

This research was funded by grants from the Japan Society for the Promotion of Science (JSPS) KAKENHI (no. 22790518, 23390377, 24790544, 26293347, JP22H04922 [AdAMS], 17H04324, 17K08970, 18K08829, 20K09259, and 21H03028), Ministry of Health, Labour, and Welfare (MHLW) of Japan (no. H26-Kakushintekigan-ippan-060), Japan Agency for Medical Research and Development (AMED; no. JP19ek0610011 and JP19dk0307071), Smoking Research Foundation (Tokyo, Japan), Japan Research Foundation for Clinical Pharmacology (JRFCP), and Asahi Kasei Pharma Open Innovation. The article processing charge was funded by the Japan Society for the Promotion of Science (JSPS) KAKENHI.

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

Data that are presented in this study are available upon request from the corresponding author.

### Supplementary Materials

### Data Availability Statement

Data that are presented in this study are available upon request from the corresponding author.

## References

1. Ikeda K., Ide S., Han W., Hayashida M., Uhl G.R., Sora I. How individual sensitivity to opiates can be predicted by gene analyses. Trends Pharmacol. Sci. 2005;26:311–317. doi: 10.1016/j.tips.2005.04.001.  [DOI](https://doi.org/10.1016/j.tips.2005.04.001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15925706/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Trends%20Pharmacol.%20Sci.&title=How%20individual%20sensitivity%20to%20opiates%20can%20be%20predicted%20by%20gene%20analyses&author=K.%20Ikeda&author=S.%20Ide&author=W.%20Han&author=M.%20Hayashida&author=G.R.%20Uhl&volume=26&publication_year=2005&pages=311-317&pmid=15925706&doi=10.1016/j.tips.2005.04.001&)

2. Nishizawa D., Ikeda K. Genome-wide association studies and human opioid sensitivity. In: Preedy V.R., editor. The Neuropathology of Drug Addictions and Substance Misuse: Volume 1. Academic Press; London, UK: 2016. pp. 909–921. (Foundations of Understanding, Tobacco, Alcohol, Cannabinoids and Opioids).  [Google Scholar](https://scholar.google.com/scholar_lookup?title=The%20Neuropathology%20of%20Drug%20Addictions%20and%20Substance%20Misuse:%20Volume%201&author=D.%20Nishizawa&author=K.%20Ikeda&publication_year=2016&)

3. Woodhouse A., Mather L.E. The minimum effective concentration of opioids: A revisitation with patient controlled analgesia fentanyl. Reg. Anesth. Pain Med. 2000;25:259–267. doi: 10.1097/00115550-200005000-00009.  [DOI](https://doi.org/10.1097/00115550-200005000-00009) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10834780/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Reg.%20Anesth.%20Pain%20Med.&title=The%20minimum%20effective%20concentration%20of%20opioids:%20A%20revisitation%20with%20patient%20controlled%20analgesia%20fentanyl&author=A.%20Woodhouse&author=L.E.%20Mather&volume=25&publication_year=2000&pages=259-267&pmid=10834780&doi=10.1097/00115550-200005000-00009&)

4. Gourlay G.K., Kowalski S.R., Plummer J.L., Cousins M.J., Armstrong P.J. Fentanyl blood concentration-analgesic response relationship in the treatment of postoperative pain. Anesth. Analg. 1988;67:329–337. doi: 10.1213/00000539-198804000-00006.  [DOI](https://doi.org/10.1213/00000539-198804000-00006) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/3354866/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anesth.%20Analg.&title=Fentanyl%20blood%20concentration-analgesic%20response%20relationship%20in%20the%20treatment%20of%20postoperative%20pain&author=G.K.%20Gourlay&author=S.R.%20Kowalski&author=J.L.%20Plummer&author=M.J.%20Cousins&author=P.J.%20Armstrong&volume=67&publication_year=1988&pages=329-337&pmid=3354866&doi=10.1213/00000539-198804000-00006&)

5. Bae J., Kwon M., Lee Y.H., Lee E.K., Choi B.M., Noh G.J. An allometric pharmacokinetic model and minimum effective analgesic concentration of fentanyl in patients undergoing major abdominal surgery. Br. J. Anaesth. 2020;125:976–985. doi: 10.1016/j.bja.2020.06.066.  [DOI](https://doi.org/10.1016/j.bja.2020.06.066) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32861508/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Anaesth.&title=An%20allometric%20pharmacokinetic%20model%20and%20minimum%20effective%20analgesic%20concentration%20of%20fentanyl%20in%20patients%20undergoing%20major%20abdominal%20surgery&author=J.%20Bae&author=M.%20Kwon&author=Y.H.%20Lee&author=E.K.%20Lee&author=B.M.%20Choi&volume=125&publication_year=2020&pages=976-985&pmid=32861508&doi=10.1016/j.bja.2020.06.066&)

6. Lehmann K.A., Heinrich C., van Heiss R. Balanced anesthesia and patient-controlled postoperative analgesia with fentanyl: Minimum effective concentrations, accumulation and acute tolerance. Acta Anaesthesiol. Belg. 1988;39:11–23.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/3369270/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Acta%20Anaesthesiol.%20Belg.&title=Balanced%20anesthesia%20and%20patient-controlled%20postoperative%20analgesia%20with%20fentanyl:%20Minimum%20effective%20concentrations,%20accumulation%20and%20acute%20tolerance&author=K.A.%20Lehmann&author=C.%20Heinrich&author=R.%20van%20Heiss&volume=39&publication_year=1988&pages=11-23&pmid=3369270&)

7. Yoshida K., Nishizawa D., Ide S., Ichinohe T., Fukuda K.I., Ikeda K. A pharmacogenetics approach to pain management. Neuropsychopharmacol. Rep. 2018;38:2–8. doi: 10.1002/npr2.12003.  [DOI](https://doi.org/10.1002/npr2.12003) | [PMC free article](/articles/PMC7292326/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30106264/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neuropsychopharmacol.%20Rep.&title=A%20pharmacogenetics%20approach%20to%20pain%20management&author=K.%20Yoshida&author=D.%20Nishizawa&author=S.%20Ide&author=T.%20Ichinohe&author=K.I.%20Fukuda&volume=38&publication_year=2018&pages=2-8&pmid=30106264&doi=10.1002/npr2.12003&)

8. Angst M.S., Phillips N.G., Drover D.R., Tingle M., Galinkin J.L., Christians U., Swan G.E., Lazzeroni L.C., Clark J.D. Opioid pharmacogenomics using a twin study paradigm: Methods and procedures for determining familial aggregation and heritability. Twin Res. Hum. Genet. 2010;13:412–425. doi: 10.1375/twin.13.5.412.  [DOI](https://doi.org/10.1375/twin.13.5.412) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20874462/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Twin%20Res.%20Hum.%20Genet.&title=Opioid%20pharmacogenomics%20using%20a%20twin%20study%20paradigm:%20Methods%20and%20procedures%20for%20determining%20familial%20aggregation%20and%20heritability&author=M.S.%20Angst&author=N.G.%20Phillips&author=D.R.%20Drover&author=M.%20Tingle&author=J.L.%20Galinkin&volume=13&publication_year=2010&pages=412-425&pmid=20874462&doi=10.1375/twin.13.5.412&)

9. Angst M.S., Phillips N.G., Drover D.R., Tingle M., Ray A., Swan G.E., Lazzeroni L.C., Clark J.D. Pain sensitivity and opioid analgesia: A pharmacogenomic twin study. Pain. 2012;153:1397–1409. doi: 10.1016/j.pain.2012.02.022.  [DOI](https://doi.org/10.1016/j.pain.2012.02.022) | [PMC free article](/articles/PMC3377769/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22444188/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pain&title=Pain%20sensitivity%20and%20opioid%20analgesia:%20A%20pharmacogenomic%20twin%20study&author=M.S.%20Angst&author=N.G.%20Phillips&author=D.R.%20Drover&author=M.%20Tingle&author=A.%20Ray&volume=153&publication_year=2012&pages=1397-1409&pmid=22444188&doi=10.1016/j.pain.2012.02.022&)

10. Branford R., Droney J., Ross J.R. Opioid genetics: The key to personalized pain control? Clin. Genet. 2012;82:301–310. doi: 10.1111/j.1399-0004.2012.01923.x.  [DOI](https://doi.org/10.1111/j.1399-0004.2012.01923.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22780883/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Genet.&title=Opioid%20genetics:%20The%20key%20to%20personalized%20pain%20control?&author=R.%20Branford&author=J.%20Droney&author=J.R.%20Ross&volume=82&publication_year=2012&pages=301-310&pmid=22780883&doi=10.1111/j.1399-0004.2012.01923.x&)

11. Droney J., Riley J., Ross J. Opioid genetics in the context of opioid switching. Curr. Opin. Support Palliat. Care. 2012;6:10–16. doi: 10.1097/SPC.0b013e32834f6a07.  [DOI](https://doi.org/10.1097/SPC.0b013e32834f6a07) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22228031/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr.%20Opin.%20Support%20Palliat.%20Care&title=Opioid%20genetics%20in%20the%20context%20of%20opioid%20switching&author=J.%20Droney&author=J.%20Riley&author=J.%20Ross&volume=6&publication_year=2012&pages=10-16&pmid=22228031&doi=10.1097/SPC.0b013e32834f6a07&)

12. Vuilleumier P.H., Stamer U.M., Landau R. Pharmacogenomic considerations in opioid analgesia. Pharmgenom. Pers. Med. 2012;5:73–87. doi: 10.2147/PGPM.S23422.  [DOI](https://doi.org/10.2147/PGPM.S23422) | [PMC free article](/articles/PMC3513230/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23226064/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmgenom.%20Pers.%20Med.&title=Pharmacogenomic%20considerations%20in%20opioid%20analgesia&author=P.H.%20Vuilleumier&author=U.M.%20Stamer&author=R.%20Landau&volume=5&publication_year=2012&pages=73-87&pmid=23226064&doi=10.2147/PGPM.S23422&)

13. Kurita G.P., Sjogren P., Ekholm O., Kaasa S., Loge J.H., Poviloniene I., Klepstad P. Prevalence and predictors of cognitive dysfunction in opioid-treated patients with cancer: A multinational study. J. Clin. Oncol. 2011;29:1297–1303. doi: 10.1200/JCO.2010.32.6884.  [DOI](https://doi.org/10.1200/JCO.2010.32.6884) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21357785/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Oncol.&title=Prevalence%20and%20predictors%20of%20cognitive%20dysfunction%20in%20opioid-treated%20patients%20with%20cancer:%20A%20multinational%20study&author=G.P.%20Kurita&author=P.%20Sjogren&author=O.%20Ekholm&author=S.%20Kaasa&author=J.H.%20Loge&volume=29&publication_year=2011&pages=1297-1303&pmid=21357785&doi=10.1200/JCO.2010.32.6884&)

14. Galvan A., Fladvad T., Skorpen F., Gao X., Klepstad P., Kaasa S., Dragani T.A. Genetic clustering of European cancer patients indicates that opioid-mediated pain relief is independent of ancestry. Pharm. J. 2012;12:412–416. doi: 10.1038/tpj.2011.27.  [DOI](https://doi.org/10.1038/tpj.2011.27) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21747411/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharm.%20J.&title=Genetic%20clustering%20of%20European%20cancer%20patients%20indicates%20that%20opioid-mediated%20pain%20relief%20is%20independent%20of%20ancestry&author=A.%20Galvan&author=T.%20Fladvad&author=F.%20Skorpen&author=X.%20Gao&author=P.%20Klepstad&volume=12&publication_year=2012&pages=412-416&pmid=21747411&doi=10.1038/tpj.2011.27&)

15. Hayashida M., Nagashima M., Satoh Y., Katoh R., Tagami M., Ide S., Kasai S., Nishizawa D., Ogai Y., Hasegawa J., et al. Analgesic requirements after major abdominal surgery are associated with OPRM1 gene polymorphism genotype and haplotype. Pharmacogenomics. 2008;9:1605–1616. doi: 10.2217/14622416.9.11.1605.  [DOI](https://doi.org/10.2217/14622416.9.11.1605) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19018716/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Analgesic%20requirements%20after%20major%20abdominal%20surgery%20are%20associated%20with%20OPRM1%20gene%20polymorphism%20genotype%20and%20haplotype&author=M.%20Hayashida&author=M.%20Nagashima&author=Y.%20Satoh&author=R.%20Katoh&author=M.%20Tagami&volume=9&publication_year=2008&pages=1605-1616&pmid=19018716&doi=10.2217/14622416.9.11.1605&)

16. Fukuda K., Hayashida M., Ide S., Saita N., Kokita Y., Kasai S., Nishizawa D., Ogai Y., Hasegawa J., Nagashima M., et al. Association between OPRM1 gene polymorphisms and fentanyl sensitivity in patients undergoing painful cosmetic surgery. Pain. 2009;147:194–201. doi: 10.1016/j.pain.2009.09.004.  [DOI](https://doi.org/10.1016/j.pain.2009.09.004) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19783098/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pain&title=Association%20between%20OPRM1%20gene%20polymorphisms%20and%20fentanyl%20sensitivity%20in%20patients%20undergoing%20painful%20cosmetic%20surgery&author=K.%20Fukuda&author=M.%20Hayashida&author=S.%20Ide&author=N.%20Saita&author=Y.%20Kokita&volume=147&publication_year=2009&pages=194-201&pmid=19783098&doi=10.1016/j.pain.2009.09.004&)

17. Nishizawa D., Nagashima M., Katoh R., Satoh Y., Tagami M., Kasai S., Ogai Y., Han W., Hasegawa J., Shimoyama N., et al. Association between KCNJ6 (GIRK2) gene polymorphisms and postoperative analgesic requirements after major abdominal surgery. PLoS ONE. 2009;4:e7060. doi: 10.1371/journal.pone.0007060.  [DOI](https://doi.org/10.1371/journal.pone.0007060) | [PMC free article](/articles/PMC2738941/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19756153/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&title=Association%20between%20KCNJ6%20(GIRK2)%20gene%20polymorphisms%20and%20postoperative%20analgesic%20requirements%20after%20major%20abdominal%20surgery&author=D.%20Nishizawa&author=M.%20Nagashima&author=R.%20Katoh&author=Y.%20Satoh&author=M.%20Tagami&volume=4&publication_year=2009&pages=e7060&pmid=19756153&doi=10.1371/journal.pone.0007060&)

18. Nishizawa D., Fukuda K., Kasai S., Hasegawa J., Aoki Y., Nishi A., Saita N., Koukita Y., Nagashima M., Katoh R., et al. Genome-wide association study identifies a potent locus associated with human opioid sensitivity. Mol. Psychiatry. 2014;19:55–62. doi: 10.1038/mp.2012.164.  [DOI](https://doi.org/10.1038/mp.2012.164) | [PMC free article](/articles/PMC3873034/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23183491/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Psychiatry&title=Genome-wide%20association%20study%20identifies%20a%20potent%20locus%20associated%20with%20human%20opioid%20sensitivity&author=D.%20Nishizawa&author=K.%20Fukuda&author=S.%20Kasai&author=J.%20Hasegawa&author=Y.%20Aoki&volume=19&publication_year=2014&pages=55-62&pmid=23183491&doi=10.1038/mp.2012.164&)

19. Nishizawa D., Fukuda K.I., Kasai S., Ogai Y., Hasegawa J., Sato N., Yamada H., Tanioka F., Sugimura H., Hayashida M., et al. Association between KCNJ6 (GIRK2) gene polymorphism rs2835859 and post-operative analgesia, pain sensitivity, and nicotine dependence. J. Pharmacol. Sci. 2014;126:253–263. doi: 10.1254/jphs.14189FP.  [DOI](https://doi.org/10.1254/jphs.14189FP) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25346042/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Pharmacol.%20Sci.&title=Association%20between%20KCNJ6%20(GIRK2)%20gene%20polymorphism%20rs2835859%20and%20post-operative%20analgesia,%20pain%20sensitivity,%20and%20nicotine%20dependence&author=D.%20Nishizawa&author=K.I.%20Fukuda&author=S.%20Kasai&author=Y.%20Ogai&author=J.%20Hasegawa&volume=126&publication_year=2014&pages=253-263&pmid=25346042&doi=10.1254/jphs.14189FP&)

20. Ide S., Nishizawa D., Fukuda K., Kasai S., Hasegawa J., Hayashida M., Minami M., Ikeda K. Haplotypes of P2RX7 gene polymorphisms are associated with both cold pain sensitivity and analgesic effect of fentanyl. Mol. Pain. 2014;10:75. doi: 10.1186/1744-8069-10-75.  [DOI](https://doi.org/10.1186/1744-8069-10-75) | [PMC free article](/articles/PMC4265416/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25472448/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Pain&title=Haplotypes%20of%20P2RX7%20gene%20polymorphisms%20are%20associated%20with%20both%20cold%20pain%20sensitivity%20and%20analgesic%20effect%20of%20fentanyl&author=S.%20Ide&author=D.%20Nishizawa&author=K.%20Fukuda&author=S.%20Kasai&author=J.%20Hasegawa&volume=10&publication_year=2014&pages=75&pmid=25472448&doi=10.1186/1744-8069-10-75&)

21. Aoki Y., Nishizawa D., Hasegawa J., Kasai S., Yoshida K., Koukita Y., Ichinohe T., Nagashima M., Katoh R., Satoh Y., et al. Association between the rs1465040 single-nucleotide polymorphism close to the transient receptor potential subfamily C member 3 (TRPC3) gene and postoperative analgesic requirements. J. Pharmacol. Sci. 2015;127:391–393. doi: 10.1016/j.jphs.2015.02.005.  [DOI](https://doi.org/10.1016/j.jphs.2015.02.005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25837939/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Pharmacol.%20Sci.&title=Association%20between%20the%20rs1465040%20single-nucleotide%20polymorphism%20close%20to%20the%20transient%20receptor%20potential%20subfamily%20C%20member%203%20(TRPC3)%20gene%20and%20postoperative%20analgesic%20requirements&author=Y.%20Aoki&author=D.%20Nishizawa&author=J.%20Hasegawa&author=S.%20Kasai&author=K.%20Yoshida&volume=127&publication_year=2015&pages=391-393&pmid=25837939&doi=10.1016/j.jphs.2015.02.005&)

22. Mieda T., Nishizawa D., Nakagawa H., Tsujita M., Imanishi H., Terao K., Yoshikawa H., Itoh K., Amano K., Tashiro J., et al. Genome-wide association study identifies candidate loci associated with postoperative fentanyl requirements after laparoscopic-assisted colectomy. Pharmacogenomics. 2016;17:133–145. doi: 10.2217/pgs.15.151.  [DOI](https://doi.org/10.2217/pgs.15.151) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26566055/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Genome-wide%20association%20study%20identifies%20candidate%20loci%20associated%20with%20postoperative%20fentanyl%20requirements%20after%20laparoscopic-assisted%20colectomy&author=T.%20Mieda&author=D.%20Nishizawa&author=H.%20Nakagawa&author=M.%20Tsujita&author=H.%20Imanishi&volume=17&publication_year=2016&pages=133-145&pmid=26566055&doi=10.2217/pgs.15.151&)

23. Amano K., Nishizawa D., Mieda T., Tsujita M., Kitamura A., Hasegawa J., Inada E., Hayashida M., Ikeda K. Opposite associations between the rs3845446 single-nucleotide polymorphism of the CACNA1E gene and postoperative pain-related phenotypes in gastrointestinal surgery versus previously reported orthognathic surgery. J. Pain. 2016;17:1126–1134. doi: 10.1016/j.jpain.2016.07.001.  [DOI](https://doi.org/10.1016/j.jpain.2016.07.001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27480382/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Pain&title=Opposite%20associations%20between%20the%20rs3845446%20single-nucleotide%20polymorphism%20of%20the%20CACNA1E%20gene%20and%20postoperative%20pain-related%20phenotypes%20in%20gastrointestinal%20surgery%20versus%20previously%20reported%20orthognathic%20surgery&author=K.%20Amano&author=D.%20Nishizawa&author=T.%20Mieda&author=M.%20Tsujita&author=A.%20Kitamura&volume=17&publication_year=2016&pages=1126-1134&pmid=27480382&doi=10.1016/j.jpain.2016.07.001&)

24. Muraoka W., Nishizawa D., Fukuda K., Kasai S., Hasegawa J., Wajima K., Nakagawa T., Ikeda K. Association between UGT2B7 gene polymorphisms and fentanyl sensitivity in patients undergoing painful orthognathic surgery. Mol. Pain. 2016;12:1744806916683182. doi: 10.1177/1744806916683182.  [DOI](https://doi.org/10.1177/1744806916683182) | [PMC free article](/articles/PMC5521342/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28256933/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Pain&title=Association%20between%20UGT2B7%20gene%20polymorphisms%20and%20fentanyl%20sensitivity%20in%20patients%20undergoing%20painful%20orthognathic%20surgery&author=W.%20Muraoka&author=D.%20Nishizawa&author=K.%20Fukuda&author=S.%20Kasai&author=J.%20Hasegawa&volume=12&publication_year=2016&pages=1744806916683182&pmid=28256933&doi=10.1177/1744806916683182&)

25. Nishizawa D., Mieda T., Tsujita M., Nakagawa H., Yamaguchi S., Kasai S., Hasegawa J., Fukuda K.I., Kitamura A., Hayashida M., et al. Genome-wide scan identifies candidate loci related to remifentanil requirements during laparoscopic-assisted colectomy. Pharmacogenomics. 2018;19:113–127. doi: 10.2217/pgs-2017-0109.  [DOI](https://doi.org/10.2217/pgs-2017-0109) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29207912/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Genome-wide%20scan%20identifies%20candidate%20loci%20related%20to%20remifentanil%20requirements%20during%20laparoscopic-assisted%20colectomy&author=D.%20Nishizawa&author=T.%20Mieda&author=M.%20Tsujita&author=H.%20Nakagawa&author=S.%20Yamaguchi&volume=19&publication_year=2018&pages=113-127&pmid=29207912&doi=10.2217/pgs-2017-0109&)

26. Takahashi K., Nishizawa D., Kasai S., Koukita Y., Fukuda K.I., Ichinohe T., Ikeda K. Genome-wide association study identifies polymorphisms associated with the analgesic effect of fentanyl in the preoperative cold pressor-induced pain test. J. Pharmacol. Sci. 2018;136:107–113. doi: 10.1016/j.jphs.2018.02.002.  [DOI](https://doi.org/10.1016/j.jphs.2018.02.002) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29502940/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Pharmacol.%20Sci.&title=Genome-wide%20association%20study%20identifies%20polymorphisms%20associated%20with%20the%20analgesic%20effect%20of%20fentanyl%20in%20the%20preoperative%20cold%20pressor-induced%20pain%20test&author=K.%20Takahashi&author=D.%20Nishizawa&author=S.%20Kasai&author=Y.%20Koukita&author=K.I.%20Fukuda&volume=136&publication_year=2018&pages=107-113&pmid=29502940&doi=10.1016/j.jphs.2018.02.002&)

27. Ohka S., Nishizawa D., Hasegawa J., Takahashi K., Nakayama K., Ebata Y., Fukuda K.I., Ikeda K. Association between rs2275913 single-nucleotide polymorphism of the interleukin-17A gene and perioperative analgesic use in cosmetic orthognathic surgery. Neuropsychopharmacol. Rep. 2018;38:67–74. doi: 10.1002/npr2.12010.  [DOI](https://doi.org/10.1002/npr2.12010) | [PMC free article](/articles/PMC7292295/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30106258/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neuropsychopharmacol.%20Rep.&title=Association%20between%20rs2275913%20single-nucleotide%20polymorphism%20of%20the%20interleukin-17A%20gene%20and%20perioperative%20analgesic%20use%20in%20cosmetic%20orthognathic%20surgery&author=S.%20Ohka&author=D.%20Nishizawa&author=J.%20Hasegawa&author=K.%20Takahashi&author=K.%20Nakayama&volume=38&publication_year=2018&pages=67-74&pmid=30106258&doi=10.1002/npr2.12010&)

28. Aoki Y., Nishizawa D., Yoshida K., Hasegawa J., Kasai S., Takahashi K., Koukita Y., Ichinohe T., Hayashida M., Fukuda K.I., et al. Association between the rs7583431 single nucleotide polymorphism close to the activating transcription factor 2 gene and the analgesic effect of fentanyl in the cold pain test. Neuropsychopharmacol. Rep. 2018;38:86–91. doi: 10.1002/npr2.12012.  [DOI](https://doi.org/10.1002/npr2.12012) | [PMC free article](/articles/PMC7292331/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30106255/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neuropsychopharmacol.%20Rep.&title=Association%20between%20the%20rs7583431%20single%20nucleotide%20polymorphism%20close%20to%20the%20activating%20transcription%20factor%202%20gene%20and%20the%20analgesic%20effect%20of%20fentanyl%20in%20the%20cold%20pain%20test&author=Y.%20Aoki&author=D.%20Nishizawa&author=K.%20Yoshida&author=J.%20Hasegawa&author=S.%20Kasai&volume=38&publication_year=2018&pages=86-91&pmid=30106255&doi=10.1002/npr2.12012&)

29. Inoue R., Nishizawa D., Hasegawa J., Nakayama K., Fukuda K.I., Ichinohe T., Mieda T., Tsujita M., Nakagawa H., Kitamura A., et al. Effects of rs958804 and rs7858836 single-nucleotide polymorphisms of the ASTN2 gene on pain-related phenotypes in patients who underwent laparoscopic colectomy and mandibular sagittal split ramus osteotomy. Neuropsychopharmacol. Rep. 2021;41:82–90. doi: 10.1002/npr2.12159.  [DOI](https://doi.org/10.1002/npr2.12159) | [PMC free article](/articles/PMC8182957/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33476460/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neuropsychopharmacol.%20Rep.&title=Effects%20of%20rs958804%20and%20rs7858836%20single-nucleotide%20polymorphisms%20of%20the%20ASTN2%20gene%20on%20pain-related%20phenotypes%20in%20patients%20who%20underwent%20laparoscopic%20colectomy%20and%20mandibular%20sagittal%20split%20ramus%20osteotomy&author=R.%20Inoue&author=D.%20Nishizawa&author=J.%20Hasegawa&author=K.%20Nakayama&author=K.I.%20Fukuda&volume=41&publication_year=2021&pages=82-90&pmid=33476460&doi=10.1002/npr2.12159&)

30. Nishizawa D., Iseki M., Arita H., Hanaoka K., Yajima C., Kato J., Ogawa S., Hiranuma A., Kasai S., Hasegawa J., et al. Genome-wide association study identifies candidate loci associated with chronic pain and postherpetic neuralgia. Mol. Pain. 2021;17:1744806921999924. doi: 10.1177/1744806921999924.  [DOI](https://doi.org/10.1177/1744806921999924) | [PMC free article](/articles/PMC8822450/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33685280/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Pain&title=Genome-wide%20association%20study%20identifies%20candidate%20loci%20associated%20with%20chronic%20pain%20and%20postherpetic%20neuralgia&author=D.%20Nishizawa&author=M.%20Iseki&author=H.%20Arita&author=K.%20Hanaoka&author=C.%20Yajima&volume=17&publication_year=2021&pages=1744806921999924&pmid=33685280&doi=10.1177/1744806921999924&)

31. Kasai S., Nishizawa D., Hasegawa J., Fukuda K.I., Ichinohe T., Nagashima M., Hayashida M., Ikeda K. Short tandem repeat variation in the CNR1 gene associated with analgesic requirements of opioids in postoperative pain management. Front. Genet. 2022;13:815089. doi: 10.3389/fgene.2022.815089.  [DOI](https://doi.org/10.3389/fgene.2022.815089) | [PMC free article](/articles/PMC8963810/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35360861/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front.%20Genet.&title=Short%20tandem%20repeat%20variation%20in%20the%20CNR1%20gene%20associated%20with%20analgesic%20requirements%20of%20opioids%20in%20postoperative%20pain%20management&author=S.%20Kasai&author=D.%20Nishizawa&author=J.%20Hasegawa&author=K.I.%20Fukuda&author=T.%20Ichinohe&volume=13&publication_year=2022&pages=815089&pmid=35360861&doi=10.3389/fgene.2022.815089&)

32. Nishizawa D., Nagashima M., Kasai S., Hasegawa J., Nakayama K., Ebata Y., Fukuda K.I., Ichinohe T., Hayashida M., Ikeda K. Associations between the C3orf20 rs12496846 polymorphism and both postoperative analgesia after orthognathic and abdominal surgeries and C3orf20 gene expression in the brain. Pharmaceutics. 2022;14:727. doi: 10.3390/pharmaceutics14040727.  [DOI](https://doi.org/10.3390/pharmaceutics14040727) | [PMC free article](/articles/PMC9028963/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35456561/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmaceutics&title=Associations%20between%20the%20C3orf20%20rs12496846%20polymorphism%20and%20both%20postoperative%20analgesia%20after%20orthognathic%20and%20abdominal%20surgeries%20and%20C3orf20%20gene%20expression%20in%20the%20brain&author=D.%20Nishizawa&author=M.%20Nagashima&author=S.%20Kasai&author=J.%20Hasegawa&author=K.%20Nakayama&volume=14&publication_year=2022&pages=727&pmid=35456561&doi=10.3390/pharmaceutics14040727&)

33. Nishizawa D., Terui T., Ishitani K., Kasai S., Hasegawa J., Nakayama K., Ebata Y., Ikeda K. Genome-wide association study identifies candidate loci associated with opioid analgesic requirements in the treatment of cancer pain. Cancers. 2022;14:4692. doi: 10.3390/cancers14194692.  [DOI](https://doi.org/10.3390/cancers14194692) | [PMC free article](/articles/PMC9564079/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36230616/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancers&title=Genome-wide%20association%20study%20identifies%20candidate%20loci%20associated%20with%20opioid%20analgesic%20requirements%20in%20the%20treatment%20of%20cancer%20pain&author=D.%20Nishizawa&author=T.%20Terui&author=K.%20Ishitani&author=S.%20Kasai&author=J.%20Hasegawa&volume=14&publication_year=2022&pages=4692&pmid=36230616&doi=10.3390/cancers14194692&)

34. Shafer S.L., Gregg K.M. Algorithms to rapidly achieve and maintain stable drug concentrations at the site of drug effect with a computer-controlled infusion pump. J. Pharmacokinet. Biopharm. 1992;20:147–169. doi: 10.1007/BF01070999.  [DOI](https://doi.org/10.1007/BF01070999) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1629794/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Pharmacokinet.%20Biopharm.&title=Algorithms%20to%20rapidly%20achieve%20and%20maintain%20stable%20drug%20concentrations%20at%20the%20site%20of%20drug%20effect%20with%20a%20computer-controlled%20infusion%20pump&author=S.L.%20Shafer&author=K.M.%20Gregg&volume=20&publication_year=1992&pages=147-169&pmid=1629794&doi=10.1007/BF01070999&)

35. Slepchenko G., Simon N., Goubaux B., Levron J.C., Le Moing J.P., Raucoules-Aimé M. Performance of target-controlled sufentanil infusion in obese patients. Anesthesiology. 2003;98:65–73. doi: 10.1097/00000542-200301000-00014.  [DOI](https://doi.org/10.1097/00000542-200301000-00014) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12502981/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anesthesiology&title=Performance%20of%20target-controlled%20sufentanil%20infusion%20in%20obese%20patients&author=G.%20Slepchenko&author=N.%20Simon&author=B.%20Goubaux&author=J.C.%20Levron&author=J.P.%20Le%20Moing&volume=98&publication_year=2003&pages=65-73&pmid=12502981&doi=10.1097/00000542-200301000-00014&)

36. Iwakiri H., Nagata O., Matsukawa T., Ozaki M., Sessler D.I. Effect-site concentration of propofol for recovery of consciousness is virtually independent of fentanyl effect-site concentration. Anesth. Analg. 2003;96:1651–1655. doi: 10.1213/01.ANE.0000062772.28479.2B.  [DOI](https://doi.org/10.1213/01.ANE.0000062772.28479.2B) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12760990/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anesth.%20Analg.&title=Effect-site%20concentration%20of%20propofol%20for%20recovery%20of%20consciousness%20is%20virtually%20independent%20of%20fentanyl%20effect-site%20concentration&author=H.%20Iwakiri&author=O.%20Nagata&author=T.%20Matsukawa&author=M.%20Ozaki&author=D.I.%20Sessler&volume=96&publication_year=2003&pages=1651-1655&pmid=12760990&doi=10.1213/01.ANE.0000062772.28479.2B&)

37. Johnson K.B., Syroid N.D., Gupta D.K., Manyam S.C., Egan T.D., Huntington J., White J.L., Tyler D., Westenskow D.R. An evaluation of remifentanil propofol response surfaces for loss of responsiveness, loss of response to surrogates of painful stimuli and laryngoscopy in patients undergoing elective surgery. Anesth. Analg. 2008;106:471–479. doi: 10.1213/ane.0b013e3181606c62.  [DOI](https://doi.org/10.1213/ane.0b013e3181606c62) | [PMC free article](/articles/PMC3050649/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18227302/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anesth.%20Analg.&title=An%20evaluation%20of%20remifentanil%20propofol%20response%20surfaces%20for%20loss%20of%20responsiveness,%20loss%20of%20response%20to%20surrogates%20of%20painful%20stimuli%20and%20laryngoscopy%20in%20patients%20undergoing%20elective%20surgery&author=K.B.%20Johnson&author=N.D.%20Syroid&author=D.K.%20Gupta&author=S.C.%20Manyam&author=T.D.%20Egan&volume=106&publication_year=2008&pages=471-479&pmid=18227302&doi=10.1213/ane.0b013e3181606c62&)

38. Kodaka M., Tsukakoshi M., Miyao H., Tsuzaki K., Ichikawa J., Komori M. The fentanyl concentration required for immobility under propofol anesthesia is reduced by pre-treatment with flurbiprofen axetil. Can. J. Anaesth. 2013;60:1204–1211. doi: 10.1007/s12630-013-0040-6.  [DOI](https://doi.org/10.1007/s12630-013-0040-6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24078554/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Can.%20J.%20Anaesth.&title=The%20fentanyl%20concentration%20required%20for%20immobility%20under%20propofol%20anesthesia%20is%20reduced%20by%20pre-treatment%20with%20flurbiprofen%20axetil&author=M.%20Kodaka&author=M.%20Tsukakoshi&author=H.%20Miyao&author=K.%20Tsuzaki&author=J.%20Ichikawa&volume=60&publication_year=2013&pages=1204-1211&pmid=24078554&doi=10.1007/s12630-013-0040-6&)

39. Chiou Y.W., Ting C.K., Wang H.Y., Tsou M.Y., Chang W.K. Enhanced recovery after surgery: Prediction for early extubation in video-assisted thoracic surgery using a response surface model in anesthesia. J. Formos. Med. Assoc. 2019;118:1450–1457. doi: 10.1016/j.jfma.2019.07.024.  [DOI](https://doi.org/10.1016/j.jfma.2019.07.024) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31471221/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Formos.%20Med.%20Assoc.&title=Enhanced%20recovery%20after%20surgery:%20Prediction%20for%20early%20extubation%20in%20video-assisted%20thoracic%20surgery%20using%20a%20response%20surface%20model%20in%20anesthesia&author=Y.W.%20Chiou&author=C.K.%20Ting&author=H.Y.%20Wang&author=M.Y.%20Tsou&author=W.K.%20Chang&volume=118&publication_year=2019&pages=1450-1457&pmid=31471221&doi=10.1016/j.jfma.2019.07.024&)

40. Masui K., Kazama T. Pharmacokinetic simulation and pharmacokinetic model. J. Clin. Anesth. 2010;34:445–455.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Anesth.&title=Pharmacokinetic%20simulation%20and%20pharmacokinetic%20model&author=K.%20Masui&author=T.%20Kazama&volume=34&publication_year=2010&pages=445-455&)

41. Hayashi K., Tsuchida K., Masui K., Sawa T. The management of intravenous patient-controlled analgesia with simulations based on the pharmacokinetics and pharmacodynamics of fentanyl. J. Jpn. Soc. Clin. Anesth. 2012;32:814–820. doi: 10.2199/jjsca.32.814.  [DOI](https://doi.org/10.2199/jjsca.32.814) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Jpn.%20Soc.%20Clin.%20Anesth.&title=The%20management%20of%20intravenous%20patient-controlled%20analgesia%20with%20simulations%20based%20on%20the%20pharmacokinetics%20and%20pharmacodynamics%20of%20fentanyl&author=K.%20Hayashi&author=K.%20Tsuchida&author=K.%20Masui&author=T.%20Sawa&volume=32&publication_year=2012&pages=814-820&doi=10.2199/jjsca.32.814&)

42. Shiota T., Kawanishi H., Inoue S., Egawa J., Kawaguchi M. Risk factors for bradypnea in a historical cohort of surgical patients receiving fentanyl-based intravenous analgesia. JA Clin. Rep. 2018;4:46. doi: 10.1186/s40981-018-0186-x.  [DOI](https://doi.org/10.1186/s40981-018-0186-x) | [PMC free article](/articles/PMC6967301/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32025969/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JA%20Clin.%20Rep.&title=Risk%20factors%20for%20bradypnea%20in%20a%20historical%20cohort%20of%20surgical%20patients%20receiving%20fentanyl-based%20intravenous%20analgesia&author=T.%20Shiota&author=H.%20Kawanishi&author=S.%20Inoue&author=J.%20Egawa&author=M.%20Kawaguchi&volume=4&publication_year=2018&pages=46&pmid=32025969&doi=10.1186/s40981-018-0186-x&)

43. de Leeuw C.A., Mooij J.M., Heskes T., Posthuma D. MAGMA: Generalized gene-set analysis of GWAS data. PLoS Comput. Biol. 2015;11:e1004219. doi: 10.1371/journal.pcbi.1004219.  [DOI](https://doi.org/10.1371/journal.pcbi.1004219) | [PMC free article](/articles/PMC4401657/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25885710/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Comput.%20Biol.&title=MAGMA:%20Generalized%20gene-set%20analysis%20of%20GWAS%20data&author=C.A.%20de%20Leeuw&author=J.M.%20Mooij&author=T.%20Heskes&author=D.%20Posthuma&volume=11&publication_year=2015&pages=e1004219&pmid=25885710&doi=10.1371/journal.pcbi.1004219&)

44. Watanabe K., Taskesen E., van Bochoven A., Posthuma D. Functional mapping and annotation of genetic associations with FUMA. Nat. Commun. 2017;8:1826. doi: 10.1038/s41467-017-01261-5.  [DOI](https://doi.org/10.1038/s41467-017-01261-5) | [PMC free article](/articles/PMC5705698/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29184056/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&title=Functional%20mapping%20and%20annotation%20of%20genetic%20associations%20with%20FUMA&author=K.%20Watanabe&author=E.%20Taskesen&author=A.%20van%20Bochoven&author=D.%20Posthuma&volume=8&publication_year=2017&pages=1826&pmid=29184056&doi=10.1038/s41467-017-01261-5&)

45. Shibutani K., Inchiosa M.A., Jr., Sawada K., Bairamian M. Accuracy of pharmacokinetic models for predicting plasma fentanyl concentrations in lean and obese surgical patients: Derivation of dosing weight (“pharmacokinetic mass”) Anesthesiology. 2004;101:603–613. doi: 10.1097/00000542-200409000-00008.  [DOI](https://doi.org/10.1097/00000542-200409000-00008) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15329584/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anesthesiology&title=Accuracy%20of%20pharmacokinetic%20models%20for%20predicting%20plasma%20fentanyl%20concentrations%20in%20lean%20and%20obese%20surgical%20patients:%20Derivation%20of%20dosing%20weight%20(%E2%80%9Cpharmacokinetic%20mass%E2%80%9D)&author=K.%20Shibutani&author=M.A.%20Inchiosa&author=K.%20Sawada&author=M.%20Bairamian&volume=101&publication_year=2004&pages=603-613&pmid=15329584&doi=10.1097/00000542-200409000-00008&)

46. Fujita Y., Yoshizawa S., Hoshika M., Inoue K., Matsushita S., Oka H., Sobue K. Effect of different surgical procedures on the accuracy of prediction of the plasma concentration of fentanyl: Comparison between mastectomy and laparoscopic prostatectomy. JA Clin. Rep. 2017;3:30. doi: 10.1186/s40981-017-0097-2.  [DOI](https://doi.org/10.1186/s40981-017-0097-2) | [PMC free article](/articles/PMC5804613/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29457074/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JA%20Clin.%20Rep.&title=Effect%20of%20different%20surgical%20procedures%20on%20the%20accuracy%20of%20prediction%20of%20the%20plasma%20concentration%20of%20fentanyl:%20Comparison%20between%20mastectomy%20and%20laparoscopic%20prostatectomy&author=Y.%20Fujita&author=S.%20Yoshizawa&author=M.%20Hoshika&author=K.%20Inoue&author=S.%20Matsushita&volume=3&publication_year=2017&pages=30&pmid=29457074&doi=10.1186/s40981-017-0097-2&)

47. Flores J.A., El Banoua F., Galan-Rodriguez B., Fernandez-Espejo E. Opiate anti-nociception is attenuated following lesion of large dopamine neurons of the periaqueductal grey: Critical role for D1 (not D2) dopamine receptors. Pain. 2004;110:205–214. doi: 10.1016/j.pain.2004.03.036.  [DOI](https://doi.org/10.1016/j.pain.2004.03.036) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15275769/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pain&title=Opiate%20anti-nociception%20is%20attenuated%20following%20lesion%20of%20large%20dopamine%20neurons%20of%20the%20periaqueductal%20grey:%20Critical%20role%20for%20D1%20(not%20D2)%20dopamine%20receptors&author=J.A.%20Flores&author=F.%20El%20Banoua&author=B.%20Galan-Rodriguez&author=E.%20Fernandez-Espejo&volume=110&publication_year=2004&pages=205-214&pmid=15275769&doi=10.1016/j.pain.2004.03.036&)

48. Kiritsy-Roy J.A., Standish S.M., Terry L.C. Dopamine D-1 and D-2 receptor antagonists potentiate analgesic and motor effects of morphine. Pharmacol. Biochem. Behav. 1989;32:717–721. doi: 10.1016/0091-3057(89)90023-3.  [DOI](https://doi.org/10.1016/0091-3057(89)90023-3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2662224/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacol.%20Biochem.%20Behav.&title=Dopamine%20D-1%20and%20D-2%20receptor%20antagonists%20potentiate%20analgesic%20and%20motor%20effects%20of%20morphine&author=J.A.%20Kiritsy-Roy&author=S.M.%20Standish&author=L.C.%20Terry&volume=32&publication_year=1989&pages=717-721&pmid=2662224&doi=10.1016/0091-3057(89)90023-3&)

49. Rodgers H.M., Yow J., Evans E., Clemens S., Brewer K.L. Dopamine D1 and D3 receptor modulators restore morphine analgesia and prevent opioid preference in a model of neuropathic pain. Neuroscience. 2019;406:376–388. doi: 10.1016/j.neuroscience.2019.03.034.  [DOI](https://doi.org/10.1016/j.neuroscience.2019.03.034) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30910641/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neuroscience&title=Dopamine%20D1%20and%20D3%20receptor%20modulators%20restore%20morphine%20analgesia%20and%20prevent%20opioid%20preference%20in%20a%20model%20of%20neuropathic%20pain&author=H.M.%20Rodgers&author=J.%20Yow&author=E.%20Evans&author=S.%20Clemens&author=K.L.%20Brewer&volume=406&publication_year=2019&pages=376-388&pmid=30910641&doi=10.1016/j.neuroscience.2019.03.034&)

50. Morgan M.J., Franklin K.B. Dopamine receptor subtypes and formalin test analgesia. Pharmacol. Biochem. Behav. 1991;40:317–322. doi: 10.1016/0091-3057(91)90560-O.  [DOI](https://doi.org/10.1016/0091-3057(91)90560-O) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1687167/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacol.%20Biochem.%20Behav.&title=Dopamine%20receptor%20subtypes%20and%20formalin%20test%20analgesia&author=M.J.%20Morgan&author=K.B.%20Franklin&volume=40&publication_year=1991&pages=317-322&pmid=1687167&doi=10.1016/0091-3057(91)90560-O&)

51. Ward L.D., Kellis M. HaploReg: A resource for exploring chromatin states, conservation, and regulatory motif alterations within sets of genetically linked variants. Nucleic Acids Res. 2012;40:D930–D934. doi: 10.1093/nar/gkr917.  [DOI](https://doi.org/10.1093/nar/gkr917) | [PMC free article](/articles/PMC3245002/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22064851/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&title=HaploReg:%20A%20resource%20for%20exploring%20chromatin%20states,%20conservation,%20and%20regulatory%20motif%20alterations%20within%20sets%20of%20genetically%20linked%20variants&author=L.D.%20Ward&author=M.%20Kellis&volume=40&publication_year=2012&pages=D930-D934&pmid=22064851&doi=10.1093/nar/gkr917&)

52. Xu Z., Taylor J.A. SNPinfo: Integrating GWAS and candidate gene information into functional SNP selection for genetic association studies. Nucleic Acids Res. 2009;37:W600–W605. doi: 10.1093/nar/gkp290.  [DOI](https://doi.org/10.1093/nar/gkp290) | [PMC free article](/articles/PMC2703930/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19417063/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&title=SNPinfo:%20Integrating%20GWAS%20and%20candidate%20gene%20information%20into%20functional%20SNP%20selection%20for%20genetic%20association%20studies&author=Z.%20Xu&author=J.A.%20Taylor&volume=37&publication_year=2009&pages=W600-W605&pmid=19417063&doi=10.1093/nar/gkp290&)

53. GTEx Consortium The Genotype-Tissue Expression (GTEx) project. Nat. Genet. 2013;45:580–585. doi: 10.1038/ng.2653.  [DOI](https://doi.org/10.1038/ng.2653) | [PMC free article](/articles/PMC4010069/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23715323/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet.&title=The%20Genotype-Tissue%20Expression%20(GTEx)%20project&volume=45&publication_year=2013&pages=580-585&pmid=23715323&doi=10.1038/ng.2653&)

54. Man M., Close S.L., Shaw A.D., Bernard G.R., Douglas I.S., Kaner R.J., Payen D., Vincent J.L., Fossceco S., Janes J.M., et al. Beyond single-marker analyses: Mining whole genome scans for insights into treatment responses in severe sepsis. Pharm. J. 2013;13:218–226. doi: 10.1038/tpj.2012.1.  [DOI](https://doi.org/10.1038/tpj.2012.1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22310353/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharm.%20J.&title=Beyond%20single-marker%20analyses:%20Mining%20whole%20genome%20scans%20for%20insights%20into%20treatment%20responses%20in%20severe%20sepsis&author=M.%20Man&author=S.L.%20Close&author=A.D.%20Shaw&author=G.R.%20Bernard&author=I.S.%20Douglas&volume=13&publication_year=2013&pages=218-226&pmid=22310353&doi=10.1038/tpj.2012.1&)

55. Lundin C., Hjorth L., Behrendtz M., Nordgren A., Palmqvist L., Andersen M.K., Biloglav A., Forestier E., Paulsson K., Johansson B. High frequency of BTG1 deletions in acute lymphoblastic leukemia in children with down syndrome. Genes Chromosomes Cancer. 2012;51:196–206. doi: 10.1002/gcc.20944.  [DOI](https://doi.org/10.1002/gcc.20944) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22072402/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genes%20Chromosomes%20Cancer&title=High%20frequency%20of%20BTG1%20deletions%20in%20acute%20lymphoblastic%20leukemia%20in%20children%20with%20down%20syndrome&author=C.%20Lundin&author=L.%20Hjorth&author=M.%20Behrendtz&author=A.%20Nordgren&author=L.%20Palmqvist&volume=51&publication_year=2012&pages=196-206&pmid=22072402&doi=10.1002/gcc.20944&)

56. Safavi S., Hansson M., Karlsson K., Biloglav A., Johansson B., Paulsson K. Novel gene targets detected by genomic profiling in a consecutive series of 126 adults with acute lymphoblastic leukemia. Haematologica. 2015;100:55–61. doi: 10.3324/haematol.2014.112912.  [DOI](https://doi.org/10.3324/haematol.2014.112912) | [PMC free article](/articles/PMC4281313/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25261097/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Haematologica&title=Novel%20gene%20targets%20detected%20by%20genomic%20profiling%20in%20a%20consecutive%20series%20of%20126%20adults%20with%20acute%20lymphoblastic%20leukemia&author=S.%20Safavi&author=M.%20Hansson&author=K.%20Karlsson&author=A.%20Biloglav&author=B.%20Johansson&volume=100&publication_year=2015&pages=55-61&pmid=25261097&doi=10.3324/haematol.2014.112912&)

57. Shi N., Feng D., Gu Y., Zheng C., Miao M. TUSC8 enhances cisplatin sensitivity of NSCLC cells through regulating VEGFA. J. BUON. 2021;26:336–344.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/34076977/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20BUON&title=TUSC8%20enhances%20cisplatin%20sensitivity%20of%20NSCLC%20cells%20through%20regulating%20VEGFA&author=N.%20Shi&author=D.%20Feng&author=Y.%20Gu&author=C.%20Zheng&author=M.%20Miao&volume=26&publication_year=2021&pages=336-344&pmid=34076977&)

58. Mahbuba W., Lambert D.G. Opioids and neovascularization: Pro or anti? Br. J. Anaesth. 2015;115:821–824. doi: 10.1093/bja/aev357.  [DOI](https://doi.org/10.1093/bja/aev357) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26537630/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Anaesth.&title=Opioids%20and%20neovascularization:%20Pro%20or%20anti?&author=W.%20Mahbuba&author=D.G.%20Lambert&volume=115&publication_year=2015&pages=821-824&pmid=26537630&doi=10.1093/bja/aev357&)

59. Gupta K., Kshirsagar S., Chang L., Schwartz R., Law P.Y., Yee D., Hebbel R.P. Morphine stimulates angiogenesis by activating proangiogenic and survival-promoting signaling and promotes breast tumor growth. Cancer Res. 2002;62:4491–4498.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/12154060/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res.&title=Morphine%20stimulates%20angiogenesis%20by%20activating%20proangiogenic%20and%20survival-promoting%20signaling%20and%20promotes%20breast%20tumor%20growth&author=K.%20Gupta&author=S.%20Kshirsagar&author=L.%20Chang&author=R.%20Schwartz&author=P.Y.%20Law&volume=62&publication_year=2002&pages=4491-4498&pmid=12154060&)

60. Suri C., Jones P.F., Patan S., Bartunkova S., Maisonpierre P.C., Davis S., Sato T.N., Yancopoulos G.D. Requisite role of angiopoietin-1, a ligand for the TIE2 receptor, during embryonic angiogenesis. Cell. 1996;87:1171–1180. doi: 10.1016/S0092-8674(00)81813-9.  [DOI](https://doi.org/10.1016/S0092-8674(00)81813-9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8980224/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell&title=Requisite%20role%20of%20angiopoietin-1,%20a%20ligand%20for%20the%20TIE2%20receptor,%20during%20embryonic%20angiogenesis&author=C.%20Suri&author=P.F.%20Jones&author=S.%20Patan&author=S.%20Bartunkova&author=P.C.%20Maisonpierre&volume=87&publication_year=1996&pages=1171-1180&pmid=8980224&doi=10.1016/S0092-8674(00)81813-9&)

61. Shafer S.L., Varvel J.R., Aziz N., Scott J.C. Pharmacokinetics of fentanyl administered by computer-controlled infusion pump. Anesthesiology. 1990;73:1091–1102. doi: 10.1097/00000542-199012000-00005.  [DOI](https://doi.org/10.1097/00000542-199012000-00005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2248388/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anesthesiology&title=Pharmacokinetics%20of%20fentanyl%20administered%20by%20computer-controlled%20infusion%20pump&author=S.L.%20Shafer&author=J.R.%20Varvel&author=N.%20Aziz&author=J.C.%20Scott&volume=73&publication_year=1990&pages=1091-1102&pmid=2248388&doi=10.1097/00000542-199012000-00005&)

62. Subramanian A., Tamayo P., Mootha V.K., Mukherjee S., Ebert B.L., Gillette M.A., Paulovich A., Pomeroy S.L., Golub T.R., Lander E.S., et al. Gene set enrichment analysis: A knowledge-based approach for interpreting genome-wide expression profiles. Proc. Natl. Acad. Sci. USA. 2005;102:15545–15550. doi: 10.1073/pnas.0506580102.  [DOI](https://doi.org/10.1073/pnas.0506580102) | [PMC free article](/articles/PMC1239896/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16199517/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&title=Gene%20set%20enrichment%20analysis:%20A%20knowledge-based%20approach%20for%20interpreting%20genome-wide%20expression%20profiles&author=A.%20Subramanian&author=P.%20Tamayo&author=V.K.%20Mootha&author=S.%20Mukherjee&author=B.L.%20Ebert&volume=102&publication_year=2005&pages=15545-15550&pmid=16199517&doi=10.1073/pnas.0506580102&)

63. Nishizawa D., Kasai S., Hasegawa J., Sato N., Yamada H., Tanioka F., Nagashima M., Katoh R., Satoh Y., Tagami M., et al. Associations between the orexin (hypocretin) receptor 2 gene polymorphism Val308Ile and nicotine dependence in genome-wide and subsequent association studies. Mol. Brain. 2015;8:50. doi: 10.1186/s13041-015-0142-x.  [DOI](https://doi.org/10.1186/s13041-015-0142-x) | [PMC free article](/articles/PMC4546081/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26289589/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Brain&title=Associations%20between%20the%20orexin%20(hypocretin)%20receptor%202%20gene%20polymorphism%20Val308Ile%20and%20nicotine%20dependence%20in%20genome-wide%20and%20subsequent%20association%20studies&author=D.%20Nishizawa&author=S.%20Kasai&author=J.%20Hasegawa&author=N.%20Sato&author=H.%20Yamada&volume=8&publication_year=2015&pages=50&pmid=26289589&doi=10.1186/s13041-015-0142-x&)

64. Benjamini Y., Hochberg Y. Controlling the false discovery rate: A practical and powerful approach to multiple testing. J. R. Stat. Soc. B. 1995;57:289–300. doi: 10.1111/j.2517-6161.1995.tb02031.x.  [DOI](https://doi.org/10.1111/j.2517-6161.1995.tb02031.x) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20R.%20Stat.%20Soc.%20B&title=Controlling%20the%20false%20discovery%20rate:%20A%20practical%20and%20powerful%20approach%20to%20multiple%20testing&author=Y.%20Benjamini&author=Y.%20Hochberg&volume=57&publication_year=1995&pages=289-300&doi=10.1111/j.2517-6161.1995.tb02031.x&)

65. Storey J. The positive False Discovery Rate: A Bayesian interpretation and the q-value. Ann. Statist. 2001;31:2013–2035. doi: 10.1214/aos/1074290335.  [DOI](https://doi.org/10.1214/aos/1074290335) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann.%20Statist.&title=The%20positive%20False%20Discovery%20Rate:%20A%20Bayesian%20interpretation%20and%20the%20q-value&author=J.%20Storey&volume=31&publication_year=2001&pages=2013-2035&doi=10.1214/aos/1074290335&)

66. Sun L., Craiu R.V., Paterson A.D., Bull S.B. Stratified false discovery control for large-scale hypothesis testing with application to genome-wide association studies. Genet. Epidemiol. 2006;30:519–530. doi: 10.1002/gepi.20164.  [DOI](https://doi.org/10.1002/gepi.20164) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16800000/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genet.%20Epidemiol.&title=Stratified%20false%20discovery%20control%20for%20large-scale%20hypothesis%20testing%20with%20application%20to%20genome-wide%20association%20studies&author=L.%20Sun&author=R.V.%20Craiu&author=A.D.%20Paterson&author=S.B.%20Bull&volume=30&publication_year=2006&pages=519-530&pmid=16800000&doi=10.1002/gepi.20164&)

67. Purcell S., Neale B., Todd-Brown K., Thomas L., Ferreira M.A., Bender D., Maller J., Sklar P., de Bakker P.I., Daly M.J., et al. PLINK: A tool set for whole-genome association and population-based linkage analyses. Am. J. Hum. Genet. 2007;81:559–575. doi: 10.1086/519795.  [DOI](https://doi.org/10.1086/519795) | [PMC free article](/articles/PMC1950838/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17701901/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Hum.%20Genet.&title=PLINK:%20A%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses&author=S.%20Purcell&author=B.%20Neale&author=K.%20Todd-Brown&author=L.%20Thomas&author=M.A.%20Ferreira&volume=81&publication_year=2007&pages=559-575&pmid=17701901&doi=10.1086/519795&)

68. Barrett J.C., Fry B., Maller J., Daly M.J. Haploview: Analysis and visualization of LD and haplotype maps. Bioinformatics. 2005;21:263–265. doi: 10.1093/bioinformatics/bth457.  [DOI](https://doi.org/10.1093/bioinformatics/bth457) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15297300/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=Haploview:%20Analysis%20and%20visualization%20of%20LD%20and%20haplotype%20maps&author=J.C.%20Barrett&author=B.%20Fry&author=J.%20Maller&author=M.J.%20Daly&volume=21&publication_year=2005&pages=263-265&pmid=15297300&doi=10.1093/bioinformatics/bth457&)

69. Faul F., Erdfelder E., Lang A.G., Buchner A. G*Power 3: A flexible statistical power analysis program for the social, behavioral, and biomedical sciences. Behav. Res. Methods. 2007;39:175–191. doi: 10.3758/BF03193146.  [DOI](https://doi.org/10.3758/BF03193146) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17695343/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Behav.%20Res.%20Methods&title=G*Power%203:%20A%20flexible%20statistical%20power%20analysis%20program%20for%20the%20social,%20behavioral,%20and%20biomedical%20sciences&author=F.%20Faul&author=E.%20Erdfelder&author=A.G.%20Lang&author=A.%20Buchner&volume=39&publication_year=2007&pages=175-191&pmid=17695343&doi=10.3758/BF03193146&)

70. Cohen J. Statistical Power Analysis for the Behavioral Sciences (Revised Edition) Academic Press; New York, NY, USA: 1977.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Statistical%20Power%20Analysis%20for%20the%20Behavioral%20Sciences%20(Revised%20Edition)&author=J.%20Cohen&publication_year=1977&)

71. Gabriel S.B., Schaffner S.F., Nguyen H., Moore J.M., Roy J., Blumenstiel B., Higgins J., DeFelice M., Lochner A., Faggart M., et al. The structure of haplotype blocks in the human genome. Science. 2002;296:2225–2229. doi: 10.1126/science.1069424.  [DOI](https://doi.org/10.1126/science.1069424) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12029063/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=The%20structure%20of%20haplotype%20blocks%20in%20the%20human%20genome&author=S.B.%20Gabriel&author=S.F.%20Schaffner&author=H.%20Nguyen&author=J.M.%20Moore&author=J.%20Roy&volume=296&publication_year=2002&pages=2225-2229&pmid=12029063&doi=10.1126/science.1069424&)
