# Candidate germline biomarkers of lenalidomide efficacy in mantle cell lymphoma: the Fondazione Italiana Linfomi MCL0208 trial

## Metadata
**Authors:** Simone Ferrero, Daniele Grimaldi, Elena Arrigoni, Mariapia Pironti, Gian Maria Zaccaria, Beatrice Alessandria, Elisa Genuardi, Gabriele De Luca, Marco Ghislieri, Rita Tavarozzi, Alice Di Rocco, Alessandro Re, Vittorio Stefoni, Federica Cavallo, Carola Boccomini, Monica Balzarotti, Vittorio Zilioli, Filipa Moita, Luca Arcaini, Elisa Lucchini, Filippo Ballerini, Andrés J M Ferreri, Benedetta Puccini, Giuseppe A Palumbo, Sara Galimberti, Sergio Cortelazzo, Antonello Di Paolo, Marco Ladetto
**Journal:** Blood Advances
**Date:** 2023 Apr 17
**DOI:** [10.1182/bloodadvances.2022009504](https://doi.org/10.1182/bloodadvances.2022009504)
**PMID:** 37058477
**PMCID:** PMC10368781
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10368781/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC10368781/pdf/main.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC10368781/pdf/main.pdf)

## Abstract

Polymorphic alleles of either ABCB1 or VEGF genes predicted better outcome in patients with MCL receiving LEN after ASCT.

ABCB1, NCF4, GSTP1, and CRBN polymorphisms were associated with toxicity during induction and with LEN dose reductions.

## Introduction

Mantle cell lymphoma (MCL) is an aggressive, mature B-cell non-Hodgkin lymphoma historically known for its poor long-term outcome.[1](#bib1) In recent years, the introduction of novel treatment platforms has led to substantial improvement in patients’ prognosis, almost uniformly including a maintenance phase, which is now considered a critical stem of successful treatment.[2](#bib2), [3](#bib3), [4](#bib4), [5](#bib5) Among maintenance regimens, lenalidomide, an oral immunomodulatory drug, has been demonstrated to have significant activity and a manageable safety profile, both alone and in combination with rituximab.[3](#bib3), [4](#bib4), [5](#bib5), [6](#bib6), [7](#bib7) However, treatment response appears nonuniform across patients, and predicting the clinical drug profile remains an unmet need.

Many factors may affect interindividual responsiveness to drugs, and the role of the host genetic background has also been investigated in lymphoma.[8](#bib8), [9](#bib9), [10](#bib10), [11](#bib11) In particular, gene polymorphisms consist of variations in the gene DNA sequence occurring in a population with a frequency ≥1% and resulting in changes in the expression, structure, and activity of the proteins encoded by these genes. Interestingly, previous reports have suggested that single-nucleotide polymorphisms (SNPs) involving drug metabolic pathways are predictive of both response and toxicity in different histotypes of non-Hodgkin lymphoma.[8](#bib8), [9](#bib9), [10](#bib10), [11](#bib11), [12](#bib12), [13](#bib13), [14](#bib14), [15](#bib15) This has been proven for several agents, including anthracyclines,[12](#bib12) monoclonal antibodies,[13](#bib13)^,^[14](#bib14) and immunomodulatory drugs, including lenalidomide, which is affected by SNPs in *ABCB1*, *ERCC5*, *XPA*, and *GSTP1*, as demonstrated in multiple myeloma (MM).[14](#bib14)^,^[15](#bib15)

The Fondazione Italiana Linfomi (FIL) MCL0208, a prospective multicenter, randomized phase 3 trial, showed a significant progression-free survival (PFS) benefit of lenalidomide maintenance (LEN) vs observation (OBS) in young (<66 years old), previously untreated patients with MCL.[3](#bib3) The study enrolled 300 patients, and a large number of biological samples were stored centrally at fixed time points to document the minimal residual disease (MRD).[16](#bib16) Moreover, several biological substudies,[17](#bib17)^,^[18](#bib18) including a pharmacogenetic analysis, were planned. In particular, we investigated specific germ line polymorphisms of transmembrane transporters, metabolic enzymes, and cell-surface receptors (*ABCB1, ABCG2, VEGFA, FCGR2A, NCF4, GSTP1*, and *CRBN*) that might predict the efficacy and safety of immunochemotherapy and LEN.

## Methods

### Patients and treatment

This study was performed on biological samples collected from the phase 3 MCL0208 study ([NCT02354313](https://clinicaltrials.gov/ct2/show/NCT02354313)) sponsored by FIL. The trial enrolled 300 previously untreated patients with MCL, aged from 18 to 65 years, without clinically significant comorbidities. The planned treatment is shown in [supplemental Figure 1](#appsec1). Briefly, patients received 3 Rituximab-Cyclophosphamide, Doxorubicin, Vincristine and Prednisone (R-CHOP21), followed by R-high-dose cyclophosphamide (4 g/m^2^), 2 cycles of R-high-dose Ara-C (2 g/m^2^ q12h x 3d), and autologous stem cell transplantation (ASCT) conditioned by using the BEAM or FEAM regimen.[3](#bib3) After ASCT, responding patients (either complete or partial response) were randomly assigned to OBS or LEN; 15 mg on days 1-21 every 28 days) for 24 months. Clinical results of the trial have already been published.[3](#bib3) All patients provided written informed consent for the use of their biological samples for research purposes, in accordance with the institutional review board’s requirements and the Helsinki Declaration. The clinical trial as well as the substudy was approved by the ethics committees of all the enrolling centers. All the samples were centralized for scientific analysis in the hematological laboratory of Torino University.

### Biological samples

Bone marrow (BM) and peripheral blood (PB) samples were collected at diagnosis and during follow-up, based on the preplanned time points of MRD analysis[16](#bib16) ([supplemental Figure 1](#appsec1)). Biological samples were identified with a subject study number that can be traced or linked back to the subject only by the site investigator.

### MRD and mutational analysis

MRD monitoring was assessed for BM and PB samples by using an allele-specific oligonucleotide real-time quantitative polymerase chain reaction (ASO RQ-PCR) approach (either on immunoglobulin heavy chain (IGH) gene or B-cell lymphoma-1 (BCL-1)/IGH rearrangements) at the indicated time points and evaluated in accordance with the criteria of the EuroMRD standardization group.[16](#bib16)

*TP53* disruptions as well as *KMT2D* mutations were identified via next-generation targeted resequencing and copy-number alteration analysis on CD-19–selected tumoral cells from BM baseline samples or as previously described.[17](#bib17)

### Pharmacogenetic analyses

For the purposes of the pharmacogenetic study, germinal DNA was extracted by using commercially available kits (QIAamp DNA Blood Mini Kit, Qiagen; Maxwell RSC Blood, Promega; and DNAzol, Invitrogen). gDNA was extracted from different specimens, preferably selecting samples devoid of lymphoma infiltration. A total of 193 patients were studied using a follow-up PB sample, mainly those who were tested to be MRD-negative or had very low levels of MRD ([supplemental Table 5](#appsec1)). When FU samples were not available, baseline PB was used (n = 60), characterized by a median tumor infiltration of 3% via flow cytometry (range, 0.03%-88%; [supplemental Table 5](#appsec1))

Specific germ line polymorphisms of transmembrane transporters, metabolic enzymes, and cell-surface receptors were evaluated. The selected polymorphisms of the *ABCB1* gene were rs1128503, rs2032582, and rs1045642. Additional SNPs belonging to other genes were chosen based on the mechanistic role of their encoded proteins in the pharmacokinetics and pharmacodynamics of lenalidomide. They included *ABCB2* rs2231142, *VEGF*-*A* rs699947, *FCGR2A* rs1801274, *NCF4* rs1883112, *GSTP1* rs1695, and *CRBN* rs1714327 and rs1705814 ([Figure 1](#fig1)). The full list of investigated SNPs with their respective putative functions is shown in [supplemental Table 2](#appsec1). For all the SNPs, a minor allele frequency of ≥0.3 in the European population was considered as a general criterion for selection.

### Figure 1.

![Figure 1.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0eda/10368781/74909002126e/BLOODA_ADV-2022-009504-gr1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=10368781_BLOODA_ADV-2022-009504-gr1.jpg)

Depiction of the biological functions of the proteins encoded by the investigated genes and the impact of relative polymorphisms. ROS, reactive oxygen species; GSTP1, glutathione-S-transferase Pi1; ATP, adenosine triphosphate; FCGR, Fc gamma receptor; VEGF-A, vascular endothelial growth factor A, ABCB1, ATP-binding cassette subfamily B member 1; Ub, ubiquitin; eNOS, endothelial nitric oxide synthase.

Patients’ genotypes with respect to the investigated SNPs were obtained by using specific TaqMan SNP Genotyping Assays (ABI, Applied Biosystems, Foster City, CA) on an ABI Prism HT7900 Sequence Detection System instrument, per the manufacturer’s instructions. Approximately 10% of samples were analyzed in duplicates, and the results showed a concordance rate of >99%. The SDS Software (Applied Biosystems) was used to impute patients’ genotypes, and allele frequencies and genotypes were calculated. The Hardy-Weinberg equilibrium was checked using the Pearson χ^2^ test (threshold value, *P* = 3.841) for each locus, whereas haplotypes and their frequencies were imputed with Arlequin software.[19](#bib19) Finally, differences in genotype (or haplotype) distributions between groups of the study (ie, the LEN vs OBS arm) were evaluated using the Pearson χ^2^ test.

### Toxicity evaluations and lenalidomide dose intensity

Toxic events were recorded per the Common Terminology Criteria for Adverse Events version 4.0. Only hematological toxicities were available in detail and were then correlated with pharmacogenetic data. In particular, granulocytes, platelets, and hemoglobin reductions were considered together across different treatment phases (induction, consolidation, and LEN/OBS). Infective toxicity was evaluated as a cumulative event per the Common Terminology Criteria for Adverse Events, version 4.0. Lenalidomide dose intensity was calculated as the ratio between the effective received dose and the planned dose.

### Statistical analysis

Because of the high number of collected biological and clinical variables, the entire data set underwent systematic post hoc quality control through data warehousing.[20](#bib20)

Statistical analysis was carried out using R version 4.0.0. For continuous variables, the Kruskal-Wallis test was used to compare medians between groups; for categorical variables, depending on the number, the χ^2^ test or Fisher exact test was used.

For survival analysis, PFS and overall survival (OS) were used as clinical end points. PFS was calculated from the date of enrollment in the clinical study to the date of disease progression (event), death from any cause (event), or last follow-up (censoring). OS was measured from the date of enrollment in the clinical study to the date of death from any cause (event) or last follow-up (censoring).[21](#bib21) Survival was estimated with the Kaplan-Meier method, and the log-rank test was applied to compare the survival distributions of the patients. The Cox proportional hazards model was implemented for the univariate and multivariate survival analysis. For all statistical analyses, the level of significance was set at *P* ≤ .05. The outcome data for the present analysis were updated as of December 2017, as planned in the clinical trial.

## Results

### Feasibility of the study

Overall, 300 patients were enrolled in the FIL-MCL0208 clinical trial, and 93% of these patients (278 of 300) were included in the pharmacogenetic study because of the availability of adequate biological samples. In particular, 96% (197 of 205) of the randomly assigned population, 97% (101 of 104) of patients randomly assigned to LEN, and 95% (96 of 101) of those randomly assigned to OBS were genotyped ([supplemental Figure 2](#appsec1)). The main clinical features of these patients are described in [supplemental Table 1](#appsec1).

### Pharmacogenetic analyses

Allele and genotype frequencies of all the investigated SNPs are detailed in [Table 1](#tbl1). The minor allele frequency values did agree with those already calculated in European populations, without significant differences between the LEN and OBS arms. Of note, all SNPs were in Hardy-Weinberg equilibrium, except for the *CRBN* locus, rs1705814.

### Table 1.

|   |   | Whole population |  | LEN arm |  | OBS arm |  | P | chi2 |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| MAF | HWE | MAF | HWE | MAF | HWE |  |  |  |  |
| ABCB1 | c.1236 | 0.444 | 0.881 | 0.436 | 0.113 | 0.448 | 1.811 | .484 | 1.454 |
| ABCB1 | c.2677 | 0.428 | 1.464 | 0.426 | 0.283 | 0.422 | 2.923 | .680 | 0.771 |
| ABCB1 | c.3435 | 0.491 | 0.236 | 0.485 | 0.008 | 0.521 | 0.181 | .732 | 0.625 |
| VEGF | c.2055 | 0.563 | 0.499 | 0.559 | 0.421 | 0.526 | 0.994 | .439 | 1.645 |
| ABCG2 | c.421 | 0.939 | 0.002 | 0.926 | 0.56 | 0.964 | 0.137 | .157 | 2.000 |
| FCGR2A | c.497 | 0.404 | 0.459 | 0.411 | 0.641 | 0.389 | 1.082 | .400 | 1.834 |
| NCF4 | −368 | 0.359 | 0.108 | 0.401 | 0.532 | 0.316 | 0.525 | .266 | 2.652 |
| GSTP1 | 313 | 0.327 | 0.237 | 0.317 | 1.730 | 0.300 | 0.048 | .660 | 0.831 |
| CRBN | rs1714327 | 0.285 | 1.046 | 0.292 | 1.316 | 0.268 | 1.268 | .817 | 0.404 |
| CRBN | rs1705814 | 0.428 | 29.977 | 0.426 | 13.563 | 0.379 | 5.456 | .390 | 1.885 |

Table 1 Caption: Allele and genotype frequencies of SNPs evaluated in the whole series

The expected linkage between the 3 *ABCB1* SNPs was confirmed ([supplemental Table 3](#appsec1)), with percentage values in the LEN and OBS arms of 18.8% and 16.7%; 51.5% and 50.0%; and 29.7% and 33.3% for wild-type (WT) homozygous (ie, CGC/CGC), heterozygous polymorphic (POL) (ie, CGC/TTT), and POL homozygous (ie, TTT/CTT) patients, respectively. No significant differences in haplotype distribution were observed between the enrolled and randomly assigned populations as well as between the 2 arms (χ^2^ = 0.264; *P* = .876).

### Impact of SNPs on treatment efficacy

Among the investigated polymorphisms, only 2, namely *ABCB1* rs2032582 and *VEGF* rs699947, were associated with lenalidomide efficacy in terms of both PFS and OS. In the randomly assigned population, 60 (31%), 107 (54%), and 30 (15%) patients were WT homozygotes (HoWT), heterozygotes (HePOL), and POL homozygotes (HoPOL), respectively, for the *ABCB1* locus. For the *VEGF* rs699947 locus, 42 (21%), 96 (49%), and 59 (30%) patients were HoWT, HePOL, and HoPOL, respectively ([supplemental Table 4](#appsec1)).

Firstly, survival analysis was performed by separately comparing these 3 groups ([supplemental Figure 4](#appsec1)), but HoPOL and HePOL patients were then grouped together because these patients showed superimposable outcomes. In the randomly assigned population, 137 patients (70%) carried at least 1 POL allele for the *ABCB1* rs2032582 locus. Interestingly, HoPOL (WW) and HePOL (GW) had better outcomes when compared with HoWT (GG) in the LEN arm (3-year PFS, 85% vs 70% [*P* = .047] and 3-year OS, 98% vs 90% [*P* = .026]) but not in the OBS arm ([Figure 2](#fig2)A-D).

### Figure 2.

![Figure 2.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0eda/10368781/0ecf14b9b628/BLOODA_ADV-2022-009504-gr2.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=10368781_BLOODA_ADV-2022-009504-gr2.jpg)

Association between ABCB1 or VEGF-A–grouped genotypes and survival in a randomly assigned population. PFS stratified based on ABCB1 genotypes in the LEN (A) and OBS arms (C); OS stratified based on ABCB1 genotypes in the LEN (B) and OBS arms (D); PFS stratified based on VEGF-A genotypes in the LEN (E) and OBS arms (G); OS stratified based on VEGF-A genotypes in the LEN (F) and OBS arms (H). ABCB1: HoWT = GG, HePOL = GW, HoPOL = WW; VEGF-A: HoWT = AA, HePOL = AC, HoPOL = CC.

Similarly, 155 patients (79%) carried at least 1 variant allele of the *VEGF-A* locus, such that HoPOL (CC) and HePOL (AC) had better outcomes than HoWT in the LEN arm (3-year PFS, 85% vs 60% [*P* = .0021] and 3-year OS, 90% vs 86.5% [*P* = .094]) but not in the OBS arm ([Figure 2](#fig2)E-H).

Accordingly, when compared with HoWTs, patients carrying variant genotypes of either gene (*ABCB1* or *VEGF-A*) showed a trend toward deeper MRD clearance via RQ-PCR in the BM after 6 months of LEN ([supplemental Figure 3](#appsec1)). In this population, as well as in the overall series, HoWT did not differ from HoPOL and HePOL based on baseline clinical features, classical prognosticators (including *TP53* aberrations), *ABCB1*, and *VEGF-A* polymorphisms ([supplemental Table 3](#appsec1)).

Moreover, by analyzing the combinatorial effect of each polymorphism in the same patients, only the small group of double WT cases (ie, *ABCB1* HoWT and *VEGF-A* HoWT) receiving lenalidomide were actually associated with poor survival, in terms of both PFS and OS ([Figure 3](#fig3)). Therefore, we reconsidered the efficacy of lenalidomide maintenance by stratifying the randomly assigned population based on its pharmacogenomic background. Survival analysis showed that patients carrying at least 1 variant allele of *ABCB1/VEGFA* POL loci experienced the highest benefit from LEN (PFS hazard ratio = 0.41 [95% confidence interval [CI], 0.22-0.75]; *P* = .004, vs PFS hazard ratio = 0.51 [95% CI, 0.30-0.87]; *P* = .012) of the entire population.[22](#bib22) In contrast, based on the limited number of patients who were double WT (n = 17), this genotype seemed to confer no benefit at all in the LEN vs the OBS arm (*P* = .632), as shown in [Figure 4](#fig4).

### Figure 3.

![Figure 3.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0eda/10368781/81b625e71a54/BLOODA_ADV-2022-009504-gr3.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=10368781_BLOODA_ADV-2022-009504-gr3.jpg)

Association between combined ABCB1/VEGF-A genotypes and survival in the LEN population. PFS (A) and OS (B) stratified based on the combination of ABCB1 and VEGF-A genotypes.

### Figure 4.

![Figure 4.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0eda/10368781/ef4766bcc9e4/BLOODA_ADV-2022-009504-gr4.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=10368781_BLOODA_ADV-2022-009504-gr4.jpg)

PFS by randomization arm stratified on the pharmacogenomic background. PFS stratified by ABCB1 and VEGF-A genotypes in the LEN and OBS arms: ABCB1/VEGF-A HoWT (A) and ABCB1/VEGF-A POL (B). HR, hazard ratio.

### Impact of SNPs on treatment toxicity

#### Lenalidomide dose reduction

Dose reduction of lenalidomide was significantly associated only with the *CRBN rs*1705814 genotype. In the studied population, 113 patients (41%) were HoWT (TT), 91 (33%) were HePOL (TC), and 73 (26%) were HoPOL (CC) ([supplemental Table 4](#appsec1)). It is worth noting that 28 patients randomly assigned to LEN (28%) were HoPOL for the *CRBN* rs1705814 locus and had a higher risk of major lenalidomide dose reduction (more than 66%) or discontinuation during the maintenance phase with respect to HoWT/HePOL (odds ratio [OR], 3.24; CI, 1.69-6.21; *P* = .013). Nonetheless, in the LEN randomization arm, no statistically significant impact of the *CRBN* rs1705814 SNP was observed on either hematological toxicities or infections ([Table 2](#tbl2)).

| Polymorphism | ABCB1 c.2677 G>T/A | NCF4 c.368G>A | GSTP1 c.313.A>G | CRBN rs1705814 T>C |
| --- | --- | --- | --- | --- |
| Hematological toxicity during the induction phase | HoPOL:OR, 0.39 (CI, 0.15-0.88); P = .033 | HoPOL/HePOL: OR, 0.56 (CI, 0.34-0.92); P = .024 | HoPOL:OR, 0.35 (CI, 0.15-0.79); P = .014 |   |
| Combined effect |   | HoWT: OR, 2.26 (CI, 1.09-4.83); P = .031 |  |   |
| Infective toxicity | HoPOL: OR, 0.53 (CI, 0.30-0.95) P = .030 |   |   | HoPOL: OR, 0.39 (CI, 0.22-0.68); P = .001 |
| Lenalidomide dose reduction |   |   |   | HoPOL: OR, 3.24 (CI, 1.69-6.21); P = .013 |

Table Caption: Association of investigated SNPs with toxic events

#### Hematological toxicity

Three polymorphisms were associated with hematological toxicity in the whole population of patients: *ABCB1* rs2032582*, NCF4* rs1883112, and *GSTP1* rs1695 (see [supplemental Table 3](#appsec1) for the distribution of genotypes). Indeed, 46 patients (17%) carrying *ABCB1* TT/AT/AA genotypes had a lower risk of hematological toxicity after the first R-CHOP cycle than did patients carrying HePOL/HoWT (OR for G ≥3 toxicity, 0.39; CI, 0.15-0.88; *P* = .033). Moreover, 162 patients (58%) carried *NCF4* AG/GG genotypes and were exposed to an overall lower risk of hematological toxicities during induction (within the R-high-dose cyclophosphamide cycle), when compared with HoWT (OR, 0.56; CI, 0.34-–0.92; *P* = .024; [Table 2](#tbl2)). Similarly, 28 patients (10%) carried the *GSTP1* GG HoPOL genotype and were exposed to a lower risk of toxicities than HePOL/HoWT (OR, 0.35; CI, 0.15-0.79; *P* = .014; [Table 2](#tbl2)). Interestingly, by combining the postulated protective effect of both SNPs, we were able to identify a subgroup of patients (*NCF4* and *GSTP1* HoWT, n = 48) who were at a higher risk of hematological toxicity during induction (OR, 2.26; CI, 1.09-4.83; *P* = .031). No impact of these SNPs was observed during later treatment phases (R-high dose Ara-C)or on hematological recovery after ASCT.

#### Infective toxicity

Two polymorphisms, *ABCB1* rs1045642 and *CRBN rs*1705814, were associated with infections in the whole series of patients during chemoimmunotherapy. In the studied population, the following allele frequencies were found: for *ABCB1 c.3435*.C>T, 70 patients (25%) were HoWT (CC), 143 (51%) were HePOL (CT), and 65 (23%) were HoPOL (TT). For *CRBN* rs1705814 T>C, 113 patients (41%) were HoWT (TT), 91 (33%) were HePOL (TC), and 73 (26%) were HoPOL (CC) ([supplemental Table 3](#appsec1)). The 65 out of 278 patients carrying the *ABCB1 TT* genotype ([supplemental Table 4](#appsec1)) were exposed to a lower risk of infections than were CT/CC individuals (OR, 0.53; CI, 0.30-0.95; *P* = .030). Similarly, 73 patients HoPOL for the *CRBN* rs1705814 locus had a lower risk of severe infections than those who were HoWT/HePOL (OR for G≥3 toxicity, 0.39; CI, 0.22-0.68; *P* = .001). On In contrast, none of the SNPs predicted the onset of infections during LEN.

## Discussion

The pharmacogenetic study of the FIL MCL0208 phase 3 randomized trial showed that SNPs of genes encoding specific cellular proteins may be associated with clinical outcome in younger patients with MCL receiving LEN after ASCT. Most importantly, we observed that

Notably, no significant differences in genotype distribution were observed between the 2 randomization arms, resetting any possible bias due to the unbalanced stratification of patients and reducing the weight of confounding factors in the LEN phase. In addition, some SNPs (namely, *ABCB1 c.3435C>T*, *NCF4 c.368G>A*, *CRBN* rs1705814 *T>C*, and *GSTP1 c.313.A>G*) were associated with hematological and infective toxicity during the immunochemotherapy phase preceding the randomization.

In recent years, several studies have underlined the potential role of interindividual genetic differences in shaping treatment response and toxicity in lymphoma.[23](#bib23)^,^[24](#bib24) Nonetheless, to date, little pharmacogenetic knowledge is available in the MCL literature,[25](#bib25), [26](#bib26), [27](#bib27), [28](#bib28) and no data relative to lenalidomide treatment in MCL have been published. In contrast, several pharmacogenetic studies have been published, mainly in MM, assessing the relation of lenalidomide activity with polymorphisms of *CRBN*, one of lenalidomide’s most investigated molecular targets.[22](#bib22)^,^[29](#bib29), [30](#bib30), [31](#bib31) Thus, to our knowledge, this study provides the first pharmacogenetic data related to lenalidomide treatment in MCL by investigating the effects of candidate genes and polymorphisms on both efficacy and toxicity. Because some of the genes may play a role in immunochemotherapy outcomes and tolerability, the investigated panel of loci was also assessed for these effects in the whole population of patients.

Our main findings suggested a key role of *ABCB1* and *VEGF-A* polymorphisms in enhancing the clinical activity of LEN after ASCT in MCL. Biologically, we might hypothesize that variant alleles of *ABCB1*, a transmembrane drug transporter, leading either to decreased gene expression in gut cells and renal tubules or to reduced ejection activity of the transporter,[32](#bib32), [33](#bib33), [34](#bib34) resulted in greater lenalidomide bioavailability, leading to an increased pharmacological effect due to higher plasma and tissue drug concentrations. This hypothesis is in line with published literature on lenalidomide pharmacokinetics in MM,[35](#bib35)^,^[36](#bib36) but in this study, no gut or renal tissue was available to investigate the hypothesis.

Similarly, the POL genotype of *VEGF-A* might lead to decreased gene expression (as suggested in other scenarios[37](#bib37), [38](#bib38), [39](#bib39)), resulting in decreased stimulation by this key cytokine in the PI3K-Akt pathway.[40](#bib40) Unfortunately, adequate lymphoma tissue for gene expression analysis was available only for a minority of the patients in this trial, so no statistically relevant conclusions might be drawn from these data.[41](#bib41) Anyway, given the direct inhibitory effect of lenalidomide on VEGF-A–mediated Akt phosphorylation, the presence of the *VEGF-A C* POL allele might additively enhance lenalidomide’s pharmacological effect. Therefore, we hypothesized that the absence of both POL genotypes might lead to lower lenalidomide and higher VEGF-A concentrations and, thus, act as a predictive biomarker of poor response to lenalidomide in MCL. Accordingly, patients carrying neither *ABCB1* nor *VEGF-A* polymorphisms (actually, 9% of the series) did not benefit from LEN when compared with OBS only ([Figure 4](#fig4)B). Thus, these results suggest a possible means of selecting patient candidates for lenalidomide to improve its efficacy and concomitantly reduce the risk of toxicity. More cumbersome is the association found between the *CRBN* rs1705814 POL genotype and lenalidomide dose reduction. We might hypothesize that this genotype could reduce the intracellular expression of cereblon, the molecular target of lenalidomide, and the minimum lenalidomide dose required to obtain the therapeutic effect. However, our analysis was not able to show any statistically significant association between this SNP and a higher risk of hematological or infectious toxicities, possibly explaining the observed lenalidomide dose reduction. Further analyses are required to explain this finding, including the roles of different toxic events not reported in the trial electronic case report form (eCRFs) as well as a wider validation on an external, well-annotated patient population, such as the recently completed EuMCLNet R2 Elderly trial.[42](#bib42) Finally, we cannot exclude the possibility that other polymorphisms might serve as possible biomarkers of lenalidomide efficacy, but none of the analyzed genes other than *ABCB1* and *VEGF-A* showed promising survival trends that deserved to be further investigated (data not shown).

Considering the standard chemotherapeutic regimens administered to patients before ASCT, it was hypothesized that some of the SNPs identified and evaluated in the present work could also act as predictive markers of tolerability because proteins encoded by the corresponding genes are involved in the transmembrane transport (ABCB1) or detoxification (GSTP1 and NCF) of cyclophosphamide, vincristine, and doxorubicin. Indeed, we found that at least 3 loci in different genes were significantly associated with hematological toxicities during the induction phase, with an increased risk for patients carrying WT genotypes. That risk was coupled with an increased incidence of severe infections, at least when considering *ABCB1*.

Our study has some strengths. Firstly, the robustness and reproducibility of RQ-PCR analysis ensure its theoretical large-scale applicability in clinical practice across different labs because the present analyses can be carried out on several instrumental platforms, including the most recent ones. Easy standardization, a patient-friendly PB source, and overall limited costs also favor this approach. Furthermore, it is intriguing to derive predictive biomarkers by investigating the genetic background of every single patient, in addition to the intrinsic alterations of tumor cells, and, hence, allow wider approaches to obtain somatic DNA (ie, buccal swab). Nevertheless, although promising, our strategy identified only a limited number of cases with an adverse genotype, and we totally lack functional studies; therefore, an external validation on an independent series of patients receiving lenalidomide is needed for validation of these candidate biomarkers. Moreover, we acknowledge that we have investigated, in this cohort, only a finite number of gene polymorphisms, selected based on data already available in the literature, and they probably do not represent all the pharmacogenomics targets potentially involved in lenalidomide’s mechanism of action. In this regard, current genome-wide sequencing tools may be applied in the future as more convenient technical approaches to study, in one-run, larger panels of different gene polymorphisms. Finally, the number of patients with MCL enrolled in this study is the largest so far. However, the sample size of this study might have limited the comprehensive detection of associations with less common alleles. Additional studies on larger MCL populations will help confirm the current findings and may reveal additional variants that influence drug-associated efficacy and toxicity. Overall, pharmacogenomics might play a role in MCL therapy, enabling clinicians to provide tailored treatment based on individual patients’ genetic profiles, possibly complementing classical mutational and expression analysis carried out on tumor cells.[17](#bib17)^,^[43](#bib43) This paradigm might also be scalable to several novel drugs currently used in MCL treatment (such as Bruton tyrosine kinase inhibitors or bispecific antibodies)[44](#bib44)^,^[45](#bib45) as well as in other lymphomas[46](#bib46) or MM, for which lenalidomide is widely used.[47](#bib47)^,^[48](#bib48) In fact, the comprehensive identification of patients who benefit the most from lenalidomide or other therapeutic approaches would not only bring clinical benefits and improved quality of life to our patients but would also be valuable from a pharmacoeconomic perspective.

In conclusion, this study demonstrates that pharmacogenetic analysis of PB samples could easily yield predictive biomarkers of poor response to LEN after ASCT in patients with MCL. Despite some methodological limitations, this approach is shown to be promising and theoretically scalable to different clinical contexts. Thus, it is a perfect example of how precision medicine might present an exceptional opportunity for our patients in the near future.

Conflict-of-interest disclosures: S.F. is a consultant for Janssen and EUSA Pharma; is on the advisory board of Janssen, EUSA Pharma, Incyte, and Clinigen; received speaker’s honoraria from Janssen, EUSA Pharma, Servier, and Gentili ; and received research funding from Gilead and MorphoSys G.A.P. received speaker’s fees from AbbVie, Bristol Myers Squibb (BMS), Incyte, and Novartis; reports service on advisory boards for AbbVie Orphan Pharmaceuticals, AstraZeneca, Bristol Myers Squibb, and Novartis; and reports support for attending meetings from AbbVie, Bristol Myers Squibb, Janssen, and Novartis. M.L. received honoraria from AbbVie, Acerta, Amgen, GSKI, Gentili, Sandoz, Gilead/Kite, Novartis, Roche, EUSA Pharma, Takeda, Regeneron, Incyte, and Jazz; received honoraria and research funding from ADC Therapeutics, BeiGene, Celgene, and Janssen. L.A. reports consulting or advisory role in Roche, Janssen-Cilag, Verastem, Incyte, EUSA Pharma, Celgene/Bristol Myers Squibb, Kite/Gilead, and ADC Therapeutics; is on speakers' bureau at EUSA Pharma and Novartis; and received research funding from Gilead Sciences. V.Z. reports consulting or advisory role in Takeda, Janssen, MSD, Roche, Servier, and Kite; received support for travel, accommodations, and expenses from Takeda and Janssen. S.G. reports being speaker at events supported by Jazz, Janssen, Pfizer, Novartis, Incyte, AstraZeneca, and AbbVie. F.C. reports advisory or consulting role for Roche and AstraZeneca; reports being speaker for Servier; and receives travel accommodation from Takeda, AstraZeneca, and Roche. C.B. reports consulting or advisory role in AbbVie and AstraZeneca; and receives support for travel, accommodations, and expenses from Janssen. The remaining authors declare no competing financial interests.

## Acknowledgments

The authors thank all the patients who participated in the study. They are grateful to Daniela Barbero, Pier Paola Fenoglio, Daniela Drandi, and Barbara Mantoan for their scientific advice and to Sonia Perticone, Antonella Ferranti, Daniela Gioia, Antonella Fiorillo, and Giulia Bondielli for their assistance.

This work was supported by Progetto di Rilevante Interesse Nazionale (PRIN2009) from the Ministero Italiano dell'Università e della Ricerca, Roma, Italy (7.07.02.60 AE01); Progetto di Ricerca Sanitaria Finalizzata 2009 (RF-2009-1469205) and 2010 (RF-2010-2307262) (S.C.), AOS Maurizio, Bolzano/Bozen, Italy; Fondi di Ricerca Locale, Università degli Studi di Torino, Italy; Fondazione Neoplasie Del Sangue, Torino, Italy; Fondazione CRT (project codes 2016.0677 and 2018.1284), Torino, Italy; AL-AIL ODV Sezione Interprovinciale di Alessandria-Asti; Associazione DaRosa, Torino, Italy; the Gilead Fellowship Program 2019, Milano, Italy; and Cancer Research UK (C355/A26819) and FC AECC and AIRC under the Accelerator Award Program.

### Authorship

Contribution: G.A.P., S.G., S.C., A.D.P., and M.L. conceived and designed the study; S.F., D.G., A.D.R., A.R., V.S., F.C., C.B., M.B., V.Z., F.M., L.A., E.L., F.B., A.J.M.F., B.P., S.C., and M.L. enrolled patients and provided biological samples; E.A., B.A., and E.G. performed experiments; S.F., D.G., M.P., G.M.Z., M.G., and A.D.P. collected and analyzed data; G.D.L. performed statistical analysis; and S.F., D.G., M.P., R.T., G.A.P., S.G., A.D.P., and M.L. wrote the paper; and all the authors approved the final version of the manuscript.

## Footnotes

## Supplementary Material

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Hermine O, Hoster E, Walewski J, et al. Addition of high-dose cytarabine to immunochemotherapy before autologous stem-cell transplantation in patients aged 65 years or younger with mantle cell lymphoma (MCL Younger): a randomised, open-label, phase 3 trial of the European Mantle Cell Lymphoma Network. Lancet. 2016;388(10044):565–575. doi: 10.1016/S0140-6736(16)00739-X.  [DOI](https://doi.org/10.1016/S0140-6736(16)00739-X) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27313086/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Addition%20of%20high-dose%20cytarabine%20to%20immunochemotherapy%20before%20autologous%20stem-cell%20transplantation%20in%20patients%20aged%2065%20years%20or%20younger%20with%20mantle%20cell%20lymphoma%20(MCL%20Younger):%20a%20randomised,%20open-label,%20phase%203%20trial%20of%20the%20European%20Mantle%20Cell%20Lymphoma%20Network&author=O%20Hermine&author=E%20Hoster&author=J%20Walewski&volume=388&issue=10044&publication_year=2016&pages=565-575&pmid=27313086&doi=10.1016/S0140-6736(16)00739-X&)

2. Le Gouill S, Thieblemont C, Oberic L, et al. Rituximab after autologous stem-cell transplantation in mantle-cell lymphoma. N Engl J Med. 2017;377(13):1250–1260. doi: 10.1056/NEJMoa1701769.  [DOI](https://doi.org/10.1056/NEJMoa1701769) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28953447/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%C2%A0Engl%20J%20Med&title=Rituximab%20after%20autologous%20stem-cell%20transplantation%20in%20mantle-cell%20lymphoma&author=S%20Le%20Gouill&author=C%20Thieblemont&author=L%20Oberic&volume=377&issue=13&publication_year=2017&pages=1250-1260&pmid=28953447&doi=10.1056/NEJMoa1701769&)

3. Ladetto M, Cortelazzo S, Ferrero S, et al. Lenalidomide maintenance after autologous haematopoietic stem-cell transplantation in mantle cell lymphoma: results of a Fondazione Italiana Linfomi (FIL) multicentre, randomised, phase 3 trial. Lancet Haematol. 2021;8(1):e34–e44. doi: 10.1016/S2352-3026(20)30358-6.  [DOI](https://doi.org/10.1016/S2352-3026(20)30358-6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33357480/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet%20Haematol&title=Lenalidomide%20maintenance%20after%20autologous%20haematopoietic%20stem-cell%20transplantation%20in%20mantle%20cell%20lymphoma:%20results%20of%20a%20Fondazione%20Italiana%20Linfomi%20(FIL)%20multicentre,%20randomised,%20phase%203%20trial&author=M%20Ladetto&author=S%20Cortelazzo&author=S%20Ferrero&volume=8&issue=1&publication_year=2021&pages=e34-e44&pmid=33357480&doi=10.1016/S2352-3026(20)30358-6&)

4. Klener P, Salek D, Pytlik R, et al. Rituximab maintenance significantly prolongs progression-free survival of patients with newly diagnosed mantle cell lymphoma treated with the Nordic MCL2 protocol and autologous stem cell transplantation. Am J Hematol. 2019;94(2):E50–E53. doi: 10.1002/ajh.25362.  [DOI](https://doi.org/10.1002/ajh.25362) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30474171/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hematol&title=Rituximab%20maintenance%20significantly%20prolongs%20progression-free%20survival%20of%20patients%20with%20newly%20diagnosed%20mantle%20cell%20lymphoma%20treated%20with%20the%20Nordic%20MCL2%20protocol%20and%20autologous%20stem%20cell%20transplantation&author=P%20Klener&author=D%20Salek&author=R%20Pytlik&volume=94&issue=2&publication_year=2019&pages=E50-E53&pmid=30474171&doi=10.1002/ajh.25362&)

5. Ribrag V, Safar V, Kluin-Nelemans H, et al. Rituximab-lenalidomide (r2) maintenance is superior to rituximab maintenance after first line immunochemotherapy in mantle cell lymphoma: results of the MCL R2 Elderly clinical trial [abstract]. Abstract 2021. Blood. 2021;138(Suppl 1):379.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Rituximab-lenalidomide%20(r2)%20maintenance%20is%20superior%20to%20rituximab%20maintenance%20after%20first%20line%20immunochemotherapy%20in%20mantle%20cell%20lymphoma:%20results%20of%20the%20MCL%20R2%20Elderly%20clinical%20trial%20%5Babstract%5D.%20Abstract%202021&author=V%20Ribrag&author=V%20Safar&author=H%20Kluin-Nelemans&volume=138&issue=Suppl%201&publication_year=2021&pages=379&)

6. Ruan J, Martin P, Christos P, et al. Five-year follow-up of lenalidomide plus rituximab as initial treatment of mantle cell lymphoma. Blood. 2018;132(19):2016–2025. doi: 10.1182/blood-2018-07-859769.  [DOI](https://doi.org/10.1182/blood-2018-07-859769) | [PMC free article](/articles/PMC6634960/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30181173/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Five-year%20follow-up%20of%20lenalidomide%20plus%20rituximab%20as%20initial%20treatment%20of%20mantle%20cell%20lymphoma&author=J%20Ruan&author=P%20Martin&author=P%20Christos&volume=132&issue=19&publication_year=2018&pages=2016-2025&pmid=30181173&doi=10.1182/blood-2018-07-859769&)

7. Trněný M, Lamy T, Walewski J, et al. SPRINT trial investigators and in collaboration with the European Mantle Cell Lymphoma Network. Lenalidomide versus investigator's choice in relapsed or refractory mantle cell lymphoma (MCL-002; SPRINT): a phase 2, randomised, multicentre trial. Lancet Oncol. 2016;17(3):319–331. doi: 10.1016/S1470-2045(15)00559-8.  [DOI](https://doi.org/10.1016/S1470-2045(15)00559-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26899778/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet%20Oncol&title=SPRINT%20trial%20investigators%20and%20in%20collaboration%20with%20the%20European%20Mantle%20Cell%20Lymphoma%20Network.%20Lenalidomide%20versus%20investigator's%20choice%20in%20relapsed%20or%20refractory%20mantle%20cell%20lymphoma%20(MCL-002;%20SPRINT):%20a%20phase%202,%20randomised,%20multicentre%20trial&author=M%20Trn%C4%9Bn%C3%BD&author=T%20Lamy&author=J%20Walewski&volume=17&issue=3&publication_year=2016&pages=319-331&pmid=26899778&doi=10.1016/S1470-2045(15)00559-8&)

8. Rossi D, Rasi S, Capello D, Gaidano G. Prognostic assessment of BCL2-938C>A polymorphism in chronic lymphocytic leukemia. Blood. 2008;111(1):466–468. doi: 10.1182/blood-2007-08-106823.  [DOI](https://doi.org/10.1182/blood-2007-08-106823) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18156504/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Prognostic%20assessment%20of%20BCL2-938C>A%20polymorphism%20in%20chronic%20lymphocytic%20leukemia&author=D%20Rossi&author=S%20Rasi&author=D%20Capello&author=G%20Gaidano&volume=111&issue=1&publication_year=2008&pages=466-468&pmid=18156504&doi=10.1182/blood-2007-08-106823&)

9. Rossi D, Rasi S, Di Rocco A, et al. The host genetic background of DNA repair mechanisms is an independent predictor of survival in diffuse large B-cell lymphoma. Blood. 2011;117(8):2405–2413. doi: 10.1182/blood-2010-07-296244.  [DOI](https://doi.org/10.1182/blood-2010-07-296244) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21156845/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=The%20host%20genetic%20background%20of%20DNA%20repair%20mechanisms%20is%20an%20independent%20predictor%20of%20survival%20in%20diffuse%20large%20B-cell%20lymphoma&author=D%20Rossi&author=S%20Rasi&author=A%20Di%20Rocco&volume=117&issue=8&publication_year=2011&pages=2405-2413&pmid=21156845&doi=10.1182/blood-2010-07-296244&)

10. Galimberti S, Nagy B, Palumbo GA, et al. Vascular endothelial growth factor polymorphisms in mantle cell lymphoma. Acta Haematol. 2010;123(2):91–95. doi: 10.1159/000267826.  [DOI](https://doi.org/10.1159/000267826) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20029173/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Acta%20Haematol&title=Vascular%20endothelial%20growth%20factor%20polymorphisms%20in%20mantle%20cell%20lymphoma&author=S%20Galimberti&author=B%20Nagy&author=GA%20Palumbo&volume=123&issue=2&publication_year=2010&pages=91-95&pmid=20029173&doi=10.1159/000267826&)

11. Rossi D, Bruscaggin A, La Cava P, et al. The genotype of MLH1 identifies a subgroup of follicular lymphoma patients who do not benefit from doxorubicin: FIL-FOLL study. Hematologica. 2015;100(4):517–524. doi: 10.3324/haematol.2014.108183.  [DOI](https://doi.org/10.3324/haematol.2014.108183) | [PMC free article](/articles/PMC4380725/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25596266/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hematologica&title=The%20genotype%20of%20MLH1%20identifies%20a%20subgroup%20of%20follicular%20lymphoma%20patients%20who%20do%20not%20benefit%20from%20doxorubicin:%20FIL-FOLL%20study&author=D%20Rossi&author=A%20Bruscaggin&author=P%20La%20Cava&volume=100&issue=4&publication_year=2015&pages=517-524&pmid=25596266&doi=10.3324/haematol.2014.108183&)

12. Ding H, Jin X, Ding N, Fu Z, Song Y, Zhu J. Single nucleotide polymorphisms of CD20 gene and their relationship with clinical efficacy of R-CHOP in patients with diffuse large B cell lymphoma. Cancer Cell Int. 2013;13:58. doi: 10.1186/1475-2867-13-58.  [DOI](https://doi.org/10.1186/1475-2867-13-58) | [PMC free article](/articles/PMC3686705/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23758737/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Cell%20Int&title=Single%20nucleotide%20polymorphisms%20of%20CD20%20gene%20and%20their%20relationship%20with%20clinical%20efficacy%20of%20R-CHOP%20in%20patients%20with%20diffuse%20large%20B%20cell%20lymphoma&author=H%20Ding&author=X%20Jin&author=N%20Ding&author=Z%20Fu&author=Y%20Song&volume=13&publication_year=2013&pages=58&pmid=23758737&doi=10.1186/1475-2867-13-58&)

13. Strefford JC, Nowicka M, Hargreaves CE, et al. Single-nucleotide Fcγ receptor polymorphisms do not impact obinutuzumab/rituximab outcome in patients with lymphoma. Blood Adv. 2021;5(15):2935–2944. doi: 10.1182/bloodadvances.2020003985.  [DOI](https://doi.org/10.1182/bloodadvances.2020003985) | [PMC free article](/articles/PMC8361457/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34323957/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood%20Adv&title=Single-nucleotide%20Fc%CE%B3%20receptor%20polymorphisms%20do%20not%20impact%20obinutuzumab/rituximab%20outcome%20in%20patients%20with%20lymphoma&author=JC%20Strefford&author=M%20Nowicka&author=CE%20Hargreaves&volume=5&issue=15&publication_year=2021&pages=2935-2944&pmid=34323957&doi=10.1182/bloodadvances.2020003985&)

14. Jakobsen Falk I, Lund J, Gréen H, et al. Pharmacogenetic study of the impact of ABCB1 single-nucleotide polymorphisms on lenalidomide treatment outcomes in patients with multiple myeloma: results from a phase IV observational study and subsequent phase II clinical trial. Cancer Chemother Pharmacol. 2018;81(1):183–193. doi: 10.1007/s00280-017-3481-8.  [DOI](https://doi.org/10.1007/s00280-017-3481-8) | [PMC free article](/articles/PMC5754426/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29177954/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Chemother%20Pharmacol&title=Pharmacogenetic%20study%20of%20the%20impact%20of%20ABCB1%20single-nucleotide%20polymorphisms%20on%20lenalidomide%20treatment%20outcomes%20in%20patients%20with%20multiple%20myeloma:%20results%20from%20a%20phase%20IV%20observational%20study%20and%20subsequent%20phase%20II%20clinical%20trial&author=I%20Jakobsen%20Falk&author=J%20Lund&author=H%20Gr%C3%A9en&volume=81&issue=1&publication_year=2018&pages=183-193&pmid=29177954&doi=10.1007/s00280-017-3481-8&)

15. de Larrea CF, Navarro A, Tovar N, et al. Impact of single nucleotide polymorphisms in genes involved in DNA repair and drug metabolism on survival after autologous stem cell transplantation in patients with multiple myeloma. Blood. 2012;120(21):2934.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Impact%20of%20single%20nucleotide%20polymorphisms%20in%20genes%20involved%20in%20DNA%20repair%20and%20drug%20metabolism%20on%20survival%20after%20autologous%20stem%20cell%20transplantation%20in%20patients%20with%20multiple%20myeloma&author=CF%20de%20Larrea&author=A%20Navarro&author=N%20Tovar&volume=120&issue=21&publication_year=2012&pages=2934&)

16. Ferrero S, Grimaldi D, Genuardi E, et al. Punctual and kinetic MRD analysis from the Fondazione Italiana Linfomi MCL0208 phase III trial in mantle cell lymphoma. Blood. 2022;140(12):1378–1389. doi: 10.1182/blood.2021014270.  [DOI](https://doi.org/10.1182/blood.2021014270) | [PMC free article](/articles/PMC9507010/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35737911/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Punctual%20and%20kinetic%20MRD%20analysis%20from%20the%20Fondazione%20Italiana%20Linfomi%20MCL0208%20phase%20III%20trial%20in%20mantle%20cell%20lymphoma&author=S%20Ferrero&author=D%20Grimaldi&author=E%20Genuardi&volume=140&issue=12&publication_year=2022&pages=1378-1389&pmid=35737911&doi=10.1182/blood.2021014270&)

17. Ferrero S, Rossi D, Rinaldi A, et al. KMT2D mutations and TP53 disruptions are poor prognostic biomarkers in mantle cell lymphoma receiving high-dose therapy: a FIL study. Haematologica. 2020;105(6):1604–1612. doi: 10.3324/haematol.2018.214056.  [DOI](https://doi.org/10.3324/haematol.2018.214056) | [PMC free article](/articles/PMC7271566/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31537689/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Haematologica&title=KMT2D%20mutations%20and%20TP53%20disruptions%20are%20poor%20prognostic%20biomarkers%20in%20mantle%20cell%20lymphoma%20receiving%20high-dose%20therapy:%20a%20FIL%20study&author=S%20Ferrero&author=D%20Rossi&author=A%20Rinaldi&volume=105&issue=6&publication_year=2020&pages=1604-1612&pmid=31537689&doi=10.3324/haematol.2018.214056&)

18. Alessandria B, Genuardi E, Civita A, et al. P1245: Clinical impact of immunoglobulin heavy chain repertoire in mantle cell lymphoma: a study from the Fondazione Italiana Iinfomi (fil) phase III mcl0208 trial. Hemasphere. 2022;6:1130–1131.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hemasphere&title=P1245:%20Clinical%20impact%20of%20immunoglobulin%20heavy%20chain%20repertoire%20in%20mantle%20cell%20lymphoma:%20a%20study%20from%20the%20Fondazione%20Italiana%20Iinfomi%20(fil)%20phase%20III%20mcl0208%20trial&author=B%20Alessandria&author=E%20Genuardi&author=A%20Civita&volume=6&publication_year=2022&pages=1130-1131&)

19. Excoffier L, Laval G, Schneider S. Arlequin ver. 3.0: an integrated software package for population genetics data analysis. Evol Bioinform Online. 2007;1:47–50.  [PMC free article](/articles/PMC2658868/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19325852/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Evol%20Bioinform%20Online&title=Arlequin%20ver.%203.0:%20an%20integrated%20software%20package%20for%20population%20genetics%20data%20analysis&author=L%20Excoffier&author=G%20Laval&author=S%20Schneider&volume=1&publication_year=2007&pages=47-50&pmid=19325852&)

20. Zaccaria GM, Ferrero S, Rosati S, et al. Applying data warehousing to a phase III clinical trial from the Fondazione Italiana Linfomi ensures superior data quality and improved assessment of clinical outcomes. JCO Clin Cancer Inform. 2019;3:1–15. doi: 10.1200/CCI.19.00049.  [DOI](https://doi.org/10.1200/CCI.19.00049) | [PMC free article](/articles/PMC6873907/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31633999/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JCO%20Clin%20Cancer%20Inform&title=Applying%20data%20warehousing%20to%20a%20phase%20III%20clinical%20trial%20from%20the%20Fondazione%20Italiana%20Linfomi%20ensures%20superior%20data%20quality%20and%20improved%20assessment%20of%20clinical%20outcomes&author=GM%20Zaccaria&author=S%20Ferrero&author=S%20Rosati&volume=3&publication_year=2019&pages=1-15&pmid=31633999&doi=10.1200/CCI.19.00049&)

21. Cheson BD, Pfistner B, Juweid ME, et al. Revised response criteria for malignant lymphoma. J Clin Oncol. 2007;25(5):579–586. doi: 10.1200/JCO.2006.09.2403.  [DOI](https://doi.org/10.1200/JCO.2006.09.2403) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17242396/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%C2%A0Clin%20Oncol&title=Revised%20response%20criteria%20for%20malignant%20lymphoma&author=BD%20Cheson&author=B%20Pfistner&author=ME%20Juweid&volume=25&issue=5&publication_year=2007&pages=579-586&pmid=17242396&doi=10.1200/JCO.2006.09.2403&)

22. Chamberlain PP, Lopez-Girona A, Miller K, et al. Structure of the human cereblon-DDB1-lenalidomide complex reveals basis for responsiveness to thalidomide analogs. Nat Struct Mol Biol. 2014;21(9):803–809. doi: 10.1038/nsmb.2874.  [DOI](https://doi.org/10.1038/nsmb.2874) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25108355/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Struct%20Mol%20Biol&title=Structure%20of%20the%20human%20cereblon-DDB1-lenalidomide%20complex%20reveals%20basis%20for%20responsiveness%20to%20thalidomide%20analogs&author=PP%20Chamberlain&author=A%20Lopez-Girona&author=K%20Miller&volume=21&issue=9&publication_year=2014&pages=803-809&pmid=25108355&doi=10.1038/nsmb.2874&)

23. Rossi D, Rasi S, Franceschetti S, et al. Analysis of the host pharmacogenetic background for prediction of outcome and toxicity in diffuse large B-cell lymphoma treated with R-CHOP21. Leukemia. 2009;23(6):1118–1126. doi: 10.1038/leu.2008.398.  [DOI](https://doi.org/10.1038/leu.2008.398) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19448608/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Leukemia&title=Analysis%20of%20the%20host%20pharmacogenetic%20background%20for%20prediction%20of%20outcome%20and%20toxicity%20in%20diffuse%20large%20B-cell%20lymphoma%20treated%20with%20R-CHOP21&author=D%20Rossi&author=S%20Rasi&author=S%20Franceschetti&volume=23&issue=6&publication_year=2009&pages=1118-1126&pmid=19448608&doi=10.1038/leu.2008.398&)

24. Rossi D, Bruscaggin A, La Cava P, et al. The genotype of MLH1 identifies a subgroup of follicular lymphoma patients who do not benefit from doxorubicin: FIL-FOLL study. Haematologica. 2015;100(4):517–524. doi: 10.3324/haematol.2014.108183.  [DOI](https://doi.org/10.3324/haematol.2014.108183) | [PMC free article](/articles/PMC4380725/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25596266/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Haematologica&title=The%20genotype%20of%20MLH1%20identifies%20a%20subgroup%20of%20follicular%20lymphoma%20patients%20who%20do%20not%20benefit%20from%20doxorubicin:%20FIL-FOLL%20study&author=D%20Rossi&author=A%20Bruscaggin&author=P%20La%20Cava&volume=100&issue=4&publication_year=2015&pages=517-524&pmid=25596266&doi=10.3324/haematol.2014.108183&)

25. Galimberti S, Palumbo GA, Caracciolo F, et al. The efficacy of rituximab plus Hyper-CVAD regimen in mantle cell lymphoma is independent of FCgammaRIIIa and FCgammaRIIa polymorphisms. J Chemother. 2007;19(3):315–321. doi: 10.1179/joc.2007.19.3.315.  [DOI](https://doi.org/10.1179/joc.2007.19.3.315) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17594928/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%C2%A0Chemother&title=The%20efficacy%20of%20rituximab%20plus%20Hyper-CVAD%20regimen%20in%20mantle%20cell%20lymphoma%20is%20independent%20of%20FCgammaRIIIa%20and%20FCgammaRIIa%20polymorphisms&author=S%20Galimberti&author=GA%20Palumbo&author=F%20Caracciolo&volume=19&issue=3&publication_year=2007&pages=315-321&pmid=17594928&doi=10.1179/joc.2007.19.3.315&)

26. Galimberti S, Nagy B, Ciancia E, et al. Outcome of patients with mantle cell lymphoma is not influenced by vascular endothelial growth factor polymorphisms. Leuk Lymphoma. 2011;52(1):142–144. doi: 10.3109/10428194.2010.527405.  [DOI](https://doi.org/10.3109/10428194.2010.527405) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21067446/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Leuk%20Lymphoma&title=Outcome%20of%20patients%20with%20mantle%20cell%20lymphoma%20is%20not%20influenced%20by%20vascular%20endothelial%20growth%20factor%20polymorphisms&author=S%20Galimberti&author=B%20Nagy&author=E%20Ciancia&volume=52&issue=1&publication_year=2011&pages=142-144&pmid=21067446&doi=10.3109/10428194.2010.527405&)

27. Cencini E, Sicuranza A, Fabbri A, et al. Study of gene polymorphisms as predictors of treatment efficacy and toxicity in patients with indolent non-hodgkin lymphomas and mantle cell lymphoma receiving bendamustine and rituximab. Br J Haematol. 2019;184(2):223–231. doi: 10.1111/bjh.15582.  [DOI](https://doi.org/10.1111/bjh.15582) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30203425/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Haematol&title=Study%20of%20gene%20polymorphisms%20as%20predictors%20of%20treatment%20efficacy%20and%20toxicity%20in%20patients%20with%20indolent%20non-hodgkin%20lymphomas%20and%20mantle%20cell%20lymphoma%20receiving%20bendamustine%20and%20rituximab&author=E%20Cencini&author=A%20Sicuranza&author=A%20Fabbri&volume=184&issue=2&publication_year=2019&pages=223-231&pmid=30203425&doi=10.1111/bjh.15582&)

28. Assis-Mendonça GR, Fattori A, Rocha RM, et al. Single nucleotide variants in immune-response genes and the tumor microenvironment composition predict progression of mantle cell lymphoma. BMC Cancer. 2021;21(1):209. doi: 10.1186/s12885-021-07891-9.  [DOI](https://doi.org/10.1186/s12885-021-07891-9) | [PMC free article](/articles/PMC7919095/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33648463/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Cancer&title=Single%20nucleotide%20variants%20in%20immune-response%20genes%20and%20the%20tumor%20microenvironment%20composition%20predict%20progression%20of%20mantle%20cell%20lymphoma&author=GR%20Assis-Mendon%C3%A7a&author=A%20Fattori&author=RM%20Rocha&volume=21&issue=1&publication_year=2021&pages=209&pmid=33648463&doi=10.1186/s12885-021-07891-9&)

29. Sardnal V, Rouquette A, Kaltenbach S, et al. A G polymorphism in the CRBN gene acts as a biomarker of response to treatment with lenalidomide in low/int-1 risk MDS without del(5q) Leukemia. 2013;27(7):1610–1613. doi: 10.1038/leu.2013.59.  [DOI](https://doi.org/10.1038/leu.2013.59) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23434730/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Leukemia&title=A%C2%A0G%20polymorphism%20in%20the%20CRBN%20gene%20acts%20as%20a%20biomarker%20of%20response%20to%20treatment%20with%20lenalidomide%20in%20low/int-1%20risk%20MDS%20without%20del(5q)&author=V%20Sardnal&author=A%20Rouquette&author=S%20Kaltenbach&volume=27&issue=7&publication_year=2013&pages=1610-1613&pmid=23434730&doi=10.1038/leu.2013.59&)

30. Lopez-Girona A, Mendy D, Ito T, et al. Cereblon is a direct protein target for immunomodulatory and antiproliferative activities of lenalidomide and pomalidomide. Leukemia. 2012;26(11):2326–2335. doi: 10.1038/leu.2012.119.  [DOI](https://doi.org/10.1038/leu.2012.119) | [PMC free article](/articles/PMC3496085/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22552008/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Leukemia&title=Cereblon%20is%20a%20direct%20protein%20target%20for%20immunomodulatory%20and%20antiproliferative%20activities%20of%20lenalidomide%20and%20pomalidomide&author=A%20Lopez-Girona&author=D%20Mendy&author=T%20Ito&volume=26&issue=11&publication_year=2012&pages=2326-2335&pmid=22552008&doi=10.1038/leu.2012.119&)

31. Zhu YX, Braggio E, Shi CX, et al. Cereblon expression is required for the anti-myeloma activity of lenalidomide and pomalidomide. Blood. 2011;118(18):4771–4779. doi: 10.1182/blood-2011-05-356063.  [DOI](https://doi.org/10.1182/blood-2011-05-356063) | [PMC free article](/articles/PMC3208291/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21860026/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Cereblon%20expression%20is%20required%20for%20the%20anti-myeloma%20activity%20of%20lenalidomide%20and%20pomalidomide&author=YX%20Zhu&author=E%20Braggio&author=CX%20Shi&volume=118&issue=18&publication_year=2011&pages=4771-4779&pmid=21860026&doi=10.1182/blood-2011-05-356063&)

32. Hoffmeyer S, Burk O, von Richter O, et al. Functional polymorphisms of the human multidrug-resistance gene: multiple sequence variations and correlation of one allele with P-glycoprotein expression and activity in vivo. Proc Natl Acad Sci U S A. 2000;97(7):3473–3478. doi: 10.1073/pnas.050585397.  [DOI](https://doi.org/10.1073/pnas.050585397) | [PMC free article](/articles/PMC16264/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10716719/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&title=Functional%20polymorphisms%20of%20the%20human%20multidrug-resistance%20gene:%20multiple%20sequence%20variations%20and%20correlation%20of%20one%20allele%20with%20P-glycoprotein%20expression%20and%20activity%20in%C2%A0vivo&author=S%20Hoffmeyer&author=O%20Burk&author=O%20von%20Richter&volume=97&issue=7&publication_year=2000&pages=3473-3478&pmid=10716719&doi=10.1073/pnas.050585397&)

33. Johne A, Köpke K, Gerloff T, et al. Modulation of steady-state kinetics of digoxin by haplotypes of the P-glycoprotein MDR1 gene. Clin Pharmacol Ther. 2002;72(5):584–594. doi: 10.1067/mcp.2002.129196.  [DOI](https://doi.org/10.1067/mcp.2002.129196) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12426522/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Modulation%20of%20steady-state%20kinetics%20of%20digoxin%20by%20haplotypes%20of%20the%20P-glycoprotein%20MDR1%20gene&author=A%20Johne&author=K%20K%C3%B6pke&author=T%20Gerloff&volume=72&issue=5&publication_year=2002&pages=584-594&pmid=12426522&doi=10.1067/mcp.2002.129196&)

34. Verstuyft C, Schwab M, Schaeffeler E, et al. Digoxin pharmacokinetics and MDR1 genetic polymorphisms. Eur J Clin Pharmacol. 2003;58(12):809–812. doi: 10.1007/s00228-003-0567-5.  [DOI](https://doi.org/10.1007/s00228-003-0567-5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12698307/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Clin%20Pharmacol&title=Digoxin%20pharmacokinetics%20and%20MDR1%20genetic%20polymorphisms&author=C%20Verstuyft&author=M%20Schwab&author=E%20Schaeffeler&volume=58&issue=12&publication_year=2003&pages=809-812&pmid=12698307&doi=10.1007/s00228-003-0567-5&)

35. Kobayashi T, Miura M, Niioka T, et al. Phase II clinical trial of lenalidomide and dexamethasone therapy in japanese elderly patients with newly diagnosed multiple myeloma to determine optimal plasma concentration of lenalidomide. Ther Drug Monit. 2018;40(3):301–309. doi: 10.1097/FTD.0000000000000499.  [DOI](https://doi.org/10.1097/FTD.0000000000000499) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29494421/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ther%20Drug%20Monit&title=Phase%20II%20clinical%20trial%20of%20lenalidomide%20and%20dexamethasone%20therapy%20in%20japanese%20elderly%20patients%20with%20newly%20diagnosed%20multiple%20myeloma%20to%20determine%20optimal%20plasma%20concentration%20of%20lenalidomide&author=T%20Kobayashi&author=M%20Miura&author=T%20Niioka&volume=40&issue=3&publication_year=2018&pages=301-309&pmid=29494421&doi=10.1097/FTD.0000000000000499&)

36. Kobayashi T, Miura M, Abumiya M, Akamine Y, Ito F, Takahashi N. Influence of ABCB1 polymorphisms on the pharmacokinetics and toxicity of lenalidomide in patients with multiple myeloma. Med Oncol. 2019;36(6):55. doi: 10.1007/s12032-019-1280-2.  [DOI](https://doi.org/10.1007/s12032-019-1280-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31089832/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Med%20Oncol&title=Influence%20of%20ABCB1%20polymorphisms%20on%20the%20pharmacokinetics%20and%20toxicity%20of%20lenalidomide%20in%20patients%20with%20multiple%20myeloma&author=T%20Kobayashi&author=M%20Miura&author=M%20Abumiya&author=Y%20Akamine&author=F%20Ito&volume=36&issue=6&publication_year=2019&pages=55&pmid=31089832&doi=10.1007/s12032-019-1280-2&)

37. Steffensen KD, Waldstrøm M, Brandslund I, Jakobsen A. The relationship of VEGF polymorphisms with serum VEGF levels and progression-free survival in patients with epithelial ovarian cancer. Gynecol Oncol. 2010;117(1):109–116. doi: 10.1016/j.ygyno.2009.11.011.  [DOI](https://doi.org/10.1016/j.ygyno.2009.11.011) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20056267/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gynecol%20Oncol&title=The%20relationship%20of%20VEGF%20polymorphisms%20with%20serum%20VEGF%20levels%20and%20progression-free%20survival%20in%20patients%20with%20epithelial%20ovarian%20cancer&author=KD%20Steffensen&author=M%20Waldstr%C3%B8m&author=I%20Brandslund&author=A%20Jakobsen&volume=117&issue=1&publication_year=2010&pages=109-116&pmid=20056267&doi=10.1016/j.ygyno.2009.11.011&)

38. Skrypnik D, Mostowska A, Jagodziński PP, Bogdański P. Association of rs699947 (-2578 C/A) and rs2010963 (-634 G/C) single nucleotide polymorphisms of the VEGF Gene, VEGF-A and leptin serum level, and cardiovascular risk in patients with excess body mass: a case-control study. J Clin Med. 2020;9(2):469. doi: 10.3390/jcm9020469.  [DOI](https://doi.org/10.3390/jcm9020469) | [PMC free article](/articles/PMC7073834/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32046348/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%C2%A0Clin%20Med&title=Association%20of%20rs699947%20(-2578%20C/A)%20and%20rs2010963%20(-634%20G/C)%20single%20nucleotide%20polymorphisms%20of%20the%20VEGF%20Gene,%20VEGF-A%20and%20leptin%20serum%20level,%20and%20cardiovascular%20risk%20in%20patients%20with%20excess%20body%20mass:%20a%20case-control%20study&author=D%20Skrypnik&author=A%20Mostowska&author=PP%20Jagodzi%C5%84ski&author=P%20Bogda%C5%84ski&volume=9&issue=2&publication_year=2020&pages=469&pmid=32046348&doi=10.3390/jcm9020469&)

39. Bhaskari J, Premalata CS, Shilpa V, et al. Vascular endothelial growth factor polymorphisms and a synchronized examination of plasma and tissue expression in epithelial ovarian cancers. Tumour Biol. 2016;37(1):1017–1023. doi: 10.1007/s13277-015-3891-3.  [DOI](https://doi.org/10.1007/s13277-015-3891-3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26264619/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Tumour%20Biol&title=Vascular%20endothelial%20growth%20factor%20polymorphisms%20and%20a%20synchronized%20examination%20of%20plasma%20and%20tissue%20expression%20in%20epithelial%20ovarian%20cancers&author=J%20Bhaskari&author=CS%20Premalata&author=V%20Shilpa&volume=37&issue=1&publication_year=2016&pages=1017-1023&pmid=26264619&doi=10.1007/s13277-015-3891-3&)

40. Andersen NF, Vogel U, Klausen TW, et al. Vascular endothelial growth factor (VEGF) gene polymorphisms may influence the efficacy of thalidomide in multiple myeloma. Int J Cancer. 2012;131(5):E636–642. doi: 10.1002/ijc.27387.  [DOI](https://doi.org/10.1002/ijc.27387) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22139971/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Cancer&title=Vascular%20endothelial%20growth%20factor%20(VEGF)%20gene%20polymorphisms%20may%20influence%20the%20efficacy%20of%20thalidomide%20in%20multiple%20myeloma&author=NF%20Andersen&author=U%20Vogel&author=TW%20Klausen&volume=131&issue=5&publication_year=2012&pages=E636-642&pmid=22139971&doi=10.1002/ijc.27387&)

41. Bomben R, Ferrero S, D'Agaro T, et al. B-cell receptor-related gene signature predicts survival in mantle cell lymphoma: results from the Fondazione Italiana Linfomi MCL-0208 trial. Haematologica. 2018;103(5):849–856. doi: 10.3324/haematol.2017.184325.  [DOI](https://doi.org/10.3324/haematol.2017.184325) | [PMC free article](/articles/PMC5927985/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29472356/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Haematologica&title=B-cell%20receptor-related%20gene%20signature%20predicts%20survival%20in%20mantle%20cell%20lymphoma:%20results%20from%20the%20Fondazione%20Italiana%20Linfomi%20MCL-0208%20trial&author=R%20Bomben&author=S%20Ferrero&author=T%20D'Agaro&volume=103&issue=5&publication_year=2018&pages=849-856&pmid=29472356&doi=10.3324/haematol.2017.184325&)

42. Ribrag V, Safar V, Kluin-Nelemans H, et al. Rituximab-lenalidomide(R2) maintenance is superior to rituximab maintenance after first line immunochemotherapy in mantle cell lymphoma: results of the MCL R2 elderly clinical trial. Blood. 2021;138(1):379. ISSN 0006-4971.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Rituximab-lenalidomide(R2)%20maintenance%20is%20superior%20to%20rituximab%20maintenance%20after%20first%20line%20immunochemotherapy%20in%20mantle%20cell%20lymphoma:%20results%20of%20the%20MCL%20R2%20elderly%20clinical%20trial&author=V%20Ribrag&author=V%20Safar&author=H%20Kluin-Nelemans&volume=138&issue=1&publication_year=2021&pages=379&)

43. Holt CL, Hussain A, Wachbroit R, Scott J. Precision medicine across the cancer continuum: implementation and implications for cancer disparities. JCO Precis Oncol. 2017;1:1–5. doi: 10.1200/PO.17.00102.  [DOI](https://doi.org/10.1200/PO.17.00102) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35172506/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JCO%20Precis%20Oncol&title=Precision%20medicine%20across%20the%20cancer%20continuum:%20implementation%20and%20implications%20for%20cancer%20disparities&author=CL%20Holt&author=A%20Hussain&author=R%20Wachbroit&author=J%20Scott&volume=1&publication_year=2017&pages=1-5&pmid=35172506&doi=10.1200/PO.17.00102&)

44. Wang ML, Rule S, Martin P, et al. Targeting BTK with ibrutinib in relapsed or refractory mantle-cell lymphoma. N Engl J Med. 2013;369(6):507–516. doi: 10.1056/NEJMoa1306220.  [DOI](https://doi.org/10.1056/NEJMoa1306220) | [PMC free article](/articles/PMC4513941/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23782157/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%C2%A0Engl%20J%20Med&title=Targeting%20BTK%20with%20ibrutinib%20in%20relapsed%20or%20refractory%20mantle-cell%20lymphoma&author=ML%20Wang&author=S%20Rule&author=P%20Martin&volume=369&issue=6&publication_year=2013&pages=507-516&pmid=23782157&doi=10.1056/NEJMoa1306220&)

45. Wang M, Rule S, Zinzani PL, et al. Acalabrutinib in relapsed or refractory mantle cell lymphoma (ACE-LY-004): a single-arm, multicentre, phase 2 trial. Lancet. 2018;391(10121):659–667. doi: 10.1016/S0140-6736(17)33108-2.  [DOI](https://doi.org/10.1016/S0140-6736(17)33108-2) | [PMC free article](/articles/PMC7864374/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29241979/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Acalabrutinib%20in%20relapsed%20or%20refractory%20mantle%20cell%20lymphoma%20(ACE-LY-004):%20a%20single-arm,%20multicentre,%20phase%202%20trial&author=M%20Wang&author=S%20Rule&author=PL%20Zinzani&volume=391&issue=10121&publication_year=2018&pages=659-667&pmid=29241979&doi=10.1016/S0140-6736(17)33108-2&)

46. Nowakowski GS, Chiappella A, Gascoyne RD, et al. ROBUST: a phase III study of lenalidomide plus R-CHOP versus placebo plus R-CHOP in previously untreated patients with ABC-type diffuse large B-cell lymphoma. J Clin Oncol. 2021;39(12):1317–1328. doi: 10.1200/JCO.20.01366.  [DOI](https://doi.org/10.1200/JCO.20.01366) | [PMC free article](/articles/PMC8078325/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33621109/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%C2%A0Clin%20Oncol&title=ROBUST:%20a%20phase%20III%20study%20of%20lenalidomide%20plus%20R-CHOP%20versus%20placebo%20plus%20R-CHOP%20in%20previously%20untreated%20patients%20with%20ABC-type%20diffuse%20large%20B-cell%20lymphoma&author=GS%20Nowakowski&author=A%20Chiappella&author=RD%20Gascoyne&volume=39&issue=12&publication_year=2021&pages=1317-1328&pmid=33621109&doi=10.1200/JCO.20.01366&)

47. Quach H, Ritchie D, Stewart AK, et al. Mechanism of action of immunomodulatory drugs (IMiDS) in multiple myeloma. Leukemia. 2010;24(1):22–32. doi: 10.1038/leu.2009.236.  [DOI](https://doi.org/10.1038/leu.2009.236) | [PMC free article](/articles/PMC3922408/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19907437/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Leukemia&title=Mechanism%20of%20action%20of%20immunomodulatory%20drugs%20(IMiDS)%20in%20multiple%20myeloma&author=H%20Quach&author=D%20Ritchie&author=AK%20Stewart&volume=24&issue=1&publication_year=2010&pages=22-32&pmid=19907437&doi=10.1038/leu.2009.236&)

48. Voorhees PM, Kaufman JL, Laubach J, et al. Daratumumab, lenalidomide, bortezomib, and dexamethasone for transplant-eligible newly diagnosed multiple myeloma: the GRIFFIN trial. Blood. 2020;136(8):936–945. doi: 10.1182/blood.2020005288.  [DOI](https://doi.org/10.1182/blood.2020005288) | [PMC free article](/articles/PMC7441167/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32325490/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Daratumumab,%20lenalidomide,%20bortezomib,%20and%20dexamethasone%20for%20transplant-eligible%20newly%20diagnosed%20multiple%20myeloma:%20the%20GRIFFIN%20trial&author=PM%20Voorhees&author=JL%20Kaufman&author=J%20Laubach&volume=136&issue=8&publication_year=2020&pages=936-945&pmid=32325490&doi=10.1182/blood.2020005288&)
