# SLCO1B1 Exome Sequencing and Statin Treatment Response in 64,000 UK Biobank Patients

## Metadata
**Authors:** Deniz Türkmen, Jack Bowden, Jane A H Masoli, David Melzer, Luke C Pilling
**Journal:** International Journal of Molecular Sciences
**Date:** 2024 Apr 17
**DOI:** [10.3390/ijms25084426](https://doi.org/10.3390/ijms25084426)
**PMID:** 38674010
**PMCID:** PMC11050003
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11050003/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC11050003/pdf/ijms-25-04426.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC11050003/pdf/ijms-25-04426.pdf)

## Abstract

The solute carrier organic anion transporter family member 1B1 (SLCO1B1) encodes the organic anion-transporting polypeptide 1B1 (OATP1B1 protein) that transports statins to liver cells. Common genetic variants in SLCO1B1, such as *5, cause altered systemic exposure to statins and therefore affect statin outcomes, with potential pharmacogenetic applications; yet, evidence is inconclusive. We studied common and rare SLCO1B1 variants in up to 64,000 patients from UK Biobank prescribed simvastatin or atorvastatin, combining whole-exome sequencing data with up to 25-year routine clinical records. We studied 51 predicted gain/loss-of-function variants affecting OATP1B1. Both SLCO1B1*5 alone and the SLCO1B1*15 haplotype increased LDL during treatment (beta*5 = 0.08 mmol/L, p = 6 × 10−8; beta*15 = 0.03 mmol/L, p = 3 × 10−4), as did the likelihood of discontinuing statin prescriptions (hazard ratio*5 = 1.12, p = 0.04; HR*15 = 1.05, p = 0.04). SLCO1B1*15 and SLCO1B1*20 increased the risk of General Practice (GP)-diagnosed muscle symptoms (HR*15 = 1.22, p = 0.003; HR*20 = 1.25, p = 0.01). We estimated that genotype-guided prescribing could potentially prevent 18% and 10% of GP-diagnosed muscle symptoms experienced by statin patients, with *15 and *20, respectively. The remaining common variants were not individually significant. Rare variants in SLCO1B1 increased LDL in statin users by up to 1.05 mmol/L, but replication is needed. We conclude that genotype-guided treatment could reduce GP-diagnosed muscle symptoms in statin patients; incorporating further SLCO1B1 variants into clinical prediction scores could improve LDL control and decrease adverse events, including discontinuation.

Keywords: cholesterol, statin, SLCO1B1, clinical response, pharmacogenomics, exome sequencing variants, epidemiology

## 1. Introduction

*SLCO1B1* (solute organic anion transporter family member 1B) encodes the ‘OATP1B1’ protein (the organic anion-transporting polypeptide 1B1) transporter to liver cells for bilirubin and statins [[1](#B1-ijms-25-04426)]. *SLCO1B1* is therefore classified as ‘very important’ on the pharmacogenetics review site PharmGKB [[2](#B2-ijms-25-04426)]. The common variants *SLCO1B1**5 (rs4149056:Val174Ala) and *SLCO1B1**1B (rs2306283:Asn130Asp—now designated *SLCO1B1**37) have European allele frequencies of ~2% and 40%, with frequencies in other populations varying from <1% to 75% [[3](#B3-ijms-25-04426)]. These variants affect statin pharmacokinetics [[1](#B1-ijms-25-04426)], with *5 linked to increased adverse events [[4](#B4-ijms-25-04426)]. However, some studies report increased statin-associated muscle symptoms (SAMS) for *SLCO1B1**5 [[5](#B5-ijms-25-04426),[6](#B6-ijms-25-04426),[7](#B7-ijms-25-04426),[8](#B8-ijms-25-04426),[9](#B9-ijms-25-04426)], but others do not [[10](#B10-ijms-25-04426),[11](#B11-ijms-25-04426),[12](#B12-ijms-25-04426)] and have found inconclusive trends [[13](#B13-ijms-25-04426)]. The *SLCO1B1* genotype is reported to increase SAMS risk in patients prescribed simvastatin or atorvastatin, but has a smaller effect on pravastatin, rosuvastatin, and fluvastatin [[14](#B14-ijms-25-04426),[15](#B15-ijms-25-04426),[16](#B16-ijms-25-04426)]. There are also conflicting reports on haplotype *15 (allele frequency 3–24%) [[3](#B3-ijms-25-04426)] and increasing SAMS [[17](#B17-ijms-25-04426)].

Common *SLCO1B1* “gain-of-function” (GoF) variants (rs11045819:Pro155Thr and rs34671512:Leu643Phe) increase simvastatin bioavailability [[18](#B18-ijms-25-04426)], but their effects on the statin response remain unclear. A recent study in 15,378 statin users combined four common *SLCO1B1* variants into a gene risk score; high-risk patients had increased odds of statin intolerance, and accounting for GoF variants improved prediction [[19](#B19-ijms-25-04426)].

Studies [[20](#B20-ijms-25-04426),[21](#B21-ijms-25-04426),[22](#B22-ijms-25-04426)] also identified rare *SLCO1B1* variants. A study of 699 patients using exome sequencing found that rare variants explained 17.8% of *SLCO1B1* variability [[23](#B23-ijms-25-04426)], and ~9% of *SLCO1B1* variants are deleterious rare variants [[20](#B20-ijms-25-04426),[22](#B22-ijms-25-04426)]. However, the impact on the statin treatment response is unknown due to the widely used genotyping technologies (microarray, imputed) and small sample sizes with exome data.

We aimed to estimate the associations between common and rare *SLCO1B1* variants with statin-related outcomes using data from 64,000 atorvastatin and simvastatin patients in UK Biobank with whole-exome sequencing (WES) data linked to primary and secondary care records. The outcomes were low-density lipoprotein (LDL) levels, General Practice-diagnosed muscle symptoms and treatment discontinuation, and hospital-diagnosed coronary heart disease and stroke. Other biomarkers investigated related to *SLCO1B1* from the GWAS (Genome-Wide Association Study) Catalog included vitamin D (risk factor for muscle weakness [[24](#B24-ijms-25-04426)]) and bilirubin (possible risk factor for increased stroke [[25](#B25-ijms-25-04426),[26](#B26-ijms-25-04426)]).

## 2. Results

A total of 67,630 participants self-reported simvastatin or atorvastatin use at the baseline assessment ([Figure 1](#ijms-25-04426-f001)); the mean age was 61.5 years, and 38.7% were female ([Table 1](#ijms-25-04426-t001)). A total of 65,513 participants were prescribed simvastatin or atorvastatin by their GP; the mean age at first prescription was 61.0 years, and 41.7% were female. See the [Supplementary Materials](#app1-ijms-25-04426) for details on the overlap.

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2f45/11050003/83710760e010/ijms-25-04426-g001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=11050003_ijms-25-04426-g001.jpg)

Cohort flowchart. CHD: coronary heart disease; GP: general practitioner. We conducted baseline analyses for common variants, including haplotypes, and rare variants. Due to the small number of events in rare-variant-carrier patients, we conducted primary care analyses in patients carrying common variants. * refers the variants described with star (e.g., *5).

### Table 1.

|   |   | Baseline Data |
| --- | --- | --- |
| Sex | N Female | 26,185 (38.7) |
| Age | Mean (SD) | 61.5 (6) |
|   | min-max | 40–70 |
|   | Median | 63 |
| Simvastatin | N (%) | 51,147 (75.5) |
| Weight | Mean (SD) | 84.3 (16.5) |
| BMI | Mean (SD) | 29.4 (5) |
| LDL | Mean (SD) | 2.75 (0.67) |
| LDL, n > 2.86 | N (%) | 28,450 (42) |
|   | N female (%) | 12,622(44.3) |
| Total cholesterol | Mean (SD) | 4.6 (0.9) |
| Triglycerides | Mean (SD) | 1.9 (1.1) |
| Direct Bilirubin | Mean (SD) | 9.5 (4.5) |
| ALT | Mean (SD) | 27.1 (14.5) |
| CRP | Mean (SD) | 2.8 (4.7) |
| Cystatin c | Mean (SD) | 0.9 (0.2) |
| Vitamin D | Mean (SD) | 49.7 (21.6) |
| HbA1c | Mean (SD) | 40.5 (10.2) |
|   | Primary care data |  |
| Sex | N Female | 27,299 (42) |
| Duration on statins^ (year) | Mean (SD) | 6.7 (4.7) |
|   | min-max | 1 to 25 |
| Number of prescriptions per year | Mean (SD) | 9 (5) |
| Muscle diagnoses 1 prior to statin ^ | N (%) | 1308 (2) |
| MI/angina diagnoses 2 prior to statin ^ | N (%) | 6718 (10.4) |
| Stroke diagnoses 2 prior to statin ^ | N (%) | 855 (1.3) |
| Muscle diagnoses 1 after first statin ^ | N (%) | 2242 (3.5) |
| MI/angina diagnoses 2 after first statin ^ | N (%) | 14,751 (22.8) |
| Stroke diagnoses 2 after first statin ^ | N (%) | 1629 (2.5) |
| Discontinuation ever | N (%) | 13,889 (22.4) |

Table 1 Caption: Characteristics of UK Biobank participants.

### 2.1. Common SLCO1B1 Variants

#### Baseline Self-Reported Statin Use and Biomarkers

**LDL cholesterol**: The mean LDL-c level was 2.75 mmol/L (SD 0.66) (2.86 mmol/L (SD 0.67) in women and 2.62 mmol/L (SD 0.65) in men. In the multivariable analysis, two variants were associated with higher LDL levels versus non-carriers: *SLCO1B1**5 (Coef: 0.08, 95%CI: 0.05 to 0.12, *p* = 6 × 10^−6^) and *SLCO1B1**15 (coefficient: 0.03, 95%CI: 0.01 to 0.04, *p* = 3 × 10^−4^) ([Figure 2](#ijms-25-04426-f002), [Table 2](#ijms-25-04426-t002)). *SLCO1B1**14 carriers (including rs11045819 and rs2306283) had a trend of a reduction in LDL levels (Coef: −0.01, *p* = 0.05).

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2f45/11050003/6bba35fdb82c/ijms-25-04426-g002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=11050003_ijms-25-04426-g002.jpg)

Associations between common variants, including haplotypes and statin-related outcomes. (A) Associations in UK Biobank baseline assessment data. Coefficient = difference in LDL (mmol/L), total cholesterol (mmol/L), and triglycerides (mmol/L) between analysis group and reference group. First four variants are common ‘*’ variants, *45 is a known ‘*’ variant, and the remaining variants are haplotypes. Models were adjusted for sex, age at baseline assessment, and genetic principal components. (B) Associations in UK Biobank primary care data. Muscle symptoms refer GP-diagnosed muscle symptoms, including myopathy, myalgia, myositis, or rhabdomyolysis. The table shows the results for patients who had been on statin prescriptions for at least 5 years. Discontinuation means patients had their last prescription at least 3 months prior to the censoring date (removal from GP list) or by 28 February 2016, if no deduction date was available. CHD: hospital inpatient coronary heart disease diagnosis. Models were adjusted for sex, age at first prescription, and genetic principal components.

| Allele |   |   |   |   |   |   |
| --- | --- | --- | --- | --- | --- | --- |
| Common variants | N | % | Coef | 95% | CI | p |
| *4-CC | 45,798 | 75.58 | ref |   |   |   |
| *4-CA | 13,324 | 21.99 | −0.02 | −0.03 | −0.009 | 8 × 10−4 |
| *4-AA | 1475 | 2.43 | −0.03 | −0.067 | −0.0008 | 0.045 |
| *5-TT | 44,821 | 72.47 | ref |   |   |   |
| *5-TC | 15,640 | 25.29 | 0.02 | 0.01 | 0.03 | 4 × 10−4 |
| *5-CC | 1383 | 2.24 | 0.08 | 0.05 | 0.12 | 6 × 10−6 |
| *37-AA | 24,413 | 40.03 | ref |   |   |   |
| *37-AG | 27,340 | 44.83 | 0.01 | −0.01 | 0.02 | 0.32 |
| *37-GG | 9235 | 15.14 | 2 × 10−3 | −0.01 | 0.02 | 0.77 |
| *19-AA | 55,469 | 89.69 | ref |   |   |   |
| *19-AC | 6223 | 10.06 | −0.01 | −0.03 | 0.01 | 0.18 |
| *19-CC | 154 | 0.25 | −0.01 | −0.12 | 0.09 | 0.83 |
| Haplotypes ^ | N | % | Coef | 95% | CI | p |
| No *14 | 23,870 | 39.91 | ref |   |   |   |
| *14 | 14,181 | 23.71 | −0.01 | −0.03 | 0.00 | 0.05 |
| No *15 | 22,015 | 36.13 | ref |   |   |   |
| *15 | 14,454 | 23.72 | 0.03 | 0.01 | 0.04 | 3 × 10-4 |
| No *20 | 24,254 | 39.80 | ref |   |   |   |
| *20 | 6173 | 10.13 | −0.01 | −0.03 | 0.01 | 0.42 |
| No *45 | 60,852 | 99.89 | ref |   |   |   |
| *45 ° | 67 | 0.11 | −0.01 | −0.17 | 0.15 | 0.90 |
| No *46 | 21,707 | 36.2 | ref |   |   |   |
| *46 | 65 | 0.1 | 0.002 | −0.01 | 0.01 | 0.70 |

Table Caption: Baseline LDL levels associations with star and common variants.

**Other biomarkers**: While *SLCO1B1**5 and *15 had decreased vitamin D levels (Coef: −2.73, 95%CI: −3.89 to −1.59, *p* = 3 × 10^−6^ and Coef: −0.56, 95%CI: −1 to −0.1, *p* = 0.02, respectively), *SLCO1B1**14 and *20 had increased vitamin D levels vs. non-carriers (Coef:0.78, 95%CI: 0.33 to 1.22, *p* = 7 × 10^−7^ and Coef: 0.67, 95%CI: 0.07 to 1.27, *p* = 0.03, respectively).

*SLCO1B1**4, *SLCO1B1**19, and *SLCO1B1**20 were associated with decreased bilirubin and the remaining variants with increased bilirubin, and *SLCO1B1**5 had the highest risk (Coef: 1.8, 95%CI: 1.6 to 2.97, *p* = 6 × 10^−54^) ([Supplementary Table S1 and File S1](#app1-ijms-25-04426)).

### 2.2. General Practice (GP) Prescribing and Statin-Related Adverse Outcomes

**Muscle symptoms**: In total, 3.4% of participants with GP-prescribed statin data had GP-diagnosed muscle symptoms. Of these, 25% were *SLCO1B1**15 homozygotes and 9.9% were *SLCO1B1**20 homozygotes ([Table 3](#ijms-25-04426-t003)). *SLCO1B1**15 carriers had increased risk of overall muscle symptoms versus non-carriers (HR: 1.16, 95%CI: 1.04 to 1.3, *p* = 0.01), of which carriers on statin for ≥5 years had a risk of 1.22 (95%CI: 1.07 to 1.39, *p* = 0.003). With 5+ years of statin prescription, *SLCO1B1**20 carriers had increased risk (HR: 1.25, 95%CI: 1.05 to 1.48, *p* = 0.01) vs. non-carriers ([Figure 1](#ijms-25-04426-f001), [Supplementary Table S2](#app1-ijms-25-04426)). Using TWIST [[27](#B27-ijms-25-04426)], we found that the RGMTE (robust genetically moderated treatment effect) and MR estimates were sufficiently similar to combine (Q-statistic *p* > 0.05) into a more precise estimate of GMTE: for *SLCO1B1**15, the per-year risk difference was 0.08% (*p* = 8 × 10^−3^). To illustrate this effect, we multiplied by the 80,685 *SLCO1B1**15 carrier patient years in the data; if we could reduce the *SLCO1B1**15 carrier’s risk by this amount, then the total cases would reduce by 65 (1457 to 1392), corresponding to a 4.5% reduction in muscle symptoms experienced by statin users, and 17.6% in the *15 carriers alone (369 to 304).

### Table 3.

|   |   | *4 |  | *5 |  | *14 |  | *15 |  | *20 |  | *37 |  | *45 |  | *19 |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
|   |   | % | n | % | n | % | n | % | n | % | n | % | n | % | n | % | n |
| SAMS | wild | 72.36 | 1542 | 67.81 | 1445 | 35.48 | 756 | 33.36 | 711 | 36.23 | 772 | 36.60 | 780 | 94.79 | 2020 | 85.83 | 1829 |
|   | heterozygotes | 20.13 | 429 | 26.09 | 556 | 36.65 | 781 | 36.60 | 780 | 48.71 | 1038 | 43.55 | 928 | 0.05 | 1 | 9.95 | 212 |
|   | homozygotes | 1.88 | 40 | 2.39 | 51 | 21.16 | 451 | 25.06 | 534 | 9.90 | 211 | 14.88 | 317 | 0.00 |   | 0.33 | 7 |
| No SAMS | wild | 71.98 | 44,126 | 69.70 | 42,730 | 37.84 | 23,194 | 34.78 | 21,319 | 38.43 | 23,559 | 38.67 | 23,707 | 94.82 | 58,129 | 86.57 | 53,070 |
|   | heterozygotes | 20.17 | 12,364 | 24.34 | 14,920 | 33.81 | 20,728 | 37.63 | 23,069 | 46.99 | 28,808 | 42.02 | 25,762 | 0.11 | 70 | 9.48 | 5810 |
|   | homozygotes | 2.24 | 1373 | 2.26 | 1388 | 21.48 | 13,169 | 22.45 | 13,763 | 9.45 | 5792 | 14.26 | 8741 | 94.82 |   | 0.28 | 169 |

Table 3 Caption: Percentage of SLCO1B1 genotype carriers with and without SAMS.

For *SLCO1B1**20, the per-year risk difference was 0.05% (*p* = 3.5 × 10^−2^). To illustrate this effect, we multiplied by the 34,076 *SLCO1B1**20 carrier patient years in the data; if we could reduce the *SLCO1B1**20 carrier’s risk by this amount, then the total cases would reduce by 17 (572 to 555), corresponding to a 3% reduction in muscle symptoms experienced by statin users, and 10.3% in the *20 carriers alone (165 to 148).

**Discontinuation**: Patients with *SLCO1B1**5 and *SLCO1B1**15 variants had slightly increased risk of discontinuation of statin therapy (HR: 1.12, 95%CI: 1.01 to 1.25, *p* = 0.04 and HR: 1.05, 95%CI: 1 to 1.1, *p* = 0.04, respectively) versus their non-carrier counterparts ([Figure 2](#ijms-25-04426-f002), [Supplementary Table S2](#app1-ijms-25-04426)). *5 was found to increase risks for women (HR: 1.19, 95%CI: 1.03 to 1.37) but not for men (HR: 1.05, 95%CI: 0.92 to 1.20) [[11](#B11-ijms-25-04426)]. No other variants increased risks in women in this study. While *SLCO1B1**5 was not significant in men, *SLCO1B1**15 increased risk in men (HR: 1.07, 95%CI: 1 to 1.13, *p* = 0.04) versus non-carriers ([Supplementary Table S3](#app1-ijms-25-04426)).

**CHD and ischemic stroke**: During the statin prescription period, 22.8% of patients had myocardial infarction or angina, while 2.5% had ischemic stroke. Carriers of *SLCO1B1**37 had reduced CHD risk versus non-carriers (HR: 0.94, 95%CI: 0.89 to 0.99, *p* = 0.02) (rates: 21.8% versus 23.1%, respectively). Other variants showed negative trends but no significant associations with CHD or stroke. The lowest stroke rates were for *SLCO1B1**5 carriers (2.38%), and the highest rates were for *SLCO1B1**20 carriers (2.8%) ([Supplementary Tables S4 and S5](#app1-ijms-25-04426)).

### 2.3. Statin Gene Risk Score

We calculated the gene scores GRS-1 (“High risk” rs4149056 -*5 CC allele, or CT for *5 and rs2306283-AA (*20), rs11045819-CC (*4), or rs34671512-AA (*19), as reported by Bigossi et al. [[19](#B19-ijms-25-04426)]) and GRS-2 (using rs2306283-GG instead of the AA allele) (see methods for details).

At baseline assessment, in participants self-reporting statin use with LDL measured (n = 61,898), the “high risk” *SLCO1B1* GRS-1 patients (n = 3667) did not have significantly higher LDL (Coef: 0.028 mmol/L: 95%Cis −0.001 to 0.061, *p* = 0.09) compared to the “low risk” GRS-1 group, whereas “high risk” *SLCO1B1* GRS-2 patients (n = 2564) had significantly raised LDL (Coef: 0.062 mmol/L: 95%Cis 0.022 to 0.101, *p* = 0.002) compared to the “low risk” GRS-2 group.

In the analysis of GP-prescribed statins, “high risk” GRS-2 patients had an increased likelihood of developing GP-diagnosed muscle symptoms (HR: 1.13, 95%CI: 1.02 to 1.27, *p* = 0.02), compared to “low risk” patients. This estimate was larger than the estimate in *5 CC homozygotes alone (HR: 1.09, 95%CI: 1.00 to 1.18, *p* = 0.048). Similarly, “high risk” GRS-2 patients had increased rates of discontinuation (HR: 1.07, 95%CI: 1.02 to 1.11, *p* = 0.005) compared to “low risk” patients, with estimates larger than those for *5 C allele carriers (HR: 1.04, 95%CI: 1 to 1.08, *p* = 0.02), but smaller than that for *5 CC homozygotes (HR: 1.12, 95%CI: 1.01 to 1.26, *p* = 0.04), compared to non-carriers.

### 2.4. Rare Variants in SLCO1B1 Identified by Exome Sequencing

We studied 47 rare variants predicted by the VEP ([https://www.ensembl.org/info/docs/tools/vep/index.html](https://www.ensembl.org/info/docs/tools/vep/index.html), accessed on 1 February 2023) tools to be missense or to have other moderate/high-impact consequences for OATP1B1 function, and that were present in at least three participants taking statins at the baseline assessment (n = 61,893). Two of these were associated with increased LDL at baseline in linear regression models adjusted for age, sex, and five principal components (FDR-adjusted *p*-value < 0.05). These rare variants had substantially larger effect sizes in comparison to *SLCO1B1**5, where the per-allele increase in LDL was 0.08 mmol/L, *p* = 6 × 10^−6^: rs374113543 (MAC = 4) increased LDL by 1.05 mmol/L (*p* = 0.001) and rs373327528 (MAC = 9) by 0.7 mmol/L (*p* = 0.003). Two further variants were nominally associated with LDL (FDR *p* > 0.05): rs960742177 (MAC = 7. Coef: 0.7, *p* = 0.009) and rs780911188 (MAC = 4. Coef: 0.65, *p* = 0.047). None of the rare variants were associated with LDL in untreated people in the TWIST analysis [[27](#B27-ijms-25-04426)] (see [Figure 3](#ijms-25-04426-f003) and [Supplementary Tables S6 and S7](#app1-ijms-25-04426) for details). The variants were not in linkage disequilibrium (LD) (R^2^ < 0.01) and were independent in the multivariable analysis.

### Figure 3.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2f45/11050003/26fb7ac350b9/ijms-25-04426-g003.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=11050003_ijms-25-04426-g003.jpg)

TWIST results demonstrating the pharmacogenetic nature of the studied variants. GMTE = genetically moderated treatment effect (Untreated = association between the variant and LDL in individuals not treated with statins at the time of assessment; Treated = association between the variant and LDL in individuals who self-reported statin treatment at the time of assessment; Robust = the ‘treated’ association accounting for any effect in ‘untreated’ individuals). Rare variants: Gly238Glu and Gly71Arg were significantly associated with LDL in treated individuals after multiple testing adjustment (FDR < 0.05, indicated with a ‘star’). The other rare variants shown were only nominally associated. See Supplementary Table S6 for details.

In the combined analysis, 24 participants carried at least one allele of the four LDL-increasing rare variants (45.8% female, mean age: 61.64 [SD 3.5]). Those 24 participants had higher LDL (0.76 mmol/L, 95%CI: 0.48 to 1.03, *p* = 7 × 10^−8^) compared to non-carriers; 45.8% had clinically high LDL (>3.5 mmol/L) compared to 11.2% in non-carriers ([Supplementary Tables S7 and S8](#app1-ijms-25-04426) for the descriptive table of rare variant and SLCO1B1*5 carriers). In the stratified analysis, the effect was substantially greater in men vs. women (Coef: 1.77, 95%CI: 1.21 to 2.33, *p* = 5.6 × 10^−10^; Coef: 0.39, 95%CI: −0.23 to 1, *p* = 0.2). A total of 62.5% of patients carrying the variants reported experiencing pain at the baseline assessment, with the majority being men (66.6%). Male participants carrying the LDL-increasing rare variants had increased risk for experiencing pain while on statins compared to non-carriers (odds ratio (OR) = 1.39, 95%CI: = 1.06 to 1.82, *p* = 0.002). It was not significant for women (OR = 1.02, 95%CI: 0.8 to 1.36, *p* = 0.91).

See [Supplementary Materials](#app1-ijms-25-04426) for details.

See [Supplementary Table S9](#app1-ijms-25-04426) for the other biomarker associations in rare variants.

### 2.5. SLCO1B1 “High Risk” Gene Score plus Rare Variants

In the combined analysis, the number of *SLCO1B1* “high risk” patients increased from 2564 (GRS-2 in the baseline LDL analysis) to 2588 when including those carrying LDL-increasing rare variants. The overall association with LDL increases modestly in this group (compared to GRS-2 associations), due to the small difference in numbers (e.g., GRS-3 LDL coefficient: 0.055, *p* = 3 × 10^−5^; GRS-2 LDL coefficient: 0.049, *p* = 2 × 10^−4^). The associations with other statin outcomes were not notably different when incorporating rare variants into the common variant *SLCO1B1* high-risk gene score.

The numbers of patients prescribed other statins were too limited to analyze. Details on the numbers can be found in [[11](#B11-ijms-25-04426)].

## 3. Discussion

Common *SLCO1B1* variants are reported to affect statin response, but the evidence on clinical outcomes is conflicting. We used whole-exome sequencing data and studied common and rare *SLCO1B1* genetic variation in a large cohort of UK Biobank statin patients to clarify risks in a community cohort. The *SLCO1B1**5 variant alone and *SLCO1B1**15 haplotype (*5 and *37 inherited together) increased LDL at UK Biobank baseline assessment and increased rates of statin prescription discontinuation in the GP data. *SLCO1B1**15 and *20 also increased the risk of GP-diagnosed muscle symptoms during statin treatment; we estimated that genotype-guided prescribing may prevent 17.6% of diagnosed muscle symptoms during statin treatment in *15 carriers, and 10.3% in the *20 carriers. This illustrates the absolute effect of these pharmacogenetic variants on SAMS events. The evidence for other common variants was weaker and less reliable. We also identified two rare mutations with large effects on LDL in statin users: 45.8% of the rare allele carriers had clinically high LDL (>3.5 mmol/L) during statin treatment, compared to 11% of non-carriers. Combining rare variants with common-variant gene risk scores to identify patients at “high risk” of statin-associated muscle symptoms (SAMS) could further improve LDL control, though replication is needed, as is cost–benefit analysis, given the predominantly modest effect sizes observed.

rs11045819 (Pro155Thr) and rs34671512 (Leu643Phe) increase simvastatin bioavailability [[18](#B18-ijms-25-04426)], suggesting GoF effects. In our study these variants improved the estimates for statin outcomes in UK Biobank patients when combined into a gene risk score (GRS); in “high risk” patients (i.e., carrying a *5 risk allele and no GoF alleles), GP-diagnosed muscle symptoms were higher (HR: 1.13, *p* = 0.02) than the estimate for *5 CC allele carriers alone (HR: 1.09, *p* = 0.048), compared to non-carriers. These findings partially replicate the Bigossi et al. [[19](#B19-ijms-25-04426)] GRS in 15,378 statin users, where incorporating GoF variants into the GRS improved the prediction of SAMS. The authors also included *SLCO1B1**37 in the GRS, defining the G-allele as protective. Other studies report that the G-allele protects against statin-related adverse events in 4196 [[6](#B6-ijms-25-04426)] and 15,378 statin patients [[19](#B19-ijms-25-04426)]. In our analysis, we found that the *37 G-allele *increased* the risk of statin adverse outcomes when included in the GRS. This difference between our study and Bigossi et al. warrants further investigation; PharmGKB reports that *37 is a “normal” function variant, rather than protective [[3](#B3-ijms-25-04426)], due to studies suggesting the *37 effect varies with the presence of *5 [[28](#B28-ijms-25-04426)]. We found that neither *5 nor *37 alone is associated with GP-diagnosed muscle symptoms, yet *15 carriers (the combination) had higher risk, suggesting—at least in UK Biobank participants—that the *37 G-allele in combination with *5a is the risk-increasing allele. More work is required to understand the context-dependent effects of *37. Additionally, we found that *37 homozygotes have a nominally decreased risk of CHD during dCCB treatment (HR 0.94, *p* = 0.02), compared to participants with no *37 alleles. This may be an anomalous result, given that we found no evidence that *37 homozygotes correlate with cholesterol, GP-diagnosed muscle symptoms, or treatment discontinuation.

For *SLCO1B1**15 (*5 and *37 in the same individual), PharmGKB reports high-level evidence for pharmacokinetic relationships, acting similarly to *5, noting limited evidence of an effect on clinical outcomes [[29](#B29-ijms-25-04426),[30](#B30-ijms-25-04426)]. Our findings extend this, showing that although estimates for the effect of *5 on LDL are greater than those for *15, the *15 individuals (a subset of *5 who also carry *37-G) had higher rates of GP-prescribed statin discontinuation and GP-diagnosed muscle symptoms. However, given the weight of prior evidence, the effect sizes and significance we observed are modest. We found that *5 and *15 are associated with lower vitamin D levels. This is consistent with a previous meta-analysis, where low vitamin D was associated with SAMS [[24](#B24-ijms-25-04426)], and supports a causal role of statin pharmacogenetics in lowering vitamin D. *SLCO1B1**20 was associated with increased vitamin D (though the association was only borderline statistically significant: *p* = 0.03) and SAMS, contrary to the other genotypes. This could be due to the complexity of SAMS’s mechanisms; we hypothesize that the reduction of unintended targets for HMG CoA Reductase inhibition, such as dolichols, ubiquinone, and prenylated proteins [[31](#B31-ijms-25-04426)], may be the reason for the inverted *SLCO1B1**20 effect, as it has been found to increase this activity, albeit with limited evidence [[3](#B3-ijms-25-04426)]. Further investigations are required.

Two studies of 717 and 699 patients report large effects of rare variants on relevant phenotypes [[19](#B19-ijms-25-04426),[28](#B28-ijms-25-04426)]. Consistent with this, we identified two rare variants that substantially increased LDL levels compared to common variants. This is a trend often seen in human genetics, due to selection pressure against large-effect variants that perturb metabolism. These variants were not associated with LDL in untreated people (the effect on LDL is therefore likely pharmacogenetic on statins). Including them in the GRS increased the overall effect size, and suggests that identifying specifically high-risk individuals for monitoring and intervention might be possible. In a recent study, both rare variants (rs374113543-rs373327528) were reported to not significantly affect OATP1B1 protein expression [[21](#B21-ijms-25-04426)], though transporter activity for these variants was not assessed. These variants have not otherwise been previously reported and should be interpreted with caution until replicated.

We also investigated other biomarkers at the UK Biobank baseline. Bilirubin hepatic transport is carried out by organic anion-transporting polypeptides (OATP) including OATP1B1 (coded by *SLCO1B1*), and its levels are increased by atorvastatin [[26](#B26-ijms-25-04426)]. In our study, *SLCO1B1* genetic variants affected bilirubin levels in untreated UK Biobank participants and those treated with statins. Higher direct bilirubin was associated with ischemic stroke (in 275 stroke patients) [[25](#B25-ijms-25-04426)], but we did not find associations between *SLCO1B1* variants and ischemic stroke, possibly due to few UK Biobank patients receiving diagnoses of stroke.

Our study has several strengths. To our knowledge, it is the largest analysis of exome sequencing data in statin patients in a general-population cohort with linked medical-record follow-up over decades. We provide clinical evidence on statin outcomes affected by *SLCO1B1* common variants, haplotypes, and rare variants. We also extend a recent [[19](#B19-ijms-25-04426)] *SLCO1B1* ‘gene risk score’ and confirm that combining *SLCO1B1**5 and GoF can improve estimates of adverse outcomes. We used the TWIST framework to estimate the genetically moderated treatment effect (number of diagnoses attributed to the genotype) at the group level. This has multiple strengths, such as including information from untreated people, and combining estimates from independent statistical methods. Whilst this framework allows us to conclude in this study that a percentage of adverse events in patients on atorvastatin and simvastatin could have been avoided if patients with the genotype received the normal effect of the treatment, it cannot yet be used to make treatment recommendations for individual patients. This would require an in-depth clinical assessment of the various treatment options available to each individual and is a question under active consideration as part of ongoing research.

This study has several limitations. UK Biobank is a volunteer cohort, and participants were healthier than the general population at baseline [[32](#B32-ijms-25-04426)], and recall bias may affect our findings. Using electronic medical records after baseline assessment helps alleviate this limitation. The ascertainment of statin-related myopathy and related diagnoses was limited due to the lack of available diagnostic codes in the GP record. SAMS’s prevalence may be overestimated in cohort studies (17%), with evidence from RCTs suggesting much lower true prevalence (4.9%) after the blinding of participants [[33](#B33-ijms-25-04426)]; in our ICD10 code-based data, we identified 3.4% prevalence, which is closer to this estimate from RCTs. Furthermore, genotype did not increase the risk of SAMs in untreated people, consistent with the statin-specific pharmacogenetic mechanism. The GP-ascertained outcomes had limited power to detect rare-variant associations, compared to quantitative baseline measures. Inconsistency in rare-variant associations necessitate further studies to establish a comprehensive understanding. We were not able to examine statin doses and other statins due to the limited availability of reliable data.

Our results show that, although patients carrying *SLCO1B1* variants are more likely to be diagnosed with muscle pain and discontinue statin prescriptions, they do not have increased cardiovascular disease risk. This may be due to GPs overcoming the effect of *SLCO1B1* variants and achieving target LDL levels; however, we observed large effects on LDL at the UK Biobank baseline; therefore, more data are needed. UK Biobank includes generally healthy individuals from socioeconomically advantaged backgrounds [[32](#B32-ijms-25-04426)]. In the general population, LDL monitoring may differ if patients do not see their GPs to report symptoms, and thus, adverse events in higher-risk groups might be more detrimental. Our study is therefore potentially significant in the context of real-world healthcare practices and may have population-wide implications. We hope that our study contributes to informing how genotype information is used in routine clinical practice, once genotype information is known for many patients; for example, following the implementation of Our Future Health [[34](#B34-ijms-25-04426)], where 5 million UK adults will have genotype information and consent for actionable results to be referred to their GP, the rate of avoided discontinuation could rise dramatically. Our findings of modest effects of SLCO1B1 variants on the outcomes studied do not support specific genotyping.

In conclusion, common *SLCO1B1* variants are associated with adverse outcomes in 64,000 statin patients from a largescale general-population cohort. Rare variants with large effects are also identified, as are improved estimates for the *SLCO1B1* gene risk score, though replication is needed.

## 4. Materials and Methods

### 4.1. UK Biobank (UKB)

UK Biobank comprises 503,325 community-based volunteers aged 40–70 years enrolled in Wales, Scotland, and England in 2006–2010. The Northwest Multi-Center Research Ethics Committee approved the collection and use of UKB data (Research Ethics Committee reference 11/NW/0382). Participants gave informed consent for the use of their data, health records, and biological materials for health-related research purposes. Access to UKB was granted under Application Number 14631. Baseline assessment included the collection of demographic, lifestyle, and health information via questionnaires, and anthropometric measures. Participants provided blood samples for genetic and biochemical analyses. This study comprises two distinct analyses: (1) data from baseline assessment, and (2) using the linked GP (General Practice) data available in 230,096 (45.7%) participants ([Figure 3](#ijms-25-04426-f003)).

GP data are up to August 2016 (TPP system supplier in England) and September 2017 (EMIS/Vision system in Wales). Drug name, quantity, date of prescription, and drug code (in Clinical Read version 2, British National Formulary [BNF] or DM + D [Dictionary of Medicines and Devices] format, depending on the provider) are available. Prescribing records included 10, 20, 40, and 80 mg simvastatin (zocor, simvastatin, simvadol) and 10, 20, 40, 60, and 80 mg atorvastatin (atorvastatin, lipitor) prescriptions. All available formulations were considered regardless of dose due to the limited availability of reliable dose data. The censor date is the date of withdrawal (deletion from the GP list, where available) or 28 February 2016 if the date of withdrawal is not available (i.e., still registered in an available clinic). Data after 28 February 2016 were incomplete (see UK Biobank document [[35](#B35-ijms-25-04426)]; details are described elsewhere [[11](#B11-ijms-25-04426)]).

### 4.2. Whole-Exome Sequencing (WES)

Detailed methods for WES analysis in UK Biobank were published by the Regeneron Genetics Center [[36](#B36-ijms-25-04426)]. In brief, participant DNA was enzymatically shared with ~200 base pairs to create a DNA library, with a unique 10-base-pair barcode added to facilitate multiplexed exome capture with a modified IDT xGen probe library. DNA was PCR-amplified and sequenced using 75-base-pair paired-end reads on an Illumina NovaSeq 6000 platform (Illumina, Inc., San Diego, CA, USA). Variant calling and quality control (QC) were performed centrally (described previously [[37](#B37-ijms-25-04426)]). ‘BWA MEM’ mapped reads to the hg38 reference genomes, and ‘WeCall’ identified small SNP and indel variants. Only called variants that have passed rigorous QC are available to researchers. Data are available on 469,914 participants.

Due to known differences in allele frequencies and pharmacogenetic effects across diverse ancestral groups, we focused on the participants genetically similar to the 1000 Genomes European (EUR) reference population (95% of UK Biobank participants), identified by principal component analysis (described previously [[38](#B38-ijms-25-04426)]).

### 4.3. SLCO1B1 Variants

We identified previously reported variants and haplotypes with the “star” nomenclature [[39](#B39-ijms-25-04426)], such as *SLCO1B1**5 and *SLCO1B1**15 (both *5 and *37—[Figure 4](#ijms-25-04426-f004) for details). In statin patients at baseline, the following *SLCO1B1* variants were observed: *15 (frequency = 23.7%), *14 (24%), *37 (15.1%), *20 (10.1%), *4 (2.4%), *5 (2.2%), *19 (0.26%), *46 (0.11%), and *45 (0.11%—heterozygotes). *31 and *47 were not observed due to the small numbers of carriers ([Figure 4](#ijms-25-04426-f004)).

### Figure 4.

![Figure 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2f45/11050003/204dbe8a8b21/ijms-25-04426-g004.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=11050003_ijms-25-04426-g004.jpg)

SLCO1B1 variants flowchart. We identified 1690 SLCO1B1 variants in whole-exome sequencing data through a variant effect predictor (VEP); of those, we filtered for high–moderate impact consequences in patients who reported statin use at baseline (n = 185) and had a GP statin prescription in their primary care records (n = 199). We included variants with mac > 3 (n = 51); of those 51, 4 were common variants. We present the minor allele frequencies of 4 common variants in the table below. We also present 6 haplotypes (represented by a ‘star’) of the 4 common variants in the table.

To identify novel variants with a potentially high impact on OATP1B1 protein function, we used the Variant Effect Predictor (VEP) [[40](#B40-ijms-25-04426)] to estimate their effect on *SLCO1B1* transcripts, protein sequence, and regulatory regions (see [Supplementary Table S10](#app1-ijms-25-04426) for VEP output). We investigated all *SLCO1B1* variants in the UKB WES data between GRChg38 coordinates chr12:21131194 and chr12:21239796 ([https://www.ncbi.nlm.nih.gov/gene/10599](https://www.ncbi.nlm.nih.gov/gene/10599), accessed on 1 February 2023). There were 1520 unique variants, of which 489 were identified as missense variants, and 618 were predicted to have a moderate or high impact: 185 unique variants were found in statin patients at baseline and 199 in patients for whom statin were prescribed by their GP, of which 51 had a minor allele count over 3, of which 4 were common variants (MAF > 0.1%) and 47 were rare-variants ([Figure 4](#ijms-25-04426-f004) for the flowchart).

We calculated the MAC (minor allele count: total number of minor alleles carried by patients) and MAF (minor allele frequency: the proportion of all alleles in the participants that were the minor allele) for each variant. We included variants in the single-variant analyses if their MAC > 3 ([Supplementary Tables S11 and S12](#app1-ijms-25-04426) for MAC and MAF for included variants).

### 4.4. Statin Gene Risk Score (GRS)

We created two GRSs.

(1) “High risk” patients are homozygous for the *5 C allele, or are heterozygous for *5 and do not carry protective alleles for *37 (i.e., are rs2306283-A homozygotes), Pro155Thr (i.e., are rs11045819-C homozygotes), or Leu643Phe (i.e., are rs34671512-A homozygotes), as reported previously [[19](#B19-ijms-25-04426)]. All remaining allele combinations are defined as “low risk”.

(2) We found that for *37, it is the G-allele that has similar trends to the detrimental *5 allele in analyses of LDL, GP-diagnosed muscle symptoms, treatment discontinuation, and hospital-diagnosed coronary heart disease (CHD) (in contrast to the previous study [[19](#B19-ijms-25-04426)]).

We therefore analyzed the GRS in two ways: (1) as reported by Bigossi [[19](#B19-ijms-25-04426)] and (2) using rs2306283-G homozygotes in the “high risk” group ([Table 4](#ijms-25-04426-t004)).

### Table 4.

|   |   | *5 | *37 | rs11045819 | rs34671512 |
| --- | --- | --- | --- | --- | --- |
|   | High risk | CC |   |   |   |
| GRS1 | High risk | TC | AA | CC | AA |
|   | Low risk | all remaining |  |  |  |
|   | High risk | CC |   |   |   |
| GRS2 | High risk | TC | GG | CC | AA |
|   | Low risk | all remaining |  |  |  |

Table 4 Caption: Statin Gene Risk Score (GRS).

### 4.5. Outcomes

We examined outcomes in two categories: baseline and GP-linked outcomes. Where data were available, we included traits linked to *SLCO1B1* in the GWAS Catalog [[41](#B41-ijms-25-04426)].

LDL, triglycerides, total cholesterol, direct bilirubin, C-reactive protein (CRP), alanine aminotransferase (ALT), cystatin C, vitamin D, and Glycated hemoglobin (Hba1c) were included (mostly from GWAS catalogue for SLCO1B1). We also extracted ‘pain experienced in last month’ variable (Data-Field 6159: head or shoulder pain, back pain, hip pain, knee pain, or pain all over body).

We utilized ICD-10 codes for conditions related to myopathy, myositis, myalgia, or rhabdomyolysis. Discontinuation was defined when patients had their last prescription at least 3 months prior to the censoring date (removal from GP list) or by 28 February 2016, if no deduction date was available. We utilized ICD-10 codes for conditions related to myopathy, myositis, myalgia, or rhabdomyolysis (G72.0, G72.8, G72.9, M60.8, M60.9, M79.1, M62.82) and converted them to the Read codes used in UK primary care records, utilizing diagnostic code maps provided by UKB.

We captured cardiovascular events from hospital admissions data, covering up to 15 years of follow-up (HES data in England until September 2021, with data from Scotland censored in August 2020 and Wales in February 2018). We identified diagnoses of CHD, including myocardial infarction and angina, using ICD-10 codes I20*, I21*, I22*, I23*, I24*, and I25*. Additionally, diagnoses of ischemic stroke were identified using ICD-10 code I63*.

### 4.6. Statistical Analysis

We conducted linear regression analyses to estimate associations between biomarkers and *SLCO1B1* variants in statin users aged 40+, adjusting for sex, age, assessment center, and principal components of genetic ancestry 1–5 to account for population substructure. We also tested associations between self-reported pain and LDL-increasing rare variants.

We performed time-to-event analyses using electronic medical record data (primary- and secondary-care-linked) for the common (*) variants. We included patients who were 40+ years old at the time of their first statin prescription and had a minimum of two months’ prescriptions during the treatment period. Cox’s proportional hazards regression models were used to assess the associations of GP-diagnosed muscle symptoms, discontinuation of statin therapy, hospital inpatient-diagnosed CHD, and ischemic stroke with the * variants, adjusting for sex, age at first statin prescription, and principal components of genetic ancestry ([Supplementary Information](#app1-ijms-25-04426) for details).

For the significant LDL-altering variants, we used our novel pharmacogenetic causal inference approach “TWIST” (Triangulation with a Study [[27](#B27-ijms-25-04426)]) to estimate the population-average effect on outcomes if all *SLCO1B1* variant carriers could experience the same treatment effect as non-carriers. This extends the well-established idea of estimating avoidable events in pharmacogenetics by incorporating information from untreated individuals and triangulating evidence for multiple statistical methods. Briefly, the method uses regression models to test several assumptions common to pharmacogenetic analysis, primarily that the genetic variants do not predict whether an individual receives statin treatment; are not associated with any measured confounders predicting statin use or the studied outcome; and only affect the outcome through the interaction with statins (see [[27](#B27-ijms-25-04426)] for details). The method robustly estimates the ‘Genetically Moderated Treatment Effect’ (GMTE): see [Supplementary Materials](#app1-ijms-25-04426) for details. We used the {twistR} R package v0.1.4 ([https://github.com/lukepilling/twistR](https://github.com/lukepilling/twistR), accessed on 1 March 2023).

## Acknowledgments

This research was conducted using the UK Biobank resource, under application 14631. We thank the UK Biobank participants and coordinators for the dataset. This study was supported by the National Institute for Health and Care Research, Exeter Biomedical Research Centre. The views expressed are those of the authors and not necessarily those of the NIHR or the Department of Health and Social Care. The authors would like to acknowledge the use of the University of Exeter’s High-Performance Computing (HPC) facility for carrying out this work. For the purpose of open access, the author has applied a ‘Creative Commons Attribution (CC BY)’ licence to any Author Accepted Manuscript version arising from this submission.

## Supplementary Materials

The following supporting information can be downloaded at: [https://www.mdpi.com/article/10.3390/ijms25084426/s1](https://www.mdpi.com/article/10.3390/ijms25084426/s1), References [[42](#B42-ijms-25-04426),[43](#B43-ijms-25-04426)] are cited in Supplementary Materials.

## Author Contributions

D.T. generated the data, performed the analyses, interpreted the results, created the figures, searched the literature, and co-wrote the manuscript. J.B. contributed to data analysis and interpretation, and contributed to the manuscript. J.A.H.M. provided expert clinical interpretation of the data and contributed to the manuscript. D.M. oversaw the interpretation and literature search. L.C.P. generated the data, performed the analyses, interpreted the results, created the figures, searched the literature, and co-wrote the manuscript. L.C.P. is the guarantor. The corresponding author attests that all listed authors meet the authorship criteria and that no others meeting the criteria have been omitted. All authors have read and agreed to the published version of the manuscript.

## Institutional Review Board Statement

The Northwest Multi-Center Research Ethics Committee approved the collection and use of UKB data (Research Ethics Committee reference 11/NW/0382).

## Data Availability Statement

Participant-level data are available on application to the UK Biobank ([www.ukbiobank.ac.uk/register-apply](http://www.ukbiobank.ac.uk/register-apply), accessed on 20 March 2024). The TWIST R package is available on GitHub ([https://github.com/lukepilling/twistR](https://github.com/lukepilling/twistR), accessed on 20 March 2024) but contains no individual-level data.

## Conflicts of Interest

Jack Bowden is employed part-time by Novo Nordisk; his company’s work is unrelated to that presented here.

## Funding Statement

D.T. was funded by the Ministry of National Education, Republic of Turkey. L.C.P. and D.M. were supported by the University of Exeter Medical School. L.C.P., D.T. and J.B. were funded by research grant MR/X011372/1. J.M. was funded by an NIHR Advanced Fellowship (NIHR302270). The views expressed in this publication are those of the author(s) and not necessarily those of the NIHR, NHS, or UK Department of Health and Social Care. The funders had no input in the study design; in the collection, analysis, and interpretation of the data; in the writing of the report; or in the decision to submit the article for publication. The researchers acted independently of the study sponsors in all aspects of this study. This study was supported by MRC grant MR/X011372/1.

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

Participant-level data are available on application to the UK Biobank ([www.ukbiobank.ac.uk/register-apply](http://www.ukbiobank.ac.uk/register-apply), accessed on 20 March 2024). The TWIST R package is available on GitHub ([https://github.com/lukepilling/twistR](https://github.com/lukepilling/twistR), accessed on 20 March 2024) but contains no individual-level data.

### Supplementary Materials

### Data Availability Statement

Participant-level data are available on application to the UK Biobank ([www.ukbiobank.ac.uk/register-apply](http://www.ukbiobank.ac.uk/register-apply), accessed on 20 March 2024). The TWIST R package is available on GitHub ([https://github.com/lukepilling/twistR](https://github.com/lukepilling/twistR), accessed on 20 March 2024) but contains no individual-level data.

## References

1. Oshiro C., Mangravite L., Klein T., Altman R. PharmGKB very important pharmacogene: SLCO1B1. Pharmacogenetics Genom. 2010;20:211–216. doi: 10.1097/FPC.0b013e328333b99c.  [DOI](https://doi.org/10.1097/FPC.0b013e328333b99c) | [PMC free article](/articles/PMC3086841/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19952871/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics%20Genom.&title=PharmGKB%20very%20important%20pharmacogene:%20SLCO1B1&author=C.%20Oshiro&author=L.%20Mangravite&author=T.%20Klein&author=R.%20Altman&volume=20&publication_year=2010&pages=211-216&pmid=19952871&doi=10.1097/FPC.0b013e328333b99c&)

2. SLCO1B1: A VIP Gene. [(accessed on 30 March 2023)]. Available online: https://www.pharmgkb.org/vip/PA166169500.  [https://www.pharmgkb.org/vip/PA166169500](https://www.pharmgkb.org/vip/PA166169500)

3. PharmGKB Gene-Specific Information Tables for SLCO1B1. [(accessed on 11 July 2023)]. Available online: https://www.pharmgkb.org/page/slco1b1RefMaterials.  [https://www.pharmgkb.org/page/slco1b1RefMaterials](https://www.pharmgkb.org/page/slco1b1RefMaterials)

4. Cooper-DeHoff R.M., Niemi M., Ramsey L.B., Luzum J.A., Tarkiainen E.K., Straka R.J., Gong L., Tuteja S., Wilke R.A., Wadelius M., et al. The Clinical Pharmacogenetics Implementation Consortium Guideline for SLCO1B1, ABCG2, and CYP2C9 genotypes and Statin-Associated Musculoskeletal Symptoms. Clin. Pharmacol. Ther. 2022;111:1007–1021. doi: 10.1002/cpt.2557.  [DOI](https://doi.org/10.1002/cpt.2557) | [PMC free article](/articles/PMC9035072/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35152405/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=The%20Clinical%20Pharmacogenetics%20Implementation%20Consortium%20Guideline%20for%20SLCO1B1,%20ABCG2,%20and%20CYP2C9%20genotypes%20and%20Statin-Associated%20Musculoskeletal%20Symptoms&author=R.M.%20Cooper-DeHoff&author=M.%20Niemi&author=L.B.%20Ramsey&author=J.A.%20Luzum&author=E.K.%20Tarkiainen&volume=111&publication_year=2022&pages=1007-1021&pmid=35152405&doi=10.1002/cpt.2557&)

5. Voora D., Shah S.H., Spasojevic I., Ali S., Reed C.R., Salisbury B.A., Ginsburg G.S. The SLCO1B1*5Genetic Variant Is Associated With Statin-Induced Side Effects. J. Am. Coll. Cardiol. 2009;54:1609–1616. doi: 10.1016/j.jacc.2009.04.053.  [DOI](https://doi.org/10.1016/j.jacc.2009.04.053) | [PMC free article](/articles/PMC3417133/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19833260/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Am.%20Coll.%20Cardiol.&title=The%20SLCO1B1*5Genetic%20Variant%20Is%20Associated%20With%20Statin-Induced%20Side%20Effects&author=D.%20Voora&author=S.H.%20Shah&author=I.%20Spasojevic&author=S.%20Ali&author=C.R.%20Reed&volume=54&publication_year=2009&pages=1609-1616&pmid=19833260&doi=10.1016/j.jacc.2009.04.053&)

6. Donnelly L.A., Doney A.S.F., Tavendale R., Lang C.C., Pearson E.R., Colhoun H.M., McCarthy M.I., Hattersley A.T., Morris A.D., Palmer C.N.A. Common Nonsynonymous Substitutions in SLCO1B1 Predispose to Statin Intolerance in Routinely Treated Individuals With Type 2 Diabetes: A Go-DARTS Study. Clin. Pharmacol. Ther. 2010;89:210–216. doi: 10.1038/clpt.2010.255.  [DOI](https://doi.org/10.1038/clpt.2010.255) | [PMC free article](/articles/PMC3353487/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21178985/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Common%20Nonsynonymous%20Substitutions%20in%20SLCO1B1%20Predispose%20to%20Statin%20Intolerance%20in%20Routinely%20Treated%20Individuals%20With%20Type%202%20Diabetes:%20A%20Go-DARTS%20Study&author=L.A.%20Donnelly&author=A.S.F.%20Doney&author=R.%20Tavendale&author=C.C.%20Lang&author=E.R.%20Pearson&volume=89&publication_year=2010&pages=210-216&pmid=21178985&doi=10.1038/clpt.2010.255&)

7. Voora D., Baye J., McDermaid A., Gowda S.N., Wilke R.A., Myrmoe A.N., Hajek C., Larson E.A. SLCO1B1*5 Allele Is Associated With Atorvastatin Discontinuation and Adverse Muscle Symptoms in the Context of Routine Care. Clin. Pharmacol. Ther. 2022;111:1075–1083. doi: 10.1002/cpt.2527.  [DOI](https://doi.org/10.1002/cpt.2527) | [PMC free article](/articles/PMC9303592/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35034348/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=SLCO1B1*5%20Allele%20Is%20Associated%20With%20Atorvastatin%20Discontinuation%20and%20Adverse%20Muscle%20Symptoms%20in%20the%20Context%20of%20Routine%20Care&author=D.%20Voora&author=J.%20Baye&author=A.%20McDermaid&author=S.N.%20Gowda&author=R.A.%20Wilke&volume=111&publication_year=2022&pages=1075-1083&pmid=35034348&doi=10.1002/cpt.2527&)

8. SEARCH Collaborative Group. Link E., Parish S., Armitage J., Bowman L., Heath S., Matsuda F., Gut I., Lathrop M., Collins R. SLCO1B1Variants and Statin-Induced Myopathy—A Genomewide Study. N. Engl. J. Med. 2008;359:789–799. doi: 10.1056/nejmoa0801936.  [DOI](https://doi.org/10.1056/nejmoa0801936) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18650507/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%20Engl.%20J.%20Med.&title=SLCO1B1Variants%20and%20Statin-Induced%20Myopathy%E2%80%94A%20Genomewide%20Study&author=E.%20Link&author=S.%20Parish&author=J.%20Armitage&author=L.%20Bowman&author=S.%20Heath&volume=359&publication_year=2008&pages=789-799&pmid=18650507&doi=10.1056/nejmoa0801936&)

9. Carr D.F., Francis B., Jorgensen A.L., Zhang E., Chinoy H., Heckbert S.R., Bis J.C., Brody J.A., Floyd J.S., Psaty B.M., et al. Genomewide Association Study of Statin-Induced Myopathy in Patients Recruited Using the UK Clinical Practice Research Datalink. Clin. Pharmacol. Ther. 2019;106:1353–1361. doi: 10.1002/cpt.1557.  [DOI](https://doi.org/10.1002/cpt.1557) | [PMC free article](/articles/PMC6896237/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31220337/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Genomewide%20Association%20Study%20of%20Statin-Induced%20Myopathy%20in%20Patients%20Recruited%20Using%20the%20UK%20Clinical%20Practice%20Research%20Datalink&author=D.F.%20Carr&author=B.%20Francis&author=A.L.%20Jorgensen&author=E.%20Zhang&author=H.%20Chinoy&volume=106&publication_year=2019&pages=1353-1361&pmid=31220337&doi=10.1002/cpt.1557&)

10. Vrablík M., Hubáček J.A., Dlouhá D., Adámková V., Zlatohlavek L., Viklický O., Hrubá P., Češka R. SLCO1B1 Polymorphism is not associated with Risk of Statin-Induced Myalgia/Myopathy in a Czech Population. Med. Sci. Monit. 2015;21:1454–1459. doi: 10.12659/MSM.893007.  [DOI](https://doi.org/10.12659/MSM.893007) | [PMC free article](/articles/PMC4450600/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25992810/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Med.%20Sci.%20Monit.&title=SLCO1B1%20Polymorphism%20is%20not%20associated%20with%20Risk%20of%20Statin-Induced%20Myalgia/Myopathy%20in%20a%20Czech%20Population&author=M.%20Vrabl%C3%ADk&author=J.A.%20Hub%C3%A1%C4%8Dek&author=D.%20Dlouh%C3%A1&author=V.%20Ad%C3%A1mkov%C3%A1&author=L.%20Zlatohlavek&volume=21&publication_year=2015&pages=1454-1459&pmid=25992810&doi=10.12659/MSM.893007&)

11. Türkmen D., Masoli J.A.H., Kuo C., Bowden J., Melzer D., Pilling L.C. Statin treatment effectiveness and the SLCO1B1*5 reduced function genotype: Long-term outcomes in women and men. Br. J. Clin. Pharmacol. 2022;88:3230–3240. doi: 10.1111/bcp.15245.  [DOI](https://doi.org/10.1111/bcp.15245) | [PMC free article](/articles/PMC9305522/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35083771/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Clin.%20Pharmacol.&title=Statin%20treatment%20effectiveness%20and%20the%20SLCO1B1*5%20reduced%20function%20genotype:%20Long-term%20outcomes%20in%20women%20and%20men&author=D.%20T%C3%BCrkmen&author=J.A.H.%20Masoli&author=C.%20Kuo&author=J.%20Bowden&author=D.%20Melzer&volume=88&publication_year=2022&pages=3230-3240&pmid=35083771&doi=10.1111/bcp.15245&)

12. Khine H., Yuet W.C., Adams-Huet B., Ahmad Z. Statin-associated muscle symptoms and SLCO1B1 rs4149056 genotype in patients with familial hypercholesterolemia. Am. Heart J. 2016;179:1–9. doi: 10.1016/j.ahj.2016.05.015.  [DOI](https://doi.org/10.1016/j.ahj.2016.05.015) | [PMC free article](/articles/PMC5014387/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27595674/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20Heart%20J.&title=Statin-associated%20muscle%20symptoms%20and%20SLCO1B1%20rs4149056%20genotype%20in%20patients%20with%20familial%20hypercholesterolemia&author=H.%20Khine&author=W.C.%20Yuet&author=B.%20Adams-Huet&author=Z.%20Ahmad&volume=179&publication_year=2016&pages=1-9&pmid=27595674&doi=10.1016/j.ahj.2016.05.015&)

13. PharmGKB Clinical Annotation for rs4149056 (SLCO1B1); Simvastatin; Statin-Related Myopathy (Level 1A Toxicity) [(accessed on 4 June 2023)]. Available online: https://www.pharmgkb.org/clinicalAnnotation/655384011.  [https://www.pharmgkb.org/clinicalAnnotation/655384011](https://www.pharmgkb.org/clinicalAnnotation/655384011)

14. Peyser B., Perry E.P., Singh K., Gill R.D., Mehan M.R., Haga S.B., Musty M.D., Milazzo N.A., Savard D., Li Y.-J., et al. Effects of Delivering SLCO1B1 Pharmacogenetic Information in Randomized Trial and Observational Settings. Circ. Genom. Precis. Med. 2018;11:e002228. doi: 10.1161/CIRCGEN.118.002228.  [DOI](https://doi.org/10.1161/CIRCGEN.118.002228) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30354330/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circ.%20Genom.%20Precis.%20Med.&title=Effects%20of%20Delivering%20SLCO1B1%20Pharmacogenetic%20Information%20in%20Randomized%20Trial%20and%20Observational%20Settings&author=B.%20Peyser&author=E.P.%20Perry&author=K.%20Singh&author=R.D.%20Gill&author=M.R.%20Mehan&volume=11&publication_year=2018&pages=e002228&pmid=30354330&doi=10.1161/CIRCGEN.118.002228&)

15. Tuteja S., Rader D.J. SLCO1B1 and Statin Therapy. Circ. Genom. Precis. Med. 2018;11:e002320. doi: 10.1161/CIRCGEN.118.002320.  [DOI](https://doi.org/10.1161/CIRCGEN.118.002320) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30354338/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circ.%20Genom.%20Precis.%20Med.&title=SLCO1B1%20and%20Statin%20Therapy&author=S.%20Tuteja&author=D.J.%20Rader&volume=11&publication_year=2018&pages=e002320&pmid=30354338&doi=10.1161/CIRCGEN.118.002320&)

16. Pasanen M.K., Fredrikson H., Neuvonen P.J., Niemi M. Different Effects of SLCO1B1 Polymorphism on the Pharmacokinetics of Atorvastatin and Rosuvastatin. Clin. Pharmacol. Ther. 2007;82:726–733. doi: 10.1038/sj.clpt.6100220.  [DOI](https://doi.org/10.1038/sj.clpt.6100220) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17473846/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Different%20Effects%20of%20SLCO1B1%20Polymorphism%20on%20the%20Pharmacokinetics%20of%20Atorvastatin%20and%20Rosuvastatin&author=M.K.%20Pasanen&author=H.%20Fredrikson&author=P.J.%20Neuvonen&author=M.%20Niemi&volume=82&publication_year=2007&pages=726-733&pmid=17473846&doi=10.1038/sj.clpt.6100220&)

17. PharmGKB Clinical Annotation for SLCO1B1*1, SLCO1B1*5, SLCO1B1*15; Fluvastatin; Statin-Related Myopathy (Level 1A Toxicity) [(accessed on 27 July 2023)]. Available online: https://www.pharmgkb.org/clinicalAnnotation/1451678626.  [https://www.pharmgkb.org/clinicalAnnotation/1451678626](https://www.pharmgkb.org/clinicalAnnotation/1451678626)

18. Mykkänen A.J.H., Taskinen S., Neuvonen M., Paile-Hyvärinen M., Tarkiainen E.K., Lilius T., Tapaninen T., Backman J.T., Tornio A., Niemi M. Genomewide Association Study of Simvastatin Pharmacokinetics. Clin. Pharmacol. Ther. 2022;112:676–686. doi: 10.1002/cpt.2674.  [DOI](https://doi.org/10.1002/cpt.2674) | [PMC free article](/articles/PMC9540481/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35652242/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Genomewide%20Association%20Study%20of%20Simvastatin%20Pharmacokinetics&author=A.J.H.%20Mykk%C3%A4nen&author=S.%20Taskinen&author=M.%20Neuvonen&author=M.%20Paile-Hyv%C3%A4rinen&author=E.K.%20Tarkiainen&volume=112&publication_year=2022&pages=676-686&pmid=35652242&doi=10.1002/cpt.2674&)

19. Bigossi M., Maroteau C., Dawed A.Y., Taylor A., Srinivasan S., Melhem A.L., Pearson E.R., Pola R., Palmer C.N.A., Siddiqui M.K. A gene risk score using missense variants in SLCO1B1 is associated with earlier onset statin intolerance. Eur. Heart J. Cardiovasc. Pharmacother. 2023;9:536–545. doi: 10.1093/ehjcvp/pvad040.  [DOI](https://doi.org/10.1093/ehjcvp/pvad040) | [PMC free article](/articles/PMC10509567/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/37253618/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur.%20Heart%20J.%20Cardiovasc.%20Pharmacother.&title=A%20gene%20risk%20score%20using%20missense%20variants%20in%20SLCO1B1%20is%20associated%20with%20earlier%20onset%20statin%20intolerance&author=M.%20Bigossi&author=C.%20Maroteau&author=A.Y.%20Dawed&author=A.%20Taylor&author=S.%20Srinivasan&volume=9&publication_year=2023&pages=536-545&pmid=37253618&doi=10.1093/ehjcvp/pvad040&)

20. Ingelman-Sundberg M., Mkrtchian S., Zhou Y., Lauschke V.M. Integrating rare genetic variants into pharmacogenetic drug response predictions. Hum. Genom. 2018;12:26. doi: 10.1186/s40246-018-0157-3.  [DOI](https://doi.org/10.1186/s40246-018-0157-3) | [PMC free article](/articles/PMC5968569/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29793534/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum.%20Genom.&title=Integrating%20rare%20genetic%20variants%20into%20pharmacogenetic%20drug%20response%20predictions&author=M.%20Ingelman-Sundberg&author=S.%20Mkrtchian&author=Y.%20Zhou&author=V.M.%20Lauschke&volume=12&publication_year=2018&pages=26&pmid=29793534&doi=10.1186/s40246-018-0157-3&)

21. Zhang L., Sarangi V., Ho M.-F., Moon I., Kalari K.R., Wang L., Weinshilboum R.M. SLCO1B1: Application and Limitations of Deep Mutational Scanning for Genomic Missense Variant Function. Drug Metab. Dispos. 2021;49:395–404. doi: 10.1124/dmd.120.000264.  [DOI](https://doi.org/10.1124/dmd.120.000264) | [PMC free article](/articles/PMC8042483/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33658230/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab.%20Dispos.&title=SLCO1B1:%20Application%20and%20Limitations%20of%20Deep%20Mutational%20Scanning%20for%20Genomic%20Missense%20Variant%20Function&author=L.%20Zhang&author=V.%20Sarangi&author=M.-F.%20Ho&author=I.%20Moon&author=K.R.%20Kalari&volume=49&publication_year=2021&pages=395-404&pmid=33658230&doi=10.1124/dmd.120.000264&)

22. Zhang B., Lauschke V.M. Genetic variability and population diversity of the human SLCO (OATP) transporter family. Pharmacol. Res. 2018;139:550–559. doi: 10.1016/j.phrs.2018.10.017.  [DOI](https://doi.org/10.1016/j.phrs.2018.10.017) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30359687/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacol.%20Res.&title=Genetic%20variability%20and%20population%20diversity%20of%20the%20human%20SLCO%20(OATP)%20transporter%20family&author=B.%20Zhang&author=V.M.%20Lauschke&volume=139&publication_year=2018&pages=550-559&pmid=30359687&doi=10.1016/j.phrs.2018.10.017&)

23. Kozyra M., Ingelman-Sundberg M., Lauschke V.M. Rare genetic variants in cellular transporters, metabolic enzymes, and nuclear receptors can be important determinants of interindividual differences in drug response. Anesth. Analg. 2017;19:20–29. doi: 10.1038/gim.2016.33.  [DOI](https://doi.org/10.1038/gim.2016.33) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27101133/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anesth.%20Analg.&title=Rare%20genetic%20variants%20in%20cellular%20transporters,%20metabolic%20enzymes,%20and%20nuclear%20receptors%20can%20be%20important%20determinants%20of%20interindividual%20differences%20in%20drug%20response&author=M.%20Kozyra&author=M.%20Ingelman-Sundberg&author=V.M.%20Lauschke&volume=19&publication_year=2017&pages=20-29&pmid=27101133&doi=10.1038/gim.2016.33&)

24. Michalska-Kasiczak M., Sahebkar A., Mikhailidis D.P., Rysz J., Muntner P., Toth P.P., Jones S.R., Rizzo M., Hovingh G.K., Farnier M., et al. Analysis of vitamin D levels in patients with and without statin-associated myalgia—A systematic review and meta-analysis of 7 studies with 2420 patients. Int. J. Cardiol. 2014;178:111–116. doi: 10.1016/j.ijcard.2014.10.118.  [DOI](https://doi.org/10.1016/j.ijcard.2014.10.118) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25464233/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Cardiol.&title=Analysis%20of%20vitamin%20D%20levels%20in%20patients%20with%20and%20without%20statin-associated%20myalgia%E2%80%94A%20systematic%20review%20and%20meta-analysis%20of%207%20studies%20with%202420%20patients&author=M.%20Michalska-Kasiczak&author=A.%20Sahebkar&author=D.P.%20Mikhailidis&author=J.%20Rysz&author=P.%20Muntner&volume=178&publication_year=2014&pages=111-116&pmid=25464233&doi=10.1016/j.ijcard.2014.10.118&)

25. Asl E.S., Taheraghdam A., Rahmani F., Javadrashid R., Golzari S.E.J., Ghaemian N., Sadeghpour Y., Esfanjani R.M., Soleimanpour H. Determination of the Predictive Value of Serum Bilirubin in Patients with Ischemic Stroke: A Prospective Descriptive Analytical Study. Adv. Pharm. Bull. 2018;8:715–719. doi: 10.15171/apb.2018.080.  [DOI](https://doi.org/10.15171/apb.2018.080) | [PMC free article](/articles/PMC6311645/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30607344/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Adv.%20Pharm.%20Bull.&title=Determination%20of%20the%20Predictive%20Value%20of%20Serum%20Bilirubin%20in%20Patients%20with%20Ischemic%20Stroke:%20A%20Prospective%20Descriptive%20Analytical%20Study&author=E.S.%20Asl&author=A.%20Taheraghdam&author=F.%20Rahmani&author=R.%20Javadrashid&author=S.E.J.%20Golzari&volume=8&publication_year=2018&pages=715-719&pmid=30607344&doi=10.15171/apb.2018.080&)

26. Mirjanic-Azaric B., Rizzo M., Jürgens G., Hallstroem S., Srdic S., Marc J., Cerne D. Atorvastatin treatment increases plasma bilirubin but not HMOX1 expression in stable angina patients. Scand. J. Clin. Lab. Investig. 2015;75:382–389. doi: 10.3109/00365513.2015.1031691.  [DOI](https://doi.org/10.3109/00365513.2015.1031691) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25922869/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Scand.%20J.%20Clin.%20Lab.%20Investig.&title=Atorvastatin%20treatment%20increases%20plasma%20bilirubin%20but%20not%20HMOX1%20expression%20in%20stable%20angina%20patients&author=B.%20Mirjanic-Azaric&author=M.%20Rizzo&author=G.%20J%C3%BCrgens&author=S.%20Hallstroem&author=S.%20Srdic&volume=75&publication_year=2015&pages=382-389&pmid=25922869&doi=10.3109/00365513.2015.1031691&)

27. Bowden J., Pilling L.C., Türkmen D., Kuo C.-L., Melzer D. The Triangulation WIthin a STudy (TWIST) framework for causal inference within pharmacogenetic research. PLoS Genet. 2021;17:e1009783. doi: 10.1371/journal.pgen.1009783.  [DOI](https://doi.org/10.1371/journal.pgen.1009783) | [PMC free article](/articles/PMC8452063/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34495953/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet.&title=The%20Triangulation%20WIthin%20a%20STudy%20(TWIST)%20framework%20for%20causal%20inference%20within%20pharmacogenetic%20research&author=J.%20Bowden&author=L.C.%20Pilling&author=D.%20T%C3%BCrkmen&author=C.-L.%20Kuo&author=D.%20Melzer&volume=17&publication_year=2021&pages=e1009783&pmid=34495953&doi=10.1371/journal.pgen.1009783&)

28. Ramsey L.B., Bruun G.H., Yang W., Treviño L.R., Vattathil S., Scheet P., Cheng C., Rosner G.L., Giacomini K.M., Fan Y., et al. Rare versus common variants in pharmacogenetics: SLCO1B1 variation and methotrexate disposition. Genome Res. 2011;22:1–8. doi: 10.1101/gr.129668.111.  [DOI](https://doi.org/10.1101/gr.129668.111) | [PMC free article](/articles/PMC3246196/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22147369/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res.&title=Rare%20versus%20common%20variants%20in%20pharmacogenetics:%20SLCO1B1%20variation%20and%20methotrexate%20disposition&author=L.B.%20Ramsey&author=G.H.%20Bruun&author=W.%20Yang&author=L.R.%20Trevi%C3%B1o&author=S.%20Vattathil&volume=22&publication_year=2011&pages=1-8&pmid=22147369&doi=10.1101/gr.129668.111&)

29. PharmGKB Clinical Annotation for SLCO1B1*1, SLCO1B1*5, SLCO1B1*15, SLCO1B1*37; Atorvastatin (Level 1A Metabolism/PK) [(accessed on 5 July 2023)]. Available online: https://www.pharmgkb.org/clinicalAnnotation/1043880630.  [https://www.pharmgkb.org/clinicalAnnotation/1043880630](https://www.pharmgkb.org/clinicalAnnotation/1043880630)

30. PharmGKB Clinical Annotation for SLCO1B1*1, SLCO1B1*5, SLCO1B1*15; Simvastatin or Simvastatin Acid (Level 1A Metabolism/PK) [(accessed on 5 July 2023)]. Available online: https://www.pharmgkb.org/clinicalAnnotation/1451681540.  [https://www.pharmgkb.org/clinicalAnnotation/1451681540](https://www.pharmgkb.org/clinicalAnnotation/1451681540)

31. Oxford Academic Effects of Statins on Skeletal Muscle: A Perspective for Physical Therapists. Physical Therapy. [(accessed on 8 March 2024)]. Available online: https://academic.oup.com/ptj/article/90/10/1530/2737797.  [https://academic.oup.com/ptj/article/90/10/1530/2737797](https://academic.oup.com/ptj/article/90/10/1530/2737797) | [DOI](https://doi.org/10.2522/ptj.20090251) | [PMC free article](/articles/PMC2949584/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20688875/)

32. Fry A., Littlejohns T.J., Sudlow C., Doherty N., Adamska L., Sprosen T., Collins R., Allen N.E. Comparison of Sociodemographic and Health-Related Characteristics of UK Biobank Participants With Those of the General Population. Am. J. Epidemiol. 2017;186:1026–1034. doi: 10.1093/aje/kwx246.  [DOI](https://doi.org/10.1093/aje/kwx246) | [PMC free article](/articles/PMC5860371/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28641372/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Epidemiol.&title=Comparison%20of%20Sociodemographic%20and%20Health-Related%20Characteristics%20of%20UK%20Biobank%20Participants%20With%20Those%20of%20the%20General%20Population&author=A.%20Fry&author=T.J.%20Littlejohns&author=C.%20Sudlow&author=N.%20Doherty&author=L.%20Adamska&volume=186&publication_year=2017&pages=1026-1034&pmid=28641372&doi=10.1093/aje/kwx246&)

33. Bytyçi I., Penson P.E., Mikhailidis D.P., Wong N.D., Hernandez A.V., Sahebkar A., Thompson P.D., Mazidi M., Rysz J., Pella D., et al. Prevalence of statin intolerance: A meta-analysis. Eur. Heart J. 2022;43:3213–3223. doi: 10.1093/eurheartj/ehac015.  [DOI](https://doi.org/10.1093/eurheartj/ehac015) | [PMC free article](/articles/PMC9757867/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35169843/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur.%20Heart%20J.&title=Prevalence%20of%20statin%20intolerance:%20A%20meta-analysis&author=I.%20Byty%C3%A7i&author=P.E.%20Penson&author=D.P.%20Mikhailidis&author=N.D.%20Wong&author=A.V.%20Hernandez&volume=43&publication_year=2022&pages=3213-3223&pmid=35169843&doi=10.1093/eurheartj/ehac015&)

34. Our Future Health. [(accessed on 3 November 2023)]. Available online: https://ourfuturehealth.org.uk/  [https://ourfuturehealth.org.uk/](https://ourfuturehealth.org.uk/)

35. Primary Care Linked Data. [(accessed on 4 June 2023)]. Available online: https://biobank.ndph.ox.ac.uk/ukb/ukb/docs/primary_care_data.pdf.  [https://biobank.ndph.ox.ac.uk/ukb/ukb/docs/primary_care_data.pdf](https://biobank.ndph.ox.ac.uk/ukb/ukb/docs/primary_care_data.pdf)

36. Van Hout C.V., Tachmazidou I., Backman J.D., Hoffman J.D., Liu D., Pandey A.K., Gonzaga-Jauregui C., Khalid S., Ye B., Banerjee N., et al. Exome sequencing and characterization of 49,960 individuals in the UK Biobank. Nature. 2020;586:749–756. doi: 10.1038/s41586-020-2853-0.  [DOI](https://doi.org/10.1038/s41586-020-2853-0) | [PMC free article](/articles/PMC7759458/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33087929/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Exome%20sequencing%20and%20characterization%20of%2049,960%20individuals%20in%20the%20UK%20Biobank&author=C.V.%20Van%20Hout&author=I.%20Tachmazidou&author=J.D.%20Backman&author=J.D.%20Hoffman&author=D.%20Liu&volume=586&publication_year=2020&pages=749-756&pmid=33087929&doi=10.1038/s41586-020-2853-0&)

37. Backman J.D., Li A.H., Marcketta A., Sun D., Mbatchou J., Kessler M.D., Benner C., Liu D., Locke A.E., Balasubramanian S., et al. Exome sequencing and analysis of 454,787 UK Biobank participants. Nature. 2021;599:628–634. doi: 10.1038/s41586-021-04103-z.  [DOI](https://doi.org/10.1038/s41586-021-04103-z) | [PMC free article](/articles/PMC8596853/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34662886/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Exome%20sequencing%20and%20analysis%20of%20454,787%20UK%20Biobank%20participants&author=J.D.%20Backman&author=A.H.%20Li&author=A.%20Marcketta&author=D.%20Sun&author=J.%20Mbatchou&volume=599&publication_year=2021&pages=628-634&pmid=34662886&doi=10.1038/s41586-021-04103-z&)

38. Pilling L.C., Tamosauskaite J., Jones G., Wood A.R., Jones L., Kuo C.-L., Kuchel G.A., Ferrucci L., Melzer D. Common conditions associated with hereditary haemochromatosis genetic variants: Cohort study in UK Biobank. BMJ. 2019;364:k5222. doi: 10.1136/bmj.k5222.  [DOI](https://doi.org/10.1136/bmj.k5222) | [PMC free article](/articles/PMC6334179/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30651232/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMJ&title=Common%20conditions%20associated%20with%20hereditary%20haemochromatosis%20genetic%20variants:%20Cohort%20study%20in%20UK%20Biobank&author=L.C.%20Pilling&author=J.%20Tamosauskaite&author=G.%20Jones&author=A.R.%20Wood&author=L.%20Jones&volume=364&publication_year=2019&pages=k5222&pmid=30651232&doi=10.1136/bmj.k5222&)

39. Ramsey L.B., Gong L., Lee S., Wagner J.B., Zhou X., Sangkuhl K., Adams S.M., Straka R.J., Empey P.E., Boone E.C., et al. PharmVar GeneFocus: SLCO1B1. Clin. Pharmacol. Ther. 2022;113:782–793. doi: 10.1002/cpt.2705.  [DOI](https://doi.org/10.1002/cpt.2705) | [PMC free article](/articles/PMC10900141/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35797228/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=PharmVar%20GeneFocus:%20SLCO1B1&author=L.B.%20Ramsey&author=L.%20Gong&author=S.%20Lee&author=J.B.%20Wagner&author=X.%20Zhou&volume=113&publication_year=2022&pages=782-793&pmid=35797228&doi=10.1002/cpt.2705&)

40. McLaren W., Gil L., Hunt S.E., Riat H.S., Ritchie G.R.S., Thormann A., Flicek P., Cunningham F. The Ensembl Variant Effect Predictor. Genome Biol. 2016;17:122. doi: 10.1186/s13059-016-0974-4.  [DOI](https://doi.org/10.1186/s13059-016-0974-4) | [PMC free article](/articles/PMC4893825/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27268795/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Biol.&title=The%20Ensembl%20Variant%20Effect%20Predictor&author=W.%20McLaren&author=L.%20Gil&author=S.E.%20Hunt&author=H.S.%20Riat&author=G.R.S.%20Ritchie&volume=17&publication_year=2016&pages=122&pmid=27268795&doi=10.1186/s13059-016-0974-4&)

41. GWAS Catalog. [(accessed on 3 November 2023)]. Available online: https://www.ebi.ac.uk/gwas/genes/SLCO1B1.  [https://www.ebi.ac.uk/gwas/genes/SLCO1B1](https://www.ebi.ac.uk/gwas/genes/SLCO1B1)

42. Expert Panel on Detection, Evaluation, and Treatment of High Blood Cholesterol in Adults Executive Summary of the third Report of the National Cholesterol Education Program (NCEP) Expert Panel on Detection, Evaluation, and Treatment of High Blood Cholesterol in Adults (Adult Treatment Panel III) JAMA. 2001;285:2486–2497. doi: 10.1001/jama.285.19.2486.  [DOI](https://doi.org/10.1001/jama.285.19.2486) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11368702/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA&title=Executive%20Summary%20of%20the%20third%20Report%20of%20the%20National%20Cholesterol%20Education%20Program%20(NCEP)%20Expert%20Panel%20on%20Detection,%20Evaluation,%20and%20Treatment%20of%20High%20Blood%20Cholesterol%20in%20Adults%20(Adult%20Treatment%20Panel%20III)&volume=285&publication_year=2001&pages=2486-2497&pmid=11368702&doi=10.1001/jama.285.19.2486&)

43. Stewart A. SLCO1B1 Polymorphisms and Statin-Induced Myopathy. PLoS Curr. 2013;5 doi: 10.1371/currents.eogt.d21e7f0c58463571bb0d9d3a19b82203.  [DOI](https://doi.org/10.1371/currents.eogt.d21e7f0c58463571bb0d9d3a19b82203) | [PMC free article](/articles/PMC3871416/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24459608/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Curr.&title=SLCO1B1%20Polymorphisms%20and%20Statin-Induced%20Myopathy&author=A.%20Stewart&volume=5&publication_year=2013&pmid=24459608&doi=10.1371/currents.eogt.d21e7f0c58463571bb0d9d3a19b82203&)
