# Genetic variants, haplotype determination, and function of novel alleles of CYP2B6 in a Han Chinese population

## Metadata
**Authors:** Li-Qun Zhang, Xin-Yue Li, Lian-Guo Chen, Zhe Chen, Ren-Ai Xu, Jian-Chang Qian, Xiao-yang Zhou, Da-Peng Dai, Guo-Xin Hu, Jian-Ping Cai
**Journal:** Heliyon
**Date:** 2024 Mar 30
**DOI:** [10.1016/j.heliyon.2024.e28952](https://doi.org/10.1016/j.heliyon.2024.e28952)
**PMID:** 38596098
**PMCID:** PMC11002277
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11002277/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC11002277/pdf/main.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC11002277/pdf/main.pdf)

## Abstract

Amino acid variants in protein may result in deleterious effects on enzymatic activity. In this study we investigate the DNA variants on activity of CYP2B6 gene in a Chinese Han population for potential use in precision medicine. All exons in CYP2B6 gene from 1483 Chinese Han adults (Zhejiang province) were sequenced using Sanger sequencing. The effects of nonsynonymous variants on recombinant protein catalytic activity were investigated in vitro with Sf12 system. The haplotype of novel nonsynonymous variants with other single nucleotide variants in the same allele was determined using Nanopore sequencing. Of 38 alleles listed on the Pharmacogene Variation Consortium, we detected 7 previously reported alleles and 18 novel variants, of which 11 nonsynonymous variants showed lower catalytic activity (0.00–0.60) on bupropion compared to CYP2B6*1. Further, these 11 novel star-alleles (CYP2B6*39–49) were assigned by the Pharmacogene Variation Consortium, which may be valuable for pharmacogenetic research and personalized medicine.

Keywords: CYP2B6, Genetic variant, Haplotype, Catalytic activity

## 1. Introduction

*Cytochrome P450s (CYPs)* are considered as the most important drug metabolizing enzymes in phase I metabolism. Human *CYPs* are categorized into 18 families and 57 genes [[1](#bib1),[2](#bib2)]. Fifteen *CYP* enzymes in families 1–4 are responsible for approximately 90% of phase I metabolism of clinical drugs [[3](#bib3),[4](#bib4)]. *CYP2B6* activity varied among human microsomes by 25 and 80 fold in the metabolism of S-mephenytoin [[5](#bib5)] and bupropion [[6](#bib6)], respectively. Such variations in *CYP2B6* activities may lead to variable blood exposure and therapeutic efficacy to drugs mainly metabolized by *CYP2B6*.

*CYP2B6* and related *CYP2B7* (pseudogene) are located in *CYP2* gene loci on chromosome 19 [[7](#bib7)]. The *CYP2B6* gene is comprised of 9 exons and encodes a 491 amino acids enzyme, which is the only functional isozyme of *CYP2B* [[2](#bib2)]. Due to the high genetic variant and inhibitors or inducers, activity of *CYP2B6* is intensively variable among individuals. For clinical drugs such as efavirenz (EFV) and nevirapine, which are used in treatment of human immunodeficiency virus (HIV), *CYP2B6* single nucleotide polymorphisms (SNPs) were shown to be useful in predicting drug responses [[8](#bib8),[9](#bib9)]. Data from Chinese HIV patients (n = 166) who had been genotyped for *CYP2B6* and tested for blood EFV levels at steady state showed trough levels for TT, GT and GG genotypes were 9.08, 3.57, and 2.94 mg/L, respectively, for variant of G516T [[10](#bib10)]. Different efficacy of EFV-based HIV therapy and other *CYP2B6* substrates may resulted from variations of the *CYP2B6* frequencies due to highly variable genetic differences among racial and ethnic populations [[11](#bib11)]. Guan et al. found that the frequencies of common *CYP2B6* variants in a Chinese Han population (n = 193) are similar to those of other Asians including Korean and Japanese, but highly different from those of Caucasians [[12](#bib12)]. Herein, we sought to investigate on the DNA variant of *CYP2B6* in a healthy Chinese Han population (n = 1483) and determine the catalytic activity of novel amino acid changes on bupropion, which may be valuable for pharmacogenetic research and personalized medicine.

## 2. Materials and methods

### 2.1. Subjects and samples

The study consisted of 1483 unrelated Han Chinese subjects (>18 y) who lived in Wenzhou (Zhejiang Province, China). This study was approved by the Institutional Ethical Committee of Beijing Hospital. All participants provided informed written consent before blood sample collection. DNA was extracted from whole blood cells using magnetic beads (Applied Biosystems, CA, USA).

### 2.2. DNA sequencing

The 9 exons and flanking introns (over 50 bp) of the *CYP2B6* were amplified by PCR with the primers listed in [Supplementary Table 1](#appsec1). After purification, the amplicons were sequenced on an Applied Biosystems 3730XL DNA Analyzer (Applied Biosystems). Acquired chromatograms of the sequences were verified manually by at least two individuals. Amplicons with inconsistent sequences relative to *CYP2B6*1* ([NG_007929.1](https://www.ncbi.nlm.nih.gov/nuccore/NG_007929.1)) were confirmed by bidirectional sequencing.

### 2.3. Data analysis

Chromatogram files were analyzed on Snapgene (version4.3.6, Illinois, USA) using *CYP2B6*1* ([NG_007929.1](https://www.ncbi.nlm.nih.gov/nuccore/NG_007929.1)) as the template. Variants that were not listed on the Pharmacogene Variation Consortium (PharmVar) were searched in the SNP database ([https://www.ncbi.nlm.nih.gov/snp/](https://www.ncbi.nlm.nih.gov/snp/)) on National Center for Biotechnology Information (NCBI) to confirm they were novel. The effects of novel nonsynonymous variants on the structure and activity of *CYP2B6* were analyzed with online tools PolyPhen-2 ([http://genetics.bwh.harvard.edu/pph2/](http://genetics.bwh.harvard.edu/pph2/)) and PROVEAN ([https://www.jcvi.org/research/provean](https://www.jcvi.org/research/provean)).

### 2.4. Haplotype sequencing

For nonsynonymous variants, the entire *CYP2B6* (30,957 bp) gene was first amplified with KOD OneTM PCR Master Mix (TOYOBO, Japan) and then purified with 0.6 × AMPure XP beads (Beckman Coulter, High Wycombe, UK). A DNA A-tailing Kit (Takara, Japan) was used for dA-tailing for 20 min at 72 °C. The A-tailed DNA amplicons were purified by 0.6 × AMPure XP beads and then ligated to barcodes (EXP-NBD104, ONT, UK). After purification, adapters were ligated to the DNA ends with the SQK-LSK110 ligation kit (ONT, Oxford, UK) and the samples were purified again with 0.6 × volume of AMPure XP beads. The adapted library was quantified with a Qubit fluorometer and 50–100 fmol DNA library was loaded onto a GridION flow-cell (R9.4.1) according to manufacturer instructions and run for 17.5 h. The raw sequencing data was base called using the MinKNOW software in default mode. The reads, in fastq format, were mapped to the human reference genome GRCh38 using Minimap2. The alignment data were transferred to bam format and sorted using samtools. The sorted and indexed bam data were viewed with IGV_2.14.0 and the haplotype of each SNP was determined manually ([Supplementary Fig. 1](#appsec1)). The novel haplotypes were submitted to DNA Data Bank of Japan and PharmVar for nomenclature.

### 2.5. Expression of CYP2B6 novel variant proteins in Sf21 cells and in vitro function analysis with UPLC-MS/MS

Briefly, the whole length of *CYP2B6* ORF was synthesized and constructed into pFastbac (Thermo) by BGI laboratory (Beijing, China). Pointed nucleotide variant was introduced with double-reversed PCR and confirmed by Sanger sequencing. Recombinant baculoviruses with high expression of both *CYP2B6* and *CYP oxidoreductase (OR)* proteins were obtained from infection of Sf21 cells as previously described method for study on *CYP2C19* variants [[13](#bib13)]. Microsomal fractions (2 μg) were separated on 10% (w/v) sodium dodecyl sulfate-polyacrylamide (SDS-PAGE) gels and western blotting was carried out for quantification of *CYP2B6* proteins. Antibodies for *CYP2B6* and *OR* were purchased from Abcam (Cambridge, UK).

Bupropion hydrochloride, midazolam and hydroxybupropion were purchased from J&K Scientific Ltd. (Hong Kong, China). Briefly, 200 μL incubation buffer containing 100 mM Tris-HCl (pH 7.4), 0.1 mg/mL *CYP2B6*1* or other novel variants, 2.7 mg/mL CYB5 and 100 mM bupropion was gently mixed and incubated for 5 min at 37 °C. Next, 1 mM NADPH was added in the solution and incubated for 40 min before transferring to −80 °C to terminate the reaction. Subsequently, 200 μL acetonitrile and 20 μL Midazolam (500 ng/mL, as internal standard) were added and vortexed for 2 min. Samples were then centrifuged at 13,000×*g* for 10 min at 4 °C and the supernatants were analyzed with UPLC-MS/MS as previously described [[14](#bib14)]. The relative ratio of hydroxybupropion to bupropion hydrochloride was calculated to assess the enzyme activities of *CYP2B6*1* and 11 novel variants. These experiments were performed in triplicate.

For Vmax/Km analysis, 1, 2, 5, 10, 20 and 50 μM of efavirenz were added in 200 μL incubation buffer containing 1 × PBS buffer, 20 μg *CYP2B6*1* or other *CYP2B6* variants, 25 μg/ml cytochrome *b*5, 1 mM NADPH. The mixture without NADPH was pre-incubated at 37 °C for 5 min; then, 1 mM of NADPH was added to initiate the reaction. After incubation for 40 min, the reaction was immediately terminated by cooling to −80 °C. Then, 200 μl of acetonitrile and 20 μl of telmisartan (10 μg/ml) as an internal standard, were added to the mixture. After being vortexed for 2 min and centrifuged at 13,000×*g* for 10 min, the supernatant was analyzed by UPLC-MS/MS to detect efavirenz and 8-OH efavirenz. The analytes were firstly separated on a BEH C18 column (2.1 × 100 mm,1.7 μm; Waters Corp., Millipore, Bedford, MA, USA), which incubated at 40 °C. The mobile phase was consisted of 0.1% formic acid-water (A) and methanol (B), and elution at 0.40 mL/min for 4.5 min with a gradient condition. The program was set as 10–90% methanol (0–2.5 min), 90-10% methanol (2.5–2.8 min), and 10% methanol (2.8–4.5 min). The monitoring transitions were *m*/*z* 329.9 → 257.9 for 8-OH efavirenz.

### 2.6. Statistical analyses

One-way analysis of variance (ANOVA) test was performed for multiple comparisons. Student's *t*-test was performed for two group comparisons. Significant level was P < 0.05.

## 3. Results

### 3.1. Allele frequencies

To investigate the overall variants of the *CYP2B6* gene in the Chinese population, 1483 healthy subjects in the south of China were recruited and 9 exons of their *CYP2B6* gene were sequenced. The results showed that there were 7 previously reported allelic variants in PharmVar and 18 novel variants identified in the present study. These include 11 nonsynonymous and 7 synonymous variants ([Table 1](#tbl1)). Additionally, most variants, except for c.12776A > T and c.18008A > C, have been reported in the library of Reference SNPs.

### Table 1.

| Exon | GRCh38.p13 position | Coding DNA position | Complementary DNA position | Effect on protein | Hit | Frequency (%) | Frequency in 1000 Genomes Project (%) | Reference SNP | Recorded on PharmVar |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| 1 | 40991369 | 64C > T | 64C > T | R22C | 109 | 3.67 | 4.80 | rs8192709 | yes |
| 2 | 41004045 | 12740G > C | 216G > C | / | 76 | 2.56 | 5.20 | rs2279341 | yes |
| 2 | 41004081 | 12776A > T | 252A > T | / | 1 | 0.03 | – | NA | no |
| 2 | 41004133 | 12828G > A | 304G > A | A102T | 1 | 0.03 | – | rs148009906 | no |
| 3 | 41004380 | 13075C > T | 418C > T | R140W | 3 | 0.10 | <0.10 | rs535039125 | no |
| 4 | 41006919 | 15614C > G | 499C > G | P167A | 10 | 0.34 | <0.10 | rs3826711 | yes |
| 4 | 41006936 | 15631G > T | 516G > T | Q172H | 546 | 18.41 | 31.60 | rs3745274 | yes |
| 4 | 41006967 | 15662G > A | 547G > A | V183I | 1 | 0.03 | 0.10 | rs58871670 | no |
| 5 | 41009313 | 18008A > C | 740A > C | H247P | 1 | 0.03 | – | NA | no |
| 5 | 41009358 | 18053A > G | 785A > G | K262R | 788 | 26.57 | – | rs2279343 | yes |
| 7 | 41012393 | 21088T > C | 1060T > C | Y354H | 1 | 0.03 | – | rs754621576 | no |
| 7 | 41012394 | 21089A > G | 1061A > G | Y354C | 1 | 0.03 | – | rs780991919 | no |
| 7 | 41012466 | 21161G > A | 1133G > A | R378Q | 2 | 0.07 | – | rs200458614 | no |
| 7 | 41012471 | 21166T > C | 1138T > C | Y380H | 12 | 0.40 | <0.10 | rs201500445 | no |
| 7 | 41012473 | 21168C > T | 1140C > T | / | 1 | 0.03 | – | rs374509631 | no |
| 7 | 41012478 | 21173T > A | 1145T > A | I382 N | 6 | 0.20 | <0.10 | rs200238771 | no |
| 8 | 41012739 | 21434C > T | 1218C > T | / | 1 | 0.03 | <0.10 | rs35661880 | no |
| 9 | 41016652 | 25347G > A | 1301G > A | R434Q | 1 | 0.03 | – | rs764288403 | no |
| 9 | 41016677 | 25372C > T | 1326C > T | / | 1 | 0.03 | – | rs1476466408 | no |
| 9 | 41016679 | 25374G > A | 1328G > A | R443H | 1 | 0.03 | – | rs374099483 | no |
| 9 | 41016686 | 25381A > G | 1335A > G | / | 1 | 0.03 | <0.10 | rs146956370 | no |
| 9 | 41016726 | 25421A > G | 1375A > G | M459V | 8 | 0.27 | <0.10 | rs3211369 | yes |
| 9 | 41016741 | 25436G > A | 1390G > A | A464T | 5 | 0.17 | <0.10 | rs117872433 | no |
| 9 | 41016743 | 25438C > T | 1392C > T | / | 8 | 0.27 | – | rs200993638 | no |
| 9 | 41016746 | 25441A > T | 1395A > T | / | 8 | 0.27 | – | rs3211370 | no |
| 9 | 41016810 | 25505C > T | 1459C > T | R487C | 5 | 0.17 | 5.40 | rs3211371 | yes |

Table 1 Caption: Single nucleotide variation (SNV) found in coding region of CYP2B6 and their frequencies in 1483 Chinese Han subjects.

As illustrated in [Table 1](#tbl1), the first three common variants were c.18053A > G (26.57%), c.15631G > T (18.41% compared to 31.6% in the 1000 Genomes Project) and c.64C > T (3.67% comparing with 4.8% in the 1000 Genomes Project). Allele frequency analysis showed that most individuals carried the wild-type *CYP2B6* allele (*CYP2B6*1*, 74.68%), and the first three common alleles were *CYP2B6*6* (combination of c.15631G > T and c.18053A > G, 12.37%), *CYP2B6*4* (c.18053A > G, 6.54%) and *CYP2B6*2* (c.64C > T, 4.28%) as shown in [Table 2](#tbl2). Additionally, four other alleles, *CYP2B6*5* (c.25505C > T), *CYP2B6*9* (c.15631G > T), *CYP2B6*23* (c.25421A > G) and *CYP2B6*26* (c.15614C > G, c.15631G > T, c.18053A > G) were also found in this study, with frequencies of no more than 0.4%. Moreover, 11 other novel alleles were reported for the first time and assigned star-allele names by PharmVar (*CYP2B6*39–49*).

### Table 2.

| Allele | Position (coding DNA) | Effect on protein | Hit | Frequency |
| --- | --- | --- | --- | --- |
| CYP2B6*1 | / | / | 2215 | 0.7468 |
| CYP2B6*2 | 64C > T | R22C | 127 | 0.0428 |
| CYP2B6*4 | 18053A > G | K262R | 194 | 0.0654 |
| CYP2B6*5 | 25505C > T | R487C | 5 | 0.0017 |
| CYP2B6*6 | 15631G > T and 18053A > G | Q172H and K262R | 367 | 0.1237 |
| CYP2B6*9 | 15631G > T | Q172H | 4 | 0.0013 |
| CYP2B6*23 | 25421A > G | M459V | 8 | 0.0027 |
| CYP2B6*26 | 15614C > G, 15631G > T, 18053A > G | P167A, Q172H and K262R | 11 | 0.0037 |
| CYP2B6*39 | 13075C > T, 15631G > T and 18053A > G | R140W, Q172H and K262R | 3 | 0.0010 |
| CYP2B6*40 | 21161G > A, 15631G > T and 18053A > G | R378Q, Q172H and K262R | 2 | 0.0007 |
| CYP2B6*41 | 21166T > C, 15631G > T and 18053A > G | Y380H, Q172H and K262R | 12 | 0.0040 |
| CYP2B6*42 | 25374G > A, 15631G > T and 18053A > G | R443H, Q172H and K262R | 1 | 0.0003 |
| CYP2B6*43 | 25436G > A, 15631G > T and 18053A > G | A464T, Q172H and K262R | 5 | 0.0017 |
| CYP2B6*44 | 12828G > A | A102T | 1 | 0.0003 |
| CYP2B6*45 | 15662G > A | V183I | 1 | 0.0003 |
| CYP2B6*46 | 18008A > C | H247P | 1 | 0.0003 |
| CYP2B6*47 | 21088T > C and 21089A > G | Y354R | 1 | 0.0003 |
| CYP2B6*48 | 21173T > A | I382 N | 7 | 0.0024 |
| CYP2B6*49 | 25347G > A | R434Q | 1 | 0.0003 |

Table 2 Caption: Allele frequencies of CYP2B6 variants detected in 1483 Chinese Han subjects.

### 3.2. Comparison of allele frequency in different biogeographical groups

The allele frequency of *CYP*2B6 in the Han Chinese population was compared to that of different biogeographical groups as illustrated in [Table 3](#tbl3). The allele frequencies of *CYP2B6* are similar to those of other Asian populations, such as Central/South Asian and East Asian, but quite different from that of European, Latino, African American/Afro-Caribbean, Near Eastern, Oceanian and Sub-Saharan African. However, *CYP2B6*1* and *CYP2B6*6* are the most prevalent alleles for all the groups.

### Table 3.

| Allele |   | Frequency |  |  |  |  |  |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Position (coding DNA) | Current study | African American/Afro-Caribbean | Central/South Asian | East Asian | European | Latino | Near Eastern | Oceanian | Sub-Saharan African |  |
| CYP2B6*1 | / | 0.7468 | 0.4135 | 0.6085 | 0.6400 | 0.4907 | 0.4974 | 0.4760 | 0.3400 | 0.3147 |
| CYP2B6*2 | 64C > T | 0.0428 | 0.0310 | 0.0410 | 0.0458 | 0.0492 | 0.0440 | 0.0390 | 0.0000 | 0.0310 |
| CYP2B6*4 | 18053A > G | 0.0654 | 0.0103 | 0.0990 | 0.0842 | 0.0409 | 0.1062 | 0.1040 | 0.0000 | 0.0000 |
| CYP2B6*5 | 25505C > T | 0.0017 | 0.0621 | NA | 0.0110 | 0.1155 | 0.0381 | 0.0240 | 0.0200 | 0.0200 |
| CYP2B6*6 | 15631G > T，18053A > G | 0.1237 | 0.3170 | 0.1850 | 0.1745 | 0.2330 | 0.2116 | 0.2310 | 0.6200 | 0.3749 |
| CYP2B6*9 | 15631G > T | 0.0013 | 0.0465 | 0.0590 | 0.0344 | 0.0147 | 0.0727 | 0.1020 | 0.0100 | NA |
| CYP2B6*23 | 25421A > G | 0.0027 | 0.0000 | 0.0000 | 0.0021 | 0.0001 | 0.0000 | NA | NA | 0.0000 |

Table 3 Caption: Allele frequencies of CYP2B6 variants in this study and other biogeographical populations (from PharmVar).

### 3.3. Genotype frequencies

Genotype analyses ([Table 4](#tbl4)) showed that the first three common genotypes were *CYP2B6*1/*1* (56.04%), *CYP2B6*1/*6* (18.41%) and *CYP2B6*1/*4* (10.92%). Additionally, *CYP2B6*1/*2*, *CYP2B6*2/*2*, *CYP2B6*2/*4*, *CYP2B6*4/*4*, *CYP2B6*1/*5*, *CYP2B6*2/*6*, *CYP2B6*6/*6*, *CYP2B6*1/*9*, *CYP2B6*1/*23* and *CYP2B6*1/*26* were also detected with frequencies of no more than 9.0%. 35 individuals had genotypes of *CYP2B6*1/39–49* (3.98%).

### Table 4.

| Genotype | Hit | Frequency (%) | Phenotype Summary | Electronic Health Record (HER) Priority Notation |
| --- | --- | --- | --- | --- |
| CYP2B6*1/*1 | 776 | 52.33 | Normal Metabolizer | Normal/Routine/Low Risk |
| CYP2B6*1/*2 | 125 | 8.43 | Normal Metabolizer | Normal/Routine/Low Risk |
| CYP2B6*2/*2 | 1 | 0.07 | Normal Metabolizer | Normal/Routine/Low Risk |
| CYP2B6*1/*4 | 162 | 10.92 | Rapid Metabolizer | Normal/Routine/Low Risk |
| CYP2B6*2/*4 | 7 | 0.47 | Rapid Metabolizer | Normal/Routine/Low Risk |
| CYP2B6*4/*4 | 16 | 1.08 | Ultrarapid Metabolizer | Normal/Routine/Low Risk |
| CYP2B6*1/*5 | 5 | 0.34 | Normal Metabolizer | Normal/Routine/Low Risk |
| CYP2B6*1/*6 | 273 | 18.41 | Intermediate Metabolizer | Abnormal/Priority/High Risk |
| CYP2B6*2/*6 | 13 | 0.88 | Intermediate Metabolizer | Abnormal/Priority/High Risk |
| CYP2B6*6/*6 | 47 | 3.17 | Poor Metabolizer | Abnormal/Priority/High Risk |
| CYP2B6*1/*9 | 4 | 0.27 | Intermediate Metabolizer | Abnormal/Priority/High Risk |
| CYP2B6*1/*23 | 8 | 0.54 | Indeterminate | none |
| CYP2B6*1/*26 | 11 | 0.74 | Intermediate Metabolizer | Abnormal/Priority/High Risk |
| CYP2B6*1/*39 | 3 | 0.20 | NA | NA |
| CYP2B6*1/*40 | 2 | 0.13 | NA | NA |
| CYP2B6*1/*41 | 12 | 0.81 | NA | NA |
| CYP2B6*1/*42 | 1 | 0.07 | NA | NA |
| CYP2B6*1/*43 | 5 | 0.34 | NA | NA |
| CYP2B6*1/*44 | 1 | 0.07 | NA | NA |
| CYP2B6*1/*45 | 1 | 0.07 | NA | NA |
| CYP2B6*2/*46 | 1 | 0.07 | NA | NA |
| CYP2B6*1/*47 | 1 | 0.07 | NA | NA |
| CYP2B6*1/*48 | 7 | 0.47 | NA | NA |
| CYP2B6*1/*49 | 1 | 0.07 | NA | NA |

Table 4 Caption: Genotype frequencies and phenotypes of CYP2B6 in 1483 Chinese Han subjects.

### 3.4. Haplotype determination of SNP

The haplotype of the novel nonsynonymous variant and all other variants in promoter, intron and untranslated region (UTR) in each sample was not clear because of the limitation of the Sanger sequencing method. Nanopore sequencing was then utilized to distinguish the haplotype of the novel variants both in coding and non-coding regions. The entire *CYP2B6* (30957 bp) gene was amplified with long-fragment PCR and finally sequenced with GridION flow-cell (R9.4.1). The haplotypes of the 11 novel nonsynonymous variants and variants in non-coding regions were determined finally ([Supplementary Fig. 2](#appsec1)), and the star allele names of these haplotypes were assigned as *CYP2B6*39–*49 by PharmVar. These results indicated that long-PCR followed by long-read sequencing was a useful tool to distinguish the haplotype of long length genes such as *CYP2B6*.

### 3.5. Functional prediction of novel variants with online PROVEAN and PolyPhen-2

Each of the 11 novel nonsynonymous variants in the *CYP2B6* gene was analyzed using the online tools PROVEAN and PolyPhen-2. Based on the analysis, it was found that the function of variants p.A102T, p.H247P, p.Y380H, p.I382 N, p.R434Q, and p.R443H was predicted to be damaged by both PROVEAN and PolyPhen-2. On the other hand, variants p.V183I, p.Y354H, and p.A464T were predicted to have a "Tolerated" or "Benign" function. The function of variants p.R140W and p.R378Q showed inconsistent predictions between PROVEAN and PolyPhen-2 ([Supplementary Table 3](#appsec1)).

### 3.6. In vitro enzymatic activities of CYP2B6 variants

A total of 11 novel nonsynonymous variants in the coding regions of the *CYP2B6* gene were identified in this study. The enzyme characteristics of these novel variants expressed in Sf21 cells were assessed with UPLC-MS/MS using bupropion and efavirenz respectively as the substrate. The relative ratio of hydroxybupropion to bupropion hydrochloride and relative clearance rate (Vmax/Km) on efavirenz were calculated to assess the enzyme activities of *CYP2B6*1* (*in vitro* activity was set as “1”) and 11 novel variants. The results showed that all the novel variants (p.A102T in **44*, p.R140W in **39*, p.V183I in **45*, p.H247P in **46*, p.Y354R in **47*, p.R378Q in **40*, p.Y380H in **41*, p.I382 N in **48*, p.R434Q in **49*, p.R443H in **42*, and p.A464T in **43*) exhibited significant lower activities than the reference *CYP2B6*1* ([Fig. 1](#fig1). A, B and C). As a result, patients with these variants may require a lower oral dose of bupropion to achieve the desired concentration in the bloodstream and therapeutic effects.

### Fig. 1.

![Fig. 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/cc86/11002277/2ee38fb191c8/gr1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=11002277_gr1.jpg)

In vitro enzymatic activities of CYP2B6 variants. A. Protein expression of CYPOR and CYP2B6 in each expression vector assessed by western blotting (Original images of gels and blots were in Supplementary Fig. 3). *1, wild type enzyme. C, control (empty vector). *38, a truncated enzyme. p.A102T, p.R140W, p.V183I, p.H247P, p.Y354H, p.R378Q, p.Y380H, p.I382 N, p.R434Q, p.R443H and p.A464T were the novel variants found in our study. B. Relative metabolic rate (hydroxybupropion to bupropion hydrochloride) of each variant comparing with wide type *1. C. Relative clearance rate (Vmax/Km) of each variant on efavirenz comparing with wide type *1. ND, not detectable. Student's t-test, *P < 0.05, **P < 0.01.

## 4. Discussion

*CYP2B6* accounts for 2–5% of total *CYP* content in liver and is thought to metabolize about 8% of clinical drugs, including bupropion, EFV, nevirapine, meperidine, methadone, ketamine, propofol, mephobarbital, and cyclophosphamide [[[15]](#bib15), [[16]](#bib16), [[17]](#bib17)]. Owing to genetic variants, inducibility or inhibition by many compounds, *CYP2B6* activity is highly variable between individuals. Some variants of *CYP2B6* affect transcriptional regulation, splicing, protein expression and thus catalytic activity.

The PharmVar website lists 38 star-alleles, each of which has a variant amino acid sequence or demonstrates functional effect (Last Updated: Jun 1st, 2022). According to the Clinical Pharmacogenetics Implementation Consortium (CPIC) Guidelines for *CYP2B6* and EFV-containing antiretroviral therapy, the *CYP2B6* phenotype can be classified into 5 categories as ultrarapid, rapid, normal, intermediate and poor metabolizer [[18](#bib18)]. According to CPIC Guidelines for *CYP2B6* variants, 63.53 % of Han Chinese population are normal metabolizers (Normal/Routine/Low Risk), 3.17% are poor metabolizers (Abnormal/Priority/High Risk), 20.84% are intermediate metabolizers (Abnormal/Priority/High Risk), 11.39% are rapid metabolizers (Normal/Routine/Low Risk), and 1.08% are ultrarapid metabolizers (Normal/Routine/Low Risk). The remaining 2.37% remained undefined ([Table 4](#tbl4)). Importantly, the “Low Risk” assigned to rapid and ultrarapid metabolizers indicates that the blood concentration of the drug may not be higher than normal, but there is a real risk of therapy failure and drug resistance because of lower drug concentrations in circulation.

Dose adjustment based on genotype demonstrated that the dose of EFV could be reduced successfully and side effects could also be decreased [[19](#bib19),[20](#bib20)]. A prior dose reduction in homozygous *CYP2B6*6* (c.516G > T and c.785A > G) patients could maintain the drug exposure in the therapeutic range [[21](#bib21)]. An *in vitro* study in mammalian cells showed that the c.516G > T variant resulted in erroneous splicing and lower expression of mRNA [[22](#bib22)]. One hypothesis suggested that the binding of splice factors to exon 4 were affected by this variant [[23](#bib23)]. All haplotypes harboring the variant c.516G > T were designated as decreased enzymatic activity (*CYP2B6*6*, **7*, **9*, **19*, **20*, **26*, **34*, and **36*) or no activity owing to the presence of other variants such as *CYP2B6*13*, **37*, and ***38 by the CPIC.

*CYP2B6*6* functional differences were also affected by variants-related inhibitors. The c.785A > G variant resulted in lower susceptibility to inhibitors such as sertraline and clopidogrel compared to *CYP2B6*1* [[24](#bib24)]. Methadone dose requirements for effective treatment of opioid addiction was shown to be signiﬁcantly reduced in carriers of this genotype [[25](#bib25)]. Further *in vivo* investigation is needed to confirm the role of genetic variants in drug–drug interaction of *CYP2B6*. Increased *in vitro* transcription [[26](#bib26)] and increased activity *in vivo* [[27](#bib27)] were observed in *CYP2B6*22* allele (c.-82T > C), which is a gain-of-function variant. Previous reports [[26](#bib26)] suggest that a c.-82T > C variant could alter the TATA-box into CCAAT, an enhancer-binding site to increase transcription from an alternative initiation site.

Some genetic studies on Han Chinese population have been performed on genetic variants of *CYP2B6* in Guangdong (unrelated healthy subjects, n = 193) [[12](#bib12)], Shanghai (HIV-infected outpatients, n = 376) [[28](#bib28)], Beijing (unrelated healthy subjects, n = 223) [[29](#bib29)] and Shandong (unrelated healthy subjects, n = 179) [[30](#bib30)]. Most of those studies did not focus on the effects of the variants on the function of *CYP2B6* and its metabolism of specific drugs. In this study, based on the Sanger sequencing method and a large number of subjects (n = 1483), a total of 11 nonsynonymous (c.12828G > A, 0.03%; c.13075C > T, 0.1%; c.15662G > A, 0.03%; c.18008A > C, 0.03%; c.21088T > C and c.21089A > G, 0.03%; c.21161G > A, 0.07%; c.21166T > C, 0.4%; c.21173T > A, 0.24%; c.25347G > A, 0.03%; c. 25374G > A, 0.03%; c. 25436G > A, 0.17%) and 7 synonymous (c. 12776A > T, 0.03%; c. 21168C > T, 0.03%; c. 21434C > T, 0.03%; c. 25372C > T, 0.03%; c. 25381A > G, 0.03%; c. 25438C > T, 0.27%; c. 25441A > T, 0.27%) variants in the coding region of the *CYP2B6* gene were detected and most of them are rare variants. To further investigate the enzymatic activity of these novel variants *in vitro*, wild-type *CYP2B6*1* and 11 novel nonsynonymous variants were expressed in Sf12 cells, and the enzymatic activities of these variants towards bupropion and efavirenz were investigated. As showed in [Fig. 1](#fig1)B and C, all 11 variants (p.A102T in **44*, p.R140W in **39*, p.V183I in **45*, p.H247P in **46*, p.Y354R in **47*, p.R378Q in **40*, p.Y380H in **41*, p.I382 N in **48*, p.R434Q in **49*, p.R443H in **42*, and p.A464T in **43*) exhibited significantly lower activity compared to *CYP2B6*1* on both bupropion and efavirenz (P < 0.05). Additionally, when using the online prediction tools PROVEAN and PolyPhen-2, it was observed that over half of these novel variants (p.A102T, p.H247P, p.Y380H, p.I382 N, p.R434Q, and p.R443H) were predicted to have damaged functions by both tools ([Supplementary Table 3](#appsec1)).

According to CPIC, the single variant c.18053A > G (**4*) leads to increased function, while the single variant c.15631G > T (**9*) results in decreased function. When these variants are combined, such as in the case of c.18053A > G (p.K262R) and c.15631G > T (p.Q172H), the overall function is decreased, indicating that c.15631G > T plays a major role in enzymatic activity. All alleles carrying variants c.15631G > T and c.18053A > G exhibit lower activity (**6*, **7*, **9*, **19*, **20*, **26*, **34*, and **36*), except for **13*, **37*, and **38*, which show no activity. In our study, we identified five novel variants (**39-*43*) that carry these two variants along with the novel base variant. Each of these individual novel variants in **39-*43* demonstrated lower enzymatic activity *in vitro* experiments. It is likely that **39-*43* would exhibit even lower or no activity when expressed as triple variants, as compared to **1*. Since c.15631G > T (18.41%) and c.18053A > G (26.57%) have a high frequency and play a significant role in determining the function of *CYP2B6*, we recommend testing these two sites in a clinical setting. It would be beneficial if they could be distinguished in trans or in cis simultaneously using advanced methods such as nanopore sequencing.

Due to differences among expression systems and substrates used *in vitro* enzymatic activity analysis may produce inconsistent results [[17](#bib17),[31](#bib31),[32](#bib32)]. For example, the *CYP2B6*6* allele is associated with enhanced cyclophosphamide metabolism, but appears to confer decreased EFV metabolism [[17](#bib17),[33](#bib33)]. (S)-methadone plasma levels in *CYP2B6*6/*6* carriers were increased and resulted in methadone associated deaths or higher risk of severe cardiac arrhythmias [[34](#bib34),[35](#bib35)]. Thus, functional data should be extrapolated with caution except for true no function alleles such as *CYP2B6*38* which has a premature stop codon and results in a truncated protein. The *in vivo* enzymatic activities of these 11 variants should be further investigated in future studies.

Because *CYP2B6* variants have been shown to affect catalytic activity, splicing, transcriptional regulation and protein expression, sequencing of the entire gene (including promoter, 5′UTR, intron and 3′UTR) was needed to investigate the relationship between different variants in different positions (haplotype and genotype). In this study, we employed the Sanger method to perform bidirectional sequencing of the promoter region, all exonic regions, and the 3′UTR region of the CYP2B6 gene in 1483 study subjects. This is, to our knowledge, the largest-scale fine sequencing of the CYP2B6 single gene conducted in China. More importantly, for the newly discovered variants, we used nanopore sequencing to further distinguish the haplotypes of these new variants, achieving, for the first time, full-length haplotype sequencing of the CYP2B6 gene at >30 kb level, which was almost unattainable with previous sequencing methods. The whole *CYP2B6* gene (30957 bp) was amplified with long-fragment PCR and finally sequenced using a GridION flow-cell (R9.4.1). In this way, we have determined the haplotypes of the 11 novel nonsynonymous variants and variants in non-coding regions ([Supplementary Fig. 2](#appsec1) and [Supplementary Table 2](#appsec1)). The results show that the relationship between different variants in the whole gene were distinguished, including both haplotype and genotype, which may provide robust evidence for metabolizer level prediction and decrease the gap between gene diagnosis and clinical phenotype.

## 5. Ethics declaration

This study was reviewed and approved by Institutional Review Boards of Beijing Hospital, with the approval number: 2019BJYYEC-054-02.

All participants provided informed consent to participate in the study.

## Funding

This work was supported by Beijing Hospital Project (BJ-2019-186) and National Key R&D Program of China (2020YFC2008300).

## Data availability statement

Sequencing data of novel variants in this study has been deposited into DNA Data Bank of Japan and National Center for Biotechnology Information with accession numbers of [LC761190](https://www.ncbi.nlm.nih.gov/nuccore/LC761190), [LC761191](https://www.ncbi.nlm.nih.gov/nuccore/LC761191), [LC761192](https://www.ncbi.nlm.nih.gov/nuccore/LC761192), [LC761193](https://www.ncbi.nlm.nih.gov/nuccore/LC761193), [LC761194](https://www.ncbi.nlm.nih.gov/nuccore/LC761194), [LC761195](https://www.ncbi.nlm.nih.gov/nuccore/LC761195), [LC761196](https://www.ncbi.nlm.nih.gov/nuccore/LC761196), [LC761197](https://www.ncbi.nlm.nih.gov/nuccore/LC761197), [LC761198](https://www.ncbi.nlm.nih.gov/nuccore/LC761198), [LC761199](https://www.ncbi.nlm.nih.gov/nuccore/LC761199) and [LC761200](https://www.ncbi.nlm.nih.gov/nuccore/LC761200). Further experimental details and data are available upon request.

## CRediT authorship contribution statement

**Li-Qun Zhang:** Writing – original draft, Formal analysis. **Xin-Yue Li:** Methodology. **Lian-Guo Chen:** Resources, Data curation. **Zhe Chen:** Resources, Methodology. **Ren-Ai Xu:** Data curation. **Jian-Chang Qian:** Validation, Methodology. **Xiao-yang Zhou:** Investigation, Data curation. **Da-Peng Dai:** Funding acquisition. **Guo-Xin Hu:** Supervision. **Jian-Ping Cai:** Supervision, Funding acquisition, Conceptualization.

## Declaration of competing interest

The authors declare the following financial interests/personal relationships which may be considered as potential competing interests: Da-peng Dai reports financial support was provided by Ministry of Science and Technology of the People's Republic of China. Da-Peng Dai reports a relationship with Ministry of Science and Technology of the People's Republic of China that includes: funding grants.

## 5. Acknowledgements

Thanks a lot for PhD. Gao-Yuan Sun and professor Fei Xiao from Beijing Hospital in providing the platform of nanopore sequencing.

## Footnotes

## Appendix ASupplementary data

The following are the Supplementary data to this article.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

Sequencing data of novel variants in this study has been deposited into DNA Data Bank of Japan and National Center for Biotechnology Information with accession numbers of [LC761190](https://www.ncbi.nlm.nih.gov/nuccore/LC761190), [LC761191](https://www.ncbi.nlm.nih.gov/nuccore/LC761191), [LC761192](https://www.ncbi.nlm.nih.gov/nuccore/LC761192), [LC761193](https://www.ncbi.nlm.nih.gov/nuccore/LC761193), [LC761194](https://www.ncbi.nlm.nih.gov/nuccore/LC761194), [LC761195](https://www.ncbi.nlm.nih.gov/nuccore/LC761195), [LC761196](https://www.ncbi.nlm.nih.gov/nuccore/LC761196), [LC761197](https://www.ncbi.nlm.nih.gov/nuccore/LC761197), [LC761198](https://www.ncbi.nlm.nih.gov/nuccore/LC761198), [LC761199](https://www.ncbi.nlm.nih.gov/nuccore/LC761199) and [LC761200](https://www.ncbi.nlm.nih.gov/nuccore/LC761200). Further experimental details and data are available upon request.

### Supplementary Materials

### Data Availability Statement

Sequencing data of novel variants in this study has been deposited into DNA Data Bank of Japan and National Center for Biotechnology Information with accession numbers of [LC761190](https://www.ncbi.nlm.nih.gov/nuccore/LC761190), [LC761191](https://www.ncbi.nlm.nih.gov/nuccore/LC761191), [LC761192](https://www.ncbi.nlm.nih.gov/nuccore/LC761192), [LC761193](https://www.ncbi.nlm.nih.gov/nuccore/LC761193), [LC761194](https://www.ncbi.nlm.nih.gov/nuccore/LC761194), [LC761195](https://www.ncbi.nlm.nih.gov/nuccore/LC761195), [LC761196](https://www.ncbi.nlm.nih.gov/nuccore/LC761196), [LC761197](https://www.ncbi.nlm.nih.gov/nuccore/LC761197), [LC761198](https://www.ncbi.nlm.nih.gov/nuccore/LC761198), [LC761199](https://www.ncbi.nlm.nih.gov/nuccore/LC761199) and [LC761200](https://www.ncbi.nlm.nih.gov/nuccore/LC761200). Further experimental details and data are available upon request.

## References

1. Nelson D.R., Koymans L., Kamataki T., Stegeman J.J., Feyereisen R., Waxman D.J., Waterman M.R., Gotoh O., Coon M.J., Estabrook R.W., Gunsalus I.C., Nebert D.W. P450 superfamily: update on new sequences, gene mapping, accession numbers and nomenclature. Pharmacogenetics. 1996;6:1–42. doi: 10.1097/00008571-199602000-00002.  [DOI](https://doi.org/10.1097/00008571-199602000-00002) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8845856/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=P450%20superfamily:%20update%20on%20new%20sequences,%20gene%20mapping,%20accession%20numbers%20and%20nomenclature&author=D.R.%20Nelson&author=L.%20Koymans&author=T.%20Kamataki&author=J.J.%20Stegeman&author=R.%20Feyereisen&volume=6&publication_year=1996&pages=1-42&pmid=8845856&doi=10.1097/00008571-199602000-00002&)

2. Nelson D.R., Zeldin D.C., Hoffman S.M., Maltais L.J., Wain H.M., Nebert D.W. Comparison of cytochrome P450 (CYP) genes from the mouse and human genomes, including nomenclature recommendations for genes, pseudogenes and alternative-splice variants. Pharmacogenetics. 2004;14:1–18. doi: 10.1097/00008571-200401000-00001.  [DOI](https://doi.org/10.1097/00008571-200401000-00001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15128046/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Comparison%20of%20cytochrome%20P450%20(CYP)%20genes%20from%20the%20mouse%20and%20human%20genomes,%20including%20nomenclature%20recommendations%20for%20genes,%20pseudogenes%20and%20alternative-splice%20variants&author=D.R.%20Nelson&author=D.C.%20Zeldin&author=S.M.%20Hoffman&author=L.J.%20Maltais&author=H.M.%20Wain&volume=14&publication_year=2004&pages=1-18&pmid=15128046&doi=10.1097/00008571-200401000-00001&)

3. Hidestrand M., Oscarson M., Salonen J.S., Nyman L., Pelkonen O., Turpeinen M., Ingelman-Sundberg M. CYP2B6 and CYP2C19 as the major enzymes responsible for the metabolism of selegiline, a drug used in the treatment of Parkinson's disease, as revealed from experiments with recombinant enzymes. Drug metabolism and disposition: the biological fate of chemicals. 2001;29:1480–1484.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/11602525/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20metabolism%20and%20disposition:%20the%20biological%20fate%20of%20chemicals&title=CYP2B6%20and%20CYP2C19%20as%20the%20major%20enzymes%20responsible%20for%20the%20metabolism%20of%20selegiline,%20a%20drug%20used%20in%20the%20treatment%20of%20Parkinson's%20disease,%20as%20revealed%20from%20experiments%20with%20recombinant%20enzymes&author=M.%20Hidestrand&author=M.%20Oscarson&author=J.S.%20Salonen&author=L.%20Nyman&author=O.%20Pelkonen&volume=29&publication_year=2001&pages=1480-1484&pmid=11602525&)

4. Purnapatre K., Khattar S.K., Saini K.S. Cytochrome P450s in the development of target-based anticancer drugs. Cancer letters. 2008;259:1–15. doi: 10.1016/j.canlet.2007.10.024.  [DOI](https://doi.org/10.1016/j.canlet.2007.10.024) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18053638/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20letters&title=Cytochrome%20P450s%20in%20the%20development%20of%20target-based%20anticancer%20drugs&author=K.%20Purnapatre&author=S.K.%20Khattar&author=K.S.%20Saini&volume=259&publication_year=2008&pages=1-15&pmid=18053638&doi=10.1016/j.canlet.2007.10.024&)

5. Ekins S., Vandenbranden M., Ring B.J., Gillespie J.S., Yang T.J., Gelboin H.V., Wrighton S.A. Further characterization of the expression in liver and catalytic activity of CYP2B6. J. Pharmacol. Exp. Therapeut. 1998;286:1253–1259.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/9732386/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Pharmacol.%20Exp.%20Therapeut.&title=Further%20characterization%20of%20the%20expression%20in%20liver%20and%20catalytic%20activity%20of%20CYP2B6&author=S.%20Ekins&author=M.%20Vandenbranden&author=B.J.%20Ring&author=J.S.%20Gillespie&author=T.J.%20Yang&volume=286&publication_year=1998&pages=1253-1259&pmid=9732386&)

6. Faucette S.R., Hawke R.L., Lecluyse E.L., Shord S.S., Yan B., Laethem R.M., Lindley C.M. Validation of bupropion hydroxylation as a selective marker of human cytochrome P450 2B6 catalytic activity. Drug metabolism and disposition: the biological fate of chemicals. 2000;28:1222–1230.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/10997944/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20metabolism%20and%20disposition:%20the%20biological%20fate%20of%20chemicals&title=Validation%20of%20bupropion%20hydroxylation%20as%20a%20selective%20marker%20of%20human%20cytochrome%20P450%202B6%20catalytic%20activity&author=S.R.%20Faucette&author=R.L.%20Hawke&author=E.L.%20Lecluyse&author=S.S.%20Shord&author=B.%20Yan&volume=28&publication_year=2000&pages=1222-1230&pmid=10997944&)

7. Hoffman S.M., Nelson D.R., Keeney D.S. Organization, structure and evolution of the CYP2 gene cluster on human chromosome 19. Pharmacogenetics. 2001;11:687–698. doi: 10.1097/00008571-200111000-00007.  [DOI](https://doi.org/10.1097/00008571-200111000-00007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11692077/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Organization,%20structure%20and%20evolution%20of%20the%20CYP2%20gene%20cluster%20on%20human%20chromosome%2019&author=S.M.%20Hoffman&author=D.R.%20Nelson&author=D.S.%20Keeney&volume=11&publication_year=2001&pages=687-698&pmid=11692077&doi=10.1097/00008571-200111000-00007&)

8. Telenti A., Zanger U.M. Pharmacogenetics of anti-HIV drugs. Annu. Rev. Pharmacol. Toxicol. 2008;48:227–256. doi: 10.1146/annurev.pharmtox.48.113006.094753.  [DOI](https://doi.org/10.1146/annurev.pharmtox.48.113006.094753) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17883329/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annu.%20Rev.%20Pharmacol.%20Toxicol.&title=Pharmacogenetics%20of%20anti-HIV%20drugs&author=A.%20Telenti&author=U.M.%20Zanger&volume=48&publication_year=2008&pages=227-256&pmid=17883329&doi=10.1146/annurev.pharmtox.48.113006.094753&)

9. Rakhmanina N.Y., van den Anker J.N. Efavirenz in the therapy of HIV infection. Expet Opin. Drug Metabol. Toxicol. 2010;6:95–103. doi: 10.1517/17425250903483207.  [DOI](https://doi.org/10.1517/17425250903483207) | [PMC free article](/articles/PMC2810555/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20001610/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Expet%20Opin.%20Drug%20Metabol.%20Toxicol.&title=Efavirenz%20in%20the%20therapy%20of%20HIV%20infection&author=N.Y.%20Rakhmanina&author=J.N.%20van%20den%20Anker&volume=6&publication_year=2010&pages=95-103&pmid=20001610&doi=10.1517/17425250903483207&)

10. Lam T.N., Hui K.H., Chan D.P., Lee S.S. Genotype-guided dose adjustment for the use of efavirenz in HIV treatment. J. Infect. 2015;71:607–609. doi: 10.1016/j.jinf.2015.07.005.  [DOI](https://doi.org/10.1016/j.jinf.2015.07.005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26196596/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Infect.&title=Genotype-guided%20dose%20adjustment%20for%20the%20use%20of%20efavirenz%20in%20HIV%20treatment&author=T.N.%20Lam&author=K.H.%20Hui&author=D.P.%20Chan&author=S.S.%20Lee&volume=71&publication_year=2015&pages=607-609&pmid=26196596&doi=10.1016/j.jinf.2015.07.005&)

11. Desta Z., El-Boraie A., Gong L., Somogyi A.A., Lauschke V.M., Dandara C., Klein K., Miller N.A., Klein T.E., Tyndale R.F., Whirl-Carrillo M., Gaedigk A. PharmVar GeneFocus: CYP2B6. Clinical pharmacology and therapeutics. 2021;110:82–97. doi: 10.1002/cpt.2166.  [DOI](https://doi.org/10.1002/cpt.2166) | [PMC free article](/articles/PMC8693800/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33448339/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clinical%20pharmacology%20and%20therapeutics&title=PharmVar%20GeneFocus:%20CYP2B6&author=Z.%20Desta&author=A.%20El-Boraie&author=L.%20Gong&author=A.A.%20Somogyi&author=V.M.%20Lauschke&volume=110&publication_year=2021&pages=82-97&pmid=33448339&doi=10.1002/cpt.2166&)

12. Guan S., Huang M., Chan E., Chen X., Duan W., Zhou S.F. Genetic polymorphisms of cytochrome P450 2B6 gene in Han Chinese, European journal of pharmaceutical sciences. official journal of the European Federation for Pharmaceutical Sciences. 2006;29:14–21. doi: 10.1016/j.ejps.2006.04.004.  [DOI](https://doi.org/10.1016/j.ejps.2006.04.004) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16815693/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=official%20journal%20of%20the%20European%20Federation%20for%20Pharmaceutical%20Sciences&title=Genetic%20polymorphisms%20of%20cytochrome%20P450%202B6%20gene%20in%20Han%20Chinese,%20European%20journal%20of%20pharmaceutical%20sciences&author=S.%20Guan&author=M.%20Huang&author=E.%20Chan&author=X.%20Chen&author=W.%20Duan&volume=29&publication_year=2006&pages=14-21&pmid=16815693&doi=10.1016/j.ejps.2006.04.004&)

13. Zhou X.Y., Hu X.X., Li M.F., Wang H., Zhang L.Q., Hu G.X., Cai J.P. Functional characterization of CYP2C19 variants in nebivolol 4-hydroxlation in vitro. Drug Test. Anal. 2018;10:807–813. doi: 10.1002/dta.2334.  [DOI](https://doi.org/10.1002/dta.2334) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29098786/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Test.%20Anal.&title=Functional%20characterization%20of%20CYP2C19%20variants%20in%20nebivolol%204-hydroxlation%20in%20vitro&author=X.Y.%20Zhou&author=X.X.%20Hu&author=M.F.%20Li&author=H.%20Wang&author=L.Q.%20Zhang&volume=10&publication_year=2018&pages=807-813&pmid=29098786&doi=10.1002/dta.2334&)

14. Hu X., Lan T., Dai D., Xu R.A., Yuan L., Zhou Q., Li Y., Cai J., Hu G. Evaluation of 24 CYP2D6 Variants on the Metabolism of Nebivolol In Vitro, Drug metabolism and disposition: the biological fate of chemicals. 2016;44:1828–1831. doi: 10.1124/dmd.116.071811.  [DOI](https://doi.org/10.1124/dmd.116.071811) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27538917/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Evaluation%20of%2024%20CYP2D6%20Variants%20on%20the%20Metabolism%20of%20Nebivolol%20In%20Vitro,%20Drug%20metabolism%20and%20disposition:%20the%20biological%20fate%20of%20chemicals&author=X.%20Hu&author=T.%20Lan&author=D.%20Dai&author=R.A.%20Xu&author=L.%20Yuan&volume=44&publication_year=2016&pages=1828-1831&pmid=27538917&doi=10.1124/dmd.116.071811&)

15. Desta Z., Saussele T., Ward B., Blievernicht J., Li L., Klein K., Flockhart D.A., Zanger U.M. Impact of CYP2B6 polymorphism on hepatic efavirenz metabolism in vitro. Pharmacogenomics. 2007;8:547–558. doi: 10.2217/14622416.8.6.547.  [DOI](https://doi.org/10.2217/14622416.8.6.547) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17559344/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Impact%20of%20CYP2B6%20polymorphism%20on%20hepatic%20efavirenz%20metabolism%20in%20vitro&author=Z.%20Desta&author=T.%20Saussele&author=B.%20Ward&author=J.%20Blievernicht&author=L.%20Li&volume=8&publication_year=2007&pages=547-558&pmid=17559344&doi=10.2217/14622416.8.6.547&)

16. Nakajima M., Komagata S., Fujiki Y., Kanada Y., Ebi H., Itoh K., Mukai H., Yokoi T., Minami H. Genetic polymorphisms of CYP2B6 affect the pharmacokinetics/pharmacodynamics of cyclophosphamide in Japanese cancer patients. Pharmacogenetics Genom. 2007;17:431–445. doi: 10.1097/FPC.0b013e328045c4fb.  [DOI](https://doi.org/10.1097/FPC.0b013e328045c4fb) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17502835/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics%20Genom.&title=Genetic%20polymorphisms%20of%20CYP2B6%20affect%20the%20pharmacokinetics/pharmacodynamics%20of%20cyclophosphamide%20in%20Japanese%20cancer%20patients&author=M.%20Nakajima&author=S.%20Komagata&author=Y.%20Fujiki&author=Y.%20Kanada&author=H.%20Ebi&volume=17&publication_year=2007&pages=431-445&pmid=17502835&doi=10.1097/FPC.0b013e328045c4fb&)

17. Zanger U.M., Klein K. Pharmacogenetics of cytochrome P450 2B6 (CYP2B6): advances on polymorphisms, mechanisms, and clinical relevance. Front. Genet. 2013;4:24. doi: 10.3389/fgene.2013.00024.  [DOI](https://doi.org/10.3389/fgene.2013.00024) | [PMC free article](/articles/PMC3588594/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23467454/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front.%20Genet.&title=Pharmacogenetics%20of%20cytochrome%20P450%202B6%20(CYP2B6):%20advances%20on%20polymorphisms,%20mechanisms,%20and%20clinical%20relevance&author=U.M.%20Zanger&author=K.%20Klein&volume=4&publication_year=2013&pages=24&pmid=23467454&doi=10.3389/fgene.2013.00024&)

18. Desta Z., Gammal R.S., Gong L., Whirl-Carrillo M., Gaur A.H., Sukasem C., Hockings J., Myers A., Swart M., Tyndale R.F., Masimirembwa C., Iwuchukwu O.F., Chirwa S., Lennox J., Gaedigk A., Klein T.E., Haas D.W. Clinical pharmacogenetics Implementation Consortium (CPIC) Guideline for CYP2B6 and efavirenz-containing antiretroviral therapy. Clinical pharmacology and therapeutics. 2019;106:726–733. doi: 10.1002/cpt.1477.  [DOI](https://doi.org/10.1002/cpt.1477) | [PMC free article](/articles/PMC6739160/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31006110/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clinical%20pharmacology%20and%20therapeutics&title=Clinical%20pharmacogenetics%20Implementation%20Consortium%20(CPIC)%20Guideline%20for%20CYP2B6%20and%20efavirenz-containing%20antiretroviral%20therapy&author=Z.%20Desta&author=R.S.%20Gammal&author=L.%20Gong&author=M.%20Whirl-Carrillo&author=A.H.%20Gaur&volume=106&publication_year=2019&pages=726-733&pmid=31006110&doi=10.1002/cpt.1477&)

19. Gatanaga H., Hayashida T., Tsuchiya K., Yoshino M., Kuwahara T., Tsukada H., Fujimoto K., Sato I., Ueda M., Horiba M., Hamaguchi M., Yamamoto M., Takata N., Kimura A., Koike T., Gejyo F., Matsushita S., Shirasaka T., Kimura S., Oka S. Successful efavirenz dose reduction in HIV type 1-infected individuals with cytochrome P450 2B6 *6 and *26, Clinical infectious diseases. an official publication of the Infectious Diseases Society of America. 2007;45:1230–1237. doi: 10.1086/522175.  [DOI](https://doi.org/10.1086/522175) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17918089/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=an%20official%20publication%20of%20the%20Infectious%20Diseases%20Society%20of%20America&title=Successful%20efavirenz%20dose%20reduction%20in%20HIV%20type%201-infected%20individuals%20with%20cytochrome%20P450%202B6%20*6%20and%20*26,%20Clinical%20infectious%20diseases&author=H.%20Gatanaga&author=T.%20Hayashida&author=K.%20Tsuchiya&author=M.%20Yoshino&author=T.%20Kuwahara&volume=45&publication_year=2007&pages=1230-1237&pmid=17918089&doi=10.1086/522175&)

20. Gatanaga H., Oka S. Successful genotype-tailored treatment with small-dose efavirenz. Aids. 2009;23:433–434. doi: 10.1097/QAD.0b013e32831940e3.  [DOI](https://doi.org/10.1097/QAD.0b013e32831940e3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19188761/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Aids&title=Successful%20genotype-tailored%20treatment%20with%20small-dose%20efavirenz&author=H.%20Gatanaga&author=S.%20Oka&volume=23&publication_year=2009&pages=433-434&pmid=19188761&doi=10.1097/QAD.0b013e32831940e3&)

21. Nyakutira C., Roshammar D., Chigutsa E., Chonzi P., Ashton M., Nhachi C., Masimirembwa C. High prevalence of the CYP2B6 516G-->T(*6) variant and effect on the population pharmacokinetics of efavirenz in HIV/AIDS outpatients in Zimbabwe. Eur. J. Clin. Pharmacol. 2008;64:357–365. doi: 10.1007/s00228-007-0412-3.  [DOI](https://doi.org/10.1007/s00228-007-0412-3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18057928/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Clin.%20Pharmacol.&title=High%20prevalence%20of%20the%20CYP2B6%20516G-->T(*6)%20variant%20and%20effect%20on%20the%20population%20pharmacokinetics%20of%20efavirenz%20in%20HIV/AIDS%20outpatients%20in%20Zimbabwe&author=C.%20Nyakutira&author=D.%20Roshammar&author=E.%20Chigutsa&author=P.%20Chonzi&author=M.%20Ashton&volume=64&publication_year=2008&pages=357-365&pmid=18057928&doi=10.1007/s00228-007-0412-3&)

22. Hofmann M.H., Blievernicht J.K., Klein K., Saussele T., Schaeffeler E., Schwab M., Zanger U.M. Aberrant splicing caused by single nucleotide polymorphism c.516G>T [Q172H], a marker of CYP2B6*6, is responsible for decreased expression and activity of CYP2B6 in liver. J. Pharmacol. Exp. Therapeut. 2008;325:284–292. doi: 10.1124/jpet.107.133306.  [DOI](https://doi.org/10.1124/jpet.107.133306) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18171905/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Pharmacol.%20Exp.%20Therapeut.&title=Aberrant%20splicing%20caused%20by%20single%20nucleotide%20polymorphism%20c.516G>T%20%5BQ172H%5D,%20a%20marker%20of%20CYP2B6*6,%20is%20responsible%20for%20decreased%20expression%20and%20activity%20of%20CYP2B6%20in%20liver&author=M.H.%20Hofmann&author=J.K.%20Blievernicht&author=K.%20Klein&author=T.%20Saussele&author=E.%20Schaeffeler&volume=325&publication_year=2008&pages=284-292&pmid=18171905&doi=10.1124/jpet.107.133306&)

23. Sadee W., Wang D., Papp A.C., Pinsonneault J.K., Smith R.M., Moyer R.A., Johnson A.D. Pharmacogenomics of the RNA world: structural RNA polymorphisms in drug therapy. Clinical pharmacology and therapeutics. 2011;89:355–365. doi: 10.1038/clpt.2010.314.  [DOI](https://doi.org/10.1038/clpt.2010.314) | [PMC free article](/articles/PMC3251919/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21289622/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clinical%20pharmacology%20and%20therapeutics&title=Pharmacogenomics%20of%20the%20RNA%20world:%20structural%20RNA%20polymorphisms%20in%20drug%20therapy&author=W.%20Sadee&author=D.%20Wang&author=A.C.%20Papp&author=J.K.%20Pinsonneault&author=R.M.%20Smith&volume=89&publication_year=2011&pages=355-365&pmid=21289622&doi=10.1038/clpt.2010.314&)

24. Talakad J.C., Kumar S., Halpert J.R. Decreased susceptibility of the cytochrome P450 2B6 variant K262R to inhibition by several clinically important drugs. Drug metabolism and disposition: the biological fate of chemicals. 2009;37:644–650. doi: 10.1124/dmd.108.023655.  [DOI](https://doi.org/10.1124/dmd.108.023655) | [PMC free article](/articles/PMC2649788/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19074527/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20metabolism%20and%20disposition:%20the%20biological%20fate%20of%20chemicals&title=Decreased%20susceptibility%20of%20the%20cytochrome%20P450%202B6%20variant%20K262R%20to%20inhibition%20by%20several%20clinically%20important%20drugs&author=J.C.%20Talakad&author=S.%20Kumar&author=J.R.%20Halpert&volume=37&publication_year=2009&pages=644-650&pmid=19074527&doi=10.1124/dmd.108.023655&)

25. Levran O., Peles E., Hamon S., Randesi M., Adelson M., Kreek M.J. CYP2B6 SNPs are associated with methadone dose required for effective treatment of opioid addiction. Addiction Biol. 2013;18:709–716. doi: 10.1111/j.1369-1600.2011.00349.x.  [DOI](https://doi.org/10.1111/j.1369-1600.2011.00349.x) | [PMC free article](/articles/PMC3735354/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21790905/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Addiction%20Biol.&title=CYP2B6%20SNPs%20are%20associated%20with%20methadone%20dose%20required%20for%20effective%20treatment%20of%20opioid%20addiction&author=O.%20Levran&author=E.%20Peles&author=S.%20Hamon&author=M.%20Randesi&author=M.%20Adelson&volume=18&publication_year=2013&pages=709-716&pmid=21790905&doi=10.1111/j.1369-1600.2011.00349.x&)

26. Zukunft J., Lang T., Richter T., Hirsch-Ernst K.I., Nussler A.K., Klein K., Schwab M., Eichelbaum M., Zanger U.M. A natural CYP2B6 TATA box polymorphism (-82T--> C) leading to enhanced transcription and relocation of the transcriptional start site, Mol. Pharmacol. 2005;67:1772–1782. doi: 10.1124/mol.104.008086.  [DOI](https://doi.org/10.1124/mol.104.008086) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15722458/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Pharmacol.&title=A%20natural%20CYP2B6%20TATA%20box%20polymorphism%20(-82T-->%20C)%20leading%20to%20enhanced%20transcription%20and%20relocation%20of%20the%20transcriptional%20start%20site,&author=J.%20Zukunft&author=T.%20Lang&author=T.%20Richter&author=K.I.%20Hirsch-Ernst&author=A.K.%20Nussler&volume=67&publication_year=2005&pages=1772-1782&pmid=15722458&doi=10.1124/mol.104.008086&)

27. Rotger M., Tegude H., Colombo S., Cavassini M., Furrer H., Decosterd L., Blievernicht J., Saussele T., Gunthard H.F., Schwab M., Eichelbaum M., Telenti A., Zanger U.M. Predictive value of known and novel alleles of CYP2B6 for efavirenz plasma concentrations in HIV-infected individuals. Clinical pharmacology and therapeutics. 2007;81:557–566. doi: 10.1038/sj.clpt.6100072.  [DOI](https://doi.org/10.1038/sj.clpt.6100072) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17235330/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clinical%20pharmacology%20and%20therapeutics&title=Predictive%20value%20of%20known%20and%20novel%20alleles%20of%20CYP2B6%20for%20efavirenz%20plasma%20concentrations%20in%20HIV-infected%20individuals&author=M.%20Rotger&author=H.%20Tegude&author=S.%20Colombo&author=M.%20Cavassini&author=H.%20Furrer&volume=81&publication_year=2007&pages=557-566&pmid=17235330&doi=10.1038/sj.clpt.6100072&)

28. Meng X.M., Li Z.R., Zheng X.Y., Liu Y.X., Niu W.J., Qiu X.Y., Lu H.Z. Effect of albumin and CYP2B6 polymorphisms on exposure of efavirenz: a population pharmacokinetic analysis in Chinese HIV-infected adults, European journal of pharmaceutical sciences : official journal of the European Federation for Pharmaceutical Sciences. 2021;167 doi: 10.1016/j.ejps.2021.105986.  [DOI](https://doi.org/10.1016/j.ejps.2021.105986) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34474119/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=official%20journal%20of%20the%20European%20Federation%20for%20Pharmaceutical%20Sciences&title=Effect%20of%20albumin%20and%20CYP2B6%20polymorphisms%20on%20exposure%20of%20efavirenz:%20a%20population%20pharmacokinetic%20analysis%20in%20Chinese%20HIV-infected%20adults,%20European%20journal%20of%20pharmaceutical%20sciences%20:&author=X.M.%20Meng&author=Z.R.%20Li&author=X.Y.%20Zheng&author=Y.X.%20Liu&author=W.J.%20Niu&volume=167&publication_year=2021&pmid=34474119&doi=10.1016/j.ejps.2021.105986&)

29. Qi G.Z., Zhang Z.Y., Wang X., Yin S.J., Lou Y.Q., Zhang G.L. Functional allele and genotype frequencies of CYP1A2, CYP2B6 and iNOS among mainland Chinese Tibetan, Mongolian, Uygur and Han populations. J. Clin. Pharm. Therapeut. 2016;41:84–91. doi: 10.1111/jcpt.12351.  [DOI](https://doi.org/10.1111/jcpt.12351) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26763760/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Pharm.%20Therapeut.&title=Functional%20allele%20and%20genotype%20frequencies%20of%20CYP1A2,%20CYP2B6%20and%20iNOS%20among%20mainland%20Chinese%20Tibetan,%20Mongolian,%20Uygur%20and%20Han%20populations&author=G.Z.%20Qi&author=Z.Y.%20Zhang&author=X.%20Wang&author=S.J.%20Yin&author=Y.Q.%20Lou&volume=41&publication_year=2016&pages=84-91&pmid=26763760&doi=10.1111/jcpt.12351&)

30. Tan L., Yu J.T., Sun Y.P., Ou J.R., Song J.H., Yu Y. The influence of cytochrome oxidase CYP2A6, CYP2B6, and CYP2C9 polymorphisms on the plasma concentrations of valproic acid in epileptic patients, Clin. Neurol. Neurosurg. 2010;112:320–323. doi: 10.1016/j.clineuro.2010.01.002.  [DOI](https://doi.org/10.1016/j.clineuro.2010.01.002) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20089352/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Neurol.%20Neurosurg.&title=The%20influence%20of%20cytochrome%20oxidase%20CYP2A6,%20CYP2B6,%20and%20CYP2C9%20polymorphisms%20on%20the%20plasma%20concentrations%20of%20valproic%20acid%20in%20epileptic%20patients,&author=L.%20Tan&author=J.T.%20Yu&author=Y.P.%20Sun&author=J.R.%20Ou&author=J.H.%20Song&volume=112&publication_year=2010&pages=320-323&pmid=20089352&doi=10.1016/j.clineuro.2010.01.002&)

31. Watanabe T., Saito T., Rico E.M.G., Hishinuma E., Kumondai M., Maekawa M., Oda A., Saigusa D., Saito S., Yasuda J., Nagasaki M., Minegishi N., Yamamoto M., Yamaguchi H., Mano N., Hirasawa N., Hiratsuka M. Functional characterization of 40 CYP2B6 allelic variants by assessing efavirenz 8-hydroxylation. Biochem. Pharmacol. 2018;156:420–430. doi: 10.1016/j.bcp.2018.09.010.  [DOI](https://doi.org/10.1016/j.bcp.2018.09.010) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30201214/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem.%20Pharmacol.&title=Functional%20characterization%20of%2040%20CYP2B6%20allelic%20variants%20by%20assessing%20efavirenz%208-hydroxylation&author=T.%20Watanabe&author=T.%20Saito&author=E.M.G.%20Rico&author=E.%20Hishinuma&author=M.%20Kumondai&volume=156&publication_year=2018&pages=420-430&pmid=30201214&doi=10.1016/j.bcp.2018.09.010&)

32. Watanabe T., Sakuyama K., Sasaki T., Ishii Y., Ishikawa M., Hirasawa N., Hiratsuka M. Functional characterization of 26 CYP2B6 allelic variants (CYP2B6.2-CYP2B6.28, except CYP2B6.22) Pharmacogenetics Genom. 2010;20:459–462. doi: 10.1097/FPC.0b013e32833bba0e.  [DOI](https://doi.org/10.1097/FPC.0b013e32833bba0e) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20517174/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics%20Genom.&title=Functional%20characterization%20of%2026%20CYP2B6%20allelic%20variants%20(CYP2B6.2-CYP2B6.28,%20except%20CYP2B6.22)&author=T.%20Watanabe&author=K.%20Sakuyama&author=T.%20Sasaki&author=Y.%20Ishii&author=M.%20Ishikawa&volume=20&publication_year=2010&pages=459-462&pmid=20517174&doi=10.1097/FPC.0b013e32833bba0e&)

33. Ariyoshi N., Ohara M., Kaneko M., Afuso S., Kumamoto T., Nakamura H., Ishii I., Ishikawa T., Kitada M. Q172H replacement overcomes effects on the metabolism of cyclophosphamide and efavirenz caused by CYP2B6 variant with Arg262. Drug metabolism and disposition: the biological fate of chemicals. 2011;39:2045–2048. doi: 10.1124/dmd.111.039586.  [DOI](https://doi.org/10.1124/dmd.111.039586) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21821736/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20metabolism%20and%20disposition:%20the%20biological%20fate%20of%20chemicals&title=Q172H%20replacement%20overcomes%20effects%20on%20the%20metabolism%20of%20cyclophosphamide%20and%20efavirenz%20caused%20by%20CYP2B6%20variant%20with%20Arg262&author=N.%20Ariyoshi&author=M.%20Ohara&author=M.%20Kaneko&author=S.%20Afuso&author=T.%20Kumamoto&volume=39&publication_year=2011&pages=2045-2048&pmid=21821736&doi=10.1124/dmd.111.039586&)

34. Crettol S., Deglon J.J., Besson J., Croquette-Krokkar M., Gothuey I., Hammig R., Monnat M., Huttemann H., Baumann P., Eap C.B. Methadone enantiomer plasma levels, CYP2B6, CYP2C19, and CYP2C9 genotypes, and response to treatment. Clinical pharmacology and therapeutics. 2005;78:593–604. doi: 10.1016/j.clpt.2005.08.011.  [DOI](https://doi.org/10.1016/j.clpt.2005.08.011) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16338275/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clinical%20pharmacology%20and%20therapeutics&title=Methadone%20enantiomer%20plasma%20levels,%20CYP2B6,%20CYP2C19,%20and%20CYP2C9%20genotypes,%20and%20response%20to%20treatment&author=S.%20Crettol&author=J.J.%20Deglon&author=J.%20Besson&author=M.%20Croquette-Krokkar&author=I.%20Gothuey&volume=78&publication_year=2005&pages=593-604&pmid=16338275&doi=10.1016/j.clpt.2005.08.011&)

35. Eap C.B., Crettol S., Rougier J.S., Schlapfer J., Sintra Grilo L., Deglon J.J., Besson J., Croquette-Krokar M., Carrupt P.A., Abriel H. Stereoselective block of hERG channel by (S)-methadone and QT interval prolongation in CYP2B6 slow metabolizers. Clinical pharmacology and therapeutics. 2007;81:719–728. doi: 10.1038/sj.clpt.6100120.  [DOI](https://doi.org/10.1038/sj.clpt.6100120) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17329992/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clinical%20pharmacology%20and%20therapeutics&title=Stereoselective%20block%20of%20hERG%20channel%20by%20(S)-methadone%20and%20QT%20interval%20prolongation%20in%20CYP2B6%20slow%20metabolizers&author=C.B.%20Eap&author=S.%20Crettol&author=J.S.%20Rougier&author=J.%20Schlapfer&author=L.%20Sintra%20Grilo&volume=81&publication_year=2007&pages=719-728&pmid=17329992&doi=10.1038/sj.clpt.6100120&)
