# Association between HLA alleles and beta-lactam antibiotics-related severe cutaneous adverse reactions

## Metadata
**Authors:** Pansakon Wattanachai, Warayuwadee Amornpinyo, Parinya Konyoung, Danklai Purimart, Usanee Khunarkornsiri, Oranuch Pattanacheewapull, Wichittra Tassaneeyakul, Nontaya Nakkam
**Journal:** Frontiers in Pharmacology
**Date:** 2023 Sep 19
**DOI:** [10.3389/fphar.2023.1248386](https://doi.org/10.3389/fphar.2023.1248386)
**PMID:** 37795024
**PMCID:** PMC10546186
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10546186/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC10546186/pdf/fphar-14-1248386.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC10546186/pdf/fphar-14-1248386.pdf)

## Abstract

Introduction: Beta-lactam antibiotics are one of the most common causes of antibiotics-related severe cutaneous adverse reactions (SCARs) including Stevens–Johnson syndrome (SJS), toxic epidermal necrolysis (TEN), drug reactions with eosinophilia and systemic symptoms (DRESS), and acute generalized exanthematous pustulosis (AGEP). Recent evidence demonstrated that the human leukocyte antigen (HLA) polymorphisms play important roles in the development of drug-related SCARs. This study aimed to extensively characterize the associations between HLA genetic polymorphisms and several phenotypes of SCARs related to beta-lactam antibiotics.

Methods: Thirty-one Thai patients with beta-lactam antibiotics-related SCARs were enrolled in the study. A total of 183 unrelated native Thai subjects without any evidence of drug allergy were recruited as the control group. Genotyping of HLA class I and class II alleles was performed.

Results: Six HLA alleles including HLA-A*01:01, HLA-B*50:01, HLA-C*06:02, HLA-DRB1*15:01, HLA-DQA1*03:01, and HLA-DQB1*03:02, were significantly associated with beta-lactam antibiotics-related SCARs. The highest risk of SCARs was observed in patients with the HLA-B*50:01 allele (OR = 12.6, 95% CI = 1.1–142.9, p = 0.042), followed by the HLA-DQB1*03:02 allele (OR = 5.8, 95% CI = 1.5–22.0, p = 0.012) and the HLA-C*06:02 allele (OR = 5.7, 95% CI = 1.6–19.9, p = 0.011). According to the phenotypes of SCARs related to beta-lactam antibiotics, the higher risk of SJS/TEN was observed in patients with HLA-A*03:02, HLA-B*46:02 (OR = 17.5, 95% CI = 1.5–201.6, p = 0.033), HLA-A*02:06, HLA-B*57:01 (OR = 9.5, 95% CI = 1.3–71.5, p = 0.028), HLA-DQB1*03:02 (OR = 7.5, 95% CI = 1.8–30.9, p = 0.008), or HLA-C*06:02 (OR = 4.9, 95% CI = 1.1–21.4, p = 0.008). While eight HLA alleles including HLA-A*02:05, HLA-A*02:11, HLA-B*37:01, HLA-B*38:01, HLA-B*50:01, HLA-C*06:02, HLA-C*03:09, and HLA-DRB1*15:01 were associated with AGEP, the highest risk of AGEP was observed in patients with the HLA-B*50:01 allele (OR = 60.7, 95% CI = 4.8–765.00, p = 0.005). Among the four HLA alleles associated with DRESS including HLA-C*04:06, HLA-DRB1*04:05, HLA-DRB1*11:01, and HLA-DQB1*04:01, the HLA-C*04:06 allele had the highest risk of beta-lactam antibiotics-related DRESS (OR = 60.0, 95% CI = 3.0–1202.1, p = 0.043). However, these associations did not achieve statistical significance after Bonferroni’s correction. Apart from the HLA risk alleles, the HLA-A*02:07 allele appeared to be a protective factor against beta-lactam antibiotic-related SCARs (OR = 0.1, 95% CI = 0.0–0.5, p = 3.7 × 10−4, Pc = 0.012).

Conclusion: This study demonstrated the candidate HLA alleles that are significantly associated with several phenotypes of beta-lactam antibiotics-related SCARs. However, whether the HLA alleles observed in this study can be used as valid genetic markers for SCARs related to beta-lactam antibiotics needs to be further explored in other ethnicities and larger cohort studies.

Keywords: beta-lactam antibiotics, severe cutaneous adverse reactions, Stevens–Johnson syndrome/toxic epidermal necrolysis, acute generalized exanthematous pustulosis, drug reactions with eosinophilia and systemic symptoms, Human Leukocyte Antigen

## 1 Introduction

Beta-lactam antibiotics are commonly prescribed medications with various clinical indications. However, they have been reported as the most common cause of antibiotic-induced cutaneous adverse drug reactions ranging in severity from mild, self-limited cutaneous eruptions such as urticaria and maculopapular exanthema (MPE) to life-threatening severe cutaneous adverse reactions (SCARs) including Stevens–Johnson syndrome (SJS), toxic epidermal necrolysis (TEN), drug reaction with eosinophilia and systemic symptoms (DRESS), and acute generalized exanthematous pustulosis (AGEP) ([Blumenthal et al., 2019](#B2)). SJS/TEN, DRESS, and AGEP vary in mortality, immunopathogenesis, and clinical characteristics. For SJS/TEN cases, cytotoxic CD8^+^ T cells accumulate in blisters and release granulysin, perforin, and granzyme B to kill keratinocytes. In the case of DRESS, CD4^+^ and CD8^+^ T cells, plasma dendritic cells (DCs), and monocytes are enriched in the dermis, and the release of TNF-α and IFN-γ was found. Moreover, interleukin 5 (IL-5) is produced from Th2 cells and group 2 innate lymphoid cells (ILC2s) in which IL-5 can induce the activation and migration of eosinophils that drive inflammation in DRESS. For AGEP, CD4^+^ T cells secrete IL-4, IL-5, IL-13, IFN-γ, TNF-α, IL-8, IL-17, and IL-22 (Th17). IL-8 drives neutrophil and the recruitment of T cells to the epidermis to form sterile pustules ([Karnes et al., 2019](#B11); [Gibson et al., 2023](#B7)). These SCARs are delayed type IV hypersensitivity reactions or T-cell-mediated immune response, with a high mortality rate of less than 1% in AGEP, less than 10% in DRESS, and 15–50% in SJS/TEN. Moreover, patients who recover from SCAR episodes may be left with sequelae or long-lasting disabilities such as blindness ([Roujeau, 2005](#B24); [Chung et al., 2016](#B4); [Gibson et al., 2023](#B7)).

Recent evidence revealed that genetic polymorphisms of human leukocyte antigen (*HLA*) genes may play important roles in the development of drug-induced life-threatening SCARs. Some *HLA* alleles have been demonstrated to be strongly associated with drug-induced SCARs and have been proposed as valid genetic markers for the prediction of drug-induced SCARs such as *HLA-B*15:02* for carbamazepine-induced SJS/TEN ([Chung et al., 2004](#B3); [Hung et al., 2006](#B8); [Tassaneeyakul et al., 2010](#B36); [Nakkam et al., 2022a](#B17)), *HLA-B*58:01* for allopurinol-induced SCARs ([Hung et al., 2005](#B9); [Tassaneeyakul et al., 2009](#B34); [Saksit et al., 2017b](#B27)), or *HLA-B*57:01* for abacavir hypersensitivity ([Mallal et al., 2002](#B14); [Mallal et al., 2008](#B15)). According to these strong associations, the regulatory agencies in several countries as well as the Clinical Pharmacogenetics Implementation Consortium (CPIC) suggest physicians to perform *HLA* screening tests in individual patients before the initiation of some drugs to identify patients who are at a higher risk of drug-induced SCARs ([Saito et al., 2016](#B25); [Phillips et al., 2018](#B21)).

A recent study of 1,078 SJS/TEN Asian patients, including Thai patients, who have been admitted to hospitals during 1998–2017, from registration databases revealed that beta-lactam antibiotics, especially aminopenicillins and cephalosporins, are the most common cause of antibiotics-induced SJS/TEN after sulfamethoxazole ([Wang et al., 2019](#B39)). The associations between genetic markers and beta-lactam antibiotics-induced adverse drug reactions have been reported, in which the *HLA-DRB* gene appeared to be associated with IgE-mediated-type hypersensitivity to penicillin in a Chinese population ([Yang et al., 2006](#B40)). Consistent with the findings in a Chinese population, *HLA-DRB1*10:01* alleles were associated with penicillin-induced immediate reactions in a European population ([Nicoletti et al., 2021](#B20)). Apart from the immediate reaction, the study in an Italian population demonstrated that the *HLA-DRB3*02:*02 allele was associated with penicillin-induced delayed hypersensitivity, especially with MPE ([Romano et al., 2022](#B23)). Moreover, *HLA-B*55:01* was related to self-reported penicillin delayed reactions in a genome-wide association study in a European population, and it should be noted that the most common phenotype of skin reactions found in this study was delayed-type rash ([Krebs et al., 2020](#B12)).

In addition, *HLA-B*48:01* was strongly associated with immediate reactions from beta-lactam hypersensitivity in Thai children, while *HLA-C*04:06*, *HLA-C*08:01*, and *HLA-DRB1*04:06* were associated with beta-lactam antibiotic-induced delayed-type hypersensitivity reactions ([Singvijarn et al., 2019](#B31)). It should be noted that only two patients with DRESS and SJS were recruited in [Singvijarn et al. (2019](#B31)). Apart from co-trimoxazole, the data on associations between SCARs and *HLA* polymorphisms in other antibiotics are still limited. The present study aimed to extensively characterize the associations between *HLA* genetic polymorphisms, including *HLA* class I and class II, and several phenotypes of SCARs related to beta-lactam antibiotics in a Thai population.

## 2 Materials and methods

### 2.1 Study population

Patients with beta-lactam antibiotics-related SCARs, who were admitted to a local hospital in Thailand from 2009 to 2022, were recruited. Patients hospitalized between 2009 and 2019 were identified retrospectively by reviewing their medical records, whereas patients who were hospitalized between 2020 and 2022 were prospectively enrolled in the study. The diagnosis of SCARs was made primarily by an internist or a dermatologist in each hospital and subsequently confirmed by a dermatologist in the investigator team. The criteria for SJS, TEN, and DRESS were classified as previously described ([Nakkam et al., 2022a](#B17); [Nakkam et al., 2022b](#B19)). While AGEP was defined as numerous non-follicular pustules arising on a widespread edematous erythema, fever, and elevated blood neutrophils, histopathology typically showed spongiform subcorneal and/or intraepidermal pustules, a marked edema of the papillary dermis, and eventually vasculitis, eosinophilia, and/or focal necrosis of keratinocytes ([Sidoroff et al., 2001](#B30)).

The causative drugs were identified by a corresponding Naranjo adverse drug reaction (ADR) score of ≥5 (probable ADR). Moreover, for SJS and TEN, the drug causality for each patient was evaluated based on the algorithm for the assessment of drug causality (ALDEN score ≥4; probable SJS/TEN) ([Sassolas et al., 2010](#B28)). The scoring system for classifying DRESS cases was based on RegiSCAR ([Kardaun et al., 2007](#B10)), and the scoring system for classifying AGEP cases was based on EuroSCAR ([Sidoroff et al., 2001](#B30)). All patients with probable RegiSCAR or EuroSCAR were enrolled as cases.

Thai subjects who had been using several medications including beta-lactam antibiotics without any evidence of drug allergy were recruited as a control group. Written informed consent was obtained from each participant. This study protocol was approved by the Ethics Committee for Human Research, Khon Kaen University, Thailand ([HE510837](https://www.ncbi.nlm.nih.gov/nuccore/HE510837)).

### 2.2 Genomic DNA preparation

Peripheral blood samples were collected into EDTA-coated tubes. Buffy coats were separated by centrifugation at 3,500 RPM, 4°C, for 15 min. Genomic DNA was then isolated from leukocytes using QIAamp DNA Blood mini kits (QIAGEN GmbH, Hilden, Germany).

### 2.3 HLA genotyping

Genotyping of *HLA* class I, including *HLA-A*, *HLA-B*, and *HLA-C*, and *HLA* class II, including *HLA-DRB1*, *HLA-DQA1*, and *HLA-DQB1*, was determined using the LIFECODES *HLA* Typing Kits (Immucor GTI Diagnostics, Waukesha, Wisconsin, United States), which are based on the reverse sequence-specific oligonucleotide (SSO) probe method coupled with xMAP Technology designed to be used with the Luminex^®^ System, as previously described ([Nakkam et al., 2022a](#B17); [Nakkam et al., 2022b](#B19)). The *HLA* alleles were analyzed using LIFECODES MATCH IT DNA Software version 1.3 based on Allele Database version 3.41. According to the previous studies ([Wang et al., 2021](#B38); [Nakkam et al., 2022a](#B17)), it was reported that the results of two-field or four-digit resolution of *HLA* genotypes from LIFECODES HLA typing were concordant with the SeCore HLA sequence-based typing (SBT) method.

### 2.4 Data statistical analysis

The demographic data and clinical characteristics of SCAR cases were carried out using descriptive statistics such as percentage, median, mean, and standard deviation. The statistical analysis among SCAR phenotypes for the clinical characteristics was carried out using two-tailed Student’s *t*-test and Fisher’s exact test.

The allele frequencies and genotype frequencies of the *HLA* class I and II alleles were determined by direct counting. The strength of associations was estimated by calculating the odds ratios (ORs) and 95% confidence intervals (CIs) using SPSS statistical software, version 28.0 for macOS (IBM, Armonk, New York, United States). ORs were determined using Haldane’s modification, which adds 0.5 to all cells to accommodate possible zero counts. The corrected *p*-values (Pc) for the multiple comparisons of *HLA* (34 for *HLA-A*, 53 for *HLA-B*, 36 for *HLA-C*, 29 for *HLA-DRB1*, 10 for *HLA-DQA1*, and 15 for *HLA-DQB1*) were calculated using Bonferroni’s correction. All *p*-values were empirical, two-tailed values, and a value less than 0.05 was considered statistically significant. The estimated linkage disequilibrium coefficients (D’) and coefficient of correlation (*r* ^2^) among identified *HLA* alleles were calculated using the PLINK (V1.07) program ([Ardlie et al., 2002](#B1); [Saksit et al., 2017a](#B26)).

## 3 Results

### 3.1 Characteristics of the study population

Thirty-one beta-lactam antibiotics-related SCARs patients who met the inclusion criteria were enrolled in this study. Among these, 21 patients (67.7%) were diagnosed with SJS/TEN (13 with SJS, one with overlap SJS/TEN, and seven with TEN), eight patients (25.8%) were diagnosed with AGEP, and two patients (6.5%) were diagnosed with DRESS.

The beta-lactam antibiotics-related SCARs patients comprised of nine men and 22 women with an average age of 52.1 ± 22.1 years (47.3 ± 24.5 in the SJS/TEN, 58.5 ± 2.1 in the DRESS, and 63.0 ± 12.7 in the AGEP groups), while the patients recruited in the control group comprised of 106 men (57.9%) and 77 women (42.1%) with an average age of 42.6 ± 13.6 years.

The mean exposure time to beta-lactam antibiotics until the occurrence of first signs of SJS/TEN (onset or latency period) was 6.1 ± 5.8 (range from 1 to 18) days, whereas those of beta-lactam antibiotics-related DRESS and AGEP were 25.5 ± 23.3 (ranging from 9 to 42) days and 7.3 ± 7.8 (ranging from 2 to 23) days, respectively. The most common comorbidities in SCARs cases included hypertension (25.8%), diabetes (25.8%), chronic renal disease (12.9%), and liver dysfunction (6.5%). For the mucosal involvement in SCARs cases, the mucosal involvements of oral (76.2%) and ocular (52.4%) were mostly reported in SJS/TEN patients. Oral mucosal involvement was found in only one (1/2, 50.0%) patient with DRESS; however, no mucosal involvement was found in AGEP patients.

The mean duration of hospital stay for SCARs treatment was 13.8 ± 15.3 (ranging from 3 to 59) days, in which the longest mean hospital stay was reported in the TEN group among the SJS, SJS/TEN overlap, DRESS, and AGEP groups. The cost of treatment in the TEN group was the highest among the other groups. Only one SCARs patient died in the hospital from sepsis, which may be complication from the SJS/TEN overlap episodes. All data are shown in [Table 1](#T1).

### TABLE 1.

| Characteristics | SJS | SJS/TEN overlap | TEN | Total SJS/TEN | AGEP | DRESS | Total SCARs |
| --- | --- | --- | --- | --- | --- | --- | --- |
| Demographic data |  |  |  |  |  |  |  |
| N | 13 | 1 | 7 | 21 | 8 | 2 | 31 |
| Age, years |   |   |   |   |   |   |   |
| Mean (SD) | 45.4 (27.0) | 69 | 47.7 (21.4) | 47.3 (24.5) | 63 (12.7) | 58.5 (2.1) | 52.1 (22.1) |
| Median [range] | 53 [3–82] | NA | 49 [8–74] | 53 [3–82] | 58 [50–82] | 58.5 [57–60] | 57 [3–82] |
| Sex, n (%) |   |   |   |   |   |   |   |
| Male | 5 (38.5) | 0 (0) | 1 (14.3) | 6 (28.6) | 2 (25) | 1 (50) | 9 (29.0) |
| Female | 8 (61.5) | 1 (100) | 6 (85.7) | 15 (71.4) | 6 (75) | 1 (50) | 22 (70.8) |
| Onset of SCARs, days |   |   |   |   |   |   |   |
| n (available data) | 13 | 1 | 6 | 20 | 6 | 2 | 28 |
| Mean (SD) | 4.4 (3.9) | 5 | 9.5 (8.1) | 6.1 (5.8) | 7.3 (7.8) | 25.5 (23.3) | 7.7 (9.0) |
| Median [range] | 4 [1–14] | NA | 10.5 [1–18] | 5 [1–18] | 4 [2–23] | 25.5 [9–42] | 4.5 [1–42] |
| Mucosal involvement, n (%) |   |   |   |   |   |   |   |
| n (available data) | 13 | 1 | 7 | 21 | 8 | 2 | 31 |
| Oral | 10 (76.9) a | 1 (100) | 5 (71.4) a | 16 (76.2) a | 0 (0) | 1 (50) | 17 (54.8) |
| Ocular | 6 (46.2) a | 1 (100) | 4 (57.1) a | 11 (52.4) a | 0 (0) | 0 (0) | 11 (35.5) |
| Genital | 4 (30.8) | 1 (100) | 3 (42.9) | 8 (38.1) | 0 (0) | 0 (0) | 8 (25.8) |
| Concomitant drugs b , n (%) |   |   |   |   |   |   |   |
| n (available data) | 13 | 1 | 7 | 21 | 8 | 2 | 31 |
| Yes | 6 (46.2) | 1 (100) | 4 (57.1) | 11 (52.4) | 2 (25) | 2 (100) | 15 (48.4) |
| Outcome from SCARs |  |  |  |  |  |  |  |
| Duration of hospital stay for the treatment of SCARs, days |   |   |   |   |   |   |   |
| n (available data) | 9 | 1 | 3 | 13 | 4 | 1 | 18 |
| Mean (SD) | 13 (17.8) | 17 | 26 (18.3) | 16.31 (17.3) | 7.7 (5.9) | 5 | 13.8 (15.3) |
| Median [range] | 7 [3–59] | NA | 30 [6–42] | 8 [3–59] | 6 [3–16] | NA | 7.5 [3–59] |
| Deceased cases, n (%) | 0 (0) | 1 (100) | 0 (0) | 1 (4.8) | 0 (0) | 0 (0) | 1 (3.2) |
| Cost of treatment |   |   |   |   |   |   |   |
| n (available data) | 5 | 1 | 2 | 8 | 4 | 0 | 12 |
| Mean (SD) | 46,263.7 (79,892.7) | 79,961 | 225,726.5 (215,879.0) | 95,350.56 (130,073.7) | 9,170.5 (9,744.3) | NA | 66,623.88 (112,219.1) |
| Median [range] | 12,081 [3,778–188,600] | NA | 225,762 [73,113–378,412] | 47,486 [3,778–378,412] | 7,307.5 [220–21,847] | NA | 169,64.3 [220–378,412] |

Table 1 Caption: Demographic and clinical data in beta-lactam antibiotics-related severe cutaneous adverse reactions.

### 3.2 Laboratory data of SCARs patients

Among 31 patients with beta-lactam antibiotics-related SCARs, 15–20% of SCARs patients, particularly patients in the SJS/TEN group, had liver involvement presented with elevated AST or ALT levels (100–500 units/L). Approximately 17% of SCARs patients had kidney involvement. Moreover, hematologic abnormality, neutrophilia, was found in 83% of patients in the AGEP group, while eosinophilia was mainly found in beta-lactam antibiotics-related DRESS patients, followed by AGEP and SJS/TEN patients ([Table 2](#T2)). In addition, fever, enlarged lymph nodes, and atypical lymphocytes were found in two DRESS patients recruited into this study.

### TABLE 2.

|   | Phenotypes, number of cases (%) |  |  |  |
| --- | --- | --- | --- | --- |
| Laboratory investigations | SJS/TEN, 21 (67.7) | AGEP, 8 (25.8) | DRESS, 2 (6.5) | Total SCARs, 31 (100.0) |
| Liver function |   |   |   |   |
| n (available data) | 14 | 4 | 2 | 20 |
| AST (units/L) |   |   |   |   |
| <100 | 10 (71.4) | 4 (100) | 2 (100) | 16 (80) |
| 100–500 | 4 (28.6) | 0 (0) | 0 (0) | 4 (20) |
| 501–1,000 | 0 (0) | 0 (0) | 0 (0) | 0 (0) |
| ALT (units/L) |   |   |   |   |
| <100 | 11 (78.6) | 4 (100) | 2 (100) | 17 (85.0) |
| 100–500 | 3 (21.4) | 0 (0) | 0 (0) | 3 (15.0) |
| 501–1,000 | 0 (0) | 0 (0) | 0 (0) | 0 (0) |
| Kidney function |   |   |   |   |
| n (available data) | 15 | 6 | 2 | 23 |
| Acute kidney injury a | 2 (13.3) | 2 (33.3) | 0 (0) | 4 (17.4) |
| Hematologic function |   |   |   |   |
| n (available data) | 15 | 6 | 2 | 23 |
| WBC (/µL) |   |   |   |   |
| <11,000/µL | 11 (73.3) | 2 (33.3) | 0 (0) | 13 (56.5) |
| ≥11,000/µL | 4 (26.7) | 4 (66.7) | 2 (100) | 10 (43.5) |
| Eosinophils (%) |   |   |   |   |
| <5% | 13 (86.7) | 4 (66.7) | 1 (50) | 18 (78.3) |
| ≥5% | 2 (13.3) | 2 (33.3) | 1 (50) | 5 (21.7) |
| Neutrophils (%) |   |   |   |   |
| <80% | 10 (66.7) | 1 (16.7) | 1 (50) | 12 (52.2) |
| ≥80% | 5 (33.3) | 5 (83.3) | 1 (50) | 11 (47.8) |

Table 2 Caption: Laboratory data of beta-lactam antibiotics-related SCARs patients.

### 3.3 Causative beta-lactam antibiotics for SCARs

Three groups of beta-lactam antibiotics were found as causative drugs of SCARs in this present study. Cephalosporins were the most common causative drug of beta-lactam antibiotics-related SCARs, accounting for 58.1% (18/31 patients), followed by penicillins (35.5%, 11/31 patients) and carbapenems (6.5%, 2/31 patients), as shown in [Figure 1](#F1). Based on the phenotypic profile of SCARs, penicillins were the predominant antibiotic class responsible for SJS/TEN (47.6%, 10/21 patients). Among these cases, the primary causative drug was amoxicillin (5/10 patients), succeeded by ampicillin (2/10 patients). Moreover, cephalosporins constituted the prevailing class of causative beta-lactam antibiotics for AGEP, encompassing 87.5% (7/8 patients), as well as DRESS, comprising 100.0% (2/2 patients). In cases of AGEP, the foremost causative drug was ceftriaxone (4/7 patients), followed by ceftazidime, cefotaxime, and cefazolin. The causative drugs of DRESS were ceftriaxone and ceftazidime.

### FIGURE 1.

![FIGURE 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/bfe7/10546186/e01af8ece509/fphar-14-1248386-g001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=10546186_fphar-14-1248386-g001.jpg)

Causative beta-lactam antibiotics for SCARs. (A) Percentages of causative beta-lactam antibiotics-related SCARs in the Thai population; (B) comparison of the causative beta-lactam antibiotics among SCARs phenotypes.

It is noteworthy that among the cohort of 31 patients, 15 individuals were found to have concomitant drug administration during the manifestation of SCARs ([Table 1](#T1)). The evaluation of all concomitant medications was conducted utilizing the established clinical scoring systems, encompassing the Naranjo ADR score for all cases and ALDEN score for SJS/TEN. Notably, it was observed that solely beta-lactam antibiotics exhibited a score indicative of a probable ADR based on both the Naranjo and ALDEN scores.

### 3.4 Associations between genetic polymorphisms of HLA and beta-lactam antibiotics-related SCARs

Genotyping results of *HLA* class I (*HLA-A*, *HLA-B*, and *HLA-C*) and *HLA* class II alleles (*HLA-DRB1*, *HLA-DQA1*, and *HLA-DQB1*) revealed that 34 alleles for *HLA-A*, 53 alleles for *HLA-B*, 36 alleles for *HLA-C*, 29 alleles for *HLA-DRB1*, 10 alleles for *HLA-DQA1*, and 15 alleles for *HLA-DQB1* were identified in this study.

The results from the *HLA* genotyping data of the SCARs group and controls revealed that the carrier frequencies of six *HLA* alleles including *HLA-A*01:01*, *HLA-B*50:01*, *HLA-C*06:02*, *HLA-DRB1*15:01*, *HLA-DQA1*03:01*, and *HLA-DQB1*03:02*, were higher in the SCARs group (*p* < 0.05) ([Table 3](#T3)). Among these six alleles, the highest risk was observed in patients with the *HLA-B*50:01* allele (approximately 12.6-fold significantly higher compared with those who did not carry this allele). Moreover, the higher risk of beta-lactam antibiotics-related SCARs was observed in patients who carried the *HLA-A*01:01* allele (OR = 4.8, 95% CI = 1.0–22.6, *p* = 0.047), the *HLA-C*06:02* allele (OR = 5.7, 95% CI = 1.6–19.9, *p* = 0.011), the *HLA-DRB1*15:01* allele (OR = 2.8, 95% CI = 1.1–7.2, *p* = 0.039), the *HLA-DQA1*03:01* allele (OR = 4.6, 95% CI = 1.4–14.9, *p* = 0.013), or the *HLA-DQB1*03:02* allele (OR = 5.8, 95% CI = 1.5–22.0, *p* = 0.012) ([Table 3](#T3)). However, it should be noted that these associations did not reach a statistically significant difference after Bonferroni’s correction.

### TABLE 3.

| HLA alleles | Controls (n = 183) | SCARs (n = 31) |  |  | SJS/TEN (n = 21) |  |  | AGEP (n = 8) |  |  | DRESS (n = 2) |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Carriers (%) | Carriers (%) | OR [95% CI] | p-value | Carriers (%) | OR [95% CI] | p-value | Carriers (%) | OR [95% CI] | p-value | Carriers (%) | OR [95% CI] | p-value |  |
| HLA-A alleles |  |  |  |  |  |  |  |  |  |  |  |  |  |
| A*11:01 | 78 (42.6) | 8 (25.8) | 0.5 [0.2–1.1] | 0.112 | 3 (14.3) | 0.2 [0.1–0.8] | 0.017 | 3 (37.5) | 0.8 [0.2–3.5] | 1.000 | 2 (100.0) | 4.0 [0.4–39.4 | 0.318 |
| A*01:01 | 4 (2.2) | 3 (9.7) | 4.8 [1.0–22.6] | 0.047 | 2 (9.5) | 4.7 [0.8–27.4] | 0.118 | 1 (12.5) | 6.4 [0.6–64.9] | 0.194 | 0 (0.0) | NA | NA |
| A*02:06 | 2 (1.1) | 2 (6.5) | 6.2 [0.9–46.1] | 0.101 | 2 (9.5) | 9.5 [1.3–71.5] | 0.028 | 0 (0.0) | NA | NA | 0 (0.0) | NA | NA |
| A*02:05 | 0 (0.0) | 1 (3.2) | 11.9 [1.1–134.9] | 0.061 | 0 (0.0) | NA | NA | 1 (12.5) | 46.0 [3.8–561.9] | 0.007 | 0 (0.0) | NA | NA |
| A*02:07 | 59 (32.2) | 1 (3.2) | 0.1 [0.0–0.5] | 3.7 × 10 −4 a | 1 (4.8) | 0.1 [0.0–0.8] | 0.009 | 0 (0.0) | NA | NA | 0 (0.0) | NA | NA |
| A*02:11 | 1 (0.5) | 1 (3.2) | 6.1 [0.37–99.62] | 0.269 | 0 (0.0) | NA | NA | 1 (12.5) | 26.00 [1.5–459.8] | 0.026 | 0 (0.0) | NA | NA |
| A*03:02 | 0 (0.0) | 1 (3.2) | 11.9 [1.1–134.9] | 0.061 | 1 (4.8) | 17.5 [1.5–201.6] | 0.033 | 0 (0.0) | NA | NA | 0 (0.0) | NA | NA |
| HLA-B alleles |  |  |  |  |  |  |  |  |  |  |  |  |  |
| B*57:01 | 2 (1.1) | 2 (6.5) | 6.2 [0.9–46.1] | 0.101 | 2 (9.5) | 9.5 [1.3–71.5] | 0.028 | 0 (0.0) | NA | NA | 0 (0.0) | NA | NA |
| B*37:01 | 0 (0.0) | 1 (3.2) | 11.9 [1.1–134.9] | 0.061 | 0 (0.0) | NA | NA | 1 (12.5) | 46.0 [3.8–561.9] | 0.007 | 0 (0.0) | NA | NA |
| B*38:01 | 0 (0.0) | 1 (3.2) | 11.9 [1.1–134.9] | 0.061 | 0 (0.0) | NA | NA | 1 (12.5) | 46.0 [3.8–561.9] | 0.007 | 0 (0.0) | NA | NA |
| B*46:02 | 0 (0.0) | 1 (3.2) | 11.9 [1.1–134.9] | 0.061 | 1 (4.8) | 17.5 [1.5–201.6] | 0.033 | 0 (0.0) | NA | NA | 0 (0.0) | NA | NA |
| B*50:01 | 1 (0.5) | 2 (6.5) | 12.6 [1.1–142.9] | 0.042 | 0 (0.0) | NA | NA | 2 (25.0) | 60.7 [4.8–765.0] | 0.005 | 0 (0.0) | NA | NA |
| HLA-C alleles |  |  |  |  |  |  |  |  |  |  |  |  |  |
| C*06:02 | 6 (3.3) | 5 (16.1) | 5.7 [1.6–19.9] | 0.011 | 3 (14.3) | 4.9 [1.1–21.4] | 0.034 | 2 (25.0) | 9.8 [1.6–59.2] | 0.038 | 0 (0.0) | NA | NA |
| C*04:06 | 3 (1.6) | 2 (6.5) | 4.1 [0.7–25.8] | 0.154 | 1 (4.8) | 3.0 [0.3–30.2] | 0.355 | 0 (0.0) | NA | NA | 1 (50.0) | 60.0 [3.0–1,202.1] | 0.043 |
| C*03:09 | 0 (0.0) | 1 (3.2) | 11.9 [1.1–134.9] | 0.061 | 0 (0.0) | NA | NA | 1 (12.5) | 46.0 [3.8–561.9] | 0.007 | 0 (0.0) | NA | NA |
| HLA-DRB1 alleles |  |  |  |  |  |  |  |  |  |  |  |  |  |
| DRB1*15:01 | 20 (10.9) | 8 (25.8) | 2.8 [1.1–7.2] | 0.039 | 5 (23.8) | 2.6 [0.8–7.7] | 0.149 | 3 (37.5) | 4.9 [1.1–22.0] | 0.039 | 0 (0.0) | NA | NA |
| DRB1*04:05 | 20 (10.9) | 6 (19.4) | 2.0 [0.7–5.3] | 0.230 | 3 (14.3) | 1.4 [0.4–5.0] | 0.713 | 1 (12.5) | 1.2 [0.1–10.0] | 1.000 | 2 (100.0) | 23.4 [2.3–235.6] | 0.007 |
| DRB1*11:01 | 9 (4.9) | 1 (3.2) | 0.6 [0.1–5.3] | 1.000 | 0 (0.0) | NA | NA | 0 (0.0) | NA | NA | 1 (50.0) | 19.3 [1.1–334.8] | 0.042 |
| HLA-DQA1 alleles |  |  |  |  |  |  |  |  |  |  |  |  |  |
| DQA1*03:01 | 6 (6.0) b | 7 (22.6) | 4.6 [1.4–14.9] | 0.013 | 4 (19.1) | 3.7 [0.9–14.5] | 0.070 | 2 (25.0) | 5.2 [0.9–31.6] | 0.108 | 1 (50.0) | 15.7 [0.9–282.5] | 0.133 |
| HLA-DQB1 alleles |  |  |  |  |  |  |  |  |  |  |  |  |  |
| DQB1*03:02 | 4 (4.0) b | 6 (19.4) | 5.8 [1.5–22.0] | 0.012 | 5 (23.8) | 7.5 [1.8–30.9] | 0.008 | 1 (12.5) | 3.4 [0.3–35.0] | 0.325 | 0 (0) | NA | NA |
| DQB1*04:01 | 11 (11.0) b | 3 (9.7) | 0.9 [0.2–3.3] | 1.000 | 0 (0.0) | NA | NA | 1 (12.5) | 1.2 [0.1–10.3] | 1.000 | 2 (100.0) | 22.5 [2.2–234.0] | 0.009 |

Table 3 Caption: List of odds ratios (ORs) and 95% confidence intervals (CIs) in individuals who carried candidate HLA class I and II alleles compared with the controls.

In contrast, the risk of beta-lactam antibiotics-related SCARs was statistically significantly lower in patients who carried the *HLA-A*02:07* allele (OR = 0.1, 95% CI = 0.0–0.5, *p* = 3.7 × 10^−4^, Pc = 0.012).

In the subgroup analysis, the frequencies of patients who carried *HLA* alleles, including *HLA-A*02:06*, *HLA-A*03:02*, *HLA-B*57:01*, *HLA-B*46:02*, *HLA-C*06:02*, and *HLA-DQB1*03:02*, were significantly higher in the SJS/TEN group compared with the control group (*p* < 0.05) ([Table 3](#T3)). Moreover, the risk of SJS/TEN was approximately 4.9- to 17.5-fold higher in patients who carried one of these six alleles compared with the controls (*HLA-A*02:06* allele, OR = 9.5, 95% CI = 1.3–71.5, *p* = 0.028; *HLA-A*03:02* allele, OR = 17.5, 95% CI = 1.5–201.6, *p* = 0.033; *HLA-B*57:01* allele, OR = 9.5, 95% CI = 1.3–71.5, *p* = 0.028; *HLA-B*46:02* allele, OR = 17.5, 95% CI = 1.5–201.6, *p* = 0.033; *HLA-C*06:02* allele, OR = 4.9, 95% CI = 1.1–21.4, *p* = 0.034; and *HLA-DQB1*03:02* allele, OR = 7.5, 95% CI = 1.8–30.9, *p* = 0.008) ([Table 3](#T3)). Interestingly, the risk of SJS/TEN related to beta-lactam antibiotics was lower in patients who carried either the *HLA-A*11:01* (OR = 0.2, 95% CI = 0.1–0.8, *p* = 0.017) or *HLA-A*02:07* allele (OR = 0.1, 95% CI = 0.0–0.8, *p* = 0.009). However, it should be noted that these associations did not reach a statistically significant difference after Bonferroni’s correction.

For the AGEP group, the carrier frequencies of eight *HLA* alleles, namely, *HLA-A*02:05*, *HLA-A*02:11*, *HLA-B*37:01*, *HLA-B*38:01*, *HLA-B*50:01*, *HLA-C*06:02*, *HLA-C*03:09*, and *HLA-DRB1*15:01*, were significantly higher compared with the control group (*p* < 0.05) ([Table 3](#T3)). Among these eight alleles, the *HLA-B*50:01* allele showed the highest risk of beta-lactam antibiotics-related AGEP*.* The strength of associations is shown in [Table 3](#T3). The frequency of patients who carried the *HLA-B*50:01* allele in the AGEP group was approximately 2.3-fold higher (2/8, 25.0%) compared with that in the control group (1/183, 0.5%), and the risk of AGEP for patients who carried the *HLA-B*50:01* allele was 60.7 (95% CI = 4.8–765.0, *p* = 0.005). However, this did not reach a statistically significant difference after Bonferroni’s correction.

When considering the associations between *HLA* and beta-lactam antibiotics-related DRESS, it was found that four *HLA* alleles including *HLA-C*04:06*, *HLA-DRB1*04:05*, *HLA-DRB1*11:01*, and *HLA-DQB1*04:01*, showed the significantly higher frequencies in the DRESS group compared with the control group (*p* < 0.05) ([Table 3](#T3)). Among these four alleles, the *HLA-C*04:06* allele showed the highest risk of DRESS with an OR of 60.0 (95% CI = 3.0–1202.1, *p* = 0.043). Moreover, the higher risk of DRESS was observed in patients who carried the *HLA-DRB1*04:05* allele (OR = 23.4, 95% CI = 2.3–235.6, *p* = 0.007), the *HLA-DRB1*11:01* allele (OR = 19.3, 95% CI = 1.1–334.8, *p* = 0.042), or the *HLA-DQB1*04:01* allele (OR = 22.5, 95% CI = 2.2–234.0, *p* = 0.009) ([Table 3](#T3)). However, it should be noted that these associations did not reach a statistically significant difference after Bonferroni’s correction.

## 4 Discussion

The majority of SCARs cases identified in the study population were SJS/TEN patients, followed by AGEP and DRESS patients. The most common beta-lactam antibiotics related to SCARs were cephalosporins, followed by penicillins and carbapenems. These findings were inconsistent with a previous study in Taiwan, which reported penicillins as the most common cause of SCARs ([Lin et al., 2014](#B13)). Furthermore, a previous study in Australia indicated that penicillins were the most common cause of cutaneous adverse drug reactions, including SCARs phenotypes ([Trubiano et al., 2016](#B33)). When considering the phenotype of SCARs, penicillins were the most frequent causative antibiotic class for SJS/TEN patients, whereas cephalosporins were the most common drug related to DRESS and AGEP.

The results of the clinical characteristics of SCARs patients in this study demonstrated that most of the SCARs patients were female, and the finding was consistent with the previous studies that the frequency of documented drug allergy is higher in women in drug allergies such as antibiotic allergies in adults ([Blumenthal et al., 2019](#B2)) and allopurinol-induced SCARs ([Saksit et al., 2017b](#B27)). The mean exposure time to beta-lactam antibiotics and the first signs of SCARs (latency) varies depending on SCARs phenotypes. DRESS showed the longest latency among other phenotypes, which was consistent with the previous studies that reported about DRESS caused by co-trimoxazole ([Nakkam et al., 2022b](#B19)), allopurinol ([Saksit et al., 2017b](#B27)), or phenytoin ([Tassaneeyakul et al., 2016](#B35)). In addition, patients with beta-lactam antibiotics-related TEN showed the longest duration of hospital stay for the treatment of SCARs and the highest cost of treatment among other SCARs phenotypes. The mucosal involvements and systemic manifestations including hepatitis and renal insufficiency were mainly noted in the SJS/TEN patients. In contrast, the hematological abnormalities including eosinophilia and neutrophilia were noted in the DRESS and AGEP patients, respectively. Of these 31 cases, only one SJS/TEN overlap patient died in the hospital due to sepsis that may be a complication from the SJS/TEN overlap episodes. This patient was a 69-year-old Thai female patient, prescribed with ceftriaxone for the treatment of infective diarrhea, and norfloxacin and metronidazole were prescribed as concurrent medications, in which only ceftriaxone showed the score of probable ADR based on the Naranjo score and probable SJS/TEN based on the ALDEN score. The SJS/TEN overlap was developed after ceftriaxone exposure for 5 days, and the duration of hospital stay of this patient was 17 days. Interestingly, this patient carried the *HLA-B*57:01* and *HLA-C*06:02* alleles in which these two alleles were proposed as candidate genetic markers for beta-lactam antibiotics-related SJS/TEN in this present study.

To the best of our knowledge, this association study is the first and largest well-defined case–control study of genetic polymorphisms of *HLA* class I and II and beta-lactam antibiotics-related SCARs. Of the *HLA* genetic polymorphisms determined, six *HLA* alleles including *HLA-A*01:01*, *HLA-B*50:01*, *HLA-C*06:02*, *HLA-DRB1*15:01*, *HLA-DQA1*03:01*, and *HLA-DQB1*03:02*, were associated with the increased risk of beta-lactam antibiotics-related SCARs. These results were not in line with the previous study in Thai children, in which *HLA-C*04:06*, *HLA-C*08:01*, and *HLA-DRB1*04:06* were significantly associated with beta-lactam antibiotics-induced delayed hypersensitivity ([Singvijarn et al., 2019](#B31)). It is important to note that the previous study included patients with various phenotypes of delayed hypersensitivity, such as urticaria, maculopapular eruption, fixed drug eruption, SJS, and DRESS ([Singvijarn et al., 2019](#B31)). In the study, there was only one ampicillin/sulbactam-induced SJS patient who carried *HLA-A*11:01/24:02*, *HLA-B*18:01/52:01*, *HLA-C*05:01/12:02*, and *HLA-DRB1*04:03/04:84* alleles and one meropenem-induced DRESS patient who carried *HLA-A*24:02/74:01*, *HLA-B*15:02/15:02*, *HLA-C*08:01/08:01*, and *HLA-DRB1*12:02/14:22* alleles ([Singvijarn et al., 2019](#B31)).

Regarding several *HLA* risk alleles of beta-lactam antibiotics related to SCARs identified in this present study, the linkage disequilibrium (LD) analysis of the study population was also performed. The results showed that there was no significance in the linkage disequilibrium (LD) of any two, three, four, or five alleles of these six *HLA* risk alleles (*HLA-A*01:01*, *HLA-B*50:01*, *HLA-C*06:02*, *HLA-DRB1*15:01*, *HLA-DQA1*03:01*, and *HLA-DQB1*03:02*) (D’ < 1 and *r* ^2^ < 1) (data not shown), suggesting that these identified *HLA* risk alleles were not linkage disequilibrium.

It is well recognized that the immunopathogenesis and clinical characteristics of each phenotype of SCARs are markedly different ([Gibson et al., 2023](#B7)). Some previous studies have proposed specific associations between *HLA* alleles and drug-induced SCARs for certain phenotypes, such as *HLA-B*15:02* for SJS/TEN induced by carbamazepine ([Hung et al., 2006](#B8); [Nakkam et al., 2022a](#B17)) and *HLA-B*13:01* for DRESS induced by co-trimoxazole ([Sukasem et al., 2020](#B32); [Wang et al., 2021](#B38); [Nakkam et al., 2022b](#B19)). Therefore, subgroup analysis based on SCARs phenotypes was performed in the present study, and the results showed that six *HLA* alleles including *HLA-A*02:06*, *HLA-A*03:02*, *HLA-B*57:01*, *HLA-B*46:02, HLA-C*06:02*, and *HLA-DQB1*03:02*, were associated with SJS/TEN. However, eight *HLA* alleles including *HLA-A*02:05*, *HLA-A*02:11*, *HLA-B*37:01*, *HLA-B*38:01*, *HLA-B*50:01*, *HLA-C*06:02*, *HLA-C*03:09*, and *HLA-DRB1*15:01*, were associated with AGEP. Moreover, four *HLA* alleles including *HLA-C*04:06*, *HLA-DRB1*04:05*, *HLA-DRB1*11:01*, and *HLA-DQB1*04:01* were associated with DRESS related to beta-lactam antibiotics.

Among six *HLA* alleles associated with SJS/TEN, the *HLA-DQB1*03:02* allele showed the highest prevalent in SJS/TEN cases (5/21, 23.8%) compared with other alleles, and the higher risk of SJS/TEN was observed in patients who carried this allele (OR = 7.5, 95% CI = 1.8–30.9, *p* = 0.008). Interestingly, when using the *HLA* data from a larger cohort of a general Thai population as a control group ([Satapornpong et al., 2020](#B29)), the association between the *HLA-DQB1*03:02* allele and SJS/TEN group was still noticed, with an OR of 3.7 (95% CI = 1.3–10.5, *p* = 0.026). This finding suggested that the *HLA-DQB1*03:02* allele may play some role in the development of SJS/TEN related to beta-lactam antibiotics.

Previous studies reported that the *HLA-B*57:01* allele was statistically significantly associated with flucloxacillin-induced liver injury ([Daly et al., 2009](#B5); [Puig et al., 2020](#B22)). Flucloxacillin is one of the beta-lactam antibiotics that has been associated with severe immune-mediated drug-induced liver injury caused by an influx of T lymphocytes targeting liver cells potentially recognizing drug-haptenated peptides in the context of *HLA-B*57:01* ([Puig et al., 2020](#B22)). Moreover, the *HLA-B*57:01* allele was reported as a valid genetic marker for abacavir hypersensitivity which is one of T-cell-mediated adverse drug reactions ([Mallal et al., 2002](#B14); [Mallal et al., 2008](#B15)). Consistent with those observed in these previous studies ([Mallal et al., 2002](#B14); [Mallal et al., 2008](#B15); [Daly et al., 2009](#B5); [Puig et al., 2020](#B22)), this present study found that *HLA-B*57:01* was identified as the risk allele for SJS/TEN related to beta-lactam antibiotics (OR = 9.5, 95% CI = 1.3–71.5, *p* = 0.028). According to the immunopathogenesis of these immune-mediated adverse drug reactions, this finding suggested that *HLA-B*57:01* may play some roles in the development of T-cell-mediated adverse reactions induced by beta-lactam antibiotics.

The previous study in a Japanese population reported that *HLA-A*02:06* was strongly associated with cold medicine-related SJS with severe ocular complications, with an OR of 5.7 (*p* = 2.8 × 10^−16^, Pc = 4.8 × 10^−15^) ([Ueta et al., 2014](#B37)), and the result was further confirmed with another Japanese cohort using next-generation sequencing (NGS)-based HLA typing ([Nakatani et al., 2019](#B16)). Interestingly, this present study found the association between *HLA-A*02:06* and beta-lactam antibiotic-induced SJS/TEN, with an OR of 9.5 (95% CI = 1.3–71.5, *p* = 0.028). The association between *HLA-A*02:06* and beta-lactam antibiotics-related SCARs deserves further investigation.

The highest risk of AGEP induced by beta-lactam antibiotics was found in patients who carried the *HLA-B*50:01* allele compared with the controls who did not carry this allele. Moreover, the risk of AGEP was determined using *HLA* data from a larger cohort of a general Thai population as the control group ([Satapornpong et al., 2020](#B29)), and the results remained the same for the *HLA-B*50:01* allele in which the risk of AGEP was statistically significantly higher in the patients who carried the *HLA-B*50:01* allele, with an OR of 201.9 (95% CI = 18.6–2,187.7, *p* = 2.6 × 10^−5^, Pc = 1.4 × 10^−3^). It should be noted that this allele was uncommon in a Thai population, and the allele frequency was approximately 0–0.28% ([Nakkam et al., 2018](#B18); [Satapornpong et al., 2020](#B29)). This finding indicated that the *HLA-B*50:01* allele may play important roles in the development of AGEP related to beta-lactam antibiotics. However, this association deserves further exploration in larger sample size and other ethnicity.

Interestingly, the results from this study demonstrated that the *HLA-C*06:02* allele was apparently associated with both SJS/TEN and AGEP. When combining all phenotypes of SCARs, the *HLA-C*06:02* allele was also observed to be the risk allele for SCARs related to beta-lactam antibiotics, with an OR of 5.7 (95% CI = 1.6–19.9, *p* = 0.011) ([Table 3](#T3)).

The risk of DRESS was observed in patients who carried *HLA-C*04:06*, *HLA-DRB1*04:05*, *HLA-DRB1*11:01*, or *HLA-DQB1*04:01*. Moreover, the risk of beta-lactam antibiotics-related DRESS was significantly higher than that in patients who carried these alleles even when determined using the general Thai population ([Satapornpong et al., 2020](#B29)) as the control group (*HLA-C*04:06*, OR = 41.7, 95% CI = 2.4–711.0, *p* = 0.010; *HLA-DRB1*04:05*, OR = 27.8, 95% CI = 2.8–272.6, *p* = 0.004; *HLA-DRB1*11:01*, OR = 30.3, 95% CI = 1.8–508.4, *p* = 0.018; and *HLA-DQB1*04:01*, OR = 36.3, 95% CI = 3.7–358.2, *p* = 0.002, Pc = 0.029). It should be noted that only two patients with DRESS were enrolled into this study.

Apart from the *HLA* risk alleles of SCARs related to beta-lactam antibiotics, the results from this study found that the *HLA-A*11:01* and *HLA-A*02:07* alleles were more common in the controls and the general Thai population ([Nakkam et al., 2018](#B18); [Satapornpong et al., 2020](#B29)) than in the cases, suggesting that they may protect against beta-lactam antibiotics-related SJS/TEN in this study. Consistent with the previous study in Iranian pediatric patients, *HLA-A*11:01* appeared to have protective effect against drug-related-SJS/TEN with an OR of 0.1 (95% CI = 0.008–0.493, *p* = 0.001), and the most common causative drug in this previous study was beta-lactam antibiotics ([Esmaeilzadeh et al., 2019](#B6)). In addition, it was observed that the *HLA-A*02:07* allele may protect against the development of SJS/TEN induced by co-trimoxazole in the Thai population ([Sukasem et al., 2020](#B32); [Nakkam et al., 2022b](#B19)).

It should be noted that there are limitations to the present study: 1) a relatively small sample size of the SCARs cases due to the rarity of beta-lactam antibiotic-related SCARs; 2) the identification of causative drugs in SCARs cases relied on clinical scoring systems (the Naranjo ADR score for all cases and the ALDEN score for SJS/TEN); and 3) *in vitro* or *in vivo* testing was not performed to validate whether beta-lactam antibiotics were indeed the causative agents.

In conclusion, the present study provided data on the clinical characteristics of beta-lactam antibiotics-related SCARs and demonstrated the candidate *HLA* alleles that showed significant associations with specific phenotypes of these drug-related SCARs in a Thai population. The identified *HLA* alleles may potentially serve as valid markers for SCARs related to beta-lactam antibiotics. However, these associations warrant further exploration in larger sample sizes and among other ethnicities.

## Acknowledgments

The authors thank all patients who participated in the study and Professor James A. Will, University of Wisconsin-Madison, for his valuable comments and critical review of the manuscript via Publication Clinic KKU, Thailand.

## Funding Statement

This work was supported by grants from the Health System Research Institute under Genomics Thailand Strategic Fund (HSRI. 65-038). Pansakon Wattanachai was supported by a Postgraduate Study Support Grant of Faculty of Medicine, Khon Kaen University.

## Data availability statement

The original contributions presented in the study are included in the article/Supplementary Material. Further inquiries can be directed to the corresponding author.

## Ethics statement

The studies involving humans were approved by the Ethics Committee for Human Research, Khon Kaen University, Thailand ([HE510837](https://www.ncbi.nlm.nih.gov/nuccore/HE510837)). The studies were conducted in accordance with the local legislation and institutional requirements. The participants provided their written informed consent to participate in this study.

## Author contributions

NN and WT designed the study. PW, WA, PK, UK, DP, OP, and NN enrolled the patients and collected the samples and clinical data. PW and NN performed the experiments. NN, PW, and WT analyzed and interpreted the data. All authors contributed to the article and approved the submitted version.

## Conflict of interest

The authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.

## Publisher’s note

All claims expressed in this article are solely those of the authors and do not necessarily represent those of their affiliated organizations, or those of the publisher, the editors, and the reviewers. Any product that may be evaluated in this article, or claim that may be made by its manufacturer, is not guaranteed or endorsed by the publisher.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Data Availability Statement

The original contributions presented in the study are included in the article/Supplementary Material. Further inquiries can be directed to the corresponding author.

### Data Availability Statement

The original contributions presented in the study are included in the article/Supplementary Material. Further inquiries can be directed to the corresponding author.

## References

1. Ardlie K. G., Kruglyak L., Seielstad M. (2002). Patterns of linkage disequilibrium in the human genome. Nat. Rev. Genet. 3 (4), 299–309. 10.1038/nrg777  [DOI](https://doi.org/10.1038/nrg777) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11967554/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Genet.&title=Patterns%20of%20linkage%20disequilibrium%20in%20the%20human%20genome&author=K.%20G.%20Ardlie&author=L.%20Kruglyak&author=M.%20Seielstad&volume=3&issue=4&publication_year=2002&pages=299-309&pmid=11967554&doi=10.1038/nrg777&)

2. Blumenthal K. G., Peter J. G., Trubiano J. A., Phillips E. J. (2019). Antibiotic allergy. Lancet 393 (10167), 183–198. 10.1016/S0140-6736(18)32218-9  [DOI](https://doi.org/10.1016/S0140-6736(18)32218-9) | [PMC free article](/articles/PMC6563335/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30558872/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Antibiotic%20allergy&author=K.%20G.%20Blumenthal&author=J.%20G.%20Peter&author=J.%20A.%20Trubiano&author=E.%20J.%20Phillips&volume=393&issue=10167&publication_year=2019&pages=183-198&pmid=30558872&doi=10.1016/S0140-6736(18)32218-9&)

3. Chung W. H., Hung S. I., Hong H. S., Hsih M. S., Yang L. C., Ho H. C., et al. (2004). Medical genetics: A marker for stevens-johnson syndrome. Nature 428 (6982), 486. 10.1038/428486a  [DOI](https://doi.org/10.1038/428486a) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15057820/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Medical%20genetics:%20A%20marker%20for%20stevens-johnson%20syndrome&author=W.%20H.%20Chung&author=S.%20I.%20Hung&author=H.%20S.%20Hong&author=M.%20S.%20Hsih&author=L.%20C.%20Yang&volume=428&issue=6982&publication_year=2004&pages=486&pmid=15057820&doi=10.1038/428486a&)

4. Chung W. H., Wang C. W., Dao R. L. (2016). Severe cutaneous adverse drug reactions. J. Dermatol 43 (7), 758–766. 10.1111/1346-8138.13430  [DOI](https://doi.org/10.1111/1346-8138.13430) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27154258/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Dermatol&title=Severe%20cutaneous%20adverse%20drug%20reactions&author=W.%20H.%20Chung&author=C.%20W.%20Wang&author=R.%20L.%20Dao&volume=43&issue=7&publication_year=2016&pages=758-766&pmid=27154258&doi=10.1111/1346-8138.13430&)

5. Daly A. K., Donaldson P. T., Bhatnagar P., Shen Y., Pe'er I., Floratos A., et al. (2009). HLA-B*5701 genotype is a major determinant of drug-induced liver injury due to flucloxacillin. Nat. Genet. 41 (7), 816–819. 10.1038/ng.379  [DOI](https://doi.org/10.1038/ng.379) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19483685/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet.&title=HLA-B*5701%20genotype%20is%20a%20major%20determinant%20of%20drug-induced%20liver%20injury%20due%20to%20flucloxacillin&author=A.%20K.%20Daly&author=P.%20T.%20Donaldson&author=P.%20Bhatnagar&author=Y.%20Shen&author=I.%20Pe'er&volume=41&issue=7&publication_year=2009&pages=816-819&pmid=19483685&doi=10.1038/ng.379&)

6. Esmaeilzadeh H., Farjadian S., Alyasin S., Nemati H., Nabavizadeh H., Esmaeilzadeh E. (2019). Epidemiology of severe cutaneous adverse drug reaction and its HLA association among pediatrics. Iran. J. Pharm. Res. 18 (1), 506–522.  [PMC free article](/articles/PMC6487414/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31089385/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Iran.%20J.%20Pharm.%20Res.&title=Epidemiology%20of%20severe%20cutaneous%20adverse%20drug%20reaction%20and%20its%20HLA%20association%20among%20pediatrics&author=H.%20Esmaeilzadeh&author=S.%20Farjadian&author=S.%20Alyasin&author=H.%20Nemati&author=H.%20Nabavizadeh&volume=18&issue=1&publication_year=2019&pages=506-522&pmid=31089385&)

7. Gibson A., Deshpande P., Campbell C. N., Krantz M. S., Mukherjee E., Mockenhaupt M., et al. (2023). Updates on the immunopathology and genomics of severe cutaneous adverse drug reactions. J. Allergy Clin. Immunol. 151 (2), 289–300.e4. 10.1016/j.jaci.2022.12.005  [DOI](https://doi.org/10.1016/j.jaci.2022.12.005) | [PMC free article](/articles/PMC9976545/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36740326/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Allergy%20Clin.%20Immunol.&title=Updates%20on%20the%20immunopathology%20and%20genomics%20of%20severe%20cutaneous%20adverse%20drug%20reactions&author=A.%20Gibson&author=P.%20Deshpande&author=C.%20N.%20Campbell&author=M.%20S.%20Krantz&author=E.%20Mukherjee&volume=151&issue=2&publication_year=2023&pages=289-300.e4&pmid=36740326&doi=10.1016/j.jaci.2022.12.005&)

8. Hung S. I., Chung W. H., Jee S. H., Chen W. C., Chang Y. T., Lee W. R., et al. (2006). Genetic susceptibility to carbamazepine-induced cutaneous adverse drug reactions. Pharmacogenet Genomics 16 (4), 297–306. 10.1097/01.fpc.0000199500.46842.4a  [DOI](https://doi.org/10.1097/01.fpc.0000199500.46842.4a) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16538176/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Genetic%20susceptibility%20to%20carbamazepine-induced%20cutaneous%20adverse%20drug%20reactions&author=S.%20I.%20Hung&author=W.%20H.%20Chung&author=S.%20H.%20Jee&author=W.%20C.%20Chen&author=Y.%20T.%20Chang&volume=16&issue=4&publication_year=2006&pages=297-306&pmid=16538176&doi=10.1097/01.fpc.0000199500.46842.4a&)

9. Hung S. I., Chung W. H., Liou L. B., Chu C. C., Lin M., Huang H. P., et al. (2005). HLA-B*5801 allele as a genetic marker for severe cutaneous adverse reactions caused by allopurinol. Proc. Natl. Acad. Sci. U. S. A. 102 (11), 4134–4139. 10.1073/pnas.0409500102  [DOI](https://doi.org/10.1073/pnas.0409500102) | [PMC free article](/articles/PMC554812/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15743917/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A.&title=HLA-B*5801%20allele%20as%20a%20genetic%20marker%20for%20severe%20cutaneous%20adverse%20reactions%20caused%20by%20allopurinol&author=S.%20I.%20Hung&author=W.%20H.%20Chung&author=L.%20B.%20Liou&author=C.%20C.%20Chu&author=M.%20Lin&volume=102&issue=11&publication_year=2005&pages=4134-4139&pmid=15743917&doi=10.1073/pnas.0409500102&)

10. Kardaun S. H., Sidoroff A., Valeyrie-Allanore L., Halevy S., Davidovici B. B., Mockenhaupt M., et al. (2007). Variability in the clinical pattern of cutaneous side-effects of drugs with systemic symptoms: does a DRESS syndrome really exist? Br. J. Dermatol 156 (3), 609–611. 10.1111/j.1365-2133.2006.07704.x  [DOI](https://doi.org/10.1111/j.1365-2133.2006.07704.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17300272/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Dermatol&title=Variability%20in%20the%20clinical%20pattern%20of%20cutaneous%20side-effects%20of%20drugs%20with%20systemic%20symptoms:%20does%20a%20DRESS%20syndrome%20really%20exist?&author=S.%20H.%20Kardaun&author=A.%20Sidoroff&author=L.%20Valeyrie-Allanore&author=S.%20Halevy&author=B.%20B.%20Davidovici&volume=156&issue=3&publication_year=2007&pages=609-611&pmid=17300272&doi=10.1111/j.1365-2133.2006.07704.x&)

11. Karnes J. H., Miller M. A., White K. D., Konvinse K. C., Pavlos R. K., Redwood A. J., et al. (2019). Applications of immunopharmacogenomics: predicting, preventing, and understanding immune-mediated adverse drug reactions. Annu. Rev. Pharmacol. Toxicol. 59 (1), 463–486. 10.1146/annurev-pharmtox010818-021818  [DOI](https://doi.org/10.1146/annurev-pharmtox010818-021818) | [PMC free article](/articles/PMC6409210/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30134124/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annu.%20Rev.%20Pharmacol.%20Toxicol.&title=Applications%20of%20immunopharmacogenomics:%20predicting,%20preventing,%20and%20understanding%20immune-mediated%20adverse%20drug%20reactions&author=J.%20H.%20Karnes&author=M.%20A.%20Miller&author=K.%20D.%20White&author=K.%20C.%20Konvinse&author=R.%20K.%20Pavlos&volume=59&issue=1&publication_year=2019&pages=463-486&pmid=30134124&doi=10.1146/annurev-pharmtox010818-021818&)

12. Krebs K., Bovijn J., Zheng N., Lepamets M., Censin J. C., Jurgenson T., et al. (2020). Genome-wide study identifies association between HLA-B*55:01 and self-reported penicillin allergy. Am. J. Hum. Genet. 107 (4), 612–621. 10.1016/j.ajhg.2020.08.008  [DOI](https://doi.org/10.1016/j.ajhg.2020.08.008) | [PMC free article](/articles/PMC7536643/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32888428/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Hum.%20Genet.&title=Genome-wide%20study%20identifies%20association%20between%20HLA-B*55:01%20and%20self-reported%20penicillin%20allergy&author=K.%20Krebs&author=J.%20Bovijn&author=N.%20Zheng&author=M.%20Lepamets&author=J.%20C.%20Censin&volume=107&issue=4&publication_year=2020&pages=612-621&pmid=32888428&doi=10.1016/j.ajhg.2020.08.008&)

13. Lin Y. F., Yang C. H., Sindy H., Lin J. Y., Rosaline Hui C. Y., Tsai Y. C., et al. (2014). Severe cutaneous adverse reactions related to systemic antibiotics. Clin. Infect. Dis. 58 (10), 1377–1385. 10.1093/cid/ciu126  [DOI](https://doi.org/10.1093/cid/ciu126) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24599767/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Infect.%20Dis.&title=Severe%20cutaneous%20adverse%20reactions%20related%20to%20systemic%20antibiotics&author=Y.%20F.%20Lin&author=C.%20H.%20Yang&author=H.%20Sindy&author=J.%20Y.%20Lin&author=C.%20Y.%20Rosaline%20Hui&volume=58&issue=10&publication_year=2014&pages=1377-1385&pmid=24599767&doi=10.1093/cid/ciu126&)

14. Mallal S., Nolan D., Witt C., Masel G., Martin A. M., Moore C., et al. (2002). Association between presence of HLA-B*5701, HLA-DR7, and HLA-DQ3 and hypersensitivity to HIV-1 reverse-transcriptase inhibitor abacavir. Lancet 359 (9308), 727–732. 10.1016/s0140-6736(02)07873-x  [DOI](https://doi.org/10.1016/s0140-6736(02)07873-x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11888582/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Association%20between%20presence%20of%20HLA-B*5701,%20HLA-DR7,%20and%20HLA-DQ3%20and%20hypersensitivity%20to%20HIV-1%20reverse-transcriptase%20inhibitor%20abacavir&author=S.%20Mallal&author=D.%20Nolan&author=C.%20Witt&author=G.%20Masel&author=A.%20M.%20Martin&volume=359&issue=9308&publication_year=2002&pages=727-732&pmid=11888582&doi=10.1016/s0140-6736(02)07873-x&)

15. Mallal S., Phillips E., Carosi G., Molina J. M., Workman C., Tomazic J., et al. (2008). HLA-B*5701 screening for hypersensitivity to abacavir. N. Engl. J. Med. 358 (6), 568–579. 10.1056/NEJMoa0706135  [DOI](https://doi.org/10.1056/NEJMoa0706135) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18256392/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%20Engl.%20J.%20Med.&title=HLA-B*5701%20screening%20for%20hypersensitivity%20to%20abacavir&author=S.%20Mallal&author=E.%20Phillips&author=G.%20Carosi&author=J.%20M.%20Molina&author=C.%20Workman&volume=358&issue=6&publication_year=2008&pages=568-579&pmid=18256392&doi=10.1056/NEJMoa0706135&)

16. Nakatani K., Ueta M., Khor S. S., Hitomi Y., Okudaira Y., Masuya A., et al. (2019). Identification of HLA-A*02:06:01 as the primary disease susceptibility HLA allele in cold medicine-related Stevens-Johnson syndrome with severe ocular complications by high-resolution NGS-based HLA typing. Sci. Rep. 9 (1), 16240. 10.1038/s41598-019-52619-2  [DOI](https://doi.org/10.1038/s41598-019-52619-2) | [PMC free article](/articles/PMC6838058/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31700100/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&title=Identification%20of%20HLA-A*02:06:01%20as%20the%20primary%20disease%20susceptibility%20HLA%20allele%20in%20cold%20medicine-related%20Stevens-Johnson%20syndrome%20with%20severe%20ocular%20complications%20by%20high-resolution%20NGS-based%20HLA%20typing&author=K.%20Nakatani&author=M.%20Ueta&author=S.%20S.%20Khor&author=Y.%20Hitomi&author=Y.%20Okudaira&volume=9&issue=1&publication_year=2019&pages=16240&pmid=31700100&doi=10.1038/s41598-019-52619-2&)

17. Nakkam N., Konyoung P., Amornpinyo W., Saksit N., Tiamkao S., Khunarkornsiri U., et al. (2022a). Genetic variants associated with severe cutaneous adverse drug reactions induced by carbamazepine. Br. J. Clin. Pharmacol. 88 (2), 773–786. 10.1111/bcp.15022  [DOI](https://doi.org/10.1111/bcp.15022) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34350628/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Clin.%20Pharmacol.&title=Genetic%20variants%20associated%20with%20severe%20cutaneous%20adverse%20drug%20reactions%20induced%20by%20carbamazepine&author=N.%20Nakkam&author=P.%20Konyoung&author=W.%20Amornpinyo&author=N.%20Saksit&author=S.%20Tiamkao&volume=88&issue=2&publication_year=2022a&pages=773-786&pmid=34350628&doi=10.1111/bcp.15022&)

18. Nakkam N., Konyoung P., Kanjanawart S., Saksit N., Kongpan T., Khaeso K., et al. (2018). HLA pharmacogenetic markers of drug hypersensitivity in a Thai population. Front. Genet. 9, 277. 10.3389/fgene.2018.00277  [DOI](https://doi.org/10.3389/fgene.2018.00277) | [PMC free article](/articles/PMC6087736/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30127801/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front.%20Genet.&title=HLA%20pharmacogenetic%20markers%20of%20drug%20hypersensitivity%20in%20a%20Thai%20population&author=N.%20Nakkam&author=P.%20Konyoung&author=S.%20Kanjanawart&author=N.%20Saksit&author=T.%20Kongpan&volume=9&publication_year=2018&pages=277&pmid=30127801&doi=10.3389/fgene.2018.00277&)

19. Nakkam N., Saksit N., Konyoung P., Amornpinyo W., Khunarkornsiri U., Purimart D., et al. (2022b). Associations of HLA and drug-metabolizing enzyme genes in co-trimoxazole-induced severe cutaneous adverse reactions. Drug Metab. Pharmacokinet. 47, 100480. 10.1016/j.dmpk.2022.100480  [DOI](https://doi.org/10.1016/j.dmpk.2022.100480) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36379177/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab.%20Pharmacokinet.&title=Associations%20of%20HLA%20and%20drug-metabolizing%20enzyme%20genes%20in%20co-trimoxazole-induced%20severe%20cutaneous%20adverse%20reactions&author=N.%20Nakkam&author=N.%20Saksit&author=P.%20Konyoung&author=W.%20Amornpinyo&author=U.%20Khunarkornsiri&volume=47&publication_year=2022b&pages=100480&pmid=36379177&doi=10.1016/j.dmpk.2022.100480&)

20. Nicoletti P., Carr D. F., Barrett S., McEvoy L., Friedmann P. S., Shear N. H., et al. (2021). Beta-lactam-induced immediate hypersensitivity reactions: A genome-wide association study of a deeply phenotyped cohort. J. Allergy Clin. Immunol. 147 (5), 1830–1837.e15. 10.1016/j.jaci.2020.10.004  [DOI](https://doi.org/10.1016/j.jaci.2020.10.004) | [PMC free article](/articles/PMC8100096/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33058932/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Allergy%20Clin.%20Immunol.&title=Beta-lactam-induced%20immediate%20hypersensitivity%20reactions:%20A%20genome-wide%20association%20study%20of%20a%20deeply%20phenotyped%20cohort&author=P.%20Nicoletti&author=D.%20F.%20Carr&author=S.%20Barrett&author=L.%20McEvoy&author=P.%20S.%20Friedmann&volume=147&issue=5&publication_year=2021&pages=1830-1837.e15&pmid=33058932&doi=10.1016/j.jaci.2020.10.004&)

21. Phillips E. J., Sukasem C., Whirl-Carrillo M., Muller D. J., Dunnenberger H. M., Chantratita W., et al. (2018). Clinical Pharmacogenetics implementation Consortium guideline for HLA genotype and use of carbamazepine and oxcarbazepine: 2017 update. Clin. Pharmacol. Ther. 103 (4), 574–581. 10.1002/cpt.1004  [DOI](https://doi.org/10.1002/cpt.1004) | [PMC free article](/articles/PMC5847474/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29392710/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Clinical%20Pharmacogenetics%20implementation%20Consortium%20guideline%20for%20HLA%20genotype%20and%20use%20of%20carbamazepine%20and%20oxcarbazepine:%202017%20update&author=E.%20J.%20Phillips&author=C.%20Sukasem&author=M.%20Whirl-Carrillo&author=D.%20J.%20Muller&author=H.%20M.%20Dunnenberger&volume=103&issue=4&publication_year=2018&pages=574-581&pmid=29392710&doi=10.1002/cpt.1004&)

22. Puig M., Ananthula S., Venna R., Kumar Polumuri S., Mattson E., Walker L. M., et al. (2020). Alterations in the HLA-B*57:01 immunopeptidome by flucloxacillin and immunogenicity of drug-haptenated peptides. Front. Immunol. 11, 629399. 10.3389/fimmu.2020.629399  [DOI](https://doi.org/10.3389/fimmu.2020.629399) | [PMC free article](/articles/PMC7900192/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33633747/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front.%20Immunol.&title=Alterations%20in%20the%20HLA-B*57:01%20immunopeptidome%20by%20flucloxacillin%20and%20immunogenicity%20of%20drug-haptenated%20peptides&author=M.%20Puig&author=S.%20Ananthula&author=R.%20Venna&author=S.%20Kumar%20Polumuri&author=E.%20Mattson&volume=11&publication_year=2020&pages=629399&pmid=33633747&doi=10.3389/fimmu.2020.629399&)

23. Romano A., Oussalah A., Chery C., Gueant-Rodriguez R. M., Gaeta F., Cornejo-Garcia J. A., et al. (2022). Next-generation sequencing and genotype association studies reveal the association of HLA-DRB3*02:02 with delayed hypersensitivity to penicillins. Allergy 77 (6), 1827–1834. 10.1111/all.15147  [DOI](https://doi.org/10.1111/all.15147) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34687232/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Allergy&title=Next-generation%20sequencing%20and%20genotype%20association%20studies%20reveal%20the%20association%20of%20HLA-DRB3*02:02%20with%20delayed%20hypersensitivity%20to%20penicillins&author=A.%20Romano&author=A.%20Oussalah&author=C.%20Chery&author=R.%20M.%20Gueant-Rodriguez&author=F.%20Gaeta&volume=77&issue=6&publication_year=2022&pages=1827-1834&pmid=34687232&doi=10.1111/all.15147&)

24. Roujeau J. C. (2005). Clinical heterogeneity of drug hypersensitivity. Toxicology 209 (2), 123–129. 10.1016/j.tox.2004.12.022  [DOI](https://doi.org/10.1016/j.tox.2004.12.022) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15767024/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Toxicology&title=Clinical%20heterogeneity%20of%20drug%20hypersensitivity&author=J.%20C.%20Roujeau&volume=209&issue=2&publication_year=2005&pages=123-129&pmid=15767024&doi=10.1016/j.tox.2004.12.022&)

25. Saito Y., Stamp L. K., Caudle K. E., Hershfield M. S., McDonagh E. M., Callaghan J. T., et al. (2016). Clinical Pharmacogenetics implementation Consortium (CPIC) guidelines for human leukocyte antigen B (HLA-B) genotype and allopurinol dosing: 2015 update. Clin. Pharmacol. Ther. 99 (1), 36–37. 10.1002/cpt.161  [DOI](https://doi.org/10.1002/cpt.161) | [PMC free article](/articles/PMC4675696/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26094938/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Clinical%20Pharmacogenetics%20implementation%20Consortium%20(CPIC)%20guidelines%20for%20human%20leukocyte%20antigen%20B%20(HLA-B)%20genotype%20and%20allopurinol%20dosing:%202015%20update&author=Y.%20Saito&author=L.%20K.%20Stamp&author=K.%20E.%20Caudle&author=M.%20S.%20Hershfield&author=E.%20M.%20McDonagh&volume=99&issue=1&publication_year=2016&pages=36-37&pmid=26094938&doi=10.1002/cpt.161&)

26. Saksit N., Nakkam N., Konyoung P., Khunarkornsiri U., Tassaneeyakul W., Chumworathayi P., et al. (2017a). Comparison between the HLA-B(*)58 : 01 allele and single-nucleotide polymorphisms in chromosome 6 for prediction of allopurinol-induced severe cutaneous adverse reactions. J. Immunol. Res. 2017, 2738784. 10.1155/2017/2738784  [DOI](https://doi.org/10.1155/2017/2738784) | [PMC free article](/articles/PMC5748122/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29392141/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Immunol.%20Res.&title=Comparison%20between%20the%20HLA-B(*)58%20:%2001%20allele%20and%20single-nucleotide%20polymorphisms%20in%20chromosome%206%20for%20prediction%20of%20allopurinol-induced%20severe%20cutaneous%20adverse%20reactions&author=N.%20Saksit&author=N.%20Nakkam&author=P.%20Konyoung&author=U.%20Khunarkornsiri&author=W.%20Tassaneeyakul&volume=2017&publication_year=2017a&pages=2738784&pmid=29392141&doi=10.1155/2017/2738784&)

27. Saksit N., Tassaneeyakul W., Nakkam N., Konyoung P., Khunarkornsiri U., Chumworathayi P., et al. (2017b). Risk factors of allopurinol-induced severe cutaneous adverse reactions in a Thai population. Pharmacogenet Genomics 27 (7), 255–263. 10.1097/FPC.0000000000000285  [DOI](https://doi.org/10.1097/FPC.0000000000000285) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28509689/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Risk%20factors%20of%20allopurinol-induced%20severe%20cutaneous%20adverse%20reactions%20in%20a%20Thai%20population&author=N.%20Saksit&author=W.%20Tassaneeyakul&author=N.%20Nakkam&author=P.%20Konyoung&author=U.%20Khunarkornsiri&volume=27&issue=7&publication_year=2017b&pages=255-263&pmid=28509689&doi=10.1097/FPC.0000000000000285&)

28. Sassolas B., Haddad C., Mockenhaupt M., Dunant A., Liss Y., Bork K., et al. (2010). ALDEN, an algorithm for assessment of drug causality in stevens-johnson syndrome and toxic epidermal necrolysis: comparison with case-control analysis. Clin. Pharmacol. Ther. 88 (1), 60–68. 10.1038/clpt.2009.252  [DOI](https://doi.org/10.1038/clpt.2009.252) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20375998/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=ALDEN,%20an%20algorithm%20for%20assessment%20of%20drug%20causality%20in%20stevens-johnson%20syndrome%20and%20toxic%20epidermal%20necrolysis:%20comparison%20with%20case-control%20analysis&author=B.%20Sassolas&author=C.%20Haddad&author=M.%20Mockenhaupt&author=A.%20Dunant&author=Y.%20Liss&volume=88&issue=1&publication_year=2010&pages=60-68&pmid=20375998&doi=10.1038/clpt.2009.252&)

29. Satapornpong P., Jinda P., Jantararoungtong T., Koomdee N., Chaichan C., Pratoomwun J., et al. (2020). Genetic diversity of HLA class I and class II alleles in Thai populations: Contribution to genotype-guided therapeutics. Front. Pharmacol. 11, 78. 10.3389/fphar.2020.00078  [DOI](https://doi.org/10.3389/fphar.2020.00078) | [PMC free article](/articles/PMC7057685/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32180714/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front.%20Pharmacol.&title=Genetic%20diversity%20of%20HLA%20class%20I%20and%20class%20II%20alleles%20in%20Thai%20populations:%20Contribution%20to%20genotype-guided%20therapeutics&author=P.%20Satapornpong&author=P.%20Jinda&author=T.%20Jantararoungtong&author=N.%20Koomdee&author=C.%20Chaichan&volume=11&publication_year=2020&pages=78&pmid=32180714&doi=10.3389/fphar.2020.00078&)

30. Sidoroff A., Halevy S., Bavinck J. N., Vaillant L., Roujeau J. C. (2001). Acute generalized exanthematous pustulosis (AGEP)--a clinical reaction pattern. J. Cutan. Pathol. 28 (3), 113–119. 10.1034/j.1600-0560.2001.028003113.x  [DOI](https://doi.org/10.1034/j.1600-0560.2001.028003113.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11168761/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Cutan.%20Pathol.&title=Acute%20generalized%20exanthematous%20pustulosis%20(AGEP)--a%20clinical%20reaction%20pattern&author=A.%20Sidoroff&author=S.%20Halevy&author=J.%20N.%20Bavinck&author=L.%20Vaillant&author=J.%20C.%20Roujeau&volume=28&issue=3&publication_year=2001&pages=113-119&pmid=11168761&doi=10.1034/j.1600-0560.2001.028003113.x&)

31. Singvijarn P., Manuyakorn W., Mahasirimongkol S., Wattanapokayakit S., Inunchot W., Wichukchinda N., et al. (2019). Association of HLA genotypes with Beta-lactam antibiotic hypersensitivity in children. Asian Pac J. Allergy Immunol. 39, 197–205. 10.12932/AP-271118-0449  [DOI](https://doi.org/10.12932/AP-271118-0449) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31012593/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Asian%20Pac%20J.%20Allergy%20Immunol.&title=Association%20of%20HLA%20genotypes%20with%20Beta-lactam%20antibiotic%20hypersensitivity%20in%20children&author=P.%20Singvijarn&author=W.%20Manuyakorn&author=S.%20Mahasirimongkol&author=S.%20Wattanapokayakit&author=W.%20Inunchot&volume=39&publication_year=2019&pages=197-205&pmid=31012593&doi=10.12932/AP-271118-0449&)

32. Sukasem C., Pratoomwun J., Satapornpong P., Klaewsongkram J., Rerkpattanapipat T., Rerknimitr P., et al. (2020). Genetic association of Co-Trimoxazole-Induced severe cutaneous adverse reactions is phenotype-specific: HLA class I genotypes and haplotypes. Clin. Pharmacol. Ther. 108 (5), 1078–1089. 10.1002/cpt.1915  [DOI](https://doi.org/10.1002/cpt.1915) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32452529/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Genetic%20association%20of%20Co-Trimoxazole-Induced%20severe%20cutaneous%20adverse%20reactions%20is%20phenotype-specific:%20HLA%20class%20I%20genotypes%20and%20haplotypes&author=C.%20Sukasem&author=J.%20Pratoomwun&author=P.%20Satapornpong&author=J.%20Klaewsongkram&author=T.%20Rerkpattanapipat&volume=108&issue=5&publication_year=2020&pages=1078-1089&pmid=32452529&doi=10.1002/cpt.1915&)

33. Trubiano J. A., Aung A. K., Nguyen M., Fehily S. R., Graudins L., Cleland H., et al. (2016). A comparative analysis between antibiotic- and nonantibiotic-associated delayed cutaneous adverse drug reactions. J. Allergy Clin. Immunol. Pract. 4 (6), 1187–1193. 10.1016/j.jaip.2016.04.026  [DOI](https://doi.org/10.1016/j.jaip.2016.04.026) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27283055/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Allergy%20Clin.%20Immunol.%20Pract.&title=A%20comparative%20analysis%20between%20antibiotic-%20and%20nonantibiotic-associated%20delayed%20cutaneous%20adverse%20drug%20reactions&author=J.%20A.%20Trubiano&author=A.%20K.%20Aung&author=M.%20Nguyen&author=S.%20R.%20Fehily&author=L.%20Graudins&volume=4&issue=6&publication_year=2016&pages=1187-1193&pmid=27283055&doi=10.1016/j.jaip.2016.04.026&)

34. Tassaneeyakul W., Jantararoungtong T., Chen P., Lin P. Y., Tiamkao S., Khunarkornsiri U., et al. (2009). Strong association between HLA-B*5801 and allopurinol-induced Stevens-Johnson syndrome and toxic epidermal necrolysis in a Thai population. Pharmacogenet Genomics 19 (9), 704–709. 10.1097/FPC.0b013e328330a3b8  [DOI](https://doi.org/10.1097/FPC.0b013e328330a3b8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19696695/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Strong%20association%20between%20HLA-B*5801%20and%20allopurinol-induced%20Stevens-Johnson%20syndrome%20and%20toxic%20epidermal%20necrolysis%20in%20a%20Thai%20population&author=W.%20Tassaneeyakul&author=T.%20Jantararoungtong&author=P.%20Chen&author=P.%20Y.%20Lin&author=S.%20Tiamkao&volume=19&issue=9&publication_year=2009&pages=704-709&pmid=19696695&doi=10.1097/FPC.0b013e328330a3b8&)

35. Tassaneeyakul W., Prabmeechai N., Sukasem C., Kongpan T., Konyoung P., Chumworathayi P., et al. (2016). Associations between HLA class I and cytochrome P450 2C9 genetic polymorphisms and phenytoin-related severe cutaneous adverse reactions in a Thai population. Pharmacogenet Genomics 26 (5), 225–234. 10.1097/FPC.0000000000000211  [DOI](https://doi.org/10.1097/FPC.0000000000000211) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26928377/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Associations%20between%20HLA%20class%20I%20and%20cytochrome%20P450%202C9%20genetic%20polymorphisms%20and%20phenytoin-related%20severe%20cutaneous%20adverse%20reactions%20in%20a%20Thai%20population&author=W.%20Tassaneeyakul&author=N.%20Prabmeechai&author=C.%20Sukasem&author=T.%20Kongpan&author=P.%20Konyoung&volume=26&issue=5&publication_year=2016&pages=225-234&pmid=26928377&doi=10.1097/FPC.0000000000000211&)

36. Tassaneeyakul W., Tiamkao S., Jantararoungtong T., Chen P., Lin S. Y., Chen W. H., et al. (2010). Association between HLA-B*1502 and carbamazepine-induced severe cutaneous adverse drug reactions in a Thai population. Epilepsia 51 (5), 926–930. 10.1111/j.1528-1167.2010.02533.x  [DOI](https://doi.org/10.1111/j.1528-1167.2010.02533.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20345939/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Epilepsia&title=Association%20between%20HLA-B*1502%20and%20carbamazepine-induced%20severe%20cutaneous%20adverse%20drug%20reactions%20in%20a%20Thai%20population&author=W.%20Tassaneeyakul&author=S.%20Tiamkao&author=T.%20Jantararoungtong&author=P.%20Chen&author=S.%20Y.%20Lin&volume=51&issue=5&publication_year=2010&pages=926-930&pmid=20345939&doi=10.1111/j.1528-1167.2010.02533.x&)

37. Ueta M., Kaniwa N., Sotozono C., Tokunaga K., Saito Y., Sawai H., et al. (2014). Independent strong association of HLA-A*02:06 and HLA-B*44:03 with cold medicine-related Stevens-Johnson syndrome with severe mucosal involvement. Sci. Rep. 4, 4862. 10.1038/srep04862  [DOI](https://doi.org/10.1038/srep04862) | [PMC free article](/articles/PMC5381277/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24781922/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&title=Independent%20strong%20association%20of%20HLA-A*02:06%20and%20HLA-B*44:03%20with%20cold%20medicine-related%20Stevens-Johnson%20syndrome%20with%20severe%20mucosal%20involvement&author=M.%20Ueta&author=N.%20Kaniwa&author=C.%20Sotozono&author=K.%20Tokunaga&author=Y.%20Saito&volume=4&publication_year=2014&pages=4862&pmid=24781922&doi=10.1038/srep04862&)

38. Wang C. W., Tassaneeyakul W., Chen C. B., Chen W. T., Teng Y. C., Huang C. Y., et al. (2021). Whole genome sequencing identifies genetic variants associated with co-trimoxazole hypersensitivity in Asians. J. Allergy Clin. Immunol. 147 (4), 1402–1412. 10.1016/j.jaci.2020.08.003  [DOI](https://doi.org/10.1016/j.jaci.2020.08.003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32791162/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Allergy%20Clin.%20Immunol.&title=Whole%20genome%20sequencing%20identifies%20genetic%20variants%20associated%20with%20co-trimoxazole%20hypersensitivity%20in%20Asians&author=C.%20W.%20Wang&author=W.%20Tassaneeyakul&author=C.%20B.%20Chen&author=W.%20T.%20Chen&author=Y.%20C.%20Teng&volume=147&issue=4&publication_year=2021&pages=1402-1412&pmid=32791162&doi=10.1016/j.jaci.2020.08.003&)

39. Wang Y. H., Chen C. B., Tassaneeyakul W., Saito Y., Aihara M., Choon S. E., et al. (2019). The medication risk of stevens-johnson syndrome and toxic epidermal necrolysis in asians: the major drug causality and comparison with the US FDA label. Clin. Pharmacol. Ther. 105 (1), 112–120. 10.1002/cpt.1071  [DOI](https://doi.org/10.1002/cpt.1071) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29569740/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=The%20medication%20risk%20of%20stevens-johnson%20syndrome%20and%20toxic%20epidermal%20necrolysis%20in%20asians:%20the%20major%20drug%20causality%20and%20comparison%20with%20the%20US%20FDA%20label&author=Y.%20H.%20Wang&author=C.%20B.%20Chen&author=W.%20Tassaneeyakul&author=Y.%20Saito&author=M.%20Aihara&volume=105&issue=1&publication_year=2019&pages=112-120&pmid=29569740&doi=10.1002/cpt.1071&)

40. Yang J., Qiao H. L., Zhang Y. W., Jia L. J., Tian X., Gao N. (2006). HLA-DRB genotype and specific IgE responses in patients with allergies to penicillins. Chin. Med. J. Engl. 119 (6), 458–466. 10.1097/00029330-200603020-00005  [DOI](https://doi.org/10.1097/00029330-200603020-00005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16584643/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chin.%20Med.%20J.%20Engl.&title=HLA-DRB%20genotype%20and%20specific%20IgE%20responses%20in%20patients%20with%20allergies%20to%20penicillins&author=J.%20Yang&author=H.%20L.%20Qiao&author=Y.%20W.%20Zhang&author=L.%20J.%20Jia&author=X.%20Tian&volume=119&issue=6&publication_year=2006&pages=458-466&pmid=16584643&doi=10.1097/00029330-200603020-00005&)
