# Precision therapeutic opioid dosing implications from genetic biomarkers and craving score

## Metadata
**Authors:** Hsin-Wen Chang, Wen-Chao Ho, Chieh-Liang Huang, Ruey-Yun Wang
**Journal:** Medicine
**Date:** 2020 May 29
**DOI:** [10.1097/MD.0000000000020429](https://doi.org/10.1097/MD.0000000000020429)
**PMID:** 32481444
**PMCID:** PMC12245365
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC12245365/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC12245365/pdf/medi-99-e20429.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC12245365/pdf/medi-99-e20429.pdf)

## Abstract

Supplemental Digital Content is available in the text

Keywords: genetic risk score, methadone maintenance treatment, prediction, quantitative trait locus

## 1. Introduction

Heroin addiction, a chronic relapsing disease characterized by compulsive drug seeking, drug abuse, tolerance, and physical dependence, is a major public health concern worldwide.^[1]^ Methadone, a synthetic racemic opioid that primarily functions as a μ-opioid receptor agonist,^[2]^ is commonly used to treat heroin addiction; it reduces cravings, withdrawal symptoms, risky behavior incidence, and relapse risk.^[3]^ When prescribing methadone, the dosing requires considerable care: excessive methadone doses are dangerous, whereas overly conservative doses are ineffective at preventing relapse to heroin use.^[4]^ However, the effective methadone maintenance therapy (MMT) dose varies for every individual and is the key to safe and successful treatment for heroin addiction.^[5]^ The clinically optimal dose should be determined on the basis of clinical signs and symptoms, which is a time-consuming process. MMT remains a crucial treatment strategy for heroin addiction.

Methadone is extensively metabolized by cytochrome P450 (*CYP*) in the liver.^[6–8]^ Several studies have suggested that numerous genetic variants may play influential roles in the etiology of methadone metabolism and μ-opioid receptors.^[7,9,10]^ Furthermore, other genes of the opioid system or genes involved in serotoninergic, dopaminergic, or noradrenergic pathways have been associated with MMT.^[5]^ Polymorphisms in genes related to these pathways may contribute to interindividual variability in optimal MMT dose and have also been associated with different aspects of drug addiction.^[11]^ However, genetic factors, which may play a key role in determining the methadone dosage, are generally not evaluated in clinical practice.

To identify genetic variants for MMT dosage that underlie heroin addiction and methadone metabolism, we performed a quantitative trait locus (QTL) association study using single nucleotide polymorphisms (SNPs) selected from candidate genes. This approach is based on physiological hypotheses in which the candidate genes were selected on the basis of their function (eg, opioidergic, dopaminergic, and serotonergic, and ethadone-metabolizing) and the related pathways (eg, cognitive function). We focused on 13 genes encoding opioidergic components (*OPRK1*, *OPRL1*, *PDYN*, and cannabinoid receptor 1 [*CNR1*]), dopaminergic and serotonin components (*COMT* and *tryptophan hydroxylases 1 [*TPH1*] and 2* [*TPH2*]), methadone-metabolizing enzymes (*CYP1A2, CYP2B6,* and *CYP2C1*), as well as cognitive function–related genes (*GRIN3A*, *GRIN3B*, and *GRM6*). To maximize the power of the study, cases were selected from the extreme margin of the specific phenotype range (eg, severe heroin addicts receiving MMT). We combined several genotypic and phenotypic factors that could potentially serve as predictors of clinically optimal MMT dose for personalized analgesic prescription.

## 2. Materials and methods

### 2.1. Patients

The study protocol was reviewed and approved by the Institutional Review Board of China Medical University Hospital (CMUH) (DMR101-IRB1-218) and was in compliance with the Declaration of Helsinki. In total, 316 heroin-dependent patients undergoing MMT were recruited at the Addiction Center of the CMUH. As assessed by senior psychiatrists experienced in heroin dependence, all recruited patients met Diagnostic and Statistical Manual of Mental Disorders, 4th Edition, criteria for heroin abuse and dependence and had been receiving MMT for at least 6 months with an unchanged dose for at least 4 weeks at CMUH. Moreover, the recruited patients provided written informed consent, had normal electrocardiography results, and were not using concurrent medications that possibly affected methadone metabolism. For each patient, the following clinical information was recorded: sex, weight (in kg), height (in cm), liver function test results, comorbidities, and daily methadone dose.

### 2.2. Heroin use and craving questionnaire

We designed a 14-item heroin use and craving (HUC) questionnaire to evaluate the behavior of heroin users undergoing MMT. Each question was scored from 0 to 4. The questionnaire was divided into 2 parts: Part I (HUC 1–6, possible total scores: 0–24), assessing the urge for heroin and whether the individual can shift the attention away from heroin, and Part II (HUC 7–14, possible total scores: 0 to 32), investigating the daily or weekly frequency of heroin usage, daily life disturbance, anxiety, and ability to overcome heroin use. In both the part, the higher the score, the more severe the heroin craving was. The score of each item was calculated according to the manual.

### 2.3. Candidate variants selection and genotyping

This study focused on the 13 aforementioned genes encoding various metabolic components: *OPRK1*, *OPRL1*, *PDYN*, *CNR1*, *COMT*, *TPH1*, *TPH2*, *CYP1A2, CYP2B6*, *CYP2C1*, *GRIN3A*, *GRIN3B*, and *GRM6*. SNPs with minor allele frequency >0.05 were selected on the basis of the previous findings and population data in SNP databases of the National Center for Biotechnology Information Han Chinese in Beijing, China In total, 44 SNPs were genotyped at the National Center for Genome Medicine, Taiwan by using Sequenom iPLEX matrix-assisted laser desorption/ionization time-of-flight mass-spectrometry technology. DNA was extracted from 3 to 10 mL of whole blood by using QIAamp DNA Blood Mini Kit (Qiagen, Valencia, CA) according to the manufacturer's protocol. Duplicate samples were randomly selected for quality control and the concordance rate was >0.99 for all SNPs assayed.

We calculated an SNP risk score based on estimates of the odds ratio per allele and risk allele frequencies assuming independence of additive risks (22). A score of 1 was given to each T allele of rs806368 in *CNR1* and each C allele of rs1386493 in *TPH2*, rs16974799 in *CYP2B6*, and rs2229205 in *OPRL1*. The overall genetic risk score (GRS) was then calculated by using the SNP risk alleles.

### 2.4. Statistical analysis

Each SNP genotype frequency distribution was examined for Hardy–Weinberg equilibrium by using the chi-square 1 degree of freedom goodness-of-fit test. Demographic characteristics and clinical parameters were evaluated with a chi-squared contingency table for categorical variables and *t* test for continuous variables for both sexes. We performed QTL mapping for maximum methadone maintenance dose. The general linear model was used to associate maximum methadone maintenance dose with genotype data, making adjustments for sex, age, and body mass index (BMI). GRSs ranged from 0 to 4. A multinomial logistic regression model including GRS and craving score, adjusted for covariates, was applied to examine the association between maximum methadone maintenance dose and genetic variants. The data were randomly separated into training and testing sets. In both splits, the training and testing data generally follow similar characteristics distributions without significant differences (Supplementary Table 1, [http://links.lww.com/MD/E366](http://links.lww.com/MD/E366)). In order to enhance the prediction accuracy and the reliability of the prediction model, we calculated the areas under the receiver operating characteristic curves (AUROCs) to compare the accuracy of maximum MMT dose. Sensitivity, specificity, and AUROCs were used to evaluate model performance in both training and testing sets. The optimal cutoff risk score threshold was identified at which both sensitivity and specificity were maximized (sensitivity + specificity). In addition, AUROC loss by stepwise selection was also performed to investigate critical and determining factors of the model.

## 3. Results

### 3.1. Patient characteristics

No significant differences were observed between male and female patients with respect to their HUC questionnaire score, maximum dose, heroin use duration, education level, and marital status, but significant differences were noted in the mean age, BMI, and heroin onset age (Table [1](#T1)).

### Table 1.

Table 1 Caption: Comparing characteristics of the study subjects.

In total, 40 SNPs in 10 genes were not associated with maximum MMT dose (Supplementary Table 2, [http://links.lww.com/MD/E366](http://links.lww.com/MD/E366)). Only 4 SNPs, rs806368 in *CNR1*, s1386493 in *TPH2*, s16974799 in *CYP2B6*, and rs2229205 in *OPRL1*, were significantly associated with maximum MMT dose (*P* < .05; Table [2](#T2))

### Table 2.

Table 2 Caption: Association between selected SNPs and maximum stabilized methadone daily doses.

We calculated the GRS by using 4 significant SNPs from Table [3](#T3). The univariate associations of GRS and the associations with maximum MMT dose are presented in a dose–response manner in Table [3](#T3). The GRS was associated with maximum MMT dose. Even after adjustments for age, sex, and BMI, the GRS remained independently associated with maximum MMT dose.

### Table 3.

Table 3 Caption: Predicted maximum stabilized methadone daily doses for genetic risk scores.

### 3.2. Selection of predictive variables for maximum MMT dose

On the basis of the results presented in Table [3](#T3), we conducted stepwise, backward and forward model analysis to select the determining factors for the best fit prediction model of maximum MMT dose. The results are displayed in Table [4](#T4). *CNR1*, *TPH2*, *CYP2B6*, *OPRL1*, and craving were included in the best fit model, with *P* set at <.05.

### Table 4.

Table 4 Caption: Selection of predictive variables for maximum dose of MMT.

In order to enhance the prediction accuracy and the reliability of the prediction model, the dataset was randomly split into training [229 (70%) subjects] and testing [97(30%) subjects] sets. Logistic regression model were trained and optimized using the training set and their performances was evaluated using the test set. This performance was replicated on the testing set and has a similar AUROC for both training set (AUROC = 0.75) and testing set (AUROC = 0.81) (Supplementary table 3, [http://links.lww.com/MD/E366](http://links.lww.com/MD/E366)).

The prediction accuracy of the maximum MMT dose in the combined dataset is presented in Table [5](#T5). The diagnostic ability for the GRS and craving was measured using AUROC in the prediction models. The AUROCs of GRS and craving score were 0.69 (95% CI: 0.61–0.77; *P* < .0001) and 0.65 (95% CI: 0.56–0.73; *P* = .0002), respectively. The AUROC of the combined GRS and craving score was a significant predictor for maximum MMT dose, with 75% sensitivity and 60% specificity (Table [5](#T5); Fig. [1](#F1)).

### Table 5.

Table 5 Caption: ROC curves for the GRS and craving were used to predict maximum stabilized methadone daily doses.

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1aab/12245365/5ae584aa7eec/medi-99-e20429-g006.jpg)

Receiver operating characteristic curves and comparison of areas under the ROC curves for the GRS and craving were used to predict maximum dose of MMT. GRS = genetic risk score, MMT = methadone maintenance treatment.

## 4. Discussion

The major findings of our study were as follows:

We observed that rs806368 in *CNR1*, rs1386493 in *TPH2*, rs16974799 in *CYP2B6*, and rs2229205 in *OPRL1* were significantly associated with maximum MMT dose. Methadone has been studied most intensively in heroin addiction pharmacogenetics research.^[12]^ Several *ABCB1, CYP2B6,* and *OPRD1* variants have been associated with methadone plasma levels in candidate gene studies.^[12,13]^ In addition, 2 genome wide association studies (GWAS) identified significant associations of *CYP2B6* and *OPRM1* with methadone dose.^[14,15]^ Because of the chemical structure of methadone, the isozymes CYP2C19 and CYP2B6 metabolize the R-enantiomer^[16]^ and S-enantiomer^[17,18]^ of methadone in the liver, respectively. Genetics can play a role in methadone metabolism and the resulting plasma concentrations. These variants can serve as surrogate markers for MMT response and efficacy.

To the best of our knowledge, the present study is the first to report the genetic association of *CNR1* and *TPH2* with maximum MMT dose. *CNR1* on chromosome 6q14–15 encodes CB1, which likely modulates the release of serotonin, norepinephrine, and dopamine, disturbances of which are well documented in depression^[19]^ and addictive disorders.^[20]^*CNR1* SNPs are associated with nicotine,^[21]^ marijuana,^[22]^ alcohol,^[23]^ cocaine,^[24]^ and polysubstance use disorder.^[25]^ Moreover, we found that rs806368 in CNR1 is associated with maximum MMT dose and heroin addiction risk—consistent with the results of a study on CNR1 and its influence on lifetime major depressive disorder and suicidal behavior in a population of opiate-dependent outpatients remitted under stable methadone treatment.^[26]^

We also observed that patients with specific genotypes of *TPH2* are associated with maximum stabilized methadone daily dose. TPH2 activation may be considered a new prospective in the treatment of neuropsychiatric diseases related to brain 5-HT levels.^[27]^ TPH2 *gene* variants are related to the efficacy of disulfiram treatment for cocaine addiction.^[28]^ Because cocaine's mechanism of action involves blocking dopamine, norepinephrine, and serotonin transporters,^[29]^ we also evaluated the role of a functional variant in TPH2. We observed that patients with the CC genotype of TPH2 exhibited more favorable responses to methadone than did those patients without this genotype. This TPH2 variant has also been associated with heroin addiction^[30]^ and quality of life in the MMT population.^[31]^

We integrated genetic variations of known pharmacokinetic and pharmacodynamic genes as well as additional genes and craving scores to predict maximum MMT dosage. The genetic risk, calculated using the individual common genetic variants, was modest; their combination resulted in a large effect on maximum MMT dosage. However, GRS with higher frequencies were rare, suggesting that variants with large effect sizes might affect a small proportion of the population. Our genetic model was associated with higher methadone dose, suggesting that combining the GRS and craving score may help identify an adequate MMT dosage for individuals. The prediction model may be useful in defining individual MMT dose, to suggest initiation of methadone treatment, and at the start of treatment to predict the optimal dose for a particular patient. The strengths of our study include comprehensive addiction scale information and the utilization of multivariate logistic regression analysis to assess the combinatory effect of genetic polymorphisms on maximum MMT dosage.

Several limitations of this study must also be acknowledged. First, our sample size was small than that applied in previous related case–control studies. We performed a QTL association to identify genetic variants for MMT dosage. Our study did not include a control group; therefore, we suggest that larger-scale studies should be conducted to confirm our prediction model in practice without a genetic risk population.

The AUROCs for GRS and craving score were 0.69 and 0.65, respectively, but their sensitivity and specificity were low. Our study had only limited phenotypic and genotypic information. Moreover, the candidate gene approach allows the scanning of a limited number of SNPs. The identification of more genes associated with MMT dosage is imperative. This should include candidate genes identified in GWAS^[14,15,32,33]^ and variants underlying heroin addiction to increase accuracy in future studies.

Although the variants identified in this study produce relative risk for the response to MMT dosage, they may uncover potential mechanisms of addictive behavior. Future studies with patients undergoing methadone treatment could investigate variability in treatment response under the influence of environmental factors (eg, other pharmacological treatments, toxic exposure, harmful or safe physical and psychological environment, and lifestyle) and acting through epigenetic modifications and gene regulation.

## 5. Conclusion

Four susceptibility SNPs are associated with maximum MMT dosage.

### 5.1. HUC questionnaire

This study designed a 14-item questionnaires, heroin using and craving (HUC) score to evaluate the behaviours, for heroin abusers received methadone maintenance therapy. Each question scores 0 to 4. The questionnaire is divided into 2 parts: part I (HUC 1–HUC 6) is to assess the urge for heroin and whether one can shift the attention from heroin. The part I sum score is calculated range from 0 to 24. The part II (HUC 7–HUC 14) is to investigate the daily or weekly frequencies of heroin usage, daily life disturbance, anxiety emotion and the ability to overcome heroin use. The part II sum calculated score ranges from 0 to 32. The higher the score, the more severe the cravings for heroin use. The score of each item would be calculated according to the manual.

### 5.2. Candidate variants selection and genotyping

This study focused on 13 genes encoding components of the ( *OPRK1, OPRL1, PDYN* and CNR1), the dopaminergic and serotonin (*COMT*, TPH1 and TPH2) and Methadone-metabolizing enzymes (*CYP1A2, CYP2B6 and CYP2C1*), as well as the Cognitive function (*GRIN3A, GRIN3B and GRM6*). SNPs with minor allele frequency greater than 0.05 were selected based on previous studies and SNP databases of the National Center for Biotechnology Information CBH (Han Chinese in Beijing, China) population data. A total of 44 SNPs were genotyping at the National Center for Genome Medicine, Taiwan by the Sequenom iPLEX matrix-assisted laser desorption/ionization time-of-flight mass-spectrometry technology. DNA was extracted from 3 to 10 mL of whole blood by using the QIAamp DNA Blood mini Kit (Qiagen, Valencia, CA) according to the manufacturer's protocol. Duplicates samples were randomly selected for quality control and the concordance rate was >0.99 for all SNPs assayed.

We calculated an SNP risk score based on estimates of the OR per allele and risk allele frequencies (p) assuming independence of additive risks (22). A score of 1 was given to each T allele of *CNR1* - rs806368 and each C allele of *TPH2*- rs1386493, C allele of *CYP2B6* - rs16974799 and each C allele of *OPRL1* -rs2229205. The overall GRS was then calculated by the SNP risk alleles.

### 5.3. Statistical analysis

Each SNP genotype frequency distribution was examined for Hardy-Weinberg equilibrium by using the chi-square 1-degree of freedom goodness-of-fit test. Demographic characteristics and clinical parameters were evaluated with a chi-squared contingency table for categorical variables and t test for continuous variables between the gender groups. We performed QTL mapping for max methadone maintenance dose. The general linear model was used for associating max methadone maintenance dose with genotype data, making adjustments for sex, age, and BMI. The ranges of GRSs were from 0 to 4. A multinomial logistic regression model including GRS and craving interaction term adjusted for covariates was applied to examine in an association between max methadone maintenance dose and genetic variants. We employed AUROC curves to compare the accuracy of maximum dose of MMT. Sensitivity, specificity, and area under the receiver operating characteristic curve (AUROC) were used to evaluate the performance of the models. The optimal cutoff risk score threshold was identified at which both sensitivity and specificity were maximized (sensitivity + specificity). In addition, AUROC loss by stepwise selection was also performed to investigate important and determining factors of the model.

## 6. Results

### 6.1. Characteristics of subjects in the CMUH

No significant differences were observed between male and female groups with respect to heroin using and craving score, maximum dose, heroin use duration, education level and marital status. Significant differences were noted in the mean age, BMI and heroin onset age of the subjects (Table [1](#T1)).

A total of 40 SNPs in 10 genes were not associated with maximum dose of MMT (supplementary table 1, [http://links.lww.com/MD/E366](http://links.lww.com/MD/E366)). We noticed that 4 SNPs, rs806368 in *CNR1* gene, s1386493 in *TPH2* gene, s16974799 in the *CYP2B6* gene, and rs2229205 in the *OPRL1*, were significantly associated with maximum dose of MMT (*P* < .05; Table [2](#T2))

We constructed GRS by using 4 significant SNPs in the Table [3](#T3). The univariate associations of GRS and associated with maximum dose of MMT are shown in a dose-response way (Table [3](#T3)). The GRS was associated with maximum dose of MMT, after adjustment for age, and gender and BMI, the GRS remained independently associated with t maximum dose of MMT.

### 6.2. Selection of predictive variables for maximum dose of MMT.

Based on the results of Table [3](#T3), we conducted stepwise model analysis to select the significant factors for the best fit to prediction model of maximum dose of MMT. The results were shown in the Table [4](#T4). CNR1 TPH2, CYP2B6, OPRL1 and craving were included in the best fit model (*P*-value was set < .05).

The prediction accuracy of the maximum dose of MMT is presented in Table [5](#T5). The diagnostic ability for GRS and craving was measured as AUROC in the prediction models. The AUROC for GRS was 0.69(95% CI: 0.61–0.77; *P* < .0001). The AUROC for craving were 0.65(95% CI: 0.56–0.73; *P* = .0002). The AUROC of the combined genetic and craving score were significant predictors for maximum dose of MMT with 46% of sensitivity and 85% of specificity for maximum dose of MMT, respectively (Table [5](#T5); Fig. [1](#F1)).

## 7. Discussion

The major findings of our study are:

Furthermore, we showed risk prediction models to identify individuals with high risk of heroin addiction and effective doses of MMT. We found that 4 SNPs, rs806368 in *CNR1* gene, rs1386493 in *TPH2* gene, rs16974799 in the *CYP2B6* gene, and rs2229205 in the *OPRL1*, were significantly associated with maximum dose of MMT. Methadone has been studied most intensively in heroin addiction pharmacogenetics research.^[12]^ Several variants in *ABCB1 CYP2B6 OPRD*1 genes were indentified association with methadone plasma levels in the candidate gene study.^[12,13]^ In addition, 2 GWAS identified *CYP2B6* and *OPRM1* genes significant association with methadone dose.^[14,15]^ Because of chemical structure of methadone, the CYP2C19 isozyme metabolizes methadone R-enantiomer^[16]^ and CYP2B6 isozyme metabolizes methadone S-enantiomer^[17,18]^ in the liver. The plasma concentration of methadone can be quantifying methadone metabolism via genetic effects. These variants can serve as a surrogate marker for the treatment responses and efficacy of MMT.

To the best of our knowledge, the present study is the first study to report the genetic association between *CNR1* and *TPH2* and maximum dose of MMT. The cannabinoid receptor 1 gene (*CNR1*) on chromosome 6q14- 15 encoded CB1 protein that is likely to modulate the release of serotonin, norepinephrine and dopamine systems, disturbances of which are well documented in depression^[19]^and addictive disorders.^[20]^ Several studies reported that SNPs of *CNR1*were associated with nicotine,^[21]^ marijuana,^[22]^ alcohol,^[23]^ cocaine,^[24]^ and poly-substance use disorder.^[25]^ We demonstrate association between polymorphisms (rs806368) in the CNR1 gene and maximum dose of MMT and the risk of heroin addiction. This finding is consistent with previous studies of CNR1 and its influence on lifetime major depressive disorder and suicidal behavior in a population of opiate-dependent outpatients remitted under stable methadone treatment.^[26]^

We have found that those patients with specific genotypes of tryptophan hydroxylase 2 genes are associated with maximum stabilized methadone daily doses. TPH2 activation may be regarded as a new prospective for neuropsychiatric diseases related to brain 5-HT levels.^[27]^ The variants inTPH2 *gene* was related to the efficacy of disulfiram treatment for cocaine addiction.^[28]^ Since cocaine's mechanism of action is through blockade of dopamine, norepinephrine and serotonin transporters,^[29]^ we also evaluated the role of a functional variant in the TPH2 gene. We found that patients with CC genotype ofTPH2 responded to methadone better than those patients without this genotype. The variant in TPH2 also has been found to be associated with heroin addiction^[30]^ and quality of life in methadone maintenance therapy population^[31]^

We integrated genetic variation at known pharmacokinetic and pharmacodynamic genes, as well as additional genes and craving score to apply prediction for maximum dosage of MMT. The genetic risk calculated by the individual common genetic variants was modest; their combination resulted in a large effect on maximum dosage of MMT. However, the frequencies of higher GRS were rare, suggesting that variants with large effect sizes will impact a small proportion of the population. Our genetic model was associated with higher methadone dose, suggesting that combine GRS and craving score may help identify an adequate dosage of MMT in the setting of addiction treatment for individual. The efficacy of prediction model may be useful in individual MMT dose requirements to suggest initiation of methadone treatment and at the start of treatment to predict the optimal dose for a particular patient. The strengths of our study include comprehensive addiction scale information and utilized multivariate logistic regression analysis to assess the combinatory effect of genetic polymorphisms on the maximum dosage of MMT.

Several limitations must also be acknowledged. First, the sample size is small compared with case/control cross many different studies. We performed a QTL association to identify genetic variants for MMT dosage. Without normal control in this study, we have suggested that larger-scale studies are warranted to further confirm this prediction model used practically without genetic risk population.

The AUROCC for GRS and craving score were 0.69 and 0.65 respectively. The sensitivity and specificity for GRS and craving were relatively lower. Our study had only limited phenotypic and genotypic information. The candidate gene approach allows a scan of a limited number of SNPs. We needed to identify more genes associated with dosage of MMT, include candidate genes identified by GWAS^[14,15,32,33]^ and variants underlying heroin addiction to increase accuracy in the future study.

Although the variants identified in this study are suggested to produce relative risk for the response to dosage of MMT, they may uncover potential mechanisms of addictive behavior. In particular, future recruitment of additional methadone- treated patients will be needed in order to study variability in treatment response under the influence of environmental factors (ie, other pharmacological treatments, toxic exposure, harmful or safe physical and psychological environment, lifestyle and so on) and acting through epigenetic modifications and gene regulation.

## 8. Conclusion

In summary, our study provided evidence that 4 susceptibility SNPs was associated with maximum dosage of MMT. For the risk prediction models, we combined GRS and craving score may be useful in evaluation of individual MMT dose requirements at the start of treatment. Predicting optimal dose for each individual can allow the clinician to provide carefully the dose to each individual patient's needs.

## Acknowledgments

We thank the National Center for Genome Medicine for the technical support.

## Author contributions

CLH and RYW were responsible for the study concept and design. HWC and WCH conducted data analysis and interpretation of findings. CLH and RYW interpreted results of genotyping and clinical data. HWC, WCH and RYW drafted the manuscript. All authors critically reviewed content and approved the final version for publication.

## Supplementary Material

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Barbotte E, Guillemin F, Chau N, et al. Prevalence of impairments, disabilities, handicaps and quality of life in the general population: a review of recent literature. Bull World Health Organ 2001;79:1047–55.  [PMC free article](/articles/PMC2566690/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11731812/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bull%20World%20Health%20Organ&title=Prevalence%20of%20impairments,%20disabilities,%20handicaps%20and%20quality%20of%20life%20in%20the%20general%20population:%20a%20review%20of%20recent%20literature&author=E%20Barbotte&author=F%20Guillemin&author=N%20Chau&volume=79&publication_year=2001&pages=1047-55&pmid=11731812&)

2. Oueslati B, Moula O, Ghachem R. The impact of OPRM1's genetic polymorphisms on methadone maintenance treatment in opioid addicts: a systematic review. Pharmacogenomics 2018;19:741–7.  [DOI](https://doi.org/10.2217/pgs-2018-0017) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29785888/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=The%20impact%20of%20OPRM1's%20genetic%20polymorphisms%20on%20methadone%20maintenance%20treatment%20in%20opioid%20addicts:%20a%20systematic%20review&author=B%20Oueslati&author=O%20Moula&author=R%20Ghachem&volume=19&publication_year=2018&pages=741-7&pmid=29785888&doi=10.2217/pgs-2018-0017&)

3. Yen YF, Yen MY, Su LW, et al. Prevalences and associated risk factors of HCV/HIV co-infection and HCV mono-infection among injecting drug users in a methadone maintenance treatment program in Taipei, Taiwan. BMC Public Health 2012;12:1066.  [DOI](https://doi.org/10.1186/1471-2458-12-1066) | [PMC free article](/articles/PMC3534443/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23227904/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Public%20Health&title=Prevalences%20and%20associated%20risk%20factors%20of%20HCV/HIV%20co-infection%20and%20HCV%20mono-infection%20among%20injecting%20drug%20users%20in%20a%20methadone%20maintenance%20treatment%20program%20in%20Taipei,%20Taiwan&author=YF%20Yen&author=MY%20Yen&author=LW%20Su&volume=12&publication_year=2012&pages=1066&pmid=23227904&doi=10.1186/1471-2458-12-1066&)

4. Maxwell S, Shinderman MS. Optimizing long-term response to methadone maintenance treatment: a 152-week follow-up using higher-dose methadone. J Addict Dis 2002;21:1–2.  [DOI](https://doi.org/10.1300/J069v21n03_01) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12094996/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Addict%20Dis&title=Optimizing%20long-term%20response%20to%20methadone%20maintenance%20treatment:%20a%20152-week%20follow-up%20using%20higher-dose%20methadone&author=S%20Maxwell&author=MS%20Shinderman&volume=21&publication_year=2002&pages=1-2&pmid=12094996&doi=10.1300/J069v21n03_01&)

5. Kreek MJ, Bart G, Lilly C, et al. Pharmacogenetics and human molecular genetics of opiate and cocaine addictions and their treatments. Pharmacol Rev 2005;57:1–26.  [DOI](https://doi.org/10.1124/pr.57.1.1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15734726/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacol%20Rev&title=Pharmacogenetics%20and%20human%20molecular%20genetics%20of%20opiate%20and%20cocaine%20addictions%20and%20their%20treatments&author=MJ%20Kreek&author=G%20Bart&author=C%20Lilly&volume=57&publication_year=2005&pages=1-26&pmid=15734726&doi=10.1124/pr.57.1.1&)

6. Kapur BM, Hutson JR, Chibber T, et al. Methadone: a review of drug-drug and pathophysiological interactions. Crit Rev Clin Lab Sci 2011;48:171–95.  [DOI](https://doi.org/10.3109/10408363.2011.620601) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22035341/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Crit%20Rev%20Clin%20Lab%20Sci&title=Methadone:%20a%20review%20of%20drug-drug%20and%20pathophysiological%20interactions&author=BM%20Kapur&author=JR%20Hutson&author=T%20Chibber&volume=48&publication_year=2011&pages=171-95&pmid=22035341&doi=10.3109/10408363.2011.620601&)

7. Tsai HJ, Wang SC, Liu SW, et al. Assessment of CYP450 genetic variability effect on methadone dose and tolerance. Pharmacogenomics 2014;15:977–86.  [DOI](https://doi.org/10.2217/pgs.14.19) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24956251/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Assessment%20of%20CYP450%20genetic%20variability%20effect%20on%20methadone%20dose%20and%20tolerance&author=HJ%20Tsai&author=SC%20Wang&author=SW%20Liu&volume=15&publication_year=2014&pages=977-86&pmid=24956251&doi=10.2217/pgs.14.19&)

8. Fonseca F, de la Torre R, Diaz L, et al. Contribution of cytochrome P450 and ABCB1 genetic variability on methadone pharmacokinetics, dose requirements, and response. PLoS One 2011;6:e19527.  [DOI](https://doi.org/10.1371/journal.pone.0019527) | [PMC free article](/articles/PMC3093392/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21589866/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Contribution%20of%20cytochrome%20P450%20and%20ABCB1%20genetic%20variability%20on%20methadone%20pharmacokinetics,%20dose%20requirements,%20and%20response&author=F%20Fonseca&author=R%20de%20la%20Torre&author=L%20Diaz&volume=6&publication_year=2011&pages=e19527&pmid=21589866&doi=10.1371/journal.pone.0019527&)

9. Beer B, Erb R, Pavlic M, et al. Association of polymorphisms in pharmacogenetic candidate genes (OPRD1, GAL, ABCB1, OPRM1) with opioid dependence in European population: a case-control study. PLoS One 2013;8:e75359.  [DOI](https://doi.org/10.1371/journal.pone.0075359) | [PMC free article](/articles/PMC3783401/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24086514/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Association%20of%20polymorphisms%20in%20pharmacogenetic%20candidate%20genes%20(OPRD1,%20GAL,%20ABCB1,%20OPRM1)%20with%20opioid%20dependence%20in%20European%20population:%20a%20case-control%20study&author=B%20Beer&author=R%20Erb&author=M%20Pavlic&volume=8&publication_year=2013&pages=e75359&pmid=24086514&doi=10.1371/journal.pone.0075359&)

10. Chiang YC, Wang RY, Huang CL, et al. Reduced dosing and liability in methadone maintenance treatment by targeting oestrogen signal for morphine addiction. J Cell Mol Med 2017;21:3552–64.  [DOI](https://doi.org/10.1111/jcmm.13266) | [PMC free article](/articles/PMC5706516/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28699698/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Cell%20Mol%20Med&title=Reduced%20dosing%20and%20liability%20in%20methadone%20maintenance%20treatment%20by%20targeting%20oestrogen%20signal%20for%20morphine%20addiction&author=YC%20Chiang&author=RY%20Wang&author=CL%20Huang&volume=21&publication_year=2017&pages=3552-64&pmid=28699698&doi=10.1111/jcmm.13266&)

11. Marie-Claire C, Jourdaine C, Lepine JP, et al. Pharmacoepigenomics of opiates and methadone maintenance treatment: current data and perspectives. Pharmacogenomics 2017;18:1359–72.  [DOI](https://doi.org/10.2217/pgs-2017-0040) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28841113/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Pharmacoepigenomics%20of%20opiates%20and%20methadone%20maintenance%20treatment:%20current%20data%20and%20perspectives&author=C%20Marie-Claire&author=C%20Jourdaine&author=JP%20Lepine&volume=18&publication_year=2017&pages=1359-72&pmid=28841113&doi=10.2217/pgs-2017-0040&)

12. Dennis BB, Bawor M, Thabane L, et al. Impact of ABCB1 and CYP2B6 genetic polymorphisms on methadone metabolism, dose and treatment response in patients with opioid addiction: a systematic review and meta-analysis. PLoS One 2014;9:e86114.  [DOI](https://doi.org/10.1371/journal.pone.0086114) | [PMC free article](/articles/PMC3906028/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24489693/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Impact%20of%20ABCB1%20and%20CYP2B6%20genetic%20polymorphisms%20on%20methadone%20metabolism,%20dose%20and%20treatment%20response%20in%20patients%20with%20opioid%20addiction:%20a%20systematic%20review%20and%20meta-analysis&author=BB%20Dennis&author=M%20Bawor&author=L%20Thabane&volume=9&publication_year=2014&pages=e86114&pmid=24489693&doi=10.1371/journal.pone.0086114&)

13. Berrettini W. A brief review of the genetics and pharmacogenetics of opioid use disorders. Dialogues Clin Neurosci 2017;19:229–36.  [DOI](https://doi.org/10.31887/DCNS.2017.19.3/wberrettini) | [PMC free article](/articles/PMC5741106/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29302220/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Dialogues%20Clin%20Neurosci&title=A%20brief%20review%20of%20the%20genetics%20and%20pharmacogenetics%20of%20opioid%20use%20disorders&author=W%20Berrettini&volume=19&publication_year=2017&pages=229-36&pmid=29302220&doi=10.31887/DCNS.2017.19.3/wberrettini&)

14. Yang HC, Chu SK, Huang CL, et al. Genome-wide pharmacogenomic study on methadone maintenance treatment identifies SNP rs17180299 and multiple haplotypes on CYP2B6, SPON1, and GSG1L associated with plasma concentrations of methadone R- and S-enantiomers in heroin-dependent patients. PLoS Genet 2016;12:e1005910.  [DOI](https://doi.org/10.1371/journal.pgen.1005910) | [PMC free article](/articles/PMC4806848/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27010727/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet&title=Genome-wide%20pharmacogenomic%20study%20on%20methadone%20maintenance%20treatment%20identifies%20SNP%20rs17180299%20and%20multiple%20haplotypes%20on%20CYP2B6,%20SPON1,%20and%20GSG1L%20associated%20with%20plasma%20concentrations%20of%20methadone%20R-%20and%20S-enantiomers%20in%20heroin-dependent%20patients&author=HC%20Yang&author=SK%20Chu&author=CL%20Huang&volume=12&publication_year=2016&pages=e1005910&pmid=27010727&doi=10.1371/journal.pgen.1005910&)

15. Smith AH, Jensen KP, Li J, et al. Genome-wide association study of therapeutic opioid dosing identifies a novel locus upstream of OPRM1. Mol Psychiatry 2017;22:346–52.  [DOI](https://doi.org/10.1038/mp.2016.257) | [PMC free article](/articles/PMC5407902/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28115739/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Psychiatry&title=Genome-wide%20association%20study%20of%20therapeutic%20opioid%20dosing%20identifies%20a%20novel%20locus%20upstream%20of%20OPRM1&author=AH%20Smith&author=KP%20Jensen&author=J%20Li&volume=22&publication_year=2017&pages=346-52&pmid=28115739&doi=10.1038/mp.2016.257&)

16. Chang Y, Fang WB, Lin SN, et al. Stereo-selective metabolism of methadone by human liver microsomes and cDNA-expressed cytochrome P450s: a reconciliation. Basic Clin Pharmacol Toxicol 2011;108:55–62.  [DOI](https://doi.org/10.1111/j.1742-7843.2010.00628.x) | [PMC free article](/articles/PMC3005981/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20825389/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Basic%20Clin%20Pharmacol%20Toxicol&title=Stereo-selective%20metabolism%20of%20methadone%20by%20human%20liver%20microsomes%20and%20cDNA-expressed%20cytochrome%20P450s:%20a%20reconciliation&author=Y%20Chang&author=WB%20Fang&author=SN%20Lin&volume=108&publication_year=2011&pages=55-62&pmid=20825389&doi=10.1111/j.1742-7843.2010.00628.x&)

17. Totah RA, Allen KE, Sheffels P, et al. Enantiomeric metabolic interactions and stereoselective human methadone metabolism. J Pharmacol Exp Ther 2007;321:389–99.  [DOI](https://doi.org/10.1124/jpet.106.117580) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17259447/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacol%20Exp%20Ther&title=Enantiomeric%20metabolic%20interactions%20and%20stereoselective%20human%20methadone%20metabolism&author=RA%20Totah&author=KE%20Allen&author=P%20Sheffels&volume=321&publication_year=2007&pages=389-99&pmid=17259447&doi=10.1124/jpet.106.117580&)

18. Totah RA, Sheffels P, Roberts T, et al. Role of CYP2B6 in stereoselective human methadone metabolism. Anesthesiology 2008;108:363–74.  [DOI](https://doi.org/10.1097/ALN.0b013e3181642938) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18292673/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anesthesiology&title=Role%20of%20CYP2B6%20in%20stereoselective%20human%20methadone%20metabolism&author=RA%20Totah&author=P%20Sheffels&author=T%20Roberts&volume=108&publication_year=2008&pages=363-74&pmid=18292673&doi=10.1097/ALN.0b013e3181642938&)

19. Hamon M, Blier P. Monoamine neurocircuitry in depression and strategies for new treatments. Prog Neuropsychopharmacol Biol Psychiatry 2013;45:54–63.  [DOI](https://doi.org/10.1016/j.pnpbp.2013.04.009) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23602950/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Prog%20Neuropsychopharmacol%20Biol%20Psychiatry&title=Monoamine%20neurocircuitry%20in%20depression%20and%20strategies%20for%20new%20treatments&author=M%20Hamon&author=P%20Blier&volume=45&publication_year=2013&pages=54-63&pmid=23602950&doi=10.1016/j.pnpbp.2013.04.009&)

20. Volkow ND, Wang GJ, Fowler JS, et al. Addiction circuitry in the human brain. Annu Rev Pharmacol Toxicol 2012;52:321–36.  [DOI](https://doi.org/10.1146/annurev-pharmtox-010611-134625) | [PMC free article](/articles/PMC3477468/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21961707/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annu%20Rev%20Pharmacol%20Toxicol&title=Addiction%20circuitry%20in%20the%20human%20brain&author=ND%20Volkow&author=GJ%20Wang&author=JS%20Fowler&volume=52&publication_year=2012&pages=321-36&pmid=21961707&doi=10.1146/annurev-pharmtox-010611-134625&)

21. Chen X, Williamson VS, An SS, et al. Cannabinoid receptor 1 gene association with nicotine dependence. Arch Gen Psychiatry 2008;65:816–24.  [DOI](https://doi.org/10.1001/archpsyc.65.7.816) | [PMC free article](/articles/PMC2733353/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18606954/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arch%20Gen%20Psychiatry&title=Cannabinoid%20receptor%201%20gene%20association%20with%20nicotine%20dependence&author=X%20Chen&author=VS%20Williamson&author=SS%20An&volume=65&publication_year=2008&pages=816-24&pmid=18606954&doi=10.1001/archpsyc.65.7.816&)

22. Agrawal A, Wetherill L, Dick DM, et al. Evidence for association between polymorphisms in the cannabinoid receptor 1 (CNR1) gene and cannabis dependence. Am J Med Genet B Neuropsychiatr Genet 2009;150B:736–40.  [DOI](https://doi.org/10.1002/ajmg.b.30881) | [PMC free article](/articles/PMC2703788/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19016476/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Med%20Genet%20B%20Neuropsychiatr%20Genet&title=Evidence%20for%20association%20between%20polymorphisms%20in%20the%20cannabinoid%20receptor%201%20(CNR1)%20gene%20and%20cannabis%20dependence&author=A%20Agrawal&author=L%20Wetherill&author=DM%20Dick&volume=150B&publication_year=2009&pages=736-40&pmid=19016476&doi=10.1002/ajmg.b.30881&)

23. Marcos M, Pastor I, de la Calle C, et al. Cannabinoid receptor 1 gene is associated with alcohol dependence. Alcohol Clin Exp Res 2012;36:267–71.  [DOI](https://doi.org/10.1111/j.1530-0277.2011.01623.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22085192/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Alcohol%20Clin%20Exp%20Res&title=Cannabinoid%20receptor%201%20gene%20is%20associated%20with%20alcohol%20dependence&author=M%20Marcos&author=I%20Pastor&author=C%20de%20la%20Calle&volume=36&publication_year=2012&pages=267-71&pmid=22085192&doi=10.1111/j.1530-0277.2011.01623.x&)

24. Clarke TK, Bloch PJ, Ambrose-Lanci LM, et al. Further evidence for association of polymorphisms in the CNR1 gene with cocaine addiction: confirmation in an independent sample and meta-analysis. Addict Biol 2013;18:702–8.  [DOI](https://doi.org/10.1111/j.1369-1600.2011.00346.x) | [PMC free article](/articles/PMC3223560/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21790903/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Addict%20Biol&title=Further%20evidence%20for%20association%20of%20polymorphisms%20in%20the%20CNR1%20gene%20with%20cocaine%20addiction:%20confirmation%20in%20an%20independent%20sample%20and%20meta-analysis&author=TK%20Clarke&author=PJ%20Bloch&author=LM%20Ambrose-Lanci&volume=18&publication_year=2013&pages=702-8&pmid=21790903&doi=10.1111/j.1369-1600.2011.00346.x&)

25. Zhang PW, Ishiguro H, Ohtsuki T, et al. Human cannabinoid receptor 1: 5’ exons, candidate regulatory regions, polymorphisms, haplotypes and association with polysubstance abuse. Mol Psychiatry 2004;9:916–31.  [DOI](https://doi.org/10.1038/sj.mp.4001560) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15289816/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Psychiatry&title=Human%20cannabinoid%20receptor%201:%205%E2%80%99%20exons,%20candidate%20regulatory%20regions,%20polymorphisms,%20haplotypes%20and%20association%20with%20polysubstance%20abuse&author=PW%20Zhang&author=H%20Ishiguro&author=T%20Ohtsuki&volume=9&publication_year=2004&pages=916-31&pmid=15289816&doi=10.1038/sj.mp.4001560&)

26. Icick R, Peoc’h K, Karsinti E, et al. A cannabinoid receptor 1 polymorphism is protective against major depressive disorder in methadone-maintained outpatients. Am J Addict 2015;24:613–20.  [DOI](https://doi.org/10.1111/ajad.12273) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26331953/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Addict&title=A%20cannabinoid%20receptor%201%20polymorphism%20is%20protective%20against%20major%20depressive%20disorder%20in%20methadone-maintained%20outpatients&author=R%20Icick&author=K%20Peoc%E2%80%99h&author=E%20Karsinti&volume=24&publication_year=2015&pages=613-20&pmid=26331953&doi=10.1111/ajad.12273&)

27. Matthes S, Mosienko V, Bashammakh S, et al. Tryptophan hydroxylase as novel target for the treatment of depressive disorders. Pharmacology 2010;85:95–109.  [DOI](https://doi.org/10.1159/000279322) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20130443/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacology&title=Tryptophan%20hydroxylase%20as%20novel%20target%20for%20the%20treatment%20of%20depressive%20disorders&author=S%20Matthes&author=V%20Mosienko&author=S%20Bashammakh&volume=85&publication_year=2010&pages=95-109&pmid=20130443&doi=10.1159/000279322&)

28. Nielsen DA, Harding MJ, Hamon SC, et al. Modifying the role of serotonergic 5-HTTLPR and TPH2 variants on disulfiram treatment of cocaine addiction: a preliminary study. Genes Brain Behav 2012;11:1001–8.  [DOI](https://doi.org/10.1111/j.1601-183X.2012.00839.x) | [PMC free article](/articles/PMC3521860/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22925276/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genes%20Brain%20Behav&title=Modifying%20the%20role%20of%20serotonergic%205-HTTLPR%20and%20TPH2%20variants%20on%20disulfiram%20treatment%20of%20cocaine%20addiction:%20a%20preliminary%20study&author=DA%20Nielsen&author=MJ%20Harding&author=SC%20Hamon&volume=11&publication_year=2012&pages=1001-8&pmid=22925276&doi=10.1111/j.1601-183X.2012.00839.x&)

29. Han DD, Gu HH. Comparison of the monoamine transporters from human and mouse in their sensitivities to psychostimulant drugs. BMC Pharmacol 2006;6:6.  [DOI](https://doi.org/10.1186/1471-2210-6-6) | [PMC free article](/articles/PMC1448202/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16515684/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Pharmacol&title=Comparison%20of%20the%20monoamine%20transporters%20from%20human%20and%20mouse%20in%20their%20sensitivities%20to%20psychostimulant%20drugs&author=DD%20Han&author=HH%20Gu&volume=6&publication_year=2006&pages=6&pmid=16515684&doi=10.1186/1471-2210-6-6&)

30. Nielsen DA, Barral S, Proudnikov D, et al. TPH2 and TPH1: association of variants and interactions with heroin addiction. Behav Genet 2008;38:133–50.  [DOI](https://doi.org/10.1007/s10519-007-9187-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18181017/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Behav%20Genet&title=TPH2%20and%20TPH1:%20association%20of%20variants%20and%20interactions%20with%20heroin%20addiction&author=DA%20Nielsen&author=S%20Barral&author=D%20Proudnikov&volume=38&publication_year=2008&pages=133-50&pmid=18181017&doi=10.1007/s10519-007-9187-7&)

31. Wang RY, Chen HJ, Huang CL, et al. Impacts of GRIN3A, GRM6 and TPH2 genetic polymorphisms on quality of life in methadone maintenance therapy population. PLoS One 2018;13:e0201408.  [DOI](https://doi.org/10.1371/journal.pone.0201408) | [PMC free article](/articles/PMC6066242/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30059533/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Impacts%20of%20GRIN3A,%20GRM6%20and%20TPH2%20genetic%20polymorphisms%20on%20quality%20of%20life%20in%20methadone%20maintenance%20therapy%20population&author=RY%20Wang&author=HJ%20Chen&author=CL%20Huang&volume=13&publication_year=2018&pages=e0201408&pmid=30059533&doi=10.1371/journal.pone.0201408&)

32. Nishizawa D, Fukuda K, Kasai S, et al. Genome-wide association study identifies a potent locus associated with human opioid sensitivity. Mol Psychiatry 2014;19:55–62.  [DOI](https://doi.org/10.1038/mp.2012.164) | [PMC free article](/articles/PMC3873034/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23183491/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Psychiatry&title=Genome-wide%20association%20study%20identifies%20a%20potent%20locus%20associated%20with%20human%20opioid%20sensitivity&author=D%20Nishizawa&author=K%20Fukuda&author=S%20Kasai&volume=19&publication_year=2014&pages=55-62&pmid=23183491&doi=10.1038/mp.2012.164&)

33. Mieda T, Nishizawa D, Nakagawa H, et al. Genome-wide association study identifies candidate loci associated with postoperative fentanyl requirements after laparoscopic-assisted colectomy. Pharmacogenomics 2016;17:133–45.  [DOI](https://doi.org/10.2217/pgs.15.151) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26566055/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Genome-wide%20association%20study%20identifies%20candidate%20loci%20associated%20with%20postoperative%20fentanyl%20requirements%20after%20laparoscopic-assisted%20colectomy&author=T%20Mieda&author=D%20Nishizawa&author=H%20Nakagawa&volume=17&publication_year=2016&pages=133-45&pmid=26566055&doi=10.2217/pgs.15.151&)
