# Effect of ARG1 Gene (rs2781666) Polymorphism on Plasma Arginase Activity and Bronchial Asthma Prevalence and Severity

## Metadata
**Authors:** Eman M Al-Ghunmieen, Alia Shatanawi, Fatima Mohammad Al-Akhras, Khaled Al Oweidat, Ebaa M Alzayadneh
**Journal:** Journal of Asthma and Allergy
**Date:** 2025 Jun 17
**DOI:** [10.2147/JAA.S505962](https://doi.org/10.2147/JAA.S505962)
**PMID:** 40547816
**PMCID:** PMC12182068
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC12182068/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC12182068/pdf/jaa-18-1023.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC12182068/pdf/jaa-18-1023.pdf)

## Abstract

**Background:** 
Bronchial asthma is a chronic inflammatory condition that affects the lungs and causes airway narrowing. Multiple genetic and environmental factors contribute to bronchial asthma. The complex interactions between these factors play a vital role in the susceptibility to and severity of asthma. Several genetic factors are associated with the prevalence and severity of bronchial asthma. Among these genes, ARG1 rs2781666 polymorphism has been found to be associated with bronchial asthma prevalence worldwide. Furthermore, increased serum arginase activity has been reported in patients with asthma, suggesting an association with bronchial asthma phenotypes.

**Purpose:** 
To determine the frequency of ARG1 rs2781666 polymorphism among Jordanians, we compared plasma arginase activity and ARG1 rs2781666 polymorphism in asthmatic patients and non-asthmatic volunteers, and analyzed the distribution of rs2781666 genotypes among asthma severity groups.

**Patients and Methods:** 
Four hundred and twenty-four asthmatic and non-asthmatic Jordanian subjects visiting the Jordan University Hospital were genotyped for ARG1 rs2781666 polymorphism using the polymerase chain reaction-restriction fragment length polymorphism method and were examined for serum arginase activity using an arginase activity assay.

**Results and conclusions:** 
There was a significant association between ARG1 rs2781666 G/T polymorphism and bronchial asthma frequency and severity. The GT genotype and T allele frequencies were significantly higher in asthmatic patients than in non-asthmatic patients. In addition, comparison between the ARG1 rs2781666 genotype distribution and asthma severity revealed that the TT genotype was more frequent in the severe asthma group. Furthermore, a comparison of plasma arginase activity between the asthmatic and non-asthmatic groups revealed that arginase activity was significantly higher in asthmatic patients than in non-asthmatic patients. In addition, the results showed a positive association between elevated plasma arginase activity and rs2781666 G/T in asthmatic patients compared to that in non-asthmatic subjects.

Keywords: arginase, SNP, inflammation, asthmatics

### Background

Bronchial asthma is a chronic inflammatory condition that affects the lungs and causes airway narrowing. Multiple genetic and environmental factors contribute to bronchial asthma. The complex interactions between these factors play a vital role in the susceptibility to and severity of asthma. Several genetic factors are associated with the prevalence and severity of bronchial asthma. Among these genes, *ARG1* rs2781666 polymorphism has been found to be associated with bronchial asthma prevalence worldwide. Furthermore, increased serum arginase activity has been reported in patients with asthma, suggesting an association with bronchial asthma phenotypes.

### Purpose

To determine the frequency of *ARG1* rs2781666 polymorphism among Jordanians, we compared plasma arginase activity and *ARG1* rs2781666 polymorphism in asthmatic patients and non-asthmatic volunteers, and analyzed the distribution of rs2781666 genotypes among asthma severity groups.

### Patients and Methods

Four hundred and twenty-four asthmatic and non-asthmatic Jordanian subjects visiting the Jordan University Hospital were genotyped for *ARG1* rs2781666 polymorphism using the polymerase chain reaction-restriction fragment length polymorphism method and were examined for serum arginase activity using an arginase activity assay.

### Results and conclusions

There was a significant association between *ARG1* rs2781666 G/T polymorphism and bronchial asthma frequency and severity. The GT genotype and T allele frequencies were significantly higher in asthmatic patients than in non-asthmatic patients. In addition, comparison between the *ARG1* rs2781666 genotype distribution and asthma severity revealed that the TT genotype was more frequent in the severe asthma group. Furthermore, a comparison of plasma arginase activity between the asthmatic and non-asthmatic groups revealed that arginase activity was significantly higher in asthmatic patients than in non-asthmatic patients. In addition, the results showed a positive association between elevated plasma arginase activity and rs2781666 G/T in asthmatic patients compared to that in non-asthmatic subjects.

## Introduction

Asthma is a major global health concern worldwide. This disease affected nearly 262 million people in 2019 and caused 455000 deaths.[1](#cit0001) It has been shown to be affecting 1–18% of the population in different countries.[2](#cit0002)^,^[3](#cit0003) In Jordan, the prevalence of asthma is approximately 10%.[4](#cit0004)^,^[5](#cit0005)

Asthma is a common chronic respiratory disease characterized by recurrent airway obstruction, airway hyper-responsiveness, and airway inflammation, and is associated with allergy and allergic rhinitis. After chronic inflammation of the airways, structural changes in the airway wall, including fibrosis and airway wall thickening, may lead to airway remodeling, which contributes to a progressive decline in lung function over time as well as the development of persistent airway hyper-responsiveness.[6](#cit0006)^,^[7](#cit0007)

Asthma symptoms included repeated episodes of coughing, wheezing, shortness of breath, and chest tightness. Asthma is a serious pulmonary condition that cannot be cured but can be managed. Asthma affects patients of all ages and often starts in early childhood.[8](#cit0008) Asthma imposes a burden on society and healthcare systems, and affects workplace productivity because of sleep disturbances.[3](#cit0003) Asthma can be medically managed by avoiding triggers that induce symptoms.[9](#cit0009) These episodic symptoms usually cause reversible airway obstruction that can be reversed spontaneously or with medication.[8](#cit0008)

Arginase (ARG) is an essential enzyme in the hepatic urea cycle that converts L-arginine to L-ornithine and urea. Additionally, arginase is constitutively expressed in airways. Arginase has two isoforms, arginase 1 (ARG1) and arginase 2 (ARG2), which are encoded and differentially expressed by various genes.[10](#cit0010) *ARG1* and *ARG2* genes were mapped to chromosome 6q23.2 and chromosome 14q24, respectively. *ARG1* encodes a 322 amino acid protein, whereas *ARG2* genes encodes 354 a acid protein.[11](#cit0011) Nitric oxide (NO) plays an important role as a signaling molecule involved in the regulation of physiological and pathophysiological processes in the lungs.[12](#cit0012) NO also plays a protective role in the airways by acting as an endogenous bronchodilator, having anti-inflammatory properties, and by inhibiting mediator release from mast cells. NO is synthesized from L-arginine by a family of NO synthase (NOS) isoenzymes.[6](#cit0006)^,^[12](#cit0012)

L-arginine is produced de novo via the citrulline-arginine pathway via the intestinal-renal axis by glutamine, which serves as a precursor. Because L-arginine is considered a substrate for both NOS and arginase, both enzymes regulate the production of endogenous NO.[13](#cit0013) By competing for L-arginine, arginase isoenzymes reduce L-arginine bioavailability for NOS, thus limiting NO production, which can contribute to airway remodeling and limit NO-induced bronchodilation and may underlie AHR in asthma.[12](#cit0012)^,^[14](#cit0014)^,^[15](#cit0015) Increased utilization of L-arginine by the ARG enzyme elevates the levels of its metabolites, including polyamines, which are involved in collagen synthesis and cell growth and proliferation, thus playing a significant role in airway remodeling (Donthi et al, 2019). Collectively, these effects of arginase imply its involvement in the pathogenesis of asthma through modulation of the L-arginine-nitric oxide pathway.[16](#cit0016) Genetic variants in *ARG1* gene have been implicated in asthma development, severity, and response to therapy.[17](#cit0017)

The ARG1 rs2781666 polymorphism has been associated with altered arginase-1 expression and activity, potentially influencing NO bioavailability and vascular function. This single nucleotide polymorphism (SNP), located in the promoter region of the ARG1 gene, may affect transcriptional regulation, leading to increased arginase expression. Several studies have linked this variant to increased cardiovascular risk and inflammation, underscoring its functional relevance in disease pathogenesis.[18](#cit0018)

The *ARG1* rs2781666 G/T genotype and elevated serum ARG 1 have also been reported to increase the risk of asthma in susceptible populations.[19](#cit0019)

Our study assessed and determined the allele frequencies of *the ARG1* gene in Jordan, measured serum arginase activity, and correlated the results with asthma in Jordan.

## Materials and Methods

### Study Design

This cross-sectional study compared patients who had been diagnosed with bronchial asthma while visiting the Pulmonary Disease Clinic at Jordan University Hospital with non-asthmatic volunteers. A total of 424 samples were collected from participants attending the Jordan University Hospital, Amman, Jordan. Of the 424 participants, 251 were asthmatic and 173 were non-asthmatic. The study protocol was approved by the Institutional Review Board (IRB) of Jordan University Hospital (reference number 2020/259). After explaining and discussing the project, written and signed consent forms were obtained from each participant before blood sampling. Demographic and clinical data were recorded using a previously prepared datasheet ([Appendix I](https://www.dovepress.com/article/supplementary_file/505962/Appendix%201_1.docx)). This study was conducted in accordance with the principles of the Declaration of Helsinki.

### Data Collection and Blood Sampling

Informed consent and patient data including age, weight, height, sex, smoking status, other diseases, and medications used were obtained.

Clinical data, FEV1, FVC, FEV1/FVC ratio, and other parameters were collected from the records of the Jordan University Hospital system.

Venous blood (3–5 mL) was collected from each subject in ethylenediaminetetraacetic acid (EDTA)-containing tubes and stored at 4° C for DNA extraction.

### Genomic DNA Extraction and Quantification

The DNA absorbance was measured at 260 nm (A260) using a UV Spectrophotometer (Thermo Scientific, USA). The concentration of the extracted genomic DNA in each sample and the ratio of the absorbance at A260/A280 nm (which evaluates the purity of the extracted DNA from protein, which should be 1.6–1.8) were obtained).[20](#cit0020) For each DNA sample, 4µL was diluted in 96 µL distilled water to a total volume of 100 µL (dilution factor = 25). Next, 100 µL of each sample was transferred into a spectrophotometer cuvette. DNA concentration was calculated using the following equation: DNA concentration (ng) = dilution factor × absorbance at 260 nm × DNA Factor (50ng/ µL).

### Genotyping

To identify the different genotypes of the patients, a specific sequence containing the SNP of interest (rs2781666) of the gene was amplified exponentially to several millionfold by Polymerase Chain Reaction (PCR) using a PCR Thermal Cycler (Bio-Rad, USA). The SNP was detected by the restriction fragment length polymorphism (RFLP) technique using a specific restriction enzyme (*Tai1*) to digest variant SNP at a specific site for the *ARG1*.

### Designation and Preparation of the Primers

The primers were chosen as in previous studies that used the same SNP and techniques.[15](#cit0015) The two primers (forward and reverse) were tested (primer-BLAST) using the NCBI website ([https://www.ncbi.nlm.nih.gov](https://www.ncbi.nlm.nih.gov)) from which we obtained all the properties and information needed for the primers and the virtually produced; to ensure primer efficiency and specificity. The forward primer sequence was 5′-CGGAAGGATCTTTAAGGTGCC-3′ and the reverse primer was 5′-CCATGTGTCCGATGCAGTTCTG-3′. PCR product size was 294 bp.

Each primer was obtained as a lyophilized powder, stored at room temperature, and spun down using centrifuged (Sigma^®^ 1–14) for one minute before opening the tube. Working solutions were prepared from the forward and reverse primers as follows: According to the supplier, both the primer vials contained 30 nmol. To prepare a 100 µM master stock solution, 300 µL of nuclease-free water was added to tubes containing the forward and reverse primers. The vials were vortexed to obtain a homogeneous solution and allowed to sit at room temperature for ten minutes before use for working stock dilutions. Twenty microliters of each master stock solution were added to 80 µL of nuclease-free water to prepare a 10 µM working solution of the forward and reverse primers. All vials were vortexed and stored at −20 °C.

### PCR Preparation of the Samples

The PCR mixture was prepared in a total volume of 50 µL for each sample, 25 µL of 2x GoTaq^®^ Green Master Mix with a final concentration of 1x was mixed with 1 µL forward and 1 µL reverse primers (10 µM), 2 µL of Genomic DNA Template (<250 ng/µL), and 21 µL NFW.

All reagents and DNA samples were prepared under sterile conditions and were placed on ice throughout the experiment. All reagent vials were spun at 2000 RPM for 20s before use. PCR was performed using a PCR Thermal Cycler (Bio-Rad, USA) according to the following protocol. Initial denaturation at 95 °C for 10 minutes, denaturation at 95 °C for 30 seconds, annealing at 57 °C for 40 seconds, extension at 72 °C for 40 seconds, and the steps 2, 3, and 4 were repeated for 35 cycles. PCR products were stored at 4 °C until further analysis.

The PCR conditions were optimized by conducting the reaction at annealing temperatures ranging from 55 to 65 °C. The most specific and intense band is observed at 57 °C. A negative control sample was used under the same conditions to ensure that the samples were not contaminated. The PCR products were confirmed by 2% agarose gel electrophoresis.

### Agarose Gel Electrophoresis Preparation

A 2% agarose gel was prepared by weighing 2 g of agarose powder (Invitrogen, USA) and dissolved in 100 mL 1X TBE buffer in volumetric flask. The mixture was shaken and heated in a microwave oven (SONA, China) for 2 min, during which the mixture was shaken every 20 seconds until the solution became clear. Next, 5 µL of RedSafe Dye (Intron, Korea) was added to the molten agarose gel to stain the mixture, and the solution was poured into a prepared electrophoresis tray and allowed to solidify. After removing the combs and casting gates for sample loading, trays were placed in a buffer tank. Next, 3 µL of the negative control and 100 bp DNA ladder were added to the first and second wells, respectively. PCR product samples were loaded sequentially. The current of gel electrophoresis was set at 120 volts for 20 min. The gel was then transferred to a UV transilluminator for qualitative analysis.

### RFLP Procedure

The amplified PCR products were subjected to restriction digestion with *Tai1* restriction enzyme (Thermo Scientific, USA) for the rs2781666 G/T position. Restriction enzyme digestion of rs2781666 PCR product components was prepared for each sample, with a total volume of 31 µL. All tubes were centrifuged at 2000 RPM for one minute and incubated in a PCR Thermal Cycler (Bio-Rad, USA) at 65 °C for one hour. The enzyme was deactivated by allowing the tubes to cool to 4 Â °C.

Ten microliters of each digestion product were loaded on 2% agarose gel along with a 100bp DNA ladder to establish the length of the digested fragments.

The gel was carried to the UV transilluminator for qualitative analysis.

### Arginase Enzyme Activity Assay

Arginase activity was determined by measuring the urea level in each sample, as the ARG enzyme catalyzes the conversion of L-arginine into urea. The produced urea reacted specifically with the color development reagent L-isonitrosopropiophenone (ISPF) to generate a colored product that was proportional to the arginase activity present in the sample. Absorbance was measured at 540 nm using a spectrophotometer (Thermo Scientific, USA). By setting up a standard curve for urea concentration versus absorbance, readings can be used to calculate ARG activity.[21](#cit0021)

In brief, 25 μL of the supernatant was heated with MnCl2 (10 mm) for 10 min at 56°C to activate the arginase. The mixture was then incubated with 50 μL L-arginine (0.5 M, pH 9.7) for 1 h at 37°C to hydrolyze L-arginine. The hydrolysis reaction was stopped with acid and the mixture was then heated at 100°C with 25 μL 9% α-ISPF in ethanol for 45 min. The samples were kept in the dark at room temperature for 10 min and the absorbance was measured at 540 nm.

### Genotyping of ARG1 rs2781666 Results

All subjects were genotyped for *ARG1* rs2781666 polymorphism. The PCR product, *ARG1* polymorphism was confirmed using 2% agarose gel electrophoresis, as shown in [Figure 1](#f0001). The PCR products (294 bp) were digested by *Tai1* restriction enzyme, resulting in three genotypes viewed on the agarose gel under a UV transilluminator: homozygous wild (GG) genotype, which appeared as a single band with a length of 294 bp; heterozygous (GT) genotype, which appeared as three bands with lengths of 112 bp, 182 bp, and 294 bp; and the homozygous mutant (TT) genotype, which appeared as two bands of lengths 112 bp and 182 bp.

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/52ba/12182068/e623ab232741/JAA-18-1023-g0001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=12182068_JAA-18-1023-g0001.jpg)

Agarose gel electrophoresis of ARG1 PCR products. Lane M refers to a 100 bp DNA ladder. Lane -ve is a negative control sample, while lanes 1–5 are random PCR products of ARG1 gene amplification.

[Figure 2](#f0002) shows the resultant restriction fragments of rs2781666 obtained by 2% agarose gel electrophoresis.

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/52ba/12182068/8cf11a04f054/JAA-18-1023-g0002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=12182068_JAA-18-1023-g0002.jpg)

Agarose gel electrophoresis of ARG1 rs2781666 genotypes. Lane M refers to a 100 bp DNA ladder, lanes (1,2,4,6,8,10,12,13,14) represent the homozygous wild-type GG genotype, lanes (5,7,9,11) are heterozygous GT genotype, and lane (3) represents the homozygous mutant TT genotype.

### Statistical Analysis

To determine genotype frequencies, the Hardy–Weinberg equation was used: p_2_ + 2pq +q_2_ =1, where p_2_ is the dominant homozygous genotype frequency.

2pq: is heterozygous genotype frequency. q_2_: is recessive homozygous genotype frequency.

To estimate allele frequency among studied subjects, the following equation was used:

| Category |
| --- |
|   |
Both mean and standard deviation (SD) were calculated using Microsoft Excel^®^ 2016, which was also used to measure p-values for continuous variables using Analysis of Variance (ANOVA), a Single Factor test.

Chi-square was calculated using the online calculator for the 2×2 contingency table ([https://www.socscistatistics.com/tests/chisquare/default2.aspx](https://www.socscistatistics.com/tests/chisquare/default2.aspx)).

The z-score was used to test for differences in minor allele frequencies among populations using the online z-score calculator for a two-tailed test at [https://www.socscistatistics.com/tests/ztest/default2.aspx](https://www.socscistatistics.com/tests/ztest/default2.aspx).

The Hardy-Weinberg equilibrium (HWE) was tested using an online calculator [https://www.wolframalpha.com/widgets/gallery/view.jsp?id=2fefa8b126607e29fe2990c722ee6cae](https://www.wolframalpha.com/widgets/gallery/view.jsp?id=2fefa8b126607e29fe2990c722ee6cae).

A *t*-test was used to test two independent sample means using an online calculator at [https://www.graphpad.com/quickcalcs/ttest1/?Format=SD](https://www.graphpad.com/quickcalcs/ttest1/?Format=SD).

## Results

### Characteristics of Study Population

Two hundred and fifty-one asthmatic patients and 173 non-asthmatic volunteers were included in this study. Among patients diagnosed with bronchial asthma 157 (63%) were females and 94 (37%) were males. The mean patient age was 46 years ± 16.059. Among non-asthmatic participants, 112 (65%) were females and 61 (35%) were males. The mean age of the non-asthmatic participants was 42 years ± 13.090. [Table 1](#t0001) shows the demographic characteristics of the study population.

### Table 1.

| Demographic Data |  |  |  | p-value (Chi-Square) |
| --- | --- | --- | --- | --- |
| Variables |  | Asthmatic | Non-Asthmatic |  |
| Male |  | (37%) 94 | (35%) 61 | 0.721 |
| Female |  | (63%) 157 | (65%) 112 |  |
| Age (years) | 18–35 | 70 | 51 | 0.0004* |
| 36–45 | 50 | 64 |  |  |
| 46–55 | 62 | 35 |  |  |
| 56–65 | 46 | 13 |  |  |
| 66–75 | 20 | 9 |  |  |
| >75 | 3 | 1 |  |  |
| Mean age ± STD (years) |  | 46 years ± 16.059 | 42 years ± 13.090 | 0.0069¶* |
| Total |  | 251 | 173 |   |

Table 1 Caption: The Demographic Data of Asthmatic and Non-Asthmatic Subjects

A chi-square test of independence was conducted to examine the association between sex and asthma status. The association was not statistically significant, *p*-value =0.721, indicating that sex distribution did not differ significantly between asthmatic and non-asthmatic individuals.

In contrast, a chi-square test comparing age group distributions showed a statistically significant difference between asthmatic and non-asthmatic participants, *p*-value < 0.0001. This suggests that asthma prevalence varied across age groups.

Mean age difference between asthmatic and non-asthmatic subjects is statistically significant. (*p*-value = 0.0069).

A multivariate logistic regression model was used to identify independent predictors of asthma. Smoking (OR = 4.25, p < 0.001), obesity (OR = 1.94, *p* = 0.013), and male gender (OR = 0.47, *p* = 0.005) were statistically significant. Additionally, individuals aged 18–35 and 36–50 had significantly lower odds of asthma compared to those aged 66 and above (*p* = 0.039 and *p* = 0.001, respectively).

### Genotype and Allele Frequencies of ARG1 rs2781666 Among Study Group

PCR-RFLP analysis of rs2781666 was performed in 424 subjects; 251 were asthmatic patients and 173 were non-asthmatic subjects. The wild-type (GG) genotype was detected in 130 (0.514) asthmatic and 117 (0.643) non-asthmatic subjects, the heterozygous (GT) genotype was detected in 100 (0.406) asthmatic and 44 (0.318) non-asthmatic subjects, and the homozygous mutant (TT) genotype was detected in 21 (0.080) asthmatic and 12 (0.039) non-asthmatic subjects. The frequency of the wild allele G was 0.717 among asthmatics volunteers and 0.803 among non-asthmatic volunteers. The frequency of the variant allele T was 0.283 in asthmatics and 0.197 in non-asthmatic volunteers.

The *ARG1* rs2781666 genotype frequencies in the control group were tested against the Hardy-Weinberg equilibrium (HWE) principle, and there were no significant differences between the observed and expected frequencies (*p > 0.05*), indicating that the rs2791666 polymorphism was in HWE.

### Comparison of ARG1 rs2781666 Genotype and Allele Frequencies Between Asthmatics and Non Asthmatic Subjects

The association between *ARG1* polymorphisms and susceptibility to bronchial asthma was investigated using the z-score test. The association was considered significant when the *p*-value was < 0.05.

[Table 2](#t0002) presents a comparison of *ARG1* polymorphism genotype frequencies between asthmatic and non-asthmatic subjects.

### Table 2.

| Genotype | Asthmatic | Non-Asthmatic | z-score | p-value | Odds Ratio (95% CI) |
| --- | --- | --- | --- | --- | --- |
| GG | 130 | 117 | 3.250- | 0.001* | 0.514 (0.343–0.769) |
| GT | 100 | 44 | 3.078 | 0.002* | 1.941 (1.268–2.971) |
| TT | 21 | 12 | 0.540 | 0.589 | 1.225 (0.586–2.560) |
| Total | 251 | 173 | -- | -- | -- |

Table 2 Caption: Comparison of Genotype Frequencies of rs2781666 Genotype Between Asthmatic Patients and Non-Asthmatic Subjects

[Table 3](#t0003) presents a comparison of the *ARG1* polymorphism allele frequencies between asthmatic and non-asthmatic subjects.

### Table 3.

| Allel | Asthmatic | Non-Asthmatic | z-score | p-value | Odds Ratio (95% CI) |
| --- | --- | --- | --- | --- | --- |
| G | 360 | 278 | 2.862- | 0.004* | --- |
| T | 142 | 68 | 2.862 | 0.004* | 1.612 (1.160–2.240) |
| Total | 502 | 346 | -- | -- | --- |

Table 3 Caption: Comparison of Allele Frequencies of rs2781666 Allele Between Asthmatic Patients and Non-Asthmatic Subjects

The GG genotype frequency was significantly higher among non-asthmatic subjects, and

The frequency of the GT genotype was significantly higher in asthmatic patients, suggesting that the GT genotype is strongly associated with bronchial asthma (*p* < 0.05). The TT genotype was not significantly different between the two groups (*p* > 0.05). Bonferroni correction was applied, the adjusted p-value = 0.0167. The GG and GT genotypes comparisons remain significant (p-value = 0.001 and 0.002 respectively).

Allelic association comparison indicated a high distribution of the G allele in non-asthmatic subjects compared to that in asthmatics subjects (*p* < 0.05), suggesting a protective role of the G allele against bronchial asthma, whereas the T allele is a risk allele for the disease.

The differences among populations in minor allele frequencies were tested using z-score for two-tailed test.

By applying chi-square test to analyze the differences of all genotypes (GG, GT, and TT) frequencies among asthmatic and non-asthmatic groups, the p-value was 0.004 which is considered significant (*p*<0.05).

In a multivariate logistic regression model adjusted for gender and age, genotype was significantly associated with asthma status (*p* = 0.006). However, when compared individually to the GG genotype, neither GT (aOR = 0.60, 95% CI: 0.28–1.29, *p* = 0.189) nor TT (aOR = 1.20, 95% CI: 0.53–2.68, *p* = 0.664) were statistically significant. Age group was a significant independent predictor (aOR = 1.43, 95% CI: 1.15–1.77, *p* = 0.001), while gender was not significant (aOR = 0.897, 95% CI: 0.591–1.360, *p* = 0.607).

A multivariate logistic regression analysis showed that the number of T alleles was significantly associated with asthma status. After adjusting for gender and age group, each additional T allele conferred a 1.57-fold increased risk of asthma (aOR = 1.569, 95% CI: 1.137–2.165, p = 0.006). Age group was also significantly associated with asthma risk (aOR = 1.439, 95% CI: 1.160–1.785, *p* = 0.001, while gender was not significant (aOR = 0.922, 95% CI: 0.610–1.394, *p* = 0.70).

### Association Between rs2781666 Polymorphism and Different Population Variables Among Study Groups

#### Association Between rs2781666 Polymorphism and Severity of Ventilatory Impairment

The asthmatic group in this study was stratified according to severity of ventilatory impairment depending on FEV1 results into; FEV1 > 80%, 60% ≤ FEV1 ≤ 80%, and FEV1< 60% as mild, moderate, and severe, respectively. The comparison of the rs2781666 genotype (GG, GT, and TT) distribution among severity groups revealed a significant association with *p*-value of 0.005, 0.016, and 0.035, respectively. The GG genotype was more frequent in the mild asthma group, GT genotype was more frequent in the moderate asthma group, and TT genotype was more frequent in the severe asthma group. Chi-square test was applied to determine whether the association between genotypes frequencies (GG,GT, and TT) vs severity groups (mild, moderate, and severe) is statistically significant. *P*-value was 0.00004 which is considered significant.

By applying Bonferroni correction, the adjusted p-value was 0.0167. The association between GG genotype and severity groups was statistically significant (*p* = 0.005). Similar is applied to the association between GT genotype and severity groups, the association was significant with *p* = 0.016. However, TT genotype distribution among severity groups revealed non-significant association with *p*-value = 0.035 at the corrected threshold.

[Table 4](#t0004) presents the distribution of rs2781666 genotypes among the severity of ventilatory impairment groups.

| Genotype | FEV1 > 80% | 60% ≤ FEV1 ≤ 80% | FEV1< 60% | p-value |
| --- | --- | --- | --- | --- |
| GG | (0.654) 85 | (0.382) 29 | (0.326) 14 | 0.005* |
| GT | (0.292) 38 | (0.539) 41 | (0.488) 21 | 0.016* |
| TT | (0.054) 7 | (0.079) 6 | (0.186) 8 | 0.035* |
| Total | 130 | 76 | 43 |   |

Table Caption: rs2781666 Genotype Distribution Among Severity Groups

The associations between the rs2781666 genotypes GG and TT and bronchial asthma among FEV1 ≥ 60% group compared to FEV1 < 60% group, were significant *(p-*value=0.006 and *p-*value=0.008, respectively). There was no significant association between the GT genotype and severity of ventilatory impairment groups *(p-*value = 0.200). However, the GT genotype showed a non-significant (*p*=0.200) higher frequency in FEV1 < 60%, group compared to FEV1 ≥ 60%) group. Applying Bonferroni correction, the adjusted p-value was 0.0167. The results remain the same at the corrected threshold. [Table 5](#t0005) presents the results of the study.

| Genotype | FEV1 ≥ 60% | FEV1 < 60% | z-score | p-value | Odds Ratio (95% CI) |
| --- | --- | --- | --- | --- | --- |
| GG | (0.553) 114 | (0.326) 14 | 2.718 | 0.006* | 2.566 (1.281–5.140) |
| GT | (0.383) 79 | (0.488) 21 | 1.276- | 0.200 | 0.651 (0.336–1.261) |
| TT | (0.063) 13 | (0.186) 8 | 2.638- | 0.008* | 0.294 (0.113–0.763) |
| Total | 206 | 43 | -- | -- | --- |

Table Caption: rs2781666 Genotype Frequencies Comparison Between FEV1 ≥ 60% and FEV1 < 60%

In addition, comparing the rs2781666 GG and GT genotypes and bronchial asthma between FEV1 > 80% group compared to FEV1 ≤ 80% group, showed a significant association (*p* < 0.0001 and *p* = 0.0002, respectively). The TT genotype showed a non-significant association with the tested groups. Applying Bonferroni correction, the adjusted p-value was 0.0167. The results remain the same at the corrected threshold. [Table 6](#t0006) presents the results of the study.

| Genotype | FEV1>80% | FEV1≤80% | z-score | p-value | Odds Ratio (95% CI) |
| --- | --- | --- | --- | --- | --- |
| GG | (0.654) 85 | (0.361) 43 | 4.613 | <0.0001* | 3.338 (1.984–5.615) |
| GT | (0.292) 38 | (0.521) 62 | 3.677- | 0.0002* | 0.379 (0.225–0.639) |
| TT | (0.054) 7 | (0.118) 14 | 1.809- | 0.0703 | 0.426 (0.166–1.097) |
| Total | 130 | 119 | --- | --- | --- |

Table Caption: rs2781666 Genotype Frequencies Comparison Between FEV1 > 80% and FEV1 ≤ 80%

#### Association Between rs2781666 Polymorphism and Gender

The rs2781666 TT polymorphism was significantly associated with male sex in the asthma group (p=0.03). However, there were no significant associations between GG and GT polymorphisms and sex when compared within the asthmatic and non-asthmatic groups. However, the GT genotype was more frequent in females (41.4%) compared to males (37.2%) in the asthmatic group. Furthermore, comparing the association of rs2781666 genotypes with sex between the study groups revealed that there was a significant association between female sex and GG and GT genotypes, with *p*-values of 0.005 and 0.002, respectively. Regarding the male sex, there was no significant association with rs2781666 genotypes between the studied groups. After applying Bonferroni correction for multiple comparisons (adjusted *p* = 0.0056), only the differences in GG (*p* = 0.005) and GT (*p* = 0.002) genotype frequencies among female asthmatic and non-asthmatic individuals remained statistically significant. The previously observed association of the TT genotype in asthmatic males (*p* = 0.030) did not remain significant after correction. This is shown in [Table 7](#t0007).

| Asthmatic Group |  |  |  |  |
| --- | --- | --- | --- | --- |
| Genotype | Female | Male | z-score | p-value |
| GG | (0.535) 84 | (0.5) 47 | 0.538 | 0.589 |
| GT | (0.414) 65 | (0.372) 35 | 0.864 | 0.390 |
| TT | (0.051) 8 | (0.127) 12 | 2.171- | 0.030* |
| Total | 157 | 94 | --- | --- |
| Non-asthmatic Group |  |  |  |  |
| Genotype | Female | Male | z-score | p-value |
| GG | (0.703) 78 | (0.603) 35 | 1.3015 | 0.193 |
| GT | (0.234) 26 | (0.310) 18 | 1.070- | 0.284 |
| TT | (0.063) 7 | (0.086) 5 | 0.556- | 0.575 |
| Total | 111 | 58 | --- | --- |
|   | Asthmatic vs Non-asthmatic(Female) |  | Asthmatic vs Non-asthmatic(Male) |  |
| Genotype | z-score | p-value | z-score | p-value |
| GG | 2.765- | 0.005* | 1.243- | 0.214 |
| GT | 3.061 | 0.002* | 0.7791 | 0.435 |
| TT | 0.424- | 0.674 | 0.787 | 0.429 |

Table Caption: Association Between rs2781666 Polymorphism and Gender in Asthmatic and Non-Asthmatic Groups

#### Association Between rs2781666 Polymorphism and BMI

Regarding BMI, participants were stratified according to BMI groups as non-obese (healthy and overweight) and obese. There was no significant difference between asthmatic and non-asthmatic obese (*p* = 0.177 and 0.453, respectively) for the GG and GT genotypes. However, the GG genotype frequency was non-significantly higher in non-asthmatic obese than in asthmatic obese groups, and the GT genotype frequency was higher in asthmatic obese than in non-asthmatic obese groups, although this did not reach statistical significance. There was a significant association between the GG and GT genotype frequencies in the non-obese asthmatic and non-asthmatic groups (*p* = 0.001). Bonferroni correction was applied and the adjusted *p*-value = 0.0083. The results remain the same at the corrected threshold. These results are shown in [Table 8](#t0008).

| Genotype | Asthmatics |  | Non-Asthmatic |  | Asthmatics vs Non-Asthmatic (Non-obese) |  | Asthmatic vs Non-Asthmatic (Obese) |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Non-obese | Obese | Non-obese | Obese | z-score | p-value | z-score | p-value |  |
| GG | 109 (0.535) | 21 (0.447) | 90 (0.709) | 27 (0.587) | 3.1502- | 0.001* | 1.3522- | 0.177 |
| GT | 78 (0.382) | 22 (0.468) | 26 (0.205) | 18 (0.391) | 3.227 | 0.001* | 0.7477 | 0.453 |
| TT | 17 (0.083) | 4 (0.085) | 11 (0.086) | 1 (0.022) | 0.1043- | 0.920 | 1.3546 | 0.177 |
| Total | 204 | 47 | 127 | 46 | -- | -- | -- | -- |

Table Caption: Association Between rs2781666 Polymorphism and BMI in Asthmatic and Non-Asthmatic Groups

### Plasma Arginase Activity Results

#### Comparison of Mean Values of Plasma ARG Activity

One hundred and seventy-seven study subjects’ plasma were assessed for ARG activity. Among them, 70 were from the asthmatic group and 107 were from the non-asthmatic group. The results are expressed as the mean ± standard deviation (SD) of ARG concentrations (mM) in both asthmatic and healthy groups. These results are presented in [Figure 3](#f0003), which shows a significant elevation in ARG activity among asthmatic patients compared to non-asthmatic subjects.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/52ba/12182068/5b0793fb1d42/JAA-18-1023-g0003.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=12182068_JAA-18-1023-g0003.jpg)

Plasma arginase activity in 107 non-asthmatic subjects and 70 asthmatic patients. Boxes represent median and interquartile range; the whiskers represent the full range of values obtained. Arginase activity was significantly higher in asthmatic patients than in non-asthmatic subjects. (*p >0.05).

The Kolmogorov–Smirnov test was used to test the study sample distribution for normality. Since the data were normally distributed, analysis of variance (ANOVA) and unpaired t-tests were used to examine the differences in continuous variables.

#### Correlation Between rs2781666 Genotype Distribution with ARG Activity Among Asthmatic and Non-Asthmatic Groups

Both asthmatic and non-asthmatic groups were stratified according to their rs2781666 genotype and arginase activity, as shown in [Table 9](#t0009). There was a significant association between the rs2781666 genotype and mean ARG activity among the asthmatic and non-asthmatic groups. The GT genotype was significantly associated with higher plasma arginase activity in asthmatic patients than that in non-asthmatic subjects (p = 0.0032). Additionally, the GG genotype was associated with significantly higher ARG activity in asthmatic patients than that in non-asthmatic volunteers (*p* = 0.0349). However, TT genotype was not significantly prevalent in the non-asthmatic group than in the asthmatic group. The Bonferroni correction was applied, the adjusted significance threshold was 0.0167. After Bonferroni correction, only GT genotype comparison remains statistically significant. The results of this study are presented in [Table 9](#t0009).

| Genotype | Arginase Activity mM (Mean+ SD) |  | P-value˧ |
| --- | --- | --- | --- |
| Asthmatic (n=70) | Non-Asthmatic (n=107) |  |  |
| GG | 171.2 ± 36.4 | 156.5 ± 28.8 | 0.0349* |
| GT | 175.6 ± 49.6 | 149.1 ± 22.2 | 0.0032* |
| TT | 141 ± 23.1 | 158.6 ± 14.6 | 0.0832 |

Table Caption: Correlation Between rs2781666 Genotype and ARG Activity Among Asthmatic and Non-Asthmatic Groups

#### Comparison Between the Distribution of rs2781666 Genotypes an ARG Activity Among Asthmatic Group

There was a non-significant association between the distribution of the rs2781666 genotypes and ARG activity (*p* >0.05). ARG activity in TT carriers was non-significantly lower than that in heterozygous GT and wild-type GG genotypes (*p* > 0.05). Applying Bonferroni correction, the adjusted p-value = 0.0167. The difference observed in GG vs TT is not statistically significant at the corrected threshold. The results are presented in [Table 10](#t0010).

| Genotype | Asthmatic Group (n=70) | p-value |
| --- | --- | --- |
| Mean ARG Activity ± SD (mM) |  |  |
| GG | 171.2 ± 36.4 | 0.1431¶ |
| GT | 175.6 ± 49.6 |  |
| TT | 141 ± 23.1 |  |
| GG vs GT |   | 0.6859˧ |
| GG vs TT |   | 0.0417˧* |
| GT vs TT |   | 0.0845˧ |

Table Caption: Comparison Between Distribution of rs2781666 Genotype and Plasma ARG Activity Among Asthmatic Group

#### Comparison Between rs2781666 Genotype Distribution and ARG Activity Among Non-Asthmatic Group

There was a non-significant association between the distribution of the rs2781666 genotypes and plasma ARG activity in the non-asthmatic group. The GT genotype showed non-significantly lower plasma ARG activity than the GG and TT genotypes (*p* >0.05). The results of this study are presented in [Table 11](#t0011).

| Genotype | Non-Asthmatic Group (n=107) | p-value |
| --- | --- | --- |
| Mean ARG Activity ± SD (mM) |  |  |
| GG | 156.5 ± 28.8 | 0.308¶ |
| GT | 149.1 ± 22.2 |  |
| TT | 158.6 ± 14.6 |  |
| GG vs GT |   | 0.167˧ |
| GT vs TT |   | 0.2266˧ |
| GG vs TT |   | 0.8318˧ |

Table Caption: Comparison Between Distribution of rs2781666 Genotype and Plasma ARG Activity Among Non-Asthmatic Group

#### Association Between ARG Activity and Severity Among Asthmatic Group

The asthmatic group was stratified according to asthma severity depending on FEV1 results into; FEV1 > 80% (Group 1), 60% ≤ FEV1 ≤ 80% (Group 2), and FEV1< 60% (Group 3) as mild, moderate, and severe asthma, respectively. [Table 12](#t0012) shows the correlation between arginase activity (mean ± SD) and asthma severity in the bronchial asthma group.

| Asthma severity | Arginase Activity(mean ± SD) (mM) | p-value¶ |
| --- | --- | --- |
| FEV1 > 80% | 166.4 ± 34.1 | 0.202 |
| 60% ≤ FEV1 ≤ 80% | 169.5 ± 41.8 |  |
| FEV1 < 60% | 180 ± 64.7 |  |

Table Caption: Comparison Between ARG Activity and Asthma Severity Among Asthmatic Groups

There was no significant association between mean arginase activity and asthma severity (*p* = 0.202). However, the results showed non-significant higher mean arginase activity in severe asthma than in mild asthma ([Figure 4](#f0004)).

![Figure 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/52ba/12182068/6efef5ccde49/JAA-18-1023-g0004.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=12182068_JAA-18-1023-g0004.jpg)

Comparison between ARG activity and severity of asthma among asthmatic groups. The whiskers represent the full range of values obtained. No significance was observed among the groups (p = 0.255).

#### Comparison of Minor Allele Frequencies of rs2781666 in Healthy Controls Between Jordanian Population and Other Populations

Regarding the minor allele frequency (T) of the rs2781666 SNP in non-asthmatic subjects, there was no significant difference in minor allele frequencies between our study and studies conducted in Caucasian, Polish, French, Pakistani, and Indian populations (*p* > 0.05).[15](#cit0015)^,^[19](#cit0019)^,^[22–24](#cit0022)

However, the minor allele frequency was significantly different from that reported in Algerian and Tunisian populations (*p* = 0.015 and 0.009, respectively).[25](#cit0025)^,^[26](#cit0026) A comparison between the present study and other studies conducted in different populations is shown in [Table 13](#t0013).

| Reference | Population | Minor Allele (T) Frequency (for controls) | SampleSize | z-score | p-value |
| --- | --- | --- | --- | --- | --- |
| This study | Jordanian | 0.196 | 173 | --- | --- |
| (Buraczynska & Zakrocka, 2021)22 | Caucasian and Polish origin | 0.213 | 400 | 0.4604- | 0.645 |
| (Shah et al, 2018)24 | Pakistani | 0.136 | 500 | 1.8976 | 0.057 |
| (Sediri et al, 2010)26 | Tunisian | 0.108 | 282 | 2.6147 | 0.009* |
| (Meroufel et al, 2009)25 | Algerian | 0.321 | 117 | 2.423- | 0.015* |
| (Donthi et al, 2018)19 | Indian | 0.21 | 150 | 0.312- | 0.756 |
| (Dumont et al, 2007)23 | French | 0.182 | 581 | 0.4161 | 0.674 |

Table Caption: Comparison of Minor Allele Frequencies Among Various Populations

## Discussion

Bronchial asthma is a chronic inflammatory disease caused by various genetic and environmental factors that impose major socioeconomic and health burdens. The interaction between these risk factors affects disease severity and response to treatment.[27](#cit0027) Its prevalence is increasing worldwide, and in Jordan, it has doubled over the past decade.[5](#cit0005)

Genetic risk factors play an important role in the development and severity of various diseases. One of these genes, *ARG1* gene, has been investigated for its effects on many inflammatory conditions, including bronchial asthma.[28](#cit0028)

To the best of our knowledge, this is the first study to investigate the role of *ARG1* rs2781666 G/T polymorphism in susceptibility to bronchial asthma, to measure plasma arginase activity, and to correlate the results with the frequency and severity of bronchial asthma in the Jordanian population.

In this study, *ARG1* rs2781666 SNP and plasma arginase activity were examined to investigate the relationship between the distribution of *ARG1* rs2781666 genotypes and bronchial asthma frequency and severity in Jordan, to examine the association of rs2781666 genotypes with other baseline characteristic variables, and to correlate *ARG1* polymorphisms with plasma arginase activity in asthmatic and non-asthmatic Jordanian populations.

Regarding the minor allele frequency (T) of the rs2781666 SNP in non-asthmatic subjects, there were no significant differences in minor allele frequencies between our study and studies conducted in Caucasian, Polish, French, Pakistani, and Indian populations (*p* > 0.05).

In this study, *ARG1* rs2781666 T variant was found to be associated with the risk of bronchial asthma and asthma severity. There was a significant association between *ARG1* rs2781666 polymorphism and bronchial asthma (*p* < 0.05). The frequency of *ARG1* polymorphism GT genotype among asthmatic group compared to non-asthmatic group was significant, with an OR of 1.941 (95% CI 1.26–2.97, *p* = 0.002), indicating the susceptibility to bronchial asthma. Also, the GG genotype was more frequent among non-asthmatic subjects than among asthmatics, with OR of 0.514 (95% CI 0.343–0.769, *p* = 0.001). However, TT genotype frequency was not significantly different between the asthmatic and non-asthmatic groups (*p* = 0.589), which is inconsistent with other reports that patients with the TT genotype are at a higher risk of bronchial asthma (OR 1.71, CI 0.74–3092, *p=*0.001).[19](#cit0019) This discrepancy may be due to the small sample size of the present study. Also, allelic frequency comparisons indicated high G allele frequency in non-asthmatic subjects compared to T allele frequency in asthmatic patients with OR = 1.61 (95% CI 1.16–2.24, *p*=0.004). These results are consistent with established association studies that showed that the frequencies of the wild GG genotype and wild G allele were more significant in non-asthmatic subjects (*p* < 0.05), and the frequencies of the heterozygous GT genotype and minor allele T were more significant in asthmatic patients (*p* < 0.05), suggesting that the GG genotype and G allele may offer protection against bronchial asthma, and the GT genotype and T allele may confer susceptibility to bronchial asthma.[19](#cit0019)

In the present study, the distribution of rs2781666 genotypes among severity of ventilatory impairment groups showed that the GG genotype was significantly more frequent in mild asthmatic patients with FEV1 level more than 80% (*p*<0.05), the GT genotype had significantly higher frequency in patients with moderate asthma, and the TT genotype had significantly higher frequency in severe asthma patients with FEV1 levels below 60% (*p*<0.05). This indicates the role of rs2781666 GT and TT genotypes in worsening bronchial asthma symptoms. Further studies with larger sample sizes and other populations are required to confirm these findings.

Furthermore, comparison the SNP genotype frequencies between FEV1 ≥ 60% group vs FEV1 < 60% group showed significantly higher frequency of the TT genotype in FEV1 < 60% group compared to FEV1 ≥ 60% group with OR, 0.29 (95% CI 0.11–0.763, *p* = 0.008). Also, the GG genotype frequency was significantly higher in FEV1 ≥ 60% group, with OR of 2.55 (95% CI 1.28–5.14, *p* = 0.006). However, GT genotype frequency was not significantly different between group patients (*p*>0.05). In addition, comparing rs2781666 genotype frequencies between FEV1 > 80% and FEV1 ≤ 80%, showed significantly higher distribution of GG genotype in FEV1 > 80% with OR of 3.33 (95% CI 1.98–5.61, *p* < 0.05), and GT genotype frequency was significantly higher in patients with FEV1 levels below 80% with OR, 0.37 (95% CI 0.22–0.63, *p* = 0.0002). However, the TT genotype was not significantly different between the groups (*p*>0.05), although in the present study, it showed a significantly higher frequency among patients with FEV1 ≤ 80% (*p* = 0.035). Altogether, these findings may provide insights into the role of *ARG1* rs2781666 genotypes in bronchial asthma severity and may introduce the TT genotype and T allele as risk factors for severe bronchial asthma.

The association of rs2781666 genotypes with gender between the study groups revealed significant associations between female gender and the GG and GT genotypes. The GG genotype was significantly higher in non-asthmatic females compared to asthmatic females (*p* = 0.005). Furthermore, the GT genotype was significantly higher in asthmatic females compared to non-asthmatic female subjects (*p* = 0.002). However, the GT genotype was more frequent in female group (41.4%) compared to male group (37.2%) in asthmatic group, although this did not reach statistically significance (*p* = 0.39). These results are consistent with Hansenet al, (2015) data that adult female patients were found to have higher prevalence of asthma (9.2%) compared to adult male patients (7.0%).[29](#cit0029) Moreover, the previously observed association of the TT genotype in asthmatic males (*p* = 0.030) did not remain significant after correction, suggesting that it may have been a chance finding. However, in both unadjusted and adjusted models, gender was not significantly associated with asthma status. Although the odds of asthma were slightly lower in males (adjusted OR = 0.92, 95% CI: 0.61–1.39), this difference was not statistically significant (*p* = 0.700), indicating no clear association between gender and asthma in this study population. The lack of a statistically significant association between gender and asthma in this study may be influenced by several limitations, including sample size imbalance, potential confounding factors, and the absence of data on asthma phenotypes and hormonal influences. Future studies with larger, gender-balanced samples and longitudinal designs are recommended to more accurately determine the independent role of gender in asthma risk, severity, and genotype interaction.

However, there was no difference between the rs2781666 polymorphism genotype distribution and smoking status among the asthmatic group (*p* > 0.05).

Regarding BMI, the stratified groups showed no significant differences between *ARG1* SNP polymorphism and BMI in asthmatic patients. There was no significant association between the rs2781666 genotype frequency and obesity in the asthmatic and non-asthmatic groups. After applying Bonferroni correction for multiple comparisons (adjusted *p* = 0.0083), only two significant associations remained: the GG and GT genotypes showed significant differences between asthmatic and non-asthmatic individuals in the non-obese subgroup (p = 0.001 for both). Specifically, the GG genotype was less frequent and the GT genotype more frequent in asthmatics, suggesting a genotype-related asthma risk that may be more evident in non-obese individuals. No statistically significant genotype differences were observed within the obese group after correction. The observation that genotype associations were significant only in non-obese individuals may reflect a gene-environment interaction. Obesity is a known risk factor for asthma and may modulate or mask genetic susceptibility by introducing additional inflammatory mechanisms. In non-obese individuals, where such metabolic influences are less pronounced, the genetic effects (particularly of GG and GT genotypes) may be more clearly expressed. Additionally, the smaller sample size in the obese subgroup may have limited the ability to detect weaker genetic associations after multiple comparison correction. This result was similar to[30](#cit0030) study that bronchial asthma prevalence in lean adults was lower than that in obese adults (7.1% and 11.1%, respectively).

In the present study, we assessed the correlation between plasma arginase activity and bronchial asthma susceptibility by comparing the mean ± SD of plasma arginase activity between the asthmatic and non-asthmatic groups. The arginase activity was significantly higher in asthmatic patients than in non-asthmatic groups (*p* = 0.003). According to the literature, elevated arginase activity contributes to a decrease in the bioavailability of L-arginine to NOS isozymes, causing a reduction in NO production, which contributes to AHR in the airways.[17](#cit0017) Our results agree with those of Oakland (USA) and Japanese populations[30](#cit0030)^,^[31](#cit0031) regarding observations of arginine activity elevation in bronchial asthma. Previous data have demonstrated high arginase activity in the serum of asthmatic patients compared to non-asthmatic patients, indicating an increased risk of bronchial asthma.[19](#cit0019)^,^[32](#cit0032)

To assess the correlation between arginase activity and the distribution of rs2781666 genotypes, the genetic analysis of *ARG1* genotypes were compared to the mean ± SD of plasma arginase activity among asthmatic and non-asthmatic groups. The results showed a positive association between elevated plasma arginase activity and the GT genotype in asthmatic patients compared to that in the non-asthmatic group (*p*=0.003). However, after Bonferroni correction applied, the arginase activity of subjects bearing the GG genotype was not significantly higher in the asthmatic group than in the non-asthmatic group (*p*=0.034), and subjects with the TT genotype had non-significantly lower arginase activity in the asthmatic group than in the non-asthmatic group (p=0.083). This is inconsistent with the data reported in the literature that the *ARG1* mutant TT genotype at rs2781666 is strongly associated with higher mean arginase activity than GT and GG genotypes in patients with essential hypertension.[24](#cit0024) Moreover, a significant association between the G/T genotype of the rs2781666 SNP in the *ARG1* gene and asthma risk, Individuals carrying the T allele (GT or TT genotypes) had a higher prevalence of asthma, and this was accompanied by elevated serum levels of arginase.[19](#cit0019) This contradiction may be due to the small sample size, which was a limitation of this study. Bronchial asthma is a heterogeneous inflammatory disease, and disease phenotypes and elevation of arginase activity cannot be predicted by a single gene. A recent study demonstrated that many inflammatory mediators and reactive oxygen species are drivers of the pathologic elevation of ARG expression and activity.[10](#cit0010)^,^[12](#cit0012)

Correlations between arginase activity and SNP distribution within each of the asthmatic and non-asthmatic groups were examined, and the results revealed that the TT genotype showed non-significantly lower plasma arginase activity than the GG and GT genotypes in asthmatic patients. In contrast, the TT genotype was not significantly associated with a higher plasma arginase activity than the GT genotype in the non-asthmatic group. In addition, when we compared both the GG and GT genotypes based on arginase activity among the non-asthmatic group, the GG genotype showed a non-significant elevation in arginase activity compared to the GT genotype (*p* > 0.05). These results were different from those reported by Shah et al, who showed that carriers of the TT genotype had maximal arginase activity, and GG genotype carriers had the least activity of arginase enzyme among patients with idiopathic dilated cardiomyopathy (*p* < 0.05).[28](#cit0028) As mentioned earlier, inconsistencies in the results may be due to the small sample size of this study. Other reasons include the need for adjustments for environmental and lifestyle factors, which may have contributed to the variation in the TT genotype’s association.

According to literature, elevated plasma arginase activity could contribute to the severity of asthma by increasing collagen and mucus production, leading to airway constriction in bronchial asthma.[19](#cit0019) To assess the published data, we stratified the asthmatic group according to disease severity and examined the association of arginase activity between the tested groups. These results were contrary to Donthi et al, (2018) study in which our study showed a non-significant association between arginase activity and the severity of asthma (*p* = 0.202), which is probably due to the small sample size of the tested groups.[15](#cit0015)

## Conclusion

The *ARG1* rs2781666 polymorphism may influence asthma severity by affecting the function of arginase 1, an enzyme involved in the metabolism of L-arginine. Arginase 1 competes with nitric oxide synthase for L-arginine, which reduces the production of nitric oxide (NO) that is vital for the relaxation of the airways and control of inflammation. Variations in *ARG1*, such as rs2781666, can lead to higher arginase activity, which can decrease NO levels thus contributing to increased airway inflammation and hyperresponsiveness. This can worsen asthma symptoms and severity. Understanding this genetic variant could help predict asthma outcomes and guide more personalized treatments, especially for patients with difficult-to-control asthma.

There was a significant association between *ARG1* rs2781666 G/T polymorphism and bronchial asthma in an adult Jordanian population.

A significant association was observed between increased plasma arginase activity and bronchial asthma frequency in the adult Jordanian population, which may serve as a biological marker of asthma.

The major allele G appears to protect against bronchial asthma, whereas the minor allele T is increased in patients with bronchial asthma.

Increased plasma arginase activity was consistent with a higher frequency of the GT genotype in the Jordanian population with asthma, representing a risk factor for bronchial asthma.

Altogether, these results indicate a role for the *ARG1* gene in the development and severity of bronchial asthma, supporting previous studies that arginase may be a potential therapeutic target in bronchial asthma.

## Funding Statement

This study was supported by The University of Jordan Deanship of Scientific Research (EMG and AS).

## Disclosure

The authors report no conflicts of interest in this work.

## References

1. WHO. WHO Asthma-World Health Organization-FACTSHEETS; 2024. Available from: https://www.who.int/news-room/fact-sheets/detail/asthma. Accessed December 12, 2024.  [https://www.who.int/news-room/fact-sheets/detail/asthma](https://www.who.int/news-room/fact-sheets/detail/asthma)

2. GINA Committee. Global Strategy for Asthma Management and Prevention 2023 Update. Global Initiative for Asthma; 2023.

3. GINA. Global initiative for asthma: asthma management and prevention, 2019. Practice Nurse. 2019;49(5).  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Practice%20Nurse&title=Global%20initiative%20for%20asthma:%20asthma%20management%20and%20prevention,%202019&volume=49&issue=5&publication_year=2019&)

4. Al Karsaneh OA, Al Anber A, Al Shboul S, et al. Impact of anxiety and depression on the level of asthma control among Jordanian Adults with Asthma. J Asthma Allergy. 2024;17:463–476. doi: 10.2147/JAA.S457875  [DOI](https://doi.org/10.2147/JAA.S457875) | [PMC free article](/articles/PMC11114137/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/38784527/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Asthma%20Allergy&title=Impact%20of%20anxiety%20and%20depression%20on%20the%20level%20of%20asthma%20control%20among%20Jordanian%20Adults%20with%20Asthma&author=OA%20Al%20Karsaneh&author=A%20Al%20Anber&author=S%20Al%20Shboul&volume=17&publication_year=2024&pages=463-476&pmid=38784527&doi=10.2147/JAA.S457875&)

5. Al-Sheyab NA, Alomari MA. Prevalence, associated factors, and control level of asthma symptoms among adolescents in Northern Jordan. Int J Adolesc Med Health. 2020;32(3). doi: 10.1515/ijamh-2017-0159  [DOI](https://doi.org/10.1515/ijamh-2017-0159) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29331100/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Adolesc%20Med%20Health&title=Prevalence,%20associated%20factors,%20and%20control%20level%20of%20asthma%20symptoms%20among%20adolescents%20in%20Northern%20Jordan&author=NA%20Al-Sheyab&author=MA%20Alomari&volume=32&issue=3&publication_year=2020&pmid=29331100&doi=10.1515/ijamh-2017-0159&)

6. North ML, Meurs H, Zaagsma J, Scott JA, Maarsingh H. Arginase in Asthma – recent developments in animal and human studies~!2009-11-12~!2010-03-23~!2010-05-04~! Open Nitric Oxide J. 2010;2(2):20–36. doi: 10.2174/1875042701002020020  [DOI](https://doi.org/10.2174/1875042701002020020) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Open%20Nitric%20Oxide%20J&title=Arginase%20in%20Asthma%20%E2%80%93%20recent%20developments%20in%20animal%20and%20human%20studies~!2009-11-12~!2010-03-23~!2010-05-04~!&author=ML%20North&author=H%20Meurs&author=J%20Zaagsma&author=JA%20Scott&author=H%20Maarsingh&volume=2&issue=2&publication_year=2010&pages=20-36&doi=10.2174/1875042701002020020&)

7. North ML, Takaro TK, Diamond ML, Ellis AK. Effects of phthalates on the development and expression of allergic disease and asthma. Ann Allergy, Asthma Immunol. 2014;112(6):496–502. doi: 10.1016/j.anai.2014.03.013  [DOI](https://doi.org/10.1016/j.anai.2014.03.013) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24726194/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Allergy,%20Asthma%20Immunol&title=Effects%20of%20phthalates%20on%20the%20development%20and%20expression%20of%20allergic%20disease%20and%20asthma&author=ML%20North&author=TK%20Takaro&author=ML%20Diamond&author=AK%20Ellis&volume=112&issue=6&publication_year=2014&pages=496-502&pmid=24726194&doi=10.1016/j.anai.2014.03.013&)

8. Bush A. Pathophysiological mechanisms of asthma. Front Pediatr. 2019;7(MAR). doi: 10.3389/fped.2019.00068  [DOI](https://doi.org/10.3389/fped.2019.00068) | [PMC free article](/articles/PMC6434661/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30941334/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front%20Pediatr&title=Pathophysiological%20mechanisms%20of%20asthma&author=A%20Bush&volume=7&issue=MAR&publication_year=2019&pmid=30941334&doi=10.3389/fped.2019.00068&)

9. Centers for Disease Control and Prevention. CDC - Asthma - Most Recent Asthma Data. Most Recent Asthma Data; 2016.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=CDC%20-%20Asthma%20-%20Most%20Recent%20Asthma%20Data&publication_year=2016&)

10. Caldwell RW, Rodriguez PC, Toque HA, Priya Narayanan S, Caldwell RB. Arginase: a multifaceted enzyme important in health and disease. Physiol Rev. 2018;98(2):641–665. doi: 10.1152/physrev.00037.2016  [DOI](https://doi.org/10.1152/physrev.00037.2016) | [PMC free article](/articles/PMC5966718/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29412048/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Physiol%20Rev&title=Arginase:%20a%20multifaceted%20enzyme%20important%20in%20health%20and%20disease&author=RW%20Caldwell&author=PC%20Rodriguez&author=HA%20Toque&author=S%20Priya%20Narayanan&author=RB%20Caldwell&volume=98&issue=2&publication_year=2018&pages=641-665&pmid=29412048&doi=10.1152/physrev.00037.2016&)

11. Yang Z, Ming XF. Functions of arginase isoforms in macrophage inflammatory responses: impact on cardiovascular diseases and metabolic disorders. Front Immunol. 2014;5(OCT). doi: 10.3389/fimmu.2014.00533  [DOI](https://doi.org/10.3389/fimmu.2014.00533) | [PMC free article](/articles/PMC4209887/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25386179/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front%20Immunol&title=Functions%20of%20arginase%20isoforms%20in%20macrophage%20inflammatory%20responses:%20impact%20on%20cardiovascular%20diseases%20and%20metabolic%20disorders&author=Z%20Yang&author=XF%20Ming&volume=5&issue=OCT&publication_year=2014&pmid=25386179&doi=10.3389/fimmu.2014.00533&)

12. Benson RC, Hardy KA, Morris CR. Arginase and Arginine dysregulation in Asthma. J Allergy. 2011;2011:1–12. doi: 10.1155/2011/736319  [DOI](https://doi.org/10.1155/2011/736319) | [PMC free article](/articles/PMC3124954/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21747870/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Allergy&title=Arginase%20and%20Arginine%20dysregulation%20in%20Asthma&author=RC%20Benson&author=KA%20Hardy&author=CR%20Morris&volume=2011&publication_year=2011&pages=1-12&pmid=21747870&doi=10.1155/2011/736319&)

13. Chen WH, Chen CH, Hsu MC, Chang RW, Wang CH, Lee TS. Advances in the molecular mechanisms of statins in regulating endothelial nitric oxide bioavailability: interlocking biology between eNOS activity and L-arginine metabolism. Biomed Pharmacother. 2024;171. doi: 10.1016/j.biopha.2024.116192  [DOI](https://doi.org/10.1016/j.biopha.2024.116192) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/38262153/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biomed%20Pharmacother&title=Advances%20in%20the%20molecular%20mechanisms%20of%20statins%20in%20regulating%20endothelial%20nitric%20oxide%20bioavailability:%20interlocking%20biology%20between%20eNOS%20activity%20and%20L-arginine%20metabolism&author=WH%20Chen&author=CH%20Chen&author=MC%20Hsu&author=RW%20Chang&author=CH%20Wang&publication_year=2024&pages=171&pmid=38262153&doi=10.1016/j.biopha.2024.116192&)

14. Cloots RHE, Sankaranarayanan S, Poynter ME, et al. Arginase 1 deletion in myeloid cells affects the inflammatory response in allergic asthma, but not lung mechanics, in female mice. BMC Pulm Med. 2017;17(1). doi: 10.1186/s12890-017-0490-7  [DOI](https://doi.org/10.1186/s12890-017-0490-7) | [PMC free article](/articles/PMC5706166/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29183288/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Pulm%20Med&title=Arginase%201%20deletion%20in%20myeloid%20cells%20affects%20the%20inflammatory%20response%20in%20allergic%20asthma,%20but%20not%20lung%20mechanics,%20in%20female%20mice&author=RHE%20Cloots&author=S%20Sankaranarayanan&author=ME%20Poynter&volume=17&issue=1&publication_year=2017&pmid=29183288&doi=10.1186/s12890-017-0490-7&)

15. Donthi S, Ganugapati J, Valluri V, Macha R, Sivasai Krovvidi SR. Molecular docking studies of ephedrine, eugenol, and their derivatives as arginase inhibitors: implications in asthma. Int J Pharm Investig. 2018;8(3). doi: 10.4103/jphi.jphi_25_18  [DOI](https://doi.org/10.4103/jphi.jphi_25_18) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Pharm%20Investig&title=Molecular%20docking%20studies%20of%20ephedrine,%20eugenol,%20and%20their%20derivatives%20as%20arginase%20inhibitors:%20implications%20in%20asthma&author=S%20Donthi&author=J%20Ganugapati&author=V%20Valluri&author=R%20Macha&author=SR%20Sivasai%20Krovvidi&volume=8&issue=3&publication_year=2018&doi=10.4103/jphi.jphi_25_18&)

16. Vlad D, Albu S. Arginase isoform expression in chronic rhinosinusitis. J Clin Med. 2019;8(11):1809. doi: 10.3390/jcm8111809  [DOI](https://doi.org/10.3390/jcm8111809) | [PMC free article](/articles/PMC6912297/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31683763/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Med&title=Arginase%20isoform%20expression%20in%20chronic%20rhinosinusitis&author=D%20Vlad&author=S%20Albu&volume=8&issue=11&publication_year=2019&pages=1809&pmid=31683763&doi=10.3390/jcm8111809&)

17. Vonk JM, Postma DS, Maarsingh H, Bruinenberg M, Koppelman GH, Meurs H. Arginase 1 and arginase 2 variations associate with asthma, asthma severity and β2 agonist and steroid response. Pharmacogenet Genomics. 2010;20(3):179–186. doi: 10.1097/FPC.0b013e328336c7fd  [DOI](https://doi.org/10.1097/FPC.0b013e328336c7fd) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20124949/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Arginase%201%20and%20arginase%202%20variations%20associate%20with%20asthma,%20asthma%20severity%20and%20%CE%B22%20agonist%20and%20steroid%20response&author=JM%20Vonk&author=DS%20Postma&author=H%20Maarsingh&author=M%20Bruinenberg&author=GH%20Koppelman&volume=20&issue=3&publication_year=2010&pages=179-186&pmid=20124949&doi=10.1097/FPC.0b013e328336c7fd&)

18. Patel C, Ghanim H, Ravishankar S, et al. Expression of arginase isoforms in human blood cells and vascular tissues and their regulation by inflammation. J Clin Endocrinol Metabol. 2012;97(7):E1202–E1210. doi: 10.1210/jc.2011-3162  [DOI](https://doi.org/10.1210/jc.2011-3162) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Endocrinol%20Metabol&title=Expression%20of%20arginase%20isoforms%20in%20human%20blood%20cells%20and%20vascular%20tissues%20and%20their%20regulation%20by%20inflammation&author=C%20Patel&author=H%20Ghanim&author=S%20Ravishankar&volume=97&issue=7&publication_year=2012&pages=E1202-E1210&doi=10.1210/jc.2011-3162&)

19. Donthi S, Neela VSK, Gaddam S, et al. Association of increased risk of asthma with elevated arginase & interleukin-13 levels in serum & rs2781666 G/T genotype of arginase I. Indian J Med Res. 2018;148(2):159–168. doi: 10.4103/ijmr.IJMR_379_16  [DOI](https://doi.org/10.4103/ijmr.IJMR_379_16) | [PMC free article](/articles/PMC6206777/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30381539/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Indian%20J%20Med%20Res&title=Association%20of%20increased%20risk%20of%20asthma%20with%20elevated%20arginase%20&%20interleukin-13%20levels%20in%20serum%20&%20rs2781666%20G/T%20genotype%20of%20arginase%20I&author=S%20Donthi&author=VSK%20Neela&author=S%20Gaddam&volume=148&issue=2&publication_year=2018&pages=159-168&pmid=30381539&doi=10.4103/ijmr.IJMR_379_16&)

20. Lucena-Aguilar G, Sánchez-López AM, Barberán-Aceituno C, Carrillo-ávila JA, López-Guerrero JA, Aguilar-Quesada R. DNA source selection for downstream applications based on DNA quality indicators analysis. Biopreservation Biobanking. 2016;14:264–270. doi: 10.1089/bio.2015.0064  [DOI](https://doi.org/10.1089/bio.2015.0064) | [PMC free article](/articles/PMC4991598/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27158753/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biopreservation%20Biobanking&title=DNA%20source%20selection%20for%20downstream%20applications%20based%20on%20DNA%20quality%20indicators%20analysis&author=G%20Lucena-Aguilar&author=AM%20S%C3%A1nchez-L%C3%B3pez&author=C%20Barber%C3%A1n-Aceituno&author=JA%20Carrillo-%C3%A1vila&author=JA%20L%C3%B3pez-Guerrero&volume=14&publication_year=2016&pages=264-270&pmid=27158753&doi=10.1089/bio.2015.0064&)

21. Shatanawi A, Momani MS. Plasma arginase activity is elevated in type 2 diabetic patients. Biomed Res. 2017;28(9).  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biomed%20Res&title=Plasma%20arginase%20activity%20is%20elevated%20in%20type%202%20diabetic%20patients&author=A%20Shatanawi&author=MS%20Momani&volume=28&issue=9&publication_year=2017&)

22. Buraczynska M, Zakrocka I. Arginase gene polymorphism increases risk of diabetic retinopathy in type 2 diabetes mellitus patients. J Clin Med. 2021;10(22):5407. doi: 10.3390/jcm10225407  [DOI](https://doi.org/10.3390/jcm10225407) | [PMC free article](/articles/PMC8620112/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34830689/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Med&title=Arginase%20gene%20polymorphism%20increases%20risk%20of%20diabetic%20retinopathy%20in%20type%202%20diabetes%20mellitus%20patients&author=M%20Buraczynska&author=I%20Zakrocka&volume=10&issue=22&publication_year=2021&pages=5407&pmid=34830689&doi=10.3390/jcm10225407&)

23. Dumont J, Zureik M, Cottel D, et al. Association of arginase 1 gene polymorphisms with the risk of myocardial infarction and common carotid intima-media thickness. J Med Genet. 2007;44(8):526–531. doi: 10.1136/jmg.2006.047449  [DOI](https://doi.org/10.1136/jmg.2006.047449) | [PMC free article](/articles/PMC2597928/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17369504/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Med%20Genet&title=Association%20of%20arginase%201%20gene%20polymorphisms%20with%20the%20risk%20of%20myocardial%20infarction%20and%20common%20carotid%20intima-media%20thickness&author=J%20Dumont&author=M%20Zureik&author=D%20Cottel&volume=44&issue=8&publication_year=2007&pages=526-531&pmid=17369504&doi=10.1136/jmg.2006.047449&)

24. Shah SFA, Iqbal T, Qamar R, Rafiq MA, Hussain S. ARG1 gene polymorphisms and their association in individuals with essential hypertension: a Case–Control Study. DNA Cell Biol. 2018;37(7):609–616. doi: 10.1089/dna.2018.4222  [DOI](https://doi.org/10.1089/dna.2018.4222) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29756997/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=DNA%20Cell%20Biol&title=ARG1%20gene%20polymorphisms%20and%20their%20association%20in%20individuals%20with%20essential%20hypertension:%20a%20Case%E2%80%93Control%20Study&author=SFA%20Shah&author=T%20Iqbal&author=R%20Qamar&author=MA%20Rafiq&author=S%20Hussain&volume=37&issue=7&publication_year=2018&pages=609-616&pmid=29756997&doi=10.1089/dna.2018.4222&)

25. Meroufel D, Dumont J, Médiène-Benchekor S, et al. Characterization of arginase 1 gene polymorphisms in the Algerian population and association with blood pressure. Clinical Biochemistry. 2009;42(10–11):1178–1182. doi: 10.1016/j.clinbiochem.2009.03.004  [DOI](https://doi.org/10.1016/j.clinbiochem.2009.03.004) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19303401/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clinical%20Biochemistry&title=Characterization%20of%20arginase%201%20gene%20polymorphisms%20in%20the%20Algerian%20population%20and%20association%20with%20blood%20pressure&author=D%20Meroufel&author=J%20Dumont&author=S%20M%C3%A9di%C3%A8ne-Benchekor&volume=42&issue=10%E2%80%9311&publication_year=2009&pages=1178-1182&pmid=19303401&doi=10.1016/j.clinbiochem.2009.03.004&)

26. Sediri Y, Kallel A, Ben Ali S, et al. Association of rs2781666 G/T polymorphism of arginase I gene with myocardial infarction in Tunisian male population. Clinical Biochemistry. 2010;43(1–2):106–109. doi: 10.1016/j.clinbiochem.2009.10.058  [DOI](https://doi.org/10.1016/j.clinbiochem.2009.10.058) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19896478/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clinical%20Biochemistry&title=Association%20of%20rs2781666%20G/T%20polymorphism%20of%20arginase%20I%20gene%20with%20myocardial%20infarction%20in%20Tunisian%20male%20population&author=Y%20Sediri&author=A%20Kallel&author=S%20Ben%20Ali&volume=43&issue=1%E2%80%932&publication_year=2010&pages=106-109&pmid=19896478&doi=10.1016/j.clinbiochem.2009.10.058&)

27. Savelieva ON, Karunas AS, Fedorova YY, et al. The role of polymorphic variants of arginase genes (ARG1, ARG2) involved in beta-2-agonist metabolism in the development and course of asthma. Vavilovskii Zhurnal Genet Selektsii. 2020;24(4):391–398. doi: 10.18699/VJ20.631  [DOI](https://doi.org/10.18699/VJ20.631) | [PMC free article](/articles/PMC7716535/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33659822/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Vavilovskii%20Zhurnal%20Genet%20Selektsii&title=The%20role%20of%20polymorphic%20variants%20of%20arginase%20genes%20(ARG1,%20ARG2)%20involved%20in%20beta-2-agonist%20metabolism%20in%20the%20development%20and%20course%20of%20asthma&author=ON%20Savelieva&author=AS%20Karunas&author=YY%20Fedorova&volume=24&issue=4&publication_year=2020&pages=391-398&pmid=33659822&doi=10.18699/VJ20.631&)

28. Shah SFA, Akram S, Iqbal T, Nawaz S, Rafiq MA, Hussain S. Association analysis between ARG1 gene polymorphisms and idiopathic dilated cardiomyopathy. Medicine. 2019;98(47):e17694. doi: 10.1097/MD.0000000000017694  [DOI](https://doi.org/10.1097/MD.0000000000017694) | [PMC free article](/articles/PMC6882636/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31764771/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Medicine&title=Association%20analysis%20between%20ARG1%20gene%20polymorphisms%20and%20idiopathic%20dilated%20cardiomyopathy&author=SFA%20Shah&author=S%20Akram&author=T%20Iqbal&author=S%20Nawaz&author=MA%20Rafiq&volume=98&issue=47&publication_year=2019&pages=e17694&pmid=31764771&doi=10.1097/MD.0000000000017694&)

29. Hansen KE, Johnson RE, Chambers KR, et al. Treatment of Vitamin D Insufficiency in postmenopausal women: a randomized clinical trial. JAMA Intern Med. 2015;175(10):1612–1621. doi: 10.1001/jamainternmed.2015.3874  [DOI](https://doi.org/10.1001/jamainternmed.2015.3874) | [PMC free article](/articles/PMC4594209/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26237520/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA%20Intern%20Med&title=Treatment%20of%20Vitamin%20D%20Insufficiency%20in%20postmenopausal%20women:%20a%20randomized%20clinical%20trial&author=KE%20Hansen&author=RE%20Johnson&author=KR%20Chambers&volume=175&issue=10&publication_year=2015&pages=1612-1621&pmid=26237520&doi=10.1001/jamainternmed.2015.3874&)

30. Akinbami LJ, Fryar CD. Current Asthma prevalence by weight status among adults. NCHS Data Brief. 2016;239.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/27019018/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=NCHS%20Data%20Brief&title=Current%20Asthma%20prevalence%20by%20weight%20status%20among%20adults&author=LJ%20Akinbami&author=CD%20Fryar&volume=239&publication_year=2016&pmid=27019018&)

31. Morris CR, Poljakovic M, Lavrisha L, et al. Decreased arginine bioavailability and increased serum arginase activity in asthma. Am J Respir Crit Care Med. 2004;170(2):148–153. doi: 10.1164/rccm.200309-1304oc  [DOI](https://doi.org/10.1164/rccm.200309-1304oc) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15070820/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Respir%20Crit%20Care%20Med&title=Decreased%20arginine%20bioavailability%20and%20increased%20serum%20arginase%20activity%20in%20asthma&author=CR%20Morris&author=M%20Poljakovic&author=L%20Lavrisha&volume=170&issue=2&publication_year=2004&pages=148-153&pmid=15070820&doi=10.1164/rccm.200309-1304oc&)

32. Ogino K, Obase Y, Takahashi N, et al. High serum arginase I levels in asthma: its correlation with high-sensitivity C-reactive protein. J Asthma. 2011;48(1):1–7. doi: 10.3109/02770903.2010.528496  [DOI](https://doi.org/10.3109/02770903.2010.528496) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21039185/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Asthma&title=High%20serum%20arginase%20I%20levels%20in%20asthma:%20its%20correlation%20with%20high-sensitivity%20C-reactive%20protein&author=K%20Ogino&author=Y%20Obase&author=N%20Takahashi&volume=48&issue=1&publication_year=2011&pages=1-7&pmid=21039185&doi=10.3109/02770903.2010.528496&)
