# Role of Single-Nucleotide Polymorphisms in Genes Implicated in Capecitabine Pharmacodynamics on the Effectiveness of Adjuvant Therapy in Colorectal Cancer

## Metadata
**Authors:** Yasmin Cura, Almudena Sánchez-Martín, Noelia Márquez-Pete, Encarnación González-Flores, Fernando Martínez-Martínez, Cristina Pérez-Ramírez, Alberto Jiménez-Morales
**Journal:** International Journal of Molecular Sciences
**Date:** 2023 Dec 20
**DOI:** [10.3390/ijms25010104](https://doi.org/10.3390/ijms25010104)
**PMID:** 38203276
**PMCID:** PMC10778960
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10778960/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC10778960/pdf/ijms-25-00104.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC10778960/pdf/ijms-25-00104.pdf)

## Abstract

Colorectal cancer (CRC) is a highly prevalent form of neoplasm worldwide. Capecitabine, an oral antimetabolite, is widely used for CRC treatment; however, there exists substantial variation in individual therapy response. This may be due to genetic variations in genes involved in capecitabine pharmacodynamics (PD). In this study, we investigated the role of single-nucleotide polymorphisms (SNPs) related to capecitabine’s PD on disease-free survival (DFS) in CRC patients under adjuvant treatment. Thirteen SNPs in the TYMS, ENOSF1, MTHFR, ERCC1/2, and XRCC1/3 genes were genotyped in 142 CRC patients using real-time PCR with predesigned TaqMan® probes. A significant association was found between favorable DFS and the ENOSF1 rs2612091-T allele (p = 0.010; HR = 0.34; 95% CI = 0.14–0.83), as well as with the TYMS/ENOSF1 region ACT haplotype (p = 0.012; HR = 0.37; 95% CI = 0.17–0.80). Other factors such as low histological grade (p = 0.009; HR = 0.34; 95% CI = 0.14–0.79) and a family history of cancer (p = 0.040; HR = 0.48; 95% CI = 0.23–0.99) were also linked to improved DFS. Therefore, the SNP ENOSF1 rs2612091 could be considered as a predictive genetic biomarker for survival in CRC patients receiving capecitabine-based adjuvant regimens.

Keywords: colorectal cancer, capecitabine, pharmacodynamics, single nucleotide polymorphisms, survival

## 1. Introduction

Colorectal cancer (CRC) is one of the most frequently diagnosed neoplasms and is considered one of the leading causes of cancer-related morbidity and mortality worldwide [[1](#B1-ijms-25-00104)]. In the United States, CRC has a significant impact, affecting approximately 135,439 new patients annually and standing as the second leading cause of cancer-related mortality, with approximately 50,260 deaths each year [[2](#B2-ijms-25-00104),[3](#B3-ijms-25-00104)]. Adenocarcinomas (ADC) constitute the majority of CRC, representing 90% of all cases, while other histological types, such as adenosquamous, spindle, squamous, and undifferentiated, are less frequently observed [[2](#B2-ijms-25-00104),[3](#B3-ijms-25-00104)]. The approach to CRC treatment is comprehensive and individualized based on patient and tumor characteristics. Therapeutic strategies encompass both local and systemic approaches. Local strategies involve surgical resection or endoscopic resection, the latter being particularly employed for favorable-risk and early-stage disease when identified in polyps. Systemic approaches include chemotherapy, targeted therapy, and immunotherapy. Chemotherapy encompasses various drug options, such as fluoropyrimidines (FP), which include 5-fluorouracil (5-FU), capecitabine, S-1, tegafur plus uracil, and other antineoplastic agents like irinotecan, trifluridine/tipiracil, and oxaliplatin. The administration of chemotherapy can be tailored to different settings, including neoadjuvant, adjuvant, metastatic, or palliative, aligning with the specific needs of each patient [[2](#B2-ijms-25-00104),[3](#B3-ijms-25-00104)]. While surgery stands as the standard treatment for patients with localized-stage CRC, those in advanced stages who are eligible for surgery typically undergo adjuvant chemotherapy or radiotherapy [[2](#B2-ijms-25-00104),[3](#B3-ijms-25-00104)]. Capecitabine, an orally administered prodrug of the antimetabolite agent 5-FU belonging to the FP family, is a standard element within adjuvant chemotherapy regimens used in CRC therapy [[4](#B4-ijms-25-00104)]. Nevertheless, high interindividual variability in treatment response is observed among patients with CRC who are treated with capecitabine-based regimens [[5](#B5-ijms-25-00104)]. Pharmacogenetics has emerged as a promising field to elucidate genetic determinants that contribute to interpatient variation in drug response [[6](#B6-ijms-25-00104)]. In fact, genotyping of variants in the dihydropyrimidine dehydrogenase (*DPYD*) gene before initiating FP therapy has become a standard practice in numerous hospitals and is considered a valuable tool for predicting severe FP toxicity. Nonetheless, it is important to note that, currently, there are no established genetic biomarkers for predicting capecitabine-based therapy effectiveness.

Single-nucleotide polymorphisms (SNPs) in genes encoding proteins involved in the pharmacodynamics (PD) pathway of capecitabine have the potential to influence treatment outcomes and drug survival [[7](#B7-ijms-25-00104)]. As a prodrug, capecitabine undergoes enzymatic bioactivation to its active form 5-FU. The active drug is subsequently metabolized into three pharmacologically active metabolites: fluorodeoxyuridine monophosphate (FdUMP), fluorouridine triphosphate (FUTP), and fluorodeoxyuridine triphosphate (FdUTP). FdUMP acts as an inhibitor of thymidylate synthase (TS), an enzyme encoded by the thymidylate synthase gene (*TYMS*), which plays a crucial role in pyrimidine and DNA synthesis. Consequently, this inhibition disrupts the conversion of 5,10-methylenetetrahydrofolate (5,10-MTHF) to dihydrofolate, a vital component of the folate cycle. Furthermore, the metabolites FUTP and FdUTP directly incorporate into RNA and DNA, respectively, leading to direct genetic damage and subsequent cell death. Therefore, apart from *TYMS*, the genes involved in the folate cycle and DNA repair pathways play an important role in the PD of capecitabine ([Figure 1](#ijms-25-00104-f001)) [[8](#B8-ijms-25-00104)]. Indeed, numerous studies have investigated the impact of several SNPs in genes related to capecitabine’s PD, such as *TYMS*, enolase superfamily member 1 gene (*ENOSF1*), methylenetetrahydrofolate reductase gene (*MTHFR*), ERCC excision repair 1, endonuclease non-catalytic subunit gene (*ERCC1*), ERCC excision repair 2, TFIIH core complex helicase subunit gene (*ERCC2*), X-ray repair cross-complementing 1 gene (*XRCC1*), and X-ray repair cross-complementing 3 gene (*XRCC3*) on the effectiveness of FP therapy in gastrointestinal neoplasms [[9](#B9-ijms-25-00104),[10](#B10-ijms-25-00104),[11](#B11-ijms-25-00104),[12](#B12-ijms-25-00104),[13](#B13-ijms-25-00104),[14](#B14-ijms-25-00104),[15](#B15-ijms-25-00104),[16](#B16-ijms-25-00104),[17](#B17-ijms-25-00104),[18](#B18-ijms-25-00104)]. However, the available evidence remains limited and characterized by conflicting findings, necessitating further investigation and exploration.

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e6f1/10778960/98ba1da1e5d8/ijms-25-00104-g001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=10778960_ijms-25-00104-g001.jpg)

Capecitabine pharmacodynamics pathway. Created with BioRender.com. 5-FU: fluorouracil; CRC: colorectal cancer; dTMP: 2′-deoxythymidine-5′-monophosphate; dUMP: 2′-deoxyridine-5′-monophosphate; DHFR: dihydrofolate reductase; ERCC1: ERCC excision repair 1, endonuclease non-catalytic subunit; ERCC2: ERCC excision repair 2, TFIIH core complex helicase subunit; ENOSF1: enolase superfamily member 1; FdUMP: fluorodeoxyuridine monophosphate; FdUTP: fluorodeoxyuridine triphosphate; FPGS: folylpolyglutamate synthase; FUTP: fluorouridine triphosphate; GGH: gamma-glutamyl hydrolase; MTHFD1: methylenetetrahydrofolate dehydrogenase; MTHFR: methylenetetrahydrofolate reductase; SHMT1: serine hydroxy methyltransferase 1; SMUG1: single-strand-selective monofunctional uracil-DNA glycosylase 1; SNP: single-nucleotide polymorphism; TDG: thymine DNA glycosylase; TP53: tumor protein P53; TYMS: thymidylate synthetase; XRCC1: X-ray repair cross-complementing 1; XRCC3: X-ray repair cross-complementing 3.

Evaluating the impact of SNPs in genes associated with the mechanism of action of capecitabine on drug survival is critical for optimizing future therapeutic strategies and improving clinical outcomes. This study aims to assess the influence of SNPs in genes involved in capecitabine’s PD pathway on disease-free survival (DFS) in CRC patients.

## 2. Results

### 2.1. Patient Characteristics

Sociodemographic and clinical characteristics of the study population (*n* = 142) are presented in [Table 1](#ijms-25-00104-t001). Median age at diagnosis was 65.00 [57.00–73.00] years. The majority of patients were men (62.68%; 89/142), had family history of cancer (59.86%; 85/142), and optimal performance status (75.35%; 107/142). All patients had European ancestry (100%; 142). Median tumor size was 4.30 [3.00–6.00] cm. Predominantly, patients had primary tumors located in the colon (57.04%; 81/142), low histological grades (88.73%; 126/142), and advanced stages of cancer at diagnosis (88.03%; 125/142). Approximately half of the patients received capecitabine in combination regimens (54.23%; 77/142). Median duration of treatment was 5.43 [5.83–4.18] months. Patients were followed up for a median period of 13.80 [7.28–22.05] months.

### Table 1.

| Characteristic | n (%) | Median [p25–p75] |
| --- | --- | --- |
| Sex |   |   |
| Female | 53 (37.32) | - |
| Male | 89 (62.68) | - |
| Age at CRC diagnosis (years) | - | 65.00 [57.00–73.00] |
| Family history of cancer |   |   |
| Yes | 85 (59.86) | - |
| No | 57 (40.14) | - |
| Performance status |   |   |
| 0 | 107 (75.35) | - |
| 1 | 28 (19.72) | - |
| 2 | 7 (4.93) | - |
| Primary tumor site |   |   |
| Colon | 81 (57.04) | - |
| Rectum | 61 (42.96) | - |
| Primary tumor size (cm) | - | 4.30 [3.00–6.00] |
| Stage |   |   |
| 0-IIC | 17 (11.97) | - |
| IIIA-IV | 125 (88.03) | - |
| Grade |   |   |
| Low | 126 (88.73) | - |
| High | 16 (11.27) | - |
| Capecitabine-based regimen |   |   |
| Monotherapy | 65 (45.77) | - |
| Combination | 77 (54.23) | - |

Table 1 Caption: Sociodemographic and clinical characteristics of 142 CRC patients.

### 2.2. Genotype Frequency and Distribution

Genotyping was successful for the 13 selected SNPs. Minor allele frequency (MAF) was >1% for all analyzed SNPs ([Table S1](#app1-ijms-25-00104)). Additionally, except for *TYMS*/*ENOSF1* rs2790 (*p* = 0.033), all the SNPs examined adhered to Hardy Weinberg equilibrium (HWE) (*p* > 0.05) ([Table S2](#app1-ijms-25-00104)). Nonetheless, no statistically significant differences were detected when comparing the observed *TYMS*/*ENOSF1* rs2790-G allele frequency with that reported for the Iberian population in Spain (IBS) at 0.274 vs. 0.248, *p* = 0.498 ([Table S3](#app1-ijms-25-00104)). None of the MAF of the SNPs included in the study population showed significant differences with the IBS (*p* > 0.050; [Table S3](#app1-ijms-25-00104)). Linkage disequilibrium (LD) analysis revealed that the SNPs *TYMS*/*ENOSF1* rs69957–*ENOSF1* rs2612091, *MTHFR* rs1801131–1801133, *ERCC2* rs13181–rs1799787, and *ERCC1* rs3212986–rs11615 were in strong LD (D’ > 0.900) ([Figure 2](#ijms-25-00104-f002); [Table S4](#app1-ijms-25-00104)). [Tables S5–S8](#app1-ijms-25-00104) present the frequency estimates of haplotypes in the *TYMS*/*ENOSF1* gene region and in the genes *MTHFR*, *ERCC2*, and *ERCC1.*

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e6f1/10778960/508270b04e4b/ijms-25-00104-g002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=10778960_ijms-25-00104-g002.jpg)

Linkage disequilibrium of selected SNPs. Color scheme: bright red: D’ = 1, LOD ≥ 2, shades of pink/red: D’ < 1, LOD ≥ 2; white: D’ < 1, LOD < 2.

### 2.3. Influence of Non-Genetic Factors on Disease-Free Survival

Patients with low histological tumor grade (*p* = 0.009; HR = 0.34; 95% CI = 0.14–0.79; [Figure 3](#ijms-25-00104-f003)) and a positive family history of cancer (*p* = 0.040; HR = 0.48; 95% CI = 0.23–0.99; [Figure 4](#ijms-25-00104-f004)) exhibited a significantly higher DFS ([Table S9](#app1-ijms-25-00104)). There was a trend towards increased DFS in patients with primary tumor located in the rectum (*p* = 0.080; HR = 0.50; 95% CI = 0.23–1.10) ([Figure S1](#app1-ijms-25-00104); [Table S9](#app1-ijms-25-00104)). In contrast, a trend of association between larger tumor size and shorter DFS was observed (*p* = 0.080; HR = 1.14; 95% CI = 0.97–1.32) ([Table S9](#app1-ijms-25-00104)). No association was found between DFS, and the rest of the sociodemographic and clinical variables collected in the study ([Figures S2–S5](#app1-ijms-25-00104); [Table S9](#app1-ijms-25-00104)).

### Figure 3.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e6f1/10778960/4d3455dac4cb/ijms-25-00104-g003.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=10778960_ijms-25-00104-g003.jpg)

Kaplan–Meier curves for the association of DFS with histological grade.

### Figure 4.

![Figure 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e6f1/10778960/998416c3bf1b/ijms-25-00104-g004.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=10778960_ijms-25-00104-g004.jpg)

Kaplan–Meier curves for the association of DFS with family history of cancer.

### 2.4. Influence of SNPs in Capecitabine Pharmacodynamics on Disease-Free Survival

A significant association was found between the *ENOSF1* rs2612091-TT genotype and increased DFS (*p* = 0.010; HR = 0.34; 95% CI = 0.14–0.83, for C vs. TT, and *p* = 0.040; HR = 0.31; 95% CI = 0.10–0.90, for CC vs. TT) ([Table S10](#app1-ijms-25-00104)). The Kaplan–Meier survival curves illustrating the associations of DFS with the *ENOSF1* rs2612091 SNP are presented in [Figure 5](#ijms-25-00104-f005) and [Figure 6](#ijms-25-00104-f006). In the bivariate Cox model, an association was observed between the *MTHFR* rs1801131-T allele and decreased DFS, as all events occurred in patients carrying this allele (*p* = 0.040) ([Table S10](#app1-ijms-25-00104)). No association was found between DFS and the remaining studied SNPs ([Figures S6–S17](#app1-ijms-25-00104); [Table S10](#app1-ijms-25-00104)).

### Figure 5.

![Figure 5](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e6f1/10778960/9923b1ff849b/ijms-25-00104-g005.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=10778960_ijms-25-00104-g005.jpg)

Kaplan–Meier curves for the association of DFS with ENOSF1 rs2612091-TT (Allele C vs. genotype TT).

### Figure 6.

![Figure 6](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e6f1/10778960/c3da8d314bcf/ijms-25-00104-g006.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=10778960_ijms-25-00104-g006.jpg)

Kaplan–Meier curves for the association of DFS with ENOSF1 rs2612091-TT (Genotype CT vs. TT).

Haplotype analysis showed an association between haplotype in *TYMS*/*ENOSF1* region (rs2790, rs699517, and rs2612091) ACT and increased DFS (*p* = 0.012; HR = 0.37; 95% CI = 0.17–0.80; for ACC vs. ACT) ([Table 2](#ijms-25-00104-t002)). No significant association was found between the remaining studied haplotypes and DFS ([Tables S11–S13](#app1-ijms-25-00104)).

### Table 2.

| H | TYMS/ENOSF1rs2790 | TYMS/ENOSF1rs699517 | ENOSF1 rs2612091 | Frequency | HR (95% CI) | p-Value |
| --- | --- | --- | --- | --- | --- | --- |
| 0 | A | C | C | 0.3830 | 1.00 | - |
| 1 | G | T | T | 0.2670 | 0.58 (0.31–1.07) | 0.083 |
| 2 | A | C | T | 0.2150 | 0.37 (0.17–0.80) | 0.012 |
| 3 | A | T | T | 0.1270 | 0.89 (0.36–2.15) | 0.796 |
| 4 | G | C | C | 0.0040 | - | - |
| 5 | G | C | T | 0.0040 | - | - |
| Goodness-of-fit: −2 × Log-likelihood (with covariates) = 278.098421 |  |  |  |  |  |  |
| Goodness-of-fit: −2 × Log-likelihood (without covariates) = 285.795061 |  |  |  |  |  |  |
| df = 3; model p-value: 0.052 |  |  |  |  |  |  |

Table 2 Caption: Effect estimation of haplotypes in TYMS/ENOSF1 gene region on disease-free survival.

The multivariate analysis revealed significant associations between higher DFS and positive family history of cancer (*p* = 0.017; HR = 0.41; 95% CI = 0.20–0.85), low tumor grade at diagnosis (*p* = 0.004; HR = 0.29; 95% CI = 0.12–0.68), as well as the TT genotype of the *ENOSF1* rs2612091 SNP (*p* = 0.009; HR = 0.30; 95% CI = 0.12–0.74). All associations remained significant after multiple comparison correction. The multivariate analysis of the association of DFS with non-genetic variables and the studied SNPs is presented in [Table 3](#ijms-25-00104-t003).

### Table 3.

| ENOSF1 rs2612091 (TT)Family history of cancer (Yes)Grade (low) | Disease-Free Survival |  |  |
| --- | --- | --- | --- |
| HR (95% CI) | p-value | p-BH * |  |
| 0.30 (0.12–0.74)0.41 (0.20–0.85)0.29 (0.12–0.68) | 0.0090.0170.004 | 0.0130.0170.012 |  |
| Model p-value < 0.001 |   |   |   |

Table 3 Caption: Multivariate analysis of disease-free survival associations with sociodemographic, clinical, and genetic variables.

## 3. Discussion

Investigating the influence of genetic variations on treatment efficacy holds significant scientific value. The absence of validated genetic biomarkers for predicting survival in FP-based treatment strategies highlights the importance of further research in this area. Our study investigated the role of SNPs related to capecitabine PD in treatment survival. Our findings revealed that the *ENOSF1* rs2612091-TT genotype and the *TYMS*/*ENOSF1* ACT haplotype were associated with longer DFS in CRC patients undergoing adjuvant capecitabine-based therapy. Furthermore, family history of cancer and low histological grade were identified as potential factors contributing to improved DFS. Understanding the role of these genetic factors could bear significant implications for personalized medicine, enabling clinicians to tailor capecitabine treatments based on individual genetic profiles and potentially leading to improved treatment survival rates.

A positive family history of cancer is a well-known risk factor for developing the disease, but its influence on prognosis remains uncertain [[19](#B19-ijms-25-00104)]. Our results suggest a significant association between positive family history of cancer and improved DFS. These findings are in line with previous studies conducted in patient cohorts with colon, gastric, and prostate cancer [[19](#B19-ijms-25-00104),[20](#B20-ijms-25-00104),[21](#B21-ijms-25-00104)]. Patients with family history of cancer tend to demonstrate (a) heightened awareness and vigilance regarding the condition, which leads to early detection and initiation of treatment, potentially improving survival outcomes; and (b) proactive behavior in maintaining a healthy lifestyle and actively participating in treatment. This behavior contributes to enhanced disease management and treatment adherence, consequently yielding improved response and prognosis [[19](#B19-ijms-25-00104),[20](#B20-ijms-25-00104),[21](#B21-ijms-25-00104)].

Histological grading is a routinely used prognostic marker in CRC [[22](#B22-ijms-25-00104),[23](#B23-ijms-25-00104)]. Specifically, a high grade is recognized as a negative prognostic factor [[22](#B22-ijms-25-00104),[24](#B24-ijms-25-00104)]. This statement correlates with what was found in our study, where a low histological grade was associated with better survival outcomes among the CRC patients investigated. Existing evidence indicates that low-grade tumors exhibit less aggressive and invasive behavior than high-grade tumors and improved survival outcomes [[22](#B22-ijms-25-00104),[25](#B25-ijms-25-00104),[26](#B26-ijms-25-00104),[27](#B27-ijms-25-00104),[28](#B28-ijms-25-00104),[29](#B29-ijms-25-00104),[30](#B30-ijms-25-00104)]. However, it is important to mention that other factors, such as tumor stage, patient age, and general health status, have also been recognized as factors that may influence disease prognosis [[23](#B23-ijms-25-00104),[24](#B24-ijms-25-00104),[31](#B31-ijms-25-00104)].

The *ENOSF1* gene, responsible for encoding the protein mitochondrial enolase superfamily member 1 (ENOF1), exhibits three distinct isoforms. One of these isoforms has been identified as an l-fuconate dehydratase, actively participating in the catabolism of L-fucose, a sugar integral to the carbohydrate composition of cellular glycoproteins. ENOF1 catalyzes the dehydration of L-fuconate to 2-keto-3-deoxy-L-fuconate by extracting the 2-proton, generating an enediolate intermediate stabilized by the presence of magnesium ion [[32](#B32-ijms-25-00104),[33](#B33-ijms-25-00104)]. Although the role of ENOF1 in the mechanism of action of FP is not fully understood, the *ENOSF1* gene has been extensively studied for its influence on the toxicity and effectiveness of FP-based treatments [[18](#B18-ijms-25-00104),[32](#B32-ijms-25-00104),[34](#B34-ijms-25-00104),[35](#B35-ijms-25-00104)]. The rationale for investigating this gene is based on its location in the genome and its potential effects on proteins that are formally related to the PD of FP. The *ENOSF1* gene partially overlaps with the *TYMS* gene (in the *TYMS*/*ENOSF1* region) on chromosome 18 and is transcribed in the opposite direction to the latter [[36](#B36-ijms-25-00104)]. It has been suggested that *ENOSF1* is involved in the regulation of the expression of *TYMS*, thus affecting the outcomes of FP-based therapy [[37](#B37-ijms-25-00104)]. Moreover, the ENOF1 enzyme has been indicated to be more sensitive to the cytotoxic effects of FP than TS [[38](#B38-ijms-25-00104)]. In-depth analysis of protein–protein interactions through the STRING interactomics platform reveals significant insights into the relationship between TS and ENOF1. Despite ENOF1 appearing to interact with approximately 10 proteins (TS, CLUL1, NPL, and DPH5, among others), it exhibits a high-confidence association score (>0.7) solely with TS (0.978). This substantial score is primarily attributed to text mining (0.957), presenting a low score in the co-expression type of association (0.075). Hence, TS emerges as the sole protein involved in this context, although not through direct physical interaction. This observation underscores the need for further exploration in understanding the functional details underlying *ENOSF1* [[39](#B39-ijms-25-00104)]. The *ENOSF1* rs2612091 variant (g.683607C>T), located in the intronic region of the gene, has been associated with *ENOSF1* mRNA expression and FP-therapy outcomes [[18](#B18-ijms-25-00104),[32](#B32-ijms-25-00104),[34](#B34-ijms-25-00104),[35](#B35-ijms-25-00104)]. In our study, we observed that patients carrying genotype *ENOSF1* rs2612091-TT exhibited superior DFS compared to C allele carriers. These findings align with the results reported by Meulendijks et al. (Caucasian population; The Netherlands; n = 185) in a pooled analysis of three prospective studies that involved gastric cancer patients undergoing capecitabine-based regimens. They observed a significant association between the *ENOSF1* rs2612091-CC genotype and lower overall survival (OS) (*p* = 0.041; HR = 1.50; 95% CI = 1.00–2.30, for TT/CT vs. CC) after adjusting for clinical covariates and the *TYMS* variable number tandem repeats (VNTR) variant (in moderate LD with *ENOSF1* rs2612091). Although, it is important to mention that the VNTR variant exhibited only a mild trend toward an association with OS (*p* = 0.076). Subgroup analysis based on disease extension revealed that the impact of *ENOSF1* rs2612091 was more pronounced in patients with locally advanced disease. In this context, patients carrying the TT genotype demonstrated notably enhanced OS (*p* = 0.001; HR = 6.5; 95% CI = 2.10–20.00, for CC vs. TT or CT) as well as progression-free survival (PFS) (*p* = 0.005; HR = 4.3, for CC vs. TT or CT) [[35](#B35-ijms-25-00104)]. However, our findings contrasted with those reported by Arjmandi et al. (Middle Eastern population; Iran; n = 97) in a study involving gastric cancer patients treated with neoadjuvant 5-FU-based therapy. The researchers reported that the *ENOSF1* rs2612091-CC genotype was linked to an increased response to 5-FU compared to the T allele (*p* = 0.017) [[18](#B18-ijms-25-00104)]. These contradictory findings may be explained by differences in the ethnicities of the populations studied, the type of therapeutic regimen under investigation, and the sample sizes of the studies. As there are no further studies exploring the impact of the *ENOSF1* rs2612091 SNP, the disparities in these reported findings underscore the need to assess this discovery within larger prospective cohorts to establish its authentic association.

While our study has provided valuable insights into the influence of SNPs in capecitabine’s PD on therapy effectiveness and has revealed the potential relevance of the *TYMS*/*ENOSF1* genetic region on DFS, it is important to recognize specific limitations that might affect the interpretation of the results. Firstly, the relatively small sample size in our study may limit the ability to detect significant differences in DFS. Secondly, the ambispective nature of the study could lead to differences in follow-up duration between retrospective and prospective cohorts, potentially influencing the estimation of survival rates. Thirdly, the exclusive inclusion of patients with European ancestry may impact the generalizability of our findings. Lastly, the median follow-up time might be considered relatively short, possibly leading to the underrepresentation of long-term events.

In summary, the findings of our study reveal a significant association between *ENOSF1* rs2612091-CC genotype ACT haplotype in the *TYMS*/*ENOSF1* gene region with DFS in CRC patients under capecitabine-based adjuvant therapy, suggesting their potential as promising genetic biomarkers of treatment prognosis. However, further research with prospective designs, longer follow-up times to consider their impact on OS, and larger sample sizes are necessary to validate and strengthen these results, ensuring their clinical applicability.

## 4. Materials and Methods

### 4.1. Design and Settings

The research was designed as an observational ambispective study. DNA samples from CRC patients treated at the University Hospital Virgen de las Nieves, Granada, Spain, who had been previously genotyped for *DPYD* variants, were requested to the Andalusian Public Health System biobank and stored at −40 °C. The study was conducted according to the principles of the Declaration of Helsinki and approved by the Biomedical Research Ethics Committee of Granada (Identification code 0632-M2-20, July 2020). The DNA samples included had written informed consent for donation to the Biobank of the Andalusian Public Health System and were identified by alphanumeric codes.

### 4.2. Study Population

The DNA samples of CRC patients were included in the study when meeting eligibility criteria. The inclusion criteria encompassed patients with advanced CRC who were candidates for capecitabine-based adjuvant therapy, aged ≥18 years, and possessed a performance status ≤ 2. Patients with previous malignancies, without available medical records, missing data, lost during follow-up, with abnormal liver or renal function, treated for less than 12 weeks or less than half of the prescribed cycles, and DNA samples with low quality were excluded from the study. The number of CRC patients with DNA samples available during the study recruitment period and who met selection criteria determined the sample size. From 2019 to 2022, 355 CRC patients underwent genotyping for *DPYD* variants. Among them, 198 individuals met the inclusion criteria, while 56 were eliminated due to exclusion criteria, resulting in a final cohort of 142 patients included in the study ([Figure 7](#ijms-25-00104-f007)).

### Figure 7.

![Figure 7](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e6f1/10778960/754263bf2172/ijms-25-00104-g007.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=10778960_ijms-25-00104-g007.jpg)

Patient flow diagram.

Capecitabine (750–2150 mg, twice daily) was administered for 14 days in 3-week cycles, either as monotherapy or in combination with other antineoplastic strategies, such as radiotherapy, oxaliplatin, or immunotherapy. Patient follow-up was performed by the medical oncology department before each new treatment cycle, every 3–6 months after treatment completion in the first year, and once a year thereafter. Follow-up visits were documented in patients’ medical records. Treatment continued until completion of scheduled cycles, disease progression, death, or occurrence of unacceptable toxicity.

### 4.3. Variables

#### 4.3.1. Endpoints

The primary endpoint of this study was DFS, defined as the time from the initiation of adjuvant therapy to disease relapse or death from any cause. Patients who were alive and without relapse at the end of follow-up were censored. Survival data were obtained from the medical records during each follow-up examination.

#### 4.3.2. Sociodemographic and Clinical Data

Sociodemographic and clinical characteristics of the patients included, such as age at CRC diagnosis, sex, primary tumor site, cancer stage (categorized as 0-IIC/IIIA-IV), histological grade (categorized as high/low), performance status, family history of cancer, and type of capecitabine-based adjuvant regimen (categorized as monotherapy/combination), were obtained from medical records.

#### 4.3.3. Genotyping Data

Samples containing 50 uL DNA were quantified using a NanoDrop 2000 UV spectrophotometer (Thermo Fisher Scientific, Waltham, MA, USA). A total of 13 SNPs in 7 genes involved in capecitabine PD were selected based on their prevalence in the European population and previous research in the scientific literature [[9](#B9-ijms-25-00104),[10](#B10-ijms-25-00104),[11](#B11-ijms-25-00104),[12](#B12-ijms-25-00104),[13](#B13-ijms-25-00104),[14](#B14-ijms-25-00104),[15](#B15-ijms-25-00104),[16](#B16-ijms-25-00104),[17](#B17-ijms-25-00104),[18](#B18-ijms-25-00104),[40](#B40-ijms-25-00104)] ([Table 4](#ijms-25-00104-t004)).

| Gene | Location * | SNP rs | SNP Type | TaqMan Assay ID |
| --- | --- | --- | --- | --- |
| TYMS | Chr. 18, g.657352T>C | rs2853741 | 2KB upstream variant | C__26612342_10 |
| TYMS-ENOSF1 | Chr. 18, g.673086A>G | rs2790 | 3 primer UTR variant/Intron variant | C___7486263_10 |
| TYMS-ENOSF1 | Chr. 18, g.673016C>T | rs699517 | 3 primer UTR variant/Non-coding transcript variant | C___7486269_10 |
| ENOSF1 | Chr. 18, g.683607C>T | rs2612091 | Intron variant | C__15908768_10 |
| MTHFR | Chr. 1, g.11794419T>G | rs1801131 | Missense variant (p.Glu469Ala) | C____850486_20 |
| MTHFR | Chr. 1, g.11796321G>A | rs1801133 | Missense variant (p.Ala262Val) | C_1202883_20 |
| ERCC1 | Chr. 19, g.45420395A>G | rs11615 | Synonymous variant (p.Asn118=) | C__2532959_10 |
| ERCC1 | Chr. 19, g.45409478C>A | rs3212986 | 3 primer UTR variant | C__2532948_10 |
| ERCC2 | Chr. 19, g.45352886G>A | rs1799787 | Intron variant | C__11973901_10 |
| ERCC2 | Chr. 19, g.45351661T>G | rs13181 | Missense variant (p.Lys751Gln) | C__3145033_10 |
| ERCC2 | Chr. 19, g.45364001C>T | rs1799793 | Missense variant (p.Asp312Asn) | C__3145050_10 |
| XRCC1 | Chr. 19, g.43551574T>C | rs25487 | Missense variant (p.Gln399Arg) | C__622564_10 |
| XRCC3 | Chr. 14, g.103699416G>A | rs861539 | Missense variant (p.Thr241Met) | C__8901525_10 |

Table Caption: Selected SNPs in capecitabine pharmacodynamic pathway.

Genotyping was performed in duplicate on the QuantStudio 3 Real Time PCR System using TaqMan probes (Thermo Fisher Scientific, Waltham, MA, USA), according to the manufacturer protocol, in the pharmacogenetics unit laboratory of HUVN. The reactions were arranged in a 96-well plate (0.2 mL), where each well comprised the DNA sample, TaqMan genotyping master mix, and the relevant TaqMan assay designed for the targeted SNP. Negative controls and known genotypic controls were included in each PCR run. The procedure commenced with a pre-read step at 60 °C for 30 s. Following this, an initial denaturation and enzyme activation step was performed at 95 °C for 10 min. The subsequent denaturation phase occurred at 95 °C for 15 s, repeated across 40 cycles. This denaturation step was followed by an annealing and extension phase at 60 °C for 60 s during each cycle. The post-read step concluded the cycle at 60 °C for 30 s. Real-time PCR data were analyzed using the QuantStudio 3/5 Real-Time PCR Software v.1.5.1. The genotyping results were interpreted based on the fluorescence signals and allelic discrimination plots generated during the amplification process. Call rates were >98%. Additionally, 10% of the samples were analyzed in duplicate with a concordance of 100%.

### 4.4. Statistical Analysis

Normality was verified by Kolmogorov–Smirnov test. Quantitative data were tabulated as median [p25–p75] and qualitative data were expressed as frequencies and percentages. Genotype frequencies of all the selected SNPs were assessed for HWE. MAF comparisons between the study population and the IBS population were conducted using the chi-square test to assess for significant differences. PLINK v1.9 and Haploview v.4.1 were used for the analysis and visualization of LD plots [[41](#B41-ijms-25-00104),[42](#B42-ijms-25-00104)]. THESIAS software v.3.1 was used for haplotype frequency estimation and association analysis between haplotypes and DFS [[43](#B43-ijms-25-00104)]. In bivariate analysis, Kaplan–Meier survival curves were generated to estimate and compare median DFS across sociodemographic, clinical, and SNP variables. Survival distributions were compared with log-rank test and univariate Cox proportional hazards regression model. Multivariate Cox hazards regression model including variables significantly associated with DFS in bivariate analysis was performed using a backward stepwise selection method. Statistical significance was established with a *p*-value < 0.05. False discovery rate (FDR) correction was applied to adjust *p*-values for multiple comparisons. An FDR threshold of 5% was set. Statistical analyses were conducted using PLINK v1.9 and R Software v.4.2.2 (R Foundation for Statistical Computing, Vienna, Austria) [[41](#B41-ijms-25-00104),[44](#B44-ijms-25-00104)].

## Acknowledgments

The results of this research are part of the doctoral thesis that will be presented by Yasmin Cura at the University of Granada, as part of the doctoral studies in “Pharmacy”.

## Abbreviations

5-FU: 5-fluorouracil; 5,10-MTHF: 5,10-methylenetetrahydrofolate; ADC: Adenocarcinomas; CRC: colorectal cancer; DFS: disease-free survival; *DPYD*: dihydropyrimidine dehydrogenase gene; *ERCC1*: ERCC excision repair 1, endonuclease non-catalytic subunit; *ERCC2*: ERCC excision repair 2, TFIIH core complex helicase subunit; ENOF1: mithocondrial-enolase-superfamily-member-1 protein; *ENOSF1*: enolase-superfamily-member-1 gene; FDR: false discovery rate; FdUMP: fluorodeoxyuridine monophosphate; FdUTP: fluorodeoxyuridine triphosphate; FP: fluoropyrimidines; FUTP: fluorouridine triphosphate; HR: hazard ratio; HWE: Hardy–Weinberg equilibrium; LD: linkage disequilibrium; MAF: minor allele frequency; *MTHFR*: methylenetetrahydrofolate-reductase; OS: overall survival; PD: pharmacodynamics; PFS: progression-free survival; SNP: single nucleotide polymorphism; TS: thymidylate synthase protein; *TYMS*: thymidylate-synthase gene; VNTR: variable number tandem repeats; *XRCC1*: X-ray repair cross-complementing 1; *XRCC3*: X-ray repair cross-complementing 3.

## Supplementary Materials

The supporting information can be downloaded at: [https://www.mdpi.com/article/10.3390/ijms25010104/s1](https://www.mdpi.com/article/10.3390/ijms25010104/s1).

## Author Contributions

Conceptualization, Y.C.; methodology, Y.C., A.S.-M. and N.M.-P.; software, Y.C., N.M.-P. and A.S.-M.; validation, C.P.-R., A.S.-M. and A.J.-M.; formal analysis, Y.C. and N.M.-P.; investigation, Y.C. and N.M.-P.; resources, E.G.-F. and A.J.-M.; data curation, Y.C. and N.M.-P.; writing—original draft preparation, Y.C.; writing—review and editing, Y.C., E.G.-F., N.M.-P., A.S.-M. and C.P.-R.; visualization, Y.C. and N.M.-P.; supervision, F.M.-M., C.P.-R. and A.J.-M.; project administration, C.P.-R. and A.J.-M.; funding acquisition, A.J.-M. All authors have read and agreed to the published version of the manuscript.

## Institutional Review Board Statement

The study was conducted in accordance with the Declaration of Helsinki and approved by the Biomedical Research Ethics Committee of Granada (Identification code 0632-M2-20, 2020).

## Informed Consent Statement

Samples provided to the Andalusian Public Health System Biobank presented informed consent from the participants involved in the study.

## Data Availability Statement

Data are unavailable due to privacy and ethical restrictions.

## Conflicts of Interest

The authors declare no conflict of interest.

## Funding Statement

The Virgen de las Nieves University Hospital Biobank was supported by grants co-funded by ERDF funds (EU) from the Instituto de Salud Carlos III (PT13/0010/0039).

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

Data are unavailable due to privacy and ethical restrictions.

### Supplementary Materials

### Data Availability Statement

Data are unavailable due to privacy and ethical restrictions.

## References

1. Siegel R.L., Miller K.D., Wagle N.S., Jemal A. Cancer Statistics, 2023. CA Cancer J. Clin. 2023;73:17–48. doi: 10.3322/caac.21763.  [DOI](https://doi.org/10.3322/caac.21763) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36633525/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=CA%20Cancer%20J.%20Clin.&title=Cancer%20Statistics,%202023&author=R.L.%20Siegel&author=K.D.%20Miller&author=N.S.%20Wagle&author=A.%20Jemal&volume=73&publication_year=2023&pages=17-48&pmid=36633525&doi=10.3322/caac.21763&)

2. Recio-Boiles A., Cagir B. StatPearls. StatPearls Publishing; Treasure Island, FL, USA: 2022. Colon Cancer.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/29262132/) | [Google Scholar](https://scholar.google.com/scholar_lookup?title=StatPearls&author=A.%20Recio-Boiles&author=B.%20Cagir&publication_year=2022&)

3. Recio-Boiles A., Kashyap S., Tsoris A., Babiker H.M. StatPearls. StatPearls Publishing; Treasure Island, FL, USA: 2022. Rectal Cancer.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/29630254/) | [Google Scholar](https://scholar.google.com/scholar_lookup?title=StatPearls&author=A.%20Recio-Boiles&author=S.%20Kashyap&author=A.%20Tsoris&author=H.M.%20Babiker&publication_year=2022&)

4. Gelibter A.J., Caponnetto S., Urbano F., Emiliani A., Scagnoli S., Sirgiovanni G., Napoli V.M., Cortesi E. Adjuvant Chemotherapy in Resected Colon Cancer: When, How and How Long? Surg. Oncol. 2019;30:100–107. doi: 10.1016/j.suronc.2019.06.003.  [DOI](https://doi.org/10.1016/j.suronc.2019.06.003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31500770/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Surg.%20Oncol.&title=Adjuvant%20Chemotherapy%20in%20Resected%20Colon%20Cancer:%20When,%20How%20and%20How%20Long?&author=A.J.%20Gelibter&author=S.%20Caponnetto&author=F.%20Urbano&author=A.%20Emiliani&author=S.%20Scagnoli&volume=30&publication_year=2019&pages=100-107&pmid=31500770&doi=10.1016/j.suronc.2019.06.003&)

5. Bertholee D., Maring J.G., van Kuilenburg A.B.P. Genotypes Affecting the Pharmacokinetics of Anticancer Drugs. Clin. Pharmacokinet. 2017;56:317–337. doi: 10.1007/s40262-016-0450-z.  [DOI](https://doi.org/10.1007/s40262-016-0450-z) | [PMC free article](/articles/PMC5340837/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27641154/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacokinet&title=Genotypes%20Affecting%20the%20Pharmacokinetics%20of%20Anticancer%20Drugs&author=D.%20Bertholee&author=J.G.%20Maring&author=A.B.P.%20van%20Kuilenburg&volume=56&publication_year=2017&pages=317-337&pmid=27641154&doi=10.1007/s40262-016-0450-z&)

6. Mroziewicz M., Tyndale R.F. Pharmacogenetics: A Tool for Identifying Genetic Factors in Drug Dependence and Response to Treatment. Addict Sci. Clin. Pract. 2010;5:17–29.  [PMC free article](/articles/PMC3120126/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22002450/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Addict%20Sci.%20Clin.%20Pract.&title=Pharmacogenetics:%20A%20Tool%20for%20Identifying%20Genetic%20Factors%20in%20Drug%20Dependence%20and%20Response%20to%20Treatment&author=M.%20Mroziewicz&author=R.F.%20Tyndale&volume=5&publication_year=2010&pages=17-29&pmid=22002450&)

7. Cura Y., Pérez-Ramírez C., Sánchez-Martín A., Membrive-Jimenez C., Valverde-Merino M.I., González-Flores E., Morales A.J. Influence of Single-Nucleotide Polymorphisms on Clinical Outcomes of Capecitabine-Based Chemotherapy in Colorectal Cancer Patients: A Systematic Review. Cancers. 2023;15:1821. doi: 10.3390/cancers15061821.  [DOI](https://doi.org/10.3390/cancers15061821) | [PMC free article](/articles/PMC10046456/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36980706/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancers&title=Influence%20of%20Single-Nucleotide%20Polymorphisms%20on%20Clinical%20Outcomes%20of%20Capecitabine-Based%20Chemotherapy%20in%20Colorectal%20Cancer%20Patients:%20A%20Systematic%20Review&author=Y.%20Cura&author=C.%20P%C3%A9rez-Ram%C3%ADrez&author=A.%20S%C3%A1nchez-Mart%C3%ADn&author=C.%20Membrive-Jimenez&author=M.I.%20Valverde-Merino&volume=15&publication_year=2023&pages=1821&pmid=36980706&doi=10.3390/cancers15061821&)

8. Thorn C.F., Marsh S., Carrillo M.W., McLeod H.L., Klein T.E., Altman R.B. PharmGKB Summary: Fluoropyrimidine Pathways. Pharmacogenet. Genom. 2011;21:237–242. doi: 10.1097/FPC.0b013e32833c6107.  [DOI](https://doi.org/10.1097/FPC.0b013e32833c6107) | [PMC free article](/articles/PMC3098754/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20601926/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet.%20Genom.&title=PharmGKB%20Summary:%20Fluoropyrimidine%20Pathways&author=C.F.%20Thorn&author=S.%20Marsh&author=M.W.%20Carrillo&author=H.L.%20McLeod&author=T.E.%20Klein&volume=21&publication_year=2011&pages=237-242&pmid=20601926&doi=10.1097/FPC.0b013e32833c6107&)

9. Ruzzo A., Graziano F., Loupakis F., Santini D., Catalano V., Bisonni R., Ficarelli R., Fontana A., Andreoni F., Falcone A., et al. Pharmacogenetic Profiling in Patients with Advanced Colorectal Cancer Treated with First-Line FOLFIRI Chemotherapy. Pharmacogenomics J. 2008;8:278–288. doi: 10.1038/sj.tpj.6500463.  [DOI](https://doi.org/10.1038/sj.tpj.6500463) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17549067/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J.&title=Pharmacogenetic%20Profiling%20in%20Patients%20with%20Advanced%20Colorectal%20Cancer%20Treated%20with%20First-Line%20FOLFIRI%20Chemotherapy&author=A.%20Ruzzo&author=F.%20Graziano&author=F.%20Loupakis&author=D.%20Santini&author=V.%20Catalano&volume=8&publication_year=2008&pages=278-288&pmid=17549067&doi=10.1038/sj.tpj.6500463&)

10. Etienne-Grimaldi M.-C., Milano G., Maindrault-Goebel F., Chibaudel B., Formento J.-L., Francoual M., Lledo G., André T., Mabro M., Mineur L., et al. Methylenetetrahydrofolate Reductase (MTHFR) Gene Polymorphisms and FOLFOX Response in Colorectal Cancer Patients. Br. J. Clin. Pharmacol. 2010;69:58–66. doi: 10.1111/j.1365-2125.2009.03556.x.  [DOI](https://doi.org/10.1111/j.1365-2125.2009.03556.x) | [PMC free article](/articles/PMC2830598/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20078613/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Clin.%20Pharmacol.&title=Methylenetetrahydrofolate%20Reductase%20(MTHFR)%20Gene%20Polymorphisms%20and%20FOLFOX%20Response%20in%20Colorectal%20Cancer%20Patients&author=M.-C.%20Etienne-Grimaldi&author=G.%20Milano&author=F.%20Maindrault-Goebel&author=B.%20Chibaudel&author=J.-L.%20Formento&volume=69&publication_year=2010&pages=58-66&pmid=20078613&doi=10.1111/j.1365-2125.2009.03556.x&)

11. Huang M.-Y., Huang M.-L., Chen M.-J., Lu C.-Y., Chen C.-F., Tsai P.-C., Chuang S.-C., Hou M.-F., Lin S.-R., Wang J.-Y. Multiple Genetic Polymorphisms in the Prediction of Clinical Outcome of Metastatic Colorectal Cancer Patients Treated with First-Line FOLFOX-4 Chemotherapy. Pharmacogenet Genom. 2011;21:18–25. doi: 10.1097/FPC.0b013e3283415124.  [DOI](https://doi.org/10.1097/FPC.0b013e3283415124) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21057378/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genom.&title=Multiple%20Genetic%20Polymorphisms%20in%20the%20Prediction%20of%20Clinical%20Outcome%20of%20Metastatic%20Colorectal%20Cancer%20Patients%20Treated%20with%20First-Line%20FOLFOX-4%20Chemotherapy&author=M.-Y.%20Huang&author=M.-L.%20Huang&author=M.-J.%20Chen&author=C.-Y.%20Lu&author=C.-F.%20Chen&volume=21&publication_year=2011&pages=18-25&pmid=21057378&doi=10.1097/FPC.0b013e3283415124&)

12. Li M., Zhao Y., Zhao E., Wang K., Lu W., Yuan L. Predictive Value of Two Polymorphisms of ERCC2, Rs13181 and Rs1799793, in Clinical Outcomes of Chemotherapy in Gastric Cancer Patients: A Meta-Analysis. Disease Markers. 2018;2018:3947626. doi: 10.1155/2018/3947626.  [DOI](https://doi.org/10.1155/2018/3947626) | [PMC free article](/articles/PMC6276480/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30581498/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Disease%20Markers&title=Predictive%20Value%20of%20Two%20Polymorphisms%20of%20ERCC2,%20Rs13181%20and%20Rs1799793,%20in%20Clinical%20Outcomes%20of%20Chemotherapy%20in%20Gastric%20Cancer%20Patients:%20A%20Meta-Analysis&author=M.%20Li&author=Y.%20Zhao&author=E.%20Zhao&author=K.%20Wang&author=W.%20Lu&volume=2018&publication_year=2018&pages=3947626&pmid=30581498&doi=10.1155/2018/3947626&)

13. Yousef A.-M., Zawiah M., Al-Yacoub S., Kadi T., Tantawi D.A., Al-Ramadhani H. The Association of Polymorphisms in Folate-Metabolizing Genes with Response to Adjuvant Chemotherapy of Colorectal Cancer. Cancer Chemother. Pharmacol. 2018;82:237–243. doi: 10.1007/s00280-018-3608-6.  [DOI](https://doi.org/10.1007/s00280-018-3608-6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29845393/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Chemother.%20Pharmacol.&title=The%20Association%20of%20Polymorphisms%20in%20Folate-Metabolizing%20Genes%20with%20Response%20to%20Adjuvant%20Chemotherapy%20of%20Colorectal%20Cancer&author=A.-M.%20Yousef&author=M.%20Zawiah&author=S.%20Al-Yacoub&author=T.%20Kadi&author=D.A.%20Tantawi&volume=82&publication_year=2018&pages=237-243&pmid=29845393&doi=10.1007/s00280-018-3608-6&)

14. Boldrin E., Malacrida S., Rumiato E., Battaglia G., Ruol A., Amadori A., Saggioro D. Association Between ERCC1 Rs3212986 and ERCC2/XPD Rs1799793 and OS in Patients with Advanced Esophageal Cancer. Front. Oncol. 2019;9:85. doi: 10.3389/fonc.2019.00085.  [DOI](https://doi.org/10.3389/fonc.2019.00085) | [PMC free article](/articles/PMC6393335/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30847299/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front.%20Oncol.&title=Association%20Between%20ERCC1%20Rs3212986%20and%20ERCC2/XPD%20Rs1799793%20and%20OS%20in%20Patients%20with%20Advanced%20Esophageal%20Cancer&author=E.%20Boldrin&author=S.%20Malacrida&author=E.%20Rumiato&author=G.%20Battaglia&author=A.%20Ruol&volume=9&publication_year=2019&pages=85&pmid=30847299&doi=10.3389/fonc.2019.00085&)

15. Boige V., Mollevi C., Gourgou S., Azria D., Seitz J.-F., Vincent M., Bigot L., Juzyna B., Miran I., Gerard J.-P., et al. Impact of Single-Nucleotide Polymorphisms in DNA Repair Pathway Genes on Response to Chemoradiotherapy in Rectal Cancer Patients: Results from ACCORD-12/PRODIGE-2 Phase III Trial. Int. J. Cancer. 2019;145:3163–3172. doi: 10.1002/ijc.32417.  [DOI](https://doi.org/10.1002/ijc.32417) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31107542/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Cancer&title=Impact%20of%20Single-Nucleotide%20Polymorphisms%20in%20DNA%20Repair%20Pathway%20Genes%20on%20Response%20to%20Chemoradiotherapy%20in%20Rectal%20Cancer%20Patients:%20Results%20from%20ACCORD-12/PRODIGE-2%20Phase%20III%20Trial&author=V.%20Boige&author=C.%20Mollevi&author=S.%20Gourgou&author=D.%20Azria&author=J.-F.%20Seitz&volume=145&publication_year=2019&pages=3163-3172&pmid=31107542&doi=10.1002/ijc.32417&)

16. Salimzadeh H., Lindskog E.B., Gustavsson B., Wettergren Y., Ljungman D. Association of DNA Repair Gene Variants with Colorectal Cancer: Risk, Toxicity, and Survival. BMC Cancer. 2020;20:409. doi: 10.1186/s12885-020-06924-z.  [DOI](https://doi.org/10.1186/s12885-020-06924-z) | [PMC free article](/articles/PMC7216326/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32397974/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Cancer&title=Association%20of%20DNA%20Repair%20Gene%20Variants%20with%20Colorectal%20Cancer:%20Risk,%20Toxicity,%20and%20Survival&author=H.%20Salimzadeh&author=E.B.%20Lindskog&author=B.%20Gustavsson&author=Y.%20Wettergren&author=D.%20Ljungman&volume=20&publication_year=2020&pages=409&pmid=32397974&doi=10.1186/s12885-020-06924-z&)

17. Jeon Y.-J., Cho S.-H., Kim E.-J., Ryu C.-S., Park H.-S., Kim J.-W., Lee J.-Y., An H.-J., Kim N.-K. 3’-UTR Polymorphisms in Thymidylate Synthase with Colorectal Cancer Prevalence and Prognosis. J. Pers. Med. 2021;11:537. doi: 10.3390/jpm11060537.  [DOI](https://doi.org/10.3390/jpm11060537) | [PMC free article](/articles/PMC8228787/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34207922/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Pers.%20Med.&title=3%E2%80%99-UTR%20Polymorphisms%20in%20Thymidylate%20Synthase%20with%20Colorectal%20Cancer%20Prevalence%20and%20Prognosis&author=Y.-J.%20Jeon&author=S.-H.%20Cho&author=E.-J.%20Kim&author=C.-S.%20Ryu&author=H.-S.%20Park&volume=11&publication_year=2021&pages=537&pmid=34207922&doi=10.3390/jpm11060537&)

18. Arjmandi K., Ameli F., Salahshourifar I., Esfandbod M., Irani S. Evaluation of TS and ENOSF1 Variants as a Biomarker in Response to Neoadjuvant Chemotherapy Based on 5FU in Gastric Cancer Patients. Asian Pac. J. Cancer Prev. 2022;23:2983–2989. doi: 10.31557/APJCP.2022.23.9.2983.  [DOI](https://doi.org/10.31557/APJCP.2022.23.9.2983) | [PMC free article](/articles/PMC9810307/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36172660/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Asian%20Pac.%20J.%20Cancer%20Prev.&title=Evaluation%20of%20TS%20and%20ENOSF1%20Variants%20as%20a%20Biomarker%20in%20Response%20to%20Neoadjuvant%20Chemotherapy%20Based%20on%205FU%20in%20Gastric%20Cancer%20Patients&author=K.%20Arjmandi&author=F.%20Ameli&author=I.%20Salahshourifar&author=M.%20Esfandbod&author=S.%20Irani&volume=23&publication_year=2022&pages=2983-2989&pmid=36172660&doi=10.31557/APJCP.2022.23.9.2983&)

19. Pesola F., Eloranta S., Martling A., Saraste D., Smedby K.E. Family History of Colorectal Cancer and Survival: A Swedish Population-Based Study. J. Intern. Med. 2020;287:723–733. doi: 10.1111/joim.13036.  [DOI](https://doi.org/10.1111/joim.13036) | [PMC free article](/articles/PMC7318575/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32012369/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Intern.%20Med.&title=Family%20History%20of%20Colorectal%20Cancer%20and%20Survival:%20A%20Swedish%20Population-Based%20Study&author=F.%20Pesola&author=S.%20Eloranta&author=A.%20Martling&author=D.%20Saraste&author=K.E.%20Smedby&volume=287&publication_year=2020&pages=723-733&pmid=32012369&doi=10.1111/joim.13036&)

20. Oh M.G., Kim J.H., Han M.A., Park J., Ryu S.Y., Choi S.W. Family History and Survival of Patients with Gastric Cancer: A Meta-Analysis. Asian Pac. J. Cancer Prev. 2014;15:3465–3470. doi: 10.7314/APJCP.2014.15.8.3465.  [DOI](https://doi.org/10.7314/APJCP.2014.15.8.3465) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24870741/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Asian%20Pac.%20J.%20Cancer%20Prev.&title=Family%20History%20and%20Survival%20of%20Patients%20with%20Gastric%20Cancer:%20A%20Meta-Analysis&author=M.G.%20Oh&author=J.H.%20Kim&author=M.A.%20Han&author=J.%20Park&author=S.Y.%20Ryu&volume=15&publication_year=2014&pages=3465-3470&pmid=24870741&doi=10.7314/APJCP.2014.15.8.3465&)

21. Brook M.N., Ní Raghallaigh H., Govindasami K., Dadaev T., Rageevakumar R., Keating D., Hussain N., Osborne A., Lophatananon A., UKGPCS Collaborators et al. Family History of Prostate Cancer and Survival Outcomes in the UK Genetic Prostate Cancer Study. Eur. Urol. 2023;83:257–266. doi: 10.1016/j.eururo.2022.11.019.  [DOI](https://doi.org/10.1016/j.eururo.2022.11.019) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36528478/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur.%20Urol.&title=Family%20History%20of%20Prostate%20Cancer%20and%20Survival%20Outcomes%20in%20the%20UK%20Genetic%20Prostate%20Cancer%20Study&author=M.N.%20Brook&author=H.%20N%C3%AD%20Raghallaigh&author=K.%20Govindasami&author=T.%20Dadaev&author=R.%20Rageevakumar&volume=83&publication_year=2023&pages=257-266&pmid=36528478&doi=10.1016/j.eururo.2022.11.019&)

22. Barresi V., Reggiani Bonetti L., Ieni A., Caruso R.A., Tuccari G. Histological Grading in Colorectal Cancer: New Insights and Perspectives. Histol. Histopathol. 2015;30:1059–1067. doi: 10.14670/HH-11-633.  [DOI](https://doi.org/10.14670/HH-11-633) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26004398/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Histol.%20Histopathol.&title=Histological%20Grading%20in%20Colorectal%20Cancer:%20New%20Insights%20and%20Perspectives&author=V.%20Barresi&author=L.%20Reggiani%20Bonetti&author=A.%20Ieni&author=R.A.%20Caruso&author=G.%20Tuccari&volume=30&publication_year=2015&pages=1059-1067&pmid=26004398&doi=10.14670/HH-11-633&)

23. Sagaert X., Vanstapel A., Verbeek S. Tumor Heterogeneity in Colorectal Cancer: What Do We Know So Far? Pathobiology. 2018;85:72–84. doi: 10.1159/000486721.  [DOI](https://doi.org/10.1159/000486721) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29414818/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pathobiology&title=Tumor%20Heterogeneity%20in%20Colorectal%20Cancer:%20What%20Do%20We%20Know%20So%20Far?&author=X.%20Sagaert&author=A.%20Vanstapel&author=S.%20Verbeek&volume=85&publication_year=2018&pages=72-84&pmid=29414818&doi=10.1159/000486721&)

24. Johncilla M., Yantiss R.K. Histology of Colorectal Carcinoma. Surg. Pathol. Clin. 2020;13:503–520. doi: 10.1016/j.path.2020.05.008.  [DOI](https://doi.org/10.1016/j.path.2020.05.008) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32773197/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Surg.%20Pathol.%20Clin.&title=Histology%20of%20Colorectal%20Carcinoma&author=M.%20Johncilla&author=R.K.%20Yantiss&volume=13&publication_year=2020&pages=503-520&pmid=32773197&doi=10.1016/j.path.2020.05.008&)

25. Cao Y., Zhang G., Bao H., Ren J., Wang Z., Zhang J., Zhao Z., Yan X., Chai Y., Zhou J. Development of a Dual-Energy Spectral Computed Tomography-Based Nomogram for the Preoperative Discrimination of Histological Grade in Colorectal Adenocarcinoma Patients. J. Gastrointest. Oncol. 2021;12:544–555. doi: 10.21037/jgo-20-368.  [DOI](https://doi.org/10.21037/jgo-20-368) | [PMC free article](/articles/PMC8107588/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34012648/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Gastrointest.%20Oncol.&title=Development%20of%20a%20Dual-Energy%20Spectral%20Computed%20Tomography-Based%20Nomogram%20for%20the%20Preoperative%20Discrimination%20of%20Histological%20Grade%20in%20Colorectal%20Adenocarcinoma%20Patients&author=Y.%20Cao&author=G.%20Zhang&author=H.%20Bao&author=J.%20Ren&author=Z.%20Wang&volume=12&publication_year=2021&pages=544-555&pmid=34012648&doi=10.21037/jgo-20-368&)

26. Zou G., Wang R., Wang M. Clinical Response and Prognostic Significance of Serum miR-497 Expression in Colorectal Cancer. Cancer Biomark. 2019;25:11–18. doi: 10.3233/CBM-181902.  [DOI](https://doi.org/10.3233/CBM-181902) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31006664/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Biomark&title=Clinical%20Response%20and%20Prognostic%20Significance%20of%20Serum%20miR-497%20Expression%20in%20Colorectal%20Cancer&author=G.%20Zou&author=R.%20Wang&author=M.%20Wang&volume=25&publication_year=2019&pages=11-18&pmid=31006664&doi=10.3233/CBM-181902&)

27. Huang X., Cheng Z., Huang Y., Liang C., He L., Ma Z., Chen X., Wu X., Li Y., Liang C., et al. CT-Based Radiomics Signature to Discriminate High-Grade from Low-Grade Colorectal Adenocarcinoma. Acad Radiol. 2018;25:1285–1297. doi: 10.1016/j.acra.2018.01.020.  [DOI](https://doi.org/10.1016/j.acra.2018.01.020) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29503175/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Acad%20Radiol.&title=CT-Based%20Radiomics%20Signature%20to%20Discriminate%20High-Grade%20from%20Low-Grade%20Colorectal%20Adenocarcinoma&author=X.%20Huang&author=Z.%20Cheng&author=Y.%20Huang&author=C.%20Liang&author=L.%20He&volume=25&publication_year=2018&pages=1285-1297&pmid=29503175&doi=10.1016/j.acra.2018.01.020&)

28. Mehrkhani F., Nasiri S., Donboli K., Meysamie A., Hedayat A. Prognostic Factors in Survival of Colorectal Cancer Patients after Surgery. Color. Dis. 2009;11:157–161. doi: 10.1111/j.1463-1318.2008.01556.x.  [DOI](https://doi.org/10.1111/j.1463-1318.2008.01556.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18462239/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Color.%20Dis.&title=Prognostic%20Factors%20in%20Survival%20of%20Colorectal%20Cancer%20Patients%20after%20Surgery&author=F.%20Mehrkhani&author=S.%20Nasiri&author=K.%20Donboli&author=A.%20Meysamie&author=A.%20Hedayat&volume=11&publication_year=2009&pages=157-161&pmid=18462239&doi=10.1111/j.1463-1318.2008.01556.x&)

29. Cho Y.B., Chun H.-K., Yun H.R., Kim H.C., Yun S.H., Lee W.Y. Histological Grade Predicts Survival Time Associated with Recurrence after Resection for Colorectal Cancer. Hepatogastroenterology. 2009;56:1335–1340.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/19950787/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hepatogastroenterology&title=Histological%20Grade%20Predicts%20Survival%20Time%20Associated%20with%20Recurrence%20after%20Resection%20for%20Colorectal%20Cancer&author=Y.B.%20Cho&author=H.-K.%20Chun&author=H.R.%20Yun&author=H.C.%20Kim&author=S.H.%20Yun&volume=56&publication_year=2009&pages=1335-1340&pmid=19950787&)

30. van den Berg I., Coebergh van den Braak R.R.J., van Vugt J.L.A., Ijzermans J.N.M., Buettner S. Actual Survival after Resection of Primary Colorectal Cancer: Results from a Prospective Multicenter Study. World J. Surg. Oncol. 2021;19:96. doi: 10.1186/s12957-021-02207-4.  [DOI](https://doi.org/10.1186/s12957-021-02207-4) | [PMC free article](/articles/PMC8022415/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33820567/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=World%20J.%20Surg.%20Oncol.&title=Actual%20Survival%20after%20Resection%20of%20Primary%20Colorectal%20Cancer:%20Results%20from%20a%20Prospective%20Multicenter%20Study&author=I.%20van%20den%20Berg&author=R.R.J.%20Coebergh%20van%20den%20Braak&author=J.L.A.%20van%20Vugt&author=J.N.M.%20Ijzermans&author=S.%20Buettner&volume=19&publication_year=2021&pages=96&pmid=33820567&doi=10.1186/s12957-021-02207-4&)

31. Liu Z., Xu Y., Xu G., Baklaushev V.P., Chekhonin V.P., Peltzer K., Ma W., Wang X., Wang G., Zhang C. Nomogram for Predicting Overall Survival in Colorectal Cancer with Distant Metastasis. BMC Gastroenterol. 2021;21:103. doi: 10.1186/s12876-021-01692-x.  [DOI](https://doi.org/10.1186/s12876-021-01692-x) | [PMC free article](/articles/PMC7934422/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33663400/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Gastroenterol.&title=Nomogram%20for%20Predicting%20Overall%20Survival%20in%20Colorectal%20Cancer%20with%20Distant%20Metastasis&author=Z.%20Liu&author=Y.%20Xu&author=G.%20Xu&author=V.P.%20Baklaushev&author=V.P.%20Chekhonin&volume=21&publication_year=2021&pages=103&pmid=33663400&doi=10.1186/s12876-021-01692-x&)

32. Hamzic S., Kummer D., Froehlich T.K., Joerger M., Aebi S., Palles C., Thomlinson I., Meulendijks D., Schellens J.H.M., García-González X., et al. Evaluating the Role of ENOSF1 and TYMS Variants as Predictors in Fluoropyrimidine-Related Toxicities: An IPD Meta-Analysis. Pharmacol. Res. 2020;152:104594. doi: 10.1016/j.phrs.2019.104594.  [DOI](https://doi.org/10.1016/j.phrs.2019.104594) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31838077/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacol.%20Res.&title=Evaluating%20the%20Role%20of%20ENOSF1%20and%20TYMS%20Variants%20as%20Predictors%20in%20Fluoropyrimidine-Related%20Toxicities:%20An%20IPD%20Meta-Analysis&author=S.%20Hamzic&author=D.%20Kummer&author=T.K.%20Froehlich&author=M.%20Joerger&author=S.%20Aebi&volume=152&publication_year=2020&pages=104594&pmid=31838077&doi=10.1016/j.phrs.2019.104594&)

33. Wichelecki D.J., Froese D.S., Kopec J., Muniz J.R.C., Yue W.W., Gerlt J.A. Enzymatic and Structural Characterization of rTSγ Provides Insights into the Function of rTSβ. Biochemistry. 2014;53:2732–2738. doi: 10.1021/bi500349e.  [DOI](https://doi.org/10.1021/bi500349e) | [PMC free article](/articles/PMC4010280/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24697329/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochemistry&title=Enzymatic%20and%20Structural%20Characterization%20of%20rTS%CE%B3%20Provides%20Insights%20into%20the%20Function%20of%20rTS%CE%B2&author=D.J.%20Wichelecki&author=D.S.%20Froese&author=J.%20Kopec&author=J.R.C.%20Muniz&author=W.W.%20Yue&volume=53&publication_year=2014&pages=2732-2738&pmid=24697329&doi=10.1021/bi500349e&)

34. Rosmarin D., Palles C., Pagnamenta A., Kaur K., Pita G., Martin M., Domingo E., Jones A., Howarth K., Freeman-Mills L., et al. A Candidate Gene Study of Capecitabine-Related Toxicity in Colorectal Cancer Identifies New Toxicity Variants at DPYD and a Putative Role for ENOSF1 Rather than TYMS. Gut. 2015;64:111–120. doi: 10.1136/gutjnl-2013-306571.  [DOI](https://doi.org/10.1136/gutjnl-2013-306571) | [PMC free article](/articles/PMC4283622/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24647007/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gut&title=A%20Candidate%20Gene%20Study%20of%20Capecitabine-Related%20Toxicity%20in%20Colorectal%20Cancer%20Identifies%20New%20Toxicity%20Variants%20at%20DPYD%20and%20a%20Putative%20Role%20for%20ENOSF1%20Rather%20than%20TYMS&author=D.%20Rosmarin&author=C.%20Palles&author=A.%20Pagnamenta&author=K.%20Kaur&author=G.%20Pita&volume=64&publication_year=2015&pages=111-120&pmid=24647007&doi=10.1136/gutjnl-2013-306571&)

35. Meulendijks D., Rozeman E.A., Cats A., Sikorska K., Joerger M., Deenen M.J., Beijnen J.H., Schellens J.H.M. Pharmacogenetic Variants Associated with Outcome in Patients with Advanced Gastric Cancer Treated with Fluoropyrimidine and Platinum-Based Triplet Combinations: A Pooled Analysis of Three Prospective Studies. Pharmacogenomics J. 2017;17:441–451. doi: 10.1038/tpj.2016.81.  [DOI](https://doi.org/10.1038/tpj.2016.81) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27995989/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J.&title=Pharmacogenetic%20Variants%20Associated%20with%20Outcome%20in%20Patients%20with%20Advanced%20Gastric%20Cancer%20Treated%20with%20Fluoropyrimidine%20and%20Platinum-Based%20Triplet%20Combinations:%20A%20Pooled%20Analysis%20of%20Three%20Prospective%20Studies&author=D.%20Meulendijks&author=E.A.%20Rozeman&author=A.%20Cats&author=K.%20Sikorska&author=M.%20Joerger&volume=17&publication_year=2017&pages=441-451&pmid=27995989&doi=10.1038/tpj.2016.81&)

36. Dolnick B.J., Angelino N.J., Dolnick R., Sufrin J.R. A Novel Function for the rTS Gene. Cancer Biol. Ther. 2003;2:364–369. doi: 10.4161/cbt.2.4.424.  [DOI](https://doi.org/10.4161/cbt.2.4.424) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14508106/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Biol.%20Ther.&title=A%20Novel%20Function%20for%20the%20rTS%20Gene&author=B.J.%20Dolnick&author=N.J.%20Angelino&author=R.%20Dolnick&author=J.R.%20Sufrin&volume=2&publication_year=2003&pages=364-369&pmid=14508106&doi=10.4161/cbt.2.4.424&)

37. Wu Q., Dolnick B.J. Detection of Thymidylate Synthase Modulators by a Novel Screening Assay. Mol. Pharmacol. 2003;63:167–173. doi: 10.1124/mol.63.1.167.  [DOI](https://doi.org/10.1124/mol.63.1.167) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12488549/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Pharmacol.&title=Detection%20of%20Thymidylate%20Synthase%20Modulators%20by%20a%20Novel%20Screening%20Assay&author=Q.%20Wu&author=B.J.%20Dolnick&volume=63&publication_year=2003&pages=167-173&pmid=12488549&doi=10.1124/mol.63.1.167&)

38. Matsusaka S., Lenz H.-J. Pharmacogenomics of Fluorouracil -Based Chemotherapy Toxicity. Expert Opin Drug Metab. Toxicol. 2015;11:811–821. doi: 10.1517/17425255.2015.1027684.  [DOI](https://doi.org/10.1517/17425255.2015.1027684) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25800061/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Expert%20Opin%20Drug%20Metab.%20Toxicol.&title=Pharmacogenomics%20of%20Fluorouracil%20-Based%20Chemotherapy%20Toxicity&author=S.%20Matsusaka&author=H.-J.%20Lenz&volume=11&publication_year=2015&pages=811-821&pmid=25800061&doi=10.1517/17425255.2015.1027684&)

39. Von Mering C., Jensen L.J., Snel B., Hooper S.D., Krupp M., Foglierini M., Jouffre N., Huynen M.A., Bork P. STRING: Known and Predicted Protein-Protein Associations, Integrated and Transferred across Organisms. Nucleic Acids Res. 2005;33:D433–D437. doi: 10.1093/nar/gki005.  [DOI](https://doi.org/10.1093/nar/gki005) | [PMC free article](/articles/PMC539959/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15608232/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&title=STRING:%20Known%20and%20Predicted%20Protein-Protein%20Associations,%20Integrated%20and%20Transferred%20across%20Organisms&author=C.%20Von%20Mering&author=L.J.%20Jensen&author=B.%20Snel&author=S.D.%20Hooper&author=M.%20Krupp&volume=33&publication_year=2005&pages=D433-D437&pmid=15608232&doi=10.1093/nar/gki005&)

40. Lautner-Csorba O., Gézsi A., Erdélyi D.J., Hullám G., Antal P., Semsei Á.F., Kutszegi N., Kovács G., Falus A., Szalai C. Roles of Genetic Polymorphisms in the Folate Pathway in Childhood Acute Lymphoblastic Leukemia Evaluated by Bayesian Relevance and Effect Size Analysis. PLoS ONE. 2013;8:e69843. doi: 10.1371/journal.pone.0069843.  [DOI](https://doi.org/10.1371/journal.pone.0069843) | [PMC free article](/articles/PMC3734218/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23940529/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&title=Roles%20of%20Genetic%20Polymorphisms%20in%20the%20Folate%20Pathway%20in%20Childhood%20Acute%20Lymphoblastic%20Leukemia%20Evaluated%20by%20Bayesian%20Relevance%20and%20Effect%20Size%20Analysis&author=O.%20Lautner-Csorba&author=A.%20G%C3%A9zsi&author=D.J.%20Erd%C3%A9lyi&author=G.%20Hull%C3%A1m&author=P.%20Antal&volume=8&publication_year=2013&pages=e69843&pmid=23940529&doi=10.1371/journal.pone.0069843&)

41. Purcell S., Neale B., Todd-Brown K., Thomas L., Ferreira M., Bender D., Maller J., Sklar P., de Bakker P.I.W., Daly M.J., et al. PLINK: A Toolset for Whole-Genome Association and Population-Based Linkage Analysis. Am. J. Hum. Genet. 2007;81:559–575. doi: 10.1086/519795.  [DOI](https://doi.org/10.1086/519795) | [PMC free article](/articles/PMC1950838/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17701901/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Hum.%20Genet.&title=PLINK:%20A%20Toolset%20for%20Whole-Genome%20Association%20and%20Population-Based%20Linkage%20Analysis&author=S.%20Purcell&author=B.%20Neale&author=K.%20Todd-Brown&author=L.%20Thomas&author=M.%20Ferreira&volume=81&publication_year=2007&pages=559-575&pmid=17701901&doi=10.1086/519795&)

42. Barrett J.C., Fry B., Maller J., Daly M.J. Haploview: Analysis and Visualization of LD and Haplotype Maps. Bioinformatics. 2005;21:263–265. doi: 10.1093/bioinformatics/bth457.  [DOI](https://doi.org/10.1093/bioinformatics/bth457) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15297300/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=Haploview:%20Analysis%20and%20Visualization%20of%20LD%20and%20Haplotype%20Maps&author=J.C.%20Barrett&author=B.%20Fry&author=J.%20Maller&author=M.J.%20Daly&volume=21&publication_year=2005&pages=263-265&pmid=15297300&doi=10.1093/bioinformatics/bth457&)

43. Tregouet D.A., Garelle V. A New JAVA Interface Implementation of THESIAS: Testing Haplotype Effects in Association Studies. Bioinformatics. 2007;23:1038–1039. doi: 10.1093/bioinformatics/btm058.  [DOI](https://doi.org/10.1093/bioinformatics/btm058) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17308338/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=A%20New%20JAVA%20Interface%20Implementation%20of%20THESIAS:%20Testing%20Haplotype%20Effects%20in%20Association%20Studies&author=D.A.%20Tregouet&author=V.%20Garelle&volume=23&publication_year=2007&pages=1038-1039&pmid=17308338&doi=10.1093/bioinformatics/btm058&)

44. R Core Team . R: A Language and Environment for Statistical Computing. R Foundation for Statistical Computing. R Core Team; Vienna, Austria: 2021.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=R:%20A%20Language%20and%20Environment%20for%20Statistical%20Computing.%20R%20Foundation%20for%20Statistical%20Computing&publication_year=2021&)
