# UGT1A1 and BLVRA allele and genotype variants in neonatal patients with hyperbilirubinemia in southern China

## Metadata
**Authors:** XiuJu Liu, Chao Zhang, LiWen Chen, ShuYan Liu, ZhenWen Zhou
**Journal:** Scientific Reports
**Date:** 2024 Oct 28
**DOI:** [10.1038/s41598-024-77499-z](https://doi.org/10.1038/s41598-024-77499-z)
**PMID:** 39468242
**PMCID:** PMC11519535
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11519535/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC11519535/pdf/41598_2024_Article_77499.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC11519535/pdf/41598_2024_Article_77499.pdf)

## Abstract

We explore the allele and genotype distribution of UGT1A1 and BLVRA variants in individuals affected by neonatal hyperbilirubinemia in southern China. Blood specimens were collected from 240 neonates: 126 cases of hyperbilirubinemia and 114 healthy controls. Serum levels of total protein, albumin, bilirubin (total and direct), urea nitrogen, creatinine, and other biochemical parameters were quantified using a biochemical analyzer. Nine UGT1A1 and five BLVRA genetic variants were genotyped using flight time mass spectrometry. The allele and genotype frequencies of these variants and their associations with neonatal hyperbilirubinemia were analyzed. The genotype frequencies of CC and CG for the UGT1A1 variant rs11888492 in the hyperbilirubinemia group were 90.48% and 9.52%, respectively (P = 0.001), in comparison with the control group. The C and G allele frequencies of rs11888492 in the hyperbilirubinemia group were 95.24% and 4.76%, respectively (P = 0.023). Similarly, in the hyperbilirubinemia group, the genotype frequencies for the UGT1A1 variant rs4148325 were 90.48% CC, 8.73% CT, and 0.79% TT (P = 0.001), with corresponding allele frequencies of 94.84% for C and 5.16% for T (P = 0.002). No notable distinctions were detected for other variants. Newborns carrying the CC genotype of rs11888492 exhibited higher total bilirubin (TBIL) levels than those carrying the GG genotype (P = 0.034), whereas newborns carrying the CC genotype of rs4148325 displayed higher TBIL levels than those carrying the CT genotype (P = 0.003). The presence of the G allele at rs11888492 was found to be significantly correlated with a decreased likelihood of developing neonatal hyperbilirubinemia (odds ratio [OR]: 0.363; 95% confidence interval [CI] 0.169–0.777). Furthermore, a substantial reduction in the risk of neonatal hyperbilirubinemia associated with the CT genotype of rs4148325 were revealed (OR = 0.242; 95% CI 0.102–0.574). Additionally, an inverse relationship was identified between TBIL concentration and the quantity of genetic variants. The UGT1A1 variants rs11888492 and rs4148325 are strongly associated with neonatal hyperbilirubinemia in southern China.

Keywords: Biliverdin reductase A (BLVRA), Neonatal hyperbilirubinemia (NHB), Single nucleotide polymorphism (SNP), Uridine diphosphate glucuronosyltransferase 1A1 (UGT1A1)

Subject terms: Diseases, Signs and symptoms

## Introduction

Hyperbilirubinemia is a prevalent clinical condition among newborns that is primarily attributed to factors such as excessive bilirubin production, compromised liver uptake and conjugation, and enhanced circulation of bilirubin in the enterohepatic system. Elevated bilirubin levels can lead to severe complications, including acute bilirubin encephalopathy and kernicterus, resulting in irreversible nervous system damage and, in extreme cases, mortality^1,2^.

The etiology of hyperbilirubinemia encompasses various maternal and neonatal factors, with increasing recognition of its genetic components^3^. Biliverdin reductase A (BLVRA) and uridine diphosphate glucuronosyltransferase 1A1 (UGT1A1) play pivotal roles in bilirubin metabolism by converting biliverdin to bilirubin and facilitating the conjugation of unconjugated bilirubin to glucuronic acid molecules, respectively^4^. Emerging evidence suggests a significant link between specific genetic variants of UGT1A1 or BLVRA and the incidence of severe hyperbilirubinemia^5–7^. Mutations in UGT1A1 cause a decrease or deletion of glucuronidase activity, affecting bilirubin synthesis in vivo and leading to hyperunbound bilirubinemia^2,8^. Genetic variants of UGT1A1, including UGT1A1*28 and UGT1A1*6, have been recognized as contributing factors for the risk of neonatal jaundice in Asian populations^9,10^.

In this study, we aimed to investigate the role of novel or less-studied genetic variants that may contribute to neonatal jaundice. While UGT1A1 variants are well-established risk factors, we sought to explore other genetic factors that may provide new insights into the condition. Therefore, our research sought to explore the allele and genotype frequencies of nine UGT1A1 and five BLVRA variants associated with neonatal hyperbilirubinemia. As far as we are aware, our study represents the inaugural report on the connection between UGT1A1 rs12479045, BLVRA rs1181576, and rs2730625 and neonatal hyperbilirubinemia.

## Results

### Clinical analysis

The clinical data of the control and NHB groups are summarized in Table [1](#Tab1). There were no substantial disparities in birth weight or gestational age (weeks) between the control and NHB groups. The average TBIL levels were 141.14 ± 46.76 µmol/L for the control group and 316.47 ± 36.46 µmol/L for NHB group. Relative to the control group, a significant elevation was observed in serum BU, BC, TP, and ALB levels within the NHB cohort (*P* < 0.0001, *P* < 0.0001, *P* = 0.0105, and *P* < 0.0001, respectively), whereas the NHB group demonstrated a notable rise in serum levels of AST, BUN, and CR (*P* < 0.0001, *P* = 0.0417, and *P* < 0.0001, respectively).

### Table 1.

| Parameter | NHB group (N = 126) | Control group (N = 114) | P-value |
| --- | --- | --- | --- |
| Birth weight (g) | 3210 ± 380 | 3120 ± 650 | 0.3298 |
| Gestational age(week) | 38.27 ± 2.46 | 39.01 ± 1.32 | 0.2354 |
| Age (days) | 8.56 ± 5.27 | 3.91 ± 4.07 | < 0.0001 |
| TBIL (µmol/L) | 316.47 ± 36.46 | 141.14 ± 46.76 | < 0.0001 |
| BU (µmol/L) | 308.41 ± 33.06 | 140.29 ± 46.70 | < 0.0001 |
| BC (µmol/L) | 6.97 ± 6.50 | 0.84 ± 1.56 | < 0.0001 |
| ALT (U/L) | 13.33 ± 8.15 | 16.11 ± 10.07 | 0.0777 |
| AST (U/L) | 38.79 ± 18.38 | 61.13 ± 31.87 | < 0.0001 |
| TP (g/L) | 54.86 ± 5.57 | 52.35 ± 5.93 | 0.0105 |
| ALB (g/L) | 38.81 ± 3.52 | 35.86 ± 3.81 | < 0.0001 |
| GLB(g/L) | 16.04 ± 3.26 | 16.49 ± 3.30 | 0.4187 |
| BUN (mmol/L) | 2.69 ± 1.35 | 3.11 ± 1.22 | 0.0417 |
| CR (mmol/L) | 39.32 ± 10.22 | 60.63 ± 18.40 | < 0.0001 |

Table 1 Caption: Clinical characteristics of control group and NHB group.

### Allele and genotype frequencies of UGT1A1 and BLVRA variants

The gene sequence accession number and the primers’sequences location are shown in Table [2](#Tab2). The allele and genotype frequencies of the UGT1A1 and BLVRA variants are shown in Table [3](#Tab3). The distribution of genetic variants for UGT1A1 and BLVRA adhered to the Hardy–Weinberg proportions, indicating no significant deviation from equilibrium (*P* > 0.05). Marked disparities were evident in the allele and genotype frequencies of rs11888492 and rs4148325 within UGT1A1 when comparing the control and NHB groups, whereas other variants did not exhibit such differences. For rs11888492, the genotype frequencies of CC and CG in the NHB group were 114 (90.48%) and 12 (9.52%), respectively, showing a notable disparity (*P* = 0.001). The C and G alleles frequencies in the NHB group were 240 (95.24%) and 12 (4.76%), respectively, showing a significant difference (*P* = 0.023). Similarly, for rs4148325, the genotype frequencies of CC, CT, and TT in the NHB group were 114 (90.48%), 11 (8.73%), and 1 (0.79%), respectively (*P* = 0.001). The C and T allele frequencies in the NHB group were 239 (94.84%) and 13 (5.16%), respectively, demonstrating a significant difference (*P* = 0.023). Thus, the variants rs11888492 and rs4148325, which showed notable distributional differences between the control and NHB groups, were incorporated into the conditional logistic regression model.

### Table 2.

| Gene | Accession number | SNP_ID | Sequence | Location |
| --- | --- | --- | --- | --- |
| Uridine diphosphate glucuronosyltransferase 1A1 (UGT1A1) | NM_000463 | rs12479045 | ACGTTGGATGAGACACTGACCATCCTTCTC ACGTTGGATGTTAAGAGCCAGAGGACTCAC | Intron2 |
| rs8330 | ACGTTGGATGCTTTAAACACACAAGGTGGC ACGTTGGATGTTGGCTTCTGCAGATGGTTG | Exon5 |  |  |
| rs1042640 | ACGTTGGATGCCATCTGCAGAAGCCAAAAG ACGTTGGATGAGTGCGGGATTCAAAGGTGG | Exon5 |  |  |
| rs3771342 | ACGTTGGATGTCCCTTCCCAAATGCATTTC ACGTTGGATGATGTGGGTTACTGGGAATGC | Intron2 |  |  |
| rs11888492 | ACGTTGGATGTAGTCCTGAATAGGAGTGCC ACGTTGGATGGTTAGGTGCTTCAACCCTTG | Exon4 |  |  |
| rs4148325 | ACGTTGGATGCAGCTCATTTCCAAGTGCTC ACGTTGGATGGAGTGGGTTTCATGAAGAGC | Intron2 |  |  |
| rs17864705 | ACGTTGGATGGTGTTGATGGTCCTATTCTC ACGTTGGATGATGCCAGGTTGTGCTTTCTC | Intron2 |  |  |
| rs4148328 | ACGTTGGATGCCCCATTAGATTTAAAACTCC ACGTTGGATGTCTGTGCAGGAAACTTATGG | Intron4 |  |  |
| rs1018124 | ACGTTGGATGTCAGTGTTCTAACCAACAGC ACGTTGGATGGACTCAAAAGTGTATTAGGG | Intron3 |  |  |
| Biliverdin reductase A (BLVRA) | NM_000712 | rs7738 | ACGTTGGATGCCCAGACTTGAAATGGAAGC ACGTTGGATGTGTCTTTGCAGTCCACTGTC | Exon11 |
| rs699512 | ACGTTGGATGAAGCATCCCTTAAACTGGTC ACGTTGGATGGTCTTTACAGTGACCGAAGG | Exon5 |  |  |
| rs10486752 | ACGTTGGATGGGAATCTGAAACTCCAGAGC ACGTTGGATGCCCCGAGTACAACAATCAGA | Intron3 |  |  |
| rs1181576 | ACGTTGGATGCAACAGTACCATTCACAGCG ACGTTGGATGCGCCCAGCTGTTTCTTTTGT | Intron7 |  |  |
| rs2730625 | ACGTTGGATGGGGCAACAAGAGCAAACATC ACGTTGGATGTCTGGGAGATGGCAACAAAG | Exon11 |  |  |

Table 2 Caption: The gene sequence accession number and the primers’ sequences location.

### Table 3.

| Gene | Variants | Genotype frequency n(%) |  |  | P-value | Allele frequency n (%) |  | P-value |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Uridine diphosphate glucuronosyltransferase 1A1 (UGT1A1) | rs1018124 (A > G) | AA | AG | GG |   | A | G |   |
| Control group | 64 (56.14) | 46 (40.35) | 4 (3.51) | 0.010 | 174 (76.32) | 54 (23.68) | 0.101 |  |
| NHB group | 70 (55.56) | 35 (27.78) | 21 (16.67) |   | 175 (69.44) | 77 (30.56) |   |  |
| rs1042640 (G > C) | CC | CG | GG |   | C | G |   |  |
| Control group | 94 (82.46) | 20 (21.28) | 0 (0.00) | 0.237 | 208 (91.23) | 20 (8.77) | 0.386 |  |
| NHB group | 111 (88.10) | 14 (11.11) | 1 (0.79) |   | 236 (93.65) | 16 (6.35) |   |  |
| rs11888492 (C > G) | CC | CG | GG |   | C | G |   |  |
| Control group | 90 (78.95) | 24 (21.05) | 4 (7.02) | 0.001 | 204 (89.47) | 32 (14.04) | 0.023 |  |
| NHB group | 114 (90.48) | 12 (9.52) | 0 (0.00) |   | 240 (95.24) | 12 (4.76) |   |  |
| rs12479045 (G > C) | GG | CG | CC |   | C | G |   |  |
| Control group | 64 (56.14) | 46 (40.35) | 4 (3.51) | 0.368 | 54 (23.68) | 174 (76.32) | 0.442 |  |
| NHB group | 80 (63.49) | 40 (31.75) | 6 (4.76) |   | 52 (20.63) | 200 (79.37) |   |  |
| rs17864705 (G > T) | GG | TG | TT |   | G | T |   |  |
| Control group | 64 (56.14) | 46 (40.35) | 4 (3.51) | 0.248 | 174 (76.32) | 54 (23.68) | 0.320 |  |
| NHB group | 82 (65.08) | 38 (30.16) | 6 (4.76) |   | 202 (80.16) | 50 (19.84) |   |  |
| rs3771342 (G > T) | GG | TG | TT |   | G | T |   |  |
| Control group | 64 (56.14) | 46 (40.35) | 4 (3.51) | 0.435 | 174 (76.32) | 54 (23.68) | 0.511 |  |
| NHB group | 79 (62.70) | 41 (32.54) | 6 (4.76) |   | 199 (78.97) | 53 (21.03) |   |  |
| rs4148325 (C > T) | CC | CT | TT |   | C | T |   |  |
| Control group | 84 (73.68) | 30 (26.32) | 0 (0.00) | 0.001 | 198 (76.23) | 30 (23.68) | 0.002 |  |
| NHB group | 114 (90.48) | 11 (8.73) | 1 (0.79) |   | 239 (94.84) | 13 (5.16) |   |  |
| rs4148328 (C > T) | CC | CT | TT |   | C | T |   |  |
| Control group | 16 (14.04) | 54 (47.37) | 44 (38.60) | 0.100 | 86 (37.72) | 142 (62.28) | 0.052 |  |
| NHB group | 13 (10.32) | 47 (37.30) | 66 (52.38) |   | 73 (28.97) | 179 (71.03) |   |  |
| rs8330 (G > C) | CC | CG | GG |   | C | G |   |  |
| Control group | 94 (64.91) | 20 (35.09) | 0 (0.00) | 0.237 | 236 (93.65) | 16 (6.35) | 0.386 |  |
| NHB group | 111 (88.10) | 14 (11.11) | 1 (0.79) |   | 208 (91.23) | 20 (8.77) |   |  |
| Biliverdin reductase A (BLVRA) | rs10486752 (A > G) | AA | AG | GG |   | A | G |   |
| Control group | 66 (57.89) | 42 (36.84) | 6 (5.26) | 0.150 | 174 (76.32) | 54 (23.68) | 0.067 |  |
| NHB group | 98 (69.84) | 24 (26.98) | 4 (3.17) |   | 210 (83.33) | 42 (16.67) |   |  |
| rs1181576 (G > A) | GG | AG | AA |   | A | G |   |  |
| Control group | 42 (36.84) | 44 (38.60) | 28 (24.56) | 0.345 | 100 (43.86) | 128 (56.14) | 0.645 |  |
| NHB group | 44 (34.92) | 59 (46.83) | 23 (18.25) |   | 105 (41.67) | 147 (58.33) |   |  |
| rs2730625 (C > T) | CC | CT | TT |   | C | T |   |  |
| Control group | 18 (15.79) | 36 (31.58) | 60 (52.63) | 0.291 | 72 (31.58) | 156 (68.42) | 0.697 |  |
| NHB group | 16 (12.70) | 52 (41.27) | 58 (46.03) |   | 84 (33.33) | 168 (66.67) |   |  |
| rs699512 (G > A) | AA | AG | GG |   | A | G |   |  |
| Control group | 60 (52.63) | 36 (31.58) | 18 (15.79) | 0.273 | 156 (68.42) | 72 (31.58) | 0.845 |  |
| NHB group | 59 (46.83) | 52 (41.27) | 15 (11.90) |   | 170 (67.46) | 82 (32.54) |   |  |
| rs7738 (A > G) | AA | AG | GG |   | A | G |   |  |
| Control group | 44 (38.60) | 46 (40.35) | 24 (21.05) | 0.664 | 134 (58.77) | 94 (41.23) | > 0.9999 |  |
| NHB group | 45 (35.71) | 58 (46.03) | 23 (18.25) |   | 148 (58.73) | 104 (41.27) |   |  |

Table 3 Caption: The genotype and allele frequency of UGT1A1 and BLVRA variants.

Table [4](#Tab4) presents the findings from the logistic regression analysis of the variants associated with neonatal hyperbilirubinemia. The logistic regression model for rs11888492 revealed that in contrast to allele C, allele G was correlated with a decreased probability of NHB occurrence (OR = 0.363; 95% CI 0.169–0.777). Similarly, analysis of rs4148325 indicated that compared to CC, the CT genotype exhibited a lowered likelihood of developing neonatal hyperbilirubinemia (OR = 0.242; 95% CI 0.102–0.574). Furthermore, compared to allele C, allele T was associated with a reduced likelihood of NHB onset (OR = 0.361; 95% CI: 0.165–0.794). However, after adjusting for clinical factors, allele T exhibited a lower risk of NHB than allele C, with an OR of 0.413 (95% CI 0.141–1.211; *P* = 0.107).

### Table 4.

| SNP | Crude |  | Adjusted |  |
| --- | --- | --- | --- | --- |
| OR (95% CI) | P-value | OR (95% CI) | P-value |  |
| rs11888492 |  |  |  |  |
| CC | Ref |   | Ref |   |
| CG | 0.681 (0.259,1.788) | 0.435 | 0.528 (0.143,1.953) | 0.339 |
| GG | – | – | – | – |
| C | Ref |   | Ref |   |
| G | 0.363 (0.169, 0.777) | 0.009 | 0.189 (0.067, 0.534) | 0.002 |
| rs4148325 |  |  |  |  |
| CC | Ref |   | Ref |   |
| CT | 0.242 (0.102,0.574) | 0.001 | 0.198 (0.056,0.697) | 0.012 |
| TT | – | – | – | – |
| C | Ref |   | Ref |   |
| T | 0.361 (0.165, 0.794) | 0.011 | 0.413 (0.141, 1.211) | 0.107 |

Table 4 Caption: Relationship between the genotype and allele frequency and hyperbilirubinemia.

### Correlation between genotype and total bilirubin level

As depicted in Fig. [1](#Fig1), newborns with the CC genotype at rs11888492 had significantly higher total bilirubin levels than those with the GG genotype (*P* = 0.034). Similarly, neonates carrying the CC genotype of rs4148325 had significantly higher total bilirubin levels than their CT counterparts (*P* = 0.003).

### Fig. 1.

![Fig. 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/009a/11519535/64251e592186/41598_2024_77499_Fig1_HTML.jpg)

the relationship between SNPs genotype and TBIL, *P < 0.05, **P < 0.01.

### Cumulative impact of UGT1A1 variants on neonatal hyperbilirubinemia

The aggregate influence of concurrent UGT1A1 variants was evaluated by examining the correlation between the quantity of variations and the likelihood of developing neonatal jaundice. The adjusted OR and 95% CI for one variant was 0.244 (95% CI 0.088–0.678; *P* = 0.007). The adjusted OR and 95% CI for two variants was 0.079 (95% CI 0.005–1.232; *P* = 0.07) (Table [5](#Tab5)). The OR exhibited an inverse correlation with the accumulative count of the two variants within individuals. As shown in Fig. [2](#Fig2), an increase in the number of genetic variations was associated with lower total bilirubin levels. Compared with the OR value for individuals with no variants, the OR value for those with one variant was statistically significant (*P* < 0.01).

### Table 5.

| Number of genetics variants | Control group (N = 114) | NHB group (N = 126) | OR | 95% CI | P-value |
| --- | --- | --- | --- | --- | --- |
| 0 | 64 | 101 | 1 | – | – |
| 1 | 46 | 24 | 0.244 | (0.088,0.678) | 0.007 |
| 2 | 4 | 1 | 0.079 | (0.005,1.232) | 0.07 |

Table 5 Caption: Adjusted ORs and 95% CI of hyperbilirubinemia associated with the number of genetic variation.

### Fig. 2.

![Fig. 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/009a/11519535/9413ca6f388a/41598_2024_77499_Fig2_HTML.jpg)

Total bilirubin values amongst different numbers of genetic variants, **P < 0.01.

## Discussion

Bilirubin metabolism is predominantly mediated by uridine diphosphate glucuronosyltransferase 1A1 (UGT1A1), which aids in glucuronidation and thereby facilitates bilirubin’s removal from the body. UGT1A1, located on chromosome 2 (2q37), covers an area of approximately 160 kbp^11–14^. Multiple genetic polymorphisms within the UGT1A1 gene, including rs8175347 [UGT1A1*28, − 53 A(TA)6TAA > A(TA)7TAA], rs4148323(UGT1A1*6, 211 G > A, Gly71Arg), rs4124874, and rs10929302, are known to influence gene expression, enzyme function, and serum bilirubin concentrations^15–19^. It is widely recognized that the − 53 A(TA)7TAA/A(TA)7TAA and 211AA genotypes predominantly contribute to Gilbert’s syndrome among populations in East Asia. Within this demographic, the 211AA genotype is primarily responsible for causing neonatal hyperbilirubinemia. In contrast, the − 53 A(TA)7TAA/A(TA)7TAA variant tends to provide a protective benefit against the development of hyperbilirubinemia in breastfed infants^17,18^. Oussalah^19^ identified rs4148323, located in the first exon’s substrate-binding domain coding region of the UGT1A1 gene, as a low-frequency and rare missense variant. This variant is in strong linkage disequilibrium with rs4148325 and is highly associated with serum bilirubin concentration. To explore the effects of more novel and less studied polymorphic sites on neonatal hyperbilirubinemia, we investigated nine UGT1A1 and five BLVRA variants.We noted statistically significant disparities in the distribution of the alleles and genotypes of rs11888492(2:233771328 C > G)and rs4148325 (2:233764663C>T) of UGT1A1 between the control and NHB groups. However, no clear relationship was observed between other variants and neonatal hyperbilirubinemia.

For rs11888492, our findings revealed that the frequencies of the CC genotype and C allele were more prevalent within the NHB cohort than in control subjects (*P* = 0.001 and *P* = 0.023, respectively). The G allele was linked to a decreased likelihood of NHB incidence (OR = 0.363; 95% CI: 0.169–0.777), while allele C increased the risk. Neonates carrying the GG genotype had significantly lower total bilirubin levels than those carrying the CC genotype. This suggests that the G allele of rs11888492 is associated with a reduced risk of neonatal hyperbilirubinemia, indicating potential biological significance in regulating bilirubin metabolism. This association could be ascribed to the variant’s impact on the splicing, transcription, or translation efficiency of the UGT1A1 gene, which could alter the function or stability of the enzyme. Research on rs11888492 is limited; according to our full-text searches in various databases, only two articles on this variant have been reported^14,20^. Zhai et al.^14^ previously observed no correlation between rs11888492 and serum TBIL levels in patients with sepsis. This discrepancy may be due to the differing study populations; our research focuses on newborns with hyperbilirubinemia, while Hyland^20^ reported that rs11888492 is not associated with esophageal squamous cell carcinoma.

For rs4148325, our investigation revealed that the genotype CC and allele C frequencies were more prevalent within the NHB cohort than in the control group (*P* = 0.001 and *P* = 0.002, respectively). When comparing neonates with the CC genotype to those with the CT genotype, there is a notably reduced risk associated with the CT genotype for the development of NHB (OR = 0.242; 95% CI 0.102–0.574). Newborns carrying the CT genotype exhibited significantly lower bilirubin levels than those with the CC genotype. The rs4148325 is located in both the proximal promoter region and intron 1 of the UGT1A1 gene. It lies very close to the UGT1A1*28 TATA box polymorphism and has a strong linkage with it, sharing the same haplotype block in our population^21^. This SNP is significantly associated with serum bilirubin levels, possibly because it affects UGT1A1 gene expression, thereby influencing bilirubin metabolism^22^. However, our findings differ from those reported in other studies that found that rs4148325 was closely linked to the highest levels of bilirubin elevation in patients receiving sarilumab treatment, likely indicating Gilbert syndrome or disorders of bilirubin excretion^23,24^. The observed inconsistency could be due to different study populations, different diseases, or the limited size of our sample group and the failure to consider the cumulative effects of numerous genetic variations in our analytical approach.

As far as we are aware, our research is pioneering in examining the interactive influence of UGT1A1 genetic variants rs11888492 and rs4148325 on neonatal hyperbilirubinemia. We found that these variants were protective factors against the development of neonatal hyperbilirubinemia, with total bilirubin levels decreasing with an increase in the number of genetic variations.

BLVRA is another important bilirubin metabolism gene responsible for the transformation of biliverdin into bilirubin. Genetic variations within the BLVRA gene may also affect TBIL levels^25^. Although genetic variations in BLVRA have been associated with unconjugated hyperbilirubinemia in adults^26^, the variant rs699512 did not show a substantial impact hyperbilirubinemia in our findings. Similarly, the correlation between rs7738 and hyperbilirubinemia remains controversial. One study demonstrated an association between rs7738 in the BLVRA and TBIL levels in coronary artery disease cases within the Chinese population^27^. However, Chiddarwar et al.^28^ observed an increased frequency in the rs7738 variant allele among individuals with hyperbilirubinemia in comparison with the control cohort, although this discrepancy lacked statistical significance (*P* < 0.14). Our study yielded results similar to those reported by Chiddarwar et al.^28^ regarding rs7738. Additionally, we observed that other variants, including rs10486752, rs1181576, and rs2730625, had no discernible effect on neonatal hyperbilirubinemia. Notably, our study marks an initial inquiry into the effects of rs1181576 and rs2730625 on neonatal hyperbilirubinemia.

This study is subject to certain constraints, including a relatively limited sample size and assessment of only 14 gene loci that potentially affect bilirubin metabolism. Neonatal hyperbilirubinemia is a complex issue influenced by multiple factors, and additional research with larger sample sizes is necessary to investigate gene–gene interactions and offer a more thorough understanding of its pathogenesis.

## Conclusions

Our study provides genetic evidence suggesting an association between the UGT1A1 rs11888492 and rs4148325 variants and neonatal hyperbilirubinemia among Chinese infants. We noted substantial distinctions in the distribution of alleles and genotypes for these variations when comparing the control group with the NHB cohort. Our findings suggest that these variants may serve as protective factors against neonatal hyperbilirubinemia, with total bilirubin levels decreasing with an increase in the number of genetic variations. However, additional studies employing more extensive sample populations is warranted to explore gene–gene interactions and gain an enhanced comprehension of the disease mechanisms behind neonatal hyperbilirubinemia. The results of this investigation could offer valuable understanding for the clinical management and drug development for this disease.

## Materials and methods

### Study participants

A total of 240 neonatal samples were gathered from the Longgang Maternal and Child Health Hospital in Shenzhen from March 2021 to March 2022. This study included 126 patients diagnosed with hyperbilirubinemia (neonatal hyperbilirubinemia group, NHB group, TBIL ≥ 257.0 µmol/L) and 114 healthy newborns (control group, TBIL < 257.0 µmol/L). The American Academy of Pediatrics established that hyperbilirubinemia was diagnosed when the serum bilirubin concentrations were at or above the 95th percentile on the neonatal hour-specific bilirubin chart. According to the Expert Consensus on the Diagnosis and Treatment of Neonatal Hyperbilirubinemia (2014), intervention is recommended for infants with a gestational age of 35 weeks or more when serum TBIL levels reach or exceed 257 µmol/L. Individuals with conditions known to affect serum total bilirubin levels, such as premature birth, asphyxia, septicemia, hepatitis, bile duct obstruction, hemolytic thalassemia, and glucose-6-phosphate dehydrogenase (G6PD) deficiency due to ABO and Rh blood group incompatibility, were excluded from both groups. The study received ethical clearance from the Longgang Maternal and Child Health Hospital Ethics Committee (Shenzhen, China)(LGFYYXLLQ-2020-023), and obtained informed consent from the participants involved in this study. All the methods in this study were strictly carried out according to the methods approved by the ethics committee.

### Biochemistry analysis of blood

Venous blood samples (2–3 mL) were obtained and the serum levels of total protein (TP), albumin (ALB), bilirubin (total and direct), urea nitrogen (BUN), aspartate transaminase (AST), alanine transaminase (ALT), and creatinine (CR) quantified using an automatic biochemical analyzer (Hitachi 7180, Japan).

### Analysis of polymorphism

All remaining blood samples from routine blood tests were subjected to genetic polymorphism analysis using the MassARRAY^®^ MALDI-TOF System (Sequenom, America). Nine single nucleotide polymorphisms (SNPs) of UGT1A1 were investigated: rs1018124, rs1042640, rs11888492, rs12479045, rs17864705, rs3771342, rs4148325, rs4148328, and rs8330. Additionally, five BLVRA SNPs were analyzed: rs10486752, rs1181576, rs2730625, rs699512, and rs7738.

### Statistical analysis

Allele and genotype frequencies were assessed through direct enumeration. Differences between control groups and neonatal hyperbilirubinemia were analyzed with the chi-squared test. The data for continuous variables are presented as averages accompanied by the standard deviation (SD), with statistical analysis conducted through the application of Student’s t-test or analysis of variance (ANOVA), as appropriate.To examine the relationship between NHB and the comparison groups, binary logistic regression was employed to derive odds ratios (ORs) and associated 95% confidence intervals (CIs), adjusting for clinical covariates, such as AST, TP, ALB, BUN, and CR. Additionally, the relationship between total bilirubin (TBIL) levels and various genotypic combinations was investigated. The statistical evaluations were carried out using GraphPad Prism 8.3.0, recognizing results as statistically significant when *P* < 0.05.

## Acknowledgements

We extend our gratitude to the Clinical Laboratory of Longgang Maternity and Child Health Institute affiliated with Shantou University Medical College, Beijing Chest Hospital’s Key Laboratory for Drug-Resistant Tuberculosis Research, and the Cardiology Department of the First Affiliated Hospital of Shantou University Medical College for their contributions to this research effort.

## Author contributions

The research presented here in is the result of a collaborative effort among the co-authors. Each author has reviewed and consented to the submission of the manuscript in its final form. ZhenWen Zhou: conducted the experiments. XiuJu Liu: conceived the experiments and wrote the manuscript. Chao Zhang: analysed the results and prepared Tables [1](#Tab1), [2](#Tab2) and [3](#Tab3); Figs. [1](#Fig1) and [2](#Fig2). LiWen Chen: analysed the results and prepared Table [4](#Tab4), and [5](#Tab5). ShuYan Liu: conceived the experiments.

## Funding

This study was funded by the Longgang District medical scientific research project (joint project of Shenzhen Longgang District Science and Technology Innovation Bureau) (LGKCYLWS2020107). The sponsor had no involvement in the study design, data gathering and analysis, or the writing of the manuscript.

## Data availability

The data are available from the corresponding author on reasonable request.

## Declarations

### Competing interests

The authors declare no competing interests.

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Data Availability Statement

The data are available from the corresponding author on reasonable request.

### Data Availability Statement

The data are available from the corresponding author on reasonable request.

## References

1. Atasilp, C. et al. Associations between UGT1A1 and SLCO1B1 polymorphisms and susceptibility to neonatal hyperbilirubinemia in Thai population. BMC Pediatr.22, 243 (2022).  [DOI](https://doi.org/10.1186/s12887-022-03311-4) | [PMC free article](/articles/PMC9059389/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35501760/)

2. Li, Y., Wu, T., Chen, L. & Zhu, Y. Associations between G6PD, OATP1B1 and BLVRA variants and susceptibility to neonatal hyperbilirubinaemia in a Chinese Han population. J. Paediatr. Child Health55, 1077–1083 (2019).  [DOI](https://doi.org/10.1111/jpc.14346) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30636082/)

3. Yang, H. et al. Clinical significance of UGT1A1 genetic analysis in Chinese neonates with severe hyperbilirubinemia. Pediatr. Neonatol.57, 310–317 (2016).  [DOI](https://doi.org/10.1016/j.pedneo.2015.08.008) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26727668/)

4. Fan, J. et al. Associations between UGT1A1, SLCO1B1, SLCO1B3, BLVRA and HMOX1 polymorphisms and susceptibility to neonatal severe hyperbilirubinemia in Chinese Han population. BMC Pediatr.24, 82 (2024).  [DOI](https://doi.org/10.1186/s12887-024-04537-0) | [PMC free article](/articles/PMC10811893/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/38279097/)

5. Morais, M. B. & Machado, M. V. Benign inheritable disorders of bilirubin metabolism manifested by conjugated hyperbilirubinemia–A narrative review. United European Gastroenterol. J.10, 745–753 (2022).  [DOI](https://doi.org/10.1002/ueg2.12279) | [PMC free article](/articles/PMC9486497/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35860851/)

6. Yang, H. et al. UGT1A1 variants in Chinese Uighur and Han newborns and its correlation with neonatal hyperbilirubinemia. PLOS ONE17, e0279059 (2022).  [DOI](https://doi.org/10.1371/journal.pone.0279059) | [PMC free article](/articles/PMC9754166/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36520959/)

7. Wisnumurti, D. A. et al. UGT1A1 genetic variations and a haplotype associated with neonatal hyperbilirubinemia in Indonesian population. BioMed Res. Int.2018, 9425843 (2018).  [DOI](https://doi.org/10.1155/2018/9425843) | [PMC free article](/articles/PMC5828093/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29607327/)

8. Deng, H., Zhou, Y., Wang, L. & Zhang, C. Ensemble learning for the early prediction of neonatal jaundice with genetic features. BMC Med. Inform. Decis. Mak.21, 338 (2021).  [DOI](https://doi.org/10.1186/s12911-021-01701-9) | [PMC free article](/articles/PMC8638201/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34852805/)

9. de Souza, M. M. T. et al. UGT1A1*28 relationship with abnormal total bilirubin levels in chronic hepatitis C patients: Outcomes from a case-control study. Medicine (Baltimore)96, e6306 (2017).  [DOI](https://doi.org/10.1097/MD.0000000000006306) | [PMC free article](/articles/PMC5369894/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28296739/)

10. Nguyen, T. T., Zhao, W., Yang, X. & Zhong, D. N. The relationship between hyperbilirubinemia and the promoter region and first exon of UGT1A1 gene polymorphisms in Vietnamese newborns. Pediatr. Res.88, 940–944 (2020).  [DOI](https://doi.org/10.1038/s41390-020-0825-6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32126570/)

11. Bai, J. et al. Combined effects of UGT1A1 and SLCO1B1 variants on Chinese adult mild unconjugated hyperbilirubinemia. Front. Genet.10, 1073 (2019).  [DOI](https://doi.org/10.3389/fgene.2019.01073) | [PMC free article](/articles/PMC6834774/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31737051/)

12. Mi, X. X. et al. Analysis of the UGT1A1 genotype in hyperbilirubinemia patients: Differences in allele frequency and distribution. BioMed Res. Int.2019, 6272174 (2019).  [DOI](https://doi.org/10.1155/2019/6272174) | [PMC free article](/articles/PMC6699345/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31467903/)

13. Zhang, X. et al. Clinical manifestations of neonatal hyperbilirubinemia are related to alterations in the gut microbiota. Children (Basel)9, 764 (2022).  [DOI](https://doi.org/10.3390/children9050764) | [PMC free article](/articles/PMC9139945/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35626941/)

14. Zhai, R. et al. Serum bilirubin levels on ICU admission are associated with ARDS development and mortality in sepsis. Thorax64, 784–790 (2009).  [DOI](https://doi.org/10.1136/thx.2009.113464) | [PMC free article](/articles/PMC2735615/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19482841/)

15. Li, Z., Song, L. & Hao, L. The role of UGT1A1 (c.-3279 T > G) gene polymorphisms in neonatal hyperbilirubinemia susceptibility. BMC Med. Genet.21, 218 (2020).  [DOI](https://doi.org/10.1186/s12881-020-01155-2) | [PMC free article](/articles/PMC7648392/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33158427/)

16. Xu, J. X. et al. Co-inheritance of G6PD deficiency and 211G to a variation of UGT1A1 in neonates with hyperbilirubinemia in eastern Guangdong. BMC Pediatr.21, 564 (2021).  [DOI](https://doi.org/10.1186/s12887-021-03010-6) | [PMC free article](/articles/PMC8665559/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34895177/)

17. Huang, M. J. et al. Bilirubin metabolism and UDP-glucuronosyltransferase 1A1 variants in Asians: Pathogenic implications and therapeutic response. Kaohsiung J. Med. Sci.38, 729–738 (2022).  [DOI](https://doi.org/10.1002/kjm2.12579) | [PMC free article](/articles/PMC11896460/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35942604/)

18. 18. Hung, Y.L. et al. Molecular biology of glucose-6-phosphate dehydrogenase and UDP-glucuronosyltransferase 1A1 in the development of neonatal unconjugated hyperbilirubinemia. Pediatr Neonatol. 65, 419–426(2024).  [DOI](https://doi.org/10.1016/j.pedneo.2024.02.003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/38480019/)

19. Oussalah, A. et al. Exome-wide association study identifies new low frequency and rare UGT1A1 coding variants and UGT1A6 coding variants influencing serum bilirubin in elderly subjects: A strobe compliant article. Med. (Baltim.). 94,e925 (2015).  [DOI](https://doi.org/10.1097/MD.0000000000000925) | [PMC free article](/articles/PMC4616369/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26039129/)

20. Hyland, P. L. et al. Genetic variants in sex hormone metabolic pathway genes and risk of esophageal squamous cell carcinoma. Carcinogenesis34, 1062–1068 (2013).  [DOI](https://doi.org/10.1093/carcin/bgt030) | [PMC free article](/articles/PMC3643422/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23358850/)

21. Johnson, A. D. et al. Genome-wide association meta-analysis for total serum bilirubin levels. Hum. Mol. Genet.18, 2700–2710 (2009).  [DOI](https://doi.org/10.1093/hmg/ddp202) | [PMC free article](/articles/PMC2701336/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19414484/)

22. Jylhävä, J. et al. A genome-wide association study identifies UGT1A1 as a regulator of serum cell-free DNA in young adults: The cardiovascular risk in young Finns study. PLoS One7, e35426 (2012).  [DOI](https://doi.org/10.1371/journal.pone.0035426) | [PMC free article](/articles/PMC3325226/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22511988/)

23. Lin, N. et al. UGT1A1 genetic variants are associated with increases in bilirubin levels in rheumatoid arthritis patients treated with sarilumab. Pharmacogenomics J.22, 160–165 (2022).  [DOI](https://doi.org/10.1038/s41397-022-00269-5) | [PMC free article](/articles/PMC9151390/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35149777/)

24. Zawistowski, M. et al. The Michigan Genomics Initiative: A biobank linking genotypes and electronic clinical records in Michigan Medicine patients. Cell Genomics3, 100257 (2023).  [DOI](https://doi.org/10.1016/j.xgen.2023.100257) | [PMC free article](/articles/PMC9932985/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36819667/)

25. 25. Lin, R. et al. Association of a BLVRA common polymorphism with essential hypertension and blood pressure in Kazaks. Clin. Exp. Hypertens.33, 294–298 (2011).  [DOI](https://doi.org/10.3109/10641963.2010.531854) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21721974/)

26. Nytofte, N. S. et al. A homozygous nonsense mutation (c.214C->A) in the biliverdin reductase alpha gene (BLVRA) results in accumulation of biliverdin during episodes of cholestasis. J. Med. Genet.48, 219–225 (2011).  [DOI](https://doi.org/10.1136/jmg.2009.074567) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21278388/)

27. Lin, R. et al. Common variants of four bilirubin metabolism genes and their association with serum bilirubin and coronary artery disease in Chinese Han population. Pharmacogenet. Genomics19, 310–318 (2009).  [DOI](https://doi.org/10.1097/FPC.0b013e328328f818) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19238116/)

28. Chiddarwar, A. S., D’Silva, S. Z., Colah, R. B., Ghosh, K. & Mukherjee, M. B. Genetic variations in bilirubin metabolism genes and their association with unconjugated hyperbilirubinemia in adults. Ann. Hum. Genet.81, 11–19 (2017).  [DOI](https://doi.org/10.1111/ahg.12179) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27943244/)
