# Estimating the In Vivo Function of CYP2D6 Alleles through Population Pharmacokinetic Modeling of Brexpiprazole

## Metadata
**Authors:** Trine Frederiksen, Johan Areberg, Arash Raoufinia, Ellen Schmidt, Tore Bjerregaard Stage, Kim Brøsen
**Journal:** Clinical Pharmacology and Therapeutics
**Date:** 2022 Nov 29
**DOI:** [10.1002/cpt.2791](https://doi.org/10.1002/cpt.2791)
**PMID:** 36350097
**PMCID:** PMC10099095
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10099095/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC10099095/pdf/CPT-113-360.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC10099095/pdf/CPT-113-360.pdf)

## Abstract

Accurate prediction of CYP2D6 phenotype from genotype information is important to support safe and efficacious pharmacotherapy with CYP2D6 substrates. To facilitate accurate CYP2D6 genotype–phenotype translation, there remains a need to investigate the enzyme activity associated with individual CYP2D6 alleles using large clinical data sets. This study aimed to quantify and compare the in vivo function of different CYP2D6 alleles through population pharmacokinetic (PopPK) modeling of brexpiprazole using data from 13 clinical studies. A PopPK model of brexpiprazole and its two metabolites, DM‐3411 and DM‐3412, was developed based on plasma concentration samples from 826 individuals. As the minor metabolite, DM‐3412, is formed via CYP2D6, the metabolic ratio of DM‐3412:brexpiprazole calculated from the PopPK parameter estimates was used as a surrogate measure of CYP2D6 activity. A CYP2D6 genotype–phenotype analysis based on 496 subjects showed that the CYP2D6*2 allele (n = 183) was associated with only 10% enzyme activity relative to the wild‐type allele (CYP2D6*1) and a low enzyme activity was consistently observed across genotypes containing CYP2D6*2. Among the decreased function alleles, the following enzyme activities relative to CYP2D6*1 were estimated: 23% for CYP2D6*9 (n = 20), 32% for CYP2D6*10 (n = 62), 64% for CYP2D6*14 (n = 1), 4% for CYP2D6*17 (n = 37), 4% for CYP2D6*29 (n = 13), and 9% for CYP2D6*41 (n = 64). These findings imply that a lower functional value would more accurately reflect the in vivo function of many reduced function CYP2D6 alleles in the metabolism of brexpiprazole. The low enzyme activity observed for CYP2D6*2, which has also been reported by others, suggests that the allele exhibits substrate‐specific enzyme activity.

**WHAT IS THE CURRENT KNOWLEDGE ON THE TOPIC?**

Accurate *CYP2D6* genotype–phenotype translation is important for pharmacotherapy, but *in vivo* evidence supporting functional assignment of individual *CYP2D6* alleles is sparse.

**WHAT QUESTION DID THIS STUDY ADDRESS?**

The objective of the study was to quantify and compare the *in vivo* CYP2D6 activity associated with different *CYP2D6* alleles through population pharmacokinetic modeling of brexpiprazole and its two metabolites.

**WHAT DOES THIS STUDY ADD TO OUR KNOWLEDGE?**

The study provides new evidence of the *in vivo* function of decreased function alleles with respect to brexpiprazole metabolism. Furthermore, it showed a reduced enzyme function associated with *CYP2D6*2*, which is normally considered fully functional.

**HOW MIGHT THIS CHANGE CLINICAL PHARMACOLOGY OR TRANSLATIONAL SCIENCE?**

The results offer valuable information to support ongoing discussions on optimizing *CYP2D6* genotype–phenotype translation.

Cytochrome P450 2D6 (CYP2D6) is involved in the metabolism of > 20% of clinically used drugs.[1](#cpt2791-bib-0001) As the *CYP2D6* gene is highly polymorphic, individuals exhibit varying degrees of CYP2D6 enzyme activity, which may affect the safety and efficacy of drugs cleared and/or activated by CYP2D6. The clinical importance of interindividual differences in CYP2D6 activity is evident from the large number of drug labels with annotations for CYP2D6[2](#cpt2791-bib-0002) and several clinical guidelines providing recommendations for CYP2D6‐guided drug therapy.[3](#cpt2791-bib-0003), [4](#cpt2791-bib-0004), [5](#cpt2791-bib-0005), [6](#cpt2791-bib-0006)

Treatment recommendations are commonly based on patients’ CYP2D6 phenotype classification as a poor metabolizer (PM), intermediate metabolizer (IM), normal metabolizer (NM), or ultra‐rapid metabolizer (UM). However, the translation of *CYP2D6* genotypes into the four phenotype categories remains ambiguous and particularly the functional assignment of decreased function alleles is heavily debated.[7](#cpt2791-bib-0007) Experts in the field argue that an activity score of 0.5 overestimates the activity conveyed by certain decreased function alleles. A recent guideline supported by the Clinical Pharmacogenetics Implementation Consortium (CPIC) recommended to downgrade the activity score of *CYP2D6*10* to 0.25, whereas other researchers argue that *CYP2D6*41* is associated with a stronger reduction in enzyme function and advocate for lowering the activity score of this allele.[7](#cpt2791-bib-0007), [8](#cpt2791-bib-0008)

One of the challenges for assigning activity scores to individual *CYP2D6* alleles is that designing clinical studies with sufficient power (i.e., adequate sample size of individual genotypes/alleles) to evaluate allele‐specific CYP2D6 activity is difficult. Furthermore, some studies suggest that certain *CYP2D6* alleles exhibit substrate‐specific behavior which complicates genotype–phenotype prediction even further.[9](#cpt2791-bib-0009), [10](#cpt2791-bib-0010), [11](#cpt2791-bib-0011)

Only a limited number of studies have quantified and compared the CYP2D6 activity associated with individual *CYP2D6* alleles in large clinical data sets. To support *CYP2D6* genotype–phenotype translation, more clinical studies with a sufficient sample size and high‐quality phenotype data of different CYP2D6 substrates are warranted.[12](#cpt2791-bib-0012)

In two previous studies, we pooled data from clinical studies of CYP2D6 substrates, vortioxetine and tedatioxetine, for two population pharmacokinetic (PopPK) meta‐analyses.[13](#cpt2791-bib-0013), [14](#cpt2791-bib-0014) Using a modeling approach, we were able to estimate the CYP2D6‐mediated clearance of a large number of individuals with different *CYP2D6* genotypes. Similar investigations of other CYP2D6 substrates could contribute to the current knowledge of *CYP2D6* genotype–phenotype relationships and aid our understanding of substrate‐specific effects.

Brexpiprazole is a drug indicated for the treatment of schizophrenia and as adjunctive therapy to antidepressants for the treatment of major depressive disorder (MDD).[15](#cpt2791-bib-0015) Brexpiprazole acts as a partial agonist at serotonin 5‐HT_1A_ and dopamine D_2_ receptors, and as an antagonist at serotonin 5‐HT_2A_ and noradrenaline α_1B_/α_2C_ receptors, all with subnanomolar affinity.

The main enzymes involved in the metabolism of brexpiprazole are CYP3A4 and CYP2D6 with minor contributions from other enzymes.[16](#cpt2791-bib-0016), [17](#cpt2791-bib-0017) The biotransformation of brexpiprazole into the major metabolite, DM‐3411, is mainly mediated by CYP3A4 and CYP2D6, whereas the formation of the minor metabolite, DM‐3412, is primarily mediated via CYP2D6 (**Figure** [1 a](#cpt2791-fig-0001)).[17](#cpt2791-bib-0017) Neither of the metabolites are believed to contribute to the pharmacological effect of brexpiprazole.[17](#cpt2791-bib-0017) According to *in vitro* studies, brexpiprazole exhibits little or no inhibition of CYP enzymes and it is not a substrate of efflux transporters, such as P‐glycoprotein (P‐gp) or breast cancer resistance protein (BCRP).[16](#cpt2791-bib-0016)

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/89a0/10099095/d615deb4506a/CPT-113-360-g002.jpg)

(a) Biotransformation of brexpiprazole, (b) Structural model for brexpiprazole and the two metabolites DM‐3411 and DM‐3412. CLM1, clearance from brexpiprazole to DM‐3412; CLM2, clearance from brexpiprazole to DM‐3411; CLMET1, DM‐3412 clearance; CLMET2, DM‐3411 clearance; k a, absorption rate constant; Q, brexpiprazole intercompartmental clearance; QMET, DM‐3412 intercompartmental clearance; V2, volume of distribution brexpiprazole central compartment; V3, volume of distribution brexpiprazole peripheral compartment; V4, volume of distribution DM‐3412 central compartment; V5, volume of distribution DM‐3412 peripheral compartment; V6, volume of distribution DM‐3411.

In the current study, brexpiprazole was used as a tool compound to estimate the CYP2D6 activity of a large number of subjects using PopPK modeling. As the minor metabolite, DM‐3412, is primarily formed via CYP2D6, the metabolic ratio of DM‐3412:brexpiprazole was used as a biomarker of CYP2D6 activity. The aim of this study was to quantify the CYP2D6 activity attributable to different *CYP2D6* genotypes and alleles using estimates from a PopPK model of brexpiprazole and its two metabolites.

## METHODS

### Study design

Data from 13 clinical studies conducted in the United States, Europe, and Asia were pooled for a PopPK meta‐analysis. An overview of the clinical studies is provided in **Table** [S1](#cpt2791-supitem-0001) of the **Supplementary Material**. All clinical studies were conducted in accordance with the principles of Good Clinical Practice and the Declaration of Helsinki. The clinical studies were all approved by local ethics committees and all subjects provided informed consent prior to study‐related procedures.

The clinical studies comprised nine phase I studies and four phase II studies, including either healthy subjects or patients diagnosed with MDD, attention deficit hyperactivity disorder, or schizophrenia. Subjects and patients were administered oral doses of 0.15–12 mg brexpiprazole either as monotherapy (*N* = 541) or as adjunctive treatment to other therapeutics (*N* = 285; see **Table** [S1](#cpt2791-supitem-0001)). Patients on concomitant treatment with moderate or strong CYP2D6 inhibitors (fluoxetine, paroxetine, or sertraline) were not included in the PopPK analysis. Concomitant use of CYP3A4 inducers and inhibitors was disallowed in all the clinical studies.

Plasma samples were collected in all studies using rich (>20 samples per individual; *N* = 245), semi‐sparse (8 samples per individual; *N* = 314), or sparse (3–4 samples per individual; *N* = 267) sampling schemes. The plasma samples were analyzed for concentrations of brexpiprazole and the two metabolites, DM‐3411 and DM‐3412, using high‐performance liquid chromatography with tandem mass spectrometric detection. The analytical method was linear over the range of 0.300 to 100 ng/mL for all analytes. Samples with a concentration below the lower limit of quantification (LLOQ; 0.300 ng/mL) were recorded in the dataset as “0.00” ng/mL and flagged as being below the LLOQ.

### Genotyping

*CYP2D6* genotyping was performed in all studies. Genomic DNA was extracted from whole blood samples and tested for different *CYP2D6* allele variants using TaqMan, DNA sequencing, and gel‐based assay technologies. The majority of the subjects (*N* = 737) were genotyped using an assay allowing identification of the following *CYP2D6* allele variants: **2*, **3*, **4*, **5*, **6*, **7*, **9*, **10*, **17*, **21*, **29*, and **41* and gene duplication. Two single ascending dose studies conducted in Japan and Korea (*N* = 89) used a *CYP2D6* genotyping assay testing for presence of the following allele variants: **2*, **4*, **5*, **10*, **14*, **18*, **21*, and **41* and gene duplication. In both assays, the wild‐type allele (*CYP2D6*1*) was assigned when no allele variants were detected.

### Population pharmacokinetic modeling

A PopPK model describing the joint pharmacokinetics (PK) of brexpiprazole and the two metabolites, DM‐3411 and DM‐3412, was developed using nonlinear mixed effects modeling in NONMEM (ICON Development Solutions, version 7.4).

Initial exploratory PopPK analyses of the three compounds individually showed that the PK for brexpiprazole was best described by a two‐compartment model with first‐order absorption and elimination, whereas DM‐3411 and DM‐3412 exhibited one‐ and two‐compartment disposition, respectively. Accordingly, the joint PopPK model initially tested consisted of 6 structural compartments including central and peripheral compartments for brexpiprazole and DM‐3412 and a central compartment for DM‐3411, as illustrated in **Figure** [1 b](#cpt2791-fig-0001).

During model development, it was tested whether additional or fewer compartments improved the model fit and different absorption models, including lag‐time absorption and transit absorption compartments, were evaluated.

Interindividual variability (IIV) of model parameters was modeled using exponential error terms and covariance between selected model parameters was tested. The residual unexplained variability (RUV) was modeled separately for each of the three compounds.

Different diagnostic tools were used to evaluate each candidate model, including the objective function value (OFV), Akaike information criterion (AIC), condition number, and goodness‐of‐fit plots. Furthermore, models were assessed by successful minimization, completion of the covariance step, and acceptable precision of parameter estimates measured by the relative standard error (%RSE).

Covariate relationships to be evaluated were selected based on knowledge from a previous PopPK analysis performed as part of the clinical development of brexpiprazole, ETA plots, and biological plausibility. The following covariates were tested in the covariate analysis: body weight, height, body‐mass index (BMI), age, gender, dose, alanine aminotransferase, creatinine clearance, and food effect.

Covariates were added to the model in a stepwise manner if they resulted in a > 6.64‐point reduction of the OFV, which corresponds to a *P* value < 0.01 for one degree of freedom. When no further covariates resulted in model improvement, the forward‐inclusion procedure was followed by backward‐elimination where each covariate was removed from the model one‐by‐one. If removal of a covariate resulted in an increase in OFV of > 10.83 points (corresponding to *P* < 0.001 for one degree of freedom), the covariate was included in the final model.

For the final model, visual predictive check (VPC) plots were generated. Furthermore, a nonparametric bootstrap analysis was performed to estimate the precision of the final model parameters. Due to a long runtime for the final PopPK model, the bootstrap analysis was only based on 100 samples.

### Estimating CYP2D6 activity

The CYP2D6 activity of each subject in the data set was estimated as the extent of formation of the minor metabolite, DM‐3412, derived from the individual PopPK model parameter estimates. As a surrogate marker of CYP2D6 activity, a metabolic ratio (MR) of the exposure of brexpiprazole (BREX) relative to DM‐3412 was estimated from the area under the curves (AUCs) using the formula:

| MR=AUCDM3412AUCBREX=dose·FDM3412/CLDM3412dose/CLBREX=FDM3412·CLBREXCLDM3412=FDM3412·CLM1+CLM2CLMET1 |
| --- |
where F_DM3412_ was calculated as the fraction of the total brexpiprazole clearance that was attributed to formation of DM‐3412:

| FDM3412=CLM1CLM1+CLM2 |
| --- |
Thereby, the formula for calculation of the MR could be further simplified to:

| MR=CLM1CLMET1 |
| --- |
### Calculation of activity scores

The contribution of individual *CYP2D6* alleles to CYP2D6 activity was estimated by multiple linear regression analysis. *CYP2D6* genotypes were coded using indicator variables reflecting the number of each variant allele in the genotype. These were used as predictors in the multiple linear regression and the log‐transformed MR was the outcome variable.

Different multiple linear regression models were tested to identify the best model describing the *CYP2D6* genotype–phenotype relationship. It was tested whether pooling the alleles catalogued as null function (*CYP2D6*3*, *CYP2D6*4*, *CYP2D6*5*, *CYP2D6*6*, *CYP2D6*7*, and *CYP2D6*21*)[18](#cpt2791-bib-0018) and those catalogued as fully functional (*CYP2D6*1* and *CYP2D6*2*),[18](#cpt2791-bib-0018) respectively, improved the model fit. The regression models were evaluated and compared using residual plots, AIC, and analysis of variance.

The coefficients (β) from the final regression model were used to calculate a CYP2D6 activity score (AS) for each allele. The AS for *CYP2D6*1* was fixed to 1, whereas the AS for the null function alleles was fixed to 0, as the latter was assumed to represent non‐CYP2D6 mediated formation of the metabolite. The AS for the remaining alleles was calculated as the relative activity to *CYP2D6*1* adjusted by the contribution from the null function alleles using the following formula:

| ASCYP2D6*X=βCYP2D6*X−βCYP2D6*nullβCYP2D6*1−βCYP2D6*null |
| --- |
Confidence intervals for each AS were estimated by nonparametric bootstrap analyses with 10,000 samples.

All statistical analyses were performed using the Open Source software environment, R (version 4.1.1), run under RStudio.

## RESULTS

### Population pharmacokinetic analysis

The PopPK analysis was based on plasma concentration data from 826 healthy subjects and patients. A summary of the subject and patient characteristics is presented in **Table** [1](#cpt2791-tbl-0001). The final PopPK analysis data set comprised a total of 8,604, 8,156 and 5,113 plasma concentrations above the LLOQ for brexpiprazole, DM‐3411, and DM‐3412, respectively.

### Table 1.

| Continuous characteristics | Population pharmacokinetic analysis |  |  |  | CYP2D6 genotype–phenotype analysis |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
| N | Median | IQR | Range | N | Median | IQR | Range |  |
| Age, years | 826 | 36 | 27–46 | 18–65 | 496 | 37 | 28–46 | 18–65 |
| Body weight, kg | 782 | 76 | 65–90 | 32–159 | 478 | 81 | 68–92 | 44–159 |
| Height, cm | 782 | 170 | 164–177 | 131–196 | 478 | 172 | 165–178 | 145–196 |
| BMI, kg/m2 | 782 | 26 | 23–30 | 16–57 | 478 | 27 | 24–31 | 17–57 |
| Creatinine clearance, mL/mina | 782 | 119 | 99‐141 | 42‐333 | 478 | 121 | 101–145 | 57–302 |
| ALAT, IU/L | 791 | 20 | 15–28 | 5–107 | 478 | 21 | 16–29 | 5–107 |

Table 1 Caption: Summary of subject characteristics

The structure of the final PopPK model is depicted in **Figure** [1 b](#cpt2791-fig-0001). The model was parameterized in terms of an absorption rate constant (*k* _a_), lag‐time (ALAG), volumes of distribution for brexpiprazole central (V2) and peripheral (V3) compartments, DM‐3412 central (V4) and peripheral (V5) compartments and DM‐3411 central (V6) compartment, two clearance parameters reflecting the clearance from brexpiprazole to DM‐3412 (CL_M1_) and DM‐3411 (CL_M2_), respectively, intercompartmental clearances for brexpiprazole (Q) and DM‐3412 (Q_MET_) and clearances for DM‐3412 (CL_MET1_) and DM‐3411 (CL_MET2_). The model included IIV terms on *k* _a_, V2, V3, V4, V6, CL_M1_, CL_M2_, V4, CL_MET1_, and CL_MET2_ and covariances among V2‐CL_M1_‐CL_M2_, V4‐CL_MET1_, and V6‐CL_MET2_. The RUV was modeled using proportional error models for all three compounds.

To stabilize the model, parameters related to the PK of brexpiprazole (*k* _a_, ALAG, V2, V3, Q, IIV_ka_, and IIV_V3_) were fixed to the final estimates from the initial PopPK model based on brexpiprazole data only.

In the covariate model development, food effect on *k* _a_ was identified as the most significant covariate relationship resulting in an OFV reduction of 281 points. In the subsequent step, the most significant covariate was BMI on V6 resulting in a 19‐point reduction in OFV. In the third step of the forward‐inclusion process, none of the tested models gave a significant reduction in OFV. The model including food effect on *k* _a_ and BMI on V6 was therefore considered the full model. In the backward‐elimination process, removal of each of the covariates resulted in a > 10.83‐point increase in OFV, and both covariates were therefore retained in the final model.

An overview of the final population parameter estimates is provided in **Table** [2](#cpt2791-tbl-0002). The bootstrap analysis indicated that most population parameters were estimated with an adequate precision, although the parameters related to DM‐3412 (CL_M1_, V4, V5, CL_MET1_, and Q_MET_) were all outside the 95% confidence intervals indicating a poorer precision of these estimates. A sensitivity analysis excluding all subjects with no measurable DM‐3412 concentrations showed great improvement in the precision of these parameters with the bootstrap 95% confidence interval capturing most of the population estimates (**Table** [S3](#cpt2791-supitem-0001)). The two clearance parameters, CL_M1_ and CL_MET1_, used to calculate the MR were estimated with an RSE < 25%. Diagnostic plots for the final PopPK model indicated a good fit of the model across all three compounds (**Figure** [S1](#cpt2791-supitem-0001)). Furthermore, the VPC plots showed a good predictive performance of the model (**Figure** [S2](#cpt2791-supitem-0001)).

### Table 2.

| Model parameter | Estimate (%RSE) | IIV (%RSE) | 95% CI |
| --- | --- | --- | --- |
| Absorption rate constant fasted | 1.02 (fixed) | 215 (fixed) | ‐ |
| Absorption rate constant fed | 0.535 (23.4) | 215 (fixed) | [0.420; 0.752] |
| Lag‐time | 0.411 (fixed) | ‐ | ‐ |
| Volume of distribution, brexpiprazole central compartment (V2) | 81.2 (fixed) | 51.7 (4.3) | ‐ |
| Volume of distribution, brexpiprazole peripheral compartment (V3) | 40.1 (fixed) | 75.3 (fixed) | ‐ |
| Inter‐compartmental clearance, brexpiprazole (Q) | 0.714 (fixed) | ‐ | ‐ |
| Clearance from brexpiprazole to DM‐3412 (CLM1) | 0.0224 (22.7) | 83.4 (9.3) | [0.0357; 0.0458] |
| Clearance from brexpiprazole to DM‐3411 (CLM2) | 1.12 (2.7) | 59.6 (5.8) | [1.07; 1.15] |
| Volume of distribution, DM‐3412 central compartment (V4) | 0.447 (41.4) | 212 (33.7) | [0.420; 2.93] |
| Volume of distribution, DM‐3412 peripheral compartment (V5) | 20.4 (22.0) | ‐ | [30.5; 40.6] |
| Inter‐compartmental clearance, DM‐3412 (QMET) | 2.16 (21.9) | ‐ | [3.16; 4.43] |
| Clearance from DM‐3412 central compartment (CLMET1) | 0.420 (21.8) | 49.5 (14.6) | [0.672; 0.855] |
| Volume of distribution, DM‐3411 central compartment (V6) | 2.44 (8.0) | 126 (7.6) | [2.11; 2.81] |
| Clearance from DM‐3411 central compartment (CLMET2) | 3.33 (3.5) | 58.3 (6.7) | [3.16; 3.40] |
| Body mass index on V6 | −0.0107 (95.3) | ‐ | [−0.0174; −0.0000303] |
| Residual error brexpiprazole | 0.0937 (0.7) | ‐ | [0.0754; 0.116] |
| Residual error DM‐3412 | 0.0422 (0.5) | ‐ | [0.0332; 0.0552] |
| Residual error DM‐3411 | 0.0848 (0.9) | ‐ | [0.0729; 0.104] |

Table 2 Caption: Parameter estimates from the final population pharmacokinetic model

### CYP2D6 genotype–phenotype analysis

Out of the 826 individuals included in the PopPK analysis, 160 did not have any plasma concentration measurements of DM‐3412 above the LLOQ. These subjects were excluded from the *CYP2D6* genotype–phenotype analysis, as it was not possible to estimate their formation of DM‐3412. Out of the remaining 666 subjects, *CYP2D6* genotype information was available for 496 subjects, whose characteristics are summarized in **Table** [1](#cpt2791-tbl-0001). An MR of DM‐3412: brexpiprazole was estimated for the 496 subjects based on individual parameter estimates from the final PopPK model. The MR was used as a surrogate measure of the CYP2D6 activity in the *CYP2D6* genotype–phenotype analysis.

**Figure** [2](#cpt2791-fig-0002) shows the estimated MR for the 496 individuals according to their *CYP2D6* genotype with shapes reflecting whether the MR estimate was based on rich (circles), semi‐sparse (triangles), or sparse (squares) PK data. The percentage relative deviation of the individual MR estimates from the median MR within each genotype was similar across the three PK sampling schemes (rich: 25%, semi‐sparse: 26%, and sparse: 26%), indicating that all three data types were able to produce reasonable estimates of the CYP2D6 activity.

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/89a0/10099095/c8c7eb6aa1d2/CPT-113-360-g003.jpg)

Box‐ and scatterplots of metabolic ratios of DM‐3412/brexpiprazole according to individuals’ CYP2D6 genotype. Colors indicate CYP2D6 genotype‐predicted phenotype and shapes indicate whether the estimate was based on rich (circles), semi‐sparse (triangles), or sparse (squares) pharmacokinetic sampling. IM, intermediate metabolizer; NM, normal metabolizer; null: CYP2D6*3, *4, *5, and *6; PM, poor metabolizer; UM, ultra‐rapid metabolizer.

There was a good agreement between the genotype‐predicted CYP2D6 phenotype and the estimated MR: the lowest MR was estimated for the CYP2D6 PMs (*N* = 17, median MR: 0.00074), followed by the CYP2D6 IMs (*N* = 173, median MR: 0.045), CYP2D6 NMs (*N* = 294, median MR: 0.088), and highest for the CYP2D6 UMs (*N* = 12, median MR: 0.10). However, some of the individual *CYP2D6* genotypes were associated with MRs lower than expected. This was particularly evident for genotypes involving the *CYP2D6*2* allele (e.g., **2/*17*, **2/*41*, **2/*2*, and **2/*29*), which is normally considered fully functional.

### CYP2D6 activity scores

A CYP2D6 AS reflecting the relative activity to the wild‐type allele (*CYP2D6*1*) was estimated for each *CYP2D6* allele based on results from a multiple linear regression analysis (see Methods section).

In the regression analysis, pooling the null function alleles into a single indicator variable, did not result in deterioration of model fit (*P =* 0.98). However, the regression model, including the fully functional alleles, *CYP2D6*1* and *CYP2D6*2*, as separate variables showed a significantly better model fit compared with the regression model pooling the two alleles into one indicator variable (*P* < 0.001). The final regression model therefore included the following alleles as predictor variables: *CYP2D6*null*, *CYP2D6*1*, *CYP2D6*2*, *CYP2D6*9*, *CYP2D6*10*, *CYP2D6*14*, *CYP2D6*17*, *CYP2D6*29*, and *CYP2D6*41*. Using this regression model, the *CYP2D6* alleles explained 67% of the variability in the MR (adjusted *R* ^2^ = 0.67).

Activity scores estimated from the final regression model are presented in **Table** [3](#cpt2791-tbl-0003).

### Table 3.

| Allele | n a | CYP2D6 activity estimateb | CYP2D6 Activity Score | 95% CIc |
| --- | --- | --- | --- | --- |
| CYP2D6*null | 176 | 0.24 | 0.00 | ‐ |
| CYP2D6*1 | 396 | 0.92 | 1.00 | ‐ |
| CYP2D6*2 | 183 | 0.31 | 0.10 | [0.05; 0.15] |
| CYP2D6*9 | 20 | 0.40 | 0.23 | [0.11; 0.34] |
| CYP2D6*10 | 62 | 0.46 | 0.32 | [0.24; 0.39] |
| CYP2D6*14 | 1 | 0.68 | 0.64 | [0.53; 0.74] |
| CYP2D6*17 | 37 | 0.27 | 0.04 | [−0.03; 0.10] |
| CYP2D6*29 | 13 | 0.27 | 0.04 | [−0.05; 0.14] |
| CYP2D6*41 | 64 | 0.30 | 0.09 | [0.02; 0.15] |

Table 3 Caption: CYP2D6 activity scores estimated for individual CYP2D6 alleles based on multiple linear regression model

### Comparison of CYP2D6 alleles and genotypes

Among the decreased function alleles, *CYP2D6*9* (*n* = 20, AS = 0.23) and *CYP2D6*10* (*n* = 62, AS = 0.32) were associated with significantly higher CYP2D6 activities compared with *CYP2D6*17* (*n* = 37, AS = 0.04), *CYP2D6*29* (*n* = 13, AS = 0.04), and *CYP2D6*41* (*n* = 64, AS = 0.09; Wald tests, *P* < 0.05). Furthermore, the *CYP2D6*2* allele was found to have a significantly lower CYP2D6 activity (*n* = 183, AS = 0.10) compared with *CYP2D6*1* (*n* = 396, AS = 1.0; Wald test, *P <* 0.001), which was surprising as the *CYP2D6*2* allele is normally considered fully functional.[18](#cpt2791-bib-0018) To further explore this finding, the CYP2D6 activity of genotypes involving *CYP2D6*1* and *CYP2D6*2* were compared (see **Figure** [3](#cpt2791-fig-0003)).

### Figure 3.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/89a0/10099095/ad3863da1038/CPT-113-360-g001.jpg)

Box‐ and scatterplots of metabolic ratios (DM‐3412/brexpiprazole) associated with CYP2D6 genotypes including CYP2D6*1 and *2. P values reflect pairwise comparisons using Wilcoxon rank‐sum tests. Null: CYP2D6*3, *4, *5, and *6.

A comparison of *CYP2D6*1/**1, *CYP2D6*1/*2*, and *CYP2D6*2/*2* showed a decreasing CYP2D6 activity as the number of *CYP2D6*2* alleles increased, and the difference among all three genotypes was statistically significant (Bonferroni–Holm corrected *P <* 0.001, Wilcoxon rank‐sum test). Likewise, a comparison of *CYP2D6*1/null* and *CYP2D6*2/null* showed a significantly lower CYP2D6 activity associated with *CYP2D6*2/null* (*P <* 0.001, Wilcoxon rank‐sum test).

Pairwise comparisons also showed significant differences (Bonferroni–Holm corrected *P <* 0.001, Wilcoxon rank‐sum test) between *CYP2D6*1/*9* vs. *CYP2D6*2/*9*, *CYP2D6*1/*10* vs. *CYP2D6*2/*10*, *CYP2D6*1/*17* vs. *CYP2D6*2/*17*, and *CYP2D6*1/*41* vs. *CYP2D6*2/*41*. The median MR for *CYP2D6*1/*29* was numerically greater than that of *CYP2D6*2/*29* but the difference was not statistically significant (*P* = 0.26), which might be due to the low number of individuals carrying these genotypes (*n* = 3 and *n* = 2, respectively).

The observed differences in CYP2D6 activity between *CYP2D6*1* and *CYP2D6*2* carriers could not be attributed to differences in age, gender, ethnicity, PK sampling scheme, or study population.

## DISCUSSION

The main objective of this study was to estimate the CYP2D6 enzyme activity associated with different *CYP2D6* genotypes and alleles using brexpiprazole as a tool compound.

The CYP2D6 activity estimated for *CYP2D6*2* in the metabolism of brexpiprazole to DM‐3412 was significantly lower than expected for an allele that is classified as fully functional.[18](#cpt2791-bib-0018) Relative to *CYP2D6*1*, *CYP2D6*2* only exhibited 10% enzyme activity and reduced CYP2D6 activity was consistently observed across genotypes containing *CYP2D6*2*.

Other authors have reported a substantially reduced *in vivo* function of the *CYP2D6*2* allele based on studies using dextromethorphan[19](#cpt2791-bib-0019), [20](#cpt2791-bib-0020) and tamoxifen[21](#cpt2791-bib-0021) to estimate CYP2D6 activity. Interestingly, Gaedigk *et al*. found that African American carriers of *CYP2D6*1/*2* and *CYP2D6*2/*2* exhibited significantly higher urinary dextromethorphan/dextrorphan metabolic ratios (indicating lower CYP2D6 activity) compared with Whites suggesting that ethnicity may contribute to variability in enzyme activity.[22](#cpt2791-bib-0022)

A recent study of Sub‐Saharan African populations described several novel single nucleotide polymorphisms (SNPs) occurring in combination with the SNPs defining *CYP2D6*2*.[23](#cpt2791-bib-0023) Many of these novel alleles were predicted *in silico* to have deleterious impacts on enzyme function.[23](#cpt2791-bib-0023) Such alleles would likely have been reported as *CYP2D6*2* by most genotyping assays, which may explain the lower enzyme activity of *CYP2D6*2* carriers of African ancestry observed in some studies. In the current study, no notable difference in CYP2D6 activity was observed between *CYP2D6*2* carriers who were African Americans compared with other ethnicities carrying identical *CYP2D6* genotypes (see **Figure** [S3](#cpt2791-supitem-0001)).

Two other studies performed with a similar PopPK methodology did not show significant differences in CYP2D6 activity between *CYP2D6*1* and *CYP2D6*2* in the metabolism of vortioxetine or tedatioxetine,[13](#cpt2791-bib-0013), [14](#cpt2791-bib-0014) respectively, which suggests that the enzyme activity of *CYP2D6*2* may be substrate‐dependent.

The *CYP2D6*2* allele is characterized by two SNPs: rs16947 (2851C>T) and rs1135840 (4181G>C). Evidence suggests that the rs16947 SNP causes reduced *CYP2D6* gene expression through altered exonic splicing.[24](#cpt2791-bib-0024) The reduced expression is believed to be compensated by the rs1135840 SNP or through enhanced transcription caused by an enhancer SNP (rs5758550) in high linkage disequilibrium with rs16947,[24](#cpt2791-bib-0024), [25](#cpt2791-bib-0025) which may explain why *CYP2D6*2* carriers usually exhibit normal enzyme function. Unfortunately, the rs5758550 SNP was not included in the genotyping assays used in this study and therefore it was not possible to investigate the potential role of the enhancer SNP in the metabolism of brexpiprazole.

The two SNPs harbored by the *CYP2D6*2* allele result in amino acid substitutions R296C and S486T. Both these positions are thought to be involved in substrate‐recognition[26](#cpt2791-bib-0026) and particularly the replacement of the positively charged arginine by cysteine at position 296 holds potential to affect substrate‐binding.[27](#cpt2791-bib-0027) *In silico* modeling has shown that the two mutations cause changes in the protein structure leading to a reduction in the number of active sites and a decreased diameter of the substrate access channel relative to the wildtype (*CYP2D6*1*), which could explain the reduced enzyme activity observed for some substrates.[27](#cpt2791-bib-0027)

The decreased function alleles observed in this study were all associated with a > 50% reduction of CYP2D6 enzyme activity relative to the *CYP2D6*1* allele, except for *CYP2D6*14* where the estimate was only based on a single *CYP2D6*1/*14* carrier. The *CYP2D6*17*, *CYP2D6*29*, and *CYP2D6*41* alleles exhibited the largest reductions with only 4–9% enzymatic activity relative to *CYP2D6*1* (see **Table** [3](#cpt2791-tbl-0003)).

In two previous *CYP2D6* genotype–phenotype analyses of vortioxetine[13](#cpt2791-bib-0013) and tedatioxetine[14](#cpt2791-bib-0014) performed using a similar methodology we also found low enzyme activity associated with *CYP2D6*17* (AS of 0.17 and 0.01) and *CYP2D6*41* (AS of 0.21 and 0.21). Other clinical studies have demonstrated a low activity of *CYP2D6*41* in the metabolism of dextromethorphan,[19](#cpt2791-bib-0019) tamoxifen,[21](#cpt2791-bib-0021) and venlafaxine.[28](#cpt2791-bib-0028) These results imply that *CYP2D6*41* exhibits a substantial reduction of enzyme activity across a range of substrates and a lower activity score may therefore better reflect the phenotype conveyed by this allele.

Regarding the *CYP2D6*17* allele, studies have shown a wide range of *in vivo* enzyme activity[13](#cpt2791-bib-0013), [14](#cpt2791-bib-0014), [29](#cpt2791-bib-0029) indicating that the allele may exhibit substrate‐specific behavior. This is supported by a study of a Tanzanian population showing that homozygous *CYP2D6*17* carriers had a greater metabolic capacity towards codeine and metoprolol compared with debrisoquine and dextromethorphan.[9](#cpt2791-bib-0009) *In vitro* investigations of different CYP2D6 substrates have also demonstrated varying degrees of enzyme activity associated with *CYP2D6*17* supporting the hypothesis of substrate‐dependent activity of the allele.[10](#cpt2791-bib-0010), [11](#cpt2791-bib-0011)

In the current analysis, an AS of 0.32 was estimated for *CYP2D6*10* (*n* = 62). This is very similar to the results from our two previous analyses of vortioxetine and tedatioxetine using similar methodologies where AS of 0.34 and 0.37, respectively, were estimated for *CYP2D6*10*.[13](#cpt2791-bib-0013), [14](#cpt2791-bib-0014) Collectively, these results support the recent decision to assign an AS of 0.25 to *CYP2D6*10*.[8](#cpt2791-bib-0008)

Large variability in CYP2D6 activity was observed among certain *CYP2D6* genotypes, particularly those involving the wildtype allele (e.g., *CYP2D6***1/*1*, *CYP2D6*1/*2*, and *CYP2D6*1/null*). Some of this variability might have been caused by allele variants not covered by the genotyping assay, as the wildtype (*CYP2D6*1*) was assigned by default when no variants were detected. In addition, several less frequent allele variants that share the two SNPs defining the *CYP2D6*2* allele (rs16947 and rs1135840) were not included in the genotyping test panels which may have led to erroneous assignment to *CYP2D6*2* for some individuals. Another limitation of the genotyping assays was that they only indicated presence of additional gene copies but did not allow identification of which allele was duplicated or the exact copy number. This may explain the large variability in, for example, the *CYP2D6*1/*4XN* genotype as the additional gene copy could originate from either the fully functional *CYP2D6*1* or the non‐functional *CYP2D6*4*. In addition, the very high CYP2D6 activity observed for *CYP2D6*1/*17XN* may suggest that multiple gene copies were present for some of these individuals.

The final PopPK model described the PK of brexpiprazole and its two metabolites, DM‐3411 and DM‐3412, well. Based on the bootstrap analysis, the population parameters related to the DM‐3412 PK were estimated with poor precision. A sensitivity analysis excluding subjects with no measurable DM‐3412 concentrations showed great improvement in the precision of the DM‐3412 parameters suggesting that the poor precision was largely driven by the <LLOQ observations for the metabolite. To avoid confounding of the results, subjects with no DM‐3412 observations > LLOQ were excluded from the subsequent genotype–phenotype analysis where individual parameter estimates (empirical Bayes estimates) from the final PopPK model were used.

The PopPK modeling approach enabled characterization of two metabolic routes, thereby allowing quantification of the DM‐3412 formation separately. As DM‐3412 is almost exclusively formed via CYP2D6, this provided a good surrogate measure of CYP2D6 activity, although other CYP enzymes contribute to the overall metabolism of brexpiprazole. Furthermore, covariate modeling allowed the PopPK model to account for other factors affecting the PKs. Food effect was identified as a significant covariate on the *k* _a_, which appeared to be driven by individuals from a multiple‐ascending dose study. However, a clinical study dedicated to evaluating food effect did not find any impact on brexpiprazole PK.[30](#cpt2791-bib-0030)

It is possible that other factors, not accounted for by the model, may have affected the extent of DM‐3412 formation. For example, other enzymes may contribute to the DM‐3412 formation and physiological and/or environmental factors may affect the metabolism (e.g., through altered *CYP2D6* gene expression). The model did not incorporate these potential sources of variability, which may account for some of the variability in the estimated CYP2D6 activity within the *CYP2D6* genotypes.

Furthermore, the major metabolite, DM‐3411, is primarily formed via CYP3A4 which is known to vary considerably between individuals, although not affected significantly by genetic polymorphism.[31](#cpt2791-bib-0031) The PopPK model included interindividual variability on the CL_M2_ parameter (i.e., the clearance resulting in formation of DM‐3411), thereby allowing this parameter to be individually estimated for each subject in the data set. Therefore, the variability in CYP3A4‐mediated metabolism was captured by the model and was not considered to be a major source for confounding.

In conclusion, a PopPK model of brexpiprazole and its two metabolites, DM‐3411 and DM‐3412, enabled estimation of CYP2D6‐mediated metabolism in almost 500 individuals carrying different *CYP2D6* genotypes. The results suggest that the existing CYP2D6 activity score system does not accurately reflect the *in vivo* function of some *CYP2D6* alleles. Particularly, the AS assigned to decreased function alleles *CYP2D6*17*, *CYP2D6*29*, and *CYP2D6*41* appear to overestimate their *in vivo* activity and the low enzyme activity observed for *CYP2D6*2* carriers in this analysis suggests that the allele may exhibit substrate‐specific activity. These findings highlight the need for further *CYP2D6* genotype–phenotype investigations to support the ongoing refinement of translation schemes to guide personalized treatment with CYP2D6 substrates.

## AUTHOR CONTRIBUTIONS

T.F., J.A., A.R., E.S., T.B.S., and K.B. wrote the manuscript. J.A. and E.S. designed the research. T.F and J.A. performed the research. T.F., J.A., T.B.S., and K.B. analyzed the data.

## FUNDING

The research was funded by Innovation Fund Denmark (grant number: 8053‐00083B) and H. Lundbeck A/S.

## CONFLICT OF INTEREST

T.F., J.A., and E.S. are employed by H. Lundbeck A/S. A.R. is employed by Otsuka Pharmaceutical Development & Commercialization, Inc. All other authors declared no competing interests for this work.

## Supporting information

## ACKNOWLEDGMENTS

The authors thank Frank Larsen for valuable scientific discussions and critical review of this paper. The data used in the analysis was provided by H. Lundbeck A/S and Otsuka Pharmaceutical Co. Ltd.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Saravanakumar, A. , Sadighi, A. , Ryu, R. & Akhlaghi, F. Physicochemical properties, biotransformation, and transport pathways of established and newly approved medications: a systematic review of the top 200 Most prescribed drugs vs. the FDA‐approved drugs between 2005 and 2016. Clin. Pharmacokinet. 58, 1281–1294 (2019).  [DOI](https://doi.org/10.1007/s40262-019-00750-8) | [PMC free article](/articles/PMC6773482/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30972694/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacokinet.&title=Physicochemical%20properties,%20biotransformation,%20and%20transport%20pathways%20of%20established%20and%20newly%20approved%20medications:%20a%20systematic%20review%20of%20the%20top%20200%20Most%20prescribed%20drugs%20vs.%20the%20FDA%E2%80%90approved%20drugs%20between%202005%20and%202016&author=A.%20Saravanakumar&author=A.%20Sadighi&author=R.%20Ryu&author=F.%20Akhlaghi&volume=58&publication_year=2019&pages=1281-1294&pmid=30972694&doi=10.1007/s40262-019-00750-8&)

2. PharmGKB CYP2D6 Drug Label Annotations <https://www.pharmgkb.org/gene/PA128/labelAnnotation>.  [https://www.pharmgkb.org/gene/PA128/labelAnnotation](https://www.pharmgkb.org/gene/PA128/labelAnnotation)

3. Hicks, J.K. et al. Clinical Pharmacogenetics implementation consortium (CPIC) guideline for CYP2D6 and CYP2C19 genotypes and dosing of selective serotonin reuptake inhibitors. Clin. Pharmacol. Ther. 98, 127–134 (2015).  [DOI](https://doi.org/10.1002/cpt.147) | [PMC free article](/articles/PMC4512908/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25974703/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Clinical%20Pharmacogenetics%20implementation%20consortium%20(CPIC)%20guideline%20for%20CYP2D6%20and%20CYP2C19%20genotypes%20and%20dosing%20of%20selective%20serotonin%20reuptake%20inhibitors&author=J.K.%20Hicks&volume=98&publication_year=2015&pages=127-134&pmid=25974703&doi=10.1002/cpt.147&)

4. Hicks, J.K. et al. Clinical Pharmacogenetics implementation consortium guideline (CPIC) for CYP2D6 and CYP2C19 genotypes and dosing of tricyclic antidepressants: 2016 update. Clin. Pharmacol. Ther. 102, 37–44 (2017).  [DOI](https://doi.org/10.1002/cpt.597) | [PMC free article](/articles/PMC5478479/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27997040/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Clinical%20Pharmacogenetics%20implementation%20consortium%20guideline%20(CPIC)%20for%20CYP2D6%20and%20CYP2C19%20genotypes%20and%20dosing%20of%20tricyclic%20antidepressants:%202016%20update&author=J.K.%20Hicks&volume=102&publication_year=2017&pages=37-44&pmid=27997040&doi=10.1002/cpt.597&)

5. Goetz, M.P. et al. Clinical pharmacogenetics implementation consortium (CPIC) guideline for CYP2D6 and tamoxifen therapy. Clin. Pharmacol. Ther. 103, 770–777 (2018).  [DOI](https://doi.org/10.1002/cpt.1007) | [PMC free article](/articles/PMC5931215/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29385237/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Clinical%20pharmacogenetics%20implementation%20consortium%20(CPIC)%20guideline%20for%20CYP2D6%20and%20tamoxifen%20therapy&author=M.P.%20Goetz&volume=103&publication_year=2018&pages=770-777&pmid=29385237&doi=10.1002/cpt.1007&)

6. Crews, K.R. et al. Clinical Pharmacogenetics implementation consortium guidelines for cytochrome P450 2D6 genotype and codeine therapy: 2014 update. Clin. Pharmacol. Ther. 95, 376–382 (2014).  [DOI](https://doi.org/10.1038/clpt.2013.254) | [PMC free article](/articles/PMC3975212/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24458010/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Clinical%20Pharmacogenetics%20implementation%20consortium%20guidelines%20for%20cytochrome%20P450%202D6%20genotype%20and%20codeine%20therapy:%202014%20update&author=K.R.%20Crews&volume=95&publication_year=2014&pages=376-382&pmid=24458010&doi=10.1038/clpt.2013.254&)

7. Molden, E. & Jukić, M.M. CYP2D6 reduced function variants and genotype/phenotype translations of CYP2D6 intermediate metabolizers: implications for personalized drug dosing in psychiatry. Front. Pharmacol. 12, 1–7 (2021).  [DOI](https://doi.org/10.3389/fphar.2021.650750) | [PMC free article](/articles/PMC8100508/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33967790/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front.%20Pharmacol.&title=CYP2D6%20reduced%20function%20variants%20and%20genotype/phenotype%20translations%20of%20CYP2D6%20intermediate%20metabolizers:%20implications%20for%20personalized%20drug%20dosing%20in%20psychiatry&author=E.%20Molden&author=M.M.%20Juki%C4%87&volume=12&publication_year=2021&pages=1-7&pmid=33967790&doi=10.3389/fphar.2021.650750&)

8. Caudle, K.E. et al. Standardizing CYP2D6 genotype to phenotype translation: consensus recommendations from the clinical Pharmacogenetics implementation consortium and Dutch Pharmacogenetics working group. Clin. Transl. Sci. 13, 116–124 (2019).  [DOI](https://doi.org/10.1111/cts.12692) | [PMC free article](/articles/PMC6951851/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31647186/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Transl.%20Sci.&title=Standardizing%20CYP2D6%20genotype%20to%20phenotype%20translation:%20consensus%20recommendations%20from%20the%20clinical%20Pharmacogenetics%20implementation%20consortium%20and%20Dutch%20Pharmacogenetics%20working%20group&author=K.E.%20Caudle&volume=13&publication_year=2019&pages=116-124&pmid=31647186&doi=10.1111/cts.12692&)

9. Wennerholm, A. et al. The African‐specific CYP2D617 allele encodes an enzyme with changed substrate specificity. Clin. Pharmacol. Ther. 71, 77–88 (2002).  [DOI](https://doi.org/10.1067/mcp.2002.120239) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11823760/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=The%20African%E2%80%90specific%20CYP2D617%20allele%20encodes%20an%20enzyme%20with%20changed%20substrate%20specificity&author=A.%20Wennerholm&volume=71&publication_year=2002&pages=77-88&pmid=11823760&doi=10.1067/mcp.2002.120239&)

10. Shen, H. et al. Comparative metabolic capabilities and inhibitory profiles of CYP2D6.1, CYP2D6.10, and CYP2D6.17. Drug Metab. Dispos. 35, 1292–1300 (2007).  [DOI](https://doi.org/10.1124/dmd.107.015354) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17470523/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab.%20Dispos.&title=Comparative%20metabolic%20capabilities%20and%20inhibitory%20profiles%20of%20CYP2D6.1,%20CYP2D6.10,%20and%20CYP2D6.17&author=H.%20Shen&volume=35&publication_year=2007&pages=1292-1300&pmid=17470523&doi=10.1124/dmd.107.015354&)

11. Bogni, A. et al. Substrate specific metabolism by polymorphic cytochrome P450 2D6 alleles. Toxicol. Vitr. 19, 621–629 (2005).  [DOI](https://doi.org/10.1016/j.tiv.2005.04.001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15893449/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Toxicol.%20Vitr.&title=Substrate%20specific%20metabolism%20by%20polymorphic%20cytochrome%20P450%202D6%20alleles&author=A.%20Bogni&volume=19&publication_year=2005&pages=621-629&pmid=15893449&doi=10.1016/j.tiv.2005.04.001&)

12. Ingelman‐Sundberg, M. Translation of pharmacogenomic drug labels into the clinic Current problems. Pharmacol. Res. 153, 104620 (2020).  [DOI](https://doi.org/10.1016/j.phrs.2019.104620) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31899313/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacol.%20Res.&title=Translation%20of%20pharmacogenomic%20drug%20labels%20into%20the%20clinic%20Current%20problems&author=M.%20Ingelman%E2%80%90Sundberg&volume=153&publication_year=2020&pages=104620&pmid=31899313&doi=10.1016/j.phrs.2019.104620&)

13. Frederiksen, T. , Areberg, J. , Schmidt, E. , Bjerregaard Stage, T. & Brøsen, K. Quantification of In vivo metabolic activity of CYP2D6 genotypes and alleles through population pharmacokinetic analysis of Vortioxetine. Clin. Pharmacol. Ther. 109, 150–159 (2020).  [DOI](https://doi.org/10.1002/cpt.1972) | [PMC free article](/articles/PMC7818194/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32599653/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Quantification%20of%20In%20vivo%20metabolic%20activity%20of%20CYP2D6%20genotypes%20and%20alleles%20through%20population%20pharmacokinetic%20analysis%20of%20Vortioxetine&author=T.%20Frederiksen&author=J.%20Areberg&author=E.%20Schmidt&author=T.%20Bjerregaard%20Stage&author=K.%20Br%C3%B8sen&volume=109&publication_year=2020&pages=150-159&pmid=32599653&doi=10.1002/cpt.1972&)

14. Frederiksen, T. , Areberg, J. , Schmidt, E. , Stage, T.B. & Brøsen, K. Cytochrome P450 2D6 genotype–phenotype characterization through population pharmacokinetic modeling of tedatioxetine. CPT Pharmacometrics Syst. Pharmacol. 10, 983–993 (2021).  [DOI](https://doi.org/10.1002/psp4.12635) | [PMC free article](/articles/PMC8452298/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33932135/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=CPT%20Pharmacometrics%20Syst.%20Pharmacol.&title=Cytochrome%20P450%202D6%20genotype%E2%80%93phenotype%20characterization%20through%20population%20pharmacokinetic%20modeling%20of%20tedatioxetine&author=T.%20Frederiksen&author=J.%20Areberg&author=E.%20Schmidt&author=T.B.%20Stage&author=K.%20Br%C3%B8sen&volume=10&publication_year=2021&pages=983-993&pmid=33932135&doi=10.1002/psp4.12635&)

15. Otsuka Pharmaceutical Co Ltd . Prescribing Information for Rexulti (Brexpiprazole) Tablets <http://www.accessdata.fda.gov> (2020).  [http://www.accessdata.fda.gov](http://www.accessdata.fda.gov)

16. Sasabe, H. et al. In vitro evaluations for pharmacokinetic drug‐drug interactions of a novel serotonin‐dopamine activity modulator, brexpiprazole. Xenobiotica 51, 522–535 (2021).  [DOI](https://doi.org/10.1080/00498254.2021.1897898) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33663326/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Xenobiotica&title=In%20vitro%20evaluations%20for%20pharmacokinetic%20drug%E2%80%90drug%20interactions%20of%20a%20novel%20serotonin%E2%80%90dopamine%20activity%20modulator,%20brexpiprazole&author=H.%20Sasabe&volume=51&publication_year=2021&pages=522-535&pmid=33663326&doi=10.1080/00498254.2021.1897898&)

17. Sasabe, H. et al. Pharmacokinetics and metabolism of brexpiprazole, a novel serotonin‐dopamine activity modulator and its main metabolite in rat, monkey and human. Xenobiotica 51, 590–604 (2021).  [DOI](https://doi.org/10.1080/00498254.2021.1890275) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33685346/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Xenobiotica&title=Pharmacokinetics%20and%20metabolism%20of%20brexpiprazole,%20a%20novel%20serotonin%E2%80%90dopamine%20activity%20modulator%20and%20its%20main%20metabolite%20in%20rat,%20monkey%20and%20human&author=H.%20Sasabe&volume=51&publication_year=2021&pages=590-604&pmid=33685346&doi=10.1080/00498254.2021.1890275&)

18. PharmGKB . Gene‐Specific Information Tables for CYP2D6 <https://www.pharmgkb.org/page/cyp2d6RefMaterials> Accessed October 28, 2022.  [https://www.pharmgkb.org/page/cyp2d6RefMaterials](https://www.pharmgkb.org/page/cyp2d6RefMaterials)

19. Abduljalil, K. et al. Assessment of activity levels for CYP2D6*1, CYP2D6*2, and CYP2D6*41 genes by population pharmacokinetics of dextromethorphan. Clin. Pharmacol. Ther. 88, 643–651 (2010).  [DOI](https://doi.org/10.1038/clpt.2010.137) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20881950/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Assessment%20of%20activity%20levels%20for%20CYP2D6*1,%20CYP2D6*2,%20and%20CYP2D6*41%20genes%20by%20population%20pharmacokinetics%20of%20dextromethorphan&author=K.%20Abduljalil&volume=88&publication_year=2010&pages=643-651&pmid=20881950&doi=10.1038/clpt.2010.137&)

20. Panserat, S. , Sica, L. , Gérard, N. , Mathieu, H. , Jacqz‐Aigrain, E. & Krishnamoorthy, R. CYP2D6 polymorphism in a Gabonese population: contribution of the CYP2D6*2 and CYP2D6*17 alleles to the high prevalence of the intermediate metabolic phenotype. Br. J. Clin. Pharmacol. 47, 121–124 (1999).  [DOI](https://doi.org/10.1046/j.1365-2125.1999.00861.x) | [PMC free article](/articles/PMC2014206/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10073750/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Clin.%20Pharmacol.&title=CYP2D6%20polymorphism%20in%20a%20Gabonese%20population:%20contribution%20of%20the%20CYP2D6*2%20and%20CYP2D6*17%20alleles%20to%20the%20high%20prevalence%20of%20the%20intermediate%20metabolic%20phenotype&author=S.%20Panserat&author=L.%20Sica&author=N.%20G%C3%A9rard&author=H.%20Mathieu&author=E.%20Jacqz%E2%80%90Aigrain&volume=47&publication_year=1999&pages=121-124&pmid=10073750&doi=10.1046/j.1365-2125.1999.00861.x&)

21. Hertz, D.L. et al. In vivo assessment of the metabolic activity of CYP2D6 diplotypes and alleles. Br. J. Clin. Pharmacol. 80, 1122–1130 (2015).  [DOI](https://doi.org/10.1111/bcp.12665) | [PMC free article](/articles/PMC4631184/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25907378/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Clin.%20Pharmacol.&title=In%20vivo%20assessment%20of%20the%20metabolic%20activity%20of%20CYP2D6%20diplotypes%20and%20alleles&author=D.L.%20Hertz&volume=80&publication_year=2015&pages=1122-1130&pmid=25907378&doi=10.1111/bcp.12665&)

22. Gaedigk, A. , Simon, S.D. , Pearce, R.E. , Bradford, L.D. , Kennedy, M.J. & Leeder, J.S. The CYP2D6 activity score: translating genotype information into a qualitative measure of phenotype. Clin. Pharmacol. Ther. 83, 234–242 (2008).  [DOI](https://doi.org/10.1038/sj.clpt.6100406) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17971818/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=The%20CYP2D6%20activity%20score:%20translating%20genotype%20information%20into%20a%20qualitative%20measure%20of%20phenotype&author=A.%20Gaedigk&author=S.D.%20Simon&author=R.E.%20Pearce&author=L.D.%20Bradford&author=M.J.%20Kennedy&volume=83&publication_year=2008&pages=234-242&pmid=17971818&doi=10.1038/sj.clpt.6100406&)

23. Wang, W.Y. et al. Characterization of novel CYP2D6 alleles across sub‐Saharan African populations. J. Pers. Med. 12, 1575 (2022).  [DOI](https://doi.org/10.3390/jpm12101575) | [PMC free article](/articles/PMC9605556/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36294714/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Pers.%20Med.&title=Characterization%20of%20novel%20CYP2D6%20alleles%20across%C2%A0sub%E2%80%90Saharan%20African%20populations&author=W.Y.%20Wang&volume=12&publication_year=2022&pages=1575&pmid=36294714&doi=10.3390/jpm12101575&)

24. Wang, D. , Poi, M.J. , Sun, X. , Gaedigk, A. , Leeder, J.S. & Sadee, W. Common CYP2D6 polymorphisms affecting alternative splicing and transcription: long‐range haplotypes with two regulatory variants modulate CYP2D6 activity. Hum. Mol. Genet. 23, 268–278 (2014).  [DOI](https://doi.org/10.1093/hmg/ddt417) | [PMC free article](/articles/PMC3857955/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23985325/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum.%20Mol.%20Genet.&title=Common%20CYP2D6%20polymorphisms%20affecting%20alternative%20splicing%20and%20transcription:%20long%E2%80%90range%20haplotypes%20with%20two%20regulatory%20variants%20modulate%20CYP2D6%20activity&author=D.%20Wang&author=M.J.%20Poi&author=X.%20Sun&author=A.%20Gaedigk&author=J.S.%20Leeder&volume=23&publication_year=2014&pages=268-278&pmid=23985325&doi=10.1093/hmg/ddt417&)

25. Zanger, U.M. , Momoi, K. , Hofmann, U. , Schwab, M. & Klein, K. Tri‐allelic haplotypes determine and differentiate functionally Normal allele CYP2D6*2 and impaired allele CYP2D6*41. Clin. Pharmacol. Ther. 109, 1256–1264 (2021).  [DOI](https://doi.org/10.1002/cpt.2078) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33043448/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Tri%E2%80%90allelic%20haplotypes%20determine%20and%20differentiate%20functionally%20Normal%20allele%20CYP2D6*2%20and%20impaired%20allele%20CYP2D6*41&author=U.M.%20Zanger&author=K.%20Momoi&author=U.%20Hofmann&author=M.%20Schwab&author=K.%20Klein&volume=109&publication_year=2021&pages=1256-1264&pmid=33043448&doi=10.1002/cpt.2078&)

26. Gotoh, O. Substrate recognition sites in cytochrome P450 family 2 (CYP2) proteins inferred from comparative analyses of amino acid and coding nucleotide sequences. J. Biol. Chem. 267, 83–90 (1992).  [PubMed](https://pubmed.ncbi.nlm.nih.gov/1730627/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem.&title=Substrate%20recognition%20sites%20in%20cytochrome%20P450%20family%202%20(CYP2)%20proteins%20inferred%20from%20comparative%20analyses%20of%20amino%20acid%20and%20coding%20nucleotide%20sequences&author=O.%20Gotoh&volume=267&publication_year=1992&pages=83-90&pmid=1730627&)

27. Dong, A.N. , Ahemad, N. , Pan, Y. , Palanisamy, U.D. , Yiap, B.C. & Ong, C.E. Functional and structural characterisation of common cytochrome P450 2D6 allelic variants—roles of Pro34 and Thr107 in catalysis and inhibition. Naunyn Schmiedebergs Arch. Pharmacol. 392, 1015–1029 (2019).  [DOI](https://doi.org/10.1007/s00210-019-01651-0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31025144/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Naunyn%20Schmiedebergs%20Arch.%20Pharmacol.&title=Functional%20and%20structural%20characterisation%20of%20common%20cytochrome%20P450%202D6%20allelic%20variants%E2%80%94roles%20of%20Pro34%20and%20Thr107%20in%20catalysis%20and%20inhibition&author=A.N.%20Dong&author=N.%20Ahemad&author=Y.%20Pan&author=U.D.%20Palanisamy&author=B.C.%20Yiap&volume=392&publication_year=2019&pages=1015-1029&pmid=31025144&doi=10.1007/s00210-019-01651-0&)

28. Haslemo, T. , Eliasson, E. , Jukic, M.M. , Ingelman‐Sundberg, M. & Molden, E. Significantly lower CYP2D6 metabolism measured as the O/N‐desmethylvenlafaxine metabolic ratio in carriers of CYP2D6*41 versus CYP2D6*9 or CYP2D6*10: a study on therapeutic drug monitoring data from 1003 genotyped Scandinavian patients. Br. J. Clin. Pharmacol. 85, 194–201 (2019).  [DOI](https://doi.org/10.1111/bcp.13788) | [PMC free article](/articles/PMC6303206/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30312494/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Clin.%20Pharmacol.&title=Significantly%20lower%20CYP2D6%20metabolism%20measured%20as%20the%20O/N%E2%80%90desmethylvenlafaxine%20metabolic%20ratio%20in%20carriers%20of%20CYP2D6*41%20versus%20CYP2D6*9%20or%20CYP2D6*10:%20a%20study%20on%20therapeutic%20drug%20monitoring%20data%20from%201003%20genotyped%20Scandinavian%20patients&author=T.%20Haslemo&author=E.%20Eliasson&author=M.M.%20Jukic&author=M.%20Ingelman%E2%80%90Sundberg&author=E.%20Molden&volume=85&publication_year=2019&pages=194-201&pmid=30312494&doi=10.1111/bcp.13788&)

29. Cai, W.M. et al. CYP2D6 genetic variation in healthy adults and psychiatric African‐American subjects: implications for clinical practice and genetic testing. Pharmacogenomics J. 6, 343–350 (2006).  [DOI](https://doi.org/10.1038/sj.tpj.6500378) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16550211/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J.&title=CYP2D6%20genetic%20variation%20in%20healthy%20adults%20and%20psychiatric%20African%E2%80%90American%20subjects:%20implications%20for%20clinical%20practice%20and%20genetic%20testing&author=W.M.%20Cai&volume=6&publication_year=2006&pages=343-350&pmid=16550211&doi=10.1038/sj.tpj.6500378&)

30. Garnock‐Jones, K.P. Brexpiprazole: a review in schizophrenia. CNS Drugs 30, 335–342 (2016).  [DOI](https://doi.org/10.1007/s40263-016-0325-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27023789/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=CNS%20Drugs&title=Brexpiprazole:%20a%20review%20in%20schizophrenia&author=K.P.%20Garnock%E2%80%90Jones&volume=30&publication_year=2016&pages=335-342&pmid=27023789&doi=10.1007/s40263-016-0325-8&)

31. Lamba, J.K. , Lin, Y.S. , Schuetz, E.G. & Thummel, K.E. Genetic contribution to variable human CYP3A‐mediated metabolism. Adv. Drug Deliv. Rev. 54, 1271–1294 (2002).  [DOI](https://doi.org/10.1016/s0169-409x(02)00066-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12406645/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Adv.%20Drug%20Deliv.%20Rev.&title=Genetic%20contribution%20to%20variable%20human%20CYP3A%E2%80%90mediated%20metabolism&author=J.K.%20Lamba&author=Y.S.%20Lin&author=E.G.%20Schuetz&author=K.E.%20Thummel&volume=54&publication_year=2002&pages=1271-1294&pmid=12406645&doi=10.1016/s0169-409x(02)00066-2&)
