# Germline variants associated with toxicity to immune checkpoint blockade

## Metadata
**Authors:** Stefan Groha, Sarah Abou Alaiwi, Wenxin Xu, Vivek Naranbhai, Amin H Nassar, Ziad Bakouny, Talal El Zarif, Renee Maria Saliby, Guihong Wan, Ahmad Rajeh, Elio Adib, Pier V Nuzzo, Andrew L Schmidt, Chris Labaki, Biagio Ricciuti, Joao Victor Alessi, David A Braun, Sachet A Shukla, Tanya E Keenan, Eliezer Van Allen, Mark M Awad, Michael Manos, Osama Rahma, Leyre Zubiri, Alexandra-Chloe Villani, Benjamin Fairfax, Christian Hammer, Zia Khan, Kerry Reynolds, Yevgeniy Semenov, Deborah Schrag, Kenneth L Kehl, Matthew L Freedman, Toni K Choueiri, Alexander Gusev
**Journal:** Nature medicine
**Date:** 2022 Dec 16
**DOI:** [10.1038/s41591-022-02094-6](https://doi.org/10.1038/s41591-022-02094-6)
**PMID:** 36526723
**PMCID:** PMC10958775
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10958775/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC10958775/pdf/nihms-1969921.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC10958775/pdf/nihms-1969921.pdf)

## Abstract

Immune checkpoint inhibitors (ICIs) have yielded remarkable responses but often lead to immune-related adverse events (irAEs). Although germline causes for irAEs have been hypothesized, no individual variant associated with developing irAEs has been identified. We carried out a genome-wide association study of 1,751 patients on ICIs across 12 cancer types. We investigated two irAE phenotypes: (1) high-grade (3–5) and (2) all-grade events. We identified 3 genome-wide significant associations (P < 5 × 10−8) in the discovery cohort associated with all-grade irAEs: rs16906115 near IL7 (combined P = 3.6 × 10−11; hazard ratio (HR) = 2.1); rs75824728 near IL22RA1 (combined P = 3.5 × 10−8; HR = 1.8); and rs113861051 on 4p15 (combined P = 1.2 × 10−8, HR = 2.0); rs16906115 was replicated in 3 independent studies. The association near IL7 colocalized with the gain of a new cryptic exon for IL7, a critical regulator of lymphocyte homeostasis. Patients carrying the IL7 germline variant exhibited significantly increased lymphocyte stability after ICI initiation, which was itself predictive of downstream irAEs and improved survival.

## Results

### GWAS of irAEs

We carried out a GWAS for two irAE phenotypes in 1,751 patients of European ancestry across 12 cancer types treated with ICIs at a single tertiary institution (Dana-Farber Cancer Institute (DFCI) cohort; [Table 1](#T1) and [Supplementary Fig. 1](#SD1)). Two irAE outcomes were defined for each patient after treatment initiation: (1) ‘high-grade’ irAEs (259 cases, 1,375 controls) determined by manual curation of records according to the National Cancer Institute (NCI) Common Terminology Criteria for Adverse Events v.5 guidelines for grade 3–5 events, with attribution of AEs as being immune-related determined based on the clinical consensus of the patient’s care team; (2) ‘all-grade’ irAEs (339 cases, 1,412 controls) algorithmically identified based on autoimmune-like electronic health record (EHR) diagnosis codes ([Supplementary Table 1](#SD2)) and including any high-grade irAEs, followed by manual review to exclude any events that were definitively linked to other causes. Detailed chart review in a subset of 44 random patients found 85% of all-grade irAEs to be consistent with grade 2 or higher events ([Methods](#S8) and [Supplementary Table 2](#SD2)). We saw no effect of sex (*P* = 0.84; [Supplementary Fig. 2](#SD1)) and a significant effect of age (hazard ratio (HR) = 1.8 (1.3–2.6), *P* = 4 × 10^−4^; [Supplementary Fig. 2](#SD1)) on all-grade irAEs. Patients on CTLA4 or a combination of programmed cell death protein 1 (PD-1)/programmed death-ligand 1 (PD-L1) and CTLA4 monoclonal antibodies experienced a significantly higher rate of irAEs, as previously found^15^ ([Extended Data Fig. 1](#F5)). Power analyses ([Supplementary Fig. 3](#SD1)) showed that this sample was sufficient to identify large effect variants, which we hypothesized to exist for treatment response outcomes given previous pharmacogenetic studies^16^.

### Table 1 |.

| Profile cohort (discovery) |  | MGH cohort (replication) |  |
| --- | --- | --- | --- |
|   | Overall (n = 1,751) |   | Overall (n = 265) |
| All-grade irAEs |   |   |   |
| Yes | 339 (19.4%) |   |   |
| No | 1,412 (80.6%) |   |   |
| High-grade irAEs |   | High-grade irAEs |   |
| Yes | 259 (14.8%) | Yes | 51 (19.2%) |
| No | 1,375 (78.5%) | No | 214 (80.8%) |
| Not curated | 117 (6.7%) |   |   |
| Cancer type |   | Cancer type |   |
| Non-small-cell lung cancer | 539 (30.8%) | Gastrointestinal cancer | 34 (12.8%) |
| Melanoma | 241 (13.8%) | Genitourinary cancer | 30 (11.3%) |
| Other | 236 (13.5%) | Lung cancer | 62 (23.4%) |
| Glioma | 112 (6.4%) | Other | 66 (24.9%) |
| Breast carcinoma | 111 (6.3%) | Skin cancer | 73 (27.5%) |
| Esophagogastric carcinoma | 111 (6.3%) |   |   |
| Renal cell carcinoma | 109 (6.2%) |   |   |
| Bladder cancer | 94 (5.4%) |   |   |
| Head and neck carcinoma | 90 (5.1%) |   |   |
| Ovarian cancer | 40 (2.3%) |   |   |
| Cancer of unknown primary origin | 34 (1.9%) |   |   |
| Colorectal cancer | 34 (1.9%) |   |   |
| Sex |   | Sex |   |
| Female | 814 (46.5%) | Female | 119 (44.9%) |
| Male | 937 (53.5%) | Male | 146 (55.1%) |
| Age |   | Age |   |
| Mean (SD) | 63.0 (12.4) | Mean (SD) | 62.3 (13.8) |
| Median (min–max) | 63.9 (19.6–102.00) | Median (min–max) | 64.0 (22.3–90.2) |
| Type of treatment |   | Type of treatment |   |
| CTLA4 | 49 (2.8%) | CTLA4 | 24 (9.1%) |
| Combination therapy | 154 (8.8%) | Combination therapy | 27 (10.2%) |
| PD-1/PD-L1 | 1,548 (88.4%) | PD-1/PD-L1 | 214 (80.8%) |
| Sequencing |   |   |   |
| Before ICI initiation | 1,363 (77.8%) |   |   |
| After ICI initiation | 388 (22.2%) |   |   |
| Start year |   |   |   |
| Before 2016 | 357 (20.4%) |   |   |
| 2016 | 416 (23.8%) |   |   |
| 2017 | 557 (31.8%) |   |   |
| 2018 | 305 (17.4%) |   |   |
| After 2018 | 116 (6.6%) |   |   |

Table 1 Caption: Cohort description of the discovery and validation cohort at the MGH

We identified three genome-wide significant loci (*P* < 5 × 10^−8^) associated with all-grade irAEs: 1 near interleukin 7 (*IL7*) at chr8q21, 1 near the interleukin 22 receptor subunit alpha 1 (*IL22RA1*) at chr1p36 and the third association at chr4p15 ([Fig. 1](#F1) and [Supplementary Fig. 4](#SD1)). No genome-wide significant associations were identified for high-grade irAEs. We tested each single-nucleotide polymorphism (SNP) for association with individual irAE subtypes and found that all three SNPs were nominally significant across multiple irAE subtypes with no clear outliers ([Supplementary Fig. 5a](#SD1) and [Supplementary Table 3](#SD2)) and were significant in the 80% of patients on PD-1 ICIs (with insufficient power to test differences by drug class such as CTLA-4 versus PD-1 ICIs; [Supplementary Fig. 6a](#SD1)). Neither variant was associated with overall survival nor with death without irAEs, even though all all-grade irAEs were associated with longer overall survival in a time-dependent analysis (HR = 0.78 (0.65–0.94), *P* = 8.6×10^−3^; [Supplementary Table 4](#SD2)), which is consistent with previous findings.

### Fig. 1 |. Manhattan plot of irAE GWAS associations.

![Fig. 1 |](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9adc/10958775/24ddb3ad3bc2/nihms-1969921-f0001.jpg)

Associations in the DFCI discovery cohort for all-grade irAEs. Each dot represents an associated SNP, with position of the SNP (x axis) and P value of the association (y axis, −log10 scale). We found three genome-wide significant associations, indicated by associations exceeding the dashed line at P = 5 × 10−8.

The lead 8q21 SNP was rs16906115, a common variant in an intron of *IL7*, with an HR = 2.0 (1.6–2.5) (*P* = 3.8 × 10^−9^, HR corrected for imputation error; [Methods](#S8), [Fig. 2a](#F2),[d](#F2) and [Supplementary Fig. 7](#SD1)). Within individual cancer types, a consistent sign was observed in 9 out of 11 cancer types (*P* = 2.7 × 10^−2^ by a one-sided binomial test) with nominal significance (*P* < 0.05) in non-small-cell lung cancer, melanoma, renal cell cancer, bladder cancer, cancer of unknown primary, as well as the collection of ‘other’ less common cancer types ([Fig. 2a](#F2)). The lead 1p36 SNP was rs75824728, a common variant in an intron of *IL22RA1*, with an HR = 1.9 (1.5–2.4) (*P* = 8.4 × 10^−9^; [Extended Data Fig. 2a](#F6)). This SNP was also nominally significantly associated with high-grade irAEs with a comparable effect size (HR = 1.5 (1.1–2.0), *P* = 1.5 × 10^−2^; [Supplementary Fig. 8a](#SD1)). Motivated by this nominal effect on high-grade irAEs, we incorporated an additional 734 patients with germline and irAE data, which further increased the significance of this association (HR = 1.4 (1.1–1.8), *P* = 4.1 × 10^−3^; [Supplementary Fig. 8b](#SD1)). Within individual cancer types, the association was nominally significant in non-small-cell lung cancer, melanoma, breast cancer, as well as the collection of ‘other’ less common cancer types ([Extended Data Fig. 2a](#F6)). The lead 4p15 SNP was rs113861051 with an HR = 2.0 (1.6–2.6) (*P* = 1.1 × 10^−8^) ([Extended Data Fig. 2b](#F6)).

### Fig. 2 |. Discovery associations and replication in MGH and CT cohorts.

![Fig. 2 |](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9adc/10958775/dda0bc7a3b53/nihms-1969921-f0002.jpg)

a–c, Forrest plot of genome-wide significant association (reference dosage for allele G) with all-grade irAEs at 8q21 in the Profile cohort (glioma did not converge due to the low number of events and was therefore excluded) (a), MGH cohort (b) and CT cohort (c). The error bars correspond to the 95% confidence interval (CI) around the mean effect size. Significance was obtained from a two-sided Wald test. d–f, Nonparametric Aalen–Johansen estimator (Methods) for the cumulative incidence of adverse events after ICI initiation stratified on SNP dosage in the DFCI discovery cohort (d), MGH replication cohort (e) and using a Kaplan–Meier estimator in the CT cohort (f). The shaded areas correspond to the 95% CIs.

We evaluated potential modifiers or interactions of the discovered associations. First, sex and age did not significantly interact with any of the genome-wide significant associations with all-grade irAEs ([Supplementary Table 5](#SD2)). Second, using a normative cohort of >23,000 pan-cancer patients not on ICIs at DFCI, no significant association between any of the three SNPs and the time from sequencing to the first toxicity (using the same International Classification of Diseases (ICD) codes as for all-grade irAEs) was observed ([Supplementary Fig. 10](#SD1)), indicating that the SNP effects were specific to the ICI setting. Likewise, none of the three lead SNPs were significantly associated with previous autoimmune disease defined based on ICD codes, nor with a polygenic risk score (PRS) for autoimmune disease^17^ either in the cohort on ICIs or in patients not on ICIs ([Methods](#S8) and [Supplementary Fig. 9](#SD1)), suggesting that these were not generic autoimmune disease variants. Fourth, we carried out a broad scan for germline, clinical and somatic features (including tumor mutational burden) associated with irAEs or interacting with the identified SNPs but observed no significant associations after multiple test correction ([Supplementary Table 6](#SD2)), underscoring the contribution of our germline findings to irAEs. Finally, we investigated various adjustments for the competing risk of death, immortal time bias and inclusion/exclusion of individuals with immune-related diagnoses at the start of treatment and observed no significant impact on these associations ([Methods](#S8) and [Supplementary Fig. 11](#SD1)).

### Independent replication of the IL7 variant

We evaluated the three discovery SNPs in two independent cohorts for replication (see [Methods](#S8) for cohort details). The rs16906115 variant near *IL7* replicated significantly (HR = 2.5 (1.4–4.5), *P* = 1.9 × 10^−3^) in an independent cohort of 265 patients on ICIs treated at Massachusetts General Hospital (MGH cohort; [Methods](#S8)) with severe irAEs requiring hospitalization and confirmed by chart review ([Fig. 2b](#F2),[e](#F2) and [Extended Data Fig. 3](#F7)). rs16906115 also replicated nominally (HR = 1.2 (1.0–1.5), *P* = 0.05) in a second cohort of 2,275 patients on clinical trials (CT cohort) for ICIs with grade 2–5 irAEs recorded as part of the trial ([Fig. 2c](#F2),[f](#F2)). Although no significant outliers were observed, a test for heterogeneity of effect sizes across trials was nominally significant (*P* = 0.02), primarily driven by the IMpassion130 triple-negative breast cancer study. Further stratifying by responders and nonresponders ([Supplementary Table 7](#SD2)), the *IL7* SNP association became more significant in responders (HR = 1.38, *P* = 0.033) and less significant in nonresponders (HR = 1.21, *P* = 0.30), although the difference was not statistically significant. Subanalyses did not show significant associations with any other event grade ([Supplementary Figs. 12](#SD1)–[14](#SD1)) or irAE subtype ([Supplementary Figs. 12](#SD1) and [15](#SD1)). The other two associations, rs75824728 near IL22RA1 and rs113861051 at 4p15, did not replicate in either independent cohorts, although all three associations were significant in a meta-analysis with the MGH cohort. (Due to data constraints, we could not perform a genome-wide meta-analysis with the CT cohort.) Since the CT cohort had detailed response data, we tested for a direct association between each of the three irAE SNPs and progression-free survival or overall survival, and none were significant. Lastly, while this manuscript was in preparation, the variant near *IL7* was independently replicated in a third cohort of 214 melanoma patients on ICIs in the UK with severe (grade 3 or above) irAEs requiring corticosteroids, which was further molecularly characterized in parallel work^18^. Thus, the *IL7*-associated variant replicated in a total of three independent cohorts ([Supplementary Table 8](#SD2)).

### Colocalization of IL7 GWAS variant with a new IL7 cryptic exon

We sought to identify a putative mechanism for the *IL7* locus by integrating our GWAS with molecular data. In tissue-specific expression quantitative trait loci (QTLs) mapped by the GTEx consortium^19^, the lead irAE SNP was significantly associated with *IL7* exon junction usage in the testis for the chr8:78,740,082–78,749,524 (hg38, [Supplementary Fig. 16](#SD1)) junction (which we called *IL7*_junc_) and had an *R*^2^ of 0.98 to the lead *IL7*_junc_ QTL (rs7816685), which was also in the irAE GWAS credible set ([Supplementary Table 9](#SD2) and [Supplementary Fig. 17](#SD1)). By inspection of the raw RNA sequencing (RNA-seq) coverage and junction plots, we observed that carriers of the risk allele exhibited splicing and activation of a new 70-base pair (bp) cryptic exon (spanning chr8:78,746,601–78,746,671, which we called *IL7*_ce_ for ‘cryptic exon’), whereas new junction reads were entirely absent from all homozygous noncarriers ([Fig. 3a](#F3)). The SNP had a stronger effect on *IL7*_ce_ and explained the association with *IL7*_junc_ in a conditional analysis, consistent with *IL7*_ce_ being the causal mediator ([Fig. 3b](#F3) and [Supplementary Fig. 16](#SD1)). rs7816685 was the most significant QTL and the only variant located in the splice region of *IL7*_ce_ and was predicted to be −1bp from an acceptor gain region for *IL7* by both SpliceAI^20^ and Pangolin^21^. Using a method for de novo iso-form reconstruction ([Methods](#S8)) identified a new transcript spanning chr8:78,732,772–78,746,671, which initiated at rs7816685 and was significantly associated with the SNP: detected in 20 out of 54 carriers and 0 out of 54 noncarriers (odds ratio = 10, *P* = 1.5 × 10^−4^; [Supplementary Table 10](#SD2) and [Extended Data Fig. 4](#F8)). rs7816685 was thus consistent with initiating a new IL7 transcript, although other correlated SNPs cannot be fully ruled out without experimental validation.

### Fig. 3 |. IL7 SNP effect cryptic exon activity in GTEx data.

![Fig. 3 |](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9adc/10958775/814f7010659e/nihms-1969921-f0003.jpg)

a, Sashimi plot of alternative splicing of IL7 stratified on the lead splice QTL, with the putative causal variant shown below and the cryptic exon highlighted (IL7ce). b, Cryptic exon activity stratified by lead splice QTL genotype (n = 322). c, Significance of coexpression of IL7 and IL7ce across GTEx tissues (Pearson correlation). Significance was obtained from linear regression (two-sided t-test). For the box plot, the bounds of the boxes are the 25th and 75th percentiles around the median; the minima and maxima of the whiskers correspond to 1.5× the interquartile range (IQR), additional points further from the median are shown as outliers.

Considering *IL7*_ce_ as the putative functional mechanism, we next quantified its activity in a broader set of tissues and cell types. Across the GTEx tissues, *IL7*_ce_ expression was low for most tissues except for testis and lymphoblastoid cell lines (LCLs) exhibiting high outlier expression ([Supplementary Fig. 18a](#SD1)), the latter consistent with the role of *IL7* in lymphoid cell development. LCLs uniquely exhibited significant correlation between *IL7*_ce_ and total *IL7* expression ([Fig. 3c](#F3)) as well as significantly higher *IL7:IL7R* coexpression in the presence of *IL7*_ce_ (*P* = 3.4 × 10^−3^; [Supplementary Fig. 18b](#SD1)), suggesting that *IL7*_ce_ may stabilize *IL7* expression or increase *IL7R* binding in lymphocytes. To better understand the precise cell type of action, we mapped *IL7*_ce_ in publicly available RNA-seq from sorted immune-related cells from patients with autoimmune diseases: *IL7*_ce_ was highly expressed in B cells and moderately expressed in CD4 T cells, with no observable expression in the other immune cell types ([Supplementary Fig. 19](#SD1)). In parallel work, the B cell-specific effect of rs16906115 on *IL7* was confirmed in patients with melanoma receiving ICIs and its influence on T cell development was further functionally characterized^18^.

### Association of IL7 variant with response in patients not on ICIs from The Cancer Genome Atlas

As noted above, the IL7 SNP was not associated with overall survival in any of our cohorts on ICIs and we further investigated this effect in treated patients from the The Cancer Genome Atlas (TCGA). We focused on 433 patients with cutaneous melanoma (SKCM), which included both primary and metastatic patients, in contrast to other TCGA cancers. Strikingly, we observed a significant association between the IL7 germline SNP and favorable progression-free survival (HR = 0.47 per irAE-increasing allele after adjusting for age and stage, *P* = 1.7 × 10^−5^, [Supplementary Table 11](#SD2)), with consistent and significant effects for overall survival and other end points ([Extended Data Figure 5](#F9)), indicating that the irAE-increasing allele can also exhibit an antitumor influence outside of the ICI setting. We attempted to quantify *IL7*_ce_ activity in the RNA-seq data but, in contrast to the newer GTEx data which used longer reads and higher mean depth, coverage of this region in the TCGA was low; therefore, no junction-spanning reads were observed. The IL7 SNP was nominally associated with raw *IL7*_ce_ coverage (*P* = 4.5 × 10^−3^) and became more significant as an interaction with deconvoluted B cell proportions^22^ (*P* = 1.5 × 10^−7^; [Methods](#S8)), consistent with B cell specificity ([Supplementary Fig. 20](#SD1)), although we caution that both *IL7*_ce_ coverage and cell type deconvolution are probabilistic estimates with uncertainty. Overall, we found clear evidence of a germline effect on antitumor response in this independent and conventionally treated population.

### Association of IL7 variant with lymphocyte homeostasis

Due to the known role of *IL7* in lymphocyte homeostasis^23^, we explored whether the influence of rs16906115 on irAEs was reflected in the peripheral blood lymphocyte count from clinical laboratory data. As a surrogate for lymphocyte expansion/homeostasis, we defined the change in relative lymphocyte count (percentage of circulating white blood cell count) using measurements 30 d before/after ICI initiation for patients in the DFCI and MGH cohorts (we refer to this as δLCICI). In the DFCI cohort, carriers of the risk allele exhibited no significant change in lymphocytes (median δLCICI=0.20 (−0.80 to 1.2], *P* = 0.69) whereas noncarriers had significantly reduced δLCICI (median δLCICI=−0.90 (−1.3 to 0.50), *P* = 2.3 × 10^−6^ by paired Wilcoxon test), which was replicated in the MGH cohort (median δLCICI=−4.9 (−6.4 to 3.5), *P* = 3.8 × 10^−11^ for non-carriers and median δLCICI=−0.125 (−3.4 to 3.35), *P* = 0.95 for carriers). The difference in δLCICI between carriers and noncarriers was significant in both the DFCI (difference in mean δLCICI=−1.1 (−2 to 0), *P* = 0.040) and MGH cohorts (δLCICI=−4.8 (−8.5 to −1.3), *P* = 0.0080; [Fig. 4](#F4)), as well as an independent cohort of patients with melanoma^18^. Similarly, δLCirAE defined 30 d before versus after irAEs was stable for carriers (*P* = 0.49) but not for noncarriers (*P* = 2.2 × 10^−3^), although this association may be complicated by steroid use ([Supplementary Fig. 21](#SD1), [Supplementary Table 13](#SD2)). Thus, the *IL7* variant had a consistent stabilizing effect on lymphocyte counts at the initiation of ICI therapy and at the onset of irAE. Results were similar when using absolute lymphocyte count. Lastly, we investigated whether this phenomenon pointed to broader lymphocyte dynamics irrespective of genotype status. Indeed, higher δLCICI was nominally associated with increased irAE incidence (HR = 1.2 per s.d., *P* = 0.018) and a concomitant increase in overall survival for those patients not experiencing any irAE (HR = 0.87, *P* = 1.6 × 10^−3^) in the DFCI cohort ([Supplementary Table 12](#SD2)), although we note that δLCICI is likely a noisy surrogate for the underlying dynamic immune process and not a direct biomarker itself.

### Fig. 4 |. IL7 SNP effect on lymphocyte homeostasis in patients on ICIs.

![Fig. 4 |](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9adc/10958775/9fec1ecc65c1/nihms-1969921-f0004.jpg)

a–e, Lymphocyte counts (percentage of total number of circulating leukocytes) up to 30 d before and after ICI initiation for cases and controls. Two-sided paired Wilcoxon test between time points in carriers and noncarriers in the DFCI (n = 1,375) (a) and MGH (n = 251) (b) cohort. Two-sided Wilcoxon test of the difference in lymphocyte counts before versus after ICI initiation between carriers and noncarriers in the Profile (n = 1,375) (c) and MGH cohort (n = 251) (d). Two-sided paired Wilcoxon test between before and after first irAE in carriers and noncarriers in the DFCI cohort (n = 337) (e). For the box plots, the bounds of the boxes are the 25th and 75th percentiles around the median; the minima and maxima of the whiskers correspond to 1.5× the IQR; additional points further from the median are shown as outliers. f,g, Association between difference in lymphocyte counts before and after ICI initiation and developing irAE (f) as well as death without an irAE (g). The error bars correspond to the 95% CI around the mean effect size. Significance was obtained with a two-sided Wald test.

## Discussion

We conducted a GWAS of irAEs in an observational pancancer setting, identifying three new genome-wide significant associations, with replication of a variant near *IL7* in three independent cohorts. This variant appeared to initiate a new cryptic isoform of *IL7*, was predictive of lymphocyte stability in patients on ICIs and improved prognosis in TCGA melanoma, which is predominantly a cohort not on ICIs. Although we focused this work on the mechanistic follow-up of IL7 due to its consistent replication, the independent associations near IL22RA1 and 4p15 may pinpoint additional mechanisms.

Although the putative *IL7* mechanism identified in this work has not previously been linked to irAEs, *IL7* has been extensively studied for its involvement in immune response and autoimmune disease. *IL7* has a critical role in the development and maturation of T cells, limits organ toxicity during antiviral immune response and supports aberrant immune activity in autoimmune disease^24^. There is evidence that *IL7* expression blocks PD-1, leading to type 1 diabetes^25^, as well as involvement in the development of chronic colitis^26^, functioning like a natural checkpoint inhibitor^27^. The administration of *IL7* in patients with cancer results in increased lymphocyte counts (particularly CD4^+^ and CD8^+^ T cell counts) and reduced regulatory T cell counts^23^. Therefore, it is plausible that the *IL7* risk variant results in a more facilitatory milieu for autoimmune/autoreactive immune responses in patients on ICIs, explaining its association with irAEs. Several studies have shown that *IL7* receptor blockade can reverse the autoimmune response^25,28^, offering a potential therapeutic avenue for managing *IL7*-mediated irAEs.

Our work highlights a complex relationship between irAEs and clinical response. In the cohorts on ICIs, the *IL7* SNP was associated with increased irAEs but was not associated with clinical responses or survival. In the TCGA cohort with melanoma, largely not treated with ICIs, the *IL7* SNP was associated with improved survival. Taken together, the IL7 SNP may thus inform treatment, with carriers exhibiting better clinical outcomes off ICIs but more irAEs on ICIs. We additionally observed *IL7* SNP carriers to exhibit increased lymphocyte stability, whereas lymphocyte stability itself, as a pharmacodynamic biomarker, was associated with both increased irAEs and improved overall survival. Thus, we hypothesize that lymphocyte stability may capture multiple immunological processes: in carriers of the germline SNP, a host autoimmune response to ICIs that leads to irAEs; in noncarriers, a broader antitumor response that can lead to improved survival and irAEs ([Extended Data Fig. 6](#F10)). While our study has uncovered a genetic instrument, detailed response and treatment history data in a large, genotyped cohort will enable further dissection of these processes.

Our study has several limitations. First, the heterogeneity of irAE presentation and severity led us to define two partially overlapping outcomes. In the discovery GWAS, irAEs were manually abstracted from clinical notes and algorithmically inferred using EHR data (followed by manual quality control) and may have thus included some events with ambiguous causes, especially for irAEs that were observed well after the treatment was administered^29^. Second, while our study was sufficiently large to discover replicating associations, the power to identify moderate effect sizes was still low and additional associations with irAEs may yet be discovered (particularly within individual cancer types). The substantial differences in power between the all-grade and high-grade outcomes also made characterization of grade-specific effects challenging. Third, the observational nature^30^ of the DFCI/MGH cohorts likely resulted in a heterogeneous patient population. Although we attempted to control for common covariates, most patients had a complex treatment history that could not be modeled. We expect this heterogeneity to primarily influence power and generalizability since germline genetic variation cannot be caused by unmodeled confounders. Fourth, we restricted our study to individuals of European ancestry to mitigate possible population stratification^31^ but further studies in individuals of non-European ancestry are warranted to understand the generalizability of these associations. In particular, the associated variant rs7816685 near *IL7* has an allele frequency of 31% in East Asian populations (compared to 6.5% in Europeans) and may thus explain more variance in irAEs in Asian patients. Fifth, the use of imputation from tumor-only panel sequencing for the discovery GWAS produced imputed variants with more noise than direct genotyping and likely excluded some difficult-to-impute or rare polymorphisms. This limitation also offers an opportunity for further analysis of this variant in existing panel sequencing datasets^32^.

The identification of genetic variants associated with irAEs is consistent with a hypothesized patient-specific immunological set point and opens avenues for future analysis to inform the genetic architecture of irAEs including: genetic correlation with other complex traits^33^; PRS for patient stratification^34^; and Mendelian randomization to estimate the causal influence of irAEs on other cancer outcomes^35^. Larger studies will enable polygenic analyses to uncover the cell types, gene sets and pathways that drive these outcomes. Ultimately, the utility of these associations to identify high-risk patients for monitoring or treatment modifications must be evaluated in prospective, randomized trials in conjunction with their influence on antitumor response.

## Methods

### Cohort definition, consent and genotyping

This research complies with all relevant ethical regulations. Analyses were carried out across three cohorts with genotyping and clinical information.

#### DFCI cohort.

A total of 1,751 patients of European ancestry (to avoid any confounding from population stratification) were treated with ICIs at the DFCI from 2013 to 2020 ([Table 1](#T1)), across 12 cancer types. Approximately 90% of patients were treated with PD-1/PD-L1 inhibitors and approximately 10% with combination immunotherapy, defined as both CTLA4 and PD-1/PD-L1 ([Supplementary Table 14](#SD2)). Patients were biopsied and sequenced on the OncoPanel tumor sequencing platform^36^ targeting 275–447 cancer genes and germline SNPs were imputed using ultra-low-coverage off-target reads^37^ with imputation accuracy evaluated using a partially overlapping set of directly genotyped individuals ([Supplementary Fig. 1](#SD1)). For normative comparisons, a pancancer control cohort of 23,763 individuals treated with non-ICI therapies at the DFCI was similarly sequenced and imputed through the same pipeline. DFCI samples were selected and sequenced from patients who were consented under institutional review board (IRB)-approved protocol 11–104 and 17–000 from the Dana-Farber/Partners Cancer Care Office for the Protection of Research Subjects. Written informed consent was obtained from participants before inclusion in this study. Secondary analyses of previously collected data were performed with approval from the Dana-Farber IRB: DFCI IRB protocol 19–033 and 19–025. Waiver of Health Insurance Portability and Accountability Act authorization was approved for both protocols.

#### MGH cohort.

This was an independent pancancer cohort of 265 patients on ICIs at the MGH with direct germline genotyping on the Illumina Multi-Ethnic Genotyping Array (MEGA) ([Table 1](#T1)). Occurrence of high-grade irAEs (33 cases, 163 controls) was obtained through the Severe Immunotherapy Complications Program at MGH for inpatient management of high-grade irAEs. Each high-grade irAE was clinically confirmed by an oncology team with expertise in diagnosing and managing irAEs and secondarily verified by organ-specific clinical irAE experts at the corresponding disease center. Cancer types with fewer than 30 patients were combined into ‘other’ cancers ([Supplementary Table 15](#SD2)). This study was approved by the Massachusetts General Brigham IRB (protocol no. 2020P002307), which waived the informed consent requirement because only deidentified data were used. The authors acknowledge that data reporting was consistent with the IRB-approved protocol for deidentified reporting of patient data.

#### CT replication cohort.

A second replication analysis of individual associations was carried out in 2,275 patients who were treated with atezolizumab (anti-PD-L1) and were of European ancestry and met sample and genetic data quality control from 12 previously published clinical trials sponsored by F. Hoffmann–La Roche/Genentech ([Supplementary Table 16](#SD2)). Studies included trials of atezolizumab in renal cell carcinoma (IMmotion), lung cancer (IMpower), triple-negative breast cancer (IMpassion), urothelial cancer (IMvigor) and advanced cancers (indoleamine 2,3-dioxygenase; majority lung, breast or ovarian). All patients provided informed consent for the respective main study. A subset of patients signed an optional Research Biosample Repository (RBR) Informed Consent Form (ICF) to provide whole-blood samples for future research, including study of inherited and noninherited genetic variation from these whole-blood samples. Ethics committees and IRBs at each study site for each clinical trial approved the clinical trial protocol, the main study ICF and the RBR ICF ([Supplementary Table 20](#SD2)). Whole-genome sequencing data were collected from whole blood (as described previously^38^) and used to compute individual variant association statistics.

#### TCGA SKCM cohort.

Data from 433 patients with melanoma (SKCM) in the TCGA were accessed through the Genomic Data Commons Data Portal. Germline data were called from normal/blood on an Affymetrix SNP 6.0 SNP array using Birdsuite v1.5.5 and then imputed using the Michigan Imputation Server. Outcomes and clinical covariates for the TCGA data were accessed from the Clinical Data Resource^39^.

### Statistical analysis

The GWAS was carried out across all variants in the DFCI and MGH cohorts for association with time to irAE separately for each irAE definition. In all cohorts, individuals were restricted to European ancestry. Due to the competing risk of death while on treatment, a cause-specific HR was computed for every SNP using a mixed-effects model^40^, equivalent to censoring on death or loss to follow-up. In each cohort, covariates were included for ancestry, age, sex and line/type of treatment (see below). Statistical fine-mapping ofgenome-wide significant loci was carried out using the SuSIE software v.0.9.57 (ref. ^41^). irAE probabilities and cumulative incidence were quantified using the Aalen–Johansen estimator, a nonparametric estimator that accounts for competing risks^42^. Associations between irAEs and overall survival were evaluated using a time-dependent covariate coded as 0 for controls and as 1 starting from the time of the first irAE. For the TCGA SKCM survival analyses, tests were performed using Cox proportional-hazards regression (with age, sex and stage as covariates) and visualized using Kaplan–Meier curves.

### Analysis of molecular data

Associations were functionally characterized using publicly available gene expression and splicing data from multiple resources. Variants were connected to putative target genes using gene expression and splicing QTLs across 44 tissues from the GTEx consortium^43^. RNA-seq BAM files were downloaded from the GTEx repository, splice junction usage was analyzed using ggsashimi v1.1.5 (ref. ^44^), de novo transcript reconstruction was conducted using Cufflinks v.2.2.1 (ref. ^45^) and candidate coding regions were inferred using TransDecoder v5.5.0. Cell sorted data across six immune cell subsets from individuals with autoimmune diseases and healthy controls were accessed from Collado-Torres et al.^46^ and the Gene Expression Omnibus (SRP045500). Pancancer RNA-seq BAM files from the TCGA were used to quantify expression across tumor sites^47^ and correlated against previously defined immune populations and signals^48^. Analyses of read-level activity and cryptic splicing were carried out using the recount2 framework^49^. Clinical laboratory measurements were extracted from EHR data via the Oncology Data Retrieval System^50^ framework for the DFCI cohort and the Research Patient Data Registry^51^ for the MGH cohort.

### Sample collection, genotype imputation and quality control

#### DFCI cohort.

The DFCI cohort was sequenced as part of the Profile project, a prospective clinical sequencing effort for consented patients undergoing routine treatment at the DFCI and affiliated hospitals. A custom targeted hybrid capture sequencing platform (OncoPanel) was used to assay genomic variation from tumor biopsies. Each sample was sequenced on 1 of 3 panel versions targeting the exons of 275,300 and 447 genes, respectively. Samples meet a minimum of 30X coverage for 80% of targets for analysis. Somatic variation (including single-nucleotide variants, insertions/deletions and copy number variation) was called by the Profile clinical bioinformatics pipeline and signed out by a pathologist at Brigham & Women’s Hospital after technical review, as described previously^36^. Off-target and on-target reads from the sequenced BAMs were imputed using the STITCH imputation software v1.6.6 (ref. ^37,52^). Imputed variants were restricted to a minor allele frequency (MAF) > 1% and imputation INFO score >0.4. Genetic ancestry was inferred using principal component projection with the SNPWEIGHTS software v2.1. Continental components were used to exclude individuals of non-European ancestry; within-Europe components were included as covariates.

A partly overlapping cohort of 833 individuals (126 overlapping patients on ICIs) with both OncoPanel tumor sequencing and direct germline SNP array genotyping (on the Illumina MEGA) was used to benchmark imputation accuracy. Pearson correlation for each SNP was computed between the tumor-imputed and germline-genotyped individuals. Mean imputed SNP correlation was 0.86 after variant quality control and highly uniform across the genome ([Supplementary Fig. 1](#SD1)). Any remaining noise was only expected to make the HRs of associations in the GWAS using imputed SNPs more conservative. Detailed analysis of variant imputation accuracy have been described separately and the imputation workflow is publicly available^37^. For visualizations where imputed patients were stratified by variant carrier/noncarrier status, the decision boundary was determined using logistic regression of carrier status on imputed dosage in the samples with both tumor sequencing and SNP array data.

#### MGH cohort.

Blood samples were collected from MGH patients and genotyped on the Illumina MEGA array. Data were imputed to the 1000 Genomes reference panel using the Haplotype Reference Consortium imputation server, followed by quality control removing variants with an MAF < 1% and INFO score < 0.9. Genetic ancestry was inferred using in-sample principal components and restricted to individuals with European ancestry.

#### CT cohort.

A subset of patients signed an optional RBR ICF to provide whole-blood samples for future research, including study of inherited and noninherited genetic variation from these whole-blood samples. Whole-genome sequencing data were collected from whole blood as described previously. Genetic ancestry was inferred using ADMIXTURE and restricted to individuals with European ancestry (ancestry > 0.7). In-sample principal components were also computed to account for any remaining population structure.

### Outcome definitions in the DFCI cohort

Mortality was collected using linkage to the National Death Index through 2019. For patients who died after 2019, a clinical death index from the EHR was used, which captured 86% of occurred deaths when evaluated for patients before 31 December 2019.

The all-grade event definition was obtained by algorithmic abstraction using EHR diagnosis codes. A list of predefined relevant diagnosis codes was used to filter all available codes for potential AEs after the start of treatment and up to 60 d after receiving the last ICI dose. Diagnosis codes present in the EHR of the respective patient before the start of treatment were excluded. Evident false positives were excluded by inspection of the diagnosis code and manual review of the patient chart at the event date to exclude events that did not occur or were clearly linked to non-ICI causes. The search terms used and manual exclusion list of search terms are shown in [Supplementary Table 1](#SD2).

### Previous autoimmune disease and PRS

We investigated the relationships between the identified risk variants and previous autoimmune disease and autoimmune disease risk. We defined patients with previous autoimmune disease based on the occurrence of an autoimmune-related ICD 10th revision codes before the start of ICI treatment. Each irAE lead SNP was then tested for association with previous autoimmune disease, while adjusting for age, sex, treatment year, panel version of the sequencing panel, treatment type, line of treatment and cancer type. As an alternative measure of autoimmune disease risk, we also inferred a PRS for any autoimmune disease from a recent UK Biobank GWAS study (see [Data availability](#S29) section). We confirmed that the PRS was significantly associated with the previous ICD-based autoimmune disease definition in the cohort on ICIs (*P* = 8.8 × 10^−4^). Each irAE lead SNP was again tested for association with the PRS, adjusting for cancer type, age, sex, panel version and the first two principal components to control for ancestry.

### Termination of treatment and steroid administration

For a subset of 44 patients, who were selected based on the highest dosage of the lead *IL7* SNP, information on continuation of treatment after irAE and steroid administration was manually annotated through manual chart review.

### Survival analysis

#### GWAS discovery.

In the DFCI discovery and MGH cohorts, discovery of GWAS variants associated with the risk of irAEs was performed using a multivariate, multistate survival framework modeling with irAE as the primary outcome and death as a competing risk. Direct modeling of competing risks is important for incidence computation to account for potential survivor bias^51^, where individuals who live longer may develop more irAEs by chance. Due to computational constraints, the mixed-effects survival GWAS methodology did not allow for stratified covariates and flexible truncation. Thus, we reestimated the top associations (*P* < 5 × 10^−6^) by fixed-effect meta-analysis over the cancer types with stratification of any covariates that exhibited a proportional-hazards violation. Lastly, to account for error in the imputation, we rescaled the HR based on the imputed/genotyped relationship, although we note this is a linear rescaling that does not impact the significance of the association.

Additionally, to account for immortal time bias, 422 patients who were sequenced after the start of ICI treatment were left truncated until sequencing. Left truncation and excluding patients with allograft surgery or immunosuppressants at the start of treatment did not influence any of the genome-wide significant associations ([Supplementary Fig. 11](#SD1)).

In the replication cohort (CT cohort), cause-specific HRs and *P* values were estimated by conventional survival analysis with censoring on death or loss to follow-up. This cause-specific hazard computation (our primary measure of effect size) is equivalent to that estimated from the multistate model, which effectively censors on death and loss to follow-up but gains a bit of statistical power modeling with similar HRs for death and irAE outcome when estimating the technical covariates.

#### Multistate modeling of competing risks.

We employed a time-to-event analysis with irAEs as the event of interest. However, since death precludes from experiencing an irAE, death events were addressed through an illness–death model, a special case of the class of multistate survival models. In this model, patients in the ‘treatment’ state can either experience a transition to ‘irAE’ or ‘death’ without having experienced an irAE. Furthermore, patients who have experienced an irAE can also transition to the death state. For any transition in the multistate survival model, censoring due to loss to follow-up and left truncation due to delayed sequencing were employed.

In the setting of multistate survival models, there are two possible HRs one might be interested in: the cause-specific hazard and the subdistribution hazard. While the subdistribution hazard quantifies the risk for the incidence of the event in the population, the cause-specific hazard quantifies the inherent risk of a patient experiencing an event conditioned on that patient being event-free. Therefore, the cause-specific HR corresponds to the infinitesimal generator of transitions in a Markov jump process with added censoring. Since we are interested in the biological mechanism of experiencing an irAE, the primary quantity of interest is the cause-specific HR (see further discussion in Austin & Fine^52^). The subdistribution hazard, which takes into account the risk of the competing death event given from the same covariate, is of secondary interest primarily from an epidemiological perspective.

To address the challenge of estimating the cumulative population-level incidence/probability of an irAE in the multistate setting, we employed the Aalen–Johansen estimator^41^. We treated irAEs as a transient state to obtain the probability over time to have experienced an irAE but be alive and irAEs as an absorbing state to obtain the cumulative incidence of irAEs over time.

#### Computational constraints.

When running a time-to-event GWAS, we sought to use methods that satisfied multiple requirements: (1) a mixed-model survival model to adjust for latent relatedness structure in the genetic data; (2) left truncation at sequencing time to alleviate immortal time bias; (3) proper adjustment for meaningful covariates, in the sense that we did not just want the best fit but to adjust such that the HR of the tested SNP is unbiased. This necessitates stratifying over covariates that do not satisfy the proportional-hazards assumption. Tools that can do all three things in finite time are, to the best of our knowledge, not available. Therefore, we ran a GWAS with coxmeg v.1.0.11, a fast tool that does not incorporate left truncation and adjustments for proportional-hazards violations, and then reran the top associations using the full multistate approach (manually implemented in R using the mstate v.0.2.11 package), in total needing 2–3 d on a cluster.

### Power analysis

We conducted a power analysis using the sample size calculator of the San Francisco Clinical and Statistical Science Institute. Using the HRs in the DFCI discovery cohort, we quantified how many events would be needed to discover the association at 80% power ([Supplementary Table 17](#SD2)). Using the number of events we observed for all-grade and high-grade irAEs, we quantified the relationship between the underlying HR and statistical power to discover an association at genome-wide significance (*P* < 5 × 10^−8^) ([Supplementary Fig. 3](#SD1)).

### Covariate adjustment

In the DFCI discovery cohort, covariates were included for: two within-Europe ancestry components (after restricting to individuals with European ancestry; see above); age at the start of treatment; self-identified sex; line of treatment as determined from the EHR medication records; start year of treatment; type of treatment (PD-1/PD-L1 or CTLA4 monotherapy or combination); concurrent alternate treatment (chemotherapy, targeted therapy); and two technical covariates adjusting for the version of the targeted panel and an indicator for sequencing after the start of treatment. Patients were grouped into cancer types with >30 individuals and the analyses were stratified or meta-analyzed over cancer types (as indicated). In the MGH cohort, covariates were included for cancer type, type of ICI, age at the start of treatment, sex and genetic ancestry. Cancer type was included as a covariate rather than a stratifying variable due to the relatively small sample size of each type and the assumption that common covariate effects could be better learned across all samples. In the CT cohort, covariates were included for five genetic ancestry components and stratified on treatment arms (which also capture cancer types).

### Reporting summary

Further information on research design is available in the Nature Portfolio Reporting Summary linked to this article.

## Extended Data

### Extended Data Fig. 1 |. Probability of irAE by treatment class.

![Extended Data Fig. 1 |](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9adc/10958775/33636a426314/nihms-1969921-f0005.jpg)

Probability of patients experiencing high-grade irAEs (a) or all-grade irAEs (b) stratified by therapy class. The difference between CTLA4 and combination therapy was not statistically significant in a log-rank test of the equivalent Kaplan-Meier estimator (p = 0.39 all-grade, p = 0.68 high-grade). The shaded areas correspond to 95% confidence intervals.

### Extended Data Fig. 2 |. Discovery associations of locus near IL22RA1 and 4p15.

![Extended Data Fig. 2 |](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9adc/10958775/6781e3df3afa/nihms-1969921-f0006.jpg)

Discovery associations with rs75824728 (a) and rs113861051 (b) stratified by cancer type. The error bars correspond to the 95% confidence interval around the mean effect size. Significance was obtained from a two-sided Wald test.

### Extended Data Fig. 3 |. Agreement between discovery cohort and MGH cohort at IL7 locus.

![Extended Data Fig. 3 |](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9adc/10958775/1344e0d1c075/nihms-1969921-f0007.jpg)

(a) Logarithmic hazard rates (effect sizes) and (b) p-values for association in the discovery DFCI cohort and the MGH cohort for the 8q21 locus, restricted to suggestive significant associations in the discovery cohort (p < 1.0 × 10−5). The shaded area of the linear fit corresponds to the 95% confidence interval. Significance was tested using a two-sided t-test on the Pearson correlations. (c) Comparison of the association strengths of variants around the top association locus in DFCI and MGH. The 95% credible set in the DFCI cohort is colored in blue. The upper red line signifies genome wide significance, the lower red line bonferroni corrected significance for SNPs tested in the MGH cohort.

### Extended Data Fig. 4 |. De novo isoform reconstruction using Cufflinks.

![Extended Data Fig. 4 |](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9adc/10958775/e06427b14b2a/nihms-1969921-f0008.jpg)

De novo isoform reconstruction using Cufflinks. There is a novel transcript spanning chr8:78732772–78746671, which initiates at rs7816685 and is highly specific to carriers.

### Extended Data Fig. 5 |. Response and overall survival in TCGA Melanoma for carriers and non-carriers of rs16906115.

![Extended Data Fig. 5 |](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9adc/10958775/2bab3ac9c421/nihms-1969921-f0009.jpg)

Response and overall survival in TCGA Melanoma for carriers and non-carriers of rs16906115. Significance was obtained using a log-rank test.

### Extended Data Fig. 6 |. Hypothesized mechanistic schematic.

![Extended Data Fig. 6 |](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9adc/10958775/07986a212376/nihms-1969921-f0010.jpg)

Hypothesized schematic of how lymphocyte stability is a marker of an active host immunity with down-stream effects on both overall survival through better anti-tumor response, as well as higher rate of irAE due to increased auto-immunity.

## Supplementary Material

## Acknowledgements

We thank all the patients who consented to participate in this study and the institutional data collection efforts that made this study possible. We thank M. Hassett, N. Lindeman, D. Liu, P. Lukasse, L. MacConnaill, P. Polak, S. Rodig, N. Zaitlen and E. Ziv for helpful discussions; the DFCI Oncology Data Retrieval System for the aggregation, management and delivery of the clinical and operational research data used in this project; and the DFCI/Brigham and Women’s Hospital Data Sharing Group for the aggregation, management and delivery of the clinical and genomics data used in this project. A.G. is supported by National Institutes of Health (NIH) grant nos. R01CA227237, R01CA244569 and R21HG010748, and awards from the Claudia Adams Barr Foundation, Louis B. Mayer Foundation, Doris Duke Charitable Foundation, Emerson Collective and Phi Beta Psi Sorority. S.A.S. acknowledges support from the National Cancer Institute (no. R50RCA211482). S.G. was supported by NIH grant no. R01CA227237 and a DFCI Trustee Fellowship. T.K.C. is supported in part by the Dana-Farber/Harvard Cancer Center Kidney SPORE (no. 2P50CA101942–16) and Program no. 5P30CA006516–56, the Kohlberg Chair at Harvard Medical School and the Trust Family, Michael Brigham, Pan-Mass Challenge and Loker Pinard Funds for Kidney Cancer Research at the DFCI. T.E.K. acknowledges grant support from the NIH (no. T32CA009172).

## Footnotes

## Data availability

Full summary association statistics for the discovery cohort are available at *Zenodo* [https://zenodo.org/record/6800429](https://zenodo.org/record/6800429). The deidentified clinical outcomes and three main associations are available in [Supplementary Table 18](#SD2) for all-grade and [Supplementary Table 19](#SD2) for high-grade irAEs. The UK Biobank association statistics for autoimmune disease were previously computed by BOLT-LMM v.2.3 and used to estimate the autoimmune disease PRS ([https://data.broadin-stitute.org/alkesgroup/UKBB/UKBB_409K/](https://data.broadin-stitute.org/alkesgroup/UKBB/UKBB_409K/)). The RNA-seq data from the GTEx and TCGA was accessed through the ReCount2 interface and API ([https://jhubiostatistics.shinyapps.io/recount/](https://jhubiostatistics.shinyapps.io/recount/)). Cell-sorted data across six immune cell subsets from individuals with autoimmune diseases and healthy controls were accessed from Chowell et al.^11^ and the GEO (SRP045500).

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

Full summary association statistics for the discovery cohort are available at *Zenodo* [https://zenodo.org/record/6800429](https://zenodo.org/record/6800429). The deidentified clinical outcomes and three main associations are available in [Supplementary Table 18](#SD2) for all-grade and [Supplementary Table 19](#SD2) for high-grade irAEs. The UK Biobank association statistics for autoimmune disease were previously computed by BOLT-LMM v.2.3 and used to estimate the autoimmune disease PRS ([https://data.broadin-stitute.org/alkesgroup/UKBB/UKBB_409K/](https://data.broadin-stitute.org/alkesgroup/UKBB/UKBB_409K/)). The RNA-seq data from the GTEx and TCGA was accessed through the ReCount2 interface and API ([https://jhubiostatistics.shinyapps.io/recount/](https://jhubiostatistics.shinyapps.io/recount/)). Cell-sorted data across six immune cell subsets from individuals with autoimmune diseases and healthy controls were accessed from Chowell et al.^11^ and the GEO (SRP045500).

### Supplementary Materials

### Data Availability Statement

Full summary association statistics for the discovery cohort are available at *Zenodo* [https://zenodo.org/record/6800429](https://zenodo.org/record/6800429). The deidentified clinical outcomes and three main associations are available in [Supplementary Table 18](#SD2) for all-grade and [Supplementary Table 19](#SD2) for high-grade irAEs. The UK Biobank association statistics for autoimmune disease were previously computed by BOLT-LMM v.2.3 and used to estimate the autoimmune disease PRS ([https://data.broadin-stitute.org/alkesgroup/UKBB/UKBB_409K/](https://data.broadin-stitute.org/alkesgroup/UKBB/UKBB_409K/)). The RNA-seq data from the GTEx and TCGA was accessed through the ReCount2 interface and API ([https://jhubiostatistics.shinyapps.io/recount/](https://jhubiostatistics.shinyapps.io/recount/)). Cell-sorted data across six immune cell subsets from individuals with autoimmune diseases and healthy controls were accessed from Chowell et al.^11^ and the GEO (SRP045500).

## References

1. Ribas A & Wolchok JD Cancer immunotherapy using checkpoint blockade. Science 359, 1350–1355 (2018).  [DOI](https://doi.org/10.1126/science.aar4060) | [PMC free article](/articles/PMC7391259/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29567705/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Cancer%20immunotherapy%20using%20checkpoint%20blockade&author=A%20Ribas&author=JD%20Wolchok&volume=359&publication_year=2018&pages=1350-1355&pmid=29567705&doi=10.1126/science.aar4060&)

2. June CH, Warshauer JT & Bluestone JA Corrigendum: Is autoimmunity the Achilles’ heel of cancer immunotherapy? Nat. Med 23, 1004 (2017).  [DOI](https://doi.org/10.1038/nm.4321) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28475571/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Med&title=Corrigendum:%20Is%20autoimmunity%20the%20Achilles%E2%80%99%20heel%20of%20cancer%20immunotherapy?&author=CH%20June&author=JT%20Warshauer&author=JA%20Bluestone&volume=23&publication_year=2017&pages=1004&pmid=28475571&doi=10.1038/nm.4321&)

3. Esfahani K et al. Moving towards personalized treatments of immune-related adverse events. Nat. Rev. Clin. Oncol 17, 504–515 (2020).  [DOI](https://doi.org/10.1038/s41571-020-0352-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32246128/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Clin.%20Oncol&title=Moving%20towards%20personalized%20treatments%20of%20immune-related%20adverse%20events&author=K%20Esfahani&volume=17&publication_year=2020&pages=504-515&pmid=32246128&doi=10.1038/s41571-020-0352-8&)

4. Boutros C et al. Safety profiles of anti-CTLA-4 and anti-PD-1 antibodies alone and in combination. Nat. Rev. Clin. Oncol 13, 473–486 (2016).  [DOI](https://doi.org/10.1038/nrclinonc.2016.58) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27141885/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Clin.%20Oncol&title=Safety%20profiles%20of%20anti-CTLA-4%20and%20anti-PD-1%20antibodies%20alone%20and%20in%20combination&author=C%20Boutros&volume=13&publication_year=2016&pages=473-486&pmid=27141885&doi=10.1038/nrclinonc.2016.58&)

5. Koon H & Atkins M Autoimmunity and immunotherapy for cancer. N. Engl. J. Med 354, 758–760 (2006).  [DOI](https://doi.org/10.1056/NEJMe058307) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16481646/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%20Engl.%20J.%20Med&title=Autoimmunity%20and%20immunotherapy%20for%20cancer&author=H%20Koon&author=M%20Atkins&volume=354&publication_year=2006&pages=758-760&pmid=16481646&doi=10.1056/NEJMe058307&)

6. Postow MA, Sidlow R & Hellmann MD Immune-related adverse events associated with immune checkpoint blockade. N. Engl. J. Med 378, 158–168 (2018).  [DOI](https://doi.org/10.1056/NEJMra1703481) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29320654/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%20Engl.%20J.%20Med&title=Immune-related%20adverse%20events%20associated%20with%20immune%20checkpoint%20blockade&author=MA%20Postow&author=R%20Sidlow&author=MD%20Hellmann&volume=378&publication_year=2018&pages=158-168&pmid=29320654&doi=10.1056/NEJMra1703481&)

7. Wang DY et al. Fatal toxic effects associated with immune checkpoint inhibitors: a systematic review and meta-analysis. JAMA Oncol 4, 1721–1728 (2018).  [DOI](https://doi.org/10.1001/jamaoncol.2018.3923) | [PMC free article](/articles/PMC6440712/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30242316/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA%20Oncol&title=Fatal%20toxic%20effects%20associated%20with%20immune%20checkpoint%20inhibitors:%20a%20systematic%20review%20and%20meta-analysis&author=DY%20Wang&volume=4&publication_year=2018&pages=1721-1728&pmid=30242316&doi=10.1001/jamaoncol.2018.3923&)

8. Eggermont AMM et al. Association between immune-related adverse events and recurrence-free survival among patients with stage III melanoma randomized to receive pembrolizumab or placebo: a secondary analysis of a randomized clinical trial. JAMA Oncol 6, 519–527 (2020).  [DOI](https://doi.org/10.1001/jamaoncol.2019.5570) | [PMC free article](/articles/PMC6990933/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31895407/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA%20Oncol&title=Association%20between%20immune-related%20adverse%20events%20and%20recurrence-free%20survival%20among%20patients%20with%20stage%20III%20melanoma%20randomized%20to%20receive%20pembrolizumab%20or%20placebo:%20a%20secondary%20analysis%20of%20a%20randomized%20clinical%20trial&author=AMM%20Eggermont&volume=6&publication_year=2020&pages=519-527&pmid=31895407&doi=10.1001/jamaoncol.2019.5570&)

9. Khan Z et al. Genetic variation associated with thyroid autoimmunity shapes the systemic immune response to PD-1 checkpoint blockade. Nat. Commun 12, 3355 (2021).  [DOI](https://doi.org/10.1038/s41467-021-23661-4) | [PMC free article](/articles/PMC8184890/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34099659/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun&title=Genetic%20variation%20associated%20with%20thyroid%20autoimmunity%20shapes%20the%20systemic%20immune%20response%20to%20PD-1%20checkpoint%20blockade&author=Z%20Khan&volume=12&publication_year=2021&pages=3355&pmid=34099659&doi=10.1038/s41467-021-23661-4&)

10. Khan Z et al. Polygenic risk for skin autoimmunity impacts immune checkpoint blockade in bladder cancer. Proc. Natl Acad. Sci. USA 117, 12288–12294 (2020).  [DOI](https://doi.org/10.1073/pnas.1922867117) | [PMC free article](/articles/PMC7275757/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32430334/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl%20Acad.%20Sci.%20USA&title=Polygenic%20risk%20for%20skin%20autoimmunity%20impacts%20immune%20checkpoint%20blockade%20in%20bladder%20cancer&author=Z%20Khan&volume=117&publication_year=2020&pages=12288-12294&pmid=32430334&doi=10.1073/pnas.1922867117&)

11. Chowell D et al. Patient HLA class I genotype influences cancer response to checkpoint blockade immunotherapy. Science 359, 582–587 (2018).  [DOI](https://doi.org/10.1126/science.aao4572) | [PMC free article](/articles/PMC6057471/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29217585/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Patient%20HLA%20class%20I%20genotype%20influences%20cancer%20response%20to%20checkpoint%20blockade%20immunotherapy&author=D%20Chowell&volume=359&publication_year=2018&pages=582-587&pmid=29217585&doi=10.1126/science.aao4572&)

12. Chowell D et al. Evolutionary divergence of HLA class I genotype impacts efficacy of cancer immunotherapy. Nat. Med 25, 1715–1720 (2019).  [DOI](https://doi.org/10.1038/s41591-019-0639-4) | [PMC free article](/articles/PMC7938381/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31700181/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Med&title=Evolutionary%20divergence%20of%20HLA%20class%20I%20genotype%20impacts%20efficacy%20of%20cancer%20immunotherapy&author=D%20Chowell&volume=25&publication_year=2019&pages=1715-1720&pmid=31700181&doi=10.1038/s41591-019-0639-4&)

13. Cubas R et al. Autoimmunity linked protein phosphatase PTPN22 as a target for cancer immunotherapy. J. Immunother. Cancer 8, e001439 (2020).  [DOI](https://doi.org/10.1136/jitc-2020-001439) | [PMC free article](/articles/PMC7604869/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33127657/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Immunother.%20Cancer&title=Autoimmunity%20linked%20protein%20phosphatase%20PTPN22%20as%20a%20target%20for%20cancer%20immunotherapy&author=R%20Cubas&volume=8&publication_year=2020&pages=e001439&pmid=33127657&doi=10.1136/jitc-2020-001439&)

14. Thorsson V et al. The immune landscape of cancer. Immunity 48, 812–830.e14 (2018).  [DOI](https://doi.org/10.1016/j.immuni.2018.03.023) | [PMC free article](/articles/PMC5982584/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29628290/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Immunity&title=The%20immune%20landscape%20of%20cancer&author=V%20Thorsson&volume=48&publication_year=2018&pages=812-830&pmid=29628290&doi=10.1016/j.immuni.2018.03.023&)

15. Martins F et al. Adverse effects of immune-checkpoint inhibitors: epidemiology, management and surveillance. Nat. Rev. Clin. Oncol 16, 563–580 (2019).  [DOI](https://doi.org/10.1038/s41571-019-0218-0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31092901/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Clin.%20Oncol&title=Adverse%20effects%20of%20immune-checkpoint%20inhibitors:%20epidemiology,%20management%20and%20surveillance&author=F%20Martins&volume=16&publication_year=2019&pages=563-580&pmid=31092901&doi=10.1038/s41571-019-0218-0&)

16. Barrett JH Genome-wide association studies of therapeutic response: addressing the complexities. Pharmacogenomics 20, 213–216 (2019).  [DOI](https://doi.org/10.2217/pgs-2018-0204) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30767725/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Genome-wide%20association%20studies%20of%20therapeutic%20response:%20addressing%20the%20complexities&author=JH%20Barrett&volume=20&publication_year=2019&pages=213-216&pmid=30767725&doi=10.2217/pgs-2018-0204&)

17. Loh P-R, Kichaev G, Gazal S, Schoech AP & Price AL Mixed-model association for biobank-scale datasets. Nat. Genet 50, 906–908 (2018).  [DOI](https://doi.org/10.1038/s41588-018-0144-6) | [PMC free article](/articles/PMC6309610/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29892013/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet&title=Mixed-model%20association%20for%20biobank-scale%20datasets&author=P-R%20Loh&author=G%20Kichaev&author=S%20Gazal&author=AP%20Schoech&author=AL%20Price&volume=50&publication_year=2018&pages=906-908&pmid=29892013&doi=10.1038/s41588-018-0144-6&)

18. Taylor C et al. Genetic variation at IL7 provides mechanistic insights into toxicity to immune checkpoint blockade. Preprint at Research Square 10.21203/rs.3.rs-1531341/v1 (2022).  [DOI](https://doi.org/10.21203/rs.3.rs-1531341/v1)

19. Aguet F et al. The GTEx Consortium atlas of genetic regulatory effects across human tissues. Science 369, 1318–1330 (2020).  [DOI](https://doi.org/10.1126/science.aaz1776) | [PMC free article](/articles/PMC7737656/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32913098/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=The%20GTEx%20Consortium%20atlas%20of%20genetic%20regulatory%20effects%20across%20human%20tissues&author=F%20Aguet&volume=369&publication_year=2020&pages=1318-1330&pmid=32913098&doi=10.1126/science.aaz1776&)

20. Jaganathan K et al. Predicting splicing from primary sequence with deep learning. Cell 176, 535–548.e24 (2019).  [DOI](https://doi.org/10.1016/j.cell.2018.12.015) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30661751/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell&title=Predicting%20splicing%20from%20primary%20sequence%20with%20deep%20learning&author=K%20Jaganathan&volume=176&publication_year=2019&pages=535-548&pmid=30661751&doi=10.1016/j.cell.2018.12.015&)

21. Zeng T & Li YI Predicting RNA splicing from DNA sequence using Pangolin. Genome Biol 23, 103 (2022).  [DOI](https://doi.org/10.1186/s13059-022-02664-4) | [PMC free article](/articles/PMC9022248/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35449021/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Biol&title=Predicting%20RNA%20splicing%20from%20DNA%20sequence%20using%20Pangolin&author=T%20Zeng&author=YI%20Li&volume=23&publication_year=2022&pages=103&pmid=35449021&doi=10.1186/s13059-022-02664-4&)

22. Li T et al. TIMER2.0 for analysis of tumor-infiltrating immune cells. Nucleic Acids Res 48, W509–W514 (2020).  [DOI](https://doi.org/10.1093/nar/gkaa407) | [PMC free article](/articles/PMC7319575/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32442275/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=TIMER2.0%20for%20analysis%20of%20tumor-infiltrating%20immune%20cells&author=T%20Li&volume=48&publication_year=2020&pages=W509-W514&pmid=32442275&doi=10.1093/nar/gkaa407&)

23. Rosenberg SA et al. IL-7 administration to humans leads to expansion of CD8+ and CD4+ cells but a relative decrease of CD4+ T-regulatory cells. J. Immunother 29, 313–319 (2006).  [DOI](https://doi.org/10.1097/01.cji.0000210386.55951.c2) | [PMC free article](/articles/PMC1473976/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16699374/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Immunother&title=IL-7%20administration%20to%20humans%20leads%20to%20expansion%20of%20CD8+%20and%20CD4+%20cells%20but%20a%20relative%20decrease%20of%20CD4+%20T-regulatory%20cells&author=SA%20Rosenberg&volume=29&publication_year=2006&pages=313-319&pmid=16699374&doi=10.1097/01.cji.0000210386.55951.c2&)

24. Barata JT, Durum SK & Seddon B Flip the coin: IL-7 and IL-7R in health and disease. Nat. Immunol 20, 1584–1593 (2019).  [DOI](https://doi.org/10.1038/s41590-019-0479-x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31745336/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Immunol&title=Flip%20the%20coin:%20IL-7%20and%20IL-7R%20in%20health%20and%20disease&author=JT%20Barata&author=SK%20Durum&author=B%20Seddon&volume=20&publication_year=2019&pages=1584-1593&pmid=31745336&doi=10.1038/s41590-019-0479-x&)

25. Penaranda C et al. IL-7 receptor blockade reverses autoimmune diabetes by promoting inhibition of effector/memory T cells. Proc. Natl Acad. Sci. USA 109, 12668–12673 (2012).  [DOI](https://doi.org/10.1073/pnas.1203692109) | [PMC free article](/articles/PMC3411948/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22733744/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl%20Acad.%20Sci.%20USA&title=IL-7%20receptor%20blockade%20reverses%20autoimmune%20diabetes%20by%20promoting%20inhibition%20of%20effector/memory%20T%20cells&author=C%20Penaranda&volume=109&publication_year=2012&pages=12668-12673&pmid=22733744&doi=10.1073/pnas.1203692109&)

26. Totsuka T et al. IL-7 is essential for the development and the persistence of chronic colitis. J. Immunol 178, 4737–4748 (2007).  [DOI](https://doi.org/10.4049/jimmunol.178.8.4737) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17404253/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Immunol&title=IL-7%20is%20essential%20for%20the%20development%20and%20the%20persistence%20of%20chronic%20colitis&author=T%20Totsuka&volume=178&publication_year=2007&pages=4737-4748&pmid=17404253&doi=10.4049/jimmunol.178.8.4737&)

27. Dooms H Interleukin-7: fuel for the autoimmune attack. J. Autoimmun 45, 40–48 (2013).  [DOI](https://doi.org/10.1016/j.jaut.2013.06.007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23831438/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Autoimmun&title=Interleukin-7:%20fuel%20for%20the%20autoimmune%20attack&author=H%20Dooms&volume=45&publication_year=2013&pages=40-48&pmid=23831438&doi=10.1016/j.jaut.2013.06.007&)

28. Belarif L et al. IL-7 receptor blockade blunts antigen-specific memory T cell responses and chronic inflammation in primates. Nat. Commun 9, 4483 (2018).  [DOI](https://doi.org/10.1038/s41467-018-06804-y) | [PMC free article](/articles/PMC6203796/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30367166/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun&title=IL-7%20receptor%20blockade%20blunts%20antigen-specific%20memory%20T%20cell%20responses%20and%20chronic%20inflammation%20in%20primates&author=L%20Belarif&volume=9&publication_year=2018&pages=4483&pmid=30367166&doi=10.1038/s41467-018-06804-y&)

29. Johnson DB, Nebhan CA, Moslehi JJ & Balko JM Immune-checkpoint inhibitors: long-term implications of toxicity. Nat. Rev. Clin. Oncol 19, 254–267 (2022).  [DOI](https://doi.org/10.1038/s41571-022-00600-w) | [PMC free article](/articles/PMC8790946/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35082367/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Clin.%20Oncol&title=Immune-checkpoint%20inhibitors:%20long-term%20implications%20of%20toxicity&author=DB%20Johnson&author=CA%20Nebhan&author=JJ%20Moslehi&author=JM%20Balko&volume=19&publication_year=2022&pages=254-267&pmid=35082367&doi=10.1038/s41571-022-00600-w&)

30. Booth CM, Karim S & Mackillop WJ Real-world data: towards achieving the achievable in cancer care. Nat. Rev. Clin. Oncol 16, 312–325 (2019).  [DOI](https://doi.org/10.1038/s41571-019-0167-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30700859/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Clin.%20Oncol&title=Real-world%20data:%20towards%20achieving%20the%20achievable%20in%20cancer%20care&author=CM%20Booth&author=S%20Karim&author=WJ%20Mackillop&volume=16&publication_year=2019&pages=312-325&pmid=30700859&doi=10.1038/s41571-019-0167-7&)

31. Price AL, Zaitlen NA, Reich D & Patterson N New approaches to population stratification in genome-wide association studies. Nat. Rev. Genet 11, 459–463 (2010).  [DOI](https://doi.org/10.1038/nrg2813) | [PMC free article](/articles/PMC2975875/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20548291/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Genet&title=New%20approaches%20to%20population%20stratification%20in%20genome-wide%20association%20studies&author=AL%20Price&author=NA%20Zaitlen&author=D%20Reich&author=N%20Patterson&volume=11&publication_year=2010&pages=459-463&pmid=20548291&doi=10.1038/nrg2813&)

32. André F et al. AACR Project GENIE: powering precision medicine through an international consortium. Cancer Discov 7, 818–831 (2017).  [DOI](https://doi.org/10.1158/2159-8290.CD-17-0151) | [PMC free article](/articles/PMC5611790/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28572459/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Discov&title=AACR%20Project%20GENIE:%20powering%20precision%20medicine%20through%20an%20international%20consortium&author=F%20Andr%C3%A9&volume=7&publication_year=2017&pages=818-831&pmid=28572459&doi=10.1158/2159-8290.CD-17-0151&)

33. van Rheenen W, Peyrot WJ, Schork AJ, Lee SH & Wray NR Genetic correlations of polygenic disease traits: from theory to practice. Nat. Rev. Genet 20, 567–581 (2019).  [DOI](https://doi.org/10.1038/s41576-019-0137-z) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31171865/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Genet&title=Genetic%20correlations%20of%20polygenic%20disease%20traits:%20from%20theory%20to%20practice&author=W%20van%20Rheenen&author=WJ%20Peyrot&author=AJ%20Schork&author=SH%20Lee&author=NR%20Wray&volume=20&publication_year=2019&pages=567-581&pmid=31171865&doi=10.1038/s41576-019-0137-z&)

34. Torkamani A, Wineinger NE & Topol EJ The personal and clinical utility of polygenic risk scores. Nat. Rev. Genet 19, 581–590 (2018).  [DOI](https://doi.org/10.1038/s41576-018-0018-x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29789686/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Genet&title=The%20personal%20and%20clinical%20utility%20of%20polygenic%20risk%20scores&author=A%20Torkamani&author=NE%20Wineinger&author=EJ%20Topol&volume=19&publication_year=2018&pages=581-590&pmid=29789686&doi=10.1038/s41576-018-0018-x&)

35. Emdin CA, Khera AV & Kathiresan S Mendelian randomization. JAMA 318, 1925–1926 (2017).  [DOI](https://doi.org/10.1001/jama.2017.17219) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29164242/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA&title=Mendelian%20randomization&author=CA%20Emdin&author=AV%20Khera&author=S%20Kathiresan&volume=318&publication_year=2017&pages=1925-1926&pmid=29164242&doi=10.1001/jama.2017.17219&)

36. Garcia EP et al. Validation of OncoPanel: a targeted next-generation sequencing assay for the detection of somatic variants in cancer. Arch. Pathol. Lab. Med 141, 751–758 (2017).  [DOI](https://doi.org/10.5858/arpa.2016-0527-OA) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28557599/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arch.%20Pathol.%20Lab.%20Med&title=Validation%20of%20OncoPanel:%20a%20targeted%20next-generation%20sequencing%20assay%20for%20the%20detection%20of%20somatic%20variants%20in%20cancer&author=EP%20Garcia&volume=141&publication_year=2017&pages=751-758&pmid=28557599&doi=10.5858/arpa.2016-0527-OA&)

37. Gusev A, Groha S, Taraszka K, Semenov YR & Zaitlen N Constructing germline research cohorts from the discarded reads of clinical tumor sequences. Genome Med 13, 179 (2021).  [DOI](https://doi.org/10.1186/s13073-021-00999-4) | [PMC free article](/articles/PMC8576948/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34749793/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Med&title=Constructing%20germline%20research%20cohorts%20from%20the%20discarded%20reads%20of%20clinical%20tumor%20sequences&author=A%20Gusev&author=S%20Groha&author=K%20Taraszka&author=YR%20Semenov&author=N%20Zaitlen&volume=13&publication_year=2021&pages=179&pmid=34749793&doi=10.1186/s13073-021-00999-4&)

38. Liu J et al. An integrated TCGA Pan-Cancer clinical data resource to drive high-quality survival outcome analytics. Cell 173, 400–416.e11 (2018).  [DOI](https://doi.org/10.1016/j.cell.2018.02.052) | [PMC free article](/articles/PMC6066282/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29625055/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell&title=An%20integrated%20TCGA%20Pan-Cancer%20clinical%20data%20resource%20to%20drive%20high-quality%20survival%20outcome%20analytics&author=J%20Liu&volume=173&publication_year=2018&pages=400-416&pmid=29625055&doi=10.1016/j.cell.2018.02.052&)

39. Fast algorithms for conducting large-scale GWAS of age-at-onset traits using Cox mixed-effects models. Genetics 215, 1191 (2020).  [DOI](https://doi.org/10.1534/genetics.119.302940) | [PMC free article](/articles/PMC7198273/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32132097/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genetics&title=Fast%20algorithms%20for%20conducting%20large-scale%20GWAS%20of%20age-at-onset%20traits%20using%20Cox%20mixed-effects%20models&volume=215&publication_year=2020&pages=1191&pmid=32132097&doi=10.1534/genetics.119.302940&)

40. Wang G, Sarkar A, Carbonetto P & Stephens M A simple new approach to variable selection in regression, with application to genetic fine mapping. J. R. Stat. Soc. Series B Stat. Methodol 82, 1273–1300 (2020).  [DOI](https://doi.org/10.1111/rssb.12388) | [PMC free article](/articles/PMC10201948/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/37220626/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20R.%20Stat.%20Soc.%20Series%20B%20Stat.%20Methodol&title=A%20simple%20new%20approach%20to%20variable%20selection%20in%20regression,%20with%20application%20to%20genetic%20fine%20mapping&author=G%20Wang&author=A%20Sarkar&author=P%20Carbonetto&author=M%20Stephens&volume=82&publication_year=2020&pages=1273-1300&pmid=37220626&doi=10.1111/rssb.12388&)

41. Aalen OO & Johansen S An empirical transition matrix for non-homogeneous Markov chains based on censored observations. Scand. Stat. Theory Appl 5, 141–150 (1978).  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Scand.%20Stat.%20Theory%20Appl&title=An%20empirical%20transition%20matrix%20for%20non-homogeneous%20Markov%20chains%20based%20on%20censored%20observations&author=OO%20Aalen&author=S%20Johansen&volume=5&publication_year=1978&pages=141-150&)

42. Garrido-Martín D, Palumbo E, Guigó R & Breschi A ggsashimi: sashimi plot revised for browser- and annotation-independent splicing visualization. PLoS Comput. Biol 14, e1006360 (2018).  [DOI](https://doi.org/10.1371/journal.pcbi.1006360) | [PMC free article](/articles/PMC6114895/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30118475/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Comput.%20Biol&title=ggsashimi:%20sashimi%20plot%20revised%20for%20browser-%20and%20annotation-independent%20splicing%20visualization&author=D%20Garrido-Mart%C3%ADn&author=E%20Palumbo&author=R%20Guig%C3%B3&author=A%20Breschi&volume=14&publication_year=2018&pages=e1006360&pmid=30118475&doi=10.1371/journal.pcbi.1006360&)

43. Trapnell C et al. Differential gene and transcript expression analysis of RNA-seq experiments with TopHat and Cufflinks. Nat. Protoc 7, 562–578 (2012).  [DOI](https://doi.org/10.1038/nprot.2012.016) | [PMC free article](/articles/PMC3334321/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22383036/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Protoc&title=Differential%20gene%20and%20transcript%20expression%20analysis%20of%20RNA-seq%20experiments%20with%20TopHat%20and%20Cufflinks&author=C%20Trapnell&volume=7&publication_year=2012&pages=562-578&pmid=22383036&doi=10.1038/nprot.2012.016&)

44. Linsley PS, Speake C, Whalen E & Chaussabel D Copy number loss of the interferon gene cluster in melanomas is linked to reduced T cell infiltrate and poor patient prognosis. PLoS ONE 9, e109760 (2014).  [DOI](https://doi.org/10.1371/journal.pone.0109760) | [PMC free article](/articles/PMC4196925/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25314013/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&title=Copy%20number%20loss%20of%20the%20interferon%20gene%20cluster%20in%20melanomas%20is%20linked%20to%20reduced%20T%20cell%20infiltrate%20and%20poor%20patient%20prognosis&author=PS%20Linsley&author=C%20Speake&author=E%20Whalen&author=D%20Chaussabel&volume=9&publication_year=2014&pages=e109760&pmid=25314013&doi=10.1371/journal.pone.0109760&)

45. Hoadley KA et al. Multiplatform analysis of 12 cancer types reveals molecular classification within and across tissues of origin. Cell 158, 929–944 (2014).  [DOI](https://doi.org/10.1016/j.cell.2014.06.049) | [PMC free article](/articles/PMC4152462/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25109877/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell&title=Multiplatform%20analysis%20of%2012%20cancer%20types%20reveals%20molecular%20classification%20within%20and%20across%20tissues%20of%20origin&author=KA%20Hoadley&volume=158&publication_year=2014&pages=929-944&pmid=25109877&doi=10.1016/j.cell.2014.06.049&)

46. Collado-Torres L et al. Reproducible RNA-seq analysis using recount2. Nat. Biotechnol 35, 319–321 (2017).  [DOI](https://doi.org/10.1038/nbt.3838) | [PMC free article](/articles/PMC6742427/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28398307/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Biotechnol&title=Reproducible%20RNA-seq%20analysis%20using%20recount2&author=L%20Collado-Torres&volume=35&publication_year=2017&pages=319-321&pmid=28398307&doi=10.1038/nbt.3838&)

47. Orechia J et al. OncDRS: an integrative clinical and genomic data platform for enabling translational research and precision medicine. Appl. Transl. Genom 6, 18–25 (2015).  [DOI](https://doi.org/10.1016/j.atg.2015.08.005) | [PMC free article](/articles/PMC4803771/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27054074/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Appl.%20Transl.%20Genom&title=OncDRS:%20an%20integrative%20clinical%20and%20genomic%20data%20platform%20for%20enabling%20translational%20research%20and%20precision%20medicine&author=J%20Orechia&volume=6&publication_year=2015&pages=18-25&pmid=27054074&doi=10.1016/j.atg.2015.08.005&)

48. Nalichowski R, Keogh D, Chueh HC & Murphy SN Calculating the benefits of a Research Patient Data Repository. AMIA Annu. Symp. Proc 2006, 1044 (2006).  [PMC free article](/articles/PMC1839563/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17238663/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=AMIA%20Annu.%20Symp.%20Proc&title=Calculating%20the%20benefits%20of%20a%20Research%20Patient%20Data%20Repository&author=R%20Nalichowski&author=D%20Keogh&author=HC%20Chueh&author=SN%20Murphy&volume=2006&publication_year=2006&pages=1044&pmid=17238663&)

49. Davies RW, Flint J, Myers S & Mott R Rapid genotype imputation from sequence without reference panels. Nat. Genet 48, 965–969 (2016).  [DOI](https://doi.org/10.1038/ng.3594) | [PMC free article](/articles/PMC4966640/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27376236/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet&title=Rapid%20genotype%20imputation%20from%20sequence%20without%20reference%20panels&author=RW%20Davies&author=J%20Flint&author=S%20Myers&author=R%20Mott&volume=48&publication_year=2016&pages=965-969&pmid=27376236&doi=10.1038/ng.3594&)

50. Chen C-Y et al. Improved ancestry inference using weights from external reference panels. Bioinformatics 29, 1399–1406 (2013).  [DOI](https://doi.org/10.1093/bioinformatics/btt144) | [PMC free article](/articles/PMC3661048/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23539302/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=Improved%20ancestry%20inference%20using%20weights%20from%20external%20reference%20panels&author=C-Y%20Chen&volume=29&publication_year=2013&pages=1399-1406&pmid=23539302&doi=10.1093/bioinformatics/btt144&)

51. Anderson JR, Cain KC & Gelber RD Analysis of survival by tumor response and other comparisons of time-to-event by outcome variables. J. Clin. Oncol 26, 3913–3915 (2008).  [DOI](https://doi.org/10.1200/JCO.2008.16.1000) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18711176/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Oncol&title=Analysis%20of%20survival%20by%20tumor%20response%20and%20other%20comparisons%20of%20time-to-event%20by%20outcome%20variables&author=JR%20Anderson&author=KC%20Cain&author=RD%20Gelber&volume=26&publication_year=2008&pages=3913-3915&pmid=18711176&doi=10.1200/JCO.2008.16.1000&)

52. Austin PC & Fine JP Practical recommendations for reporting Fine-Gray model analyses for competing risk data. Stat. Med 36, 4391–4400 (2017).  [DOI](https://doi.org/10.1002/sim.7501) | [PMC free article](/articles/PMC5698744/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28913837/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Stat.%20Med&title=Practical%20recommendations%20for%20reporting%20Fine-Gray%20model%20analyses%20for%20competing%20risk%20data&author=PC%20Austin&author=JP%20Fine&volume=36&publication_year=2017&pages=4391-4400&pmid=28913837&doi=10.1002/sim.7501&)
