# Genomic variations associated with risk and protection against vincristine-induced peripheral neuropathy in pediatric cancer patients

## Metadata
**Authors:** Kheireddin Mufti, Miguel Cordova, Erika N Scott, Jessica N Trueman, Jessica M Lovnicki, Catrina M Loucks, Shahrad R Rassekh, Colin J D Ross, Bruce C Carleton, On behalf of the Canadian Pharmacogenomics Network for Drug Safety Consortium
**Journal:** NPJ Genomic Medicine
**Date:** 2024 Nov 5
**DOI:** [10.1038/s41525-024-00443-7](https://doi.org/10.1038/s41525-024-00443-7)
**PMID:** 39500896
**PMCID:** PMC11538333
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11538333/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC11538333/pdf/41525_2024_Article_443.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC11538333/pdf/41525_2024_Article_443.pdf)

## Abstract

Vincristine-induced peripheral neuropathy is a common and highly debilitating toxicity from vincristine treatment that affects quality of life and often requires dose reduction, potentially affecting survival. Although previous studies demonstrated genetic factors are associated with vincristine neuropathy risk, the clinical relevance of most identified variants is limited by small sample sizes and unclear clinical phenotypes. A genome-wide association study was conducted in 1100 cases and controls matched by vincristine dose and genetic ancestry, uncovering a statistically significant (p < 5.0 × 10−8) variant in MCM3AP gene that substantially increases the risk of neuropathy and 12 variants protective against neuropathy within/near SPDYA, METTL8, PDE4D, FBN2, ZFAND3, NFIB, PAPPA, LRRTM3, NRG3, VTI1A, ARHGAP5, and ACTN1. A follow-up pathway analysis reveals the involvement of four key pathways, including nerve structure and development, myelination, neuronal transmission, and cytoskeleton/microfibril function pathways. These findings present potential actionable genomic markers of vincristine neuropathy and offer opportunities for tailored interventions to improve vincristine safety in children with cancer. This study is registered with ClinicalTrials.gov under the title National Active Surveillance Network and Pharmacogenomics of Adverse Drug Reactions in Children (ID NCT00414115, registered on December 21, 2006).

Subject terms: Risk factors, Predictive markers, Prognostic markers, Genetic testing, Genetic markers

## Introduction

Vincristine is one of the most widely used chemotherapeutic agents for the treatment of pediatric cancers, including leukemias, lymphomas, neuroblastoma, rhabdomyosarcoma, Wilm’s tumor, and brain tumors^1–3^. Despite its high efficacy, up to 78% of patients receiving vincristine develop clinically significant vincristine-induced peripheral neuropathy^4–6^. The symptoms are characterized by progressive sensory, motor, and autonomic nerve dysfunctions (e.g., paresthesia, impaired motor skills, loss of deep tendon reflexes, and altered gait). Such symptoms are one of the main reasons for interruption or discontinuation of vincristine treatment, which can lead to an overall loss of therapeutic efficacy and an increased relapse risk, and consequently affects patients’ survival and quality of life during and after cancer therapy^7,8^.

The risk of developing vincristine-induced peripheral neuropathy is influenced by several factors, including patient-related and treatment-related factors. Age, sex, and ancestry represent the three patient-related factors that impact the risk of vincristine neuropathy, while treatment-related risk factors include vincristine cumulative dose and concomitant use of medications such as nifedipine, clarithromycin and anti-fungal azoles (e.g., fluconazole, ketoconazole, posaconazole, itraconazole, and voriconazole)^9–15^.

In addition, several genetic variations have been associated with the risk of vincristine-induced neuropathy. Since the first pharmacogenetic study of vincristine-induced neuropathy by Aplenc et al. demonstrating the role of genetic variants in *CYP3A4* and *CYP3A5* in 2003^16^, additional studies have implicated genetic variations in genes encoding components of vincristine pharmacokinetics: metabolism (e.g., *CYP3A4*, *CYP3A5*, and *DPYD*)^16–18^, distribution (e.g., *ABCC1* and *ABCC2*)^18,19^, and elimination (e.g., *ABCB1* and *ABCC2*)^19,20^; as well as vincristine pharmacodynamics: vincristine cellular targets (e.g., *CEP72*, *CAPG*, and *MAPT*)^20–22^ and neuropathy-related pathways (e.g., *SLC5A7* and *NKAIN3*)^18^.

Thus far, drug treatment of neuropathy results in modest improvements at best^15^. Nonetheless, emerging pharmacogenomic findings offer opportunities for further study of personalized or adaptive dosing strategies (e.g., treatment regimens with higher or lower dose intensity) and/or closer monitoring and proactive management (e.g., the addition of pre-emptive physiotherapy in patients at high-risk) to alleviate or prevent vincristine-induced peripheral neuropathy, which can adversely affect patients’ quality of life and longevity due to dose reduction or treatment omission despite the improved survival rate of cancers^7,8^. This study aims to investigate genomic risk factors of vincristine peripheral neuropathy in a large cohort of pediatric cancer patients using a genome-wide approach to identify pharmacogenomic markers implicated with neuropathy pathology and risk.

## Results

### Patient characteristics

The median age at diagnosis of the cases was 5.48 years (interquartile range [IQR]: 3.37–10.3), 50.5% (*n* = 278) of whom were males. The median age for the controls was 4.95 years (IQR: 2.80–9.35), and 51.8% (*n* = 285) were males. Cases and controls received a median cumulative dose of 48 mg/m^2^ (IQR: 27.0–65.0) and 45 mg/m^2^ (IQR: 18.0–64.5), respectively. Table [1](#Tab1) summarizes the clinical and demographic characteristics of the current cohort, and Supplementary Fig. [1](#MOESM1) (CONSORT diagram) provides a comprehensive overview of included patients.

### Table 1.

| Patient characteristics | Cases Neuropathy grade ≥ 2 (N = 550) | Controls Neuropathy grade 0 (N = 550) | Overall (N = 1100) | p-Value |
| --- | --- | --- | --- | --- |
| Sex |  |  |  |  |
| Female (%) | 272 (49.5%) | 265 (48.2%) | 537 (48.8%) | 0.46 |
| Male (%) | 278 (50.5%) | 285 (51.8%) | 563 (51.2%) |   |
| Age at diagnosis |  |  |  |  |
| Years, median (IQR) | 5.48 (3.37–10.3) | 4.95 (2.80–9.35) | 5.24 (3.06–9.92) | 0.003 |
| Vincristine total cumulative dose |  |  |  |  |
| mg/m2, median (IQR) | 48.0 (27.0–65.0) | 45.0 (18.0–64.5) | 47.0 (23.5–65.0) | 0.11 |
| Concomitant medications (%) |  |  |  |  |
| Antifungal azolesa | 68 (12.36%) | 67 (12.2%) | 135 (12.27%) | 1.0 |
| Clarithromycin | 11 (2.0%) | 42 (7.6%) | 53 (4.81%) | 0.0001 |
| Nifedipine | 50 (9.0%) | 35 (6.4%) | 85 (7.72%) | 0.11 |
| Phenytoin | 8 (1.5%) | 6 (1.1%) | 14 (1.3%) | 0.78 |
| PC1 |  |  |  |  |
| African from non-African, median (IQR) | 0.00938 (0.00826 –0.00966) | 0.00933 (0.00726–0.00967) | 0.00936 (0.00778–0.00966) | 0.32 |
| PC2 |  |  |  |  |
| East Asian from non-East Asian, median (IQR) | 0.0118 (0.0106–0.0121) | 0.0117 (0.00964–0.0121) | 0.0117 (0.0101–0.0121) | 0.10 |
| PC3 |  |  |  |  |
| Admixed American from non-admixed American, median (IQR) | 0.00483 (0.00407–0.00559) | 0.00483 (0.00409–0.00576) | 0.00483 (0.00408–0.00565) | 0.11 |
| PC4 |  |  |  |  |
| South Asian from non-South Asian, median (IQR) | −0.00459 (−0.00577 to −0.00240) | −0.00456 (−0.00591 to −0.00253) | −0.00458 (−0.00586 to −0.00248) | 0.61 |

Table 1 Caption: Clinical and demographic characteristics of the CPNDS pediatric patients

### Cohort exclusions

Exclusions were made based on clinical criteria to ensure high quality and homogeneity of the cohort. These include samples exhibiting discrepancies in sex assignment (*n* = 3) or displaying high genetic relatedness beyond first-degree relatives (*n* = 7) after medical chart confirmation of sex or repeat genotyping failed to resolve discrepancies. Individuals whose DNA samples were collected post-bone marrow transplant (*n* = 24) were excluded due to potential alterations in genetic background post-transplantation. Patients with mild neuropathy (CTCAE grade 1; *n* = 155) were excluded because these cases are mildly affected, do not require medical intervention, and may not progress to a more severe grade. All cases underwent expert review to ensure accurate classification according to CTCAE criteria, with those that were ambiguous excluded (*n* = 59).

Additional exclusions included non-pediatric patients (age at cancer diagnosis > 18 years old; *n* = 8), those lacking clinical charts with data on vincristine dosing or details about potential neuropathy (*n* = 19), patients who did not receive vincristine (*n* = 2), and duplicate entries (*n* = 8). Cases where neuropathy reaction was unclear if due to vincristine (*n* = 59), patients with pre-existing neurological conditions prior to vincristine administration (*n* = 27), and cases with unclear vincristine dosing/timing in relation to neuropathy onset (*n* = 23) were also excluded.

### Multivariable analysis and confounding variables

The multivariable analysis accounted for key confounding variables, specifically the genetic ancestry and vincristine cumulative dose. Of the other variables analyzed, age and concomitant clarithromycin use were statistically significantly different between the case and control groups. Concomitant clarithromycin was more frequently given in controls (7.6%) than in cases (2.0%), and the median age at diagnosis was slightly higher in cases (5.48 years, IQR: 3.37–10.3) than in controls (4.95 years, IQR: 2.80–9.35). Given clarithromycin’s effect as a potent CYP450 3A4 inhibitor, increasing vincristine plasma concentrations, the more frequent use in controls suggests it is unlikely to impact vincristine peripheral neuropathy risk. Supplementary Table [1](#MOESM1) details the timing and overlap of concomitant medications with neuropathy onset. The observed nominal age difference (6 months and 10 days) between cases and controls is also unlikely to be clinically significant in terms of substantially increasing vincristine neuropathy risk. The multivariate analysis, therefore, focused on clinically relevant confounding variables: vincristine exposure (cumulative dose) and genetic ancestry (top 4 principal components).

### Time to neuropathy

We investigated the time to first documentation of vincristine-induced peripheral neuropathy (CTCAE grade ≥ 2) in the 550 cases to understand the timing of vincristine neuropathy in relation to vincristine treatment. To calculate the treatment duration to the first neuropathy reaction, we used the date of the first dose of vincristine and the date neuropathy was first documented in the medical record. Three-hundred and thirty-one cases (60.18%) reported symptoms of peripheral neuropathy (CTCAE grade ≥ 2) within the first month of chemotherapy, while 525 (95.45%) had developed neuropathy symptoms by 14 months and 500 (100%) by 55 months (illustrated in Supplementary Fig. [2](#MOESM1)). Controls were followed up for an average duration of 12.5 years (standard deviation [SD] ± 5.93) confirming the absence of vincristine-induced neuropathy symptoms, having received on average 28 doses of vincristine (SD ± 17).

### Genome-wide analysis

To identify genetic risk loci associated with the risk of vincristine-induced neuropathy (CTCAE grade ≥ 2), we performed a genome-wide analysis of vincristine-induced peripheral neuropathy cases and controls matched by vincristine dose and genetic ancestry (top 4 PCs, allelic model). The GWAS identified 13 significant associations (*p* < 5 × 10^−^^8^), including variants within or near the genes *SPDYA*, *METLL8*, *PDE4D*, *FBN2*, *ZFAND3*, *NFIB*, *PAPPA*, *LRRTM3*, *VTI1A*, *NRG3*, *ARHGAP5*, *ACTN1*, and *MCM3AP* (Fig. [1](#Fig1)). Individual LocusZoom plots can be found in Supplementary Fig. [3](#MOESM1).

### Fig. 1. Manhattan plot of the vincristine-induced peripheral neuropathy GWAS summary statistics.

![Fig. 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4b69/11538333/b944e493c611/41525_2024_443_Fig1_HTML.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=11538333_41525_2024_443_Fig1_HTML.jpg)

Manhattan plot highlighting the GWAS-nominated loci associated with the risk of vincristine-induced neuropathy. Each point represents the log p-value at each genomic position from Chromosome 1 to Chromosome 22. The genome-wide significance threshold was set at 5 × 10−8 (black dashed line).

Of the 13 significant GWAS loci, one SNP (rs1815857) in *MCM3AP* increased the risk of vincristine-induced neuropathy by 6-fold (OR = 6.25, 95% CI: 2.91–12.11, *p* = 3.13 × 10^−8^) while 12 SNPs showed a protective effect against neuropathy risk (Table [2](#Tab2)). *FBN2* rs12656510 is found in LD with a missense variant rs32209 in the same gene (*R*^2^ = 0.72, *D’* = 0.85) with a combined annotation depletion dependent (CADD) score of 21.2 (top 1% of deleterious variants in the genome), and *MCM3AP* rs1815857 is in LD (*R*^2^ = 0.90, *D’* = 0.96) with an intronic variant rs17177067 within the same gene (CADD = 11.2; top 10% of pathogenic variants across the genome). In addition, 9 of the 13 identified genes are widely expressed across the nervous system, including 4 genes (*NFIB*, *ACTN1, ZFAND3*, and *MCM3AP*) that are very highly expressed in the tibial nerve (Supplementary Fig. [4](#MOESM1)).

### Table 2.

| Variant | Chr | Position (GRCh37/hg19) | Annotation | Nearest Gene(s) | EA | EAF cases | EAF controls | Global EAF (GnomAD v3.1.2) | Odds ratio (95% CI) | p-value |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| rs12474420 | 2 | 29068092 | intronic | SPDYA | T | 1.6% | 6.2% | 2.6% | 0.25 (0.14–0.43) | 4.40 × 10−8 |
| rs79802223 | 2 | 172161437 | intergenic | METTL8 | G | 0.9% | 5.0% | 2.5% | 0.17 (0.09–0.35) | 2.46 × 10−8 |
| rs12658429 | 5 | 58532218 | intronic | PDE4D | C | 1.5% | 6.1% | 3.0% | 0.24 (0.14–0.42) | 4.14 × 10−8 |
| rs12656510 | 5 | 127751056 | intronic | FBN2 | T | 3.8% | 9.7% | 6.7% | 0.36 (0.25–0.53) | 4.06 × 10−8 |
| rs200858088 | 6 | 37981515 | 5′UTR | ZFAND3 | C | 0.3% | 4.5% | 1.6% | 0.07 (0.02–0.21) | 2.53 × 10−10 |
| rs10961381 | 9 | 14117145 | intronic | NFIB | G | 0.2% | 3.6% | 1.1% | 0.07 (0.02–0.24) | 2.50 × 10−8 |
| rs12235805 | 9 | 118440926 | intergenic | PAPPA | A | 1.2% | 5.5% | 2.9% | 0.20 (0.11–0.30) | 3.04 × 10−8 |
| rs10997459 | 10 | 68730118 | intronic | LRRTM3 | G | 0.6% | 4.2% | 2.5% | 0.14 (0.06–0.32) | 4.84 × 10−8 |
| rs12253008 | 10 | 84153036 | intronic | NRG3 | T | 0.7% | 4.5% | 3.2% | 0.15 (0.07–0.33) | 3.19 × 10−8 |
| rs17129858 | 10 | 114313869 | intronic | VTI1A | T | 2.2% | 7.7% | 3.6% | 0.27 (0.17–0.43) | 4.10 × 10−9 |
| rs8006511 | 14 | 32449334 | intergenic | ARHGAP5 | G | 33.5% | 45.3% | 42.5% | 0.60 (0.51–0.72) | 2.90 × 10−8 |
| rs2268979 | 14 | 69415962 | intronic | ACTN1 | T | 8.3% | 16.2% | 10.4% | 0.46 (0.35–0.61) | 2.19 × 10−8 |
| rs1815857 | 21 | 47700847 | intronic | MCM3AP | T | 99.1% | 95.1% | 97.4% | 6.25 (2.91–12.11) | 3.13 × 10−8 |

Table 2 Caption: SNPs associated with vincristine-induced peripheral neuropathy

### Pathway analysis

The nominated cellular components include glutamatergic synapse (*p* = 0.0004), synapse (*p* = 0.001), and voltage-gated calcium channel complex (*p* = 0.02). Biological processes include principal sensory nucleus of trigeminal nerve development (*p* = 0.001), cell communication by electrical coupling (*p* = 0.02) and regulation of cellular component movement by cytoskeleton (*p* = 0.02), and molecular functions include protein kinase activator activity (*p* = 0.001), ion channel binding (*p* = 0.002) and structural constituent of post-synapse (*p* = 0.005) (Fig. [2a](#Fig2) and Supplementary Table [2](#MOESM1)). All *p*-values were Bonferroni corrected for multiple testing (number of tested pathways).

### Fig. 2. Vincristine neuropathy pathways associated with genes nominated by GWAS.

![Fig. 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4b69/11538333/5a38a018d667/41525_2024_443_Fig2_HTML.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=11538333_41525_2024_443_Fig2_HTML.jpg)

a Vincristine-induced peripheral neuropathy GWAS pathways nominated by gene-set enrichment analysis using WebGestalt (http://www.webgestalt.org/) in cellular components, biological processes, and molecular functions84. The bars represent the Bonferroni corrected p-values, and the red dashed line indicates a significance threshold of p = 0.05. b Key GWAS-significant loci across different pathophysiology, where a significant overlap is observed between the gene sets involved in different pathways, suggesting that an interaction between these pathways contributes to the overall risk of vincristine peripheral neuropathy.

## Discussion

In this largest GWAS of vincristine-induced peripheral neuropathy, we identified 13 significant vincristine peripheral neuropathy-associated loci, including 12 variants that strongly protect against peripheral neuropathy by 40–93% and one variant that increases the odds of developing peripheral neuropathy by 6-fold (Table [2](#Tab2)). The pathway analysis showed specificity for the involvement of four key pathways: nerve structure and development (genes involved: *ACTN1*, *LRRTM3*, *NRG3*, *VTI1A*, *NFIB*, *FBN2*, and *MCM3AP)*, nerve myelination (*MCM3AP* and *ZFAND3)*, neuronal transmission (*PDE4D*, *ACTN1*, *NRG3*, *VTI1A*, *SPDYA*, and *MCM3AP)*, and cytoskeleton/microfibril function (*ACTN1*, *NRG3*, *FBN2*, and *ARHGAP5*) pathways; all are required for the stability and normal function of the peripheral nervous system. Notably, the identified genetic variations within all the identified genes, except for *MCM3AP*, exhibit a protective effect against the development and progression of neuropathy, potentially modulating biological processes that contribute to the overall integrity of nerve structure and function.

The identified vincristine peripheral neuropathy genes involved in nerve structure pathway (*ACTN1*, *LRRTM3*, *NRG3*, *VTI1A*, *NFIB*, and *FBN2*) are known to play critical roles in the development and maintenance of neuronal networks. Specifically, *ACTN1*, *LRRTM3*, and *NRG3* are implicated in synapse formation and synaptic plasticity^23–28^, suggesting their involvement in regulating neuronal connectivity and communication. *VTI1A* is primarily associated with the trans-Golgi network in the neuron body and is known to play a key role in neuronal development^29,30^. NFIB is a transcription factor that regulates neurogenesis of the pontine neurons^31–33^, which manage sensory and pain signals^34^. Haploinsufficiency of *NFIB* in humans is found to result in neurodevelopmental deficits and neuronal malformations^35^. *FBN2* is a key component of the extracellular matrix of peripheral nerve^36^, and rare pathogenic missense variants in *FBN2* were found to cause early-onset peripheral neuropathy^37^. Variants in all these genes confer significant protection against neuropathy, reducing the risk by 54–93%.

In addition, genes identified in the nerve myelination pathway involve *ZFAND3* and *MCM3AP. MCM3AP* encodes for germinal center-associated nuclear protein (GANP), which is a major regulator of gene expression in developing motor neurons^38^. *MCM3AP* biallelic variants that affect protein function cause nerve demyelination and autosomal recessive forms of childhood-onset peripheral neuropathy (Charcot–Marie–Tooth disease; CMT)^39–41^. While *MCM3AP*-related neuropathy is primarily axonal and predominantly motor, sensory impairments such as altered pain or reduced sensory nerve action potentials are also present in some affected individuals^42^. *ZFAND3* plays a role in preventing CMT potentially through its role in facilitating the degradation and clearance of aberrant stress granules (SGs) that are responsible for increasing stress vulnerability of peripheral motor nerves^43,44^. Such aberrant SGs are implicated with CMT risk, and their disruption was found to alleviate motor weaknesses in CMT mice^45^. *ZFAND3* has also been identified through previous GWAS studies to be associated (OR = 1.51; CI 95%: 0.54–4.22) with an increased risk of Type 2 Diabetes (T2D) neuropathy^46,47^, and variants in *ZFAND3* T2D locus (rs58692659 and rs9470794) discovered in patients of East Asian ancestry are in high LD with the top SNP in the current GWAS (rs200858088; *R*^2^ = 0.952, *D’* = 0.975, and *R*^2^ = 0.985, *D’* = 1.0, respectively). The identified variant in *ZFAND3* (rs200858088) decreased vincristine neuropathy reaction risk by 93% (OR = 0.07; CI 95%: 0.02–0.21), while *MCM3AP* rs1815857 increased the risk of peripheral neuropathy (OR = 6.25; CI 95%: 2.91–12.11).

Vincristine peripheral neuropathy-related genes involved in the neuronal transmission pathway were *PDE4D*, *ACTN1*, *NRG3*, *LRRTM3*, and *VTI1A*, which are notably linked with various aspects of neuronal signaling and synaptic transmission. PDE4D is an enzyme that modulates cyclic adenosine monophosphate (cAMP) levels in neurons^48,49^, and it plays a role in intracellular cascades involved in post-synaptic signaling^50^. Inhibition of *PDE4* has shown promising results in reversing neuropathic pain in mice^51^ and is considered a therapeutic target for improving neurological symptoms of neurodegenerative disorders such as Parkinson’s disease and Huntington’s disease^52^. This can explain the neuroprotective effect of *PDE4D* rs12658429 against peripheral neuropathy risk in the current study. *ACTN1* regulates post-synaptic growth and neuronal plasticity through its interaction and activation of calcium/calmodulin-dependent protein kinase II (CaMKII)^26–28^. *NRG3* and *LRRTM3* promote excitatory synapse formation and synaptic plasticity of the peripheral nerves through their regulation of ErbB signaling and neurexin binding, respectively^23–25,53^. *NRG3* is a paralog of *NRG1*, which reduces pain hypersensitivity and inhibits neuropathic pain^54,55^, and ablation of *NRG3* results in reduced excitatory synapse numbers on GABAergic interneurons (involved in mediating pain), altered short-term plasticity and disinhibition of the hippocampal network^24^. *VTI1A* has been implicated in synaptic vesicle trafficking and neurotransmitter release, and its absence leads to impairments in neuronal development and communication^29,30,56^. The presence of protective variants in these genes may influence crucial processes such as synaptic vesicle trafficking, neurotransmitter release, and synaptic membrane dynamics, ultimately contributing to efficient neuronal transmission.

In addition to the previously discussed pathways, defects in cytoskeleton and microfibril architectures have also been implicated in peripheral neuropathies^57,58^. This study identified *ACTN1*, *NRG3*, *FBN2*, and *ARHGAP5* that are involved in maintaining cytoskeletal integrity and regulating cellular processes related to microfibril function. Such processes are essential for ensuring neuronal morphology stability and axonal transport^27^. *FBN2* encodes for microfibril^36^, while *ACTN1*, *NRG3*, and *ARHGAP5* are associated with actin cytoskeleton organization and dynamics^59–61^. *ACTN1* encodes for alpha-actinin and plays a critical role in axonal growth and guidance^62^. *ARHGAP5* is involved in the regulation of Rho GTPases, which are key regulators of cytoskeletal remodeling and cellular motility processes within the peripheral nervous system^61^. Variations within these genes have shown a 40–86% protection against peripheral neuropathy risk, which suggests their contribution to maintaining neuronal structural stability and cellular motility, thereby decreasing the sensitivity of the peripheral nervous system to vincristine-neurotoxicity. Of note, a significant overlap exists in the contributing genes within each pathophysiology (Fig. [2b](#Fig2)), suggesting a potential interplay between these pathways underlies protection against susceptibility of vincristine-induced peripheral neuropathy. Based on these results, we hypothesize that vincristine neuropathy risk is influenced by a complex interaction involving multiple neuropathy-related pathophysiology mechanisms (Fig. [3](#Fig3)).

### Fig. 3. Vincristine-induced peripheral neuropathy-related pathophysiology.

![Fig. 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4b69/11538333/546d1fe4d974/41525_2024_443_Fig3_HTML.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=11538333_41525_2024_443_Fig3_HTML.jpg)

Schematic overview of the key GWAS-significant loci across different vincristine peripheral neuropathy-related pathophysiology. a VTI1A regulates neuronal development, Golgi secretion, and extracellular transport of neurons85,86; NRG3 plays a role in intracellular signal transduction and nervous system development87; Actin provides structural support for neurons and regulates neuronal shape88. b MCM3AP is implicated in axonal and demyelinating CMT neuropathy39,40; ZFAND proteins regulate the clearance of aberrant stress granules (SGs) via the ubiquitin/proteasome system, preventing stress vulnerability in motoneurons and CMT caused by aberrant-SGs43–45,89; Neuregulin interacts with ErbB to promote myelination of peripheral axons and motor axon maturation90–94. c Actin filaments regulate axon growth, guidance, and active axonal transport88,95; ARHGAP5 mediates cytoskeleton changes and promotes actin polymerization through regulating Rho GTPases61; FBN2 provides mechanical and functional support to peripheral nerves and is implicated in early-onset neuropathies37,96–99. d VTI1A regulates the fusion of synaptic vesicles with the presynaptic membrane via interacting with t-SNAREs complex proteins56,85,100; Actinin regulates neurotransmitter release from presynaptic terminals and modulates synaptic plasticity via activating CaMKII in post-synapse neurons95,101–103; NRG3 promotes excitatory synapse formation and plasticity, and modulates synaptic transmission through activating ErbB424,25,87; LRRTM3 induces synapse development and differentiation through interacting with pre-synapse neurexins23,104–106. The figure was created with BioRender.com and Adobe Illustrator® Creative Cloud.

Several trials sought to determine whether medications such as amitriptyline, Org 2766, gabapentin, vitamin B6, and glutamine can be used to treat or prevent vincristine-induced peripheral neuropathy^63–71^. Out of these, only gabapentin has been found to improve neuropathy symptoms in pediatric patients^70,71^. However, these trials suffered from limitations such as a small sample, insufficient power, a high drop-out rate, and inconsistencies in primary outcomes, limiting the comparability of results. In addition, these trials were performed in a variety of treatment settings with various chemotherapy regimens, including different combinations of other neurotoxic medications (e.g., vinblastine, platinum derivatives, and taxanes), making the interpretation and extrapolation of the results a major challenge. As a result, despite the widespread interest and numerous clinical trials aiming to treat peripheral neuropathy, no clear standard has been determined or can be recommended at the current time.

Our findings present potential actionable genomic markers, encompassing variants in multiple genes associated with significant alterations in vincristine peripheral neuropathy risk, including *MCM3AP* (rs1815857) with a 6-fold increased risk, and protective variants such as *FBN2* (rs12656510), *ZFAND3* (rs200858088), *SPDYA* (rs12474420), *METLL8* (rs79802223), *PDE4D* (rs12658429), *NFIB* (rs10961381), *PAPPA* (rs12235805), *LRRTM3* (rs10997459), *VTI1A* (rs17129858), *NRG3* (rs12253008), *ARHGAP5* (rs8006511), and *ACTN1* (rs2268979). These genomic markers can be potentially leveraged to identify patients’ risk of developing peripheral neuropathy prior to treatment start and tailor therapeutic strategies (i.e., vincristine doses) accordingly. Specifically, patients carrying the high-risk variant in *MCM3AP* (rs1815857) may warrant closer monitoring and proactive management (i.e., the addition of pre-emptive physiotherapy) to mitigate the increased risk. Conversely, individuals with protective variants, such as those identified in *NRG3* (rs12253008) and *ACTN1* (rs2268979)—most overlapping genes in the pathophysiology of neuropathy, may benefit from treatment regimens with higher vincristine dose intensity.

Of note, while the current GWAS study identified 13 genetic variants strongly associated with vincristine-induced peripheral neuropathy risk (Table [2](#Tab2)), the analysis did not replicate previously identified variants, namely the promoter variation (rs924607) in the centrosomal Protein 72 (*CEP72*) gene, which was first identified by Diouf et al.^21^, where an association was observed in cases of neuropathy grade 2–4 during the continuation phase of chemotherapy treatment^21^. This significant association was evident only in patients after prolonged vincristine treatment (20–40 doses over 1–2 years) and was not replicated by the same research group (Diouf et al.^73^) and other studies when investigating neuropathy risk during the induction phase^72,73^. This suggests the effect of *CEP72* is particularly relevant to longer-term therapy. In the current study, the time-to-neuropathy analysis showed that 60.2% of vincristine-induced neuropathy cases reported symptoms early (i.e., within the first month of chemotherapy), and 90% developed neuropathy prior to the completion of seven months of cancer treatment. This difference in neuropathy onset can explain the non-significant association of *CEP72* rs924607 observed in our study and suggests potential etiological differences in neuropathy between induction and continuation phases of treatment. The lack of replication of previously identified genetic markers highlights the complexity of vincristine-induced neuropathy and underscores the importance of considering the phenotypic variability in genomic studies when moving forward with results validation.

To determine if our findings are relevant to the adult population, a dedicated study in adults is needed. The detection and diagnosis, as well as incidence and severity of vincristine-induced peripheral neuropathy, may differ between pediatric and adult populations due to differences in physiology, comorbidities, and the use of concomitant medications. The incidence of neuropathy in pediatric patients can be as high as 78%^5^, whereas in adults, it ranges from 30–50%^74,75^. Such difference in the incidence potentially suggests that different genetic and biological factors may influence neuropathy risk between children and adults. While predictive pharmacogenetic biomarkers remain constant across the lifespan, their expression and impact can differ significantly between children and adults. In addition, the dynamic developmental changes such as drug-metabolizing enzymes and organ function over time necessitate dedicated research to optimize drug therapy in children and adult populations. Unlike adults, children typically have fewer comorbidities and take fewer concomitant medications, which can mask genetic predictors of adverse drug reactions. This highlights the necessity for independent pharmacogenomic studies in children and adults to identify genetic predictors of vincristine-induced neuropathy.

While our findings provide valuable insights into the genomic associations with vincristine-induced peripheral neuropathy risk, the identified genetic variants in this study should be validated in independent cohorts of children to establish the generalizability and robustness of the findings. Given the rarity of pediatric cancer, securing a sufficiently large independent cohort for replication is challenging. However, validation of the current findings is a critical next step in this work and is now being planned. Functional studies using cellular (e.g., neurons derived from human induced pluripotent stem cells) or animal (e.g., mice, zebrafish, and *Caenorhabditis elegans*) models are likely to be useful for better understanding the mechanism by which the identified variants are involved in the pathophysiology of vincristine neuropathy. Validation of these findings will be a critical step forward in our efforts to mitigate the burden incurred by this severe reaction on children battling cancer, their families, and caregivers.

## Methods

### Patient recruitment and characterization

A total of 2043 patients who received vincristine chemotherapy were consecutively recruited and enrolled between 2005 and 2018 from 10 pediatric health centers within the Canadian Pharmacogenomics Network for Drug Safety (CPNDS) in Canada. This included sites in Vancouver (BC Children’s Hospital), Montreal (CHU Sainte-Justine Hospital), Toronto (The Hospital for Sick Children), Winnipeg (Children’s Hospital at the Health Sciences Center), London (Children’s Hospital at London Health Sciences Center), Ottawa (Children’s Hospital of Eastern Ontario), Calgary (Alberta Children’s Hospital), Halifax (IWK Children’s Hospital), Edmonton (Stollery Children’s Hospital), and Saskatoon (Jim Pattison Children’s Hospital). Written informed consent and/or assent were obtained from all patients or their parents/legal guardians/substitute decision-makers before enrolling in the study. The study was conducted in accordance with the Declaration of Helsinki and received ethical approval from the University of British Columbia Children’s and Women’s Research Ethics Board (reference number: H04-70358). The trial is publicly accessible with ClinicalTrials.gov under the registry name “National Active Surveillance Network and Pharmacogenomics of Adverse Drug Reactions in Children” and registration number [NCT00414115](https://clinicaltrials.gov/ct2/show/NCT00414115), registered on December 21, 2006.

Relevant clinical and demographic data were collected from medical charts and electronic medical records and were characterized for the entire cohort. These included data on patient age at cancer diagnosis, sex, cancer diagnosis, tumor risk and relapse, start/end date of vincristine treatment, chemotherapy protocol, date of first notation of neuropathy, vincristine total cumulative dose received and cumulative dose to the time of toxicity in cases (mg/m^2^), and concomitant medications used that potentially impact vincristine metabolism and elimination (i.e., nifedipine, clarithromycin, and phenytoin). We also recorded concomitant treatment with anti-fungal azoles (i.e., fluconazole, itraconazole, ketoconazole, posaconazole, and voriconazole) which are strong CYP3A4 inhibitors and increase the risk of neuropathy when administered concomitantly with vincristine at its therapeutic doses^76^.

Vincristine-induced peripheral neuropathy was assessed and graded according to the Common Terminology Criteria of Adverse Events (CTCAE) version 5.0. Vincristine neuropathy reactions were defined by neurological symptoms clinically characterized by numbness and paresthesia, loss of sensation/tendon reflexes, alterations of fine gait and motor skills, or uncontrolled pain and paralysis that may lead to death. Cases of vincristine-neuropathy (CTCAE grade ≥ 2, *n* = 669) and controls (*n* = 683) were matched by vincristine cumulative dose and genetic ancestry (top 4 principal components [PCs]) using propensity score matching (R package [MatchIt], data included: case/control status, treatment duration, vincristine dose, PC1–PC4) to reduce case-control selection bias^77^, resulting in 550 cases matched with 550 controls. The CONSORT diagram (Supplementary Fig. [1](#MOESM1)) provides detailed descriptions of all patients and exclusions.

### Genotyping and quality control

DNA extraction and genotyping were performed on the collected biological samples (blood or saliva). Genotype data was obtained through the Illumina Infinium^®^ (Illumina, San Diego, CA, USA) Global Screening Array (GSA v2.0) with a custom multi-disease add-on panel. Using the 1000 Genomes Phase 3 samples as a reference, genotype phasing and imputation were conducted with SHAPEIT2 (v2.0)^78^ and IMPUTE2 (v2.3.2)^79^ to include variants that were not directly genotyped, and imputed single nucleotide polymorphisms (SNPs) with missingness > 0.05, call threshold < 0.9, and info score < 0.5 were removed. A total of 488,268 directly genotyped SNPs remained after the quality control (QC) steps. 10,239,025 variants were analyzed following imputation, and 6,476,118 SNPs remained available for genomic analysis after QC filtering.

PLINK v1.9 was used for quality control filtering and variant calling through BC Platforms^80^. Initial quality control was completed through Genome Studio to remove failed assays identified by visual inspection of raw genotype data. At the sample level, individuals were screened for call rate (>95%) and genetic sex using PLINK v1.9 in BC Platforms. Samples with sex discrepancies that were not resolved by re-checking the patient chart, re-genotyping or re-collection were excluded. Identity-by-descent (IBD) check was performed using PLINK v1.9 GWAS pipeline to identify relatedness between the samples. Highly genetically related samples (closer than cousins; PI cut-off ≥ 0.125) were checked for genetic quality, and the samples with the lowest call rate were removed. At the SNP level, further quality control was performed to filter SNPs based on variant missingness (<0.05), deviation from Hardy-Weinberg Equilibrium (HWE; Fisher’s exact test *p* < 1 × 10^−6^), and minor allele frequency (MAF) < 0.01 using PLINK (v1.9). Genetic ancestry was calculated using principal component analysis (PCA) of patient genotypes via the EIGENSTRAT method, in conjunction with genome-wide genotyping data from the 1000 Genomes Phase 3 reference population, after Genotype Harmonizer strand alignment^81^. PCA was conducted and visualized using RStudio and related packages (i.e., AssocTests, ggplot2, and plyr), and the top 4 principal components (Supplementary Fig. [5](#MOESM1)) were accounted for in the genome-wide analysis to control for any potential population stratification.

### Statistical analyses

Analyses of the clinical and demographic characteristics were performed using R and related packages (i.e., plyr, reshape2, dplyr, broom, and finalfit) with the Wilcoxon–Mann–Whitney test for testing continuous variables and Fisher exact test for categorical variables. Table [1](#Tab1) summarizes and reviews all the clinical and demographic characteristics analyzed in the current cohort (additional characteristics related to tumor types are listed in Supplementary Table [3](#MOESM1)). We performed a matched case-control genome-wide analysis using an allelic genetic model (PLINK v1.9 in BC Platforms). The genome-wide significance level was established at *p* = 5 × 10^−8^ for a priori power calculations (power calculations are presented in Supplementary Fig. [6](#MOESM1)). We sought to identify potentially causal/driver variants by testing linkage disequilibrium (LD) between the top GWAS hits and previously identified SNPs within the 1000 Genomes reference cohort (considering all cohorts together [i.e., African, American, European, East Asian, and South Asian] and non-Finnish European alone) embedded in LDLink ([https://ldlink.nci.nih.gov/](https://ldlink.nci.nih.gov/))^82^. Additionally, we used the Genotype-Tissue Expression database (GTEx consortium v8.0; embedded in FUMA: [https://fuma.ctglab.nl/](https://fuma.ctglab.nl/)) to test the effects of the identified variants on gene expression^83^. To examine whether specific pathways are involved according to the GWAS results, pathway enrichment analysis (gene-set enrichment) was performed using the GWAS-nominated genes for cellular components, biological processes, and molecular functions through the publicly available online tool WebGestalt ([http://www.webgestalt.org/](http://www.webgestalt.org/))^84^. An enrichment score was computed to indicate the extent to which the genes identified through GWAS are overexpressed in various functional pathways. Statistical significance was determined using a permutation test (Fisher’s exact test), and adjustment for multiple testing (number of pathways examined) was applied according to Bonferroni correction.

## Supplementary information

## Acknowledgements

We extend our heartfelt appreciation and thankfulness to all the patients and their families who have generously supported the work. This study is part of the “Genomic and Outcomes Databank for Pharmacogenomics and Implementation Studies (Go-PGx)” and is supported by Genome Canada, Genome British Columbia (funding #272PGX), and the Canadian Institutes of Health Research (funding #GP1-155872) through the Large Scale Applied Research Project Competition. Additional support has been provided by British Columbia’s Provincial Health Services Authority (PHSA), BC Children’s Hospital Foundation, and Health Canada. K.M. is supported by a 4-Year Doctoral Fellowship from the University of British Columbia (UBC). E.N.S. is currently supported by an International Association for the Study of Pain John J. Bonica Trainee Fellowship and was supported by a 4-Year Doctoral Fellowship from UBC. C.M.L. was supported through a BC Children’s Hospital Research Institute (BCCHR) Bertram Hoffmeister Postdoctoral Fellowship, CIHR Drug Safety and Effectiveness Cross-Disciplinary Training (DSECT) Awards, and a CIHR Fellowship (FRN 164641). C.M.L. is currently supported by a BCCHR Investigator Establishment Award. S.R.R. has received funding from a law firm for expert testimony on a patent case involving sodium thiosulfate and has served on a science advisory board for Jazz Pharmaceutics on an unrelated product (Vyxeos). C.J.D.R. was supported by a Michael Smith Foundation for Health Research Scholar Award. B.C.C. receives financial support for his pharmacogenomics from the following government sources: Canadian Institutes of Health Research, Genome Canada, Genome British Columbia, Health Canada, the BC Children’s Hospital Foundation, the BC Provincial Health Services Authority and the US Centers for Disease Control and Prevention. B.C.C. is a past consultant to the United Health Group and a current consultant to Dynacare Next Specialized Diagnostics and Neopharm Labs in the area of pharmacogenomics and pharmacogenetic testing.

## Author contributions

K.M., C.M.L., S.R.R., C.J.D.R., and B.C.C. designed and conceptualized the study. J.N.T. and J.M.L. contributed to the acquisition and preparation of study data. K.M. and J.M.L. performed quality control. KM performed case-control matching, analysis of clinical data, and genome-wide association study, M.C. and E.N.S provided help with these analyses. K.M. performed time-to-neuropathy and pathway analyses. K.M. generated figures and tables, and wrote the initial draft of the paper with primary review by C.J.D.R. and B.C.C. All members of the CPNDS consortium (listed below) critically reviewed and edited the paper.

## Data availability

Patient-level data underlying the study cannot be made publicly available due to restrictions mandated by the Research Ethics Board at the University of British Columbia, where the data reside. Datasets analyzed during the current study can be shared with co-investigators on request. If co-investigators are outside Canada, data must reside within the country for access. These stipulations are per the ethical approval of our study. The data were collected based on written informed consent, and sharing of patient-specific data, even when de-identified, without further written informed consent from the participants is not permitted. Interested researchers may request access to these data via direct contact with the corresponding authors.

## Code availability

No custom codes were specifically developed for the data analysis in this study; the analysis relied on established software tools and packages as described above.

## Competing interests

The authors declare no competing interests.

## Footnotes

## Contributor Information

Colin J. D. Ross, Email: colin.ross@ubc.ca

Bruce C. Carleton, Email: bcarleton@popi.ubc.ca

On behalf of the Canadian Pharmacogenomics Network for Drug Safety Consortium:

[Gabriella S. S. Groeneweg](https://pubmed.ncbi.nlm.nih.gov/?term=%22Groeneweg%20GSS%22%5BAuthor%5D), [Michelle Higginson](https://pubmed.ncbi.nlm.nih.gov/?term=%22Higginson%20M%22%5BAuthor%5D), [Wan-Chun Chang](https://pubmed.ncbi.nlm.nih.gov/?term=%22Chang%20WC%22%5BAuthor%5D), [Kathy Li](https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20K%22%5BAuthor%5D), [Fudan Miao](https://pubmed.ncbi.nlm.nih.gov/?term=%22Miao%20F%22%5BAuthor%5D), [Derek Yau](https://pubmed.ncbi.nlm.nih.gov/?term=%22Yau%20D%22%5BAuthor%5D), [Lucie Pecheux](https://pubmed.ncbi.nlm.nih.gov/?term=%22Pecheux%20L%22%5BAuthor%5D), [Bina Gyawali](https://pubmed.ncbi.nlm.nih.gov/?term=%22Gyawali%20B%22%5BAuthor%5D), [Amanda Perreault](https://pubmed.ncbi.nlm.nih.gov/?term=%22Perreault%20A%22%5BAuthor%5D), [Fatema Abbasi](https://pubmed.ncbi.nlm.nih.gov/?term=%22Abbasi%20F%22%5BAuthor%5D), [Gregory Guilcher](https://pubmed.ncbi.nlm.nih.gov/?term=%22Guilcher%20G%22%5BAuthor%5D), [Gesche Riabowol](https://pubmed.ncbi.nlm.nih.gov/?term=%22Riabowol%20G%22%5BAuthor%5D), [Geert ’t Jong](https://pubmed.ncbi.nlm.nih.gov/?term=%22Jong%20G%E2%80%99%22%5BAuthor%5D), [Michelle Staub](https://pubmed.ncbi.nlm.nih.gov/?term=%22Staub%20M%22%5BAuthor%5D), [Geoff Cuvelier](https://pubmed.ncbi.nlm.nih.gov/?term=%22Cuvelier%20G%22%5BAuthor%5D), [Kathleen Felton](https://pubmed.ncbi.nlm.nih.gov/?term=%22Felton%20K%22%5BAuthor%5D), [Sara Khalaj](https://pubmed.ncbi.nlm.nih.gov/?term=%22Khalaj%20S%22%5BAuthor%5D), [Michael Rieder](https://pubmed.ncbi.nlm.nih.gov/?term=%22Rieder%20M%22%5BAuthor%5D), [Awatif Abuzgaia](https://pubmed.ncbi.nlm.nih.gov/?term=%22Abuzgaia%20A%22%5BAuthor%5D), [Tamorah Lewis](https://pubmed.ncbi.nlm.nih.gov/?term=%22Lewis%20T%22%5BAuthor%5D), [Himal Ghimire](https://pubmed.ncbi.nlm.nih.gov/?term=%22Ghimire%20H%22%5BAuthor%5D), [Paul Nathan](https://pubmed.ncbi.nlm.nih.gov/?term=%22Nathan%20P%22%5BAuthor%5D), [Donna Johnston](https://pubmed.ncbi.nlm.nih.gov/?term=%22Johnston%20D%22%5BAuthor%5D), [Mounira Ibrahim](https://pubmed.ncbi.nlm.nih.gov/?term=%22Ibrahim%20M%22%5BAuthor%5D), [Jean‐François Bussières](https://pubmed.ncbi.nlm.nih.gov/?term=%22Bussi%C3%A8res%20JF%22%5BAuthor%5D), [Thaïna-Rafi Jean-Baptiste](https://pubmed.ncbi.nlm.nih.gov/?term=%22Jean-Baptiste%20TR%22%5BAuthor%5D), [Denis Lebel](https://pubmed.ncbi.nlm.nih.gov/?term=%22Lebel%20D%22%5BAuthor%5D), [Maja Krajinovic](https://pubmed.ncbi.nlm.nih.gov/?term=%22Krajinovic%20M%22%5BAuthor%5D), [Thai Hoa Tran](https://pubmed.ncbi.nlm.nih.gov/?term=%22Tran%20TH%22%5BAuthor%5D), [Kerry Goralski](https://pubmed.ncbi.nlm.nih.gov/?term=%22Goralski%20K%22%5BAuthor%5D), [Zara Forbrigger](https://pubmed.ncbi.nlm.nih.gov/?term=%22Forbrigger%20Z%22%5BAuthor%5D), and [Ketan Kulkarni](https://pubmed.ncbi.nlm.nih.gov/?term=%22Kulkarni%20K%22%5BAuthor%5D)

## Supplementary information

The online version contains supplementary material available at 10.1038/s41525-024-00443-7.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

Patient-level data underlying the study cannot be made publicly available due to restrictions mandated by the Research Ethics Board at the University of British Columbia, where the data reside. Datasets analyzed during the current study can be shared with co-investigators on request. If co-investigators are outside Canada, data must reside within the country for access. These stipulations are per the ethical approval of our study. The data were collected based on written informed consent, and sharing of patient-specific data, even when de-identified, without further written informed consent from the participants is not permitted. Interested researchers may request access to these data via direct contact with the corresponding authors.

No custom codes were specifically developed for the data analysis in this study; the analysis relied on established software tools and packages as described above.

### Supplementary Materials

### Data Availability Statement

Patient-level data underlying the study cannot be made publicly available due to restrictions mandated by the Research Ethics Board at the University of British Columbia, where the data reside. Datasets analyzed during the current study can be shared with co-investigators on request. If co-investigators are outside Canada, data must reside within the country for access. These stipulations are per the ethical approval of our study. The data were collected based on written informed consent, and sharing of patient-specific data, even when de-identified, without further written informed consent from the participants is not permitted. Interested researchers may request access to these data via direct contact with the corresponding authors.

No custom codes were specifically developed for the data analysis in this study; the analysis relied on established software tools and packages as described above.

## References

1. Pui, C.-H. & Evans, W. E. Treatment of acute lymphoblastic leukemia. N. Engl. J. Med.354, 166–178 (2006).  [DOI](https://doi.org/10.1056/NEJMra052603) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16407512/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Pui,%20C.-H.%20&%20Evans,%20W.%20E.%20Treatment%20of%20acute%20lymphoblastic%20leukemia.%20N.%20Engl.%20J.%20Med.354,%20166%E2%80%93178%20(2006).)

2. Said, R. & Tsimberidou, A. M. Pharmacokinetic evaluation of vincristine for the treatment of lymphoid malignancies. Expert Opin. Drug Metab. Toxicol.10, 483–494 (2014).  [DOI](https://doi.org/10.1517/17425255.2014.885016) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24512004/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Said,%20R.%20&%20Tsimberidou,%20A.%20M.%20Pharmacokinetic%20evaluation%20of%20vincristine%20for%20the%20treatment%20of%20lymphoid%20malignancies.%20Expert%20Opin.%20Drug%20Metab.%20Toxicol.10,%20483%E2%80%93494%20(2014).)

3. Pizzo, P.A. & Poplack, D.G. Principles and Practice of Pediatric Oncology. (Lippincott Williams & Wilkins, 2015).

4. Gidding, C., Kellie, S., Kamps, W. & De Graaf, S. Vincristine revised. (1999).  [DOI](https://doi.org/10.1016/s1040-8428(98)00023-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10226730/)

5. Lavoie Smith, E. M. et al. Patterns and severity of vincristine‐induced peripheral neuropathy in children with acute lymphoblastic leukemia. J. Peripheral Nerv. Syst.20, 37–46 (2015).  [DOI](https://doi.org/10.1111/jns.12114) | [PMC free article](/articles/PMC4610712/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25977177/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Lavoie%20Smith,%20E.%20M.%20et%20al.%20Patterns%20and%20severity%20of%20vincristine%E2%80%90induced%20peripheral%20neuropathy%20in%20children%20with%20acute%20lymphoblastic%20leukemia.%20J.%20Peripheral%20Nerv.%20Syst.20,%2037%E2%80%9346%20(2015).)

6. Nama, N. et al. Vincristine-induced peripheral neurotoxicity: a prospective cohort. Pediatr. Hematol. Oncol.37, 15–28 (2020).  [DOI](https://doi.org/10.1080/08880018.2019.1677832) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31682156/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Nama,%20N.%20et%20al.%20Vincristine-induced%20peripheral%20neurotoxicity:%20a%20prospective%20cohort.%20Pediatr.%20Hematol.%20Oncol.37,%2015%E2%80%9328%20(2020).)

7. Smith, E. M. L. et al. Characteristics and patterns of pediatric chemotherapy-induced peripheral neuropathy: a systematic review. Cancer Treat. Res. Commun.28, 100420 (2021).  [DOI](https://doi.org/10.1016/j.ctarc.2021.100420) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34225104/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Smith,%20E.%20M.%20L.%20et%20al.%20Characteristics%20and%20patterns%20of%20pediatric%20chemotherapy-induced%20peripheral%20neuropathy:%20a%20systematic%20review.%20Cancer%20Treat.%20Res.%20Commun.28,%20100420%20(2021).)

8. van de Velde, M. E. et al. The association between vincristine‐induced peripheral neuropathy and health‐related quality of life in children with cancer. Cancer Med.10, 8172–8181 (2021).  [DOI](https://doi.org/10.1002/cam4.4289) | [PMC free article](/articles/PMC8607258/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34725942/) | [Google Scholar](https://scholar.google.com/scholar_lookup?van%20de%20Velde,%20M.%20E.%20et%20al.%20The%20association%20between%20vincristine%E2%80%90induced%20peripheral%20neuropathy%20and%20health%E2%80%90related%20quality%20of%20life%20in%20children%20with%20cancer.%20Cancer%20Med.10,%208172%E2%80%938181%20(2021).)

9. Jeng, M. R. & Feusner, J. Itraconazole-enhanced vincristine neurotoxicity in a child with acute lymphoblastic leukemia. Pediatr. Hematol. Oncol.18, 137–142 (2001).  [DOI](https://doi.org/10.1080/088800101300002973) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11255732/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Jeng,%20M.%20R.%20&%20Feusner,%20J.%20Itraconazole-enhanced%20vincristine%20neurotoxicity%20in%20a%20child%20with%20acute%20lymphoblastic%20leukemia.%20Pediatr.%20Hematol.%20Oncol.18,%20137%E2%80%93142%20(2001).)

10. van Schie, R. M., Brüggemann, R. J., Hoogerbrugge, P. M. & Te Loo, D. Effect of azole antifungal therapy on vincristine toxicity in childhood acute lymphoblastic leukaemia. J. Antimicrob. Chemother.66, 1853–1856 (2011).  [DOI](https://doi.org/10.1093/jac/dkr223) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21652620/) | [Google Scholar](https://scholar.google.com/scholar_lookup?van%20Schie,%20R.%20M.,%20Br%C3%BCggemann,%20R.%20J.,%20Hoogerbrugge,%20P.%20M.%20&%20Te%20Loo,%20D.%20Effect%20of%20azole%20antifungal%20therapy%20on%20vincristine%20toxicity%20in%20childhood%20acute%20lymphoblastic%20leukaemia.%20J.%20Antimicrob.%20Chemother.66,%201853%E2%80%931856%20(2011).)

11. Thackray, J., Spatz, K. & Steinherz, P. G. Vincristine toxicity with co-administration of fluconazole: long-term concerns. Pediatric Blood Cancer64, 101002 (2017).  [DOI](https://doi.org/10.1002/pbc.26652) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28544423/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Thackray,%20J.,%20Spatz,%20K.%20&%20Steinherz,%20P.%20G.%20Vincristine%20toxicity%20with%20co-administration%20of%20fluconazole:%20long-term%20concerns.%20Pediatric%20Blood%20Cancer64,%20101002%20(2017).)

12. Lin, M. J.-Y., Paul, M. R. & Kuo, D. J. Severe neuropathic pain with concomitant administration of vincristine and posaconazole. J. Pediatr. Pharmacol. Therap.23, 417–420 (2018).  [DOI](https://doi.org/10.5863/1551-6776-23.5.417) | [PMC free article](/articles/PMC6213623/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30429697/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Lin,%20M.%20J.-Y.,%20Paul,%20M.%20R.%20&%20Kuo,%20D.%20J.%20Severe%20neuropathic%20pain%20with%20concomitant%20administration%20of%20vincristine%20and%20posaconazole.%20J.%20Pediatr.%20Pharmacol.%20Therap.23,%20417%E2%80%93420%20(2018).)

13. Tay, N. et al. Chemotherapy-induced peripheral neuropathy in children and adolescent cancer patients. Front. Mol. Biosci.9, 1015746 (2022).  [DOI](https://doi.org/10.3389/fmolb.2022.1015746) | [PMC free article](/articles/PMC9614173/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36310587/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Tay,%20N.%20et%20al.%20Chemotherapy-induced%20peripheral%20neuropathy%20in%20children%20and%20adolescent%20cancer%20patients.%20Front.%20Mol.%20Biosci.9,%201015746%20(2022).)

14. Sabus, A. et al. Incidence and severity of neuropathy with concurrent use of voriconazole and vincristine in pediatric patients with cancer. J. Hematol. Oncol. Pharmacy13, 77 (2023).  [Google Scholar](https://scholar.google.com/scholar_lookup?Sabus,%20A.%20et%20al.%20Incidence%20and%20severity%20of%20neuropathy%20with%20concurrent%20use%20of%20voriconazole%20and%20vincristine%20in%20pediatric%20patients%20with%20cancer.%20J.%20Hematol.%20Oncol.%20Pharmacy13,%2077%20(2023).)

15. Triarico, S. et al. Vincristine-induced peripheral neuropathy (VIPN) in pediatric tumors: mechanisms, risk factors, strategies of prevention and treatment. Int. J. Mol. Sci.22, 4112 (2021).  [DOI](https://doi.org/10.3390/ijms22084112) | [PMC free article](/articles/PMC8073828/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33923421/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Triarico,%20S.%20et%20al.%20Vincristine-induced%20peripheral%20neuropathy%20(VIPN)%20in%20pediatric%20tumors:%20mechanisms,%20risk%20factors,%20strategies%20of%20prevention%20and%20treatment.%20Int.%20J.%20Mol.%20Sci.22,%204112%20(2021).)

16. Aplenc, R. et al. CYP3A genotypes and treatment response in paediatric acute lymphoblastic leukaemia. Br. J. Haematol.122, 240–244 (2003).  [DOI](https://doi.org/10.1046/j.1365-2141.2003.04430.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12846892/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Aplenc,%20R.%20et%20al.%20CYP3A%20genotypes%20and%20treatment%20response%20in%20paediatric%20acute%20lymphoblastic%20leukaemia.%20Br.%20J.%20Haematol.122,%20240%E2%80%93244%20(2003).)

17. Kishi, S. et al. Ancestry and pharmacogenetics of antileukemic drug toxicity. Blood109, 4151–4157 (2007).  [DOI](https://doi.org/10.1182/blood-2006-10-054528) | [PMC free article](/articles/PMC1885506/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17264302/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Kishi,%20S.%20et%20al.%20Ancestry%20and%20pharmacogenetics%20of%20antileukemic%20drug%20toxicity.%20Blood109,%204151%E2%80%934157%20(2007).)

18. Wright, G. E. et al. Pharmacogenomics of vincristine‐induced peripheral neuropathy implicates pharmacokinetic and inherited neuropathy genes. Clin. Pharmacol. Therap.105, 402–410 (2019).  [DOI](https://doi.org/10.1002/cpt.1179) | [PMC free article](/articles/PMC6519044/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29999516/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Wright,%20G.%20E.%20et%20al.%20Pharmacogenomics%20of%20vincristine%E2%80%90induced%20peripheral%20neuropathy%20implicates%20pharmacokinetic%20and%20inherited%20neuropathy%20genes.%20Clin.%20Pharmacol.%20Therap.105,%20402%E2%80%93410%20(2019).)

19. Lopez-Lopez, E. et al. Vincristine pharmacokinetics pathway and neurotoxicity during early phases of treatment in pediatric acute lymphoblastic leukemia. Pharmacogenomics17, 731–741 (2016).  [DOI](https://doi.org/10.2217/pgs-2016-0001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27180762/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Lopez-Lopez,%20E.%20et%20al.%20Vincristine%20pharmacokinetics%20pathway%20and%20neurotoxicity%20during%20early%20phases%20of%20treatment%20in%20pediatric%20acute%20lymphoblastic%20leukemia.%20Pharmacogenomics17,%20731%E2%80%93741%20(2016).)

20. Ceppi, F. et al. Polymorphisms of the vincristine pathway and response to treatment in children with childhood acute lymphoblastic leukemia. Pharmacogenomics15, 1105–1116 (2014).  [DOI](https://doi.org/10.2217/pgs.14.68) | [PMC free article](/articles/PMC4443746/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25084203/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Ceppi,%20F.%20et%20al.%20Polymorphisms%20of%20the%20vincristine%20pathway%20and%20response%20to%20treatment%20in%20children%20with%20childhood%20acute%20lymphoblastic%20leukemia.%20Pharmacogenomics15,%201105%E2%80%931116%20(2014).)

21. Diouf, B. et al. Association of an inherited genetic variant with vincristine-related peripheral neuropathy in children with acute lymphoblastic leukemia. J. Am. Med. Assoc.313, 815–823 (2015).  [DOI](https://doi.org/10.1001/jama.2015.0894) | [PMC free article](/articles/PMC4377066/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25710658/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Diouf,%20B.%20et%20al.%20Association%20of%20an%20inherited%20genetic%20variant%20with%20vincristine-related%20peripheral%20neuropathy%20in%20children%20with%20acute%20lymphoblastic%20leukemia.%20J.%20Am.%20Med.%20Assoc.313,%20815%E2%80%93823%20(2015).)

22. Martin-Guerrero, I. et al. Variants in vincristine pharmacodynamic genes involved in neurotoxicity at induction phase in the therapy of pediatric acute lymphoblastic leukemia. Pharmacogenom. J.19, 564–569 (2019).  [DOI](https://doi.org/10.1038/s41397-019-0081-5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30723315/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Martin-Guerrero,%20I.%20et%20al.%20Variants%20in%20vincristine%20pharmacodynamic%20genes%20involved%20in%20neurotoxicity%20at%20induction%20phase%20in%20the%20therapy%20of%20pediatric%20acute%20lymphoblastic%20leukemia.%20Pharmacogenom.%20J.19,%20564%E2%80%93569%20(2019).)

23. Um, J. W. et al. LRRTM3 regulates excitatory synapse development through alternative splicing and neurexin binding. Cell Rep.14, 808–822 (2016).  [DOI](https://doi.org/10.1016/j.celrep.2015.12.081) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26776509/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Um,%20J.%20W.%20et%20al.%20LRRTM3%20regulates%20excitatory%20synapse%20development%20through%20alternative%20splicing%20and%20neurexin%20binding.%20Cell%20Rep.14,%20808%E2%80%93822%20(2016).)

24. Müller, T. et al. Neuregulin 3 promotes excitatory synapse formation on hippocampal interneurons. EMBO J.37, e98858 (2018).  [DOI](https://doi.org/10.15252/embj.201798858) | [PMC free article](/articles/PMC6120667/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30049711/) | [Google Scholar](https://scholar.google.com/scholar_lookup?M%C3%BCller,%20T.%20et%20al.%20Neuregulin%203%20promotes%20excitatory%20synapse%20formation%20on%20hippocampal%20interneurons.%20EMBO%20J.37,%20e98858%20(2018).)

25. Ledonne, A. & Mercuri, N. B. On the modulatory roles of neuregulins/ErbB signaling on synaptic plasticity. Int. J. Mol. Sci.21, 275 (2019).  [DOI](https://doi.org/10.3390/ijms21010275) | [PMC free article](/articles/PMC6981567/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31906113/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Ledonne,%20A.%20&%20Mercuri,%20N.%20B.%20On%20the%20modulatory%20roles%20of%20neuregulins/ErbB%20signaling%20on%20synaptic%20plasticity.%20Int.%20J.%20Mol.%20Sci.21,%20275%20(2019).)

26. Robison, A., Bartlett, R. K., Bass, M. A. & Colbran, R. J. Differential modulation of Ca2+/calmodulin-dependent protein kinase II activity by regulated interactions with N-methyl-D-aspartate receptor NR2B subunits and α-actinin. J. Biol. Chem.280, 39316–39323 (2005).  [DOI](https://doi.org/10.1074/jbc.M508189200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16172120/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Robison,%20A.,%20Bartlett,%20R.%20K.,%20Bass,%20M.%20A.%20&%20Colbran,%20R.%20J.%20Differential%20modulation%20of%20Ca2+/calmodulin-dependent%20protein%20kinase%20II%20activity%20by%20regulated%20interactions%20with%20N-methyl-D-aspartate%20receptor%20NR2B%20subunits%20and%20%CE%B1-actinin.%20J.%20Biol.%20Chem.280,%2039316%E2%80%9339323%20(2005).)

27. Hell, J. W. CaMKII: claiming center stage in postsynaptic function and organization. Neuron81, 249–265 (2014).  [DOI](https://doi.org/10.1016/j.neuron.2013.12.024) | [PMC free article](/articles/PMC4570830/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24462093/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Hell,%20J.%20W.%20CaMKII:%20claiming%20center%20stage%20in%20postsynaptic%20function%20and%20organization.%20Neuron81,%20249%E2%80%93265%20(2014).)

28. Fink, C. C. & Meyer, T. Molecular mechanisms of CaMKII activation in neuronal plasticity. Curr. Opin. Neurobiol.12, 293–299 (2002).  [DOI](https://doi.org/10.1016/s0959-4388(02)00327-6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12049936/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Fink,%20C.%20C.%20&%20Meyer,%20T.%20Molecular%20mechanisms%20of%20CaMKII%20activation%20in%20neuronal%20plasticity.%20Curr.%20Opin.%20Neurobiol.12,%20293%E2%80%93299%20(2002).)

29. Kunwar, A. J. et al. Lack of the endosomal SNAREs vti1a and vti1b led to significant impairments in neuronal development. Proc. Natl Acad. Sci. USA108, 2575–2580 (2011).  [DOI](https://doi.org/10.1073/pnas.1013891108) | [PMC free article](/articles/PMC3038762/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21262811/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Kunwar,%20A.%20J.%20et%20al.%20Lack%20of%20the%20endosomal%20SNAREs%20vti1a%20and%20vti1b%20led%20to%20significant%20impairments%20in%20neuronal%20development.%20Proc.%20Natl%20Acad.%20Sci.%20USA108,%202575%E2%80%932580%20(2011).)

30. Bollmann, C. et al. Primary neurons lacking the SNAREs vti1a and vti1b show altered neuronal development. Neural Dev.17, 1–13 (2022).  [DOI](https://doi.org/10.1186/s13064-022-00168-2) | [PMC free article](/articles/PMC9682837/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36419086/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Bollmann,%20C.%20et%20al.%20Primary%20neurons%20lacking%20the%20SNAREs%20vti1a%20and%20vti1b%20show%20altered%20neuronal%20development.%20Neural%20Dev.17,%201%E2%80%9313%20(2022).)

31. Gronostajski, R. M. Roles of the NFI/CTF gene family in transcription and development. Gene249, 31–45 (2000).  [DOI](https://doi.org/10.1016/s0378-1119(00)00140-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10831836/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Gronostajski,%20R.%20M.%20Roles%20of%20the%20NFI/CTF%20gene%20family%20in%20transcription%20and%20development.%20Gene249,%2031%E2%80%9345%20(2000).)

32. Mason, S., Piper, M., Gronostajski, R. M. & Richards, L. J. Nuclear factor one transcription factors in CNS development. Mol. Neurobiol.39, 10–23 (2009).  [DOI](https://doi.org/10.1007/s12035-008-8048-6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19058033/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Mason,%20S.,%20Piper,%20M.,%20Gronostajski,%20R.%20M.%20&%20Richards,%20L.%20J.%20Nuclear%20factor%20one%20transcription%20factors%20in%20CNS%20development.%20Mol.%20Neurobiol.39,%2010%E2%80%9323%20(2009).)

33. Kumbasar, A., Plachez, C., Gronostajski, R. M., Richards, L. J. & Litwack, E. D. Absence of the transcription factor Nfib delays the formation of the basilar pontine and other mossy fiber nuclei. J. Comp. Neurol.513, 98–112 (2009).  [DOI](https://doi.org/10.1002/cne.21943) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19107796/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Kumbasar,%20A.,%20Plachez,%20C.,%20Gronostajski,%20R.%20M.,%20Richards,%20L.%20J.%20&%20Litwack,%20E.%20D.%20Absence%20of%20the%20transcription%20factor%20Nfib%20delays%20the%20formation%20of%20the%20basilar%20pontine%20and%20other%20mossy%20fiber%20nuclei.%20J.%20Comp.%20Neurol.513,%2098%E2%80%93112%20(2009).)

34. Yam, M. F. et al. General pathways of pain sensation and the major neurotransmitters involved in pain regulation. Int. J. Mol. Sci.19, 2164 (2018).  [DOI](https://doi.org/10.3390/ijms19082164) | [PMC free article](/articles/PMC6121522/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30042373/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Yam,%20M.%20F.%20et%20al.%20General%20pathways%20of%20pain%20sensation%20and%20the%20major%20neurotransmitters%20involved%20in%20pain%20regulation.%20Int.%20J.%20Mol.%20Sci.19,%202164%20(2018).)

35. Zenker, M. et al. Variants in nuclear factor I genes influence growth and development. In: American Journal of Medical Genetics Part C: Seminars in Medical Genetics. (Wiley Online Library, 2019).  [DOI](https://doi.org/10.1002/ajmg.c.31747) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31730271/)

36. Jensen, S. A., Robertson, I. B. & Handford, P. A. Dissecting the fibrillin microfibril: structural insights into organization and function. Structure20, 215–225 (2012).  [DOI](https://doi.org/10.1016/j.str.2011.12.008) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22325771/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Jensen,%20S.%20A.,%20Robertson,%20I.%20B.%20&%20Handford,%20P.%20A.%20Dissecting%20the%20fibrillin%20microfibril:%20structural%20insights%20into%20organization%20and%20function.%20Structure20,%20215%E2%80%93225%20(2012).)

37. Peeters, S. et al. Delineation of a new fibrillin-2-opathy with evidence for a role of FBN2 in the pathogenesis of carpal tunnel syndrome. J. Med. Genet.58, 778–782 (2021).  [DOI](https://doi.org/10.1136/jmedgenet-2020-107085) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32900841/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Peeters,%20S.%20et%20al.%20Delineation%20of%20a%20new%20fibrillin-2-opathy%20with%20evidence%20for%20a%20role%20of%20FBN2%20in%20the%20pathogenesis%20of%20carpal%20tunnel%20syndrome.%20J.%20Med.%20Genet.58,%20778%E2%80%93782%20(2021).)

38. Sedghi, M. et al. Recessive charcot-marie-tooth and multiple sclerosis associated with a variant in MCM3AP. Brain Commun.1, fcz011 (2019).  [DOI](https://doi.org/10.1093/braincomms/fcz011) | [PMC free article](/articles/PMC7425404/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32954258/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Sedghi,%20M.%20et%20al.%20Recessive%20charcot-marie-tooth%20and%20multiple%20sclerosis%20associated%20with%20a%20variant%20in%20MCM3AP.%20Brain%20Commun.1,%20fcz011%20(2019).)

39. Karakaya, M. et al. Biallelic MCM3AP mutations cause Charcot-Marie-Tooth neuropathy with variable clinical presentation. Brain140, e65 (2017).  [DOI](https://doi.org/10.1093/brain/awx222) | [PMC free article](/articles/PMC6272822/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28969388/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Karakaya,%20M.%20et%20al.%20Biallelic%20MCM3AP%20mutations%20cause%20Charcot-Marie-Tooth%20neuropathy%20with%20variable%20clinical%20presentation.%20Brain140,%20e65%20(2017).)

40. Ylikallio, E. et al. MCM3AP in recessive Charcot-Marie-Tooth neuropathy and mild intellectual disability. Brain140, 2093–2103 (2017).  [DOI](https://doi.org/10.1093/brain/awx138) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28633435/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Ylikallio,%20E.%20et%20al.%20MCM3AP%20in%20recessive%20Charcot-Marie-Tooth%20neuropathy%20and%20mild%20intellectual%20disability.%20Brain140,%202093%E2%80%932103%20(2017).)

41. Kennerson, M. L., Corbett, A. C., Ellis, M., Perez-Siles, G. & Nicholson, G. A. A novel MCM3AP mutation in a Lebanese family with recessive Charcot-Marie-Tooth neuropathy. Brain141, e66 (2018).  [DOI](https://doi.org/10.1093/brain/awy184) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29982295/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Kennerson,%20M.%20L.,%20Corbett,%20A.%20C.,%20Ellis,%20M.,%20Perez-Siles,%20G.%20&%20Nicholson,%20G.%20A.%20A%20novel%20MCM3AP%20mutation%20in%20a%20Lebanese%20family%20with%20recessive%20Charcot-Marie-Tooth%20neuropathy.%20Brain141,%20e66%20(2018).)

42. Woldegebriel, R. Identification and role of MCM3AP disease gene in neurological disease spectrum and its neuronal modelling. Dissertationes Scholae Doctoralis Ad Sanitatem Investigandam Universitatis Helsinkiensis (2021).

43. Cristofani, R., Rusmini, P. & Galbiati, M. The regulation of the small heat shock protein B8 in misfolding protein diseases causing motoneuronal and muscle cell death. Front. Neurosci.13, 796 (2019).  [DOI](https://doi.org/10.3389/fnins.2019.00796) | [PMC free article](/articles/PMC6688727/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31427919/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Cristofani,%20R.,%20Rusmini,%20P.%20&%20Galbiati,%20M.%20The%20regulation%20of%20the%20small%20heat%20shock%20protein%20B8%20in%20misfolding%20protein%20diseases%20causing%20motoneuronal%20and%20muscle%20cell%20death.%20Front.%20Neurosci.13,%20796%20(2019).)

44. Turakhiya, A. et al. ZFAND1 recruits p97 and the 26S proteasome to promote the clearance of arsenite-induced stress granules. Mol. Cell70, 906–919.e907 (2018).  [DOI](https://doi.org/10.1016/j.molcel.2018.04.021) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29804830/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Turakhiya,%20A.%20et%20al.%20ZFAND1%20recruits%20p97%20and%20the%2026S%20proteasome%20to%20promote%20the%20clearance%20of%20arsenite-induced%20stress%20granules.%20Mol.%20Cell70,%20906%E2%80%93919.e907%20(2018).)

45. Cui, Q. et al. Diverse CMT2 neuropathies are linked to aberrant G3BP interactions in stress granules. Cell186, 803–820.e825 (2023).  [DOI](https://doi.org/10.1016/j.cell.2022.12.046) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36738734/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Cui,%20Q.%20et%20al.%20Diverse%20CMT2%20neuropathies%20are%20linked%20to%20aberrant%20G3BP%20interactions%20in%20stress%20granules.%20Cell186,%20803%E2%80%93820.e825%20(2023).)

46. Ndiaye, F. K. et al. Expression and functional assessment of candidate type 2 diabetes susceptibility genes identify four new genes contributing to human insulin secretion. Mol. Metab.6, 459–470 (2017).  [DOI](https://doi.org/10.1016/j.molmet.2017.03.011) | [PMC free article](/articles/PMC5444093/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28580277/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Ndiaye,%20F.%20K.%20et%20al.%20Expression%20and%20functional%20assessment%20of%20candidate%20type%202%20diabetes%20susceptibility%20genes%20identify%20four%20new%20genes%20contributing%20to%20human%20insulin%20secretion.%20Mol.%20Metab.6,%20459%E2%80%93470%20(2017).)

47. Song, C. et al. Ten SNPs may affect type 2 diabetes risk in interaction with prenatal exposure to Chinese famine. Nutrients12, 3880 (2020).  [DOI](https://doi.org/10.3390/nu12123880) | [PMC free article](/articles/PMC7766924/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33353041/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Song,%20C.%20et%20al.%20Ten%20SNPs%20may%20affect%20type%202%20diabetes%20risk%20in%20interaction%20with%20prenatal%20exposure%20to%20Chinese%20famine.%20Nutrients12,%203880%20(2020).)

48. Houslay, M. D. & Adams, D. R. PDE4 cAMP phosphodiesterases: modular enzymes that orchestrate signalling cross-talk, desensitization and compartmentalization. Biochem. J.370, 1–18 (2003).  [DOI](https://doi.org/10.1042/BJ20021698) | [PMC free article](/articles/PMC1223165/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12444918/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Houslay,%20M.%20D.%20&%20Adams,%20D.%20R.%20PDE4%20cAMP%20phosphodiesterases:%20modular%20enzymes%20that%20orchestrate%20signalling%20cross-talk,%20desensitization%20and%20compartmentalization.%20Biochem.%20J.370,%201%E2%80%9318%20(2003).)

49. Baillie, G. S., Tejeda, G. S. & Kelly, M. P. Therapeutic targeting of 3′, 5′-cyclic nucleotide phosphodiesterases: inhibition and beyond. Nat. Rev. Drug Discov.18, 770–796 (2019).  [DOI](https://doi.org/10.1038/s41573-019-0033-4) | [PMC free article](/articles/PMC6773486/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31388135/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Baillie,%20G.%20S.,%20Tejeda,%20G.%20S.%20&%20Kelly,%20M.%20P.%20Therapeutic%20targeting%20of%203%E2%80%B2,%205%E2%80%B2-cyclic%20nucleotide%20phosphodiesterases:%20inhibition%20and%20beyond.%20Nat.%20Rev.%20Drug%20Discov.18,%20770%E2%80%93796%20(2019).)

50. Datta, D. et al. Mapping phosphodiesterase 4D (PDE4D) in macaque dorsolateral prefrontal cortex: postsynaptic compartmentalization in layer III pyramidal cell circuits. Front. Neuroanat.14, 578483 (2020).  [DOI](https://doi.org/10.3389/fnana.2020.578483) | [PMC free article](/articles/PMC7714912/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33328902/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Datta,%20D.%20et%20al.%20Mapping%20phosphodiesterase%204D%20(PDE4D)%20in%20macaque%20dorsolateral%20prefrontal%20cortex:%20postsynaptic%20compartmentalization%20in%20layer%20III%20pyramidal%20cell%20circuits.%20Front.%20Neuroanat.14,%20578483%20(2020).)

51. Zhang, F. F. et al. Inhibition of phosphodiesterase‐4 in the spinal dorsal horn ameliorates neuropathic pain via cAMP‐cytokine‐Cx43 signaling in mice. CNS Neurosci. Therapeutics28, 749–760 (2022).  [DOI](https://doi.org/10.1111/cns.13807) | [PMC free article](/articles/PMC8981432/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35156776/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Zhang,%20F.%20F.%20et%20al.%20Inhibition%20of%20phosphodiesterase%E2%80%904%20in%20the%20spinal%20dorsal%20horn%20ameliorates%20neuropathic%20pain%20via%20cAMP%E2%80%90cytokine%E2%80%90Cx43%20signaling%20in%20mice.%20CNS%20Neurosci.%20Therapeutics28,%20749%E2%80%93760%20(2022).)

52. Bhat, A. et al. Phosphodiesterase-4 enzyme as a therapeutic target in neurological disorders. Pharmacol. Res.160, 105078 (2020).  [DOI](https://doi.org/10.1016/j.phrs.2020.105078) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32673703/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Bhat,%20A.%20et%20al.%20Phosphodiesterase-4%20enzyme%20as%20a%20therapeutic%20target%20in%20neurological%20disorders.%20Pharmacol.%20Res.160,%20105078%20(2020).)

53. Gomez, A. M., Traunmüller, L. & Scheiffele, P. Neurexins: molecular codes for shaping neuronal synapses. Nat. Rev. Neurosci.22, 137–151 (2021).  [DOI](https://doi.org/10.1038/s41583-020-00415-7) | [PMC free article](/articles/PMC7612283/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33420412/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Gomez,%20A.%20M.,%20Traunm%C3%BCller,%20L.%20&%20Scheiffele,%20P.%20Neurexins:%20molecular%20codes%20for%20shaping%20neuronal%20synapses.%20Nat.%20Rev.%20Neurosci.22,%20137%E2%80%93151%20(2021).)

54. Wan, C. et al. Neuregulin1-ErbB4 signaling in spinal cord participates in electroacupuncture analgesia in inflammatory pain. Front Neurosci.15, 636348 (2021).  [DOI](https://doi.org/10.3389/fnins.2021.636348) | [PMC free article](/articles/PMC7875897/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33584196/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Wan,%20C.%20et%20al.%20Neuregulin1-ErbB4%20signaling%20in%20spinal%20cord%20participates%20in%20electroacupuncture%20analgesia%20in%20inflammatory%20pain.%20Front%20Neurosci.15,%20636348%20(2021).)

55. Dai, D. W. et al. Distinct roles of neuregulin in different models of neuropathic pain. Neurol. Sci.35, 531–536 (2014).  [DOI](https://doi.org/10.1007/s10072-013-1537-z) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24057116/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Dai,%20D.%20W.%20et%20al.%20Distinct%20roles%20of%20neuregulin%20in%20different%20models%20of%20neuropathic%20pain.%20Neurol.%20Sci.35,%20531%E2%80%93536%20(2014).)

56. Antonin, W., Riedel, D. & Von Mollard, G. F. The SNARE Vti1a-β is localized to small synaptic vesicles and participates in a novel SNARE complex. J. Neurosci.20, 5724–5732 (2000).  [DOI](https://doi.org/10.1523/JNEUROSCI.20-15-05724.2000) | [PMC free article](/articles/PMC6772562/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10908612/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Antonin,%20W.,%20Riedel,%20D.%20&%20Von%20Mollard,%20G.%20F.%20The%20SNARE%20Vti1a-%CE%B2%20is%20localized%20to%20small%20synaptic%20vesicles%20and%20participates%20in%20a%20novel%20SNARE%20complex.%20J.%20Neurosci.20,%205724%E2%80%935732%20(2000).)

57. Prior, R., Van Helleputte, L., Benoy, V. & Van Den Bosch, L. Defective axonal transport: a common pathological mechanism in inherited and acquired peripheral neuropathies. Neurobiol. Dis.105, 300–320 (2017).  [DOI](https://doi.org/10.1016/j.nbd.2017.02.009) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28238949/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Prior,%20R.,%20Van%20Helleputte,%20L.,%20Benoy,%20V.%20&%20Van%20Den%20Bosch,%20L.%20Defective%20axonal%20transport:%20a%20common%20pathological%20mechanism%20in%20inherited%20and%20acquired%20peripheral%20neuropathies.%20Neurobiol.%20Dis.105,%20300%E2%80%93320%20(2017).)

58. Pareyson, D., Saveri, P., Sagnelli, A. & Piscosquito, G. Mitochondrial dynamics and inherited peripheral nerve diseases. Neurosci. Lett.596, 66–77 (2015).  [DOI](https://doi.org/10.1016/j.neulet.2015.04.001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25847151/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Pareyson,%20D.,%20Saveri,%20P.,%20Sagnelli,%20A.%20&%20Piscosquito,%20G.%20Mitochondrial%20dynamics%20and%20inherited%20peripheral%20nerve%20diseases.%20Neurosci.%20Lett.596,%2066%E2%80%9377%20(2015).)

59. Otey, C. A. & Carpen, O. α‐actinin revisited: a fresh look at an old player. Cell Motil. Cytoskelet.58, 104–111 (2004).  [DOI](https://doi.org/10.1002/cm.20007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15083532/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Otey,%20C.%20A.%20&%20Carpen,%20O.%20%CE%B1%E2%80%90actinin%20revisited:%20a%20fresh%20look%20at%20an%20old%20player.%20Cell%20Motil.%20Cytoskelet.58,%20104%E2%80%93111%20(2004).)

60. Kuramochi, Y., Guo, X. & Sawyer, D. B. Neuregulin activates erbB2-dependent src/FAK signaling and cytoskeletal remodeling in isolated adult rat cardiac myocytes. J. Mol. Cell. Cardiol.41, 228–235 (2006).  [DOI](https://doi.org/10.1016/j.yjmcc.2006.04.007) | [PMC free article](/articles/PMC1847613/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16769082/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Kuramochi,%20Y.,%20Guo,%20X.%20&%20Sawyer,%20D.%20B.%20Neuregulin%20activates%20erbB2-dependent%20src/FAK%20signaling%20and%20cytoskeletal%20remodeling%20in%20isolated%20adult%20rat%20cardiac%20myocytes.%20J.%20Mol.%20Cell.%20Cardiol.41,%20228%E2%80%93235%20(2006).)

61. Héraud, C., Pinault, M., Lagrée, V. & Moreau, V. p190RhoGAPs, the ARHGAP35-and ARHGAP5-encoded proteins, in health and disease. Cells8, 351 (2019).  [DOI](https://doi.org/10.3390/cells8040351) | [PMC free article](/articles/PMC6523970/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31013840/) | [Google Scholar](https://scholar.google.com/scholar_lookup?H%C3%A9raud,%20C.,%20Pinault,%20M.,%20Lagr%C3%A9e,%20V.%20&%20Moreau,%20V.%20p190RhoGAPs,%20the%20ARHGAP35-and%20ARHGAP5-encoded%20proteins,%20in%20health%20and%20disease.%20Cells8,%20351%20(2019).)

62. Moradi, M. et al. Differential roles of α-, β-, and γ-actin in axon growth and collateral branch formation in motoneurons. J. Cell Biol.216, 793–814 (2017).  [DOI](https://doi.org/10.1083/jcb.201604117) | [PMC free article](/articles/PMC5346967/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28246119/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Moradi,%20M.%20et%20al.%20Differential%20roles%20of%20%CE%B1-,%20%CE%B2-,%20and%20%CE%B3-actin%20in%20axon%20growth%20and%20collateral%20branch%20formation%20in%20motoneurons.%20J.%20Cell%20Biol.216,%20793%E2%80%93814%20(2017).)

63. Kautio, A.-L. et al. Amitriptyline in the prevention of chemotherapy-induced neuropathic symptoms. Anticancer Res.29, 2601–2606 (2009).  [PubMed](https://pubmed.ncbi.nlm.nih.gov/19596934/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Kautio,%20A.-L.%20et%20al.%20Amitriptyline%20in%20the%20prevention%20of%20chemotherapy-induced%20neuropathic%20symptoms.%20Anticancer%20Res.29,%202601%E2%80%932606%20(2009).)

64. Koeppen, S. et al. Lack of neuroprotection by an ACTH (4–9) analogue. A randomized trial in patients treated with vincristine for Hodgkin’s or non-Hodgkin’s lymphoma. J. Cancer Res. Clin. Oncol.130, 153–160 (2004).  [DOI](https://doi.org/10.1007/s00432-003-0524-9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14727106/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Koeppen,%20S.%20et%20al.%20Lack%20of%20neuroprotection%20by%20an%20ACTH%20(4%E2%80%939)%20analogue.%20A%20randomized%20trial%20in%20patients%20treated%20with%20vincristine%20for%20Hodgkin%E2%80%99s%20or%20non-Hodgkin%E2%80%99s%20lymphoma.%20J.%20Cancer%20Res.%20Clin.%20Oncol.130,%20153%E2%80%93160%20(2004).)

65. Van Kooten, B. et al. A pilot study on the influence of a corticotropin (4-9) analogue on vinca alkaloid—induced neuropathy. Arch. Neurol.49, 1027–1031 (1992).  [DOI](https://doi.org/10.1001/archneur.1992.00530340043016) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1329697/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Van%20Kooten,%20B.%20et%20al.%20A%20pilot%20study%20on%20the%20influence%20of%20a%20corticotropin%20(4-9)%20analogue%20on%20vinca%20alkaloid%E2%80%94induced%20neuropathy.%20Arch.%20Neurol.49,%201027%E2%80%931031%20(1992).)

66. Rao, R. D. et al. Efficacy of gabapentin in the management of chemotherapy‐induced peripheral neuropathy: a phase 3 randomized, double‐blind, placebo‐controlled, crossover trial (N00C3). Cancer110, 2110–2118 (2007).  [DOI](https://doi.org/10.1002/cncr.23008) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17853395/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Rao,%20R.%20D.%20et%20al.%20Efficacy%20of%20gabapentin%20in%20the%20management%20of%20chemotherapy%E2%80%90induced%20peripheral%20neuropathy:%20a%20phase%203%20randomized,%20double%E2%80%90blind,%20placebo%E2%80%90controlled,%20crossover%20trial%20(N00C3).%20Cancer110,%202110%E2%80%932118%20(2007).)

67. Anghelescu, D. L. et al. Prospective randomized trial of interventions for vincristine‐related neuropathic pain. Pediatr. Blood Cancer67, e28539 (2020).  [DOI](https://doi.org/10.1002/pbc.28539) | [PMC free article](/articles/PMC8149969/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32618122/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Anghelescu,%20D.%20L.%20et%20al.%20Prospective%20randomized%20trial%20of%20interventions%20for%20vincristine%E2%80%90related%20neuropathic%20pain.%20Pediatr.%20Blood%20Cancer67,%20e28539%20(2020).)

68. Jackson, D. V. et al. Clinical trial of pyridoxine to reduce vincristine neurotoxicity. J. Neuro-Oncol.4, 37–41 (1986).  [DOI](https://doi.org/10.1007/BF02158000) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/3528407/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Jackson,%20D.%20V.%20et%20al.%20Clinical%20trial%20of%20pyridoxine%20to%20reduce%20vincristine%20neurotoxicity.%20J.%20Neuro-Oncol.4,%2037%E2%80%9341%20(1986).)

69. Bradfield, S. M., Sandler, E., Geller, T., Tamura, R. N. & Krischer, J. P. Glutamic acid not beneficial for the prevention of vincristine neurotoxicity in children with cancer. Pediatr. Blood Cancer62, 1004–1010 (2015).  [DOI](https://doi.org/10.1002/pbc.25384) | [PMC free article](/articles/PMC4405446/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25545757/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Bradfield,%20S.%20M.,%20Sandler,%20E.,%20Geller,%20T.,%20Tamura,%20R.%20N.%20&%20Krischer,%20J.%20P.%20Glutamic%20acid%20not%20beneficial%20for%20the%20prevention%20of%20vincristine%20neurotoxicity%20in%20children%20with%20cancer.%20Pediatr.%20Blood%20Cancer62,%201004%E2%80%931010%20(2015).)

70. Mokhtar, G. M., Shaaban, S. Y., Elbarbary, N. S. & Fayed, W. A. A trial to assess the efficacy of glutamic acid in prevention of vincristine-induced neurotoxicity in pediatric malignancies: a pilot study. J. Pediatr. Hematol./Oncol.32, 594–600 (2010).  [DOI](https://doi.org/10.1097/MPH.0b013e3181e9038d) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20921909/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Mokhtar,%20G.%20M.,%20Shaaban,%20S.%20Y.,%20Elbarbary,%20N.%20S.%20&%20Fayed,%20W.%20A.%20A%20trial%20to%20assess%20the%20efficacy%20of%20glutamic%20acid%20in%20prevention%20of%20vincristine-induced%20neurotoxicity%20in%20pediatric%20malignancies:%20a%20pilot%20study.%20J.%20Pediatr.%20Hematol./Oncol.32,%20594%E2%80%93600%20(2010).)

71. Sands, S. et al. Glutamine for the treatment of vincristine-induced neuropathy in children and adolescents with cancer. Supportive Care Cancer25, 701–708 (2017).  [DOI](https://doi.org/10.1007/s00520-016-3441-6) | [PMC free article](/articles/PMC5598352/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27830395/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Sands,%20S.%20et%20al.%20Glutamine%20for%20the%20treatment%20of%20vincristine-induced%20neuropathy%20in%20children%20and%20adolescents%20with%20cancer.%20Supportive%20Care%20Cancer25,%20701%E2%80%93708%20(2017).)

72. Gutierrez-Camino, A. et al. Lack of association of the CEP72 rs924607 TT genotype with vincristine-related peripheral neuropathy during the early phase of pediatric acute lymphoblastic leukemia treatment in a Spanish population. Pharmacogenet. Genom.26, 100–102 (2016).  [DOI](https://doi.org/10.1097/FPC.0000000000000191) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26618658/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Gutierrez-Camino,%20A.%20et%20al.%20Lack%20of%20association%20of%20the%20CEP72%20rs924607%20TT%20genotype%20with%20vincristine-related%20peripheral%20neuropathy%20during%20the%20early%20phase%20of%20pediatric%20acute%20lymphoblastic%20leukemia%20treatment%20in%20a%20Spanish%20population.%20Pharmacogenet.%20Genom.26,%20100%E2%80%93102%20(2016).)

73. Diouf, B., Crews, K.R. & Evans, W.E. Vincristine pharmacogenomics:‘winner’s curse’or a different phenotype?. LWW (2016).  [DOI](https://doi.org/10.1097/FPC.0000000000000192) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26704670/)

74. Rummel, M. J. et al. Bendamustine plus rituximab versus CHOP plus rituximab as first-line treatment for patients with indolent and mantle-cell lymphomas: an open-label, multicentre, randomised, phase 3 non-inferiority trial. Lancet381, 1203–1210 (2013).  [DOI](https://doi.org/10.1016/S0140-6736(12)61763-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23433739/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Rummel,%20M.%20J.%20et%20al.%20Bendamustine%20plus%20rituximab%20versus%20CHOP%20plus%20rituximab%20as%20first-line%20treatment%20for%20patients%20with%20indolent%20and%20mantle-cell%20lymphomas:%20an%20open-label,%20multicentre,%20randomised,%20phase%203%20non-inferiority%20trial.%20Lancet381,%201203%E2%80%931210%20(2013).)

75. Bartlett, N. L. et al. Dose-adjusted EPOCH-R compared with R-CHOP as frontline therapy for diffuse large B-cell lymphoma: clinical outcomes of the phase III intergroup trial alliance/CALGB 50303. J. Clin. Oncol.37, 1790 (2019).  [DOI](https://doi.org/10.1200/JCO.18.01994) | [PMC free article](/articles/PMC6774813/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30939090/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Bartlett,%20N.%20L.%20et%20al.%20Dose-adjusted%20EPOCH-R%20compared%20with%20R-CHOP%20as%20frontline%20therapy%20for%20diffuse%20large%20B-cell%20lymphoma:%20clinical%20outcomes%20of%20the%20phase%20III%20intergroup%20trial%20alliance/CALGB%2050303.%20J.%20Clin.%20Oncol.37,%201790%20(2019).)

76. Moriyama, B. et al. Adverse interactions between antifungal azoles and vincristine: review and analysis of cases. Mycoses55, 290–297 (2012).  [DOI](https://doi.org/10.1111/j.1439-0507.2011.02158.x) | [PMC free article](/articles/PMC3345292/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22126626/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Moriyama,%20B.%20et%20al.%20Adverse%20interactions%20between%20antifungal%20azoles%20and%20vincristine:%20review%20and%20analysis%20of%20cases.%20Mycoses55,%20290%E2%80%93297%20(2012).)

77. Medaglio, D., Stephens‐Shields, A. J. & Leonard, C. E. Research and scholarly methods: propensity scores. J. Am. Coll. Clin. Pharm.5, 467–475 (2022).  [DOI](https://doi.org/10.1002/jac5.1591) | [PMC free article](/articles/PMC9980423/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36873057/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Medaglio,%20D.,%20Stephens%E2%80%90Shields,%20A.%20J.%20&%20Leonard,%20C.%20E.%20Research%20and%20scholarly%20methods:%20propensity%20scores.%20J.%20Am.%20Coll.%20Clin.%20Pharm.5,%20467%E2%80%93475%20(2022).)

78. Delaneau, O., Zagury, J.-F. & Marchini, J. Improved whole-chromosome phasing for disease and population genetic studies. Nat. Methods10, 5–6 (2013).  [DOI](https://doi.org/10.1038/nmeth.2307) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23269371/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Delaneau,%20O.,%20Zagury,%20J.-F.%20&%20Marchini,%20J.%20Improved%20whole-chromosome%20phasing%20for%20disease%20and%20population%20genetic%20studies.%20Nat.%20Methods10,%205%E2%80%936%20(2013).)

79. Howie, B. N., Donnelly, P. & Marchini, J. A flexible and accurate genotype imputation method for the next generation of genome-wide association studies. PLoS Genet.5, e1000529 (2009).  [DOI](https://doi.org/10.1371/journal.pgen.1000529) | [PMC free article](/articles/PMC2689936/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19543373/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Howie,%20B.%20N.,%20Donnelly,%20P.%20&%20Marchini,%20J.%20A%20flexible%20and%20accurate%20genotype%20imputation%20method%20for%20the%20next%20generation%20of%20genome-wide%20association%20studies.%20PLoS%20Genet.5,%20e1000529%20(2009).)

80. Purcell, S. et al. PLINK: a tool set for whole-genome association and population-based linkage analyses. Am. J. Hum. Genet.81, 559–575 (2007).  [DOI](https://doi.org/10.1086/519795) | [PMC free article](/articles/PMC1950838/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17701901/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Purcell,%20S.%20et%20al.%20PLINK:%20a%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses.%20Am.%20J.%20Hum.%20Genet.81,%20559%E2%80%93575%20(2007).)

81. Deelen, P. et al. Genotype harmonizer: automatic strand alignment and format conversion for genotype data integration. BMC Res. Notes7, 1–4 (2014).  [DOI](https://doi.org/10.1186/1756-0500-7-901) | [PMC free article](/articles/PMC4307387/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25495213/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Deelen,%20P.%20et%20al.%20Genotype%20harmonizer:%20automatic%20strand%20alignment%20and%20format%20conversion%20for%20genotype%20data%20integration.%20BMC%20Res.%20Notes7,%201%E2%80%934%20(2014).)

82. Machiela, M. J. & Chanock, S. J. LDlink: a web-based application for exploring population-specific haplotype structure and linking correlated alleles of possible functional variants. Bioinformatics31, 3555–3557 (2015).  [DOI](https://doi.org/10.1093/bioinformatics/btv402) | [PMC free article](/articles/PMC4626747/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26139635/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Machiela,%20M.%20J.%20&%20Chanock,%20S.%20J.%20LDlink:%20a%20web-based%20application%20for%20exploring%20population-specific%20haplotype%20structure%20and%20linking%20correlated%20alleles%20of%20possible%20functional%20variants.%20Bioinformatics31,%203555%E2%80%933557%20(2015).)

83. Watanabe, K., Taskesen, E., van Bochoven, A. & Posthuma, D. FUMA: functional mapping and annotation of genetic associations. Eur. Neuropsychopharmacol.29, S789–S790 (2019).  [DOI](https://doi.org/10.1038/s41467-017-01261-5) | [PMC free article](/articles/PMC5705698/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29184056/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Watanabe,%20K.,%20Taskesen,%20E.,%20van%20Bochoven,%20A.%20&%20Posthuma,%20D.%20FUMA:%20functional%20mapping%20and%20annotation%20of%20genetic%20associations.%20Eur.%20Neuropsychopharmacol.29,%20S789%E2%80%93S790%20(2019).)

84. Wang, J., Vasaikar, S., Shi, Z., Greer, M. & Zhang, B. WebGestalt 2017: a more comprehensive, powerful, flexible and interactive gene set enrichment analysis toolkit. Nucleic Acids Res.45, W130–W137 (2017).  [DOI](https://doi.org/10.1093/nar/gkx356) | [PMC free article](/articles/PMC5570149/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28472511/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Wang,%20J.,%20Vasaikar,%20S.,%20Shi,%20Z.,%20Greer,%20M.%20&%20Zhang,%20B.%20WebGestalt%202017:%20a%20more%20comprehensive,%20powerful,%20flexible%20and%20interactive%20gene%20set%20enrichment%20analysis%20toolkit.%20Nucleic%20Acids%20Res.45,%20W130%E2%80%93W137%20(2017).)

85. Tang, F. et al. The role of Vti1a in biological functions and its possible role in nervous system disorders. Front. Mol. Neurosci.15, 918664 (2022).  [DOI](https://doi.org/10.3389/fnmol.2022.918664) | [PMC free article](/articles/PMC9197314/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35711736/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Tang,%20F.%20et%20al.%20The%20role%20of%20Vti1a%20in%20biological%20functions%20and%20its%20possible%20role%20in%20nervous%20system%20disorders.%20Front.%20Mol.%20Neurosci.15,%20918664%20(2022).)

86. Tang, B. L. Vesicle transport through interaction with t-SNAREs 1a (Vti1a)’s roles in neurons. Heliyon6, e04600 (2020).  [DOI](https://doi.org/10.1016/j.heliyon.2020.e04600) | [PMC free article](/articles/PMC7398939/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32775753/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Tang,%20B.%20L.%20Vesicle%20transport%20through%20interaction%20with%20t-SNAREs%201a%20(Vti1a)%E2%80%99s%20roles%20in%20neurons.%20Heliyon6,%20e04600%20(2020).)

87. Ou, G.-Y., Lin, W.-W. & Zhao, W.-J. Neuregulins in neurodegenerative diseases. Front. Aging Neurosci.13, 662474 (2021).  [DOI](https://doi.org/10.3389/fnagi.2021.662474) | [PMC free article](/articles/PMC8064692/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33897409/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Ou,%20G.-Y.,%20Lin,%20W.-W.%20&%20Zhao,%20W.-J.%20Neuregulins%20in%20neurodegenerative%20diseases.%20Front.%20Aging%20Neurosci.13,%20662474%20(2021).)

88. Kevenaar, J. T. & Hoogenraad, C. C. The axonal cytoskeleton: from organization to function. Front. Mol. Neurosci.8, 44 (2015).  [DOI](https://doi.org/10.3389/fnmol.2015.00044) | [PMC free article](/articles/PMC4536388/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26321907/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Kevenaar,%20J.%20T.%20&%20Hoogenraad,%20C.%20C.%20The%20axonal%20cytoskeleton:%20from%20organization%20to%20function.%20Front.%20Mol.%20Neurosci.8,%2044%20(2015).)

89. Lee, D., Takayama, S. & Goldberg, A. L. ZFAND5/ZNF216 is an activator of the 26S proteasome that stimulates overall protein degradation. Proc. Natl Acad. Sci. USA115, E9550–E9559 (2018).  [DOI](https://doi.org/10.1073/pnas.1809934115) | [PMC free article](/articles/PMC6187164/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30254168/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Lee,%20D.,%20Takayama,%20S.%20&%20Goldberg,%20A.%20L.%20ZFAND5/ZNF216%20is%20an%20activator%20of%20the%2026S%20proteasome%20that%20stimulates%20overall%20protein%20degradation.%20Proc.%20Natl%20Acad.%20Sci.%20USA115,%20E9550%E2%80%93E9559%20(2018).)

90. Mei, L. & Nave, K.-A. Neuregulin-ERBB signaling in the nervous system and neuropsychiatric diseases. Neuron83, 27–49 (2014).  [DOI](https://doi.org/10.1016/j.neuron.2014.06.007) | [PMC free article](/articles/PMC4189115/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24991953/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Mei,%20L.%20&%20Nave,%20K.-A.%20Neuregulin-ERBB%20signaling%20in%20the%20nervous%20system%20and%20neuropsychiatric%20diseases.%20Neuron83,%2027%E2%80%9349%20(2014).)

91. Lee, S. M., Chin, L.-S. & Li, L. Dysregulation of ErbB receptor trafficking and signaling in demyelinating Charcot-Marie-Tooth disease. Mol. Neurobiol.54, 87–100 (2017).  [DOI](https://doi.org/10.1007/s12035-015-9668-2) | [PMC free article](/articles/PMC4935639/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26732592/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Lee,%20S.%20M.,%20Chin,%20L.-S.%20&%20Li,%20L.%20Dysregulation%20of%20ErbB%20receptor%20trafficking%20and%20signaling%20in%20demyelinating%20Charcot-Marie-Tooth%20disease.%20Mol.%20Neurobiol.54,%2087%E2%80%93100%20(2017).)

92. Michailov, G. V. et al. Axonal neuregulin-1 regulates myelin sheath thickness. Science304, 700–703 (2004).  [DOI](https://doi.org/10.1126/science.1095862) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15044753/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Michailov,%20G.%20V.%20et%20al.%20Axonal%20neuregulin-1%20regulates%20myelin%20sheath%20thickness.%20Science304,%20700%E2%80%93703%20(2004).)

93. Brinkmann, B. G. et al. Neuregulin-1/ErbB signaling serves distinct functions in myelination of the peripheral and central nervous system. Neuron59, 581–595 (2008).  [DOI](https://doi.org/10.1016/j.neuron.2008.06.028) | [PMC free article](/articles/PMC2628490/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18760695/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Brinkmann,%20B.%20G.%20et%20al.%20Neuregulin-1/ErbB%20signaling%20serves%20distinct%20functions%20in%20myelination%20of%20the%20peripheral%20and%20central%20nervous%20system.%20Neuron59,%20581%E2%80%93595%20(2008).)

94. Kong, L. et al. Boosting neuregulin 1 type-III expression hastens SMA motor axon maturation. Acta Neuropathol. Commun.11, 53 (2023).  [DOI](https://doi.org/10.1186/s40478-023-01551-8) | [PMC free article](/articles/PMC10061791/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36997967/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Kong,%20L.%20et%20al.%20Boosting%20neuregulin%201%20type-III%20expression%20hastens%20SMA%20motor%20axon%20maturation.%20Acta%20Neuropathol.%20Commun.11,%2053%20(2023).)

95. Gentile, J. E., Carrizales, M. G. & Koleske, A. J. Control of synapse structure and function by actin and its regulators. Cells11, 603 (2022).  [DOI](https://doi.org/10.3390/cells11040603) | [PMC free article](/articles/PMC8869895/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35203254/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Gentile,%20J.%20E.,%20Carrizales,%20M.%20G.%20&%20Koleske,%20A.%20J.%20Control%20of%20synapse%20structure%20and%20function%20by%20actin%20and%20its%20regulators.%20Cells11,%20603%20(2022).)

96. Charbonneau, N. L. et al. Fibrillins can co-assemble in fibrils, but fibrillin fibril composition displays cell-specific differences. J. Biol. Chem.278, 2740–2749 (2003).  [DOI](https://doi.org/10.1074/jbc.M209201200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12429739/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Charbonneau,%20N.%20L.%20et%20al.%20Fibrillins%20can%20co-assemble%20in%20fibrils,%20but%20fibrillin%20fibril%20composition%20displays%20cell-specific%20differences.%20J.%20Biol.%20Chem.278,%202740%E2%80%932749%20(2003).)

97. Peeters, S., De Kinderen, P., Meester, J. A., Verstraeten, A. & Loeys, B. L. The fibrillinopathies: new insights with focus on the paradigm of opposing phenotypes for both FBN1 and FBN2. Hum. Mutat.43, 815–831 (2022).  [DOI](https://doi.org/10.1002/humu.24383) | [PMC free article](/articles/PMC9322447/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35419902/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Peeters,%20S.,%20De%20Kinderen,%20P.,%20Meester,%20J.%20A.,%20Verstraeten,%20A.%20&%20Loeys,%20B.%20L.%20The%20fibrillinopathies:%20new%20insights%20with%20focus%20on%20the%20paradigm%20of%20opposing%20phenotypes%20for%20both%20FBN1%20and%20FBN2.%20Hum.%20Mutat.43,%20815%E2%80%93831%20(2022).)

98. Yoneyama, T. et al. Association of positional and functional candidate genes FGF1, FBN2, and LOX on 5q31 with intracranial aneurysm. J. Hum. Genet.48, 309–314 (2003).  [DOI](https://doi.org/10.1007/s10038-003-0030-6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12750963/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Yoneyama,%20T.%20et%20al.%20Association%20of%20positional%20and%20functional%20candidate%20genes%20FGF1,%20FBN2,%20and%20LOX%20on%205q31%20with%20intracranial%20aneurysm.%20J.%20Hum.%20Genet.48,%20309%E2%80%93314%20(2003).)

99. Ratnapriya, R. et al. Rare and common variants in extracellular matrix gene Fibrillin 2 (FBN2) are associated with macular degeneration. Hum. Mol. Genet.23, 5827–5837 (2014).  [DOI](https://doi.org/10.1093/hmg/ddu276) | [PMC free article](/articles/PMC4189898/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24899048/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Ratnapriya,%20R.%20et%20al.%20Rare%20and%20common%20variants%20in%20extracellular%20matrix%20gene%20Fibrillin%202%20(FBN2)%20are%20associated%20with%20macular%20degeneration.%20Hum.%20Mol.%20Genet.23,%205827%E2%80%935837%20(2014).)

100. Ramirez, D. M., Khvotchev, M., Trauterman, B. & Kavalali, E. T. Vti1a identifies a vesicle pool that preferentially recycles at rest and maintains spontaneous neurotransmission. Neuron73, 121–134 (2012).  [DOI](https://doi.org/10.1016/j.neuron.2011.10.034) | [PMC free article](/articles/PMC3259527/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22243751/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Ramirez,%20D.%20M.,%20Khvotchev,%20M.,%20Trauterman,%20B.%20&%20Kavalali,%20E.%20T.%20Vti1a%20identifies%20a%20vesicle%20pool%20that%20preferentially%20recycles%20at%20rest%20and%20maintains%20spontaneous%20neurotransmission.%20Neuron73,%20121%E2%80%93134%20(2012).)

101. Nelson, J. C., Stavoe, A. K. & Colón-Ramos, D. A. The actin cytoskeleton in presynaptic assembly. Cell Adhes. Migr.7, 379–387 (2013).  [DOI](https://doi.org/10.4161/cam.24803) | [PMC free article](/articles/PMC3739815/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23628914/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Nelson,%20J.%20C.,%20Stavoe,%20A.%20K.%20&%20Col%C3%B3n-Ramos,%20D.%20A.%20The%20actin%20cytoskeleton%20in%20presynaptic%20assembly.%20Cell%20Adhes.%20Migr.7,%20379%E2%80%93387%20(2013).)

102. Jalan-Sakrikar, N., Bartlett, R. K., Baucum, A. J. & Colbran, R. J. Substrate-selective and calcium-independent activation of CaMKII by α-actinin. J. Biol. Chem.287, 15275–15283 (2012).  [DOI](https://doi.org/10.1074/jbc.M112.351817) | [PMC free article](/articles/PMC3346149/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22427672/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Jalan-Sakrikar,%20N.,%20Bartlett,%20R.%20K.,%20Baucum,%20A.%20J.%20&%20Colbran,%20R.%20J.%20Substrate-selective%20and%20calcium-independent%20activation%20of%20CaMKII%20by%20%CE%B1-actinin.%20J.%20Biol.%20Chem.287,%2015275%E2%80%9315283%20(2012).)

103. Dhavan, R., Greer, P. L., Morabito, M. A., Orlando, L. R. & Tsai, L.-H. The cyclin-dependent kinase 5 activators p35 and p39 interact with the α-subunit of Ca2+/calmodulin-dependent protein kinase II and α-actinin-1 in a calcium-dependent manner. J. Neurosci.22, 7879–7891 (2002).  [DOI](https://doi.org/10.1523/JNEUROSCI.22-18-07879.2002) | [PMC free article](/articles/PMC6758084/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12223541/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Dhavan,%20R.,%20Greer,%20P.%20L.,%20Morabito,%20M.%20A.,%20Orlando,%20L.%20R.%20&%20Tsai,%20L.-H.%20The%20cyclin-dependent%20kinase%205%20activators%20p35%20and%20p39%20interact%20with%20the%20%CE%B1-subunit%20of%20Ca2+/calmodulin-dependent%20protein%20kinase%20II%20and%20%CE%B1-actinin-1%20in%20a%20calcium-dependent%20manner.%20J.%20Neurosci.22,%207879%E2%80%937891%20(2002).)

104. Roppongi, R. T., Karimi, B. & Siddiqui, T. J. Role of LRRTMs in synapse development and plasticity. Neurosci. Res.116, 18–28 (2017).  [DOI](https://doi.org/10.1016/j.neures.2016.10.003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27810425/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Roppongi,%20R.%20T.,%20Karimi,%20B.%20&%20Siddiqui,%20T.%20J.%20Role%20of%20LRRTMs%20in%20synapse%20development%20and%20plasticity.%20Neurosci.%20Res.116,%2018%E2%80%9328%20(2017).)

105. Siddiqui, T. J., Pancaroglu, R., Kang, Y., Rooyakkers, A. & Craig, A. M. LRRTMs and neuroligins bind neurexins with a differential code to cooperate in glutamate synapse development. J. Neurosci.30, 7495–7506 (2010).  [DOI](https://doi.org/10.1523/JNEUROSCI.0470-10.2010) | [PMC free article](/articles/PMC2896269/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20519524/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Siddiqui,%20T.%20J.,%20Pancaroglu,%20R.,%20Kang,%20Y.,%20Rooyakkers,%20A.%20&%20Craig,%20A.%20M.%20LRRTMs%20and%20neuroligins%20bind%20neurexins%20with%20a%20differential%20code%20to%20cooperate%20in%20glutamate%20synapse%20development.%20J.%20Neurosci.30,%207495%E2%80%937506%20(2010).)

106. Yamagata, A. et al. Structural insights into modulation and selectivity of transsynaptic neurexin–LRRTM interaction. Nat. Commun.9, 3964 (2018).  [DOI](https://doi.org/10.1038/s41467-018-06333-8) | [PMC free article](/articles/PMC6160412/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30262834/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Yamagata,%20A.%20et%20al.%20Structural%20insights%20into%20modulation%20and%20selectivity%20of%20transsynaptic%20neurexin%E2%80%93LRRTM%20interaction.%20Nat.%20Commun.9,%203964%20(2018).)
