# A frequent CYP2D6 variant promotes skipping of exon 3 and reduces CYP2D6 protein expression in human liver samples

## Metadata
**Authors:** Joseph M Collins, Hannah Lester, Samia Shabnaz, Danxin Wang
**Journal:** Frontiers in Pharmacology
**Date:** 2023 Jul 27
**DOI:** [10.3389/fphar.2023.1186540](https://doi.org/10.3389/fphar.2023.1186540)
**PMID:** 37576811
**PMCID:** PMC10412816
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10412816/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC10412816/pdf/fphar-14-1186540.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC10412816/pdf/fphar-14-1186540.pdf)

## Abstract

CYP2D6 is one of the most polymorphic drug-metabolizing enzymes in the liver. While genetic CYP2D6 variants serve as clinical biomarkers to predict CYP2D6 activity, large inter-person variability in CYP2D6 expression remains unaccounted for. Previous results suggest that there is variable expression of a CYP2D6 splice isoform with an in-frame deletion of exon 3 (CYP2D6ΔE3) encoding a protein lacking numerous active site residues. Here, using fragment analysis and RT-qPCR, we revealed that rs1058164 G (MAF = 27%–43%) is associated with increased formation of CYP2D6∆E3 in human liver samples (1.4–2.5-fold) and transfected cells. Furthermore, western blots showed that rs1058164 G was associated with a 50% decrease in full-length hepatic CYP2D6 protein expression. In addition, by studying a larger liver cohort, we confirmed our previous results that rs16947 (CYP2D6*2) reduces full-length CYP2D6 mRNA by increasing the production of an unstable splice isoform lacking exon 6 (CYP2D6ΔE6) and that the impact of CYP2D6ΔE6 is offset in carriers of the downstream enhancer variant rs5758550. The three frequent SNPs (rs1058164, rs16947, and rs5758550) form various 3-SNP-haplotypes, each with distinct CYP2D6 expression characteristics. Using an expression score (ES) system, we tested the impact of the 3-SNP-haplotype on improving the standard model to predict hepatic CYP2D6 protein expression based on genotype. A model that incorporates the 3-SNP-haplotype provided the best fit for CYP2D6 expression and also accounted for more variability in CYP2D6 protein levels (59%) than a model based on the accepted standard (36%) or one that only adds rs16947 and rs5758550 (42%). Clinical studies are needed to determine whether including the 3-SNP-haplotype alongside current standard CYP2D6 models improves the predictive value of CYP2D6 panels.

Keywords: cytochrome P450s, CYP2D6, single nucleotide polymorphisms, drug metabolism, genetic test, metabolic phenotype, splice variant, protein expression

## 1 Introduction

As one of the most important drug-metabolizing cytochrome P450 (CYP) enzymes in the liver CYP2D6 metabolizes ∼25% of currently used medications. CYP2D6 is highly polymorphic, and there are over 100 CYP2D6 star (*) alleles (PharmVar, [https://www.pharmvar.org](https://www.pharmvar.org)). In addition to CYP2D6 inhibitors, genetic variants are a primary cause of the large inter-person variability in CYP2D6 enzymatic activity ([Ingelman-Sundberg, 2005](#B20)) and profoundly impact the therapeutic outcomes of CYP2D6 substrate drugs ([Goetz et al., 2018](#B15); [Brown et al., 2019](#B6)). CYP2D6 genetic testing has become a useful tool for personalized drug therapy and the Clinical Pharmacogenomics Implementation Consortium (CPIC) has published guidelines for using CYP2D6 genetic information to guide therapy for many medications ([Crews et al., 2014](#B9); [Hicks et al., 2015](#B18); [Bell et al., 2017](#B4); [Hicks et al., 2017](#B19); [Goetz et al., 2018](#B15)).

Nine common star alleles (*1, *2, *3, *4, *9, *10, *17, *29, and *41) and gene copy number variations (CNVs) (e.g., gene deletion (*5), gene duplication and multiplications) are included in most CYP2D6 panels. An activity score (AS) system ([Gaedigk et al., 2008](#B14)) is commonly employed to predict the enzymatic activity of CYP2D6 based on star allele genotype according to the CPIC standards ([Crews et al., 2014](#B9); [Hicks et al., 2017](#B19)), which is then translated into CYP2D6 metabolizer phenotypes, including poor metabolizers, intermediate metabolizers, normal metabolizers, and ultra-rapid metabolizers ([Crews et al., 2014](#B9); [Hicks et al., 2017](#B19)). With the exception of poor metabolizers, the relationship between CYP2D6 genotype and phenotype remains ambiguous, with large variability within genotype/haplotype groups ([Pedersen et al., 2005](#B31); [Gaedigk et al., 2008](#B14); [Llerena et al., 2012](#B24); [Montané Jaime et al., 2013](#B26)), suggesting that the full complexity of genetic CYP2D6 variation remains uncertain. Numerous rare genetic changes have been identified in CYP2D6 through the extensive genome, exome, and long-read sequencing ([Nofziger et al., 2019](#B29)), but their rarity precludes their ability to explain population-level variation in CYP2D6. In particular, observed variability in the expression of CYP2D6 at both the mRNA and protein levels has not been fully characterized. Since CYP2D6 is not inducible by environmental factors ([Bock et al., 1994](#B5)), it appears that the likely cause for differences in CYP2D6 expression arises from cis-acting genetic factors, as opposed to trans-acting factors.

We previously identified two interacting variants that regulate the expression of CY2D6: an enhancer variant (rs5758550) that increases CYP2D6 transcription and an exonic variant rs16947 (core SNP of *2) that promotes an alternate and unstable splice isoform missing exon 6, which is out-of-frame and undergoes nonsense-mediated decay, thereby reducing expression of full-length CYP2D6 ([Wang et al., 2014](#B43); [Wang et al., 2015](#B42)). The combination of rs16947 and rs5758550 form alternate CYP2D6 haplotypes with varying enzymatic activities and occur at different frequencies in different populations ([Wang et al., 2014](#B43); [Ray et al., 2019](#B33); [Elias et al., 2020](#B12)). In a recent cross-ancestry GWAS, the compensatory effect of rs5758550 on the deleterious effects of rs16947 was confirmed for CYP2D6-dependent endoxifen plasma concentration ([Khor et al., 2023](#B22)). However, there are inconsistent results regarding the impact of rs5758550 and rs16947 on *in vivo* CYP2D6 protein expression/activity ([Sanchez-Spitman et al., 2017](#B36); [Ning et al., 2018](#B28); [Thomas et al., 2020](#B39); [Dinh et al., 2022](#B11); [Khor et al., 2023](#B22)). Furthermore, overall CYP2D6 mRNA levels do not seem to correlate well with protein levels ([Ning et al., 2018](#B28)), indicating that additional uncharacterized post-transcriptional events influence CYP2D6 expression. In this study, we identified a frequent synonymous variant rs1058164 that is associated with increased formation of a CYP2D6 splice isoform missing exon 3 (CYP2D6∆E3). Based on LD structure and different allele frequencies, rs5758550, rs16947, and rs1058164 form various 3-SNP haplotypes in different populations ([Machiela and Chanock, 2015](#B25)). Therefore, we evaluated their combined effects by considering the different 3-SNP haplotypes on full-length CYP2D6 expression in a liver sample cohort derived from both European American (EA) and African American (AA) donors (*n* = 244). Our results demonstrated that including all known regulatory SNPs better predicted full-length CYP2D6 protein expression in the liver samples.

## 2 Materials and methods

### 2.1 DNA and RNA preparation from liver tissue samples

Human liver samples (121 EA and 123 AA) were obtained from The Cooperative Human Tissue Network under an exempted protocol approved by the University of Florida IRB committee. Liver sample characteristics are published ([Collins and Wang, 2021a](#B7)). Biopsy liver tissues were immediately frozen in liquid nitrogen, shipped on dry ice, and stored at −80°C until use. Genomic DNA was prepared using DNeasy Blood & Tissue kits (Qiagen, United States). Total RNA was prepared from liver samples using direct-zol RNA miniprep plus kits (Zymos Research, CA, United States). RNA quality control and cDNA synthesis were performed as described ([Collins and Wang, 2021b](#B8)).

### 2.2 Genotyping

CYP2D6 SNPs were genotyped using PCR amplification of the entire CYP2D6 gene (∼5 kb) followed by multiplex primer extension assays (SnapShot assays), as described ([Sistonen et al., 2005](#B38)). Twelve SNPs (*10—rs1065852, *17—rs28371706, *6—rs5030655, *4—rs3892097, *3—rs35742686, *9—rs5030656, *2—rs16947, *39—rs1135840, *41—rs28371725, *29—rs59421388, rs5758550, and rs1058164) were genotyped (see [Supplementary Table S1](#s11) for primers and probes). Copy number variation (CNV) was genotyped using TaqMan copy number variation assays with two probes targeting intron 6 and exon 9 of CYP2D6 (CNV assay information is in [Supplementary Table S1](#s11)). Consistent results from both probes were required for determining CNV, as samples with inconsistent results may carry hybrid genes, which were subsequently excluded from further analysis. The genotype (or haplotype) of duplicated genes was assigned by comparing the allelic ratio of CNV samples to the average allelic ratio from samples with two copies of CYP2D6, similar to genotyping using allele quantification-based pyrosequencing, as described ([Langaee et al., 2015](#B23)).

### 2.3 CYP2D6 mRNA quantification

The expression levels of CYP2D6 mRNA and the two splice isoforms (CYP2D6∆E3 and CYP2D6∆E6) were measured with real-time qPCR using commercial (Thermo Fisher Scientific) or self-designed TaqMan assays ([Zanger et al., 2021](#B46)) or with SYBR Green using gene-specific primers (primers in [Supplementary Table S1](#s11)), as described ([Collins and Wang, 2021a](#B7)). Multiple commercial or self-designed TaqMan assays were tested; only those that were found to be robust and specific for CYP2D6 were used in this study ([Supplementary Table S1](#s11)). Measurements were conducted on a Quantabio Q real-time PCR instrument (VWR, PA, United States) with *β*-actin as an internal control. Our approach for measuring CYP2D6 mRNA involved three qPCR primer sets. Firstly, the total pool of CYP2D6 mRNA was measured using primers targeting exon 9, found in full-length CYP2D6, CYP2D6∆E3, and CYP2D6∆E6. Secondly, CYP2D6∆E3-isoform-specific primers were designed to measure any CYP2D6 mRNA isoform missing exon 3. Finally, qPCR primers spanning exons 6 and 7 were used to detect any CYP2D6 isoforms containing exon 6 but not CYP2D6∆E6. The amount of the CYP2D6∆E3 isoform was also measured using fragment analysis, as described ([Wang et al., 2014](#B43)).

### 2.4 CYP2D6 protein quantification

Liver lysate preparation (*n* = 189) and CYP2D6 protein quantification using the capillary Western blotting Jess system (ProteinSimple Inc., CA, United States) were conducted as described ([Collins and Wang, 2021a](#B7)). CYP2D6 protein was detected with a rabbit anti-CYP2D6 antibody (1:10, Novus NBP2-67020) followed by an HRP-conjugated anti-rabbit secondary antibody (Biotechne, San Jose, CA, United States). Full-length CYP2D6 protein expression levels were normalized by *β*-actin detected in the same capillary by a mouse anti-β-actin antibody (1:25, Biotechne) followed by a NIR-conjugated anti-mouse secondary antibody (Biotechne, San Jose, CA, United States). The CYP2D6 antibody used in this study is highly specific to human CYP2D6; only a single 53 kD (full-length CYP2D6) or 48 kD (CYP2D6∆E3) was detectable in CYP2D6 cDNA transfected HEK cells, but not in mock-transfected cells nor liver samples with homozygous null allele genotypes (*4/*4 or *5/*3) ([Supplementary Table S1](#s11)). The CYP2D6∆E6 mRNA has a shifted open reading frame and is targeted for nonsense-mediated decay. It is unknown whether CYP2D6∆E6 encodes a protein, although the resulting protein would not be detected by the CYP2D6 antibody used here since the CYP2D6 antibody targets protein residues 301–497 of CYP2D6, which are encoded by 3′portion of exon 6 and the downstream exons, which would be missing from the predicted CYP2D6∆E6 protein (a c-terminal truncated protein of 289 aa) ([Wang et al., 2014](#B43)).

### 2.5 CYP2D6 expression plasmids

Expression plasmids containing CYP2D6 cDNA corresponding to full-length ([NM_000106](https://www.ncbi.nlm.nih.gov/nuccore/NM_000106), catalog #SC104446) or CYP2D6∆E3 ([NM_001025161](https://www.ncbi.nlm.nih.gov/nuccore/NM_001025161), catalog #SC302357) were purchased from Origen (Rockville, MD, United States). The entire CYP2D6 gene (5.1 kb including introns) was PCR amplified from liver gDNA samples and cloned into the pcDNA3 vector using the Infusion cloning kit (Clontech), as described ([Wang et al., 2014](#B43)). Eight clones harboring either rs1065852 G or C alleles (four for each) were selected, representing either the CYP2D6*1 or CYP2D5*10 haplotype, respectively. Haplotypes containing rs16947 were avoided since rs16947 reduces total CPY2D6 expression ([Wang et al., 2014](#B43)), which would have potentially interfered with comparing CYP2D6∆E3 to the total CYP2D6 transcript pool. All constructs were sequenced to ensure sequence fidelity.

### 2.6 Cell culture and transfection

Cells were cultured at 5% CO2/37°C in a humidified incubator in DMEM (HepG2) or DMEM/F12 (HEK293) supplemented with 10% fetal bovine serum, 100 U/mL penicillin, and 100 μg/mL streptomycin. Cells were plated into 12-well plates the day before transfection. Transfection of plasmid DNA was performed using lipofectamine 3,000 (Invitrogen Life Technologies, CA, United States) according to the manufacturer’s instructions. Cells were harvested 48 h after transfection for total RNA preparation or lysed with RIPA buffer for Western blotting. In the CYP2D6 genomic DNA transfection experiments, mRNA was purified from total RNA using the PolyA Tract mRNA isolation system (Promega) to avoid contamination of plasmid DNA. To determine the mRNA stability of the CYP2D6 splice isoforms, HepG2 cells (expressing full-length CYP2D6, CYP2D6∆E3, or CYP2D6∆E6) were treated with actinomycin D (20 μg/mL) for different time periods before harvesting. To test protein stability, CYP2D6, and CYP2D6∆E3 transfected HEK cells were treated with 10 μg/mL cycloheximide for 6 h before harvesting.

### 2.7 Data analysis

After the log10 transformation, the expression levels of all CYP2D6 transcripts followed a normal distribution. *β*-actin normalized full-length CYP2D6 protein levels were normally distributed without transformation. A multiple linear regression model was used to test the association between genotype and the expression levels of the CYP2D6 splice isoforms using the Minitab software (version 21). Forward and backward stepwise regression were used to select the best set of predictors to include in the multiple linear regression models with a cutoff *p*-value < 0.05. The tested candidate predictors were: sex, race, age, CYP2D6 genotypes, and several transcription factors (TFs) known to affect CYP2D6 expression ([Collins and Wang, 2021b](#B8)). To test the association between enhancer SNP rs5758550 or *41 SNP rs28371725 with the overall CYP2D6 mRNA level, race, and the expression levels of several TFs (RXRA, NR1I3, HNF4A, ESR1, FOXA2, and ARNT) were included as covariates. For analysis of the association between rs1058164 and CYP2D6∆E3, age, sex, race, and TFs were not significant, and therefore none were included, while CYP2D6*2 SNP rs16947 and CYP2D6*4 SNP were significant, and were thus included in the model. Age, sex, race, and TFs were not included in the CYP2D6 protein analysis since none of them were significant. A simple linear regression model also tested associations between genotype-predicted expression score (ES) and CYP2D6 protein levels. The goodness-of-fit was judged by residual and normal quartile plots. The Student’s t-test was used to compare the means between two groups and ANOVA for more than two groups.

## 3 Results

### 3.1 CYP2D6 genotype and haplotype frequency in the liver cohort

The allele frequencies of the twelve common CYP2D6 SNPs were similar to those reported in the 1,000 genome database ([1000 Genomes Project Consortium et al., 2012](#B2)) ([Supplementary Table S2](#s11)), with the *10, *6, *9 alleles being more prevalent in EA and *17 and *29 being more prevalent in AA. Gene deletion frequencies were similar between EA and AA (∼5%), while CNVs with >2 copies were more prevalent in AA (20%) than in EA (6%) ([Supplementary Table S3](#s11)). The frequencies of CYP2D6 diplotype in EA and AA are shown in [Table 1](#T1).

### TABLE 1.

| Diplotype | AA+ EA |  | AA |  | EA |  |
| --- | --- | --- | --- | --- | --- | --- |
| Count | % | Count | % | Count | % |  |
| *1/*1 | 25 | 10.25 | 5 | 4.07 | 20 | 16.53 |
| *1/*2 | 42 | 17.21 | 19 | 15.45 | 23 | 19.01 |
| *1/*4 | 13 | 5.33 | 4 | 3.25 | 9 | 7.44 |
| *1/*5 | 3 | 1.23 | 3 | 2.44 | 0 | 0.00 |
| *1/*6 | 1 | 0.41 | 1 | 0.81 | 0 | 0.00 |
| *1/*9 | 1 | 0.41 | 1 | 0.81 | 0 | 0.00 |
| *1/*10 | 5 | 2.05 | 2 | 1.63 | 3 | 2.48 |
| *1/*17 | 12 | 4.92 | 10 | 8.13 | 2 | 1.65 |
| *1/*29 | 5 | 2.05 | 4 | 3.25 | 1 | 0.83 |
| *1/*41 | 11 | 4.51 | 8 | 6.50 | 3 | 2.48 |
| *2/*2 | 12 | 4.92 | 4 | 3.25 | 8 | 6.61 |
| *2/*3 | 11 | 4.51 | 2 | 1.63 | 9 | 7.44 |
| *2/*5 | 3 | 1.23 | 2 | 1.63 | 1 | 0.83 |
| *2/*9 | 1 | 0.41 | 0 | 0.00 | 1 | 0.83 |
| *2/*10 | 6 | 2.46 | 4 | 3.25 | 2 | 1.65 |
| *2/*17 | 3 | 1.23 | 2 | 1.63 | 1 | 0.83 |
| *2/*29 | 5 | 2.05 | 3 | 2.44 | 2 | 1.65 |
| *2/*41 | 7 | 2.87 | 2 | 1.63 | 5 | 4.13 |
| *3/*9 | 1 | 0.41 | 0 | 0.00 | 1 | 0.83 |
| *4/*10 | 3 | 1.23 | 1 | 0.81 | 2 | 1.65 |
| *4/*17 | 6 | 2.46 | 0 | 0.00 | 6 | 4.96 |
| *4/*4 | 2 | 0.82 | 0 | 0.00 | 2 | 1.65 |
| *4/*5 | 1 | 0.41 | 0 | 0.00 | 1 | 0.83 |
| *4/*6 | 1 | 0.41 | 0 | 0.00 | 1 | 0.83 |
| *4/*9 | 5 | 2.05 | 5 | 4.07 | 0 | 0.00 |
| *4/*29 | 5 | 2.05 | 5 | 4.07 | 0 | 0.00 |
| *4/*41 | 1 | 0.41 | 1 | 0.81 | 0 | 0.00 |
| *5/*17 | 1 | 0.41 | 1 | 0.81 | 0 | 0.00 |
| *5/*41 | 2 | 0.82 | 1 | 0.81 | 1 | 0.83 |
| *6/*10 | 1 | 0.41 | 0 | 0.00 | 1 | 0.83 |
| *6/*29 | 1 | 0.41 | 1 | 0.81 | 0 | 0.00 |
| *9/*41 | 1 | 0.41 | 0 | 0.00 | 1 | 0.83 |
| *10/*10 | 1 | 0.41 | 0 | 0.00 | 1 | 0.83 |
| *10/*17 | 3 | 1.23 | 2 | 1.63 | 1 | 0.83 |
| *10/*41 | 2 | 0.82 | 0 | 0.00 | 2 | 1.65 |
| *17/*17 | 2 | 0.82 | 1 | 0.81 | 1 | 0.83 |
| *17/*29 | 3 | 1.23 | 3 | 2.44 | 0 | 0.00 |
| *17/*41 | 1 | 0.41 | 1 | 0.81 | 0 | 0.00 |
| *29/*29 | 3 | 1.23 | 2 | 1.63 | 1 | 0.83 |
| *29/*41 | 1 | 0.41 | 1 | 0.81 | 0 | 0.00 |
| *41/*41 | 3 | 1.23 | 1 | 0.81 | 2 | 1.65 |
| *1/*1 × 2 | 3 | 1.23 | 2 | 1.63 | 1 | 0.83 |
| *1/*2 × 2 | 1 | 0.41 | 1 | 0.81 | 0 | 0.00 |
| *1/*4 × 2 | 3 | 1.23 | 3 | 2.44 | 0 | 0.00 |
| *1/*17 × 2 | 1 | 0.41 | 1 | 0.81 | 0 | 0.00 |
| *1/*29 × 2 | 3 | 1.23 | 3 | 2.44 | 0 | 0.00 |
| *2/*4 × 2 | 2 | 0.82 | 1 | 0.81 | 1 | 0.83 |
| *2/*1 × 2 | 1 | 0.41 | 1 | 0.81 | 0 | 0.00 |
| *2 × 2/*2 × 2 | 2 | 0.82 | 2 | 1.63 | 0 | 0.00 |
| *2 × 2/*4 × 3 | 1 | 0.41 | 1 | 0.81 | 0 | 0.00 |
| *4/*1 × 2 | 1 | 0.41 | 1 | 0.81 | 0 | 0.00 |
| *4/*4 × 2 | 1 | 0.41 | 1 | 0.81 | 0 | 0.00 |
| *4/*29 × 2 | 1 | 0.41 | 1 | 0.81 | 0 | 0.00 |
| *9/*29 × 2 | 1 | 0.41 | 0 | 0.00 | 1 | 0.83 |
| *10/*2 × 2 | 1 | 0.41 | 1 | 0.81 | 0 | 0.00 |
| *10/*4 × 2 | 2 | 0.82 | 1 | 0.81 | 1 | 0.83 |
| *29/*2 × 2 | 1 | 0.41 | 0 | 0.00 | 1 | 0.83 |
| *41/*4 × 2 | 1 | 0.41 | 1 | 0.81 | 0 | 0.00 |
| *2 × 2/*4 × 2 | 1 | 0.41 | 0 | 0.00 | 1 | 0.83 |
| *1 × 4/*2 × 5 | 1 | 0.41 | 0 | 0.00 | 1 | 0.83 |

Table 1 Caption: Frequency of CYP2D6 diplotype in liver samples.

### 3.2 A commonly-occurring exon 3 SNP rs1058164 is associated with increased expression of a CYP2D6 splice isoform lacking exon 3

There are two common alternative splice isoforms for CYP2D6, one lacking exon 6 (CYP2D6∆E6) and another lacking exon 3 (CYP2D6∆E3). A SNP in exon 6 (rs16947, *2) increases the production of CYP2D6∆E6 ([Wang et al., 2014](#B43)). To determine whether any SNPs may also control the expression of CYP2D6∆E3, we used fragment analysis to quantify the fraction of CYP2D6∆E3 isoforms in the total CYP2D6 transcript pool for each sample, as described ([Wang et al., 2014](#B43)). The average fraction of CYP2D6∆E3 in liver samples was 21% (ranging from 1%–48%, with very low levels in CYP2D6*4 carriers), similar to a previous report ([Sangar et al., 2010](#B37)). Using this data, we then tested for association between genotype and the production of CYP2D6∆E3. In a multiple linear regression model that included several SNPs (rs1058164, rs3892097, and rs16947), the most significantly associated SNP with CYP2D6∆E3 was rs1058164 (1662 G >C), with each G allele being associated with 1.48-fold increased CYP2D6∆E3 (*p* = 0.001) ([Table 2](#T2)). To validate these results, we then used real-time qPCR to specifically quantify CYP2D6∆E3. The expression of CYP2D6∆E3 was normalized by total CYP2D6 expression (CYP2D6∆E3/total CYP2D6) to control for overall variable transcription of CYP2D6 between samples. Again, rs1058164 G was associated with 2.5-fold increased CYP2D6∆E3 (*p* = 4.7 × 10^−4^), and the median levels of CYP2D6∆E3 for the various genotype were CC (0.68), CG (1.18) and GG (2.02) ([Table 2](#T2); [Figure 1A](#F1)), after adjusting for rs3892097 and rs16947. Furthermore, when CYP2D6 genomic DNA is expressed in HEK293 T cells, cells carrying the rs1058164 G allele produced more CYP2D6∆E3 than those transfected with the C allele (*p* < 0.0001) ([Figure 1B](#F1)).

### TABLE 2.

| SNP ID (independent variable) | Dependent variable | Coefficient | p-Value |
| --- | --- | --- | --- |
| Fragment analysis |  |  |  |
| rs1058164 G | log (CYP2D6∆E3) | 0.1708 | 0.001 |
| rs3892097 (*4) | log (CYP2D6∆E3) | −0.2849 | 0.003 |
| rs16947 (*2) | log (CYP2D6∆E3) | 0.1552 | 0.003 |
| Real-time qPCR - CYP2D6∆E3 |  |  |  |
| rs1058164 G | log (CYP2D6∆E3/total) | 0.419 | 4.7E-4 |
| rs3892097 (*4) | log (CYP2D6∆E3/total) | −0.133 | 0.187 |
| rs16947 (*2) | log (CYP2D6∆E3/total) | 0.332 | 0.004 |
| Real-time qPCR - CYP2D6∆E6 |  |  |  |
| rs16947 (*2) a | Log (CYP2D6 with E6/total) | −0.1379 | 0.055 |
| rs28371725 (*41) a | Log (CYP2D6 with E6/total) | −0.069 | 0.661 |
| Real-time qPCR - CYP2D6 |  |  |  |
| rs5758550 b | log (CYP2D6 total) | 0.0987 | 0.009 |
| rs28371725 (*41) b | log (CYP2D6 total) | −0.047 | 0.571 |

Table 2 Caption: Associations between CYP2D6 genotype and levels of CYP2D6∆E3, E6-containing CYP2D6, and total CYP2D6 mRNA in liver samples using multiple linear regression. Only significant variables were included in the regression models after screening, as described in the method section. The p-value for each variable is shown, and the coefficient is in a log10 scale.

### FIGURE 1.

![FIGURE 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8cd9/10412816/b2a419625ccd/fphar-14-1186540-g001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=10412816_fphar-14-1186540-g001.jpg)

The frequent synonymous SNP rs1058164 G is associated with higher levels of the CYP2D6 splice isoform missing exon 3 (CYP2D6∆E3). (A) CYP2D6∆E3 (log) levels in liver samples grouped by rs1058164 genotypes, adjusted by rs16947 and rs3892097 genotypes. The differences between groups were analyzed with multiple linear regression. The box and horizontal lines show the 25th and 75th percentiles surrounding the mean, and the whiskers show values up to 1.5*IQR. (B) The expression levels of CYP2D6∆E3 in HEK293 T cells after transfection with CYP2D6 genomic DNA expression plasmids containing rs1085164 G or C alleles. Data are combined results from different clones (4 for each genotype) and two independent experiments with triplicates. The horizontal bar indicates the average (mean).

We also re-evaluated the effects of the enhancer SNP rs5758550, rs16947 (*2), and rs28371725 (*41) on the expression of CYP2D6 and its splice isoforms. As expected, rs5758550 is associated with increased mRNA expression of total CYP2D6 (1.25-fold, *p* = 0.009, [Table 2](#T2)) after adjusting for the expression of several TFs known to regulate CYP2D6 ([Collins and Wang, 2021a](#B7)). To test the impact of rs16947 on skipping of exon 6, we first attempted to use a custom TaqMan probe that specifically quantifies CYP2D6∆E6 ([Zanger et al., 2021](#B46)), but the assay generated very low amplification efficiency and inconsistent results. Instead, we used a custom exon 6/exon 7-spanning TaqMan assay ([Zanger et al., 2021](#B46)) to specifically quantify CYP2D6 isoforms that contain exon 6. Again, this was normalized by total CYP2D6 expression (CYP2D6 with E6/total CYP2D6). Consistent with our previous results ([Wang et al., 2014](#B43)), rs16947 was associated with decreased expression of CYP2D6 isoforms containing exon 6 (0.87-fold, *p* = 0.055, [Table 2](#T2)), supporting that rs16947 promotes skipping of exon 6, although not quite reaching statistical significance. Conversely, rs28371725 (*41) was not associated with either the overall expression of CYP2D6 or specifically with CYP2D6 isoforms containing exon 6 ([Table 2](#T2)), consistent with our previous study ([Wang et al., 2014](#B43)).

### 3.3 The CYP2D6∆E3 protein is less stable than full-length CYP2D6

CYP2D6∆E6 isoforms have a shifted reading frame, causing decreased CYP2D6 mRNA expression due to nonsense-mediated RNA decay ([Wang et al., 2014](#B43)). In contrast, CYP2D6∆E3 isoforms contain an in-frame deletion that lacks 153 bp. We tested whether the CYP2D6∆E3 mRNA is stable by treating HepG2 cells with actinomycin D, which blocks transcription and enables measurement of mRNA longevity. Unlike CYP2D6∆E6, CYP2D6∆E3 was as stable as full-length CYP2D6 ([Figure 2A](#F2)) and therefore did not decrease the total CYP2D6 transcript pool. However, at the protein level, CYP2D6∆E3 appears to be less stable than full-length CYP2D6. In transfected HEK293 T cells, CYP2D6∆E3 protein levels decreased by 53% after treatment with the protein synthesis inhibitor cycloheximide for 6 hours, compared to only a 33% decrease in full-length CYP2D6 ([Figure 2B](#F2)). However, as reported previously ([Sangar et al., 2010](#B37)), we only observed a singular band corresponding to full-length CYP2D6 in liver samples ([Supplementary Figure S1](#s11)), suggesting that the CYP2D6∆E3 protein is highly unstable *in vivo*.

### FIGURE 2.

![FIGURE 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8cd9/10412816/f28eacb6514b/fphar-14-1186540-g002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=10412816_fphar-14-1186540-g002.jpg)

Testing mRNA and protein stabilities of full-length CYP2D6 and its splice isoforms. (A) Time course of the relative expression levels of full-length CYP2D6, CYP2D6∆E3, and CYP2D6∆E6 in HepG2 cells after actinomycin D treatment. (B) Western blot image of CYP2D6 protein isoforms in HEK293 T cells transfected with plasmid DNAs expressing full-length (HEK2D6F), CYP2D6∆E3 (HEK2D6S), a mix of the two (mix-2D6F/2D6S), or a mix of the two and treated with cycloheximide (Cyclo-mix-2D6F/2D6S).

### 3.4 Liver samples with rs1058164 have decreased full-length CYP2D6 protein expression

We next tested for the relationship between the genotype of the aforementioned regulatory variants (rs5758550, rs16947, *41, and rs1058164) and full-length CYP2D6 protein levels in liver samples (*n* = 189) using a simple linear regression model. The overall correlation between total CYP2D6 mRNA and full-length protein is weak (r = 0.356, *p* = 1.11 × 10^−6^), consistent with previous report ([Ning et al., 2018](#B28)). To avoid confounding by CNVs and complex haplotype structure of null alleles, 81 samples carrying CNVs and/or null alleles (*3, *4, *6) were excluded in these analyses, leaving an effective sample size of 108. While *41 was associated with decreased full-length CYP2D6 protein, it did not reach statistical significance (*p* = 0.082), in agreement with its lack of association with total CYP2D6 mRNA levels. Consistent with its effects on total CYP2D6 mRNA expression, rs5758550 was associated with increased full-length CYP2D6 protein (*p* = 1.09 × 10^−12^) ([Table 3](#T3)). rs16947 was also associated with increased full-length CYP2D6 protein in simple linear regression analysis, however, we expected that this was due to its high LD with rs5758550. Indeed, after adjusting for the rs5758550 genotype, rs16947 was instead associated with reduced full-length CYP2D6 protein expression (*p* = 9.14 × 10^−4^) ([Table 3](#T3)), as expected.

### TABLE 3.

| SNP ID | Coefficient | p-Value |
| --- | --- | --- |
| Simple linear regression |  |  |
| rs5758550 | 65.11 | 1.09E-12 |
| rs16947 | 28.8 | 0.006 |
| rs28371725 (*41) | −25.4 | 0.082 |
| rs1058164 G | −42.1 | 5.97E-5 |
| Multiple linear regression |  |  |
| rs1058164 G | −41.8 | 6.08E-4 |
| rs5758550 | 72.3 | 5.66E-12 |
| rs16947 (*2) | −44.5 | 9.14E-4 |

Table 3 Caption: Association between CYP2D6 genotype and full-length CYP2D6 protein level in liver samples using simple or multiple linear regression.

The rs1058164 G allele was associated with decreased full-length CYP2D6 protein level in simple linear regression analysis (*p* = 5.97 × 10^−5^). A similar association (*p* = 6.08 × 10^−4^) was seen in a multiple linear regression analysis after adjusting for rs5758550 and rs16947 ([Figure 3A](#F3); [Table 3](#T3)), with each G allele being associated with a ∼50% reduction in full-length CYP2D6 protein. Notably, since the CYP2D6*1 haplotype contains rs1058164 G and most of the CYP2D6*2 haplotypes contain rs1058164 C, the CYP2D6*1 haplotype would be expected to have lower protein than *2 haplotypes that also contain rs5758550. To test this, we specifically looked at protein expression in samples with only one functional copy of CYP2D6 (i.e., samples were *1/*null or *2/*null) so that full-length CYP2D6 protein expression would be solely driven by either a single *1 or *2 allele. In this subset (*n* = 29), samples with CYP2D6*1 had significantly lower full-length CYP2D6 protein levels than samples with rs16947 and rs5758550 (78 ± 28 vs. 120 ± 42, *p* = 0.004) ([Figure 3B](#F3)).

### FIGURE 3.

![FIGURE 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8cd9/10412816/5be9ac7229df/fphar-14-1186540-g003.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=10412816_fphar-14-1186540-g003.jpg)

Comparison of full-length CYP2D6 protein expression in liver samples between different genotypes. (A) Full-length CYP2D6 protein levels in liver samples (adjusted by rs5758550 and rs16947 genotypes) grouped by rs1058164 genotype. (B) Full-length CYP2D6 protein expression in CYP2D6*1 samples (carrying rs1058164 G) and CYP2D6*2 samples (carrying rs16947, rs1058164 C, and rs5758550). Only samples with a single copy of a functional CYP2D6 gene were analyzed. In both plots, the box and horizontal lines show the 25th and 75th percentiles surrounding the mean, and the whiskers show values up to 1.5*IQR.

### 3.5 The impact of CNVs on CYP2D6 protein expression

According to the accepted standards, CYP2D6 gene duplication is thought to double CYP2D6 expression and activity if the duplicated genes are either *1 or *2 ([Crews et al., 2014](#B9); [Hicks et al., 2017](#B19)). However, duplication of the CYP2D6 gene does not include duplication of the distal enhancer located >100 kb downstream of the CYP2D6 promoter, which is critical for CYP2D6 expression ([Wang et al., 2014](#B43)). Therefore, the duplicated CYP2D6 gene may not be fully expressed. We then tested the relationship between CYP2D6 CNVs and full-length CYP2D6 protein expression in the liver samples. Samples were excluded if they had duplicated null alleles (*3, *4, *6) and *41 (considered to be a reduced expression allele ([Crews et al., 2014](#B9); [Hicks et al., 2017](#B19))) or if copy number >4 (due to difficulties in assigning haplotypes for each copy). Samples with duplication of amino acid-changing alleles (*9, *10, *17, and *29) were included since there is no direct evidence showing that these SNPs alter full-length CYP2D6 protein expression. Overall, there is a good correlation between CYP2D6 CNV and full-length CYP2D6 protein (r = 0.622, *p* = 1.0 × 10^−16^) ([Figure 4](#F4)). However, the correlation is mainly driven by samples with CNV ≤ 2 (0, 1 and 2), as the correlation was relatively unchanged (r = 0.646, *p* = 2.0 × 10^−16^) after omission of samples with CNV 3 or 4 and the correlation disappeared (r = 0.165, *p* = 0.105) when samples with CNV 0 and 1 were removed. Also, the comparison of mean CYP2D6 protein levels between CNV groups showed a significant increase between 0 vs. 1 copies and 1 vs. 2 copies, but not with additional CNVs: 2 vs. 3 (*p* = 0.84), 3 vs. 4 (*p* = 0.65), and 2 vs. 4 (*p* = 0.31) ([Figure 4](#F4)).

### FIGURE 4.

![FIGURE 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8cd9/10412816/c852917c1f16/fphar-14-1186540-g004.jpg)

Relationship between CYP2D6 CNV and full-length CYP2D6 protein expression in liver samples. Samples with duplicated null alleles (*3, *4, *6) and *41 CNV >4 were excluded.

### 3.6 A model incorporating all three regulatory variants better predicts CYP2D6 protein expression

We then generated a modified version of the AS system ([Gaedigk et al., 2008](#B14)) to predict the full-length CYP2D6 protein expression of each haplotype based on the combined effects of all regulatory variants and CNVs, which we labeled as an expression score (ES) system. We tested three different models for ES assignments ([Supplementary Table S4](#s11)).

Model 1 was based on the accepted standard ([Hicks et al., 2017](#B19)), with ES = 0 for null alleles (*3, *4, *5, *6), ES = 0.5 for the reduced expression allele *41, and ES = 1 for *1, *2, and *39. Star alleles with nonsynonymous changes (*9, *10, *17, and *29) were also assigned ES = 1 as there is no evidence that these SNPs change the expression of full-length CYP2D6 protein. For CNV, ES = (number of CYP2D6 copies) × (ES of corresponding alleles).

Model 2 is a modified version of model 1 that also includes rs5758550 and rs16947 and is based on previous reports ([Wang et al., 2014](#B43); [Ray et al., 2019](#B33)). Haplotypes with both rs5758550 and rs16947 were assigned ES = 1, haplotypes containing only rs16947 and not rs5758550 were assigned ES = 0.5, and haplotypes containing rs5758550 without rs16947 were assigned ES = 2. This alteration to model 1 applies to all rs16947-containing alleles, such as *17, *29, and *41. For CNV, each additional normal allele only increases the ES by 1.5 instead of two, as in model 1. For example, duplication of normally expressing alleles (e.g., *1) was assigned ES = 1.5 (instead of 2), and duplication of a reduced expression allele (i.e., rs16947 without rs5758550) had an ES = 0.75 (ES = 0.5 + 0.25). Model 3 further expanded on model 2 to also include rs1058164, with a 50% reduction in ES score for each rs1058164 G allele.

As shown in both [Table 4](#T4) and [Figure 5](#F5), in all three models, ES was significantly associated with measured full-length CYP2D6 protein level (*p* < 10^–13^). Consistent with previous reports ([Wang et al., 2014](#B43); [Ray et al., 2019](#B33)), the addition of rs16947 and rs575850 in model 2 improved the linearized relationship between ES and CYP2D6 protein level and explained 14.18% more variability in full-length CYP2D6 protein expression than did model 1. Model 3, with its inclusion of rs1058164, had the best-linearized relationship between ES and measured full-length CYP2D6 protein and could further explain an additional 17.27% variability in full-length CYP2D6 protein expression (R^2^ = 68.09%). Model 3 also performed best when the two populations were analyzed separately ([Table 4](#T4)). Notably, improvements from model 1 to model 2 were greater for AA than EA (differences in R^2^, 22.5% vs. 6.55%), while the improvement from model 2 to model 3 was greater in EA than AA (differences in R^2^, 27.58% vs. 3.73%).

### TABLE 4.

| Model | AA+ EA |  |  | AA only |  |  | EA only |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Coefficient | p-value | R 2 (%) | Coefficient | p-value | R 2 (%) | Coefficient | p-value | R 2 (%) |  |
| Model 1 | 57.94 | < 10−13 | 36.64 | 55.61 | 1E-11 | 40.08 | 62.75 | 4.27E-11 | 35.62 |
| Model 2 | 77.44 | < 10−13 | 50.82 | 81.61 | < 10−13 | 62.58 | 72.82 | 2.15E-13 | 42.17 |
| Model 3 | 106.15 | < 10−13 | 68.09 | 98.63 | < 10−13 | 66.31 | 112.78 | < 10−13 | 69.75 |

Table 4 Caption: Association between full-length CYP2D6 protein level and predicted expression scores (ES) based on the three models using simple linear regression.

### FIGURE 5.

![FIGURE 5](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8cd9/10412816/de2f31508fb4/fphar-14-1186540-g005.jpg)

Relationship between full-length CYP2D6 protein expression and expression score derived from genotype. (A) CYP2D6 protein level as predicted by ES using Model 1. (B) ES-based prediction of CYP2D6 protein via model 2. (C) Model 3 ES-based prediction of CYP2D6 protein. In all three plots, open circles represent samples that did not move ES values compared to model 1, while closed squares are samples which moved to new ES values based on the particular model.

## 4 Discussion

We identified a frequent synonymous SNP rs1058164 as a novel regulator for a CYP2D6 alternative splice isoform CYP2D6∆E3. Our results showed that the rs1058164 G allele was associated with increased CYP2D6∆E3 production and decreased full-length CYP2D6 protein expression. Since all CYP2D6*1 haplotypes contain rs1058164 G, this result indicates that the expression or activity of CYP2D6*1 may currently be over-estimated. Using a liver sample cohort, we also re-evaluated the effects of the enhancer SNP rs5758550 and the *2 core SNP rs16947 on the expression of CYP2D6 mRNA and protein levels and found results consistent with our previous report ([Wang et al., 2014](#B43)). Moreover, using an ES system to predict full-length CYP2D6 protein expression from genotypes, we demonstrated that two ES-modified models (model 2 and model 3) that further incorporate CYP2D6 regulatory SNPs better predicted full-length CYP2D6 protein expression than the currently accepted standard (model 1) ([Crews et al., 2014](#B9); [Hicks et al., 2017](#B19)). Taken together, our results indicate that various 3-SNP-haplotypes (formed by rs5758550, rs16947, and rs1058164) explain a significant portion of inter-person variability in the expression of CYP2D6 protein.

Alternative splicing is an essential mechanism in the regulation of gene expression and protein diversity across eukaryotes. Splicing is regulated by a complex network of trans-acting factors and cis-acting genetic elements ([Wang et al., 2015](#B44)). It has been suggested that genetic alterations to splicing are one of the primary mechanisms underlying genetic associations ([GTEx Consortium, 2020](#B16)), and the most prevalent type of alternative splicing arising from genetic variation is exon skipping, with the majority of the causal SNPs occurring within the skipped exons ([Amoah et al., 2021](#B3)). The two CYP2D6 SNPs (rs16947 and rs1058164) described in this report are also located within their respective exons and may potentially be driving the exon skipping. Embedded within CYP2D6*1, rs1058164 G is a minor allele with a frequency ranging from 0.27 to 0.43 in different populations, while the major allele C is on >70 star alleles (PharmVar), including several common ones (*10, *17, *29). With the exception of one *2 sub-haplotype (CYP2D6*2.004), most of the rs16947-variant T containing alleles also have rs1058164 C (PharmVar), indicating that rs1058164 G and rs16947 T rarely co-occur. Indeed, we only found seven rs16947 T (3% of all rs16947 containing alleles) haplotypes that also had rs1058164 G in our liver samples. Since skipping of both exon 3 and exon 6 reduced the expression of full-length CYP2D6 protein, it is possible that the apparent mutual exclusivity of rs16947 T and rs1058164 G may be due to a need to balance the expression/function of CYP2D6 proteins.

Based on protein structure ([Rowland et al., 2006](#B35)), the CYP2D6∆E3 isoform lacks numerous residues that form the active site cavity (residues 118–122) or are involved in heme binding (Trp-128 and Arg-132), likely causing it to have severely reduced or no enzymatic activity. While CYP2D6∆E3 has previously been detected at the mRNA level by us and others ([Zhuge and Yu, 2004](#B47); [Sangar et al., 2010](#B37); [Wang et al., 2014](#B43)), to our knowledge, it has not been detected at the protein level *in vivo*. As CYP enzymes form protein complexes that can directly ([Reed et al., 2012](#B34)) or indirectly ([Kandel and Lampe, 2014](#B21)) impact the enzymatic activity of other CYPs, it is possible that CYP2D6∆E3 isoforms are actively targeted for degradation to limit their potential deleterious effect on the other CYP enzymes.

Although considered as a reference or wildtype allele, our results indicated that CYP2D6*1 harbors a variant associated with reduced full-length CYP2D6 protein expression, below even CYP2D6*2 alleles containing the enhancer SNP rs5758550 ([Figure 3B](#F3)). These results are consistent with a previous report showing lower CYP2D6 protein levels in liver samples carrying *1 than *2A, a *2 allele with a promoter SNP rs1080985 (that is in high LD with rs5758550) ([Zanger et al., 2001](#B45)). Our results also agree with a recent GWAS study showing that rs1058164 C is associated with increased levels of CYP2D6 protein in liver samples and CYP2D6-dependent endoxifen ([Khor et al., 2023](#B22)). In our liver cohort, the frequency of the CYP2D6*1 haplotype was 31%, with 10% of the samples being homozygous for CYP2D6*1. Therefore, our results indicate that CYP2D6 metabolic phenotype assignment may be overestimated for a significant portion of individuals. Large variability has been observed in CYP2D6 phenotype groups based on the AS system ([Crews et al., 2014](#B9); [Hicks et al., 2017](#B19)), which may in part be driven by both *2 and *1 being assigned a normal AS of 1, instead of *1 being considered a reduced expression allele. However, *in vitro* and population pharmacokinetic studies have shown that the amino acid change (R296C) caused by rs16947 in CYP2D6*2 reduces enzymatic activity for some substrates ([Abduljalil et al., 2010](#B1); [Muroi et al., 2014](#B27); [Hertz et al., 2015](#B17); [van der Lee et al., 2021](#B41); [Frederiksen et al., 2022](#B13)) and further separation of CYP2D6*1 and CYP2D6*2 phenotypes may be necessary for different substrates.

We re-evaluated the impacts of the enhancer SNP rs5758550 and rs16947 on the expression of CYP2D6 mRNA, splice isoforms, and protein in liver samples. Our results showed strong effects of rs5758550 and rs16947 on full-length CYP2D6 protein expression, which, together with rs1058164, explained 45% ([Table 3](#T3)) of the variability in full-length CYP2D6 protein abundance in the liver samples. These results are consistent with previous reports showing that including rs5758550 and rs16947 into a new AS system better predicts CYP2D6 activity ([Wang et al., 2014](#B43); [Ray et al., 2019](#B33); [Thomas et al., 2020](#B39)). However, there have been a few studies that debate the influence of rs5758550 and rs16947. For example, rs5758550 apparently does not alter CYP2D6*2 activity towards the metabolism of tamoxifen ([Sanchez-Spitman et al., 2017](#B36)) or dextromethorphan ([Dinh et al., 2022](#B11)) but does impact atomoxetine metabolism ([Dinh et al., 2022](#B11)). These discrepancies may be due to a variety of reasons, including the highly polymorphic nature of CYP2D6, the small sample sizes used in the studies, the analysis methods used, and the capacity of a substrate drug to serve as a surrogate for CYPD6-specific enzyme activity and/or protein abundance. In support of our results, a recent report with a relatively large sample size (n = 497) showed that distal regulatory variants were associated with CYP2D6-dependent metabolism of tamoxifen (independent of CYP2D6 variants in the AS system), and rs5758550 was found to compensate for the deleterious effects of rs16947 ([Khor et al., 2023](#B22)).

We did not find an association between rs28371725 (core SNP of *41) and CYP2D6 mRNA, CYP2D6∆E6 isoform production, or full-length CYP2D6 protein expression ([Table 2](#T2); [Table 3](#T3)), which agrees with our previous results ([Wang et al., 2014](#B43)). However, there are multiple studies showing reduced CYP2D6 expression/activity in *41 ([Abduljalil et al., 2010](#B1); [van der Lee et al., 2021](#B41); [Khor et al., 2023](#B22)), which we hypothesize is due to rs28371725 acting as a surrogate marker for the haplotypes comprised of rs16947 without rs5758550 (i.e., H3-haplotype described in ([Ray et al., 2019](#B33))). Unlike the other reports ([Abduljalil et al., 2010](#B1); [van der Lee et al., 2021](#B41); [Khor et al., 2023](#B22)), our liver cohort consisted of ∼50% AA samples where the concordance between *41 and H3 is the lowest ([Ray et al., 2019](#B33)). In our cohort, all *41 alleles were found in H3-haplotypes in EA, whereas only 81% of the *41 alleles were on H3-haplotypes in AA. Also, while only 6.5% of the H3-haplotypes in EA samples did not have rs28371725 (*41), 18.1% of AA H3-haplotypes did not (allele frequencies were consistent with ([Ray et al., 2019](#B33))). Therefore, this LD breakage of rs28371725 (*41) and the H3-haplotypes in our cohort enabled us to dissect the roles of rs28371725, rs16947, and rs5758550 on full-length CYP2D6 protein expression and our results did not support an independent effect of rs28371725 on CYP2D6 transcripts and protein expression.

Moreover, the molecular mechanism underlying how rs28371725 (*41) impacts CYP2D6 expression is still unclear; it is an intronic variant that was proposed to cause partial intron 6 retention ([Rau et al., 2006](#B32)) and skipping of exon 6 ([Toscano et al., 2006](#B40)). Recently, a tri-allelic haplotype (consisting of rs16947, *41, and a SNP in exon 9 rs1135840 C) was reported to cause CYP2D6∆E6 in Huh-7 and COS-7 transfection experiments ([Zanger et al., 2021](#B46)). These results disagree with our previous study showing no differences in CYP2D6∆E6 levels between *2 and *41 in either liver samples or in transfected HEK293 T cells ([Wang et al., 2014](#B43)). Unlike *41, the rs1135840 C variant always co-occurs with *2 and is in high LD with rs1058164 C. It is unclear how a nucleotide substitution in exon 9 could affect the skipping of exon 6. Also, as Huh-7 cells express detectable levels of CYP2D6 (Ct value ∼26 indicating moderate expression), natively-expressed CYP2D6 may have confounded their quantification of transfected CYP2D6. Moreover, many of the rare haplotypes (for example, rs16947 T-rs1058164G) presented in ([Zanger et al., 2021](#B46)) were generated by site-directed mutagenesis with *1 as the template. While it is unclear whether rs1058164 G (found on *1) can affect CYP2D6∆E6 production, our results presented here indicated that CYP2D6∆E3 (produced by rs1058164 G) and CYP2D6∆E6 may be mutually exclusive.

Our results also indicated that CYP2D6 expression may be over-estimated in individuals with gene duplication/multiplications since we found no linear relationship between CYP2D6 expression and CNV number in samples carrying CNV >2. Several studies have shown that CYP2D6 activity overlaps between UM (gene duplication being the main cause) and EM phenotypes ([Hertz et al., 2015](#B17); [de Andres et al., 2021](#B10); [van der Lee et al., 2021](#B41)), which may arise from an overestimation of the impact of CYP2D6 CNVs. Previously, we found that deletion of the downstream enhancer reduced CYP2D6 expression by half ([Wang et al., 2014](#B43)), and as enhancer-mediated regulation of every CYP2D6 copy may not be equal and additional duplicated genes may be under-expressed compared to a singular copy. As enhancer-mediated regulation is complex, future experiments to determine the role of the downstream enhancer in regulating multiple CYP2D6 gene copies are warranted.

While the new models made significant improvements for predicting CYP2D6 protein expression, we observed differences in how they performed for the two populations. Model 2 was more robust for predicting CYP2D6 protein levels in AA, while model 3 made a larger difference for predicting EA protein ([Table 4](#T4)). This is likely due to population differences in the distribution and LD of the three regulatory SNPs and CYP2D6 CNV frequencies. For example, in the EA population, most of the rs16947-containing haplotypes also contained rs5758550, and in model 2, they were assigned an ES of 1, which is the same value they received in model 1. However, in AA, the concordance between rs16947 and rs5758550 was low, and therefore the ES for many rs16947-containing alleles was misassigned in model 1. Also, AA samples had a higher frequency of >2 CYP2D6 CNVs than EA. Since model 2 specifically corrected misassigned rs16947 alleles and CNVs, it makes sense that it made such a large improvement for the prediction of full-length CYP2D6 protein abundance in AA. In contrast, model 3 addressed the contribution of rs1058164 G alternative splicing on the CYP2D6*1 allele, which is more frequent in EA than in AA, and likely drives the better improvement of model 3 for EA. Many pharmacogenomic panels show differences in their ability to predict phenotypes for diverse populations ([Ortega and Meyers, 2014](#B30)), and our results echo the need for more research on diverse populations to help better design panels for use in personalized drug therapy.

A limitation of this study is that only common SNPs were genotyped, and rare SNPs that may affect full-length CYP2D6 protein expression (e.g., *11, *15, *19, *etc.*) were not captured. However, although these rare alleles can potentially impact CYP2D6 protein expression in individuals, their impact on variability at a population level is expected to be limited. We measured CYP2D6 protein levels instead of activity since we focused on testing the effect of regulatory variants on CYP2D6 mRNA and protein expression. However, although we cannot fully account for the extent that the CYP2D6 regulatory variants influence *in vivo* CYP2D6 activity, their impact may be large as CYP2D6 protein level is the major contributor to interindividual variability in CYP2D6-mediated drug metabolism ([Ning et al., 2018](#B28)). Considering the improvements made by model 3 to predicting full-length CYP2D6 expression, additional studies to evaluate the impact of these regulatory SNPs on CYP2D6 activity and their applicability to additional populations are warranted.

## Funding Statement

This work was supported by a research grant from the National Institute of Health R35 GM140845 to DW.

## Data availability statement

The original contributions presented in the study are included in the article/[Supplementary Materials](#s11), further inquiries can be directed to the corresponding author.

## Ethics statement

The studies involving human liver samples were reviewed and approved by the University of Florida IRB committee. Written informed consent for participation was not required for this study in accordance with the national legislation and the institutional requirements.

## Author contributions

JC designed and performed experiments and wrote manuscript, HL and SS performed experiments, DW designed and performed experiments, analysed data, and wrote manuscript. All authors contributed to the article and approved the submitted version.

## Conflict of interest

The authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.

## Publisher’s note

All claims expressed in this article are solely those of the authors and do not necessarily represent those of their affiliated organizations, or those of the publisher, the editors and the reviewers. Any product that may be evaluated in this article, or claim that may be made by its manufacturer, is not guaranteed or endorsed by the publisher.

## Supplementary material

The Supplementary Material for this article can be found online at: [https://www.frontiersin.org/articles/10.3389/fphar.2023.1186540/full#supplementary-material](https://www.frontiersin.org/articles/10.3389/fphar.2023.1186540/full#supplementary-material)

## Abbreviations

AA; African American, AS; activity score, CNV; gene copy number variation, CPIC; Clinical Pharmacogenomics Implementation Consortium, CYP; cytochrome P450, EA; European American, ES; expression score, TF; transcription factor, *−; star allele.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

The original contributions presented in the study are included in the article/[Supplementary Materials](#s11), further inquiries can be directed to the corresponding author.

### Supplementary Materials

### Data Availability Statement

The original contributions presented in the study are included in the article/[Supplementary Materials](#s11), further inquiries can be directed to the corresponding author.

## References

1. Abduljalil K., Frank D., Gaedigk A., Klaassen T., Tomalik-Scharte D., Jetter A., et al. (2010). Assessment of activity levels for CYP2D6*1, CYP2D6*2, and CYP2D6*41 genes by population pharmacokinetics of dextromethorphan. Clin. Pharmacol. Ther. 88 (5), 643–651. 10.1038/clpt.2010.137  [DOI](https://doi.org/10.1038/clpt.2010.137) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20881950/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Assessment%20of%20activity%20levels%20for%20CYP2D6*1,%20CYP2D6*2,%20and%20CYP2D6*41%20genes%20by%20population%20pharmacokinetics%20of%20dextromethorphan&author=K.%20Abduljalil&author=D.%20Frank&author=A.%20Gaedigk&author=T.%20Klaassen&author=D.%20Tomalik-Scharte&volume=88&issue=5&publication_year=2010&pages=643-651&pmid=20881950&doi=10.1038/clpt.2010.137&)

2. 1000 Genomes Project Consortium Abecasis G. R., Auton A., Brooks L. D., DePristo M. A., Durbin R. M., Handsaker R. E., et al. (2012). An integrated map of genetic variation from 1,092 human genomes. Nature 491 (7422), 56–65. 10.1038/nature11632  [DOI](https://doi.org/10.1038/nature11632) | [PMC free article](/articles/PMC3498066/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23128226/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=An%20integrated%20map%20of%20genetic%20variation%20from%201,092%20human%20genomes&author=G.%20R.%20Abecasis&author=A.%20Auton&author=L.%20D.%20Brooks&author=M.%20A.%20DePristo&author=R.%20M.%20Durbin&volume=491&issue=7422&publication_year=2012&pages=56-65&pmid=23128226&doi=10.1038/nature11632&)

3. Amoah K., Hsiao Y. E., Bahn J. H., Sun Y., Burghard C., Tan B. X., et al. (2021). Allele-specific alternative splicing and its functional genetic variants in human tissues. Genome Res. 31 (3), 359–371. 10.1101/gr.265637.120  [DOI](https://doi.org/10.1101/gr.265637.120) | [PMC free article](/articles/PMC7919445/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33452016/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res.&title=Allele-specific%20alternative%20splicing%20and%20its%20functional%20genetic%20variants%20in%20human%20tissues&author=K.%20Amoah&author=Y.%20E.%20Hsiao&author=J.%20H.%20Bahn&author=Y.%20Sun&author=C.%20Burghard&volume=31&issue=3&publication_year=2021&pages=359-371&pmid=33452016&doi=10.1101/gr.265637.120&)

4. Bell G. C., Caudle K. E., Whirl-Carrillo M., Gordon R. J., Hikino K., Prows C. A., et al. (2017). Clinical Pharmacogenetics Implementation Consortium (CPIC) guideline for CYP2D6 genotype and use of ondansetron and tropisetron. Clin. Pharmacol. Ther. 102 (2), 213–218. 10.1002/cpt.598  [DOI](https://doi.org/10.1002/cpt.598) | [PMC free article](/articles/PMC5479760/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28002639/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Clinical%20Pharmacogenetics%20Implementation%20Consortium%20(CPIC)%20guideline%20for%20CYP2D6%20genotype%20and%20use%20of%20ondansetron%20and%20tropisetron&author=G.%20C.%20Bell&author=K.%20E.%20Caudle&author=M.%20Whirl-Carrillo&author=R.%20J.%20Gordon&author=K.%20Hikino&volume=102&issue=2&publication_year=2017&pages=213-218&pmid=28002639&doi=10.1002/cpt.598&)

5. Bock K. W., Schrenk D., Forster A., Griese E. U., Mörike K., Brockmeier D., et al. (1994). The influence of environmental and genetic factors on CYP2D6, CYP1A2 and UDP-glucuronosyltransferase in man using sparteine, caffeine, and paracetamol as probes. Pharmacogenetics 4 (4), 209–218. 10.1097/00008571-199408000-00005  [DOI](https://doi.org/10.1097/00008571-199408000-00005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7987405/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=The%20influence%20of%20environmental%20and%20genetic%20factors%20on%20CYP2D6,%20CYP1A2%20and%20UDP-glucuronosyltransferase%20in%20man%20using%20sparteine,%20caffeine,%20and%20paracetamol%20as%20probes&author=K.%20W.%20Bock&author=D.%20Schrenk&author=A.%20Forster&author=E.%20U.%20Griese&author=K.%20M%C3%B6rike&volume=4&issue=4&publication_year=1994&pages=209-218&pmid=7987405&doi=10.1097/00008571-199408000-00005&)

6. Brown J. T., Bishop J. R., Sangkuhl K., Nurmi E. L., Mueller D. J., Dinh J. C., et al. (2019). Clinical pharmacogenetics implementation consortium guideline for cytochrome P450 (CYP)2D6 genotype and atomoxetine therapy. Clin. Pharmacol. Ther. 106 (1), 94–102. 10.1002/cpt.1409  [DOI](https://doi.org/10.1002/cpt.1409) | [PMC free article](/articles/PMC6612570/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30801677/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Clinical%20pharmacogenetics%20implementation%20consortium%20guideline%20for%20cytochrome%20P450%20(CYP)2D6%20genotype%20and%20atomoxetine%20therapy&author=J.%20T.%20Brown&author=J.%20R.%20Bishop&author=K.%20Sangkuhl&author=E.%20L.%20Nurmi&author=D.%20J.%20Mueller&volume=106&issue=1&publication_year=2019&pages=94-102&pmid=30801677&doi=10.1002/cpt.1409&)

7. Collins J. M., Wang D. (2021a). Co-Expression of drug metabolizing cytochrome P450 enzymes and estrogen receptor alpha (ESR1) in human liver: Racial differences and the regulatory role of ESR1. Drug Metab. Pers. Ther. 36 (3), 205–214. 10.1515/dmpt-2020-0160  [DOI](https://doi.org/10.1515/dmpt-2020-0160) | [PMC free article](/articles/PMC8376797/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33823094/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab.%20Pers.%20Ther.&title=Co-Expression%20of%20drug%20metabolizing%20cytochrome%20P450%20enzymes%20and%20estrogen%20receptor%20alpha%20(ESR1)%20in%20human%20liver:%20Racial%20differences%20and%20the%20regulatory%20role%20of%20ESR1&author=J.%20M.%20Collins&author=D.%20Wang&volume=36&issue=3&publication_year=2021a&pages=205-214&pmid=33823094&doi=10.1515/dmpt-2020-0160&)

8. Collins J. M., Wang D. (2021b). Cytochrome P450 3A4 (CYP3A4) protein quantification using capillary Western blot technology and total protein normalization. J. Pharmacol. Toxicol. Methods 112, 107117. 10.1016/j.vascn.2021.107117  [DOI](https://doi.org/10.1016/j.vascn.2021.107117) | [PMC free article](/articles/PMC8616831/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34474151/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Pharmacol.%20Toxicol.%20Methods&title=Cytochrome%20P450%203A4%20(CYP3A4)%20protein%20quantification%20using%20capillary%20Western%20blot%20technology%20and%20total%20protein%20normalization&author=J.%20M.%20Collins&author=D.%20Wang&volume=112&publication_year=2021b&pages=107117&pmid=34474151&doi=10.1016/j.vascn.2021.107117&)

9. Crews K. R., Gaedigk A., Dunnenberger H. M., Leeder J. S., Klein T. E., Caudle K. E., et al. (2014). Clinical pharmacogenetics implementation consortium guidelines for cytochrome P450 2D6 genotype and codeine therapy: 2014 update. Clin. Pharmacol. Ther. 95 (4), 376–382. 10.1038/clpt.2013.254  [DOI](https://doi.org/10.1038/clpt.2013.254) | [PMC free article](/articles/PMC3975212/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24458010/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Clinical%20pharmacogenetics%20implementation%20consortium%20guidelines%20for%20cytochrome%20P450%202D6%20genotype%20and%20codeine%20therapy:%202014%20update&author=K.%20R.%20Crews&author=A.%20Gaedigk&author=H.%20M.%20Dunnenberger&author=J.%20S.%20Leeder&author=T.%20E.%20Klein&volume=95&issue=4&publication_year=2014&pages=376-382&pmid=24458010&doi=10.1038/clpt.2013.254&)

10. de Andres F., Altamirano-Tinoco C., Ramirez-Roa R., Montes-Mondragon C. F., Dorado P., Penas-Lledo E. M. (2021). Relationships between CYP1A2, CYP2C9, CYP2C19, CYP2D6 and CYP3A4 metabolic phenotypes and genotypes in a Nicaraguan Mestizo population. Pharmacogenomics J. 21 (2), 140–151. 10.1038/s41397-020-00190-9  [DOI](https://doi.org/10.1038/s41397-020-00190-9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33024249/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J.&title=Relationships%20between%20CYP1A2,%20CYP2C9,%20CYP2C19,%20CYP2D6%20and%20CYP3A4%20metabolic%20phenotypes%20and%20genotypes%20in%20a%20Nicaraguan%20Mestizo%20population&author=F.%20de%20Andres&author=C.%20Altamirano-Tinoco&author=R.%20Ramirez-Roa&author=C.%20F.%20Montes-Mondragon&author=P.%20Dorado&volume=21&issue=2&publication_year=2021&pages=140-151&pmid=33024249&doi=10.1038/s41397-020-00190-9&)

11. Dinh J. C., Boone E. C., Staggs V. S., Pearce R. E., Wang W. Y., Gaedigk R., et al. (2022). The impact of the CYP2D6 "enhancer" single nucleotide polymorphism on CYP2D6 activity. Clin. Pharmacol. Ther. 111 (3), 646–654. 10.1002/cpt.2469  [DOI](https://doi.org/10.1002/cpt.2469) | [PMC free article](/articles/PMC8825689/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34716917/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=The%20impact%20of%20the%20CYP2D6%20%22enhancer%22%20single%20nucleotide%20polymorphism%20on%20CYP2D6%20activity&author=J.%20C.%20Dinh&author=E.%20C.%20Boone&author=V.%20S.%20Staggs&author=R.%20E.%20Pearce&author=W.%20Y.%20Wang&volume=111&issue=3&publication_year=2022&pages=646-654&pmid=34716917&doi=10.1002/cpt.2469&)

12. Elias A. B. R., Araujo G. S., de Souza S. J., Suarez-Kurtz G. (2020). Distribution and linkage disequilibrium of the enhancer SNP rs5758550 among Latin American populations: Influence of continental ancestry. Pharmacogenet Genomics 30 (4), 67–72. 10.1097/FPC.0000000000000398  [DOI](https://doi.org/10.1097/FPC.0000000000000398) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32187157/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Distribution%20and%20linkage%20disequilibrium%20of%20the%20enhancer%20SNP%20rs5758550%20among%20Latin%20American%20populations:%20Influence%20of%20continental%20ancestry&author=A.%20B.%20R.%20Elias&author=G.%20S.%20Araujo&author=S.%20J.%20de%20Souza&author=G.%20Suarez-Kurtz&volume=30&issue=4&publication_year=2020&pages=67-72&pmid=32187157&doi=10.1097/FPC.0000000000000398&)

13. Frederiksen T., Areberg J., Raoufinia A., Schmidt E., Stage T. B., Brosen K. (2022). Quantification of in vivo metabolic activity of CYP2D6 genotypes and alleles through population pharmacokinetic analysis of vortioxetine. Clin. Pharmacol. Ther. 109, 150–159. 10.1002/cpt.1972  [DOI](https://doi.org/10.1002/cpt.1972) | [PMC free article](/articles/PMC7818194/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32599653/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Quantification%20of%20in%20vivo%20metabolic%20activity%20of%20CYP2D6%20genotypes%20and%20alleles%20through%20population%20pharmacokinetic%20analysis%20of%20vortioxetine&author=T.%20Frederiksen&author=J.%20Areberg&author=A.%20Raoufinia&author=E.%20Schmidt&author=T.%20B.%20Stage&volume=109&publication_year=2022&pages=150-159&pmid=32599653&doi=10.1002/cpt.1972&)

14. Gaedigk A., Simon S. D., Pearce R. E., Bradford L. D., Kennedy M. J., Leeder J. S. (2008). The CYP2D6 activity score: Translating genotype information into a qualitative measure of phenotype. Clin. Pharmacol. Ther. 83 (2), 234–242. 10.1038/sj.clpt.6100406  [DOI](https://doi.org/10.1038/sj.clpt.6100406) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17971818/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=The%20CYP2D6%20activity%20score:%20Translating%20genotype%20information%20into%20a%20qualitative%20measure%20of%20phenotype&author=A.%20Gaedigk&author=S.%20D.%20Simon&author=R.%20E.%20Pearce&author=L.%20D.%20Bradford&author=M.%20J.%20Kennedy&volume=83&issue=2&publication_year=2008&pages=234-242&pmid=17971818&doi=10.1038/sj.clpt.6100406&)

15. Goetz M. P., Sangkuhl K., Guchelaar H. J., Schwab M., Province M., Whirl-Carrillo M., et al. (2018). Clinical pharmacogenetics implementation consortium (CPIC) guideline for CYP2D6 and tamoxifen therapy. Clin. Pharmacol. Ther. 103 (5), 770–777. 10.1002/cpt.1007  [DOI](https://doi.org/10.1002/cpt.1007) | [PMC free article](/articles/PMC5931215/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29385237/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Clinical%20pharmacogenetics%20implementation%20consortium%20(CPIC)%20guideline%20for%20CYP2D6%20and%20tamoxifen%20therapy&author=M.%20P.%20Goetz&author=K.%20Sangkuhl&author=H.%20J.%20Guchelaar&author=M.%20Schwab&author=M.%20Province&volume=103&issue=5&publication_year=2018&pages=770-777&pmid=29385237&doi=10.1002/cpt.1007&)

16. GTEx Consortium (2020). The GTEx Consortium atlas of genetic regulatory effects across human tissues. Science 369 (6509), 1318–1330. 10.1126/science.aaz1776  [DOI](https://doi.org/10.1126/science.aaz1776) | [PMC free article](/articles/PMC7737656/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32913098/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=The%20GTEx%20Consortium%20atlas%20of%20genetic%20regulatory%20effects%20across%20human%20tissues&volume=369&issue=6509&publication_year=2020&pages=1318-1330&pmid=32913098&doi=10.1126/science.aaz1776&)

17. Hertz D. L., Snavely A. C., McLeod H. L., Walko C. M., Ibrahim J. G., Anderson S., et al. (2015). In vivo assessment of the metabolic activity of CYP2D6 diplotype and alleles. Br. J. Clin. Pharmacol. 80 (5), 1122–1130. 10.1111/bcp.12665  [DOI](https://doi.org/10.1111/bcp.12665) | [PMC free article](/articles/PMC4631184/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25907378/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Clin.%20Pharmacol.&title=In%20vivo%20assessment%20of%20the%20metabolic%20activity%20of%20CYP2D6%20diplotype%20and%20alleles&author=D.%20L.%20Hertz&author=A.%20C.%20Snavely&author=H.%20L.%20McLeod&author=C.%20M.%20Walko&author=J.%20G.%20Ibrahim&volume=80&issue=5&publication_year=2015&pages=1122-1130&pmid=25907378&doi=10.1111/bcp.12665&)

18. Hicks J. K., Bishop J. R., Sangkuhl K., Muller D. J., Ji Y., Leckband S. G., et al. (2015). Clinical pharmacogenetics implementation consortium (CPIC) guideline for CYP2D6 and CYP2C19 genotypes and dosing of selective serotonin reuptake inhibitors. Clin. Pharmacol. Ther. 98 (2), 127–134. 10.1002/cpt.147  [DOI](https://doi.org/10.1002/cpt.147) | [PMC free article](/articles/PMC4512908/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25974703/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Clinical%20pharmacogenetics%20implementation%20consortium%20(CPIC)%20guideline%20for%20CYP2D6%20and%20CYP2C19%20genotypes%20and%20dosing%20of%20selective%20serotonin%20reuptake%20inhibitors&author=J.%20K.%20Hicks&author=J.%20R.%20Bishop&author=K.%20Sangkuhl&author=D.%20J.%20Muller&author=Y.%20Ji&volume=98&issue=2&publication_year=2015&pages=127-134&pmid=25974703&doi=10.1002/cpt.147&)

19. Hicks J. K., Sangkuhl K., Swen J. J., Ellingrod V. L., Müller D. J., Shimoda K., et al. (2017). Clinical pharmacogenetics implementation consortium guideline (CPIC) for CYP2D6 and CYP2C19 genotypes and dosing of tricyclic antidepressants: 2016 update. Clin. Pharmacol. Ther. 102 (1), 37–44. 10.1002/cpt.597  [DOI](https://doi.org/10.1002/cpt.597) | [PMC free article](/articles/PMC5478479/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27997040/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Clinical%20pharmacogenetics%20implementation%20consortium%20guideline%20(CPIC)%20for%20CYP2D6%20and%20CYP2C19%20genotypes%20and%20dosing%20of%20tricyclic%20antidepressants:%202016%20update&author=J.%20K.%20Hicks&author=K.%20Sangkuhl&author=J.%20J.%20Swen&author=V.%20L.%20Ellingrod&author=D.%20J.%20M%C3%BCller&volume=102&issue=1&publication_year=2017&pages=37-44&pmid=27997040&doi=10.1002/cpt.597&)

20. Ingelman-Sundberg M. (2005). Genetic polymorphisms of cytochrome P450 2D6 (CYP2D6): Clinical consequences, evolutionary aspects and functional diversity. Pharmacogenomics J. 5 (1), 6–13. 10.1038/sj.tpj.6500285  [DOI](https://doi.org/10.1038/sj.tpj.6500285) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15492763/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J.&title=Genetic%20polymorphisms%20of%20cytochrome%20P450%202D6%20(CYP2D6):%20Clinical%20consequences,%20evolutionary%20aspects%20and%20functional%20diversity&author=M.%20Ingelman-Sundberg&volume=5&issue=1&publication_year=2005&pages=6-13&pmid=15492763&doi=10.1038/sj.tpj.6500285&)

21. Kandel S. E., Lampe J. N. (2014). Role of protein-protein interactions in cytochrome P450-mediated drug metabolism and toxicity. Chem. Res. Toxicol. 27 (9), 1474–1486. 10.1021/tx500203s  [DOI](https://doi.org/10.1021/tx500203s) | [PMC free article](/articles/PMC4164225/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25133307/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chem.%20Res.%20Toxicol.&title=Role%20of%20protein-protein%20interactions%20in%20cytochrome%20P450-mediated%20drug%20metabolism%20and%20toxicity&author=S.%20E.%20Kandel&author=J.%20N.%20Lampe&volume=27&issue=9&publication_year=2014&pages=1474-1486&pmid=25133307&doi=10.1021/tx500203s&)

22. Khor C. C., Winter S., Sutiman N., Murdter T. E., Chen S., Lim J. S. L., et al. (2023). Cross-ancestry genome-wide association study defines the extended CYP2D6 locus as the principal genetic determinant of endoxifen plasma concentrations. Clin. Pharmacol. Ther. 113, 712–723. 10.1002/cpt.2846  [DOI](https://doi.org/10.1002/cpt.2846) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36629403/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Cross-ancestry%20genome-wide%20association%20study%20defines%20the%20extended%20CYP2D6%20locus%20as%20the%20principal%20genetic%20determinant%20of%20endoxifen%20plasma%20concentrations&author=C.%20C.%20Khor&author=S.%20Winter&author=N.%20Sutiman&author=T.%20E.%20Murdter&author=S.%20Chen&volume=113&publication_year=2023&pages=712-723&pmid=36629403&doi=10.1002/cpt.2846&)

23. Langaee T., Hamadeh I., Chapman A. B., Gums J. G., Johnson J. A. (2015). A novel simple method for determining CYP2D6 gene copy number and identifying allele(s) with duplication/multiplication. PLoS One 10 (1), e0113808. 10.1371/journal.pone.0113808  [DOI](https://doi.org/10.1371/journal.pone.0113808) | [PMC free article](/articles/PMC4308104/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25625348/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=A%20novel%20simple%20method%20for%20determining%20CYP2D6%20gene%20copy%20number%20and%20identifying%20allele(s)%20with%20duplication/multiplication&author=T.%20Langaee&author=I.%20Hamadeh&author=A.%20B.%20Chapman&author=J.%20G.%20Gums&author=J.%20A.%20Johnson&volume=10&issue=1&publication_year=2015&pages=e0113808&pmid=25625348&doi=10.1371/journal.pone.0113808&)

24. Llerena A., Dorado P., Ramírez R., González I., Alvarez M., Peñas-Lledó E. M., et al. (2012). CYP2D6 genotype and debrisoquine hydroxylation phenotype in Cubans and Nicaraguans. Pharmacogenomics J. 12 (2), 176–183. 10.1038/tpj.2010.85  [DOI](https://doi.org/10.1038/tpj.2010.85) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21135868/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J.&title=CYP2D6%20genotype%20and%20debrisoquine%20hydroxylation%20phenotype%20in%20Cubans%20and%20Nicaraguans&author=A.%20Llerena&author=P.%20Dorado&author=R.%20Ram%C3%ADrez&author=I.%20Gonz%C3%A1lez&author=M.%20Alvarez&volume=12&issue=2&publication_year=2012&pages=176-183&pmid=21135868&doi=10.1038/tpj.2010.85&)

25. Machiela M. J., Chanock S. J. (2015). LDlink: A web-based application for exploring population-specific haplotype structure and linking correlated alleles of possible functional variants. Bioinformatics 31 (21), 3555–3557. 10.1093/bioinformatics/btv402  [DOI](https://doi.org/10.1093/bioinformatics/btv402) | [PMC free article](/articles/PMC4626747/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26139635/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=LDlink:%20A%20web-based%20application%20for%20exploring%20population-specific%20haplotype%20structure%20and%20linking%20correlated%20alleles%20of%20possible%20functional%20variants&author=M.%20J.%20Machiela&author=S.%20J.%20Chanock&volume=31&issue=21&publication_year=2015&pages=3555-3557&pmid=26139635&doi=10.1093/bioinformatics/btv402&)

26. Montané Jaime L. K., Lalla A., Steimer W., Gaedigk A. (2013). Characterization of the CYP2D6 gene locus and metabolic activity in indo- and afro-Trinidadians: Discovery of novel allelic variants. Pharmacogenomics 14 (3), 261–276. 10.2217/pgs.12.207  [DOI](https://doi.org/10.2217/pgs.12.207) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23394389/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Characterization%20of%20the%20CYP2D6%20gene%20locus%20and%20metabolic%20activity%20in%20indo-%20and%20afro-Trinidadians:%20Discovery%20of%20novel%20allelic%20variants&author=L.%20K.%20Montan%C3%A9%20Jaime&author=A.%20Lalla&author=W.%20Steimer&author=A.%20Gaedigk&volume=14&issue=3&publication_year=2013&pages=261-276&pmid=23394389&doi=10.2217/pgs.12.207&)

27. Muroi Y., Saito T., Takahashi M., Sakuyama K., Niinuma Y., Ito M., et al. (2014). Functional characterization of wild-type and 49 CYP2D6 allelic variants for N-desmethyltamoxifen 4-hydroxylation activity. Drug Metab. Pharmacokinet. 29 (5), 360–366. 10.2133/dmpk.dmpk-14-rg-014  [DOI](https://doi.org/10.2133/dmpk.dmpk-14-rg-014) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24647041/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab.%20Pharmacokinet.&title=Functional%20characterization%20of%20wild-type%20and%2049%20CYP2D6%20allelic%20variants%20for%20N-desmethyltamoxifen%204-hydroxylation%20activity&author=Y.%20Muroi&author=T.%20Saito&author=M.%20Takahashi&author=K.%20Sakuyama&author=Y.%20Niinuma&volume=29&issue=5&publication_year=2014&pages=360-366&pmid=24647041&doi=10.2133/dmpk.dmpk-14-rg-014&)

28. Ning M., Duarte J. D., Rubin L. H., Jeong H. (2018). CYP2D6 protein level is the major contributor to interindividual variability in CYP2D6-mediated drug metabolism in healthy human liver tissue. Clin. Pharmacol. Ther. 104 (5), 974–982. 10.1002/cpt.1032  [DOI](https://doi.org/10.1002/cpt.1032) | [PMC free article](/articles/PMC6053340/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29349771/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=CYP2D6%20protein%20level%20is%20the%20major%20contributor%20to%20interindividual%20variability%20in%20CYP2D6-mediated%20drug%20metabolism%20in%20healthy%20human%20liver%20tissue&author=M.%20Ning&author=J.%20D.%20Duarte&author=L.%20H.%20Rubin&author=H.%20Jeong&volume=104&issue=5&publication_year=2018&pages=974-982&pmid=29349771&doi=10.1002/cpt.1032&)

29. Nofziger C., Turner A. J., Sangkuhl K., Whirl-Carrillo M., Agúndez J. A. G., Black J. L., et al. (2019). PharmVar GeneFocus: CYP2D6. Clin. Pharmacol. Ther. 107, 154–170. 10.1002/cpt.1643  [DOI](https://doi.org/10.1002/cpt.1643) | [PMC free article](/articles/PMC6925641/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31544239/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=PharmVar%20GeneFocus:%20CYP2D6&author=C.%20Nofziger&author=A.%20J.%20Turner&author=K.%20Sangkuhl&author=M.%20Whirl-Carrillo&author=J.%20A.%20G.%20Ag%C3%BAndez&volume=107&publication_year=2019&pages=154-170&pmid=31544239&doi=10.1002/cpt.1643&)

30. Ortega V. E., Meyers D. A. (2014). Pharmacogenetics: Implications of race and ethnicity on defining genetic profiles for personalized medicine. J. Allergy Clin. Immunol. 133 (1), 16–26. 10.1016/j.jaci.2013.10.040  [DOI](https://doi.org/10.1016/j.jaci.2013.10.040) | [PMC free article](/articles/PMC3933289/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24369795/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Allergy%20Clin.%20Immunol.&title=Pharmacogenetics:%20Implications%20of%20race%20and%20ethnicity%20on%20defining%20genetic%20profiles%20for%20personalized%20medicine&author=V.%20E.%20Ortega&author=D.%20A.%20Meyers&volume=133&issue=1&publication_year=2014&pages=16-26&pmid=24369795&doi=10.1016/j.jaci.2013.10.040&)

31. Pedersen R. S., Damkier P., Brosen K. (2005). Tramadol as a new probe for cytochrome P450 2D6 phenotyping: A population study. Clin. Pharmacol. Ther. 77 (6), 458–467. 10.1016/j.clpt.2005.01.014  [DOI](https://doi.org/10.1016/j.clpt.2005.01.014) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15961977/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Tramadol%20as%20a%20new%20probe%20for%20cytochrome%20P450%202D6%20phenotyping:%20A%20population%20study&author=R.%20S.%20Pedersen&author=P.%20Damkier&author=K.%20Brosen&volume=77&issue=6&publication_year=2005&pages=458-467&pmid=15961977&doi=10.1016/j.clpt.2005.01.014&)

32. Rau T., Diepenbruck S., Diepenbruck I., Eschenhagen T. (2006). The 2988G>A polymorphism affects splicing of a CYP2D6 minigene. Clin. Pharmacol. Ther. 80 (5), 555–558. 10.1016/j.clpt.2006.08.008  [DOI](https://doi.org/10.1016/j.clpt.2006.08.008) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17112815/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=The%202988G>A%20polymorphism%20affects%20splicing%20of%20a%20CYP2D6%20minigene&author=T.%20Rau&author=S.%20Diepenbruck&author=I.%20Diepenbruck&author=T.%20Eschenhagen&volume=80&issue=5&publication_year=2006&pages=555-558&pmid=17112815&doi=10.1016/j.clpt.2006.08.008&)

33. Ray B., Ozcagli E., Sadee W., Wang D. (2019). CYP2D6 haplotypes with enhancer single-nucleotide polymorphism rs5758550 and rs16947 (*2 allele): Implications for CYP2D6 genotyping panels. Pharmacogenet Genomics 29 (2), 39–47. 10.1097/FPC.0000000000000363  [DOI](https://doi.org/10.1097/FPC.0000000000000363) | [PMC free article](/articles/PMC6441960/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30520769/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=CYP2D6%20haplotypes%20with%20enhancer%20single-nucleotide%20polymorphism%20rs5758550%20and%20rs16947%20(*2%20allele):%20Implications%20for%20CYP2D6%20genotyping%20panels&author=B.%20Ray&author=E.%20Ozcagli&author=W.%20Sadee&author=D.%20Wang&volume=29&issue=2&publication_year=2019&pages=39-47&pmid=30520769&doi=10.1097/FPC.0000000000000363&)

34. Reed J. R., Connick J. P., Cheng D., Cawley G. F., Backes W. L. (2012). Effect of homomeric P450-P450 complexes on P450 function. Biochem. J. 446 (3), 489–497. 10.1042/BJ20120636  [DOI](https://doi.org/10.1042/BJ20120636) | [PMC free article](/articles/PMC3518449/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22738171/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem.%20J.&title=Effect%20of%20homomeric%20P450-P450%20complexes%20on%20P450%20function&author=J.%20R.%20Reed&author=J.%20P.%20Connick&author=D.%20Cheng&author=G.%20F.%20Cawley&author=W.%20L.%20Backes&volume=446&issue=3&publication_year=2012&pages=489-497&pmid=22738171&doi=10.1042/BJ20120636&)

35. Rowland P., Blaney F. E., Smyth M. G., Jones J. J., Leydon V. R., Oxbrow A. K., et al. (2006). Crystal structure of human cytochrome P450 2D6. J. Biol. Chem. 281 (11), 7614–7622. 10.1074/jbc.M511232200  [DOI](https://doi.org/10.1074/jbc.M511232200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16352597/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem.&title=Crystal%20structure%20of%20human%20cytochrome%20P450%202D6&author=P.%20Rowland&author=F.%20E.%20Blaney&author=M.%20G.%20Smyth&author=J.%20J.%20Jones&author=V.%20R.%20Leydon&volume=281&issue=11&publication_year=2006&pages=7614-7622&pmid=16352597&doi=10.1074/jbc.M511232200&)

36. Sanchez-Spitman A. B., Moes D. A., Gelderblom H., Dezentje V. O., Swen J. J., Guchelaar H. J. (2017). The effect of rs5758550 on CYP2D6*2 phenotype and formation of endoxifen in breast cancer patients using tamoxifen. Pharmacogenomics 18 (12), 1125–1132. 10.2217/pgs-2017-0080  [DOI](https://doi.org/10.2217/pgs-2017-0080) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28745555/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=The%20effect%20of%20rs5758550%20on%20CYP2D6*2%20phenotype%20and%20formation%20of%20endoxifen%20in%20breast%20cancer%20patients%20using%20tamoxifen&author=A.%20B.%20Sanchez-Spitman&author=D.%20A.%20Moes&author=H.%20Gelderblom&author=V.%20O.%20Dezentje&author=J.%20J.%20Swen&volume=18&issue=12&publication_year=2017&pages=1125-1132&pmid=28745555&doi=10.2217/pgs-2017-0080&)

37. Sangar M. C., Anandatheerthavarada H. K., Martin M. V., Guengerich F. P., Avadhani N. G. (2010). Identification of genetic variants of human cytochrome P450 2D6 with impaired mitochondrial targeting. Mol. Genet. Metab. 99 (1), 90–97. 10.1016/j.ymgme.2009.08.009  [DOI](https://doi.org/10.1016/j.ymgme.2009.08.009) | [PMC free article](/articles/PMC2794965/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19781968/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Genet.%20Metab.&title=Identification%20of%20genetic%20variants%20of%20human%20cytochrome%20P450%202D6%20with%20impaired%20mitochondrial%20targeting&author=M.%20C.%20Sangar&author=H.%20K.%20Anandatheerthavarada&author=M.%20V.%20Martin&author=F.%20P.%20Guengerich&author=N.%20G.%20Avadhani&volume=99&issue=1&publication_year=2010&pages=90-97&pmid=19781968&doi=10.1016/j.ymgme.2009.08.009&)

38. Sistonen J., Fuselli S., Levo A., Sajantila A. (2005). CYP2D6 genotyping by a multiplex primer extension reaction. Clin. Chem. 51 (7), 1291–1295. 10.1373/clinchem.2004.046466  [DOI](https://doi.org/10.1373/clinchem.2004.046466) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15905314/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Chem.&title=CYP2D6%20genotyping%20by%20a%20multiplex%20primer%20extension%20reaction&author=J.%20Sistonen&author=S.%20Fuselli&author=A.%20Levo&author=A.%20Sajantila&volume=51&issue=7&publication_year=2005&pages=1291-1295&pmid=15905314&doi=10.1373/clinchem.2004.046466&)

39. Thomas C. D., Mosley S. A., Kim S., Lingineni K., El Rouby N., Langaee T. Y., et al. (2020). Examination of metoprolol pharmacokinetics and pharmacodynamics across CYP2D6 genotype-derived activity scores. CPT Pharmacometrics Syst. Pharmacol. 9 (12), 678–685. 10.1002/psp4.12563  [DOI](https://doi.org/10.1002/psp4.12563) | [PMC free article](/articles/PMC7762806/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33067866/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=CPT%20Pharmacometrics%20Syst.%20Pharmacol.&title=Examination%20of%20metoprolol%20pharmacokinetics%20and%20pharmacodynamics%20across%20CYP2D6%20genotype-derived%20activity%20scores&author=C.%20D.%20Thomas&author=S.%20A.%20Mosley&author=S.%20Kim&author=K.%20Lingineni&author=N.%20El%20Rouby&volume=9&issue=12&publication_year=2020&pages=678-685&pmid=33067866&doi=10.1002/psp4.12563&)

40. Toscano C., Klein K., Blievernicht J., Schaeffeler E., Saussele T., Raimundo S., et al. (2006). Impaired expression of CYP2D6 in intermediate metabolizers carrying the *41 allele caused by the intronic SNP 2988G>A: Evidence for modulation of splicing events. Pharmacogenetics Genomics 16 (10), 755–766. 10.1097/01.fpc.0000230112.96086.e0  [DOI](https://doi.org/10.1097/01.fpc.0000230112.96086.e0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17001295/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics%20Genomics&title=Impaired%20expression%20of%20CYP2D6%20in%20intermediate%20metabolizers%20carrying%20the%20*41%20allele%20caused%20by%20the%20intronic%20SNP%202988G>A:%20Evidence%20for%20modulation%20of%20splicing%20events&author=C.%20Toscano&author=K.%20Klein&author=J.%20Blievernicht&author=E.%20Schaeffeler&author=T.%20Saussele&volume=16&issue=10&publication_year=2006&pages=755-766&pmid=17001295&doi=10.1097/01.fpc.0000230112.96086.e0&)

41. van der Lee M., Allard W. G., Vossen R., Baak-Pablo R. F., Menafra R., Deiman B., et al. (2021). Toward predicting CYP2D6-mediated variable drug response from CYP2D6 gene sequencing data. Sci. Transl. Med. 13 (603), eabf3637. 10.1126/scitranslmed.abf3637  [DOI](https://doi.org/10.1126/scitranslmed.abf3637) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34290055/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sci.%20Transl.%20Med.&title=Toward%20predicting%20CYP2D6-mediated%20variable%20drug%20response%20from%20CYP2D6%20gene%20sequencing%20data&author=M.%20van%20der%20Lee&author=W.%20G.%20Allard&author=R.%20Vossen&author=R.%20F.%20Baak-Pablo&author=R.%20Menafra&volume=13&issue=603&publication_year=2021&pages=eabf3637&pmid=34290055&doi=10.1126/scitranslmed.abf3637&)

42. Wang D., Papp A. C., Sun X. (2015a). Functional characterization of CYP2D6 enhancer polymorphisms. Hum. Mol. Genet. 24 (6), 1556–1562. 10.1093/hmg/ddu566  [DOI](https://doi.org/10.1093/hmg/ddu566) | [PMC free article](/articles/PMC4381757/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25381333/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum.%20Mol.%20Genet.&title=Functional%20characterization%20of%20CYP2D6%20enhancer%20polymorphisms&author=D.%20Wang&author=A.%20C.%20Papp&author=X.%20Sun&volume=24&issue=6&publication_year=2015a&pages=1556-1562&pmid=25381333&doi=10.1093/hmg/ddu566&)

43. Wang D., Poi M. J., Sun X., Gaedigk A., Leeder J. S., Sadee W. (2014). Common CYP2D6 polymorphisms affecting alternative splicing and transcription: Long-range haplotypes with two regulatory variants modulate CYP2D6 activity. Hum. Mol. Genet. 23 (1), 268–278. 10.1093/hmg/ddt417  [DOI](https://doi.org/10.1093/hmg/ddt417) | [PMC free article](/articles/PMC3857955/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23985325/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum.%20Mol.%20Genet.&title=Common%20CYP2D6%20polymorphisms%20affecting%20alternative%20splicing%20and%20transcription:%20Long-range%20haplotypes%20with%20two%20regulatory%20variants%20modulate%20CYP2D6%20activity&author=D.%20Wang&author=M.%20J.%20Poi&author=X.%20Sun&author=A.%20Gaedigk&author=J.%20S.%20Leeder&volume=23&issue=1&publication_year=2014&pages=268-278&pmid=23985325&doi=10.1093/hmg/ddt417&)

44. Wang Y., Liu J., Huang B. O., Xu Y. M., Li J., Huang L. F., et al. (2015b). Mechanism of alternative splicing and its regulation. Biomed. Rep. 3 (2), 152–158. 10.3892/br.2014.407  [DOI](https://doi.org/10.3892/br.2014.407) | [PMC free article](/articles/PMC4360811/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25798239/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biomed.%20Rep.&title=Mechanism%20of%20alternative%20splicing%20and%20its%20regulation&author=Y.%20Wang&author=J.%20Liu&author=B.%20O.%20Huang&author=Y.%20M.%20Xu&author=J.%20Li&volume=3&issue=2&publication_year=2015b&pages=152-158&pmid=25798239&doi=10.3892/br.2014.407&)

45. Zanger U. M., Fischer J., Raimundo S., Stüven T., Evert B. O., Schwab M., et al. (2001). Comprehensive analysis of the genetic factors determining expression and function of hepatic CYP2D6. Pharmacogenetics Genomics 11 (7), 573–585. 10.1097/00008571-200110000-00004  [DOI](https://doi.org/10.1097/00008571-200110000-00004) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11668217/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics%20Genomics&title=Comprehensive%20analysis%20of%20the%20genetic%20factors%20determining%20expression%20and%20function%20of%20hepatic%20CYP2D6&author=U.%20M.%20Zanger&author=J.%20Fischer&author=S.%20Raimundo&author=T.%20St%C3%BCven&author=B.%20O.%20Evert&volume=11&issue=7&publication_year=2001&pages=573-585&pmid=11668217&doi=10.1097/00008571-200110000-00004&)

46. Zanger U. M., Momoi K., Hofmann U., Schwab M., Klein K. (2021). Tri-allelic haplotypes determine and differentiate functionally normal allele CYP2D6*2 and impaired allele CYP2D6*41. Clin. Pharmacol. Ther. 109 (5), 1256–1264. 10.1002/cpt.2078  [DOI](https://doi.org/10.1002/cpt.2078) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33043448/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Tri-allelic%20haplotypes%20determine%20and%20differentiate%20functionally%20normal%20allele%20CYP2D6*2%20and%20impaired%20allele%20CYP2D6*41&author=U.%20M.%20Zanger&author=K.%20Momoi&author=U.%20Hofmann&author=M.%20Schwab&author=K.%20Klein&volume=109&issue=5&publication_year=2021&pages=1256-1264&pmid=33043448&doi=10.1002/cpt.2078&)

47. Zhuge J., Yu Y. N. (2004). Three new alternative splicing variants of human cytochrome P450 2D6 mRNA in human extratumoral liver tissue. World J. Gastroenterol. 10 (22), 3356–3360. 10.3748/wjg.v10.i22.3356  [DOI](https://doi.org/10.3748/wjg.v10.i22.3356) | [PMC free article](/articles/PMC4572313/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15484318/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=World%20J.%20Gastroenterol.&title=Three%20new%20alternative%20splicing%20variants%20of%20human%20cytochrome%20P450%202D6%20mRNA%20in%20human%20extratumoral%20liver%20tissue&author=J.%20Zhuge&author=Y.%20N.%20Yu&volume=10&issue=22&publication_year=2004&pages=3356-3360&pmid=15484318&doi=10.3748/wjg.v10.i22.3356&)
