# Exploratory Genome-Wide Association Analysis to Identify Pharmacogenetic Determinants of Response to R-CHOP in Diffuse Large B-Cell Lymphoma

## Metadata
**Authors:** Gabriele Perrone, Luigi Rigacci, Sara Urru, Sofya Kovalchuk, Marco Brugia, Alberto Fabbri, Lorenzo Iovino, Benedetta Puccini, Emanuele Cencini, Enrico Orciuolo, Silvia Birtolo, Alessandro Melosi, Simone Santini, Ida Landini, Giandomenico Roviello, Raffaella Santi, Alessandra Macciotta, Fulvio Ricceri, Alberto Bosi, Monica Bocchia, Mario Petrini, Enrico Mini, Stefania Nobili
**Journal:** Cancers
**Date:** 2023 May 13
**DOI:** [10.3390/cancers15102753](https://doi.org/10.3390/cancers15102753)
**PMID:** 37345090
**PMCID:** PMC10216814
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10216814/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC10216814/pdf/cancers-15-02753.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC10216814/pdf/cancers-15-02753.pdf)

## Abstract

**Simple Summary:** 
The standard treatment of patients affected by diffuse large B-cell lymphomas (DLBCLs) is represented by a chemoimmunotherapeutic regimen (R-CHOP) that is successful in about 60% of patients. Currently, we cannot know in advance if their disease will respond to R-CHOP. This study aimed at identifying predictive genetic biomarkers for R-CHOP response through the analysis of a very high number of host genetic polymorphisms. Of the 216 enrolled chemonaïve patients candidate to R-CHOP, 185 were eligible. Highly statistically significant associations were found between progression-free survival and six polymorphisms (i.e., rs116665727, rs1607795, rs75614943, rs77241831, rs117500207, rs78466241), and between the overall survival (OS) and five polymorphisms (i.e., rs74832512, rs117500207, rs35789195, rs11721010, rs12356569). Overall, wild-type patients (i.e., without these polymorphisms) showed prolonged survival compared with polymorphic patients. In the future, these polymorphisms, alone or in combination, after a proper validation in a further number of patients, could contribute to improving the prediction of R-CHOP response.

**Abstract:** 
R-CHOP standard chemotherapy is successful in about 60% of diffuse large B-cell lymphoma (DLBCL) patients. Unresponsive patients have a poor prognosis, and predictive biomarkers of response to R-CHOP are lacking. We conducted the first prospective GWAS study aimed at exploring constitutional biomarkers predictive of R-CHOP efficacy and toxicity. Overall, 216 any-stage chemonaïve DLBCL patients candidate to R-CHOP were enrolled. The median age of the 185 eligible patients was 59.2 years, 49.7% were women and 45.4% were stage I–II patients. According to the Revised International Prognostic Index (R-IPI), 14.1%, 56.8% and 29.2% were in the very good, good and poor prognosis groups, respectively. Of the patients, 85.9% produced a complete response. Highly significant associations (i.e., p < 5 × 10−8) were found between progression-free survival (PFS) and six SNPs (i.e., rs116665727, rs1607795, rs75614943, rs77241831, rs117500207, rs78466241). Additionally, five SNPs (i.e., rs74832512, rs117500207, rs35789195, rs11721010, rs12356569) were highly associated with overall survival (OS). Wild-type patients showed a prolonged PFS or OS compared with patients carrying deleterious alleles (p < 0.001). No association with the adequate significant threshold was observed between SNPs and the objective response or toxicity. In the future, these SNPs, alone or in combination, after a proper validation in an independent cohort, could contribute to improving the prediction of R-CHOP response.

Keywords: diffuse large B-cell lymphoma, R-CHOP, predictive biomarkers, pharmacogenetics, single-nucleotide polymorphism, genome-wide association study

### Simple Summary

The standard treatment of patients affected by diffuse large B-cell lymphomas (DLBCLs) is represented by a chemoimmunotherapeutic regimen (R-CHOP) that is successful in about 60% of patients. Currently, we cannot know in advance if their disease will respond to R-CHOP. This study aimed at identifying predictive genetic biomarkers for R-CHOP response through the analysis of a very high number of host genetic polymorphisms. Of the 216 enrolled chemonaïve patients candidate to R-CHOP, 185 were eligible. Highly statistically significant associations were found between progression-free survival and six polymorphisms (i.e., rs116665727, rs1607795, rs75614943, rs77241831, rs117500207, rs78466241), and between the overall survival (OS) and five polymorphisms (i.e., rs74832512, rs117500207, rs35789195, rs11721010, rs12356569). Overall, wild-type patients (i.e., without these polymorphisms) showed prolonged survival compared with polymorphic patients. In the future, these polymorphisms, alone or in combination, after a proper validation in a further number of patients, could contribute to improving the prediction of R-CHOP response.

### Abstract

R-CHOP standard chemotherapy is successful in about 60% of diffuse large B-cell lymphoma (DLBCL) patients. Unresponsive patients have a poor prognosis, and predictive biomarkers of response to R-CHOP are lacking. We conducted the first prospective GWAS study aimed at exploring constitutional biomarkers predictive of R-CHOP efficacy and toxicity. Overall, 216 any-stage chemonaïve DLBCL patients candidate to R-CHOP were enrolled. The median age of the 185 eligible patients was 59.2 years, 49.7% were women and 45.4% were stage I–II patients. According to the Revised International Prognostic Index (R-IPI), 14.1%, 56.8% and 29.2% were in the very good, good and poor prognosis groups, respectively. Of the patients, 85.9% produced a complete response. Highly significant associations (i.e., *p* < 5 × 10^−8^) were found between progression-free survival (PFS) and six SNPs (i.e., rs116665727, rs1607795, rs75614943, rs77241831, rs117500207, rs78466241). Additionally, five SNPs (i.e., rs74832512, rs117500207, rs35789195, rs11721010, rs12356569) were highly associated with overall survival (OS). Wild-type patients showed a prolonged PFS or OS compared with patients carrying deleterious alleles (*p* < 0.001). No association with the adequate significant threshold was observed between SNPs and the objective response or toxicity. In the future, these SNPs, alone or in combination, after a proper validation in an independent cohort, could contribute to improving the prediction of R-CHOP response.

## 1. Introduction

Diffuse large B-cell lymphoma (DLBCL) is the most common type of adult non-Hodgkin’s lymphoma, representing about 40% of all newly diagnosed and 90% of aggressive lymphomas [[1](#B1-cancers-15-02753),[2](#B2-cancers-15-02753)]. DLBCL is a highly heterogeneous disease [[3](#B3-cancers-15-02753)]. Gene expression profiling (GEP) seminal studies identified categories according to the ‘cell-of-origin’ principle, that may help to stratify DLBCL patients with different prognoses [[3](#B3-cancers-15-02753),[4](#B4-cancers-15-02753),[5](#B5-cancers-15-02753),[6](#B6-cancers-15-02753)]. Germinal centre B-cell-like (GCB) DLBCL patients have a significantly better overall survival than those with an activated B-cell-like (ABC) DLBCL. A recent GEP study [[7](#B7-cancers-15-02753)] identified, at the single cell level, GCB cell subpopulations characterized by different signatures, thus providing more detailed assignments of GCB DLBCL able to recognize additional prognostic subgroups. In clinical practice, GEPs are replaced by immunohistochemical algorithms [[8](#B8-cancers-15-02753),[9](#B9-cancers-15-02753),[10](#B10-cancers-15-02753)], although their prognostic predictive power is lower compared to GEP [[9](#B9-cancers-15-02753)]. Studies on the genetics of tumour cells have identified further sub-sets of DLBCLs characterized by different alterations and outcomes [[11](#B11-cancers-15-02753)]. To date, the main recognized poor-prognosis genetic subtypes display tumour *MYC* and *BCL2* translocations (‘double-hit lymphoma’) or *MYC*, *BCL2* and *BCL6* translocations (‘triple-hit lymphoma) [[12](#B12-cancers-15-02753),[13](#B13-cancers-15-02753),[14](#B14-cancers-15-02753)].

Currently, a DLBCL outcome is clinically predicted by the International Prognostic Index (IPI) that has been revised (R-IPI) after the addition of the anti-CD20 monoclonal antibody rituximab to the CHOP regimen (doxorubicin, cyclophosphamide, vincristine, prednisone) [[15](#B15-cancers-15-02753)].

These prognostic biomarkers are useful to indicate long-term clinical outcomes for DLBCL patients, but they are not informative of the likelihood of patients to respond to R-CHOP in relation to the sensitivity/resistance of the tumour to the drug treatment.

Although R-CHOP significantly improved the outcomes of DLBCL patients compared to CHOP alone [[16](#B16-cancers-15-02753)], about 40% of DLBCL patients, due to tumour heterogeneity, is refractory or relapses after initial complete remission, and most of them ultimately die, with only a minority cured by the salvage therapy [[17](#B17-cancers-15-02753)]. Thus, DLBCL patients would require personalized treatment. However, at present, based on the fundamental clinical trials performed at the beginning of the rituximab era [[18](#B18-cancers-15-02753),[19](#B19-cancers-15-02753),[20](#B20-cancers-15-02753)], of which the results were successively confirmed in long-term studies [[21](#B21-cancers-15-02753),[22](#B22-cancers-15-02753)], the R-CHOP regimen represents the upfront standard of care for all DLBCL stages and for both GCB and non-GCB subtypes [[23](#B23-cancers-15-02753)]. Very recently, the modified regimen R-CHOP, represented by pola-R-CHP in which vincristine is replaced with the anti-CD79b antibody drug conjugate polatuzumab vedotin [[24](#B24-cancers-15-02753)], has been approved for the treatment of previously untreated DLBCL patients. A more aggressive first-line treatment is only recommended for the double/triple-hit lymphomas [[25](#B25-cancers-15-02753),[26](#B26-cancers-15-02753)]. The results of the “frontMIND” phase III trial that compares an intensified treatment (i.e., the anti-CD19 tafasitamab monoclonal antibody *plus* lenalidomide *plus* R-CHOP vs. R-CHOP) in newly diagnosed high-intermediate and high-risk DLBCL patients, will only be available in the next years [[27](#B27-cancers-15-02753)]. Tafasitamab and other anti-CD19 monoclonal antibodies, as well as the anti-CD19 CAR T-cell therapy, are currently approved only for the treatment of patients with recurrent or refractory DLBCL.

Overall, although a large amount of information is available on the mutational landscape of DLBCL [[14](#B14-cancers-15-02753),[28](#B28-cancers-15-02753)], recently integrated by an elegant study on genotyping of DLBCL mutations on liquid biopsies [[29](#B29-cancers-15-02753)], less information is available for host genetics in DLBCL.

Germline genetic variations are good candidates to characterize the role of the host in predicting response to drug treatments (antitumor activity, efficacy and host toxicity), since they are constitutional, stable over time and easy for evaluating. Compared with somatic mutations, host genetic variations are not affected by tissue heterogeneity and/or by the evolution of molecular changes that may ultimately lead to changes in molecular profiles within the tumour or during the different stages of tumour development. Pharmacogenetics is, in fact, a recognized discipline for predicting drug response in several tumour types [[30](#B30-cancers-15-02753)], including DLBC [[31](#B31-cancers-15-02753)]. According to a candidate gene-based approach, genes involved in pharmacokinetics/pharmacodynamics (PK/PD) of drugs used in DLBCL or genes associated with the immune function were found to be associated with R-CHOP efficacy [[31](#B31-cancers-15-02753),[32](#B32-cancers-15-02753)]. However, GWAS represents a hypothesis-free approach that increases the chance of identifying genetic variants in novel genes that cannot be easily included in candidate gene-based approaches. These genes may be involved in several relevant molecular processes comprising drug action and disposition, tumour drug resistance and pathobiology [[33](#B33-cancers-15-02753)], and thus play a potential role in drug response. On this basis, a number of GWAS studies have been performed to identify novel associations between drug efficacy and genetic variants in several cancers, including breast cancer [[34](#B34-cancers-15-02753)], lung cancer [[35](#B35-cancers-15-02753),[36](#B36-cancers-15-02753)], pancreatic cancer [[37](#B37-cancers-15-02753)], renal cell carcinoma [[38](#B38-cancers-15-02753)], hepatocellular carcinoma [[39](#B39-cancers-15-02753)], as well as DLBCL [[40](#B40-cancers-15-02753)].

To date, only one GWAS analysis has explored the constitutional DNA in DLBCL patients to identify associations between SNPs and clinical outcomes [[40](#B40-cancers-15-02753)], although patients analysed were not uniformly treated.

Thus, due to the substantial lack of data on constitutional biomarkers potentially predictive of response to R-CHOP in DLBCL patients, we designed an exploratory multicentre prospective pharmacogenetic study, based on GWAS analysis. The study primary endpoint was the identification of potential associations between polymorphisms and progression-free survival (PFS) or toxicity. The secondary endpoint was the identification of potential associations between polymorphisms and the overall survival (OS) or objective response.

## 2. Materials and Methods

### 2.1. Participants and Treatment

Study participants were enrolled in this GWAS prospective observational clinical trial from November 2012 to October 2017 at six Divisions of Haematology belonging to Italian university hospitals and first-level hospitals. The inclusion criteria were: signed informed consent; histologically confirmed diagnosis of DLBCL according to WHO classification systems [[41](#B41-cancers-15-02753),[42](#B42-cancers-15-02753)]; age 18–75 years; PS ECOG ≤ 2; patients eligible for standard R-CHOP; no previous chemotherapy and radiotherapy; availability of blood samples. Exclusion criteria were: a different diagnosis from DLBCL; liver or renal disfunction; previous neoplasms; contraindications for doxorubicin or vincristine treatment. The patient’s prognosis was evaluated according to the Revised IPI (R-IPI) [[15](#B15-cancers-15-02753)] and patients were classified in the following three prognostic groups: R-IPI 0, R-IPI 1–2, R-IPI 3–5, the R-IPI 0 and R-IPI 3–5 groups being those with a more favourable and unfavourable prognosis, respectively. Overall, 216 chemonaïve DLBCL patients (Ann Arbor stages I–IV) eligible for R-CHOP treatment were enrolled. A total of 187 patients were evaluable based on clinical, pathological and genetic information.

R-CHOP was administered as follows: rituximab 375 mg/m^2^ on day 0, cyclophosphamide 750 mg/m^2^ on day 1, doxorubicin 50 mg/m^2^ on day 1, vincristine 1.4 mg/m^2^ (max. 2 mg) on day 1, prednisone 50 mg/m^2^ days 1–5, every 21 days for 6 cycles.

The study was approved by the local IRB (Careggi University Hospital, Florence, Italy) of the coordinator centre (Prot. 2012/0033535) and by those of the participant centres. All patients provided written informed consent.

### 2.2. DNA Extraction and Genotyping Platform

Peripheral whole-blood samples were collected in EDTA vacutainer tubes before the first cycle of R-CHOP. Genomic DNA was extracted from peripheral blood according to standard procedures, by using the Puregene Blood Core Kit (Qiagen, Hilden, Germany). The DNA concentration and quality were evaluated with a Qubit 3.0 Fluorometer. The DNA integrity was assessed with an Agilent 2100 Bioanalyzer.

According to an Affymetrix-based methodology, total gDNA was amplified and, after fragmentation and hybridization, genotyped using the UK Biobank Axiom Array on a GeneTitan MC System INTL instrument (Thermo Fisher, Waltham, MA, USA). The array contained 820,967 SNPs. In this system, GWAS markers were selected using Affymetrix’ imputation-aware marker choice algorithms to provide the GW coverage of common (estimated minor allele frequency (EMAF) ≥ 5%) and low-frequency (1% < EMAF) markers. Array data were uploaded on the NCBI GEO repository as [GSE186441](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE186441).

### 2.3. Bioinformatic Analysis

Bioinformatic analysis to identify host genetic biomarkers associated with R-CHOP efficacy or toxicity was performed. A total of 187 study samples were analysed. Files of .CEL type were imported in the Axiom Analysis Suite (4.0.3 software) (Thermo Fisher) [[43](#B43-cancers-15-02753)] and processed using the Axiom UKB WCSG array type to evaluate sample and SNP quality.

Four quality metrics were used to assess the plate and sample quality [[44](#B44-cancers-15-02753)]: the plate pass rate (PPR) and average plate control rate (APCR) filters were applied to plates, whereas the dish quality control (DQC) and quality control call rate (QCCR) metrics were used to evaluate the sample quality ([Supplementary Materials](#app1-cancers-15-02753)).

Confidence scores (CS) defined as CS = 1 − *p*, where *p* is the posterior probability that the point belongs to the assigned genotype cluster, were calculated to assess the genotype call quality. The CS ranges in [0, 1] and values above the 0.15 threshold generate No Calls.

Sex was automatically assessed by the analysis tool. In the case of unknown sex assignment, X and Y chromosome intensities and heterozygosity rates were checked.

Allele calling and quality control of the genotyping data were performed on a total of 836,727 probesets. In the case of more than one probeset interrogating the same SNP, the best probeset was chosen according to Axiom Analysis Suite metrics, and then the SNP quality was checked. In particular, monomorphic SNPs, SNPs with a call rate < 95%, minor allele frequency < 1%, and those that significantly deviated (*p* < 10^−10^) from the Hardy–Weinberg equilibrium were excluded. To evaluate the SNP quality, Fisher’s linear discriminant, homozygous ratio offset and heterozygous strength offset default thresholds were also considered. Principal components analysis was performed to identify the eventual substructure in the population. Finally preliminary association tests were carried out considering only the samples and SNPs passing quality filters.

### 2.4. Patient Population and Quality Control of Genotypic Data

Of the 216 patients enrolled, 13 were not evaluable due to the lack of sufficient clinical information. In addition, 3 patients who interrupted the treatment early, 2 patients who underwent a treatment other than the R-CHOP regimen, 1 who was affected by follicular lymphoma, and 2 patients who were not of Caucasian origin, were excluded. Eight patients were excluded due to sample and/or DNA degradation. A total of 187 study samples were analysed, considering ~800,000 probesets.

Study samples were distributed into four plates: three were defined as being of high quality because of PPR ≥ 95% and APCR > 99%, the remaining one had PPR = 92.31%. The DQC and QCCR metrics were used to evaluate the sample quality and, because of the low QCCR (<97%), two samples were excluded from the subsequent analysis. The median CS was in the order of 10^−5^, confirming the high-quality genotype calls.

Two samples were identified with an unknown sex, so X and Y chromosome intensities and heterozygosity rates were checked. This analysis led to the male sex for both samples, in agreement with the collected data. PCA identified two components which were used to adjust the association tests.

### 2.5. R-CHOP Efficacy and Toxicity Evaluation

Objective response was evaluated during (e.g., third cycle) and at the end (sixth cycle) of the R-CHOP treatment by the Cheson standardized response criteria [[45](#B45-cancers-15-02753)] until 2014, and subsequently by the Cheson revised response criteria [[46](#B46-cancers-15-02753)]. According to the Cheson criteria, the treatment response was classified as complete remission (CR), partial remission (PR), stable disease (SD) and progression disease (PD). The PFS duration was calculated from the date of diagnosis until the PD or last follow-up, whereas the OS duration from the date of the diagnosis until death or the last follow-up.

The R-CHOP toxicity was evaluated according to the NCI-CTCAE version 4.03 criteria.

### 2.6. Statistical Analysis

Treatment efficacy was measured *via* PFS, OS and objective response.

To identify correlations between clinical/pathological characteristics or SNPs and the study endpoints, Cox (PFS and OS) and logistic regressions (objective response) were used. For correlation studies, patients were grouped as complete response (CR) and not CR (i.e., partial response, stable disease, progression) patients. The age was analysed as a continuous variable.

Five types of toxicity were considered as a coprimary endpoint: neutropenia, haematological toxicity, gastrointestinal toxicity, infection and maximum toxicity (i.e., the highest G among the five study types of toxicity). To test the SNP association, two logistic regression models were performed for every type of toxicity: the first one considered toxicity (grades from 1 to 4, i.e., G1–G4) versus no toxicity (grade 0, i.e., G0), and the second considered severe toxicity (grades 3 and 4, i.e., G3–G4) versus low toxicity (grades 0, 1 and 2, i.e., G0–G2).

The models were then adjusted for principal components, sex, R-IPI and all of them and Manhattan plots were generated to show the association results. The models adjusted for age and stage were not considered, as these parameters were included in the R-IPI.

GWAS analysis were performed using the GWASTools package of the statistical software R version 4.0.2 (R Core Team (2020)—[https://www.r-project.org/](https://www.r-project.org/), accessed on 15 December 2022).

The Kaplan–Meier survival plots were generated for the SNPs that were found to be associated with PFS and/or OS through bioinformatic analysis with *p*-values < 5 × 10^−8^. *p* < 0.05 was considered statistically significant; the statistical analysis was performed using the SPSS software v.26 (IBM, Armonk, NY, USA).

### 2.7. Functional Enrichment and Network Analysis

The lists of SNPs associated with the PFS or OS, with or without adjustment for sex and/or R-IPI, were uploaded into the DAVID bioinformatics tool ([https://david.ncifcrf.gov/](https://david.ncifcrf.gov/), accessed on16 January 2023) to perform functional enrichment analysis, or into Metacore^TM^ to perform network analysis. The latter is a web-based bioinformatics suite that allows researchers to upload data analysis results from genomic experiments and provides functional analysis to identify the most relevant pathways, networks and cellular processes in the data. The shortest path function was applied allowing for one intermediary to connect the uploaded elements.

## 3. Results

### 3.1. Relationships between Efficacy Parameters and Clinical/Pathological Characteristics

Overall, 185 patients were evaluable. Clinical and pathological characteristics are listed in [Table 1](#cancers-15-02753-t001). Sex was well balanced (i.e., 50.3% men). At presentation, 45.4% of patients were early-stage (stage I–II), 70.8% had good or very good R-IPI. The performance status was 0 in 62.2% of the cases. The median follow-up was 45 months, and the median PFS and OS were not reached at the time of the analysis. The objective response was observed in 176 patients (95.1%) (i.e., CR 89%, *n* = 159; PR 9.2%, *n* = 17). One patient (0.5%) showed SD and eight patients (4.3%) showed PD.

### Table 1.

|   |   | No. |
| --- | --- | --- |
| No. |   | 185 |
| Age (mean (SD)) |   | 59.17 (13.56) |
| Sex | Male | 93 (50.3) |
|   | Female | 92 (49.7) |
| Disease stage (%) | I | 20 (10.8) |
|   | II | 64 (34.6) |
|   | III | 39 (21.1) |
|   | IV | 62 (33.5) |
| R-IPI (%) | Very good (0) | 26 (14.1) |
|   | Good (1–2) | 105 (56.8) |
|   | Poor (3–5) | 54 (29.2) |
| “B” symptoms (%) | Yes | 41 (22.2) |
|   | No | 139 (75.1) |
|   | Missing | 5 (2.7) |
| Bulky disease (%) | Yes | 58 (31.4) |
|   | No | 122 (65.9) |
|   | Missing | 5 (2.7) |
| Bone marrow involvement (%) | Yes | 23 (12.4) |
|   | No | 140 (75.7) |
|   | Missing | 22 (11.9) |
| Performance status (ECOG) (%) | 0 | 115 (62.2) |
|   | 1 | 58 (31.4) |
|   | 2 | 12 (6.5) |

Table 1 Caption: Clinical and pathological characteristics of patients.

According to Cox regression analysis, PFS was significantly associated with sex (more prolonged for female), R-IPI (shorter for R-IPI 3–5) and stage (shorter for stage III–IV) ([Figure 1](#cancers-15-02753-f001)). The OS was associated with sex (i.e., prolonged in females), age (i.e., shorter in older patients), R-IPI (shorter for R-IPI 3–5) and stage (shorter for stage III–IV). No other clinical parameter was found to be associated with the PFS or OS. The objective response was significantly associated with the stage and R-IPI; in particular, stage III–IV and R-IPI-poor patients had a worse response (odds ratio (OR) 0.39, CI 0.15–0.95, *p* = 0.046 and OR 0.37, CI 0.14–0.92, *p* = 0.032, respectively). No significant association was found between the objective response and sex or age ([Supplementary Materials](#app1-cancers-15-02753)).

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/04eb/10216814/c29a6c82c219/cancers-15-02753-g001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=10216814_cancers-15-02753-g001.jpg)

Kaplan–Meyer curves of progression-free survival according to the clinical/pathological characteristics of DLBCL patients (n = 185). (A) Sex (log rank test, p = 0.002; Cox regression: HR 0.34; CI 0.17–0.69; p = 0.003); (B) stage (log rank test, p = 0.016; Cox regression: HR 3.12; CI 1.48–6.59; p = 0.003); (C) R-IPI (log rank test, p = 0.002; Cox regression: HR 2.55; CI 1.34–4.88; p = 0.005).

### 3.2. Relationships between Efficacy Parameters and SNPs

Overall, highly significant associations (i.e., *p* < 5 × 10^–8^) were found between PFS and six SNPs (i.e., rs116665727, rs1607795, rs75614943, rs77241831, rs117500207 and rs78466241) after R-IPI and/or sex-unadjusted or -adjusted analyses. Similarly, five SNPs (i.e., rs74832512, rs117500207, rs35789195, rs11721010 and rs12356569) were highly associated with the OS after the same unadjusted or adjusted analyses ([Supplementary Materials](#app1-cancers-15-02753)). Wild-type patients showed a prolonged PFS and OS compared with patients carrying deleterious alleles (*p* < 0.001). No association with the adequate significant threshold was observed between the SNPs and the objective response, with the *p*-value of the highest associations in the order of 10^−5^ ([Supplementary Materials](#app1-cancers-15-02753)).

#### 3.2.1. PFS Associations

Preliminary association tests (i.e., unadjusted analysis) with the PFS identified four SNPs reaching the significance threshold (*p* < 5 × 10^–8^): rs116665727 in the *TNIP3* gene (HR = 6.77, *p* = 1.0 × 10^−9^), rs1607795 in the *LNC00882* gene (HR = 18.56, *p* = 1.3 × 10^−8^), rs75614943 in the *CIDEA* gene (HR = 15.73, *p* = 1.9 × 10^−8^) and rs77241831 in the *NOS1* gene (HR = 6.66, *p* = 3.1 × 10^−8^). Twelve further SNPs were identified with a lower, but still relevant, significant association level (from *p* ≥ 5 × 10^−8^ to *p* = 10^−7^) ([Figure 2](#cancers-15-02753-f002); [Supplementary Materials](#app1-cancers-15-02753)).

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/04eb/10216814/4e33f71f423a/cancers-15-02753-g002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=10216814_cancers-15-02753-g002.jpg)

Progression-free survival according to the SNPs that reached the adequate GWAS statistical significance (p < 5 × 10–8). Wild-type patients showed a prolonged PFS compared with polymorphic patients. § Unadjusted analysis; # adjusted for R-IPI; ¶ adjusted for sex and R-IPI.

Two out of these four SNPs, i.e., rs75614943 in the *CIDEA* gene (HR = 20.67, *p* = 7.0 × 10^−9^) and rs77241831 in the *NOS1* gene (HR = 6.58, *p* = 4.5 × 10^−8^) were also highly associated with the PFS after R-IPI adjustment. In addition to SNPs in the *CIDEA* and *NOS1* genes, a third SNP, i.e., rs117500207 in the *PPP6R3* gene, was found to be associated with the PFS after R-IPI adjustment (HR = 24.38, *p* = 2.6 × 10^−8^) ([Figure 2](#cancers-15-02753-f002); [Supplementary Materials](#app1-cancers-15-02753)).

Two additional SNPs, rs1607795 in the *LNC00882* and rs74832512 in the *SLC35F4* gene, were exactly on (HR = 15.98, *p* = 5.0 × 10^−8^) and just above (HR = 30.69, *p* = 7.9 × 10^−8^) the threshold, respectively ([Figure 2](#cancers-15-02753-f002) and [Supplementary Materials](#app1-cancers-15-02753)), after adjustment for R-IPI. A further nine SNPs were associated with *p*-values ranging from 5x10^−8^ to 10^−7^.

Adjusting the analysis for R-IPI and sex, one SNP was highly statistically associated with the PFS, i.e., the rs78466241 in *CSRP3-AS1* intron variant (HR = 32.42, *p* = 6.0 × 10^−9^) ([Figure 2](#cancers-15-02753-f002) and [Supplementary Materials](#app1-cancers-15-02753)), whereas two further SNPs, i.e., rs12356569 in the *LYZL1* (HR = 27.64, *p* = 5.1 × 10^−8^) and rs68362253 in the *PPP6R3* (HR = 22.08, *p* = 6.8 × 10^−8^) genes, were just above the threshold ([Supplementary Materials](#app1-cancers-15-02753)).

#### 3.2.2. OS Associations

By preliminary association tests between the OS and SNPs, only one SNP reached the significance threshold: rs74832512 in the *SLC35F4* gene (HR = 26.48, *p* = 1.7 × 10^−8^). This result was also confirmed by adjusting the analysis for R-IPI (HR = 56.03, *p* = 7.0 × 10^−9^), sex (HR = 30.63, *p* = 2.3 × 10^−8^) and R-IPI *plus* sex (HR = 53.91, *p* = 3.4 × 10^–8^) ([Figure 3](#cancers-15-02753-f003) and [Supplementary Materials](#app1-cancers-15-02753)). After adjustment for sex, the association between rs11721010 in the *OXNAD1* gene and the OS was also highly significant (HR = 20.30; *p* = 2.7 × 10^−8^) ([Figure 3](#cancers-15-02753-f003) and [Supplementary Materials](#app1-cancers-15-02753)).

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/04eb/10216814/e925ff844de9/cancers-15-02753-g003.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=10216814_cancers-15-02753-g003.jpg)

Overall survival according to the SNPs that reached the adequate GWAS statistical significance (p < 5 × 10–8). Wild-type patients showed a prolonged OS compared with polymorphic patients. § Unadjusted analysis; # adjusted for R-IPI; † adjusted for sex; ¶ adjusted for sex and R-IPI.

Among the SNPs associated with the OS after adjusting for R-IPI, in addition to rs74832512 in the *SLC35F4* gene, two other SNPs showed an adequate GWAS *p*-value: rs117500207 in the *PPP6R3* (HR = 25.46, *p* = 2.6 × 10^−8^) and rs35789195 in the *CDH3* genes (HR = 9.65, *p* = 4.9 × 10^−8^). Just below the threshold, a further SNP, rs61759901 on gene *RGS11* (HR = 57.30, *p* = 5.1 × 10^−8^), was associated with the OS ([Figure 3](#cancers-15-02753-f003) and [Supplementary Materials](#app1-cancers-15-02753)).

When the analysis was adjusted for R-IPI and sex, in addition to rs74832512 in the *SLC35F4* gene, a further SNP, i.e., rs12356569 in the *LYZL1* gene (HR = 39.70, *p* = 4.0 × 10^−8^), was identified with adequate statistical significance ([Figure 3](#cancers-15-02753-f003) and [Supplementary Materials](#app1-cancers-15-02753)). Interestingly, among the five SNPs significantly associated with the OS, in addition to rs117500207 in the *PPP6R3* gene that was also significantly associated (*p* < 5 × 10^−8^) with the PFS, rs74832512 in the *SLC35F4* and rs12356569 in the *LYZL1* genes were associated with the PFS, with a *p*-value > 5 × 10^−8^–10^−7^. rs35789195 in the *CDH3* and rs11721010 in the *OXNAD1* genes were associated with the PFS, with *p*-values of 1.3 × 10^−6^ and 1.4 × 10^−6^, respectively ([Supplementary Materials](#app1-cancers-15-02753)).

### 3.3. Relationships between Toxicity and Clinical/Pathological Characteristics

Overall, G1–4 toxicity was reported in 80.5% of the patients. The comparisons between G0–2 and G3–4 total toxicity, neutropenia, hematological toxicity showed an association with age with a higher risk in older patients compared with younger patients (*p* < 0.01). The comparison between G0 and G1–4 toxicity confirmed the association with age (*p* = 0.004) ([Supplementary Materials](#app1-cancers-15-02753)).

### 3.4. Relationships between Toxicity and SNPs

When preliminary association tests were performed comparing G0 against G1–4 toxicity, the smaller *p*-value was 10^−5^. The same level of association was observed when adjusting for sex or R-IPI, whereas adjusting for R-IPI and sex, the smaller *p*-value was 10^−6^ ([Supplementary Materials](#app1-cancers-15-02753)).

When the association analysis was performed between toxicity (G0–2 vs. G3–4) and SNPs, no SNP reached the requested GWAS statistical significance, with the smaller *p*-values around 10^−6^, even after sex and/or R-IPI corrections ([Supplementary Materials](#app1-cancers-15-02753)).

No association was observed between the SNPs and hematological toxicity, gastrointestinal toxicity or infection.

### 3.5. Functional Enrichment and Network Analysis

To better characterize the biological role of the genes linked to the SNPs associated with survival, functional enrichment and network analysis were performed taking into consideration not only genes of which the associations reached *p* < 5 × 10^−8^, but also those reaching *p* < 10^−7^. Overall, 14 and 13 genes were considered for PFS and OS, respectively. The list of genes linked to PFS-associated SNPs is mostly enriched in the regulation of chromosome organization (GO:0033044, with *PHF2*, *DYNC1LI*, *NOS1*), histone modifications (GO:0016570, with *PHF2*, *PBXIP1* and *NOS1*), the regulation of nucleobase-containing compound metabolic process (GO:0019219, with *PHF2*, *TNIP3*, *CIDEA*, *PBXIP1*, *NOS1*, *TRIM34)* and processes related with the positive regulation of transcription, with the *PHF2*, *TNIP3*, *PBXIP1* and *NOS1* genes. Interestingly, the phagosome pathway was overrepresented as well, with the *DYNC1LI1* and *NOS1* genes. Gene network analysis also revealed that all 14 genes, except for *TRIM34*, are interconnected by addition of only one intermediary, with *PHF2* and *PBXIP1* being the most interconnected ([Figure 4](#cancers-15-02753-f004)).

### Figure 4.

![Figure 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/04eb/10216814/3cd5fdff56a4/cancers-15-02753-g004.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=10216814_cancers-15-02753-g004.jpg)

Network analysis of genes of which the SNPs were associated to the PFS, with p-values lower than 10−7 (genes included in the analysis: TNIP3, CIDEA, NOS1, NPC1L1, DYNC1LI1, PBXIP1, PHF2, LOC339568, SLC35F4, TRIM34, TRPC7, ACTL8, FAM169B, LOC101927873) (MetacoreTM software).

Only one GO biological process was overrepresented within the list of genes linked to OS-associated SNPs: the regulation of signal transduction (GO:0009966, with *CDH3*, *AFAP1*, *RGS11*, *NOS1*, *TRIM34*).

## 4. Discussion

The high heterogeneity of DLBCL that has been shown at genetic, transcriptomic and immunophenotypic levels, reflects the variability in DLBCL prognosis and drug response [[14](#B14-cancers-15-02753)]. However, despite this knowledge, the prediction of prognosis for DLBCL is mainly based on IPI that includes clinical, pathological and haemato-chemical factors, but not molecular factors. Thus, integrating the current factors with molecular factors could improve the predictive value of IPI. To date, the analysis of specific biomarkers of prognosis drives the treatment of double/triple-hit lymphoma patients, to whom more aggressive treatments (e.g., dose-adjusted EPOCH-R—etoposide, prednisone, vincristine, cyclophosphamide, doxorubicin, and rituximab) may be offered [[23](#B23-cancers-15-02753),[25](#B25-cancers-15-02753)]. Otherwise, independently of the IPI score, the R-CHOP regimen is the recommended first-line chemo-immunotherapy for most DLBCL patients, based on its higher efficacy compared with CHOP alone [[18](#B18-cancers-15-02753),[19](#B19-cancers-15-02753),[20](#B20-cancers-15-02753)].

Nevertheless, about 40% of patients will relapse after R-CHOP first-line treatment.

To date, molecular prognostic and/or predictive biomarkers have been mainly studied at the tumour level [[47](#B47-cancers-15-02753),[48](#B48-cancers-15-02753),[49](#B49-cancers-15-02753)]. Host genetics has been, instead, poorly investigated in terms of DLBCL prognosis and therapeutic outcome. The available studies are substantially based on a candidate gene approach, and few polymorphisms in genes involved in the ADME of R-CHOP drugs or in immunity, inflammation and apoptotic processes have been identified as potential predictors of DLBCL outcomes [[31](#B31-cancers-15-02753),[32](#B32-cancers-15-02753)]. However, these data, commonly obtained in retrospective studies, need validation in prospective clinical trials. Currently, no prospective trials specifically designed to investigate SNP prediction of R-CHOP outcomes is ongoing ([www.clinicaltrials.gov](http://www.clinicaltrials.gov), accessed on 4 May 2023). Today, only one GWAS analysis investigating the relationships between host genetics and immunochemotherapy in DLBCL patients not uniformly treated, is available [[40](#B40-cancers-15-02753)]. It is based on a meta-analysis of GWAS study datasets, in which some associations were identified. The top loci for event-free survival (EFS) were marked by rs7712513 at 5q23.2 (*p* = 2.08 × 10^−7^), and rs7765004 at 6q21 (*p* = 7.09 × 10^−7^). Both SNPs were also associated with OS (*p* = 3.53 × 10^−8^ and 5.36 × 10^−8^, respectively). These authors also performed an exploratory analysis by building a two-SNP risk score that was found to be highly predictive of EFS (*p* = 1.78 × 10^−12^).

Thus, to contribute to the current knowledge on the pharmacogenetics of DLBCL, we designed this exploratory pharmacogenetic prospective multicentre study based on an unsupervised analysis of constitutional SNPs, and aimed at identifying SNPs potentially predictive for R-CHOP efficacy.

Through bioinformatic analysis, we identified a total of six SNPs highly associated (*p* < 5 × 10^−8^) with the PFS (i.e., *TNIP3*, *LNC00882*, *CIDEA*, *NOS1, PPP6R3*, *CSRP3-AS1*) and five SNPs with the OS (i.e., *SLC35F4*, *PPP6R3*, *CDH3*, *OXNAD1*, *LYZL1*). One of them, rs117500207 in the *PPP6R3* gene, was found to be highly associated with both the PFS and OS. Some SNPs highly associated with the PFS were also associated with the OS, although with a slightly higher *p*-value (*p* ≤ 10^−7^) (e.g., *NOS1*), and all the SNPs highly associated with the OS were associated with the PFS with a high trend (i.e., *LYZL1*, *p* = 5 × 10^−8^; *SLC35F4 p* = 7.9 × 10^−8^) or *p*-values equal to 10^−6^ ([Supplementary Materials](#app1-cancers-15-02753)). However, the reason for which only one SNP (i.e., rs117500207 in *PPP6R3*) reached the *p*-value planned cut-off in both PFS and OS may be due to the fact that among the relapsed patients, only 64% had died at the time of the analysis. Thus, the lower number of events observed in OS (death) compared with PFS (progression), may have provided the differences in *p*-values observed for the associations between the same SNPs and the two survival parameters. Among genes of which the SNPs were associated with the clinical outcomes of this cohort of patients, some have been previously related to cancer, including lymphomas (e.g., *CDH3*, *OXNAD1*). However, none of the ten genes of which the SNPs were associated with survival parameters have never been found to be mutated in ctDNA from DLBCL) [[50](#B50-cancers-15-02753),[51](#B51-cancers-15-02753),[52](#B52-cancers-15-02753)]. The rs77241831 SNP in the *NOS1* gene was found to be highly associated with the PFS (*p* < 5 × 10^−8^) after unadjusted analysis and R-IPI adjustment. NOS1 belongs to the family of nitric oxide synthases that synthesize nitric oxide (NO) starting from L-arginine. The evaluation of NOS isoform expression levels in B-cell non-Hodgkin lymphoma tissues showed NOS1 more frequently expressed compared with NOS2 and NOS3 (94% of cases vs. 87% vs. 6%, respectively) [[53](#B53-cancers-15-02753)]. Interestingly, some studies highlighted associations between NOS genetic polymorphisms and non-Hodgkin lymphoma risk [[54](#B54-cancers-15-02753),[55](#B55-cancers-15-02753)].

The SNP, rs75614943, in the cell death inducing DFFA (DNA fragmentation factor subunit Alpha), such as effector A (*CIDEA),* gene was highly associated with PFS after preliminary analysis and R-IPI adjustment (*p* < 5 × 10^−8^). The CIDEA protein has been shown to play a role in lipolysis in animal models and humans [[56](#B56-cancers-15-02753)], and has also been suggested to play a role in human cancer cachexia [[57](#B57-cancers-15-02753)].

As far as the SNPrs116665727 in the *TNIP3* gene is concerned, it was found to be highly associated with the PFS after preliminary analysis (*p* < 5 × 10^−8^). A number of autoimmune diseases have been associated with the TNIP3 protein and related mutations [[58](#B58-cancers-15-02753)].

The remaining three genes of which the SNPs were significantly associated with the PFS were long intergenic non-protein coding RNA 882 (*LNC00882*), cysteine and glycine rich protein 3, and E2F8 antisense RNA 1 (*CSRP3-AS1*) and *PPP6R3*. No information on the biological and/or pathological role of *CSRP3-AS1* is available, while a relationship between *LNC00882* and pathological processes, including cancer (e.g., colorectal cancer) [[59](#B59-cancers-15-02753)] has been suggested.

The rs117500207 SNP in *PPP6R3* gene was found to be highly associated to both the PFS and OS after R-IPI adjustment. PPP6R3 is a regulatory subunit for the PPP6 catalytic subunit that is involved in the turnover of serine and threonine phosphorylation events during mitosis. This protein is ubiquitary-expressed in human tissues and highly expressed in immune cells and lymphoid tissues. The *PPP6R3* gene was recently included in a 95-gene blood transcriptome signature that represents a link between *C. burnetii*, the agent that causes Q fever, and non-Hodgkin lymphoma [[60](#B60-cancers-15-02753)].

In addition to *PPP6R3*, other four genes were significantly associated with the OS, with or without adjustment for R-IPI and/or sex (i.e., rs74832512 in *SLC35F4)*, after adjustment for sex (i.e., rs11721010 in *OXNAD1*), after adjustment for R-IPI (i.e., rs35789195 in *CDH3*), after adjustment for R-IPI and sex (i.e., rs12356560 in *LYZL1*).

The *SLC35F4* gene codifies the SLC35F4 transporter belonging to the solute carrier 35 (SLC35) transporter family. Overall, *SLC35F4* has been poorly investigated. Available information derived from a complex miRNA–mRNA analysis in different lymphoma types, including DLBCL, reports that this gene is targeted by the hsa-miR-4524a-3p through a miRNA–mRNA interaction with negative correlation [[61](#B61-cancers-15-02753)], thus suggesting that this gene is part of the regulatory pathways relevant in lymphomagenesis.

*OXNAD1* (oxidoreductase NAD-binding domain containing 1) gene codifies the oxidoreductase NAD-binding domain-containing protein 1 protein, an ubiquitous protein of which the main expression was detected in lymph nodes. Interestingly, aberrations in this gene (2p24.3) have been found in translocation-positive follicular lymphoma [[62](#B62-cancers-15-02753)]. However, no information is available in relation to the potential role of the *OXNAD1* gene in drug efficacy of lymphomas or other cancers.

The SNP, rs12356569 in the *LYZL1* (lysozyme like 1) gene was found to be highly associated with the OS, and with a very high trend with the PFS (i.e., *p* = 5 × 10^−8^) after adjustment for sex and R-IPI. The expression of the LYZL1 protein has been detected in a small number of cancer tissues, mainly ovarian and liver cancers, and not in lymphomas. Overall, its role in disease or cancer has not been characterized, while the potential role of *LYZL1* in anticancer drug resistance has been suggested. In fact, in a recent study, *LYZL1* gene expression was found to vary according to the expression of two biomarkers that have been suggested to be involved in ovarian cancer paclitaxel resistance (i.e., low *MAD2*/high *TLR4* expression: increased expression of *LYZL1*; high *MAD2*/low *TLR4* expression: decreased expression of *LYZL1*) [[63](#B63-cancers-15-02753)].

The *CDH3* (cadherin 3) gene codifies placenta (P-) cadherin, a Ca^2+^-dependent cell–cell adhesion protein, belonging to the cadherin superfamily. As known, cadherins in addition to their physiological roles [[64](#B64-cancers-15-02753)] also play a recognized role in carcinogenesis, tumour invasion and metastasis [[64](#B64-cancers-15-02753),[65](#B65-cancers-15-02753)].

In relation to survival parameters, we also performed a network analysis that included all genes associated with survival, with *p* < 10^−7^ (i.e., 14 genes with the PFS and 13 genes with the OS). The aim was to obtain information on the biological processes in which these genes were involved regardless of a forced selection, based on the GWAS *p*-value cut-off (*p* < 5 × 10^−8^) that could underestimate the relevant biological information. This knowledge that informs of the common or related biological functions of genes belonging to the same connected component of the network also allows to highlight the co-regulated genes that could play a role in the clinical outcome also by their potential involvement in the mechanisms of action/resistance of the administered drugs. The biological processes that we identified were mainly related to histone and chromatin modifications, the positive regulation of RNA (i.e., transcription, synthetic and metabolic processes) and the regulation of signal transduction. These observations suggest a crosstalk between signal transduction and chromatin and/or epigenetic modifications that may ultimately reflect changes in RNA levels. Overall, the genes identified could contribute to refine genetically altered transcriptional regulators, as well as epigenetic regulators that, although recognized as relevant in B-cell lymphomas [[66](#B66-cancers-15-02753)], have been substantially poorly investigated.

Finally, no association endowed by the adequate statistical significance was found between the SNPs and toxicity or objective response. However, although the observed side effects were substantially expected, SNPs that have been associated with toxicity with 10^−6^ *p*-values should be further investigated in an independent cohort of patients to better understand their potential role in R-CHOP toxicity independently of the achievement of the GWAS significant *p*-value. The lack of associations with the objective response could be related to the very early determination of this efficacy parameter. In fact, the analysis took into consideration the objective response established at the end of the immunochemotherapy treatment, when DLBCL is usually well controlled.

To the best of our knowledge, this is the first prospective study designed with the aim of evaluating the associations between SNPs and R-CHOP response by GWAS. Thus, it overcomes the potential limits of retrospective analyses. However, we acknowledge some limitations in this study; for instance, the absence of an independent replication data set. This was due to the difficulty to find a cohort of DLBCL patients whose DNA was available, uniformly treated with R-CHOP, with adequate clinical information (treatment efficacy/toxicity data) and a long follow-up. In addition, since only another GWAS study performed in non-Hodgkin lymphoma patients took into consideration pharmacogenetic relationships [[40](#B40-cancers-15-02753)], no data are currently available in public repositories and this also hampered the possibility to carry out an in silico validation. For this reason, a future development of this work could be the enrolment of a new cohort of DLBCL patients to validate the identified SNPs.

## 5. Conclusions

Overall, our exploratory study acquires relevance in the framework of the pharmacological treatment of this tumour that, as mentioned, is currently substantially limited to R-CHOP in the first-line treatment. In our opinion, among the ten identified constitutional genetic variations, those that will be successfully validated in an independent cohort of DLBCL patients treated with R-CHOP could integrate the currently known clinical and tumour genetic/genomic risk factors that are used at present following first-line chemo-immunotherapy, to identify patients candidate to intensified treatments in order to circumvent tumour drug resistance upfront. These SNPs could also be helpful in characterizing the new mechanisms of DLBCL drug resistance and contribute to the implementation of personalized treatments through the use of potential not cross-resistant drug combination regimens.

In conclusion, our exploratory analysis discovered novel SNPs potentially predictive for R-CHOP efficacy in DLBCL patients, thus providing support to the hypothesis that the implementation of pharmacogenetics in DLBCL may contribute to a personalized patient treatment selection in the future.

## Supplementary Materials

The following supporting information can be downloaded at: [https://www.mdpi.com/article/10.3390/cancers15102753/s1](https://www.mdpi.com/article/10.3390/cancers15102753/s1). Refs. [[67](#B67-cancers-15-02753),[68](#B68-cancers-15-02753),[69](#B69-cancers-15-02753)] are cited in Supplementary Materials.

## Author Contributions

Conceptualization: L.R., A.B., E.M. and S.N. conceived and designed the study; Methodology: G.P., I.L. and S.N. performed experiments. Formal analysis: G.P., S.U., A.M. (Alessandra Macciotta) and S.N. analyzed data. Validation: G.P., S.U., A.M. (Alessandra Macciotta), F.R., E.M. and S.N. interpreted data. Investigation: L.R., A.F., E.O., S.B., A.M. (Alessandro Melosi), S.S., A.B., M.B. (Monica Bocchia), M.P. enrolled patients, provided patient’s samples and data and R.S. provided pathological information. Data curation: G.P., S.K., M.B. (Marco Brugia), L.I., B.P., E.C., G.R. collected and cured patient’s data. Supervision: L.R., A.B., E.M. and S.N. supervised the study. Writing—original draft preparation: G.P., S.U., E.M. and S.N. Writing—review and editing: G.P., L.R., S.U., A.F., A.B., M.B. (Monica Bocchia), M.P., F.R., E.M. and S.N. revised the manuscript. Funding acquisition: E.M. and S.N. provided funds. All authors have read and agreed to the published version of the manuscript.

## Institutional Review Board Statement

The study was conducted in accordance with the Declaration of Helsinki, and approved by the Institutional Review Board of the Careggi University Hospital (Florence, Italy) of the coordinator centre (Prot. 2012/0033535) and by those of the participant centres.

## Informed Consent Statement

Informed consent was obtained from all subjects involved in the study.

## Data Availability Statement

Array data are available on the NCBI GEO repository as [GSE186441](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE186441).

## Conflicts of Interest

The authors declare no conflict of interest.

## Funding Statement

This research was funded by the Associazione Giacomo Onlus 2012–2020 (Castiglioncello, Italy) (to E.M.), Fondazione Cassa di Risparmio di Firenze—ref. no. 2013.0842 and ref. no. 2014.0969 (Firenze, Italy) (to S.N.).

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

Array data are available on the NCBI GEO repository as [GSE186441](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE186441).

### Supplementary Materials

### Data Availability Statement

Array data are available on the NCBI GEO repository as [GSE186441](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE186441).

## References

1. Siegel R.L., Miller K.D., Wagle N.S., Jemal A. Cancer statistics, 2023. CA Cancer J. Clin. 2023;73:17–48. doi: 10.3322/caac.21763.  [DOI](https://doi.org/10.3322/caac.21763) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36633525/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=CA%20Cancer%20J.%20Clin.&title=Cancer%20statistics,%202023&author=R.L.%20Siegel&author=K.D.%20Miller&author=N.S.%20Wagle&author=A.%20Jemal&volume=73&publication_year=2023&pages=17-48&pmid=36633525&doi=10.3322/caac.21763&)

2. Armitage J.O., Gascoyne R.D., Lunning M.A., Cavalli F. Non-Hodgkin lymphoma. Lancet. 2017;390:298–310. doi: 10.1016/S0140-6736(16)32407-2.  [DOI](https://doi.org/10.1016/S0140-6736(16)32407-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28153383/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Non-Hodgkin%20lymphoma&author=J.O.%20Armitage&author=R.D.%20Gascoyne&author=M.A.%20Lunning&author=F.%20Cavalli&volume=390&publication_year=2017&pages=298-310&pmid=28153383&doi=10.1016/S0140-6736(16)32407-2&)

3. Alaggio R., Amador C., Anagnostopoulos I., Attygalle A.D., Araujo I.B.O., Berti E., Bhagat G., Borges A.M., Boyer D., Calaminici M., et al. The 5th edition of the World Health Organization Classification of Haematolymphoid Tumours: Lymphoid Neoplasms. Leukemia. 2022;36:1720–1748. doi: 10.1038/s41375-022-01620-2.  [DOI](https://doi.org/10.1038/s41375-022-01620-2) | [PMC free article](/articles/PMC9214472/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35732829/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Leukemia&title=The%205th%20edition%20of%20the%20World%20Health%20Organization%20Classification%20of%20Haematolymphoid%20Tumours:%20Lymphoid%20Neoplasms&author=R.%20Alaggio&author=C.%20Amador&author=I.%20Anagnostopoulos&author=A.D.%20Attygalle&author=I.B.O.%20Araujo&volume=36&publication_year=2022&pages=1720-1748&pmid=35732829&doi=10.1038/s41375-022-01620-2&)

4. Alizadeh A.A., Eisen M.B., Davis R.E., Ma C., Lossos I.S., Rosenwald A., Boldrick J.C., Sabet H., Tran T., Yu X., et al. Distinct types of diffuse large B-cell lymphoma identified by gene expression profiling. Nature. 2000;403:503–511. doi: 10.1038/35000501.  [DOI](https://doi.org/10.1038/35000501) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10676951/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Distinct%20types%20of%20diffuse%20large%20B-cell%20lymphoma%20identified%20by%20gene%20expression%20profiling&author=A.A.%20Alizadeh&author=M.B.%20Eisen&author=R.E.%20Davis&author=C.%20Ma&author=I.S.%20Lossos&volume=403&publication_year=2000&pages=503-511&pmid=10676951&doi=10.1038/35000501&)

5. Rosenwald A., Wright G., Chan W.C., Connors J.M., Campo E., Fisher R.I., Gascoyne R.D., Muller-Hermelink H.K., Smeland E.B., Giltnane J.M., et al. The use of molecular profiling to predict survival after chemotherapy for diffuse large-B-cell lymphoma. N. Engl. J. Med. 2002;346:1937–1947. doi: 10.1056/NEJMoa012914.  [DOI](https://doi.org/10.1056/NEJMoa012914) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12075054/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%20Engl.%20J.%20Med.&title=The%20use%20of%20molecular%20profiling%20to%20predict%20survival%20after%20chemotherapy%20for%20diffuse%20large-B-cell%20lymphoma&author=A.%20Rosenwald&author=G.%20Wright&author=W.C.%20Chan&author=J.M.%20Connors&author=E.%20Campo&volume=346&publication_year=2002&pages=1937-1947&pmid=12075054&doi=10.1056/NEJMoa012914&)

6. Lenz G., Wright G., Dave S.S., Xiao W., Powell J., Zhao H., Xu W., Tan B., Goldschmidt N., Iqbal J., et al. Stromal gene signatures in large-B-cell lymphomas. N. Engl. J. Med. 2008;359:2313–2323. doi: 10.1056/NEJMoa0802885.  [DOI](https://doi.org/10.1056/NEJMoa0802885) | [PMC free article](/articles/PMC9103713/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19038878/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%20Engl.%20J.%20Med.&title=Stromal%20gene%20signatures%20in%20large-B-cell%20lymphomas&author=G.%20Lenz&author=G.%20Wright&author=S.S.%20Dave&author=W.%20Xiao&author=J.%20Powell&volume=359&publication_year=2008&pages=2313-2323&pmid=19038878&doi=10.1056/NEJMoa0802885&)

7. Holmes A.B., Corinaldesi C., Shen Q., Kumar R., Compagno N., Wang Z., Nitzan M., Grunstein E., Pasqualucci L., Dalla-Favera R., et al. Single-cell analysis of germinal-center B cells informs on lymphoma cell of origin and outcome. J. Exp. Med. 2020;217:e20200483. doi: 10.1084/jem.20200483.  [DOI](https://doi.org/10.1084/jem.20200483) | [PMC free article](/articles/PMC7537389/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32603407/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Exp.%20Med.&title=Single-cell%20analysis%20of%20germinal-center%20B%20cells%20informs%20on%20lymphoma%20cell%20of%20origin%20and%20outcome&author=A.B.%20Holmes&author=C.%20Corinaldesi&author=Q.%20Shen&author=R.%20Kumar&author=N.%20Compagno&volume=217&publication_year=2020&pages=e20200483&pmid=32603407&doi=10.1084/jem.20200483&)

8. Meyer P.N., Fu K., Greiner T.C., Smith L.M., Delabie J., Gascoyne R.D., Ott G., Rosenwald A., Braziel R.M., Campo E., et al. Immunohistochemical methods for predicting cell of origin and survival in patients with diffuse large B-cell lymphoma treated with rituximab. J. Clin. Oncol. 2011;29:200–207. doi: 10.1200/JCO.2010.30.0368.  [DOI](https://doi.org/10.1200/JCO.2010.30.0368) | [PMC free article](/articles/PMC3058275/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21135273/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Oncol.&title=Immunohistochemical%20methods%20for%20predicting%20cell%20of%20origin%20and%20survival%20in%20patients%20with%20diffuse%20large%20B-cell%20lymphoma%20treated%20with%20rituximab&author=P.N.%20Meyer&author=K.%20Fu&author=T.C.%20Greiner&author=L.M.%20Smith&author=J.%20Delabie&volume=29&publication_year=2011&pages=200-207&pmid=21135273&doi=10.1200/JCO.2010.30.0368&)

9. Scott D.W. Cell-of-Origin in Diffuse Large B-Cell Lymphoma: Are the Assays Ready for the Clinic? Am. Soc. Clin. Oncol. Educ. Book. 2015;35:e458–e466. doi: 10.14694/EdBook_AM.2015.35.e458.  [DOI](https://doi.org/10.14694/EdBook_AM.2015.35.e458) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25993210/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20Soc.%20Clin.%20Oncol.%20Educ.%20Book&title=Cell-of-Origin%20in%20Diffuse%20Large%20B-Cell%20Lymphoma:%20Are%20the%20Assays%20Ready%20for%20the%20Clinic?&author=D.W.%20Scott&volume=35&publication_year=2015&pages=e458-e466&pmid=25993210&doi=10.14694/EdBook_AM.2015.35.e458&)

10. Hans C.P., Weisenburger D.D., Greiner T.C., Gascoyne R.D., Delabie J., Ott G., Müller-Hermelink H.K., Campo E., Braziel R.M., Jaffe E.S., et al. Confirmation of the molecular classification of diffuse large B-cell lymphoma by immunohistochemistry using a tissue microarray. Blood. 2004;103:275–282. doi: 10.1182/blood-2003-05-1545.  [DOI](https://doi.org/10.1182/blood-2003-05-1545) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14504078/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Confirmation%20of%20the%20molecular%20classification%20of%20diffuse%20large%20B-cell%20lymphoma%20by%20immunohistochemistry%20using%20a%20tissue%20microarray&author=C.P.%20Hans&author=D.D.%20Weisenburger&author=T.C.%20Greiner&author=R.D.%20Gascoyne&author=J.%20Delabie&volume=103&publication_year=2004&pages=275-282&pmid=14504078&doi=10.1182/blood-2003-05-1545&)

11. Chapuy B., Stewart C., Dunford A.J., Kim J., Kamburov A., Redd R.A., Lawrence M.S., Roemer M.G.M., Li A.J., Ziepert M., et al. Molecular subtypes of diffuse large B cell lymphoma are associated with distinct pathogenic mechanisms and outcomes. Nat. Med. 2018;24:679–690. doi: 10.1038/s41591-018-0016-8.  [DOI](https://doi.org/10.1038/s41591-018-0016-8) | [PMC free article](/articles/PMC6613387/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29713087/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Med.&title=Molecular%20subtypes%20of%20diffuse%20large%20B%20cell%20lymphoma%20are%20associated%20with%20distinct%20pathogenic%20mechanisms%20and%20outcomes&author=B.%20Chapuy&author=C.%20Stewart&author=A.J.%20Dunford&author=J.%20Kim&author=A.%20Kamburov&volume=24&publication_year=2018&pages=679-690&pmid=29713087&doi=10.1038/s41591-018-0016-8&)

12. Vermaat J.S., Pals S.T., Younes A., Dreyling M., Federico M., Aurer I., Radford J., Kersten M.J., EHA Lymphoma Group, a Scientific Working Group of the European Hematology Association Precision medicine in diffuse large B-cell lymphoma: Hitting the target. Haematologica. 2015;100:989–993. doi: 10.3324/haematol.2015.128371.  [DOI](https://doi.org/10.3324/haematol.2015.128371) | [PMC free article](/articles/PMC5004412/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26314080/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Haematologica&title=Precision%20medicine%20in%20diffuse%20large%20B-cell%20lymphoma:%20Hitting%20the%20target&author=J.S.%20Vermaat&author=S.T.%20Pals&author=A.%20Younes&author=M.%20Dreyling&author=M.%20Federico&volume=100&publication_year=2015&pages=989-993&pmid=26314080&doi=10.3324/haematol.2015.128371&)

13. Coiffier B., Sarkozy C. Diffuse large B-cell lymphoma: R-CHOP failure-what to do? Hematol. Am. Soc. Hematol. Educ. Program. 2016;2016:366–378. doi: 10.1182/asheducation-2016.1.366.  [DOI](https://doi.org/10.1182/asheducation-2016.1.366) | [PMC free article](/articles/PMC6142522/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27913503/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hematol.%20Am.%20Soc.%20Hematol.%20Educ.%20Program&title=Diffuse%20large%20B-cell%20lymphoma:%20R-CHOP%20failure-what%20to%20do?&author=B.%20Coiffier&author=C.%20Sarkozy&volume=2016&publication_year=2016&pages=366-378&pmid=27913503&doi=10.1182/asheducation-2016.1.366&)

14. Pasqualucci L., Dalla-Favera R. Genetics of diffuse large B-cell lymphoma. Blood. 2018;131:2307–2319. doi: 10.1182/blood-2017-11-764332.  [DOI](https://doi.org/10.1182/blood-2017-11-764332) | [PMC free article](/articles/PMC5969374/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29666115/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Genetics%20of%20diffuse%20large%20B-cell%20lymphoma&author=L.%20Pasqualucci&author=R.%20Dalla-Favera&volume=131&publication_year=2018&pages=2307-2319&pmid=29666115&doi=10.1182/blood-2017-11-764332&)

15. Sehn L.H., Berry B., Chhanabhai M., Fitzgerald C., Gill K., Hoskins P., Klasa R., Savage K.J., Shenkier T., Sutherland J., et al. The revised International Prognostic Index (R-IPI) is a better predictor of outcome than the standard IPI for patients with diffuse large B-cell lymphoma treated with R-CHOP. Blood. 2007;109:1857–1861. doi: 10.1182/blood-2006-08-038257.  [DOI](https://doi.org/10.1182/blood-2006-08-038257) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17105812/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=The%20revised%20International%20Prognostic%20Index%20(R-IPI)%20is%20a%20better%20predictor%20of%20outcome%20than%20the%20standard%20IPI%20for%20patients%20with%20diffuse%20large%20B-cell%20lymphoma%20treated%20with%20R-CHOP&author=L.H.%20Sehn&author=B.%20Berry&author=M.%20Chhanabhai&author=C.%20Fitzgerald&author=K.%20Gill&volume=109&publication_year=2007&pages=1857-1861&pmid=17105812&doi=10.1182/blood-2006-08-038257&)

16. Kubuschok B., Held G., Pfreundschuh M. Management of diffuse large B-cell lymphoma (DLBCL) Cancer Treat. Res. 2015;165:271–288. doi: 10.1007/978-3-319-13150-4_11.  [DOI](https://doi.org/10.1007/978-3-319-13150-4_11) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25655614/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Treat.%20Res.&title=Management%20of%20diffuse%20large%20B-cell%20lymphoma%20(DLBCL)&author=B.%20Kubuschok&author=G.%20Held&author=M.%20Pfreundschuh&volume=165&publication_year=2015&pages=271-288&pmid=25655614&doi=10.1007/978-3-319-13150-4_11&)

17. González-Barca E., Boumendil A., Blaise D., Trněný M., Masszi T., Finel H., Michieli M.G., Bittenbring J.T., Gritti G., Snowden J.A., et al. Outcome in patients with diffuse large B-cell lymphoma who relapse after autologous stem cell transplantation and receive active therapy. A retrospective analysis of the Lymphoma Working Party of the European Society for Blood and Marrow Transplantation (EBMT) Bone Marrow Transpl. 2020;55:393–399. doi: 10.1038/s41409-019-0650-x.  [DOI](https://doi.org/10.1038/s41409-019-0650-x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31541205/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bone%20Marrow%20Transpl.&title=Outcome%20in%20patients%20with%20diffuse%20large%20B-cell%20lymphoma%20who%20relapse%20after%20autologous%20stem%20cell%20transplantation%20and%20receive%20active%20therapy.%20A%20retrospective%20analysis%20of%20the%20Lymphoma%20Working%20Party%20of%20the%20European%20Society%20for%20Blood%20and%20Marrow%20Transplantation%20(EBMT)&author=E.%20Gonz%C3%A1lez-Barca&author=A.%20Boumendil&author=D.%20Blaise&author=M.%20Trn%C4%9Bn%C3%BD&author=T.%20Masszi&volume=55&publication_year=2020&pages=393-399&pmid=31541205&doi=10.1038/s41409-019-0650-x&)

18. Coiffier B., Lepage E., Briere J., Herbrecht R., Tilly H., Bouabdallah R., Morel P., Van Den Neste E., Salles G., Gaulard P., et al. CHOP chemotherapy plus rituximab compared with CHOP alone in elderly patients with diffuse large-B-cell lymphoma. N. Engl. J. Med. 2002;346:235–242. doi: 10.1056/NEJMoa011795.  [DOI](https://doi.org/10.1056/NEJMoa011795) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11807147/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%20Engl.%20J.%20Med.&title=CHOP%20chemotherapy%20plus%20rituximab%20compared%20with%20CHOP%20alone%20in%20elderly%20patients%20with%20diffuse%20large-B-cell%20lymphoma&author=B.%20Coiffier&author=E.%20Lepage&author=J.%20Briere&author=R.%20Herbrecht&author=H.%20Tilly&volume=346&publication_year=2002&pages=235-242&pmid=11807147&doi=10.1056/NEJMoa011795&)

19. Sehn L.H., Donaldson J., Chhanabhai M., Fitzgerald C., Gill K., Klasa R., MacPherson N., O’Reilly S., Spinelli J.J., Sutherland J., et al. Introduction of combined CHOP plus rituximab therapy dramatically improved outcome of diffuse large B-cell lymphoma in British Columbia. J. Clin. Oncol. 2005;23:5027–5033. doi: 10.1200/JCO.2005.09.137.  [DOI](https://doi.org/10.1200/JCO.2005.09.137) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15955905/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Oncol.&title=Introduction%20of%20combined%20CHOP%20plus%20rituximab%20therapy%20dramatically%20improved%20outcome%20of%20diffuse%20large%20B-cell%20lymphoma%20in%20British%20Columbia&author=L.H.%20Sehn&author=J.%20Donaldson&author=M.%20Chhanabhai&author=C.%20Fitzgerald&author=K.%20Gill&volume=23&publication_year=2005&pages=5027-5033&pmid=15955905&doi=10.1200/JCO.2005.09.137&)

20. Pfreundschuh M., Trümper L., Osterborg A., Pettengell R., Trneny M., Imrie K., Ma D., Gill D., Walewski J., Zinzani P.L., et al. CHOP-like chemotherapy plus rituximab versus CHOP-like chemotherapy alone in young patients with good-prognosis diffuse large-B-cell lymphoma: A randomised controlled trial by the MabThera International Trial (MInT) Group. Lancet Oncol. 2006;7:379–391. doi: 10.1016/S1470-2045(06)70664-7.  [DOI](https://doi.org/10.1016/S1470-2045(06)70664-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16648042/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet%20Oncol.&title=CHOP-like%20chemotherapy%20plus%20rituximab%20versus%20CHOP-like%20chemotherapy%20alone%20in%20young%20patients%20with%20good-prognosis%20diffuse%20large-B-cell%20lymphoma:%20A%20randomised%20controlled%20trial%20by%20the%20MabThera%20International%20Trial%20(MInT)%20Group&author=M.%20Pfreundschuh&author=L.%20Tr%C3%BCmper&author=A.%20Osterborg&author=R.%20Pettengell&author=M.%20Trneny&volume=7&publication_year=2006&pages=379-391&pmid=16648042&doi=10.1016/S1470-2045(06)70664-7&)

21. Coiffier B., Thieblemont C., Van Den Neste E., Lepeu G., Plantier I., Castaigne S., Lefort S., Marit G., Macro M., Sebban C., et al. Long-term outcome of patients in the LNH-98.5 trial, the first randomized study comparing rituximab-CHOP to standard CHOP chemotherapy in DLBCL patients: A study by the Groupe d’Etudes des Lymphomes de l’Adulte. Blood. 2010;116:2040–2045. doi: 10.1182/blood-2010-03-276246.  [DOI](https://doi.org/10.1182/blood-2010-03-276246) | [PMC free article](/articles/PMC2951853/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20548096/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Long-term%20outcome%20of%20patients%20in%20the%20LNH-98.5%20trial,%20the%20first%20randomized%20study%20comparing%20rituximab-CHOP%20to%20standard%20CHOP%20chemotherapy%20in%20DLBCL%20patients:%20A%20study%20by%20the%20Groupe%20d%E2%80%99Etudes%20des%20Lymphomes%20de%20l%E2%80%99Adulte&author=B.%20Coiffier&author=C.%20Thieblemont&author=E.%20Van%20Den%20Neste&author=G.%20Lepeu&author=I.%20Plantier&volume=116&publication_year=2010&pages=2040-2045&pmid=20548096&doi=10.1182/blood-2010-03-276246&)

22. Pfreundschuh M., Kuhnt E., Trümper L., Osterborg A., Trneny M., Shepherd L., Gill D.S., Walewski J., Pettengell R., Jaeger U., et al. CHOP-like chemotherapy with or without rituximab in young patients with good-prognosis diffuse large-B-cell lymphoma: 6-year results of an open-label randomised study of the MabThera International Trial (MInT) Group. Lancet Oncol. 2011;12:1013–1022. doi: 10.1016/S1470-2045(11)70235-2.  [DOI](https://doi.org/10.1016/S1470-2045(11)70235-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21940214/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet%20Oncol.&title=CHOP-like%20chemotherapy%20with%20or%20without%20rituximab%20in%20young%20patients%20with%20good-prognosis%20diffuse%20large-B-cell%20lymphoma:%206-year%20results%20of%20an%20open-label%20randomised%20study%20of%20the%20MabThera%20International%20Trial%20(MInT)%20Group&author=M.%20Pfreundschuh&author=E.%20Kuhnt&author=L.%20Tr%C3%BCmper&author=A.%20Osterborg&author=M.%20Trneny&volume=12&publication_year=2011&pages=1013-1022&pmid=21940214&doi=10.1016/S1470-2045(11)70235-2&)

23. NCCN Clinical Practice Guidelines in Oncology B-Cell lymphomas (V. 1.2023) 2023. [(accessed on 14 April 2023)]. Available online: https://www.nccn.org/professionals/physician_gls/pdf/b-cell.pdf.  [https://www.nccn.org/professionals/physician_gls/pdf/b-cell.pdf](https://www.nccn.org/professionals/physician_gls/pdf/b-cell.pdf)

24. Tilly H., Morschhauser F., Sehn L.H., Friedberg J.W., Trněný M., Sharman J.P., Herbaux C., Burke J.M., Matasar M., Rai S., et al. Polatuzumab Vedotin in Previously Untreated Diffuse Large B-Cell Lymphoma. N. Engl. J. Med. 2022;386:351–363. doi: 10.1056/NEJMoa2115304.  [DOI](https://doi.org/10.1056/NEJMoa2115304) | [PMC free article](/articles/PMC11702892/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34904799/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%20Engl.%20J.%20Med.&title=Polatuzumab%20Vedotin%20in%20Previously%20Untreated%20Diffuse%20Large%20B-Cell%20Lymphoma&author=H.%20Tilly&author=F.%20Morschhauser&author=L.H.%20Sehn&author=J.W.%20Friedberg&author=M.%20Trn%C4%9Bn%C3%BD&volume=386&publication_year=2022&pages=351-363&pmid=34904799&doi=10.1056/NEJMoa2115304&)

25. Dunleavy K. Optimal Management of Double-Hit Lymphoma. J. Oncol. Pract. 2016;12:241–242. doi: 10.1200/JOP.2015.010421.  [DOI](https://doi.org/10.1200/JOP.2015.010421) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26962168/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Oncol.%20Pract.&title=Optimal%20Management%20of%20Double-Hit%20Lymphoma&author=K.%20Dunleavy&volume=12&publication_year=2016&pages=241-242&pmid=26962168&doi=10.1200/JOP.2015.010421&)

26. Johnson N.A., Slack G.W., Savage K.J., Connors J.M., Ben-Neriah S., Rogic S., Scott D.W., Tan K.L., Steidl C., Sehn L.H., et al. Concurrent expression of MYC and BCL2 in diffuse large B-cell lymphoma treated with rituximab plus cyclophosphamide, doxorubicin, vincristine, and prednisone. J. Clin. Oncol. 2012;30:3452–3459. doi: 10.1200/JCO.2011.41.0985.  [DOI](https://doi.org/10.1200/JCO.2011.41.0985) | [PMC free article](/articles/PMC3454768/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22851565/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Oncol.&title=Concurrent%20expression%20of%20MYC%20and%20BCL2%20in%20diffuse%20large%20B-cell%20lymphoma%20treated%20with%20rituximab%20plus%20cyclophosphamide,%20doxorubicin,%20vincristine,%20and%20prednisone&author=N.A.%20Johnson&author=G.W.%20Slack&author=K.J.%20Savage&author=J.M.%20Connors&author=S.%20Ben-Neriah&volume=30&publication_year=2012&pages=3452-3459&pmid=22851565&doi=10.1200/JCO.2011.41.0985&)

27. Tafasitamab + Lenalidomide + R-CHOP Versus R-CHOP in Newly Diagnosed High-intermediate and High Risk DLBCL Patients (frontMIND) [(accessed on 14 April 2023)]; Available online: https://clinicaltrials.gov/ct2/show/NCT04824092.  [https://clinicaltrials.gov/ct2/show/NCT04824092](https://clinicaltrials.gov/ct2/show/NCT04824092)

28. El Hussein S., Shaw K.R.M., Vega F. Evolving insights into the genomic complexity and immune landscape of diffuse large B-cell lymphoma: Opportunities for novel biomarkers. Mod. Pathol. 2020;33:2422–2436. doi: 10.1038/s41379-020-0616-y.  [DOI](https://doi.org/10.1038/s41379-020-0616-y) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32620919/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mod.%20Pathol.&title=Evolving%20insights%20into%20the%20genomic%20complexity%20and%20immune%20landscape%20of%20diffuse%20large%20B-cell%20lymphoma:%20Opportunities%20for%20novel%20biomarkers&author=S.%20El%20Hussein&author=K.R.M.%20Shaw&author=F.%20Vega&volume=33&publication_year=2020&pages=2422-2436&pmid=32620919&doi=10.1038/s41379-020-0616-y&)

29. Rossi D., Diop F., Spaccarotella E., Monti S., Zanni M., Rasi S., Deambrogi C., Spina V., Bruscaggin A., Favini C., et al. Diffuse large B-cell lymphoma genotyping on the liquid biopsy. Blood. 2017;129:1947–1957. doi: 10.1182/blood-2016-05-719641.  [DOI](https://doi.org/10.1182/blood-2016-05-719641) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28096087/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Diffuse%20large%20B-cell%20lymphoma%20genotyping%20on%20the%20liquid%20biopsy&author=D.%20Rossi&author=F.%20Diop&author=E.%20Spaccarotella&author=S.%20Monti&author=M.%20Zanni&volume=129&publication_year=2017&pages=1947-1957&pmid=28096087&doi=10.1182/blood-2016-05-719641&)

30. Mini E., Nobili S. Pharmacogenetics of Cancer. OAE Publishing Inc.; Alhambra, CA, USA: 2020.  [DOI](https://doi.org/10.20517/cdr.2020.10) | [PMC free article](/articles/PMC9090591/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35582607/) | [Google Scholar](https://scholar.google.com/scholar_lookup?title=Pharmacogenetics%20of%20Cancer&author=E.%20Mini&author=S.%20Nobili&publication_year=2020&)

31. Lavacchi D., Landini I., Perrone G., Roviello G., Mini E., Nobili S. Pharmacogenetics in diffuse large B-cell lymphoma treated with R-CHOP: Still an unmet challenge. Pharmacol. Ther. 2022;229:107924. doi: 10.1016/j.pharmthera.2021.107924.  [DOI](https://doi.org/10.1016/j.pharmthera.2021.107924) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34175369/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacol.%20Ther.&title=Pharmacogenetics%20in%20diffuse%20large%20B-cell%20lymphoma%20treated%20with%20R-CHOP:%20Still%20an%20unmet%20challenge&author=D.%20Lavacchi&author=I.%20Landini&author=G.%20Perrone&author=G.%20Roviello&author=E.%20Mini&volume=229&publication_year=2022&pages=107924&pmid=34175369&doi=10.1016/j.pharmthera.2021.107924&)

32. Falduto A., Cimino F., Speciale A., Musolino C., Gangemi S., Saija A., Allegra A. How gene polymorphisms can influence clinical response and toxicity following R-CHOP therapy in patients with diffuse large B cell lymphoma. Blood Rev. 2017;31:235–249. doi: 10.1016/j.blre.2017.02.005.  [DOI](https://doi.org/10.1016/j.blre.2017.02.005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28262268/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood%20Rev.&title=How%20gene%20polymorphisms%20can%20influence%20clinical%20response%20and%20toxicity%20following%20R-CHOP%20therapy%20in%20patients%20with%20diffuse%20large%20B%20cell%20lymphoma&author=A.%20Falduto&author=F.%20Cimino&author=A.%20Speciale&author=C.%20Musolino&author=S.%20Gangemi&volume=31&publication_year=2017&pages=235-249&pmid=28262268&doi=10.1016/j.blre.2017.02.005&)

33. Low S.K., Takahashi A., Mushiroda T., Kubo M. Genome-wide association study: A useful tool to identify common genetic variants associated with drug toxicity and efficacy in cancer pharmacogenomics. Clin. Cancer Res. 2014;20:2541–2552. doi: 10.1158/1078-0432.CCR-13-2755.  [DOI](https://doi.org/10.1158/1078-0432.CCR-13-2755) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24831277/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Cancer%20Res.&title=Genome-wide%20association%20study:%20A%20useful%20tool%20to%20identify%20common%20genetic%20variants%20associated%20with%20drug%20toxicity%20and%20efficacy%20in%20cancer%20pharmacogenomics&author=S.K.%20Low&author=A.%20Takahashi&author=T.%20Mushiroda&author=M.%20Kubo&volume=20&publication_year=2014&pages=2541-2552&pmid=24831277&doi=10.1158/1078-0432.CCR-13-2755&)

34. Kiyotani K., Mushiroda T., Tsunoda T., Morizono T., Hosono N., Kubo M., Tanigawara Y., Imamura C.K., Flockhart D.A., Aki F., et al. A genome-wide association study identifies locus at 10q22 associated with clinical outcomes of adjuvant tamoxifen therapy for breast cancer patients in Japanese. Hum. Mol. Genet. 2012;21:1665–1672. doi: 10.1093/hmg/ddr597.  [DOI](https://doi.org/10.1093/hmg/ddr597) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22180457/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum.%20Mol.%20Genet.&title=A%20genome-wide%20association%20study%20identifies%20locus%20at%2010q22%20associated%20with%20clinical%20outcomes%20of%20adjuvant%20tamoxifen%20therapy%20for%20breast%20cancer%20patients%20in%20Japanese&author=K.%20Kiyotani&author=T.%20Mushiroda&author=T.%20Tsunoda&author=T.%20Morizono&author=N.%20Hosono&volume=21&publication_year=2012&pages=1665-1672&pmid=22180457&doi=10.1093/hmg/ddr597&)

35. Wu X., Ye Y., Rosell R., Amos C.I., Stewart D.J., Hildebrandt M.A., Roth J.A., Minna J.D., Gu J., Lin J., et al. Genome-wide association study of survival in non-small cell lung cancer patients receiving platinum-based chemotherapy. J. Natl. Cancer Inst. 2011;103:817–825. doi: 10.1093/jnci/djr075.  [DOI](https://doi.org/10.1093/jnci/djr075) | [PMC free article](/articles/PMC3096796/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21483023/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Natl.%20Cancer%20Inst.&title=Genome-wide%20association%20study%20of%20survival%20in%20non-small%20cell%20lung%20cancer%20patients%20receiving%20platinum-based%20chemotherapy&author=X.%20Wu&author=Y.%20Ye&author=R.%20Rosell&author=C.I.%20Amos&author=D.J.%20Stewart&volume=103&publication_year=2011&pages=817-825&pmid=21483023&doi=10.1093/jnci/djr075&)

36. Hu L., Wu C., Zhao X., Heist R., Su L., Zhao Y., Han B., Cao S., Chu M., Dai J., et al. Genome-wide association study of prognosis in advanced non-small cell lung cancer patients receiving platinum-based chemotherapy. Clin. Cancer Res. 2012;18:5507–5514. doi: 10.1158/1078-0432.CCR-12-1202.  [DOI](https://doi.org/10.1158/1078-0432.CCR-12-1202) | [PMC free article](/articles/PMC3723686/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22872573/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Cancer%20Res.&title=Genome-wide%20association%20study%20of%20prognosis%20in%20advanced%20non-small%20cell%20lung%20cancer%20patients%20receiving%20platinum-based%20chemotherapy&author=L.%20Hu&author=C.%20Wu&author=X.%20Zhao&author=R.%20Heist&author=L.%20Su&volume=18&publication_year=2012&pages=5507-5514&pmid=22872573&doi=10.1158/1078-0432.CCR-12-1202&)

37. Innocenti F., Owzar K., Cox N.L., Evans P., Kubo M., Zembutsu H., Jiang C., Hollis D., Mushiroda T., Li L., et al. A genome-wide association study of overall survival in pancreatic cancer patients treated with gemcitabine in CALGB 80303. Clin. Cancer Res. 2012;18:577–584. doi: 10.1158/1078-0432.CCR-11-1387.  [DOI](https://doi.org/10.1158/1078-0432.CCR-11-1387) | [PMC free article](/articles/PMC3412624/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22142827/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Cancer%20Res.&title=A%20genome-wide%20association%20study%20of%20overall%20survival%20in%20pancreatic%20cancer%20patients%20treated%20with%20gemcitabine%20in%20CALGB%2080303&author=F.%20Innocenti&author=K.%20Owzar&author=N.L.%20Cox&author=P.%20Evans&author=M.%20Kubo&volume=18&publication_year=2012&pages=577-584&pmid=22142827&doi=10.1158/1078-0432.CCR-11-1387&)

38. Diekstra M.H.M., Swen J.J., van der Zanden L.F.M., Vermeulen S.H., Boven E., Mathijssen R.H.J., Fukunaga K., Mushiroda T., Hongo F., Oosterwijk E., et al. Genome-Wide Meta-Analysis Identifies Variants in DSCAM and PDLIM3 That Correlate with Efficacy Outcomes in Metastatic Renal Cell Carcinoma Patients Treated with Sunitinib. Cancers. 2022;14:2838. doi: 10.3390/cancers14122838.  [DOI](https://doi.org/10.3390/cancers14122838) | [PMC free article](/articles/PMC9220885/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35740506/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancers&title=Genome-Wide%20Meta-Analysis%20Identifies%20Variants%20in%20DSCAM%20and%20PDLIM3%20That%20Correlate%20with%20Efficacy%20Outcomes%20in%20Metastatic%20Renal%20Cell%20Carcinoma%20Patients%20Treated%20with%20Sunitinib&author=M.H.M.%20Diekstra&author=J.J.%20Swen&author=L.F.M.%20van%20der%20Zanden&author=S.H.%20Vermeulen&author=E.%20Boven&volume=14&publication_year=2022&pages=2838&pmid=35740506&doi=10.3390/cancers14122838&)

39. Lin C.L., Liang K.H., Hu C.C., Chien C.H., Chen L.W., Chien R.N., Lin Y.H., Yeh C.T. A Single Nucleotide Polymorphism rs1010816 Predicts Sorafenib Therapeutic Outcomes in Advanced Hepatocellular Carcinoma. Int. J. Mol. Sci. 2023;24:1681. doi: 10.3390/ijms24021681.  [DOI](https://doi.org/10.3390/ijms24021681) | [PMC free article](/articles/PMC9862766/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36675198/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Mol.%20Sci.&title=A%20Single%20Nucleotide%20Polymorphism%20rs1010816%20Predicts%20Sorafenib%20Therapeutic%20Outcomes%20in%20Advanced%20Hepatocellular%20Carcinoma&author=C.L.%20Lin&author=K.H.%20Liang&author=C.C.%20Hu&author=C.H.%20Chien&author=L.W.%20Chen&volume=24&publication_year=2023&pages=1681&pmid=36675198&doi=10.3390/ijms24021681&)

40. Ghesquieres H., Slager S.L., Jardin F., Veron A.S., Asmann Y.W., Maurer M.J., Fest T., Habermann T.M., Bene M.C., Novak A.J., et al. Genome-Wide Association Study of Event-Free Survival in Diffuse Large B-Cell Lymphoma Treated With Immunochemotherapy. J. Clin. Oncol. 2015;33:3930–3937. doi: 10.1200/JCO.2014.60.2573.  [DOI](https://doi.org/10.1200/JCO.2014.60.2573) | [PMC free article](/articles/PMC4879713/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26460308/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Oncol.&title=Genome-Wide%20Association%20Study%20of%20Event-Free%20Survival%20in%20Diffuse%20Large%20B-Cell%20Lymphoma%20Treated%20With%20Immunochemotherapy&author=H.%20Ghesquieres&author=S.L.%20Slager&author=F.%20Jardin&author=A.S.%20Veron&author=Y.W.%20Asmann&volume=33&publication_year=2015&pages=3930-3937&pmid=26460308&doi=10.1200/JCO.2014.60.2573&)

41. Swerdlow S.H., Campo E., Harris N.L., Jaffe E.S., Pileri S.A., Stein H., Thiele J., Vardiman J.W. WHO Classification of Tumours of Haematopoietic and Lymphoid Tissues-WHO Classification of Tumours. 4th ed. Volume 2 United States Environmental Protection Agency; Washington, DC, USA: 2008.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=WHO%20Classification%20of%20Tumours%20of%20Haematopoietic%20and%20Lymphoid%20Tissues-WHO%20Classification%20of%20Tumours&author=S.H.%20Swerdlow&author=E.%20Campo&author=N.L.%20Harris&author=E.S.%20Jaffe&author=S.A.%20Pileri&publication_year=2008&)

42. Swerdlow S.H., Campo E., Harris N.L., Jaffe E.S., Pileri S.A., Stein H., Thiele J. WHO Classification of Tumours of Haematopoietic and Lymphoid Tissues-WHO Classification of Tumours, revised. 4th ed. Volume 2 United States Environmental Protection Agency; Washington, DC, USA: 2017.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=WHO%20Classification%20of%20Tumours%20of%20Haematopoietic%20and%20Lymphoid%20Tissues-WHO%20Classification%20of%20Tumours,%20revised&author=S.H.%20Swerdlow&author=E.%20Campo&author=N.L.%20Harris&author=E.S.%20Jaffe&author=S.A.%20Pileri&publication_year=2017&)

43. Thermo Fisher Scientific . Axiom Genotyping Solution Data Analysis User Guide. Thermo Fisher Scientific; Hong Kong, China: 2020. Publication Number MAN0018363 Tech.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Axiom%20Genotyping%20Solution%20Data%20Analysis%20User%20Guide&publication_year=2020&)

44. Laurie C.C., Doheny K.F., Mirel D.B., Pugh E.W., Bierut L.J., Bhangale T., Boehm F., Caporaso N.E., Cornelis M.C., Edenberg H.J., et al. Quality control and quality assurance in genotypic data for genome-wide association studies. Genet Epidemiol. 2010;34:591–602. doi: 10.1002/gepi.20516.  [DOI](https://doi.org/10.1002/gepi.20516) | [PMC free article](/articles/PMC3061487/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20718045/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genet%20Epidemiol.&title=Quality%20control%20and%20quality%20assurance%20in%20genotypic%20data%20for%20genome-wide%20association%20studies&author=C.C.%20Laurie&author=K.F.%20Doheny&author=D.B.%20Mirel&author=E.W.%20Pugh&author=L.J.%20Bierut&volume=34&publication_year=2010&pages=591-602&pmid=20718045&doi=10.1002/gepi.20516&)

45. Cheson B.D., Pfistner B., Juweid M.E., Gascoyne R.D., Specht L., Horning S.J., Coiffier B., Fisher R.I., Hagenbeek A., Zucca E., et al. Revised response criteria for malignant lymphoma. J. Clin. Oncol. 2007;25:579–586. doi: 10.1200/JCO.2006.09.2403.  [DOI](https://doi.org/10.1200/JCO.2006.09.2403) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17242396/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Oncol.&title=Revised%20response%20criteria%20for%20malignant%20lymphoma&author=B.D.%20Cheson&author=B.%20Pfistner&author=M.E.%20Juweid&author=R.D.%20Gascoyne&author=L.%20Specht&volume=25&publication_year=2007&pages=579-586&pmid=17242396&doi=10.1200/JCO.2006.09.2403&)

46. Cheson B.D., Fisher R.I., Barrington S.F., Cavalli F., Schwartz L.H., Zucca E., Lister T.A. Recommendations for initial evaluation, staging, and response assessment of Hodgkin and non-Hodgkin lymphoma: The Lugano classification. J. Clin. Oncol. 2014;32:3059–3068. doi: 10.1200/JCO.2013.54.8800.  [DOI](https://doi.org/10.1200/JCO.2013.54.8800) | [PMC free article](/articles/PMC4979083/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25113753/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Oncol.&title=Recommendations%20for%20initial%20evaluation,%20staging,%20and%20response%20assessment%20of%20Hodgkin%20and%20non-Hodgkin%20lymphoma:%20The%20Lugano%20classification&author=B.D.%20Cheson&author=R.I.%20Fisher&author=S.F.%20Barrington&author=F.%20Cavalli&author=L.H.%20Schwartz&volume=32&publication_year=2014&pages=3059-3068&pmid=25113753&doi=10.1200/JCO.2013.54.8800&)

47. Nobili S., Napoli C., Puccini B., Landini I., Perrone G., Brugia M., Benelli G., Doria M., Martelli M., Finolezzi E., et al. Identification of pharmacogenomic markers of clinical efficacy in a dose-dense therapy regimen (R-CHOP14) in diffuse large B-cell lymphoma. Leuk Lymphoma. 2014;55:2071–2078. doi: 10.3109/10428194.2013.866665.  [DOI](https://doi.org/10.3109/10428194.2013.866665) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24289107/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Leuk%20Lymphoma&title=Identification%20of%20pharmacogenomic%20markers%20of%20clinical%20efficacy%20in%20a%20dose-dense%20therapy%20regimen%20(R-CHOP14)%20in%20diffuse%20large%20B-cell%20lymphoma&author=S.%20Nobili&author=C.%20Napoli&author=B.%20Puccini&author=I.%20Landini&author=G.%20Perrone&volume=55&publication_year=2014&pages=2071-2078&pmid=24289107&doi=10.3109/10428194.2013.866665&)

48. Bouroumeau A., Bussot L., Bonnefoix T., Fournier C., Chapusot C., Casasnovas O., Martin L., McLeer A., Col E., David-Boudet L., et al. c-MYC and p53 expression highlight starry-sky pattern as a favourable prognostic feature in R-CHOP-treated diffuse large B-cell lymphoma. J. Pathol. Clin. Res. 2021;7:604–615. doi: 10.1002/cjp2.223.  [DOI](https://doi.org/10.1002/cjp2.223) | [PMC free article](/articles/PMC8503894/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34374220/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Pathol.%20Clin.%20Res.&title=c-MYC%20and%20p53%20expression%20highlight%20starry-sky%20pattern%20as%20a%20favourable%20prognostic%20feature%20in%20R-CHOP-treated%20diffuse%20large%20B-cell%20lymphoma&author=A.%20Bouroumeau&author=L.%20Bussot&author=T.%20Bonnefoix&author=C.%20Fournier&author=C.%20Chapusot&volume=7&publication_year=2021&pages=604-615&pmid=34374220&doi=10.1002/cjp2.223&)

49. Yu H., Fu D., Xu P.P., Cheng S., Wang L., Zhang Y.Z., Zhao W.L. Implication of immune cell signature of tumor microenvironment in diffuse large B-cell lymphoma. Hematol. Oncol. 2021;39:616–624. doi: 10.1002/hon.2905.  [DOI](https://doi.org/10.1002/hon.2905) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34331367/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hematol.%20Oncol.&title=Implication%20of%20immune%20cell%20signature%20of%20tumor%20microenvironment%20in%20diffuse%20large%20B-cell%20lymphoma&author=H.%20Yu&author=D.%20Fu&author=P.P.%20Xu&author=S.%20Cheng&author=L.%20Wang&volume=39&publication_year=2021&pages=616-624&pmid=34331367&doi=10.1002/hon.2905&)

50. Kurtz D.M., Scherer F., Jin M.C., Soo J., Craig A.F.M., Esfahani M.S., Chabon J.J., Stehr H., Liu C.L., Tibshirani R., et al. Circulating Tumor DNA Measurements As Early Outcome Predictors in Diffuse Large B-Cell Lymphoma. J. Clin. Oncol. 2018;36:2845–2853. doi: 10.1200/JCO.2018.78.5246.  [DOI](https://doi.org/10.1200/JCO.2018.78.5246) | [PMC free article](/articles/PMC6161832/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30125215/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Oncol.&title=Circulating%20Tumor%20DNA%20Measurements%20As%20Early%20Outcome%20Predictors%20in%20Diffuse%20Large%20B-Cell%20Lymphoma&author=D.M.%20Kurtz&author=F.%20Scherer&author=M.C.%20Jin&author=J.%20Soo&author=A.F.M.%20Craig&volume=36&publication_year=2018&pages=2845-2853&pmid=30125215&doi=10.1200/JCO.2018.78.5246&)

51. Rivas-Delgado A., Nadeu F., Enjuanes A., Casanueva-Eliceiry S., Mozas P., Magnano L., Castrejón de Anta N., Rovira J., Dlouhy I., Martín S., et al. Mutational Landscape and Tumor Burden Assessed by Cell-free DNA in Diffuse Large B-Cell Lymphoma in a Population-Based Study. Clin. Cancer Res. 2021;27:513–521. doi: 10.1158/1078-0432.CCR-20-2558.  [DOI](https://doi.org/10.1158/1078-0432.CCR-20-2558) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33122345/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Cancer%20Res.&title=Mutational%20Landscape%20and%20Tumor%20Burden%20Assessed%20by%20Cell-free%20DNA%20in%20Diffuse%20Large%20B-Cell%20Lymphoma%20in%20a%20Population-Based%20Study&author=A.%20Rivas-Delgado&author=F.%20Nadeu&author=A.%20Enjuanes&author=S.%20Casanueva-Eliceiry&author=P.%20Mozas&volume=27&publication_year=2021&pages=513-521&pmid=33122345&doi=10.1158/1078-0432.CCR-20-2558&)

52. Li M., Mi L., Wang C., Wang X., Zhu J., Qi F., Yu H., Ye Y., Wang D., Cao J., et al. Clinical implications of circulating tumor DNA in predicting the outcome of diffuse large B cell lymphoma patients receiving first-line therapy. BMC Med. 2022;20:369. doi: 10.1186/s12916-022-02562-3.  [DOI](https://doi.org/10.1186/s12916-022-02562-3) | [PMC free article](/articles/PMC9594942/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36280874/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Med.&title=Clinical%20implications%20of%20circulating%20tumor%20DNA%20in%20predicting%20the%20outcome%20of%20diffuse%20large%20B%20cell%20lymphoma%20patients%20receiving%20first-line%20therapy&author=M.%20Li&author=L.%20Mi&author=C.%20Wang&author=X.%20Wang&author=J.%20Zhu&volume=20&publication_year=2022&pages=369&pmid=36280874&doi=10.1186/s12916-022-02562-3&)

53. Mendes R.V., Martins A.R., de Nucci G., Murad F., Soares F.A. Expression of nitric oxide synthase isoforms and nitrotyrosine immunoreactivity by B-cell non-Hodgkin’s lymphomas and multiple myeloma. Histopathology. 2001;39:172–178. doi: 10.1046/j.1365-2559.2001.01189.x.  [DOI](https://doi.org/10.1046/j.1365-2559.2001.01189.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11493334/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Histopathology&title=Expression%20of%20nitric%20oxide%20synthase%20isoforms%20and%20nitrotyrosine%20immunoreactivity%20by%20B-cell%20non-Hodgkin%E2%80%99s%20lymphomas%20and%20multiple%20myeloma&author=R.V.%20Mendes&author=A.R.%20Martins&author=G.%20de%20Nucci&author=F.%20Murad&author=F.A.%20Soares&volume=39&publication_year=2001&pages=172-178&pmid=11493334&doi=10.1046/j.1365-2559.2001.01189.x&)

54. Han X., Zheng T., Lan Q., Zhang Y., Kilfoy B.A., Qin Q., Rothman N., Zahm S.H., Holford T.R., Leaderer B. Genetic polymorphisms in nitric oxide synthase genes modify the relationship between vegetable and fruit intake and risk of non-Hodgkin lymphoma. Cancer Epidemiol. Biomark. Prev. 2009;18:1429–1438. doi: 10.1158/1055-9965.EPI-09-0001.  [DOI](https://doi.org/10.1158/1055-9965.EPI-09-0001) | [PMC free article](/articles/PMC2965355/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19423521/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Epidemiol.%20Biomark.%20Prev.&title=Genetic%20polymorphisms%20in%20nitric%20oxide%20synthase%20genes%20modify%20the%20relationship%20between%20vegetable%20and%20fruit%20intake%20and%20risk%20of%20non-Hodgkin%20lymphoma&author=X.%20Han&author=T.%20Zheng&author=Q.%20Lan&author=Y.%20Zhang&author=B.A.%20Kilfoy&volume=18&publication_year=2009&pages=1429-1438&pmid=19423521&doi=10.1158/1055-9965.EPI-09-0001&)

55. Kim C., Zheng T., Lan Q., Chen Y., Foss F., Chen X., Holford T., Leaderer B., Boyle P., Chanock S.J., et al. Genetic polymorphisms in oxidative stress pathway genes and modification of BMI and risk of non-Hodgkin lymphoma. Cancer Epidemiol. Biomark. Prev. 2012;21:866–868. doi: 10.1158/1055-9965.EPI-12-0010.  [DOI](https://doi.org/10.1158/1055-9965.EPI-12-0010) | [PMC free article](/articles/PMC3394153/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22374993/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Epidemiol.%20Biomark.%20Prev.&title=Genetic%20polymorphisms%20in%20oxidative%20stress%20pathway%20genes%20and%20modification%20of%20BMI%20and%20risk%20of%20non-Hodgkin%20lymphoma&author=C.%20Kim&author=T.%20Zheng&author=Q.%20Lan&author=Y.%20Chen&author=F.%20Foss&volume=21&publication_year=2012&pages=866-868&pmid=22374993&doi=10.1158/1055-9965.EPI-12-0010&)

56. Slayton M., Gupta A., Balakrishnan B., Puri V. CIDE Proteins in Human Health and Disease. Cells. 2019;8:238. doi: 10.3390/cells8030238.  [DOI](https://doi.org/10.3390/cells8030238) | [PMC free article](/articles/PMC6468517/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30871156/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cells&title=CIDE%20Proteins%20in%20Human%20Health%20and%20Disease&author=M.%20Slayton&author=A.%20Gupta&author=B.%20Balakrishnan&author=V.%20Puri&volume=8&publication_year=2019&pages=238&pmid=30871156&doi=10.3390/cells8030238&)

57. Laurencikiene J., Stenson B.M., Arvidsson Nordström E., Agustsson T., Langin D., Isaksson B., Permert J., Rydén M., Arner P. Evidence for an important role of CIDEA in human cancer cachexia. Cancer Res. 2008;68:9247–9254. doi: 10.1158/0008-5472.CAN-08-1343.  [DOI](https://doi.org/10.1158/0008-5472.CAN-08-1343) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19010897/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res.&title=Evidence%20for%20an%20important%20role%20of%20CIDEA%20in%20human%20cancer%20cachexia&author=J.%20Laurencikiene&author=B.M.%20Stenson&author=E.%20Arvidsson%20Nordstr%C3%B6m&author=T.%20Agustsson&author=D.%20Langin&volume=68&publication_year=2008&pages=9247-9254&pmid=19010897&doi=10.1158/0008-5472.CAN-08-1343&)

58. Nititham J., Taylor K.E., Gupta R., Chen H., Ahn R., Liu J., Seielstad M., Ma A., Bowcock A.M., Criswell L.A., et al. Meta-analysis of the TNFAIP3 region in psoriasis reveals a risk haplotype that is distinct from other autoimmune diseases. Genes Immun. 2015;16:120–126. doi: 10.1038/gene.2014.75.  [DOI](https://doi.org/10.1038/gene.2014.75) | [PMC free article](/articles/PMC4526682/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25521225/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genes%20Immun.&title=Meta-analysis%20of%20the%20TNFAIP3%20region%20in%20psoriasis%20reveals%20a%20risk%20haplotype%20that%20is%20distinct%20from%20other%20autoimmune%20diseases&author=J.%20Nititham&author=K.E.%20Taylor&author=R.%20Gupta&author=H.%20Chen&author=R.%20Ahn&volume=16&publication_year=2015&pages=120-126&pmid=25521225&doi=10.1038/gene.2014.75&)

59. Song B., Li H., Guo S., Yang T., Li L., Cao L., Wang J. LINC00882 Plays a Tumor-promoter Role in Colorectal Cancer by Targeting miR-3619-5p to Up-regulate CTNNB1. Arch. Med. Res. 2022;53:29–36. doi: 10.1016/j.arcmed.2021.06.001.  [DOI](https://doi.org/10.1016/j.arcmed.2021.06.001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34399990/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arch.%20Med.%20Res.&title=LINC00882%20Plays%20a%20Tumor-promoter%20Role%20in%20Colorectal%20Cancer%20by%20Targeting%20miR-3619-5p%20to%20Up-regulate%20CTNNB1&author=B.%20Song&author=H.%20Li&author=S.%20Guo&author=T.%20Yang&author=L.%20Li&volume=53&publication_year=2022&pages=29-36&pmid=34399990&doi=10.1016/j.arcmed.2021.06.001&)

60. Melenotte C., Mezouar S., Ben Amara A., Benatti S., Chiaroni J., Devaux C., Costello R., Kroemer G., Mege J.L., Raoult D. A transcriptional signature associated with non-Hodgkin lymphoma in the blood of patients with Q fever. PLoS ONE. 2019;14:e0217542. doi: 10.1371/journal.pone.0217542.  [DOI](https://doi.org/10.1371/journal.pone.0217542) | [PMC free article](/articles/PMC6557487/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31181104/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&title=A%20transcriptional%20signature%20associated%20with%20non-Hodgkin%20lymphoma%20in%20the%20blood%20of%20patients%20with%20Q%20fever&author=C.%20Melenotte&author=S.%20Mezouar&author=A.%20Ben%20Amara&author=S.%20Benatti&author=J.%20Chiaroni&volume=14&publication_year=2019&pages=e0217542&pmid=31181104&doi=10.1371/journal.pone.0217542&)

61. Hezaveh K., Kloetgen A., Bernhart S.H., Mahapatra K.D., Lenze D., Richter J., Haake A., Bergmann A.K., Brors B., Burkhardt B., et al. Alterations of microRNA and microRNA-regulated messenger RNA expression in germinal center B-cell lymphomas determined by integrative sequencing analysis. Haematologica. 2016;101:1380–1389. doi: 10.3324/haematol.2016.143891.  [DOI](https://doi.org/10.3324/haematol.2016.143891) | [PMC free article](/articles/PMC5394868/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27390358/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Haematologica&title=Alterations%20of%20microRNA%20and%20microRNA-regulated%20messenger%20RNA%20expression%20in%20germinal%20center%20B-cell%20lymphomas%20determined%20by%20integrative%20sequencing%20analysis&author=K.%20Hezaveh&author=A.%20Kloetgen&author=S.H.%20Bernhart&author=K.D.%20Mahapatra&author=D.%20Lenze&volume=101&publication_year=2016&pages=1380-1389&pmid=27390358&doi=10.3324/haematol.2016.143891&)

62. Krijgsman O., Gonzalez P., Ponz O.B., Roemer M.G., Slot S., Broeks A., Braaf L., Kerkhoven R.M., Bot F., van Groningen K., et al. Dissecting the gray zone between follicular lymphoma and marginal zone lymphoma using morphological and genetic features. Haematologica. 2013;98:1921–1929. doi: 10.3324/haematol.2013.085118.  [DOI](https://doi.org/10.3324/haematol.2013.085118) | [PMC free article](/articles/PMC3856968/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23850804/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Haematologica&title=Dissecting%20the%20gray%20zone%20between%20follicular%20lymphoma%20and%20marginal%20zone%20lymphoma%20using%20morphological%20and%20genetic%20features&author=O.%20Krijgsman&author=P.%20Gonzalez&author=O.B.%20Ponz&author=M.G.%20Roemer&author=S.%20Slot&volume=98&publication_year=2013&pages=1921-1929&pmid=23850804&doi=10.3324/haematol.2013.085118&)

63. Bates M., Spillane C.D., Gallagher M.F., McCann A., Martin C., Blackshields G., Keegan H., Gubbins L., Brooks R., Brooks D., et al. The role of the MAD2-TLR4-MyD88 axis in paclitaxel resistance in ovarian cancer. PLoS ONE. 2020;15:e0243715. doi: 10.1371/journal.pone.0243715.  [DOI](https://doi.org/10.1371/journal.pone.0243715) | [PMC free article](/articles/PMC7769460/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33370338/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&title=The%20role%20of%20the%20MAD2-TLR4-MyD88%20axis%20in%20paclitaxel%20resistance%20in%20ovarian%20cancer&author=M.%20Bates&author=C.D.%20Spillane&author=M.F.%20Gallagher&author=A.%20McCann&author=C.%20Martin&volume=15&publication_year=2020&pages=e0243715&pmid=33370338&doi=10.1371/journal.pone.0243715&)

64. Stemmler M.P. Cadherins in development and cancer. Mol. Biosyst. 2008;4:835–850. doi: 10.1039/b719215k.  [DOI](https://doi.org/10.1039/b719215k) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18633485/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Biosyst.&title=Cadherins%20in%20development%20and%20cancer&author=M.P.%20Stemmler&volume=4&publication_year=2008&pages=835-850&pmid=18633485&doi=10.1039/b719215k&)

65. Vieira A.F., Paredes J. P-cadherin and the journey to cancer metastasis. Mol. Cancer. 2015;14:178. doi: 10.1186/s12943-015-0448-4.  [DOI](https://doi.org/10.1186/s12943-015-0448-4) | [PMC free article](/articles/PMC4595126/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26438065/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Cancer&title=P-cadherin%20and%20the%20journey%20to%20cancer%20metastasis&author=A.F.%20Vieira&author=J.%20Paredes&volume=14&publication_year=2015&pages=178&pmid=26438065&doi=10.1186/s12943-015-0448-4&)

66. Yang H., Green M.R. Harnessing lymphoma epigenetics to improve therapies. Hematology Am. Soc. Hematol. Educ. Program. 2020;2020:95–100. doi: 10.1182/hematology.2020006908.  [DOI](https://doi.org/10.1182/hematology.2020006908) | [PMC free article](/articles/PMC7727577/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33275741/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hematology%20Am.%20Soc.%20Hematol.%20Educ.%20Program.&title=Harnessing%20lymphoma%20epigenetics%20to%20improve%20therapies&author=H.%20Yang&author=M.R.%20Green&volume=2020&publication_year=2020&pages=95-100&pmid=33275741&doi=10.1182/hematology.2020006908&)

67. Thermo Fisher Scientific . Axiom Genotyping Solution—Data Analysis Guide. Thermo Fisher Scientific; Waltham, MA, USA: 2011. Technical Report.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Axiom%20Genotyping%20Solution%E2%80%94Data%20Analysis%20Guide&publication_year=2011&)

68. GENEVA Coordinating Center . GWASTools Data Cleaning. GENEVA Coordinating Center; Genève, Switzerland: 2020. Technical Report.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=GWASTools%20Data%20Cleaning&publication_year=2020&)

69. Wiggs J. Quality Control Report for Genotypic Data. NIH; Bethesda, MD, USA: 2016. Technical Report.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Quality%20Control%20Report%20for%20Genotypic%20Data&author=J.%20Wiggs&publication_year=2016&)
