# An association study of HLA with levofloxacin-induced severe cutaneous adverse drug reactions in Han Chinese

## Metadata
**Authors:** Menglin Jiang, Jin Yang, Linlin Yang, Lina Wang, Ting Wang, Shengna Han, Ye Cheng, Zihua Chen, Yu Su, Lirong Zhang, Fanping Yang, Sheng-an Chen, Jin Zhang, Hao Xiong, Lanting Wang, Zhen Zhang, Li Ma, Xiaoqun Luo, Qinghe Xing
**Journal:** iScience
**Date:** 2023 Jul 13
**DOI:** [10.1016/j.isci.2023.107391](https://doi.org/10.1016/j.isci.2023.107391)
**PMID:** 37554438
**PMCID:** PMC10404721
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10404721/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC10404721/pdf/main.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC10404721/pdf/main.pdf)

## Abstract

Levofloxacin-induced severe cutaneous adverse drug reactions (LEV-SCARs) remain unexplored. An association study of human leukocyte antigen (HLA) alleles with LEV-SCARs among 12 patients, 806 healthy subjects, and 100 levofloxacin-tolerant individuals was performed. The carrier frequencies of HLA-B∗13:01 (odds ratio [OR]: 4.50; 95% confidence interval [CI]: 1.15–17.65; p = 0.043), HLA-B∗13:02 (OR: 6.14; 95% CI: 1.73–21.76; p = 0.0072), and serotype B13 (OR: 17.73; 95% CI: 3.61–86.95; p = 4.85 × 10−5) in patients with LEV-SCARs were significantly higher than those of levofloxacin-tolerant individuals. Molecular docking analysis suggested that levofloxacin formed more stable binding models with HLA-B∗13:01 and HLA-B∗13:02 than with non-risk HLA-B∗46:01. Mass spectrometry revealed that nonapeptides bound to HLA-B∗13:02 shifted at several positions after exposure to levofloxacin. Prospective screening for serotype B13 (sensitivity: 83%, specificity: 78%) and alternative drug treatment for carriers may significantly decrease the incidence of LEV-SCARs.

Subject areas: Health sciences, Dermatology, Genetics, Immunology

## Introduction

Levofloxacin (LEV) is a third-generation fluoroquinolone antibiotic and the S-isomer of racemic ofloxacin. It treats various infectious diseases due to its wide antimicrobial spectrum and powerful activities against Gram-negative bacteria, Gram-positive bacteria, and atypical bacterial pathogens. Furthermore, LEV is on the World Health Organization’s List of Essential Medicines, where it is considered the most effective and safe medicine needed in a health system.[1](#bib1)

However, LEV has been linked to various life-threatening severe cutaneous adverse drug reactions (SCARs), including Stevens-Johnson syndrome (SJS), toxic epidermal necrolysis (TEN), drug reaction with eosinophilia and systemic symptoms (DRESS), and acute generalized exanthematous pustulosis (AGEP). LEV-induced TEN was first reported in a Caucasian woman in 2002.[2](#bib2) Subsequently, several cases of LEV-induced SCARs (LEV-SCARs) have occasionally been reported, with some cases resulting in mortality.[3](#bib3)^,^[4](#bib4)^,^[5](#bib5)^,^[6](#bib6) Despite the observed low incidence, the mortality rates for SJS and TEN are 1–5% and 25–35%, respectively.[7](#bib7) In addition, because LEV is always co-administered with other drugs, all treatments must be stopped when a patient develops SCARs. After the patient recovers, preventing re-exposure to the culprit drug while also permitting the use of other medicines is of the utmost importance. However, future use of all the dosed drugs becomes contraindicated due to the inability to implicate any one drug based on clinical grounds alone. Consequently, LEV-SCARs seriously affect the medication’s safety and significantly restrict treatment options. Thus, an effective method to predict the risk of SCARs in patients taking LEV is required.

SJS/TEN, DRESS, and AGEP are all T cell-mediated drug reactions. Previous evidence supporting that the pathogenesis of SCARs involves major histocompatibility complex (MHC) restricted presentation of drug or drug metabolites for T cell activation, and drug-specific T cells orchestrate the effector response and produce the final SCARs phenotype. The pharmacogenetic study of allopurinol-induced SCARs identified a strong association of *HLA-B∗58:01* with allopurinol-induced DRESS and SJS/TEN in Han Chinese[8](#bib8) and Portuguese population.[9](#bib9) Another study of carbamazepine hypersensitivity revealed that *HLA-A∗31:01* was associated with carbamazepine-induced DRESS and SJS/TEN, and one subject with AGEP also was positive for *HLA-A∗31:01*.[10](#bib10) Overall, the evidence points to different SCAR entities could sometimes have the same association with *HLA* polymorphisms.

Previously implicated genes in the development of SCARs with clinical application prospects are mostly concentrated in the human leukocyte antigen (HLA) loci, especially in HLA class I loci. For instance, *HLA-B∗15:02*, *HLA-B∗57:01*, and *HLA-B∗58:01* have been recommended as predictors of carbamazepine-induced SJS/TEN,[11](#bib11) abacavir-induced hypersensitivity,[12](#bib12) and allopurinol-induced SCARs,[13](#bib13) respectively. Research has also demonstrated that pre-prescription genotyping for the *HLA* risk allele and withholding causative drugs from positive patients dramatically diminishes the incidence of related SCARs. However, the genetic factors that may be relevant to LEV-SCARs have not been well studied. Therefore, we conducted an association study of 12 patients with LEV-SCARs to identify their genetic risk factors. Furthermore, we employed mass spectroscopy to analyze the effect of LEV on the interactions between HLA molecules and peptides to explore the potential molecular mechanism of LEV-SCARs.

## Results

### Clinical characteristics of LEV-SCAR patients

Twelve subjects who met the inclusion criteria for clinical diagnosis of LEV-SCARs were recruited in our study, including two patients with DRESS, one with AGEP, four with SJS, four with TEN, and one with SJS-TEN overlap ([Table 1](#tbl1)). The LEV was identified as the most potential culprit drug based on the algorithm of drug causality for epidermal necrolysis (ALDEN) or Naranjo algorithm ([Table S1](#mmc1)). There were nine males and three females, with a mean age of 46.1 ± 21.2 years (range 19–89 years). The sex and age of SCAR patients showed no significant difference with two control groups. The latency period from LEV initiation to the first symptoms of SCARs ranged from 1 to 25 days, with an average of 9.8 ± 8.5 days. Therein, the latency period of two patients with DRESS were 12 days and three days, respectively. Ten SCAR patients were exposed to combined medication, but no adverse reactions related to the prescribed drugs other than LEV were found in their medical records.

### Table 1.

| No. | Gender/Age (years) | SCARs | Combined drugs | Latency (days) | Mucosal involvement | Systemic manifestations | HLA-A genotype | HLA-B genotype | HLA-C genotype |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| 1 | M/27 | SJS | Warfarin; Perindopril; Cefaclor; Hydrochlorothiazide | 22 | Oral | LFI | 11:01/11:02 | 18:01/51:02 | 07:04/15:02 |
| 2 | F/59 | TEN | Amoxicillin, Armillarisin A | 25 | Oral, genitalia | LFI, RFI | 02:01/30:01 | 13:02/40:01 | 06:02/07:02 |
| 3 | M/19 | SJS | Cefuroxim; Cefamedin; Ibuprofen; Metamizole | 3 | Oral, eye, genitalia | LFI | 02:53/24:02 | 13:01/55:02 | 01:02/03:04 |
| 4 | F/32 | SJS | Asmeton | 7 | Oral, genitalia | None | 03:01/26:01 | 08:01/13:02 | 06:02/07:02 |
| 5 | M/46 | DRESS | Azithromycin; Cold medicine | 12 | Oral, eye | LFI | 02:10/30:01 | 13:02/40:06 | 06:02/08:01 |
| 6 | M/33 | TEN | – | 2 | Oral, genitalia | LFI | 30:01/33:03 | 13:02/44:03 | 06:02/07:01 |
| 7 | F/59 | TEN | Isepamicin | Unclear | Oral, genitalia | RFI | 24:02/30:01 | 13:02/40:01 | 06:02/07:02 |
| 8 | M/59 | TEN | Clarithromycin; Rabeprazole | 9 | Oral, eye | LFI | 11:01/30:01 | 13:02/15:02 | 06:02/08:01 |
| 9 | M/74 | SJS | Cefixime; Cefotiam; Ceftazidime; Fluconazole; Aminopyrine | 21 | Oral, genitalia | LFI, RFI | 11:01/33:03 | 40:01/58:01 | 03:02/03:04 |
| 10 | M/89 | SJS-TEN | Cefmetazole | 3 | Oral, genitalia | RFI | 11:01/11:01 | 13:01/15:02 | 03:04/08:01 |
| 11 | M/19 | AGEP | – | 1 | None | None | 11:01/24:02 | 13:01/40:01 | 03:04/07:02 |
| 12 | M/37 | DRESS | Compound Aminopyrine and Antipyrine; Acetaminophen | 3 | None |   | 02:01/11:01 | 13:01/38:02 | 03:04/07:02 |

Table 1 Caption: Clinical characteristics and HLA genotypes of patients with LEV-SCARs

### A comparative study with the general population controls

Eight classical HLA loci were successfully genotyped using whole-exome sequencing (WES) data from 11 patients with SCARs and 806 population controls. A total of 31 *HLA* alleles were discovered in SCAR patients with carrier frequencies >10.0%. Of these alleles, *HLA-B∗13:01* (OR: 7.37; 95% CI: 2.10–25.91; *P*c = 0.027) and *HLA-B∗13:02* (OR: 5.42; 95% CI: 1.63–18.01; *P*c = 0.031) were both dominantly enriched in patients with LEV-SCARs, demonstrating carrier frequencies of 36.4% and 54.5%, respectively, while the carrier frequencies in the population controls were 7.2% and 18.1%, respectively ([Table 2](#tbl2)). Furthermore, the HLA-B13 serotype, including the principal alleles of *HLA-B∗13:01* and *HLA-B∗13:02*, showed a stronger association with LEV-SCARs (OR: 30.91; 95% CI: 3.93–243.01; p = 7.91 × 10^−6^). Two strongly linked alleles (r^2^ = 0.92), *HLA-DRB1∗12:02* (OR: 5.95; 95% CI: 1.78–19.86; *P*c = 0.030) and *HLA-DQA1∗06:01* (OR: 5.56; 95% CI: 1.67–18.55; *P*c = 0.038), also showed strong associations with LEV-SCARs that were lost after conditioning with serotype B13.

### Table 2.

| HLA genotype | Carrier, No. (%) |  | P | Pca | Odd ratio (95% CI) |
| --- | --- | --- | --- | --- | --- |
| LEV-SCAR patients, (n = 11) | Population controls, (n = 806) |  |  |  |  |
| A∗30:01 | 5 (45.5) | 132 (16.4) | 0.024 | 0.12 | 4.26 (1.28–14.15) |
| B∗13:01 | 4 (36.4) | 58 (7.2) | 6.63 × 10−3 | 0.027 | 7.37 (2.1–25.91) |
| B∗13:02 | 6 (54.5) | 146 (18.1) | 7.68 × 10−3 | 0.031 | 5.42 (1.63–18.02) |
| B∗15:02 | 2 (18.2) | 32 (4) | 0.073 | 0.29 | 5.38 (1.12–25.9) |
| C∗03:04 | 5 (45.5) | 122 (15.1) | 0.018 | 0.071 | 4.67 (1.4–15.55) |
| C∗06:02 | 6 (54.5) | 205 (25.4) | 0.039 | 0.16 | 3.52 (1.06–11.65) |
| DQA1∗02:01 | 6 (54.5) | 198 (24.6) | 0.033 | 0.13 | 3.68 (1.11–12.2) |
| DQA1∗06:01 | 5 (45.5) | 105 (13) | 0.01 | 0.038 | 5.56 (1.67–18.55) |
| DQB1∗02:02 | 6 (54.5) | 168 (20.8) | 0.015 | 0.061 | 4.56 (1.37–15.11) |
| DQB1∗03:01 | 8 (72.7) | 290 (36) | 0.022 | 0.088 | 4.74 (1.25–18.02) |
| DRB1∗07:01 | 6 (54.5) | 198 (24.6) | 0.033 | 0.13 | 3.68 (1.11–12.2) |
| DRB1∗12:02 | 5 (45.5) | 99 (12.3) | 7.48 × 10−3 | 0.03 | 5.95 (1.78–19.86) |
| Serotype B13 | 10 (90.9) | 197 (24.4) | 7.91 × 10−6 |   | 30.91 (3.93–243.01) |

Table 2 Caption: Frequencies of HLA alleles in patients with LEV-SCARs and population controls

Previous studies have indicated that SCARs are more likely to be associated with HLA class I alleles, and serotype B13 showed the most significant association with LEV-SCARs. Thus, high-resolution *HLA-A*, *-B*, and *-C* genotyping was performed for 12 SCAR patients, including 11 cases analyzed by WES, to validate the accuracy of *HLA* genotyping based on WES data. We demonstrated that the alleles of *HLA-A*, *-B*, and *-C* genotyped from the WES data were reliable and consistent with the results of the PCR-SSO analysis, except for one patient whose *HLA-C* genotype was *HLA-C∗07:01*/*-C∗06:02* while *HLA-C∗07:06*/*-C∗06:02* by exome sequencing.

### A comparative study with the LEV-tolerant controls

Next, we recruited 100 LEV-tolerant patients and performed a comparative study between the SCARs and tolerant groups to exclude the influence of clinical factors. The carrier frequencies of *HLA-B∗13:01* (OR: 4.50; 95% CI: 1.15–17.65; p = 0.043) and *HLA-B∗13:02* (OR: 6.14; 95% CI: 1.73–21.76; p = 0.0072) in the tolerant group were 10.0% and 14.0%, respectively, which were significantly lower than those in the patients with LEV-SCARs ([Table 3](#tbl3)). The B13 serotype was present in 83.3% (10/12) of the patients with LEV-SCARs, but only in 22.0% (22/100) of LEV-tolerant individuals (OR: 17.73; 95% CI: 3.61–86.95; p = 4.85 × 10^−5^). This suggests that B13 serotype is a significant risk factor for LEV-SCARs, with the risk of LEV-SCARs in carriers of B13 serotype being 17.73 times that in non-carriers.

### Table 3.

| HLA genotype | Carrier, No. (%) |  | P | Odd ratio (95% CI) |
| --- | --- | --- | --- | --- |
| LEV-SCAR patients, (n = 12) | Tolerant controls, (n = 100) |  |  |  |
| B∗13:01 | 4 (33.3) | 10 (10.0) | 0.043 | 4.5 (1.15–17.65) |
| B∗13:02 | 6 (50.0) | 14 (14.0) | 7.21 × 10−3 | 6.14 (1.73–21.76) |
| Serotype B13 | 10 (83.3) | 22 (22.0) | 4.85 × 10−5 | 17.73 (3.61–86.95) |

Table 3 Caption: Frequencies of HLA alleles in patients with LEV-SCARs and LEV-tolerant controls

### Molecular docking analyses of HLA proteins with LEV

Genetic analysis indicated that *HLA-B∗13:01*, *HLA-B∗13:02*, and serotype B13 were associated with LEV-SCARs. To confirm this, molecular simulations were performed to analyze the interactions between HLA proteins and LEV. HLA-B∗46:01, present at a higher frequency of carriers in the population controls than in the patients with LEV-SCARs, was used as a control.

The analysis demonstrated that LEV can stably bind to HLA-B∗13:01 and HLA-B∗13:02, with docking scores of −5.70 and −6.83 respectively ([Figure 1](#fig1)A). Strong hydrophobic forces were formed between the LEV and hydrophobic cores in both proteins consisting of Y74, Y9, I66, Y7, Y159, Y99, and L156. Moreover, LEV was closer to the hydrophobic core in HLA-B∗13:02 due to the variant R97T in the antigen-binding pocket ([Figures 1](#fig1)B and 1C). Compared to HLA-B∗13:01 and HLA-B∗13:02, the common protein HLA-B∗46:01 possesses more polar variants, such as I66K and V152E, around the groove region, resulting in the formation of stable salt bridges with LEV that prevent the ligand from entering the groove region ([Figures 1](#fig1)B and 1D). As a result, the interaction between LEV and HLA-B∗46:01 occurred outside the groove region, demonstrating a worse docking score (−5.07; [Figure 1](#fig1)A). Thus, molecular docking analysis suggested that HLA-B∗13:01 and HLA-B∗13:02 formed stable binding models with LEV, providing further evidence for the results of genetic analyses. As the combination therapy was common in the case group, the interactions between risk HLA proteins and the involved drugs other than LEV were analyzed. The results supported LEV as the most suspicious culprit drug in LEV-SCAR patients ([Figures S1](#mmc1) and [S2](#mmc1)).

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ffca/10404721/06f80fd07f53/gr1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=10404721_gr1.jpg)

Interactions between levofloxacin and HLA proteins (A) Binding modes of levofloxacin (LEV, green) in HLA-B∗13:01(slate), HLA-B∗13:02(white), and HLA-B∗46:01(salmon). LEV and residues forming the binding pocket are shown in sticks. (B) Sequence alignments of HLA-B∗13:01, HLA-B∗13:02, and HLA-B∗46:01. Different residues forming the binding pocket are marked with blue arrows. (C) Superimposition of the binding modes in panel A. For clarity, HLA proteins are shown as transparent cartoon. Residues and LEV are shown in sticks. (D) Electrostatic surfaces of HLA-B∗13:01, HLA-B∗13:02, and HLA-B∗46:01 generated by PyMOL. The PyMOL Charge-smoothed potential of each protein is shown under the picture.

### Pharmacogenetic predictors of LEV-SCARs

Based on the association study, *HLA-B∗13:01*, *HLA-B∗13:02*, and serotype B13 are candidate predictors of SCARs in patients treated with LEV. As predictors for LEV-SCARs, the presence of *HLA-B∗13:01* and *HLA-B∗13:02* had sensitivities of 33% and 50%, respectively, as well as specificities of 90% and 86%, respectively. As a risk predictor for LEV-SCARs in the Han Chinese population, serotype B13 was present in 10 (83.3%) of the 12 subjects with LEV-SCARs and only in 22 (22%) of the 100 LEV-tolerant controls (sensitivity: 83%; specificity: 78%). As regards the SJS/TEN, *HLA-B∗13:01*, *HLA-B∗13:02*, and serotype B13 showed the sensitivities of 22%, 56%, and 78%, respectively. These data suggest that serotype B13 is a superior predictor for LEV-SCARs in terms of sensitivity and specificity.

### Analysis of the effect of LEV on the interactions between HLA-B∗13:02 and peptides

The structural changes in the antigen-binding clefts affect the specific interactions between HLA and antigen peptides. Since abacavir-hypersensitivity has been identified to be associated with alterations in the antigenic repertoire of HLA-B∗57:01 induced by the drug molecule,[14](#bib14) we explored whether LEV-SCARs have a similar mechanism by analyzing peptides bound to HLA-B∗13:02 in the presence or absence of LEV using mass spectrometry. This was performed on HLA-B∗13:02 due to its more significant association with LEV-SCARs compared with that of HLA-B∗13:01. In total, we sequenced 1,824 and 2,449 peptides in the LEV and control groups, respectively, with nonapeptides accounting for the highest proportion at approximately 50%. Among these peptides, 131 and 279 nonapeptides were sequenced specifically in the LEV and control groups, respectively. The sequences of the nonapeptides bound to HLA-B∗13:02 in the presence of LEV demonstrated shifts at several positions. The average frequencies of L2 and Y8 of the nonapeptides in the control group were 18.4% and 0.5%, respectively. Whereas these were significantly decreased after exposure to LEV, demonstrating frequencies of 16.7% and 0.1%, respectively. Moreover, multiple amino acids at position 2 of the nonapeptides were increased after drug treatment, including N2 (from 3.3 to 4.1%), G2 (from 3.6 to 4.0%), R2 (from 3 to 3.4%), and F2 (from 0.9 to 1.2%), potentially resulting in improved hydrophilicity. However, the magnitude of the LEV-induced repertoire shift was smaller than that observed in abacavir-hypersensitivity.

The peptide QFKDLKVNL (pep-Q), which was only detected in the LEV group, demonstrated poor affinity for HLA-B∗13:02.[15](#bib15) In contrast, peptide RQDLTTLDV (pep-R), which was only detected in the control group, showed high affinity for HLA-B∗13:02.[15](#bib15) Thus, we selected these peptides for further analysis of the interactions among HLA-B∗13:02, LEV, and peptides through molecular simulation ([Figure 2](#fig2)). In the absence of LEV, pep-R tightly bound to the antigen-binding cleft, forming extensive hydrogen bonds with polar residues along the binding pocket ([Figure 2](#fig2)B). Meanwhile, L4 and L7 of pep-R made hydrophobic contact with the bottom of the cleft consisting of Y9, Y74, W95, Y99, L116, and Y123 ([Figure 2](#fig2)D). Compared with pep-R, pep-Q possesses more residues with larger side chains, such as F2, K3 and K6, so it could not fit in the narrow cleft as well as pep-R ([Figure 2](#fig2)A), and the C-terminal end even protruded from the pocket ([Figure 2](#fig2)C), resulting in a worse docking score. When LEV was bound to HLA-B∗13:02, it occupied a large proportion of the antigen-binding cleft, preventing both peptides from binding at the base of the pocket. As a result, pep-R was located at an exterior site with fewer polar interactions, and L4 and L7 were exposed to the unfavorable solvent environment, leading to much reduced binding affinity ([Figures 2](#fig2)B and 2D). Conversely, the larger space outside the cleft could better accommodate pep-Q. More direct HLA-B∗13:02-pep-Q interactions were observed, and F2, L5, and V7 of pep-Q formed strong hydrophobic patterns with LEV ([Figures 2](#fig2)A and 2C), which provides a basis for understanding why LEV enhanced the affinity of pep-Q with HLA-B∗13:02.

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ffca/10404721/5045ce2da261/gr2.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=10404721_gr2.jpg)

Effects of levofloxacin on the interactions between HLA-B∗13:02 and candidate peptides (A and B) Different binding models of peptide QFKDLKVNL (pep-Q, A) and peptide RQDLTTLDV (pep-R, B) in HLA-B∗13:02 in the absence or presence of levofloxacin (LEV, green). All amino acid residues from peptides are depicted as sticks. Resides involved in polar interactions are labeled. (C and D) Comparison of binding modes in two situations. Resides involved in hydrophobic interactions are labeled.

These observations for LEV-HLA-B∗13:02-peptide complexes provided further evidences for the impact of LEV on the interaction between HLA-B∗13:02 and peptides as the possible pathogenic mechanism underlying LEV-SCARs.

## Discussion

The relationships between SCARs and *HLA* alleles are complicated and specific to drug, phenotype, and race. Not all patients with carbamazepine-induced SJS/TEN carried *HLA-B∗15:02*; other alleles, including *HLA-B∗15:21*, *HLA-B∗15:11*, *HLA-B∗15:08*, and *HLA-A∗31:01*, were also associated with carbamazepine hypersensitivity.[10](#bib10)^,^[16](#bib16)^,^[17](#bib17) The related alleles mostly belonged to serotype B75 and shared in their antigen-binding groove amino acids that play important roles in the interactions between culprit drugs and risk HLA proteins.[18](#bib18) Another relationship has been demonstrated between HLA allelic groupings (including *HLA-C∗04:01*, *HLA-C∗05:01* and *HLA-C∗18:01*) and nevirapine hypersensitivity, which may be attributed to similar peptide binding properties and F pocket structure.[19](#bib19) In the present study, serotype B13 was strongly associated with LEV-SCARs, demonstrating a risk in LEV-SCARs development in carriers 17.73 times higher than that in the non-carriers. Based on the HLA directory (2008)[20](#bib20) and the allele frequency database,[21](#bib21) the serotype B13 HLA allele mainly includes *HLA-B∗13:01* and *HLA-B∗13:02* in the Chinese population. The two molecules demonstrated only three amino acid variation, and multiple shared amino acids form a hydrophobic antigen-binding pocket to properly accommodate LEV. However, *HLA-B∗13:01* and *HLA-B∗13:02* only identify 83% of ‘at-risk’ patients. Other alleles involving in the development of LEV-SCARs may not have been discovered.

The SCARs are known as T-cell-mediated immune disorders elicited by drugs. The key points of molecular mechanisms of SCARs include specific drug antigens interacting with the specific HLA molecule, involvement of specific T cell receptors, induction of T-cell-mediated responses, and cell death mechanism.[22](#bib22) The clinical heterogeneity of SCARs might be explained by the activation of different effector or regulatory cells secreting specific cytokines.[23](#bib23) Our study revealed the same association of serotype B13 with LEV-induced different SCAR entities, including SJS/TEN, DRESS, and AGEP. The risk allele of *HLA-B∗13:01* has been previously reported to be associated with the dapsone hypersensitivity[24](#bib24) and salazosulfapyridine-induced DRESS.[25](#bib25) We have checked the medical history and verified that the patients in the study had not been treated with sulfonamides or dapsone prior to the development of the cutaneous reactions.

In the present study, we identified that LEV could hydrophobically bind to the A-E subpocket of the antigen-binding groove in HLA-B∗13:01, while dapsone and salazosulfapyridine have been reported to bind toward the F pocket of the same groove.[25](#bib25)^,^[26](#bib26)^,^[27](#bib27)^,^[28](#bib28) The differences in binding site may be attributed to their different chemical scaffolds. LEV has a bigger structure compared to dapsone and salazosulfapyridine, which was not suitable in the narrow F pocket and, therefore, favorably located in the larger cleft consisting of A-E pocket. Nevertheless, as all these small molecules possess aromatic rings, they presented a common binding characteristic that formed extensive hydrophobic interactions with HLA-B∗13:01. These models also provided clues to ligand selectivity. The different residues R97 in HLA-B∗13:01 and T97 in HLA-B∗13:02 significantly changed LEV binding poses in two receptors, making LEV closer to the bottom of the antigen-binding cleft in HLA-B∗13:02 than in HLA-B∗13:01, which agrees with the docking models of dapsone.[28](#bib28) Jiang et al. demonstrated that three different amino acids in the antigen-binding site (I94, I95, and R97 in HLA-B∗13:01, and T94, W95, and T97 in HLA-B∗13:02) played important roles in dapsone selectivity through creating steric hindrance, changing van der Waals interactions and affecting the electrostatic properties of the dapsone-binding pocket.[26](#bib26)

In 2012, the three-dimensional crystal structure of the abacavir-HLA-B∗57:01-peptide complex was resolved, which explained the molecular mechanism behind abacavir-induced hypersensitivity.[14](#bib14)^,^[29](#bib29) Under normal circumstances, the self-peptides displayed by HLA-B∗57:01 are ignored by CD8^+^ T cells. However, abacavir binds to HLA-B∗57:01, occupying the C-F pocket region of the peptide binding groove and changing its shape and chemistry. The HLA-B∗57:01 immunopeptidome dramatically changed, and these newly presented peptides would be recognized as foreign, thus stimulating CD8^+^ T-cell proliferative responses and triggering an abnormal immune response.[14](#bib14) To investigate the molecular mechanism of LEV-SCARs, we characterized the peptides bound to affinity-purified HLA-B∗13:02 molecules from untreated and LEV-treated cells by mass spectrometry. This helps to understand the drug antigen presentation stage, a common phase of LEV-SCARs. We observed some shifts in the preferred amino acid induced by LEV, which showed smaller magnitudes than the changes in abacavir hypersensitivity. This variation may be resulted from the different conformation alterations in the peptide binding cleft of risk HLA molecules. In spite of this, the results prompted that the affinity of peptides to HLA-B∗13:02 is influenced by LEV, underpinning the development of LEV-SCARs.

Delayed-type hypersensitivity is well-documented as T cell mediated.[30](#bib30) The antigen signals presented by HLA on the surface of antigen-presenting cells need to be recognized by T cells to initiate an immune response. Thus, the TCR repertoire is expansive to respond to diverse antigenic peptides. Previous studies have identified the proliferation of drug-specific T cells and revealed oligoclonal expansion of TCR in hypersensitivity induced by abacavir, carbamazepine, and allopurinol.[14](#bib14)^,^[31](#bib31)^,^[32](#bib32) Pan et al. found a predominate and public TCR clonotype in patients with carbamazepine-induced SJS/TEN, whereas broadly polyclonal TCR usage was discovered in abacavir hypersensitivity. The distinct diversity in TCR repertoire was consistent with the different pathogenesis of hypersensitivity. The development of carbamazepine-induced SJS/TEN was related to the ‘p-i’ concept, while abacavir hypersensitivity was associated with the ‘altered peptide repertoire’ model. Hence, the TCR repertoire in LEV-SCARs will be explored to further illuminate the underlying mechanism.

In the development of quinolone antibiotics, it has been observed that the molecular structures of drugs are not only related to pharmacological activity, but also to drug toxicity. The substituents of an amino or methyl at position 5, fluorine at position 6, and fluorine or chlorine at position 8 are related to genotoxicity.[33](#bib33) Drug toxicity is a key factor in druggability. About 4% of new drug withdrawals from the market are due to adverse reactions, which poses a substantial challenge in drug development.[34](#bib34) Abacavir-induced severe hypersensitivity reactions occur in 4–8% of patients receiving therapy, associated with considerable global morbidity and mortality. Paul et al. indicated that certain analogues of abacavir could retain antiviral activity, but discard the T cell immunogenicity.[35](#bib35) Therefore, investigating the molecular mechanism of LEV-SCARs and their interactions with chemical structures can provide guidance for the effective and rational optimization of quinolone antibiotics.

In summary, this study revealed the association of LEV-SCARs with strong genetic predispositions, identifying serotype B13 as a risk factor for LEV-SCARs with a predictive sensitivity of 83% and specificity of 78%. We believe that the prospective screening of serotype B13, coupled with an alternative drug treatment for carriers, may significantly decrease the incidence of LEV-SCARs in the Han Chinese population.

### Limitation of the study

One limitation of this study was the small sample size. The prevalence of combination therapy in patients receiving LEV posed a huge challenge for the identification of causative drugs and sample collection. However, the relatively small sample size in our study is sufficient to capture clinically useful genetic factors with remarkable genetic effects.[36](#bib36) Moreover, conservative Bonferroni corrections for multiple comparisons were conducted to eliminate false-positive results. Accordingly, the likelihood of a type I error was considerably low in the present study. Our study was powered to identify a strong association between serotype B13 and LEV-SCARs in a Han Chinese population and we cannot generalize at this point to people of other ancestries in whom HLA associations with LEV-SCARs will need to be independently studied. In addition, due to the influences of cell numbers and detergents, the peptides obtained from mass spectrometry were numbered. Furthermore, we did not validate these peptides using T cells in patients with LEV-SCARs, making it non-conducive to analyzing the effect of LEV on the antigenic repertoire of HLA-B∗13:02.

## STAR★Methods

### Key resources table

| REAGENT or RESOURCE | SOURCE | IDENTIFIER |
| --- | --- | --- |
| Biological samples |  |  |
|   |  |  |
| All patients of the LEV-SCARs group, LEV-tolerant group, and the population control group | Huashan Hospital of Fudan University (Shanghai, China) | N/A |
|   |  |  |
| Critical commercial assays |  |  |
|   |  |  |
| SureSelect Human All Exon V6 | Agilent | Cat#5190-8865 |
| ANTI-Flag M2 affinity gel | Sigma-Aldrich | Cat#A2220 |
| Amicon Ultra-0.5 | Merck | Cat#UFC500308 |
| HiPPR detergent removal resin | Thermo Fisher Scientific | Cat#88306 |
|   |  |  |
| Chemicals, peptides, and recombinant proteins |  |  |
|   |  |  |
| Saline | China Otsuka | H44020184 |
| Levofloxacin | Xinchang Pharmaceutical Factory | H20033965 |
| Trifluoroacetic acid | Sangon Biotech | Cat#A501480 |
| NP-40 | Thermo Fisher Scientific | Cat#85125 |
|   |  |  |
| Experimental models: Cell lines |  |  |
|   |  |  |
| HLA class I-defective cell line HMy2.C1R | National Collection of Authenticated Cell Cultures | Cat#GNHu21; RRID: CVCL_3714 |
| HMy2.C1R transfected with the HLA-B∗13:02 plasmid | This study | N/A |
|   |  |  |
| Recombinant DNA |  |  |
|   |  |  |
| Plasmid: HLA-B∗13:02 FLAG tag at C-terminus | This study | Vector: pCMV6-Entry |
|   |  |  |
| Deposited data |  |  |
|   |  |  |
| HLA genotyping data with WES | This study | Figshare: http://doi.org/10.6084/m9.figshare.23311421 |
|   |  |  |
| Software and algorithms |  |  |
|   |  |  |
| HLA-HD | Kawaguchi et al.37 | http://www.genome.med.kyoto-u.ac.jp/HLA-HD |
| HPEPDOCK | N/A | http://huanglab.phys.hust.edu.cn/hpepdock/ |
| R Studio | N/A | https://www.rstudio.com/ |
### Resource availability

#### Lead contact

Requests for additional information and resources should be directed to the lead contact, Qinghe Xing ([xingqinghe@hotmail.com](http://mailto:xingqinghe@hotmail.com)).

#### Materials availability

The plasmid and cell line generated in this study are available upon request from the [lead contact](#sec4.2.1) with a completed Material Transfer Agreement.

### Experimental model and study participant details

#### Patient recruitment

Twelve patients diagnosed with LEV-SCARs were recruited from the dermatology ward and emergency ward of Huashan Hospital of Fudan University (Shanghai, China) between 2007 and 2020. All the diagnoses were performed by two independent dermatologists who examined photographs, pathological slides, and medical records. The diagnosis of SJS/TEN was based on the appearance of mucous membrane erosions and epidermal detachment (positive Nikolsky’s sign), and confirmed by skin biopsies showing full-thickness necrosis of the epidermis on pathological examination.[38](#bib38) The diagnostic criterion for DRESS was the scoring system of the Registry of Severe Cutaneous Adverse Reactions (RegiSCAR) group,[39](#bib39) including cutaneous involvement with typical skin eruptions, fever (≥38.5°C), enlarged lymph nodes (two or more sites, ≥1 cm), presence of atypical lymphocytes and eosinophilia, organ involvement (e.g., liver, kidney, and lung), time of resolution, and the evaluation of other potential causes. The patients diagnosed with probable or definite DRESS by the RegiSCAR were recruited in the investigation. Diagnosis of AGEP was evaluated according to the validation system of Sidoroff.[40](#bib40) The scores of the algorithm of drug causality for epidermal necrolysis (ALDEN)[41](#bib41) and Naranjo algorithm[42](#bib42) were calculated for each drug taken by patients. Only cases in which the LEV was considered highly probable or probable were included in the study.

Two control groups were included in this study. A total of 806 Han Chinese subjects without any clinical history of SCARs were used as the population control group. The LEV-tolerant group consisted of 100 individuals from the same hospital, who had received LEV without any evidence of adverse reactions for at least three months. Both case and control groups comprised unrelated Han Chinese patients. This study was approved by the ethics committee of Huashan Hospital with informed consent obtaining from all participants.

### Method details

#### Whole-exome sequencing (WES) and HLA genotyping

We performed WES on 11 out of the 12 patients with LEV-SCARs and on all 806 subjects in the population control group, except for one patient without a sufficient blood sample. All samples were prepared using SureSelect Human All Exon V6 (Agilent) and subsequently sequenced using Illumina DNA sequencing. Eight classical HLA alleles (*HLA-A, -B, -C, -DPA1, -DPB1, -DQA1, -DQB1*, and *-DRB1*) were genotyped using HLA-HD[37](#bib37) version 1.3.0 with WES data.

High-resolution *HLA* genotyping of *HLA-A*, *-B*, and *-C* in 12 patients with LEV-SCARs and 100 LEV-tolerant individuals was performed using Mentality Bio-Tech (Beijing, China) with the polymerase chain reaction-sequence specific oligonucleotide (PCR-SSO) method.

#### Molecular docking analysis of LEV with HLA molecules

To investigate the interaction modes between LEV and HLA molecules, docking analyses were performed. HLA-B∗13:01, HLA-B∗13:02, and HLA-B∗46:01 were set as the receptors, and LEV was set as the ligand. The structures of HLA-B∗13:01 and HLA-B∗13:02 were obtained using homology modeling based on the crystal structure of HLA-B∗52:01 (PDB: [3W39](http://pdb:3W39)) with 96% identity, and the crystal structure of HLA-B∗46:01 was obtained from the PDB database (PDB: [4LCY](http://pdb:4LCY)). The structures of HLA and LEV were processed to obtain the pdbqt format files. The receptor-grid file was generated using an enclosing box with dimensions 28 Å × 28 Å × 28 Å centered on the groove region of the receptor. The processed drug molecules were then docked into the assumed binding pocket of HLA with default values, outputting the top 50 conformations of each ligand respectively. The most reliable binding poses were selected according to the favorable interaction energy and visual inspection.

#### Analysis of the interactions among LEV, HLA-B∗13:02, and peptides

An HLA class I-defective cell line HMy2.C1R (RRID: CVCL_3714) transfected with the *HLA-B∗13:02* allele was used for peptide elution.[43](#bib43)^,^[44](#bib44) The cells were treated with LEV at 50 μg/mL (LEV group) or an equal volume of saline (the control group) for 48h for peptide elution. The immunoaffinity purification were performed with ANTI-Flag M2 affinity gel (Sigma-Aldrich), and bound complexes were eluted by acidification with 6% trifluoroacetic acid. The eluent was ultrafiltered through Amicon Ultra 3K (Merck), and the remaining NP-40 (MW < 3000) was removed using HiPPR detergent removal resin (Thermo Fisher Scientific). High-performance liquid chromatography-tandem mass spectrometry analysis was performed on a Nano Aquity UPLC system (Waters Corporation) connected to a quadrupole-Orbitrap mass spectrometer (Thermo Fisher Scientific) equipped with an online nano-electrospray ion source. To further investigate the effect of LEV on the interactions between HLA-B∗13:02 and candidate peptides, we docked peptides into both an apo HLA-B∗13:02 molecule and HLA-B∗13:02-LEV complexes using the HPEPDOCK server ([http://huanglab.phys.hust.edu.cn/hpepdock/](http://huanglab.phys.hust.edu.cn/hpepdock/)).[45](#bib45) For each docking run, top 10 conformations were generated and then analyzed by visual inspection.

### Quantification and statistical analysis

The carrier frequency was calculated as the ratio of the number of subjects carrying the allele to the total number of the group. Alleles with carrier frequencies >10.0% in the SCARs group were compared between LEV-SCARs patients and two control groups using Fisher’s exact test. Overall odd ratios (ORs) with corresponding 95% confidence intervals (CIs) were calculated to quantify the association between the presence of the *HLA* allele and LEV-SCARs. The *p* values were corrected using Bonferroni correction for multiple comparisons (*P*c). Associations were deemed statistically significant at *p* value <0.05.

## Acknowledgments

This work was supported by the National Natural Science Foundation of China [grant numbers 81872525, 81472873, 31972880, 31371274, 82173422, 31900895]; National Key Research and Development Program [grant numbers 2021YFC2700800, 2016YFC0905000, 2016YFC0905001, 2018YFA0508100]; Natural Science Foundation of Henan Province [grant numbers 182300410010]; and Shanghai Municipal Commission of Science and Technology Research Project [grant numbers 19JC1411001]. We also thank the support of Beijing PARATERA Tech CO., Ltd (URL: [https://paratera.com/](https://paratera.com/)) and National Supercomputing Center in Zhengzhou for providing computational resources for this study. The funder had no role in study design, data collection, data analysis, interpretation, or writing of this report.

### Author contributions

Q.X., X.L., M.J., J.Y., and L.Y. developed the overall concept and designed research. M.J., L.Y., L.W., T.W., Y.C., and H.X. conducted the experiments. M.J., J.Y., L.Y., L.W., T.W., and Y.C. analyzed data. S.H., Z.C., Y.S., L.Z., F.Y., S-A.C., J.Z., H.X., L.W., Z.Z., and L.M. provided reagents and/or conceptual input. M.J., L.Y., and Q.X. drafted the manuscript. Q.X., X.L., L.Y., and S.H. provided funding. Q.X. and X.L provided supervision. All authors discussed the results and contributed to the final manuscript. Besides, we would like to thank Editage ([www.editage.cn](http://www.editage.cn)) for English language editing.

### Declaration of interests

The authors declare no competing interests.

Published: July 13, 2023

## Footnotes

## Contributor Information

Xiaoqun Luo, Email: luoxiaoqun913@163.com.

Qinghe Xing, Email: xingqinghe@hotmail.com.

## Supplemental information

## Data and code availability

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

### Supplementary Materials

### Data Availability Statement

## References

1. WHO Expert Committee . Vol. 994. World Health Organization; 2015. pp. 1–546. (The Selection and Use of Essential Medicines).  [Google Scholar](https://scholar.google.com/scholar_lookup?WHO%20Expert%20Committee%20.%20Vol.%20994.%20World%20Health%20Organization;%202015.%20pp.%201%E2%80%93546.%20(The%20Selection%20and%20Use%20of%20Essential%20Medicines).)

2. Digwood-Lettieri S., Reilly K.J., Haith L.R., Jr., Patton M.L., Guilday R.J., Cawley M.J., Ackerman B.H. Levofloxacin-induced toxic epidermal necrolysis in an elderly patient. Pharmacotherapy. 2002;22:789–793. doi: 10.1592/phco.22.9.789.34074.  [DOI](https://doi.org/10.1592/phco.22.9.789.34074) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12066972/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacotherapy&title=Levofloxacin-induced%20toxic%20epidermal%20necrolysis%20in%20an%20elderly%20patient&author=S.%20Digwood-Lettieri&author=K.J.%20Reilly&author=L.R.%20Haith&author=M.L.%20Patton&author=R.J.%20Guilday&volume=22&publication_year=2002&pages=789-793&pmid=12066972&doi=10.1592/phco.22.9.789.34074&)

3. Teo W.L., Pang S.M. Levofloxacin-induced toxic epidermal necrolysis treated with intravenous immunoglobulin. Ann. Acad. Med. Singap. 2010;39:660–661.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/20838711/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann.%20Acad.%20Med.%20Singap.&title=Levofloxacin-induced%20toxic%20epidermal%20necrolysis%20treated%20with%20intravenous%20immunoglobulin&author=W.L.%20Teo&author=S.M.%20Pang&volume=39&publication_year=2010&pages=660-661&pmid=20838711&)

4. Davila G., Ruiz-Hornillos J., Rojas P., De Castro F., Zubeldia J.M. Toxic epidermal necrolysis induced by levofloxacin. Ann. Allergy Asthma Immunol. 2009;102:441–442. doi: 10.1016/S1081-1206(10)60521-2.  [DOI](https://doi.org/10.1016/S1081-1206(10)60521-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19492670/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann.%20Allergy%20Asthma%20Immunol.&title=Toxic%20epidermal%20necrolysis%20induced%20by%20levofloxacin&author=G.%20Davila&author=J.%20Ruiz-Hornillos&author=P.%20Rojas&author=F.%20De%20Castro&author=J.M.%20Zubeldia&volume=102&publication_year=2009&pages=441-442&pmid=19492670&doi=10.1016/S1081-1206(10)60521-2&)

5. Christie M.J., Wong K., Ting R.H., Tam P.Y., Sikaneta T.G. Generalized seizure and toxic epidermal necrolysis following levofloxacin exposure. Ann. Pharmacother. 2005;39:953–955. doi: 10.1345/aph.1E587.  [DOI](https://doi.org/10.1345/aph.1E587) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15827068/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann.%20Pharmacother.&title=Generalized%20seizure%20and%20toxic%20epidermal%20necrolysis%20following%20levofloxacin%20exposure&author=M.J.%20Christie&author=K.%20Wong&author=R.H.%20Ting&author=P.Y.%20Tam&author=T.G.%20Sikaneta&volume=39&publication_year=2005&pages=953-955&pmid=15827068&doi=10.1345/aph.1E587&)

6. Islam A.F.M.S., Rahman M.S. Levofloxacin-induced fatal toxic epidermal necrolysis. Ann. Pharmacother. 2005;39:1136–1137. doi: 10.1345/aph.1E613.  [DOI](https://doi.org/10.1345/aph.1E613) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15886293/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann.%20Pharmacother.&title=Levofloxacin-induced%20fatal%20toxic%20epidermal%20necrolysis&author=A.F.M.S.%20Islam&author=M.S.%20Rahman&volume=39&publication_year=2005&pages=1136-1137&pmid=15886293&doi=10.1345/aph.1E613&)

7. Hoetzenecker W., Nägeli M., Mehra E.T., Jensen A.N., Saulite I., Schmid-Grendelmeier P., Guenova E., Cozzio A., French L.E. Adverse cutaneous drug eruptions: current understanding. Semin. Immunopathol. 2016;38:75–86. doi: 10.1007/s00281-015-0540-2.  [DOI](https://doi.org/10.1007/s00281-015-0540-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26553194/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Semin.%20Immunopathol.&title=Adverse%20cutaneous%20drug%20eruptions:%20current%20understanding&author=W.%20Hoetzenecker&author=M.%20N%C3%A4geli&author=E.T.%20Mehra&author=A.N.%20Jensen&author=I.%20Saulite&volume=38&publication_year=2016&pages=75-86&pmid=26553194&doi=10.1007/s00281-015-0540-2&)

8. Hung S.I., Chung W.H., Liou L.B., Chu C.C., Lin M., Huang H.P., Lin Y.L., Lan J.L., Yang L.C., Hong H.S., et al. HLA-B∗5801 allele as a genetic marker for severe cutaneous adverse reactions caused by allopurinol. Proc. Natl. Acad. Sci. USA. 2005;102:4134–4139. doi: 10.1073/pnas.0409500102.  [DOI](https://doi.org/10.1073/pnas.0409500102) | [PMC free article](/articles/PMC554812/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15743917/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&title=HLA-B%E2%88%975801%20allele%20as%20a%20genetic%20marker%20for%20severe%20cutaneous%20adverse%20reactions%20caused%20by%20allopurinol&author=S.I.%20Hung&author=W.H.%20Chung&author=L.B.%20Liou&author=C.C.%20Chu&author=M.%20Lin&volume=102&publication_year=2005&pages=4134-4139&pmid=15743917&doi=10.1073/pnas.0409500102&)

9. Gonçalo M., Coutinho I., Teixeira V., Gameiro A.R., Brites M.M., Nunes R., Martinho A. HLA-B∗58:01 is a risk factor for allopurinol-induced DRESS and Stevens-Johnson syndrome/toxic epidermal necrolysis in a Portuguese population. Br. J. Dermatol. 2013;169:660–665. doi: 10.1111/bjd.12389.  [DOI](https://doi.org/10.1111/bjd.12389) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23600531/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Dermatol.&title=HLA-B%E2%88%9758:01%20is%20a%20risk%20factor%20for%20allopurinol-induced%20DRESS%20and%20Stevens-Johnson%20syndrome/toxic%20epidermal%20necrolysis%20in%20a%20Portuguese%20population&author=M.%20Gon%C3%A7alo&author=I.%20Coutinho&author=V.%20Teixeira&author=A.R.%20Gameiro&author=M.M.%20Brites&volume=169&publication_year=2013&pages=660-665&pmid=23600531&doi=10.1111/bjd.12389&)

10. McCormack M., Alfirevic A., Bourgeois S., Farrell J.J., Kasperavičiūtė D., Carrington M., Sills G.J., Marson T., Jia X., de Bakker P.I.W., et al. HLA-A∗3101 and carbamazepine-induced hypersensitivity reactions in Europeans. N. Engl. J. Med. 2011;364:1134–1143. doi: 10.1056/NEJMoa1013297.  [DOI](https://doi.org/10.1056/NEJMoa1013297) | [PMC free article](/articles/PMC3113609/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21428769/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%C2%A0Engl.%20J.%20Med.&title=HLA-A%E2%88%973101%20and%20carbamazepine-induced%20hypersensitivity%20reactions%20in%20Europeans&author=M.%20McCormack&author=A.%20Alfirevic&author=S.%20Bourgeois&author=J.J.%20Farrell&author=D.%20Kasperavi%C4%8Di%C5%ABt%C4%97&volume=364&publication_year=2011&pages=1134-1143&pmid=21428769&doi=10.1056/NEJMoa1013297&)

11. Chen P., Lin J.J., Lu C.S., Ong C.T., Hsieh P.F., Yang C.C., Tai C.T., Wu S.L., Lu C.H., Hsu Y.C., et al. Carbamazepine-induced toxic effects and HLA-B∗1502 screening in Taiwan. N. Engl. J. Med. 2011;364:1126–1133. doi: 10.1056/NEJMoa1009717.  [DOI](https://doi.org/10.1056/NEJMoa1009717) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21428768/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%C2%A0Engl.%20J.%20Med.&title=Carbamazepine-induced%20toxic%20effects%20and%20HLA-B%E2%88%971502%20screening%20in%20Taiwan&author=P.%20Chen&author=J.J.%20Lin&author=C.S.%20Lu&author=C.T.%20Ong&author=P.F.%20Hsieh&volume=364&publication_year=2011&pages=1126-1133&pmid=21428768&doi=10.1056/NEJMoa1009717&)

12. Mallal S., Phillips E., Carosi G., Molina J.M., Workman C., Tomazic J., Jägel-Guedes E., Rugina S., Kozyrev O., Cid J.F., et al. HLA-B∗5701 screening for hypersensitivity to abacavir. N. Engl. J. Med. 2008;358:568–579. doi: 10.1056/NEJMoa0706135.  [DOI](https://doi.org/10.1056/NEJMoa0706135) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18256392/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%C2%A0Engl.%20J.%20Med.&title=HLA-B%E2%88%975701%20screening%20for%20hypersensitivity%20to%20abacavir&author=S.%20Mallal&author=E.%20Phillips&author=G.%20Carosi&author=J.M.%20Molina&author=C.%20Workman&volume=358&publication_year=2008&pages=568-579&pmid=18256392&doi=10.1056/NEJMoa0706135&)

13. Ko T.M., Tsai C.Y., Chen S.Y., Chen K.S., Yu K.H., Chu C.S., Huang C.M., Wang C.R., Weng C.T., Yu C.L., et al. Use of HLA-B∗58:01 genotyping to prevent allopurinol induced severe cutaneous adverse reactions in Taiwan: national prospective cohort study. BMJ. 2015;351:h4848. doi: 10.1136/bmj.h4848.  [DOI](https://doi.org/10.1136/bmj.h4848) | [PMC free article](/articles/PMC4579807/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26399967/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMJ&title=Use%20of%20HLA-B%E2%88%9758:01%20genotyping%20to%20prevent%20allopurinol%20induced%20severe%20cutaneous%20adverse%20reactions%20in%20Taiwan:%20national%20prospective%20cohort%20study&author=T.M.%20Ko&author=C.Y.%20Tsai&author=S.Y.%20Chen&author=K.S.%20Chen&author=K.H.%20Yu&volume=351&publication_year=2015&pages=h4848&pmid=26399967&doi=10.1136/bmj.h4848&)

14. Illing P.T., Vivian J.P., Dudek N.L., Kostenko L., Chen Z., Bharadwaj M., Miles J.J., Kjer-Nielsen L., Gras S., Williamson N.A., et al. Immune self-reactivity triggered by drug-modified HLA-peptide repertoire. Nature. 2012;486:554–558. doi: 10.1038/nature11147.  [DOI](https://doi.org/10.1038/nature11147) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22722860/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Immune%20self-reactivity%20triggered%20by%20drug-modified%20HLA-peptide%20repertoire&author=P.T.%20Illing&author=J.P.%20Vivian&author=N.L.%20Dudek&author=L.%20Kostenko&author=Z.%20Chen&volume=486&publication_year=2012&pages=554-558&pmid=22722860&doi=10.1038/nature11147&)

15. Karosiene E., Lundegaard C., Lund O., Nielsen M. NetMHCcons: a consensus method for the major histocompatibility complex class I predictions. Immunogenetics. 2012;64:177–186. doi: 10.1007/s00251-011-0579-8.  [DOI](https://doi.org/10.1007/s00251-011-0579-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22009319/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Immunogenetics&title=NetMHCcons:%20a%20consensus%20method%20for%20the%20major%20histocompatibility%20complex%20class%20I%20predictions&author=E.%20Karosiene&author=C.%20Lundegaard&author=O.%20Lund&author=M.%20Nielsen&volume=64&publication_year=2012&pages=177-186&pmid=22009319&doi=10.1007/s00251-011-0579-8&)

16. Kim S.H., Lee K.W., Song W.J., Kim S.H., Jee Y.K., Lee S.M., Kang H.R., Park H.W., Cho S.H., Park S.H., et al. Carbamazepine-induced severe cutaneous adverse reactions and HLA genotypes in Koreans. Epilepsy Res. 2011;97:190–197. doi: 10.1016/j.eplepsyres.2011.08.010.  [DOI](https://doi.org/10.1016/j.eplepsyres.2011.08.010) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21917426/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Epilepsy%20Res.&title=Carbamazepine-induced%20severe%20cutaneous%20adverse%20reactions%20and%20HLA%20genotypes%20in%20Koreans&author=S.H.%20Kim&author=K.W.%20Lee&author=W.J.%20Song&author=S.H.%20Kim&author=Y.K.%20Jee&volume=97&publication_year=2011&pages=190-197&pmid=21917426&doi=10.1016/j.eplepsyres.2011.08.010&)

17. Kaniwa N., Saito Y., Aihara M., Matsunaga K., Tohkin M., Kurose K., Furuya H., Takahashi Y., Muramatsu M., Kinoshita S., et al. HLA-B∗1511 is a risk factor for carbamazepine-induced Stevens-Johnson syndrome and toxic epidermal necrolysis in Japanese patients. Epilepsia. 2010;51:2461–2465. doi: 10.1111/j.1528-1167.2010.02766.x.  [DOI](https://doi.org/10.1111/j.1528-1167.2010.02766.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21204807/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Epilepsia&title=HLA-B%E2%88%971511%20is%20a%20risk%20factor%20for%20carbamazepine-induced%20Stevens-Johnson%20syndrome%20and%20toxic%20epidermal%20necrolysis%20in%20Japanese%20patients&author=N.%20Kaniwa&author=Y.%20Saito&author=M.%20Aihara&author=K.%20Matsunaga&author=M.%20Tohkin&volume=51&publication_year=2010&pages=2461-2465&pmid=21204807&doi=10.1111/j.1528-1167.2010.02766.x&)

18. Cao Z.H., Wei Z.Y., Zhu Q.Y., Zhang J.Y., Yang L., Qin S.Y., Shao L.Y., Zhang Y.T., Xuan J.K., Li Q.L., et al. HLA-B∗58:01 allele is associated with augmented risk for both mild and severe cutaneous adverse reactions induced by allopurinol in Han Chinese. Pharmacogenomics. 2012;13:1193–1201. doi: 10.2217/pgs.12.89.  [DOI](https://doi.org/10.2217/pgs.12.89) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22909208/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=HLA-B%E2%88%9758:01%20allele%20is%20associated%20with%20augmented%20risk%20for%20both%20mild%20and%20severe%20cutaneous%20adverse%20reactions%20induced%20by%20allopurinol%20in%20Han%20Chinese&author=Z.H.%20Cao&author=Z.Y.%20Wei&author=Q.Y.%20Zhu&author=J.Y.%20Zhang&author=L.%20Yang&volume=13&publication_year=2012&pages=1193-1201&pmid=22909208&doi=10.2217/pgs.12.89&)

19. Pavlos R., McKinnon E.J., Ostrov D.A., Peters B., Buus S., Koelle D., Chopra A., Schutte R., Rive C., Redwood A., et al. Shared peptide binding of HLA Class I and II alleles associate with cutaneous nevirapine hypersensitivity and identify novel risk alleles. Sci. Rep. 2017;7:8653. doi: 10.1038/s41598-017-08876-0.  [DOI](https://doi.org/10.1038/s41598-017-08876-0) | [PMC free article](/articles/PMC5561238/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28819312/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&title=Shared%20peptide%20binding%20of%20HLA%20Class%20I%20and%20II%20alleles%20associate%20with%20cutaneous%20nevirapine%20hypersensitivity%20and%20identify%20novel%20risk%20alleles&author=R.%20Pavlos&author=E.J.%20McKinnon&author=D.A.%20Ostrov&author=B.%20Peters&author=S.%20Buus&volume=7&publication_year=2017&pages=8653&pmid=28819312&doi=10.1038/s41598-017-08876-0&)

20. Holdsworth R., Hurley C.K., Marsh S.G.E., Lau M., Noreen H.J., Kempenich J.H., Setterholm M., Maiers M. The HLA dictionary 2008: a summary of HLA-A, -B, -C, -DRB1/3/4/5, and -DQB1 alleles and their association with serologically defined HLA-A, -B, -C, -DR, and -DQ antigens. Tissue Antigens. 2009;73:95–170. doi: 10.1111/j.1399-0039.2008.01183.x.  [DOI](https://doi.org/10.1111/j.1399-0039.2008.01183.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19140825/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Tissue%20Antigens&title=The%20HLA%20dictionary%202008:%20a%20summary%20of%20HLA-A,%20-B,%20-C,%20-DRB1/3/4/5,%20and%20-DQB1%20alleles%20and%20their%20association%20with%20serologically%20defined%20HLA-A,%20-B,%20-C,%20-DR,%20and%20-DQ%20antigens&author=R.%20Holdsworth&author=C.K.%20Hurley&author=S.G.E.%20Marsh&author=M.%20Lau&author=H.J.%20Noreen&volume=73&publication_year=2009&pages=95-170&pmid=19140825&doi=10.1111/j.1399-0039.2008.01183.x&)

21. Gonzalez-Galarza F.F., McCabe A., Santos E.J.M.D., Jones J., Takeshita L., Ortega-Rivera N.D., Cid-Pavon G.M.D., Ramsbottom K., Ghattaoraya G., Alfirevic A., et al. Allele frequency net database (AFND) 2020 update: gold-standard data classification, open access genotype data and new query tools. Nucleic Acids Res. 2020;48:D783–D788. doi: 10.1093/nar/gkz1029.  [DOI](https://doi.org/10.1093/nar/gkz1029) | [PMC free article](/articles/PMC7145554/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31722398/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&title=Allele%20frequency%20net%20database%20(AFND)%202020%20update:%20gold-standard%20data%20classification,%20open%20access%20genotype%20data%20and%20new%20query%20tools&author=F.F.%20Gonzalez-Galarza&author=A.%20McCabe&author=E.J.M.D.%20Santos&author=J.%20Jones&author=L.%20Takeshita&volume=48&publication_year=2020&pages=D783-D788&pmid=31722398&doi=10.1093/nar/gkz1029&)

22. Chung W.H., Wang C.W., Dao R.L. Severe cutaneous adverse drug reactions. J. Dermatol. 2016;43:758–766. doi: 10.1111/1346-8138.13430.  [DOI](https://doi.org/10.1111/1346-8138.13430) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27154258/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%C2%A0Dermatol.&title=Severe%20cutaneous%20adverse%20drug%20reactions&author=W.H.%20Chung&author=C.W.%20Wang&author=R.L.%20Dao&volume=43&publication_year=2016&pages=758-766&pmid=27154258&doi=10.1111/1346-8138.13430&)

23. Duong T.A., Valeyrie-Allanore L., Wolkenstein P., Chosidow O. Severe cutaneous adverse reactions to drugs. Lancet. 2017;390:1996–2011. doi: 10.1016/S0140-6736(16)30378-6.  [DOI](https://doi.org/10.1016/S0140-6736(16)30378-6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28476287/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Severe%20cutaneous%20adverse%20reactions%20to%20drugs&author=T.A.%20Duong&author=L.%20Valeyrie-Allanore&author=P.%20Wolkenstein&author=O.%20Chosidow&volume=390&publication_year=2017&pages=1996-2011&pmid=28476287&doi=10.1016/S0140-6736(16)30378-6&)

24. Zhang F.R., Liu H., Irwanto A., Fu X.A., Li Y., Yu G.Q., Yu Y.X., Chen M.F., Low H.Q., Li J.H., et al. HLA-B∗13:01 and the dapsone hypersensitivity syndrome. N. Engl. J. Med. 2013;369:1620–1628. doi: 10.1056/NEJMoa1213096.  [DOI](https://doi.org/10.1056/NEJMoa1213096) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24152261/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%C2%A0Engl.%20J.%20Med.&title=HLA-B%E2%88%9713:01%20and%20the%20dapsone%20hypersensitivity%20syndrome&author=F.R.%20Zhang&author=H.%20Liu&author=A.%20Irwanto&author=X.A.%20Fu&author=Y.%20Li&volume=369&publication_year=2013&pages=1620-1628&pmid=24152261&doi=10.1056/NEJMoa1213096&)

25. Yang F., Gu B., Zhang L., Xuan J., Luo H., Zhou P., Zhu Q., Yan S., Chen S.A., Cao Z., et al. HLA-B∗13:01 is associated with salazosulfapyridine-induced drug rash with eosinophilia and systemic symptoms in Chinese Han population. Pharmacogenomics. 2014;15:1461–1469. doi: 10.2217/pgs.14.69.  [DOI](https://doi.org/10.2217/pgs.14.69) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25303297/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=HLA-B%E2%88%9713:01%20is%20associated%20with%20salazosulfapyridine-induced%20drug%20rash%20with%20eosinophilia%20and%20systemic%20symptoms%20in%20Chinese%20Han%20population&author=F.%20Yang&author=B.%20Gu&author=L.%20Zhang&author=J.%20Xuan&author=H.%20Luo&volume=15&publication_year=2014&pages=1461-1469&pmid=25303297&doi=10.2217/pgs.14.69&)

26. Jiang H., Wang C.W., Wang Z., Dai Y., Zhu Y., Lee Y.S., Cao Y., Chung W.H., Ouyang S., Wang H. Functional and structural characteristics of HLA-B∗13:01-mediated specific T cells reaction in dapsone-induced drug hypersensitivity. J. Biomed. Sci. 2022;29:58. doi: 10.1186/s12929-022-00845-8.  [DOI](https://doi.org/10.1186/s12929-022-00845-8) | [PMC free article](/articles/PMC9375929/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35964029/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%C2%A0Biomed.%20Sci.&title=Functional%20and%20structural%20characteristics%20of%20HLA-B%E2%88%9713:01-mediated%20specific%20T%C2%A0cells%20reaction%20in%20dapsone-induced%20drug%20hypersensitivity&author=H.%20Jiang&author=C.W.%20Wang&author=Z.%20Wang&author=Y.%20Dai&author=Y.%20Zhu&volume=29&publication_year=2022&pages=58&pmid=35964029&doi=10.1186/s12929-022-00845-8&)

27. Satapornpong P., Pratoomwun J., Rerknimitr P., Klaewsongkram J., Nakkam N., Rungrotmongkol T., Konyoung P., Saksit N., Mahakkanukrauh A., Amornpinyo W., et al. HLA-B∗13 :01 is a predictive marker of dapsone-induced severe cutaneous adverse reactions in Thai patients. Front. Immunol. 2021;12:661135. doi: 10.3389/fimmu.2021.661135.  [DOI](https://doi.org/10.3389/fimmu.2021.661135) | [PMC free article](/articles/PMC8130671/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34017337/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front.%20Immunol.&title=HLA-B%E2%88%9713%20:01%20is%20a%20predictive%20marker%20of%20dapsone-induced%20severe%20cutaneous%20adverse%20reactions%20in%20Thai%20patients&author=P.%20Satapornpong&author=J.%20Pratoomwun&author=P.%20Rerknimitr&author=J.%20Klaewsongkram&author=N.%20Nakkam&volume=12&publication_year=2021&pages=661135&pmid=34017337&doi=10.3389/fimmu.2021.661135&)

28. Watanabe H., Watanabe Y., Tashiro Y., Mushiroda T., Ozeki T., Hashizume H., Sueki H., Yamamoto T., Utsunomiya-Tate N., Gouda H., et al. A docking model of dapsone bound to HLA-B∗13:01 explains the risk of dapsone hypersensitivity syndrome. J. Dermatol. Sci. 2017;88:320–329. doi: 10.1016/j.jdermsci.2017.08.007.  [DOI](https://doi.org/10.1016/j.jdermsci.2017.08.007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28870516/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%C2%A0Dermatol.%20Sci.&title=A%20docking%20model%20of%20dapsone%20bound%20to%20HLA-B%E2%88%9713:01%20explains%20the%20risk%20of%20dapsone%20hypersensitivity%20syndrome&author=H.%20Watanabe&author=Y.%20Watanabe&author=Y.%20Tashiro&author=T.%20Mushiroda&author=T.%20Ozeki&volume=88&publication_year=2017&pages=320-329&pmid=28870516&doi=10.1016/j.jdermsci.2017.08.007&)

29. Ostrov D.A., Grant B.J., Pompeu Y.A., Sidney J., Harndahl M., Southwood S., Oseroff C., Lu S., Jakoncic J., de Oliveira C.A.F., et al. Drug hypersensitivity caused by alteration of the MHC-presented self-peptide repertoire. Proc. Natl. Acad. Sci. USA. 2012;109:9959–9964. doi: 10.1073/pnas.1207934109.  [DOI](https://doi.org/10.1073/pnas.1207934109) | [PMC free article](/articles/PMC3382472/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22645359/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&title=Drug%20hypersensitivity%20caused%20by%20alteration%20of%20the%20MHC-presented%20self-peptide%20repertoire&author=D.A.%20Ostrov&author=B.J.%20Grant&author=Y.A.%20Pompeu&author=J.%20Sidney&author=M.%20Harndahl&volume=109&publication_year=2012&pages=9959-9964&pmid=22645359&doi=10.1073/pnas.1207934109&)

30. Pavlos R., Mallal S., Ostrov D., Buus S., Metushi I., Peters B., Phillips E. T cell-mediated hypersensitivity reactions to drugs. Annu. Rev. Med. 2015;66:439–454. doi: 10.1146/annurev-med-050913-022745.  [DOI](https://doi.org/10.1146/annurev-med-050913-022745) | [PMC free article](/articles/PMC4295772/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25386935/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annu.%20Rev.%20Med.&title=T%20cell-mediated%20hypersensitivity%20reactions%20to%20drugs&author=R.%20Pavlos&author=S.%20Mallal&author=D.%20Ostrov&author=S.%20Buus&author=I.%20Metushi&volume=66&publication_year=2015&pages=439-454&pmid=25386935&doi=10.1146/annurev-med-050913-022745&)

31. Pan R.Y., Chu M.T., Wang C.W., Lee Y.S., Lemonnier F., Michels A.W., Schutte R., Ostrov D.A., Chen C.B., Phillips E.J., et al. Identification of drug-specific public TCR driving severe cutaneous adverse reactions. Nat. Commun. 2019;10:3569. doi: 10.1038/s41467-019-11396-2.  [DOI](https://doi.org/10.1038/s41467-019-11396-2) | [PMC free article](/articles/PMC6687717/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31395875/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&title=Identification%20of%20drug-specific%20public%20TCR%20driving%20severe%20cutaneous%20adverse%20reactions&author=R.Y.%20Pan&author=M.T.%20Chu&author=C.W.%20Wang&author=Y.S.%20Lee&author=F.%20Lemonnier&volume=10&publication_year=2019&pages=3569&pmid=31395875&doi=10.1038/s41467-019-11396-2&)

32. Chung W.H., Pan R.Y., Chu M.T., Chin S.W., Huang Y.L., Wang W.C., Chang J.Y., Hung S.I. Oxypurinol-specific T cells possess preferential TCR clonotypes and express granulysin in allopurinol-induced severe cutaneous adverse reactions. J. Invest. Dermatol. 2015;135:2237–2248. doi: 10.1038/jid.2015.165.  [DOI](https://doi.org/10.1038/jid.2015.165) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25946710/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%C2%A0Invest.%20Dermatol.&title=Oxypurinol-specific%20T%C2%A0cells%20possess%20preferential%20TCR%20clonotypes%20and%20express%20granulysin%20in%20allopurinol-induced%20severe%20cutaneous%20adverse%20reactions&author=W.H.%20Chung&author=R.Y.%20Pan&author=M.T.%20Chu&author=S.W.%20Chin&author=Y.L.%20Huang&volume=135&publication_year=2015&pages=2237-2248&pmid=25946710&doi=10.1038/jid.2015.165&)

33. Pham T.D.M., Ziora Z.M., Blaskovich M.A.T. Quinolone antibiotics. Medchemcomm. 2019;10:1719–1739. doi: 10.1039/c9md00120d.  [DOI](https://doi.org/10.1039/c9md00120d) | [PMC free article](/articles/PMC6836748/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31803393/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Medchemcomm&title=Quinolone%20antibiotics&author=T.D.M.%20Pham&author=Z.M.%20Ziora&author=M.A.T.%20Blaskovich&volume=10&publication_year=2019&pages=1719-1739&pmid=31803393&doi=10.1039/c9md00120d&)

34. Smith D.A., Schmid E.F. Drug withdrawals and the lessons within. Curr. Opin. Drug Discov. Devel. 2006;9:38–46.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/16445116/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr.%20Opin.%20Drug%20Discov.%20Devel.&title=Drug%20withdrawals%20and%20the%20lessons%20within&author=D.A.%20Smith&author=E.F.%20Schmid&volume=9&publication_year=2006&pages=38-46&pmid=16445116&)

35. Thomson P.J., Illing P.T., Farrell J., Alhaidari M., Bell C.C., Berry N., O'Neill P.M., Purcell A.W., Park K.B., Naisbitt D.J. Modification of the cyclopropyl moiety of abacavir provides insight into the structure activity relationship between HLA-B∗57:01 binding and T-cell activation. Allergy. 2020;75:636–647. doi: 10.1111/all.14057.  [DOI](https://doi.org/10.1111/all.14057) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31549414/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Allergy&title=Modification%20of%20the%20cyclopropyl%20moiety%20of%20abacavir%20provides%20insight%20into%20the%20structure%20activity%20relationship%20between%20HLA-B%E2%88%9757:01%20binding%20and%20T-cell%20activation&author=P.J.%20Thomson&author=P.T.%20Illing&author=J.%20Farrell&author=M.%20Alhaidari&author=C.C.%20Bell&volume=75&publication_year=2020&pages=636-647&pmid=31549414&doi=10.1111/all.14057&)

36. Nelson M.R., Bacanu S.A., Mosteller M., Li L., Bowman C.E., Roses A.D., Lai E.H., Ehm M.G. Genome-wide approaches to identify pharmacogenetic contributions to adverse drug reactions. Pharmacogenomics J. 2009;9:23–33. doi: 10.1038/tpj.2008.4.  [DOI](https://doi.org/10.1038/tpj.2008.4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18301416/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J.&title=Genome-wide%20approaches%20to%20identify%20pharmacogenetic%20contributions%20to%20adverse%20drug%20reactions&author=M.R.%20Nelson&author=S.A.%20Bacanu&author=M.%20Mosteller&author=L.%20Li&author=C.E.%20Bowman&volume=9&publication_year=2009&pages=23-33&pmid=18301416&doi=10.1038/tpj.2008.4&)

37. Kawaguchi S., Higasa K., Shimizu M., Yamada R., Matsuda F. HLA-HD: An accurate HLA typing algorithm for next-generation sequencing data. Hum. Mutat. 2017;38:788–797. doi: 10.1002/humu.23230.  [DOI](https://doi.org/10.1002/humu.23230) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28419628/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum.%20Mutat.&title=HLA-HD:%20An%20accurate%20HLA%20typing%20algorithm%20for%20next-generation%20sequencing%20data&author=S.%20Kawaguchi&author=K.%20Higasa&author=M.%20Shimizu&author=R.%20Yamada&author=F.%20Matsuda&volume=38&publication_year=2017&pages=788-797&pmid=28419628&doi=10.1002/humu.23230&)

38. Roujeau J.C., Stern R.S. Severe adverse cutaneous reactions to drugs. N. Engl. J. Med. 1994;331:1272–1285. doi: 10.1056/NEJM199411103311906.  [DOI](https://doi.org/10.1056/NEJM199411103311906) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7794310/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%C2%A0Engl.%20J.%20Med.&title=Severe%20adverse%20cutaneous%20reactions%20to%20drugs&author=J.C.%20Roujeau&author=R.S.%20Stern&volume=331&publication_year=1994&pages=1272-1285&pmid=7794310&doi=10.1056/NEJM199411103311906&)

39. Kardaun S.H., Sekula P., Valeyrie-Allanore L., Liss Y., Chu C.Y., Creamer D., Sidoroff A., Naldi L., Mockenhaupt M., Roujeau J.C., et al. Drug reaction with eosinophilia and systemic symptoms (DRESS): an original multisystem adverse drug reaction. Results from the prospective RegiSCAR study. Br. J. Dermatol. 2013;169:1071–1080. doi: 10.1111/bjd.12501.  [DOI](https://doi.org/10.1111/bjd.12501) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23855313/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Dermatol.&title=Drug%20reaction%20with%20eosinophilia%20and%20systemic%20symptoms%20(DRESS):%20an%20original%20multisystem%20adverse%20drug%20reaction.%20Results%20from%20the%20prospective%20RegiSCAR%20study&author=S.H.%20Kardaun&author=P.%20Sekula&author=L.%20Valeyrie-Allanore&author=Y.%20Liss&author=C.Y.%20Chu&volume=169&publication_year=2013&pages=1071-1080&pmid=23855313&doi=10.1111/bjd.12501&)

40. Sidoroff A., Halevy S., Bavinck J.N., Vaillant L., Roujeau J.C. Acute generalized exanthematous pustulosis (AGEP)--a clinical reaction pattern. J. Cutan. Pathol. 2001;28:113–119. doi: 10.1034/j.1600-0560.2001.028003113.x.  [DOI](https://doi.org/10.1034/j.1600-0560.2001.028003113.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11168761/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%C2%A0Cutan.%20Pathol.&title=Acute%20generalized%20exanthematous%20pustulosis%20(AGEP)--a%20clinical%20reaction%20pattern&author=A.%20Sidoroff&author=S.%20Halevy&author=J.N.%20Bavinck&author=L.%20Vaillant&author=J.C.%20Roujeau&volume=28&publication_year=2001&pages=113-119&pmid=11168761&doi=10.1034/j.1600-0560.2001.028003113.x&)

41. Sassolas B., Haddad C., Mockenhaupt M., Dunant A., Liss Y., Bork K., Haustein U.F., Vieluf D., Roujeau J.C., Le Louet H. ALDEN, an algorithm for assessment of drug causality in Stevens-Johnson Syndrome and toxic epidermal necrolysis: comparison with case-control analysis. Clin. Pharmacol. Ther. 2010;88:60–68. doi: 10.1038/clpt.2009.252.  [DOI](https://doi.org/10.1038/clpt.2009.252) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20375998/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=ALDEN,%20an%20algorithm%20for%20assessment%20of%20drug%20causality%20in%20Stevens-Johnson%20Syndrome%20and%20toxic%20epidermal%20necrolysis:%20comparison%20with%20case-control%20analysis&author=B.%20Sassolas&author=C.%20Haddad&author=M.%20Mockenhaupt&author=A.%20Dunant&author=Y.%20Liss&volume=88&publication_year=2010&pages=60-68&pmid=20375998&doi=10.1038/clpt.2009.252&)

42. Naranjo C.A., Busto U., Sellers E.M., Sandor P., Ruiz I., Roberts E.A., Janecek E., Domecq C., Greenblatt D.J. A method for estimating the probability of adverse drug reactions. Clin. Pharmacol. Ther. 1981;30:239–245. doi: 10.1038/clpt.1981.154.  [DOI](https://doi.org/10.1038/clpt.1981.154) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7249508/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=A%20method%20for%20estimating%20the%20probability%20of%20adverse%20drug%20reactions&author=C.A.%20Naranjo&author=U.%20Busto&author=E.M.%20Sellers&author=P.%20Sandor&author=I.%20Ruiz&volume=30&publication_year=1981&pages=239-245&pmid=7249508&doi=10.1038/clpt.1981.154&)

43. Di Marco M., Schuster H., Backert L., Ghosh M., Rammensee H.G., Stevanović S. Unveiling the peptide motifs of HLA-C and HLA-G from naturally presented peptides and generation of binding prediction matrices. J. Immunol. 2017;199:2639–2651. doi: 10.4049/jimmunol.1700938.  [DOI](https://doi.org/10.4049/jimmunol.1700938) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28904123/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%C2%A0Immunol.&title=Unveiling%20the%20peptide%20motifs%20of%20HLA-C%20and%20HLA-G%20from%20naturally%20presented%20peptides%20and%20generation%20of%20binding%20prediction%20matrices&author=M.%20Di%20Marco&author=H.%20Schuster&author=L.%20Backert&author=M.%20Ghosh&author=H.G.%20Rammensee&volume=199&publication_year=2017&pages=2639-2651&pmid=28904123&doi=10.4049/jimmunol.1700938&)

44. Yang C.W.O., Hung S.I., Juo C.G., Lin Y.P., Fang W.H., Lu I.H., Chen S.T., Chen Y.T. HLA-B∗1502-bound peptides: implications for the pathogenesis of carbamazepine-induced Stevens-Johnson syndrome. J. Allergy Clin. Immunol. 2007;120:870–877. doi: 10.1016/j.jaci.2007.06.017.  [DOI](https://doi.org/10.1016/j.jaci.2007.06.017) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17697703/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%C2%A0Allergy%20Clin.%20Immunol.&title=HLA-B%E2%88%971502-bound%20peptides:%20implications%20for%20the%20pathogenesis%20of%20carbamazepine-induced%20Stevens-Johnson%20syndrome&author=C.W.O.%20Yang&author=S.I.%20Hung&author=C.G.%20Juo&author=Y.P.%20Lin&author=W.H.%20Fang&volume=120&publication_year=2007&pages=870-877&pmid=17697703&doi=10.1016/j.jaci.2007.06.017&)

45. Zhou P., Jin B., Li H., Huang S.Y. HPEPDOCK: a web server for blind peptide-protein docking based on a hierarchical algorithm. Nucleic Acids Res. 2018;46:W443–W450. doi: 10.1093/nar/gky357.  [DOI](https://doi.org/10.1093/nar/gky357) | [PMC free article](/articles/PMC6030929/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29746661/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&title=HPEPDOCK:%20a%20web%20server%20for%20blind%20peptide-protein%20docking%20based%20on%20a%20hierarchical%20algorithm&author=P.%20Zhou&author=B.%20Jin&author=H.%20Li&author=S.Y.%20Huang&volume=46&publication_year=2018&pages=W443-W450&pmid=29746661&doi=10.1093/nar/gky357&)
