# Clinical implications of a gain-of-function genetic polymorphism in DPYD (rs4294451) in colorectal cancer patients treated with fluoropyrimidines

## Metadata
**Authors:** Elena De Mattia, Jerry Polesel, Lucia Scarabel, Erika Cecchin
**Journal:** Frontiers in Pharmacology
**Date:** 2024 Dec 5
**DOI:** [10.3389/fphar.2024.1516375](https://doi.org/10.3389/fphar.2024.1516375)
**PMID:** 39703399
**PMCID:** PMC11655210
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11655210/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC11655210/pdf/fphar-15-1516375.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC11655210/pdf/fphar-15-1516375.pdf)

## Abstract

Dihydropyrimidine dehydrogenase (DPD, encoded by the DPYD gene) is the rate-limiting enzyme for the detoxification of fluoropyrimidines (FLs). Rs4294451 is a regulatory DPYD polymorphism that has recently been functionally characterized and associated with increased DPD expression in the liver. The aim of the present study was to test the clinical implications of being a carrier of rs4294451 in a cohort of 645 FL-treated colorectal cancer patients. Carriers of at least one DPYD rs4294451-T variant allele had a lower risk of developing NCI-CTC grade 4–5 hematological [odds ratio (OR) = 0.39; 95% confidence interval (CI): 0.15–0.98; additive model] and hematological/non-hematological (OR = 0.44; 95% CI: 0.22–0.88; dominant model) FL-related toxicity. Patients with the DPYD rs4294451-T allele also had a longer time to severe toxicity development after starting FL treatment [hematological, Hazard ratio (HR) = 0.27; 95% CI: 0.09–0.79; Fine–Gray test = 0.1569; hematological/non-hematological: HR = 0.38, 95% CI: 0.17–0.85; Fine–Gray test = 0.0444]. It is worth noting that while being at lower risk of toxicity, DPYD rs4294451-T allele carriers also tend to present a shorter overall survival (HR = 1.41; 95% CI: 1.05–1.90; log-rank p = 0.0406). These findings demonstrate a clinical effect of DPYD-rs4294451 polymorphism coherent with the recently described functional effect. Further investigation is warranted to elucidate the potential clinical value to the rs4294451 polymorphism as toxicity and especially as an efficacy marker in colorectal cancer.

Keywords: DPYD, rs4294451, fluoropyrimidine, toxicity, colorectal cancer

## Introduction

Fluoropyrimidines (FLs), mainly 5-fluorouracil (5-FU) and its oral prodrug capecitabine, represent the cornerstone of several antineoplastic regimens currently used to treat a broad spectrum of solid tumors, including gastrointestinal tract, breast, and head and neck cancers ([Knikman et al., 2021](#B11); [Lunenburg et al., 2016](#B13)). Severe toxicity is observed in up to 30% of patients ([Knikman et al., 2021](#B11); [Lunenburg et al., 2016](#B13)), with a negative impact on the effectiveness of cancer treatment (i.e., an increased risk of chemotherapy interruption and, consequently, disease progression), patients’ quality of life, and medical costs. For approximately 1% of patients, this toxicity could be fatal ([Knikman et al., 2021](#B11); [Lunenburg et al., 2016](#B13)). Dihydropyrimidine dehydrogenase (DPD, encoded by the *DPYD* gene) is the first and rate-limiting enzyme in the detoxification pathway of FL, which is constitutively deficient in approximately 3%–5% of Caucasians ([Boisdron-Celle et al., 2017](#B2)). At present, four variants in the *DPYD* gene (*DPYD*2A,* rs3918290*; DPYD*13,* rs55886062; c.2846A>T, rs67376798; and c.1236G>A-HapB3, rs56038477) are validated for their clinical impact on FL-related toxicity and recommended for pre-treatment testing by European regulatory agencies according to specific clinical guidelines for drug adjustments ([Amstutz et al., 2018](#B1); [EMA, 2020](#B6); [Lunenburg et al., 2020](#B14)). However, the routinely tested four *DPYD* variants identify only a minimal percentage (approximately 17%) of patients experiencing severe FL-related toxicity ([De Mattia et al., 2023](#B4); [De Mattia et al., 2022](#B5)), and additional validated predictive genetic markers are therefore required. The Association for Molecular Pathology (AMP) PGx Working Group guidelines for *DPYD* testing in clinical practice have recently recommended an extended panel that is more representative of genetic diversity across populations ([Pratt et al., 2024](#B16)).

A recent study ([Zhang et al., 2024](#B19)), using human liver tissues and cellular models to characterize a novel cis-enhancer element capable of modulating DPD expression, has shown that the allelic status of the common germline variant *DPYD* rs4294451A>T could affect CEBPB-driven DPD expression and sensitivity/resistance to 5-FU. Based on the data, the author hypothesized that the higher systemic detoxification of 5-FU due to increased liver DPD expression in carriers of the *DPYD* rs4294451-T allele results in lower exposure to active anti-tumor metabolites of 5-FU. These findings make the *DPYD* rs4294451 polymorphism a strong candidate for the prediction of 5-FU toxicity risk and, potentially, tumor resistance to 5-FU-based therapy. However, the clinical impact of this common genetic variant (22.7% of the minor allele frequency in Non-Finnish Europeans according to gnomAD v3.1.2, [https://www.ensembl.org/](https://www.ensembl.org/)) has never been tested.

The aim of the present study is to elucidate the clinical role of the *DPYD* rs4294451 variant as predictive pharmacogenetic markers of the clinical outcome (severe toxicity and prognosis) related to an FL-based treatment in a cohort of 645 patients with colorectal cancer (CRC).

## Patients and methods

### Patient cohorts and clinical data collection

Between 1999 and 2019, clinical data and biological samples (blood) from patients receiving FL-based chemotherapy were collected at the Clinical and Experimental Pharmacology Unit of the Centro di Riferimento Oncologico (CRO) in Aviano (PN). From a database of 1,122 clinical cases, the study population was selected based on the following inclusion criteria: (1) diagnosis of colorectal carcinoma; (2) available peripheral biological blood sample; (3) age ≥18 years; (4) assumption of treatment containing FLs (5-FU or capecitabine); (5) available detailed clinical data; and 6) signed written informed. Patients were previously genotyped for *DPYD**2A, *13, c2846A>T, and c.1236C>T ([Dalle Fratte et al., 2018](#B3)), and carriers of at least one of the DPYD variants were excluded from the study.

Patients’ medical records were reviewed to collect the following clinical information: (1) baseline clinical–demographic data (e.g., gender, age, and tumor location); (2) chemotherapy information (e.g., FL type, FL starting dose, concurrent administration of chemotherapeutic agents or radiotherapy, start and end dates of therapy, and discontinuation of therapy); (3) toxicity data recorded at each chemotherapy cycle, including severity assessment and start date; (4) patient follow-up data regarding patients’ death.

Adverse events were recorded throughout the entire chemotherapy period and graded according to the National Cancer Institute’s Common Terminology Criteria for Adverse Events v5.0 (NCI-CTCAE). Only toxicities that were more likely to be related to the FL–DPYD interaction were considered, and these were categorized as hematological (i.e., neutropenia) or non-hematological (i.e., nausea, vomiting, diarrhea, hand-foot syndrome, and mucositis). “Any type” toxicity defines both hematological and non-hematological toxicity.

All patients in the study were self-reported Caucasians. The study protocol complied with the ethical guidelines of the 1975 Declaration of Helsinki and was approved by the local ethical committee. All experiments were carried out in accordance with the relevant guidelines and regulations of Centro di Riferimento Oncologico di Aviano (PN).

### Genotyping methods

Patients’ peripheral blood was collected in EDTA-containing tubes. Genomic DNA was extracted from blood samples using the EZ1 DNA Blood 200-mL Kit (QIAGEN) and the BioRobot EZ1 (QIAGEN). Alternatively, genomic DNA was extracted manually using the High Pure Template Preparation Kit. Patients were previously genotyped for *DPYD*2A, DPYD*13, DPYD* c.2846A>T, and c.1236G>A-*HapB3* ([Dalle Fratte et al., 2018](#B3)).

The analysis of the *DPYD* rs4294451 polymorphism was performed using pre-designed TaqMan SNP Genotyping assays (assay ID: C_32478960_10; functionally tested) according to the manufacturer’s instructions with the TaqMan™ Universal PCR Master Mix on the ABI7500 Real-Time PCR System instrument (Applied Biosystems). Negative and positive controls from previous genotyping were used in each analysis.

### Statistical analysis

#### Socio-demographic and clinical characteristics were reported as absolute frequency and percentage.

To estimate the risk of developing a G4 toxicity associated to the *DPYD* rs4294451 A>T genotype, patients were first classified as having experienced at least an NCI-CTC grade (G) 4 or 5 event (cases) or not (controls). The odds ratio (OR) of developing a G4 toxicity, and the corresponding 95% confidence interval (CI), was estimated through the unconditional logistic regression model, including terms for potential confounders (i.e., gender, age, and cotreatments). Dominant, recessive, and additive genetic models were considered by combining heterozygous with homozygous genotypes; the best-fitting genetic model was selected according to the Wald chi-square test.

Furthermore, to account for the time to G4 toxicity development, a survival analysis was performed. The time at risk of G4 toxicity was calculated from the date of the therapy start to the date of G4 event, death, or last follow-up, whichever occurred first. To account for competing risks, the cumulative incidence of G4 toxicity was calculated ([Fine and Gray, 1999](#B7)), and differences according to the *DPYD*-rs4294451 A>T genotype were tested through Gray’s test ([Gray, 1998](#B8)). Finally, the overall survival was estimated according to the Kaplan–Meier method, and the difference by the rs4294451 A>T genotype was tested through the log-rank test ([Kalbfleisch and Prentice, 2002](#B10)).

## Results

### Patients and genotyping

In total, 689 CRC patients receiving FL-based treatment were selected from our biobank according to the eligibility criteria specified above. Forty-four patients, positive for one of the four previously tested *DPYD* variants, were excluded from the study: *DPYD**2A (n = 9), *DPYD**13 (n = 0), c2846A>T (n = 8), and c.1236C>T (n = 27). The final study population included 645 cases and the main clinical–demographic characteristics are summarized in [Table 1](#T1).

### TABLE 1.

| Characteristic | n | (%) |
| --- | --- | --- |
| Gender |  |  |
| Female | 250 | (38.8) |
| Male | 395 | (61.2) |
| Age (years) |  |  |
| <55 | 145 | (22.6) |
| 55–64 | 210 | (32.7) |
| 65–69 | 131 | (20.4) |
| ≥70 | 157 | (24.4) |
| Tumor localization |  |  |
| Left colon | 173 | (26.8) |
| Right colon | 139 | (21.6) |
| Transversal colon | 77 | (11.9) |
| Rectum | 256 | (39.7) |
| Chemotherapy |  |  |
| Fluoropyrimidines |  |  |
| 5-FU | 461 | (71.5) |
| Capecitabine | 184 | (28.5) |
| Monotherapy | 73 | (11.3) |
| Association with radiotherapy | 119 | (18.5) |
| Association with irinotecan | 239 | (37.1) |
| Association with oxaliplatin | 202 | (31.3) |
| Other | 12 | (1.9) |
| Genotype for DPYD rs4294451 a |  |  |
| AA | 438 | (67.9) |
| AT | 181 | (28.1) |
| TT | 26 | (4.0) |

Table 1 Caption: Socio-demographic and clinical characteristics of the study population (n = 645).

*DPYD* rs4294451 ([NC_000001.11](https://www.ncbi.nlm.nih.gov/nuccore/NC_000001.11):g.97930158T>A, GRCh38.p14 chr 1) genotyping was successfully performed for all the 645 patients in the study population. Genotype frequency is given in [Table 1](#T1).

### DPYD rs4294451 variant and toxicity risk

Overall, 60 patients (9.3%) developed at least one G4 toxicity, while no patient experienced G5 toxicity. The distribution of toxicity according to the rs4294451 genotype is given in [Table 2](#T2). The *DPYD* rs4294451-T allele was significantly associated with reduced risk of developing a G4 hematological (OR = 0.38; 95% CI: 0.14–0.99; additive model) or G4 “any-type” (OR = 0.45; 95% CI: 0.23–0.88; dominant model) toxicity. These associations were confirmed by adjusted logistic regression analysis, both for hematological (OR = 0.39; 95% CI: 0.15–0.98) and “any-type” (OR = 0.44; 95% CI: 0.22–0.88) toxicity. Although not statistically significant, the same trend was observed for the non-hematological toxicity.

### TABLE 2.

|   | rs4294451 A>T |  |  | Model | OR (95% CI) a | OR (95% CI) b |
| --- | --- | --- | --- | --- | --- | --- |
| AA (n = 438) | At (n = 181) | TT (n = 26) |  |  |  |  |
| Toxicity type | G4 n (%) | G4 n (%) | G4 n (%) |  |  |  |
| Hematological | 27 (6.16) | 5 (2.76) | 0 (0) | Additive | 0.38 (0.14–0.99) | 0.39 (0.15–0.98) |
| Non-hematological | 23 (5.25) | 5 (2.76) | 1 (3.85) | Dominant | 0.54 (0.22–1.34) | 0.54 (0.21–1.36) |
| Any type | 49 (11.19) | 10 (5.52) | 1 (3.85) | Dominant | 0.45 (0.23–0.88) | 0.44 (0.22–0.88) |

Table 2 Caption: Odds ratio (OR) and corresponding confidence intervals (CIs) for G4 toxicity according to the DPYD rs4294451 variant. Associations with p-value < 0.05 are in bold.

To account for potential competing risk of death, the cumulative incidence of G4 toxicity after treatment initiation was also estimated according to the *DPYD* rs4294451 variant ([Figure 1A–C](#F1)). Within 6 months of treatment initiation, the cumulative incidence of any type of G4 toxicity was 10.3% for the *DPYD* rs4294451-AA genotype and 5.3% for the *DPYD* rs4294451-AT/TT genotypes ([Figure 1C](#F1); *p* = 0.0444). No significant difference emerged for hematological and non-hematological-specific toxicities ([Figure 1A, B](#F1)), even though the cumulative incidence was lower for *DPYD* rs4294451-AT/TT than for the AA genotype. After adjusting for sex, age, and cotreatments, the rs4294451-T allele was associated with a lower probability of early toxicity development, and this association became statistically significant for the hematological (HR = 0.27; 95% CI: 0.09–0.79; additive model) and “any-type” (HR = 0.38; 95% CI: 0.17–0.85; dominant model) G4 toxicities ([Figure 1D](#F1)).

### FIGURE 1.

![FIGURE 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1592/11655210/7e4e695f941d/fphar-15-1516375-g001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=11655210_fphar-15-1516375-g001.jpg)

Cumulative incidence of G4 toxicity according to the DPYD rs4294451 variant. (A) Hematological toxicity. (B) Non-hematological toxicity. (C) Any-type toxicity. (D) Hazard ratio (HR) and corresponding confidence intervals (CIs) for G4 toxicity according to DPYD polymorphisms.

### DPYD rs4294451 variant and overall survival

The impact of the *DPYD* rs4294451 genotype on patients’ overall survival was also estimated. Patients carrying the *DPYD* rs4294451 AT/TT genotype had a shorter survival than those with rs4294451-AA, with a 10-year overall survival of 26.2% and 60.3%, respectively (*p* = 0.0406; [Figure 2](#F2).) After adjusting for sex, age, and cotreatments, a significant excess of risk emerged for the DPYD rs4294451 AT/TT genotype compared to the AA genotype (HR = 1.41; 95% CI: 1.05–1.90).

### FIGURE 2.

![FIGURE 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1592/11655210/b2fb3a1b4a4c/fphar-15-1516375-g002.jpg)

Overall survival according to the rs4294451 variant.

## Discussion

Despite many years of clinical use and optimization of administration methods, treatment with FL still causes severe toxicity in up to 30% of patients, leading not only to treatment interruption but also patient death in approximately 1% of cases. Pre-treatment *DPYD* genotyping for the four validated markers (*DPYD**2A, *DPYD**13, c.2846A>T, and c.1236G>A-HapB3) is currently entered into clinical practice and permits the identification of patients at increased risk of developing severe FL-related toxicity before starting therapy ([Amstutz et al., 2018](#B1); [Knikman et al., 2021](#B11); [Lunenburg et al., 2020](#B14)). However, the *DPYD* panel of four variants has high specificity (between 99% and 100%) but a low sensitivity (1%–12%) for detecting patients at risk of toxicity ([Toffoli et al., 2015](#B17)), suggesting the need for further investigation of the *DPYD* genotype.

The present study investigated the *DPYD* rs4294451 variant as a potential genetic marker for optimizing FL-based therapy and, for the first time, demonstrated a clinical impact of this polymorphism, which has been recently functionally characterized by [Zhang et al. (2024)](#B19). In particular, in the current analysis, being a carrier of an rs4294451-T allele demonstrated a protective effect against the risk of developing G4 toxicity at any time during treatment and a lower probability of developing early toxicity after starting treatment. At the same time, being a carrier of an rs4294451-T allele was related to a worse prognosis in terms of overall survival. The observed clinical impact was consistent with the functional effect described for *DPYD* rs4294451 polymorphism in the work by [Zhang et al. (2024)](#B19). The rs4294451 polymorphism is located in an enhancer element that controls *DPYD* transcription by recruiting the transcription factor CEBPB ([Zhang et al., 2024](#B19)). [Zhang et al. (2024)](#B19) showed an association between the rs4294451-T allele and increased CEBPB-driven *DPYD* expression, which leads to higher catabolism of 5-FU and lower exposure to the drug. The resulting lower sensitivity to 5-FU associated with the rs4294451-T allele agreed with the decreased risk of developing G4 toxicity reported in the present work for patients carrying the same rs4294451-T allele. These patients also consistently had a poor prognosis in terms of survival.

The present study data support the potential clinical utility of the *DPYD* rs4294451 variant in further refining the genotype-guided FL dosing to prevent severe adverse effects. Previous *DPYD* pharmacogenetic association studies focused on toxicities higher than G3, whereas in the present analysis, we investigated the effect of the variant on extremely high-toxicity events such as life-threatening grade 4–5 toxicities. Those toxicities are of particular clinical importance for both the patient quality of life and management costs, often leading to patient hospitalization and are by far the most relevant to prevent ([Le Teuff et al., 2024](#B12)).

According to gnomAD v3.1.2 ([https://www.ensembl.org/](https://www.ensembl.org/)), the rs4294451-T allele is quite common, with a frequency of 40.4% in the African/African American population, 22.7% in the non-Finnish European population, and 24.8%–7.3% in the Asian population, making it a good candidate for improving FL treatment optimization based on the patient’s genetic profile. The discovery of novel predictive markers for personalized FL dose adjustment is of particular importance, especially in populations such as African or Asian populations, where the *DPYD* four variants routinely tested in European countries are less common and therefore of less clinical importance.

The *DPYD* rs4294451 has been reported to be in linkage disequilibrium with other two variants located in the *DPYD* coding region, c.85T>C (rs1801265) and c.496G>A (rs2297595) ([Zhang et al., 2024](#B19)). These variants, in haplotype combination with a third variant (c.1129–5923C>G, rs75017182), have been associated with altered systemic DPD activity ([Hamzic et al., 2021](#B9)), which translates into a different risk of severe toxicity ([Medwid et al., 2023](#B15)). Unfortunately, it was not possible in the present study to analyze the clinical impact of the haplotype combination of rs4294451 with other *DPYD* functional polymorphisms, including the four validated *DPYD* markers, because the study sample number was insufficient for this type of analysis. Further studies are warranted to elucidate the effect of an integrated haplotype, including *DPYD* rs4294451 polymorphism, on the clinical outcome of patients treated with FL.

To highlight the specific effect of the *DPYD* rs4294451 variant, patients who were carriers of the four *DPYD* validated markers were excluded from this analysis. The *DPYD* c.1236G>A variant was adopted to tag the HapB3 haplotype. However, it should be noted that a recent report identified rare cases of missing linkage disequilibrium between c.1236G>A and c.1129–5923C>G polymorphism, which is causative for reduced DPD function ([Turner et al., 2024](#B18)).

While most of the studies that investigated the clinical effect of *DPYD* genetics in patients treated with FL focused on the prevention of toxicity, this study reports for the first time a promising impact of rs4294451 on the patient’s prognosis, consistent with the potentially higher detoxification of the drug reported for the polymorphic allele. Since the *DPYD* rs4294451 variant was associated with increased *DPYD* expression, potentially impacting the drug systemic exposure, an open question remains regarding the contribution of this polymorphism to the mechanism of resistance to 5-FU therapy and, consequently, its effect on the clinical tumor response to FL-based therapy. However, due to the heterogeneity of the study population with respect to baseline clinicopathologic characteristics, it is difficult to draw definitive conclusions about the role of the DPYD rs4294451 variant in modulating patient survival. Moreover, the design of the present study was not able to distinguish the predictive or prognostic value of this DPYD polymorphism, so further analyses are needed to better elucidate the impact of the DPYD rs4294451 variant on FL efficacy and survival.

In conclusion, the *DPYD* variant rs4294451 may be a good candidate for further refining the personalization of FL-based therapy based on the patient genotype in terms of both toxicity and efficacy, indicating the need for further research efforts to validate the preliminary data reported here.

## Funding Statement

The author(s) declare that financial support was received for the research, authorship, and/or publication of this article. This work was supported by the Italian Ministry of Health (Ricerca Corrente).

## Data availability statement

The raw data supporting the conclusion of this article will be made available by the authors, without undue reservation.

## Ethics statement

The studies involving humans were approved by Centro di Riferimento Oncologico (CRO)–IRCCS Ethical Committee. The studies were conducted in accordance with the local legislation and institutional requirements. The participants provided their written informed consent to participate in this study.

## Author contributions

ED: visualization and writing–original draft. JP: formal analysis, visualization, and writing–review and editing. LS: data curation, investigation, and writing–review and editing. EC: supervision, writing–review and editing, and conceptualization.

## Conflict of interest

The authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.

The author(s) declared that they were an editorial board member of Frontiers, at the time of submission. This had no impact on the peer review process and the final decision.

## Generative AI statement

The author(s) declare that no Generative AI was used in the creation of this manuscript.

## Publisher’s note

All claims expressed in this article are solely those of the authors and do not necessarily represent those of their affiliated organizations, or those of the publisher, the editors, and the reviewers. Any product that may be evaluated in this article, or claim that may be made by its manufacturer, is not guaranteed or endorsed by the publisher.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Data Availability Statement

The raw data supporting the conclusion of this article will be made available by the authors, without undue reservation.

### Data Availability Statement

The raw data supporting the conclusion of this article will be made available by the authors, without undue reservation.

## References

1. Amstutz U., Henricks L. M., Offer S. M., Barbarino J., Schellens J. H. M., Swen J. J., et al. (2018). Clinical pharmacogenetics implementation consortium (CPIC) guideline for dihydropyrimidine dehydrogenase genotype and fluoropyrimidine dosing: 2017 update. Clin. Pharmacol. Ther. 103 (2), 210–216. 10.1002/cpt.911  [DOI](https://doi.org/10.1002/cpt.911) | [PMC free article](/articles/PMC5760397/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29152729/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Clinical%20pharmacogenetics%20implementation%20consortium%20(CPIC)%20guideline%20for%20dihydropyrimidine%20dehydrogenase%20genotype%20and%20fluoropyrimidine%20dosing:%202017%20update&author=U.%20Amstutz&author=L.%20M.%20Henricks&author=S.%20M.%20Offer&author=J.%20Barbarino&author=J.%20H.%20M.%20Schellens&volume=103&issue=2&publication_year=2018&pages=210-216&pmid=29152729&doi=10.1002/cpt.911&)

2. Boisdron-Celle M., Capitain O., Faroux R., Borg C., Metges J. P., Galais M. P., et al. (2017). Prevention of 5-fluorouracil-induced early severe toxicity by pre-therapeutic dihydropyrimidine dehydrogenase deficiency screening: assessment of a multiparametric approach. Semin. Oncol. 44 (1), 13–23. 10.1053/j.seminoncol.2017.02.008  [DOI](https://doi.org/10.1053/j.seminoncol.2017.02.008) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28395758/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Semin.%20Oncol.&title=Prevention%20of%205-fluorouracil-induced%20early%20severe%20toxicity%20by%20pre-therapeutic%20dihydropyrimidine%20dehydrogenase%20deficiency%20screening:%20assessment%20of%20a%20multiparametric%20approach&author=M.%20Boisdron-Celle&author=O.%20Capitain&author=R.%20Faroux&author=C.%20Borg&author=J.%20P.%20Metges&volume=44&issue=1&publication_year=2017&pages=13-23&pmid=28395758&doi=10.1053/j.seminoncol.2017.02.008&)

3. Dalle Fratte C., Polesel J., Roncato R., De Mattia E., Ecca F., Bignucolo A., et al. (2018). DPYD gene activity score predicts dose-limiting toxicity in fluoropyrimidine-treated colorectal cancer patients. J. Mol. Clin. Med. 1 (3), 143–149. 10.31083/j.jmcm.2018.03.003  [DOI](https://doi.org/10.31083/j.jmcm.2018.03.003) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Mol.%20Clin.%20Med.&title=DPYD%20gene%20activity%20score%20predicts%20dose-limiting%20toxicity%20in%20fluoropyrimidine-treated%20colorectal%20cancer%20patients&author=C.%20Dalle%20Fratte&author=J.%20Polesel&author=R.%20Roncato&author=E.%20De%20Mattia&author=F.%20Ecca&volume=1&issue=3&publication_year=2018&pages=143-149&doi=10.31083/j.jmcm.2018.03.003&)

4. De Mattia E., Polesel J., Silvestri M., Roncato R., Scarabel L., Calza S., et al. (2023). The burden of rare variants in DPYS gene is a novel predictor of the risk of developing severe fluoropyrimidine-related toxicity. Hum. Genomics 17 (1), 99. 10.1186/s40246-023-00546-9  [DOI](https://doi.org/10.1186/s40246-023-00546-9) | [PMC free article](/articles/PMC10633914/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/37946254/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum.%20Genomics&title=The%20burden%20of%20rare%20variants%20in%20DPYS%20gene%20is%20a%20novel%20predictor%20of%20the%20risk%20of%20developing%20severe%20fluoropyrimidine-related%20toxicity&author=E.%20De%20Mattia&author=J.%20Polesel&author=M.%20Silvestri&author=R.%20Roncato&author=L.%20Scarabel&volume=17&issue=1&publication_year=2023&pages=99&pmid=37946254&doi=10.1186/s40246-023-00546-9&)

5. De Mattia E., Silvestri M., Polesel J., Ecca F., Mezzalira S., Scarabel L., et al. (2022). Rare genetic variant burden in DPYD predicts severe fluoropyrimidine-related toxicity risk. Biomed. Pharmacother. 154, 113644. 10.1016/j.biopha.2022.113644  [DOI](https://doi.org/10.1016/j.biopha.2022.113644) | [PMC free article](/articles/PMC9463069/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36063648/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biomed.%20Pharmacother.&title=Rare%20genetic%20variant%20burden%20in%20DPYD%20predicts%20severe%20fluoropyrimidine-related%20toxicity%20risk&author=E.%20De%20Mattia&author=M.%20Silvestri&author=J.%20Polesel&author=F.%20Ecca&author=S.%20Mezzalira&volume=154&publication_year=2022&pages=113644&pmid=36063648&doi=10.1016/j.biopha.2022.113644&)

6. EMA (2020). EMA recommendations on DPD testing prior to treatment with fluorouracil, capecitabine, tegafur and flucytosine. European Medicines Agency. Available at: https://www.ema.europa.eu/en/news/ema-recommendations-dpd-testing-prior-treatment-fluorouracil-capecitabine-tegafur-flucytosine (Accessed October 01, 2024).  [https://www.ema.europa.eu/en/news/ema-recommendations-dpd-testing-prior-treatment-fluorouracil-capecitabine-tegafur-flucytosine](https://www.ema.europa.eu/en/news/ema-recommendations-dpd-testing-prior-treatment-fluorouracil-capecitabine-tegafur-flucytosine) | [Google Scholar](https://scholar.google.com/scholar_lookup?title=EMA%20recommendations%20on%20DPD%20testing%20prior%20to%20treatment%20with%20fluorouracil,%20capecitabine,%20tegafur%20and%20flucytosine&publication_year=2020&)

7. Fine J. P., Gray R. J. (1999). A proportional hazards model for the subdistribution of a competing risk. J. Am. Stat. Assoc. 94 (446), 496–509. 10.1080/01621459.1999.10474144  [DOI](https://doi.org/10.1080/01621459.1999.10474144) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Am.%20Stat.%20Assoc.&title=A%20proportional%20hazards%20model%20for%20the%20subdistribution%20of%20a%20competing%20risk&author=J.%20P.%20Fine&author=R.%20J.%20Gray&volume=94&issue=446&publication_year=1999&pages=496-509&doi=10.1080/01621459.1999.10474144&)

8. Gray R. J. (1998). A class of $K$-Sample tests for comparing the cumulative incidence of a competing risk. Ann. Statistics 16 (3), 1141–1154. 10.1214/aos/1176350951  [DOI](https://doi.org/10.1214/aos/1176350951) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann.%20Statistics&title=A%20class%20of%20%24K%24-Sample%20tests%20for%20comparing%20the%20cumulative%20incidence%20of%20a%20competing%20risk&author=R.%20J.%20Gray&volume=16&issue=3&publication_year=1998&pages=1141-1154&doi=10.1214/aos/1176350951&)

9. Hamzic S., Scharer D., Offer S. M., Meulendijks D., Nakas C., Diasio R. B., et al. (2021). Haplotype structure defines effects of common DPYD variants c.85T > C (rs1801265) and c.496A > G (rs2297595) on dihydropyrimidine dehydrogenase activity: implication for 5-fluorouracil toxicity. Br. J. Clin. Pharmacol. 87 (8), 3234–3243. 10.1111/bcp.14742  [DOI](https://doi.org/10.1111/bcp.14742) | [PMC free article](/articles/PMC8359980/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33491253/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Clin.%20Pharmacol.&title=Haplotype%20structure%20defines%20effects%20of%20common%20DPYD%20variants%20c.85T%20>%20C%20(rs1801265)%20and%20c.496A%20>%20G%20(rs2297595)%20on%20dihydropyrimidine%20dehydrogenase%20activity:%20implication%20for%205-fluorouracil%20toxicity&author=S.%20Hamzic&author=D.%20Scharer&author=S.%20M.%20Offer&author=D.%20Meulendijks&author=C.%20Nakas&volume=87&issue=8&publication_year=2021&pages=3234-3243&pmid=33491253&doi=10.1111/bcp.14742&)

10. Kalbfleisch J. D., Prentice R. L. (2002). The statistical analysis of failure time data. 2nd Edition.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=The%20statistical%20analysis%20of%20failure%20time%20data&author=J.%20D.%20Kalbfleisch&author=R.%20L.%20Prentice&publication_year=2002&)

11. Knikman J. E., Gelderblom H., Beijnen J. H., Cats A., Guchelaar H. J., Henricks L. M. (2021). Individualized dosing of fluoropyrimidine-based chemotherapy to prevent severe fluoropyrimidine-related toxicity: what are the options? Clin. Pharmacol. Ther. 109 (3), 591–604. 10.1002/cpt.2069  [DOI](https://doi.org/10.1002/cpt.2069) | [PMC free article](/articles/PMC7983939/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33020924/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Individualized%20dosing%20of%20fluoropyrimidine-based%20chemotherapy%20to%20prevent%20severe%20fluoropyrimidine-related%20toxicity:%20what%20are%20the%20options?&author=J.%20E.%20Knikman&author=H.%20Gelderblom&author=J.%20H.%20Beijnen&author=A.%20Cats&author=H.%20J.%20Guchelaar&volume=109&issue=3&publication_year=2021&pages=591-604&pmid=33020924&doi=10.1002/cpt.2069&)

12. Le Teuff G., Cozic N., Boyer J. C., Boige V., Diasio R. B., Taieb J., et al. (2024). Dihydropyrimidine dehydrogenase gene variants for predicting grade 4-5 fluoropyrimidine-induced toxicity: FUSAFE individual patient data meta-analysis. Br. J. Cancer 130 (5), 808–818. 10.1038/s41416-023-02517-2  [DOI](https://doi.org/10.1038/s41416-023-02517-2) | [PMC free article](/articles/PMC10912560/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/38225422/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Cancer&title=Dihydropyrimidine%20dehydrogenase%20gene%20variants%20for%20predicting%20grade%204-5%20fluoropyrimidine-induced%20toxicity:%20FUSAFE%20individual%20patient%20data%20meta-analysis&author=G.%20Le%20Teuff&author=N.%20Cozic&author=J.%20C.%20Boyer&author=V.%20Boige&author=R.%20B.%20Diasio&volume=130&issue=5&publication_year=2024&pages=808-818&pmid=38225422&doi=10.1038/s41416-023-02517-2&)

13. Lunenburg C., Henricks L. M., Guchelaar H. J., Swen J. J., Deenen M. J., Schellens J. H. M., et al. (2016). Prospective DPYD genotyping to reduce the risk of fluoropyrimidine-induced severe toxicity: ready for prime time. Eur. J. Cancer 54, 40–48. 10.1016/j.ejca.2015.11.008  [DOI](https://doi.org/10.1016/j.ejca.2015.11.008) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26716401/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Cancer&title=Prospective%20DPYD%20genotyping%20to%20reduce%20the%20risk%20of%20fluoropyrimidine-induced%20severe%20toxicity:%20ready%20for%20prime%20time&author=C.%20Lunenburg&author=L.%20M.%20Henricks&author=H.%20J.%20Guchelaar&author=J.%20J.%20Swen&author=M.%20J.%20Deenen&volume=54&publication_year=2016&pages=40-48&pmid=26716401&doi=10.1016/j.ejca.2015.11.008&)

14. Lunenburg C., van der Wouden C. H., Nijenhuis M., Crommentuijn-van Rhenen M. H., de Boer-Veger N. J., Buunk A. M., et al. (2020). Dutch Pharmacogenetics Working Group (DPWG) guideline for the gene-drug interaction of DPYD and fluoropyrimidines. Eur. J. Hum. Genet. 28 (4), 508–517. 10.1038/s41431-019-0540-0  [DOI](https://doi.org/10.1038/s41431-019-0540-0) | [PMC free article](/articles/PMC7080718/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31745289/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Hum.%20Genet.&title=Dutch%20Pharmacogenetics%20Working%20Group%20(DPWG)%20guideline%20for%20the%20gene-drug%20interaction%20of%20DPYD%20and%20fluoropyrimidines&author=C.%20Lunenburg&author=C.%20H.%20van%20der%20Wouden&author=M.%20Nijenhuis&author=M.%20H.%20Crommentuijn-van%20Rhenen&author=N.%20J.%20de%20Boer-Veger&volume=28&issue=4&publication_year=2020&pages=508-517&pmid=31745289&doi=10.1038/s41431-019-0540-0&)

15. Medwid S., Wigle T. J., Kim R. B. (2023). Fluoropyrimidine-associated toxicity and DPYD variants c.85T>C, c.496A>G, and c.1236G>A: impact of haplotype. Cancer Chemother. Pharmacol. 91 (1), 97–102. 10.1007/s00280-022-04491-7  [DOI](https://doi.org/10.1007/s00280-022-04491-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36357798/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Chemother.%20Pharmacol.&title=Fluoropyrimidine-associated%20toxicity%20and%20DPYD%20variants%20c.85T>C,%20c.496A>G,%20and%20c.1236G>A:%20impact%20of%20haplotype&author=S.%20Medwid&author=T.%20J.%20Wigle&author=R.%20B.%20Kim&volume=91&issue=1&publication_year=2023&pages=97-102&pmid=36357798&doi=10.1007/s00280-022-04491-7&)

16. Pratt V. M., Cavallari L. H., Fulmer M. L., Gaedigk A., Hachad H., Ji Y., et al. (2024). DPYD genotyping recommendations: a joint consensus recommendation of the association for molecular Pathology, American college of medical genetics and genomics, clinical pharmacogenetics implementation consortium, college of American pathologists, Dutch pharmacogenetics working Group of the royal Dutch pharmacists association, European society for pharmacogenomics and personalized therapy, pharmacogenomics knowledgebase, and pharmacogene variation consortium. J. Mol. Diagn 26 (10), 851–863. 10.1016/j.jmoldx.2024.05.015  [DOI](https://doi.org/10.1016/j.jmoldx.2024.05.015) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/39032821/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Mol.%20Diagn&title=DPYD%20genotyping%20recommendations:%20a%20joint%20consensus%20recommendation%20of%20the%20association%20for%20molecular%20Pathology,%20American%20college%20of%20medical%20genetics%20and%20genomics,%20clinical%20pharmacogenetics%20implementation%20consortium,%20college%20of%20American%20pathologists,%20Dutch%20pharmacogenetics%20working%20Group%20of%20the%20royal%20Dutch%20pharmacists%20association,%20European%20society%20for%20pharmacogenomics%20and%20personalized%20therapy,%20pharmacogenomics%20knowledgebase,%20and%20pharmacogene%20variation%20consortium&author=V.%20M.%20Pratt&author=L.%20H.%20Cavallari&author=M.%20L.%20Fulmer&author=A.%20Gaedigk&author=H.%20Hachad&volume=26&issue=10&publication_year=2024&pages=851-863&pmid=39032821&doi=10.1016/j.jmoldx.2024.05.015&)

17. Toffoli G., Giodini L., Buonadonna A., Berretta M., De Paoli A., Scalone S., et al. (2015). Clinical validity of a DPYD-based pharmacogenetic test to predict severe toxicity to fluoropyrimidines. Int. J. Cancer 137 (12), 2971–2980. 10.1002/ijc.29654  [DOI](https://doi.org/10.1002/ijc.29654) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26099996/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Cancer&title=Clinical%20validity%20of%20a%20DPYD-based%20pharmacogenetic%20test%20to%20predict%20severe%20toxicity%20to%20fluoropyrimidines&author=G.%20Toffoli&author=L.%20Giodini&author=A.%20Buonadonna&author=M.%20Berretta&author=A.%20De%20Paoli&volume=137&issue=12&publication_year=2015&pages=2971-2980&pmid=26099996&doi=10.1002/ijc.29654&)

18. Turner A. J., Haidar C. E., Yang W., Boone E. C., Offer S. M., Empey P. E., et al. (2024). Updated DPYD HapB3 haplotype structure and implications for pharmacogenomic testing. Clin. Transl. Sci. 17 (1), e13699. 10.1111/cts.13699  [DOI](https://doi.org/10.1111/cts.13699) | [PMC free article](/articles/PMC10777430/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/38129972/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Transl.%20Sci.&title=Updated%20DPYD%20HapB3%20haplotype%20structure%20and%20implications%20for%20pharmacogenomic%20testing&author=A.%20J.%20Turner&author=C.%20E.%20Haidar&author=W.%20Yang&author=E.%20C.%20Boone&author=S.%20M.%20Offer&volume=17&issue=1&publication_year=2024&pages=e13699&pmid=38129972&doi=10.1111/cts.13699&)

19. Zhang T., Ambrodji A., Huang H., Bouchonville K. J., Etheridge A. S., Schmidt R. E., et al. (2024). Germline cis variant determines epigenetic regulation of the anti-cancer drug metabolism gene dihydropyrimidine dehydrogenase (DPYD). Elife 13. 10.7554/eLife.94075  [DOI](https://doi.org/10.7554/eLife.94075) | [PMC free article](/articles/PMC11060711/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/38686795/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Elife&title=Germline%20cis%20variant%20determines%20epigenetic%20regulation%20of%20the%20anti-cancer%20drug%20metabolism%20gene%20dihydropyrimidine%20dehydrogenase%20(DPYD)&author=T.%20Zhang&author=A.%20Ambrodji&author=H.%20Huang&author=K.%20J.%20Bouchonville&author=A.%20S.%20Etheridge&volume=13&publication_year=2024&pmid=38686795&doi=10.7554/eLife.94075&)
