# SLCO1B1 Functional Variants, Bilirubin, Statin‐Induced Myotoxicity, and Recent Sub‐Saharan African Ancestry: A Precision Medicine Health Equity Study

## Metadata
**Authors:** Tanushree Haldar, Mark Kvale, Jia Yang, Michael P Douglas, Willow Coyote‐Maetas, Linda Kachuri, John S Witte, Carlos Iribarren, Marisa W Medina, Ronald M Krauss, Sook Wah Yee, Akinyemi Oni‐Orisan
**Journal:** Clinical Pharmacology and Therapeutics
**Date:** 2025 Mar 6
**DOI:** [10.1002/cpt.3624](https://doi.org/10.1002/cpt.3624)
**PMID:** 40047317
**PMCID:** PMC12087686
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC12087686/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC12087686/pdf/CPT-117-1696.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC12087686/pdf/CPT-117-1696.pdf)

## Abstract

Statin pharmacogenetic implementation guidelines are derived from evidence of primarily Eurocentrically biased study populations. Functional SLCO1B1 variants that are rare in these study populations have not been equitably investigated and are thus missing from guidelines. The objective of this precision medicine health equity study was to determine the clinical validity of understudied candidate functional SLCO1B1 variants common in people with 1,000 Genomes sub‐Saharan African superpopulation (1KG‐AFR‐like) genetic similarity. We conducted our analyses using the real‐world evidence of participants from three large, electronic health record‐linked biobanks. We used bilirubin levels (as an endogenous substrate of organic anion transporting polypeptide [OATP1B1] function) and severe statin‐induced myotoxicity phenotypes. Loss‐of‐function splice variant rs77271279 (P = 1.1 × 10−17) had the strongest association with elevated total bilirubin levels in Black participants (mean 84% AFR‐like genetic similarity) followed by missense variant rs59502379 (P = 7.4 × 10−12) then missense variant rs4149056 (P = 6.0 × 10−5). In an exploratory subset of the Black study population who used statins (n = 77 severe statin‐induced myotoxicity cases), rs59502379 (odds ratio [OR] = 2.85, 95% confidence interval [CI] 1.08–7.52), but not rs77271279 (OR = 1.75, 95% CI 0.62–4.73) was associated with myotoxicity. Sensitivity analyses in participants with >5% AFR‐like genetic similarity corroborated these findings. For white participants, rs77271279 and rs59502379 were rare precluding subsequent analyses. Our findings highlight the clinical relevance for understudied SLCO1B1 variants on pharmacogenetic testing panels with a potential immediate impact on reducing the risk of severe statin‐induced myotoxicity primarily in Black patients, a group historically excluded from genomic research. Future studies require larger statin user study populations with less heterogeneity by statin type.

Variants known to impact OATP1B1 function that are rare in Eurocentrically biased study populations have not been investigated in cohorts as a determinant of statin myotoxicity and are thus missing from guideline inclusion. Thus, pharmacogenetic tests for statin therapy are not providing equitable benefits in preventing the risk of myotoxicity.

This health equity study determined the clinical validity of understudied candidate functional *SLCO1B1* variants common in people with recent genealogical ancestors from Africa.

This investigation is the first to demonstrate a strong association between two *SLCO1B1* polymorphisms and levels of an endogenous biomarker for OATP1B1 function. Effect sizes were larger than what was found with the well‐established c.521T>C variant.

Eventual incorporation of these variants into preemptive pharmacogenetic testing panels could have a significantly positive impact on the risk of severe statin‐induced myotoxicity, especially in Black patients, who make up a group historically excluded from genomic research.

3‐Hydroxy‐3‐methylglutaryl coenzyme A (HMG‐CoA) reductase inhibitors (more commonly known as statins) are among the most efficacious therapies for the prevention of atherosclerotic cardiovascular disease (ASCVD). Their broader population‐wide utilization, which would reduce the burden of this major public health issue, is substantially hampered by the adverse drug reaction of myotoxicity.[1](#cpt3624-bib-0001) Pharmacogenetic findings linking elevated statin drug levels to myotoxicity[2](#cpt3624-bib-0002), [3](#cpt3624-bib-0003) suggest that these debilitating adverse drug reactions cannot be entirely attributed to a nocebo affect and that a precision medicine approach to statin selection improves outcomes.

Currently, the Clinical Pharmacogenetics Implementation Consortium (CPIC) provides recommendations with the highest strength of evidence for statin prescribing based on *SLCO1B1* (which encodes the liver‐specific organic anion transporting polypeptide [OATP1B1]) genetic test results to improve therapeutic safety by reducing the risk of myotoxicity.[3](#cpt3624-bib-0003) Consequently, statin pharmacogenetics is among the most common clinical pharmacogenetic implementation examples in practice[4](#cpt3624-bib-0004); the inclusion of *SLCO1B1* as part of a 12‐gene pharmacogenetic test panel in a landmark randomized clinical trial was a major contributor to the primary outcome showing a 30% relative risk reduction for general adverse drug reactions.[5](#cpt3624-bib-0005) However, the evidence for which CPIC recommendations of statins and other therapies originate from Eurocentrically biased study populations (i.e., participants who predominantly self‐identify as white, derive recent ancestry from any region within the continent of Europe, and/or have genetic similarity to participants of the 1,000 Genomes European superpopulation [1KG‐EUR‐like]).[2](#cpt3624-bib-0002), [6](#cpt3624-bib-0006) As a result, variants known to impact OATP1B1 function that are rare in these groups but common in other groups (i.e., one or more of the diverse groups that do not fit the above definition of a Eurocentrically biased study population) have not been investigated as a determinant of statin myotoxicity and are thus missing from guideline inclusion.

Taken together, these prior findings demonstrate that pharmacogenetic tests for statin therapy are not providing equitable benefits in preventing the risk of adverse drug reactions. Enhanced investigation in study populations not Eurocentrically biased will generate the evidence base necessary for more inclusive pharmacogenetic prescribing guidelines. The objective of this health equity study was to determine the clinical validity of understudied candidate functional *SLCO1B1* variants common in people with recent genealogical ancestors from Africa. To do this, we investigated the association between our variants of interest and endogenous total bilirubin levels as a routinely measured OATP1B1 activity biomarker.

## METHODS

### Data resources

We conducted our studies using de‐identified, individual‐level, population‐based data capturing the health of participants from three independent electronic health record‐linked biobanks (each with genome‐wide genotype data): the National Institutes of Health (NIH) All Of Us research program (AoURP), the Genetic Epidemiology Research on Adult Health and Aging (GERA) Cohort, and the UK Biobank (UKB).[7](#cpt3624-bib-0007), [8](#cpt3624-bib-0008), [9](#cpt3624-bib-0009) Before inclusion and exclusion criteria were applied, we had access to ~900,000 participants across the three cohorts. Institutional Review Board (IRB) approval was obtained from both Kaiser Permanente (Mid‐Atlantic States Region) and the University of California. Participants gave written informed consent. See **Supplementary Material** [S1](#cpt3624-supitem-0001) for further details on these data resources.

### Dataset types

Within each data resource, we used electronic health record, self‐administered survey, physical measurement, and genomic data. Further details on our dataset types are available in the **Supplementary Material** [S1](#cpt3624-supitem-0001).

### Justification for the use of population descriptors in genetic studies

Primarily, participants who self‐identified as white or Black were included in the study. We also included participants with >5% genetic similarity to at least one subpopulation group within the 1,000 Genomes African superpopulation (i.e., 1KG‐AFR‐like, people with recent genealogical ancestors from Africa).

The selection of these descriptors was carefully done to (1) highlight a health disparity essential for achieving our primary study objective and (2) avoid the common practice of reporting Eurocentrically biased results as universally beneficial. Our investigation meets the NIH National Institute on Minority Health and Health Disparities (NIMHD) definition of health disparities research because we report “a health difference, on the basis of one or more health outcomes, that adversely affects disadvantaged populations”[10](#cpt3624-bib-0010). Furthermore, this study fits under the classification of health equity because it reflects a “commitment to reduce—and, ultimately, eliminate—disparities in health” by attempting to improve health outcomes for people who are understudied in pharmacogenetics.[11](#cpt3624-bib-0011) When race was reported, it was not used as a proxy for genetic similarity, rather it was used to underscore the health equity aspect of the study. The >5% 1KG‐AFR‐like genetic similarity threshold, on the other hand, was used to ensure that we maximized the coverage of participants carrying our variants of interest while avoiding a threshold so liberal (e.g., pooling together all participants) that it would dilute the effect. Our terminology and methods are in concordance with guidance from the National Academies of Sciences, Engineering, and Medicine[12](#cpt3624-bib-0012) and also informed by recent health disparities precision medicine research.[13](#cpt3624-bib-0013) Further details on our chosen population descriptors are available in the Discussion section and the **Supplementary Material** [S1](#cpt3624-supitem-0001). The reference groups used for the genetic similarity population description are shown in the Genetic Similarity section. The specific definitions for the race categories of each cohort are described in the Data Resources section of the **Supplementary Material** [S1](#cpt3624-supitem-0001).

### Genetic similarity

The output of genome‐wide principal component analyses was used as a measure of continental genetic similarity. These analyses were performed using microarray genotyping results from GERA and AoURP with PLINK2 software. Linkage disequilibrium (LD) pruning was conducted on the genotyped data to generate the principal components using a subset of independent variants. We used the following settings to LD prune (window size = 1,000 variants, step size = 50 variants, *r* ^2^ threshold = 0.05). First‐ and second‐degree relatives (kinship coefficient cutoff of 0.12) were removed prior to LD pruning. For UKB, continental genetic similarity principal components were previously computed using fastPCA based on a set of 407,219 unrelated samples and 147,604 genetic markers.[14](#cpt3624-bib-0014)

For AoURP, continental genetic ancestry was estimated using Rye[15](#cpt3624-bib-0015) with a reference data set that consisted of 3,433 participants from 1,000 Genome Project and Human Genome Diversity Project (1KGPHGDP). AFR‐like genetic ancestry was based on reference participants from Esan in Nigeria (ESN), Gambian in Western Division—Mandinka (GWD), Luhya in Webuye, Kenya (LWK), Mende in Sierra Leone (MSL), and Yoruba in Ibadan, Nigeria (YRI). For GERA, continental genetic ancestry was previously estimated[16](#cpt3624-bib-0016) using the full maximum‐likelihood software package *frappe*.

### Genotype

Consistent with the overall objective of this study, we selected candidate *SLCO1B1* variants with evidence of mechanistic function that are common (>1% allele frequency) in subpopulation groups with 1KG‐AFR‐like genetic similarity. Subpopulation groups were gathered from the online Genome Aggregation Database (gnomAD) browser: [https://gnomad.broadinstitute.org/](https://gnomad.broadinstitute.org/). A systematic search for the *SLCO1B1* variants was conducted within gnomAD and PubMed database. Two independent investigators (S.W.Y. and A.O.) conducted the search using relevant genetic terms (“SLCO1B1” or “OATP1B1”) in combination with relevant populations descriptors (“sub‐Saharan,” “African,” “Black,” “AFR”). Through this search, we identified 3 variants that met the above criteria: missense variant reference single nucleotide polymorphism (rs)4,149,056 (nucleotide c.521T>C, amino acid OATP1B1‐Val174Ala), missense variant rs59502379 (c.1463G>C, OATP1B1‐Gly488Ala), and loss‐of‐function splice variant rs77271279 (c.481+1G>T).[17](#cpt3624-bib-0017), [18](#cpt3624-bib-0018), [19](#cpt3624-bib-0019) The direction of effect from each polymorphism was reduced OATP1B1 function.[18](#cpt3624-bib-0018), [20](#cpt3624-bib-0020) Among the 3 identified variants, we considered the latter 2 as understudied since only c.521T>C was mentioned in recent CPIC guidelines.[3](#cpt3624-bib-0003), [21](#cpt3624-bib-0021) Only c.521T>C among the 3 had a minor allele frequency considered common (>0.01) in ≥1 group outside of 1KG‐AFR. Further details on our genotypes including the systematic search are available in the **Supplementary Material** [S1](#cpt3624-supitem-0001).

### Phenotype (bilirubin levels)

As the primary outcome of our study, we used bilirubin, an endogenous substrate for OATP1B1.[22](#cpt3624-bib-0022) We used total bilirubin levels as a biomarker for OATP1B1 activity. Findings from multiple genome‐wide association studies for total bilirubin support its use as a biomarker for OATP1B1 activity.[23](#cpt3624-bib-0023), [24](#cpt3624-bib-0024) To rule out the effect of bilirubin levels impacted largely by disease or organ dysfunction, we excluded levels that were outside of normal limits. The thresholds used to set the limit of normal were 1.71–20.50 micromoles per liter (UKB) or 0.1–1.2 milligrams (AoURP and GERA) based on standard of care. Further details are available in the **Supplementary Material** [S1](#cpt3624-supitem-0001).

### Phenotype (severe statin‐induced myotoxicity)

We generated a phenotype for severe statin‐induced myotoxicity, as previously described[25](#cpt3624-bib-0025) with modifications. Briefly, a participant was considered to have experienced a severe statin‐induced myotoxicity event if they had evidence of statin‐induced rhabdomyolysis or statin‐induced myopathy (creatine kinase levels >5× the upper limit of normal). We intentionally chose a creatine kinase level threshold higher than often used for standard myopathy definitions[26](#cpt3624-bib-0026), [27](#cpt3624-bib-0027) to capture only the most severe cases of statin‐induced myotoxicity. Due to the smaller sample size compared to the participants with total bilirubin levels, this phenotype was used as a secondary outcome. This drug response phenotype was generated in AoURP and GERA only. Incomplete pharmacy records prohibited the generation of a drug response phenotype in UKB (UKB statin data were only cross‐sectional, based on self‐administered survey data, and lacked details on type/dose). Further details are available in both the Methods section and **Table** [S1](#cpt3624-supitem-0001) of the **Supplementary Material** [S1](#cpt3624-supitem-0001).

### Cell‐based assay studies

We conducted *in vitro* mechanistic studies to support our population‐based studies and extend the current functional evidence for our *SLCO1B1* variants of interest. Briefly, we established HEK293T cell lines expressing either OATP1B1 reference or one of our two reduced function amino acid changes (OATP1B1‐Val174Ala or OATP1B1‐Gly488Ala). We then performed a variety of assays to assess the functional impact of these amino acid variants. See **Supplementary Material** [S1](#cpt3624-supitem-0001) for details.

### Statistical analysis

As a primary analysis, we investigated the association between our candidate variants and log_10_‐transformed total bilirubin levels in self‐identified Black participants using linear regression (with age, sex, body mass index, and 10 principal components of continental genetic similarity as covariates). The inclusion of principal components as covariates was used to account for population stratification as a potential confounder. The first 10 principal components accounted for 88% of the total population structure variance from the first 30 components. Meta‐analyses of these results from AoURP, GERA, and UKB were conducted using METAL.[28](#cpt3624-bib-0028) First, effect sizes for each variant within a cohort were normalized as Z‐scores. Then Z‐scores were combined across cohorts weighted based on cohort size (weights proportional to the square root of the sample size). As a sensitivity analysis, we repeated our primary analyses except specifically in participants with >5% genetic similarity to the 1KG‐AFR‐like superpopulation. A subsequent sensitivity analysis repeated the primary analyses in strata based on the UGT1A1 genotype.

For the secondary analysis, we determined the relationship between our candidate *SLCO1B1* variants (c.521T>C, c.481+1G>T, c.1463G>C) and severe statin‐induced myotoxicity using multiple logistic regression in race‐stratified groups from both cohorts from which we made drug response phenotypes (AoURP, GERA). Participant age, sex assigned at birth, and the first 10 principal components of continental genetic similarity were included as covariates. We conducted these analyses in race‐stratified meta‐analyses of each variant across cohorts for all statin types combined. Under the assumption that genetic effects do not vary by cohort, we used fixed‐effect meta‐analyses to model these pharmacogenetic relationships.[29](#cpt3624-bib-0029) We also performed the meta‐analyses associations in participants with >5% genetic similarity to the 1KG‐AFR‐like superpopulation.

Our tertiary analyses were cell‐based studies as described above and in the Methods section of the **Supplementary Material** [S1](#cpt3624-supitem-0001).

All analyses were conducted using R software (version 4.3.1; R Foundation for Statistical Computing; [www.R‐project.org](http://www.r-project.org); Vienna, Austria). STROBE guidelines were followed for reporting the results of our population‐based studies.[30](#cpt3624-bib-0030) *P* values <0.05 were considered statistically significant.

### Role of the funding source

This study was funded by the National Institutes of Health and the Office of the Director – All of Us (OD‐AoURP). These sponsors did not have any role in the study design, conduct, or reporting.

## RESULTS

### Characterization of the study population

A combined total of 527,500 participants from all 3 cohorts met the criteria of study inclusion for our primary analysis. Among these, 28,332 (5%) self‐identified as Black and 499,168 (95%) as white. For our sensitivity analysis, 41,703 participants (including 28,367 participants self‐identifying as Black [68%]) had >5% genetic similarity to the 1KG‐AFR‐like superpopulation. Full baseline characteristics in this total bilirubin study population are shown in **Tables** [S2](#cpt3624-supitem-0001) **and** [S3](#cpt3624-supitem-0001).

For our secondary analysis, a total of 578 severe statin‐induced myotoxicity cases met the criteria of study inclusion. Among these, 77 identified as Black and 501 as white (**Table** [S4](#cpt3624-supitem-0001)). The number of matched statin user controls was 657 and 5,161, respectively for Black and white participants.

### The relationship between SLCO1B1 variants and total bilirubin levels

Within our study population of Black participants across all 3 cohorts, each of the 3 candidate variants was associated with elevated total bilirubin levels. The strongest association was observed for c.481+1G>T (Z‐score transformed beta = 8.57, standard error [SE] = 1.00, *P* = 1.1 × 10^−17^) followed by c.1463G>C (beta = 6.85, SE = 1.00, *P* = 7.4 × 10^−12^) and c.521T>C (beta = 4.01, SE = 1.00, *P* = 6.0 × 10^−5^, **Table** [1](#cpt3624-tbl-0001)). Within our white study population, c.521T>C was associated with elevated total bilirubin levels (beta = 31.25, SE = 1.00, *P* = 2.1 × 10^−473^). No association was observed between our 1KG‐AFR‐specific variants (c.481 + 1G>T and c.1463G>C) and total bilirubin in white participants (**Table** [S5](#cpt3624-supitem-0001)). Consistent results were generally observed within individual cohorts (**Figure** [1](#cpt3624-fig-0001), **Table** [1](#cpt3624-tbl-0001)). Sensitivity analyses within our participants with >5% 1KG‐AFR‐like genetic similarity showed gene‐bilirubin results nearly identical to those of the Black participants (**Table** [S6](#cpt3624-supitem-0001)). Similar gene‐bilirubin results were observed within strata defined by the *UGT1A1*28* genotype (**Table** [S7](#cpt3624-supitem-0001)). Finally, our *SLCO1B1* variants were independently associated with bilirubin compared to *UGT1A1*28* (**Table** [S8](#cpt3624-supitem-0001)).

### Table 1.

| SLCO1B1 variant | Analysis groupb | AFR genetic similarityc | Effect allele frequency | Cohort | Sample size | Betad | SE | P‐value |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
| c.481+1G>T | Black |   |   | Meta‐analysis | 28,332 | 8.57 | 1.00 | 1.1 × 10−17 |
| 0.86 | 0.030 | AoURP | 19,673 | 0.035 | 0.005 | 7.6 × 10−11 |  |  |
| 0.71 | 0.023 | GERA | 2,198 | 0.225 | 0.099 | 0.02 |  |  |
| 0.84 | 0.027 | UKB | 6,461 | 0.100 | 0.020 | 9.8 × 10−8 |  |  |
| c.1463G>C | Black |   |   | Meta‐analysis | 28,332 | 6.85 | 1.00 | 7.4 × 10−12 |
| 0.86 | 0.036 | AoURP | 19,673 | 0.025 | 0.005 | 5.0 × 10−7 |  |  |
| 0.71 | 0.026 | GERA | 2,198 | 0.181 | 0.095 | 0.06 |  |  |
| 0.84 | 0.024 | UKB | 6,461 | 0.071 | 0.016 | 7.0 × 10−6 |  |  |
| c.521T>C | Black |   |   | Meta‐analysis | 28,332 | 4.01 | 1.00 | 6.0 × 10−5 |
| 0.86 | 0.029 | AoURP | 19,673 | 0.020 | 0.005 | 1.4 × 10−4 |  |  |
| 0.71 | 0.049 | GERA | 2,198 | 0.137 | 0.070 | 0.05 |  |  |
| 0.84 | 0.043 | UKB | 6,461 | 0.012 | 0.02 | 0.57 |  |  |
| Whitee |   |   | Meta‐analysis | 499,168 | 46.57 | 1.00 | 2.1 × 10−473 |  |
| 0.0004 | 0.160 | AoURP | 67,462 | 0.027 | 0.001 | 1.6 × 10−104 |  |  |
| 0.0062 | 0.155 | GERA | 50,979 | 0.160 | 0.009 | 7.0 × 10−76 |  |  |
| 0.0030 | 0.151 | UKB | 380,727 | 0.058 | 0.001 | 9.3 × 10−681 |  |  |

Table 1 Caption: The relationship a between candidate functional SLCO1B1 variants and total bilirubin levels

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0ebc/12087686/b6f87f6ebd0e/CPT-117-1696-g001.jpg)

The relationship1 between SLCO1B1 variants and total bilirubin levels (micromoles per liter) in self‐identified Black participants (n = 6,461) from the UK Biobank (UKB). Loss‐of‐function splice variant c.481+1G>T (rs77271279) had the strongest association (P = 9.8 × 10−8) with elevated total bilirubin levels (an endogenous substrate of organic anion transporting polypeptide [OATP1B1] function that we used as a biomarker for enzyme activity) followed by missense variant c.1463G>C (rs59502379, P = 7.0 × 10−6). Missense variant c.521T>C (rs4149056) was not associated with total bilirubin levels (P = 0.57). Minor allele frequencies were similar for all variants (0.020.04). Age, sex, body mass index, and 10 principal components of continental genetic similarity were added as covariates. Data presented as mean and standard error range. *P < 0.05.

### The relationship between SLCO1B1 variants and severe statin‐induced myotoxicity

Our meta‐analysis results for the secondary analysis demonstrated an increased risk of severe statin‐induced myotoxicity with c.1463G>C (odds ratio [OR] = 2.85, 95% confidence interval [CI] 1.08–7.52, *P* = 0.03), but not c.481 + 1G>T (OR = 1.75, 95% CI 0.62–4.73, *P* = 0.09) or c.521T>C (OR = 1.47, 95% CI 0.63–3.42, *P* = 0.37) for Black participants across all statin types (**Table** [2](#cpt3624-tbl-0002)). The most common statin type used within this subset was atorvastatin (~40% of cases). An exploratory analysis within the atorvastatin users yielded similar effect sizes relative to all statins combined: OR = 3.16 for c.1463G>C (95% CI 0.88–11.32, *P* = 0.08), OR = 1.59 for c.481+1G>T (95% CI 0.37–6.87, *P* = 0.53), and OR = 1.11 for c.521T>C (95% CI 0.22–5.54, *P* = 0.90). Further statin type‐specific results are available in **Table** [S9](#cpt3624-supitem-0001). Similar to the bilirubin association results, these gene‐drug response relationships among our participants with >5% genetic similarity to the 1KG‐AFR‐like superpopulation mirrored those of our Black participants (**Tables** [S10](#cpt3624-supitem-0001) **and** [S11](#cpt3624-supitem-0001)).

### Table 2.

| SLCO1B1 variant | Analysis groupb | Effect allele frequency | AFR genetic similarityc | Cohort | Sample size (case/control) | Beta | SE | Odds ratio | 95% Cl | P‐value |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| c.481+1G>T | Black |   |   | Meta‐analysis | 77/770 | 0.54 | 0.52 | 1.71 | 0.62–4.73 | 0.30 |
| 0.035 | 0.88 | AoURP | 36/360 | −0.26 | 0.77 |   |   |   |  |  |
| 0.017 | 0.77 | GERA | 41/410 | 1.20 | 0.70 |   |   |   |  |  |
| c.1463G>C | Black |   |   | Meta‐analysis | 77/770 | 1.05 | 0.49 | 2.85 | 1.08–7.52 | 0.03 |
| 0.037 | 0.88 | AoURP | 36/360 | 1.47 | 0.53 |   |   |   |  |  |
| 0.030 | 0.77 | GERA | 41/410 | −1.90 | 1.40 |   |   |   |  |  |
| c.521T>C | Black |   |   | Meta‐analysis | 77/770 | 0.39 | 0.43 | 1.47 | 0.63–3.42 | 0.37 |
| 0.023 | 0.88 | AoURP | 36/360 | 0.21 | 0.79 |   |   |   |  |  |
| 0.045 | 0.77 | GERA | 41/410 | 0.46 | 0.51 |   |   |   |  |  |
| Whited |   |   | Meta‐analysis | 501/5,010 | 0.31 | 0.08 | 1.36 | 1.16–1.61 | 2.4 × 10−4 |  |
| 0.162 | 0.00 | AoURP | 54/540 | 0.10 | 0.27 |   |   |   |  |  |
| 0.156 | 0.0043 | GERA | 447/4,470 | 0.33 | 0.09 |   |   |   |  |  |

Table 2 Caption: Relationship a between candidate functional SLCO1B1 variants and severe statin‐induced myotoxicity

For White participants, c.521T>C was significantly associated with the increased risk of severe statin‐induced myotoxicity with a similar effect as observed for Black participants (OR = 1.41, 95% CI 1.20–1.67, *P* = 5.4 × 10^−5^). Cohort‐specific results are displayed in **Table** [2](#cpt3624-tbl-0002).

### Cell‐based mechanistic studies for OATP1B1‐Val174Ala and OATP1B1‐Gly488Ala

Following a 24‐hour transient transfection of CMV‐mNG2‐1‐10‐P2A‐mCherry[31](#cpt3624-bib-0031) into the landing cells expressing OATP1B1 reference, OATP1B1‐Val174Ala, or OATP1B1‐Gly488Ala, cells were treated with 2 μg/mL doxycycline to induce transporter expression. HEK293T landing cells transiently overexpressing OATP1B1 reference, OATP1B1‐Val174Ala, and OATP1B1‐Gly488Ala exhibited substantially higher levels of mNeonGreen abundance compared to HEK293T landing cells without transfection of CMV‐mNG2‐1‐10‐P2A‐mCherry (**Figure** [2 a](#cpt3624-fig-0002)). The OATP1B1‐Val174Ala and OATP1B1‐Gly488Ala amino acid substitution exhibited similar mNeonGreen fluorescence when compared to the OATP1B1 reference (**Figure** [2 a](#cpt3624-fig-0002)). Moreover, OATP1B1 reference, OATP1B1‐Val174Ala, and OATP1B1‐Gly488Ala each had plasma membrane fluorescence green fluorescence protein (GFP) expression (**Figure** [2 b](#cpt3624-fig-0002)).

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0ebc/12087686/831ee57c85ed/CPT-117-1696-g002.jpg)

Functional characterizations of two missense variants in OATP1B1. (a) OATP1B1 reference (REF) and the two missense OATP1B1 variants (V174A and G488A) had similar green fluorescent protein (GFP) expression as well as levels greater than that of HEK293T landing cells without GFP (left). OATP1B1 REF has increased uptake of OATP1B1 substrate fluoresceine‐methotrexate (FL‐MTX) compared to the missense variants (right). (b) Cellular localization of GFP‐tagged OATP1B1 REF and V174A overlaps with the plasma membrane marker wheat germ agglutinin (WGA). OATP1B1 G488A showed weaker expression on the plasma membrane and some intracellular localization. (c) Cytotoxicity of rosuvastatin was greatest in HEK293T landing pad cells with doxycycline‐ (Dox‐) induced OATP1B1 transporter expression (red), followed by OATP1B1‐V174A (green) then OATP1B1‐G488A (blue) compared to cells without OATP1B1 expression (open circle). Both missense OATP1B1 variants (V174A and G488A) showed significantly reduced uptake of radioligands (d) [3H]‐estradiol‐17beta‐glucuronide (EG) and (e) [3H]‐esterone sulfate (ES) compared to OATP1B1 REF.

Despite similar GFP expression compared to OATP1B1 reference, the amino acid substitutions demonstrated significantly reduced intracellular fluorescein‐methotrexate as well as reduced uptake of both estrone sulfate and estradiol‐17β‐glucuronide (**Figure** [2 a,d,e](#cpt3624-fig-0002)). Furthermore, only OATP1B1‐Gly488Ala showed mixed localization of GFP abundance on the plasma membrane and intracellularly (**Figure** [2 b](#cpt3624-fig-0002)). There was a slight reduction of OATP1B1‐Gly488Ala on the plasma membrane compared to OATP1B1‐Val174Ala relative to reference (**Figure** [2 b](#cpt3624-fig-0002)). Accordingly, OATP1B1‐Gly488Ala cells showed a reduced uptake of estrone sulfate and less cytotoxicity to rosuvastatin (**Figure** [2 c,e](#cpt3624-fig-0002)) compared to OATP1B1‐Val174Ala. However, the uptake of estradiol‐17β‐glucuronide was not significantly different between OATP1B1‐Val174Ala and OATP1B1‐Gly488Ala (**Figure** [2 d](#cpt3624-fig-0002)).

## DISCUSSION

This investigation offers the first indication toward clinical validity for statin‐induced myopathy of variants that are specific to people with recent genealogical ancestors from Africa. Using electronic health record‐linked biobanks, we found that two functional *SLCO1B1* variants extremely rare in Eurocentrically biased study populations were strongly associated with an endogenous biomarker of OATP1B1 function, an effect of greater magnitude than our results for the well‐established c.521T>C variant within any group. The relative effect of these associations was reinforced in our cell‐based functional studies for the variants that were investigated by those assays. Finally, we generated the largest cohort to date of statin‐induced myopathy in study populations of self‐identified Black participants. Our findings rationalize the further investigation of these traditionally overlooked variants with the strong potential for future integration into pharmacogenetic testing panels.

Adverse drug reactions are a common, underappreciated cause of morbidity and mortality; two million hospitalizations each year in the United States are caused by these events.[32](#cpt3624-bib-0032) Preemptive pharmacogenetic prescribing has demonstrated efficacy in reducing the relative risk of adverse drug reactions by almost one third[5](#cpt3624-bib-0005) and is gaining wider acceptance as well as increased coverage by insurance plans.[33](#cpt3624-bib-0033), [34](#cpt3624-bib-0034)

However, there is also a growing concern among clinicians and scientists that, if not done carefully, clinical pharmacogenetics has the potential to exacerbate existing health disparities.[35](#cpt3624-bib-0035) Clinical tools developed in Eurocentrically biased study populations may exhibit lower accuracy in diverse patient groups, leading to suboptimal healthcare outcomes.[36](#cpt3624-bib-0036) A proposed solution to prevent further exacerbation is to conduct analyses in study populations with considerable genetic diversity.[37](#cpt3624-bib-0037)

For example, one study found that using a “normal” range for absolute neutrophil count (“normal” is based on results from Eurocentrically biased study populations) as a tool to guide azathioprine immunosuppressant therapy (a standard of care practice in United States health systems) leads to disproportionately higher rates of unnecessary therapeutic discontinuation in Black patients.[13](#cpt3624-bib-0013) Even after adjusting for non‐genetic factors, this disparity was attributed entirely to an *ACKR1* variant (present in >80% of Black and <0.5% of white participants, [gnomad.broadinstitute.org](http://gnomad.broadinstitute.org)), a variant associated with low (relative to “normal”) but safe neutrophil counts.[13](#cpt3624-bib-0013) Findings suggest that pharmacogenetic testing for *ACKR1* in all patients to guide azathioprine therapy could partially mitigate the existing racial disparity with immunosuppressants—which have root causes unrelated to pharmacogenetics[38](#cpt3624-bib-0038)—and prevent its exacerbation. These and many other examples[39](#cpt3624-bib-0039), [40](#cpt3624-bib-0040), [41](#cpt3624-bib-0041), [42](#cpt3624-bib-0042), [43](#cpt3624-bib-0043), [44](#cpt3624-bib-0044), [45](#cpt3624-bib-0045) demonstrate the usefulness of population descriptor‐based stratification to identify potential precision medicine‐relevant health disparity exacerbations that otherwise couldn't be identified.

The potential exacerbation of existing health disparities from precision medicine is similar to statins, a therapy where >500 million prescriptions are filled annually for Black Americans.[46](#cpt3624-bib-0046) Racial disparities in the utilization rates of statin therapy exist, an observation attributable to implicit racism.[47](#cpt3624-bib-0047) Given these observed racial disparities, the importance of developing statin precision medicine tools to prevent myotoxicity (and thereby improve utilization) equitably across diverse patient groups becomes even more critical. This is especially true considering that statins are consistently among the most commonly chosen medication classes included in clinical pharmacogenetic implementation programs[5](#cpt3624-bib-0005), [33](#cpt3624-bib-0033) and are among the most “actionable” (i.e., patients taking statins are carriers of genotypes for which guidelines would suggest a therapeutic regimen change).[5](#cpt3624-bib-0005) CPIC guidelines recommend specific statin types and doses for people with OATP1B1 decreased or poor function phenotype for whom the drug is indicated.[3](#cpt3624-bib-0003) This high strength of the recommendation may mistakenly imply an assumption that these recommendations are universally beneficial across the indicated population. However, the guidelines dictate that the OATP1B1 phenotype should be determined only by haplotypes containing c.521T>C (e.g.,**5* and **15*), which have lower allele frequency in people with recent genealogical ancestors from Africa (<0.5% prevalence in multiple subgroups of the 1,000 Genomes African superpopulation) including self‐identified Black Americans. This suggests that the currently recommended *SLCO1B1*‐based pharmacogenetic test may be less beneficial in patients from these subgroups. Our results replicated these allele frequencies, thereby validating the hypothesis that this pharmacogenetic test does not benefit groups equitably on a population‐wide scale.

In contrast to c.521T>C, there are functional *SLCO1B1* variants that are rare in Eurocentrically biased study populations. In particular, two *SLCO1B1* polymorphisms (missense variant c.1463G>C and splice donor c.481+1G>T) have mechanistic evidence for reduced transport function.[17](#cpt3624-bib-0017), [18](#cpt3624-bib-0018), [19](#cpt3624-bib-0019) These variants were discovered in study populations not Eurocentrically biased and are only common (>1% allele frequency) in people with recent genealogical ancestors from Africa. However, they have been entirely understudied as pharmacogenetic markers. Thus, not only do the current *SLCO1B1*‐based pharmacogenetic guideline recommendations have inequitable benefits across patient groups, but they also risk misguiding clinicians into believing carriers of the understudied variants have normal OATP1B1 function.

Therefore, we sought to strengthen the clinical validity of these understudied variants. Using a validated biomarker of OATP1B1 function, we observed effect sizes for these variants larger than what we observed for the c.521T>C association. Results from our cell‐based studies support these relative effect sizes. Furthermore, between the two 1KG‐AFR‐specific variants, c.481+1G>T had the strongest effect based on our bilirubin results; this is consistent with the relative functional effects observed by complete loss of function variants relative to missense polymorphisms partially reducing function.[48](#cpt3624-bib-0048) It is also important to remark that these 1KG‐AFR‐specific variants may have been considered too rare (<0.1%) for inclusion if our analyses had been conducted in a pooled study population without group stratification. Altogether, our findings provide evidence for the potential clinical relevance of these variants for drugs that are OATP1B1 substrates.

In a substantially smaller subset of our study population, we found a significant pharmacogenetic association for severe statin‐induced myopathy with one of our AFR‐specific variants but not the other. Small sample size and heterogeneity in statin type may explain these results. In fact, our prior study shows a weaker pharmacogenetic association between *SLCO1B1* and atorvastatin compared to other statin types.[25](#cpt3624-bib-0025) Atorvastatin was the most common type in the current study and our findings suggest that results were driven by this type. It is unknown if atorvastatin is more influenced by c.1463G>C compared to other *SLCO1B1* variants, though our results suggest this may be the case. Future studies require larger statin user study populations for a more robust investigation of statin type with these understudied variants.

Our findings are consistent with a case study reporting atorvastatin‐induced rhabdomyolysis in a “51‐year‐old African American male”.[49](#cpt3624-bib-0049) The patient carried no copies of the well‐established *SLCO1B1* c.521T>C polymorphism. However, he was found to carry one copy of c.481+1G>T, which was thought to be the causative factor of this adverse drug reaction. This patient was ultimately switched to a non‐statin lipid‐lowering alternative. Another clinical case describes “an adult woman of African ancestry” hospitalized for rosuvastatin‐induced rhabdomyolysis and ultimately found to carry both *SLCO1B1* c.1463G>C and c.481+1G>T.[50](#cpt3624-bib-0050) These clinical cases illustrate the potential for our novel AFR‐specific variants to prevent life‐threatening statin‐induced myotoxicity before therapy initiation if included in preemptive pharmacogenetic tests.

This study is not without limitations. First—due to the use of electronic health records as our primary data resource—this study is not designed to investigate clinical utility. Nevertheless, our findings from these invaluable real‐world data sources set the stage for the inclusion of the studied variants (as part of preemptive pharmacogenetic panels) in future randomized controlled trials. Second, bilirubin as a substrate for OATP1B1 is non‐specific to that enzyme and doesn't distinguish between conjugated versus unconjugated forms.[22](#cpt3624-bib-0022) That being said, total bilirubin can be used as a biomarker for OATP1B1 activity[23](#cpt3624-bib-0023), [24](#cpt3624-bib-0024) and validates the established *SLCO1B1* c.521T>C in the current study. Moreover, other OATP1B1 substrates such as bile acids, steroid hormones, thyroid hormones, and steroid sulfates are also non‐specific or are not routinely measured in clinical care.[22](#cpt3624-bib-0022) Third, c.481+1G>T was not interrogated in our cell‐based assay studies since our methods precluded the generation of loss‐of‐function splice variants. Fourth, the selection of two substrates with two biological replicates each for our cell‐based studies did not cover the full diversity of potential agents (e.g., docetaxel) that are substrates for OATP1B1. However, our findings are consistent with those of other studies that have examined the impact of OATP1B1‐Val174Ala and OATP1B1‐Gly488Ala.[18](#cpt3624-bib-0018), [19](#cpt3624-bib-0019)

In summary, our findings provide a strong rationale for further pharmacogenetic investigation of 1KG‐AFR‐specific *SLCO1B1* variants. The eventual incorporation of these variants into preemptive pharmacogenetic testing panels for everyone could have a significantly positive impact on the risk of severe statin‐induced myotoxicity. This is especially true for Black patients, a group historically excluded from genomic research. Most importantly, these findings set the foundation for future studies that deviate from the sole reliance on Eurocentrically biased study populations to improve pharmacological health outcomes in all more equitably.

## FUNDING

Research reported in this publication was supported by the National Human Genome Research Institute (R01HG012824: T.H., M.P.D., A.O.) and the National Institute of General Medical Sciences (R01GM117163: S.W.Y.) of the NIH as well as the OD‐AoURP (which also sponsored R01HG012824: T.H., M.P.D., A.O.).

## CONFLICT OF INTEREST

The authors declared no competing interests for this work.

## AUTHOR CONTRIBUTIONS

T.H., M.K., J.Y., M.P.D., L.K., J.S.W., C.I., M.W.M., S.W.Y., and A.O. wrote the manuscript; S.W.Y. and A.O. designed the research; T.H., M.K., J.Y., and S.W.Y. performed the research; T.H., M.K., J.Y., S.W.Y., and A.O. analyzed the data. W.C., L.K., J.S.W., R.M.K., and A.O. contributed new reagents/analytical tools.

## Supporting information

## ACKNOWLEDGMENT

The content is solely the responsibility of the authors and does not necessarily represent the official views of the National Institutes of Health or the OD‐AoURP. This study was made possible through data provided by the Kaiser Permanente Division of Research Program on Genes, Environment, and Health (RPGEH) funded by the National Institute on Aging, the National Institute of Mental Health, the National Institute of Health Office of the Director, the Robert Wood Johnson Foundation, and the Kaiser Permanente Community Benefits Program. We would like to express our gratitude to the members of the All of Us Evenings with Genetics Research Team at Baylor College of Medicine, particularly Drs. J. Wang and S.E. Scherer, for their valuable contributions and support throughout this research endeavor.

## Contributor Information

Sook Wah Yee, Email: sookwah.yee@gmail.com.

Akinyemi Oni‐Orisan, Email: akinyemi.oni-orisan@ucsf.edu.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Vinci, P. et al. Statin‐associated myopathy: emphasis on mechanisms and targeted therapy. Int. J. Mol. Sci. 22, 11687 (2021).  [DOI](https://doi.org/10.3390/ijms222111687) | [PMC free article](/articles/PMC8583847/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34769118/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Mol.%20Sci.&title=Statin%E2%80%90associated%20myopathy:%20emphasis%20on%20mechanisms%20and%20targeted%20therapy&author=P.%20Vinci&volume=22&publication_year=2021&pages=11687&pmid=34769118&doi=10.3390/ijms222111687&)

2. SEARCH Collaborative Group et al. SLCO1B1 variants and statin‐induced myopathy – a genomewide study. N. Engl. J. Med. 359, 789–799 (2008).  [DOI](https://doi.org/10.1056/NEJMoa0801936) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18650507/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%20Engl.%20J.%20Med.&title=SLCO1B1%20variants%20and%20statin%E2%80%90induced%20myopathy%20%E2%80%93%20a%20genomewide%20study&volume=359&publication_year=2008&pages=789-799&pmid=18650507&doi=10.1056/NEJMoa0801936&)

3. Cooper‐DeHoff, R.M. et al. The clinical pharmacogenetics implementation consortium guideline for SLCO1B1, ABCG2, and CYP2C9 genotypes and statin‐associated musculoskeletal symptoms. Clin. Pharmacol. Ther. 111, 1007–1021 (2022).  [DOI](https://doi.org/10.1002/cpt.2557) | [PMC free article](/articles/PMC9035072/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35152405/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=The%20clinical%20pharmacogenetics%20implementation%20consortium%20guideline%20for%20SLCO1B1,%20ABCG2,%20and%20CYP2C9%20genotypes%20and%20statin%E2%80%90associated%20musculoskeletal%20symptoms&author=R.M.%20Cooper%E2%80%90DeHoff&volume=111&publication_year=2022&pages=1007-1021&pmid=35152405&doi=10.1002/cpt.2557&)

4. Oni‐Orisan, A. et al. An introductory tutorial on cardiovascular pharmacogenetics for healthcare providers. Clin. Pharmacol. Ther. 114, 275–287 (2023).  [DOI](https://doi.org/10.1002/cpt.2957) | [PMC free article](/articles/PMC10406163/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/37303270/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=An%20introductory%20tutorial%20on%20cardiovascular%20pharmacogenetics%20for%20healthcare%20providers&author=A.%20Oni%E2%80%90Orisan&volume=114&publication_year=2023&pages=275-287&pmid=37303270&doi=10.1002/cpt.2957&)

5. Swen, J.J. et al. A 12‐gene pharmacogenetic panel to prevent adverse drug reactions: an open‐label, multicentre, controlled, cluster‐randomised crossover implementation study. Lancet 401, 347–356 (2023).  [DOI](https://doi.org/10.1016/S0140-6736(22)01841-4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36739136/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=A%2012%E2%80%90gene%20pharmacogenetic%20panel%20to%20prevent%20adverse%20drug%20reactions:%20an%20open%E2%80%90label,%20multicentre,%20controlled,%20cluster%E2%80%90randomised%20crossover%20implementation%20study&author=J.J.%20Swen&volume=401&publication_year=2023&pages=347-356&pmid=36739136&doi=10.1016/S0140-6736(22)01841-4&)

6. Carr, D.F. et al. Genomewide association study of statin‐induced myopathy in patients recruited using the UK clinical practice research datalink. Clin. Pharmacol. Ther. 106, 1353–1361 (2019).  [DOI](https://doi.org/10.1002/cpt.1557) | [PMC free article](/articles/PMC6896237/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31220337/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Genomewide%20association%20study%20of%20statin%E2%80%90induced%20myopathy%20in%20patients%20recruited%20using%20the%20UK%20clinical%20practice%20research%20datalink&author=D.F.%20Carr&volume=106&publication_year=2019&pages=1353-1361&pmid=31220337&doi=10.1002/cpt.1557&)

7. Oni‐Orisan, A. et al. Polygenic risk score and statin relative risk reduction for primary prevention of myocardial infarction in a real‐world population. Clin. Pharmacol. Ther. 112, 1070–1078 (2022).  [DOI](https://doi.org/10.1002/cpt.2715) | [PMC free article](/articles/PMC10112337/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35862449/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Polygenic%20risk%20score%20and%20statin%20relative%20risk%20reduction%20for%20primary%20prevention%20of%20myocardial%20infarction%20in%20a%20real%E2%80%90world%20population&author=A.%20Oni%E2%80%90Orisan&volume=112&publication_year=2022&pages=1070-1078&pmid=35862449&doi=10.1002/cpt.2715&)

8. Fry, A. et al. Comparison of sociodemographic and health‐related characteristics of UK biobank participants with those of the general population. Am. J. Epidemiol. 186, 1026–1034 (2017).  [DOI](https://doi.org/10.1093/aje/kwx246) | [PMC free article](/articles/PMC5860371/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28641372/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Epidemiol.&title=Comparison%20of%20sociodemographic%20and%20health%E2%80%90related%20characteristics%20of%20UK%20biobank%20participants%20with%20those%20of%20the%20general%20population&author=A.%20Fry&volume=186&publication_year=2017&pages=1026-1034&pmid=28641372&doi=10.1093/aje/kwx246&)

9. All of us research program genomics investigators genomic data in the all of us research program. Nature 627, 340–346 (2024).  [DOI](https://doi.org/10.1038/s41586-023-06957-x) | [PMC free article](/articles/PMC10937371/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/38374255/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=All%20of%20us%20research%20program%20genomics%20investigators%20genomic%20data%20in%20the%20all%20of%20us%20research%20program&volume=627&publication_year=2024&pages=340-346&pmid=38374255&doi=10.1038/s41586-023-06957-x&)

10. Alvidrez, J. , Castille, D. , Laude‐Sharp, M. , Rosario, A. & Tabor, D. The national institute on minority health and health disparities research framework. Am. J. Public Health 109, S16–S20 (2019).  [DOI](https://doi.org/10.2105/AJPH.2018.304883) | [PMC free article](/articles/PMC6356129/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30699025/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Public%20Health&title=The%20national%20institute%20on%20minority%20health%20and%20health%20disparities%20research%20framework&author=J.%20Alvidrez&author=D.%20Castille&author=M.%20Laude%E2%80%90Sharp&author=A.%20Rosario&author=D.%20Tabor&volume=109&publication_year=2019&pages=S16-S20&pmid=30699025&doi=10.2105/AJPH.2018.304883&)

11. Braveman, P. What are health disparities and health equity? We need to be clear. Public Health Rep. 129(2), 5–8 (2014).  [DOI](https://doi.org/10.1177/00333549141291S203) | [PMC free article](/articles/PMC3863701/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24385658/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Public%20Health%20Rep.&title=What%20are%20health%20disparities%20and%20health%20equity?%20We%20need%20to%20be%20clear&author=P.%20Braveman&volume=129&issue=2&publication_year=2014&pages=5-8&pmid=24385658&doi=10.1177/00333549141291S203&)

12. National Academies of Sciences, Engineering, and Medicine , Division of Behavioral and Social Sciences and Education , Health and Medicine Division , Committee on Population , Board on Health Sciences Policy & Committee on the Use of Race, Ethnicity, and Ancestry as Population Descriptors in Genomics Research . Using Population Descriptors in Genetics and Genomics Research: A New Framework for an Evolving Field (National Academies Press (US), Washington (DC), 2023). 10.17226/26902.  [DOI](https://doi.org/10.17226/26902) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36989389/) | [Google Scholar](https://scholar.google.com/scholar_lookup?title=Using%20Population%20Descriptors%20in%20Genetics%20and%20Genomics%20Research:%20A%20New%20Framework%20for%20an%20Evolving%20Field&publication_year=2023&)

13. Dickson, A.L. et al. Race, genotype, and azathioprine discontinuation: a cohort study. Ann. Intern. Med. 175, 1092–1099 (2022).  [DOI](https://doi.org/10.7326/M21-4675) | [PMC free article](/articles/PMC9378477/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35724382/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann.%20Intern.%20Med.&title=Race,%20genotype,%20and%20azathioprine%20discontinuation:%20a%20cohort%20study&author=A.L.%20Dickson&volume=175&publication_year=2022&pages=1092-1099&pmid=35724382&doi=10.7326/M21-4675&)

14. Bycroft, C. et al. The UK biobank resource with deep phenotyping and genomic data. Nature 562, 203–209 (2018).  [DOI](https://doi.org/10.1038/s41586-018-0579-z) | [PMC free article](/articles/PMC6786975/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30305743/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=The%20UK%20biobank%20resource%20with%20deep%20phenotyping%20and%20genomic%20data&author=C.%20Bycroft&volume=562&publication_year=2018&pages=203-209&pmid=30305743&doi=10.1038/s41586-018-0579-z&)

15. Conley, A.B. et al. Rye: genetic ancestry inference at biobank scale. Nucleic Acids Res. 51, e44 (2023).  [DOI](https://doi.org/10.1093/nar/gkad149) | [PMC free article](/articles/PMC10164567/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36928108/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&title=Rye:%20genetic%20ancestry%20inference%20at%20biobank%20scale&author=A.B.%20Conley&volume=51&publication_year=2023&pages=e44&pmid=36928108&doi=10.1093/nar/gkad149&)

16. Banda, Y. et al. Characterizing race/ethnicity and genetic ancestry for 100,000 subjects in the genetic epidemiology research on adult health and aging (GERA) cohort. Genetics 200, 1285–1295 (2015).  [DOI](https://doi.org/10.1534/genetics.115.178616) | [PMC free article](/articles/PMC4574246/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26092716/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genetics&title=Characterizing%20race/ethnicity%20and%20genetic%20ancestry%20for%20100,000%20subjects%20in%20the%20genetic%20epidemiology%20research%20on%20adult%20health%20and%20aging%20(GERA)%20cohort&author=Y.%20Banda&volume=200&publication_year=2015&pages=1285-1295&pmid=26092716&doi=10.1534/genetics.115.178616&)

17. van de Steeg, E. et al. Complete OATP1B1 and OATP1B3 deficiency causes human rotor syndrome by interrupting conjugated bilirubin reuptake into the liver. J. Clin. Invest. 122, 519–528 (2012).  [DOI](https://doi.org/10.1172/JCI59526) | [PMC free article](/articles/PMC3266790/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22232210/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Invest.&title=Complete%20OATP1B1%20and%20OATP1B3%20deficiency%20causes%20human%20rotor%20syndrome%20by%20interrupting%20conjugated%20bilirubin%20reuptake%20into%20the%20liver&author=E.%20van%20de%20Steeg&volume=122&publication_year=2012&pages=519-528&pmid=22232210&doi=10.1172/JCI59526&)

18. Tirona, R.G. , Leake, B.F. , Merino, G. & Kim, R.B. Polymorphisms in OATP‐C: identification of multiple allelic variants associated with altered transport activity among European‐ and African‐Americans. J. Biol. Chem. 276, 35669–35675 (2001).  [DOI](https://doi.org/10.1074/jbc.M103792200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11477075/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem.&title=Polymorphisms%20in%20OATP%E2%80%90C:%20identification%20of%20multiple%20allelic%20variants%20associated%20with%20altered%20transport%20activity%20among%20European%E2%80%90%20and%20African%E2%80%90Americans&author=R.G.%20Tirona&author=B.F.%20Leake&author=G.%20Merino&author=R.B.%20Kim&volume=276&publication_year=2001&pages=35669-35675&pmid=11477075&doi=10.1074/jbc.M103792200&)

19. Ho, R.H. et al. Drug and bile acid transporters in rosuvastatin hepatic uptake: function, expression, and pharmacogenetics. Gastroenterology 130, 1793–1806 (2006).  [DOI](https://doi.org/10.1053/j.gastro.2006.02.034) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16697742/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastroenterology&title=Drug%20and%20bile%20acid%20transporters%20in%20rosuvastatin%20hepatic%20uptake:%20function,%20expression,%20and%20pharmacogenetics&author=R.H.%20Ho&volume=130&publication_year=2006&pages=1793-1806&pmid=16697742&doi=10.1053/j.gastro.2006.02.034&)

20. Yu, B. et al. Loss‐of‐function variants influence the human serum metabolome. Sci. Adv. 2, e1600800 (2016).  [DOI](https://doi.org/10.1126/sciadv.1600800) | [PMC free article](/articles/PMC5007069/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27602404/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sci.%20Adv.&title=Loss%E2%80%90of%E2%80%90function%20variants%20influence%20the%20human%20serum%20metabolome&author=B.%20Yu&volume=2&publication_year=2016&pages=e1600800&pmid=27602404&doi=10.1126/sciadv.1600800&)

21. Ramsey, L.B. et al. The clinical pharmacogenetics implementation consortium guideline for SLCO1B1 and simvastatin‐induced myopathy: 2014 update. Clin. Pharmacol. Ther. 96, 423–428 (2014).  [DOI](https://doi.org/10.1038/clpt.2014.125) | [PMC free article](/articles/PMC4169720/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24918167/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=The%20clinical%20pharmacogenetics%20implementation%20consortium%20guideline%20for%20SLCO1B1%20and%20simvastatin%E2%80%90induced%20myopathy:%202014%20update&author=L.B.%20Ramsey&volume=96&publication_year=2014&pages=423-428&pmid=24918167&doi=10.1038/clpt.2014.125&)

22. Chu, X. et al. Clinical probes and endogenous biomarkers as substrates for transporter drug‐drug interaction evaluation: perspectives from the international transporter consortium. Clin. Pharmacol. Ther. 104, 836–864 (2018).  [DOI](https://doi.org/10.1002/cpt.1216) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30347454/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Clinical%20probes%20and%20endogenous%20biomarkers%20as%20substrates%20for%20transporter%20drug%E2%80%90drug%20interaction%20evaluation:%20perspectives%20from%20the%20international%20transporter%20consortium&author=X.%20Chu&volume=104&publication_year=2018&pages=836-864&pmid=30347454&doi=10.1002/cpt.1216&)

23. Johnson, A.D. et al. Genome‐wide association meta‐analysis for total serum bilirubin levels. Hum. Mol. Genet. 18, 2700–2710 (2009).  [DOI](https://doi.org/10.1093/hmg/ddp202) | [PMC free article](/articles/PMC2701336/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19414484/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum.%20Mol.%20Genet.&title=Genome%E2%80%90wide%20association%20meta%E2%80%90analysis%20for%20total%20serum%20bilirubin%20levels&author=A.D.%20Johnson&volume=18&publication_year=2009&pages=2700-2710&pmid=19414484&doi=10.1093/hmg/ddp202&)

24. Sinnott‐Armstrong, N. et al. Genetics of 35 blood and urine biomarkers in the UK biobank. Nat. Genet. 53, 185–194 (2021).  [DOI](https://doi.org/10.1038/s41588-020-00757-z) | [PMC free article](/articles/PMC7867639/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33462484/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet.&title=Genetics%20of%2035%20blood%20and%20urine%20biomarkers%20in%20the%20UK%20biobank&author=N.%20Sinnott%E2%80%90Armstrong&volume=53&publication_year=2021&pages=185-194&pmid=33462484&doi=10.1038/s41588-020-00757-z&)

25. Lu, B. et al. Effect of SLCO1B1 T521C on statin‐related myotoxicity with use of lovastatin and atorvastatin. Clin. Pharmacol. Ther. 110, 733–740 (2021).  [DOI](https://doi.org/10.1002/cpt.2337) | [PMC free article](/articles/PMC8376784/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34114646/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Effect%20of%20SLCO1B1%20T521C%20on%20statin%E2%80%90related%20myotoxicity%20with%20use%20of%20lovastatin%20and%20atorvastatin&author=B.%20Lu&volume=110&publication_year=2021&pages=733-740&pmid=34114646&doi=10.1002/cpt.2337&)

26. Alfirevic, A. et al. Phenotype standardization for statin‐induced myotoxicity. Clin. Pharmacol. Ther. 96, 470–476 (2014).  [DOI](https://doi.org/10.1038/clpt.2014.121) | [PMC free article](/articles/PMC4172546/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24897241/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Phenotype%20standardization%20for%20statin%E2%80%90induced%20myotoxicity&author=A.%20Alfirevic&volume=96&publication_year=2014&pages=470-476&pmid=24897241&doi=10.1038/clpt.2014.121&)

27. Oni‐Orisan, A. et al. Development and application of an algorithm for statin‐induced myopathy based on electronic health record‐derived structured elements. medRxiv (2023). 10.1101/2023.04.24.23289059.  [DOI](https://doi.org/10.1101/2023.04.24.23289059) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=medRxiv&title=Development%20and%20application%20of%20an%20algorithm%20for%20statin%E2%80%90induced%20myopathy%20based%20on%20electronic%20health%20record%E2%80%90derived%20structured%20elements&author=A.%20Oni%E2%80%90Orisan&publication_year=2023&doi=10.1101/2023.04.24.23289059&)

28. Willer, C.J. , Li, Y. & Abecasis, G.R. METAL: fast and efficient meta‐analysis of genomewide association scans. Bioinformatics 26, 2190–2191 (2010).  [DOI](https://doi.org/10.1093/bioinformatics/btq340) | [PMC free article](/articles/PMC2922887/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20616382/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=METAL:%20fast%20and%20efficient%20meta%E2%80%90analysis%20of%20genomewide%20association%20scans&author=C.J.%20Willer&author=Y.%20Li&author=G.R.%20Abecasis&volume=26&publication_year=2010&pages=2190-2191&pmid=20616382&doi=10.1093/bioinformatics/btq340&)

29. Kanters, S. Fixed‐ and random‐effects models. Methods Mol. Biol. 2345, 41–65 (2022).  [DOI](https://doi.org/10.1007/978-1-0716-1566-9_3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34550583/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Methods%20Mol.%20Biol.&title=Fixed%E2%80%90%20and%20random%E2%80%90effects%20models&author=S.%20Kanters&volume=2345&publication_year=2022&pages=41-65&pmid=34550583&doi=10.1007/978-1-0716-1566-9_3&)

30. von Elm, E. et al. The strengthening the reporting of observational studies in epidemiology (STROBE) statement: guidelines for reporting observational studies. Ann. Intern. Med. 147, 573–577 (2007).  [DOI](https://doi.org/10.7326/0003-4819-147-8-200710160-00010) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17938396/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann.%20Intern.%20Med.&title=The%20strengthening%20the%20reporting%20of%20observational%20studies%20in%20epidemiology%20(STROBE)%20statement:%20guidelines%20for%20reporting%20observational%20studies&author=E.%20von%20Elm&volume=147&publication_year=2007&pages=573-577&pmid=17938396&doi=10.7326/0003-4819-147-8-200710160-00010&)

31. Yee, S.W. et al. The full spectrum of SLC22 OCT1 mutations illuminates the bridge between drug transporter biophysics and pharmacogenomics. Mol Cell. 84, 1932–1947(2023).  [DOI](https://doi.org/10.1016/j.molcel.2024.04.008) | [PMC free article](/articles/PMC11382353/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/38703769/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Cell.&title=The%20full%20spectrum%20of%20SLC22%20OCT1%20mutations%20illuminates%20the%20bridge%20between%20drug%20transporter%20biophysics%20and%20pharmacogenomics&author=S.W.%20Yee&volume=84&publication_year=2023&pages=1932-1947&pmid=38703769&doi=10.1016/j.molcel.2024.04.008&)

32. Lazarou, J. , Pomeranz, B.H. & Corey, P.N. Incidence of adverse drug reactions in hospitalized patients: a meta‐analysis of prospective studies. JAMA 279, 1200–1205 (1998).  [DOI](https://doi.org/10.1001/jama.279.15.1200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9555760/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA&title=Incidence%20of%20adverse%20drug%20reactions%20in%20hospitalized%20patients:%20a%20meta%E2%80%90analysis%20of%20prospective%20studies&author=J.%20Lazarou&author=B.H.%20Pomeranz&author=P.N.%20Corey&volume=279&publication_year=1998&pages=1200-1205&pmid=9555760&doi=10.1001/jama.279.15.1200&)

33. Luzum, J.A. et al. The pharmacogenomics research network translational pharmacogenetics program: outcomes and metrics of pharmacogenetic implementations across diverse healthcare systems. Clin. Pharmacol. Ther. 102, 502–510 (2017).  [DOI](https://doi.org/10.1002/cpt.630) | [PMC free article](/articles/PMC5511786/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28090649/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=The%20pharmacogenomics%20research%20network%20translational%20pharmacogenetics%20program:%20outcomes%20and%20metrics%20of%20pharmacogenetic%20implementations%20across%20diverse%20healthcare%20systems&author=J.A.%20Luzum&volume=102&publication_year=2017&pages=502-510&pmid=28090649&doi=10.1002/cpt.630&)

34. Jarvis, J.P. et al. Real‐world impact of a pharmacogenomics‐enriched comprehensive medication management program. J. Pers. Med. 12, 421(2022).  [DOI](https://doi.org/10.3390/jpm12030421) | [PMC free article](/articles/PMC8949247/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35330421/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Pers.%20Med.&title=Real%E2%80%90world%20impact%20of%20a%20pharmacogenomics%E2%80%90enriched%20comprehensive%20medication%20management%20program&author=J.P.%20Jarvis&volume=12&publication_year=2022&pages=421&pmid=35330421&doi=10.3390/jpm12030421&)

35. Oni‐Orisan, A. , Mavura, Y. , Banda, Y. , Thornton, T.A. & Sebro, R. Embracing genetic diversity to improve black health. N. Engl. J. Med. 384, 1163–1167 (2021).  [DOI](https://doi.org/10.1056/NEJMms2031080) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33567186/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%20Engl.%20J.%20Med.&title=Embracing%20genetic%20diversity%20to%20improve%20black%20health&author=A.%20Oni%E2%80%90Orisan&author=Y.%20Mavura&author=Y.%20Banda&author=T.A.%20Thornton&author=R.%20Sebro&volume=384&publication_year=2021&pages=1163-1167&pmid=33567186&doi=10.1056/NEJMms2031080&)

36. Martin, A.R. et al. Clinical use of current polygenic risk scores may exacerbate health disparities. Nat. Genet. 51, 584–591 (2019).  [DOI](https://doi.org/10.1038/s41588-019-0379-x) | [PMC free article](/articles/PMC6563838/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30926966/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet.&title=Clinical%20use%20of%20current%20polygenic%20risk%20scores%20may%20exacerbate%20health%20disparities&author=A.R.%20Martin&volume=51&publication_year=2019&pages=584-591&pmid=30926966&doi=10.1038/s41588-019-0379-x&)

37. Kehinde, O. , Ramsey, L.B. , Gaedigk, A. & Oni‐Orisan, A. Advancing CYP2D6 pharmacogenetics through a Pharmacoequity lens. Clin. Pharmacol. Ther. 114, 69–76 (2023).  [DOI](https://doi.org/10.1002/cpt.2890) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36924260/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Advancing%20CYP2D6%20pharmacogenetics%20through%20a%20Pharmacoequity%20lens&author=O.%20Kehinde&author=L.B.%20Ramsey&author=A.%20Gaedigk&author=A.%20Oni%E2%80%90Orisan&volume=114&publication_year=2023&pages=69-76&pmid=36924260&doi=10.1002/cpt.2890&)

38. Nguyen, G.C. , LaVeist, T.A. , Harris, M.L. , Wang, M.H. , Datta, L.W. & Brant, S.R. Racial disparities in utilization of specialist care and medications in inflammatory bowel disease. Am. J. Gastroenterol. 105, 2202–2208 (2010).  [DOI](https://doi.org/10.1038/ajg.2010.202) | [PMC free article](/articles/PMC3170037/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20485281/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Gastroenterol.&title=Racial%20disparities%20in%20utilization%20of%20specialist%20care%20and%20medications%20in%20inflammatory%20bowel%20disease&author=G.C.%20Nguyen&author=T.A.%20LaVeist&author=M.L.%20Harris&author=M.H.%20Wang&author=L.W.%20Datta&volume=105&publication_year=2010&pages=2202-2208&pmid=20485281&doi=10.1038/ajg.2010.202&)

39. Pardiñas, A.F. et al. Pharmacokinetics and pharmacogenomics of clozapine in an ancestrally diverse sample: a longitudinal analysis and genome‐wide association study using UK clinical monitoring data. Lancet Psychiatry 10, 209–219 (2023).  [DOI](https://doi.org/10.1016/S2215-0366(23)00002-0) | [PMC free article](/articles/PMC10824469/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36804072/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet%20Psychiatry&title=Pharmacokinetics%20and%20pharmacogenomics%20of%20clozapine%20in%20an%20ancestrally%20diverse%20sample:%20a%20longitudinal%20analysis%20and%20genome%E2%80%90wide%20association%20study%20using%20UK%20clinical%20monitoring%20data&author=A.F.%20Pardi%C3%B1as&volume=10&publication_year=2023&pages=209-219&pmid=36804072&doi=10.1016/S2215-0366(23)00002-0&)

40. Wu, A.H. , White, M.J. , Oh, S. & Burchard, E. The Hawaii clopidogrel lawsuit: the possible effect on clinical laboratory testing. Pers. Med. 12, 179–181 (2015).  [DOI](https://doi.org/10.2217/pme.15.4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29771642/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pers.%20Med.&title=The%20Hawaii%20clopidogrel%20lawsuit:%20the%20possible%20effect%20on%20clinical%20laboratory%20testing&author=A.H.%20Wu&author=M.J.%20White&author=S.%20Oh&author=E.%20Burchard&volume=12&publication_year=2015&pages=179-181&pmid=29771642&doi=10.2217/pme.15.4&)

41. Hoshitsuki, K. et al. Challenges in clinical implementation of CYP2D6 genotyping: choice of variants to test affects phenotype determination. Genet. Med. 22, 232–233 (2020).  [DOI](https://doi.org/10.1038/s41436-019-0614-y) | [PMC free article](/articles/PMC7702299/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31341243/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genet.%20Med.&title=Challenges%20in%20clinical%20implementation%20of%20CYP2D6%20genotyping:%20choice%20of%20variants%20to%20test%20affects%20phenotype%20determination&author=K.%20Hoshitsuki&volume=22&publication_year=2020&pages=232-233&pmid=31341243&doi=10.1038/s41436-019-0614-y&)

42. Jordan, I.K. , Sharma, S. & Mariño‐Ramírez, L. Population pharmacogenomics for health equity. Gen 14(10), 1840 (2023).  [DOI](https://doi.org/10.3390/genes14101840) | [PMC free article](/articles/PMC10606908/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/37895188/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gen&title=Population%20pharmacogenomics%20for%20health%20equity&author=I.K.%20Jordan&author=S.%20Sharma&author=L.%20Mari%C3%B1o%E2%80%90Ram%C3%ADrez&volume=14&issue=10&publication_year=2023&pages=1840&pmid=37895188&doi=10.3390/genes14101840&)

43. Kimmel, S.E. et al. A pharmacogenetic versus a clinical algorithm for warfarin dosing. N. Engl. J. Med. 369, 2283–2293 (2013).  [DOI](https://doi.org/10.1056/NEJMoa1310669) | [PMC free article](/articles/PMC3942158/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24251361/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%20Engl.%20J.%20Med.&title=A%20pharmacogenetic%20versus%20a%20clinical%20algorithm%20for%20warfarin%20dosing&author=S.E.%20Kimmel&volume=369&publication_year=2013&pages=2283-2293&pmid=24251361&doi=10.1056/NEJMoa1310669&)

44. Lindley, K.J. et al. Warfarin dosing in patients with CYP2C9*5 variant alleles. Clin. Pharmacol. Ther. 111, 950–955 (2022).  [DOI](https://doi.org/10.1002/cpt.2549) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35108398/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Warfarin%20dosing%20in%20patients%20with%20CYP2C9*5%20variant%20alleles&author=K.J.%20Lindley&volume=111&publication_year=2022&pages=950-955&pmid=35108398&doi=10.1002/cpt.2549&)

45. Cavallari, L.H. et al. Genetic and clinical predictors of warfarin dose requirements in African Americans. Clin. Pharmacol. Ther. 87, 459–464 (2010).  [DOI](https://doi.org/10.1038/clpt.2009.223) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20072124/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Genetic%20and%20clinical%20predictors%20of%20warfarin%20dose%20requirements%20in%20African%20Americans&author=L.H.%20Cavallari&volume=87&publication_year=2010&pages=459-464&pmid=20072124&doi=10.1038/clpt.2009.223&)

46. Van Driest, S.L. et al. Clinically actionable genotypes among 10,000 patients with preemptive pharmacogenomic testing. Clin. Pharmacol. Ther. 95, 423–431 (2014).  [DOI](https://doi.org/10.1038/clpt.2013.229) | [PMC free article](/articles/PMC3961508/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24253661/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Clinically%20actionable%20genotypes%20among%2010,000%20patients%20with%20preemptive%20pharmacogenomic%20testing&author=S.L.%20Van%20Driest&volume=95&publication_year=2014&pages=423-431&pmid=24253661&doi=10.1038/clpt.2013.229&)

47. Frank, D.A. , Johnson, A.E. , Hausmann, L.R.M. , Gellad, W.F. , Roberts, E.T. & Vajravelu, R.K. Disparities in guideline‐recommended statin use for prevention of atherosclerotic cardiovascular disease by race, ethnicity, and gender: a nationally representative cross‐sectional analysis of adults in the United States. Ann. Intern. Med. 176, 1057–1066 (2023).  [DOI](https://doi.org/10.7326/M23-0720) | [PMC free article](/articles/PMC10804313/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/37487210/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann.%20Intern.%20Med.&title=Disparities%20in%20guideline%E2%80%90recommended%20statin%20use%20for%20prevention%20of%20atherosclerotic%20cardiovascular%20disease%20by%20race,%20ethnicity,%20and%20gender:%20a%20nationally%20representative%20cross%E2%80%90sectional%20analysis%20of%20adults%20in%20the%20United%20States&author=D.A.%20Frank&author=A.E.%20Johnson&author=L.R.M.%20Hausmann&author=W.F.%20Gellad&author=E.T.%20Roberts&volume=176&publication_year=2023&pages=1057-1066&pmid=37487210&doi=10.7326/M23-0720&)

48. Gerasimavicius, L. , Livesey, B.J. & Marsh, J.A. Loss‐of‐function, gain‐of‐function and dominant‐negative mutations have profoundly different effects on protein structure. Nat. Commun. 13, 3895 (2022).  [DOI](https://doi.org/10.1038/s41467-022-31686-6) | [PMC free article](/articles/PMC9259657/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35794153/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&title=Loss%E2%80%90of%E2%80%90function,%20gain%E2%80%90of%E2%80%90function%20and%20dominant%E2%80%90negative%20mutations%20have%20profoundly%20different%20effects%20on%20protein%20structure&author=L.%20Gerasimavicius&author=B.J.%20Livesey&author=J.A.%20Marsh&volume=13&publication_year=2022&pages=3895&pmid=35794153&doi=10.1038/s41467-022-31686-6&)

49. Kiage, J. , Venkatanarayan, A. , Roth, M. & Elam, M. Atorvastatin‐associated rhabdomyolysis in a patient with a novel variant of the SLCO1B1 gene: a case report. J. Clin. Lipidol. 16, 23–27 (2022).  [DOI](https://doi.org/10.1016/j.jacl.2021.11.007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34887219/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Lipidol.&title=Atorvastatin%E2%80%90associated%20rhabdomyolysis%20in%20a%20patient%20with%20a%20novel%20variant%20of%20the%20SLCO1B1%20gene:%20a%20case%20report&author=J.%20Kiage&author=A.%20Venkatanarayan&author=M.%20Roth&author=M.%20Elam&volume=16&publication_year=2022&pages=23-27&pmid=34887219&doi=10.1016/j.jacl.2021.11.007&)

50. Medwid, S. , Deckert, R. , Gryn, S.E. & Kim, R.B. SLCO1B1 variants in a patient of African ancestry presenting with rosuvastatin‐induced rhabdomyolysis: a case report. Br. J. Clin. Pharmacol. 91, 232–235 (2025).  [DOI](https://doi.org/10.1111/bcp.16329) | [PMC free article](/articles/PMC11671340/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/39511784/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Clin.%20Pharmacol.&title=SLCO1B1%20variants%20in%20a%20patient%20of%20African%20ancestry%20presenting%20with%20rosuvastatin%E2%80%90induced%20rhabdomyolysis:%20a%20case%20report&author=S.%20Medwid&author=R.%20Deckert&author=S.E.%20Gryn&author=R.B.%20Kim&volume=91&publication_year=2025&pages=232-235&pmid=39511784&doi=10.1111/bcp.16329&)
