# Metabolic activity of CYP2C19 and CYP2D6 on antidepressant response from 13 clinical studies using genotype imputation: a meta-analysis

## Metadata
**Authors:** Danyang Li, Oliver Pain, Chiara Fabbri, Win Lee Edwin Wong, Chris Wai Hang Lo, Stephan Ripke, Annamaria Cattaneo, Daniel Souery, Mojca Z Dernovsek, Neven Henigsberg, Joanna Hauser, Glyn Lewis, Ole Mors, Nader Perroud, Marcella Rietschel, Rudolf Uher, Wolfgang Maier, Bernhard T Baune, Joanna M Biernacka, Guido Bondolfi, Katharina Domschke, Masaki Kato, Yu-Li Liu, Alessandro Serretti, Shih-Jen Tsai, Richard Weinshilboum, the GSRD Consortium, the Major Depressive Disorder Working Group of the Psychiatric Genomics Consortium, Andrew M McIntosh, Cathryn M Lewis
**Journal:** Translational Psychiatry
**Date:** 2024 Jul 19
**DOI:** [10.1038/s41398-024-02981-1](https://doi.org/10.1038/s41398-024-02981-1)
**PMID:** 39025838
**PMCID:** PMC11258238
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11258238/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC11258238/pdf/41398_2024_Article_2981.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC11258238/pdf/41398_2024_Article_2981.pdf)

## Abstract

Cytochrome P450 enzymes including CYP2C19 and CYP2D6 are important for antidepressant metabolism and polymorphisms of these genes have been determined to predict metabolite levels. Nonetheless, more evidence is needed to understand the impact of genetic variations on antidepressant response. In this study, individual clinical and genetic data from 13 studies of European and East Asian ancestry populations were collected. The antidepressant response was clinically assessed as remission and percentage improvement. Imputed genotype was used to translate genetic polymorphisms to metabolic phenotypes (poor, intermediate, normal, and rapid+ultrarapid) of CYP2C19 and CYP2D6. CYP2D6 structural variants cannot be imputed from genotype data, limiting the determination of metabolic phenotypes, and precluding testing for association with response. The association of CYP2C19 metabolic phenotypes with treatment response was examined using normal metabolizers as the reference. Among 5843 depression patients, a higher remission rate was found in CYP2C19 poor metabolizers compared to normal metabolizers at nominal significance but did not survive after multiple testing correction (OR = 1.46, 95% CI [1.03, 2.06], p = 0.033, heterogeneity I2 = 0%, subgroup difference p = 0.72). No metabolic phenotype was associated with percentage improvement from baseline. After stratifying by antidepressants primarily metabolized by CYP2C19, no association was found between metabolic phenotypes and antidepressant response. Metabolic phenotypes showed differences in frequency, but not effect, between European- and East Asian-ancestry studies. In conclusion, metabolic phenotypes imputed from genetic variants using genotype were not associated with antidepressant response. CYP2C19 poor metabolizers could potentially contribute to antidepressant efficacy with more evidence needed. Sequencing and targeted pharmacogenetic testing, alongside information on side effects, antidepressant dosage, depression measures, and diverse ancestry studies, would more fully capture the influence of metabolic phenotypes.

Subject terms: Depression, Predictive markers

## Introduction

Antidepressants are the first-line treatment for moderate or severe depression, however efficacy varies, and side effects are common [[1](#CR1)]. Approximately 35% of patients reach remission after treatment with a single antidepressant but a substantial proportion require further treatment, with many developing treatment-resistant depression [[2](#CR2)–[5](#CR5)]. Even within the same antidepressant class, treatment responses vary substantially. For example, selective serotonin reuptake inhibitors (SSRIs), the most widely prescribed antidepressants, could lead to remission in 30–45% of patients [[6](#CR6)]. Differences in response rate may be due to many factors including drug-drug interactions [[7](#CR7)], depression subtypes [[8](#CR8), [9](#CR9)], comorbidity [[10](#CR10)], smoking [[11](#CR11)], and genetic variation, particularly in drug metabolism genes.

Pharmacogenetics utilizes genetic variation that plays a role in medication action and metabolism to facilitate individualized prescription, thus improving the treatment efficacy, and reducing undesirable effects [[12](#CR12)]. In antidepressants, current evidence and prescribing guidelines support cytochrome P450 (CYP) genes for pharmacogenetic testing, in which *CYP2C19* and *CYP2D6* have been widely examined for drug efficacy and side effects [[12](#CR12)–[15](#CR15)]. Both *CYP2C19* and *CYP2D6* are highly polymorphic, with genetic haplotypes defined by the star allele nomenclature [[16](#CR16)]. These star alleles can be classified into different metabolic phenotypes, such as poor metabolizers (PMs), intermediate metabolizers (IMs), normal metabolizers (NMs), rapid and ultrarapid metabolizers (RMs/UMs) according to Clinical Pharmacogenetics Implementation Consortium (CPIC) guidelines [[14](#CR14), [15](#CR15)]. Compared to NMs, PMs and IMs have an increased risk of adverse effects because of a lower metabolism rate and elevated drug serum concentrations, which may also increase treatment efficacy. RMs and UMs, on the other hand, facilitate the metabolic process to reduce drug exposure and may lead to treatment failure through a lack of efficacy.

Clinical studies have shown that genetic variation in these metabolizing enzymes is clearly associated with metabolite levels, but the link between genetic variation and treatment response or side effects is more complicated. For example, in the GENDEP study, *CYP2C19* and *CYP2D6* genotypes were associated with serum concentration of escitalopram and nortriptyline, but did not predict treatment response [[17](#CR17)]. A meta-analysis of 94 studies assessed the relationship between psychiatric drug exposure (dose-normalized plasma level) and metabolising status of CYP2C19 and CYP2D6, observing exposure differences in escitalopram and sertraline [[18](#CR18)]. However, treatment effectiveness of these antidepressants was not associated with *CYP2C19* genotypes in a large retrospective study based on participant self-report [[19](#CR19)].

Guidelines have been developed for antidepressant use based on CYP2D6 and CYP2C19 metabolizing status. For instance, CPIC guidelines for CYP2C19 suggest a 50% dose reduction of citalopram, escitalopram and sertraline for PMs, and alternative antidepressants that are not predominantly metabolized by CYP2C19 are advised for UMs [[15](#CR15)]. However, evidence is still accruing to confirm the role of pharmacogenetic testing to guide antidepressant prescribing [[12](#CR12)].

Both *CYP2C19* and *CYP2D6* require in-depth assessment of variation to fully determine star alleles and metabolizer status but it is complicated by structural variants and pseudogenes, particularly for *CYP2D6*. Full assessment cannot be achieved through genotyping but requires pharmacogenetic-specific tests (e.g., targeted arrays, sequencing) [[20](#CR20)–[22](#CR22)]. However, many research studies have genome-wide genotyping and lack full pharmacogenetic assessment. In this study, we combined clinical and genetic data from 13 clinical studies, with 5843 participants, of European and East Asian ancestry. Imputation from genome-wide genotyping was used to estimate the metabolic status of CYP2C19 and CYP2D6. Association of CYP2C19 metabolic phenotypes with clinically evaluated treatment response was performed to investigate whether genotype-determined PMs, IMs, and RMs/UMs showed differential antidepressant efficacy, compared to NMs. This unique resource provides additional evidence of the relationship between CYP gene metabolic phenotypes and treatment response, and may further determine whether genotype-determined metabolizer status could add useful information for individualized prescribing of antidepressants.

## Methods

### Samples

The clinical studies analyzed have been described in detail previously [[3](#CR3)]. In brief, 10 studies with European ancestry and 3 studies from East Asia were included. All participants had a diagnosis of major depressive disorder (MDD) and received at least one antidepressant, with treatment response collected at baseline, and for 4–12 weeks post-baseline. Informed consent was obtained from all participants. We assessed two antidepressant response outcomes of remission and percentage improvement. Remission was a binary outcome defined as a reduction of the depression symptoms to a prespecified criteria of the rating scale. Percentage improvement was a continuous measure calculated from the proportional decrease (or increase) of depression symptom score from baseline. The percentage improvement was standardized (mean 0, standard deviation 1) within study to allow comparability of different scales across the studies (e.g., HAMD (Hamilton Depression Rating Scale), MADRS (Montgomery Åsberg Depression Rating Scale), QIDSC (Quick Inventory of Depressive Symptomatology)). Demographic and clinical variables of age, sex, MDD baseline severity and antidepressant prescription information were available in each study (Supplementary Table [1](#MOESM1)).

Detailed procedures of genotyping have been reported elsewhere [[23](#CR23)–[31](#CR31)]. Quality control and imputation were processed using the standard ‘RICOPILI’ pipeline from the Psychiatric Genomics Consortium (PGC) with 1000 Genomes Project multi-ancestry reference panel [[32](#CR32)]. Each step was performed separately in European and East Asian ancestry studies following standard PGC protocols. Study details can be found in Supplementary Table [1](#MOESM1) and the previous study [[3](#CR3)].

### Star alleles and metabolic phenotypes

Using best guess imputed genotype calls, phasing was conducted separately on the genetic regions of *CYP2C19* and *CYP2D6* obtained from PharmGKB ([https://www.pharmgkb.org/](https://www.pharmgkb.org/)). The haplotype was determined in each sample using SHAPEIT4 software and the 1000 Genomes Project multi-ancestry reference panel [[33](#CR33)]. To fully utilize phased SNPs and translate them to star alleles, we first extracted all SNPs used to define *CYP2C19* and *CYP2D6* star alleles from the CPIC definition tables ([https://cpicpgx.org/](https://cpicpgx.org/); downloaded June 2022). These SNPs were then matched to the phased data, and matching SNPs were assigned to star alleles following the CPIC guidelines. If a star allele was defined by more than one SNP, it was counted only when all the defined SNPs were observed. Each star allele was annotated as having no, decreased, normal, or increased function with corresponding activity value based on CPIC definition tables and the previous literature (Supplementary Table [2](#MOESM1)) [[34](#CR34)]. The reference allele (*1) was assigned to haplotypes that had no annotated functional star alleles or had uncertain or unknown functional alleles of *CYP2D6*. Because structural variants cannot be determined from genotype data, CYP2D6 rapid and ultrarapid metabolizers were not included. Next, we calculated the activity score for each individual by adding the activity values of the two star alleles. Metabolic phenotypes (PM, IM, NM) of CYPC19 were classified based on CPIC and CYP2C19 rapid+ultrarapid metabolizers were defined as individuals carrying at least one increased functional allele (*17) [[15](#CR15)]. CYP2D6 phenotypes (PM, IM, NM) were determined following consensus recommendations from the CPIC and the Dutch Pharmacogenetics Working Group (DPWG) [[35](#CR35)]. To validate the defined metabolic phenotypes, we compared phenotype concordance with that previously derived in the GENDEP using Roche AmpliChip CYP450 microarray and TaqMan SNP genotyping [[17](#CR17)]. After harmonizing the metabolizer status, the concordance rate (percentage of individuals assigned the same metabolic phenotypes) was 96.4% for CYP2C19 and 79.9% for CYP2D6 (Supplementary Table [3](#MOESM1)). The proportion of misclassification in CYP2C19 and CYP2D6 metabolic phenotypes can be found in Supplementary Table [3](#MOESM1).

### Statistical analyses

#### Associations

Given the suboptimal performance of genotype-based metabolic phenotype imputation in CYP2D6, we exclusively focused on CYP2C19 in the association and meta-analyses to ensure the validity of the results. We used the NMs in CYP2C19 as the reference group to examine the effect of other metabolizer groups on antidepressant response. For remission, logistic regression was used to evaluate the association with CYP2C19 metabolic phenotypes in each study, including age, sex, and MDD baseline severity as covariates. For percentage improvement, linear regression with CYP2C19 metabolic phenotypes, adjusting for age and sex, was used to test for association with metabolic phenotypes. The correlation between MDD baseline score and percentage improvement was very low (Pearson correlation = 0.042), so we did not add MDD baseline severity as a covariate. We next stratified into ‘antidepressant groups’, with drugs that were primarily metabolized by CYP2C19, based on the clinical annotation of Level 1 A in PharmGKB [[13](#CR13), [36](#CR36)](Supplementary Table [4](#MOESM1)). Stratifying participants by CYP2C19-metabolized antidepressants, we repeated the analyses of remission and percentage improvement in 10 studies with CYP2C19-metabolized antidepressants (3390 participants) (Supplementary Figure [1](#MOESM1)).

#### Meta-analyses

In each study, odds ratios (ORs) of remission, and Standard Mean Differences (SMDs, Cohen’s D) of percentage improvement, with standard errors of both effect sizes, for each metabolizer group were extracted. We applied random effect meta-analysis since the true effects were assumed to be heterogeneous due to the difference in factors such as study populations, antidepressants prescribed, and outcome measurements. The effect sizes in each study were pooled, and inverse-variance weighted. The between-study heterogeneity was quantified by *I*^2^ statistic and heterogeneity variance *τ*^2^ using the Paule-Mandel method for ORs and restricted maximum-likelihood estimator for SMDs. The significance was tested by Cochran’s Q at *p* < 0.05. Additionally, subgroup meta-analyses were applied to test the hypothesis that effects differed between European and East Asian ancestry. We assumed both ancestries shared a common between-study heterogeneity (*τ*^2^) due to a small number of studies from East Asia. Cochran’s Q was used to determine whether the differences between subgroups could be explained by true effect differences or by sampling errors alone. We performed meta-analyses in all samples for CYP2C19 metabolic phenotypes and then stratified the analyses by antidepressant groups for the corresponding metabolizer effects. We used *p*-value < 0.05 as nominal significance, and corrected for multiple testing for the 3 independent tests of metabolic phenotypes compared with NMs (3 phenotypes in CYP2C19), giving a Bonferroni corrected *p*-value of 0.017 (0.05/3). No correction across outcomes (remission and percentage improvement) was applied, due to their high correlations. All meta-analyses were performed by ‘meta’ package in R 4.2.1.

The power of the meta-analysis was calculated by ‘dmetar’ package in R 4.2.1. Using the sample size of PMs (*N* = 179) and NMs (*N* = 2289) in CYP2C19, the meta-analysis had over 80% power to detect SMD of 0.074 and OR 1.15 with no effect heterogeneity, or SMD 0.085 and OR 1.17 with low heterogeneity, at a significance level *p* = 0.01.

### Sensitivity tests

Four sensitivity analyses were performed. Firstly, each participant’s activity score was calculated as a continuous measure to assess metabolic activity and compared to the metabolizer groups. We tested CYP2C19 metabolic effects represented by activity scores using the same analyses described above. For the percentage improvement outcome, correlations were assessed between activity scores and residuals of percentage improvement after regressing out age and sex, and restricted maximum-likelihood estimator was used to estimate between-study heterogeneity of correlations in the meta-analyses. Secondly, the impact of baseline depression severity on percentage improvement was assessed by including it as a covariate in the linear regression analyses. Thirdly, to test how small studies might be impacting results, we reran the meta-analysis of CYP2C19 PM on the remission outcome including only studies with at least 10 PMs present. Finally, the association of CYP2C19 metabolic phenotype with citalopram and escitalopram were measured to compare the effect with all samples and CYP2C19 antidepressant group.

## Results

### Characteristics of star alleles and metabolic phenotypes

Seven star alleles in *CYP2C19* and 16 alleles in *CYP2D6* were identified from the imputed genotype data and were classified as having no, decreased, normal and increased function (Supplementary Table [2](#MOESM1)). In general, alleles had similar frequencies in studies of the same ancestry group (Supplementary Figure [2](#MOESM1)). The reference alleles (*1) were the most common, with mean frequency 62.8% in *CYP2C19*, and 39.2% in *CYP2D6* in European ancestry studies, and frequencies of 62.1% and 34.2% in East Asian studies. Other high frequency alleles in European-ancestry studies were *17 (22.0%) in *CYP2C19* and *4 (19.8%) in *CYP2D6*, while *CYP2C19* *2 (30.5%) and *CYP2D6* *10 (48.6%) had high frequencies in East Asian studies. Structural variants including *5 in *CYP2D6* cannot be imputed from genotype, making assessment of *CYP2D6* star alleles and metabolizer status incomplete. A total of 5843 individuals with remission or percentage improvement outcome in 13 studies were analyzed. Four metabolizer groups (PMs, IMs, NMs, RMs+UMs) for CYP2C19 and three metabolizer groups (PMs, IMs, and NMs) for CYP2D6 were translated from star alleles. In both genes, the most common metabolizer group was NMs, and the rarest was PMs (Table [1](#Tab1)). Compared with the East Asians, the European-ancestry studies had a lower proportion of CYP2C19 PMs and IMs, and higher proportion of RMs+UMs. CYP2D6 PMs were only found in the European-ancestry studies (Fig. [1](#Fig1), differences between ancestries, Wilcoxon test: CYP2C19 PMs *p* = 0.007, CYP2C19 IMs *p* = 0.007, CYP2C19 RMs+UMs *p* = 0.007, CYP2D6 PMs *p* = 0.014). For the 12 antidepressants metabolized primarily by either CYP2C19 or CYP2D6, the same distribution of metabolic phenotypes was found in both antidepressant groups (Supplementary Table [5](#MOESM1)).

### Table 1.

|   | CYP2C19 |  |  |  | CYP2D6 (incomplete assessment*) |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- |
| PM (N = 179) | IM (N = 1601) | NM (N = 2289) | RM + UM (N = 1774) | PM (N = 249) | IM (N = 2087) | NM (N = 3507) |  |
| Remission | 89 (49.7%) | 607 (37.9%) | 885 (38.7%) | 662 (37.3%) | 96 (38.6%) | 796 (38.1%) | 1351 (38.5%) |
| Percentage Improvement | 0.12 (1.10) | 0.00 (0.98) | −0.01 (1.01) | 0.00 (0.99) | −0.02 (0.98) | 0.01 (0.98) | 0.00 (1.00) |
| Age | 45.15 (14.53) | 44.76 (14.64) | 44.44 (14.19) | 44.95 (14.17) | 43.41 (14.62) | 44.51 (14.29) | 44.91 (14.31) |
| Sex (female) | 112 (62.6%) | 987 (61.6%) | 1438 (62.8%) | 1112 (62.7%) | 154 (61.8%) | 1301 (62.3%) | 2194 (62.6%) |
| Ancestry (European) | 110 (61.5%) | 1360 (84.9%) | 2078 (90.8%) | 1768 (99.7%) | 249 (100%) | 1902 (91.1%) | 3165 (90.2%) |
| CYP2D6 (incomplete assessment*) |  |  |  |  |  |  |  |
| PM | 7 (3.9%) | 55 (3.4%) | 96 (4.2%) | 91 (5.1%) | - | - | - |
| IM | 58 (32.4%) | 598 (37.4%) | 785 (34.3%) | 646 (36.4%) | - | - | - |
| NM | 114 (63.7%) | 948 (59.2%) | 1408 (61.5%) | 1037 (58.5%) | - | - | - |

Table 1 Caption: Sample characteristics.

### Fig. 1. Proportion of metabolic phenotypes in each cohort.

![Fig. 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/bee1/11258238/711fd0074612/41398_2024_2981_Fig1_HTML.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=11258238_41398_2024_2981_Fig1_HTML.jpg)

DAST Depression and Sequence of Treatment, GENDEP Genome Based Therapeutic Drugs for Depression, GENPOD GENetic and clinical Predictors Of treatment response in Depression, GODS Geneva Outpatient Depression Study, GSK: Glaxo Smith Kline, GSRD Group for the Study of Resistant Depression, PFZ Pfizer, PGRN Pharmacogenomics Research Network Antidepressant Medication Pharmacogenomic Study, STARD Sequenced Treatment Alternatives to Relieve Depression. PM poor metabolizer, IM intermediate metabolizer, NM normal metabolizer, RM + UM rapid+ultrarapid metabolizer. *Due to undetected variants in genotype, imputation of CYP2D6 metabolic phenotypes was less accurate.

### Meta-analyses of CYP2C19 metabolic phenotypes in all samples

Given the incomplete representativeness of CYP2D6 metabolic status using genotype, we focused on CYP2C19 metabolic phenotypes in the association test. The association of metabolizer status with antidepressant response was first assessed in all samples, across all antidepressants. The remission rate and mean percentage improvement in each metabolizer group are presented in Table [1](#Tab1). Overall, PMs in CYP2C19 showed a higher remission rate with nominal significance (OR = 1.46, 95% CI [1.03, 2.06], *p* = 0.033, Fig. [2a](#Fig2)) but did not meet correction for multiple testing. The percentage improvement analysis showed a non-significant higher efficacy in PMs (SMD = 0.13, 95% CI [−0.03, 0.29], *p* = 0.101). Other metabolic phenotypes in CYP2C19 had no difference from NMs in both outcomes (Fig. [2a](#Fig2)). Subgroup meta-analyses found no heterogeneity in the effect of CYP2C19 PMs in all cohorts or between ancestry groups (all cohorts: I^2^ = 0%, τ^2^ = 0, *p* = 0.81; between groups: χ^2^ = 0.13, *p* = 0.72). In other metabolic phenotypes, no significant heterogeneity was detected (Supplementary Figure [3](#MOESM1)).

### Fig. 2. Association of CYP2C19 metabolizer status with antidepressant outcomes.

![Fig. 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/bee1/11258238/48fbcdbfed80/41398_2024_2981_Fig2_HTML.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=11258238_41398_2024_2981_Fig2_HTML.jpg)

a Association of CYP2C19 metabolic phenotypes in all samples. b Association of CYP2C19 metabolic phenotypes stratified by CYP2C19-metabolized antidepressants. PM poor metabolizer, IM intermediate metabolizer, RM + UM rapid+ultrarapid metabolizer, OR odd ratio, SMD standard mean difference, CI confidence interval.

### Meta-analyses of CYP2C19 metabolic phenotypes stratified by antidepressant groups

Next, to determine if the metabolic activity was associated with response in antidepressants that were primarily metabolized by CYP2C19 [[13](#CR13)], meta-analyses were stratified with 7 CYP2C19-metabolized antidepressants (Supplementary Table [4](#MOESM1)). CYP2C19 PMs showed a similar trend to the results in all samples, with a higher remission rate and percentage improvement compared to NMs (remission: OR = 1.47, 95% CI [0.90, 2.39], *p* = 0.121; percentage improvement: SMD = 0.12, 95% CI [–0.10, 0.34], *p* = 0.282, Fig. [2b, c](#Fig2)) but the association was not significant. Other metabolizer groups were not associated with response. Detailed results for each study can be found in Supplementary Figure [4](#MOESM1). As a comparison, metabolic effect was tested in the antidepressant groups that were not primarily metabolized by CYP2C19. Detailed results are shown in Supplementary Figure [5](#MOESM1).

### Sensitivity test

Finally, four sensitivity tests were performed. First, the meta-analyses were repeated using the activity score as a quantitative measurement of metabolic activity to compare the results with the primary analyses. The activity scores differed between European and East Asian studies, with Europeans having higher scores for both CYP2C19 and CYP2D6 (Wilcoxon test: CYP2C19 *p* = 0.007; CYP2D6 *p* = 0.028, Supplementary Figure [6](#MOESM1)). However, activity score of CYP2C19 was not associated with the outcomes of remission or percentage improvement (Supplementary Table [6](#MOESM1)). In the second sensitivity test, baseline severity of depression was added as an additional covariate in the analyses of percentage improvement. As in the primary analyses, PMs in CYP2C19 had higher, but non-significant SMD of percentage improvement (SMD = 0.13, 95% CI [−0.03, 0.29], *p* = 0.103). No clear pattern was found in tests of other metabolizers (Supplementary Table [7](#MOESM1)). Furthermore, we meta-analyzed the CYP2C19 PMs for remission by including only studies with more than 10 CYP2C19 PMs. A higher rate of remission was observed in CYP2C19 PMs from 8 studies confirming the association found in the main analyses (OR = 1.56, 95% CI [1.09; 2.24], *p* = 0.016). Lastly, we tested the effect of CYP2C19 on citalopram and escitalopram antidepressants. No significant findings were detected, with the strongest effects found in CYP2C19 PMs, showing a non-significant increase in the remission rate and percentage improvement compared to NMs (remission: OR = 1.41, 95% CI [0.84, 2.34], *p* = 0.192, percentage improvement: SMD = 0.069, 95% CI [−0.16, 0.30], *p* = 0.559, Supplementary Table [8](#MOESM1)).

## Discussion

In this study, we leveraged 13 clinical studies (10 of European-ancestry and 3 from East Asia) to identify the metabolic status of CYP2C19 and CYP2D6 through genome-wide genotyping. A meta-analysis was performed for the association between CYP2C19 metabolic phenotypes and antidepressant response, using remission and percentage improvement as outcome measures. Using the available imputed genotype data, we identified 7 star alleles of *CYP2C19* and 16 star alleles of *CYP2D6*. We found CYP2C19 PMs had a higher remission rate compared to CYP2C19 NMs in all samples (OR = 1.46; 95% CI [1.03, 2.06]), which reached nominal significance but was not significant at the multiple testing threshold. CYP2C19 PMs also had a higher remission rate in antidepressants primarily metabolized by CYP2C19 (OR = 1.47, 95% CI [0.90, 2.39]) but differences were not significant. No difference in percentage improvement was seen between PMs and NMs. Other metabolizer groups in CYP2C19 showed no association with either remission or percentage improvement. Although there were differences in the frequency of star alleles and in the proportions of metabolic phenotypes between European and East Asian ancestry studies, the impact of metabolic phenotypes was similar.

In *CYP2C19*, our analysis pipeline detected 7 star alleles including all tier 1 alleles (*2, *3 and *17) and two tier 2 alleles (*8, *35) demonstrating a good coverage of imputed genotype for *CYP2C19* region [[37](#CR37)]. Nevertheless, only a moderate relationship was detected with CYP2C19 PMs with the remission outcome. Other metabolizer statuses were not associated with treatment outcomes. When testing the PMs restricted to antidepressants largely metabolized by CYP2C19, a similar effect size was detected but showed no significance, suggesting a loss of power. Other meta-analyses, retrospective studies, and clinical cohorts have replicated a higher antidepressant efficacy of CYP2C19 PMs [[19](#CR19), [38](#CR38)–[40](#CR40)]. However, a null effect or an opposite association of CYP2C19 slow metabolizers for lower antidepressant efficacy was observed in smaller samples [[17](#CR17), [41](#CR41), [42](#CR42)]. This discrepancy may be due to different criteria for study participants, MDD severity, dropout rates, medication prescribed, and lack of information on other associated factors such as antidepressant dosage. Given the heterogeneity of patients and potential confounding variables, our results need further replication to understand the role of CYP2C19 metabolizers. In addition to treatment efficacy, PMs of CYP2C19 were also associated with worse antidepressant tolerability, although these features were not assessed in our study [[19](#CR19), [39](#CR39)]. CPIC and the Dutch Pharmacogenetics Working Group (DPWG) have recommended reducing the starting dose of escitalopram, citalopram, and sertraline for CYP2C19 PMs because of the increased probability of adverse effects [[15](#CR15), [43](#CR43)]. Appropriate support could be provided to patients at the beginning of the treatment to reduce the dropout rate and maximize the drug effect.

In *CYP2D6*, genotype data had lower ability to identify star alleles. Our study detected 16 star alleles of *CYP2D6*, which were classified as having no, decreased, or normal function. No structural variants could be detected, so increased function alleles (*xN) were not called, and fewer PMs/IMs were reported (such as undetected deletion *5). Approximately 7% of *CYP2D6* variants are structural variants, so the star allele calls, diplotype assignment and metabolic phenotype can be affected by missing structural variants [[22](#CR22), [44](#CR44)]. The low concordance rate of CYP2D6 metabolic phenotype with previous assessment in the GENDEP study [[17](#CR17)] (79.9%) indicates a limited allele coverage in the genotype, leading to a higher proportion of NMs and misclassification of other metabolic phenotypes (Supplementary Table [3](#MOESM1)).

Activity score was also applied for the assignment of metabolic phenotype. Using clinical guidelines, each allele from *CYP2C19* and *CYP2D6* is assigned an activity value and the value is summed across the two alleles carried to give an activity score representing the individual’s metabolic activity [[34](#CR34), [35](#CR35), [40](#CR40)]. We found no effect of CYP2C19 activity score on the outcomes of remission and percentage improvement but not the limitations of performing this across all drugs. These antidepressant results contrast to antipsychotic response, where higher CYP2C19 activity score was associated with lower symptom severity [[34](#CR34)]. The previous association of CYP2C19 PMs with remission outcome was not detected in the activity score analysis. This is likely because PMs have a low frequency and represent only the lower tail of the activity score distribution, so the effect is diluted when combining phenotype groups.

Our analyses included both European and East Asian ancestry populations. The frequencies of star alleles were clustered by ancestry. For example, European population had lower frequencies of *2, *3 in *CYP2C19* and *10 in *CYP2D6*, but higher frequencies of *CYP2C19* *17 and *CYP2D6* *4, than the East Asian population, leading to fewer PMs and IMs for CYP2C19 but higher proportions of CYP2C19 RMs+UMs and CYP2D6 PMs. These ancestry differences align with the CPIC guideline and other reports [[15](#CR15), [45](#CR45)]. When connecting the cytochrome enzyme status with antidepressant response, few studies have been performed in the East Asian population. A clinical trial of 100 depression patients from Taipei found CYP2C19 poor metabolizers had higher serum levels of antidepressants [[46](#CR46)]. In some antipsychotics metabolized by specific cytochrome enzymes, the plasma concentrations of drugs are higher in East Asian populations than in European populations [[47](#CR47)]. In contrast, modelling has suggested that the metabolic contributions of CYP2C19 on escitalopram would be similar across European and Asian populations [[48](#CR48)]. As there is little evidence of differentiation by ancestry, current clinical guidelines provide the same antidepressant dosing recommendations across populations [[15](#CR15)]. Our subgroup meta-analyses between the European and East Asian studies found no difference in metabolic effect for each phenotype of CYP2C19 but the low sample size in East Asian studies (9% of all samples) implies much reduced power compared to the European studies.

Some study limitations should be considered. In addition to the incomplete assessment of star alleles from genotype data considered above, larger sample sizes are needed specifically in different ancestries and drug groups to evaluate drug-specific metabolic effect. Too few CYP2C19 PMs (2.1% in European, 3.1% in all participants) were present to show a statistically significant association after correcting for multiple testing. Similarly, CYP2C19 RMs+UMs (1.1%) were rare in the East Asian population. Citalopram and escitalopram were the most prescribed drugs, accounting for 54% of all samples and 93% of the CYP2C19 antidepressant group, so the metabolic effect on treatment response was mainly determined by these two drugs. In addition, no data from clinical evaluations or the environment (e.g., dosage, concomitant drugs, smoking, diet) were analyzed, and these factors could influence symptom improvement and cytochrome metabolic activity. Although no significant heterogeneity was detected in the meta-analyses, we should acknowledge the differences among cohorts in study design, patient selection, and response measures. No significant differences between IMs/RMs+UMs and NMs could be detected, and higher power is probably needed to effectively test between metabolizer groups. Side effects were also not available in our data, which are associated with metabolic phenotypes. We analyzed only the final depression score, at the end of the study treatment, to determine remission and calculate the percentage improvement. Other studies have suggested using longitudinal measures throughout treatment period as repeated measures in a mixed linear model to improve the statistical power [[41](#CR41)]. While our study concentrates on *CYP2C19* and *CYP2D6*, there is potential for extending pharmacogenetic testing to other pharmacokinetic (*CYP2B6*) and pharmacodynamic genes (*SLC6A4*, *HTR2A*) to understand their impact on antidepressant efficacy and tolerability [[15](#CR15)]. Finally, the imputed genotype showed a promising utility in detecting *CYP2C19* star alleles, but in *CYP2D6*, no RMs/UMs and fewer PMs/IMs were identified due to undetected rare and structural variants. Deeper imputation panels that detect structure variants, or further genetic studies using sequencing or modern targeted array would be necessary for a full assessment of CYP2D6 metabolizer status [[49](#CR49)].

In conclusion, using imputed genotype data, our meta-analysis showed no significant association between CYP2C19 metabolic phenotypes with antidepressant response. Moderate evidence of an association with CYP2C19 poor metabolizers was indicated, which had higher rates of antidepressant remission. Metabolic phenotypes of CYP2C19 differed in frequency between European and East Asian populations but did not differ in their effect on treatment outcomes. Research studies with genotype are limited in their assessment of pharmacogenetic variation especially for CYP2D6. Fuller assessment of metabolizer status together with information on clinical factors and broader ancestry diversity could reduce the heterogeneity and improve power to evaluate the effect of metabolic phenotypes on antidepressant response.

## Supplementary information

## Acknowledgements

The collection of the sample from the Group for the Study of Resistant Depression (GSRD) Consortium was supported by an unrestricted grant from Lundbeck for the GSRD. Lundbeck had no further role in the study design and the collection, analysis, and interpretation of data. The GENDEP (Genome Based Therapeutic Drugs for Depression) study was funded by a European Commission Framework 6 grant (EC Contract Ref. No. LSHB-CT-2003–503428). H. Lundbeck provided nortriptyline and escitalopram for the GENDEP study. GlaxoSmithKline and the UK National Institute for Health Research of the Department of Health contributed to the funding of the sample collection at the Institute of Psychiatry, London. GENDEP Illumina array genotyping was funded in part by a joint grant from the UK Medical Research Council and GlaxoSmithKline (Grant No. G0701420). The GENPOD (GENetic and clinical Predictors Of treatment response in Depression) trial was funded by the UK Medical Research Council and supported by the UK Mental Health Research Network. The genotyping of GENPOD samples was supported by the Innovative Medicine Initiative Joint Undertaking under Grant No. 115008, of which resources are composed of European Union and the European Federation of Pharmaceutical Industries and Associations (EFPIA) in-kind contribution and financial contribution from the European Union’s Seventh Framework Programme (Grant No. FP7/2007–2013). EFPIA members Pfizer, GlaxoSmithKline, and F. Hoffmann-La Roche have contributed work and samples to the project presented here. The PFZ (Pfizer), GSK (GlaxoSmithKline), and GODS were supported by the Innovative Medicine Initiative Joint Undertaking (IMI-JU) under Grant No. 115008 of which resources are composed of European Union and EFPIA) in-kind contribution and financial contribution from the European Union’s Seventh Framework Programme (FP7/2007–2013). EFPIA members Pfizer, GlaxoSmithKline, and F. Hoffmann La-Roche have contributed work and samples to the project presented here. The PGRN-AMPS (Pharmacogenomics Research Network Antidepressant Medication Pharmacogenomic Study) study data were obtained via the database of Genotypes and Phenotypes (dbGAP) (Accession No. phs000670.v1.p1). Funding support for the PGRN-AMPS study was provided by the National Institute of General Medical Sciences, National Institutes of Health, through the PGRN grant to Principal Investigators R. Weinshilboum and L. Wang (Grant No. U19 GM61388). D. Mrazek served as the Principal Investigator for the PGRN-AMPS study within the Mayo Clinic PGRN program. Genome-wide genotyping was performed at the RIKEN Center for Genomic Medicine, with funding provided by RIKEN. The datasets used for the analyses described in this manuscript were obtained from the dbGaP at [http://www.ncbi.nlm.nih.gov/gap/](http://www.ncbi.nlm.nih.gov/gap/). Major funding for the Psychiatric Genetics Consortium (PGC) is from the U.S. National Institutes of Health (Grant Nos. U01 MH109528 and U01 MH109532). For the purposes of open access, the author has applied a Creative Commons Attribution (CC BY) licence to any Accepted Author Manuscript version arising from this submission.

## Author contributions

CML and DL conceived and designed the study. CF, AC, DS, MZD, NH, JH, GL, OM, NP, MR, RU, WM, BTB, JMB, GB, KD, MK, YLL, AS, SJT, and RW led the original studies and provided the data. DL conducted the analyses with the support from OP and CF. DL and CML wrote the original manuscript. All authors reviewed and approved the final manuscript.

## Funding

This study was supported by the NIMH ([MH124873](https://www.ncbi.nlm.nih.gov/nuccore/MH124873)), the Wellcome Trust (226770/Z/22/Z) and by the NIHR Maudsley Biomedical Research Centre at South London and Maudsley NHS Foundation Trust and King’s College London. Chiara Fabbri and Alessandro Serretti were partly supported by #NEXTGENERATIONEU (NGEU), funded by the Ministry of University and Research (MUR), National Recovery and Resilience Plan (NRRP), project MNESYS (PE0000006) - A Multiscale integrated approach to the study of the nervous system in health and disease (DN. 1553 11.10.2022).

## Data availability

The data that support the findings of this study are available from the corresponding authors upon request.

## Code availability

Analysis code is available at: [https://github.com/DanyangLi107/PGC_CYP2gene](https://github.com/DanyangLi107/PGC_CYP2gene).

## Competing interests

CML serves on the scientific advisory board for Myriad Neuroscience, and is a consultant for UCB. AS is or has been consultant/speaker for: Abbott, AbbVie, Angelini, AstraZeneca, Clinical Data, Boehringer, Bristol Myers Squibb, Eli Lilly, GlaxoSmithKline, InnovaPharma, Italfarmaco, Janssen, Lundbeck, Naurex, Pfizer, Polifarma, Sanofi, and Servier. AMM has received research support from the Sackler Trust and speaker fees from Janssen and Illumina. MK has received grant funding from the Japanese Ministry of Health, Labor and Welfare, the Japan Society for the Promotion of Science, SENSHIN Medical Research Foundation, the Japan Research Foundation for Clinical Pharmacology and the Japanese Society of Clinical Neuropsychopharmacology and speaker’s honoraria from Sumitomo Pharma, Otsuka, Meiji-Seika Pharma, Eli Lilly, MSD K.K., Pfizer, Janssen Pharmaceutical, Shionogi, Mitsubishi Tanabe Pharma, Takeda Pharmaceutical, Lundbeck Viatris Inc, Eisai Co., Ltd. and Ono Pharmaceutical and participated in an advisory/review board for Otsuka, Sumitomo Pharma, Shionogi and Boehringer Ingelheim. DS has received grant/research support from GlaxoSmithKline and Lundbeck; and served as a consultant or on advisory boards for AstraZeneca, Bristol Myers Squibb, Eli Lilly, Janssen, and Lundbeck. CF was a speaker for Janssen. NP is or has been consultant/speaker for: Takeda, Janssen and Lundbeck. All other authors report no biomedical financial interests or potential conflicts of interest.

## Ethics approval and consent to participate

This was a secondary analysis of studies shared by investigators with the Psychiatric Genomics Consortium. Each original study had received ethics approval.

## Footnotes

## Supplementary information

The online version contains supplementary material available at 10.1038/s41398-024-02981-1.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

The data that support the findings of this study are available from the corresponding authors upon request.

Analysis code is available at: [https://github.com/DanyangLi107/PGC_CYP2gene](https://github.com/DanyangLi107/PGC_CYP2gene).

### Supplementary Materials

### Data Availability Statement

The data that support the findings of this study are available from the corresponding authors upon request.

Analysis code is available at: [https://github.com/DanyangLi107/PGC_CYP2gene](https://github.com/DanyangLi107/PGC_CYP2gene).

## References

1. National Institute for Health and Care Excellence. Depression in adults: treatment and management. NICE guideline NG222. 2022. https://www.nice.org.uk/guidance/ng222.  [https://www.nice.org.uk/guidance/ng222](https://www.nice.org.uk/guidance/ng222) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35977056/)

2. Fabbri C, Hagenaars SP, John C, Williams AT, Shrine N, Moles L, et al. Genetic and clinical characteristics of treatment-resistant depression using primary care records in two UK cohorts. Mol Psychiatry. 2021;26:3363–73.  [DOI](https://doi.org/10.1038/s41380-021-01062-9) | [PMC free article](/articles/PMC8505242/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33753889/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Fabbri%20C,%20Hagenaars%20SP,%20John%20C,%20Williams%20AT,%20Shrine%20N,%20Moles%20L,%20et%20al.%20Genetic%20and%20clinical%20characteristics%20of%20treatment-resistant%20depression%20using%20primary%20care%20records%20in%20two%20UK%20cohorts.%20Mol%20Psychiatry.%202021;26:3363%E2%80%9373.)

3. Pain O, Hodgson K, Trubetskoy V, Ripke S, Marshe VS, Adams MJ, et al. Identifying the Common Genetic Basis of Antidepressant Response. Biol Psychiatry Glob Open Sci. 2022;2:115–26.  [DOI](https://doi.org/10.1016/j.bpsgos.2021.07.008) | [PMC free article](/articles/PMC9117153/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35712048/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Pain%20O,%20Hodgson%20K,%20Trubetskoy%20V,%20Ripke%20S,%20Marshe%20VS,%20Adams%20MJ,%20et%20al.%20Identifying%20the%20Common%20Genetic%20Basis%20of%20Antidepressant%20Response.%20Biol%20Psychiatry%20Glob%20Open%20Sci.%202022;2:115%E2%80%9326.)

4. Trivedi MH, Rush AJ, Wisniewski SR, Nierenberg AA, Warden D, Ritz L, et al. Evaluation of Outcomes With Citalopram for Depression Using Measurement-Based Care in STAR*D: Implications for Clinical Practice. Am J Psychiatry. 2006;163:28–40.  [DOI](https://doi.org/10.1176/appi.ajp.163.1.28) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16390886/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Trivedi%20MH,%20Rush%20AJ,%20Wisniewski%20SR,%20Nierenberg%20AA,%20Warden%20D,%20Ritz%20L,%20et%20al.%20Evaluation%20of%20Outcomes%20With%20Citalopram%20for%20Depression%20Using%20Measurement-Based%20Care%20in%20STAR*D:%20Implications%20for%20Clinical%20Practice.%20Am%20J%20Psychiatry.%202006;163:28%E2%80%9340.)

5. Sforzini L, Worrell C, Kose M, Anderson IM, Aouizerate B, Arolt V, et al. A Delphi-method-based consensus guideline for definition of treatment-resistant depression for clinical trials. Mol Psychiatry. 2022;27:1286–99.  [DOI](https://doi.org/10.1038/s41380-021-01381-x) | [PMC free article](/articles/PMC9095475/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34907394/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Sforzini%20L,%20Worrell%20C,%20Kose%20M,%20Anderson%20IM,%20Aouizerate%20B,%20Arolt%20V,%20et%20al.%20A%20Delphi-method-based%20consensus%20guideline%20for%20definition%20of%20treatment-resistant%20depression%20for%20clinical%20trials.%20Mol%20Psychiatry.%202022;27:1286%E2%80%9399.)

6. Carvalho AF, Cavalcante JL, Castelo MS, Lima MCO. Augmentation strategies for treatment-resistant depression: a literature review: Augmentation strategies for TRD. J Clin Pharm Ther. 2007;32:415–28.  [DOI](https://doi.org/10.1111/j.1365-2710.2007.00846.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17875106/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Carvalho%20AF,%20Cavalcante%20JL,%20Castelo%20MS,%20Lima%20MCO.%20Augmentation%20strategies%20for%20treatment-resistant%20depression:%20a%20literature%20review:%20Augmentation%20strategies%20for%20TRD.%20J%20Clin%20Pharm%20Ther.%202007;32:415%E2%80%9328.)

7. Bleakley S. Antidepressant drug interactions: evidence and clinical significance: Antidepressant drug interactions. Prog Neurol Psychiatry. 2016;20:21–7.  [Google Scholar](https://scholar.google.com/scholar_lookup?Bleakley%20S.%20Antidepressant%20drug%20interactions:%20evidence%20and%20clinical%20significance:%20Antidepressant%20drug%20interactions.%20Prog%20Neurol%20Psychiatry.%202016;20:21%E2%80%937.)

8. Fabbri C, Pain O, Hagenaars SP, Lewis CM, Serretti A. Transcriptome-wide association study of treatment-resistant depression and depression subtypes for drug repurposing. Neuropsychopharmacology. 2021;46:1821–9.  [DOI](https://doi.org/10.1038/s41386-021-01059-6) | [PMC free article](/articles/PMC8357803/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34158615/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Fabbri%20C,%20Pain%20O,%20Hagenaars%20SP,%20Lewis%20CM,%20Serretti%20A.%20Transcriptome-wide%20association%20study%20of%20treatment-resistant%20depression%20and%20depression%20subtypes%20for%20drug%20repurposing.%20Neuropsychopharmacology.%202021;46:1821%E2%80%939.)

9. Fava M, Uebelacker LA, Alpert JE, Nierenberg AA, Pava JA, Rosenbaum JF. Major depressive subtypes and treatment response. Biol Psychiatry. 1997;42:568–76.  [DOI](https://doi.org/10.1016/S0006-3223(96)00440-4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9376453/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Fava%20M,%20Uebelacker%20LA,%20Alpert%20JE,%20Nierenberg%20AA,%20Pava%20JA,%20Rosenbaum%20JF.%20Major%20depressive%20subtypes%20and%20treatment%20response.%20Biol%20Psychiatry.%201997;42:568%E2%80%9376.)

10. Iosifescu DV, Bankier B, Fava M. Impact of medical comorbid disease on antidepressant treatment of major depressive disorder. Curr Psychiatry Rep. 2004;6:193–201.  [DOI](https://doi.org/10.1007/s11920-004-0064-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15142472/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Iosifescu%20DV,%20Bankier%20B,%20Fava%20M.%20Impact%20of%20medical%20comorbid%20disease%20on%20antidepressant%20treatment%20of%20major%20depressive%20disorder.%20Curr%20Psychiatry%20Rep.%202004;6:193%E2%80%93201.)

11. Oliveira P, Ribeiro J, Donato H, Madeira N. Smoking and antidepressants pharmacokinetics: a systematic review. Ann Gen Psychiatry. 2017;16:17.  [DOI](https://doi.org/10.1186/s12991-017-0140-8) | [PMC free article](/articles/PMC5340025/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28286537/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Oliveira%20P,%20Ribeiro%20J,%20Donato%20H,%20Madeira%20N.%20Smoking%20and%20antidepressants%20pharmacokinetics:%20a%20systematic%20review.%20Ann%20Gen%20Psychiatry.%202017;16:17.)

12. Bousman CA, Bengesser SA, Aitchison KJ, Amare AT, Aschauer H, Baune BT, et al. Review and Consensus on Pharmacogenomic Testing in Psychiatry. Pharmacopsychiatry. 2021;54:5–17.  [DOI](https://doi.org/10.1055/a-1288-1061) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33147643/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Bousman%20CA,%20Bengesser%20SA,%20Aitchison%20KJ,%20Amare%20AT,%20Aschauer%20H,%20Baune%20BT,%20et%20al.%20Review%20and%20Consensus%20on%20Pharmacogenomic%20Testing%20in%20Psychiatry.%20Pharmacopsychiatry.%202021;54:5%E2%80%9317.)

13. Bousman CA, Zierhut H, Müller DJ. Navigating the Labyrinth of Pharmacogenetic Testing: A Guide to Test Selection. Clin Pharmacol Ther. 2019;106:309–12.  [DOI](https://doi.org/10.1002/cpt.1432) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31004441/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Bousman%20CA,%20Zierhut%20H,%20M%C3%BCller%20DJ.%20Navigating%20the%20Labyrinth%20of%20Pharmacogenetic%20Testing:%20A%20Guide%20to%20Test%20Selection.%20Clin%20Pharmacol%20Ther.%202019;106:309%E2%80%9312.)

14. Hicks JK, Sangkuhl K, Swen JJ, Ellingrod VL, Müller DJ, Shimoda K, et al. Clinical pharmacogenetics implementation consortium guideline (CPIC) for CYP2D6 and CYP2C19 genotypes and dosing of tricyclic antidepressants: 2016 update. Clin Pharmacol Ther. 2017;102:37–44.  [DOI](https://doi.org/10.1002/cpt.597) | [PMC free article](/articles/PMC5478479/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27997040/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Hicks%20JK,%20Sangkuhl%20K,%20Swen%20JJ,%20Ellingrod%20VL,%20M%C3%BCller%20DJ,%20Shimoda%20K,%20et%20al.%20Clinical%20pharmacogenetics%20implementation%20consortium%20guideline%20(CPIC)%20for%20CYP2D6%20and%20CYP2C19%20genotypes%20and%20dosing%20of%20tricyclic%20antidepressants:%202016%20update.%20Clin%20Pharmacol%20Ther.%202017;102:37%E2%80%9344.)

15. Bousman CA, Stevenson JM, Ramsey LB, Sangkuhl K, Hicks JK, Strawn JR, et al. Clinical Pharmacogenetics Implementation Consortium (CPIC) Guideline for CYP2D6, CYP2C19, CYP2B6, SLC6A4, and HTR2A Genotypes and Serotonin Reuptake Inhibitor Antidepressants. Clin Pharmacol Ther. 2023;114:51–68.  [DOI](https://doi.org/10.1002/cpt.2903) | [PMC free article](/articles/PMC10564324/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/37032427/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Bousman%20CA,%20Stevenson%20JM,%20Ramsey%20LB,%20Sangkuhl%20K,%20Hicks%20JK,%20Strawn%20JR,%20et%20al.%20Clinical%20Pharmacogenetics%20Implementation%20Consortium%20(CPIC)%20Guideline%20for%20CYP2D6,%20CYP2C19,%20CYP2B6,%20SLC6A4,%20and%20HTR2A%20Genotypes%20and%20Serotonin%20Reuptake%20Inhibitor%20Antidepressants.%20Clin%20Pharmacol%20Ther.%202023;114:51%E2%80%9368.)

16. Kalman LV, Agúndez JAG, Appell ML, Black JL, Bell GC, Boukouvala S, et al. Pharmacogenetic allele nomenclature: International workgroup recommendations for test result reporting. Clin Pharmacol Ther. 2016;99:172–85.  [DOI](https://doi.org/10.1002/cpt.280) | [PMC free article](/articles/PMC4724253/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26479518/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Kalman%20LV,%20Ag%C3%BAndez%20JAG,%20Appell%20ML,%20Black%20JL,%20Bell%20GC,%20Boukouvala%20S,%20et%20al.%20Pharmacogenetic%20allele%20nomenclature:%20International%20workgroup%20recommendations%20for%20test%20result%20reporting.%20Clin%20Pharmacol%20Ther.%202016;99:172%E2%80%9385.)

17. Hodgson K, Tansey K, Dernovšek MZ, Hauser J, Henigsberg N, Maier W, et al. Genetic differences in cytochrome P450 enzymes and antidepressant treatment response. J Psychopharmacol (Oxf). 2014;28:133–41.  [DOI](https://doi.org/10.1177/0269881113512041) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24257813/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Hodgson%20K,%20Tansey%20K,%20Dernov%C5%A1ek%20MZ,%20Hauser%20J,%20Henigsberg%20N,%20Maier%20W,%20et%20al.%20Genetic%20differences%20in%20cytochrome%20P450%20enzymes%20and%20antidepressant%20treatment%20response.%20J%20Psychopharmacol%20(Oxf).%202014;28:133%E2%80%9341.)

18. Milosavljević F, Bukvić N, Pavlović Z, Miljević C, Pešić V, Molden E. et al. Association of CYP2C19 and CYP2D6 Poor and Intermediate Metabolizer Status With Antidepressant and Antipsychotic Exposure: A Systematic Review and Meta-analysis. JAMA Psychiatry. 2021;78:270–80.  [DOI](https://doi.org/10.1001/jamapsychiatry.2020.3643) | [PMC free article](/articles/PMC7702196/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33237321/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Milosavljevi%C4%87%20F,%20Bukvi%C4%87%20N,%20Pavlovi%C4%87%20Z,%20Miljevi%C4%87%20C,%20Pe%C5%A1i%C4%87%20V,%20Molden%20E.%20et%20al.%20Association%20of%20CYP2C19%20and%20CYP2D6%20Poor%20and%20Intermediate%20Metabolizer%20Status%20With%20Antidepressant%20and%20Antipsychotic%20Exposure:%20A%20Systematic%20Review%20and%20Meta-analysis.%20JAMA%20Psychiatry.%202021;78:270%E2%80%9380.)

19. Campos AI, Byrne EM, Mitchell BL, Wray NR, Lind PA, Licinio J, et al. Impact of CYP2C19 metaboliser status on SSRI response: a retrospective study of 9500 participants of the Australian Genetics of Depression Study. Pharmacogenomics J. 2022;22:130–5.  [DOI](https://doi.org/10.1038/s41397-022-00267-7) | [PMC free article](/articles/PMC8975743/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35094016/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Campos%20AI,%20Byrne%20EM,%20Mitchell%20BL,%20Wray%20NR,%20Lind%20PA,%20Licinio%20J,%20et%20al.%20Impact%20of%20CYP2C19%20metaboliser%20status%20on%20SSRI%20response:%20a%20retrospective%20study%20of%209500%20participants%20of%20the%20Australian%20Genetics%20of%20Depression%20Study.%20Pharmacogenomics%20J.%202022;22:130%E2%80%935.)

20. Carvalho Henriques B, Buchner A, Hu X, Wang Y, Yavorskyy V, Wallace K, et al. Methodology for clinical genotyping of CYP2D6 and CYP2C19. Transl Psychiatry. 2021;11:596.  [DOI](https://doi.org/10.1038/s41398-021-01717-9) | [PMC free article](/articles/PMC8608805/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34811360/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Carvalho%20Henriques%20B,%20Buchner%20A,%20Hu%20X,%20Wang%20Y,%20Yavorskyy%20V,%20Wallace%20K,%20et%20al.%20Methodology%20for%20clinical%20genotyping%20of%20CYP2D6%20and%20CYP2C19.%20Transl%20Psychiatry.%202021;11:596.)

21. Nofziger C, Turner AJ, Sangkuhl K, Whirl‐Carrillo M, Agúndez JAG, Black JL, et al. PharmVar GeneFocus: CYP2D6. Clin Pharmacol Ther. 2020;107:154–70.  [DOI](https://doi.org/10.1002/cpt.1643) | [PMC free article](/articles/PMC6925641/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31544239/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Nofziger%20C,%20Turner%20AJ,%20Sangkuhl%20K,%20Whirl%E2%80%90Carrillo%20M,%20Ag%C3%BAndez%20JAG,%20Black%20JL,%20et%20al.%20PharmVar%20GeneFocus:%20CYP2D6.%20Clin%20Pharmacol%20Ther.%202020;107:154%E2%80%9370.)

22. Pratt VM, Cavallari LH, Del Tredici AL, Gaedigk A, Hachad H, Ji Y, et al. Recommendations for Clinical CYP2D6 Genotyping Allele Selection: A Joint Consensus Recommendation of the Association for Molecular Pathology, College of American Pathologists, Dutch Pharmacogenetics Working Group of the Royal Dutch Pharmacists Association, and the European Society for Pharmacogenomics and Personalized Therapy. J Mol. Diagn. 2021;23:1047–64.  [DOI](https://doi.org/10.1016/j.jmoldx.2021.05.013) | [PMC free article](/articles/PMC8579245/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34118403/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Pratt%20VM,%20Cavallari%20LH,%20Del%20Tredici%20AL,%20Gaedigk%20A,%20Hachad%20H,%20Ji%20Y,%20et%20al.%20Recommendations%20for%20Clinical%20CYP2D6%20Genotyping%20Allele%20Selection:%20A%20Joint%20Consensus%20Recommendation%20of%20the%20Association%20for%20Molecular%20Pathology,%20College%20of%20American%20Pathologists,%20Dutch%20Pharmacogenetics%20Working%20Group%20of%20the%20Royal%20Dutch%20Pharmacists%20Association,%20and%20the%20European%20Society%20for%20Pharmacogenomics%20and%20Personalized%20Therapy.%20J%20Mol.%20Diagn.%202021;23:1047%E2%80%9364.)

23. Baffa A, Hohoff C, Baune BT, Müller-Tidow C, Tidow N, Freitag C, et al. Norepinephrine and Serotonin Transporter Genes: Impact on Treatment Response in Depression. Neuropsychobiology. 2010;62:121–31.  [DOI](https://doi.org/10.1159/000317285) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20588071/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Baffa%20A,%20Hohoff%20C,%20Baune%20BT,%20M%C3%BCller-Tidow%20C,%20Tidow%20N,%20Freitag%20C,%20et%20al.%20Norepinephrine%20and%20Serotonin%20Transporter%20Genes:%20Impact%20on%20Treatment%20Response%20in%20Depression.%20Neuropsychobiology.%202010;62:121%E2%80%9331.)

24. Baune BT, Hohoff C, Berger K, Neumann A, Mortensen S, Roehrs T, et al. Association of the COMT val158met Variant with Antidepressant Treatment Response in Major Depression. Neuropsychopharmacology. 2008;33:924–32.  [DOI](https://doi.org/10.1038/sj.npp.1301462) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17522626/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Baune%20BT,%20Hohoff%20C,%20Berger%20K,%20Neumann%20A,%20Mortensen%20S,%20Roehrs%20T,%20et%20al.%20Association%20of%20the%20COMT%20val158met%20Variant%20with%20Antidepressant%20Treatment%20Response%20in%20Major%20Depression.%20Neuropsychopharmacology.%202008;33:924%E2%80%9332.)

25. Biernacka JM, Sangkuhl K, Jenkins G, Whaley RM, Barman P, Batzler A, et al. The International SSRI Pharmacogenomics Consortium (ISPC): a genome-wide association study of antidepressant treatment response. Transl Psychiatry. 2015;5:e553.  [DOI](https://doi.org/10.1038/tp.2015.47) | [PMC free article](/articles/PMC4462610/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25897834/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Biernacka%20JM,%20Sangkuhl%20K,%20Jenkins%20G,%20Whaley%20RM,%20Barman%20P,%20Batzler%20A,%20et%20al.%20The%20International%20SSRI%20Pharmacogenomics%20Consortium%20(ISPC):%20a%20genome-wide%20association%20study%20of%20antidepressant%20treatment%20response.%20Transl%20Psychiatry.%202015;5:e553.)

26. Domschke K, Hohoff C, Mortensen LS, Roehrs T, Deckert J, Arolt V, et al. Monoamine oxidase A variant influences antidepressant treatment response in female patients with Major Depression. Prog Neuropsychopharmacol Biol Psychiatry. 2008;32:224–8.  [DOI](https://doi.org/10.1016/j.pnpbp.2007.08.011) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17884271/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Domschke%20K,%20Hohoff%20C,%20Mortensen%20LS,%20Roehrs%20T,%20Deckert%20J,%20Arolt%20V,%20et%20al.%20Monoamine%20oxidase%20A%20variant%20influences%20antidepressant%20treatment%20response%20in%20female%20patients%20with%20Major%20Depression.%20Prog%20Neuropsychopharmacol%20Biol%20Psychiatry.%202008;32:224%E2%80%938.)

27. Fabbri C, Kasper S, Kautzky A, Bartova L, Dold M, Zohar J, et al. Genome-wide association study of treatment-resistance in depression and meta-analysis of three independent samples. Br J Psychiatry. 2019;214:36–41.  [DOI](https://doi.org/10.1192/bjp.2018.256) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30468137/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Fabbri%20C,%20Kasper%20S,%20Kautzky%20A,%20Bartova%20L,%20Dold%20M,%20Zohar%20J,%20et%20al.%20Genome-wide%20association%20study%20of%20treatment-resistance%20in%20depression%20and%20meta-analysis%20of%20three%20independent%20samples.%20Br%20J%20Psychiatry.%202019;214:36%E2%80%9341.)

28. Garriock HA, Kraft JB, Shyn SI, Peters EJ, Yokoyama JS, Jenkins GD, et al. A Genome-Wide Association Study of Citalopram Response in Major Depressive Disorder. Biol Psychiatry. 2010;67:133–8.  [DOI](https://doi.org/10.1016/j.biopsych.2009.08.029) | [PMC free article](/articles/PMC2794921/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19846067/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Garriock%20HA,%20Kraft%20JB,%20Shyn%20SI,%20Peters%20EJ,%20Yokoyama%20JS,%20Jenkins%20GD,%20et%20al.%20A%20Genome-Wide%20Association%20Study%20of%20Citalopram%20Response%20in%20Major%20Depressive%20Disorder.%20Biol%20Psychiatry.%202010;67:133%E2%80%938.)

29. Mrazek DA, Biernacka JM, McAlpine DE, Benitez J, Karpyak VM, Williams MD, et al. Treatment Outcomes of Depression: The Pharmacogenomic Research Network Antidepressant Medication Pharmacogenomic Study. J Clin Psychopharmacol. 2014;34:313–7.  [DOI](https://doi.org/10.1097/JCP.0000000000000099) | [PMC free article](/articles/PMC3992481/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24743713/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Mrazek%20DA,%20Biernacka%20JM,%20McAlpine%20DE,%20Benitez%20J,%20Karpyak%20VM,%20Williams%20MD,%20et%20al.%20Treatment%20Outcomes%20of%20Depression:%20The%20Pharmacogenomic%20Research%20Network%20Antidepressant%20Medication%20Pharmacogenomic%20Study.%20J%20Clin%20Psychopharmacol.%202014;34:313%E2%80%937.)

30. Tansey KE, Guipponi M, Perroud N, Bondolfi G, Domenici E, Evans D, et al. Genetic Predictors of Response to Serotonergic and Noradrenergic Antidepressants in Major Depressive Disorder: A Genome-Wide Analysis of Individual-Level Data and a Meta-Analysis. PLoS Med. 2012; 9. 10.1371/journal.pmed.1001326.  [DOI](https://doi.org/10.1371/journal.pmed.1001326) | [PMC free article](/articles/PMC3472989/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23091423/)

31. Uher R, Perroud N, Ng MYM, Hauser J, Henigsberg N, Maier W, et al. Genome-wide pharmacogenetics of antidepressant response in the GENDEP project. Am J Psychiatry. 2010;167:555–64.  [DOI](https://doi.org/10.1176/appi.ajp.2009.09070932) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20360315/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Uher%20R,%20Perroud%20N,%20Ng%20MYM,%20Hauser%20J,%20Henigsberg%20N,%20Maier%20W,%20et%20al.%20Genome-wide%20pharmacogenetics%20of%20antidepressant%20response%20in%20the%20GENDEP%20project.%20Am%20J%20Psychiatry.%202010;167:555%E2%80%9364.)

32. Lam M, Awasthi S, Watson HJ, Goldstein J, Panagiotaropoulou G, Trubetskoy V, et al. RICOPILI: Rapid Imputation for COnsortias PIpeLIne. Bioinformatics. 2020;36:930–3.  [DOI](https://doi.org/10.1093/bioinformatics/btz633) | [PMC free article](/articles/PMC7868045/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31393554/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Lam%20M,%20Awasthi%20S,%20Watson%20HJ,%20Goldstein%20J,%20Panagiotaropoulou%20G,%20Trubetskoy%20V,%20et%20al.%20RICOPILI:%20Rapid%20Imputation%20for%20COnsortias%20PIpeLIne.%20Bioinformatics.%202020;36:930%E2%80%933.)

33. Delaneau O, Zagury J-F, Robinson MR, Marchini JL, Dermitzakis ET. Accurate, scalable and integrative haplotype estimation. Nat Commun. 2019;10:5436.  [DOI](https://doi.org/10.1038/s41467-019-13225-y) | [PMC free article](/articles/PMC6882857/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31780650/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Delaneau%20O,%20Zagury%20J-F,%20Robinson%20MR,%20Marchini%20JL,%20Dermitzakis%20ET.%20Accurate,%20scalable%20and%20integrative%20haplotype%20estimation.%20Nat%20Commun.%202019;10:5436.)

34. Okhuijsen-Pfeifer C, van der Horst MZ, Bousman CA, Lin B, van Eijk KR, Ripke S, et al. Genome-wide association analyses of symptom severity among clozapine-treated patients with schizophrenia spectrum disorders. Transl Psychiatry. 2022;12:145.  [DOI](https://doi.org/10.1038/s41398-022-01884-3) | [PMC free article](/articles/PMC8989876/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35393395/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Okhuijsen-Pfeifer%20C,%20van%20der%20Horst%20MZ,%20Bousman%20CA,%20Lin%20B,%20van%20Eijk%20KR,%20Ripke%20S,%20et%20al.%20Genome-wide%20association%20analyses%20of%20symptom%20severity%20among%20clozapine-treated%20patients%20with%20schizophrenia%20spectrum%20disorders.%20Transl%20Psychiatry.%202022;12:145.)

35. Caudle KE, Sangkuhl K, Whirl-Carrillo M, Swen JJ, Haidar CE, Klein TE, et al. Standardizing CYP2D6 Genotype to Phenotype Translation: Consensus Recommendations from the Clinical Pharmacogenetics Implementation Consortium and Dutch Pharmacogenetics Working Group. Clin Transl Sci. 2020;13:116–24.  [DOI](https://doi.org/10.1111/cts.12692) | [PMC free article](/articles/PMC6951851/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31647186/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Caudle%20KE,%20Sangkuhl%20K,%20Whirl-Carrillo%20M,%20Swen%20JJ,%20Haidar%20CE,%20Klein%20TE,%20et%20al.%20Standardizing%20CYP2D6%20Genotype%20to%20Phenotype%20Translation:%20Consensus%20Recommendations%20from%20the%20Clinical%20Pharmacogenetics%20Implementation%20Consortium%20and%20Dutch%20Pharmacogenetics%20Working%20Group.%20Clin%20Transl%20Sci.%202020;13:116%E2%80%9324.)

36. Whirl-Carrillo M, Huddart R, Gong L, Sangkuhl K, Thorn CF, Whaley R, et al. An Evidence-Based Framework for Evaluating Pharmacogenomics Knowledge for Personalized Medicine. Clin Pharmacol Ther. 2021;110:563–72.  [DOI](https://doi.org/10.1002/cpt.2350) | [PMC free article](/articles/PMC8457105/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34216021/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Whirl-Carrillo%20M,%20Huddart%20R,%20Gong%20L,%20Sangkuhl%20K,%20Thorn%20CF,%20Whaley%20R,%20et%20al.%20An%20Evidence-Based%20Framework%20for%20Evaluating%20Pharmacogenomics%20Knowledge%20for%20Personalized%20Medicine.%20Clin%20Pharmacol%20Ther.%202021;110:563%E2%80%9372.)

37. Pratt VM, Del Tredici AL, Hachad H, Ji Y, Kalman LV, Scott SA, et al. Recommendations for Clinical CYP2C19 Genotyping Allele Selection: A Report of the Association for Molecular Pathology. J Mol Diagn. 2018;20:269–76.  [DOI](https://doi.org/10.1016/j.jmoldx.2018.01.011) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29474986/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Pratt%20VM,%20Del%20Tredici%20AL,%20Hachad%20H,%20Ji%20Y,%20Kalman%20LV,%20Scott%20SA,%20et%20al.%20Recommendations%20for%20Clinical%20CYP2C19%20Genotyping%20Allele%20Selection:%20A%20Report%20of%20the%20Association%20for%20Molecular%20Pathology.%20J%20Mol%20Diagn.%202018;20:269%E2%80%9376.)

38. Calabrò M, Fabbri C, Kasper S, Zohar J, Souery D, Montgomery S, et al. Metabolizing status of CYP2C19 in response and side effects to medications for depression: Results from a naturalistic study. Eur Neuropsychopharmacol. 2022;56:100–11.  [DOI](https://doi.org/10.1016/j.euroneuro.2022.01.008) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35152032/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Calabr%C3%B2%20M,%20Fabbri%20C,%20Kasper%20S,%20Zohar%20J,%20Souery%20D,%20Montgomery%20S,%20et%20al.%20Metabolizing%20status%20of%20CYP2C19%20in%20response%20and%20side%20effects%20to%20medications%20for%20depression:%20Results%20from%20a%20naturalistic%20study.%20Eur%20Neuropsychopharmacol.%202022;56:100%E2%80%9311.)

39. Fabbri C, Tansey KE, Perlis RH, Hauser J, Henigsberg N, Maier W, et al. Effect of cytochrome CYP2C19 metabolizing activity on antidepressant response and side effects: Meta-analysis of data from genome-wide association studies. Eur Neuropsychopharmacol. 2018;28:945–54.  [DOI](https://doi.org/10.1016/j.euroneuro.2018.05.009) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30135031/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Fabbri%20C,%20Tansey%20KE,%20Perlis%20RH,%20Hauser%20J,%20Henigsberg%20N,%20Maier%20W,%20et%20al.%20Effect%20of%20cytochrome%20CYP2C19%20metabolizing%20activity%20on%20antidepressant%20response%20and%20side%20effects:%20Meta-analysis%20of%20data%20from%20genome-wide%20association%20studies.%20Eur%20Neuropsychopharmacol.%202018;28:945%E2%80%9354.)

40. Mrazek DA, Biernacka JM, O’kane DJ, Black JL, Cunningham JM, Drews MS, et al. CYP2C19 Variation and Citalopram Response. Pharmacogenet Genomics. 2011;21:1–9.  [DOI](https://doi.org/10.1097/fpc.0b013e328340bc5a) | [PMC free article](/articles/PMC3090085/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21192344/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Mrazek%20DA,%20Biernacka%20JM,%20O%E2%80%99kane%20DJ,%20Black%20JL,%20Cunningham%20JM,%20Drews%20MS,%20et%20al.%20CYP2C19%20Variation%20and%20Citalopram%20Response.%20Pharmacogenet%20Genomics.%202011;21:1%E2%80%939.)

41. Islam F, Marshe VS, Magarbeh L, Frey BN, Milev RV, Soares CN, et al. Effects of CYP2C19 and CYP2D6 gene variants on escitalopram and aripiprazole treatment outcome and serum levels: results from the CAN-BIND 1 study. Transl Psychiatry. 2022;12:366.  [DOI](https://doi.org/10.1038/s41398-022-02124-4) | [PMC free article](/articles/PMC9448818/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36068210/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Islam%20F,%20Marshe%20VS,%20Magarbeh%20L,%20Frey%20BN,%20Milev%20RV,%20Soares%20CN,%20et%20al.%20Effects%20of%20CYP2C19%20and%20CYP2D6%20gene%20variants%20on%20escitalopram%20and%20aripiprazole%20treatment%20outcome%20and%20serum%20levels:%20results%20from%20the%20CAN-BIND%201%20study.%20Transl%20Psychiatry.%202022;12:366.)

42. Joković D, Milosavljević F, Stojanović Z, Šupić G, Vojvodić D, Uzelac B, et al. CYP2C19 slow metabolizer phenotype is associated with lower antidepressant efficacy and tolerability. Psychiatry Res. 2022;312:114535.  [DOI](https://doi.org/10.1016/j.psychres.2022.114535) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35398660/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Jokovi%C4%87%20D,%20Milosavljevi%C4%87%20F,%20Stojanovi%C4%87%20Z,%20%C5%A0upi%C4%87%20G,%20Vojvodi%C4%87%20D,%20Uzelac%20B,%20et%20al.%20CYP2C19%20slow%20metabolizer%20phenotype%20is%20associated%20with%20lower%20antidepressant%20efficacy%20and%20tolerability.%20Psychiatry%20Res.%202022;312:114535.)

43. Brouwer JMJL, Nijenhuis M, Soree B, Guchelaar HJ, Swen JJ, van Schaik RHN, et al. Dutch Pharmacogenetics Working Group (DPWG) guideline for the gene-drug interaction between CYP2C19 and CYP2D6 and SSRIs. Eur J Hum Genet. 2021. 10.1038/s41431-021-01004-7.  [DOI](https://doi.org/10.1038/s41431-021-01004-7) | [PMC free article](/articles/PMC9553948/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34782755/)

44. Del Tredici AL, Malhotra A, Dedek M, Espin F, Roach D, Zhu G, et al. Frequency of CYP2D6 Alleles Including Structural Variants in the United States. Front Pharmacol. 2018;9:305.  [DOI](https://doi.org/10.3389/fphar.2018.00305) | [PMC free article](/articles/PMC5895772/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29674966/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Del%20Tredici%20AL,%20Malhotra%20A,%20Dedek%20M,%20Espin%20F,%20Roach%20D,%20Zhu%20G,%20et%20al.%20Frequency%20of%20CYP2D6%20Alleles%20Including%20Structural%20Variants%20in%20the%20United%20States.%20Front%20Pharmacol.%202018;9:305.)

45. Zhou Y, Ingelman-Sundberg M, Lauschke VM. Worldwide Distribution of Cytochrome P450 Alleles: A Meta-analysis of Population-scale Sequencing Projects. Clin Pharmacol Ther. 2017;102:688–700.  [DOI](https://doi.org/10.1002/cpt.690) | [PMC free article](/articles/PMC5600063/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28378927/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Zhou%20Y,%20Ingelman-Sundberg%20M,%20Lauschke%20VM.%20Worldwide%20Distribution%20of%20Cytochrome%20P450%20Alleles:%20A%20Meta-analysis%20of%20Population-scale%20Sequencing%20Projects.%20Clin%20Pharmacol%20Ther.%202017;102:688%E2%80%93700.)

46. Tsai M-H, Lin K-M, Hsiao M-C, Shen WW, Lu M-L, Tang H-S, et al. Genetic polymorphisms of cytochrome P450 enzymes influence metabolism of the antidepressant escitalopram and treatment response. Pharmacogenomics. 2010;11:537–46.  [DOI](https://doi.org/10.2217/pgs.09.168) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20350136/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Tsai%20M-H,%20Lin%20K-M,%20Hsiao%20M-C,%20Shen%20WW,%20Lu%20M-L,%20Tang%20H-S,%20et%20al.%20Genetic%20polymorphisms%20of%20cytochrome%20P450%20enzymes%20influence%20metabolism%20of%20the%20antidepressant%20escitalopram%20and%20treatment%20response.%20Pharmacogenomics.%202010;11:537%E2%80%9346.)

47. Lin S-K. Racial/Ethnic Differences in the Pharmacokinetics of Antipsychotics: Focusing on East Asians. J Pers Med. 2022;12:1362.  [DOI](https://doi.org/10.3390/jpm12091362) | [PMC free article](/articles/PMC9504618/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36143147/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Lin%20S-K.%20Racial/Ethnic%20Differences%20in%20the%20Pharmacokinetics%20of%20Antipsychotics:%20Focusing%20on%20East%20Asians.%20J%20Pers%20Med.%202022;12:1362.)

48. Zhou L, Sharma P, Yeo KR, Higashimori M, Xu H, Al-Huniti N, et al. Assessing pharmacokinetic differences in Caucasian and East Asian (Japanese, Chinese and Korean) populations driven by CYP2C19 polymorphism using physiologically-based pharmacokinetic modelling. Eur J Pharm Sci. 2019;139:105061.  [DOI](https://doi.org/10.1016/j.ejps.2019.105061) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31479720/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Zhou%20L,%20Sharma%20P,%20Yeo%20KR,%20Higashimori%20M,%20Xu%20H,%20Al-Huniti%20N,%20et%20al.%20Assessing%20pharmacokinetic%20differences%20in%20Caucasian%20and%20East%20Asian%20(Japanese,%20Chinese%20and%20Korean)%20populations%20driven%20by%20CYP2C19%20polymorphism%20using%20physiologically-based%20pharmacokinetic%20modelling.%20Eur%20J%20Pharm%20Sci.%202019;139:105061.)

49. McInnes G, Lavertu A, Sangkuhl K, Klein TE, Whirl-Carrillo M, Altman RB. Pharmacogenetics at Scale: An Analysis of the UK Biobank. Clin Pharmacol Ther. 2021;109:1528–37.  [DOI](https://doi.org/10.1002/cpt.2122) | [PMC free article](/articles/PMC8144239/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33237584/) | [Google Scholar](https://scholar.google.com/scholar_lookup?McInnes%20G,%20Lavertu%20A,%20Sangkuhl%20K,%20Klein%20TE,%20Whirl-Carrillo%20M,%20Altman%20RB.%20Pharmacogenetics%20at%20Scale:%20An%20Analysis%20of%20the%20UK%20Biobank.%20Clin%20Pharmacol%20Ther.%202021;109:1528%E2%80%9337.)
