# Genetic variants of NEUROD1 target genes are associated with clinical outcomes of small‐cell lung cancer patients

## Metadata
**Authors:** Sunwoong Lee, Seung Soo Yoo, Jin Eun Choi, Mi Jeong Hong, Sook Kyung Do, Jang Hyuck Lee, Won Ki Lee, Ji Eun Park, Sun Ha Choi, Hyewon Seo, Jaehee Lee, Shin Yup Lee, Seung Ick Cha, Chang Ho Kim, Hyo‐Gyoung Kang, Jae Yong Park
**Journal:** Thoracic Cancer
**Date:** 2023 Mar 19
**DOI:** [10.1111/1759-7714.14839](https://doi.org/10.1111/1759-7714.14839)
**PMID:** 36935366
**PMCID:** PMC10151137
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10151137/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC10151137/pdf/TCA-14-1145.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC10151137/pdf/TCA-14-1145.pdf)

## Abstract

**Background:** 
Neurogenic differentiation factor 1 (NEUROD1) is frequently overexpressed in small‐cell lung cancer (SCLC). NEUROD1 plays an important role in promoting malignant behavior and survival.

**Methods:** 
In this study, we evaluated the association between putative functional polymorphisms in 45 NEUROD1 target genes and chemotherapy response and survival outcomes in 261 patients with SCLC. Among the 100 single nucleotide polymorphisms (SNPs) studied, two were significantly associated with both chemotherapy response and overall survival (OS) of patients with SCLC.

**Results:** 
The SNP rs3806915C>A in semaphorin 6A (SEMA6A) gene was significantly associated with better chemotherapy response and OS (p = 0.04 and p = 0.04, respectively). The SNP rs11265375C>T in nescient helix–loop helix 1 (NHLH1) gene was also associated with better chemotherapy response and OS (p = 0.04 and p = 0.02, respectively). Luciferase assay showed a significantly higher promoter activity of SEMA6A with the rs3806915 A allele than C allele in H446 lung cancer cells (p = 4 × 10−6). The promoter activity of NHLH1 showed a significantly higher with the rs11265375 T allele than C allele (p = 0.001).

**Conclusion:** 
These results suggest that SEMA6A rs3806915C>A and NHLH1 rs11265375C>T polymorphisms affect the promoter activity and expression of the genes, which may affect the survival outcome of patients with SCLC.

Keywords: NEUROD1, NHLH1, SCLC, SEMA6A, SNP, Survival, Variant

### Background

Neurogenic differentiation factor 1 (NEUROD1) is frequently overexpressed in small‐cell lung cancer (SCLC). NEUROD1 plays an important role in promoting malignant behavior and survival.

### Methods

In this study, we evaluated the association between putative functional polymorphisms in 45 NEUROD1 target genes and chemotherapy response and survival outcomes in 261 patients with SCLC. Among the 100 single nucleotide polymorphisms (SNPs) studied, two were significantly associated with both chemotherapy response and overall survival (OS) of patients with SCLC.

### Results

The SNP rs3806915C>A in semaphorin 6A (*SEMA6A*) gene was significantly associated with better chemotherapy response and OS (*p* = 0.04 and *p* = 0.04, respectively). The SNP rs11265375C>T in nescient helix–loop helix 1 (*NHLH1*) gene was also associated with better chemotherapy response and OS (*p* = 0.04 and *p* = 0.02, respectively). Luciferase assay showed a significantly higher promoter activity of *SEMA6A* with the rs3806915 A allele than C allele in H446 lung cancer cells (*p* = 4 × 10^−6^). The promoter activity of *NHLH1* showed a significantly higher with the rs11265375 T allele than C allele (*p* = 0.001).

### Conclusion

These results suggest that *SEMA6A* rs3806915C>A and *NHLH1* rs11265375C>T polymorphisms affect the promoter activity and expression of the genes, which may affect the survival outcome of patients with SCLC.

## INTRODUCTION

Lung cancer remains the leading cause of cancer‐related deaths worldwide. In 2020, more than 2.2 million new cases and 1.8 million deaths because of lung cancer were recorded.[1](#tca14839-bib-0001) Lung cancer is largely divided into non–small‐cell lung cancer (NSCLC) and small‐cell lung cancer (SCLC), which account for ~85% and 15% of all cases, respectively. In the past 10 years, the 5‐year survival rate of lung cancer has increased from 16% to 21% owing to innovations in cancer treatment such as immunotherapy and targeted anticancer drugs.[1](#tca14839-bib-0001), [2](#tca14839-bib-0002) However, these novel therapeutics have shown significant beneficial effects in NSCLC, but not in SCLC. The 2‐year survival rate for extensive disease (ED) SCLC, which accounts for approximately 70% of SCLC cases, is only 8%.[1](#tca14839-bib-0001) Although the recent addition of immune checkpoint inhibitors (ICIs) to cytotoxic chemotherapy has improved the overall survival (OS) in ED‐SCLC,[3](#tca14839-bib-0003), [4](#tca14839-bib-0004) this is still marginal compared to the significant breakthroughs in the treatment of NSCLC. Therefore, there is a need to identify predictive biomarkers or develop new therapeutics to improve the survival outcomes of SCLC.

SCLC is a highly aggressive pulmonary neuroendocrine tumor characterized by rapid tumor growth, high vascularity, genomic instability, and early metastasis compared with NSCLC.[5](#tca14839-bib-0005) Our understanding of the biology and genomic alterations in SCLC has broadened over the past decade. The majority of SCLCs are characterized by inactivation of TP53 and RB1 tumor suppressor genes.[6](#tca14839-bib-0006) It was also known that MYC amplification, commonly found in SCLC, is related to short survival time.[7](#tca14839-bib-0007) With the advancements in cancer genetics, efforts are being made to further classify SCLCs from two subtypes (variant and classical) based on gene expression profiles. Many researchers have classified SCLC based on gene expression of achaete‐scute homologue 1 (ASCL1) and neurogenic differentiation factor 1 (NEUROD1)[8](#tca14839-bib-0008), [9](#tca14839-bib-0009), [10](#tca14839-bib-0010), [11](#tca14839-bib-0011) into three types as: ASCL1‐high, NEUROD1‐high, and double negative, or into four types by further dividing the double negatives.[8](#tca14839-bib-0008), [9](#tca14839-bib-0009), [10](#tca14839-bib-0010), [11](#tca14839-bib-0011)

The basic helix–loop–helix transcription factors ASCL1 and NEUROD1 play important roles in promoting malignant behavior and survival of SCLC.[8](#tca14839-bib-0008) ASCL1 is essential for neuroendocrine differentiation in the lungs and plays a crucial role in SCLC carcinogenesis.[12](#tca14839-bib-0012), [13](#tca14839-bib-0013) ASCL1 is expressed in ~75% of SCLCs and functions as a lineage‐specific oncogene.[6](#tca14839-bib-0006), [14](#tca14839-bib-0014) NEUROD1 is also critical for promoting neuronal differentiation and maturation.[15](#tca14839-bib-0015), [16](#tca14839-bib-0016) NEUROD1 is expressed in ~15% of SCLCs and is associated with the variant subtype.[6](#tca14839-bib-0006), [10](#tca14839-bib-0010) NEUROD1 is thought to promote tumor cell migration and therefore, contribute to metastasis in SCLC.[17](#tca14839-bib-0017) Its role as a regulatory hub in SCLC, through signaling molecules such as tyrosine kinase tropomyosin‐related kinase B and neural cell adhesion molecule, has been reported.[18](#tca14839-bib-0018) Therefore, ASCL1, NEUROD1, and their target genes are potential therapeutic targets for SCLC.[5](#tca14839-bib-0005), [14](#tca14839-bib-0014), [18](#tca14839-bib-0018)

In a previous study, we found that a polymorphism in dopa decarboxylase, an ASCL1 target gene, was associated with survival outcomes in patients with SCLC.[19](#tca14839-bib-0019) We hypothesized that functional polymorphisms in NEUROD1 target genes may also affect the clinical outcomes of patients with SCLC, as NEUROD1 plays a crucial role in SCLC carcinogenesis. To test this hypothesis, we evaluated the association between putative functional polymorphisms in 45 NEUROD1 target genes and the chemotherapy response and survival outcomes of patients with SCLC.

## RESULTS

### Patient characteristics

The baseline characteristics of the 261 patients are presented in Table [1](#tca14839-tbl-0001). The response rate to first line chemotherapy was 72.8% (95% confidence interval [CI], 67.4–78.2) and was higher with irinotecan‐cisplatin (IP) regimen than with etoposide‐cisplatin (EP) regimen (78.7% vs. 67.2%, *p* = 0.04). However, the OS did not differ between the regimens. The median survival time was 10.5 months (95% CI, 9.3–11.4). Younger age, limited‐stage disease, good performance status, low neuron‐specific enolase level, no weight loss, receiving second line chemotherapy, and radiation to the tumor were associated with better OS (Table [1](#tca14839-tbl-0001)). These variables were adjusted in subsequent studies to determine their association with the polymorphisms.

### TABLE 1.

| Variables | No. of cases | Response to chemotherapy |  |  |  | Overall survival |  |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Responders (CR + PR) a | Non‐responders (SD + PD) a | OR (95% CI) | p | MST (months) | 95% CI | Log‐rank p | HR (95% CI) | p |  |  |
| Overall | 261 | 190 (72.8) | 71 (27.2) | b |   | 10.5 | 9.3–11.4 |   |   |   |
| Age (years) |  |  |  |  |  |  |  |  |  |  |
| <68 | 129 | 100 (77.5) | 29 (22.5) | 1.00 |   | 11.8 | 11.0–13.5 |   | 1.00 |   |
| ≥68 | 132 | 90 (68.2) | 42 (31.8) | 0.62 (0.36–1.08) | 0.09 | 7.9 | 7.1–9.4 | 3 × 10−4 | 1.63 (1.25–2.13) | 3 × 10−4 |
| Gender |  |  |  |  |  |  |  |  |  |  |
| Male | 226 | 163 (72.1) | 63 (27.9) | 1.00 |   | 10.5 | 9.2–11.4 |   | 1.00 |   |
| Female | 35 | 27 (77.1) | 8 (22.9) | 1.30 (0.56–3.02) | 0.54 | 11.0 | 6.4–15.3 | 0.75 | 0.94 (0.63–1.40) | 0.75 |
| Smoking status |  |  |  |  |  |  |  |  |  |  |
| Never | 19 | 16 (84.2) | 3 (15.8) | 1.00 |   | 11.4 | 6.4–15.5 |   | 1.00 |   |
| Ever | 242 | 174 (71.9) | 68 (28.1) | 0.48 (0.14–1.70) | 0.26 | 10.3 | 9.2–11.3 | 0.82 | 1.06 (0.64–1.76) | 0.82 |
| Clinical stage |  |  |  |  |  |  |  |  |  |  |
| LD | 66 | 46 (69.7) | 20 (30.3) | 1.00 |   | 13.0 | 10.7–15.5 |   | 1.00 |   |
| ED | 195 | 144 (73.8) | 51 (26.2) | 1.23 (0.66–2.27) | 0.51 | 9.6 | 8.2–10.9 | 0.001 | 1.67 (1.21–2.30) | 2 × 10−3 |
| ECOG |  |  |  |  |  |  |  |  |  |  |
| 0–1 | 215 | 162 (75.4) | 53 (24.6) | 1.00 |   | 10.9 | 10.0–11.7 |   | 1.00 |   |
| 2 | 46 | 28 (60.9) | 18 (39.1) | 0.51 (0.26–0.99) | 0.05 | 7.2 | 4.3–9.2 | 3 × 10−4 | 1.82 (1.31–2.53) | 4 × 10−4 |
| NSE |  |  |  |  |  |  |  |  |  |  |
| <14.7 | 96 | 66 (68.8) | 30 (31.2) | 1 |   | 11.4 | 10.2–14.0 |   | 1.00 |   |
| ≥14.7 | 147 | 109 (74.2) | 38 (25.8) | 1.30 (0.74–2.30) | 0.36 | 9.4 | 7.7–10.5 | 0.02 | 1.41 (1.07–1.87) | 0.02 |
| Weight loss |  |  |  |  |  |  |  |  |  |  |
| No | 185 | 139 (75.1) | 46 (24.9) | 1.00 |   | 11.2 | 10.2–12.2 |   | 1.00 |   |
| Yes | 76 | 51 (67.1) | 25 (32.9) | 0.68 (0.38–1.21) | 0.19 | 8.1 | 7.2–10.2 | 0.01 | 1.42 (1.07–1.89) | 0.01 |
| Chemotherapy regimen |  |  |  |  |  |  |  |  |  |  |
| EP | 134 | 90 (67.2) | 44 (32.8) | 1.00 |   | 10.9 | 9.0–12.4 |   | 1.00 |   |
| IP | 127 | 100 (78.7) | 27 (21.3) | 1.81 (1.04–3.16) | 0.04 | 10.2 | 8.9–11.4 | 0.88 | 0.98 (0.75–1.28) | 0.88 |
| 2nd line chemotherapy |  |  |  |  |  |  |  |  |  |  |
| Yes | 140 |   |   |   |   | 12.2 | 11.2–13.7 |   | 1.00 |   |
| No | 121 |   |   |   |   | 7.2 | 6.2–8.4 | 1 × 10−5 | 1.80 (1.38–2.35) | 1 × 10−5 |
| Radiation to tumor |  |  |  |  |  |  |  |  |  |  |
| Yes | 34 |   |   |   |   | 16.7 | 13.0–30.8 |   | 1.00 |   |
| No | 227 |   |   |   |   | 9.6 | 8.2–10.8 | 1 × 10−5 | 3.02 (1.78–5.10) | 1 × 10−5 |

Table 1 Caption: Univariate analysis for response to chemotherapy and overall survival by clinical variables.

### Association between single nucleotide polymorphisms and treatment outcomes

Among the 100 single nucleotide polymorphisms (SNPs) evaluated, two showed significant association with both chemotherapy response and OS. *SEMA6A* rs3806915C>A was significantly associated with better chemotherapy response and OS (under a codominant model, adjusted odds ratio [aOR], 1.74; 95% CI, 1.02–2.95; *p* = 0.04, and aHR, 0.78; 95% CI, 0.62–0.99; *p* = 0.04, respectively) (Table [2](#tca14839-tbl-0002) and Figure [1](#tca14839-fig-0001)). *NHLH1* rs11265375C>T was also significantly associated with better chemotherapy response and OS (under a dominant model, aOR, 1.95; 95% CI, 1.04–3.65; *p* = 0.04, and aHR, 0.70; 95% CI, 0.52–0.95; *p* = 0.02, respectively) (Table [2](#tca14839-tbl-0002) and Figure [1](#tca14839-fig-0001)).

### TABLE 2.

| Gene polymorphism | Genotype | Responders (%) a | Non‐responders (%) a | OR (95% CI) b | p b | No. of cases (%) c | L‐R‐P | HR (95% CI) d | p d |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| SEMA6A rs3806915C>A | CC | 116 (70.3) | 49 (29.7) | 1.00 |   | 165 (64.0) | 0.11 | 1.00 |   |
| CA | 58 (74.4) | 20 (25.6) | 1.34 (0.70–2.57) | 0.38 | 78 (30.2) |   | 0.87 (0.63–1.19) | 0.37 |  |
| AA | 14 (93.3) | 2 (6.7) | 7.77 (0.97–62.55) | 0.05 | 15 (5.8) |   | 0.50 (0.25–0.97) | 0.04 |  |
| Dominant |   |   | 1.63 (0.87–3.07) | 0.13 |   | 0.10 | 0.78 (0.58–1.05) | 0.11 |  |
| Recessive |   |   | 7.03 (0.88–55.88) | 0.07 |   | 0.07 | 0.52 (0.27–1.00) | 0.05 |  |
| Codominant |   |   | 1.74 (1.02–2.95) | 0.04 |   |   | 0.78 (0.62–0.99) | 0.04 |  |
| NHLH1 rs11265375C>T | CC | 98 (66.2) | 50 (33.8) | 1.00 |   | 148 (58.1) | 0.22 | 1.00 |   |
| CT | 70 (82.4) | 15 (17.6) | 2.16 (1.08–4.32) | 0.03 | 85 (33.3) |   | 0.62 (0.45–0.87) | 0.005 |  |
| TT | 17 (77.3) | 5 (22.73) | 1.34 (0.44–4.12) | 0.61 | 22 (8.6) |   | 1.11 (0.66–1.87) | 0.70 |  |
| Dominant |   |   | 1.95 (1.04–3.65) | 0.04 |   | 0.09 | 0.70 (0.52–0.95) | 0.02 |  |
| Recessive |   |   | 1.06 (0.35–3.20) | 0.92 |   | 0.90 | 1.28 (0.76–2.15) | 0.35 |  |
| Codominant |   |   | 1.51 (0.92–2.48) | 0.10 |   |   | 0.84 (0.66–1.07) | 0.16 |  |

Table 2 Caption: The association between SEMA6A rs3806915C>A and NHLH1 rs11265375C>T and the response to chemotherapy and overall survival.

### FIGURE 1.

![FIGURE 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/30f8/10151137/32132895967e/TCA-14-1145-g002.jpg)

Kaplan–Meier curves for overall survival according to polymorphisms (a) SEMA6A rs3806915C>A and (b) NHLH1 rs11265375C>T. p values were calculated using multivariate Cox proportional hazard models (rs3806915C>A under a codominant model and rs11265375C>T under a dominant model).

### Effect of SNPs on the promoter activity of SEMA6A and NHLH1

The SNP rs3806915C>A is located in the *SEMA6A* promoter region (−1621 base pairs [bp] from the transcription start site). We performed a luciferase assay to assess the effect of rs3806915C>A on *SEMA6A* promoter activity. Promoter activity was significantly higher for the rs3806915 A allele than for the rs3806915 C allele in H446 lung cancer cells (*p* = 4 × 10^−6^) (Figure [2](#tca14839-fig-0002)).

### FIGURE 2.

![FIGURE 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/30f8/10151137/1f54b6e7cd52/TCA-14-1145-g003.jpg)

Relative luciferase activity according to polymorphisms. The effect of (a) SEMA6A rs3806915C>A and (b) NHLH1 rs11265375C>T genotypes on the promoter activity of the respective gene in H446 lung cancer cells. Data are presented as mean ± standard error of mean. p values are based on a t‐test.

SNP rs11265375C>A is located in the first intron of the *NHLH1* gene. However, based on the high chromatin accessibility (as measured by DNase I hypersensitivity)[20](#tca14839-bib-0020) and strong signal for active histone markers (H3K4Me3 and H3K27Ac)[21](#tca14839-bib-0021) at the chromosomal position of rs11265375C>A in the University of California Santa Cruz (UCSC) genome browser, rs11265375C>A was predicted to affect promoter activity (Figure [3](#tca14839-fig-0003)). The luciferase assay also showed a significantly higher promoter activity for the rs11265375 T allele than for the rs11265375 C allele in H446 lung cancer cells (*p* = 0.001) (Figure [2](#tca14839-fig-0002)).

### FIGURE 3.

![FIGURE 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/30f8/10151137/ddea48bbab45/TCA-14-1145-g001.jpg)

Bioinformatics annotation of NHLH1 promoter region using the University of California Santa Cruz (UCSC) genome browser. (a) UCSC genome browser view of chromosome 1q23.2 with data from the transcription factor ChIP‐seq, DNase 1 hypersensitivity, histone modifications from the ENCODE project. The histone modification tracks show the level of enrichment of the histone marks across the genome as determined by a ChIP‐seq assay using the seven cell lines of the ENCODE project. The next track shows DNase I hypersensitivity clusters. The last track represents transcription factor ChIP‐seq clusters (338 factors from 130 cell types) from ENCODE 3. The gray box encloses each peak cluster of transcription factor occupancy: the darkness of the box is proportional to the maximum signal strength observed in any cell type contributing to the cluster. (b) Definitions of track colors are listed.

## DISCUSSION

In this study, we investigated the association between genetic variants of NEUROD1 target genes and the clinical outcomes of patients with SCLC. We found that two SNPs, *SEMA6A* rs3806915C>A and *NHLH1* rs11265375C>T, were significantly associated with both chemotherapy response and OS in patients with SCLC. Additionally, we found that the promoter activity of each gene was significantly higher in the variant allele than in the wild‐type allele in in vitro functional studies.

*SEMA6A* is a member of the semaphorin family, which is known to regulate cell motility and attachment during axon guidance, vascular growth, immune cell regulation, and tumor progression.[22](#tca14839-bib-0022) *SEMA6A* has been proposed to be a prognostic biomarker that reduces cancer cell proliferation, migration, and invasion in glioblastoma.[23](#tca14839-bib-0023) Recently, a few studies have analyzed the role of *SEMA6A* in lung cancer.[24](#tca14839-bib-0024), [25](#tca14839-bib-0025) Chen et al.[24](#tca14839-bib-0024) reported that overexpression of *SEMA6A* decreases lung cancer cell migration and suggested the role of *SEMA6A* in inhibition of cancer cell migration. Shen et al.[25](#tca14839-bib-0025) showed that overexpression of *SEMA6A* reduces the proliferation of lung cancer cells and increases the rate of apoptosis. As rs3806915C>A is located in the *SEMA6A* promoter region (−1621 bp from the transcription start site), it may alter the promoter activity of *SEMA6A*. Results of the luciferase assay revealed that the promoter activity of *SEMA6A* was higher for the rs3806915 A allele than for the C allele. Furthermore, we found that *SEMA6A* rs3806915C>A was significantly associated with better chemotherapy response and OS. This is consistent with the results of the aforementioned studies. *SEMA6A* rs3806915C>A increases the promoter activity of *SEMA6A*, thereby increasing the expression of *SEMA6A*, which in turn reduces lung cancer cell migration and proliferation and increases apoptosis, leading to better OS. Dhanabal et al.[26](#tca14839-bib-0026) reported that recombinant *SEMA6A*‐1 soluble extracellular domain inhibits growth factor and tumor‐induced angiogenesis in vivo, suggesting the potential therapeutic role of *SEMA6A*. Therefore, *SEMA6A* represents an attractive therapeutic target for treating lung cancer.

*NHLH1*, also known as *HEN1* and *NSCL1*, encodes helix–loop–helix protein 1, which plays a role in the growth and development of a wide variety of tissues, particularly in regulating neurogenesis.[27](#tca14839-bib-0027), [28](#tca14839-bib-0028) Misexpression of *NSCL1* leads to abnormal brain development in chicks.[29](#tca14839-bib-0029) *NHLH1* has been reported to be associated with neuroblastoma and medulloblastoma.[30](#tca14839-bib-0030), [31](#tca14839-bib-0031) However, its association with other cancers is unknown. In this study, *NHLH1* rs11265375C>T was found to be significantly associated with chemotherapy and OS in patients with SCLC. *NHLH1* rs11265375C>T is located in an intron of *NHLH1*. As technological advances in sequencing have expanded our understanding of the genome, it has become clear that introns are not merely junk DNA and that variants in introns can also affect gene expression.[32](#tca14839-bib-0032), [33](#tca14839-bib-0033) The SNP rs11265375C>T was predicted to affect *NHLH1* promoter activity in the UCSC genome browser. A luciferase assay confirmed that the variant allele had higher *NHLH1* promoter activity than the wild‐type allele. Although the role of NEUROD1 in SCLC and as an upstream regulator of *NHLH1* (aliases of *NSCL1*) is known, the direct relationship between *NHLH1* and SCLC remains unknown.[6](#tca14839-bib-0006), [17](#tca14839-bib-0017), [18](#tca14839-bib-0018), [34](#tca14839-bib-0034) Further research is required to clarify this in the future.

ICIs are therapeutic agents that are revolutionizing the treatment of lung cancer, especially NSCLC. ICIs have also brought about a paradigm shift in the treatment methods for SCLC. In recent studies, the combined use of ICI with conventional platinum doublet chemotherapy was shown to extend the median OS in patients with ED‐SCLC from ~10 months to 12 to 13 months.[3](#tca14839-bib-0003), [4](#tca14839-bib-0004) As our study included patients from before ICIs were introduced as a standard treatment for SCLC, it is not known how these polymorphisms affect the clinical outcomes of patients with SCLC treated with ICI‐combination therapy. Therefore, it would be interesting to study the effects of these two polymorphisms in patients who receive ICI‐combination therapy.

This study has some limitations. First, all the patients enrolled in this study were of Korean descent; therefore, caution should be exercised in generalizing the results of this study to other ethnic groups. The frequency of SNPs varies between races and may have different effects; therefore, validation in different ethnic groups is necessary. In addition, although the variant alleles affected the promoter activity of the respective genes in the lung cancer cells, the effect of the variants on gene expression could not be confirmed in actual SCLC tissues. Unlike NSCLC, SCLC tissues are difficult to obtain because they are rarely resectable. Therefore, previous studies on *SEMA6A* [25](#tca14839-bib-0025), [26](#tca14839-bib-0026) were also performed on NSCLC tissues or lung cancer cell lines.

In summary, we investigated the effect of genetic variants of NEUROD1 target genes on clinical outcomes in patients with SCLC. *SEMA6A* rs3806915C>A and *NHLH1* rs11265375C>T were significantly associated with better chemotherapy response and OS. Functional studies suggested that these SNPs may influence clinical outcomes in patients with SCLC by affecting promoter activity and gene expression.

## METHODS

### Study population

The study population has been described in our previous study.[19](#tca14839-bib-0019) Briefly, 261 patients diagnosed with SCLC between 1997 and 2017 at the Kyungpook National University Hospital (KNUH) who received at least two cycles of the EP regimen or the IP regimen chemotherapy as first line treatment were enrolled. Patients treated with concurrent chemoradiotherapy were excluded because radiotherapy may affect the evaluation of chemotherapy response. Patients who received radiation therapy after chemotherapy were included. Treatment was discontinued in case of disease progression or major toxicity, or as determined by the patient or physician. Chemotherapy response was assessed after every two cycles of treatment by computed tomography using the Response Evaluation Criteria in Solid Tumors. Patients displaying complete or partial response to first‐line chemotherapy were classified as responders, and those with stable or progressive disease were classified as non‐responders.

This study was approved by the Institutional Review Board of the KNUH. Blood samples for genotyping were provided by the National Biobank of Korea‐KNUH, which is supported by the Ministry of Health, Welfare, and Family Affairs (approval no. KNUCH 2020‐03‐040). All blood samples were obtained before the first chemotherapy session. Informed consent was obtained from all subjects or their legal guardians. All methods were performed in accordance with relevant guidelines and regulations.

### Selection of SNPs and genotyping

We selected 45 NEUROD1 target genes by searching public databases and related articles. We collected 33 917 SNPs using a public database ([http://www.ncbi.nlm.nih.gov/SNP](http://www.ncbi.nlm.nih.gov/SNP)). To identify potentially functional polymorphisms, we used FuncPred utility for functional SNP prediction in the SNPinfo web server ([https://snpinfo.niehs.nih.gov/](https://snpinfo.niehs.nih.gov/)). After excluding SNPs with low minor allele frequencies (≤0.1 by HapMap‐JPT data), 180 potentially functional SNPs were collected. Using the TagSNP utility for linkage disequilibrium (LD)‐tagged SNP selection, 59 LD polymorphisms (r2 ≥ 0.8) were excluded, and the remaining 121 SNPs were prepared for genotyping. We designed primers of 28plex at the multiplex level and excluded 10 SNPs during the primer combination. A three‐step polymerase chain reaction (PCR) was performed for the remaining 111 SNPs. Genotyping was performed using Sequenom MassARRAY iPLEX assay (Sequenom) following the manufacturer's instructions. Of the 111 SNPs, 100 SNPs (excluding 11 with call rates <95% or *p* value for Hardy–Weinberg equilibrium [HWE] <0.05) in the NEUROD1 target genes were processed for statistical analysis (Table [S1](#tca14839-supitem-0001)).

### Promoter‐luciferase constructs and luciferase assay

To verify the functional relevance of the two genetic variants, we investigated whether rs3806915C>A and rs11265375C>T regulate the promoter activity of *SEMA6A* and NHLH1, respectively.

A 1754 bp fragment (from −1403 to +351 bp based on the transcription start site) that included rs3806915C>A was synthesized by PCR using genomic DNA from a donor carrying a heterozygote. The *SEMA6A* genomic sequence was used as the PCR template, and the pGL3‐basic genomic sequence was used as the PCR template primers. The sequences of the primers used were as follows: Insert_fwd: 5′‐ttctctatcgataCGAGGCTGGCTCTTGAAGCC‐3′, Insert_rev: 5′‐agagctcggtaccTCTGCGCCGATTAACAAGTCATTTC‐3′, pGL3‐basic_fwd: 5′‐aatcggcgcagaGGTACCGAGCTCTTACGCGTG‐3′ and pGL3‐basic_rev: 5′‐gagccagcctcgTATCGATAGAGAAATGTTCTGGCACC‐3′ (the overlapping sequences of vectors and inserts are indicated by lowercase letters). PCR products were assembled into the pGL3‐basic‐*SEMA6A* construct containing the rs3806915 C or A allele using the NEBuilder™ HiFi DNA Assembly Master Mix Kit (New England Biolabs), according to the manufacturer's instructions. A 398 bp fragment (from +173 to +571 bp based on the transcription start site) that included rs11265375C>T was synthesized by PCR using genomic DNA from a donor carrying a heterozygote. The forward primer with the Kpn I restriction site (5′‐CGGGGTACCCTAGAAAGCTGGTCACTAAC‐3′) and reverse primer with the Xho I restriction site (5′‐CCGCTCGAGGCAGCAGCTTCTATTTACCC‐3′) were used. The PCR products were cloned into the Kpn I/Xho I site of the pGL3‐basic vector (Promega), resulting in pGL3‐basic‐NHLH1 constructs containing either rs11265375 C or T alleles. All constructs were verified by genome sequencing before use.

H446 lung cancer cells were transfected with the pRL‐SV40 vector (Promega) and the pGL3‐basic vector using Lipofectamine™ (Qiagen). The cells were harvested 48 hours following transfection, and lysates were prepared using the Dual‐Luciferase Reporter Assay System (Promega). Luciferase activity was measured using a Synergy HTX Multi‐Mode Microplate Reader (BioTek Instruments), and the activity was normalized to that of pRL‐SV40 Renilla luciferase activity.

### Statistical analysis

The Statistical Analysis System version 9.2 for Windows (SAS Institute) software was used for statistical analysis. Response to chemotherapy was analyzed as the proportion of responders and non‐responders based on clinical variables and genotypes. OS was defined as the period from the day of the first chemotherapy to the date of patient death or last follow‐up. The estimated OS based on the clinical variables and genotypes was analyzed using the log‐rank test and Kaplan–Meier method. Adjusted hazard ratios (aHR) and 95% CIs were calculated for the multivariate statistical models (Cox proportional hazards models). Adjustment variables were as follows: age, sex, smoking status, clinical stage, Eastern Cooperative Oncology Group performance status, weight loss, chemotherapy regimen, second line chemotherapy, neuron‐specific enolase, and radiation to the tumor.

## AUTHOR CONTRIBUTIONS

*Conceived and designed the experiments*: Hyo‐Gyoung Kang and Jae Yong Park. *Performed the experiments*: Sunwoong Lee, Hyo‐Gyoung Kang, Jin Eun Choi, Mi Jeung Hong, Sook Kyung Do, and Jang Hyuck Lee. *Acquired clinical data*: Seung Soo Yoo, Won Ki Lee, Ji Eun Park, Sun Ha Choi, Hyewon Seo, Jaehee Lee, ShinYup Lee, Seung Ick Cha, Chang Ho Kim, and Jae Yong Park. *Analyzed and interpreted the data*: Sunwoong Lee, Seung Soo Yoo, Hyo‐Gyoung Kang, and Jae Yong Park. *Wrote the main manuscript text*: Sunwoong Lee, Seung Soo Yoo, Hyo‐Gyoung Kang, and Jae Yong Park. *Supervised the study*: Hyo‐Gyoung Kang, and Jae Yong Park. All authors reviewed the manuscript.

## FUNDING INFORMATION

Kyungpook National University Development Project Research Fund 2020.

## CONFLICT OF INTEREST STATEMENT

The authors declare no competing interests.

## Supporting information

## ACKNOWLEDGMENTS

This research was supported by the Kyungpook National University Development Project Research Fund, 2020.

Lee S, Yoo SS, Choi JE, Hong MJ, Do SK, Lee JH, et al. Genetic variants of NEUROD1 target genes are associated with clinical outcomes of small‐cell lung cancer patients. Thorac Cancer. 2023;14(13):1145–1152. 10.1111/1759-7714.14839

## Contributor Information

Hyo‐Gyoung Kang, Email: pearlblue0@gmail.com.

Jae Yong Park, Email: jaeyong@knu.ac.kr.

## DATA AVAILABILITY STATEMENT

All data generated or analyzed during this study are included in this published article and its supplementary information files.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

All data generated or analyzed during this study are included in this published article and its supplementary information files.

### Supplementary Materials

### Data Availability Statement

All data generated or analyzed during this study are included in this published article and its supplementary information files.

## References

1. Siegel RL, Miller KD, Fuchs HE, Jemal A. Cancer statistics, 2021. CA Cancer J Clin. 2021;71:7–33.  [DOI](https://doi.org/10.3322/caac.21654) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33433946/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=CA%20Cancer%20J%20Clin&title=Cancer%20statistics,%202021&author=RL%20Siegel&author=KD%20Miller&author=HE%20Fuchs&author=A%20Jemal&volume=71&publication_year=2021&pages=7-33&pmid=33433946&doi=10.3322/caac.21654&)

2. Siegel R, Ward E, Brawley O, Jemal A. Cancer statistics, 2011: the impact of eliminating socioeconomic and racial disparities on premature cancer deaths. CA Cancer J Clin. 2011;61:212–36.  [DOI](https://doi.org/10.3322/caac.20121) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21685461/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=CA%20Cancer%20J%20Clin&title=Cancer%20statistics,%202011:%20the%20impact%20of%20eliminating%20socioeconomic%20and%20racial%20disparities%20on%20premature%20cancer%20deaths&author=R%20Siegel&author=E%20Ward&author=O%20Brawley&author=A%20Jemal&volume=61&publication_year=2011&pages=212-36&pmid=21685461&doi=10.3322/caac.20121&)

3. Horn L, Mansfield AS, Szczęsna A, Havel L, Krzakowski M, Hochmair MJ, et al. First‐line Atezolizumab plus chemotherapy in extensive‐stage small‐cell lung cancer. N Engl J Med. 2018;379:2220–9.  [DOI](https://doi.org/10.1056/NEJMoa1809064) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30280641/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=First%E2%80%90line%20Atezolizumab%20plus%20chemotherapy%20in%20extensive%E2%80%90stage%20small%E2%80%90cell%20lung%20cancer&author=L%20Horn&author=AS%20Mansfield&author=A%20Szcz%C4%99sna&author=L%20Havel&author=M%20Krzakowski&volume=379&publication_year=2018&pages=2220-9&pmid=30280641&doi=10.1056/NEJMoa1809064&)

4. Goldman JW, Dvorkin M, Chen Y, Reinmuth N, Hotta K, Trukhin D, et al. Durvalumab, with or without tremelimumab, plus platinum‐etoposide versus platinum‐etoposide alone in first‐line treatment of extensive‐stage small‐cell lung cancer (CASPIAN): updated results from a randomised, controlled, open‐label, phase 3 trial. Lancet Oncol. 2021;22:51–65.  [DOI](https://doi.org/10.1016/S1470-2045(20)30539-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33285097/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet%20Oncol&title=Durvalumab,%20with%20or%20without%20tremelimumab,%20plus%20platinum%E2%80%90etoposide%20versus%20platinum%E2%80%90etoposide%20alone%20in%20first%E2%80%90line%20treatment%20of%20extensive%E2%80%90stage%20small%E2%80%90cell%20lung%20cancer%20(CASPIAN):%20updated%20results%20from%20a%20randomised,%20controlled,%20open%E2%80%90label,%20phase%203%20trial&author=JW%20Goldman&author=M%20Dvorkin&author=Y%20Chen&author=N%20Reinmuth&author=K%20Hotta&volume=22&publication_year=2021&pages=51-65&pmid=33285097&doi=10.1016/S1470-2045(20)30539-8&)

5. Gazdar AF, Bunn PA, Minna JD. Small‐cell lung cancer: what we know, what we need to know and the path forward. Nat Rev Cancer. 2017;17:765.  [DOI](https://doi.org/10.1038/nrc.2017.106) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29123245/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Cancer&title=Small%E2%80%90cell%20lung%20cancer:%20what%20we%20know,%20what%20we%20need%20to%20know%20and%20the%20path%20forward&author=AF%20Gazdar&author=PA%20Bunn&author=JD%20Minna&volume=17&publication_year=2017&pages=765&pmid=29123245&doi=10.1038/nrc.2017.106&)

6. George J, Lim JS, Jang SJ, Cun Y, Ozretić L, Kong G, et al. Comprehensive genomic profiles of small cell lung cancer. Nature. 2015;524:47–53.  [DOI](https://doi.org/10.1038/nature14664) | [PMC free article](/articles/PMC4861069/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26168399/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Comprehensive%20genomic%20profiles%20of%20small%20cell%20lung%20cancer&author=J%20George&author=JS%20Lim&author=SJ%20Jang&author=Y%20Cun&author=L%20Ozreti%C4%87&volume=524&publication_year=2015&pages=47-53&pmid=26168399&doi=10.1038/nature14664&)

7. Alves Rde C, Meurer RT, Roehe AV. MYC amplification is associated with poor survival in small cell lung cancer: a chromogenic in situ hybridization study. J Cancer Res Clin Oncol. 2014;140:2021–5.  [DOI](https://doi.org/10.1007/s00432-014-1769-1) | [PMC free article](/articles/PMC11824018/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25012251/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Cancer%20Res%20Clin%20Oncol&title=MYC%20amplification%20is%20associated%20with%20poor%20survival%20in%20small%20cell%20lung%20cancer:%20a%20chromogenic%20in%20situ%20hybridization%20study&author=C%20Alves%20Rde&author=RT%20Meurer&author=AV%20Roehe&volume=140&publication_year=2014&pages=2021-5&pmid=25012251&doi=10.1007/s00432-014-1769-1&)

8. Borromeo MD, Savage TK, Kollipara RK, He M, Augustyn A, Osborne JK, et al. ASCL1 and NEUROD1 reveal heterogeneity in pulmonary neuroendocrine tumors and regulate distinct genetic programs. Cell Rep. 2016;16:1259–72.  [DOI](https://doi.org/10.1016/j.celrep.2016.06.081) | [PMC free article](/articles/PMC4972690/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27452466/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Rep&title=ASCL1%20and%20NEUROD1%20reveal%20heterogeneity%20in%20pulmonary%20neuroendocrine%20tumors%20and%20regulate%20distinct%20genetic%20programs&author=MD%20Borromeo&author=TK%20Savage&author=RK%20Kollipara&author=M%20He&author=A%20Augustyn&volume=16&publication_year=2016&pages=1259-72&pmid=27452466&doi=10.1016/j.celrep.2016.06.081&)

9. Mollaoglu G, Guthrie MR, Böhm S, Brägelmann J, Can I, Ballieu PM, et al. MYC drives progression of small cell lung cancer to a variant neuroendocrine subtype with vulnerability to Aurora kinase inhibition. Cancer Cell. 2017;31:270–85.  [DOI](https://doi.org/10.1016/j.ccell.2016.12.005) | [PMC free article](/articles/PMC5310991/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28089889/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Cell&title=MYC%20drives%20progression%20of%20small%20cell%20lung%20cancer%20to%20a%20variant%20neuroendocrine%20subtype%20with%20vulnerability%20to%20Aurora%20kinase%20inhibition&author=G%20Mollaoglu&author=MR%20Guthrie&author=S%20B%C3%B6hm&author=J%20Br%C3%A4gelmann&author=I%20Can&volume=31&publication_year=2017&pages=270-85&pmid=28089889&doi=10.1016/j.ccell.2016.12.005&)

10. Rudin CM, Poirier JT, Byers LA, Dive C, Dowlati A, George J, et al. Molecular subtypes of small cell lung cancer: a synthesis of human and mouse model data. Nat Rev Cancer. 2019;19:289–97.  [DOI](https://doi.org/10.1038/s41568-019-0133-9) | [PMC free article](/articles/PMC6538259/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30926931/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Cancer&title=Molecular%20subtypes%20of%20small%20cell%20lung%20cancer:%20a%20synthesis%20of%20human%20and%20mouse%20model%20data&author=CM%20Rudin&author=JT%20Poirier&author=LA%20Byers&author=C%20Dive&author=A%20Dowlati&volume=19&publication_year=2019&pages=289-97&pmid=30926931&doi=10.1038/s41568-019-0133-9&)

11. Baine MK, Hsieh MS, Lai WV, Egger JV, Jungbluth AA, Daneshbod Y, et al. SCLC subtypes defined by ASCL1, NEUROD1, POU2F3, and YAP1: a comprehensive immunohistochemical and histopathologic characterization. J Thorac Oncol. 2020;15:1823–35.  [DOI](https://doi.org/10.1016/j.jtho.2020.09.009) | [PMC free article](/articles/PMC8362797/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33011388/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Thorac%20Oncol&title=SCLC%20subtypes%20defined%20by%20ASCL1,%20NEUROD1,%20POU2F3,%20and%20YAP1:%20a%20comprehensive%20immunohistochemical%20and%20histopathologic%20characterization&author=MK%20Baine&author=MS%20Hsieh&author=WV%20Lai&author=JV%20Egger&author=AA%20Jungbluth&volume=15&publication_year=2020&pages=1823-35&pmid=33011388&doi=10.1016/j.jtho.2020.09.009&)

12. Borges M, Linnoila RI, van de Velde HJK, Chen H, Nelkin BD, Mabry M, et al. An achaete‐scute homologue essential for neuroendocrine differentiation in the lung. Nature. 1997;386:852–5.  [DOI](https://doi.org/10.1038/386852a0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9126746/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=An%20achaete%E2%80%90scute%20homologue%20essential%20for%20neuroendocrine%20differentiation%20in%20the%20lung&author=M%20Borges&author=RI%20Linnoila&author=HJK%20van%20de%20Velde&author=H%20Chen&author=BD%20Nelkin&volume=386&publication_year=1997&pages=852-5&pmid=9126746&doi=10.1038/386852a0&)

13. Jiang T, Collins BJ, Jin N, Watkins DN, Brock MV, Matsui W, et al. Achaete‐scute complex homologue 1 regulates tumor‐initiating capacity in human small cell lung cancer. Cancer Res. 2009;69:845–54.  [DOI](https://doi.org/10.1158/0008-5472.CAN-08-2762) | [PMC free article](/articles/PMC2919317/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19176379/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=Achaete%E2%80%90scute%20complex%20homologue%201%20regulates%20tumor%E2%80%90initiating%20capacity%20in%20human%20small%20cell%20lung%20cancer&author=T%20Jiang&author=BJ%20Collins&author=N%20Jin&author=DN%20Watkins&author=MV%20Brock&volume=69&publication_year=2009&pages=845-54&pmid=19176379&doi=10.1158/0008-5472.CAN-08-2762&)

14. Augustyn A, Borromeo M, Wang T, Fujimoto J, Shao C, Dospoy PD, et al. ASCL1 is a lineage oncogene providing therapeutic targets for high‐grade neuroendocrine lung cancers. Proc Natl Acad Sci U S A. 2014;111:14788–93.  [DOI](https://doi.org/10.1073/pnas.1410419111) | [PMC free article](/articles/PMC4205603/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25267614/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&title=ASCL1%20is%20a%20lineage%20oncogene%20providing%20therapeutic%20targets%20for%20high%E2%80%90grade%20neuroendocrine%20lung%20cancers&author=A%20Augustyn&author=M%20Borromeo&author=T%20Wang&author=J%20Fujimoto&author=C%20Shao&volume=111&publication_year=2014&pages=14788-93&pmid=25267614&doi=10.1073/pnas.1410419111&)

15. Pang ZP, Yang N, Vierbuchen T, Ostermeier A, Fuentes DR, Yang TQ, et al. Induction of human neuronal cells by defined transcription factors. Nature. 2011;476:220–3.  [DOI](https://doi.org/10.1038/nature10202) | [PMC free article](/articles/PMC3159048/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21617644/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Induction%20of%20human%20neuronal%20cells%20by%20defined%20transcription%20factors&author=ZP%20Pang&author=N%20Yang&author=T%20Vierbuchen&author=A%20Ostermeier&author=DR%20Fuentes&volume=476&publication_year=2011&pages=220-3&pmid=21617644&doi=10.1038/nature10202&)

16. Pataskar A, Jung J, Smialowski P, Noack F, Calegari F, Straub T, et al. NeuroD1 reprograms chromatin and transcription factor landscapes to induce the neuronal program. EMBO J. 2016;35:24–45.  [DOI](https://doi.org/10.15252/embj.201591206) | [PMC free article](/articles/PMC4718003/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26516211/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=EMBO%20J&title=NeuroD1%20reprograms%20chromatin%20and%20transcription%20factor%20landscapes%20to%20induce%20the%20neuronal%20program&author=A%20Pataskar&author=J%20Jung&author=P%20Smialowski&author=F%20Noack&author=F%20Calegari&volume=35&publication_year=2016&pages=24-45&pmid=26516211&doi=10.15252/embj.201591206&)

17. Ikematsu Y, Tanaka K, Toyokawa G, Ijichi K, Ando N, Yoneshima Y, et al. NEUROD1 is highly expressed in extensive‐disease small cell lung cancer and promotes tumor cell migration. Lung Cancer. 2020;146:97–104.  [DOI](https://doi.org/10.1016/j.lungcan.2020.05.012) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32526603/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lung%20Cancer&title=NEUROD1%20is%20highly%20expressed%20in%20extensive%E2%80%90disease%20small%20cell%20lung%20cancer%20and%20promotes%20tumor%20cell%20migration&author=Y%20Ikematsu&author=K%20Tanaka&author=G%20Toyokawa&author=K%20Ijichi&author=N%20Ando&volume=146&publication_year=2020&pages=97-104&pmid=32526603&doi=10.1016/j.lungcan.2020.05.012&)

18. Osborne JK, Larsen JE, Shields MD, Gonzales JX, Shames DS, Sato M, et al. NeuroD1 regulates survival and migration of neuroendocrine lung carcinomas via signaling molecules TrkB and NCAM. Proc Natl Acad Sci U S A. 2013;110:6524–9.  [DOI](https://doi.org/10.1073/pnas.1303932110) | [PMC free article](/articles/PMC3631659/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23553831/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&title=NeuroD1%20regulates%20survival%20and%20migration%20of%20neuroendocrine%20lung%20carcinomas%20via%20signaling%20molecules%20TrkB%20and%20NCAM&author=JK%20Osborne&author=JE%20Larsen&author=MD%20Shields&author=JX%20Gonzales&author=DS%20Shames&volume=110&publication_year=2013&pages=6524-9&pmid=23553831&doi=10.1073/pnas.1303932110&)

19. Kim JH, Lee SY, Choi JE, do SK, Lee JH, Hong MJ, et al. Polymorphism in ASCL1 target gene DDC is associated with clinical outcomes of small cell lung cancer patients. Thorac Cancer. 2020;11:19–28.  [DOI](https://doi.org/10.1111/1759-7714.13212) | [PMC free article](/articles/PMC6938757/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31691490/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Thorac%20Cancer&title=Polymorphism%20in%20ASCL1%20target%20gene%20DDC%20is%20associated%20with%20clinical%20outcomes%20of%20small%20cell%20lung%20cancer%20patients&author=JH%20Kim&author=SY%20Lee&author=JE%20Choi&author=SK%20do&author=JH%20Lee&volume=11&publication_year=2020&pages=19-28&pmid=31691490&doi=10.1111/1759-7714.13212&)

20. Wang YM, Zhou P, Wang LY, Li ZH, Zhang YN, Zhang YX, Correlation between DNase I hypersensitive site distribution and gene expression in HeLa S3 cells. 2012, e42414.  [DOI](https://doi.org/10.1371/journal.pone.0042414) | [PMC free article](/articles/PMC3416863/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22900019/)

21. Birney E, Stamatoyannopoulos JA, Dutta A, Guigó R, Gingeras TR, Margulies EH, et al. Identification and analysis of functional elements in 1% of the human genome by the ENCODE pilot project. Nature 2007;447:799–816.  [DOI](https://doi.org/10.1038/nature05874) | [PMC free article](/articles/PMC2212820/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17571346/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Identification%20and%20analysis%20of%20functional%20elements%20in%201%%20of%20the%20human%20genome%20by%20the%20ENCODE%20pilot%20project&author=E%20Birney&author=JA%20Stamatoyannopoulos&author=A%20Dutta&author=R%20Guig%C3%B3&author=TR%20Gingeras&volume=447&publication_year=2007&pmid=17571346&doi=10.1038/nature05874&)

22. Kruger RP, Aurandt J, Guan KL. Semaphorins command cells to move. Nat Rev Mol Cell Biol. 2005;6:789–800.  [DOI](https://doi.org/10.1038/nrm1740) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16314868/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Mol%20Cell%20Biol&title=Semaphorins%20command%20cells%20to%20move&author=RP%20Kruger&author=J%20Aurandt&author=KL%20Guan&volume=6&publication_year=2005&pages=789-800&pmid=16314868&doi=10.1038/nrm1740&)

23. Zhao J, Tang H, Zhao H, Che W, Zhang L, Liang P. SEMA6A is a prognostic biomarker in glioblastoma. Tumour Biol. 2015;36:8333–40.  [DOI](https://doi.org/10.1007/s13277-015-3584-y) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26014517/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Tumour%20Biol&title=SEMA6A%20is%20a%20prognostic%20biomarker%20in%20glioblastoma&author=J%20Zhao&author=H%20Tang&author=H%20Zhao&author=W%20Che&author=L%20Zhang&volume=36&publication_year=2015&pages=8333-40&pmid=26014517&doi=10.1007/s13277-015-3584-y&)

24. Chen LH, Liao CY, Lai LC, Tsai MH, Chuang EY. Semaphorin 6A attenuates the migration capability of lung cancer cells via the NRF2/HMOX1 Axis. Sci Rep. 2019;9:13302.  [DOI](https://doi.org/10.1038/s41598-019-49874-8) | [PMC free article](/articles/PMC6746772/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31527696/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sci%20Rep&title=Semaphorin%206A%20attenuates%20the%20migration%20capability%20of%20lung%20cancer%20cells%20via%20the%20NRF2/HMOX1%20Axis&author=LH%20Chen&author=CY%20Liao&author=LC%20Lai&author=MH%20Tsai&author=EY%20Chuang&volume=9&publication_year=2019&pages=13302&pmid=31527696&doi=10.1038/s41598-019-49874-8&)

25. Shen C‐Y, Chang YC, Chen LH, Lin WC, Lee YH, Yeh ST, et al. The extracellular SEMA domain attenuates intracellular apoptotic signaling of semaphorin 6A in lung cancer cells. Oncogenesis. 2018;7:95–5.  [DOI](https://doi.org/10.1038/s41389-018-0105-z) | [PMC free article](/articles/PMC6281666/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30518871/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncogenesis&title=The%20extracellular%20SEMA%20domain%20attenuates%20intracellular%20apoptotic%20signaling%20of%20semaphorin%206A%20in%20lung%20cancer%20cells&author=C%E2%80%90Y%20Shen&author=YC%20Chang&author=LH%20Chen&author=WC%20Lin&author=YH%20Lee&volume=7&publication_year=2018&pages=95-5&pmid=30518871&doi=10.1038/s41389-018-0105-z&)

26. Dhanabal M, Wu F, Alvarez E, McQueeney KD, Jeffers M, MacDougall J, et al. Recombinant semaphorin 6A‐1 ectodomain inhibits in vivo growth factor and tumor cell line‐induced angiogenesis. Cancer Biol Ther. 2005;4:659–68.  [DOI](https://doi.org/10.4161/cbt.4.6.1733) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15917651/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Biol%20Ther&title=Recombinant%20semaphorin%206A%E2%80%901%20ectodomain%20inhibits%20in%20vivo%20growth%20factor%20and%20tumor%20cell%20line%E2%80%90induced%20angiogenesis&author=M%20Dhanabal&author=F%20Wu&author=E%20Alvarez&author=KD%20McQueeney&author=M%20Jeffers&volume=4&publication_year=2005&pages=659-68&pmid=15917651&doi=10.4161/cbt.4.6.1733&)

27. Bao J, Talmage DA, Role LW, Gautier J. Regulation of neurogenesis by interactions between HEN1 and neuronal LMO proteins. Development. 2000;127:425–35.  [DOI](https://doi.org/10.1242/dev.127.2.425) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10603358/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Development&title=Regulation%20of%20neurogenesis%20by%20interactions%20between%20HEN1%20and%20neuronal%20LMO%20proteins&author=J%20Bao&author=DA%20Talmage&author=LW%20Role&author=J%20Gautier&volume=127&publication_year=2000&pages=425-35&pmid=10603358&doi=10.1242/dev.127.2.425&)

28. Begley CG, Lipkowitz S, Göbel V, Mahon KA, Bertness V, Green AR, et al. Molecular characterization of NSCL, a gene encoding a helix‐loop‐helix protein expressed in the developing nervous system. Proc Natl Acad Sci USA. 1992;89:38–42.  [DOI](https://doi.org/10.1073/pnas.89.1.38) | [PMC free article](/articles/PMC48170/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1729708/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20USA&title=Molecular%20characterization%20of%20NSCL,%20a%20gene%20encoding%20a%20helix%E2%80%90loop%E2%80%90helix%20protein%20expressed%20in%20the%20developing%20nervous%20system&author=CG%20Begley&author=S%20Lipkowitz&author=V%20G%C3%B6bel&author=KA%20Mahon&author=V%20Bertness&volume=89&publication_year=1992&pages=38-42&pmid=1729708&doi=10.1073/pnas.89.1.38&)

29. Li CM, Yan RT, Wang SZ. Misexpression of a bHLH gene, cNSCL1, results in abnormal brain development. Dev Dyn. 1999;215:238–47.  [DOI](https://doi.org/10.1002/(SICI)1097-0177(199907)215:3<238::AID-AJA6>3.0.CO;2-F) | [PMC free article](/articles/PMC1764459/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10398534/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Dev%20Dyn&title=Misexpression%20of%20a%20bHLH%20gene,%20cNSCL1,%20results%20in%20abnormal%20brain%20development&author=CM%20Li&author=RT%20Yan&author=SZ%20Wang&volume=215&publication_year=1999&pages=238-47&pmid=10398534&doi=10.1002/(SICI)1097-0177(199907)215:3<238::AID-AJA6>3.0.CO;2-F&)

30. Brown L, Espinosa R 3rd, le Beau MM, Siciliano MJ, Baer R. HEN1 and HEN2: a subgroup of basic helix‐loop‐helix genes that are coexpressed in a human neuroblastoma. Proc Natl Acad Sci USA. 1992;89:8492–6.  [DOI](https://doi.org/10.1073/pnas.89.18.8492) | [PMC free article](/articles/PMC49946/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1528853/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20USA&title=HEN1%20and%20HEN2:%20a%20subgroup%20of%20basic%20helix%E2%80%90loop%E2%80%90helix%20genes%20that%20are%20coexpressed%20in%20a%20human%20neuroblastoma&author=L%20Brown&author=R%20Espinosa&author=MM%20le%20Beau&author=MJ%20Siciliano&author=R%20Baer&volume=89&publication_year=1992&pages=8492-6&pmid=1528853&doi=10.1073/pnas.89.18.8492&)

31. De Smaele E, Fragomeli C, Ferretti E, Pelloni M, Po A, Canettieri G, et al. An integrated approach identifies Nhlh1 and Insm1 as sonic hedgehog‐regulated genes in developing cerebellum and medulloblastoma. Neoplasia. 2008;10:89–98.  [DOI](https://doi.org/10.1593/neo.07891) | [PMC free article](/articles/PMC2213903/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18231642/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neoplasia&title=An%20integrated%20approach%20identifies%20Nhlh1%20and%20Insm1%20as%20sonic%20hedgehog%E2%80%90regulated%20genes%20in%20developing%20cerebellum%20and%20medulloblastoma&author=E%20De%20Smaele&author=C%20Fragomeli&author=E%20Ferretti&author=M%20Pelloni&author=A%20Po&volume=10&publication_year=2008&pages=89-98&pmid=18231642&doi=10.1593/neo.07891&)

32. Dunham I, Kundaje A, Aldred SF, Collins PJ, Davis CA, Doyle F. An integrated encyclopedia of DNA elements in the human genome. Nature. 2012;489:57–74.  [DOI](https://doi.org/10.1038/nature11247) | [PMC free article](/articles/PMC3439153/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22955616/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=An%20integrated%20encyclopedia%20of%20DNA%20elements%20in%20the%20human%20genome&author=I%20Dunham&author=A%20Kundaje&author=SF%20Aldred&author=PJ%20Collins&author=CA%20Davis&volume=489&publication_year=2012&pages=57-74&pmid=22955616&doi=10.1038/nature11247&)

33. Jo BS, Choi SS. Introns: the functional benefits of introns in genomes. Genomics Inform. 2015;13:112–8.  [DOI](https://doi.org/10.5808/GI.2015.13.4.112) | [PMC free article](/articles/PMC4742320/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26865841/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genomics%20Inform&title=Introns:%20the%20functional%20benefits%20of%20introns%20in%20genomes&author=BS%20Jo&author=SS%20Choi&volume=13&publication_year=2015&pages=112-8&pmid=26865841&doi=10.5808/GI.2015.13.4.112&)

34. Kim WY. NeuroD1 is an upstream regulator of NSCL1. Biochem Biophys Res Commun. 2012;419:27–31.  [DOI](https://doi.org/10.1016/j.bbrc.2012.01.100) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22310718/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20Biophys%20Res%20Commun&title=NeuroD1%20is%20an%20upstream%20regulator%20of%20NSCL1&author=WY%20Kim&volume=419&publication_year=2012&pages=27-31&pmid=22310718&doi=10.1016/j.bbrc.2012.01.100&)
