# A genome-wide approach to identify genetic variants that contribute to etoposide-induced cytotoxicity

## Metadata
**Authors:** R Stephanie Huang, Shiwei Duan, Wasim K Bleibel, Emily O Kistner, Wei Zhang, Tyson A Clark, Tina X Chen, Anthony C Schweitzer, John E Blume, Nancy J Cox, M Eileen Dolan
**Journal:** Proceedings of the National Academy of Sciences of the United States of America
**Date:** 2007 May 30
**DOI:** [10.1073/pnas.0703736104](https://doi.org/10.1073/pnas.0703736104)
**PMID:** 17537913
**PMCID:** PMC1887589
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1887589/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC1887589/pdf/zpq9758.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC1887589/pdf/zpq9758.pdf)

## Abstract

Large interindividual variance has been observed in sensitivity to drugs. To comprehensively decipher the genetic contribution to these variations in drug susceptibility, we present a genome-wide model using human lymphoblastoid cell lines from the International HapMap consortium, of which extensive genotypic information is available, to identify genetic variants that contribute to chemotherapeutic agent-induced cytotoxicity. Our model integrated genotype, gene expression, and sensitivity of HapMap cell lines to drugs. Cell lines derived from 30 trios of European descent (Center d'Etude du Polymorphisme Humain population) and 30 trios of African descent (Yoruban population) were used. Cell growth inhibition at increasing concentrations of etoposide for 72 h was determined by using alamarBlue assay. Gene expression on 176 HapMap cell lines (87 Center d'Etude du Polymorphisme Humain population and 89 Yoruban population) was determined by using the Affymetrix GeneChip Human Exon 1.0ST Array. We evaluated associations between genotype and cytotoxicity, genotype and gene expression and correlated gene expression of the identified candidates with cytotoxicity. The analysis identified 63 genetic variants that contribute to etoposide-induced toxicity through their effect on gene expression. These include genes that may play a role in cancer (AGPAT2, IL1B, and WNT5B) and genes not yet known to be associated with sensitivity to etoposide. This unbiased method can be used to elucidate genetic variants contributing to a wide range of cellular phenotypes induced by chemotherapeutic agents.

Keywords: HapMap, pharmacogenomics, toxicity, whole-genome association

## Results

### Cell Cytotoxicity.

Using the alamarBlue cytotoxicity assay, 87 and 89 cell lines derived from Center d'Etude du Polymorphisme Humain population (CEU) and Yoruban population (YRI) trios were exposed to increasing concentrations of etoposide (0.02–2.5 μM) for 72 h. Although our intention was to evaluate 90 CEU and 90 YRI lines, two CEU and one YRI cell lines failed to reach 85% viability on the experiment day on more than three attempts and therefore were not further evaluated. Additionally, one CEU cell line (GM12236) was not available from Coriell at the time of phenotyping. Similar dose-dependent etoposide cell growth inhibition was observed in cell lines from both populations. Interindividual variation in the IC_50_ was 433- and 222-fold in CEU and YRI cell lines, respectively ([Fig. 2](#F2)). The median IC_50_ in CEU and YRI cell lines was 0.43 and 0.40 μM, respectively ([5](#B5)).

### Fig. 2.

![Fig. 2.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ea6f/1887589/72dfc497c953/zpq0230765970002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=1887589_zpq0230765970002.jpg)

Frequency distribution of Box–Cox transformed etoposide IC50 in the CEU (n = 87) and YRI (n = 89).

### Quantitative Transmission Disequilibrium Test (QTDT) Genotype–Cytotoxicity Association.

SNPs (387,417), which represent 22,667 genes, were evaluated for their association with etoposide IC_50_. This covered ≈85% annotated genes of the whole human genome. Using an arbitrary *P* value threshold (*P* ≤ 0.0001), 49, 122, and 51 SNPs were found to significantly associate with etoposide IC_50_ in the combined, CEU and YRI and individual CEU and YRI, respectively. The binomial tests between the probability of our significant findings and the probability of random discovery showed *P* = 0.046, <10^−8^, 0.024 in the combined, CEU, and YRI, respectively, which indicates our association test results are not likely to be random effects. These SNPs were located in or within 10 kb up- and downstream of 26, 35, and 22 genes ([Table 1](#T1)). A list of these SNPs can be found in [supporting information (SI) Table 3](http://www.pnas.org/cgi/content/full/0703736104/DC1). Gene ontology (GO) analysis performed on these significant genes in comparison to 22,667 total genes tested indicated that these genes are enriched in cell organization and biogenesis, endocytosis, cell adhesion, intracellular protein transport, intracellular signaling, and cell differentiation process ([Table 2](#T2)).

### Table 1.

| Model | Combined populations | CEU | YRI |
| --- | --- | --- | --- |
| SNP associated with etoposide IC50 (P ≤ 0.0001) | 49 SNPs (in or near 26 genes) | 122 SNPs (in or near 35 genes) | 51 SNPs (in or near 22 genes) |
| SNP associated with etoposide IC50 and with gene expression (Bonferroni corrected P < 0.05) | 7 trans-acting events (6 genes) | 2 cis- and 132 trans-acting events (21 genes) | 45 trans-acting events (40 genes) |
| Gene expression correlated with etoposide IC50(P < 0.05) | 3 genes (associated with 4 SNPs) | 18 genes (associated with 54 SNPs) | 24 genes (associated with 6 SNPs) |

Table 1 Caption: Three-way model significant results in combined, CEU, or YRI

### Table 2.

| Gene symbol* | GO biological function | Corrected P value |
| --- | --- | --- |
| Determined from association analysis between genotype and IC50 |  |  |
| FHOD3, FMN2 | Cell organization and biogenesis | 2 × 10−4 |
| GATA3, PCDH15 | Sensory perception of sound | 5.5 × 10−3 |
| DNM3, AP4S1 | Endocytosis | 5.9 × 10−3 |
| FBN1, PCDH15, GRM8 | Visual perception | 6.6 × 10−3 |
| FHOD3, FMN2 | Actin cytoskeleton organization and biogenesis | 6.6 × 10−3 |
| PCDH15, CDH2 | Homophilic cell adhesion | 7.7 × 10−3 |
| DANJC6, PTPRD | Protein amino acid dephosphorylation | 7.8 × 10−3 |
| SPON1, PCDH15, CNTN5, HNT, CDH2 | Cell adhesion | 8.3 × 10−3 |
| STX18, AP4S1 | Intracellular protein transport | 9.3 × 10−3 |
| AP4S1, SORBS2, NUP205, SLC35C2 | Transport | 9.7 × 10−3 |
| KCNN3, SLC24A4 | Potassium ion transport | 9.9 × 10−3 |
| GATA3, KLF13 | Transcription from RNA polymerase II promoter | 0.01 |
| KCNN3, GRM8 | Synaptic transmission | 0.01 |
| ADCY2, FMN2, KSR2 | Intracellular signaling cascade | 0.01 |
| SPON1, FBN1, FMN2 | Development | 0.02 |
| KCNN3, SLIT1 | Nervous system development | 0.03 |
| SLIT1, BMP7 | Cell differentiation | 0.03 |
| SLC24A4, GRID1 | Ion transport | 0.03 |
| Determined from association analysis between genotype and expression |  |  |
| CAPNS1, CAPN1, TCIRG1, POU3F2 | Positive regulation of cell proliferation | 2.3 × 10−4 |
| TCIRG1, ATP13A1, ATP2A3 | Proton transport | 2.4 × 10−4 |
| TUBA1, C9orf48 | Microtubule-based movement | 6.3 × 10−3 |
| SLC27A4, ATP13A1, ATP2A3, AGPAT2 | Metabolism | 7.6 × 10−3 |
| ATP13A1, ATP2A3 | Cation transport | 0.01 |
| GBP2, GBP4, GBP7, NOTCH1 | Immune response | 0.01 |
| IGSF8, ACTN4 | Cell motility | 0.01 |
| CASP10, CAPN1, PEPD, ANPEP | Proteolysis | 0.01 |
| RECQL4, WNT5B, HOXB9, PTMA | Development | 0.01 |
| RECQL4, UHRF1 | DNA repair | 0.02 |
| NEU3, PYGB | Carbohydrate metabolism | 0.03 |
| NOTCH1, ANPEP | Cell differentiation | 0.03 |
| PTMA, IL1B | Regulation of progression through cell cycle | 0.03 |
| IGSF8, IL1B | Cell proliferation | 0.03 |
| IL1B, TNFRSF6B | Apoptosis | 0.04 |

Table 2 Caption: Biological process of GO for host and target genes obtained from association studies of genotype and etoposide IC50 or of genotype and gene expression for combined, CEU, and YRI

### QTDT Genotype and Gene Expression Association.

To obtain equally enriched gene expression data, we generated expression data by using the Affymetrix GeneChip Human Exon 1.0 ST Array (exon array). This exon array contains ≈1.4 million probe sets designed to represent all known and predicted exon regions within the human genome (Build 34). Gene expression analysis was performed on 176 lymphoblastoid cell lines (LCLs) (87 CEU and 89 YRI) by using the exon array. The QTDT association analysis was conducted between gene expression and the SNPs that were significantly associated with etoposide IC_50_ ([Table 2](#T2)). After normalization and robust multiarray average summary by the core sets of exon probes, we obtained gene/transcript cluster signal intensity ranging from 4 to 13. Only 14,722 genes that had the mean sample gene expression intensity of greater than five, indicating expression in both CEU and YRI, were included in the analysis. We found 7 trans-acting relationships in the combined populations, 2 cis- and 132 trans-acting relationships in CEU, and 45 trans-acting relationships in YRI (Bonferroni corrected *P* < 0.05; [Table 1](#T1)). These cis–trans-acting relationships involved 7 SNPs located in or within 10 kb up- or downstream of 3 genes and associated with 6 genes in the combined population; 56 SNPs located in or within 10 kb up- or downstream of 9 genes and associated with 21 genes in CEU; and 6 SNPs located in or within 10 kb up- or downstream of 5 genes and associated with 40 genes in YRI ([SI Table 3](http://www.pnas.org/cgi/content/full/0703736104/DC1)). We identified 45 SNPs in *SLC2A9* gene (chromosome 4) that were significantly associated with gene expression of *CAPNS1* (chromosome 19), *GBP7*/*GBP2*/*GBP4* (chromosome 1), *NOTCH1* (chromosome 9), *MBD5* (chromosome 2), and *CWF19L1*/ *BLOC1S2* (chromosome 10) in CEU ([Fig. 4](#F4)). We use the host genes to refer to the genes hosting the SNPs of cis- or trans-acting regulations and target genes to refer to the genes whose mRNA levels are regulated by the cis- or trans-acting regulators. The GO test on 65 target genes of >14,722 reference genes showed that these target genes are enriched in biological processes, including cell proliferation regulation, cell metabolism, development, cell motility, DNA repair, proteolysis, and apoptosis ([Table 2](#T2)).

### Fig. 4.

![Fig. 4.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ea6f/1887589/2f1aa51549b8/zpq0230765970004.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=1887589_zpq0230765970004.jpg)

Relationship between SNPs located in SLC2A9 gene, gene expression and etoposide IC50 in CEU population. (A) SLC2A9 gene conservation in 17 other species (obtained from University of California Santa Cruz Genome Browser on Human March 2006 Assembly; http://genome.ucsc.edu). The circle indicates the highly conserved region. (B) HapMap LD plot of 45 significant SNPs associated with etoposide IC50 and gene expression. The black lines show significant associations between SNPs in SLC2A9 and potential target genes.

### Linear Regression of Gene Expression and Etoposide IC50.

We examined the correlation between gene expression and etoposide IC_50_ by using a general linear model that was constructed to reflect the trio relationship in our data. Six gene expressions were evaluated in the combined population. Three had significant correlation to etoposide IC_50_ (*P* < 0.05). They were *TSPAN7*, *CAPNS1*, and *AGPAT2* ([Fig. 3](#F3) and [SI Table 3](http://www.pnas.org/cgi/content/full/0703736104/DC1)). We also found 18 and 24 genes whose expression significantly correlated to etoposide IC_50_ in CEU and YRI, respectively (*P* < 0.05; [SI Table 3](http://www.pnas.org/cgi/content/full/0703736104/DC1)). We identified 4, 54, and 6 SNPs that were significantly associated with etoposide IC_50_ through regulation of 3, 18, and 24 gene expressions in the combined CEU and YRI, respectively. After taking into consideration linkage disequilibrium (LD), our final findings consist of 3, 7, and 5 representative SNPs that significantly associated with 3, 18, and 24 gene expressions. One example was the significant association between the genotype of rs446112 (located in intron 1 of *ZNF663* gene on chromosome 20) and etoposide IC_50_ (*P* = 5 × 10^−5^). This SNP was associated with the expression of *AGPAT2* gene (located on chromosome 9, *P* = 2 × 10^−6^), whose expression significantly correlated to etoposide IC_50_ (*P* = 0.03; [Fig. 3](#F3)). In the individual CEU population, we identified 45 significant SNPs (located in *SLC2A9* gene on chromosome 4 and in high LD) whose genotypes were associated with etoposide IC_50_ and expression of 8 genes ([Fig. 4](#F4)). The expression of these 8 genes was significantly correlated with etoposide IC_50_ ([SI Table 3](http://www.pnas.org/cgi/content/full/0703736104/DC1)). Also in the CEU population, we identified rs6539870 (located in the 5′-tail end of *SLC6A15* gene on chromosome 12) as associated with etoposide IC_50_ (*P* = 2 × 10^−5^) and *IL1B* gene expression (*P* = 7 × 10^−7^). We also found a strong correlation between the *IL1B* expression and etoposide IC_50_ (*P* = 2 × 10^−5^ and [Fig. 5](#F5)). In the YRI, we identified a strong association between the genotype of SNP rs2784917, located in *SLIT1* gene on chromosome 10, and the expression of *WNT5B* gene, located on chromosome 12 (*P* = 7 × 10^−8^) along with a significant SNP genotype and etoposide IC_50_ association (*P* = 5 × 10^−5^). The TT genotype of rs2784917 was associated with higher *WNT5B* gene expression and lower etoposide IC_50_. This was further indicated by the inverse correlation found between *WNT5B* gene expression and etoposide IC_50_ (*P* = 4 × 10^−6^ and [Fig. 6](#F6)).

### Fig. 3.

![Fig. 3.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ea6f/1887589/325e63f5f012/zpq0230765970003.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=1887589_zpq0230765970003.jpg)

Relationship between rs446112, AGPAT2 gene expression and etoposide IC50 in combined CEU and YRI. (A) rs446112 genotype and Box–Cox transformed etoposide IC50 association. (B) rs446112 genotype and log2 transformed AGPAT2 expression association. (C) Log2 transformed AGPAT2 expression and Box-Cox transformed etoposide IC50 correlation.

### Fig. 5.

![Fig. 5.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ea6f/1887589/bc31fb33b78a/zpq0230765970005.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=1887589_zpq0230765970005.jpg)

Relationship among rs6539870, IL1B gene expression, and etoposide IC50 in CEU. (A) rs6539870 genotype and Box–Cox-transformed etoposide IC50 association. (B) rs6539870 genotype and log2 transformed IL1B expression association. (C) Log2-transformed IL1B expression and Box–Cox-transformed etoposide IC50 correlation.

### Fig. 6.

![Fig. 6.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ea6f/1887589/77affec3c8f4/zpq0230765970006.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=1887589_zpq0230765970006.jpg)

Relationship among rs2784917, WNT5B gene expression, and etoposide IC50 in YRI. (A) rs2784917 genotype and Box–Cox-transformed etoposide IC50 association. (B) rs2784917 genotype and log2-transformed WNT5B expression association. (C) Log2-transformed WNT5B expression and Box–Cox-transformed etoposide IC50 correlation.

### Multivariate Models to Predict Etoposide IC50 with Genotypes.

To examine the overall contributions of our selected genetic variants to sensitivity of etoposide, additional general linear models were constructed. All SNP genotypes that were significantly associated with etoposide IC_50_ through their effects on gene expression were included as the independent variables to predict etoposide IC_50_ as the dependent variable in each tested population. The backwards elimination approach was applied for models reduction. In the combined population, three of the four tested SNPs were included in the final model (*P* < 0.0003 for all SNPs). Specifically, rs460869, rs6588131, and rs16965867 were all significant predictors of etoposide IC_50_. Computing a weighted sum of *R*^2^ from each group of unrelated individuals gives an overall estimate of *R*^2^ = 0.30, indicating 30% of the variation in etoposide IC_50_ can be explained by these three SNPs in the combined populations. In the CEU population, rs10018204, rs11222869, rs16965867, rs1846644, and rs6539870 were included in the final model (*P* < 0.015 for all five SNPs). The indicator of gender is a significant predictor (*P* < 0.0077), which is in agreement with our previous finding of a significant difference in etoposide IC_50_ between female and male within the YRI ([5](#B5)). The overall estimate of *R*^2^ = 0.55, indicating 55% of the etoposide IC_50_ variation can be explained by these five SNPs in the CEU population. In the YRI, four of the six tested SNPs were included in the final model (*P* < 0.015 for all SNPs). Specifically, rs10061997, rs12190776, rs2784917, and rs9730073 were all significant predictors of etoposide IC_50_. The overall estimate of *R*^2^ = 0.40, indicates 40% of the etoposide IC_50_ variation can be explained by these 4 SNPs in the YRI.

## Discussion

We have developed a genome-wide approach to identify genetic variants that are important in chemotherapy-induced cytotoxicity as well as other quantitative traits that can be measured in HapMap cell lines. Our model includes complementary approaches merging whole-genome association between genotype and phenotype (sensitivity to drug) and association between genotype and gene expression, as well as linear regression analysis between gene expression and phenotype to identify genetic variants that are important to drug-induced cytotoxicity through modulation of gene expression.

Previously, our laboratory used cell lines derived from large pedigrees to demonstrate that a significant genetic component contributed to susceptibility to the cytotoxic effects of cisplatin, a chemotherapeutic agent ([6](#B6)). Variation in susceptibility to two additional chemotherapeutic agents, fluorouracil (5-FU) and docetaxel, were also shown to have a significant genetic component ([7](#B7)). Cheung's group has shown that expression of a considerable number of genes is directly controlled by cis- or trans-acting elements of the genotype ([8](#B8)–[10](#B10)). Many studies have also demonstrated the correlation between gene expression and treatment effect ([11](#B11)–[19](#B19)). The present study links genotype to gene expression to cell line treatment response and ultimately identified genetic variants that influence drug-induced cytotoxicity.

For etoposide, our model allowed us to uncover previously unknown genetic variants important in drug cytotoxicity. We applied our model to the combined populations (CEU + YRI), which provided more power to detect significant genetic variants and because both populations showed similar sensitivity to etoposide at all concentrations studied. However, the possibility exists that, even though both populations showed similar sensitivity to etoposide, this phenotype could be controlled by different genetic variants in each population; therefore, we also applied our model independently to CEU and YRI. One challenge in the application of this genome-wide approach is that the multiple testing may result in false discovery. To decrease the false discovery rate, a conventional approach is to use a stringent statistical cutoff ([20](#B20)). However, these arbitrarily chosen cutoff values may not hold true biological meaning. Our stepwise approach linked SNP genotype, gene expression, and sensitivity to drugs together. We limited our search to identify genetic variants that associated sensitivity to drugs through regulation of gene expression. Without the gene expression element of the model, the QTDT analysis between genotype and etoposide IC_50_ yielded 49, 122, and 51 significant SNPs (*P* ≤ 0.0001) associated with etoposide IC_50_ from the combined CEU- and YRI-derived cell lines, respectively. Taking into account these associations composed of those SNPs associated and not associated with expression, we further reduced our list to only those associated with expression to obtain 4, 54, and 6 SNPs through the expression of 3, 18, and 24 genes in combined, CEU and YRI, respectively. The stepwise approach allows us to narrow down genes whose expression is correlated to etoposide cytotoxicity and provides us with greater confidence in the SNPs identified for further validation.

All SNP genotypes and gene expressions found through this model can be considered relevant to sensitivity to etoposide. Functional studies are currently underway to confirm the role of these genetic variants contribution. Of the 63 genetic variants we identified, some of them have not been described previously, whereas others are supported by literature to play a role in the cell sensitivity to etoposide. For example, we identified a strong association among genetic variants of SNP rs446112, gene expression of *AGPAT2*, and susceptibility to etoposide-induced cytotoxicity in the combined populations. Niesporek *et al.* ([11](#B11)) have shown the increased expression of *AGPAT2* was significantly linked to reduced overall survival time as well as to shorter progression-free survival time in ovarian cancer patients. Our study showed that the AA genotype of SNP rs446112 was associated with higher *AGPAT2* gene expression, which correlated with lower sensitivity to etoposide. Further studies to confirm *AGPAT2* as a mechanism of resistance to etoposide are warranted.

An association between genetic variants of SNP rs6539870, gene expression of *IL1B*, and etoposide IC_50_ was identified in CEU. *IL1B* is an important cytokine, which mediates the inflammatory response and is involved in a variety of cellular activities, including cell proliferation, differentiation, and apoptosis ([21](#B21)). It has been shown that IL-1α, a close member of the same interleukin protein family, can dramatically increase the sensitivity of etoposide in osteosarcoma cells ([22](#B22)). The synergistic antitumor effects of IL-1α and etoposide have also been observed in melanoma and ovarian cell lines ([23](#B23), [24](#B24)). Our study showed that the GG genotype of SNP rs6539870 associated with higher *IL1B* gene expression and greater sensitivity to etoposide. Another association was identified among genetic variants of rs2784917, gene expression of *WNT5B*, and etoposide IC_50_ in the YRI. WNT signaling pathway plays a key role in carcinogenesis and embryogenesis and has been found to be up-regulated in gastric, esophageal, pancreatic, breast cancer cell line, and uterine leiomyoma cells ([25](#B25), [26](#B26)). Our study showed that the TT genotype of SNP rs2784917 was associated with higher *WNT5B* gene expression and greater sensitivity to etoposide. These genes may be targets for synergizing tumor cells to etoposide. Additionally, many genes identified through our model were not previously studied and could also be targets for etoposide treatment induced toxicity.

We also found that the expression levels of many genes shared the same regulatory region. For example, we found 45 SNPs with high LD located in the *SLC2A9* gene. These SNPs are significantly associated with the expression of eight genes. Among the 45 SNPs, we identified haplotypes associated with *NOTCH1* and *GBP2*/*4*/*7* gene expression as well as haplotypes associated with *CWF19L1*/*BLOC1S2* gene expression. An intronic region located in *SLC2A9* gene that has been shown to be highly conserved across 17 species (University of California Santa Cruz Human Genome Browser, March 2006 assembly; [http://genome.ucsc.edu](http://genome.ucsc.edu)) contains two of our significant SNPs (rs6449178 and rs6449179). Further study is needed to evaluate the role of this region, because it may host a new gene or contain an alternative splicing site or noncoding, micro-, or siRNA.

The ultimate goal of the model is to identify genetic variants important in drug response or toxicity, so patients at risk for nonresponse or toxicities can be given alternative therapy. However, one limitation of the model is that it represents only one type of cell line and may not represent protein expression in a tissue of known toxicity or tumor. Furthermore, candidate genes known to contribute to the pharmacokinetics of etoposide, e.g., *CYP3A*, *UGT1A1*, and *ABCB1*, are not expressed or are expressed at very low levels in LCLs (data not shown). The advantage to the model is that genes important in the pharmacodynamics of the drug can be identified without confounding variables from pharmacokinetic variables. Although the model allowed us to identify several genetic variants significantly associated with etoposide-induced cytotoxicity through gene expression, functional studies are required to confirm the contribution of these genetic variants to etoposide-induced cytotoxicity. There are also other genetic variants that are not associated through expression that may contribute to the variation in cytotoxicity that will require further study.

In summary, this genome-wide approach successfully integrated genotype, gene expression, and sensitivity to drug information to identify genetic variants that are important in drug treatment. It can be used to uncover important genetic variants contributing to a wide range of phenotypes that can be measured in LCLs.

## Methods

### Cell Lines.

EBV-transformed B LCLs derived from 30 Center d'Etude du Polymorphisme Humain trios (mother, father, and child) from Utah residents with ancestry from Northern and Western Europe (HAPMAPPT01, CEU) and 30 trios collected from the Yoruba in Ibadan, Nigeria (HAPMAPPT03, YRI) were purchased from the Coriell Institute for Medical Research (Camden, NJ). Cell lines were maintained in RPMI medium 1640 (Mediatech, Herndon, VA) supplemented with 15% FBS (HyClone, Logan, UT) and 1% l-glutamine (Invitrogen, Carlsbad, CA). Cell lines were passaged three times per week and seeded at a concentration of 350,000 cells/ml at 37°C in a 95% humidified 5% CO_2_ atmosphere.

### Drug.

Etoposide (NSC-141540) was provided by the Drug Synthesis and Chemistry Branch, Division of Cancer Treatment, National Cancer Institute, Bethesda, MD. PBS (pH 7.4) was from Invitrogen, and DMSO was from Sigma–Aldrich (St. Louis, MO).

### Cytotoxicity Assay.

The cytotoxic effect was determined by using the nontoxic colorimetric-based assay, alamarBlue (Biosource, Camarillo, CA). Exponentially growing lymphoblastoid cells with >85% viability, as determined by the trypan blue dye exclusion method by using Vi-Cell XR viability analyzer (Beckman Coulter, Fullerton, CA), were plated in triplicate at a density of 1 × 10^5^ cells/ml in 96-well round-bottom plates (Corning, Corning, NY) for 24 h. Etoposide was initially dissolved in DMSO and further diluted with media. Cells were treated with either vehicle (0.025% DMSO) or 0.02, 0.1, 0.5, and 2.5 μM etoposide for 72 h. AlamarBlue was added 24 h before absorbance reading at wavelengths 570 and 600 nm by using the Synergy-HT multidetection plate reader (BioTek, Winooski, VT). Percent survival was quantified by using the manufacturer's protocol. Final percent survival was averaged from at least six replicates from two independent experiments. IC_50_ was determined for each cell line by curve fitting of percent cell survival against concentrations of drug.

### Genotype and Cytotoxicity Association Analysis.

SNP genotypes were downloaded from the International HapMap database ([www.HapMap.org](http://www.HapMap.org)) (release 21). To perform a high-quality whole-genome association study, several data filters were used. To reduce possible genotyping errors, we excluded 100,536 and 138,533 SNPs with Mendelian allele transmission errors on 22 autosomes in the 30 CEU and 30 YRI HapMap trios, respectively. To exclude the extreme outliers and increase the power of the association studies within our limited number of samples, we included only the SNPs that met the criteria of having three genotypes (homozygous wild type, heterozygous, and homozygous variant) and containing a minimum of two samples for each genotype in the unrelated individuals of each population. To obtain functionally relevant SNPs, we further filtered the SNPs by location. Only SNPs located within 10-kb up- or downstream and within a gene were included. Thus, our final data set consisted of 387,417 very informative SNPs covering 22,667 well annotated genes.

Eighty-seven CEU and 89 YRI HapMap samples were phenotyped for etoposide cytotoxicity. A Box–Cox transformation was applied to these 176 IC_50_ values by using MINITAB14 (Minitab, State College, PA) followed by a Kolmogorov–Smirnov test of normality. The transformed data with *P* > 0.05 were considered normally distributed. The QTDT was performed to identify any genotype–cytotoxicity association by using QTDT software ([27](#B27)) ([www.sph.umich.edu/csg/abecasis/QTDT](http://www.sph.umich.edu/csg/abecasis/QTDT)). Because of the possible heterogeneity between and within each population, we performed QTDT studies in these two ethnic groups separately by using sex as a covariate and together by using sex and race as covariates. *P* ≤ 0.0001 was considered statistically significant. The binomial tests were performed between the probability of our significant findings and the probability of random discovery.

### Gene Expression Assessment.

Before sample collection, LCLs were diluted to 500,000 cells/ml in RPMI growth media four times over 2 wk. LCLs (8.5 million cells) were in log-growth phase at the time of collection with ≥85% viability. Trypan blue exclusion test was used to assess viability and concentration. RNA from 87 CEU and 89 YRI cell lines were extracted at the fourth dilution by using RNeasy Plus Mini kits (Qiagen, Valencia, CA). RNA quality was assessed by using the RNA 6000 Nano Assay (Agilent Technologies, Palo Alto, CA). RNA samples were further purified and prepared according to the manufacturer's protocol by using Affymetrix's GeneChip Whole Transcript Sense Target Labeling Assay with reagents specifically designed for the Human Exon arrays. Gene expression profiles were assessed by using Affymetrix GeneChip Human Exon 1.0 ST array. Probe-signal intensities were sketch-normalized by using a subset of the 1.4 million probe sets. Transcript cluster expression was summarized by using a robust multiarray average method ([28](#B28)) with a core set of well annotated exons (≈200,000). The ExACT program developed by Affymetrix with its Integrated Genome Browser was used to determine gene signal.

### Genotype and Gene Expression Association Analysis.

A second QTDT test that integrated candidate SNPs with mRNA level gene expression was performed to identify possible genotype-expression association. Significant SNPs generated from the genotype–cytotoxicity association in CEU, YRI, or combined populations were tested for their association with gene expression in the same population. Genes with average intensity greater than five from Affymetrix GeneChip Human Exon 1.0 ST Array analysis were considered as expressed genes and included in this association analysis. The QTDT test was carried out by using gene expression in CEU and YRI separately and in combined CEU and YRI with sex and race (in the combined samples) as covariates. We examined not only the cis-acting gene, defined as gene expression associated with SNP(s) within 2.5 Mb on the same chromosome, but also the trans-acting gene, defined as gene expression associated with SNP(s) on different chromosome(s). A Bonferroni correction (*P* = 0.05) was used to adjust raw *P* values after QTDT analysis.

### GO Analysis.

To further characterize sets of functionally related genes that may affect drug response, we used Onto Express ([29](#B29)–[32](#B32)) to classify genes according to their GO annotations ([33](#B33)). The genes hosting SNPs that were significantly associated with etoposide IC_50_, and those genes that were regulated by SNPs in a cis or trans manner were tested for their potential enrichment in a particular GO term by using biological process function. Hypergeometric distribution (Bioconductor Vignette, [http://bioconductor.org/docs/vignettes.html](http://bioconductor.org/docs/vignettes.html)) was used with each GO term having an associated *P* value relating to the number of genes that are annotated at that term. GO terms enriched in our sets of genes relative to the references (as total number of genes tested) was indicated at a Benjamin–Hochberg FDR of 5%.

### Gene Expression and Etoposide IC50 Linear Regression Analysis.

To examine the relationship between gene expression and sensitivity to etoposide, a general linear model was constructed with etoposide IC_50_ (transformed by using the Box–Cox transformation) as the dependent variable and robust multiarray average-summarized log_2_-transformed gene expression level together with an indicator for gender as the independent variables. The dependent variable was transformed to satisfy the assumption of normality. Trios were treated as units of analysis, and members of different families were considered independent. The covariance structure within a trio was modeled by using a Toeplitz structure with two diagonal bands, such that the trios were ordered father, offspring, and then mother. With this covariance structure, mother and father IC_50_ values were independent, but the offspring's value was allowed to covary with both father's and mother's values. If a SNP was significantly associated with etoposide IC_50_, and the same SNP was significantly associated with gene expression, then the above approach was used to test whether gene expression significantly predicted IC_50_. Sixteen genes/transcripts were tested for their expression correlation with etoposide IC_50_ in the CEU population, and 33 genes were tested in the YRI. Only six genes were tested by using combined CEU and YRI. With the combined approach a predictor of population was included in the model. *P* < 0.05 was considered statistically significant. The model was programmed by using the PROC MIXED procedure in SAS/STAT software (Version 9.1, SAS Institute, Cary, NC). The REPEATED statement was used to model the Toeplitz covariance structure. The LD of significant SNPs within each population was evaluated by using Haploview version 3.32 ([www.broad.mit.edu/mpg/haploview](http://www.broad.mit.edu/mpg/haploview)).

### Multivariate Model to Predict Etoposide IC50 with Genotypes.

To examine the overall genetic variant contributions to sensitivity of etoposide, additional general linear models were constructed with transformed etoposide IC_50_ as the dependent variable. The independent variables included all of the significant SNP genotypes (assuming an additive genetic effect) that were selected from the three-way model in the combined populations and the two populations independently. These SNP genotypes are significantly associated with etoposide IC_50_ through their effect on gene expression. For the model of combined populations, indicators of race and sex were also included as predictors. Trios were analyzed as independent units. The covariance was modeled as described above. Models were reduced by using a backwards elimination approach. SNPs included in each of the final models were statistically significant at the α = 0.05 level. Using the final model, predicted transformed IC_50_ values were computed. Within the unrelated individuals (parents from the trios and separately offspring from the trios) an *R*^2^ was estimated between observed IC_50_ and the predicted IC_50_ from the final model. Last, a weighted average of the two *R*^2^ estimates was computed to quantify the amount of variation in etoposide IC_50_ explained by the selected SNP genotypes.

## Supplementary Material

## Acknowledgments

We thank Dr. Jeong-Ah Kang for excellent technical support in maintaining the cell lines and Cheryl Roe for critical review of the manuscript. This Pharmacogenetics of Anticancer Agents Research Group ([http://pharmacogenetics.org](http://pharmacogenetics.org)) study was supported by the National Institutes of Health (NIH)/National Institute of General Medical Sciences (NIGMS) (Grant GM61393). This research is supported by the NIH/NIGMS Pharmacogenetics Research Network and Database, U01GM61374, Russ Altman, principal investigator ([www.pharmgkb.org](http://www.pharmgkb.org)).

## Abbreviations

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. The International HapMap Consortium. Nature. 2005;437:1299–1320. doi: 10.1038/nature04226.  [DOI](https://doi.org/10.1038/nature04226) | [PMC free article](/articles/PMC1880871/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16255080/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&volume=437&publication_year=2005&pages=1299-1320&pmid=16255080&doi=10.1038/nature04226&)

2. Sinha BK, Haim N, Dusre L, Kerrigan D, Pommier Y. Cancer Res. 1988;48:5096–5100.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/2842045/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&author=BK%20Sinha&author=N%20Haim&author=L%20Dusre&author=D%20Kerrigan&author=Y%20Pommier&volume=48&publication_year=1988&pages=5096-5100&pmid=2842045&)

3. Mistry AR, Felix CA, Whitmarsh RJ, Mason A, Reiter A, Cassinat B, Parry A, Walz C, Wiemels JL, Segal MR, et al. N Engl J Med. 2005;352:1529–1538. doi: 10.1056/NEJMoa042715.  [DOI](https://doi.org/10.1056/NEJMoa042715) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15829534/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&author=AR%20Mistry&author=CA%20Felix&author=RJ%20Whitmarsh&author=A%20Mason&author=A%20Reiter&volume=352&publication_year=2005&pages=1529-1538&pmid=15829534&doi=10.1056/NEJMoa042715&)

4. Ratain MJ, Kaminer LS, Bitran JD, Larson RA, Le Beau MM, Skosey C, Purl S, Hoffman PC, Wade J, Vardiman JW. Blood. 1987;70:1412–1417.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/2822173/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&author=MJ%20Ratain&author=LS%20Kaminer&author=JD%20Bitran&author=RA%20Larson&author=MM%20Le%20Beau&volume=70&publication_year=1987&pages=1412-1417&pmid=2822173&)

5. Huang RS, Kistner EO, Bleibel WK, Shukla SJ, Dolan ME. Mol Cancer Ther. 2007;6:31–36. doi: 10.1158/1535-7163.MCT-06-0591.  [DOI](https://doi.org/10.1158/1535-7163.MCT-06-0591) | [PMC free article](/articles/PMC2669540/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17237264/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Cancer%20Ther&author=RS%20Huang&author=EO%20Kistner&author=WK%20Bleibel&author=SJ%20Shukla&author=ME%20Dolan&volume=6&publication_year=2007&pages=31-36&pmid=17237264&doi=10.1158/1535-7163.MCT-06-0591&)

6. Dolan ME, Newbold KG, Nagasubramanian R, Wu X, Ratain MJ, Cook EH, Jr, Badner JA. Cancer Res. 2004;64:4353–6. doi: 10.1158/0008-5472.CAN-04-0340.  [DOI](https://doi.org/10.1158/0008-5472.CAN-04-0340) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15205351/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&author=ME%20Dolan&author=KG%20Newbold&author=R%20Nagasubramanian&author=X%20Wu&author=MJ%20Ratain&volume=64&publication_year=2004&pages=4353-6&pmid=15205351&doi=10.1158/0008-5472.CAN-04-0340&)

7. Watters JW, Kraja A, Meucci MA, Province MA, McLeod HL. Proc Natl Acad Sci USA. 2004;101:11809–11814. doi: 10.1073/pnas.0404580101.  [DOI](https://doi.org/10.1073/pnas.0404580101) | [PMC free article](/articles/PMC511056/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15282376/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20USA&author=JW%20Watters&author=A%20Kraja&author=MA%20Meucci&author=MA%20Province&author=HL%20McLeod&volume=101&publication_year=2004&pages=11809-11814&pmid=15282376&doi=10.1073/pnas.0404580101&)

8. Cheung V, Conlin L, Weber T, Arcaro M, Jen K, Morley M, Spielman R. Nat Genet. 2003;33:422–425. doi: 10.1038/ng1094.  [DOI](https://doi.org/10.1038/ng1094) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12567189/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&author=V%20Cheung&author=L%20Conlin&author=T%20Weber&author=M%20Arcaro&author=K%20Jen&volume=33&publication_year=2003&pages=422-425&pmid=12567189&doi=10.1038/ng1094&)

9. Cheung VG, Spielman RS, Ewens KG, Weber TM, Morley M, Burdick JT. Nature. 2005;437:1365–1369. doi: 10.1038/nature04244.  [DOI](https://doi.org/10.1038/nature04244) | [PMC free article](/articles/PMC3005311/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16251966/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&author=VG%20Cheung&author=RS%20Spielman&author=KG%20Ewens&author=TM%20Weber&author=M%20Morley&volume=437&publication_year=2005&pages=1365-1369&pmid=16251966&doi=10.1038/nature04244&)

10. Morley M, Molony CM, Weber TM, Devlin JL, Ewens KG, Spielman RS, Cheung VG. Nature. 2004;430:743–747. doi: 10.1038/nature02797.  [DOI](https://doi.org/10.1038/nature02797) | [PMC free article](/articles/PMC2966974/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15269782/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&author=M%20Morley&author=CM%20Molony&author=TM%20Weber&author=JL%20Devlin&author=KG%20Ewens&volume=430&publication_year=2004&pages=743-747&pmid=15269782&doi=10.1038/nature02797&)

11. Niesporek S, Denkert C, Weichert W, Köbel M, Noske A, Sehouli J, Singer JW, Dietel M, Hauptmann S. Br J Cancer. 2005;92:1729–1736. doi: 10.1038/sj.bjc.6602528.  [DOI](https://doi.org/10.1038/sj.bjc.6602528) | [PMC free article](/articles/PMC2362024/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15841084/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Cancer&author=S%20Niesporek&author=C%20Denkert&author=W%20Weichert&author=M%20K%C3%B6bel&author=A%20Noske&volume=92&publication_year=2005&pages=1729-1736&pmid=15841084&doi=10.1038/sj.bjc.6602528&)

12. Hirano T, Kizaki M, Kato K, Abe F, Masuda N, Umezawa K. Leuk Res. 2002;26:1097–1103. doi: 10.1016/s0145-2126(02)00052-8.  [DOI](https://doi.org/10.1016/s0145-2126(02)00052-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12443882/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Leuk%20Res&author=T%20Hirano&author=M%20Kizaki&author=K%20Kato&author=F%20Abe&author=N%20Masuda&volume=26&publication_year=2002&pages=1097-1103&pmid=12443882&doi=10.1016/s0145-2126(02)00052-8&)

13. Debret R, Le Naour R, Sallenave J, Deshorgue A, Hornebeck W, Guenounou M, Bernard P, Antonicelli F. J Invest Dermatol. 2006;126:1860–1868. doi: 10.1038/sj.jid.5700337.  [DOI](https://doi.org/10.1038/sj.jid.5700337) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16675961/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Invest%20Dermatol&author=R%20Debret&author=R%20Le%20Naour&author=J%20Sallenave&author=A%20Deshorgue&author=W%20Hornebeck&volume=126&publication_year=2006&pages=1860-1868&pmid=16675961&doi=10.1038/sj.jid.5700337&)

14. Breit S, Stanulla M, Flohr T, Schrappe M, Ludwig W-D, Tolle G, Happich M, Muckenthaler MU, Kulozik AE. Blood. 2006;108:1151–1157. doi: 10.1182/blood-2005-12-4956.  [DOI](https://doi.org/10.1182/blood-2005-12-4956) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16614245/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&author=S%20Breit&author=M%20Stanulla&author=T%20Flohr&author=M%20Schrappe&author=W-D%20Ludwig&volume=108&publication_year=2006&pages=1151-1157&pmid=16614245&doi=10.1182/blood-2005-12-4956&)

15. Cohen L, Bourbonniere M, Sabbagh L, Bouchard A, Chew T, Jeannequin P, Lazure C, Sekaly R. Cell Death Differ. 2005;12:243–254. doi: 10.1038/sj.cdd.4401568.  [DOI](https://doi.org/10.1038/sj.cdd.4401568) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15650752/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Death%20Differ&author=L%20Cohen&author=M%20Bourbonniere&author=L%20Sabbagh&author=A%20Bouchard&author=T%20Chew&volume=12&publication_year=2005&pages=243-254&pmid=15650752&doi=10.1038/sj.cdd.4401568&)

16. Nefedova Y, Cheng P, Alsina M, Dalton WS, Gabrilovich DI. Blood. 2004;103:3503–3510. doi: 10.1182/blood-2003-07-2340.  [DOI](https://doi.org/10.1182/blood-2003-07-2340) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14670925/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&author=Y%20Nefedova&author=P%20Cheng&author=M%20Alsina&author=WS%20Dalton&author=DI%20Gabrilovich&volume=103&publication_year=2004&pages=3503-3510&pmid=14670925&doi=10.1182/blood-2003-07-2340&)

17. Kakugawa Y, Wada T, Yamaguchi K, Yamanami H, Ouchi K, Sato I, Miyagi T. Proc Natl Acad Sci USA. 2002;99:10718–10723. doi: 10.1073/pnas.152597199.  [DOI](https://doi.org/10.1073/pnas.152597199) | [PMC free article](/articles/PMC125023/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12149448/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20USA&author=Y%20Kakugawa&author=T%20Wada&author=K%20Yamaguchi&author=H%20Yamanami&author=K%20Ouchi&volume=99&publication_year=2002&pages=10718-10723&pmid=12149448&doi=10.1073/pnas.152597199&)

18. Suzuki S, Takahashi S, Takahashi S, Takeshita K, Hikosaka A, Wakita T, Nishiyama N, Fujita T, Okamura T, Shirai T. Prostate. 2006;66:463–469. doi: 10.1002/pros.20385.  [DOI](https://doi.org/10.1002/pros.20385) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16353248/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Prostate&author=S%20Suzuki&author=S%20Takahashi&author=S%20Takahashi&author=K%20Takeshita&author=A%20Hikosaka&volume=66&publication_year=2006&pages=463-469&pmid=16353248&doi=10.1002/pros.20385&)

19. Chung Y, Kim T, Kim D, Namkoong H, Kim H, Ha S, Kim S, Shin S, Kim J, Lee Y, et al. Leukemia. 2006;20:1542–1550. doi: 10.1038/sj.leu.2404310.  [DOI](https://doi.org/10.1038/sj.leu.2404310) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16855633/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Leukemia&author=Y%20Chung&author=T%20Kim&author=D%20Kim&author=H%20Namkoong&author=H%20Kim&volume=20&publication_year=2006&pages=1542-1550&pmid=16855633&doi=10.1038/sj.leu.2404310&)

20. Stranger BE, Forrest MS, Clark AG, Minichiello MJ, Deutsch S, Lyle R, Hunt S, Kahl B, Antonarakis SE, Tavaré S, et al. PLoS Genet. 2005;1:e78. doi: 10.1371/journal.pgen.0010078.  [DOI](https://doi.org/10.1371/journal.pgen.0010078) | [PMC free article](/articles/PMC1315281/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16362079/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet&author=BE%20Stranger&author=MS%20Forrest&author=AG%20Clark&author=MJ%20Minichiello&author=S%20Deutsch&volume=1&publication_year=2005&pages=e78&pmid=16362079&doi=10.1371/journal.pgen.0010078&)

21. Roy D, Sarkar S, Felty Q. Front Biosci. 2006;11:889–898. doi: 10.2741/1845.  [DOI](https://doi.org/10.2741/1845) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16146780/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front%20Biosci&author=D%20Roy&author=S%20Sarkar&author=Q%20Felty&volume=11&publication_year=2006&pages=889-898&pmid=16146780&doi=10.2741/1845&)

22. Jia S, Zwelling L, McWatters A, An T, Kleinerman E. J Exp Ther Oncol. 2002;2:27–36. doi: 10.1046/j.1359-4117.2002.01003.x.  [DOI](https://doi.org/10.1046/j.1359-4117.2002.01003.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12415617/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Exp%20Ther%20Oncol&author=S%20Jia&author=L%20Zwelling&author=A%20McWatters&author=T%20An&author=E%20Kleinerman&volume=2&publication_year=2002&pages=27-36&pmid=12415617&doi=10.1046/j.1359-4117.2002.01003.x&)

23. Usui N, Matsushima K, Pilaro A, Longo D, Wiltrout R. Biotherapy. 1996;9:199–208. doi: 10.1007/BF02620733.  [DOI](https://doi.org/10.1007/BF02620733) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9012539/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biotherapy&author=N%20Usui&author=K%20Matsushima&author=A%20Pilaro&author=D%20Longo&author=R%20Wiltrout&volume=9&publication_year=1996&pages=199-208&pmid=9012539&doi=10.1007/BF02620733&)

24. Monti E, Mimnaugh E, Sinha B. Biochim Biophys Acta. 1993;1180:231–235. doi: 10.1016/0925-4439(93)90043-z.  [DOI](https://doi.org/10.1016/0925-4439(93)90043-z) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8422427/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochim%20Biophys%20Acta&author=E%20Monti&author=E%20Mimnaugh&author=B%20Sinha&volume=1180&publication_year=1993&pages=231-235&pmid=8422427&doi=10.1016/0925-4439(93)90043-z&)

25. Saitoh T, Katoh M. Int J Mol Med. 2002;10:345–349.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/12165812/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Mol%20Med&author=T%20Saitoh&author=M%20Katoh&volume=10&publication_year=2002&pages=345-349&pmid=12165812&)

26. Mangioni S, Vigano P, Lattuada D, Abbiati A, Vignali M, Di Blasio AM. J Clin Endocrinol Metab. 2005;90:5349–5355. doi: 10.1210/jc.2005-0272.  [DOI](https://doi.org/10.1210/jc.2005-0272) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15972578/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Endocrinol%20Metab&author=S%20Mangioni&author=P%20Vigano&author=D%20Lattuada&author=A%20Abbiati&author=M%20Vignali&volume=90&publication_year=2005&pages=5349-5355&pmid=15972578&doi=10.1210/jc.2005-0272&)

27. Abecasis G, Cardon L, Cookson W. Am J Hum Genet. 2000;66:279–292. doi: 10.1086/302698.  [DOI](https://doi.org/10.1086/302698) | [PMC free article](/articles/PMC1288332/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10631157/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&author=G%20Abecasis&author=L%20Cardon&author=W%20Cookson&volume=66&publication_year=2000&pages=279-292&pmid=10631157&doi=10.1086/302698&)

28. Irizarry RA, Hobbs B, Collin F, Beazer-Barclay YD, Antonellis KJ, Scherf U, Speed TP. Biostatistics. 2003;4:249–264. doi: 10.1093/biostatistics/4.2.249.  [DOI](https://doi.org/10.1093/biostatistics/4.2.249) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12925520/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biostatistics&author=RA%20Irizarry&author=B%20Hobbs&author=F%20Collin&author=YD%20Beazer-Barclay&author=KJ%20Antonellis&volume=4&publication_year=2003&pages=249-264&pmid=12925520&doi=10.1093/biostatistics/4.2.249&)

29. Draghici S, Khatri P, Bhavsar P, Shah A, Krawetz SA, Tainsky MA. Nucleic Acids Res. 2003;31:3775–3781. doi: 10.1093/nar/gkg624.  [DOI](https://doi.org/10.1093/nar/gkg624) | [PMC free article](/articles/PMC169030/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12824416/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&author=S%20Draghici&author=P%20Khatri&author=P%20Bhavsar&author=A%20Shah&author=SA%20Krawetz&volume=31&publication_year=2003&pages=3775-3781&pmid=12824416&doi=10.1093/nar/gkg624&)

30. Draghici S, Khatri P, Martins R, Ostermeier G, Krawetz S. Genomics. 2003;81:98–104. doi: 10.1016/s0888-7543(02)00021-6.  [DOI](https://doi.org/10.1016/s0888-7543(02)00021-6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12620386/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genomics&author=S%20Draghici&author=P%20Khatri&author=R%20Martins&author=G%20Ostermeier&author=S%20Krawetz&volume=81&publication_year=2003&pages=98-104&pmid=12620386&doi=10.1016/s0888-7543(02)00021-6&)

31. Khatri P, Draghici S, Ostermeier G, Krawetz S. Genomics. 2002;79:266–270. doi: 10.1006/geno.2002.6698.  [DOI](https://doi.org/10.1006/geno.2002.6698) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11829497/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genomics&author=P%20Khatri&author=S%20Draghici&author=G%20Ostermeier&author=S%20Krawetz&volume=79&publication_year=2002&pages=266-270&pmid=11829497&doi=10.1006/geno.2002.6698&)

32. Khatri P, Sellamuthu S, Malhotra P, Amin K, Done A, Draghici S. Nucleic Acids Res. 2005;33:W762–W765. doi: 10.1093/nar/gki472.  [DOI](https://doi.org/10.1093/nar/gki472) | [PMC free article](/articles/PMC1160233/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15980579/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&author=P%20Khatri&author=S%20Sellamuthu&author=P%20Malhotra&author=K%20Amin&author=A%20Done&volume=33&publication_year=2005&pages=W762-W765&pmid=15980579&doi=10.1093/nar/gki472&)

33. Ashburner M, Ball C, Blake J, Botstein D, Butler H, Cherry J, Davis A, Dolinski K, Dwight S, Eppig J, et al. Nat Genet. 2000;25:25–29. doi: 10.1038/75556.  [DOI](https://doi.org/10.1038/75556) | [PMC free article](/articles/PMC3037419/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10802651/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&author=M%20Ashburner&author=C%20Ball&author=J%20Blake&author=D%20Botstein&author=H%20Butler&volume=25&publication_year=2000&pages=25-29&pmid=10802651&doi=10.1038/75556&)
