# Genetic polymorphisms of GGT1 gene (rs8135987, rs5751901 and rs2017869) are associated with neoadjuvant chemotherapy efficacy and toxicities in breast cancer patients

## Metadata
**Authors:** Lu Sun, Ziping Wu, Yanping Lin, Shuguang Xu, Yumei Ye, Wenjin Yin, Liheng Zhou, Jingsong Lu
**Journal:** BMC Medical Genomics
**Date:** 2023 Oct 27
**DOI:** [10.1186/s12920-023-01685-7](https://doi.org/10.1186/s12920-023-01685-7)
**PMID:** 37891571
**PMCID:** PMC10612355
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10612355/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC10612355/pdf/12920_2023_Article_1685.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC10612355/pdf/12920_2023_Article_1685.pdf)

## Abstract

**Background:** 
Our previous study illustrated the predictive value of serum gamma-glutamyl transpeptidase (GGT) for neoadjuvant chemotherapy (NAC) sensitivity in breast cancer patients. In this study we aim to determine whether single nucleotide polymorphisms (SNPs) in the gamma-glutamyltransferase 1 (GGT1) gene are related to the NAC response and adverse events and to find out a genetic marker in predicting NAC sensitivity.

**Methods:** 
Three SNP loci (rs8135987, rs5751901, rs2017869) of GGT1 gene were selected and tested among breast cancer patients reciving NAC. Four genotype models were used in SNP analysis: co-dominant model compared AA vs. Aa vs. aa; dominant model compared AA vs. Aa + aa; recessive model compared AA + Aa vs. aa; over-dominant model compared AA + aa vs. Aa. Chi-squared test and multivariable logistic regression analysis were performed between SNP genotypes, haplotypes and pathological complete response(pCR), adverse events as well as serum GGT level.

**Results:** 
A total of 143 patients were included in the study. For SNP rs8135987 (T > C), the TC genotype in over-dominant model was inversely related with pCR (adjusted OR = 0.30, 95% CI 0.10–0.88, p = 0.029) as well as the risk of peripheral neuropathy (adjusted OR = 0.39, 95% CI 0.15–0.96, p = 0.042). The TC genotype in dominant model was significantly associated with elevated serum GGT level (OR = 3.11, 95% CI 1.07–9.02, p = 0.036). For rs2017869 (G > C), the occurrence of grade 2 or greater neutropenia (OR = 0.39, 95% CI 0.08–0.84, p = 0.025) and leukopenia (OR = 0.24, 95% CI 0.08–0.78, p = 0.017) were both significantly reduced in patients with CC genotypes. For rs5751901(T > C), the CC genotype could significantly reduce the risk of grade 2 or greater neutropenia (OR = 0.29, 95% CI 0.09–0.96, p = 0.036) and leukopenia (OR = 0.27, 95% CI 0.09–0.84, p = 0.024) in recessive model.

**Conclusions:** 
The GGT1 gene SNPs might be an independent risk factor for poor response of NAC in breast cancer patients, providng theoretical basis for further precision therapy.

**Supplementary Information:** 
The online version contains supplementary material available at 10.1186/s12920-023-01685-7.

Keywords: GGT1 gene, Single nucleotide polymorphism, Breast cancer, Neoadjuvant therapy

### Background

Our previous study illustrated the predictive value of serum gamma-glutamyl transpeptidase (GGT) for neoadjuvant chemotherapy (NAC) sensitivity in breast cancer patients. In this study we aim to determine whether single nucleotide polymorphisms (SNPs) in the gamma-glutamyltransferase 1 (*GGT1*) gene are related to the NAC response and adverse events and to find out a genetic marker in predicting NAC sensitivity.

### Methods

Three SNP loci (rs8135987, rs5751901, rs2017869) of *GGT1* gene were selected and tested among breast cancer patients reciving NAC. Four genotype models were used in SNP analysis: co-dominant model compared AA vs. Aa vs. aa; dominant model compared AA vs. Aa + aa; recessive model compared AA + Aa vs. aa; over-dominant model compared AA + aa vs. Aa. Chi-squared test and multivariable logistic regression analysis were performed between SNP genotypes, haplotypes and pathological complete response(pCR), adverse events as well as serum GGT level.

### Results

A total of 143 patients were included in the study. For SNP rs8135987 (T > C), the TC genotype in over-dominant model was inversely related with pCR (adjusted OR = 0.30, 95% CI 0.10–0.88, *p* = 0.029) as well as the risk of peripheral neuropathy (adjusted OR = 0.39, 95% CI 0.15–0.96, *p* = 0.042). The TC genotype in dominant model was significantly associated with elevated serum GGT level (OR = 3.11, 95% CI 1.07–9.02, *p* = 0.036). For rs2017869 (G > C), the occurrence of grade 2 or greater neutropenia (OR = 0.39, 95% CI 0.08–0.84, *p* = 0.025) and leukopenia (OR = 0.24, 95% CI 0.08–0.78, *p* = 0.017) were both significantly reduced in patients with CC genotypes. For rs5751901(T > C), the CC genotype could significantly reduce the risk of grade 2 or greater neutropenia (OR = 0.29, 95% CI 0.09–0.96, *p* = 0.036) and leukopenia (OR = 0.27, 95% CI 0.09–0.84, *p* = 0.024) in recessive model.

### Conclusions

The *GGT1* gene SNPs might be an independent risk factor for poor response of NAC in breast cancer patients, providng theoretical basis for further precision therapy.

### Supplementary Information

The online version contains supplementary material available at 10.1186/s12920-023-01685-7.

## Background

Breast cancer is the most common malignant tumor among females all around the world [[1](#CR1)]. Neoadjuvant chemotherapy (NAC) has been widely used in patients with locally advanced breast cancer as an important part of comprehensive breast cancer treatment. Clinical studies had revealed a better prognosis in patients achieving pathologic complete response (pCR) [[2](#CR2)]. Thus, it is necessary to find out biomarkers to predict NAC sensitivity. Gamma-glutamyltransferase (GGT) is a membrane-bound enzyme encoded by *GGT1* gene. The previous study had demonstrated it could protect cells from being damaged by radicals and oxidative stress, which is considered to be related to the resistance of antitumor drugs [[3](#CR3)]. In our previous study, we had discovered the predictive value of serum GGT in neoadjuvant chemotherapy for breast cancer. Patients with low pre-therapeutic serum GGT levels are more likely to have higher pCR rates, better RFS and DFS, and higher hematologic toxicity [[4](#CR4)]. However, the underlying mechanism for the relationship between serum GGT level and NAC sensitivity needs to be further explored.

Single nucleotide polymorphisms (SNPs) are DNA sequence polymorphisms resulting from mutations in a specific nucleotide in the genome of a chromosome. SNPs were reported to be correlated with disease susceptibility as well as drug resistance in vivo [[5](#CR5)–[9](#CR9)]. The *GGT1* gene, located on human chromosome 22, is the encoding gene of GGT protein. *GGT1* gene. Diergaarde et al. analyzed 26 SNPs and found out that the SNPs rs2017869 and rs8135987 of *GGT1* gene were significantly associated with the incidence and development of pancreatic cancer [[3](#CR3)]. Brand et al. reported the correlation between rs8135987 and rs4820599 and the risk of chronic pancreatitis [[4](#CR4)]. When it comes to drug toxicity, Khrunin et al. explored the relationship between SNPs rs5751901 and drug toxicity in patients with ovarian cancer and demonstrated that patients with TT genotype of rs5751901 had an increased risk of nephrotoxicity during cisplatin-based chemotherapy [[10](#CR10)]. However, there are few studies related to SNPs of *GGT* gene in breast cancer, especially in the neoadjuvant setting.

The serum GGT protein level is affected by both environmental and genetic factors. The correlation between serum GGT protein level and *GGT1* gene SNPs had been identified in different ethnic groups [[11](#CR11), [12](#CR12)]. Melzer et al. found that serum GGT protein level was associated with rs5751901 [[11](#CR11)]. The rs4820599 variant in *GGT1* was significantly related to circulating GGT protein level in a large-scale genome-wide association studies (GWAS) meta-analysis in East Asians as well [[12](#CR12)].

Based on these premises, we tested a hypothesis that *GGT1* gene SNPs might affect the level of GGT protein in tissues and serum, which would, in turn, affect the efficacy and toxicity of neoadjuvant chemotherapy in breast cancer. We searched for *GGT1* SNPs by using the public database and then did literature research. Finally, three *GGT1* SNPs (rs5751901, rs8135987 and rs2017869), which might have predictive value for disease or prognosis in tumors, were selected in this study. We performed a study to demonstrate whether the SNPs located in *GGT1* gene had an effect on serum GGT protein level and the susceptibility of breast cancer patients undergoing neoadjuvant chemotherapy, in order to find out a genetic marker in predicting NAC sensitivity.

## Method

### Study population

This study consists of 143 newly diagnosed breast cancer patients from December 2013 to January 2018 in Shanghai Jiao Tong university affiliated Renji hospital. All the patients were enrolled in the SHPD001 and SHPD002 clinical trials. The study design and recruitment methods have been described in detail previously [[13](#CR13), [14](#CR14)]. Briefly, all the patients were scheduled to received NAC before surgery. The chemotherapy regimen is a combination of weekly paclitaxel and cisplatin. Human epidermal growth factor receptor-2 (HER-2) positive patients in SHPD001 were recommended to receive concurrent trastuzumab. All HER-2 positive patient in SHPD002 received trastuzumab concomitantly at a weekly basis. For hormone receptor positive patients in SHPD002, endocrine therapy (aromatase inhibitor or gonadotropin releasing hormone agonist) was randomized together with chemotherapy according to their menstrual status.

The pCR (ypT0, defined as the absence of either invasive cancer or cancer in situ in the breast) was used to estimate the efficacy of NAC. HER-2 positive was defined as IHC staining 3 + or FISH (florescent in situ hybridization) showing HER2 gene amplification. Clinical staging was based on the eighth edition of American Joint Committee on Cancer (AJCC) TNM classification. The clinical stage of the patient was determined by CT, MRI and bone scan before treatment. Fine needle or core needle biopsy was performed for clinically significant enlarged lymph nodes. Adverse events were assessed at each visit and recorded according to CTCAE v4.03. Peripheral blood specimens were collected within one week prior to the first cycle of NAC for evaluation of serum GGT and genetic analysis (stored at -80℃). Serum GGT was assayed by the standard method recommended by the International Federation for Clinical Chemistry [[15](#CR15)]. The normal range of serum GGT for female was 7–32 U/L at our institution.

### SNP selection and genotyping assays

We selected SNPs by using the public database (NCBI/TargetScan) if they met the following criteria: a) Located in the 3'UTR region or 5'UTR region or intron region of the *GGT1* gene; b) Minimum allele frequency (MAF) > 0.10; c) Reported by other researchers with predicting the value of disease incidence or prognosis.

The *GGT1* gene information of the Han Chinese in Beijing and the Southern Han Chinese using for Hardy–Weinberg equilibrium (HWE) and linkage disequilibrium analysis was obtained using 1000 Genomes Browser.

A total of three SNPs was selected for further studying: rs8135987, rs5751901, rs2017869. Genomic DNA was extracted from the peripheral blood samples using the TIANamp Genomic DNA Kit (Beijing, People’s Republic of China) according to the manufacturer’s protocols (primer sequencing see Supplementary Table [1](#MOESM1)). The candidate SNPs were genotyped at Shanghai Benegene Biotechnology Co., Ltd (Shanghai, People’s Republic of China), using the MassARRAY system (Sequenom, San Diego, CA, USA).

### Statistical analysis

This study established four genotype models. Assuming A is the major allele and a is the minor allele. The co-dominant model compares AA vs. Aa vs. aa; the dominant model compares AA vs. Aa + aa; the recessive model compares AA + Aa vs. aa; the over-dominant model compares AA + aa vs. Aa [[16](#CR16)].

Pearson's chi-square test was used for assessment of Hardy–Weinberg equilibrium (HWE) [[17](#CR17)] and the frequency differences in the genotype and haplotype distribution between groups (different pCR outcomes and serum GGT levels). Multivariable logistic regression analyses were used to analyze the associations between different SNP genotypes, haplotypes and pCR outcomes, serum GGT level as well as adverse events. Results were shown as p-value, odds ratio (OR), and 95% confidence interval (95%CI). A two-sided p value < 0.05 was considered statistically significant. Stata SE 14.1 (Stata Corp LP, USA) was used in statistical analysis. Haploview 4.1 and PHASE 2.1 was used for haplotype analysis.

## Results

### Genotype distributions

No deviations from HWE were detected (p > 0.05) (Table [1](#Tab1)). SNPs rs5751901 was in linkage disequilibrium with rs2017869 (D' = 0.97, r2 = 0.91) (Fig. [1](#Fig1)), further haplotype analysis were carried out with these two loci. A total of 143 patients were included in this study.

### Table 1.

| Gene | SNP | SNP location | Major allele | Minor allele | Genotype Distribution A N (%) | MAF | HWE |
| --- | --- | --- | --- | --- | --- | --- | --- |
| GGT1 gene | rs8135987 | chr22: 25,012,854 | T | C | 69(48.3)/58(40.5)/16(11.2) | 0.303 | 0.1546 |
| rs5751901 | chr22: 24,992,266 | T | C | 57(39.9)/62(43.3)/24(16.8) | 0.344 | 0.1029 |  |
| rs2017869 | chr22: 24,997,309 | G | C | 59(41.3)/61(42.6)/23(16.1) | 0.353 | 0.0687 |  |

Table 1 Caption: Genotype distribution and HWE

### Fig. 1.

![Fig. 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d0fe/10612355/44c79fac545f/12920_2023_1685_Fig1_HTML.jpg)

D and r2 value of paired GGT1 gene SNP loci. a D value of paired SNP loci. b r2 value of paired SNP loci

### Clinic-pathological characteristics and the relationship between three SNPs and the efficacy of NAC

The average age of all patients was 51.5 years (range, 26–70 years). A total of 50 patients achieved pCR in NAC with a pCR rate of 34.97% (Table [2](#Tab2)). For the SNP rs8135987, patients with CC + TC genotype had significantly lower pCR rate compared with the TT genotype in the dominant model (*p* = 0.039). Results from the multivariable logistic regression analysis revealed that the odds of obtaining pCR for TC genotype were 0.30 times the odds of obtaining pCR for TT genotype in co-dominant model (95% CI 0.09–0.90, *p* = 0.033) and 0.30 times the odds of obtaining pCR for TT + CC genotype in over-dominant model (95% CI 0.10–0.88, *p* = 0.029). In other words, TC genotype was less likely to obtain pCR in both co-dominant model and over dominant model (Table [3](#Tab3)).

### Table 2.

| Patient characteristics | N(%) |
| --- | --- |
| Age at first diagnosis (years) |  |
| ≤ 50 | 61 (42.66) |
| > 50 | 82 (57.34) |
| Menses |  |
| Premenopausal | 59(41.26) |
| Postmenopausal | 84(58.74) |
| Body Mass Index (BMI) |  |
| ≤ 23 | 69(48.25) |
| > 23 | 74(51.75) |
| Estrogen Receptor (ER) (%) |  |
| < 10 | 50(34.97) |
| ≥ 10 | 93(65.03) |
| Progesterone Receptor (PR) (%) |  |
| < 10 | 42(29.37) |
| ≥ 10 | 101(70.63) |
| Ki-67 (%) |  |
| < 30 | 35(24.48) |
| ≥ 30 & < 60 | 65(45.45) |
| ≥ 60 | 43(30.07) |
| HER-2 statusA |  |
| Negative | 84(58.74) |
| Positive | 59(41.26) |
| Clinical Tumor stageB |  |
| cT1-cT2 | 72(50.35) |
| cT3-cT4 | 71(49.65) |
| Clinical Nodal statusB |  |
| cN0 | 19(13.29) |
| cN1-3 | 124(86.71) |
| Efficacy |  |
| Non-pCR | 93(65.03) |
| pCR | 50(34.97) |

Table 2 Caption: Clinic-pathological characteristics of the study population

### Table 3.

|   | pCR group | Non-pCR group | χ2 | Pearson pA | Logit pB | OR(95%CI) C |
| --- | --- | --- | --- | --- | --- | --- |
| N(%) | N(%) |  |  |  |  |  |
| Genotype Distribution |  |  |  |  |  |  |
| rs8135987 (T > C) |  |  |  |  |  |  |
| TT | 30(60.0) | 39(41.9) | - | - | - | - |
| TC | 16(32.0) | 42(45.2) | 3.450 | 0.063 | 0.033 | 0.30(0.09–0.90) |
| CC | 4(8.0) | 12(12.9) | 1.848 | 0.258 | 0.976 | 1.03(0.14–7.14) |
| TT | 30(60.0) | 39(41.9) | - | - | - | - |
| TC + CC | 20(40.0) | 54(58.1) | 4.250 | 0.039 | 0.060 | 0.37(0.13–1.04) |
| TT + TC | 46(92.0) | 81(87.1) | - | - | - | - |
| CC | 4(8.00) | 12(12.9) | 0.787 | 0.422 | 0.591 | 1.64(0.26–10.10) |
| TT + CC | 34(68.0) | 51(54.8) |   | - | - | - |
| TC | 16(32.0) | 42(45.1) | 2.336 | 0.126 | 0.029 | 0.30(0.10–0.88) |
| Allele |  |  |  |  |  |  |
| C | 24(24.0) | 66(35.5) | - | - | - | - |
| T | 76(76.0) | 120(64.5) | 3.977 | 0.046 | - | - |
| rs5751901 (T > C) |  |  |  |  |  |  |
| TT | 21(42.0) | 36(38.7) | - | - | - | - |
| TC | 20(40.0) | 42(45.2) | 0.276 | 0.599 | 0.475 | 0.68 (0.23–1.95) |
| CC | 9(18.0) | 15(16.1) | 0.003 | 0.955 | 0.629 | 1.42 (0.33–5.96) |
| TT | 21(42.0) | 36(38.7) | - | - | - | - |
| TC + CC | 29(58.0) | 57(61.2) | 0.147 | 0.702 | 0.705 | 0.83(0.31–2.18) |
| TT + TC | 41(82.0) | 78(83.9) | - | - | - | - |
| CC | 9(18.0) | 15(26.1) | 0.081 | 0.817 | 0.436 | 1.67(0.44–6.31) |
| TT + CC | 30(60.0) | 51(54.8) | - | - | - | - |
| TC | 20(40.0) | 42(45.2) | 0.352 | 0.553 | 0.352 | 0.62(0.23–1.68) |
| Allele |  |  |  |  |  |  |
| C | 38(38.0) | 72(38.7) | - | - | - | - |
| T | 62(62.0) | 114(61.3) | 0.138 | 0.906 | - | - |
| rs2017869 (G > C) |  |  |  |  |  |  |
| GG | 21(42.0) | 38(40.8) | - | - | - | - |
| GC | 21(42.0) | 40(43.0) | 0.018 | 0.893 | 0.800 | 0.87(0.31–2.49) |
| CC | 8(16.0) | 15(16.1) | 0.005 | 1.000 | 0.616 | 1.45(0.34–6.23) |
| GG | 21(42.0) | 38(40.9) | - | - | - | - |
| GC + CC | 29(58.0) | 55(59.1) | 0.017 | 0.895 | 0.998 | 0.99(0.37–2.61) |
| GG + GC | 42(84) | 78(83.9) | - | - | - | - |
| CC | 8(16) | 15(16.1) | 0.000 | 1.000 | 0.537 | 1.50(0.38–6.11) |
| GG + CC | 29(58.0) | 53(57.0) | - | - | - | - |
| GC | 21(42.0) | 40(43.0) | 0.014 | 0.907 | 0.662 | 0.80(0.29–2.16) |
| Allele |  |  |  |  |  |  |
| G | 63(63.0) | 116(62.4) | - | - | - | - |
| C | 37(37.0) | 70(37.6) | 0.011 | 0.916 | - | - |
| Haplotype (rs5751901—rs2017869) |  |  |  |  |  |  |
| TG | 42(58.3) | 78(58.2) | 0.000 | 0.984 | - | - |
| CC | 28(38.9) | 55(41.0) | 0.132 | 0.717 | - | - |
| CG | 2(2.8) | 1(0.8) | 0.004 | 0.952 | - | - |

Table 3 Caption: Association between selected SNPs and pCR

For SNP rs5751901 and rs2017869, no significant evidence supported a correlation between SNP genotypes, haplotypes and pCR outcomes (Table [3](#Tab3)).

### Relationship between three SNPs and adverse events of NAC

A total of 106 patients with available data of adverse events were included in analysis. For SNP rs2017869, multivariable analysis showed that the odds of neutropenia for CC genotype was 0.39 times the odds of neutropenia for TT + TC genotype (95% CI 0.08–0.84, *p* = 0.025) in recessive model. The the odds of leukopenia for CC genotype was 0.24 times the odds of leukopenia for TT + TC genotype (95% CI 0.08–0.78, *p* = 0.017) in recessive model. For SNP rs5751901, the CC genotype decreased the risk of ouccuring neutropenia (adjusted OR = 0.29, 95% CI 0.09–0.96, *p* = 0.036) and leukopenia (adjusted OR = 0.27, 95% CI 0.09–0.84, *p* = 0.024) compared with TT + TC genotype in the recessive model. For SNP rs8135987, it was pronounced correlated with grade 2 or greater peripheral neuropathy, the risk of which was significantly lower in TC genotype compared with the TT + CC genotype in the over-dominant model (adjusted OR = 0.39, 95% CI 0.15–0.96, *p* = 0.042). In addition, the CC genotype in the recessive model was an independent protective factor for leukopenia (adjusted OR = 0.16, 95% CI 0.05–0.68, *p* = 0.014), while it was an independent risk factor for elevated AST (adjusted OR = 4.5, 95% CI 1.11–18.27, *p* = 0.035). The detailed adverse events spectrum is shown in Table [4](#Tab4). In haplotype analysis for rs5751901 and rs2017869, the occurance of grade 2 or greater neutropenia for TG haplotype was increased by 3.05 times (adjusted OR = 4.05, 95% CI 1.27–12.92, *p* = 0.018) and the occurance of grade 2 or greater leukopenia for TG haplotype was increased by 3.26 times (adjusted OR = 4.26, 95% CI 1.36–13.30, *p* = 0.013) (Table [5](#Tab5)).

### Table 4.

| Toxic reactions | SNP | Genotypes | Toxicity grade |  | OR(95%CI) | pA |
| --- | --- | --- | --- | --- | --- | --- |
| Neutropenia |   |   | Grade < 2 | Grade ≥ 2 |   |   |
| rs8135987 (T > C) | TT | 9(42.9) | 38(44.7) |   |   |  |
|   | TC | 7(33.3) | 39(45.9) | 1.66(0.52–5.24) | 0.391 |  |
|   | CC | 5(23.8) | 8(9.4) | 0.47(0.11–2.04) | 0.314 |  |
|   | TT vs. TC + CC |   |   | 1.15(0.42–3.18) | 0.790 |  |
|   | TT + TC vs. CC |   |   | 0.38(0.09–1.54) | 0.175 |  |
|   | TT + CC vs. TC |   |   | 1.96(0.66–5.83) | 0.227 |  |
| rs5751901 (T > C) | TT | 6(28.6) | 36(42.4) |   |   |  |
|   | TC | 8(38.1) | 38(44.7) | 0.70(0.25–2.96) | 0.824 |  |
|   | CC | 7(33.3) | 11(12.9) | 0.26(0.07–1.01) | 0.052 |  |
|   | TT vs. TC + CC |   |   | 0.56(0.19–1.67) | 0.300 |  |
|   | TT + TC vs. CC |   |   | 0.29(0.09–0.96) | 0.036 |  |
|   | TT + CC vs. TC |   |   | 1.43(0.49–4.12) | 0.504 |  |
| rs2017869 (G > C) | GG | 7(33.3) | 37(43.5) |   |   |  |
|   | GC | 7(33.3) | 38(44.7) | 1.23(0.36–4.20) | 0.740 |  |
|   | CC | 7(33.3) | 10(11.8) | 0.28(0.08–1.04) | 0.059 |  |
|   | GG vs. GC + CC |   |   | 0.70(0.24–2.03) | 0.514 |  |
|   | GG + GC vs. CC |   |   | 0.39(0.08–0.84) | 0.025 |  |
|   | GG + CC vs. GC |   |   | 1.90(0.63–5.72) | 0.251 |  |
| Leukopenia |   |   | Grade < 2 | Grade ≥ 2 |   |   |
| rs8135987 (T > C) | TT | 8(33.3) | 39(47.6) |   |   |  |
|   | TC | 9(37.5) | 37(45.1) | 0.85(0.36–4.20) | 0.765 |  |
|   | CC | 7(29.2) | 6(7.3) | 0.16(0.05–0.68) | 0.014 |  |
|   | TT vs. TC + CC |   |   | 0.54(0.19–1.47) | 0.229 |  |
|   | TT + TC vs. CC |   |   | 0.17(0.04–0.67) | 0.012 |  |
|   | TT + CC vs. TC |   |   | 1.30(0.48–3.52) | 0.602 |  |
| rs5751901 (T > C) | TT | 8(33.3) | 34(41.5) |   |   |  |
|   | TC | 8(33.3) | 38(46.3) | 0.99(0.31–3.13) | 0.998 |  |
|   | CC | 8(33.3) | 10(12.2) | 0.27(0.07–0.96) | 0.045 |  |
|   | TT vs. TC + CC |   |   | 0.63(0.23–1.71) | 0.363 |  |
|   | TT + TC vs. CC |   |   | 0.27(0.09–0.84) | 0.024 |  |
|   | TT + CC vs. TC |   |   | 1.59(0.58–4.34) | 0.370 |  |
| rs2017869 (G > C) | GG | 9(37.5) | 35(42.7) |   |   |  |
|   | GC | 7(29.2) | 38(46.3) | 1.29 (0.40–4.13) | 0.663 |  |
|   | CC | 8(33.3) | 9(11.0) | 0.27 (0.08–0.97) | 0.045 |  |
|   | GG vs. GC + CC |   |   | 0.71(0.27–1.95) | 0.523 |  |
|   | GG + GC vs. CC |   |   | 0.24(0.08–0.78) | 0.017 |  |
|   | GG + CC vs. GC |   |   | 1.97(0.69–5.64) | 0.204 |  |
| Peripheral neuropathy |   |   | Grade < 2 | Grade ≥ 2 |   |   |
| rs8135987 (T > C) | TT | 26(39.4) | 21(52.5) |   |   |  |
|   | TC | 34(51.5) | 12(30.0) | 0.43(0.17–1.09) | 0.078 |  |
|   | CC | 6(9.1) | 7(17.5) | 1.70(0.41–7.07) | 0.459 |  |
|   | TT vs. TC + CC |   |   | 0.60(0.26–1.40) | 0.236 |  |
|   | TT + TC vs. CC |   |   | 2.42(0.61–9.59) | 0.206 |  |
|   | TT + CC vs. TC |   |   | 0.39(0.15–0.96) | 0.042 |  |
| rs5751901 (T > C) | TT | 27(40.9) | 15(37.5) |   |   |  |
|   | TC | 30(45.5) | 16(40.0) | 1.95(0.59–6.45) | 0.270 |  |
|   | CC | 9(13.6) | 9(22.5) | 0.96(0.37–2.47) | 0.928 |  |
|   | TT vs. TC + CC |   |   | 1.19 (0.50–2.85) | 0.688 |  |
|   | TT + TC vs. CC |   |   | 2.00(0.67–5.96) | 0.213 |  |
|   | TT + CC vs. TC |   |   | 0.77(0.32–1.83) | 0.561 |  |
| rs2017869 (G > C) | GG | 28(42.4) | 16(40.0) |   |   |  |
|   | GC | 29(43.9) | 16(40.0) | 0.96(0.37–2.49) | 0.939 |  |
|   | CC | 9(13.6) | 8(20.0) | 0.69(0.50–5.65) | 0.393 |  |
|   | GG vs. GC + CC |   |   | 1.15(0.48–2.73) | 0.757 |  |
|   | GG + GC vs. CC |   |   | 1.72(0.56–5.25) | 0.340 |  |
|   | GG + CC vs. GC |   |   | 0.83(0.34–1.98) | 0.668 |  |
| AST increased |   |   | Grade < 1 | Grade ≥ 1 |   |   |
| rs8135987 (T > C) | TT | 29(51.8) | 18(36.0) |   |   |  |
|   | TC | 22(29.3) | 24(48.0) | 1.50(0.60–3.66) | 0.386 |  |
|   | CC | 5(8.9) | 8(16.0) | 5.40(1.24–23.4) | 0.024 |  |
|   | TT vs. TC + CC |   |   | 1.94(0.84–4.50) | 0.122 |  |
|   | TT + TC vs. CC |   |   | 4.50(1.11–18.27) | 0.035 |  |
|   | TT + CC vs. TC |   |   | 1.09(0.46–2.53) | 0.844 |  |
| rs5751901 (T > C) | TT | 25(44.6) | 17(34.0) |   |   |  |
|   | TC | 22(39.3) | 24(48.0) | 1.67(0.50–5.47) | 0.401 |  |
|   | CC | 9(16.1) | 9(18.0) | 1.28(0.51–3.20) | 0.591 |  |
|   | TT vs. TC + CC |   |   | 1.39(0.59–3.22) | 0.448 |  |
|   | TT + TC vs. CC |   |   | 1.47(0.49–4.41) | 0.493 |  |
|   | TT + CC vs. TC |   |   | 1.10(0.48–2.55) | 0.818 |  |
| rs2017869 (G > C) | GG | 26(49.1) | 18(36.0) |   |   |  |
|   | GC | 22(39.3) | 23(46.0) | 1.13(0.45–2.86) | 0.784 |  |
|   | CC | 8(14.3) | 9(18.0) | 1.96(0.58–6.60) | 0.279 |  |
|   | GG vs. GC + CC |   |   | 1.34(.057–3.10) | 0.500 |  |
|   | GG + GC vs. CC |   |   | 1.84(4.58–5.77) | 0.293 |  |
|   | GG + CC vs. GC |   |   | 0.95(0.40–2.23) | 0.904 |  |

Table 4 Caption: Association between SNPs and patients’ adverse events during NAC

### Table 5.

| Haplotype | N(%) |  | χ2 | Pearson pA | Logit pB | OR(95%CI) C |
| --- | --- | --- | --- | --- | --- | --- |
| TG | 89 (61.8) |  |   |   |   |   |
| CC | 61 (36.8) |  |   |   |   |   |
| CG | 3 (1.4) |  |   |   |   |   |
| Neutropenia | Grade < 2 | Grade ≥ 2 |   |   |   |   |
| TG | 14 (15.7) | 75(84.3) | 5.818 | 0.016 | 0.018 | 4.05(1.27–12.92) |
| CC | 14(22.9) | 47(77.1) | 0.892 | 0.345 | 0.413 | 0.65(0.23–1.83) |
| CG | 1(33.3) | 2(66.7) | 0.355 | 0.551 | 0.393 | 0.33(0.03–4.23) |
| Leukopenia | Grade < 2 | Grade ≥ 2 |   |   |   |   |
| TG | 16(18.0) | 73(82.0) | 6.892 | 0.009 | 0.013 | 4.26(1.36–13.30) |
| CC | 15(24.6) | 46(75.4) | 0.312 | 0.577 | 0.404 | 0.65(0.24–1.77) |
| CG | 1(33.3) | 2(66.7) | 0.202 | 0.541 | 0.605 | 0.51(0.34–6.56) |
| Peripheral neuropathy | Grade < 2 | Grade ≥ 2 |   |   |   |   |
| TG | 58(65.2) | 31(34.8) | 1.992 | 0.158 | 0.142 | 0.43(0.14–1.32) |
| CC | 37(60.7) | 24(39.3) | 0.158 | 0.691 | 0.512 | 1.34(0.56–3.21) |
| CG | 1(33.3) | 2(66.7) | 1.100 | 0.294 | 0.371 | 3.18(0.25–40.30) |
| AST increased | Grade < 1 | Grade ≥ 1 |   |   |   |   |
| TG | 48 (53.9) | 41(46.1) | 0.271 | 0.603 | 0.411 | 0.63(0.21–1.89) |
| CC | 30(49.2) | 31(50.8) | 0.768 | 0.381 | 0.596 | 1.25(0.55–2.88) |
| CG | 2(66.7) | 1(33.3) | 0.237 | 0.626 | 0.602 | 0.51(0.39–6.59) |

Table 5 Caption: Occurrence of adverse events during NAC according to haplotypes

### GGT SNPs and serum GGT level

Patients were divided into a high-level group (≥ 29 U/L) and a low-level group (< 29 U/L) according to the serum GGT level according to our previous study [[5](#CR5)]. For SNP rs8135987, the TC genotype of both the dominant model (adjusted OR = 3.11, 95% CI 1.07–9.02, *p* = 0.036) and the co-dominant model (adjusted OR = 2.20, 95% CI 1.25–12.62, *p* = 0.019) were significantly associated with elevated serum GGT level in multivariable analysis. For SNPs rs2017869, the CC genotype of the recessive model was significantly related with higher serum GGT level (adjusted OR = 3.09, 95% CI 1.02–9.36, *p* = 0.046) in multivariable analysis. For SNPs rs5751901, the distribution of the CC genotype of co-dominant model (*p* = 0.034) and recessive models (*p* = 0.028) were both significantly difference among GGT levels, while the difference was not statistically significant in multivariable analysis (Table [6](#Tab6)).

### Table 6.

| Genotypes | Serum GGT level N (%) |  | pA | pB | OR(95%CI) C |
| --- | --- | --- | --- | --- | --- |
| Low level < 29 U/L | High level ≥ 29U/L |  |  |  |  |
| rs8135987 (T > C) |  |  |  |  |  |
| TT | 60(52.6) | 9(31.0) |   |   |   |
| TC | 43(37.7) | 15(51.7) | 0.073 | 0.019 | 2.20(1.25–12.62) |
| CC | 11(9.7) | 5(17.3) | 0.127 | 0.351 | 3.97(0.41–11.59) |
| TT vs. TC + CC |   |   | 0.060 | 0.036 | 3.11(1.07–9.02) |
| TT + TC vs. CC |   |   | 0.319 | 0.791 | 1.22(0.27–5.45) |
| TT + CC vs. TC |   |   | 0.170 | 0.050 | 2.76(0.99–7.61) |
| rs5751901 (T > C) |  |  |  |  |  |
| TT | 49(43.0) | 8(27.6) |   |   |   |
| TC | 50(43.9) | 12(41.4) | 0.472 | 0.497 | 1.48(0.47–4.64) |
| CC | 15(13.1) | 9(31.0) | 0.034 | 0.068 | 3.32(0.91–12.13) |
| TT vs. TC + CC |   |   | 0.144 | 0.262 | 1.79(0.65–4.98) |
| TT + TC vs. CC |   |   | 0.028 | 0.067 | 2.78(0.93–8.29) |
| TT + CC vs. TC |   |   | 0.810 | 0.714 | 0.84(0.32–2.17) |
| rs2017869 (G > C) |  |  |  |  |  |
| GG | 50(43.8) | 9(31.0) |   |   |   |
| GC | 50(43.8) | 11(38.0) | 0.808 | 0.825 | 3.32(0.93–11.91) |
| CC | 14(12.4) | 9(31.0) | 0.035 | 0.065 | 1.14(0.36–3.54) |
| GG vs. GC + CC |   |   | 0.291 | 0.442 | 1.47(0.54–3.96) |
| GG + GC vs. CC |   |   | 0.022 | 0.046 | 3.09(1.02–9.36) |
| GG + CC vs. GC |   |   | 0.564 | 0.411 | 0.67(0.25–1.76) |
| Haplotype |  |  |  |  |  |
| TG |   |   | 0.059 | 0.157 | 0.44(0.14–1.37) |
| CC |   |   | 0.361 | 0.477 | 1.44(0.53–3.95) |
| CG |   |   | 0.570 | 0.677 | 1.74(0.13–23.40) |

Table 6 Caption: Association between SNPs and pre-therapeutic serum GGT level

## Discussion

To the best of our knowledge, this study is the first one to explore the relationship between *GGT1* gene SNPs and NAC efficacy and adverse events, as well as serum GGT levels in breast cancer. Here we found out that *GGT1* gene SNPs have potential value in predicting the efficacy and tolerability of neoadjuvant chemotherapy. In our findings, we revealed for the first time the relationship between SNP rs8135987 (T > C) and NAC efficacy, adverse events, and serum GGT level. The TC genotype of SNP rs8135987 showed negative relation to pCR in both the over-dominant model and co-dominant model. Furthermore, we found that the TC genotype of SNP rs8135987 was an independent protective factor for the occurrence of peripheral neuropathy. Thus, the TC genotype of rs8135987 may be a novel biomarker of the resistance to NAC, which may lead to reduced treatment sensitivity as well as the incidence of adverse events.

Based on previous studies, SNPs in and near *GGT1* gene could influence the expression level and activity of serum GGT [[18](#CR18), [19](#CR19)]. The observed genetic covariance in a twin study indicated that plasma GGT levels may caused by genetics [[20](#CR20)]. The serum GGT is extremely important in mediating the intracellular glutathione (GSH) levels. The most critical biological function of GSH is anti-oxidation and neutralizing free radicals [[21](#CR21)]. Thus, serum GGT plays a vital role in protecting the cell against oxidative stress and further resisting the toxicity of the promoting agents, which means it may enhance the resistance to pro-oxidant cancer therapy [[22](#CR22)]. Several previous studies had confirmed that GGT can influence the sensitivity of tumor cells to drugs [[3](#CR3), [23](#CR23)]. Our previous study also illuminated the predictive value of serum GGT level in NAC [[5](#CR5)]. In this study, we found that the TC genotype of rs8135987 was associated with elevated GGT level. It may revealed the underlying mechanisms of the insensitivity of TC genotype of SNP rs8135987 to NAC in breast cancer. In addition, the recessive models of rs2017869 and rs5751901 were both significantly associated with pre-treatment serum GGT levels, consisting with the results of other research [[10](#CR10)], which found that each minor allele of rs5751901 was associated with a 0.21 standard deviation increase in GGT1 protein level. Moreover, Sciskalska et al. also found that SNP rs5751901 may cause changes in GGT activity. The TC genotype for SNP rs5751901 had an increased blood GGT activity compared to with CC genotypes in smokers [[24](#CR24)].

Our study was the first to find out the relationship between *GGT1* gene SNPs and the neurotoxicity and hematotoxicity of chemotherapy. In this study, we found that TC genotype of SNP rs8135987 was an independent protective factor for the occurrence of peripheral neuropathy. What’s more, significant negative correlation exists between the recessive models of rs2017869, rs5751901 and neutropenia as well as leukopenia, suggesting that SNPs rs2017869 and rs5751901 have certain predictive value for patient’s tolerability of NAC. Haplotype analysis also confirmed this result, the TG haplotype of rs5751901 and rs2017869 had an increased occurrence of neutropenia and leukopenia. Practically, the tolerability to chemotherapy may be related to the metabolism of the drugs in vivo. Khrunin et al. found that the TT genotype of rs5751901 was significantly associated with cisplatin nephrotoxicity in patients with ovarian cancer. The frequency of TT genotype was approximately 50% in patients with renal failure, while it was only 31% in those with normal renal function [[9](#CR9)]. However, all patients in our study received small-dose weekly cisplatin combined with paclitaxel, none of them had severe renal impairment, and that might be why there was no significant correlation between the *GGT1* gene SNPs and elevated creatinine (Supplementary Table [2](#MOESM1)).

The selected SNP loci fall within the non-coding regions of GGT1 gene. As we know, introns’ sequences account for nearly 24% of the entire human genome [[25](#CR25)]. The human genome contains millions of SNPs and many of them are intronic and have unknown functional significance. Previous study showed that intron-located SNPs affect splicing, alternative splicing and splicing efficiency and confer risk for the development of different multifactorial human diseases [[26](#CR26)–[29](#CR29)]. A GWAS study suggested that SNPs located in the intron may as well alter the protein levels by the linkage disequilibrium with adjacent alleles or regulating gene expression in ways of affecting mRNA splicing. SNP rs5751901 was found to be in linkage disequilibrium with rs6519519 (r^2^ = 0.71), which was related to the transcript abundance of *GGT1* gene [[11](#CR11)]. SNP rs5751901 was also in linkage disequilibrium with rs4820599, which was located in the *GGT1* gene transcript region as a potential transcriptional binding site [[18](#CR18)]. SNP rs8135987 is located in an intron of *GGT1* gene and within 2 kb of exon 9, which is found to be crucial for substrate binding and catalysis [[30](#CR30)]. These might be the potential biological mechanisms by which these non-coding SNPs affected serum GGT protein level and thus affect pCR outcome and adverse events. This hypothesis might be further studied and validated in the future.

A limitation of our study is the relatively small sample size, which might undermine the statistical power. However, since the object of the study was the patients undergoing prospective clinical studies, who have complete and highly reliable clinical and pathological information, our results can prompt an inherent law for predicting chemotherapeutic sensitivity. Nevertheless, it is necessary to expand the number of the samples or to conduct multi-center prospective studies in the future, so as to better verify the predictive value of *GGT1* gene varieties in NAC of breast cancer.

In summary, there is convincing evidence that *GGT1* gene SNPs were potential markers in predicting the therapeutic effect and adverse events of NAC. Moreover, the correlation between *GGT1* gene SNPs and serum GGT protein level was verified. This study represents a step forward toward a better understanding of the effect of *GGT1* gene in breast cancer, providing a theoretical and clinical basis for individualized treatment.

## Conclusions

The *GGT1* gene SNPs might be a novel biomarker of the sensitivity of NAC in breast cancer patients, providing theoretical basis for further precision therapy.

## Supplementary Information

## Acknowledgements

Not applicable.

## Abbreviations

## Authors’ contributions

LS, ZPW contributed equally to this study. LS: Conceptualization, Data curation, Investigation, Methodology, Visualization, Writing—original draft; LHZ: Writing—review & editing; ZPW: Formal analysis, Funding acquisition, Writing—review & editing; YPL,YMY: Resources; SGX: Data curation; WJY: Conceptualization, Funding acquisition; JSL: Conceptualization, Funding acquisition, Project administration, Writing—review & editing. The author(s) read and approved the final manuscript.

## Funding

This study was funded by National Natural Science Foundation of China (No. 82103695 and 82,173,115), Clinical Research Plan of Shanghai Hospital Development Center (No. SHDC2020CR3003A), Science and Technology Commission of Shanghai Municipality (No. 20DZ2201600), Beijing Foundation of Medicine Award (No. YXJL-2020–0941-0737), Shanghai Municipal Key Clinical Specialty, Shanghai ‘Rising Stars of Medical Talent’ Youth Development Program for Outstanding Youth Medical Talents (No. 2018–16), Multidisciplinary Cross Research Foundation of Shanghai Jiao Tong University (No. YG2019QNA28), Clinical Research Innovation Nurturing Fund of Renji Hospital and United Imaging (No. 2021RJLY-002), Nurturing Fund of Renji Hospital (No. PYIII20-09, PY2018-III-15 and RJPY-LX-002).

## Availability of data and materials

The data that support the findings of this study are available from the corresponding author, Jingsong Lu, upon reasonable request.

## Declarations

### Ethics approval and consent to participate

All procedures performed in studies involving human participants were in accordance with the ethical standards of the institutional and/or national research committee and with the 1964 Helsinki declaration and its later amendments or comparable ethical standards. Both clinical trials (SHPD001 and SHPD002) have been approved by the Ethics Committee of Renji Hospital, School of Medicine, Shanghai Jiaotong University. The informed consent was obtained from all subjects and/or their legal guardian(s).

### Consent for publication

Not applicable.

### Competing interests

The authors declare no competing interests.

## Footnotes

## Contributor Information

Liheng Zhou, Email: zhouliheng@renji.com.

Jingsong Lu, Email: lujingsong@renji.com.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

The data that support the findings of this study are available from the corresponding author, Jingsong Lu, upon reasonable request.

### Supplementary Materials

### Data Availability Statement

The data that support the findings of this study are available from the corresponding author, Jingsong Lu, upon reasonable request.

## References

1. Siegel RL, Miller KD, Fuchs HE, Jemal A. Cancer statistics, 2022. CA Cancer J Clin. 2022;72(1):7–33. doi: 10.3322/caac.21708.  [DOI](https://doi.org/10.3322/caac.21708) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35020204/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=CA%20Cancer%20J%20Clin&title=Cancer%20statistics,%202022&author=RL%20Siegel&author=KD%20Miller&author=HE%20Fuchs&author=A%20Jemal&volume=72&issue=1&publication_year=2022&pages=7-33&pmid=35020204&doi=10.3322/caac.21708&)

2. Cortazar P, Zhang L, Untch M, Mehta K, Costantino JP, Wolmark N, Bonnefoi H, Cameron D, Gianni L, Valagussa P, et al. Pathological complete response and long-term clinical benefit in breast cancer: the CTNeoBC pooled analysis. Lancet. 2014;384(9938):164–172. doi: 10.1016/S0140-6736(13)62422-8.  [DOI](https://doi.org/10.1016/S0140-6736(13)62422-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24529560/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Pathological%20complete%20response%20and%20long-term%20clinical%20benefit%20in%20breast%20cancer:%20the%20CTNeoBC%20pooled%20analysis&author=P%20Cortazar&author=L%20Zhang&author=M%20Untch&author=K%20Mehta&author=JP%20Costantino&volume=384&issue=9938&publication_year=2014&pages=164-172&pmid=24529560&doi=10.1016/S0140-6736(13)62422-8&)

3. Pompella A, De Tata V, Paolicchi A, Zunino F. Expression of gamma-glutamyltransferase in cancer cells and its significance in drug resistance. Biochem Pharmacol. 2006;71(3):231–238. doi: 10.1016/j.bcp.2005.10.005.  [DOI](https://doi.org/10.1016/j.bcp.2005.10.005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16303117/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20Pharmacol&title=Expression%20of%20gamma-glutamyltransferase%20in%20cancer%20cells%20and%20its%20significance%20in%20drug%20resistance&author=A%20Pompella&author=V%20De%20Tata&author=A%20Paolicchi&author=F%20Zunino&volume=71&issue=3&publication_year=2006&pages=231-238&pmid=16303117&doi=10.1016/j.bcp.2005.10.005&)

4. Sun L, Yin W, Wu Z, Wang Y, Lu J. The predictive value of pre-therapeutic serum gamma-glutamyl transferase in efficacy and adverse reactions to neoadjuvant chemotherapy among breast cancer patients. J Breast Cancer. 2020;23(5):509–520. doi: 10.4048/jbc.2020.23.e59.  [DOI](https://doi.org/10.4048/jbc.2020.23.e59) | [PMC free article](/articles/PMC7604370/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33154826/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Breast%20Cancer&title=The%20predictive%20value%20of%20pre-therapeutic%20serum%20gamma-glutamyl%20transferase%20in%20efficacy%20and%20adverse%20reactions%20to%20neoadjuvant%20chemotherapy%20among%20breast%20cancer%20patients&author=L%20Sun&author=W%20Yin&author=Z%20Wu&author=Y%20Wang&author=J%20Lu&volume=23&issue=5&publication_year=2020&pages=509-520&pmid=33154826&doi=10.4048/jbc.2020.23.e59&)

5. Diergaarde B, Brand R, Lamb J, Cheong SY, Stello K, Barmada MM, Feingold E, Whitcomb DC. Pooling-based genome-wide association study implicates gamma-glutamyltransferase 1 (GGT1) gene in pancreatic carcinogenesis. Pancreatology. 2010;10(2–3):194–200. doi: 10.1159/000236023.  [DOI](https://doi.org/10.1159/000236023) | [PMC free article](/articles/PMC2899150/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20484958/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pancreatology&title=Pooling-based%20genome-wide%20association%20study%20implicates%20gamma-glutamyltransferase%201%20(GGT1)%20gene%20in%20pancreatic%20carcinogenesis&author=B%20Diergaarde&author=R%20Brand&author=J%20Lamb&author=SY%20Cheong&author=K%20Stello&volume=10&issue=2%E2%80%933&publication_year=2010&pages=194-200&pmid=20484958&doi=10.1159/000236023&)

6. Brand H, Diergaarde B, O'Connell MR, Whitcomb DC, Brand RE. Variation in the gamma-glutamyltransferase 1 gene and risk of chronic pancreatitis. Pancreas. 2013;42(5):836–840. doi: 10.1097/MPA.0b013e318279f720.  [DOI](https://doi.org/10.1097/MPA.0b013e318279f720) | [PMC free article](/articles/PMC3676446/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23462328/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pancreas&title=Variation%20in%20the%20gamma-glutamyltransferase%201%20gene%20and%20risk%20of%20chronic%20pancreatitis&author=H%20Brand&author=B%20Diergaarde&author=MR%20O'Connell&author=DC%20Whitcomb&author=RE%20Brand&volume=42&issue=5&publication_year=2013&pages=836-840&pmid=23462328&doi=10.1097/MPA.0b013e318279f720&)

7. Perez-Ramirez C, Canadas-Garre M, Molina MA, Cabeza Barrera J, Faus-Dader MJ. Impact of single nucleotide polymorphisms on the efficacy and toxicity of EGFR tyrosine kinase inhibitors in advanced non-small cell lung cancer patients. Mutat Res. 2019;781:63–70. doi: 10.1016/j.mrrev.2019.04.001.  [DOI](https://doi.org/10.1016/j.mrrev.2019.04.001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31416579/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mutat%20Res&title=Impact%20of%20single%20nucleotide%20polymorphisms%20on%20the%20efficacy%20and%20toxicity%20of%20EGFR%20tyrosine%20kinase%20inhibitors%20in%20advanced%20non-small%20cell%20lung%20cancer%20patients&author=C%20Perez-Ramirez&author=M%20Canadas-Garre&author=MA%20Molina&author=J%20Cabeza%20Barrera&author=MJ%20Faus-Dader&volume=781&publication_year=2019&pages=63-70&pmid=31416579&doi=10.1016/j.mrrev.2019.04.001&)

8. Maeng CH, Yi JH, Lee J, Hong JY, Choi MK, Jung HA, Park JO, Park SH, Park YS, Kang WK, et al. Effects of single nucleotide polymorphisms on treatment outcomes and toxicity in patients treated with sunitinib. Anticancer Res. 2013;33(10):4619–4626.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/24123039/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anticancer%20Res&title=Effects%20of%20single%20nucleotide%20polymorphisms%20on%20treatment%20outcomes%20and%20toxicity%20in%20patients%20treated%20with%20sunitinib&author=CH%20Maeng&author=JH%20Yi&author=J%20Lee&author=JY%20Hong&author=MK%20Choi&volume=33&issue=10&publication_year=2013&pages=4619-4626&pmid=24123039&)

9. Giovannetti E, Zucali PA, Peters GJ, Cortesi F, D'Incecco A, Smit EF, Falcone A, Burgers JA, Santoro A, Danesi R, et al. Association of polymorphisms in AKT1 and EGFR with clinical outcome and toxicity in non-small cell lung cancer patients treated with gefitinib. Mol Cancer Ther. 2010;9(3):581–593. doi: 10.1158/1535-7163.MCT-09-0665.  [DOI](https://doi.org/10.1158/1535-7163.MCT-09-0665) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20159991/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Cancer%20Ther&title=Association%20of%20polymorphisms%20in%20AKT1%20and%20EGFR%20with%20clinical%20outcome%20and%20toxicity%20in%20non-small%20cell%20lung%20cancer%20patients%20treated%20with%20gefitinib&author=E%20Giovannetti&author=PA%20Zucali&author=GJ%20Peters&author=F%20Cortesi&author=A%20D'Incecco&volume=9&issue=3&publication_year=2010&pages=581-593&pmid=20159991&doi=10.1158/1535-7163.MCT-09-0665&)

10. Khrunin AV, Moiseev AA, Khokhrin DV, Gorbunova VA, Limborska SA. Polymorphisms of OCT2, GGT1, HO1, and DNASE1 genes and nephrotoxicity of cysplatin in ovarian cancer patients. Mol Genet Microbiol Virol. 2010;25(4):163–166. doi: 10.3103/S089141681004004X.  [DOI](https://doi.org/10.3103/S089141681004004X) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Genet%20Microbiol%20Virol&title=Polymorphisms%20of%20OCT2,%20GGT1,%20HO1,%20and%20DNASE1%20genes%20and%20nephrotoxicity%20of%20cysplatin%20in%20ovarian%20cancer%20patients&author=AV%20Khrunin&author=AA%20Moiseev&author=DV%20Khokhrin&author=VA%20Gorbunova&author=SA%20Limborska&volume=25&issue=4&publication_year=2010&pages=163-166&doi=10.3103/S089141681004004X&)

11. Melzer D, Perry JR, Hernandez D, Corsi AM, Stevens K, Rafferty I, Lauretani F, Murray A, Gibbs JR, Paolisso G, et al. A genome-wide association study identifies protein quantitative trait loci (pQTLs) PLoS Genet. 2008;4(5):e1000072. doi: 10.1371/journal.pgen.1000072.  [DOI](https://doi.org/10.1371/journal.pgen.1000072) | [PMC free article](/articles/PMC2362067/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18464913/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet&title=A%20genome-wide%20association%20study%20identifies%20protein%20quantitative%20trait%20loci%20(pQTLs)&author=D%20Melzer&author=JR%20Perry&author=D%20Hernandez&author=AM%20Corsi&author=K%20Stevens&volume=4&issue=5&publication_year=2008&pages=e1000072&pmid=18464913&doi=10.1371/journal.pgen.1000072&)

12. Kim YJ, Go MJ, Hu C, Hong CB, Kim YK, Lee JY, Hwang JY, Oh JH, Kim DJ, Kim NH, et al. Large-scale genome-wide association studies in East Asians identify new genetic loci influencing metabolic traits. Nat Genet. 2011;43(10):990–995. doi: 10.1038/ng.939.  [DOI](https://doi.org/10.1038/ng.939) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21909109/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Large-scale%20genome-wide%20association%20studies%20in%20East%20Asians%20identify%20new%20genetic%20loci%20influencing%20metabolic%20traits&author=YJ%20Kim&author=MJ%20Go&author=C%20Hu&author=CB%20Hong&author=YK%20Kim&volume=43&issue=10&publication_year=2011&pages=990-995&pmid=21909109&doi=10.1038/ng.939&)

13. Zhou L, Xu S, Yin W, Lin Y, Du Y, Jiang Y, Wang Y, Zhang J, Wu Z, Lu J. Weekly paclitaxel and cisplatin as neoadjuvant chemotherapy with locally advanced breast cancer: a prospective, single arm, phase II study. Oncotarget. 2017;8(45):79305–79314. doi: 10.18632/oncotarget.17954.  [DOI](https://doi.org/10.18632/oncotarget.17954) | [PMC free article](/articles/PMC5668042/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29108309/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncotarget&title=Weekly%20paclitaxel%20and%20cisplatin%20as%20neoadjuvant%20chemotherapy%20with%20locally%20advanced%20breast%20cancer:%20a%20prospective,%20single%20arm,%20phase%20II%20study&author=L%20Zhou&author=S%20Xu&author=W%20Yin&author=Y%20Lin&author=Y%20Du&volume=8&issue=45&publication_year=2017&pages=79305-79314&pmid=29108309&doi=10.18632/oncotarget.17954&)

14. Bayarmaa B, Wu Z, Peng J, Wang Y, Xu S, Yan T, Yin W, Lu J, Zhou L. Association of LncRNA MEG3 polymorphisms with efficacy of neoadjuvant chemotherapy in breast cancer. BMC Cancer. 2019;19(1):877. doi: 10.1186/s12885-019-6077-3.  [DOI](https://doi.org/10.1186/s12885-019-6077-3) | [PMC free article](/articles/PMC6727505/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31488093/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Cancer&title=Association%20of%20LncRNA%20MEG3%20polymorphisms%20with%20efficacy%20of%20neoadjuvant%20chemotherapy%20in%20breast%20cancer&author=B%20Bayarmaa&author=Z%20Wu&author=J%20Peng&author=Y%20Wang&author=S%20Xu&volume=19&issue=1&publication_year=2019&pages=877&pmid=31488093&doi=10.1186/s12885-019-6077-3&)

15. Shaw LM, Stromme JH, London JL, Theodorsen L. International Federation of Clinical Chemistry, (IFCC), Scientific Committee, Analytical Section. IFCC methods for the measurement of catalytic concentration of enzymes. Part 4. IFCC method for gamma-glutamyltransferase [(gamma-glutamyl)-peptide: amino acid gamma-glutamyltransferase, EC 2.3.2.2] J Clin Chem Clin Biochem. 1983;21(10):633–646.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/6139407/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Chem%20Clin%20Biochem&title=International%20Federation%20of%20Clinical%20Chemistry,%20(IFCC),%20Scientific%20Committee,%20Analytical%20Section.%20IFCC%20methods%20for%20the%20measurement%20of%20catalytic%20concentration%20of%20enzymes.%20Part%204.%20IFCC%20method%20for%20gamma-glutamyltransferase%20%5B(gamma-glutamyl)-peptide:%20amino%20acid%20gamma-glutamyltransferase,%20EC%202.3.2.2%5D&author=LM%20Shaw&author=JH%20Stromme&author=JL%20London&author=L%20Theodorsen&volume=21&issue=10&publication_year=1983&pages=633-646&pmid=6139407&)

16. Horita N, Kaneko T. Genetic model selection for a case-control study and a meta-analysis. Meta Gene. 2015;5:1–8. doi: 10.1016/j.mgene.2015.04.003.  [DOI](https://doi.org/10.1016/j.mgene.2015.04.003) | [PMC free article](/articles/PMC4443430/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26042205/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Meta%20Gene&title=Genetic%20model%20selection%20for%20a%20case-control%20study%20and%20a%20meta-analysis&author=N%20Horita&author=T%20Kaneko&volume=5&publication_year=2015&pages=1-8&pmid=26042205&doi=10.1016/j.mgene.2015.04.003&)

17. Wigginton JE, Cutler DJ, Abecasis GR. A note on exact tests of hardy-weinberg equilibrium. Am J Hum Genet. 2005;76(5):887–893. doi: 10.1086/429864.  [DOI](https://doi.org/10.1086/429864) | [PMC free article](/articles/PMC1199378/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15789306/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=A%20note%20on%20exact%20tests%20of%20hardy-weinberg%20equilibrium&author=JE%20Wigginton&author=DJ%20Cutler&author=GR%20Abecasis&volume=76&issue=5&publication_year=2005&pages=887-893&pmid=15789306&doi=10.1086/429864&)

18. Yuan X, Waterworth D, Perry JR, Lim N, Song K, Chambers JC, Zhang W, Vollenweider P, Stirnadel H, Johnson T, et al. Population-based genome-wide association studies reveal six loci influencing plasma levels of liver enzymes. Am J Hum Genet. 2008;83(4):520–528. doi: 10.1016/j.ajhg.2008.09.012.  [DOI](https://doi.org/10.1016/j.ajhg.2008.09.012) | [PMC free article](/articles/PMC2561937/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18940312/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Population-based%20genome-wide%20association%20studies%20reveal%20six%20loci%20influencing%20plasma%20levels%20of%20liver%20enzymes&author=X%20Yuan&author=D%20Waterworth&author=JR%20Perry&author=N%20Lim&author=K%20Song&volume=83&issue=4&publication_year=2008&pages=520-528&pmid=18940312&doi=10.1016/j.ajhg.2008.09.012&)

19. Kamatani Y, Matsuda K, Okada Y, Kubo M, Hosono N, Daigo Y, Nakamura Y, Kamatani N. Genome-wide association study of hematological and biochemical traits in a Japanese population. Nat Genet. 2010;42(3):210–215. doi: 10.1038/ng.531.  [DOI](https://doi.org/10.1038/ng.531) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20139978/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Genome-wide%20association%20study%20of%20hematological%20and%20biochemical%20traits%20in%20a%20Japanese%20population&author=Y%20Kamatani&author=K%20Matsuda&author=Y%20Okada&author=M%20Kubo&author=N%20Hosono&volume=42&issue=3&publication_year=2010&pages=210-215&pmid=20139978&doi=10.1038/ng.531&)

20. Loomba R, Rao F, Zhang L, Khandrika S, Ziegler MG, Brenner DA, O'Connor DT. Genetic covariance between gamma-glutamyl transpeptidase and fatty liver risk factors: role of beta2-adrenergic receptor genetic variation in twins. Gastroenterology. 2010;139(3):836–845 845 e831. doi: 10.1053/j.gastro.2010.06.009.  [DOI](https://doi.org/10.1053/j.gastro.2010.06.009) | [PMC free article](/articles/PMC3038676/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20537997/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastroenterology&title=Genetic%20covariance%20between%20gamma-glutamyl%20transpeptidase%20and%20fatty%20liver%20risk%20factors:%20role%20of%20beta2-adrenergic%20receptor%20genetic%20variation%20in%20twins&author=R%20Loomba&author=F%20Rao&author=L%20Zhang&author=S%20Khandrika&author=MG%20Ziegler&volume=139&issue=3&publication_year=2010&pages=836-845%20845%20e831&pmid=20537997&doi=10.1053/j.gastro.2010.06.009&)

21. Lim JS, Yang JH, Chun BY, Kam S, Jacobs DR, Jr, Lee DH. Is serum gamma-glutamyltransferase inversely associated with serum antioxidants as a marker of oxidative stress? Free Radic Biol Med. 2004;37(7):1018–1023. doi: 10.1016/j.freeradbiomed.2004.06.032.  [DOI](https://doi.org/10.1016/j.freeradbiomed.2004.06.032) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15336318/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Free%20Radic%20Biol%20Med&title=Is%20serum%20gamma-glutamyltransferase%20inversely%20associated%20with%20serum%20antioxidants%20as%20a%20marker%20of%20oxidative%20stress?&author=JS%20Lim&author=JH%20Yang&author=BY%20Chun&author=S%20Kam&author=DR%20Jacobs&volume=37&issue=7&publication_year=2004&pages=1018-1023&pmid=15336318&doi=10.1016/j.freeradbiomed.2004.06.032&)

22. Hanigan MH. Gamma-glutamyl transpeptidase: redox regulation and drug resistance. Adv Cancer Res. 2014;122:103–141. doi: 10.1016/B978-0-12-420117-0.00003-7.  [DOI](https://doi.org/10.1016/B978-0-12-420117-0.00003-7) | [PMC free article](/articles/PMC4388159/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24974180/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Adv%20Cancer%20Res&title=Gamma-glutamyl%20transpeptidase:%20redox%20regulation%20and%20drug%20resistance&author=MH%20Hanigan&volume=122&publication_year=2014&pages=103-141&pmid=24974180&doi=10.1016/B978-0-12-420117-0.00003-7&)

23. Franzini M, Corti A, Lorenzini E, Paolicchi A, Pompella A, De Cesare M, Perego P, Gatti L, Leone R, Apostoli P, et al. Modulation of cell growth and cisplatin sensitivity by membrane gamma-glutamyltransferase in melanoma cells. Eur J Cancer. 2006;42(15):2623–2630. doi: 10.1016/j.ejca.2006.04.016.  [DOI](https://doi.org/10.1016/j.ejca.2006.04.016) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16928443/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Cancer&title=Modulation%20of%20cell%20growth%20and%20cisplatin%20sensitivity%20by%20membrane%20gamma-glutamyltransferase%20in%20melanoma%20cells&author=M%20Franzini&author=A%20Corti&author=E%20Lorenzini&author=A%20Paolicchi&author=A%20Pompella&volume=42&issue=15&publication_year=2006&pages=2623-2630&pmid=16928443&doi=10.1016/j.ejca.2006.04.016&)

24. Sciskalska M, Oldakowska M, Marek G, Milnerowicz H. Increased risk of acute pancreatitis occurrence in smokers with rs5751901 polymorphisms in GGT1 gene. Int J Med Sci. 2020;17(2):242–254. doi: 10.7150/ijms.38657.  [DOI](https://doi.org/10.7150/ijms.38657) | [PMC free article](/articles/PMC6990886/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32038108/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Med%20Sci&title=Increased%20risk%20of%20acute%20pancreatitis%20occurrence%20in%20smokers%20with%20rs5751901%20polymorphisms%20in%20GGT1%20gene&author=M%20Sciskalska&author=M%20Oldakowska&author=G%20Marek&author=H%20Milnerowicz&volume=17&issue=2&publication_year=2020&pages=242-254&pmid=32038108&doi=10.7150/ijms.38657&)

25. Grzybowska EA. Human intronless genes: functional groups, associated diseases, evolution, and mRNA processing in absence of splicing. Biochem Biophys Res Commun. 2012;424(1):1–6. doi: 10.1016/j.bbrc.2012.06.092.  [DOI](https://doi.org/10.1016/j.bbrc.2012.06.092) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22732409/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20Biophys%20Res%20Commun&title=Human%20intronless%20genes:%20functional%20groups,%20associated%20diseases,%20evolution,%20and%20mRNA%20processing%20in%20absence%20of%20splicing&author=EA%20Grzybowska&volume=424&issue=1&publication_year=2012&pages=1-6&pmid=22732409&doi=10.1016/j.bbrc.2012.06.092&)

26. Wang D, Poi MJ, Sun X, Gaedigk A, Leeder JS, Sadee W. Common CYP2D6 polymorphisms affecting alternative splicing and transcription: long-range haplotypes with two regulatory variants modulate CYP2D6 activity. Hum Mol Genet. 2014;23(1):268–278. doi: 10.1093/hmg/ddt417.  [DOI](https://doi.org/10.1093/hmg/ddt417) | [PMC free article](/articles/PMC3857955/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23985325/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=Common%20CYP2D6%20polymorphisms%20affecting%20alternative%20splicing%20and%20transcription:%20long-range%20haplotypes%20with%20two%20regulatory%20variants%20modulate%20CYP2D6%20activity&author=D%20Wang&author=MJ%20Poi&author=X%20Sun&author=A%20Gaedigk&author=JS%20Leeder&volume=23&issue=1&publication_year=2014&pages=268-278&pmid=23985325&doi=10.1093/hmg/ddt417&)

27. Ramirez-Bello J, Jimenez-Morales M. Functional implications of single nucleotide polymorphisms (SNPs) in protein-coding and non-coding RNA genes in multifactorial diseases. Gac Med Mex. 2017;153(2):238–250.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/28474710/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gac%20Med%20Mex&title=Functional%20implications%20of%20single%20nucleotide%20polymorphisms%20(SNPs)%20in%20protein-coding%20and%20non-coding%20RNA%20genes%20in%20multifactorial%20diseases&author=J%20Ramirez-Bello&author=M%20Jimenez-Morales&volume=153&issue=2&publication_year=2017&pages=238-250&pmid=28474710&)

28. Chiang HL, Wu JY, Chen YT. Identification of functional single nucleotide polymorphisms in the branchpoint site. Hum Genomics. 2017;11(1):27. doi: 10.1186/s40246-017-0122-6.  [DOI](https://doi.org/10.1186/s40246-017-0122-6) | [PMC free article](/articles/PMC5680774/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29121990/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Genomics&title=Identification%20of%20functional%20single%20nucleotide%20polymorphisms%20in%20the%20branchpoint%20site&author=HL%20Chiang&author=JY%20Wu&author=YT%20Chen&volume=11&issue=1&publication_year=2017&pages=27&pmid=29121990&doi=10.1186/s40246-017-0122-6&)

29. Zhou YL, Wu WP, Cheng J, Liang LL, Cen JM, Chen C, Liu X, Xiong XD. CircFOXO3 rs12196996, a polymorphism at the gene flanking intron, is associated with circFOXO3 levels and the risk of coronary artery disease. Aging (Albany NY) 2020;12(13):13076–13089. doi: 10.18632/aging.103398.  [DOI](https://doi.org/10.18632/aging.103398) | [PMC free article](/articles/PMC7377899/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32614786/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Aging%20(Albany%20NY)&title=CircFOXO3%20rs12196996,%20a%20polymorphism%20at%20the%20gene%20flanking%20intron,%20is%20associated%20with%20circFOXO3%20levels%20and%20the%20risk%20of%20coronary%20artery%20disease&author=YL%20Zhou&author=WP%20Wu&author=J%20Cheng&author=LL%20Liang&author=JM%20Cen&volume=12&issue=13&publication_year=2020&pages=13076-13089&pmid=32614786&doi=10.18632/aging.103398&)

30. Ikeda Y, Fujii J, Taniguchi N. Significance of Arg-107 and Glu-108 in the catalytic mechanism of human gamma-glutamyl transpeptidase. identification by site-directed mutagenesis. J Biol Chem. 1993;268(6):3980–3985. doi: 10.1016/S0021-9258(18)53567-5.  [DOI](https://doi.org/10.1016/S0021-9258(18)53567-5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8095045/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Significance%20of%20Arg-107%20and%20Glu-108%20in%20the%20catalytic%20mechanism%20of%20human%20gamma-glutamyl%20transpeptidase.%20identification%20by%20site-directed%20mutagenesis&author=Y%20Ikeda&author=J%20Fujii&author=N%20Taniguchi&volume=268&issue=6&publication_year=1993&pages=3980-3985&pmid=8095045&doi=10.1016/S0021-9258(18)53567-5&)
