# Exploring the contribution of genetic variants to high sunitinib exposure in patients with cancer

## Metadata
**Authors:** Eline L Giraud, Stefanie D Krens, Stefan Böhringer, Ingrid M E Desar, Sita H Vermeulen, Lambertus A Kiemeney, Alwin D R Huitema, Neeltje Steeghs, Nielka P van Erp, Jesse J Swen
**Journal:** British Journal of Clinical Pharmacology
**Date:** 2024 Aug 6
**DOI:** [10.1111/bcp.16196](https://doi.org/10.1111/bcp.16196)
**PMID:** 39107874
**PMCID:** PMC11773116
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11773116/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC11773116/pdf/BCP-91-297.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC11773116/pdf/BCP-91-297.pdf)

## Abstract

**Aims:** 
Sunitinib exhibits considerable interindividual variability in exposure. While the target total plasma concentration of sunitinib and its active metabolite is 50–87.5 ng/mL for the intermittent dosing schedule, ~10–21% of patients experience higher exposures (>87.5 ng/mL), correlated with an increased risk for toxicity. Previous research identified single nucleotide variants (SNVs) in genes from the sunitinib pharmacokinetic pathway to be associated with efficacy and toxicity. However, significant interindividual variability in exposure remains unexplained. Our aim was to identify genetic variants associated with supratherapeutic exposure of sunitinib.

**Methods:** 
This was a genome‐wide association study. Cases were identified during routine therapeutic drug monitoring and consisted of patients with dose‐normalized sunitinib plasma concentrations >87.5 ng/mL (intermittent dosing) or >75 ng/mL (continuous dosing). Controls were sampled from the historical cohort EuroTARGET who tolerated the standard dose of 50 mg in an intermittent schedule. SNVs were tested for an association with sunitinib exposure. A P‐value ≤5 × 10−8 was considered significant and a P‐value between 5 × 10−8 and 5 × 10−6 was considered suggestive.

**Results:** 
Sixty‐nine cases and 345 controls were included for association analysis. One SNV (rs6923761), located on the gene glucagon‐like peptide 1 receptor, was significantly associated with increased sunitinib exposure (P = 7.86 × 10−19). Twelve SNVs were suggestive for an association with sunitinib exposure (P ≤ 5 × 10−6).

**Conclusions:** 
While rs6923761 is associated with high sunitinib exposure, the underlying mechanism is not yet clarified and warrants further investigation. [Corrections made on 23 September 2024, after first online publication: In the preceding sentence, identifier rs6923671 has been changed to rs6923761 in this version.] We could not confirm the earlier found associations between SNVs in candidate genes involved in the pharmacokinetic pathway of sunitinib and its efficacy and toxicity.

Keywords: cancer, genetic polymorphism, genetics and pharmacogenetics, pharmacokinetics, toxicity

### Aims

Sunitinib exhibits considerable interindividual variability in exposure. While the target total plasma concentration of sunitinib and its active metabolite is 50–87.5 ng/mL for the intermittent dosing schedule, ~10–21% of patients experience higher exposures (>87.5 ng/mL), correlated with an increased risk for toxicity. Previous research identified single nucleotide variants (SNVs) in genes from the sunitinib pharmacokinetic pathway to be associated with efficacy and toxicity. However, significant interindividual variability in exposure remains unexplained. Our aim was to identify genetic variants associated with supratherapeutic exposure of sunitinib.

### Methods

This was a genome‐wide association study. Cases were identified during routine therapeutic drug monitoring and consisted of patients with dose‐normalized sunitinib plasma concentrations >87.5 ng/mL (intermittent dosing) or >75 ng/mL (continuous dosing). Controls were sampled from the historical cohort EuroTARGET who tolerated the standard dose of 50 mg in an intermittent schedule. SNVs were tested for an association with sunitinib exposure. A *P*‐value ≤5 × 10^−8^ was considered significant and a *P*‐value between 5 × 10^−8^ and 5 × 10^−6^ was considered suggestive.

### Results

Sixty‐nine cases and 345 controls were included for association analysis. One SNV (rs6923761), located on the gene *glucagon‐like peptide 1 receptor*, was significantly associated with increased sunitinib exposure (*P =* 7.86 × 10^−19^). Twelve SNVs were suggestive for an association with sunitinib exposure *(P* ≤ 5 × 10^−6^).

### Conclusions

While rs6923761 is associated with high sunitinib exposure, the underlying mechanism is not yet clarified and warrants further investigation. [Corrections made on 23 September 2024, after first online publication: In the preceding sentence, identifier rs6923671 has been changed to rs6923761 in this version.] We could not confirm the earlier found associations between SNVs in candidate genes involved in the pharmacokinetic pathway of sunitinib and its efficacy and toxicity.

## What is already known about this subject

## What is already known about this subject

## What this study adds

## 1. INTRODUCTION

Sunitinib is an oral multitarget tyrosine kinase inhibitor approved for the treatment of metastatic renal cell carcinoma (mRCC), pancreatic neuroendocrine tumours and gastrointestinal stromal tumours (GISTs). It is prescribed at a fixed dose of 50 mg once daily (OD) in a 4‐weeks‐on/2‐weeks‐off (4/2) schedule or 37.5 mg OD taken continuously. Sunitinib is mainly metabolized through cytochrome P450 (CYP)3A4 into its equipotent metabolite N‐desethyl sunitinib (SU12662), and subsequently into other metabolites.[1](#bcp16196-bib-0001) SU12662 accounts for 23–37% of the total exposure, with the sum of sunitinib and SU12662 representing the total active drug in plasma.[1](#bcp16196-bib-0001) A clear exposure–response relationship has been established for sunitinib with a well‐defined target therapeutic range (plasma trough level, C_trough_) within which improved efficacy but acceptable toxicity is observed.[2](#bcp16196-bib-0002) These C_trough_ ranges have previously been established in pharmacokinetic (PK) studies and range from 37.5–75 ng/mL and 50–87.5 ng/mL, depending on the dose and dosing schedule of sunitinib.[3](#bcp16196-bib-0003) Due to the relatively high interpatient variability of sunitinib sum plasma concentrations (34–59%), only 38% of patients show C_trough_ levels within the predefined target range.[3](#bcp16196-bib-0003), [4](#bcp16196-bib-0004) Indeed, around half of patients show drug levels below and ~10–21% above the target range.[4](#bcp16196-bib-0004), [5](#bcp16196-bib-0005) Supratherapeutic sunitinib sum plasma concentrations have been correlated with dose‐limiting toxicities including fatigue, mucositis and hand‐foot syndrome.[3](#bcp16196-bib-0003), [6](#bcp16196-bib-0006) Hence, approximately 33–50% of the sunitinib treated mRCC patients require a dose reduction in the first 3 cycles due to toxicity.[5](#bcp16196-bib-0005), [7](#bcp16196-bib-0007), [8](#bcp16196-bib-0008)

To date, it remains unclear why ~10–21% of sunitinib treated patients develop supratherapeutic concentrations under standard dosing.[5](#bcp16196-bib-0005) Previous research revealed that parameters like tumour type, Asian race, sex and body weight described part of the variability in clearance for sunitinib and its metabolite, although very limited.[9](#bcp16196-bib-0009) Also, multiple exploratory studies have been conducted based on hypothesis driven approaches, reporting associations between pharmacodynamic parameters, efficacy, and toxicity of sunitinib and single nucleotide variants (SNVs) in candidate genes involved in the sunitinib PK pathway.[8](#bcp16196-bib-0008), [10](#bcp16196-bib-0010), [11](#bcp16196-bib-0011), [12](#bcp16196-bib-0012), [13](#bcp16196-bib-0013), [14](#bcp16196-bib-0014) In detail, genetic variants in PK‐related genes, including [CYP3A5](https://www.guidetopharmacology.org/GRAC/FamilyDisplayForward?familyId=263%26objId=1338#1338) and [CYP1A1](https://www.guidetopharmacology.org/GRAC/ObjectDisplayForward?objectId=1318), have been reported to be associated with sunitinib‐induced toxicity and efficacy.[8](#bcp16196-bib-0008), [10](#bcp16196-bib-0010), [11](#bcp16196-bib-0011), [12](#bcp16196-bib-0012), [13](#bcp16196-bib-0013) Additionally, correlations between multiple polymorphisms in the efflux transporters [ABCG2](https://www.guidetopharmacology.org/GRAC/ObjectDisplayForward?objectId=792) and [ABCB1](https://www.guidetopharmacology.org/GRAC/ObjectDisplayForward?objectId=768) and toxicity and efficacy outcomes have been reported.[8](#bcp16196-bib-0008), [11](#bcp16196-bib-0011), [13](#bcp16196-bib-0013), [14](#bcp16196-bib-0014) Only 1 study investigating the association between genetic variants in ABCB1 and ABCG2 and toxicity also involved PK data to support their hypothesis.[15](#bcp16196-bib-0015) However, all of these studies were limited in sample size, assessed a small set of candidate genes, and the identified SNVs together explained only a small part of the observed interpatient variability of sunitinib plasma concentrations.

By contrast, a genome‐wide association study (GWAS) offers a hypothesis‐free approach to test common genetic variants across the genome for associations with an outcome of interest.[16](#bcp16196-bib-0016) Thus far, there have been few studies that used a GWAS approach to identify genes linked to altered PK.[17](#bcp16196-bib-0017), [18](#bcp16196-bib-0018) However, although not frequent and actual implementation can take very long, some findings from GWAS studies did lead to actual changes in the standard practice, such as *NUDT15* for azathioprine and *SLCO1B1* for statins.[19](#bcp16196-bib-0019), [20](#bcp16196-bib-0020) Here, we report the results of a GWAS to identify genetic variants associated with high sunitinib exposure.

## 2. METHODS

### 2.1. Study population and design

The study population consisted of patients with mRCC and GIST treated with sunitinib (Figure [1](#bcp16196-fig-0001)). Cases were defined as patients who received sunitinib at the standard starting dose, defined as 50 mg OD in an intermittent schedule (either 4/2 or 2‐weeks‐on/1‐week‐off) or 37.5 mg OD continuously, with sum plasma concentrations exceeding respectively 87.5 or 75 ng/mL. Cases were identified during routine therapeutic drug monitoring in standard patient care in the Radboud University Medical Centre (Radboudumc, Nijmegen, the Netherlands) and The Netherlands Cancer Institute (Amsterdam, the Netherlands) between December 2013 and December 2019.

### FIGURE 1.

![FIGURE 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e4f4/11773116/79c7f7f4c88a/BCP-91-297-g002.jpg)

Patient selection. OD, once daily.

Controls were sampled from the EuroTARGET study. This is an international multicentre observational study composed of 1210 patients who were diagnosed with mRCC.[21](#bcp16196-bib-0021) We only selected patients receiving 50 mg sunitinib OD in a 4/2 schedule. No PK data were available for the EuroTARGET cohort. To exclude patients with high sunitinib exposure from the EuroTARGET study data, all patients who received a dose reduction at any timepoint during their treatment were excluded from the analysis. This approach assumes that patients with supratherapeutic sunitinib and SU12662 C_trough_ levels require a dose reduction and is supported by available literature.[5](#bcp16196-bib-0005) In addition, this cut‐off is supported by the fact that all case patients in our study (sunitinib concentrations >87.5 ng/mL) experienced toxicities requiring dose reductions.

The protocol was approved by the Institutional Review Board of the Radboudumc, Nijmegen, The Netherlands (2019–5393), as well as the Institutional Review Board of The Netherlands Cancer Institute, Amsterdam, The Netherlands (IRBml19–237). For patients from the Radboudumc, data from routine clinical care were used. Informed consent was obtained from those patients still alive at the time of patient selection. For patients of The Netherlands Cancer Institute, informed consent for use of biological samples and clinical data was acquired at the start of a patients' treatment in the hospital, in accordance with the hospital wide opt‐out procedure. The EuroTARGET study was approved by the ethics committees of participating centres and all patients provided written informed consent.[21](#bcp16196-bib-0021)

### 2.2. Data collection and genotyping

For cases, baseline characteristics (age, sex, tumour type, sunitinib starting dose and dosing schedule, C_trough_) and treatment information were retrospectively retrieved from electronic health records. Germline DNA was isolated from stored EDTA plasma collected during routine patient care. Genotyping was conducted with the Illumina Infinium Global Screening Array BeadChip (GSA Illumina, San Diego, CA, USA), containing 725 829 markers.

For controls, clinical and genotyping data were available from EuroTARGET. Germline DNA was isolated from EDTA‐whole blood, peripheral blood mononuclear cell samples, serum or plasma and was collected before or during sunitinib treatment.[21](#bcp16196-bib-0021) Genotyping was performed using the Illumina Human OmniExpress BeadChips 12v1–1, 24v1–0, and 24v1–1 containing 680 729 markers (Illumina, San Diego, CA, USA).[21](#bcp16196-bib-0021) Genetic data from controls and cases were merged for the final analysis (Figure [2](#bcp16196-fig-0002)). Hereby, 139 918 markers were present on both chips. Data were coded and stored in Castor EDC.[22](#bcp16196-bib-0022)

### FIGURE 2.

![FIGURE 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e4f4/11773116/cb2f93574b46/BCP-91-297-g003.jpg)

Flowchart of combined arrays—cases and controls. SNV, single nucleotide variant; MAF, minor allele frequency; HWE, Hardy–Weinberg equilibrium.

### 2.3. Quality control

Quality control (QC) checks were performed using R software version 4.2.3 and PLINK software, version 1.07.[23](#bcp16196-bib-0023) Cases and controls were genotyped using different arrays with varying SNV coverage. Therefore, to reduce the risk of excluding all cases, patients were excluded based on an individual genotype call rate threshold of 47.5%, gender mismatch between reported and estimated sex based on genotypes of the X‐chromosome (using PLINK), or excess of heterozygous genotypes as measured by the inbreeding coefficient. The inbreeding F‐statistic was used to detect an excess of heterozygosity based on outlier detection. For QC steps, genetic markers were filtered at a minor allele frequency (MAF) of 0.5%. Individual genetic markers were excluded based on a call rate <80%, and a *P*‐value ≤10^−7^ for the Hardy–Weinberg equilibrium goodness‐of‐fit test. As a result of the relatively low number of cases compared to controls and the call rate of <80% for the individual genetic markers, all markers uniquely present on the Illumina Infinium Global Screening Array BeadChip were excluded from the analyses. Only markers uniquely on the Illumina Human OmniExpress BeadChip and overlapping SNVs remained present in analyses.

After the exclusion of patients and markers in these QC steps, the remaining set was used for integrative QC assessment. To evaluate the possibility of population stratification or outliers, multidimensional scaling (MDS) analysis was performed using PLINK. SNVs were pruned using PLINK prior to performing the MDS. Additionally, pairwise identity by state (IBS) statistics were calculated to identify potential duplicates. MDS and IBS were computed using PLINK. Patients who were identified as outliers based on IBS clustering were excluded from the analysis. MDS coordinates were extracted and used as covariates in the association analysis. Seven dimensions were used for the MDS map.

### 2.4. Imputation

SNV imputation was performed using the programs SHAPEIT and IMPUTE2 with default parameters using the 1000 genomes reference panel (build version 3) with the total, cosmopolitan, set of individuals.[24](#bcp16196-bib-0024) An MDS plot was used to assess ethnicity and batch effects. By removing the SNVs only present on the array used for the case group, the potential bias introduced by genotyping imputation was minimized. After imputation, the number of markers that reached statistical significance increased excessively to 9522 (inflation factor [IF] = 1.68). Therefore, only non‐imputed results will be discussed in further analysis. The results after imputation can be found in Table [S1](#bcp16196-supitem-0001).

### 2.5. Statistical analysis

Associations of SNVs with sunitinib exposure were evaluated using logistic regression, treating the outcome as binary variable (cases with high exposure *vs*. controls). Association analyses were corrected for sex and age. SNVs were filtered based on a MAF of 2.0% before performing regression analyses. Statistical analyses were performed in R statistics version 4.2.3. Associations with a *P*‐value ≤5 × 10^−8^ were considered statistically genome‐wide significant. Associations with a *P*‐value ≤5 × 10^−6^ were considered suggestive for association.[25](#bcp16196-bib-0025) Post‐association QC was performed by visual inspection of quantile‐quantile plots of *P*‐values of association tests and computation of the IF.[26](#bcp16196-bib-0026) To improve readability, the list of top SNVs according to *P*‐values only contains the best association in a window of 100 kb. A full list is provided as a supplement (Table [S2](#bcp16196-supitem-0002)).

### 2.6. Nomenclature of targets and ligands

Key protein targets and ligands in this article are hyperlinked to corresponding entries in [http://www.guidetopharmacology.org](http://www.guidetopharmacology.org), and are permanently archived in the Concise Guide to PHARMACOLOGY 2023/24.[27](#bcp16196-bib-0027), [28](#bcp16196-bib-0028), [29](#bcp16196-bib-0029)

## 3. RESULTS

### 3.1. Quality control

For 695 individuals, both genotype and clinical data were available and these data entered the QC steps (Figure [1](#bcp16196-fig-0001)). Two individuals (0.3%) failed the quality control check based on the individual genotype call rate and 3 individuals failed the sex check (0.4%). No individuals were excluded based on the inbreeding analysis. Mean inbreeding coefficient was 0.01 (95% confidence interval −0.03, 0.04). Altogether, 4 individuals were excluded (0.6% from baseline), resulting in 691 remaining patients. Next, of 1 228 263 available markers, 595 231 markers (48.2% from baseline) were excluded as they failed the QC criteria (Figure [2](#bcp16196-fig-0002)) based on the allele frequency (*n =* 214 641; 17.5%), MAF (*n =* 567 021; 46.2%) or Hardy–Weinberg equilibrium (*n =* 4434; 0.4%). No outliers were detected by the IBS analysis. In addition, 277 patients from the EuroTARGET cohort were excluded as they had received dose reductions during their treatment with sunitinib (*n =* 276) or due to missing data (*n =* 1). After this, results were again filtered based on a MAF of <2.0%, resulting in the exclusion of another 18 336 markers. A total of 614 696 markers in 414 patients, of which 69 cases and 345 controls, remained in the final analysis.

### 3.2. Baseline characteristics

Baseline patient characteristics for both the case and control arm are described in Table [1](#bcp16196-tbl-0001). In the case arm, most patients had mRCC (88.4%) and were treated with sunitinib in an intermittent schedule (*n =* 47, 68.1%). All patients in the control arm were treated in an intermittent 4/2 schedule for mRCC. The median dose‐normalized sunitinib exposure in the case group was 97 ng/mL (interquartile range [IQR] 91–115).

### TABLE 1.

|   | All patients | Cases | Controls |
| --- | --- | --- | --- |
| (n = 414) | (n = 69) | (n = 345) |  |
| Median age*, years | 63 | 63 | 62 |
| Interquartile range | 54–69 | 54–69 | 54–69 |
| Sex, n (%) |  |  |  |
| Male | 318 (76.8) | 45 (65.2) | 273 (79.1) |
| Female | 96 (23.2) | 24 (34.8) | 72 (20.9) |
| Tumour type, n (%) |  |  |  |
| Metastatic renal cell carcinoma | 406 (98.1) | 61 (88.4) | 345 (100.0) |
| Gastrointestinal stromal tumour | 5 (1.2) | 5 (7.2) | 0 (0.0) |
| Other | 3 (0.7) | 3 (4.3) | 0 (0.0) |
| Sunitinib dose*, n (%) |  |  |  |
| 25.0 mg | 5 (1.2) | 5 (7.2) | 0 (0.0) |
| 37.5 mg | 27 (6.5) | 27 (39.1) | 0 (0.0) |
| 50.0 mg | 380 (91.8) | 35 (50.7) | 345 (100.0) |
| Other | 2 (0.5) | 2 (2.9) | 0 (0.0) |
| Dosing schedule*, n (%) |  |  |  |
| Continuous | 22 (5.3) | 22 (31.9) | 0 (0.0) |
| 4 weeks on/2 weeks off | 376 (90.8) | 31 (44.9) | 345 (100.0) |
| 2 weeks on/1 week off | 16 (3.9) | 16 (23.2) | 0 (0.0) |
| Median C trough , ng/mL Interquartile range | N/A | 97 91–115 | N/A |

Table 1 Caption: Baseline characteristics. Ctrough, trough plasma concentration. *Age at the time of the plasma sample (cases) or at the start of sunitinib treatment (controls). Please note that percentages may not add up to 100 due to rounding.

### 3.3. Genome‐wide association analysis

Based on the MDS plots, at least 2 dimensions contributed to clustering in the data and should be corrected for in regression analysis. Due to the complete separation of cases and controls, this was not possible. For further analysis, we therefore did not include MDS dimensions into the model. The IF was 1.10 indicating moderate inflation. The Manhattan plot for the analysis is shown in Figure [3](#bcp16196-fig-0003). Only 1 marker reached genome‐wide statistical significance, rs6923761 (chromosome 6; *P* = 7.86 × 10^−19^), located in the glucagon‐like peptide 1 receptor ([GLP1R](https://www.guidetopharmacology.org/GRAC/ObjectDisplayForward?objectId=249)) gene. This SNV had a MAF of 0.3 and an effect size β_M_ of −3.18 (odds ratio 0.04), suggesting that carriers of this variant allele have a decreased sunitinib concentration. Furthermore, 12 SNVs showed a *P*‐value ≤5 × 10^−6^ suggestive for an association with sunitinib exposure. These SNVs were scrutinized for a plausible biological association with sunitinib exposure. Out of the 12, 3 SNVs were located in a gene (*GPRC5C*, *FAR2* and *NUF2)*, whereas 9 SNVs were not located in protein‐coding regions. Details on the abovementioned 13 SNVs are listed in Table [2](#bcp16196-tbl-0002).

### FIGURE 3.

![FIGURE 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e4f4/11773116/d75236a0fe5b/BCP-91-297-g001.jpg)

Manhattan plot displaying the P‐values for all markers associated with sunitinib exposure. rs6923761 (7.86 × 10−19) is encircled in green.

### TABLE 2.

|   | P (SNV) | Marker | Chr | Pos | A | Af | P (HWE) | βM | Gene |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| 1 | 7.86 × 10−19 | rs6923761 | 6 | 39 034 072 | A | 0.30 | 3.30 × 10−2 | −3.18 | GLP1R |
| 2 | 6.18 × 10−8 | rs16978038 | 17 | 72 422 081 | C | 0.11 | 2.23 × 10−1 | −3.17 | GPRC5C |
| 3 | 1.44 × 10−7 | rs6630111 | 23 | 25 718 259 | T | 0.28 | 8.97 × 10−1 | 1.10 | 178 kb 5′ of RP11‐86A5.1 |
| 4 | 5.26 × 10−7 | rs11868247 | 17 | 74 820 857 | T | 0.18 | 1.30 × 10−1 | −2.16 | 44 kb 5′ of MGAT5B |
| 5 | 7.50 × 10−7 | rs7236584 | 18 | 63 866 787 | T | 0.02 | 1.05 × 10−1 | 2.76 | 279 kb 5′ of RP11‐389 J22.3 |
| 6 | 1.31 × 10−6 | rs2516090 | 22 | 27 406 436 | A | 0.14 | 1.47 × 10−1 | −2.13 | 25 kb 5′ of U6 |
| 7 | 1.31 × 10−6 | rs7680581 | 4 | 33 206 192 | A | 0.02 | 5.81 × 10−1 | 2.48 | 229 kb 5′ of RP11‐802H3.2 |
| 8 | 1.56 × 10−6 | rs10843346 | 12 | 29 373 899 | A | 0.46 | 2.01 × 10−1 | −0.94 | FAR2 |
| 9 | 1.68 × 10−6 | rs213212 | 6 | 33 185 918 | C | 0.27 | 9.08 × 10−1 | −1.38 | 5.4 kb 3′ of RING1 |
| 10 | 1.97 × 10−6 | rs11802875 | 1 | 163 313 539 | T | 0.21 | 8.69 × 10−2 | 1.40 | NUF2 |
| 11 | 2.53 × 10−6 | rs7001569 | 8 | 83 176 155 | A | 0.10 | 1.70 × 10−1 | 1.43 | 103 kb 5′ of RP11‐99H20.1 |
| 12 | 3.72 × 10−6 | rs9318254 | 13 | 74 871 048 | A | 0.14 | 5.23 × 10−1 | −1.74 | 7.7 kb 5′ of RNY1P5 |
| 13 | 4.74 × 10−6 | rs7881910 | 23 | 4 628 697 | C | 0.48 | 8.15 × 10−1 | −0.82 | 8.1 kb 3′ of AC074035.1 |

Table 2 Caption: SNVs with P‐value ≤5 × 10−6. SNV, single nucleotide variants; Chr, chromosome; Pos, position; A, effect allele; Af, allele frequency of effect allele; HWE, Hardy–Weinberg equilibrium; βM, effect size of the marker; kb, kilobase.

## 4. DISCUSSION

In this study, we identified 1 marker in *GLP1R* on chromosome 6, rs6923761, that reached genome wide statistical significance *(P =* 7.86 × 10^−19^) for association with supratherapeutic sunitinib exposure. In addition, we identified 12 SNVs suggestive for an association with supratherapeutic sunitinib exposure.

rs6923761 is a missense variant located on the *GLP1R* gene and results in the substitution of serine for glycine at position 168 (Gly168Ser). *GLP1R* codes for a 7‐transmembrane protein that functions as a receptor for the glucagon‐like peptide 1 (GLP‐1) hormone.[30](#bcp16196-bib-0030) Protein expression of this receptor was mainly found in pancreatic β‐cells, where it becomes internalized in response to GLP‐1 or GLP‐1 analogues.[31](#bcp16196-bib-0031) It thereby plays a key role in the signalling cascades leading to glucose‐induced insulin secretion, particularly in the context of diabetes. In literature, rs6923761 has been associated with a decreased β‐cell responsiveness to GLP‐1 and thus subsequent decreased insulin secretion.[32](#bcp16196-bib-0032) Although the label of sunitinib mentions a potential decrease in blood glucose, the underlying mechanism is not yet understood.[33](#bcp16196-bib-0033) Theoretically, GLP‐1 could affect drug absorption due to delayed gastric emptying. However, an earlier interaction study did not show any clinically relevant effect of liraglutide, a GLP‐1‐agonist, on the PK of concomitant drugs.[34](#bcp16196-bib-0034) Based on these results, the involvement of *GLP1R* in changes in the PK of sunitinib remains unclear, yet a possible association between GLP‐1(R) and high sunitinib exposure cannot be ruled out. Out of the 12 SNVs suggestive for an association with supratherapeutic sunitinib exposure, 9 were not located on protein coding regions. The remaining 3 were located on the genes *GPRC5C*, *FAR2* and *NUF2*. Overall, none of the identified significant or suggestive hits appears to be a plausible biological candidate for supratherapeutic sunitinib exposure.

In the current study, a hypothesis‐free GWAS approach was used to test for an association between a large set of genetic variants and high sunitinib exposure. The relatively small sample size of the case group dictates that our study has only enough statistical power to identify variants with a substantial effect size. Previous candidate gene studies reported associations between SNVs in *CYP3A5*, *ABCB1* and *ABCG2* and sunitinib toxicity and efficacy. Our study could not confirm these associations. Of note, rs2231137, 1 of the top SNVs reported in previous studies located on the *ABCG2* gene, showed a *P*‐value of 6.52 × 10^−6^. While above our predefined thresholds, this does not completely exclude involvement of *ABCG2* in sunitinib PK, though the effect size will be small and not likely to be clinically relevant.[8](#bcp16196-bib-0008), [13](#bcp16196-bib-0013)

To the best of our knowledge, this is the first GWAS investigating the association between genetic variants and supratherapeutic sunitinib exposure. A strength of our analysis is the large set of cases with both genetic and sunitinib exposure data available, which might be considered unique given that only ~10–21% of patients have plasma concentrations exceeding 87.5 ng/mL. Despite the uniqueness of our dataset, several limitations have to be taken into account. First, sunitinib exposure levels were only measured in the cases. While we anticipated that excluding all patients requiring dose reductions would eliminate any individuals with supratherapeutic sunitinib exposure from the control group, we cannot assert this with absolute certainty. This could have potentially diluted the contrast between cases and controls limiting the ability to detect true signals. However, given that all cases in our study (with sunitinib concentrations >87.5 ng/mL) suffered from toxicities that required dose reductions, we consider this risk negligible.[5](#bcp16196-bib-0005) In the control group, all patients experiencing dose‐limiting toxicities were excluded, regardless of whether this was related to high sunitinib exposure or not. It should be noted that not all sunitinib toxicity is related to supratherapeutic exposure. Indeed, while 33–50% of patients require dose reductions due to toxicity only 10–21% exhibit supratherapeutic sunitinib exposure. Consequently, patients requiring dose reductions with sunitinib exposure <87.5 ng/mL may have been falsely excluded. In addition, roughly 70% of cases used sunitinib in an intermittent schedule compared to 100% of controls, which could potentially create differences in the risk of developing toxicity. Nonetheless, prior studies in patients with mRCC and GIST demonstrated similar toxicity profiles between both dosing regimens.[35](#bcp16196-bib-0035), [36](#bcp16196-bib-0036) We therefore believe this difference is negligible. Furthermore, the availability of sunitinib concentrations for both cases and controls would have allowed us to build a linear regression model with sunitinib exposure as a continuous outcome which would have substantially increased statistical power to detect variants associated with sunitinib PK.

Considering the number of patients typically needed to perform a GWAS, our sample size was relatively small. This might have led to overestimation of the effect sizes of significant SNVs (winner's curse), and the approximation of small *P*‐values might become inaccurate. We cannot exclude such effects in our study. However, there are several SNVs with higher MAFs in our top list suggesting that the chances are slim that these are all false positives. Furthermore, we used data from 2 different cohorts and patients were treated in different settings, which may have led to differences in their treatment. Moreover, the case patients were a mix of patients with different indications, while controls where only patients with mRCC. Treatment‐related effects can thus not be excluded. Both cohorts have also been genotyped with different arrays. The fact that cases and controls were not randomized across the arrays might have introduced confounding. Indeed, the MDS analysis clearly showed that batch effects are present. With an IF of 1.1, our study shows moderate inflation but there does not seem to be systematic induction of association due to the batch effects. Still, genotyping imputation increased genetic differences between the batches resulting in clear inflation, and imputed results could not be used for the analysis.

## 5. CONCLUSIONS

We found rs6923761 located in *GLP1R* to be associated with high sunitinib exposure. However, the underlying biological mechanism is not (yet) understood. We could not confirm previously reported associations between SNVs in candidate genes involved in the PK pathway of sunitinib and its efficacy and toxicity. Together, these results suggest that it is not likely that genetic variation is the predominant cause of supratherapeutic sunitinib exposure. With respect to clinical implications, TDM currently represents the best approach we can offer to guide patients towards achieving therapeutic sunitinib levels.

## AUTHOR CONTRIBUTIONS

E.L.G. wrote the manuscript. S.D.K., N.P.v.E. and J.J.S. designed the research. S.D.K. and S.L.G. collected clinical data and patient samples. S.H.V. performed the germline DNA isolation of control samples. E.L.G., S.D.K., S.B. and J.J.S. analysed the data. S.D.K., S.B., S.H.V., B.L.K., I.M.E.D., A.D.R.H., N.S., N.P.v.E. and J.J.S. were responsible for critical revision of the manuscript. All authors gave final approval of the version to be submitted.

## CONFLICT OF INTEREST STATEMENT

The authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest. S.D.K. received speaker honoraria from GSK outside the submitted work. N.S. provided consultation or attended advisory boards for Boehringer Ingelheim, Ellipses Pharma, GlaxoSmithKline, Incyte, Luszana, and received research grants from Abbvie, Actuate Therapeutics, Amgen, Array, Ascendis Pharma, AstraZeneca, Bayer, Blueprint Medicines, Boehringer Ingelheim, BridgeBio, Bristol‐Myers Squibb, Cantargia, CellCentric, Cogent Biosciences, Crescendo Biologics, Cytovation, Deciphera, Dragonfly, Eli Lilly, Exelixis, Genentech, GlaxoSmithKline, IDRx, Immunocore, Incyte, InteRNA, Janssen, Kinnate Biopharma, Kling Biotherapeutics, Lixte, Luszana, Merck, Merck Sharp & Dohme, Merus, Molecular Partners, Navire Pharma, Novartis, Numab Therapeutics, Pfizer, Relay Pharmaceuticals, Revolution Medicin, Roche, Sanofi, Seattle Genetics, Taiho, Takeda; all paid to the institute and all outside the submitted work. N.v.E. reports research funding paid to the institute from Astellas and Ipsen; all outside the submitted work.

## ETHICS APPROVAL

The protocol was approved by the Institutional Review Board of both the Radboud University Medical Centre (Radboudumc) and The Netherlands Cancer Institute. For patients from the Radboudumc, data from routine clinical care were used. For patients of The Netherlands Cancer Institute, informed consent for use of biological samples and clinical data was acquired at the start of a patients' treatment in the hospital, in accordance with the hospital wide opt‐out procedure. The EuroTARGET study was approved by the ethics committees of participating centres.

## INFORMED CONSENT

Informed consent was obtained from all patients.

## Supporting information

## ACKNOWLEDGEMENTS

The authors would like to thank Steffie Groenland (S.L.G.) for her help with the data collection in The Netherlands Cancer Institute.

Giraud EL, Krens SD, Böhringer S, et al. Exploring the contribution of genetic variants to high sunitinib exposure in patients with cancer. Br J Clin Pharmacol. 2025;91(2):297‐305. doi: 10.1111/bcp.16196

## DATA AVAILABILITY STATEMENT

The data generated during and/or analysed during the current study is available from the corresponding author on reasonable request.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

The data generated during and/or analysed during the current study is available from the corresponding author on reasonable request.

### Supplementary Materials

### Data Availability Statement

The data generated during and/or analysed during the current study is available from the corresponding author on reasonable request.

## References

1. European Medicines Agency . Sunitinib summary of product characteristics. Accessed 11‐4‐2022, 2022. https://www.ema.europa.eu/en/documents/product-information/sutent-epar-product-information_en.pdf  [https://www.ema.europa.eu/en/documents/product-information/sutent-epar-product-information_en.pdf](https://www.ema.europa.eu/en/documents/product-information/sutent-epar-product-information_en.pdf) | [Google Scholar](https://scholar.google.com/scholar_lookup?title=Sunitinib%20summary%20of%20product%20characteristics&)

2. Suttle AB, Ball HA, Molimard M, et al. Relationships between pazopanib exposure and clinical safety and efficacy in patients with advanced renal cell carcinoma. Br J Cancer. 2014;111(10):1909‐1916. doi: 10.1038/bjc.2014.503  [DOI](https://doi.org/10.1038/bjc.2014.503) | [PMC free article](/articles/PMC4229638/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25349968/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Cancer&title=Relationships%20between%20pazopanib%20exposure%20and%20clinical%20safety%20and%20efficacy%20in%20patients%20with%20advanced%20renal%20cell%20carcinoma&author=AB%20Suttle&author=HA%20Ball&author=M%20Molimard&volume=111&issue=10&publication_year=2014&pages=1909-1916&pmid=25349968&doi=10.1038/bjc.2014.503&)

3. Westerdijk K, Desar IME, Steeghs N, van der Graaf WTA, van Erp NP, on behalf of the Dutch Pharmacology and Oncology Group (DPOG) . Imatinib, sunitinib and pazopanib: from flat‐fixed dosing towards a pharmacokinetically guided personalized dose. Br J Clin Pharmacol. 2020;86(2):258‐273. doi: 10.1111/bcp.14185  [DOI](https://doi.org/10.1111/bcp.14185) | [PMC free article](/articles/PMC7015742/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31782166/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=Imatinib,%20sunitinib%20and%20pazopanib:%20from%20flat%E2%80%90fixed%20dosing%20towards%20a%20pharmacokinetically%20guided%20personalized%20dose&author=K%20Westerdijk&author=IME%20Desar&author=N%20Steeghs&author=WTA%20van%20der%20Graaf&author=NP%20van%20Erp&volume=86&issue=2&publication_year=2020&pages=258-273&pmid=31782166&doi=10.1111/bcp.14185&)

4. Lankheet NAG, Desar IME, Mulder SF, et al. Optimizing the dose in cancer patients treated with imatinib, sunitinib and pazopanib. Br J Clin Pharmacol. 2017;83(10):2195‐2204. doi: 10.1111/bcp.13327  [DOI](https://doi.org/10.1111/bcp.13327) | [PMC free article](/articles/PMC5595974/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28500677/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=Optimizing%20the%20dose%20in%20cancer%20patients%20treated%20with%20imatinib,%20sunitinib%20and%20pazopanib&author=NAG%20Lankheet&author=IME%20Desar&author=SF%20Mulder&volume=83&issue=10&publication_year=2017&pages=2195-2204&pmid=28500677&doi=10.1111/bcp.13327&)

5. Westerdijk K, Krens SD, van der Graaf WTA, et al. The relationship between sunitinib exposure and both efficacy and toxicity in real‐world patients with renal cell carcinoma and gastrointestinal stromal tumour. Br J Clin Pharmacol. 2021;87(2):326‐335. doi: 10.1111/bcp.14332  [DOI](https://doi.org/10.1111/bcp.14332) | [PMC free article](/articles/PMC9328649/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32358810/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=The%20relationship%20between%20sunitinib%20exposure%20and%20both%20efficacy%20and%20toxicity%20in%20real%E2%80%90world%20patients%20with%20renal%20cell%20carcinoma%20and%20gastrointestinal%20stromal%20tumour&author=K%20Westerdijk&author=SD%20Krens&author=WTA%20van%20der%20Graaf&volume=87&issue=2&publication_year=2021&pages=326-335&pmid=32358810&doi=10.1111/bcp.14332&)

6. de Wit D, Guchelaar HJ, den Hartigh J, Gelderblom H, van Erp NP. Individualized dosing of tyrosine kinase inhibitors: are we there yet? Drug Discov Today. 2015;20(1):18‐36. doi: 10.1016/j.drudis.2014.09.007  [DOI](https://doi.org/10.1016/j.drudis.2014.09.007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25245169/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Discov%20Today&title=Individualized%20dosing%20of%20tyrosine%20kinase%20inhibitors:%20are%20we%20there%20yet?&author=D%20de%20Wit&author=HJ%20Guchelaar&author=J%20den%20Hartigh&author=H%20Gelderblom&author=NP%20van%20Erp&volume=20&issue=1&publication_year=2015&pages=18-36&pmid=25245169&doi=10.1016/j.drudis.2014.09.007&)

7. van der Veldt AA, Boven E, Helgason HH, et al. Predictive factors for severe toxicity of sunitinib in unselected patients with advanced renal cell cancer. Br J Cancer. 2008;99(2):259‐265. doi: 10.1038/sj.bjc.6604456  [DOI](https://doi.org/10.1038/sj.bjc.6604456) | [PMC free article](/articles/PMC2480961/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18594533/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Cancer&title=Predictive%20factors%20for%20severe%20toxicity%20of%20sunitinib%20in%20unselected%20patients%20with%20advanced%20renal%20cell%20cancer&author=AA%20van%20der%20Veldt&author=E%20Boven&author=HH%20Helgason&volume=99&issue=2&publication_year=2008&pages=259-265&pmid=18594533&doi=10.1038/sj.bjc.6604456&)

8. van Erp NP, Eechoute K, van der Veldt AA, et al. Pharmacogenetic pathway analysis for determination of sunitinib‐induced toxicity. J Clin Oncol. 2009;27(26):4406‐4412. doi: 10.1200/JCO.2008.21.7679  [DOI](https://doi.org/10.1200/JCO.2008.21.7679) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19667267/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Pharmacogenetic%20pathway%20analysis%20for%20determination%20of%20sunitinib%E2%80%90induced%20toxicity&author=NP%20van%20Erp&author=K%20Eechoute&author=AA%20van%20der%20Veldt&volume=27&issue=26&publication_year=2009&pages=4406-4412&pmid=19667267&doi=10.1200/JCO.2008.21.7679&)

9. Houk BE, Bello CL, Kang D, Amantea M. A population pharmacokinetic meta‐analysis of sunitinib malate (SU11248) and its primary metabolite (SU12662) in healthy volunteers and oncology patients. Clin Cancer Res. 2009;15(7):2497‐2506. doi: 10.1158/1078-0432.CCR-08-1893  [DOI](https://doi.org/10.1158/1078-0432.CCR-08-1893) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19258444/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=A%20population%20pharmacokinetic%20meta%E2%80%90analysis%20of%20sunitinib%20malate%20(SU11248)%20and%20its%20primary%20metabolite%20(SU12662)%20in%20healthy%20volunteers%20and%20oncology%20patients&author=BE%20Houk&author=CL%20Bello&author=D%20Kang&author=M%20Amantea&volume=15&issue=7&publication_year=2009&pages=2497-2506&pmid=19258444&doi=10.1158/1078-0432.CCR-08-1893&)

10. Diekstra MH, Klumpen HJ, Lolkema MP, et al. Association analysis of genetic polymorphisms in genes related to sunitinib pharmacokinetics, specifically clearance of sunitinib and SU12662. Clin Pharmacol Ther. 2014;96(1):81‐89. doi: 10.1038/clpt.2014.47  [DOI](https://doi.org/10.1038/clpt.2014.47) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24566734/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Association%20analysis%20of%20genetic%20polymorphisms%20in%20genes%20related%20to%20sunitinib%20pharmacokinetics,%20specifically%20clearance%20of%20sunitinib%20and%20SU12662&author=MH%20Diekstra&author=HJ%20Klumpen&author=MP%20Lolkema&volume=96&issue=1&publication_year=2014&pages=81-89&pmid=24566734&doi=10.1038/clpt.2014.47&)

11. Diekstra MH, Swen JJ, Boven E, et al. CYP3A5 and ABCB1 polymorphisms as predictors for sunitinib outcome in metastatic renal cell carcinoma. Eur Urol. 2015;68(4):621‐629. doi: 10.1016/j.eururo.2015.04.018  [DOI](https://doi.org/10.1016/j.eururo.2015.04.018) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25930089/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20Urol&title=CYP3A5%20and%20ABCB1%20polymorphisms%20as%20predictors%20for%20sunitinib%20outcome%20in%20metastatic%20renal%20cell%20carcinoma&author=MH%20Diekstra&author=JJ%20Swen&author=E%20Boven&volume=68&issue=4&publication_year=2015&pages=621-629&pmid=25930089&doi=10.1016/j.eururo.2015.04.018&)

12. Patel ND, Chakrabory K, Messmer G, Krishnan K, Bossaer JB. Severe sunitinib‐induced myelosuppression in a patient with a CYP 3A4 polymorphism. J Oncol Pharm Pract. 2018;24(8):623‐626. doi: 10.1177/1078155217724863  [DOI](https://doi.org/10.1177/1078155217724863) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28782406/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Oncol%20Pharm%20Pract&title=Severe%20sunitinib%E2%80%90induced%20myelosuppression%20in%20a%20patient%20with%20a%20CYP%203A4%20polymorphism&author=ND%20Patel&author=K%20Chakrabory&author=G%20Messmer&author=K%20Krishnan&author=JB%20Bossaer&volume=24&issue=8&publication_year=2018&pages=623-626&pmid=28782406&doi=10.1177/1078155217724863&)

13. van der Veldt AA, Eechoute K, Gelderblom H, et al. Genetic polymorphisms associated with a prolonged progression‐free survival in patients with metastatic renal cell cancer treated with sunitinib. Clin Cancer Res. 2011;17(3):620‐629. doi: 10.1158/1078-0432.CCR-10-1828  [DOI](https://doi.org/10.1158/1078-0432.CCR-10-1828) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21097692/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=Genetic%20polymorphisms%20associated%20with%20a%20prolonged%20progression%E2%80%90free%20survival%20in%20patients%20with%20metastatic%20renal%20cell%20cancer%20treated%20with%20sunitinib&author=AA%20van%20der%20Veldt&author=K%20Eechoute&author=H%20Gelderblom&volume=17&issue=3&publication_year=2011&pages=620-629&pmid=21097692&doi=10.1158/1078-0432.CCR-10-1828&)

14. Kim HR, Park HS, Kwon WS, et al. Pharmacogenetic determinants associated with sunitinib‐induced toxicity and ethnic difference in Korean metastatic renal cell carcinoma patients. Cancer Chemother Pharmacol. 2013;72(4):825‐835. doi: 10.1007/s00280-013-2258-y  [DOI](https://doi.org/10.1007/s00280-013-2258-y) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24013576/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Chemother%20Pharmacol&title=Pharmacogenetic%20determinants%20associated%20with%20sunitinib%E2%80%90induced%20toxicity%20and%20ethnic%20difference%20in%20Korean%20metastatic%20renal%20cell%20carcinoma%20patients&author=HR%20Kim&author=HS%20Park&author=WS%20Kwon&volume=72&issue=4&publication_year=2013&pages=825-835&pmid=24013576&doi=10.1007/s00280-013-2258-y&)

15. Mizuno T, Fukudo M, Terada T, et al. Impact of genetic variation in breast cancer resistance protein (BCRP/ABCG2) on sunitinib pharmacokinetics. Drug Metab Pharmacokinet. 2012;27(6):631‐639. doi: 10.2133/dmpk.dmpk-12-rg-026  [DOI](https://doi.org/10.2133/dmpk.dmpk-12-rg-026) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22673043/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Pharmacokinet&title=Impact%20of%20genetic%20variation%20in%20breast%20cancer%20resistance%20protein%20(BCRP/ABCG2)%20on%20sunitinib%20pharmacokinetics&author=T%20Mizuno&author=M%20Fukudo&author=T%20Terada&volume=27&issue=6&publication_year=2012&pages=631-639&pmid=22673043&doi=10.2133/dmpk.dmpk-12-rg-026&)

16. Diekstra MHM, Swen JJ, van der Zanden LFM, et al. Genome‐wide meta‐analysis identifies variants in DSCAM and PDLIM3 that correlate with efficacy outcomes in metastatic renal cell carcinoma patients treated with sunitinib. Cancers (Basel). 2022;14(12):2838. doi: 10.3390/cancers14122838  [DOI](https://doi.org/10.3390/cancers14122838) | [PMC free article](/articles/PMC9220885/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35740506/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancers%20(Basel)&title=Genome%E2%80%90wide%20meta%E2%80%90analysis%20identifies%20variants%20in%20DSCAM%20and%20PDLIM3%20that%20correlate%20with%20efficacy%20outcomes%20in%20metastatic%20renal%20cell%20carcinoma%20patients%20treated%20with%20sunitinib&author=MHM%20Diekstra&author=JJ%20Swen&author=LFM%20van%20der%20Zanden&volume=14&issue=12&publication_year=2022&pages=2838&pmid=35740506&doi=10.3390/cancers14122838&)

17. Mykkanen AJH, Tarkiainen EK, Taskinen S, et al. Genome‐wide association study of atorvastatin pharmacokinetics: associations with SLCO1B1, UGT1A3, and LPP. Clin Pharmacol Ther. 2024;115(6):1428‐1440. doi: 10.1002/cpt.3236  [DOI](https://doi.org/10.1002/cpt.3236) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/38493369/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Genome%E2%80%90wide%20association%20study%20of%20atorvastatin%20pharmacokinetics:%20associations%20with%20SLCO1B1,%20UGT1A3,%20and%20LPP&author=AJH%20Mykkanen&author=EK%20Tarkiainen&author=S%20Taskinen&volume=115&issue=6&publication_year=2024&pages=1428-1440&pmid=38493369&doi=10.1002/cpt.3236&)

18. Mykkanen AJH, Taskinen S, Neuvonen M, et al. Genomewide association study of simvastatin pharmacokinetics. Clin Pharmacol Ther. 2022;112(3):676‐686. doi: 10.1002/cpt.2674  [DOI](https://doi.org/10.1002/cpt.2674) | [PMC free article](/articles/PMC9540481/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35652242/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Genomewide%20association%20study%20of%20simvastatin%20pharmacokinetics&author=AJH%20Mykkanen&author=S%20Taskinen&author=M%20Neuvonen&volume=112&issue=3&publication_year=2022&pages=676-686&pmid=35652242&doi=10.1002/cpt.2674&)

19. Yang SK, Hong M, Baek J, et al. A common missense variant in NUDT15 confers susceptibility to thiopurine‐induced leukopenia. Nat Genet. 2014;46(9):1017‐1020. doi: 10.1038/ng.3060  [DOI](https://doi.org/10.1038/ng.3060) | [PMC free article](/articles/PMC4999337/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25108385/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=A%20common%20missense%20variant%20in%20NUDT15%20confers%20susceptibility%20to%20thiopurine%E2%80%90induced%20leukopenia&author=SK%20Yang&author=M%20Hong&author=J%20Baek&volume=46&issue=9&publication_year=2014&pages=1017-1020&pmid=25108385&doi=10.1038/ng.3060&)

20. Group SC , Link E, Parish S, et al. SLCO1B1 variants and statin‐induced myopathy‐‐a genomewide study. N Engl J Med. 2008;359(8):789‐799. doi: 10.1056/NEJMoa0801936  [DOI](https://doi.org/10.1056/NEJMoa0801936) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18650507/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=SLCO1B1%20variants%20and%20statin%E2%80%90induced%20myopathy%E2%80%90%E2%80%90a%20genomewide%20study&author=E%20Link&author=S%20Parish&volume=359&issue=8&publication_year=2008&pages=789-799&pmid=18650507&doi=10.1056/NEJMoa0801936&)

21. van der Zanden LFM, Vermeulen SH, Oskarsdottir A, et al. Description of the EuroTARGET cohort: a European collaborative project on TArgeted therapy in renal cell cancer‐GEnetic‐ and tumor‐related biomarkers for response and toxicity. Urol Oncol. 2017;35(8):529 e9‐529 e16. doi: 10.1016/j.urolonc.2017.03.009  [DOI](https://doi.org/10.1016/j.urolonc.2017.03.009) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28385611/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Urol%20Oncol&title=Description%20of%20the%20EuroTARGET%20cohort:%20a%20European%20collaborative%20project%20on%20TArgeted%20therapy%20in%20renal%20cell%20cancer%E2%80%90GEnetic%E2%80%90%20and%20tumor%E2%80%90related%20biomarkers%20for%20response%20and%20toxicity&author=LFM%20van%20der%20Zanden&author=SH%20Vermeulen&author=A%20Oskarsdottir&volume=35&issue=8&publication_year=2017&pages=529%20e9-529%20e16&pmid=28385611&doi=10.1016/j.urolonc.2017.03.009&)

22. Castor EDC . Castor electronic data capture. https://castoredc.com  [https://castoredc.com](https://castoredc.com) | [Google Scholar](https://scholar.google.com/scholar_lookup?title=Castor%20electronic%20data%20capture&)

23. Purcell S, Neale B, Todd‐Brown K, et al. PLINK: a tool set for whole‐genome association and population‐based linkage analyses. Am J Hum Genet. 2007;81(3):559‐575. doi: 10.1086/519795  [DOI](https://doi.org/10.1086/519795) | [PMC free article](/articles/PMC1950838/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17701901/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=PLINK:%20a%20tool%20set%20for%20whole%E2%80%90genome%20association%20and%20population%E2%80%90based%20linkage%20analyses&author=S%20Purcell&author=B%20Neale&author=K%20Todd%E2%80%90Brown&volume=81&issue=3&publication_year=2007&pages=559-575&pmid=17701901&doi=10.1086/519795&)

24. Marchini J, Howie B, Myers S, McVean G, Donnelly P. A new multipoint method for genome‐wide association studies by imputation of genotypes. Nat Genet. 2007;39(7):906‐913. doi: 10.1038/ng2088  [DOI](https://doi.org/10.1038/ng2088) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17572673/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=A%20new%20multipoint%20method%20for%20genome%E2%80%90wide%20association%20studies%20by%20imputation%20of%20genotypes&author=J%20Marchini&author=B%20Howie&author=S%20Myers&author=G%20McVean&author=P%20Donnelly&volume=39&issue=7&publication_year=2007&pages=906-913&pmid=17572673&doi=10.1038/ng2088&)

25. Pe'er I, Yelensky R, Altshuler D, Daly MJ. Estimation of the multiple testing burden for genomewide association studies of nearly all common variants. Genet Epidemiol. 2008;32(4):381‐385. doi: 10.1002/gepi.20303  [DOI](https://doi.org/10.1002/gepi.20303) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18348202/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genet%20Epidemiol&title=Estimation%20of%20the%20multiple%20testing%20burden%20for%20genomewide%20association%20studies%20of%20nearly%20all%20common%20variants&author=I%20Pe'er&author=R%20Yelensky&author=D%20Altshuler&author=MJ%20Daly&volume=32&issue=4&publication_year=2008&pages=381-385&pmid=18348202&doi=10.1002/gepi.20303&)

26. Devlin B, Roeder K. Genomic control for association studies. Biometrics. 1999;55(4):997‐1004. doi: 10.1111/j.0006-341x.1999.00997.x  [DOI](https://doi.org/10.1111/j.0006-341x.1999.00997.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11315092/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biometrics&title=Genomic%20control%20for%20association%20studies&author=B%20Devlin&author=K%20Roeder&volume=55&issue=4&publication_year=1999&pages=997-1004&pmid=11315092&doi=10.1111/j.0006-341x.1999.00997.x&)

27. Alexander SPH, Fabbro D, Kelly E, et al. The concise guide to PHARMACOLOGY 2023/24: transporters. Brit J Pharmacol. 2023;180:S374‐S469. doi: 10.1111/bph.16182  [DOI](https://doi.org/10.1111/bph.16182) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/38123156/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Brit%20J%20Pharmacol&title=The%20concise%20guide%20to%20PHARMACOLOGY%202023/24:%20transporters&author=SPH%20Alexander&author=D%20Fabbro&author=E%20Kelly&volume=180&publication_year=2023&pages=S374-S469&pmid=38123156&doi=10.1111/bph.16182&)

28. Alexander SPH, Christopoulos A, Davenport AP, et al. The concise guide to PHARMACOLOGY 2023/24: G protein‐coupled receptors. Brit J Pharmacol. 2023;180:S23‐S144. doi: 10.1111/bph.16177  [DOI](https://doi.org/10.1111/bph.16177) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/38123151/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Brit%20J%20Pharmacol&title=The%20concise%20guide%20to%20PHARMACOLOGY%202023/24:%20G%20protein%E2%80%90coupled%20receptors&author=SPH%20Alexander&author=A%20Christopoulos&author=AP%20Davenport&volume=180&publication_year=2023&pages=S23-S144&pmid=38123151&doi=10.1111/bph.16177&)

29. Alexander SPH, Fabbro D, Kelly E, et al. The concise guide to PHARMACOLOGY 2023/24: enzymes. Brit J Pharmacol. 2023;180:S289‐S373. doi: 10.1111/bph.16181  [DOI](https://doi.org/10.1111/bph.16181) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/38123154/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Brit%20J%20Pharmacol&title=The%20concise%20guide%20to%20PHARMACOLOGY%202023/24:%20enzymes&author=SPH%20Alexander&author=D%20Fabbro&author=E%20Kelly&volume=180&publication_year=2023&pages=S289-S373&pmid=38123154&doi=10.1111/bph.16181&)

30. Muller TD, Finan B, Bloom SR, et al. Glucagon‐like peptide 1 (GLP‐1). Mol Metab. 2019;30:72‐130. doi: 10.1016/j.molmet.2019.09.010  [DOI](https://doi.org/10.1016/j.molmet.2019.09.010) | [PMC free article](/articles/PMC6812410/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31767182/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Metab&title=Glucagon%E2%80%90like%20peptide%201%20(GLP%E2%80%901)&author=TD%20Muller&author=B%20Finan&author=SR%20Bloom&volume=30&publication_year=2019&pages=72-130&pmid=31767182&doi=10.1016/j.molmet.2019.09.010&)

31. Doyle ME, Egan JM. Mechanisms of action of glucagon‐like peptide 1 in the pancreas. Pharmacol Ther. 2007;113(3):546‐593. doi: 10.1016/j.pharmthera.2006.11.007  [DOI](https://doi.org/10.1016/j.pharmthera.2006.11.007) | [PMC free article](/articles/PMC1934514/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17306374/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacol%20Ther&title=Mechanisms%20of%20action%20of%20glucagon%E2%80%90like%20peptide%201%20in%20the%20pancreas&author=ME%20Doyle&author=JM%20Egan&volume=113&issue=3&publication_year=2007&pages=546-593&pmid=17306374&doi=10.1016/j.pharmthera.2006.11.007&)

32. Sathananthan A, Man CD, Micheletto F, et al. Common genetic variation in GLP1R and insulin secretion in response to exogenous GLP‐1 in nondiabetic subjects: a pilot study. Diabetes Care. 2010;33(9):2074‐2076. doi: 10.2337/dc10-0200  [DOI](https://doi.org/10.2337/dc10-0200) | [PMC free article](/articles/PMC2928367/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20805279/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Diabetes%20Care&title=Common%20genetic%20variation%20in%20GLP1R%20and%20insulin%20secretion%20in%20response%20to%20exogenous%20GLP%E2%80%901%20in%20nondiabetic%20subjects:%20a%20pilot%20study&author=A%20Sathananthan&author=CD%20Man&author=F%20Micheletto&volume=33&issue=9&publication_year=2010&pages=2074-2076&pmid=20805279&doi=10.2337/dc10-0200&)

33. European Medicines Agency . Sutent: EPAR ‐ product information. 06–07‐2023, 2023. Accessed June 7, 2023. https://www.ema.europa.eu/en/medicines/human/EPAR/sutent  [https://www.ema.europa.eu/en/medicines/human/EPAR/sutent](https://www.ema.europa.eu/en/medicines/human/EPAR/sutent) | [Google Scholar](https://scholar.google.com/scholar_lookup?title=Sutent:%20EPAR%20%E2%80%90%20product%20information.%2006%E2%80%9307%E2%80%902023&publication_year=2023&)

34. Malm‐Erjefalt M, Ekblom M, Vouis J, Zdravkovic M, Lennernas H. Effect on the gastrointestinal absorption of drugs from different classes in the biopharmaceutics classification system, when treating with liraglutide. Mol Pharm. 2015;12(11):4166‐4173. doi: 10.1021/acs.molpharmaceut.5b00278  [DOI](https://doi.org/10.1021/acs.molpharmaceut.5b00278) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26426736/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Pharm&title=Effect%20on%20the%20gastrointestinal%20absorption%20of%20drugs%20from%20different%20classes%20in%20the%20biopharmaceutics%20classification%20system,%20when%20treating%20with%20liraglutide&author=M%20Malm%E2%80%90Erjefalt&author=M%20Ekblom&author=J%20Vouis&author=M%20Zdravkovic&author=H%20Lennernas&volume=12&issue=11&publication_year=2015&pages=4166-4173&pmid=26426736&doi=10.1021/acs.molpharmaceut.5b00278&)

35. George S, Blay JY, Casali PG, et al. Clinical evaluation of continuous daily dosing of sunitinib malate in patients with advanced gastrointestinal stromal tumour after imatinib failure. Eur J Cancer. 2009;45(11):1959‐1968. doi: 10.1016/j.ejca.2009.02.011  [DOI](https://doi.org/10.1016/j.ejca.2009.02.011) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19282169/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Cancer&title=Clinical%20evaluation%20of%20continuous%20daily%20dosing%20of%20sunitinib%20malate%20in%20patients%20with%20advanced%20gastrointestinal%20stromal%20tumour%20after%20imatinib%20failure&author=S%20George&author=JY%20Blay&author=PG%20Casali&volume=45&issue=11&publication_year=2009&pages=1959-1968&pmid=19282169&doi=10.1016/j.ejca.2009.02.011&)

36. Motzer RJ, Hutson TE, Olsen MR, et al. Randomized phase II trial of sunitinib on an intermittent versus continuous dosing schedule as first‐line therapy for advanced renal cell carcinoma. J Clin Oncol. 2012;30(12):1371‐1377. doi: 10.1200/JCO.2011.36.4133  [DOI](https://doi.org/10.1200/JCO.2011.36.4133) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22430274/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Randomized%20phase%20II%20trial%20of%20sunitinib%20on%20an%20intermittent%20versus%20continuous%20dosing%20schedule%20as%20first%E2%80%90line%20therapy%20for%20advanced%20renal%20cell%20carcinoma&author=RJ%20Motzer&author=TE%20Hutson&author=MR%20Olsen&volume=30&issue=12&publication_year=2012&pages=1371-1377&pmid=22430274&doi=10.1200/JCO.2011.36.4133&)
