# Durable response of lung carcinoma patients to EGFR tyrosine kinase inhibitors is determined by germline polymorphisms in some immune-related genes

## Metadata
**Authors:** Lorraine Dalens, Julie Niogret, Corentin Richard, Sandy Chevrier, Pascal Foucher, Bruno Coudert, Aurélie Lagrange, Laure Favier, Virginie Westeel, Stefano Kim, Olivier Adotevi, Caroline Chapusot, Laurent Martin, Laurent Arnould, Courèche-Guillaume Kaderbhai, Romain Boidot
**Journal:** Molecular Cancer
**Date:** 2023 Jul 29
**DOI:** [10.1186/s12943-023-01829-4](https://doi.org/10.1186/s12943-023-01829-4)
**PMID:** 37516818
**PMCID:** PMC10385908
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10385908/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC10385908/pdf/12943_2023_Article_1829.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC10385908/pdf/12943_2023_Article_1829.pdf)

## Abstract

**Background:** 
Non-small cell lung cancer is a very poor prognosis disease. Molecular analyses have highlighted several genetic alterations which may be targeted by specific therapies. In clinical practice, progression-free survival on EGFR TKI treatment is between 12 and 14 months. However, some patients progress rapidly in less than 6 months, while others remain free of progression for 16 months or even longer during EGFR TKI treatment.

**Methods:** 
We sequenced tumor exomes from 135 lung cancer patients (79 with EGFR-wildtype (WT), 56 with EGFR-mutant tumors) enrolled in the ALCAPONE trial (genomic analysis of lung cancers by next generation sequencing for personalized treatment).

**Results:** 
Some germline polymorphisms were enriched in the EGFR-mutant subset compared to EGFR-WT tumors or to a reference population. However, the most interesting observation was the negative impact of some germline SNPs in immunity-related genes on survival on EGFR TKI treatment. Indeed, the presence of one of three particular SNPs in the HLA-DRB5 gene was associated with a decreased PFS on EGFR TKI. Moreover, some SNPs in the KIR3DL1 and KIR3DL2 genes were linked to a decrease in both progression-free and overall survival of patients with EGFR-mutant tumors.

**Conclusion:** 
Our data suggest that SNPs in genes expressed by immune cells may influence the response to targeted treatments, such as EGFR TKIs. This indicates that the impact of these cells may not be limited to modulating the response to immunotherapies. Further studies are needed to determine the exact mechanisms underlying this influence and to identify the associated predictive and prognostic markers that would allow to refine treatments and so improve lung cancer patient outcomes.

**Trial registration:** 
NCT02281214: NGS Genome Analysis in Personalization of Lung Cancer Treatment (ALCAPONE).

**Supplementary Information:** 
The online version contains supplementary material available at 10.1186/s12943-023-01829-4.

Keywords: Lung carcinoma, Exome analysis, SNPs, Innate immunity, EGFR TKI

### Background

Non-small cell lung cancer is a very poor prognosis disease. Molecular analyses have highlighted several genetic alterations which may be targeted by specific therapies. In clinical practice, progression-free survival on EGFR TKI treatment is between 12 and 14 months. However, some patients progress rapidly in less than 6 months, while others remain free of progression for 16 months or even longer during EGFR TKI treatment.

### Methods

We sequenced tumor exomes from 135 lung cancer patients (79 with *EGFR*-wildtype (WT), 56 with *EGFR*-mutant tumors) enrolled in the ALCAPONE trial (genomic analysis of lung cancers by next generation sequencing for personalized treatment).

### Results

Some germline polymorphisms were enriched in the *EGFR*-mutant subset compared to *EGFR*-WT tumors or to a reference population. However, the most interesting observation was the negative impact of some germline SNPs in immunity-related genes on survival on EGFR TKI treatment. Indeed, the presence of one of three particular SNPs in the *HLA-DRB5* gene was associated with a decreased PFS on EGFR TKI*.* Moreover, some SNPs in the *KIR3DL1* and *KIR3DL2* genes were linked to a decrease in both progression-free and overall survival of patients with *EGFR*-mutant tumors.

### Conclusion

Our data suggest that SNPs in genes expressed by immune cells may influence the response to targeted treatments, such as EGFR TKIs. This indicates that the impact of these cells may not be limited to modulating the response to immunotherapies. Further studies are needed to determine the exact mechanisms underlying this influence and to identify the associated predictive and prognostic markers that would allow to refine treatments and so improve lung cancer patient outcomes.

### Trial registration

[NCT02281214](https://clinicaltrials.gov/ct2/show/NCT02281214): NGS Genome Analysis in Personalization of Lung Cancer Treatment (ALCAPONE).

### Supplementary Information

The online version contains supplementary material available at 10.1186/s12943-023-01829-4.

## Introduction

Lung cancer is one of the most frequent cancers and one of the leading causes of cancer death worldwide. Extensive studies of the mutational landscape in lung carcinoma, particularly in non-small cell lung carcinoma (NSCLC), have revealed the existence of specific alterations, many of which have emerged as therapeutic targets or biomarkers predicting response to treatment over the past decade. The most well-known targetable alterations in NSCLC are mutations in the *EGFR* gene, *ALK* fusions, *ROS1* fusions, mutations in splicing sites around exon 14 of the *MET* gene, and—more recently added to the list of actionable variants—the p.(Val600Glu) mutation in the *BRAF* gene and the p.(Gly12Cys) mutation in the *KRAS* gene. Alterations in other genes, such as *KEAP1*, *STK11*, or *TP53,* have been described as biomarkers of response to immunotherapy.

Among therapeutics targeting these specific molecular alterations, EGFR tyrosine kinase inhibitors (TKIs) have been in use for nearly 20 years. While third generation EGFR TKIs, like osimertinib, have been approved as firstline treatment for metastatic NSCLC, first- and second-generation EGFR TKIs are still in use. Despite the overall efficacy of these targeted therapies in patients with *EGFR*-mutant tumors, some patients do not achieve a durable response to EGFR TKI treatment. Some mechanisms of resistance to EGFR TKIs have been described, including the emergence of the p.(Thr790Met) mutation in the *EGFR* gene, conferring resistance to the first- and second-generation inhibitors, as well as the *EGFR* p.(Cys797Ser) mutation, which confers resistance to the third-generation TKIs. However, reliable markers of durable treatment response remain elusive. To date, only two predictive markers have been described in the literature: sensitizing mutations in the *EGFR* gene and *CDK4/6* amplification, neither of which indicates a marker of durable treatment response.

Single Nucleotide Polymorphisms (SNPs) are common genetic variations found in all individuals. While most SNPs are neutral and do not affect protein function, some can influence protein activity. For example, SNPs in the *DPYD* gene have been linked to toxicity to 5-fluorouracil. Genome-wide studies have identified numerous SNPs that appear to be linked to susceptibility to different diseases or disorders.

In this study, we carried out whole-exome sequencing on 135 lung carcinomas with known *EGFR* mutational status (79 tumors with wild-type *EGFR* and 56 with *EGFR*-mutant tumors) to determine whether some variants may impact the durability of response to treatment in patients with *EGFR*-mutant tumors.

## Methods

### Study design and patients

The patients included in this study were participants in a multicenter, prospective clinical trial, ALCAPONE ([NCT02281214](https://clinicaltrials.gov/ct2/show/NCT02281214)) (genomic analysis of lung cancer by next-generation sequencing for personalization of treatment). This trial was initiated in November 2014, and aimed to investigate possible associations between NGS-identified genomic biomarkers and the effectiveness of some treatments in patients with advanced lung cancer. The participants were recruited from among patients treated at the Dijon Cancer Center (*Centre Georges-Francois Leclerc*), the Dijon University Hospital, or the Besançon University Hospital (France). A total of 165 patients were recruited between November 2014 and August 2018. Patients were considered as eligible if they presented a locally advanced inoperable or metastatic lung cancer. We only included patients with incurable disease. Detailed inclusion and exclusion criteria are available on ClinicalTrials.gov. The clinical data of the population was presented as [Supplementary Results](#MOESM1).

All patients signed an informed consent to participate in the trial. The trial protocol was approved by an institutional review committee and by the relevant French regional ethical committee (*Comité de protection des personnes Est*). Of note, the information on race/ethnicity of the study participants was not collected due to legal constraints (collecting such data is forbidden in France). However, based on the location of participating hospitals and the patients’ family names, we can presume that a vast majority of the patients were of Caucasian origins. Therefore, we used the Non-Finnish European (NFE) population (corresponding to Caucasian population) as a reference. In genetics, SNP frequency depends on ethnic origin, and some SNPs are specific to certain races. Hence, using a reference population that corresponds to the studied population is necessary.

### DNA isolation

DNA was isolated from formalin-fixed paraffin-embedded (FFPE) tumor tissue using the Maxwell 16 FFPE Plus LEV DNA Purification kit (Promega, Madison, WI, USA) following the manufacturer’s instructions. The tumor cell content was assessed by a pathologist on a hematoxylin-and-eosin-stained slide from the same core biopsy which was used for DNA extraction. Only samples containing more than 20% of tumor cells were included in the molecular analyses. The quantity of the extracted genomic DNA was assessed with a fluorimetric method using a Qubit device (Fisher Scientific, Waltham, MA, USA).

### Whole-exome capture and sequencing

The library was prepared from 200ng genomic DNA using the Agilent SureSelect XT reagent kit (AgilentTechnologies, SantaClara, USA) according to the manufacturer’s protocol. The totality of the enriched library was hybridized and captured with the SureSelect All Exon v5 (Agilent Technologies) baits. Following hybridization, the captured libraries were purified according to the manufacturer's recommendations and amplified by polymerase chain reaction (12 cycles). Normalized libraries were pooled and DNA was sequenced on an Illumina NextSeq500 device using 2*151 bp paired-end reads, and then multiplexed. Tumor DNA sequencing generated mean target coverage of 80X.

### Exome analysis workflow

Raw DNA sequencing data were aligned to the hg19 genome built using the Burrows-Wheeler Aligner (BWA), version 0.7.15. Duplicates were marked with Picard version 2.5.0. Base quality scores recalibration and variant calling were performed using GATK tools, version 3.6. Single Nucleotide Variations were annotated using the Variant Studio Illumina software. Filters of candidate variants included the coverage depth of 10X or higher and a variant nucleotide allelic fraction in tumor DNA above 5%.

All genomic analyses were performed at the Dijon Cancer Center.

### Statistical analysis

To decrease the number of nucleotide alterations to test and decrease the probability of false-positive results, we have applied filters to select variants (SNPs and somatic mutations) present in more than 10% of individuals but in less than 90% of patients. These filters were applied on all groups or subsets on which we performed statistical analyses (full cohort, mutated EGFR patients, WT EGFR patients, PFS, OS…) and only the common variants were kept for the analysis. This step enabled us to select only 461 SNPs for statistical analysis.

Differences between the two groups were assessed using the Chi-Square test for categorial variables and the Student’s t-test for continuous variables. The objective response rate (ORR) was defined as the proportion of patients with a complete response or partial response to treatment according to the Response Evaluation Criteria in Solid Tumors (RECIST). Progression-free survival (PFS) was calculated as the time from the first metastatic treatment initiation to either evidence of progression, or death. Overall survival (OS) was defined as the time from the first metastatic treatment initiation to death. Both survival parameters were censored at the last follow-up timepoint if no event was recorded. The Kaplan–Meier estimator was used to calculate survival proportions. Log-rank tests were performed to assess the difference in survival between the groups. Univariate analyses aiming to determine the possible association of different DNA variants with PFS and OS were performed using Cox proportional hazard regression models in order to calculate hazard ratios (HR) with 95% confidence intervals. All *p-values* were adjusted for multi-testing by using the Benjamini–Hochberg procedure. All statistical tests were two-sided with *p*-values below 0.01 considered as indicative of a statistical significance. Statistical analyses were performed with R language v4.1.0 (R Core Team, 2021) and GraphPad Prism software v8.0.2 (GraphPad Software, La Jolla, CA, USA).

## Results

### Some germline polymorphisms are enriched in EGFR-mutant tumors

It is well known that *EGFR* mutations are mutually exclusive with some other gene variants in lung cancers, especially those in the *KRAS* gene. The analysis of all variants detected with whole-exome sequencing confirmed that only *EGFR*-WT tumors had *KRAS* mutations, with 17 tumors (21.52%) harboring gain-of-function mutations in this gene. Interestingly, three other genes were also altered only in *EGFR*-WT tumors: *KEAP1 (*20 patients; 25.32%), *STK11* (16 patients; 20.25%), and *UNC80* (11 patients; 13.92%; Supplementary Figure S[3](#MOESM4)A). In the subset of *EGFR*-mutant tumors, all variants found in these three genes were Single Nucleotide Polymorphisms (SNPs; Supplementary Table S[2](#MOESM6)). These gene variants were present in 43 (54.43%) patients with *EGFR*-WT tumors. Twenty-seven (34.18%) of them had variants in one gene, 11 (13.92%) had tumor variants in two genes, and 5 (6.33%) in all the three genes (Supplementary Figure S[3](#MOESM4)A).

In contrast, we did not observe any specific variants occurring only in *EGFR*-mutant tumors. Nevertheless, some variants – classified as SNPs - were enriched in *EGFR*-mutant tumors compared to *EGFR*-WT tumors (Supplementary Fig. [3](#MOESM4)B, Supplementary Table S[3](#MOESM7)). The *CHRNA3* c.67_69del variant was significantly more frequent in *EGFR*-mutant tumors compared to *EGFR*-WT tumors (*p* < 0.001). The same association was observed for three other SNPs: *COL18A1* c.3378_3386del (*p* = 0.001), *PLBD1* c.74_76del (*p* < 0.001), and *ZFPM1* c.1335_1340del (*p* < 0.001).

### Polymorphisms in genes of the antigen processing and presentation pathway are linked to worse PFS on EGFR-TKI

We then analyzed whether some genetic variants might impact objective response to first-line treatment and progression-free survival of patients. We confirmed that mutations in exon 20 of the *EGFR* gene were associated with lack of response to treatment and negatively affected PFS (Supplementary Figure S[1](#MOESM2)A). Hence, we excluded the two patients with exon-20 mutations from further analyses. No variant was significantly associated with the rate of objective response to treatment. However, polymorphisms in five genes impacted PFS of patients receiving EGFR TKIs (Fig. [1](#Fig1)A). Two SNPs were found in the *CTBP2* gene: rs376695472 (c.2036 T > C, p.(Ile679Thr)) and rs945665113 (c.2059C > T, p.(Arg687Trp)), and both of them were associated with a better PFS of patients receiving EGFR TKI treatment (HR = 0.36 [0.19–0.67]; *p* = 0.0009 and HR = 0.39 [0.22–0.70]; *p* = 0.0012). Conversely, these SNPs did not affect PFS of patients with *EGFR*-WT tumors (Fig. [1](#Fig1)A). The presence of both *CTBP2* variants in the tumor was associated with a median PFS of 20.1 months, the presence of either of the two SNPs was associated with a median PFS of 15.9 months, while the patients lacking both variants had a median PFS of 9.5 months (Fig. [1](#Fig1)B).

### Fig. 1.

![Fig. 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/59e5/10385908/736d9ba915b4/12943_2023_1829_Fig1_HTML.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=10385908_12943_2023_1829_Fig1_HTML.jpg)

The impact of genetic variants on progression-free survival (PFS) of lung carcinoma patients with EGFR-mutant (M +) or EGFR-wildtype (WT) tumors. A Forest plot showing the effects of variants with a significant impact on PFS of patients with EGFR M + patients treated with EGFR tyrosine kinase inhibitors (TKIs) compared to PFS of patients with EGFR-WT tumors receiving another firstline treatment. B-E Kaplan–Meier curves for PFS. Patients with EGFR-mutant tumors were stratified according to the number of variants in the CTBP2 gene (B), the absence or presence of at least one variant in the HLA-DRB5 (C), in KIR3DL1 (D), or in the KIR3DL2 gene (E). F Heatmap summarizing the effects of all these variants on PFS of patients with EGFR M + tumors

Seven polymorphisms spread along three other genes: *HLA-DRB5, KIR3DL1,* and *KIR3DL2,* were associated with a worse PFS in patients receiving an EGFR TKI treatment. These included the following three variants in the *HLA-DRB5 gene*: rs1071748 (c.119A > G, p.(Asp40Gly)), rs184278615 (c.160C > T, p.(Arg54Trp)), and rs201469165 (c.163 T > A, p.(Phe55Ile)) (HR = 2.52 [1.24–5.09] *p* = 0.008, HR = 3.49 [1.58–7.68] *p* = 0.001, and HR = 2.78 [1.31–5.89] *p* = 0.005, respectively (Fig. [1](#Fig1)A)). Similar to the two polymorphisms in the *CTBP2 gene*, the negative effect on PFS was observed only for patients with *EGFR*-mutant tumors. Overall, patients with tumors that did not harbor any of these three variants had better PFS than those with at least one of them (median PFS 14.6 versus 8.1 months; *p* < 0.001; Fig. [1](#Fig1)C). Similarly, three polymorphisms in the *KIR3DL1* gene: rs199822561 (c.1277 T > C, p.(Ile426Thr)), rs200821495 (c.1279 T > A, p.(Leu427Met)), and rs72629105 (c.1286C > T, p.(Thr429Met)), were associated with a decreased PFS only in patients with *EGFR*-mutant tumors treated with an EGFR TKI (HR = 2.79 [1.26–6.14]; *p* = 0.008; Fig. [1](#Fig1)A). The presence of at least one of these three polymorphisms in the tumor decreased the median PFS from 13.6 months to 7.2 months (Fig. [1](#Fig1)D). A similar effect was found for the rs973541788 (c.1156C > G, p.(Gln386Glu)) polymorphism in the *KIR3DL2* gene: worse PFS for patients with mutant tumors compared to that of patients whose tumors did not harbor this variant (HR = 2.32 [1.27–4.24] *p* = 0.005) (Fig. [1](#Fig1)A). The presence of the SNP decreased the median PFS from 15.2 to 9.7 months (Fig. [1](#Fig1)E). The negative impact of all variants except those in the *CTBP2* gene on PFS, as well as the positive effect of the latter were consistent with the heatmap representation (Fig. [1](#Fig1)F). Another SNP—in the *MUC17* gene—was also linked to worse PFS, but this effect was not specific to patients with *EGFR*-mutant tumors.

### Some polymorphisms in immunity-related genes are linked to better OS of patients receiving EGFR-TKI

Next, we wondered whether the positive effect of different SNPs on PFS of patients receiving an EGFR TKI treatment affected their OS as well. We found that among the nine SNPs impacting PFS, only those in the *KIR3DL1* and *KIR3DL2* genes were significantly associated with a worse OS (Fig. [2](#Fig2)A). Indeed, the presence of at least one of the following *KIR3DL1* SNPs: rs72629105, rs200821495, or rs199822561, was associated with a worse OS in patients with *EGFR*-mutant tumors (median OS of 10.6 months versus 32.5 months; *p* < 0.001; Fig. [2](#Fig2)B).

### Fig. 2.

![Fig. 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/59e5/10385908/28123a829d7e/12943_2023_1829_Fig2_HTML.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=10385908_12943_2023_1829_Fig2_HTML.jpg)

Genetic variants and overall survival (OS) of lung carcinoma patients with EGFR-mutant (M +) or EGFR-wildtype (WT) tumors. A Forest plot showing how variants with a significant impact on PFS affect the OS of patients with EGFR M + tumors treated with EGFR TKIs (in orange). Effects on the OS of patients with EGFR-WT tumors are displayed in blue. B, C Kaplan–Meier curves for PFS. Patients with EGFR-mutant tumors were stratified according to (B) the absence or presence of at least one variant in the KIR3DL1 gene, or (C) the absence or presence of the variant in the KIR3DL2 gene. D Forest plot illustrating the effect of variants with a significant impact on OS for patients with EGFR M + tumors treated with EGFR TKIs (in orange). The effects on the OS of patients with EGFR-WT tumors are displayed in blue. E Heatmap summarizing the effect of all these variants on the OS of patients with EGFR M + tumors

The same association was observed for the rs973541788 variant in the *KIR3DL2* gene, with a median patient OS of 18 months if the variant was present in the tumor versus 42.6 months in case of its absence (Fig. [2](#Fig2)C). Moreover, we tested whether other variants detected by whole-exome sequencing might influence OS of patients with *EGFR*-mutant tumors. In addition to the four above-mentioned SNPs impacting both PFS and OS, we found that eleven other variants, mainly in genes encoding proteins involved in antigen processing and presentation, were linked to a worse OS only in the subset of patients with *EGFR*-mutant tumors (Fig. [2](#Fig2)D, Supplementary Table S[4](#MOESM8)). Indeed, it appeared that the frequency of these variants was higher in patients with a poor OS (Fig. [2](#Fig2)E).

## Discussion

The use of targeted therapies in lung cancer has revolutionized patient care in the past several years, especially for patients with *EGFR*-mutant tumors treated with EGFR TKIs. However, despite the specificity of EGFR TKIs, some patients do not experience long-lasting benefits from these treatments. Even with the early administration of the third-generation EGFR TKIs which are now used as the first line of treatment, some patients progress earlier than others. The development of resistance to EGFR TKIs has been thoroughly studied but, to date, no biomarker of response duration has been identified, except the presence of activating mutations in the *EGFR* gene itself. To address this gap, we conducted a study analyzing tumors from 135 lung cancer patients (including 56 patients with *EGFR*-mutant tumors and 79 with *EGFR*-WT tumors) using whole-exome sequencing.

We first confirmed that *KRAS* mutations were mutually exclusive with *EGFR* mutations. Moreover, we observed that three other genes were mutated only in *EGFR*-WT tumors: *KEAP1*, *STK11*, and *UNC80*. Two of these genes: *STK11* and *KEAP1*, are known to impact response to immunotherapy whereas the third one: *UNC80*, encodes a component of an ion channel complex and has never been found to be directly involved in any cancer-related mechanism [[1](#CR1)]. However, the UNC80 protein has been shown to bind Src kinases [[2](#CR2)] that do play some role in cancer progression. We did not identify any recurrent somatic variations associated with *EGFR* mutations. Nonetheless, we observed a significant enrichment of four SNPs: *CHRNA3* rs66793222*, COL18A1 c.3378_3386del, PLBD1* rs147342083, and *ZFPM1* rs149145771*,* in *EGFR*-mutant tumors.

The *CHRNA3* gene codes for a member of the nicotinic acetylcholine receptor family. Polymorphisms in this gene have been reported to be associated with heavy smoking [[3](#CR3)], but also with susceptibility to lung cancer in the Chinese population [[4](#CR4)]. We hypothesize that the enrichment of the *CHRNA3* rs66793222 variant we observed in patients with *EGFR*-mutant tumors compared to the reference Non-Finnish European (NFE) population may reflect a possible preventive effect of this SNP on tobacco dependence (people with this SNP are less susceptible to develop nicotine dependence). Indeed, it is well known that *EGFR* mutations are more frequent in lung tumors in never-smokers, and the lower frequency of *rs66793222* in *EGFR*-WT tumors seems to support this hypothesis.

The second SNP we identified was an in-frame deletion c.3378_3386del in the *COL18A1* gene that encodes the alpha chain of type XVIII collagen. Its frequency in patients with *EGFR*-WT tumors was similar to that in the NFE population. On the contrary, the frequency of this SNP in *EGFR*-mutant tumors was 12-fold higher than in NFE population, with more than 32% of patients with *EGFR*-mutant tumors harboring this SNP. While this particular SNP has not been described in the literature, another SNP in the *COL18A1* gene, *D104N*, has been shown to increase the risk of developing osteosarcoma [[5](#CR5)] and sporadic breast cancer [[6](#CR6)]. Our results suggest that the c.3378_3386del polymorphism might be associated with an increased prevalence of *EGFR*-mutant lung cancer. However, further studies are needed to confirm this hypothesis.

The two other SNPs enriched in patients with *EGFR*-mutant tumors were *PLBD1* rs147342083 and *ZFPM1* rs149145771. *PLBD1* codes for a phospholipase expressed in neutrophils, with its exact role being poorly understood. On the other hand, whereas the *ZFPM1* gene encodes a transcription factor, FOG1, well known for its role in erythroid and megakaryocytic cell differentiation [[7](#CR7)]. We found that the frequencies of both variants in these two genes were very low in *EGFR*-WT tumors compared to the NFE population, suggesting that these SNPs might protect from *EGFR*-WT lung cancer.

While the presence of an *EGFR* mutation in the tumor is a known biomarker of sensitivity to EGFR TKIs, no other marker predicting the duration of response to EGFR TKI treatment has ever been reported in the literature. Therefore, we investigated whether the presence of variants in some other genes found using the exome analysis might influence progression-free survival (PFS) in patients receiving EGFR TKI. We found that polymorphisms in four genes—*CTBP2, HLA-DRB5*, *KIR3DL1*, and *KIR3DL2*—might have an impact. The *CTBP2* gene codes for a transcriptional repressor involved in tumorigenesis and cancer progression. Moreover, a high mRNA expression of CTBP2 was linked to a poor prognosis in patients with different cancers, including lung adenocarcinoma [[8](#CR8)]. In our study, we focused on nucleotide variations in the *CTBP2* gene. It appeared that patients with the *CTBP2* reference genotype had a worse PFS than those harboring one SNP in this gene, and those with one SNP fared worse than those with two SNPs in the gene. This suggests that the *CTBP2* genotype may impact the activity of the resulting protein, particularly when it is overexpressed in cancer. Further analysis of the Wnt/β-catenin pathway activation, that could be triggered by CTBP2, may provide additional insights into the effect of the *CTBP2* genotype on patient survival.

On the contrary, the presence of a SNP in *HLA-DRB5*, *KIR3DL1*, and/or *KIR3DL2* immune-related genes decreased PFS of patients receiving EGFR TKI. The HLA-DRB5 mRNA expression has previously been associated with a better prognosis in lung adenocarcinoma [[9](#CR9)]. In our study, the presence of any of the three aforementioned SNPs was significantly associated with worse PFS in patients receiving EGFR TKIs. This suggests that immunity may play a role in response to EGFR TKIs. This hypothesis is reinforced by our finding that particular *KIR3DL1* and *KIR3DL2* SNPs may impact the PFS of patients receiving such a treatment. Indeed, as for *HLA-DRB5*, patients with the reference sequence of *KIR3DL1* and *KIR3DL2* had a better PFS than those with at least one of the identified SNPs. Interestingly, we observed that patients with the reference sequence of *KIR3DL1* or *KIR3DL2* had also a better OS than those harboring at least one SNP in both genes. In addition, another gene of the KIR family, *KIR2DL4*, had a similar impact on survival as *KIR3DL1* and *KIR3DL2*, with better PFS for patients with the reference sequence than those who had at least one SNP in this gene. Of note, even though we detected these SNPs in the tumor exome analysis, they are also present in non-cancer cells, including immune cells. *KIR3DL1/2* and *KIR2DL4* are killer immunoglobulin-like receptors (KIR), mainly expressed by Natural Killer (NK) cells. They belong to haplotype A, present in about 50% of humans [[10](#CR10)], and localized in the telomeric KIR region. It seems that ˗ at least in the South European population ˗ *KIR3DL2* and *KIR2DL4* are present in 100% of KIR profiles, and *KIR3DL1* is present in almost 82% of profiles [[11](#CR11)]. KIRs interact with HLA class I molecules. When KIRs detect a self-molecule, NK cells are inhibited, avoiding the destruction of normal cells. When they detect a tumor cell, the interaction between HLA class I molecule with KIR activates NK cells to destroy abnormal cells. EGFR TKIs were shown to increase the expression of HLA class I molecules at the surface of tumor cells [[12](#CR12)]. They also increase the tumor-induced chemotaxis of immune cells, which results in an increased tumor infiltration by immune cells, including NK cells [[12](#CR12)]. In addition, it was reported that erlotinib treatment of tumors with the p.(Leu858Arg) *EGFR*-mutation enhances the antigen-presenting capabilities of antigen-presenting cells [[13](#CR13)]. Finally, the hypothesis that the involvement of NK cells is important for effective treatment of *EGFR*-mutant lung adenocarcinomas is supported by the positive impact of autologous NK cell immunotherapy in advanced lung adenocarcinoma, especially in patients with *EGFR*-mutant tumors [[14](#CR14)]. Of note, a phase I/II clinical trial studying the combination of natural killer T cells and gefitinib in the treatment of advanced *EGFR*-mutant lung cancer started is currently underway [[15](#CR15)].

## Conclusions

In conclusion, our work shows that SNPs may impact the response to tumor-targeting therapeutics such as tyrosine kinase inhibitors in lung cancer patients, probably by affecting the tumor microenvironment and non-tumor cells, especially immune cells.

## Supplementary Information

## Acknowledgements

We warmly acknowledge Katarzyna Szymanska for English editing of the manuscript.

## Authors’ contributions

Study design and supervision: RB; Data curation: LD, JN, CGK; Funding acquisition: BC, RB; Investigation: PF, BC, AL, LF, VW, SK, CGK, OA, CC, LM, LA; Result analysis: CR, SC; Result visualization: CR, RB; Manuscript writing: RB, CR.

## Funding

The ALCAPONE trial was partly funded by Boehringer Ingelheim.

## Availability of data and materials

Genomic data may be shared upon reasonable request to the corresponding author in accordance with the French law on genomic data.

## Declarations

### Ethics approval and consent to participate

All patients signed an informed consent to participate in the trial. The trial protocol was approved by an institutional review committee and by the relevant French regional ethical committee (*Comité de protection des personnes Est*).

### Consent for publication

All patients, by giving their consent to participate, have given their consent for publication.

### Competing interests

The authors declare no competing interests.

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

Genomic data may be shared upon reasonable request to the corresponding author in accordance with the French law on genomic data.

### Supplementary Materials

### Data Availability Statement

Genomic data may be shared upon reasonable request to the corresponding author in accordance with the French law on genomic data.

## References

1. Bramswig NC, Zaki MS. UNC80 Deficiency. 2017 Sep 21 [updated 2023 May 18]. In: Adam MP, Mirzaa GM, Pagon RA, Wallace SE, Bean LJH, Gripp KW, Amemiya A, editors. GeneReviews® [Internet]. Seattle: University of Washington, Seattle; 1993–2023. PMID: 28933810.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/28933810/)

2. Wang H, Ren D. UNC80 functions as a scaffold for Src kinases in NALCN channel function. Channels (Austin) 2009;3:161–163. doi: 10.4161/chan.3.3.8853.  [DOI](https://doi.org/10.4161/chan.3.3.8853) | [PMC free article](/articles/PMC2810456/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19535918/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Channels%20(Austin)&title=UNC80%20functions%20as%20a%20scaffold%20for%20Src%20kinases%20in%20NALCN%20channel%20function&author=H%20Wang&author=D%20Ren&volume=3&publication_year=2009&pages=161-163&pmid=19535918&doi=10.4161/chan.3.3.8853&)

3. Perez-Morales R, Gonzalez-Zamora A, Gonzalez-Delgado MF, Calleros Rincon EY, Olivas Calderon EH, Martinez-Ramirez OC, Rubio J. CHRNA3 rs1051730 and CHRNA5 rs16969968 polymorphisms are associated with heavy smoking, lung cancer, and chronic obstructive pulmonary disease in a Mexican population. Ann Hum Genet. 2018;82:415–424. doi: 10.1111/ahg.12264.  [DOI](https://doi.org/10.1111/ahg.12264) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29993116/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Hum%20Genet&title=CHRNA3%20rs1051730%20and%20CHRNA5%20rs16969968%20polymorphisms%20are%20associated%20with%20heavy%20smoking,%20lung%20cancer,%20and%20chronic%20obstructive%20pulmonary%20disease%20in%20a%20Mexican%20population&author=R%20Perez-Morales&author=A%20Gonzalez-Zamora&author=MF%20Gonzalez-Delgado&author=EY%20Calleros%20Rincon&author=EH%20Olivas%20Calderon&volume=82&publication_year=2018&pages=415-424&pmid=29993116&doi=10.1111/ahg.12264&)

4. Qu X, Wang K, Dong W, Shen H, Wang Y, Liu Q, Du J. Association between two CHRNA3 variants and susceptibility of lung cancer: a meta-analysis. Sci Rep. 2016;6:20149. doi: 10.1038/srep20149.  [DOI](https://doi.org/10.1038/srep20149) | [PMC free article](/articles/PMC4735583/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26831765/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sci%20Rep&title=Association%20between%20two%20CHRNA3%20variants%20and%20susceptibility%20of%20lung%20cancer:%20a%20meta-analysis&author=X%20Qu&author=K%20Wang&author=W%20Dong&author=H%20Shen&author=Y%20Wang&volume=6&publication_year=2016&pages=20149&pmid=26831765&doi=10.1038/srep20149&)

5. Guo Z, Zhang T, Wu J, Wang H, Liu X, Tian L. Genetic polymorphisms in COL18A1 influence the development of osteosarcoma. Int J Clin Exp Pathol. 2015;8:11531–11536.  [PMC free article](/articles/PMC4637702/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26617886/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Clin%20Exp%20Pathol&title=Genetic%20polymorphisms%20in%20COL18A1%20influence%20the%20development%20of%20osteosarcoma&author=Z%20Guo&author=T%20Zhang&author=J%20Wu&author=H%20Wang&author=X%20Liu&volume=8&publication_year=2015&pages=11531-11536&pmid=26617886&)

6. Lourenco GJ, Cardoso-Filho C, Goncales NS, Shinzato JY, Zeferino LC, Nascimento H, Costa FF, Gurgel MS, Lima CS. A high risk of occurrence of sporadic breast cancer in individuals with the 104NN polymorphism of the COL18A1 gene. Breast Cancer Res Treat. 2006;100:335–338. doi: 10.1007/s10549-006-9259-z.  [DOI](https://doi.org/10.1007/s10549-006-9259-z) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16807676/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Breast%20Cancer%20Res%20Treat&title=A%20high%20risk%20of%20occurrence%20of%20sporadic%20breast%20cancer%20in%20individuals%20with%20the%20104NN%20polymorphism%20of%20the%20COL18A1%20gene&author=GJ%20Lourenco&author=C%20Cardoso-Filho&author=NS%20Goncales&author=JY%20Shinzato&author=LC%20Zeferino&volume=100&publication_year=2006&pages=335-338&pmid=16807676&doi=10.1007/s10549-006-9259-z&)

7. Mancini E, Sanjuan-Pla A, Luciani L, Moore S, Grover A, Zay A, Rasmussen KD, Luc S, Bilbao D, O'Carroll D, et al. FOG-1 and GATA-1 act sequentially to specify definitive megakaryocytic and erythroid progenitors. EMBO J. 2012;31:351–365. doi: 10.1038/emboj.2011.390.  [DOI](https://doi.org/10.1038/emboj.2011.390) | [PMC free article](/articles/PMC3261555/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22068055/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=EMBO%20J&title=FOG-1%20and%20GATA-1%20act%20sequentially%20to%20specify%20definitive%20megakaryocytic%20and%20erythroid%20progenitors&author=E%20Mancini&author=A%20Sanjuan-Pla&author=L%20Luciani&author=S%20Moore&author=A%20Grover&volume=31&publication_year=2012&pages=351-365&pmid=22068055&doi=10.1038/emboj.2011.390&)

8. Li B, Xiang Z, Xiong F, Yan B, Huang Q. C-terminal binding protein-2 is a prognostic marker for lung adenocarcinomas. Medicine (Baltimore) 2020;99:e20076. doi: 10.1097/MD.0000000000020076.  [DOI](https://doi.org/10.1097/MD.0000000000020076) | [PMC free article](/articles/PMC7402787/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32756072/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Medicine%20(Baltimore)&title=C-terminal%20binding%20protein-2%20is%20a%20prognostic%20marker%20for%20lung%20adenocarcinomas&author=B%20Li&author=Z%20Xiang&author=F%20Xiong&author=B%20Yan&author=Q%20Huang&volume=99&publication_year=2020&pages=e20076&pmid=32756072&doi=10.1097/MD.0000000000020076&)

9. Schaafsma E, Fugle CM, Wang X, Cheng C. Pan-cancer association of HLA gene expression with cancer prognosis and immunotherapy efficacy. Br J Cancer. 2021;125:422–432. doi: 10.1038/s41416-021-01400-2.  [DOI](https://doi.org/10.1038/s41416-021-01400-2) | [PMC free article](/articles/PMC8329209/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33981015/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Cancer&title=Pan-cancer%20association%20of%20HLA%20gene%20expression%20with%20cancer%20prognosis%20and%20immunotherapy%20efficacy&author=E%20Schaafsma&author=CM%20Fugle&author=X%20Wang&author=C%20Cheng&volume=125&publication_year=2021&pages=422-432&pmid=33981015&doi=10.1038/s41416-021-01400-2&)

10. Benson DM, Jr, Caligiuri MA. Killer immunoglobulin-like receptors and tumor immunity. Cancer Immunol Res. 2014;2:99–104. doi: 10.1158/2326-6066.CIR-13-0219.  [DOI](https://doi.org/10.1158/2326-6066.CIR-13-0219) | [PMC free article](/articles/PMC3938196/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24592397/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Immunol%20Res&title=Killer%20immunoglobulin-like%20receptors%20and%20tumor%20immunity&author=DM%20Benson&author=MA%20Caligiuri&volume=2&publication_year=2014&pages=99-104&pmid=24592397&doi=10.1158/2326-6066.CIR-13-0219&)

11. Cisneros E, Moraru M, Gomez-Lozano N, Muntasell A, Lopez-Botet M, Vilches C. Haplotype-Based Analysis of KIR-Gene Profiles in a South European Population-Distribution of Standard and Variant Haplotypes, and Identification of Novel Recombinant Structures. Front Immunol. 2020;11:440. doi: 10.3389/fimmu.2020.00440.  [DOI](https://doi.org/10.3389/fimmu.2020.00440) | [PMC free article](/articles/PMC7089957/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32256494/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front%20Immunol&title=Haplotype-Based%20Analysis%20of%20KIR-Gene%20Profiles%20in%20a%20South%20European%20Population-Distribution%20of%20Standard%20and%20Variant%20Haplotypes,%20and%20Identification%20of%20Novel%20Recombinant%20Structures&author=E%20Cisneros&author=M%20Moraru&author=N%20Gomez-Lozano&author=A%20Muntasell&author=M%20Lopez-Botet&volume=11&publication_year=2020&pages=440&pmid=32256494&doi=10.3389/fimmu.2020.00440&)

12. Li F, Lizee G, Hwu P, Du X, Deng L, Talukder A, Katailiha A, Zou Q, Roszik J, Hawke D, Jackson K, Bradley S, Wang Y, Ataullakhanov R, Bagaev A, Kotlov N, Svekolkin V, Miheechev N, Frenkel F, Sonnemann H. The role of EGFR inhibitor (EGFRi) in immune cell infiltration and CD8+ T-cell activation in EGFR mutant lung cancer. Ann Oncol. 2019;30:v2. doi: 10.1093/annonc/mdz238.004.  [DOI](https://doi.org/10.1093/annonc/mdz238.004) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Oncol&title=The%20role%20of%20EGFR%20inhibitor%20(EGFRi)%20in%20immune%20cell%20infiltration%20and%20CD8+%20T-cell%20activation%20in%20EGFR%20mutant%20lung%20cancer&author=F%20Li&author=G%20Lizee&author=P%20Hwu&author=X%20Du&author=L%20Deng&volume=30&publication_year=2019&pages=v2&doi=10.1093/annonc/mdz238.004&)

13. Venugopalan A, Lee MJ, Niu G, Medina-Echeverz J, Tomita Y, Lizak MJ, Cultraro CM, Simpson RM, Chen X, Trepel JB, Guha U. EGFR-targeted therapy results in dramatic early lung tumor regression accompanied by imaging response and immune infiltration in EGFR mutant transgenic mouse models. Oncotarget. 2016;7:54137–54156. doi: 10.18632/oncotarget.11021.  [DOI](https://doi.org/10.18632/oncotarget.11021) | [PMC free article](/articles/PMC5338915/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27494838/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncotarget&title=EGFR-targeted%20therapy%20results%20in%20dramatic%20early%20lung%20tumor%20regression%20accompanied%20by%20imaging%20response%20and%20immune%20infiltration%20in%20EGFR%20mutant%20transgenic%20mouse%20models&author=A%20Venugopalan&author=MJ%20Lee&author=G%20Niu&author=J%20Medina-Echeverz&author=Y%20Tomita&volume=7&publication_year=2016&pages=54137-54156&pmid=27494838&doi=10.18632/oncotarget.11021&)

14. Hong G, Chen X, Sun X, Zhou M, Liu B, Li Z, Yu Z, Gao W, Liu T. Effect of autologous NK cell immunotherapy on advanced lung adenocarcinoma with EGFR mutations. Precis Clin Med. 2019;2:235–245. doi: 10.1093/pcmedi/pbz023.  [DOI](https://doi.org/10.1093/pcmedi/pbz023) | [PMC free article](/articles/PMC8985770/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35693880/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Precis%20Clin%20Med&title=Effect%20of%20autologous%20NK%20cell%20immunotherapy%20on%20advanced%20lung%20adenocarcinoma%20with%20EGFR%20mutations&author=G%20Hong&author=X%20Chen&author=X%20Sun&author=M%20Zhou&author=B%20Liu&volume=2&publication_year=2019&pages=235-245&pmid=35693880&doi=10.1093/pcmedi/pbz023&)

15. Yu W, Ye F, Yuan X, Ma Y, Mao C, Li X, Li J, Dai C, Qian F, Li J, et al. A phase I/II clinical trial on the efficacy and safety of NKT cells combined with gefitinib for advanced EGFR-mutated non-small-cell lung cancer. BMC Cancer. 2021;21:877. doi: 10.1186/s12885-021-08590-1.  [DOI](https://doi.org/10.1186/s12885-021-08590-1) | [PMC free article](/articles/PMC8325186/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34332557/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Cancer&title=A%20phase%20I/II%20clinical%20trial%20on%20the%20efficacy%20and%20safety%20of%20NKT%20cells%20combined%20with%20gefitinib%20for%20advanced%20EGFR-mutated%20non-small-cell%20lung%20cancer&author=W%20Yu&author=F%20Ye&author=X%20Yuan&author=Y%20Ma&author=C%20Mao&volume=21&publication_year=2021&pages=877&pmid=34332557&doi=10.1186/s12885-021-08590-1&)
