# Novel Independent Trans- and Cis-Genetic Variants Associated with CYP2D6 Expression and Activity in Human Livers

## Metadata
**Authors:** Dylan Smith, Bing He, Jian Shi, Hao-Jie Zhu, Xinwen Wang
**Journal:** Drug Metabolism and Disposition
**Date:** 2024 Feb
**DOI:** [10.1124/dmd.123.001548](https://doi.org/10.1124/dmd.123.001548)
**PMID:** 38050015
**PMCID:** PMC10801631
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10801631/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC10801631/pdf/dmd.123.001548.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC10801631/pdf/dmd.123.001548.pdf)

## Abstract

**SIGNIFICANT STATEMENT:** 
Using individual human livers, we identified four novel cis- and trans-pQTLs/aQTLs (protein quantitative trait loci/activity quantitative trait loci) of Cytochrome P450 2D6 (CYP2D6) that are independent from known functional CYP2D6 star alleles. This study connects the CYP2D6 gene expression and activity, enhancing our understanding of the genetic variants associated with CYP2D6 protein expression and activity, potentially advancing our insight into the interindividual variability in CYP2D6 substrate medication response.

### SIGNIFICANT STATEMENT

Using individual human livers, we identified four novel cis- and trans-pQTLs/aQTLs (protein quantitative trait loci/activity quantitative trait loci) of Cytochrome P450 2D6 (CYP2D6) that are independent from known functional CYP2D6 star alleles. This study connects the CYP2D6 gene expression and activity, enhancing our understanding of the genetic variants associated with CYP2D6 protein expression and activity, potentially advancing our insight into the interindividual variability in CYP2D6 substrate medication response.

## Introduction

Cytochrome P450 2D6 (CYP2D6), localized to human chromosome 22q13.2 and predominantly expressed in the human liver, stands as a significant member in the cytochrome P450 (P450) oxidase family ([Owen et al., 2009](#B28); [Ray et al., 2019](#B33)). It is believed to catalyze the metabolism of approximately 25% of clinically used medications across diverse therapeutic areas, including antidepressants (e.g., paroxetine, fluoxetine, and venlafaxine), antipsychotics (e.g., aripiprazole, clozapine, and pimozide), antineoplastic agents (e.g., tamoxifen), adrenergic antagonists (e.g., carvedilol and metoprolol), and analgesics (e.g., codeine and tramadol) ([Nofziger et al., 2020](#B27)). Apart from its role as a xenobiotic-metabolizing enzyme, CYP2D6 plays various physiologic roles in the brain ([Kirchheiner et al., 2011](#B20)). One such role is its involvement in catalyzing the synthesis of endogenous neurotransmitters such as dopamine ([Hiroi et al., 1998](#B18)) and serotonin ([Yu et al., 2003](#B48)). The activity of CYP2D6 has been associated with conditions like Parkinson’s disease ([Smith et al., 1992](#B37); [Tao et al., 1998](#B42)) and certain personality traits (e.g., anxiety and impulsivity) ([Bertilsson et al., 1989](#B2)).

CYP2D6 gene is highly polymorphic, with over 135 unique star alleles cataloged by the Pharmacogene Variation (PharmVar) Consortium, which is among the most characterized drug-metabolizing enzyme genes ([Nofziger et al., 2020](#B27)). Some of these alleles can significantly alter CYP2D6 expression and activity, leading to marked variations in response to CYP2D6 substrate medications ([Relling and Evans, 2015](#B34); [Tracy et al., 2016](#B43); [Hicks et al., 2017](#B16); [Brown et al., 2019](#B4); [Crews et al., 2021](#B7)). CYP2D6 alleles are typically divided into two tiers based on their known impact on enzyme function. Tier1 alleles are recognized for their well established and clinically significant effects on enzyme activity, meriting their inclusion in the “must-test” list published by the Association for Molecular Pathology (AMP) Pharmacogenomics (PGx). The Tier2 alleles have a moderate effect on CYP2D6 activity, which are typically less prevalent in populations (minor allele frequencies, MAFs ≤1%) ([Pratt et al., 2021](#B32)). Each allele is assigned an activity value on a scale of 0 to 1. An allele receives an activity value of 0 if it lacks functionality, 0.25 if its function is severely diminished, 0.5 if its function is reduced, and 1 if its function is normal. By summing up the activity values of all alleles present in an individual’s genotype, an activity score (AS) is obtained ([Gaedigk et al., 2008](#B11)). The AS is then translated into a metabolizer phenotype, which is categorized as poor metabolizer (PM, AS = 0), intermediate metabolizer (IM, AS = 0.25–1), normal metabolizer (NM, AS = 1.25–2.25), or ultrarapid metabolizer (UM, AS >2.25) ([Caudle et al., 2020](#B5); [Pratt et al., 2021](#B32)).

Moreover, several studies have highlighted additional genetic variants that are associated with CYP2D6 transcription, namely expression quantitative loci (eQTLs) and/or the metabolism of CYP2D6 substrate drugs, namely activity quantitative loci (aQTLs) ([Pan et al., 2017b](#B31)). For example, variants such as rs1080985 (−1584C>G) ([Zanger et al., 2001](#B49); [Dorado et al., 2009](#B8); [Llerena et al., 2013](#B25)), rs5758550 ([Wang et al., 2014](#B45); [Ray et al., 2019](#B33)), rs8138080, rs5751247, and rs17478227 ([Yang et al., 2010](#B47)) have been reportedly linked to enhanced or decreased CYP2D6 mRNA levels and activities. These variants are located within a range of 1.58 kilobase (kb) to 127 kb downstream or upstream of CYP2D6 gene ([Pan et al., 2017b](#B31)). Despite considering all known functional variants, only a fraction of the variability in CYP2D6 function could be accounted for ([Shakeel et al., 2018](#B35)). It is important to note that these studies primarily relied on CYP2D6 mRNA levels to evaluate the impact of specific SNPs ([Pan et al., 2017b](#B31)). However, the overall correlation between CYP2D6 mRNA and protein is weak (r = 0.356, *P* = 1.11 × 10^−6^) ([Shakeel et al., 2018](#B35)). Therefore, protein expression appears to be a more reliable predictor of CYP2D6-mediated drug metabolism compared with mRNA expression ([Shakeel et al., 2018](#B35)). Hence, there is an urgent need to identify novel and independent genetic variants associated with both CYP2D6 protein expression and activity.

In the current study, we performed a genome-wide association study (GWAS) to identify novel independent variants associated with CYP2D6 protein expressions in human livers. The effect of known functional CYP2D6 star alleles was controlled by conditional analysis to reveal independent variants. To validate our findings, we further assessed the effects of these independent single nuclear polymorphisms (SNPs) on CYP2D6 activity in a validation cohort of 100 human liver samples. Our study led to the discovery that, independent from the established functional CYP2D6 star alleles, five SNPs—rs1807493, rs4073010, rs1062753, rs80274432, and rs729559—were significantly associated with CYP2D6 protein expressions in human livers. Notably, we were able to confirm a consistent impact of four SNPs—rs1807493, rs4073010, rs1062753, and rs729559—on CYP2D6 activities in human livers. The findings suggest that these four SNPs might play a significant role in contributing to CYP2D6 variability in both protein expressions and activities, potentially serving as new genetic biomarkers alongside the established star alleles.

## Materials and Methods

### Materials

PureLink Genomic DNA Mini Kit, Pierce BCA Protein Assay Kit, phosphate-buffered saline (PBS), urea, ammonium bicarbonate (NH_4_HCO_3_), dl-dithiothreitol (DTT), trifluoroacetic acid (TFA), acetone, and water with 0.1% formic acid [liquid chromatography mass spectrometry (LC-MS) reagent] were from Thermo Fisher Scientific (Waltham, MA). Multi-Ethnic Genotyping Array (MEGA) was obtained from Illumina (San Diego, CA). Recombinant human NADPH-P450 oxidoreductase, NADPH- generating reagents (0.1 mg/ml of yeast glucose-6-phosphate dehydrogenase, 3 mg/ml of NADP+, and 0.07 M glucose-6-phosphate), human cytochrome b5, dextromethorphan hydrobromide, dextrorphan, d3-hydroxyomeprazole, and other P450 probe substrates and metabolites were supplied by Sigma-Aldrich (St. Louis, MO). TPCK-treated trypsin was obtained from Worthington Biochemical Corporation (Freehold, NJ). Lysyl endopeptidase (Lys-C) was acquired from Wako Chemicals (Richmond, VA). The synthetic iRT standards solution was from Biognosys AG (Cambridge, MA). Iodoacetamide (IAA) was the product of Acros Organics (Morris Plains, NJ). Acetonitrile with 0.1% formic acid (LC-MS reagent) was purchased from JT Baker Chemical Co. (Phillipsburg, NJ). Water Oasis HLB columns were obtained through Waters Corporation (Milford, MA). All other chemicals and reagents were of analytical grade and commercially available.

A total of 287 normal individual human liver samples were obtained from XenoTech LLC (Kansas City, KS), the Cooperative Human Tissue Network (CHTN, Columbus, OH), and the University of Minnesota Liver Tissue Cell Distribution System. The livers samples used in this study comprised normal tissues collected from adult donors. The donors consisted of 114 males, 156 females, and 17 with unavailable gender information. Among them, 172 were identified as Caucasian, 29 as Black, three as ‘other,’ and the ethnicity of 83 donors were unavailable.

### Genome-Wide Genotyping and Imputation

The genome-wide genotyping and imputation were conducted using the previously published method ([He et al., 2020](#B14); [Smith et al., 2023](#B38)). Briefly, DNA was extracted from the 287 human liver samples using PureLink Genomic DNA Mini Kit. A total of 500 ng of DNA at the concentration of 100 ng/*μ*l was used for genotyping with the Illumina Multi-Ethnic Global Array (MEGA). MEGA consisted of 1,779,819 markers, providing comprehensive genomic coverage for various global populations. SNP calling was performed with Genome Studio (Illumina, Miami, FL), whereas subsequent quality control (QC) analysis was conducted using PLINK (version 1.9). SNP imputation was conducted using the Michigan Imputation Server ([https://imputationserver.sph.umich.edu](https://imputationserver.sph.umich.edu)). The specific threshold for quality control and genotype imputation have been reported by [He et al. (2020](#B14)).

### Preparation of Human Liver S9 Fractions

A total of 287 individual human liver s9 fraction (HLS9) samples were prepared from human liver samples according to the method described previously ([Wang et al., 2020](#B46)). In brief, approximately 200 mg individual liver tissues were homogenized in 0.5 ml PBS on ice. The homogenates were then subjected to two rounds of centrifugation at 9000 × *g* for 20 minutes at 4°C, and the fat layers were removed after the first round. The supernatant, referred to as HLS9, was collected in a clean tube. Pooled HLS9 samples were prepared by combining protein content-normalized portions from all individuals. Throughout the preparation process, the samples were consistently kept on ice or at 4°C. After preparation, the HLS9 samples were stored at −80°C until use.

### Data-Independent Acquisition–Absolute Protein Quantification Proteomics for CYP2D6 Absolute Quantification

A total of 287 HLS9 samples were prepared for proteomic analysis using the method detailed previously ([Shi et al., 2018](#B36); [Wang et al., 2020](#B46)). A SCIEX TripleTOF 5600 plus mass spectrometer system coupled with an Eksigent 2D plus LC system was applied for the data-independent acquisition (DIA) proteomic analysis. An aliquot of the peptides derived from 6 *μ*g protein-normalized samples was loaded to the trapping column. After elution to the analytical column for separation, the peptides were injected to the mass spectrometer. To minimize sample carryover, each sample injection was followed by an injection of water. The mass spectrometer operated in positive ion mode with the ion spray voltage at 5500 V and the source temperature set at 280°C. The data-independent acquisition (DIA) scheme included a 250-millisecond time-of-flight (TOF)-MS scan from 400 to 1250 Da, as well as MS/MS scans from 100 to 1500 Da, with a 100-variable isolation window scheme. Each isolation window had an accumulation time of 25 milliseconds, resulting in a total cycle time of 2.8 seconds ([Shi et al., 2018](#B36); [Wang et al., 2020](#B46)). We applied the previously published DIA–absolute protein quantification (DIA-APQ) algorithm to obtain the absolute CYP2D6 protein expression levels ([He et al., 2019](#B13)).

### P450 Cocktail Incubation and LC-MS/MS Analysis for CYP2D6 Activities

The CYP2D6 activities were assessed in 100 HLS9 samples through the metabolism rates of its probe substrate, dextromethorphan. The rates of dextromethorphan O-demethylation were obtained using a P450 cocktail incubation method published previously ([Spaggiari et al., 2014](#B40)), with minor modifications. In brief, HLS9 at a final protein concentration of 2 mg/ml were preincubated with a five-P450 substrate cocktail, including 20 *μ*M phenacetin (CYP1A2), 50 *μ*M tolbutamide (CYP2C9), 2 *μ*M omeprazole (CYP2C19), 2 *μ*M dextromethorphan hydrobromide (CYP2D6), and 2 *μ*M midazolam (CYP3A4), in a phosphate-buffered saline (PBS) solution containing 20 mM HEPES, pH 7.4, 0.1 mM EDTA, and 1.25 mM MgCl_2_ at 37°C for 5 minutes. The reaction was then initiated by addition of an NADPH-generating system (containing 50 parts 0.1M glucose-6-phosphate, 25 parts 10 mg/ml NADP+, and one part 1 mg/ml yeast glucose-6- phosphate dehydrogenase). After a 30-minute incubation at 37°C, the reaction was terminated by adding a 2-fold volume of acetonitrile containing the internal standard (IS) d3-hydroxy-omeprazole (20 ng/ml). The samples were vortexed and then centrifuged at 13.2 × 1000 rpm at 4°C for 20 minutes. At last, the supernatant was transferred for the LC-MS/MS assay. Dextrorphan, the product of dextromethorphan O-demethylation, was measured using a Shimadzu HPLC system (Shimadzu, Tokyo, Japan) coupled with an Applied Biosystems API 4000 triple quadrupole/linear ion trap (QTRAP) mass spectrometer (Foster City, CA). A Shimadzu VP-ODS column (5 *μ*m, 150 × 2.0 mm; Shimadzu, Tokyo, Japan) was used to separate the analytes. The mobile phase consisted of water (A) and methanol (B) containing 6 mM ammonium acetate and 0.2% formic acid at a flow rate of 0.2 ml/min. A gradient elution was applied for the separation with the following time program: mobile phase B was increased from 5% to 50% during the first 2 minutes and continued to increase to 70% from 2 to 7 minutes, then returned to 5% at 7.5 minutes and maintained at 5% until the end of the elution at 12 minutes. The mass spectrometer was operated in the positive ion mode. The five metabolites and IS were monitored with the following transitions and collision energy (CE) in a multiple reaction monitoring (MRM) mode: acetaminophen (151.9 > 110.2, CE: 20 eV), 4-hydroxy-tolbutamide (287.3 > 188.3, CE: 15 eV), 5-hydroxy-omerprazole (362.3 > 213.9, CE: 20 eV), dextrorphan (258.0 > 199.2, CE: 40 eV), 1’-hydroxy-midazolam (342.5 > 203.2, CE: 40 eV), and d3-hydroxy-omerprazole (IS, 365.3 > 213.9, CE: 20 eV).

## Statistical Analysis

The absolute CYP2D6 protein quantity in the 287 HLS9 sample was used as the phenotype for the GWAS. The statistically significant GWAS *P* value threshold was set at 5.0 × 10^−7^ to account for multiple comparisons. Conditional analysis was executed using Plink (version 1.9) to identify the SNPs exhibiting independent effect from each other and the established CYP2D6 functional star alleles with an MAF equal to or greater than 0.01, which include CYP2D6 *3 (rs35742686), *4 (rs3892097), *6 (rs5030655), *9 (rs5030656), *10 (rs1065852), and *41 (rs28371725) ([Yang et al., 2010](#B47); [Ray et al., 2019](#B33); [Pratt et al., 2021](#B32)). The association between specific SNP genotypes and CYP2D6 activities was assessed by analysis of variance (ANOVA) using GraphPad Prism version 9.5.1 (GraphPad Software, San Diego, CA). A *P* value adjusted for multiple tests <0.05 was considered statistically significant.

## Results

### GWAS Identifies Variants Associated with CYP2D6 Protein Levels in Human Livers

The absolute CYP2D6 expressions in 287 individual human liver samples were determined as 0.1033 ± 0.0696 *μ*g/(mg HLS9 *proteins*), with a coefficient of variation (CV) of 67.30% ([Fig. 1A](#F1)). Employing a GWAS *P* value threshold of 5.0 × 10^−7^, a total of 44 variants exhibited significant association with CYP2D6 protein expressions in the liver samples ([Fig. 2](#F2)). Among these variants, seven were located on chromosome 1 and five on chromosome 2, whereas the remaining 32 SNPs were located on the same chromosome (Chr22) as CYP2D6 gene ([Table 1](#T1)).

### Fig. 1.

![Fig. 1.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/27f9/10801631/fd9761734e1a/dmd.123.001548f1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=10801631_dmd.123.001548f1.jpg)

(A) The absolute CYP2D6 protein expressions in 287 human livers determined using DIA-APQ and the CYP2D6 activities on metabolizing dextromethorphan to dextrorphan in 100 human livers. (B) Correlation analysis illustrating the relationship between CYP2D6 protein expressions and activities across the 100 human liver samples.

### Fig. 2.

![Fig. 2.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/27f9/10801631/509abf6e66b0/dmd.123.001548f2.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=10801631_dmd.123.001548f2.jpg)

Manhattan plot of the GWAS results for CYP2D6 protein expression in 287 individual human liver microsomes. On the y-axis, unadjusted P values are represented, whereas the x-axis corresponds to the genomic coordinates of genotyped and imputed SNPs. The significance threshold is indicated by a red line at an unadjusted P value of 5.0 × −7.

### TABLE 1.

| dbSNP | CHR | POS (GRCh37) | Distance to CYP2D6 (kb) −Upstream, +Downstream | UNADJ | BONF | FDR |
| --- | --- | --- | --- | --- | --- | --- |
| rs80274432 | 1 | 20831301 |   | 5.17E-08 | 0.006817 | 0.001136 |
| rs12566937 | 1 | 20832674 |   | 5.17E-08 | 0.006817 | 0.001136 |
| rs7544348 | 1 | 20851834 |   | 8.07E-09 | 0.001065 | 0.001021 |
| rs76440167 | 1 | 20883582 |   | 1.74E-08 | 0.002292 | 0.001021 |
| rs12569006 | 1 | 20997748 |   | 3.10E-08 | 0.004085 | 0.001021 |
| rs35753119 | 1 | 22834927 |   | 2.51E-07 | 0.03314 | 0.004142 |
| rs74877287 | 1 | 22846434 |   | 2.51E-07 | 0.03314 | 0.004142 |
| rs1550430 | 2 | 42227952 |   | 3.20E-07 | 0.04765 | 0.00953 |
| rs12474678 | 2 | 42232512 |   | 3.20E-07 | 0.04765 | 0.00953 |
| rs729559 | 2 | 42272984 |   | 1.03E-07 | 0.0153 | 0.005101 |
| rs729558 | 2 | 42273091 |   | 7.98E-08 | 0.01189 | 0.005101 |
| rs4073010 | 2 | 216745391 |   | 7.04E-08 | 0.0105 | 0.005101 |
| rs133299 | 22 | 42388841 | 137.97 | 9.11E-15 | 1.73E-10 | 1.33E-11 |
| rs133302 | 22 | 42391176 | 135.64 | 4.38E-11 | 8.30E-07 | 3.46E-08 |
| rs1062753 | 22 | 42392811 | 134.00 | 2.53E-15 | 4.79E-11 | 5.28E-12 |
| rs8138080 | 22 | 42396371 | 130.44 | 2.53E-15 | 4.79E-11 | 5.28E-12 |
| rs9623489 | 22 | 42397026 | 129.79 | 2.53E-15 | 4.79E-11 | 5.28E-12 |
| rs11090066 | 22 | 42398195 | 128.62 | 2.53E-15 | 4.79E-11 | 5.28E-12 |
| rs9623490 | 22 | 42398266 | 128.55 | 2.53E-15 | 4.79E-11 | 5.28E-12 |
| rs9611700 | 22 | 42401514 | 125.30 | 2.53E-15 | 4.79E-11 | 5.28E-12 |
| rs2413663 | 22 | 42402399 | 124.41 | 2.53E-15 | 4.79E-11 | 5.28E-12 |
| rs9620007 | 22 | 42405657 | 121.16 | 1.09E-14 | 2.06E-10 | 1.38E-11 |
| rs9611711 | 22 | 42410695 | 116.12 | 1.09E-14 | 2.06E-10 | 1.38E-11 |
| rs133331 | 22 | 42411603 | 115.21 | 6.07E-15 | 1.15E-10 | 9.57E-12 |
| rs133332 | 22 | 42411664 | 115.15 | 6.18E-12 | 1.17E-07 | 5.32E-09 |
| rs133333 | 22 | 42412365 | 114.45 | 6.18E-12 | 1.17E-07 | 5.32E-09 |
| rs5758550 | 22 | 42412711 | 114.10 | 6.18E-12 | 1.17E-07 | 5.32E-09 |
| rs9607868 | 22 | 42414181 | 112.63 | 1.25E-14 | 2.37E-10 | 1.48E-11 |
| rs4822082 | 22 | 42414198 | 112.61 | 6.07E-15 | 1.15E-10 | 9.57E-12 |
| rs2301521 | 22 | 42423110 | 103.70 | 3.01E-07 | 0.005698 | 0.0001965 |
| rs1807493 | 22 | 42473860 | 52.95 | 3.43E-16 | 6.50E-12 | 5.28E-12 |
| rs1080985 | 22 | 42528382 | −1.57 | 7.65E-16 | 1.45E-11 | 5.28E-12 |
| rs768013142 | 22 | 42544551 | −17.74 | 1.49E-10 | 2.83E-06 | 1.09E-07 |
| rs742086 | 22 | 42552253 | −25.44 | 1.46E-11 | 2.77E-07 | 1.21E-08 |
| rs6519313 | 22 | 42552875 | −26.06 | 2.79E-15 | 5.28E-11 | 5.28E-12 |
| rs4453786 | 22 | 42563308 | −36.50 | 1.36E-10 | 2.58E-06 | 1.03E-07 |
| rs75993207 | 22 | 42578609 | −51.80 | 3.17E-07 | 0.00601 | 0.0002003 |
| rs932376 | 22 | 42614362 | −87.55 | 1.52E-13 | 2.88E-09 | 1.69E-10 |
| rs56906457 | 22 | 42628335 | −101.52 | 6.45E-10 | 1.22E-05 | 4.52E-07 |
| rs775324395 | 22 | 42651536 | −124.72 | 3.50E-07 | 0.006629 | 0.0002139 |
| rs134874 | 22 | 42661144 | −134.33 | 4.19E-09 | 7.93E-05 | 2.83E-06 |
| rs134883 | 22 | 42671487 | −144.68 | 5.99E-12 | 1.14E-07 | 5.32E-09 |
| rs134904 | 22 | 42685537 | −158.73 | 7.65E-16 | 1.45E-11 | 5.28E-12 |
| rs9611766 | 22 | 42688639 | −161.83 | 3.73E-13 | 7.07E-09 | 3.93E-10 |

Table 1 Caption: rsID numbers, chromosomal location, distance to CYP2D6 gene, unadjusted P value, Bonferroni-corrected P value, and false discovery rate (FDR) of the 44 SNPs that passed the GWAS P value threshold of 5.0 × 10−7

### The Impact of Established CYP2D6 Star Alleles/Metabolizer Phenotypes on CYP2D6 Protein Expressions and Activities in Human Livers

We assigned CYP2D6 Tier1 star alleles to the 287 human livers. These included *1, *2, *3, *4, *6, *9, *10, *17, *29, and *41, determined based on their respective signature SNPs ([Table 2](#T2)). The MAF of CYP2D6 star alleles in our liver samples were comparable to the reference MAF in Caucasian on PharmGKB ([Table 2](#T2)). By summing the activity values of each allele in an individual, we calculated the activity scores (AS) in the 287 human livers. The AS of 0, 0.25, 0.5, 1, 1.25, 1.5, and 2 represented 3%, 12%, 1%, 21%, 2%, 11%, and 50% of the liver samples, respectively. We subsequently categorized the livers into four metabolizer phenotypes based on their AS: ultrarapid metabolizers (UM, AS >2.25), normal metabolizers (NM, AS 1.25–2.25), intermediate metabolizers (IM, AS 0.25–1), and poor metabolizers (PM, AS 0). This resulted in the classification of 182 samples as UM/NM (63%), 97 as IM (34%), and eight as PM (3%). As anticipated, the CYP2D6 phenotypes were consistent with both protein expressions and activities ([Fig. 3](#F3), *P* < 0.0001), demonstrating a significant effect of CYP2D6 genotypes on its phenotypes.

### TABLE 2.

| Star Allele | Activity Value | MAF | Ref MAF | Signature SNPs |
| --- | --- | --- | --- | --- |
| *1 | 1 | 0.35 | 0.29 |   |
| *2 | 1 | 0.36 | 0.19 | rs16947, rs1135840 |
| *3 | 0 | 0.01 | 0.02 | rs35742686 |
| *4 | 0 | 0.17 | 0.18 | rs3892097 |
| *6 | 0 | 0.01 | 0.01 | rs5030655 |
| *9 | 0.5 | 0.03 | 0.03 | rs5030656 |
| *10 | 0.25 | 0.09 | 0.02 | rs1065852, rs1135840 |
| *17 | 0.5 | 0.01 | 0.00 | rs28371706, rs16947, rs1135840 |
| *29 | 0.5 | 0.00 | 0.00 | rs59421388, rs61736512, rs1058164, rs16947, rs1135840 |
| *41 | 0.5 | 0.04 | 0.09 | rs28371725, rs16947, rs1135840 |

Table 2 Caption: Signature SNPs for assigning CYP2D6 star alleles and corresponding CYP2D6 activity values A comparison of minor allele frequencies (MAFs) of CYP2D6 star alleles in 287 human liver samples with the reference MAF in Caucasian as reported on PharmGKB

### Fig. 3.

![Fig. 3.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/27f9/10801631/705776a5bb59/dmd.123.001548f3.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=10801631_dmd.123.001548f3.jpg)

CYP2D6 expressions (n = 287) and activities (n = 100) across the phenotype groups of ultrarapid/normal metabolizer (UM/NM, AS ≥ 1.25), intermediate metabolizer (IM, AS = 0.25–1), and poor metabolizer (PM, AS = 0).

### Conditional Analysis to Identify Novel and Independent GWAS SNPs

To control the effect of known functional CYP2D6 genetic variants, we performed a conditional analysis by controlling an SNP set composed of established functional CYP2D6 star alleles with MAF of 1% or higher. The control SNP set included CYP2D6 *3 (rs35742686), *4 (rs3892097), *6 (rs5030655), *9 (rs5030656), *10 (rs1065852), and *41 (rs28371725). Through a five-step conditional analysis, we demonstrated that 39 out of the 44 significant SNPs were in high linkage disequilibrium (LD) with the control SNPs and that the remaining five SNPs, including rs1807493 (Chr22), rs1062753 (Chr22), rs4073010 (Chr2), rs729559 (Chr2), and rs80274432 (Chr1), showed independent effect on CYP2D6 expression with a Bonferroni-corrected *P* value less than 0.05 ([Supplemental Table 2](http://dmd.aspetjournals.org/lookup/suppl/doi:10.1124/dmd.123.001548/-/DC1)). To visualize the influence of these SNPs on CYP2D6 protein expressions, we conducted a comparative analysis across different genotypes in the 287 human liver samples. Specifically, the CYP2D6 protein expressions in the rs1062753 carriers displayed a declining trend with an increasing number of minor allele A copies. In contrast, the minor alleles of the remaining four SNPs—rs1807493, rs4073010, rs729559 and rs80274432—were associated with the increases of the CYP2D6 expression levels ([Fig. 4](#F4)).

### Fig. 4.

![Fig. 4.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/27f9/10801631/d843bab592c9/dmd.123.001548f4.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=10801631_dmd.123.001548f4.jpg)

The CYP2D6 protein expressions across genotypes of the five SNPs rs1807493, rs1062753, rs4073010, rs729559, and rs80274432 in 287 liver samples.

### Validation of Independent SNPs through the Association Analysis with CYP2D6 Activities

The CYP2D6 activities on the biotransformation of dextromethorphan to dextrorphan in 100 human liver samples were determined as 0.1535 ± 0.1724 pmol/min/mg protein ([Fig. 1A](#F1)). A significant association was observed between the CYP2D6 activities and the CYP2D6 protein expressions in the human livers (R^2^ = 0.4009, *P* < 0.0001; [Fig. 1B](#F1)). Importantly, four out of the five independent GWAS-SNPs—rs1807493 (*P* = 0.0011), rs4073010 (*P* = 0.0020), rs1062753 (*P* = 0.0001), and rs729559 (*P* = 0.0001) —displayed significant association with CYP2D6 activities, which was consistent with their impact on CYP2D6 protein expressions. The number of minor alleles of rs1062753 was associated with a decrease in CYP2D6 activities, whereas the other three SNPs exhibited a positive association with the CYP2D6 activities ([Fig. 5](#F5)). Inconsistent with the CYP2D6 protein results, rs80274432 (*P* = 0.8381), located on chromosome 1, did not show a significant association with CYP2D6 activities ([Fig. 5](#F5)).

### Fig. 5.

![Fig. 5.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/27f9/10801631/a807df370959/dmd.123.001548f5.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=10801631_dmd.123.001548f5.jpg)

CYP2D6 activities in 100 human liver samples, categorized by the genotypes of five SNPs, namely rs1807493, rs1062753, rs4073010, rs729559, and rs80274432. ANOVA test was applied to analyze the impact of these variants on CYP2D6 activities. P value < 0.05 was considered statistically significant.

### Impact of the Four Novel SNPs on CYP2D6 Protein and Activity Levels in Livers Classified into the Same CYP2D6 Phenotype Categories

The genotypes of four SNPs—rs1807493 (positive association, *P* < 0.0001), rs4073010 (positive association, *P* = 0.0002), rs729559 (positive association, *P* = 0.0010), and rs1062753 (negative association, *P* < 0.0001) —significantly contributed to explaining the variability in CYP2D6 protein expressions ([Fig. 6A](#F6)) in 182 human livers classified as CYP2D6 normal metabolizers (NM), as determined based on the established CYP2D6 star alleles and assigned activity scores. Notably, these SNPs exhibited consistent effect on CYP2D6 activities within the CYP2D6 NM group ([Fig. 6B](#F6)), with rs1807493 (*P* = 0.0451), rs4073010 (*P* = 0.0379), rs729559 (*P* = 0.0010), and rs1062753 (*P* = 0.0001).

### Fig. 6.

![Fig. 6.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/27f9/10801631/1589924d4f18/dmd.123.001548f6.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=10801631_dmd.123.001548f6.jpg)

Absolute protein quantities (A) and activities (B) of CYP2D6 in individual human livers across various genotypes for four specific SNPs—rs1807493, rs4073010, rs729559, and rs1062753—within individuals classified as CYP2D6 normal metabolizers based on the established CYP2D6 star alleles. ANOVA test was applied to analyze the impact of these variants on CYP2D6 protein expressions and activities. P value < 0.05 was considered statistically significant.

In addition to the normal metabolizers, we also assessed the contribution of these SNPs in predicting the CYP2D6 protein and activity levels in 97 livers within the intermediate metabolizer group (IM). Due to the relatively lower MAF of rs4073010, no carriers of rs4073010 variant were identified in this group. Thus, we analyzed the effect of the remaining three SNPs in the IM group. Specifically, rs1807493 (*P* = 0.0002), rs729559 (*P* = 0.0524), and rs1062753 (*P* = 0.0289) showed a significant impact on CYP2D6 protein expressions within the IM group ([Fig. 7A](#F7)). Similarly, the SNPs rs1807493 (*P* = 0.1250), rs729559 (*P* = 0.0133), and rs1062753 (*P* = 0.0112) exhibited a consistent effect on CYP2D6 activities in a manner consistent with their impacts on proteins ([Fig. 7B](#F7)).

### Fig. 7.

![Fig. 7.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/27f9/10801631/54ad1e8608e8/dmd.123.001548f7.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=10801631_dmd.123.001548f7.jpg)

Absolute protein quantities (A) and activities (B) of CYP2D6 in individual human livers across various genotypes for three specific SNPs—rs1807493, rs729559, and rs1062753—within individuals classified as CYP2D6 intermediate metabolizers as determined by the established CYP2D6 star alleles. Two-tailed Mann-Whitney test and ANOVA test were applied to analyze the impact of these variants on CYP2D6 protein expressions and activities. P value < 0.05 was considered statistically significant.

## Discussion

CYP2D6 is recognized as one of the very important pharmacogenes (VIPs) in the realm of pharmacogenomics implementation, attributed to its extensive polymorphism and its crucial role in metabolizing up to a quarter of clinically used medications ([Nofziger et al., 2020](#B27); [Kim et al., 2021](#B19)). As of today, a count of 75 US Food and Drug Administration (FDA)-approved drugs have incorporated CYP2D6 genetic biomarker information into their drug labels ([Kim et al., 2021](#B19)) ([https://www.fda.gov/drugs/science-and-research-drugs/table-pharmacogenomic-biomarkers-drug-labeling](https://www.fda.gov/drugs/science-and-research-drugs/table-pharmacogenomic-biomarkers-drug-labeling)). Six Clinical Pharmacogenetics Implementation Consortium (CPIC) guidelines involving CYP2D6 pharmacogenetics have been published, implying its significant clinical relevance ([Bell et al., 2017](#B1); [Hicks et al., 2017](#B16); [Goetz et al., 2018](#B12); [Brown et al., 2019](#B4); [Crews et al., 2021](#B7); [Bousman et al., 2023](#B3)). For example, with the prodrug codeine, CYP2D6 poor metabolizers (AS = 0), like those with CYP2D6*3/*4 and *4/*4 genotypes, produce minimal active metabolite (morphine), resulting in insufficient pain relief. On the other end of the spectrum, CYP2D6 ultrarapid metabolizers (AS >2.25), with genotypes like CYP2D6*1/*1x*N*, *1/*2x*N*, produce excessive morphine, elevating the risk of toxicity. Compared with the standard codeine therapy, CYP2D6 pharmacogenetics-guided therapy has demonstrated a marked improvement in the safety and efficacy of codeine treatment ([Crews et al., 2021](#B7)). Atomoxetine is another substrate drug of CYP2D6, which is commonly used for attention deficit hyperactivity disorder (ADHD). CYP2D6 poor metabolizers (AS = 0) have significantly decreased metabolism of atomoxetine, leading to higher blood concentrations and potential adverse effects. Conversely, ultrarapid metabolizers CYP2D6 (AS >2) may not achieve desired therapeutic levels ([Brown et al., 2019](#B4)). Despite the well established implementation of the functional CYP2D6 star alleles as clinical biomarkers to predict CYP2D6-mediated drug metabolism, a considerable degree of interindividual variability in CYP2D6 expression and activity remains unexplained ([Gaedigk et al., 2018](#B10)), presenting ongoing challenges. For instance, substantial variations persist among the individuals who do not carry any known functional CYP2D6 star alleles, and significant differences in response to CYP2D6 substrate medications are observed even within the same phenotype groups ([Solomon et al., 2019](#B39); [Frederiksen et al., 2023](#B9)). Hence, a growing body of literature is focused on identifying CYP2D6 eQTLs or clinical aQTLs. In these studies, the CYP2D6 mRNA levels have traditionally served as the primary phenotype for eQTL analysis ([Pan et al., 2017b](#B31)). Recognizing that the absolute CYP2D6 protein quantities offer a more accurate indicator of catalytic function ([Shakeel et al., 2018](#B35)), our investigation employed CYP2D6 absolute protein quantities in human livers as a direct phenotype for GWAS. Furthermore, we validated the genetic impact on hepatic CYP2D6 activities, bridging a crucial gap between CYP2D6 expression and functional consequences. In contrast to assessing the clinical outcomes related to CYP2D6 metabolism, measuring hepatic CYP2D6 activity through in vitro incubation offers a less confounded approach with reduced demands for large sample size. By assessing both the CYP2D6 protein levels and its activity, this dual screening strategy provided a robust foundation for understanding the relationships between genetic variants, protein expression, and enzymatic function within the context of CYP2D6 metabolism.

To validate our genotyping data, we compared the MAFs of CYP2D6 Tier1 star alleles and the MAF of their respective signature SNPs obtained from our 287 human liver samples with the reference MAFs for Caucasian. The frequencies of the signature SNPs of the CYP2D6 Tier1 star alleles obtained from our study population generally aligned with the reference MAFs ([Supplemental Table 1](http://dmd.aspetjournals.org/lookup/suppl/doi:10.1124/dmd.123.001548/-/DC1)). The MAFs obtained from our data were in close agreement with reference values for most of the star alleles such as CYP2D6*1, CYP2D6*3, CYP2D6*4, CYP2D6*6, CYP2D6*9, CYP2D6*17, and CYP2D6*29. However, we observed slight variations in the MAF of CYP2D6*2 (0.36 vs. 0.19), CYP2D6*10 (0.09 vs. 0.02), and CYP2D6*41 (0.04 vs. 0.09) ([Table 2](#T2)). It is important to note that although the majority (60%) of our liver samples were identified as Caucasians, 10% were identified as Black, 1% were categorized as ‘other,’ and the ethnicity of 29% of our samples remained unknown. Thus, the disparities in MAF between our cohort and the Caucasian reference may be attributed to the differences in ethnicity and the relatively smaller sample size in our study compared with the reference dataset. Another factor contributing to the higher incidence of CYP2D6*2 and lower incidence of CYP2D6*41 could be the presence of two shared SNPs (*rs16947* and *rs1135840*) in both alleles, posing challenges in distinguishing between them in some cases ([Hicks et al., 2014](#B17)). We further assessed CYP2D6 protein and activity levels across various CYP2D6 phenotype metabolizers. As expected, CYP2D6 protein expression and activity levels exhibited a gradient from high to low, corresponding to UM/NM, IM to PM, supporting the reliability of our genotyping data and the measurements of CYP2D6 protein and activity levels ([Fig. 3](#F3)).

In the GWAS analysis, we identified a total of 44 SNPs displaying a significant association with CYP2D6 protein expression levels using a *P* value threshold of 5.0 × 10^−7^. Among the 32 GWAS-SNPs located on chromosome 22, several exhibited genetic effects that align well with prior studies ([Yang et al., 2010](#B47); [Wang et al., 2014](#B45), [2015](#B44); [Pan et al., 2017b](#B31)). For example, five highly linked SNPs—rs1080985 (*P* = 6.18 × 10^−12^), rs133333 (*P* = 6.18 × 10^−12^), rs5758550 (*P* = 6.18 × 10^−12^), rs4822082 (*P* = 6.07 × 10^−15^), and rs8138080 (*P* = 2.53×10^−15^)—demonstrated significant associations with absolute CYP2D6 protein expressions in our cohort ([Table 1](#T1)). These findings agree with previous reports that documented these SNPs as being associated with increased CYP2D6 transcription and substrate drug metabolism ([Zanger et al., 2001](#B49); [Dorado et al., 2009](#B8); [Yang et al., 2010](#B47); [Llerena et al., 2013](#B25); [Wang et al., 2014](#B45), [2015](#B44); [Pan et al., 2017b](#B31)). Among these SNPs, rs5758550, located approximately 115 kb downstream of the CYP2D6 gene within an enhancer region, was shown to regulate enhancer activity in a reporter gene assay and was validated in HepG2 cells via clustered regularly interspaced short palindromic repeats (CRISPR)-mediated genome editing ([Wang et al., 2015](#B44)). Our data, in addition to transcriptional evidence, provided compelling protein-level support for the regulatory impact of rs5758550 genetic polymorphisms on CYP2D6 expression levels ([Wang et al., 2014](#B45), [2015](#B44); [Ray et al., 2019](#B33)).

Given the notable effect of the established functional CYP2D6 star alleles, we conducted a conditional analysis to uncover novel and independent protein quantitative trait loci (pQTLs). This conditional analysis employed the signature SNPs of common functional CYP2D6 Tier1 star alleles (MAF ≥ 1%) as a control SNP set. Through this analysis, we identified two cis-pQTLs, rs1807493 (*P* = 3.43 × 10^−16^) and rs1062753 (2.53 × 10^−15^), associated with increased and decreased CYP2D6 protein expressions, respectively ([Fig. 4](#F4); [Supplemental Table 2](http://dmd.aspetjournals.org/lookup/suppl/doi:10.1124/dmd.123.001548/-/DC1)). Importantly, these effects were independent of the common functional CYP2D6 Tier1 star alleles. Moreover, we confirmed the influence of these two SNPs on CYP2D6 activity ([Fig. 5](#F5)). Specifically, rs1807493, located within the SMDT1 and PHETA2 genes, which is approximately 53 kb downstream of the CYP2D6 gene, exhibited high LD with two other SNPs, rs5758550 (D’ = 0.956, R^2^ = 0.885) and −1584C>G ([Dorado et al., 2009](#B8); [LLerena et al., 2013](#B25); [Pan et al., 2017b](#B31)) (rs1080985, D’ = 0.962, R^2^ = 0.915). The two SNPs rs5758550 and rs1080985 have been previously associated with elevated expression levels and increased CYP2D6 substrate metabolism by multiple independent research groups ([Zanger et al., 2001](#B49); [Dorado et al., 2009](#B8); [Llerena et al., 2013](#B25); [Wang et al., 2014](#B45), [2015](#B44); [Ray et al., 2019](#B33)). Notably, our GWAS data revealed a stronger association between rs1807493 (*P* = 3.43 × 10^−16^) and CYP2D6 protein expression compared with rs5758550 (*P* = 6.18 × 10^−12^) and rs1080985 (*P* = 6.18 × 10^−12^). The other independent cis-pQTL, rs1062753, was located on SEPTIN gene, approximately 134 kb downstream from the CYP2D6 gene. The association between rs1062753 and CYP2D6 expression or activity has not been reported before. However, a significant association between CYP2D6 transcription and the genotypes of rs8138080, an SNP in high LD with rs1062753, has been documented ([Yang et al., 2010](#B47); [Wang et al., 2014](#B45), [2015](#B44); [Pan et al., 2017b](#B31); [Hennig et al., 2019](#B15)). In this study, rs1807493 and rs1062753 were revealed as the CYP2D6 pQTLs/aQTLs for the first time. Further investigations are warranted to elucidate the underlying mechanisms linking these two variants to CYP2D6 protein expression and activity.

In addition to the cis-pQTLs/aQTLs, the conditional analysis also identified three trans-variants rs4073010 (Chr2), rs729559 (Chr2) and rs80274432 (Chr1) that showed significant associations with CYP2D6 expressions. One highlight of our research lies in its capacity to validate GWAS-pQTLs through CYP2D6 activity assessment. By evaluating the influence of these trans-pQTLs on CYP2D6 activity, we excluded rs80274432, located on MUL1 gene in chromosome 1, due to its lack of a significant effect on activity ([Fig. 5](#F5)). The remaining two trans-pQTLs, rs4073010 and rs729559, situated in noncoding regions on chromosome 2, may encompass intergenic regions or noncoding RNAs. Importantly, no prior publications involving these two SNPs have been published as of the present date. Our research has thus uncovered these CYP2D6 trans-eQTLs/aQTLs, suggesting their potential to influence the CYP2D6 expression or function via complex interactions involving regulatory elements, chromatin conformation, or long-range regulation ([Churchill, 2006](#B6); [Liu et al., 2019](#B24); [Morgan et al., 2020](#B26)). Nevertheless, a comprehensive understanding of the precise mechanisms and interactions requires further investigation.

Among these four independent CYP2D6 cis- and trans-pQTLs/aQTLs, three of them—rs4073010 on Chr2, rs729559 on Chr2, and rs1807493 on Chr22—demonstrated a positive association between the number of minor alleles and the CYP2D6 protein expressions and activities in human livers. Notably, when these three variants, rs4073010, rs729559, and rs1807493, were considered together, a consistent increase in CYP2D6 protein expression ([Supplemental Fig. 1A](http://dmd.aspetjournals.org/lookup/suppl/doi:10.1124/dmd.123.001548/-/DC1)) and activity ([Supplemental Fig. 2A](http://dmd.aspetjournals.org/lookup/suppl/doi:10.1124/dmd.123.001548/-/DC1)) was observed, as the number of mutations at these SNPs ranges from zero to four. However, we also noted that there was no clear trend observed when the number of mutations increased from four to five. This lack of consistency in the trend may be attributed to the small sample size in these two groups (*n* = 2), which may not be sufficient to accurately reflect the change ([Supplemental Figs. 1A](http://dmd.aspetjournals.org/lookup/suppl/doi:10.1124/dmd.123.001548/-/DC1) and 2A). When we examined the combined effect of any two of these three CYP2D6 pQTLs/aQTLs, a consistent additive effect was also observed ([Supplemental Figs. 1](http://dmd.aspetjournals.org/lookup/suppl/doi:10.1124/dmd.123.001548/-/DC1) and 2). In particular, for rs4073010 (Chr2) and rs729559 (Chr2), the CYP2D6 protein expression and activity followed an overall increasing trend with an increase in mutations at these two SNPs. Similar to the combined effect observed when considering all three SNPs, the increasing trend became less clear when the sample sizes were too small (*n* = 1 or *n* = 2) ([Supplemental Figs. 1B](http://dmd.aspetjournals.org/lookup/suppl/doi:10.1124/dmd.123.001548/-/DC1) and 2B). Notably, when examining the combined effect of rs729559 (Chr2) and rs1807493 (Chr22) and that of rs4073010 (Chr2) and rs1807493 (Chr22), the CYP2D6 protein levels ([Supplemental Fig. 1, C and D](http://dmd.aspetjournals.org/lookup/suppl/doi:10.1124/dmd.123.001548/-/DC1)) and activities ([Supplemental Fig. 2, C and D](http://dmd.aspetjournals.org/lookup/suppl/doi:10.1124/dmd.123.001548/-/DC1)) consistently showed an increasing trend along with an increase in the number of mutations at the respective two SNPs. Furthermore, we assessed whether the concurrent SNPs might exhibit a synergistic effect by comparing the CYP2D6 protein expressions in the livers carrying one minor allele of each of two SNPs to those homozygotes for only one of the two SNPs. It was found that the CYP2D6 expression in the heterozygous samples with both concurrent SNPs was not significantly higher than their respective homozygous counterparts ([Supplemental Fig. 3](http://dmd.aspetjournals.org/lookup/suppl/doi:10.1124/dmd.123.001548/-/DC1)), suggesting an additive, rather than synergistic, effect for the examined SNPs. However, it is important to note that this study is limited to association-based analysis, and further mechanistic studies are required to confirm the absence of a synergistic effect among these SNPs.

It is essential to emphasize the independent effect of the four newly identified CYP2D6 pQTLs/aQTLs—rs1807493, rs4073010, rs729559, and rs1062753—in comparison with the established CYP2D6 functional polymorphisms. When we examined CYP2D6 protein and activity levels across various genotypes of these pQTLs/aQTLs within the same phenotype groups categorized by the genotypes of the established CYP2D6 star alleles, significant associations were observed between their genotypes and the CYP2D6 protein expressions ([Fig. 6A](#F6), NM; [Fig. 7A](#F7), IM) as well as activities ([Fig. 6B](#F6), NM and [Fig. 7B](#F7), IM). The association trend appeared less clear in the CYP2D6 IM group compared with the NM group, which is likely due to the very limited sample size for carriers of some SNPs. For instance, there were no carriers for rs4073010, no rs1807493 homozygote, and only one rs729559 carrier among the 97 CYP2D6 IM livers ([Fig. 7](#F7)). These four novel CYP2D6 pQTLs/aQTLs appear to significantly contribute to explaining additional variations in CYP2D6 expression and function among individuals, complementing the known functional CYP2D6 polymorphisms. Moreover, when we examined the combined effect of the three SNPs that exhibited a positive association with CYP2D6 protein and activity levels (rs4073010, rs729559, and rs1807493) in livers within the same metabolizer phenotype groups, we consistently observed an additive effect on both CYP2D6 protein and activity within specific CYP2D6 phenotype groups, including NM ([Supplemental Fig. 4,](http://dmd.aspetjournals.org/lookup/suppl/doi:10.1124/dmd.123.001548/-/DC1)[A–B](http://dmd.aspetjournals.org/lookup/suppl/doi:10.1124/dmd.123.001548/-/DC1)) and IM ([Supplemental Fig. 4,](http://dmd.aspetjournals.org/lookup/suppl/doi:10.1124/dmd.123.001548/-/DC1)[C–D](http://dmd.aspetjournals.org/lookup/suppl/doi:10.1124/dmd.123.001548/-/DC1)). These data further underscore that the impact of these novel CYP2D6 pQTLs/aQTLs is independent of the known functional CYP2D6 polymorphisms, enhancing our ability to predict variability in CYP2D6 expression/function beyond the current knowledge base.

Analyzing the MAFs of these four newly discovered CYP2D6 pQTLs/aQTLs in this study across diverse ethnic groups reveals significant variations in their prevalence. The ratios of the highest to lowest MAF across these groups were 4.61, 39.57, 8.23 and 3.51 for rs1807493, rs4073010, rs1062753, and rs729559, respectively. Consistent with the ethnicity composition of our liver samples, which were primarily Caucasian, the MAF of these SNPs in our cohort aligns most closely to the Caucasian ([Supplemental Table 3](http://dmd.aspetjournals.org/lookup/suppl/doi:10.1124/dmd.123.001548/-/DC1)). Among these four SNPs, rs4073010, with an MAF of 0.038 in our cohort, displays the most significant ethnic disparities. The SNP is most prevalent in the Asian population, where it has an MAF of 0.277, while being quite rare in the Caucasian population, with an MAF of 0.007. Given its high prevalence in the Asian population (MAF = 0.277) and its commonality in the Black African (MAF = 0.092) and Hispanic (MAF = 0.180) populations, rs4073010 could potentially serve as a significant CYP2D6 pQTL/aQTL in these populations ([Supplemental Table 3](http://dmd.aspetjournals.org/lookup/suppl/doi:10.1124/dmd.123.001548/-/DC1)). Similarly, rs729559 is most common in the Hispanic and Asian populations with MAFs higher than 0.400 while relatively less common in Caucasian and Black African. In contrast, rs1062753 is most common in the Hispanic population, with an MAF of 0.428, and it also exhibits prevalence in the Black African (MAF = 0.320) and Caucasian (MAF = 0.297) populations, while being less common in the Asian population (MAF = 0.052). Lastly, rs1807493 is more common in the Caucasian, Hispanic, and Asian populations, with MAFs of 0.212, 0.115, and 0.108, respectively. In Black African, although less frequent, it is still a common SNP with MAF of 0.046 ([Supplemental Table 3](http://dmd.aspetjournals.org/lookup/suppl/doi:10.1124/dmd.123.001548/-/DC1)). The disparities in the prevalence of these SNPs across ethnicities suggest their potential critical role in specific ethnic groups and warrant further investigation into their impact on CYP2D6 expression/function in respective prevalent populations.

Several limitations of this study have been recognized. Firstly, the sample size of this study, 287, is relatively small for a typical GWAS study. As a result, our study may have limited power to detect SNPs with low MAF. However, the absolute CYP2D6 protein levels in liver samples were used as a phenotype in this study, which are significantly less affected by confounding factors than the clinical therapeutic outcomes. Additionally, we use the CYP2D6 activity to validate the results, which greatly lowers the probability of having false positive results. Secondly, the race of our liver samples was primarily Caucasian. Some variants with high MAF in other races/ethnicities but not Caucasian could not be identified in our assay. Thirdly, the genotypes were obtained through Illumina MEGA chip, which is primarily designed for genotyping SNPs across the genome. Although it provided valuable information regarding SNPs, it is not suitable for determining the gene copy number variations. Thus, we were unable to detect CYP2D6*5 (deletion) or gene duplication of CYP2D6. As a result, we could not distinguish the ultrarapid metabolizers (UM) from the normal metabolizers (NM), and these two groups were combined as UM/NM in our analysis. The interactions between the copy number variants and the novel variants identified in this study could not be analyzed either. Lastly, besides genetic variants, nongenetic factors such as developmental stages ([Stevens et al., 2008](#B41)), pregnancy ([Koh et al., 2011](#B23), [2014a](#B21),b), and CYP2D6 inhibitors/inducers in the livers ([Pan and Jeong, 2015](#B29); [Pan et al., 2017a](#B30)) might have contributed to the variations in its activity, which could not be revealed by the current study.

In summary, our study performed a conditional GWAS, controlling for the effects of established functional CYP2D6 star alleles. This investigation aimed to identify CYP2D6 pQTLs in a cohort of 287 individual human liver samples. To enhance the robustness of our findings, we subsequently validated these pQTLs by examining their impact on CYP2D6 enzymatic activities using 100 liver samples. Our research contributed to the understanding of the genetic factors associated with both CYP2D6 expression (pQTL) and activity (aQTL). By employing both protein levels and enzymatic activity as phenotypes, the study provided invaluable insights into the relationship between genetic polymorphisms, CYP2D6 protein levels, and activities. In this study, four SNPs—rs1807493, rs4073010, rs1062753, and rs729559—emerged as novel pQTLs/aQTLs of CYP2D6. Importantly, these pQTLs/aQTLs are independent of the effect of established CYP2D6 star alleles. Thus, the findings have great potential to enhance our current predictive capabilities for CYP2D6-mediated drug disposition, which has traditionally relied on established CYP2D6 polymorphisms. Furthermore, these four SNPs exhibit an additive effect on CYP2D6 expression and activity while also demonstrating high prevalence in specific ethnic groups, warranting further clinical studies to validate the impact of these SNPs within their respective prevalent ethnic populations. This study is of immense clinical relevance, particularly in the context of CYP2D6-mediated drug metabolism. It has the potential to contribute to the advancement of personalized pharmacotherapy for a broad spectrum of medications such as atomoxetine for ADHD ([Brown et al., 2019](#B4)), opioids for pain management ([Crews et al., 2021](#B7)), ondansetron and tropisetron for antiemetic purposes ([Bell et al., 2017](#B1)), tamoxifen for breast cancer ([Goetz et al., 2018](#B12)), selective serotonin reuptake inhibitors (SSRIs) ([Bousman et al., 2023](#B3)) and tricyclic antidepressants ([Hicks et al., 2017](#B16)) for anxiety and depression.

## Data Availability

All of the necessary data for assessing this work are present in the paper and/or the [Supplemental Material](http://dmd.aspetjournals.org/lookup/suppl/doi:10.1124/dmd.123.001548/-/DC1). All proteomics data have been deposited to the ProteomeXchange Consortium under the dataset identifier PXD019169 ([He et al., 2020](#B14)). Any additional data related to this study may be requested from the authors.

## Abbreviations

## Authorship Contributions

*Participated in research design:* Zhu, Wang.

*Conducted experiments:* Shi, Wang.

*Performed data analysis:* Smith, He, Wang.

*Contributed new reagents or analytic tools:* Zhu.

*Wrote and contributed to the writing of the manuscript:* Smith, Zhu, Wang.

## Footnotes

## References

1. Bell GCCaudle KEWhirl-Carrillo MGordon RJHikino KProws CAGaedigk AAgundez JSadhasivam SKlein TE, et al. (2017) Clinical Pharmacogenetics Implementation Consortium (CPIC) guideline for CYP2D6 genotype and use of ondansetron and tropisetron. Clin Pharmacol Ther 102:213–218.  [DOI](https://doi.org/10.1002/cpt.598) | [PMC free article](/articles/PMC5479760/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28002639/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Clinical%20Pharmacogenetics%20Implementation%20Consortium%20(CPIC)%20guideline%20for%20CYP2D6%20genotype%20and%20use%20of%20ondansetron%20and%20tropisetron&volume=102&publication_year=2017&pages=213-218&pmid=28002639&doi=10.1002/cpt.598&)

2. Bertilsson L, Alm C, De Las Carreras C, Widen J, Edman G, Schalling D (1989) Debrisoquine hydroxylation polymorphism and personality. Lancet 1:555.  [DOI](https://doi.org/10.1016/s0140-6736(89)90094-9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2564084/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Debrisoquine%20hydroxylation%20polymorphism%20and%20personality&volume=1&publication_year=1989&pages=555&pmid=2564084&doi=10.1016/s0140-6736(89)90094-9&)

3. Bousman CAStevenson JMRamsey LBSangkuhl KHicks JKStrawn JRSingh ABRuaño GMueller DJTsermpini EE, et al. (2023) Clinical Pharmacogenetics Implementation Consortium (CPIC) guideline for CYP2D6, CYP2C19, CYP2B6, SLC6A4, and HTR2A genotypes and serotonin reuptake inhibitor antidepressants. Clin Pharmacol Ther 114:51–68.  [DOI](https://doi.org/10.1002/cpt.2903) | [PMC free article](/articles/PMC10564324/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/37032427/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Clinical%20Pharmacogenetics%20Implementation%20Consortium%20(CPIC)%20guideline%20for%20CYP2D6,%20CYP2C19,%20CYP2B6,%20SLC6A4,%20and%20HTR2A%20genotypes%20and%20serotonin%20reuptake%20inhibitor%20antidepressants&volume=114&publication_year=2023&pages=51-68&pmid=37032427&doi=10.1002/cpt.2903&)

4. Brown JTBishop JRSangkuhl KNurmi ELMueller DJDinh JCGaedigk AKlein TECaudle KEMcCracken JT, et al. (2019) Clinical Pharmacogenetics Implementation Consortium guideline for cytochrome P450 (CYP)2D6 genotype and atomoxetine therapy. Clin Pharmacol Ther 106:94–102.  [DOI](https://doi.org/10.1002/cpt.1409) | [PMC free article](/articles/PMC6612570/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30801677/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Clinical%20Pharmacogenetics%20Implementation%20Consortium%20guideline%20for%20cytochrome%20P450%20(CYP)2D6%20genotype%20and%20atomoxetine%20therapy&volume=106&publication_year=2019&pages=94-102&pmid=30801677&doi=10.1002/cpt.1409&)

5. Caudle KESangkuhl KWhirl-Carrillo MSwen JJHaidar CEKlein TEGammal RSRelling MVScott SAHertz DL, et al. (2020) Standardizing CYP2D6 genotype to phenotype translation: consensus recommendations from the Clinical Pharmacogenetics Implementation Consortium and Dutch Pharmacogenetics Working Group. Clin Transl Sci 13:116–124.  [DOI](https://doi.org/10.1111/cts.12692) | [PMC free article](/articles/PMC6951851/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31647186/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Transl%20Sci&title=Standardizing%20CYP2D6%20genotype%20to%20phenotype%20translation:%20consensus%20recommendations%20from%20the%20Clinical%20Pharmacogenetics%20Implementation%20Consortium%20and%20Dutch%20Pharmacogenetics%20Working%20Group&volume=13&publication_year=2020&pages=116-124&pmid=31647186&doi=10.1111/cts.12692&)

6. Churchill GA (2006) The genetics of gene expression. Mamm Genome 17:465.  [DOI](https://doi.org/10.1007/s00335-006-1100-9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16783627/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mamm%20Genome&title=The%20genetics%20of%20gene%20expression&volume=17&publication_year=2006&pages=465&pmid=16783627&doi=10.1007/s00335-006-1100-9&)

7. Crews KRMonte AAHuddart RCaudle KEKharasch EDGaedigk ADunnenberger HMLeeder JSCallaghan JTSamer CF, et al. (2021) Clinical Pharmacogenetics Implementation Consortium guideline for CYP2D6, OPRM1, and COMT genotypes and select opioid therapy. Clin Pharmacol Ther 110:888–896.  [DOI](https://doi.org/10.1002/cpt.2149) | [PMC free article](/articles/PMC8249478/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33387367/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Clinical%20Pharmacogenetics%20Implementation%20Consortium%20guideline%20for%20CYP2D6,%20OPRM1,%20and%20COMT%20genotypes%20and%20select%20opioid%20therapy&volume=110&publication_year=2021&pages=888-896&pmid=33387367&doi=10.1002/cpt.2149&)

8. Dorado P, Peñas-Lledó EM, de la Rubia A, Llerena A (2009) Relevance of CYP2D6 -1584C>G polymorphism for thioridazine:mesoridazine plasma concentration ratio in psychiatric patients. Pharmacogenomics 10:1083–1089.  [DOI](https://doi.org/10.2217/pgs.09.57) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19604081/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Relevance%20of%20CYP2D6%20-1584C>G%20polymorphism%20for%20thioridazine:mesoridazine%20plasma%20concentration%20ratio%20in%20psychiatric%20patients&volume=10&publication_year=2009&pages=1083-1089&pmid=19604081&doi=10.2217/pgs.09.57&)

9. Frederiksen T, Areberg J, Schmidt E, Stage TB, Brøsen K (2023) Does ethnicity impact CYP2D6 genotype-phenotype relationships? Clin Transl Sci 16:1012–1020.  [DOI](https://doi.org/10.1111/cts.13506) | [PMC free article](/articles/PMC10264946/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36869607/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Transl%20Sci&title=Does%20ethnicity%20impact%20CYP2D6%20genotype-phenotype%20relationships?&volume=16&publication_year=2023&pages=1012-1020&pmid=36869607&doi=10.1111/cts.13506&)

10. Gaedigk A, Dinh JC, Jeong H, Prasad B, Leeder JS (2018) Ten years’ experience with the CYP2D6 activity score: a perspective on future investigations to improve clinical predictions for precision therapeutics. J Pers Med 8:15.  [DOI](https://doi.org/10.3390/jpm8020015) | [PMC free article](/articles/PMC6023391/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29673183/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pers%20Med&title=Ten%20years%E2%80%99%20experience%20with%20the%20CYP2D6%20activity%20score:%20a%20perspective%20on%20future%20investigations%20to%20improve%20clinical%20predictions%20for%20precision%20therapeutics&volume=8&publication_year=2018&pages=15&pmid=29673183&doi=10.3390/jpm8020015&)

11. Gaedigk A, Simon SD, Pearce RE, Bradford LD, Kennedy MJ, Leeder JS (2008) The CYP2D6 activity score: translating genotype information into a qualitative measure of phenotype. Clin Pharmacol Ther 83:234–242.  [DOI](https://doi.org/10.1038/sj.clpt.6100406) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17971818/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=The%20CYP2D6%20activity%20score:%20translating%20genotype%20information%20into%20a%20qualitative%20measure%20of%20phenotype&volume=83&publication_year=2008&pages=234-242&pmid=17971818&doi=10.1038/sj.clpt.6100406&)

12. Goetz MPSangkuhl KGuchelaar HJSchwab MProvince MWhirl-Carrillo MSymmans WFMcLeod HLRatain MJZembutsu H, et al. (2018) Clinical Pharmacogenetics Implementation Consortium (CPIC) guideline for CYP2D6 and tamoxifen therapy. Clin Pharmacol Ther 103:770–777.  [DOI](https://doi.org/10.1002/cpt.1007) | [PMC free article](/articles/PMC5931215/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29385237/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Clinical%20Pharmacogenetics%20Implementation%20Consortium%20(CPIC)%20guideline%20for%20CYP2D6%20and%20tamoxifen%20therapy&volume=103&publication_year=2018&pages=770-777&pmid=29385237&doi=10.1002/cpt.1007&)

13. He B, Shi J, Wang X, Jiang H, Zhu HJ (2019) Label-free absolute protein quantification with data-independent acquisition. J Proteomics 200:51–59.  [DOI](https://doi.org/10.1016/j.jprot.2019.03.005) | [PMC free article](/articles/PMC6533198/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30880166/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Proteomics&title=Label-free%20absolute%20protein%20quantification%20with%20data-independent%20acquisition&volume=200&publication_year=2019&pages=51-59&pmid=30880166&doi=10.1016/j.jprot.2019.03.005&)

14. He B, Shi J, Wang X, Jiang H, Zhu HJ (2020) Genome-wide pQTL analysis of protein expression regulatory networks in the human liver. BMC Biol 18:97.  [DOI](https://doi.org/10.1186/s12915-020-00830-3) | [PMC free article](/articles/PMC7418398/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32778093/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Biol&title=Genome-wide%20pQTL%20analysis%20of%20protein%20expression%20regulatory%20networks%20in%20the%20human%20liver&volume=18&publication_year=2020&pages=97&pmid=32778093&doi=10.1186/s12915-020-00830-3&)

15. Hennig EE, Piątkowska M, Goryca K, Pośpiech E, Paziewska A, Karczmarski J, Kluska A, Brewczyńska E, Ostrowski J (2019) Non-CYP2D6 variants selected by a GWAS improve the prediction of impaired tamoxifen metabolism in patients with breast cancer. J Clin Med 8:1087.  [DOI](https://doi.org/10.3390/jcm8081087) | [PMC free article](/articles/PMC6722498/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31344832/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Med&title=Non-CYP2D6%20variants%20selected%20by%20a%20GWAS%20improve%20the%20prediction%20of%20impaired%20tamoxifen%20metabolism%20in%20patients%20with%20breast%20cancer&volume=8&publication_year=2019&pages=1087&pmid=31344832&doi=10.3390/jcm8081087&)

16. Hicks JKSangkuhl KSwen JJEllingrod VLMüller DJShimoda KBishop JRKharasch EDSkaar TCGaedigk A, et al. (2017) Clinical Pharmacogenetics Implementation Consortium guideline (CPIC) for CYP2D6 and CYP2C19 genotypes and dosing of tricyclic antidepressants: 2016 update. Clin Pharmacol Ther 102:37–44.  [DOI](https://doi.org/10.1002/cpt.597) | [PMC free article](/articles/PMC5478479/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27997040/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Clinical%20Pharmacogenetics%20Implementation%20Consortium%20guideline%20(CPIC)%20for%20CYP2D6%20and%20CYP2C19%20genotypes%20and%20dosing%20of%20tricyclic%20antidepressants:%202016%20update&volume=102&publication_year=2017&pages=37-44&pmid=27997040&doi=10.1002/cpt.597&)

17. Hicks JK, Swen JJ, Gaedigk A (2014) Challenges in CYP2D6 phenotype assignment from genotype data: a critical assessment and call for standardization. Curr Drug Metab 15:218–232.  [DOI](https://doi.org/10.2174/1389200215666140202215316) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24524666/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Drug%20Metab&title=Challenges%20in%20CYP2D6%20phenotype%20assignment%20from%20genotype%20data:%20a%20critical%20assessment%20and%20call%20for%20standardization&volume=15&publication_year=2014&pages=218-232&pmid=24524666&doi=10.2174/1389200215666140202215316&)

18. Hiroi T, Imaoka S, Funae Y (1998) Dopamine formation from tyramine by CYP2D6. Biochem Biophys Res Commun 249:838–843.  [DOI](https://doi.org/10.1006/bbrc.1998.9232) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9731223/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20Biophys%20Res%20Commun&title=Dopamine%20formation%20from%20tyramine%20by%20CYP2D6&volume=249&publication_year=1998&pages=838-843&pmid=9731223&doi=10.1006/bbrc.1998.9232&)

19. Kim JA, Ceccarelli R, Lu CY (2021) Pharmacogenomic biomarkers in US FDA-approved drug labels (2000-2020). J Pers Med 11:179.  [DOI](https://doi.org/10.3390/jpm11030179) | [PMC free article](/articles/PMC8000585/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33806453/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pers%20Med&title=Pharmacogenomic%20biomarkers%20in%20US%20FDA-approved%20drug%20labels%20(2000-2020)&volume=11&publication_year=2021&pages=179&pmid=33806453&doi=10.3390/jpm11030179&)

20. Kirchheiner J, Seeringer A, Godoy AL, Ohmle B, Maier C, Beschoner P, Sim EJ, Viviani R (2011) CYP2D6 in the brain: genotype effects on resting brain perfusion. Mol Psychiatry 16:237–, 333–341..  [DOI](https://doi.org/10.1038/mp.2010.42) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20368706/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Psychiatry&title=CYP2D6%20in%20the%20brain:%20genotype%20effects%20on%20resting%20brain%20perfusion&volume=16&publication_year=2011&pages=237-,%20333%E2%80%93341&pmid=20368706&doi=10.1038/mp.2010.42&)

21. Koh KH, Pan X, Shen HW, Arnold SL, Yu AM, Gonzalez FJ, Isoherranen N, Jeong H (2014a) Altered expression of small heterodimer partner governs cytochrome P450 (CYP) 2D6 induction during pregnancy in CYP2D6-humanized mice. J Biol Chem 289:3105–3113.  [DOI](https://doi.org/10.1074/jbc.M113.526798) | [PMC free article](/articles/PMC3916516/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24318876/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Altered%20expression%20of%20small%20heterodimer%20partner%20governs%20cytochrome%20P450%20(CYP)%202D6%20induction%20during%20pregnancy%20in%20CYP2D6-humanized%20mice&volume=289&publication_year=2014a&pages=3105-3113&pmid=24318876&doi=10.1074/jbc.M113.526798&)

22. Koh KH, Pan X, Zhang W, McLachlan A, Urrutia R, Jeong H (2014b) Krüppel-like factor 9 promotes hepatic cytochrome P450 2D6 expression during pregnancy in CYP2D6-humanized mice. Mol Pharmacol 86:727–735.  [DOI](https://doi.org/10.1124/mol.114.093666) | [PMC free article](/articles/PMC4244597/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25217496/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Pharmacol&title=Kr%C3%BCppel-like%20factor%209%20promotes%20hepatic%20cytochrome%20P450%202D6%20expression%20during%20pregnancy%20in%20CYP2D6-humanized%20mice&volume=86&publication_year=2014b&pages=727-735&pmid=25217496&doi=10.1124/mol.114.093666&)

23. Koh KH, Xie H, Yu AM, Jeong H (2011) Altered cytochrome P450 expression in mice during pregnancy. Drug Metab Dispos 39:165–169.  [DOI](https://doi.org/10.1124/dmd.110.035790) | [PMC free article](/articles/PMC3033691/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20971892/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Altered%20cytochrome%20P450%20expression%20in%20mice%20during%20pregnancy&volume=39&publication_year=2011&pages=165-169&pmid=20971892&doi=10.1124/dmd.110.035790&)

24. Liu X, Li YI, Pritchard JK (2019) Trans effects on gene expression can drive omnigenic inheritance. Cell 177:1022–1034.e6.  [DOI](https://doi.org/10.1016/j.cell.2019.04.014) | [PMC free article](/articles/PMC6553491/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31051098/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell&title=Trans%20effects%20on%20gene%20expression%20can%20drive%20omnigenic%20inheritance&volume=177&publication_year=2019&pages=1022-1034.e6&pmid=31051098&doi=10.1016/j.cell.2019.04.014&)

25. Llerena A, Dorado P, Ramírez R, Calzadilla LR, Peñas-Lledó E, Álvarez M, Naranjo MEG, González I, Pérez B; CEIBA Consortium of Ibero–American Network of Pharmacogenetics & Pharmacogenomics RIBEF (2013) CYP2D6 -1584C>G promoter polymorphism and debrisoquine ultrarapid hydroxylation in healthy volunteers. Pharmacogenomics 14:1973–1977.  [DOI](https://doi.org/10.2217/pgs.13.181) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24279852/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=CYP2D6%20-1584C>G%20promoter%20polymorphism%20and%20debrisoquine%20ultrarapid%20hydroxylation%20in%20healthy%20volunteers&volume=14&publication_year=2013&pages=1973-1977&pmid=24279852&doi=10.2217/pgs.13.181&)

26. Morgan MD, Patin E, Jagla B, Hasan M, Quintana-Murci L, Marioni JC (2020) Quantitative genetic analysis deciphers the impact of cis and trans regulation on cell-to-cell variability in protein expression levels. PLoS Genet 16:e1008686.  [DOI](https://doi.org/10.1371/journal.pgen.1008686) | [PMC free article](/articles/PMC7094872/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32168362/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet&title=Quantitative%20genetic%20analysis%20deciphers%20the%20impact%20of%20cis%20and%20trans%20regulation%20on%20cell-to-cell%20variability%20in%20protein%20expression%20levels&volume=16&publication_year=2020&pages=e1008686&pmid=32168362&doi=10.1371/journal.pgen.1008686&)

27. Nofziger CTurner AJSangkuhl KWhirl-Carrillo MAgúndez JAGBlack JLDunnenberger HMRuano GKennedy MAPhillips MS, et al. (2020) PharmVar GeneFocus: CYP2D6. Clin Pharmacol Ther 107:154–170.  [DOI](https://doi.org/10.1002/cpt.1643) | [PMC free article](/articles/PMC6925641/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31544239/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=PharmVar%20GeneFocus:%20CYP2D6&volume=107&publication_year=2020&pages=154-170&pmid=31544239&doi=10.1002/cpt.1643&)

28. Owen RP, Sangkuhl K, Klein TE, Altman RB (2009) Cytochrome P450 2D6. Pharmacogenet Genomics 19:559–562.  [DOI](https://doi.org/10.1097/FPC.0b013e32832e0e97) | [PMC free article](/articles/PMC4373606/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19512959/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Cytochrome%20P450%202D6&volume=19&publication_year=2009&pages=559-562&pmid=19512959&doi=10.1097/FPC.0b013e32832e0e97&)

29. Pan X, Jeong H (2015) Estrogen-induced cholestasis leads to repressed CYP2D6 expression in CYP2D6-humanized mice. Mol Pharmacol 88:106–112.  [DOI](https://doi.org/10.1124/mol.115.098822) | [PMC free article](/articles/PMC4468640/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25943116/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Pharmacol&title=Estrogen-induced%20cholestasis%20leads%20to%20repressed%20CYP2D6%20expression%20in%20CYP2D6-humanized%20mice&volume=88&publication_year=2015&pages=106-112&pmid=25943116&doi=10.1124/mol.115.098822&)

30. Pan X, Kent R, Won KJ, Jeong H (2017a) Cholic acid feeding leads to increased CYP2D6 expression in CYP2D6-humanized mice. Drug Metab Dispos 45:346–352.  [DOI](https://doi.org/10.1124/dmd.116.074013) | [PMC free article](/articles/PMC5363697/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28153841/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Cholic%20acid%20feeding%20leads%20to%20increased%20CYP2D6%20expression%20in%20CYP2D6-humanized%20mice&volume=45&publication_year=2017a&pages=346-352&pmid=28153841&doi=10.1124/dmd.116.074013&)

31. Pan X, Ning M, Jeong H (2017b) Transcriptional regulation of CYP2D6 expression. Drug Metab Dispos 45:42–48.  [DOI](https://doi.org/10.1124/dmd.116.072249) | [PMC free article](/articles/PMC5193069/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27698228/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Transcriptional%20regulation%20of%20CYP2D6%20expression&volume=45&publication_year=2017b&pages=42-48&pmid=27698228&doi=10.1124/dmd.116.072249&)

32. Pratt VMCavallari LHDel Tredici ALGaedigk AHachad HJi YKalman LVLy RCMoyer AMScott SA, et al. (2021) Recommendations for Clinical CYP2D6 genotyping allele selection: a joint consensus recommendation of the Association for Molecular Pathology, College of American Pathologists, Dutch Pharmacogenetics Working Group of the Royal Dutch Pharmacists Association, and the European Society for Pharmacogenomics and Personalized Therapy. J Mol Diagn 23:1047–1064.  [DOI](https://doi.org/10.1016/j.jmoldx.2021.05.013) | [PMC free article](/articles/PMC8579245/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34118403/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Mol%20Diagn&title=Recommendations%20for%20Clinical%20CYP2D6%20genotyping%20allele%20selection:%20a%20joint%20consensus%20recommendation%20of%20the%20Association%20for%20Molecular%20Pathology,%20College%20of%20American%20Pathologists,%20Dutch%20Pharmacogenetics%20Working%20Group%20of%20the%20Royal%20Dutch%20Pharmacists%20Association,%20and%20the%20European%20Society%20for%20Pharmacogenomics%20and%20Personalized%20Therapy&volume=23&publication_year=2021&pages=1047-1064&pmid=34118403&doi=10.1016/j.jmoldx.2021.05.013&)

33. Ray B, Ozcagli E, Sadee W, Wang D (2019) CYP2D6 haplotypes with enhancer single-nucleotide polymorphism rs5758550 and rs16947 (*2 allele): implications for CYP2D6 genotyping panels. Pharmacogenet Genomics 29:39–47.  [DOI](https://doi.org/10.1097/FPC.0000000000000363) | [PMC free article](/articles/PMC6441960/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30520769/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=CYP2D6%20haplotypes%20with%20enhancer%20single-nucleotide%20polymorphism%20rs5758550%20and%20rs16947%20(*2%20allele):%20implications%20for%20CYP2D6%20genotyping%20panels&volume=29&publication_year=2019&pages=39-47&pmid=30520769&doi=10.1097/FPC.0000000000000363&)

34. Relling MV, Evans WE (2015) Pharmacogenomics in the clinic. Nature 526:343–350.  [DOI](https://doi.org/10.1038/nature15817) | [PMC free article](/articles/PMC4711261/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26469045/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Pharmacogenomics%20in%20the%20clinic&volume=526&publication_year=2015&pages=343-350&pmid=26469045&doi=10.1038/nature15817&)

35. Shakeel N, Sletner L, Falk RS, Slinning K, Martinsen EW, Jenum AK, Eberhard-Gran M (2018) Prevalence of postpartum depressive symptoms in a multiethnic population and the role of ethnicity and integration. J Affect Disord 241:49–58.  [DOI](https://doi.org/10.1016/j.jad.2018.07.056) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30096592/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Affect%20Disord&title=Prevalence%20of%20postpartum%20depressive%20symptoms%20in%20a%20multiethnic%20population%20and%20the%20role%20of%20ethnicity%20and%20integration&volume=241&publication_year=2018&pages=49-58&pmid=30096592&doi=10.1016/j.jad.2018.07.056&)

36. Shi J, Wang X, Lyu L, Jiang H, Zhu HJ (2018) Comparison of protein expression between human livers and the hepatic cell lines HepG2, Hep3B, and Huh7 using SWATH and MRM-HR proteomics: Focusing on drug-metabolizing enzymes. Drug Metab Pharmacokinet 33:133–140.  [DOI](https://doi.org/10.1016/j.dmpk.2018.03.003) | [PMC free article](/articles/PMC6309175/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29610054/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Pharmacokinet&title=Comparison%20of%20protein%20expression%20between%20human%20livers%20and%20the%20hepatic%20cell%20lines%20HepG2,%20Hep3B,%20and%20Huh7%20using%20SWATH%20and%20MRM-HR%20proteomics:%20Focusing%20on%20drug-metabolizing%20enzymes&volume=33&publication_year=2018&pages=133-140&pmid=29610054&doi=10.1016/j.dmpk.2018.03.003&)

37. Smith CADWolf CRGough ACSpurr NKLeigh PNSummers BAHarding AEMaraganore DMSturman SGWilliams AC, et al. (1992) Debrisoquine hydroxylase gene polymorphism and susceptibility to Parkinson’s disease. [published correction appears in Lancet (1992) 340:64]. Lancet 339:1375–1377.  [DOI](https://doi.org/10.1016/0140-6736(92)91196-f) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1350805/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Debrisoquine%20hydroxylase%20gene%20polymorphism%20and%20susceptibility%20to%20Parkinson%E2%80%99s%20disease&volume=339&publication_year=1992&pages=1375-1377&pmid=1350805&doi=10.1016/0140-6736(92)91196-f&)

38. Smith LS, Wang X, Shi J, He B, Zhu HJ (2023) Genome-wide association study for the genetic determinants of thiopurine methyltransferase protein expression in human livers and racial differences. Pharm Res DOI: 10.1007/s11095-023-03558-1 [published ahead of print].  [10.1007/s11095-023-03558-1](http://10.1007/s11095-023-03558-1) | [DOI](https://doi.org/10.1007/s11095-023-03558-1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/37430149/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharm%20Res&title=Genome-wide%20association%20study%20for%20the%20genetic%20determinants%20of%20thiopurine%20methyltransferase%20protein%20expression%20in%20human%20livers%20and%20racial%20differences&publication_year=2023&pmid=37430149&doi=10.1007/s11095-023-03558-1&)

39. Solomon HV, Cates KW, Li KJ (2019) Does obtaining CYP2D6 and CYP2C19 pharmacogenetic testing predict antidepressant response or adverse drug reactions? Psychiatry Res 271:604–613.  [DOI](https://doi.org/10.1016/j.psychres.2018.12.053) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30554109/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Psychiatry%20Res&title=Does%20obtaining%20CYP2D6%20and%20CYP2C19%20pharmacogenetic%20testing%20predict%20antidepressant%20response%20or%20adverse%20drug%20reactions?&volume=271&publication_year=2019&pages=604-613&pmid=30554109&doi=10.1016/j.psychres.2018.12.053&)

40. Spaggiari D, Geiser L, Daali Y, Rudaz S (2014) Phenotyping of CYP450 in human liver microsomes using the cocktail approach. Anal Bioanal Chem 406:4875–4887.  [DOI](https://doi.org/10.1007/s00216-014-7915-4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24894520/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anal%20Bioanal%20Chem&title=Phenotyping%20of%20CYP450%20in%20human%20liver%20microsomes%20using%20the%20cocktail%20approach&volume=406&publication_year=2014&pages=4875-4887&pmid=24894520&doi=10.1007/s00216-014-7915-4&)

41. Stevens JC, Marsh SA, Zaya MJ, Regina KJ, Divakaran K, Le M, Hines RN (2008) Developmental changes in human liver CYP2D6 expression. Drug Metab Dispos 36:1587–1593.  [DOI](https://doi.org/10.1124/dmd.108.021873) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18474679/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Developmental%20changes%20in%20human%20liver%20CYP2D6%20expression&volume=36&publication_year=2008&pages=1587-1593&pmid=18474679&doi=10.1124/dmd.108.021873&)

42. Tao E, Liu Z, Chen B, Pan X, Shao M (1998) Association between the debrisoquine hydroxylase gene polymorphism and the genetic susceptibility of Parkinson’s disease. Zhonghua Yi Xue Yi Chuan Xue Za Zhi 15:281–283.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/9758873/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Zhonghua%20Yi%20Xue%20Yi%20Chuan%20Xue%20Za%20Zhi&title=Association%20between%20the%20debrisoquine%20hydroxylase%20gene%20polymorphism%20and%20the%20genetic%20susceptibility%20of%20Parkinson%E2%80%99s%20disease&volume=15&publication_year=1998&pages=281-283&pmid=9758873&)

43. Tracy TSChaudhry ASPrasad BThummel KESchuetz EGZhong XBTien YCJeong HPan XShireman LM, et al. (2016) Interindividual variability in cytochrome P450-mediated drug metabolism. Drug Metab Dispos 44:343–351.  [DOI](https://doi.org/10.1124/dmd.115.067900) | [PMC free article](/articles/PMC4767386/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26681736/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Interindividual%20variability%20in%20cytochrome%20P450-mediated%20drug%20metabolism&volume=44&publication_year=2016&pages=343-351&pmid=26681736&doi=10.1124/dmd.115.067900&)

44. Wang D, Papp AC, Sun X (2015) Functional characterization of CYP2D6 enhancer polymorphisms. Hum Mol Genet 24:1556–1562.  [DOI](https://doi.org/10.1093/hmg/ddu566) | [PMC free article](/articles/PMC4381757/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25381333/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=Functional%20characterization%20of%20CYP2D6%20enhancer%20polymorphisms&volume=24&publication_year=2015&pages=1556-1562&pmid=25381333&doi=10.1093/hmg/ddu566&)

45. Wang D, Poi MJ, Sun X, Gaedigk A, Leeder JS, Sadee W (2014) Common CYP2D6 polymorphisms affecting alternative splicing and transcription: long-range haplotypes with two regulatory variants modulate CYP2D6 activity. Hum Mol Genet 23:268–278.  [DOI](https://doi.org/10.1093/hmg/ddt417) | [PMC free article](/articles/PMC3857955/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23985325/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=Common%20CYP2D6%20polymorphisms%20affecting%20alternative%20splicing%20and%20transcription:%20long-range%20haplotypes%20with%20two%20regulatory%20variants%20modulate%20CYP2D6%20activity&volume=23&publication_year=2014&pages=268-278&pmid=23985325&doi=10.1093/hmg/ddt417&)

46. Wang X, He B, Shi J, Li Q, Zhu HJ (2020) Comparative proteomics analysis of human liver microsomes and S9 fractions. Drug Metab Dispos 48:31–40.  [DOI](https://doi.org/10.1124/dmd.119.089235) | [PMC free article](/articles/PMC6918043/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31699809/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Comparative%20proteomics%20analysis%20of%20human%20liver%20microsomes%20and%20S9%20fractions&volume=48&publication_year=2020&pages=31-40&pmid=31699809&doi=10.1124/dmd.119.089235&)

47. Yang XZhang BMolony CChudin EHao KZhu JGaedigk ASuver CZhong HLeeder JS, et al. (2010) Systematic genetic and genomic analysis of cytochrome P450 enzyme activities in human liver. Genome Res 20:1020–1036.  [DOI](https://doi.org/10.1101/gr.103341.109) | [PMC free article](/articles/PMC2909567/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20538623/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res&title=Systematic%20genetic%20and%20genomic%20analysis%20of%20cytochrome%20P450%20enzyme%20activities%20in%20human%20liver&volume=20&publication_year=2010&pages=1020-1036&pmid=20538623&doi=10.1101/gr.103341.109&)

48. Yu A-M, Idle JR, Byrd LG, Krausz KW, Küpfer A, Gonzalez FJ (2003) Regeneration of serotonin from 5-methoxytryptamine by polymorphic human CYP2D6. Pharmacogenetics 13:173–181.  [DOI](https://doi.org/10.1097/01.fpc.0000054066.98065.7b) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12618595/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Regeneration%20of%20serotonin%20from%205-methoxytryptamine%20by%20polymorphic%20human%20CYP2D6&volume=13&publication_year=2003&pages=173-181&pmid=12618595&doi=10.1097/01.fpc.0000054066.98065.7b&)

49. Zanger UM, Fischer J, Raimundo S, Stüven T, Evert BO, Schwab M, Eichelbaum M (2001) Comprehensive analysis of the genetic factors determining expression and function of hepatic CYP2D6. Pharmacogenetics 11:573–585.  [DOI](https://doi.org/10.1097/00008571-200110000-00004) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11668217/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Comprehensive%20analysis%20of%20the%20genetic%20factors%20determining%20expression%20and%20function%20of%20hepatic%20CYP2D6&volume=11&publication_year=2001&pages=573-585&pmid=11668217&doi=10.1097/00008571-200110000-00004&)
