# Impact of CYP2D6 and CYP2B6 phenotypes on the response to tramadol in patients with acute post‐surgical pain

## Metadata
**Authors:** Ana Casajús, Pablo Zubiaur, Enrique Alday, Paula Soria‐Chacartegui, Miriam Saiz‐Rodríguez, Lara Gutierrez, Catalina Aragonés, Diana Campodónico, Antía Gómez‐Fernández, Marcos Navares‐Gómez, Gonzalo Villapalos‐García, Gina Mejía‐Abril, Dolores Ochoa, Francisco Abad‐Santos
**Journal:** Clinical and Translational Science
**Date:** 2024 Jan 1
**DOI:** [10.1111/cts.13698](https://doi.org/10.1111/cts.13698)
**PMID:** 38140786
**PMCID:** PMC10787143
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10787143/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC10787143/pdf/CTS-17-e13698.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC10787143/pdf/CTS-17-e13698.pdf)

## Abstract

Tramadol is an important minor opioid prescribed for pain management. In this study, we analyzed the well‐known impact of CYP2D6 genetic variation and 60 additional variants in eight candidate genes (i.e., ABCG2, SLCO1B1, CYP2D6, CYP2B6, CYP2C19, CYP2C9, CYP3A5, and CYP3A4) on tramadol efficacy and safety. Some 108 patients with pain after surgery admitted to a post‐anesthesia care unit (PACU) and prescribed tramadol were recruited. They were genotyped, and tramadol M1/M2 metabolite concentrations were determined by a newly validated HPLC‐MS/MS method. CYP2D6 intermediate (IM) and poor (PM) metabolizers showed lower M1 concentrations adjusted for dose/weight at 30 and 120 min compared to ultrarapid (UM) and normal (NM) metabolizers (univariate p < 0.001 and 0.020, multivariate p < 0.001 and 0.001, unstandardized β coefficients = 0.386 and 0.346, R 2 = 0.146 and 0.120, respectively). CYP2B6 PMs (n = 10) were significantly related to a higher reduction in pain 30 min after tramadol intake (univariate p = 0.038, multivariate p = 0.016, unstandardized β coefficient = 0.224, R 2 = 0.178), to lower PACU admission time (p = 0.007), and to lower incidence of adverse drug reactions (p = 0.038) compared to the other phenotypes. CYP3A4 IMs and PMs showed a higher prevalence of drowsiness and dizziness (p = 0.028 and 0.005, respectively). Our results suggest that the interaction of CYP2B6 and CYP2D6 phenotypes may be clinically relevant, pending validation of these results in large, independent cohorts. Additional research is required to clarify the impact of CYP3A4 genetic variation on tramadol response.

Nowadays, *CYP2D6* is considered the only clinically relevant pharmacogenetic biomarker for tramadol prescription; however, polymorphism of other metabolizing enzymes may be clinically relevant.

In this work, CYP2B6 genotype‐informed phenotype was related to the response and tolerability to tramadol in patients with acute post‐surgical pain.

CYP2B6 phenotype may be more relevant than previously thought in relation to tramadol prescription; our results suggest that the interaction of CYP2B6 and CYP2D6 phenotypes may be clinically relevant, pending validation of these results in large, independent cohorts.

*CYP2D6* was the main predictor of tramadol M1 metabolite concentrations but had a modest impact on drug effectiveness and safety.

## INTRODUCTION

Tramadol is a centrally acting, analgesic drug used for the treatment of moderate pain or, in association with non‐steroidal anti‐inflammatory drugs (NSAIDs), severe pain. It can be administered p.o. (per os), intramuscularly (i.m.), subcutaneously (s.c.), or intravenously (i.v.) at a dose of 50–100 mg every 6–8 h without exceeding a maximum daily dose of 400 mg.[1](#cts13698-bib-0001) After oral administration, it is almost completely absorbed (90%) and rapidly distributed, reaching maximum concentrations (*C* _max_) in 1–2 h (*T* _max_).[1](#cts13698-bib-0001) It is metabolized in the liver mainly by O‐ and N‐demethylation and by conjugation reactions forming glucuronides and sulfates.[2](#cts13698-bib-0002) O‐Demethylation of tramadol to M1, the main active metabolite, is catalyzed by cytochrome P450 (CYP) 2D6 isoform (CYP2D6), whereas N‐demethylation to M2 is catalyzed by CYP2B6 and CYP3A4 (Figure [1](#cts13698-fig-0001)). M1 and M2 may then be further metabolized to secondary metabolites and M1 may also be inactivated in the liver by glucuronidation, mostly via UGT2B7 and UGT1A8 (Figure [1](#cts13698-fig-0001)). Tramadol and its metabolites are excreted mainly by the renal route with an elimination half‐life (*T* _1/2_) of approximately 6 h.[1](#cts13698-bib-0001)

### FIGURE 1.

![FIGURE 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/924c/10787143/f8873ddf728e/CTS-17-e13698-g001.jpg)

Tramadol's metabolic pathway. Blue arrows: hydroxylation reactions. Orange arrows: conjugation reactions. Figured inspired by PharmGKB's tramadol pharmacokinetics pathway. 2

Tramadol has two chiral centers and is administered as a racemic mixture of the (+)‐tramadol (R, R) enantiomer and the (−)‐tramadol (S, S) enantiomer. It acts by a dual mechanism of action: pure agonist of μ‐opioid receptors (effect produced mainly by the (+)‐enantiomer of the M1 metabolite), and inhibition of neuronal reuptake of noradrenaline and serotonin and increased serotonin release (effect produced by (+)‐tramadol and (−)‐tramadol).[1](#cts13698-bib-0001) The relationship between serum concentrations and analgesic effect is dose‐dependent and may vary considerably among patients. Generally, a tramadol serum concentration of 100–300 ng/mL is safe and effective.[1](#cts13698-bib-0001), [3](#cts13698-bib-0003), [4](#cts13698-bib-0004) Therapeutic drug monitoring (TDM) of serum levels allows adjustment of the treatment based on drugs and an individual's characteristics, to an effective and safe level.[5](#cts13698-bib-0005) Moreover, the gene encoding CYP2D6, which is apparently the main enzyme responsible for tramadol metabolism, is subject to high genetic variation[6](#cts13698-bib-0006) but other polymorphic genes such as *CYP2B6* and *CYP3A4* are also involved. As all the variability associated with genetic variation is not yet controlled, although there are specific drug recommendations, there is no consensus on them. Therefore, TDM could be potentially useful in this case, especially after the first administration (early TDM, as performed with voriconazole).

Given the high genetic polymorphism of *CYP2D6*, both the Clinical Pharmacogenetics Implementation Consortium (CPIC) and the Dutch Pharmacogenetics Working Group (DPWG) of the Royal Dutch Pharmacists Association (KNMP) issued pharmacogenetic guidelines containing prescribing recommendations.[6](#cts13698-bib-0006), [7](#cts13698-bib-0007) However, these recommendations are rather dissimilar: on the one hand, CPIC recommends that CYP2D6 ultrarapid (UM) and poor metabolizers (PM) avoid tramadol and receive non‐opioid analgesics, and, if needed, non‐codeine opioids (such as oxycodone).[6](#cts13698-bib-0006) On the other hand, the DPWG recommends that UMs receive an alternative drug not metabolized or metabolized to a lesser extent by CYP2D6, but if tramadol use is required, a 40% of the standard dose is indicated. DPWG guideline also recommends against tramadol use for PMs.[7](#cts13698-bib-0007)

A potential explanation for this lack of consensus is that other genetic factors contribute to tramadol response apart from *CYP2D6* variation. Given the complexity of the tramadol metabolic pathway and the involvement of several enzymes in parallel, it is understandable that the impact of genetic variation in a single enzyme (i.e., *CYP2D6*) on drug efficacy and safety has not been clearly established. Alternatively, we hypothesize that a polygenic model may better classify patients predisposed to lack of efficacy or toxicity. Considering tramadol's pathway, it is reasonable to think that *CYP3A4* or *CYP2B6* genetic variation may influence tramadol therapy outcomes. Thus, in this study we aimed to shed light on which is the best predictive model of tramadol pharmacotherapy outcomes. For this purpose, we evaluated the effects of CYP2D6 genotype‐informed phenotype and of variants in seven additional pharmacogenes (e.g., *CYP3A4* and *CYP2B6*) together with demographic and clinical characteristics and M1/M2 concentrations on the response, tolerability, and incidence of adverse drug reactions (ADRs) to tramadol in patients with acute post‐surgical pain. The present work is part of the La Princesa Multidisciplinary Initiative for the Implementation of Pharmacogenetics (PriME‐PGx).[8](#cts13698-bib-0008)

## METHODS

### Study design and population

The present work is an observational and prospective study that aimed to assess the impact of CYP2D6 genotype‐informed phenotype and of several variants in additional relevant pharmacogenes (i.e., *CYP3A5*, *CYP2C19*, *CYP2C9*, *CYP2B6*, *SLCO1B1*, *ABCG2*, *UGT1A1*, and *UGT2B15*; genotyped variants indicated in Table [1](#cts13698-tbl-0001)), as well as tramadol M1 and M2 metabolite concentrations, demographic and clinical characteristics, and pain rescue treatment with any analgesic and/or anti‐inflammatory drug. Patients over 18 years old presenting acute pain after a surgical procedure with a visual analog scale (VAS) score greater than or equal to 4 cm during their stay in the post‐anesthesia care unit (PACU) were treated with tramadol 100 mg i.v. The exclusion criteria were as follows: subjects undergoing chronic treatment with opioid analgesics and pregnant or breastfeeding women. All patients signed an informed consent for their participation in the present work. The study was approved by the Institutional Review Board (IRB) of the Hospital Universitario de La Princesa (Madrid, Spain) in accordance with Spanish legislation and the Revised Declaration of Helsinki.[9](#cts13698-bib-0009), [10](#cts13698-bib-0010) The project was classified as a prospective, follow‐up, observational study and authorized by the Spanish Drugs Agency.

### TABLE 1.

| Gene | Allele | SNV | Gene | Allele | SNP |
| --- | --- | --- | --- | --- | --- |
| CYP2D6 | *3 | rs35742686 | CYP2C19 | *2 | rs4244285, rs12769205 |
| *4 | rs3892097 | *3 | rs4986893 |  |  |
| *6 | rs5030655 | *4 | rs28399504, rs12248560 |  |  |
| *7 | rs5030867 | *5 | rs56337013 |  |  |
| *8 | rs5030865 | *6 | rs72552267 |  |  |
| *9 | rs5030656 | *7 | rs72558186 |  |  |
| *10 | rs1065852 | *8 | rs41291556 |  |  |
| *12 | rs5030862 | *9 | rs17884712 |  |  |
| *14 | rs5030865 | *17 | rs12248560 |  |  |
| *15 | rs774671100 | *35 | rs12769205 |  |  |
| *17 | rs28371706 | CYP2C9 | *2 | rs1799853 |  |
| *19 | rs72549353 | *3 | rs1057910 |  |  |
| *29 | rs59421388 | *5 | rs28371686 |  |  |
| *41 | rs28371725 | *8 | rs7900194, rs9332094 |  |  |
| *56 | rs72549347 | *11 | rs28371685 |  |  |
| *59 | rs79292917 | CYP3A4 | *2 | rs55785340, rs61469810 |  |
| CYP3A5 | *3 | rs776746 | *3 | rs4986910 |  |
| *6 | rs10264272 | *4 | rs55951658 |  |  |
| *7 | rs41303343 | *5 | rs55901263 |  |  |
| ABCG2 | C>T | rs2231137 | *6 | rs4646438 |  |
| G>T | rs2231142 | *18 | rs28371759 |  |  |
| G>A | rs7699188 | *20 | rs67666821 |  |  |
| SLCO1B1 | *2 | rs56101265 | *22 | rs35599367 |  |
| *3 | rs56061388 | CYP2B6 | *4 | rs2279343 |  |
| *4 | rs11045819 | *5 | rs3211371 |  |  |
| *5 | rs4149056 | *6 | rs3745274, rs2279343 |  |  |
| *6 | rs55901008 | *7 | rs3745274, rs2279343, rs3211371 |  |  |
| *9 | rs59502379 | *9 | rs3745274 |  |  |
| *10 | rs56199088 | *18 | rs28399499 |  |  |
| *14 | rs2306283 | *22 | rs34223104 |  |  |
| UGT1A1 | *6 | rs4148323 | UGT2B15 |   | rs1902023 |
| *80 | rs887829 |  |  |  |  |

Table 1 Caption: Genotyped variants, alleles, and genes.

### Pharmacokinetics and safety

Two EDTA‐K_2_ blood tubes were collected 30 min after tramadol administration and on discharge from PACU for genotyping and pharmacokinetic profiling. The samples were centrifuged to extract plasma in <1 h and were stored at −80°C until drug plasma determination. Tramadol, M1, and M2 concentrations were determined with a high‐performance liquid chromatography–tandem mass spectrometry (HPLC‐MS/MS) instrument. The analytical method was validated according to the US Food and Drug Administration (FDA) bioanalytical method validation guidance for linearity and calibration range, selectivity, specificity, sensitivity, accuracy, and precision.[11](#cts13698-bib-0011) Validation data are shown in the results section of this article. Because this study was classified as observational, no modifications of routine clinical practice were allowed. Consequently, the same i.v. line used for tramadol administration was used to collect the samples. Tramadol carryover was expectedly high, and therefore tramadol concentrations were excluded from the analysis. Conversely, metabolites concentrations showed no carryover effect and were included in the study.

Adverse event (AE) identification was performed by means of open questions to the patients. Furthermore, spontaneously reported AEs were registered. The determination of causality of the AE was carried out using the Spanish Pharmacovigilance System algorithm.[12](#cts13698-bib-0012) Only AEs with a possible, probable, or definitive relationship with tramadol were classified as ADRs and considered for the present study.

### Genotyping, haplotyping, and phenotyping

EDTA‐K2 tubes containing patients’ blood were thawed for DNA extraction with a Maxwell® RSC Instrument (Promega Biotech Ibérica S.L., Spain). Genotyping was performed by real‐time polymerase chain reaction (qPCR) with TaqMan® hydrolysis probes. For this, a QuantStudio 12 K Flex qPCR instrument (Applied Biosystems, ThermoFisher, USA) with an OpenArray™ thermal block and a customized array were used to genotype the variants listed in Table [1](#cts13698-tbl-0001). *SLC22A1*, *UGT2B27*, *UGT1A8*, and *ABCC2* were not analyzed since there are no clinically or functionally validated single nucleotide polymorphism (SNPs) for them. Furthermore, a *CYP2D6* gene copy number assay (CNV) was performed using the same instrument with a 96‐well thermal block.[13](#cts13698-bib-0013)

Alleles were defined according to CPIC allele definition tables (available at [https://cpicpgx.org/](https://cpicpgx.org/)) and PharmVar database (available at [https://www.pharmvar.org/](https://www.pharmvar.org/)). Pharmacogenetic phenotypes were inferred based on the obtained genotypes. *CYP2D6*, *CYP3A5*, *CYP2C19*, *CYP2C9*, *CYP2B6*, *SLCO1B1*, and *ABCG2* variants were used to infer the pharmacogenetic phenotype following the CPIC guidelines.[14](#cts13698-bib-0014), [15](#cts13698-bib-0015), [16](#cts13698-bib-0016), [17](#cts13698-bib-0017), [18](#cts13698-bib-0018), [19](#cts13698-bib-0019) The genotype‐informed phenotype for CYP3A4 was performed according to the DPWG indications.[20](#cts13698-bib-0020)

### Statistical analysis

The statistical analysis was performed using SPSS statistics software version 23 (SPSS Inc., Chicago, IL, USA). M1 and M2 concentrations were divided by the dose/weight (DW) ratio (M1/DW, M2/DW, respectively) to correct the effect of weight on relative dosage as part of a secondary analysis.

Patients with M1 levels below the lower limit of quantification (LLOQ) were assigned a 2 ng/mL value; M2 levels below the LLOQ (i.e., 25 ng/mL) were excluded from the analysis. Shapiro–Wilk and Kolmogorov–Smirnov tests revealed that M1/M2 levels, %VAS_30_ and %VAS_120_ (i.e., the percentual reduction in the VAS score at 30 min and at discharge, or around 2 h, respectively) did not follow a normal distribution, and they were therefore analyzed nonparametrically, or parametrically after logarithmic transformation when possible.

An initial descriptive analysis of demographic characteristics was performed. Second, a univariate analysis was conducted where the main dependent variables (i.e., %VAS_30_, %VAS_120_, M1/_30_, M1/_120_, M2/_30_, and M2/_120_) were analyzed according to categorical variables (e.g., sex, need of pain rescue treatment, genotypes, and phenotypes). When analyzing M1_30_, M1_120_, %VAS_30_, %VAS_120_, M2_30_, and M2_120_, the nonparametric tests Mann–Whitney U test (for two‐category variables) and Kruskal–Wallis (for variables with three or more categories) were used. Statistical significance was considered to be *p <* 0.05. Afterwards, all parameters were individually analyzed with multivariate analysis, using linear regression. Only variables with *p <* 0.05 in the univariate analysis were included as independent variables and were transformed into dummy variables when necessary. In the multivariate analyses of %VAS variables, the rescue treatment was introduced as a covariate. Univariate analysis was initially carried out comparing the incidence of ADRs according to sex, use of pain rescue treatment, and genetic variables (i.e., genotypes or phenotypes). For this analysis, a Pearson's chi‐squared or a Fisher's exact test (if more than 20% of cells had expected frequencies lower than 5) were used. In the case of time spent in the PACU since the administration of tramadol, the same methodology was followed. The Bonferroni correction for multiple comparisons was applied in the multivariate analyses.

## RESULTS

### Optimization and validation of the analytical method

The Supplementary Materials contain a full description of this section (Table [S1](#cts13698-supitem-0001)).

### Clinical study

A total of 108 patients participated in the study (57 men and 51 women). Overall, men exhibited higher weight and height compared to women (*p <* 0.001 and *p <* 0.001, respectively) with no significant differences in age and body mass index (Table [2](#cts13698-tbl-0002)). Due to their heterogeneity, surgeries were classified into six groups: abdominal surgery (*n* = 47), head and neck (*n* = 16), trauma (*n* = 23), vascular (*n* = 8), urological (*n* = 4), gynecological (*n* = 5), and ocular surgery (*n* = 4). No differences in surgery type were observed according to sex (*p* > 0.05), except for two types of surgery (i.e., urological and gynecological surgeries) as all individuals were expectedly males and females, respectively. Women experienced greater pain after surgery (VAS_0_) than men (*p* = 0.038) with no significant differences at the other times (Table [2](#cts13698-tbl-0002)). Only 22 patients (20%) required rescue treatments; morphine was administered in most cases (*n* = 14/22 patients), followed by paracetamol (*n* = 5), dexketoprofen (*n* = 2), fentanyl (*n* = 1), and bupivacaine (*n* = 1). All rescue treatments were administered solely, except for one patient that received morphine and paracetamol.

### TABLE 2.

| Variable | Men (n = 57) |  | Women (n = 51) |  |
| --- | --- | --- | --- | --- |
| Mean | SD | Mean | SD |  |
| Age (years) | 56.65 | 16.75 | 60.06 | 16.39 |
| Weight (kg) | 81.47* | 15.91 | 66.85 | 14.10 |
| Height (m) | 1.74* | 0.06 | 1.58 | 0.07 |
| BMI (kg/m2) | 26.76 | 4.71 | 26.63 | 5.70 |
| VAS0 | 6.74 | 1.41 | 7.16* | 1.67 |
| VAS30 | 3.47 | 2.01 | 2.64 | 2.08 |
| VAS120 | 2.04 | 1.85 | 2.08 | 1.87 |

Table 2 Caption: Baseline characteristics of the study population.

The %VAS_30_ and %VAS_120_ differed according to rescue treatment and CYP2B6 phenotype, but no differences were observed according to sex and other pharmacogenetic biomarkers (Table [3](#cts13698-tbl-0003)). Patients requiring rescue treatment showed a significantly lower %VAS_30_ and a lower %VAS_120_ (univariate *p <* 0.001, multivariate *p <* 0.001, unstandardized β coefficient = 0.386, and *R* ^2^ = 0.146 and univariate *p =* 0.020, multivariate *p <* 0.001, unstandardized β coefficient = 0.346, and *R* ^2^ = 0.120). CYP2B6 PMs showed a significantly higher %VAS_30_ than intermediate metabolizers (IMs) + normal metabolizers) NMs + RMs (univariate *p =* 0.038, multivariate *p =* 0.016, unstandardized β coefficient = 0.224, and *R* ^2^ = 0.178) (Table [3](#cts13698-tbl-0003)).

### TABLE 3.

| Variable |  | %VAS30 |  |  | %VAS120 |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- |
| N | Mean | SD | N | Mean | SD |  |  |
| Total |  | 108 | 49.78 | 28.84 | 106 | 70.74 | 24.67 |
| Sex | Men | 57 | 49.31 | 25.78 | 57 | 70.58 | 24.83 |
| Women | 51 | 50.31 | 32.19 | 49 | 70.92 | 24.72 |  |
| Pain rescue treatment | No | 86 | 53.34* | 28.21 | 84 | 75.13* | 23.12 |
| Yes | 22 | 33.92 | 27.54 | 22 | 53.98 | 23.61 |  |
| CYP2D6 | UM | 10 | 35.93 | 23.00 | 10 | 59.71 | 21.30 |
| NM | 69 | 52.88 | 30.32 | 68 | 73.16 | 25.19 |  |
| IM | 22 | 47.12 | 23.15 | 21 | 65.63 | 23.35 |  |
| PM | 3 | 20.00 | 20.00 | 3 | 57.78 | 23.41 |  |
| CYP2B6 | RM | 27 | 55.05 | 26.12 | 26 | 73.80 | 19.89 |
| NM | 22 | 46.13 | 29.98 | 22 | 66.79 | 29.24 |  |
| IM | 47 | 44.43 | 29.66 | 46 | 68.69 | 25.67 |  |
| PM | 10 | 69.96** | 24.40 | 10 | 83.74 | 19.42 |  |
| CYP2C19 | UM | 4 | 54.91 | 41.27 | 4 | 65.27 | 12.11 |
| RM | 20 | 57.25 | 20.07 | 20 | 76.52 | 21.70 |  |
| NM | 50 | 47.17 | 30.56 | 48 | 71.58 | 24.60 |  |
| IM | 31 | 48.58 | 30.54 | 31 | 65.91 | 27.78 |  |
| PM | 2 | 40.28 | 21.60 | 2 | 63.89 | 19.64 |  |
| CYP2C9 | NM | 71 | 51.08 | 30.48 | 70 | 71.73 | 26.21 |
| IM | 34 | 48.45 | 26.02 | 33 | 68.99 | 22.21 |  |
| PM | 3 | 34.17 | 19.42 | 3 | 66.67 | 15.27 |  |
| CYP3A4 | NM | 102 | 49.27 | 29.48 | 102 | 70.98 | 25.06 |
| IM/PM | 6 | 58.53 | 12.61 | 6 | 66.70 | 17.74 |  |
| CYP3A5 | IM | 11 | 37.85 | 26.17 | 11 | 61.01 | 24.29 |
| PM | 95 | 51.18 | 29.21 | 93 | 71.66 | 24.65 |  |
| SLCO1B1 | NF | 80 | 50.59 | 27.22 | 79 | 70.24 | 23.68 |
| DF/PF | 28 | 47.47 | 33.49 | 27 | 72.19 | 27.75 |  |
| ABCG2 | NF | 94 | 51.99 | 27.06 | 92 | 71.03 | 24.71 |
| DF | 14 | 34.97 | 36.61 | 14 | 68.84 | 25.18 |  |
| UGT1A1 | NM | 30 | 49.38 | 28.72 | 30 | 70.02 | 22.80 |
| IM | 50 | 50.85 | 29.65 | 49 | 74.19 | 24.17 |  |
| PM | 18 | 50.00 | 23.03 | 18 | 66.00 | 26.45 |  |
| UGT2B15 | T/T | 18 | 51.23 | 28.02 | 18 | 74.65 | 23.56 |
| T/G | 60 | 49.46 | 29.28 | 59 | 70.84 | 24.57 |  |
| G/G | 19 | 50.21 | 24.92 | 19 | 69.52 | 24.92 |  |

Table 3 Caption: %VAS reduction according to demographic characteristics and phenotypes.

M1_30_ (univariate *p* = 0.030) and M1_120_ (univariate *p* = 0.001, multivariate *p* = 0.003, unstandardized β coefficient = 10.02, *R* ^2^ = 0.21) levels were significantly higher in women compared to men (Table [4](#cts13698-tbl-0004)). In contrast, M1/DW_30_ and M1/DW_120_ concentrations did not differ according to sex or rescue treatment (Table [S2](#cts13698-supitem-0001)). CYP2D6 IMs + PMs showed significantly lower M1_30_ and M1_120_ levels than NMs + UMs (univariate *p* < 0.001 and *p* < 0.001, multivariate *p* < 0.001 and *p* = 0.002, unstandardized β coefficient = −13.40 and −14.21, respectively); the association between M1_120_ levels and CYP2D6 phenotype reached the statistical threshold for significance after the Bonferroni correction for multiple comparisons (i.e., 0.05/3 factors, i.e., sex, pain rescue treatment, and CYP2D6 phenotype = 0.017). IMs and PMs for CYP2D6 showed a lower M1/DW_30_ and M1/DW_120_ concentration, the latter being much lower at both times, than UMs and NMs (univariate *p =* 0.013, 0.003, multivariate *p =* 0.013 and 0.003, unstandardized β coefficients = 0.245 and 0.290, and *R* ^2^ = 0.060 and 0.084, respectively) M2 was only quantifiable in 20 subjects. Patients who required rescue medication had a lower M2/DW_120_ concentration (*p =* 0.032) (Table [S2](#cts13698-supitem-0001)). CYP2D6 UMs or NMs that are also CYP2B6 IMs or PMs were classified as CYP2D6‐CYP2B6 “fast‐slow” metabolizers; CYP2D6 IMs or PMs that are also CYP2B6 UMs, RMs, or NMs were classified as “slow‐fast” metabolizers. The CYP2D6‐CYP2B6 “fast‐slow” group was related to significantly higher M1_30_ and M1_120_ levels compared to the “slow‐fast” group, and these differences were greater in magnitude than the differences observed for CYP2D6 UMs + NMs versus CYP2D6 IMs + PMs (Table [5](#cts13698-tbl-0005)).

### TABLE 4.

| Variable |  | M130 (ng/mL) |  |  | M230 (ng/mL) |  |  | M1120 (ng/mL) |  |  | M2120 (ng/mL) |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| N | Mean | SD | N | Mean | SD | N | Mean | SD | N | Mean | SD |  |  |
| Sex | Men | 57 | 22.90 | 16.16 | 5 | 31.55 | 8.81 | 57 | 24.67 | 13.84 | 10 | 31.40 | 12.66 |
| Women | 51 | 31.16* | 20.85 | 7 | 35.52 | 8.40 | 51 | 36.78* | 20.31 | 11 | 35.84 | 12.40 |  |
| Pain rescue treatment | No | 81 | 26.19 | 19.56 | 12 | 33.86 | 8.42 | 86 | 30.28 | 18.10 | 19 | 34.42 | 12.88 |
| Yes | 20 | 29.37 | 16.39 | 0 |   |   | 22 | 33.91 | 15.18 | 2 | 27.13 | 1.23 |  |
| CYP2D6 | UM | 10 | 33.89 | 21.35 | 5 | 24.15 | 2.31 | 10 | 40.38 | 17.08 | 1 | 37.10 |   |
| NM | 69 | 29.22 | 18.72 | 3 | 19.02 | 0.38 | 69 | 34.27 | 16.47 | 13 | 35.05 | 13.48 |  |
| IM | 22 | 18.88* | 14.15 | 4 | 25.42 | 11.70 | 22 | 20.89* | 15.26 | 4 | 34.18 | 9.97 |  |
| PM | 3 | 2.89* | 1.55 | 0 |   |   | 4 | 4.36* | 1.88 | 3 | 26.25 | 14.16 |  |
| CYP2B6 | RM | 27 | 26.75 | 22.24 | 4 | 39.62 | 10.20 | 27 | 29.77 | 22.80 | 6 | 42.24 | 15.03 |
| NM | 22 | 23.81 | 19.36 | 5 | 31.60 | 8.10 | 22 | 30.00 | 14.80 | 5 | 35.89 | 12.81 |  |
| IM | 47 | 30.04 | 17.90 | 3 | 29.96 | 1.87 | 47 | 33.95 | 16.03 | 8 | 27.75 | 15.03 |  |
| PM | 10 | 21.85 | 10.92 | 0 |   |   | 10 | 26.46 | 13.45 | 0 |   |   |  |
| CYP2C19 | UM | 4 | 27.46 | 8.12 | 0 |   |   | 4 | 39.87 | 2.22 | 1 | 26.71 |   |
| RM | 20 | 28.87 | 22.43 | 3 | 31.84 | 0.16 | 20 | 30.54 | 16.30 | 4 | 36.28 | 8.12 |  |
| NM | 50 | 26.23 | 20.60 | 5 | 34.93 | 9.71 | 50 | 31.60 | 20.25 | 11 | 31.76 | 12.79 |  |
| IM | 31 | 26.86 | 15.09 | 4 | 34.05 | 11.32 | 31 | 29.02 | 15.00 | 5 | 37.41 | 16.32 |  |
| PM | 2 | 18.68 | 23.59 | 0 |   |   | 2 | 37.06 | 21.71 |   |   |   |  |
| CYP2C9 | NM | 71 | 27.09 | 18.90 | 8 | 34.46 | 8.40 | 71 | 31.78 | 17.51 | 12 | 35.83 | 13.09 |
| IM | 34 | 27.65 | 19.00 | 4 | 32.68 | 9.63 | 34 | 31.26 | 17.20 | 8 | 33.54 | 9.10 |  |
| PM | 3 | 10.44 | 14.62 | 0 |   |   | 3 | 10.30 | 14.00 | 1 | 9.93 |   |  |
| CYP3A4 | NM | 102 | 26.98 | 19.24 | 10 | 32.83 | 8.34 | 102 | 31.47 | 17.85 | 19 | 32.59 | 11.91 |
| IM | 6 | 23.58 | 11.88 | 2 | 39.05 | 10.25 | 6 | 23.33 | 8.62 | 2 | 44.55 | 16.46 |  |
| CYP3A5 | IM | 11 | 24.09 | 14.29 | 1 | 26.75 |   | 11 | 30.54 | 18.21 | 2 | 29.93 | 5.18 |
| PM | 97 | 27.57 | 19.31 | 11 | 34.51 | 8.51 | 95 | 31.56 | 17.43 | 18 | 34.46 | 13.25 |  |
| SLCO1B1 | NF | 80 | 25.91 | 18.64 | 9 | 32.84 | 8.48 | 80 | 30.95 | 17.84 | 15 | 33.77 | 13.10 |
| DF/PF | 28 | 29.32 | 19.70 | 3 | 36.92 | 9.12 | 28 | 31.23 | 17.00 | 6 | 33.62 | 11.74 |  |
| ABCG2 | NF | 94 | 25.95 | 19.11 | 10 | 32.83 | 8.24 | 94 | 31.31 | 18.01 | 18 | 32.44 | 12.24 |
| DF | 14 | 25.89 | 18.89 | 2 | 39.05 | 10.25 | 14 | 29.10 | 14.41 | 3 | 41.44 | 12.83 |  |
| UGT1A1 | NM | 30 | 30.65 | 20.21 | 4 | 30.26 | 9.60 | 30 | 31.75 | 16.78 | 5 | 33.12 | 11.23 |
| IM | 50 | 21.49 | 16.63 | 7 | 34.15 | 7.04 | 50 | 27.29 | 17.81 | 12 | 32.75 | 13.43 |  |
| PM | 18 | 30.00 | 21.84 | 1 | 46.29 |   | 18 | 31.27 | 22.13 | 3 | 41.02 | 13.64 |  |
| UGT2B15 | T/T | 18 | 23.82 | 13.64 | 5 | 36.74 | 9.42 | 18 | 27.56 | 14.76 | 4 | 44.04 | 17.94 |
| T/G | 60 | 27.52 | 19.12 | 6 | 32.66 | 8.95 | 60 | 31.53 | 19.32 | 14 | 31.06 | 10.86 |  |
| G/G | 19 | 24.54 | 24.00 | 1 | 26.75 |   | 19 | 25.72 | 17.46 | 2 | 35.35 | 2.48 |  |
| Total |  | 108 | 26.80 | 18.89 | 12 | 33.86 | 8.42 | 108 | 31.02 | 17.54 | 21 | 33.73 | 12.42 |

Table 4 Caption: M1 and M2 levels according to demographic characteristics and phenotypes.

### TABLE 5.

| Group | n | M130 (ng/mL) |  | p‐Value | M1120 (ng/mL) |  | p‐Value |
| --- | --- | --- | --- | --- | --- | --- | --- |
| Mean | SD | Mean | SD |  |  |  |  |
| Fast‐slow | 43 | 29.71 | 16.29 | 0.0002 | 33.73 | 15.98 | 0.0001 |
| Slow‐fast | 13 | 11.02 | 9.09 | 12.58 | 9.76 |  |  |
| CYP2D6 UM + NM | 79 | 29.81 | 18.98 | 0.02 | 34.17 | 17.66 | 0.00023 |
| CYP2D6 IM + PM | 25 | 16.96 | 14.27 | 18.9 | 15.3 |  |  |

Table 5 Caption: Comparison of M130 and M1120 differences according to CYP2D6‐CYP2B6metabolic groups and CYP2D6 phenotypes.

Patients with ADRs had longer PACU lengths of stay than those without ADRs (*p* = 0.008). Patients who took rescue treatment for pain also had longer PACU lengths of stay (*p* = 0.027) (Table [6](#cts13698-tbl-0006)). Conversely, CYP2B6 PMs had shorter PACU length stay times than IMs + NMs + RMs (*p* = 0.007) (Table [6](#cts13698-tbl-0006)).

### TABLE 6.

| Variable |  | N | Time (h) | SD |
| --- | --- | --- | --- | --- |
| Sex | Men | 56 | 2:05 | 0:54 |
| Women | 50 | 2:16 | 1:02 |  |
| ADRs | No | 53 | 1:55 | 0:48 |
| Yes | 53 | 2:25* | 1:03 |  |
| Pain rescue treatment | No | 84 | 2:04 | 0:56 |
| Yes | 22 | 2:34* | 0:57 |  |
| CYP2D6 | UM | 10 | 2:23 | 1:07 |
| NM | 69 | 2:03 | 0:53 |  |
| IM/PM | 23 | 2:29 | 1:05 |  |
| CYP2B6 | RM | 27 | 2:31 | 0:55 |
| NM | 21 | 1:57 | 0:53 |  |
| IM | 46 | 2:15 | 1:01 |  |
| PM | 10 | 1:22** | 0:20 |  |
| CYP2C19 | UM | 4 | 2:27 | 0:57 |
| RM | 20 | 1:43 | 0:42 |  |
| NM | 49 | 2:12 | 1:01 |  |
| IM/PM | 32 | 2:21 | 1:00 |  |
| CYP2C9 | NM | 70 | 2:07 | 0:58 |
| IM/PM | 36 | 2:15 | 0:57 |  |
| CYP3A5 | IM | 10 | 2:22 | 0:48 |
| PM | 94 | 2:08 | 0:58 |  |
| SLCO1B1 | NF | 79 | 2:07 | 0:51 |
| DF/PF | 27 | 2:19 | 1:15 |  |
| CYP3A4 | NM | 100 | 2:10 | 0:59 |
| IM/PM | 6 | 2:01 | 0:33 |  |
| ABCG2 | NF | 93 | 2:09 | 0:56 |
| DF | 13 | 2:15 | 1:11 |  |
| UGT1A1 | NM | 29 | 2:15 | 1:00 |
| IM | 49 | 1:58 | 0:48 |  |
| PM | 18 | 2:06 | 0:52 |  |
| UGT2B15 | T/T | 18 | 2:03 | 0:56 |
| T/G | 58 | 2:05 | 0:49 |  |
| G/G | 19 | 2:04 | 0:52 |  |

Table 6 Caption: Time from tramadol administration to discharge from PACU.

Some 53 patients (49.1%) experienced ADRs, drowsiness being the most frequent (*n* = 28), followed by dry mouth, nausea, dizziness, hyperhidrosis, headache, and vomiting (Table [7](#cts13698-tbl-0007)). CYP2B6 PMs showed fewer ADRs than IMs + NMs + RMs (*p =* 0.038). CYP3A4 IMs and PMs showed a higher prevalence of dizziness and drowsiness compared to NMs (*p =* 0.028 and 0.005, respectively). Although statistical significance was not reached, a tendency in CYP2D6 UMs toward higher ADR prevalence was observed (70.0%) compared to NMs (49.3%) and IMs + PMs (49.0%).

### TABLE 7.

| Variable |  | N | ADR yes (n (%)) | Dizziness (n (%)) | Headache (n (%)) | Drowsiness (n (%)) | Nausea (n (%)) | Vomiting (n (%)) | Dry mouth (n (%)) | Hyperhidrosis (n (%)) | Other (n (%)) |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Total |  | 108 | 53 (49.1) | 14 (13.0) | 3 (2.8) | 28 (26.4) | 19 (17.6) | 3 (2.8) | 20 (18.5) | 5 (4.6) | 3 (2.8) |
| Sex | Men | 57 | 24 (42.1) | 5 (8.8) | 2 (3.5) | 14 (25.0) | 7 (12.3) | 2 (3.5) | 7 (12.3) | 3 (5.3) | 3 (5.3) |
| Women | 51 | 29 (56.1) | 9 (17.6) | 1 (2.0) | 14 (28.0) | 12 (23.5) | 1 (2.0) | 13 (25.5) | 2 (3.9) | 0 (0.0) |  |
| Pain rescue treatment | No | 86 | 41 (47.7) | 11 (12.8) | 2 (2.3) | 20 (23.8) | 16 (18.6) | 3 (3.5) | 15 (17.4) | 3 (3.5) | 3 (3.5) |
| Yes | 22 | 12 (54.5) | 3 (13.6) | 1 (4.5) | 8 (36.4) | 3 (13.6) | 0 (0.0) | 5 (22.7) | 2 (9.1) | 0 (0.0) |  |
| CYP2D6 | UM | 10 | 7 (70.0) | 0 (0.0) | 0 (0.0) | 4 (40.0) | 1 (10.0) | 0 (0.0) | 2 (20.0) | 0 (0.0) | 1 (10.0) |
| NM | 69 | 34 (49.3) | 10 (14.5) | 3 (4.3) | 15 (22.4) | 11 (15.9) | 2 (2.9) | 12 (17.4) | 4 (5.8) | 2 (2.9) |  |
| IM/PM | 25 | 10 (40.0) | 3 (12.0) | 0 (0.0) | 8 (32.0) | 5 (20.0) | 1 (4.0) | 6 (24.0) | 1 (4.0) | 0 (0.0) |  |
| CYP2B6 | RM | 27 | 17 (63.0) | 7 (25.9) | 2 (7.4) | 10 (37.0) | 7 (25.9) | 2 (7.4) | 3 (11.1) | 2 (7.4) | 1 (3.7) |
| NM | 22 | 10 (45.5) | 2 (9.1) | 0 (0.0) | 5 (22.7) | 3 (13.6) | 0 (0.0) | 6 (27.3) | 0 (0.0) | 0 (0.0) |  |
| IM | 47 | 24 (51.1) | 5 (10.6) | 1 (2.1) | 13 (27.7) | 8 (17.0) | 1 (2.1) | 11 (23.4) | 2 (4.3) | 2 (4.3) |  |
| PM | 10 | 1 (10.0)* | 0 (0.0) | 0 (0.0) | 0 (0.0) | 1 (10.0) | 0 (0.0) | 0 (0.0) | 0 (0.0) | 0 (0.0) |  |
| CYP2C19 | UM | 4 | 3 (75.0) | 1 (25.0) | 0 (0.0) | 0 (0.0) | 1 (25.0) | 0 (0.0) | 1 (25.0) | 1 (25.0) | 0 (0.0) |
| RM | 20 | 9 (45.0) | 1 (5.0) | 0 (0.0) | 3 (15.0) | 2 (10.0) | 0 (0.0) | 2 (10.0) | 1 (5.0) | 2 (10.0) |  |
| NM | 50 | 24 (48.0) | 7 (14.0) | 1 (2.0) | 16 (32.0) | 10 (20.0) | 1 (2.0) | 9 (18.0) | 1 (2.0) | 1 (2.0) |  |
| IM/PM | 33 | 16 (48.5) | 5 (15.2) | 2 (6.1) | 9 (27.3) | 6 (18.2) | 2 (6.1) | 7 (21.2%) | 1 (3.0 | 0 (0.0) |  |
| CYP2C9 | NM | 71 | 35 (49.3) | 11 (15.5) | 3 (4.2) | 19 (26.8) | 14 (19.7) | 2 (2.8) | 13 (18.3) | 3 (4.2) | 3 (4.2) |
| IM/PM | 37 | 18 (48.6) | 3 (8.1) | 0 (0.0) | 9 (24.3) | 5 (13.5) | 1 (2.7) | 7 (18.9) | 2 (5.4) | 0 (0.0) |  |
| CYP3A5 | IM | 11 | 7 (63.6) | 3 (27.3) | 1 (9.1) | 4 (36.4) | 3 (27.3) | 0 (0.0) | 3 (27.3) | 1 (9.1) | 0 (0.0) |
| PM | 95 | 45 (47.4) | 11 (11.6) | 2 (2.1) | 24 (25.3) | 16 (16.8) | 3 (3.2) | 17 (17.9) | 3 (3.2) | 3 (3.2) |  |
| SLCO1B1 | NF | 80 | 38 (47.5) | 10 (12.5) | 3 (3.8) | 22 (28.2) | 13 (16.3) | 2 (2.5) | 14 (17.5) | 2 (2.5) | 3 (3.8) |
| DF/PF | 28 | 15 (53.6) | 4 (14.3) | 0 (0.0) | 6 (21.4) | 6 (21.4) | 1 (3.6) | 6 (21.4) | 3 (10.7) | 0 (0.0) |  |
| CYP3A4 | NM | 102 | 48 (47.1) | 11 (10.8) | 3 (2.9) | 23 (22.5) | 17 (16.7) | 3 (2.9) | 19 (18.6) | 4 (3.9) | 3 (2.9) |
| IM/PM | 6 | 5 (83.3 | 3 (50.0)* | 0 (0.0) | 5 (83.3)* | 2 (33.3) | 0 (0.0) | 1 (16.7) | 1 (16.7%) | 0 (0.0) |  |
| ABCG2 | NF | 94 | 45 (47.9) | 12 (12.8) | 3 (3.2) | 25 (26.6) | 15 (16.0) | 2 (2.1) | 18 (19.1) | 5 (5.3) | 3 (3.2) |
| DF | 14 | 8 (57.1) | 2 (14.3) | 0 (0.0) | 3 (21.4) | 4 (28.6) | 1 (7.1) | 2 (14.3) | 0 (0.0) | 0 (0.0) |  |
| UGT1A1 | NM | 30 | 11 (36.7) | 3 (10.0) | 0 (0.0) | 5 (17.2) | 3 (10.0) | 1 (3.3) | 4 (13.3) | 2 (6.7) | 1 (3.3) |
| IM | 50 | 22 (44.0) | 6 (12.0) | 2 (4.0) | 9 (18.4) | 10 (20.0) | 2 (4.0) | 5 (10.0) | 2 (4.0) | 2 (4.0) |  |
| PM | 18 | 10 (55.6) | 3 (12.2) | 1 (5.6) | 7 (38.9) | 3 (16.7) | 0 (0.0) | 5 (27.8) | 0 (0.0) | 0 (0.0) |  |
| UGT2B15 | NM | 18 | 10 (55.6) | 1 (5.6) | 1 (5.6) | 7 (38.9) | 6 (33.3) | 1 (5.6) | 2 (11.1) | 0 (0.0) | 1 (5.6) |
| IM | 60 | 24 (40.0) | 7 (11.7) | 1 (1.7) | 11 (19.0) | 5 (8.3) | 2 (3.3) | 9 (15.0) | 3 (5.0) | 1 (1.7) |  |
| PM | 19 | 9 (47.4) | 4 (21.1) | 1 (5.3) | 3 (15.8) | 5 (26.3) | 0 (0.0) | 3 (15.8) | 1 (5.3) | 1 (5.3) |  |

Table 7 Caption: ADRs after tramadol administration according to demographic characteristics and phenotype.

## DISCUSSION

Based on the publication of pharmacogenetic guidelines, *CYP2D6* genotype (or genotype‐informed phenotype) is the only clinically relevant biomarker that can help individualize tramadol pharmacotherapy. However, CPIC and DPWG differ in their recommendations for UMs and PMs, which implies a level of uncertainty for the clinician.[14](#cts13698-bib-0014), [15](#cts13698-bib-0015), [16](#cts13698-bib-0016), [17](#cts13698-bib-0017), [18](#cts13698-bib-0018), [19](#cts13698-bib-0019), [20](#cts13698-bib-0020) Furthermore, the clinical relevance of the genetic variation in other genes on tramadol is still uncertain. This study provides clarification of the impact of CYP2D6 genotype‐informed phenotype and genetic variation at other pharmacogenes on the effectiveness and safety of this drug.

The variations in baseline pain levels after surgery align with the findings from studies suggesting that women tend to exhibit higher pain sensitivity compared to men.[21](#cts13698-bib-0021) In this work, tramadol was demonstrated to be a rather effective analgesic drug as the mean pain reduction was 50% at 30 min and 70% at discharge, with only 20% of patients requiring rescue treatment. Expectedly, patients who required rescue treatment had a significant smaller %VAS_30_ and %VAS_120_ than those who did not. CYP2B6 PMs showed a significantly greater decrease in %VAS_30_ than RMs, NMs, and IMs. These results support other studies in which a higher area under the curve (AUC) of tramadol was found in CYP2B6 PMs and IMs, the accumulation of which (and its consequent transformation into M1) is the reason for its greater reduction in pain.[22](#cts13698-bib-0022) Furthermore, CYP2B6 PMs presented lower PACU admission times, which confirms the higher effectiveness of tramadol, which is consistent with the lower incidence of ADRs (i.e., not enough time to document them) and the accumulation of active moieties. Unfortunately, we were unable to confirm tramadol (parent drug) accumulation experimentally for the carryover issue explained earlier. Further research should confirm this hypothesis. A similar effect was expected for CYP3A4 IM or PM phenotypes, which was not observed, which suggests that CYP2B6 plays a major role in the N‐demethylation of tramadol compared to CYP3A4. Alternatively, the lack of association may be explained by the modest sample size (i.e., only 5 CYP3A4 IMs and 1 PM were identified). However, CYP3A4 IMs + PMs showed a higher prevalence of drowsiness and dizziness, which suggests that tramadol may indeed accumulate in these patients. Additional research is required to evaluate how *CYP3A4* genetic variation affects tramadol pharmacotherapy.

CYP2D6 PMs showed a not statistically significant 20% reduction in the VAS at 30 min compared to a reduction of 36%–53% in the other phenotypes at 30 min. The lack of statistical significance may be explained because only three patients were PMs (one of them showing a 0% reduction). However, this tendency could not be observed at 120 min. This modest effect suggests that the interplay of additional predictors could mask *CYP2D6* effect, particularly with small sample sizes. Previous studies have suggested an additive impact of *CYP2D6* and *CYP2B6* variation on tramadol pharmacokinetics.[23](#cts13698-bib-0023) Our work confirms that tramadol effectiveness may be better predicted with a combination of CYP2B6 and CYP2D6 genotype‐informed phenotypes, which is consistent with tramadol's metabolic pathway. This would explain the controversy on tramadol dose adjustment recommendations and the lack of consensus between DPWG and CPIC; since CYP2B6 is not considered, CYP2D6 only predicts part of the variability, and therefore the recommendations may not be accurate. For instance, a patient with a PM phenotype for CYP2B6 and PM for CYP2D6 may not form M2 (inactive), and therefore may accumulate tramadol+M1 (active), showing a good response to tramadol and a higher risk for ADRs. Or a patient with a UM phenotype for both enzymes may convert tramadol to M2 and M1 to a greater extent, with the M1+tramadol (active) concentration being low since M1 is further metabolized; consequently, this patient would be predicted to respond poorly to tramadol. In summary, the individual clinical context of each patient and genetic variation information (including *CYP2B6* and *CYP2D6*) should be considered to inform clinical decisions.

Concerning tramadol's metabolites, CYP2D6 IMs and PMs showed a lower M1/DW_30_ and M1/DW_120_ concentration than UMs and NMs. These findings are consistent with several prospective clinical trials where CYP2D6 PMs more often fail to exhibit analgesia response to tramadol.[3](#cts13698-bib-0003), [24](#cts13698-bib-0024) These results also support the recommendations made by CPIC, whereby PMs are recommended to receive an alternative analgesic drug due to the suboptimal or lack of effect of tramadol.[6](#cts13698-bib-0006) Additionally, they somehow also support the dose increase suggested by the DPWG, as a higher dose may potentially lead to higher tramadol+M1 concentrations. However, in the present work, although CYP2D6 PMs indeed showed lower M1/DW exposure, only a tendency toward lower effectiveness was observed at 30 min, which disappeared at 120 min. This suggests that although M1 is an active metabolite and contributes to tramadol effectiveness, the parent drug should be considered, and tramadol+M1 concentrations (i.e., the sum of the active moieties) may be a better predictor of tramadol effectiveness and safety. In this regard, CYP2B6 mainly and CYP3A4 to a lesser extent may be clinically relevant as both contribute to tramadol+M1 concentrations by transforming tramadol into M2. Furthermore, M1 is reportedly more potent at the μ‐opioid receptor, but tramadol is more active at inhibiting the serotonin and norepinephrine reuptake transporters.[25](#cts13698-bib-0025) Based on our data, the non‐opiodergic pharmacodynamics of tramadol may be involved in pain reduction more than expected.

Interestingly, CYP2D6 UMs presented a higher incidence of ADRs than other phenotypes, although this association did not reach the threshold for significance. A high concentration of M1 shortly after drug administration could explain this observation. However, this finding may change with long‐term administration of tramadol. These results support CYP2D6 as an important predictor of drug safety, but also of drug effectiveness (indirectly), which is consistent with CPIC/DPWG guidelines and the literature published to date. Moreover, these results support CPIC guidelines where Ums are discouraged to use tramadol to avoid the risk of ADRs with label‐recommended dosing.[12](#cts13698-bib-0012)

### Limitations

This study has several limitations. First, tramadol (parent drug) levels could not be analyzed, since the blood tubes for pharmacokinetic determinations were collected from the same i.v. access through which tramadol was administered; consequently, tramadol+M1 concentrations could not be analyzed either. Furthermore, not having tramadol levels complicates the interpretation of the associations with M1 levels; however, CYP2D6 phenotype is a well‐known predictor of M1 levels, and we were able to observe this association, which suggests that our model is adequate. In addition, because of the high LLOQ of the analytical method for M2 determination, most patients (81%) showed analyte abundances below the LLOQ and, therefore, M2 concentrations could not be accounted for. In contrast, the LLOQ for M1 was significantly lower, and since M1 is active and M2 is not, this limitation does not seem that meaningful. Second, the sample size was limited and arbitrary. It would be desirable to increase the sample size to enrich our population in carriers of low‐prevalence genotypes (e.g., CYP3A4 PMs), which would improve the statistical power of the study. Third, the mechanisms of enzyme induction and inhibition may significantly differ from patients with long‐term tramadol treatment. This could mean that some of the associations observed here may not be observed in patients with chronic tramadol treatment, or that some of the associations occurring in such patients may not have been observed clearly in our work. Finally, the enantiomers of tramadol and their specific metabolism by CYP enzymes, as well as their specific pharmacodynamics, comprise an additional layer of complexity that we could not control for as we did not measure or analyze enantiomers but the racemic mixtures.

## CONCLUSIONS

Today, *CYP2D6* is considered the only clinically relevant pharmacogenetic biomarker for tramadol prescription. However, in this work, CYP2B6 genotype‐informed phenotype was related to drug effectiveness and safety, while CYP2D6 phenotype had only a modest effect on these variables. The lack of agreement between CPIC and DPWG guidelines on their recommendations for tramadol prescription may be explained as they only consider CYP2D6 genotype‐informed phenotype. Our results suggest that CYP2B6 phenotype may be more relevant than previously thought; following the validation of these results in large, independent cohorts, CYP2B6 phenotype information could be used in combination with CYP2D6 phenotype to inform routine pharmacotherapy. Additional research is required to clarify the impact of *CYP3A4* genetic variation on tramadol response.

## AUTHOR CONTRIBUTIONS

A.C., P.S.‐C., P.Z., and F.A.‐S. wrote the manuscript, designed the research, and analyzed the data. A.C., P.S.‐C., P.Z., E.A., M.S.‐R., D.C., A.G.‐F., G.V.‐G., M.N.‐G., L.G., C.A., G.M.‐A., D.O., and F.A.‐S. performed the research. F.A.‐S. contributed new reagents and analytical tools.

## FUNDING INFORMATION

P.S.‐C. is financed by Universidad Autónoma de Madrid (FPI‐UAM, 2021). G.V.‐G. is co‐financed by Instituto de Salud Carlos III (ISCIII) and the European Social Fund “Investing in Your Future” (PFIS predoctoral grant, number FI20/00090). M.N.‐G. is financed by the ICI20/00131 grant, Acción Estratégica en Salud 2017–2020, ISCIII. P.Z. is financed by Universidad Autónoma de Madrid, Margarita Salas contract, grants for the requalification of the Spanish university system. M.S.‐R. contract was supported by Instituto de Salud Carlos III (ISCIII), Spanish Ministry of Science and Innovation, through the Sara Borrell Program (CD21/00022).

## CONFLICT OF INTEREST STATEMENT

F.A.‐S. and D.O. have been consultants or investigators in clinical trials sponsored by the following pharmaceutical companies: Abbott, Alter, Chemo, Cinfa, FAES, Farmalíder, Ferrer, GlaxoSmithKline, Galenicum, Gilead, Italfarmaco, Janssen‐Cilag, Kern, MSD, Moderna, Normon, Novartis, Servier, Silverpharma, Teva, and Zambon. All the other authors declared no competing interests for this work.

## Supporting information

Casajús A, Zubiaur P, Alday E, et al. Impact of CYP2D6 and CYP2B6 phenotypes on the response to tramadol in patients with acute post‐surgical pain. Clin Transl Sci. 2024;17:e13698. doi: 10.1111/cts.13698

## Contributor Information

Pablo Zubiaur, Email: pablo.zubiaur@uam.es.

Francisco Abad‐Santos, Email: francisco.abad@uam.es.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Grond S, Sablotzki A. Clinical pharmacology of tramadol. Clin Pharmacokinetics. 2004;43:879‐923.  [DOI](https://doi.org/10.2165/00003088-200443130-00004) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15509185/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacokinetics&title=Clinical%20pharmacology%20of%20tramadol&author=S%20Grond&author=A%20Sablotzki&volume=43&publication_year=2004&pages=879-923&pmid=15509185&doi=10.2165/00003088-200443130-00004&)

2. Gong L, Stamer UM, Tzvetkov MV, Altman RB, Klein TE. PharmGKB summary: tramadol pathway. Pharmacogenet Genomics. 2014;24:374‐380.  [DOI](https://doi.org/10.1097/FPC.0000000000000057) | [PMC free article](/articles/PMC4100774/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24849324/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=PharmGKB%20summary:%20tramadol%20pathway&author=L%20Gong&author=UM%20Stamer&author=MV%20Tzvetkov&author=RB%20Altman&author=TE%20Klein&volume=24&publication_year=2014&pages=374-380&pmid=24849324&doi=10.1097/FPC.0000000000000057&)

3. Poulsen L, Arendt‐Nielsen L, Brøsen K, Sindrup SH. The hypoalgesic effect of tramadol in relation to CYP2D6*. Clin Pharmacol Ther. 1996;60:636‐644.  [DOI](https://doi.org/10.1016/S0009-9236(96)90211-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8988065/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=The%20hypoalgesic%20effect%20of%20tramadol%20in%20relation%20to%20CYP2D6*&author=L%20Poulsen&author=L%20Arendt%E2%80%90Nielsen&author=K%20Br%C3%B8sen&author=SH%20Sindrup&volume=60&publication_year=1996&pages=636-644&pmid=8988065&doi=10.1016/S0009-9236(96)90211-8&)

4. Enggaard TP, Poulsen L, Arendt‐Nielsen L, Brøsen K, Ossig J, Sindrup SH. The analgesic effect of tramadol after intravenous injection in healthy volunteers in relation to CYP2D6. Anesth Analg. 2006;102:146‐150.  [DOI](https://doi.org/10.1213/01.ane.0000189613.61910.32) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16368820/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anesth%20Analg&title=The%20analgesic%20effect%20of%20tramadol%20after%20intravenous%20injection%20in%20healthy%20volunteers%20in%20relation%20to%20CYP2D6&author=TP%20Enggaard&author=L%20Poulsen&author=L%20Arendt%E2%80%90Nielsen&author=K%20Br%C3%B8sen&author=J%20Ossig&volume=102&publication_year=2006&pages=146-150&pmid=16368820&doi=10.1213/01.ane.0000189613.61910.32&)

5. Kang J‐S, Lee M‐H. Overview of therapeutic drug monitoring. Korean J Intern Med. 2009;24:1‐10.  [DOI](https://doi.org/10.3904/kjim.2009.24.1.1) | [PMC free article](/articles/PMC2687654/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19270474/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Korean%20J%20Intern%20Med&title=Overview%20of%20therapeutic%20drug%20monitoring&author=J%E2%80%90S%20Kang&author=M%E2%80%90H%20Lee&volume=24&publication_year=2009&pages=1-10&pmid=19270474&doi=10.3904/kjim.2009.24.1.1&)

6. Crews KR, Monte AA, Huddart R, et al. Clinical pharmacogenetics implementation consortium guideline forCYP2D6,OPRM1, and COMTGenotypes and select opioid therapy. Clin Pharmacol Ther. 2021;110:888‐896.  [DOI](https://doi.org/10.1002/cpt.2149) | [PMC free article](/articles/PMC8249478/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33387367/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Clinical%20pharmacogenetics%20implementation%20consortium%20guideline%20forCYP2D6,OPRM1,%20and%20COMTGenotypes%20and%20select%20opioid%20therapy&author=KR%20Crews&author=AA%20Monte&author=R%20Huddart&volume=110&publication_year=2021&pages=888-896&pmid=33387367&doi=10.1002/cpt.2149&)

7. Matic M, Nijenhuis M, Soree B, et al. Dutch pharmacogenetics working group (DPWG) guideline for the gene–drug interaction between CYP2D6 and opioids (codeine, tramadol and oxycodone). Eur J Hum Genet. 2021;30:1105‐1113. doi: 10.1038/s41431-021-00920-y  [DOI](https://doi.org/10.1038/s41431-021-00920-y) | [PMC free article](/articles/PMC9553935/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34267337/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Hum%20Genet&title=Dutch%20pharmacogenetics%20working%20group%20(DPWG)%20guideline%20for%20the%20gene%E2%80%93drug%20interaction%20between%20CYP2D6%20and%20opioids%20(codeine,%20tramadol%20and%20oxycodone)&author=M%20Matic&author=M%20Nijenhuis&author=B%20Soree&volume=30&publication_year=2021&pages=1105-1113&pmid=34267337&doi=10.1038/s41431-021-00920-y&)

8. Zubiaur P, Mejía‐Abril G, Navares‐Gómez M, et al. PriME‐PGx: La Princesa University hospital multidisciplinary initiative for the implementation of pharmacogenetics. JCMM. 2021;10:3772.  [DOI](https://doi.org/10.3390/jcm10173772) | [PMC free article](/articles/PMC8432257/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34501219/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JCMM&title=PriME%E2%80%90PGx:%20La%20Princesa%20University%20hospital%20multidisciplinary%20initiative%20for%20the%20implementation%20of%20pharmacogenetics&author=P%20Zubiaur&author=G%20Mej%C3%ADa%E2%80%90Abril&author=M%20Navares%E2%80%90G%C3%B3mez&volume=10&publication_year=2021&pages=3772&pmid=34501219&doi=10.3390/jcm10173772&)

9. World Medical Association Declaration of Helsinki: Ethical principles for medical research involving human subjects. JAMA. 2013;310:2191.  [DOI](https://doi.org/10.1001/jama.2013.281053) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24141714/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA&title=World%20Medical%20Association%20Declaration%20of%20Helsinki:%20Ethical%20principles%20for%20medical%20research%20involving%20human%20subjects&volume=310&publication_year=2013&pages=2191&pmid=24141714&doi=10.1001/jama.2013.281053&)

10. Vijayananthan A, Nawawi O. The importance of good clinical practice guidelines and its role in clinical trials. Biomed Imaging Interv J. 2008;4:1‐4.  [DOI](https://doi.org/10.2349/biij.4.1.e5) | [PMC free article](/articles/PMC3097692/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21614316/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biomed%20Imaging%20Interv%20J&title=The%20importance%20of%20good%20clinical%20practice%20guidelines%20and%20its%20role%20in%20clinical%20trials&author=A%20Vijayananthan&author=O%20Nawawi&volume=4&publication_year=2008&pages=1-4&pmid=21614316&doi=10.2349/biij.4.1.e5&)

11. Food and Drug Administration (FDA) . Bioanalytical Method Validation. Guidance for Industry; 2018. https://www.fda.gov/media/70858/download  [https://www.fda.gov/media/70858/download](https://www.fda.gov/media/70858/download%3e) | [Google Scholar](https://scholar.google.com/scholar_lookup?title=Bioanalytical%20Method%20Validation&publication_year=2018&)

12. Aguirre C, García M. Evaluación de la causalidad en las comunicaciones de reacciones adversas a medicamentos. Algoritmo del Sistema Español de Farmacovigilancia. Med Clin. 2016;147:461‐464.  [DOI](https://doi.org/10.1016/j.medcli.2016.06.012) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27450163/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Med%20Clin&title=Evaluaci%C3%B3n%20de%20la%20causalidad%20en%20las%20comunicaciones%20de%20reacciones%20adversas%20a%20medicamentos.%20Algoritmo%20del%20Sistema%20Espa%C3%B1ol%20de%20Farmacovigilancia&author=C%20Aguirre&author=M%20Garc%C3%ADa&volume=147&publication_year=2016&pages=461-464&pmid=27450163&doi=10.1016/j.medcli.2016.06.012&)

13. Belmonte C, Ochoa D, Román M, et al. Influence of CYP2D6, CYP3A4, CYP3A5 and ABCB1 polymorphisms on pharmacokinetics and safety of aripiprazole in healthy volunteers. Basic Clin Pharmacol Toxicol. 2018;122:596‐605.  [DOI](https://doi.org/10.1111/bcpt.12960) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29325225/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Basic%20Clin%20Pharmacol%20Toxicol&title=Influence%20of%20CYP2D6,%20CYP3A4,%20CYP3A5%20and%20ABCB1%20polymorphisms%20on%20pharmacokinetics%20and%20safety%20of%20aripiprazole%20in%20healthy%20volunteers&author=C%20Belmonte&author=D%20Ochoa&author=M%20Rom%C3%A1n&volume=122&publication_year=2018&pages=596-605&pmid=29325225&doi=10.1111/bcpt.12960&)

14. Desta Z, Gammal RS, Gong L, et al. Clinical pharmacogenetics implementation consortium (CPIC) guideline for CYP2B6 and efavirenz‐containing antiretroviral therapy. Clin Pharmacol Ther. 2019;106:726‐733.  [DOI](https://doi.org/10.1002/cpt.1477) | [PMC free article](/articles/PMC6739160/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31006110/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Clinical%20pharmacogenetics%20implementation%20consortium%20(CPIC)%20guideline%20for%20CYP2B6%20and%20efavirenz%E2%80%90containing%20antiretroviral%20therapy&author=Z%20Desta&author=RS%20Gammal&author=L%20Gong&volume=106&publication_year=2019&pages=726-733&pmid=31006110&doi=10.1002/cpt.1477&)

15. Lee CR, Luzum JA, Sangkuhl K, et al. Clinical pharmacogenetics implementation consortium guideline for CYP2C19 genotype and clopidogrel therapy: 2022 update. Clin Pharma Therap. 2022;cpt.2526:959‐967. doi: 10.1002/cpt.2526  [DOI](https://doi.org/10.1002/cpt.2526) | [PMC free article](/articles/PMC9287492/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35034351/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharma%20Therap&title=Clinical%20pharmacogenetics%20implementation%20consortium%20guideline%20for%20CYP2C19%20genotype%20and%20clopidogrel%20therapy:%202022%20update&author=CR%20Lee&author=JA%20Luzum&author=K%20Sangkuhl&volume=cpt.2526&publication_year=2022&pages=959-967&pmid=35034351&doi=10.1002/cpt.2526&)

16. Goetz MP, Sangkuhl K, Guchelaar HJ, et al. Clinical pharmacogenetics implementation consortium (CPIC) guideline for CYP2D6 and tamoxifen therapy. Clin Pharmacol Ther. 2018;103:770‐777.  [DOI](https://doi.org/10.1002/cpt.1007) | [PMC free article](/articles/PMC5931215/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29385237/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Clinical%20pharmacogenetics%20implementation%20consortium%20(CPIC)%20guideline%20for%20CYP2D6%20and%20tamoxifen%20therapy&author=MP%20Goetz&author=K%20Sangkuhl&author=HJ%20Guchelaar&volume=103&publication_year=2018&pages=770-777&pmid=29385237&doi=10.1002/cpt.1007&)

17. Birdwell K, Decker B, Barbarino JM, et al. Clinical pharmacogenetics implementation consortium (CPIC) guidelines for CYP3A5 genotype and tacrolimus dosing. Clin Pharmacol Ther. 2015;98:19‐24.  [DOI](https://doi.org/10.1002/cpt.113) | [PMC free article](/articles/PMC4481158/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25801146/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Clinical%20pharmacogenetics%20implementation%20consortium%20(CPIC)%20guidelines%20for%20CYP3A5%20genotype%20and%20tacrolimus%20dosing&author=K%20Birdwell&author=B%20Decker&author=JM%20Barbarino&volume=98&publication_year=2015&pages=19-24&pmid=25801146&doi=10.1002/cpt.113&)

18. Ramsey LB, Johnson SG, Caudle KE, et al. The clinical pharmacogenetics implementation consortium guideline for SLCO1B1 and simvastatin‐induced myopathy: 2014 update. Clin Pharmacol Ther. 2014;96:423‐428.  [DOI](https://doi.org/10.1038/clpt.2014.125) | [PMC free article](/articles/PMC4169720/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24918167/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=The%20clinical%20pharmacogenetics%20implementation%20consortium%20guideline%20for%20SLCO1B1%20and%20simvastatin%E2%80%90induced%20myopathy:%202014%20update&author=LB%20Ramsey&author=SG%20Johnson&author=KE%20Caudle&volume=96&publication_year=2014&pages=423-428&pmid=24918167&doi=10.1038/clpt.2014.125&)

19. Theken KN, Lee CR, Gong L, et al. Clinical pharmacogenetics implementation consortium guideline (CPIC) for CYP2C9 and nonsteroidal anti‐inflammatory drugs. Clin Pharmacol Ther. 2020;108:191‐200.  [DOI](https://doi.org/10.1002/cpt.1830) | [PMC free article](/articles/PMC8080882/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32189324/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Clinical%20pharmacogenetics%20implementation%20consortium%20guideline%20(CPIC)%20for%20CYP2C9%20and%20nonsteroidal%20anti%E2%80%90inflammatory%20drugs&author=KN%20Theken&author=CR%20Lee&author=L%20Gong&volume=108&publication_year=2020&pages=191-200&pmid=32189324&doi=10.1002/cpt.1830&)

20. Dutch Pharmacogenetics Working Group (DPWG) CYP3A4: quetiapine. https://www.g‐standaard.nl/risicoanalyse/B0005991.PDF  [https://www.g‐standaard.nl/risicoanalyse/B0005991.PDF](https://www.g-standaard.nl/risicoanalyse/B0005991.PDF%3e)

21. Bartley EJ, Fillingim RB. Sex differences in pain: a brief review of clinical and experimental findings. Br J Anaesth. 2013;111:52‐58.  [DOI](https://doi.org/10.1093/bja/aet127) | [PMC free article](/articles/PMC3690315/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23794645/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Anaesth&title=Sex%20differences%20in%20pain:%20a%20brief%20review%20of%20clinical%20and%20experimental%20findings&author=EJ%20Bartley&author=RB%20Fillingim&volume=111&publication_year=2013&pages=52-58&pmid=23794645&doi=10.1093/bja/aet127&)

22. Saiz‐Rodríguez M, Ochoa D, Román M, et al. Involvement of CYP2D6 and CYP2B6 on tramadol pharmacokinetics. Pharmacogenomics. 2020;21:663‐675.  [DOI](https://doi.org/10.2217/pgs-2020-0026) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32538291/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Involvement%20of%20CYP2D6%20and%20CYP2B6%20on%20tramadol%20pharmacokinetics&author=M%20Saiz%E2%80%90Rodr%C3%ADguez&author=D%20Ochoa&author=M%20Rom%C3%A1n&volume=21&publication_year=2020&pages=663-675&pmid=32538291&doi=10.2217/pgs-2020-0026&)

23. Haage P, Kronstrand R, Josefsson M, et al. Enantioselective pharmacokinetics of tramadol and its three main metabolites; impact ofCYP2D6,CYP2B6, andCYP3A4genotype. Pharmacol Res Perspect. 2018;6:e00419.  [DOI](https://doi.org/10.1002/prp2.419) | [PMC free article](/articles/PMC6034060/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29992026/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacol%20Res%20Perspect&title=Enantioselective%20pharmacokinetics%20of%20tramadol%20and%20its%20three%20main%20metabolites;%20impact%20ofCYP2D6,CYP2B6,%20andCYP3A4genotype&author=P%20Haage&author=R%20Kronstrand&author=M%20Josefsson&volume=6&publication_year=2018&pages=e00419&pmid=29992026&doi=10.1002/prp2.419&)

24. Stamer UM, Musshoff F, Kobilay M, Madea B, Hoeft A, Stuber F. Concentrations of tramadol and O‐desmethyltramadol enantiomers in different CYP2D6 genotypes. Clin Pharmacol Ther. 2007;82:41‐47.  [DOI](https://doi.org/10.1038/sj.clpt.6100152) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17361124/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Concentrations%20of%20tramadol%20and%20O%E2%80%90desmethyltramadol%20enantiomers%20in%20different%20CYP2D6%20genotypes&author=UM%20Stamer&author=F%20Musshoff&author=M%20Kobilay&author=B%20Madea&author=A%20Hoeft&volume=82&publication_year=2007&pages=41-47&pmid=17361124&doi=10.1038/sj.clpt.6100152&)

25. Smith HS. Opioid Metabolism. Mayo Clin Proc. 2009;84:613‐624.  [DOI](https://doi.org/10.1016/S0025-6196(11)60750-7) | [PMC free article](/articles/PMC2704133/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19567715/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mayo%20Clin%20Proc&title=Opioid%20Metabolism&author=HS%20Smith&volume=84&publication_year=2009&pages=613-624&pmid=19567715&doi=10.1016/S0025-6196(11)60750-7&)
