# Evaluating the Frequencies of CNOT3, GRIA1, NFATC2, and PNPLA3 Variant Alleles and Their Association with L-Asparaginase Hypersensitivity in Pediatric Acute Lymphoblastic Leukemia in Addis Ababa, Ethiopia

## Metadata
**Authors:** Awol Mekonnen Ali, Haileyesus Adam, Daniel Hailu, Rawleigh Howe, Teferra Abula, Marieke J H Coenen
**Journal:** The Application of Clinical Genetics
**Date:** 2023 Aug 2
**DOI:** [10.2147/TACG.S404695](https://doi.org/10.2147/TACG.S404695)
**PMID:** 37551203
**PMCID:** PMC10404408
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10404408/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC10404408/pdf/tacg-16-131.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC10404408/pdf/tacg-16-131.pdf)

## Abstract

**Introduction:** 
L-asparaginase is a vital component for the treatment of childhood acute lymphoblastic leukemia (ALL); however, hypersensitivity reactions and hepatotoxicity hinder its anti-neoplastic efficacy. Previous reports indicated that genetic variants in CNOT3, GRIA1, and NFATC2 genes might be associated with hypersensitivity reactions and PNPLA3 with liver function.

**Objective:** 
In this study, it was investigated whether this association also exists in a pediatric ALL cohort from Ethiopia.

**Methods:** 
Three variants GRIA1 rs4958351, CNOT3 rs73062673, and NFATC2 rs6021191 were genotyped in a cohort of 160 patients. Association analysis to investigate the association with hypersensitivity reactions was performed using logistic regression analyses. Besides these variants, a variant in PNPLA3 (rs738409) was genotyped to assess the association with liver function.

**Results:** 
Genotype frequencies of GRIA1 rs4958351, CNOT3 rs73062673, and NFATC2 rs6021191 were higher/lower than previously reported. One hundred and forty-four patients were included in the association analysis of which, 18 (12.5%) developed L-ASP hypersensitivity. Though the frequency of hypersensitivity was higher in patients that carried the risk alleles of the three investigated genes, no statistically significant differences were observed. Association analysis between PNPLA3 rs738409 and liver function could not be investigated due to a lack of clinical information.

**Conclusion:** 
In conclusion, none of the tested genes did predict L-asparaginase hypersensitivity in an Ethiopian pediatric ALL patients.

Keywords: acute lymphoblastic leukemia, L-asparaginase, hypersensitivity

### Introduction

L-asparaginase is a vital component for the treatment of childhood acute lymphoblastic leukemia (ALL); however, hypersensitivity reactions and hepatotoxicity hinder its anti-neoplastic efficacy. Previous reports indicated that genetic variants in *CNOT3, GRIA1*, and *NFATC2* genes might be associated with hypersensitivity reactions and *PNPLA3* with liver function.

### Objective

In this study, it was investigated whether this association also exists in a pediatric ALL cohort from Ethiopia.

### Methods

Three variants *GRIA1* rs4958351, *CNOT3* rs73062673, and *NFATC2* rs6021191 were genotyped in a cohort of 160 patients. Association analysis to investigate the association with hypersensitivity reactions was performed using logistic regression analyses. Besides these variants, a variant in *PNPLA3* (rs738409) was genotyped to assess the association with liver function.

### Results

Genotype frequencies of *GRIA1* rs4958351, *CNOT3* rs73062673, and *NFATC2* rs6021191 were higher/lower than previously reported. One hundred and forty-four patients were included in the association analysis of which, 18 (12.5%) developed L-ASP hypersensitivity. Though the frequency of hypersensitivity was higher in patients that carried the risk alleles of the three investigated genes, no statistically significant differences were observed. Association analysis between *PNPLA3* rs738409 and liver function could not be investigated due to a lack of clinical information.

### Conclusion

In conclusion, none of the tested genes did predict L-asparaginase hypersensitivity in an Ethiopian pediatric ALL patients.

## Introduction

L-asparaginase (L-ASP) is the first therapeutic enzyme with antineoplastic properties.[1](#cit0001) It has been the backbone of acute lymphoblastic leukemia (ALL) treatment protocol for nearly 40 years.[2](#cit0002) All pediatric regimens for the treatment of ALL and the majority of adult protocols consist of L-ASP in remission induction and intensification treatment protocols.[3](#cit0003) Currently, three L-ASP preparations are available; the native asparaginase derived from *E. coli*, a PEGylated form of this enzyme (PEG-asparaginase), and one isolated from *Erwinia chrysanthemi*.[4](#cit0004) All L-ASP preparations share the same mechanism of action.[5](#cit0005) They deplete serum asparagine levels and subsequently inhibit protein synthesis leading to cytotoxicity.[6](#cit0006)

Unfortunately, L-ASP treatment can be accompanied by side effects such as hypersensitivity, immunosuppression, hepatotoxicity, pancreatitis, and coagulation dysfunction.[7](#cit0007) Hypersensitivity is characterized as an allergic reaction with signs and symptoms consistent with an immune response to a known antigen.[8](#cit0008) Hypersensitivity reactions develop in 10–30% of the patients during L-ASP treatment.[8](#cit0008),[9](#cit0009) Factors that may affect the incidence of hypersensitivity include the L-ASP preparation, intensity of dosing, route of administration, concurrent chemotherapy, and time point in therapy.[10](#cit0010) Several studies indicate that specific genetic variants might also be linked to L-ASP hypersensitivity, eg, variants in Glutamate Ionotropic Receptor AMPA Type Subunit 1 (*GRIA1*), CCR4-NOT Transcription Complex Subunit 3 (*CNOT3*), and Nuclear Factor of Activated T Cells 2 (*NFATC2*).[11](#cit0011) The intronic variant rs4958351 in the *GRIA1* gene appeared to be a strong risk factor for L-ASP hypersensitivity both in genome-wide and candidate gene studies.[12](#cit0012),[13](#cit0013) Another genome-wide association study (GWAS) led to the identification of an association between L-ASP hypersensitivity and a SNP on chromosome 19 (rs73062673) in the non-coding region close to *CNOT3*. This gene regulates the transcription of *HLA* genes.[14](#cit0014) The *NFATC2* rs6021191 variant has been found to be significantly associated with the risk of developing L-ASP hypersensitivity in children with ALL.[15](#cit0015) Mice studies show that *NFATC2* knockout significantly reduces the risk of developing L-ASP hypersensitivity.[16](#cit0016)

Another candidate gene is *PNPLA3*. Several genetic studies showed that the common nonsynonymous variant c.444C>G (rs738409) of this gene is the key genetic determinant of fatty liver disease severity in pediatric and adult patients.[17](#cit0017),[18](#cit0018) In a study focusing on patients with pediatric ALL, this variant showed an association with hepatotoxicity during the induction phase of ALL therapy.[19](#cit0019)

So far, GWAS and candidate gene studies have identified several genes that could be associated with L-ASP hypersensitivity reactions and hepatotoxicity. To the best of our knowledge, there have been no previous reports of the frequency of variants that are linked to L-ASP toxicity in Ethiopia, with a population of over 100 million people. Therefore, the present study was designed to evaluate the frequency of variants that are previously linked to L-ASP hypersensitivity and liver toxicity in a cohort of pediatric ALL patients from Ethiopia. In addition, it was investigated if the variants are associated with hypersensitivity reactions.

## Method

### Patients and Treatment Protocol

This study was conducted at Tikur Anbessa specialized referral teaching hospital, Addis Ababa University, Addis Ababa, Ethiopia. A total of 160 pediatric ALL patients were enrolled. ALL patients are stratified into standard risk (SR), intermediate risk (IR), and high-risk group (HR) based on physical examination, age, initial white blood cell count (WBC), central nervous system status, and early prednisolone response. Informed consent was obtained from all participants’ caregivers before study enrolment. Patients with liver and kidney problems were excluded. This study was conducted in accordance with the Declaration of Helsinki. The study protocol was approved by the Institutional Review Board of the College of Health Sciences, Addis Ababa University (AAU) (021/18/Pharma), the Armauer Hansen Research Institute Ethical Review Committee (PO51/18), and the Ethiopian National Research Ethics Review Committee.

### Asparaginase in ALL Protocol

Patients were treated according to the standard ALL treatment protocol for low and middle income countries.[20](#cit0020) Intravenous (IV) native *E. coli* L-ASP is part of the induction, consolidation, and delayed intensification phases of this treatment protocol. It is administered nine times at a dose of 6000 U/m^2^ both in the induction and delayed intensification phase irrespective of the risk group. Patients in the HR group receive additional L-ASP at a dose of 10,000 U/m^2^ for eight times in the consolidation phase of the protocol.

### L-ASP Hypersensitivity Phenotyping

The baseline data, demographic, clinical presentation during diagnosis, complete blood count (CBC) at diagnosis, peripheral morphology, peripheral and bone marrow blast, liver and kidney function test, risk group, and L-ASP hypersensitivity reaction were collected from medical records.

### Sample and Genotyping

EDTA whole blood samples were collected from ALL patients aged less than 12 years old at the time of diagnosis. Genomic DNA was isolated from peripheral leukocytes in 1mL of whole blood using QIAamp Blood Midi Kit (Qiagen GmbH, Hilden, Germany) following the manufacturer’s instructions. The DNA quality was assessed by gel electrophoresis and NanoDrop™ ND-2000c Spectrophotometer (Thermo Scientific, Isogen, the Netherlands). Genotyping of SNPs rs4958351 in *GRIA1* and rs738409 in *PNPLA3* was performed using a KASPar-On-Demand (KOD) assay (LGC Genomics, Hoddesdon, UK) as described previously with little modification.[21](#cit0021) The final volume for each reaction was 5 µL containing 1 µL of DNA (10 ng/µL), 2.5 µL of KASP 5000 V4.0 Low ROX (2x; LGC Genomics), 0.0625 µL of the KASPar assay (40x), and 1.44 µL of MilliQ grade water. The PCR conditions consisted of an initial denaturation at 94°C for 15 min, followed by 10 cycles of denaturation at 94°C for 20s and annealing/extension at 61°C for 60s, including a drop of 0.6°C for each cycle. This was followed by 26 cycles of denaturation at 94°C for 10s and annealing/extension at 55°C for 60s, followed by 12 cycles of denaturation at 94°C for 20s and annealing/extension at 57°C for 60s. *CNOT3* rs73062673 and *NFATC2* rs6021191 variants were genotyped using TaqMan^®^ SNP Genotyping Assays (Assay ID number C__98092291__20 for rs73062673, C_29689803_10 for rs6021191) as described before.[22](#cit0022) The final reaction volume of 5 µL was prepared by mixing 1 µL of DNA (10 ng/µL), 2.5 µL of TaqMan^®^ Universal PCR Master Mix (2x; Applied Biosystems by Thermo Fisher scientific, Warrington, UK), 0.0625 µL of TaqMan^®^ SNP Genotyping Assay (40x; Applied Biosystems by Thermo Fisher Scientific), and 1.44 µL of MilliQ grade water. The PCR conditions included an initial stage at 95°C for 12 min and followed by 50 cycles with step 1 at 92°C for 15s followed by step 2 with 60°C for 90s. MilliQ grade water and positive controls were included for quality control in both TaqMan^®^ and KASP assay. The genomic DNA samples were amplified in Veriti™ 96-well Fast Thermal Cycler PCR (Applied Biosystems, Singapore) and results were analyzed using QuantStudio™ 3 v1.5. software (Applied Biosystems by Thermo Fisher Scientific, Singapore).

### Statistical Analysis

Observed minor allele frequencies in this cohort were compared with minor allele frequencies reported in dbSNP. All statistical analyses were conducted using SPSS version 26. The Hardy-Weinberg equilibrium was assessed using the chi-square test. Demographic characteristics, clinical profiles, and genotype frequencies of participants were analyzed using descriptive statistics. Bivariable logistic regression analyses were performed to determine the association between genetic variants and asparaginase hypersensitivity. Factors previously reported to be linked with asparaginase hypersensitivity (age, gender, risk group) were included as co-variates. Tests were considered significant when the *p*-value was lower than 0.05.

## Results

### Incidence of Asparaginase Hypersensitivity

The basic characteristics of the enrolled patients are shown in [Table 1](#t0001). A total of 160 patients participated in this study, of which 144 (90%) were included in the final L-ASP hypersensitivity association analysis as data on (possible) hypersensitivity was missing for 16 patients. Fifty (34.7%) of the patients were girls, a bit more than half of the patients presented with hepatomegaly (59.7%) and splenomegaly (54.7%) in isolation or combination. One hundred and twenty-two (84.7%) of the study participants were 10 years and younger. 53.5% of the patients belonged to the high-risk group and 46.5% were assigned to the standard risk group. Eighteen (12.5%) of the patients included in the final L-ASP hypersensitivity association analysis experienced hypersensitivity reactions.

### Table 1.

|   |  | n (%) |
| --- | --- | --- |
| Child’s sex | Male | 94 (65.3) |
| Female | 50 (34.7) |  |
| Child’s age (years) | ≤10 | 122 (84.7) |
| >10 | 22 (15.3) |  |
| Hepatosplenomegaly | Hepatomegaly | 83 (59.7) |
| Splenomegaly | 76 (54.7) |  |
| Risk group | Standard risk | 67 (46.5) |
| High risk | 77 (53.5) |  |
| L-ASP hypersensitivity | Yes | 18 (12.5) |

Table 1 Caption: Socio-Demographic Characteristics and Frequency of Asparaginase Hypersensitivity (N = 144)

### Allele Frequencies

Genotypes were determined for all patients included in the study (n = 160) ([Table 2](#t0002)). *GRIA1* rs4958351 minor allele frequency (17.8%) is low compared to African general frequency (25.2% dbSNP). The minor allele frequency (6.9%) of *CNOT3* rs73062673 is a bit higher as compared to the frequencies of the African population reported in dbSNP (4.6%). A low minor allele frequency (5.6%) of *NFATC2* rs6021191 is observed in the present study compared to African general frequency (15.2% dbSNP). The minor allele frequency (16.9%) of *PNPLA3* rs738409 is higher compared to the reported African general allele frequency (8.9% dbSNP).

### Table 2.

|   | Genotype | Genotype Frequency (%) | Minor Allele Frequency of this Study | dbSNP Minor Allele Frequency (African Population) |
| --- | --- | --- | --- | --- |
| GRIA1 rs4958351 | GG | 108 (67.5) | 17.8% (Allele A) | 25.2% (Allele A) |
| GA | 47 (29.4) |  |  |  |
| AA | 5 (3.1) |  |  |  |
| CNOT3 rs73062673 | TT | 140 (87.5) | 6.9% (Allele C) | 4.6% (Allele C) |
| TC | 18 (11.3) |  |  |  |
| CC | 2 (1.3) |  |  |  |
| NFATC2 rs6021191 | AA | 142 (88.8) | 5.6% (Allele T) | 15.2% (Allele T) |
| AT | 18 (11.3) |  |  |  |
| PNPLA3 rs738409 | CC | 111 (69.4) | 16.9% (Allele G) | 8.9% (Allele G) |
| CG | 44 (27.5) |  |  |  |
| GG | 5 (3.1) |  |  |  |

Table 2 Caption: Genotype and Allele Frequencies of Candidate Gene SNP (N = 160)

### Association Analysis for L-ASP Hypersensitivity

Association analysis for L-ASP hypersensitivity was performed for 144 patients with complete information on hypersensitivity. Bivariable logistic regression analyses revealed that none of the patient profiles (age, gender, and risk group) or genetic factors showed a significant association with L-ASP hypersensitivity. A comparison of L-ASP hypersensitivity frequency in relation to the patient’s genotype is presented in [Figure 1](#f0001). The frequency of L-ASP hypersensitivity is higher in patients with a variant allele compared to a patient without a variant allele, though not statistically significant. Due to a lack of clinical data on hepatotoxicity, no association analysis was performed for *PNPLA3* rs738409.

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/02e8/10404408/4a366eab3483/TACG-16-131-g0001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=10404408_TACG-16-131-g0001.jpg)

Frequency of L-ASP hypersensitivity in relation to the investigated SNPs. For each SNP the reference genotype is depicted in the first bar. (A) GRIA1 rs4958351 (B) CNOT3 rs73062673 (C) NFATC2 rs6021191.

## Discussion

Patients who develop L-ASP hypersensitivity reactions have increased L-ASP clearance leading to suboptimal concentrations of the drug in the serum.[23](#cit0023),[24](#cit0024) Failure to receive the full course of L-ASP treatment due to toxicities has been linked with poor outcomes in ALL.[25](#cit0025),[26](#cit0026) We investigated genetic variants in genes previously associated with L-ASP hypersensitivity.

The incidence of L-ASP hypersensitivity in this cohort (12.5%) is very low compared to incidence reports from Spain (55%),[27](#cit0027) America (41%),[12](#cit0012),[28](#cit0028) and Slovenia (49.3%).[13](#cit0013) There is no clear explanation for this discrepancy; however, the frequency of L-ASP hypersensitivity has been associated with the L-ASP preparation, intensity and consistency of dosing, route of administration, concurrent chemotherapy, a time point in therapy, racial ancestry, and patient genetics.[10](#cit0010),[12](#cit0012),[15](#cit0015) Racial ancestry may play a major role for lower incidence of L-ASP hypersensitivity, as the rate of hypersensitivity is lower in the black population.

In this study, previously reported genetic variants in *GRIA1, CNOT3, NFATC2*, and *PNPLA3* were investigated. Lower allele frequencies as compared to the general African population were observed for the variants in *GRIA1* and *NFATC2*. In addition, the minor allele frequency of *GRIA1* rs4958351 (17.8% in this study) is lower than reported in America (27%),[12](#cit0012) Hungary (34%),[29](#cit0029) and Caucasian (36.9%).[13](#cit0013) The minor allele frequency of *NFATC2* rs6021191 (5.6%) was higher than a study performed in the USA (2.6%) but lower than the minor allele frequency of *NFATC2* rs6021191 in African Americans (14.2%).[15](#cit0015) Higher allele frequencies of *CNOT3* and *PNPLA3* were observed as compared to the general African population. Besides, the minor allele frequency of *CNOT3* rs73062673 was lower than previous findings (6.9% versus 10%).[14](#cit0014) Also, the minor allele frequency of *PNPLA3* rs738409 (16.9%) was lower compared to those from other studies done in North India (29.7%),[30](#cit0030) Germany (31.3%),[31](#cit0031) and Brazil (43.3%).[32](#cit0032) However, the minor allele frequency was higher than that of African Americans (13.9%).[33](#cit0033)

Due to a lack of complete clinical data, no association analysis was conducted for *PNPLA3*. However, several previous studies showed that this SNP was significantly associated with nonalcoholic fatty liver disease,[33](#cit0033),[34](#cit0034) and L-ASP induced liver toxicity.[19](#cit0019) The associations of *GRIA1* rs4958351, *CNOT3* rs73062673, and *NFATC2* rs602119 with L-ASP hypersensitivity were not replicated in this cohort. GWAS study by Chen et al,[12](#cit0012) and candidate gene studies by Rajić et al,[13](#cit0013) and Fernandez et al,[15](#cit0015) identified a significantly higher incidence of L-ASP hypersensitivity in patients with *GRIA1* rs4958351 risk allele carrier. However, our study is in line with a report by Kutszegi et al,[29](#cit0029) who also showed no association between *GRIA1* rs4958351 and *E. coli*-ASP hypersensitivity. The first study showing association between *CNOT3* and PEG-asparaginase hypersensitivity was a genome-wide association study.[14](#cit0014) To the best of our knowledge, we are the first to perform a replication study, unfortunately, we were unable to show association between the *CNOT3* SNP rs73062673 and L-ASP hypersensitivity. In a previous study, Fernandez et al[15](#cit0015) found that the rs6021191 in the *NFATC2* was associated with L-ASP hypersensitivity. But, this is not replicated in the present study. Likewise, no association was found between *NFATC2* rs6021191 and PEG-asparaginase hypersensitivity in a study done by Liu et al.[35](#cit0035)

In general, it is interesting to note that the frequency of L-ASP hypersensitivity is higher in variant carriers compared to non-risk allele carriers for all three genotyped SNPs. Maybe an association might still exist; however, a larger population will be necessary to prove this.

Non-genetic variables like gender, age, racial ancestry, and risk arm have previously been associated with the risk of L-ASP hypersensitivity.[12](#cit0012),[15](#cit0015) But other reports indicate that age and gender were not linked to L-ASP hypersensitivity.[13](#cit0013) The present study also investigated non-genetic factors (age, gender, risk group) for association with L-ASP hypersensitivity. We were unable to identify an association with these factors. In the current study, a possible reason for the lack of association between the predicting factors and L-ASP hypersensitivity could be a lesser incidence of the hypersensitivity reactions compounded by a smaller sample size.

The major limitation of this study is the small sample size although it is the largest group of pediatric ALL patients from Ethiopia. Unfortunately, the data on liver function were largely lacking and therefore we could not address all research questions. Besides, it might be of interest to know the asparaginase enzyme activity as this would allow for detecting subclinical L-ASP hypersensitivity. The strength of our study is that we investigated a group of patients from a large African country for which knowledge on allele frequencies and information on whether previously reported associations are translatable to this population is often missing.

## Conclusion

In conclusion, the incidence of L-ASP hypersensitivity among pediatric ALL patients in Tikur Anbessa, Addis Ababa, Ethiopia is low. In the current study, there was no significant association between hypersensitivity and three previously investigated candidate genes, though the frequency of hypersensitivity is high in patients carrying the risk allele.

## Acknowledgments

We thank the study participants and staff members of cancer center of Tikur Anbessa specialized hospital and Armauer Hansen Research Institute for their support during the study period. We would like to extend our sincere appreciation to the staff members of RadboudUMC. The current address for Marieke JH Coenen is Erasmus Medical Center, Erasmus University, Rotterdam, Netherlands.

## Disclosure

The authors report no conflicts of interest in this work.

## References

1. Batool T, Makky EA, Jalal M, Yusoff MM. A comprehensive review on l-asparaginase and its applications. Appl Biochem Biotechnol. 2016;178(5):900–923. doi: 10.1007/s12010-015-1917-3  [DOI](https://doi.org/10.1007/s12010-015-1917-3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26547852/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Appl%20Biochem%20Biotechnol&title=A%20comprehensive%20review%20on%20l-asparaginase%20and%20its%20applications&author=T%20Batool&author=EA%20Makky&author=M%20Jalal&author=MM%20Yusoff&volume=178&issue=5&publication_year=2016&pages=900-923&pmid=26547852&doi=10.1007/s12010-015-1917-3&)

2. Fu CH, Sakamoto KM. PEG-asparaginase. Expert Opin Pharmacother. 2007;8(12):1977–1984. doi: 10.1517/14656566.8.12.1977  [DOI](https://doi.org/10.1517/14656566.8.12.1977) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17696798/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Expert%20Opin%20Pharmacother&title=PEG-asparaginase&author=CH%20Fu&author=KM%20Sakamoto&volume=8&issue=12&publication_year=2007&pages=1977-1984&pmid=17696798&doi=10.1517/14656566.8.12.1977&)

3. Pieters R, Hunger SP, Boos J, et al. L-asparaginase treatment in acute lymphoblastic leukemia: a focus on Erwinia asparaginase. Cancer. 2011;117(2):238–249. doi: 10.1002/cncr.25489  [DOI](https://doi.org/10.1002/cncr.25489) | [PMC free article](/articles/PMC3000881/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20824725/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer&title=L-asparaginase%20treatment%20in%20acute%20lymphoblastic%20leukemia:%20a%20focus%20on%20Erwinia%20asparaginase&author=R%20Pieters&author=SP%20Hunger&author=J%20Boos&volume=117&issue=2&publication_year=2011&pages=238-249&pmid=20824725&doi=10.1002/cncr.25489&)

4. Brumano LP, da Silva FVS, Costa-Silva TA, et al. Development of L-asparaginase biobetters: current research status and review of the desirable quality profiles. Front Bioeng Biotechnol. 2019;6:212. doi: 10.3389/fbioe.2018.00212  [DOI](https://doi.org/10.3389/fbioe.2018.00212) | [PMC free article](/articles/PMC6335324/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30687702/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front%20Bioeng%20Biotechnol&title=Development%20of%20L-asparaginase%20biobetters:%20current%20research%20status%20and%20review%20of%20the%20desirable%20quality%20profiles&author=LP%20Brumano&author=FVS%20da%20Silva&author=TA%20Costa-Silva&volume=6&publication_year=2019&pages=212&pmid=30687702&doi=10.3389/fbioe.2018.00212&)

5. Egler RA, Ahuja SP, Matloub Y. L-asparaginase in the treatment of patients with acute lymphoblastic leukemia. J Pharmacol Pharmacother. 2016;7(2):62–71. doi: 10.4103/0976-500X.184769  [DOI](https://doi.org/10.4103/0976-500X.184769) | [PMC free article](/articles/PMC4936081/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27440950/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacol%20Pharmacother&title=L-asparaginase%20in%20the%20treatment%20of%20patients%20with%20acute%20lymphoblastic%20leukemia&author=RA%20Egler&author=SP%20Ahuja&author=Y%20Matloub&volume=7&issue=2&publication_year=2016&pages=62-71&pmid=27440950&doi=10.4103/0976-500X.184769&)

6. El-Nagga NEA, El-Ewasy SM, El-Shweihy NM. Microbial L-asparaginase as a potential therapeutic agent for the treatment of acute lymphoblastic leukemia: the pros and cons. Int J Pharmacol. 2014;10(4):182–199. doi: 10.3923/ijp.2014.182.199  [DOI](https://doi.org/10.3923/ijp.2014.182.199) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Pharmacol&title=Microbial%20L-asparaginase%20as%20a%20potential%20therapeutic%20agent%20for%20the%20treatment%20of%20acute%20lymphoblastic%20leukemia:%20the%20pros%20and%20cons&author=NEA%20El-Nagga&author=SM%20El-Ewasy&author=NM%20El-Shweihy&volume=10&issue=4&publication_year=2014&pages=182-199&doi=10.3923/ijp.2014.182.199&)

7. Bade NA, Lu C, Patzke CL, et al. Optimizing pegylated asparaginase use: an institutional guideline for dosing, monitoring, and management. J Oncol Pharm Pract. 2020;26(1):74–92. doi: 10.1177/1078155219838316  [DOI](https://doi.org/10.1177/1078155219838316) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30917738/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Oncol%20Pharm%20Pract&title=Optimizing%20pegylated%20asparaginase%20use:%20an%20institutional%20guideline%20for%20dosing,%20monitoring,%20and%20management&author=NA%20Bade&author=C%20Lu&author=CL%20Patzke&volume=26&issue=1&publication_year=2020&pages=74-92&pmid=30917738&doi=10.1177/1078155219838316&)

8. Burke MJ. How to manage asparaginase hypersensitivity in acute lymphoblastic leukemia. Future Oncol. 2014;10(16):2615–2627. doi: 10.2217/fon.14.138  [DOI](https://doi.org/10.2217/fon.14.138) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24983955/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Future%20Oncol&title=How%20to%20manage%20asparaginase%20hypersensitivity%20in%20acute%20lymphoblastic%20leukemia&author=MJ%20Burke&volume=10&issue=16&publication_year=2014&pages=2615-2627&pmid=24983955&doi=10.2217/fon.14.138&)

9. Hijiya N, van der Sluis IM. Asparaginase-associated toxicity in children with acute lymphoblastic leukemia. Leuk Lymphoma. 2016;57(4):748–757. doi: 10.3109/10428194.2015.1101098  [DOI](https://doi.org/10.3109/10428194.2015.1101098) | [PMC free article](/articles/PMC4819847/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26457414/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Leuk%20Lymphoma&title=Asparaginase-associated%20toxicity%20in%20children%20with%20acute%20lymphoblastic%20leukemia&author=N%20Hijiya&author=IM%20van%20der%20Sluis&volume=57&issue=4&publication_year=2016&pages=748-757&pmid=26457414&doi=10.3109/10428194.2015.1101098&)

10. Lopez-Santillan M, Iparraguirre L, Martin-Guerrero I, Gutierrez-Camino A, Garcia-Orad A. Review of pharmacogenetics studies of L-asparaginase hypersensitivity in acute lymphoblastic leukemia points to variants in the GRIA1 gene. Drug Metab Pers Ther. 2017;32(1):1–9. doi: 10.1515/dmpt-2016-0033  [DOI](https://doi.org/10.1515/dmpt-2016-0033) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28259867/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Pers%20Ther&title=Review%20of%20pharmacogenetics%20studies%20of%20L-asparaginase%20hypersensitivity%20in%20acute%20lymphoblastic%20leukemia%20points%20to%20variants%20in%20the%20GRIA1%20gene&author=M%20Lopez-Santillan&author=L%20Iparraguirre&author=I%20Martin-Guerrero&author=A%20Gutierrez-Camino&author=A%20Garcia-Orad&volume=32&issue=1&publication_year=2017&pages=1-9&pmid=28259867&doi=10.1515/dmpt-2016-0033&)

11. Abaji R, Krajinovic M. Pharmacogenetics of asparaginase in acute lymphoblastic leukemia. CDR. 2019. doi: 10.20517/cdr.2018.24  [DOI](https://doi.org/10.20517/cdr.2018.24) | [PMC free article](/articles/PMC8992626/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35582721/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=CDR&title=Pharmacogenetics%20of%20asparaginase%20in%20acute%20lymphoblastic%20leukemia&author=R%20Abaji&author=M%20Krajinovic&publication_year=2019&pmid=35582721&doi=10.20517/cdr.2018.24&)

12. Chen SH, Pei D, Yang W, et al. Genetic variations in GRIA1 on chromosome 5q33 related to asparaginase hypersensitivity. Clin Pharmacol Ther. 2010;88(2):191–196. doi: 10.1038/clpt.2010.94  [DOI](https://doi.org/10.1038/clpt.2010.94) | [PMC free article](/articles/PMC3000799/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20592726/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Genetic%20variations%20in%20GRIA1%20on%20chromosome%205q33%20related%20to%20asparaginase%20hypersensitivity&author=SH%20Chen&author=D%20Pei&author=W%20Yang&volume=88&issue=2&publication_year=2010&pages=191-196&pmid=20592726&doi=10.1038/clpt.2010.94&)

13. Rajić V, Debeljak M, Goričar K, Jazbec J. Polymorphisms in GRIA1 gene are a risk factor for asparaginase hypersensitivity during the treatment of childhood acute lymphoblastic leukemia. Leuk Lymphoma. 2015;56(11):3103–3108. doi: 10.3109/10428194.2015.1020802  [DOI](https://doi.org/10.3109/10428194.2015.1020802) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25697915/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Leuk%20Lymphoma&title=Polymorphisms%20in%20GRIA1%20gene%20are%20a%20risk%20factor%20for%20asparaginase%20hypersensitivity%20during%20the%20treatment%20of%20childhood%20acute%20lymphoblastic%20leukemia&author=V%20Raji%C4%87&author=M%20Debeljak&author=K%20Gori%C4%8Dar&author=J%20Jazbec&volume=56&issue=11&publication_year=2015&pages=3103-3108&pmid=25697915&doi=10.3109/10428194.2015.1020802&)

14. Højfeldt SG, Wolthers BO, Tulstrup M, et al. Genetic predisposition to PEG-asparaginase hypersensitivity in children treated according to NOPHO ALL2008. Br J Haematol. 2019;184(3):405–417. doi: 10.1111/bjh.15660  [DOI](https://doi.org/10.1111/bjh.15660) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30450575/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Haematol&title=Genetic%20predisposition%20to%20PEG-asparaginase%20hypersensitivity%20in%20children%20treated%20according%20to%20NOPHO%20ALL2008&author=SG%20H%C3%B8jfeldt&author=BO%20Wolthers&author=M%20Tulstrup&volume=184&issue=3&publication_year=2019&pages=405-417&pmid=30450575&doi=10.1111/bjh.15660&)

15. Fernandez CA, Smith C, Yang W, et al. HLA-DRB1*07:01 is associated with a higher risk of asparaginase allergies. Blood. 2014;124(8):1266–1276. doi: 10.1182/blood-2014-03-563742  [DOI](https://doi.org/10.1182/blood-2014-03-563742) | [PMC free article](/articles/PMC4141516/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24970932/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=HLA-DRB1*07:01%20is%20associated%20with%20a%20higher%20risk%20of%20asparaginase%20allergies&author=CA%20Fernandez&author=C%20Smith&author=W%20Yang&volume=124&issue=8&publication_year=2014&pages=1266-1276&pmid=24970932&doi=10.1182/blood-2014-03-563742&)

16. Rathod S, Ramsey M, Finkelman FD, Fernandez CA. Genetic inhibition of NFATC2 attenuates asparaginase hypersensitivity in mice. Blood Adv. 2020;4(18):4406–4416. doi: 10.1182/bloodadvances.2020002478  [DOI](https://doi.org/10.1182/bloodadvances.2020002478) | [PMC free article](/articles/PMC7509855/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32931581/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood%20Adv&title=Genetic%20inhibition%20of%20NFATC2%20attenuates%20asparaginase%20hypersensitivity%20in%20mice&author=S%20Rathod&author=M%20Ramsey&author=FD%20Finkelman&author=CA%20Fernandez&volume=4&issue=18&publication_year=2020&pages=4406-4416&pmid=32931581&doi=10.1182/bloodadvances.2020002478&)

17. Krawczyk M, Liebe R, Lammert F. Toward genetic prediction of nonalcoholic fatty liver disease trajectories: PNPLA3 and beyond. Gastroenterology. 2020;158(7):1865–1880.e1. doi: 10.1053/j.gastro.2020.01.053  [DOI](https://doi.org/10.1053/j.gastro.2020.01.053) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32068025/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastroenterology&title=Toward%20genetic%20prediction%20of%20nonalcoholic%20fatty%20liver%20disease%20trajectories:%20PNPLA3%20and%20beyond&author=M%20Krawczyk&author=R%20Liebe&author=F%20Lammert&volume=158&issue=7&publication_year=2020&pages=1865-1880.e1&pmid=32068025&doi=10.1053/j.gastro.2020.01.053&)

18. Salari N, Darvishi N, Mansouri K, et al. Association between PNPLA3 rs738409 polymorphism and nonalcoholic fatty liver disease: a systematic review and meta-analysis. BMC Endocr Disord. 2021;21(1):125. doi: 10.1186/s12902-021-00789-4  [DOI](https://doi.org/10.1186/s12902-021-00789-4) | [PMC free article](/articles/PMC8214766/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34147109/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Endocr%20Disord&title=Association%20between%20PNPLA3%20rs738409%20polymorphism%20and%20nonalcoholic%20fatty%20liver%20disease:%20a%20systematic%20review%20and%20meta-analysis&author=N%20Salari&author=N%20Darvishi&author=K%20Mansouri&volume=21&issue=1&publication_year=2021&pages=125&pmid=34147109&doi=10.1186/s12902-021-00789-4&)

19. Liu Y, Fernandez C, Smith C, et al. Genome-wide study links PNPLA3 variant with elevated hepatic transaminase after acute lymphoblastic leukemia therapy. Clin Pharmacol Ther. 2017;102(1):131–140. doi: 10.1002/cpt.629  [DOI](https://doi.org/10.1002/cpt.629) | [PMC free article](/articles/PMC5511775/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28090653/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Genome-wide%20study%20links%20PNPLA3%20variant%20with%20elevated%20hepatic%20transaminase%20after%20acute%20lymphoblastic%20leukemia%20therapy&author=Y%20Liu&author=C%20Fernandez&author=C%20Smith&volume=102&issue=1&publication_year=2017&pages=131-140&pmid=28090653&doi=10.1002/cpt.629&)

20. Hunger SP, Sung L, Howard SC. Treatment strategies and regimens of graduated intensity for childhood acute lymphoblastic leukemia in low-income countries: a proposal. Pediatr Blood Cancer. 2009;52(5):559–565. doi: 10.1002/pbc.21889  [DOI](https://doi.org/10.1002/pbc.21889) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19127567/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pediatr%20Blood%20Cancer&title=Treatment%20strategies%20and%20regimens%20of%20graduated%20intensity%20for%20childhood%20acute%20lymphoblastic%20leukemia%20in%20low-income%20countries:%20a%20proposal&author=SP%20Hunger&author=L%20Sung&author=SC%20Howard&volume=52&issue=5&publication_year=2009&pages=559-565&pmid=19127567&doi=10.1002/pbc.21889&)

21. Vos HI, Guchelaar HJ, Gelderblom H, et al. Replication of a genetic variant in ACYP2 associated with cisplatin-induced hearing loss in patients with osteosarcoma. Pharmacogenet Genomics. 2016;26(5):243–247. doi: 10.1097/FPC.0000000000000212  [DOI](https://doi.org/10.1097/FPC.0000000000000212) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26928270/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Replication%20of%20a%20genetic%20variant%20in%20ACYP2%20associated%20with%20cisplatin-induced%20hearing%20loss%20in%20patients%20with%20osteosarcoma&author=HI%20Vos&author=HJ%20Guchelaar&author=H%20Gelderblom&volume=26&issue=5&publication_year=2016&pages=243-247&pmid=26928270&doi=10.1097/FPC.0000000000000212&)

22. Kumagai H, Miyamoto-Mikami E, Hirata K, et al. ESR1 rs2234693 polymorphism is associated with muscle injury and muscle stiffness. Med Sci Sports Exerc. 2019;51(1):19–26. doi: 10.1249/MSS.0000000000001750  [DOI](https://doi.org/10.1249/MSS.0000000000001750) | [PMC free article](/articles/PMC6310456/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30113520/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Med%20Sci%20Sports%20Exerc&title=ESR1%20rs2234693%20polymorphism%20is%20associated%20with%20muscle%20injury%20and%20muscle%20stiffness&author=H%20Kumagai&author=E%20Miyamoto-Mikami&author=K%20Hirata&volume=51&issue=1&publication_year=2019&pages=19-26&pmid=30113520&doi=10.1249/MSS.0000000000001750&)

23. Burke MJ, Zalewska-Szewczyk B. Hypersensitivity reactions to asparaginase therapy in acute lymphoblastic leukemia: immunology and clinical consequences. Future Oncol. 2022;18(10):1285–1299. doi: 10.2217/fon-2021-1288  [DOI](https://doi.org/10.2217/fon-2021-1288) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35107320/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Future%20Oncol&title=Hypersensitivity%20reactions%20to%20asparaginase%20therapy%20in%20acute%20lymphoblastic%20leukemia:%20immunology%20and%20clinical%20consequences&author=MJ%20Burke&author=B%20Zalewska-Szewczyk&volume=18&issue=10&publication_year=2022&pages=1285-1299&pmid=35107320&doi=10.2217/fon-2021-1288&)

24. Zalewska-Szewczyk B, Andrzejewski W, Młynarski W, Jędrychowska-Dańska K, Witas H, Bodalski J. The anti-asparagines antibodies correlate with l -asparagines activity and may affect clinical outcome of childhood acute lymphoblastic leukemia. Leuk Lymphoma. 2007;48(5):931–936. doi: 10.1080/10428190701292049  [DOI](https://doi.org/10.1080/10428190701292049) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17487737/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Leuk%20Lymphoma&title=The%20anti-asparagines%20antibodies%20correlate%20with%20l%20-asparagines%20activity%20and%20may%20affect%20clinical%20outcome%20of%20childhood%20acute%20lymphoblastic%20leukemia&author=B%20Zalewska-Szewczyk&author=W%20Andrzejewski&author=W%20M%C5%82ynarski&author=K%20J%C4%99drychowska-Da%C5%84ska&author=H%20Witas&volume=48&issue=5&publication_year=2007&pages=931-936&pmid=17487737&doi=10.1080/10428190701292049&)

25. Silverman LB. Improved outcome for children with acute lymphoblastic leukemia: results of Dana-Farber consortium protocol 91-01. Blood. 2001;97(5):1211–1218. doi: 10.1182/blood.V97.5.1211  [DOI](https://doi.org/10.1182/blood.V97.5.1211) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11222362/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Improved%20outcome%20for%20children%20with%20acute%20lymphoblastic%20leukemia:%20results%20of%20Dana-Farber%20consortium%20protocol%2091-01&author=LB%20Silverman&volume=97&issue=5&publication_year=2001&pages=1211-1218&pmid=11222362&doi=10.1182/blood.V97.5.1211&)

26. Gottschalk Højfeldt S, Grell K, Abrahamsson J, et al. Relapse risk following truncation of pegylated asparaginase in childhood acute lymphoblastic leukemia. Blood. 2021;137(17):2373–2382. doi: 10.1182/blood.2020006583  [DOI](https://doi.org/10.1182/blood.2020006583) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33150360/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Relapse%20risk%20following%20truncation%20of%20pegylated%20asparaginase%20in%20childhood%20acute%20lymphoblastic%20leukemia&author=S%20Gottschalk%20H%C3%B8jfeldt&author=K%20Grell&author=J%20Abrahamsson&volume=137&issue=17&publication_year=2021&pages=2373-2382&pmid=33150360&doi=10.1182/blood.2020006583&)

27. Ovalle BP, Azócar MM, Nicklas DC, Villarroel CM, Morales VJ. Hypersensitivity reactions associated with the use of asparaginase in children with acute lymphoblastic leukemia. Andes Pediatr. 2021;92(2):182. doi: 10.32641/andespediatr.v92i2.2151  [DOI](https://doi.org/10.32641/andespediatr.v92i2.2151) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34106156/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Andes%20Pediatr&title=Hypersensitivity%20reactions%20associated%20with%20the%20use%20of%20asparaginase%20in%20children%20with%20acute%20lymphoblastic%20leukemia&author=BP%20Ovalle&author=MM%20Az%C3%B3car&author=DC%20Nicklas&author=CM%20Villarroel&author=VJ%20Morales&volume=92&issue=2&publication_year=2021&pages=182&pmid=34106156&doi=10.32641/andespediatr.v92i2.2151&)

28. Panosyan EH, Seibel NL, Martin-Aragon S, et al. Asparaginase antibody and asparaginase activity in children with higher-risk acute lymphoblastic leukemia: childrens cancer group study CCG-1961. J Pediatr Hematol Oncol. 2004;26(4):217–226. doi: 10.1097/00043426-200404000-00002  [DOI](https://doi.org/10.1097/00043426-200404000-00002) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15087948/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pediatr%20Hematol%20Oncol&title=Asparaginase%20antibody%20and%20asparaginase%20activity%20in%20children%20with%20higher-risk%20acute%20lymphoblastic%20leukemia:%20childrens%20cancer%20group%20study%20CCG-1961&author=EH%20Panosyan&author=NL%20Seibel&author=S%20Martin-Aragon&volume=26&issue=4&publication_year=2004&pages=217-226&pmid=15087948&doi=10.1097/00043426-200404000-00002&)

29. Kutszegi N, Semsei ÁF, Gézsi A, et al. Subgroups of paediatric acute lymphoblastic leukaemia might differ significantly in genetic predisposition to asparaginase hypersensitivity. PLoS One. 2015;10(10):e0140136. doi: 10.1371/journal.pone.0140136  [DOI](https://doi.org/10.1371/journal.pone.0140136) | [PMC free article](/articles/PMC4601692/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26457809/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Subgroups%20of%20paediatric%20acute%20lymphoblastic%20leukaemia%20might%20differ%20significantly%20in%20genetic%20predisposition%20to%20asparaginase%20hypersensitivity&author=N%20Kutszegi&author=%C3%81F%20Semsei&author=A%20G%C3%A9zsi&volume=10&issue=10&publication_year=2015&pages=e0140136&pmid=26457809&doi=10.1371/journal.pone.0140136&)

30. Dutta AK. Genetic factors affecting susceptibility to alcoholic liver disease in an Indian population. Ann Hepatol. 2013;12(6):901–907. doi: 10.1016/S1665-2681(19)31295-5  [DOI](https://doi.org/10.1016/S1665-2681(19)31295-5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24114820/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Hepatol&title=Genetic%20factors%20affecting%20susceptibility%20to%20alcoholic%20liver%20disease%20in%20an%20Indian%20population&author=AK%20Dutta&volume=12&issue=6&publication_year=2013&pages=901-907&pmid=24114820&doi=10.1016/S1665-2681(19)31295-5&)

31. Nischalke HD, Berger C, Luda C, et al. The PNPLA3 rs738409 148M/M genotype is a risk factor for liver cancer in alcoholic cirrhosis but shows no or weak association in hepatitis C cirrhosis. PLoS One. 2011;6(11):e27087. doi: 10.1371/journal.pone.0027087  [DOI](https://doi.org/10.1371/journal.pone.0027087) | [PMC free article](/articles/PMC3210131/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22087248/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=The%20PNPLA3%20rs738409%20148M/M%20genotype%20is%20a%20risk%20factor%20for%20liver%20cancer%20in%20alcoholic%20cirrhosis%20but%20shows%20no%20or%20weak%20association%20in%20hepatitis%20C%20cirrhosis&author=HD%20Nischalke&author=C%20Berger&author=C%20Luda&volume=6&issue=11&publication_year=2011&pages=e27087&pmid=22087248&doi=10.1371/journal.pone.0027087&)

32. Manchiero C, da S NAK, Magri MC, et al. The rs738409 polymorphism of the PNPLA3 gene is associated with hepatic steatosis and fibrosis in Brazilian patients with chronic hepatitis C. BMC Infect Dis. 2017;17(1):780. doi: 10.1186/s12879-017-2887-6  [DOI](https://doi.org/10.1186/s12879-017-2887-6) | [PMC free article](/articles/PMC5735770/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29258449/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Infect%20Dis&title=The%20rs738409%20polymorphism%20of%20the%20PNPLA3%20gene%20is%20associated%20with%20hepatic%20steatosis%20and%20fibrosis%20in%20Brazilian%20patients%20with%20chronic%20hepatitis%20C&author=C%20Manchiero&author=NAK%20da%20S&author=MC%20Magri&volume=17&issue=1&publication_year=2017&pages=780&pmid=29258449&doi=10.1186/s12879-017-2887-6&)

33. Romeo S, Kozlitina J, Xing C, et al. Genetic variation in PNPLA3 confers susceptibility to nonalcoholic fatty liver disease. Nat Genet. 2008;40(12):1461–1465. doi: 10.1038/ng.257  [DOI](https://doi.org/10.1038/ng.257) | [PMC free article](/articles/PMC2597056/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18820647/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Genetic%20variation%20in%20PNPLA3%20confers%20susceptibility%20to%20nonalcoholic%20fatty%20liver%20disease&author=S%20Romeo&author=J%20Kozlitina&author=C%20Xing&volume=40&issue=12&publication_year=2008&pages=1461-1465&pmid=18820647&doi=10.1038/ng.257&)

34. Chen LZ, Xin YN, Geng N, Jiang M, Zhang DD, Xuan SY. PNPLA3 I148M variant in nonalcoholic fatty liver disease: demographic and ethnic characteristics and the role of the variant in nonalcoholic fatty liver fibrosis. WJG. 2015;21(3):794. doi: 10.3748/wjg.v21.i3.794  [DOI](https://doi.org/10.3748/wjg.v21.i3.794) | [PMC free article](/articles/PMC4299331/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25624712/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=WJG&title=PNPLA3%20I148M%20variant%20in%20nonalcoholic%20fatty%20liver%20disease:%20demographic%20and%20ethnic%20characteristics%20and%20the%20role%20of%20the%20variant%20in%20nonalcoholic%20fatty%20liver%20fibrosis&author=LZ%20Chen&author=YN%20Xin&author=N%20Geng&author=M%20Jiang&author=DD%20Zhang&volume=21&issue=3&publication_year=2015&pages=794&pmid=25624712&doi=10.3748/wjg.v21.i3.794&)

35. Liu S, Gao C, Wu Y, et al. HLA-DRB1 *16:02 is associated with PEG-asparaginase hypersensitivity. Pharmacogenomics. 2021;22(17):1135–1142. doi: 10.2217/pgs-2021-0107  [DOI](https://doi.org/10.2217/pgs-2021-0107) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34747637/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=HLA-DRB1%20*16:02%20is%20associated%20with%20PEG-asparaginase%20hypersensitivity&author=S%20Liu&author=C%20Gao&author=Y%20Wu&volume=22&issue=17&publication_year=2021&pages=1135-1142&pmid=34747637&doi=10.2217/pgs-2021-0107&)
