# Genome-Wide Association Study Identifies Pharmacogenomic Variants Associated With Metformin Glycemic Response in African American Patients With Type 2 Diabetes

## Metadata
**Authors:** Baojun Wu, Sook Wah Yee, Shujie Xiao, Fei Xu, Sneha B Sridhar, Mao Yang, Samantha Hochstadt, Whitney Cabral, David E Lanfear, Monique M Hedderson, Kathleen M Giacomini, L Keoki Williams
**Journal:** Diabetes Care
**Date:** 2023 Aug 28
**DOI:** [10.2337/dc22-2494](https://doi.org/10.2337/dc22-2494)
**PMID:** 37639712
**PMCID:** PMC10834390
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10834390/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC10834390/pdf/dc222494.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC10834390/pdf/dc222494.pdf)

## Abstract

**OBJECTIVE:** 
Metformin is the most common treatment for type 2 diabetes (T2D). However, there have been no pharmacogenomic studies for T2D in which a population of color was used in the discovery analysis. This study sought to identify genomic variants associated with metformin response in African American patients with diabetes.

**RESEARCH DESIGN AND METHODS:** 
Patients in the discovery set were adult, African American participants from the Diabetes Multi-omic Investigation of Drug Response (DIAMOND), a cohort study of patients with T2D from a health system serving southeast Michigan. DIAMOND participants had genome-wide genotype data and longitudinal electronic records of laboratory results and medication fills. The genome-wide discovery analysis identified polymorphisms correlated to changes in glycated hemoglobin (HbA1c) levels among individuals on metformin monotherapy. Lead associations were assessed for replication in an independent cohort of African American participants from Kaiser Permanente Northern California (KPNC) and in European American participants from DIAMOND.

**RESULTS:** 
The discovery set consisted of 447 African American participants, whereas the replication sets included 353 African American KPNC participants and 466 European American DIAMOND participants. The primary analysis identified a variant, rs143276236, in the gene ARFGEF3, which met the threshold for genome-wide significance, replicated in KPNC African Americans, and was still significant in the meta-analysis (P = 1.17 × 10−9). None of the significant discovery variants replicated in European Americans DIAMOND participants.

**CONCLUSIONS:** 
We identified a novel and biologically plausible genetic variant associated with a change in HbA1c levels among African American patients on metformin monotherapy. These results highlight the importance of diversity in pharmacogenomic studies.

### OBJECTIVE

Metformin is the most common treatment for type 2 diabetes (T2D). However, there have been no pharmacogenomic studies for T2D in which a population of color was used in the discovery analysis. This study sought to identify genomic variants associated with metformin response in African American patients with diabetes.

### RESEARCH DESIGN AND METHODS

Patients in the discovery set were adult, African American participants from the Diabetes Multi-omic Investigation of Drug Response (DIAMOND), a cohort study of patients with T2D from a health system serving southeast Michigan. DIAMOND participants had genome-wide genotype data and longitudinal electronic records of laboratory results and medication fills. The genome-wide discovery analysis identified polymorphisms correlated to changes in glycated hemoglobin (HbA_1c_) levels among individuals on metformin monotherapy. Lead associations were assessed for replication in an independent cohort of African American participants from Kaiser Permanente Northern California (KPNC) and in European American participants from DIAMOND.

### RESULTS

The discovery set consisted of 447 African American participants, whereas the replication sets included 353 African American KPNC participants and 466 European American DIAMOND participants. The primary analysis identified a variant, rs143276236, in the gene *ARFGEF3*, which met the threshold for genome-wide significance, replicated in KPNC African Americans, and was still significant in the meta-analysis (*P* = 1.17 × 10^−9^). None of the significant discovery variants replicated in European Americans DIAMOND participants.

### CONCLUSIONS

We identified a novel and biologically plausible genetic variant associated with a change in HbA_1c_ levels among African American patients on metformin monotherapy. These results highlight the importance of diversity in pharmacogenomic studies.

## Introduction

In the U.S., ∼34 million people (10.5% of the total population) have diabetes, and the majority of these individuals (90–95%) have had type 2 diabetes (T2D) ([1](#B1)). For more than a decade, the American Diabetes Association and the European Association for the Study of Diabetes have recommended metformin as the initial drug of choice for the treatment of T2D ([2](#B2),[3](#B3)). According to the 2018 Medical Expenditure Panel Survey, metformin is estimated to be the sixth most commonly used medication in the U.S. in terms of the number of patients filling a prescription and the fourth most common medication in terms of total prescriptions filled ([4](#B4)). Nevertheless, the mechanism by which metformin exerts its effect on blood glucose levels and gluconeogenesis is not completely known and may involve both AMPK-dependent and AMPK-independent pathways ([5](#B5)).

Heritability for the change in glycated hemoglobin (HbA_1c_) levels while on metformin has been estimated to be between 20% and 34%, suggesting that a considerable proportion of treatment response is genetically determined ([6](#B6)). To date, there have been four published genome-wide association studies (GWAS) to identify pharmacogenomic variants associated with metformin treatment response ([7](#B7)–[11](#B11)). Of note, using the additive genetic model, the variants identified in these studies did not meet genome-wide statistical significance in the discovery analysis alone but, rather, reached statistical significance only after meta-analyzing results with other cohorts.

In other pharmacogenomic analyses, we and others have demonstrated that risk variants identified in one population group often do not replicate in other groups ([12](#B12)). We have also demonstrated the advantages of genetic association studies in diverse population groups, particularly African Americans, where lower linkage disequilibrium (LD) can facilitate finer mapping resolution ([13](#B13)).

In this study, we performed an ancestry-stratified genome-wide analysis for pharmacogenomic variants associated with glycemic response among African American individuals treated with metformin. Study participants were from the Diabetes Multi-omic Investigation of Drug Response (DIAMOND), a diverse cohort of individuals with T2D from southeast Michigan and the Detroit metropolitan area.

## Research Design and Methods

### Study Populations and Settings

The cohorts included in this study were approved by the institutional review boards of the Henry Ford Health System; the University of California, San Francisco; and Kaiser Permanente. All participants provided written informed consent at the time of enrollment prior to the collection of study-related data and samples. Eligible DIAMOND participants were aged ≥18 years, were members of the health system and possessed affiliated health insurance with pharmaceutical coverage, and resided in southeast Michigan. Among this group, individuals with two or more recorded diagnoses of T2D and at least one HbA_1c_ value ≥6.5% were invited to participate. Individuals with a diagnosis of type 1 diabetes, gestational diabetes mellitus alone, or drug-associated diabetes (e.g., because of corticosteroid use) were excluded. For the discovery analysis, we restricted the study sample to individuals who self-identified as African American and who had been treated with metformin alone (i.e., metformin monotherapy) for diabetes.

The replication cohort consisted of African American patients who received care at Kaiser Permanente Northern California (KPNC), and White individuals of European descent (European Americans) from the DIAMOND cohort. The KPNC cohort has been described in detail elsewhere ([9](#B9),[14](#B14)). Electronic medical records were used to identify patients with T2D who had received at least 6 months of care in the health system and who had not used other diabetes medications prior to their first metformin prescription. Patients had to have an HbA_1c_ measurement within 90 days and ≥90 days following metformin initiation. Patients who met these criteria were contacted via letter and invited to participate.

### DNA Collection, Genotyping, and Quality Control

At the time of enrollment, participants underwent a detailed evaluation that included completing a research survey, a clinical examination, and specimen collection. DNA was primarily obtained from blood, but occasionally from saliva (Oragene DNA Kit; DNA Genotek Inc.). Genomic DNA was isolated and assessed for its quality and quantity using a Nanodrop spectrophotometer and PicoGreen dsDNA quantification assay (Thermo Fisher Scientific). Genome-wide genotyping was performed with the Axiom Precision Medicine Research Array (Thermo Fisher Scientific) at the University of North Carolina’s Functional Genomics Core. The genotyping quality control (QC) threshold for including an individual’s results was a dish QC >0.82 and an overall call rate >97%. A total of 2,158 DIAMOND participants were genotyped; 13 (0.6%) samples did not pass genotyping QC.

Variant QC included assessment of overall call rate and Hardy-Weinberg equilibrium (*P* < 10^−6^). Of 855,849 single nucleotide polymorphisms (SNPs) genotyped, 797,368 (93%) passed our QC criteria. The internal QC filter of the Michigan Imputation Server (reference panel TOPMed-r2 version 1.0.0 [hg38]) excluded an additional 70,265 variants, and the remaining set of variants were used to impute missing genotypes ([15](#B15)). Variants with an imputation score ≥0.8 were used in the analysis. We also restricted our association analyses to biallelic SNPs with a minor allele frequency (MAF) ≥1%.

Population structure was characterized using principal components (PCs) ([16](#B16),[17](#B17)). Individuals ≥6 SDs from their self-reported race group in the first 10 PCs were removed from the analysis (no African Americans and eight European American participants were excluded based on this criterion). An additional eight samples were removed (three African American and five European American participants) based on an inbreeding coefficient ≥6 SDs from the rest of the group. Relatedness among participants was estimated using kinship coefficients computed in PLINK 2.0 ([18](#B18)). In situations where participants were related (i.e., pairwise kinship coefficient >0.1), one individual’s sample was randomly excluded from the analysis set. Twenty-six individuals (20 African American and 6 European American participants) were removed based on relatedness.

Individuals in the KPNC cohort and their samples underwent a similar QC process. These individuals were genotyped on the Illumina Infinium OmniExpress-24 version 1.2 array. After QC, 725,468 SNPs were used to impute missing genotypes on the Michigan Imputation Server.

Local and global genetic ancestries were assessed in both the discovery and replication groups using the programs RFMix version 1.5.4 ([19](#B19)) and ADMIXTURE 1.3.0 ([20](#B20)). The reference populations for ancestry estimation were the YRI (Yoruba in Ibadan, Nigeria) and CEU (Northern Europeans from Utah) groups from the 1000 Genomes Project ([21](#B21)).

### Clinical Data and Drug Exposure Measures

The HbA_1c_ laboratory results used for assessing metformin response had to be at least 120 days apart; metformin use had to span this interval such that a 120-day window of drug exposure could be calculated and related to the change in HbA_1c_. The first set of HbA_1c_ measurements fulfilling these criteria (i.e., the interval closest to or including treatment initiation) was used in calculating the primary outcome.

Metformin exposure was estimated using available pharmacy claims data in the manner similar to what we have done previously ([22](#B22)); a schematic is shown in [Supplementary Fig. 1](https://doi.org/10.2337/figshare.23925912). We used the claims information on each metformin fill (i.e., preparation strength, total amount dispensed) to calculate the total amount of medication received. The time interval between refills was then used to equally distribute the total amount dispensed over that interval, thereby assigning an estimated daily amount consumed to each day between fills. Anchoring on the date of the second HbA_1c_ test result (for the observation window used), we averaged the estimated daily amount of metformin consumed for the preceding 120-day period (i.e., the period of glycemic control reflected in the HbA_1c_ result). Baseline HbA_1c_ was the first HbA_1c_ measurement in the selected observation window. Serum creatinine measures taken within the 6 months of baseline HbA_1c_ were used for the estimated glomerular filtration rate (eGFR) ([23](#B23)).

### Statistical Analysis

The primary outcome was the change in HbA_1c_ levels while on metformin monotherapy. In our initial analysis, we assessed the relationship between genotype and the primary outcome, restricted to individuals whose estimated daily metformin use was ≥425 mg. This dose was selected to represent the equivalent of 80% adherence to the lowest possible daily dose of metformin (i.e., 500 mg/day) or 50% adherence of the next lower pill size (850 mg) to define a consistent level of exposure; this is the approximate range of adherence observed in studies of metformin use by us and others ([24](#B24)). To validate this threshold, we also evaluated the relationship between metformin use and change in HbA_1c_ levels ([Supplementary Fig. 2](https://doi.org/10.2337/figshare.23925912)) and the distribution of metformin exposure ([Supplementary Fig. 3](https://doi.org/10.2337/figshare.23925912)). These analyses suggested that 425 mg/day of metformin constituted an inflection point in terms of treatment response (i.e., the point above which increasing metformin use lowered HbA_1c_ levels). Change in HbA_1c_ was normalized using rank-based inverse normal transformation ([25](#B25)). The transformed change in HbA_1c_ levels was regressed on genotype, using an additive genetic model and restricted to biallelic SNPs. Additional covariates included patient age, sex, eGFR, and baseline HbA_1c_ levels (i.e., the initial HbA_1c_ level in the interval assessed). Association testing was performed using the software package GENESIS ([26](#B26)) and was restricted to regions identified as homozygous for African ancestry based on results from RFMix. Genome-wide statistical significance was defined as *P* < 5 × 10^−8^.

Variants that met the threshold for statistical significance were reassessed in the KPNC cohort and in European American DIAMOND participants. A *P* value of 0.05 was used as the significance threshold for the replication. The regression models used in the replication analysis were the same as those used for the discovery analysis. Results from African American participants in the discovery and replication groups were meta-analyzed using the program METAL ([27](#B27)). Based on a discovery group sample size of 447, *P* < 5 × 10^−8^ significance threshold, and an MAF of either ≥0.01 or ≥0.03, we estimated 80% power to detect effect sizes of ≥2.07 and ≥1.21, respectively. For the replication group using a sample size of 353, *P* = 0.05 significance threshold, and an MAF of either ≥0.01 or ≥0.03, we estimated 80% power to detect effect sizes of ≥1.05 and ≥0.61, respectively. We also performed a post hoc sensitivity analysis to account for the periods used in our evaluation with respect to metformin initiation. These analyses included separately stratifying the group by the time of assessment, as well as by adjusting for time since initiation in the regression model.

Our second approach for identifying pharmacogenomic variants associated with metformin treatment response involved evaluating for gene (genotype) × drug interactions associated with change in HbA_1c_. Rank-based inverse normal transformed change in HbA_1c_ levels was regressed on genotype, metformin exposure (i.e., <425 vs. ≥425 mg/day), and an interaction term between genotype and metformin use. The models adjusted for patient age, sex, eGFR, and baseline HbA_1c_ levels. Statistical significance was assessed by comparing the full model (i.e., the model with age, sex, eGFR, baseline HbA_1c_ level, metformin exposure, genotype, and a genotype × metformin interaction) with a parsimonious model (i.e., the model without the genotype and genotype × metformin interaction terms). Therefore, the joint association test assessed the combined effect of adding genotype and the interaction term. The genome-wide statistical significance threshold for the joint association was also set at *P* < 5 × 10^−8^. See the [Supplementary Methods and Discussion](https://doi.org/10.2337/figshare.23925912) for additional commentary on the analyses performed.

## Results

### Study Populations

The discovery set comprised 447 African American DIAMOND participants with genome-wide genotype data and consistent metformin use (i.e., an estimated ≥425 mg/day during the observation period). Similarly, there were 353 African American participants from the KPNC cohort and 466 European American participants from the DIAMOND cohort with similar data for replication. The characteristics of these participants are shown in [Table 1](#T1) and [Supplementary Table 1](https://doi.org/10.2337/figshare.23925912). The average age among African American DIAMOND participants, African American KPNC participants, and European American DIAMOND participants was 56.4, 55.2, and 59.5 years, respectively. The sex distribution in these groups was 69.1%, 66.3%, and 41.8% female, respectively. Baseline HbA_1c_ levels in these groups were 7.6%, 7.3%, and 7.3%, and the average absolute change in HbA_1c_ among individuals whose average metformin use was ≥425 mg/day during the study observation period was −0.57, −0.66, and −0.44, respectively.

### Table 1.

|   | Discovery group | Replication groups |  |
| --- | --- | --- | --- |
| African American DIAMOND participants (n = 447) | African American KPNC participants (n = 353) | European American DIAMOND participants (n = 466) |  |
| Age (years)† | 56.4 ± 10.3 | 55.2 ± 10.0 | 59.5 ± 10.5 |
| Female sex, n (%) | 309 (69.1) | 234 (66.3) | 195 (41.8) |
| BMI (kg/m2)‡ | 35.0 ± 7.1 (n = 328) | 35.3 ± 7.2 (n = 287) | 34.2 ± 6.0 (n = 306) |
| Proportion of African ancestry§ | 81.7 ± 10.3 | 79.4 ± 14.3 | 0.06 ± 2.0 |
| Baseline HbA1c level (%)ǁ | 7.6 ± 1.8 | 7.3 ± 1.8 | 7.3 ± 1.4 |
| Change in HbA1c on metformin treatment** | −0.57 ± 1.76 | −0.66 ± 1.78 | −0.44 ± 1.40 |
| Metformin use (mg/day)†† | 937.5 ± 419.0 | 963.5 ± 471.7 | 1,088.5 ± 502.9 |
| eGFR (mL/min/1.73 m2)‡‡ | 93.7 ± 20.4 (n = 447) | 94.4 ± 19.1 (n = 353) | 87.0 ± 18.5 (n = 466) |

Table 1 Caption: Characteristics of discovery and replication groups used in the primary analysis*

### Genome-Wide Association Identifies Relationship Between Variant rs143276236 and HbA1c Change Among Individuals With T2D on Metformin Therapy

Results from the genome-wide association analysis are shown in [Table 2](#T2) and [Fig. 1](#F1). The quantile-quantile plot is shown in [Supplementary Fig. 4](https://doi.org/10.2337/figshare.23925912). In the discovery analysis, 11 SNPs had an association *P* value less than the genome-wide significance threshold of *P* < 5.0 × 10^−8^ in regions homozygous for African ancestry. Variants rs143276236 and rs116251012 in the gene *ARFGEF3* on chromosome 6 had a consistent direction of effect in both DIAMOND and KPNC cohorts, and the variants had a replication *P* < 0.05 in KPNC cohort. A meta-analysis of the signal in regions homozygous for African ancestry in DIAMOND and KPNC participants was statistically significant for rs143276236 (*P* = 1.17 × 10^−9^) and for rs116251012 (*P* = 1.59 × 10^−8^). A locus zoom plot of the region ([Supplementary Fig. 5](https://doi.org/10.2337/figshare.23925912)) demonstrates that the lead variant rs143276236 was in high LD with variant rs116251012 (*r*^2^ = 0.78). Only two of the variants identified in African ancestry regions were sufficiently frequent to assess in European American DIAMOND participants ([Supplementary Table 2](https://doi.org/10.2337/figshare.23925912)). Neither of these variants reached statistical significance among European American participants or when meta-analyzed in all three study groups.

### Table 2.

| Variant | Chr | Position (Hg38) | Alleles | Nearest gene | Functional consequence | DIAMOND cohort (discovery set) |  |  |  | KPNC cohort (replication set) |  |  |  |   |   |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| n | MAF | Parameter estimate† | P | n | MAF | Parameter estimate† | P | Direction of effect | Meta-analysis P |  |  |  |  |  |  |
| rs150979496 | 18 | 8501049 | T/C | THEMIS3P | None known | 300 | 0.012 | 1.90 | 5.04 × 10−9 | — | NA | NA | NA | NA | NA |
| rs143276236 | 6 | 138176343 | A/C | ARFGEF3 | Intron variant | 292 | 0.027 | 1.20 | 6.39 × 10−9 | 227 | 0.024 | 0.60 | 0.0072 | +/+ | 1.17 × 10−9 |
| rs59699971 | 12 | 40649666 | G/A | MUC19 | None known | 288 | 0.031 | 1.16 | 9.57 × 10−9 | 219 | 0.041 | −0.32 | 0.0977 | +/− | 0.006 |
| rs115567566 | 12 | 40671268 | G/A | MUC19 | None known | 287 | 0.031 | 1.16 | 1.13 × 10−8 | 218 | 0.039 | −0.39 | 0.0482 | +/− | 0.011 |
| rs142017898 | 16 | 83500006 | C/T | CDH13 | Intron variant | 287 | 0.017 | 1.52 | 1.15 × 10−8 | 219 | 0.025 | −0.09 | 0.6884 | +/− | 0.001 |
| rs138642807 | 9 | 109970865 | T/G | PALM2-AKAP2 | Intron variant | 285 | 0.012 | 1.88 | 1.25 × 10−8 | 221 | 0.018 | 0.20 | 0.4432 | +/+ | 3.17 × 10−5 |
| rs146090661 | 9 | 109975903 | G/A | PALM2-AKAP2 | Intron variant | 285 | 0.012 | 1.88 | 1.25 × 10−8 | 221 | 0.018 | 0.20 | 0.4432 | +/+ | 3.17 × 10−5 |
| rs12297682 | 12 | 40920062 | G/A | CNTN1 | Intron variant | 287 | 0.052 | 0.85 | 4.09 × 10−8 | 219 | 0.055 | −0.24 | 0.1506 | +/− | 0.003 |
| rs79162841 | 12 | 40926234 | G/A | CNTN1 | Intron variant | 287 | 0.052 | 0.85 | 4.09 × 10−8 | 219 | 0.055 | −0.24 | 0.1506 | +/− | 0.003 |
| rs116251012 | 6 | 138191883 | T/C | ARFGEF3 | Intron variant | 292 | 0.031 | 1.08 | 4.80 × 10−8 | 227 | 0.035 | 0.49 | 0.0092 | +/+ | 1.59 × 10−8 |
| rs149911554 | 10 | 106606746 | A/G | SORCS1 | Intron variant | 274 | 0.013 | 1.54 | 4.99 × 10−8 | 210 | 0.024 | 0.44 | 0.0775 | +/+ | 8.45 × 10−7 |
| rs114019854 | 10 | 106606829 | T/C | SORCS1 | Intron variant | 274 | 0.013 | 1.54 | 4.99 × 10−8 | 210 | 0.024 | 0.44 | 0.0775 | +/+ | 8.45 × 10−7 |

Table 2 Caption: Genome-wide significant associations for change in HbA1c levels among individuals on metformin monotherapy*

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5aac/10834390/59d0d70f1436/dc222494f1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=10834390_dc222494f1.jpg)

Manhattan plot of the discovery analysis relating genome-wide polymorphisms to changes in HbA1c among African American patients with T2D on metformin monotherapy from the DIAMOND cohort. Discovery association results were restricted to regions where individuals were homozygous for African ancestry. The dashed line denotes the threshold for genome-wide statistical significance (P < 5.0 × 10−8).

### Unique African and European Haplotypes in Chromosome 6 Region Associated With Metformin Glycemic Response

[Supplementary Fig. 6](https://doi.org/10.2337/figshare.23925912) demonstrates the haplotype structure in the region surrounding the leading variant rs143276236 on chromosome 6. As can be seen, the extent of LD was lower and the size of the haplotype blocks smaller in a background homozygous for African ancestry ([Supplementary Fig. 6A](https://doi.org/10.2337/figshare.23925912)) compared with the same region homozygous for European ancestry ([Supplementary Fig. 6B](https://doi.org/10.2337/figshare.23925912)).

### An Ancestry-Informed Gene × Metformin Interaction Association Analysis for Glycemic Response Identifies Similar Top Genetic Variant Associations

In DIAMOND, we had 203 additional African American participants who were on metformin monotherapy but had an average estimated exposure <425 mg/day in the observation period ([Supplementary Table 3](https://doi.org/10.2337/figshare.23925912)). Average estimated metformin use in this group was 224.2 mg/day. As would be expected, the absolute improvement in HbA_1c_ levels was lower in individuals whose average exposure was <425 mg/day compared with those using ≥425 mg/day (−0.22 vs. −0.57, respectively; *P* = 0.024). Both the high and low metformin exposure groups (i.e., ≥425 mg/day and <425 mg/day) were used to assess for gene × metformin interactions (i.e., variants whose effect on HbA_1c_ appeared to differ based on metformin exposure). The association analysis was again restricted to intraindividual genomic regions homozygous for African ancestry. The quantile-quantile plot is shown in [Supplementary Fig. 7](https://doi.org/10.2337/figshare.23925912). A joint test was used to assess the combined effect of genotype and genotype × drug interaction ([Supplementary Table 4](https://doi.org/10.2337/figshare.23925912)). The effect of metformin on the change in HbA_1c_ was evinced by the consistent inverse relationship between level of metformin exposure and the outcome variable. Many of the same variants identified in the previous analysis ([Table 2](#T2)) were again observed in the interaction analysis. *ARFGEF3* was also identified in the gene × metformin interaction analysis; significant variants in this gene included rs143276236 (noted previously) and the two SNPs rs141012141 and rs80340144 located within 137 base pairs of each other. The joint test *P* values for these three *ARFGEF3* variants were 1.08 × 10^−8^, 3.48 ×10^−8^, 3.48 × 10^−8^, respectively; however, only rs143276236 suggested replication in the KPNC cohort (joint test *P* = 0.06).

### Association Between Variants in ARFGEF3 and HbA1c Change by Degree of Metformin Use

Differences in the apparent relationship among degree of metformin use, genotype, and HbA_1c_ improvement are shown in [Fig. 2](#F2). Among individuals with ≥425 mg/day average daily exposure to metformin, individuals with the *ARFGEF3* rs143276236 CC genotype had an average absolute HbA_1c_ reduction of 0.59%, whereas individuals with the AC genotype had an average absolute HbA_1c_ reduction of 0.28%; these differences were statistically significant (*P* = 0.0083) ([Fig. 2A](#F2)). The genotype relationship was not statistically significant among individuals with average metformin exposure <425 mg/day (*P* = 0.11). A similar pattern was also observed using the HbA_1c_ change adjusted for baseline HbA_1c_. The difference between the two genotypes were statistically significant (*P* = 0.012) ([Fig. 2B](#F2)) with average metformin exposure ≥425 mg/day, but the difference was not statistically significant among individuals with average metformin exposure <425 mg/day (*P* = 0.91).

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5aac/10834390/5a668e483c29/dc222494f2.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=10834390_dc222494f2.jpg)

Absolute changes (A) and adjusted changes (B) in HbA1c levels with metformin treatment among African American DIAMOND participants stratified by the ARFGEF3 variant genotype rs143276236. Results are also stratified by average levels of daily metformin use as assessed using pharmacy records of patient medication fills. Among individuals on metformin monotherapy for T2D, high exposure was considered ≥425 mg/day (left), and low exposure was considered <425 mg/day (right). Only one individual was AA homozygous for the ARFGEF3 rs143276236 variant (noted by dashed line).

### Post Hoc Analyses Accounting for Time of Treatment

Since the exposure window used in our assessment may not have coincided with metformin initiation, we performed a post hoc analysis to assess the potential effect of timing on our results. Most assessments were performed at the time of metformin initiation ([Supplementary Fig. 8](https://doi.org/10.2337/figshare.23925912)). The durability of metformin response appeared to last to day 403 following treatment initiation, but it was most evident in the first 146 days ([Supplementary Fig. 9](https://doi.org/10.2337/figshare.23925912)). The lead variant rs143276236 in the association analysis had a consistent magnitude and direction of effect between 0 and 146 days and between 147 and 403 days postinitiation ([Supplementary Table 5](https://doi.org/10.2337/figshare.23925912)), and it demonstrated genome-wide significance within the 403 days in which the metformin treatment response was evident (*P* = 9.72 × 10^−9^) ([Supplementary Table 6](https://doi.org/10.2337/figshare.23925912)). Variant rs143276236 was also associated with change in HbA_1c_ after adjusting for time since treatment initiation in both the discovery (*P* = 9.09 × 10^−9^) and replication sets (*P* = 0.035) ([Supplementary Table 7](https://doi.org/10.2337/figshare.23925912)).

## Conclusions

Despite rapid progress in genomics, U.S. minority populations are considerably underrepresented in existing studies ([28](#B28)). For example, individuals of African descent constitute ∼2% of all GWAS participants ([29](#B29),[30](#B30)), and these figures are much smaller when one considers the number of studies where people of color comprise the discovery population or are sufficiently represented for adequately powered subgroup analyses. Without such studies, these population groups would not share in precision medication advancements that are predicated on this foundational research ([31](#B31)).

To leverage the strength of our study’s diversity, we used a technique that we have used successfully on other projects involving fine mapping of variant associations in regions homozygous for African ancestry ([13](#B13)). This approach has multiple potential advantages. First, it can be used to assess for associations in an ancestral background not previously evaluated, thereby identifying novel or population-specific risk variants. Second, it may permit better mapping resolution of causal variants, especially if ancestral group differences in allele frequency, LD, or genetic variation permit better signal isolation ([32](#B32)). We also used methods developed by us to assess metformin exposure in large patient populations to ensure that medication use was taken into account ([22](#B22),[24](#B24)).

Using the above approach, we evaluated for variants associated with a change in HbA_1c_ levels while on metformin monotherapy. The most significant allele replicated was rs143276236, an intronic SNP located in the gene *ARFGEF3* that is also known as brefeldin A-inhibited guanine nucleotide-exchange protein 3 (*BIG3*). Experiments in mice have found *BIG3* to be highly expressed in islet α- and β-cells, and *BIG3* expression is inversely associated with both glucagon and insulin granule biogenesis in both types of cells, respectively ([33](#B33),[34](#B34)). *BIG3* knockout mice also had a 35% increase in insulin granule content in β-cells and enhanced insulin secretion; however, they were also more likely to develop postprandial hyperglycemia and hyperinsulinemia, consistent with peripheral insulin resistance attributable to chronic oversecretion ([33](#B33)). In short, *BIG3* appears to function as a negative regulator of insulin secretion through its role in regulating early granule biogenesis in the Golgi apparatus. While the mechanistic relationship between *ARFGEF3*/*BIG3* and metformin use remain to be determined, the gene itself is a plausible mediator of blood glucose levels.

To investigate the possible effects that our lead variant rs143276236 may have on gene expression, we performed a post hoc analysis using HaploReg version 4.1 ([35](#B35)) to assess potential effects on transcription factor binding. The algorithm estimated that the variant may alter activating transcription factor 3 (*ATF3*) and *CACD2* binding motifs with reduced binding affinity. Studies have suggested that *ATF3* may influence both T2D and metformin’s effect on hepatic gluconeogenesis ([36](#B36),[37](#B37)). Furthermore, a study showed that *ATF3* is the top gene that is regulated by metformin in human liver hepatocytes ([38](#B38)).

A previous GWAS identified variant rs8192675 in the glucose transporter gene *SLC2A2* to be associated with metformin glycemic response among White European individuals with T2D ([9](#B9)). We did not find any significant variants within the *SLC2A2* among participants of African ancestry, and the association of rs8192675 with change in HbA_1c_ obtained a *P* value of 0.81 in our discovery set. The discrepancy highlights the potential importance of diversity in pharmacogenomic studies. In addition, Mahajan et al. ([39](#B39)) performed a meta-analysis of 122 separate studies comprising 180,834 individuals with T2D and 1,159,055 control subjects; the composition of the group was 51.1% White European, 28.4% East Asian, 8.3% South Asian, 6.6% African (including African Americans), and 5.6% Latino (of any ancestry group). The investigators identified 277 separate genetic loci associated with T2D at a significance level of *P* < 5 × 10^−8^. Again, we did not find any significant variants overlapping with these 277 loci, suggesting that the variant that we identified may be only relevant in the presence of metformin, as was observed in our study. Recently, Li et al. ([11](#B11)) conducted a GWAS of individuals with prediabetes from the Diabetes Prevention Program (DPP). The investigators assessed both for variants associated with various glycemic traits, including HbA_1c_, in the treatment arm and for gene × drug interactions (using both treatment and placebo arms). Variants in *ENOSF1* and *LOC101928519* were associated with a 1-year change in HbA_1c_ levels in the metformin-only group, and variants in *LINC01093*, *MAN2B2*, and *SOX5* were associated with change in HbA_1c_ in the interaction model. Only *LOC101928519* was found to be statistically significant among African American DPP participants, and none of these variants were identified in our current study.

Although the mechanism of metformin’s effect on blood glucose levels is not known, it is widely speculated that the principal effect involves hepatic gluconeogenesis ([40](#B40)). Curiously, our top associations in *ARFGEF*/*BIG3* suggest that metformin may also affect glucose and insulin levels through islet cell granule formation. However, our observational study design did not allow us to investigate whether metformin has a direct or indirect effect on these processes. Nevertheless, the consistency of our findings in terms of site of action, robustness to different analytic methods, and replication in an independent sample suggests that the pharmacogenomic variants identified here deserve additional functional validation.

The use of observational data to assess medication response has its limitations. For example, we had to use strict criteria to define drug exposure using retrospective electronic medical record data. We have previously demonstrated that these exposure measures are strongly related to clinical outcomes ([22](#B22)); hence, these data provide an opportunity to evaluate population groups that are understudied in clinical trials. We also acknowledge that 425 mg/day does not correspond to an actual metformin pill size; however, this level of exposure did appear to constitute an inflection point for metformin treatment response ([Supplementary Fig. 2](https://doi.org/10.2337/figshare.23925912)). It is important to note that since patients often do not take their full dose as prescribed ([24](#B24)), average levels of exposure may not perfectly coincide with pill strengths. To best leverage the data available to us, we also used exposure times that did not always include metformin initiation. Nevertheless, in post hoc analyses, we demonstrated that our lead variant was robust to various methods to account for time since treatment initiation. Most importantly, we demonstrated that rs143276236 was a leading pharmacogenomic variant using two very different analytic approaches: an exposure-only analysis and a genotype × drug interaction analysis.

In summary, we have identified at least one variant that appears to influence metformin’s glycemic effect in African American individuals with T2D. Performing pharmacogenomic studies in diverse population groups is necessary for a number of reasons. First, pharmacogenomic information for U.S. minority patients is lacking, so without these studies, these population groups will not equally benefit from downstream advances in precision medicine. For example, genetic predictive risk scores developed using data from European ancestry groups perform poorly in other population groups ([31](#B31)); therefore, more diverse pharmacogenomic studies are needed as a basis for more broadly applicable risk scores ([22](#B22)). Second, LD differences between population groups may actually improve fine-mapping resolution and causal variant identification ([13](#B13)). While further studies are needed to replicate our findings, our approach provides a useful roadmap for others to follow in terms of leveraging diversity to improve pharmacogenomic discovery.

This article contains supplementary material online at [https://doi.org/10.2337/figshare.23925912](https://doi.org/10.2337/figshare.23925912).

## Article Information

**Acknowledgments.** The authors acknowledge Dr. Michiaki Kubo, Dr. Atsushi Takahashi, Dr. Shiro Maeda, Dr. Yukihide Momozawa, and Dr. Yoichiro Kamatani at the RIKEN Institute for performing genotyping of the KPNC cohort on the Illumina Infinium OmniExpress-24, version 1.2 array.

**Funding.** This project was supported by the Fund for Henry Ford Hospital and National Institute of Diabetes and Digestive and Kidney Diseases grants R01DK064695 and R01DK113003 (to L.K.W.); National Heart, Lung, and Blood Institute grants R01HL079055, R01HL118267, and R01HL141845 (to L.K.W.), National Institute of Allergy and Infectious Diseases grants R01AI079139 and R56AI165903 (to L.K.W.), and National Institute of General Medical Sciences grants U01GM061390 and R01GM117163 (to S.W.Y., K.M.G., and M.M.H.).

**Duality of Interest.** No potential conflicts of interest relevant to this article were reported.

**Author Contributions.** B.W., S.W.Y., S.X., F.X., S.B.S., M.Y., S.H., W.C., D.E.L., M.M.H., K.M.G., and L.K.W. contributed to the data collection. B.W., S.W.Y., S.X., and L.K.W. contributed to the conception and design of the study and to the data analysis. B.W., S.X., and L.K.W. contributed to the initial manuscript writing. All authors revised the manuscript for important content and approved of the final version. B.W. and L.K.W. are the guarantors of this work and, as such, had full access to all the data in the study and take responsibility for the integrity of the data and the accuracy of the analysis.

## Funding Statement

This project was supported by the Fund for Henry Ford Hospital and National Institute of Diabetes and Digestive and Kidney Diseases grants R01DK064695 and R01DK113003 (to L.K.W.); National Heart, Lung, and Blood Institute grants R01HL079055, R01HL118267, and R01HL141845 (to L.K.W.), National Institute of Allergy and Infectious Diseases grants R01AI079139 and R56AI165903 (to L.K.W.), and National Institute of General Medical Sciences grants U01GM061390 and R01GM117163 (to S.W.Y., K.M.G., and M.M.H.).

## Footnotes

## References

1. Centers for Disease Control and Prevention , Ed. National Diabetes Statistics Report, 2020. Atlanta, GA, Centers for Disease Control and Prevention, U.S. Department of Health and Human Services, 2020  [Google Scholar](https://scholar.google.com/scholar_lookup?title=National%20Diabetes%20Statistics%20Report,%202020&publication_year=2020&)

2. Nathan DM, Buse JB, Davidson MB, et al.; American Diabetes Association; European Association for Study of Diabetes . Medical management of hyperglycemia in type 2 diabetes: a consensus algorithm for the initiation and adjustment of therapy: a consensus statement of the American Diabetes Association and the European Association for the Study of Diabetes. Diabetes Care 2009;32:193–203  [DOI](https://doi.org/10.2337/dc08-9025) | [PMC free article](/articles/PMC2606813/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18945920/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Diabetes%20Care&title=Medical%20management%20of%20hyperglycemia%20in%20type%202%20diabetes:%20a%20consensus%20algorithm%20for%20the%20initiation%20and%20adjustment%20of%20therapy:%20a%20consensus%20statement%20of%20the%20American%20Diabetes%20Association%20and%20the%20European%20Association%20for%20the%20Study%20of%20Diabetes&volume=32&publication_year=2009&pages=193-203&pmid=18945920&doi=10.2337/dc08-9025&)

3. Buse JB, Wexler DJ, Tsapas A, et al. 2019 Update to: Management of Hyperglycemia in Type 2 Diabetes, 2018. A Consensus Report by the American Diabetes Association (ADA) and the European Association for the Study of Diabetes (EASD). Diabetes Care 2020;43:487–493  [DOI](https://doi.org/10.2337/dci19-0066) | [PMC free article](/articles/PMC6971782/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31857443/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Diabetes%20Care&title=2019%20Update%20to:%20Management%20of%20Hyperglycemia%20in%20Type%202%20Diabetes,%202018.%20A%20Consensus%20Report%20by%20the%20American%20Diabetes%20Association%20(ADA)%20and%20the%20European%20Association%20for%20the%20Study%20of%20Diabetes%20(EASD)&volume=43&publication_year=2020&pages=487-493&pmid=31857443&doi=10.2337/dci19-0066&)

4. Agency for Healthcare Research and Quality (Ed.). Total purchases in thousands by prescribed drug, United States, 1996-2018. In: Medical Expenditure Panel Survey. Rockville, MD, Agency for Healthcare Research and Quality, Rockville, Maryland, 2021. Accessed 1 August 2023. Available from https://datatools.ahrq.gov/meps-hc/?tab=prescribed-drugs&dash=18  [https://datatools.ahrq.gov/meps-hc/?tab=prescribed-drugs&dash=18](https://datatools.ahrq.gov/meps-hc/?tab=prescribed-drugs&dash=18) | [Google Scholar](https://scholar.google.com/scholar_lookup?title=Medical%20Expenditure%20Panel%20Survey&publication_year=2021&)

5. Rena G, Hardie DG, Pearson ER. The mechanisms of action of metformin. Diabetologia 2017;60:1577–1585  [DOI](https://doi.org/10.1007/s00125-017-4342-z) | [PMC free article](/articles/PMC5552828/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28776086/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Diabetologia&title=The%20mechanisms%20of%20action%20of%20metformin&volume=60&publication_year=2017&pages=1577-1585&pmid=28776086&doi=10.1007/s00125-017-4342-z&)

6. Zhou K, Donnelly L, Yang J, et al.; Wellcome Trust Case Control Consortium 2 . Heritability of variation in glycaemic response to metformin: a genome-wide complex trait analysis. Lancet Diabetes Endocrinol 2014;2:481–487  [DOI](https://doi.org/10.1016/S2213-8587(14)70050-6) | [PMC free article](/articles/PMC4038749/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24731673/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet%20Diabetes%20Endocrinol&title=Heritability%20of%20variation%20in%20glycaemic%20response%20to%20metformin:%20a%20genome-wide%20complex%20trait%20analysis&volume=2&publication_year=2014&pages=481-487&pmid=24731673&doi=10.1016/S2213-8587(14)70050-6&)

7. GoDarts and UKPDS Diabetes Pharmacogenetics Study Group; Wellcome Trust Case Control Consortium 2 . Common variants near ATM are associated with glycemic response to metformin in type 2 diabetes. Nat Genet 2011;43:117–120  [DOI](https://doi.org/10.1038/ng.735) | [PMC free article](/articles/PMC3030919/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21186350/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Common%20variants%20near%20ATM%20are%20associated%20with%20glycemic%20response%20to%20metformin%20in%20type%202%20diabetes&volume=43&publication_year=2011&pages=117-120&pmid=21186350&doi=10.1038/ng.735&)

8. van Leeuwen N, Nijpels G, Becker ML, et al. A gene variant near ATM is significantly associated with metformin treatment response in type 2 diabetes: a replication and meta-analysis of five cohorts. Diabetologia 2012;55:1971–1977  [DOI](https://doi.org/10.1007/s00125-012-2537-x) | [PMC free article](/articles/PMC3369131/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22453232/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Diabetologia&title=A%20gene%20variant%20near%20ATM%20is%20significantly%20associated%20with%20metformin%20treatment%20response%20in%20type%202%20diabetes:%20a%20replication%20and%20meta-analysis%20of%20five%20cohorts&volume=55&publication_year=2012&pages=1971-1977&pmid=22453232&doi=10.1007/s00125-012-2537-x&)

9. Zhou K, Yee SW, Seiser EL, et al.; MetGen Investigators; DPP Investigators; ACCORD Investigators . Variation in the glucose transporter gene SLC2A2 is associated with glycemic response to metformin. Nat Genet 2016;48:1055–1059  [DOI](https://doi.org/10.1038/ng.3632) | [PMC free article](/articles/PMC5007158/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27500523/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Variation%20in%20the%20glucose%20transporter%20gene%20SLC2A2%20is%20associated%20with%20glycemic%20response%20to%20metformin&volume=48&publication_year=2016&pages=1055-1059&pmid=27500523&doi=10.1038/ng.3632&)

10. Rotroff DM, Yee SW, Zhou K, et al.; MetGen Investigators; ACCORD/ACCORDion Investigators . Genetic variants in CPA6 and PRPF31 are associated with variation in response to metformin in individuals with type 2 diabetes. Diabetes 2018;67:1428–1440  [DOI](https://doi.org/10.2337/db17-1164) | [PMC free article](/articles/PMC6014560/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29650774/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Diabetes&title=Genetic%20variants%20in%20CPA6%20and%20PRPF31%20are%20associated%20with%20variation%20in%20response%20to%20metformin%20in%20individuals%20with%20type%202%20diabetes&volume=67&publication_year=2018&pages=1428-1440&pmid=29650774&doi=10.2337/db17-1164&)

11. Li JH, Perry JA, Jablonski KA, et al.; Diabetes Prevention Program Research Group . Identification of genetic variation influencing metformin response in a multiancestry genome-wide association study in the Diabetes Prevention Program (DPP). Diabetes 2023;72:1161–1172  [DOI](https://doi.org/10.2337/db22-0702) | [PMC free article](/articles/PMC10382652/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36525397/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Diabetes&title=Identification%20of%20genetic%20variation%20influencing%20metformin%20response%20in%20a%20multiancestry%20genome-wide%20association%20study%20in%20the%20Diabetes%20Prevention%20Program%20(DPP)&volume=72&publication_year=2023&pages=1161-1172&pmid=36525397&doi=10.2337/db22-0702&)

12. Levin AM, Gui H, Hernandez-Pacheco N, et al. Integrative approach identifies corticosteroid response variant in diverse populations with asthma. J Allergy Clin Immunol 2019;143:1791–1802  [DOI](https://doi.org/10.1016/j.jaci.2018.09.034) | [PMC free article](/articles/PMC6482107/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30367910/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Allergy%20Clin%20Immunol&title=Integrative%20approach%20identifies%20corticosteroid%20response%20variant%20in%20diverse%20populations%20with%20asthma&volume=143&publication_year=2019&pages=1791-1802&pmid=30367910&doi=10.1016/j.jaci.2018.09.034&)

13. Gui H, Levin AM, Hu D, et al. Mapping the 17q12-21.1 locus for variants associated with early-onset asthma in African Americans. Am J Respir Crit Care Med 2021;203:424–436  [DOI](https://doi.org/10.1164/rccm.202006-2623OC) | [PMC free article](/articles/PMC7885840/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32966749/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Respir%20Crit%20Care%20Med&title=Mapping%20the%2017q12-21.1%20locus%20for%20variants%20associated%20with%20early-onset%20asthma%20in%20African%20Americans&volume=203&publication_year=2021&pages=424-436&pmid=32966749&doi=10.1164/rccm.202006-2623OC&)

14. Goswami S, Yee SW, Xu F, et al. A longitudinal HbA1c model elucidates genes linked to disease progression on metformin. Clin Pharmacol Ther 2016;100:537–547  [DOI](https://doi.org/10.1002/cpt.428) | [PMC free article](/articles/PMC5534241/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27415606/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=A%20longitudinal%20HbA1c%20model%20elucidates%20genes%20linked%20to%20disease%20progression%20on%20metformin&volume=100&publication_year=2016&pages=537-547&pmid=27415606&doi=10.1002/cpt.428&)

15. Das S, Forer L, Schönherr S, et al. Next-generation genotype imputation service and methods. Nat Genet 2016;48:1284–1287  [DOI](https://doi.org/10.1038/ng.3656) | [PMC free article](/articles/PMC5157836/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27571263/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Next-generation%20genotype%20imputation%20service%20and%20methods&volume=48&publication_year=2016&pages=1284-1287&pmid=27571263&doi=10.1038/ng.3656&)

16. Patterson N, Price AL, Reich D. Population structure and eigenanalysis. PLoS Genet 2006;2:e190.  [DOI](https://doi.org/10.1371/journal.pgen.0020190) | [PMC free article](/articles/PMC1713260/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17194218/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet&title=Population%20structure%20and%20eigenanalysis&volume=2&publication_year=2006&pages=e190&pmid=17194218&doi=10.1371/journal.pgen.0020190&)

17. Price AL, Zaitlen NA, Reich D, Patterson N. New approaches to population stratification in genome-wide association studies. Nat Rev Genet 2010;11:459–463  [DOI](https://doi.org/10.1038/nrg2813) | [PMC free article](/articles/PMC2975875/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20548291/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Genet&title=New%20approaches%20to%20population%20stratification%20in%20genome-wide%20association%20studies&volume=11&publication_year=2010&pages=459-463&pmid=20548291&doi=10.1038/nrg2813&)

18. Chang CC, Chow CC, Tellier LC, Vattikuti S, Purcell SM, Lee JJ. Second-generation PLINK: rising to the challenge of larger and richer datasets. Gigascience 2015;4:7.  [DOI](https://doi.org/10.1186/s13742-015-0047-8) | [PMC free article](/articles/PMC4342193/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25722852/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gigascience&title=Second-generation%20PLINK:%20rising%20to%20the%20challenge%20of%20larger%20and%20richer%20datasets&volume=4&publication_year=2015&pages=7&pmid=25722852&doi=10.1186/s13742-015-0047-8&)

19. Maples BK, Gravel S, Kenny EE, Bustamante CD. RFMix: a discriminative modeling approach for rapid and robust local-ancestry inference. Am J Hum Genet 2013;93:278–288  [DOI](https://doi.org/10.1016/j.ajhg.2013.06.020) | [PMC free article](/articles/PMC3738819/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23910464/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=RFMix:%20a%20discriminative%20modeling%20approach%20for%20rapid%20and%20robust%20local-ancestry%20inference&volume=93&publication_year=2013&pages=278-288&pmid=23910464&doi=10.1016/j.ajhg.2013.06.020&)

20. Alexander DH, Novembre J, Lange K. Fast model-based estimation of ancestry in unrelated individuals. Genome Res 2009;19:1655–1664  [DOI](https://doi.org/10.1101/gr.094052.109) | [PMC free article](/articles/PMC2752134/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19648217/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res&title=Fast%20model-based%20estimation%20of%20ancestry%20in%20unrelated%20individuals&volume=19&publication_year=2009&pages=1655-1664&pmid=19648217&doi=10.1101/gr.094052.109&)

21. Genomes Project Consortium . An integrated map of genetic variation from 1,092 human genomes. Nature 2012;491:56–65  [DOI](https://doi.org/10.1038/nature11632) | [PMC free article](/articles/PMC3498066/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23128226/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=An%20integrated%20map%20of%20genetic%20variation%20from%201,092%20human%20genomes&volume=491&publication_year=2012&pages=56-65&pmid=23128226&doi=10.1038/nature11632&)

22. Williams LK, Padhukasahasram B, Ahmedani BK, et al. Differing effects of metformin on glycemic control by race-ethnicity. J Clin Endocrinol Metab 2014;99:3160–3168  [DOI](https://doi.org/10.1210/jc.2014-1539) | [PMC free article](/articles/PMC4154100/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24921653/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Endocrinol%20Metab&title=Differing%20effects%20of%20metformin%20on%20glycemic%20control%20by%20race-ethnicity&volume=99&publication_year=2014&pages=3160-3168&pmid=24921653&doi=10.1210/jc.2014-1539&)

23. Delgado C, Baweja M, Crews DC, et al. A unifying approach for GFR estimation: recommendations of the NKF-ASN Task Force on Reassessing the Inclusion of Race in Diagnosing Kidney Disease. Am J Kidney Dis 2022;79:268–288.e1  [DOI](https://doi.org/10.1053/j.ajkd.2021.08.003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34563581/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Kidney%20Dis&title=A%20unifying%20approach%20for%20GFR%20estimation:%20recommendations%20of%20the%20NKF-ASN%20Task%20Force%20on%20Reassessing%20the%20Inclusion%20of%20Race%20in%20Diagnosing%20Kidney%20Disease&volume=79&publication_year=2022&pages=268-288.e1&pmid=34563581&doi=10.1053/j.ajkd.2021.08.003&)

24. Pladevall M, Williams LK, Potts LA, Divine G, Xi H, Lafata JE. Clinical outcomes and adherence to medications measured by claims data in patients with diabetes. Diabetes Care 2004;27:2800–2805  [DOI](https://doi.org/10.2337/diacare.27.12.2800) | [PMC free article](/articles/PMC1262687/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15562188/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Diabetes%20Care&title=Clinical%20outcomes%20and%20adherence%20to%20medications%20measured%20by%20claims%20data%20in%20patients%20with%20diabetes&volume=27&publication_year=2004&pages=2800-2805&pmid=15562188&doi=10.2337/diacare.27.12.2800&)

25. McCaw ZR, Lane JM, Saxena R, Redline S, Lin X. Operating characteristics of the rank-based inverse normal transformation for quantitative trait analysis in genome-wide association studies. Biometrics 2020;76:1262–1272  [DOI](https://doi.org/10.1111/biom.13214) | [PMC free article](/articles/PMC8643141/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31883270/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biometrics&title=Operating%20characteristics%20of%20the%20rank-based%20inverse%20normal%20transformation%20for%20quantitative%20trait%20analysis%20in%20genome-wide%20association%20studies&volume=76&publication_year=2020&pages=1262-1272&pmid=31883270&doi=10.1111/biom.13214&)

26. Gogarten SM, Sofer T, Chen H, et al. Genetic association testing using the GENESIS R/Bioconductor package. Bioinformatics 2019;35:5346–5348  [DOI](https://doi.org/10.1093/bioinformatics/btz567) | [PMC free article](/articles/PMC7904076/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31329242/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=Genetic%20association%20testing%20using%20the%20GENESIS%20R/Bioconductor%20package&volume=35&publication_year=2019&pages=5346-5348&pmid=31329242&doi=10.1093/bioinformatics/btz567&)

27. Willer CJ, Li Y, Abecasis GR. METAL: fast and efficient meta-analysis of genomewide association scans. Bioinformatics 2010;26:2190–2191  [DOI](https://doi.org/10.1093/bioinformatics/btq340) | [PMC free article](/articles/PMC2922887/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20616382/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=METAL:%20fast%20and%20efficient%20meta-analysis%20of%20genomewide%20association%20scans&volume=26&publication_year=2010&pages=2190-2191&pmid=20616382&doi=10.1093/bioinformatics/btq340&)

28. Morales J, Welter D, Bowler EH, et al. A standardized framework for representation of ancestry data in genomics studies, with application to the NHGRI-EBI GWAS Catalog. Genome Biol 2018;19:21.  [DOI](https://doi.org/10.1186/s13059-018-1396-2) | [PMC free article](/articles/PMC5815218/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29448949/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Biol&title=A%20standardized%20framework%20for%20representation%20of%20ancestry%20data%20in%20genomics%20studies,%20with%20application%20to%20the%20NHGRI-EBI%20GWAS%20Catalog&volume=19&publication_year=2018&pages=21&pmid=29448949&doi=10.1186/s13059-018-1396-2&)

29. Sirugo G, Williams SM, Tishkoff SA. The missing diversity in human genetic studies. Cell 2019;177:26–31  [DOI](https://doi.org/10.1016/j.cell.2019.02.048) | [PMC free article](/articles/PMC7380073/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30901543/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell&title=The%20missing%20diversity%20in%20human%20genetic%20studies&volume=177&publication_year=2019&pages=26-31&pmid=30901543&doi=10.1016/j.cell.2019.02.048&)

30. Mills MC, Rahal C. The GWAS Diversity Monitor tracks diversity by disease in real time. Nat Genet 2020;52:242–243  [DOI](https://doi.org/10.1038/s41588-020-0580-y) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32139905/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=The%20GWAS%20Diversity%20Monitor%20tracks%20diversity%20by%20disease%20in%20real%20time&volume=52&publication_year=2020&pages=242-243&pmid=32139905&doi=10.1038/s41588-020-0580-y&)

31. Martin AR, Kanai M, Kamatani Y, Okada Y, Neale BM, Daly MJ. Clinical use of current polygenic risk scores may exacerbate health disparities. Nat Genet 2019;51:584–591  [DOI](https://doi.org/10.1038/s41588-019-0379-x) | [PMC free article](/articles/PMC6563838/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30926966/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Clinical%20use%20of%20current%20polygenic%20risk%20scores%20may%20exacerbate%20health%20disparities&volume=51&publication_year=2019&pages=584-591&pmid=30926966&doi=10.1038/s41588-019-0379-x&)

32. Gurdasani D, Barroso I, Zeggini E, Sandhu MS. Genomics of disease risk in globally diverse populations. Nat Rev Genet 2019;20:520–535  [DOI](https://doi.org/10.1038/s41576-019-0144-0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31235872/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Genet&title=Genomics%20of%20disease%20risk%20in%20globally%20diverse%20populations&volume=20&publication_year=2019&pages=520-535&pmid=31235872&doi=10.1038/s41576-019-0144-0&)

33. Li H, Wei S, Cheng K, et al. BIG3 inhibits insulin granule biogenesis and insulin secretion. EMBO Rep 2014;15:714–722  [DOI](https://doi.org/10.1002/embr.201338181) | [PMC free article](/articles/PMC4197882/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24711543/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=EMBO%20Rep&title=BIG3%20inhibits%20insulin%20granule%20biogenesis%20and%20insulin%20secretion&volume=15&publication_year=2014&pages=714-722&pmid=24711543&doi=10.1002/embr.201338181&)

34. Li H, Liu T, Lim J, Gounko NV, Hong W, Han W. Increased biogenesis of glucagon-containing secretory granules and glucagon secretion in BIG3-knockout mice. Mol Metab 2015;4:246–252  [DOI](https://doi.org/10.1016/j.molmet.2015.01.001) | [PMC free article](/articles/PMC4338310/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25737957/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Metab&title=Increased%20biogenesis%20of%20glucagon-containing%20secretory%20granules%20and%20glucagon%20secretion%20in%20BIG3-knockout%20mice&volume=4&publication_year=2015&pages=246-252&pmid=25737957&doi=10.1016/j.molmet.2015.01.001&)

35. Ward LD, Kellis M. HaploReg v4: systematic mining of putative causal variants, cell types, regulators and target genes for human complex traits and disease. Nucleic Acids Res 2016;44:D877–D881  [DOI](https://doi.org/10.1093/nar/gkv1340) | [PMC free article](/articles/PMC4702929/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26657631/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=HaploReg%20v4:%20systematic%20mining%20of%20putative%20causal%20variants,%20cell%20types,%20regulators%20and%20target%20genes%20for%20human%20complex%20traits%20and%20disease&volume=44&publication_year=2016&pages=D877-D881&pmid=26657631&doi=10.1093/nar/gkv1340&)

36. Kim JY, Park KJ, Hwang JY, et al. Activating transcription factor 3 is a target molecule linking hepatic steatosis to impaired glucose homeostasis. J Hepatol 2017;67:349–359  [DOI](https://doi.org/10.1016/j.jhep.2017.03.023) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28365312/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Hepatol&title=Activating%20transcription%20factor%203%20is%20a%20target%20molecule%20linking%20hepatic%20steatosis%20to%20impaired%20glucose%20homeostasis&volume=67&publication_year=2017&pages=349-359&pmid=28365312&doi=10.1016/j.jhep.2017.03.023&)

37. Allen-Jennings AE, Hartman MG, Kociba GJ, Hai T. The roles of ATF3 in liver dysfunction and the regulation of phosphoenolpyruvate carboxykinase gene expression. J Biol Chem 2002;277:20020–20025  [DOI](https://doi.org/10.1074/jbc.M200727200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11916968/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=The%20roles%20of%20ATF3%20in%20liver%20dysfunction%20and%20the%20regulation%20of%20phosphoenolpyruvate%20carboxykinase%20gene%20expression&volume=277&publication_year=2002&pages=20020-20025&pmid=11916968&doi=10.1074/jbc.M200727200&)

38. Luizon MR, Eckalbar WL, Wang Y, et al. Genomic characterization of metformin hepatic response. PLoS Genet 2016;12:e1006449.  [DOI](https://doi.org/10.1371/journal.pgen.1006449) | [PMC free article](/articles/PMC5130177/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27902686/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet&title=Genomic%20characterization%20of%20metformin%20hepatic%20response&volume=12&publication_year=2016&pages=e1006449&pmid=27902686&doi=10.1371/journal.pgen.1006449&)

39. Mahajan A, Spracklen CN, Zhang W, et al.; FinnGen; eMERGE Consortium . Multi-ancestry genetic study of type 2 diabetes highlights the power of diverse populations for discovery and translation. Nat Genet 2022;54:560–572  [DOI](https://doi.org/10.1038/s41588-022-01058-3) | [PMC free article](/articles/PMC9179018/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35551307/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Multi-ancestry%20genetic%20study%20of%20type%202%20diabetes%20highlights%20the%20power%20of%20diverse%20populations%20for%20discovery%20and%20translation&volume=54&publication_year=2022&pages=560-572&pmid=35551307&doi=10.1038/s41588-022-01058-3&)

40. LaMoia TE, Shulman GI. Cellular and molecular mechanisms of metformin action. Endocr Rev 2021;42:77–96  [DOI](https://doi.org/10.1210/endrev/bnaa023) | [PMC free article](/articles/PMC7846086/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32897388/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Endocr%20Rev&title=Cellular%20and%20molecular%20mechanisms%20of%20metformin%20action&volume=42&publication_year=2021&pages=77-96&pmid=32897388&doi=10.1210/endrev/bnaa023&)
