# In vitro comparative analysis of metabolic capabilities and inhibitory profiles of selected CYP2D6 alleles on tramadol metabolism

## Metadata
**Authors:** Noor Ahmed Nahid, Siva Rama Raju Kanumuri, Abhisheak Sharma, Danxin Wang, Julie A Johnson
**Journal:** Clinical and Translational Science
**Date:** 2025 Jan 27
**DOI:** [10.1111/cts.70059](https://doi.org/10.1111/cts.70059)
**PMID:** 39870079
**PMCID:** PMC11772017
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11772017/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC11772017/pdf/CTS-18-e70059.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC11772017/pdf/CTS-18-e70059.pdf)

## Abstract

Tramadol, the 41st most prescribed drug in the United States in 2021 is a prodrug activated by CYP2D6, which is highly polymorphic. Previous studies showed enzyme‐inhibitor affinity varied between different CYP2D6 allelic variants with dextromethorphan and atomoxetine metabolism. However, no study has compared tramadol metabolism in different CYP2D6 alleles with different CYP2D6 inhibitors. We hypothesize that the inhibitory effects of CYP2D6 inhibitors on CYP2D6‐mediated tramadol metabolism are inhibitor‐ and CYP2D6‐allele‐specific. We performed comparative analyses of CYP2D6*1, CYP2D6*2, CYP2D6*10, and CYP2D6*17 using recombinant enzymes to metabolize tramadol to O‐desmethyltramadol, measured via UPLC‐MS/MS. The Michaelis constant (Km) and maximum velocity (Vmax) for each CYP2D6 allele, and IC50 values for different inhibitors were determined by nonlinear regression analysis. Intrinsic clearance was calculated as Vmax/Km. The intrinsic clearance of tramadol was almost double for CYP2D6*2 (180%) but was much lower for CYP2D6*10 and *17 (20% and 10%, respectively) compared to CYP2D6*1. The inhibitor potencies (defined by Ki) for the various inhibitors for the CYP2D6*1 allele were quinidine > terbinafine > paroxetine ≈ duloxetine >>bupropion. CYP2D6*2 showed the next greatest inhibition, with Ki ratios compared to CYP2D6*1 ranging from 0.96 to 3.87. For each inhibitor tested, CYP2D6*10 and CYP2D6*17 were more resistant to inhibition than CYP2D6*1 or CYP2D6*2, with most Ki ratios in the 3–9 range. Three common CYP2D6 allelic variants showed different metabolic capacities toward tramadol and genotype‐dependent inhibition compared to CYP2D6*1. Further studies are warranted to understand the clinical consequences of inhibitor and CYP2D6 genotype‐dependent drug–drug interactions on tramadol bioactivation.

Previous studies showed metabolic capacity and enzyme‐inhibitor affinity varies between different CYP2D6 allelic variants. However, comparative data on the metabolism of tramadol, a commonly prescribed CYP2D6‐dependent prodrug, in different CYP2D6 alleles with or without the presence of different CYP2D6 inhibitors are limited.

What are the comparative metabolic capabilities of CYP2D6 alleles CYP2D6*1, CYP2D6*2, CYP2D6*10, and CYP2D6*17 in the metabolism of tramadol, and how do these allelic variants respond to inhibition by common CYP2D6 inhibitors?

Three common CYP2D6 allelic variants CYP2D6*2, CYP2D6*10, and CYP2D6*17 showed different metabolic capacities toward tramadol and genotype‐dependent enzyme‐inhibitor affinity as compared to CYP2D6*1. Moreover, we observed that CYP2D6*1 followed by CYP2D6*2 are more susceptible to inhibition than other variant alleles.

Different CYP2D6 alleles exhibit varying metabolic activities toward tramadol, and the inhibitory effects of CYP2D6 inhibitors on enzyme activity are specific to both the inhibitor and CYP2D6 allele. If validated through further studies, knowledge of such inhibitors and CYP2D6 genotype‐dependent drug–drug interactions could improve our ability to predict pain control by tramadol and be considered in clinical pharmacogenomics guidelines.

## INTRODUCTION

Tramadol is widely prescribed to treat moderate to severe pain and was the 41st most prescribed drug in the US in 2021.[1](#cts70059-bib-0001) Tramadol is classified as a prodrug, necessitating bioactivation by the drug‐metabolizing enzyme cytochrome P450 2D6 (CYP2D6) into an active metabolite for its analgesic effects, as the parent compound exhibits little to no analgesic activity compared to the active metabolite.[2](#cts70059-bib-0002), [3](#cts70059-bib-0003), [4](#cts70059-bib-0004)

The drug‐metabolizing enzyme CYP2D6 is encoded by *CYP2D6*, which is highly polymorphic, with more than 100 alleles identified.[5](#cts70059-bib-0005) Many of the CYP2D6 variant alleles display little to no enzymatic activity, resulting in an intermediate (IM) or poor metabolizer (PM) phenotype. Gene duplication/multiplication can also occur, leading to an ultra‐rapid metabolizer phenotype (UM) when fully functional alleles are duplicated.[3](#cts70059-bib-0003) In this study, we focused on CYP2D6*1, CYP2D6*2, CYP2D6*10, and CYP2D6*17. The CYP2D6*1 and CYP2D6*2 alleles are considered normal functions, each with an activity score of 1.0. The CYP2D6*17 allele exhibits reduced function with an activity score of 0.5, while CYP2D6*10 exhibits much‐reduced function with an activity score of 0.25.[6](#cts70059-bib-0006) CYP2D6 phenotype is also influenced by drug interactions via enzyme inhibition by several commonly used medications (e.g., fluoxetine, bupropion, duloxetine). The FDA defines specific drugs as strong, moderate, or weak CYP2D6 inhibitors defined by the increase in AUC of sensitive index substrates. Specifically, a strong inhibitor increases plasma concentration (i.e., AUC) ≥5‐fold, whereas moderate and weak inhibitors increase AUC by ≥2 to <5‐fold and ≥1.25 to <2‐fold, respectively.[7](#cts70059-bib-0007) Strong and moderate inhibitors cause phenoconversion of CYP2D6*1 to PM or IM phenotypes, respectively,[6](#cts70059-bib-0006), [8](#cts70059-bib-0008) while weak inhibitors are generally considered clinically insignificant.[7](#cts70059-bib-0007), [9](#cts70059-bib-0009), [10](#cts70059-bib-0010) Therefore, both genetic variants and drug–drug interactions influence CYP2D6 metabolic phenotypes.

CYP2D6 activity is typically determined using dextromethorphan and debrisoquine as probe drugs.[11](#cts70059-bib-0011) Clinically, the activity of the alleles is treated as constant across drugs.[6](#cts70059-bib-0006) Thus, the enzymatic activity obtained with one probe substrate is usually extrapolated to all substrates of the same enzyme.[12](#cts70059-bib-0012) Moreover, the potential impact of CYP coding variants on CYP‐substrate interactions is often not considered.[11](#cts70059-bib-0011) Collecting data from multiple in vitro kinetic studies showed that different variants of CYP2D6 alleles have varied specificity toward given substrates.[11](#cts70059-bib-0011) When the same inhibitor is evaluated using different probes for the same P450 enzyme activity, the outcome of the drug–drug interactions can also be different.[12](#cts70059-bib-0012) For example, Wang et al. examined the mutual inhibition among the four commonly used CYP3A4 substrates testosterone, terfenadine, midazolam, and nifedipine. They found that although testosterone partially inhibits the hydroxylation of terfenadine and midazolam, it does not inhibit nifedipine oxidation,[13](#cts70059-bib-0013) Moreover, mixed effects were obtained on the functionally reduced CYP2D6 variants in enzyme‐substrate or enzyme‐inhibitor affinity, which is lower, higher, or comparable to that for CYP2D6*1.[14](#cts70059-bib-0014), [15](#cts70059-bib-0015), [16](#cts70059-bib-0016), [17](#cts70059-bib-0017) These studies suggest CYP‐substrate/inhibitors interactions may be genotype and substrate‐specific and raise questions of direct extrapolation of results from one substrate with CYP2D6*1 to other substrates and CYP2D6 variants.[14](#cts70059-bib-0014) The dependence of tramadol on CYP2D6 for bioactivation and concurrent use of tramadol with CYP2D6 inhibitors pose a risk of drug–drug interaction and treatment failure with tramadol. To provide more effective treatment with tramadol, the Clinical Pharmacogenetics Implementation Consortium (CPIC) provides guidelines for using *CYP2D6* data with tramadol in the clinical setting.[6](#cts70059-bib-0006) However, data on tramadol biotransformation in different CYP2D6 alleles in the presence or absence of varying CYP2D6 inhibitors are inadequate.

In this study, we aimed to compare the metabolic capabilities of CYP2D6 alleles (CYP2D6*1, CYP2D6*2, CYP2D6*10, and CYP2D6*17) in the metabolism of tramadol and to compare the inhibition potency of several CYP2D6 inhibitors that the FDA has classified as strong (quinidine, terbinafine, paroxetine, and bupropion) or moderate CYP2D6 inhibitor (duloxetine). These inhibitors were chosen for study because quinidine and terbinafine are known potent CYP2D6 inhibitors, and bupropion, paroxetine, and duloxetine are widely used in patients who are also prescribed tramadol. We hypothesized that the metabolic activities toward tramadol metabolism differ among CYP2D6 alleles and that inhibitory effects of CYP2D6 inhibitors on CYP2D6‐mediated tramadol biotransformation are inhibitor and *CYP2D6* allele (protein‐coding variant) dependent.

## MATERIALS AND METHODS

### Materials

The CYP2D6 substrate tramadol hydrochloride, the metabolite‐ *O*‐desmethyl tramadol hydrochloride, the internal standard of the metabolite *O*‐desmethyl tramadol‐*D*6 hydrochloride, the CYP2D6 inhibitors‐ quinidine, terbinafine, duloxetine, bupropion, and paroxetine, were purchased from Cayman Chemical (Ann Arbor, MI, USA). NADPH regeneration system (NADP+, glucose‐6‐phosphate, and the enzyme glucose‐6‐phosphate dehydrogenase) was obtained from Promega (Madison, WI, USA). Full‐length Human CYP2D6*1, CYP2D6*2, CYP2D6*10, and CYP2D6*17 EasyCYP Bactosomes® were purchased from BioIVT (Westbury, NY, USA). These Bactosomes are prepared using the Human CYP2D6 alleles and human CYP‐reductase coexpressed in Escherichia coli.[18](#cts70059-bib-0018) All other chemicals and reagents were commercially available and of the highest analytical grade.

### Comparison of the Enzyme Kinetics of CYP2D6*1, CYP2D6*2, CYP2D6*10, and CYP2D6*17

Various concentrations of tramadol (0.1 μM to 1000 μM) were incubated with four CYP2D6 alleles, that is, CYP2D6*1, CYP2D6*2, CYP2D6*10 and CYP2D6*17. Incubations were performed in duplicate in 96‐well plates in a total assay volume of 200 μL with each well containing CYP2D6 Bactosomes® (1 pmol P450/mL), substrate (10 different concentrations within the range), 100 mM potassium phosphate buffer containing 5 mM MgCl_2_ (pH 7.4). The mixture was preincubated for 3 min at 37°C before a 10 mM NADPH‐generation system was added to initiate the reaction. The incubation time for biotransformation was 30 minutes, which was observed to be optimal from the time course experiment, and at that time the rate of product formation remained linear for time and enzyme concentration. After incubation, the reaction was terminated by adding a fourfold volume of ice‐cold methanol (with 0.05% formic acid) containing 10 ng/mL of internal standard. The resulting samples were centrifuged at 1500 rpm for 3 min at 5°C to remove protein before the supernatant was applied to the ultra‐performance liquid chromatography–tandem mass spectrometry (UPLC‐MS/MS) analysis.

### In Vitro Inhibition Study with CYP2D6 Inhibitors

The half‐maximal inhibitory concentration (IC_50_) of the CYP2D6 inhibitors on the biotransformation of tramadol *O*‐demethylation of four CYP2D6 alleles was measured. The inhibition constant (K_i_) was then estimated from Equation ([1](#cts70059-disp-0001)).[19](#cts70059-bib-0019)

| IC50=Ki1+Skm | (1) |
| --- | --- |
| So,IfS=kmthenKi=IC502 | (2) |
| --- | --- |
For determination of the IC_50_, kinetic studies were performed with the K_m_ concentrations of tramadol for each allele (60 μM for CYP2D6*1 and CYP2D6*2, 204 μM for CYP2D6*10, and 150 μM for CYP2D6*17) and five CYP2D6 inhibitors (quinidine, terbinafine, duloxetine, bupropion, and paroxetine). Eight different concentrations of each inhibitor were used with ranges of 2–5000 μM for quinidine, 10–20,000 μM for terbinafine, 10–20,000 μM for duloxetine, 50–50,000 μM for bupropion, and 50–10,000 μM for paroxetine. These ranges were selected to reflect the varying potencies of each inhibitor, ensuring comprehensive coverage of their inhibitory effects on the target enzyme. Incubations were performed in duplicate in 96‐well plates in a total assay volume of 100 μL CYP2D6 Bactosomes (1 pmol P450/mL), substrate, 100 mM potassium phosphate buffer containing 5 mM MgCl2 (pH 7.4). Before the commencement of the reaction, by adding a 10 mM NADPH‐generation system, the incubation mixture was preincubated for 3 min at 37°C. The resulting samples were then centrifuged before the supernatant was applied to UPLC‐MS/MS analysis.

### UPLC‐MS/MS Analysis

*O*‐desmethyltramadol, the active metabolite of tramadol, was quantified using the UPLC‐MS/MS method. The UPLC‐MS/MS was a Waters Acquity Class‐I chromatograph coupled with a Xevo TQ‐S Micro triple quadrupole mass spectrometer (Waters, Milford, MA). Waters Acquity UPLC BEH C18 column (1.7 mm, 2.1 × 50 mm) fitted with a Vanguard precolumn of the same chemistry was used with an aqueous mobile phase of 0.1% formic acid in water (A), and an organic mobile phase of 0.1% formic acid in acetonitrile (B). Gradient elution was used where 80% A was supplied for 0.5 min and then decreased linearly to 68% up to 2.2 min and then 62% up to 3.5 min, after which it was raised back up to 80%, where it was maintained until 5.5 min, at a flow rate of 0.35 mL/min. Mass spectrometric analysis was performed via electrospray ionization in positive mode, and the mass transitions of *m/z* 250.18 → 57.92 were monitored for *O*‐desmethyltramadol. The inter‐day and intraday assay precision, as well as accuracy, were within 10%.

### Data analysis

Michaelis constant (K_m_) and maximum velocity (V_max_) values were determined by nonlinear regression analysis using the Michaelis–Menten equation (rate of metabolite formation as a function of substrate concentration) using GraphPad Prism (version 10.2.3). The *r*‐squared value of goodness of fit was 0.9663, 0.9844, 0.9889, and 0.9865 for CYP2D6*1, CYP2D6*2, CYP2D6*10 and CYP2D6*17 respectively. The ratio of V_max_ and K_m_ was used to calculate in vitro intrinsic clearance. For the inhibition assays, K_i_ values were estimated from IC_50_ values. K_i_ ratio (K_i_ of the variant allele)/K_i_ of the wild‐type allele (CYP2D6*1) was then calculated to compare the inhibitor potency of the inhibitors. All data visualizations were performed using GraphPad Prism software.

## RESULTS

### Enzyme Kinetics of CYP2D6*1, CYP2D6*2, CYP2D6*10, and CYP2D6*17

Figure [1](#cts70059-fig-0001) shows the plots of tramadol concentration versus *O*‐desmethyltramadol formation rate with four CYP2D6 alleles: CYP2D6*1, CYP2D6*2, CYP2D6*10, and CYP2D6*17. The parameters K_m_ and V_max_ for tramadol and the corresponding in vitro intrinsic clearance (CL_int_ = V_max_/K_m_) values for each allele are presented in Table [1](#cts70059-tbl-0001). The intrinsic clearance ratio related to CYP2D6*1 was also calculated to provide relative comparisons of the other three CYP2D6 alleles to CYP2D6*1. Compared to CYP2D6*1, CYP2D6*2 had a similar K_m_ but nearly twofold higher V_max_, resulting in 80% higher intrinsic clearance than CYP2D6*1. For CYP2D6*10 and CYP2D6*17, the K_m_ values were very high and V_max_ values were very low, resulting in 20% and 10% intrinsic clearance compared to CYP2D6*1. The estimated tramadol biotransformation intrinsic clearance values decreased in the order CYP2D6*2 > CYP2D6*1 > CYP2D6*10 > CYP2D6*17 (Table [1](#cts70059-tbl-0001)).

### FIGURE 1.

![FIGURE 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/08e9/11772017/f6a89cf1935c/CTS-18-e70059-g002.jpg)

Enzyme kinetic plot of O‐desmethyl tramadol formation versus tramadol concentration in different CYP2D6 alleles. The error bars indicate the standard deviation of a data set relative to the mean.

### TABLE 1.

| Different alleles | Km | Vmax | CLint | CLint ratio |
| --- | --- | --- | --- | --- |
| μM | pmol/min/pmol P450 | μL/min/pmol P450 | CYP2D6*X/CYP2D6*1 |  |
| CYP2D6*1 | 59.6 | 22.6 | 0.4 |   |
| CYP2D6*2 | 59.4 | 39.7 | 0.7 | 1.8 |
| CYP2D6*10 | 204.2 | 13.6 | 0.1 | 0.2 |
| CYP2D6*17 | 149.0 | 5.6 | 0.0 | 0.1 |

Table 1 Caption: Comparison of enzyme kinetic parameters for CYP2D6*1, CYP2D6*2, CYP2D6*10, and CYP2D6*17.

### Inhibition of CYP2D6‐mediated tramadol biotransformation in CYP2D6*1, CYP2D6*2, CYP2D6*10, and CYP2D6*17

Figure [2](#cts70059-fig-0002) shows the inhibition curves for different inhibitors on different CYP2D6 alleles. Table [2](#cts70059-tbl-0002) lists the K_i_ values for different inhibitors among different alleles and the K_i_ ratio for the variant alleles as compared to the CYP2D6*1 allele. The five inhibitors showed varied inhibitory potency toward CYP2D6*1 with K_i_ values ranging from 4.4 nM to 12 μM. CYP2D6 variant alleles showed increased K_i_ values compared to CYP2D6*1 for all five inhibitors except for the bupropion/CYP2D6*2 pair. Quinidine showed the lowest K_i_ values for CYP2D6 alleles, consistent with quinidine's known potent inhibition of CYP2D6. Terbinafine is also classified as a strong CYP2D6 inhibitor and showed a low K_i_ for the CYP2D6*1 allele and higher K_i_ values for the variant alleles. Although bupropion is classified as a strong CYP2D6 inhibitor by the FDA, our study showed very high K_i_ for bupropion that differed among the various CYP2D6 alleles. Duloxetine is a moderate inhibitor, and the K_i_ value of duloxetine was 0.2559 μM for the CYP2D6*1 allele, with the K_i_ ratio of the other CYP2D6 alleles vs. CYP2D6*1 ranging from 2.4 to 8. Although paroxetine is a strong inhibitor, the K_i_ value was similar to duloxetine in the CYP2D6*1 allele, suggesting moderate inhibitory capability against tramadol metabolism. In general, CYP2D6 variant alleles (CYP2D6*2, CYP2D6*10, and CYP2D6*17) showed 1.6–5.6 times higher K_i_ values compared to CYP2D6*1, indicating weaker inhibition by CYP2D6 inhibitors of the variant alleles.

### FIGURE 2.

![FIGURE 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/08e9/11772017/c979bd60217f/CTS-18-e70059-g001.jpg)

Inhibition curves for various CYP2D6 inhibitors in different CYP2D6 alleles. The error bars indicate the standard deviation of a data set relative to the mean.

### TABLE 2.

| Inhibitors | Ki (μM) |  |  |  | Ki Ratio |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- |
| CYP2D6*1 | CYP2D6*2 | CYP2D6*10 | CYP2D6*17 | CYP2D6*2/CYP2D6*1 | CYP2D6*10/CYP2D6*1 | CYP2D6*17/CYP2D6*1 |  |
| Quinidine | 0.0044 | 0.0129 | 0.0198 | 0.0417 | 2.93 | 4.50 | 9.48 |
| Terbinafine | 0.0173 | 0.0670 | 0.2355 | 0.8437 | 3.87 | 13.61 | 48.77 |
| Duloxetine | 0.2559 | 0.6142 | 2.0090 | 0.7864 | 2.40 | 7.85 | 3.07 |
| Bupropion | 11.9260 | 11.5035 | 55.9084 | 20.2406 | 0.96 | 4.69 | 1.70 |
| Paroxetine | 0.2143 | 0.3468 | 0.6203 | 1.1940 | 1.62 | 2.89 | 5.57 |

Table 2 Caption: Inhibition of tramadol‐o‐demethylation by different inhibitors and different CYP2D6 alleles.

## DISCUSSION

In this study, we measured and compared tramadol intrinsic clearance of four *CYP2D6* alleles using EasyCYP Bactosomes. Our results showed that the CYP2D6‐mediated tramadol biotransformation varied among different CYP2D6 alleles. Additionally, we found that the inhibitory potency of different CYP2D6 inhibitors against CYP2D6‐mediated tramadol biotransformation varied markedly among five inhibitors and four CYP2D6 alleles, indicating that the inhibitory effects of CYP2D6 inhibitors on tramadol biotransformation are inhibitor and CYP2D6 genotype (protein‐coding variant) dependent. This is the first study comparing the inhibition of five CYP2D6 inhibitors on the four most common CYP2D6 coding alleles using tramadol as a substrate in an in vitro bactosomes system.

### Impact of CYP2D6 genotype on CYP2D6‐mediated tramadol biotransformation

CYP2D6*2 is considered a normal activity allele with an assigned activity score of 1.[6](#cts70059-bib-0006) We observed the intrinsic clearance of CYP2D6*2 higher than the assigned consensus activity score. The results are consistent with other studies that showed CYP2D6*2 displayed the largest variation in activity for different substrates.[11](#cts70059-bib-0011), [16](#cts70059-bib-0016), [20](#cts70059-bib-0020), [21](#cts70059-bib-0021), [22](#cts70059-bib-0022), [23](#cts70059-bib-0023), [24](#cts70059-bib-0024) The CYP2D6*2 haplotype is characterized by the 2851C>T (R296C) and 4181C>G (S486T) variants, which may induce structural changes within the enzyme's active site. These morphological alterations are thought to enhance ligand docking and binding, leading to increased intrinsic clearance of substrates such as venlafaxine.[11](#cts70059-bib-0011), [25](#cts70059-bib-0025)

CYP2D6*10 is a reduced activity allele with an assigned consensus activity score of 0.25.[6](#cts70059-bib-0006) We found the intrinsic clearance of tramadol by CYP2D6*10 was 20% of that by CYP2D6*1, consistent with the assigned activity score (0.25).[6](#cts70059-bib-0006) Shen et al.,[14](#cts70059-bib-0014) using microsomes prepared from insect cells, observed the intrinsic clearance value of tramadol for CYP2D6*10 was 6.9% of that for CYP2D6*1. The SNPs included in the characterization of CYP2D6*10 are 4181C>G(S486T) and 100C>T(P34S) of which P34S is found to cause reduced clearance of various drugs which might be due to the change to the active site that decreases the access to the ligand binding site.[11](#cts70059-bib-0011), [25](#cts70059-bib-0025)

CYP2D6*17 is also a reduced activity allele with an assigned activity score of 0.5.[6](#cts70059-bib-0006) However, the intrinsic clearance of CYP2D6*17 in our study was very low relative to CYP2D6*1, substantially lower than the clearance expected from the activity score of 0.5 (or 50%)[6](#cts70059-bib-0006) and the findings (36%) reported by Shen et al.[14](#cts70059-bib-0014) when using tramadol as substrate. The reason for this discrepancy is unclear and may arise from enzyme sources, and incubation conditions. CYP2D6*17 also showed different ranges of activity for other substrates,[14](#cts70059-bib-0014), [16](#cts70059-bib-0016), [20](#cts70059-bib-0020), [21](#cts70059-bib-0021), [24](#cts70059-bib-0024) and more in line with our observations, Lee et al. reported that, on average, the in vitro CYP2D6*17 activity for different substrates was 0.28 ± 0.21 (range: 0.06–0.80).[11](#cts70059-bib-0011) CYP2D6*17 is characterized by 2851C>T (R296C), 4181C>G (S486T), and 1022C>T(T107I). T107I was observed to change the morphology decreasing the binding of the ligand to the active site and thus was found to decrease the clearance of various drugs.[11](#cts70059-bib-0011), [25](#cts70059-bib-0025)

Most in vitro studies assessing CYP2D6 activity have focused on the common CYP2D6 probe substrates, such as dextromethorphan or bufuralol. Inter‐substrate variability for CYP2D6 alleles is prevalent,[11](#cts70059-bib-0011) and predicting the extent to which metabolism is affected by a specific variant for different substrates is challenging.[17](#cts70059-bib-0017), [23](#cts70059-bib-0023), [26](#cts70059-bib-0026) These previous results and our results highlight the necessity of measuring tramadol biotransformation, instead of relying on data from probe drugs, for more accurately assessing the capacity of different CYP2D6 alleles on CYP2D6‐mediated tramadol biotransformation. As tramadol is a prodrug that depends on CYP2D6 activity for biotransformation to active metabolites, higher activity of CYP2D6*2 would be expected to produce higher amounts of active metabolites of tramadol, with potential for increased pain relief efficacy and/or increased risk of tramadol adverse effects that may require dose reduction.

### CYP2D6 genotype‐specific inhibition

FDA has classified 11 drugs as CYP2D6 strong and moderate inhibitors.[7](#cts70059-bib-0007) Of those, we studied five. To compare the inhibitory potency of different inhibitors across different CYP2D6 alleles, we estimated the K_i_ from Equation [2](#cts70059-disp-0002) as we described in the methods section using a substrate concentration equal to K_m_ for the specific alleles. A retrospective analysis of 343 in vitro experiments suggests that, under appropriate experimental conditions with a substrate concentration equal to K_m_, values of K_i_ for direct, reversible inhibition can be reliably estimated from values of IC_50_.[27](#cts70059-bib-0027) Thus, our calculated K_i_ from IC_50_ should reliably estimate the differences in inhibitory potencies of different inhibitors among different CYP2D6 alleles.

The binding affinity (K_i_) of different inhibitors varied among the different variants in the presence of tramadol (Table [2](#cts70059-tbl-0002)). CYP2D6*1 demonstrated the greatest susceptibility to inhibition by all five inhibitors studied compared to the CYP2D6 variant alleles. For CYP2D6*1, the Ki for quinidine was the lowest (indicating the highest inhibitory potency), followed by terbinafine, paroxetine, and duloxetine, with the highest Ki observed for bupropion.

CYP2D6*2 is the second most susceptible to inhibition among the alleles studied with Ki twofold to fourfold higher than CYP2D6*1. Akiyoshi et al. reported that the K_i_ of terbinafine for CYP2D6*2 was seven times higher than that of CYP2D6*1 when the metabolism of dextromethorphan was studied.[28](#cts70059-bib-0028) Our results indicate that CYP2D6 variant alleles that showed reduced affinity (higher K_m_) for tramadol compared to CYP2D6*1 also showed reduced affinity for inhibitors (higher K_i_). Although CYP2D6*2 showed increased intrinsic clearance (Table [1](#cts70059-tbl-0001)), this increase in clearance is likely driven by its increased velocity rather than binding affinity toward tramadol because the Km for CYP2D6*2 did not differ from CYP2D6*1. Thus, compared to other CYP2D6 variant alleles, CYP2D6*2 appears to be more variable toward different substrates or inhibitors and warrants further investigation. CYP2D6*10 and CYP2D6*17 were more resistant to inhibition by the inhibitors than CYP2D6*1 or CYP2D6*2, with most Ki ratios in the 3–9 range.

Quinidine is a strong inhibitor of CYP2D6.[29](#cts70059-bib-0029) We also observed high binding affinity and inhibitory potency (IC_50_ values were listed in Table [S1](#cts70059-supitem-0001)) of quinidine across all the CYP2D6 alleles, although allele‐specific effects were observed for tramadol metabolism. Though bupropion is classified as a strong inhibitor of CYP2D6 by the FDA, data of K_i_ values from our study suggest bupropion is a less potent inhibitor of CYP2D6‐mediated metabolism of tramadol in vitro. However, a previous study reported that in vivo drug–drug interactions caused by bupropion cannot be explained by the in vitro data,[30](#cts70059-bib-0030) indicating other factors (e.g., the impact of other metabolites or other CYP enzymes) may influence bupropion inhibition that are not captured in in vitro assays. Terbinafine, duloxetine, and paroxetine also showed varying binding affinity for different CYP2D6 alleles. In a previous study, terbinafine exhibited K_i_ values 6.8 and 35 times higher for the CYP2D6*2 and CYP2D6*17 alleles, respectively, compared to CYP2D6*1, using the substrate dextromethorphan.[28](#cts70059-bib-0028) These findings align with our results, albeit with varying magnitudes. Overall, the data suggest the inhibitory potency of an inhibitor may vary between different CYP2D6 variant alleles and be dependent on the substrate being inhibited.

### Clinical implications

Our results showed the clearance of tramadol by CYP2D6*10 and CYP2D6*17 alleles was already very low, raising questions about the clinical importance of considering the impact of CYP2D6 inhibitors on tramadol biotransformation in individuals carrying those alleles. However, CYP2D6*2 showed increased intrinsic clearance but less susceptibility to inhibition than CYP2D6*1. Thus inhibition of tramadol metabolism via CYP2D6*2 appears to be clinically important, which requires further investigation.

The results reported here have potential implications for treatment outcomes with tramadol, precision medicine, and translational research. Our data suggest allele and substrate‐specific effects should be determined for commonly used drugs rather than relying on data from probe substrates or classical inhibitors. Replication of our results in further studies could have important implications for improving our ability to predict pain control by tramadol and could be considered in the clinical pharmacogenetics guidelines.

There are some limitations in our study. Although as described above, the estimation of K_i_ from IC_50_ in situations that our study met is reliable, K_i_ values obtained from different substrate concentrations would have been more precise. The in vitro study used recombinant enzyme Bactosomes may not fully replicate the complexity of human physiology. The advantage of using recombinant enzymes is the influence from other enzymes was excluded. However, we do not know the effect of other CYP enzymes on the metabolism of the inhibitors, whether such metabolism affects CYP2D6 inhibition in vivo, or other physiological factors that would impair the in vitro to in vivo interpretation.

In conclusion, this study highlights the different metabolic activities of CYP2D6 variant alleles in metabolizing tramadol compared to the CYP2D6*1 allele. Additionally, it shows the differences in inhibitory potency of various CYP2D6 inhibitors on tramadol metabolism among CYP2D6 genetic variants. Specifically, all the variant alleles exhibited less sensitivity to inhibition by the inhibitors compared to the CYP2D6*1 allele and there is variability within the CYP2D6*1 allele across different inhibitors (even within the same inhibitor strength category as classified by FDA). Additional investigations with liver microsomes and in vivo studies are warranted to confirm the findings. Incorporating substrate and allele‐specific enzymatic and inhibition effects in clinical pharmacogenetics guidelines could further enhance treatment outcomes with tramadol.

## AUTHOR CONTRIBUTIONS

N.A.N., D.W., and J.A.J. wrote the manuscript; N.A.N., D.W., J.A.J., and A.S. designed the research; N.A.N. and S.R.R.K. performed the research; N.A.N., S.R.R.K., and D.W. analyzed the data; S.R.R.K, and A.S contributed analytical tools.

## FUNDING INFORMATION

The paper has been partially supported by the NIH R35 GM140845 to DW.

## CONFLICT OF INTEREST STATEMENT

The authors declared no competing interests for this work.

## Supporting information

Nahid NA, Kanumuri SRR, Sharma A, Wang D, Johnson JA. In vitro comparative analysis of metabolic capabilities and inhibitory profiles of selected CYP2D6 alleles on tramadol metabolism. Clin Transl Sci. 2025;18:e70059. doi: 10.1111/cts.70059

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. The Top 300 of 2022. Accessed January 28, 2024. https://clincalc.com/DrugStats/Top300Drugs.aspx  [https://clincalc.com/DrugStats/Top300Drugs.aspx](https://clincalc.com/DrugStats/Top300Drugs.aspx)

2. Volpe DA, McMahon Tobin GA, Mellon RD, et al. Uniform assessment and ranking of opioid μ receptor binding constants for selected opioid drugs. Regul Toxicol Pharmacol. 2011;59(3):385‐390.  [DOI](https://doi.org/10.1016/j.yrtph.2010.12.007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21215785/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Regul%20Toxicol%20Pharmacol&title=Uniform%20assessment%20and%20ranking%20of%20opioid%20%CE%BC%20receptor%20binding%20constants%20for%20selected%20opioid%20drugs&author=DA%20Volpe&author=GA%20McMahon%20Tobin&author=RD%20Mellon&volume=59&issue=3&publication_year=2011&pages=385-390&pmid=21215785&doi=10.1016/j.yrtph.2010.12.007&)

3. Crews KR, Gaedigk A, Dunnenberger HM, et al. Clinical pharmacogenetics implementation consortium guidelines for cytochrome P450 2D6 genotype and codeine therapy: 2014 update. Clin Pharmacol Ther. 2014;95(4):376‐382.  [DOI](https://doi.org/10.1038/clpt.2013.254) | [PMC free article](/articles/PMC3975212/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24458010/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Clinical%20pharmacogenetics%20implementation%20consortium%20guidelines%20for%20cytochrome%20P450%202D6%20genotype%20and%20codeine%20therapy:%202014%20update&author=KR%20Crews&author=A%20Gaedigk&author=HM%20Dunnenberger&volume=95&issue=4&publication_year=2014&pages=376-382&pmid=24458010&doi=10.1038/clpt.2013.254&)

4. Foster A, Mobley E, Wang Z. Complicated pain management in a CYP450 2D6 poor metabolizer. Pain Pract. 2007;7(4):352‐356.  [DOI](https://doi.org/10.1111/j.1533-2500.2007.00153.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17986163/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pain%20Pract&title=Complicated%20pain%20management%20in%20a%20CYP450%202D6%20poor%20metabolizer&author=A%20Foster&author=E%20Mobley&author=Z%20Wang&volume=7&issue=4&publication_year=2007&pages=352-356&pmid=17986163&doi=10.1111/j.1533-2500.2007.00153.x&)

5. Pharmacogene Variation Consortium (PharmVar) . (Gaedigk et al. 2018, CPT 103:399; Gaedigk et al. 2019, CPT 105:29). Accessed June 8, 2022. www.PharmVar.org  [www.PharmVar.org](http://www.pharmvar.org)

6. Crews KR, Monte AA, Huddart R, et al. Clinical pharmacogenetics implementation consortium guideline for CYP2D6, OPRM1, and COMT genotypes and select opioid therapy. Clin Pharmacol Ther. 2021;110(4):888‐896.  [DOI](https://doi.org/10.1002/cpt.2149) | [PMC free article](/articles/PMC8249478/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33387367/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Clinical%20pharmacogenetics%20implementation%20consortium%20guideline%20for%20CYP2D6,%20OPRM1,%20and%20COMT%20genotypes%20and%20select%20opioid%20therapy&author=KR%20Crews&author=AA%20Monte&author=R%20Huddart&volume=110&issue=4&publication_year=2021&pages=888-896&pmid=33387367&doi=10.1002/cpt.2149&)

7. U.S. Food and Drug Administration . Drug Development and Drug Interactions: Table of Substrates, Inhibitors and Inducers. Accessed June 8, 2022. https://www.fda.gov/drugs/developmentapprovalprocess/developmentresources/druginteractionslabeling/ucm093664.htm  [https://www.fda.gov/drugs/developmentapprovalprocess/developmentresources/druginteractionslabeling/ucm093664.htm](https://www.fda.gov/drugs/developmentapprovalprocess/developmentresources/druginteractionslabeling/ucm093664.htm)

8. Frost DA, Soric MM, Kaiser R, Neugebauer RE. Efficacy of tramadol for pain management in patients receiving strong cytochrome P450 2D6 inhibitors. Pharmacotherapy. 2019;39(6):724‐729.  [DOI](https://doi.org/10.1002/phar.2269) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31038218/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacotherapy&title=Efficacy%20of%20tramadol%20for%20pain%20management%20in%20patients%20receiving%20strong%20cytochrome%20P450%202D6%20inhibitors&author=DA%20Frost&author=MM%20Soric&author=R%20Kaiser&author=RE%20Neugebauer&volume=39&issue=6&publication_year=2019&pages=724-729&pmid=31038218&doi=10.1002/phar.2269&)

9. Nahid NA, Johnson JA. CYP2D6 pharmacogenetics and phenoconversion in personalized medicine. Expert Opin Drug Metab Toxicol. 2022;18(11):769‐785.  [DOI](https://doi.org/10.1080/17425255.2022.2160317) | [PMC free article](/articles/PMC9891304/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36597259/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Expert%20Opin%20Drug%20Metab%20Toxicol&title=CYP2D6%20pharmacogenetics%20and%20phenoconversion%20in%20personalized%20medicine&author=NA%20Nahid&author=JA%20Johnson&volume=18&issue=11&publication_year=2022&pages=769-785&pmid=36597259&doi=10.1080/17425255.2022.2160317&)

10. Cicali EJ, Elchynski AL, Cook KJ, et al. How to integrate CYP2D6 Phenoconversion into clinical pharmacogenetics: a tutorial. Clin Pharmacol Ther. 2021;110(3):677‐687.  [DOI](https://doi.org/10.1002/cpt.2354) | [PMC free article](/articles/PMC8404400/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34231197/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=How%20to%20integrate%20CYP2D6%20Phenoconversion%20into%20clinical%20pharmacogenetics:%20a%20tutorial&author=EJ%20Cicali&author=AL%20Elchynski&author=KJ%20Cook&volume=110&issue=3&publication_year=2021&pages=677-687&pmid=34231197&doi=10.1002/cpt.2354&)

11. van der Lee M, Guchelaar HJ, Swen JJ. Substrate specificity of CYP2D6 genetic variants. Pharmacogenomics. 2021;22(16):1081‐1089.  [DOI](https://doi.org/10.2217/pgs-2021-0093) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34569808/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Substrate%20specificity%20of%20CYP2D6%20genetic%20variants&author=M%20van%20der%20Lee&author=HJ%20Guchelaar&author=JJ%20Swen&volume=22&issue=16&publication_year=2021&pages=1081-1089&pmid=34569808&doi=10.2217/pgs-2021-0093&)

12. Yuan R, Madani S, Wei XX, Reynolds K, Huang SM. Evaluation of cytochrome P450 probe substrates commonly used by the pharmaceutical industry to study in vitro drug interactions. Drug Metab Dispos. 2002;30(12):1311‐1319.  [DOI](https://doi.org/10.1124/dmd.30.12.1311) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12433797/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Evaluation%20of%20cytochrome%20P450%20probe%20substrates%20commonly%20used%20by%20the%20pharmaceutical%20industry%20to%20study%20in%C2%A0vitro%20drug%20interactions&author=R%20Yuan&author=S%20Madani&author=XX%20Wei&author=K%20Reynolds&author=SM%20Huang&volume=30&issue=12&publication_year=2002&pages=1311-1319&pmid=12433797&doi=10.1124/dmd.30.12.1311&)

13. Wang RW, Newton DJ, Liu N, Atkins WM, Lu AY. Human cytochrome P‐450 3A4: in vitro drug–drug interaction patterns are substrate‐dependent. Drug Metab Dispos. 2000;28(3):360‐366.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/10681383/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Human%20cytochrome%20P%E2%80%90450%203A4:%20in%C2%A0vitro%20drug%E2%80%93drug%20interaction%20patterns%20are%20substrate%E2%80%90dependent&author=RW%20Wang&author=DJ%20Newton&author=N%20Liu&author=WM%20Atkins&author=AY%20Lu&volume=28&issue=3&publication_year=2000&pages=360-366&pmid=10681383&)

14. Shen H, He MM, Liu H, et al. Comparative metabolic capabilities and inhibitory profiles of CYP2D6.1, CYP2D6.10, and CYP2D6.17. Drug Metab Dispos. 2007;35(8):1292‐1300.  [DOI](https://doi.org/10.1124/dmd.107.015354) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17470523/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Comparative%20metabolic%20capabilities%20and%20inhibitory%20profiles%20of%20CYP2D6.1,%20CYP2D6.10,%20and%20CYP2D6.17&author=H%20Shen&author=MM%20He&author=H%20Liu&volume=35&issue=8&publication_year=2007&pages=1292-1300&pmid=17470523&doi=10.1124/dmd.107.015354&)

15. Ramamoorthy Y, Yu AM, Suh N, Haining RL, Tyndale RF, Sellers EM. Reduced (+/−)‐3,4‐methylenedioxymethamphetamine (“Ecstasy”) metabolism with cytochrome P450 2D6 inhibitors and pharmacogenetic variants in vitro. Biochem Pharmacol. 2002;63(12):2111‐2119.  [DOI](https://doi.org/10.1016/s0006-2952(02)01028-6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12110370/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20Pharmacol&title=Reduced%20(+/%E2%88%92)%E2%80%903,4%E2%80%90methylenedioxymethamphetamine%20(%E2%80%9CEcstasy%E2%80%9D)%20metabolism%20with%20cytochrome%20P450%202D6%20inhibitors%20and%20pharmacogenetic%20variants%20in%C2%A0vitro&author=Y%20Ramamoorthy&author=AM%20Yu&author=N%20Suh&author=RL%20Haining&author=RF%20Tyndale&volume=63&issue=12&publication_year=2002&pages=2111-2119&pmid=12110370&doi=10.1016/s0006-2952(02)01028-6&)

16. Bapiro TE, Hasler JA, Ridderström M, Masimirembwa CM. The molecular and enzyme kinetic basis for the diminished activity of the cytochrome P450 2D6.17 (CYP2D6.17) variant. Potential implications for CYP2D6 phenotyping studies and the clinical use of CYP2D6 substrate drugs in some African populations. Biochem Pharmacol. 2002;64(9):1387‐1398.  [DOI](https://doi.org/10.1016/s0006-2952(02)01351-5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12392820/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20Pharmacol&title=The%20molecular%20and%20enzyme%20kinetic%20basis%20for%20the%20diminished%20activity%20of%20the%20cytochrome%20P450%202D6.17%20(CYP2D6.17)%20variant.%20Potential%20implications%20for%20CYP2D6%20phenotyping%20studies%20and%20the%20clinical%20use%20of%20CYP2D6%20substrate%20drugs%20in%20some%20African%20populations&author=TE%20Bapiro&author=JA%20Hasler&author=M%20Ridderstr%C3%B6m&author=CM%20Masimirembwa&volume=64&issue=9&publication_year=2002&pages=1387-1398&pmid=12392820&doi=10.1016/s0006-2952(02)01351-5&)

17. Bogni A, Monshouwer M, Moscone A, et al. Substrate specific metabolism by polymorphic cytochrome P450 2D6 alleles. Toxicol In Vitro. 2005;19(5):621‐629.  [DOI](https://doi.org/10.1016/j.tiv.2005.04.001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15893449/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Toxicol%20In%C2%A0Vitro&title=Substrate%20specific%20metabolism%20by%20polymorphic%20cytochrome%20P450%202D6%20alleles&author=A%20Bogni&author=M%20Monshouwer&author=A%20Moscone&volume=19&issue=5&publication_year=2005&pages=621-629&pmid=15893449&doi=10.1016/j.tiv.2005.04.001&)

18. c2d6‐2r004av2‐1.pdf. Accessed February 26, 2024. xenotech.com  [xenotech.com](http://xenotech.com)

19. Greenblatt DJ, Zhao Y, Venkatakrishnan K, et al. Mechanism of cytochrome P450‐3A inhibition by ketoconazole. J Pharm Pharmacol. 2011;63(2):214‐221.  [DOI](https://doi.org/10.1111/j.2042-7158.2010.01202.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21235585/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharm%20Pharmacol&title=Mechanism%20of%20cytochrome%20P450%E2%80%903A%20inhibition%20by%20ketoconazole&author=DJ%20Greenblatt&author=Y%20Zhao&author=K%20Venkatakrishnan&volume=63&issue=2&publication_year=2011&pages=214-221&pmid=21235585&doi=10.1111/j.2042-7158.2010.01202.x&)

20. Cai J, Dai DP, Geng PW, et al. Effects of 22 novel CYP2D6 variants found in the Chinese population on the Bufuralol and dextromethorphan metabolisms in vitro. Basic Clin Pharmacol Toxicol. 2016;118(3):190‐199.  [DOI](https://doi.org/10.1111/bcpt.12478) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26310775/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Basic%20Clin%20Pharmacol%20Toxicol&title=Effects%20of%2022%20novel%20CYP2D6%20variants%20found%20in%20the%20Chinese%20population%20on%20the%20Bufuralol%20and%20dextromethorphan%20metabolisms%20in%C2%A0vitro&author=J%20Cai&author=DP%20Dai&author=PW%20Geng&volume=118&issue=3&publication_year=2016&pages=190-199&pmid=26310775&doi=10.1111/bcpt.12478&)

21. Marcucci KA, Pearce RE, Crespi C, Steimel DT, Leeder JS, Gaedigk A. Characterization of cytochrome P450 2D6.1 (CYP2D6.1), CYP2D6.2, and CYP2D6.17 activities toward model CYP2D6 substrates dextromethorphan, bufuralol, and debrisoquine. Drug Metab Dispos. 2002;30(5):595‐601.  [DOI](https://doi.org/10.1124/dmd.30.5.595) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11950793/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Characterization%20of%20cytochrome%20P450%202D6.1%20(CYP2D6.1),%20CYP2D6.2,%20and%20CYP2D6.17%20activities%20toward%20model%20CYP2D6%20substrates%20dextromethorphan,%20bufuralol,%20and%20debrisoquine&author=KA%20Marcucci&author=RE%20Pearce&author=C%20Crespi&author=DT%20Steimel&author=JS%20Leeder&volume=30&issue=5&publication_year=2002&pages=595-601&pmid=11950793&doi=10.1124/dmd.30.5.595&)

22. Wennerholm A, Johansson I, Hidestrand M, Bertilsson L, Gustafsson LL, Ingelman‐Sundberg M. Characterization of the CYP2D6*29 allele commonly present in a black Tanzanian population causing reduced catalytic activity. Pharmacogenetics. 2001;11(5):417‐427.  [DOI](https://doi.org/10.1097/00008571-200107000-00005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11470994/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Characterization%20of%20the%20CYP2D6*29%20allele%20commonly%20present%20in%20a%20black%20Tanzanian%20population%20causing%20reduced%20catalytic%20activity&author=A%20Wennerholm&author=I%20Johansson&author=M%20Hidestrand&author=L%20Bertilsson&author=LL%20Gustafsson&volume=11&issue=5&publication_year=2001&pages=417-427&pmid=11470994&doi=10.1097/00008571-200107000-00005&)

23. Muroi Y, Saito T, Takahashi M, et al. Functional characterization of wild‐type and 49 CYP2D6 allelic variants for N‐desmethyltamoxifen 4‐hydroxylation activity. Drug Metab Pharmacokinet. 2014;29(5):360‐366.  [DOI](https://doi.org/10.2133/dmpk.dmpk-14-rg-014) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24647041/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Pharmacokinet&title=Functional%20characterization%20of%20wild%E2%80%90type%20and%2049%20CYP2D6%20allelic%20variants%20for%20N%E2%80%90desmethyltamoxifen%204%E2%80%90hydroxylation%20activity&author=Y%20Muroi&author=T%20Saito&author=M%20Takahashi&volume=29&issue=5&publication_year=2014&pages=360-366&pmid=24647041&doi=10.2133/dmpk.dmpk-14-rg-014&)

24. Yu A, Kneller BM, Rettie AE, Haining RL. Expression, purification, biochemical characterization, and comparative function of human cytochrome P450 2D6.1, 2D6.2, 2D6.10, and 2D6.17 allelic isoforms. J Pharmacol Exp Ther. 2002;303(3):1291‐1300.  [DOI](https://doi.org/10.1124/jpet.102.039891) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12438554/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacol%20Exp%20Ther&title=Expression,%20purification,%20biochemical%20characterization,%20and%20comparative%20function%20of%20human%20cytochrome%20P450%202D6.1,%202D6.2,%202D6.10,%20and%202D6.17%20allelic%20isoforms&author=A%20Yu&author=BM%20Kneller&author=AE%20Rettie&author=RL%20Haining&volume=303&issue=3&publication_year=2002&pages=1291-1300&pmid=12438554&doi=10.1124/jpet.102.039891&)

25. Dong AN, Ahemad N, Pan Y, Palanisamy UD, Yiap BC, Ong CE. Functional and structural characterisation of common cytochrome P450 2D6 allelic variants‐roles of Pro34 and Thr107 in catalysis and inhibition. Naunyn Schmiedeberg's Arch Pharmacol. 2019;392(8):1015‐1029.  [DOI](https://doi.org/10.1007/s00210-019-01651-0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31025144/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Naunyn%20Schmiedeberg's%20Arch%20Pharmacol&title=Functional%20and%20structural%20characterisation%20of%20common%20cytochrome%20P450%202D6%20allelic%20variants%E2%80%90roles%20of%20Pro34%20and%20Thr107%20in%20catalysis%20and%20inhibition&author=AN%20Dong&author=N%20Ahemad&author=Y%20Pan&author=UD%20Palanisamy&author=BC%20Yiap&volume=392&issue=8&publication_year=2019&pages=1015-1029&pmid=31025144&doi=10.1007/s00210-019-01651-0&)

26. Marcath LA, Pasternak AL, Hertz DL. Challenges to assess substrate‐dependent allelic effects in CYP450 enzymes and the potential clinical implications. Pharmacogenomics J. 2019;19(6):501‐515.  [DOI](https://doi.org/10.1038/s41397-019-0105-1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31616046/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Challenges%20to%20assess%20substrate%E2%80%90dependent%20allelic%20effects%20in%20CYP450%20enzymes%20and%20the%20potential%20clinical%20implications&author=LA%20Marcath&author=AL%20Pasternak&author=DL%20Hertz&volume=19&issue=6&publication_year=2019&pages=501-515&pmid=31616046&doi=10.1038/s41397-019-0105-1&)

27. Haupt LJ, Kazmi F, Ogilvie BW, et al. The reliability of estimating Ki values for direct, reversible inhibition of cytochrome P450 enzymes from corresponding IC50 values: a retrospective analysis of 343 experiments. Drug Metab Dispos. 2015;43(11):1744‐1750.  [DOI](https://doi.org/10.1124/dmd.115.066597) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26354951/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=The%20reliability%20of%20estimating%20Ki%20values%20for%20direct,%20reversible%20inhibition%20of%20cytochrome%20P450%20enzymes%20from%20corresponding%20IC50%20values:%20a%20retrospective%20analysis%20of%20343%20experiments&author=LJ%20Haupt&author=F%20Kazmi&author=BW%20Ogilvie&volume=43&issue=11&publication_year=2015&pages=1744-1750&pmid=26354951&doi=10.1124/dmd.115.066597&)

28. Akiyoshi T, Ishiuchi M, Imaoka A, Ohtani H. Variation in the inhibitory potency of terbinafine among genetic variants of CYP2D6. Drug Metab Pharmacokinet. 2015;30(4):321‐324.  [DOI](https://doi.org/10.1016/j.dmpk.2015.04.003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26195224/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Pharmacokinet&title=Variation%20in%20the%20inhibitory%20potency%20of%20terbinafine%20among%20genetic%20variants%20of%20CYP2D6&author=T%20Akiyoshi&author=M%20Ishiuchi&author=A%20Imaoka&author=H%20Ohtani&volume=30&issue=4&publication_year=2015&pages=321-324&pmid=26195224&doi=10.1016/j.dmpk.2015.04.003&)

29. McLaughlin LA, Paine MJ, Kemp CA, et al. Why is quinidine an inhibitor of cytochrome P450 2D6? The role of key active‐site residues in quinidine binding. J Biol Chem. 2005;280(46):38617‐38624.  [DOI](https://doi.org/10.1074/jbc.M505974200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16162505/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Why%20is%20quinidine%20an%20inhibitor%20of%20cytochrome%20P450%202D6?%20The%20role%20of%20key%20active%E2%80%90site%20residues%20in%20quinidine%20binding&author=LA%20McLaughlin&author=MJ%20Paine&author=CA%20Kemp&volume=280&issue=46&publication_year=2005&pages=38617-38624&pmid=16162505&doi=10.1074/jbc.M505974200&)

30. Sager JE, Tripathy S, Price LSL, et al. Corrigendum to “in vitro to in vivo extrapolation of the complex drug–drug interaction of bupropion and its metabolites with CYP2D6; simultaneous reversible inhibition and CYP2D6 downregulation”. Biochem Pharmacol. 2021;183:114306.  [DOI](https://doi.org/10.1016/j.bcp.2020.114306) | [PMC free article](/articles/PMC7769866/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33161206/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20Pharmacol&title=Corrigendum%20to%20%E2%80%9Cin%20vitro%20to%20in%C2%A0vivo%20extrapolation%20of%20the%20complex%20drug%E2%80%93drug%20interaction%20of%20bupropion%20and%20its%20metabolites%20with%20CYP2D6;%20simultaneous%20reversible%20inhibition%20and%20CYP2D6%20downregulation%E2%80%9D&author=JE%20Sager&author=S%20Tripathy&author=LSL%20Price&volume=183&publication_year=2021&pages=114306&pmid=33161206&doi=10.1016/j.bcp.2020.114306&)
