# Discovery of ancestry-specific variants associated with clopidogrel response among Caribbean Hispanics

## Metadata
**Authors:** Guang Yang, Pablo González, Mariangeli Moneró, Kelvin Carrasquillo, Jessicca Y Renta, Dagmar F Hernandez-Suarez, Mariana R Botton, Kyle Melin, Stuart A Scott, Gualberto Ruaño, Abiel Roche-Lima, Cristina Alarcon, Marylyn D Ritchie, Minoli A Perera, Jorge Duconge
**Journal:** NPJ Genomic Medicine
**Date:** 2025 Mar 7
**DOI:** [10.1038/s41525-025-00479-3](https://doi.org/10.1038/s41525-025-00479-3)
**PMID:** 40055373
**PMCID:** PMC11889249
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11889249/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC11889249/pdf/41525_2025_Article_479.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC11889249/pdf/41525_2025_Article_479.pdf)

## Abstract

High on-treatment platelet reactivity (HTPR) with clopidogrel predicts ischemic events in adults with coronary artery disease, and while HTPR varies by ethnicity, no genome-wide association study (GWAS) of clopidogrel response has been conducted in Caribbean Hispanics. This study aimed to identify genetic predictors of HTPR in a cohort of 511 Puerto Rican cardiovascular patients treated with clopidogrel, stratified by P2Y12 reaction units (PRU) into responders and non-responders (HTPR). Local ancestry inference (LAI) and traditional GWAS identified variants in the CYP2C19 region associated with HTPR, primarily in individuals with European ancestry. Three variants (OSBPL10 rs1376606, DERL3 rs5030613, RGS6 rs9323567) showed suggestive significance, and a variant in UNC5C was linked to increased HTPR risk. These findings highlight the unique genetic landscape of Caribbean Hispanics and challenge the significance of CYP2C19*2 in predicting clopidogrel response in patients with high non-European ancestry. Further studies are needed to replicate these results in other diverse cohorts.

Subject terms: Genetic markers, Predictive markers

## Introduction

Clopidogrel is a P2Y12 receptor inhibitor that is widely recommended for secondary prevention of coronary artery disease (CAD) to reduce major adverse cardiovascular and cerebrovascular events (MACCEs). Despite its proven effectiveness as part of dual antiplatelet therapy (DAPT), some patients do not attain adequate antiplatelet effects^1,2^. Studies have shown that certain genetic variants in clinically relevant pharmacogenes (e.g., *CYP2C19**2) are significantly associated with a diminished clopidogrel response^2,3^. A meta-analysis of 9 clinical trials and 9,685 patients with ACS showed that carriers of one or two loss-of-function (LOF) *CYP2C19* alleles had 1.6-fold greater risk for MACCEs and 2.8-fold increased risk of stent thrombosis compared to non-carriers^4^. In a separate study, carrying at least one LOF *CYP2C19* allele resulted in a higher rate of MACCEs when compared to wild-types (i.e., 12.1% vs. 8.0%)^5^. However, *CYP2C19* accounts for only ~12% of the observed variability in clopidogrel response^1^. Further, we have preliminarily found that approximately 22% of poor responders to clopidogrel do not carry any known LOF *CYP2C19* allele, suggesting the presence of unknown genetic variants for clopidogrel response among Caribbean Hispanics. A GWAS conducted by the International Clopidogrel Pharmacogenomics Consortium (ICPC) confirmed the significance of *CYP2C19**2 in clopidogrel resistance^6^. However, most of these studies primarily involved individuals of European ancestry, with very limited representation from Hispanics and other minorities populations^7^.

Although routine implementation of *CYP2C19* testing to guide antiplatelet therapy selection is not a universal standard of care, the American College of Cardiology Foundation/American Heart Association (ACC/AHA) ACS guidelines note that genetic testing might be considered on a case-by-case basis to identify whether a high-risk patient is predisposed to inadequate platelet inhibition with clopidogrel^8^. The Clinical Pharmacogenetics Implementation Consortium (CPIC) recommends an alternative antiplatelet therapy (if not contraindicated) when *CYP2C19* genotype status is known for carriers of *CYP2C19* risk alleles, particularly for those with a poor metabolizer (PM) phenotype^9^. A *CYP2C19* genotype-guided strategy for optimal selection of antiplatelet therapy has been proven beneficial in several clinical studies; however, none of these trials were performed with Caribbean Hispanic patients^7,10–12^.

To date, a substantial knowledge gap exists in our understanding of clopidogrel pharmacogenomics among Caribbean Hispanics, who make up less than 1% of participants in previous studies. This lack of representation exacerbates existing healthcare disparities^13,14^. This study addresses this gap by conducting a novel GWAS of residual on-treatment platelet reactivity in Caribbean Hispanic patients to identify genetic determinants of clopidogrel responsiveness. Furthermore, it incorporates local ancestry information into the genetic analysis of clopidogrel response, adding a novel dimension to our understanding of this critical pharmacogenomic trait.

## Results

Figure [1](#Fig1) depicts the participants who underwent screening, were enrolled, and ultimately completed the study. We recruited and analyzed a cohort of 511 patients on maintenance dose of clopidogrel (75 mg/day). Table [1](#Tab1) describes the baseline characteristics of the participants from the study population with complete genetic profiles and clinical data. The average age of all participants in this study was ~68 years old, with 45% identified as female. Most were middle-aged with a high prevalence of conventional risk factors (i.e., high BMI (28.4 kg/m^2^), 83.8% hypertension, 71.9% dyslipidemias, and 54.8% diabetes). Furthermore, 20% of participants were on proton pump inhibitors (mainly pantoprazole), and statins and calcium channel blockers were prescribed in 79.1% and 26.8% of patients, respectively. Patients who were taking aspirin administered as part of DAPT represented 63.3% of the total cohort. Among 72.6% of participants, clopidogrel was given for stable CAD or ACS. About 18% of participants received a loading dose of clopidogrel (300 or 600 mg) on the scheduled date for PCI, while 78% were already receiving maintenance doses of clopidogrel 75 mg/daily by the time they underwent the intervention and, therefore, a loading dose was not given. Since no differences in outcomes or baseline characteristics existed among patients with different dose schemes, they were combined for further analyses. Cases tended to be women, whose proportion significantly exceeded that of controls (59.8% vs. 40.6%; *p* < 0.001). Cases also showed a higher proportion of patients diagnosed with diabetes (63% vs. 52%; *p* = 0.034), but less smokers (*p* = 0.025). The cases also had a higher BMI on average (*p* = 0.002).

### Fig. 1. Participant Enrollment and Data Selection Process.

![Fig. 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e49c/11889249/04ce333c9fd3/41525_2025_479_Fig1_HTML.jpg)

Flowchart to illustrate the number of cardiovascular (CV) participants who were enrolled in the study protocol from seven different medical facilities across the Commonwealth of Puerto Rico, between January 2018 and June 2020 and the total number of patients with full data considered for the final association analyses.

### Table 1.

| Variables | All Patients (n = 511) |  |  |  |  |  | Controls (n = 399) |  |  |  |  |  | Cases (n = 112) |  |  |  |  |  | p-value d |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| mean | SD | SEM | Min | Max | median | mean | SD | SEM | Min | Max | median | mean | SD | SEM | Min | Max | median |  |  |
| Age (years) | 68.01 | 10.95 | 0.51 | 27.00 | 94.00 | 66.00 | 67.72 | 11.1 | 0.59 | 27.0 | 94.00 | 66.00 | 69.02 | 10.44 | 1.03 | 37.00 | 89.00 | 79.00 | 0.322 |
| BMI (kg/m2) | 28.40 | 5.71 | 0.27 | 11.48 | 52.67 | 27.46 | 27.89 | 5.3 | 0.28 | 11.48 | 47.02 | 25.82 | 30.15 | 6.73 | 0.67 | 15.95 | 52.67 | 29.41 | 0.002 |
| AMR | 0.115 | 0.04 | 0.002 | 0.01 | 0.56 | 0.12 | 0.11 | 0.04 | 0.002 | 0.01 | 0.56 | 0.12 | 0.11 | 0.03 | 0.003 | 0.02 | 0.19 | 0.12 | 0.370 |
| IBS | 0.680 | 0.13 | 0.006 | 0.16 | 0.96 | 0.77 | 0.70 | 0.13 | 0.007 | 0.23 | 0.96 | 0.77 | 0.70 | 0.13 | 0.01 | 0.16 | 0.93 | 0.77 | 0.821 |
| YRI | 0.205 | 0.14 | 0.006 | 0.004 | 0.83 | 0.11 | 0.20 | 0.14 | 0.007 | 0.004 | 0.72 | 0.34 | 0.19 | 0.14 | 0.01 | 0.03 | 0.83 | 0.11 | 0.606 |
|   | N |  |  | % |  |  | N |  |  | % |  |  | N |  |  | % |  |  |   |
| Type 2 Diabetes Mellitus | 280 |  |  | 54.78 |  |  | 209 |  |  | 52.38 |  |  | 71 |  |  | 63.39 |  |  | 0.034 |
| Hypertension | 428 |  |  | 83.75 |  |  | 330 |  |  | 82.71 |  |  | 98 |  |  | 87.50 |  |  | 0.190 |
| Dyslipidemias | 367 |  |  | 71.82 |  |  | 286 |  |  | 71.68 |  |  | 81 |  |  | 72.32 |  |  | 0.894 |
| Smoking | 69 |  |  | 13.50 |  |  | 60 |  |  | 15.04 |  |  | 9 |  |  | 8.03 |  |  | 0.025 |
| MACCEsa | 42 |  |  | 8.22 |  |  | 32 |  |  | 8.02 |  |  | 10 |  |  | 8.93 |  |  | 0.763 |
| Non-fatal MIa | 19 |  |  | 3.72 |  |  | 16 |  |  | 4.01 |  |  | 3 |  |  | 2.68 |  |  | 0.464 |
| Stent Thrombosisa | 15 |  |  | 2.93 |  |  | 12 |  |  | 3.01 |  |  | 3 |  |  | 2.68 |  |  | 0.850 |
| Ischemic Strokes /TIAa | 14 |  |  | 2.74 |  |  | 11 |  |  | 2.76 |  |  | 2 |  |  | 1.78 |  |  | 0.512 |
| CV Deathsa | 4 |  |  | 0.78 |  |  | 3 |  |  | 0.75 |  |  | 1 |  |  | 0.89 |  |  | 0.887 |
| Bleedings Eventsb | 83 |  |  | 16.24 |  |  | 67 |  |  | 16.79 |  |  | 16 |  |  | 14.29 |  |  | 0.518 |
| Aspirin Use | 323 |  |  | 63.21 |  |  | 251 |  |  | 62.91 |  |  | 72 |  |  | 64.29 |  |  | 0.830 |
| Statins Use | 404 |  |  | 79.06 |  |  | 323 |  |  | 80.95 |  |  | 81 |  |  | 72.32 |  |  | 0.160 |
| CCB | 156 |  |  | 30.53 |  |  | 122 |  |  | 30.58 |  |  | 34 |  |  | 30.36 |  |  | 0.751 |
| PPI | 123 |  |  | 24.07 |  |  | 82 |  |  | 20.55 |  |  | 41 |  |  | 36.61 |  |  | 0.913 |
| LVEF ≤ 40% | 42 |  |  | 8.22 |  |  | 33 |  |  | 8.27 |  |  | 9 |  |  | 8.04 |  |  | 0.991 |
| ACS & Stable CAD | 371 |  |  | 72.60 |  |  | 295 |  |  | 73.93 |  |  | 76 |  |  | 67.86 |  |  | 0.120 |
| Coronary Artery Stentingc | 191 |  |  | 37.38 |  |  | 159 |  |  | 39.85 |  |  | 32 |  |  | 28.57 |  |  | 0.495 |
| PAD | 123 |  |  | 24.07 |  |  | 94 |  |  | 23.56 |  |  | 29 |  |  | 25.89 |  |  | 0.378 |
| Sex (Males) | 282 |  |  | 55.18 |  |  | 237 |  |  | 59.40 |  |  | 45 |  |  | 40.18 |  |  | < 0.001 |

Table 1 Caption: Baseline characteristics of 511 Caribbean Hispanic participants in the study cohort

Figure [2](#Fig2) shows the overall and relative distribution of ancestral proportions for the three major ancestral populations (i.e., Native American, European, and West-African) to the trihybrid, admixed genomic makeup of individuals in the study cohort. Four representative LAI plots (RFMix karyograms) from individuals on the extremes and the middle of the Fig. [2C](#Fig2) (PCA) can be found as a supplemental material (Supplementary Fig. [1](#MOESM1)). Recently admixed populations, like Caribbean Hispanics, have a genome that is reminiscent of a mosaic, with sections inherited from Africa and others inherited from Europe or native/indigenous populations. LA is an important consideration in drug response, particularly in admixed population, at the gene level. Therefore, we performed both traditional GWAS, which was adjusted for genomic PCs, and LAI GWAS, incorporating LA into our association analyses. Supplementary Fig. [2](#MOESM1) shows the results of the association tests between PCs and either HTPR or PRU and illustrates the reason for choosing only 2 PCs in further analyses.

### Fig. 2. Genetic admixture in Caribbean Hispanics.

![Fig. 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e49c/11889249/0dd8e5ad70ca/41525_2025_479_Fig2_HTML.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=11889249_41525_2025_479_Fig2_HTML.jpg)

A Density plots of the study cohort with green representing Native American (AMR: 103 Native Mesoamerican and South American individuals from the Human Genome Diversity Project/ Centre d’Etude du Polymorphism database (HGDP-CEPH); representing Maya and Pima in Mexico, Colombian in Colombia, Karitiana and Surui in Brazil), purple representing African (YRI: 61 Yoruba in Ibadan, Nigeria from 1KGP), and cyan representing European (IBS: 107 Iberian populations in Spain from 1KGP) ancestries, respectively42,43. There are different proportions of African and European contributions to the overall genetic background of each participant. Native American curve suggests this ancestral component is less variable among participants, but also smaller in terms of global proportion. B The individual proportions of the three parental components in the study cohort (ancestral fractions). Each column represents an individual in the study cohort. C Principal component analysis (PCA) in each participant of the study (PR: Caribbean Hispanics from Puerto Rico, in red color) along with the European (IBS), African (YRI), and Native American (AMR) reference populations. D Bar chart of the average ancestral proportions across the cohort. When merged with non-overlapping SNPs typed previously in all individuals from our study cohort (PR), the resulting data for admixture analysis at both the population and regional (continental) level consist of over 30 million SNPs.

We conducted both linear and logistic regression analyses to identify genetic association to PRU and HTPR, respectively. Results from the traditional SNP-based GWAS analyses of PRU and HTPR, adjusted by covariates and PCs, are shown in Fig. [3](#Fig3) as both Manhattan and Q-Q plots. The time since PCI was not included as a covariate in the multivariable regression model. Top association hits from these GWAS with the beta (β) coefficients for the linear (PRU) or Odds Ratios (ORs) for the logistic regressions (HTPR), as well as their corresponding *p*-values and 95% confidence intervals (CI), are presented in Table [2](#Tab2). Notably, after functional annotation of lead SNPs and SNPs in LD using ANNOVAR, an overwhelming majority of the lead SNPs (~99%) were situated within intronic, intergenic, or long non-coding RNA (lncRNA)/regulatory regions. Conversely, less than 1% of the variants correspond to exonic variants occurring within coding regions. Although the multivariable linear regression GWAS did not yield any loci with genome-wide significant signals, three loci reached the suggestive significance level of 10^−6^ (Fig. [3](#Fig3)). In Fig. [3](#Fig3), we observed noteworthy associations to PRU levels on chromosomes 3, 14, and 22. Specifically, the variant rs1376606G located in an intergenic region near the *OSBPL10* (Oxysterol Binding Protein Like 10 gene) on chromosome 3p23, exhibited a statistically significant relationship with the PRU value. The association suggests that as the number of G alleles a patient carries at this locus increases, their PRU value decreases, which is indicated by the negative beta coefficient (β: −31.38, 95% CI: −43.66 to −19.09, *p* = 7.75·10^−7^). This SNP also resides within a distal enhancer element and serves as an expression quantitative trait locus (eQTL) for *OSBPL10* in multiple GTEx tissues. In the intronic region of *DERL3* (degradation in endoplasmic reticulum protein 3) on chromosome 22q11.23, the rs5030613G>A SNP was associated with a decreased PRU value (β: −33.53, 95% CI: −46.76 to −20.3, *p* = 9.42·10^−7^). On chromosome 14q24.2, rs9323567C>T SNP within the intronic region of *RGS6* (Regulator of G Protein Signaling 6 gene), was also associated with a decreased PRU value with β of −26.71 (95%CI: −37.27 to −16.15, *p* = 9.85·10^−7^). In the multivariable logistic regression GWAS of the HTPR phenotype, we identified rs116022080G>A on chromosome 4q22.3 to be associated with an increased risk of HTPR (OR: 3.689, 95%CI: 2.191 to 6.209, *p* = 9.02·10^−7^). This SNP is an intronic variant at the *UNC5C* gene (Unc-5 Netrin Receptor C).

### Fig. 3. Manhattan and QQ plot from traditional GWAS.

![Fig. 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e49c/11889249/b9dbd67edc6a/41525_2025_479_Fig3_HTML.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=11889249_41525_2025_479_Fig3_HTML.jpg)

Manhattan plot representing the association between single nucleotide polymorphisms (SNPs) and Platelet Reactivity Units (PRU) or High on Treatment Platelet Reactivity (HTPR). The x-axis shows the position of each SNP on the chromosome, while the y-axis displays the association (−log10 p-value) between the SNP and the trait of interest. The blue line represents the suggestive level of significance (p = 1·10−6). Threshold for genome-wide significance (p = 5·10−8) not shown. The QQ plot displays the observed p-values of the genome-wide association study (GWAS) summary data on the y-axis, compared to the expected p-values under the null hypothesis of no association on the x-axis. Deviations from the diagonal line indicate departures from the null hypothesis, where points above the line suggest more significant associations than expected, while points below the line suggest fewer significant associations than expected. A The GWAS of PRU using linear regression identified 3 loci (ZNF860 on chromosome 3, DERL3 on chromosome 22, and RGS6 on chromosome 14) to be associated with the changes of PRU value at the suggestive level of significance (p < 1·10−6). B QQ plot for linear regression model GWAS. The genomic control value for the GWAS was λ = 1.01, showing no inflation. C The GWAS of HTPR using logistic regression showed that one locus on chromosome 4 associated with increased risk of HTPR at the suggestive level of significance (p < 1·10−6). D QQ plot for logistic regression model GWAS. The genomic control value for the GWAS was λ = 1.03, showing no inflation.

### Table 2.

| Linear regression model GWAS |  |  |  |  |  |  |  |  |  |  |   |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| SNPa | Chr | Position | Allele1 | Allele2 | MAF | Beta | SE | 95% CIs | p-value | Gene | p-value/rep |
| rs1376606 | 3 | 32010168 | G | C | 0.13 | −31.38 | 6.27 | −43.66, −19.09 | 7.75E-07 | OSBPL10 | 0.68 |
| rs5030613 | 22 | 23834100 | A | G | 0.11 | −33.53 | 6.75 | −46.76, −20.30 | 9.42E-07 | DERL3 | NA |
| rs9323567 | 14 | 72268639 | T | C | 0.19 | −26.71 | 5.39 | −37.27, −16.15 | 9.85E-07 | RGS6 | 0.41 |
| rs4244285 | 10 | 94781859 | A | G | 0.13 | 23.94 | 6.15 | 11.89, 35.99 | 0.0001125 | CYP2C19 | 1.67E-33 |

Table 2 Caption: Top SNPs identified by traditional GWAS of platelet reactivity response in 511 Caribbean Hispanics on clopidogrel

Of note, the previously validated *CYP2C19*2* (rs4244285) variant was not significant in this study cohort, with *p* values of 1.5·10^−5^ for HTPR and 1.1·10^−4^ for PRU (Table [2](#Tab2)). This coding variant had previously been strongly associated with a decreased PRU (*p* = 1.67·10^−33^) in a large GWAS conducted by the ICPC among individuals of European ancestry^6^.

Results of the LAI adjusted GWAS, conducted to increase the power of detecting association in clopidogrel-treated patients from this admixed cohort and to minimize the impact of local ancestry bias, are depicted in Fig. [4](#Fig4). While no variant was genome-wide significant in the ancestry-specific analysis (Supplementary Fig. [3](#MOESM1)), the LAI GWAS meta-analysis identified a near significant association of the intronic SNP rs12571421 (c.819+228 A > G), located in the *CYP2C19* gene on chromosome 10, with a higher risk of HTPR (OR: 2.06, 95% CI: 1.54 to 2.74, *p* = 8.37·10^−7^). Moreover, we found that *CYP2C19*2* (rs4244285) was associated with a higher risk of HTPR (OR: 2.01, 95% CI: 1.51 to 2.31, *p* = 1.67·10^−6^) in our patients (Fig. [4](#Fig4)).

### Fig. 4. Manhattan plot of LAI adjusted GWAS.

![Fig. 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e49c/11889249/02a152810ca0/41525_2025_479_Fig4_HTML.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=11889249_41525_2025_479_Fig4_HTML.jpg)

After tractor analysis, we used METAL to perform a meta-analysis across all the ancestry to obtain associating results that were adjusted for LAI (A) No variant reached the suggestive significance level in linear regression model local ancestry inference GWAS. B The logistic regression model local ancestry inference GWAS showed one locus on chromosome 10 associated with increased risk of HTPR at the suggestive level of significance (p < 1·10-6).

Notably, significant effects were observed in individuals of European (*p* = 0.00088) and African (*p* = 0.00076) ancestry, but not in those of Native American ancestry (*p* = 0.26) (Table [3](#Tab3)). Furthermore, the *CYP2C19*2* (rs4244285) is in strong linkage disequilibrium (LD) with the top SNP hit of this LAI GWAS (*r*^2^ = 0.96, D’ = 1.0 among Hispanics) (Fig. [5](#Fig5)). The two SNPs are in an LD block of 6 kb, but the intronic SNP was removed from the *CYP2C19*2.003* allele definition by PharmVar due to unknown functionality^15^. Our findings suggest that *CYP2C19* SNPs may have distinct effect sizes on clopidogrel response based on local genetic ancestry. Importantly, this effect was only seen as suggestive of significance in the LAI adjusted GWAS, suggesting the need for LA adjustment in GWAS of admixed populations such as Caribbean Hispanics. No association was found in the LAI GWAS to PRU. Supplementary Fig. [4](#MOESM1) depicts QQ plot to compare the power of regular GWAS and LAI GWAS.

### Table 3.

| SNPa | Chr | Position | Allele1 | Allele2 | MAF (AFR, AMR, EUR) | OR | SE | Pval (AFR, AMR, EUR) | p-value/rep |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| rs12571421 | 10 | 96541982 | G | A | 0.13 (0.18, 0.09, 0.10) | 2.06 | 0.15 | 8.37E-07 (6.10E-04, 0.26, 5.20E-04) | 5.05E-33 |
| rs4244285 | 10 | 96541616 | A | G | 0.13 (0.19, 0.09, 0.11) | 2.01 | 0.15 | 1.67E-06 (7.58E-04, 0.26, 8.81E-04) | 1.67E-33 |

Table 3 Caption: Top SNPs in LAI adjusted GWAS case-control study of HTPR in 511 Caribbean Hispanics on clopidogrel

### Fig. 5. Region plot of top SNP from LAI logistic GWAS.

![Fig. 5](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e49c/11889249/7dadd4e66587/41525_2025_479_Fig5_HTML.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=11889249_41525_2025_479_Fig5_HTML.jpg)

The colors of the circles refer to linkage disequilibrium (LD) (r2) between top SNP rs12571421 (purple diamond) and nearby SNPs (based on pairwise r2 values from the 1000 Genomes Project reference panel). The blue line and right y-axis show the estimated recombination rate. The x-axis represents the genomic position in chromosome 10 and the left y-axis represents the −log10 p-value of association with HTPR in discovery cohort. Region plot showed CYP2C19*2 variant rs4244285 is high LD with top SNP rs12571421.

The two top signals on chromosome 10 found in our cohort of Caribbean Hispanics from Puerto Rico after performing the LAI-adjusted GWAS driven by the European ancestry/tract were replicated in a previous traditional GWAS of all-European individuals from the ICPC consortium (Table [3](#Tab3), rs4244285: *p*-value of 1.67·10^−33^; rs12571421: *p*-value of 5.05·10^−33^). However, none of the top hits detected in our traditional GWAS in Caribbean Hispanics following linear and logistic regressions were replicated in the ICPC cohort (Table [2](#Tab2)) or when tested in either the ACCOuNT consortium cohort of African Americans or the Brazilian cohort (Supplementary Fig. [5](#MOESM1)). This may be due to differences in local and global ancestry within the replication cohorts or spurious associations. Additional studies with matched replication cohorts are needed.

## Discussion

We conducted the first GWAS of clopidogrel response in a diverse and previously understudied Caribbean Hispanic population. Although our patient population was smaller than that of the ICPC, they displayed a greater degree of genetic admixture and heterogeneity compared to those in the ICPC study (i.e., exclusively Whites of self-reported European ancestry)^6^. Unlike the Native American component, the density plots of ancestral proportions for African and European heritages are broad and shallow and can be interpreted as confirmatory evidence of a diverse genomic architecture and heterogeneity within Caribbean Hispanics (Fig. [2A](#Fig2)). Genetically inferred ancestries show a varying degree of admixture proportions (i.e., continuum clines as in Fig. [2B](#Fig2)) that also scatter in the PCA plot (Fig. [2C](#Fig2)). Such unique combinations of individual proportions of Native American, West-African, and European ancestry give rise to rich repertoires of allelic combinations and haplotype blocks with very distinctive LD patterns that ultimately mask true associations in GWAS, becoming an important source of systematic bias. To account for any confounder effect from the variable degree of ancestry on the expected pharmacogenomic associations of clopidogrel resistance in Caribbean Hispanics, we adjusted by PC as a covariate in the regression analyses of regular GWAS and used LAI measures in the Tractor local ancestry GWAS.

Our study is unique in that we used a combination of traditional and LAI adjusted GWAS approaches to identify both novel population-specific genetic regions associated with clopidogrel response and confirm GWAS hits among Caribbean Hispanics from previous studies in Europeans. A recent study provided new insights into the role of local ancestry on clopidogrel response in African Americans from the ACCOuNT consortium^16^. In our LAI adjusted GWAS meta-analysis, the intronic variant rs12571421 in *CYP2C19*, which is in high LD with *CYP2C19**2, was identified as the top signal (OR: 2.06, *p* = 8.37·10^−7^). Previous in silico analyses predicted that this intronic SNP is a potential splicing alteration due to activation of a cryptic donor site within *CYP2C19*^17^. Additional functional studies are needed to determine if the identified association is driven solely by *CYP2C19*2* or a combination of functional genetic effects.

Admixed genomes pose methodological challenges for GWASs in populations like Caribbean Hispanics, including the need to adjust for population stratification. A recent publication assessed the significant impact of cross-ancestry genetic architecture and the resulting allelic heterogeneity that is given by differences in estimated effect sizes for risk variants across distinct ancestral backgrounds, on GWAS statistics in admixed populations^18^. Authors found that controlling for and conditioning effect sizes on LAI will significantly reduce statistical power^18^. However, our LAI GWAS in Caribbean Hispanics improved power to detect top signals (Supplementary Fig. [4](#MOESM1)) and revealed distinct effect sizes and *p*-values of relevant *CYP2C19* SNPs on clopidogrel response based on local genetic ancestry. Notably, these signals on chromosome 10 were replicated in the ICPC cohort (Table [3](#Tab3)). More important, the effect of rs12571421 was strongest in the European tract and weakest in the Native American ancestry suggesting that this allele may be less predictive among individuals from the Caribbean Hispanic population with greater Native American ancestry.

Compared with the traditional GWAS, the QQ plot of LAI GWAS showed the improved power to detect the top signal (Supplementary Fig. [4](#MOESM1)). These findings emphasize the need to account for LAI when conducting a GWAS in highly diverse, admixed populations to control for population stratification and resulting genetic heterogeneity because some significant loci seem to be driven by a particular ancestry group. This is critical and highlights a lack of transferability and poor portability of GWAS findings across diverse populations, particularly in those underrepresented groups with mixed genetic ancestries.

Although no GWAS signals were detected using traditional GWAS methods for PRU and HTPR, a total of 51 loci in 15 different genomic regions including chromosome 10 showed suggestive evidence of association with PRU and HTPR in our cohort of admixed Caribbean Hispanics (*p* ≤ 5·10^−6^). Despite only showing suggestive evidence of genome-wide association with clopidogrel antiplatelet effects, the predictability of PRU variability at 4 h was substantially improved by novel variants detected in chromosomes 9, 18, and 21 of 115 Chinese patients with coronary heart disease (CHD)^19^. In previous GWAS primarily involving individuals of European ancestry, the *CYP2C19* locus on chromosome 10 has consistently demonstrated to be the strongest genetic determinant of the diminished platelet response to clopidogrel, its active metabolite formation, and poorer cardiovascular outcomes^1,2,6,20^.

In this study, we were able to identify loci nominally associated with platelet reactivity in Caribbean Hispanics. The effect sizes of these novel loci could likely be a direct consequence of the relatively large percentage of heritability explained by them in this population and the well-defined biological nature of the PRU phenotype. In fact, Shuldiner et al have previously established that platelet response to clopidogrel is highly heritable (i.e., h(2) = 0.73; *p* < 0.001)^1^. GWAS are generally aimed at finding very small effect sizes and not very low frequency variants, except when such rare SNPs have relatively large effect sizes on the outcome of interest. Accordingly, there are some exceptions to the usually large number of samples needed to confirm small differences with statistical confidence^21^.

The *OSBPL10* gene encodes a member of the OSBP family, a group of intracellular lipid receptors that acts as sterol sensors^22^. *OSBPL10* has been previously reported as a novel candidate gene for high triglyceride trait in dyslipidemic Europeans as well as a regulator of cellular lipid metabolism and apolipoprotein B secretion^23^. A GWAS in Japanese found that *OSBPL10* SNPs are associated with susceptibility to peripheral arterial disease (PAD)^24^. Dyslipidemia has been largely associated with increased risk of premature CHD and is considered a modifiable factor affecting HTPR in clopidogrel-treated patients^25,26^. The intergenic variant rs1376606 harbored within a distal enhancer element on chromosome 3 is an eQTL for *OSBPL10* in several GTEx tissues, which may explain the observed association with lower PRUs (*p* = 7.75·10^−7^) in this study.

The intronic rs5030613 variant in *DERL3* was also related to decreased PRU values in our cohort (*p* = 9.42·10^−7^). The *DERL3* gene encodes a member of the Derlin family of proteins, Derlin-3, which plays a role in ATF6-induced endoplasmic reticulum-associated degradation (ERAD) of unfolded and misfolded glycoproteins to maintain homeostasis under stress^27^. Derlin-3 was found to be up-regulated in the heart following myocardial infarction (MI), showing cardioprotective effects^28^. Likewise, an intronic variant within the *RGS6* locus was associated with reduced PRU in our study (rs9323567 SNP; *p* = 9.85·10^−7^). *RGS6* expression is increased in the myocardium of patients with heart disease, and a regulatory role of RGS6 in cardiomyocytes during the aggravation of pathological cardiac hypertrophy has been reported^29^. *RGS6* encodes a member of the R7 subfamily of RGS, which is robustly expressed in the heart and is reportedly a key regulator of ischemic injury^30^. In contrast, the intronic rs116022080 variant in *UNC5C* was associated with an increased HTPR risk in the logistic regression (OR: 3.689, *p* = 9.02·10^−7^). This encoded protein belongs to the UNC-5 family of netrin receptors. At least one *UNC5C* variant was listed among those earlier identified genome-wide significant CAD risk loci, though its role in CAD pathophysiology is still uncertain^31,32^.

Since all the above-mentioned genetic loci have also been found to be associated with high cardiovascular risk, a dual role of these hits as genetic markers of predictive value for both cardiovascular disease susceptibility and clopidogrel resistance cannot be ruled out. However, further studies are warranted to confirm this potential interaction. Despite no individual variant being associated with platelet reactivity at GWAS significance in our clopidogrel-treated patients, an admixture-adjusted polygenic risk score-driven model using top hits from this study and LAI as weighting factors is expected to help improve predictability of clopidogrel response among Caribbean Hispanics. We firmly believe this approach will ultimately facilitate DAPT optimization in this underrepresented population.

The modest sample size of the discovery cohort was a limitation of this study. There is an urgent need to increase the representation of diverse populations in pharmacogenomic studies. The European-centric bias of earlier genetic studies, if not properly mitigated, will limit our understanding of underlying determinants of poor clopidogrel response among Caribbean Hispanics. The current paucity of data from non-European individuals will preclude any further extrapolation of derived prediction models to the population at large. Consequently, findings from this work are expected to help in part by addressing such an unmet need. This is of remarkable importance given the well-known differences in haplotypes and allele frequency distributions, effect sizes, varying admixture patterns, and unique genomic architectures across diverse ethnicities of distinct ancestries.

*CYP2C19**2 is well-established as a functional variant with reproducible effects on platelet response to clopidogrel among Europeans^1,2,6^. However, the lack of association observed in the AMR tract may be attributed to a combination of factors, including potential modifier SNPs that could diminish the effects of *CYP2C19**2 or the reduced statistical power in this tract due to the smaller sample size. In populations with varying degrees of ancestry from different genetic backgrounds, LD patterns can differ significantly, which may lead to the loss of signal. Moreover, allelic heterogeneity and differing allele frequencies, particularly when SNPs exhibit higher frequency differentiation by ancestry, could further influence the ability to detect associations. These factors highlight the complexity of genetic associations in admixed populations and underscore the challenges of translating findings across diverse genetic backgrounds. Further studies examining ancestry-specific effects in populations with varying genetic ancestries are warranted to fully elucidate the underlying cause.

All LAI GWAS associations should be considered as providing supplementary insights into phenotype associations, complementing rather than replacing findings from traditional GWAS. Potential biases associated with the LAI method include inaccuracies in ancestry assignment, which can arise from the reference populations used and a variable phase quality. Since existing reference panels do not thoroughly represent the broad genetic diversity of our study population, this may lead to misclassification of ancestry and consequently affect the interpretation of associations^33,34^. Additionally, the resolution of the LAI can vary depending on the extent of admixture in the population and time since admixture. Given that Caribbean Hispanics exhibit a recent and complex multi-way admixture pattern, the estimation of local ancestry may be less reliable, potentially introducing biases into our findings. There is also the possibility of allelic imbalance, where differences in allele frequencies between ancestral groups can skew results. Hou et al. suggested that the loss of power observed in LAI-adjusted GWAS might arise from biases in local ancestry corrections, especially when causal variants exhibit differing allelic effects across ancestries. They also noted that the power of LAI-based methods is diminished when there is insufficient heterogeneity in allelic effects or when imperfect tagging occurs due to linkage disequilibrium^34^. To mitigate these biases, we emphasize the importance of careful interpretation of our findings and recommend further validation in larger, more diverse cohorts.

In this work, we present groundbreaking findings that shed light on the effect of locus-specific ancestry in the pharmacogenomics of clopidogrel. Interestingly, our data suggest that the more native ancestry a Caribbean Hispanic patient has on their genetic background, the less associated the *CYP2C19**2 allele is with platelet reactivity. Since Caribbean Hispanics were not included in the original studies by the ICPC team, these discoveries hold significant implications for the field. Furthermore, we strongly believe this may have implications for other admixed populations, but unfortunately, such data does not currently exist. Consequently, the findings highlighted in this work are poised to address the existing paucity of relevant data concerning clopidogrel pharmacogenomics in non-European populations. This, in turn, could facilitate a more comprehensive understanding and implementation of CPIC-based actionable recommendations in minority and underrepresented populations, thus potentially improving healthcare outcomes for a broader range of individuals, and facilitating more inclusive and equitable practices. In light of the existing CPIC guideline for clopidogrel, which predominantly relies on data from ‘European-centric’ studies (i.e., particularly the GWAS of the ICPC cohort), we believe there is a need for cautious consideration. It is essential to recognize the potential limitations and biases inherent in such studies when applying CPIC guidelines to non-European ACS/PCI patients. Therefore, we suggest a revisited approach to antiplatelet therapy guidance for these populations, mindful of their genomic diversity.

Important to bear in mind that the published GWAS of clopidogrel conducted within the all-European ICPC cohort has not been replicated in Caribbean Hispanics or other minority populations. Despite this, the results of that GWAS served as the foundation for the current CPIC guideline on clopidogrel, which centers primarily on the utility of the *CYP2C19**2 variant as the primary risk allele for predicting poor antiplatelet response and increased likelihood of MACCEs in ACS patients undergoing PCI, thus informing recommendations for escalation/de-escalation strategies. Indeed, there is a critical void in the current CPIC guidelines on clopidogrel because of the lack of adequate replications of major findings (i.e., *CYP2C19**2) in minority populations such as Caribbean Hispanics. Although no GWAS significant hits were found, our work is the first to provide credible evidence of the lack of a significant effect of *CYP2C19**2 as a risk allele in predicting poor response to clopidogrel among those Caribbean Hispanic patients who have a high contribution of non-European ancestry on this locus, which we think is a very relevant contribution to the field.

## Methods

This was a multicenter, observational, case-control, clinical study protocol to perform an unbiased GWAS of clopidogrel response in Caribbean Hispanics. Five hundred and eleven patients of Hispanic descent who reside in the Commonwealth of Puerto Rico and received either a 600 or 300 mg loading dose and a daily 75 mg maintenance dose of clopidogrel (i.e., alone or as a component of dual antiplatelet therapy (DAPT)) were recruited into the study. Enrollment occurred between January 2018 and June 2020 for patients who were prescribed clopidogrel after either percutaneous coronary interventions (PCI) with ACS or for stable CAD with elective stenting. All patients received a daily maintenance dose of clopidogrel for a minimum of 7 days prior to enrollment. This study received Institutional Review Board approval (#A4070417) and adhered to the study protocol ([NCT03419325](https://clinicaltrials.gov/ct2/show/NCT03419325)). We confirm that we complied with all relevant ethical regulations, including the Declaration of Helsinki, in conducting this study. Figure [1](#Fig1) shows the number of participants who were screened, enrolled, and subsequently completed the study from seven different medical facilities across the island. The study used minimal inclusion and exclusion criteria (Table [4](#Tab4)) to recruit a diverse population representative of real-world clinical practice. Medication adherence was evaluated through self-reporting and record reviews, following a standardized methodology as previously described^35^.

### Table 4.

| Inclusion criteria | Exclusion criteria |
| --- | --- |
| Caribbean Hispanics residing in Puerto Rico Both genders (i.e., males/females) Age ≥ 21 years old Receiving clopidogrel (75 mg/day) alone or in combination with aspirin as DAPT for therapeutic indications (ACS, stable CAD, PAD). No clinically active hepatic abnormality. The ability to understand the requirements of the study. The ability to comply with the study procedures and protocols. A female patient is eligible to enter the study if she is of child-bearing potential and not pregnant or nursing, or not of child-bearing potential. | Non-Hispanic patients Currently enrolled in another active research protocols BUN > 30 and creatinine > 2.0 mg/dL Platelet count <100,000/mm3 Hematocrit (Hct) ≤ 25% Nasogastric or enteral feedings Acute illness (e.g., sepsis, infection, anemia) HIV/AIDS, Hepatitis B patients Alcoholism and drug abuse Patients with any cognitive and mental health impairment Sickle cell patients Active malignancy Patients taking another antiplatelet (i.e., other than clopidogrel and aspirin) |

Table 4 Caption: The criteria for inclusion and exclusion in the study were as follows

Blood samples (20 mL) for genetic testing and rapid ex vivo residual platelet function analysis were collected at a single time-point on the recruitment day while under clopidogrel maintenance daily dosing of at least 7 days (i.e., within a window of 7 to 10 days after starting a 75 mg maintenance dose without any additional loading dose). Platelet reactivity was measured using the VerifyNow P2Y12 assay (Accumetrics, CA, USA), following the manufacturer’s instructions. The results were expressed as P2Y12 reaction units (PRU) and used to determine inhibition of platelets by clopidogrel in each participant. High on-treatment platelet reactivity (HTPR) was defined as PRU ≥ 230, which indicated poor clopidogrel response^36,37^.

### Genotyping and imputation

Genomic DNA was extracted using standard methods (Qiagen, CA, USA). Genotyping was performed using the HiScan® system with the Infinium® Multi-Ethnic Hispanic AMR/AFR MEGA BeadChip array according to manufacturer instructions (Illumina, CA, USA). Standard quality control (QC) procedures were implemented. Any genotyped SNPs that deviated from Hardy-Weinberg equilibrium (HWE) were flagged but not removed. Gender misspecification was checked using X chromosome zygosity. Individuals who did not match known sample data were excluded. Identity-by-descent (IBD) check was performed to identify sample duplicates, contaminated samples, and cryptic relationships. For each pair of samples with estimated IBD coefficients greater than 0.185, only the sample with the highest call rate was retained. Only SNPs with a minor allele frequency (MAF) greater than 5% were included in our analyses. We then imputed the QCed genotype data using the TOPMed Imputation server ([https://imputation.biodatacatalyst.nhlbi.nih.gov/#!](https://imputation.biodatacatalyst.nhlbi.nih.gov/)), which were performed following standard procedures described elsewhere^38^.

### Genome-wide association studies (GWAS) analyses

Covariate frequencies (e.g., means ± SD) and statistics for the study population as well as measures of HWE and linkage disequilibrium (LD) metrics were calculated using R and PLINK v1.9^39^. After QC, 12,343,367 genotyped and imputed variants were evaluated for associations. Manhattan and quantile-quantile (Q-Q) plots were generated to visualize results. GWAS regression analyses using an additive genetic model were adjusted for relevant covariates (e.g., age, sex, body mass index (BMI), diabetes) and the first two principal components (PCs). SNPs with *p*-value ≤ 5·10^−8^ were considered as genome-wide significant, whereas SNPs with *p*-value ≤ 10^−6^ were considered as suggestive association. LocusZoom.js v0.12 was used to capture and visualize regions of interest^40^. All statistical analyses were performed using PLINK v1.9^39^. A separate multivariable logistic regression GWAS was performed to estimate the odds ratio and 95%CI for HTPR adjusted for covariates using PLINK v1.9^39^. Functional annotation of SNPs was obtained by ANNOVAR^41^.

### Population and local specific ancestry estimations

Global and local ancestry for each individual level were estimated to adjust the association analysis by admixture. Three different ancestral populations were utilized as reference panels for ancestry estimation. Genotype data from a total of 107 individuals of European ancestry (IBS: Iberian populations in Spain) and 61 of African ancestry (YRI: Yoruba in Ibadan, Nigeria) from the Phase-3 1000 Genomes project (1KGP)^42^, and 103 Native Mesoamerican and South American individuals from the Human Genome Diversity Project/*Centre d’Etude du Polymorphism* (HGDP-CEPH) database (AMR: Native American populations distributed as follows: Maya from southern Mexico; Colombian; Karitiana from western Amazon, Brazil; Surui from Mato-Grosso, Brazil; and Pima from central/southern Arizona and northwestern Mexico) were used to infer ancestry proportions and local ancestry^43^.

The reference panels were combined into a single VCF file that included the study cohort data using BCFtools v1.9^44^ and VCFtools v0.1.13^45^, for pre-processing and combining references with study samples. We used PLINK v1.9^39^ software to convert the combined VCF file into a PLINK binary file (BED/BIM/FAM), which was used as input for ADMIXTUREv1.3.0^46^ and RFMixv2^47^ software. For quality assurance and control, we performed a multidimensional PC analysis using PLINK to observe clustering between references and study data. We used ADMIXTURE to estimate proportions of ancestry using a *K* = 3, considering our three reference populations (YRI, IBS, and AMR). We used RFMix with a window size of 0.2 cM, number of generations equal to 8, and the number of trees to generate per Random Forest set to 100.

### Local ancestry inferences (LAI) adjusted GWAS

The LAI GWAS was performed to improve the power to detect association over standard GWAS using the Tractor software package^48^ adjusted for age, sex, diabetes, BMI, PC1, and PC2. Briefly, the RFMix ancestry calls were converted to Tractor format, which includes genotype dosage and haplotype count information for African, Native American, and European ancestry at each SNP position. The Tractor local ancestry GWAS was then performed using PLINK. This analysis allowed us to use the LAI at each SNPs position as an SNP-based covariate in the GWAS for this admixed cohort. All clinical covariates that showed association to PRU or HTPR were included as covariates in the analysis. The deconvoluted model within TRACTOR was used in this analysis^48^. Meta-analyses on the deconvoluted AFR, AMR, and EUR summary statistics were conducted using METAL using a random-effect model. We pre-specified SNPs at *p*-value ≤ 5·10^−8^ as significant, and those at *p*-value ≤ 10^−6^ as suggestive.

### Replication cohorts

The replication cohorts consisted of 167 African Americans and 200 Brazilians on maintenance doses of clopidogrel enrolled from 5 hospital systems in Chicago and Washington DC (University of Chicago Medical Center, University of Illinois and Northwestern Memorial Hospital, George Washington University Hospital, and the Washington DC VA Medical Center) through the African American Cardiovascular Pharmacogenomics Consortium (ACCOuNT)^49^, *Hospital de Clínicas de Porto Alegre*, and biorepository of the Universidade Federal do Rio Grande do Sul, in Porto Alegre, Brazil; these samples were independent from those used in the discovery cohort. All patients provided written, informed consent for study participation and were genotyped as described above via the Infinium® Multi-Ethnic MEGA BeadChip array. Available clinical data from the replication dataset included sex, age, BMI, PRU measures, concomitant medications, diabetes, and indication for therapy. HTPR was pre-specified as PRU ≥ 230 on clopidogrel therapy as previously described. LAIs of each subject were estimated with RFMix v2^47^ as described above but using YRI and CEU samples from 1KGP^42^ as the reference populations (ACCOuNT cohort only). For the replication models, we selected the same clinical covariates used in the discovery models. Both PRUs and HTPR were modeled using significant covariates and the top hits by the linear and logistic regression model functions in PLINK v1.9^39^. Because we tested only the three SNPs that were significant by regular GWAS in our discovery cohort when using a linear model (PRU) and the one identified after performing logistic regression (HTPR), as well as the top two SNPs in LAI GWAS, we did not correct for multiple hypothesis testing (i.e., *p*-value ≤ 0.05 significance was used instead). In addition, we used available data from the GWAS conducted in the ICPC cohort for external validation. A detailed description of the ICPC cohort and results can be found elsewhere^6^.

## Supplementary information

## Acknowledgements

We thank the patients for their voluntary participation in this study. Special thanks to the Research Design and Biostatistics Core of The Alliance/Puerto Rico Clinical and Translational Research Consortium (PRCTRC), supported by NIMHD and NIAID (award # U54MD007587), for assistance with study design and sample size calculations. We also appreciate the Bioinformatics Core of the CCRHD-RCMI Program at UPR-MSC for their support with GWAS analysis, admixture, and ancestry measurements. Thanks to Edgardo R González, Dr. Ednalise Santiago, Dr. Eduardo Tosado, Dr. Héctor Nuñez, Dr. Ariel González, Dr. Laura Ileana Fernandez-Morales, Dr. Luis Antonio Velez-Figueroa, Dr. Orlando Arce, Frances Marín-Maldonado, Andrés López-Reyes, and Marines Rosario for help with data collection and patient recruitment. We are grateful to Dr. Mara Helena Hutz from the Department of Genetics, Instituto de Biociências, Universidade Federal do Rio Grande do Sul, Brazil, for providing Brazilian samples from their biorepository. This work was supported by CCRHD-RCMI grant #2U54 MD007600-33 (NIMHD/NIH), the 2023 Pharmacogenomics Global Research Network (PGRN) Research Collaboration Grant, 1U54 MD010723 (NIMHD/NIH), and the NIGMS-Research Training Initiative for Student Enhancement (RISE) Program grant R25 GM061838. No funded writing assistance was used. The views expressed in this manuscript do not reflect those of the NIH or the U.S. Government.

## Author contributions

All the authors have accepted responsibility for the entire content of this research article and approved submission. G.Y., P.G., M.M., J.D.S., and M.A.P. drafted the manuscript; S.A.S., G.R., A.R.L., D.F.H.S., K.M., M.D.R. assisted with review and editing of the final manuscript; J.D.S., M.A.P., S.A.S., G.R., A.R.L. designed the study and help interpreted results; P.G., M.M., J.Y.R., D.F.H.S., M.R.B., K.M. and J.D. assisted with patient recruitments and data collection; D.F.H.S., K.M. assessed patient events and adjudicated MACCEs; M.M., J.Y.R., M.R.B., K.M., J.D. performed genotyping and PRU testing; M.R.B. and M.D.R. facilitated access to available databases; G.Y., P.G., M.M., K.C., C.A., J.D., M.A.P. performed data analyses, statistical tests and run imputations, QC, GWAS, meta-analysis.

## Data availability

The datasets used and/or analyzed during the current study are available from the corresponding author on reasonable request.

## Competing Interests

The authors declare no competing interests.

## Footnotes

## Supplementary information

The online version contains supplementary material available at 10.1038/s41525-025-00479-3.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

The datasets used and/or analyzed during the current study are available from the corresponding author on reasonable request.

### Supplementary Materials

### Data Availability Statement

The datasets used and/or analyzed during the current study are available from the corresponding author on reasonable request.

## References

1. Shuldiner, A. R. et al. Association of cytochrome P450 2C19 genotype with the antiplatelet effect and clinical efficacy of clopidogrel therapy. JAMA302, 849–857 (2009).  [DOI](https://doi.org/10.1001/jama.2009.1232) | [PMC free article](/articles/PMC3641569/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19706858/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Shuldiner,%20A.%20R.%20et%20al.%20Association%20of%20cytochrome%20P450%202C19%20genotype%20with%20the%20antiplatelet%20effect%20and%20clinical%20efficacy%20of%20clopidogrel%20therapy.%20JAMA302,%20849%E2%80%93857%20(2009).)

2. van den Broek, W. W. A. et al. Genotype-guided antiplatelet therapy: JACC review topic of the week. J. Am. Coll. Cardiol.84, 1107–1118 (2024).  [DOI](https://doi.org/10.1016/j.jacc.2024.06.038) | [PMC free article](/articles/PMC11495226/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/39260933/) | [Google Scholar](https://scholar.google.com/scholar_lookup?van%20den%20Broek,%20W.%20W.%20A.%20et%20al.%20Genotype-guided%20antiplatelet%20therapy:%20JACC%20review%20topic%20of%20the%20week.%20J.%20Am.%20Coll.%20Cardiol.84,%201107%E2%80%931118%20(2024).)

3. Lewis, J. P. & Shuldiner, A. R. Clopidogrel pharmacogenetics: beyond candidate genes and genome-wide association studies. Clin. Pharm. Ther.101, 323–325 (2017).  [DOI](https://doi.org/10.1002/cpt.519) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27649515/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Lewis,%20J.%20P.%20&%20Shuldiner,%20A.%20R.%20Clopidogrel%20pharmacogenetics:%20beyond%20candidate%20genes%20and%20genome-wide%20association%20studies.%20Clin.%20Pharm.%20Ther.101,%20323%E2%80%93325%20(2017).)

4. Mega, J. L. et al. Reduced-function CYP2C19 genotype and risk of adverse clinical outcomes among patients treated with clopidogrel predominantly for PCI: a meta-analysis. JAMA304, 1821–1830 (2010).  [DOI](https://doi.org/10.1001/jama.2010.1543) | [PMC free article](/articles/PMC3048820/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20978260/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Mega,%20J.%20L.%20et%20al.%20Reduced-function%20CYP2C19%20genotype%20and%20risk%20of%20adverse%20clinical%20outcomes%20among%20patients%20treated%20with%20clopidogrel%20predominantly%20for%20PCI:%20a%20meta-analysis.%20JAMA304,%201821%E2%80%931830%20(2010).)

5. Mega, J. L. et al. Cytochrome p-450 polymorphisms and response to clopidogrel. N. Engl. J. Med.360, 354–362 (2009).  [DOI](https://doi.org/10.1056/NEJMoa0809171) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19106084/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Mega,%20J.%20L.%20et%20al.%20Cytochrome%20p-450%20polymorphisms%20and%20response%20to%20clopidogrel.%20N.%20Engl.%20J.%20Med.360,%20354%E2%80%93362%20(2009).)

6. Verma, S. S. et al. Genomewide association study of platelet reactivity and cardiovascular response in patients treated with clopidogrel: a study by the international clopidogrel pharmacogenomics consortium. Clin. Pharm. Ther.108, 1067–1077 (2020).  [DOI](https://doi.org/10.1002/cpt.1911) | [PMC free article](/articles/PMC7689744/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32472697/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Verma,%20S.%20S.%20et%20al.%20Genomewide%20association%20study%20of%20platelet%20reactivity%20and%20cardiovascular%20response%20in%20patients%20treated%20with%20clopidogrel:%20a%20study%20by%20the%20international%20clopidogrel%20pharmacogenomics%20consortium.%20Clin.%20Pharm.%20Ther.108,%201067%E2%80%931077%20(2020).)

7. Sirugo, G., Williams, S. M. & Tishkoff, S. A. The missing diversity in human genetic studies. Cell177, 1080 (2019).  [DOI](https://doi.org/10.1016/j.cell.2019.02.048) | [PMC free article](/articles/PMC7380073/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30901543/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Sirugo,%20G.,%20Williams,%20S.%20M.%20&%20Tishkoff,%20S.%20A.%20The%20missing%20diversity%20in%20human%20genetic%20studies.%20Cell177,%201080%20(2019).)

8. Levine, G. N. et al. 2016 ACC/AHA guideline focused update on duration of dual antiplatelet therapy in patients with coronary artery disease: a report of the American College of Cardiology/American Heart Association Task Force on Clinical Practice Guidelines. J. Thorac. Cardiovasc Surg.152, 1243–1275 (2016).  [DOI](https://doi.org/10.1016/j.jtcvs.2016.07.044) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27751237/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Levine,%20G.%20N.%20et%20al.%202016%20ACC/AHA%20guideline%20focused%20update%20on%20duration%20of%20dual%20antiplatelet%20therapy%20in%20patients%20with%20coronary%20artery%20disease:%20a%20report%20of%20the%20American%20College%20of%20Cardiology/American%20Heart%20Association%20Task%20Force%20on%20Clinical%20Practice%20Guidelines.%20J.%20Thorac.%20Cardiovasc%20Surg.152,%201243%E2%80%931275%20(2016).)

9. Lee, C. R. et al. Clinical pharmacogenetics implementation consortium guideline for CYP2C19 genotype and clopidogrel therapy: 2022 update. Clin. Pharm. Ther.112, 959–967 (2022).  [DOI](https://doi.org/10.1002/cpt.2526) | [PMC free article](/articles/PMC9287492/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35034351/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Lee,%20C.%20R.%20et%20al.%20Clinical%20pharmacogenetics%20implementation%20consortium%20guideline%20for%20CYP2C19%20genotype%20and%20clopidogrel%20therapy:%202022%20update.%20Clin.%20Pharm.%20Ther.112,%20959%E2%80%93967%20(2022).)

10. Claassens, D. M. F. et al. Clopidogrel versus ticagrelor or prasugrel after primary percutaneous coronary intervention according to CYP2C19 genotype: a POPular genetics subanalysis. Circ. Cardiovasc. Inter.14, e009434 (2021).  [DOI](https://doi.org/10.1161/CIRCINTERVENTIONS.120.009434) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33722066/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Claassens,%20D.%20M.%20F.%20et%20al.%20Clopidogrel%20versus%20ticagrelor%20or%20prasugrel%20after%20primary%20percutaneous%20coronary%20intervention%20according%20to%20CYP2C19%20genotype:%20a%20POPular%20genetics%20subanalysis.%20Circ.%20Cardiovasc.%20Inter.14,%20e009434%20(2021).)

11. Klein, M. D., Williams, A. K., Lee, C. R. & Stouffer, G. A. Clinical utility of CYP2C19 genotyping to guide antiplatelet therapy in patients with an acute coronary syndrome or undergoing percutaneous coronary intervention. Arterioscler Thromb. Vasc. Biol.39, 647–652 (2019).  [DOI](https://doi.org/10.1161/ATVBAHA.118.311963) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30760018/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Klein,%20M.%20D.,%20Williams,%20A.%20K.,%20Lee,%20C.%20R.%20&%20Stouffer,%20G.%20A.%20Clinical%20utility%20of%20CYP2C19%20genotyping%20to%20guide%20antiplatelet%20therapy%20in%20patients%20with%20an%20acute%20coronary%20syndrome%20or%20undergoing%20percutaneous%20coronary%20intervention.%20Arterioscler%20Thromb.%20Vasc.%20Biol.39,%20647%E2%80%93652%20(2019).)

12. Malik, A. H. et al. Effect of genotype-guided oral P2Y12 inhibitor selection after percutaneous coronary intervention: a systematic review and meta-analysis of randomized clinical trials. Cardiovasc. Revasc. Med.41, 115–121 (2022).  [DOI](https://doi.org/10.1016/j.carrev.2022.01.005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35033458/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Malik,%20A.%20H.%20et%20al.%20Effect%20of%20genotype-guided%20oral%20P2Y12%20inhibitor%20selection%20after%20percutaneous%20coronary%20intervention:%20a%20systematic%20review%20and%20meta-analysis%20of%20randomized%20clinical%20trials.%20Cardiovasc.%20Revasc.%20Med.41,%20115%E2%80%93121%20(2022).)

13. Duconge, J. & Ruaño, G. Admixture and ethno-specific alleles: missing links for global pharmacogenomics. Pharmacogenomics17, 1479–1482 (2016).  [DOI](https://doi.org/10.2217/pgs-2016-0115) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27537834/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Duconge,%20J.%20&%20Rua%C3%B1o,%20G.%20Admixture%20and%20ethno-specific%20alleles:%20missing%20links%20for%20global%20pharmacogenomics.%20Pharmacogenomics17,%201479%E2%80%931482%20(2016).)

14. Duconge, J. & Ruaño, G. Preventing the exacerbation of health disparities by iatrogenic pharmacogenomic applications: lessons from warfarin. Pharmacogenomics19, 875–881 (2018).  [DOI](https://doi.org/10.2217/pgs-2018-0055) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29898627/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Duconge,%20J.%20&%20Rua%C3%B1o,%20G.%20Preventing%20the%20exacerbation%20of%20health%20disparities%20by%20iatrogenic%20pharmacogenomic%20applications:%20lessons%20from%20warfarin.%20Pharmacogenomics19,%20875%E2%80%93881%20(2018).)

15. Botton, M. R. et al. PharmVar GeneFocus: CYP2C19. Clin. Pharm. Ther.109, 352–366 (2021).  [DOI](https://doi.org/10.1002/cpt.1973) | [PMC free article](/articles/PMC7769975/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32602114/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Botton,%20M.%20R.%20et%20al.%20PharmVar%20GeneFocus:%20CYP2C19.%20Clin.%20Pharm.%20Ther.109,%20352%E2%80%93366%20(2021).)

16. Yang, G. et al. The role of global and local ancestry on clopidogrel response in African Americans. Pac. Symp. Biocomput.28, 221–232 (2023).  [PMC free article](/articles/PMC9782753/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36540979/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Yang,%20G.%20et%20al.%20The%20role%20of%20global%20and%20local%20ancestry%20on%20clopidogrel%20response%20in%20African%20Americans.%20Pac.%20Symp.%20Biocomput.28,%20221%E2%80%93232%20(2023).)

17. Angulo-Aguado, M. et al. A pharmacogenetic study of CYP2C19 in acute coronary syndrome patients of Colombian origin reveals new polymorphisms potentially related to clopidogrel therapy. J. Pers. Med.11, 400 (2021).  [DOI](https://doi.org/10.3390/jpm11050400) | [PMC free article](/articles/PMC8150782/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34065778/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Angulo-Aguado,%20M.%20et%20al.%20A%20pharmacogenetic%20study%20of%20CYP2C19%20in%20acute%20coronary%20syndrome%20patients%20of%20Colombian%20origin%20reveals%20new%20polymorphisms%20potentially%20related%20to%20clopidogrel%20therapy.%20J.%20Pers.%20Med.11,%20400%20(2021).)

18. Mester, R. et al. Impact of cross-ancestry genetic architecture on GWASs in admixed populations. Am. J. Hum. Genet.110, 927–939 (2023).  [DOI](https://doi.org/10.1016/j.ajhg.2023.05.001) | [PMC free article](/articles/PMC10257009/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/37224807/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Mester,%20R.%20et%20al.%20Impact%20of%20cross-ancestry%20genetic%20architecture%20on%20GWASs%20in%20admixed%20populations.%20Am.%20J.%20Hum.%20Genet.110,%20927%E2%80%93939%20(2023).)

19. Zhong, W. P. et al. Genomewide association study identifies novel genetic loci that modify antiplatelet effects and pharmacokinetics of clopidogrel. Clin. Pharm. Ther.101, 791–802 (2017).  [DOI](https://doi.org/10.1002/cpt.589) | [PMC free article](/articles/PMC5485718/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27981573/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Zhong,%20W.%20P.%20et%20al.%20Genomewide%20association%20study%20identifies%20novel%20genetic%20loci%20that%20modify%20antiplatelet%20effects%20and%20pharmacokinetics%20of%20clopidogrel.%20Clin.%20Pharm.%20Ther.101,%20791%E2%80%93802%20(2017).)

20. Backman, J. D. et al. Genome-wide analysis of clopidogrel active metabolite levels identifies novel variants that influence antiplatelet response. Pharmacogenet. Genom.27, 159–163 (2017).  [DOI](https://doi.org/10.1097/FPC.0000000000000272) | [PMC free article](/articles/PMC5346037/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28207573/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Backman,%20J.%20D.%20et%20al.%20Genome-wide%20analysis%20of%20clopidogrel%20active%20metabolite%20levels%20identifies%20novel%20variants%20that%20influence%20antiplatelet%20response.%20Pharmacogenet.%20Genom.27,%20159%E2%80%93163%20(2017).)

21. SEARCH Collaborative Group et al. SLCO1B1 variants and statin-induced myopathy-a genomewide study. N. Engl. J. Med.359, 789–799 (2008).  [DOI](https://doi.org/10.1056/NEJMoa0801936) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18650507/) | [Google Scholar](https://scholar.google.com/scholar_lookup?SEARCH%20Collaborative%20Group%20et%20al.%20SLCO1B1%20variants%20and%20statin-induced%20myopathy-a%20genomewide%20study.%20N.%20Engl.%20J.%20Med.359,%20789%E2%80%93799%20(2008).)

22. Perry, R. J. & Ridgway, N. D. Oxysterol-binding protein and vesicle-associated membrane protein-associated protein are required for sterol-dependent activation of the ceramide transport protein. Mol. Biol. Cell17, 2604–2616 (2006).  [DOI](https://doi.org/10.1091/mbc.E06-01-0060) | [PMC free article](/articles/PMC1474796/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16571669/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Perry,%20R.%20J.%20&%20Ridgway,%20N.%20D.%20Oxysterol-binding%20protein%20and%20vesicle-associated%20membrane%20protein-associated%20protein%20are%20required%20for%20sterol-dependent%20activation%20of%20the%20ceramide%20transport%20protein.%20Mol.%20Biol.%20Cell17,%202604%E2%80%932616%20(2006).)

23. Perttilä, J. et al. OSBPL10, a novel candidate gene for high triglyceride trait in dyslipidemic Finnish subjects, regulates cellular lipid metabolism. J. Mol. Med.87, 825–835 (2009).  [DOI](https://doi.org/10.1007/s00109-009-0490-z) | [PMC free article](/articles/PMC2707950/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19554302/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Perttil%C3%A4,%20J.%20et%20al.%20OSBPL10,%20a%20novel%20candidate%20gene%20for%20high%20triglyceride%20trait%20in%20dyslipidemic%20Finnish%20subjects,%20regulates%20cellular%20lipid%20metabolism.%20J.%20Mol.%20Med.87,%20825%E2%80%93835%20(2009).)

24. Koriyama, H. et al. Identification of evidence suggestive of an association with peripheral arterial disease at the OSBPL10 locus by genome-wide investigation in the Japanese population. J. Atheroscler. Thromb.17, 1054–1062 (2010).  [DOI](https://doi.org/10.5551/jat.4291) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20610895/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Koriyama,%20H.%20et%20al.%20Identification%20of%20evidence%20suggestive%20of%20an%20association%20with%20peripheral%20arterial%20disease%20at%20the%20OSBPL10%20locus%20by%20genome-wide%20investigation%20in%20the%20Japanese%20population.%20J.%20Atheroscler.%20Thromb.17,%201054%E2%80%931062%20(2010).)

25. Sibbing, D. et al. Updated expert consensus statement on platelet function and genetic testing for guiding P2Y12 receptor inhibitor treatment in percutaneous. Coron. Interv. JACC Cardiovasc. Interv.12, 1521–1537 (2019).  [DOI](https://doi.org/10.1016/j.jcin.2019.03.034) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31202949/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Sibbing,%20D.%20et%20al.%20Updated%20expert%20consensus%20statement%20on%20platelet%20function%20and%20genetic%20testing%20for%20guiding%20P2Y12%20receptor%20inhibitor%20treatment%20in%20percutaneous.%20Coron.%20Interv.%20JACC%20Cardiovasc.%20Interv.12,%201521%E2%80%931537%20(2019).)

26. Genest, J. J. Jr. et al. Familial lipoprotein disorders in patients with premature coronary artery disease. Circulation85, 2025–2033 (1992).  [DOI](https://doi.org/10.1161/01.cir.85.6.2025) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1534286/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Genest,%20J.%20J.%20Jr.%20et%20al.%20Familial%20lipoprotein%20disorders%20in%20patients%20with%20premature%20coronary%20artery%20disease.%20Circulation85,%202025%E2%80%932033%20(1992).)

27. Belmont, P. J. et al. Roles for endoplasmic reticulum-associated degradation and the novel endoplasmic reticulum stress response gene Derlin-3 in the ischemic heart. Circ. Res.106, 307–316 (2010).  [DOI](https://doi.org/10.1161/CIRCRESAHA.109.203901) | [PMC free article](/articles/PMC3330119/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19940266/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Belmont,%20P.%20J.%20et%20al.%20Roles%20for%20endoplasmic%20reticulum-associated%20degradation%20and%20the%20novel%20endoplasmic%20reticulum%20stress%20response%20gene%20Derlin-3%20in%20the%20ischemic%20heart.%20Circ.%20Res.106,%20307%E2%80%93316%20(2010).)

28. Eura, Y., Miyata, T. & Kokame, K. Derlin-3 is required for changes in ERAD complex formation under ER stress. Int. J. Mol. Sci.21, 6146 (2020).  [DOI](https://doi.org/10.3390/ijms21176146) | [PMC free article](/articles/PMC7504720/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32858914/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Eura,%20Y.,%20Miyata,%20T.%20&%20Kokame,%20K.%20Derlin-3%20is%20required%20for%20changes%20in%20ERAD%20complex%20formation%20under%20ER%20stress.%20Int.%20J.%20Mol.%20Sci.21,%206146%20(2020).)

29. Huang, Z. et al. Regulator of G protein signaling 6 facilities cardiac hypertrophy by activating apoptosis signal-regulating kinase 1-P38/c-JUN N-terminal kinase 1/2 signaling. J. Am. Heart Assoc.7, e009179 (2018).  [DOI](https://doi.org/10.1161/JAHA.118.009179) | [PMC free article](/articles/PMC6404897/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30371330/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Huang,%20Z.%20et%20al.%20Regulator%20of%20G%20protein%20signaling%206%20facilities%20cardiac%20hypertrophy%20by%20activating%20apoptosis%20signal-regulating%20kinase%201-P38/c-JUN%20N-terminal%20kinase%201/2%20signaling.%20J.%20Am.%20Heart%20Assoc.7,%20e009179%20(2018).)

30. Rorabaugh, B. R. et al. Regulator of G protein signaling 6 protects the heart from ischemic injury. J. Pharm. Exp. Ther.360, 409–416 (2017).  [DOI](https://doi.org/10.1124/jpet.116.238345) | [PMC free article](/articles/PMC5325075/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28035008/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Rorabaugh,%20B.%20R.%20et%20al.%20Regulator%20of%20G%20protein%20signaling%206%20protects%20the%20heart%20from%20ischemic%20injury.%20J.%20Pharm.%20Exp.%20Ther.360,%20409%E2%80%93416%20(2017).)

31. van der Harst, P. & Verweij, N. The identification of 64 novel genetic loci provides an expanded view on the genetic architecture of coronary artery disease. Circ. Res.122, 433–443 (2017).  [DOI](https://doi.org/10.1161/CIRCRESAHA.117.312086) | [PMC free article](/articles/PMC5805277/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29212778/) | [Google Scholar](https://scholar.google.com/scholar_lookup?van%20der%20Harst,%20P.%20&%20Verweij,%20N.%20The%20identification%20of%2064%20novel%20genetic%20loci%20provides%20an%20expanded%20view%20on%20the%20genetic%20architecture%20of%20coronary%20artery%20disease.%20Circ.%20Res.122,%20433%E2%80%93443%20(2017).)

32. Erdmann, J., Kessler, T., Munoz-Venegas, L. & Schunkert, H. A decade of genome-wide association studies for coronary artery disease: the challenges ahead. Cardiovasc. Res.114, 1241–1257 (2018).  [DOI](https://doi.org/10.1093/cvr/cvy084) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29617720/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Erdmann,%20J.,%20Kessler,%20T.,%20Munoz-Venegas,%20L.%20&%20Schunkert,%20H.%20A%20decade%20of%20genome-wide%20association%20studies%20for%20coronary%20artery%20disease:%20the%20challenges%20ahead.%20Cardiovasc.%20Res.114,%201241%E2%80%931257%20(2018).)

33. Baran, Y. et al. Fast and accurate inference of local ancestry in Latino populations. Bioinformatics28, 1359–1367 (2012).  [DOI](https://doi.org/10.1093/bioinformatics/bts144) | [PMC free article](/articles/PMC3348558/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22495753/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Baran,%20Y.%20et%20al.%20Fast%20and%20accurate%20inference%20of%20local%20ancestry%20in%20Latino%20populations.%20Bioinformatics28,%201359%E2%80%931367%20(2012).)

34. Hou, K., Bhattacharya, A., Mester, R., Burch, K. S. & Pasaniuc, B. On powerful GWAS in admixed populations. Nat. Genet.53, 1631–1633 (2021).  [DOI](https://doi.org/10.1038/s41588-021-00953-5) | [PMC free article](/articles/PMC8939372/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34824480/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Hou,%20K.,%20Bhattacharya,%20A.,%20Mester,%20R.,%20Burch,%20K.%20S.%20&%20Pasaniuc,%20B.%20On%20powerful%20GWAS%20in%20admixed%20populations.%20Nat.%20Genet.53,%201631%E2%80%931633%20(2021).)

35. Hernandez-Suarez, D. F. et al. Implementing a pharmacogenetic-driven algorithm to guide dual antiplatelet therapy (DAPT) in Caribbean Hispanics: protocol for a non-randomised clinical trial. BMJ Open10, e038936 (2020).  [DOI](https://doi.org/10.1136/bmjopen-2020-038936) | [PMC free article](/articles/PMC7412606/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32764090/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Hernandez-Suarez,%20D.%20F.%20et%20al.%20Implementing%20a%20pharmacogenetic-driven%20algorithm%20to%20guide%20dual%20antiplatelet%20therapy%20(DAPT)%20in%20Caribbean%20Hispanics:%20protocol%20for%20a%20non-randomised%20clinical%20trial.%20BMJ%20Open10,%20e038936%20(2020).)

36. Aradi, D. et al. Bleeding and stent thrombosis on P2Y12-inhibitors: collaborative analysis on the role of platelet reactivity for risk stratification after percutaneous coronary intervention. Eur. Heart J.36, 1762–1771 (2015).  [DOI](https://doi.org/10.1093/eurheartj/ehv104) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25896078/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Aradi,%20D.%20et%20al.%20Bleeding%20and%20stent%20thrombosis%20on%20P2Y12-inhibitors:%20collaborative%20analysis%20on%20the%20role%20of%20platelet%20reactivity%20for%20risk%20stratification%20after%20percutaneous%20coronary%20intervention.%20Eur.%20Heart%20J.36,%201762%E2%80%931771%20(2015).)

37. Hernandez-Suarez, D. F. et al. Pharmacogenetic association study on clopidogrel response in Puerto Rican Hispanics with cardiovascular disease: a novel characterization of a Caribbean population. Pharmgenomics Pers. Med.11, 95–106 (2018).  [DOI](https://doi.org/10.2147/PGPM.S165805) | [PMC free article](/articles/PMC5996853/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29922082/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Hernandez-Suarez,%20D.%20F.%20et%20al.%20Pharmacogenetic%20association%20study%20on%20clopidogrel%20response%20in%20Puerto%20Rican%20Hispanics%20with%20cardiovascular%20disease:%20a%20novel%20characterization%20of%20a%20Caribbean%20population.%20Pharmgenomics%20Pers.%20Med.11,%2095%E2%80%93106%20(2018).)

38. Verma, S. S. et al. Imputation and quality control steps for combining multiple genome-wide datasets. Front. Genet.5, 370 (2014).  [DOI](https://doi.org/10.3389/fgene.2014.00370) | [PMC free article](/articles/PMC4263197/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25566314/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Verma,%20S.%20S.%20et%20al.%20Imputation%20and%20quality%20control%20steps%20for%20combining%20multiple%20genome-wide%20datasets.%20Front.%20Genet.5,%20370%20(2014).)

39. Purcell, S. et al. PLINK: a tool set for whole-genome association and population-based linkage analyses. Am. J. Hum. Genet.81, 559–575 (2007).  [DOI](https://doi.org/10.1086/519795) | [PMC free article](/articles/PMC1950838/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17701901/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Purcell,%20S.%20et%20al.%20PLINK:%20a%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses.%20Am.%20J.%20Hum.%20Genet.81,%20559%E2%80%93575%20(2007).)

40. Pruim, R. J. et al. LocusZoom: regional visualization of genome-wide association scan results. Bioinformatics26, 2336–2337 (2010).  [DOI](https://doi.org/10.1093/bioinformatics/btq419) | [PMC free article](/articles/PMC2935401/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20634204/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Pruim,%20R.%20J.%20et%20al.%20LocusZoom:%20regional%20visualization%20of%20genome-wide%20association%20scan%20results.%20Bioinformatics26,%202336%E2%80%932337%20(2010).)

41. Wang, K., Li, M. & Hakonarson, H. ANNOVAR: functional annotation of genetic variants from high-throughput sequencing data. Nucleic Acids Res38, e164 (2010).  [DOI](https://doi.org/10.1093/nar/gkq603) | [PMC free article](/articles/PMC2938201/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20601685/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Wang,%20K.,%20Li,%20M.%20&%20Hakonarson,%20H.%20ANNOVAR:%20functional%20annotation%20of%20genetic%20variants%20from%20high-throughput%20sequencing%20data.%20Nucleic%20Acids%20Res38,%20e164%20(2010).)

42. 1000 Genomes Project Consortium et al. A global reference for human genetic variation. Nature526, 68–74 (2015).  [DOI](https://doi.org/10.1038/nature15393) | [PMC free article](/articles/PMC4750478/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26432245/)

43. Cavalli-Sforza, L. L. The Human Genome Diversity Project: past, present and future. Nat. Rev. Genet.6, 333–340 (2005).  [DOI](https://doi.org/10.1038/nrg1596) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15803201/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Cavalli-Sforza,%20L.%20L.%20The%20Human%20Genome%20Diversity%20Project:%20past,%20present%20and%20future.%20Nat.%20Rev.%20Genet.6,%20333%E2%80%93340%20(2005).)

44. Danecek, P. et al. Twelve years of SAMtools and BCFtools. Gigascience10, giab008 (2021).  [DOI](https://doi.org/10.1093/gigascience/giab008) | [PMC free article](/articles/PMC7931819/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33590861/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Danecek,%20P.%20et%20al.%20Twelve%20years%20of%20SAMtools%20and%20BCFtools.%20Gigascience10,%20giab008%20(2021).)

45. Danecek, P. et al. The variant call format and VCFtools. Bioinformatics27, 2156–2158 (2011).  [DOI](https://doi.org/10.1093/bioinformatics/btr330) | [PMC free article](/articles/PMC3137218/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21653522/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Danecek,%20P.%20et%20al.%20The%20variant%20call%20format%20and%20VCFtools.%20Bioinformatics27,%202156%E2%80%932158%20(2011).)

46. Alexander, D. H., Novembre, J. & Lange, K. Fast model-based estimation of ancestry in unrelated individuals. Genome Res.19, 1655–1664 (2009).  [DOI](https://doi.org/10.1101/gr.094052.109) | [PMC free article](/articles/PMC2752134/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19648217/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Alexander,%20D.%20H.,%20Novembre,%20J.%20&%20Lange,%20K.%20Fast%20model-based%20estimation%20of%20ancestry%20in%20unrelated%20individuals.%20Genome%20Res.19,%201655%E2%80%931664%20(2009).)

47. Maples, B. K., Gravel, S., Kenny, E. E. & Bustamante, C. D. RFMix: a discriminative modeling approach for rapid and robust local-ancestry inference. Am. J. Hum. Genet.93, 278–288 (2013).  [DOI](https://doi.org/10.1016/j.ajhg.2013.06.020) | [PMC free article](/articles/PMC3738819/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23910464/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Maples,%20B.%20K.,%20Gravel,%20S.,%20Kenny,%20E.%20E.%20&%20Bustamante,%20C.%20D.%20RFMix:%20a%20discriminative%20modeling%20approach%20for%20rapid%20and%20robust%20local-ancestry%20inference.%20Am.%20J.%20Hum.%20Genet.93,%20278%E2%80%93288%20(2013).)

48. Atkinson, E. G. et al. Tractor uses local ancestry to enable the inclusion of admixed individuals in GWAS and to boost power. Nat. Genet.53, 195–204 (2021).  [DOI](https://doi.org/10.1038/s41588-020-00766-y) | [PMC free article](/articles/PMC7867648/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33462486/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Atkinson,%20E.%20G.%20et%20al.%20Tractor%20uses%20local%20ancestry%20to%20enable%20the%20inclusion%20of%20admixed%20individuals%20in%20GWAS%20and%20to%20boost%20power.%20Nat.%20Genet.53,%20195%E2%80%93204%20(2021).)

49. Friedman, P. N. et al. The ACCOuNT consortium: a model for the discovery, translation, and implementation of precision medicine in African Americans. Clin. Transl. Sci.12, 209–217 (2019).  [DOI](https://doi.org/10.1111/cts.12608) | [PMC free article](/articles/PMC6510376/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30592548/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Friedman,%20P.%20N.%20et%20al.%20The%20ACCOuNT%20consortium:%20a%20model%20for%20the%20discovery,%20translation,%20and%20implementation%20of%20precision%20medicine%20in%20African%20Americans.%20Clin.%20Transl.%20Sci.12,%20209%E2%80%93217%20(2019).)
