# Genetic signatures of ERCC1 and ERCC2 expression, along with SNPs variants, unveil favorable prognosis in SCLC patients undergoing platinum-based chemotherapy

## Metadata
**Authors:** ENRICO CALIMAN, SARA FANCELLI, FEDERICO SCOLARI, ADRIANO PASQUI, CLARA MANNESCHI, DANIELE LAVACCHI, FRANCESCA MAZZONI, FRANCESCA GENSINI, VALERIA PASINI, CAMILLA EVA COMIN, LUCA VOLTOLINI, SERENA PILLOZZI, LORENZO ANTONUZZO
**Journal:** Oncology Research
**Date:** 2024 Dec 20
**DOI:** [10.32604/or.2024.050161](https://doi.org/10.32604/or.2024.050161)
**PMID:** 39735668
**PMCID:** PMC11671403
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11671403/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC11671403/pdf/OncolRes-33-50161.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC11671403/pdf/OncolRes-33-50161.pdf)

## Abstract

**Background:** 
Platinum chemotherapy (CT) remains the backbone of systemic therapy for patients with small-cell lung cancer (SCLC). The nucleotide excision repair (NER) pathway plays a central role in the repair of the DNA damage exerted by platinum agents. Alteration in this repair mechanism may affect patients’ survival.

**Materials and Methods:** 
We conducted a retrospective analysis of data from 38 patients with extensive disease (ED)-SCLC who underwent platinum-CT at the Clinical Oncology Unit, Careggi University Hospital, Florence (Italy), from 2015 to 2020. mRNA expression analysis and single nucleotide polymorphism (SNP) characterization of three NER pathway genes—namely ERCC1, ERCC2, and ERCC5—were performed on patient tumor samples.

**Results:** 
Overall, elevated expression of ERCC genes was observed in SCLC patients compared to healthy controls. Patients with low ERCC1 and ERCC5 expression levels exhibited a better median progression-free survival (mPFS = 7.1 vs. 4.9 months, p = 0.39 for ERCC1 and mPFS = 6.9 vs. 4.8 months, p = 0.093 for ERCC5) and overall survival (mOS = 8.7 vs. 6.0 months, p = 0.4 for ERCC1 and mOS = 7.2 vs. 6.2 months, p = 0.13 for ERCC5). Genotyping analysis of five SNPs of ERCC genes showed a longer survival in patients harboring the wild-type genotype or the heterozygous variant of the ERCC1 rs11615 SNP (p = 0.24 for PFS and p = 0.14 for OS) and of the rs13181 and rs1799793 ERCC2 SNPs (p = 0.43 and p = 0.26 for PFS and p = 0.21 and p = 0.16 for OS, respectively) compared to patients with homozygous mutant genotypes.

**Conclusions:** 
The comprehensive analysis of ERCC gene expression and SNP variants appears to identify patients who derive greater survival benefits from platinum-CT.

Keywords: Small cell lung cancer (SCLC), Nucleotide excision repair (NER) pathway, ERCC genes, Single nucleotide polymorphisms (SNPs), Platinum-chemotherapy (CT)

### Background

Platinum chemotherapy (CT) remains the backbone of systemic therapy for patients with small-cell lung cancer (SCLC). The nucleotide excision repair (NER) pathway plays a central role in the repair of the DNA damage exerted by platinum agents. Alteration in this repair mechanism may affect patients’ survival.

### Materials and Methods

We conducted a retrospective analysis of data from 38 patients with extensive disease (ED)-SCLC who underwent platinum-CT at the Clinical Oncology Unit, Careggi University Hospital, Florence (Italy), from 2015 to 2020. mRNA expression analysis and single nucleotide polymorphism (SNP) characterization of three NER pathway genes—namely *ERCC1*, *ERCC2*, and *ERCC5*—were performed on patient tumor samples.

### Results

Overall, elevated expression of *ERCC* genes was observed in SCLC patients compared to healthy controls. Patients with low *ERCC1* and *ERCC5* expression levels exhibited a better median progression-free survival (mPFS = 7.1 *vs*. 4.9 months, *p* = 0.39 for *ERCC1* and mPFS = 6.9 *vs*. 4.8 months, *p* = 0.093 for *ERCC5*) and overall survival (mOS = 8.7 *vs*. 6.0 months, *p* = 0.4 for *ERCC1* and mOS = 7.2 *vs*. 6.2 months, *p* = 0.13 for *ERCC5*). Genotyping analysis of five SNPs of *ERCC* genes showed a longer survival in patients harboring the wild-type genotype or the heterozygous variant of the *ERCC1* rs11615 SNP (*p* = 0.24 for PFS and *p* = 0.14 for OS) and of the rs13181 and rs1799793 *ERCC2* SNPs (*p* = 0.43 and *p* = 0.26 for PFS and *p* = 0.21 and *p* = 0.16 for OS, respectively) compared to patients with homozygous mutant genotypes.

### Conclusions

The comprehensive analysis of *ERCC* gene expression and SNP variants appears to identify patients who derive greater survival benefits from platinum-CT.

## Introduction

Small-cell lung cancer (SCLC) accounts for approximately 10%–20% of primary lung tumors and most patients present with extensive disease (ED) at the time of diagnosis. Despite a high rate of response to first-line treatment, responses are transient, and patients often experience a rapid disease relapse resulting in poor 2-year survival outcomes [[1](#ref-1)]. Recently, the introduction of immune checkpoint inhibitors (ICIs) in combination with platinum-based chemotherapy (CT) has set a new standard of care for the treatment of ED-SCLC. Nonetheless, platinum-based CT remains the backbone of first-line systemic therapy for these patients [[2](#ref-2)–[4](#ref-4)].

The dismal prognosis of SCLC is strongly related to the development of CT resistance mechanisms by the tumor, including those based on DNA damage repair systems. Platinum agents, particularly cisplatin and carboplatin, exert their cytotoxic effects by inducing DNA damage through the formation of intrastrand adducts and interstrand cross-links that inhibit DNA synthesis and transcription [[5](#ref-5)]. Nucleotide excision repair (NER) pathway plays a central role among DNA damage repair systems and is strictly involved in the detection and repair of DNA adducts and cross-links induced by platinum activity [[6](#ref-6),[7](#ref-7)]. The NER pathway consists of several steps: recognition, incision/excision of DNA lesion, restoration through the synthesis of new nucleotides, and ligation [[8](#ref-8)]. Over 30 different proteins participate in this process and converge at the DNA repair site into the TFIIH transcription initiation complex [[9](#ref-9)]. Proteins that play a crucial role in this pathway, including excision repair cross-complementing group 1 (*ERCC1*), *ERCC2*, and *ERCC5* have been widely studied in different types of tumors, including lung cancer [[10](#ref-10)]. Different expression levels of these proteins can alter DNA repair capacity, modulate cancer susceptibility, and, eventually, impact therapy response and survival of lung cancer patients treated with platinum compounds [[11](#ref-11)]. Moreover, several studies have shown that genetic factors such as single nucleotide polymorphisms (SNPs) in specific genes of the NER pathway may influence the inter-individual differences in platinum effectiveness and toxicity [[12](#ref-12),[13](#ref-13)]. However, the results still remain controversial. To date, far fewer studies have investigated the prognostic and predictive role of these NER pathway genes in patients with SCLC, with no definitive evidence of their influence on platinum-CT responsiveness or patient survival [[14](#ref-14)–[17](#ref-17)].

The aim of our study was to determine the correlation between these genes of the NER pathway and survival outcomes in patients with SCLC. To this aim, we retrospectively analyzed the *ERCC1*, *ERCC2*, and *ERCC5* expression levels and the genotypic characterization of a panel of five single nucleotide polymorphisms (SNPs) of these genes in a cohort of patients with ED-SCLC who received platinum/etoposide regimens.

## Materials and Methods

### Patients

In this single-center observational study, we retrospectively collected clinical-demographic data from medical records of patients with ED-SCLC (according to the VII edition of TNM staging system), who were treated at the Clinical Oncology Unit, Careggi University Hospital, Florence (Italy) between January 2015 to December 2020. All patients enrolled in the analysis received platinum-based standard of care CT as first-line treatment, which was administered up to a maximum of six cycles or was discontinued earlier due to disease progression, intolerable toxicity, clinical decision, or patient refusal. Patients’ responses to CT were assessed with radiological imaging every three months, according to clinical practice. The radiological complete response (CR), partial response (PR), stable disease (SD), and progression disease (PD) were defined in accordance with the Response Evaluation Criteria in Solid Tumors (RECIST) version 1.1. The measured clinical outcomes were the following: Objective Response Rate (ORR, defined as CR plus PR) and Disease Control Rate (DCR, defined as the percentage of patients who have achieved CR or PR or SD), the Progression-Free Survival (PFS) and the Overall Survival (OS). The cut-off date for follow-up analysis was 31 December 2021.

### Data collection

The following characteristics were reviewed from each patient’s medical record: a) clinical-demographic data, i.e., age, sex, Eastern Cooperative Oncology Group Performance Status (ECOG PS), smoking status, and metastasis sites at diagnosis; b) histopathological data from Formalin-Fixed Paraffin-Embedded (FFPE) archival diagnostic tumor tissues of the study population, obtained in collaboration with the Histopathology and Molecular Pathology Unit, Careggi University Hospital, Florence) data about treatment received by patients, such as the type of chemotherapy administered in the first and subsequent lines, the duration of each line of therapy, the radiological responses obtained, and the type of radiotherapy performed, if any (symptomatic or prophylactic cranial irradiation, PCI). Archival FFPE tumor samples of patients were collected and further analyzed for mRNA expression and genotype characterization of key genes of the NER pathway, namely *ERCC1*, *ERCC2*, and *ERCC5*. In detail, mRNA expression was available from a cohort of n = 26 samples, while the analysis of a panel of five SNPs of the genes was assessed in the entire study population (n = 38). Four lung tissue samples from healthy donors were used as a control group for the gene expression analysis.

### DNA extraction and genotyping by SNP assay

DNA was extracted from SCLC FFPE sections using a QIAamp DNA Blood mini kit (Qiagen, Manchester, UK). Allelic discrimination was performed using TaqMan SNP Genotyping Assays (Applied Biosystems, Forster City, CA, USA) on the Rotor-gene 6000 instrument (Qiagen, Manchester, UK). The assay consisted of two allele-specific minor groove binding (MGB) probes, labeled with the fluorescent dyes VIC and FAM. Real-time PCR was performed in 15 μL reaction mixtures containing 7.5 μL of Taqpath ProAmp MasterMix (Applied Biosystems, Forster City, CA, USA). TaqMan-MGB genotyping assay mixes were supplied at 40X concentration and 2 μL of sample DNA. Thermocycler conditions were an initial hold step for 30 s at 60°C and 5 min at 95°C, followed by 40 cycles of 95°C for 15 s and 60°C for 60 s, and finally, a last post read step for 30 s at 60°C. Genotypes were analyzed by measuring allele-specific fluorescence using the Rotor-gene software for allelic discrimination (Qiagen, Manchester, UK).

### RNA extraction and quantitative real-time reverse transcription PCR (qPCR)

Total RNA was extracted from SCLC FFPE sections using the RNeasy Mini kit (Qiagen, Manchester, UK). The quantity and the quality of RNA were evaluated using a Nanodrop spectrophotometer. 50 ng of RNA from each sample were retro-transcribed with the PrimeScript™ RT Reagent Kit (Takara Bio, Japan); the resulting cDNA was then used for qPCR analysis with “Powerup Sybr Green” (Applied Biosystems, Forster City, CA, USA) on a Rotor Gene Q (Qiagen, Manchester, UK) using *ERCC1* (F 5′CTCAAGGAGCTGGCTAAGATGT3′; R 5′CATAGGCCTTGTAGGTCTCCAG3′), *ERCC2* (F 5′CTGGAGGTGACCAAACTCATCTA3′ R 5′CCTGCTTCTCATAGAAGTTGAGC3′) and *ERCC5* (F 5′CAGACACAGCTCCGAATTGA3′ 5′TTCTGGGTTTTTCGTTTTGC3′) primers. The relative quantification was performed using LinRegPCR software and the data were normalized to 18s rRNA. The primers used are listed below.

### Statistical analysis

Patient characteristics were presented by descriptive statistics. Statistical analysis was performed using R [[18](#ref-18)]. The frequency distribution of the five SNPs in our cohort and the general population has been compared by chi-squared test. Relative expression levels have been obtained with the 2^−∆∆Ct^ method [[19](#ref-19)], using as reference gene 18S RNA and as negative control the geometric mean of the expression levels in 4 healthy tissue samples. The correlation between *ERCC* genes’ relative expression and clinical or genotypic parameters has been computed by the Wilcoxon test. Multiple testing corrections have been performed with the Bonferroni method. For each of the three evaluated genes, the cutoff point for “high” and “low” expression levels was estimated by maximizing the Youden Index of the Receiver Operating Characteristic (ROC) curve. The ROC curve was plotted using as parameters the gene expression of each *ERCC* gene and the 12 months overall survival. This analysis has been performed with the ROCR package [[20](#ref-20)]. Kaplan-Meier curves have been computed with the package for survival analysis in R, illustrating the correlation between expression levels or SNP variants and OS or PFS. Linkage Disequilibrium (LD) analysis was performed and plotted using the Gaston package for R.

### Ethics approval

The present study was approved by the Regional Ethics Committee for Clinical Trials of the Tuscany Region (“QUASAR” protocol, code “19822_BIO”). All informed consent documents are in compliance with the International Conference on Harmonization (ICH) guideline on good clinical practice (GCP). The study protocol is performed in accordance with the principles of the Declaration of Helsinki and in compliance with GCP and the applicable laws and regulations. Each patient will be identified by a code instead of the patient’s name in order to protect the patient’s identity when reporting study-related data.

## Results

### Patient characteristics

Thirty-eight patients with advanced or relapsed SCLC who received CT as per clinical practice during the study period were included in the analysis. The clinical baseline characteristics of the enrolled patients are summarized in [Table 1](#table-1). The median age at the time of diagnosis was 68 years (range 41–81 years), 17 were males (44.74%) and 21 were females (55.26%). All patients had a history of tobacco exposure (n = 14 former and n = 24 current smokers). At enrolling time, the ECOG PS was 1 in half of the patients (n = 19, 50.00%), 0 in 11 patients (28.95%), and 2 in 8 patients (21.05%). All patients had ES-SCLC and received platinum-based CT as frontline therapy. Carboplatin plus etoposide was used as first-line treatment in 34 patients (89.47%), while 4 (10.53%) patients were treated with cisplatin plus etoposide. Most patients received 4 up to 6 cycles of CT (n = 26, 68.42%), while 12 patients (31.58%) received 3 or fewer cycles. The most frequent metastasis sites at diagnosis were respectively: liver (n = 18, 47.37%), bone (n = 18, 47.37%), adrenal glands (n = 15, 39.47%), and brain (n = 7, 18.42%).

### Table 1. Patients’ baseline characteristics.

| Patients (n = 38) |   |
| --- | --- |
| Age |   |
| Median (range)–years | 68 (41–81) |
| Sex–no. (%) |   |
| Male | 17 (44.74) |
| Female | 21 (55.26) |
| Smoking status–no. (%) |   |
| Never | 0 (0) |
| Current | 24 (63.16) |
| Former | 14 (36.84) |
| ECOG performance status–no. (%) |   |
| 0 | 11 (28.95) |
| 1 | 19 (50.00) |
| 2 | 8 (21.05) |
| Metastasis sites–no. (%) |   |
| Liver | 18 (47.37) |
| Bone | 18 (47.37) |
| Adrenal gland | 15 (39.47) |
| Brain | 7 (18.42) |
| Treatments |   |
| I line–no. (%) | 38 (100) |
| Carboplatin plus etoposide | 34 (89.47) |
| Cisplatin plus etoposide | 4 (10.53) |
| Number of cycles |   |
| ≤3 | 12 (31.58) |
| 4–6 | 26 (68.42) |
| II line–no. (%) | 8 (21.05) |
| III line–no. (%) | 3 (7.89) |
| Best response to treatment–no. (%) |   |
| Complete response | 1 (2.63) |
| Partial response | 21 (56.26) |
| Stable disease | 1 (2.63) |
| Progression disease | 15 (39.47) |
| Radiotherapy–no. (%) |   |
| Chest/mediastin | 7 (18.42) |
| Bone | 4 (10.53) |
| Brain | 4 (10.53) |
| Liver | 0 (0) |
| Adrenal gland | 0 (0) |
| PCI–no. (%) | 5 (13.16) |
In the overall population, 1 patient achieved CR (2.63%), 21 PR (56.26%), and 1 SD (2.63%), while 15 had PD as best response (39.47%). All patients experienced PD or death at the cut-off date for the follow-up analysis. Preferred sites of disease recurrence were the chest, liver, bone, brain, or adrenal glands. Eight patients (21.05%) received a second-line CT with topotecan (n = 1, 2.63%) or irinotecan (n = 4, 10.53%), or platinum plus etoposide (n = 3, 7.89%) or other (n = 1, 2.63%), while only 3 patients (7.89%) were fit for a third line. Fifteen patients (39.47%) underwent palliative-symptomatic radiotherapy to the chest, bone, or brain, while in 5 patients (13.16%), who achieved an excellent response rate to first-line CT, prophylactic cranial irradiation (PCI) was performed.

### Expression of NER pathway genes and their association with clinicopathological characteristics

The expression levels of the three genes of interest involved in the NER pathway (i.e., *ERCC1*, *ERCC2*, and *ERCC5*) were obtained from diagnostic FFPE tumor samples. Tumor samples showed higher levels of gene expression than healthy controls: in detail, the median expression values of the three genes in the study population compared to the control group were 3.41 (95% Confidence Interval [CI], 0.27–19.20) for *ERCC1*, 5.89 (95% CI, 0.19–18.84) for *ERCC2* and 9.24 (95% CI, 2.12–23.17) for *ERCC5*, respectively ([Fig. 1A](#fig-1)).

### Figure 1. (A): Relative mRNA expression values of the genes ERCC1, ERCC2 and ERCC5 in a cohort of 26 patients with extended stage-SCLC compared to the healthy group. Values were normalized to the control group, value = 1. (B–G): Association between mRNA expression levels of the genes and age (< or ≥70 years) (B) (p = 0.49 for ERCC1, p = 0.47 for ERCC2, p = 0.32 for ERCC5), sex (C) (p = 0.47 for ERCC1, p = 0.54 for ERCC2, p = 0.81 for ERCC5), response to therapy (D) (p = 0.40 for ERCC1, p = 0.49 for ERCC2, p = 0.60 for ERCC5) and site of metastasis at diagnosis: brain (E) (p = 0.85 for ERCC1, p = 0.70 for ERCC2, p = 0.77 for ERCC5), liver (F) (p = 0.57 for ERCC1, p = 015 for ERCC2, p = 0.71 for ERCC5) and bone (G) (p = 0.32 for ERCC1, p = 0.83 for ERCC2, p = 0.34 for ERCC5).

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7b49/11671403/59247c04ee00/OncolRes-33-50161-f001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=11671403_OncolRes-33-50161-f001.jpg)

### SNPs analysis

We then genotyped five polymorphic alterations in a panel of SNPs belonging to relevant genes of the NER pathway: rs11615 for *ERCC1*, rs13181 and rs1799793 for *ERCC2*, rs2296147 and rs1047768 for *ERCC5*. Data was available for all patients enrolled in the study (n = 38). Genotyping of rs11615 in the *ERCC1* gene indicated that the most common genotype was A/G (42%; 16/38), while 53% (20/38) and 58% (22/38) of patients showed T/G genotype in rs13181 and C/T genotype of rs1799793 in the *ERCC2* gene, and 50% (19/38) and 47% (18/38) of patients showed T/C genotype in rs2296147 and T/C or CC genotype of rs1047768 in the *ERCC5* gene (Frequencies of genotypes are shown in Table S1). Linkage disequilibrium (LD) coefficient r2 was calculated to evaluate the non-random association of *ERCC* SNPs alleles at different loci. A moderate LD was observed between; i) *ERCC2* SNPs rs13181 and re1799793; ii) *ERCC5* SNPs rs1047768 and rs2296147; iii) *ERCC1* SNP rs11615 and *ERCC2* SNP rs1799793 (Fig. S1).

As far as the correlation between SNPs variants and clinicopathological characteristics of patients, our results showed no significant association between SNPs and patient features (data not shown).

In the 26 patients for whom mRNA expression values were available, we assessed whether they correlated with the main clinicopathological features of the study population, such as age (< or ≥70 years), sex, site of metastasis, and response to therapy (responder *vs*. no responder). No statistically significant association was found between gene expression levels and age, sex, or response to treatment ([Fig. 1B](#fig-1)–[D](#fig-1)). With regard to the association with the site of metastasis, results showed no statistical correlation between gene expression and different sites of metastasis ([Fig. 1E](#fig-1)–[G](#fig-1)).

### Association between SNPs variants and gene expression

We next analyzed the correlation between SNPs variants of the three genes and their expression: patients harboring mutant homozygous variants of the *ERCC1* rs11615 and of the two SNPs of *ERCC2* (rs13181 and rs1799793) had significantly higher expression of the corresponding gene ([Fig. 2A](#fig-2)). Of note, patients with the mutant homozygous variant of the *ERCC1* SNP also had higher expressions of the other two genes, *ERCC2* and *ERCC5*. Similarly, patients harboring the mutant homozygous variants of the two SNPs of *ERCC2* showed increased gene expression of *ERCC1* and *ERCC5*. However, the mutant homozygous variants of the two SNPs of *ERCC5* (rs1047768 and rs2296147) were associated with a different gene expression pattern ([Fig. 2A](#fig-2)). We then assessed whether alteration of the NER pathway by the presence of variants in the SNPs was associated with different expression levels of the three genes. In detail, patients were considered deficient in the NER pathway if at least one of the three genes harbored a mutant homozygous variant in one of the five SNPs. Consistently with previous results, a positive association was found between NER pathway deficiency and higher gene expression levels of *ERCC1* (*p* = 0.069), *ERCC2* (*p* = 0.26) and *ERCC5* (*p* = 0.048) ([Fig. 2B](#fig-2)).

### Figure 2. (A) Correlation between SNPs variants of the three genes and their expression. Five SNPs of the genes of interest were analyzed: rs11615 for ERCC1 (top row on the left: 2A1), rs13181 and rs1799793 for ERCC2 (second and third rows on the left: 2A2 and 2A3), rs2296147 and rs1047768 for ERCC5 (first and second rows on the right: 2A4 and 2A5). Mutant homozygous variants of ERCC1 and ERCC2 SNPs (right of each boxplot) were statistically associated with increased expression of all three genes compared to wild-type homozygous and alternative heterozygous variants (left of each boxplot). Different gene expression patterns were found for variants of the two ERCC5 SNPs. The data refer to the 26 patients for whom the mRNA expression levels of each gene were available. (B) Association between NER pathway deficiency and gene expression. Deficiency of the NER pathway correlates with higher gene expression levels of ERCC1, ERCC2 and ERCC5 (NER pathway was considered deficient [“altered”] if at least one of the three genes had a mutant homozygous variant in one of the five SNPs of interest). Note: The corresponding p-values are indicated in each figure for ease of reference.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7b49/11671403/d43c0a59f2c9/OncolRes-33-50161-f002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=11671403_OncolRes-33-50161-f002.jpg)

### Survival outcomes analysis

In the overall population, our results showed a median PFS (mPFS) of 4.5 months (95% CI, 4.0–7.2) and a mOS of 5.7 months (95% CI, 4.4–7.2). For the analysis of survival outcomes, the study population was divided into high and low expression groups for each of the three genes, based on the cutoff point estimated by ROC curve analysis. Patients with low *ERCC1* and *ERCC5* expression showed a longer PFS compared to patients with high expression (mPFS = 7.1 months (95% CI, 4.8–NR) *vs*. 4.9 months (95% CI, 2.5–7.2) *p* = 0.39, for *ERCC1* and mPFS = 6.9 months (95% CI, 4.0–NR) *vs*. 4.8 months (95% CI, 2.5–NR) *p* = 0.093, for *ERCC5*) ([Fig. 3A](#fig-3),[C](#fig-3)).

### Figure 3. Progression free survival (PFS) and overall survival (OS) in the low- vs. high-expression groups for each of the three genes: ERCC1 (A and D), ERCC2 (B and E) and ERCC5 (C and F). The data refer to the 26 patients for whom the mRNA expression levels of each gene were available.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7b49/11671403/51a3072ad51e/OncolRes-33-50161-f003.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=11671403_OncolRes-33-50161-f003.jpg)

Conversely, patients with low *ERCC2* expression had lower PFS than patients with high expression (mPFS = 2.2 months (95% CI, 2.2–NR) *vs*. 5.1 months (95% CI, 4.4–7.2) in low and high expression groups, respectively, *p* = 0.38) ([Fig. 3B](#fig-3)). Consistently, a trend for a better OS was reported in the low expression groups for *ERCC1* (mOS = 8.7 months (95% CI, 5.6–NR) *vs*. 6.0 months (95% CI, 4.0–8.9), *p* = 0.4) and for *ERCC5* (mOS = 7.2 months (95% CI, 4.9–NR) *vs*. 6.2 months (95% CI, 2.5–NR), *p* = 0.13) and in the high expression group for *ERCC2* (mOS = 6.2 months (95% CI, 4.9–8.9) *vs*. 2.9 months (95% CI, 2.9–NR), *p* = 0.35) ([Fig. 3D](#fig-3)–[F](#fig-3)).

Survival outcomes analysis was also performed according to SNPs variants in the entire study population. Overall, better PFS and OS were reported in patients with wild-type or heterozygous genotypes (NER pathway “proficient”) than in patients with homozygous mutant variants (NER pathway “deficient”): mPFS = 5.5 months (95% CI, 4.0–NR) *vs*. 4.9 months (95% CI, 2.5–7.2) (*p* = 0.26) and mOS = 7.0 months (95% CI, 4.6–NR) *vs*. 5.6 months (95% CI, 2.9–7.2) (*p* = 0.15), respectively ([Fig. 4A](#fig-4),[B](#fig-4)). Accordingly, a non-significant trend for a better PFS and OS was reported for patients with a wild-type genotype or heterozygous variant than for patients with a homozygous mutant variant for all SNPs examined (Table S2).

### Figure 4. Progression free survival (PFS) (A) and overall survival (OS) (B) in patients with wild-type or heterozygous genotypes (NER pathway “proficient”) and in patients with homozygous mutant variant in at least one of the five SNPs (NER pathway “deficient”).

![Figure 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7b49/11671403/78633c23ab46/OncolRes-33-50161-f004.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=11671403_OncolRes-33-50161-f004.jpg)

Finally, as *ERCC1* and *ERCC2* genes are both located in chromosome 19 and the linkage disequilibrium analysis showed a moderate association of their SNPs alleles, we assessed the survival outcomes of patients based on gene expression together with SNPs variants of the two genes. As shown in [Fig. 5](#fig-5), patients with low *ERCC1* or *ERCC2* expression showed a statistically significantly better OS than patients with high gene expression (mOS = 10.6 months [95% CI, 5.6–NR] *vs*. 5.6 months (95% CI, 4.0–7.2), *p* = 0.042) ([Fig. 5C](#fig-5)). Similarly, patients with wild-type or heterozygous SNPs variants in *ERCC1* or *ERCC2* SNPs had a mOS of 6.3 months (95% CI, 4.9–8.9), compared to mOS of 4.4 months (95% CI, 2.4–NR) in patients with at least one mutant homozygous variant (*p* = 0.052) ([Fig. 5D](#fig-5)). A similar trend, although not statistically significant, was also reported for PFS in these two patient subgroups ([Fig. 5A](#fig-5),[B](#fig-5)).

### Figure 5. Progression Free Survival (PFS) and Overall Survival (OS) according to ERCC1 and ERCC2 status. PFS and OS was assessed in patients with low ERCC1 or ERCC2 expression compared to high-expression subgroup (A and B) and in patients with at least one mutant homozygous variant in ERCC1 or ERCC2 SNPs vs. patients with wild-type or heterozygous SNPs variant (C and D).

![Figure 5](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7b49/11671403/790b0f090713/OncolRes-33-50161-f005.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=11671403_OncolRes-33-50161-f005.jpg)

## Discussion

The NER pathway is one of the major repair mechanisms for platinum-DNA adducts and cross-links, playing a key role in the detection and repair process of the DNA damage exerted by the platinum compounds, particularly cisplatin and carboplatin [[6](#ref-6),[7](#ref-7)]. Several studies have described those genetic alterations in the SNPs of NER pathway genes and deregulation of NER pathway proteins may affect the DNA repair functions, influence the efficacy of platinum-CT, and, eventually, impact on clinical outcomes of patients with lung cancer [[10](#ref-10),[21](#ref-21)].

In our study we investigated a homogeneous cohort of 38 patients with ED-SCLC, assessing the role of three key genes encoding crucial factors of the NER pathway, *ERCC1*, *ERCC2*, and *ERCC5*. Our results showed a higher expression of these genes in SCLC patients compared to the healthy controls; of note, patients with low *ERCC1* and *ERCC5* expression had better mPFS and mOS, while an inverse trend emerged in survival outcomes for *ERCC2* expression. As SNPs can influence gene functions and phenotype, potentially leading to increased mRNA expression [[22](#ref-22)], genotyping of *ERCC* genes was conducted to investigate how various *ERCC* SNP profiles might correlate with altered expression levels. Specifically, the analysis examined five polymorphic alterations within a panel of clinically relevant SNPs associated with these genes. The results revealed that patients with homozygous mutant genotypes in the *ERCC1* rs11615 SNP and both the *ERCC2* rs13181 and rs1799793 SNPs exhibited heightened expression of all three genes. Conversely, a distinct gene expression pattern emerged for the genetic variants of the two *ERCC5* SNPs (rs1047768 and rs2296147). These findings are consistent with similar results previously reported by our group [[23](#ref-23)]. Finally, we investigated how genetic alterations in these SNPs may influence survival outcomes of SCLC patients. Overall, our data showed a longer PFS and OS, although not significant, in patients harboring the wild-type genotype or the heterozygous variant of the *ERCC1* rs11615 SNP and of the rs13181 and rs1799793 *ERCC2* SNPs compared to patients with the corresponding homozygous mutant genotype. On the other hand, the two polymorphisms of *ERCC5* showed no association with patients’ survival.

The higher gene expression of *ERCC1*, *ERCC2*, and *ERCC5* in tumor samples compared to healthy control is consistent with previous studies that reported high tumor tissue levels of NER pathways genes, mainly *ERCC1*, in SCLC and in other solid tumors [[14](#ref-14),[15](#ref-15),[24](#ref-24),[25](#ref-25)]. The high expression of NER pathway genes could lead to an increased ability of tumor cells to repair the DNA damage exerted by platinum-CT, conferring resistance to anti-cancer treatment. This may explain why, despite initial response to therapy, almost all ED-SCLC patients eventually experience disease progression and often early disease relapse. Consistent with this observation, *ERCC1* has been extensively investigated as a predictive biomarker for the efficacy of platinum agents and as a negative prognostic factor in various malignancies, including NSCLC and SCLC. Several studies have shown a significant correlation between low *ERCC1* expression and both higher response rates to CT and better clinical outcomes in cancer patients [[11](#ref-11),[26](#ref-26)–[29](#ref-29)]. As far as SCLC, protein or mRNA expression of *ERCC1* was reported to be predictive of treatment efficacy and a prognostic factor for survival [[14](#ref-14),[30](#ref-30)–[32](#ref-32)]. Moreover, in a large retrospective study of 184 SCLC patients, the low expression of *ERCC1*, as part of a favorable expression signature, was significantly correlated with better PFS and OS in both limited disease (LD) and ED-SCLC [[16](#ref-16)]. Interestingly, Lee et al. also reported that expression of *ERCC1* in SCLC is lower compared to studies of tissue from NSCLC, suggesting that the greater biological aggressiveness of SCLC could be related to the loss of *ERCC1* gene, which leads to impaired DNA-repair functions of the NER pathway [[33](#ref-33)]. According to previous data, our findings showed that patients with low *ERCC1* expression had longer PFS and OS. Similarly, better outcomes were also found in patients with low *ERCC5*, for whom far less data is available to date [[34](#ref-34),[35](#ref-35)]. Of note, Simon and colleagues reported that increased expression of *ERCC1* is an independent predictor of improved survival in resected patients with NSCLC and that this may be secondary to a decreased accumulation of genomic aberrations as a result of efficient DNA-damage repair system [[36](#ref-36)]. The favorable prognostic value of *ERCC1* expression in resected-NSCLC patients was also confirmed by Olaussen et al. [[37](#ref-37)]. Consequently, upregulation of *ERCC1* on the one hand seems to be a negative prognostic factor in advanced disease, as it reduces the benefit of platinum-CT, while on the other hand may act as a predictor of better survival in limited disease, as it reduces the risk of relapse after definitive treatment. Interestingly, it was reported that the *ERCC1* gene generates different isoforms by alternative splicing and that only one isoform is implicated in the repair of platinum-DNA adducts [[38](#ref-38)]. Consequently, the expression of different isoforms could be a major concern as it would lead to a tumor being considered *ERCC1*-positive, although the expressed protein might be non-functional.

In contrast with *ERCC1* and *ERCC5*, our results showed that patients with low *ERCC2* expression had an opposite survival trend with shorter PFS and OS. Although this finding should be taken with caution given the small size of *ERCC2*-low subgroup in our population, some previous data have reported a correlation between *ERCC2* upregulation with a more aggressive cancer phenotype in head and neck tumors and in NSCLC cell lines, suggesting an inter-tissue variation in NER genes and chemoresistance [[39](#ref-39)–[41](#ref-41)].

The genotyping analysis of SNPs on NER genes in our cohorts showed that the allele frequencies of the three genes of interest differed between SCLC patients and the general population, particularly for the SNP of *ERCC5* rs2296147 (*p* < 0.0001), suggesting a crucial role of the NER pathway in these patients. As wild-type and variant genotypes of the NER pathway are likely associated with differential activity of DNA repair functions [[42](#ref-42)], polymorphic alterations in these genes may influence the variability of DNA damage repair activity. Consequently, we assessed whether specific genotypes of these genes may also impact the clinical outcomes of SCLC patients. Our data showed that homozygous mutant genotypes in the SNPs of *ERCC1* and *ERCC2* are associated with decreased PFS and OS compared to wild-type or heterozygous variant genotypes. To date, very few studies have investigated the role of genetic polymorphisms in the NER pathway for SCLC. Nicos and colleagues evaluated the genotypes of the *ERCC1* in a cohort of SCLC patients, reporting that patients harboring the heterozygous genotype in rs11615 had a significantly shorter OS compared to the wild-type genotype, which instead was a favorable prognostic factor. Moreover, the different genotypes in two SNPs of the *ERCC1* showed also a correlation with the hematological toxicity of the treatment [[17](#ref-17)]. Conversely much more data about the role of gene polymorphisms in the NER pathway are available for NSCLC, where they have been extensively investigated, sometimes with contradictory results [[10](#ref-10)]. The SNP profiling of the *ERCC1* mostly indicated clinical relevance for response to platinum-based CT and association with survival [[12](#ref-12),[43](#ref-43)–[46](#ref-46)] but its association with clinical outcomes remains unclear. Several studies also evaluated the role of *ERCC2* SNPs rs13181 and rs1799793 in the prognosis and survival outcomes of advanced NSCLC. Generally, and in accordance with our results, variant genotypes were reported to be associated with decreased OS in these patients [[13](#ref-13),[44](#ref-44),[47](#ref-47)–[49](#ref-49)]. Interestingly, two meta-analyses reported that SNPs in both *ERCC1* and *ERCC2* genes may play a significant role in lung cancer risk. In detail, Xu et al. indicated that individuals carrying at least one wild-type allele in *ERCC1* rs11615 have a reduced risk of lung cancer development [[12](#ref-12)], while a large meta-analysis by Zhan et al., including 22 studies about *ERCC2* rs13181 and rs1799793 polymorphisms suggested that mutant genotypes of these SNPs are correlated with lung cancer development [[50](#ref-50)]. Taken together, these data highlight the dual role of DNA repair genes, both in carcinogenesis and in the restoration of platinum-induced DNA damage. For that reason, DNA repair systems have been described as a double-edged sword [[7](#ref-7),[51](#ref-51)] because, on the one hand, a deficiency in DNA repair functions may increase cancer susceptibility, while, on the other hand, it may improve survival in patients already diagnosed with cancer, when treated with platinum agents.

A novelty of the present study is that the integrated analysis of the expression of *ERCC1* and *ERCC2* and their SNPs variants was able to confirm a trend for a longer PFS and a significantly better OS for patients with a favorable signature (low gene expression and no homozygous mutant variants) than those with unfavorable signature (high gene expression or presence of homozygous mutant variants of the SNPs). This finding supports the prognostic role of the NER pathway genes and suggests that an integrated analysis of both mRNA expression and gene polymorphisms in the major components of the NER pathway may identify SCLC patients with different survival outcomes.

Limitations of the present study include its retrospective design and the relatively small sample size that greatly weakens the statistical power of survival analysis. Furthermore, it is known that the DNA repair process includes a complex set of different mechanisms, and thus several genetic polymorphisms in other DNA repair pathways could influence the clinical outcome of SCLC. Finally, our study population received platinum-based CT, as combination treatment with ICI had not yet been approved for patients with ED-SCLC at the time of the design of the study. However, we consider that our research may help to identify a subgroup of patients who achieve substantial benefit from standard CT and with a better prognosis. Therefore, future large sample and prospective studies are warranted to validate the role of expression and polymorphisms in NER pathway genes on the prognosis of SCLC patients.

Finally, in the new era of chemo-immunotherapy and in a future therapeutic landscape in which new targeted therapies may also play a role in the therapeutic strategy of SCLC, the role of platinum-based CT will probably still remain relevant in maximizing the therapeutic benefit for these patients. Thus, future prospective trials may further investigate NER pathway genes and their role as prognostic factors for SCLC in a new and integrated therapeutic scenario.

## Supplementary Materials

## Acknowledgments

Not applicable.

## Abbreviations

## Funding Statement

This research did not receive any specific grant from funding agencies in the public, commercial, or not-for-profit sectors.

## Author Contributions

Enrico Caliman, Serena Pillozzi: conceptualization; formal analysis; interpretation of data. Enrico Caliman, Clara Manneschi: writing-original draft preparation. FS: software; methodology. Adriano Pasqui, Sara Fancelli, Daniele Lavacchi; Francesca Mazzoni: resources; data curation; investigation. Sara Fancelli, Daniele Lavacchi, Francesca Mazzoni, Francesca Gensini, Valeria Pasini: writing-review and editing. Francesca Gensini, Camilla Eva Comin, Luca Voltolini, Serena Pillozzi, Lorenzo Antonuzzo: writing-review and editing; supervision. Lorenzo Antonuzzo: project administration. All authors reviewed the results and approved the final version of the manuscript.

## Availability of Data and Materials

The datasets generated during and/or analyzed during the current study are available from the corresponding author upon reasonable request.

## Ethics Approval

The present study was approved by the Regional Ethics Committee for Clinical Trials of the Tuscany Region (“QUASAR” protocol, code “19822_BIO”). All informed consent documents are in compliance with the International Conference on Harmonization (ICH) guideline on good clinical practice (GCP). The study protocol is performed in accordance with the principles of the Declaration of Helsinki and in compliance with GCP and the applicable laws and regulations. Each patient will be identified by a code instead of the patient’s name in order to protect the patient’s identity when reporting study-related data.

## Conflicts of Interest

The authors declare no conflicts of interest to report regarding the present study.

## Supplementary Materials

The supplementary material is available online at [https://doi.org/10.32604/or.2024.050161](https://doi.org/10.32604/or.2024.050161).

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

The datasets generated during and/or analyzed during the current study are available from the corresponding author upon reasonable request.

### Supplementary Materials

### Data Availability Statement

The datasets generated during and/or analyzed during the current study are available from the corresponding author upon reasonable request.

## References

1. Rudin, C. M., Brambilla, E., Faivre-Finn, C., Sage, J. (2021). Small-cell lung cancer. Nature Reviews Disease Primers , 7(1), 3. 10.1038/s41572-020-00235-0;  [DOI](https://doi.org/10.1038/s41572-020-00235-0) | [PMC free article](/articles/PMC8177722/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33446664/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature%20Reviews%20Disease%20Primers&title=Small-cell%20lung%20cancer&volume=7&issue=1&publication_year=2021&pages=3&pmid=33446664&doi=10.1038/s41572-020-00235-0&)

2. Horn, L., Mansfield, A. S., Szczęsna, A., Havel, L., Krzakowski, M.et al. (2018). First-line atezolizumab plus chemotherapy in extensive-stage small-cell lung cancer. The New England Journal of Medicine , 379(23), 2220–2229. 10.1056/NEJMoa1809064;  [DOI](https://doi.org/10.1056/NEJMoa1809064) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30280641/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=The%20New%20England%20Journal%20of%20Medicine&title=First-line%20atezolizumab%20plus%20chemotherapy%20in%20extensive-stage%20small-cell%20lung%20cancer&volume=379&issue=23&publication_year=2018&pages=2220-2229&pmid=30280641&doi=10.1056/NEJMoa1809064&)

3. Paz-Ares, L., Dvorkin, M., Chen, Y., Reinmuth, N., Hotta, K.et al. (2019). Durvalumab plus platinum-etoposide versus platinum-etoposide in first-line treatment of extensive-stage small-cell lung cancer (CASPIAN): A randomised, controlled, open-label, phase 3 trial. The Lancet , 394(10212), 1929–1939. 10.1016/S0140-6736(19)32222-6;  [DOI](https://doi.org/10.1016/S0140-6736(19)32222-6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31590988/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=The%20Lancet&title=Durvalumab%20plus%20platinum-etoposide%20versus%20platinum-etoposide%20in%20first-line%20treatment%20of%20extensive-stage%20small-cell%20lung%20cancer%20(CASPIAN):%20A%20randomised,%20controlled,%20open-label,%20phase%203%20trial&volume=394&issue=10212&publication_year=2019&pages=1929-1939&pmid=31590988&doi=10.1016/S0140-6736(19)32222-6&)

4. Caliman, E., Fancelli, S., Petroni, G., Gatta Michelet, M. R., Cosso, F.et al. (2022). Challenges in the treatment of small cell lung cancer in the era of immunotherapy and molecular classification. Lung Cancer , 175, 88–100;  [DOI](https://doi.org/10.1016/j.lungcan.2022.11.014) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36493578/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lung%20Cancer&title=Challenges%20in%20the%20treatment%20of%20small%20cell%20lung%20cancer%20in%20the%20era%20of%20immunotherapy%20and%20molecular%20classification&volume=175&publication_year=2022&pages=88-100&pmid=36493578&doi=10.1016/j.lungcan.2022.11.014&)

5. Kartalou, M., Essigmann, J. M. (2001). Recognition of cisplatin adducts by cellular proteins. Mutation Research , 478(1–2), 1–21;  [DOI](https://doi.org/10.1016/s0027-5107(01)00142-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11406166/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mutation%20Research&title=Recognition%20of%20cisplatin%20adducts%20by%20cellular%20proteins&volume=478&issue=1%E2%80%932&publication_year=2001&pages=1-21&pmid=11406166&doi=10.1016/s0027-5107(01)00142-7&)

6. García-Campelo, R., Alonso-Curbera, G., Antón Aparicio, L. M., Rosell, R. (2005). Pharmacogenomics in lung cancer: An analysis of DNA repair gene expression in patients treated with platinum-based chemotherapy. Expert Opinion on Pharmacotherapy , 6(12), 2015–2026. 10.1517/14656566.6.12.2015;  [DOI](https://doi.org/10.1517/14656566.6.12.2015) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16197356/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Expert%20Opinion%20on%20Pharmacotherapy&title=Pharmacogenomics%20in%20lung%20cancer:%20An%20analysis%20of%20DNA%20repair%20gene%20expression%20in%20patients%20treated%20with%20platinum-based%20chemotherapy&volume=6&issue=12&publication_year=2005&pages=2015-2026&pmid=16197356&doi=10.1517/14656566.6.12.2015&)

7. Rosell, R., Lord, R. V. N., Taron, M., Reguart, N. (2002). DNA repair and cisplatin resistance in non-small-cell lung cancer. Lung Cancer , 38(3), 217–227. 10.1016/S0169-5002(02)00224-6;  [DOI](https://doi.org/10.1016/S0169-5002(02)00224-6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12445742/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lung%20Cancer&title=DNA%20repair%20and%20cisplatin%20resistance%20in%20non-small-cell%20lung%20cancer&volume=38&issue=3&publication_year=2002&pages=217-227&pmid=12445742&doi=10.1016/S0169-5002(02)00224-6&)

8. Petruseva, I. O., Evdokimov, A. N., Lavrik, O. I. (2014). Molecular mechanism of global genome nucleotide excision repair. Acta Naturae , 6(20), 23–34;  [PMC free article](/articles/PMC3999463/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24772324/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Acta%20Naturae&title=Molecular%20mechanism%20of%20global%20genome%20nucleotide%20excision%20repair&volume=6&issue=20&publication_year=2014&pages=23-34&pmid=24772324&)

9. Spivak, G. (2015). Nucleotide excision repair in humans. DNA Repair , 36, 13–18. 10.1016/j.dnarep.2015.09.003;  [DOI](https://doi.org/10.1016/j.dnarep.2015.09.003) | [PMC free article](/articles/PMC4688078/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26388429/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=DNA%20Repair&title=Nucleotide%20excision%20repair%20in%20humans&volume=36&publication_year=2015&pages=13-18&pmid=26388429&doi=10.1016/j.dnarep.2015.09.003&)

10. Pérez-Ramírez, C., Cañadas-Garre, M., Molina, M.Á., Robles, A. I., Faus-Dáder, M. J.et al. (2017). Contribution of genetic factors to platinum-based chemotherapy sensitivity and prognosis of non-small cell lung cancer. Mutation Research. Reviews in Mutation Research , 771, 32–58. 10.1016/j.mrrev.2016.11.003;  [DOI](https://doi.org/10.1016/j.mrrev.2016.11.003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28342452/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mutation%20Research.%20Reviews%20in%20Mutation%20Research&title=Contribution%20of%20genetic%20factors%20to%20platinum-based%20chemotherapy%20sensitivity%20and%20prognosis%20of%20non-small%20cell%20lung%20cancer&volume=771&publication_year=2017&pages=32-58&pmid=28342452&doi=10.1016/j.mrrev.2016.11.003&)

11. Hubner, R. A., Riley, R. D., Billingham, L. J., Popat, S. (2011). Excision repair cross-complementation group 1 (ERCC1) status and lung cancer outcomes: A meta-analysis of published studies and recommendations. PLoS One , 6(10), 1–10. 10.1371/journal.pone.0025164;  [DOI](https://doi.org/10.1371/journal.pone.0025164) | [PMC free article](/articles/PMC3194810/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22022380/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Excision%20repair%20cross-complementation%20group%201%20(ERCC1)%20status%20and%20lung%20cancer%20outcomes:%20A%20meta-analysis%20of%20published%20studies%20and%20recommendations&volume=6&issue=10&publication_year=2011&pages=1-10&pmid=22022380&doi=10.1371/journal.pone.0025164&)

12. Xu, T. P., Shen, H., Liu, L. X., Shu, Y. Q. (2013). Association of ERCC1-C118T and -C8092A polymorphisms with lung cancer risk and survival of advanced-stage non-small cell lung cancer patients receiving platinum-based chemotherapy: A pooled analysis based on 39 reports. Gene , 526(2), 265–274. 10.1016/j.gene.2013.05.021;  [DOI](https://doi.org/10.1016/j.gene.2013.05.021) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23727606/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gene&title=Association%20of%20ERCC1-C118T%20and%20-C8092A%20polymorphisms%20with%20lung%20cancer%20risk%20and%20survival%20of%20advanced-stage%20non-small%20cell%20lung%20cancer%20patients%20receiving%20platinum-based%20chemotherapy:%20A%20pooled%20analysis%20based%20on%2039%20reports&volume=526&issue=2&publication_year=2013&pages=265-274&pmid=23727606&doi=10.1016/j.gene.2013.05.021&)

13. Lan, X., Li, Y., Wu, Y., Li, X., Xu, L. (2022). The association of ERCC1 and ERCC5 polymorphisms with lung cancer risk in han Chinese. Journal of Cancer , 13(2), 517–526. 10.7150/jca.59277;  [DOI](https://doi.org/10.7150/jca.59277) | [PMC free article](/articles/PMC8771503/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35069899/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Cancer&title=The%20association%20of%20ERCC1%20and%20ERCC5%20polymorphisms%20with%20lung%20cancer%20risk%20in%20han%20Chinese&volume=13&issue=2&publication_year=2022&pages=517-526&pmid=35069899&doi=10.7150/jca.59277&)

14. Sereno Moyano, M., Cejas, P., Moreno, V., Belda-Iniesta, C., López, R.et al. (2012). ERCC1 and topoisomerase I expression in small cell lung cancer: Prognostic and predictive implications. International Journal of Oncology , 40(6), 2104–2110. 10.3892/ijo.2012.1378;  [DOI](https://doi.org/10.3892/ijo.2012.1378) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22344449/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=International%20Journal%20of%20Oncology&title=ERCC1%20and%20topoisomerase%20I%20expression%20in%20small%20cell%20lung%20cancer:%20Prognostic%20and%20predictive%20implications&volume=40&issue=6&publication_year=2012&pages=2104-2110&pmid=22344449&doi=10.3892/ijo.2012.1378&)

15. Sodja, E., Knez, L., Kern, I., Ovčariček, T., Sadikov, A.et al. (2012). Impact of ERCC1 expression on treatment outcome in small-cell lung cancer patients treated with platinum-based chemotherapy. European Journal of Cancer , 48(18), 3378–3385. 10.1016/j.ejca.2012.06.011;  [DOI](https://doi.org/10.1016/j.ejca.2012.06.011) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22795264/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=European%20Journal%20of%20Cancer&title=Impact%20of%20ERCC1%20expression%20on%20treatment%20outcome%20in%20small-cell%20lung%20cancer%20patients%20treated%20with%20platinum-based%20chemotherapy&volume=48&issue=18&publication_year=2012&pages=3378-3385&pmid=22795264&doi=10.1016/j.ejca.2012.06.011&)

16. Karachaliou, N., Papadaki, C., Lagoudaki, E., Trypaki, M., Sfakianaki, M.et al. (2013). Predictive value of BRCA1, ERCC1, ATP7B, PKM2, TOPOI, TOPΟ-IIA, TOPOIIB and C-MYC genes in patients with small cell lung cancer (SCLC) who received first line therapy with cisplatin and etoposide. PLoS One , 8(9), 1–9.  [DOI](https://doi.org/10.1371/journal.pone.0074611) | [PMC free article](/articles/PMC3772910/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24058603/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Predictive%20value%20of%20BRCA1,%20ERCC1,%20ATP7B,%20PKM2,%20TOPOI,%20TOP%CE%9F-IIA,%20TOPOIIB%20and%20C-MYC%20genes%20in%20patients%20with%20small%20cell%20lung%20cancer%20(SCLC)%20who%20received%20first%20line%20therapy%20with%20cisplatin%20and%20etoposide&volume=8&issue=9&publication_year=2013&pages=1-9&pmid=24058603&doi=10.1371/journal.pone.0074611&)

17. Nicoś, M., Rolska-Kopińska, A., Krawczyk, P., Grenda, A., Bozyk, A.et al. (2021). Effect of TOP2A and ERCC1 gene polymorphisms on the efficacy and toxicity of cisplatin and etoposide-based chemotherapy in small cell lung cancer patients. Archives of Medical Science , 17(2), 474–480. 10.5114/aoms.2020.92572;  [DOI](https://doi.org/10.5114/aoms.2020.92572) | [PMC free article](/articles/PMC7959040/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33747282/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Archives%20of%20Medical%20Science&title=Effect%20of%20TOP2A%20and%20ERCC1%20gene%20polymorphisms%20on%20the%20efficacy%20and%20toxicity%20of%20cisplatin%20and%20etoposide-based%20chemotherapy%20in%20small%20cell%20lung%20cancer%20patients&volume=17&issue=2&publication_year=2021&pages=474-480&pmid=33747282&doi=10.5114/aoms.2020.92572&)

18. R Core Team (2021). R: a language and environment for statistical computing. Vienna, Austria. R Foundation for Statistical Computing. https://www.r-project.org/ (accessed on 23/04/2024).  [https://www.r-project.org/](https://www.r-project.org/) | [Google Scholar](https://scholar.google.com/scholar_lookup?title=R:%20a%20language%20and%20environment%20for%20statistical%20computing&publication_year=2021&)

19. Livak, K. J., Schmittgen, T. D. (2001). Analysis of relative gene expression data using real-time quantitative PCR and the 2–ΔΔCT method. Methods , 25(4), 402–408. 10.1006/meth.2001.1262;  [DOI](https://doi.org/10.1006/meth.2001.1262) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11846609/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Methods&title=Analysis%20of%20relative%20gene%20expression%20data%20using%20real-time%20quantitative%20PCR%20and%20the%202%E2%80%93%CE%94%CE%94CT%20method&volume=25&issue=4&publication_year=2001&pages=402-408&pmid=11846609&doi=10.1006/meth.2001.1262&)

20. Sing, T., Sander, O., Beerenwinkel, N., Lengauer, T. (2005). ROCR: Visualizing classifier performance in R. Bioinformatics , 21(20), 3940–3941. 10.1093/bioinformatics/bti623;  [DOI](https://doi.org/10.1093/bioinformatics/bti623) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16096348/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=ROCR:%20Visualizing%20classifier%20performance%20in%20R&volume=21&issue=20&publication_year=2005&pages=3940-3941&pmid=16096348&doi=10.1093/bioinformatics/bti623&)

21. Olaussen, K. A., Postel-Vinay, S. (2016). Predictors of chemotherapy efficacy in non-small-cell lung cancer: A challenging landscape. Annals of Oncology , 27(11), 2004–2016. 10.1093/annonc/mdw321;  [DOI](https://doi.org/10.1093/annonc/mdw321) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27502726/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annals%20of%20Oncology&title=Predictors%20of%20chemotherapy%20efficacy%20in%20non-small-cell%20lung%20cancer:%20A%20challenging%20landscape&volume=27&issue=11&publication_year=2016&pages=2004-2016&pmid=27502726&doi=10.1093/annonc/mdw321&)

22. Shastry, B. S. (2009). SNPs: Impact on gene function and phenotype. Methods in Molecular Biology , 578, 3–22. 10.1007/978-1-60327-411-1  [DOI](https://doi.org/10.1007/978-1-60327-411-1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19768584/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Methods%20in%20Molecular%20Biology&title=SNPs:%20Impact%20on%20gene%20function%20and%20phenotype&volume=578&publication_year=2009&pages=3-22&pmid=19768584&doi=10.1007/978-1-60327-411-1&)

23. Pasqui, A., Boddi, A., Campanacci, D. A., Scoccianti, G., Grasso, D.et al. (2022). Alteration of the nucleotide excision repair (NER) pathway in soft tissue sarcoma. International Journal of Molecular Sciences , 23(15), 8360. 10.3390/ijms23158360;  [DOI](https://doi.org/10.3390/ijms23158360) | [PMC free article](/articles/PMC9369086/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35955506/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=International%20Journal%20of%20Molecular%20Sciences&title=Alteration%20of%20the%20nucleotide%20excision%20repair%20(NER)%20pathway%20in%20soft%20tissue%20sarcoma&volume=23&issue=15&publication_year=2022&pages=8360&pmid=35955506&doi=10.3390/ijms23158360&)

24. Suh, D. H., Park, W. H., Kim, M., Kim, K., No, J. H.et al. (2023). HOXB9 overexpression confers chemoresistance to ovarian cancer cells by inducing ERCC-1, MRP-2, and XIAP. International Journal of Molecular Sciences , 24(2), 1249. 10.3390/ijms24021249;  [DOI](https://doi.org/10.3390/ijms24021249) | [PMC free article](/articles/PMC9865712/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36674764/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=International%20Journal%20of%20Molecular%20Sciences&title=HOXB9%20overexpression%20confers%20chemoresistance%20to%20ovarian%20cancer%20cells%20by%20inducing%20ERCC-1,%20MRP-2,%20and%20XIAP&volume=24&issue=2&publication_year=2023&pages=1249&pmid=36674764&doi=10.3390/ijms24021249&)

25. Das., A. P., Saini, S., Tyagi, S., Chaudhary, N., Agarwal, S. M. (2023). Elucidation of increased cervical cancer risk due to polymorphisms in XRCC1 (R399Q and R194W), ERCC5 (D1104H), and NQO1 (P187S). Reproductive Sciences , 30(4), 1118–1132. 10.1007/s43032-022-01096-6;  [DOI](https://doi.org/10.1007/s43032-022-01096-6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36195778/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Reproductive%20Sciences&title=Elucidation%20of%20increased%20cervical%20cancer%20risk%20due%20to%20polymorphisms%20in%20XRCC1%20(R399Q%20and%20R194W),%20ERCC5%20(D1104H),%20and%20NQO1%20(P187S)&volume=30&issue=4&publication_year=2023&pages=1118-1132&pmid=36195778&doi=10.1007/s43032-022-01096-6&)

26. Ganzinelli, M., Linardou, H., Alvisi, M. F., Caiola, E., Lo Russo, G.et al. (2021). Single-arm, open label prospective trial to assess prediction of the role of ERCC1/XPF complex in the response of advanced NSCLC patients to platinum-based chemotherapy. ESMO Open , 6(1), 100034. 10.1016/j.esmoop.2020.100034;  [DOI](https://doi.org/10.1016/j.esmoop.2020.100034) | [PMC free article](/articles/PMC7809372/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33422766/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=ESMO%20Open&title=Single-arm,%20open%20label%20prospective%20trial%20to%20assess%20prediction%20of%20the%20role%20of%20ERCC1/XPF%20complex%20in%20the%20response%20of%20advanced%20NSCLC%20patients%20to%20platinum-based%20chemotherapy&volume=6&issue=1&publication_year=2021&pages=100034&pmid=33422766&doi=10.1016/j.esmoop.2020.100034&)

27. Deek, M. P., Yegya-Raman, N., Daroui, P., Balasubramanian, S., Malhotra, J.et al. (2021). Overexpression of excision repair cross-complementing 1 gene associates with higher risk of therapeutic failure after definitive chemoradiation for unresectable non-small cell lung cancer. Annals of Palliative Medicine , 10(7), 7205–7213. 10.21037/apm-21-182;  [DOI](https://doi.org/10.21037/apm-21-182) | [PMC free article](/articles/PMC10845888/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34263627/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annals%20of%20Palliative%20Medicine&title=Overexpression%20of%20excision%20repair%20cross-complementing%201%20gene%20associates%20with%20higher%20risk%20of%20therapeutic%20failure%20after%20definitive%20chemoradiation%20for%20unresectable%20non-small%20cell%20lung%20cancer&volume=10&issue=7&publication_year=2021&pages=7205-7213&pmid=34263627&doi=10.21037/apm-21-182&)

28. Li, G., Cheng, D. (2020). Meta-analysis of ERCC1 protein expression and platinum chemosensitivity in non-small-cell lung cancer. Evidence-Based Complementary and Alternative Medicine: 2020 , 7376568. 10.1155/2020/7376568;  [DOI](https://doi.org/10.1155/2020/7376568) | [PMC free article](/articles/PMC7210550/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32419821/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Evidence-Based%20Complementary%20and%20Alternative%20Medicine:%202020&title=Meta-analysis%20of%20ERCC1%20protein%20expression%20and%20platinum%20chemosensitivity%20in%20non-small-cell%20lung%20cancer&publication_year=2020&pages=7376568&pmid=32419821&doi=10.1155/2020/7376568&)

29. Tsyganov, M. M., Rodionov, E. O., Ibragimova, M. K., Miller, S. V., Cheremisina, O. V.et al. (2022). Personalized prescription of chemotherapy based on assessment of mRNA expression of BRCA1, RRM1, ERCC1, TOP1, TOP2α, TUBβ3, TYMS, and GSTP1 genes in tumors compared to standard chemotherapy in the treatment of non-small-cell lung cancer. Journal of Personalized Medicine , 12(10), 1647. 10.3390/jpm12101647;  [DOI](https://doi.org/10.3390/jpm12101647) | [PMC free article](/articles/PMC9605448/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36294786/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Personalized%20Medicine&title=Personalized%20prescription%20of%20chemotherapy%20based%20on%20assessment%20of%20mRNA%20expression%20of%20BRCA1,%20RRM1,%20ERCC1,%20TOP1,%20TOP2%CE%B1,%20TUB%CE%B23,%20TYMS,%20and%20GSTP1%20genes%20in%20tumors%20compared%20to%20standard%20chemotherapy%20in%20the%20treatment%20of%20non-small-cell%20lung%20cancer&volume=12&issue=10&publication_year=2022&pages=1647&pmid=36294786&doi=10.3390/jpm12101647&)

30. Ceppi, P., Longo, M., Volante, M., Novello, S., Cappia, S.et al. (2008). Excision repair cross complementing-1 and topoisomerase IIα gene expression in small-cell lung cancer patients treated with platinum and etoposide: A retrospective study. Journal of Thoracic Oncology , 3(6), 583–589. 10.1097/JTO.0b013e3181734f24;  [DOI](https://doi.org/10.1097/JTO.0b013e3181734f24) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18520795/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Thoracic%20Oncology&title=Excision%20repair%20cross%20complementing-1%20and%20topoisomerase%20II%CE%B1%20gene%20expression%20in%20small-cell%20lung%20cancer%20patients%20treated%20with%20platinum%20and%20etoposide:%20A%20retrospective%20study&volume=3&issue=6&publication_year=2008&pages=583-589&pmid=18520795&doi=10.1097/JTO.0b013e3181734f24&)

31. Kim, Y. H., Ishii, G., Goto, K., Ota, S., Kubota, K.et al. (2009). Expression of breast cancer resistance protein is associated with a poor clinical outcome in patients with small-cell lung cancer. Lung Cancer , 65(1), 105–111. 10.1016/j.lungcan.2008.10.008;  [DOI](https://doi.org/10.1016/j.lungcan.2008.10.008) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19036469/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lung%20Cancer&title=Expression%20of%20breast%20cancer%20resistance%20protein%20is%20associated%20with%20a%20poor%20clinical%20outcome%20in%20patients%20with%20small-cell%20lung%20cancer&volume=65&issue=1&publication_year=2009&pages=105-111&pmid=19036469&doi=10.1016/j.lungcan.2008.10.008&)

32. Chiappori, A. A., Zheng, Z., Chen, T., Rawal, B., Schell, M. J.et al. (2010). Features of potentially predictive biomarkers of chemotherapeutic efficacy in small cell lung cancer. Journal of Thoracic Oncology , 5(4), 484–490. 10.1097/JTO.0b013e3181ccb27b;  [DOI](https://doi.org/10.1097/JTO.0b013e3181ccb27b) | [PMC free article](/articles/PMC2861290/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20107425/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Thoracic%20Oncology&title=Features%20of%20potentially%20predictive%20biomarkers%20of%20chemotherapeutic%20efficacy%20in%20small%20cell%20lung%20cancer&volume=5&issue=4&publication_year=2010&pages=484-490&pmid=20107425&doi=10.1097/JTO.0b013e3181ccb27b&)

33. Lee, H. W., Han, J. H., Kim, J. H., Lee, M. H., Jeong, S. H.et al. (2008). Expression of excision repair cross-complementation group 1 protein predicts poor outcome in patients with small cell lung cancer. Lung Cancer , 59(1), 95–104. 10.1016/j.lungcan.2007.07.023;  [DOI](https://doi.org/10.1016/j.lungcan.2007.07.023) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17889401/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lung%20Cancer&title=Expression%20of%20excision%20repair%20cross-complementation%20group%201%20protein%20predicts%20poor%20outcome%20in%20patients%20with%20small%20cell%20lung%20cancer&volume=59&issue=1&publication_year=2008&pages=95-104&pmid=17889401&doi=10.1016/j.lungcan.2007.07.023&)

34. Zou, T., Liu, J. Y., Qin, Q., Guo, J., Zhou, W. Z.et al. (2023). Role of rs873601 polymorphisms in prognosis of lung cancer patients treated with platinum-based chemotherapy. Biomedicines , 11(12), 3133. 10.3390/biomedicines11123133;  [DOI](https://doi.org/10.3390/biomedicines11123133) | [PMC free article](/articles/PMC10741124/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/38137354/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biomedicines&title=Role%20of%20rs873601%20polymorphisms%20in%20prognosis%20of%20lung%20cancer%20patients%20treated%20with%20platinum-based%20chemotherapy&volume=11&issue=12&publication_year=2023&pages=3133&pmid=38137354&doi=10.3390/biomedicines11123133&)

35. Abdalkhalek, E. S., Wakeel, L. M. E., Nagy, A. A., Sabri, N. A. (2022). Variants of ERCC5 and the outcome of platinum-based regimens in non-small cell lung cancer: A prospective cohort study. Medical Oncology , 39(10), 152. 10.1007/s12032-022-01741-9;  [DOI](https://doi.org/10.1007/s12032-022-01741-9) | [PMC free article](/articles/PMC9296400/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35852645/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Medical%20Oncology&title=Variants%20of%20ERCC5%20and%20the%20outcome%20of%20platinum-based%20regimens%20in%20non-small%20cell%20lung%20cancer:%20A%20prospective%20cohort%20study&volume=39&issue=10&publication_year=2022&pages=152&pmid=35852645&doi=10.1007/s12032-022-01741-9&)

36. Simon, G. R., Sharma, S., Cantor, A., Smith, P., Bepler, G. (2005). ERCC1 expression is a predictor of survival in resected patients with non-small cell lung cancer. Chest , 127(3), 978–983. 10.1378/chest.127.3.978;  [DOI](https://doi.org/10.1378/chest.127.3.978) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15764785/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chest&title=ERCC1%20expression%20is%20a%20predictor%20of%20survival%20in%20resected%20patients%20with%20non-small%20cell%20lung%20cancer&volume=127&issue=3&publication_year=2005&pages=978-983&pmid=15764785&doi=10.1378/chest.127.3.978&)

37. Olaussen, K. A., Dunant, A., Fouret, P., Brambilla, E., André, F.et al. (2006). DNA repair by ERCC1 in non-small-cell lung cancer and cisplatin-based adjuvant chemotherapy. The New England Journal of Medicine , 355(10), 983–991. 10.1056/NEJMoa060570;  [DOI](https://doi.org/10.1056/NEJMoa060570) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16957145/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=The%20New%20England%20Journal%20of%20Medicine&title=DNA%20repair%20by%20ERCC1%20in%20non-small-cell%20lung%20cancer%20and%20cisplatin-based%20adjuvant%20chemotherapy&volume=355&issue=10&publication_year=2006&pages=983-991&pmid=16957145&doi=10.1056/NEJMoa060570&)

38. Friboulet, L., Olaussen, K. A., Pignon, J. P., Shepherd, F. A., Tsao, M. S.et al. (2013). ERCC1 isoform expression and DNA repair in non-small-cell lung cancer. The New England Journal of Medicine , 368(12), 1101–1110. 10.1056/NEJMoa1214271;  [DOI](https://doi.org/10.1056/NEJMoa1214271) | [PMC free article](/articles/PMC4054818/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23514287/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=The%20New%20England%20Journal%20of%20Medicine&title=ERCC1%20isoform%20expression%20and%20DNA%20repair%20in%20non-small-cell%20lung%20cancer&volume=368&issue=12&publication_year=2013&pages=1101-1110&pmid=23514287&doi=10.1056/NEJMoa1214271&)

39. Zafeer, M., Mahjabeen, I., Kayani, M. A. (2016). Increased expression of ERCC2 gene in head and neck cancer is associated with aggressive tumors: A systematic review and case-control study. The International Journal of Biological Markers , 31(1), e17–25. 10.5301/jbm.5000186;  [DOI](https://doi.org/10.5301/jbm.5000186) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26659720/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=The%20International%20Journal%20of%20Biological%20Markers&title=Increased%20expression%20of%20ERCC2%20gene%20in%20head%20and%20neck%20cancer%20is%20associated%20with%20aggressive%20tumors:%20A%20systematic%20review%20and%20case-control%20study&volume=31&issue=1&publication_year=2016&pages=e17-25&pmid=26659720&doi=10.5301/jbm.5000186&)

40. Wang, Y. Y., Fang, P. T., Su, C. W., Chen, Y. K., Huang, J. J.et al. (2021). Excision repair cross-complementing group 2 upregulation is a potential predictive biomarker for oral squamous cell carcinoma recurrence. Oncology Letters , 21(6), 450. 10.3892/ol  [DOI](https://doi.org/10.3892/ol) | [PMC free article](/articles/PMC8045162/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33868488/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncology%20Letters&title=Excision%20repair%20cross-complementing%20group%202%20upregulation%20is%20a%20potential%20predictive%20biomarker%20for%20oral%20squamous%20cell%20carcinoma%20recurrence&volume=21&issue=6&publication_year=2021&pages=450&pmid=33868488&doi=10.3892/ol&)

41. Sito, H., Sharzehan, M. A. K., Islam, M. A., Tan, S. C. (2024). Genetic variants associated with response to platinum-based chemotherapy in non-small cell lung cancer patients: A field synopsis and meta-analysis. British Journal of Biomedical Science , 81, 11835. 10.3389/bjbs.2024.11835;  [DOI](https://doi.org/10.3389/bjbs.2024.11835) | [PMC free article](/articles/PMC10914946/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/38450253/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=British%20Journal%20of%20Biomedical%20Science&title=Genetic%20variants%20associated%20with%20response%20to%20platinum-based%20chemotherapy%20in%20non-small%20cell%20lung%20cancer%20patients:%20A%20field%20synopsis%20and%20meta-analysis&volume=81&publication_year=2024&pages=11835&pmid=38450253&doi=10.3389/bjbs.2024.11835&)

42. Lunn, R. M., Helzlsouer, K. J., Parshad, R., Umbach, D. M., Harris, E. L.et al. (2000). XPD polymorphisms: Effects on DNA repair proficiency. Carcinogenesis , 21(4), 551–555. 10.1093/carcin/21.4.551;  [DOI](https://doi.org/10.1093/carcin/21.4.551) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10753184/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Carcinogenesis&title=XPD%20polymorphisms:%20Effects%20on%20DNA%20repair%20proficiency&volume=21&issue=4&publication_year=2000&pages=551-555&pmid=10753184&doi=10.1093/carcin/21.4.551&)

43. Zhao, X., Zhang, Z., Yuan, Y., Yuan, X. (2014). Polymorphisms in ERCC1 gene could predict clinical outcome of platinum-based chemotherapy for non-small cell lung cancer patients. Tumor Biology , 35(8), 8335–8341. 10.1007/s13277-014-2033-7;  [DOI](https://doi.org/10.1007/s13277-014-2033-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24859833/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Tumor%20Biology&title=Polymorphisms%20in%20ERCC1%20gene%20could%20predict%20clinical%20outcome%20of%20platinum-based%20chemotherapy%20for%20non-small%20cell%20lung%20cancer%20patients&volume=35&issue=8&publication_year=2014&pages=8335-8341&pmid=24859833&doi=10.1007/s13277-014-2033-7&)

44. Pérez-Ramírez, C., Cañadas-Garre, M., Alnatsha, A., Villar, E., Valdivia-Bautista, J.et al. (2019). Pharmacogenetics of platinum-based chemotherapy: Impact of DNA repair and folate metabolism gene polymorphisms on prognosis of non-small cell lung cancer patients. The Pharmacogenomics Journal , 19(2), 164–177. 10.1038/s41397-018-0014-8;  [DOI](https://doi.org/10.1038/s41397-018-0014-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29662106/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=The%20Pharmacogenomics%20Journal&title=Pharmacogenetics%20of%20platinum-based%20chemotherapy:%20Impact%20of%20DNA%20repair%20and%20folate%20metabolism%20gene%20polymorphisms%20on%20prognosis%20of%20non-small%20cell%20lung%20cancer%20patients&volume=19&issue=2&publication_year=2019&pages=164-177&pmid=29662106&doi=10.1038/s41397-018-0014-8&)

45. Mlak, R., Krawczyk, P., Homa-Mlak, I., Powrózek, T., Ciesielka, M.et al. (2019). Predictive value of single nucleotide polymorphisms of ERCC1, involved in NER mechanism in patients with advanced NSCLC treated with cisplatin and gemcitabine. Pathology Oncology Research , 25(3), 1035–1045. 10.1007/s12253-018-0459-8;  [DOI](https://doi.org/10.1007/s12253-018-0459-8) | [PMC free article](/articles/PMC6614130/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30066234/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pathology%20Oncology%20Research&title=Predictive%20value%20of%20single%20nucleotide%20polymorphisms%20of%20ERCC1,%20involved%20in%20NER%20mechanism%20in%20patients%20with%20advanced%20NSCLC%20treated%20with%20cisplatin%20and%20gemcitabine&volume=25&issue=3&publication_year=2019&pages=1035-1045&pmid=30066234&doi=10.1007/s12253-018-0459-8&)

46. Mazzoni, F., Cecere, F. L., Meoni, G., Giuliani, C., Boni, L.et al. (2013). Phase II trial of customized first line chemotherapy according to ERCC1 and RRM1 SNPs in patients with advanced non-small-cell lung cancer. Lung Cancer , 82(2), 288–293. 10.1016/j.lungcan.2013.08.018;  [DOI](https://doi.org/10.1016/j.lungcan.2013.08.018) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24045016/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lung%20Cancer&title=Phase%20II%20trial%20of%20customized%20first%20line%20chemotherapy%20according%20to%20ERCC1%20and%20RRM1%20SNPs%20in%20patients%20with%20advanced%20non-small-cell%20lung%20cancer&volume=82&issue=2&publication_year=2013&pages=288-293&pmid=24045016&doi=10.1016/j.lungcan.2013.08.018&)

47. Karaağaç, M., Geredeli, Ç., Yıldırım, M. S., Altınok, T., Dede, I.et al. (2023). The XRCC1 and TP53 gene polymorphisms are associated with advanced-stage disease and early distant metastasis at diagnosis in non-small cell lung cancer. Journal of Cancer Research and Therapeutics , 19(5), 1248–1254. 10.4103/jcrt.jcrt_1657_21;  [DOI](https://doi.org/10.4103/jcrt.jcrt_1657_21) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/37787291/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Cancer%20Research%20and%20Therapeutics&title=The%20XRCC1%20and%20TP53%20gene%20polymorphisms%20are%20associated%20with%20advanced-stage%20disease%20and%20early%20distant%20metastasis%20at%20diagnosis%20in%20non-small%20cell%20lung%20cancer&volume=19&issue=5&publication_year=2023&pages=1248-1254&pmid=37787291&doi=10.4103/jcrt.jcrt_1657_21&)

48. Afifah, N. N., Diantini, A., Intania, R., Abdulah, R., Barliana, M. I. (2020). Genetic polymorphisms and the efficacy of platinum-based chemotherapy: Review. Pharmacogenomics and Personalized Medicine , 13, 427–444. 10.2147/PGPM.S267625;  [DOI](https://doi.org/10.2147/PGPM.S267625) | [PMC free article](/articles/PMC7549502/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33116759/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20and%20Personalized%20Medicine&title=Genetic%20polymorphisms%20and%20the%20efficacy%20of%20platinum-based%20chemotherapy:%20Review&volume=13&publication_year=2020&pages=427-444&pmid=33116759&doi=10.2147/PGPM.S267625&)

49. Zhang, H., Li, Y., Guo, S., Wang, Y., Wang, H.et al. (2020). Effect of ERCC2 rs13181 and rs1799793 polymorphisms and environmental factors on the prognosis of patients with lung cancer. American Journal of Translational Research , 12(10), 6941–6953;  [PMC free article](/articles/PMC7653631/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33194084/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=American%20Journal%20of%20Translational%20Research&title=Effect%20of%20ERCC2%20rs13181%20and%20rs1799793%20polymorphisms%20and%20environmental%20factors%20on%20the%20prognosis%20of%20patients%20with%20lung%20cancer&volume=12&issue=10&publication_year=2020&pages=6941-6953&pmid=33194084&)

50. Zhan, P., Wang, Q., Wei, S. Z., Wang, J., Qian, Q.et al. (2010). ERCC2/XPD Lys751Gln and Asp312Asn gene polymorphism and lung cancer risk: A meta-analysis involving 22 case-control studies. Journal of Thoracic Oncology , 5(9), 1337–1345. 10.1097/JTO.0b013e3181e7fe2a;  [DOI](https://doi.org/10.1097/JTO.0b013e3181e7fe2a) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20651612/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Thoracic%20Oncology&title=ERCC2/XPD%20Lys751Gln%20and%20Asp312Asn%20gene%20polymorphism%20and%20lung%20cancer%20risk:%20A%20meta-analysis%20involving%2022%20case-control%20studies&volume=5&issue=9&publication_year=2010&pages=1337-1345&pmid=20651612&doi=10.1097/JTO.0b013e3181e7fe2a&)

51. Wei, Q., Frazier, M. L., Levin, B. (2000). DNA repair: A double-edged sword. Journal of the National Cancer Institute , 92(6), 440–441. 10.1093/jnci/92.6.440;  [DOI](https://doi.org/10.1093/jnci/92.6.440) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10716953/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20the%20National%20Cancer%20Institute&title=DNA%20repair:%20A%20double-edged%20sword&volume=92&issue=6&publication_year=2000&pages=440-441&pmid=10716953&doi=10.1093/jnci/92.6.440&)
