# The Influence of CYP2B6, GSTP1, and SLCO1B1 Star Allele‐Predicted Phenotypes and CBR1 Genetic Variants on Effectiveness Outcomes in Patients With Hepatocellular Carcinoma Receiving Doxorubicin via Transarterial Chemoembolization

## Metadata
**Authors:** Sireen Abdul Rahim Shilbayeh, Omnia A Abd El‐Baset, Mohammad A Alshabeeb, Abdalrhman Hamdan Alanizi, Naglaa F Khedr, Rehab H Werida
**Journal:** Pharmacology Research & Perspectives
**Date:** 2025 May 22
**DOI:** [10.1002/prp2.70114](https://doi.org/10.1002/prp2.70114)
**PMID:** 40405401
**PMCID:** PMC12098310
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC12098310/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC12098310/pdf/PRP2-13-e70114.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC12098310/pdf/PRP2-13-e70114.pdf)

## Abstract

We investigated the influence of CYP2B6, GSTP1, and SLCO1B1 star allele‐predicted phenotypes and CBR1 variants on clinical outcomes in patients with HCC receiving DOX via TACE. A prospective cohort of patients with HCC underwent DOX therapy via TACE. Selected genes were genotyped in germline DNA samples from the final cohort (82 patients) via Axiom Precision Medicine Diversity (PMD) Research Array technology. The Kaplan–Meier (KM) method and Cox proportional hazards (CPH) model were employed to find independent clinical and genetic predictors of overall survival (OS) and progression‐free survival (PFS) after TACE. Based on univariate and combined association analyses of genetic factors, the star alleles predicting the phenotypic status of three genes (CYP2B6, GSTP1, and SLCO1B1) did not significantly modify the response potential of DOX via TACE, as indicated by OS or PFS. Conversely, we found a novel association between two CBR1 polymorphisms (rs3787728 and rs1005695) and interindividual differences in OS and PFS. The presence of a heterozygous genotype (TC or CG at either locus, which were highly frequent in our cohort), probably with greater CBR metabolic activity, appeared to have an expressive influence by negatively modulating the consequences of DOX locoregional therapy on HCC by shortening the median OS (KM p = 0.02 and 0.04, respectively) and median PFS (KM p = 0.05 and 0.023, respectively) in comparison to those with other haplotypes. Exploratory PGx studies involving a wider HCC cohort and targeting more DOX‐related genes are needed to replicate our findings.

Trial Registration: NCT06313047 (Study Details | Pharmacogenetic of Doxorubicin in HCC. | clinicaltrials.gov)

Keywords: array genotyping, CBR1, doxorubicin, hepatocellular carcinoma, pharmacogenetic testing, transarterial chemoembolization

## 1. Introduction

Hepatocellular carcinoma (HCC) is the sixth most common malignancy worldwide and the fourth leading cause of cancer‐related death [[1](#prp270114-bib-0001)]. The incidence and risk factors for HCC differ across regions and countries. Incidence rates of hepatocellular carcinoma are highest in Eastern Asia, Northern Africa, and Southeast Asia [[1](#prp270114-bib-0001)].

In the Arab world, HCC is a major health problem, especially in Egypt, which has the highest incidence rate of HCC in Africa [[2](#prp270114-bib-0002)]. The incidence of HCC in Saudi Arabia is considered in the middle of international trends; however, the majority of cases are diagnosed at late stages and have poor prognoses [[3](#prp270114-bib-0003)]. The main risk factor for HCC in Egypt is infection with hepatitis C virus (HCV), which affects about 15% of the population [[2](#prp270114-bib-0002)]. In other Arab countries, such as Saudi Arabia, United Arab Emirates (UAE), and Morocco, hepatitis B virus (HBV) is the leading cause of HCC [[4](#prp270114-bib-0004)]. Other factors that contribute to HCC in the Arab region include obesity, diabetes, alcohol consumption, and nonalcoholic steatohepatitis (NASH) [[1](#prp270114-bib-0001), [5](#prp270114-bib-0005)].

Early‐stage HCC is best managed by curative treatment, which includes surgical resection, ablation, or transplantation. Patients with intermediate‐stage disease, particularly unresectable HCC, often receive palliative locoregional therapy, while systemic treatment, including chemotherapy, tyrosine kinase inhibitors, or immunotherapy, is reserved for patients with advanced disease [[6](#prp270114-bib-0006)].

One of the methods of locoregional treatment is transarterial chemoembolization (TACE). This involves injecting a chemotherapy drug (such as doxorubicin [DOX], epirubicin, cisplatin, or adriamycin) mixed with an embolic agent (such as ethiodized oil or microspheres) into the hepatic artery that supplies blood to the tumor. Such localized administration will cause an infarct and thus necrosis of the tumor. This is preferred over systemic chemotherapy, as the drug can directly target the cancer cells and block their blood supply, while minimizing the exposure to the rest of the body [[7](#prp270114-bib-0007)]. Unfortunately, the majority of HCC cases are discovered much later in life, which contributes to the dire prognosis of this disease. Finding a reliable serum biomarker, or combination of pharmacogenomic (PGx) indicators, is still a critical research area for achieving early diagnosis as well as improving clinical outcomes after chemotherapy with TACE sessions [[8](#prp270114-bib-0008), [9](#prp270114-bib-0009)].

DOX is the most common cytotoxic drug used for TACE [[7](#prp270114-bib-0007), [10](#prp270114-bib-0010)], alone or in combination with chemotherapy [[11](#prp270114-bib-0011), [12](#prp270114-bib-0012), [13](#prp270114-bib-0013)] at fixed or variable doses, depending on patient variables (body surface area, weight) and tumor size [[7](#prp270114-bib-0007)]. As a cytotoxic anthracycline antibiotic, DOX operates in two ways: It intercalates between DNA base pairs, and it uncoils the DNA helix. This hinders DNA synthesis and prevents DNA topoisomerase II from breaking and resealing DNA properly. The other mechanism involves the production of free radicals, which damage biological membranes, DNA, and proteins [[14](#prp270114-bib-0014), [15](#prp270114-bib-0015)]. These effects cause quickly dividing cells to undergo apoptosis. DOX is effective against numerous types of cancer, including acute leukemia, lymphomas, sarcomas, and solid tumors. DOX was approved for use in the United States in 1974 and is still a key drug in many cancer chemotherapy regimens [[15](#prp270114-bib-0015), [16](#prp270114-bib-0016), [17](#prp270114-bib-0017)].

DOX is degraded in the liver by microsomal enzymes, which produce toxic or immunogenic intermediates, mainly doxorubicinol and DOX semiquinone radicals [[14](#prp270114-bib-0014)]. In brief, DOX can be transformed to doxorubicinol through two‐electron reduction. Doxorubicinol is less antineoplastic but more cardiotoxic than the parent drug; the rate of this reaction may influence both the antitumor activity and the danger of DOX‐induced heart failure [[16](#prp270114-bib-0016), [18](#prp270114-bib-0018)]. Two carbonic anhydrase isoforms (CBR1 and CRB3) and two aldoketoreductase isoforms (AKR1A1 and AKR1C3) were deemed possible doxorubicinol‐producing enzymes inside the mitochondria that can interrupt respiration and cause the release of cytochrome C, which initiates apoptosis [[15](#prp270114-bib-0015), [17](#prp270114-bib-0017)].

Cytochrome P450 2B6 (CYP2B6) is responsible for the metabolism of anticancer medicines. Population variation caused by single nucleotide polymorphisms (SNPs) in the *CYP2B6* gene results in variations in drug metabolism, which can lead to unpleasant reactions or therapeutic failure. So far, around 30 nonsynonymous variants in the *CYP2B6* gene have been discovered. The occurrence of these polymorphisms reveals intra‐ and interpopulation variability, which influences therapeutic efficacy at the individual and population levels [[19](#prp270114-bib-0019)].

The glutathione S transferase (GST) family is made up of isoenzymes that participate in the Phase II detoxification of xenobiotics by glutathione conjugation. GST is a detoxifying enzyme that regulates cell sensitivity to anticancer medicines. The *GST* gene is involved in DOX resistance [[20](#prp270114-bib-0020), [21](#prp270114-bib-0021)]. Genetic variations in *GST* genes have been linked to limited chemotherapeutic responses. The *GSTP1* Ile to Val substitution has been associated with reduced enzyme activity in the removal of chemical agents, which can lead to several toxicities during chemotherapy [[21](#prp270114-bib-0021)].

P‐glycoproteins, including ATP‐binding cassette (ABC) family members such as ABCB1 transporters (also known as multidrug resistance 1 [MDR1] proteins), are responsible for DOX cell efflux, by which they regulate both the intra‐ and extracellular concentrations and bioavailability of the drug and its metabolites [[22](#prp270114-bib-0022)]. The existence of DOX at lower effective dosages in circulation and/or cancer tissue may result in cancer cell survival and drug resistance [[23](#prp270114-bib-0023)]. *ABCB1* gene polymorphisms have a significant impact on DOX pharmacokinetics (PK) and pharmacodynamics (PD), influencing the DOX concentration and clearance [[24](#prp270114-bib-0024)]. Influx drug transporter genes, including solute carrier (SLC) family transporters such as *SLC22A16*, may have an impact on the systemic PD of DOX, leading to various types of hematological toxicity [[22](#prp270114-bib-0022), [25](#prp270114-bib-0025), [26](#prp270114-bib-0026), [27](#prp270114-bib-0027)].

Carbonyl reductase 1 (CBR1), a monomeric NADPH‐dependent cytosolic enzyme, is the primary hepatic DOX reductase [[16](#prp270114-bib-0016)]. However, the variability in CBR1 enzyme activity may influence the outcomes of therapy with DOX and other CBR1 substrates [[27](#prp270114-bib-0027), [28](#prp270114-bib-0028)]. Previous sequencing of full‐length *CBR1* genes revealed some nonsynonymous SNPs that determine variations in hepatic CBR1 expression [[29](#prp270114-bib-0029), [30](#prp270114-bib-0030)], which displayed unique catalytic and thermodynamic features [[31](#prp270114-bib-0031)]. Among the studied SNPs in the context of DOX PK are *133G>A (rs9024), 720C>T (rs20572), 262G>A (rs1143663), and 391C>T (rs41557318) [[28](#prp270114-bib-0028)]. Breast cancer patients who carried even one of the variants of these SNPs displayed increased DOX exposure levels resulting from decreased clearance, implying the possibility that decreased catalytic activity of the CBR1 enzyme leads to reduced intracellular conversion of DOX to doxorubicinol [[28](#prp270114-bib-0028)]. Another study reported an increased risk of recurrent vomiting induced by DOX in breast cancer homozygous CC carriers of the SNP rs20572 [[27](#prp270114-bib-0027)]. To the best of our knowledge, no *CBR1* polymorphisms have been investigated in terms of their impact on survival outcomes in any cancer setting. Additionally, their influences on DOX PK and PD were neither examined nor replicated in other cancer types.

A review of the current literature revealed that some PGx studies were conducted to examine the influence of candidate markers in certain gene groups (metabolic genes) in modulating the clinical effectiveness of DOX in various types of cancers, though not HCC [[22](#prp270114-bib-0022), [25](#prp270114-bib-0025), [27](#prp270114-bib-0027), [28](#prp270114-bib-0028), [32](#prp270114-bib-0032), [33](#prp270114-bib-0033), [34](#prp270114-bib-0034), [35](#prp270114-bib-0035), [36](#prp270114-bib-0036), [37](#prp270114-bib-0037), [38](#prp270114-bib-0038), [39](#prp270114-bib-0039), [40](#prp270114-bib-0040), [41](#prp270114-bib-0041), [42](#prp270114-bib-0042), [43](#prp270114-bib-0043), [44](#prp270114-bib-0044), [45](#prp270114-bib-0045), [46](#prp270114-bib-0046)]. Additionally, these investigations were inconclusive in terms of association results and limited to certain ethnogeographic groups that mostly received DOX in combination with cyclophosphamide [[28](#prp270114-bib-0028), [33](#prp270114-bib-0033), [39](#prp270114-bib-0039)] or in polychemotherapy regimens [[27](#prp270114-bib-0027), [32](#prp270114-bib-0032), [34](#prp270114-bib-0034), [45](#prp270114-bib-0045)]. These factors could limit the extrapolation of the findings of the studied PGx markers to all cancer settings or diverse ethnic populations, where germline or somatic genetic mutations could be dissimilar in terms of their population frequencies and expression (due to the heterogeneity of the tumors). Within the context of HCC and the use of DOX via the TACE procedure, only one study has examined the impact of SNPs in one glycosyltransferase gene (*GALNT14*) on drug response and survival in a group of Taiwanese patients [[47](#prp270114-bib-0047)]. Despite the significant association between *GALNT14* genotype and cancer cell apoptosis it found [[47](#prp270114-bib-0047)], we speculate that polymorphisms in other genes directly related to the DOX PK/PD pathway have roles in survival outcomes in patients with HCC.

On the above background, the present study was carried out to estimate the frequencies of all genetic variants in the top four rated genes (based on PharmGKB: *CYP2B6*, *GSTP1*, *SLCO1B1*, and *CBR1*) [[15](#prp270114-bib-0015), [17](#prp270114-bib-0017)] reported to be involved in the PK and possibly the PD of DOX and to examine their association with the overall clinical effectiveness outcome in a sample of Arabic HCC patients.

## 2. Materials and Methods

### 2.1. Study Design and Patients

This was a prospective cohort study conducted between August 2022 and December 2023 at qualified oncology departments for the management of HCC in university hospitals. The Institutional Review Board (IRB) of Princess Nourah University (PNU) (IRB Log Number: 23–0177) approved all procedures, which were conducted in compliance with recognized ethical norms. Initially, 224 consecutive HCC patients, both male and female, were screened for potential inclusion. Patients underwent a comprehensive review of their medical history, physical status, and laboratory results. Eligible patients had their baseline viral indicators for HBV and HCV evaluated. Eligibility criteria included: aged ≥ 18 years, with a confirmed diagnosis of intermediate or advanced‐stage HCC (proven by histological, radiological, or pathological analysis) according to the American Association for the Study of Liver Diseases (AASLD) guidelines [[6](#prp270114-bib-0006)], and amenable to receive DOX via TACE technology based on our expert opinion (tumors were unresectable or noncandidate for radiofrequency ablation). Individuals with any of the following criteria were excluded: age > 75 years, presence of portal vein thrombosis, white blood cell count < 3 × 10^9^9^/L, platelet count < 50 × 10^9^9^/L, serum bilirubin > 3 mg/dL, serum creatinine concentration > 1.5 mg/dL, a history of co‐occurring illnesses, or a diagnosis with different types of cancer. All patients provided written informed consent before enrollment. Primary data concerning demographic and clinical information, including age, sex, HCC medical background information (etiology, staging, tumor characteristics, and prognostic criteria), baseline laboratory values, and concurrent medication use were collected from medical records. All data were entered in an anonymous fashion into an electronic annotated data entry program, which was specifically designed for this project by the Research Electronic Data Capture (REDCap) software project linked to PNU [[48](#prp270114-bib-0048)].

Based on a standard conventional TACE treatment protocol, each patient was intra‐arterially infused with a combination emulsion containing two medications: ethionized oil and DOX. For tumors with a diameter less than 5 cm, the ethionized oil dose was < 5 mL. Tumors larger than 5 cm were given a maximum dose of 10 mL. The DOX dose ranged between 30 and 60 mg/m^2^, depending on the tumor size, extent, and blood supply.

### 2.2. Study Oversight

As this study was observational, a follow‐up period was conducted over a period of 13 months to enable the full capture of clinical outcomes, including effectiveness and safety. Triple pelvic abdominal computed tomography (CT) scans were performed both before and 1 month after TACE to identify recurrence and the need for another therapeutic cycle. Follow‐up appointments were scheduled for all patients, including those who achieved complete response (CR), to identify any negative effects and to undergo evaluations of hematologic, kidney, and liver function.

### 2.3. Clinical Outcome Definitions

To evaluate the effectiveness of doxorubicin mediated through TACE, the following outcome measures were employed.

#### 2.3.1. Primary Outcomes

The primary outcomes were overall survival (OS), which was expressed as the number of deaths that occurred at any time after TACE treatment (OSe), as well as the median OS time, which was measured from the date of TACE until the date of death from any cause. If death did not occur, the patients who survived were censored, and they were counted as surviving to the end date of the follow‐up period (December 31, 2023).

PFS events (PFSEs) were defined objectively and subjectively based on two separate measures: radiological examination (by imaging tests) or clear clinical ascertainment of symptomatic deterioration status. Objective radiologic progression was identified based on independent radiologic review, and the patients were classified according to modified Response Evaluation Criteria in Solid Tumors (mRECIST) [[49](#prp270114-bib-0049)] into four groups: complete response, partial response, stable disease, or progressive disease. Subjective symptomatic deterioration status was determined based on a global deterioration of health status requiring discontinuation of treatment or severe symptoms leading to death (as confirmed by the oncologist's documentation in a patient's file). Every effort was made to document patients who did not have any progressive disease as censored events in the PFS analysis at the date of the last follow‐up (December 31, 2023), the date of the start of the next treatment, or the date of death (without hepatic tumor progression). Consequently, the median PFS time was measured from the date of the first TACE session until the date of any evidence of PFSE (tumor radiological or clear clinical symptomatic progression), death, or last follow‐up (December 31, 2023).

The objective response rate (ORR) (which is calculated from the number of events of complete and partial radiological response) and disease control rate (DCR) (which is equal to the ORR plus the number of stable disease events) were determined according to radiological assessments based on the RECIST criteria [[50](#prp270114-bib-0050)]. These surrogate endpoints were considered valid for enhancing the power of detecting genetic associations with effectiveness estimates in the present study, as they were previously validated for their significant correlation with OS in HCC patients treated with TACE [[50](#prp270114-bib-0050), [51](#prp270114-bib-0051)] or immunotherapy [[52](#prp270114-bib-0052)].

#### 2.3.2. Secondary Outcome (α‐Fetoprotein [AFP])

In several studies, AFP was deemed a significant biomarker for tumor biological response due to its ability to predict complete tumor necrosis as well as OS in patients with HCC [[53](#prp270114-bib-0053), [54](#prp270114-bib-0054), [55](#prp270114-bib-0055)]. However, contradictory findings have been reported about AFP sensitivity according to tumor size in HCC [[56](#prp270114-bib-0056), [57](#prp270114-bib-0057), [58](#prp270114-bib-0058)]. Therefore, in the present study, AFP concentrations (ng/mL) were measured before and after TACE treatment (at 46 weeks). The AFP percentage change was estimated based on a comparison between the baseline AFP and the lowest post‐TACE measurement (in the case that the patient received 2 or more cycles). In this study, a more than 20% reduction in the AFP level after TACE was considered a satisfactory endpoint for tumor response (i.e., protection of liver function to the greatest extent, control of the tumor, or downregulation of the tumor stage)‐ [[53](#prp270114-bib-0053)].

### 2.4. Genotyping Method and Quality Control

On the first day of the visit, blood samples for genotyping were collected into two 5‐mL EDTA tubes. Genomic DNA was extracted from whole blood by a QIAGEN Symphony^SP^ automated extraction system via a QIAGEN DSP DNA Midi Kit according to the manufacturer's instructions (Qiagen). A Nanodrop spectrophotometer was used to determine the concentration and purity of the extracted DNA. The extracted genomic DNA samples were amplified by multiplex PCR using a QIAGEN Multiplex PCR Kit according to the manufacturer's instructions (Qiagen). This step was done to allow high‐resolution genotyping of extremely homologous regions of the genome. Following amplification, genome‐wide genotyping of the participants was performed using the Applied Biosystems Axiom Precision Medicine Diversity (PMD) Research Array technology on an automated Applied Biosystems GeneTitan Multi‐Channel (MC) instrument (Affymetrix Inc., Santa Clara, CA, USA). This axiom array solution offers comprehensive coverage of > 850,000 single nucleotide polymorphisms (SNPs), insertions or deletions (indels), and copy number variations (CNVs) with dense whole‐genome coverage across diverse populations (Catalog identifier: 951961; ThermoFisher Scientific, [https://www.thermofisher.com/order/catalog/product/951961/](https://www.thermofisher.com/order/catalog/product/951961/)) [[59](#prp270114-bib-0059)], as well as thorough analysis of > 5000 PGx markers in > 1100 core and extended PK/PD genes across clinical annotation levels of evidence 1–4 (as established by PharmGKB) that influence the absorption, distribution, metabolism, and excretion (ADME) of commonly prescribed medications. In addition, this array facilitates CNV analysis of regions of key pharmacogenes (*CYP2A6*, *CYP2D6*, *GSTM1*, *GSTT1*, *UGT2B17*, and *SULT1A1*).

The genotyping profiles generated were analyzed with Applied Biosystems Axiom Analysis Suite software v5.2 (Thermo Fisher Scientific, Santa Clara, CA, USA). Specifically, for genes that have well‐defined star allele nomenclature, the software enables the translation of genotype calls to functional allele calls (star allele nomenclature), which enables the prediction of gene metabolism (classification into UM, ultrarapid metabolizer; RM, rapid metabolizer; NM, normal metabolizer; IM, intermediate metabolizer; PM, poor metabolizer) or functional status (classification into PF, poor function; DF, decreased function; NF, normal function; IF, increased function) based on detected diplotypes. The databases used to create the allele translation tables were the PharmGKB–Stanford University pharmacogenomics reference database and the Pharmacogene Variation (PharmVar) Consortium [[60](#prp270114-bib-0060)]. All genotype calls of the genes that did not have a star allele nomenclature are reported as homozygous for the major allele, heterozygous, or homozygous for a variant at each polymorphic locus. Markers of the candidate genes having a genotyping call rate < 95%, minor allele frequency (MAF) < 0.05, or Hardy–Weinberg equilibrium (HWE) *p* < 0.001 were excluded from association analysis.

### 2.5. Statistical Analysis

All the data were reviewed, coded, and analyzed via SPSS statistical software version 29 (IBM SPSS Statistics, IBM Inc., NY, USA). The Kolmogorov–Smirnov test was used to assess the normality of the continuous variables, which are presented as mean ± standard deviation, median (interquartile range), or the range. The unpaired Student's *t*‐test was used to compare the means of unrelated parametric values, while the Mann–Whitney test was used to analyze unrelated nonparametric data. Categorical data are summarized as percentages and numerical values. Depending on the normality test, chi‐squared analysis or Fisher's exact test was performed to examine the individual impact of the population categorical data, as well as the separate effect of star allele‐translated phenotypes or genotypes on each of the study outcomes, including death, tumor progression, and biological response (% change in AFP). The Kaplan–Meier (KM) method was used to estimate the survival probability (median OS or median PFS) with respect to different groups (demographics, clinical data, and star allele‐translated phenotypes or genotypes), and the log rank test was used to compare survival (event‐time distributions). Cox proportional hazard analysis was conducted to examine the associations of the variables with the main effectiveness outcomes (OS or PFS) and to obtain the hazard ratio (HR) with its 95% confidence interval (CI) for each factor or covariate separately. Associations of study variables (genetic and nongenetic) with the AFP response were examined by logistic regression analysis and are presented as odds ratios (ORs) and their 95% CIs.

Variables that demonstrated significance in the univariate analyses were included in a multivariate regression to find the most significant predictors of the study outcomes (i.e., multivariate Cox proportional hazard models for OS and PFS or multiple logistic regression analysis for AFP response). For all tests, statistical significance was defined as *p* < 0.05 for a two‐tailed distribution.

## 3. Results

### 3.1. Baseline Patient Demographic Characteristics and Clinical Variables

Out of 224 screened HCC patients, 100 were eligible for the study. 88 patients were initially enrolled in the study, but six patients dropped out of the final analysis because of incomplete data. The patient recruitment, enrollment, and follow‐up data are depicted in Figure [1](#prp270114-fig-0001). The clinical and demographic characteristics of the 82 included patients are given in Table [1](#prp270114-tbl-0001).

### FIGURE 1.

![FIGURE 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/495e/12098310/4c5ee074d80b/PRP2-13-e70114-g002.jpg)

Flowchart for patient screening, recruitment, and follow‐up.

### TABLE 1.

| Demographic data | Value, n (%) |
| --- | --- |
| Sex |  |
| Male | 60 (73.2) |
| Female | 22 (26.8) |
| Age (years) |  |
| Mean ± SD | 62.29 ± 6.57 |
| Median (range) | 61.5 (48–74) |
| Albumin, g/dL, mean ± SD | 3.65 ± 0.54 |
| Hemoglobin, g/dl, mean ± SD | 12.16 ± 1.90 |
| Bilirubin, mg/dL, mean ± SD | 1.01 ± 0.39 |
| ALT, U/L, median (range) | 39 (15–219) |
| AST, U/L, median (range) | 51 (18–248) |
| AFP, ng/mL, median (range) | 46.9 (0.63–36 979) |
| TACE Frequency |  |
| 1 | 75 (91.5) |
| 2 | 4 (4.9) |
| 3 | 2 (2.4) |
| Systemic anticancer therapy |  |
| Sorafenib | 8 (9.8) |
| Cytotoxic chemotherapy | 77 (93.9) |
| Hormonal therapy | 5 (6.1) |
| Antiviral therapy | 19 (23.2) |
| History of comorbidities and prognostic criteria |  |
| Hepatitis C | 78 (95.1) |
| Hepatitis B | 4 (4.9) |
| Liver cirrhosis | 52 (63.4) |
| Ascites | 18 (22) |
| Macroscopic vascular invasion | 19 (23.2) |
| Extrahepatic spread | 6 (7.3) |
| Child–Pugh class |  |
| A | 58 (70.7) |
| B | 22 (26.8) |
| C | 2 (2.4) |
| Tumor characteristics |  |
| BCLC stage |  |
| 0 | 9 (11) |
| A | 35 (42.7) |
| B | 38 (46.3) |
| HCC HFL |  |
| 1 HCC < 2 cm carcinoma in situ | 11 (13.4) |
| 1 HCC or 3 nodules < 3 cm | 32 (39) |
| Multi nodular | 39 (47.6) |
| Disease extension |  |
| Liver only | 67 (81.7) |
| Metastatic | 15 (18.3) |

Table 1 Caption: Baseline demographics and clinical characteristics of the included patients (n = 82).

The mean age of the participants was 62.29 ± 6.57 years, and 73.2% were male. About 92.7% of the patients had hepatitis C, 4.9% had hepatitis B, and the etiological cause was undefined in 3.7%. The presence of liver cirrhosis and ascites was clinically confirmed in 63.4% and 22%, respectively. The patients' history of comorbidities and prognostic criteria are presented in Table [1](#prp270114-tbl-0001). A total of 9.8% of patients required treatment with sorafenib, 93.9% were on cytotoxic chemotherapy, 6.1% were on hormonal therapy, and 23.2% received concomitant antiviral therapy (Table [1](#prp270114-tbl-0001)).

According to the Child–Pugh liver functional classification, most patients were initially class A (70.7%), indicating adequate liver function. According to the Barcelona Clinic Liver Cancer Staging and Treatment Allocation (BCLC) system [[61](#prp270114-bib-0061)], the majority were categorized as class B (46.3%), indicating an intermediate stage with multinodular status, and fewer were classified as early‐stage A (42.7%), associated with 1 or 3 nodules < 3 cm in length, or as very early class 0 (11%), associated with one lesion of less than 2 cm in size.

Macroscopic vascular invasion was identified in 19 patients (23.2%), and extrahepatic spread or metastasis was confirmed in 6 (7.3%) and 15 (18.3%) patients, respectively. By Eastern Cooperative Oncology Group (ECOG) staging [[62](#prp270114-bib-0062)], the performance status was equal to 0 for all patients, indicating that they were fully active and able to complete all their daily activities as before their illness.

Baseline laboratory results were as follows: albumin (3.65 ± 0.54 g/dL), hemoglobin (12.16 ± 1.90 g/dL), bilirubin (1.01 ± 0.39 mg/dL), alanine transaminase (ALT) (39 [range: 15–219] U/L), aspartate transaminase (AST) (51 [range: 18–248] U/L), and AFP (46.9 [range: 0.63–36 979] ng/mL). The TACE procedure frequency during the current study period was 1, 2, and 3 in 92.6%, 4.9%, and 2.5% of the included cohort patients, respectively.

### 3.2. Distribution of the Star Allele‐Predicted Phenotypes of the CYP2B6 , GSTP1 , and SLCO1B1 Genes

Based on the PMD Research Array genotyping module, a total of 192 (120, 14, and 58) probe sets (markers) were targeted in the analysis to identify genotype calls and star alleles (haplotypes) of the *CYP2B6*, *GSTP1*, and *SLCO1B1* genes, respectively. The average call rate of the selected markers was 99.6%. As shown in Table [2](#prp270114-tbl-0002), the functional status of three studied genes (*CYP2B6*, *GSTP1*, and *SLCO1B1*) was determined based on their corresponding known haplotype–diplotype‐based star allele calling via Axiom Analysis Suite software (AxAS, Metabolizer file). Notably, among the three genes identified in the HCC cohort in this study, CYP2B6 NM (47.6%), GSTP1 NM (51.2%), and SLCO1B1 NF (46.3%) were the most common phenotypes.

### TABLE 2.

| Gene a | Frequency, N (%) | Associated genotypes b |
| --- | --- | --- |
| CYP2B6 |  |  |
| PM | 12 (14.6) | *6/*6 |
| IM | 27 (32.9) | *1/*6, *2/*6, *4/*6 |
| RM | 4 (4.9) | *1/*4, *2/*4 |
| NM | 39 (47.6) | *1/*1, *1/*2, *2/*2 |
| GSTP1 |  |  |
| IM | 40 (48.8) | *A/*B, *A/*C, *B/*B, *B/*C, *C/*C |
| NM | 42 (51.2) | *A/*A |
| SLCO1B1 |  |  |
| PF | 3 (3.7) | *15/*15, *5/*15, *5/*5 |
| DF | 22 (26.8) | *1A/*15, *1A/*5, *1B/*15, *1B/*17 |
| IF | 8 (9.8) | *1A/*14, *1B/*14 |
| NF | 38 (46.3) | *14/*15, *14/*21, *1A/*1A, *1A/*1B, *1B/*1B, *5/*14 |
| UF | 11 (13.4) | *15/*27, *17/*21, *1A/*21, *1A/*27, *1B/*21, *21/*21 |

Table 2 Caption: Phenotype frequencies and associated genotypes (known calls) for the considered genes (n = 82).

### 3.3. Distribution of CBR1 Genotypes

From the *CBR1* gene, out of seven markers identified in the PMD Research Array genotyping module, three SNPs were selected based on standard parameters (biallelic, passed the dish QC, MAF > 0.05, and HWE *p* > 0.001) for inclusion in the final association analysis. The MAFs of the three included CRB1 SNPs (rs2835265, rs3787728, and rs1005695) in the current HCC cohort were 0.0526, 0.237, and 0.324, respectively. As a nonstar allele gene, *CBR1* had genotypes presented as homozygous for the major allele, homozygous for the variant allele, or heterozygous at each SNP. *CBR1* rs2835265 SNP analysis revealed that the major‐allele CC genotype was present in 74 patients (99.2%), while the heterozygous genotype TC was present in eight patients (8.9%). The *CBR1* rs2835265 homozygote TT was not detected in this cohort. *CBR1* rs3787728 had the major‐allele TT genotype in 4 patients (4.9%), while the variant genotypes TC and CC were found in 30 patients (36.6%) and 48 patients (58.5%), respectively. For *CRB1* rs1005695, the major‐allele GG genotype was present in 8 patients (9.8%), while the variant genotypes GC and CC were present in 36 patients (43.9%) and 38 patients (46.3%), respectively.

### 3.4. Effectiveness Outcomes

In the present HCC cohort, the median follow‐up was 9.13 months (range 1.77–95.27). At the end of the follow‐up period, the cumulative OS rate was 90.1%, while the cumulative PFS rate was 54.3%.

The median PFS time was 21.73 months (95% CI 10.87–107.87), and the median OS time was 84.34 months (95% CI 77.16–91.52). A biological response (20% reduction in AFP) was obtained in 36 patients (44.4%). The distribution of the radiological response level attained at the end of the study period, based on the mRECIST criteria, was as follows: complete response in 20 patients (24.7%), partial response in 21 patients (25.9%), stable response in 3 patients (3.7%), and progressive response in 37 patients (45.7%).

#### 3.4.1. Association of Nongenetic Predictors With Effectiveness

As shown in Table [S1](#prp270114-supitem-0001), the initial univariate analysis by Kaplan–Meier and Cox proportional hazard methods revealed some clinical and prognostic patient criteria that were found to have a significant negative impact on both survival outcomes median OS and median PFS. Clearly, the presence of clinical factors such as ascites, macroscopic vascular invasion, metastatic disease extension, and progression of radiological response (based on mRECIST) was associated with shorter OS and PFS. On the other hand, certain other clinical factors, such as AFP response status and hepatitis B etiology, were significant negative predictors of median PFS but not median OS. Other demographic data (such as age and sex), baseline laboratory data (including albumin, hemoglobin, bilirubin, ALT, ALT, and AFP), concomitant use of systemic therapy (sorafenib, cytotoxic chemotherapy, and antivirals), presence of liver cirrhosis, Child–Pugh class, TACE frequency, and BCLC stage had nonsignificant influences on both OS and PFS in this HCC cohort (Table [S1](#prp270114-supitem-0001)). A multivariate Cox proportional hazards model with a forward likelihood ratio revealed that the presence of metastatic disease extension was the most significant predictor of shorter OS, with an HR of 8.36 (95% CI 1.1–63.5, *p* = 0.04), while hepatitis B etiology was the strongest predictor of shorter PFS (HR: 5.04, 95% CI 1.26–20.1, *p* = 0.022).

As for the biological response, univariate association analyses between the response status (response vs. nonresponse) and baseline patient characteristics are given in Table [S2](#prp270114-supitem-0001). In this HCC cohort, none of the demographic or clinical criteria were associated with changes in AFP after TACE therapy.

#### 3.4.2. Association of Genetic Predictors With Effectiveness

The distributions of median OS and median PFS with respect to each star allele gene phenotype status are displayed in Table [3](#prp270114-tbl-0003). For CYP2B6, the longest median OS and PFS were observed among individuals with RM (27.7 and 54.4 months, respectively) compared to those with other phenotypes, but the differences did not reach statistical significance (log rank *p* = 0.43 and 0.2, respectively). For GSTP1, the estimated OS and PFS were longer among HCC patients characterized by IM status (86.8 and 32 months, respectively) than among those characterized by NM (81.6 and 21.7 months, respectively), but again the differences were not statistically significant (log rank *p* = 0.45 and 0.8, respectively). For the *SLCO1B1* gene, like CYP2B6, the increased functional status of the SLCO1B1 IF phenotype group was associated with longer OS and PFS (29.2 and 61.1 months, respectively). The estimates of OS and PFS tended to be much shorter among the patients with a lower‐functioning *SLCO1B1* gene of the PF/DF (10.1 and 16.6 months) or NM kind (18.1 and 21.7 months), though again the differences were not statistically significant (log rank *p* = 0.74 and 0.3, respectively).

| Gene | Effectiveness measure |  |  |  |  |  |  |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Overall survival (OS) |  |  |  |  | Progression‐free survival (PFS) |  |  |  |  |  |
| p a | Median OS (95% CI) | p b | Unadjusted HR (95% CI) | p c | p a | Median PFS (95% CI) | p b | Unadjusted HR (95% CI) | p c |  |
| CYP2B6 |  |  |  |  |  |  |  |  |  |  |
| PM | 0.47 | 23.8 (5.2–42.5) | 0.43 | 0 | 0.99 | 0.59 | 65.5 (41.7–89.1) | 0.2 | 1.58 (0.18–14.2) | 0.68 |
| IM |   | 15.5 (6.2–24.8) |   | 1.73 (0.43–6.96) | 0.44 |   | 12.7 (0.12–25.3) |   | 3.35 (0.44–25.6) | 0.24 |
| PM/IM |   | 18 (9.8–26.3) |   | 1.16 (0.29–4.6) | 0.84 |   | 48.2 (31.5–64.8) |   | 1.1 (0.56–2.1) | 0.82 |
| RM |   | 27.7 (18.8–74.1) |   | 0 | 0.99 |   | 54.4 (27.1–81.6) |   | 0.4 (0.05–3.1) | 0.38 |
| NM |   | 16.1 (11.4–20.9) |   | 2.57 (0.35–18.8) | 0.35** |   | 21.7 (10.2–33.3) |   | 2.49 (0.33–18.8) | 0.38** |
| GSTP1 |  |  |  |  |  |  |  |  |  |  |
| IM | 0.71 | 86.8 (77.8–95.8) | 0.45 | 0.58 (0.138–2.4) | 0.45 | 0.66 | 32 (0–69.9) | 0.8 | 0.92 (0.48–1.77) | 0.8 |
| NM |   | 81.6 (70.3–92.8) |   |   |   |   | 21.7 (10.7–32.7) |   |   |   |
| SLCO1B1 |  |  |  |  |  |  |  |  |  |  |
| PF/DF | 0.69 | 10.1 (8.1–12) | 0.74 | 1.0 (0.23–4.5) | 0.99 | 0.27 | 16.6 (13.7–19.6) | 0.3 | 0.675 (0.29–1.6) | 0.35 |
| IF |   | 29.2 (3.7–54.6) |   | 0.93 (0.1–8.3) | 0.95 |   | 60.1 (28.9–91.3) |   | 0.537 (0.16–1.8) | 0.32 |
| UF |   | 24.7 (4.7–44.7) |   | 0.75 (0.19–2.99) | 0.98 |   | 19.8 (17.9–21.8) |   | 0.664 (0.25–1.8) | 0.41 |
| NF |   | 18.1 (11–25.2) |   | 0.39 (0.053–2.9) | 0.68* |   | 21.7 (10.5–32.9) |   | 0.640 (0.33–1.2) | 0.18* |

Table Caption: Univariate associations of the star allele‐predicted phenotypes of the CYP2B6, GSTP1, and SLCO1B1 genes with effectiveness outcomes (overall survival, progression‐free survival).

Regarding the biological response, as noted in Table [4](#prp270114-tbl-0004), none of the three genes' star alleles predicted phenotypes displayed a significant impact on the AFP% change after TACE therapy.

| Gene | Effectiveness according to AFP % change |  | p a | OR (95% CI) b | p c |
| --- | --- | --- | --- | --- | --- |
| Responder, N (%) | Nonresponder, N (%) |  |  |  |  |
| CYP2B6 |  |  |  |  |  |
| PM | 4 (11.1) | 7 (16.3) | 0.490 | 1.14 (0.08–16.95) | 0.923 |
| IM | 15 (41.7) | 11 (25.6) |   | 2.73 (0.23–34) | 0.436 |
| RM | 1 (2.8) | 2 (4.7) |   | 0.72 (0.06–8.6) | 0.794 |
| NM | 16 (44.4) | 23 (53.5) |   | 1.391 (0.12–16.7) | 0.794 |
| GSTP1 |  |  |  |  |  |
| IM | 19 (52.8) | 19 (44.2) | 0.502 | 1.4 (0.58–3.4) | 0.447 |
| NM | 17 (47.2) | 24 (55.8) |   |   |   |
| SLCO1B1 |  |  |  |  |  |
| PF/DF | 9 (25) | 15 (34.9) | 0.540 | 0.75 (0.26–2.2) | 0.593 |
| IF | 4 (11.1) | 4 (9.3) |   | 1.25 (0.27–5.8) | 0.776 |
| UF | 7 (19.4) | 4 (9.3) |   | 2.19 (0.54–8.8) | 0.271 |
| NF | 16 (44.4) | 20 (46.5) |   | 1.7 (0.15–19.6) | 0.668 |

Table Caption: Univariate associations of the star allele‐predicted phenotypes of the CYP2B6, GSTP1, and SLCO1B1 genes with the biological response (% change in AFP).

The distributions of OS and PFS medians according to the five identified *CRB1* SNPs (rs2835265, rs3787728, and rs1005695) are displayed in Table [5](#prp270114-tbl-0005). For *CRB1* rs2835265, despite the observed reduction in median OS and PFS associated with the variant genotype TC (19.8 and 14.6 months, respectively) compared with the major‐allele CC genotype (84.3 and 32 months, respectively), the survival probabilities estimated by the KM method (log rank *p* = 0.86 and 0.62, respectively) and Cox proportional hazard analysis (HR_OS_ 1.2 [CI 0.148–9.8] and HR_PFS_ 1.3 [CI 0.457–3.7]) were similar (*p* > 0.05).

| Marker name | SNP rsID | Chr. | Genotype | N frequency (%) | Effectiveness measure |  |  |  |  |  |  |  |  |  | Effectiveness according to AFP % change |  | p a | OR (95% CI) d | p e |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Overall survival (OS) |  |  |  |  | Progression‐free survival (PFS) |  |  |  |  | Responder (%) | Nonresponder (%) |  |  |  |  |  |  |  |  |
| Univariate Analysis P a | Median OS (95% CI) | p b | Unadjusted HR (95% CI) | p c | Univariate Analysis p a | Median PFS (95% CI) | p b | Unadjusted HR (95% CI) | p c |  |  |  |  |  |  |  |  |  |  |
| c.398‐48C>T (CBR1) | rs2835265 | 21 | Homozygote (variant) | 0 (0) |   |   |   |   | 1 |   |   |   |   |   | — |   |   |   |   |
| Heterozygote | 8 (9.8) | 0.582 | 19.8 (16.2–23.4) | 0.864 | 1.2 (0.148–9.8) | 0.864 | 1.0 | 14.6 (2.47–26.8) | 0.615 | 1.3 (0.457–3.7) | 0.623 | 4 (11.1) | 4 (9.3) | 0.791 | 1.2 (0.28–5.3) | 0.79 |  |  |  |
| Homozygote (major) | 74 (90.2) |   | 84.3 (76.5–92) |   |   |   |   | 32 (0–72.96) |   |   |   | 32 (88.9) | 39 (90.7) |   |   |   |  |  |  |
| c.397 + 538 T>C (CBR1) | rs3787728 | 21 | Homozygote (variant) | 48 (58.5) | 0.062 | 19.3 (12.7–25.9) | 0.046 | 5184 (0–8.6E + 134) | 0.96 ‡ | 0.263 | 59.4 (10.9–107.9) | 0.054 | 2.1 (0.26–15.5) | 0.482 ‡ | 21 (58.3) | 25 (58.1) | 0.150 | 1.01 (0.41–2.5) | 0.99* |
| Heterozygote | 30 (36.6) |   | 13.7 (7.2–20.1) |   | 5.3 (1.07–26.3) | 0.04 ** |   | 7.1 (0–21.7) |   | 1.82 (0.938–3.5) | 0.077 ** | 15 (41.7) | 14 (32.6) |   | 1.3 (0.5–3.2) | 0.61** |  |  |  |
|   |   |   |   |   | 4.4 (0.89–22) | 0.068 †† |   |   |   | 1.57 (0.82–3.01) | 0.173 †† |   |   |   |   |   |  |  |  |
| Homozygote (major) | 4 (4.9) |   | 27.4 (18.3–73.1) |   | 0 | 0.99 |   | 56 (31.5–80.5) |   |   |   | 0 1 (0) | 4 (9.3) |   |   |   |  |  |  |
| c.397 + 210G>C (CBR1) | rs1005695 | 21 | Homozygote (variant) | 38 (46.3) | 0.131 | 92.8 (87.9–97.6) | 0.136 | 0.22 (0.014–3.6) | 0.287 | 0.123 | 59.4 (39.7–73.1) | 0.027 | 1.039 (0.293–3.7) | 0.953 | 16 (44.4) | 20 (46.5) | 0.837 | 1.3 (0.28–6.4) | 0.72 ‡ |
|   |   |   |   |   | 0.163 (0.02–1.3) | 0.089 * |   |   |   | 0.55 (0.28–1.1) | 0.085 * |   |   |   | 0.92 (0.38–2.2) | 0.85* |  |  |  |
| Heterozygote | 36 (43.9) |   | 56.1 (45.9–66.3) |   | 1.46 (0.175–12) | 0.729 |   | 14.6 (0–30.8) |   | 2.175 (0.635–7.5) | 0.216 | 17 (47.2) | 18 (41.9) |   | 1.6 (0.33–7.6) | 0.57 † |  |  |  |
|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 1.5 (0.32–6.5) | 0.63*** |  |  |  |
| Homozygote (major) | 8 (9.8) |   | 83.96 (63.5–104.4) |   |   |   |   | 57.4 (24.1–90.7) |   |   |   | 3 (8.3) | 5 (11.6) |   |   |   |  |  |  |
| c.*133G>A (CBR1) | rs9024 | 21 | Homozygote (variant) | 0 (0) | 0.531 |   |   |   |   | 0.697 |   |   |   |   | — | — |   |   |   |
| Heterozygote | 7 (8.5) |   | 19.5 (15.4–23.6) | 0.731 |   |   |   | 14.6 (2.3–26.97) | 0.43 |   |   | 3 (8.3) | 4 (9.3) | 1 | 0.89 (0.19–4.3) | 0.88 |  |  |  |
| Homozygote (major) | 75 (91.5) |   | 84.5 (76.9–92.05) |   | 0.69 (0.085–5.6) | 0.442 |   | 32.03 (0–72.8) |   | 0.66 (0.23–1.9) | 0.442 | 33 (91.7) | 39 (90.7) |   |   |   |  |  |  |
| c.627C>T (CBR1) | rs20572 | 21 | Homozygote (variant) | 0 (0) | 0.531 |   |   |   |   | 0.697 |   |   |   |   | — | — |   |   |   |
| Heterozygote | 7 (8.5) |   | 19.5 (15.4–23.6) | 0.731 |   |   |   | 14.6 (2.3–26.97) | 0.43 |   |   | 3 (8.3) | 4 (9.3) | 1 |   |   |  |  |  |
| Homozygote (major) | 75 (91.5) |   | 84.5 (76.9–92.05) |   | 0.83 (0.29–2.4) | 0.733 |   | 32.03 (0–72.8) |   | 0.815 (0.48–1.4) | 0.442 | 33 (91.7) | 39 (90.7) |   | 1.13 (0.24–5.4) | 0.88 |  |  |  |

Table Caption: Genotype frequencies of common markers in the CBR1 gene and their univariate associations with effectiveness outcomes (overall survival, progression‐free survival, and biological response).

However, for the second *CRB1* SNP (rs3787728), the heterozygous TC genotype was significantly associated with reduced cumulative OS (Figure [2](#prp270114-fig-0002): A1) and greater hazard for death (HR_OS_ 5.3 (CI 1.07–26.3)) than the homozygous CC genotype. Further association analysis was conducted by pooling the TC genotype with the major‐allele TT genotype (four patients), which also revealed a significantly lower probability of OS with this combination than with the homozygous CC genotype (Figure [2](#prp270114-fig-0002) (A2): log rank *p* = 0.045). The latter comparison confirmed the impact of the T allele (reference) on shortening OS compared with the C allele in this HCC cohort. Similarly, examination of the impact of this SNP (rs3787728) on PFS reinforced the influence of the T allele on reducing progression survival (Table [5](#prp270114-tbl-0005) and Figure [3](#prp270114-fig-0003): B1 and B2), where the TC genotypes demonstrated a markedly reduced median PFS interval of 7.1 months (CI 0–21.7) in comparison with that of the homozygote CC genotype at 59.4 months (CI 10.9–107.9).

![FIGURE 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/495e/12098310/eb1c91cd4537/PRP2-13-e70114-g003.jpg)

Kaplan–Meier analysis of overall survival (OS) in HCC patients receiving doxorubicin chemotherapy mediated through TACE. (A1) OS in patients with different CRB1 rs3787728 genotypes. (A2) OS in CRB1 rs3787728 T allele carriers (TC and TT genotypes) versus non‐T carriers (CC genotype). (A3) OS in patients with different CRB1 rs1005695 genotypes. (A4) OS in CRB1 rs1005695 G allele carriers (GC and GG genotypes) versus non‐G carriers (CC genotype).

![FIGURE 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/495e/12098310/8c6eaf21b2e7/PRP2-13-e70114-g004.jpg)

Kaplan–Meier analysis of progression‐free survival (PFS) in HCC patients receiving doxorubicin chemotherapy mediated through TACE. (B1) PFS in patients with different CRB1 rs3787728 genotypes. (B2) PFS in CRB1 rs3787728 T allele carriers (TC and TT genotypes) versus non‐T carriers (CC genotype). (B3) PFS in patients with different CRB1 rs1005695 genotypes. (B4) PFS in CRB1 rs1005695 G allele carriers (GC and GG genotypes) versus non‐G carriers (CC genotype).

For the third *CRB1* SNP (rs1005695), carriers of the G allele (reference) in the GG and CG groups had a significantly shorter OS interval than CC homozygotes (Figure [2](#prp270114-fig-0002): A3 and A4). The impact of the G allele of the *CRB1* SNP (rs1005695) on PFS was also evident (Figure [3](#prp270114-fig-0003): B3 and B4). The other two minorly prevalent *CRB1* SNPs (rs9024 and rs20572) did not significantly affect either OS or PFS.

Similarly (as noted earlier for other gene phenotypes concerning the biological response), none of the *CRB1* SNPs displayed a significant association with the percentage change in AFP after TACE therapy (Table [5](#prp270114-tbl-0005)).

#### 3.4.3. Combined Genetic and Nongenetic Multivariate Association Analyses

Initially, combined analysis of genetic factors revealed that T allele carriers of *CRB1* rs3787728 (TC and TT genotypes) were more likely to have reduced OS than G allele carriers of *CRB1* rs1005695 (CG and GG genotypes) (HR 4.4, CI 0.9–22.02, *p* = 0.068).

Subsequently, nongenetic factors (including the presence of ascites, macroscopic vascular invasion, and metastatic disease extension), which revealed a significant impact on OS in the univariate analysis, were incorporated with *CRB1* polymorphisms (rs3787728 and rs1005695) in multivariate Cox regression analysis by the forward stepwise likelihood ratio. The obtained model revealed that metastatic disease extension (*p* = 0.015), followed by ascites (*p* = 0.062), was the most powerful predictive factor for modulating the impact of doxorubicin on OS than genotypic variants within both *CRB1* SNPs.

When we conducted combined multivariate Cox regression analysis of genetic factors to explore their impact on PFS, we found that G allele carriers of *CRB1* rs1005695 (GC and GG genotypes) had a greater risk of shortening PFS than T allele carriers of *CRB1* rs3787728 (TC and TT genotypes). However, exploration of nongenetic factors (including AFP response status, hepatitis B etiology, and other aforementioned OS prognostic factors) together with *CRB1* polymorphisms (rs3787728 and rs1005695) demonstrated the greater influence of nongenetic prognostic factors on PFS. The AFP response status (*p* = 0.006), the presence of macroscopic vascular invasion (*p* = 0.022), metastatic disease extension (*p* = 0.028), and hepatitis B etiology (*p* = 0.06) were independent predictors of PFS. The statistical significance of *CRB1* polymorphisms (rs3787728 and rs1005695) was not retained in the final model.

## 4. Discussion

Based on the efforts of PharmGKB, a number of top evidence‐rated genes, together with their associated variants, were integrated to aid further research on precision medicine and its future implementation in clinical practice [[15](#prp270114-bib-0015), [17](#prp270114-bib-0017)]. These PGx markers are deemed essentially useful as an initial skeleton to explain interindividual and interethnic variability of certain chemotherapeutic drugs, such as DOX PK and PD, and hence how they could modulate the clinical response [[15](#prp270114-bib-0015)]. However, the heterogeneous nature of tumors in different cancer sites, the interethnic variability in germline genetic variant frequencies, and the ongoing advancements in genotyping techniques and bioinformatics tools may hinder the extrapolation of all current findings to global cancer settings in diverse ethnogeography groups. Our study examining the associations of polymorphisms in four candidate genes with survival outcomes in Arabic HCC patients receiving DOX via TACE has yielded findings relevant to our population in this cancer setting.

### 4.1. Star Allele‐Predicted Phenotypes and Survival Outcomes

The first group of findings concerning the predicted phenotypes of the star alleles was obtained via pharmacogenomic testing with the Axiom Array, which enables the examination of a broad collection of probe sets targeting all potential genetic variants within the three studied metabolic genes (*CYP2B6*, *GSTP1*, and *SLCO1B1*). The definitions of the star alleles and their corresponding metabolic statuses were based on the updated PharmGKB–Stanford University pharmacogenomics reference database and the Pharmacogene Variation (PharmVar) Consortium [[60](#prp270114-bib-0060), [63](#prp270114-bib-0063), [64](#prp270114-bib-0064)]. The implementation of this genotyping technique with its accurate translation of star allele calls to distinct phenotypes in our HCC patients made this study a unique attempt in comparison to previous DOX PGx studies in which a limited number of SNPs in the three genes were genotyped without subgrouping of patient genotypes according to known metabolic capacity phenotypes. Notably, in our study, despite interindividual variations in OS and PFS among the three gene phenotypes, the differences were not statistically significant. This finding implies that the three metabolic genes had nonsignificant impacts on modulating the effect of DOX on improving survival outcomes in patients with HCC.

Regarding the *CYP2B6* gene, the *2 (normal function), *8 (no function), *9 (decreased function), and *4 (increased function) alleles [[65](#prp270114-bib-0065)] were associated with a poor outcome (in terms of OS, PFS, or both), while *CYP2B6**5 (normal function) was associated with a longer PFS [[33](#prp270114-bib-0033)] in the setting of breast cancer patients receiving a combination therapy of DOX and cyclophosphamide [[33](#prp270114-bib-0033)]. Consistent with our findings, other studies have failed to show a statistically significant impact of *CYP2B6* SNPs on breast cancer survival [[34](#prp270114-bib-0034), [39](#prp270114-bib-0039)]. These conflicting findings could be attributed to the fact that the metabolic influence of *CYP2B6* could be substrate specific as well as to differences in the frequency of ethnogeographic star alleles among different cohorts [[66](#prp270114-bib-0066)]. On the other hand, *CYP1B1* SNPs, which are involved in the metabolism of DOX, were found to be associated with the incidence of cardiotoxicity in an animal model study [[67](#prp270114-bib-0067)], but no evidence of their impact on OS or PFS was noted in childhood acute lymphoblastic patients receiving DOX in polychemotherapy [[32](#prp270114-bib-0032)].

For the *GSTP1* gene, our observation of a nonsignificant impact of its predicted phenotypes (IMs vs. NMs) on predicting survival was consistent with some previous studies examining the influence of its common SNP (*GSTP1**B_c.313A>G(I105V), rs1695), which is associated with reduced enzymatic activity. In agreement with our findings, the mutated variant 105 V had no effect on OS or PFS in childhood acute lymphoblastic leukemia [[32](#prp270114-bib-0032)] or other malignancies [[34](#prp270114-bib-0034), [39](#prp270114-bib-0039), [44](#prp270114-bib-0044), [68](#prp270114-bib-0068)]. In contrast to our findings, decreased PFS has been found among variant carriers of this reduced‐function *GSTP1* SNP in patients with osteosarcoma [[45](#prp270114-bib-0045)] or decreased OS among major‐allele homozygotes (*A/*A) with advanced gastric cancer [[69](#prp270114-bib-0069)] receiving DOX as multiagent chemotherapy. On the other hand, *GSTP1* homozygous *GSTP1* (*B/*B genotype) has been associated with a reduced risk of chemoresistance to doxorubicin in breast cancer patients [[42](#prp270114-bib-0042)]. Collectively, these negative findings on survival outcomes were mainly attributed to the altered sensitivity of cancer cells to DOX chemotherapy induced by bearing either *GSTP1* genotypes resulting in chemoresistance (in major‐allele homozygotes) [[42](#prp270114-bib-0042), [69](#prp270114-bib-0069), [70](#prp270114-bib-0070)] or increased toxicity associated with less DOX detoxification in variant carriers (such as leukopenia and cardiotoxicity) [[45](#prp270114-bib-0045), [71](#prp270114-bib-0071)]. However, these interpretations were not supported by the use of DOX via TACE in the current HCC cohort, possibly due to the overexpression of *GSTP1* in patients with liver cirrhosis or in patients with HCC (as previously reported), regardless of the germline phenotype [[20](#prp270114-bib-0020), [72](#prp270114-bib-0072)]. Our results are more in line with studies [[42](#prp270114-bib-0042), [69](#prp270114-bib-0069), [70](#prp270114-bib-0070)] reporting the favorable impact of harboring the mutant allele *B, as the IMs in our cohort tended to have longer OS and PFS than the NMs. Nevertheless, further molecular studies estimating the GSTP1 protein level in HCC tumor tissues are speculated to undermine or highlight the significance of GSP1 germline phenotyping before its use as a PGx marker for DOX chemoresistance or safety in HCC clinical practice.

The polymorphisms of *SLCO1B1*, a gene encoding a liver‐specific member of the organic anion transporter family [[73](#prp270114-bib-0073), [74](#prp270114-bib-0074)], were speculated to have an impact on DOX transport from the blood into hepatocytes in the HCC setting. Despite the observed variability among the predicted phenotypes of *SLCO1B1* star alleles in terms of OS and FPS median survival time, the difference was nonsignificant in the present cohort for both survival terms. Polymorphism in another transporter gene, *SLC22A16* (a member of the organic cation transporter family), which is highly expressed in bone marrow but minimally expressed in hepatocytes [[75](#prp270114-bib-0075)], was shown to modulate DOX blood levels [[25](#prp270114-bib-0025)] and hence increase toxicity (mainly leucopenia) [[33](#prp270114-bib-0033)] and neutropenia [[22](#prp270114-bib-0022), [27](#prp270114-bib-0027)] in breast cancer patients. In line with our results, these *SLC22A16* SNPs have not shown a proven impact on survival (OS and PFS) [[25](#prp270114-bib-0025), [33](#prp270114-bib-0033)]. Importantly, patients in former cancer sites were receiving systemic DOX‐based chemotherapy [[22](#prp270114-bib-0022), [25](#prp270114-bib-0025), [27](#prp270114-bib-0027), [33](#prp270114-bib-0033)], while our patient received DOX in the form of locoregional administration directed into the hepatic artery, a factor that minimizes exposure of normal body cells to DOX. Whether *SLCO1B1* or *SLC22A16* polymorphisms impact the DOX safety profile (laboratory or other adverse events) in the current HCC cohort remains an area of interest for our future multiplex genotyping research in HCC. These investigations are deemed mandatory for prioritizing the selection of relevant PGx transporter markers according to the method of DOX administration in various clinical cancer practice venues.

### 4.2. Common CBR1 Genotypes and Survival Outcomes

The second group of findings of this study is related to *CBR1* polymorphisms. In fact, in addition to known clinical nongenetic prognostic HCC factors, *CBR1* interindividual genetic variation had the greatest genetic impact on OS and PFS in this cohort. For the first time, the presence of reference alleles of two novel SNPs in *CBR1* (rs3787728 and rs1005695), which occurred at a high frequency in our cohort (41.5% and 53.7%, respectively), appeared to influence the effects of DOX locoregional therapy on OS and PFS in patients with HCC. Notably, these SNPs were not examined in earlier studies. The present observation of their negative impact is presumably attributed to the higher activity associated with harboring the reference alleles in either SNP, leading to greater CBR metabolic activity (higher clearance rate of DOX) with faster conversion to its more toxic but less antineoplastic metabolite (doxorubicinol) [[16](#prp270114-bib-0016), [18](#prp270114-bib-0018)]. Furthermore, another possible explanation for the observed association of *CRB1* polymorphisms with survival outcomes in parallel with the less pronounced association of other studied genes in the present study could be related to the higher expression levels of the CRB1 enzyme in hepatocytes [[16](#prp270114-bib-0016)], the confined site where the DOX TACE procedure is targeted in HCC patients. Both assumptions require validation in additional molecular and functional PK studies. Similarly, a previous PK study reported that homozygosity for the reference allele at the *CBR1 c.627C*>T (rs20572) and c.*133G>A (rs9024) loci (both in strong linkage disequilibrium) were associated with increased clearance and reduced exposure to DOX in Asian breast cancer patients [[28](#prp270114-bib-0028)]. Our attempt to further examine the power of these similar SNPs [[28](#prp270114-bib-0028)] did not yield a significant impact on either OS or PFS (Table [5](#prp270114-tbl-0005)), probably due to their much lower prevalence in our cohort (0.047 for both SNPs). In contrast, the allele frequency of the c.627C>T polymorphism was reported to be higher in earlier studies involving other ethnic groups (Indian (0.18), Caucasian (0.15), and Chinese (0.26)) [[29](#prp270114-bib-0029)]. Therefore, these findings are expected to demonstrate greater consequences on the survival outcomes of patients with HCC within the aforementioned populations. Interestingly, the prevalence of the well‐characterized nonsynonymous V88I (c.262G>A, rs1143663) polymorphism in the *CBR1* gene [[30](#prp270114-bib-0030)], which appears at a low frequency among African Americans (0.014) and results in a marked reduction in the kinetic and thermodynamic properties of CBR1 isoforms [[31](#prp270114-bib-0031)], was not detected in our cohort (MAF = 0, which is consistent with its frequency within South Asian populations) [[28](#prp270114-bib-0028)].

Nevertheless, the current study has shed more light on the significance of incorporating all *CBR1* polymorphisms in the PGx marker panel for HCC patients who are candidates for DOX therapy, though replication of association findings in larger, diverse ethnic studies is still needed before their implementation can be recommended for routine practice.

### 4.3. Strengths and Limitations

As one of the PGx studies in a continuously evolving oncology research area, our study has certain strengths and limitations. First, the current study provided novel evidence for the relative significance of four genes directly involved in DOX PK on survival outcomes and effectiveness in a very rarely studied cancer group, HCC patients with tumors deemed unresectable or noncandidate for radiofrequency ablation. All previous PGx studies involving HCC patients, as identified in the literature, were concerned with systemic therapy, sorafenib being the last choice for advanced‐stage HCC patients [[76](#prp270114-bib-0076), [77](#prp270114-bib-0077), [78](#prp270114-bib-0078), [79](#prp270114-bib-0079), [80](#prp270114-bib-0080)]. Only one study involving DOX via the TACE procedure in HCC has been conducted; however, that study focused on one gene SNP and was limited to a certain ethnic group [[47](#prp270114-bib-0047)]. Second, the employment of a fast microarray technique (including up‐to‐date clinically relevant content with all PharmGKB markers) enabled the genotyping of all known variants within the four genes, specifically with more accurate metabolic or transport capacity‐based phenotyping of the nomenclature of the three gene star alleles (*CYP2B6, GSTP1, and SLCO1B1*). In particular, this point may strengthen the findings obtained in the present analyses of their relevant associations with the outcomes of localized DOX therapy in patients with HCC. Previous PGx studies of DOX in other cancer types [[22](#prp270114-bib-0022), [27](#prp270114-bib-0027), [32](#prp270114-bib-0032), [33](#prp270114-bib-0033), [34](#prp270114-bib-0034), [41](#prp270114-bib-0041), [42](#prp270114-bib-0042), [44](#prp270114-bib-0044), [45](#prp270114-bib-0045), [71](#prp270114-bib-0071)] mostly targeted a few star alleles based on the estimated frequencies in related populations, which could have missed the examination of certain minor star alleles and their impacts on the global estimation of an individual metabolic or transport phenotype for DOX. Third, this is the first study to examine and provide preliminary evidence for the potential association between *CBR1* genetic variants and survival outcomes after DOX therapy in HCC patients. Fourth, the prospective design with an extended follow‐up period employing both radiological and biological marker (AFP) assessments enabled the capture of more robust endpoints, such as tumor progression and death.

However, our study has several limitations. First, the modest sample size and its confinement to the Arabic ethnicity could have limited the power of detecting the significant association of star allele‐predicted phenotypes with DOX effectiveness and may limit extrapolation to a wider HCC population with diverse ethnogeographic origins.

Second, as a single gene may not be adequate to elucidate its influence on xenobiotic effectiveness, other gene polymorphisms known to be involved in DOX PK (such as *ABCB1 [efflux], CBR3, AKR1A1*, and *AKR1C3* [all three metabolize DOX to doxorubicinol]) or the DOX PD pathway (such as DNA repair genes) could have additive roles in OS and PFS and warrant further investigation in future work. However, these genes were reported to have less specific expression in hepatocytes [[81](#prp270114-bib-0081)]. Third, consistent with previous epidemiological studies [[82](#prp270114-bib-0082)] and few PGx studies in patients with HCC receiving sorafenib [[47](#prp270114-bib-0047), [80](#prp270114-bib-0080), [83](#prp270114-bib-0083)], the combined analysis in our study revealed that nongenetic prognostic factors such as metastatic disease extension, hepatitis B etiology, presence of macroscopic vascular invasion, or ascites appeared more powerful in predicting and modulating the impact of doxorubicin on OS and PFS than genotypic variants (within two *CRB1* SNPs in our patients). This finding could undermine the significance of PGx marker germline genotyping in a subset of HCC patients and may comprehensively indicate the ineffectiveness of DOX locoregional therapy in HCC patients with these clinical features. These findings require replication in larger studies and could be of interest as a target for our future exploratory multivariant somatic genomic testing research in HCC.

## 5. Conclusions

The present study provides the first evidence for the predictive value of two *CBR1* SNPs in modulating survival outcomes after DOX therapy via TACE in HCC patients.

In agreement with the findings of previous studies, these variants of the *CBR1* gene appear to be associated with higher CBR1 protein levels and metabolic activity and, accordingly, with a higher clearance rate of DOX with faster conversion to its more toxic but less antineoplastic metabolite doxorubicinol. We hypothesize that high levels of doxorubicinol‐induced toxicity are associated with reduced OS and PFS. Despite our view of the potential value of implementing *CRB1* polymorphism genotyping in real practice given its practicality and lower cost in comparison to gene expression analyses, the prognostic significance of *CRB1* polymorphisms was limited to a subset of HCC patients in the present cohort who are devoid of known metastatic risk factors. This point of view suggests that further extensive exploratory PGx studies on a wider set of HCC clinical factors and targeting more germline and somatic genes are still needed to replicate our findings or reveal other DOX‐related PK and PD pathway markers that could have additive predictive value for improving OS or PFS in HCC patients.

## Author Contributions

**Sireen Abdul Rahim Shilbayeh, Rehab H. Werida:** conceptualization, methodology. **Mohammad A. Alshabeeb, Abdalrhman Hamdan Alanizi, Naglaa F. Khedr:** validation. **Sireen Abdul Rahim Shilbayeh, Omnia A. Abd El‐Baset, Naglaa F. Khedr:** laboratory work. **Mohammad A. Alshabeeb, Rehab H. Werida, Sireen Abdul Rahim Shilbayeh:** formal analysis. **Omnia A. Abd El‐Baset, Abdalrhman Hamdan Alanizi, Mohammad A. Alshabeeb:** recruitment and clinical investigation. **Sireen Abdul Rahim Shilbayeh:** resources. **Omnia A. Abd El‐Baset, Rehab H. Werida and Sireen Abdul Rahim Shilbayeh:** data curation. **Sireen Abdul Rahim Shilbayeh, Naglaa F. Khedr, Rehab H. Werida:** writing – original draft preparation. All authors: writing – review and editing. **Mohammad A. Alshabeeb:** visualization. **Sireen Abdul Rahim Shilbayeh:** supervision, project administration, funding acquisition. All authors revised and approved the final manuscript.

## Conflicts of Interest

The authors declare no conflicts of interest.

## Supporting information

## Acknowledgments

The authors have nothing to report.

## Data Availability Statement

The datasets supporting the conclusions of this article are included within the article and its additional files (Tables [S1](#prp270114-supitem-0001) and [S2](#prp270114-supitem-0001)).

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

The datasets supporting the conclusions of this article are included within the article and its additional files (Tables [S1](#prp270114-supitem-0001) and [S2](#prp270114-supitem-0001)).

### Supplementary Materials

### Data Availability Statement

The datasets supporting the conclusions of this article are included within the article and its additional files (Tables [S1](#prp270114-supitem-0001) and [S2](#prp270114-supitem-0001)).

## References

1. Rumgay H., Ferlay J., De Martel C., et al., “Global, Regional and National Burden of Primary Liver Cancer by Subtype,” European Journal of Cancer 161 (2022): 108–118, 10.1016/j.ejca.2021.11.023.  [DOI](https://doi.org/10.1016/j.ejca.2021.11.023) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34942552/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=European%20Journal%20of%20Cancer&title=Global,%20Regional%20and%20National%20Burden%20of%20Primary%20Liver%20Cancer%20by%20Subtype&author=H.%20Rumgay&author=J.%20Ferlay&author=C.%20De%20Martel&volume=161&publication_year=2022&pages=108-118&pmid=34942552&doi=10.1016/j.ejca.2021.11.023&)

2. Abudeif A., “Epidemiology and Risk Factors of Hepatocellular Carcinoma in Egypt,” Sohag Medical Journal 23 (2019): 8–12, 10.21608/smj.2019.13376.1019.  [DOI](https://doi.org/10.21608/smj.2019.13376.1019) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sohag%20Medical%20Journal&title=Epidemiology%20and%20Risk%20Factors%20of%20Hepatocellular%20Carcinoma%20in%20Egypt&author=A.%20Abudeif&volume=23&publication_year=2019&pages=8-12&doi=10.21608/smj.2019.13376.1019&)

3. Althubiti M. and Alfayez M., “Insights on Hepatocellular Carcinoma in Saudi Arabia,” in Liver Cancer in the Middle East, ed. Carr B. I. (Springer International Publishing, 2021), 247–257.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Liver%20Cancer%20in%20the%20Middle%20East&author=M.%20Althubiti&author=M.%20Alfayez&publication_year=2021&)

4. El‐Kassas M. and Elbadry M., “Hepatocellular Carcinoma in Africa: Challenges and Opportunities,” Frontiers in Medicine 9 (2022): 899420, 10.3389/fmed.2022.899420.  [DOI](https://doi.org/10.3389/fmed.2022.899420) | [PMC free article](/articles/PMC9263092/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35814750/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Medicine&title=Hepatocellular%20Carcinoma%20in%20Africa:%20Challenges%20and%20Opportunities&author=M.%20El%E2%80%90Kassas&author=M.%20Elbadry&volume=9&publication_year=2022&pages=899420&pmid=35814750&doi=10.3389/fmed.2022.899420&)

5. Hashim M. J., Rizvi S. S., and Khan G., “Hepatocellular Carcinoma in The United Arab Emirates,” in Liver Cancer in the Middle East, ed. Carr B. I. (Springer International Publishing, 2021), 101–108.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Liver%20Cancer%20in%20the%20Middle%20East&author=M.%20J.%20Hashim&author=S.%20S.%20Rizvi&author=G.%20Khan&publication_year=2021&)

6. Singal A. G., Llovet J. M., Yarchoan M., et al., “AASLD Practice Guidance on Prevention, Diagnosis, and Treatment of Hepatocellular Carcinoma,” Hepatology 78 (2023): 1922–1965, 10.1097/hep.0000000000000466.  [DOI](https://doi.org/10.1097/hep.0000000000000466) | [PMC free article](/articles/PMC10663390/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/37199193/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hepatology&title=AASLD%20Practice%20Guidance%20on%20Prevention,%20Diagnosis,%20and%20Treatment%20of%20Hepatocellular%20Carcinoma&author=A.%20G.%20Singal&author=J.%20M.%20Llovet&author=M.%20Yarchoan&volume=78&publication_year=2023&pages=1922-1965&pmid=37199193&doi=10.1097/hep.0000000000000466&)

7. Bessar A. A., Farag A., Monem S. M. A., et al., “Transarterial Chemoembolisation in Patients With Hepatocellular Carcinoma: Low‐Dose Doxorubicin Reduces Post‐Embolisation Syndrome Without Affecting Survival‐Prospective Interventional Study,” European Radiology Experimental 5, no. 1 (2021): 10, 10.1186/s41747-021-00204-6.  [DOI](https://doi.org/10.1186/s41747-021-00204-6) | [PMC free article](/articles/PMC7921261/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33649930/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=European%20Radiology%20Experimental&title=Transarterial%20Chemoembolisation%20in%20Patients%20With%20Hepatocellular%20Carcinoma:%20Low%E2%80%90Dose%20Doxorubicin%20Reduces%20Post%E2%80%90Embolisation%20Syndrome%20Without%20Affecting%20Survival%E2%80%90Prospective%20Interventional%20Study&author=A.%20A.%20Bessar&author=A.%20Farag&author=S.%20M.%20A.%20Monem&volume=5&issue=1&publication_year=2021&pages=10&pmid=33649930&doi=10.1186/s41747-021-00204-6&)

8. Lee C. W., Tsai H. I., Lee W. C., et al., “Normal Alpha‐Fetoprotein Hepatocellular Carcinoma: Are They Really Normal?,” Journal of Clinical Medicine 8 (2019): 1736, 10.3390/jcm8101736.  [DOI](https://doi.org/10.3390/jcm8101736) | [PMC free article](/articles/PMC6832124/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31635078/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Clinical%20Medicine&title=Normal%20Alpha%E2%80%90Fetoprotein%20Hepatocellular%20Carcinoma:%20Are%20They%20Really%20Normal?&author=C.%20W.%20Lee&author=H.%20I.%20Tsai&author=W.%20C.%20Lee&volume=8&publication_year=2019&pages=1736&pmid=31635078&doi=10.3390/jcm8101736&)

9. Mohan P. B., Lochan R., and Shetty S., “Biomarker in Hepatocellular Carcinoma,” Indian Journal of Surgical Oncology 15, no. S2 (2024): 261–268, 10.1007/s13193-023-01858-x.  [DOI](https://doi.org/10.1007/s13193-023-01858-x) | [PMC free article](/articles/PMC11133295/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/38817995/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Indian%20Journal%20of%20Surgical%20Oncology&title=Biomarker%20in%20Hepatocellular%20Carcinoma&author=P.%20B.%20Mohan&author=R.%20Lochan&author=S.%20Shetty&volume=15&issue=S2&publication_year=2024&pages=261-268&pmid=38817995&doi=10.1007/s13193-023-01858-x&)

10. Tam K., “The Roles of Doxorubicin in Hepatocellular Carcinoma,” ADMET DMPK 1, no. 3 (2013): 29–44, 10.5599/admet.1.3.7.  [DOI](https://doi.org/10.5599/admet.1.3.7) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=ADMET%20DMPK&title=The%20Roles%20of%20Doxorubicin%20in%20Hepatocellular%20Carcinoma&author=K.%20Tam&volume=1&issue=3&publication_year=2013&pages=29-44&doi=10.5599/admet.1.3.7&)

11. Fan Y. P., Liao J. Z., Lu Y. Q., et al., “MiR‐375 and Doxorubicin Co‐Delivered by Liposomes for Combination Therapy of Hepatocellular Carcinoma,” Molecular Therapy ‐ Nucleic Acids 7 (2017): 181–189, 10.1016/j.omtn.2017.03.010.  [DOI](https://doi.org/10.1016/j.omtn.2017.03.010) | [PMC free article](/articles/PMC5415965/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28624193/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Molecular%20Therapy%20%E2%80%90%20Nucleic%20Acids&title=MiR%E2%80%90375%20and%20Doxorubicin%20Co%E2%80%90Delivered%20by%20Liposomes%20for%20Combination%20Therapy%20of%20Hepatocellular%20Carcinoma&author=Y.%20P.%20Fan&author=J.%20Z.%20Liao&author=Y.%20Q.%20Lu&volume=7&publication_year=2017&pages=181-189&pmid=28624193&doi=10.1016/j.omtn.2017.03.010&)

12. Feng F., Jiang Q., Jia H., et al., “Which Is the Best Combination of TACE and Sorafenib for Advanced Hepatocellular Carcinoma Treatment? A Systematic Review and Network Meta‐Analysis,” Pharmacological Research 135 (2018): 89–101, 10.1016/j.phrs.2018.06.021.  [DOI](https://doi.org/10.1016/j.phrs.2018.06.021) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29959032/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacological%20Research&title=Which%20Is%20the%20Best%20Combination%20of%20TACE%20and%20Sorafenib%20for%20Advanced%20Hepatocellular%20Carcinoma%20Treatment?%20A%20Systematic%20Review%20and%20Network%20Meta%E2%80%90Analysis&author=F.%20Feng&author=Q.%20Jiang&author=H.%20Jia&volume=135&publication_year=2018&pages=89-101&pmid=29959032&doi=10.1016/j.phrs.2018.06.021&)

13. Liu B., Huang J. W., Li Y., et al., “Single‐Agent Versus Combination Doxorubicin‐Based Transarterial Chemoembolization in the Treatment of Hepatocellular Carcinoma: A Single‐Blind, Randomized, Phase II Trial,” Oncology 89 (2015): 23–30, 10.1159/000371522.  [DOI](https://doi.org/10.1159/000371522) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25613214/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncology&title=Single%E2%80%90Agent%20Versus%20Combination%20Doxorubicin%E2%80%90Based%20Transarterial%20Chemoembolization%20in%20the%20Treatment%20of%20Hepatocellular%20Carcinoma:%20A%20Single%E2%80%90Blind,%20Randomized,%20Phase%20II%20Trial&author=B.%20Liu&author=J.%20W.%20Huang&author=Y.%20Li&volume=89&publication_year=2015&pages=23-30&pmid=25613214&doi=10.1159/000371522&)

14. Kciuk M., Gielecińska A., Mujwar S., et al., “Doxorubicin‐An Agent With Multiple Mechanisms of Anticancer Activity,” Cells 12 (2023): 659, 10.3390/cells12040659.  [DOI](https://doi.org/10.3390/cells12040659) | [PMC free article](/articles/PMC9954613/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36831326/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cells&title=Doxorubicin%E2%80%90An%20Agent%20With%20Multiple%20Mechanisms%20of%20Anticancer%20Activity&author=M.%20Kciuk&author=A.%20Gieleci%C5%84ska&author=S.%20Mujwar&volume=12&publication_year=2023&pages=659&pmid=36831326&doi=10.3390/cells12040659&)

15. Thorn C. F., Oshiro C., Marsh S., et al., “Doxorubicin Pathways: Pharmacodynamics and Adverse Effects,” Pharmacogenetics and Genomics 21, no. 7 (2011): 440–446, 10.1097/FPC.0b013e32833ffb56.  [DOI](https://doi.org/10.1097/FPC.0b013e32833ffb56) | [PMC free article](/articles/PMC3116111/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21048526/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics%20and%20Genomics&title=Doxorubicin%20Pathways:%20Pharmacodynamics%20and%20Adverse%20Effects&author=C.%20F.%20Thorn&author=C.%20Oshiro&author=S.%20Marsh&volume=21&issue=7&publication_year=2011&pages=440-446&pmid=21048526&doi=10.1097/FPC.0b013e32833ffb56&)

16. Kassner N., Huse K., Martin H. J., et al., “Carbonyl Reductase 1 Is a Predominant Doxorubicin Reductase in the Human Liver,” Drug Metabolism and Disposition 36 (2008): 2113–2120, 10.1124/dmd.108.022251.  [DOI](https://doi.org/10.1124/dmd.108.022251) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18635746/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metabolism%20and%20Disposition&title=Carbonyl%20Reductase%201%20Is%20a%20Predominant%20Doxorubicin%20Reductase%20in%20the%20Human%20Liver&author=N.%20Kassner&author=K.%20Huse&author=H.%20J.%20Martin&volume=36&publication_year=2008&pages=2113-2120&pmid=18635746&doi=10.1124/dmd.108.022251&)

17. Thorn C. F., Klein T. E., and Altman R. B., “PharmGKB: The Pharmacogenomics Knowledge Base,” Methods in Molecular Biology 1015 (2013): 311–320, 10.1007/978-1-62703-435-7_20.  [DOI](https://doi.org/10.1007/978-1-62703-435-7_20) | [PMC free article](/articles/PMC4084821/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23824865/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Methods%20in%20Molecular%20Biology&title=PharmGKB:%20The%20Pharmacogenomics%20Knowledge%20Base&author=C.%20F.%20Thorn&author=T.%20E.%20Klein&author=R.%20B.%20Altman&volume=1015&publication_year=2013&pages=311-320&pmid=23824865&doi=10.1007/978-1-62703-435-7_20&)

18. Volkova M. and Russell R., “Anthracycline Cardiotoxicity: Prevalence, Pathogenesis and Treatment,” Current Cardiology Reviews 7 (2011): 214–220, 10.2174/157340311799960645.  [DOI](https://doi.org/10.2174/157340311799960645) | [PMC free article](/articles/PMC3322439/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22758622/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Current%20Cardiology%20Reviews&title=Anthracycline%20Cardiotoxicity:%20Prevalence,%20Pathogenesis%20and%20Treatment&author=M.%20Volkova&author=R.%20Russell&volume=7&publication_year=2011&pages=214-220&pmid=22758622&doi=10.2174/157340311799960645&)

19. Langmia I. M., Just K. S., Yamoune S., Brockmöller J., Masimirembwa C., and Stingl J. C., “CYP2B6 Functional Variability in Drug Metabolism and Exposure Across Populations‐Implication for Drug Safety, Dosing, and Individualized Therapy,” Frontiers in Genetics 12 (2021): 692234, 10.3389/fgene.2021.692234.  [DOI](https://doi.org/10.3389/fgene.2021.692234) | [PMC free article](/articles/PMC8313315/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34322158/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Genetics&title=CYP2B6%20Functional%20Variability%20in%20Drug%20Metabolism%20and%20Exposure%20Across%20Populations%E2%80%90Implication%20for%20Drug%20Safety,%20Dosing,%20and%20Individualized%20Therapy&author=I.%20M.%20Langmia&author=K.%20S.%20Just&author=S.%20Yamoune&author=J.%20Brockm%C3%B6ller&author=C.%20Masimirembwa&volume=12&publication_year=2021&pages=692234&pmid=34322158&doi=10.3389/fgene.2021.692234&)

20. Cui J., Li G., Yin J., et al., “GSTP1 and Cancer: Expression, Methylation, Polymorphisms and Signaling (Review),” International Journal of Oncology 56 (2020): 867–878, 10.3892/ijo.2020.4979.  [DOI](https://doi.org/10.3892/ijo.2020.4979) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32319549/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=International%20Journal%20of%20Oncology&title=GSTP1%20and%20Cancer:%20Expression,%20Methylation,%20Polymorphisms%20and%20Signaling%20(Review)&author=J.%20Cui&author=G.%20Li&author=J.%20Yin&volume=56&publication_year=2020&pages=867-878&pmid=32319549&doi=10.3892/ijo.2020.4979&)

21. Ma J., Zhu S. L., Liu Y., Huang X. Y., and Su D. K., “GSTP1 Polymorphism Predicts Treatment Outcome and Toxicities for Breast Cancer,” Oncotarget 8 (2017): 72939–72949, 10.18632/oncotarget.18513.  [DOI](https://doi.org/10.18632/oncotarget.18513) | [PMC free article](/articles/PMC5641181/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29069838/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncotarget&title=GSTP1%20Polymorphism%20Predicts%20Treatment%20Outcome%20and%20Toxicities%20for%20Breast%20Cancer&author=J.%20Ma&author=S.%20L.%20Zhu&author=Y.%20Liu&author=X.%20Y.%20Huang&author=D.%20K.%20Su&volume=8&publication_year=2017&pages=72939-72949&pmid=29069838&doi=10.18632/oncotarget.18513&)

22. Faraji A., Manshadi H. R. D., Mobaraki M., Zare M., and Houshmand M., “Association of ABCB1 and SLC22A16 Gene Polymorphisms With Incidence of Doxorubicin‐Induced Febrile Neutropenia: A Survey of Iranian Breast Cancer Patients,” PLoS One 11 (2016): e0168519, 10.1371/journal.pone.0168519.  [DOI](https://doi.org/10.1371/journal.pone.0168519) | [PMC free article](/articles/PMC5201260/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28036387/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Association%20of%20ABCB1%20and%20SLC22A16%20Gene%20Polymorphisms%20With%20Incidence%20of%20Doxorubicin%E2%80%90Induced%20Febrile%20Neutropenia:%20A%20Survey%20of%20Iranian%20Breast%20Cancer%20Patients&author=A.%20Faraji&author=H.%20R.%20D.%20Manshadi&author=M.%20Mobaraki&author=M.%20Zare&author=M.%20Houshmand&volume=11&publication_year=2016&pages=e0168519&pmid=28036387&doi=10.1371/journal.pone.0168519&)

23. Wang X., Zhang H., and Chen X., “Drug Resistance and Combating Drug Resistance in Cancer,” Cancer Drug Resistance (Alhambra, Calif.) 2 (2019): 141–160, 10.20517/cdr.2019.10.  [DOI](https://doi.org/10.20517/cdr.2019.10) | [PMC free article](/articles/PMC8315569/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34322663/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Drug%20Resistance%20(Alhambra,%20Calif.)&title=Drug%20Resistance%20and%20Combating%20Drug%20Resistance%20in%20Cancer&author=X.%20Wang&author=H.%20Zhang&author=X.%20Chen&volume=2&publication_year=2019&pages=141-160&pmid=34322663&doi=10.20517/cdr.2019.10&)

24. Bagdasaryan A. A., Chubarev V. N., Smolyarchuk E. A., et al., “Pharmacogenetics of Drug Metabolism: The Role of Gene Polymorphism in the Regulation of Doxorubicin Safety and Efficacy,” Cancers (Basel) 14, no. 21 (2022): 5436, 10.3390/cancers14215436.  [DOI](https://doi.org/10.3390/cancers14215436) | [PMC free article](/articles/PMC9659104/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36358854/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancers%20(Basel)&title=Pharmacogenetics%20of%20Drug%20Metabolism:%20The%20Role%20of%20Gene%20Polymorphism%20in%20the%20Regulation%20of%20Doxorubicin%20Safety%20and%20Efficacy&author=A.%20A.%20Bagdasaryan&author=V.%20N.%20Chubarev&author=E.%20A.%20Smolyarchuk&volume=14&issue=21&publication_year=2022&pages=5436&pmid=36358854&doi=10.3390/cancers14215436&)

25. Lal S., Wong Z. W., Jada S. R., et al., “Novel SLC22A16 Polymorphisms and Influence on Doxorubicin Pharmacokinetics in Asian Breast Cancer Patients,” Pharmacogenomics 8, no. 6 (2007): 567–575, 10.2217/14622416.8.6.567.  [DOI](https://doi.org/10.2217/14622416.8.6.567) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17559346/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Novel%20SLC22A16%20Polymorphisms%20and%20Influence%20on%20Doxorubicin%20Pharmacokinetics%20in%20Asian%20Breast%20Cancer%20Patients&author=S.%20Lal&author=Z.%20W.%20Wong&author=S.%20R.%20Jada&volume=8&issue=6&publication_year=2007&pages=567-575&pmid=17559346&doi=10.2217/14622416.8.6.567&)

26. Mordente A., Minotti G., Martorana G. E., Silvestrini A., Giardina B., and Meucci E., “Anthracycline Secondary Alcohol Metabolite Formation in Human or Rabbit Heart: Biochemical Aspects and Pharmacologic Implications,” Biochemical Pharmacology 66 (2003): 989–998, 10.1016/s0006-2952(03)00442-8.  [DOI](https://doi.org/10.1016/s0006-2952(03)00442-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12963485/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochemical%20Pharmacology&title=Anthracycline%20Secondary%20Alcohol%20Metabolite%20Formation%20in%20Human%20or%20Rabbit%20Heart:%20Biochemical%20Aspects%20and%20Pharmacologic%20Implications&author=A.%20Mordente&author=G.%20Minotti&author=G.%20E.%20Martorana&author=A.%20Silvestrini&author=B.%20Giardina&volume=66&publication_year=2003&pages=989-998&pmid=12963485&doi=10.1016/s0006-2952(03)00442-8&)

27. Tecza K., Pamula‐Pilat J., Lanuszewska J., Butkiewicz D., and Grzybowska E., “Pharmacogenetics of Toxicity of 5‐Fluorouracil, Doxorubicin and Cyclophosphamide Chemotherapy in Breast Cancer Patients,” Oncotarget 9 (2018): 9114–9136, 10.18632/oncotarget.24148.  [DOI](https://doi.org/10.18632/oncotarget.24148) | [PMC free article](/articles/PMC5823653/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29507678/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncotarget&title=Pharmacogenetics%20of%20Toxicity%20of%205%E2%80%90Fluorouracil,%20Doxorubicin%20and%20Cyclophosphamide%20Chemotherapy%20in%20Breast%20Cancer%20Patients&author=K.%20Tecza&author=J.%20Pamula%E2%80%90Pilat&author=J.%20Lanuszewska&author=D.%20Butkiewicz&author=E.%20Grzybowska&volume=9&publication_year=2018&pages=9114-9136&pmid=29507678&doi=10.18632/oncotarget.24148&)

28. Lal S., Sandanaraj E., Wong Z. W., et al., “CBR1 and CBR3 Pharmacogenetics and Their Influence on Doxorubicin Disposition in Asian Breast Cancer Patients,” Cancer Science 99, no. 10 (2008): 2045–2054, 10.1111/j.1349-7006.2008.00903.x.  [DOI](https://doi.org/10.1111/j.1349-7006.2008.00903.x) | [PMC free article](/articles/PMC11160041/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19016765/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Science&title=CBR1%20and%20CBR3%20Pharmacogenetics%20and%20Their%20Influence%20on%20Doxorubicin%20Disposition%20in%20Asian%20Breast%20Cancer%20Patients&author=S.%20Lal&author=E.%20Sandanaraj&author=Z.%20W.%20Wong&volume=99&issue=10&publication_year=2008&pages=2045-2054&pmid=19016765&doi=10.1111/j.1349-7006.2008.00903.x&)

29. Gonzalez‐Covarrubias V., Ghosh D., Lakhman S. S., Pendyala L., and Blanco J. G., “A Functional Genetic Polymorphism on Human Carbonyl Reductase 1 (CBR1 V88I) Impacts on Catalytic Activity and NADPH Binding Affinity,” Drug Metabolism and Disposition 35 (2007): 973–980, 10.1124/dmd.107.014779.  [DOI](https://doi.org/10.1124/dmd.107.014779) | [PMC free article](/articles/PMC2442771/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17344335/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metabolism%20and%20Disposition&title=A%20Functional%20Genetic%20Polymorphism%20on%20Human%20Carbonyl%20Reductase%201%20(CBR1%20V88I)%20Impacts%20on%20Catalytic%20Activity%20and%20NADPH%20Binding%20Affinity&author=V.%20Gonzalez%E2%80%90Covarrubias&author=D.%20Ghosh&author=S.%20S.%20Lakhman&author=L.%20Pendyala&author=J.%20G.%20Blanco&volume=35&publication_year=2007&pages=973-980&pmid=17344335&doi=10.1124/dmd.107.014779&)

30. Gonzalez‐Covarrubias V., Zhang J., Kalabus J. L., Relling M. V., and Blanco J. G., “Pharmacogenetics of Human Carbonyl Reductase 1 (CBR1) in Livers From Black and White Donors,” Drug Metabolism and Disposition 37 (2009): 400–407, 10.1124/dmd.108.024547.  [DOI](https://doi.org/10.1124/dmd.108.024547) | [PMC free article](/articles/PMC2680526/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19022938/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metabolism%20and%20Disposition&title=Pharmacogenetics%20of%20Human%20Carbonyl%20Reductase%201%20(CBR1)%20in%20Livers%20From%20Black%20and%20White%20Donors&author=V.%20Gonzalez%E2%80%90Covarrubias&author=J.%20Zhang&author=J.%20L.%20Kalabus&author=M.%20V.%20Relling&author=J.%20G.%20Blanco&volume=37&publication_year=2009&pages=400-407&pmid=19022938&doi=10.1124/dmd.108.024547&)

31. Bains O. S., Karkling M. J., Grigliatti T. A., Reid R. E., and Riggs K. W., “Two Nonsynonymous Single Nucleotide Polymorphisms of Human Carbonyl Reductase 1 Demonstrate Reduced In Vitro Metabolism of Daunorubicin and Doxorubicin,” Drug Metabolism and Disposition 37 (2009): 1107–1114, 10.1124/dmd.108.024711.  [DOI](https://doi.org/10.1124/dmd.108.024711) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19204081/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metabolism%20and%20Disposition&title=Two%20Nonsynonymous%20Single%20Nucleotide%20Polymorphisms%20of%20Human%20Carbonyl%20Reductase%201%20Demonstrate%20Reduced%20In%C2%A0Vitro%20Metabolism%20of%20Daunorubicin%20and%20Doxorubicin&author=O.%20S.%20Bains&author=M.%20J.%20Karkling&author=T.%20A.%20Grigliatti&author=R.%20E.%20Reid&author=K.%20W.%20Riggs&volume=37&publication_year=2009&pages=1107-1114&pmid=19204081&doi=10.1124/dmd.108.024711&)

32. Abo‐Bakr A., Mossallam G., El Azhary N., Hafez H., and Badawy R., “Impact of CYP1A1, GSTP1 and XRCC1 Genes Polymorphisms on Toxicity and Response to Chemotherapy in Childhood Acute Lymphoblastic Leukemia,” Journal of the Egyptian National Cancer Institute 29 (2017): 127–133, 10.1016/j.jnci.2017.07.002.  [DOI](https://doi.org/10.1016/j.jnci.2017.07.002) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28844589/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20the%20Egyptian%20National%20Cancer%20Institute&title=Impact%20of%20CYP1A1,%20GSTP1%20and%20XRCC1%20Genes%20Polymorphisms%20on%20Toxicity%20and%20Response%20to%20Chemotherapy%20in%20Childhood%20Acute%20Lymphoblastic%20Leukemia&author=A.%20Abo%E2%80%90Bakr&author=G.%20Mossallam&author=N.%20El%20Azhary&author=H.%20Hafez&author=R.%20Badawy&volume=29&publication_year=2017&pages=127-133&pmid=28844589&doi=10.1016/j.jnci.2017.07.002&)

33. Bray J., Sludden J., Griffin M. J., et al., “Influence of Pharmacogenetics on Response and Toxicity in Breast Cancer Patients Treated With Doxorubicin and Cyclophosphamide,” British Journal of Cancer 102 (2010): 1003–1009, 10.1038/sj.bjc.6605587.  [DOI](https://doi.org/10.1038/sj.bjc.6605587) | [PMC free article](/articles/PMC2844036/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20179710/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=British%20Journal%20of%20Cancer&title=Influence%20of%20Pharmacogenetics%20on%20Response%20and%20Toxicity%20in%20Breast%20Cancer%20Patients%20Treated%20With%20Doxorubicin%20and%20Cyclophosphamide&author=J.%20Bray&author=J.%20Sludden&author=M.%20J.%20Griffin&volume=102&publication_year=2010&pages=1003-1009&pmid=20179710&doi=10.1038/sj.bjc.6605587&)

34. Brooks J. D., Teraoka S. N., Bernstein L., et al., “Common Variants in Genes Coding for Chemotherapy Metabolizing Enzymes, Transporters, and Targets: A Case‐Control Study of Contralateral Breast Cancer Risk in the WECARE Study,” Cancer Causes & Control 24, no. 8 (2013): 1605–1614, 10.1007/s10552-013-0237-6.  [DOI](https://doi.org/10.1007/s10552-013-0237-6) | [PMC free article](/articles/PMC3709075/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23775025/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Causes%20&%20Control&title=Common%20Variants%20in%20Genes%20Coding%20for%20Chemotherapy%20Metabolizing%20Enzymes,%20Transporters,%20and%20Targets:%20A%20Case%E2%80%90Control%20Study%20of%20Contralateral%20Breast%20Cancer%20Risk%20in%20the%20WECARE%20Study&author=J.%20D.%20Brooks&author=S.%20N.%20Teraoka&author=L.%20Bernstein&volume=24&issue=8&publication_year=2013&pages=1605-1614&pmid=23775025&doi=10.1007/s10552-013-0237-6&)

35. Fan L., Goh B. C., Wong C. I., et al., “Genotype of Human Carbonyl Reductase CBR3 Correlates With Doxorubicin Disposition and Toxicity,” Pharmacogenetics and Genomics 18, no. 7 (2008): 621–631, 10.1097/FPC.0b013e328301a869.  [DOI](https://doi.org/10.1097/FPC.0b013e328301a869) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18551042/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics%20and%20Genomics&title=Genotype%20of%20Human%20Carbonyl%20Reductase%20CBR3%20Correlates%20With%20Doxorubicin%20Disposition%20and%20Toxicity&author=L.%20Fan&author=B.%20C.%20Goh&author=C.%20I.%20Wong&volume=18&issue=7&publication_year=2008&pages=621-631&pmid=18551042&doi=10.1097/FPC.0b013e328301a869&)

36. Gándara‐Mireles J. A., Lares‐Asseff I., Espinoza E. A. R., et al., “Association of Genetic Polymorphisms NCF4 rs1883112, CBR3 rs1056892, and ABCC1 rs3743527 With the Cardiotoxic Effects of Doxorubicin in Children With Acute Lymphoblastic Leukemia,” Pharmacogenetics and Genomics 31, no. 5 (2021): 108–115, 10.1097/fpc.0000000000000428.  [DOI](https://doi.org/10.1097/fpc.0000000000000428) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34096893/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics%20and%20Genomics&title=Association%20of%20Genetic%20Polymorphisms%20NCF4%20rs1883112,%20CBR3%20rs1056892,%20and%20ABCC1%20rs3743527%20With%20the%20Cardiotoxic%20Effects%20of%20Doxorubicin%20in%20Children%20With%20Acute%20Lymphoblastic%20Leukemia&author=J.%20A.%20G%C3%A1ndara%E2%80%90Mireles&author=I.%20Lares%E2%80%90Asseff&author=E.%20A.%20R.%20Espinoza&volume=31&issue=5&publication_year=2021&pages=108-115&pmid=34096893&doi=10.1097/fpc.0000000000000428&)

37. Gelderblom H., Blay J. Y., Seddon B. M., et al., “Brostallicin Versus Doxorubicin as First‐Line Chemotherapy in Patients With Advanced or Metastatic Soft Tissue Sarcoma: An European Organisation for Research and Treatment of Cancer Soft Tissue and Bone Sarcoma Group Randomised Phase II and Pharmacogenetic Study,” European Journal of Cancer 50 (2014): 388–396, 10.1016/j.ejca.2013.10.002.  [DOI](https://doi.org/10.1016/j.ejca.2013.10.002) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24215845/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=European%20Journal%20of%20Cancer&title=Brostallicin%20Versus%20Doxorubicin%20as%20First%E2%80%90Line%20Chemotherapy%20in%20Patients%20With%20Advanced%20or%20Metastatic%20Soft%20Tissue%20Sarcoma:%20An%20European%20Organisation%20for%20Research%20and%20Treatment%20of%20Cancer%20Soft%20Tissue%20and%20Bone%20Sarcoma%20Group%20Randomised%20Phase%20II%20and%20Pharmacogenetic%20Study&author=H.%20Gelderblom&author=J.%20Y.%20Blay&author=B.%20M.%20Seddon&volume=50&publication_year=2014&pages=388-396&pmid=24215845&doi=10.1016/j.ejca.2013.10.002&)

38. Hertz D. L., Caram M. V., Kidwell K. M., et al., “Evidence for Association of SNPs in ABCB1 and CBR3, but Not RAC2, NCF4, SLC28A3 or TOP2B, With Chronic Cardiotoxicity in a Cohort of Breast Cancer Patients Treated With Anthracyclines,” Pharmacogenomics 17 (2016): 231–240, 10.2217/pgs.15.162.  [DOI](https://doi.org/10.2217/pgs.15.162) | [PMC free article](/articles/PMC5558515/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26799497/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Evidence%20for%20Association%20of%20SNPs%20in%20ABCB1%20and%20CBR3,%20but%20Not%20RAC2,%20NCF4,%20SLC28A3%20or%20TOP2B,%20With%20Chronic%20Cardiotoxicity%20in%20a%20Cohort%20of%20Breast%20Cancer%20Patients%20Treated%20With%20Anthracyclines&author=D.%20L.%20Hertz&author=M.%20V.%20Caram&author=K.%20M.%20Kidwell&volume=17&publication_year=2016&pages=231-240&pmid=26799497&doi=10.2217/pgs.15.162&)

39. Kalra S., Kaur R. P., Ludhiadch A., et al., “Association of CYP2C19*2 and ALDH1A1*1/*2 Variants With Disease Outcome in Breast Cancer Patients: Results of a Global Screening Array,” European Journal of Clinical Pharmacology 74 (2018): 1291–1298, 10.1007/s00228-018-2505-6.  [DOI](https://doi.org/10.1007/s00228-018-2505-6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29938344/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=European%20Journal%20of%20Clinical%20Pharmacology&title=Association%20of%20CYP2C19*2%20and%20ALDH1A1*1/*2%20Variants%20With%20Disease%20Outcome%20in%20Breast%20Cancer%20Patients:%20Results%20of%20a%20Global%20Screening%20Array&author=S.%20Kalra&author=R.%20P.%20Kaur&author=A.%20Ludhiadch&volume=74&publication_year=2018&pages=1291-1298&pmid=29938344&doi=10.1007/s00228-018-2505-6&)

40. Lal S., Sutiman N., Ooi L. L., et al., “Pharmacogenetics of ABCB5, ABCC5 and RLIP76 and Doxorubicin Pharmacokinetics in Asian Breast Cancer Patients,” Pharmacogenomics Journal 17 (2017): 337–343, 10.1038/tpj.2016.17.  [DOI](https://doi.org/10.1038/tpj.2016.17) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26975227/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20Journal&title=Pharmacogenetics%20of%20ABCB5,%20ABCC5%20and%20RLIP76%20and%20Doxorubicin%20Pharmacokinetics%20in%20Asian%20Breast%20Cancer%20Patients&author=S.%20Lal&author=N.%20Sutiman&author=L.%20L.%20Ooi&volume=17&publication_year=2017&pages=337-343&pmid=26975227&doi=10.1038/tpj.2016.17&)

41. Nyangwara V. A., Mazhindu T., Chikwambi Z., et al., “Cardiotoxicity and Pharmacogenetics of Doxorubicin in Black Zimbabwean Breast Cancer Patients,” British Journal of Clinical Pharmacology 90, no. 8 (2023): 1782–1789, 10.1111/bcp.15659.  [DOI](https://doi.org/10.1111/bcp.15659) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36630266/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=British%20Journal%20of%20Clinical%20Pharmacology&title=Cardiotoxicity%20and%20Pharmacogenetics%20of%20Doxorubicin%20in%20Black%20Zimbabwean%20Breast%20Cancer%20Patients&author=V.%20A.%20Nyangwara&author=T.%20Mazhindu&author=Z.%20Chikwambi&volume=90&issue=8&publication_year=2023&pages=1782-1789&pmid=36630266&doi=10.1111/bcp.15659&)

42. Romero A., Martín M., Oliva B., et al., “Glutathione S‐Transferase P1 c.313A > G Polymorphism Could Be Useful in the Prediction of Doxorubicin Response in Breast Cancer Patients,” Annals of Oncology 23 (2012): 1750–1756, 10.1093/annonc/mdr483.  [DOI](https://doi.org/10.1093/annonc/mdr483) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22052985/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annals%20of%20Oncology&title=Glutathione%20S%E2%80%90Transferase%20P1%20c.313A%20>%20G%20Polymorphism%20Could%20Be%20Useful%20in%20the%20Prediction%20of%20Doxorubicin%20Response%20in%20Breast%20Cancer%20Patients&author=A.%20Romero&author=M.%20Mart%C3%ADn&author=B.%20Oliva&volume=23&publication_year=2012&pages=1750-1756&pmid=22052985&doi=10.1093/annonc/mdr483&)

43. Ruiz‐Pinto S., Martin M., Pita G., et al., “Pharmacogenetic Variants and Response to Neoadjuvant Single‐Agent Doxorubicin or Docetaxel: A Study in Locally Advanced Breast Cancer Patients Participating in the NCT00123929 Phase 2 Randomized Trial,” Pharmacogenetics and Genomics 28 (2018): 245–250, 10.1097/fpc.0000000000000354.  [NCT00123929](https://clinicaltrials.gov/ct2/show/NCT00123929) | [DOI](https://doi.org/10.1097/fpc.0000000000000354) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30334909/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics%20and%20Genomics&title=Pharmacogenetic%20Variants%20and%20Response%20to%20Neoadjuvant%20Single%E2%80%90Agent%20Doxorubicin%20or%20Docetaxel:%20A%20Study%20in%20Locally%20Advanced%20Breast%20Cancer%20Patients%20Participating%20in%20the%20NCT00123929%20Phase%202%20Randomized%20Trial&author=S.%20Ruiz%E2%80%90Pinto&author=M.%20Martin&author=G.%20Pita&volume=28&publication_year=2018&pages=245-250&pmid=30334909&doi=10.1097/fpc.0000000000000354&)

44. Tecza K., Pamula‐Pilat J., Lanuszewska J., and Grzybowska E., “Genetic Polymorphisms and Response to 5‐Fluorouracil, Doxorubicin and Cyclophosphamide Chemotherapy in Breast Cancer Patients,” Oncotarget 7 (2016): 66790–66808, 10.18632/oncotarget.11053.  [DOI](https://doi.org/10.18632/oncotarget.11053) | [PMC free article](/articles/PMC5341838/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27527855/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncotarget&title=Genetic%20Polymorphisms%20and%20Response%20to%205%E2%80%90Fluorouracil,%20Doxorubicin%20and%20Cyclophosphamide%20Chemotherapy%20in%20Breast%20Cancer%20Patients&author=K.%20Tecza&author=J.%20Pamula%E2%80%90Pilat&author=J.%20Lanuszewska&author=E.%20Grzybowska&volume=7&publication_year=2016&pages=66790-66808&pmid=27527855&doi=10.18632/oncotarget.11053&)

45. Windsor R. E., Strauss S. J., Kallis C., Wood N. E., and Whelan J. S., “Germline Genetic Polymorphisms May Influence Chemotherapy Response and Disease Outcome in Osteosarcoma: A Pilot Study,” Cancer 118 (2012): 1856–1867, 10.1002/cncr.26472.  [DOI](https://doi.org/10.1002/cncr.26472) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21887680/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer&title=Germline%20Genetic%20Polymorphisms%20May%20Influence%20Chemotherapy%20Response%20and%20Disease%20Outcome%20in%20Osteosarcoma:%20A%20Pilot%20Study&author=R.%20E.%20Windsor&author=S.%20J.%20Strauss&author=C.%20Kallis&author=N.%20E.%20Wood&author=J.%20S.%20Whelan&volume=118&publication_year=2012&pages=1856-1867&pmid=21887680&doi=10.1002/cncr.26472&)

46. Wojnowski L., Kulle B., Schirmer M., et al., “NAD(P)H Oxidase and Multidrug Resistance Protein Genetic Polymorphisms Are Associated With Doxorubicin‐Induced Cardiotoxicity,” Circulation 112 (2005): 3754–3762, 10.1161/circulationaha.105.576850.  [DOI](https://doi.org/10.1161/circulationaha.105.576850) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16330681/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation&title=NAD(P)H%20Oxidase%20and%20Multidrug%20Resistance%20Protein%20Genetic%20Polymorphisms%20Are%20Associated%20With%20Doxorubicin%E2%80%90Induced%20Cardiotoxicity&author=L.%20Wojnowski&author=B.%20Kulle&author=M.%20Schirmer&volume=112&publication_year=2005&pages=3754-3762&pmid=16330681&doi=10.1161/circulationaha.105.576850&)

47. Liang K. H., Lin C. L., Chen S. F., et al., “GALNT14 Genotype Effectively Predicts the Therapeutic Response in Unresectable Hepatocellular Carcinoma Treated With Transcatheter Arterial Chemoembolization,” Pharmacogenomics 17 (2016): 353–366, 10.2217/pgs.15.179.  [DOI](https://doi.org/10.2217/pgs.15.179) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26871639/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=GALNT14%20Genotype%20Effectively%20Predicts%20the%20Therapeutic%20Response%20in%20Unresectable%20Hepatocellular%20Carcinoma%20Treated%20With%20Transcatheter%20Arterial%20Chemoembolization&author=K.%20H.%20Liang&author=C.%20L.%20Lin&author=S.%20F.%20Chen&volume=17&publication_year=2016&pages=353-366&pmid=26871639&doi=10.2217/pgs.15.179&)

48. Harris P. A., Taylor R., Minor B. L., et al., “The REDCap Consortium: Building an International Community of Software Platform Partners,” Journal of Biomedical Informatics 95 (2019): 103208, 10.1016/j.jbi.2019.103208.  [DOI](https://doi.org/10.1016/j.jbi.2019.103208) | [PMC free article](/articles/PMC7254481/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31078660/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Biomedical%20Informatics&title=The%20REDCap%20Consortium:%20Building%20an%20International%20Community%20of%20Software%20Platform%20Partners&author=P.%20A.%20Harris&author=R.%20Taylor&author=B.%20L.%20Minor&volume=95&publication_year=2019&pages=103208&pmid=31078660&doi=10.1016/j.jbi.2019.103208&)

49. Lencioni R. and Llovet J. M., “Modified RECIST (mRECIST) Assessment for Hepatocellular Carcinoma,” Seminars in Liver Disease 30 (2010): 52–60, 10.1055/s-0030-1247132.  [DOI](https://doi.org/10.1055/s-0030-1247132) | [PMC free article](/articles/PMC12268942/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20175033/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Seminars%20in%20Liver%20Disease&title=Modified%20RECIST%20(mRECIST)%20Assessment%20for%20Hepatocellular%20Carcinoma&author=R.%20Lencioni&author=J.%20M.%20Llovet&volume=30&publication_year=2010&pages=52-60&pmid=20175033&doi=10.1055/s-0030-1247132&)

50. Bruix J., “Endpoints in Clinical Trials for Liver Cancer and Their Value in Evidence‐Based Clinical Decision Making: An Unresolved Gordian Knot,” Journal of Hepatology 74 (2021): 1483–1488, 10.1016/j.jhep.2021.01.033.  [DOI](https://doi.org/10.1016/j.jhep.2021.01.033) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33556420/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Hepatology&title=Endpoints%20in%20Clinical%20Trials%20for%20Liver%20Cancer%20and%20Their%20Value%20in%20Evidence%E2%80%90Based%20Clinical%20Decision%20Making:%20An%20Unresolved%20Gordian%20Knot&author=J.%20Bruix&volume=74&publication_year=2021&pages=1483-1488&pmid=33556420&doi=10.1016/j.jhep.2021.01.033&)

51. Celsa C., Cabibbo G., Enea M., et al., “Are Radiological Endpoints Surrogate Outcomes of Overall Survival in Hepatocellular Carcinoma Treated With Transarterial Chemoembolization?,” Liver International 41 (2021): 1105–1116, 10.1111/liv.14822.  [DOI](https://doi.org/10.1111/liv.14822) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33587814/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Liver%20International&title=Are%20Radiological%20Endpoints%20Surrogate%20Outcomes%20of%20Overall%20Survival%20in%20Hepatocellular%20Carcinoma%20Treated%20With%20Transarterial%20Chemoembolization?&author=C.%20Celsa&author=G.%20Cabibbo&author=M.%20Enea&volume=41&publication_year=2021&pages=1105-1116&pmid=33587814&doi=10.1111/liv.14822&)

52. Cabibbo G., Celsa C., Enea M., et al., “Progression‐Free Survival Early Assessment Is a Robust Surrogate Endpoint of Overall Survival in Immunotherapy Trials of Hepatocellular Carcinoma,” Cancers (Basel) 13, no. 1 (2020): 90, 10.3390/cancers13010090.  [DOI](https://doi.org/10.3390/cancers13010090) | [PMC free article](/articles/PMC7796103/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33396833/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancers%20(Basel)&title=Progression%E2%80%90Free%20Survival%20Early%20Assessment%20Is%20a%20Robust%20Surrogate%20Endpoint%20of%20Overall%20Survival%20in%20Immunotherapy%20Trials%20of%20Hepatocellular%20Carcinoma&author=G.%20Cabibbo&author=C.%20Celsa&author=M.%20Enea&volume=13&issue=1&publication_year=2020&pages=90&pmid=33396833&doi=10.3390/cancers13010090&)

53. Liu G., Ouyang Q., Xia F., et al., “Alpha‐Fetoprotein Response Following Transarterial Chemoembolization Indicates Improved Survival for Intermediate‐Stage Hepatocellular Carcinoma,” HPB: The Official Journal of the International Hepato Pancreato Biliary Association 21 (2019): 107–113, 10.1016/j.hpb.2018.06.1800.  [DOI](https://doi.org/10.1016/j.hpb.2018.06.1800) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30017783/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=HPB:%20The%20Official%20Journal%20of%20the%20International%20Hepato%20Pancreato%20Biliary%20Association&title=Alpha%E2%80%90Fetoprotein%20Response%20Following%20Transarterial%20Chemoembolization%20Indicates%20Improved%20Survival%20for%20Intermediate%E2%80%90Stage%20Hepatocellular%20Carcinoma&author=G.%20Liu&author=Q.%20Ouyang&author=F.%20Xia&volume=21&publication_year=2019&pages=107-113&pmid=30017783&doi=10.1016/j.hpb.2018.06.1800&)

54. Masior Ł., Krasnodębski M., Kuncewicz M., et al., “Alpha‐Fetoprotein Response After First Transarterial Chemoembolization (TACE) and Complete Pathologic Response in Patients With Hepatocellular Cancer,” Cancers (Basel) 15 (2023): 3952, 10.3390/cancers15153962.  [DOI](https://doi.org/10.3390/cancers15153962) | [PMC free article](/articles/PMC10417598/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/37568778/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancers%20(Basel)&title=Alpha%E2%80%90Fetoprotein%20Response%20After%20First%20Transarterial%20Chemoembolization%20(TACE)%20and%20Complete%20Pathologic%20Response%20in%20Patients%20With%20Hepatocellular%20Cancer&author=%C5%81.%20Masior&author=M.%20Krasnod%C4%99bski&author=M.%20Kuncewicz&volume=15&publication_year=2023&pages=3952&pmid=37568778&doi=10.3390/cancers15153962&)

55. Sánchez A. I. P., Roces L. V., García I. Z., et al., “Value of α‐Fetoprotein as an Early Biomarker for Treatment Response to Sorafenib Therapy in Advanced Hepatocellular Carcinoma,” Oncology Letters 15 (2018): 8863–8870, 10.3892/ol.2018.8400.  [DOI](https://doi.org/10.3892/ol.2018.8400) | [PMC free article](/articles/PMC5950578/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29805623/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncology%20Letters&title=Value%20of%20%CE%B1%E2%80%90Fetoprotein%20as%20an%20Early%20Biomarker%20for%20Treatment%20Response%20to%20Sorafenib%20Therapy%20in%20Advanced%20Hepatocellular%20Carcinoma&author=A.%20I.%20P.%20S%C3%A1nchez&author=L.%20V.%20Roces&author=I.%20Z.%20Garc%C3%ADa&volume=15&publication_year=2018&pages=8863-8870&pmid=29805623&doi=10.3892/ol.2018.8400&)

56. Hanif H., Ali M. J., Susheela A. T., et al., “Update on the Applications and Limitations of Alpha‐Fetoprotein for Hepatocellular Carcinoma,” World Journal of Gastroenterology 28 (2022): 216–229, 10.3748/wjg.v28.i2.216.  [DOI](https://doi.org/10.3748/wjg.v28.i2.216) | [PMC free article](/articles/PMC8776528/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35110946/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=World%20Journal%20of%20Gastroenterology&title=Update%20on%20the%20Applications%20and%20Limitations%20of%20Alpha%E2%80%90Fetoprotein%20for%20Hepatocellular%20Carcinoma&author=H.%20Hanif&author=M.%20J.%20Ali&author=A.%20T.%20Susheela&volume=28&publication_year=2022&pages=216-229&pmid=35110946&doi=10.3748/wjg.v28.i2.216&)

57. Nakamura S., Nouso K., Sakaguchi K., et al., “Sensitivity and Specificity of Des‐Gamma‐Carboxy Prothrombin for Diagnosis of Patients With Hepatocellular Carcinomas Varies According to Tumor Size,” American Journal of Gastroenterology 101 (2006): 2038–2043, 10.1111/j.1572-0241.2006.00681.x.  [DOI](https://doi.org/10.1111/j.1572-0241.2006.00681.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16848811/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=American%20Journal%20of%20Gastroenterology&title=Sensitivity%20and%20Specificity%20of%20Des%E2%80%90Gamma%E2%80%90Carboxy%20Prothrombin%20for%20Diagnosis%20of%20Patients%20With%20Hepatocellular%20Carcinomas%20Varies%20According%20to%20Tumor%20Size&author=S.%20Nakamura&author=K.%20Nouso&author=K.%20Sakaguchi&volume=101&publication_year=2006&pages=2038-2043&pmid=16848811&doi=10.1111/j.1572-0241.2006.00681.x&)

58. Saffroy R., Pham P., Reffas M., Takka M., Lemoine A., and Debuire B., “New Perspectives and Strategy Research Biomarkers for Hepatocellular Carcinoma,” Clinical Chemistry and Laboratory Medicine 45 (2007): 1169–1179, 10.1515/cclm.2007.262.  [DOI](https://doi.org/10.1515/cclm.2007.262) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17635075/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clinical%20Chemistry%20and%20Laboratory%20Medicine&title=New%20Perspectives%20and%20Strategy%20Research%20Biomarkers%20for%20Hepatocellular%20Carcinoma&author=R.%20Saffroy&author=P.%20Pham&author=M.%20Reffas&author=M.%20Takka&author=A.%20Lemoine&volume=45&publication_year=2007&pages=1169-1179&pmid=17635075&doi=10.1515/cclm.2007.262&)

59. Thermo Fisher Scientific , “Axiom Precision Medicine Diversity Array Plus,” 2024. accessed January 1, 2024, https://www.thermofisher.com/order/catalog/product/951961.  [https://www.thermofisher.com/order/catalog/product/951961](https://www.thermofisher.com/order/catalog/product/951961)

60. Perreault L. P. L., Zaïd N., Cameron M., Mongrain I., and Dubé M. P., “Pharmacogenetic Content of Commercial Genome‐Wide Genotyping Arrays,” Pharmacogenomics 19 (2018): 1159–1167, 10.2217/pgs-2017-0129.  [DOI](https://doi.org/10.2217/pgs-2017-0129) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30272537/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Pharmacogenetic%20Content%20of%20Commercial%20Genome%E2%80%90Wide%20Genotyping%20Arrays&author=L.%20P.%20L.%20Perreault&author=N.%20Za%C3%AFd&author=M.%20Cameron&author=I.%20Mongrain&author=M.%20P.%20Dub%C3%A9&volume=19&publication_year=2018&pages=1159-1167&pmid=30272537&doi=10.2217/pgs-2017-0129&)

61. Bruix J. and Sherman M., “Management of Hepatocellular Carcinoma,” Hepatology 42 (2005): 1208–1236, 10.1002/hep.20933.  [DOI](https://doi.org/10.1002/hep.20933) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16250051/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hepatology&title=Management%20of%20Hepatocellular%20Carcinoma&author=J.%20Bruix&author=M.%20Sherman&volume=42&publication_year=2005&pages=1208-1236&pmid=16250051&doi=10.1002/hep.20933&)

62. Oken M. M., Creech R. H., Tormey D. C., et al., “Toxicity and Response Criteria of the Eastern Cooperative Oncology Group,” American Journal of Clinical Oncology 5, no. 6 (1982): 649–655, 10.1097/00000421-198212000-00014.  [DOI](https://doi.org/10.1097/00000421-198212000-00014) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7165009/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=American%20Journal%20of%20Clinical%20Oncology&title=Toxicity%20and%20Response%20Criteria%20of%20the%20Eastern%20Cooperative%20Oncology%20Group&author=M.%20M.%20Oken&author=R.%20H.%20Creech&author=D.%20C.%20Tormey&volume=5&issue=6&publication_year=1982&pages=649-655&pmid=7165009&doi=10.1097/00000421-198212000-00014&)

63. Whirl‐Carrillo M., Huddart R., Gong L., et al., “An Evidence‐Based Framework for Evaluating Pharmacogenomics Knowledge for Personalized Medicine,” Clinical Pharmacology and Therapeutics 110 (2021): 563–572, 10.1002/cpt.2350.  [DOI](https://doi.org/10.1002/cpt.2350) | [PMC free article](/articles/PMC8457105/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34216021/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clinical%20Pharmacology%20and%20Therapeutics&title=An%20Evidence%E2%80%90Based%20Framework%20for%20Evaluating%20Pharmacogenomics%20Knowledge%20for%20Personalized%20Medicine&author=M.%20Whirl%E2%80%90Carrillo&author=R.%20Huddart&author=L.%20Gong&volume=110&publication_year=2021&pages=563-572&pmid=34216021&doi=10.1002/cpt.2350&)

64. Nguyen D. T., Tran T. T. H., Tran M. H., et al., “A Comprehensive Evaluation of Polygenic Score and Genotype Imputation Performances of Human SNP Arrays in Diverse Populations,” Scientific Reports 12 (2022): 17556, 10.1038/s41598-022-22215-y.  [DOI](https://doi.org/10.1038/s41598-022-22215-y) | [PMC free article](/articles/PMC9585077/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36266455/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Scientific%20Reports&title=A%20Comprehensive%20Evaluation%20of%20Polygenic%20Score%20and%20Genotype%20Imputation%20Performances%20of%20Human%20SNP%20Arrays%20in%20Diverse%20Populations&author=D.%20T.%20Nguyen&author=T.%20T.%20H.%20Tran&author=M.%20H.%20Tran&volume=12&publication_year=2022&pages=17556&pmid=36266455&doi=10.1038/s41598-022-22215-y&)

65. Desta Z., Gammal R. S., Gong L., et al., “Clinical Pharmacogenetics Implementation Consortium (CPIC) Guideline for CYP2B6 and Efavirenz‐Containing Antiretroviral Therapy,” Clinical Pharmacology and Therapeutics 106 (2019): 726–733, 10.1002/cpt.1477.  [DOI](https://doi.org/10.1002/cpt.1477) | [PMC free article](/articles/PMC6739160/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31006110/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clinical%20Pharmacology%20and%20Therapeutics&title=Clinical%20Pharmacogenetics%20Implementation%20Consortium%20(CPIC)%20Guideline%20for%20CYP2B6%20and%20Efavirenz%E2%80%90Containing%20Antiretroviral%20Therapy&author=Z.%20Desta&author=R.%20S.%20Gammal&author=L.%20Gong&volume=106&publication_year=2019&pages=726-733&pmid=31006110&doi=10.1002/cpt.1477&)

66. Zhang H., Sridar C., Kenaan C., Amunugama H., Ballou D. P., and Hollenberg P. F., “Polymorphic Variants of Cytochrome P450 2B6 (CYP2B6.4‐CYP2B6.9) Exhibit Altered Rates of Metabolism for Bupropion and Efavirenz: A Charge‐Reversal Mutation in the K139E Variant (CYP2B6.8) Impairs Formation of a Functional Cytochrome p450‐Reductase Complex,” Journal of Pharmacology and Experimental Therapeutics 338 (2011): 803–809, 10.1124/jpet.111.183111.  [DOI](https://doi.org/10.1124/jpet.111.183111) | [PMC free article](/articles/PMC3164347/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21659470/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Pharmacology%20and%20Experimental%20Therapeutics&title=Polymorphic%20Variants%20of%20Cytochrome%20P450%202B6%20(CYP2B6.4%E2%80%90CYP2B6.9)%20Exhibit%20Altered%20Rates%20of%20Metabolism%20for%20Bupropion%20and%20Efavirenz:%20A%20Charge%E2%80%90Reversal%20Mutation%20in%20the%20K139E%20Variant%20(CYP2B6.8)%20Impairs%20Formation%20of%20a%20Functional%20Cytochrome%20p450%E2%80%90Reductase%20Complex&author=H.%20Zhang&author=C.%20Sridar&author=C.%20Kenaan&author=H.%20Amunugama&author=D.%20P.%20Ballou&volume=338&publication_year=2011&pages=803-809&pmid=21659470&doi=10.1124/jpet.111.183111&)

67. Maayah Z. H., Althurwi H. N., Abdelhamid G., Lesyk G., Jurasz P., and El‐Kadi A. O., “CYP1B1 Inhibition Attenuates Doxorubicin‐Induced Cardiotoxicity Through a Mid‐Chain HETEs‐Dependent Mechanism,” Pharmacological Research 105 (2016): 28–43, 10.1016/j.phrs.2015.12.016.  [DOI](https://doi.org/10.1016/j.phrs.2015.12.016) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26772815/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacological%20Research&title=CYP1B1%20Inhibition%20Attenuates%20Doxorubicin%E2%80%90Induced%20Cardiotoxicity%20Through%20a%20Mid%E2%80%90Chain%20HETEs%E2%80%90Dependent%20Mechanism&author=Z.%20H.%20Maayah&author=H.%20N.%20Althurwi&author=G.%20Abdelhamid&author=G.%20Lesyk&author=P.%20Jurasz&volume=105&publication_year=2016&pages=28-43&pmid=26772815&doi=10.1016/j.phrs.2015.12.016&)

68. Marsh S., Paul J., King C. R., Gifford G., McLeod H. L., and Brown R., “Pharmacogenetic Assessment of Toxicity and Outcome After Platinum Plus Taxane Chemotherapy in Ovarian Cancer: The Scottish Randomised Trial in Ovarian Cancer,” Journal of Clinical Oncology 25 (2007): 4528–4535, 10.1200/jco.2006.10.4752.  [DOI](https://doi.org/10.1200/jco.2006.10.4752) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17925548/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Clinical%20Oncology&title=Pharmacogenetic%20Assessment%20of%20Toxicity%20and%20Outcome%20After%20Platinum%20Plus%20Taxane%20Chemotherapy%20in%20Ovarian%20Cancer:%20The%20Scottish%20Randomised%20Trial%20in%20Ovarian%20Cancer&author=S.%20Marsh&author=J.%20Paul&author=C.%20R.%20King&author=G.%20Gifford&author=H.%20L.%20McLeod&volume=25&publication_year=2007&pages=4528-4535&pmid=17925548&doi=10.1200/jco.2006.10.4752&)

69. Ruzzo A., Graziano F., Kawakami K., et al., “Pharmacogenetic Profiling and Clinical Outcome of Patients With Advanced Gastric Cancer Treated With Palliative Chemotherapy,” Journal of Clinical Oncology 24 (2006): 1883–1891, 10.1200/jco.2005.04.8322.  [DOI](https://doi.org/10.1200/jco.2005.04.8322) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16622263/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Clinical%20Oncology&title=Pharmacogenetic%20Profiling%20and%20Clinical%20Outcome%20of%20Patients%20With%20Advanced%20Gastric%20Cancer%20Treated%20With%20Palliative%20Chemotherapy&author=A.%20Ruzzo&author=F.%20Graziano&author=K.%20Kawakami&volume=24&publication_year=2006&pages=1883-1891&pmid=16622263&doi=10.1200/jco.2005.04.8322&)

70. Huang G., Mills L., and Worth L. L., “Expression of Human Glutathione S‐Transferase P1 Mediates the Chemosensitivity of Osteosarcoma Cells,” Molecular Cancer Therapeutics 6 (2007): 1610–1619, 10.1158/1535-7163.Mct-06-0580.  [DOI](https://doi.org/10.1158/1535-7163.Mct-06-0580) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17513610/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Molecular%20Cancer%20Therapeutics&title=Expression%20of%20Human%20Glutathione%20S%E2%80%90Transferase%20P1%20Mediates%20the%20Chemosensitivity%20of%20Osteosarcoma%20Cells&author=G.%20Huang&author=L.%20Mills&author=L.%20L.%20Worth&volume=6&publication_year=2007&pages=1610-1619&pmid=17513610&doi=10.1158/1535-7163.Mct-06-0580&)

71. Ramos D. L., Gaspar J. F., Pingarilho M., et al., “Genotoxic Effects of Doxorubicin in Cultured Human Lymphocytes With Different Glutathione S‐Transferase Genotypes,” Mutation Research 724 (2011): 28–34, 10.1016/j.mrgentox.2011.04.013.  [DOI](https://doi.org/10.1016/j.mrgentox.2011.04.013) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21640195/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mutation%20Research&title=Genotoxic%20Effects%20of%20Doxorubicin%20in%20Cultured%20Human%20Lymphocytes%20With%20Different%20Glutathione%20S%E2%80%90Transferase%20Genotypes&author=D.%20L.%20Ramos&author=J.%20F.%20Gaspar&author=M.%20Pingarilho&volume=724&publication_year=2011&pages=28-34&pmid=21640195&doi=10.1016/j.mrgentox.2011.04.013&)

72. Liu X., Tan N., Liao H., et al., “High GSTP1 Inhibits Cell Proliferation by Reducing Akt Phosphorylation and Is Associated With a Better Prognosis in Hepatocellular Carcinoma,” Oncotarget 9, no. 10 (2018): 8957–8971, 10.18632/oncotarget.23420.  [DOI](https://doi.org/10.18632/oncotarget.23420) | [PMC free article](/articles/PMC5823662/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29507666/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncotarget&title=High%20GSTP1%20Inhibits%20Cell%20Proliferation%20by%20Reducing%20Akt%20Phosphorylation%20and%20Is%20Associated%20With%20a%20Better%20Prognosis%20in%20Hepatocellular%20Carcinoma&author=X.%20Liu&author=N.%20Tan&author=H.%20Liao&volume=9&issue=10&publication_year=2018&pages=8957-8971&pmid=29507666&doi=10.18632/oncotarget.23420&)

73. Fagerberg L., Hallström B. M., Oksvold P., et al., “Analysis of the Human Tissue‐Specific Expression by Genome‐Wide Integration of Transcriptomics and Antibody‐Based Proteomics,” Molecular & Cellular Proteomics 13 (2014): 397–406, 10.1074/mcp.M113.035600.  [DOI](https://doi.org/10.1074/mcp.M113.035600) | [PMC free article](/articles/PMC3916642/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24309898/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Molecular%20&%20Cellular%20Proteomics&title=Analysis%20of%20the%20Human%20Tissue%E2%80%90Specific%20Expression%20by%20Genome%E2%80%90Wide%20Integration%20of%20Transcriptomics%20and%20Antibody%E2%80%90Based%20Proteomics&author=L.%20Fagerberg&author=B.%20M.%20Hallstr%C3%B6m&author=P.%20Oksvold&volume=13&publication_year=2014&pages=397-406&pmid=24309898&doi=10.1074/mcp.M113.035600&)

74. Niemi M., Pasanen M. K., and Neuvonen P. J., “Organic Anion Transporting Polypeptide 1B1: A Genetically Polymorphic Transporter of Major Importance for Hepatic Drug Uptake,” Pharmacological Reviews 63 (2011): 157–181, 10.1124/pr.110.002857.  [DOI](https://doi.org/10.1124/pr.110.002857) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21245207/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacological%20Reviews&title=Organic%20Anion%20Transporting%20Polypeptide%201B1:%20A%20Genetically%20Polymorphic%20Transporter%20of%20Major%20Importance%20for%20Hepatic%20Drug%20Uptake&author=M.%20Niemi&author=M.%20K.%20Pasanen&author=P.%20J.%20Neuvonen&volume=63&publication_year=2011&pages=157-181&pmid=21245207&doi=10.1124/pr.110.002857&)

75. Okabe M., Unno M., Harigae H., et al., “Characterization of the Organic Cation Transporter SLC22A16: A Doxorubicin Importer,” Biochemical and Biophysical Research Communications 333 (2005): 754–762, 10.1016/j.bbrc.2005.05.174.  [DOI](https://doi.org/10.1016/j.bbrc.2005.05.174) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15963465/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochemical%20and%20Biophysical%20Research%20Communications&title=Characterization%20of%20the%20Organic%20Cation%20Transporter%20SLC22A16:%20A%20Doxorubicin%20Importer&author=M.%20Okabe&author=M.%20Unno&author=H.%20Harigae&volume=333&publication_year=2005&pages=754-762&pmid=15963465&doi=10.1016/j.bbrc.2005.05.174&)

76. Cabral L. K. D., Tiribelli C., and Sukowati C. H. C., “Sorafenib Resistance in Hepatocellular Carcinoma: The Relevance of Genetic Heterogeneity,” Cancers (Basel) 12 (2020): 1576, 10.3390/cancers12061576.  [DOI](https://doi.org/10.3390/cancers12061576) | [PMC free article](/articles/PMC7352671/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32549224/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancers%20(Basel)&title=Sorafenib%20Resistance%20in%20Hepatocellular%20Carcinoma:%20The%20Relevance%20of%20Genetic%20Heterogeneity&author=L.%20K.%20D.%20Cabral&author=C.%20Tiribelli&author=C.%20H.%20C.%20Sukowati&volume=12&publication_year=2020&pages=1576&pmid=32549224&doi=10.3390/cancers12061576&)

77. Casadei‐Gardini A., Marisi G., Dadduzio V., et al., “Association of NOS3 and ANGPT2 Gene Polymorphisms With Survival in Patients With Hepatocellular Carcinoma Receiving Sorafenib: Results of the Multicenter Prospective INNOVATE Study,” Clinical Cancer Research 26 (2020): 4485–4493, 10.1158/1078-0432.Ccr-19-3897.  [DOI](https://doi.org/10.1158/1078-0432.Ccr-19-3897) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32371540/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clinical%20Cancer%20Research&title=Association%20of%20NOS3%20and%20ANGPT2%20Gene%20Polymorphisms%20With%20Survival%20in%20Patients%20With%20Hepatocellular%20Carcinoma%20Receiving%20Sorafenib:%20Results%20of%20the%20Multicenter%20Prospective%20INNOVATE%20Study&author=A.%20Casadei%E2%80%90Gardini&author=G.%20Marisi&author=V.%20Dadduzio&volume=26&publication_year=2020&pages=4485-4493&pmid=32371540&doi=10.1158/1078-0432.Ccr-19-3897&)

78. Casadei‐Gardini A., Marisi G., Faloppi L., et al., “eNOS Polymorphisms and Clinical Outcome in Advanced HCC Patients Receiving Sorafenib: Final Results of the ePHAS Study,” Oncotarget 7 (2016): 27988–27999, 10.18632/oncotarget.8569.  [DOI](https://doi.org/10.18632/oncotarget.8569) | [PMC free article](/articles/PMC5053704/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27058899/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncotarget&title=eNOS%20Polymorphisms%20and%20Clinical%20Outcome%20in%20Advanced%20HCC%20Patients%20Receiving%20Sorafenib:%20Final%20Results%20of%20the%20ePHAS%20Study&author=A.%20Casadei%E2%80%90Gardini&author=G.%20Marisi&author=L.%20Faloppi&volume=7&publication_year=2016&pages=27988-27999&pmid=27058899&doi=10.18632/oncotarget.8569&)

79. Hodo Y., Honda M., Tanaka A., et al., “Association of Interleukin‐28B Genotype and Hepatocellular Carcinoma Recurrence in Patients With Chronic Hepatitis C,” Clinical Cancer Research 19, no. 7 (2013): 1827–1837, 10.1158/1078-0432.Ccr-12-1641.  [DOI](https://doi.org/10.1158/1078-0432.Ccr-12-1641) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23426277/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clinical%20Cancer%20Research&title=Association%20of%20Interleukin%E2%80%9028B%20Genotype%20and%20Hepatocellular%20Carcinoma%20Recurrence%20in%20Patients%20With%20Chronic%20Hepatitis%20C&author=Y.%20Hodo&author=M.%20Honda&author=A.%20Tanaka&volume=19&issue=7&publication_year=2013&pages=1827-1837&pmid=23426277&doi=10.1158/1078-0432.Ccr-12-1641&)

80. Lin C. L., Liang K. H., Hu C. C., et al., “A Single Nucleotide Polymorphism rs1010816 Predicts Sorafenib Therapeutic Outcomes in Advanced Hepatocellular Carcinoma,” International Journal of Molecular Sciences 24 (2023): 1681, 10.3390/ijms24021681.  [DOI](https://doi.org/10.3390/ijms24021681) | [PMC free article](/articles/PMC9862766/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36675198/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=International%20Journal%20of%20Molecular%20Sciences&title=A%20Single%20Nucleotide%20Polymorphism%20rs1010816%20Predicts%20Sorafenib%20Therapeutic%20Outcomes%20in%20Advanced%20Hepatocellular%20Carcinoma&author=C.%20L.%20Lin&author=K.%20H.%20Liang&author=C.%20C.%20Hu&volume=24&publication_year=2023&pages=1681&pmid=36675198&doi=10.3390/ijms24021681&)

81. Karlsson M., Zhang C., Méar L., et al., “A Single‐Cell Type Transcriptomics Map of Human Tissues,” Science Advances 7 (2021): eabh2169, 10.1126/sciadv.abh2169.  [DOI](https://doi.org/10.1126/sciadv.abh2169) | [PMC free article](/articles/PMC8318366/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34321199/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science%20Advances&title=A%20Single%E2%80%90Cell%20Type%20Transcriptomics%20Map%20of%20Human%20Tissues&author=M.%20Karlsson&author=C.%20Zhang&author=L.%20M%C3%A9ar&volume=7&publication_year=2021&pages=eabh2169&pmid=34321199&doi=10.1126/sciadv.abh2169&)

82. Philips C. A., Rajesh S., Nair D. C., Ahamed R., Abduljaleel J. K., and Augustine P., “Hepatocellular Carcinoma in 2021: An Exhaustive Update,” Cureus 13 (2021): e19274, 10.7759/cureus.19274.  [DOI](https://doi.org/10.7759/cureus.19274) | [PMC free article](/articles/PMC8569837/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34754704/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cureus&title=Hepatocellular%20Carcinoma%20in%202021:%20An%20Exhaustive%20Update&author=C.%20A.%20Philips&author=S.%20Rajesh&author=D.%20C.%20Nair&author=R.%20Ahamed&author=J.%20K.%20Abduljaleel&volume=13&publication_year=2021&pages=e19274&pmid=34754704&doi=10.7759/cureus.19274&)

83. Marisi G., Petracci E., Raimondi F., et al., “ANGPT2 and NOS3 Polymorphisms and Clinical Outcome in Advanced Hepatocellular Carcinoma Patients Receiving Sorafenib,” Cancers (Basel) 11 (2019): 1023, 10.3390/cancers11071023.  [DOI](https://doi.org/10.3390/cancers11071023) | [PMC free article](/articles/PMC6679015/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31330833/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancers%20(Basel)&title=ANGPT2%20and%20NOS3%20Polymorphisms%20and%20Clinical%20Outcome%20in%20Advanced%20Hepatocellular%20Carcinoma%20Patients%20Receiving%20Sorafenib&author=G.%20Marisi&author=E.%20Petracci&author=F.%20Raimondi&volume=11&publication_year=2019&pages=1023&pmid=31330833&doi=10.3390/cancers11071023&)
