# Improved Prediction of CYP2D6 Catalyzed Drug Metabolism by Taking Variant Substrate Specificities and Novel Polymorphic Haplotypes into Account

## Metadata
**Authors:** Inger Johansson, Emmanouella M Xanthopoulou, Yitian Zhou, Anabel Sanchez‐Spitman, Maaike van der Lee, Birgit M Wollmann, Elisabet Størset, Jesse J Swen, Henk‐Jan Guchelaar, Espen Molden, Marin M Jukic, Magnus Ingelman‐Sundberg
**Journal:** Clinical Pharmacology and Therapeutics
**Date:** 2025 Apr 22
**DOI:** [10.1002/cpt.3680](https://doi.org/10.1002/cpt.3680)
**PMID:** 40261922
**PMCID:** PMC12166262
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC12166262/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC12166262/pdf/CPT-118-218.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC12166262/pdf/CPT-118-218.pdf)

## Abstract

The polymorphic CYP2D6 enzyme plays a pivotal role in the metabolism of approximately 25% of clinically prescribed drugs. However, the impact of specific genetic variants on the interindividual variability in CYP2D6‐mediated drug metabolism remains insufficiently quantified. This translational study sought to address this gap by analyzing the genotypes and phenotypes of patients in two large clinical cohorts, focusing on the metabolism of the CYP2D6 substrates risperidone and desmethyltamoxifen. The analysis incorporated novel polymorphic haplotypes and substrate‐specific differences among the CYP2D6.1, CYP2D6.2, and CYP2D6.35 enzyme variants. The study revealed that CYP2D6.2 and CYP2D6.35 exhibit reduced metabolic capacity for these substrates, both in vivo and in an in vitro expression model. This was evidenced by decreased catalytic turnover (Kcat), decreased substrate affinity, and altered substrate docking. Furthermore, novel polymorphic haplotypes on the CYP2D6*1, CYP2D6*2, and CYP2D6*35 backgrounds were identified, each associated with a 30–40% increase in CYP2D6 activity. Incorporating these findings into prediction equations significantly improved the genetic prediction accuracy (R 2) for CYP2D6‐mediated metabolism of desmethyltamoxifen from 59% to 71% and risperidone, also metabolized by CYP3A4, from 42% to 46%. These results highlight the importance of accounting for drug‐specific interactions with enzyme variants and integrating distinct polymorphic haplotypes into CYP pharmacogenomic models and guidelines for better translation into clinical practice.

CYP2D6 is a key polymorphic enzyme involved in the metabolism of numerous clinically prescribed drugs. Pharmacogenomic predictions based on *CYP2D6* genotypes have the potential to optimize drug therapy across multiple therapeutic fields. However, further characterization of *CYP2D6* genetic variants is essential to refine these predictions.

This study aimed to address how two common CYP2D6 enzyme variants, CYP2D6.2 and CYP2D6.35, as well as novel CYP2D6 haplotypes, influence the metabolism of desmethyltamoxifen and risperidone. Using both *in vivo* data from two independent patient cohorts and *in vitro* expression systems and *in silico* docking, we evaluated the impact of these variants to enhance prediction models for CYP2D6‐dependent drug metabolism.

Our results indicate that the metabolism of both desmethyltamoxifen and risperidone is reduced when catalyzed by CYP2D6.2 and CYP2D6.35, compared to CYP2D6.1. Additionally, we identified new haplotypes that significantly increase the expression of *CYP2D6*2 and CYP2D6**35. Incorporating these findings into an adjusted algorithm, we improved the prediction of desmethyltamoxifen metabolism, with adjusted *R* ^2^ increasing from 59% to 71.2% compared to the CPIC‐derived activity scores. The novel algorithm also improved prediction of risperidone metabolism, however, prediction accuracy remained limited, with *R* ^2^ values increasing from 22.7% to 25.3%.

Our findings suggest that CYP2D6 pharmacogenomic predictions can be improved by considering specific drug substrates and including distinct polymorphic haplotypes near the *CYP2D* locus. Applications of these refinements may lead to more accurate, personalized predictions of CYP‐catalyzed drug metabolism, advancing the field of clinical pharmacology and translational science.

Pharmacogenomics has become increasingly important in modern medicine, with its clinical application facilitated by defining gene‐drug pairs where pharmacogenomic biomarkers can enhance the quality of drug prescriptions, as noted in the FDA's table of pharmacogenetic associations[1](#cpt3680-bib-0001) where pharmacogenetic aspects support therapeutic management recommendations for 59 different drugs. Most of these variants are related to drug pharmacokinetics (PK). However, while PGx biomarkers explain a substantial part of interindividual variation in drug PK, a major part of the genetic contribution to PK variation is still to be unveiled, as evident from twin studies.[2](#cpt3680-bib-0002) This is in part due to a limited understanding of the entire spectrum of pharmacogenomic factors contributing to interindividual variability in drug metabolism and response. As previously highlighted,[3](#cpt3680-bib-0003), [4](#cpt3680-bib-0004) incomplete knowledge of inherited genetic variants that contribute to interindividual differences, a phenomenon referred to as “missing heritability”, is likely also of substantial relevance for the understanding of the pharmacogenomic influence on drug metabolism.[2](#cpt3680-bib-0002)

Two major polymorphic genes, *CYP2C19* and *CYP2D6*, play a crucial role in interindividual differences in drug metabolism, with CYP2D6 genetic polymorphism being particularly prolific for the metabolism of a broad range of substrates.[1](#cpt3680-bib-0001) Notably, CYP2D6 is responsible for the metabolism of approximately 25% of all clinically used drugs that are predominantly eliminated through metabolic pathways.[3](#cpt3680-bib-0003), [5](#cpt3680-bib-0005) The *CYP2D6* gene is highly polymorphic, with over 165 defined allelic variants that encode enzymes exhibiting a wide range of catalytic activities, from ultrarapid to completely nonfunctional phenotypes.[6](#cpt3680-bib-0006) The Clinical Pharmacogenetics Implementation Consortium (CPIC) assigns activity values to each variant, and then the sum defines the activity score, which is then translated to overall enzymatic activity.[7](#cpt3680-bib-0007) The genetic variation in the *CYP2D* locus can modulate gene expression through mechanisms including: (i) gene copy number variations, (ii) mutations in the open reading frame, and (iii) mutations in introns and in flanking regulatory regions.

The lack of understanding of the genetic influence on CYP2D6‐dependent drug metabolism includes hitherto unidentified rare mutations as well as the lack of investigations regarding the extent by which mutations causing amino acid changes can result in altered substrate binding and catalysis by the enzyme, which causes clinically relevant interindividual alterations in drug metabolism. It has been shown *in vitro* that the CYP2D6 variants CYP2D6.1, CYP2D6.2, CYP2D6.17, and CYP2D6.35 differ in their substrate specificities, although there has been little consensus among the studies performed.[8](#cpt3680-bib-0008), [9](#cpt3680-bib-0009), [10](#cpt3680-bib-0010), [11](#cpt3680-bib-0011) Thus, for enzyme variants with amino acid changes, specific gene variant‐drug pairs often need to be considered when assessing activity scores. This means that a single gene variant may have different activity scores depending on the substrate. However, such enzyme activity score assessments have not yet been implemented in pharmacogenomic guidelines cf.[3](#cpt3680-bib-0003), [12](#cpt3680-bib-0012)

Polymorphic variations in drug metabolism have traditionally been attributed to differences in enzyme expression, with the role of substrate specificity among polymorphic CYP enzymes receiving comparatively less attention.[10](#cpt3680-bib-0010) In this study, we examined the extent to which substrate specificity differences in CYP2D6 variant enzymes, along with additional polymorphic loci adjacent to the *CYP2D6* gene, contribute to the unexplained heritability in CYP2D6‐dependent drug metabolism. We specifically evaluated substrate specificity differences among the common CYP2D6 variants CYP2D6.1, CYP2D6.2, and CYP2D6.35, classified as exhibiting normal function.[6](#cpt3680-bib-0006) Using desmethyltamoxifen and risperidone as biomarker substrates, we analyzed data from two distinct cohorts: (i) the CYPTAM study, comprising 608 tamoxifen‐treated women of Caucasian ethnicity with breast cancer from different hospitals in The Netherlands where previous data analyses of an algorithm based on long‐read gene sequencing were applied[11](#cpt3680-bib-0011), [13](#cpt3680-bib-0013) and (ii) a Norwegian clinical cohort from Diakonhjemmet Hospital, Oslo, Norway, including 512 risperidone‐treated patients.[14](#cpt3680-bib-0014) To further elucidate the molecular basis of altered substrate specificities, we employed heterologous cell expression systems and in silico modeling techniques. Additionally, we assessed genotype‐based predictions of CYP2D6 activity using new, substrate‐specific activity scores for CYP2D6.2 and CYP2D6.35 as well as the importance of novel haplotypes.

## MATERIAL AND METHODS

### Patient data

Data from two distinct cohort studies were analyzed. The first was the CYPTAM study, which included a cohort of 608 tamoxifen‐treated women (self‐reported Caucasian ethnicity) with breast cancer, with steady‐state through levels of desmethyltamoxifen and endoxifen, along with CNV data and genome‐wide association study (GWAS) data obtained using the Infinium Global Screening Array v3. This provided genotype calls for 686,082 single nucleotide polymorphisms (SNPs) from 608 women.[13](#cpt3680-bib-0013) The second cohort was from the Center for Psychopharmacology, Diakonhjemmet Hospital in Oslo, where risperidone and 9‐hydroxyrisperidone levels obtained from the therapeutic drug monitoring database were available from 512 patients, along with data on the common *CYP2D6* **2‐6, *9, *10, *35, *41* alleles genotyped within the clinical routine, as previously described.[14](#cpt3680-bib-0014)

### Metabolic capacity encoded by CYP2D6*1, CYP2D6*2, and CYP2D6*35 in vivo

The metabolic capacity encoded by the *CYP2D6*1, CYP2D6*2*, and *CYP2D6*35* alleles was calculated using a previously established method for determining activity scores.[15](#cpt3680-bib-0015) For the risperidone cohort, exclusion criteria comprised inconclusive genotyping, age > 65 years, parent drug or metabolite concentrations below the limit of quantification (1 nmol/L), or when the time between the last dose intake and sample withdrawal fell outside the 10–30 h range, whereafter 393 subjects remained for analyses. The exclusion criteria for the CYPTAM study included other malignancies, hormone receptor‐negative tumors, a history of thromboembolic events, pregnancy or breastfeeding, prolonged QT intervals, and abnormal blood values.[16](#cpt3680-bib-0016)

To quantify CYP2D6 metabolic activity, product‐to‐substrate ratios were calculated: [9OH‐risperidone]/[risperidone] and [endoxifen]/[desmethyltamoxifen]. Initial screening analysis revealed similar effects of the *CYP2D6*2* and *CYP2D6*35* alleles on metabolic activity (**Supplementary Figure** [S1](#cpt3680-supitem-0001)). As a result, these two alleles were functionally combined into a composite *CYP2D6*2–*35* allele for the purpose of this analysis. For cross‐drug analysis, the product‐to‐substrate ratios was harmonized using the formula: (M/P − a)/(b − a), where a represents the mean ratio for individuals with a *CYP2D6Null/Null* genotype (set to 0% metabolic activity) and b represents the mean M/P for individuals with a *CYP2D6*1/*1* genotype (set to 200% metabolic activity). This standardization enabled pooling results from patients treated with different substrates. To estimate the relative contributions of the *CYP2D6*1* and *CYP2D6*2–*35* alleles, regression equations were calculated separately for patients treated with risperidone, tamoxifen, and both drugs. Metabolic activity was the dependent variable, while the number of *CYP2D6*2–*35* alleles served as the independent variable. Activity scores used for CYP2D6.2, CYP2D6.35, and other CYP2D6 functional alleles used for both substrates are given in **Table** [1](#cpt3680-tbl-0001).

### Table 1.

| Haplotype activity scores |  |  |  |
| --- | --- | --- | --- |
| Allele | Activity score without the influence of the J haplotype | Activity score with or without the J haplotype | Activity score (CPIC) |
| CYP2D6*1 | 1 | 1 | 1 |
| CYP2D6*2 | 0.5 | 0.45 | 1 |
| CYP2D6*2 + J | 0.5 | 0.6 | 1 |
| CYP2D6*35 | 0.5 | 0.45 | 1 |
| CYP2D6*35 + J | 0.5 | 0.6 | 1 |
| CYP2D6*9 | 0.35 | 0.35 | 0.25 |
| CYP2D6*10 | 0.35 | 0.35 | 0.25 |
| CYP2D6*41 | 0.15 | 0.15 | 0.25 |

Table 1 Caption: Haplotype and diplotype activity scores for CYP2D6 genetic variants analyzed in this study for risperidone and desmethyltamoxifen. The metabolic capacity encoded by the CYP2D6*1, CYP2D6*2, and CYP2D6*35 alleles was calculated using a previously established method for determining activity scores. 15 (J) denotes alleles in Linkage Disequilibrium (LD) to the J haplotype

### Algorithm predictive power evaluation

To address the skewness of risperidone metabolic product‐to‐substrate ratios caused by very low risperidone concentrations, the ratios were logarithmically transformed to achieve a normal distribution of residuals before harmonization. Next, a suggestion for modification of activity scores for major *CYP2D6* alleles was based on previously published calculations for *CYP2D6*9*, *CYP2D6*10*, and *CYP2D6*41* alleles[15](#cpt3680-bib-0015) and calculations for *CYP2D6*2* and *CYP2D6*35* herein (**Table** [1](#cpt3680-tbl-0001)). The calculation of metabolic capacity for each algorithm was performed by summing the activity scores of haplotypes, weighted by the number of alleles carried by each individual. The predictive accuracy of the algorithms was evaluated using the adjusted *R* ^2^, which quantified the proportion of the variability in the measured in vivo metabolic capacity explained by the predicted metabolic capacity (based on patient genotypes). Finally, we compared the predictive accuracy of the algorithms derived from activity scores as defined by CPIC, Gene‐specific Information Tables for CYP2D6,[17](#cpt3680-bib-0017) and derived from modified activity scores as proposed in **Table** [1](#cpt3680-tbl-0001). The algorithms differed only in the specific activity scores assigned to each allele, as detailed in **Table** [1](#cpt3680-tbl-0001).

### Construction of variant CYP2D6 expression plasmids

The expression plasmid pCMV4 containing *CYP2D6*2* cDNA (in house) was used as a template for the construction of the different allelic CYP2D6 variants using the QuikChange Lightning Multi Site‐Directed Mutagenesis Kit (Agilent, Santa Clara, CA, USA) according to the manufacturer's protocol. Sequences of mutagenesis primers are presented in **Supplementary Table** [S1](#cpt3680-supitem-0001). All cDNAs were verified by DNA sequencing (KI Gene, core facility, Karolinska Institutet). Plasmids were isolated with the Qiagen Plasmid Plus Midi kit (Hilden, Germany).

### Heterologous expression of CYP2D6 variants

CYP2D6 variant expression plasmids were transfected into HEK293 cells, cultured in DMEM supplemented with 4.5 g/L glucose, 10% fetal bovine serum, and penicillin–streptomycin (100 U/mL and 100 μg/mL), until they reached 70–80% confluence in 6‐well plates. Transfection of the pCMV plasmids containing the CYP2D6 variants was carried out using Lipofectamine™ 3000 Transfection Reagent (Invitrogen, Thermo Fisher Scientific, Waltham, MA, USA) following the manufacturer's instructions. After 48 hours of incubation, cells were harvested, and the pellets were stored at −80°C. The cell pellets were later resuspended in 100 mM sodium phosphate buffer (pH 7.4), sonicated 20 times for 1 second each, and centrifuged at 800 ** g ** for 10 minutes. The resulting supernatants were aliquoted and stored at −80°C.

Our analysis included structurally related variants of CYP2D6, namely CYP2D6.2, CYP2D6.28, and CYP2D6.35, which share the amino acid substitutions R296C and S486T, with CYP2D6.28 also carrying V7M and Q151E, and CYP2D6.35 additionally containing V11M compared to CYP2D6.2. The relative enzyme expression and activities for bufuralol (a traditional marker substrate for CYP2D6 activity), risperidone, and desmethyltamoxifen were assessed in the transfected cells.

Protein determinations were performed using *DC* Protein Assay Reagents (BioRad) and SpectraMax iD3 spectrophotometer (Molecular Devices LCC.). CYP2D6 expression levels were verified with SDS‐WB analysis with CYP antibodies from Daiichi, Tokyo, Japan. Levels of expression were similar between allelic variants and between experiments.

### Bufuralol hydroxylation

The rate of bufuralol hydroxylation, a conventional reaction for the assessment of CYP2D6 activity *in vitro*, and here used as a control for CYP2D6 activity, was analyzed by incubations containing 800 ** g ** supernatant corresponding to 200 μg of protein, 0.1 M sodium phosphate buffer, 0.6–50 μM bufuralol (racemate), and 1 mM NADPH in a total volume of 150 μL. The reactions performed at 37°C were terminated by the addition of 14 μL of 70% perchloric acid. After centrifugation, the supernatant was analyzed by HPLC as described by Kronbach et al.[18](#cpt3680-bib-0018)

### Risperidone hydroxylation

Incubations were performed as above but with 0.5–10 μM risperidone. The reactions were terminated by the addition of 0.5 volume of acetonitrile. Risperidone and the metabolite 9‐OH‐risperidone were quantified using HPLC as described by Lenk et al.[19](#cpt3680-bib-0019)

### Desmethyltamoxifen metabolism

Incubations were performed as above using 6.25–100 mM desmethyltamoxifen. The desmethyltamoxifen and endoxifen were quantified as described in Sanchez‐Spitman et al.[13](#cpt3680-bib-0013)

### Haplotype analysis

Analysis of haplotype and linkage disequilibrium was performed using LDlink[20](#cpt3680-bib-0020) and Ensembl.[21](#cpt3680-bib-0021)

### Molecular docking analysis

The crystal structure of wild‐type human CYP2D6 was obtained from the Protein Data Bank (PDB ID: 3TBG). Structures of CYP2D6.2 and CYP2D6.35 were modeled using AlphaFold2 on the Colab platform[22](#cpt3680-bib-0022) Docking was performed using AutoDock Vina in UCSF Chimera.[23](#cpt3680-bib-0023) Specifically, structures of desmethyltamoxifen and risperidone obtained from the PubChem database (CID: 6378383 and 5073, respectively) were docked to the active site cavity of CYP2D6.[24](#cpt3680-bib-0024) Docking centers were defined accordingly and default parameters were used for the receptor, ligand, and advanced options. Ten poses with different RMSD values were generated, and the optimal poses were selected for intra‐molecule contact calculation in UCSF Chimera. PyMOL (version 3.0.2) was further used for docking pose visualization.

## RESULTS

### Differences between CYP2D6.1, CYP2D6.2, and CYP2D6.35 in the in vivo metabolism of desmethyltamoxifen or risperidone

As shown in **Figure** [1 a](#cpt3680-fig-0001), individuals carrying the *CYP2D6*2* and *CYP2D6*35* alleles demonstrated reduced metabolic capacity for both risperidone and desmethyltamoxifen. Both alleles exhibited a comparable decrease in activity, measured as the log [product]/[substrate] ratio (**Figure** [1 a](#cpt3680-fig-0001)). However, activity values displayed a broader spread in the risperidone cohort. As shown in **Figure** [1 a](#cpt3680-fig-0001), heterozygous carriers of either variant allele demonstrated a 35% reduction in activity, while individuals homozygous for the variant alleles had approximately a 65% reduction compared to patients homozygous for *CYP2D6*1*. These findings indicate that *CYP2D6*2* and *CYP2D6*35* are not normal‐function alleles but rather reduced‐function variants for desmethyltamoxifen and risperidone. The impact of these alleles for discrimination between phenotypes was higher for desmethyltamoxifen, likely because risperidone is also metabolized by CYP3A4[25](#cpt3680-bib-0025) and among patients with reduced CYP2D6 activity to some extent directly eliminated via the kidneys.[26](#cpt3680-bib-0026)

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f9ec/12166262/f2e88c445875/CPT-118-218-g005.jpg)

Effect of CYP2D6.2/CYP2D6.35 vs. CYP2D6.1. Effect of CYP2D6.2/CYP2D6.35 vs. CYP2D6.1 on (a) log ratio [endoxifen]/[desmethyltamoxifen] and log ratio [9‐hydroxyrisperidone]/[risperidone] and on (b) endoxifen and 9‐hydroxyrisperidone levels.

This difference is clearly illustrated when the levels of the enzyme products, endoxifen, and 9‐OH‐risperidone, are compared among patients homozygous or heterozygous for the three variant *CYP2D6* alleles (**Figure** [1 b](#cpt3680-fig-0001)). A significant reduction in endoxifen production was observed in carriers of *CYP2D6*2* and *CYP2D6*35*, while these alleles did not significantly affect the 9‐OH‐risperidone levels. Although CYP3A4 contributes to the conversion of tamoxifen to desmethyltamoxifen, its influence on endoxifen production appears minimal compared to its more prominent role of CYP3A4 in the 9‐hydroxylation of risperidone (**Figure** [1 b](#cpt3680-fig-0001)).

### Pharmacokinetic and molecular analyses of the variant CYP2D6 enzymes in vitro and in silico

We investigated the characteristics of different CYP2D6 enzyme variants by transfecting HEK293 cells with expression plasmids for each variant, with results presented in **Table** [2](#cpt3680-tbl-0002) and **Figure** [2 a](#cpt3680-fig-0002).

### Table 2.

| A. | Bufuralol |  |  |  |  |
| --- | --- | --- | --- | --- | --- |
| V max |  | K m | K cat/K m |  |  |
| pmol product/1.5 h, 133 μg protein | Relative CYP2D6.1 | μM |   | Relative CYP2D6.1 |  |
| CYP2D6.1 | 22.0 ± 10.8 |   | 10.9 ± 3.0 | 2.01 ± 0.68 |   |
| CYP2D6.2 | 17.7 ± 9.9 | 0.69 ± 0.12 * | 13.7 ± 4.0 | 1.29 ± 0.47 | 0.63 ± 0.03*** |
| CYP2D6.28 | 16.6 ± 6.1 | 0.72 ± 0.23 | 13.04 ± 6.9 | 1.63 ± 1.03 | 0.76 ± 0.35 |
| CYP2D6.35 | 11.9 ± 4.7 | 0.63 ± 0.29 | 14.2 ± 12.5 | 1.24 ± 0.69 | 0.58 ± 0.20* |

Table 2 Caption: Pharmacokinetic properties of bufuralol (a). risperidone (b) and desmethyltamoxifen (c) metabolism by different CYP2D6 enzyme variants expressed in HEK293 cells. Results are from three independent expression experiments; desmethyltamoxifen data for CYP2D6.35 is from two experiments. Statistical analysis was performed using Student t‐test, ***P < 0.001, **P < 0.01, *P < 0.05

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f9ec/12166262/94ea00f300da/CPT-118-218-g002.jpg)

Kinetic analysis of different CYP2D6 enzyme variants. Kinetic analysis of the metabolism of (a) bufuralol, risperidone, and desmethyltamoxifen catalyzed by the different CYP2D6 enzyme variants. Graphs from representative experiments are shown. (b) Binding of desmethyltamoxifen and risperidone to the CYP2D6.1, CYP2D6.2, and CYP2D6.35 variants. Structures of desmethyltamoxifen and risperidone binding to CYP2D6 and its variant enzymes were analyzed using molecular docking as described under Methods. The active site of CYP2D6 is highlighted in the surface with the rainbow spectrum, and the substrate is shown in a red stick. The figure indicates the different modes of substrate binding for each of the CYP2D6 variants.

For bufuralol, no significant differences in Km and Vmax values were observed between the CYP2D6 variants examined. However, the relative Kcat/Km values for desmethyltamoxifen were only 30–35% for both CYP2D6.2 and CYP2D6.35 and 48% for CYP2D6.28, as compared to CYP2D6.1, indicating reduced catalytic efficiency for this substrate (**Table** [2](#cpt3680-tbl-0002)). In the case of risperidone, CYP2D6.2, CYP2D6.28, and CYP2D6.35 displayed significantly higher Km values than CYP2D6.1. The relative Kcat/Km values demonstrated that the catalytic efficiencies of CYP2D6.2 and CYP2D6.35 for risperidone were reduced by approximately 80%, compared to CYP2D6.1 (**Table** [2](#cpt3680-tbl-0002)). The observed effects of these alleles on the production of endoxifen from desmethyltamoxifen and 9‐hydroxyrisperidone from risperidone were in line with the *in vivo* data (**Figure** [1](#cpt3680-fig-0001)) although more pronounced.

### Docking of risperidone and desmethyltamoxifen into CYP2D6 active sites

Docking simulations were carried out based on the crystal structure of CYP2D6.1, where CYP2D6.2 and CYP2D6.35 were modeled using AlphaFold2 on the Colab platform. The results, shown in **Figure** [2 b](#cpt3680-fig-0002), reveal distinct differences in the docking of risperidone and desmethyltamoxifen into the active sites of CYP2D6.1 compared to CYP2D6.2 and CYP2D6.35. Notably, desmethyltamoxifen interacts with residues outside the CYP2D6 active site and exhibits altered positions and orientations in variant proteins. In contrast, risperidone can enter the active site and become fully enclosed by the variant proteins. These findings are in line with the kinetic data and strongly suggest that the reduced activities of CYP2D6.2 and CYP2D6.35 are primarily due to impaired substrate binding and catalysis at the active site.

### Identification of novel haplotypes of importance for CYP2D6 activity

CYP2D6 expression has been shown to be influenced by polymorphisms in the nuclear factor NFIB on chromosome 9,[19](#cpt3680-bib-0019) as well as by genetic variants located within 500 kb of the *CYP2D6* gene on chromosome 22.[13](#cpt3680-bib-0013), [27](#cpt3680-bib-0027) To further identify genetic variants that may regulate CYP2D6 expression, we analyzed haplotypes based on the GWAS data in the genomic regions surrounding the *CYP2D6* gene, aiming to uncover additional polymorphic regions that could play a critical role in its regulation. Using data from Sanchez‐Spitman,[13](#cpt3680-bib-0013) we employed LDlink[20](#cpt3680-bib-0020) and Ensembl[21](#cpt3680-bib-0021) to identify haplotypes in linkage disequilibrium (LD) with an *r* ^2^ between 0.9 and 1. This analysis revealed 27 distinct haplotypes within 500 kb of the *CYP2D6* gene (**Figure** [3](#cpt3680-fig-0003); **Supplementary Table** [S2](#cpt3680-supitem-0002)).

### Figure 3.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f9ec/12166262/c1937b6c8b85/CPT-118-218-g006.jpg)

Schematic representation of the distribution of haplotypes identified on chromosome 22. Schematic representation of the genomic distribution of haplotypes identified on chromosome 22. Haplotypes B1, C2, E, J1, L, and O1. Marked red were significant for log M/P [desmethyltamoxifen]/[endoxifen] among carriers of CYP2D6*1, CYP2D6*2, and CYP2D6*35. Red dots indicate the localization of the significant SNPs from the GWAS that were not grouped into haplotypes.

We then assessed the association of each haplotype with CYP2D6 expression by analyzing the log [product]/[substrate] ratio of [desmethyltamoxifen] to [endoxifen] concentrations in individuals carrying the respective haplotypes. Since many of these haplotypes are in LD with known defective or reduced‐function *CYP2D6* alleles, we focused the analysis on patients who exclusively carried the *CYP2D6*1, CYP2D6*2*, and/or *CYP2D6*35* alleles (**Supplementary Figure** [S2](#cpt3680-supitem-0001)).

### Targeted analyses of haplotypes in carriers of the CYP2D6*1, CYP2D6*2, and CYP2D6*35 alleles

Of the 27 haplotypes identified, we found six variant haplotypes significantly associated with altered CYP2D6 activity (**Figure** [4](#cpt3680-fig-0004)). The effects of these six haplotypes on CYP2D6 activity, their associations with other genes and genomic regions, and their frequencies are presented in **Table** [3](#cpt3680-tbl-0003). Among these, haplotypes J1 and L, both with high frequencies, were linked to a 36% and 39% increase in the log [product]/[substrate] ratio, respectively. Haplotype E and O1 were associated with a 15–20% increase in activity, while B1 and C2 were linked to a 10–15% decrease in activity.

### Figure 4.

![Figure 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f9ec/12166262/680ec3ecf9e5/CPT-118-218-g003.jpg)

Effect of haplotypes on log [endoxifen]/[desmethyltamoxifen] ratio. Effect of haplotypes J1, L, C2, E, B1, and O1 on log [endoxifen]/[desmethyltamoxifen] ratio in patients carrying CYP2D6*1 and being heterozygous for CYP2D6*2 or CYP2D6*35. ***P < 0.001, **P < 0.01, *P < 0.05 according to Mann–Whitney U test.

### Table 3.

| Haplotype | Position GRCh37 (GRCh38) | Gene(s) | Marker SNP | Effect on ratio [endoxifen]/[desmethyl‐tamoxifen]a |
| --- | --- | --- | --- | --- |
| B1 | 22:42255525–22:42281429 (22:41859521–22:41885425) | SREBF2 intron 1, intron 3, 4, 10 SEPTIN3 | rs133290 | CC > CA 10% lower* |
| C2 | 22:42378070–22:42538509 (22:41982066–22:42538509) | SEPTIN3 CYP2D7 | rs11914200 | GG > GA 15% lower* |
| E | 22:42344297–22:42365073 (22:41948293–22:41969069) | CENPM LINC00634 intergenic SEPTIN3 | rs8140869 | GA > AA 15% higher* |
| J1 | 22:42189230–22:42246392 (22:41793226–22:41850388) | MEI1 CCDC134 SREBF2 intron 1 | rs2267439 | CC > TT 39% higher** CT > TT 28% higher*** |
| L | 22:42640606–22:42680800 (22:42244600–22:42284794) | TCF20 5′‐UTR, intron 1 | rs5751251 | GG > CC 36% higher* GG > GC 20% higher* |
| O1 | 22:42225018–22:42247695 (22:41829014–22:41851691) | CCDC134 SREBF2 intron 1, 5′‐UTR | rs9607850 | CC > TT 20% higher* |

Table 3 Caption: Properties of 6 different haplotypes identified associarted with altered rate of desnethyltamoxiden metabolism in patients

Most genetic variants linked to these haplotypes are located downstream of the *CYP2D6* gene. Notably, variants within haplotypes B1, J1, and O1 are situated within intron 1 of the *SREBF2* gene (**Table** [3](#cpt3680-tbl-0003), **Figure** [3](#cpt3680-fig-0003)). Haplotype E contains single nucleotide polymorphisms (SNPs) in the *SMIM45* gene, which encodes small integral membrane protein 45 and long non‐coding RNA 634 (LINC00634), while all genetic variants in haplotype L are located in the *TCF20 gene*, over 100 kb upstream of the *CYP2D6* gene (**Table** [3](#cpt3680-tbl-0003), **Figure** [3](#cpt3680-fig-0003)).

An analysis of haplotype distribution among *CYP2D6* alleles revealed that 90% of *CYP2D6*1* alleles carried haplotype J1, compared to only 62% of *CYP2D6**2 alleles, suggesting a genetic contribution to the reduced activity observed in *CYP2D6*2* carriers (**Supplementary Table** [S3](#cpt3680-supitem-0001)). In addition to these haplotype associations, we identified 15 of the SNPs from the GWAS, which were not grouped into haplotypes, to be significant for log MR [desmethyltamoxifen]/[endoxifen] among carriers of *CYP2D6*1*, *CYP2D6*2*, and *CYP2D6*35* (**Figure** [3](#cpt3680-fig-0003)). Five of these genetic variants are located within the *SREBF2* gene, and six are located in the *TCF20* gene (**Figure** [3](#cpt3680-fig-0003)).

### Pharmacogenomic prediction of CYP2D6‐dependent metabolism of desmethyltamoxifen and risperidone

Based on substrate specificity differences among CYP2D6 enzyme variants and the presence of the haplotype J1, we developed prediction algorithms to evaluate the influence of allelic activity scores (AS) of different CYP2D6 variants on the metabolism of desmethyltamoxifen and risperidone. According to CPIC guidelines, the activity scores for *CYP2D6*2* and *CYP2D6*35* are considered equivalent to *CYP2D6*1*,[28](#cpt3680-bib-0028) while activity scores for *CYP2D6*9, CYP2D6*10*, and *CYP2D6*41* are set at 25%. However, based on our previously published data involving nearly 5,000 patients[15](#cpt3680-bib-0015) those for *CYP2D6*9* and *CYP2D6*10* are approximately 35%, and the activity score for *CYP2D6*41* is about 15%. Based on the regression analyses of the rates of metabolism of desmethyltamoxifen and risperidone (**Supplementary Figure** [S3](#cpt3680-supitem-0001)) we find that the activity scores for *CYP2D6*2* and *CYP2D6*35* are 50%. Using the revised activity scores (see **Table** [1](#cpt3680-tbl-0001)), we developed a predictive algorithm and compared its performance against the CPIC‐defined activity score‐based algorithm. The differences in activity scores relative to CPIC are primarily attributed to the 50% activity decrease in *CYP2D6*2* and *CYP2D6*35*, whereas the altered activity scores for *CYP2D6*9, CYP2D6*10*, and *CYP2D6*41* contribute to the 10% of the change (**Supplementary Table** [S4](#cpt3680-supitem-0001)).

This comparison enabled us to evaluate the predictive power of our approach in accounting for the metabolic variability of desmethyltamoxifen and risperidone metabolism, highlighting the impact of updated activity score estimates. This comparison revealed that the adjusted *R* ^2^ value improved from 59% using the CPIC activity definitions to 70.6% for desmethyltamoxifen when using the AS:s provided in **Table** [1](#cpt3680-tbl-0001) (**Figure** [5 a,b](#cpt3680-fig-0005)). Further incorporating the *CYP2D6*2* and *CYP2D6*35* alleles having the J1 haplotype (Cf. **Figure** [4](#cpt3680-fig-0004)) increased the *R* ^2^ value to 71.2% (**Figure** [5 c](#cpt3680-fig-0005)). Much lower prediction rates were found for risperidone, where the adjusted *R* ^2^ using CPIC AS:s was 42.2% and increased to 46.2% using the AS:s in **Table** [1](#cpt3680-tbl-0001) (**Figure** [6 a,b](#cpt3680-fig-0006)) which are not statistically significant. The smaller increase in predictability for risperidone is expected due to the lower influence on risperidone metabolism by CYP2D6.2 and CYP2D6.35 (**Figure** [1](#cpt3680-fig-0001)).

### Figure 5.

![Figure 5](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f9ec/12166262/4a7dc167cbdb/CPT-118-218-g004.jpg)

Evaluation of the CYP2D6*2 and CYP2D6*35 haplotype desmethyltamoxifen activity score. Evaluation of the CYP2D6*2 and CYP2D6*35 haplotype activity score based on the desmethyltamoxifen product‐to‐substrate ratio and its relevance to the prediction of CYP2D6 metabolic capacity by CYP2D6 genotype. The improvement in the prediction of the desmethyltamoxifen product‐to‐substrate ratio by CYP2D6 genotype if activity scores for CYP2D6 alleles were revised as provided in Table 3 is illustrated on the plot showing (a) the prediction power based on activity scores taken as currently suggested by the CPIC guidelines, (b) the presently proposed revised activity scores, (c) the presently proposed revised activity scores with discrimination between carriers and non‐carriers of J Haplotype among *2 and *35 carriers.

### Figure 6.

![Figure 6](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f9ec/12166262/326ecd26d02a/CPT-118-218-g001.jpg)

Evaluation of the CYP2D6*2 and CYP2D6*35 haplotype risperidone activity score. Evaluation of the CYP2D6*2 and CYP2D6*35 haplotype activity score based on risperidone product‐to‐substrate ratios and its relevance to the prediction of CYP2D6 metabolic capacity by CYP2D6 genotype. The improvement in the prediction of the risperidone product‐to‐substrate ratio by CYP2D6 genotype if activity scores for CYP2D6 alleles were revised as provided in Table 1 is illustrated in the plot showing (a) the prediction power based on activity scores taken as currently suggested by the CPIC guidelines, (b) the presently proposed revised activity scores.

## DISCUSSION

In this contribution we initially focused on the role of CYP2D6.2 and CYP2D6.35 on the interindividual variability in the metabolism of desmethyltamoxifen and risperidone *in vivo* and found that the adjusted *R* ^2^ genetically predicted value of desmethyltamoxifen metabolism increased from 59% to 70.6% taking the reduced capacity of the CYP2D6.2 and CYP2D6.35 enzymes into account (**Figure** [5](#cpt3680-fig-0005)). Furthermore, we outline the importance of novel haplotypes within chromosome 22 as regulators of CYP2D6 expression, further contributing to reducing the extent of missing heritability evident from a further increase of *R* ^2^ to 71.4% when such variants were taken into account. In the study by van der Lee et al[11](#cpt3680-bib-0011) Significant effort was dedicated to analyzing the same cohort used in this study to identify genetic variants associated with endoxifen levels. Using an algorithm trained for the genetic variants seen based on long‐read sequencing of the *CYP2D6* gene, van der Lee et al. achieved a prediction accuracy of 79% for interindividual variability in CYP2D6 activity. In a replication cohort of tamoxifen‐treated patients, the model demonstrated an adjusted *R* ^2^ of 0.66, compared to an adjusted *R* ^2^ of 0.35 for the star‐allele‐based approach. Thus, in our case, the star‐allele‐based analyses in the model resulted in better prediction than the long‐read based algorithm in the replication cohorts. Since our model is mainly based on allelic variants being analyzed routinely by RT‐PCR and arrays, the application does not require DNA sequencing.[14](#cpt3680-bib-0014), [29](#cpt3680-bib-0029), [30](#cpt3680-bib-0030)

This study highlights the varying activity scores observed in specific gene variant–drug pairs, particularly among CYP2D6 enzyme variants with different combinations of amino acid substitutions. These differences suggest that a single gene variant may exhibit distinct activity scores depending on the substrate. However, such detailed assessments of drug‐specific CYP2D6 variant activity scores have not yet been incorporated into pharmacogenomic guidelines from CPIC or DPWG, nor in regulatory documents from the FDA.

Given that these CYP2D6 enzyme variants are relatively common in European populations and have been shown to exhibit significant differences in substrate specificity in vitro, their inclusion in pharmacogenomic guidelines should be considered[29](#cpt3680-bib-0029). This is particularly relevant for CYP2D6.2 *and* CYP2D6.35 in combination with drugs such as risperidone and desmethyltamoxifen. Recent findings by Størset et al.[14](#cpt3680-bib-0014) further support this, indicating that both CYP2D6.2 and CYP2D6.35 are associated with reduced risperidone clearance. These observed differences serve as proof of principle and are likely applicable to many of the other > 100 CYP2D6‐metabolized drugs, an area that needs future investigations. With respect to the substrates here considered, it is clear that the prediction effects seen are greater for desmethyltamoxifen compared to risperidone most likely because risperidone is also metabolized by CYP3A4[25](#cpt3680-bib-0025) and directly eliminated via the kidneys, especially in patients having reduced CYP2D6 activity[26](#cpt3680-bib-0026), [31](#cpt3680-bib-0031) which means that renal function also affects the [product]/[substrate] ratios.

The marked reduction in CYP2D6.2‐dependent metabolism of desmethyltamoxifen and risperidone in relation to CYP2D6.1 (**Figure** [1](#cpt3680-fig-0001)) might explain the nonlinear increase in CYP2D6 metabolism observed for these substrates in patients classified as ultra‐rapid metabolizers (UMs) based on copy number. Thus, for specific substrates, the rate of CYP2D6 activity in individuals with *CYP2D6*2* duplication is lower than expected, with UMs with a CYP2D6*2 duplication typically exhibiting only a 20–30% higher risperidone metabolism activity than those homozygous for *CYP2D6*1*.[15](#cpt3680-bib-0015), [19](#cpt3680-bib-0019), [32](#cpt3680-bib-0032) Among UMs, over 90% carry duplicated *CYP2D6*2* alleles. In contrast, duplicated *CYP2D6*1* alleles are predominantly found in Micronesia, arising from a separate evolutionary event leading to gene duplication.[33](#cpt3680-bib-0033) Thus, it is essential to analyze which specific *CYP2D6* allele is duplicated in order to provide precise genotype–phenotype predictions of CYP2D6‐dependent drug metabolism for diplotypes with increased copy number.

*In vitro* analysis of the three CYP2D6 enzyme variants showed similar bufuralol metabolism across all variants. However, CYP2D6.2 and CYP2D6.35 exhibited lower Kcat/Km ratios for risperidone and desmethyltamoxifen compared to CYP2D6.1, consistent with patient data. The observed differences in substrate handling as the only explanation for lower CYP2D6.2 activity may be partly biased by the J1 haplotype of *CYP2D6*1*, which enhances CYP2D6.1 activity by 33% and is more common in *CYP2D6*1* than in *CYP2D6*2* (**Supplementary Table** [S4](#cpt3680-supitem-0001)).

Spitman et al. identified 430 genetic variants in the *TCF20* and *WBP2NL* genes, which are in strong linkage disequilibrium with *CYP2D6* gene variants.[13](#cpt3680-bib-0013) We used a haplotype‐based approach within a 500 kb region, which revealed 27 distinct haplotypes (**Figure** [3](#cpt3680-fig-0003); **Supplementary Table** [S2](#cpt3680-supitem-0001)). Of these, six haplotypes, obtained after the elimination of haplotypes in LD to known functional SNPs, were significantly associated with the ratio [endoxifen]/[desmethyltamoxifen] (**Table** [1](#cpt3680-tbl-0001)). Notably, two haplotypes (J1 and L) were linked to a 36–39% higher log [endoxifen]/[desmethyltamoxifen] ratio (**Table** [3](#cpt3680-tbl-0003)), with haplotype J1 having the most substantial impact. The effect of the J1 haplotype is restricted to *CYP2D6*2* and *CYP2D6*35* alleles (**Supplementary Figure** [S4](#cpt3680-supitem-0001)) due to a much higher frequency of the J1 haplotype in the *CYP2D6*1* allele (**Supplementary Table** [S3](#cpt3680-supitem-0001)).

The marker SNPs for haplotype B1 had previously been associated with endoxifen levels,[34](#cpt3680-bib-0034) where the authors also identified rs1894714 and rs11914200, which are part of haplotypes J2 and C2, respectively. SNPs significantly associated with endoxifen levels and linked to the C2 haplotype were also found in the GWAS by Khor et al.[27](#cpt3680-bib-0027) This SNP, located in the *CYP2D7* gene, is in linkage disequilibrium (*r* ^2^ = 0.53) with the *CYP2D6*4* allele. However, its effect persisted when we tested the C2 haplotype in subjects carrying *CYP2D6*1, CYP2D6*2*, or *CYP2D6*35* alleles (**Table** [3](#cpt3680-tbl-0003)).

The region extending approximately 150 kb upstream and 350 kb downstream from the *CYP2D6* gene clearly contains several genetic variants that significantly influence its expression. While the specific polymorphic elements responsible have yet to be identified, advancing our understanding of these variants will likely enhance our ability to predict interindividual differences in *CYP2D6* gene expression.

The *CYP2D6*2* allele is highly prevalent, according to PharmGKB[17](#cpt3680-bib-0017) and PharmFreq[35](#cpt3680-bib-0035) having frequencies between 18 to 27% in different parts of the world. According to PharmVar, CPIC, and DPWG, the allele, as mentioned, is classified as having normal activity. However, based on the *in vivo* and *in vitro* data presented here, as well as *in vitro* data from Hiratsuka's group,[8](#cpt3680-bib-0008) it is evident that *CYP2D6*2* encodes an enzyme with reduced activity for certain substrates, such as risperidone and desmethyltamoxifen. However, CYP2D6.2 does not impact the CYP2D6‐mediated metabolism of bufuralol (**Table** [2](#cpt3680-tbl-0002)), dextromethorphan,[10](#cpt3680-bib-0010) and codeine[36](#cpt3680-bib-0036) This is an important issue that evidently should be implemented in current as well as in the future CPIC and DPWG guidelines concomitant to the increased information on this point.

In conclusion, there is compelling evidence to support the inclusion of *CYP2D6*2 and CYP2D6*35* as reduced‐function alleles in future pharmacogenomic guidelines for specific drug substrates. Notably, the principle of substrate‐specific differences in CYP2D6 activity caused by amino acid substitutions extends to other common CYP2D6 enzyme variants, such as CYP2D6.2 in Europe, CYP2D6.10 in Asia, and CYP2D6.17 in Africa, underscoring its broad applicability. This principle is also relevant to many of the 100 additional drugs metabolized by CYP2D6, particularly those with high selectivity for the enzyme, whose metabolism is significantly influenced by the substrate specificity of the different CYP2D6 variants. We strongly advocate for pharmacogenomic guidelines from CPIC and DPWG to incorporate substrate‐specific polymorphisms where applicable, given their substantial clinical impact. Furthermore, we recommend that regulatory bodies such as the FDA and EMA integrate these considerations into drug product information sheets to improve the clinical value of pharmacogenomic recommendations.

## FUNDING

The research was funded by the Swedish Research Council (grant 2021‐02732), (MIS & EM) the ZonMW Hotel grant 435004018 (JJS), The Swedish Cancer Society, (grant 23‐0763PT) (YZ), The Science Fund of the Republic of Serbia (grant 6066800) (MMJ), and the Swedish Brain Foundation (grant FO2023‐0139) (MIS & MMJ).

## CONFLICT OF INTEREST

Magnus Ingelman‐Sundberg, Co‐founder and shareholder in HepaPredict AB. All other authors declared no competing interests for this work.

## AUTHOR CONTRIBUTIONS

MIS, IJ, JJS, HJG, MMJ, EM wrote the manuscript; MIS, IJ, MMJ, HJG designed the research; AS, JS, ES, EM performed the research; MIS, IJ, JJS, HJG, MMJ, EM, ES analyzed the data.

## Supporting information

## DATA AVAILABILITY STATEMENT

All data are available from the authors.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

All data are available from the authors.

### Supplementary Materials

### Data Availability Statement

All data are available from the authors.

## References

1. Table of Pharmacogenetic Associations . <https://www.fda.gov/medical‐devices/precision‐medicine/table‐pharmacogenetic‐associations#section1>.  [https://www.fda.gov/medical‐devices/precision‐medicine/table‐pharmacogenetic‐associations#section1](https://www.fda.gov/medical-devices/precision-medicine/table-pharmacogenetic-associations#section1)

2. Matthaei, J. et al. Heritability of metoprolol and torsemide pharmacokinetics. Clin. Pharmacol. Ther. 98, 611–621 (2015).  [DOI](https://doi.org/10.1002/cpt.258) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26344676/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Heritability%20of%20metoprolol%20and%20torsemide%20pharmacokinetics&author=J.%20Matthaei&volume=98&publication_year=2015&pages=611-621&pmid=26344676&doi=10.1002/cpt.258&)

3. Ingelman‐Sundberg, M. Pharmacogenomic prescribing guidelines: are they always useful? Clin. Pharmacol. Ther. 0, 1–3 (2024).  [DOI](https://doi.org/10.1002/cpt.3403) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/39101255/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Pharmacogenomic%20prescribing%20guidelines:%20are%20they%20always%20useful?&author=M.%20Ingelman%E2%80%90Sundberg&volume=0&publication_year=2024&pages=1-3&pmid=39101255&doi=10.1002/cpt.3403&)

4. Jukic, M. , Milosavljević, F. , Molden, E. & Ingelman‐Sundberg, M. Pharmacogenomics in treatment of depression and psychosis: an update. Trends Pharmacol. Sci. 43, 1055–1069 (2022).  [DOI](https://doi.org/10.1016/j.tips.2022.09.011) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36307251/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Trends%20Pharmacol.%20Sci.&title=Pharmacogenomics%20in%20treatment%20of%20depression%20and%20psychosis:%20an%20update&author=M.%20Jukic&author=F.%20Milosavljevi%C4%87&author=E.%20Molden&author=M.%20Ingelman%E2%80%90Sundberg&volume=43&publication_year=2022&pages=1055-1069&pmid=36307251&doi=10.1016/j.tips.2022.09.011&)

5. Zanger, U.M. & Schwab, M. Cytochrome P450 enzymes in drug metabolism: regulation of gene expression, enzyme activities, and impact of genetic variation. Pharmacol. Ther. 138, 103–141 (2013).  [DOI](https://doi.org/10.1016/j.pharmthera.2012.12.007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23333322/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacol.%20Ther.&title=Cytochrome%20P450%20enzymes%20in%20drug%20metabolism:%20regulation%20of%20gene%20expression,%20enzyme%20activities,%20and%20impact%20of%20genetic%20variation&author=U.M.%20Zanger&author=M.%20Schwab&volume=138&publication_year=2013&pages=103-141&pmid=23333322&doi=10.1016/j.pharmthera.2012.12.007&)

6. Pharmacogene Variation Consortium . <https://www.pharmvar.org/>.  [https://www.pharmvar.org/](https://www.pharmvar.org/)

7. Gaedigk, A. , Dinh, J.C. , Jeong, H. , Prasad, B. & Leeder, J.S. Ten years' experience with the CYP2D6 activity score: a perspective on future investigations to improve clinical predictions for precision therapeutics. J. Pers. Med. 8, 1–15 (2018).  [DOI](https://doi.org/10.3390/jpm8020015) | [PMC free article](/articles/PMC6023391/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29673183/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Pers.%20Med.&title=Ten%20years'%20experience%20with%20the%20CYP2D6%20activity%20score:%20a%20perspective%20on%20future%20investigations%20to%20improve%20clinical%20predictions%20for%20precision%20therapeutics&author=A.%20Gaedigk&author=J.C.%20Dinh&author=H.%20Jeong&author=B.%20Prasad&author=J.S.%20Leeder&volume=8&publication_year=2018&pages=1-15&pmid=29673183&doi=10.3390/jpm8020015&)

8. Muroi, Y. et al. Functional characterization of wild‐type and 49 CYP2D6 allelic variants for N‐desmethyltamoxifen 4‐hydroxylation activity. Drug Metab. Pharmacokinet. 29, 360–366 (2014).  [DOI](https://doi.org/10.2133/dmpk.dmpk-14-rg-014) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24647041/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab.%20Pharmacokinet.&title=Functional%20characterization%20of%20wild%E2%80%90type%20and%2049%20CYP2D6%20allelic%20variants%20for%20N%E2%80%90desmethyltamoxifen%204%E2%80%90hydroxylation%20activity&author=Y.%20Muroi&volume=29&publication_year=2014&pages=360-366&pmid=24647041&doi=10.2133/dmpk.dmpk-14-rg-014&)

9. Oscarson, M. , Hidestrand, M. , Johansson, I. & Ingelman‐Sundberg, M. A combination of mutations in the CYP2D6*17 (CYP2D6Z) allele causes alterations in enzyme function. Mol. Pharmacol. 52, 1034–1040 (1997).  [DOI](https://doi.org/10.1124/mol.52.6.1034) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9415713/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Pharmacol.&title=A%20combination%20of%20mutations%20in%20the%20CYP2D6*17%20(CYP2D6Z)%20allele%20causes%20alterations%20in%20enzyme%20function&author=M.%20Oscarson&author=M.%20Hidestrand&author=I.%20Johansson&author=M.%20Ingelman%E2%80%90Sundberg&volume=52&publication_year=1997&pages=1034-1040&pmid=9415713&doi=10.1124/mol.52.6.1034&)

10. Van Der Lee, M. , Guchelaar, H.J. & Swen, J.J. Substrate specificity of CYP2D6 genetic variants. Pharmacogenomics 22, 1081–1089 (2021).  [DOI](https://doi.org/10.2217/pgs-2021-0093) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34569808/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Substrate%20specificity%20of%20CYP2D6%20genetic%20variants&author=M.%20Van%20Der%20Lee&author=H.J.%20Guchelaar&author=J.J.%20Swen&volume=22&publication_year=2021&pages=1081-1089&pmid=34569808&doi=10.2217/pgs-2021-0093&)

11. Van der Lee, M. , Allard, W.G. , Vossen, R.H.A.M. et al. Toward predicting CYP2D6‐mediated variable drug response from CYP2D6 gene sequencing data. Sci. Transl. Med. 13, abf3637 (2021). 10.1126/scitranslmed.abf3637.  [DOI](https://doi.org/10.1126/scitranslmed.abf3637) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34290055/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sci.%20Transl.%20Med.&title=Toward%20predicting%20CYP2D6%E2%80%90mediated%20variable%20drug%20response%20from%20CYP2D6%20gene%20sequencing%20data&author=M.%20Van%20der%20Lee&author=W.G.%20Allard&author=R.H.A.M.%20Vossen&volume=13&publication_year=2021&pages=abf3637&pmid=34290055&doi=10.1126/scitranslmed.abf3637&)

12. Kevin Hicks, J. , Jesse, J.S. & Andrea, G. Challenges in CYP2D6 phenotype assignment from genotype data: a critical assessment and call for standardization. Curr. Drug Metab. 15, 218–232 (2014).  [DOI](https://doi.org/10.2174/1389200215666140202215316) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24524666/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr.%20Drug%20Metab.&title=Challenges%20in%20CYP2D6%20phenotype%20assignment%20from%20genotype%20data:%20a%20critical%20assessment%20and%20call%20for%20standardization&author=J.%20Kevin%20Hicks&author=J.S.%20Jesse&author=G.%20Andrea&volume=15&publication_year=2014&pages=218-232&pmid=24524666&doi=10.2174/1389200215666140202215316&)

13. Sanchez‐Spitman, A.B. , Böhringer, S. , Dezentjé, V.O. , Gelderblom, H. , Swen, J.J. & Guchelaar, H.J. A genome‐wide association study of Endoxifen serum concentrations and adjuvant tamoxifen efficacy in early‐stage breast cancer patients. Clin. Pharmacol. Ther. 116, 155–164 (2024).  [DOI](https://doi.org/10.1002/cpt.3255) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/38501904/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=A%20genome%E2%80%90wide%20association%20study%20of%20Endoxifen%20serum%20concentrations%20and%20adjuvant%20tamoxifen%20efficacy%20in%20early%E2%80%90stage%20breast%20cancer%20patients&author=A.B.%20Sanchez%E2%80%90Spitman&author=S.%20B%C3%B6hringer&author=V.O.%20Dezentj%C3%A9&author=H.%20Gelderblom&author=J.J.%20Swen&volume=116&publication_year=2024&pages=155-164&pmid=38501904&doi=10.1002/cpt.3255&)

14. Størset, E. , Bråten, L.S. , Ingelman‐Sundberg, M. , Johansson, I. , Molden, E. & Kringen, M.K. Impact of CYP2D6*2, CYP2D6*35, rs5758550, and related haplotypes on risperidone clearance in vivo. Eur. J. Clin. Pharmacol. 80, 1531–1541 (2024).  [DOI](https://doi.org/10.1007/s00228-024-03721-6) | [PMC free article](/articles/PMC11393095/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/38963454/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Clin.%20Pharmacol.&title=Impact%20of%20CYP2D6*2,%20CYP2D6*35,%20rs5758550,%20and%20related%20haplotypes%20on%20risperidone%20clearance%20in%C2%A0vivo&author=E.%20St%C3%B8rset&author=L.S.%20Br%C3%A5ten&author=M.%20Ingelman%E2%80%90Sundberg&author=I.%20Johansson&author=E.%20Molden&volume=80&publication_year=2024&pages=1531-1541&pmid=38963454&doi=10.1007/s00228-024-03721-6&)

15. Jukić, M.M. , Smith, R.L. , Molden, E. & Ingelman‐Sundberg, M. Evaluation of the CYP2D6 haplotype activity scores based on metabolic ratios of 4,700 patients treated with three different CYP2D6 substrates. Clin. Pharmacol. Ther. 110, 750–758 (2021).  [DOI](https://doi.org/10.1002/cpt.2246) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33792048/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Evaluation%20of%20the%20CYP2D6%20haplotype%20activity%20scores%20based%20on%20metabolic%20ratios%20of%204,700%20patients%20treated%20with%20three%20different%20CYP2D6%20substrates&author=M.M.%20Juki%C4%87&author=R.L.%20Smith&author=E.%20Molden&author=M.%20Ingelman%E2%80%90Sundberg&volume=110&publication_year=2021&pages=750-758&pmid=33792048&doi=10.1002/cpt.2246&)

16. CYPTAM study . <https://onderzoekmetmensen.nl/en/node/23389/pdf>.  [https://onderzoekmetmensen.nl/en/node/23389/pdf](https://onderzoekmetmensen.nl/en/node/23389/pdf)

17. PharmGKB . <https://www.pharmgkb.org/>.  [https://www.pharmgkb.org/](https://www.pharmgkb.org/)

18. Kronbach, T. , Mathys, D. , Gut, J. , Catin, T. & Meyer, U.A. High‐performance liquid chromatographic assays for bufuralol 1′‐hydroxylase, debrisoquine 4‐hydroxylase, and dextromethorphan O‐demethylase in microsomes and purified cytochrome P‐450 isozymes of human liver. Anal. Biochem. 162, 24–32 (1987).  [DOI](https://doi.org/10.1016/0003-2697(87)90006-6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/3605590/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anal.%20Biochem.&title=High%E2%80%90performance%20liquid%20chromatographic%20assays%20for%20bufuralol%201%E2%80%B2%E2%80%90hydroxylase,%20debrisoquine%204%E2%80%90hydroxylase,%20and%20dextromethorphan%20O%E2%80%90demethylase%20in%20microsomes%20and%20purified%20cytochrome%20P%E2%80%90450%20isozymes%20of%20human%20liver&author=T.%20Kronbach&author=D.%20Mathys&author=J.%20Gut&author=T.%20Catin&author=U.A.%20Meyer&volume=162&publication_year=1987&pages=24-32&pmid=3605590&doi=10.1016/0003-2697(87)90006-6&)

19. Lenk, H.Ç. et al. The polymorphic nuclear factor NFIB regulates hepatic CYP2D6 expression and influences risperidone metabolism in psychiatric patients. Clin. Pharmacol. Ther. 111, 1165–1174 (2022).  [DOI](https://doi.org/10.1002/cpt.2571) | [PMC free article](/articles/PMC9314634/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35253216/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=The%20polymorphic%20nuclear%20factor%20NFIB%20regulates%20hepatic%20CYP2D6%20expression%20and%20influences%20risperidone%20metabolism%20in%20psychiatric%20patients&author=H.%C3%87.%20Lenk&volume=111&publication_year=2022&pages=1165-1174&pmid=35253216&doi=10.1002/cpt.2571&)

20. LDLlink . <https://ldlink.nih.gov>.  [https://ldlink.nih.gov](https://ldlink.nih.gov)

21. Ensembl . <www.ensembl.org>.  [www.ensembl.org](http://www.ensembl.org)

22. Mirdita, M. , Schütze, K. , Moriwaki, Y. , Heo, L. , Ovchinnikov, S. & Steinegger, M. ColabFold: making protein folding accessible to all. Nat. Methods 19, 679–682 (2022).  [DOI](https://doi.org/10.1038/s41592-022-01488-1) | [PMC free article](/articles/PMC9184281/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35637307/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Methods&title=ColabFold:%20making%20protein%20folding%20accessible%20to%20all&author=M.%20Mirdita&author=K.%20Sch%C3%BCtze&author=Y.%20Moriwaki&author=L.%20Heo&author=S.%20Ovchinnikov&volume=19&publication_year=2022&pages=679-682&pmid=35637307&doi=10.1038/s41592-022-01488-1&)

23. Eberhardt, J. , Santos‐Martins, D. , Tillack, A.F. & Forli, S. AutoDock Vina 1.2.0: new docking methods, expanded force field, and python bindings. J. Chem. Inf. Model. 61, 3891–3898 (2021).  [DOI](https://doi.org/10.1021/acs.jcim.1c00203) | [PMC free article](/articles/PMC10683950/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34278794/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Chem.%20Inf.%20Model.&title=AutoDock%20Vina%201.2.0:%20new%20docking%20methods,%20expanded%20force%20field,%20and%20python%20bindings&author=J.%20Eberhardt&author=D.%20Santos%E2%80%90Martins&author=A.F.%20Tillack&author=S.%20Forli&volume=61&publication_year=2021&pages=3891-3898&pmid=34278794&doi=10.1021/acs.jcim.1c00203&)

24. Rowland, P. et al. Crystal structure of human cytochrome P450 2D6. J. Biol. Chem. 281, 7614–7622 (2006).  [DOI](https://doi.org/10.1074/jbc.M511232200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16352597/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem.&title=Crystal%20structure%20of%20human%20cytochrome%20P450%202D6&author=P.%20Rowland&volume=281&publication_year=2006&pages=7614-7622&pmid=16352597&doi=10.1074/jbc.M511232200&)

25. Dantonio, A.L. , Doran, A.C. & Obach, R.S. Intersystem extrapolation factors are substrate‐dependent for CYP3A4: impact on cytochrome P450 reaction phenotyping. Drug Metab. Dispos. 50, 249–257 (2022).  [DOI](https://doi.org/10.1124/dmd.121.000758) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34903590/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab.%20Dispos.&title=Intersystem%20extrapolation%20factors%20are%20substrate%E2%80%90dependent%20for%20CYP3A4:%20impact%20on%20cytochrome%20P450%20reaction%20phenotyping&author=A.L.%20Dantonio&author=A.C.%20Doran&author=R.S.%20Obach&volume=50&publication_year=2022&pages=249-257&pmid=34903590&doi=10.1124/dmd.121.000758&)

26. Mannens, G. , Huang, M.L. , Meuldermans, W. , Hendrickx, J. , Woestenborghs, R. & Heykants, J. Absorption, metabolism, and excretion of risperidone in humans. Drug Metab. Dispos. 21, 1134–1141 (1993).  [PubMed](https://pubmed.ncbi.nlm.nih.gov/7507814/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab.%20Dispos.&title=Absorption,%20metabolism,%20and%20excretion%20of%20risperidone%20in%20humans&author=G.%20Mannens&author=M.L.%20Huang&author=W.%20Meuldermans&author=J.%20Hendrickx&author=R.%20Woestenborghs&volume=21&publication_year=1993&pages=1134-1141&pmid=7507814&)

27. Khor, C.C. et al. Cross‐ancestry genome‐wide association study defines the extended CYP2D6 locus as the principal genetic determinant of Endoxifen plasma concentrations. Clin. Pharmacol. Ther. 113, 712–723 (2023).  [DOI](https://doi.org/10.1002/cpt.2846) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36629403/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Cross%E2%80%90ancestry%20genome%E2%80%90wide%20association%20study%20defines%20the%20extended%20CYP2D6%20locus%20as%20the%20principal%20genetic%20determinant%20of%20Endoxifen%20plasma%20concentrations&author=C.C.%20Khor&volume=113&publication_year=2023&pages=712-723&pmid=36629403&doi=10.1002/cpt.2846&)

28. CPIC guidelines . <https://cpicpgx.org/guidelines/>.  [https://cpicpgx.org/guidelines/](https://cpicpgx.org/guidelines/)

29. Saito, T. et al. Functional characterization of 50 CYP2D6 allelic variants by assessing primaquine 5‐hydroxylation. Drug Metab. Pharmacokinet. 33, 250–257 (2018).  [DOI](https://doi.org/10.1016/j.dmpk.2018.08.004) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30366777/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab.%20Pharmacokinet.&title=Functional%20characterization%20of%2050%20CYP2D6%20allelic%20variants%20by%20assessing%20primaquine%205%E2%80%90hydroxylation&author=T.%20Saito&volume=33&publication_year=2018&pages=250-257&pmid=30366777&doi=10.1016/j.dmpk.2018.08.004&)

30. Caudle, K.E. et al. Standardizing CYP2D6 genotype to phenotype translation: consensus recommendations from the clinical pharmacogenetics implementation consortium and Dutch pharmacogenetics working group. Clin. Transl. Sci. 13, 116–124 (2020).  [DOI](https://doi.org/10.1111/cts.12692) | [PMC free article](/articles/PMC6951851/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31647186/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Transl.%20Sci.&title=Standardizing%20CYP2D6%20genotype%20to%20phenotype%20translation:%20consensus%20recommendations%20from%20the%20clinical%20pharmacogenetics%20implementation%20consortium%20and%20Dutch%20pharmacogenetics%20working%20group&author=K.E.%20Caudle&volume=13&publication_year=2020&pages=116-124&pmid=31647186&doi=10.1111/cts.12692&)

31. Gründer, P. , Augustin, M. , Paulzen, M. & Gründer, G. Influence of kidney function on serum risperidone concentrations in patients treated with risperidone. J. Clin. Psychiatry 80, E1–E5 (2019).  [DOI](https://doi.org/10.4088/JCP.19m12890) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31774943/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Psychiatry&title=Influence%20of%20kidney%20function%20on%20serum%20risperidone%20concentrations%20in%20patients%20treated%20with%20risperidone&author=P.%20Gr%C3%BCnder&author=M.%20Augustin&author=M.%20Paulzen&author=G.%20Gr%C3%BCnder&volume=80&publication_year=2019&pages=E1-E5&pmid=31774943&doi=10.4088/JCP.19m12890&)

32. Jukic, M.M. , Smith, R.L. , Haslemo, T. , Molden, E. & Ingelman‐Sundberg, M. Effect of CYP2D6 genotype on exposure and efficacy of risperidone and aripiprazole: a retrospective, cohort study. Lancet Psychiatry 6, 418–426 (2019).  [DOI](https://doi.org/10.1016/S2215-0366(19)30088-4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31000417/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet%20Psychiatry&title=Effect%20of%20CYP2D6%20genotype%20on%20exposure%20and%20efficacy%20of%20risperidone%20and%20aripiprazole:%20a%20retrospective,%20cohort%20study&author=M.M.%20Jukic&author=R.L.%20Smith&author=T.%20Haslemo&author=E.%20Molden&author=M.%20Ingelman%E2%80%90Sundberg&volume=6&publication_year=2019&pages=418-426&pmid=31000417&doi=10.1016/S2215-0366(19)30088-4&)

33. Ingelman‐Sundberg, M. Genetic polymorphisms of cytochrome P450 2D6 (CYP2D6): clinical consequences, evolutionary aspects and functional diversity. Pharmacogenomics J. 5, 6–13 (2005).  [DOI](https://doi.org/10.1038/sj.tpj.6500285) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15492763/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J.&title=Genetic%20polymorphisms%20of%20cytochrome%20P450%202D6%20(CYP2D6):%20clinical%20consequences,%20evolutionary%20aspects%20and%20functional%20diversity&author=M.%20Ingelman%E2%80%90Sundberg&volume=5&publication_year=2005&pages=6-13&pmid=15492763&doi=10.1038/sj.tpj.6500285&)

34. Hennig, E.E. , Piatkowska, M. , Goryca, K. et al. Non‐CYP2D6 variants selected by a GWAS improve the prediction of impaired tamoxifen metabolism in patients with breast cancer. J. Clin. Med. 8, 1–16 (2019).  [DOI](https://doi.org/10.3390/jcm8081087) | [PMC free article](/articles/PMC6722498/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31344832/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Med.&title=Non%E2%80%90CYP2D6%20variants%20selected%20by%20a%20GWAS%20improve%20the%20prediction%20of%20impaired%20tamoxifen%20metabolism%20in%20patients%20with%20breast%20cancer&author=E.E.%20Hennig&author=M.%20Piatkowska&author=K.%20Goryca&volume=8&publication_year=2019&pages=1-16&pmid=31344832&doi=10.3390/jcm8081087&)

35. PharmFreq . <https://www.pharmfreq.com>.  [https://www.pharmfreq.com](https://www.pharmfreq.com)

36. Ashraf, M.W. et al. Population pharmacokinetic quantification of CYP2D6 activity in codeine metabolism in ambulatory surgical patients for model‐informed precision dosing. Clin. Pharmacokinet. 63, 1547–1560 (2024).  [DOI](https://doi.org/10.1007/s40262-024-01433-9) | [PMC free article](/articles/PMC11573879/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/39441506/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacokinet.&title=Population%20pharmacokinetic%20quantification%20of%20CYP2D6%20activity%20in%20codeine%20metabolism%20in%20ambulatory%20surgical%20patients%20for%20model%E2%80%90informed%20precision%20dosing&author=M.W.%20Ashraf&volume=63&publication_year=2024&pages=1547-1560&pmid=39441506&doi=10.1007/s40262-024-01433-9&)
