# Associations between UGT1A1, SLCO1B1, SLCO1B3, BLVRA and HMOX1 polymorphisms and susceptibility to neonatal severe hyperbilirubinemia in Chinese Han population

## Metadata
**Authors:** Juan Fan, Hua-Yun He, Huan-Huan Li, Pi-Liu Wu, Lei Tang, Bo-Yin Deng, Wen-Hui Dong, Jian-Hui Wang
**Journal:** BMC Pediatrics
**Date:** 2024 Jan 26
**DOI:** [10.1186/s12887-024-04537-0](https://doi.org/10.1186/s12887-024-04537-0)
**PMID:** 38279097
**PMCID:** PMC10811893
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10811893/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC10811893/pdf/12887_2024_Article_4537.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC10811893/pdf/12887_2024_Article_4537.pdf)

## Abstract

**Background:** 
Severe neonatal hyperbilirubinemia could lead to kernicterus and neonatal death. This study aimed to analyze the association between single nucleotide polymorphisms in genes involved in bilirubin metabolism and the incidence of severe hyperbilirubinemia.

**Methods:** 
A total of 144 neonates with severe hyperbilirubinemia and 50 neonates without or mild hyperbilirubinemia were enrolled in 3 institutions between 2019 and 2020. Twelve polymorphisms of 5 genes (UGT1A1, SLCO1B1, SLCO1B3, BLVRA, and HMOX1) were analyzed by PCR amplification of genomic DNA. Genotyping was performed using an improved multiplex ligation detection reaction technique based on ligase detection reaction.

**Results:** 
The frequencies of the A allele in UGT1A1-rs4148323 and the C allele in SLCO1B3-rs2417940 in the severe hyperbilirubinemia group (30.2% and 90.6%, respectively) were significantly higher than those in the controls (30.2% vs.13.0%, 90.6% vs. 78.0%, respectively, both p < 0.05). Haplotype analysis showed the ACG haplotype of UGT1A1 were associated with an increased hyperbilirubinemia risk (OR 3.122, p = 0.001), whereas the GCG haplotype was related to a reduced risk (OR 0.523, p = 0.018).

**Conclusion:** 
The frequencies of the A allele in rs4148323 and the C allele in rs2417940 are highly associated with the incidence of severe hyperbilirubinemia in Chinese Han neonates.

**Trial registration:** 
Trial registration number:ChiCTR1800020424; Date of registration:2018-12-29.

Keywords: Single nucleotide polymorphism, UGT1A1, SLCO1B3, Neonatal hyperbilirubinemia

### Background

Severe neonatal hyperbilirubinemia could lead to kernicterus and neonatal death. This study aimed to analyze the association between single nucleotide polymorphisms in genes involved in bilirubin metabolism and the incidence of severe hyperbilirubinemia.

### Methods

A total of 144 neonates with severe hyperbilirubinemia and 50 neonates without or mild hyperbilirubinemia were enrolled in 3 institutions between 2019 and 2020. Twelve polymorphisms of 5 genes (UGT1A1, SLCO1B1, SLCO1B3, BLVRA, and HMOX1) were analyzed by PCR amplification of genomic DNA. Genotyping was performed using an improved multiplex ligation detection reaction technique based on ligase detection reaction.

### Results

The frequencies of the A allele in UGT1A1-rs4148323 and the C allele in SLCO1B3-rs2417940 in the severe hyperbilirubinemia group (30.2% and 90.6%, respectively) were significantly higher than those in the controls (30.2% vs.13.0%, 90.6% vs. 78.0%, respectively, both *p* < 0.05). Haplotype analysis showed the ACG haplotype of UGT1A1 were associated with an increased hyperbilirubinemia risk (OR 3.122, *p* = 0.001), whereas the GCG haplotype was related to a reduced risk (OR 0.523, *p* = 0.018).

### Conclusion

The frequencies of the A allele in rs4148323 and the C allele in rs2417940 are highly associated with the incidence of severe hyperbilirubinemia in Chinese Han neonates.

### Trial registration

Trial registration number:ChiCTR1800020424; Date of registration:2018-12-29.

## Introduction

Neonatal hyperbilirubinemia(NH) and resultant jaundice are commonly seen in 50–60% of newborns and, to a greater extent, in premature infants [[1](#CR1), [2](#CR2)]. Although most cases are benign and do not result in severe consequences, some infants will develop hazardous levels of bilirubin that directly threaten brain damage and may result in neuro-developmental abnormalities, such as hearing loss, athetosis, and intellectual deficit [[2](#CR2)]. In pathological hyperbilirubinemia, increased production of bilirubin, deficiency in hepatic uptake, impaired conjugation of bilirubin, and/or increased enterohepatic circulation of bilirubin are observed [[3](#CR3)]. Severe hyperbilirubinemia can be caused by maternofetal blood group isoimmunization (especially ABO hemolysis), G-6-PD deficiency, and severe infection [[4](#CR4)]. Laboratory testing can predict this part of infants in advance and help guide active intervention to prevent exchange transfusion and reduce the risk of bilirubin encephalopathy. However, routine laboratory tests do not identify the etiology of severe hyperbilirubinemia in over 50% of infants [[5](#CR5), [6](#CR6)].

The American Academy of Pediatrics recommendations in 2004 identified East Asian ancestry, particularly mainland China, as a substantial risk factor for severe hyperbilirubinemia [[7](#CR7)]. A positive family history can be a marker for shared genetic susceptibility; several studies [[8](#CR8), [9](#CR9)] have identified a previous sibling with a history of neonatal jaundice as an essential risk factor for neonatal hyperbilirubinemia. If an earlier sibling had a serum bilirubin > 15 mg/dL (257 µmol/L), the risk in subsequent siblings increased to 12.5-fold greater than controls [[10](#CR10)].In addition, Ebbesen’s [[11](#CR11)] research on bilirubin levels in identical and fraternal twins showed that after controlling for the factors known to regulate neonatal hyperbilirubinemia, genetics still has an important influence on neonatal bilirubin levels. Thus, genetic factors play an important role in the development of neonatal hyperbilirubinemia, especially genetic polymorphism of key enzymes involved in bilirubin metabolism. A growing body of evidence suggested that genetic variants in uridine diphosphate glucuronosyltransferase 1A1 (UGT1A1), solute carrier organic anion transporter family members 1B1 and 1B3 (SLCO1B1, SLCO1B3), glucose-6-phosphate dehydrogenase deficiency (G-6-PD), heme oxygenase 1 (HMOX1), and biliverdin reductase A (BLVRA) are closely associated with the incidence of severe hyperbilirubinemia.

However, the combined action of multiple genes on the occurrence of severe neonatal hyperbilirubinemia remains unclear. This study aimed to investigate the association between single nucleotide polymorphisms (SNPs) in genetic variants in bilirubin metabolism and the risk of severe hyperbilirubinemia.

## Materials and methods

### Patient characteristics

Between January 1, 2019, and June 30, 2020, 144 neonates admitted to three hospitals in southwest China with severe hyperbilirubinemia and meeting the following inclusion criteria were enrolled as the case group: (1) gestational age(GA) ≥ 35 weeks, within seven days of birth; (2) indirect bilirubin was the predominate component of serum bilirubin, accounting for more than 80% of the total bilirubin; (3) total bilirubin level met the exchange transfusion criteria in compliance with the guideline of management of hyperbilirubinemia endorsed by American Academy of Pediatrics in 2004 [[7](#CR7)]. Meanwhile, a control group of 50 neonates was recruited, with GA ≥ 35 weeks and an age exceeding 7 days, who was in the absence of hyperbilirubinemia after brith or no phototherapy history. Additionally, their transcutaneous bilirubin levels monitored at community-based health facility after birth and blood bilirubin levels at admission were below the criteria requiring phototherapy [[7](#CR7)]. Those with definite infection, multiple organ malformations, evident ABO or Rh incompatibility, and G-6-PD deficiency were excluded from this study. The experimental protocol was established, according to the ethical guidelines of the Helsinki Declaration and was approved by the Human Ethics Committee of the Children ‘s Hospital of Chongqing Medical University. Written informed consent was obtained from the parents or guardians.

### SNP selection

UGT1A1, SLCO1B1, SLCO1B3, HMOX1, and BLVRA genes were chosen for their crucial involvement in bilirubin metabolism. TagSNPs selected for the validation were based on the following criteria: (1) SNPs within 2 kb upstream and downstream of the gene area; (2) when multiple associated SNPs were in strong linkage disequilibrium (LD, r2 > 0.8), SNPs previously reported in the literature were prioritized. Twelve tagSNPs were chosen based on these parameters, including six from UGT1A1, two from SLCO1B1, two from SLCO1B3, one from BLVRA, and one from HMOX1(Table [1](#Tab1)).

### Table 1.

| Gene (chromosome) | Reference SNP ID | Chromosome position | Region | MAF1000G_CHB | MAF1000G_EAS | Allele |
| --- | --- | --- | --- | --- | --- | --- |
| UGT1A1(2) | rs4148323 | 234,669,144 | exonic | 0.192 | 0.138 | A |
|   | rs3771341 | 234,673,239 | intronic | 0.113 | 0.127 | A |
|   | rs34946978 | 234,676,872 | exonic | 0.024 | 0.011 | T |
|   | rs114982090 | 234,680,955 | exonic | 0.005 | 0.007 | T |
|   | rs35350960 | 234,669,619 | exonic | 0.007 | 0.014 | A |
|   | rs34993780 | 234,681,059 | exonic | 0.002 | 0.003 | G |
| SLCO1B1(12) | rs4149056 | 21,331,549 | exonic | 0.127 | 0.123 | C |
|   | rs1564370 | 21,335,190 | intronic | 0.221 | 0.244 | G |
| SLCO1B3(12) | rs2417940 | 21,017,875 | intronic | 0.173 | 0.195 | T |
|   | rs2117032 | 21,074,122 | 3’-flanking | 0.421 | 0.461 | C |
| BLVRA(7) | rs699512 | 43,810,764 | exonic | 0.310 | 0.317 | G |
| HMOX1(22) | rs2071747 | 35,777,185 | exonic | 0.055 | 0.054 | C |

Table 1 Caption: Genotyping data for 12 tagSNPs genotyped from the candidate genes UGT1A1, SLCO1B1, SLCO1B3, BLVRA, and HMOX1

### DNA isolation

Venous blood samples (2 to 3 ml) were collected and transferred to EDTA-anticoagulated tubes(Suzhou Bidi Medical Devices Co., Ltd.; Suzhou, China). The genomic DNA was isolated from EDTA-anticoagulated blood samples using the QIAamp DNA Kit (Qiagen, Hilden, Germany) and stored at − 80℃ for the subsequent experiments.

### SNP genotyping [12]

The genotyping of 12 SNP loci in one ligation reaction was done in this work using an enhanced multiplex ligation detection reaction method. The 12 SNP sites weres amplified using a multiplex of PCR procedures. The PCR reaction in 10 µl contained 1x Takara GC-I buffer, 3.0 mM MgCl_2_, 0.3 mM dNTP mix, 1 U HotStar Taq polymerase(Qiagen Inc.), 1 µl of primer mixture and 1 µl of genomic DNA.

The PCR program for all reactions was 95℃ for 2 min; 11 cycles x (94℃ 20 s, 65℃-0.5℃/cycle 40 s, 72℃ 1 min 30 s); 24 cycles x (94℃ 20 s, 59℃ 30 s, 72℃1 min 30 s); and 72℃ for 2 min; holding at 4℃. Exonuclease I and shrimp alkaline phosphatase, which were digested at 37 °C for 1 h and 75 °C for 15 min, were used to purify 10 µl of PCR products. The labeling oligo mixture, probe mixture, 2 µl of ligation buffer, 80 U of Taq DNA Ligase (NEB), and 5 µl of the purified PCR product mixture are all included in the ligation reaction, which is contained in 10 µl.

The ligation cycling program was 95℃ for 2 min; 38 cycles x (94℃ 1 min, 56℃ 4 min); holding at 4℃. A 0.5 µl of ligation product was loaded in an ABI3730XL, and the raw data were analyzed by GeneMapper 4.1.All of the primers, probes and labeling oligos were designed by and ordered from Genesky Biotechnologies Inc (Shanghai, China).

### Statistical analysis

Statistical analysis was performed by SPSS (version 19.0; SPSS, Inc., Chicago, IL, USA). Quantitative data were expressed as the mean ± SD and qualitative data as a percentage. Allelic frequencies were calculated by the gene-counting method. An unpaired Student’s t-test was utilized to compare the two groups. The Hardy-Weinberg test was used to evaluate the hereditary equilibrium. Linkage disequilibrium (LD) analyses were conducted using HaploView version 4.2 (Broad Institute, Cambridge, MA, United States) in Han Chinese in Beijing (CHB) populations from the 1000 Genomes Project phase 3 genotype data [[13](#CR13)]. The chi-square test or Fisher’s precision probability test was used to assess the frequencies of alleles and genotypes in the two groups under three distinct genetic models: dominant, recessive, and additive. We used multivariable logistic regression adjusting for GA and age. The genotype relative risk was calculated using odds ratios (ORs) and 95% confidence intervals (CIs). Logistic regression analysis was used to calculate the significance of differences in genotype and allele frequencies and investigate the association of tested SNPs with hyperbilirubinemia risk. *P* < 0.05 was considered to be statistically significant.

## Results

### Clinical and demographic characteristics

A total of 194 newborns, including 144 neonates in the case group and 50 neonates in the control group, had blood samples obtained for DNA extraction and genotyping. All enrolled neonates were from the Chinese Han population,and did not receive any medications that may affect bilirubin metabolism during the perinatal period, such as phenobarbital. The demographic characteristics of the two groups are shown in Table [2](#Tab2). GA, birth weight, sex, and feeding patterns did not significantly differ between the groups. A significant difference in TSB and days of blood sampling was noted between the two groups because of the study grouping design.

### Table 2.

|   |  | Case Group(N = 144) | Control Group(N = 50) | P value |
| --- | --- | --- | --- | --- |
| Gestational age (weeks) |  | 38.4 ± 1.2 | 38.6 ± 1.4 | 0.519 |
| Birth weight (g) |  | 3117 ± 465 | 3204 ± 433 | 0.245 |
| Gender (male/female) |  | 69/75 | 28/22 | 0.325 |
| feeding patterns | breast feeding | 47 | 16 | 0.464 |
| artificial feeding | 28 | 13 |  |  |
| mixed feeding | 69 | 21 |  |  |
| Days of blood sampling (days) |  | 3.7 ± 1.8 | 16.5 ± 9.1 | 0.000 |
| TSB(mg/dl) |  | 20.8 ± 4.0 | 5.4 ± 3.7 | 0.000 |

Table 2 Caption: Clinical and demographic characteristics of neonates in Case and Control Groups

### Hardy-Weinberg equilibrium test

Tests were performed for twelve loci between the groups, and the *P* values were > 0.05, indicating all loci were in Hardy-Weinberg equilibrium (Table [3](#Tab3)).

### Table 3.

| Gene | Reference SNP ID | Case GroupHWE-p | Control GroupHWE-p |
| --- | --- | --- | --- |
| UGT1A1 | rs4148323 | 0.766 | 0.818 |
|   | rs3771341 | 0.276 | 0.651 |
|   | rs34946978 | 1.000 | 1.000 |
|   | rs114982090 | 1.000 | 1.000 |
|   | rs35350960 | 1.000 | 1.000 |
|   | rs34993780 | 1.000 | 1.000 |
| SLCO1B1 | rs4149056 | 1.000 | 1.000 |
|   | rs1564370 | 1.000 | 0.452 |
| SLCO1B3 | rs2417940 | 1.000 | 0.506 |
|   | rs2117032 | 0.532 | 0.263 |
| BLVRA | rs699512 | 0.318 | 0.617 |
| HMOX1 | rs2071747 | 1.000 | 1.000 |

Table 3 Caption: Hardy-Weinberg equilibrium in Case and Control Groups

### Distribution of genotype, allele, and genetic susceptibility to hyperbilirubinemia of gene polymorphisms

Table [4](#Tab4) displays the allelic and genotypic frequencies of UGT1A1, SLCO1B1, SLCO1B3, BLVRA, and HMOX1 variants between the case and control groups. Significant differences in genotype and allele distributions in UGT1A1-rs4148323 and SLCO1B3-rs2417940 were noted between the two groups. Compared to the control group, the case group had higher frequencies of GA heterozygotes and AA mutant homozygotes at the rs4148323 locus (GA, 38.2% vs. 26.0%, *P* = 0.036 and AA, 11.1% vs. 0.0%, *P* = 0.003, respectively). Furthermore, the frequency of CC homozygotes was higher in the case group than the control group at the rs2417940 locus(81.3% vs. 58.0%, *P* = 0.002). Allele frequencies analysis revealed that the rs4148323 A allele in UGT1A1 gene and the allele C of SLCO1B3-rs2417940 were associated with hyperbilirubinemia (OR 2.897, *p* = 0.001 and OR 2.726,*P* = 0.001, respectively). However, there were no statistical differences in the genotype and allele distributions in UGT1A1-rs3771341, -rs34946978, -rs114982090, -rs114982090, SLCO1B1-rs4149056, -rs1564370, SLCO1B3-rs2117032, BLVRA-rs699512 and HMOX1-rs2071747 between the two groups.

### Table 4.

| Genotypes/Alleles | Case group (n = 144)n (%) | Control group (n = 50)n (%) | P value | OR (95%CI) |
| --- | --- | --- | --- | --- |
| UGT1A1-rs4148323 |   |   |   |   |
| GG | 73 (50.7) | 37 (74.0) |   | 1 |
| GA | 55 (38.2) | 13 (26.0) | 0.036 | 2.144 (1.041–4.416) |
| AA | 16 (11.1) | 0 (0) | 0.003* | 0.820 (0.744–0.904) |
| A allele | 87 (30.2) | 13 (13.0) |   |   |
| G allele | 201 (69.8) | 87 (87.0) | 0.001 | 2.897 (1.535–5.465) |
| UGT1A1-rs3771341 |   |   |   |   |
| GG | 134 (93.1) | 42 (84.0) |   | 1 |
| GA | 9 (6.2) | 7 (14.0) | 0.129* | 0.403 (0.141–1.148) |
| AA | 1 (0.7) | 1 (2.0) | 0.426* | 0.313 (0.019–5.120) |
| A allele | 11 (3.8) | 9 (9.0) |   |   |
| G allele | 277 (96.2) | 91 (91.0) | 0.063 | 0.402 (0.161-1.000) |
| UGT1A1-rs34946978 |   |   |   |   |
| CC | 136 (94.4) | 48 (96.0) |   |   |
| CT | 8 (5.6) | 2 (4.0) | 1.000* | 1.412 (0.290–6.882) |
| C allele | 280 (97.2) | 98 (98.0) |   |   |
| T allele | 8 (2.8) | 2 (2.0) | 1.000* | 0.714 (0.149–3.421) |
| UGT1A1-rs114982090 |   |   |   |   |
| CC | 138 (95.8) | 50 (100.0) |   |   |
| CT | 6 (4.2) | 0 (0) | 0.342* | 0.958 (0.926–0.992) |
| C allele | 282 (97.9) | 100 (100.0) |   |   |
| T allele | 6 (2.1) | 0 (0) | 0.345* | 0.979 (0.963–0.996) |
| SLCO1B1-rs4149056 |   |   |   |   |
| TT | 118 (81.9) | 41 (82.0) |   | 1 |
| CT | 25 (17.4) | 9 (18.0) | 0.934 | 0.965 (0.416–2.237) |
| CC | 1 (0.7) | 0 (0) | 1.000* | 0.992 (0.975–1.008) |
| C allele | 27 (9.4) | 9 (9.0) |   |   |
| T allele | 261 (90.6) | 91 (91.0) | 0.911 | 1.046 (0.474–2.308) |
| SLCO1B1-rs1564370 |   |   |   |   |
| CC | 89 (61.8) | 34 (68.0) |   | 1 |
| CG | 48 (33.3) | 13 (26.0) | 0.354 | 0.709 (0.342–1.470) |
| GG | 7 (4.9) | 3 (6.0) | 1.000* | 1.122 (0.274–4.591) |
| C allele | 226 (78.5) | 81 (81.0) |   |   |
| G allele | 62 (21.5) | 19 (19.0) | 0.669 | 0.855 (0.482–1.517) |
| SLCO1B3- rs2417940 |   |   |   |   |
| CC | 117 (81.3) | 29 (58.0) |   | 1 |
| CT | 27 (18.7) | 20 (40.0) | 0.002 | 2.989 (1.474–6.060) |
| TT | 0 (0) | 1 (2.0) | 0.204* | 1.034 (0.968–1.106) |
| C allele | 261 (90.6) | 78 (78.0) |   |   |
| T allele | 27 (9.4) | 22 (22.0) | 0.001 | 2.726 (1.471–5.054) |
| SLCO1B3-rs2117032 |   |   |   |   |
| CC | 23 (16.0) | 8 (16.0) |   | 1 |
| CT | 66 (45.8) | 30 (60.0) | 0.565 | 1.307 (0.524–3.256) |
| TT | 55 (38.2) | 12 (24.0) | 0.367 | 0.627 (0.227–1.737) |
| C allele | 112 (38.9) | 46 (46.0) |   |   |
| T allele | 176 (61.1) | 54 (54.0) | 0.212 | 0.747 (0.472–1.182) |
| BLVRA- rs699512 |   |   |   |   |
| AA | 73 (50.7) | 27 (54.0) |   | 1 |
| GA | 60 (41.7) | 18 (36.0) | 0.550 | 0.811 (0.408–1.613) |
| GG | 11 (7.6) | 5 (10.0) | 0.766* | 1.229 (0.391–3.864) |
| A allele | 206 (71.5) | 72 (72.0) |   |   |
| G allele | 82 (28.5) | 28 (28.0) | 0.928 | 0.977 (0.589–1.620) |
| HMOX1- rs2071747 |   |   |   |   |
| GC | 13 (9.0) | 2 (4.0) |   |   |
| GG | 131 (91.0) | 48 (96.0) | 0.362* | 2.382 (0.518–10.944) |
| C allele | 13 (4.5) | 2 (2.0) |   |   |
| G allele | 275 (95.5) | 98 (98.0) | 0.372* | 2.316 (0.514–10.448) |

Table 4 Caption: Genotypes and allele frequencies of SNPs in the UGT1A1, SLCO1B1, SLCO1B3, BLVRA, and HMOX1 genes

### Linkage disequilibrium correlations of UGT1A1, SLCO1B1 and SLCO1B3 haplotypes with hyperbilirubinemia susceptibility

The linkage disequilibrium distributions of six UGT1A1 loci, two SLCO1B1 loci, and two SLCO1B3 loci are depicted in Fig. [1](#Fig1). Haplotypes were allocated based on linkage disequilibrium distribution, and relative distribution frequencies of haplotypes with in case and control groups were investigated. Two haplotypes of UGT1A1 were associated with hyperbilirubinemia susceptibility, of which ACG (OR 3.122, *p* = 0.001) was discovered to increase the risk of hyperbilirubinemia while GCG (OR 0.523, *p* = 0.018) reduced the risk. None of the haplotypes of SLCO1B1 and SLCO1B3 was found to be associated with hyperbilirubinemia (Table [5](#Tab5)).

### Fig. 1.

![Fig. 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1788/10811893/a4276b2291ea/12887_2024_4537_Fig1_HTML.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=10811893_12887_2024_4537_Fig1_HTML.jpg)

Linkage disequilibrium structures of SNPs of the candidate genes UGT1A1, SLCO1B1, and SLCO1B3

### Table 5.

| Gene | SNPs constituted | Haplotypes | Frequencies |  | P value | OR (95%CI) |
| --- | --- | --- | --- | --- | --- | --- |
| Case group | Control group |  |  |  |  |  |
| UGT1A1 | rs4148323,rs35350960,rs3771341 | GCA | 0.034 | 0.080 | 0.114 | 0.414 (0.159–1.079) |
| GCG | 0.663 | 0.790 | 0.018 | 0.523 (0.305–0.898) |  |  |
| ACA | 0.004 | 0.010 | 0.450 | 0.345 (0.021–5.567) |  |  |
| ACG | 0.299 | 0.120 | 0.001 | 3.122 (1.624–6.003) |  |  |
| rs34946978,rs114982090,rs34993780 | CCT | 0.951 | 0.980 | 0.343 | 0.399 (0.089–1.789) |  |
| TCT | 0.028 | 0.020 | 0.955 | 1.400 (0.292–6.706) |  |  |
| CTT | 0.021 | / | 0.345 | / |  |  |
| SLCO1B1 | rs4149056,rs1564370 | CC | 0.094 | 0.090 | 0.911 | 1.046 (0.474–1.373) |
| TC | 0.691 | 0.730 | 0.463 | 0.827 (0.498–1.373) |  |  |
| TG | 0.215 | 0.180 | 0.452 | 1.250 (0.698–2.238) |  |  |

Table 5 Caption: Frequencies of haplotypes assigned by SNPs of UGT1A1, SLCO1B1, and SLCO1B3 genes in Case and Control Groups

### Unconditional logistic analysis of SNPs and their susceptibility to hyperbilirubinemia

In the dominant model, UGT1A1-rs4148323 AA + GA genotypes were associated with an increased risk of hyperbilirubinemia (OR 2.768,*p* = 0.004). In the dominant model, the SLCO1B3-rs2417940 CC genotype was related to a higher risk of hyperbilirubinemia (OR 3.138, *p* = 0.001). There were statistical differences in the frequencies of the UGT1A1-rs4148323 AA genotype between the two groups by Fisher’s precision probability test. In the additive model, alleles of the SLCO1B3-rs2417940 minor C (OR 2.889, *p* = 0.003) were associated with an increased risk of hyperbilirubinemia. The frequencies of the remaining 10 SNPs did not differ between the two groups(Table [6](#Tab6)).

### Table 6.

| Gene | tagSNPs | Allele | Dominant model |  | Recessive model |  | Additive model |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| P value | OR (95%CI) | P value | OR (95%CI) | P value |  | OR (95%CI) |  |  |  |
| UGT1A1 | rs4148323 | A | 0.004 | 2.768 (1.359–5.638) | 0.013* | / | 0.119 | 1.759 (0.860–3.598) |  |
| rs3771341 | A | 0.086 | 0.392 (0.145–1.057) | 0.450* | 0.343 (0.021–5.583) | 0.131* | 0.410 (0.144–1.165) |  |  |
| rs34946978 | T | 1.000* | 1.412 (0.290–6.882) | / | / | 1.000* | 1.412 (0.290–6.882) |  |  |
| rs114982090 | T | 0.342* | 0.958 (0.926–0.992) | / | / | 0.342* | 0.958 (0.926–0.992) |  |  |
| SLCO1B1 | rs4149056 | C | 0.993 | 1.004 (0.435–2.318) | 1.000* | 0.993 (0.980–1.007) | 0.918 | 0.957 (0.413–2.218) |  |
|   | rs1564370 | G | 0.433 | 1.313 (0.663–2.599) | 0.720* | 0.800 (0.199–3.222) | 0.336 | 1.423 (0.692–2.926) |  |
| SLCO1B3 | rs2417940 | T | 0.001 | 3.138 (1.558–6.321) | 0.258* | 1.020 (0.981–1.062) | 0.003 | 2.889 (1.429–5.839) |  |
|   | rs2117032 | C | 0.069 | 1.957 (0.942–4.064) | 0.996 | 1.002 (0.417–2.410) | 0.084 | 0.564 (0.293–1.085) |  |
| BLVRA | rs699512 | G | 0.687 | 0.876 (0.460–1.669) | 0.563* | 1.343 (0.443–4.076) | 0.481 | 1.270 (0.653–2.471) |  |
| HMOX1 | rs2071747 | C | 0.362* | 2.382 (0.518–10.944) | / | / | 0.362* | 2.382 (0.518–10.94) |  |

Table 6 Caption: Comparison of allele and genotype distributions and frequencies in three genetic models between case and control groups

## Discussion

Hyperbilirubinemia, presenting as jaundice, is a ubiquitous and frequently benign condition in newborn babies, albeit a leading cause of hospitalization in the first week of life [[14](#CR14)]. Although the exact etiology of hyperbilirubinemia remains unclear [[15](#CR15)], the role of genetic factors in the pathogenesis of hyperbilirubinemia has received significant attention from neonatologists [[16](#CR16)]. The current study aims to determine the roles of several genetic bilirubin metabolism modifiers in developing severe hyperbilirubinemia in Chinese Han newborns. Our research indicated striking relationships between the vulnerability to severe hyperbilirubinemia and the gene polymorphisms for UGT1A1 and SLCO1B3.

UGT1A1 is a member of the UGT1 family of microsomal membranes and plays an essential role in converting the toxic form of bilirubin to its nontoxic form [[17](#CR17)]. The present study demonstrated that rs4148323 in the UGT1A1 gene is independently associated with total bilirubin levels. The frequency of the A allele in rs4148323 was associated with the incidence of severe hyperbilirubinemia. A prior study has reported that the A allele in rs4148323 is common in the East Asian population, with allele frequencies of 19.2% in Han Chinese and 19% in Korean populations [[18](#CR18)], and 16.2% in the Japanese population as well [[19](#CR19)], whereas it is monomorphic in the European populations ([http://www.ncbi.nlm.nih.gov/snp](http://www.ncbi.nlm.nih.gov/snp)), indicating that rs4148323 might be specifically associated with serum bilirubin levels in Asians. In the control group, the A allele frequency was 13%, similar to another study in China [[20](#CR20)], and is much higher than in the severe hyperbilirubinemia group which had a frequency of 30.2%, similar to that observed in Japanese studies [[21](#CR21)].

SLCO1B3, located at 12p12, is highly expressed in the basolateral hepatocyte membrane. It is an organic anion transporter gene coding an organic anion transporter polypeptide (OATP/1B3) that mediates the extrusion of bilirubin [[22](#CR22)]. Genetic variants of SLCO1B3 contributed to idiopathic mild unconjugated hyperbilirubinemia [[23](#CR23)]. The variant of rs2417940, located in intron 7 of SLCO1B3, was significantly associated with total serum bilirubin levels [[18](#CR18)]. Our study demonstrated that the genotype CC and the frequency of allele C in the SLCO1B3-rs2417940 significantly correlated to the incidence of hyperbilirubinemia (81.3% vs. 58.0%, *P* = 0.002, OR 2.989 and 90.6% vs. 78.0%, *P* = 0.001, OR 2.726, respectively). Dai et al. [[24](#CR24)] reported that smoking and rs2417940 polymorphism in SLCO1B3 on total bilirubin levels had a significant interaction, and rs2417940 had a much stronger effect on serum bilirubin levels in nonsmokers than in smokers. The frequency of allele T in the control group with mild or no hyperbilirubinemia was 22.0%, similar to another report in China (18.3%) [[25](#CR25)].

We acknowledge that this study had several limitations. In this article, only twelve loci of genes that may affect bilirubin metabolism were assessed, including UGT1A1, SLCO1B1, SLCO1B3, HMOX1, and BLVRA. It is uncertain whether the polymorphism of other SNPs in the above genes or other genes would affect the serum bilirubin level. However, to our best knowledge, the current analysis was cost-effective, containing all relevant variations in these genes described in the Asian population. Another obvious limitation is that we could not examine the effect of medications on the metabolism of bilirubin levels or the activity of the target genes. However, medications are rarely prescribed for most neonates with hyperbilirubinemia in clinical settings.

## Conclusion

This study demonstrated the influence of genetic polymorphisms of several hyperbilirubinemia-related genes, illuminating the complicated nature of this condition. However, severe neonatal hyperbilirubinemia is a multifactorial issue. Future studies focusing on the interactions of the bilirubin metabolism gene, other genes, and nongenetic factors will provide a more holistic insight into the pathogenesis of neonatal hyperbilirubinemia.

## Acknowledgements

We would like to thank the laboratory staff Genesky Biotechnologies Inc and NICU nursing staff of Children’s Hospital of Chongqing Medical University, Chongqing Jiulongpo People’s Hospital and Chongqing Bishan Maternal and Child Health Care Hospital for their contribution to this study.

## Abbreviations

## Author contributions

This work was carried out in collaboration between all authors.Jian-Hui Wang and Wen-Hui Dong designed the study. Juan Fan, Hua-Yun He, Huan-Huan Li,Huan-Huan Li,and Lei Tang acquired the data. Wen-Hui Dong and Bo-Yin Deng, analyzed the data. Wen-Hui Dong, Jian-Hui Wang, and Bo-Yin Deng wrote the manuscript. Juan Fan and Jian-Hui Wang gave technical support and conceptual advice. All authors read and approved the final manuscript.

## Funding

This study is funded by Chongqing medical scientific research project (Joint project of Chongqing Health Commission and Science and Technology Bureau) (2022MSXM137). The funder had no role in the design of the study and collection, analysis, and interpretation of data and in writing the manuscript.

## Data availability

Full data set and other materials on this study can be obtained from the corresponding author on reasonable request.

## Declarations

### Ethics approval and consent to participate

All authors hereby declare that all experiments have been examined and approved by the Ethics Committee of the Children ‘s Hospital of Chongqing Medical University in 2018 (No. 2018108) and have therefore been performed in accordance with the ethical standards laid down in the 1964 Declaration of Helsinki. A written informed consent was obtained from the parents before inclusion into the study.

### Consent for publication

Not applicable.

### Competing interests

The authors declare that they have no conflict of interest.

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Data Availability Statement

Full data set and other materials on this study can be obtained from the corresponding author on reasonable request.

### Data Availability Statement

Full data set and other materials on this study can be obtained from the corresponding author on reasonable request.

## References

1. Campistol J, Galvez H, Cazorla AG, Malaga I, Iriondo M, Cusi V. [Neurological dysfunction induced by bilirrubin] Neurologia. 2012;27(4):202–11. doi: 10.1016/j.nrl.2010.03.013.  [DOI](https://doi.org/10.1016/j.nrl.2010.03.013) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21163242/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neurologia&title=%5BNeurological%20dysfunction%20induced%20by%20bilirrubin%5D&author=J%20Campistol&author=H%20Galvez&author=AG%20Cazorla&author=I%20Malaga&author=M%20Iriondo&volume=27&issue=4&publication_year=2012&pages=202-11&pmid=21163242&doi=10.1016/j.nrl.2010.03.013&)

2. Mitra S, Rennie J. Neonatal jaundice: aetiology, diagnosis and treatment. Br J Hosp Med (Lond) 2017;78(12):699–704. doi: 10.12968/hmed.2017.78.12.699.  [DOI](https://doi.org/10.12968/hmed.2017.78.12.699) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29240507/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Hosp%20Med%20(Lond)&title=Neonatal%20jaundice:%20aetiology,%20diagnosis%20and%20treatment&author=S%20Mitra&author=J%20Rennie&volume=78&issue=12&publication_year=2017&pages=699-704&pmid=29240507&doi=10.12968/hmed.2017.78.12.699&)

3. Ulgenalp A, Duman N, Schaefer FV, Whetsell L, Bora E, Gulcan H, Kumral A, Oren H, Giray O, Ercal D, et al. Analyses of polymorphism for UGT1*1 exon 1 promoter in neonates with pathologic and prolonged jaundice. Biol Neonate. 2003;83(4):258–62. doi: 10.1159/000069487.  [DOI](https://doi.org/10.1159/000069487) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12743455/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biol%20Neonate&title=Analyses%20of%20polymorphism%20for%20UGT1*1%20exon%201%20promoter%20in%20neonates%20with%20pathologic%20and%20prolonged%20jaundice&author=A%20Ulgenalp&author=N%20Duman&author=FV%20Schaefer&author=L%20Whetsell&author=E%20Bora&volume=83&issue=4&publication_year=2003&pages=258-62&pmid=12743455&doi=10.1159/000069487&)

4. Yu C, Li H, Zhang Q, He H, Chen X, Hua Z. Report about term infants with severe hyperbilirubinemia undergoing exchange transfusion in Southwestern China during an 11-year period, from 2001 to 2011. PLoS ONE. 2017;12(6):e0179550. doi: 10.1371/journal.pone.0179550.  [DOI](https://doi.org/10.1371/journal.pone.0179550) | [PMC free article](/articles/PMC5491324/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28662083/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&title=Report%20about%20term%20infants%20with%20severe%20hyperbilirubinemia%20undergoing%20exchange%20transfusion%20in%20Southwestern%20China%20during%20an%2011-year%20period,%20from%202001%20to%202011&author=C%20Yu&author=H%20Li&author=Q%20Zhang&author=H%20He&author=X%20Chen&volume=12&issue=6&publication_year=2017&pages=e0179550&pmid=28662083&doi=10.1371/journal.pone.0179550&)

5. Kuzniewicz MW, Wickremasinghe AC, Wu YW, McCulloch CE, Walsh EM, Wi S, Newman TB. Incidence, etiology, and outcomes of hazardous hyperbilirubinemia in newborns. Pediatrics. 2014;134(3):504–9. doi: 10.1542/peds.2014-0987.  [DOI](https://doi.org/10.1542/peds.2014-0987) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25092943/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pediatrics&title=Incidence,%20etiology,%20and%20outcomes%20of%20hazardous%20hyperbilirubinemia%20in%20newborns&author=MW%20Kuzniewicz&author=AC%20Wickremasinghe&author=YW%20Wu&author=CE%20McCulloch&author=EM%20Walsh&volume=134&issue=3&publication_year=2014&pages=504-9&pmid=25092943&doi=10.1542/peds.2014-0987&)

6. Christensen RD, Lambert DK, Henry E, Eggert LD, Yaish HM, Reading NS, Prchal JT. Unexplained extreme hyperbilirubinemia among neonates in a multihospital healthcare system. Blood Cells Mol Dis. 2013;50(2):105–9. doi: 10.1016/j.bcmd.2012.10.004.  [DOI](https://doi.org/10.1016/j.bcmd.2012.10.004) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23123126/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood%20Cells%20Mol%20Dis&title=Unexplained%20extreme%20hyperbilirubinemia%20among%20neonates%20in%20a%20multihospital%20healthcare%20system&author=RD%20Christensen&author=DK%20Lambert&author=E%20Henry&author=LD%20Eggert&author=HM%20Yaish&volume=50&issue=2&publication_year=2013&pages=105-9&pmid=23123126&doi=10.1016/j.bcmd.2012.10.004&)

7. American Academy of Pediatrics Subcommittee on H Management of hyperbilirubinemia in the newborn infant 35 or more weeks of gestation. Pediatrics. 2004;114(1):297–316. doi: 10.1542/peds.114.1.297.  [DOI](https://doi.org/10.1542/peds.114.1.297) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15231951/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pediatrics&title=Management%20of%20hyperbilirubinemia%20in%20the%20newborn%20infant%2035%20or%20more%20weeks%20of%20gestation&volume=114&issue=1&publication_year=2004&pages=297-316&pmid=15231951&doi=10.1542/peds.114.1.297&)

8. Gale R, Seidman DS, Dollberg S, Stevenson DK. Epidemiology of neonatal jaundice in the Jerusalem population. J Pediatr Gastroenterol Nutr. 1990;10(1):82–6. doi: 10.1097/00005176-199001000-00016.  [DOI](https://doi.org/10.1097/00005176-199001000-00016) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2324884/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pediatr%20Gastroenterol%20Nutr&title=Epidemiology%20of%20neonatal%20jaundice%20in%20the%20Jerusalem%20population&author=R%20Gale&author=DS%20Seidman&author=S%20Dollberg&author=DK%20Stevenson&volume=10&issue=1&publication_year=1990&pages=82-6&pmid=2324884&doi=10.1097/00005176-199001000-00016&)

9. Newman TB, Xiong B, Gonzales VM, Escobar GJ. Prediction and prevention of extreme neonatal hyperbilirubinemia in a mature health maintenance organization. Arch Pediatr Adolesc Med. 2000;154(11):1140–7. doi: 10.1001/archpedi.154.11.1140.  [DOI](https://doi.org/10.1001/archpedi.154.11.1140) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11074857/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arch%20Pediatr%20Adolesc%20Med&title=Prediction%20and%20prevention%20of%20extreme%20neonatal%20hyperbilirubinemia%20in%20a%20mature%20health%20maintenance%20organization&author=TB%20Newman&author=B%20Xiong&author=VM%20Gonzales&author=GJ%20Escobar&volume=154&issue=11&publication_year=2000&pages=1140-7&pmid=11074857&doi=10.1001/archpedi.154.11.1140&)

10. Watchko JF, Lin Z. Exploring the genetic architecture of neonatal hyperbilirubinemia. Semin Fetal Neonatal Med. 2010;15(3):169–75. doi: 10.1016/j.siny.2009.11.003.  [DOI](https://doi.org/10.1016/j.siny.2009.11.003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20022574/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Semin%20Fetal%20Neonatal%20Med&title=Exploring%20the%20genetic%20architecture%20of%20neonatal%20hyperbilirubinemia&author=JF%20Watchko&author=Z%20Lin&volume=15&issue=3&publication_year=2010&pages=169-75&pmid=20022574&doi=10.1016/j.siny.2009.11.003&)

11. Ebbesen F, Mortensen BB. Difference in plasma bilirubin concentration between monozygotic and dizygotic newborn twins. Acta Paediatr. 2003;92(5):569–73. doi: 10.1111/j.1651-2227.2003.tb02508.x.  [DOI](https://doi.org/10.1111/j.1651-2227.2003.tb02508.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12839286/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Acta%20Paediatr&title=Difference%20in%20plasma%20bilirubin%20concentration%20between%20monozygotic%20and%20dizygotic%20newborn%20twins&author=F%20Ebbesen&author=BB%20Mortensen&volume=92&issue=5&publication_year=2003&pages=569-73&pmid=12839286&doi=10.1111/j.1651-2227.2003.tb02508.x&)

12. Shen W, Kuang P, Wang B, Zeng Q, Chen C, Lin X. Genetic polymorphisms of LPCAT1, CHPT1 and PCYT1B and risk of neonatal respiratory distress syndrome among a Chinese Han Population. Pediatr Neonatol. 2020;61(3):318–24. doi: 10.1016/j.pedneo.2019.12.012.  [DOI](https://doi.org/10.1016/j.pedneo.2019.12.012) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31964590/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pediatr%20Neonatol&title=Genetic%20polymorphisms%20of%20LPCAT1,%20CHPT1%20and%20PCYT1B%20and%20risk%20of%20neonatal%20respiratory%20distress%20syndrome%20among%20a%20Chinese%20Han%20Population&author=W%20Shen&author=P%20Kuang&author=B%20Wang&author=Q%20Zeng&author=C%20Chen&volume=61&issue=3&publication_year=2020&pages=318-24&pmid=31964590&doi=10.1016/j.pedneo.2019.12.012&)

13. Genomes Project C, Auton A, Brooks LD, Durbin RM, Garrison EP, Kang HM, Korbel JO, Marchini JL, McCarthy S, McVean GA, et al. A global reference for human genetic variation. Nature. 2015;526(7571):68–74. doi: 10.1038/nature15393.  [DOI](https://doi.org/10.1038/nature15393) | [PMC free article](/articles/PMC4750478/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26432245/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=A%20global%20reference%20for%20human%20genetic%20variation&author=C%20Genomes%20Project&author=A%20Auton&author=LD%20Brooks&author=RM%20Durbin&author=EP%20Garrison&volume=526&issue=7571&publication_year=2015&pages=68-74&pmid=26432245&doi=10.1038/nature15393&)

14. Olusanya BO, Kaplan M, Hansen TWR. Neonatal hyperbilirubinaemia: a global perspective. Lancet Child Adolesc Health. 2018;2(8):610–20. doi: 10.1016/S2352-4642(18)30139-1.  [DOI](https://doi.org/10.1016/S2352-4642(18)30139-1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30119720/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet%20Child%20Adolesc%20Health&title=Neonatal%20hyperbilirubinaemia:%20a%20global%20perspective&author=BO%20Olusanya&author=M%20Kaplan&author=TWR%20Hansen&volume=2&issue=8&publication_year=2018&pages=610-20&pmid=30119720&doi=10.1016/S2352-4642(18)30139-1&)

15. Lee B, Piersante T, Calkins KL. Neonatal hyperbilirubinemia. Pediatr Ann. 2022;51(6):e219–27. doi: 10.3928/19382359-20220407-02.  [DOI](https://doi.org/10.3928/19382359-20220407-02) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35667102/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pediatr%20Ann&title=Neonatal%20hyperbilirubinemia&author=B%20Lee&author=T%20Piersante&author=KL%20Calkins&volume=51&issue=6&publication_year=2022&pages=e219-27&pmid=35667102&doi=10.3928/19382359-20220407-02&)

16. Mei H, Dong X, Wu B, Wang H, Lu Y, Hu L, Wang J, Cao Y, Zhang R, Cheng G, et al. Clinical and genetic etiologies of neonatal unconjugated hyperbilirubinemia in the China neonatal genomes project. J Pediatr. 2022;243:53–60e59. doi: 10.1016/j.jpeds.2021.12.038.  [DOI](https://doi.org/10.1016/j.jpeds.2021.12.038) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34953813/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pediatr&title=Clinical%20and%20genetic%20etiologies%20of%20neonatal%20unconjugated%20hyperbilirubinemia%20in%20the%20China%20neonatal%20genomes%20project&author=H%20Mei&author=X%20Dong&author=B%20Wu&author=H%20Wang&author=Y%20Lu&volume=243&publication_year=2022&pages=53-60e59&pmid=34953813&doi=10.1016/j.jpeds.2021.12.038&)

17. Yang H, Lin F, Chen ZK, Zhang L, Xu JX, Wu YH, Gu JY, Ma YB, Li JD, Yang LY. UGT1A1 mutation association with increased bilirubin levels and severity of unconjugated hyperbilirubinemia in ABO incompatible newborns of China. BMC Pediatr. 2021;21(1):259. doi: 10.1186/s12887-021-02726-9.  [DOI](https://doi.org/10.1186/s12887-021-02726-9) | [PMC free article](/articles/PMC8167307/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34074250/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Pediatr&title=UGT1A1%20mutation%20association%20with%20increased%20bilirubin%20levels%20and%20severity%20of%20unconjugated%20hyperbilirubinemia%20in%20ABO%20incompatible%20newborns%20of%20China&author=H%20Yang&author=F%20Lin&author=ZK%20Chen&author=L%20Zhang&author=JX%20Xu&volume=21&issue=1&publication_year=2021&pages=259&pmid=34074250&doi=10.1186/s12887-021-02726-9&)

18. Kang TW, Kim HJ, Ju H, Kim JH, Jeon YJ, Lee HC, Kim KK, Kim JW, Lee S, Kim JY, et al. Genome-wide association of serum bilirubin levels in Korean population. Hum Mol Genet. 2010;19(18):3672–8. doi: 10.1093/hmg/ddq281.  [DOI](https://doi.org/10.1093/hmg/ddq281) | [PMC free article](/articles/PMC2928134/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20639394/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=Genome-wide%20association%20of%20serum%20bilirubin%20levels%20in%20Korean%20population&author=TW%20Kang&author=HJ%20Kim&author=H%20Ju&author=JH%20Kim&author=YJ%20Jeon&volume=19&issue=18&publication_year=2010&pages=3672-8&pmid=20639394&doi=10.1093/hmg/ddq281&)

19. Tadaka S, Hishinuma E, Komaki S, Motoike IN, Kawashima J, Saigusa D, Inoue J, Takayama J, Okamura Y, Aoki Y, et al. jMorp updates in 2020: large enhancement of multi-omics data resources on the general Japanese population. Nucleic Acids Res. 2021;49(D1):D536–44. doi: 10.1093/nar/gkaa1034.  [DOI](https://doi.org/10.1093/nar/gkaa1034) | [PMC free article](/articles/PMC7779038/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33179747/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=jMorp%20updates%20in%202020:%20large%20enhancement%20of%20multi-omics%20data%20resources%20on%20the%20general%20Japanese%20population&author=S%20Tadaka&author=E%20Hishinuma&author=S%20Komaki&author=IN%20Motoike&author=J%20Kawashima&volume=49&issue=D1&publication_year=2021&pages=D536-44&pmid=33179747&doi=10.1093/nar/gkaa1034&)

20. Zhou J, Yang C, Zhu W, Chen S, Zeng Y, Wang J, Zhao H, Chen Y, Lin F. Identification of Genetic Risk Factors for Neonatal Hyperbilirubinemia in Fujian Province, Southeastern China: A Case-Control Study. Biomed Res Int 2018, 2018:7803175.  [DOI](https://doi.org/10.1155/2018/7803175) | [PMC free article](/articles/PMC6157199/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30298137/)

21. Hanafusa H, Abe S, Ohyama S, Kyono Y, Kido T, Nakasone R, Ashina M, Tanimura K, Nozu K, Fujioka K. Influence of UGT1A1 genetic variants on free bilirubin levels in Japanese newborns: a preliminary study. Int J Environ Res Public Health 2022, 19(20).  [DOI](https://doi.org/10.3390/ijerph192013090) | [PMC free article](/articles/PMC9603041/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/36293671/)

22. Morais MB, Machado MV. Benign inheritable disorders of bilirubin metabolism manifested by conjugated hyperbilirubinemia-A narrative review. United Eur Gastroenterol J. 2022;10(7):745–53. doi: 10.1002/ueg2.12279.  [DOI](https://doi.org/10.1002/ueg2.12279) | [PMC free article](/articles/PMC9486497/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35860851/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=United%20Eur%20Gastroenterol%20J&title=Benign%20inheritable%20disorders%20of%20bilirubin%20metabolism%20manifested%20by%20conjugated%20hyperbilirubinemia-A%20narrative%20review&author=MB%20Morais&author=MV%20Machado&volume=10&issue=7&publication_year=2022&pages=745-53&pmid=35860851&doi=10.1002/ueg2.12279&)

23. Sanna S, Busonero F, Maschio A, McArdle PF, Usala G, Dei M, Lai S, Mulas A, Piras MG, Perseu L, et al. Common variants in the SLCO1B3 locus are associated with bilirubin levels and unconjugated hyperbilirubinemia. Hum Mol Genet. 2009;18(14):2711–8. doi: 10.1093/hmg/ddp203.  [DOI](https://doi.org/10.1093/hmg/ddp203) | [PMC free article](/articles/PMC2701337/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19419973/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=Common%20variants%20in%20the%20SLCO1B3%20locus%20are%20associated%20with%20bilirubin%20levels%20and%20unconjugated%20hyperbilirubinemia&author=S%20Sanna&author=F%20Busonero&author=A%20Maschio&author=PF%20McArdle&author=G%20Usala&volume=18&issue=14&publication_year=2009&pages=2711-8&pmid=19419973&doi=10.1093/hmg/ddp203&)

24. Dai X, Wu C, He Y, Gui L, Zhou L, Guo H, Yuan J, Yang B, Li J, Deng Q, et al. A genome-wide association study for serum bilirubin levels and gene-environment interaction in a Chinese population. Genet Epidemiol. 2013;37(3):293–300. doi: 10.1002/gepi.21711.  [DOI](https://doi.org/10.1002/gepi.21711) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23371916/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genet%20Epidemiol&title=A%20genome-wide%20association%20study%20for%20serum%20bilirubin%20levels%20and%20gene-environment%20interaction%20in%20a%20Chinese%20population&author=X%20Dai&author=C%20Wu&author=Y%20He&author=L%20Gui&author=L%20Zhou&volume=37&issue=3&publication_year=2013&pages=293-300&pmid=23371916&doi=10.1002/gepi.21711&)

25. Yang H, Wang Q, Zheng L, Lin M, Zheng XB, Lin F, Yang LY. Multiple genetic modifiers of Bilirubin Metabolism involvement in significant neonatal hyperbilirubinemia in patients of Chinese descent. PLoS ONE. 2015;10(7):e0132034. doi: 10.1371/journal.pone.0132034.  [DOI](https://doi.org/10.1371/journal.pone.0132034) | [PMC free article](/articles/PMC4493094/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26146841/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&title=Multiple%20genetic%20modifiers%20of%20Bilirubin%20Metabolism%20involvement%20in%20significant%20neonatal%20hyperbilirubinemia%20in%20patients%20of%20Chinese%20descent&author=H%20Yang&author=Q%20Wang&author=L%20Zheng&author=M%20Lin&author=XB%20Zheng&volume=10&issue=7&publication_year=2015&pages=e0132034&pmid=26146841&doi=10.1371/journal.pone.0132034&)
