# A polymorphism in the VKORC1-regulator calumenin predicts higher warfarin doses in African-Americans

## Metadata
**Authors:** D Voora, DC Koboldt, CR King, PA Lenzini, CS Eby, R Porche-Sorbet, E Deych, M Crankshaw, PE Milligan, HL McLeod, SR Patel, LH Cavallari, PM Ridker, GR Grice, RD Miller, BF Gage
**Journal:** Clinical pharmacology and therapeutics
**Date:** 2010 Mar 3
**DOI:** [10.1038/clpt.2009.291](https://doi.org/10.1038/clpt.2009.291)
**PMID:** 20200517
**PMCID:** PMC2928561
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2928561/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC2928561/pdf/nihms197329.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC2928561/pdf/nihms197329.pdf)

## Abstract

**Background:** 
Warfarin demonstrates wide interindividual variability that is partly mediated by variants in CYP2C9 and VKORC1. Whether variants in CALU (vitamin K reductase regulator) influence warfarin dose is unknown.

**Methods and Results:** 
We resequenced CALU regions in a discovery cohort of dose-outliers: patients with high(>90th percentile, n=55) or low(<10th percentile, n=53) dose requirements(after accounting for known genetic and nongenetic variables). One CALU variant, rs339097, was associated with high-doses(p=0.01). We validated this variant as a predictor of higher warfarin doses in two replication cohorts: 1)496 patients of mixed ethnicity, 2)194 African-American patients. The G allele of rs339097(African-American and Caucasian allele frequency 0.14 and 0.002, respectively), was associated with a 14.5%(SD±7%) greater therapeutic dose(p=0.03) in the first replication cohort and a higher than predicted dose in the second replication cohort(allele frequency=0.14, one-sided p=0.03).

**Conclusions:** 
CALU rs339097 A>G is associated with higher warfarin dose requirements independent of known genetic and nongenetic predictors of warfarin dose in African-Americans.

Keywords: anticoagulants, calumenin, genetic polymorphism, pharmacogenetics, warfarin/administration & dosage

### Background

Warfarin demonstrates wide interindividual variability that is partly mediated by variants in *CYP2C9* and *VKORC1*. Whether variants in *CALU* (vitamin K reductase regulator) influence warfarin dose is unknown.

### Methods and Results

We resequenced *CALU* regions in a discovery cohort of dose-outliers: patients with high(>90th percentile, n=55) or low(<10th percentile, n=53) dose requirements(after accounting for known genetic and nongenetic variables). One *CALU* variant, rs339097, was associated with high-doses(p=0.01). We validated this variant as a predictor of higher warfarin doses in two replication cohorts: 1)496 patients of mixed ethnicity, 2)194 African-American patients. The G allele of rs339097(African-American and Caucasian allele frequency 0.14 and 0.002, respectively), was associated with a 14.5%(SD±7%) greater therapeutic dose(p=0.03) in the first replication cohort and a higher than predicted dose in the second replication cohort(allele frequency=0.14, one-sided p=0.03).

### Conclusions

*CALU* rs339097 A>G is associated with higher warfarin dose requirements independent of known genetic and nongenetic predictors of warfarin dose in African-Americans.

## Introduction

Warfarin sodium is a widely used anticoagulant for the prevention and treatment of venous thromboembolism, myocardial infarction, and stroke. Warfarin is characterized by a wide interindividual variability in dose response. In addition, warfarin has a narrow therapeutic index: International Normalized Ratio (INR) values greater than 3 or 4 increase the risk of bleeding while those significantly less than 2 are less effective.([1](#R1)) Several single nucleotide polymorphisms (SNPs) in Cytochrome P450 2C9 and 4F2 (*CYP2C9* and *CYP4F2,* respectively) and vitamin K 2,3-epoxide reductase complex subunit 1 (*VKORC1*) are associated with dose-response, but together explain only 35% of the variation in warfarin dose requirements in Caucasians ([2](#R2), [3](#R3)) and 10% in African-Americans([4](#R4))

Although recent whole genome association studies failed to identify novel variants outside of these three loci, they were limited to Caucasian patients.([5](#R5)–[7](#R7)) Thus, additional genes may help predict warfarin dose, especially in African-American populations, where *VKORC1* and *CYP2C9* SNPs are not as predictive of the dose-response([8](#R8), [9](#R9)) and where warfarin resistance may be more common([10](#R10)).

Studies of warfarin resistant rats and *in vitro* mRNA silencing experiments have identified calumenin (*CALU*) as a regulator of vitamin K 2,3-epoxide reductase (VKOR) and warfarin sensitivity.([11](#R11)–[13](#R13)) In humans, limited studies of genetic variation in *CALU* in unselected warfarin treated patients have yielded unconvincing results. For example, an exonic *CALU* SNP, rs2290228, was found in one patient with an exceptionally high warfarin dose([14](#R14)) but *CALU* SNPs have not been significantly associated with dose in other studies.([15](#R15), [16](#R16))

To systematically investigate if genetic variation in *CALU* is associated with warfarin sensitivity or resistance, we chose a two-stage study design ([Figure 1](#F1)). In the first stage we selected patients from a well-characterized parent cohort who were at the extremes of warfarin dose requirements after accounting for known genetic and nongenetic factors from a parent cohort of patients taking warfarin chronically ([Figure 1](#F1), left hand panel). In the second stage, we used two replication cohorts: 1) a larger, mixed ethnicity sample of the parent cohort ([Figure 1](#F1), right hand panel) and 2) an external cohort of African-American patients. We used these replication cohorts to validate how *CALU* SNPs correlated with higher warfarin doses while accounting for known genetic and clinical factors. This two-stage strategy has been shown to be powerful([17](#R17)) and successful at identifying novel variants using candidate genes (*ABCA1* for HDL cholesterol([18](#R18)) and *PCSK9* for LDL cholesterol([19](#R19))) or whole genome (*NOS1AP* for QT interval([20](#R20))) approaches. To identify novel variants in *CALU* we directly sequenced exonic regions of *CALU* as well as evolutionary conserved putative regulatory regions. To identify these regulatory elements we used a comparative genomics approach to identify noncoding elements that were conserved in the rat and mouse orthologs. We hypothesized that patients who required unusually high or low doses of warfarin (compared to their predicted dose based on a validated pharmacogenetic algorithm that incorporates genetic and nongenetic factors([9](#R9))) would be more likely to have clinically relevant SNPs in *CALU*. We then validated these associations in two larger replication cohorts.

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c1e1/2928561/8c0599d03991/nihms197329f1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=2928561_nihms197329f1.jpg)

Schematic of two-stage study design using a parent cohort of patients on stable warfarin therapy. In the discovery cohort (left panel, we sequenced CALU in patients who had the largest or smallest residual dosing errors (shaded) within the parent cohort. In the second stage, significant (P < 0.05, FDR < 0.010) SNPs from the discovery stage were then genotyped in the remainder of the parent cohort (shaded, replication cohort 1, right panel). Predicted doses were based on a validated pharmacogenetics algorithm.9 CALU, calumenin; FDR, false-discovery rate; SNP; single-nucleotide polymorphism.

## Results

### Discovery Cohort

From the parent cohort we selected 108 outliers: 53 patients formed the low-dose group and 55 patients formed the high-dose group ([Figure 1](#F1), **left panel**). Despite having similar predicted doses (4.7±1.9 vs. 5.2±1.9) based on the validated pharmac algorithm ([9](#R9)), the average dose in the high-dose group was five times greater than in the low-dose group ([Table 1](#T1)). As designed, the dose ratio (therapeutic/predicted doses) was higher in the high-dose group compared to the low-dose group: 2.4 ± 1.4 vs. 0.55 ± 0.07, p < 0.0001. The two groups were well matched with respect to other important determinants of warfarin responsiveness. ([Table 1](#T1))

### Table 1.

|   | Discovery |  | ReplicationCohort #1(n = 496) | ReplicationCohort #2(n = 194) |
| --- | --- | --- | --- | --- |
|   |  |  |  |  |
| Low-Dose(n=53) | High-Dose(n=55) |  |  |  |
|   |  |  |  |  |
| Therapeutic dose, mean±SD(mg/day) † | 2.6±1.1 | 12.7±8.3 | 4.8 ± 1.6 | 5.9 ± 2.3 |
|   |  |  |  |  |
| Age, mean±SD (years) | 62±15 | 59±15 | 61 ± 15 | 57 ± 16 |
|   |  |  |  |  |
| BSA, mean±SD (m2) | 2.01±0.41 | 2.04±0.31 | 2.0 ± 0.3 | 2.07 ± 0.33 |
|   |  |  |  |  |
| Current Smoker, N (%) | 7 (13) | 11 (20) | 60 (12) | 35 (18) |
|   |  |  |  |  |
| Medications |   |   |   |   |
| Amiodarone, N (%) | 4 (8) | 1 (2) | 9 (2) | 4 (2) |
| Simvastatin, N (%)* | 4 (8) | 0 (0) | 56 (11) | 64 (33) |
|   |  |  |  |  |
| Male | 20 (38%) | 29 (53%) | 255 (51%) | 47 (24%) |
|   |  |  |  |  |
| Race |   |   |   |   |
| Black, N (%) | 18 (34) | 21 (38) | 70 (14) | 194 (100) |
| White, N (%) | 34 (64) | 33 (60) | 407 (82) | 0 |
| Other, N (%) | 1 (2) | 1 (2) | 20 (4) | 0 |
|   |  |  |  |  |
| CYP2C9*2 (% carriers) | 30 | 23 | 11 | 5 |
|   |  |  |  |  |
| CYP2C9*3 (% carriers) | 13 | 15 | 6 | 2 |
|   |  |  |  |  |
| VKORC1-1639 A (% carriers) | 51 | 49 | 33 | 18 |

Table 1 Caption: Baseline Characteristics of Discovery and Replication Cohorts

The comparative genomics approach identified 10 highly conserved regions for resequencing that included all known *CALU* exons, portions of the 5’ and 3’ UTRs and promoter region, and several conserved transcription factor binding sites. ([Supplementary Methods](#SD1)) Our resequencing strategy identified eight polymorphisms in *CALU*: seven previously identified in public databases (dbSNP Build ID: 130) and one novel SNP. ([Table 2](#T2)) All eight SNPs were in Hardy-Weinberg Equilibrium (data not shown). Among these eight SNPs, the proportion of patients carrying the minor allele of one SNP, rs339097, was significantly different between groups. ([Table 2](#T2)) This A>G SNP was a significant (p=0.01, false discovery rate 9%) predictor of the high-dose group among African-American outliers: it was present in 7 of 17 (41%) high-dose outliers (minor allele frequency [MAF] 20%) vs. 1 of 19 (5%) low-dose outliers (MAF = 3%). In contrast, the allele was invariant in Caucasian outliers.

### Table 2.

| SNP | Region | Position | Alleles | MAF by Race inlow dose group(AA/white/O,%) | MAF by Race inhigh dose group(AA/white/O,%) | P-valuein whitepts | P-valuein AApts |
| --- | --- | --- | --- | --- | --- | --- | --- |
| rs12538139 | 5' utr | 128175777 | G/A | 10/23/0 | 8/21/50 | 0.7 | 0.9 |
| rs2290228 | exon | 128175884 | G/A | 12/16/50 | 10/23/0 | 0.09 | 0.5 |
| rs2290227 | intron | 128176217 | G/A | 0/1/0 | 5/6/0 | 0.1 | 0.9 |
| rs2307040 | intron | 128181842 | C/T | 12/37/0 | 13/26/50 | 0.3 | 0.8 |
| rs41274227 | intron | 128182044 | C/T | 0/3/0 | 5/6/0 | 0.1 | 0.9 |
| rs339097 | intron | 128186460 | A/G | 3/0/0 | 20/0/0 | 1.0 | 0.01 |
| 38412 | intron | 128195082 | C/T | 6/0/0 | 3/0/0 | 1.0 | 0.6 |
| rs1043550 | 3' utr | 128196461 | A/G | 16/36/0 | 15/32/50 | 0.8 | 0.9 |

Table 2 Caption: Variants discovered in CALU and association with high dose in outlier group (n = 108)

### Replication Cohorts

To validate these findings we first identified 496 patients from the parent cohort who had DNA and complete clinical data (replication cohort #1, [Figure 1](#F1) right hand panel). We genotyped these participants for *CALU* rs339097 and three *CALU* SNPs identified in the literature (rs2290228, rs2290227, and rs1043550)([14](#R14), [15](#R15), [21](#R21)). In a univariate analysis, only the *CALU* SNP rs339097 was significantly (p = 0.003) associated with therapeutic dose. Carriers of the minor (G) allele of rs339097 were more likely African-American ([Table 3](#T3)). In a multivariable regression model that accounted for *CYP2C9* and *VKORC1* genotypes and clinical factors including race, carriers of the G allele of *CALU* SNP rs339097 averaged (± SD) 14.5% (± 7%) higher therapeutic doses compared to noncarriers (p=0.03). In the entire parent cohort (discovery plus replication cohort #1), 25 out of the 31 carriers of the CALU rs339097 minor allele (83%) had larger therapeutic doses than predicted by the validated pharmacogenetic algorithm([9](#R9)) ([Figure 2](#F2)). In African-Americans, rs339097 individually explained 5.7% (R^2^ = 0.057) of the residual variation in warfarin. However, the allele was too rare in Caucasians to evaluate its influence on the R^2^ in that group. There was no significant two-way interaction between the *CALU* SNPs and either *VKORC1* (*VKORC1* T-1639A) or *CYP2C9* SNPs (*2, and *3).

### Table 3.

|   | Replication Cohort #1 |  | Replication Cohort #2 |  |
| --- | --- | --- | --- | --- |
|   |  |  |  |  |
| Characteristic | rs339097GG or AG(n = 21) | rs339097AA(n= 475) | rs339097GG or AG(n = 52) | rs339097AA(n= 142) |
|   |  |  |  |  |
| Therapeutic warfarindose (mg/day, mean§± SD ) | 6.5 ± 1.4 | 4.8 ± 1.6** | 6.3 ± 2.3 | 5.7 ± 2.3 |
|   |  |  |  |  |
| Age (years, mean ±SD) | 57 ± 19 | 62 ± 15 | 58 ± 15 | 56 ± 16 |
|   |  |  |  |  |
| Race (%) |   |   |   |   |
| African-American | 86% | 11%** | 100% | 100% |
| Caucasian | 10% | 85% | 0% | 0% |
| Other | 5% | 4% | 0% | 0% |
|   |  |  |  |  |
| BSA (m2, mean ±SD) | 2.0 ± 0.2 | 2.0 ± 0.3 | 2.0 ± 0.4 | 2.1 ± 0.3 |
|   |  |  |  |  |
| VKORC1 -1639 A† | 10% | 37% | 12% | 20% |
|   |  |  |  |  |
| CYP2C9*2† | 2% | 12% | 2% | 6% |
|   |  |  |  |  |
| CYP2C9*3† | 2% | 6% | 2% | 1% |
|   |  |  |  |  |
| Current smoker | 29% | 14%* | 10% | 21% |
|   |  |  |  |  |
| Current amiodaroneuse | 16% | 2%* | 2% | 2% |

Table 3 Caption: Characteristics of replication cohorts, stratified by CALU SNP rs339097

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c1e1/2928561/0393a6b36db9/nihms197329f2.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=2928561_nihms197329f2.jpg)

Carriers of CALU SNP rs339097 require higher-than-predicted therapeutic warfarin doses in the combined discovery and replication cohort 1. Therapeutic dose vs. predicated dose (based on a validated pharmacogenetic algorithm9) in carriers (•) and noncarriers (+) of the G allele for CALU rs339097. In this combined cohort, 25 of 31 (83%) carriers of the G allele required a higher-than-predicated warfarin dose to achieve a therapeutic INR. CALU, calumenin; INR, international normalized ratio; SNP, single-nucleotide polymorphism; WT, wild type.

Because the prevalence of the G allele for rs339097 varied significantly by race, we sought additional replication in 194 African-American patients on warfarin (replication cohort #2). The baseline characteristics of this group were similar, except for higher average dose requirements and a lower percentage of men ([Table 1](#T1)). In this second cohort, carriers of the G allele (MAF 14%) had a higher residual dosing error (therapeutic dose − predicted dose([9](#R9))) compared to noncarriers: mean residual (± SD) in carriers vs. noncarriers: 0.20 (± 2.0) vs. − 0.41 ± 1.9, one-sided p-value = 0.03, two-sided p-value = 0.06), confirming that despite known genetic/nongenetic factors, carriers of the G for *CALU* rs339097 allele required higher warfarin doses.

In a pooled analysis of all African-American patients who participated in this study, the minor allele of *CALU* rs339097 was associated with an 11% increase in warfarin dose requirements after accounting for other factors ([Table 4](#T4)).

### Table 4.

| Factor |   | Effect on warfarin dose |
| --- | --- | --- |
| Age in years, mean (SD) | 58 (16) | −6% per decade |
| BSA, m2, mean (SD) | 2.1 (0.4) | 24% per 0.5m2 |
| Percent Smokers | 20% | 13% |
| Percent Using Amiodarone | 3% | −16% |
| CALU Allele frequency | 0.14 | 11% per allele |
| CYP2C9 *2 Allele Frequency | 0.04 | −20% per allele |
| CYP2C9 *3 Allele Frequency | 0.01 | −34% per allele |
| VKORC1 3673A Frequency | 0.11 | −24% per allele |
| Percent with indication of DVT or PE | 43% | 2% |
| Target INR | 2.5 (0.13) | 4% per 0.25 increase in target INR |

Table 4 Caption: Influence of pharmacogenetic factors on therapeutic warfarin dose in pooled African-American patients (n = 241)

### In silico validation

Because hepatic *CALU* is overexpressed in rodents with warfarin resistance([11](#R11)) and rs339097 is intronic, we used publicly available gene expression data to determine whether genetic variation at rs339097 altered *CALU* gene expression in humans. In immortalized lymphoblastoid cell lines of 30 trios of African descent (parents, n = 60 and their adult children, n = 30), the number of G alleles was associated with higher *CALU* gene expression (mean normalized *CALU* expression ± SE in those with 0, 1, or 2 alleles: 10.01 ± 0.02, 10.17 ± 0.69, and 10.33 ± 0.69; respectively; p-value: 0.02).

## Discussion

Warfarin is the most widely prescribed oral anticoagulant and is characterized by wide interindividual variability. Although recent studies have identified SNPs in *VKORC1* and *CYP2C9* in conjunction with nongenetic factors ([9](#R9), [22](#R22), [23](#R23)) as significant predictors of dose requirements, more than half of the variability in warfarin dose has remained unexplained. Calumenin, a recently identified regulator of VKOR([13](#R13)), is overexpressed in warfarin resistant rats([11](#R11)) and has been suggested to underlie warfarin sensitivity in humans([14](#R14)), but had not been evaluated thoroughly.

In this pharmacogenetic study we found that carriers of the minor allele of the third intronic *CALU* SNP rs339097 required a dose that was 14.5% greater than predicted by *VKORC1*, *CYP2C9* and clinical factors. The A>G allele was rare in Caucasian (allele frequency 0.2%), but common in African-Americans (allele frequency 11%–14%).

This genetic association study had several methodological strengths: 1) use of a candidate gene approach, 2) resequencing of conserved regions to identify influential regions within this gene, and 3) employing a two-stage approach([18](#R18)–[20](#R20)) that first examines extreme phenotypes in a population and then carries forward significant SNPs to a larger, more general samples of patients. The two–stage approach allowed us to discover a *CALU* SNP associated with a high-dose warfarin group and then demonstrate that carriers of the minor, G allele, of rs339097 had higher than predicted doses in two general populations. The decision to study *CALU* was backed by evidence of its role in rodent warfarin resistance([11](#R11)) and *in vitro* evidence of calumenin inhibiting VKOR([13](#R13)). We resequenced highly conserved regions of *CALU* to maximize our ability to identify influential regions outside of the coding regions, and used the false discovery rate to account for multiple comparisons.

Calumenin is a member of the CREC family of proteins and localizes in the endoplasmic reticulum,([24](#R24)) where γ – carboxylase([25](#R25)) and VKOR([26](#R26)) also localize. In humans, two calumenin isoforms are expressed that use two different second exons (exon 2-1 and exon 2-2) and both isoforms were sequenced. In the rat, calumenin interacts with and inhibits VKOR, thereby inhibiting γ - carboxylase. Furthermore, certain warfarin-resistant rats have increased hepatic levels of calumenin that protect VKOR from inhibition by warfarin. The VKOR and γ - carboxylase isolated from these rats have normal warfarin sensitivity.([11](#R11), [12](#R12)) In cells genetically engineered cells to produce Factor IX or VII, *in vitro* silencing of *CALU* mRNA increases VKOR and γ - carboxylase activity and production of these VKOR dependent clotting factors.([13](#R13), [27](#R27)) In humans, a SNP in the 3’ UTR of *CALU* was associated with increased activity of one other vitamin-K dependent protein (protein S).([15](#R15)) Therefore, the available animal and *in vitro* evidence suggest that calumenin regulates VKOR and γ – carboxylase.

Although the molecular implications of rs3390907 are not known, using publicly available gene expression data from lymphoblastoid cell lines generated from trios of the YRI cohort([28](#R28)) we observed that the G allele of rs3390907 is associated with higher expression of CALU. Interestingly, this association with higher gene expression in carriers of the G allele for rs339097 is consistent with the findings in warfarin resistant rats where CALU was found to be overexpressed([11](#R11)). Based on HapMap linkage disequilibrium data of the YRI population, rs339097 is in the center of an 18-Kb haplotype block (Haploview([29](#R29))). This large block contains several conserved regions, particularly in the second intron, that were not examined as part of the current study. Therefore it is unclear if this SNP directly leads to alterations in calumenin expression or mRNA stability or is in linkage disequilibrium with a causative SNP within this haplotype block.

The *CALU* SNP identified in our populations, rs339097, had minor allele frequencies of 11% to 14% in African-Americans and 0.2% in Caucasians. Although we recruited subjects with a wide variety of clinical conditions, these allele frequencies are similar to those found in the YRI and CEU (Centre d'Etude du Polymorphisme Humain from Utah) populations studied by the HapMap consortium (19.5% and 0%, respectively). Thus, among patients starting warfarin, this SNP will have greater potential to benefit patients of African descent.

In the multivariable regression model that accounted for race, rs339097 remained a significant predictor of higher warfarin dose in the first replication cohort. Though the overall contribution of this SNP to the residual variance in African-Americans was modest, patients who carried this SNP had a 14.5% increase in their warfarin requirement per allele. The dose-effect is similar to the *CYP2C9*2* allele, which is associated with a 19% decrease, and to the *CYP4F2* rs2108622 allele, which is associated with a 4% to 12% increase in warfarin requirements. Identification of novel SNPs in the African-American population is important because lower allele frequencies of *VKORC1* and *CYP2C9* SNPs in this population make current pharmacogenetic models of warfarin dosing less accurate in that population.([8](#R8), [9](#R9), [30](#R30)) Therefore, future studies of pharmacogenetic warfarin therapy, particularly those with larger numbers of patients of African descent, could use additional SNPs and perhaps initial INR response([31](#R31)) to improve predictive accuracy. To facilitate these studies, we have incorporated the present findings into an online dosing algorithm: [www.warfarindosing.org](http://www.warfarindosing.org).

In summary, we found that carriers of the minor allele of the *CALU* SNP rs339097 require higher doses of warfarin. Because the incremental cost of adding a single SNP into commercial genotyping platforms is low, *CALU* rs339097 could be incorporated into these platforms and into clinical trials that include populations with African ancestry.

## Methods

### Patient Selection

Our parent cohort is comprised of approximately 700 patients on therapeutic doses of warfarin who attended an outpatient anticoagulation clinic affiliated with Barnes–Jewish Hospital at Washington University Medical Center (St. Louis), who participated in the PREVENT (**PRE**vention of **VEN**ous **T**hromboembolism) study, or who were initiated on warfarin during a hospitalization for orthopedic surgery.([2](#R2), [9](#R9), [32](#R32), [33](#R33)) These patients provided written informed consent, DNA, demographic variables, and medication histories as previously described.([9](#R9)) The parent cohort is representative of most ambulatory patients on warfarin therapy ([Table 1](#T1)) except that it also included four patients who were referred for unusual warfarin doses (> 20 mg/day or < 1 mg/day). Because of missing data or unavailable DNA, only 604 (86%) patients were available to construct the discovery and replication cohorts. The research protocol was approved by the institutional review board at Washington University in St. Louis.

#### Calumenin SNP discovery cohort

For all patients in the parent cohort, the therapeutic daily warfarin dose (in mg) was compared to the dose predicted by a validated pharmacogenetic algorithm ([9](#R9))). We then chose the 10% of this cohort with the largest residual dosing error (defined as therapeutic dose - predicted dose/therapeutic dose) to identify 108 outliers with DNA who were either warfarin sensitive (N = 53) or resistant (N = 55) ([Figure 1](#F1), **left panel**).

#### Calumenin SNP replication cohorts

From the remaining patients in the parent cohort we selected the 496 patients with DNA and complete clinical data recruited at Washington University Medical Center for further genotyping (Replication cohort #1, [Figure 1](#F1), right panel). Of these, three did not reach a stable dose and their therapeutic dose was estimated by two independent clinical pharmacists (PEM and GRG). The pharmacists were blinded to genotype and agreed in their estimation of therapeutic dose.

To validate our findings in an African-American population we (SRP, LHC) genotyped 194 African-American patients at the University of Illinois at Chicago (replication cohort #2)([34](#R34)). We excluded those (n = 7) with missing *VKORC1* and *CYP2C9* genotypes or those on known inducers of Cytochrome P450 enzymes.

#### CALU in silico gene expression replication

Using software developed by Ge et al.,([35](#R35)) we merged publicly available gene expression data([36](#R36)) from lymphoblastoid cell lines with genotype data from the HapMap project to test for an association between the minor (G) allele of rs339097 and *CALU* gene expression in trios of Yoruba people of Ibadan, Nigeria (YRI population, HapMap project).

### CALU Target Region Selection

*CALU* is a 32Kb gene located on human 7q32 and contains six exons. In humans two *CALU* isoforms are expressed that use two different exons 2 (exon 2-1 and exon 2-2).

We used two complementary methods to target region selection for direct sequencing of *CALU*. We used two comparative genomics approaches to find 10 highly conserved (>90% across rat and mouse orthologs) regions of *CALU* and 5Kb of its 5' and 3' flanking regions using the VISTA Browser ([http://genome.lbl.gov/vista/index.shtml](http://genome.lbl.gov/vista/index.shtml)) and the University of California at Santa Cruz (UCSC) Genome Browser's "knownGene" database file and mRNAs from GenBank ([http://genome.ucsc.edu/cgi-bin/hgTrackUi?hgsid=66289578&c=chr7&g=knownGene](http://genome.ucsc.edu/cgi-bin/hgTrackUi?hgsid=66289578&c=chr7&g=knownGene), March 2004). These 10 regions contained all six exons and the alternatively spliced exon 2 (as expected) and a 69-bp segment of the 5’UTR. In addition, we identified two regions in the 3’UTR and one in the 5’UTR that were highly conserved. We designed primers and sequenced these 10 regions as previously described (correction to original description ([37](#R37)): 100 µM each of the 4 dNTPs was used). ([Supplementary Methods](#SD1)).

### Genotyping Protocols

To validate the SNPs identified by sequencing we first genotyped the 108 outliers for the SNPs identified by resequencing using PCR and Pyrosequencing^®^ ([38](#R38), [39](#R39)). PCR primers were designed using the UCSC Golden Path Human Genome Browser May 2004 Build ([http://genome.ucsc.edu/cgi-bin/hgGateway](http://genome.ucsc.edu/cgi-bin/hgGateway)) and *CALU* Refseq [NM_001219](https://www.ncbi.nlm.nih.gov/nuccore/NM_001219). This confirmatory step revealed eight polymorphic *CALU* SNPs for analysis in the discovery cohort: c38412t, rs41274227, rs1043550, rs12538139, rs2290227, rs2290228, rs2307040, rs339097. For the replication cohort, we used Pyrosequencing^®^ ([40](#R40)) to genotype selected SNPs.

### Statistical Analyses

In the SNP discovery analysis, we used a chi-squared analysis to compare the prevalence of SNPs between the high-dose and low-dose groups and the Fisher’s exact test when cell frequencies were low. As recommended,([41](#R41)) we calculated and report false discovery rates.

In replication cohort #1, we tested *CALU* SNPs (rs339097, rs2290228, rs2290227, and rs1043550) in all subjects and in the subsets of Caucasians and African-Americans. We tested for Hardy-Weinberg Equilibrium using Haploview 3.32; if the SNP was in HWE (all were), we used simple linear regression to regress the therapeutic daily warfarin dose (log transformed) onto the number of minor alleles in univariate and multivariate models that included *CYP2C9* and *VKORC1* genotypes, age, smoking, body surface area, concomitant medications, target INR, venous thromboembolism indication, and race. We used an unpaired t-test to compare therapeutic doses (log transformed) of carriers and non carriers of the *CALU* minor allele. We tested for two-way interactions between SNPs in *CALU, VKORC1,* and *CYP2C9*.

In replication cohort #2, we tested for association of *CALU* SNP rs339097 A>G with residual dosing error (ln(therapeutic) − ln(predicted warfarin dose)([9](#R9))) using a one-sided (unpaired) t-test. The normality of residuals was confirmed using the Kolmogorov-Smirnov test.

For the *in silico* replication we use a generalized estimating equation to regress the normalized *CALU* gene expression onto the number of minor alleles of rs339097 (coded as 0, 1, 2) in YRI children and parents using SAS procedure GENMOD to account for the family structure of the trios. Normality of residuals was confirmed Kolmogorov-Smirnov test.

We performed all statistical analyses in SAS (Version 9.1; SAS Institute, Inc; Cary, NC) except that replication cohort #2 was analyzed in SPSS (Release 15.0; SPSS Inc., Chicago, IL). Statistical significance refers to a p-value < 0.05.

## Supplementary Material

## Acknowledgements

This study was funded by the National Institutes of Health, R01 HL074724

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Oden A, Fahlen M, Hart RG. Optimal INR for prevention of stroke and death in atrial fibrillation: a critical appraisal. Thromb Res. 2006;117:493–499. doi: 10.1016/j.thromres.2004.11.025.  [DOI](https://doi.org/10.1016/j.thromres.2004.11.025) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16517250/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Thromb%20Res&title=Optimal%20INR%20for%20prevention%20of%20stroke%20and%20death%20in%20atrial%20fibrillation:%20a%20critical%20appraisal&author=A%20Oden&author=M%20Fahlen&author=RG%20Hart&volume=117&publication_year=2006&pages=493-499&pmid=16517250&doi=10.1016/j.thromres.2004.11.025&)

2. Rieder MJ, Reiner AP, Gage BF, Nickerson DA, Eby CS, McLeod HL, et al. Effect of VKORC1 haplotypes on transcriptional regulation and warfarin dose. N Engl J Med. 2005;352:2285–2293. doi: 10.1056/NEJMoa044503.  [DOI](https://doi.org/10.1056/NEJMoa044503) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15930419/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Effect%20of%20VKORC1%20haplotypes%20on%20transcriptional%20regulation%20and%20warfarin%20dose&author=MJ%20Rieder&author=AP%20Reiner&author=BF%20Gage&author=DA%20Nickerson&author=CS%20Eby&volume=352&publication_year=2005&pages=2285-2293&pmid=15930419&doi=10.1056/NEJMoa044503&)

3. Caldwell MD, Awad T, Johnson JA, Gage BF, Falkowski M, Gardina P, et al. CYP4F2 genetic variant alters required warfarin dose. Blood. 2008 doi: 10.1182/blood-2007-11-122010. blood-2007-11-122010.  [DOI](https://doi.org/10.1182/blood-2007-11-122010) | [PMC free article](/articles/PMC2288721/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18250228/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=CYP4F2%20genetic%20variant%20alters%20required%20warfarin%20dose&author=MD%20Caldwell&author=T%20Awad&author=JA%20Johnson&author=BF%20Gage&author=M%20Falkowski&publication_year=2008&pmid=18250228&doi=10.1182/blood-2007-11-122010&)

4. Limdi NA, Arnett DK, Goldstein JA, Beasley TM, McGwin G, Adler BK, et al. Influence of CYP2C9 and VKORC1 on warfarin dose, anticoagulation attainment and maintenance among European-Americans and African-Americans. Pharmacogenomics. 2008;9:511–526. doi: 10.2217/14622416.9.5.511.  [DOI](https://doi.org/10.2217/14622416.9.5.511) | [PMC free article](/articles/PMC2757655/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18466099/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Influence%20of%20CYP2C9%20and%20VKORC1%20on%20warfarin%20dose,%20anticoagulation%20attainment%20and%20maintenance%20among%20European-Americans%20and%20African-Americans&author=NA%20Limdi&author=DK%20Arnett&author=JA%20Goldstein&author=TM%20Beasley&author=G%20McGwin&volume=9&publication_year=2008&pages=511-526&pmid=18466099&doi=10.2217/14622416.9.5.511&)

5. Cooper GM, Johnson JA, Langaee TY, Feng H, Stanaway IB, Schwarz UI, et al. A genome-wide scan for common genetic variants with a large influence on warfarin maintenance dose. Blood. 2008 doi: 10.1182/blood-2008-01-134247.  [DOI](https://doi.org/10.1182/blood-2008-01-134247) | [PMC free article](/articles/PMC2515139/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18535201/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=A%20genome-wide%20scan%20for%20common%20genetic%20variants%20with%20a%20large%20influence%20on%20warfarin%20maintenance%20dose&author=GM%20Cooper&author=JA%20Johnson&author=TY%20Langaee&author=H%20Feng&author=IB%20Stanaway&publication_year=2008&pmid=18535201&doi=10.1182/blood-2008-01-134247&)

6. Takeuchi F, McGinnis R, Bourgeois S, Barnes C, Eriksson N, Soranzo N, et al. A genome-wide association study confirms VKORC1, CYP2C9, and CYP4F2 as principal genetic determinants of warfarin dose. PLoS Genet. 2009;5:e1000433. doi: 10.1371/journal.pgen.1000433.  [DOI](https://doi.org/10.1371/journal.pgen.1000433) | [PMC free article](/articles/PMC2652833/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19300499/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet&title=A%20genome-wide%20association%20study%20confirms%20VKORC1,%20CYP2C9,%20and%20CYP4F2%20as%20principal%20genetic%20determinants%20of%20warfarin%20dose&author=F%20Takeuchi&author=R%20McGinnis&author=S%20Bourgeois&author=C%20Barnes&author=N%20Eriksson&volume=5&publication_year=2009&pages=e1000433&pmid=19300499&doi=10.1371/journal.pgen.1000433&)

7. Teichert M, Eijgelsheim M, Rivadeneira F, Uitterlinden AG, van Schaik RHN, Hofman A, et al. A genome-wide association study of acenocoumarol maintenance dosage. Hum Mol Genet. 2009 doi: 10.1093/hmg/ddp309. ddp309.  [DOI](https://doi.org/10.1093/hmg/ddp309) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19578179/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=A%20genome-wide%20association%20study%20of%20acenocoumarol%20maintenance%20dosage&author=M%20Teichert&author=M%20Eijgelsheim&author=F%20Rivadeneira&author=AG%20Uitterlinden&author=RHN%20van%20Schaik&publication_year=2009&pmid=19578179&doi=10.1093/hmg/ddp309&)

8. Schelleman H, Chen Z, Kealey C, Whitehead AS, Christie J, Price M, et al. Warfarin response and vitamin K epoxide reductase complex 1 in African Americans and Caucasians. Clin Pharmacol Ther. 2007;81:742–747. doi: 10.1038/sj.clpt.6100144.  [DOI](https://doi.org/10.1038/sj.clpt.6100144) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17329985/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Warfarin%20response%20and%20vitamin%20K%20epoxide%20reductase%20complex%201%20in%20African%20Americans%20and%20Caucasians&author=H%20Schelleman&author=Z%20Chen&author=C%20Kealey&author=AS%20Whitehead&author=J%20Christie&volume=81&publication_year=2007&pages=742-747&pmid=17329985&doi=10.1038/sj.clpt.6100144&)

9. Gage B, Eby C, Johnson J, Deych E, Rieder M, Ridker P, et al. Use of Pharmacogenetic and Clinical Factors to Predict the Therapeutic Dose of Warfarin. Clin Pharmacol Ther. 2008 doi: 10.1038/clpt.2008.10.  [DOI](https://doi.org/10.1038/clpt.2008.10) | [PMC free article](/articles/PMC2683977/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18305455/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Use%20of%20Pharmacogenetic%20and%20Clinical%20Factors%20to%20Predict%20the%20Therapeutic%20Dose%20of%20Warfarin&author=B%20Gage&author=C%20Eby&author=J%20Johnson&author=E%20Deych&author=M%20Rieder&publication_year=2008&pmid=18305455&doi=10.1038/clpt.2008.10&)

10. Diab F, Feffer S. Hereditary warfarin resistance. South Med J. 1994;87:407–409. doi: 10.1097/00007611-199403000-00023.  [DOI](https://doi.org/10.1097/00007611-199403000-00023) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8134868/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=South%20Med%20J&title=Hereditary%20warfarin%20resistance&author=F%20Diab&author=S%20Feffer&volume=87&publication_year=1994&pages=407-409&pmid=8134868&doi=10.1097/00007611-199403000-00023&)

11. Wallin R, Hutson SM, Cain D, Sweatt A, Sane DC. A molecular mechanism for genetic warfarin resistance in the rat. Faseb J. 2001;15:2542–2544. doi: 10.1096/fj.01-0337fje.  [DOI](https://doi.org/10.1096/fj.01-0337fje) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11641264/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Faseb%20J&title=A%20molecular%20mechanism%20for%20genetic%20warfarin%20resistance%20in%20the%20rat&author=R%20Wallin&author=SM%20Hutson&author=D%20Cain&author=A%20Sweatt&author=DC%20Sane&volume=15&publication_year=2001&pages=2542-2544&pmid=11641264&doi=10.1096/fj.01-0337fje&)

12. Wajih N, Sane DC, Hutson SM, Wallin R. The inhibitory effect of calumenin on the vitamin K-dependent gamma-carboxylation system. Characterization of the system in normal and warfarin-resistant rats. J Biol Chem. 2004;279:25276–25283. doi: 10.1074/jbc.M401645200.  [DOI](https://doi.org/10.1074/jbc.M401645200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15075329/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=The%20inhibitory%20effect%20of%20calumenin%20on%20the%20vitamin%20K-dependent%20gamma-carboxylation%20system.%20Characterization%20of%20the%20system%20in%20normal%20and%20warfarin-resistant%20rats&author=N%20Wajih&author=DC%20Sane&author=SM%20Hutson&author=R%20Wallin&volume=279&publication_year=2004&pages=25276-25283&pmid=15075329&doi=10.1074/jbc.M401645200&)

13. Wajih N, Hutson SM, Wallin R. siRNA silencing of calumenin enhances functional factor IX production. Blood. 2006;108:3757–3760. doi: 10.1182/blood-2006-02-004671.  [DOI](https://doi.org/10.1182/blood-2006-02-004671) | [PMC free article](/articles/PMC1895465/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16902154/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=siRNA%20silencing%20of%20calumenin%20enhances%20functional%20factor%20IX%20production&author=N%20Wajih&author=SM%20Hutson&author=R%20Wallin&volume=108&publication_year=2006&pages=3757-3760&pmid=16902154&doi=10.1182/blood-2006-02-004671&)

14. Vecsler M, Loebstein R, Almog S, Kurnik D, Goldman B, Halkin H, et al. Combined genetic profiles of components and regulators of the vitamin K-dependent gamma-carboxylation system affect individual sensitivity to warfarin. Thromb Haemost. 2006;95:205–211. doi: 10.1160/TH05-06-0446.  [DOI](https://doi.org/10.1160/TH05-06-0446) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16493479/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Thromb%20Haemost&title=Combined%20genetic%20profiles%20of%20components%20and%20regulators%20of%20the%20vitamin%20K-dependent%20gamma-carboxylation%20system%20affect%20individual%20sensitivity%20to%20warfarin&author=M%20Vecsler&author=R%20Loebstein&author=S%20Almog&author=D%20Kurnik&author=B%20Goldman&volume=95&publication_year=2006&pages=205-211&pmid=16493479&doi=10.1160/TH05-06-0446&)

15. Kimura R, Kokubo Y, Miyashita K, Otsubo R, Nagatsuka K, Otsuki T, et al. Polymorphisms in vitamin K-dependent gamma-carboxylation-related genes influence interindividual variability in plasma protein C and protein S activities in the general population. Int J Hematol. 2006;84:387–397. doi: 10.1532/IJH97.06082.  [DOI](https://doi.org/10.1532/IJH97.06082) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17189218/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Hematol&title=Polymorphisms%20in%20vitamin%20K-dependent%20gamma-carboxylation-related%20genes%20influence%20interindividual%20variability%20in%20plasma%20protein%20C%20and%20protein%20S%20activities%20in%20the%20general%20population&author=R%20Kimura&author=Y%20Kokubo&author=K%20Miyashita&author=R%20Otsubo&author=K%20Nagatsuka&volume=84&publication_year=2006&pages=387-397&pmid=17189218&doi=10.1532/IJH97.06082&)

16. Wadelius M, Chen LY, Eriksson N, Bumpstead S, Ghori J, Wadelius C, et al. Association of warfarin dose with genes involved in its action and metabolism. Hum Genet. 2007;121:23–34. doi: 10.1007/s00439-006-0260-8.  [DOI](https://doi.org/10.1007/s00439-006-0260-8) | [PMC free article](/articles/PMC1797064/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17048007/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Genet&title=Association%20of%20warfarin%20dose%20with%20genes%20involved%20in%20its%20action%20and%20metabolism&author=M%20Wadelius&author=LY%20Chen&author=N%20Eriksson&author=S%20Bumpstead&author=J%20Ghori&volume=121&publication_year=2007&pages=23-34&pmid=17048007&doi=10.1007/s00439-006-0260-8&)

17. Satagopan JM, Venkatraman ES, Begg CB. Two-stage designs for gene-disease association studies with sample size constraints. Biometrics. 2004;60:589–597. doi: 10.1111/j.0006-341X.2004.00207.x.  [DOI](https://doi.org/10.1111/j.0006-341X.2004.00207.x) | [PMC free article](/articles/PMC8985053/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15339280/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biometrics&title=Two-stage%20designs%20for%20gene-disease%20association%20studies%20with%20sample%20size%20constraints&author=JM%20Satagopan&author=ES%20Venkatraman&author=CB%20Begg&volume=60&publication_year=2004&pages=589-597&pmid=15339280&doi=10.1111/j.0006-341X.2004.00207.x&)

18. Cohen JC, Kiss RS, Pertsemlidis A, Marcel YL, McPherson R, Hobbs HH. Multiple rare alleles contribute to low plasma levels of HDL cholesterol. Science. 2004;305:869–872. doi: 10.1126/science.1099870.  [DOI](https://doi.org/10.1126/science.1099870) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15297675/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Multiple%20rare%20alleles%20contribute%20to%20low%20plasma%20levels%20of%20HDL%20cholesterol&author=JC%20Cohen&author=RS%20Kiss&author=A%20Pertsemlidis&author=YL%20Marcel&author=R%20McPherson&volume=305&publication_year=2004&pages=869-872&pmid=15297675&doi=10.1126/science.1099870&)

19. Cohen J, Pertsemlidis A, Kotowski IK, Graham R, Garcia CK, Hobbs HH. Low LDL cholesterol in individuals of African descent resulting from frequent nonsense mutations in PCSK9. Nat Genet. 2005;37:161–165. doi: 10.1038/ng1509.  [DOI](https://doi.org/10.1038/ng1509) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15654334/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Low%20LDL%20cholesterol%20in%20individuals%20of%20African%20descent%20resulting%20from%20frequent%20nonsense%20mutations%20in%20PCSK9&author=J%20Cohen&author=A%20Pertsemlidis&author=IK%20Kotowski&author=R%20Graham&author=CK%20Garcia&volume=37&publication_year=2005&pages=161-165&pmid=15654334&doi=10.1038/ng1509&)

20. Arking DE, Pfeufer A, Post W, Kao WH, Newton-Cheh C, Ikeda M, et al. A common genetic variant in the NOS1 regulator NOS1AP modulates cardiac repolarization. Nat Genet. 2006;38:644–651. doi: 10.1038/ng1790.  [DOI](https://doi.org/10.1038/ng1790) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16648850/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=A%20common%20genetic%20variant%20in%20the%20NOS1%20regulator%20NOS1AP%20modulates%20cardiac%20repolarization&author=DE%20Arking&author=A%20Pfeufer&author=W%20Post&author=WH%20Kao&author=C%20Newton-Cheh&volume=38&publication_year=2006&pages=644-651&pmid=16648850&doi=10.1038/ng1790&)

21. Gonzalez-Conejero R, Corral J, Roldan V, Ferrer F, Sanchez-Serrano I, Sanchez-Blanco JJ, et al. The genetic interaction between VKORC1 c1173t and calumenin a29809g modulates the anticoagulant response of acenocoumarol. J Thromb Haemost. 2007;5:1701–1706. doi: 10.1111/j.1538-7836.2007.02630.x.  [DOI](https://doi.org/10.1111/j.1538-7836.2007.02630.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17596133/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Thromb%20Haemost&title=The%20genetic%20interaction%20between%20VKORC1%20c1173t%20and%20calumenin%20a29809g%20modulates%20the%20anticoagulant%20response%20of%20acenocoumarol&author=R%20Gonzalez-Conejero&author=J%20Corral&author=V%20Roldan&author=F%20Ferrer&author=I%20Sanchez-Serrano&volume=5&publication_year=2007&pages=1701-1706&pmid=17596133&doi=10.1111/j.1538-7836.2007.02630.x&)

22. Wadelius M, Chen LY, Lindh JD, Eriksson N, Ghori MJR, Bumpstead S, et al. The largest prospective warfarin-treated cohort supports genetic forecasting. Blood. 2008 doi: 10.1182/blood-2008-04-149070. blood-2008-04-149070.  [DOI](https://doi.org/10.1182/blood-2008-04-149070) | [PMC free article](/articles/PMC2630264/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18574025/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=The%20largest%20prospective%20warfarin-treated%20cohort%20supports%20genetic%20forecasting&author=M%20Wadelius&author=LY%20Chen&author=JD%20Lindh&author=N%20Eriksson&author=MJR%20Ghori&publication_year=2008&pmid=18574025&doi=10.1182/blood-2008-04-149070&)

23. Thacker SM, Grice GR, Milligan PE, Gage BF. Dosing anticoagulant therapy with coumarin drugs: is genotyping clinically useful? Yes. J Thromb Haemost. 2008;6:1445–1449. doi: 10.1111/j.1538-7836.2008.03074.x.  [DOI](https://doi.org/10.1111/j.1538-7836.2008.03074.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18627440/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Thromb%20Haemost&title=Dosing%20anticoagulant%20therapy%20with%20coumarin%20drugs:%20is%20genotyping%20clinically%20useful?%20Yes&author=SM%20Thacker&author=GR%20Grice&author=PE%20Milligan&author=BF%20Gage&volume=6&publication_year=2008&pages=1445-1449&pmid=18627440&doi=10.1111/j.1538-7836.2008.03074.x&)

24. Vorum H, Liu X, Madsen P, Rasmussen HH, Honore B. Molecular cloning of a cDNA encoding human calumenin, expression in Escherichia coli and analysis of its Ca2+-binding activity. Biochim Biophys Acta. 1998;1386:121–131. doi: 10.1016/s0167-4838(98)00089-2.  [DOI](https://doi.org/10.1016/s0167-4838(98)00089-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9675259/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochim%20Biophys%20Acta&title=Molecular%20cloning%20of%20a%20cDNA%20encoding%20human%20calumenin,%20expression%20in%20Escherichia%20coli%20and%20analysis%20of%20its%20Ca2+-binding%20activity&author=H%20Vorum&author=X%20Liu&author=P%20Madsen&author=HH%20Rasmussen&author=B%20Honore&volume=1386&publication_year=1998&pages=121-131&pmid=9675259&doi=10.1016/s0167-4838(98)00089-2&)

25. Presnell SR, Stafford DW. The vitamin K-dependent carboxylase. Thromb Haemost. 2002;87:937–946.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/12083499/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Thromb%20Haemost&title=The%20vitamin%20K-dependent%20carboxylase&author=SR%20Presnell&author=DW%20Stafford&volume=87&publication_year=2002&pages=937-946&pmid=12083499&)

26. Rost S, Fregin A, Ivaskevicius V, Conzelmann E, Hortnagel K, Pelz HJ, et al. Mutations in VKORC1 cause warfarin resistance and multiple coagulation factor deficiency type 2. Nature. 2004;427:537–541. doi: 10.1038/nature02214.  [DOI](https://doi.org/10.1038/nature02214) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14765194/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Mutations%20in%20VKORC1%20cause%20warfarin%20resistance%20and%20multiple%20coagulation%20factor%20deficiency%20type%202&author=S%20Rost&author=A%20Fregin&author=V%20Ivaskevicius&author=E%20Conzelmann&author=K%20Hortnagel&volume=427&publication_year=2004&pages=537-541&pmid=14765194&doi=10.1038/nature02214&)

27. Wajih N, Owen J, Wallin R. Enhanced functional recombinant factor VII production by HEK 293 cells stably transfected with VKORC1 where the gamma-carboxylase inhibitor calumenin is stably suppressed by shRNA transfection. Thromb Res. 2008;122:405–410. doi: 10.1016/j.thromres.2007.11.002.  [DOI](https://doi.org/10.1016/j.thromres.2007.11.002) | [PMC free article](/articles/PMC2556075/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18177690/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Thromb%20Res&title=Enhanced%20functional%20recombinant%20factor%20VII%20production%20by%20HEK%20293%20cells%20stably%20transfected%20with%20VKORC1%20where%20the%20gamma-carboxylase%20inhibitor%20calumenin%20is%20stably%20suppressed%20by%20shRNA%20transfection&author=N%20Wajih&author=J%20Owen&author=R%20Wallin&volume=122&publication_year=2008&pages=405-410&pmid=18177690&doi=10.1016/j.thromres.2007.11.002&)

28. Stranger BE, Forrest MS, Dunning M, Ingle CE, Beazley C, Thorne N, et al. Relative Impact of Nucleotide and Copy Number Variation on Gene Expression Phenotypes. Science. 2007;315:848–853. doi: 10.1126/science.1136678.  [DOI](https://doi.org/10.1126/science.1136678) | [PMC free article](/articles/PMC2665772/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17289997/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Relative%20Impact%20of%20Nucleotide%20and%20Copy%20Number%20Variation%20on%20Gene%20Expression%20Phenotypes&author=BE%20Stranger&author=MS%20Forrest&author=M%20Dunning&author=CE%20Ingle&author=C%20Beazley&volume=315&publication_year=2007&pages=848-853&pmid=17289997&doi=10.1126/science.1136678&)

29. Barrett JC, Fry B, Maller J, Daly MJ. Haploview: analysis and visualization of LD and haplotype maps. Bioinformatics. 2005;21:263–265. doi: 10.1093/bioinformatics/bth457.  [DOI](https://doi.org/10.1093/bioinformatics/bth457) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15297300/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=Haploview:%20analysis%20and%20visualization%20of%20LD%20and%20haplotype%20maps&author=JC%20Barrett&author=B%20Fry&author=J%20Maller&author=MJ%20Daly&volume=21&publication_year=2005&pages=263-265&pmid=15297300&doi=10.1093/bioinformatics/bth457&)

30. Limdi NA, Beasley TM, Crowley MR, Goldstein JA, Rieder MJ, Flockhart DA, et al. VKORC1 polymorphisms, haplotypes and haplotype groups on warfarin dose among African-Americans and European-Americans. Pharmacogenomics. 2008;9:1445–1458. doi: 10.2217/14622416.9.10.1445.  [DOI](https://doi.org/10.2217/14622416.9.10.1445) | [PMC free article](/articles/PMC2586955/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18855533/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=VKORC1%20polymorphisms,%20haplotypes%20and%20haplotype%20groups%20on%20warfarin%20dose%20among%20African-Americans%20and%20European-Americans&author=NA%20Limdi&author=TM%20Beasley&author=MR%20Crowley&author=JA%20Goldstein&author=MJ%20Rieder&volume=9&publication_year=2008&pages=1445-1458&pmid=18855533&doi=10.2217/14622416.9.10.1445&)

31. Lenzini PA, Grice GR, Milligan PE, Dowd MB, Subherwal S, Deych E, et al. Laboratory and clinical outcomes of pharmacogenetic vs. clinical protocols for warfarin initiation in orthopedic patients. J Thromb Haemost. 2008 doi: 10.1111/j.1538-7836.2008.03095.x.  [DOI](https://doi.org/10.1111/j.1538-7836.2008.03095.x) | [PMC free article](/articles/PMC2920450/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18662264/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Thromb%20Haemost&title=Laboratory%20and%20clinical%20outcomes%20of%20pharmacogenetic%20vs.%20clinical%20protocols%20for%20warfarin%20initiation%20in%20orthopedic%20patients&author=PA%20Lenzini&author=GR%20Grice&author=PE%20Milligan&author=MB%20Dowd&author=S%20Subherwal&publication_year=2008&pmid=18662264&doi=10.1111/j.1538-7836.2008.03095.x&)

32. Voora D, Eby C, Linder MW, Milligan PE, Bukaveckas BL, McLeod HL, et al. Prospective dosing of warfarin based on cytochrome P-450 2C9 genotype. Thromb Haemost. 2005;93:700–705. doi: 10.1160/TH04-08-0542.  [DOI](https://doi.org/10.1160/TH04-08-0542) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15841315/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Thromb%20Haemost&title=Prospective%20dosing%20of%20warfarin%20based%20on%20cytochrome%20P-450%202C9%20genotype&author=D%20Voora&author=C%20Eby&author=MW%20Linder&author=PE%20Milligan&author=BL%20Bukaveckas&volume=93&publication_year=2005&pages=700-705&pmid=15841315&doi=10.1160/TH04-08-0542&)

33. Ridker PM, Goldhaber SZ, Danielson E, Rosenberg Y, Eby CS, Deitcher SR, et al. Long-term, low-intensity warfarin therapy for the prevention of recurrent venous thromboembolism. N Engl J Med. 2003;348:1425–1434. doi: 10.1056/NEJMoa035029.  [DOI](https://doi.org/10.1056/NEJMoa035029) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12601075/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Long-term,%20low-intensity%20warfarin%20therapy%20for%20the%20prevention%20of%20recurrent%20venous%20thromboembolism&author=PM%20Ridker&author=SZ%20Goldhaber&author=E%20Danielson&author=Y%20Rosenberg&author=CS%20Eby&volume=348&publication_year=2003&pages=1425-1434&pmid=12601075&doi=10.1056/NEJMoa035029&)

34. Momary KM, Shapiro NL, Viana MA, Nutescu EA, Helgason CM, Cavallari LH. Factors influencing warfarin dose requirements in African-Americans. Pharmacogenomics. 2007;8:1535–1544. doi: 10.2217/14622416.8.11.1535.  [DOI](https://doi.org/10.2217/14622416.8.11.1535) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18034618/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Factors%20influencing%20warfarin%20dose%20requirements%20in%20African-Americans&author=KM%20Momary&author=NL%20Shapiro&author=MA%20Viana&author=EA%20Nutescu&author=CM%20Helgason&volume=8&publication_year=2007&pages=1535-1544&pmid=18034618&doi=10.2217/14622416.8.11.1535&)

35. Ge D, Zhang K, Need AC, Martin O, Fellay J, Urban TJ, et al. WGAViewer: Software for genomic annotation of whole genome association studies. Genome Res. 2008 doi: 10.1101/gr.071571.107. gr.071571.107.  [DOI](https://doi.org/10.1101/gr.071571.107) | [PMC free article](/articles/PMC2279251/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18256235/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res&title=WGAViewer:%20Software%20for%20genomic%20annotation%20of%20whole%20genome%20association%20studies&author=D%20Ge&author=K%20Zhang&author=AC%20Need&author=O%20Martin&author=J%20Fellay&publication_year=2008&pmid=18256235&doi=10.1101/gr.071571.107&)

36. Stranger BE, Nica AC, Forrest MS, Dimas A, Bird CP, Beazley C, et al. Population genomics of human gene expression. Nat Genet. 2007;39:1217–1224. doi: 10.1038/ng2142.  [DOI](https://doi.org/10.1038/ng2142) | [PMC free article](/articles/PMC2683249/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17873874/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Population%20genomics%20of%20human%20gene%20expression&author=BE%20Stranger&author=AC%20Nica&author=MS%20Forrest&author=A%20Dimas&author=CP%20Bird&volume=39&publication_year=2007&pages=1217-1224&pmid=17873874&doi=10.1038/ng2142&)

37. Miller RD, Duan S, Lovins EG, Kloss EF, Kwok PY. Efficient high-throughput resequencing of genomic DNA. Genome Res. 2003;13:717–720. doi: 10.1101/gr.886203.  [DOI](https://doi.org/10.1101/gr.886203) | [PMC free article](/articles/PMC430165/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12654721/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res&title=Efficient%20high-throughput%20resequencing%20of%20genomic%20DNA&author=RD%20Miller&author=S%20Duan&author=EG%20Lovins&author=EF%20Kloss&author=PY%20Kwok&volume=13&publication_year=2003&pages=717-720&pmid=12654721&doi=10.1101/gr.886203&)

38. Marsh S, King CR, Garsa AA, McLeod HL. Pyrosequencing of clinically relevant polymorphisms. Methods Mol Biol. 2005;311:97–114. doi: 10.1385/1-59259-957-5:097.  [DOI](https://doi.org/10.1385/1-59259-957-5:097) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16100402/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Methods%20Mol%20Biol&title=Pyrosequencing%20of%20clinically%20relevant%20polymorphisms&author=S%20Marsh&author=CR%20King&author=AA%20Garsa&author=HL%20McLeod&volume=311&publication_year=2005&pages=97-114&pmid=16100402&doi=10.1385/1-59259-957-5:097&)

39. King CR, Scott-Horton T. Pyrosequencing: a simple method for accurate genotyping. Methods Mol Biol. 2007;373:39–56. doi: 10.1385/1-59745-377-3:39.  [DOI](https://doi.org/10.1385/1-59745-377-3:39) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17185756/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Methods%20Mol%20Biol&title=Pyrosequencing:%20a%20simple%20method%20for%20accurate%20genotyping&author=CR%20King&author=T%20Scott-Horton&volume=373&publication_year=2007&pages=39-56&pmid=17185756&doi=10.1385/1-59745-377-3:39&)

40. Ronaghi M, Uhlen M, Nyren P. A sequencing method based on real-time pyrophosphate. Science. 1998;281:363. doi: 10.1126/science.281.5375.363. 5.  [DOI](https://doi.org/10.1126/science.281.5375.363) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9705713/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=A%20sequencing%20method%20based%20on%20real-time%20pyrophosphate&author=M%20Ronaghi&author=M%20Uhlen&author=P%20Nyren&volume=281&publication_year=1998&pages=363&pmid=9705713&doi=10.1126/science.281.5375.363&)

41. Benjamini Y, Hochberg Y. Controlling the false discovery rate: a practical and powerful approach to multiple testing. J R Stat Soc B. 1995;57:289–300.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20R%20Stat%20Soc%20B&title=Controlling%20the%20false%20discovery%20rate:%20a%20practical%20and%20powerful%20approach%20to%20multiple%20testing&author=Y%20Benjamini&author=Y%20Hochberg&volume=57&publication_year=1995&pages=289-300&)
