# Single-nucleotide polymorphism in the human mu opioid receptor gene alters β-endorphin binding and activity: Possible implications for opiate addiction

## Metadata
**Authors:** Cherie Bond, K Steven LaForge, Mingting Tian, Dorothy Melia, Shengwen Zhang, Lisa Borg, Jianhua Gong, James Schluger, Judith A Strong, Suzanne M Leal, Jay A Tischfield, Mary Jeanne Kreek, Lei Yu
**Journal:** Proceedings of the National Academy of Sciences of the United States of America
**Date:** 1998 Aug 4
**DOI:** [10.1073/pnas.95.16.9608](https://doi.org/10.1073/pnas.95.16.9608)
**PMID:** 9689128
**PMCID:** PMC21386
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC21386/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC21386/pdf/pq009608.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC21386/pdf/pq009608.pdf)

## Abstract

Opioid drugs play important roles in the clinical management of pain, as well as in the development and treatment of drug abuse. The mu opioid receptor is the primary site of action for the most commonly used opioids, including morphine, heroin, fentanyl, and methadone. By sequencing DNA from 113 former heroin addicts in methadone maintenance and 39 individuals with no history of drug or alcohol abuse or dependence, we have identified five different single-nucleotide polymorphisms (SNPs) in the coding region of the mu opioid receptor gene. The most prevalent SNP is a nucleotide substitution at position 118 (A118G), predicting an amino acid change at a putative N-glycosylation site. This SNP displays an allelic frequency of approximately 10% in our study population. Significant differences in allele distribution were observed among ethnic groups studied. The variant receptor resulting from the A118G SNP did not show altered binding affinities for most opioid peptides and alkaloids tested. However, the A118G variant receptor binds β-endorphin, an endogenous opioid that activates the mu opioid receptor, approximately three times more tightly than the most common allelic form of the receptor. Furthermore, β-endorphin is approximately three times more potent at the A118G variant receptor than at the most common allelic form in agonist-induced activation of G protein-coupled potassium channels. These results show that SNPs in the mu opioid receptor gene can alter binding and signal transduction in the resulting receptor and may have implications for normal physiology, therapeutics, and vulnerability to develop or protection from diverse diseases including the addictive diseases.

## MATERIALS AND METHODS

#### Study Subjects and Procedures.

Addictive disease patients, who were long-term former heroin addicts currently in chronic methadone maintenance treatment, and also normal control subjects with no history of any drug or alcohol abuse, were included in this study. All study subjects were characterized extensively with respect to drug abuse, the addictive diseases, psychological and psychiatric profiles, and medical and ethnic family backgrounds. No study subjects were related.

Previously heroin-addicted patients admitted to the study conformed to the federally regulated criteria for admission to a methadone or l-alpha-acetyl-methadone (LAAM) maintenance program. These criteria are: (*i*) 1 or more years of daily multiple-dose self-administration of heroin or other opiates, (*ii*) the development of tolerance and dependence, and (*iii*) drug-seeking behavior ([18](#B18)). All opioid-dependent study subjects were former heroin addicts currently in treatment at methadone maintenance clinics in New York City, primarily, two clinics closely associated for more than 25 years with the Laboratory of the Biology of Addictive Diseases at The Rockefeller University, the Adolescent Development Program and Adult Clinic at the New York Hospital–Cornell Medical Center. Current or prior abuse of other drugs was not used as an exclusion criterion for this study group as long as long-term opiate addiction continued to be the primary diagnosis.

Healthy control subjects were recruited primarily through posting of notices and newspaper advertisements or by referral as possible research volunteers by physicians or staff at The Rockefeller University Hospital. Individuals with any ongoing drug or alcohol abuse, or prior periods of alcohol or drug abuse, were excluded from this category. The exclusion criteria were defined as follows: for current or continuing abuse of alcohol, at least one instance of drinking to intoxication during the previous 30 days; for opiates, cocaine, amphetamines, or other illicit drugs (excluding cannabis), any use during the previous 30 days. Users of nicotine or caffeine were not excluded, nor were individuals who had abused cannabis for up to 12 days during the previous 30 days. For prior abuse, subjects were excluded who had used illicit drugs (with the exception of cannabis) for at least three times a week for a period of at least 1 month at any point in their lifetime.

All study subjects were screened rigorously by specially trained research personnel, including psychiatrists and research nurses, to ensure appropriate characterization of addictive diseases, status of treatment, and presence or absence of any polydrug or alcohol abuse. Both addictive disease patients and normal volunteers admitted to the study were assessed by a psychiatrist or research nurse with several general and specific psychiatric and psychological instruments. The Addiction Severity Index was included, with previously established interobserver reliability for the trained clinical research staff involved in this study ([19](#B19)). Study subjects also were administered a detailed personal, medical, and specific addictive disease questionnaire. Urine specimens were obtained on all study subjects, both those in the addictive disease group (in whom regular, ongoing urine collections are made and analyzed both for clinical care and other research purposes), and in the healthy, normal volunteer group. Urine specimens were analyzed for multiple drugs of abuse. Although not utilized in the present study, a family medical history and addictive disease questionnaire, designed to provide information regarding any substance abuse and major mental illnesses of first- and second-degree relatives, was administered.

Study subjects provided detailed information regarding family origin and ethnic background, including country or geographic area of birth. This information was obtained for both the study subjects and their immediate ancestors (parents, grandparents, and great-grandparents) to the extent that the information was known by the study subjects. Study subjects were classified into five groups: African-American, Caucasian, Hispanic (Caribbean and Central or South American origin), Native North American, and Other.

Subjects entering the study were required to be competent to understand the study procedures and understand and sign the Institutional Review Board-approved informed consent. Patients with schizophrenia or other psychotic mental illnesses were excluded from the study by this criterion. The presence of any known serological markers for hepatitis B or C or HIV was not used as an exclusion criterion.

After the informed consent, psychiatric and behavioral assessment, and family history acquisition, venipuncture on each study subject was performed and a blood specimen was taken. Blood samples were processed for DNA extraction both at The Rockefeller University and at Indiana University. Epstein–Barr virus transformation was used to create stable cell lines that were stored at both institutions for future studies. All blood samples were coded for continued maintenance of full confidentiality. The psychiatrists and nurses who performed psychiatric and psychological assessments were blind to the ultimately determined genotypes of the study subjects. Similarly, the identity and categorization of the study subjects was and remained unknown to the laboratory research personnel.

#### Exon Amplification and Sequencing.

Sequences from the noncoding regions of the human mu opioid receptor gene were used to design PCR primers for use in the amplification of the coding regions. PCR primers were synthesized for three of the four exons of the gene; the fourth exon was not included in this study because this exon is small (4 or 12 aa residues) and alternative splicing in this exon has been shown to occur ([20](#B20)). Exon 1 forward primer sequences were based on the 5′ untranslated region of the receptor ([11](#B11)). Exon 1 reverse, exon 2 forward and reverse, and exon 3 forward primer sequences were based on partial intron sequence data obtained from inverse PCR of genomic DNA sequences for the receptor gene (data not shown). Exon 3 reverse primers were based on reported intron 3 sequence ([20](#B20)). Two sets of primers were designed for each exon to allow for nested PCRs to increase amplification specificity of the exonic regions. Only one reverse primer was used for exon 1. The PCRs were performed with 300–400 ng of genomic DNA, PCR products were separated on agarose gels, and the DNA fragments were purified for DNA sequencing. DNA polymorphisms were confirmed by both manual and automated sequencing.

#### Mutagenesis.

*In vitro* site-directed mutagenesis was performed to generate human mu opioid receptor (hMOR) cDNA containing one of two major variants found, the A118G SNP. Complementary oligonucleotides containing the desired mutation were synthesized and annealed to the pcDNA3 plasmid containing the most common allelic form of hMOR. Primer 1, TTGTCCCACTTAGATGGCGACCTGTCCGACCCA; and primer 2, ACCGCATGGGTCGGACAGGTCGCCATCTAAGTG. Primers were extended and the product was amplified by PCR using hMOR dsDNA as the template, and *Dpn*I restriction enzyme was added afterward to digest the methylated, nonmutated, most common dsDNA. After transformation into *Escherichia coli* cells, DNA from individual colonies was examined by restriction enzyme digestion and DNA sequencing to confirm success of mutagenesis.

#### Cell Transfection and Binding Analysis.

Stable transfection of the A118G SNP plasmid into AV-12 cells was performed ([21](#B21)). Individual colonies were picked, expanded, and tested for expression levels by performing binding assays. Cells were harvested by washing with PBS at room temperature, scraped into homogenization solution (0.3 M sucrose/25 mM Tris⋅HCl, pH 7.4/0.05% BSA and protease inhibitor cocktail, including 0.5 mM phenylmethylsulfonyl fluoride/0.1 μg/ml leupeptin/0.01% aprotinin), transferred to Dounce homogenizer, and homogenized on ice. The suspension was centrifuged at 1,000 × *g* for 10 min, and the supernatant was saved in a clean tube. The cell pellet was resuspended in homogenization buffer, homogenized, and centrifuged as described above. The supernatants from both extractions were combined and centrifuged at 30,000 × *g* for 20 min. The pelleted membranes were resuspended in binding buffer (50 mM Tris⋅HCl, pH 7.4); binding assays were carried out using membrane protein preparations as described ([11](#B11)). It has been shown that N-linked glycosylation is carried out by AV-12 cells ([22](#B22), [23](#B23)).

#### Electrophysiology.

Preparation of *Xenopus* oocytes was as reported previously ([11](#B11)). Oocytes were injected with *in vitro* transcribed mRNAs for the most common or A118G variant mu opioid receptors together with the G protein-activated, inwardly rectifying K^+^ channels (GIRK1 and GIRK2). Two to 3 days after RNA injection, oocytes were voltage-clamped in ND96 solution (96 mM NaCl/2 mM KCl/1 mM MgCl_2_/1.8 mM CaCl_2_/5 mM Hepes, pH 7.6) by using a two-electrode voltage clamp (Axon Instruments). Cells then were superfused with a high potassium solution (98 mM KCl/1 mM MgCl_2_/1.8 mM CaCl_2_/5 mM Hepes, pH 7.6) and stimulated with opioid ligands to measure the resulting potassium current. *Xenopus* oocytes have been shown to carry out N-linked glycosylation of exogenously expressed proteins ([24](#B24), [25](#B25)).

## RESULTS AND DISCUSSION

#### Polymorphisms in the Human mu Opioid Receptor Gene.

Molecular cloning of the mu opioid receptor ([8](#B8)–[11](#B11)) has made it possible to determine potential sequence polymorphism. To identify SNPs in the mu opioid receptor, we used a PCR-based strategy to amplify the coding regions of the mu opioid receptor gene and to determine the DNA sequence of the amplified exonic regions. Using this method, we sequenced DNA samples from 152 subjects, including both former heroin addicts in methadone maintenance treatment and individuals with no history of drug or alcohol dependence.

Inclusion criteria as defined in the *Materials and Methods* section were met by these 152 individuals. One hundred and thirteen of the study subjects (74.3%) were former heroin addicts in methadone maintenance treatment, with or without previous or current codependency for other substances; 39 study subjects (25.7%) had no history of drug or alcohol dependence or any ongoing illicit opiate or other drug use, or alcohol abuse. Study subjects included 69 females (45.4%) and 83 males (56.4%).

Within the group of former heroin addicts in methadone maintenance treatment, the mean years in treatment was 6.7 with a range from 2 months to 30 years (*n* = 112, only one patient’s history could not be verified by an additional interview or by medical chart records). Before treatment, the mean years of heroin addiction was 10.1 years, with a range from 1 to 30 years (*n* = 109; the exact duration of heroin use before entry into methadone treatment could not be verified for four subjects). The mean daily methadone dose of opioid dependent patients in stable treatment was 84 mg/day, with a range from 30 to 120 mg/day. Only patients (*n* = 106) with established stable methadone doses were included in this calculation; the other patients were undergoing induction, increasing, tapering, or elimination dose schedules.

The ethnic breakdown of the study subject populations was as follows: African-American, 31 (20.3%), Caucasian, 52 (34.2%), Hispanic, 67 (44.1%), Native North American, 1 (0.7%), and Other, 1 (0.7%). Several individuals reported that one parent’s ancestry was from one ethnic group and the other parent’s ancestry was from a second ethnic group, including four individuals (2.6%) who reported one African-American parent and one Caucasian parent, and five individuals (3.3%) who reported one Caucasian parent and one Hispanic. For the genotype calculations, the former study subjects were classified as African-American and the latter, as Hispanic.

By sequencing PCR-amplified DNA from the study subjects, we determined that the previously reported sequence for the human mu opioid receptor ([10](#B10), [11](#B11)) was the most common allele found in our study population. We also identified five different variants, all SNPs. For the purpose of this report, we use the term “most common” to denote the predominant mu opioid receptor allele and the corresponding receptor that was reported originally by cDNA cloning ([10](#B10), [11](#B11)), and the term “variant” to denote the allelic genes or the resulting receptors containing polymorphic variations. Table [1](#T1) shows these SNPs, with information on the position of amino acid substitutions, and overall frequency of the variant alleles in the study population. Two recent studies ([12](#B12), [13](#B13)) also have identified one or both of these five SNPs in the coding region (A118G and C17T), but not the other three SNPs (G24A, G779A, and G942A).

### Table 1.

| Variant name | Nucleotide position | Exon location | Corresponding amino acid change | Protein domain | Allele frequency |
| --- | --- | --- | --- | --- | --- |
| A118G | 118 | 1 | Asn-40 → Asp (N40D) | N terminal | 10.5% (26 heterozygous individual and 3 homozygous individuals, n = 152) |
| C17T | 17 | 1 | Ala-6 → Val (A6V) | N terminal | 6.6% (14 heterozygous individuals and 3 homozygous individuals, n = 152) |
| G24A | 24 | 1 | Silent mutation | N terminal | 2% (six heterozygous individuals, n = 152) |
| G779A | 779 | 3 | Arg-260 → His (R260H) | CL3 | One heterozygous individual |
| G942A | 942 | 3 | Silent mutation | EL3 | One heterozygous individual |

Table 1 Caption: SNPs in the human mu opioid receptor gene

Genotype and allele frequencies for the two most prevalent allelic variants in our population sample, the A118G and C17T polymorphisms, are shown in Table [2](#T2), with overall allelic frequencies of 10.5 and 6.6%, respectively. One of the two earlier reports, comparing healthy volunteers with cocaine- and/or opiate-dependent individuals, found an allelic frequency of 10 and 22% for the C17T receptor SNP in their two study groups, respectively ([12](#B12)). This study included Caucasian and African-American subjects. The A118G SNP was not identified ([12](#B12)). The second previously reported study included American Caucasian, Finnish Caucasian, and Native North American study subjects; an allelic frequency of 10.5–16.3% for the A118G variant among the three groups was found, but with only “rare” occurrence of the C17T polymorphism ([13](#B13)). Table [2](#T2) gives associations of each frequency in our study population broken down by ethnicity, gender, and opioid dependence. Because the number of individuals homozygous for the variant alleles was small, we used allele frequencies, rather than genotype frequencies, to test for significant differences.

### Table 2.

|   | A118G Genotypes |  |  | Total | A118G Alleles |  | Total | C17T Genotypes |  |  | Total | C17T Alleles |  | Total |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
|   | A/A | A/G | G/G | A | G | C/C | C/T | T/T | C | T |  |  |  |  |
| Ethnicity* |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
| African-American | 30 (0.968) | 1 (0.032) | 0 — | 31 | 61 (0.984) | 1 (0.016) | 62 | 21 (0.677) | 7 (0.226) | 3 (0.097) | 31 | 49 (0.790) | 13 (0.210) | 62 |
| Caucasian | 41 (0.788) | 10 (0.192) | 1 (0.019) | 52 | 92 (0.885) | 12 (0.115) | 104 | 50 (0.962) | 2 (0.038) | 0 — | 52 | 102 (0.981) | 2 (0.019) | 104 |
| Hispanic | 50 (0.746) | 15 (0.225) | 2 (0.030) | 67 | 115 (0.858) | 19 (0.142) | 134 | 62 (0.925) | 5 (0.075) | 0 — | 67 | 129 (0.963) | 5 (0.037) | 134 |
|   |   |   |   |   | χ(2)2=7.15 (P=0.028) |  |  |   |   |   |   | χ(2)2=26.0 (P=0.000002) |  |  |
| Gender |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
| Female | 59 (0.855) | 8 (0.116) | 2 (0.029) | 69 | 126 (0.913) | 12 (0.087) | 138 | 58 (0.841) | 9 (0.130) | 2 (0.029) | 69 | 125 (0.906) | 13 (0.094) | 138 |
| Male | 64 (0.771) | 18 (0.217) | 1 (0.012) | 83 | 146 (0.880) | 20 (0.120) | 166 | 77 (0.928) | 5 (0.060) | 1 (0.012) | 83 | 159 (0.958) | 7 (0.042) | 166 |
|   |   |   |   | χ(1)2 = 0.90 (P = 0.343) |  |  |  |  | Yate’s corrected χ(1)2 = 2.53 (P = 0.112) |  |  |  |  |  |
| Opioid Dependence |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
| Dependent | 94 (0.832) | 18 (0.159) | 1 (0.009) | 113 | 206 (0.912) | 20 (0.088) | 226 | 97 (0.858) | 13 (0.115) | 3 (0.027) | 113 | 207 (0.916) | 19 (0.084) | 226 |
| Nondependent | 29 (0.744) | 8 (0.205) | 2 (0.051) | 39 | 66 (0.846) | 12 (0.154) | 78 | 38 (0.974) | 1 (0.026) | 0 — | 39 | 77 (0.987) | 1 (0.013) | 78 |
|   |   |   | Yale’s corrected χ(1)2 = 1.98 (P = 0.159) |  |  |  |  |  | Yate’s corrected χ(1)2 = 3.70 (P = 0.054) |  |  |  |  |  |

Table 2 Caption: Genotype and allele frequency associations

We tested for differences of allele frequencies among the three most common ethnic groups, in our study, African-American, Caucasian, and Hispanic, irrespective of opioid-dependency status. We found significant differences of allele frequencies among the ethnic groups studied for both the A118G [χ_(2)_^2^ = 7.15 (*P* = 0.028)] and the C17T [χ_(2)_^2^ = 26.0 (*P* = 0.000002)]. If the individuals who reported one parent from one ethnic group and one from another ethnic group were excluded from this analysis, similar significance levels were obtained for differences of both SNPs among ethnic groups. This result is not surprising because allele frequencies are known to vary among ethnic groups. However, it is important to consider these differences, which can confound association analyses. No significant association of gender with either polymorphism was observed.

For the A118G polymorphism, there was no significant difference in allele frequencies between opioid-dependent and nondependent study subjects with all ethnic groups combined. Similarly, no association between this allele and alcoholism or drug abuse of unspecified type was found in the study reported by Bergen *et al.* ([13](#B13)). However, within the Hispanic study subject group, we found that the A118G variant allele was present in a significantly higher proportion of non-opioid-dependent subjects compared with the opioid-dependent subjects [Yates-corrected Chi-square χ_(1)_^2^ = 8.22 (*P* = 0.0041)]. Although this finding could be explained by population admixture within this group, it suggests the possibility that the A118G SNP might confer a relative protection against opioid dependence. Further studies with a larger sample size would be necessary to test this hypothesis.

In contrast to the A118G allele, the C17T variant was present in a higher overall proportion of opioid-dependent persons in our sample at a marginal significance level [Yates-corrected Chi-square χ_(1)_^2^ = 3.70 (*P* = 0.054)]. This result is similar to that obtained by Berrettini *et al.* ([12](#B12)).

Table [3](#T3) shows data of this study stratified by ethnic group and opioid-dependency status for both the A118G and the C17T SNPs. The pooled Relative Risk (RR) and the Mantel–Haenszel Chi-square ([26](#B26)) were calculated. For the A118G polymorphism there was no significant difference in allele frequencies between former heroin addicts in treatment (cases) and those with no history of drug or alcohol abuse or dependence (controls) [RR = 0.48 χ_(1)_^2^ = 2.76 (*P* = 0.096)]. Although not significant, with respect to the Relative Risk there was evidence of heterogeneity among ethnic groups [χ_(2)_^2^ = 5.16 (*P* = 0.076)]. It should be noted that the direction of the Relative Risk, i.e., less than one, shows here that the A118G polymorphism was more frequent in control subjects than in opioid-dependent subjects and, again, suggests that the A118G polymorphism might confer some level of protection against opiate addiction, which is of particular interest given the differences in receptor activity (see below). There was a marginally significant difference in the allele frequencies for the C17T polymorphism between opioid-dependent and healthy control subjects [RR = 7.83 χ_(1)_^2^ = 3.73 (*P* = 0.05)]. The test for heterogeneity of the Relative Risk among ethnic groups for the C17T polymorphism was not significant [χ_(2)_^2^ = 3.95 (*P* = 0.14)].

### Table 3.

|   | A118G |  |  |  | C17T |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
|   | Opioid-dependent (cases) |  | Nondependent (controls) |  | Opioid-dependent (cases) |  | Nondependent (controls) |  |
|   | A | G | A | G | C | T | C | T |
| African-American | 45 | 1 | 16 | 0 | 33 | 13 | 16 | 0 |
| Caucasian | 53 | 7 | 39 | 5 | 59 | 1 | 43 | 1 |
| Hispanic | 104 | 12 | 11 | 7 | 111 | 5 | 18 | 0 |
| Combined | 202 | 20 | 66 | 12 | 203 | 19 | 77 | 1 |

Table 3 Caption: Stratification of alleles of opioid-dependent and nondependent study subjects by ethnicity

#### The A118G SNP Affects β-Endorphin-Binding Affinity of the mu Opioid Receptor.

The most prevalent genetic polymorphism we identified is the A118G SNP with a substitution at the nucleotide position 118 with respect to the first base of the initiator codon for methionine (Fig. [1](#F1)). This allele was observed in 29 of the 152 subjects, with 26 subjects being heterozygous and 3 being homozygous for the variant allele. Overall, this gives an allele frequency of 10.5% in the subject population that we have examined in this study. Nucleotide 118 is the first base in codon 40 of the human mu opioid receptor, and the A118G variant predicts an Asn-to-Asp change in amino acid residue 40 of the receptor (N40D). The Asn residue at amino acid position 40 in the most common sequence of the mu receptor is a putative site for N-glycosylation ([11](#B11)). Thus, the A118G variant would result in the loss of a putative N-glycosylation site. The position of amino acid 40 is in the N-terminal region of the mu opioid receptor ([11](#B11)). Based on sequence motif similarities with other G protein-coupled receptors ([27](#B27)), the N-terminal region of opioid receptors, including that of the mu opioid receptor, is predicted to be in the extracellular space ([28](#B28)). To explore any potential effects of the A118G polymorphism on the mu opioid receptor, we mutated the position 118 of the most common mu receptor cDNA by site-directed mutagenesis and generated a cDNA clone for the human mu opioid receptor containing the A118G variant. This way, both the most common and the A118G variant receptors could be expressed in cells to determine their cellular activity and their binding affinities.

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/aa2c/21386/dd3573c22981/pq1682230001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=21386_pq1682230001.jpg)

DNA sequences showing the A118G polymorphism. Examples of DNA sequence are shown from individuals homozygous for the most common allele (Left), heterozygous (Center), and homozygous for the A118G variant allele (Right). The arrows indicate the position of nucleotide 118, with the adjacent sequences shown.

We performed radioligand-binding assays with cell lines stably transfected with either the A118G variant or the most common mu receptor to determine whether the A118G polymorphism changes the receptor’s ability to bind opioid ligands, especially endogenous opioid peptides, because they are the physiological agonists for the mu opioid receptor. The A118G variant and the most common mu receptors yielded similar binding affinity values for most of the opioid ligands tested, including the small endogenous peptide agonists Met- and Leu-enkephalin, each with five aa residues; endomorphin-1 and -2, each with four residues; the mu-selective synthetic opioid peptide DAMGO, with five amino acid residues; the endogenous ligand for the kappa opioid receptor dynorphin A ([1](#B1)–[17](#B17)); as well as the mu-preferring opioid alkaloid agonists morphine, fentanyl, methadone, and the opioid antagonist naloxone (Fig. [2](#F2), and data not shown). These results suggest that the A118G polymorphism does not change the overall binding properties of the mu opioid receptor. This is not unexpected, because the predicted amino acid change as a result of the A118G SNP is a single residue substitution in the N-terminal region in the extracellular space and is unlikely to drastically affect the overall tertiary structure of the receptor.

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/aa2c/21386/ce0843a2ad77/pq1682230002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=21386_pq1682230002.jpg)

Binding of endogenous opioid peptides to the most common and A118G mu opioid receptors. Membrane preparations from cells expressing either the most common (■) or the A118G (▵) receptors were used in binding experiments to displace the [3H]DAMGO binding. Shown are examples of displacement binding for four endogenous peptides: Met-enkephalin, dynorphin A, β-endorphin, and endomorphin-1.

There was a substantial change for the A118G variant receptor binding of human β-endorphin, a much larger endogenous opioid peptide, which has 31 aa residues and which activates the mu opioid receptor. Whereas the other, smaller, endogenous opioid peptides and alkaloid agonists and antagonist displayed similar binding affinities for both receptors, the A118G variant receptor showed higher binding affinity for β-endorphin than the most common receptor (Fig. [2](#F2)), with the ratio of the *K*_i_ of the most common allele to A118G variant SNP being 3.46 ± 0.31 (mean ± SEM, *n* = 3). These results indicate that although the A118G polymorphism did not alter the overall profile of ligand binding to the receptor, it specifically influenced the β-endorphin binding and resulted in much higher affinity for this large, endogenous opioid peptide.

#### The A118G SNP Changes the Agonist Potency of β-Endorphin for Activating the mu Opioid Receptor.

An important cellular activity of the mu opioid receptor is inhibition of neuronal excitability by receptor-mediated inhibition of presynaptic calcium channels and activation of postsynaptic potassium channels ([29](#B29), [30](#B30)). The major effector potassium channels for the mu receptor, as well as for many other G protein-coupled receptors, are the G protein-activated, inwardly rectifying K^+^ (GIRK) channels ([31](#B31), [32](#B32)). Coexpression studies have shown that the mu opioid receptor can readily activate GIRK channels via a G protein-mediated mechanism ([11](#B11), [33](#B33), [34](#B34)). To examine the effect of the A118G polymorphism, we used the *Xenopus* oocyte expression to compare the A118G variant receptor with the most common mu opioid receptor. Agonist stimulation of the A118G variant receptor activated a potassium current similar to that seen with the most common mu opioid receptor ([11](#B11), [33](#B33)). The EC_50_ values for endomorphin-1 are 4.6 nM for the most common receptor and 4.9 nM for the A118G variant receptor (Fig. [3](#F3)), indicating that endomorphin-1 activated both receptors with similar potency. The EC_50_ values for β-endorphin, however, differed about 3-fold between the A118G variant and the most common mu opioid receptors (Fig. [3](#F3)), consistent with the change in the binding affinity (Fig. [2](#F2)). These data indicate that, as a result of the SNP in the receptor gene, the A118G variant receptor may be functionally different from the most common mu opioid receptor.

### Figure 3.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/aa2c/21386/829227edc7ca/pq1682230003.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=21386_pq1682230003.jpg)

Comparison of the most common and the A118G variant mu opioid receptors in coupling to G protein-activated, inwardly rectifying K+ (GIRK) channels. (A) Example of current trace showing the experimental protocol and calculation method for the agonist-induced response. Oocytes were clamped at a holding potential of −80 mV and superfused with different solutions as indicated. Imax, maximum K+ currents evoked by DAMGO at a saturating concentration (100 nM). ITest, K+ currents evoked by the test dose of agonists. The trace example shows the response of the A118G variant receptor to 20 nM β-endorphin. (B) Dose-response curves of receptor activation. The tested concentrations of agonists ranged from 0.1 nM to 1 μM. The response to a test dose is expressed as the fraction of the maximum activation by 100 nM DAMGO (ITest/Imax). Data are presented as mean ± SEM (n = 4–5). All oocytes were used only once to avoid desensitization.

An endogenous opioid with wide distribution in both the central nervous system (CNS) and the periphery, β-endorphin has been postulated to play a role in diverse biological functions ([35](#B35), [36](#B36)). As a neuropeptide, it can modulate neurotransmitter actions in the CNS to mediate antinociception. It is also a mediator in the stress response, of potential importance for the pathophysiology of the addictive diseases ([37](#B37)–[42](#B42)). β-endorphin can regulate the secretion of both stress and reproductive hormones, thereby influencing a variety of physiological functions. Given the diverse roles of β-endorphin, it is particularly interesting that the A118G polymorphism may change the receptor with respect to the binding affinity of β-endorphin and the potency of its cellular activity. The approximately 3-fold difference in binding affinity and potency of β-endorphin (Figs. [2](#F2) and [3](#F3)) suggests that individuals carrying the variant receptor gene (A118G) may show differences in some of the functions mediated by β-endorphin action at the altered mu opioid receptors. Thus, for example, response to stress, reproductive function, and pain perception could be altered. Moreover, the data suggest that subtle changes, such as the SNP studied with respect to its binding and activity, could have significant effects on the susceptibility or vulnerability to develop multifactorial diseases such as specific addictions ([7](#B7), [8](#B8), [43](#B43)).

## Acknowledgments

We thank W. Ai, L. S. Bye, J. Chen, A. Ho, Z. Lai, J. Ott, G. Perret, M. R. Vasko, L. Walsh, P. Young, and X. Yue for technical assistance, comments, and suggestions. We also thank Drs. E. Khuri and A. Wells, directors of the clinics that participated in the study. This research was supported in part by the National Institutes of Health Research Grants DA09444 (L.Y. and M.J.K.), DA05130 and DA00049 (M.J.K.), HG00008 (S.M.L.), and DA09116 and DA11891 (L.Y.), and by a General Clinical Research Center Grant (M01-RR00102) from the National Center for Research Resources at the National Institutes of Health.

## ABBREVIATION

## References

1. Zadina J E, Hackler L, Ge L J, Kastin A J. Nature (London) 1997;386:499–502. doi: 10.1038/386499a0.  [DOI](https://doi.org/10.1038/386499a0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9087409/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature%20(London)&author=J%20E%20Zadina&author=L%20Hackler&author=L%20J%20Ge&author=A%20J%20Kastin&volume=386&publication_year=1997&pages=499-502&pmid=9087409&doi=10.1038/386499a0&)

2. Basbaum A I, Fields H L. Annu Rev Neurosci. 1984;7:309–338. doi: 10.1146/annurev.ne.07.030184.001521.  [DOI](https://doi.org/10.1146/annurev.ne.07.030184.001521) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/6143527/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annu%20Rev%20Neurosci&author=A%20I%20Basbaum&author=H%20L%20Fields&volume=7&publication_year=1984&pages=309-338&pmid=6143527&doi=10.1146/annurev.ne.07.030184.001521&)

3. Pasternak G W. Clin Neuropharmacol. 1993;16:1–18. doi: 10.1097/00002826-199302000-00001.  [DOI](https://doi.org/10.1097/00002826-199302000-00001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8093680/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Neuropharmacol&author=G%20W%20Pasternak&volume=16&publication_year=1993&pages=1-18&pmid=8093680&doi=10.1097/00002826-199302000-00001&)

4. Pasternak G W. The Opiate Receptors. Totowa, NJ: Humana; 1988.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=The%20Opiate%20Receptors&author=G%20W%20Pasternak&publication_year=1988&)

5. Goldstein A. Trends Pharmacol Sci. 1987;8:456–459.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Trends%20Pharmacol%20Sci&author=A%20Goldstein&volume=8&publication_year=1987&pages=456-459&)

6. Kreek M J. Mol Psychiatry. 1996;1:232–254.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/9118348/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Psychiatry&author=M%20J%20Kreek&volume=1&publication_year=1996&pages=232-254&pmid=9118348&)

7. Kreek M J. Neurochem Res. 1996;21:1469–1488. doi: 10.1007/BF02532387.  [DOI](https://doi.org/10.1007/BF02532387) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8947936/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neurochem%20Res&author=M%20J%20Kreek&volume=21&publication_year=1996&pages=1469-1488&pmid=8947936&doi=10.1007/BF02532387&)

8. Chen Y, Mestek A, Liu J, Hurley J A, Yu L. Mol Pharmacol. 1993;44:8–12.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/8393525/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Pharmacol&author=Y%20Chen&author=A%20Mestek&author=J%20Liu&author=J%20A%20Hurley&author=L%20Yu&volume=44&publication_year=1993&pages=8-12&pmid=8393525&)

9. Wang J B, Imai Y, Eppler C M, Gregor P, Spivak C E, Uhl G R. Proc Natl Acad Sci USA. 1993;90:10230–10234. doi: 10.1073/pnas.90.21.10230.  [DOI](https://doi.org/10.1073/pnas.90.21.10230) | [PMC free article](/articles/PMC47748/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8234282/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20USA&author=J%20B%20Wang&author=Y%20Imai&author=C%20M%20Eppler&author=P%20Gregor&author=C%20E%20Spivak&volume=90&publication_year=1993&pages=10230-10234&pmid=8234282&doi=10.1073/pnas.90.21.10230&)

10. Wang J B, Johnson P S, Persico A M, Hawkins A L, Griffin C A, Uhl G R. FEBS Lett. 1994;338:217–222. doi: 10.1016/0014-5793(94)80368-4.  [DOI](https://doi.org/10.1016/0014-5793(94)80368-4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7905839/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=FEBS%20Lett&author=J%20B%20Wang&author=P%20S%20Johnson&author=A%20M%20Persico&author=A%20L%20Hawkins&author=C%20A%20Griffin&volume=338&publication_year=1994&pages=217-222&pmid=7905839&doi=10.1016/0014-5793(94)80368-4&)

11. Mestek A, Hurley J H, Bye L S, Campbell A D, Chen Y, Tian M, Schulman H, Yu L. J Neurosci. 1995;15:2396–2406. doi: 10.1523/JNEUROSCI.15-03-02396.1995.  [DOI](https://doi.org/10.1523/JNEUROSCI.15-03-02396.1995) | [PMC free article](/articles/PMC6578163/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7891175/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Neurosci&author=A%20Mestek&author=J%20H%20Hurley&author=L%20S%20Bye&author=A%20D%20Campbell&author=Y%20Chen&volume=15&publication_year=1995&pages=2396-2406&pmid=7891175&doi=10.1523/JNEUROSCI.15-03-02396.1995&)

12. Berrettini W H, Hoehe M R, Ferrada T N, Gottheil E. Addiction Biol. 1997;2:303–308. doi: 10.1080/13556219772598.  [DOI](https://doi.org/10.1080/13556219772598) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26735786/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Addiction%20Biol&author=W%20H%20Berrettini&author=M%20R%20Hoehe&author=T%20N%20Ferrada&author=E%20Gottheil&volume=2&publication_year=1997&pages=303-308&pmid=26735786&doi=10.1080/13556219772598&)

13. Bergen A W, Kokoszka J, Peterson R, Long J C, Virkkunen M, Linnoila M, Goldman D. Mol Psychiatry. 1997;2:490–494. doi: 10.1038/sj.mp.4000331.  [DOI](https://doi.org/10.1038/sj.mp.4000331) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9399694/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Psychiatry&author=A%20W%20Bergen&author=J%20Kokoszka&author=R%20Peterson&author=J%20C%20Long&author=M%20Virkkunen&volume=2&publication_year=1997&pages=490-494&pmid=9399694&doi=10.1038/sj.mp.4000331&)

14. Kieffer B L. Cell Mol Neurobiol. 1995;15:615–635. doi: 10.1007/BF02071128.  [DOI](https://doi.org/10.1007/BF02071128) | [PMC free article](/articles/PMC11563145/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8719033/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Mol%20Neurobiol&author=B%20L%20Kieffer&volume=15&publication_year=1995&pages=615-635&pmid=8719033&doi=10.1007/BF02071128&)

15. Shenker A, Laue L, Kosugi S, Merendino J J, Jr, Minegishi T, Cutler G B., Jr Nature (London) 1993;365:652–654. doi: 10.1038/365652a0.  [DOI](https://doi.org/10.1038/365652a0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7692306/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature%20(London)&author=A%20Shenker&author=L%20Laue&author=S%20Kosugi&author=J%20J%20Merendino&author=T%20Minegishi&volume=365&publication_year=1993&pages=652-654&pmid=7692306&doi=10.1038/365652a0&)

16. Parma J, Duprez L, Van Sande J, Cochaux P, Gervy C, Mockel J, Dumont J, Vassart G. Nature (London) 1993;365:649–651. doi: 10.1038/365649a0.  [DOI](https://doi.org/10.1038/365649a0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8413627/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature%20(London)&author=J%20Parma&author=L%20Duprez&author=J%20Van%20Sande&author=P%20Cochaux&author=C%20Gervy&volume=365&publication_year=1993&pages=649-651&pmid=8413627&doi=10.1038/365649a0&)

17. Rosenthal W, Antaramian A, Gilbert S, Birnbaumer M. J Biol Chem. 1993;268:13030–13033.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/8514744/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&author=W%20Rosenthal&author=A%20Antaramian&author=S%20Gilbert&author=M%20Birnbaumer&volume=268&publication_year=1993&pages=13030-13033&pmid=8514744&)

18. Rettig R A, Yarmolinsky A. Federal Regulation of Methadone Treatment. Washington, DC: Natl. Acad. Press; 1995.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/25121195/) | [Google Scholar](https://scholar.google.com/scholar_lookup?title=Federal%20Regulation%20of%20Methadone%20Treatment&author=R%20A%20Rettig&author=A%20Yarmolinsky&publication_year=1995&)

19. McLellan A T, Luborsky L, Woody G E, O’Brien C P. J Nerv Ment Dis. 1980;168:26–33. doi: 10.1097/00005053-198001000-00006.  [DOI](https://doi.org/10.1097/00005053-198001000-00006) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7351540/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Nerv%20Ment%20Dis&author=A%20T%20McLellan&author=L%20Luborsky&author=G%20E%20Woody&author=C%20P%20O%E2%80%99Brien&volume=168&publication_year=1980&pages=26-33&pmid=7351540&doi=10.1097/00005053-198001000-00006&)

20. Bare L A, Mansson E, Yang D. FEBS Lett. 1994;354:213–216. doi: 10.1016/0014-5793(94)01129-x.  [DOI](https://doi.org/10.1016/0014-5793(94)01129-x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7957926/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=FEBS%20Lett&author=L%20A%20Bare&author=E%20Mansson&author=D%20Yang&volume=354&publication_year=1994&pages=213-216&pmid=7957926&doi=10.1016/0014-5793(94)01129-x&)

21. Chen Y, Liu J, Yu L. Addiction Biol. 1996;1:49–59. doi: 10.1080/1355621961000124686.  [DOI](https://doi.org/10.1080/1355621961000124686) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12893486/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Addiction%20Biol&author=Y%20Chen&author=J%20Liu&author=L%20Yu&volume=1&publication_year=1996&pages=49-59&pmid=12893486&doi=10.1080/1355621961000124686&)

22. Parkinson J F, Grinnell B W, Hoskins J, Vlahos C J, Bang N U. J Biol Chem. 1990;265:12602–12610.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/2165069/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&author=J%20F%20Parkinson&author=B%20W%20Grinnell&author=J%20Hoskins&author=C%20J%20Vlahos&author=N%20U%20Bang&volume=265&publication_year=1990&pages=12602-12610&pmid=2165069&)

23. Grinnell B W, Walls J D, Gerlitz B. J Biol Chem. 1991;266:9778–9785.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/2033065/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&author=B%20W%20Grinnell&author=J%20D%20Walls&author=B%20Gerlitz&volume=266&publication_year=1991&pages=9778-9785&pmid=2033065&)

24. Hayes G, Busch A, Lotscher M, Waldegger S, Lang F, Verrey F, Biber J, Murer H. J Biol Chem. 1994;269:24143–24149.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/7929070/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&author=G%20Hayes&author=A%20Busch&author=M%20Lotscher&author=S%20Waldegger&author=F%20Lang&volume=269&publication_year=1994&pages=24143-24149&pmid=7929070&)

25. Gurnett C A, De Waard M, Campbell K P. Neuron. 1996;16:431–440. doi: 10.1016/s0896-6273(00)80061-6.  [DOI](https://doi.org/10.1016/s0896-6273(00)80061-6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8789958/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neuron&author=C%20A%20Gurnett&author=M%20De%20Waard&author=K%20P%20Campbell&volume=16&publication_year=1996&pages=431-440&pmid=8789958&doi=10.1016/s0896-6273(00)80061-6&)

26. Mantel N, Haenszel W. J Natl Cancer Inst. 1959;22:719–748.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/13655060/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Natl%20Cancer%20Inst&author=N%20Mantel&author=W%20Haenszel&volume=22&publication_year=1959&pages=719-748&pmid=13655060&)

27. O’Dowd B F, Lefkowitz R J, Caron M G. Annu Rev Neurosci. 1989;12:67–83. doi: 10.1146/annurev.ne.12.030189.000435.  [DOI](https://doi.org/10.1146/annurev.ne.12.030189.000435) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2648960/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annu%20Rev%20Neurosci&author=B%20F%20O%E2%80%99Dowd&author=R%20J%20Lefkowitz&author=M%20G%20Caron&volume=12&publication_year=1989&pages=67-83&pmid=2648960&doi=10.1146/annurev.ne.12.030189.000435&)

28. Chen Y, Mestek A, Liu J, Yu L. Biochem J. 1993;295:625–628. doi: 10.1042/bj2950625.  [DOI](https://doi.org/10.1042/bj2950625) | [PMC free article](/articles/PMC1134603/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8240267/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20J&author=Y%20Chen&author=A%20Mestek&author=J%20Liu&author=L%20Yu&volume=295&publication_year=1993&pages=625-628&pmid=8240267&doi=10.1042/bj2950625&)

29. North R A. In: Handbook of Experimental Pharmacology: Opioids I. Herz A, editor. Vol. 104. Berlin: Springer; 1993. pp. 773–797.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Handbook%20of%20Experimental%20Pharmacology:%20Opioids%20I&author=R%20A%20North&publication_year=1993&)

30. Chavkin C. In: The Opiate Receptors. Pasternak G W, editor. Totowa, NJ: Humana; 1988. pp. 273–303.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=The%20Opiate%20Receptors&author=C%20Chavkin&publication_year=1988&)

31. Kubo Y, Reuveny E, Slesinger P A, Jan Y N, Jan L Y. Nature (London) 1993;364:802–806. doi: 10.1038/364802a0.  [DOI](https://doi.org/10.1038/364802a0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8355805/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature%20(London)&author=Y%20Kubo&author=E%20Reuveny&author=P%20A%20Slesinger&author=Y%20N%20Jan&author=L%20Y%20Jan&volume=364&publication_year=1993&pages=802-806&pmid=8355805&doi=10.1038/364802a0&)

32. Dascal N, Schreibmayer W, Lim N F, Wang W, Chavkin C, DiMagno L, Labarca C, Kieffer B L, Gaveriaux-Ruff C, Trollinger D, et al. Proc Natl Acad Sci USA. 1993;90:10235–10239. doi: 10.1073/pnas.90.21.10235.  [DOI](https://doi.org/10.1073/pnas.90.21.10235) | [PMC free article](/articles/PMC47749/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8234283/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20USA&author=N%20Dascal&author=W%20Schreibmayer&author=N%20F%20Lim&author=W%20Wang&author=C%20Chavkin&volume=90&publication_year=1993&pages=10235-10239&pmid=8234283&doi=10.1073/pnas.90.21.10235&)

33. Chen Y, Yu L. J Biol Chem. 1994;269:7839–7842.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/8132498/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&author=Y%20Chen&author=L%20Yu&volume=269&publication_year=1994&pages=7839-7842&pmid=8132498&)

34. Kovoor A, Henry D J, Chavkin C. J Biol Chem. 1995;270:589–595. doi: 10.1074/jbc.270.2.589.  [DOI](https://doi.org/10.1074/jbc.270.2.589) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7822283/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&author=A%20Kovoor&author=D%20J%20Henry&author=C%20Chavkin&volume=270&publication_year=1995&pages=589-595&pmid=7822283&doi=10.1074/jbc.270.2.589&)

35. Kreek M J, Hartman N. Ann N Y Acad Sci. 1982;398:151–172. doi: 10.1111/j.1749-6632.1982.tb39489.x.  [DOI](https://doi.org/10.1111/j.1749-6632.1982.tb39489.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/6130734/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20N%20Y%20Acad%20Sci&author=M%20J%20Kreek&author=N%20Hartman&volume=398&publication_year=1982&pages=151-172&pmid=6130734&doi=10.1111/j.1749-6632.1982.tb39489.x&)

36. Zhou Y, Spangler R, LaForge K S, Maggos C E, Ho A, Kreek M J. Peptides. 1996;17:435–441. doi: 10.1016/0196-9781(96)00014-9.  [DOI](https://doi.org/10.1016/0196-9781(96)00014-9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8735970/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Peptides&author=Y%20Zhou&author=R%20Spangler&author=K%20S%20LaForge&author=C%20E%20Maggos&author=A%20Ho&volume=17&publication_year=1996&pages=435-441&pmid=8735970&doi=10.1016/0196-9781(96)00014-9&)

37. Kreek M J, Wardlaw S L, Friedman J, Schneider B, Frantz A G. In: Advances in Endogenous and Exogenous Opioids. Simon E, Takagi H, editors. Tokyo: Kodansha; 1981. pp. 364–366.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Advances%20in%20Endogenous%20and%20Exogenous%20Opioids&author=M%20J%20Kreek&author=S%20L%20Wardlaw&author=J%20Friedman&author=B%20Schneider&author=A%20G%20Frantz&publication_year=1981&)

38. Kreek M J, Ragunath J, Plevy S, Hamer D, Schneider B, Hartman N. Neuropeptides. 1984;5:277–278. doi: 10.1016/0143-4179(84)90081-7.  [DOI](https://doi.org/10.1016/0143-4179(84)90081-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/6099512/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neuropeptides&author=M%20J%20Kreek&author=J%20Ragunath&author=S%20Plevy&author=D%20Hamer&author=B%20Schneider&volume=5&publication_year=1984&pages=277-278&pmid=6099512&doi=10.1016/0143-4179(84)90081-7&)

39. Kosten T R, Kreek M J, Swift C, Carney M K, Ferdinands L. Life Sci. 1987;41:1071–1076. doi: 10.1016/0024-3205(87)90623-0.  [DOI](https://doi.org/10.1016/0024-3205(87)90623-0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2956475/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Life%20Sci&author=T%20R%20Kosten&author=M%20J%20Kreek&author=C%20Swift&author=M%20K%20Carney&author=L%20Ferdinands&volume=41&publication_year=1987&pages=1071-1076&pmid=2956475&doi=10.1016/0024-3205(87)90623-0&)

40. Kosten T R, Kreek M J, Ragunath J, Kleber H D. Life Sci. 1986;39:55–59. doi: 10.1016/0024-3205(86)90437-6.  [DOI](https://doi.org/10.1016/0024-3205(86)90437-6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2941636/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Life%20Sci&author=T%20R%20Kosten&author=M%20J%20Kreek&author=J%20Ragunath&author=H%20D%20Kleber&volume=39&publication_year=1986&pages=55-59&pmid=2941636&doi=10.1016/0024-3205(86)90437-6&)

41. Kennedy J A, Hartman N, Sbriglio R, Khuri E, Kreek M J. Br J Addiction. 1990;85:1133–1140. doi: 10.1111/j.1360-0443.1990.tb03438.x.  [DOI](https://doi.org/10.1111/j.1360-0443.1990.tb03438.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2224193/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Addiction&author=J%20A%20Kennedy&author=N%20Hartman&author=R%20Sbriglio&author=E%20Khuri&author=M%20J%20Kreek&volume=85&publication_year=1990&pages=1133-1140&pmid=2224193&doi=10.1111/j.1360-0443.1990.tb03438.x&)

42. Culpepper-Morgan J A, Kreek M J. Metab Clin Exp. 1997;46:130–134. doi: 10.1016/s0026-0495(97)90289-4.  [DOI](https://doi.org/10.1016/s0026-0495(97)90289-4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9030816/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Metab%20Clin%20Exp&author=J%20A%20Culpepper-Morgan&author=M%20J%20Kreek&volume=46&publication_year=1997&pages=130-134&pmid=9030816&doi=10.1016/s0026-0495(97)90289-4&)

43. Kreek M J. Pharmacol Biochem Behav. 1997;57:551–569. doi: 10.1016/s0091-3057(96)00440-6.  [DOI](https://doi.org/10.1016/s0091-3057(96)00440-6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9218280/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacol%20Biochem%20Behav&author=M%20J%20Kreek&volume=57&publication_year=1997&pages=551-569&pmid=9218280&doi=10.1016/s0091-3057(96)00440-6&)
