# Influence of Leukotriene Pathway Polymorphisms on Response to Montelukast in Asthma

## Metadata
**Authors:** John J Lima, Shu Zhang, Audrey Grant, Lianhe Shao, Kelan G Tantisira, Hooman Allayee, Jianwei Wang, James Sylvester, Janet Holbrook, Robert Wise, Scott T Weiss, Kathleen Barnes
**Journal:** American Journal of Respiratory and Critical Care Medicine
**Date:** 2005 Nov 17
**DOI:** [10.1164/rccm.200509-1412OC](https://doi.org/10.1164/rccm.200509-1412OC)
**PMID:** 16293801
**PMCID:** PMC2662939
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2662939/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC2662939/pdf/AJRCCM1734379.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC2662939/pdf/AJRCCM1734379.pdf)

## Abstract

Rationale: Interpatient variability in montelukast response may be related to variation in leukotriene pathway candidate genes.

Objective: To determine associations between polymorphisms in leukotriene pathway candidate genes with outcomes in patients with asthma receiving montelukast for 6 mo who participated in a clinical trial.

Methods: Polymorphisms were typed using Sequenom matrix-assisted laser desorption/ionization time-of-flight (MALDI-TOF) mass array spectrometry and published methods; haplotypes were imputed using single nucleotide polymorphism–expectation maximization (SNP-EM). Analysis of variance and logistic regression models were used to test for changes in outcomes by genotype. In addition, χ2 and likelihood ratio tests were used to test for differences between groups. Case-control comparisons were analyzed using the SNP-EM Omnibus likelihood ratio test.

Measurements: Outcomes were asthma exacerbation rate and changes in FEV1 compared with baseline.

Results: DNA was collected from 252 participants: 69% were white, 26% were African American. Twenty-eight SNPs in the ALOX5, LTA4H, LTC4S, MRP1, and cysLT1R genes, and an ALOX5 repeat polymorphism were successfully typed. There were racial disparities in allele frequencies in 17 SNPs and in the repeat polymorphism. Association analyses were performed in 61 whites. Associations were found between genotypes of SNPs in the ALOX5 (rs2115819) and MRP1 (rs119774) genes and changes in FEV1 (p < 0.05), and between two SNPs in LTC4S (rs730012) and in LTA4H (rs2660845) genes for exacerbation rates. Mutant ALOX5 repeat polymorphism was associated with decreased exacerbation rates. There was strong linkage disequilibrium between ALOX5 SNPs. Associations between ALOX5 haplotypes and risk of exacerbations were found.

Conclusions: Genetic variation in leukotriene pathway candidate genes contributes to variability in montelukast response.

Keywords: antiinflammatory, montelukast, pharmacodynamic, pharmacogenetic

**Keywords:** antiinflammatory, montelukast, pharmacodynamic, pharmacogenetic

## METHODS

### Study Design and Patient Studies

This pharmacogenetic study was ancillary to a randomized, double-masked, parallel-designed trial that compared the efficacy of placebo, theophylline (Theochron Extended Release; Inwood Laboratories, Inc., Inwood, NY) 300 mg daily, and montelukast (Singulair; Merck and Co., Inc., Whitehouse Station, NJ) 10 mg daily, as add-on therapy in patients with poorly controlled mild to moderate persistent asthma. Doses were identically masked within opaque gelatin capsules. Briefly, 488 patients were recruited from 19 centers in the American Lung Association Asthma Clinical Research Centers Network. Before randomization, all patients completed a questionnaire that included queries about demographic characteristics, smoking history, age at onset of asthma, hospitalizations, unscheduled health care visits for asthma, or courses of oral corticosteroids during the preceding 12 mo. In addition, participants completed the Asthma Symptom Utility Index ([28](#bib28)), Asthma Control Questionnaire ([29](#bib29)), spirometry with bronchodilator, and measurement of peak expiratory flow. DNA was collected from 252 participants who volunteered for the trial and for the pharmacogenetic study. The institutional review boards of each participating center approved the protocols for the trial and for the ancillary pharmacogenetic study.

### Outcomes

Associations between genetic variants with two outcomes were analyzed: percentage of change in % predicted FEV_1_ after 6 mo of montelukast treatment compared with % predicted FEV_1_ recorded at baseline, and the binary risk of having an asthma exacerbation (no exacerbation or at least one exacerbation) during 6 mo of montelukast treatment. An asthma exacerbation was defined as 1 or more of the following: a more than 30% decrease in peak expiratory flow rate for 2 consecutive days; a course of oral steroids; an unscheduled visit to the clinic, the emergency room, or hospital; or an increase of four puffs of rescue inhaler use in 1 d.

### Genotyping

Single nucleotide polymorphisms (SNPs) were genotyped via a Sequenom matrix-assisted laser desorption/ionization time-of-flight (MALDI-TOF) mass array spectrometer (Sequenom, San Diego, CA), using a semiautomated primer design program (Spectro Designer; Sequenom) ([30](#bib30)–[32](#bib32)). In addition, the *LTC4S* C-444A promoter SNP was genotyped and the number of Sp1 binding motifs (5′GGGCGG3′) in the *ALOX5* promoter was determined as previously described ([16](#bib16), [33](#bib33)). Hardy-Weinberg equilibrium (HWE) between expected and observed genotype distributions was calculated using χ^2^ goodness-of-fit tests. For SNPs on the *CYSLTR1*, HWE was determined in females because of the location of this gene on the X chromosome.

### Determination of Haplotype

Haplotypes for the *ALOX5*, *LTA4H*, and *MRP1* SNPs were imputed using an expectation-maximization (EM) algorithm (SNP-EM) ([34](#bib34), [35](#bib35)). HWE between expected and observed genotype distributions was calculated using χ^2^ goodness-of-fit tests. Linkage disequilibrium (LD) was assessed and displayed using Haploview ([http://www.broad.mit.edu/mpg/haploview/](http://www.broad.mit.edu/mpg/haploview/)).

### Association Analyses

An analysis of variance (Stata 8.0; StataCorp, College Station, TX) was used to test for differences in mean percentage changes in FEV_1_ at 6 mo of treatment compared with baseline FEV_1_ by genotype. Logistic regression models were used to test for increases in risk of exacerbations at any time point during montelukast treatment for particular marker genotypes. A χ^2^ test and two-sample tests of proportions were used to test the differences in frequencies between two groups. Case-control (participants experiencing at least one exacerbation vs. participants with no exacerbations) comparisons were analyzed by an omnibus test (SNP-EM Omnibus likelihood ratio test) ([34](#bib34), [35](#bib35)); p values less than 0.05 were considered significant. For polymorphisms significantly associated with an outcome in participants on montelukast, associations were analyzed in participants assigned to placebo.

## RESULTS

### Patients

Baseline characteristics of participants are shown in [Table 1](#tbl1). A total of 252 individuals participated in the pharmacogenetic study: 88 were randomized to receive montelukast, 77 received theophylline, and 86 received placebo (baseline data were not collected on one participant assigned to theophylline treatment). Baseline characteristics were reasonably evenly distributed between the three groups. Approximately 77 to 79% were on inhaled corticosteroids at randomization. Mean values of post-bronchodilator pulmonary function measures and scores of Asthma Symptom Utility Index and of Asthma Control Questionnaire indicated that this cohort had mild to moderately severe persistent asthma that was not well controlled at baseline. The percentage of participants who smoked and who were exposed to second-hand smoke was reasonably evenly distributed among the three treatment groups (data not shown). Asthma exacerbation rates in participants after 6 mo of placebo, montelukast, and theophylline treatment were 6.1, 3.7, and 5.2 events/person-year, respectively. Whites and African Americans comprised 69 and 24%, respectively, of the participants randomized to receive montelukast for 6 mo ([Table 1](#tbl1)). Because of the relatively low number of African Americans and the potential for population stratification ([36](#bib36)), analyses were restricted to 61 whites in the montelukast arm.

### TABLE 1.

|   | Treatment Assignment* |  |  |
| --- | --- | --- | --- |
|   | T | M | P |
| No. | 77 | 88 | 86 |
| Mean age at randomization, yr (SD) | 41 (15) | 40 (15) | 42 (13) |
| Male, % | 23 | 30 | 36 |
| Race or ethnic group, % |   |   |   |
| White | 69 | 69 | 67 |
| African American | 27 | 24 | 28 |
| Hispanic | 0 | 6 | 5 |
| Other | 4 | 1 | 0 |
| Asthma characteristics |   |   |   |
| Mean age of asthma onset, yr (SD) | 21 (18) | 19 (17) | 21 (16) |
| > 1 scheduled health care visits for asthma in previous 12 mo, % | 38 | 45 | 43 |
| > 1 course of oral corticosteroids in previous 12 mo, % | 42 | 41 | 48 |
| Use of inhaled short-acting β-agonist > times/wk, % | 40 | 47 | 47 |
| Use of nebulized β-agonist > time/wk, % | 1 | 5 | 8 |
| On inhaled corticosteroids at baseline, % | 78 | 77 | 79 |
| Mean asthma scores (SD) |   |   |   |
| ASUI score, range: 0–1 | 0.67 (0.17) | 0.69 (0.14) | 0.70 (0.15) |
| ACQ, range: 1–6 | 2.3 (0.6) | 2.2 (0.6) | 2.3 (0.6) |
| Mean pulmonary function measures (SD)* |   |   |   |
| FEV1, L | 2.6 (0.7) | 2.5 (0.8) | 2.4 (0.8) |
| FEV1, % predicted | 87.3 (15.4) | 83.5 (18.1) | 83.8 (15.8) |
| FVC, % predicted | 94.4 (17.1) | 92.0 (17.6) | 88.5 (13.9) |
| Peak flow rate, L/min | 382 (108) | 374 (112) | 349 (94) |
| Peak flow, % predicted | 85.5 (19.8) | 83.2 (20.9) | 77.8 (19.4) |

Table 1 Caption: BASELINE CHARACTERISTICS AMONG PHARMACOGENETIC STUDY PARTICIPANTS

### Allele Frequencies, HWE, and LD

A total 42 SNPs and the *ALOX5* promoter sp1 tandem repeat polymorphism were genotyped. Three nonsynonymous SNPs failed optimization, nine were monomorphic, and two were dropped because they did not pass quality control for discordant samples. The overall percentage of successful genotyping calls was 96%. [Table 2](#tbl2) lists p values for HWEs, minor allele frequencies, and racial differences of the remaining 28 SNPs. Two SNPs in whites (rs2247570 on *LTA4H* and rs152033 in *MRP1*) and three in African Americans (rs129081, rs35587N_N, and re3902401 in MRP1) were not in HWE. There were significant racial disparities in allele frequencies for 16 SNPs.

### TABLE 2.

|   |   | Whites |  |  | African Americans |  |  |   |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Gene | SNP | n* | HWE p Value | MAF | n* | HWE p Value | MAF | p Value‡ |
| ALOX5 | rs2029523 (A>G) | 336 | 0.558 | 0.429 | 124 | 0.182 | 0.250 | 0.001 |
|   | rs2115819 (A>G) | 322 | 0.220 | 0.472 | 120 | 0.071 | 0.233 | < 0.001 |
|   | rs745986 (A>G) | 328 | 0.751 | 0.217 | 124 | 0.843 | 0.169 | 0.258 |
|   | rs892690 (C>T) | 330 | 0.684 | 0.421 | 126 | 0.649 | 0.040 | < 0.001 |
|   | rs892691 (G>A) | 330 | 0.089 | 0.233 | 124 | 0.198 | 0.315 | 0.074 |
| cysLTR1 | rs2412222 (G>A) | 350 | 0.931† | 0.249 | 128 | 0.976† | 0.305 | 0.401 |
|   | rs320995F_F (A>G) | 334 | 0.902† | 0.261 | 128 | 0.936† | 0.313 | 0.422 |
|   | rs321029 (G>A) | 332 | 0.834† | 0.280 | 126 | 0.747† | 0.142 | 0.013 |
|   | rs321092 (A>G) | 328 | 0.819† | 0.284 | 134 | 0.795† | 0.216 | 0.065 |
| LTA4H | rs2241136 (G>A) | 330 | 0.041 | 0.106 | 126 | 0.521 | 0.056 | 0.099 |
|   | rs2247570 (T>C) | 328 | 0.255 | 0.320 | 126 | 0.418 | 0.429 | 0.029 |
|   | rs2660845 (A>G) | 330 | 0.560 | 0.297 | 120 | 0.513 | 0.392 | 0.057 |
| LTC4S | rs272431 (G>T) | 302 | 0.954 | 0.003 | 100 | 0.652 | 0.260 | < 0.001 |
|   | rs272440 (G>A) | 330 | 0.956 | 0.003 | 126 | 0.966 | 0.254 | < 0.001 |
|   | rs730012 (A>C) | 328 | 1.000 | 0.302 | 130 | 1.000 | 0.086 | < 0.001 |
| MRP1 | rs119774 (C>T) | 256 | 0.243 | 0.070 | 106 | 0.768 | 0.028 | 0.119 |
|   | rs129081 (G>C) | 330 | 0.990 | 0.349 | 120 | 0.040 | 0.467 | 0.023 |
|   | rs152033 (C>T) | 330 | 0.022 | 0.118 | 126 | 0.570 | 0.191 | 0.043 |
|   | rs1967120 (A>G) | 336 | 0.914 | 0.286 | 128 | 0.462 | 0.313 | 0.568 |
|   | rs212081 (G>A) | 328 | 0.416 | 0.378 | 120 | 0.998 | 0.342 | 0.484 |
|   | rs215066 (G>A) | 330 | 0.337 | 0.052 | 124 | 0.383 | 0.177 | < 0.001 |
|   | rs2239330S_S (G>A) | 332 | 0.164 | 0.283 | 120 | 0.156 | 0.058 | < 0.001 |
|   | rs2239996 (A>G) | 326 | 0.473 | 0.485 | 126 | 0.986 | 0.127 | < 0.001 |
|   | rs246221V_V (T>C) | 328 | 0.613 | 0.256 | 108 | 0.105 | 0.472 | < 0.001 |
|   | rs35587N_N (T>C) | 334 | 0.402 | 0.267 | 122 | 0.007 | 0.475 | < 0.001 |
|   | rs3887893 (T>C) | 324 | 0.080 | 0.395 | 126 | 0.381 | 0.405 | 0.846 |
|   | rs3902401 (C>T) | 322 | 0.275 | 0.059 | 126 | 0.009 | 0.064 | 0.842 |
|   | rs4148356R_Q (G>A) | 330 | 0.956 | 0.003 | 126 | 0.929 | 0.008 | 0.470 |

Table 2 Caption: HARDY-WEINBERG EQUILIBRIA AND MINOR ALLELE FREQUENCIES OF 28 SINGLE NUCLEOTIDE POLYMORPHISMS IN LEUKOTRIENE PATHWAY CANDIDATE GENES IN WHITES AND AFRICAN AMERICANS

The allelic and genotypic frequencies of Sp1 binding motifs (5′GGGCGG3′) in the *ALOX5* promoter polymorphisms for whites and African Americans are shown in [Table 3](#tbl3). The percentage of successful genotyping calls was 97%. In whites and African Americans, 80 and 47%, respectively, carried five tandem repeats (p < 0.001), followed by four repeats, which represented 17% in both races. One-third of African-American alleles carried three repeats compared with less than 1% in whites. The most common genotype in whites was 5/5 followed by 4/6. In African Americans, the most common genotype was 3/5, followed by 5/5, 4/5, and 3/4 ([Table 3](#tbl3)). When collapsed into three genotypes based on the wild-type (n = 5) and the mutant form X (n ≠ 5), significant racial differences were observed. The distribution of genotypes in whites in our study was similar to those published previously ([12](#bib12), [37](#bib37)–[39](#bib39)).

### TABLE 3.

| Whites |  | African Americans |  |   |
| --- | --- | --- | --- | --- |
| Allele | n (%) | Allele | n (%) | p Value |
| 3 | 2 (0.6) | 3 | 44 (34) | < 0.001 |
| 4 | 56 (17) | 4 | 22 (17) | 0.99 |
| 5 | 264 (80) | 5 | 61 (47) | < 0.001 |
| 6 | 7 (2.1) | 6 | 3 (2.3) | 0.9 |
| 7 | 1 (0.3) | 7 | 0 (0) | 0.5 |
| Total | 330 (100) |   | 130 (100) |   |
| Genotype |   | Genotype |   |   |
| 3/4 | 1 (0.6) | 3/3 | 3 (4.6) |   |
| 3/5 | 1 (0.6) | 3/4 | 10 (15) |   |
| 4/4 | 4 (2.4) | 3/5 | 26 (40) |   |
| 4/5 | 46 (28) | 3/6 | 2 (3.1) |   |
| 4/7 | 1 (0.6) | 4/5 | 11 (17) |   |
| 5/5 | 105 (64) | 4/6 | 1 (1.5) |   |
| 5/6 | 7 (4.2) | 5/5 | 12 (18.5) |   |
| Total | 165 (100) |   | 65 (100) |   |
| 5/5 | 105 (64) | 5/5 | 12 (18.5) | < 0.001 |
| 5/X | 54 (33) | 5/X | 37 (57) | 0.0007 |
| X/X | 6 (3.6) | X/X | 16 (24.6) | < 0.001 |
| Total | 165 (100) |   | 65 (100) |   |

Table 3 Caption: ALLELIC AND GENOTYPIC FREQUENCIES OF Sp1 BINDING MOTIFS IN THE ALOX5 PROMOTER FOR 165 WHITES AND 65 AFRICAN AMERICANS

Figure E1R (*see* online supplement) shows LD between *ALOX5* SNPs. Strong pairwise LD was observed between *ALOX5* SNPs 1 (rs892690) and 5 (rs892691), SNPs 2 (rs745986) and 3 (rs2029253), and between SNPs 3 and 4 (rs2115819), as determined by D′ values greater than 0.9 (×100). Modest LD was found between SNPs 1 and 3, SNPs 1 and 4, and between SNPs 3 and 5 (rs892691). For the *LTA4H* gene, SNPs rs2241136 and rs26606845 were in modest pairwise LD with a D′ value of 0.69 (data not shown).

Figure E2R shows LD between *MRP1* SNPs. Strong pairwise LD was observed between SNPs rs246271 and rs35587 (SNPs 5 and 6), SNPs 1, 2, and 3, and SNPs 2 and 3.

### Genotype Association Analysis

Significant associations were found between two LT pathway SNPs and the change in % predicted FEV_1_ observed after 6 mo of montelukast treatment compared with baseline ([Figure 1](#fig1)). Compared with CC homozygotes (n = 41), heterozygotes (n = 8) for the *MRP1* rs119774 SNP had higher percentage changes in % predicted FEV_1_: 24% (95% confidence interval [CI], −0.105 to 0.577) versus 2.2% (95% CI, −0.005 to 0.049) increase (p = 0.004). Compared with AA homozygotes (n = 11) and heterozygotes (n = 38), GG homozygotes (n = 6) for the *ALOX5* rs2115819 SNP had a significantly higher FEV_1_ response to montelukast at 6 mo of treatment: 30% (95% CI = −0.017 to 1.21) versus 4.4% (95% CI, −0.025 to 0.66) and 2.0% (95% CI, 0.013–0.075) in the AA and AG genotype groups, respectively (p = 0.017). No significant associations were observed between changes in FEV_1_ and *MRP1* rs119774 (p = 0.56) and *ALOX5* rs2115819 (p = 0.33) in participants assigned to placebo.

### Figure 1.

![Figure 1.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1f13/2662939/a0ad4be9cd77/AJRCCM1734379fig1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=2662939_AJRCCM1734379fig1.jpg)

Influence of genotype on percentage change in % predicted FEV1 over baseline in 61 white participants taking montelukast. Percentage change in % predicted FEV1 over baseline after 6 mo of montelukast treatment was compared by genotypes of MRP1 marker rs119774 and of ALOX5 marker rs2115819.

[Table 4](#tbl4) summarizes the influence of LT pathway polymorphisms on the risk of having at least one asthma exacerbation in participants receiving montelukast. Individuals carrying a variant number (either 2, 3, 4, 6, or 7) of repeats of the *ALOX5* promoter on one allele had a 73% reduction in the risk of having one or more asthma exacerbations compared with homozygotes for the five repeat alleles (p = 0.045). For participants on placebo for 6 mo, there were no differences in exacerbation risk by genotype (p = 0.134).

### TABLE 4.

| Gene (variant) | Genotype | Frequency (n) | Odds Ratio (95% CI) | p Value |
| --- | --- | --- | --- | --- |
| ALOX5 (repeat variant) | 5/5 | 0.64 (37) | 1.0 | 0.045 |
|   | 5/X | 0.36 (21) | 0.27 (0.08, 0.97) |   |
| LTA4H (rs2660845) | AA | 0.49 (28) | 1.0 | 0.021 |
|   | AG | 0.44 (25) | 4.0 (1.23, 12.99) | 0.133 |
|   | GG | 0.08 (5) | 4.5 (0.63, 31.95) |   |
| LTC4S (rs730012) | AA | 0.51 (30) | 1.0 | 0.023 |
|   | AC | 0.38 (22) | 0.24 (0.07, 0.83) | 0.106 |
|   | CC | 0.11 (6) | 0.16 (0.02, 1.49) |   |

Table 4 Caption: GENOTYPE AND FREQUENCIES OF ALOX5, LTA4H, AND LTC4S POLYMORPHISMS AND ODDS RATIOS OF ASSOCIATIONS BETWEEN GENOTYPE AND LIKELIHOOD OF HAVING AN ASTHMA EXACERBATION IN WHITES TAKING MONTELUKAST FOR 6 MO

For the *LTA4H* rs2660845 SNP, the risk of having at least one exacerbation was 4- to 4.5-fold higher in heterozygotes and GG homozygotes compared with AA homozygotes. The odds ratio for asthma exacerbations for GG homozygotes did not achieve statistical significance, which was probably related to the small number of individuals carrying this genotype. When collapsed into carriers of the G allele (AG + GG), the odds ratio for having an exacerbation was greater than 4.0 (p < 0.001). For participants receiving placebo, no differences in exacerbation rate were noted by genotype (p = 0.85 for AG genotype; p = 0.776 for GG homozygotes).

For the *LTC4S* A-444C SNP (rs730012), heterozygotes receiving montelukast had a 76% reduced risk of having an asthma exacerbation compared with AA homozygotes (p = 0.023). The risk of having an exacerbation was reduced even more in CC homozygotes; however, this difference was not statistically significant. This may be related to the relatively low frequency of CC homozygotes (11%). When collapsed into carriers of the C allele (AC + CC), the risk was reduced by 80% compared with AA homozygotes (p < 0.001). Heterozygotes assigned to placebo had a 74% reduced risk of having an asthma exacerbation compared with AA homozygotes (p = 0.034). The risk of having an exacerbation in CC homozygotes was no different compared with AA homozygotes (p = 0.57). When collapsed into carriers of the C allele, the risk was reduced by 69% compared with AA (p = 0.05).

### Haplotype Association Analysis

Significant associations were found between *ALOX5* 2-, 3- and 4-SNP haplotypes and exacerbation rates using the omnibus logistical regression test (SNP-EM; data not shown). A χ^2^ analysis was used to compare differences in cases (frequency of participants having at least one asthma exacerbation while receiving montelukast) and control subjects (frequency of participants with no exacerbations) among haplotypes ([Table 5](#tbl5)). Haplotypes with the A alleles for SNPs 2 and 3 are strongly associated with the risk of having an asthma exacerbation and addition of the C allele from SNP 1 to two or three SNP haplotypes (CAA, CAAA) tended to increase the strength of the association.

### TABLE 5.

| ALOX5 Haplotypes |  |  |  |  | Estimated Frequency |  | χ2 p Value |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
| rs892690 no. 1 | rs745986 no. 2 | rs202925 no. 3 | rs2115819 no. 4 | rs892691 no. 5 | Cases* | Control Subjects† |   |   |
|   | A | A |   |   | 0.521 | 0.286 | 7.8 | 0.002 |
|   | G | A |   |   | 0.104 | 0.286 | 5.6 | 0.007 |
| C | A | A |   |   | 0.558 | 0.286 | 8.9 | 0.001 |
|   | A | A | A |   | 0.534 | 0.313 | 5.4 | 0.01 |
| C | A | A | A |   | 0.585 | 0.313 | 8.1 | 0.003 |
|   | A | A | A | A | 0.326 | 0.137 | 5.7 | 0.037 |

Table 5 Caption: ASSOCIATION ANALYSES FOR ALOX5 HAPLOTYPES AND FREQUENCIES OF WHITE PARTICIPANTS HAVING AT LEAST ONE ASTHMA EXACERBATION DURING MONTELUKAST TREATMENT

## DISCUSSION

Montelukast is recommended as an alternative to low-dose inhaled corticosteroids for patients with mild persistent asthma and as alternative add-on therapy to inhaled corticosteroid treatment in patients with moderate persistent (step 3) and severe persistent (step 4) asthma ([2](#bib2)). Although the drug is safe and effective in controlling asthma symptoms, responsiveness is highly variable among patients, which is believed to be due to genetic variation. Several studies have reported that the repeat polymorphism in the *ALOX5* promoter ([12](#bib12)) and the *LTC4S* A-444C SNP ([13](#bib13)–[16](#bib16)) contributes to the variability in response montelukast and other LT modifiers. However, the allele frequency of the *ALOX5* repeat polymorphism in whites is too low to contribute much to the variability in response to LT modifiers, and the influence of the *LTC4S* A-444C SNP on response to LT receptor antagonists has been questioned ([40](#bib40), [41](#bib41)).

The present study explored associations between polymorphisms in candidate genes encoding key proteins in the LT pathway with response in patients randomized to montelukast treatment as participants in a large clinical trial. We identified five polymorphisms that were associated with changes in FEV_1_ or with the risk of exacerbations while receiving montelukast. When analyzed in participants assigned to placebo, no associations were found between outcomes and genotype, with the possible exception of heterozygotes for the *LTC4S* –444 SNP (*see below*). Our results support the idea that genetic variation contributes in a significant way to the interpatient variability in response to montelukast and other LT receptor antagonists. In addition, our data point to the possibility of individualizing LT receptor antagonist treatment using genotyping information.

The *ALOX5* gene located on 10q11.21 encodes a key enzyme in the synthesis of cysLTs ([18](#bib18)). Early studies identified addition and deletion variants in the core promoter of the *ALOX5* gene that were associated with diminished promoter-reporter activity in tissue culture ([33](#bib33)). In a later study, Drazen and colleagues ([12](#bib12)) hypothesized that there would be decreased ALOX5 product production and diminished response to drugs treating this pathway because of diminished gene transcription associated with addition and deletion variants. Indeed, ABT-761, an ALOX5 inhibitor, increased FEV_1_ over baseline in wild-type homozygotes (5/5) and heterozygotes (5/X) compared with variant allele homozygotes (X/X) ([12](#bib12)). The results of the present study are not consistent with expectations based on this study. We found that montelukast was associated with a 73% reduced risk of an exacerbation in carriers of the mutant allele (X/X and 5/X) compared with wild-type homozygotes, suggesting that mutant variants up-regulated ALOX5 activity. Consistent with our data, Dwyer and coworkers ([39](#bib39)) reported that compared with wild-type and heterozygotes (5/5 + 5/X), homozygous mutants (X/X) had increased carotid intima-media thickness, an atherogenic effect that was exacerbated by increased intake of dietary arachidonic acid, and had higher C-reactive protein levels. In addition, patients with aspirin-intolerant asthma, who are known to be responsive to LT antagonists, carrying the mutant allele (X) showed increased hyperresponsiveness compared with patients with the wild-type genotype ([42](#bib42)). Taken together, these data suggest that the repeat polymorphism in the *ALOX5* promoter is an important pharmacogenetic locus, and underscore the need for additional pharmacogenetic studies that target the *ALOX5* gene.

LTC_4_ synthase catalyzes the formation of LTC_4_ from LTA_4_ ([18](#bib18)). In the present study, the *LTC4S* A-444C promoter SNP (rs730012; 5q35) was associated with a reduced risk of an asthma exacerbation: the C allele reduced risk by 80% compared with AA homozygotes receiving montelukast ([Table 4](#tbl4)). In the placebo group, the exacerbation rate was significantly reduced in heterozygotes compared with A homozygotes, which questions our findings with montelukast. However, the exacerbation risk in C homozygotes was not different compared with A homozygotes (p = 0.569). Moreover, when placebo participants were collapsed into carriers of the C allele, they were not at greater risk of an exacerbation compared with A homozygotes (p = 0.05). In contrast, carriers of the C allele on montelukast had an 80% reduced risk of an exacerbation compared with A homozygotes (p < 0.001). This suggests that the significant association observed in heterozygotes on placebo is spurious, probably because of small numbers. Therefore, we conclude that the *LTC4S* –A444C SNP contributes to the variability in response to montelukast. These data are in agreement with previous studies reporting that carriers of the C allele responded better to LT receptor antagonists compared with AA homozygotes ([14](#bib14)–[16](#bib16)). The mechanisms underlying the favorable response to montelukast in carriers of the C allele compared with AA homozygotes may be related to up-regulation of LTC_4_ synthase expression, which would result in higher concentrations of cysLTs and increased inflammation ([43](#bib43)). Thus, our study replicates previous studies and supports the idea that *LTC4S* is an important gene, which contributes to variability in response to LT receptor antagonists.

The present study identified three novel associations between LT pathway SNPs and responsiveness to montelukast. The genotype of rs2115819 located in intron 2 of *ALOX5* was associated with differences in the FEV_1_ response to montelukast ([Figure 1](#fig1)), and is in tight LD with rs2029253 (Figure E1R). Moreover, it is one of four SNPs that comprise a haplotype that is associated with the highest proportion of participants having an asthma exacerbation ([Table 5](#tbl5)). It is also possible that one or more of these intronic SNPs could by themselves be functional. Further studies are required to replicate these data in a larger clinical trial, and to identify the functional SNPs that may be in LD with rs2115819.

The *LTA4H* gene, located on chromosome 12q22, encodes the enzyme that catalyzes the formation of LTB_4_ ([44](#bib44)), a potent chemoattractant agent ([45](#bib45), [46](#bib46)), from LTA_4_. In the present study, compared with AA homozygotes, carriers of the G allele of rs2660845 had a four- to fivefold increased probability of having an asthma exacerbation while receiving montelukast. The mechanism underlying the association between the genotype of this SNP and the risk of an asthma exacerbation is unknown. One possibility could be related to the G allele down-regulating the activity of LTA_4_H, which would result in shunting LTA_4_ away from the LTA_4_H pathway and increasing the formation of cysLTs.

LTC_4_ is transported to the extracellular space by *MRP1*, a member of the ABC family of transmembrane transport proteins ([22](#bib22), [47](#bib47)). MRP1 is highly expressed in human bronchial epithelial cells ([48](#bib48)). The *MRP1* gene is located on 16p13.12 and is highly polymorphic ([49](#bib49), [50](#bib50)). A mutation in the last transmembrane segment influences LTC_4_ transport ([51](#bib51)) and it is possible that *MRP1* genetic variants could have significant effects on LTC_4_ transport, cysLT expression, and response to montelukast. The genotype of rs119774, which is located in intron 1, was associated with increases in % predicted FEV_1_ in participants receiving montelukast for 6 mo ([Figure 1](#fig1)): heterozygotes (CT) had a 24% increase in % predicted FEV_1_ compared with a 2% increase in CC homozygotes. No association between genotype and changes in % predicted FEV_1_ was observed in participants on placebo. In addition, rs119774 is in fairly tight LD with rs215066, which is also in intron 1and showed a positive trend for an association between montelukast-evoked changes in % predicted FEV_1_ and rs215066 (p = 0.066). To our knowledge, this is the first report of a genotype–phenotype association for the *MRP1* gene in patients with asthma receiving montelukast and warrants further study.

The present study has several limitations. Gene variants that contribute to variable drug response in complex phenotypes, like asthma, may have modest effects, thus requiring large sample sizes to detect associations ([52](#bib52)). Our sample size was small, and it is possible that the associations we observed between LT pathway SNPs and responsiveness to montelukast could represent false-positive results. Because of our small sample size and the potential for population stratification ([36](#bib36)), we restricted our analysis to whites. We did not correct for multiple hypothesis testing, which could also contribute to false-positive associations ([52](#bib52)). We chose not to adjust for multiple comparisons because, given the small numbers of participants, we reasoned that it is important not to dismiss differences that could be real. For these reasons, the results of our study should be regarded as exploratory and underscore the need for replication in larger, more diverse populations.

In summary, we found significant associations between several common polymorphisms in LT pathway candidate genes with either the risk of having an asthma exacerbation or an increase in % predicted FEV_1_ over baseline in whites with asthma who received montelukast for 6 mo. For two polymorphisms, the *ALOX5* tandem repeat promoter polymorphism and the *LTC4S* A-444C SNP, our results replicate previous studies; for three SNPs in *ALOX5*, *LTA4H*, and *MRP1* genes, our results show novel associations. Further studies are required to replicate our associations.

## Supplementary Material

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Drazen J. Clinical pharmacology of leukotriene receptor antagonists and 5-lipoxygenase inhibitors. Am J Respir Crit Care Med 1998;157:S233–S237.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/9620945/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Respir%20Crit%20Care%20Med&volume=157&publication_year=1998&pages=S233&pmid=9620945&)

2. National Asthma Education and Prevention Program. Expert Panel Report 2: guidelines for the diagnosis and management of asthma, April 1997. Bethesda, MD: U.S. Department of Health and Human Services, Public Health Service, National Institutes of Health, National Heart, Lung, and Blood Institute; 1997. Publication No. 97–4051.

3. Nayak A. A review of montelukast in the treatment of asthma and allergic rhinitis. Expert Opin Pharmacother 2004;5:679–686.  [DOI](https://doi.org/10.1517/14656566.5.3.679) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15013935/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Expert%20Opin%20Pharmacother&volume=5&publication_year=2004&pages=679&pmid=15013935&doi=10.1517/14656566.5.3.679&)

4. Bisgaard H, Zielen S, Garcia-Garcia ML, Johnston SL, Gilles L, Menten J, Tozzi CA, Polos P. Montelukast reduces asthma exacerbations in 2- to 5-year-old children with intermittent asthma. Am J Respir Crit Care Med 2005;171:315–322.  [DOI](https://doi.org/10.1164/rccm.200407-894OC) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15542792/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Respir%20Crit%20Care%20Med&volume=171&publication_year=2005&pages=315&pmid=15542792&doi=10.1164/rccm.200407-894OC&)

5. Malmstrom K, Rodriguez-Gomez G, Guerra J, Villaran C, Pineiro A, Wei LX, Seidenberg BC, Reiss TF. Oral montelukast, inhaled beclomethasone, and placebo for chronic asthma: a randomized, controlled trial. Montelukast/Beclomethasone Study Group. Ann Intern Med 1999;130:487–495.  [DOI](https://doi.org/10.7326/0003-4819-130-6-199903160-00005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10075616/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Intern%20Med&volume=130&publication_year=1999&pages=487&pmid=10075616&doi=10.7326/0003-4819-130-6-199903160-00005&)

6. Israel E, Chervinsky PS, Friedman B, Van Bavel J, Skalky CS, Ghannam AF, Bird SR, Edelman JM. Effects of montelukast and beclomethasone on airway function and asthma control. J Allergy Clin Immunol 2002;110:847–854.  [DOI](https://doi.org/10.1067/mai.2002.129413) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12464949/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Allergy%20Clin%20Immunol&volume=110&publication_year=2002&pages=847&pmid=12464949&doi=10.1067/mai.2002.129413&)

7. Szefler SJ, Phillips BR, Martinez FD, Chinchilli VM, Lemanske RF, Strunk RC, Zeiger RS, Larsen G, Spahn JD, Bacharier LB, et al. Characterization of within-subject responses to fluticasone and montelukast in childhood asthma. J Allergy Clin Immunol 2005;115:233–242.  [DOI](https://doi.org/10.1016/j.jaci.2004.11.014) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15696076/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Allergy%20Clin%20Immunol&volume=115&publication_year=2005&pages=233&pmid=15696076&doi=10.1016/j.jaci.2004.11.014&)

8. Drazen JM, Silverman EK, Lee TH. Heterogeneity of therapeutic responses in asthma. Br Med Bull 2000;56:1054–1070.  [DOI](https://doi.org/10.1258/0007142001903535) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11359637/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20Med%20Bull&volume=56&publication_year=2000&pages=1054&pmid=11359637&doi=10.1258/0007142001903535&)

9. Silverman ES, Liggett SB, Gelfand EW, Rosenwasser LJ, Baron RM, Weiss ST, Drazen JM. The pharmacogenetics of asthma: a candidate gene approach. Pharmacogenomics J 2001;1:27–37.  [DOI](https://doi.org/10.1038/sj.tpj.6500005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11913724/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&volume=1&publication_year=2001&pages=27&pmid=11913724&doi=10.1038/sj.tpj.6500005&)

10. Lima JJ, Wang J. Respiratory diseases. In: Thomas S, editor. Pharmacogenomics: applications to patient care. Kansas City, MO: American College of Clinical Pharmacy; 2004. pp. 571–611.

11. Tantisira KG, Weiss ST. The pharmacogenetics of asthma: an update. Curr Opin Mol Ther 2005;7:209–217.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/15977417/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Opin%20Mol%20Ther&volume=7&publication_year=2005&pages=209&pmid=15977417&)

12. Drazen JM, Yandava CN, Dube L, Szczerback N, Hippensteel R, Pillari A, Israel E, Schork N, Silverman ES, Katz DA, et al. Pharmacogenetic association between ALOX5 promoter genotype and the response to anti-asthma treatment. Nat Genet 1999;22:168–170.  [DOI](https://doi.org/10.1038/9680) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10369259/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&volume=22&publication_year=1999&pages=168&pmid=10369259&doi=10.1038/9680&)

13. Szczeklik A, Mastalerz L, Nizankowska E, Sanak M. Montelukast for persistent asthma. Lancet 2001;358:1456–1457.  [DOI](https://doi.org/10.1016/S0140-6736(01)06499-6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11705519/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&volume=358&publication_year=2001&pages=1456&pmid=11705519&doi=10.1016/S0140-6736(01)06499-6&)

14. Sampson AP, Siddiqui S, Buchanan D, Howarth PH, Holgate ST, Holloway JW, Sayers I. Variant LTC(4) synthase allele modifies cysteinyl leukotriene synthesis in eosinophils and predicts clinical response to zafirlukast. Thorax 2000;55:S28–S31.  [DOI](https://doi.org/10.1136/thorax.55.suppl_2.S28) | [PMC free article](/articles/PMC1765955/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10992553/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Thorax&volume=55&publication_year=2000&pages=S28&pmid=10992553&doi=10.1136/thorax.55.suppl_2.S28&)

15. Asano K, Shiomi T, Hasegawa N, Nakamura H, Kudo H, Matsuzaki T, Hakuno H, Fukunaga K, Suzuki Y, Kanazawa M, et al. Leukotriene C4 synthase gene A(-444)C polymorphism and clinical response to a CYS-LT(1) antagonist, pranlukast, in Japanese patients with moderate asthma. Pharmacogenetics 2002;12:565–570.  [DOI](https://doi.org/10.1097/00008571-200210000-00009) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12360108/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&volume=12&publication_year=2002&pages=565&pmid=12360108&doi=10.1097/00008571-200210000-00009&)

16. Whelan GJ, Blake K, Kissoon N, Duckworth LJ, Wang J, Sylvester JE, Lima JJ. Effect of montelukast on time-course of exhaled nitric oxide in asthma: influence of LTC4 synthase A(-444)C polymorphism. Pediatr Pulmonol 2003;36:413–420.  [DOI](https://doi.org/10.1002/ppul.10385) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14520724/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pediatr%20Pulmonol&volume=36&publication_year=2003&pages=413&pmid=14520724&doi=10.1002/ppul.10385&)

17. Drazen JM, Israel E, O'Byrne PM. Treatment of asthma with drugs modifying the leukotriene pathway. N Engl J Med 1999;340:197–206.  [DOI](https://doi.org/10.1056/NEJM199901213400306) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9895400/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&volume=340&publication_year=1999&pages=197&pmid=9895400&doi=10.1056/NEJM199901213400306&)

18. Kanaoka Y, Boyce JA. Cysteinyl leukotrienes and their receptors: cellular distribution and function in immune and inflammatory responses. J Immunol 2004;173:1503–1510.  [DOI](https://doi.org/10.4049/jimmunol.173.3.1503) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15265876/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Immunol&volume=173&publication_year=2004&pages=1503&pmid=15265876&doi=10.4049/jimmunol.173.3.1503&)

19. Woods JW, Evans JF, Ethier D, Scott S, Vickers PJ, Hearn L, Heibein JA, Charleson S, Singer II. 5-lipoxygenase and 5-lipoxygenase-activating protein are localized in the nuclear envelope of activated human leukocytes. J Exp Med 1993;178:1935–1946.  [DOI](https://doi.org/10.1084/jem.178.6.1935) | [PMC free article](/articles/PMC2191287/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8245774/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Exp%20Med&volume=178&publication_year=1993&pages=1935&pmid=8245774&doi=10.1084/jem.178.6.1935&)

20. Dahlen SE, Hedqvist P, Hammarstrom S, Samuelsson B. Leukotrienes are potent constrictors of human bronchi. Nature 1980;288:484–486.  [DOI](https://doi.org/10.1038/288484a0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/6108512/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&volume=288&publication_year=1980&pages=484&pmid=6108512&doi=10.1038/288484a0&)

21. Lewis RA, Austen KF, Soberman RJ. Leukotrienes and other products of the 5-lipoxygenase pathway: biochemistry and relation to pathobiology in human diseases. N Engl J Med 1990;323:645–655.  [DOI](https://doi.org/10.1056/NEJM199009063231006) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2166915/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&volume=323&publication_year=1990&pages=645&pmid=2166915&doi=10.1056/NEJM199009063231006&)

22. Lam BK, Owen WF Jr, Austen KF, Soberman RJ. The identification of a distinct export step following the biosynthesis of leukotriene C4 by human eosinophils. J Biol Chem 1989;264:12885–12889.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/2753893/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&volume=264&publication_year=1989&pages=12885&pmid=2753893&)

23. Anderson ME, Allison RD, Meister A. Interconversion of leukotrienes catalyzed by purified gamma-glutamyl transpeptidase: concomitant formation of leukotriene D4 and gamma- glutamyl amino acids. Proc Natl Acad Sci USA 1982;79:1088–1091.  [DOI](https://doi.org/10.1073/pnas.79.4.1088) | [PMC free article](/articles/PMC345905/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/6122208/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20USA&volume=79&publication_year=1982&pages=1088&pmid=6122208&doi=10.1073/pnas.79.4.1088&)

24. Lee CW, Lewis RA, Corey EJ, Austen KF. Conversion of leukotriene D4 to leukotriene E4 by a dipeptidase released from the specific granule of human polymorphonuclear leucocytes. Immunology 1983;48:27–35.  [PMC free article](/articles/PMC1453997/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/6293969/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Immunology&volume=48&publication_year=1983&pages=27&pmid=6293969&)

25. Drazen JM, Austen KF. Leukotrienes and airway responses. Am Rev Respir Dis 1987;136:985–998.  [DOI](https://doi.org/10.1164/ajrccm/136.4.985) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2821857/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20Rev%20Respir%20Dis&volume=136&publication_year=1987&pages=985&pmid=2821857&doi=10.1164/ajrccm/136.4.985&)

26. Busse W, Kraft M. Cysteinyl leukotrienes in allergic inflammation: strategic target for therapy. Chest 2005;127:1312–1326.  [DOI](https://doi.org/10.1378/chest.127.4.1312) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15821210/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chest&volume=127&publication_year=2005&pages=1312&pmid=15821210&doi=10.1378/chest.127.4.1312&)

27. Figueroa DJ, Breyer RM, Defoe SK, Kargman S, Daugherty BL, Waldburger K, Liu Q, Clements M, Zeng Z, O'Neill GP, et al. Expression of the cysteinyl leukotriene 1 receptor in normal human lung and peripheral blood leukocytes. Am J Respir Crit Care Med 2001;163:226–233.  [DOI](https://doi.org/10.1164/ajrccm.163.1.2003101) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11208650/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Respir%20Crit%20Care%20Med&volume=163&publication_year=2001&pages=226&pmid=11208650&doi=10.1164/ajrccm.163.1.2003101&)

28. Revicki DA, Leidy NK, Brennan-Diemer F, Sorensen S, Togias A. Integrating patient preferences into health outcomes assessment: the multiattribute Asthma Symptom Utility Index. Chest 1998;114:998–1007.  [DOI](https://doi.org/10.1378/chest.114.4.998) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9792568/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chest&volume=114&publication_year=1998&pages=998&pmid=9792568&doi=10.1378/chest.114.4.998&)

29. Juniper EF, O'Byrne PM, Guyatt GH, Ferrie PJ, King DR. Development and validation of a questionnaire to measure asthma control. Eur Respir J 1999;14:902–907.  [DOI](https://doi.org/10.1034/j.1399-3003.1999.14d29.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10573240/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20Respir%20J&volume=14&publication_year=1999&pages=902&pmid=10573240&doi=10.1034/j.1399-3003.1999.14d29.x&)

30. Sun X, Ding H, Hung K, Guo B. A new MALDI-TOF based mini-sequencing assay for genotyping of SNPS. Nucleic Acids Res 2000;28:E68.  [DOI](https://doi.org/10.1093/nar/28.12.e68) | [PMC free article](/articles/PMC102753/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10871391/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&volume=28&publication_year=2000&pages=E68&pmid=10871391&doi=10.1093/nar/28.12.e68&)

31. Fei Z, Smith LM. Analysis of single nucleotide polymorphisms by primer extension and matrix-assisted laser desorption/ionization time-of-flight mass spectrometry. Rapid Commun Mass Spectrom 2000;14:950–959.  [DOI](https://doi.org/10.1002/(SICI)1097-0231(20000615)14:11<950::AID-RCM971>3.0.CO;2-3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10844731/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Rapid%20Commun%20Mass%20Spectrom&volume=14&publication_year=2000&pages=950&pmid=10844731&doi=10.1002/(SICI)1097-0231(20000615)14:11<950::AID-RCM971>3.0.CO;2-3&)

32. Bray MS, Boerwinkle E, Doris PA. High-throughput multiplex SNP genotyping with MALDI-TOF mass spectrometry: practice, problems and promise. Hum Mutat 2001;17:296–304.  [DOI](https://doi.org/10.1002/humu.27) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11295828/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mutat&volume=17&publication_year=2001&pages=296&pmid=11295828&doi=10.1002/humu.27&)

33. In KH, Asano K, Beier D, Grobholz J, Finn PW, Silverman EK, Silverman ES, Collins T, Fischer AR, Keith TP, et al. Naturally occurring mutations in the human 5-lipoxygenase gene promoter that modify transcription factor binding and reporter gene transcription. J Clin Invest 1997;99:1130–1137.  [DOI](https://doi.org/10.1172/JCI119241) | [PMC free article](/articles/PMC507922/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9062372/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Invest&volume=99&publication_year=1997&pages=1130&pmid=9062372&doi=10.1172/JCI119241&)

34. Fallin D, Schork NJ. Accuracy of haplotype frequency estimation for biallelic loci, via the expectation-maximization algorithm for unphased diploid genotype data. Am J Hum Genet 2000;67:947–959.  [DOI](https://doi.org/10.1086/303069) | [PMC free article](/articles/PMC1287896/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10954684/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&volume=67&publication_year=2000&pages=947&pmid=10954684&doi=10.1086/303069&)

35. Fallin D, Cohen A, Essioux L, Chumakov I, Blumenfeld M, Cohen D, Schork NJ. Genetic analysis of case/control data using estimated haplotype frequencies: application to APOE locus variation and Alzheimer's disease. Genome Res 2001;11:143–151.  [DOI](https://doi.org/10.1101/gr.148401) | [PMC free article](/articles/PMC311030/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11156623/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res&volume=11&publication_year=2001&pages=143&pmid=11156623&doi=10.1101/gr.148401&)

36. Pritchard JK, Rosenberg NA. Use of unlinked genetic markers to detect population stratification in association studies. Am J Hum Genet 1999;65:220–228.  [DOI](https://doi.org/10.1086/302449) | [PMC free article](/articles/PMC1378093/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10364535/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&volume=65&publication_year=1999&pages=220&pmid=10364535&doi=10.1086/302449&)

37. Fowler SJ, Hall IP, Wilson AM, Wheatley AP, Lipworth BJ. 5-Lipoxygenase polymorphism and in-vivo response to leukotriene receptor antagonists. Eur J Clin Pharmacol 2002;58:187–190.  [DOI](https://doi.org/10.1007/s00228-002-0458-1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12107604/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Clin%20Pharmacol&volume=58&publication_year=2002&pages=187&pmid=12107604&doi=10.1007/s00228-002-0458-1&)

38. Sayers I, Barton S, Rorke S, Sawyer J, Peng Q, Beghe B, Ye S, Keith T, Clough JB, Holloway JW, et al. Promoter polymorphism in the 5-lipoxygenase (ALOX5) and 5-lipoxygenase-activating protein (ALOX5AP) genes and asthma susceptibility in a Caucasian population. Clin Exp Allergy 2003;33:1103–1110.  [DOI](https://doi.org/10.1046/j.1365-2222.2003.01733.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12911785/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Exp%20Allergy&volume=33&publication_year=2003&pages=1103&pmid=12911785&doi=10.1046/j.1365-2222.2003.01733.x&)

39. Dwyer JH, Allayee H, Dwyer KM, Fan J, Wu H, Mar R, Lusis AJ, Mehrabian M. Arachidonate 5-lipoxygenase promoter genotype, dietary arachidonic acid, and atherosclerosis. N Engl J Med 2004;350:29–37.  [DOI](https://doi.org/10.1056/NEJMoa025079) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14702425/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&volume=350&publication_year=2004&pages=29&pmid=14702425&doi=10.1056/NEJMoa025079&)

40. Currie GP, Lima JJ, Sylvester JE, Lee DK, Cockburn WJ, Lipworth BJ. Leukotriene C4 synthase polymorphisms and responsiveness to leukotriene antagonists in asthma. Br J Clin Pharmacol 2003;56:422–426.  [DOI](https://doi.org/10.1046/j.1365-2125.2003.01952.x) | [PMC free article](/articles/PMC1884371/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12968987/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&volume=56&publication_year=2003&pages=422&pmid=12968987&doi=10.1046/j.1365-2125.2003.01952.x&)

41. Currie GP, Lee D. Uncertain biological relevance of polymorphism of leukotriene C4 synthase in asthma. J Allergy Clin Immunol 2005;115:205.  [DOI](https://doi.org/10.1016/j.jaci.2004.07.040) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15637573/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Allergy%20Clin%20Immunol&volume=115&publication_year=2005&pages=205&pmid=15637573&doi=10.1016/j.jaci.2004.07.040&)

42. Kim SH, Bae JS, Suh CH, Nahm DH, Holloway JW, Park HS. Polymorphism of tandem repeat in promoter of 5-lipoxygenase in ASA-intolerant asthma: a positive association with airway hyperresponsiveness. Allergy 2005;60:760–765.  [DOI](https://doi.org/10.1111/j.1398-9995.2005.00780.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15876305/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Allergy&volume=60&publication_year=2005&pages=760&pmid=15876305&doi=10.1111/j.1398-9995.2005.00780.x&)

43. Sanak M, Pierzchalska M, Bazan-Socha S, Szczeklik A. Enhanced expression of the leukotriene C(4) synthase due to overactive transcription of an allelic variant associated with aspirin-intolerant asthma. Am J Respir Cell Mol Biol 2000;23:290–296.  [DOI](https://doi.org/10.1165/ajrcmb.23.3.4051) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10970818/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Respir%20Cell%20Mol%20Biol&volume=23&publication_year=2000&pages=290&pmid=10970818&doi=10.1165/ajrcmb.23.3.4051&)

44. Haeggstrom JZ. Structure, function, and regulation of leukotriene A4 hydrolase. Am J Respir Crit Care Med 2000;161:S25–S31.  [DOI](https://doi.org/10.1164/ajrccm.161.supplement_1.ltta-6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10673222/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Respir%20Crit%20Care%20Med&volume=161&publication_year=2000&pages=S25&pmid=10673222&doi=10.1164/ajrccm.161.supplement_1.ltta-6&)

45. Ford-Hutchinson AW, Bray MA, Doig MV, Shipley ME, Smith MJ. Leukotriene B, a potent chemokinetic and aggregating substance released from polymorphonuclear leukocytes. Nature 1980;286:264–265.  [DOI](https://doi.org/10.1038/286264a0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/6250050/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&volume=286&publication_year=1980&pages=264&pmid=6250050&doi=10.1038/286264a0&)

46. Jala VR, Haribabu B. Leukotrienes and atherosclerosis: new roles for old mediators. Trends Immunol 2004;25:315–322.  [DOI](https://doi.org/10.1016/j.it.2004.04.003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15145321/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Trends%20Immunol&volume=25&publication_year=2004&pages=315&pmid=15145321&doi=10.1016/j.it.2004.04.003&)

47. Kawabe T, Chen ZS, Wada M, Uchiumi T, Ono M, Akiyama S, Kuwano M. Enhanced transport of anticancer agents and leukotriene C4 by the human canalicular multispecific organic anion transporter (cMOAT/MRP2). FEBS Lett 1999;456:327–331.  [DOI](https://doi.org/10.1016/s0014-5793(99)00979-5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10456333/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=FEBS%20Lett&volume=456&publication_year=1999&pages=327&pmid=10456333&doi=10.1016/s0014-5793(99)00979-5&)

48. Brechot JM, Hurbain I, Fajac A, Daty N, Bernaudin JF. Different pattern of MRP localization in ciliated and basal cells from human bronchial epithelium. J Histochem Cytochem 1998;46:513–517.  [DOI](https://doi.org/10.1177/002215549804600411) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9524197/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Histochem%20Cytochem&volume=46&publication_year=1998&pages=513&pmid=9524197&doi=10.1177/002215549804600411&)

49. Saito S, Iida A, Sekine A, Miura Y, Ogawa C, Kawauchi S, Higuchi S, Nakamura Y. Identification of 779 genetic variations in eight genes encoding members of the ATP-binding cassette, subfamily C (ABCC/MRP/CFTR). J Hum Genet 2002;47:147–171.  [DOI](https://doi.org/10.1007/s100380200018) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12166651/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Hum%20Genet&volume=47&publication_year=2002&pages=147&pmid=12166651&doi=10.1007/s100380200018&)

50. van der Deen M, de Vries EG, Timens W, Scheper RJ, Timmer-Bosscha H, Postma DS. ATP-binding cassette (ABC) transporters in normal and pathological lung. Respir Res 2005;6:59.  [DOI](https://doi.org/10.1186/1465-9921-6-59) | [PMC free article](/articles/PMC1200430/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15967026/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Respir%20Res&volume=6&publication_year=2005&pages=59&pmid=15967026&doi=10.1186/1465-9921-6-59&)

51. Ito S, Ieiri I, Tanabe M, Suzuki A, Higuchi S, Otsubo K. Polymorphism of the ABC transporter genes, MDR1, MRP1 and MRP2/cMOAT, in healthy Japanese subjects. Pharmacogenetics 2001;11:175–184.  [DOI](https://doi.org/10.1097/00008571-200103000-00008) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11266082/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&volume=11&publication_year=2001&pages=175&pmid=11266082&doi=10.1097/00008571-200103000-00008&)

52. Hirschhorn JN, Daly MJ. Genome-wide association studies for common diseases and complex traits. Nat Rev Genet 2005;6:95–108.  [DOI](https://doi.org/10.1038/nrg1521) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15716906/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Genet&volume=6&publication_year=2005&pages=95&pmid=15716906&doi=10.1038/nrg1521&)
