# Glutathione Pathway Genetic Polymorphisms and Lung Cancer Survival After Platinum-Based Chemotherapy

## Metadata
**Authors:** Ann M Moyer, Zhifu Sun, Anthony J Batzler, Liang Li, Daniel J Schaid, Ping Yang, Richard M Weinshilboum
**Journal:** Cancer epidemiology, biomarkers & prevention : a publication of the American Association for Cancer Research, cosponsored by the American Society of Preventive Oncology
**Date:** 2010 Mar 3
**DOI:** [10.1158/1055-9965.EPI-09-0871](https://doi.org/10.1158/1055-9965.EPI-09-0871)
**PMID:** 20200426
**PMCID:** PMC2837367
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2837367/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC2837367/pdf/nihms171463.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC2837367/pdf/nihms171463.pdf)

## Abstract

**Background:** 
Lung cancer is commonly treated with platinum compounds. The “glutathione pathway” participates in the metabolism of platinum compounds. We set out to test the hypotheses that single nucleotide polymorphisms (SNPs) or copy number variation for genes within the glutathione pathway might influence survival in lung cancer patients treated with these drugs.

**Methods:** 
Germline DNA samples from 973 lung cancer patients were genotyped for 290 glutathione pathway SNPs. GSTT1 copy number was also assayed. We determined the association of these polymorphisms with survival for lung cancer patients, followed by functional genomic validation.

**Results:** 
We observed suggestive associations between survival and GSTT1 copy number (p=0.017), and GSTA5, GSTM4, and ABCC4 SNPs, adjusted for covariates (p=0.018, 0.002, 0.002, respectively) or not (p=0.005, 0.011, 0.002). One hundred lymphoblastoid cell lines were then treated with cisplatin, and IC50 values were significantly associated with the GSTM4 SNP (p=0.019). Furthermore, GSTM4, GSTT1, and ABCC4 overexpression significantly decreased cisplatin sensitivity in lung cancer and HEK293T cell lines.

**Conclusions:** 
These results suggest that GSTM4 polymorphisms are biomarkers for the prediction of cisplatin response. ABCC4 polymorphisms, as well as GSTT1 copy number, may also help to predict cisplatin response, but further validation is required.

**Impact:** 
These results represent a step toward the individualized chemotherapy of lung cancer.

Keywords: pharmacogenomics, pharmacogenetics, single nucleotide polymorphism, copy number variant, lung cancer, cisplatin, glutathione S-transferase, glutathione pathway, platinum antineoplastic, GSTT1, GSTM4, GSTA5, ABCC4

### Background

Lung cancer is commonly treated with platinum compounds. The “glutathione pathway” participates in the metabolism of platinum compounds. We set out to test the hypotheses that single nucleotide polymorphisms (SNPs) or copy number variation for genes within the glutathione pathway might influence survival in lung cancer patients treated with these drugs.

### Methods

Germline DNA samples from 973 lung cancer patients were genotyped for 290 glutathione pathway SNPs. GSTT1 copy number was also assayed. We determined the association of these polymorphisms with survival for lung cancer patients, followed by functional genomic validation.

### Results

We observed suggestive associations between survival and *GSTT1* copy number (p=0.017), and *GSTA5*, *GSTM4*, and *ABCC4* SNPs, adjusted for covariates (p=0.018, 0.002, 0.002, respectively) or not (p=0.005, 0.011, 0.002). One hundred lymphoblastoid cell lines were then treated with cisplatin, and IC_50_ values were significantly associated with the *GSTM4* SNP (p=0.019). Furthermore, *GSTM4, GSTT1*, and *ABCC4* overexpression significantly decreased cisplatin sensitivity in lung cancer and HEK293T cell lines.

### Conclusions

These results suggest that *GSTM4* polymorphisms are biomarkers for the prediction of cisplatin response. *ABCC4* polymorphisms, as well as *GSTT1* copy number, may also help to predict cisplatin response, but further validation is required.

### Impact

These results represent a step toward the individualized chemotherapy of lung cancer.

## Introduction

Platinum compounds are widely used in the treatment of many forms of cancer and they are a mainstay in the therapy of lung, testicular, ovarian, head and neck, and bladder cancer ([1](#R1)-[3](#R3)). However, there are large inter-individual variations in response to treatment with these agents, with response rates of only 30% for cisplatin combination chemotherapy for lung cancer ([4](#R4)). After platinum chemotherapeutic agents enter the cell and their chloride leaving groups are aquated, they exert their cytotoxic effects by covalently binding DNA purine bases and forming adducts ([5](#R5)-[8](#R8)). If unrecognized by the DNA damage recognition and repair machinery, these adducts can interfere with DNA replication and result in mutations or cell death via apoptosis or necrosis ([9](#R9), [10](#R10)). When used in therapeutic doses, cisplatin can cause significant nephrotoxicity and neurotoxicity, both of which can be severe enough to require discontinuation of therapy.

Platinum compounds can be inactivated by glutathione conjugation. The “glutathione pathway” ([Figure 1](#F1)), which includes enzymes responsible for glutathione synthesis and redox status, as well as glutathione *S*-transferases and transporters that remove glutathione conjugates from cells, is highly genetically polymorphic. This pathway has been implicated in cellular resistance to cancer chemotherapy as a result of the glutathione conjugation of active metabolites, followed by the excretion of glutathione conjugates ([11](#R11)-[13](#R13)).

### Figure 1. The “Glutathione Pathway”.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2cbf/2837367/777ee30d1162/nihms171463f1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=2837367_nihms171463f1.jpg)

The figure shows a schematic representation of the “glutathione pathway.” Glutathione (GSH) is synthesized from glutamate, cysteine, and glycine by γ-glutamylcysteine synthetase and glutathione synthetase. Glutathione redox state is regulated, in part, by glutathione peroxidases, forming oxidized glutathione (GSSG), and by a reaction catalyzed by glutathione reductase. Glutathione is conjugated to substrates both through the action of the glutathione S-transferases and through non-enzymatic reactions. Glutathione conjugates can be excreted from the cells by members of the ABCC transporter family.

Lung cancer is the leading cause of cancer death in both men and women ([14](#R14)). The prognosis for this disease is poor because of late stage tumor detection, a narrow therapeutic index for drugs used in its treatment, and lack of algorithms to predict the best drug and dose for individual patients ([15](#R15)). We set out to study the possible contribution of glutathione pathway pharmacogenomics to variation in response to platinum therapy of patients with lung cancer – followed by functional validation of candidate genes identified during the clinical study using cell-based model systems. Specifically, we genotyped germline DNA from 973 patients with lung cancer for 290 glutathione pathway SNPs ([Table 1](#T1)), as well as number of copies of the *GSTT1* and *GSTM1* genes. We will use the term “copy number polymorphism” (CNP) subsequently to refer to these gene deletion polymorphisms in *GSTT1* and *GSTM1*. We then evaluated the effect of candidate SNPs identified during the clinical association study on cisplatin toxicity using a “Human Variation Panel” cell line model system. Finally, we also determined the effect of altering the expression of the candidate genes identified during the clinical study on cisplatin toxicity in cancer cell lines. This series of clinical and complementary laboratory-based functional studies identified several candidate SNPs and genes that might be useful biomarkers in predicting both platinum compound toxicity and patient survival after the treatment of lung cancer with platinum-based antineoplastic agents.

### Table 1.

| Gene Name | Panel SNPs | SNPs Analyzed |
| --- | --- | --- |
|   |  |  |
| ABCC1 | 29 | 28 |
| ABCC2 | 17 | 14 |
| ABCC3 | 15 | 13 |
| ABCC4 | 68 | 65 |
| GCLC | 16 | 16 |
| GCLM | 3 | 3 |
| GPX1 | 3 | 1 |
| GPX2 | 3 | 3 |
| GPX3 | 6 | 5 |
| GPX5 | 3 | 3 |
| GPX6 | 3 | 3 |
| GPX7 | 2 | 2 |
| GSR | 7 | 6 |
| GSS | 7 | 7 |
| GSTA1 | 1 | 1 |
| GSTA2 | 5 | 4 |
| GSTA3 | 6 | 5 |
| GSTA4 | 10 | 8 |
| GSTA5 | 8 | 6 |
| GSTM1 | 10 | 5 |
| GSTM2 | 2 | 1 |
| GSTM3 | 3 | 3 |
| GSTM4 | 5 | 4 |
| GSTM5 | 7 | 4 |
| GSTO1 | 8 | 6 |
| GSTO2 | 12 | 9 |
| GSTP1 | 12 | 8 |
| GSTT1 | 7 | 7 |
| GSTT2 | 0 | 0 |
| GSTZ1 | 12 | 11 |
|   |  |  |
| Total | 290 | 251 |

Table 1 Caption: Glutathione Pathway panel genes. “Panel SNPs” indicates the number of SNPs in each gene originally selected for genotyping for each gene, while “SNPs Analyzed” indicates the number of SNPs that passed all quality control checks and were included in the final analysis.

## Materials and Methods

### Patient Samples

The study group included 973 patients with pathologically confirmed primary lung cancer, including both small cell and non-small cell lung cancer, who were treated with platinum-based chemotherapy. Study subjects were Caucasian patients who were consecutively enrolled in this protocol from a population of patients diagnosed and/or treated at the Mayo Clinic (Rochester, MN) between 1997 and 2006. Details with regard to clinical characteristics of patients, patient enrollment, and data collection procedures were described previously ([16](#R16)-[18](#R18)). Briefly, each case was identified through the Mayo Clinic pathologic diagnostic (Co-Path) system. After written informed consent had been obtained, patient medical records were abstracted by a trained nurse and a blood sample was collected. Vital status and cause of death were determined by reviewing the Mayo Clinic registration database and medical records, correspondence from patients' next-of-kin, death certificates, obituary documents, the Mayo Clinic Tumor Registry, and the Social Security Death Index website. Additional patient information was collected with a mail-in questionnaire sent to participants or to their next-of-kin. As of June 2007 (when patients were last followed and the data were “frozen” for analysis), 694 were deceased and 279 were alive. All patients provided written informed consent for their participation in this study, and the study was reviewed and approved by the Mayo Clinic Institutional Review Board.

### Cell Lines

“Human Variation Panel” lymphoblastoid cell lines from sample set HD 100CAU, corresponding to 100 Caucasian-American subjects, were obtained from the Coriell Cell Repository (Camden, NJ). The National Institute of General Medical Sciences had obtained and anonymized these cell lines before deposit, and all subjects had provided written informed consent for the use of their samples for research purposes. Use of these cell lines for research purposes was reviewed and approved by the Mayo Clinic Institutional Review Board.

Human lung adenocarcinoma cell lines CRL-5872 and CRL-5985 were obtained from the ATCC (Manassas, VA). HEK293T cell lines stably expressing *ABCC4* were a generous gift from Dr. Piet Borst of The Netherlands Cancer Institute ([19](#R19)).

### Genotyping

The Illumina Golden Gate platform was used to genotype 290 SNPs within genes encoding proteins in the glutathione pathway ([Figure 1](#F1), [Table 1](#T1)). SNPs were selected on the basis of in-depth resequencing studies ([20](#R20)-[22](#R22)) for genes previously resequenced in our laboratory and/or HapMap data to include all other genes in the glutathione pathway. Specifically, SNPs in *GSTT1*, *GSTM1*, *GSTP1*, *GSTO1*, and *GSTO2* were based on in-depth resequencing studies and were selected by use of a haplotype-tagging algorithm ([20](#R20)-[25](#R25)). We also genotyped all nonsynonymous SNPs found in these genes in the Caucasian-American population. Haploview was used to select additional HapMap tag SNPs in these and in other genes studied. Specifically, data from the Caucasian (CEU) HapMap population were used to select SNPs with the following parameters: ignore pairwise comparisons of markers >500 kb apart, exclude individuals with >50% missing genotypes, Hardy-Weinberg p-value cutoff=0.001, minimum genotype=75%, maximum number of Mendelian errors=1, Minimum minor allele frequency=0.001, aggressive tagging, r^2^ threshold=0.8, LOD threshold for multi-marker tests=3.0. In order to obtain *GSTT1* and *GSTM1* CNP information, fluorescent-based fragment analysis was performed as described previously ([20](#R20)).

The “Human Variation Panel” lymphoblastoid cell line DNA was obtained from the Coriell Cell Repository. This DNA was genotyped with Illumina HumanHap550k BeadChips. However, only data for rs4715354, rs560018, and rs7984157 (SNPs for glutathione pathway genes identified during the clinical study) were used in this study. *GSTT1* copy number data for these same cell lines had been obtained previously ([20](#R20)).

### Cisplatin Cytotoxicity

Cisplatin was obtained from Sigma-Aldrich (St. Louis, MO) and was dissolved in DMSO immediately prior to use. Cells were incubated in cisplatin for either 72 hours (lymphoblastoid cells and HEK293) or 120 hours (lung cancer cell lines) in the presence of 8 drug concentrations ranging either from 0.1 to 80 μM (lymphoblastoid and HEK293T cells) or 0.3 to 320 μM (lung cancer cell lines) – all with DMSO concentrations of less than 0.1% in a dark incubator. The cytotoxic effect of cisplatin was evaluated by determining the concentration of cisplatin required to inhibit growth and/or survival by 50% (IC_50_) using the CellTiter Blue (Promega, Madison, WI) assay to perform the cytotoxicity assays.

### Transient Overexpression in Cell Lines

To assess the effect of increased gene expression on cisplatin cytotoxicity in lung adenocarcinoma, *GSTT1*, *GSTM4*, and *GSTA5* were transiently overexpressed in the CRL-5872 and CRL-5985 cell lines. Expression constructs for the wild type cDNA were either obtained from OriGene (Rockville, MD) or were created as described previously ([20](#R20)). Expression constructs or empty vector were transfected into the cell lines using the TransFast reagent (Promega, Madison, WI). After 24 hours, the cells were treated with cisplatin as described for the cytotoxicity experiments and data for cells transfected with expression constructs were compared with results obtained after transfection with “empty” vector.

### Data Analysis

A total of 251 glutathione pathway SNPs were included in the analysis for the clinical association study (see [Table 1](#T1)), as well as *GSTT1* and *GSTM1* copy number. SNPs were excluded on the basis of genotyping failure, ambiguous “clustering”, monomorphic genotyping, minor allele frequency less than 0.01, or significant departures from Hardy-Weinberg Equilibrium (p<0.001). SNPs that had call rates >95% and passed all other quality control checks were included in the study. Associations of SNP genotypes with overall survival were analyzed by the Cox proportional hazards model, using the common homozygote as baseline, to contrast with heterozygotes and homozygotes for the rare allele. For markers with three genotypes, score tests with 2 degrees of freedom (df) were used, while 1 df tests were used if no homozygotes for the rare allele were observed. The associations of copy number for the *GSTT1* and *GSTM1* genes were analyzed in a similar fashion, except the copy number of 0 was used as baseline, to contrast with subjects with 1 or with 2 copies. To correct for multiple testing of the 251 SNPs assayed in the initial experiment, the Bonferroni corrected p-value threshold of 0.0002 was used to determine statistically significant associations. To determine whether associations with SNPs should be adjusted for the clinical covariates of age at diagnosis, gender, smoking status, disease stage and treatment, backward selection was performed. Here stage was divided into five categories: small cell lung cancer with stages limited versus extensive; non-small cell lung cancer with stages I+II, versus III versus IV.

IC_50_ values were obtained from the cisplatin cytotoxicity data using GraphPad Prism 4.03 to fit the data to a sigmoidal dose-response curve, with no constraints. Correlations of the log transformed cisplatin IC_50_ values obtained from the “Human Variation Panel” cell-based model system, and SNP and CNP genotypes were assessed with Pearson correlations and ANOVA. For the overexpression experiments, group mean values were compared using Student's *t* test. Because the associations of SNPs and CNP with survival were used to screen out the variants most likely to have functional importance, and because only a small number of SNPs/CNPs were pursued with functional studies, the associations of IC50 values with SNPs/CNPs were determined to be statistically significant with a p-value threshold of 0.05.

## Results

### Survival Analyses for Glutathione Pathway Genetic Variation and Lung Cancer Patient Samples

Overall survival and genotype data for 973 lung cancer patients were included in this study. Basic demographic and clinical data with regard to the patients studied are listed in [Table 2](#T2). At last follow-up, 694 of these patients were deceased and 279 were alive. Four covariates – disease stage (which also took into account non-small cell vs. small cell), treatment, age at diagnosis, and gender – were statistically significant predictors of survival, each adjusted for the others. Smoking history was not significant after adjusting for the other covariates (p=0.15), so our results were not adjusted for smoking history. Three tag SNPs, one each in *GSTA5* (rs4715354), *GSTM4* (rs560018), and *ABCC4* (rs7984157), were associated with survival, whether adjusted for the covariates of stage, treatment, gender and age at diagnosis (p=0.018, 0.002, 0.002, respectively) or not (p=0.005, 0.011, 0.002) ([Figure 2](#F2), [Table 3](#T3)), although these SNPs were not statistically significant after correcting for multiple testing. Similar data for all of the polymorphisms studied are listed in [Supplementary Table 1](#SD1). We should point out that the associations for *GSTM4* and *GSTA5* both seemed to be “driven” by the heterozygous samples – a trend that we also noticed in the cell line-based validation studies described subsequently. The biological explanation for these observations is not obvious. *GSTT1* copy number was associated with survival before adjustment for covariates (p=0.017), but the strength of the association decreased after adjusting for all covariates (p=0.079) ([Figure 2](#F2)).

### Table 2.

| Characteristics at the Diagnosis and Treatment | Values and Percentages |
| --- | --- |
| Age at Diagnosis |   |
| Mean (standard deviation) | 62.1 (10.4) |
| Median (range) | 63.0 (34.0-93.0) |
| Gender, N (%) |   |
| Female | 439 (45.1%) |
| Male | 534 (54.9%) |
| Cigarette smoking status |   |
| Non-smokers | 155 (15.9%) |
| Smokers | 818 (84.1%) |
| Lung cancer stage |   |
| Unknown | 10 (1.0%) |
| SCLC: LIMITED | 103 (10.7%) |
| EXTENSIVE | 68 (7.1%) |
| NSCLC: I | 96 (9.9%) |
| II | 65 (6.7%) |
| III | 340 (35.3%) |
| IV | 291 (30.2%) |
| Histologic cell type |   |
| Adenocarcinoma | 443 (45.5%) |
| Squamous cell carcinoma | 169 (17.4%) |
| Small cell carcinoma | 172 (17.7%) |
| Large cell carcinoma | 30 (3.1%) |
| Mixed and unspecified NSCLC | 152 (15.5%) |
| Carcinoid or salivary gland tumors | 7 (0.7%) |
| Tumor differentiation grade |   |
| Nongradable or unknown | 76 (7.8%) |
| Well Differentiated | 75 (7.7%) |
| Moderately Differentiated | 319 (32.8%) |
| Poor/Undifferentiated | 503 (51.7%) |
| Treatment Modality |   |
| Only platinum drugs | 334 (34.3%) |
| Surgery & platinum drugs | 189 (19.4%) |
| Radiation & platinum drugs | 285 (29.3%) |
| Surgery & radiation & platinum drugs | 165 (17.0%) |

Table 2 Caption: Description of 973 patients who were diagnosed with primary lung cancer and received platinum-based drug therapy, Mayo Clinic, 1997-2006. Stage for NSCLC is described on the basis of the TNM classification. The SCLC staged is described as suggested by the American Cancer Society (48) as either “EXTENSIVE” or “LIMITED”.

### Figure 2. Lung Cancer Patient Survival Curves.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2cbf/2837367/f14f19123096/nihms171463f2.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=2837367_nihms171463f2.jpg)

Kaplan-Meier survival curves for lung cancer patients are shown for the GSTT1 CNP and for SNPs in GSTM4, GSTA5, and ABCC4. The x-axis on each graph indicates years since lung cancer diagnosis, while the y-axis indicates the percentage of patients surviving. The p-values unadjusted for covariates (p) and after adjustment for covariates (p adj) are indicated for each variant. For GSTT1, 0, 1, and 2 indicate the number of copies of GSTT1. For GSTM4, GSTA5, and ABCC4, CC indicates two copies of the common allele, CR refers to heterozygotes, and RR indicates patients having two copies of the rare allele.

### Table 3.

| Polymorphism | Gene | p-value | p-value adj | CR Hazard Ratio | RR Hazard Ratio | MAF |
| --- | --- | --- | --- | --- | --- | --- |
|   |  |  |  |  |  |  |
| RS7984157 | ABCC4 | 0.0017 | 0.0018 | 0.90 (0.76-1.07) | 2.02 (1.30-3.13) | 0.164 |
| RS4715354 | GSTA5 | 0.0047 | 0.0182 | 1.26 (1.06-1.51) | 0.97 (0.78-1.20) | 0.484 |
| RS560018 | GSTM4 | 0.0105 | 0.0021 | 1.25 (1.07-1.48) | 0.99 (0.79-1.24) | 0.376 |
| RS1332018 | GSTM3 | 0.0126 | 0.0617 | 1.28 (1.08-1.52) | 1.10 (0.88-1.36) | 0.427 |
| RS943288 | ABCC4 | 0.0160 | 0.0278 | 0.79 (0.66-0.94) | 1.29 (0.80-2.09) | 0.136 |
| RS4925 | GSTO1 | 0.0227 | 0.0872 | 0.94 (0.80-1.11) | 1.31 (1.04-1.64) | 0.322 |
| RS1564351 | ABCC4 | 0.0230 | 0.0104 | 1.29 (1.07-1.56) | 1.41 (0.67-2.98) | 0.096 |
| RS212082 | ABCC1 | 0.0245 | 0.0366 | 1.09 (0.92-1.28) | 0.50 (0.28-0.88) | 0.168 |
| RS152023 | ABCC1 | 0.0296 | 0.0419 | 1.06 (0.91-1.24) | 1.40 (1.09-1.80) | 0.325 |
| RS1800668 | GPX1 | 0.0309 | 0.0162 | 0.82 (0.70-0.97) | 0.80 (0.61-1.03) | 0.303 |
| RS11597282 | ABCC2 | 0.0324 | 0.0016 | 1.43 (1.07-1.92) | 2.55 (0.36-18.18) | 0.032 |
| RS2164624 | GSTO1 | 0.0339 | 0.1274 | 0.91 (0.77-1.07) | 1.22 (0.98-1.52) | 0.346 |
| GSTO2_1 | GSTO2 | 0.0401 | 0.0702 | 0.94 (0.79-1.10) | 1.28 (1.01-1.61) | 0.315 |
| RS1189446 | ABCC4 | 0.0490 | 0.0351 | 1.16 (0.99-1.37) | 0.70 (0.43-1.14) | 0.178 |
| RS9370155 | GSTA5 | 0.0506 | 0.5660 | 1.14 (0.95-1.36) | 1.79 (1.03-3.10) | 0.122 |
| RS929166 | GSTM5 | 0.0524 | 0.1614 | 0.92 (0.79-1.08) | 0.70 (0.52-0.94) | 0.273 |
| RS7483 | GSTM3 | 0.0536 | 0.0761 | 0.88 (0.75-1.02) | 0.74 (0.55-0.98) | 0.289 |
| RS4636781 | ABCC4 | 0.0544 | 0.1123 | 0.81 (0.68-0.96) | 1.00 (0.66-1.50) | 0.168 |
| RS2274405 | ABCC4 | 0.0572 | 0.1625 | 0.83 (0.71-0.97) | 0.85 (0.66-1.09) | 0.325 |
| RS2889517 | ABCC1 | 0.0602 | 0.3203 | 1.09 (0.93-1.27) | 1.36 (1.05-1.75) | 0.302 |

Table 3 Caption: The 20 SNPs with lowest unadjusted p-values associated with the survival of lung cancer patients treated with platinum compounds are listed. The gene in which the SNP is located is indicated as well as the p-value unadjusted for covariates (p-value), the p-value adjusted for covariates (p-value Adj), hazard ratio of one rare SNP allele with confidence intervals (CR Hazard Ratio), and hazard ratio of two rare SNP alleles with confidence intervals (RR Hazard Ratio).

Adjusting for disease stage had the greatest impact on the p-value for *GSTT1* copy number, primarily because 63% of patients with limited stage disease had less than two copies of *GSTT1*, while only 56% of the patients with more advanced disease stage were homozygous or heterozygous for the *GSTT1* gene deletion. The correlation of *GSTT1* copy number with stage could result from a contribution of *GSTT1* copy number to disease severity in terms of disease stage, in which case it would not be appropriate to adjust for stage. When we adjusted only for treatment, sex, and age, the association of *GSTT1* copy number with survival did not diminish (p=0.039) compared to when stage was adjusted. Compared to subjects with no copies of *GSTT1*, subjects with a single copy had a 21% increased risk of death (Hazard ratio, [HR] 95% confidence interval, CI=0.96-1.54) and subjects with two copies of *GSTT1* had a 40% increased risk of death (HR 95% CI=1.10-1.76). When *GSTT1* copy number was included with the three tag SNPs that we had identified in a Cox regression model together with treatment, age, and sex, subjects with two copies of *GSTT1* remained at increased risk of death (p=0.03; HR=1.31, 95% CI=1.03-11.66); subjects heterozygous for the rare SNP allele in *GSTA5* were at increased risk (p=0.009, HR 1.23, 95% CI=1.05-1.42); subjects heterozygous for the rare SNP allele in *GSTM4* were at increased risk (p=0.001, HR=1.29, 95% CI=1.11-1.51), and subjects homozygous for the rare SNP allele in *ABCC4* were also at increased risk (p=0.0001, HR=2.35, 95% CI=1.51-3.65). These results suggest that *GSTT1* copy number and the three tag SNPs contribute independent effects to the risk for death, supporting a polygenic mechanism.

### SNP-IC50 Correlation Analysis in Lymphoblastoid Cell Lines

The three SNPs identified during the patient survival analysis, as well as the *GSTT1* CNP, were next pursued by *in vitro* functional validation studies performed with a cell-based model system, “Human Variation Panel” lymphoblastoid cells obtained from 100 healthy Caucasian-American subjects. Cells obtained from Caucasian-American subjects were selected to correspond to the Caucasian patients in the clinical association study. This “model system” has been used to generate and test pharmacogenetic hypotheses for other antineoplastic drugs ([26](#R26)). Specifically, these 100 cell lines were used to perform cytotoxicity studies with cisplatin so the association of SNPs with IC_50_ could be determined. In this system, rs560018 (the SNP in *GSTM4*) was significantly associated with IC_50_, p=0.019 ([Figure 3](#F3)), but rs4715354 in *GSTA5* and rs7984157 in *ABCC4* were not significantly associated with cisplatin IC_50_ (p=0.582 and 0.598, respectively) ([Figure 3](#F3)). There was also not a statistically significant association between *GSTT1* copy number and IC_50_ (p=0.509) in this cell line model system ([Figure 3](#F3)). We next utilized cancer cell lines to perform cytotoxicity experiments as a second cell line-based functional validation.

### Figure 3. Association of SNPs and Cisplatin IC50 Values in Lymphoblastoid Cells.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2cbf/2837367/b7db87316022/nihms171463f3.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=2837367_nihms171463f3.jpg)

The plots show log cisplatin log IC50 values by copy number or SNP genotype in 100 lymphoblastoid cell lines from Caucasian-American subjects. Each dot represents an individual sample. The green line across the center of each diamond represents the group mean, while the vertical span of each diamond represents the 95% confidence interval for each group. Mean values and standard deviation lines are shown as blue lines. The p-value for each ANOVA is listed at the bottom left of the plot.

### Transient Overexpression in Cancer Cell Lines

SNPs have been shown to influence the activity of enzymes and the function of transporters by either altering the quantity of protein as a result of an effect on transcription or protein synthesis/degradation, or as a result of an effect on “active sites” ([27](#R27), [28](#R28)). The SNPs identified in our clinical association study were located in introns, making the mechanism by which they may be associated with clinical outcome unclear. They might be linked to other functional SNPs or they might alter promoter elements – found in introns approximately one-third of the time ([29](#R29)) – thus affecting transcription and the level of enzyme or transporter expressed. Without resequencing each gene in its entirety, it is impossible to decipher whether the SNPs that we identified were the functional SNPs or if they are linked to other, functionally important SNPs. Therefore, we attempted to determine the effect of altered level of the enzymes and transporter identified during our clinical association study on cisplatin cytotoxicity in cancer cell lines as a functional approach to the validation of the possible impact of these and/or linked SNPs. Specifically, we overexpressed the genes in which the SNPs were found in either lung cancer or in HEK293T cells. Adenocarcinoma cell lines were selected because adenocarcinoma represented the largest percentage of patients in our clinical association study. We found that, compared to IC_50_ values for CRL-5872 and CRL-5985 lung cancer cells transfected with empty vector as a control, there were significant changes in IC_50_ values, with increased resistance after the overexpression of GSTM4 (159±18.9%, p=0.014 and 170±30.7%, p=0.050, respectively) and GSTT1 (161±17.5%, p=0.017 and 144±5.0%, p=0.013, respectively) ([Figure 4](#F4)). Overexpression of GSTA5 did not result in significant alterations of IC_50_ values in either cell line (95.7±6.3%, p=0.383, and 141±25.5%, p=0.188) ([Figure 4](#F4)). ABCC4 overexpression was only tested in HEK293T cell lines. The two clones studied, both stably overexpressing ABCC4, displayed IC_50_ values that increased slightly in the direction of increased resistance (118±3.3%, p=0.058, and 119±3.6%, p=0.035) ([Figure 4](#F4)).

### Figure 4. Effect Overexpression on Cisplatin IC50.

![Figure 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2cbf/2837367/2fc255f7dbe7/nihms171463f4.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=2837367_nihms171463f4.jpg)

Relative cisplatin IC50 values, after overexpression of the gene indicated, are shown in the cell lines indicated. The IC50 value after transfection with the empty vector control for each experiment was defined as 100%, and IC50 values are reported as a percentage of that for the control. All experiments were performed at least six times. * indicates p<0.05

## Discussion

Platinum-containing agents are a mainstay in the chemotherapy of lung cancer. Cisplatin response and toxicity both exhibit inter-individual variation. That is due, in part, to multiple mechanisms for resistance to these compounds, including reduced drug accumulation, enhanced DNA repair, increased drug detoxification, and/or increased removal of glutathione-conjugates from the cell ([30](#R30)-[33](#R33)). Glutathione conjugation is one mechanism by which platinum compounds can be detoxified either by the conjugation of free cisplatin or by binding DNA-platinum mono-adducts, thus preventing cytotoxic DNA crosslink formation ([31](#R31), [34](#R34), [35](#R35)). In the present study, we evaluated the possible contribution of genetic variation in the “glutathione pathway” to differences in response to platinum antineoplastic agents by performing an epidemiologic genotype-phenotype association study of lung cancer patient survival after treatment with platinum-containing agents. We then used two different cell line-based approaches in an attempt to biologically validate the results of our clinical association study – lymphoblastoid cell lines and complementary tumor cell line experiments. Specifically, we studied SNPs with the lowest p-values observed during the clinical association study using a lymphoblastoid cell-based model system, and by evaluating the effect on cisplatin IC_50_ of altered expression of genes in which these SNPs were found in lung cancer and HEK293T cell lines.

The clinical association study presented here utilized a patient cohort from a single institution. All lung cancer patients treated with a platinum compound were included in our study in an attempt to be representative of the general population of patients with lung cancer. It should be emphasized that germline mutations have been shown to be of great importance in pharmacogenomics. In addition, although non-small cell and small cell lung cancer have many differences, both cancers are treated very similarly with platinum agents ([36](#R36), [37](#R37)). Including both non-small cell and small cell lung cancer patients allowed us to have greater power when studying the possible effects of germline mutations on patient survival after adjusting for disease stage—including histologic type and severity of lung cancer.

*GSTT1* copy number polymorphism and three SNPs, one each in *GSTM4, GSTA5*, and *ABCC4*, were associated with survival in lung cancer patients treated with platinum-containing agents, although none of these associations achieved statistical significance when we accounted for all 251 SNPs evaluated. *GSTT1*, *GSTM4*, and *GSTA5* are all glutathione *S*-transferase enzymes that can catalyze the conjugation of glutathione to reactive electrophiles, while *ABCC4* encodes a cell membrane transporter, MRP4, that contributes to drug efflux from cells ([38](#R38)). The *GSTT1* CNP has been reported previously to be associated with response to cisplatin in combination with other genotypes ([15](#R15), [39](#R39)). In the present study, we found that this association appeared to be directly related to *GSTT1* copy number. That is not surprising, since we have previously shown that GSTT1 expression increases with increasing copies of *GSTT1* ([20](#R20)). Therefore, it is likely that the *GSTT1* CNP may be associated with outcome as a result of increased levels of GSTT1 enzyme and, therefore, increased GSTT1-mediated conjugation. Obviously, failure to observe significant associations with the phenotype of interest, survival, for the other genes included in our study does not rule out a significant association for those genes if additional SNPs had been studied.

The SNP in *GSTM4* that we identified was associated with both the survival of lung cancer patients treated with platinum-containing agents and with cisplatin IC_50_ in the lymphoblastoid cell line-based model system. Overexpression of GSTM4 resulted in a greater than 50% change in IC_50_ in a direction indicating increased resistance in both lung cancer cell lines exposed to cisplatin. While this was a robust association and, to our knowledge, is the first time that a specific *GSTM4* SNP has been reported to play a role in cisplatin resistance, this is not the first time that GSTM4 has been implicated as a factor in cisplatin response. When the expression pattern of an MCF-7 cisplatin-resistant derivative cell line was compared to the parental cell line, GSTM4 was one of 28 genes found to be differentially expressed between the two cell lines ([40](#R40)). Furthermore, a different SNP in *GSTM4*, rs650985, has also been associated with risk for the development of lung cancer ([41](#R41)) and that polymorphism is in linkage disequilibrium with the SNP identified in our study (D′=1.0) ([Figure 5](#F5)). Obviously, future studies will be required to determine the possible role(s) of *GSTM4* in lung cancer carcinogenesis and treatment response – although those studies would be technically challenging because of the high degree of sequence similarity among genes encoding GSTM family members.

### Figure 5. GSTM4 and GSTA5 Haplotype Structures.

![Figure 5](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2cbf/2837367/2d913743c3b8/nihms171463f5.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=2837367_nihms171463f5.jpg)

The SNPs that we identified in both GSTM4 and GSTA5 are linked to other SNPs in these genes. The haplotype structures of the two genes were obtained from the HapMap. SNP pairs with high D′ values are shown in red, while SNP pairs with lower D′ values are shown in lighter red, grey, or white. The SNPs which we identified are “boxed” in red, while a SNP in GSTM4 that we did not identify but which is discussed in the text is indicated with a “broken” red box.

*GSTA5* was first identified in 2002 and it remains controversial whether this gene is even expressed, so the function of GSTA5 remains to be clarified ([42](#R42)). This is the first report of an association between a genetic polymorphism in this gene, a polymorphism located in intron 1, and a phenotype, survival of lung cancer patients after platinum-treatment. The SNP that we identified has a high minor allele frequency of approximately 50%, and is linked to several other *GSTA5* polymorphisms ([Figure 5](#F5)). To date, no studies have been performed that might clarify the functional significance of *GSTA5* polymorphisms.

*ABCC4* is a highly polymorphic gene encoding a member of the ATP-binding cassette family of membrane transporters, the multidrug resistance protein 4 (MRP4). Five *ABCC4* SNPs in addition to rs7984157 were among the 20 SNPs with the lowest unadjusted p-values associated with the survival of our lung cancer patients ([Table 2](#T2)). Four of the 5 additional *ABCC4* SNPs were in moderate to high linkage disequilibrium with rs7984157 (rs1564351, D′=0.85, r^2^=0.384; rs2274405, D′=0.434, r^2^=0.023; rs943290, D′=0.606, r^2^=0.021; rs4636781, D′=1.0, r^2^=0.057). There is evidence that cisplatin can form complexes with glutathione and that these complexes are substrates for organic anion transporters, but drug resistance caused by increased transport of these complexes appears to be complex and to depend on a combination of changes in the glutathione pathway ([31](#R31), [43](#R43), [44](#R44)). MRP4 is expressed at low levels in only a few tissues, including lung, kidney, and bladder ([45](#R45)). Although MRP4 overexpression was not detected in cisplatin-resistant cell lines in one study ([45](#R45)), Savaraj et al. reported that MRP4 was overexpressed in a cisplatin-resistant variant small cell lung cancer cell line ([46](#R46)). Wakamatsu et al. also found that MRP4 expression was increased 11.3-fold over expression in wild type cells in hepatocellular carcinoma cells resistant to cisplatin ([47](#R47)). These data suggest that cisplatin resistance due to alterations in transporters is complex, and that polymorphisms, whether present in germline DNA or induced in the tumor genome, might cause alterations in *ABCC4* expression and might result in altered glutathione-conjugate efflux.

Although we identified one CNP and 3 SNPs that may be associated with overall survival after the treatment of lung cancer patients with platinum compounds, only one of those polymorphisms was associated with cisplatin IC_50_ in the 100 lymphoblastoid cell-based model system that we also studied. That may be due, in part, to a different cellular environment in the tumor, resulting in differential effects of sequence variation in *GSTT1*, *GSTM4*, *GSTA5*, and *ABCC4*. However, it has been demonstrated repeatedly that germline polymorphisms can be important for pharmacogenomic effects in drug response during the treatment of cancer[†](#FN2). Because the GSTs can be induced by many chemicals, it is possible that the importance of these SNPs may not be evident until patients have been exposed to an agent which might cause these genes to be regulated differently than in the controlled environment that exists in a cell culture system. In addition, the associations that we observed using DNA from lung cancer patients were not statistically significant after correction for multiple comparisons, so it is also possible that these polymorphisms failed to demonstrate significant associations in the model system because they were false positives during the clinical study, but that explanation is less likely for those that were functionally validated by altering cisplatin IC_50_ values after overexpression in lung cancer cell lines. Resequencing of each of these genes and identifying linkage patterns would be useful to help identify functionally important SNPs and, thus, to identify mechanisms underlying our observed associations.

Finally, the approach that we used in this series of studies, beginning with a clinical association study and then moving to the laboratory in an attempt to validate and/or study mechanisms responsible for the observed statistical associations, is one that may find increasing application as the expense of genotyping continues to decrease, the number of polymorphisms queried rises and the statistical challenges of addressing the issue of multiple comparisons increases.

In conclusion, we have studied the importance of glutathione pathway gene sequence and copy number variation in the response of lung cancer patients and cell line systems to platinum-based antineoplastic agents. A *GSTT1* copy number variant and one SNP each in *GSTM4*, *GSTA5*, and *ABCC4* were associated with overall survival in lung cancer patients treated with platinum-containing antineoplastics. The SNP in *GSTM4* was also associated with cisplatin IC_50_ in a lymphoblastoid cell line-based model system that has been used to study individual genetic variation. Furthermore, overexpression of *GSTT1*, *GSTM4*, and *ABCC4* altered cisplatin IC_50_ values in cancer cell lines, resulting in increased cisplatin resistance.

These results suggest that *GSTM4* genetic variants may represent useful biomarkers for the prediction of cisplatin response. *GSTT1* and *ABCC4* genetic variants may also ultimately prove to be useful biomarkers, but they require further validation. However, future studies designed to explore mechanisms by which variation in this important pathway for drug response might contribute to cisplatin resistance/sensitivity will be needed. Those studies might result in the identification of useful biomarkers and improved algorithms for the treatment of cancer with platinum-containing agents.

## Supplementary Material

## Acknowledgments

We thank Dr. Piet Borst of The Netherlands Cancer Institute for the HEK293T cells overexpressing *ABCC4* that were used in this study and Luanne Wussow for her assistance with the preparation of this manuscript.

This work was supported in part by National Institutes of Health (NIH) grants R01 GM28157, R01 CA80127, R01 CA84354, R01 CA115857, R01 CA132780, R03 CA77118, U01 GM61388 (The Pharmacogenetics Research Network), as well as a PhRMA Foundation “Center of Excellence in Clinical Pharmacology” Award.

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Kelland L. The resurgence of platinum-based cancer chemotherapy. Nat Rev Cancer. 2007;7:573–84. doi: 10.1038/nrc2167.  [DOI](https://doi.org/10.1038/nrc2167) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17625587/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Cancer&title=The%20resurgence%20of%20platinum-based%20cancer%20chemotherapy&author=L%20Kelland&volume=7&publication_year=2007&pages=573-84&pmid=17625587&doi=10.1038/nrc2167&)

2. Cisplatin Reed E, Carboplatin, Oxaliplatin . In: Cancer Chemotherapy and Biotherapy. Chabner B, editor. Lippincott Williams & Wilkins; 2006. pp. 332–343.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Cancer%20Chemotherapy%20and%20Biotherapy&author=Reed%20E%20Cisplatin&author=%20Carboplatin&author=%20Oxaliplatin&publication_year=2006&)

3. Rosenberg B, Vancamp L, Krigas T. Inhibition of Cell Division in Escherichia Coli by Electrolysis Products from a Platinum Electrode. Nature. 1965;205:698–9. doi: 10.1038/205698a0.  [DOI](https://doi.org/10.1038/205698a0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14287410/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Inhibition%20of%20Cell%20Division%20in%20Escherichia%20Coli%20by%20Electrolysis%20Products%20from%20a%20Platinum%20Electrode&author=B%20Rosenberg&author=L%20Vancamp&author=T%20Krigas&volume=205&publication_year=1965&pages=698-9&pmid=14287410&doi=10.1038/205698a0&)

4. Schiller JH, Harrington D, Belani CP, et al. Comparison of four chemotherapy regimens for advanced non-small-cell lung cancer. N Engl J Med. 2002;346:92–8. doi: 10.1056/NEJMoa011954.  [DOI](https://doi.org/10.1056/NEJMoa011954) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11784875/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Comparison%20of%20four%20chemotherapy%20regimens%20for%20advanced%20non-small-cell%20lung%20cancer&author=JH%20Schiller&author=D%20Harrington&author=CP%20Belani&volume=346&publication_year=2002&pages=92-8&pmid=11784875&doi=10.1056/NEJMoa011954&)

5. Eastman A. Characterization of the adducts produced in DNA by cis-diamminedichloroplatinum(II) and cis-dichloro(ethylenediamine)platinum(II) Bioc. doi: 10.1021/bi00285a031.  [DOI](https://doi.org/10.1021/bi00285a031) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/6225458/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioc&title=Characterization%20of%20the%20adducts%20produced%20in%20DNA%20by%20cis-diamminedichloroplatinum(II)%20and%20cis-dichloro(ethylenediamine)platinum(II)&author=A%20Eastman&pmid=6225458&doi=10.1021/bi00285a031&)

6. Fichtinger-Schepman AM, van der Veer JL, den Hartog JH, Lohman PH, Reedijk J. Adducts of the antitumor drug cis-diamminedichloroplatinum(II) with DNA: formation, identification, and quantitation. Biochemistry. 1985;24:707–13. doi: 10.1021/bi00324a025.  [DOI](https://doi.org/10.1021/bi00324a025) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/4039603/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochemistry&title=Adducts%20of%20the%20antitumor%20drug%20cis-diamminedichloroplatinum(II)%20with%20DNA:%20formation,%20identification,%20and%20quantitation&author=AM%20Fichtinger-Schepman&author=JL%20van%20der%20Veer&author=JH%20den%20Hartog&author=PH%20Lohman&author=J%20Reedijk&volume=24&publication_year=1985&pages=707-13&pmid=4039603&doi=10.1021/bi00324a025&)

7. Lindauer E, Holler E. Cellular distribution and cellular reactivity of platinum (II) complexes. Biochem Pharmacol. 1996;52:7–14. doi: 10.1016/0006-2952(96)00106-2.  [DOI](https://doi.org/10.1016/0006-2952(96)00106-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8678910/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20Pharmacol&title=Cellular%20distribution%20and%20cellular%20reactivity%20of%20platinum%20(II)%20complexes&author=E%20Lindauer&author=E%20Holler&volume=52&publication_year=1996&pages=7-14&pmid=8678910&doi=10.1016/0006-2952(96)00106-2&)

8. Lippard SJ. Chemistry and Molecular Biology of Platinum Anticancer Drugs. Pure & Applied Chemistry. 1987;59:731–742.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pure%20&%20Applied%20Chemistry&title=Chemistry%20and%20Molecular%20Biology%20of%20Platinum%20Anticancer%20Drugs&author=SJ%20Lippard&volume=59&publication_year=1987&pages=731-742&)

9. Harder HC, Rosenberg B. Inhibitory effects of anti-tumor platinum compounds on DNA, RNA and protein syntheses in mammalian cells in vitro. Int J Cancer. 1970;6:207–16. doi: 10.1002/ijc.2910060207.  [DOI](https://doi.org/10.1002/ijc.2910060207) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/5479434/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Cancer&title=Inhibitory%20effects%20of%20anti-tumor%20platinum%20compounds%20on%20DNA,%20RNA%20and%20protein%20syntheses%20in%20mammalian%20cells%20in%20vitro&author=HC%20Harder&author=B%20Rosenberg&volume=6&publication_year=1970&pages=207-16&pmid=5479434&doi=10.1002/ijc.2910060207&)

10. Wang D, Lippard SJ. Cellular processing of platinum anticancer drugs. Nat Rev Drug Discov. 2005;4:307–20. doi: 10.1038/nrd1691.  [DOI](https://doi.org/10.1038/nrd1691) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15789122/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Drug%20Discov&title=Cellular%20processing%20of%20platinum%20anticancer%20drugs&author=D%20Wang&author=SJ%20Lippard&volume=4&publication_year=2005&pages=307-20&pmid=15789122&doi=10.1038/nrd1691&)

11. Rabik CA, Dolan ME. Molecular mechanisms of resistance and toxicity associated with platinating agents. Cancer Treat Rev. 2007;33:9–23. doi: 10.1016/j.ctrv.2006.09.006.  [DOI](https://doi.org/10.1016/j.ctrv.2006.09.006) | [PMC free article](/articles/PMC1855222/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17084534/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Treat%20Rev&title=Molecular%20mechanisms%20of%20resistance%20and%20toxicity%20associated%20with%20platinating%20agents&author=CA%20Rabik&author=ME%20Dolan&volume=33&publication_year=2007&pages=9-23&pmid=17084534&doi=10.1016/j.ctrv.2006.09.006&)

12. Wang G, Reed E, Li QQ. Molecular basis of cellular response to cisplatin chemotherapy in non-small cell lung cancer (Review) Oncol Rep. 2004;12:955–65.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/15492778/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncol%20Rep&title=Molecular%20basis%20of%20cellular%20response%20to%20cisplatin%20chemotherapy%20in%20non-small%20cell%20lung%20cancer%20(Review)&author=G%20Wang&author=E%20Reed&author=QQ%20Li&volume=12&publication_year=2004&pages=955-65&pmid=15492778&)

13. Yang P, Ebbert JO, Sun Z, Weinshilboum RM. Role of the glutathione metabolic pathway in lung cancer treatment and prognosis: a review. J Clin Oncol. 2006;24:1761–9. doi: 10.1200/JCO.2005.02.7110.  [DOI](https://doi.org/10.1200/JCO.2005.02.7110) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16603718/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Role%20of%20the%20glutathione%20metabolic%20pathway%20in%20lung%20cancer%20treatment%20and%20prognosis:%20a%20review&author=P%20Yang&author=JO%20Ebbert&author=Z%20Sun&author=RM%20Weinshilboum&volume=24&publication_year=2006&pages=1761-9&pmid=16603718&doi=10.1200/JCO.2005.02.7110&)

14. Devesa SS, Grauman DJ, Blot WJ, Pennello GA, Hoover RN, Fraumeni JF., Jr . Atlas of cancer mortality in the United States, 1950-94. Washington, DC: US Govt Print Off; 1999.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Atlas%20of%20cancer%20mortality%20in%20the%20United%20States,%201950-94&author=SS%20Devesa&author=DJ%20Grauman&author=WJ%20Blot&author=GA%20Pennello&author=RN%20Hoover&publication_year=1999&)

15. Yang P, Yokomizo A, Tazelaar HD, et al. Genetic determinants of lung cancer short-term survival: the role of glutathione-related genes. Lung Cancer. 2002;35:221–9. doi: 10.1016/s0169-5002(01)00426-3.  [DOI](https://doi.org/10.1016/s0169-5002(01)00426-3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11844594/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lung%20Cancer&title=Genetic%20determinants%20of%20lung%20cancer%20short-term%20survival:%20the%20role%20of%20glutathione-related%20genes&author=P%20Yang&author=A%20Yokomizo&author=HD%20Tazelaar&volume=35&publication_year=2002&pages=221-9&pmid=11844594&doi=10.1016/s0169-5002(01)00426-3&)

16. Sun Z, Aubry MC, Deschamps C, et al. Histologic grade is an independent prognostic factor for survival in non-small cell lung cancer: an analysis of 5018 hospital- and 712 population-based cases. J Thorac Cardiovasc Surg. 2006;131:1014–20. doi: 10.1016/j.jtcvs.2005.12.057.  [DOI](https://doi.org/10.1016/j.jtcvs.2005.12.057) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16678584/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Thorac%20Cardiovasc%20Surg&title=Histologic%20grade%20is%20an%20independent%20prognostic%20factor%20for%20survival%20in%20non-small%20cell%20lung%20cancer:%20an%20analysis%20of%205018%20hospital-%20and%20712%20population-based%20cases&author=Z%20Sun&author=MC%20Aubry&author=C%20Deschamps&volume=131&publication_year=2006&pages=1014-20&pmid=16678584&doi=10.1016/j.jtcvs.2005.12.057&)

17. Visbal AL, Williams BA, Nichols FC, 3rd, et al. Gender differences in non-small-cell lung cancer survival: an analysis of 4,618 patients diagnosed between 1997 and 2002. Ann Thorac Surg. 2004;78:209–15. doi: 10.1016/j.athoracsur.2003.11.021. discussion 215.  [DOI](https://doi.org/10.1016/j.athoracsur.2003.11.021) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15223430/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Thorac%20Surg&title=Gender%20differences%20in%20non-small-cell%20lung%20cancer%20survival:%20an%20analysis%20of%204,618%20patients%20diagnosed%20between%201997%20and%202002&author=AL%20Visbal&author=BA%20Williams&author=FC%20Nichols&volume=78&publication_year=2004&pages=209-15&pmid=15223430&doi=10.1016/j.athoracsur.2003.11.021&)

18. Yang P, Allen MS, Aubry MC, et al. Clinical features of 5,628 primary lung cancer patients: experience at Mayo Clinic from 1997 to 2003. Chest. 2005;128:452–62. doi: 10.1378/chest.128.1.452.  [DOI](https://doi.org/10.1378/chest.128.1.452) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16002972/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chest&title=Clinical%20features%20of%205,628%20primary%20lung%20cancer%20patients:%20experience%20at%20Mayo%20Clinic%20from%201997%20to%202003&author=P%20Yang&author=MS%20Allen&author=MC%20Aubry&volume=128&publication_year=2005&pages=452-62&pmid=16002972&doi=10.1378/chest.128.1.452&)

19. Wielinga PR, Reid G, Challa EE, et al. Thiopurine metabolism and identification of the thiopurine metabolites transported by MRP4 and MRP5 overexpressed in human embryonic kidney cells. Mol Pharmacol. 2002;62:1321–31. doi: 10.1124/mol.62.6.1321.  [DOI](https://doi.org/10.1124/mol.62.6.1321) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12435799/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Pharmacol&title=Thiopurine%20metabolism%20and%20identification%20of%20the%20thiopurine%20metabolites%20transported%20by%20MRP4%20and%20MRP5%20overexpressed%20in%20human%20embryonic%20kidney%20cells&author=PR%20Wielinga&author=G%20Reid&author=EE%20Challa&volume=62&publication_year=2002&pages=1321-31&pmid=12435799&doi=10.1124/mol.62.6.1321&)

20. Moyer AM, Salavaggione OE, Hebbring SJ, et al. Glutathione S-transferase T1 and M1: gene sequence variation and functional genomics. Clin Cancer Res. 2007;13:7207–16. doi: 10.1158/1078-0432.CCR-07-0635.  [DOI](https://doi.org/10.1158/1078-0432.CCR-07-0635) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18056202/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=Glutathione%20S-transferase%20T1%20and%20M1:%20gene%20sequence%20variation%20and%20functional%20genomics&author=AM%20Moyer&author=OE%20Salavaggione&author=SJ%20Hebbring&volume=13&publication_year=2007&pages=7207-16&pmid=18056202&doi=10.1158/1078-0432.CCR-07-0635&)

21. Moyer AM, Salavaggione OE, Wu TY, et al. Glutathione S-transferase P1: gene sequence variation and functional genomic studies. Cancer Res. 2008;68:4791–801. doi: 10.1158/0008-5472.CAN-07-6724.  [DOI](https://doi.org/10.1158/0008-5472.CAN-07-6724) | [PMC free article](/articles/PMC2490824/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18559526/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=Glutathione%20S-transferase%20P1:%20gene%20sequence%20variation%20and%20functional%20genomic%20studies&author=AM%20Moyer&author=OE%20Salavaggione&author=TY%20Wu&volume=68&publication_year=2008&pages=4791-801&pmid=18559526&doi=10.1158/0008-5472.CAN-07-6724&)

22. Mukherjee B, Salavaggione OE, Pelleymounter LL, et al. Glutathione S-transferase omega 1 and omega 2 pharmacogenomics. Drug Metab Dispos. 2006;34:1237–46. doi: 10.1124/dmd.106.009613.  [DOI](https://doi.org/10.1124/dmd.106.009613) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16638819/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Glutathione%20S-transferase%20omega%201%20and%20omega%202%20pharmacogenomics&author=B%20Mukherjee&author=OE%20Salavaggione&author=LL%20Pelleymounter&volume=34&publication_year=2006&pages=1237-46&pmid=16638819&doi=10.1124/dmd.106.009613&)

23. Hartl D, Clark AG. Chapter 3. Organization of genetic variation. Principles of Population Genetics. Sunderland, MA: Sinauer Associates, Inc.; 2000. pp. 95–107.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Chapter%203.%20Organization%20of%20genetic%20variation.%20Principles%20of%20Population%20Genetics&author=D%20Hartl&author=AG%20Clark&publication_year=2000&)

24. Hendrick P. Genetics of Populations. Sudbury, MA: Jones and Bartlett Publishing; 2000. pp. 396–405.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Genetics%20of%20Populations&author=P%20Hendrick&publication_year=2000&)

25. Schaid DJ, Rowland CM, Tines DE, Jacobson RM, Poland GA. Score tests for association between traits and haplotypes when linkage phase is ambiguous. Am J Hum Genet. 2002;70:425–34. doi: 10.1086/338688.  [DOI](https://doi.org/10.1086/338688) | [PMC free article](/articles/PMC384917/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11791212/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Score%20tests%20for%20association%20between%20traits%20and%20haplotypes%20when%20linkage%20phase%20is%20ambiguous&author=DJ%20Schaid&author=CM%20Rowland&author=DE%20Tines&author=RM%20Jacobson&author=GA%20Poland&volume=70&publication_year=2002&pages=425-34&pmid=11791212&doi=10.1086/338688&)

26. Li L, Fridley B, Kalari K, et al. Gemcitabine and cytosine arabinoside cytotoxicity: association with lymphoblastoid cell expression. Cancer Res. 2008;68:7050–7058. doi: 10.1158/0008-5472.CAN-08-0405.  [DOI](https://doi.org/10.1158/0008-5472.CAN-08-0405) | [PMC free article](/articles/PMC2562356/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18757419/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=Gemcitabine%20and%20cytosine%20arabinoside%20cytotoxicity:%20association%20with%20lymphoblastoid%20cell%20expression&author=L%20Li&author=B%20Fridley&author=K%20Kalari&volume=68&publication_year=2008&pages=7050-7058&pmid=18757419&doi=10.1158/0008-5472.CAN-08-0405&)

27. Wang L, Sullivan W, Toft D, Weinshilboum R. Thiopurine S-methyltransferase pharmacogenetics: chaperone protein association and allozyme degradation. Pharmacogenetics. 2003;13:555–64. doi: 10.1097/01.fpc.0000054124.14659.99.  [DOI](https://doi.org/10.1097/01.fpc.0000054124.14659.99) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12972954/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Thiopurine%20S-methyltransferase%20pharmacogenetics:%20chaperone%20protein%20association%20and%20allozyme%20degradation&author=L%20Wang&author=W%20Sullivan&author=D%20Toft&author=R%20Weinshilboum&volume=13&publication_year=2003&pages=555-64&pmid=12972954&doi=10.1097/01.fpc.0000054124.14659.99&)

28. Salavaggione OE, Wang L, Wiepert M, Yee VC, Weinshilboum RM. Thiopurine S-methyltransferase pharmacogenetics: variant allele functional and comparative genomics. Pharmacogenet Genomics. 2005;15:801–815. doi: 10.1097/01.fpc.0000174788.69991.6b.  [DOI](https://doi.org/10.1097/01.fpc.0000174788.69991.6b) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16220112/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Thiopurine%20S-methyltransferase%20pharmacogenetics:%20variant%20allele%20functional%20and%20comparative%20genomics&author=OE%20Salavaggione&author=L%20Wang&author=M%20Wiepert&author=VC%20Yee&author=RM%20Weinshilboum&volume=15&publication_year=2005&pages=801-815&pmid=16220112&doi=10.1097/01.fpc.0000174788.69991.6b&)

29. Cheung VG, Spielman RS, Ewens KG, Weber TM, Morley M, Burdick JT. Mapping determinants of human gene expression by regional and genome-wide association. Nature. 2005;437:1365–9. doi: 10.1038/nature04244.  [DOI](https://doi.org/10.1038/nature04244) | [PMC free article](/articles/PMC3005311/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16251966/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Mapping%20determinants%20of%20human%20gene%20expression%20by%20regional%20and%20genome-wide%20association&author=VG%20Cheung&author=RS%20Spielman&author=KG%20Ewens&author=TM%20Weber&author=M%20Morley&volume=437&publication_year=2005&pages=1365-9&pmid=16251966&doi=10.1038/nature04244&)

30. Eastman A, Schulte N. Enhanced DNA repair as a mechanism of resistance to cis-diamminedichloroplatinum(II) Biochemistry. 1988;27:4730–4. doi: 10.1021/bi00413a022.  [DOI](https://doi.org/10.1021/bi00413a022) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/3167012/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochemistry&title=Enhanced%20DNA%20repair%20as%20a%20mechanism%20of%20resistance%20to%20cis-diamminedichloroplatinum(II)&author=A%20Eastman&author=N%20Schulte&volume=27&publication_year=1988&pages=4730-4&pmid=3167012&doi=10.1021/bi00413a022&)

31. Ishikawa T, Ali-Osman F. Glutathione-associated cis-diamminedichloroplatinum(II) metabolism and ATP-dependent efflux from leukemia cells. Molecular characterization of glutathione-platinum complex and its biological significance. J Biol Chem. 1993;268:20116–25.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/8376370/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Glutathione-associated%20cis-diamminedichloroplatinum(II)%20metabolism%20and%20ATP-dependent%20efflux%20from%20leukemia%20cells.%20Molecular%20characterization%20of%20glutathione-platinum%20complex%20and%20its%20biological%20significance&author=T%20Ishikawa&author=F%20Ali-Osman&volume=268&publication_year=1993&pages=20116-25&pmid=8376370&)

32. Loh SY, Mistry P, Kelland LR, Abel G, Harrap KR. Reduced drug accumulation as a major mechanism of acquired resistance to cisplatin in a human ovarian carcinoma cell line: circumvention studies using novel platinum (II) and (IV) ammine/amine complexes. Br J Cancer. 1992;66:1109–15. doi: 10.1038/bjc.1992.419.  [DOI](https://doi.org/10.1038/bjc.1992.419) | [PMC free article](/articles/PMC1978040/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1457352/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Cancer&title=Reduced%20drug%20accumulation%20as%20a%20major%20mechanism%20of%20acquired%20resistance%20to%20cisplatin%20in%20a%20human%20ovarian%20carcinoma%20cell%20line:%20circumvention%20studies%20using%20novel%20platinum%20(II)%20and%20(IV)%20ammine/amine%20complexes&author=SY%20Loh&author=P%20Mistry&author=LR%20Kelland&author=G%20Abel&author=KR%20Harrap&volume=66&publication_year=1992&pages=1109-15&pmid=1457352&doi=10.1038/bjc.1992.419&)

33. Meijer C, Mulder NH, Timmer-Bosscha H, Sluiter WJ, Meersma GJ, de Vries EG. Relationship of cellular glutathione to the cytotoxicity and resistance of seven platinum compounds. Cancer Res. 1992;52:6885–9.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/1458477/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=Relationship%20of%20cellular%20glutathione%20to%20the%20cytotoxicity%20and%20resistance%20of%20seven%20platinum%20compounds&author=C%20Meijer&author=NH%20Mulder&author=H%20Timmer-Bosscha&author=WJ%20Sluiter&author=GJ%20Meersma&volume=52&publication_year=1992&pages=6885-9&pmid=1458477&)

34. Eastman A. Cross-linking of glutathione to DNA by cancer chemotherapeutic platinum coordination complexes. Chem Biol Interact. 1987;61:241–8. doi: 10.1016/0009-2797(87)90004-4.  [DOI](https://doi.org/10.1016/0009-2797(87)90004-4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/3568194/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chem%20Biol%20Interact&title=Cross-linking%20of%20glutathione%20to%20DNA%20by%20cancer%20chemotherapeutic%20platinum%20coordination%20complexes&author=A%20Eastman&volume=61&publication_year=1987&pages=241-8&pmid=3568194&doi=10.1016/0009-2797(87)90004-4&)

35. Godwin AK, Meister A, O'Dwyer PJ, Huang CS, Hamilton TC, Anderson ME. High resistance to cisplatin in human ovarian cancer cell lines is associated with marked increase of glutathione synthesis. Proc Natl Acad Sci U S A. 1992;89:3070–4. doi: 10.1073/pnas.89.7.3070.  [DOI](https://doi.org/10.1073/pnas.89.7.3070) | [PMC free article](/articles/PMC48805/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1348364/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&title=High%20resistance%20to%20cisplatin%20in%20human%20ovarian%20cancer%20cell%20lines%20is%20associated%20with%20marked%20increase%20of%20glutathione%20synthesis&author=AK%20Godwin&author=A%20Meister&author=PJ%20O'Dwyer&author=CS%20Huang&author=TC%20Hamilton&volume=89&publication_year=1992&pages=3070-4&pmid=1348364&doi=10.1073/pnas.89.7.3070&)

36. Hann CL, Rudin CM. Management of small-cell lung cancer: incremental changes but hope for the future. Oncology (Williston Park) 2008;22:1486–1492.  [PMC free article](/articles/PMC4124612/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19133604/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncology%20(Williston%20Park)&title=Management%20of%20small-cell%20lung%20cancer:%20incremental%20changes%20but%20hope%20for%20the%20future&author=CL%20Hann&author=CM%20Rudin&volume=22&publication_year=2008&pages=1486-1492&pmid=19133604&)

37. Sanborn RE. Cisplatin versus carboplatin in NSCLC: is there one “best” answer? Curr Treat Options Oncol. 2008;9:326–342. doi: 10.1007/s11864-009-0085-5.  [DOI](https://doi.org/10.1007/s11864-009-0085-5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19225891/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Treat%20Options%20Oncol&title=Cisplatin%20versus%20carboplatin%20in%20NSCLC:%20is%20there%20one%20%E2%80%9Cbest%E2%80%9D%20answer?&author=RE%20Sanborn&volume=9&publication_year=2008&pages=326-342&pmid=19225891&doi=10.1007/s11864-009-0085-5&)

38. van Aubel RA, Smeets PH, Peters JG, Bindels RJ, Russel FG. The MRP4/ABCC4 gene encodes a novel apical organic anion transporter in human kidney proximal tubules: putative efflux pump for urinary cAMP and cGMP. J Am Soc Nephrol. 2002;13:595–603. doi: 10.1681/ASN.V133595.  [DOI](https://doi.org/10.1681/ASN.V133595) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11856762/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Am%20Soc%20Nephrol&title=The%20MRP4/ABCC4%20gene%20encodes%20a%20novel%20apical%20organic%20anion%20transporter%20in%20human%20kidney%20proximal%20tubules:%20putative%20efflux%20pump%20for%20urinary%20cAMP%20and%20cGMP&author=RA%20van%20Aubel&author=PH%20Smeets&author=JG%20Peters&author=RJ%20Bindels&author=FG%20Russel&volume=13&publication_year=2002&pages=595-603&pmid=11856762&doi=10.1681/ASN.V133595&)

39. Medeiros R, Pereira D, Afonso N, et al. Platinum/paclitaxel-based chemotherapy in advanced ovarian carcinoma: glutathione S-transferase genetic polymorphisms as predictive biomarkers of disease outcome. Int J Clin Oncol. 2003;8:156–61. doi: 10.1007/s10147-003-0318-8.  [DOI](https://doi.org/10.1007/s10147-003-0318-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12851839/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Clin%20Oncol&title=Platinum/paclitaxel-based%20chemotherapy%20in%20advanced%20ovarian%20carcinoma:%20glutathione%20S-transferase%20genetic%20polymorphisms%20as%20predictive%20biomarkers%20of%20disease%20outcome&author=R%20Medeiros&author=D%20Pereira&author=N%20Afonso&volume=8&publication_year=2003&pages=156-61&pmid=12851839&doi=10.1007/s10147-003-0318-8&)

40. Watson MB, Lind MJ, Smith L, Drew PJ, Cawkwell L. Expression microarray analysis reveals genes associated with in vitro resistance to cisplatin in a cell line model. Acta Oncol. 2007;46:651–8. doi: 10.1080/02841860601156157.  [DOI](https://doi.org/10.1080/02841860601156157) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17562441/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Acta%20Oncol&title=Expression%20microarray%20analysis%20reveals%20genes%20associated%20with%20in%20vitro%20resistance%20to%20cisplatin%20in%20a%20cell%20line%20model&author=MB%20Watson&author=MJ%20Lind&author=L%20Smith&author=PJ%20Drew&author=L%20Cawkwell&volume=46&publication_year=2007&pages=651-8&pmid=17562441&doi=10.1080/02841860601156157&)

41. Liloglou T, Walters M, Maloney P, Youngson J, Field JK. A T2517C polymorphism in the GSTM4 gene is associated with risk of developing lung cancer. Lung Cancer. 2002;37:143–6. doi: 10.1016/s0169-5002(02)00078-8.  [DOI](https://doi.org/10.1016/s0169-5002(02)00078-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12140136/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lung%20Cancer&title=A%20T2517C%20polymorphism%20in%20the%20GSTM4%20gene%20is%20associated%20with%20risk%20of%20developing%20lung%20cancer&author=T%20Liloglou&author=M%20Walters&author=P%20Maloney&author=J%20Youngson&author=JK%20Field&volume=37&publication_year=2002&pages=143-6&pmid=12140136&doi=10.1016/s0169-5002(02)00078-8&)

42. Morel F, Rauch C, Coles B, Le Ferrec E, Guillouzo A. The human glutathione transferase alpha locus: genomic organization of the gene cluster and functional characterization of the genetic polymorphism in the hGSTA1 promoter. Pharmacogenetics. 2002;12:277–86. doi: 10.1097/00008571-200206000-00003.  [DOI](https://doi.org/10.1097/00008571-200206000-00003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12042665/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=The%20human%20glutathione%20transferase%20alpha%20locus:%20genomic%20organization%20of%20the%20gene%20cluster%20and%20functional%20characterization%20of%20the%20genetic%20polymorphism%20in%20the%20hGSTA1%20promoter&author=F%20Morel&author=C%20Rauch&author=B%20Coles&author=E%20Le%20Ferrec&author=A%20Guillouzo&volume=12&publication_year=2002&pages=277-86&pmid=12042665&doi=10.1097/00008571-200206000-00003&)

43. Muller M, Meijer C, Zaman GJ, et al. Overexpression of the gene encoding the multidrug resistance-associated protein results in increased ATP-dependent glutathione S-conjugate transport. Proc Natl Acad Sci U S A. 1994;91:13033–7. doi: 10.1073/pnas.91.26.13033.  [DOI](https://doi.org/10.1073/pnas.91.26.13033) | [PMC free article](/articles/PMC45575/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7809167/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&title=Overexpression%20of%20the%20gene%20encoding%20the%20multidrug%20resistance-associated%20protein%20results%20in%20increased%20ATP-dependent%20glutathione%20S-conjugate%20transport&author=M%20Muller&author=C%20Meijer&author=GJ%20Zaman&volume=91&publication_year=1994&pages=13033-7&pmid=7809167&doi=10.1073/pnas.91.26.13033&)

44. Zaman GJ, Lankelma J, van Tellingen O, et al. Role of glutathione in the export of compounds from cells by the multidrug-resistance-associated protein. Proc Natl Acad Sci U S A. 1995;92:7690–4. doi: 10.1073/pnas.92.17.7690.  [DOI](https://doi.org/10.1073/pnas.92.17.7690) | [PMC free article](/articles/PMC41211/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7644478/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&title=Role%20of%20glutathione%20in%20the%20export%20of%20compounds%20from%20cells%20by%20the%20multidrug-resistance-associated%20protein&author=GJ%20Zaman&author=J%20Lankelma&author=O%20van%20Tellingen&volume=92&publication_year=1995&pages=7690-4&pmid=7644478&doi=10.1073/pnas.92.17.7690&)

45. Kool M, de Haas M, Scheffer GL, et al. Analysis of expression of cMOAT (MRP2), MRP3, MRP4, and MRP5, homologues of the multidrug resistance-associated protein gene (MRP1), in human cancer cell lines. Cancer Res. 1997;57:3537–47.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/9270026/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=Analysis%20of%20expression%20of%20cMOAT%20(MRP2),%20MRP3,%20MRP4,%20and%20MRP5,%20homologues%20of%20the%20multidrug%20resistance-associated%20protein%20gene%20(MRP1),%20in%20human%20cancer%20cell%20lines&author=M%20Kool&author=M%20de%20Haas&author=GL%20Scheffer&volume=57&publication_year=1997&pages=3537-47&pmid=9270026&)

46. Savaraj N, Wu C, Wangpaichitr M, et al. Overexpression of mutated MRP4 in cisplatin resistant small cell lung cancer cell line: collateral sensitivity to azidothymidine. Int J Oncol. 2003;23:173–9.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/12792791/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Oncol&title=Overexpression%20of%20mutated%20MRP4%20in%20cisplatin%20resistant%20small%20cell%20lung%20cancer%20cell%20line:%20collateral%20sensitivity%20to%20azidothymidine&author=N%20Savaraj&author=C%20Wu&author=M%20Wangpaichitr&volume=23&publication_year=2003&pages=173-9&pmid=12792791&)

47. Wakamatsu T, Nakahashi Y, Hachimine D, Seki T, Okazaki K. The combination of glycyrrhizin and lamivudine can reverse the cisplatin resistance in hepatocellular carcinoma cells through inhibition of multidrug resistance-associated proteins. Int J Oncol. 2007;31:1465–72.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/17982673/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Oncol&title=The%20combination%20of%20glycyrrhizin%20and%20lamivudine%20can%20reverse%20the%20cisplatin%20resistance%20in%20hepatocellular%20carcinoma%20cells%20through%20inhibition%20of%20multidrug%20resistance-associated%20proteins&author=T%20Wakamatsu&author=Y%20Nakahashi&author=D%20Hachimine&author=T%20Seki&author=K%20Okazaki&volume=31&publication_year=2007&pages=1465-72&pmid=17982673&)

48. http://www.cancer.org/docroot/CRI/content/CRI_2_4_3x_How_Is_Small_Cell_Lung_Cancer_Staged.asp  [http://www.cancer.org/docroot/CRI/content/CRI_2_4_3x_How_Is_Small_Cell_Lung_Cancer_Staged.asp](http://www.cancer.org/docroot/CRI/content/CRI_2_4_3x_How_Is_Small_Cell_Lung_Cancer_Staged.asp)
