# CACNA1C gene polymorphisms, cardiovascular disease outcomes and treatment response

## Metadata
**Authors:** Amber L Beitelshees, Hrishikesh Navare, Danxin Wang, Yan Gong, Jennifer Wessel, James Moss, Taimour Y Langaee, Rhonda M Cooper-DeHoff, Wolfgang Sadee, Carl J Pepine, Nicolas J Schork, Julie A Johnson
**Journal:** Circulation. Cardiovascular genetics
**Date:** 2009 Jun 3
**DOI:** [10.1161/CIRCGENETICS.109.857839](https://doi.org/10.1161/CIRCGENETICS.109.857839)
**PMID:** 20031608
**PMCID:** PMC2761685
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2761685/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC2761685/pdf/nihms131313.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC2761685/pdf/nihms131313.pdf)

## Abstract

**Background:** 
The gene encoding the target of calcium channel blockers, the α1c-subunit of the L-type calcium channel (CACNA1C) has not been well characterized and only small pharmacogenetic studies testing this gene have been published to date.

**Methods and Results:** 
Resequencing of CACNA1C was performed followed by a nested case-control study of the INternational VErapamil SR/trandolapril STudy (INVEST) GENEtic Substudy (INVEST-GENES). Of 46 polymorphisms identified, eight were assessed in the INVEST-GENES. Rs1051375 was found to have a significant interaction with treatment strategy (p=0.0001). Rs1051375 A/A genotype was associated with a 46% reduction in the primary outcome among those randomized to verapamil SR treatment compared to atenolol treatment (OR 0.54 95% CI 0.32-0.92). In heterozygous A/G individuals, there was no difference in the occurrence of the primary outcome when randomized to verapamil SR versus atenolol treatment (OR 1.47 95% CI 0.86-2.53), while homozygous G/G individuals had a greater than 4-fold increased risk of the primary outcome with verapamil treatment compared to those randomized to atenolol treatment (OR 4.59 95% CI 1.67-12.67). We did not identify allelic expression imbalance or differences in mRNA expression in heart tissue by rs1051375 genotype.

**Conclusions:** 
Variation in CACNA1C is associated with treatment response among hypertensive patients with stable coronary artery disease. Our data suggest a genetically-defined group of patients that benefit most from calcium channel blocker therapy, a group that benefits most from β-blocker therapy, and a third group in which calcium channel blocker and β-blocker therapy are equivalent.

Keywords: genetics, pharmacology, ion channels, calcium, pharmacogenetics

### Background

The gene encoding the target of calcium channel blockers, the α1c-subunit of the L-type calcium channel (*CACNA1C*) has not been well characterized and only small pharmacogenetic studies testing this gene have been published to date.

### Methods and Results

Resequencing of *CACNA1C* was performed followed by a nested case-control study of the INternational VErapamil SR/trandolapril STudy (INVEST) GENEtic Substudy (INVEST-GENES). Of 46 polymorphisms identified, eight were assessed in the INVEST-GENES. Rs1051375 was found to have a significant interaction with treatment strategy (p=0.0001). Rs1051375 A/A genotype was associated with a 46% reduction in the primary outcome among those randomized to verapamil SR treatment compared to atenolol treatment (OR 0.54 95% CI 0.32-0.92). In heterozygous A/G individuals, there was no difference in the occurrence of the primary outcome when randomized to verapamil SR versus atenolol treatment (OR 1.47 95% CI 0.86-2.53), while homozygous G/G individuals had a greater than 4-fold increased risk of the primary outcome with verapamil treatment compared to those randomized to atenolol treatment (OR 4.59 95% CI 1.67-12.67). We did not identify allelic expression imbalance or differences in mRNA expression in heart tissue by rs1051375 genotype.

### Conclusions

Variation in *CACNA1C* is associated with treatment response among hypertensive patients with stable coronary artery disease. Our data suggest a genetically-defined group of patients that benefit most from calcium channel blocker therapy, a group that benefits most from β-blocker therapy, and a third group in which calcium channel blocker and β-blocker therapy are equivalent.

## INTRODUCTION

The α1c subunit is the major pore-forming subunit of the L-type calcium channel and is the binding site for all currently available calcium channel blockers (i.e. dihydropyridines, phenylalkylamines, and benzothiazepines).[1](#R1) The gene that encodes the α1c subunit of the L-type calcium channel, *CACNA1C*, is a large gene, nearly 300 kb in size, located on chromosome 12p13.3.[2](#R2) It is made up of 44 invariant and 6 alternative exons with a coding region of over 8 kb.[2](#R2) Although NCBI’s dbSNP reports nearly 2,000 SNPs in *CACNA1C*, a relatively small proportion of those are validated. Furthermore, HapMap data suggest a very low degree of linkage disequilibrium in this region of the genome, making tagSNP approaches for association studies of this gene problematic.

Perhaps related to these issues, very few studies have been published evaluating genetic influences of any drug target candidate genes on calcium channel blocker response.[3](#R3)^-^[5](#R5) The lack of pharmacogenetic data with calcium channel blockers is surprising given that amlodipine, verapamil, diltiazem, and nifedipine are all among the top 300 prescription drugs based on prescription numbers in 2005 (Rx List 2005). The studies published to date evaluating *CACNA1C* and calcium channel blocker response have been somewhat limited in that small numbers of individuals were evaluated (n=120-160) and that many different calcium channel blockers were all studied together.[4](#R4)^, ^[5](#R5)

Because of the small number of coding region variants in the public domain databases in *CACNA1C*, and because this gene is not in an extensively studied ENCODE region of HapMap, we undertook a resequencing effort for SNP discovery. We then performed a nested case-control study using the genetic substudy of the INternational VErapamil SR/Trandolapril STudy (INVEST-GENES) to evaluate the impact of selected SNPs on adverse outcomes and response to verapamil SR.

## METHODS

### Resequencing

Unrelated genomic DNA samples were purchased from the Coriell Cell Repository: 20 Native American samples (panels HD-17 and HD-18), 20 African American samples, (first 20 samples from panel AA50), and 20 Caucasian samples (from the apparently healthy collection, sample numbers available upon request).

PCR primers were designed using Mutation Discovery to create amplicons containing an exon (or portion of an exon) and at least 50 base pairs (bp) of intron upstream and downstream from the exons. Exon boundaries and numbering were defined according to Soldatov, et al.[2](#R2) PCR primers and conditions for the 54 amplicons are shown in [Supplemental Table 1](#SD2).

For each amplicon, a reference sample that did not form a heteroduplex was chosen. Temperatures for DHPLC were chosen using Navigator® software (Transgenomic, Omaha, NE,) ([Supplemental Table 1](#SD2)). Each of the 60 samples was then pooled with the reference sample in a two to one ratio, denatured at 95 °C, and allowed to slowly reanneal over 30 minutes to allow heteroduplex formation. Pooled samples were then run on DHPLC under partially dentaturing conditions according the gradient calculated by the Navigator software. Reference samples and samples forming heteroduplexes were sent for direct sequencing to determine the nature and location of the variation present in the amplicon. Sequencing was performed in the forward and reverse direction using the same primers as those used for PCR amplification (Amersham, MegaBACE 1000) ([Supplemental Table 1](#SD2)). In addition to the human samples, one chimpanzee and one gorilla sample were also purchased from the Coriell Cell Repository and sequenced in order to estimate the ancestral alleles of polymorphisms discovered.

### Prediction of Functional Consequences of Polymorphisms and SNP Selection

PolyMAPr was run on all discovered polymorphisms to predict the functional consequences.[6](#R6) Conserved noncoding regions were determined using VISTA browser ([http://pipeline.lbl.gov/cgi-bin/gateway2](http://pipeline.lbl.gov/cgi-bin/gateway2)).

SNPs to study for clinical association were chosen if they had a minor allele frequency ≥ 0.10 and met one of the following critera: 1) non-synonymous in nature or putative functional significance based on *in silico* data (i.e. located within putative transcription factor binding site, exonic splicing enhancer region, or splice sites) or 2) located in conserved noncoding sequence. SNPs that met these criteria were then assessed for pairwise LD and redundant SNPs were eliminated.

### INVEST-GENES Clinical Cohort

INVEST was a randomized trial of 22,576 patients with hypertension and stable coronary artery disease. Patients were randomized to an atenolol-or verapamil-based treatment strategy with other antihypertensives added in order to achieve blood pressure control.[7](#R7) INVEST-GENES has been described previously.[3](#R3) Briefly, genetic samples were collected from 5,979 INVEST patients residing in mainland United States and Puerto Rico. All patients provided written informed consent for participation in the genetic substudy and the study was approved by the University of Florida IRB. Initial genotype data became available in August of 2007 and final genotype data for *CACNA1C* became available on March 18, 2008. Using the 5,979 patients with genetic samples as previously described, a nested case-control group consisting of all of the 258 patients who experienced the primary outcome (first occurrence of death, nonfatal myocardial infarction, or nonfatal stroke) and 774 age-, sex-, and race-frequency-matched controls from INVEST-GENES were assessed. The nested case control study provides nearly the same statistical power as genotyping the entire cohort as only the number of controls is decreased. We have previously demonstrated with four other genes and seven SNPs that this nested case control approach yields similar results as genotyping the entire cohort of 5,979 samples (*ADRB1, ADRB2, KCNMB1, and ADD1*). [3](#R3)^, ^[8](#R8)^, ^[9](#R9)

### Genotyping

Genomic DNA was extracted from buccal cells collected in mouthwash samples according to standard protocols.[10](#R10) Polymorphisms were genotyped by pyrosequencing (PSQ HS 96A) or Taqman^®^ methods. The PSQ HS 96 genotyping platform (Biotage AB, Uppsala, Sweden) was used for the pyrosequencing assays for Rs215976, SNP37 and rs2239128 (primer sequences available upon request). PCR reactions were carried out using HotStar Taq mix, 10 pmmol each of forward and reverse primers, water, and 20ng of genomic DNA. The Applied Biosystems 7900 HT SNP genotyping platform was used for the Taqman^®^ assay. The PCR primers and probes for rs216008, rs1051375, rs10848683, rs2239050 and rs2238032 assays (IDs C___7499713_1_, C___2877394_1_, C___2877389_10, C__16173701_10, C__16171390_10, respectively) were purchased from Applied Biosystems (Applied Biosystems, Foster City, USA). 5 μL reactions in 384-well plate were prepared and the assays were performed and analyzed according to the manufacture’s recommendations. Haplotypes were predicted using Polymorphism and Haplotype Analysis Suite version 0.9.[11](#R11) Analysis was performed using the most likely haplotype. The genotype and primary event data have been deposited in the Pharmacogenomics Knowledge Base ([www.pharmgkb.org](http://www.pharmgkb.org)).

### Functional Assessment of Selected Single Nucleotide Polymorphisms

To test whether rs1051375 changes CANA1C mRNA levels via RNA processing or splicing, we measured allelic RNA ratio with rs1051375 as a marker in heterozygous human ventricular heart tissues, as described previously.[12](#R12) A segment of DNA or cDNA surrounding the marker SNP was amplified by PCR using primer sets to amplify different splice variants. Primer sequences are as follows: E44F: TCGTCCACCGGCTCCA; E44R: CTGAGCTTCCACGCCACCT; E40F: GGCCCTGAGGATCAAAACAG; E46R: CACTTCATAGGTCTCATCCTGAGAC. PCR products were then subjected to a primer extension assay using extension primer (CCGGCTACCCCAGCAC). Allelic mRNA ratios were normalized by DNA ratios. If rs1051375 affects mRNA splicing, we would expect to observe differences in RNA ratios after amplification with different primer sets. Total CACNA1C mRNA in human ventricular tissues was measured using real-time PCR as described previously.[12](#R12)

### Statistical Analysis

Baseline characteristics were compared by genotype using chi-square or ANOVA, as appropriate. Hardy-Weinberg equilibrium was calculated separately by race/ethnicity using chi-square test with one degree of freedom. All statistical analyses were conducted using SAS version 9.1 (Cary, NC) or SPSS version 11.5 (Chicago, IL). Analyses by treatment strategy were based on patients receiving at least one dose of the randomized drug in their assigned strategy. A two-sided p< 0.006 (0.05/8 SNPs assessed) was considered significant for all analyses. As a method of further ensuring that we did not have excess type I errors given that there were actually more than 8 tests performed (e.g. by race, by genotype, and individual components of the primary outcome), we also performed the false discovery rate (FDR) according to the method of Benjamini and colleagues to adjust for multiple comparisons.[13](#R13) Unadjusted and adjusted odds ratios (ORs) and 95% confidence intervals (CIs) for occurrence of the primary outcome were calculated using logistic regression for the case-control group. The model contained the following covariates: age, sex, race/ethnicity, genomic ancestry proportions (see below), BMI, smoking, INVEST treatment strategy, previous MI, previous stroke, heart failure, diabetes, peripheral vascular disease, renal insufficiency, and baseline systolic blood pressure (SBP) and diastolic blood pressure (DBP). Additionally, we included genotype/haplotype and the interaction term between genotype/haplotype and treatment strategy. The modeling was also conducted separately by genotype/haplotype and by race/ethnicity. Genotype and haplotypes were entered in models according to additive models of inheritance.

Considering our case-control group, we had 80% power to detect an odds ratio of 1.83 for the primary outcome with an alpha of 0.006 and minor allele frequency of 0.10.

To control for the potential of population stratification in our racially and ethnically diverse population, we used a total panel including up to 87 ancestry informative markers (AIMs) (mean 68 ± 24 markers per subject), selected to show large allele frequency differences across three parental populations (West Africans, Indigenous Americans, and Europeans) selected from a large panel of over 10,000 SNPs [14](#R14). Maximum likelihood was then used to estimate each patient’s individual genomic ancestry proportions on these three axes and these terms were included in statistical models in addition to the race/ethnicity term. In order to ensure accurate ancestry proportion estimates, at least 30 AIMs had to be successfully genotyped in each individual to be included in analyses.

In addition to controlling for potential confounding by population stratification through inclusion of race/ethnicity and ancestry estimates in the models, we also analyzed each racial/ethnic group separately. While we do not have sufficient power in each group, we sought to determine if the direction of the association was similar in each group. When similar, we then pooled the data, still adjusting for race and ancestry.

## RESULTS

### SNP Discovery and In Silico Modeling

Upon screening with DHPLC, 8 of the 54 amplicons contained no variation based on the DHPLC chromatograms. For those 8 amplicons, only the reference sequence was sent for sequencing. All other amplicons either clearly contained samples with heteroduplex formation (30 amplicons) or were associated with some degree of ambiguity (16 amplicons) and selected samples were sent for direct sequencing.

A total of 46 polymorphisms were identified, 44 SNPs and 2 insertion/deletion polymorphisms ([Supplemental Table 2](#SD2)). The linkage disequilibrium plots for the discovered SNPs in each population are shown in [Figure 1](#F1). Thirty-one were located in introns, 11 were synonymous, and 4 were non-synonymous. Of the 46 polymorphisms, 18 (39%) were novel, not found in dbSNP. Of the novel polymorphisms, 11 were intronic, 5 synonymous, and 2 non-synonymous ([Supplemental Table 2](#SD2)).

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/900a/2761685/bdd59733ee55/nihms131313f1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=2761685_nihms131313f1.jpg)

Linkage disequilibrium plots for CACNA1C resequencing data. Linkage disequilibrium plots (with D’ values) are shown with all 46 discovered SNPs for the 3 populations resequenced: Caucasian, Native American, and African American.

The estimated minor allele frequencies of the discovered polymorphisms range from being polymorphic only in a single population to 50% ([Supplemental Table 2](#SD2)). [Figure 2](#F2) depicts all common SNPs (>5% overall minor allele frequency) mapped to the gene structure with degree of conservation with mouse shown. None of the discovered polymorphisms occurred in conserved noncoding regions ([Figure 2](#F2)), although this finding may have been biased by our resequencing strategy which focused more heavily on coding regions of the gene.

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/900a/2761685/bae3835236d6/nihms131313f2.jpg)

Conservation, CACNA1C gene structure, and SNP minor allele frequencies. Panel A shows degree of conservation with mouse. Turquoise areas indicate conserved untranslated regions, blue indicates exonic regions, and peach indicates noncoding regions. Panel B shows the CACNA1C gene structure with exons shown as boxes and introns shown as the lines connecting the boxes. Panel C shows the minor allele frequency of common SNPs and their relative location in CACNA1C.

*In silico* functional analysis of the discovered exonic SNPs in *CACNA1C* revealed potential exonic splicing enhancer (ESE) motifs in Thr1149Thr (novel, SNP23) and Thr1396Thr (novel, SNP27) for SF2/ASF and SRp40. Leu1058Leu (novel, SNP20) contains a potential ESE for SRp55. A potential ESE motif for SF2/ASF was identified in Asp812Asp (rs215976) and Asp1543Asp (rs41276710). Asn1812Asn (novel, SNP37) and Phe1282Phe (rs216008) contain potential ESE motifs for SRp40. Arg2056Gln (novel, SNP45) contains a potential ESE for SC35 and Thr1835Thr (rs1051375) for SC35 and SF2/ASF. The non-synonymous SNP, Pro1868Leu (rs10848683), was predicted to be possibly damaging. Both P1868L and M1869V (rs10774053) are located in alternate exon 45, which is not designated as an exon in NCBI’s dbSNP.

### Association Studies

Based on the resequencing data, subsequent *in silico* studies, and LD analysis, we identified five SNPs to genotype in the INVEST-GENES case-control set (rs215976, rs216008, novel SNP37, rs1051375, and rs10848683). Additionally, we genotyped the three SNPs identified by Bremer et al as being associated with calcium antagonist response (rs2239050, rs2238032, and rs2239128) for a total of eight SNPs.[4](#R4)

Genotyping was complete for 1010 (98%) for rs2238032, 972 (94%) for rs2239050, 1017 (99%) for rs215976, 961 (93%) for rs216008, 977 (95%) for SNP37, 952 (92%) for rs2239128, 975 (94%) for rs1051375, 1002 (97%) for rs10848683. Genotype frequencies are shown in [Table 1](#T1). All genotype frequencies were consistent with those predicted by Hardy-Weinberg equilibrium, with the exception of rs2239050, which was out of Hardy-Weinberg equilibrium in Hispanic individuals (p=0.035). When genotype frequencies for rs2239050 were calculated among only those with greater than 75% European ancestry, they were consistent with those predicted by Hardy-Weinberg equilibrium, suggesting that this SNP may have been out of equilibrium due to admixture among the Hispanic individuals. The pairwise LD plots for each of the populations are shown in [Supplemental Figure 1](#SD2). Baseline characteristics for cases and controls are shown in [Table 2](#T2). As outlined in the methods, any differences in baseline characteristics between cases and controls were included as covariates in the analysis. Smoking was less common, BMI was greater, and angina history was more prevalent in rs1051375 variant homozygote G/G than in A/G or A/A individuals and the minor allele frequency was lower in Caucasians than other racial/ethnic groups. All characteristics were similar by other genotype groups with the exception of race (rs2238032, rs2239050, rs2239128, and rs10848683), BMI (rs2239050), baseline DBP (rs10848683, rs2239128, rs2238032, and rs2239050), and history of arrhythmia (SNP37) (data not shown). The differences in BMI and DBP by genotype at baseline appeared to be due to racial differences in allele frequencies because they were no longer significant when compared in Caucasians only.

### Table 1.

|   | INVEST case-control (n=1032) | Caucasian (n=616) | Hispanic (n=262) | African American (n=131) | Other (n=23) |
| --- | --- | --- | --- | --- | --- |
| rs2238032 |   |   |   |   |   |
| T/T | 936 | 593 | 233 | 87 | 23 |
| T/G | 72 | 6 | 27 | 39 | 0 |
| G/G | 2 | 0 | 0 | 2 | 0 |
| m.a.f. | 0.04 | 0.005 | 0.05 | 0.17 | 0.0 |
| rs2239050 |   |   |   |   |   |
| G/G | 634 | 455 | 145 | 17 | 17 |
| G/C | 250 | 109 | 85 | 52 | 4 |
| C/C | 88 | 8 | 24 | 54 | 2 |
| m.a.f. | 0.22 | 0.11 | 0.26 | 0.65 | 0.17 |
| rs215976 |   |   |   |   |   |
| C/C | 853 | 512 | 217 | 103 | 21 |
| C/T | 160 | 92 | 40 | 26 | 3 |
| T/T | 4 | 1 | 2 | 1 | 0 |
| m.a.f. | 0.08 | 0.08 | 0.08 | 0.11 | 0.06 |
| rs216008 |   |   |   |   |   |
| C/C | 639 | 392 | 165 | 69 | 13 |
| C/T | 278 | 160 | 71 | 40 | 7 |
| T/T | 44 | 23 | 10 | 10 | 1 |
| m.a.f. | 0.19 | 0.18 | 0.19 | 0.25 | 0.21 |
| rs2239128 |   |   |   |   |   |
| C/C | 511 | 349 | 128 | 25 | 8 |
| C/T | 357 | 192 | 100 | 55 | 10 |
| T/T | 84 | 27 | 14 | 41 | 2 |
| m.a.f. | 0.28 | 0.22 | 0.26 | 0.57 | 0.35 |
| SNP37 |   |   |   |   |   |
| C/C | 865 | 499 | 226 | 117 | 23 |
| T/C | 108 | 78 | 24 | 6 | 0 |
| T/T | 4 | 4 | 0 | 0 | 0 |
| m.a.f. | 0.06 | 0.07 | 0.05 | 0.02 | 0.0 |
| rs1051375 |   |   |   |   |   |
| A/A | 396 | 273 | 96 | 17 | 10 |
| A/G | 402 | 237 | 108 | 48 | 9 |
| G/G | 177 | 71 | 42 | 60 | 4 |
| m.a.f. | 0.39 | 0.33 | 0.39 | 0.67 | 0.37 |
| rs10848683 |   |   |   |   |   |
| T/T | 580 | 406 | 135 | 28 | 11 |
| T/C | 340 | 170 | 91 | 70 | 9 |
| C/C | 82 | 25 | 25 | 29 | 3 |
| m.a.f. | 0.25 | 0.18 | 0.28 | 0.50 | 0.33 |

Table 1 Caption: Genotype and Allele Frequencies in Overall Group and by Race/ethnicity

### Table 2.

| Characteristic | Cases (N = 258) | Controls (N = 774) |
| --- | --- | --- |
| Age, mean (SD), years | 71.5 (9.9) | 70.2 (9.3) |
| Women | 131 (51%) | 393 (51%) |
| BP, mean (SD), mmHg |   |   |
| Systolic | 150.6 (19.0) | 147.5 (18.9) |
| Diastolic | 83.6 (11.1) | 83.3 (11.1) |
| Race/ethnicity |   |   |
| White | 154 (60%) | 462 (60%) |
| Black | 33 (13%) | 98 (13%) |
| Hispanic | 63 (24%) | 199 (26%) |
| Other/multiracial | 8 (3%) | 15 (2%) |
| BMI, mean (SD), kg/m2 | 27.4 (4.8) | 29.0 (5.5) |
| Myocardial infarction | 96 (37%) | 230 (30%) |
| Angina pectoris | 153 (59%) | 483 (62%) |
| Revascularization >1 month ago | 103 (40%) | 272 (35%) |
| Stroke/TIA | 36 (14%) | 71 (9%) |
| Left ventricular hypertrophy | 46 (18%) | 136 (18%) |
| Heart failure (class I-III) | 28 (11%) | 29 (4%) |
| Peripheral vascular disease | 43 (17%) | 88 (11%) |
| Arrhythmia | 25 (10%) | 74 (10%) |
| Smoking |   |   |
| Past | 133 (52%) | 355 (46%) |
| Within 30 days | 34 (13%) | 83 (11%) |
| Never |   |   |
| Diabetes‡ | 102 (40%) | 224 (29%) |
| Hypercholesterolemia‡ | 161 (62%) | 485 (63%) |
| Renal impairment† | 14 (5%) | 18 (2%) |
| Treatment randomization, verapamil SR | 137 (53%) | 372 (48%) |
| Treatment with all three study drugs (randomized plus trandolapril plus HCTZ) | 119 (46%) | 457 (59%) |

Table 2 Caption: Baseline Characteristics

In INVEST-GENES, the crossover rate to beta-blocker use in the calcium channel blocker arm was 0% and crossover to calcium channel blocker use in the beta-blocker arm was 0.03%. As allowed per the INVEST protocol for refractory angina, calcium channel blocker use at any time in the beta-blocker strategy was 18.73% and beta-blocker use at any time in the CCB strategy was 12.44%.

### Main Effects of CACNA1C SNPs on Outcomes

None of the eight SNPs genotyped were associated with a main effect on outcomes with a p<0.006. Two SNPs exhibited trends relative to the composite primary outcome with p values < 0.05 but > 0.006, rs2238032 (OR 0.41 95% confidence interval [CI] 0.21-0.83, p=0.01) and rs10848683 (OR 0.74 95% CI 0.56-0.98, p=0.03). The main effect of both SNPs remained consistent with AIMs included in the model.

### Treatment x Genotype Interactions

One SNP was identified with a significant interaction with treatment strategy, rs1051375 (p=0.0001) ([Table 3](#T3)). This SNP remained significant with FDR adjustment (p=0.01). Rs10848683 trended toward a significant interaction with treatment strategy (p=0.10) and rs2238032, which had a modest main effect, had no evidence for a pharmacogenetic effect on outcomes (p=0.95). Rs1051375 and rs10848683 are in significant LD with D’=0.87 and r^2^=0.36 in Caucasians. We performed analyses stratified by genotype for rs1051375 and found that A/A individuals randomized to verapamil SR were less likely to experience a primary outcome than those randomized to atenolol ([Table 3](#T3) and [Figure 3](#F3)). G/G individuals randomized to verapamil SR were more likely to experience the primary outcome than those randomized to atenolol ([Table 3](#T3) and [Figure 3](#F3)). No difference in the occurrence of the primary outcome was noted between treatment groups in heterozygous individuals ([Table 3](#T3) and [Figure 3](#F3)). When Caucasians, our largest racial group, were analyzed alone, the effects of rs1051375 remained consistent ([Table 3](#T3)). Rs1051375’s interactions were consistent across all components of the primary outcome (p=0.01 for death, p=0.01 for myocardial infarction, and p=0.06 for stroke).

### Table 3.

| Main effect | SNP*ASSIGN p value | A/A | A/G | G/G |
| --- | --- | --- | --- | --- |
| Primary outcome |  |  |  |  |
| INVEST-GENES case control |  |  |  |  |
| 0.87 (0.68-1.10) | 0.0001 | 0.54 (0.32-0.92) | 1.47 (0.86-2.53) | 4.59 (1.67-12.67) |
| Caucasians only (n=631) |  |  |  |  |
| 0.91 (0.66-1.26) | 0.002 | 0.38 (0.18-0.79) | 1.22 (0.61-2.45) | >999 (0.036->999) |
| Death |  |  |  |  |
| INVEST-GENES case control |  |  |  |  |
| 0.99 (0.69-1.36) | 0.01 | 0.98 (0.47-2.02) | 1.07 (0.42-2.74) | 7.76 (1.57-38.3) |
| Cardiac death |  |  |  |  |
| 0.93 (0.56-1.54) | 0.16 | 1.07 (0.38-2.99) | 0.58 (0.13-2.61) | 179.8 (0.32->999) |
| Myocardial infarction |  |  |  |  |
| 0.68 (0.48-0.96) | 0.01 | 0.57 (0.29-1.13) | 0.77 (0.35-1.69) | 4.37 (0.68-28.3) |
| Stroke |  |  |  |  |
| 1.19 (0.84-1.68) | 0.06 | 0.45 (0.18-1.10) | 2.27 (1.01-5.13) | 1.77 (0.48-6.49) |

Table 3 Caption: Adjusted Odds Ratios and 95% Confidence Intervals for Main Effects and Verapamil SR Effects for rs1051375

### Figure 3.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/900a/2761685/bfa5ffeec4e3/nihms131313f3.jpg)

Adjusted odds ratios and 95% confidence intervals for treatment strategy by rs1051375 genotype. Reference is atenolol treatment strategy. Genotype*treatment strategy interaction p values=0.0001. CCB=calcium channel blocker (verapamil SR) treatment arm. BB=β-blocker (atenolol) treatment arm.

In post hoc analysis, we also assessed average treatment blood pressure and the number of antihypertensive drugs required by rs1051375 genotype, given that the INVEST study design called for the addition of additional drugs in order to achieve blood pressure goals. The average treatment blood pressure was 135.7/76.5 ± 11.5/7.2 mmHg in A/As, 134.5/76.6 ± 11.0/6.9 mmHg in A/Gs, and 138.3/78.2 ± 12.2/7.0 mmHg in G/Gs (overall ANOVA p=0.001 for SBP and p=0.024 for DBP). This trend for a higher treatment blood pressure in G/G individuals was similar in the atenolol-and verapamil SR-based treatment groups. Consistent with worse outcomes among rs1051375 A/A patients in the atenolol strategy, significantly more A/A patients randomized to the atenolol strategy (41%) required ≥four drugs (including nonstudy drugs) for blood pressure control than those randomized to the verapamil SR strategy (24%), p=0.0005.

### Haplotype Analysis

We conducted haplotype analyses in two ways. First, we imputed haplotypes across all eight SNPs genotyped in *CACNA1C*. Second, because of the relatively low degree of LD across the gene and the fact that a block was evident in Caucasians across SNP37, rs1051375, and rs10848683, we assigned haplotypes for just those three SNPs. The haplotype analysis results were largely consistent with the single SNP analyses, whereby individuals with haplotypes containing the rs1051375 A allele had improved outcomes with verapamil SR randomization and those with haplotypes contraining rs1051375 G had worse outcomes with verapmail SR than with atenolol (data not shown).

### Functional Assessment

When using rs1051375 as a marker to measure allelic mRNA expression in heart tissues, we did not identify allelic expression imbalance (AEI) in CACNA1C ([Figure 4](#F4)), indicating this SNP does not affect mRNA expression. From E40 to E46, six splice variants have been reported, E40-E41, E40-E41+57nt, E44-E46, E44-E45*-E46, E44-E45-E46 and E44-E45*-187nt-E46, but the major splice variants E40-E41 and E44-E46 comprise more than 95% of total transcripts. If rs1051375 was to affect the splicing in this locus, and the splice variants are unstable or undergo nonsense mediated decay, we would see AEI when using primers spanning different exons, like E40F/E44R and E44F/E46R. However, this result failed to identify AEI after PCR amplification using different primers ([Figure 4](#F4)), suggesting that rs1051375 does not affect mRNA splicing in this locus in ventricular tissue. We also measured CACNA1C mRNA expression in ventricular tissue by rs1051375 genotype and did not find differences ([Figure 5](#F5)).

### Figure 4.

![Figure 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/900a/2761685/98f264d6eef8/nihms131313f4.jpg)

Allelic mRNA expression in 8 heart samples using rs1051375 as marker. cDNAs were amplified using three pairs of primers as indicated at the top of the figure. Results are normalized to DNA ratio.

### Figure 5.

![Figure 5](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/900a/2761685/c43c3f7e33bd/nihms131313f5.jpg)

CACNA1C mRNA expression grouped by rs1051375 genotypes. CACNA1C mRNA expression was measured by primers spanning exon 3 and exon 4, and normalized by β-actin expression.

## DISCUSSION

We performed an extensive SNP discovery effort of the *CACNA1C* coding region and intron/exon junctions and a clinical association study using the INVEST-GENES to assess the impact of *CACNA1C* genetic variation on outcomes and treatment response. Of eight SNPs tested in the genetic association study, we identified one SNP with a significant interaction with treatment strategy. This effect of this interaction was such that individuals homozygous for the major allele (A/A) randomized to verapamil SR-based treatment regimens had a 45% reduced risk of the primary outcome compared to A/A individuals randomized to atenolol-based regimens. In contrast, individuals homozygous for the minor allele (G/G) randomized to verapamil SR-based treatment had a 4.5-fold increase in the primary outcome compared to G/G individuals randomized to atenolol-based treatment. These findings suggest that individuals with rs1051375 A/A would benefit from treatment with a calcium channel blocker, those with the G/G genotype would benefit from treatment with a beta blocker, and in those with the heterozygous genotype it would not matter which treatment was chosen.

Given the study design of INVEST, it is difficult to determine whether the differences in treatment outcomes observed are due to differences in blood pressure response. In our post hoc analysis, we observed overall average treatment blood pressures that were higher among those with the G/G genotype. Additionally, patients with the A/A genotype who were randomized to the atenolol treatment strategy were more likely to require four or more drugs for blood pressure control than A/A patients randomized to the verapamil SR treatment strategy, suggesting that blood pressure response differences may be playing a role in differences in treatment outcomes.

Although the mechanism of the SNP*treatment interaction is unclear at this time, one explanation is that variants in *CACNA1C* result in reduced function of the L-type calcium channel. If this were the case, individuals with these genotypes might gain more benefit from a treatment approach involving a mechanism of action not dependent on the L-type calcium channel (e.g. beta-blocker instead of calcium channel blocker). In contrast, the major alleles have greater L-type calcium channel function and thus benefit more from treatment with calcium channel blockade. Last, it is possible that variation in *CACNA1C* might directly influence treatment response through interactions between calcium signaling and beta-adrenergic signaling pathways since protein kinase A activation via the beta-1-adrenergic receptor results in activation of the L-type calcium channel.

Two of the SNPs we found to be associated with outcomes or treatment response, rs1051375 and rs10848683, are in a fair degree of LD (D’=0.84 and r^2^=0.36 in Caucasians). Therefore, it is unclear whether either or both of these SNPs are functional, or both might be tagSNPs for the actual functional SNP. The results of our haplotype analysis as well as the more significant p values in individual SNP analysis suggest that rs1051375 is the more likely causative SNP of the two or in stronger LD with the causative SNP. Rs1051375 is synonymous, Thr1835Thr, so the functional relevance of this SNP is not immediately clear. It was selected for analysis because it is located at putative ESE sites for SC35 and SF2/ASF. In addition, other synonymous SNPs have recently been identified as having functional importance in the *ABCB1* gene.[15](#R15) Whether one or both of these SNPs are functional or whether they are linked to another functional SNP needs to be elucidated.

Although we have not yet discovered the mechanisms underlying our observed associations, we have made substantial progress toward eliminating several possible functional mechanisms. Based on our work presented here and previously published, we can now exclude differences in expression levels in the myocardium by genotype and that rs1051375 alters splicing in myocardial tissue as potential explanations for the functional basis.[12](#R12) It is still possible, however, that expression or splicing differences in vasculature smooth muscle may exist.

We were unable to replicate the findings of Bremer et al in which rs2239050, rs2238032, and rs2239128 were found to be associated with calcium channel blocker antihypertensive response. However, rs2238032 was associated with a main effect on outcomes in our study, although it did not meet our predefined level for significance after adjustment for multiple comparisons. An additional study published during the revision of this manuscript found one SNP in *CACNA1C*, although it was not one of the SNPs we assessed, and two SNPs in *CACNA1D* to be associated with response to dihydropyridine CCBs among 161 Japanese individuals.[5](#R5)

One of the major strengths of our study is the fact that our population came from a randomized clinical trial. This study design greatly reduces possible biases that can be introduced into observational studies where many factors influence treatment decisions. Additionally, all endpoints in the clinical trial were adjudicated by a blinded endpoints committee which also strengthens our phenotype.

### Limitations

Our study has some limitations that should be addressed. First, although we have eliminated some possible functional mechanisms, we do not know the functional mechanism underlying the association we observed between *CACNA1C* variants and cardiovascular outcomes. These mechanistic studies will be important in order to understand how we might use this information in the future for genotype-guided treatment decisions. A second limitation of our study is that we do not have a replication cohort for our findings. Unfortunately, it is very challenging to find replication cohorts for pharmacogenetic studies with detailed drug phenotype data and similar patient populations, particularly when the phenotype is adverse cardiovascular outcomes. However, we are working toward replicating these findings in other cardiovascular outcomes studies.

### Conclusions

We have identified a SNP in *CACNA1C*, the binding site for calcium channel blockers, with a significant interaction with treatment strategy in a group of hypertensive patients with CAD. Individuals homozygous for the major allele had a reduction in the occurrence of death, nonfatal MI, or nonfatal stroke when treated with a calcium channel blocker-based treatment regimen compared to those treated with a beta-blocker-based treatment regimen. In contrast, individuals homozygous for the minor allele had a reduction in adverse outcomes when treated with beta-blocker-based regimens. If validated, these findings might be used in the future to help guide choice of therapy in the treatment of hypertension. Of great interest, our findings suggest the potential of targeting an individual’s underlying molecular mechanism of disease to improve clinical outcomes.

## Supplementary Material

## Acknowledgments

**Funding Sources**: This study was supported by NIH grants HL074730, GM074492, and RR017568 and grants from the University of Florida Opportunity Fund and Abbott Laboratories.

**Disclosures**: NIH research grants (ALB, YG, RMCD, CJP, NJS, JAJ), CJP reports receiving research grants from Baxter, Bioheart, Cardium, Pfizer, Viron, Abbott, and Berlex Lab/Bayer HealthCare, RMCD reports receiving honoraria from the Preventive Cardiovascular Nurses Association and the American College of Cardiology, CJP reports serving as a consultant for Abbott as DSMB Chair, Forest Laboratories, Novartis/Cleveland Clinic DSMB Chair, Pfizer, CV Therapeutics, NicOx, Angiobalst DSMB member, Indigo, Boerhinger Ingleheim, DCRI/The Medicines Company-Interim Analysis Committee Research, Reliant Pharmaceuticals, Schering-Plough, and Sanofi Aventis, CJP reports receiving unrestricted educational grants from Astra Zeneca, Boehringer Ingelheim, CV Therapeutics, Pfizer, Sanofi Aventis, Schering Plough, Daiichi-Sankyo, Merck, Novartis, The Medicines Company, GSK, and Reliant Pharmaceuticals.

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Abernethy DR, Schwartz JB. Calcium-antagonist drugs. N Engl J Med. 1999;341:1447–1457. doi: 10.1056/NEJM199911043411907.  [DOI](https://doi.org/10.1056/NEJM199911043411907) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10547409/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Calcium-antagonist%20drugs&author=DR%20Abernethy&author=JB%20Schwartz&volume=341&publication_year=1999&pages=1447-1457&pmid=10547409&doi=10.1056/NEJM199911043411907&)

2. Soldatov NM. Genomic structure of human L-type Ca2+ channel. Genomics. 1994;22:77–87. doi: 10.1006/geno.1994.1347.  [DOI](https://doi.org/10.1006/geno.1994.1347) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7959794/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genomics&title=Genomic%20structure%20of%20human%20L-type%20Ca2+%20channel&author=NM%20Soldatov&volume=22&publication_year=1994&pages=77-87&pmid=7959794&doi=10.1006/geno.1994.1347&)

3. Beitelshees AL, Gong Y, Wang D, Schork NJ, Cooper-Dehoff RM, Langaee TY, Shriver MD, Sadee W, Knot HJ, Pepine CJ, Johnson JA. KCNMB1 genotype influences response to verapamil SR and adverse outcomes in the INternational VErapamil SR/Trandolapril STudy (INVEST) Pharmacogenet Genomics. 2007;17:719–729. doi: 10.1097/FPC.0b013e32810f2e3c.  [DOI](https://doi.org/10.1097/FPC.0b013e32810f2e3c) | [PMC free article](/articles/PMC2713584/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17700361/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=KCNMB1%20genotype%20influences%20response%20to%20verapamil%20SR%20and%20adverse%20outcomes%20in%20the%20INternational%20VErapamil%20SR/Trandolapril%20STudy%20(INVEST)&author=AL%20Beitelshees&author=Y%20Gong&author=D%20Wang&author=NJ%20Schork&author=RM%20Cooper-Dehoff&volume=17&publication_year=2007&pages=719-729&pmid=17700361&doi=10.1097/FPC.0b013e32810f2e3c&)

4. Bremer T, Man A, Kask K, Diamond C. CACNA1C polymorphisms are associated with the efficacy of calcium channel blockers in the treatment of hypertension. Pharmacogenomics. 2006;7:271–279. doi: 10.2217/14622416.7.3.271.  [DOI](https://doi.org/10.2217/14622416.7.3.271) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16610939/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=CACNA1C%20polymorphisms%20are%20associated%20with%20the%20efficacy%20of%20calcium%20channel%20blockers%20in%20the%20treatment%20of%20hypertension&author=T%20Bremer&author=A%20Man&author=K%20Kask&author=C%20Diamond&volume=7&publication_year=2006&pages=271-279&pmid=16610939&doi=10.2217/14622416.7.3.271&)

5. Kamide K, Yang J, Matayoshi T, Takiuchi S, Horio T, Yoshii M, Miwa Y, Yasuda H, Yoshihara F, Nakamura S, Nakahama H, Miyata T, Kawano Y. Genetic Polymorphisms of L-Type Calcium Channel alpha1(C) and alpha1(D) Subunit Genes are Associated With Sensitivity to the Antihypertensive Effects of L-Type Dihydropyridine Calcium-Channel Blockers. Circ J. 2009;73:732–740. doi: 10.1253/circj.cj-08-0761.  [DOI](https://doi.org/10.1253/circj.cj-08-0761) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19225208/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circ%20J&title=Genetic%20Polymorphisms%20of%20L-Type%20Calcium%20Channel%20alpha1(C)%20and%20alpha1(D)%20Subunit%20Genes%20are%20Associated%20With%20Sensitivity%20to%20the%20Antihypertensive%20Effects%20of%20L-Type%20Dihydropyridine%20Calcium-Channel%20Blockers&author=K%20Kamide&author=J%20Yang&author=T%20Matayoshi&author=S%20Takiuchi&author=T%20Horio&volume=73&publication_year=2009&pages=732-740&pmid=19225208&doi=10.1253/circj.cj-08-0761&)

6. Freimuth RR, Stormo GD, McLeod HL. PolyMAPr: programs for polymorphism database mining, annotation, and functional analysis. Hum Mutat. 2005;25:110–117. doi: 10.1002/humu.20123.  [DOI](https://doi.org/10.1002/humu.20123) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15643605/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mutat&title=PolyMAPr:%20programs%20for%20polymorphism%20database%20mining,%20annotation,%20and%20functional%20analysis&author=RR%20Freimuth&author=GD%20Stormo&author=HL%20McLeod&volume=25&publication_year=2005&pages=110-117&pmid=15643605&doi=10.1002/humu.20123&)

7. Pepine CJ, Handberg EM, Cooper-DeHoff RM, Marks RG, Kowey P, Messerli FH, Mancia G, Cangiano JL, Garcia-Barreto D, Keltai M, Erdine S, Bristol HA, Kolb HR, Bakris GL, Cohen JD, Parmley WW. A calcium antagonist vs a non-calcium antagonist hypertension treatment strategy for patients with coronary artery disease. The International Verapamil-Trandolapril Study (INVEST): a randomized controlled trial. JAMA. 2003;290:2805–2816. doi: 10.1001/jama.290.21.2805.  [DOI](https://doi.org/10.1001/jama.290.21.2805) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14657064/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA&title=A%20calcium%20antagonist%20vs%20a%20non-calcium%20antagonist%20hypertension%20treatment%20strategy%20for%20patients%20with%20coronary%20artery%20disease.%20The%20International%20Verapamil-Trandolapril%20Study%20(INVEST):%20a%20randomized%20controlled%20trial&author=CJ%20Pepine&author=EM%20Handberg&author=RM%20Cooper-DeHoff&author=RG%20Marks&author=P%20Kowey&volume=290&publication_year=2003&pages=2805-2816&pmid=14657064&doi=10.1001/jama.290.21.2805&)

8. Gerhard T, Gong Y, Beitelshees AL, Mao X, Lobmeyer MT, Cooper-DeHoff RM, Langaee TY, Schork NJ, Shriver MD, Pepine CJ, Johnson JA. Alpha-adducin polymorphism associated with increased risk of adverse cardiovascular outcomes: results from GENEtic Substudy of the INternational VErapamil SR-trandolapril STudy (INVEST-GENES) Am Heart J. 2008;156:397–404. doi: 10.1016/j.ahj.2008.03.007.  [DOI](https://doi.org/10.1016/j.ahj.2008.03.007) | [PMC free article](/articles/PMC2547143/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18657677/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20Heart%20J&title=Alpha-adducin%20polymorphism%20associated%20with%20increased%20risk%20of%20adverse%20cardiovascular%20outcomes:%20results%20from%20GENEtic%20Substudy%20of%20the%20INternational%20VErapamil%20SR-trandolapril%20STudy%20(INVEST-GENES)&author=T%20Gerhard&author=Y%20Gong&author=AL%20Beitelshees&author=X%20Mao&author=MT%20Lobmeyer&volume=156&publication_year=2008&pages=397-404&pmid=18657677&doi=10.1016/j.ahj.2008.03.007&)

9. Pacanowski MA, Gong Y, Cooper-Dehoff RM, Schork NJ, Shriver MD, Langaee TY, Pepine CJ, Johnson JA. beta-adrenergic receptor gene polymorphisms and beta-blocker treatment outcomes in hypertension. Clin Pharmacol Ther. 2008;84:715–721. doi: 10.1038/clpt.2008.139.  [DOI](https://doi.org/10.1038/clpt.2008.139) | [PMC free article](/articles/PMC2675574/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18615004/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=beta-adrenergic%20receptor%20gene%20polymorphisms%20and%20beta-blocker%20treatment%20outcomes%20in%20hypertension&author=MA%20Pacanowski&author=Y%20Gong&author=RM%20Cooper-Dehoff&author=NJ%20Schork&author=MD%20Shriver&volume=84&publication_year=2008&pages=715-721&pmid=18615004&doi=10.1038/clpt.2008.139&)

10. Andrisin TE, Humma LM, Johnson JA. Collection of genomic DNA by the noninvasive mouthwash method for use in pharmacogenetic studies. Pharmacotherapy. 2002;22:954–960. doi: 10.1592/phco.22.12.954.33598.  [DOI](https://doi.org/10.1592/phco.22.12.954.33598) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12173798/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacotherapy&title=Collection%20of%20genomic%20DNA%20by%20the%20noninvasive%20mouthwash%20method%20for%20use%20in%20pharmacogenetic%20studies&author=TE%20Andrisin&author=LM%20Humma&author=JA%20Johnson&volume=22&publication_year=2002&pages=954-960&pmid=12173798&doi=10.1592/phco.22.12.954.33598&)

11. Schaid DJ, McDonnell SK, Wang L, Cunningham JM, Thibodeau SN. Caution on pedigree haplotype inference with software that assumes linkage equilibrium. Am J Hum Genet. 2002;71:992–995. doi: 10.1086/342666.  [DOI](https://doi.org/10.1086/342666) | [PMC free article](/articles/PMC378555/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12387273/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Caution%20on%20pedigree%20haplotype%20inference%20with%20software%20that%20assumes%20linkage%20equilibrium&author=DJ%20Schaid&author=SK%20McDonnell&author=L%20Wang&author=JM%20Cunningham&author=SN%20Thibodeau&volume=71&publication_year=2002&pages=992-995&pmid=12387273&doi=10.1086/342666&)

12. Wang D, Papp AC, Binkley PF, Johnson JA, Sadee W. Highly variable mRNA expression and splicing of L-type voltage-dependent calcium channel alpha subunit 1C in human heart tissues. Pharmacogenet Genomics. 2006;16:735–745. doi: 10.1097/01.fpc.0000230119.34205.8a.  [DOI](https://doi.org/10.1097/01.fpc.0000230119.34205.8a) | [PMC free article](/articles/PMC2688811/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17001293/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Highly%20variable%20mRNA%20expression%20and%20splicing%20of%20L-type%20voltage-dependent%20calcium%20channel%20alpha%20subunit%201C%20in%20human%20heart%20tissues&author=D%20Wang&author=AC%20Papp&author=PF%20Binkley&author=JA%20Johnson&author=W%20Sadee&volume=16&publication_year=2006&pages=735-745&pmid=17001293&doi=10.1097/01.fpc.0000230119.34205.8a&)

13. Benjamini Y, Hochberg Y. Controlling the false discovery rate: a practical and powerful approach to multiple testing. Journal of the Royal Statistical Society, Series B. 1995;57:289–300.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20the%20Royal%20Statistical%20Society,%20Series%20B&title=Controlling%20the%20false%20discovery%20rate:%20a%20practical%20and%20powerful%20approach%20to%20multiple%20testing&author=Y%20Benjamini&author=Y%20Hochberg&volume=57&publication_year=1995&pages=289-300&)

14. Shriver MD, Mei R, Parra EJ, Sonpar V, Halder I, Tishkoff SA, Schurr TG, Zhadanov SI, Osipova LP, Brutsaert TD, Friedlaender J, Jorde LB, Watkins WS, Bamshad MJ, Gutierrez G, Loi H, Matsuzaki H, Kittles RA, Argyropoulos G, Fernandez JR, Akey JM, Jones KW. Large-scale SNP analysis reveals clustered and continuous patterns of human genetic variation. Hum Genomics. 2005;2:81–89. doi: 10.1186/1479-7364-2-2-81.  [DOI](https://doi.org/10.1186/1479-7364-2-2-81) | [PMC free article](/articles/PMC3525270/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16004724/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Genomics&title=Large-scale%20SNP%20analysis%20reveals%20clustered%20and%20continuous%20patterns%20of%20human%20genetic%20variation&author=MD%20Shriver&author=R%20Mei&author=EJ%20Parra&author=V%20Sonpar&author=I%20Halder&volume=2&publication_year=2005&pages=81-89&pmid=16004724&doi=10.1186/1479-7364-2-2-81&)

15. Kimchi-Sarfaty C, Oh JM, Kim IW, Sauna ZE, Calcagno AM, Ambudkar SV, Gottesman MM. A “silent” polymorphism in the MDR1 gene changes substrate specificity. Science. 2007;315:525–528. doi: 10.1126/science.1135308.  [DOI](https://doi.org/10.1126/science.1135308) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17185560/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=A%20%E2%80%9Csilent%E2%80%9D%20polymorphism%20in%20the%20MDR1%20gene%20changes%20substrate%20specificity&author=C%20Kimchi-Sarfaty&author=JM%20Oh&author=IW%20Kim&author=ZE%20Sauna&author=AM%20Calcagno&volume=315&publication_year=2007&pages=525-528&pmid=17185560&doi=10.1126/science.1135308&)
