# Positive Association between Aspirin-Intolerant Asthma and Genetic Polymorphisms of FSIP1: a Case-Case Study

## Metadata
**Authors:** Jason Yongha Kim, Jeong Hyun Kim, Tae Joon Park, Joon Seol Bae, Jin Sol Lee, Charisse Flerida Pasaje, Byung Lae Park, Hyun Sub Cheong, Jong-Sook Park, Sung-Woo Park, Soo-Taek Uh, Mi-Kyeong Kim, Inseon S Choi, Sang Heon Cho, Byoung Whui Choi, Choon-Sik Park, Hyoung Doo Shin
**Journal:** BMC Pulmonary Medicine
**Date:** 2010 Jun 1
**DOI:** [10.1186/1471-2466-10-34](https://doi.org/10.1186/1471-2466-10-34)
**PMID:** 20513247
**PMCID:** PMC2896935
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2896935/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC2896935/pdf/1471-2466-10-34.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC2896935/pdf/1471-2466-10-34.pdf)

## Abstract

**Background:** 
Aspirin-intolerant asthma (AIA), which is caused by non-steroidal anti-inflammatory drugs (NSAIDs) such as aspirin, causes lung inflammation and reversal bronchi reduction, leading to difficulty in breathing. Aspirin is known to affect various parts inside human body, ranging from lung to spermatogenesis. FSIP1, also known as HDS10, is a recently discovered gene that encodes fibrous sheath interacting protein 1, and is regulated by amyloid beta precursor protein (APP). Recently, it has been reported that a peptide derived from APP is cleaved by α disintegrin and metalloproteinase 33 (ADAM33), which is an asthma susceptibility gene. It has also been known that the FSIP1 gene is expressed in airway epithelium.

**Objectives:** 
Aim of this study is to find out whether FSIP1 polymorphisms affect the onset of AIA in Korean population, since it is known that AIA is genetically affected by various genes.

**Methods:** 
We conducted association study between 66 single nucleotide polymorphisms (SNPs) of the FSIP1 gene and AIA in total of 592 Korean subjects including 163 AIA and 429 aspirin-tolerant asthma (ATA) patients. Associations between polymorphisms of FSIP1 and AIA were analyzed with sex, smoking status, atopy, and body mass index (BMI) as covariates.

**Results:** 
Initially, 18 SNPs and 4 haplotypes showed associations with AIA. However, after correcting the data for multiple testing, only one SNP showed an association with AIA (corrected P-value = 0.03, OR = 1.63, 95% CI = 1.23-2.16), showing increased susceptibility to AIA compared with that of ATA cases. Our findings suggest that FSIP1 gene might be a susceptibility gene for aspirin intolerance in asthmatics.

**Conclusion:** 
Although our findings did not suggest that SNPs of FSIP1 had an effect on the reversibility of lung function abnormalities in AIA patients, they did show significant evidence of association between the variants in FSIP1 and AIA occurrence among asthmatics in a Korean population.

### Background

Aspirin-intolerant asthma (AIA), which is caused by non-steroidal anti-inflammatory drugs (NSAIDs) such as aspirin, causes lung inflammation and reversal bronchi reduction, leading to difficulty in breathing. Aspirin is known to affect various parts inside human body, ranging from lung to spermatogenesis. *FSIP1*, also known as *HDS10*, is a recently discovered gene that encodes fibrous sheath interacting protein 1, and is regulated by amyloid beta precursor protein (APP). Recently, it has been reported that a peptide derived from APP is cleaved by α disintegrin and metalloproteinase 33 (*ADAM33*), which is an asthma susceptibility gene. It has also been known that the *FSIP1 *gene is expressed in airway epithelium.

### Objectives

Aim of this study is to find out whether *FSIP1 *polymorphisms affect the onset of AIA in Korean population, since it is known that AIA is genetically affected by various genes.

### Methods

We conducted association study between 66 single nucleotide polymorphisms (SNPs) of the *FSIP1 *gene and AIA in total of 592 Korean subjects including 163 AIA and 429 aspirin-tolerant asthma (ATA) patients. Associations between polymorphisms of *FSIP1 *and AIA were analyzed with sex, smoking status, atopy, and body mass index (BMI) as covariates.

### Results

Initially, 18 SNPs and 4 haplotypes showed associations with AIA. However, after correcting the data for multiple testing, only one SNP showed an association with AIA (corrected *P*-value = 0.03, OR = 1.63, 95% CI = 1.23-2.16), showing increased susceptibility to AIA compared with that of ATA cases. Our findings suggest that *FSIP1 *gene might be a susceptibility gene for aspirin intolerance in asthmatics.

### Conclusion

Although our findings did not suggest that SNPs of *FSIP1 *had an effect on the reversibility of lung function abnormalities in AIA patients, they did show significant evidence of association between the variants in *FSIP1 *and AIA occurrence among asthmatics in a Korean population.

## Background

Asthma is a disease that affects a large number of people globally, which is estimated to be about 300 million worldwide [[1](#B1)] and about 3 million asthma patients in South Korea. The disease is defined by the lung inflammation and difficulty of breathing when the suspect is under the influence of various factors that trigger the asthma reaction. Among these factors, non-steroidal anti-inflammatory drugs such as aspirin are known to cause aspirin-intolerant asthma. The AIA was first described in 1922 [[2](#B2),[3](#B3)], and its most noticeable symptoms include aspirin sensitivity, bronchial asthma, and chronic rhinosinusitis with nasal polyposis [[4](#B4)-[6](#B6)]. When non-steroidal anti-inflammatory drugs such as aspirin are ingested, the drugs cause bronchoconstriction in the patients. Aspirin, also known as acetylsalicylic acid, is primarily used as pain and fever reliever, and as an anti-inflammatory medication. One function of aspirin is decrease of prostaglandin production, with the most significant effect of decrease in the prostaglandin on the alleviation of inflammation and pain.

Recently, there have been several reports that found the relation between AIA and genetic polymorphisms, such as Adenosine A1 receptor (*ADORA1*) polymorphisms 1405C > T and A102A [[5](#B5)]; several polymorphisms in prostaglandin receptors and thromboxane receptor [[7](#B7)]; Interleukin 10 (*IL10*) polymorphism -1082 A > G [[8](#B8)]; cysteinyl leukotriene receptor 1 (*CYSLTR1*) promoter polymorphism -634C > T [[6](#B6)]; angiotensin I converting enzyme (*ACE*) polymorphisms -262 A > T and -115 T > C [[9](#B9)]; and Fc Fragment of IgE (*FCER1G*) polymorphism -237A > G [[10](#B10)]. These results suggest that other genes or complex signal pathways might be related to the development of aspirin hypersensitivity in asthmatics.

Fibrous sheath interacting protein 1 gene (*FSIP1*), also known as *HSD10*, is a recently discovered gene that was first described in 2003 [[11](#B11)]. With its primary function on protein binding, the *FSIP1 *gene is expressed in airway epithelium ([GSE4498](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE4498) and GDS2486, GEO database) [[12](#B12)]. It has been reported that *FSIP1 *is regulated by amyloid beta (Aβ) precursor protein (APP) [[13](#B13)]. APP is an integral membrane protein expressed in many tissues, and particularly, in the synapses of neurons. There has been a report that APP is cleaved by α disintegrin and metalloproteinase 33 *(ADAM33)*, which is an asthma susceptibility gene [[14](#B14),[15](#B15)]. Based on these findings, we hypothesized *FSIP1 *gene could have an effect on the mechanism of aspirin on the various levels, including onset of AIA, and conducted association analyses of *FSIP1 *gene polymorphisms between AIA and ATA patients. We also conducted association analyses between SNPs of *FSIP1 *and fall rate of FEV_1 _by aspirin provocation.

## Methods

### Study Subjects

This study was conducted in compliance with the Global Initiative for Asthma (GINA) Global strategy for asthma management and prevention Study. A total of 592 subjects were recruited from the Asthma Genome Research Center comprising hospitals of Soonchunhyang Seoul and Bucheon Hospital, Chungnam, Chungbuk, and Seoul national university in Korea. All subjects provided informed consent, and the protocols were approved by the Institutional Review Board of each hospital. Each patient showed airway reversibility such as the inhalant bronchodilator-induced improvement with over 15% of forced expiratory volume in 1 s (FEV_1_) and/or the airway hyperresponsiveness to the provocative concentration of P20 methacholine [[16](#B16)]. Aspirin challenge was performed as previously described [[17](#B17)]. Subjects showing a fall rate of FEV_1 _less than 15% without skin manifestations were included in ATA group. The clinical profiles of subjects are listed in Table [1](#T1).

### Table 1.

| Clinical profile | Asthmatics (all subject) | AIA | ATA |
| --- | --- | --- | --- |
| N | 592 | 163 | 429 |
| Age of first medical examination (mean (range)) | 46.15 (15.40-77.88) | 43.13 (17.22-72.73) | 47.30 (15.40-77.88) |
| Height (cm) | 160.78 ± 8.63 | 161.72 ± 8.69 | 160.42 ± 8.39 |
| Weight (kg) | 62.81 ± 10.84 | 61.25 ± 10.38 | 63.40 ± 10.97 |
| Fall rate (%) | 9.27 ± 13.24 | 24.63 ± 16.11 | 3.54 ± 4.85 |
| Blood eosinophil (%) | 6.01 ± 5.73 | 5.96 ± 5.21 | 6.03 ± 5.92 |
| FVC %, predicted | 88.54 ± 14.08 | 90.35 ± 14.04 | 87.85 ± 14.05 |
| FEV1 %, predicted | 90.54 ± 16.97 | 87.58 ± 16.94 | 91.66 ± 16.87 |
| PC20, methacholine (mg/ml) | 6.43 ± 8.67 | 5.02 ± 7.83 | 6.91 ± 8.90 |
| Total IgE (IU/ml)* | 156 (62, 394) | 164 (78, 357) | 154 (53, 416) |
| Sex (male/female) | 206/386 | 59/104 | 147/282 |
| Current Smoker (%) | 27.70 | 21.47 | 30.07 |
| Positive rate of skin test (%) | 56.42 | 52.76 | 57.81 |

Table 1 Caption: Clinical profiles of AIA patients and ATA cases.

All patients had a history of dyspnea and wheezing during the previous 12 months, plus one of the following: 1) >15% increase in FEV1 or >12% increase plus 200 mL following inhalation of a short-acting bronchodilator, 2) <10 mg/mL PC20 methacholine, and 3) >20% increase in FEV1 following 2 weeks of treatment with inhaled steroids and long-acting bronchodilators. Twenty-four common inhalant allergens were used for a skin prick test [[9](#B9)]. Total IgE was measured by the CAP system (Pharmacia Diagnostics, Uppsala, Sweden). Atopy was defined as having a wheal reaction equal to or greater than histamine or 3 mm in diameter. The asthmatic patients had experienced no exacerbation of asthma and respiratory tract infection in the 6 weeks preceding oral aspirin challenge (OAC). OAC was performed with increasing doses of aspirin using methods slightly modified from those described previously [[9](#B9),[18](#B18)]. Changes in FEV_1 _were followed for 5 hours after the last aspirin challenge dose. Aspirin-induced bronchospasms, as reflected by rate (%) of FEV_1 _decline, were calculated as the pre-challenge FEV_1 _minus the post-challenge FEV_1 _divided by the pre-challenge FEV_1_. OAC reactions were categorized into 2 groups as follows: 15% or greater decreases in FEV_1 _with naso-ocular or cutaneous reactions (aspirin intolerant asthma: AIA), and less than 15% decreases in FEV_1 _without naso-ocular or cutaneous reactions (aspirin tolerant asthma: ATA).

### SNP selection and genotyping

We selected candidate SNPs showing polymorphic in the National Center for Biotechnology Information (build 36), and then genotyped in 163 AIA and 429 ATA subjects. Genotyping was performed at a multiplex level of using the Illumina Golden Gate genotyping system [[19](#B19)] and data quality was assessed by duplicate DNAs (n = 10). The genotype quality score for retaining data was set to 0.25. SNPs that could not satisfy the following criteria were excluded: (i) a minimum call rate of 90%; (ii) no duplicate error; (iii) Hardy-Weinberg equilibrium greater than *P *> 0.001. A total of 66 SNPs from *FSIP1 *were successfully genotyped.

### Statistics

We examined Lewontin's D' (|*D'*|) and the linkage disequilibrium (LD) coefficient *r*^2 ^between all pairs of biallelic loci. Linkage disequilibrium was inferred using the algorithm developed by the Broad Institute (using the program Haploview) [[20](#B20)]. Haplotypes were first estimated by using PHASE software [[21](#B21)], and then computed by logistic analyses using the Statistical Analysis System (SAS). Subjects harboring missing genotypes were omitted in the analysis of individual single-nucleotide polymorphisms and haplotypes. The genotyping success rate was >99%, so it is unlikely that omitting a small number of individuals introduced any bias in the analysis. The genotype and haplotype association with AIA were analyzed using logistic models with age (continuous value), gender (male = 0, female = 1), smoking status (non-smoker = 0, ex-smoker = 1, smoker = 2), atopy (absence = 0, presence = 1), and BMI as covariates. Significant associations are shown in bold face (*P *< 0.05). The common alleles were used as the referent genotype to the heterozygote and homozygote of the minor allele in referent analysis. The association analyses of differences in the fall rates in FEV_1 _following aspirin challenge with the genotypes and haplotypes were examined by regression analysis using SAS. The data were managed and analyzed using SAS. The effective numbers of independent marker loci in each genes were calculated to correct for multiple testing using the software SNPSpD [[22](#B22)], which is based on the spectral decomposition (SpD) of matrices of pair-wise LD between SNPs [[22](#B22)].

## Results

### Subject Characteristics

A total of 592 asthma patients were recruited for this study, and there were 163 AIA patients and 429 ATA cases (Table [1](#T1)). First of all, the fall rate by aspirin provocation in AIA patients showed significantly higher fall rate (24.63%) than ATA cases (3.54%) (*P *< 0.0001, Table [1](#T1)). Among the total subjects, 27.70% were current smokers, with more smokers within ATA patients (30.07%) than AIA patients (21.47%). Also, the positive skin test rate of AIA was smaller than that of ATA patients (52.76 and 57.81, respectively).

### SNP Analyses

Figure [1](#F1) shows the physical map of *FSIP1*, which contains 66 SNPs of the *FSIP1 *gene along with positions at exons and introns. Most of the variants were located in intronic regions (Additional File [1](#S1)). In case of SNPs in coding region, two nonsynonymous SNPs, *rs10152640 *and *rs16969386*, were located in exon11 region, and they also caused amino acid change, with the former causing cysteine to arginine amino acid change at position 402, while the latter caused glycine to alanine amino acid change at position 528.

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/bcf3/2896935/0a857c4ac0bc/1471-2466-10-34-1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=2896935_1471-2466-10-34-1.jpg)

A physical map of FSIP1 on chromosome 15q14. Exons, introns, UTR, and SNPs are shown. Coding exons are marked by shaded blocks; UTRs by white blocks. SNPs with complete LD are represented by r2 = 1.

In logistic analysis for associations between SNPs of *FSIP1 *and AIA in a Korean population with adjustment for age, sex, smoking status, atopy and BMI, there was 18 SNPs showing initial significant associations and one SNP showing significant association after correction (*rs7179742*, *P *= 0.0007, *P*^cor. ^= 0.03, OR = 1.63, 95% CI = 1.23-2.16). Detailed analyses of *rs7179742*, along with other SNPs in the same haplotype block (*FSIP1_BL1*), are shown in Table [2](#T2). Although *rs7179742 *was the only SNP with significant relation after correction for multiple testing, SNPs in *FSIP1_BL1 *had considerably significant *P*-values before correction for multiple testing. Furthermore, SNPs in *FSIP1_BL1 *including *rs7179742 *were observed to have an effect on the increased fall rate of FEV_1 _by aspirin provocation (Table [3](#T3)).

### Table 2.

| rs# | Position | MAF |  |  | Heterozygosity | HWE | OR(95%CI) | P | Pcor. |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
|   |   |   |  |  |   |   |   |   |   |
|   |   | AIA | ATA | Total |   |   |   |   |   |
| rs1602543 | Intron2 | 0.199 | 0.235 | 0.226 | 0.350 | 0.997 | 0.81 (0.59-1.11) | 0.19 | 1.00 |
| rs7179742 | Intron2 | 0.358 | 0.262 | 0.287 | 0.409 | 0.765 | 1.63 (1.23-2.16) | 0.0007 | 0.03 |
| rs1021411 | Intron3 | 0.292 | 0.227 | 0.242 | 0.367 | 0.885 | 1.47 (1.09-1.98) | 0.01 | 0.55 |
| rs1021410 | Intron3 | 0.352 | 0.262 | 0.284 | 0.407 | 0.751 | 1.60 (1.20-2.13) | 0.001 | 0.06 |
| rs12440567 | Intron3 | 0.325 | 0.240 | 0.261 | 0.386 | 0.788 | 1.58 (1.18-2.10) | 0.002 | 0.09 |
| rs2576932 | Intron3 | 0.443 | 0.508 | 0.490 | 0.500 | 0.786 | 0.75 (0.58-0.98) | 0.03 | 1.00 |
| rs10520151 | Intron5 | 0.027 | 0.023 | 0.024 | 0.047 | 0.274 | 1.17 (0.53-2.59) | 0.70 | 1.00 |
| rs16969802 | Intron5 | 0.322 | 0.239 | 0.260 | 0.384 | 0.705 | 1.56 (1.17-2.08) | 0.002 | 0.11 |
| rs12911842 | Intron5 | 0.211 | 0.260 | 0.242 | 0.367 | 0.467 | 0.73 (0.54-0.99) | 0.04 | 1.00 |
| rs7179417 | Intron5 | 0.446 | 0.509 | 0.492 | 0.500 | 0.815 | 0.76 (0.59-0.98) | 0.04 | 1.00 |
| rs4115216 | Intron5 | 0.446 | 0.510 | 0.493 | 0.500 | 0.877 | 0.76 (0.58-0.98) | 0.03 | 1.00 |
| rs8033957 | Intron5 | 0.527 | 0.468 | 0.485 | 0.500 | 0.511 | 1.28 (1.00-1.66) | 0.05 | 1.00 |
| rs6492915 | Intron5 | 0.527 | 0.468 | 0.485 | 0.500 | 0.511 | 1.28 (1.00-1.66) | 0.05 | 1.00 |
| rs8031241 | Intron5 | 0.452 | 0.513 | 0.497 | 0.500 | 0.905 | 0.76 (0.59-0.99) | 0.04 | 1.00 |
| rs1906164 | Intron6 | 0.232 | 0.249 | 0.247 | 0.372 | 0.852 | 0.91 (0.67-1.23) | 0.53 | 1.00 |

Table 2 Caption: Logistic and statistical analyses of FSIP1 SNPs in haplotype block 1.

### Table 3.

| rs# | C/C |  | C/R |  | R/R |  | P* |
| --- | --- | --- | --- | --- | --- | --- | --- |
|   |   |  |   |  |   |  |   |
|   | n | Fall rate | n | Fall rate | n | Fall rate |   |
| rs1602543 | 356 | 9.74 ± 13.12 | 205 | 8.25 ± 13.03 | 31 | 9.88 ± 15.55 | 0.43 |
| rs7179742 | 300 | 8.29 ± 12.05 | 241 | 9.49 ± 14.15 | 51 | 13.54 ± 14.55 | 0.01 |
| rs1021411 | 335 | 8.84 ± 12.89 | 223 | 9.58 ± 13.87 | 34 | 10.88 ± 12.14 | 0.26 |
| rs1021410 | 299 | 8.30 ± 12.03 | 245 | 9.56 ± 14.13 | 48 | 13.38 ± 14.86 | 0.01 |
| rs12440567 | 322 | 8.36 ± 12.33 | 227 | 9.81 ± 14.15 | 43 | 12.76 ± 14.11 | 0.02 |
| rs2576932 | 157 | 11.33 ± 15.8 | 290 | 8.34 ± 11.80 | 144 | 8.76 ± 12.69 | 0.06 |
| rs10520151 | 564 | 9.15 ± 13.02 | 27 | 0.71 ± 17.32 | 1 | 13.8 | 0.50 |
| rs16969802 | 324 | 8.38 ± 12.32 | 225 | 9.80 ± 14.18 | 43 | 12.68 ± 14.14 | 0.02 |
| rs12911842 | 341 | 9.57 ± 13.93 | 210 | 9.30 ± 12.54 | 41 | 6.09 ± 9.94 | 0.14 |
| rs7179417 | 156 | 11.40 ± 15.8 | 291 | 8.25 ± 11.78 | 145 | 8.87 ± 12.64 | 0.06 |
| rs4115216 | 155 | 11.46 ± 15.8 | 292 | 8.23 ± 11.77 | 145 | 8.87 ± 12.64 | 0.06 |
| rs8033957 | 162 | 9.08 ± 13.21 | 284 | 8.22 ± 11.59 | 145 | 11.41 ± 15.8 | 0.10 |
| rs6492915 | 162 | 9.08 ± 13.21 | 284 | 8.22 ± 11.59 | 145 | 11.41 ± 15.8 | 0.10 |
| rs8031241 | 153 | 11.52 ± 15.9 | 291 | 8.20 ± 11.76 | 147 | 8.91 ± 12.62 | 0.06 |
| rs1906164 | 339 | 9.25 ± 12.60 | 216 | 9.21 ± 13.94 | 37 | 9.24 ± 14.74 | 0.98 |

Table 3 Caption: Association analysis between SNPs of FSIP1 gene and the fall of FEV1 by aspirin provocation.

Logarithms for *P*-values of all SNPs were calculated for the better representation of the values (Figure [2](#F2)). Simultaneously, negative log of 0.05 and 0.0001, where latter value equals to 0.05 after correction for multiple testing (multiple testing correction number = 49.39), show the lines for significant association. In particular, SNPs with its value exceeding lower line (negative log of 0.05) would have initial association with AIA, and if it exceeds upper line (negative log of 0.0001), it would still have significant association after correction for multiple testing. As shown in Figure [2](#F2), there were 18 SNPs out of 66 that showed initial significant association with AIA. 12 SNPs out of the 18 showed association with the susceptibility to the risk of AIA, furthermore, rs7179742 had significantly higher MAF for AIA than that of ATA, showing increased susceptibility of AIA (MAF = 0.358 and 0.262, respectively). On the other hand, six SNPs out of the 18 showed lower minor allele frequencies in AIA patients compared to those of ATA cases.

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/bcf3/2896935/07cd65a7a986/1471-2466-10-34-2.jpg)

A graph of negative log of P-values of SNPs. After correction for multiple testing, all SNPs are calculated for the better representation of the values. The log = 0.0001 equals to P = 0.05. SNP rs7179742, which is the only SNP to have significant P-value after the multiple corrections, is highlighted in black.

### Haplotypes Analyses

Haplotypes were calculated using PHASE software, and there were 3 haplotype blocks identified (Figure [3](#F3)). Interestingly, pair-wise comparisons showed tight LD with three LD blocks among 66 SNPs of the *FSIP1 *gene. From each haplotype blocks, a total of 13 haplotypes with frequencies higher than 0.05 are listed (Figure [3A](#F3)). Among them, 4 haplotypes showed initial association with AIA, as detailed analyses of the haplotypes are listed in Table [4](#T4). Three haplotypes, *FSIP1_BL1_ht4 *(*P *= 0.03), *FSIP1_BL2_ht4 *(*P *= 0.02), *FSIP1_BL3_ht3 *(*P *= 0.04) and *FSIP1_BL3_ht4 *(*P *= 0.01), were frequent in AIA patients, whereas *FSIP1_BL3_ht3 *revealed a lower frequency in AIA compared to that of ATA cases, respectively. However, their signals disappeared after the correction for multiple testing.

### Figure 3.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/bcf3/2896935/a2b31a8e4604/1471-2466-10-34-3.jpg)

Haplotypes and LDs of FSIP1 gene. (A) Haplotypes of FSIP1 in the Korean population. They are divided into 3 haplotype blocks. Only those with frequencies over 0.05 are shown. (B) LD coefficients graphical plot (|D'|) among SNPs based on the genotypes of whole study subjects in this study (n = 592).

### Table 4.

| Haplotypes | MAF |  |  | Heterozygosity | HWE | OR (95%CI) | P | Pcor. |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
|   |   |  |  |   |   |   |   |   |
|   | AIA | ATA | Total |   |   |   |   |   |
| FSIP1-_ BL1_ht1 | 0.235 | 0.256 | 0.254 | 0.379 | 0.770 | 0.91 (0.67-1.23) | 0.52 | 1.00 |
| FSIP1-_BL1_ ht2 | 0.202 | 0.241 | 0.227 | 0.351 | 0.683 | 0.76 (0.55-1.03) | 0.08 | 1.00 |
| FSIP1-_BL1_ ht3 | 0.187 | 0.214 | 0.208 | 0.330 | 0.498 | 0.85 (0.62-1.16) | 0.30 | 1.00 |
| FSIP1-_BL1_ ht4 | 0.244 | 0.190 | 0.202 | 0.322 | 0.546 | 1.42 (1.04-1.94) | 0.03 | 1.00 |
| FSIP1-_BL2_ ht1 | 0.262 | 0.302 | 0.295 | 0.416 | 0.614 | 0.82 (0.62-1.09) | 0.18 | 1.00 |
| FSIP1-_BL2_ ht2 | 0.199 | 0.226 | 0.214 | 0.336 | 0.207 | 0.82 (0.60-1.11) | 0.20 | 1.00 |
| FSIP1-_BL2_ ht3 | 0.130 | 0.148 | 0.146 | 0.250 | 0.400 | 0.88 (0.61-1.27) | 0.49 | 1.00 |
| FSIP1-_BL2_ ht4 | 0.151 | 0.098 | 0.111 | 0.197 | 0.908 | 1.61 (1.10-2.37) | 0.02 | 0.76 |
| FSIP1-_BL2_ ht5 | 0.123 | 0.104 | 0.109 | 0.195 | 0.839 | 1.34 (0.89-2.01) | 0.17 | 1.00 |
| FSIP1-_BL3_ ht1 | 0.238 | 0.291 | 0.283 | 0.406 | 0.011 | 0.78 (0.59-1.04) | 0.09 | 1.00 |
| FSIP1-_BL3_ ht2 | 0.298 | 0.269 | 0.278 | 0.401 | 0.600 | 1.21 (0.91-1.60) | 0.19 | 1.00 |
| FSIP1-_BL3_ ht3 | 0.157 | 0.202 | 0.187 | 0.304 | 0.575 | 0.70 (0.50-0.98) | 0.04 | 1.00 |
| FSIP1-_BL3_ ht4 | 0.117 | 0.072 | 0.085 | 0.155 | 0.200 | 1.76 (1.13-2.74) | 0.01 | 0.66 |

Table 4 Caption: Logistic and statistical analyses of FSIP1 haplotypes.

## Discussion

Most of the SNPs were located in intronic region, but two SNPs, *rs10152640 *and *rs16969386*, were located in exon region. They also caused amino acid change, as *rs10152640 *caused cysteine to arginine amino acid change at position 402 and *rs16969386 *caused glycine to alanine amino acid change at position 528. Nonsynonymous amino acid changes have significant effect on the expression and function of the protein [[23](#B23),[24](#B24)]. In particular, the hydrophobicity change from cysteine to arginine (2.5 and -4.5 of hydrophobicity, respectively) on the *rs10152640 *could induce a critical conformational or functional change of the protein. However, when the association between two nonsynonymous SNPs of *FSIP1 *and the fall rate of FEV_1 _by aspirin provocation was analyzed, no significant association was found (data not shown), indicating that these nonsynonymous variants could not have a direct effect on the activities of FSIP1 protein.

From the association analyses of *FSIP1 *gene with AIA, we initially found 18 polymorphisms that showed association. In addition, significant associations of several SNPs in *FSIP1_BL1 *and *FSIP1_BL2 *with the fall rate of FEV_1 _by aspirin provocation were found (data for *FSIP1_BL2 *not shown). However, after multiple testing correction, only *rs7179742 *(*P *= 0.0007, *P*^cor. ^= 0.03, OR = 1.63, 95% CI = 1.23-2.16) showed increased susceptibility of AIA. The possible susceptibility of *rs7179742 *to AIA was reinforced by the significant association of this variant with the increased fall rate of FEV_1 _by aspirin provocation (*P *= 0.01 in co-dominant model, Table [3](#T3)). Although this SNP is located in the intronic region of the gene, intronic region can still affect overall transcription in different ways. It was reported that introns can affect transcription efficiency [[25](#B25)], and another study demonstrated that variations in intronic region can lead to splicing abnormalities, which in turn may lead to human diseases [[26](#B26)]. It was also reported recently that a SNP located in intronic region showed an association with asthma [[27](#B27)]. On the other hand, in additional analysis of LD near *FSIP1 *in Asian populations (Chinese and Japanese) from the International HapMap Project [http://hapmap.ncbi.nlm.nih.gov/](http://hapmap.ncbi.nlm.nih.gov/), the *FSIP1 *gene is in LD with the *thrombospondin-1 *(*THBS1 *or *TSP-1*) gene (Additional File [2](#S2)). The *THBS1 *gene has been implicated in the network underlying the pulmonary response to oxidative stress in asthma [[28](#B28)]. More interestingly, aspirin, as an inhibitor of THBS1, has recently shown to lead to reduction in THBS1 levels [[29](#B29)]. This suggests that *FSIP1 *might have an effect on aspirin hypersensitivity in asthma, with relation to the nearby potential gene of *THBS1*.

The *FSIP1 *gene has been discovered very recently. *FSIP1 *is regulated by APP, an integral member protein that is found in many tissues. APP, in turn, is cleaved by an asthma susceptible gene called *ADAM33 *[[30](#B30)]. *ADAM33 *polymorphisms have been associated with asthma risk in various populations, including Korea [[31](#B31)], Thai, Han Chinese and Japanese [[32](#B32)-[34](#B34)], who are closely related with Korean population, especially Han Chinese and Japanese. The *ADAM33 *polymorphisms have also been related with asthma risk of smoking population [[35](#B35),[36](#B36)]. These results suggest that the *FSIP1 *gene could have a correlation with APP and/or *ADAM33 *in development of asthma. In addition, the fact that aspirin affects wide range of human body, from lung to blood clotting to spermatogenesis, has to be considered. Naturally, the mechanism and pathways of aspirin is intricate and complicated.

One of limitation of our study is the small number of AIA patient, which might result in decreasing the statistical power of this study. However, considering the rareness of AIA condition, our results could provide supporting information, with further needs of replication in large number of subjects. Taken together, even though current knowledge of *FSIP1 *gene implicates that it only affects a small part out of aspirin pathways, our findings suggest a new relation between *FSIP1 *and aspirin hypersensitivity that explains the association found in this study.

## Conclusions

In summary, we found 18 SNPs and 4 haplotypes of *FSIP1 *showing associations with AIA initially, and after multiple testing correction, one SNP, *rs7179742 *showed significant association with AIA. Although the relation between the *FSIP1 *gene and AIA is not yet clearly understood, our findings suggest that *FSIP1*-related regulations of APP and *ADAM33 *may play a role for the development of aspirin hypersensitivity in asthmatics, along with the fact that FSIP1 is expressed in airway epithelium. Future researches should concentrate on this part to discover more about the role of *FSIP1 *gene and also the relation between the gene and various diseases including AIA.

## Abbreviations

*FSIP1*: *Fibrous sheath-interacting protein 1*; AIA: Aspirin-intolerant asthma; ATA: Aspirin-tolerant asthma; APP: Amyloid beta (Aβ) precursor protein; *ADAM33*: *α disintegrin and metalloproteinase 33*; SNP: Single nucleotide polymorphism; BMI: Body mass index; FEV_1_: Forced expiratory volume in 1 second; IgE: Immuglobulin E; OAC: Oral aspirin challenge; PAF: Platelet activating factor; ht: Haplotype; FVC: Forced Vital Capacity; *P*: *P *value, and *P*^cor.^: Corrected *P *value; LD: Linkage Disequilibrium; THBS1: Thrombospondin-1

## Competing interests

The authors declare that they have no competing interests.

## Authors' contributions

JYK and JHK developed tables/figures, and drafted the manuscript. TJP, JSL, and CFP participated in preparation and quality control of samples. JSB, BLP, and HSC performed the statistical analysis. JSP, SWP, STU, MKK, ISC, SHC, and BWC helped to recruit subjects. CSP and HDS managed all of this study and helped to draft the manuscript. All authors read and approved the final manuscript.

## Pre-publication history

The pre-publication history for this paper can be accessed here:

[http://www.biomedcentral.com/1471-2466/10/34/prepub](http://www.biomedcentral.com/1471-2466/10/34/prepub)

## Supplementary Material

## Contributor Information

Jason Yongha Kim, Email: imblue870227@gmail.com.

Jeong Hyun Kim, Email: soykm@sogang.ac.kr.

Tae Joon Park, Email: pires81@naver.com.

Joon Seol Bae, Email: jsbae@sogang.ac.kr.

Jin Sol Lee, Email: shadow2386@naver.com.

Charisse Flerida Pasaje, Email: charissepass@yahoo.com.

Byung Lae Park, Email: blpark@snp-genetics.com.

Hyun Sub Cheong, Email: chhs@snp-genetics.com.

Jong-Sook Park, Email: js1221@schbc.ac.kr.

Sung-Woo Park, Email: swpark@schbc.ac.kr.

Soo-Taek Uh, Email: uhs@hosp.sch.ac.kr.

Mi-Kyeong Kim, Email: kimmk@chungbuk.ac.kr.

Inseon S Choi, Email: ischoi@chonnam.ac.kr.

Sang Heon Cho, Email: shcho@plaza.snu.ac.kr.

Byoung Whui Choi, Email: bwchoimd@cau.ac.kr.

Choon-Sik Park, Email: mdcspark@unitel.co.kr.

Hyoung Doo Shin, Email: hdshin@sogang.ac.kr.

## Acknowledgements

This work was supported by a grant from the Korea Health 21 R&D Project (A010249); a grant number M1-0302-00-0073 from Korea Science and Engineering Foundation (KOSEF) funded by the Korea government (MEST) (No. 2009-0080157); an Intramural Research Grant of the Korea National Institute of Health (grant number 4800-4845-300-260-00); an Intramural Research Grant from Sogang University (grant number 200810021.01); and a Priority Research Centers Program through the National Research Foundation of Korea (NRF) funded by the Ministry of Education, Science and Technology (2009-0093822). The DNA samples were generously provided by the Soonchunhyang University, Bucheon Hospital Biobank, a member of the National Biobank of Korea, supported by the Ministry of Health, Welfare and Family Affairs, Republic of Korea.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Fanta CH. Asthma. N Engl J Med. 2009;360:1002–1014. doi: 10.1056/NEJMra0804579.  [DOI](https://doi.org/10.1056/NEJMra0804579) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19264689/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Asthma&author=CH%20Fanta&volume=360&publication_year=2009&pages=1002-1014&pmid=19264689&doi=10.1056/NEJMra0804579&)

2. Widal F, Abrami P, Lermoyez J. Anaphylaxie et idiosyncrasie. 1992 [Anaphylaxis and idiosyncrasy. 1992] Allergy Proc. 1993;14:373–376. doi: 10.2500/108854193778774065. discussion 371-372.  [DOI](https://doi.org/10.2500/108854193778774065) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8288122/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Allergy%20Proc&title=Anaphylaxie%20et%20idiosyncrasie.%201992%20%5BAnaphylaxis%20and%20idiosyncrasy.%201992%5D&author=F%20Widal&author=P%20Abrami&author=J%20Lermoyez&volume=14&publication_year=1993&pages=373-376&pmid=8288122&doi=10.2500/108854193778774065&)

3. Babu KS, Salvi SS. Aspirin and asthma. Chest. 2000;118:1470–1476. doi: 10.1378/chest.118.5.1470.  [DOI](https://doi.org/10.1378/chest.118.5.1470) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11083703/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chest&title=Aspirin%20and%20asthma&author=KS%20Babu&author=SS%20Salvi&volume=118&publication_year=2000&pages=1470-1476&pmid=11083703&doi=10.1378/chest.118.5.1470&)

4. Kim SH, Hur GY, Choi JH, Park HS. Pharmacogenetics of aspirin-intolerant asthma. Pharmacogenomics. 2008;9:85–91. doi: 10.2217/14622416.9.1.85.  [DOI](https://doi.org/10.2217/14622416.9.1.85) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18154450/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Pharmacogenetics%20of%20aspirin-intolerant%20asthma&author=SH%20Kim&author=GY%20Hur&author=JH%20Choi&author=HS%20Park&volume=9&publication_year=2008&pages=85-91&pmid=18154450&doi=10.2217/14622416.9.1.85&)

5. Kim SH, Kim YK, Park HW, Kim SH, Kim SH, Ye YM, Min KU, Park HS. Adenosine deaminase and adenosine receptor polymorphisms in aspirin-intolerant asthma. Respir Med. 2009;103:356–363. doi: 10.1016/j.rmed.2008.10.008.  [DOI](https://doi.org/10.1016/j.rmed.2008.10.008) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19019667/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Respir%20Med&title=Adenosine%20deaminase%20and%20adenosine%20receptor%20polymorphisms%20in%20aspirin-intolerant%20asthma&author=SH%20Kim&author=YK%20Kim&author=HW%20Park&author=SH%20Kim&author=SH%20Kim&volume=103&publication_year=2009&pages=356-363&pmid=19019667&doi=10.1016/j.rmed.2008.10.008&)

6. Kim SH, Ye YM, Hur GY, Lee SK, Sampson AP, Lee HY, Park HS. CysLTR1 promoter polymorphism and requirement for leukotriene receptor antagonist in aspirin-intolerant asthma patients. Pharmacogenomics. 2007;8:1143–1150. doi: 10.2217/14622416.8.9.1143.  [DOI](https://doi.org/10.2217/14622416.8.9.1143) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17924829/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=CysLTR1%20promoter%20polymorphism%20and%20requirement%20for%20leukotriene%20receptor%20antagonist%20in%20aspirin-intolerant%20asthma%20patients&author=SH%20Kim&author=YM%20Ye&author=GY%20Hur&author=SK%20Lee&author=AP%20Sampson&volume=8&publication_year=2007&pages=1143-1150&pmid=17924829&doi=10.2217/14622416.8.9.1143&)

7. Kim SH, Kim YK, Park HW, Jee YK, Kim SH, Bahn JW, Chang YS, Kim SH, Ye YM, Shin ES. Association between polymorphisms in prostanoid receptor genes and aspirin-intolerant asthma. Pharmacogenet Genomics. 2007;17:295–304. doi: 10.1097/01.fpc.0000239977.61841.fe.  [DOI](https://doi.org/10.1097/01.fpc.0000239977.61841.fe) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17496729/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Association%20between%20polymorphisms%20in%20prostanoid%20receptor%20genes%20and%20aspirin-intolerant%20asthma&author=SH%20Kim&author=YK%20Kim&author=HW%20Park&author=YK%20Jee&author=SH%20Kim&volume=17&publication_year=2007&pages=295-304&pmid=17496729&doi=10.1097/01.fpc.0000239977.61841.fe&)

8. Kim SH, Yang EM, Lee HN, Cho BY, Ye YM, Park HS. Combined effect of IL-10 and TGF-beta1 promoter polymorphisms as a risk factor for aspirin-intolerant asthma and rhinosinusitis. Allergy. 2009;64:1221–1225. doi: 10.1111/j.1398-9995.2009.01989.x.  [DOI](https://doi.org/10.1111/j.1398-9995.2009.01989.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19222424/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Allergy&title=Combined%20effect%20of%20IL-10%20and%20TGF-beta1%20promoter%20polymorphisms%20as%20a%20risk%20factor%20for%20aspirin-intolerant%20asthma%20and%20rhinosinusitis&author=SH%20Kim&author=EM%20Yang&author=HN%20Lee&author=BY%20Cho&author=YM%20Ye&volume=64&publication_year=2009&pages=1221-1225&pmid=19222424&doi=10.1111/j.1398-9995.2009.01989.x&)

9. Kim TH, Chang HS, Park SM, Nam BY, Park JS, Rhim T, Park HS, Kim MK, Choi IS, Cho SH. Association of angiotensin I-converting enzyme gene polymorphisms with aspirin intolerance in asthmatics. Clin Exp Allergy. 2008;38:1727–1737. doi: 10.1111/j.1365-2222.2008.02965.x.  [DOI](https://doi.org/10.1111/j.1365-2222.2008.02965.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18727619/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Exp%20Allergy&title=Association%20of%20angiotensin%20I-converting%20enzyme%20gene%20polymorphisms%20with%20aspirin%20intolerance%20in%20asthmatics&author=TH%20Kim&author=HS%20Chang&author=SM%20Park&author=BY%20Nam&author=JS%20Park&volume=38&publication_year=2008&pages=1727-1737&pmid=18727619&doi=10.1111/j.1365-2222.2008.02965.x&)

10. Palikhe NS, Kim SH, Cho BY, Ye YM, Hur GY, Park HS. Association of three sets of high-affinity IgE receptor (FcepsilonR1) polymorphisms with aspirin-intolerant asthma. Respir Med. 2008;102:1132–1139. doi: 10.1016/j.rmed.2008.03.017.  [DOI](https://doi.org/10.1016/j.rmed.2008.03.017) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18595682/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Respir%20Med&title=Association%20of%20three%20sets%20of%20high-affinity%20IgE%20receptor%20(FcepsilonR1)%20polymorphisms%20with%20aspirin-intolerant%20asthma&author=NS%20Palikhe&author=SH%20Kim&author=BY%20Cho&author=YM%20Ye&author=GY%20Hur&volume=102&publication_year=2008&pages=1132-1139&pmid=18595682&doi=10.1016/j.rmed.2008.03.017&)

11. Brown PR, Miki K, Harper DB, Eddy EM. A-kinase anchoring protein 4 binding proteins in the fibrous sheath of the sperm flagellum. Biol Reprod. 2003;68:2241–2248. doi: 10.1095/biolreprod.102.013466.  [DOI](https://doi.org/10.1095/biolreprod.102.013466) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12606363/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biol%20Reprod&title=A-kinase%20anchoring%20protein%204%20binding%20proteins%20in%20the%20fibrous%20sheath%20of%20the%20sperm%20flagellum&author=PR%20Brown&author=K%20Miki&author=DB%20Harper&author=EM%20Eddy&volume=68&publication_year=2003&pages=2241-2248&pmid=12606363&doi=10.1095/biolreprod.102.013466&)

12. Harvey BG, Heguy A, Leopold PL, Carolan BJ, Ferris B, Crystal RG. Modification of gene expression of the small airway epithelium in response to cigarette smoking. J Mol Med. 2007;85:39–53. doi: 10.1007/s00109-006-0103-z.  [DOI](https://doi.org/10.1007/s00109-006-0103-z) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17115125/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Mol%20Med&title=Modification%20of%20gene%20expression%20of%20the%20small%20airway%20epithelium%20in%20response%20to%20cigarette%20smoking&author=BG%20Harvey&author=A%20Heguy&author=PL%20Leopold&author=BJ%20Carolan&author=B%20Ferris&volume=85&publication_year=2007&pages=39-53&pmid=17115125&doi=10.1007/s00109-006-0103-z&)

13. Wen GY, Yang SY, Kaczmarski W, He XY, Pappas KS. Presence of hydroxysteroid dehydrogenase type 10 in amyloid plaques (APs) of Hsiao's APP-Sw transgenic mouse brains, but absence in APs of Alzheimer's disease brains. Brain Res. 2002;954:115–122. doi: 10.1016/S0006-8993(02)03354-1.  [DOI](https://doi.org/10.1016/S0006-8993(02)03354-1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12393239/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Brain%20Res&title=Presence%20of%20hydroxysteroid%20dehydrogenase%20type%2010%20in%20amyloid%20plaques%20(APs)%20of%20Hsiao's%20APP-Sw%20transgenic%20mouse%20brains,%20but%20absence%20in%20APs%20of%20Alzheimer's%20disease%20brains&author=GY%20Wen&author=SY%20Yang&author=W%20Kaczmarski&author=XY%20He&author=KS%20Pappas&volume=954&publication_year=2002&pages=115-122&pmid=12393239&doi=10.1016/S0006-8993(02)03354-1&)

14. Sakagami T, Jinnai N, Nakajima T, Sekigawa T, Hasegawa T, Suzuki E, Inoue I, Gejyo F. ADAM33 polymorphisms are associated with aspirin-intolerant asthma in the Japanese population. J Hum Genet. 2007;52:66–72. doi: 10.1007/s10038-006-0081-6.  [DOI](https://doi.org/10.1007/s10038-006-0081-6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17061022/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Hum%20Genet&title=ADAM33%20polymorphisms%20are%20associated%20with%20aspirin-intolerant%20asthma%20in%20the%20Japanese%20population&author=T%20Sakagami&author=N%20Jinnai&author=T%20Nakajima&author=T%20Sekigawa&author=T%20Hasegawa&volume=52&publication_year=2007&pages=66-72&pmid=17061022&doi=10.1007/s10038-006-0081-6&)

15. Zou J, Zhu F, Liu J, Wang W, Zhang R, Garlisi CG, Liu YH, Wang S, Shah H, Wan Y. Catalytic activity of human ADAM33. J Biol Chem. 2004;279:9818–9830. doi: 10.1074/jbc.M309696200.  [DOI](https://doi.org/10.1074/jbc.M309696200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14676211/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Catalytic%20activity%20of%20human%20ADAM33&author=J%20Zou&author=F%20Zhu&author=J%20Liu&author=W%20Wang&author=R%20Zhang&volume=279&publication_year=2004&pages=9818-9830&pmid=14676211&doi=10.1074/jbc.M309696200&)

16. Crapo RO, Casaburi R, Coates AL, Enright PL, Hankinson JL, Irvin CG, MacIntyre NR, McKay RT, Wanger JS, Anderson SD. Guidelines for methacholine and exercise challenge testing-1999. This official statement of the American Thoracic Society was adopted by the ATS Board of Directors, July 1999. Am J Respir Crit Care Med. 2000;161:309–329. doi: 10.1164/ajrccm.161.1.ats11-99.  [DOI](https://doi.org/10.1164/ajrccm.161.1.ats11-99) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10619836/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Respir%20Crit%20Care%20Med&title=Guidelines%20for%20methacholine%20and%20exercise%20challenge%20testing-1999.%20This%20official%20statement%20of%20the%20American%20Thoracic%20Society%20was%20adopted%20by%20the%20ATS%20Board%20of%20Directors,%20July%201999&author=RO%20Crapo&author=R%20Casaburi&author=AL%20Coates&author=PL%20Enright&author=JL%20Hankinson&volume=161&publication_year=2000&pages=309-329&pmid=10619836&doi=10.1164/ajrccm.161.1.ats11-99&)

17. Lee SH, Rhim T, Choi YS, Min JW, Kim SH, Cho SY, Paik YK, Park CS. Complement C3a and C4a increased in plasma of patients with aspirin-induced asthma. Am J Respir Crit Care Med. 2006;173:370–378. doi: 10.1164/rccm.200505-740OC.  [DOI](https://doi.org/10.1164/rccm.200505-740OC) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16293803/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Respir%20Crit%20Care%20Med&title=Complement%20C3a%20and%20C4a%20increased%20in%20plasma%20of%20patients%20with%20aspirin-induced%20asthma&author=SH%20Lee&author=T%20Rhim&author=YS%20Choi&author=JW%20Min&author=SH%20Kim&volume=173&publication_year=2006&pages=370-378&pmid=16293803&doi=10.1164/rccm.200505-740OC&)

18. Cormican LJ, Farooque S, Altmann DR, Lee TH. Improvements in an oral aspirin challenge protocol for the diagnosis of aspirin hypersensitivity. Clin Exp Allergy. 2005;35:717–722. doi: 10.1111/j.1365-2222.2005.02261.x.  [DOI](https://doi.org/10.1111/j.1365-2222.2005.02261.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15969660/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Exp%20Allergy&title=Improvements%20in%20an%20oral%20aspirin%20challenge%20protocol%20for%20the%20diagnosis%20of%20aspirin%20hypersensitivity&author=LJ%20Cormican&author=S%20Farooque&author=DR%20Altmann&author=TH%20Lee&volume=35&publication_year=2005&pages=717-722&pmid=15969660&doi=10.1111/j.1365-2222.2005.02261.x&)

19. Oliphant A, Barker DL, Stuelpnagel JR, Chee MS. BeadArray technology: enabling an accurate, cost-effective approach to high-throughput genotyping. Biotechniques. 2002. pp. 56–58. 60-51.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/12083399/)

20. Barrett JC, Fry B, Maller J, Daly MJ. Haploview: analysis and visualization of LD and haplotype maps. Bioinformatics. 2005;21:263–265. doi: 10.1093/bioinformatics/bth457.  [DOI](https://doi.org/10.1093/bioinformatics/bth457) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15297300/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=Haploview:%20analysis%20and%20visualization%20of%20LD%20and%20haplotype%20maps&author=JC%20Barrett&author=B%20Fry&author=J%20Maller&author=MJ%20Daly&volume=21&publication_year=2005&pages=263-265&pmid=15297300&doi=10.1093/bioinformatics/bth457&)

21. Stephens M, Smith NJ, Donnelly P. A new statistical method for haplotype reconstruction from population data. Am J Hum Genet. 2001;68:978–989. doi: 10.1086/319501.  [DOI](https://doi.org/10.1086/319501) | [PMC free article](/articles/PMC1275651/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11254454/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=A%20new%20statistical%20method%20for%20haplotype%20reconstruction%20from%20population%20data&author=M%20Stephens&author=NJ%20Smith&author=P%20Donnelly&volume=68&publication_year=2001&pages=978-989&pmid=11254454&doi=10.1086/319501&)

22. Single Nucleotide Polymorphism Spectral Decomposition (SNPSpD) http://genepi.qimr.edu.au/general/daleN/SNPSpD/  [http://genepi.qimr.edu.au/general/daleN/SNPSpD/](http://genepi.qimr.edu.au/general/daleN/SNPSpD/)

23. Shastry BS. SNPs: impact on gene function and phenotype. Methods Mol Biol. 2009;578:3–22. doi: 10.1007/978-1-60327-411-1_1. full_text.  [DOI](https://doi.org/10.1007/978-1-60327-411-1_1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19768584/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Methods%20Mol%20Biol&title=SNPs:%20impact%20on%20gene%20function%20and%20phenotype&author=BS%20Shastry&volume=578&publication_year=2009&pages=3-22&pmid=19768584&doi=10.1007/978-1-60327-411-1_1&)

24. Barenboim M, Jamison DC, Vaisman II. Statistical geometry approach to the study of functional effects of human nonsynonymous SNPs. Hum Mutat. 2005;26:471–476. doi: 10.1002/humu.20238.  [DOI](https://doi.org/10.1002/humu.20238) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16200641/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mutat&title=Statistical%20geometry%20approach%20to%20the%20study%20of%20functional%20effects%20of%20human%20nonsynonymous%20SNPs&author=M%20Barenboim&author=DC%20Jamison&author=II%20Vaisman&volume=26&publication_year=2005&pages=471-476&pmid=16200641&doi=10.1002/humu.20238&)

25. Brinster RL, Allen JM, Behringer RR, Gelinas RE, Palmiter RD. Introns increase transcriptional efficiency in transgenic mice. Proc Natl Acad Sci USA. 1988;85:836–840. doi: 10.1073/pnas.85.3.836.  [DOI](https://doi.org/10.1073/pnas.85.3.836) | [PMC free article](/articles/PMC279650/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/3422466/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20USA&title=Introns%20increase%20transcriptional%20efficiency%20in%20transgenic%20mice&author=RL%20Brinster&author=JM%20Allen&author=RR%20Behringer&author=RE%20Gelinas&author=RD%20Palmiter&volume=85&publication_year=1988&pages=836-840&pmid=3422466&doi=10.1073/pnas.85.3.836&)

26. Pagani F, Baralle FE. Genomic variants in exons and introns: identifying the splicing spoilers. Nat Rev Genet. 2004;5:389–396. doi: 10.1038/nrg1327.  [DOI](https://doi.org/10.1038/nrg1327) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15168696/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Genet&title=Genomic%20variants%20in%20exons%20and%20introns:%20identifying%20the%20splicing%20spoilers&author=F%20Pagani&author=FE%20Baralle&volume=5&publication_year=2004&pages=389-396&pmid=15168696&doi=10.1038/nrg1327&)

27. Jung JS, Park BL, Cheong HS, Bae JS, Kim JH, Chang HS, Rhim T, Park JS, Jang AS, Lee YM. Association of IL-17RB gene polymorphism with asthma. Chest. 2009;135:1173–1180. doi: 10.1378/chest.08-1595.  [DOI](https://doi.org/10.1378/chest.08-1595) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19118269/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chest&title=Association%20of%20IL-17RB%20gene%20polymorphism%20with%20asthma&author=JS%20Jung&author=BL%20Park&author=HS%20Cheong&author=JS%20Bae&author=JH%20Kim&volume=135&publication_year=2009&pages=1173-1180&pmid=19118269&doi=10.1378/chest.08-1595&)

28. Freishtat RJ, Benton AS, Watson AM, Wang Z, Rose MC, Hoffman EP. Delineation of a gene network underlying the pulmonary response to oxidative stress in asthma. J Investig Med. 2009;57:756–764. doi: 10.231/JIM.0b013e3181b91a83.  [DOI](https://doi.org/10.231/JIM.0b013e3181b91a83) | [PMC free article](/articles/PMC3328512/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19730131/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Investig%20Med&title=Delineation%20of%20a%20gene%20network%20underlying%20the%20pulmonary%20response%20to%20oxidative%20stress%20in%20asthma&author=RJ%20Freishtat&author=AS%20Benton&author=AM%20Watson&author=Z%20Wang&author=MC%20Rose&volume=57&publication_year=2009&pages=756-764&pmid=19730131&doi=10.231/JIM.0b013e3181b91a83&)

29. Fouladkou F, Lu C, Jiang C, Zhou L, She Y, Walls JR, Kawabe H, Brose N, Henkelman RM, Huang A. The ubiquitin ligase Nedd4-1 is required for heart development and is a suppressor of thrombospondin-1. J Biol Chem. pp. 6770–6780.  [DOI](https://doi.org/10.1074/jbc.M109.082347) | [PMC free article](/articles/PMC2825471/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20026598/)

30. Weiss ST, Raby BA, Rogers A. Asthma genetics and genomics 2009. Curr Opin Genet Dev. 2009;19:279–282. doi: 10.1016/j.gde.2009.05.001.  [DOI](https://doi.org/10.1016/j.gde.2009.05.001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19481925/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Opin%20Genet%20Dev&title=Asthma%20genetics%20and%20genomics%202009&author=ST%20Weiss&author=BA%20Raby&author=A%20Rogers&volume=19&publication_year=2009&pages=279-282&pmid=19481925&doi=10.1016/j.gde.2009.05.001&)

31. Lee JH, Park HS, Park SW, Jang AS, Uh ST, Rhim T, Park CS, Hong SJ, Holgate ST, Holloway JW. ADAM33 polymorphism: association with bronchial hyper-responsiveness in Korean asthmatics. Clin Exp Allergy. 2004;34:860–865. doi: 10.1111/j.1365-2222.2004.01977.x.  [DOI](https://doi.org/10.1111/j.1365-2222.2004.01977.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15196271/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Exp%20Allergy&title=ADAM33%20polymorphism:%20association%20with%20bronchial%20hyper-responsiveness%20in%20Korean%20asthmatics&author=JH%20Lee&author=HS%20Park&author=SW%20Park&author=AS%20Jang&author=ST%20Uh&volume=34&publication_year=2004&pages=860-865&pmid=15196271&doi=10.1111/j.1365-2222.2004.01977.x&)

32. Thongngarm T, Jameekornrak A, Limwongse C, Sangasapaviliya A, Jirapongsananuruk O, Assawamakin A, Chaiyaratana N, Luangwedchakarn V, Thongnoppakhun W. Association between ADAM33 polymorphisms and asthma in a Thai population. Asian Pac J Allergy Immunol. 2008;26:205–211.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/19317339/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Asian%20Pac%20J%20Allergy%20Immunol&title=Association%20between%20ADAM33%20polymorphisms%20and%20asthma%20in%20a%20Thai%20population&author=T%20Thongngarm&author=A%20Jameekornrak&author=C%20Limwongse&author=A%20Sangasapaviliya&author=O%20Jirapongsananuruk&volume=26&publication_year=2008&pages=205-211&pmid=19317339&)

33. Noguchi E, Ohtsuki Y, Tokunaga K, Yamaoka-Sageshima M, Ichikawa K, Aoki T, Shibasaki M, Arinami T. ADAM33 polymorphisms are associated with asthma susceptibility in a Japanese population. Clin Exp Allergy. 2006;36:602–608. doi: 10.1111/j.1365-2222.2006.02471.x.  [DOI](https://doi.org/10.1111/j.1365-2222.2006.02471.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16650044/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Exp%20Allergy&title=ADAM33%20polymorphisms%20are%20associated%20with%20asthma%20susceptibility%20in%20a%20Japanese%20population&author=E%20Noguchi&author=Y%20Ohtsuki&author=K%20Tokunaga&author=M%20Yamaoka-Sageshima&author=K%20Ichikawa&volume=36&publication_year=2006&pages=602-608&pmid=16650044&doi=10.1111/j.1365-2222.2006.02471.x&)

34. Su D, Zhang X, Sui H, Lu F, Jin L, Zhang J. Association of ADAM33 gene polymorphisms with adult allergic asthma and rhinitis in a Chinese Han population. BMC Med Genet. 2008;9:82. doi: 10.1186/1471-2350-9-82.  [DOI](https://doi.org/10.1186/1471-2350-9-82) | [PMC free article](/articles/PMC2553063/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18778489/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Med%20Genet&title=Association%20of%20ADAM33%20gene%20polymorphisms%20with%20adult%20allergic%20asthma%20and%20rhinitis%20in%20a%20Chinese%20Han%20population&author=D%20Su&author=X%20Zhang&author=H%20Sui&author=F%20Lu&author=L%20Jin&volume=9&publication_year=2008&pages=82&pmid=18778489&doi=10.1186/1471-2350-9-82&)

35. Sadeghnejad A, Ohar JA, Zheng SL, Sterling DA, Hawkins GA, Meyers DA, Bleecker ER. Adam33 polymorphisms are associated with COPD and lung function in long-term tobacco smokers. Respir Res. 2009;10:21. doi: 10.1186/1465-9921-10-21.  [DOI](https://doi.org/10.1186/1465-9921-10-21) | [PMC free article](/articles/PMC2664793/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19284602/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Respir%20Res&title=Adam33%20polymorphisms%20are%20associated%20with%20COPD%20and%20lung%20function%20in%20long-term%20tobacco%20smokers&author=A%20Sadeghnejad&author=JA%20Ohar&author=SL%20Zheng&author=DA%20Sterling&author=GA%20Hawkins&volume=10&publication_year=2009&pages=21&pmid=19284602&doi=10.1186/1465-9921-10-21&)

36. Reijmerink NE, Kerkhof M, Koppelman GH, Gerritsen J, de Jongste JC, Smit HA, Brunekreef B, Postma DS. Smoke exposure interacts with ADAM33 polymorphisms in the development of lung function and hyperresponsiveness. Allergy. 2009;64:898–904. doi: 10.1111/j.1398-9995.2009.01939.x.  [DOI](https://doi.org/10.1111/j.1398-9995.2009.01939.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19236319/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Allergy&title=Smoke%20exposure%20interacts%20with%20ADAM33%20polymorphisms%20in%20the%20development%20of%20lung%20function%20and%20hyperresponsiveness&author=NE%20Reijmerink&author=M%20Kerkhof&author=GH%20Koppelman&author=J%20Gerritsen&author=JC%20de%20Jongste&volume=64&publication_year=2009&pages=898-904&pmid=19236319&doi=10.1111/j.1398-9995.2009.01939.x&)
