# Genome-wide association scan identifies candidate polymorphisms associated with differential response to anti-TNF treatment in Rheumatoid Arthritis

## Metadata
**Authors:** Chunyu Liu, Franak Batliwalla, Wentian Li, Annette Lee, Ronenn Roubenoff, Evan Beckman, Houman Khalili, Aarti Damle, Marlena Kern, Robert M Plenge, Marieke Coenen, Timothy W Behrens, Richard Furie, John P Carulli, Peter K Gregersen
**Journal:** Molecular Medicine
**Date:** 2008 Jul 10
**DOI:** [10.2119/2008-00056.Liu](https://doi.org/10.2119/2008-00056.Liu)
**PMID:** 18615156
**PMCID:** PMC2276142
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2276142/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC2276142/pdf/mol-14-9-0561.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC2276142/pdf/mol-14-9-0561.pdf)

## Abstract

The prediction of response (or non-response) to anti-TNF treatment for Rheumatoid Arthritis is a pressing clinical problem. We conducted a genome wide association study using the Illumina HapMap300 SNP chip on 89 RA patients prospectively followed after beginning anti-TNF therapy as part of Autoimmune Biomarkers Collaborative Network (ABCoN) patient cohort. Response to therapy was determined by the change in Disease Activity Score (DAS28) observed after 14 weeks. We used a two part analysis that treated the change in DAS28 as a continuous trait and then incorporated into a dichotomous trait of “good responder” and “nonresponder” by EULAR criteria. We corrected for multiple tests by permutation, and adjusted for potential population stratification using Eigenstrat. Multiple SNP markers showed significant associations near or within loci including the v-maf musculoaponeurotic fibrosarcoma oncogene homolog B (MAFB) gene on chromosome 20, the type I interferon gene IFNk on chromosome 9, and in a locus on chromosome 7 that includes the paraoxonase I (PON1) gene A SNP in the IL10 promoter (rs1800896) that was previously reported as associated with anti-TNF response was weakly associated with response in this cohort. Replications of these results in independent and larger data sets are clearly required. We provide a reference list of candidate SNPs (p<0.01) that can be investigated in future pharmacogenomic studies.

Keywords: Tumor Necrosis Factor, Rheumatoid arthritis, Genome Wide Association, Pharmacogenetics

## Introduction

Tumor necrosis factor – alpha (TNF-α) is a key regulator of the inflammatory cascade in rheumatoid arthritis (RA) and several other inflammatory diseases([1](#b1-08_56_liu)–[3](#b3-08_56_liu)). To date, three TNF antagonists, Infliximab (Remicade), Etanercept (Enbrel) and Adalimumab (Humira), have been approved by the FDA to treat rheumatoid arthritis and other inflammatory diseases. The molecular mechanisms for these three TNF inhibitors are similar: they block the binding of TNF-α to its cell-surface receptors and limit subsequent cell signaling pathways that are induced or regulated by TNF-α. Etanercept is a dimeric TNF receptor-IgG fusion protein and mimics the inhibition effects of soluble TNF receptor by binding to TNF-α. Infliximab is a chimeric mouse-human antibody, while Adalimumab is a fully humanized antibody.

Although the therapeutic utility of TNF-α antagonism is well established, patients display substantial heterogeneity in their response to anti-TNF therapies, and the efficacy of any anti-TNF agent in a given patient is unpredictable. Approximately one third of patients have minimal or no response to these agents ([4](#b4-08_56_liu)). A genetic influence has been suggested based on candidate gene studies([5](#b5-08_56_liu), [6](#b6-08_56_liu)), but no comprehensive analysis of this issue has been reported. In a recent review, Coenen et al ([7](#b7-08_56_liu)) summarized 17 pharmacogenetic studies of anti-TNF treatment that were conducted after 2001. All of the 17 studies focused on polymorphisms in genes known to be involved in RA pathogenesis, genes encoding TNF-α receptors, or genes implicated in TNF-α metabolism. Several groups reported that a single nucleotide polymorphism (SNP), -308G>A, in the promoter of TNFA is significantly associated with the outcome to anti-TNF treatment ([5](#b5-08_56_liu), [8](#b8-08_56_liu)–[14](#b14-08_56_liu)). This positive association is also supported by a meta-analysis that was performed using 311 patients combined from several studies ([13](#b13-08_56_liu)).

In order to identify biomarkers influencing response to anti-TNF therapy, the Autoimmune Biomarkers Collaborative Network (ABCoN) has prospectively enrolled cohort of Rheumatoid Arthritis patients beginning anti-TNF treatment. Using these patient samples, we took an unbiased genome wide approach to finding common genetic variations that could be responsible for individual differences in response to the three anti-TNF agents. We report results for SNPs with lowest p-values from the GWA study, and provide details for selected candidate genes.

## Methods

### Patients

The Autoimmune Biomarkers Collaborative Network (ABCoN) was established in order to explore the use of new technologies for biomarker discovery in both RA and Systemic Lupus (SLE). The ABCoN Rheumatoid Arthritis cohort includes 116 active RA patients followed prospectively to evaluate efficacy of the three available anti-TNF agents. In order to examine the response to anti-TNF therapy in RA, blood samples, laboratory and clinical data were collected at baseline (prior to anti-TNF therapy), 6 weeks, 3 months, 6 months and 1-year post treatment. DNA, RNA, peripheral blood cells, plasma, serum and urine were obtained at the time of each study visit. Enrollment criteria included having a minimum of six swollen joints at enrollment, and no previous exposure to anti-TNF agents during the six months prior to enrollment in the study. We did not enroll patients taking more than 10 mg of oral steroid therapy per day at the time of enrollment. All the patients provided written informed consent. The study protocols were approved by local ethics committees.

### Efficacy measurements

Disease severity was evaluated using the DAS28 score which is the Disease Activity Score that includes 28-joint counts and C-reactive protein([15](#b15-08_56_liu)). DAS28 was measured at three time points: baseline, 6 weeks, and 14 weeks. Two scales were considered to evaluate efficacy of anti-TNF treatment. First, a relative improvement in disease activity was calculated for each patient using the DAS28 scores at baseline and at week 14:

| relDAS28=(DAS28visit1-DAS28visit3DAS28visit1)%. |
| --- |
*RelDAS*28 has a continuous scale and is approximately normal. Second, according to the EULAR definition published elsewhere ([16](#b16-08_56_liu)), patients are classified as good, moderate or non-responders, using the individual amount of change in the DAS28 (ΔDAS28) and DAS28 values at 14 weeks ([16](#b16-08_56_liu)). Briefly, a good responder is classified if ΔDAS28 1.2 and DAS28 at 14 weeks 3.2; a moderate responders are patients with (ΔDAS28 1.2 and DAS28 at 14 weeks > 3.2) or (0.6 <ΔDAS28 1.2 and DAS28 at 14 weeks 5.1). Patients are classified as non-responders if they do not fall into any of these categories ([16](#b16-08_56_liu)).

### Genotyping and quality control

The patients’ genomic DNA was extracted from peripheral blood using standard protocols. We genotyped 317,000 Single Nucleotide Polymorphisms (SNPs) on 102 anti-TNF treated patients using an Illumina Beadstation and Illumina HAP300 chips according to the Illumina Infinium 2 assay manual (Illumina), as previously described ([17](#b17-08_56_liu)). The HAP300 chip includes, on average, a SNP every 10 kilobases across all the autosomes, and interrogates approximately 87% of the common genetic variation in populations of European descent ([18](#b18-08_56_liu)).

To ensure SNP marker quality and reduce the possibility of false associations, quality control procedures were performed on each of the 317k SNPs before association tests were carried out. The SNP set was filtered on the basis of genotype call rates ( 95%), minor allele frequency (MAF 0.05). Hardy–Weinberg equilibrium (HWE) was calculated for individual SNPs using the Exact test. All of the SNPs reported in this manuscript have HWE p-values > 0.001. After filtering, 283,348 polymorphic SNPs were analyzed on chromosome 1 through chromosome 22. The average call rate for the filtered SNPs was 99.5%. We removed patients if their percentage of missing genotypes was more than 5% or if there was evidence of possible contamination in their DNA sample.

### Statistical analysis

We evaluated associations between SNP markers and response to the anti-TNF therapies in two stages. In the first stage of our analysis, we used *relDAS*28 as a continuous dependent variable to evaluate the associations. Because the sample size is relatively small in our study, a continuous scale of the dependent variable has more power than a categorized variable. Linear regression was carried out to evaluate association between individual SNP markers and response to anti-TNF therapies (i.e., *relDAS*28) in the context of additive genetic effect model. A t- statistic was derived from regression and used to evaluate association between individual SNP markers and response to anti-TNF therapies (i.e., *relDAS*28). The t-statistic is robust, and thus some departure from normality for the dependent variable is acceptable. Because more than 283,348 tests were involved in this study, a permutation test was carried out to account for multiple testing on each chromosome. The permutation test can also address the slight deviation from normality in the dependent variable. To obtain adjusted p-values, the phenotypic values were randomly shuffled to break the relationship between phenotype and genotype. The entire analysis was repeated on the shuffled data; therefore, the shuffled data is representative of the null hypothesis. The 1,000 smallest p-values were obtained from each of the N = 1,000 permutation iterations for the whole set of SNPs on individual chromosomes.

In the second stage analysis, we use a categorized dependent variable – response status to the anti-TNF therapy (i.e., non-responders vs. good responders) to evaluate association with SNPs selected from the first stage. The probability of being a non-responder was modeled using logistic regression with additive genetic effect model. Unless specified, all calculations for statistical analysis were carried out using the R software package (Version 2.2.1).

Population admixture (i.e., sampling of subjects from two or more subpopulations) has been recognized as a major cause for inconsistent results and spurious associations for genetic studies([19](#b19-08_56_liu), [20](#b20-08_56_liu)). U.S. populations are genetically admixed. Although self-reported ethnicity data was recorded in this study, it is incomplete and may be inaccurate. To accurately classify individuals according to ancestry and to remove any possible related individuals, we calculated pair-wise identity-by-state (IBS) distance for the 102 subjects and performed subsequent complete linkage agglomerative clustering and multidimensional scaling using genome-wide SNP markers in Plink software (version 1.00, [http://pngu.mgh.harvard.edu/~purcell/plink/index.shtml](http://pngu.mgh.harvard.edu/~purcell/plink/index.shtml)). Clustering data were plotted to identify major population subdivisions. In addition to removing outliers from the dataset, we further evaluated potential effect from subpopulations (e.g. northern and southern Europeans) by the EIGENSTRAT program with genome wide SNP data ([21](#b21-08_56_liu)). The top ten principal components (PCs) were obtained. Correlation analysis between the top PCs and the phenotypes (*delDAS*28 and dichotomous response status to anti-TNF) was performed to detect if the phenotypic difference among individuals were due to population stratification. If the spread of samples in these principal components was purely due to population stratification, it can be removed by forcing all samples to have zero value in these principal components. Then a "virtual" genotype can be obtained by rotating the corrected principal components back to the original genotype space. Pearson's chi-square test was performed for association between selected SNPs and response status to the anti-TNF therapy (i.e., non-responders vs. good responders)

## Results

Among the 102 active RA patients with complete genotypic and phenotypic data, self-reported ancestry included 83 Europeans, four Asians, three American Africans, as well as three subjects with reported Latino ethnicity. Nine patients have missing information for their ethnicity. Linkage agglomerative clustering and multidimensional scaling identified 89 patients with primarily European ancestry (Supplementary [Fig. 1](#SD1)), among them 83 was self-reported to be white and 6 had missing ethnicity data. These 89 patients were used for subsequent association analysis.

The baseline characteristics of the 89 patients are summarized in [Table 1](#t1-08_56_liu). At the time of diagnosis, they were 47±14 (mean ± S.D) yr old, their disease duration was 8±9 years, 75% were women, and 15% were current smokers, and the average serum CRP level (mg/dl) at baseline was 1.7 (S.D. = 2.0). On average, the DAS28 at the baseline (before anti-TNF therapy) was 5.22 (S.D. = 0.80), indicating that the RA disease activity was high for most of the patients([16](#b16-08_56_liu)). 54 subjects were treated with Etanercept, 37 with Infliximab, and 25 with Adalimumab. These patients had a mean disease duration of 8 years; 46% had been previously treated with other DMARDs, and seven patients had been previously treated with TNF inhibitors.

### Table 1.

| Patient Characteristics: European descent (n=89) |  |
| --- | --- |
| Age (yr) | 57+13.5 |
| Age at diagnosis (yr) | 47 ± 15 |
| Women: (%) | 75 |
| Disease duration (yr) | 8 ± 8 |
| Current Smokers (%) | 15 |
| Pain VAS | 50.0 ± 22.3 |
| Health VAS | 46.4 ± 17.5 |
| Tender (no.) | 11.8 ± 6.2 |
| Swollen (no.) | 11.2 ± 4.8 |
| HAQ | 1.13 ± 0.61 |
| Physician’s global assessment | 49.4 ± 19.0 |
| RF at baseline | 238.2 ± 369.6 |
| RF+ % | 83.75%* |
| Serum CRP level (mg/dl) at baseline | 1.7 ± 2.0 |
| CCP+ % | 61.9%** |
| DAS 28 at baseline | 5.22 ± 0.80 |
| DAS 28 at 6 weeks | 3.99 ± 1.13 |
| DAS 28 at 14 weeks | 3.72 ± 1.32 |
| anti-TNF drug |  |
| Enbrel | n = 39 |
| Humira | n = 18 |
| Remicade | n = 32 |
| Other Medication |  |
| Steroids | 63% |
| Methotrexate | 63% |
| DMARDS | 46% |
| Arava | n = 16 |
| Azathioprine | n = 1 |
| Sulfasalazine | n = 18 |
| Plaquenil | n = 15 |

Table 1 Caption: *11 subjects missing RF data **5 subjects missing CCP data

### Genome-wide association studies: 1st stage analysis

The GWA analyses were performed to test for association between 283,348 polymorphic SNPs and the relative change in DAS28 (*relDAS*28) using an additive genetic model. We did not further adjust the principal components in our regression model, because they did not significantly correlate with *relDAS*28 (i.e. the major phenotypic difference among individuals was not due to population stratification) (Supplementary [Table 1](#SD1)). The chromosomal distribution of p-values for the genome wide association is shown in [Fig. 1](#f1-08_56_liu). To address multiple testing issues, a chromosome-wide permutation test was performed. Sixteen SNPs remain significant (permutation exact p 0.05) or borderline significant (0.05 < permutation exact p 0.1) after permutation test to obtain exact significance levels. The p-values along with annotation information for these 16 SNPs are shown in [Table 2](#t2-08_56_liu). Among these sixteen SNPs, four are located within genes, one is in the 3’UTR, and eleven are in the flanking regions of genes. All these SNPs are common polymorphisms (minor allele frequency > 0.1). In addition to [Table 2](#t2-08_56_liu), we also provide data for 2985 SNPs with *relDAS*28 p-values 0.01 in Supplementary [Table 1](#SD1). We used the Illumina annotation file “HumanHap317K_geneannotation.txt” together with the UCSC Genome Browser to annotate SNP details ([22](#b22-08_56_liu)).

### Fig. 1.

![Fig. 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/bcdb/2536555/cd38c98d8767/mol-14-9-0561f1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=2536555_mol-14-9-0561f1.jpg)

Genome wide association p-value plots showing the association of single-nucleotide polymorphism (SNPs) with relative change in DAS28 score (relDAS28). Chromosomal location is shown on the abscissa. P values shown on the ordinate are uncorrected.

### Table 2.

| SNP | P-value1 of RelDAS28 | Permut. p-value (RelDAS28) | Polymorphism | Allele associated with non-response (Allele Frequency) | Chr | Physical position | Known Genes | Location3 | P-value2 Non-responders vs. Good responders (Adjusted)4 | Odds Ratio (OR) being non-responder (95% CI) 2 |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| rs983332 | 0.000005 | 0.008 | A/C | A (0.21) | 1 | 87844401 | LMO4 | F3U | 0.00007 (0.00009) | 10.2 (2.6, 59.2) |
| rs928655 | 0.00003 | 0.07 | A/G | A (0.77) | 1 | 89561595 | GBP6 | I | 0.0009 (0.0004) | 5.5 (1.8, 20.2) |
| rs13393173 | 0.000004 | 0.02 | A/G | A (0.12) | 2 | 169214598 | LASS6 | I | 0.004 (0.02) | 6.8 (1.7, 40.3) |
| rs437943 | 0.000004 | 0.1 | A/G | G (0.33) | 4 | 35194664 | CENTD1 | F3U | 0.0007 (0.002) | 4.6 (1.8, 12.3) |
| rs10945919 | 0.0000003 | 0.004 | A/G | G (0.32) | 6 | 164157088 | QKI | F3U | 0.0007 (0.0008) | 4.6 (1.8, 12.3) |
| rs854555 | 0.000002 | 0.03 | A/C | A (0.34) | 7 | 94575042 | PON1 | I | 0.0006 (0.001) | 4.6 (1.8, 12.3) |
| rs854548 | 0.000003 | 0.06 | A/G | A (0.27) | 7 | 94570471 | PON1 | F3U | 0.00004 (0.0003) | 8.5 (2.6, 36.5) |
| rs854547 | 0.000006 | 0.1 | A/G | A (0.63) | 7 | 94568507 | PON1 | F3U | 0.003 (0.004) | 3.6 (1.5, 9.3) |
| rs7046653 | 0.0000005 | 0.01 | A/G | A (0.26) | 9 | 27480967 | IFNK | F5U | 0.0004 (0.002) | 4.9 (1.8, 14.0) |
| rs868856 | 0.0000005 | 0.01 | C/T | T (0.26) | 9 | 27479251 | MOBKL2B | I | 0.0005 (0.002) | 4.9 (1.8, 14.0) |
| rs774359 | 0.0000006 | 0.01 | C/T | C (0.22) | 9 | 27551049 | C9orf72 | 3UTR | 0.0005 (0.005) | 5.4 (1.9, 17.3) |
| rs2814707 | 0.000002 | 0.04 | A/G | A (0.22) | 9 | 27526397 | MOBKL2B | F5U | 0.0006 (0.007) | 5.2 (1.8, 16.7) |
| rs3849942 | 0.000005 | 0.07 | A/G | A (0.21) | 9 | 27533281 | C9orf72 | F3U | 0.001 (0.01) | 5.0 (1.7, 15.8) |
| rs6028945 | 0.0000002 | 0.003 | G/T | T (0.12) | 20 | 38254219 | MAFB | F3U | 0.0004 (0.006) | 11.2 (2.3, 108.1) |
| rs6138150 | 0.000003 | 0.05 | C/T | T (0.84) | 20 | 23795009 | CST5 | F3U | 0.0002 (0.002) | 11.1 (2.5, 103.3) |
| rs6071980 | 0.000003 | 0.05 | C/T | C (0.13) | 20 | 38301990 | MAFB | F3U | 0.0009 (0.01) | 7.6 (1.9, 44.6) |

Table 2 Caption: Sixteen SNPs with smallest p-values with permutation exact p-values 0.1

Among these leading associations, the rs6028945 marker ~500Kb 3’ of the MAFB locus on chromosome 20 is marginally the strongest association (p=0.003 corrected) when the relative change in DAS28 is considered as a continuous variable; a second marker in the region of MAFB, rs6071980 also shows evidence of association (p=0.05, corrected). Likewise, multimarker evidence of association is seen with markers in the Paraoxinase 1 gene (PON1) as well as in a region of chromosome 9 that contains the interferon kappa (IFNK), MOBKL2B and C9orf72 loci. Other loci showing some evidence of association include a guanylate nucleotide binding protein (GBP6) at 1p22.2, LAG1 (longevity assurance homolog 6, LASS6) at chr2, cystatin D (CST5) at 20p11.21, centaurin, delta 1 (CENTD1) at 4p14, quaking homolog, and KH domain RNA binding (QK1) at 6q26-q27 ([Table 2](#t2-08_56_liu)).

### Association between SNPs and response to anti-TNF therapy: 2nd stage analysis

According to the criteria of EULAR( [16](#b16-08_56_liu)), the 89 patients under study can be categorized into three groups: 23 non-responders, 31 good responders and 35 moderate responders using the baseline DAS28 and the change in DAS28 at 14 weeks. We compared allele frequencies between non-responders and good responders using the Fisher’s exact test ([23](#b23-08_56_liu)). The last two columns of [Table 2](#t2-08_56_liu) display Fishers exact p-values and Odds Ratios (ORs) with 95% confidence intervals (CIs) for the non-responder status to the anti-TNF therapies. Note that these 95% CIs are very wide, reflecting the small sample size (i.e. 23 non-responders vs. 31 good-responders) in this study. The major principal components did not appear to predict good responder vs. nonresponder status (Supplementary [Fig. 2](#SD1)). Because the sample size is quite small (23 non-responders vs. 31 responders), a minor effect due to subpopulation may have large impact on the 2X2 table estimate. Therefore, we further adjusted subpopulation structure to give adjusted Chi-square p-values using EIGENSTRAT. As shown in [Table 2](#t2-08_56_liu), these adjusted Chi-square p-values are generally larger than the Fisher’s exact test p-values.

### SNPs in candidate genes

Of the markers present on the Illumina HapMap 300 chip, four SNP associations with TNF response have been reported previously in candidate gene studies: rs1800896 in IL10 ([8](#b8-08_56_liu), [24](#b24-08_56_liu)), rs419598 in interleukin 1 receptor antagonist (IL1RN) ([14](#b14-08_56_liu)), rs1041981 in LTA([25](#b25-08_56_liu)–[27](#b27-08_56_liu)), and rs4149570 in TNFRSF1A([26](#b26-08_56_liu)). These markers were evaluated separately in view of their increased prior probability. In the current study, only rs1800896 shows evidence of association, with a p-value of 0.0132 (uncorrected) with the *delDAS*28 and 0.0183 (corrected for stratification only) with responding status to anti-TNF therapy.

## Discussion

In this report, we describe the first genome-wide association study to evaluate pharmacogenetic effects on the response to anti-TNF treatment for rheumatoid arthritis. Several SNPs show significant association with the change in DAS28 observed in these patients over a 14-week period of treatment.

Two SNPs located approximately 500kb downstream of the MAFB (v-maf musculoaponeurotic fibrosarcoma oncogene homolog B) locus show significant evidence of association after correction by permutation and after accounting for the possible effects of population stratification. This gene is a member of the Maf family of bZIP transcription factors that are generally involved in cellular differentiation([28](#b28-08_56_liu)). MAFB is a putative tumor suppressor in the myeloid lineage, with a key role in monopoiesis([29](#b29-08_56_liu)) as well as monocytedendritic cell differentiation([30](#b30-08_56_liu)). Little is known about the biology of this gene in the context of inflammatory disease.

Another interesting association with anti-TNF response was found in the paraoxonase (PON1) locus. Paraoxonase is an enzyme associated with high density lipoproteins that may play a role in inflammatory disease([31](#b31-08_56_liu)). Low serum levels of PON1 have been reported in inflammatory disorders ([32](#b32-08_56_liu), [33](#b33-08_56_liu)), including rheumatoid arthritis([34](#b34-08_56_liu)), although in the setting of autoimmunity, most attention as been on the anti-inflammatory effects of PON1 in risk for cardiovascular disease([35](#b35-08_56_liu)). Genetic regulation of Paraoxonase 1 levels has been documented([36](#b36-08_56_liu), [37](#b37-08_56_liu)), although not always by polymorphisms within PON1 ([38](#b38-08_56_liu)). TNF-α, IL6 and PON1 levels appear to be correlated in the setting of rheumatoid arthritis([39](#b39-08_56_liu)), although it is certainly not clear why genetic variation in PON1 should influence response to TNF inhibition.

Five SNPs in a region of chromosome 9 across a 70kb region also showed association with treatment response. Of the three genes in this region with high LD, IFNκ is the most compelling candidate for involvement in inflammation and/or response to anti-TNF treatment, since type I IFNs clearly play a role in inflammatory disease([40](#b40-08_56_liu)).

Only 4 specific SNPs previously tested for association with anti-TNF treatment were included on the 317K chip. Of these, we found that the G allele of rs1800896 (flanking region of IL10) is associated with good response to anti-TNF therapies (OR = 2.7 (95%CI: 1.2~6.7), Fisher’s p = 0.0183), which is consistent with previous result that the combination of GG at this locus was associated with good response to etanercept ([8](#b8-08_56_liu)). One SNP reported to be associated with response to anti-TNF treatment is rs1800629, located in the promoter of the TNF-α gene ([7](#b7-08_56_liu)). Neither this SNP nor perfect surrogates are on the Illumina 317K array. Thus, our data cannot replicate this previous finding.

As with other previous reports on the pharmacogenetics of TNF response, this study is limited by a relatively small sample size. Therefore, instead of a dichotomous classification of patients into non-responders vs. good-responders, we first utilized the continuous score of the relative change in DAS28 (i.e., *delDAS*28) as the dependent variable in a regression model, so that we were able utilize all subjects and therefore maximize the statistical power of the study. We also performed chromosome-wide permutation tests to address the issue of multiple testing. Bonferroni adjustments are often used in adjusting statistical significance for multiple tests. However, the Bonferroni test is highly conservative, particularly in the way it tests the overall null hypothesis, i.e. all null hypotheses are true simultaneously ([41](#b41-08_56_liu)). Furthermore, Bonferroni assumes all tests are independent, and since there is linkage disequilibrium among some of the SNPs on the 317K chip the tests are not truly independent. Finally, in addition to analyzing the change in DAS28 as a continuous trait, we classified the patients into 3 response groups (i.e. non-, moderate, and good responders) and analyzed association to anti-TNF response using the two extreme groups. The Fisher’s exact p-values for the dichotomous trait are consistent with the *relDAS*28 p-values in most circumstances (i.e. smaller Fisher’s exact p-values are consistent with smaller *relDAS*28 p-values).

In addition to carefully choosing the analytical approach in this analysis, we thoroughly investigated sub-population structure in our study subjects to reduce false positive association that might be caused by population admixture ([19](#b19-08_56_liu), [20](#b20-08_56_liu)). We calculated IBS estimation and further performed agglomerative clustering analysis using the PLINK software to detect population admixture and strata. We selected 89 subjects with predominantly European ancestry and performed subsequently statistical analyses using only these subjects. Furthermore, we performed principal component analysis using the EIGENSTRAT software to detect possible minor phenotypic difference in individuals due to any subpopulation structure and provided adjusted Chi-square p-values for selected SNPs in this report.

A further caution regarding these results relates to the fact that our population was rather diverse clinically, with some patients having quite longstanding disease and nearly 40% of the patients were CCP antibody negative. Seven of the 89 patients had previously been treated with a TNF inhibitor. Therefore, this population is clearly not representative of a new onset cohort, where information about TNF responsiveness might be most clinically useful. Clearly, replications of these results in independent and much larger data sets are required to confirm these findings. Given the fact that so many patients have been treated with these agents over the last decade, it is disappointing that these population resources are not readily available. Nevertheless, several European RA cohorts may be useful for such replication studies ([7](#b7-08_56_liu), [42](#b42-08_56_liu)). In addition, we and others are actively engaged in an effort to recruit a U.S. cohort of up to 1,000 RA patients beginning treatment with TNF inhibitors. Samples sizes of this magnitude or larger will be required in order to provide robust replication of the current results.

A test predicting response or non-response to anti-TNF treatment would be an important tool in the hands of physicians. TNF inhibitors are currently a mainstay of biologic therapy in RA. However, only a fraction of patients show a clinically meaningful response to anti-TNF treatment after 3–6 months. Furthermore, many patients lose response over time. After patients fail their first TNF inhibitor, physicians are faced with a choice between trying a second TNF inhibitor or moving to a biologic therapy with a different mechanism of action([43](#b43-08_56_liu)). Additional biologic agents with different mechanisms of action are available for RA patients who have an inadequate response to TNF inhibitors and more potential therapies are in late phases of clinical development. The ability to determine whether patients are likely to be a non-responder to a TNF inhibitor would make these treatment decisions more rational. The current results suggest that this may be achievable.

## Supplemental Data

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Feldmann M, Brennan FM, Maini RN. Role of cytokines in rheumatoid arthritis. Annu Rev Immunol. 1996;14:397–440. doi: 10.1146/annurev.immunol.14.1.397.  [DOI](https://doi.org/10.1146/annurev.immunol.14.1.397) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8717520/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annu%20Rev%20Immunol&title=Role%20of%20cytokines%20in%20rheumatoid%20arthritis&author=M%20Feldmann&author=FM%20Brennan&author=RN%20Maini&volume=14&publication_year=1996&pages=397-440&pmid=8717520&doi=10.1146/annurev.immunol.14.1.397&)

2. Feldmann M. What is the mechanism of action of anti-tumour necrosis factor-alpha antibody in rheumatoid arthritis? Int Arch Allergy Immunol. 1996;111:362–365. doi: 10.1159/000237393.  [DOI](https://doi.org/10.1159/000237393) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8957109/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20Arch%20Allergy%20Immunol&title=What%20is%20the%20mechanism%20of%20action%20of%20anti-tumour%20necrosis%20factor-alpha%20antibody%20in%20rheumatoid%20arthritis?&author=M%20Feldmann&volume=111&publication_year=1996&pages=362-365&pmid=8957109&doi=10.1159/000237393&)

3. Saxne T, Palladino MA, Jr, Heinegard D, Talal N, Wollheim FA. Detection of tumor necrosis factor alpha but not tumor necrosis factor beta in rheumatoid arthritis synovial fluid and serum. Arthritis and rheumatism. 1988;31:1041–1045. doi: 10.1002/art.1780310816.  [DOI](https://doi.org/10.1002/art.1780310816) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/3136775/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arthritis%20and%20rheumatism&title=Detection%20of%20tumor%20necrosis%20factor%20alpha%20but%20not%20tumor%20necrosis%20factor%20beta%20in%20rheumatoid%20arthritis%20synovial%20fluid%20and%20serum&author=T%20Saxne&author=MA%20Palladino&author=D%20Heinegard&author=N%20Talal&author=FA%20Wollheim&volume=31&publication_year=1988&pages=1041-1045&pmid=3136775&doi=10.1002/art.1780310816&)

4. Bathon JM, Martin RW, Fleischmann RM, et al. A comparison of etanercept and methotrexate in patients with early rheumatoid arthritis. N Engl J Med. 2000;343:1586–1593. doi: 10.1056/NEJM200011303432201.  [DOI](https://doi.org/10.1056/NEJM200011303432201) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11096165/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=A%20comparison%20of%20etanercept%20and%20methotrexate%20in%20patients%20with%20early%20rheumatoid%20arthritis&author=JM%20Bathon&author=RW%20Martin&author=RM%20Fleischmann&volume=343&publication_year=2000&pages=1586-1593&pmid=11096165&doi=10.1056/NEJM200011303432201&)

5. Mugnier B, Balandraud N, Darque A, Roudier C, Roudier J, Reviron D. Polymorphism at position -308 of the tumor necrosis factor alpha gene influences outcome of infliximab therapy in rheumatoid arthritis. Arthritis and rheumatism. 2003;48:1849–1852. doi: 10.1002/art.11168.  [DOI](https://doi.org/10.1002/art.11168) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12847678/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arthritis%20and%20rheumatism&title=Polymorphism%20at%20position%20-308%20of%20the%20tumor%20necrosis%20factor%20alpha%20gene%20influences%20outcome%20of%20infliximab%20therapy%20in%20rheumatoid%20arthritis&author=B%20Mugnier&author=N%20Balandraud&author=A%20Darque&author=C%20Roudier&author=J%20Roudier&volume=48&publication_year=2003&pages=1849-1852&pmid=12847678&doi=10.1002/art.11168&)

6. Martinez A, Salido M, Bonilla G, et al. Association of the major histocompatibility complex with response to infliximab therapy in rheumatoid arthritis patients. Arthritis and rheumatism. 2004;50:1077–1082. doi: 10.1002/art.20154.  [DOI](https://doi.org/10.1002/art.20154) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15077289/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arthritis%20and%20rheumatism&title=Association%20of%20the%20major%20histocompatibility%20complex%20with%20response%20to%20infliximab%20therapy%20in%20rheumatoid%20arthritis%20patients&author=A%20Martinez&author=M%20Salido&author=G%20Bonilla&volume=50&publication_year=2004&pages=1077-1082&pmid=15077289&doi=10.1002/art.20154&)

7. Coenen MJ, Toonen EJ, Scheffer H, Radstake TR, Barrera P, Franke B. Pharmacogenetics of anti-TNF treatment in patients with rheumatoid arthritis. Pharmacogenomics. 2007;8:761–773. doi: 10.2217/14622416.8.7.761.  [DOI](https://doi.org/10.2217/14622416.8.7.761) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17638513/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Pharmacogenetics%20of%20anti-TNF%20treatment%20in%20patients%20with%20rheumatoid%20arthritis&author=MJ%20Coenen&author=EJ%20Toonen&author=H%20Scheffer&author=TR%20Radstake&author=P%20Barrera&volume=8&publication_year=2007&pages=761-773&pmid=17638513&doi=10.2217/14622416.8.7.761&)

8. Padyukov L, Lampa J, Heimburger M, et al. Genetic markers for the efficacy of tumour necrosis factor blocking therapy in rheumatoid arthritis. Ann Rheum Dis. 2003;62:526–529. doi: 10.1136/ard.62.6.526.  [DOI](https://doi.org/10.1136/ard.62.6.526) | [PMC free article](/articles/PMC1754569/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12759288/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Rheum%20Dis&title=Genetic%20markers%20for%20the%20efficacy%20of%20tumour%20necrosis%20factor%20blocking%20therapy%20in%20rheumatoid%20arthritis&author=L%20Padyukov&author=J%20Lampa&author=M%20Heimburger&volume=62&publication_year=2003&pages=526-529&pmid=12759288&doi=10.1136/ard.62.6.526&)

9. Balog A, Klausz G, Gal J, et al. Investigation of the prognostic value of TNF-alpha gene polymorphism among patients treated with infliximab, and the effects of infliximab therapy on TNF-alpha production and apoptosis. Pathobiology. 2004;71:274–280. doi: 10.1159/000080062.  [DOI](https://doi.org/10.1159/000080062) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15459487/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pathobiology&title=Investigation%20of%20the%20prognostic%20value%20of%20TNF-alpha%20gene%20polymorphism%20among%20patients%20treated%20with%20infliximab,%20and%20the%20effects%20of%20infliximab%20therapy%20on%20TNF-alpha%20production%20and%20apoptosis&author=A%20Balog&author=G%20Klausz&author=J%20Gal&volume=71&publication_year=2004&pages=274-280&pmid=15459487&doi=10.1159/000080062&)

10. Fonseca JE, Carvalho T, Cruz M, et al. Polymorphism at position -308 of the tumour necrosis factor alpha gene and rheumatoid arthritis pharmacogenetics. Ann Rheum Dis. 2005;64:793–794. doi: 10.1136/ard.2004.028167.  [DOI](https://doi.org/10.1136/ard.2004.028167) | [PMC free article](/articles/PMC1755496/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15834068/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Rheum%20Dis&title=Polymorphism%20at%20position%20-308%20of%20the%20tumour%20necrosis%20factor%20alpha%20gene%20and%20rheumatoid%20arthritis%20pharmacogenetics&author=JE%20Fonseca&author=T%20Carvalho&author=M%20Cruz&volume=64&publication_year=2005&pages=793-794&pmid=15834068&doi=10.1136/ard.2004.028167&)

11. Cuchacovich M, Soto L, Edwardes M, et al. Tumour necrosis factor (TNF)alpha -308 G/G promoter polymorphism and TNFalpha levels correlate with a better response to adalimumab in patients with rheumatoid arthritis. Scand J Rheumatol. 2006;35:435–440. doi: 10.1080/03009740600904284.  [DOI](https://doi.org/10.1080/03009740600904284) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17343250/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Scand%20J%20Rheumatol&title=Tumour%20necrosis%20factor%20(TNF)alpha%20-308%20G/G%20promoter%20polymorphism%20and%20TNFalpha%20levels%20correlate%20with%20a%20better%20response%20to%20adalimumab%20in%20patients%20with%20rheumatoid%20arthritis&author=M%20Cuchacovich&author=L%20Soto&author=M%20Edwardes&volume=35&publication_year=2006&pages=435-440&pmid=17343250&doi=10.1080/03009740600904284&)

12. Seitz M, Wirthmuller U, Moller B, Villiger PM. The -308 tumour necrosis factor-alpha gene polymorphism predicts therapeutic response to TNFalpha-blockers in rheumatoid arthritis and spondyloarthritis patients. Rheumatology (Oxford) 2007;46:93–96. doi: 10.1093/rheumatology/kel175.  [DOI](https://doi.org/10.1093/rheumatology/kel175) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16720636/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Rheumatology%20(Oxford)&title=The%20-308%20tumour%20necrosis%20factor-alpha%20gene%20polymorphism%20predicts%20therapeutic%20response%20to%20TNFalpha-blockers%20in%20rheumatoid%20arthritis%20and%20spondyloarthritis%20patients&author=M%20Seitz&author=U%20Wirthmuller&author=B%20Moller&author=PM%20Villiger&volume=46&publication_year=2007&pages=93-96&pmid=16720636&doi=10.1093/rheumatology/kel175&)

13. Lee YH, Rho YH, Choi SJ, Ji JD, Song GG. Association of TNF-alpha -308 G/A polymorphism with responsiveness to TNF-alpha-blockers in rheumatoid arthritis: a meta-analysis. Rheumatol Int. 2006;27:157–161. doi: 10.1007/s00296-006-0175-7.  [DOI](https://doi.org/10.1007/s00296-006-0175-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16909270/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Rheumatol%20Int&title=Association%20of%20TNF-alpha%20-308%20G/A%20polymorphism%20with%20responsiveness%20to%20TNF-alpha-blockers%20in%20rheumatoid%20arthritis:%20a%20meta-analysis&author=YH%20Lee&author=YH%20Rho&author=SJ%20Choi&author=JD%20Ji&author=GG%20Song&volume=27&publication_year=2006&pages=157-161&pmid=16909270&doi=10.1007/s00296-006-0175-7&)

14. Marotte H, Pallot-Prades B, Grange L, et al. The shared epitope is a marker of severity associated with selection for, but not with response to, infliximab in a large rheumatoid arthritis population. Ann Rheum Dis. 2006;65:342–347. doi: 10.1136/ard.2005.037150.  [DOI](https://doi.org/10.1136/ard.2005.037150) | [PMC free article](/articles/PMC1798045/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16096333/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Rheum%20Dis&title=The%20shared%20epitope%20is%20a%20marker%20of%20severity%20associated%20with%20selection%20for,%20but%20not%20with%20response%20to,%20infliximab%20in%20a%20large%20rheumatoid%20arthritis%20population&author=H%20Marotte&author=B%20Pallot-Prades&author=L%20Grange&volume=65&publication_year=2006&pages=342-347&pmid=16096333&doi=10.1136/ard.2005.037150&)

15. Prevoo ML, van 't Hof MA, Kuper HH, van Leeuwen MA, van de Putte LB, van Riel PL. Modified disease activity scores that include twenty-eight-joint counts. Development and validation in a prospective longitudinal study of patients with rheumatoid arthritis. Arthritis and rheumatism. 1995;38:44–48. doi: 10.1002/art.1780380107.  [DOI](https://doi.org/10.1002/art.1780380107) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7818570/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arthritis%20and%20rheumatism&title=Modified%20disease%20activity%20scores%20that%20include%20twenty-eight-joint%20counts.%20Development%20and%20validation%20in%20a%20prospective%20longitudinal%20study%20of%20patients%20with%20rheumatoid%20arthritis&author=ML%20Prevoo&author=MA%20van%20't%20Hof&author=HH%20Kuper&author=MA%20van%20Leeuwen&author=LB%20van%20de%20Putte&volume=38&publication_year=1995&pages=44-48&pmid=7818570&doi=10.1002/art.1780380107&)

16. van Gestel AM, Haagsma CJ, van Riel PL. Validation of rheumatoid arthritis improvement criteria that include simplified joint counts. Arthritis and rheumatism. 1998;41:1845–1850. doi: 10.1002/1529-0131(199810)41:10<1845::AID-ART17>3.0.CO;2-K.  [DOI](https://doi.org/10.1002/1529-0131(199810)41:10<1845::AID-ART17>3.0.CO;2-K) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9778226/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arthritis%20and%20rheumatism&title=Validation%20of%20rheumatoid%20arthritis%20improvement%20criteria%20that%20include%20simplified%20joint%20counts&author=AM%20van%20Gestel&author=CJ%20Haagsma&author=PL%20van%20Riel&volume=41&publication_year=1998&pages=1845-1850&pmid=9778226&doi=10.1002/1529-0131(199810)41:10<1845::AID-ART17>3.0.CO;2-K&)

17. Duerr RH, Taylor KD, Brant SR, et al. A genome-wide association study identifies IL23R as an inflammatory bowel disease gene. Science. 2006;314:1461–1463. doi: 10.1126/science.1135245.  [DOI](https://doi.org/10.1126/science.1135245) | [PMC free article](/articles/PMC4410764/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17068223/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=A%20genome-wide%20association%20study%20identifies%20IL23R%20as%20an%20inflammatory%20bowel%20disease%20gene&author=RH%20Duerr&author=KD%20Taylor&author=SR%20Brant&volume=314&publication_year=2006&pages=1461-1463&pmid=17068223&doi=10.1126/science.1135245&)

18. Pe'er I, de Bakker PI, Maller J, Yelensky R, Altshuler D, Daly MJ. Evaluating and improving power in whole-genome association studies using fixed marker sets. Nat Genet. 2006;38:663–667. doi: 10.1038/ng1816.  [DOI](https://doi.org/10.1038/ng1816) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16715096/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Evaluating%20and%20improving%20power%20in%20whole-genome%20association%20studies%20using%20fixed%20marker%20sets&author=I%20Pe'er&author=PI%20de%20Bakker&author=J%20Maller&author=R%20Yelensky&author=D%20Altshuler&volume=38&publication_year=2006&pages=663-667&pmid=16715096&doi=10.1038/ng1816&)

19. Deng HW. Population admixture may appear to mask, change or reverse genetic effects of genes underlying complex traits. Genetics. 2001;159:1319–1323. doi: 10.1093/genetics/159.3.1319.  [DOI](https://doi.org/10.1093/genetics/159.3.1319) | [PMC free article](/articles/PMC1461858/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11729172/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genetics&title=Population%20admixture%20may%20appear%20to%20mask,%20change%20or%20reverse%20genetic%20effects%20of%20genes%20underlying%20complex%20traits&author=HW%20Deng&volume=159&publication_year=2001&pages=1319-1323&pmid=11729172&doi=10.1093/genetics/159.3.1319&)

20. Seldin MF, Price AL. Application of ancestry informative markers to association studies in European Americans. PLoS genetics. 2008;4:e5. doi: 10.1371/journal.pgen.0040005.  [DOI](https://doi.org/10.1371/journal.pgen.0040005) | [PMC free article](/articles/PMC2211545/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18208330/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20genetics&title=Application%20of%20ancestry%20informative%20markers%20to%20association%20studies%20in%20European%20Americans&author=MF%20Seldin&author=AL%20Price&volume=4&publication_year=2008&pages=e5&pmid=18208330&doi=10.1371/journal.pgen.0040005&)

21. Price AL, Patterson NJ, Plenge RM, Weinblatt ME, Shadick NA, Reich D. Principal components analysis corrects for stratification in genome-wide association studies. Nat Genet. 2006;38:904–909. doi: 10.1038/ng1847.  [DOI](https://doi.org/10.1038/ng1847) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16862161/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Principal%20components%20analysis%20corrects%20for%20stratification%20in%20genome-wide%20association%20studies&author=AL%20Price&author=NJ%20Patterson&author=RM%20Plenge&author=ME%20Weinblatt&author=NA%20Shadick&volume=38&publication_year=2006&pages=904-909&pmid=16862161&doi=10.1038/ng1847&)

22. Kent WJ, Sugnet CW, Furey TS, et al. The human genome browser at UCSC. Genome Res. 2002;12:996–1006. doi: 10.1101/gr.229102.  [DOI](https://doi.org/10.1101/gr.229102) | [PMC free article](/articles/PMC186604/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12045153/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res&title=The%20human%20genome%20browser%20at%20UCSC&author=WJ%20Kent&author=CW%20Sugnet&author=TS%20Furey&volume=12&publication_year=2002&pages=996-1006&pmid=12045153&doi=10.1101/gr.229102&)

23. Fisher RA. On the interpretation of χ2 from contingency tables, and the calculation of P. Journal of the Royal Statistical Society. 1922;85:87–94.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20the%20Royal%20Statistical%20Society&title=On%20the%20interpretation%20of%20%CF%872%20from%20contingency%20tables,%20and%20the%20calculation%20of%20P&author=RA%20Fisher&volume=85&publication_year=1922&pages=87-94&)

24. Turner DM, Williams DM, Sankaran D, Lazarus M, Sinnott PJ, Hutchinson IV. An investigation of polymorphism in the interleukin-10 gene promoter. Eur J Immunogenet. 1997;24:1–8. doi: 10.1111/j.1365-2370.1997.tb00001.x.  [DOI](https://doi.org/10.1111/j.1365-2370.1997.tb00001.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9043871/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Immunogenet&title=An%20investigation%20of%20polymorphism%20in%20the%20interleukin-10%20gene%20promoter&author=DM%20Turner&author=DM%20Williams&author=D%20Sankaran&author=M%20Lazarus&author=PJ%20Sinnott&volume=24&publication_year=1997&pages=1-8&pmid=9043871&doi=10.1111/j.1365-2370.1997.tb00001.x&)

25. Kang CP, Lee KW, Yoo DH, Kang C, Bae SC. The influence of a polymorphism at position -857 of the tumour necrosis factor alpha gene on clinical response to etanercept therapy in rheumatoid arthritis. Rheumatology (Oxford) 2005;44:547–552. doi: 10.1093/rheumatology/keh550.  [DOI](https://doi.org/10.1093/rheumatology/keh550) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15695296/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Rheumatology%20(Oxford)&title=The%20influence%20of%20a%20polymorphism%20at%20position%20-857%20of%20the%20tumour%20necrosis%20factor%20alpha%20gene%20on%20clinical%20response%20to%20etanercept%20therapy%20in%20rheumatoid%20arthritis&author=CP%20Kang&author=KW%20Lee&author=DH%20Yoo&author=C%20Kang&author=SC%20Bae&volume=44&publication_year=2005&pages=547-552&pmid=15695296&doi=10.1093/rheumatology/keh550&)

26. Criswell LA, Lum RF, Turner KN, et al. The influence of genetic variation in the HLA-DRB1 and LTA-TNF regions on the response to treatment of early rheumatoid arthritis with methotrexate or etanercept. Arthritis and rheumatism. 2004;50:2750–2756. doi: 10.1002/art.20469.  [DOI](https://doi.org/10.1002/art.20469) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15457442/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arthritis%20and%20rheumatism&title=The%20influence%20of%20genetic%20variation%20in%20the%20HLA-DRB1%20and%20LTA-TNF%20regions%20on%20the%20response%20to%20treatment%20of%20early%20rheumatoid%20arthritis%20with%20methotrexate%20or%20etanercept&author=LA%20Criswell&author=RF%20Lum&author=KN%20Turner&volume=50&publication_year=2004&pages=2750-2756&pmid=15457442&doi=10.1002/art.20469&)

27. Koss K, Satsangi J, Fanning GC, Welsh KI, Jewell DP. Cytokine (TNF alpha, LT alpha and IL-10) polymorphisms in inflammatory bowel diseases and normal controls: differential effects on production and allele frequencies. Genes Immun. 2000;1:185–190. doi: 10.1038/sj.gene.6363657.  [DOI](https://doi.org/10.1038/sj.gene.6363657) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11196710/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genes%20Immun&title=Cytokine%20(TNF%20alpha,%20LT%20alpha%20and%20IL-10)%20polymorphisms%20in%20inflammatory%20bowel%20diseases%20and%20normal%20controls:%20differential%20effects%20on%20production%20and%20allele%20frequencies&author=K%20Koss&author=J%20Satsangi&author=GC%20Fanning&author=KI%20Welsh&author=DP%20Jewell&volume=1&publication_year=2000&pages=185-190&pmid=11196710&doi=10.1038/sj.gene.6363657&)

28. Yang Y, Cvekl A. Large Maf Transcription Factors: Cousins of AP-1 Proteins and Important Regulators of Cellular Differentiation. Einstein J Biol Med. 2007;23:2–11. doi: 10.23861/ejbm20072347.  [DOI](https://doi.org/10.23861/ejbm20072347) | [PMC free article](/articles/PMC2151748/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18159220/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Einstein%20J%20Biol%20Med&title=Large%20Maf%20Transcription%20Factors:%20Cousins%20of%20AP-1%20Proteins%20and%20Important%20Regulators%20of%20Cellular%20Differentiation&author=Y%20Yang&author=A%20Cvekl&volume=23&publication_year=2007&pages=2-11&pmid=18159220&doi=10.23861/ejbm20072347&)

29. Gemelli C, Montanari M, Tenedini E, et al. Virally mediated MafB transduction induces the monocyte commitment of human CD34+ hematopoietic stem/progenitor cells. Cell death and differentiation. 2006;13:1686–1696. doi: 10.1038/sj.cdd.4401860.  [DOI](https://doi.org/10.1038/sj.cdd.4401860) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16456583/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20death%20and%20differentiation&title=Virally%20mediated%20MafB%20transduction%20induces%20the%20monocyte%20commitment%20of%20human%20CD34+%20hematopoietic%20stem/progenitor%20cells&author=C%20Gemelli&author=M%20Montanari&author=E%20Tenedini&volume=13&publication_year=2006&pages=1686-1696&pmid=16456583&doi=10.1038/sj.cdd.4401860&)

30. Bakri Y, Sarrazin S, Mayer UP, et al. Balance of MafB and PU.1 specifies alternative macrophage or dendritic cell fate. Blood. 2005;105:2707–2716. doi: 10.1182/blood-2004-04-1448.  [DOI](https://doi.org/10.1182/blood-2004-04-1448) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15598817/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Balance%20of%20MafB%20and%20PU.1%20specifies%20alternative%20macrophage%20or%20dendritic%20cell%20fate&author=Y%20Bakri&author=S%20Sarrazin&author=UP%20Mayer&volume=105&publication_year=2005&pages=2707-2716&pmid=15598817&doi=10.1182/blood-2004-04-1448&)

31. Navab M, Anantharamaiah GM, Reddy ST, Van Lenten BJ, Ansell BJ, Fogelman AM. Mechanisms of disease: proatherogenic HDL--an evolving field. Nature clinical practice. 2006;2:504–511. doi: 10.1038/ncpendmet0245.  [DOI](https://doi.org/10.1038/ncpendmet0245) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16957764/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature%20clinical%20practice&title=Mechanisms%20of%20disease:%20proatherogenic%20HDL--an%20evolving%20field&author=M%20Navab&author=GM%20Anantharamaiah&author=ST%20Reddy&author=BJ%20Van%20Lenten&author=BJ%20Ansell&volume=2&publication_year=2006&pages=504-511&pmid=16957764&doi=10.1038/ncpendmet0245&)

32. Bodolay E, Seres I, Szodoray P, et al. Evaluation of paraoxonase activity in patients with mixed connective tissue disease. The Journal of rheumatology. 2008;35:237–243.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/18085736/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=The%20Journal%20of%20rheumatology&title=Evaluation%20of%20paraoxonase%20activity%20in%20patients%20with%20mixed%20connective%20tissue%20disease&author=E%20Bodolay&author=I%20Seres&author=P%20Szodoray&volume=35&publication_year=2008&pages=237-243&pmid=18085736&)

33. Kiss E, Seres I, Tarr T, Kocsis Z, Szegedi G, Paragh G. Reduced paraoxonase1 activity is a risk for atherosclerosis in patients with systemic lupus erythematosus. Annals of the New York Academy of Sciences. 2007;1108:83–91. doi: 10.1196/annals.1422.009.  [DOI](https://doi.org/10.1196/annals.1422.009) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17893973/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annals%20of%20the%20New%20York%20Academy%20of%20Sciences&title=Reduced%20paraoxonase1%20activity%20is%20a%20risk%20for%20atherosclerosis%20in%20patients%20with%20systemic%20lupus%20erythematosus&author=E%20Kiss&author=I%20Seres&author=T%20Tarr&author=Z%20Kocsis&author=G%20Szegedi&volume=1108&publication_year=2007&pages=83-91&pmid=17893973&doi=10.1196/annals.1422.009&)

34. Tanimoto N, Kumon Y, Suehiro T, et al. Serum paraoxonase activity decreases in rheumatoid arthritis. Life sciences. 2003;72:2877–2885. doi: 10.1016/s0024-3205(03)00195-4.  [DOI](https://doi.org/10.1016/s0024-3205(03)00195-4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12697270/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Life%20sciences&title=Serum%20paraoxonase%20activity%20decreases%20in%20rheumatoid%20arthritis&author=N%20Tanimoto&author=Y%20Kumon&author=T%20Suehiro&volume=72&publication_year=2003&pages=2877-2885&pmid=12697270&doi=10.1016/s0024-3205(03)00195-4&)

35. Batuca JR, Ames PR, Isenberg DA, Alves JD. Antibodies toward high-density lipoprotein components inhibit paraoxonase activity in patients with systemic lupus erythematosus. Annals of the New York Academy of Sciences. 2007;1108:137–146. doi: 10.1196/annals.1422.016.  [DOI](https://doi.org/10.1196/annals.1422.016) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17893980/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annals%20of%20the%20New%20York%20Academy%20of%20Sciences&title=Antibodies%20toward%20high-density%20lipoprotein%20components%20inhibit%20paraoxonase%20activity%20in%20patients%20with%20systemic%20lupus%20erythematosus&author=JR%20Batuca&author=PR%20Ames&author=DA%20Isenberg&author=JD%20Alves&volume=1108&publication_year=2007&pages=137-146&pmid=17893980&doi=10.1196/annals.1422.016&)

36. Brophy VH, Jampsa RL, Clendenning JB, McKinstry LA, Jarvik GP, Furlong CE. Effects of 5' regulatory-region polymorphisms on paraoxonasegene (PON1) expression. American journal of human genetics. 2001;68:1428–1436. doi: 10.1086/320600.  [DOI](https://doi.org/10.1086/320600) | [PMC free article](/articles/PMC1226129/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11335891/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=American%20journal%20of%20human%20genetics&title=Effects%20of%205'%20regulatory-region%20polymorphisms%20on%20paraoxonasegene%20(PON1)%20expression&author=VH%20Brophy&author=RL%20Jampsa&author=JB%20Clendenning&author=LA%20McKinstry&author=GP%20Jarvik&volume=68&publication_year=2001&pages=1428-1436&pmid=11335891&doi=10.1086/320600&)

37. Garin MC, James RW, Dussoix P, et al. Paraoxonase polymorphism Met-Leu54 is associated with modified serum concentrations of the enzyme. A possible link between the paraoxonase gene and increased risk of cardiovascular disease in diabetes. The Journal of clinical investigation. 1997;99:62–66. doi: 10.1172/JCI119134.  [DOI](https://doi.org/10.1172/JCI119134) | [PMC free article](/articles/PMC507768/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9011577/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=The%20Journal%20of%20clinical%20investigation&title=Paraoxonase%20polymorphism%20Met-Leu54%20is%20associated%20with%20modified%20serum%20concentrations%20of%20the%20enzyme.%20A%20possible%20link%20between%20the%20paraoxonase%20gene%20and%20increased%20risk%20of%20cardiovascular%20disease%20in%20diabetes&author=MC%20Garin&author=RW%20James&author=P%20Dussoix&volume=99&publication_year=1997&pages=62-66&pmid=9011577&doi=10.1172/JCI119134&)

38. Sanghera DK, Manzi S, Minster RL, et al. Genetic variation in the paraoxonase-3 (PON3) gene is associated with serum PON1 activity. Annals of human genetics. 2008;72:72–81. doi: 10.1111/j.1469-1809.2007.00400.x.  [DOI](https://doi.org/10.1111/j.1469-1809.2007.00400.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17900266/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annals%20of%20human%20genetics&title=Genetic%20variation%20in%20the%20paraoxonase-3%20(PON3)%20gene%20is%20associated%20with%20serum%20PON1%20activity&author=DK%20Sanghera&author=S%20Manzi&author=RL%20Minster&volume=72&publication_year=2008&pages=72-81&pmid=17900266&doi=10.1111/j.1469-1809.2007.00400.x&)

39. Kerekes G, Szekanecz Z, Der H, et al. Endothelial dysfunction and atherosclerosis in rheumatoid arthritis: a multiparametric analysis using imaging techniques and laboratory markers of inflammation and autoimmunity. The Journal of rheumatology. 2008;35:398–406.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/18203326/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=The%20Journal%20of%20rheumatology&title=Endothelial%20dysfunction%20and%20atherosclerosis%20in%20rheumatoid%20arthritis:%20a%20multiparametric%20analysis%20using%20imaging%20techniques%20and%20laboratory%20markers%20of%20inflammation%20and%20autoimmunity&author=G%20Kerekes&author=Z%20Szekanecz&author=H%20Der&volume=35&publication_year=2008&pages=398-406&pmid=18203326&)

40. Selmi C, Lleo A, Zuin M, Podda M, Rossaro L, Gershwin ME. Interferon alpha and its contribution to autoimmunity. Curr Opin Investig Drugs. 2006;7:451–456.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/16729722/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Opin%20Investig%20Drugs&title=Interferon%20alpha%20and%20its%20contribution%20to%20autoimmunity&author=C%20Selmi&author=A%20Lleo&author=M%20Zuin&author=M%20Podda&author=L%20Rossaro&volume=7&publication_year=2006&pages=451-456&pmid=16729722&)

41. Perneger TV. What's wrong with Bonferroni adjustments. Bmj. 1998;316:1236–1238. doi: 10.1136/bmj.316.7139.1236.  [DOI](https://doi.org/10.1136/bmj.316.7139.1236) | [PMC free article](/articles/PMC1112991/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9553006/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bmj&title=What's%20wrong%20with%20Bonferroni%20adjustments&author=TV%20Perneger&volume=316&publication_year=1998&pages=1236-1238&pmid=9553006&doi=10.1136/bmj.316.7139.1236&)

42. Sverdrup B, Kallberg H, Bengtsson C, et al. Association between occupational exposure to mineral oil and rheumatoid arthritis: results from the Swedish EIRA case-control study. Arthritis research & therapy. 2005;7:R1296–1303. doi: 10.1186/ar1824.  [DOI](https://doi.org/10.1186/ar1824) | [PMC free article](/articles/PMC1297574/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16277683/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arthritis%20research%20&%20therapy&title=Association%20between%20occupational%20exposure%20to%20mineral%20oil%20and%20rheumatoid%20arthritis:%20results%20from%20the%20Swedish%20EIRA%20case-control%20study&author=B%20Sverdrup&author=H%20Kallberg&author=C%20Bengtsson&volume=7&publication_year=2005&pages=R1296-1303&pmid=16277683&doi=10.1186/ar1824&)

43. Bokarewa M, Lindholm C, Zendjanchi K, Nadali M, Tarkowski A. Efficacy of anti-CD20 treatment in patients with rheumatoid arthritis resistant to a combination of methotrexate/anti-TNF therapy. Scandinavian journal of immunology. 2007;66:476–483. doi: 10.1111/j.1365-3083.2007.01995.x.  [DOI](https://doi.org/10.1111/j.1365-3083.2007.01995.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17850593/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Scandinavian%20journal%20of%20immunology&title=Efficacy%20of%20anti-CD20%20treatment%20in%20patients%20with%20rheumatoid%20arthritis%20resistant%20to%20a%20combination%20of%20methotrexate/anti-TNF%20therapy&author=M%20Bokarewa&author=C%20Lindholm&author=K%20Zendjanchi&author=M%20Nadali&author=A%20Tarkowski&volume=66&publication_year=2007&pages=476-483&pmid=17850593&doi=10.1111/j.1365-3083.2007.01995.x&)
