# ARG1 Is a Novel Bronchodilator Response Gene: Screening and Replication in Four Asthma Cohorts

## Metadata
**Authors:** Augusto A Litonjua, Jessica Lasky-Su, Kady Schneiter, Kelan G Tantisira, Ross Lazarus, Barbara Klanderman, John J Lima, Charles G Irvin, Stephen P Peters, John P Hanrahan, Stephen B Liggett, Gregory A Hawkins, Deborah A Meyers, Eugene R Bleecker, Christoph Lange, Scott T Weiss
**Journal:** American Journal of Respiratory and Critical Care Medicine
**Date:** 2008 Jul 10
**DOI:** [10.1164/rccm.200709-1363OC](https://doi.org/10.1164/rccm.200709-1363OC)
**PMID:** 18617639
**PMCID:** PMC2556451
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2556451/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC2556451/pdf/AJRCCM1787688.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC2556451/pdf/AJRCCM1787688.pdf)

## Abstract

Rationale: Inhaled β-agonists are one of the most widely used classes of drugs for the treatment of asthma. However, a substantial proportion of patients with asthma do not have a favorable response to these drugs, and identifying genetic determinants of drug response may aid in tailoring treatment for individual patients.

Objectives: To screen variants in candidate genes in the steroid and β-adrenergic pathways for association with response to inhaled β-agonists.

Methods: We genotyped 844 single nucleotide polymorphisms (SNPs) in 111 candidate genes in 209 children and their parents participating in the Childhood Asthma Management Program. We screened the association of these SNPs with acute response to inhaled β-agonists (bronchodilator response [BDR]) using a novel algorithm implemented in a family-based association test that ranked SNPs in order of statistical power. Genes that had SNPs with median power in the highest quartile were then taken for replication analyses in three other asthma cohorts.

Measurements and Main Results: We identified 17 genes from the screening algorithm and genotyped 99 SNPs from these genes in a second population of patients with asthma. We then genotyped 63 SNPs from four genes with significant associations with BDR, for replication in a third and fourth population of patients with asthma. Evidence for association from the four asthma cohorts was combined, and SNPs from ARG1 were significantly associated with BDR. SNP rs2781659 survived Bonferroni correction for multiple testing (combined P value = 0.00048, adjusted P value = 0.047).

Conclusions: These findings identify ARG1 as a novel gene for acute BDR in both children and adults with asthma.

Keywords: pharmacogenetics, asthma, bronchodilator agents

## AT A GLANCE COMMENTARY

## AT A GLANCE COMMENTARY

### Scientific Knowledge on the Subject

Investigations on asthma pharmacogenetics of the β-agonist response to date have mostly studied only one or a few single nucleotide polymorphisms in the β_2_-adrenergic receptor gene (*ADRB2*). Because response to inhaled β-agonists in asthma is a complex phenotype, it is likely that other genes are involved.

### What This Study Adds to the Field

This study identifies the arginase 1 gene (*ARG1*) as a potential β-agonist response gene, using a family-based method to screen variants in genes in the steroid and β-adrenergic pathways.

## METHODS

### Study Populations

We used DNA samples from four clinical trials. All patients or their legal guardians consented to each trial study protocol and ancillary genetic testing. The population we used for the screening algorithm was the Childhood Asthma Management Program (CAMP). Trial design and methodology have been published ([16](#bib16), [17](#bib17)). A total of 209 white probands (randomized to the placebo group) and their parents were included as part of parent–child trios for the screening analyses. Only subjects randomized to the placebo group were used for the screening analyses to avoid confounding effects of medications (corticosteroids and nedocromil) other than inhaled β-agonist.

The population we used for the first replication study (hereafter called the Asthma Trial) was composed of 432 white subjects with asthma ([18](#bib18), [19](#bib19)) who were part of an asthma medication trial conducted by Sepracor, Inc., in the United States. Two completed trials conducted by the American Lung Association Asthma Clinical Research Centers, the Leukotriene Modifier or Corticosteroid or Corticosteroid Salmeterol trial (LOCCS) ([20](#bib20)), and the Effectiveness of Low Dose Theophylline as Add-on Treatment in Asthma (LODO) trial ([21](#bib21)) were used as the second and third replication samples. The 166 white subjects from the LOCCS trial and the 155 white subjects from the LODO trial for whom DNA was available were used for this analysis. Detailed information on these subjects has been previously published and is included in the online supplement.

### Selection of Genes and SNPs: Genotyping

We genotyped 844 SNPs in 111 candidate genes: 42 genes involved in β-adrenergic signaling and regulation; 28 genes involved in innate glucocorticoid synthesis and metabolism, cellular receptors, and transcriptional regulators; and 41 genes from prior asthma association studies that had been previously conducted in the CAMP dataset (Table E1 in the online supplement). Candidate genes in the β-adrenergic and corticosteroid pathways were selected based on prior studies in the literature, their known involvement in metabolic pathways ([22](#bib22), [23](#bib23)), and on expert opinion (S.B.L. and K.G.T.). Corticosteroid pathway candidate genes were included in this analysis because of the known interactions between β_2_-agonists and corticosteroids ([24](#bib24), [25](#bib25)). Finally, we included the candidate genes that our group had previously genotyped and studied in CAMP, because these were already available and so as to appropriately adjust our current analyses for all prior tests conducted with these genes. SNPs were primarily selected using public databases, although resequencing of several core genes was performed. We oversampled exonic and promoter regions and attempted coverage of at least one SNP every 10 kb. We emphasized golden-gate validated and linkage disequilibrium tag SNPs, where available.

SNPs were genotyped via an Illumina BeadStation 500G (Illumima, Inc., San Diego, CA) and via a Sequenom MassArray matrix-assisted laser desorption/ionization time-of-flight mass spectrometer (Sequenom, San Diego, CA). Further details are included in the online supplement. SNPs were also checked for mendelian inconsistencies and for Hardy-Weinberg equilibrium.

### Statistical Methodology

The primary outcome measure of the association analyses was acute response to inhaled bronchodilator (BDR), and calculated as the percentage difference between the pre- and post-bronchodilator FEV_1_ value (BDR = 100 × [post-FEV_1_ − pre-FEV_1_]/pre-FEV_1_). In all analyses, both screening and replication, BDR was treated as a continuous variable. We initially screened the genotypic association with BDR in CAMP using a modified version of the screening algorithm as detailed by Van Steen and associates ([11](#bib11)). Further details are included in the online supplement. The rationale for using CAMP data as the screening set is that the screening methodology was designed for family data. No screening methodology has yet been published for population-based data. We used the 11 repeated measures of BDR over the 4 years of the trial in the placebo group using the FBAT-PC statistic ([26](#bib26)) to maximize the heritability of a given marker and thereby maximize power for the screening stage. Only additive genetic models were evaluated and all analyses in the screening stage were adjusted for age, sex, height, and baseline FEV_1_. We selected the most powerful candidate genes by first ranking the individual SNPs based on conditional power and then evaluating the median rank of all of the SNPs within a given candidate gene. We selected genes whose median SNP ranks for power were within the top 25% of all SNPs genotyped to be taken forward for genotyping in the Asthma Trial population. For those selected SNPs, we evaluated the FBAT-PC statistics for directionality of the association and *P* value for each SNP, and this allowed us to conduct one-sided tests in the replication analyses. We selected 99 SNPs from 17 genes for replication in the Asthma Trial population.

The analyses of the replicate populations were performed using generalized linear models as incorporated into PROC GLM of the SAS statistical analysis software (version 9.0; SAS Institute, Cary, NC), and SNP genotypes were coded for additive models. Only BDR calculated from spirometric measurements at all baseline visits (before randomization) for all the trials were used. All analyses in the replication populations were adjusted for age, sex, height, and baseline FEV_1_. Genes with at least one SNP that was at least marginally associated with BDR (one-sided *P* value < 0.05) were then genotyped in the final two replicate populations. For each SNP in which the direction of the association was in the same direction in each of the four populations, we then combined the *P* values (two-sided) from the original family-based analysis and the one-sided *P* values from the replication cohorts using Fisher's method ([27](#bib27)) to increase statistical efficiency ([28](#bib28)). No evidence for population stratification was found in any of the three populations. Further details on analytic issues are included in the online supplement.

## RESULTS

The baseline characteristics of participants of the four asthma cohorts are shown in [Table 1](#tbl1). The CAMP subjects on whom we performed the initial screen were composed of children, whereas the three replication cohorts were primarily adults with asthma. In the CAMP subjects, we screened 844 SNPs from 111 candidate genes (Table E1 and [Figure 1](#fig1)). In the screening analysis, we ranked the individual SNPs based on the highest to lowest power estimates from the FBAT screening analysis. From these, we identified 19 genes whose median SNP ranks for power were within the top 25% of all SNPs genotyped (Table E1). Because the family-based analysis suggested directionality of the association, we conducted one-sided tests in the replication datasets. The first replication analysis was conducted on 432 white adults with asthma who had participated in a clinical trial of an asthma medication (Asthma Trial). We successfully genotyped 99 SNPs (11.7% of all the SNPs that were screened) from 17 genes in the Asthma Trial. In this first replication analysis, nine genes contained at least one SNP that was at least marginally (one-sided *P* ≤ 0.05) associated with BDR. We then genotyped 63 SNPs from these nine genes and tested them in the final two asthma clinical trial populations from the American Lung Association Asthma Clinical Research Network: the LOCCS trial and the LODO trial.

### TABLE 1.

|   | CAMP | Asthma Trial | LOCCS | LODO |
| --- | --- | --- | --- | --- |
|   | (n = 209) | (n = 432) | (n = 166) | (n = 155) |
| Age, mean yr (SD) | 8.8 (2.1) | 32.5 (13.7) | 34.5 (15.3) | 42.9 (14.7) |
| Range | 5.2–13.2 | 12–80 | 7–71 | 15–76 |
| Sex, n (%) |   |   |   |   |
| Male | 125 (59.8) | 216 (50.0) | 58 (34.9) | 39 (25.2) |
| Female | 84 (40.2) | 216 (50.0) | 108 (65.1) | 116 (74.8) |
| FEV1, mean %predicted (SD) | 95.0 (13.1) | 61.4 (6.9) | 90.8 (9.7) | 78.2 (16.5) |
| BDR, mean % (SD) | 10.4 (9.4) | 39.9 (20.4) | 6.4 (6.1) | 9.7 (11.1) |

Table 1 Caption: BASELINE CHARACTERISTICS OF PARTICIPANTS IN THE FOUR ASTHMA TRIALS

### Figure 1.

![Figure 1.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/bce6/2556451/e432518d5553/AJRCCM1787688f1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=2556451_AJRCCM1787688f1.jpg)

Overall strategy of screening and replication. ACRC = Asthma Clinical Research Centers; BDR = bronchodilator response; CAMP = Childhood Asthma Management Program; LOCCS = Leukotriene Modifier or Corticosteroid or Corticosteroid Salmeterol; LODO = Effectiveness of Low Dose Theophylline as Add-on Treatment in Asthma.

[Table 2](#tbl2) summarizes the results of the replication analyses. Several SNPs were individually associated with BDR in the four populations. The *P* values from each of these populations were combined, and four SNPs from *ARG1* (rs2781659, rs2781663, rs2781665, and rs2749935) showed the strongest evidence for association with BDR. After applying Bonferroni correction for the 99 tests in the initial replication analysis, SNP rs22781659 remained significantly associated with BDR (combined *P* value = 0.00048; Bonferroni-corrected *P* value = 0.047). Evidence for association of SNPs rs2781663 and rs2781665 was borderline significant after adjustment for multiple testing (Bonferroni-corrected *P* values of 0.075 and 0.085, respectively).

### TABLE 2.

| Gene Name | rs Number | CAMP P Value | Asthma Trial P Value | LODO P Value | LOCCS P Value | Combined P Value |
| --- | --- | --- | --- | --- | --- | --- |
| ARG1 | rs2781659 | 0.310 | 0.030 | 0.029 | 0.003 | 0.00048 |
| ARG1 | rs2781663 | 0.428 | 0.022 | 0.060 | 0.003 | 0.00076 |
| ARG1 | rs2781665 | 0.559 | 0.031 | 0.036 | 0.003 | 0.00086 |
| ARG1 | rs2749935 | 0.918 | 0.072 | 0.011 | 0.029 | 0.00596 |
| CRHR2 | rs1003929 | 0.159 | 0.003 | 0.484 | 0.222 | 0.01244 |
| CRHR2 | rs2190242 | 0.022 | 0.071 | 0.176 | 0.281 | 0.01509 |
| CPM | rs1144961 | 0.401 | 0.265 | 0.140 | 0.009 | 0.02189 |
| CRHR2 | rs2240403 | 0.053 | 0.270 | 0.040 | 0.342 | 0.02917 |
| CRHR2 | rs2284220 | 0.209 | 0.016 | 0.263 | 0.315 | 0.03747 |
| CRHR2 | rs917195 | 0.097 | 0.106 | 0.107 | 0.270 | 0.03906 |
| CRHR2 | rs2284217 | 0.472 | 0.068 | 0.154 | 0.084 | 0.04869 |
| CREBL2 | rs4555 | 0.860 | 0.077 | 0.280 | 0.023 | 0.05003 |
| CREM | rs10827492 | 0.827 | 0.009 | 0.465 | 0.136 | 0.05331 |
| CREM | rs4934736 | 0.884 | 0.009 | 0.260 | 0.242 | 0.05694 |
| CREM | rs1148247 | 0.610 | 0.016 | 0.397 | 0.140 | 0.05758 |
| CRHR2 | rs929377 | 0.652 | 0.180 | 0.076 | 0.066 | 0.06155 |
| CRHR2 | rs7793837 | 0.873 | 0.096 | 0.015 | 0.480 | 0.06336 |
| CRHR2 | rs2267716 | 0.342 | 0.366 | 0.026 | 0.244 | 0.07444 |
| CREM | rs7077242 | 0.835 | 0.024 | 0.232 | 0.174 | 0.07590 |
| CREM | rs10827493 | 0.956 | 0.010 | 0.322 | 0.273 | 0.07867 |

Table 2 Caption: SUMMARY OF RESULTS OF TESTING AND REPLICATION IN THE FOUR ASTHMA CLINICAL TRIALS*

We examined the effect of each of these three SNPs on the magnitude of BDR in each of the populations ([Table 3](#tbl3)). In each case, the presence of the minor allele was associated with lower adjusted BDR compared with the homozygous major allele, consistent with the initial FBAT-PC results. [Figure 2](#fig2) compares the LD patterns of the five ARG1 SNPs in the four asthma populations by plotting pairwise *r*^2^ in physical order. The pairwise *r*^2^ values are similar in each of the populations. The three SNPs that were associated with BDR (rs2781659, rs2781663, and rs2781665) were in strong LD with each other (*r*^2^ values ranging from 95 to 100%, depending on the population). For example, in the CAMP population, *r*^2^ between SNPs rs2781659 and rs2781663 was 100%, whereas the *r*^2^ between SNPs rs2781659 and rs2781665 was 99%. In contrast, the other SNP that was more weakly associated with BDR, rs2749935, was not as tightly linked with the other three SNPs (*r*^2^ was 55% with rs2781659, rs2781663, and rs2781665).

### TABLE 3.

|   | Asthma Trial |  |  | LOCCS |  |  | LODO |  |  |  |  |  |  |  |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Genotype | n (%) | BDR | SE | n (%) | BDR | SE | n (%) | BDR | SE |  |  |  |  |  |  |  |  |
|   | ARG1_RS2781665 |  |  |  |  |  |  |  |  |   |   |   |   |   |   |   |   |
| AA | 189 (46.55) | 41.08 | 1.33 | 71 (44.38) | 7.53 | 0.69 | 70 (47.30) | 11.53 | 1.17 |  |  |  |  |  |  |  |  |
| AT | 169 (41.63) | 39.44 | 1.41 | 68 (42.50) | 6.42 | 0.72 | 62 (41.89) | 7.95 | 1.25 |  |  |  |  |  |  |  |  |
| TT | 48 (11.82) | 35.33 | 2.65 | 21 (13.13) | 3.14 | 1.29 | 16 (10.81) | 8.59 | 2.44 |  |  |  |  |  |  |  |  |
|   | ARG1_RS2781663 |  |  |  |  |  |  |  |  |   |   |   |   |   |   |   |   |
| TT | 190 (44.92) | 41.71 | 1.32 | 71 (44.38) | 7.43 | 0.68 | 67 (46.85) | 10.78 | 1.12 |  |  |  |  |  |  |  |  |
| AT | 184 (43.50) | 39.14 | 1.34 | 67 (41.88) | 6.68 | 0.70 | 61 (42.66) | 7.85 | 1.18 |  |  |  |  |  |  |  |  |
| AA | 49 (11.58) | 36.28 | 2.62 | 22 (13.75) | 2.99 | 1.22 | 15 (10.49) | 8.40 | 2.38 |  |  |  |  |  |  |  |  |
|   | ARG1_RS2781659 |  |  |  |  |  |  |  |  |   |   |   |   |   |   |   |   |
| AA | 193 (45.43) | 41.39 | 1.31 | 74 (47.13) | 7.35 | 0.69 | 71 (47.65) | 11.53 | 1.15 |  |  |  |  |  |  |  |  |
| AG | 183 (42.86) | 39.50 | 1.35 | 62 (33.49) | 6.39 | 0.76 | 61 (40.94) | 7.51 | 1.24 |  |  |  |  |  |  |  |  |
| GG | 50 (11.71) | 35.93 | 2.59 | 21 (13.38) | 2.90 | 1.30 | 17 (11.41) | 8.73 | 2.35 |  |  |  |  |  |  |  |  |

Table 3 Caption: EFFECTS OF ARG1 SINGLE NUCLEOTIDE POLYMORPHISMSS ON BRONCHODILATOR RESPONSE*

### Figure 2.

![Figure 2.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/bce6/2556451/33cfe1a99e0e/AJRCCM1787688f2.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=2556451_AJRCCM1787688f2.jpg)

Linkage disequilibrium patterns of ARG1 single nucleotide polymorphisms (SNPs) in the four asthma cohorts. Numbers in the individual blocks represent r2 values for each pair of SNPs (blank values = 100%), with the colors corresponding to D′ values. Plots were created using the program Haploview (http://www.broad.mit.edu/mpg/haploview/).

## DISCUSSION

Prior investigations into the pharmacogenetics of asthma have generally been limited to one or a few SNPs from one gene. We investigated 844 SNPs from 111 candidate genes selected from asthma β-agonist and corticosteroid pathways, and from our prior candidate gene studies, and screened these SNPs for association with BDR using a family-based screening algorithm that allowed us to rank the SNPs based on estimated power for replication. We then genotyped 99 SNPs from 17 genes in a population-based cohort of patients with asthma, who participated in an asthma clinical trial. Finally, we genotyped 83 SNPs in seven genes in two separate cohorts of subjects with asthma. We found SNPs in the *ARG1* gene to be associated with BDR in these three asthma populations, after adjusting for multiple comparisons.

*ARG1* has recently been implicated in asthma. Zimmermann and colleagues ([29](#bib29)) reported increased expression of *ARG1* and *ARG2* in murine lung, and also found increased arginase 1 protein expression from human asthma bronchoalveolar lavage cells. Variants in *ARG1* were associated with atopy in a cohort of Mexicans with asthma ([30](#bib30)). *ARG1* maps to chromosome 6q23 and encodes one isoform of the enzyme arginase, which metabolizes l-arginine. l-Arginine homeostasis is involved in the regulation of airway function, because the availability of this amino acid to nitric oxide synthase (NOS) determines the production of the endogenous bronchodilator nitric oxide (NO) ([31](#bib31)). Changes in l-arginine homeostasis may contribute to many of the features of asthma, such as airway hyperresponsiveness, airway inflammation, and airway remodeling ([32](#bib32)). Intracellular l-arginine levels are regulated by at least three distinct mechanisms (reviewed by Maarsingh and colleagues [32]): (*1*) cellular uptake by cationic amino acid transporters, (*2*) recycling from l-citrulline, and (*3*) metabolism by NOS and arginase. Arginase is postulated to be involved in asthma by depleting stores of l-arginine, an NOS substrate, which leads to decreased production of NO, a potent bronchial smooth muscle relaxer ([33](#bib33), [34](#bib34)), and it has been shown to inhibit airway smooth muscle relaxation ([35](#bib35), [36](#bib36)). Finally, RNA interference of arginase 1 in the lungs resulted in complete loss of airway hyperresponsiveness to methacholine due to IL-13 treatment ([37](#bib37)). This correlated with arginase 1 expression, which suggests that the polymorphisms involved with the current findings in human asthma may cause a loss of expression or function of arginase 1.

We used a gene-based strategy to select SNPs to take forward for replication. In this method, after ranking SNPs from 1 (most power) to 844 (least power), we grouped all SNPs for each gene and calculated the median SNP rank for that gene. Thus, whereas some genes had one or two SNPs that were assigned high ranks, these genes may not be taken forward because the median SNP rank did not meet the predetermined cutoff. We adopted this strategy because we were not sure that LD patterns across the four asthma populations would be similar. It is interesting to note that *ADRB2*, a gene that has been widely studied in asthma pharmacogenetics ([38](#bib38)), was not one of the genes that was selected using this strategy, despite including 18 SNPs from this gene in the screening analysis. It is possible that there was insufficient power in the screening stage, because we only analyzed the 209 trios in the placebo group in CAMP. However, it should be noted that a prior analysis using all 400 trios also did not find an association with any of the *ADRB2* SNPs and BDR ([39](#bib39)). Furthermore, the phenotype that we investigated is different from that reported in other studies reporting on the pharmacogenetic effect of *ADRB2* ([9](#bib9), [10](#bib10)). We are currently performing additional genotyping and analyses using an SNP-based strategy for replication, rather than the gene-based strategy that we used here, to see if we identify important SNPs in this gene and others for association with BDR.

Our analysis used the phenotype of acute response to a short-acting β_2_-agonist, albuterol, in part because this was the phenotype that was common to all asthma cohorts. In the screening algorithm, we used the information from repeated measures of BDR among the 209 white children randomized to the placebo group in the CAMP study over the 4 years of the trial. This was done to increase the power for the screening method. In contrast, for the replication cohorts, we only used the information on BDR response on entry into the respective studies, to standardize the phenotype. Thus, our results may not be applicable to patients with asthma who are on regular β_2_-agonist treatment (either short- or long-acting). We also did not address interactions with any other class of asthma medication, because baseline medication was different for all the populations: BDR was performed in both CAMP and the Adult Trial populations after several weeks of being off all asthma medications; LOCCS subjects were receiving inhaled corticosteroids for 4–6 weeks before BDR testing; and drug regimens for LODO subjects were not changed before entry into the trial.

We used a novel method of screening a large number of SNPs for association analysis ([11](#bib11)). This method has been successfully used to identify disease-susceptibility genes ([12](#bib12)–[14](#bib14)). Because this method has only been developed for family-based studies and not population-based studies, we used the CAMP population for screening the original 844 SNPs. The traditional method would have been to analyze all the SNPs in one population, determine which SNPs were associated with BDR at a predetermined level of significance, then test these SNPs in the replication populations. However, if we had used this usual method for gene finding, we would then have had to adjust our overall results for the 844 SNPs that were originally tested, and it is likely that no finding would have survived this adjustment for multiple testing, even if the association was real. In our method, because we screened on power and not *P* value, we only needed to adjust for the 99 tests in the first replication step. Thus, this screening method allows the use of modest-sized populations for gene discovery because it limits the number of tests that are actually being performed.

The population to which we applied our screening algorithm was a cohort of childhood asthmatics, whereas the three asthma replication cohorts were composed predominantly of adults with asthma. As we stated previously, the rationale for this is that the screening method was developed for the setting of family-based studies and not for population-based studies. There is no similar screening method that has yet been developed for population-based studies. Because our replication populations were of small to modest sizes, we applied the screening method as a means of minimizing the number of tests. Although there were only 209 parent–child trios included in the screening analysis, we maximized the power in the screening stage by using the 11 repeated measures of BDR over the 4 years of the trial. In addition, there were differences in the asthma severity and in the magnitude of the BDR between the populations as shown in [Table 1](#tbl1). Despite these differences, we were able to detect associations between SNPs in *ARG1* and BDR in each of the three replication populations. Although the association between these SNPs and BDR in CAMP was not statistically significant, the effect sizes of each SNP were of sufficient magnitude for them to be selected based on power in the screening analysis. We can only surmise at this point that there may be age-related effects associated with the SNPs in this gene. We believe, therefore, that the results of the association between *ARG1* polymorphisms and BDR are robust and applicable to both children and adults with asthma in a variety of settings.

The three *ARG1* SNPs that were associated with BDR were all in the promoter region of the gene and were in tight LD with each other. Genotyping of all known SNPs in the gene or resequencing of the gene will need to be performed to determine if these three promoter SNPs are in LD with the functional mutation. There is mounting evidence for “cross-talk” between pathways involved with the relaxation and constriction of airway smooth muscle ([40](#bib40), [41](#bib41)). It is thus not entirely unexpected that a gene involved with airway hyperresponsiveness in mouse studies is associated with a bronchodilator response in human asthma. However, arginase 1 has not been previously identified as one of the proteins involved in such cross-talk. This unexpected finding shows the potential value of whole-genome coverage to study drug response to uncover novel genetic determinants. The screening method that we used for this analysis would be easily applicable to the case of whole-genome association.

In summary, we have identified SNPs in *ARG1* as novel BDR determinants. Further studies will need to identify the functional SNP or SNPs in this gene. Other pharmacogenetic studies using long-acting β_2_-agonists, either alone or in conjunction with corticosteroids, and investigation of other phenotypes (e.g., FEV_1_, peak flow) are needed to clarify the effects of variants in this gene. Our analysis shows the utility of a family-based algorithm to effectively screen SNPs for replication in other cohorts. This method is easily applicable to the case of whole-genome association.

## Supplementary Material

## Acknowledgments

The authors thank all families for their enthusiastic participation in the CAMP Genetics Ancillary Study. They thank the CAMP investigators and research team, for collection of CAMP Genetic Ancillary Study data. The authors also acknowledge the American Lung Association (ALA) and the ALA's Asthma Clinical Research Centers investigators and research teams for use of LOCCS and LoDo data, and Nemours Children's Clinic. The authors also acknowledge Sepracor, Inc., for use of the Asthma Trial data.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Masoli M, Fabian D, Holt S, Beasley R; Global Initiative for Asthma (GINA) Program. Global burden of asthma: executive summary of the GINA Dissemination Committee report. Allergy 2004;469–478.  [DOI](https://doi.org/10.1111/j.1398-9995.2004.00526.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15080825/)

2. Mannino DM, Homa DM, Akinbami LJ, Moorman JE, Gwynn C, Redd SC. Surveillance for asthma—United States, 1980–1999. MMWR Surveill Summ 2002;51:1–13.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/12420904/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=MMWR%20Surveill%20Summ&volume=51&publication_year=2002&pages=1&pmid=12420904&)

3. Gergen PJ, Mullally DI, Evans R III. National survey of prevalence of asthma among children in the United States, 1976. to 1980. Pediatrics 1988;81:1–7.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/3336575/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pediatrics&volume=81&publication_year=1976&pages=1&pmid=3336575&)

4. Smith DH, Malone DC, Lawson KA, Okamoto LJ, Battista C, Saunders WB. A national estimate of the economic costs of asthma. Am J Respir Crit Care Med 1997;156:787–793.  [DOI](https://doi.org/10.1164/ajrccm.156.3.9611072) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9309994/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Respir%20Crit%20Care%20Med&volume=156&publication_year=1997&pages=787&pmid=9309994&doi=10.1164/ajrccm.156.3.9611072&)

5. Sears MR, Lotvall J. Past, present and future: beta2-adrenoceptor agonists in asthma management. Respir Med 2005;99:152–170.  [DOI](https://doi.org/10.1016/j.rmed.2004.07.003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15715182/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Respir%20Med&volume=99&publication_year=2005&pages=152&pmid=15715182&doi=10.1016/j.rmed.2004.07.003&)

6. Drazen JM, Silverman EK, Lee TH. Heterogeneity of therapeutic responses in asthma. Br Med Bull 2000;56:1054–1070.  [DOI](https://doi.org/10.1258/0007142001903535) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11359637/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20Med%20Bull&volume=56&publication_year=2000&pages=1054&pmid=11359637&doi=10.1258/0007142001903535&)

7. Weiss ST, Litonjua AA, Lange C, Lazarus R, Liggett SB, Bleecker ER, Tantisira KG. Overview of the pharmacogenetics of asthma treatment. Pharmacogenomics J 2006;6:311–326.  [DOI](https://doi.org/10.1038/sj.tpj.6500387) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16568148/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&volume=6&publication_year=2006&pages=311&pmid=16568148&doi=10.1038/sj.tpj.6500387&)

8. Litonjua AA. The significance of beta2-adrenergic receptor polymorphisms in asthma. Curr Opin Pulm Med 2006;12:12–17.  [DOI](https://doi.org/10.1097/01.mcp.0000198068.50457.95) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16357573/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Opin%20Pulm%20Med&volume=12&publication_year=2006&pages=12&pmid=16357573&doi=10.1097/01.mcp.0000198068.50457.95&)

9. Israel E, Chinchilli VM, Ford JG, Boushey HA, Cherniack R, Craig TJ, Deykin A, Fagan JK, Fahy JV, Fish J, et al. Use of regularly scheduled albuterol treatment in asthma: genotype-stratified, randomised, placebo-controlled cross-over trial. Lancet 2004;364:1505–1512.  [DOI](https://doi.org/10.1016/S0140-6736(04)17273-5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15500895/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&volume=364&publication_year=2004&pages=1505&pmid=15500895&doi=10.1016/S0140-6736(04)17273-5&)

10. Israel E, Drazen JM, Liggett SB, Boushey HA, Cherniack RM, Chinchilli VM, Cooper DM, Fahy JV, Fish JE, Ford JG, et al. The effect of polymorphisms of the beta(2)-adrenergic receptor on the response to regular use of albuterol in asthma. Am J Respir Crit Care Med 2000;162:75–80.  [DOI](https://doi.org/10.1164/ajrccm.162.1.9907092) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10903223/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Respir%20Crit%20Care%20Med&volume=162&publication_year=2000&pages=75&pmid=10903223&doi=10.1164/ajrccm.162.1.9907092&)

11. Van Steen K, McQueen MB, Herbert A, Raby B, Lyon H, Demeo DL, Murphy A, Su J, Datta S, Rosenow C, et al. Genomic screening and replication using the same data set in family-based association testing. Nat Genet 2005;37:683–691.  [DOI](https://doi.org/10.1038/ng1582) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15937480/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&volume=37&publication_year=2005&pages=683&pmid=15937480&doi=10.1038/ng1582&)

12. Herbert A, Gerry NP, McQueen MB, Heid IM, Pfeufer A, Illig T, Wichmann HE, Meitinger T, Hunter D, Hu FB, et al. A common genetic variant is associated with adult and childhood obesity. Science 2006;312:279–283.  [DOI](https://doi.org/10.1126/science.1124779) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16614226/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&volume=312&publication_year=2006&pages=279&pmid=16614226&doi=10.1126/science.1124779&)

13. Lyon HN, Emilsson V, Hinney A, Heid IM, Lasky-Su J, Zhu X, Thorleifsson G, Gunnarsdottir S, Walters GB, Thorsteinsdottir U, et al. The association of a SNP upstream of INSIG2 with body mass index is reproduced in several but not all cohorts. PLoS Genet 2007;3:e61.  [DOI](https://doi.org/10.1371/journal.pgen.0030061) | [PMC free article](/articles/PMC1857727/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17465681/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet&volume=3&publication_year=2007&pages=e61&pmid=17465681&doi=10.1371/journal.pgen.0030061&)

14. Lasky-Su J, Lyon HN, Emilsson V, Heid IM, Molony C, Raby BA, Lazarus R, Klanderman B, Soto-Quiros ME, Avila L, et al. On the replication of genetic associations: timing can be everything! Am J Hum Genet 2008;82:849–858.  [DOI](https://doi.org/10.1016/j.ajhg.2008.01.018) | [PMC free article](/articles/PMC2427263/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18387595/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&volume=82&publication_year=2008&pages=849&pmid=18387595&doi=10.1016/j.ajhg.2008.01.018&)

15. Litonjua AA, Tantisira KG, Su JA, Lazarus R, Klanderman B, Lange C, Weiss ST. Polymorphisms in ARG1 are determinants of bronchodilator response: screening and replication in 4 asthma cohorts [abstract]. Am J Respir Crit Care Med 2007;175:A968.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Respir%20Crit%20Care%20Med&volume=175&publication_year=2007&pages=A968&)

16. Childhood Asthma Management Program Research Group. The Childhood Asthma Management Program (CAMP): design, rationale, and methods. Control Clin Trials 1999;20:91–120.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/10027502/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Control%20Clin%20Trials&volume=20&publication_year=1999&pages=91&pmid=10027502&)

17. Childhood Asthma Management Program Research Group. Long-term effects of budesonide or nedocromil in children with asthma. N Engl J Med 2000;343:1054–1063.  [DOI](https://doi.org/10.1056/NEJM200010123431501) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11027739/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&volume=343&publication_year=2000&pages=1054&pmid=11027739&doi=10.1056/NEJM200010123431501&)

18. Baron RM, Palmer LJ, Tantisira K, Gabriel S, Sonna LA, Le L, Hallock A, Libermann TA, Drazen JM, Weiss ST, et al. DNA sequence variants in epithelium-specific ETS-2 and ETS-3 are not associated with asthma. Am J Respir Crit Care Med 2002;166:927–932.  [DOI](https://doi.org/10.1164/rccm.200201-048OC) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12359648/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Respir%20Crit%20Care%20Med&volume=166&publication_year=2002&pages=927&pmid=12359648&doi=10.1164/rccm.200201-048OC&)

19. Silverman ES, Palmer LJ, Subramaniam V, Hallock A, Mathew S, Vallone J, Faffe DS, Shikanai T, Raby BA, Weiss ST, et al. Transforming growth factor-β1 promoter polymorphism C-509T is associated with asthma. Am J Respir Crit Care Med 2004;169:214–219.  [DOI](https://doi.org/10.1164/rccm.200307-973OC) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14597484/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Respir%20Crit%20Care%20Med&volume=169&publication_year=2004&pages=214&pmid=14597484&doi=10.1164/rccm.200307-973OC&)

20. Peters SP, Anthonisen N, Castro M, Holbrook JT, Irvin CG, Smith LJ, Wise RA. Randomized comparison of strategies for reducing treatment in mild persistent asthma. N Engl J Med 2007;356:2027–2039.  [DOI](https://doi.org/10.1056/NEJMoa070013) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17507702/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&volume=356&publication_year=2007&pages=2027&pmid=17507702&doi=10.1056/NEJMoa070013&)

21. American Lung Association Asthma Clinical Research Centers. Clinical trial of low-dose theophylline and montelukast in patients with poorly controlled asthma. Am J Respir Crit Care Med 2007;175:235–242.  [DOI](https://doi.org/10.1164/rccm.200603-416OC) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16998094/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Respir%20Crit%20Care%20Med&volume=175&publication_year=2007&pages=235&pmid=16998094&doi=10.1164/rccm.200603-416OC&)

22. Litonjua A, Thorn CF, Liggett SB. β-Agonist and β-blocker pathway [diagram on the Internet]. Palo Alto (CA): PharmGKB, Stanford University; c2008 [updated 2007 Aug 1; accessed 2008 May 11]. Available from: http://www.pharmgkb.org/do/serve?objId=PA2024  [http://www.pharmgkb.org/do/serve?objId=PA2024](http://www.pharmgkb.org/do/serve?objId=PA2024)

23. Weiss S, Litonjua A, Tantisira K, Wong M-L, Thorn C, Licinio J. Glucocorticoid and inflammatory genes pathway [diagram on the Internet]. Palo Alto (CA): PharmGKB, Stanford University; c2008 [updated 2007 Aug 1; accessed 2008 May 11]. Available from: http://www.pharmgkb.org/search/pathway/glucocorticoid/hpa-axis.jsp#  [http://www.pharmgkb.org/search/pathway/glucocorticoid/hpa-axis.jsp#](http://www.pharmgkb.org/search/pathway/glucocorticoid/hpa-axis.jsp#)

24. Barnes PJ. Scientific rationale for using a single inhaler for asthma control. Eur Respir J 2007;29:587–595.  [DOI](https://doi.org/10.1183/09031936.00080306) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17329493/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20Respir%20J&volume=29&publication_year=2007&pages=587&pmid=17329493&doi=10.1183/09031936.00080306&)

25. Johnson M. Interactions between corticosteroids and β2-agonists in asthma and chronic obstructive pulmonary disease. Proc Am Thorac Soc 2004;1:200–206.  [DOI](https://doi.org/10.1513/pats.200402-010MS) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16113435/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Am%20Thorac%20Soc&volume=1&publication_year=2004&pages=200&pmid=16113435&doi=10.1513/pats.200402-010MS&)

26. Lange C, van Steen K, Andrew T, Lyon H, DeMeo DL, Raby B, Murphy A, Silverman EK, MacGregor A, Weiss ST, et al. A family-based association test for repeatedly measured quantitative traits adjusting for unknown environmental and/or polygenic effects. Stat Appl Genet Mol Biol [serial on the Internet]. 2004 [accessed 2008 May 11];3:Article 17. Available from: http://www.bepress.com/sagmb/vol3/iss1/art17  [http://www.bepress.com/sagmb/vol3/iss1/art17](http://www.bepress.com/sagmb/vol3/iss1/art17) | [DOI](https://doi.org/10.2202/1544-6115.1067) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16646795/)

27. Fisher RA. Statistical methods for research workers. New York: Hafner; 1950.

28. Skol AD, Scott LJ, Abecasis GR, Boehnke M. Joint analysis is more efficient than replication-based analysis for two-stage genome-wide association studies. Nat Genet 2006;38:209–213.  [DOI](https://doi.org/10.1038/ng1706) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16415888/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&volume=38&publication_year=2006&pages=209&pmid=16415888&doi=10.1038/ng1706&)

29. Zimmermann N, King NE, Laporte J, Yang M, Mishra A, Pope SM, Muntel EE, Witte DP, Pegg AA, Foster PS, et al. Dissection of experimental asthma with DNA microarray analysis identifies arginase in asthma pathogenesis. J Clin Invest 2003;111:1863–1874.  [DOI](https://doi.org/10.1172/JCI17912) | [PMC free article](/articles/PMC161427/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12813022/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Invest&volume=111&publication_year=2003&pages=1863&pmid=12813022&doi=10.1172/JCI17912&)

30. Li H, Romieu I, Sienra-Monge JJ, Ramirez-Aguilar M, Estela Del Rio-Navarro B, Kistner EO, Gjessing HK, Lara-Sanchez Idel C, Chiu GY, London SJ. Genetic polymorphisms in arginase i and ii and childhood asthma and atopy. J Allergy Clin Immunol 2006;117:119–126.  [DOI](https://doi.org/10.1016/j.jaci.2005.09.026) | [PMC free article](/articles/PMC1450009/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16387594/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Allergy%20Clin%20Immunol&volume=117&publication_year=2006&pages=119&pmid=16387594&doi=10.1016/j.jaci.2005.09.026&)

31. Ricciardolo FL, Sterk PJ, Gaston B, Folkerts G. Nitric oxide in health and disease of the respiratory system. Physiol Rev 2004;84:731–765.  [DOI](https://doi.org/10.1152/physrev.00034.2003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15269335/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Physiol%20Rev&volume=84&publication_year=2004&pages=731&pmid=15269335&doi=10.1152/physrev.00034.2003&)

32. Maarsingh H, Zaagsma J, Meurs H. Arginine homeostasis in allergic asthma. Eur J Pharmacol 2008;585:375–384.  [DOI](https://doi.org/10.1016/j.ejphar.2008.02.096) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18410920/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Pharmacol&volume=585&publication_year=2008&pages=375&pmid=18410920&doi=10.1016/j.ejphar.2008.02.096&)

33. Zimmermann N, Rothenberg ME. The arginine-arginase balance in asthma and lung inflammation. Eur J Pharmacol 2006;533:253–262.  [DOI](https://doi.org/10.1016/j.ejphar.2005.12.047) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16458291/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Pharmacol&volume=533&publication_year=2006&pages=253&pmid=16458291&doi=10.1016/j.ejphar.2005.12.047&)

34. Meurs H, Maarsingh H, Zaagsma J. Arginase and asthma: novel insights into nitric oxide homeostasis and airway hyperresponsiveness. Trends Pharmacol Sci 2003;24:450–455.  [DOI](https://doi.org/10.1016/S0165-6147(03)00227-X) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12967769/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Trends%20Pharmacol%20Sci&volume=24&publication_year=2003&pages=450&pmid=12967769&doi=10.1016/S0165-6147(03)00227-X&)

35. Maarsingh H, Leusink J, Bos IS, Zaagsma J, Meurs H. Arginase strongly impairs neuronal nitric oxide-mediated airway smooth muscle relaxation in allergic asthma. Respir Res 2006;7:6.  [DOI](https://doi.org/10.1186/1465-9921-7-6) | [PMC free article](/articles/PMC1363345/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16409620/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Respir%20Res&volume=7&publication_year=2006&pages=6&pmid=16409620&doi=10.1186/1465-9921-7-6&)

36. Maarsingh H, Tio MA, Zaagsma J, Meurs H. Arginase attenuates inhibitory nonadrenergic noncholinergic nerve-induced nitric oxide generation and airway smooth muscle relaxation. Respir Res 2005;6:23.  [DOI](https://doi.org/10.1186/1465-9921-6-23) | [PMC free article](/articles/PMC555585/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15748286/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Respir%20Res&volume=6&publication_year=2005&pages=23&pmid=15748286&doi=10.1186/1465-9921-6-23&)

37. Yang M, Rangasamy D, Matthaei KI, Frew AJ, Zimmmermann N, Mahalingam S, Webb DC, Tremethick DJ, Thompson PJ, Hogan SP, et al. Inhibition of arginase I activity by RNA interference attenuates IL-13-induced airways hyperresponsiveness. J Immunol 2006;177:5595–5603.  [DOI](https://doi.org/10.4049/jimmunol.177.8.5595) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17015747/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Immunol&volume=177&publication_year=2006&pages=5595&pmid=17015747&doi=10.4049/jimmunol.177.8.5595&)

38. Liggett SB, Hall IP. Beta2-adrenergic receptor polymorphisms and asthmatic phenotypes. In: Postma DS, Weiss ST, editors. Genetics of asthma and chronic obstructive pulmonary disease. New York: Informa Healthcare USA; 2007. pp. 299–316.

39. Silverman EK, Kwiatkowski DJ, Sylvia JS, Lazarus R, Drazen JM, Lange C, Laird NM, Weiss ST. Family-based association analysis of beta2-adrenergic receptor polymorphisms in the childhood asthma management program. J Allergy Clin Immunol 2003;112:870–876.  [DOI](https://doi.org/10.1016/s0091-6749(03)02023-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14610472/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Allergy%20Clin%20Immunol&volume=112&publication_year=2003&pages=870&pmid=14610472&doi=10.1016/s0091-6749(03)02023-2&)

40. McGraw DW, Almoosa KF, Paul RJ, Kobilka BK, Liggett SB. Antithetic regulation by beta-adrenergic receptors of GQ receptor signaling via phospholipase C underlies the airway beta-agonist paradox. J Clin Invest 2003;112:619–626.  [DOI](https://doi.org/10.1172/JCI18193) | [PMC free article](/articles/PMC171392/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12925702/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Invest&volume=112&publication_year=2003&pages=619&pmid=12925702&doi=10.1172/JCI18193&)

41. McGraw DW, Elwing JM, Fogel KM, Wang WC, Glinka CB, Mihlbachler KA, Rothenberg ME, Liggett SB. Crosstalk between GI and GQ/GS pathways in airway smooth muscle regulates bronchial contractility and relaxation. J Clin Invest 2007;117:1391–1398.  [DOI](https://doi.org/10.1172/JCI30489) | [PMC free article](/articles/PMC1838924/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17415415/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Invest&volume=117&publication_year=2007&pages=1391&pmid=17415415&doi=10.1172/JCI30489&)
