# Genetic Variants Contributing to Daunorubicin-Induced Cytotoxicity

## Metadata
**Authors:** R Stephanie Huang, Shiwei Duan, Emily O Kistner, Wasim K Bleibel, Shannon M Delaney, Donna L Fackenthal, Soma Das, M Eileen Dolan
**Journal:** Cancer research
**Date:** 2008 May 1
**DOI:** [10.1158/0008-5472.CAN-07-6381](https://doi.org/10.1158/0008-5472.CAN-07-6381)
**PMID:** 18451141
**PMCID:** PMC2714371
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2714371/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC2714371/pdf/nihms101217.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC2714371/pdf/nihms101217.pdf)

## Abstract

Identifying heritable genetic variants responsible for chemo-therapeutic toxicities has been challenging due in part to its multigenic nature. To date, there is a paucity of data on genetic variants associated with patients experiencing severe myelosuppression or cardiac toxicity following treatment with daunorubicin. We present a genome-wide model using International HapMap cell lines that integrate genotype and gene expression to identify genetic variants that contribute to daunorubicin-induced cytotoxicity. A cell growth inhibition assay was used to measure variations in the cytotoxicity of daunorubicin. Gene expression was determined using the Affymetrix GeneChip Human Exon 1.0ST Array. Using sequential analysis, we evaluated the associations between genotype and cytotoxicity, those significant genotypes with gene expression and correlated gene expression of the identified candidates with cytotoxicity. A total of 26, 9, and 18 genetic variants were identified to contribute to daunorubicin-induced cytotoxicity through their effect on 16, 9, and 36 gene expressions in the combined, Centre d’ Etude du Polymorphisme Humain (CEPH), and Yoruban populations, respectively. Using 50 non-HapMap CEPH cell lines, single nucleotide polymorphisms generated through our model predicted 29% of the overall variation in daunorubicin sensitivity and the expression of CYP1B1 was significantly correlated with sensitivity to daunorubicin. In the CEPH validation set, rs120525235 and rs3750518 were significant predictors of transformed daunorubicin IC50 (P = 0.005 and P = 0.0008, respectively), and rs1551315 trends toward significance (P = 0.089). This unbiased method can be used to elucidate genetic variants contributing to a wide range of cellular phenotypes.

## Introduction

Daunorubicin, an anthracycline chemotherapeutic agent, is commonly used to treat leukemia, lymphoma, and advanced HIV-associated Kaposi’s sarcoma ([1](#R1), [2](#R2)). The drug acts through DNA and RNA synthesis inhibition by intercalating with DNA base pairs, stabilizing the double-stranded DNA cleavage normally catalyzed by topoisomerase II and causing inhibition of religation of DNA breaks ([3](#R3), [4](#R4)). Daunorubicin also produces free radicals which have been established as a cause of toxicity ([5](#R5), [6](#R6)). Despite its wide usage, daunorubicin is associated with severe myelosuppression and can cause cardiac toxicity ([7](#R7)–[9](#R9)). The incidence of daunorubicin treatment–induced toxicities are highly variable and are associated with both treatment dosage and patient age ([10](#R10)).

Our laboratory has used EBV-transformed B-lymphoblastoid cell lines (LCL) derived from healthy individuals within 24 large Centre d’ Etude du Polymorphisme Humain (CEPH) pedigrees to show that f29% of human variation in susceptibility to daunorubicin-induced cytotoxicity is due to a genetic component ([11](#R11)). Thus, genetics is likely to play a role in overall human variation in daunorubicin-induced response and toxicity; however, only a few clinical studies have evaluated the predictive value of genetic variants. Wojnowski et al. have found significant association between anthracycline-induced cardiotoxicity and genetic polymorphisms of the NAD(P)H oxidase and efflux transporters (*MRP1* and *MRP2*) in peripheral blood lymphocytes of patients with non–Hodgkins lymphoma ([6](#R6)). In patients with either myeloma or acute myelogenous leukemia who were treated with combination chemotherapy that included daunorubicin, genetic polymorphisms in *GSTP1* and *GSTT1* genes have been shown to associate with clinical outcomes (e.g., progression-free survival or overall survival; refs. [12](#R12), [13](#R13)). Furthermore, it has been shown that African American patients who received doxorubicin, a structural analogue of daunorubicin, developed cardiotoxicity more frequently, indicative of population-specific effects ([14](#R14)). There is a need to comprehensively elucidate the genetic variants important in daunorubicin-induced cytotoxicity within and among populations.

Recently, our group developed a comprehensive, genome-wide approach that integrates genotype, gene expression, and cytotoxicity data to identify potentially functional single nucleotide polymorphisms (SNP) associated with chemotherapy-induced cytotoxicity through their effect on expression ([15](#R15), [16](#R16)). The International HapMap cell lines derived from individuals of African and European descent allowed us to define a set of genetic variants unique to those populations and common among both populations. Although we recognize that there are limitations with the use of cell lines, these studies will likely help direct further clinical studies by providing a strong list of SNPs/candidate genes to evaluate in clinical trials. The long-term goal is to identify genetic polymorphisms that influence chemotherapeutic-induced toxicity in individuals and to identify patients “at risk” for adverse events associated with this agent.

## Materials and Methods

### Cell lines and drugs

EBV-transformed LCLs derived from 30 CEPH trios from Utah residents with ancestry from Northern and Western Europe (HAPMAPPT01, CEU) and 30 trios collected from the Yoruba in Ibadan, Nigeria (HAPMAPPT03, YRI) were purchased from the Coriell Institute for Medical Research.[4](#FN3) The cell lines used for the validation set included GM06986, GM06995, GM07002, GM07014, GM07016, GM07017, GM07049, GM07050, GM07340, GM07341, GM07347, GM10837, GM10840, GM10841, GM10844, GM10845, GM10848, GM10858, GM11843, GM11891, GM11893, GM11894, GM11917, GM11918, GM11919, GM11920, GM11931, GM12045, GM12058, GM12096, GM12099, GM12116, GM12117, GM12718, GM12748, GM12749, GM12889, GM12890, GM13042, GM13044, GM13045, GM13046, GM13047, GM13048, GM13049, GM13050, GM13051, GM13052, GM13060, and GM13133. Detailed cell maintenance was described in our previous publication ([17](#R17)). Daunorubicin (NSC-82151) was kindly provided by the Drug Synthesis and Chemistry Branch, Division of Cancer Treatment, National Cancer Institute (Bethesda, MD).

### Genotype and cytotoxicity association analysis

Cell growth inhibition was tested in 175 LCLs as described previously ([17](#R17)). SNP genotypes were downloaded from the International HapMap database[5](#FN4) (release 21) and filtered. Details for the SNP filtration criteria can be found in our previous publication ([16](#R16)). A total of 387,417 SNPs having a high minor allele frequency and located in or near genes were included in the analysis.

Using Microsoft Excel software, an inverse normalization of the percentile rank function was applied to the 175 combined IC_50_ values and 86 CEU or 89 YRI separate IC_50_ values for different testing populations, respectively. The quantitative transmission disequilibrium test (QTDT) was performed to identify any genotype-cytotoxicity association using QTDT software ([18](#R18)).[6](#FN5) Due to the possible heterogeneity between and within each population, we performed association studies in these two ethnic groups separately using sex as a covariate, and together using sex and race as covariates. *P* ≤ 0.0001 was considered statistically significant. False discovery rate was calculated using the Benjamini-Hochberg correction method ([19](#R19)).

### Genotype and gene expression association analysis

Baseline gene expression was evaluated in 87 CEU and 89 YRI LCLs using Affymetrix GeneChip Human Exon 1.0 ST array (Exon Array) as described previously ([16](#R16)). A second QTDT test which integrated only those SNPs found from the genotype cytotoxicity association was performed with mRNA gene expression to identify possible genotype-expression association ([16](#R16)). A Bonferroni correction (*P* < 0.05 based on the number of transcript clusters) was used to adjust raw *P* values after QTDT analysis.

### Gene expression and daunorubicin IC50 linear regression analysis

To examine the relationship between gene expression and sensitivity to daunorubicin, a general linear model was constructed with daunorubicin IC_50_ (transformed using the inverse normalization of the percentile rank function) as the dependent variable and robust multiarray average–summarized log_2_-transformed gene expression level together with an indicator for gender as the independent variables. Only gene expression significantly associated with SNPs identified in the genotype and gene expression associations described above were included in the analysis. A detailed description of this model can be found in our previous publication ([16](#R16)). *P* ≤ 0.05 was considered statistically significant. The linkage disequilibrium of significant SNPs within each population was evaluated using Haploview version 3.32.[7](#FN6)

### Multivariate model to predict daunorubicin IC50 with genotypes

To examine the overall genetic variant contributions to sensitivity of daunorubicin, additional general linear models were constructed with transformed daunorubicin IC_50_ as the dependent variable. The independent variables included all the significant SNP genotypes (assuming an additive genetic effect) that were selected from the three-way model in the combined populations and the two populations independently. These SNP genotypes were significantly associated with daunorubicin IC_50_ through their effect on gene expression. Details on constructing the multivariate model was described elsewhere ([16](#R16)).

### Genotype analysis on validation sample set

A set of 50 unrelated LCLs that are part of CEPH/Utah pedigrees but are not HapMap LCLs was used to evaluate the validity of using SNP genotypes to predict daunorubicin sensitivity in CEU. IC_50_ was obtained for each LCL by performing cytotoxicity assays as described above. Data was transformed using an inverse normalization of percentile ranks. Genotype analysis for the following SNPs: rs12052523, rs10083335, rs3750518, rs623360, rs2195830, and rs1551315 was performed by single-base extension using the SNaPshot Multiplex method (Applied Biosystems). All SNPs were amplified in separate PCR amplicons using the following conditions: reactions were denatured initially at 95°C for 15 min then cycled at 95°C for 30 s, at 57°C to 58°C for 30 s and at 72°C for 45 s for 40 cycles. Amplicons were pooled (20% of each PCR product) and cleaned up by exonuclease I and shrimp alkaline phosphatase treatment to remove excess primers and deoxynucleotide triphosphates prior to the single-base extension reaction. SNPs were pooled in duplex (rs2195830, rs1551315) and quadruplex (rs12052523, rs10083335, rs3750518, rs623360) single-base extension reactions, respectively. The single-base extension reactions were performed according to the manufacturer’s recommended protocol (Applied Biosystems). The extension products were separated on an ABI 3130 and analyzed by the GeneMapper software version 3.0 (Applied Biosystems).

### Multivariable model estimates daunorubicin sensitivity on validation sample set

To obtain a better genetic prediction effect for daunorubicin IC_50_, we evaluated additive, dominant, and recessive models for each of the six SNPs in the original HapMap CEU samples using the same linear regression approach. The genetic effect that best fit the transformed IC_50_ data was determined by computing *R*^2^ for each univariate model. After selecting the best genetic effect for each marker, the SNPs were entered into a multivariable model to predict daunorubicin IC_50_ in the validation samples using the same models.

### Real-time quantitative PCR for CYP1B1 expression

*CYP1B1* mRNA expression was evaluated in 50 unrelated CEPH non–HapMap LCLs. Exponentially growing cells were diluted at a density of 3.5 × 10^5^ cells/mL per flask. A total of 5 × 10^6^ cells were pelleted and washed in ice-cold PBS and centrifuged to remove PBS. All cell pellets were flash-frozen and stored at −80°C until RNA isolation. Total RNA was extracted using the RNeasy Mini kit (QIAGEN Inc.) following the manufacturer’s protocol. RNA quality assessment and quantification were conducted using the optical spectrometry (260/280 nm) ratio. Subsequently, mRNA was reverse transcribed to cDNA using Applied Biosystems High-Capacity Reverse Transcription kit (Applied Biosystems). The final concentration of cDNA was 50 ng/µL. Real-time reverse transcription-PCR was performed for *CYP1B1* and an endogenous control (*huB2M*) using TaqMan Gene Expression Assays (Applied Biosystems) on the Applied Biosystems 7500 real-time PCR system. Total reaction was carried out in a 25 µL volume which consisted of 12.5 µL of 2× TaqMan Gene Expression-master mix, 1.25 µL primers, and probe mix ( final concentration of 900 nmol/L forward and reverse primers and 250 nmol/L of probe) along with 10 µL of 1.25 ng/µL cDNA. The *CYP1B1* TaqMan primers and probe were labeled with the FAM reporter dye and the MGB quencher dye. *huB2M* primer/probe mixture was labeled with the VIC reporter dye and the MGB quencher dye. The thermocyler variables were 50°C for 2 min, 95°C for 10 min, and 40 cycles of 95°C for 15 s/60°C for 1 min. Each cycle threshold (Ct) value obtained for *CYP1B1* was normalized using *huB2M* independently. A relative standard curve method was used to obtain the relative *CYP1B1* expression in our LCL samples (Guide to performing relative qualification of gene expression using real-time quantitative PCR),[8](#FN7) with the lowest *CYP1B1* expression set as the calibrator for all other LCLs. Each experiment was conducted at least twice and samples were run in triplicate per experiment. Linear regression was then performed between the inverse normalization of percentile ranks–transformed daunorubicin IC_50_ and the relative *CYP1B1* expression. *P* < 0.05 was considered statistically significant.

## Results

### QTDT genotype-cytotoxicity association

Using the alamarBlue cytotoxicity assay, 86 and 89 cell lines derived from CEU and YRI trios were exposed to increasing concentrations of daunorubicin (0.0125–1 µmol/L) for 72 hours and the IC_50_ was determined by curve-fitting the percentage of cell survival against concentrations of the drug ([17](#R17)). Interindividual variation in the IC_50_ was 40-fold and 54-fold with the median IC_50_ at 0.04 and 0.05 µmol/L in CEU and YRI cell lines, respectively ([17](#R17)). We started by evaluating 387,417 SNPs, representing 22,667 genes, for significant association with daunorubicin IC_50_. Using a *P* value threshold (*P* ≤ 0.0001), 207, 233, and 189 SNPs were found to significantly associate with daunorubicin IC_50_ in the combined CEU and YRI populations, and individual CEU and YRI populations, respectively ([Table 1](#T1)). These SNPs were located in or within 10 kb upstream/downstream of 115, 105, and 104 genes. Raw *P* ≤ 0.0001 corresponded to a false discovery rate of 0.167, 0.145, and 0.184 in the combined, CEU, and YRI populations, respectively.

### Table 1.

| Approach | Combined populations | CEU | YRI |
| --- | --- | --- | --- |
| SNP associated with daunorubicin, IC50 (P ≤ 0.0001) | 207 SNPs | 233 SNPs | 189 SNPs |
| SNP associated with daunorubicin, IC50 and gene expression (Bonferroni-corrected, P < 0.05) | 38 SNPs (36 genes) | 30 SNPs (17 genes) | 20 SNPs (42 genes) |
| Gene expression associated with SNP and correlated with daunorubicin, IC50 (P ≤ 0.05) | 16 genes (26 SNPs) | 9 genes (9 SNPs) | 36 genes (18 SNPs) |

Table 1 Caption: Significant results from the sequential three-way model in combined, CEU, or YRI populations

### QTDT genotype and gene expression association

To obtain equally enriched gene expression data, we generated expression data on 176 LCLs (87 CEU and 89 YRI) using Exon Array. The QTDT association analysis was conducted between gene expression and the SNPs that were significantly associated with daunorubicin IC_50_. A total of 13,314 transcript clusters with a mean log_2_-transformed gene expression intensity of >5, indicating expression in both CEU and YRI samples, were included in the analysis. In terms of expression, we found 15 *cis*- and 54 *trans*-acting relationships in the combined populations, 9 *cis*- and 48 *trans*-acting relationships in CEU, and 1 *cis*- and 47 *trans*-acting relationships in YRI (Bonferroni-corrected *P* < 0.05 based on number of transcript clusters). Among all observed *cis*- and *trans*-acting relationships, some SNPs were significantly associated with more than one gene expression, whereas some gene expressions were associated with more than one SNP. Therefore, the final *cis/trans*-acting relationships were represented by 38 SNPs that significantly associated with 36 gene expressions in the combined population, 30 SNPs that significantly associated with 17 gene expressions in CEU, and 20 SNPs that significantly associated with 42 gene expressions in YRI ([Table 1](#T1), [Supplementary Table S1](#SD1)).

### Linear regression of gene expression and daunorubicin IC50

We examined the correlation between gene expression and daunorubicin IC_50_ using a general linear model that was constructed to reflect the relationship among the trio structure of samples in our data. Fourteen transcript clusters (representing 16 genes) had significant correlations with daunorubicin IC_50_ in the combined populations (*P* ≤ 0.05; [Table 1](#T1)). In the same manner, we found 9 and 36 genes whose expressions significantly correlated with daunorubicin IC_50_ in the CEU and YRI populations, respectively (*P* ≤ 0.05; [Table 1](#T1)).

When combining the results generated from association tests between genotypes, daunorubicin IC_50_ and gene expression, as well as the linear regression between gene expression and daunorubicin IC_50_, we identified 26, 9, and 18 SNPs that were significantly associated with daunorubicin IC_50_ through association with 16, 9, and 36 gene expressions in the combined, CEU, and YRI populations, respectively. One example was the significant association between the genotype of rs10932125 (chromosome 2) and daunorubicin IC_50_ (*P* = 1 × 10^−4^). This SNP was associated with the expression of the *CYP1B1* gene (*P* = 1 × 10^−6^), which significantly correlated with daunorubicin IC_50_ (*P* = 3 × 10^−5^; [Fig. 1](#F1)). This data indicates that CYP1B1 confers resistance to daunorubicin. In the individual CEU population, we identified rs3750518 (chromosome 9) to be associated with daunorubicin IC_50_ (*P* = 8 × 10^−5^) and *HNRPD* gene expression (*P* = 3 × 10^−6^). We also found an inverse correlation between the *HNRPD* expression and daunorubicin IC_50_ (*P* = 5 × 10^−4^; [Fig. 2](#F2)). In the YRI population, we identified a strong association between the genotype of SNP rs6603859 and daunorubicin IC_50_ (*P* = 5 × 10^−5^). This SNP genotype was also significantly associated with the expression of the *TAP2* gene located on chromosome 6 (*P* = 3 × 10^−6^). The GG genotype of rs6603859 was associated with higher *TAP2* gene expression and higher daunorubicin IC_50_. This was further indicated by the positive correlation found between *TAP2* gene expression and daunorubicin IC_50_ (*P* = 7 × 10^−3^; [Fig. 3](#F3)), which indicates that this gene confers resistance to daunorubicin. In addition, we identified a significant association between daunorubicin IC_50_ and the genotype of SNP rs7929521 in YRI (*P* = 5 × 10^−5^). This same SNP genotype was significantly associated with the expression of the *IKBKE* gene located on chromosome 1 (*P* = 2 × 10^−6^). The GG genotype of rs7929521 was associated with higher *IKBKE* gene expression and higher daunorubicin IC_50_. This was further indicated by the positive correlation found between *IKBKE* gene expression and daunorubicin IC_50_ (*P* = 3 × 10^−3^; [Fig. 4](#F4)).

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e2c6/2714371/daa900043307/nihms101217f1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=2714371_nihms101217f1.jpg)

Relationship between rs10932125, CYP1B1 gene expression, and daunorubicin IC50 in the combined CEU and YRI populations. A, rs10932125 genotype and inverse-transformed daunorubicin IC50 association. B, rs10932125 genotype and log2-transformed CYP1B1 expression association. C, log2-transformed CYP1B1 expression and inverse-transformed daunorubicin IC50 correlation.

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e2c6/2714371/378d6f2bf12a/nihms101217f2.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=2714371_nihms101217f2.jpg)

Relationship between rs3750518, HNRPD gene expression, and daunorubicin IC50 in the CEU population. A, rs3750518 genotype and inverse-transformed daunorubicin IC50 association. B, rs3750518 genotype and log2-transformed HNRPD expression association. C, log2-transformed HNRPD expression and inverse-transformed daunorubicin IC50 correlation.

### Figure 3.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e2c6/2714371/b3a261c3e1aa/nihms101217f3.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=2714371_nihms101217f3.jpg)

Relationship between rs6603859, TAP2 gene expression, and daunorubicin IC50 in the YRI population. A, rs6603859 genotype and inverse-transformed daunorubicin IC50 association. B, rs6603859 genotype and log2-transformed TAP2 expression association. C, log2-transformed TAP2 expression and inverse-transformed daunorubicin IC50 correlation.

### Figure 4.

![Figure 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e2c6/2714371/7b646e02c037/nihms101217f4.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=2714371_nihms101217f4.jpg)

Relationship between rs7929521, IKBKE gene expression, and daunorubicin IC50 in the YRI population. A, rs7929521 genotype and inverse-transformed daunorubicin IC50 association. B, rs7929521 genotype and log2-transformed IKBKE expression association. C, log2-transformed IKBKE expression and inverse-transformed daunorubicin IC50 correlation.

### Multivariate model to predict daunorubicin IC50 with genotypes

To examine the overall contributions of our selected genetic variants to the sensitivity of daunorubicin, additional general linear models were constructed. In the combined populations, 8 of the 26 tested SNPs were selected in the final model (*P* < 0.04 for all SNPs). Specifically, rs10932125, rs17128525, rs2053456, rs2664420, rs11743052, rs1412312, rs6859237, and rs7704526 were all significant predictors of daunorubicin IC_50_. Race is also a significant predictor of daunorubicin IC_50_ (*P* = 0.0042). This was in agreement with our phenotypic finding that daunorubicin IC_50_ was significantly different between the CEU and YRI populations ([17](#R17)). Computing a weighted sum of *r*^2^ from each group of unrelated individuals gives an overall estimate of *r*^2^ = 0.47, indicating that 47% of the variation in daunorubicin IC_50_ can be explained by these eight SNPs in the combined populations. In the CEU population, six of the nine tested SNPs were included in the final model (*P* < 0.04 for rs10083335, rs12052523, rs2195830, rs3750518, and rs623360; and *P* = 0.05 for rs1551315). When all six SNPs were included in the model, the overall estimate was *r*^2^ = 0.61, indicating that 61% of the daunorubicin IC_50_ variation could be explained by these six SNPs in the CEU population. In the YRI population, 6 of the 18 tested SNPs were included in the final model (*P* < 0.04 for all SNPs). Specifically, rs9730073, rs12250538, rs2255796, rs293381, rs6552429, and rs12783321 were all significant predictors of daunorubicin IC_50_. The overall estimate of *r*^2^ = 0.63 indicated that 63% of the daunorubicin IC_50_ variation could be explained by these six SNPs in the YRI populations.

### SNP genetic effect validation

A set of 50 non-HapMap CEU cell lines were evaluated for their sensitivity to daunorubicin at concentrations between 0.0125 and 1 µmol/L for 72 hours, and IC_50_ determined. Using the single-base extension method, we obtained genotyping information for all six SNP genotypes. We tested dominant and recessive effects for each of these six SNPs in the HapMap CEU samples and used the same genetic model assumptions for the non-HapMap CEU validation set. Upon evaluation in HapMap CEU, rs1551315 best fits the data when dominant genetic effects were assumed, rs12052523 best fits the data when recessive genetic effects were assumed, whereas rs2195830, rs623360, rs10083335, and rs3750518 best fit the data when additive genetic effects were assumed explaining 61% of the variation in HapMap cells. Therefore, these genetic models were entered into a multivariable model to predict daunorubicin IC_50_ in the validation set. With this exploratory approach, the total variation of the transformed daunorubicin IC_50_ explained by the six SNPs was 0.29. Two markers, rs120525235 and rs3750518, were significant predictors of transformed daunorubicin IC_50_ (*P* = 0.005 and *P* = 0.0008, respectively). One marker is trending towards statistical significance in the replication set (rs1551315, *P* = 0.089). The remaining three SNPs do not significantly predict IC_50_. The validation step identifies which SNPs are more likely to be true positives.

### Effect of CYP1B1 on daunorubicin sensitivity

Our data shows a significant correlation between *CYP1B1* expression and daunorubicin IC_50_ ([Fig. 1](#F1)), we therefore set out to evaluate the effect of *CYP1B1* expression on the cellular sensitivity to daunorubicin in the 50 independent CEPH non-HapMap LCLs. Real-time quantitative PCR was performed for *CYP1B1* using expression of *huB2M* as an endogenous control. A linear regression was performed between the relative *CYP1B1* expression and the inverse normalization of percentile ranks–transformed daunorubicin IC_50_. We found a significant correlation between *CYP1B1* expression and daunorubicin IC_50_ in agreement with our findings in the HapMap samples ([Fig. 5](#F5)).

### Figure 5.

![Figure 5](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e2c6/2714371/a7ea627c901a/nihms101217f5.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=2714371_nihms101217f5.jpg)

Relative CYP1B1 expression is correlated with cellular sensitivity to daunorubicin. CYP1B1 relative to endogenous control huB2M expression is correlated with daunorubicin IC50 in 50 independent non-HapMap CEPH samples.

## Discussion

Using a genome-wide, unbiased approach that sequentially evaluated whole genome association between genotype and phenotype (sensitivity to drug), followed by the association between genotype and gene expression, as well as linear regression analysis between gene expression and phenotype, we successfully identified 53 genetic variants (26 in the combined, 9 in the CEU, 18 in the YRI populations) significantly associated with daunorubicin-induced cytotoxicity through the expression of 61 genes in CEU and/or YRI cell lines. A multivariate model indicated that ~47%, 61%, and 63% of the observed daunorubicin IC_50_ variations could be explained by eight, six, and six SNPs identified through our model in the combined, CEU, and YRI populations, respectively. The CEU results were further tested in an independent set of 50 unrelated CEPH LCLs. Two of the six SNPs generated as predictors from our HapMap samples were able to explain ~29% of the daunorubicin sensitivity in the validation set. Furthermore, in this same independent set of samples, we found a significant correlation between *CYP1B1* expression and daunorubicin IC_50_ suggesting that CYP1B1 confers resistance to daunorubicin.

Until recently, few studies used LCLs to determine whether genetic factors contributed to chemotherapeutic-induced cytotoxicity. Using LCLs derived from large pedigrees, our laboratory found that 47% of the variations in susceptibility to cisplatin were due to genetic factors ([20](#R20)). Variation in cellular susceptibility to 5-fluorouracil and docetaxel ([21](#R21)), and daunorubicin (29% of variation in IC_50_ due to genetic factors, *P* = 8 × 10^−7^; ref. [11](#R11)) were also shown to have a significant genetic component.

We have previously described a linkage-directed association approach to identify genetic variants that contribute to daunorubicin cytotoxicity ([11](#R11)). Although our previous approach is cellbased, the previous study differs substantially from the cell-based model described herein. Differences include (*a*) in the previous publication ([11](#R11)), 24 large pedigrees consisting of 324 individual cell lines were used to perform linkage analysis followed by linkage directed QTDT on a subset of samples (87 HapMap CEU samples). The rationale for using large pedigrees was to determine regions on the genome that harbored genetic variation contributing to susceptibility to differing drug concentrations in the CEU population. That approach allowed us to test if different genes (upstream or downstream of the damage) contribute to variations in the sensitivity to low versus high concentrations of daunorubicin. In the current study, we are evaluating genetic variants that are associated with daunorubicin cytotoxicity through their effect on baseline gene expression in CEU, YRI, and combined populations using the IC_50_ as the cytotoxic phenotype. We further validated experimental findings for the genetic predictors and gene expression of CYP1B1 in a separate set of non-HapMap CEU LCLs. Of the significant SNPs that were associated with daunorubicin IC_50_ through gene expressions using our current approach, five are under suggestive linkage peaks. These SNPs are rs1831567 (13q32), rs1412312 (13q32), rs6897941 (5q12.3), rs7704526 (5q35), and rs12052523 (2q36.3), providing greater confidence in their importance in contributing to variation in sensitivity to daunorubicin.

One limitation of multiple testing is the resulting false discovery rate. Stringent statistical cutoffs are used to decrease the false discovery rates ([22](#R22)). We have a relatively small sample size (90 CEU or YRI populations or 180 for the combined populations) to perform genome-wide association tests on the complex trait (e.g., susceptibility to drugs). Our goal was to set a cutoff that would detect genetic contributions to the trait, but also control for the false discovery rate; therefore, a threshold of *P* ≤ 10^−4^ was chosen for the association between SNP genotype and daunorubicin IC_50_. Theoretically, given the 387,417 SNPs tested, this cutoff would produce 39 significant SNPs by chance, although our test found 207, 233, and 189 SNPs that met this cutoff and were associated with daunorubicin IC_50_ in combined, CEU, and YRI populations, respectively. Once these were identified, we narrowed down our genetic variants list to 26, 9, and 18 SNPs that significantly associated with the expression of 16, 9, and 36 genes that were also significantly correlated with daunorubicin IC_50_ (*P* < 0.05) in combined, CEU, and YRI populations, respectively. Given the stringent statistical cutoffs, we did not observe any overlap of SNPs between the CEU and YRI populations. This was not surprising due to the heterogeneity between the CEU and YRI samples. Neither did we observe SNPs that overlapped between the combined populations and the individual populations. The larger number of samples in the combined populations provides more power to detect smaller genetic effects on cytotoxicity and gene expression; however, the unique genetic findings in one population may be masked by the noise produced by a lack of genetic effects in the other population. Thus, we interpreted the final results in each separate population as population-specific genetic variants that contribute to daunorubicin toxicity, whereas those in the combined population were genetic variants important to drug-induced toxicity regardless of population tested.

Our model allowed us to uncover previously unknown genetic variants that are likely important in daunorubicin cytotoxicity. In the combined population, we identified three SNPs located within the *ChGn* gene on chromosome 8 that *trans*-regulated the expression of *ADD3* and also associated with daunorubicin IC_50_. Gyorffy et al. showed a significant correlation between ADD3 expression and resistance of 30 cancer cell lines towards doxorubicin ([23](#R23)). In addition, we identified a significant association between the genotype of rs10932125 (chromosome 2), expression of *CYP1B1* (chromosome 2) and daunorubicin IC_50_. It has been shown in rat liver microsomes that CYP2B1 may play a role in the metabolism of doxorubicin ([24](#R24)); however, in the literature, evidence for genes important in daunorubicin metabolism is sparse. Our data in HapMap cell lines and in non-HapMap CEPH LCLs supports the concept that expression of *CYP1B1* confers resistance to daunorubicin. We are studying the metabolic conversion of daunorubicin by CYP1B1.

Significant associations between genetic variants of rs3750518, *HNRPD* expression, and daunorubicin IC_50_ were found in the CEU population. This SNP was significantly associated with the expression of the *HNRPD* gene located on chromosome 4. The protein encoded by HNRPD [heterogeneous nuclear ribonucleo-protein D (AU-rich element RNA-binding protein 1) or AUF1] is a member of the heterogeneous nuclear ribonucleoproteins (hnRNP) subfamily, which are complexed with heterogeneous nRNA and regulate the mRNA stability. Lapucci et al. have shown that HNRPD can bind to AU-rich elements in the 3′-untranslated region of the *bcl-2* gene and consequently induce apoptosis after UVC irradiation in the transfected Jurkat T-cell leukemia cell line ([25](#R25)). Furthermore, it has been shown that cell death–inhibiting RNA inhibits apoptosis by acting as a competitive inhibitor of AUF1, preventing AUF1 from binding to its targets ([26](#R26)). Our study showed that the GG genotype of SNP rs3750518 associated with higher *HNRPD* gene expression and greater sensitivity to daunorubicin in agreement with evidence in the literature that higher *HNRPD* expression increases apoptosis, and thus, could potentially lead to higher sensitivity to cellular stress induced by daunorubicin.

In the YRI population, we identified a strong association between genetic variants of SNP rs6603859 (chr1) and the expression of six genes (*TAP2*, chr6; *OGDH*, chr7; *RASSF7*, chr11; *PYGB*, chr20; *SAPS2*, chr22; and *LOC442582*, chr22). This SNP was also associated with the susceptibility to daunorubicin-induced cytotoxicity. Furthermore, the expressions of all six genes were significantly correlated with daunorubicin IC_50_. Interestingly, one of the target genes, *TAP2* [transporter 2, ATP-binding cassette, subfamily B (MDR/TAP)], forms a heterodimer with ABCB2 in order to transport peptides from the cytoplasm to the endoplasmic reticulum. Transfection of the *TAP* gene into TAP-deficient lymphoblastoid T2 cells conferred mild resistance to etoposide, vincristine, and doxorubicin (2-fold to 2.5-fold; ref. [27](#R27)). Our study showed that the GG genotype of this SNP was associated with higher *TAP2* gene expression and correlated with lower sensitivity to daunorubicin.

Another interesting finding in YRI was the significant association between genetic variants of SNP rs7929521, gene expression of *IKBKE* [inhibitor of κ light polypeptide gene enhancer in B cells, kinase ε] and susceptibility to daunorubicin-induced cytotoxicity. This SNP is significantly associated with the expression of five genes (*RPL7A, RASSF5, IKBKE, IDH3A*, and *SLC9A8*). In most cells, nuclear factor κ B (NFκB) is trapped in the cytoplasm by IκB. Under stimulation, IκB undergoes phosphorylation by IκB kinase complex (IKK or IKBK), ubiquitinylation, and degradation. Released NFκB migrates into the nucleus and activates target gene expression ([28](#R28), [29](#R29)). It has been shown that inhibition of NFκB activation can result in increased sensitivity to anthracyclines ([30](#R30), [31](#R31)). Tergaonkar et al. have shown IKKα-independent and IKKβ-independent activation of NFκB by doxorubicin ([32](#R32)). Buss et al. have indicated that interleukin-1–inducible phosphorylation of p65 NFκB at Ser^536^ is mediated by multiple protein kinases including IKKα, IKKβ, IKKε, TRAF family member-associated–binding kinase 1 (TBK1), and an unknown kinase ([33](#R33)). Moreover, Adli et al. have shown IKBKE in controlling the proliferation of certain cancer cells through regulation of constitutive NFκB activity ([34](#R34)). Our study revealed that the expression of *IKBKE* was significantly associated with daunorubicin cytotoxicity.

Candidate gene approaches have commonly focused on one gene/pathway, the expression of which is associated with sensitivity to the drug. For example, increased activity of drug efflux transporter, i.e., ABCB1 (p-glycoprotein), ABCG2, and LRP ([35](#R35)–[38](#R38)), and the decreased activity of topoisomerase II have been shown to be associated with resistance to doxorubicin in cancer cell lines ([39](#R39)). In addition, the expression of *c-raf, bcl-2* ([40](#R40)), *p53* ([41](#R41), [42](#R42)), and *CYP2B1* ([24](#R24)) were also shown to correlate with resistance to anthracyclines. Further studies to evaluate whether genetic polymorphisms within these candidate genes were associated with sensitivity to doxorubicin were negative ([43](#R43)–[45](#R45)). This could be due in part to the multigenic nature of drug-induced toxicity and the limited selection of genetic variants within or close to the gene of interest. Recently, several groups have shown that expression of a considerable number of genes is directly controlled by *trans*-acting elements ([22](#R22), [46](#R46)–[50](#R50)). Limited information is available on genetic polymorphisms associated with daunorubicin; thus, our genome-wide approach giving equal weight to all genes and identifying multiple genetic polymorphisms both *cis* and *trans* to the genes associated with daunorubicin-induced cytotoxicity offers investigators genetic variants worth further investigation. The final list of genes in our approach only includes genetic variants that affect cytotoxicity through effects on gene expression but does not include genetic variants that may change expression following drug exposure or variants affecting protein structure, activity, and/or degradation. However, the association test between daunorubicin IC_50_ and genotype (the first step of our model) is inclusive of all genetic variants that act through other means. In addition, we used 387,417 very informative SNPs (located in or near genes) in the genome-wide association to facilitate the most likely candidates for follow-up functional validation studies. An important caveat is that these SNPs are sparser than the full HapMap genotypes; therefore, it is plausible that these SNPs may only be ‘‘surrogate’’ genetic markers whereas other SNPs located in other regions of the genome are the actual causative genetic markers.

In summary, this genome-wide approach successfully integrated genotype, gene expression, and sensitivity with drug information to identify genetic variants that are important in drug cytotoxicity. The model can be used to uncover important genetic variants contributing to a wide range of phenotypes that can be measured in lymphoblastoid cell lines and is likely to find utility for many researchers interested in genotype–expression studies, expression–phenotype studies, or genotype–expression–phenotype studies.

## Supplementary Material

## Acknowledgments

**Grant support:** This Pharmacogenetics of Anticancer Agents Research group ([http://pharmacogenetics.org](http://pharmacogenetics.org)) study was supported by NIH/NIGMS grants UO1GM61393 and UO1GM61374 ([http://pharmgkb.org/](http://pharmgkb.org/)).

We are grateful for the excellent technical support provided by Dr. Jeong-Ah Kang in maintaining the cell lines. In addition, we are grateful to Drs. T.A. Clark, T.X. Chen, A.C. Schweitzer, and J.E. Blume at Expression Research, Affymetrix Laboratory, Affymetrix, Inc., Santa Clara, CA, for their contribution in generating the exon array data on the CEU and YRI cell lines.

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Davis H, Davis T. Daunorubicin and adriamycin in cancer treatment: an analysis of their roles and limitations. Cancer Treat Rep. 1979;63:809–815.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/378369/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Treat%20Rep&title=Daunorubicin%20and%20adriamycin%20in%20cancer%20treatment:%20an%20analysis%20of%20their%20roles%20and%20limitations&author=H%20Davis&author=T%20Davis&volume=63&publication_year=1979&pages=809-815&pmid=378369&)

2. Schurmann D, Dormann A, Grunewald T, Ruf B. Successful treatment of AIDS-related pulmonary Kaposi’ sarcoma with liposomal daunorubicin. Eur Respir J. 1994;7:824–825. doi: 10.1183/09031936.94.07040824.  [DOI](https://doi.org/10.1183/09031936.94.07040824) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8005268/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20Respir%20J&title=Successful%20treatment%20of%20AIDS-related%20pulmonary%20Kaposi%E2%80%99%20sarcoma%20with%20liposomal%20daunorubicin&author=D%20Schurmann&author=A%20Dormann&author=T%20Grunewald&author=B%20Ruf&volume=7&publication_year=1994&pages=824-825&pmid=8005268&doi=10.1183/09031936.94.07040824&)

3. Goodman MF, Bessman MJ, Bachur NR. Adriamycin and daunorubicin inhibition of mutant T4 DNA polymerases. Proc Natl Acad Sci U S A. 1974;71:1193–1196. doi: 10.1073/pnas.71.4.1193.  [DOI](https://doi.org/10.1073/pnas.71.4.1193) | [PMC free article](/articles/PMC388190/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/4524632/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&title=Adriamycin%20and%20daunorubicin%20inhibition%20of%20mutant%20T4%20DNA%20polymerases&author=MF%20Goodman&author=MJ%20Bessman&author=NR%20Bachur&volume=71&publication_year=1974&pages=1193-1196&pmid=4524632&doi=10.1073/pnas.71.4.1193&)

4. Bachur N, Yu F, Johnson R, Hickey R, Wu Y, Malkas L. Helicase inhibition by anthracycline anticancer agents. Mol Pharmacol. 1992;41:993–998.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/1614415/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Pharmacol&title=Helicase%20inhibition%20by%20anthracycline%20anticancer%20agents&author=N%20Bachur&author=F%20Yu&author=R%20Johnson&author=R%20Hickey&author=Y%20Wu&volume=41&publication_year=1992&pages=993-998&pmid=1614415&)

5. Booser D, Hortobagyi G. Anthracycline antibiotics in cancer therapy: focus on drug resistance. Drugs. 1994;47:223–258. doi: 10.2165/00003495-199447020-00002.  [DOI](https://doi.org/10.2165/00003495-199447020-00002) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7512899/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drugs&title=Anthracycline%20antibiotics%20in%20cancer%20therapy:%20focus%20on%20drug%20resistance&author=D%20Booser&author=G%20Hortobagyi&volume=47&publication_year=1994&pages=223-258&pmid=7512899&doi=10.2165/00003495-199447020-00002&)

6. Wojnowski L, Kulle B, Schirmer M, et al. NAD(P)H oxidase and multidrug resistance protein genetic polymorphisms are associated with doxorubicin-induced cardiotoxicity. Circulation. 2005;112:3754–3762. doi: 10.1161/CIRCULATIONAHA.105.576850.  [DOI](https://doi.org/10.1161/CIRCULATIONAHA.105.576850) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16330681/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation&title=NAD(P)H%20oxidase%20and%20multidrug%20resistance%20protein%20genetic%20polymorphisms%20are%20associated%20with%20doxorubicin-induced%20cardiotoxicity&author=L%20Wojnowski&author=B%20Kulle&author=M%20Schirmer&volume=112&publication_year=2005&pages=3754-3762&pmid=16330681&doi=10.1161/CIRCULATIONAHA.105.576850&)

7. Lipshultz S. Exposure to anthracyclines during childhood causes cardiac injury. Semin Oncol. 2006;33:S8–S14. doi: 10.1053/j.seminoncol.2006.04.019.  [DOI](https://doi.org/10.1053/j.seminoncol.2006.04.019) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16781284/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Semin%20Oncol&title=Exposure%20to%20anthracyclines%20during%20childhood%20causes%20cardiac%20injury&author=S%20Lipshultz&volume=33&publication_year=2006&pages=S8-S14&pmid=16781284&doi=10.1053/j.seminoncol.2006.04.019&)

8. Seiter K. Toxicity of the topoisomerase II inhibitors. Expert Opin Drug Saf. 2005;4:219–234. doi: 10.1517/14740338.4.2.219.  [DOI](https://doi.org/10.1517/14740338.4.2.219) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15794715/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Expert%20Opin%20Drug%20Saf&title=Toxicity%20of%20the%20topoisomerase%20II%20inhibitors&author=K%20Seiter&volume=4&publication_year=2005&pages=219-234&pmid=15794715&doi=10.1517/14740338.4.2.219&)

9. Young R, Ozols R, Myers C. The anthracycline antineoplastic drugs. N Engl J Med. 1981;305:139–153. doi: 10.1056/NEJM198107163050305.  [DOI](https://doi.org/10.1056/NEJM198107163050305) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7017406/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=The%20anthracycline%20antineoplastic%20drugs&author=R%20Young&author=R%20Ozols&author=C%20Myers&volume=305&publication_year=1981&pages=139-153&pmid=7017406&doi=10.1056/NEJM198107163050305&)

10. Shan K, Lincoff M, Young J. Anthracycline-induced cardiotoxicity. Ann Intern Med. 1996;125:47–58. doi: 10.7326/0003-4819-125-1-199607010-00008.  [DOI](https://doi.org/10.7326/0003-4819-125-1-199607010-00008) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8644988/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Intern%20Med&title=Anthracycline-induced%20cardiotoxicity&author=K%20Shan&author=M%20Lincoff&author=J%20Young&volume=125&publication_year=1996&pages=47-58&pmid=8644988&doi=10.7326/0003-4819-125-1-199607010-00008&)

11. Duan S, Bleibel WK, Huang RS, et al. Mapping genes that contribute to daunorubicin-induced cytotoxicity. Cancer Res. 2007;67:5425–5433. doi: 10.1158/0008-5472.CAN-06-4431.  [DOI](https://doi.org/10.1158/0008-5472.CAN-06-4431) | [PMC free article](/articles/PMC2735868/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17545624/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=Mapping%20genes%20that%20contribute%20to%20daunorubicin-induced%20cytotoxicity&author=S%20Duan&author=WK%20Bleibel&author=RS%20Huang&volume=67&publication_year=2007&pages=5425-5433&pmid=17545624&doi=10.1158/0008-5472.CAN-06-4431&)

12. Dasgupta R, Adamson P, Davies F, et al. Polymorphic variation in GSTP1 modulates outcome following therapy for multiple myeloma. Blood. 2003;102:2345–2350. doi: 10.1182/blood-2003-02-0444.  [DOI](https://doi.org/10.1182/blood-2003-02-0444) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12791655/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Polymorphic%20variation%20in%20GSTP1%20modulates%20outcome%20following%20therapy%20for%20multiple%20myeloma&author=R%20Dasgupta&author=P%20Adamson&author=F%20Davies&volume=102&publication_year=2003&pages=2345-2350&pmid=12791655&doi=10.1182/blood-2003-02-0444&)

13. Naoe T, Tagawa Y, Kiyoi H, et al. Prognostic significance of the null genotype of glutathione S-transferase-T1 in patients with acute myeloid leukemia: increased early death after chemotherapy. Leukemia. 2002;16:203–208. doi: 10.1038/sj.leu.2402361.  [DOI](https://doi.org/10.1038/sj.leu.2402361) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11840286/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Leukemia&title=Prognostic%20significance%20of%20the%20null%20genotype%20of%20glutathione%20S-transferase-T1%20in%20patients%20with%20acute%20myeloid%20leukemia:%20increased%20early%20death%20after%20chemotherapy&author=T%20Naoe&author=Y%20Tagawa&author=H%20Kiyoi&volume=16&publication_year=2002&pages=203-208&pmid=11840286&doi=10.1038/sj.leu.2402361&)

14. Hasan S, Dinh K, Lombardo F, Kark J. Doxorubicin cardiotoxicity in African Americans. J Natl Med Assoc. 2004;96:196–199.  [PMC free article](/articles/PMC2594938/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14977278/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Natl%20Med%20Assoc&title=Doxorubicin%20cardiotoxicity%20in%20African%20Americans&author=S%20Hasan&author=K%20Dinh&author=F%20Lombardo&author=J%20Kark&volume=96&publication_year=2004&pages=196-199&pmid=14977278&)

15. Huang RS, Duan S, Bleibel WK, et al. A genome-wide approach to identify genetic variants that contribute to etoposide-induced cytotoxicity. Proc Natl Acad Sci U S A. 2007;104:9758–9763. doi: 10.1073/pnas.0703736104.  [DOI](https://doi.org/10.1073/pnas.0703736104) | [PMC free article](/articles/PMC1887589/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17537913/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&title=A%20genome-wide%20approach%20to%20identify%20genetic%20variants%20that%20contribute%20to%20etoposide-induced%20cytotoxicity&author=RS%20Huang&author=S%20Duan&author=WK%20Bleibel&volume=104&publication_year=2007&pages=9758-9763&pmid=17537913&doi=10.1073/pnas.0703736104&)

16. Huang RS, Duan S, Shukla SJ, et al. Identification of genetic variants contributing to cisplatin-induced cytotoxicity using a genome-wide approach. Am J Hum Genet. 2007;81:427–437. doi: 10.1086/519850.  [DOI](https://doi.org/10.1086/519850) | [PMC free article](/articles/PMC1950832/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17701890/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Identification%20of%20genetic%20variants%20contributing%20to%20cisplatin-induced%20cytotoxicity%20using%20a%20genome-wide%20approach&author=RS%20Huang&author=S%20Duan&author=SJ%20Shukla&volume=81&publication_year=2007&pages=427-437&pmid=17701890&doi=10.1086/519850&)

17. Huang RS, Kistner EO, Bleibel WK, Shukla SJ, Dolan ME. Effect of population and gender on chemotherapeutic agent-induced cytotoxicity. Mol Cancer Ther. 2007;6:31–36. doi: 10.1158/1535-7163.MCT-06-0591.  [DOI](https://doi.org/10.1158/1535-7163.MCT-06-0591) | [PMC free article](/articles/PMC2669540/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17237264/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Cancer%20Ther&title=Effect%20of%20population%20and%20gender%20on%20chemotherapeutic%20agent-induced%20cytotoxicity&author=RS%20Huang&author=EO%20Kistner&author=WK%20Bleibel&author=SJ%20Shukla&author=ME%20Dolan&volume=6&publication_year=2007&pages=31-36&pmid=17237264&doi=10.1158/1535-7163.MCT-06-0591&)

18. Abecasis G, Cardon L, Cookson W. A general test of association for quantitative traits in nuclear families. Am J Hum Genet. 2000;66:279–292. doi: 10.1086/302698.  [DOI](https://doi.org/10.1086/302698) | [PMC free article](/articles/PMC1288332/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10631157/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=A%20general%20test%20of%20association%20for%20quantitative%20traits%20in%20nuclear%20families&author=G%20Abecasis&author=L%20Cardon&author=W%20Cookson&volume=66&publication_year=2000&pages=279-292&pmid=10631157&doi=10.1086/302698&)

19. Benjamini Y, Hochberg Y. Controlling the false discovery rate: a practical and powerful approach to multiple testing. J R Stat Soc B. 1995;57:289–300.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20R%20Stat%20Soc%20B&title=Controlling%20the%20false%20discovery%20rate:%20a%20practical%20and%20powerful%20approach%20to%20multiple%20testing&author=Y%20Benjamini&author=Y%20Hochberg&volume=57&publication_year=1995&pages=289-300&)

20. Dolan ME, Newbold KG, Nagasubramanian R, et al. Heritability and linkage analysis of sensitivity to cisplatin-induced cytotoxicity. Cancer Res. 2004;64:4353–4356. doi: 10.1158/0008-5472.CAN-04-0340.  [DOI](https://doi.org/10.1158/0008-5472.CAN-04-0340) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15205351/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=Heritability%20and%20linkage%20analysis%20of%20sensitivity%20to%20cisplatin-induced%20cytotoxicity&author=ME%20Dolan&author=KG%20Newbold&author=R%20Nagasubramanian&volume=64&publication_year=2004&pages=4353-4356&pmid=15205351&doi=10.1158/0008-5472.CAN-04-0340&)

21. Watters JW, Kraja A, Meucci MA, Province MA, McLeod HL. Genome-wide discovery of loci influencing chemotherapy cytotoxicity. Proc Natl Acad Sci U S A. 2004;101:11809–11814. doi: 10.1073/pnas.0404580101.  [DOI](https://doi.org/10.1073/pnas.0404580101) | [PMC free article](/articles/PMC511056/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15282376/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&title=Genome-wide%20discovery%20of%20loci%20influencing%20chemotherapy%20cytotoxicity&author=JW%20Watters&author=A%20Kraja&author=MA%20Meucci&author=MA%20Province&author=HL%20McLeod&volume=101&publication_year=2004&pages=11809-11814&pmid=15282376&doi=10.1073/pnas.0404580101&)

22. Stranger BE, Forrest MS, Clark AG, et al. Genome-wide associations of gene expression variation in humans. PLoS Genet. 2005;1:e78. doi: 10.1371/journal.pgen.0010078.  [DOI](https://doi.org/10.1371/journal.pgen.0010078) | [PMC free article](/articles/PMC1315281/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16362079/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet&title=Genome-wide%20associations%20of%20gene%20expression%20variation%20in%20humans&author=BE%20Stranger&author=MS%20Forrest&author=AG%20Clark&volume=1&publication_year=2005&pages=e78&pmid=16362079&doi=10.1371/journal.pgen.0010078&)

23. Gyorffy B, Burowiak P, Kiesslich O, et al. Gene expression profiling of 30 cancer cell lines predicts resistance towards 11 anticancer drugs at clinically achieved concentrations. Int J Cancer. 2006;118:1699–1712. doi: 10.1002/ijc.21570.  [DOI](https://doi.org/10.1002/ijc.21570) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16217747/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Cancer&title=Gene%20expression%20profiling%20of%2030%20cancer%20cell%20lines%20predicts%20resistance%20towards%2011%20anticancer%20drugs%20at%20clinically%20achieved%20concentrations&author=B%20Gyorffy&author=P%20Burowiak&author=O%20Kiesslich&volume=118&publication_year=2006&pages=1699-1712&pmid=16217747&doi=10.1002/ijc.21570&)

24. Goeptar A, Te Koppele J, Lamme E, Pique J, Vermeulen N. Cytochrome P450 2B1-mediated one-electron reduction of adriamycin: a study with rat liver microsomes and purified enzymes. Mol Pharmacol. 1993;44:1267–1277.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/8264564/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Pharmacol&title=Cytochrome%20P450%202B1-mediated%20one-electron%20reduction%20of%20adriamycin:%20a%20study%20with%20rat%20liver%20microsomes%20and%20purified%20enzymes&author=A%20Goeptar&author=J%20Te%20Koppele&author=E%20Lamme&author=J%20Pique&author=N%20Vermeulen&volume=44&publication_year=1993&pages=1267-1277&pmid=8264564&)

25. Lapucci A, Donnini M, Papucci L, et al. AUF1 is a bcl-2 A + U-rich element-binding protein involved in bcl-2 mRNA destabilization during apoptosis. J Biol Chem. 2002;277:16139–16146. doi: 10.1074/jbc.M201377200.  [DOI](https://doi.org/10.1074/jbc.M201377200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11856759/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=AUF1%20is%20a%20bcl-2%20A%20+%20U-rich%20element-binding%20protein%20involved%20in%20bcl-2%20mRNA%20destabilization%20during%20apoptosis&author=A%20Lapucci&author=M%20Donnini&author=L%20Papucci&volume=277&publication_year=2002&pages=16139-16146&pmid=11856759&doi=10.1074/jbc.M201377200&)

26. Shchors K, Yehiely F, Kular R, Kotlo K, Brewer G, Deiss L. Cell death inhibiting RNA (CDIR) derived from a 3′-untranslated region binds AUF1 and heat shock protein 27. J Biol Chem. 2002;277:47061–47072. doi: 10.1074/jbc.M202272200.  [DOI](https://doi.org/10.1074/jbc.M202272200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12356764/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Cell%20death%20inhibiting%20RNA%20(CDIR)%20derived%20from%20a%203%E2%80%B2-untranslated%20region%20binds%20AUF1%20and%20heat%20shock%20protein%2027&author=K%20Shchors&author=F%20Yehiely&author=R%20Kular&author=K%20Kotlo&author=G%20Brewer&volume=277&publication_year=2002&pages=47061-47072&pmid=12356764&doi=10.1074/jbc.M202272200&)

27. Izquierdo M, Neefjes J, Mathari A, Flens M, Scheffer G, Scheper R. Overexpression of the ABC transporter TAP in multidrug-resistant human cancer cell lines. Br J Cancer. 1996;74:1961–1967. doi: 10.1038/bjc.1996.660.  [DOI](https://doi.org/10.1038/bjc.1996.660) | [PMC free article](/articles/PMC2074820/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8980397/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Cancer&title=Overexpression%20of%20the%20ABC%20transporter%20TAP%20in%20multidrug-resistant%20human%20cancer%20cell%20lines&author=M%20Izquierdo&author=J%20Neefjes&author=A%20Mathari&author=M%20Flens&author=G%20Scheffer&volume=74&publication_year=1996&pages=1961-1967&pmid=8980397&doi=10.1038/bjc.1996.660&)

28. Huxford T, Huang D-B, Malek S, Ghosh G. The crystal structure of the IκBα/NF-κB complex reveals mechanisms of NF-κB inactivation. Cell. 1998;95:759. doi: 10.1016/s0092-8674(00)81699-2.  [DOI](https://doi.org/10.1016/s0092-8674(00)81699-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9865694/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell&title=The%20crystal%20structure%20of%20the%20I%CE%BAB%CE%B1/NF-%CE%BAB%20complex%20reveals%20mechanisms%20of%20NF-%CE%BAB%20inactivation&author=T%20Huxford&author=D-B%20Huang&author=S%20Malek&author=G%20Ghosh&volume=95&publication_year=1998&pages=759&pmid=9865694&doi=10.1016/s0092-8674(00)81699-2&)

29. Jacobs MD, Harrison SC. Structure of an IkBa/NF-κB complex. Cell. 1998;95:749. doi: 10.1016/s0092-8674(00)81698-0.  [DOI](https://doi.org/10.1016/s0092-8674(00)81698-0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9865693/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell&title=Structure%20of%20an%20IkBa/NF-%CE%BAB%20complex&author=MD%20Jacobs&author=SC%20Harrison&volume=95&publication_year=1998&pages=749&pmid=9865693&doi=10.1016/s0092-8674(00)81698-0&)

30. Cherbonnier C, Déas O, Carvalho G, et al. Potentiation of tumour apoptosis by human growth hormone via glutathione production and decreased NF-B activity. Br J Cancer. 2003;89:1108–1115. doi: 10.1038/sj.bjc.6601223.  [DOI](https://doi.org/10.1038/sj.bjc.6601223) | [PMC free article](/articles/PMC2376966/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12966434/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Cancer&title=Potentiation%20of%20tumour%20apoptosis%20by%20human%20growth%20hormone%20via%20glutathione%20production%20and%20decreased%20NF-B%20activity&author=C%20Cherbonnier&author=O%20D%C3%A9as&author=G%20Carvalho&volume=89&publication_year=2003&pages=1108-1115&pmid=12966434&doi=10.1038/sj.bjc.6601223&)

31. Cherbonnier C, Deas O, Vassal G, et al. Human growth hormone gene transfer into tumor cells may improve cancer chemotherapy. Cancer Gene Ther. 2002;9:497–504. doi: 10.1038/sj.cgt.7700467.  [DOI](https://doi.org/10.1038/sj.cgt.7700467) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12032660/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Gene%20Ther&title=Human%20growth%20hormone%20gene%20transfer%20into%20tumor%20cells%20may%20improve%20cancer%20chemotherapy&author=C%20Cherbonnier&author=O%20Deas&author=G%20Vassal&volume=9&publication_year=2002&pages=497-504&pmid=12032660&doi=10.1038/sj.cgt.7700467&)

32. Tergaonkar V, Bottero V, Ikawa M, Li Q, Verma IM. I{κ}B Kinase-Independent I{κ}B{α} degradation path-way: functional NF-{κ}B activity and implications for cancer therapy. Mol Cell Biol. 2003;23:8070–8083. doi: 10.1128/MCB.23.22.8070-8083.2003.  [DOI](https://doi.org/10.1128/MCB.23.22.8070-8083.2003) | [PMC free article](/articles/PMC262380/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14585967/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Cell%20Biol&title=I%7B%CE%BA%7DB%20Kinase-Independent%20I%7B%CE%BA%7DB%7B%CE%B1%7D%20degradation%20path-way:%20functional%20NF-%7B%CE%BA%7DB%20activity%20and%20implications%20for%20cancer%20therapy&author=V%20Tergaonkar&author=V%20Bottero&author=M%20Ikawa&author=Q%20Li&author=IM%20Verma&volume=23&publication_year=2003&pages=8070-8083&pmid=14585967&doi=10.1128/MCB.23.22.8070-8083.2003&)

33. Buss H, Dorrie A, Schmitz M, Hoffmann E, Resch K, Kracht M. Constitutive and interleukin-1-inducible phosphorylation of p65 NF-{κ}B at serine 536 is mediated by multiple protein kinases including I{κ}B kinase (IKK)-{α}, IKK{β}, IKK{ε}, TRAF family member-associated (TANK)-binding kinase 1 (TBK1), and an unknown kinase and couples p65 to TATA-binding protein-associated factor II31-mediated interleukin-8 transcription. J Biol Chem. 2004;279:55633–55643. doi: 10.1074/jbc.M409825200.  [DOI](https://doi.org/10.1074/jbc.M409825200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15489227/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Constitutive%20and%20interleukin-1-inducible%20phosphorylation%20of%20p65%20NF-%7B%CE%BA%7DB%20at%20serine%20536%20is%20mediated%20by%20multiple%20protein%20kinases%20including%20I%7B%CE%BA%7DB%20kinase%20(IKK)-%7B%CE%B1%7D,%20IKK%7B%CE%B2%7D,%20IKK%7B%CE%B5%7D,%20TRAF%20family%20member-associated%20(TANK)-binding%20kinase%201%20(TBK1),%20and%20an%20unknown%20kinase%20and%20couples%20p65%20to%20TATA-binding%20protein-associated%20factor%20II31-mediated%20interleukin-8%20transcription&author=H%20Buss&author=A%20Dorrie&author=M%20Schmitz&author=E%20Hoffmann&author=K%20Resch&volume=279&publication_year=2004&pages=55633-55643&pmid=15489227&doi=10.1074/jbc.M409825200&)

34. Adli M, Baldwin A. IKK-i/IKKε controls constitutive, cancer cell-associated NF-κB activity via regulation of Ser-536 p65/RelA phosphorylation. J Biol Chem. 2006;281:26976–26984. doi: 10.1074/jbc.M603133200.  [DOI](https://doi.org/10.1074/jbc.M603133200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16840782/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=IKK-i/IKK%CE%B5%20controls%20constitutive,%20cancer%20cell-associated%20NF-%CE%BAB%20activity%20via%20regulation%20of%20Ser-536%20p65/RelA%20phosphorylation&author=M%20Adli&author=A%20Baldwin&volume=281&publication_year=2006&pages=26976-26984&pmid=16840782&doi=10.1074/jbc.M603133200&)

35. Fukushima K, Okai Y, Matsuura S, Tsujimoto H, Endo Y. Molecular cloning of feline lung resistance-related protein (LRP) cDNA and its expression in a feline lymphoma cell line and adriamycin-resistant subline. J Vet Med Sci. 2006;68:885–890. doi: 10.1292/jvms.68.885.  [DOI](https://doi.org/10.1292/jvms.68.885) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16953094/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Vet%20Med%20Sci&title=Molecular%20cloning%20of%20feline%20lung%20resistance-related%20protein%20(LRP)%20cDNA%20and%20its%20expression%20in%20a%20feline%20lymphoma%20cell%20line%20and%20adriamycin-resistant%20subline&author=K%20Fukushima&author=Y%20Okai&author=S%20Matsuura&author=H%20Tsujimoto&author=Y%20Endo&volume=68&publication_year=2006&pages=885-890&pmid=16953094&doi=10.1292/jvms.68.885&)

36. Pesic M, Markovic J, Jankovic D, et al. Induced resistance in the human non small cell lung carcinoma (NCI-H460) cell line in vitro by anticancer drugs. J Chemother. 2006;18:66–73. doi: 10.1179/joc.2006.18.1.66.  [DOI](https://doi.org/10.1179/joc.2006.18.1.66) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16572896/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Chemother&title=Induced%20resistance%20in%20the%20human%20non%20small%20cell%20lung%20carcinoma%20(NCI-H460)%20cell%20line%20in%20vitro%20by%20anticancer%20drugs&author=M%20Pesic&author=J%20Markovic&author=D%20Jankovic&volume=18&publication_year=2006&pages=66-73&pmid=16572896&doi=10.1179/joc.2006.18.1.66&)

37. Han B, Zhang J. Multidrug resistance in cancer chemotherapy and xenobiotic protection mediated by the half ATP-binding cassette transporter ABCG2. Curr Med Chem Anticancer Agents. 2004;4:31–42. doi: 10.2174/1568011043482205.  [DOI](https://doi.org/10.2174/1568011043482205) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14754410/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Med%20Chem%20Anticancer%20Agents&title=Multidrug%20resistance%20in%20cancer%20chemotherapy%20and%20xenobiotic%20protection%20mediated%20by%20the%20half%20ATP-binding%20cassette%20transporter%20ABCG2&author=B%20Han&author=J%20Zhang&volume=4&publication_year=2004&pages=31-42&pmid=14754410&doi=10.2174/1568011043482205&)

38. Cheng SH, Lam W, Lee ASK, Fung KP, Wu RSS, Fong WF. Low-level doxorubicin resistance in benzo[a]-pyrene-treated KB-3-1 cells is associated with increased LRP expression and altered subcellular drug distribution. Toxicol Appl Pharmacol. 2000;164:134–142. doi: 10.1006/taap.2000.8903.  [DOI](https://doi.org/10.1006/taap.2000.8903) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10764626/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Toxicol%20Appl%20Pharmacol&title=Low-level%20doxorubicin%20resistance%20in%20benzo%5Ba%5D-pyrene-treated%20KB-3-1%20cells%20is%20associated%20with%20increased%20LRP%20expression%20and%20altered%20subcellular%20drug%20distribution&author=SH%20Cheng&author=W%20Lam&author=ASK%20Lee&author=KP%20Fung&author=RSS%20Wu&volume=164&publication_year=2000&pages=134-142&pmid=10764626&doi=10.1006/taap.2000.8903&)

39. Scheltema JMW, Romijn JC, Steenbrugge GJv, Beck WT, Schröder FH, Mickisch GH. Decreased levels of topoisomerase IIα in human renal cell carcinoma lines resistant to etoposide. J Cancer Res Clin Oncol. 1997;123:546–554. doi: 10.1007/s004320050103.  [DOI](https://doi.org/10.1007/s004320050103) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9393588/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Cancer%20Res%20Clin%20Oncol&title=Decreased%20levels%20of%20topoisomerase%20II%CE%B1%20in%20human%20renal%20cell%20carcinoma%20lines%20resistant%20to%20etoposide&author=JMW%20Scheltema&author=JC%20Romijn&author=GJv%20Steenbrugge&author=WT%20Beck&author=FH%20Schr%C3%B6der&volume=123&publication_year=1997&pages=546-554&pmid=9393588&doi=10.1007/s004320050103&)

40. Cioca D, Aoki Y, Kiyosawa K. RNA interference is a functional pathway with therapeutic potential in human myeloid leukemia cell lines. Cancer Gene Ther. 2003;10:125–133. doi: 10.1038/sj.cgt.7700544.  [DOI](https://doi.org/10.1038/sj.cgt.7700544) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12536201/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Gene%20Ther&title=RNA%20interference%20is%20a%20functional%20pathway%20with%20therapeutic%20potential%20in%20human%20myeloid%20leukemia%20cell%20lines&author=D%20Cioca&author=Y%20Aoki&author=K%20Kiyosawa&volume=10&publication_year=2003&pages=125-133&pmid=12536201&doi=10.1038/sj.cgt.7700544&)

41. Brown J, Wouters B. Apoptosis, p53, and tumor cell sensitivity to anticancer agents. Cancer Res. 1999;59:1391–1399.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/10197600/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=Apoptosis,%20p53,%20and%20tumor%20cell%20sensitivity%20to%20anticancer%20agents&author=J%20Brown&author=B%20Wouters&volume=59&publication_year=1999&pages=1391-1399&pmid=10197600&)

42. Lowe S, Bodis S, McClatchey A, et al. p53 status and the efficacy of cancer therapy in vivo. Science. 1994;266:807–810. doi: 10.1126/science.7973635.  [DOI](https://doi.org/10.1126/science.7973635) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7973635/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=p53%20status%20and%20the%20efficacy%20of%20cancer%20therapy%20in%20vivo&author=S%20Lowe&author=S%20Bodis&author=A%20McClatchey&volume=266&publication_year=1994&pages=807-810&pmid=7973635&doi=10.1126/science.7973635&)

43. Efferth T, Sauerbrey A, Steinbach D, et al. Analysis of single nucleotide polymorphism C3435T of the multi-drug resistance gene MDR1 in acute lymphoblastic leukemia. Int J Oncol. 2003;23:509–517.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/12851703/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Oncol&title=Analysis%20of%20single%20nucleotide%20polymorphism%20C3435T%20of%20the%20multi-drug%20resistance%20gene%20MDR1%20in%20acute%20lymphoblastic%20leukemia&author=T%20Efferth&author=A%20Sauerbrey&author=D%20Steinbach&volume=23&publication_year=2003&pages=509-517&pmid=12851703&)

44. Staalesen V, Leirvaag B, Lillehaug JR, Lonning PE. Genetic and epigenetic changes in p21 and p21B do not correlate with resistance to doxorubicin or mitomycin and 5-fluorouracil in locally advanced breast cancer. Clin Cancer Res. 2004;10:3438–3443. doi: 10.1158/1078-0432.CCR-03-0796.  [DOI](https://doi.org/10.1158/1078-0432.CCR-03-0796) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15161699/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=Genetic%20and%20epigenetic%20changes%20in%20p21%20and%20p21B%20do%20not%20correlate%20with%20resistance%20to%20doxorubicin%20or%20mitomycin%20and%205-fluorouracil%20in%20locally%20advanced%20breast%20cancer&author=V%20Staalesen&author=B%20Leirvaag&author=JR%20Lillehaug&author=PE%20Lonning&volume=10&publication_year=2004&pages=3438-3443&pmid=15161699&doi=10.1158/1078-0432.CCR-03-0796&)

45. Sturm I, Bosanguet A, Hummel M, Dorken B, Daniel P. In B-CLL, the codon 72 polymorphic variants of p53 are not related to drug resistance and disease prognosis. BMC Cancer. 2005;18:105. doi: 10.1186/1471-2407-5-105.  [DOI](https://doi.org/10.1186/1471-2407-5-105) | [PMC free article](/articles/PMC1208864/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16109171/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Cancer&title=In%20B-CLL,%20the%20codon%2072%20polymorphic%20variants%20of%20p53%20are%20not%20related%20to%20drug%20resistance%20and%20disease%20prognosis&author=I%20Sturm&author=A%20Bosanguet&author=M%20Hummel&author=B%20Dorken&author=P%20Daniel&volume=18&publication_year=2005&pages=105&pmid=16109171&doi=10.1186/1471-2407-5-105&)

46. Cheung V, Conlin L, Weber T, et al. Natural variation in human gene expression assessed in lymphoblastoid cells. Nat Genet. 2003;33:422–425. doi: 10.1038/ng1094.  [DOI](https://doi.org/10.1038/ng1094) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12567189/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Natural%20variation%20in%20human%20gene%20expression%20assessed%20in%20lymphoblastoid%20cells&author=V%20Cheung&author=L%20Conlin&author=T%20Weber&volume=33&publication_year=2003&pages=422-425&pmid=12567189&doi=10.1038/ng1094&)

47. Cheung VG, Spielman RS, Ewens KG, Weber TM, Morley M, Burdick JT. Mapping determinants of human gene expression by regional and genome-wide association. Nature. 2005;437:1365–1369. doi: 10.1038/nature04244.  [DOI](https://doi.org/10.1038/nature04244) | [PMC free article](/articles/PMC3005311/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16251966/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Mapping%20determinants%20of%20human%20gene%20expression%20by%20regional%20and%20genome-wide%20association&author=VG%20Cheung&author=RS%20Spielman&author=KG%20Ewens&author=TM%20Weber&author=M%20Morley&volume=437&publication_year=2005&pages=1365-1369&pmid=16251966&doi=10.1038/nature04244&)

48. Deutsch S, Lyle R, Dermitzakis ET, et al. Gene expression variation and expression quantitative trait mapping of human chromosome 21 genes. Hum Mol Genet. 2005;14:3741–3749. doi: 10.1093/hmg/ddi404.  [DOI](https://doi.org/10.1093/hmg/ddi404) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16251198/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=Gene%20expression%20variation%20and%20expression%20quantitative%20trait%20mapping%20of%20human%20chromosome%2021%20genes&author=S%20Deutsch&author=R%20Lyle&author=ET%20Dermitzakis&volume=14&publication_year=2005&pages=3741-3749&pmid=16251198&doi=10.1093/hmg/ddi404&)

49. Monks S, Leonardson A, Zhu H, et al. Genetic inheritance of gene expression in human cell lines. Am J Hum Genet. 2004;75:1094–1105. doi: 10.1086/426461.  [DOI](https://doi.org/10.1086/426461) | [PMC free article](/articles/PMC1182144/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15514893/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Genetic%20inheritance%20of%20gene%20expression%20in%20human%20cell%20lines&author=S%20Monks&author=A%20Leonardson&author=H%20Zhu&volume=75&publication_year=2004&pages=1094-1105&pmid=15514893&doi=10.1086/426461&)

50. Morley M, Molony CM, Weber TM, et al. Genetic analysis of genome-wide variation in human gene expression. Nature. 2004;430:743–747. doi: 10.1038/nature02797.  [DOI](https://doi.org/10.1038/nature02797) | [PMC free article](/articles/PMC2966974/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15269782/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Genetic%20analysis%20of%20genome-wide%20variation%20in%20human%20gene%20expression&author=M%20Morley&author=CM%20Molony&author=TM%20Weber&volume=430&publication_year=2004&pages=743-747&pmid=15269782&doi=10.1038/nature02797&)
