# Pharmacogenetics of antipsychotic response in the CATIE trial: a candidate gene analysis

## Metadata
**Authors:** Anna C Need, Richard SE Keefe, Dongliang Ge, Iris Grossman, Sam Dickson, Joseph P McEvoy, David B Goldstein
**Journal:** European Journal of Human Genetics
**Date:** 2009 Jan 21
**DOI:** [10.1038/ejhg.2008.264](https://doi.org/10.1038/ejhg.2008.264)
**PMID:** 19156168
**PMCID:** PMC2986499
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2986499/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC2986499/pdf/ejhg2008264a.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC2986499/pdf/ejhg2008264a.pdf)

## Abstract

The Clinical Antipsychotic Trials of Intervention Effectiveness (CATIE) Phase 1 Schizophrenia trial compared the effectiveness of one typical and four atypical antipsychotic medications. Although trials such as CATIE present important opportunities for pharmacogenetics research, the very richness of the clinical data presents challenges for statistical interpretation, and in particular the risk that data mining will lead to false-positive discoveries. For this reason, it is both misleading and unhelpful to perpetuate the current practice of reporting association results for these trials one gene at a time, ignoring the fact that multiple gene-by-phenotype tests are being carried out on the same data set. On the other hand, suggestive associations in such trials may lead to new hypotheses that can be tested through both replication efforts and biological experimentation. The appropriate handling of these forms of data therefore requires dissemination of association statistics without undue emphasis on select findings. Here we attempt to illustrate this approach by presenting association statistics for 2769 polymorphisms in 118 candidate genes evaluated for 21 pharmacogenetic phenotypes. On current evidence it is impossible to know which of these associations may be real, although in total they form a valuable resource that is immediately available to the scientific community.

Keywords: schizophrenia, neurocognition, RIMS1, quetiapine, GRM8, discontinuation

## Introduction

One notable result of the Clinical Antipsychotic Trials of Intervention Effectiveness (CATIE) trial of antipsychotic effectiveness in schizophrenia^1^ was the high discontinuation rates among the four atypical and one typical antipsychotic drugs trialed – ranging from 64% for olanzapine to 82% for quetiapine. An even more striking outcome was the rough equivalence among the medicines in most measures when patients on each drug are considered as a group. Although current medicines therefore do not appear better or worse on average, at the individual patient level, variation in response is pronounced. A major goal of psychiatric pharmacogenetics is therefore the prediction of the drug that will work best, with the fewest side effects, for individual patients with schizophrenia.

Unfortunately, our capacity to predict therapeutic response and clinically significant side effects in individual patients is currently extremely limited. There are, for example, no effective means by which to match individual patients to medications that offer them more symptom control or that reduce their chances or severity of adverse reactions. In general, studies implicating genetic variants in antipsychotic response have not been well replicated, with follow-up studies often using different genetic markers and models, testing heterogeneous study populations, and affected by a publication bias of positive findings.^2, 3^ There may be some cumulative evidence for associations with weight gain and tardive dyskinesia, although effect sizes are generally small.^3, 4, 5, 6^ There are examples of genetic associations with antipsychotic response that have led to the development of predictive tests: a human leukocyte antigen test that measures the risk of developing agranulocytosis when treated with clozapine,^7^ and a combination of polymorphisms that can be used in certain patient groups to predict clozapine response;^8^ however, both these tests are of very limited clinical utility.^5^ Finally, the AmpliChip CYP450 pharmacogenetic test was hoped to help predict antipsychotic response.^9^ However, a recent study showed that these genotypes cannot predict antipsychotic dose or response in real-world settings,^10^ and the AmpliChip too is expected to be of little utility in antipsychotic prescribing.^11^

Here we have identified 21 response phenotypes, some of which are partially correlated, and evaluated them in the 756 CATIE participants consented for genetic studies. Although in the CATIE trial, the primary phenotype was discontinuation of treatment for any cause, we felt that this end point was too heterogeneous for a genetic study. Instead, we examined CATIE secondary outcome measures, including discontinuation due to inefficacy, weight gain and change in PANSS scores. To reduce the scope for data mining, our philosophy for defining these pharmacogenetic phenotypes was, wherever possible, to follow how CATIE investigators evaluated and reported responses in the trial. For example, a weight-gain ‘case' was defined as a patient who gained 7% or more of his or her baseline body weight (cf. Lieberman *et al.*^1^). Finally, as neurocognitive impairment in schizophrenia is a significant predictor of outcome^12^ and has been suggested to be a particularly suitable trait for the genetic study of schizophrenia,^13^ we searched for genetic predictors of cognitive change, utilizing data from detailed cognitive assessments performed throughout the CATIE trial and reflecting various measures of verbal learning and memory, working memory, motor function, attention and executive function. All phenotypes in this paper were defined before genetic analyses were performed to prevent false positives due to unconstrained or biased data mining.

## Materials and methods

A detailed description of the participants consented for genetic study is provided in Sullivan *et al.*^14^ SNPs were genotyped using Illumina GoldenGate technology. For further description of the CATIE trial and participants and detailed description of gene and SNP selection, genotyping and quality control, phenotype definitions and statistical analyses, please see Supplementary Methods. All genes examined and their SNP coverage are shown in [Table 1](#tbl1); all phenotypes and sample sizes for each association test are shown in [Table 2](#tbl2).

### Table 1. Coverage for each gene in relevant HapMap populations.

| Gene | No. of original tags | No. of tags passed QC | Average r2CEU | Average r2YRI | Gene | No. of original tags | No. of tags passed QC | Average r2CEU | Average r2YRI | Gene | No. of original tags | No. of tags passed QC | Average r2CEU | Average r2YRI |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| ACE | 19 | 17 | 0.88 | 0.58 | DRD5 | 3 | 2 | 0.00 | 0.00 | HTR2B | 9 | 8 | 0.59 | 0.47 |
| ACHE | 6 | 4 | 0.00 | 0.77 | GAD1 | 13 | 12 | 0.84 | 0.52 | HTR2C | 36 | 30 | 0.93 | 0.75 |
| ADCY1 | 33 | 25 | 0.76 | 0.62 | GAD2 | 22 | 21 | 0.92 | 0.47 | HTR3A | 11 | 10 | 0.52 | 0.51 |
| ADCY2 | 80 | 75 | 0.75 | 0.53 | GLS | 19 | 16 | 0.84 | 0.67 | HTR3B | 13 | 12 | 0.69 | 0.67 |
| ADCY8 | 84 | 75 | 0.82 | 0.67 | GLUD1 | 8 | 7 | 0.78 | 0.52 | HTR4 | 36 | 33 | 0.81 | 0.58 |
| ADCY9 | 44 | 41 | 0.67 | 0.53 | GLUD2 | 4 | 1 | 0.00 | 0.00 | HTR5A | 11 | 9 | 0.63 | 0.51 |
| ADORA2A | 9 | 7 | 0.42 | 0.92 | GLUL | 8 | 8 | 0.82 | 0.67 | HTR6 | 8 | 8 | 0.90 | 0.57 |
| BCHE | 9 | 8 | 0.77 | 0.56 | GRIA1 | 49 | 47 | 0.73 | 0.68 | HTR7 | 13 | 12 | 0.78 | 0.60 |
| BDNF | 9 | 8 | 0.80 | 0.59 | GRIA2 | 14 | 13 | 0.74 | 0.68 | MAOA | 9 | 8 | 0.86 | 0.28 |
| CACNG2 | 48 | 40 | 0.69 | 0.56 | GRIA3 | 86 | 74 | 0.78 | 0.50 | MAOB | 19 | 19 | 0.71 | 0.60 |
| CAMK2A | 26 | 25 | 0.70 | 0.35 | GRIA4 | 42 | 39 | 0.82 | 0.65 | MTHFR | 12 | 12 | 0.73 | 0.63 |
| CAMK2B | 21 | 14 | 0.69 | 0.44 | GRIN1 | 7 | 6 | 0.34 | 0.43 | NOTCH4 | 29 | 23 | 0.73 | 0.56 |
| CCL2 | 6 | 4 | 0.92 | 0.63 | GRIN2A | 75 | 75 | 0.79 | 0.60 | NTRK2 | 77 | 70 | 0.84 | 0.65 |
| CHAT | 22 | 21 | 0.66 | 0.48 | GRIN2B | 114 | 109 | 0.70 | 0.63 | PPP1R1B | 3 | 3 | 0.82 | 0.67 |
| CHRM1 | 10 | 9 | 0.61 | 0.52 | GRIN2C | 8 | 6 | 0.88 | 0.54 | PPP3R1 | 9 | 9 | 0.87 | 0.71 |
| CHRM2 | 31 | 29 | 0.80 | 0.58 | GRIN2D | 16 | 13 | 0.78 | 0.33 | PPP3R2 | 1 | 1 | 0.01 | 0.01 |
| CHRM3 | 55 | 51 | 0.71 | 0.50 | GRIN3A | 42 | 35 | 0.84 | 0.71 | PRNP | 13 | 11 | 0.88 | 0.49 |
| CHRM4 | 2 | 2 | 0.00 | 0.24 | GRIN3B | 10 | 7 | 0.55 | 0.52 | RELN | 8 | 6 | 0.20 | 0.13 |
| CHRM5 | 11 | 9 | 0.76 | 0.62 | GRM1 | 53 | 50 | 0.86 | 0.69 | RGS9 | 12 | 9 | 0.82 | 0.52 |
| CHRNA10 | 7 | 5 | 0.58 | 0.36 | GRM2 | 4 | 3 | 0.00 | 0.00 | RIMS1 | 71 | 58 | 0.73 | 0.60 |
| CHRNA2 | 16 | 15 | 0.72 | 0.51 | GRM3 | 33 | 31 | 0.85 | 0.63 | SLC17A6 | 15 | 15 | 0.73 | 0.52 |
| CHRNA3 | 4 | 4 | 0.78 | 0.39 | GRM4 | 32 | 28 | 0.66 | 0.54 | SLC17A7 | 9 | 7 | 0.50 | 0.15 |
| CHRNA4 | 8 | 6 | 0.67 | 0.49 | GRM5 | 76 | 73 | 0.82 | 0.60 | SLC18A1 | 15 | 13 | 0.77 | 0.52 |
| CHRNA5 | 12 | 11 | 0.97 | 0.77 | GRM6 | 14 | 13 | 0.67 | 0.62 | SLC18A2 | 17 | 17 | 0.63 | 0.33 |
| CHRNA6 | 4 | 4 | 0.73 | 0.64 | GRM7 | 200 | 181 | 0.73 | 0.53 | SLC1A1 | 51 | 48 | 0.63 | 0.51 |
| CHRNA7 | 18 | 17 | 0.70 | 0.45 | GRM8 | 275 | 244 | 0.90 | 0.77 | SLC1A2 | 37 | 35 | 0.86 | 0.55 |
| CHRNA9 | 13 | 10 | 0.84 | 0.40 | GSK3A | 2 | 2 | 0.00 | 0.00 | SLC1A3 | 28 | 27 | 0.60 | 0.48 |
| CHRNB2 | 10 | 9 | 0.24 | 0.36 | GSK3B | 21 | 19 | 0.94 | 0.88 | SLC1A6 | 9 | 8 | 0.56 | 0.51 |
| CHRNB3 | 11 | 11 | 0.93 | 0.60 | HDC | 15 | 14 | 0.60 | 0.59 | SLC6A3 | 19 | 17 | 0.67 | 0.61 |
| CHRNB4 | 8 | 8 | 0.56 | 0.29 | HNMT | 17 | 17 | 0.96 | 0.51 | SLC6A4 | 11 | 10 | 0.59 | 0.35 |
| COMT | 23 | 18 | 0.81 | 0.44 | HRH1 | 7 | 5 | 0.84 | 0.42 | SNAP25 | 27 | 25 | 0.69 | 0.44 |
| CREB1 | 13 | 13 | 0.89 | 0.78 | HRH2 | 10 | 9 | 0.51 | 0.05 | SOD1 | 8 | 8 | 0.77 | 0.64 |
| CREBBP | 24 | 23 | 0.62 | 0.52 | HRH3 | 8 | 7 | 0.29 | 0.15 | SOD2 | 7 | 4 | 0.65 | 0.36 |
| DAO | 9 | 8 | 0.77 | 0.63 | HRH4 | 14 | 12 | 0.57 | 0.52 | SYT11 | 9 | 9 | 0.86 | 0.39 |
| DBH | 27 | 22 | 0.62 | 0.37 | HTR1A | 4 | 4 | 0.00 | 0.00 | SYT4 | 5 | 4 | 0.37 | 0.61 |
| DDC | 21 | 17 | 0.71 | 0.62 | HTR1B | 10 | 10 | 1.00 | 1.00 | TH | 7 | 6 | 0.22 | 0.24 |
| DRD1 | 8 | 8 | 0.69 | 0.31 | HTR1D | 3 | 3 | 0.00 | 0.00 | TPH2 | 25 | 24 | 0.90 | 0.63 |
| DRD2 | 25 | 23 | 0.87 | 0.57 | HTR1E | 15 | 14 | 0.76 | 0.72 | ZDHHC8 | 10 | 7 | 0.48 | 0.47 |
| DRD3 | 17 | 17 | 0.84 | 0.55 | HTR1F | 5 | 5 | 1.00 | 0.90 |   |   |   |   |   |
| DRD4 | 4 | 4 | 0.00 | 0.19 | HTR2A | 23 | 22 | 0.70 | 0.43 |   |   |   |   |   |

Table 1 Caption: CEU, CEPH samples from Utah; YRI, Yoruban samples from Nigeria. Displayed are both the number of original SNPs (n=3072) and those remaining after poorly genotyped SNPs were removed (n=2769). The average r2 calculations were based on phase II HapMap data including only SNPs with MAF>0.05 in one or both of the populations. For each gene, each HapMap SNP was tested against all tags for that gene, and the highest pairwise r2 value with any tag was used to calculate the average.

### Table 2. Phenotypes analyzed.

| Phenotype (description) | Sample size (case–control) | Description | Statistical analysis |
| --- | --- | --- | --- |
| Visuospatial working memory change (computerized test of spatial working memory) | 402 | Neurocognitive baseline score subtracted from score at 2 months, only for subjects who are still on phase I drug at 2 months | All drug groups analyzed together, using linear regression with additive genetic model and including the Eigenstrat axis, sex, self-described race, baseline and phase I drug as covariates |
| Letter/number sequencing change (reorder auditorily presented letter-number clusters) | 442 |   |   |
| FAS change (name words beginning with F, A or S) | 447 |   |   |
| Categories change (name words in particular category) | 447 |   |   |
| Digit symbol change ((WAIS-R) draw symbols associated with digits) | 447 |   |   |
| WCST perseverative error change (sort cards by correct sorting strategy) | 421 |   |   |
| Mazes change ((WISC-R), timed navigation of pencil through mazes) | 439 |   |   |
| Verbal change (recall of 12 nouns verbally presented 3 times) | 447 |   |   |
| Continuous performance change (detecting rapidly presented repeated numbers) | 381 |   |   |
| Neurocognitive change principal component 1 (PC1) | 447 |   |   |
|   |   |   |   |
| PANSS total change | 524 | PANSS baseline score subtracted from score at 3 months, only for subjects who are still on phase I drug at 3 months. Negative score reflects improvement as the higher the PANSS score the worse the symptoms | As for cognitive change |
| PANSS positive change | 524 |   |   |
| PANSS negative change | 524 |   |   |
| PANSS general psychopathology change | 524 |   |   |
|   |   |   |   |
| Weight gain quantitative | 653 | Maximum % weight gain at any point during phase I | As for cognitive change |
| Weight gain case–control | 472/181 | Cases gained ≥7% of baseline weight at any point during phase I | Logistic regression, otherwise as for cognitive change |
| Discontinuation olanzapine inefficacy | 127/21 | Phase I data only, cases defined as patients who did not continue phase I drug until the end of the trial for whom the primary reason for discontinuation was recorded as ‘inadequate therapeutic effect' | Logistic regression with additive genetic model including the Eigenstrat axis, sex and race as covariates |
| Discontinuation perphenazine inefficacy | 79/35 |   |   |
| Discontinuation quetiapine inefficacy | 87/47 |   |   |
| Discontinuation risperidone inefficacy | 101/40 |   |   |
| Discontinuation ziprasidone inefficacy | 50/19 |   |   |
## Results

In total, 21 phenotypes were tested ([Table 2](#tbl2)). Two associations were significant after permutation correction for all tested SNPs (see below). No associations were significant after correction for all SNPs and all phenotypes tested (*P*<9 × 10^−7^).

### Neurocognition

#### Study-wide significant association

SNP rs7778604, located in intron 2 of the *GRM8* gene, was associated with antipsychotic-induced change in verbal memory, and this remained significant after permutation correcting for all studied SNPs (corrected *P*=0.02, uncorrected *P*=0.00001, *n*=447; [Figure 1](#fig1)). The SNP was present in both African-American and European-American participants, and to ensure that the association was not caused by uncorrected stratification, we repeated the analysis of this phenotype with 10 axes of ancestry, instead of a single axis. The association remained significant at 0.00037 (uncorrected), but was no longer significant after correction by permutation for all tested SNPs (*P*=0.58). Those patients carrying the rarer variant (*n*=97) tended to improve in verbal memory with 8 weeks of treatment, whereas those homozygous for the common allele (*n*=369) performed worse after 8 weeks of phase 1 treatment. The associated SNP is in a highly alternatively spliced gene but has no obvious function. It is in strong linkage disequilibrium (LD) with a number of other intronic SNPs, including rs17863182, which lies in the middle of a highly conserved region 3 kb upstream from another alternative exon.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f6d5/2986499/c7c39e7dee61/ejhg2008264f1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=2986499_ejhg2008264f1.jpg)

Association of GRM8 SNP rs7778604 with change in verbal memory (n=447; uncorrected P=0.00001). On average, the patients with the less common allele showed an improvement in memory, whereas the patients homozygous for the common allele did not.

#### Multiple hits

To identify SNPs with a general effect across cognitive domains, we first looked at the top hits associated with the principal component axis drawn from the change measures of all tests. The strongest association was with rs10954143, an intronic SNP in *GRM8* (*P*=0.0003) that was associated with many other intronic SNPs. This association seemed to be primarily driven by letter-number sequencing (*P*=0.001), continuous performance test (0.002) and mazes (*P*=0.03), so it is associated with a number of different domains. Similarly, the next strongest associated SNPs were two tightly linked SNPs in the 3′-UTR of *ADCY1,* rs2461127 and rs2471267 (each with *P*=0.0005), and these were also driven by associations in letter-number sequencing (*P*=0.01) and continuous performance (*P*=0.06), as well as verbal recall (*P*=0.01) and FAS (*P*=0.06).

We then looked at the uncorrected *P*-values to examine all hits in common across multiple cognitive tests (<0.05), not including PC1 ([Table 3](#tbl3)). No SNPs were associated with all nine tests. The SNP that was associated with most tests (5/9) was rs1011427, an intronic SNP in *HTR4*, and was in complete LD with another intronic SNP. *HTR4* codes for the serotonin receptor 4 subunit, and has been associated with enhanced cognition in animals.^15, 16, 17^ This SNP was associated with verbal fluency categories and letters, visuospatial working memory, verbal recall and digit symbol score ([Table 3](#tbl3)). As expected, it was also associated with the PC1 measure (*P*=0.006).

| SNP | Gene | Position | Number of hits | Categories (verbal fluency) | FAS (verbal fluency) | Letter/number sequencing | Visuospatial working memory | Verbal recall | Continuous performance test | Digit symbol | Mazes | WCST perseverative errors |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| rs1011427 | HTR4 | Intronic | 5 | 0.0441 | 0.0218 |   | 0.0143 | 0.0446 |   | 0.03995 |   |   |
| rs11766222 | ADCY1 | Intronic | 4 |   | 0.0275 | 0.0436 |   |   | 0.0313 |   | 0.0304 |   |
| rs339015 | GRM7 | Intronic | 4 | 0.0200 | 0.0172 | 0.0116 |   |   |   |   | 0.0447 |   |
| rs1390938 | SLC18A1 | Non-synonymous | 4 | 0.0045 |   | 0.0014 |   | 0.0398 |   | 0.01083 |   |   |
| rs1544938 | ADCY2 | Intronic | 3 |   | 0.0295 | 0.0116 |   |   | 0.0724 | 0.0837 |   | 0.0056 |
| rs919322 | ADCY2 | Intronic | 3 |   | 0.0222 | 0.0316 |   |   | 0.0862 |   |   | 0.0274 |
| rs2067477 | CHRM1 | Intronic | 3 |   | 0.0612 | 0.0238 |   | 0.0442 |   | 0.01195 | 0.0897 |   |
| rs3920243 | CHRNA7 | Intronic | 3 |   |   | 0.0762 |   | 0.0327 | 0.0495 |   |   | 0.0017 |
| rs965435 | CHRNA7 | Upstream | 3 |   |   | 0.0103 | 0.0103 |   |   |   |   | 0.0352 |
| rs129968 | CREBBP | Intronic | 3 |   |   |   |   | 0.0211 | 0.0033 | 0.009104 |   |   |
| rs6539460 | DAO | Intronic | 3 |   |   | 0.0259 | 0.0150 | 0.0219 |   |   |   |   |
| rs324029 | DRD3 | Intronic | 3 |   |   |   | 0.0164 | 0.0446 |   |   |   | 0.0239 |
| rs3734120 | HTR4 | Downstream | 3 |   |   | 0.0427 |   |   | 0.0448 |   | 0.0066 |   |
| rs4934292 | GLUD1 | Intronic | 3 |   | 0.0303 |   |   |   | 0.0415 | 0.03173 |   |   |
| rs10512287 | GRIN3A | Intronic | 3 | 0.0236 | 0.0604 |   |   |   | 0.0230 | 0.01025 |   |   |
| rs1323423 | GRIN3A | Intronic | 3 | 0.0065 | 0.0387 |   |   |   |   | 0.03957 |   |   |
| rs1323426 | GRIN3A | Intronic | 3 | 0.0065 | 0.0387 |   |   |   |   | 0.03957 |   |   |
| rs1323427 | GRIN3A | Intronic | 3 | 0.0134 |   | 0.0004 |   |   | 0.0455 |   |   |   |
| rs1572560 | GRIN3A | Intronic | 3 | 0.0056 | 0.0141 |   |   |   |   | 0.03631 |   |   |
| rs2485523 | GRIN3A | Intronic | 3 | 0.0044 | 0.0152 |   |   |   |   | 0.04913 |   |   |
| rs163326 | GRM7 | Intronic | 3 | 0.0590 | 0.0292 | 0.0020 |   |   |   |   | 0.0040 |   |
| rs756084 | GRM7 | Intronic | 3 |   |   |   | 0.0090 |   | 0.0489 |   | 0.0369 |   |
| rs10954143 | GRM8 | Intronic | 3 |   |   | 0.0010 |   |   | 0.0018 |   | 0.0289 |   |
| rs2237801 | GRM8 | Intronic | 3 |   |   |   |   | 0.0411 | 0.0267 |   | 0.0381 |   |
| rs1187362 | NTRK2 | Intronic | 3 |   |   | 0.0184 |   |   | 0.0384 |   |   | 0.0109 |
| rs3776572 | SLC1A3 | Intronic | 3 | 0.0115 |   | 0.0447 |   | 0.0279 |   |   |   |   |

Table Caption: Highlighted in bold are P-values below 0.005, in light gray are borderline hits 0.1>x>0.05.

Other SNPs of particular interest include an intronic SNP in *ADCY1*, rs11766222 (PC1, *P*=0.002). *ADCY1* codes for the neurospecific type I adenylyl cyclase, involved in hippocampal long-term potentiation and certain forms of learning and memory. This SNP is in complete LD (*r*^2^=1) with several other SNPs surrounding exons 12, 13 and 14, and associates with verbal fluency (letters), the auditory letter-number test, the continuous performance test and mazes. Also, of interest was rs1390938 (The136Ile), a non-synonymous coding SNP in *SLC18A1*, the gene that codes for the vesicular monoamine transporter 1 (PC1, *P*=0.008). This SNP has previously been associated with bipolar disorder^18^ and in this study associated with verbal fluency (categories), the auditory letter-number test, verbal recall and digit symbol. All SNPs that associated with three or more of the cognitive phenotypes are displayed in [Table 3](#tbl3).

#### Top hits

We next examined the strongest cognitive associations. The lowest *P*-value was for rs17866959, an intronic SNP in *GRM8* that was present only in African-American patients and associated with change in Wisconsin card sorting test (WCST) score. The *P*-value for association with change in WCST errors was *P*=8.3 × 10^−6^, and considering only African-Americans, the *P-*value was 2.4 × 10^−6^. Those patients carrying the variant allele all declined in WCST performance except one participant whose score did not change; however, those participants not carrying this variant tended to slightly improve in performance with treatment.

The next two strongest hits, and the only others with *P*<10^−4^, were both intronic SNPs in *GRM7* that were associated with a change in visuospatial memory. The SNPs are rs7627369 and rs3864075, and they are in partial LD with each other (*r*^2^=0.25 in this data set). Those individuals, both African-American and European-American, that carried the rare variant showed a stronger improvement in score across the 8 weeks of treatment (eg, rs7627369 homozygotes carrying the common variant made an average of 0.07 fewer errors *versus* those carrying the rare variant, who made 0.90 fewer errors after 8 weeks of treatment). SNP rs7627396 also associates with change in PANSS negative (*P*=0.0002) and PANSS general psychopathology (*P*=0.0004) scores.

We annotated the list of cognitive hits (<0.05) using WGAviewer,^19^ and some of the SNPs seemed to be of particular note due to their position or the position of associated SNPs. SNPs that had a *P*-value <0.001, and also SNPs that had a *P*-value <0.01 and were located in a coding or otherwise putatively functional region of the gene, are displayed in [Table 4](#tbl4). All SNPs with a *P*-value of association with a cognitive phenotype <0.05, along with their annotation, are displayed in Supplementary Results Table S1.

| Phenotype | SNP | No. of patients | P-value | Gene | Gene product | Position |
| --- | --- | --- | --- | --- | --- | --- |
| WCST perseverative errors | rs17866959 | 421 | 8.27E−06 | GRM8 | Metabotropic glutamate receptor 8 | Intronic |
| Verbal recall | rs7778604 | 447 | 1.00E−05 | GRM8 | Metabotropic glutamate receptor 8 | Intronic |
| Visuospatial working memory | rs7627369 | 402 | 2.90E−05 | GRM7 | Metabotropic glutamate receptor 7 | Intronic |
| Visuospatial working memory | rs3864075 | 402 | 9.46E−05 | GRM7 | Metabotropic glutamate receptor 7 | Intronic |
| Digit symbol | rs1491850 | 447 | 0.0001 | BDNF | Brain-derived neurotrophic factor | Intergenic |
| Continuous performance | rs779760 | 381 | 0.0002 | GRM7 | Metabotropic glutamate receptor 7 | Intronic |
| Categories (verbal fluency) | rs949731 | 447 | 0.0003 | GRM7 | Metabotropic glutamate receptor 7 | Intronic |
| Change PC1 | rs10954143 | 447 | 0.0003 | GRM8 | Metabotropic glutamate receptor 8 | Intronic |
| Verbal recall | rs17867741 | 447 | 0.0004 | GRM8 | Metabotropic glutamate receptor 8 | Intronic |
| Verbal recall | rs11563409 | 447 | 0.0004 | GRM8 | Metabotropic glutamate receptor 8 | Intronic |
| Letter/number sequencing | rs1323427 | 442 | 0.0004 | GRIN3A | NMDA receptor 3A | Intronic |
| WCST perseverative errors | rs7972662 | 421 | 0.0005 | GRIN2B | NMDA receptor 2B | Intronic |
| Change PC1 | rs2461127 | 447 | 0.0005 | ADCY1 | Adenylate cyclase 1 | 3′-UTR |
| Change PC1 | rs2471267 | 447 | 0.0005 | ADCY1 | Adenylate cyclase 1 | 3′-UTR |
| WCST perseverative errors | rs10488597 | 421 | 0.0006 | CHRM2 | Muscarinic receptor 2 | Intronic |
| Verbal recall | rs10512289 | 447 | 0.0006 | GRIN3A | NMDA receptor 3A | Intergenic |
| Mazes | rs1361957 | 439 | 0.0007 | GRM8 | Metabotropic glutamate receptor 8 | Intronic |
| WCST perseverative errors | rs2228703 | 421 | 0.0008 | CACNG2 | Stargazin | Intergenic |
| Continuous performance | rs1361956 | 381 | 0.0009 | GRM8 | Metabotropic glutamate receptor 8 | Intronic |
| Change PC1 | rs6774660 | 447 | 0.0009 | GRM7 | Metabotropic glutamate receptor 7 | Intronic |
| Verbal recall | rs11563780 | 447 | 0.0009 | GRM8 | Metabotropic glutamate receptor 8 | Intronic |
| FAS (verbal fluency) | rs3824519 | 447 | 0.0009 | NTRK2 | Neurotrophic tyrosine kinase receptor 2 | Intronic |
| Mazes | rs10455255 | 439 | 0.001 | RIMS1 | RAB3A-interacting molecule 1 | 3′-UTR |
| Letter/number sequencing | rs1390938 | 441 | 0.001 | SLC18A1 | Vesicular monoamine transporter 1 | Non-synonymous |
| FAS (verbal fluency) | rs707176 | 447 | 0.002 | GRIA1 | NMDA receptor subunit 1 | Synonymous |
| Categories (verbal fluency) | rs1948 | 447 | 0.002 | CHRNB4 | Neuronal nicotinic receptor B4 | 3′-UTR |
| Change PC1 | rs2067477 | 447 | 0.002 | CHRM1 | Muscarinic acetylcholine receptor 1 | Synonymous |
| WCST perseverative errors | rs11575542 | 421 | 0.003 | DDC | Dopa decarboxylase | Non-synonymous |
| WCST perseverative errors | rs3744959 | 421 | 0.003 | SYT4 | Synaptotagmin 4 | 3′-UTR |
| Categories (verbal fluency) | rs3743075 | 447 | 0.003 | CHRNA3 | Neuronal nicotinic receptor A3 | Synonymous |
| Letter/number sequencing | rs942142 | 442 | 0.004 | GRIN3A | NMDA receptor 3A | Synonymous |
| Letter/number sequencing | rs10512285 | 439 | 0.004 | GRIN3A | NMDA receptor 3A | Synonymous |
| Change PC1 | rs10249706 | 447 | 0.004 | ADCY1 | Adenylate cyclase 1 | 3′-UTR |
| Verbal recall | rs3829210 | 447 | 0.004 | ADCY8 | Adenylate cyclase 8 | 5′-UTR |
| Auditory letter-number test | rs10989589 | 442 | 0.004 | GRIN3A | NMDA receptor 3A | Frameshift |
| Digit symbol | rs10225980 | 447 | 0.004 | ADCY1 | Adenylate cyclase 1 | 3′-UTR |
| Verbal recall | rs7301328 | 447 | 0.004 | GRIN2B | NMDA receptor 2B | Synonymous |
| Categories (verbal fluency) | rs1390938 | 446 | 0.005 | SLC18A1 | Vesicular monoamine transporter 1 | Non-synonymous |
| Continuous performance | rs2073440 | 381 | 0.005 | HDC | Histidine decarboxylase | Non-synonymous |
| Change PC1 | rs10225980 | 447 | 0.005 | ADCY1 | Adenylate cyclase 1 | 3′-UTR |
| FAS (verbal fluency) | rs3743075 | 447 | 0.006 | CHRNA3 | Neuronal nicotinic receptor A3 | Synonymous |
| Digit symbol | rs10249706 | 447 | 0.006 | ADCY1 | Adenylate cyclase 1 | 3′-UTR |
| FAS (verbal fluency) | rs1549521 | 447 | 0.006 | HDC | Histidine decarboxylase | Synonymous |
| FAS (verbal fluency) | rs2273689 | 447 | 0.006 | SLC1A2 | Glutamate transporter 1 | Splice site, intronic |
| Categories (verbal fluency) | rs3134942 | 447 | 0.006 | NOTCH4 | Notch4 | Synonymous |
| WCST perseverative errors | rs11692815 | 421 | 0.006 | PPP3R1 | Calcineurin b, type 1 | Synonymous |
| Verbal recall | rs2471267 | 447 | 0.007 | ADCY1 | Adenylate cyclase 1 | 3′-UTR |
| Mazes | rs891398 | 439 | 0.007 | CHRNA2 | Neuronal nicotinic receptor A2 | Non-synonymous |
| Change PC1 | rs1390938 | 446 | 0.008 | SLC18A1 | Vesicular monoamine transporter 1 | Non-synonymous |
| Verbal recall | rs6280 | 447 | 0.008 | DRD3 | Dopamine receptor 3 | Non-synonymous |
| WCST perseverative errors | rs3820594 | 421 | 0.008 | SYT11 | Synaptotagmin 11 | 5′-UTR |
| Visuospatial working memory | rs10022491 | 402 | 0.009 | CHRNA9 | Neuronal nicotinic receptor A9 | Synonymous |
| Change PC1 | rs11575542 | 447 | 0.009 | DDC | Dopa Decarboxylase | Non-synonymous |
| Letter/number sequencing | rs2471267 | 442 | 0.009 | ADCY1 | Adenylate cyclase 1 | 3′-UTR |
| Letter/number sequencing | rs1549521 | 442 | 0.009 | HDC | Histidine decarboxylase | Synonymous |
| Change PC1 | rs7030238 | 446 | 0.009 | GRIN3A | NMDA receptor 3A | 3′-UTR |
#### PANSS

No PANSS association stood up to correction for all tested SNPs. All *P*-values described are uncorrected.

#### Multiple hits

To see if any SNP is associated with an overall improvement in symptoms, we looked first at SNPs that associated with PANSS total, and thus associated with multiple PANSS measures. [Table 5](#tbl5) shows all 22 SNPS that are associated with PANSS total score with an uncorrected *P*-value of <0.01, along with their *P*-value for association with each of the three individual PANSS scores. The strongest PANSS total association was rs7627369, an intronic SNP in the metabotropic glutamate receptor gene *GRM7* that is of low frequency (4%) in both European- and African-Americans (and is also associated with visuospatial working memory, above). It is located in intron 8 of the gene and is associated with other SNPs in introns 8 and 9. Altogether, there were 19 participants who carried the rare allele at this locus who also had change scores for PANSS available at 3 months on the phase I drug. For each PANSS category, those with the rare allele showed greater improvement in score across the 3 months. On average, patients carrying the rare allele dropped 17.3 PANSS total points in comparison with 6.5 points dropped by patients homozygous for the common allele.

|   |   |   |   | PANSS P-value |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- |
| SNP | Gene | Gene product | Position | Positive | Negative | General psychopathology | Total |
| rs3828628 | ADCY2 | Adenylate cyclase 2 | Intronic | 0.01 | 0.009 | 0.06 | 0.004 |
| rs2241695 | CAMK2A | α-CaM kinase II | Intronic | 0.02 | 0.001 | 0.007 | 0.0006 |
| rs869191 | CAMK2A | α-CaM kinase II | Intronic | 0.03 | 0.01 | 0.002 | 0.0009 |
| rs4586 | CCL2 | Small inducible cytokine A2 | Synonymous | 0.01 | 0.007 | 0.02 | 0.001 |
| rs4795893 | CCL2 | Small inducible cytokine A2 | 7.8 kb upstream | 0.006 | 0.1 | 0.02 | 0.004 |
| rs7164043 | CHRNA7 | Neuronal nicotinic receptor A7 | Intronic | 0.005 | 0.02 | 0.14 | 0.007 |
| rs6808291 | DRD3 | Dopamine receptor D3 | 9 kb upstream | 0.02 | 0.001 | 0.007 | 0.0006 |
| rs2867383 | DRD5 | Dopamine receptor D5 | 2.3 kb downstream | 0.12 | 0.02 | 0.006 | 0.004 |
| rs1654670 | GRIN2D | NMDA receptor 2D | Intronic | 0.004 | 0.03 | 0.002 | 0.0005 |
| rs362835 | GRM1 | Metabotropic glutamate receptor 1 | Intronic | 0.003 | 0.17 | 0.02 | 0.004 |
| rs12275483 | GRM5 | Metabotropic glutamate receptor 5 | Intronic | 0.02 | 0.03 | 0.06 | 0.009 |
| rs7733067 | GRM6 | Metabotropic glutamate receptor 6 | 3.6 kb upstream | 0.006 | 0.15 | 0.03 | 0.007 |
| rs7627369 | GRM7 | Metabotropic glutamate receptor 7 | Intronic | 0.09 | 0.0002 | 0.0004 | 0.0001 |
| rs6774660 | GRM7 | Metabotropic glutamate receptor 7 | Intronic | 0.01 | 0.008 | 0.008 | 0.0009 |
| rs17869476 | GRM8 | Metabotropic glutamate receptor 8 | Intronic | 0.004 | 0.34 | 0.009 | 0.005 |
| rs1923882 | HTR2A | Serotonin receptor 2A | Intronic | 0.01 | 0.03 | 0.04 | 0.006 |
| rs10061244 | HTR4 | Serotonin receptor 4 | Intronic | 0.09 | 0.06 | 0.004 | 0.004 |
| rs1957893 | PRKCH | Protein kinase C eta | Intronic | 0.02 | 0.12 | 0.02 | 0.007 |
| rs8103212 | SLC1A6 | Excitatory amino-acid transporter 4 | Intronic | 0.006 | 0.04 | 0.02 | 0.002 |
| rs3746295 | SLC1A6 | Excitatory amino-acid transporter 4 | Synonymous | 0.03 | 0.04 | 0.006 | 0.002 |
| rs3803927 | SLC1A6 | Excitatory amino-acid transporter 4 | Intronic | 0.04 | 0.03 | 0.01 | 0.004 |

Table Caption: Light gray coloring indicates that the SNP did not associate significantly with that test. Bold text indicates that the two same-gene SNPs are in LD (r2≥0.5).

*SLC1A6*, a glutamate transporter gene, showed three associated SNPs but only two of the SNPs were in LD, indicating two independent hits in the same gene. One of the associated SNPs, rs3746295, is a synonymous coding SNP in the first exon.

There was another SNP located in a coding region that was associated with PANSS score. This was rs4586, in exon 2 of *CCL2* (aka *SCYA2*, *MCP1* and *MCAF*), the monocyte chemoattractant protein-1 gene. A promoter polymorphism in this gene (rs1024611) has earlier been associated with antipsychotic resistance.^20^ SNP rs4586 is in LD with rs1024611 (*D*′=1.00, *r*^2^=0.62); therefore, it is possible that the association detected could be driven by rs1024611, which is not directly genotyped in this study.

#### Top hits for PANSS negative, positive and general psychosis

No PANSS change hits were significant after correction for multiple testing. All of the strongest hits were in glutamate receptor genes. The overall top hit was rs7628369 (*GRM7* SNP, see above) associating with a change in PANSS negative score (*P*=0.0002), and the next strongest was the same SNP associating with PANSS general psychosis score. After this, the strongest association was with rs2106190, a *GRM8* SNP that was associated with change in positive PANSS score at *P*=0.0008. The next three strongest hits were all associated with change in PANSS negative score: rs2237547, in *GRM3*, *P*=0.0009; rs10512287, in *GRIN3A*, *P*=0.0010 and rs757656, in *GRM3, P*=0.0010. All other associations were above *P*=0.001. All PANSS change associations below 0.05, along with their locations and associated SNPs of interest, are shown in Supplementary Table S2.

### Discontinuation

#### Study-wide significant association

An SNP in intron 2 of *RIMS1*, rs502046, was associated with discontinuation from quetiapine due to inefficacy (raw *P*-value=0.00009; study-wide corrected *P*=0.021, *n*=129, odds ratio=3.1; [Figure 2](#fig2)). The SNP is in strong LD with a number of other SNPs in introns 2, 3 and 4 ([Figure 2](#fig2)). The patients, both European- and African-Americans, carrying the rare variant were more likely to continue with quetiapine treatment than those without (TT: 33/38 discontinued; CT: 36/62 discontinued; CC 13/29 discontinued). To confirm that the association was not caused by stratification beyond that accounted for by the first PCA axis, we repeated the analysis with 10 significant axes of ancestry included as covariates. Again, the resulting raw *P*-value was still strongly significant at *P*=0.00027, but when corrected by permutation for all SNPs, this becomes 0.31. This SNP was not associated with discontinuation of any of the other drugs.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f6d5/2986499/b9d42efa206e/ejhg2008264f2.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=2986499_ejhg2008264f2.jpg)

Association of RIMS1 SNP rs502046 with quetiapine discontinuation (n=129; uncorrected P=0.00009). The patients carrying the rarer variant were much less likely to discontinue than those without.

#### Top hits

All significant hits (uncorrected *P*<0.05), along with their genic position, are shown in Supplementary Table S3. None of the strongest hits for discontinuation from a drug due to inefficacy made any strong biological sense. The top hit for discontinuation from olanzapine (*n*=141) was an intronic SNP in *GRM7* that was associated with various other intronic SNPs within the gene, none of which had any obvious function (proximity to an exon, association with gene expression, location in known regulatory region). The top hit for discontinuation from perphenazine (*n*=106), the single typical antipsychotic tested in CATIE, was rs2756271, located 1.5 kb upstream of the start of the prion protein gene *PRNP*, and again the SNP showed no obvious function, nor was it associated with any SNPs closer to the gene. The top hit for discontinuation from risperidone (*n*=136) was rs10495449, an SNP just downstream from the muscarinic acetylcholine receptor gene *CHRM3.* The SNP shows no association with any other SNP within the gene and has no obvious function. Finally, the top hit for ziprasidone (*n*=60) was rs7614915, a synonymous coding SNP in exon 8 of the *GRM7* metabotropic glutamate receptor gene.

#### Weight gain

No weight-gain association stood up to correction for multiple testing. All *P*-values described are uncorrected.

#### Top hits

The strongest association with maximum weight gain during phase I was an intronic SNP, rs7164043, in the *α*7-nicotinic acetylcholine receptor subunit gene *CHRNA7* that was associated with quantitative weight gain *P*=0.0001. The SNP was also associated with change in PANSS total (*P*=0.006), positive (*P*=0.01), negative (*P*=0.03) and general (0.04) scores, change in visuospatial working memory (*P*=0.01) and change in verbal memory score (0.03). This SNP, however, is only present in African-American populations, with the exception of one patient self-described as ‘White alone' and identified as of European ancestry by Eigenstrat analysis. Restricting the association of weight to African-Americans increases the significance of the association, such that those carrying the rare variant (*n*=20) gained an average of 7.02% of their baseline weight, whereas those without the variant allele (*n*=176) gained, on average, only 0.17% of their baseline weight.

The strongest hit for the weight-gain case–control association test was an intronic SNP in the serotonin receptor gene *HTR2A*, rs1928042 (*P*=0.001). This SNP was not associated with the quantitative measure of weight gain, so it is possible that the effect is a false positive. All significant hits (uncorrected *P*<0.05) for weight gain are displayed in Supplementary Table S4, and the top hits plus others that are of interest due to their genic position are displayed in [Table 6](#tbl6).

| Test | SNP | P-value | Gene | Gene product | Type |
| --- | --- | --- | --- | --- | --- |
| Quantitative | rs7164043 | 0.0001 | CHRNA7 | Neuronal nicotinic receptor A7 | Intronic |
| Quantitative | rs2079731 | 0.0003 | ADCY9 | Adenylate cyclase 9 | Intronic |
| Quantitative | rs1934124 | 0.0004 | RIMS1 | RAB3A-interacting molecule 1 | Intronic |
| Case–control | rs1928042 | 0.0010 | HTR2A | Serotonin receptor 2A | Intronic |
| Case–control | rs2770292 | 0.0017 | HTR2A | Serotonin receptor 2A | Intronic |
| Quantitative | rs632994 | 0.0017 | HRH2 | Histamine receptor H2 | 7.6 kb downstream |
| Case–control | rs10505778 | 0.0019 | GRIN2B | NMDA receptor 2b | Intronic |
| Case–control | rs480409 | 0.0019 | GRM7 | Metabotropic glutamate receptor 7 | Intronic |
| Case–control | rs5320 | 0.006 | DBH | Dopamine β-hydroxylase | Non-synonymous |
| Case–control | rs8192591 | 0.006 | NOTCH4 | NOTCH 4 (drosophila homolog) | Non-synonymous |
| Case–control | rs1390939 | 0.007 | SLC18A1 | Vesicular monoamine transporter 1 | 5′-UTR |
| Quantitative | rs1805482 | 0.009 | GRIN2B | NMDA receptor 2B | Synonymous |
| Quantitative | rs11575553 | 0.014 | DDC | DOPA decarboxylase | 3′-UTR |
| Quantitative | rs1805522 | 0.02 | GRIN2B | NMDA receptor 2B | Synonymous |
| Case–control | rs1948 | 0.02 | CHRNB4 | Neuronal nicotinic receptor B4 | 3′-UTR |
| Case–control | rs11575553 | 0.02 | DDC | DOPA decarboxylase | 3′-UTR |
| Quantitative | rs326175 | 0.03 | ADCY2 | Adenylate cyclase 2 | 3′-UTR |
| Case–control | rs130003 | 0.03 | CREBBP | CREB-binding protein | Synonymous |
| Case–control | rs8178990 | 0.03 | CHAT | Choline acetyltransferase | Non-synonymous |
| Case–control | rs660652 | 0.03 | CHRNA3 | Neuronal nicotinic receptor, A3 | 3′-UTR |
| Quantitative | rs3746295 | 0.04 | SLC1A6 | Excitatory amino-acid transporter 4 | Synonymous |
| Quantitative | rs3749380 | 0.05 | GRM7 | Metabotropic glutamate receptor 7 | Synonymous |
| Case–control | rs2270641 | 0.05 | SLC18A1 | Vesicular monoamine transporter 1 | Non-synonymous |

Table Caption: All drugs included in a single analysis.

#### Genotype imputation analysis

Using tagging SNPs, we found strongly suggestive associations with different SNPs in the *GRM7* and *GRM8* genes with neurocognitive, PANSS and discontinuation phenotypes, and a strongly suggestive association of an *RIMS1* SNP with quetiapine discontinuation. As not all common variants in these genes have been genotyped, it is possible that there are ungenotyped SNPs that are more strongly associated with these phenotypes. We therefore imputed genotypes for all the HapMap SNPs for *GRM7*, *GRM8* and *RIMS1* using MaCH (Supplementary Methods). We first checked to see if there was a stronger association with discontinuation from quetiapine for any *RIMS1* SNP. We found that the genotyped SNP, rs502046, was the lowest associated SNP in this gene, even after examining associations with all permuted genotypes.

As *GRM7* and *GRM8* were associated particularly strongly with many phenotypes, we looked to see if imputed SNPs had stronger associations with any phenotype. For most phenotypes, the lowest *P*-value for an imputed SNP was of the same order of magnitude as that of the most strongly associated genotyped SNP. However, some phenotypes showed much stronger association with an imputed SNP; these are documented in Supplementary Table S1. None of the strongest associated phenotype detailed for *GRM7* and *GRM8* (above) improved when imputed SNPs were examined.

#### Previous findings

Where possible, we also checked to see if we were able to replicate other findings of SNPs determining antipsychotic response in schizophrenia. In many cases, this was not possible, as previously associated polymorphisms have been non-SNP variants, such as VNTRs or in/dels. In addition, some important SNPs failed genotyping quality control (see Supplementary Information). Previously associated SNPs that we successfully genotyped here include *COMT* val108/158met (rs4680), *DRD2* -241-A/G (rs1799978), *DRD3* Ser9Gly (rs6280) and *HTR2A* 102-T/C(rs6313). The *COMT* SNP has been associated with antipsychotic-related improvement in working memory and negative symptoms^21, 22^ and better treatment response^23^ (as well as other phenotypes not captured here). We found no significant associations of *COMT* val108/158met with any phenotype evaluated in this study. The *DRD2* SNP −241-A/G has been associated of a faster response to antipsychotic treatment,^24^ but again we saw no association with any treatment response measures at the time points measured (PANSS scores: 12 weeks; neurocognitive change: 8 weeks). The *DRD3* Ser9Gly SNP has earlier been associated with a better response to risperidone treatment in Chinese (carriers of ser allele better response^25^), worse response to atypical antipsychotic treatment in Caucasians (ser/ser worse than other genotypes^26^) and improvement in positive symptoms while on olanzapine.^27^ In this study, the SNP was associated with change in verbal memory at 8 weeks (uncorrected *P*=0.008), change in visuospatial memory at 8 weeks (uncorrected *P*=0.03) and discontinuation from risperidone due to inefficacy (uncorrected *P*=0.02, gly/gly less likely to discontinue due to inefficacy). Finally, the *HTR2A* 102-T/C SNP has been associated with better response to risperidone in Chinese (CC better response^28^) and with non-response to antipsychotic treatment in Caucasians (CC overrepresented in non-responders^29^). In this study, we saw only a very borderline association with change in verbal fluency (uncorrected *P*=0.05). In summary, although previously associated SNPs and related phenotypes were tested here, we could not provide strong support for the previous associations.

## Discussion

This study has identified suggestive associations that if confirmed could be of considerable clinical significance. Notable among these findings is a polymorphism that is strongly associated with discontinuation of quetiapine (rs502046, in *RIMS1*). When randomized onto quetiapine in phase 1 of the CATIE trial, a minority of patients stayed on the drug (35%) but most discontinued. We have shown that those patients who continued on the drug, presumably responding well, carried a rare variant at the associated locus, and for those patients homozygous for the rare allele, the continuation rate was 55%. If this association is confirmed, quetiapine would likely emerge as a first-line choice for the 22% of patients who carry the favorable genotype. Attempted replication therefore of this association is a priority.

The second study-wide significant association was between a *GRM8* variant (rs7778604) and improvement in verbal memory over 8 weeks of antipsychotic treatment. No measure of cognitive improvement, including verbal memory, differed between the medications used in this trial,^30^ nor did the genetic association with rs7778604 differ significantly between treatment groups. This variant, therefore, appears to convey a general increased responsiveness to the cognitive benefits of antipsychotic treatment. An improved understanding of the mechanisms of this effect could be useful in developing adjunctive medications for improving cognitive response to antipsychotic drugs in schizophrenia. This may have great relevance to the study of cognitive deficits in schizophrenia. Numerous pharmaceutical industry drug-development programs and government efforts, such as the NIH-sponsored project Measurement and Treatment Research to Improve Cognition in Schizophrenia,^31^ are currently devoted to finding pharmacologic compounds or behavioral strategies that may improve cognitive impairment in schizophrenia.

On balance, these results, like most findings in psychiatric genetics, remain equivocal. There was no single association that stood up to correction for all SNPs and all phenotypes analyzed. However, given the study size and the number of pharmacogenetic questions addressed, there is clearly potential for type II error. This could have been reduced in part by matching particular candidate genes up to the phenotypes that they could most plausibly affect, rather than testing all genes against all phenotypes. However, with the current state of knowledge, we felt that such matching would have to be based at least in part on guesswork and consequently opted for the broader approach presented here. For these reasons, we felt that it was important to make available all associations with uncorrected *P*-values <0.05, for potential targeted replication by other interested researchers. It should also be noted that as this study was limited to common SNPs there could be unrepresented polymorphisms in these genes with stronger effects on the pharmacogenetic phenotypes than we have found here such as rare SNPs, large structural variants recently implicated in schizophrenia,^32, 33^ in-dels or VNTRs.

In addition, it is important to appreciate that the CATIE study was not designed with genetic hypotheses in mind, and it is not well powered to test many of the most pressing pharmacogenetic questions. Clinical studies such as CATIE could be an invaluable resource for conducting genetic studies that have direct relevant to patient populations. However, for these studies to be effective, it is an absolute requirement that the genetic component is considered *a priori* and a set of genetic hypotheses established in advance to ensure sufficient power to explore the relevant questions and reduce the potential for data mining. All study patients should be encouraged to take part in the genetic analyses both to maximize power for the genetic hypotheses and so that the genetic component accurately reflects the overall sample composition. In addition, it is important that psychiatric practitioners facilitate complementary collections in the regular care setting that allow the same hypotheses to be tested and replication analyses to be performed. This is the only way to realize the potential of these trials for advancing personalized medicine.

By contemporary standards for association genetics, nothing in psychiatric genetics has yet qualified as a definitive association.^34^ A key question for the field is what to do with its results that do not pass contemporary standards. Although we agree that suggestive results need to be widely available, it is difficult to see any advantage in the continued publication of only slightly suggestive results individually, polymorphism by polymorphism and phenotype by phenotype. In many cases, the quality of a gene as a candidate for the condition has made up for very modest *P*-values to motivate separate publication.^14, 35, 36, 37, 38, 39^ Large-scale genotyping being carried out here and elsewhere, however, makes clear such an approach is untenable. In our study, there are at least 856 and 368 genetic associations with raw *P*-values superior to the 0.049 and 0.015 reported for *RGS4* SNPs rs2661319 and rs2842030 published in a recent paper^35^ (see Supplementary Information Tables, noting that the figures 856 and 368 for nominal genetic associations include only the strongest associated SNP per gene to prevent inflation of estimates by LD). Each of these is in genes that can be argued to be plausibly involved (as they were selected as candidate genes). Clearly, it would be both unnecessary and misleading to publish these associations one by one in individual research papers. These considerations demonstrate that the earlier practice in psychiatric genetics of individual reports for suggestive candidate genes is no longer appropriate given the scale of genetic data being generated, including for large numbers of suggestive candidate genes.

The alternative we propose is to report all associations resulting from large-scale screens that both explicitly and implicitly convey the magnitude of testing actually taking place in the community and allow ready interpretation and comparison of the results. To facilitate this effort, we have included as Supplementary Information all raw *P*-values (<0.05) in a file format that contains the phenotype, SNP and *P*-value. These results can therefore be easily interrogated by any software tool for comparison with other data sets. With the increase in larger scale genetic studies, and in particular, the increasing prevalence of whole-genome analyses, the risk of producing type II errors when correcting for thousands of SNPs increases exponentially. Many SNPs with real effects will not stand up to correction for study-wide significance. This mandates the use of software programs such as WGAviewer^19^ to allow the examination and interpretation of results in their genomic context, and necessitates data sharing between studies so that replications of SNPs that may not come out in the top *P*-values can be detected and followed up.

## Acknowledgments

We thank Kevin Shianna and team for genotype data, Jeff Dawson for management of the phenotype database and Mike Weale for tagging. We are indebted to the patients and families who took part in this study. We also thank Sonia Davies for her help in providing and interpreting the CATIE phenotype data. The CATIE project was supported by NIMH contract N01 MH90001.

## Footnotes

## Supplementary Material

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Lieberman JA, Stroup TS, McEvoy JP, et al. Effectiveness of antipsychotic drugs in patients with chronic schizophrenia. N Engl J Med. 2005;353:1209–1223. doi: 10.1056/NEJMoa051688.  [DOI](https://doi.org/10.1056/NEJMoa051688) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16172203/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Effectiveness%20of%20antipsychotic%20drugs%20in%20patients%20with%20chronic%20schizophrenia&author=JA%20Lieberman&author=TS%20Stroup&author=JP%20McEvoy&volume=353&publication_year=2005&pages=1209-1223&pmid=16172203&doi=10.1056/NEJMoa051688&)

2. Bakker PR, van Harten PN, van Os J. Antipsychotic-induced tardive dyskinesia and polymorphic variations in COMT, DRD2, CYP1A2 and MnSOD genes: a meta-analysis of pharmacogenetic interactions. Mol Psychiatry. 2008;13:544–556. doi: 10.1038/sj.mp.4002142.  [DOI](https://doi.org/10.1038/sj.mp.4002142) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18180754/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Psychiatry&title=Antipsychotic-induced%20tardive%20dyskinesia%20and%20polymorphic%20variations%20in%20COMT,%20DRD2,%20CYP1A2%20and%20MnSOD%20genes:%20a%20meta-analysis%20of%20pharmacogenetic%20interactions&author=PR%20Bakker&author=PN%20van%20Harten&author=J%20van%20Os&volume=13&publication_year=2008&pages=544-556&pmid=18180754&doi=10.1038/sj.mp.4002142&)

3. Reynolds GP. The impact of pharmacogenetics on the development and use of antipsychotic drugs. Drug Discov Today. 2007;12:953–959. doi: 10.1016/j.drudis.2007.07.018.  [DOI](https://doi.org/10.1016/j.drudis.2007.07.018) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17993414/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Discov%20Today&title=The%20impact%20of%20pharmacogenetics%20on%20the%20development%20and%20use%20of%20antipsychotic%20drugs&author=GP%20Reynolds&volume=12&publication_year=2007&pages=953-959&pmid=17993414&doi=10.1016/j.drudis.2007.07.018&)

4. Bakker PR, van Harten PN, van Os J. Antipsychotic-induced tardive dyskinesia and the Ser9Gly polymorphism in the DRD3 gene: a meta analysis. Schizophr Res. 2006;83:185–192. doi: 10.1016/j.schres.2006.01.010.  [DOI](https://doi.org/10.1016/j.schres.2006.01.010) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16513329/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Schizophr%20Res&title=Antipsychotic-induced%20tardive%20dyskinesia%20and%20the%20Ser9Gly%20polymorphism%20in%20the%20DRD3%20gene:%20a%20meta%20analysis&author=PR%20Bakker&author=PN%20van%20Harten&author=J%20van%20Os&volume=83&publication_year=2006&pages=185-192&pmid=16513329&doi=10.1016/j.schres.2006.01.010&)

5. Arranz MJ, Kapur S. Pharmacogenetics in psychiatry: are we ready for widespread clinical use. Schizophr Bull. 2008;34:1130–1144. doi: 10.1093/schbul/sbn114.  [DOI](https://doi.org/10.1093/schbul/sbn114) | [PMC free article](/articles/PMC2632495/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18753306/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Schizophr%20Bull&title=Pharmacogenetics%20in%20psychiatry:%20are%20we%20ready%20for%20widespread%20clinical%20use&author=MJ%20Arranz&author=S%20Kapur&volume=34&publication_year=2008&pages=1130-1144&pmid=18753306&doi=10.1093/schbul/sbn114&)

6. De Luca V, Mueller DJ, de Bartolomeis A, Kennedy JL. Association of the HTR2C gene and antipsychotic induced weight gain: a meta-analysis. Int J Neuropsychopharmacol. 2007;10:697–704. doi: 10.1017/S1461145707007547.  [DOI](https://doi.org/10.1017/S1461145707007547) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17291373/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Neuropsychopharmacol&title=Association%20of%20the%20HTR2C%20gene%20and%20antipsychotic%20induced%20weight%20gain:%20a%20meta-analysis&author=V%20De%20Luca&author=DJ%20Mueller&author=A%20de%20Bartolomeis&author=JL%20Kennedy&volume=10&publication_year=2007&pages=697-704&pmid=17291373&doi=10.1017/S1461145707007547&)

7. Athanasiou MC, Malhotra AK, Xu C, Stephens JC. Discovery and utilization of haplotypes for pharmacogenetic studies of psychotropic drug response. Psychiatr Genet. 2002;12:89–96. doi: 10.1097/00041444-200206000-00006.  [DOI](https://doi.org/10.1097/00041444-200206000-00006) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12131761/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Psychiatr%20Genet&title=Discovery%20and%20utilization%20of%20haplotypes%20for%20pharmacogenetic%20studies%20of%20psychotropic%20drug%20response&author=MC%20Athanasiou&author=AK%20Malhotra&author=C%20Xu&author=JC%20Stephens&volume=12&publication_year=2002&pages=89-96&pmid=12131761&doi=10.1097/00041444-200206000-00006&)

8. Arranz MJ, Munro J, Birkett J, et al. Pharmacogenetic prediction of clozapine response. Lancet. 2000;355:1615–1616. doi: 10.1016/s0140-6736(00)02221-2.  [DOI](https://doi.org/10.1016/s0140-6736(00)02221-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10821369/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Pharmacogenetic%20prediction%20of%20clozapine%20response&author=MJ%20Arranz&author=J%20Munro&author=J%20Birkett&volume=355&publication_year=2000&pages=1615-1616&pmid=10821369&doi=10.1016/s0140-6736(00)02221-2&)

9. de Leon J, Armstrong SC, Cozza KL. Clinical guidelines for psychiatrists for the use of pharmacogenetic testing for CYP450 2D6 and CYP450 2C19. Psychosomatics. 2006;47:75–85. doi: 10.1176/appi.psy.47.1.75.  [DOI](https://doi.org/10.1176/appi.psy.47.1.75) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16384813/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Psychosomatics&title=Clinical%20guidelines%20for%20psychiatrists%20for%20the%20use%20of%20pharmacogenetic%20testing%20for%20CYP450%202D6%20and%20CYP450%202C19&author=J%20de%20Leon&author=SC%20Armstrong&author=KL%20Cozza&volume=47&publication_year=2006&pages=75-85&pmid=16384813&doi=10.1176/appi.psy.47.1.75&)

10. Grossman I, Sullivan PF, Walley N, et al. Genetic determinants of variable metabolism have little impact on the clinical use of leading antipsychotics in the CATIE study. Genet Med. 2008;10:720–729. doi: 10.1097/GIM.0b013e3181863239.  [DOI](https://doi.org/10.1097/GIM.0b013e3181863239) | [PMC free article](/articles/PMC3697113/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18813134/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genet%20Med&title=Genetic%20determinants%20of%20variable%20metabolism%20have%20little%20impact%20on%20the%20clinical%20use%20of%20leading%20antipsychotics%20in%20the%20CATIE%20study&author=I%20Grossman&author=PF%20Sullivan&author=N%20Walley&volume=10&publication_year=2008&pages=720-729&pmid=18813134&doi=10.1097/GIM.0b013e3181863239&)

11. Grossman I. Routine pharmacogenetic testing in clinical practice: dream or reality. Pharmacogenomics. 2007;8:1449–1459. doi: 10.2217/14622416.8.10.1449.  [DOI](https://doi.org/10.2217/14622416.8.10.1449) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17979518/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Routine%20pharmacogenetic%20testing%20in%20clinical%20practice:%20dream%20or%20reality&author=I%20Grossman&volume=8&publication_year=2007&pages=1449-1459&pmid=17979518&doi=10.2217/14622416.8.10.1449&)

12. Green MF, Kern RS, Heaton RK. Longitudinal studies of cognition and functional outcome in schizophrenia: implications for MATRICS. Schizophr Res. 2004;72:41. doi: 10.1016/j.schres.2004.09.009.  [DOI](https://doi.org/10.1016/j.schres.2004.09.009) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15531406/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Schizophr%20Res&title=Longitudinal%20studies%20of%20cognition%20and%20functional%20outcome%20in%20schizophrenia:%20implications%20for%20MATRICS&author=MF%20Green&author=RS%20Kern&author=RK%20Heaton&volume=72&publication_year=2004&pages=41&pmid=15531406&doi=10.1016/j.schres.2004.09.009&)

13. Braff DL, Greenwood TA, Swerdlow NR, Light GA, Schork NJ. Advances in endophenotyping schizophrenia. World Psychiatry. 2008;7:11–18. doi: 10.1002/j.2051-5545.2008.tb00140.x.  [DOI](https://doi.org/10.1002/j.2051-5545.2008.tb00140.x) | [PMC free article](/articles/PMC2359727/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18458787/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=World%20Psychiatry&title=Advances%20in%20endophenotyping%20schizophrenia&author=DL%20Braff&author=TA%20Greenwood&author=NR%20Swerdlow&author=GA%20Light&author=NJ%20Schork&volume=7&publication_year=2008&pages=11-18&pmid=18458787&doi=10.1002/j.2051-5545.2008.tb00140.x&)

14. Sullivan PF, Keefe RS, Lange LA, et al. NCAM1 and neurocognition in schizophrenia. Biol Psychiatry. 2007;61:902–910. doi: 10.1016/j.biopsych.2006.07.036.  [DOI](https://doi.org/10.1016/j.biopsych.2006.07.036) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17161382/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biol%20Psychiatry&title=NCAM1%20and%20neurocognition%20in%20schizophrenia&author=PF%20Sullivan&author=RS%20Keefe&author=LA%20Lange&volume=61&publication_year=2007&pages=902-910&pmid=17161382&doi=10.1016/j.biopsych.2006.07.036&)

15. Mohler EG, Shacham S, Noiman S, et al. VRX-03011, a novel 5-HT4 agonist, enhances memory and hippocampal acetylcholine efflux. Neuropharmacology. 2007;53:563–573. doi: 10.1016/j.neuropharm.2007.06.016.  [DOI](https://doi.org/10.1016/j.neuropharm.2007.06.016) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17692343/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neuropharmacology&title=VRX-03011,%20a%20novel%205-HT4%20agonist,%20enhances%20memory%20and%20hippocampal%20acetylcholine%20efflux&author=EG%20Mohler&author=S%20Shacham&author=S%20Noiman&volume=53&publication_year=2007&pages=563-573&pmid=17692343&doi=10.1016/j.neuropharm.2007.06.016&)

16. Micale V, Leggio GM, Mazzola C, Drago F. Cognitive effects of SL65.0155, a serotonin 5-HT4 receptor partial agonist, in animal models of amnesia. Brain Res. 2006;1121:207–215. doi: 10.1016/j.brainres.2006.08.108.  [DOI](https://doi.org/10.1016/j.brainres.2006.08.108) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17011531/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Brain%20Res&title=Cognitive%20effects%20of%20SL65.0155,%20a%20serotonin%205-HT4%20receptor%20partial%20agonist,%20in%20animal%20models%20of%20amnesia&author=V%20Micale&author=GM%20Leggio&author=C%20Mazzola&author=F%20Drago&volume=1121&publication_year=2006&pages=207-215&pmid=17011531&doi=10.1016/j.brainres.2006.08.108&)

17. Manuel-Apolinar L, Rocha L, Pascoe D, Castillo E, Castillo C, Meneses A. Modifications of 5-HT4 receptor expression in rat brain during memory consolidation. Brain Res. 2005;1042:73–81. doi: 10.1016/j.brainres.2005.02.020.  [DOI](https://doi.org/10.1016/j.brainres.2005.02.020) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15823255/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Brain%20Res&title=Modifications%20of%205-HT4%20receptor%20expression%20in%20rat%20brain%20during%20memory%20consolidation&author=L%20Manuel-Apolinar&author=L%20Rocha&author=D%20Pascoe&author=E%20Castillo&author=C%20Castillo&volume=1042&publication_year=2005&pages=73-81&pmid=15823255&doi=10.1016/j.brainres.2005.02.020&)

18. Lohoff FW, Dahl JP, Ferraro TN, et al. Variations in the vesicular monoamine transporter 1 gene (VMAT1/SLC18A1) are associated with bipolar i disorder. Neuropsychopharmacology. 2006;31:2739–2747. doi: 10.1038/sj.npp.1301196.  [DOI](https://doi.org/10.1038/sj.npp.1301196) | [PMC free article](/articles/PMC2507868/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16936705/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neuropsychopharmacology&title=Variations%20in%20the%20vesicular%20monoamine%20transporter%201%20gene%20(VMAT1/SLC18A1)%20are%20associated%20with%20bipolar%20i%20disorder&author=FW%20Lohoff&author=JP%20Dahl&author=TN%20Ferraro&volume=31&publication_year=2006&pages=2739-2747&pmid=16936705&doi=10.1038/sj.npp.1301196&)

19. Ge D, Zhang K, Need AC, et al. WGAViewer: software for genomic annotation of whole genome association studies. Genome Res. 2008;18:640–643. doi: 10.1101/gr.071571.107.  [DOI](https://doi.org/10.1101/gr.071571.107) | [PMC free article](/articles/PMC2279251/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18256235/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res&title=WGAViewer:%20software%20for%20genomic%20annotation%20of%20whole%20genome%20association%20studies&author=D%20Ge&author=K%20Zhang&author=AC%20Need&volume=18&publication_year=2008&pages=640-643&pmid=18256235&doi=10.1101/gr.071571.107&)

20. Mundo E, Altamura AC, Vismara S, et al. MCP-1 gene (SCYA2) and schizophrenia: a case–control association study. Am J Med Genet B Neuropsychiatr Genet. 2005;132:1–4. doi: 10.1002/ajmg.b.30100.  [DOI](https://doi.org/10.1002/ajmg.b.30100) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15389752/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Med%20Genet%20B%20Neuropsychiatr%20Genet&title=MCP-1%20gene%20(SCYA2)%20and%20schizophrenia:%20a%20case%E2%80%93control%20association%20study&author=E%20Mundo&author=AC%20Altamura&author=S%20Vismara&volume=132&publication_year=2005&pages=1-4&pmid=15389752&doi=10.1002/ajmg.b.30100&)

21. Weickert TW, Goldberg TE, Mishara A, et al. Catechol-O-methyltransferase val108/158met genotype predicts working memory response to antipsychotic medications. Biol Psychiatry. 2004;56:677–682. doi: 10.1016/j.biopsych.2004.08.012.  [DOI](https://doi.org/10.1016/j.biopsych.2004.08.012) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15522252/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biol%20Psychiatry&title=Catechol-O-methyltransferase%20val108/158met%20genotype%20predicts%20working%20memory%20response%20to%20antipsychotic%20medications&author=TW%20Weickert&author=TE%20Goldberg&author=A%20Mishara&volume=56&publication_year=2004&pages=677-682&pmid=15522252&doi=10.1016/j.biopsych.2004.08.012&)

22. Bertolino A, Caforio G, Blasi G, et al. Interaction of COMT Val108/158 Met genotype and olanzapine treatment on prefrontal cortical function in patients with schizophrenia. Am J Psychiatry. 2004;161:1798–1805. doi: 10.1176/ajp.161.10.1798.  [DOI](https://doi.org/10.1176/ajp.161.10.1798) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15465976/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Psychiatry&title=Interaction%20of%20COMT%20Val108/158%20Met%20genotype%20and%20olanzapine%20treatment%20on%20prefrontal%20cortical%20function%20in%20patients%20with%20schizophrenia&author=A%20Bertolino&author=G%20Caforio&author=G%20Blasi&volume=161&publication_year=2004&pages=1798-1805&pmid=15465976&doi=10.1176/ajp.161.10.1798&)

23. Anttila S, Illi A, Kampman O, Mattila KM, Lehtimaki T, Leinonen E. Interaction between NOTCH4 and catechol-O-methyltransferase genotypes in schizophrenia patients with poor response to typical neuroleptics. Pharmacogenetics. 2004;14:303–307. doi: 10.1097/00008571-200405000-00005.  [DOI](https://doi.org/10.1097/00008571-200405000-00005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15115916/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Interaction%20between%20NOTCH4%20and%20catechol-O-methyltransferase%20genotypes%20in%20schizophrenia%20patients%20with%20poor%20response%20to%20typical%20neuroleptics&author=S%20Anttila&author=A%20Illi&author=O%20Kampman&author=KM%20Mattila&author=T%20Lehtimaki&volume=14&publication_year=2004&pages=303-307&pmid=15115916&doi=10.1097/00008571-200405000-00005&)

24. Schafer M, Rujescu D, Giegling I, et al. Association of short-term response to haloperidol treatment with a polymorphism in the dopamine D(2) receptor gene. Am J Psychiatry. 2001;158:802–804. doi: 10.1176/appi.ajp.158.5.802.  [DOI](https://doi.org/10.1176/appi.ajp.158.5.802) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11329406/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Psychiatry&title=Association%20of%20short-term%20response%20to%20haloperidol%20treatment%20with%20a%20polymorphism%20in%20the%20dopamine%20D(2)%20receptor%20gene&author=M%20Schafer&author=D%20Rujescu&author=I%20Giegling&volume=158&publication_year=2001&pages=802-804&pmid=11329406&doi=10.1176/appi.ajp.158.5.802&)

25. Reynolds GP, Yao Z, Zhang X, Sun J, Zhang Z. Pharmacogenetics of treatment in first-episode schizophrenia: D3 and 5-HT2C receptor polymorphisms separately associate with positive and negative symptom response. Eur Neuropsychopharmacol. 2005;15:143–151. doi: 10.1016/j.euroneuro.2004.07.001.  [DOI](https://doi.org/10.1016/j.euroneuro.2004.07.001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15695058/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20Neuropsychopharmacol&title=Pharmacogenetics%20of%20treatment%20in%20first-episode%20schizophrenia:%20D3%20and%205-HT2C%20receptor%20polymorphisms%20separately%20associate%20with%20positive%20and%20negative%20symptom%20response&author=GP%20Reynolds&author=Z%20Yao&author=X%20Zhang&author=J%20Sun&author=Z%20Zhang&volume=15&publication_year=2005&pages=143-151&pmid=15695058&doi=10.1016/j.euroneuro.2004.07.001&)

26. Szekeres G, Keri S, Juhasz A, et al. Role of dopamine D3 receptor (DRD3) and dopamine transporter (DAT) polymorphism in cognitive dysfunctions and therapeutic response to atypical antipsychotics in patients with schizophrenia. Am J Med Genet B Neuropsychiatr Genet. 2004;124:1–5. doi: 10.1002/ajmg.b.20045.  [DOI](https://doi.org/10.1002/ajmg.b.20045) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14681904/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Med%20Genet%20B%20Neuropsychiatr%20Genet&title=Role%20of%20dopamine%20D3%20receptor%20(DRD3)%20and%20dopamine%20transporter%20(DAT)%20polymorphism%20in%20cognitive%20dysfunctions%20and%20therapeutic%20response%20to%20atypical%20antipsychotics%20in%20patients%20with%20schizophrenia&author=G%20Szekeres&author=S%20Keri&author=A%20Juhasz&volume=124&publication_year=2004&pages=1-5&pmid=14681904&doi=10.1002/ajmg.b.20045&)

27. Staddon S, Arranz MJ, Mancama D, Mata I, Kerwin RW. Clinical applications of pharmacogenetics in psychiatry. Psychopharmacology (Berl) 2002;162:18–23. doi: 10.1007/s00213-002-1084-4.  [DOI](https://doi.org/10.1007/s00213-002-1084-4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12107612/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Psychopharmacology%20(Berl)&title=Clinical%20applications%20of%20pharmacogenetics%20in%20psychiatry&author=S%20Staddon&author=MJ%20Arranz&author=D%20Mancama&author=I%20Mata&author=RW%20Kerwin&volume=162&publication_year=2002&pages=18-23&pmid=12107612&doi=10.1007/s00213-002-1084-4&)

28. Lane HY, Chang YC, Chiu CC, Chen ML, Hsieh MH, Chang WH. Association of risperidone treatment response with a polymorphism in the 5-HT(2A) receptor gene. Am J Psychiatry. 2002;159:1593–1595. doi: 10.1176/appi.ajp.159.9.1593.  [DOI](https://doi.org/10.1176/appi.ajp.159.9.1593) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12202283/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Psychiatry&title=Association%20of%20risperidone%20treatment%20response%20with%20a%20polymorphism%20in%20the%205-HT(2A)%20receptor%20gene&author=HY%20Lane&author=YC%20Chang&author=CC%20Chiu&author=ML%20Chen&author=MH%20Hsieh&volume=159&publication_year=2002&pages=1593-1595&pmid=12202283&doi=10.1176/appi.ajp.159.9.1593&)

29. Joober R, Benkelfat C, Brisebois K, et al. T102C polymorphism in the 5HT2A gene and schizophrenia: relation to phenotype and drug response variability. J Psychiatry Neurosci. 1999;24:141–146.  [PMC free article](/articles/PMC1188995/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10212557/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Psychiatry%20Neurosci&title=T102C%20polymorphism%20in%20the%205HT2A%20gene%20and%20schizophrenia:%20relation%20to%20phenotype%20and%20drug%20response%20variability&author=R%20Joober&author=C%20Benkelfat&author=K%20Brisebois&volume=24&publication_year=1999&pages=141-146&pmid=10212557&)

30. Keefe RS, Bilder RM, Davis SM, et al. Neurocognitive effects of antipsychotic medications in patients with chronic schizophrenia in the CATIE trial. Arch Gen Psychiatry. 2007;64:633–647. doi: 10.1001/archpsyc.64.6.633.  [DOI](https://doi.org/10.1001/archpsyc.64.6.633) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17548746/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arch%20Gen%20Psychiatry&title=Neurocognitive%20effects%20of%20antipsychotic%20medications%20in%20patients%20with%20chronic%20schizophrenia%20in%20the%20CATIE%20trial&author=RS%20Keefe&author=RM%20Bilder&author=SM%20Davis&volume=64&publication_year=2007&pages=633-647&pmid=17548746&doi=10.1001/archpsyc.64.6.633&)

31. Marder SR, Fenton W. Measurement and treatment research to improve cognition in schizophrenia: NIMH MATRICS initiative to support the development of agents for improving cognition in schizophrenia. Schizophr Res. 2004;72:5–9. doi: 10.1016/j.schres.2004.09.010.  [DOI](https://doi.org/10.1016/j.schres.2004.09.010) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15531402/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Schizophr%20Res&title=Measurement%20and%20treatment%20research%20to%20improve%20cognition%20in%20schizophrenia:%20NIMH%20MATRICS%20initiative%20to%20support%20the%20development%20of%20agents%20for%20improving%20cognition%20in%20schizophrenia&author=SR%20Marder&author=W%20Fenton&volume=72&publication_year=2004&pages=5-9&pmid=15531402&doi=10.1016/j.schres.2004.09.010&)

32. Stone JL, O'Donovan MC, Gurling H, et al. Rare chromosomal deletions and duplications increase risk of schizophrenia. Nature. 2008;455(7210):237–241. doi: 10.1038/nature07239.  [DOI](https://doi.org/10.1038/nature07239) | [PMC free article](/articles/PMC3912847/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18668038/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Rare%20chromosomal%20deletions%20and%20duplications%20increase%20risk%20of%20schizophrenia&author=JL%20Stone&author=MC%20O'Donovan&author=H%20Gurling&volume=455&publication_year=2008&pages=237-241&pmid=18668038&doi=10.1038/nature07239&)

33. Walsh T, McClellan JM, McCarthy SE, et al. Rare structural variants disrupt multiple genes in neurodevelopmental pathways in schizophrenia. Science. 2008;320:539–543. doi: 10.1126/science.1155174.  [DOI](https://doi.org/10.1126/science.1155174) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18369103/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Rare%20structural%20variants%20disrupt%20multiple%20genes%20in%20neurodevelopmental%20pathways%20in%20schizophrenia&author=T%20Walsh&author=JM%20McClellan&author=SE%20McCarthy&volume=320&publication_year=2008&pages=539-543&pmid=18369103&doi=10.1126/science.1155174&)

34. McCarthy MI, Abecasis GR, Cardon LR, et al. Genome-wide association studies for complex traits: consensus, uncertainty and challenges. Nat Rev Genet. 2008;9:356–369. doi: 10.1038/nrg2344.  [DOI](https://doi.org/10.1038/nrg2344) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18398418/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Genet&title=Genome-wide%20association%20studies%20for%20complex%20traits:%20consensus,%20uncertainty%20and%20challenges&author=MI%20McCarthy&author=GR%20Abecasis&author=LR%20Cardon&volume=9&publication_year=2008&pages=356-369&pmid=18398418&doi=10.1038/nrg2344&)

35. Campbell DB, Ebert PJ, Skelly T, et al. Ethnic stratification of the association of RGS4 variants with antipsychotic treatment response in schizophrenia. Biol Psychiatry. 2008;63:32–41. doi: 10.1016/j.biopsych.2007.04.018.  [DOI](https://doi.org/10.1016/j.biopsych.2007.04.018) | [PMC free article](/articles/PMC2194758/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17588543/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biol%20Psychiatry&title=Ethnic%20stratification%20of%20the%20association%20of%20RGS4%20variants%20with%20antipsychotic%20treatment%20response%20in%20schizophrenia&author=DB%20Campbell&author=PJ%20Ebert&author=T%20Skelly&volume=63&publication_year=2008&pages=32-41&pmid=17588543&doi=10.1016/j.biopsych.2007.04.018&)

36. Crowley JJ, Keefe RS, Perkins DO, Stroup TS, Lieberman JA, Sullivan PF. The neuregulin 1 promoter polymorphism rs6994992 is not associated with chronic schizophrenia or neurocognition. Am J Med Genet B Neuropsychiatr Genet. 2008;147B:1298–1300. doi: 10.1002/ajmg.b.30727.  [DOI](https://doi.org/10.1002/ajmg.b.30727) | [PMC free article](/articles/PMC3875465/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18286587/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Med%20Genet%20B%20Neuropsychiatr%20Genet&title=The%20neuregulin%201%20promoter%20polymorphism%20rs6994992%20is%20not%20associated%20with%20chronic%20schizophrenia%20or%20neurocognition&author=JJ%20Crowley&author=RS%20Keefe&author=DO%20Perkins&author=TS%20Stroup&author=JA%20Lieberman&volume=147B&publication_year=2008&pages=1298-1300&pmid=18286587&doi=10.1002/ajmg.b.30727&)

37. Campbell DB, Lange LA, Skelly T, Lieberman J, Levitt P, Sullivan PF. Association of RGS2 and RGS5 variants with schizophrenia symptom severity. Schizophr Res. 2008;101:67–75. doi: 10.1016/j.schres.2008.01.006.  [DOI](https://doi.org/10.1016/j.schres.2008.01.006) | [PMC free article](/articles/PMC2486404/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18262772/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Schizophr%20Res&title=Association%20of%20RGS2%20and%20RGS5%20variants%20with%20schizophrenia%20symptom%20severity&author=DB%20Campbell&author=LA%20Lange&author=T%20Skelly&author=J%20Lieberman&author=P%20Levitt&volume=101&publication_year=2008&pages=67-75&pmid=18262772&doi=10.1016/j.schres.2008.01.006&)

38. Pinheiro AP, Keefe RS, Skelly T, et al. AKT1 and neurocognition in schizophrenia. Aust N Z J Psychiatry. 2007;41:169–177. doi: 10.1080/00048670601109956.  [DOI](https://doi.org/10.1080/00048670601109956) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17464696/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Aust%20N%20Z%20J%20Psychiatry&title=AKT1%20and%20neurocognition%20in%20schizophrenia&author=AP%20Pinheiro&author=RS%20Keefe&author=T%20Skelly&volume=41&publication_year=2007&pages=169-177&pmid=17464696&doi=10.1080/00048670601109956&)

39. Sullivan PF, Keefe RS, Lange LA, et al. NCAM1 and neurocognition in schizophrenia. Biol Psychiatry. 2007;61:902–910. doi: 10.1016/j.biopsych.2006.07.036.  [DOI](https://doi.org/10.1016/j.biopsych.2006.07.036) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17161382/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biol%20Psychiatry&title=NCAM1%20and%20neurocognition%20in%20schizophrenia&author=PF%20Sullivan&author=RS%20Keefe&author=LA%20Lange&volume=61&publication_year=2007&pages=902-910&pmid=17161382&doi=10.1016/j.biopsych.2006.07.036&)
