# A Common Variant in Fibroblast Growth Factor Binding Protein 1 (FGFBP1) is Associated with Bone Mineral Density and Influences Gene Expression In Vitro

## Metadata
**Authors:** Nicole Hoppman, John C McLenithan, Daniel J McBride, Haiqing Shen, Jan Bruder, Richard L Bauer, John R Shaffer, Jie Liu, Elizabeth A Streeten, Alan R Shuldiner, Candace M Kammerer, Braxton D Mitchell
**Journal:** Bone
**Date:** 2010 May 5
**DOI:** [10.1016/j.bone.2010.04.607](https://doi.org/10.1016/j.bone.2010.04.607)
**PMID:** 20450993
**PMCID:** PMC2902654
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2902654/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC2902654/pdf/nihms-203738.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC2902654/pdf/nihms-203738.pdf)

## Abstract

We previously detected strong evidence for linkage of forearm bone mineral density (BMD) to chromosome 4p (lod = 4.3) in a set of 29 large Mexican American families. Fibroblast growth factor binding protein 1 (FGFBP1) is a strong candidate gene for bone homeostasis in this region. We sequenced the coding region of FGFBP1 in a subset of our Mexican American study population and performed association studies with BMD on SNPs genotyped in the entire cohort. We then attempted to replicate these findings in an independent study cohort and performed invitro functional studies on replicated, potentially functional polymorphisms using a luciferase reporter construct to evaluate influence on gene expression. Several SNPs spanning the gene, all in one large block of linkage disequilibrium, were significantly associated with BMD at various skeletal sites (n=872, p = 0.001-0.04). The associations were then replicated in an independent population of European ancestry (n = 972; p = 0.02-0.04). Sex-stratified association analyses in both study populations suggest this association is much stronger in men. Subsequent luciferase reporter gene assays revealed marked differences in FGFBP1 expression among the three common haplotypes. Further experiments revealed that a promoter polymorphism, rs12503796, results in decreased expression of FGFBP1 and inhibits upregulation of the gene by testosterone in vitro. Collectively, these findings suggest that sequence variation in FGFBP1 may contribute to variation in BMD, possibly influencing osteoporosis risk.

Keywords: FGFBP1, polymorphisms, bone mineral density, osteoporosis, testosterone

## Introduction

Osteoporosis, a disease characterized by low bone mass and structural deterioration of bone, is a major threat to public health. It is estimated that approximately 10 million Americans are affected while another 34 million are at risk due to low BMD. Fractures resulting from osteoporosis are very costly in terms of public health dollars as well as quality of life. Osteoporosis is estimated to be responsible for more than 1.5 million fractures annually, of which 300,000 are fractures of the hip and 250,000 are fractures of the forearm. The estimated cost for osteoporotic hip fractures in the US was $18 billion dollars in 2002, and the cost is rising [[1](#R1)].

Because BMD is such an important predictor of future fracture, efforts are underway to understand the genetic and environmental factors that influence BMD. Twin and family studies suggest that BMD has a strong genetic component [[2](#R2)-[5](#R5)]. Several chromosomal regions potentially harboring quantitative trait loci (QTL) that influence BMD have been identified through linkage analysis, though few have been replicated across multiple studies. The paucity of replicated linkages is not surprising, since locus-specific effects on BMD are likely to be small as bone homeostasis is a complex process that is influenced by many factors. Nevertheless, determining the genetic causes and mechanism of this disease may make it possible to improve diagnosis, treatment, and offer better screening and prevention to at-risk individuals.

We previously performed a genome-wide linkage scan in 664 Mexican Americans comprising 29 large pedigrees and detected very strong evidence for linkage of forearm (radius midpoint) BMD to an 18 megabase region on chromosome 4p with the peak LOD near marker D4S2639 (LOD = 4.33, genome-wide p = 0.006) [[6](#R6)]. This region includes over 30 known genes and several hypothetical proteins.

An attractive candidate gene in this region is fibroblast growth factor binding protein 1 (*FGFBP1*). *FGFBP1*, also known as heparin-binding growth factor binding protein 17 (*HBP17*), is located at chromosome 4p15.32, which is directly under our linkage peak in Mexican Americans. The gene is approximately 2.8 kilobases long, and its 234 amino acid protein product is secreted from the cell. FGFBP1 has been shown to bind various fibroblast growth factors (FGFs), such as FGF-1 and FGF-2 [[7](#R7)], as well as FGFs 7, 10, and 22 [[8](#R8)]. Additional studies suggest that FGFBP1 has the ability to mobilize FGF-2 from the extracellular matrix and enhance the biological activities of both FGF-1 and FGF-2 [[9](#R9), [10](#R10)]. The ability of FGFBP1 to bind and modify the properties of FGF-2 is particularly relevant to the study of BMD for many reasons. FGF-2 is essential in regulating the differentiation, proliferation, and apoptosis of human osteoblasts [[11](#R11)]. FGF-2 binds cell surface receptors (FGFRs) on osteoblasts and activates mitogen-activated protein (MAP) kinase signal transduction pathways involving various MAP kinases [[12](#R12)-[14](#R14)]. Each of these pathways ultimately activates transcription of genes involved in osteoblast differentiation and function, such as type I collagen and alkaline phosphatase. Since FGFBP1 is secreted it has the potential to act on FGF-2 upstream of this pathway before it binds its receptor on osteoblast cell membranes.

In this follow-up to our linkage study, we report identification of several SNPs associated with BMD in our Mexican American population and replicate the association in an independent population. We then demonstrate by luciferase reporter gene assay that the three common haplotypes show marked differences in *FGFBP1* expression with a promoter polymorphism responsible for abolishing this gene's response to testosterone *in vitro*. Therefore, variation in *FGFBP1* may be important in expression of this gene and possibly subsequent risk of osteoporosis.

## Materials and Methods

### Mexican American Subjects

The San Antonio Family Osteoporosis Study (SAFOS) was initiated in 1997 to identify the genetic determinants of BMD in Mexican Americans. The baseline phase of the SAFOS included 897 individuals from 34 families (872 of whom had BMD measured at the hip and/or spine and are included in the present analyses) who were invited to participate because of their concurrent participation in a follow-up examination of the San Antonio Family Heart Study (SAFHS). Eligibility criteria for probands were that they be 40-60 years of age and have large families in the San Antonio area. First, second, and third degree relatives of each proband and the proband's spouse were invited to participate irrespective of medical history. Details of sampling and recruitment procedures were previously described [[15](#R15)].

Examinations were conducted at the General Clinical Research Center located at the Audie Murphy Veteran's Administration Hospital on the campus of the Health Science Center in San Antonio. Body mass index (BMI) was calculated as weight in kilograms divided by height in meters squared. BMD was assessed using a Hologic Model 1500W dual energy X-ray absorptiometer (DXA, Hologic, Inc., Bedford, MA). Relevant to this report, we measured BMD at the forearm (radius one-third), lumbar spine (L1-L4), and hip (trochanter and femoral neck). Areal BMD (g/cm^2^) was determined by the manufacturer's software by dividing the bone mineral content (BMC, g) by the projected area of the region scanned (cm^2^). The short-term in vivo precision of the BMD was determined on 27 subjects who were examined twice on the same day. The precision of the lumbar spine was 0.009 g/cm^2^ (CV% = 1.0%) and precision of the total hip was 0.007 g/cm^2^ (CV% = 0.87%). The precision of the manufacturer's spine phantom was 0.0017 g/cm^2^ (CV% = 0.17%).

### Amish Replication Subjects

The Amish Family Osteoporosis Study (AFOS) was also initiated in 1997 with the goal of identifying the genetic determinants of osteoporosis. Details of ascertainment, phenotyping and clinical characteristics of participants have been reported previously [[16](#R16)-[19](#R19)]. Ascertainment for this study was based on families with a proband believed to be at risk for osteoporosis by virtue of fracture history or prior bone mineral density measurements. The AFOS also comprised large extended families and included adults aged 20 years and older.

Exams were conducted at the Amish Research Clinic in Strasburg, PA. Height was measured by using a stadiometer and weight was recorded without shoes. BMI was calculated as weight in kilograms divided by height in meters squared. BMD was measured by DXA using a Hologic Model 4500 W (Hologic Inc., Bedford, MA) at the same sites as for the Mexican American study.

The protocols and procedures for the San Antonio Family Osteoporosis Study and Amish Family Osteoporosis Study were approved by the Institutional Review Boards at the respective institutions and all subjects gave written informed consent.

### Sequencing and Genotyping

*FGFBP1* is 2.8 kilobases in length and contains two exons. Primers were designed using Primer3 software (available at [http://frodo.wi.mit.edu/cgi-bin/primer3/primer3_www.cgi](http://frodo.wi.mit.edu/cgi-bin/primer3/primer3_www.cgi)) to amplify approximately 1500 nucleotides of the promoter, both exons including 5′ and 3′ untranslated regions (UTR), and splice junctions. We sequenced PCR products from 40 Mexican Americans. To enhance our chances of detecting a highly penetrant rare allele, we selected 20 subjects with high BMD and 20 with low BMD from the families showing the greatest evidence of linkage to 4p. There was an equal number of males and females in the sequencing subset. The probability of not observing a variant in 40 individuals (80 chromosomes) whose frequency in the full set is 4% or higher was 3.8%. Both strands were sequenced from all samples on an ABI 3700 DNA sequencer and analyzed with Sequence Analysis 3.2 software (Applied Biosystems). We used Sequencher (GeneCodes) to align individual sequences and identify polymorphisms.

All SNPs were genotyped using SNPlex Genotyping System (Applied Biosystems) according to the manufacturer's protocol [[20](#R20)]. SNPs were analyzed using GeneMapper version 4.0 software (Applied Biosystems). Genotyping error rates ranged from 0-2% in Mexican Americans (n = 50 replicated samples) and from 0-1.5% in Amish (n = 180 replicated samples).

### Association Analyses

All genotypes were checked for Mendelian errors using Pedcheck software [[21](#R21)] and errors were resolved or removed prior to analysis. Linkage disequilibrium (LD) among SNPs was assessed using the program Haploview [[22](#R22)].

Because both the Mexican American and Amish studies included large extended families, we utilized variance component methods to test for associations between each SNP genotype and BMD measures at different sites because these methods account for the relatedness among family members. Briefly, we used SOLAR to estimate the effects of genotype on BMD while incorporating the effects of sex, age, age squared, age by sex interactions, and BMI, as well as the residual correlations in BMD existing between related individuals. We initially tested for significance using additive genetic models, that is, linear allele dosage effects. When statistically significant genotype effects were observed, we then evaluated dominant and recessive genetic models that compared trait values between carriers and noncarriers of each allele. Only a recessive model was used in the Amish replication study based on findings in the Mexican American study population. Statistical significance was assessed by likelihood ratio tests in which we compared the likelihood of the data given the pedigree structure for a model in which genotype effects were estimated to that of a model in which genotype effects were constrained to be zero. Twice the difference between the logarithm of the likelihoods of the two models is distributed asymptotically as a χ^2^ statistic with degrees of freedom equal to the difference in the numbers of independent factors in the models being compared. In the initial Mexican American sample, we used a Bonferroni correction to account for the number of `independent' SNPs tested by excluding SNPs in high LD (r^2^ > 0.80) with another tested SNP. We did not adjust for the multiple BMD phenotypes tested because of the high correlations in BMD across sites.

### Vector Preparation and Cloning

There were three common haplotypes at *FGFBP1*, which comprised 92% of haplotypes in Mexican Americans and 99% of haplotypes in the Amish. A 3.12 kb region of the gene (promoter, exon 1 (untranslated), intron 1, and first 7 bases of exon 2 prior to the translation start site) was amplified from DNA samples isolated from individuals homozygous for each haplotype. PCR was performed with Advantage 2 Polymerase Mix (BD Biosciences), according to the manufacturer's protocol. The amplified fragments and pGL3-Basic Reporter vector were digested with *Xho*I (New England Biolabs) and PCR products were cloned into the linearized vector using BD In-Fusion PCR Cloning Kit (BD Biosciences). Constructs were transformed into *E. coli* Fusion-Blue chemically competent cells (BD Biosciences) and isolated using QIAprep Spin Miniprep Kit (Qiagen). Constructs with single SNPs were created from the HapA construct using the QuikChange Site-Directed Mutagenesis Kit (Stratagene). All recombinant plasmid vectors were sequenced to confirm the sequence and orientation of inserts. Correct constructs were transformed into chemically competent *E. coli* cells and plasmids were isolated using HiSpeed Plasmid Maxi Kit (Qiagen).

### Cell Culture and Transfection

Human bronchial epithelial cells (ATCC designation HBE135-E6E7), which were shown to express FGFBP1 at high levels by quantitative RT-PCR (data not shown), were obtained from American Type Culture Collection (ATCC). Osteoblast cell lines were not used because they did not express FGFBP1 at sufficient levels for *in vitro* studies to be performed (data not shown). Cells were subcultured and plated onto 12- or 24-well culture dishes (Costar) and grown in keratinocyte serum-free media (GIBCO) supplemented with 5 ng/mL human recombinant EGF (GIBCO), 0.05 mg/ml bovine pituitary extract (GIBCO), 0.005 mg/ml insulin (Sigma), and 500 ng/ml hydrocortisone (Sigma). Cells were incubated at 37°C with 95% air and 5% CO 2 and were transfected at 50-70% confluence. For each construct, at least two wells were transfected using FuGENE HD Reagent (Roche) according to manufacturer's protocol using a 6:2 ratio (6 uL reagent: 2 ug DNA). 0.1 ug of pRL-CMV was cotransfected per well in order to normalize for transfection efficiency. Cells were incubated for 2 days, after which media was removed and cells were lysed for reporter gene assays as described below. Dexamethasone (100 nM) (Sigma), 3,3′,5-Triiodo-L-thyronine sodium salt (thyroid hormone/T3, 50 nM) (Sigma), testosterone (100 nM) (Sigma), and 9-cis-retinoic acid (100 nM) (Sigma) were added to cells 24 hours post-transfection at the concentrations indicated; cells were lysed and assayed for luciferase activity 24 hours after addition.

### Luciferase Activity

The Dual-Luciferase Reporter Assay System (Promega) was used to quantitate luciferase activity in transfected cells. Firefly luciferase activity was determined by mixing 20 uL of cell lysate with 100 uL of Luciferase Assay Reagent II. Relative light units were measured with a TD-20/20 Luminometer (Turner Designs). Renilla luciferase activity was then measured by the addition of 100 uL of Stop & Glo Reagent and subsequent measurement on the TD-20/20. Each well was read three times, and the average of the readings was used in statistical analyses. Relative luciferase activity was corrected for Renilla luciferase activity, expression levels among wells were standardized within experiments relative to HapA, and at least 3 experiments were combined for statistical analysis. The Student's T-test was used to test for expression differences between luciferase reporter gene constructs.

### Quantitative RT-PCR (qRT-PCR)

Total RNA was extracted from HBE cells stimulated with testosterone or not for 24 hours (100nM) using RNeasy Mini Kits (Qiagen,). Total RNA was reverse transcribed using the Roche Transcriptor cDNA kit (Roche Applied Science). cDNA was quantified using the LightCycler 480, LightCycler 480 Probes Master kit (Roche Applied Science) and Taqman probe/primer sets for *FGFBP1* and cyclophilin (Applied Biosystems). Normalized gene expression of FGFBP1 was expressed as a ratio of FGFBP1/cyclophilin mRNA.

## Results

### SNP Discovery and Association with BMD in Mexican Americans

Characteristics of the 872 Mexican American (initial) and 972 Amish (replication) samples are given in [Table 1](#T1). Compared to the Amish sample, the Mexican American sample was on average younger (42-43 versus 50-51 yrs) and heavier (mean BMI 29.5 and 31.1 kg/m^2^ in men and women, respectively versus. 26.4 and 28.0 kg/m^2^. Consistent with their younger age and higher BMI, Mexican Americans also had higher BMD at the spine and femoral neck.

### Table 1.

|   | Mexican Americans (initial sample; n = 872) |  | Amish (replication sample; n = 972) |  |
| --- | --- | --- | --- | --- |
| Trait | Males (n = 335) | Females (n = 537) | Males (n = 376) | Females N = 596) |
| Mean age (yrs) | 41.9 ± 16.4 | 43.1 ± 15.4 | 50.2 ± 16.0 | 50.7 ± 16.2 |
| Mean BMI (kg/m2) | 29.5 ± 5.7 | 31.1 ± 6.9 | 26.4 ± 4.2 | 28.0 ± 6.0 |
| BMD (g/cm2) |   |   |   |   |
| Spine | 1.062 ± 0.145 | 1.008 ± 0.145 | 0.941 ± 0.141 | 0.907 ± 0.153 |
| Femoral neck | 0.905 ± 0.148 | 0.847 ± 0.139 | 0.853 ± 0.132 | 0.810 ± 0.155 |
| Total hip | 1.116 ± 0.151 | 1.010 ± 0.150 | 0.985 ± 0.128 | 0.893 ± 0.157 |
| Trochanter | 0.766 ± 0.120 | 0.683 ± 0.116 | 0.767 ± 0.104 | 0.689 ± 0.124 |
| Radius midpoint | 0.679 ± 0.066 | 0.577 ± 0.063 | 0.681 ± 0.065 | 0.547 ± 0.077 |
| Radius one-third | 0.778 ± 0.068 | 0.652 ± 0.067 | 0.802 ± 0.066 | 0.673 ± 0.075 |

Table 1 Caption: Characteristics of Mexican American and Amish study subjects (mean ± SD).

Sequencing of *FGFBP1* in 40 Mexican American subjects revealed 9 polymorphisms ([Figure 1](#F1)). All of the variants detected by sequencing were present in the HapMap database with the lone exception of rs2246196, which lies over 1kb upstream of the *FGFBP1* promoter and is monomorphic in the Amish cohort. We genotyped 7 of the 9 SNPs in the full set of SAFOS participants; the two remaining SNPs, rs2246196 and rs732244, failed genotyping but were in near-perfect correlation (r^2^ > 0.90) with other genotyped SNPs in the 40 sequencing samples (rs2246196 in LD with rs2531174 and rs2072313; and rs732244 in LD with rs732245 and rs12503796). In the full SAFOS sample there was extensive linkage disequilibrium among the 7 genotyped SNPs, indicating that *FGFBP1* comprised essentially one large haplotype block ([Figure 2A](#F2)).

### Fig.1.

![Fig.1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3ef5/2902654/0eb7dc05ace5/nihms-203738-f0001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=2902654_nihms-203738-f0001.jpg)

Polymorphisms identified in Mexican Americans and the Amish. Exons are numbered below the gene schematic; black regions correspond to UTR and white regions correspond to coding sequence. Direction of transcription is indicated by arrow. Abbreviations: SNP, single nucleotide polymorphism; N/A, not genotyped due to multiplex restrictions; UTR, untranslated region *Positions given are cDNA coordinates.

### Fig. 2.

![Fig. 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3ef5/2902654/f42c1d81f334/nihms-203738-f0002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=2902654_nihms-203738-f0002.jpg)

Linkage disequilibrium (D' top, r2 bottom) pattern for all SNPs genotyped in Mexican Americans (a) and the Amish (b). For both D' and r2, values range from 0 (no LD) to 100 (complete LD). Schematic of the FGFBP1 gene is shown above to indicate approximate locations of each SNP. Arrow indicates direction of transcription; white box indicates coding region. Value in each box indicates D' or r2 value between 2 SNPs (intersection). For D' figures, no value indicates complete LD (100). Rs2531174 and rs2072313 were nonpolymorphic in the Amish and are therefore not shown in 2b

Two of the seven genotyped SNPs (rs2531174 and rs2072313) had minor allele frequencies ≤ 0.02 and were not analyzed further ([Figure 1](#F1)). Associations of the remaining five SNPs with BMD are shown in [Figure 3](#F3). In the full set of Mexican Americans, four of the five SNPs were significantly associated with BMD at one or more sites with statistically significant p-values for the recessive model ranging from 0.004 to 0.042. The two most strongly associated SNPs, rs2245964 and rs16892645, were in high LD (r^2^ = 0.86) and subjects homozygous for the minor allele (frequency = 0.18-0.20; n = 32 subjects for rs16892645) had higher BMD than subjects with one or two copies of the major allele. Two additional SNPs were also in high LD with each other (rs12503796 and rs732245), and so we considered our threshold p-value for statistical significance in the initial Mexican American cohort to be 0.05/3, or 0.017.

### Fig. 3.

![Fig. 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3ef5/2902654/986bf8ca7576/nihms-203738-f0003.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=2902654_nihms-203738-f0003.jpg)

P-values reflecting association of FGFBP1 SNPs with BMD in Mexican Americans and Amish. P-values indicated correspond to a recessive genetic model. Covariates include age, age squared, sex, BMI (and in Mexican Americans diabetes status and physical activity levels. P-values not shown are not significant (p > 0.05)

### Replication Study in Old Order Amish

We next sought to replicate the associations in an independent sample consisting of 972 Amish individuals. The LD patterns of the five common *FGFBP1* SNPs in the Amish closely resembled those estimated in Mexican Americans ([Figure 2B](#F2)) as well as the HapMap CEPH population (data not shown). As shown in [Figure 3](#F3), both SNPs were associated with trochanter BMD, again with subjects homozygous for the minor allele having higher BMD than subjects with one or two copies of the major allele (p=0.02 for each; 2-tailed p-value with adjustments for age, sex, and BMI). Modest associations (p < 0.05) were also observed in the Amish between the other three *FGFBP1* SNPs and BMD at other sites.

[Figure 4](#F4) demonstrates the association between genotype and BMD at the trochanter in both populations for one representative SNP (rs16892645), adjusted for age, sex, BMI, and other covariates, demonstrating a similar effect in both populations. This particular SNP accounts for 1.2% and 0.3% of the total residual variation in trochanter BMD after accounting for the effects of age, sex, and BMI (and other covariates) in Mexican Americans and Amish respectively.

### Fig. 4.

![Fig. 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3ef5/2902654/fd79abe509b7/nihms-203738-f0004.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=2902654_nihms-203738-f0004.jpg)

Mean BMD (± standard error) at the trochanter according to rs16892645 genotype in Mexican Americans and the Amish. BMD is adjusted for age, sex, BMI, diabetes status, and metabolic equivalents in Mexican Americans, and for age, sex, and BMI in the Amish. P-values shown are for recessive genetic mode. N=572 (GG), 245 (GA), and 32 (AA) for Mexican Americans and N=848 (GG), 110 (GA), and 7 (AA)

Because BMD is known to differ between men and women, we also assessed whether these SNPs were differentially associated with BMD in men and women in both the Mexican American and Amish cohorts. Sex-specific analyses in both populations revealed significant associations with BMD in males while very weak or even lacking in females. In Mexican Americans, for example, the associations with rs16892645 remained highly statistically significant in men despite a markedly reduced sample size (p: 0.037-0.001; n = 359), but not in women (p > 0.05 for all; n = 528) ([Figure 3](#F3)). However, only 15 males and 17 females in Mexican Americans, and only 3 males and 4 females in the Amish, had the high BMD genotype. A formal genotype × sex interaction was carried out in Mexican Americans and revealed no strong evidence supporting a stronger genotype effect on BMD in men than in women (interaction p-value > 0.05). There was insufficient power to detect a genotype × sex interaction involving rs16892645 in the Amish due to the low frequency of this SNP (MAF = 0.06) and small number of subjects with the high BMD-associated genotype.

### In Vitro Functional Studies

In order to determine if one or more of the associated SNPs in *FGFBP1* influence gene expression we created luciferase reporter gene constructs which included 1.6 kb of the promoter, the entire 5′ UTR, and entire intron of *FGFBP1*, totaling 3.12 kb in length. Constructs included the three common haplotypes formed by SNPs rs12503796, rs2245964, and rs16892645 in this region. These haplotypes, hereafter referred to as HapA (GCG), HapB (TCG), and HapC (GGA), occurred in frequencies of 0.31, 0.48, and 0.18 in the population. Because of the high LD existing between the UTR and intronic SNPs (rs2245964 and rs16892645), only the CG and GA haplotypes were observed at these two SNPs ([Figure 5a](#F5)). We observed significantly decreased luciferase expression with HapB as compared to HapA (p = 6.2 × 10^−6^, while HapC showed only slightly decreased expression (p = 0.017) ([Figure 5b](#F5)). Expression of HapB is approximately 25% lower than HapA. To confirm this finding, a construct containing only the promoter SNP (designated PRO), created by site-directed mutagenesis of HapA, also showed significantly decreased expression compared to HapA, confirming the effect of the promoter SNP on expression (p = 6.51 × 10^−5^, data not shown). Site-directed mutagenesis was also used to separate rs2245964 and rs16892645 in order to determine whether one or both influence expression when isolated; however we observed no significant difference in expression from HapA for either SNP (data not shown).

### Fig. 5.

![Fig. 5](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3ef5/2902654/ea7a53462419/nihms-203738-f0005.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=2902654_nihms-203738-f0005.jpg)

A. Diagram of native haplotype expression vector constructs. Differences from HapA are shown in red. B. Luciferase activity values (adjusted for Renilla luciferase activity and normalized to HapA) for native haplotypes (n = 11 wells). Bars represent standard deviation. All data is normalized to Renilla luciferase activity and adjusted relative to HapA expression (1). *p = 6.2 × 10−6 for HapB vs HapA, **p = 0.017 for HapC vs HapA. C. A polymorphism in the promoter of FGFBP1 inhibits up-regulation of luciferase reporter gene expression (adjusted for Renilla luciferase activity) by testosterone. Bars represent standard error (n = 6 wells). All data is normalized to Renilla luciferase activity and adjusted relative to HapA expression without testosterone (1). *p = 0.00087; with versus without testosterone, **p = 0.0098; with versus without testosterone.

[Table 2](#T2) lists several transcription factors that are predicted to bind to the sequence containing the promoter polymorphism according to three different prediction software programs—Patch [[23](#R23)], Promo [[24](#R24), [25](#R25)], and TFsearch [[26](#R26)]. It is possible that one or more of these transcription factors may directly or indirectly influence bone homeostasis. To determine whether any of these factors does indeed bind to one of these polymorphic loci, we added several nuclear hormone receptor agonists to the HBE cells transfected with HapA, HapB, and HapC and measured luciferase expression. Three agonists, dexamethasone, thyroid hormone and retinoic acid, did not influence expression of the luciferase reporter gene. However, when testosterone was added to cultures, we observed an increase in luciferase expression for both HapA (p = 0.00087) and HapC (p = 0.0098), but not HapB (p = 0.752), implying that the promoter polymorphism (rs12503796) found in HapB may inhibit response to testosterone. To confirm this result, we performed the experiment with the PRO construct and observed the same lack of response to testosterone (p = 0.59, [Figure 5c](#F5)).

### Table 2.

| SNP | Location | Allele | Construct | Promo Transcription Factor Prediction | PATCH Transcription Factor Prediction | TFsearch Transcription Factor Prediction |
| --- | --- | --- | --- | --- | --- | --- |
| rs12503796 | promoter | G | HapA HapC | RXRα | H4TF-2 | RunX1 |
|   |   | T | HapB | AR, TFII-I | GR, T3Rα, T3Rβ1, T3Rβ2, RARα1, RXRβ | none |

Table 2 Caption: Potential transcription factor binding sites for sequences containing polymorphisms. Those relevant to bone homeostasis and therefore tested in vitro are shown in bold. No relevant sites were predicted for the two other polymorphisms tested (rs2245964 and rs16892645).

We next measured endogenous gene expression of *FGFBP1* in HBE cells by qRT-PCR to verify that the regulation of the endogenous *FGFBP1* gene by testosterone was similar to the observed regulation of *FGFBP1* promoter/ luciferase constructs. *FGFBP1* gene expression (as a ratio of cyclophilin mRNA) increased 1.5 fold over basal expression after addition of testosterone (0.93 ± 0.07 vs 1.43 ± 0.2, respectively, p < 0.05, n=12). Thus, testosterone regulation of the endogenous *FGFBP1* gene was of similar magnitude and direction as the promoter/luciferase constructs (data not shown).

## Discussion

Despite the high heritability of BMD, efforts to identify individual genes that explain a large proportion of the variation in this trait have been largely unsuccessful. Numerous candidate genes have been proposed, some of which have been associated with BMD in multiple populations [[27](#R27), [28](#R28)], and recent genome-wide association studies have identified a small number of additional loci that appear to be robustly associated with BMD [[29](#R29)-[34](#R34)]. However, very few association studies have resulted in the identification of functional variants with demonstrable effects on gene expression. We have performed an association study in two very different populations, a genetically heterogenous population with a high degree of admixture and a genetically homogenous population of European ancestry, and followed up with functional studies of associated haplotypes. The fact that the *FGFBP1* associations demonstrated in our study were observed across two populations, taken together with our *in vitro* studies showing a dramatic effect on gene expression, make this gene a very convincing candidate for bone homeostasis.

The mechanisms for differences in *FGFBP1* expression are not clear; however, we provide evidence that the promoter polymorphism (rs12503796) may be involved. Rs12503796 is located in the promoter approximately 280 bases upstream of the transcription start site. It is not located in a known transcription factor binding site; however, it is a predicted binding site for several transcription factors that may be involved in bone metabolism. In this regard we did, in fact, observe enhanced *in vitro* luciferase expression when transfected cells were co-cultured with testosterone, although the testosterone-enhancing effect was obliterated in constructs having the rs12503796 T allele, suggesting that this allele may destroy a binding site for the androgen receptor. An attractive interpretation of this hypothesis is that loss of a testosterone-enhancing effect on BMD might be expected to have a greater effect on BMD in men than in women, which is also consistent with our observed association results. However, the “T” allele is predicted to bind the androgen receptor, while the “G” allele is not, so if the prediction is correct, other mechanisms to account for the differences in haplotype expression must be in play. For example, there may be other transcription factors not tested, either alone or through interaction with the androgen receptor, that mediate this effect. In fact, the promoter polymorphism is also a predicted binding site for the general transcription factor TFII-I and the rs12503796 polymorphism might alter TFII-I binding, possibly explaining why basal expression levels of the constructs were different in the absence of hormones. Finally, it is also possible that *FGFBP1* expression differences may have nothing to do with the rs12503796 promoter polymorphism because *FGFBP1* expression was decreased in HapC, despite the presence of the G allele at this promoter site. It is also possible that this effect is not true as this construct did not have significantly different expression from HapA in the presence of testosterone, which is likely more representative of true physiological conditions in which testosterone is always present at some level.

Despite the parallel associations observed in these data of *FGFBP1* SNPs with BMD and of *FGFBP1* haplotypes with gene expression, there are some limitations and inconsistencies in our study. First, Mexican Americans are an admixed population and our analysis of *FGFBP1* SNPS did not account for population substructure. However, replication in a non-admixed population, the Amish, mitigates this limitation. Second, the expression experiments may not have been performed in the most physiologically relevant tissue. *FGFBP1* expression in osteoblasts was measured; expression was detected but at insufficient amounts to carry out in vitro studies (data not shown). However, because FGFBP1 is secreted, it is possible that high expression in tissues other than bone could certainly influence bone mineral density. Third, the allele most strongly associated with higher BMD (e.g. rs16892645 A allele) is not by itself associated with gene expression. Fourth, the T allele at the promoter polymorphism (rs12503796) appears to block the increased expression of this gene in the presence of testosterone. How this potential decrease in activity and/or lack of stimulation leads to increased BMD is unclear since FGFBP1 is thought to enhance FGF-2 binding to its receptors, which should lead to osteoblast stimulation. Therefore, one might expect decreased expression or function of FGFBP1 to lead to decreased BMD. It is possible that in bone, FGFBP1 acts to sequester FGF-2 instead of enhancing binding. Indeed, some data suggests that while FGFBP1 stimulates FGF-2 at low concentrations, it can also inhibit FGF-2 at higher concentrations [[7](#R7), [35](#R35)]. Likewise, FGF-2 has also been found to have dual effects on osteoblasts. FGF-2 induces proliferation and differentiation in immature osteoblasts but has also been found to block mineralization and induce apoptosis in differentiated osteoblasts [[13](#R13)]. FGF-2 is also thought to influence osteoclast maturation and activity [[36](#R36)-[40](#R40)]; hence we would expect in this scenario that decreased expression of FGFBP1 would lead to increased BMD, as we observed, via decreased osteoclast activity. These hypotheses require further testing in order to determine the mechanism of this effect. Additionally, the cell type used in our in vitro experiments may not be representative of the pathway in relevant tissues *in vitro*.

The possibility that the effects of *FGFBP1* SNPs may manifest differently between men and women is intriguing, but must be considered speculative. While the intronic rs16892645 variant was more strongly associated in Mexican American men than women, the estimated effect size in men was not statistically greater than that observed in women in a formal sex by SNP interaction analysis. Moreover, we had little power to test for sex differences in the association of this SNP with BMD in the Amish because of the low frequency of the rs16892645 variant in this population. Nevertheless, coupled with the observation that expression was significantly enhanced in the presence of testosterone, the possibility of a sex interaction makes an interesting hypothesis. Future studies of *FGFBP1* should consider the possibility of differential effects between men and women.

It seems unlikely that variation in *FGFBP1* is responsible for our previously observed linkage peak due to the modest, albeit replicated, associations observed with BMD. Although linkage analysis can detect rare genetic effects, our prior linkage was exclusively with forearm BMD, while the current analyses indicate association across multiple BMD sites, particularly at the hip. It is unclear why our previous linkage study detected little evidence for linkage to hip BMD; likewise, despite modest association of rs2245964 and rs16892645 with forearm BMD, a subsequent (not shown) linkage analysis indicated essentially no change in the evidence for linkage to forearm BMD after accounting for the associated SNPs from this study. Other genes in this region must therefore be considered as candidates for their role in driving the forearm BMD linkage result.

In summary, we have identified several polymorphisms in *FGFBP1* that are associated with BMD at several sites in both Mexican Americans and the Old Order Amish. The minor alleles of these polymorphisms tend to be associated with higher BMD in both populations and there is a suggestion that effects may be stronger in males than in females. A promoter polymorphism (rs12503796) was shown to influence gene expression *in vitro*, possibly due to alteration of a transcription factor binding site as demonstrated by abolishment of response to testosterone *in vitro*. However, while these data provide evidence that *FGFBP1* is involved in bone remodeling, the precise mechanism governing the regulation of gene expression is not clear. This mechanism likely involves an interaction between sequence variation and hormonal or other influences on transcription. Further elucidation of the factors regulating expression of this gene and the downstream effects of altered gene expression of skeletal phenotypes seems warranted.

## Acknowledgements

We are deeply grateful for the cooperation of the families participating in the SAFOS and AFOS. The San Antonio Family Osteoporosis Study is supported by research grants RO1-AR43351, RO1- HL69313, and PO1-HL45522 awarded by the National Institutes of Health. Support for the Frederic C. Bartter General Clinical Research Center was made available by Clinical National Institutes of Health Grant MO1-RR-01346. The Amish Family Osteoporosis Study is supported by research grants R01-AR46838 awarded by the National Institutes of Health, University of Maryland General Clinical Research Center Grant M01 RR165001, General Clinical Research Centers Program, National Center for Research Resources (NCRR), and the Geriatrics Research and Education Clinical Center, Baltimore Veterans Administration Medical Center. We would like to thank Faith Rowland for help with sequencing and Jessica Pray and Samuel Woodle for help with the functional analyses.

## Footnotes

## References

1. Bone Health and Osteoporosis: A Report of the Surgeon General. U.S. Department of Health and Human Services, Office of the Surgeon General; Rockville, MD: 2004.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/20945569/) | [Google Scholar](https://scholar.google.com/scholar_lookup?title=Bone%20Health%20and%20Osteoporosis:%20A%20Report%20of%20the%20Surgeon%20General&publication_year=2004&)

2. Arden NK, Spector TD. Genetic influences on muscle strength, lean body mass, and bone mineral density: a twin study. J Bone Miner Res. 1997;12:2076–81. doi: 10.1359/jbmr.1997.12.12.2076.  [DOI](https://doi.org/10.1359/jbmr.1997.12.12.2076) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9421240/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Bone%20Miner%20Res&title=Genetic%20influences%20on%20muscle%20strength,%20lean%20body%20mass,%20and%20bone%20mineral%20density:%20a%20twin%20study&author=NK%20Arden&author=TD%20Spector&volume=12&publication_year=1997&pages=2076-81&pmid=9421240&doi=10.1359/jbmr.1997.12.12.2076&)

3. Dequeker J, Nijs J, Verstraeten A, Geusens P, Gevers G. Genetic determinants of bone mineral content at the spine and radius: a twin study. Bone. 1987;8:207–9. doi: 10.1016/8756-3282(87)90166-9.  [DOI](https://doi.org/10.1016/8756-3282(87)90166-9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/3446256/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bone&title=Genetic%20determinants%20of%20bone%20mineral%20content%20at%20the%20spine%20and%20radius:%20a%20twin%20study&author=J%20Dequeker&author=J%20Nijs&author=A%20Verstraeten&author=P%20Geusens&author=G%20Gevers&volume=8&publication_year=1987&pages=207-9&pmid=3446256&doi=10.1016/8756-3282(87)90166-9&)

4. Pocock NA, Eisman JA, Hopper JL, Yeates MG, Sambrook PN, Eberl S. Genetic determinants of bone mass in adults. A twin study. J Clin Invest. 1987;80:706–10. doi: 10.1172/JCI113125.  [DOI](https://doi.org/10.1172/JCI113125) | [PMC free article](/articles/PMC442294/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/3624485/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Invest&title=Genetic%20determinants%20of%20bone%20mass%20in%20adults.%20A%20twin%20study&author=NA%20Pocock&author=JA%20Eisman&author=JL%20Hopper&author=MG%20Yeates&author=PN%20Sambrook&volume=80&publication_year=1987&pages=706-10&pmid=3624485&doi=10.1172/JCI113125&)

5. Smith DM, Nance WE, Kang KW, Christian JC, Johnston CC., Jr Genetic factors in determining bone mass. J Clin Invest. 1973;52:2800–8. doi: 10.1172/JCI107476.  [DOI](https://doi.org/10.1172/JCI107476) | [PMC free article](/articles/PMC302548/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/4795916/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Invest&title=Genetic%20factors%20in%20determining%20bone%20mass&author=DM%20Smith&author=WE%20Nance&author=KW%20Kang&author=JC%20Christian&author=CC%20Johnston&volume=52&publication_year=1973&pages=2800-8&pmid=4795916&doi=10.1172/JCI107476&)

6. Kammerer CM, Schneider JL, Cole SA, Hixson JE, Samollow PB, O'Connell JR, Perez R, Dyer TD, Almasy L, Blangero J, Bauer RL, Mitchell BD. Quantitative trait loci on chromosomes 2p, 4p, and 13q influence bone mineral density of the forearm and hip in Mexican Americans. J Bone Miner Res. 2003;18:2245–52. doi: 10.1359/jbmr.2003.18.12.2245.  [DOI](https://doi.org/10.1359/jbmr.2003.18.12.2245) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14672361/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Bone%20Miner%20Res&title=Quantitative%20trait%20loci%20on%20chromosomes%202p,%204p,%20and%2013q%20influence%20bone%20mineral%20density%20of%20the%20forearm%20and%20hip%20in%20Mexican%20Americans&author=CM%20Kammerer&author=JL%20Schneider&author=SA%20Cole&author=JE%20Hixson&author=PB%20Samollow&volume=18&publication_year=2003&pages=2245-52&pmid=14672361&doi=10.1359/jbmr.2003.18.12.2245&)

7. Wu DQ, Kan MK, Sato GH, Okamoto T, Sato JD. Characterization and molecular cloning of a putative binding protein for heparin-binding growth factors. J Biol Chem. 1991;266:16778–85.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/1885605/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Characterization%20and%20molecular%20cloning%20of%20a%20putative%20binding%20protein%20for%20heparin-binding%20growth%20factors&author=DQ%20Wu&author=MK%20Kan&author=GH%20Sato&author=T%20Okamoto&author=JD%20Sato&volume=266&publication_year=1991&pages=16778-85&pmid=1885605&)

8. Beer HD, Bittner M, Niklaus G, Munding C, Max N, Goppelt A, Werner S. The fibroblast growth factor binding protein is a novel interaction partner of FGF-7, FGF-10 and FGF-22 and regulates FGF activity: implications for epithelial repair. Oncogene. 2005;24:5269–77. doi: 10.1038/sj.onc.1208560.  [DOI](https://doi.org/10.1038/sj.onc.1208560) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15806171/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncogene&title=The%20fibroblast%20growth%20factor%20binding%20protein%20is%20a%20novel%20interaction%20partner%20of%20FGF-7,%20FGF-10%20and%20FGF-22%20and%20regulates%20FGF%20activity:%20implications%20for%20epithelial%20repair&author=HD%20Beer&author=M%20Bittner&author=G%20Niklaus&author=C%20Munding&author=N%20Max&volume=24&publication_year=2005&pages=5269-77&pmid=15806171&doi=10.1038/sj.onc.1208560&)

9. Aigner A, Butscheid M, Kunkel P, Krause E, Lamszus K, Wellstein A, Czubayko F. An FGF-binding protein FGF-BP exerts its biological function by parallel paracrine stimulation of tumor cell and endothelial cell proliferation through FGF-2 release. Int J Cancer. 2001;92:510–7. doi: 10.1002/1097-0215(20010515)92:4<510::aid-ijc1227>3.0.co;2-h.  [DOI](https://doi.org/10.1002/1097-0215(20010515)92:4<510::aid-ijc1227>3.0.co;2-h) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11304685/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Cancer&title=An%20FGF-binding%20protein%20FGF-BP%20exerts%20its%20biological%20function%20by%20parallel%20paracrine%20stimulation%20of%20tumor%20cell%20and%20endothelial%20cell%20proliferation%20through%20FGF-2%20release&author=A%20Aigner&author=M%20Butscheid&author=P%20Kunkel&author=E%20Krause&author=K%20Lamszus&volume=92&publication_year=2001&pages=510-7&pmid=11304685&doi=10.1002/1097-0215(20010515)92:4<510::aid-ijc1227>3.0.co;2-h&)

10. Tassi E, Al-Attar A, Aigner A, Swift MR, McDonnell K, Karavanov A, Wellstein A. Enhancement of fibroblast growth factor FGF activity by an FGF-binding protein. J Biol Chem. 2001;276:40247–53. doi: 10.1074/jbc.M104933200.  [DOI](https://doi.org/10.1074/jbc.M104933200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11509569/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Enhancement%20of%20fibroblast%20growth%20factor%20FGF%20activity%20by%20an%20FGF-binding%20protein&author=E%20Tassi&author=A%20Al-Attar&author=A%20Aigner&author=MR%20Swift&author=K%20McDonnell&volume=276&publication_year=2001&pages=40247-53&pmid=11509569&doi=10.1074/jbc.M104933200&)

11. Marie PJ, Debiais F, Hay E. Regulation of human cranial osteoblast phenotype by FGF-2, FGFR-2 and BMP-2 signaling. Histol Histopathol. 2002;17:877–85. doi: 10.14670/HH-17.877.  [DOI](https://doi.org/10.14670/HH-17.877) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12168799/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Histol%20Histopathol&title=Regulation%20of%20human%20cranial%20osteoblast%20phenotype%20by%20FGF-2,%20FGFR-2%20and%20BMP-2%20signaling&author=PJ%20Marie&author=F%20Debiais&author=E%20Hay&volume=17&publication_year=2002&pages=877-85&pmid=12168799&doi=10.14670/HH-17.877&)

12. Debiais F, Lemonnier J, Hay E, Delannoy P, Caverzasio J, Marie PJ. Fibroblast growth factor-2 FGF-2 increases N-cadherin expression through protein kinase C and Src-kinase pathways in human calvaria osteoblasts. J Cell Biochem. 2001;81:68–81. doi: 10.1002/1097-4644(20010401)81:1<68::aid-jcb1024>3.0.co;2-s.  [DOI](https://doi.org/10.1002/1097-4644(20010401)81:1<68::aid-jcb1024>3.0.co;2-s) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11180398/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Cell%20Biochem&title=Fibroblast%20growth%20factor-2%20FGF-2%20increases%20N-cadherin%20expression%20through%20protein%20kinase%20C%20and%20Src-kinase%20pathways%20in%20human%20calvaria%20osteoblasts&author=F%20Debiais&author=J%20Lemonnier&author=E%20Hay&author=P%20Delannoy&author=J%20Caverzasio&volume=81&publication_year=2001&pages=68-81&pmid=11180398&doi=10.1002/1097-4644(20010401)81:1<68::aid-jcb1024>3.0.co;2-s&)

13. Mansukhani A, Bellosta P, Sahni M, Basilico C. Signaling by fibroblast growth factors FGF and fibroblast growth factor receptor 2 FGFR2-activating mutations blocks mineralization and induces apoptosis in osteoblasts. J Cell Biol. 2000;149:1297–308. doi: 10.1083/jcb.149.6.1297.  [DOI](https://doi.org/10.1083/jcb.149.6.1297) | [PMC free article](/articles/PMC2175120/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10851026/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Cell%20Biol&title=Signaling%20by%20fibroblast%20growth%20factors%20FGF%20and%20fibroblast%20growth%20factor%20receptor%202%20FGFR2-activating%20mutations%20blocks%20mineralization%20and%20induces%20apoptosis%20in%20osteoblasts&author=A%20Mansukhani&author=P%20Bellosta&author=M%20Sahni&author=C%20Basilico&volume=149&publication_year=2000&pages=1297-308&pmid=10851026&doi=10.1083/jcb.149.6.1297&)

14. Newberry EP, Willis D, Latifi T, Boudreaux JM, Towler DA. Fibroblast growth factor receptor signaling activates the human interstitial collagenase promoter via the bipartite Ets-AP1 element. Mol Endocrinol. 1997;11:1129–44. doi: 10.1210/mend.11.8.9958.  [DOI](https://doi.org/10.1210/mend.11.8.9958) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9212060/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Endocrinol&title=Fibroblast%20growth%20factor%20receptor%20signaling%20activates%20the%20human%20interstitial%20collagenase%20promoter%20via%20the%20bipartite%20Ets-AP1%20element&author=EP%20Newberry&author=D%20Willis&author=T%20Latifi&author=JM%20Boudreaux&author=DA%20Towler&volume=11&publication_year=1997&pages=1129-44&pmid=9212060&doi=10.1210/mend.11.8.9958&)

15. Mitchell BD, Kammerer CM, Schneider JL, Perez R, Bauer RL. Genetic and environmental determinants of bone mineral density in Mexican Americans: results from the San Antonio Family Osteoporosis Study. Bone. 2003;33:839–46. doi: 10.1016/s8756-3282(03)00246-1.  [DOI](https://doi.org/10.1016/s8756-3282(03)00246-1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14623060/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bone&title=Genetic%20and%20environmental%20determinants%20of%20bone%20mineral%20density%20in%20Mexican%20Americans:%20results%20from%20the%20San%20Antonio%20Family%20Osteoporosis%20Study&author=BD%20Mitchell&author=CM%20Kammerer&author=JL%20Schneider&author=R%20Perez&author=RL%20Bauer&volume=33&publication_year=2003&pages=839-46&pmid=14623060&doi=10.1016/s8756-3282(03)00246-1&)

16. Brown LB, Streeten EA, Shapiro JR, McBride D, Shuldiner AR, Peyser PA, Mitchell BD. Genetic and environmental influences on bone mineral density in pre- and post-menopausal women. Osteoporos Int. 2005;16:1849–56. doi: 10.1007/s00198-005-1948-7.  [DOI](https://doi.org/10.1007/s00198-005-1948-7) | [PMC free article](/articles/PMC4049290/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15997421/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Osteoporos%20Int&title=Genetic%20and%20environmental%20influences%20on%20bone%20mineral%20density%20in%20pre-%20and%20post-menopausal%20women&author=LB%20Brown&author=EA%20Streeten&author=JR%20Shapiro&author=D%20McBride&author=AR%20Shuldiner&volume=16&publication_year=2005&pages=1849-56&pmid=15997421&doi=10.1007/s00198-005-1948-7&)

17. Brown LB, Streeten EA, Shuldiner AR, Almasy LA, Peyser PA, Mitchell BD. Assessment of sex-specific genetic and environmental effects on bone mineral density. Genet Epidemiol. 2004;27:153–61. doi: 10.1002/gepi.20009.  [DOI](https://doi.org/10.1002/gepi.20009) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15305331/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genet%20Epidemiol&title=Assessment%20of%20sex-specific%20genetic%20and%20environmental%20effects%20on%20bone%20mineral%20density&author=LB%20Brown&author=EA%20Streeten&author=AR%20Shuldiner&author=LA%20Almasy&author=PA%20Peyser&volume=27&publication_year=2004&pages=153-61&pmid=15305331&doi=10.1002/gepi.20009&)

18. Streeten EA, McBride DJ, Lodge AL, Pollin TI, Stinchcomb DG, Agarwala R, Schaffer AA, Shapiro JR, Shuldiner AR, Mitchell BD. Reduced incidence of hip fracture in the Old Order Amish. J Bone Miner Res. 2004;19:308–13. doi: 10.1359/JBMR.0301223.  [DOI](https://doi.org/10.1359/JBMR.0301223) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14969401/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Bone%20Miner%20Res&title=Reduced%20incidence%20of%20hip%20fracture%20in%20the%20Old%20Order%20Amish&author=EA%20Streeten&author=DJ%20McBride&author=AL%20Lodge&author=TI%20Pollin&author=DG%20Stinchcomb&volume=19&publication_year=2004&pages=308-13&pmid=14969401&doi=10.1359/JBMR.0301223&)

19. Streeten EA, McBride DJ, Pollin TI, Ryan K, Shapiro J, Ott S, Mitchell BD, Shuldiner AR, O'Connell JR. Quantitative trait loci for BMD identified by autosome-wide linkage scan to chromosomes 7q and 21q in men from the Amish Family Osteoporosis Study. J Bone Miner Res. 2006;21:1433–42. doi: 10.1359/jbmr.060602.  [DOI](https://doi.org/10.1359/jbmr.060602) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16939402/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Bone%20Miner%20Res&title=Quantitative%20trait%20loci%20for%20BMD%20identified%20by%20autosome-wide%20linkage%20scan%20to%20chromosomes%207q%20and%2021q%20in%20men%20from%20the%20Amish%20Family%20Osteoporosis%20Study&author=EA%20Streeten&author=DJ%20McBride&author=TI%20Pollin&author=K%20Ryan&author=J%20Shapiro&volume=21&publication_year=2006&pages=1433-42&pmid=16939402&doi=10.1359/jbmr.060602&)

20. De la Vega FM, Lazaruk KD, Rhodes MD, Wenz MH. Assessment of two flexible and compatible SNP genotyping platforms: TaqMan SNP Genotyping Assays and the SNPlex Genotyping System. Mutat Res. 2005;573:111–35. doi: 10.1016/j.mrfmmm.2005.01.008.  [DOI](https://doi.org/10.1016/j.mrfmmm.2005.01.008) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15829242/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mutat%20Res&title=Assessment%20of%20two%20flexible%20and%20compatible%20SNP%20genotyping%20platforms:%20TaqMan%20SNP%20Genotyping%20Assays%20and%20the%20SNPlex%20Genotyping%20System&author=FM%20De%20la%20Vega&author=KD%20Lazaruk&author=MD%20Rhodes&author=MH%20Wenz&volume=573&publication_year=2005&pages=111-35&pmid=15829242&doi=10.1016/j.mrfmmm.2005.01.008&)

21. O'Connell JR, Weeks DE. PedCheck: a program for identification of genotype incompatibilities in linkage analysis. Am J Hum Genet. 1998;63:259–66. doi: 10.1086/301904.  [DOI](https://doi.org/10.1086/301904) | [PMC free article](/articles/PMC1377228/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9634505/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=PedCheck:%20a%20program%20for%20identification%20of%20genotype%20incompatibilities%20in%20linkage%20analysis&author=JR%20O'Connell&author=DE%20Weeks&volume=63&publication_year=1998&pages=259-66&pmid=9634505&doi=10.1086/301904&)

22. Barrett JC, Fry B, Maller J, Daly MJ. Haploview: analysis and visualization of LD and haplotype maps. Bioinformatics. 2005;21:263–5. doi: 10.1093/bioinformatics/bth457.  [DOI](https://doi.org/10.1093/bioinformatics/bth457) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15297300/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=Haploview:%20analysis%20and%20visualization%20of%20LD%20and%20haplotype%20maps&author=JC%20Barrett&author=B%20Fry&author=J%20Maller&author=MJ%20Daly&volume=21&publication_year=2005&pages=263-5&pmid=15297300&doi=10.1093/bioinformatics/bth457&)

23. PATCH: Pattern Search for Transcription Factor Binding Sites. Available at: http://www.gene-regulation.com/pub/programs.html. Last accessed January 2007.  [http://www.gene-regulation.com/pub/programs.html](http://www.gene-regulation.com/pub/programs.html)

24. Farre D, Roset R, Huerta M, Adsuara JE, Rosello L, Alba MM, Messeguer X. Identification of patterns in biological sequences at the ALGGEN server: PROMO and MALGEN. Nucleic Acids Res. 2003;31:3651–3. doi: 10.1093/nar/gkg605.  [DOI](https://doi.org/10.1093/nar/gkg605) | [PMC free article](/articles/PMC169011/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12824386/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=Identification%20of%20patterns%20in%20biological%20sequences%20at%20the%20ALGGEN%20server:%20PROMO%20and%20MALGEN&author=D%20Farre&author=R%20Roset&author=M%20Huerta&author=JE%20Adsuara&author=L%20Rosello&volume=31&publication_year=2003&pages=3651-3&pmid=12824386&doi=10.1093/nar/gkg605&)

25. Messeguer X, Escudero R, Farre D, Nunez O, Martinez J, Alba MM. PROMO: detection of known transcription regulatory elements using species-tailored searches. Bioinformatics. 2002;18:333–4. doi: 10.1093/bioinformatics/18.2.333.  [DOI](https://doi.org/10.1093/bioinformatics/18.2.333) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11847087/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=PROMO:%20detection%20of%20known%20transcription%20regulatory%20elements%20using%20species-tailored%20searches&author=X%20Messeguer&author=R%20Escudero&author=D%20Farre&author=O%20Nunez&author=J%20Martinez&volume=18&publication_year=2002&pages=333-4&pmid=11847087&doi=10.1093/bioinformatics/18.2.333&)

26. Heinemeyer T, Wingender E, Reuter I, Hermjakob H, Kel AE, Kel OV, Ignatieva EV, Ananko EA, Podkolodnaya OA, Kolpakov FA, Podkolodny NL, Kolchanov NA. Databases on transcriptional regulation: TRANSFAC, TRRD and COMPEL. Nucleic Acids Res. 1998;26:362–7. doi: 10.1093/nar/26.1.362.  [DOI](https://doi.org/10.1093/nar/26.1.362) | [PMC free article](/articles/PMC147251/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9399875/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=Databases%20on%20transcriptional%20regulation:%20TRANSFAC,%20TRRD%20and%20COMPEL&author=T%20Heinemeyer&author=E%20Wingender&author=I%20Reuter&author=H%20Hermjakob&author=AE%20Kel&volume=26&publication_year=1998&pages=362-7&pmid=9399875&doi=10.1093/nar/26.1.362&)

27. Ralston SH. Genetic determinants of osteoporosis. Curr Opin Rheumatol. 2005;17:475–9. doi: 10.1097/01.bor.0000166385.62851.92.  [DOI](https://doi.org/10.1097/01.bor.0000166385.62851.92) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15956846/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Opin%20Rheumatol&title=Genetic%20determinants%20of%20osteoporosis&author=SH%20Ralston&volume=17&publication_year=2005&pages=475-9&pmid=15956846&doi=10.1097/01.bor.0000166385.62851.92&)

28. Liu YJ, Shen H, Xiao P, Xiong DH, Li LH, Recker RR, Deng HW. Molecular genetic studies of gene identification for osteoporosis: a 2004 update. J Bone Miner Res. 2006;21:1511–35. doi: 10.1359/JBMR.051002.  [DOI](https://doi.org/10.1359/JBMR.051002) | [PMC free article](/articles/PMC1829484/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16995806/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Bone%20Miner%20Res&title=Molecular%20genetic%20studies%20of%20gene%20identification%20for%20osteoporosis:%20a%202004%20update&author=YJ%20Liu&author=H%20Shen&author=P%20Xiao&author=DH%20Xiong&author=LH%20Li&volume=21&publication_year=2006&pages=1511-35&pmid=16995806&doi=10.1359/JBMR.051002&)

29. Cho YS, Go MJ, Kim YJ, Heo JY, Oh JH, Ban HJ, Yoon D, Lee MH, Kim DJ, Park M, Cha SH, Kim JW, Han BG, Min H, Ahn Y, Park MS, Han HR, Jang HY, Cho EY, Lee JE, Cho NH, Shin C, Park T, Park JW, Lee JK, Cardon L, Clarke G, McCarthy MI, Lee JY, Lee JK, Oh B, Kim HL. A large-scale genome-wide association study of Asian populations uncovers genetic factors influencing eight quantitative traits. Nat Genet. 2009;41:527–34. doi: 10.1038/ng.357.  [DOI](https://doi.org/10.1038/ng.357) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19396169/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=A%20large-scale%20genome-wide%20association%20study%20of%20Asian%20populations%20uncovers%20genetic%20factors%20influencing%20eight%20quantitative%20traits&author=YS%20Cho&author=MJ%20Go&author=YJ%20Kim&author=JY%20Heo&author=JH%20Oh&volume=41&publication_year=2009&pages=527-34&pmid=19396169&doi=10.1038/ng.357&)

30. Kiel DP, Demissie S, Dupuis J, Lunetta KL, Murabito JM, Karasik D. Genome-wide association with bone mass and geometry in the Framingham Heart Study. BMC Med Genet. 2007;8(Suppl 1):S14. doi: 10.1186/1471-2350-8-S1-S14.  [DOI](https://doi.org/10.1186/1471-2350-8-S1-S14) | [PMC free article](/articles/PMC1995606/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17903296/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Med%20Genet&title=Genome-wide%20association%20with%20bone%20mass%20and%20geometry%20in%20the%20Framingham%20Heart%20Study&author=DP%20Kiel&author=S%20Demissie&author=J%20Dupuis&author=KL%20Lunetta&author=JM%20Murabito&volume=8&issue=Suppl%201&publication_year=2007&pages=S14&pmid=17903296&doi=10.1186/1471-2350-8-S1-S14&)

31. Richards JB, Rivadeneira F, Inouye M, Pastinen TM, Soranzo N, Wilson SG, Andrew T, Falchi M, Gwilliam R, Ahmadi KR, Valdes AM, Arp P, Whittaker P, Verlaan DJ, Jhamai M, Kumanduri V, Moorhouse M, van Meurs JB, Hofman A, Pols HA, Hart D, Zhai G, Kato BS, Mullin BH, Zhang F, Deloukas P, Uitterlinden AG, Spector TD. Bone mineral density, osteoporosis, and osteoporotic fractures: a genome-wide association study. Lancet. 2008;371:1505–12. doi: 10.1016/S0140-6736(08)60599-1.  [DOI](https://doi.org/10.1016/S0140-6736(08)60599-1) | [PMC free article](/articles/PMC2679414/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18455228/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Bone%20mineral%20density,%20osteoporosis,%20and%20osteoporotic%20fractures:%20a%20genome-wide%20association%20study&author=JB%20Richards&author=F%20Rivadeneira&author=M%20Inouye&author=TM%20Pastinen&author=N%20Soranzo&volume=371&publication_year=2008&pages=1505-12&pmid=18455228&doi=10.1016/S0140-6736(08)60599-1&)

32. Styrkarsdottir U, Halldorsson BV, Gretarsdottir S, Gudbjartsson DF, Walters GB, Ingvarsson T, Jonsdottir T, Saemundsdottir J, Center JR, Nguyen TV, Bagger Y, Gulcher JR, Eisman JA, Christiansen C, Sigurdsson G, Kong A, Thorsteinsdottir U, Stefansson K. Multiple genetic loci for bone mineral density and fractures. N Engl J Med. 2008;358:2355–65. doi: 10.1056/NEJMoa0801197.  [DOI](https://doi.org/10.1056/NEJMoa0801197) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18445777/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Multiple%20genetic%20loci%20for%20bone%20mineral%20density%20and%20fractures&author=U%20Styrkarsdottir&author=BV%20Halldorsson&author=S%20Gretarsdottir&author=DF%20Gudbjartsson&author=GB%20Walters&volume=358&publication_year=2008&pages=2355-65&pmid=18445777&doi=10.1056/NEJMoa0801197&)

33. Styrkarsdottir U, Halldorsson BV, Gretarsdottir S, Gudbjartsson DF, Walters GB, Ingvarsson T, Jonsdottir T, Saemundsdottir J, Snorradottir S, Center JR, Nguyen TV, Alexandersen P, Gulcher JR, Eisman JA, Christiansen C, Sigurdsson G, Kong A, Thorsteinsdottir U, Stefansson K. New sequence variants associated with bone mineral density. Nat Genet. 2009;41:15–7. doi: 10.1038/ng.284.  [DOI](https://doi.org/10.1038/ng.284) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19079262/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=New%20sequence%20variants%20associated%20with%20bone%20mineral%20density&author=U%20Styrkarsdottir&author=BV%20Halldorsson&author=S%20Gretarsdottir&author=DF%20Gudbjartsson&author=GB%20Walters&volume=41&publication_year=2009&pages=15-7&pmid=19079262&doi=10.1038/ng.284&)

34. Xiong DH, Liu XG, Guo YF, Tan LJ, Wang L, Sha BY, Tang ZH, Pan F, Yang TL, Chen XD, Lei SF, Yerges LM, Zhu XZ, Wheeler VW, Patrick AL, Bunker CH, Guo Y, Yan H, Pei YF, Zhang YP, Levy S, Papasian CJ, Xiao P, Lundberg YW, Recker RR, Liu YZ, Liu YJ, Zmuda JM, Deng HW. Genome-wide association and follow-up replication studies identified ADAMTS18 and TGFBR3 as bone mass candidate genes in different ethnic groups. Am J Hum Genet. 2009;84:388–98. doi: 10.1016/j.ajhg.2009.01.025.  [DOI](https://doi.org/10.1016/j.ajhg.2009.01.025) | [PMC free article](/articles/PMC2667986/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19249006/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Genome-wide%20association%20and%20follow-up%20replication%20studies%20identified%20ADAMTS18%20and%20TGFBR3%20as%20bone%20mass%20candidate%20genes%20in%20different%20ethnic%20groups&author=DH%20Xiong&author=XG%20Liu&author=YF%20Guo&author=LJ%20Tan&author=L%20Wang&volume=84&publication_year=2009&pages=388-98&pmid=19249006&doi=10.1016/j.ajhg.2009.01.025&)

35. Chen JH, Wang XC, Sato JD. Bifunctional effects of heparin-binding protein HBp17 on DNA synthesis in cells. Cell Biol Int. 2001;25:567–70. doi: 10.1006/cbir.2000.0688.  [DOI](https://doi.org/10.1006/cbir.2000.0688) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11407864/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Biol%20Int&title=Bifunctional%20effects%20of%20heparin-binding%20protein%20HBp17%20on%20DNA%20synthesis%20in%20cells&author=JH%20Chen&author=XC%20Wang&author=JD%20Sato&volume=25&publication_year=2001&pages=567-70&pmid=11407864&doi=10.1006/cbir.2000.0688&)

36. Chikazu D, Hakeda Y, Ogata N, Nemoto K, Itabashi A, Takato T, Kumegawa M, Nakamura K, Kawaguchi H. Fibroblast growth factor FGF-2 directly stimulates mature osteoclast function through activation of FGF receptor 1 and p42/p44 MAP kinase. J Biol Chem. 2000;275:31444–50. doi: 10.1074/jbc.M910132199.  [DOI](https://doi.org/10.1074/jbc.M910132199) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10896947/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Fibroblast%20growth%20factor%20FGF-2%20directly%20stimulates%20mature%20osteoclast%20function%20through%20activation%20of%20FGF%20receptor%201%20and%20p42/p44%20MAP%20kinase&author=D%20Chikazu&author=Y%20Hakeda&author=N%20Ogata&author=K%20Nemoto&author=A%20Itabashi&volume=275&publication_year=2000&pages=31444-50&pmid=10896947&doi=10.1074/jbc.M910132199&)

37. Kawaguchi H, Chikazu D, Nakamura K, Kumegawa M, Hakeda Y. Direct and indirect actions of fibroblast growth factor 2 on osteoclastic bone resorption in cultures. J Bone Miner Res. 2000;15:466–73. doi: 10.1359/jbmr.2000.15.3.466.  [DOI](https://doi.org/10.1359/jbmr.2000.15.3.466) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10750561/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Bone%20Miner%20Res&title=Direct%20and%20indirect%20actions%20of%20fibroblast%20growth%20factor%202%20on%20osteoclastic%20bone%20resorption%20in%20cultures&author=H%20Kawaguchi&author=D%20Chikazu&author=K%20Nakamura&author=M%20Kumegawa&author=Y%20Hakeda&volume=15&publication_year=2000&pages=466-73&pmid=10750561&doi=10.1359/jbmr.2000.15.3.466&)

38. Nakano K, Okada Y, Saito K, Tanaka Y. Induction of RANKL expression and osteoclast maturation by the binding of fibroblast growth factor 2 to heparan sulfate proteoglycan on rheumatoid synovial fibroblasts. Arthritis Rheum. 2004;50:2450–8. doi: 10.1002/art.20367.  [DOI](https://doi.org/10.1002/art.20367) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15334457/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arthritis%20Rheum&title=Induction%20of%20RANKL%20expression%20and%20osteoclast%20maturation%20by%20the%20binding%20of%20fibroblast%20growth%20factor%202%20to%20heparan%20sulfate%20proteoglycan%20on%20rheumatoid%20synovial%20fibroblasts&author=K%20Nakano&author=Y%20Okada&author=K%20Saito&author=Y%20Tanaka&volume=50&publication_year=2004&pages=2450-8&pmid=15334457&doi=10.1002/art.20367&)

39. Shimoaka T, Ogasawara T, Yonamine A, Chikazu D, Kawano H, Nakamura K, Itoh N, Kawaguchi H. Regulation of osteoblast, chondrocyte, and osteoclast functions by fibroblast growth factor FGF-18 in comparison with FGF-2 and FGF-10. J Biol Chem. 2002;277:7493–500. doi: 10.1074/jbc.M108653200.  [DOI](https://doi.org/10.1074/jbc.M108653200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11741978/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Regulation%20of%20osteoblast,%20chondrocyte,%20and%20osteoclast%20functions%20by%20fibroblast%20growth%20factor%20FGF-18%20in%20comparison%20with%20FGF-2%20and%20FGF-10&author=T%20Shimoaka&author=T%20Ogasawara&author=A%20Yonamine&author=D%20Chikazu&author=H%20Kawano&volume=277&publication_year=2002&pages=7493-500&pmid=11741978&doi=10.1074/jbc.M108653200&)

40. Zuo J, Jiang J, Dolce C, Holliday LS. Effects of basic fibroblast growth factor on osteoclasts and osteoclast-like cells. Biochem Biophys Res Commun. 2004;318:162–7. doi: 10.1016/j.bbrc.2004.04.013.  [DOI](https://doi.org/10.1016/j.bbrc.2004.04.013) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15110768/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20Biophys%20Res%20Commun&title=Effects%20of%20basic%20fibroblast%20growth%20factor%20on%20osteoclasts%20and%20osteoclast-like%20cells&author=J%20Zuo&author=J%20Jiang&author=C%20Dolce&author=LS%20Holliday&volume=318&publication_year=2004&pages=162-7&pmid=15110768&doi=10.1016/j.bbrc.2004.04.013&)
