# Genome-wide association study of acute post-surgical pain in humans

## Metadata
**Authors:** Hyungsuk Kim, Edward Ramsay, Hyewon Lee, Sharon Wahl, Raymond A Dionne
**Journal:** Pharmacogenomics
**Date:** 2009 Feb
**DOI:** [10.2217/14622416.10.2.171](https://doi.org/10.2217/14622416.10.2.171)
**PMID:** 19207018
**PMCID:** PMC2696643
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2696643/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC2696643/pdf/nihms110731.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC2696643/pdf/nihms110731.pdf)

## Abstract

**Aims:** 
Testing a relatively small genomic region with a few hundred SNPs provides limited information. Genome-wide association studies (GWAS) provide an opportunity to overcome the limitation of candidate gene association studies. Here, we report the results of a GWAS for the responses to an NSAID analgesic.

**Materials & methods:** 
European Americans (60 females and 52 males) undergoing oral surgery were genotyped with Affymetrix 500K SNP assay. Additional SNP genotyping was performed from the gene in linkage disequilibrium with the candidate SNP revealed by the GWAS.

**Results:** 
GWAS revealed a candidate SNP (rs2562456) associated with analgesic onset, which is in linkage disequilibrium with a gene encoding a zinc finger protein. Additional SNP genotyping of ZNF429 confirmed the association with analgesic onset in humans (p = 1.8 × 10−10, degrees of freedom = 103, F = 28.3). We also found candidate loci for the maximum post-operative pain rating (rs17122021, p = 6.9 × 10−7) and post-operative pain onset time (rs6693882, p = 2.1 × 10−6), however, correcting for multiple comparisons did not sustain these genetic associations.

**Conclusion:** 
GWAS for acute clinical pain followed by additional SNP genotyping of a neighboring gene suggests that genetic variations in or near the loci encoding DNA binding proteins play a role in the individual variations in responses to analgesic drugs.

Keywords: analgesic onset, GWAS, pain

### Aims

Testing a relatively small genomic region with a few hundred SNPs provides limited information. Genome-wide association studies (GWAS) provide an opportunity to overcome the limitation of candidate gene association studies. Here, we report the results of a GWAS for the responses to an NSAID analgesic.

### Materials & methods

European Americans (60 females and 52 males) undergoing oral surgery were genotyped with Affymetrix 500K SNP assay. Additional SNP genotyping was performed from the gene in linkage disequilibrium with the candidate SNP revealed by the GWAS.

### Results

GWAS revealed a candidate SNP (rs2562456) associated with analgesic onset, which is in linkage disequilibrium with a gene encoding a zinc finger protein. Additional SNP genotyping of *ZNF429* confirmed the association with analgesic onset in humans (p = 1.8 × 10^−10^, degrees of freedom = 103, F = 28.3). We also found candidate loci for the maximum post-operative pain rating (rs17122021, p = 6.9 × 10^−7^) and post-operative pain onset time (rs6693882, p = 2.1 × 10^−6^), however, correcting for multiple comparisons did not sustain these genetic associations.

### Conclusion

GWAS for acute clinical pain followed by additional SNP genotyping of a neighboring gene suggests that genetic variations in or near the loci encoding DNA binding proteins play a role in the individual variations in responses to analgesic drugs.

## Materials & methods

### Subjects

The study was approved by the Institutional Review Board of the National Institute of Dental and Craniofacial Research (NIH, MD, USA) and informed consent was obtained from all subjects. Among 221 patients of mixed ethnic population undergoing oral surgery, only European Americans (60 females and 52 males), with ages ranging from 17 to 35 years, were analyzed in this study to minimize population stratification and potential bias caused by age differences. It is usually not recommended that bony impacted third molars be removed when patients are out of this age range. Patients underwent standardized surgery by the same oral surgeon removing third molar teeth that included at least one bony impacted mandibular third molar. After receiving premedication with intravenous midazolam (4.9 ± 0.2 mg) and local anesthesia with 2% lidocaine (250.6 ± 43.0 mg) with epinephrine 1:100,000, a mucoperiosteal flap was raised and retracted, bone removed, and the teeth were sectioned as needed to facilitate extraction of the impacted lower third molars.

### Genotyping

For GWAS, genomic DNA extracted from white blood cells with the Puregene™ DNA isolation kit (Gentra Systems Inc., MN, USA) was hybridized to SNP chips following manufacturer’s instruction (Affymetrix, CA, USA). Quality control procedures including call rates and the Hardy–Weinberg Equilibrium test were performed as suggested [[19](#R19)], and 255,785 SNPs that passed quality control filters were analyzed with dependent variables. Briefly, quality control procedures excluded 99,330 SNPs with call rates greater than 0.85, as well as 26,437 SNPs not in Hardy–Weinberg equilibrium (p < 0.001) in the control sample, yielding 374,975 SNPs available for analysis. The primary analytic modality involved computation of likelihood ratios (degrees of freedom [df] = 1) for the best possible genotypic split (e.g., recessive or dominant models) for each SNP, with the constraint that a minimum of ten subjects populate each split group (thereby excluding monomorphic and very rare SNPs), yielding 255,785 SNP splits. For GWAS statistical analyses, we used the recursive partitioning approach based on F-statistics, using Interactive Tree Analysis of Helix Tree 6.2.0 software (Golden Helix, MT, USA). The detailed description for the approach is available in the [Appendix](#APP1).

Next, we sought to extend the GWAS finding by conducting common SNP associations in the candidate gene region based on linkage disequilibrium (LD) of the HapMap data. The gene encoding zinc finger protein 429 (*ZNF429*) is in an LD with the SNP suggested from GWAS analysis. Genotyping was performed with Assays-by-Design SNP Genotyping Products (Applied Biosystems, CA, USA). A total of 5 ng of DNA was mixed in a well containing 2.5 µl of Taqman® universal master mix (Applied Biosystems) 0.25 µl of genotyping assay mix and 2.25 µl of DNAse free water. PCR was performed under the following conditions: 95°C, 10 min followed by 40 cycles of 92°C, 15 s and 60°C, 1 min in a Perkin–Elmer™ 9700 thermocycler (Perkin–Elmer Inc., MA, USA). Following PCR, fluorescence of each well was measured using the ABI Prism 7900 Sequence Detection System (Applied Biosystems). Genotype discrimination was performed using Taqman Sequence Detector version 2.0 software. Samples that failed to amplify were not included in the final analysis.

### Clinical pain measurement

Clinically induced pain was recorded with a paper and pencil form of a 100 mm visual analog scale (VAS). After the extraction of the impacted third molars, pain was recorded every 20 min by VAS until subjects requested analgesic medication as the local anesthesia was eliminated and post-operative pain onset occurred. Ketorolac tromethamine (Toradol) was administered intravenously at the recommended dose (30 mg) and pain was recorded by VAS again at 15 min intervals for 180 min. When subjects requested analgesic medication as the local anesthesia dissipated (mean = 125 min; 95% confidence interval [CI]: 117–132 min) and post-operative pain (VAS, mean = 58.4 mm; 95% CI: 54.9–61.9) onset occurred. The maximum post-operative pain rating, post-operative pain onset time and the analgesic onset time after ketorolac administration were used as measures of clinical pain and the onset of nonsteroidal anti-inflammatory drug (NSAID) analgesia.

### Data analysis

A total of 11 SNPs from the *ZNF429* in an LD with the most statistically significant SNP in Helix Tree results were analyzed. To resolve phase unknown genotypes and estimate population frequencies in unrelated individuals we employed PHASE method, a probability based Bayesian algorithm. To quantify LD (allelic association between marker and genes), widely used D′ and r^2^ were calculated. A value of zero implies independence, whereas 1.0 means complete co-transfer. Haploblocks based on CI method were generated by Haploview version 3.11 with identical genomic region of HapMap.

For the CNV analysis, Partek GS version 6.2 (Partek, MO, USA) was used to compare CNs between slow analgesic onset (≥ 30 min, n = 7) and rapid analgesic onset (≤ 5 min, n = 33) patients. According to the manufacturer’s recommendation for generating baselines for CN analysis (n > 25), rapid analgesic onset group was defined as 5 min or less. Slow analgesic onset was defined conversely as 5% of the patients with the slowest reported onset of analgesia.

## Results

Genome-wide association study analysis revealed a significant association (p < 3.3 × 10^−8^, p-value threshold was conservatively calculated for 500,000 SNPs × 3 tests from Bonferroni multiple test correction) with analgesic onset after ketorolac administration with Helix Tree analysis ([FIGURE 1A](#F1)). This revealed that a SNP (rs2562456, p = 2.3 × 10^−10^) of an unknown gene *LOC400680*, which is not annotated in current National Center for Biotechnology Information (NCBI) assembly, is significantly associated with analgesic onset at the false-discovery rate (FDR) of 1.18 × 10^−4^. Haploview analysis combining with the HapMap identified that this SNP is in an LD with SNPs in *ZNF429* ([FIGURE 1B](#F1)). Additional 11 SNPs of *ZNF429* region ([TABLE 1](#T1), [NT_011295.10](https://www.ncbi.nlm.nih.gov/nuccore/NT_011295.10)) including four tag SNPs suggested by HapMap data were genotyped. It revealed that the D′ among these 11 SNPs are 1.0 ([TABLE 2](#T2)) and four SNPs are significantly associated with analgesic onset (rs2650825, rs1879234, rs2562408, rs2562466). Homozygous minor allele subjects of these SNPs show approximately four-times slower analgesic onset (n = 9, mean = 36.2 min; 95% CI: 13.3–59.2) compared with major homozygous (n = 61, mean = 9.4 min; 95% CI: 7.8–11.0) and heterozygotes (n = 34, mean = 9.0 min; 95% CI: 6.9–11.1) with analysis-of-variance, p = 1.8 × 10^−10^, df = 103, F = 28.3 ([FIGURE 1C](#F1)). Strong effects on analgesic onset (p = 3.5 × 10^−8^ to p = 2.7 × 10^−7^) in three different novel regions across the human genome were also detected, though Bonferroni multiple test corrections denied the significant associations for these borderline SNPs: rs7295290 (p = 3.5 × 10^−8^ at FDR = 4.51 × 10^−3^, rs10014562 (p = 2.6 × 10^−7^ at FDR = 2.93 × 10^−2^) and rs17011183 (p = 2.7 × 10^−7^ at FDR = 2.48 × 10^−2^) ([FIGURE 1A](#F1)).

### Figure 1. Multilayer genetic association study.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/92b7/2696643/542303282bd4/nihms110731f1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=2696643_nihms110731f1.jpg)

(A) Plot of statistical significance (−log10P) values of genotyped SNPs across the whole genome for analgesic onset time after ketorolac administration. Reference line indicates threshold for genome wide significance (p = 4.2 × 10−7). Horizontal axis represent reference SNPs for the chromosomal location. (B) A single haploblock from HapMap containing significant associated SNPs and ZNF429. (C) Associations between four ZNF429 SNPs and analgesic onset (p = 1.8 × 10−10, degrees of freedom = 103, F = 28.3, mean ± standard error).

### Table 1.

| SNP | SNP ID | Location on NCBI assembly | Nucleotide variation | Rarer allele frequency |
| --- | --- | --- | --- | --- |
| 1 | rs1984771 | 21,483,637 | A/G | 0.17 |
| 2 | rs2650825 | 21,491,065 | A/G | 0.25 |
| 3 | rs1520071 | 21,493,880 | G/T | 0.17 |
| 4 | rs2650757 | 21,495,385 | A/G | 0.17 |
| 5 | rs2681365 | 21,495,804 | A/G | 0.17 |
| 6 | rs1879234 | 21,496,319 | G/T | 0.25 |
| 7 | rs11085455 | 21,496,885 | C/T | 0.17 |
| 8 | rs2173724 | 21,500,997 | A/G | 0.17 |
| 9 | rs2133816 | 21,501,221 | C/T | 0.17 |
| 10 | rs2562408 | 21,501,721 | C/G | 0.25 |
| 11 | rs2562466 | 21,506,044 | C/T | 0.25 |

Table 1 Caption: SNPs genotyped from ZNF429.

### Table 2.

| L1 | L2 | D′ | LOD | r2 | Cllow | Clhi | Dist | T-int |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
| rs1984771 | rs2650825 | 1.0 | 4.48 | 0.071 | 0.67 | 1.0 | 7428 | 115.55 |
|   | rs1520071 | 1.0 | 35.53 | 1.0 | 0.94 | 1.0 | 10243 | – |
|   | rs2650757 | 1.0 | 35.53 | 1.0 | 0.94 | 1.0 | 11748 | – |
|   | rs2681365 | 1.0 | 35.53 | 1.0 | 0.94 | 1.0 | 12167 | – |
|   | rs1879234 | 1.0 | 4.48 | 0.071 | 0.67 | 1.0 | 12682 | – |
|   | rs11085455 | 1.0 | 30.89 | 0.921 | 0.91 | 1.0 | 13248 | – |
|   | rs2173724 | 1.0 | 35.53 | 1.0 | 0.94 | 1.0 | 17360 | – |
|   | rs2133816 | 1.0 | 35.53 | 1.0 | 0.94 | 1.0 | 17584 | – |
|   | rs2562408 | 1.0 | 4.48 | 0.071 | 0.67 | 1.0 | 18084 | – |
|   | rs2562466 | 1.0 | 4.48 | 0.071 | 0.67 | 1.0 | 22407 | – |
|   | rs1520071 | 1.0 | 4.48 | 0.071 | 0.67 | 1.0 | 2815 | 205.86 |
|   |  |  |  |  |  |  |  |  |
| rs2650825 | rs2650757 | 1.0 | 4.48 | 0.071 | 0.67 | 1.0 | 4320 | – |
|   | rs2681365 | 1.0 | 4.48 | 0.071 | 0.67 | 1.0 | 4739 | – |
|   | rs1879234 | 1.0 | 46.31 | 1.0 | 0.95 | 1.0 | 5254 | – |
|   | rs11085455 | 1.0 | 4.15 | 0.065 | 0.64 | 1.0 | 5820 | – |
|   | rs2173724 | 1.0 | 4.48 | 0.071 | 0.67 | 1.0 | 9932 | – |
|   | rs2133816 | 1.0 | 4.48 | 0.071 | 0.67 | 1.0 | 10156 | – |
|   | rs2562408 | 1.0 | 46.31 | 1.0 | 0.95 | 1.0 | 10656 | – |
|   | rs2562466 | 1.0 | 46.31 | 1.0 | 0.95 | 1.0 | 14979 | – |
|   |  |  |  |  |  |  |  |  |
| rs1520071 | rs2650757 | 1.0 | 35.53 | 1.0 | 0.94 | 1.0 | 1505 | 347.82 |
|   | rs2681365 | 1.0 | 35.53 | 1.0 | 0.94 | 1.0 | 1924 | – |
|   | rs1879234 | 1.0 | 4.48 | 0.071 | 0.67 | 1.0 | 2439 | – |
|   | rs11085455 | 1.0 | 30.89 | 0.921 | 0.91 | 1.0 | 3005 | – |
|   | rs2173724 | 1.0 | 35.53 | 1.0 | 0.94 | 1.0 | 7117 | – |
|   | rs2133816 | 1.0 | 35.53 | 1.0 | 0.94 | 1.0 | 7341 | – |
|   | rs2562408 | 1.0 | 4.48 | 0.071 | 0.67 | 1.0 | 7841 | – |
|   | rs2562466 | 1.0 | 4.48 | 0.071 | 0.67 | 1.0 | 12164 | – |
|   |  |  |  |  |  |  |  |  |
| rs2650757 | rs2681365 | 1.0 | 35.53 | 1.0 | 0.94 | 1.0 | 419 | 489.78 |
|   | rs1879234 | 1.0 | 4.48 | 0.071 | 0.67 | 1.0 | 934 | – |
|   | rs11085455 | 1.0 | 30.89 | 0.921 | 0.91 | 1.0 | 1500 | – |
|   | rs2173724 | 1.0 | 35.53 | 1.0 | 0.94 | 1.0 | 5612 | – |
|   | rs2133816 | 1.0 | 35.53 | 1.0 | 0.94 | 1.0 | 5836 | – |
|   | rs2562408 | 1.0 | 4.48 | 0.071 | 0.67 | 1.0 | 6336 | – |
|   | rs2562466 | 1.0 | 4.48 | 0.071 | 0.67 | 1.0 | 10659 | – |
|   |  |  |  |  |  |  |  |  |
| rs2681365 | rs1879234 | 1.0 | 4.48 | 0.071 | 0.67 | 1.0 | 515 | 549.37 |
|   | rs11085455 | 1.0 | 30.89 | 0.921 | 0.91 | 1.0 | 1081 | – |
|   | rs2173724 | 1.0 | 35.53 | 1.0 | 0.94 | 1.0 | 5193 | – |
|   | rs2133816 | 1.0 | 35.53 | 1.0 | 0.94 | 1.0 | 5417 | – |
|   | rs2562408 | 1.0 | 4.48 | 0.071 | 0.67 | 1.0 | 5917 | – |
|   | rs2562466 | 1.0 | 4.48 | 0.071 | 0.67 | 1.0 | 10240 | – |
|   |  |  |  |  |  |  |  |  |
| rs1879234 | rs11085455 | 1.0 | 4.15 | 0.065 | 0.64 | 1.0 | 566 | 544.19 |
|   | rs2173724 | 1.0 | 4.48 | 0.071 | 0.67 | 1.0 | 4678 | – |
|   | rs2133816 | 1.0 | 4.48 | 0.071 | 0.67 | 1.0 | 4902 | – |
|   | rs2562408 | 1.0 | 46.31 | 1.0 | 0.95 | 1.0 | 5402 | – |
|   | rs2562466 | 1.0 | 46.31 | 1.0 | 0.95 | 1.0 | 9725 | – |
|   |  |  |  |  |  |  |  |  |
| rs11085455 | rs2173724 | 1.0 | 30.89 | 0.921 | 0.91 | 1.0 | 4112 | 411.72 |
|   | rs2133816 | 1.0 | 30.89 | 0.921 | 0.91 | 1.0 | 4336 | – |
|   | rs2562408 | 1.0 | 4.15 | 0.065 | 0.64 | 1.0 | 4836 | – |
|   | rs2562466 | 1.0 | 4.15 | 0.065 | 0.64 | 1.0 | 9159 | – |
|   |  |  |  |  |  |  |  |  |
| rs2173724 | rs2133816 | 1.0 | 35.53 | 1.0 | 0.94 | 1.0 | 224 | 269.76 |
|   | rs2562408 | 1.0 | 4.48 | 0.071 | 0.67 | 1.0 | 724 | – |
|   | rs2562466 | 1.0 | 4.48 | 0.071 | 0.67 | 1.0 | 5047 | – |
|   |  |  |  |  |  |  |  |  |
| rs2133816 | rs2562408 | 1.0 | 4.48 | 0.071 | 0.67 | 1.0 | 500 | 127.8 |
|   | rs2562466 | 1.0 | 4.48 | 0.071 | 0.67 | 1.0 | 4823 | – |
|   |  |  |  |  |  |  |  |  |
| rs2562408 | rs2562466 | 1.0 | 46.31 | 1.0 | 0.95 | 1.0 | 4323 | 105.73 |

Table 2 Caption: Linkage disequilibrium of SNPs genotyped from ZNF429.

Comparison of CN across the whole genome revealed that patients with slow analgesic onset (≥30 min, n = 7, CN mean = 3.78; 95% CI: 2.3–5.3) have higher CNs than those with rapid analgesic onset (≤5 min, n = 33, CN mean = 2.06; 95% CI: 1.9–2.2) at SNP rs17011183 (p = 4.5 × 10^−6^, df = 38, F = 41.4), at the same SNP showed strong association above, but this effect was not significant with multiple test correction.

We also found candidate loci for the maximum post-operative pain rating (rs17122021, p = 6.9 × 10^−7^) and post-operative pain onset time (rs6693882, p = 2.1 × 10^−6^), however, correcting for multiple comparisons did not sustain these genetic associations.

## Discussion

Even after the multiple test corrections, the SNP (rs2562456) located in chromosome 19 showed a significant association with the onset time of analgesia after ketorolac administration. The annotation of the genomic region around this SNP is not available yet, though it is included in the unknown gene *LOC400680*; the potential function of this hypothetical gene is not known at present. However, the additionally genotyped 11 SNPs neighboring around this region showed high LD values across approximately 19 kbp (data not shown). According to HapMap, this high LD extends up to 88 kbp size of a single haploblock (from rs8107940 to rs2681384, based on 95% CI) including a gene relevant to DNA binding protein: *ZNF429*. This gene spans approximately 32.3 kbp in the human genome with five exons and the size of mRNA is approximately 1.9 kbp. More than 20 SNPs whose heterozygosity is greater than 0.2, have been reported across the entire gene. Zinc finger proteins control the processes that modulate the frequency, rate or extent of DNA dependent transcription by binding to DNA. Except for this general function as a zinc finger protein, specific biological function of ZNF429 in analgesic drug metabolism and pain pathways has not been reported yet.

Further investigation of *ZNF429* including the role of frequent SNPs in this gene revealed that genetic polymorphisms in this region are strongly associated with NSAID analgesia. We could screen SNPs from introns only because few numbers of SNPs with low minor allele frequency have been reported so far in *ZNF429*. It is not surprising to confirm that *ZNF429* is in a strong LD as HapMap data already suggest in Caucasians. However, the GWAS missed the associations of *ZNF429* SNPs with analgesic onset because we filtered out SNPs with less than ten subjects as the minimum number for one group. The number of minor homozygotes of *ZNF429* was nine. It is clearly demonstrated that dense genotyping around the candidate region can reduce the risk of missing important associations.

We later analyzed analgesic drug efficacy and found that the slower analgesic onset homozygotes tend to report less analgesic effectiveness (p = 0.017, 77% pain reduction, 95% CI: 62–92%) compared with heterozygotes (89% pain reduction, 95% CI: 84–94%) and major homozygotes (90% pain reduction, 95% CI: 87–93%). Even though the statistical significance did not sustain with multiple test correction and virtually all subjects reported significant pain reduction (t-test, p = 1.75 × 10^−12^, VAS, mean = 23.7 mm; 95% CI: 20.3–27.2), it is noticeable that slower analgesic onset patients may also have less benefit from the analgesic drugs in terms of efficacy.

Considering overly conservative Bonferroni correction, a modified Bayesian formula with recent estimates of the total number of human genes as 20,000 has been suggested and results in a p-value threshold of approximately 4.2 × 10^−7^ [[19](#R19)]. This approach introduces three additional SNPs as candidate regions for association with analgesic onset. One of these is in chromosome 4 but this region is not annotated yet. The remaining two are:

Even though we found marginal associations between these additional genetic variations and analgesic onset after ketorolac administration, it is still not clear how these genes or genetic variations influence analgesic responses. Genomic regions around those SNPs were not further investigated in this study because they did not overcome the Bonferroni multiple test corrections. However, this marginal but strong association deserves future investigation in analgesic drug responses. Overall, further investigation is needed initially to characterize the function of those genes both in animals and humans.

We did not find statistically significant association between CNVs across human genome and NSAID analgesia. SNP rs17011183 in the *WDFY4* gene showed a marginal association between analgesic onset and its CNV. Considering the marginal association between analgesic onset and the genotypes of this SNP ([FIGURE 1A](#F1), p = 2.7 × 10^−7^), genomic region of the *WDFY4* gene including rs17011183 may contribute both qualitatively and quantitatively to analgesic drug responses.

Unlike NSAID analgesic onset, only weak associations were found in maximum post-operative pain rating and post-operative pain onset time but those associations were not significant with multiple test corrections. Post-operative pain is also influenced by psychological and environmental factors. It is more likely that genetic factors exert only subtle effects in more complex phenotypes. Our results imply that pharmacogenomics of analgesic responses may be a more feasible target for genetics of pain studies. Considering these weak associations for acute inflammatory pain in the well-characterized oral surgery model [[20](#R20)–[23](#R23)], finding loci responsible for multifactorial chronic pain may be very challenging. Even for a clearly heritable trait such as height, the first genetic locus reported only explains 0.3% of individual differences [[24](#R24)].

Whereas our results implicate a significant role of genetic factors in the responses to analgesic drugs, this study is not free from some limitation. Most of all, the sample size is relatively small for a GWAS since the sample size required is increased by the large number of hypotheses that are tested. Because we genotyped 500K SNPs from collected samples retrospectively, the sample size was not based on power analysis. We did not have required baseline data for power analysis either since GWAS related to analgesic drug efficacy has never been reported, to the best of our knowledge. Our result should be considered as preliminary because of this limitation, though our results can be used in power analysis for the future GWAS of analgesic drug responses with a larger scale of various ethnic populations. It should be also considered that current genotyping platforms represent only about two-thirds of all known common genetic variations throughout the human genome [[25](#R25)]. These limitations may increase the risk of false discovery. Furthermore, our results are limited to European American population only and cannot be generalized to other ethnic populations since pain responses including analgesic efficacy and genetic variations are very different among ethnic populations [[26](#R26)–[29](#R29)]. As with other genetic association studies, replications with larger sample sizes with various ethnic backgrounds by independent investigators are critical to confirm novel findings. Since the genetic association studies can only identify statistical relationships, studies are needed to characterize the underlying mechanisms. When the candidate genetic loci lack annotation, extensive additional work is needed in both animals and humans.

Even though the sample size is relatively small for a GWAS, the genome-wide significant result of this study appears robust. This study suggests that genetic variations in or near the loci encoding DNA binding proteins play a role in the individual variations in responses to analgesic drugs. These observations also suggest that inter-individual variability in analgesic drug response may be induced by variations in genes that modulate transcription in addition to genes that directly affect pharmacodynamics or pharmacokinetics.

## Conclusion

A GWAS of acute clinical pain followed by additional SNP genotyping of neighboring gene suggests that genetic variations in or near the loci encoding DNA binding proteins play a role in the individual variations in responses to analgesic drugs.

## Future perspective

Even with the relatively small sample size, this study provides evidence that whole-genome scale SNP genotyping is a valuable tool for hypothesis generating when searching for novel candidate genomic regions responsible for pain and analgesia in humans. It is necessary to confirm these associations with independent subjects in a bigger sample size as well as with functional genomic studies.

#### Genome-wide association study (GWAS) of analgesic drug response

#### Genome-wide association study (GWAS) of analgesic drug response

#### Genome-wide association study (GWAS) of analgesic drug response

#### Multilayer study

#### ZNF429

## Acknowledgments

This study was supported by the Division of Intramural Research, National Institute of Nursing Research and National Institute of Dental and Craniofacial Research, National Institutes of Health, Bethesda, MD 20892, USA.

## Appendix

The p-value, which is labeled ‘p =’ in a tree node display, can be calculated as follows.

Suppose there are n observations to be split into k subgroups with D unique values of the continuous or ordinal predictor (not counting a possible missing value). Let *F*_0_ be the sum of squared deviations from the mean over all responses and *F*_k_ be the sum over all segments of the squared deviations from the mean responses of each respective segment. Then F-statistic can be defined as:

| F=(F0−Fk)(n−k)Fk(k−1) |
| --- |
The p-value can be obtained from performing an F-test with k-1 and n-k degrees of freedom. The optimal set of segments is found with the smallest p-value and such sets are constructed by recursive partitioning.

## Footnotes

## References

1. Woodcock J, Witter J, Dionne RA. Stimulating the development of mechanism-based, individualized pain therapies. Nat. Rev. Drug Discov. 2007;6(9):703–710. doi: 10.1038/nrd2335.  [DOI](https://doi.org/10.1038/nrd2335) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17762885/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Drug%20Discov&title=Stimulating%20the%20development%20of%20mechanism-based,%20individualized%20pain%20therapies&author=J%20Woodcock&author=J%20Witter&author=RA%20Dionne&volume=6&issue=9&publication_year=2007&pages=703-710&pmid=17762885&doi=10.1038/nrd2335&)

2. Zubieta JK, Heitzeg MM, Smith YR, et al. COMT val158met genotype affects µ-opioid neurotransmitter responses to a pain stressor. Science. 2003;299(5610):1240–1243. doi: 10.1126/science.1078546. ▪ First report of association between COMT and acute experimental pain.

3. Klepstad P, Rakvag TT, Kaasa S, et al. The 118 A > G polymorphism in the human µ-opioid receptor gene may increase morphine requirements in patients with pain caused by malignant disease. Acta Anaesthesiol. Scand. 2004;48(10):1232–1239. doi: 10.1111/j.1399-6576.2004.00517.x.  [DOI](https://doi.org/10.1111/j.1399-6576.2004.00517.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15504181/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Acta%20Anaesthesiol.%20Scand&title=The%20118%20A%20>%20G%20polymorphism%20in%20the%20human%20%C2%B5-opioid%20receptor%20gene%20may%20increase%20morphine%20requirements%20in%20patients%20with%20pain%20caused%20by%20malignant%20disease&author=P%20Klepstad&author=TT%20Rakvag&author=S%20Kaasa&volume=48&issue=10&publication_year=2004&pages=1232-1239&pmid=15504181&doi=10.1111/j.1399-6576.2004.00517.x&)

4. Janicki PK, Schuler G, Francis D, et al. A genetic association study of the functional A118G polymorphism of the human µ-opioid receptor gene in patients with acute and chronic pain. Anesth. Analg. 2006;103(4):1011–1017. doi: 10.1213/01.ane.0000231634.20341.88.  [DOI](https://doi.org/10.1213/01.ane.0000231634.20341.88) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17000822/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anesth.%20Analg&title=A%20genetic%20association%20study%20of%20the%20functional%20A118G%20polymorphism%20of%20the%20human%20%C2%B5-opioid%20receptor%20gene%20in%20patients%20with%20acute%20and%20chronic%20pain&author=PK%20Janicki&author=G%20Schuler&author=D%20Francis&volume=103&issue=4&publication_year=2006&pages=1011-1017&pmid=17000822&doi=10.1213/01.ane.0000231634.20341.88&)

5. Tegeder I, Costigan M, Griffn RS, et al. GTP cyclohydrolase and tetrahydrobiopterin regulate pain sensitivity and persistence. Nat. Med. 2006;12(11):1269–1277. doi: 10.1038/nm1490. ▪ Reported the role of GCH1 genetic variations in acute experimental and chronic clinical pains from multiple clinics. However, population stratification of human samples may cause false-positive findings.

6. Kim H, Dionne RA. Lack of influence of GTP cyclohydrolase gene (GCH1) variations on pain sensitivity in humans. Mol. Pain. 2007;3:6. doi: 10.1186/1744-8069-3-6. ▪ Reported failure to repeat the findings of Tegeder et al. [5] in larger samples.  [5](#R5)

7. Reyes-Gibby CC, Shete S, Rakvag T, et al. Exploring joint effects of genes and the clinical efficacy of morphine for cancer pain: OPRM1 and COMT gene. Pain. 2007;130(1–2):25–30. doi: 10.1016/j.pain.2006.10.023. ▪ First report of joining effect of multiple genetic variations. However, the result shows opposite effect of COMT compared to Zubieta et al. [2].  [2](#R2)

8. Cartegni L, Chew SL, Krainer AR. Listening to silence and understanding nonsense: exonic mutations that affect splicing. Nat. Rev. Genet. 2002;3(4):285–298. doi: 10.1038/nrg775.  [DOI](https://doi.org/10.1038/nrg775) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11967553/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Genet&title=Listening%20to%20silence%20and%20understanding%20nonsense:%20exonic%20mutations%20that%20affect%20splicing&author=L%20Cartegni&author=SL%20Chew&author=AR%20Krainer&volume=3&issue=4&publication_year=2002&pages=285-298&pmid=11967553&doi=10.1038/nrg775&)

9. Mattick JS. The functional genomics of noncoding RNA. Science. 2005;309(5740):1527–1528. doi: 10.1126/science.1117806.  [DOI](https://doi.org/10.1126/science.1117806) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16141063/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=The%20functional%20genomics%20of%20noncoding%20RNA&author=JS%20Mattick&volume=309&issue=5740&publication_year=2005&pages=1527-1528&pmid=16141063&doi=10.1126/science.1117806&)

10. Scuteri A, Sanna S, Chen WM, et al. Genome-wide association scan shows genetic variants in the FTO gene are associated with obesity-related traits. PLoS Genet. 2007;3(7):E115. doi: 10.1371/journal.pgen.0030115.  [DOI](https://doi.org/10.1371/journal.pgen.0030115) | [PMC free article](/articles/PMC1934391/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17658951/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet&title=Genome-wide%20association%20scan%20shows%20genetic%20variants%20in%20the%20FTO%20gene%20are%20associated%20with%20obesity-related%20traits&author=A%20Scuteri&author=S%20Sanna&author=WM%20Chen&volume=3&issue=7&publication_year=2007&pages=E115&pmid=17658951&doi=10.1371/journal.pgen.0030115&)

11. Salonen JT, Uimari P, Aalto JM, et al. Type 2 diabetes whole-genome association study in four populations: the DiaGen consortium. Am. J. Hum. Genet. 2007;81(2):338–345. doi: 10.1086/520599.  [DOI](https://doi.org/10.1086/520599) | [PMC free article](/articles/PMC1950819/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17668382/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Hum.%20Genet&title=Type%202%20diabetes%20whole-genome%20association%20study%20in%20four%20populations:%20the%20DiaGen%20consortium&author=JT%20Salonen&author=P%20Uimari&author=JM%20Aalto&volume=81&issue=2&publication_year=2007&pages=338-345&pmid=17668382&doi=10.1086/520599&)

12. Plenge RM, Seielstad M, Padyukov L, et al. TRAF1-C5 as a risk locus for rheumatoid arthritis – a genomewide study. N. Engl. J. Med. 2007;357(12):1199–1209. doi: 10.1056/NEJMoa073491.  [DOI](https://doi.org/10.1056/NEJMoa073491) | [PMC free article](/articles/PMC2636867/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17804836/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%20Engl.%20J.%20Med&title=TRAF1-C5%20as%20a%20risk%20locus%20for%20rheumatoid%20arthritis%20%E2%80%93%20a%20genomewide%20study&author=RM%20Plenge&author=M%20Seielstad&author=L%20Padyukov&volume=357&issue=12&publication_year=2007&pages=1199-1209&pmid=17804836&doi=10.1056/NEJMoa073491&)

13. Shifman S, Bhomra A, Smiley S, et al. A whole genome association study of neuroticism using DNA pooling. Mol. Psychiatry. 2007;13(3):302–321. doi: 10.1038/sj.mp.4002048.  [DOI](https://doi.org/10.1038/sj.mp.4002048) | [PMC free article](/articles/PMC4004964/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17667963/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Psychiatry&title=A%20whole%20genome%20association%20study%20of%20neuroticism%20using%20DNA%20pooling&author=S%20Shifman&author=A%20Bhomra&author=S%20Smiley&volume=13&issue=3&publication_year=2007&pages=302-321&pmid=17667963&doi=10.1038/sj.mp.4002048&)

14. Redon R, Ishikawa S, Fitch KR, et al. Global variation in copy number in the human genome. Nature. 2006;444(7118):444–454. doi: 10.1038/nature05329.  [DOI](https://doi.org/10.1038/nature05329) | [PMC free article](/articles/PMC2669898/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17122850/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Global%20variation%20in%20copy%20number%20in%20the%20human%20genome&author=R%20Redon&author=S%20Ishikawa&author=KR%20Fitch&volume=444&issue=7118&publication_year=2006&pages=444-454&pmid=17122850&doi=10.1038/nature05329&)

15. Feuk L, Carson AR, Scherer SW. Structural variation in the human genome. Nat. Rev. Genet. 2006;7(2):85–97. doi: 10.1038/nrg1767.  [DOI](https://doi.org/10.1038/nrg1767) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16418744/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Genet&title=Structural%20variation%20in%20the%20human%20genome&author=L%20Feuk&author=AR%20Carson&author=SW%20Scherer&volume=7&issue=2&publication_year=2006&pages=85-97&pmid=16418744&doi=10.1038/nrg1767&)

16. Gonzalez E, Kulkarni H, Bolivar H, et al. The influence of CCL3L1 gene-containing segmental duplications on HIV-1/AIDS susceptibility. Science. 2005;307(5714):1434–1440. doi: 10.1126/science.1101160.  [DOI](https://doi.org/10.1126/science.1101160) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15637236/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=The%20influence%20of%20CCL3L1%20gene-containing%20segmental%20duplications%20on%20HIV-1/AIDS%20susceptibility&author=E%20Gonzalez&author=H%20Kulkarni&author=H%20Bolivar&volume=307&issue=5714&publication_year=2005&pages=1434-1440&pmid=15637236&doi=10.1126/science.1101160&)

17. Matsushima S, Ide T, Yamato M, et al. Overexpression of mitochondrial peroxiredoxin-3 prevents left ventricular remodeling and failure after myocardial infarction in mice. Circulation. 2006;113(14):1779–1786. doi: 10.1161/CIRCULATIONAHA.105.582239.  [DOI](https://doi.org/10.1161/CIRCULATIONAHA.105.582239) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16585391/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation&title=Overexpression%20of%20mitochondrial%20peroxiredoxin-3%20prevents%20left%20ventricular%20remodeling%20and%20failure%20after%20myocardial%20infarction%20in%20mice&author=S%20Matsushima&author=T%20Ide&author=M%20Yamato&volume=113&issue=14&publication_year=2006&pages=1779-1786&pmid=16585391&doi=10.1161/CIRCULATIONAHA.105.582239&)

18. Ikeuchi M, Matsusaka H, Kang D, et al. Overexpression of mitochondrial transcription factor A ameliorates mitochondrial deficiencies and cardiac failure after myocardial infarction. Circulation. 2005;112(5):683–690. doi: 10.1161/CIRCULATIONAHA.104.524835.  [DOI](https://doi.org/10.1161/CIRCULATIONAHA.104.524835) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16043643/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation&title=Overexpression%20of%20mitochondrial%20transcription%20factor%20A%20ameliorates%20mitochondrial%20deficiencies%20and%20cardiac%20failure%20after%20myocardial%20infarction&author=M%20Ikeuchi&author=H%20Matsusaka&author=D%20Kang&volume=112&issue=5&publication_year=2005&pages=683-690&pmid=16043643&doi=10.1161/CIRCULATIONAHA.104.524835&)

19. Lencz T, Morgan TV, Athanasiou M, et al. Converging evidence for a pseudoautosomal cytokine receptor gene locus in schizophrenia. Mol. Psychiatry. 2007;12(6):572–580. doi: 10.1038/sj.mp.4001983.  [DOI](https://doi.org/10.1038/sj.mp.4001983) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17522711/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Psychiatry&title=Converging%20evidence%20for%20a%20pseudoautosomal%20cytokine%20receptor%20gene%20locus%20in%20schizophrenia&author=T%20Lencz&author=TV%20Morgan&author=M%20Athanasiou&volume=12&issue=6&publication_year=2007&pages=572-580&pmid=17522711&doi=10.1038/sj.mp.4001983&)

20. Hargreaves K, Dionne R. Evaluating endogenous mediators of pain and analgesia in clinical studies. In: Max M, Portenoy R, Laska E, editors. Advances in Pain Research and Therapy. Volume 18. New York, NY, USA: Raven Press; 1991. pp. 579–598.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Advances%20in%20Pain%20Research%20and%20Therapy&author=K%20Hargreaves&author=R%20Dionne&publication_year=1991&)

21. Gordon SM, Brahim JS, Rowan J, Kent A, Dionne RA. Peripheral prostanoid levels and nonsteroidal anti-inflammatory drug analgesia: replicate clinical trials in a tissue injury model. Clin. Pharmacol. Ther. 2002;72(2):175–183. doi: 10.1067/mcp.2002.126501.  [DOI](https://doi.org/10.1067/mcp.2002.126501) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12189364/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther&title=Peripheral%20prostanoid%20levels%20and%20nonsteroidal%20anti-inflammatory%20drug%20analgesia:%20replicate%20clinical%20trials%20in%20a%20tissue%20injury%20model&author=SM%20Gordon&author=JS%20Brahim&author=J%20Rowan&author=A%20Kent&author=RA%20Dionne&volume=72&issue=2&publication_year=2002&pages=175-183&pmid=12189364&doi=10.1067/mcp.2002.126501&)

22. Cooper SA, Beaver WT. A model to evaluate mild analgesics in oral surgery outpatients. Clin. Pharmacol. Ther. 1976;20(2):241–250. doi: 10.1002/cpt1976202241.  [DOI](https://doi.org/10.1002/cpt1976202241) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/780041/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther&title=A%20model%20to%20evaluate%20mild%20analgesics%20in%20oral%20surgery%20outpatients&author=SA%20Cooper&author=WT%20Beaver&volume=20&issue=2&publication_year=1976&pages=241-250&pmid=780041&doi=10.1002/cpt1976202241&)

23. Hargreaves KM, Schmidt EA, Mueller GP, Dionne RA. Dexamethasone alters plasma levels of β-endorphin and postoperative pain. Clin. Pharmacol. Ther. 1987;42(6):601–607. doi: 10.1038/clpt.1987.206.  [DOI](https://doi.org/10.1038/clpt.1987.206) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2961501/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther&title=Dexamethasone%20alters%20plasma%20levels%20of%20%CE%B2-endorphin%20and%20postoperative%20pain&author=KM%20Hargreaves&author=EA%20Schmidt&author=GP%20Mueller&author=RA%20Dionne&volume=42&issue=6&publication_year=1987&pages=601-607&pmid=2961501&doi=10.1038/clpt.1987.206&)

24. Weedon MN, Lettre G, Freathy RM, et al. A common variant of HMGA2 is associated with adult and childhood height in the general population. Nat. Genet. 2007;39(10):1245–1250. doi: 10.1038/ng2121.  [DOI](https://doi.org/10.1038/ng2121) | [PMC free article](/articles/PMC3086278/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17767157/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet&title=A%20common%20variant%20of%20HMGA2%20is%20associated%20with%20adult%20and%20childhood%20height%20in%20the%20general%20population&author=MN%20Weedon&author=G%20Lettre&author=RM%20Freathy&volume=39&issue=10&publication_year=2007&pages=1245-1250&pmid=17767157&doi=10.1038/ng2121&)

25. Pe’er I, de Bakker PI, Maller J, et al. Evaluating and improving power in whole-genome association studies using fixed marker sets. Nat. Genet. 2006;38(6):663–667. doi: 10.1038/ng1816. ▪ Emphasizes that even in the highly heritable trait such as height, mostly significant SNP from GWAS can only explain small portion of individual differences.

26. Edwards C, Fillingim R, Keefe F. Race, ethnicity and pain. Pain. 2001;94(2):133–137. doi: 10.1016/S0304-3959(01)00408-0.  [DOI](https://doi.org/10.1016/S0304-3959(01)00408-0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11690726/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pain&title=Race,%20ethnicity%20and%20pain&author=C%20Edwards&author=R%20Fillingim&author=F%20Keefe&volume=94&issue=2&publication_year=2001&pages=133-137&pmid=11690726&doi=10.1016/S0304-3959(01)00408-0&)

27. Gabriel SB, Schaffner SF, Nguyen H, et al. The structure of haplotype blocks in the human genome. Science. 2002;296(5576):2225–2229. doi: 10.1126/science.1069424.  [DOI](https://doi.org/10.1126/science.1069424) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12029063/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=The%20structure%20of%20haplotype%20blocks%20in%20the%20human%20genome&author=SB%20Gabriel&author=SF%20Schaffner&author=H%20Nguyen&volume=296&issue=5576&publication_year=2002&pages=2225-2229&pmid=12029063&doi=10.1126/science.1069424&)

28. Green CR, Ndao-Brumblay SK, Nagrant AM, Baker TA, Rothman E. Race, age, and gender influences among clusters of African American and white patients with chronic pain. J. Pain. 2004;5(3):171–182. doi: 10.1016/j.jpain.2004.02.227.  [DOI](https://doi.org/10.1016/j.jpain.2004.02.227) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15106130/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Pain&title=Race,%20age,%20and%20gender%20influences%20among%20clusters%20of%20African%20American%20and%20white%20patients%20with%20chronic%20pain&author=CR%20Green&author=SK%20Ndao-Brumblay&author=AM%20Nagrant&author=TA%20Baker&author=E%20Rothman&volume=5&issue=3&publication_year=2004&pages=171-182&pmid=15106130&doi=10.1016/j.jpain.2004.02.227&)

29. Kim H, Dionne RA. Comment on Diatchenko et al. Catechol-O-methyltransferase gene polymorphisms are associated with multiple pain-evoking stimuli. Pain. 2007;129(3):365–366. doi: 10.1016/j.pain.2007.02.011. author reply 366–370.  [DOI](https://doi.org/10.1016/j.pain.2007.02.011) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17407801/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pain&title=Comment%20on%20Diatchenko%20et%20al.%20Catechol-O-methyltransferase%20gene%20polymorphisms%20are%20associated%20with%20multiple%20pain-evoking%20stimuli&author=H%20Kim&author=RA%20Dionne&volume=129&issue=3&publication_year=2007&pages=365-366&pmid=17407801&doi=10.1016/j.pain.2007.02.011&)
