# Genome-wide association scan of tag SNPs identifies a susceptibility locus for lung cancer at 15q25.1

## Metadata
**Authors:** Christopher I Amos, Xifeng Wu, Peter Broderick, Ivan P Gorlov, Jian Gu, Timothy Eisen, Qiong Dong, Qing Zhang, Xiangjun Gu, Jayaram Vijayakrishnan, Kate Sullivan, Athena Matakidou, Yufei Wang, Gordon Mills, Kimberly Doheny, Ya-Yu Tsai, Wei Vivien Chen, Sanjay Shete, Margaret R Spitz, Richard S Houlston
**Journal:** Nature genetics
**Date:** 2008 Apr 2
**DOI:** [10.1038/ng.109](https://doi.org/10.1038/ng.109)
**PMID:** 18385676
**PMCID:** PMC2713680
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2713680/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC2713680/pdf/nihms115953.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC2713680/pdf/nihms115953.pdf)

## Abstract

To identify risk variants for lung cancer, we conducted a multistage genome-wide association study. In the discovery phase, we analyzed 315,450 tagging SNPs in 1,154 current and former (ever) smoking cases of European ancestry and 1,137 frequency-matched, ever-smoking controls from Houston, Texas. For replication, we evaluated the ten SNPs most significantly associated with lung cancer in an additional 711 cases and 632 controls from Texas and 2,013 cases and 3,062 controls from the UK. Two SNPs, rs1051730 and rs8034191, mapping to a region of strong linkage disequilibrium within 15q25.1 containing PSMA4 and the nicotinic acetylcholine receptor subunit genes CHRNA3 and CHRNA5, were significantly associated with risk in both replication sets. Combined analysis yielded odds ratios of 1.32 (P < 1 × 10−17) for both SNPs. Haplotype analysis was consistent with there being a single risk variant in this region. We conclude that variation in a region of 15q25.1 containing nicotinic acetylcholine receptors genes contributes to lung cancer risk.

## METHODS

### Study populations

For detailed descriptions of the component studies, see [Supplementary Methods](#SD1) online. The study protocols were approved by the Institutional Review Board of The University of Texas M.D. Anderson Cancer Center and by a review board at the Institute for Cancer Research Foundation. Informed consent was obtained from all participants.

### Genotyping

Genotyping procedures and quality control approaches are described in [Supplementary Methods](#SD1). We retained data from 315,860 SNPs from Illumina analysis that had genotyping results in 90% or more subjects, but 410 were monomorphic for analysis in individuals of European descent (and hence not informative). Confirmatory genotyping in Houston was conducted on an independent sample of 711 cases and 632 controls using a Taqman genotyping platform for the ten most significant SNPs identified in the discovery phase. The Texas replication sample comprised independent cases and controls from the discovery set who were from the same study population source but who tended to be more recently enrolled participants with incomplete frequency matching. Genotyping of UK samples was conducted by competitive allele-specific PCR KASPar chemistry (KBiosciences).

### Statistical analyses

We used similarity in genotypes as implemented in PLINK to identify individuals and clusters of individuals who deviated by more than 4 standard deviations from other study subjects, and we excluded these outliers. We identified genetically related subjects using PLINK software, which uses the similarity in identity by state of genotypes to estimate identity by descent values[17](#R17), setting the clustering value at 0.0001 and excluding 639 markers that deviated from Hardy-Weinberg equilibrium in the controls (*P* < 0.0001) and 584 SNPs with minor allele frequency (MAF) < 0.01.

Association between SNP genotype and disease status was primarily assessed using the allelic 1 degree-of-freedom (d.f.) test or Fisher’s exact test where an expected cell count was <5. We also carried out association analysis using the Armitage-Doll trend test[18](#R18). The ORs associated with each SNP and the 95% confidence intervals were estimated by allele and by genotype using unconditional logistic regression. None of the markers associated with lung cancer risk showed deviations from Hardy-Weinberg equilibrium (*P* > 0.05).

We evaluated the adequacy of the case-control matching and the possibility of differential genotyping of cases and controls using quantile-quantile plots of test statistics. A test inflation factor λ was calculated by dividing the median of the test statistics by the expected median from a χ^2^ distribution with 1 d.f.[19](#R19). The mean and median of the χ^2^ tests in [Figure 1](#F1) were 1.0196 and 0.4675, very close to the expected values of 1.00 and 0.456. Comparison of the median χ^2^ test with its expected value yielded a λ value of 1.025, very close to expected, indicating that population substructure, if present, did not have any substantial effect upon the discovery stage analyses presented here.

We used HelixTree for preliminary analyses and for initial data manipulation; we then transferred data to PLINK[17](#R17) and EIGENSTRAT[20](#R20). We evaluated the association of markers with lung cancer risk allowing for potential effects of population substructure by using a Cochran-Mantel-Haenszel test[22](#R22) in PLINK. Strata were defined by a nearest neighbor cluster analysis of genetic similarity, which identified 44 clusters. We also carried out a second analysis to allow for substructure effects using EIGENSTRAT[20](#R20). All genetic data from the discovery set were used to obtain correlation matrices among the subjects. Spectral analysis was done to extract those eigenvectors explaining the largest proportion of interindividual variation. A scree plot of the associated eigenvalues showed a point of inflection when three eigenvalues were included, and these three eigenvalues alone exceeded 2.0 (results not shown). Results from all analyses were very similar for significantly associated SNPs whether or not adjustments for population structure were made ([Supplementary Table 1](#SD2)).

We used SAS Genetics v9.1 to conduct association tests for Hardy-Weinberg equilibrium and to perform haplotype analyses. Logistic regression, implemented in SAS version 9.1, was used to perform analyses adjusting for smoking and other covariates. We conducted joint analysis of data generated from multiple phases using standard methods for combining raw data based on the Mantel-Haenszel method. We used Cochran’s Q statistic to test for heterogeneity.

We used Haploview[21](#R21) software (v3.2) to infer the LD structure of the genome in the regions containing loci associated with disease risk. To impute SNPs from multimarker tags, we used a procedure described previously[22](#R22) based upon haplotype frequencies from HapMap release 21, build 35.

### Statistical methods for obtaining P values

We obtained *P* values combining data from the discovery phase as well as the two replication phases following a procedure outlined previously[23](#R23). Specifically, we set the critical value for the discovery phase to be the least significant result among the ten SNPs retained for follow up (*P* = 4.9 × 10^−5^). We obtained the joint test statistics by comparing allele frequencies in cases versus controls from all studies according to their sample sizes (results from the two replication phases were combined prior to joint analysis). We used the joint statistic value conditioning on the critical value *P* = 4.9 × 10^−5^ using the program CaTS to estimate the *P* value required to reach observed joint *Z* value. The pointwise *P* value so derived can be adjusted for multiple testing using a Bonferroni approach by multiplying the pointwise *P* value by the number of tests (results not shown). For several cases in which the replication *P* value was very much larger than the discovery *P* value, the CaTS software could not provide a result because of numerical overflow, and these results were indicated by > 1 × 10^−5^, which was the least significant *P* value obtained before the overflow. We also provided *P* values from the replication phase only by combining results from the Texas replication and UK studies, and adjusting for center effects using a Cochran-Mantel-Haenszel procedure implemented in SAS.

### URLs

Haploview, [http://www.broad.mit.edu/mpg/haploview/](http://www.broad.mit.edu/mpg/haploview/); Eigenstrat, [http://genepath.med.harvard.edu/~reich/EIGENSTRAT.htm](http://genepath.med.harvard.edu/~reich/EIGENSTRAT.htm); CaTS, [http://www.sph.umich.edu/csg/abecasis/CaTS/](http://www.sph.umich.edu/csg/abecasis/CaTS/).

## Supplementary Material

## ACKNOWLEDGMENTS

Partial support for this study has been provided by US National Institutes of Health grants R01CA133996, R01CA55769, P50 CA70907 and R01CA121197, the Kleberg Center for Molecular Markers at M.D. Anderson Cancer Center, and by support from the Flight Attendants Medical Research Institute. Genotyping services were provided by the Center for Inherited Disease Research (CIDR). CIDR is fully funded through a federal contract from the National Institutes of Health to The Johns Hopkins University, Contract Number N01-HG-65403. We thank the Kelsey Research Foundation for facilitating control selection in Texas. At the Institute for Cancer Research, work was undertaken with support primarily from Cancer Research UK. We are also grateful to the National Cancer Research Network, HEAL and Sanofi-Aventis. A. Matakidou was the recipient of a clinical research fellowship from the Allan J. Lerner Fund. We are also thankful for the unstinting efforts of the study coordinators and interviewers, including S. Honn, P. Porter, S. Ritter and J. Rogers. We also thank the study participants, who had the most critical role in this research.

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Tokuhata GK, Lilienfeld AM. Familial aggregation of lung cancer in humans. J. Natl. Cancer Inst. 1963;30:289–312.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/13985327/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Natl.%20Cancer%20Inst&title=Familial%20aggregation%20of%20lung%20cancer%20in%20humans&author=GK%20Tokuhata&author=AM%20Lilienfeld&volume=30&publication_year=1963&pages=289-312&pmid=13985327&)

2. Amos CI, Xu W, Spitz MR. Is there a genetic basis for lung cancer susceptibility? Recent Results Cancer Res. 1999;151:3–12. doi: 10.1007/978-3-642-59945-3_1.  [DOI](https://doi.org/10.1007/978-3-642-59945-3_1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10337715/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Recent%20Results%20Cancer%20Res&title=Is%20there%20a%20genetic%20basis%20for%20lung%20cancer%20susceptibility?&author=CI%20Amos&author=W%20Xu&author=MR%20Spitz&volume=151&publication_year=1999&pages=3-12&pmid=10337715&doi=10.1007/978-3-642-59945-3_1&)

3. Jonsson S, et al. Familial risk of lung carcinoma in the Icelandic population. J. Am. Med. Assn. 2004;22:2977–2983. doi: 10.1001/jama.292.24.2977.  [DOI](https://doi.org/10.1001/jama.292.24.2977) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15613665/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Am.%20Med.%20Assn&title=Familial%20risk%20of%20lung%20carcinoma%20in%20the%20Icelandic%20population&author=S%20Jonsson&volume=22&publication_year=2004&pages=2977-2983&pmid=15613665&doi=10.1001/jama.292.24.2977&)

4. Hwang SJ, et al. Lung cancer risk in germline p53 mutation carriers: association between an inherited cancer predisposition, cigarette smoking, and cancer risk. Hum. Genet. 2003;113:238–243. doi: 10.1007/s00439-003-0968-7.  [DOI](https://doi.org/10.1007/s00439-003-0968-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12802680/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum.%20Genet&title=Lung%20cancer%20risk%20in%20germline%20p53%20mutation%20carriers:%20association%20between%20an%20inherited%20cancer%20predisposition,%20cigarette%20smoking,%20and%20cancer%20risk&author=SJ%20Hwang&volume=113&publication_year=2003&pages=238-243&pmid=12802680&doi=10.1007/s00439-003-0968-7&)

5. Sanders BM, Jay M, Draper GJ, Roberts EM. Non-ocular cancer in relatives of retinoblastoma patients. Br. J. Cancer. 1989;60:358–365. doi: 10.1038/bjc.1989.285.  [DOI](https://doi.org/10.1038/bjc.1989.285) | [PMC free article](/articles/PMC2247167/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2789942/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Cancer&title=Non-ocular%20cancer%20in%20relatives%20of%20retinoblastoma%20patients&author=BM%20Sanders&author=M%20Jay&author=GJ%20Draper&author=EM%20Roberts&volume=60&publication_year=1989&pages=358-365&pmid=2789942&doi=10.1038/bjc.1989.285&)

6. Kleinerman RA, et al. Hereditary retinoblastoma and risk of lung cancer. J. Natl. Cancer Inst. 2000;92:2037–2039. doi: 10.1093/jnci/92.24.2037.  [DOI](https://doi.org/10.1093/jnci/92.24.2037) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11121467/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Natl.%20Cancer%20Inst&title=Hereditary%20retinoblastoma%20and%20risk%20of%20lung%20cancer&author=RA%20Kleinerman&volume=92&publication_year=2000&pages=2037-2039&pmid=11121467&doi=10.1093/jnci/92.24.2037&)

7. Takemiya M, Shiraishi S, Teramoto T, Miki Y. Bloom’s syndrome with porokeratosis of Mibelli and multiple cancers of the skin, lung and colon. Clin. Genet. 1987;31:35–44. doi: 10.1111/j.1399-0004.1987.tb02764.x.  [DOI](https://doi.org/10.1111/j.1399-0004.1987.tb02764.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/3568432/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Genet&title=Bloom%E2%80%99s%20syndrome%20with%20porokeratosis%20of%20Mibelli%20and%20multiple%20cancers%20of%20the%20skin,%20lung%20and%20colon&author=M%20Takemiya&author=S%20Shiraishi&author=T%20Teramoto&author=Y%20Miki&volume=31&publication_year=1987&pages=35-44&pmid=3568432&doi=10.1111/j.1399-0004.1987.tb02764.x&)

8. Yamanaka A, Hirai T, Ohtake Y, Kitagawa M. Lung cancer associated with Werner’s syndrome: a case report and review of the literature. Jpn. J. Clin. Oncol. 1997;27:415–418. doi: 10.1093/jjco/27.6.415.  [DOI](https://doi.org/10.1093/jjco/27.6.415) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9438005/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Jpn.%20J.%20Clin.%20Oncol&title=Lung%20cancer%20associated%20with%20Werner%E2%80%99s%20syndrome:%20a%20case%20report%20and%20review%20of%20the%20literature&author=A%20Yamanaka&author=T%20Hirai&author=Y%20Ohtake&author=M%20Kitagawa&volume=27&publication_year=1997&pages=415-418&pmid=9438005&doi=10.1093/jjco/27.6.415&)

9. Bailey-Wilson JE, et al. A major lung cancer susceptibility locus maps to chromosome 6q23–25. Am. J. Hum. Genet. 2004;75:460–474. doi: 10.1086/423857.  [DOI](https://doi.org/10.1086/423857) | [PMC free article](/articles/PMC1182024/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15272417/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Hum.%20Genet&title=A%20major%20lung%20cancer%20susceptibility%20locus%20maps%20to%20chromosome%206q23%E2%80%9325&author=JE%20Bailey-Wilson&volume=75&publication_year=2004&pages=460-474&pmid=15272417&doi=10.1086/423857&)

10. Webb EL, et al. Search for low penetrance alleles for colorectal cancer through a scan of 1467 non-synonymous SNPs in 2575 cases and 2707 controls with validation by kin-cohort analysis of 14,704 first-degree relatives. Hum. Mol. Genet. 2006;15:3263–3271. doi: 10.1093/hmg/ddl401.  [DOI](https://doi.org/10.1093/hmg/ddl401) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17000706/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum.%20Mol.%20Genet&title=Search%20for%20low%20penetrance%20alleles%20for%20colorectal%20cancer%20through%20a%20scan%20of%201467%20non-synonymous%20SNPs%20in%202575%20cases%20and%202707%20controls%20with%20validation%20by%20kin-cohort%20analysis%20of%2014,704%20first-degree%20relatives&author=EL%20Webb&volume=15&publication_year=2006&pages=3263-3271&pmid=17000706&doi=10.1093/hmg/ddl401&)

11. Zhang Q, et al. Nicotine induces hypoxia-inducible factor-1alpha expression in human lung cancer cells via nicotinic acetylcholine receptor-mediated signaling pathways. Clin. Cancer Res. 2007;13:4686–4694. doi: 10.1158/1078-0432.CCR-06-2898.  [DOI](https://doi.org/10.1158/1078-0432.CCR-06-2898) | [PMC free article](/articles/PMC4166418/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17699846/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Cancer%20Res&title=Nicotine%20induces%20hypoxia-inducible%20factor-1alpha%20expression%20in%20human%20lung%20cancer%20cells%20via%20nicotinic%20acetylcholine%20receptor-mediated%20signaling%20pathways&author=Q%20Zhang&volume=13&publication_year=2007&pages=4686-4694&pmid=17699846&doi=10.1158/1078-0432.CCR-06-2898&)

12. Lam DC, et al. Expression of nicotinic acetylcholine receptor subunit genes in non-small-cell lung cancer reveals differences between smokers and nonsmokers. Cancer Res. 2007;67:4638–4647. doi: 10.1158/0008-5472.CAN-06-4628.  [DOI](https://doi.org/10.1158/0008-5472.CAN-06-4628) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17510389/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=Expression%20of%20nicotinic%20acetylcholine%20receptor%20subunit%20genes%20in%20non-small-cell%20lung%20cancer%20reveals%20differences%20between%20smokers%20and%20nonsmokers&author=DC%20Lam&volume=67&publication_year=2007&pages=4638-4647&pmid=17510389&doi=10.1158/0008-5472.CAN-06-4628&)

13. Minna JD. Nicotine exposure and bronchial epithelial cell nicotinic acetylcholine receptor expression in the pathogenesis of lung cancer. J. Clin. Invest. 2003;111:31–33. doi: 10.1172/JCI17492.  [DOI](https://doi.org/10.1172/JCI17492) | [PMC free article](/articles/PMC151841/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12511585/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Invest&title=Nicotine%20exposure%20and%20bronchial%20epithelial%20cell%20nicotinic%20acetylcholine%20receptor%20expression%20in%20the%20pathogenesis%20of%20lung%20cancer&author=JD%20Minna&volume=111&publication_year=2003&pages=31-33&pmid=12511585&doi=10.1172/JCI17492&)

14. Trombino S, et al. Alpha7-nicotinic acetylcholine receptors affect growth regulation of human mesothelioma cells: role of mitogen-activated protein kinase pathway. Cancer Res. 2004;64:135–145. doi: 10.1158/0008-5472.can-03-1672.  [DOI](https://doi.org/10.1158/0008-5472.can-03-1672) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14729617/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=Alpha7-nicotinic%20acetylcholine%20receptors%20affect%20growth%20regulation%20of%20human%20mesothelioma%20cells:%20role%20of%20mitogen-activated%20protein%20kinase%20pathway&author=S%20Trombino&volume=64&publication_year=2004&pages=135-145&pmid=14729617&doi=10.1158/0008-5472.can-03-1672&)

15. Ho YS, et al. Tobacco-specific carcinogen 4-(methylnitrosamino)-1-(3-pyridyl)-1-butanone (NNK) induces cell proliferation in normal human bronchial epithelial cells through NFkappaB activation and cyclin D1 up-regulation. Toxicol. Appl. Pharmacol. 2005;205:133–148. doi: 10.1016/j.taap.2004.09.019.  [DOI](https://doi.org/10.1016/j.taap.2004.09.019) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15893541/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Toxicol.%20Appl.%20Pharmacol&title=Tobacco-specific%20carcinogen%204-(methylnitrosamino)-1-(3-pyridyl)-1-butanone%20(NNK)%20induces%20cell%20proliferation%20in%20normal%20human%20bronchial%20epithelial%20cells%20through%20NFkappaB%20activation%20and%20cyclin%20D1%20up-regulation&author=YS%20Ho&volume=205&publication_year=2005&pages=133-148&pmid=15893541&doi=10.1016/j.taap.2004.09.019&)

16. Saccone SF, et al. Cholinergic nicotinic receptor genes implicated in a nicotine dependence association study targeting 348 candidate genes with 3713 SNPs. Hum. Mol. Genet. 2007;16:36–49. doi: 10.1093/hmg/ddl438.  [DOI](https://doi.org/10.1093/hmg/ddl438) | [PMC free article](/articles/PMC2270437/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17135278/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum.%20Mol.%20Genet&title=Cholinergic%20nicotinic%20receptor%20genes%20implicated%20in%20a%20nicotine%20dependence%20association%20study%20targeting%20348%20candidate%20genes%20with%203713%20SNPs&author=SF%20Saccone&volume=16&publication_year=2007&pages=36-49&pmid=17135278&doi=10.1093/hmg/ddl438&)

17. Purcell S, et al. PLINK: a tool set for whole-genome association and population-based linkage analyses. Am. J. Hum. Genet. 2007;81:559–575. doi: 10.1086/519795.  [DOI](https://doi.org/10.1086/519795) | [PMC free article](/articles/PMC1950838/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17701901/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Hum.%20Genet&title=PLINK:%20a%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses&author=S%20Purcell&volume=81&publication_year=2007&pages=559-575&pmid=17701901&doi=10.1086/519795&)

18. Balding DJ. A tutorial on statistical methods for population association studies. Nat. Rev. Genet. 2006;7:781–791. doi: 10.1038/nrg1916.  [DOI](https://doi.org/10.1038/nrg1916) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16983374/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Genet&title=A%20tutorial%20on%20statistical%20methods%20for%20population%20association%20studies&author=DJ%20Balding&volume=7&publication_year=2006&pages=781-791&pmid=16983374&doi=10.1038/nrg1916&)

19. Devlin B, Roeder K, Wasserman L. Genomic control, a new approach to genetic-based association studies. Theor. Popul. Biol. 2001;60:155–166. doi: 10.1006/tpbi.2001.1542.  [DOI](https://doi.org/10.1006/tpbi.2001.1542) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11855950/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Theor.%20Popul.%20Biol&title=Genomic%20control,%20a%20new%20approach%20to%20genetic-based%20association%20studies&author=B%20Devlin&author=K%20Roeder&author=L%20Wasserman&volume=60&publication_year=2001&pages=155-166&pmid=11855950&doi=10.1006/tpbi.2001.1542&)

20. Price AL, et al. Principal components analysis corrects for stratification in genome-wide association studies. Nat. Genet. 2006;38:904–909. doi: 10.1038/ng1847.  [DOI](https://doi.org/10.1038/ng1847) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16862161/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet&title=Principal%20components%20analysis%20corrects%20for%20stratification%20in%20genome-wide%20association%20studies&author=AL%20Price&volume=38&publication_year=2006&pages=904-909&pmid=16862161&doi=10.1038/ng1847&)

21. Barrett JC, Fry B, Maller J, Daly MJ. Haploview: analysis and visualization of LD and haplotype maps. Bioinformatics. 2005;21:263–265. doi: 10.1093/bioinformatics/bth457.  [DOI](https://doi.org/10.1093/bioinformatics/bth457) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15297300/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=Haploview:%20analysis%20and%20visualization%20of%20LD%20and%20haplotype%20maps&author=JC%20Barrett&author=B%20Fry&author=J%20Maller&author=MJ%20Daly&volume=21&publication_year=2005&pages=263-265&pmid=15297300&doi=10.1093/bioinformatics/bth457&)

22. de Bakker PI, et al. Efficiency and power in genetic association studies. Nat. Genet. 2005;37:1217–1223. doi: 10.1038/ng1669.  [DOI](https://doi.org/10.1038/ng1669) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16244653/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet&title=Efficiency%20and%20power%20in%20genetic%20association%20studies&author=PI%20de%20Bakker&volume=37&publication_year=2005&pages=1217-1223&pmid=16244653&doi=10.1038/ng1669&)

23. Skol AD, Scott LJ, Abecasis GR, Boehnke M. Joint analysis is more efficient than replication-based analysis for two-stage genome-wide association studies. Nat. Genet. 2006;38:209–213. doi: 10.1038/ng1706.  [DOI](https://doi.org/10.1038/ng1706) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16415888/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet&title=Joint%20analysis%20is%20more%20efficient%20than%20replication-based%20analysis%20for%20two-stage%20genome-wide%20association%20studies&author=AD%20Skol&author=LJ%20Scott&author=GR%20Abecasis&author=M%20Boehnke&volume=38&publication_year=2006&pages=209-213&pmid=16415888&doi=10.1038/ng1706&)
