# CYP2C9*1B Promoter Polymorphisms, in Linkage with CYP2C19*2, Affect Phenytoin Autoinduction of Clearance and Maintenance Dose

## Metadata
**Authors:** Amarjit S Chaudhry, Thomas J Urban, Jatinder K Lamba, Angela K Birnbaum, Rory P Remmel, Murali Subramanian, Stephen Strom, Joyce H You, Dalia Kasperaviciute, Claudia B Catarino, Rodney A Radtke, Sanjay M Sisodiya, David B Goldstein, Erin G Schuetz
**Journal:** The Journal of Pharmacology and Experimental Therapeutics
**Date:** 2010 Feb
**DOI:** [10.1124/jpet.109.161026](https://doi.org/10.1124/jpet.109.161026)
**PMID:** 19855097
**PMCID:** PMC2812115
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2812115/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC2812115/pdf/zpt599.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC2812115/pdf/zpt599.pdf)

## Abstract

The commonly prescribed antiepileptic drug phenytoin has a narrow therapeutic range and wide interindividual variability in clearance explained in part by CYP2C9 and CYP2C19 coding variants. After finding a paradoxically low urinary phenytoin metabolite (S)/(R) ratio in subjects receiving phenytoin maintenance therapy with a CYP2C9*1/*1 and CYP2C19*1/*2 genotype, we hypothesized that CYP2C9 regulatory polymorphisms (rPMs), G-3089A and −2663delTG, in linkage disequilibrium with CYP2C19*2 were responsible. These rPMs explained as much as 10% of the variation in phenytoin maintenance dose in epileptic patients, but were not correlated with other patients' warfarin dose requirements or with phenytoin metabolite ratio in human liver microsomes. We hypothesized the rPMs affected CYP2C9 induction by phenytoin, a pregnane X receptor (PXR), and constitutive androstane receptor (CAR) activator. Transfection studies showed that CYP2C9 reporters with wild-type versus variant alleles had similar basal activity but significantly greater phenytoin induction by cotransfected PXR, CAR, and Nrf2 and less Yin Yang 1 transcription factor repression. Phenytoin induction of CYP2C9 was greater in human hepatocytes with the CYP2C9 wild type versus variant haplotype. Therefore, CYP2C9 rPMs affect phenytoin-dependent induction of CYP2C9 and phenytoin metabolism in humans, with an effect size comparable with that for CYP2C9*2 and 2C9*3. These findings may also be relevant to the clinical use of other PXR, CAR, and Nrf2 activators.

## Materials and Methods

### 

#### Materials.

QuikChange II XL site-directed mutagenesis kit was obtained from Stratagene (La Jolla, CA). TRIzol and Superscript reverse transcription-polymerase chain reaction (PCR) systems were from Invitrogen (Carlsbad, CA). The luciferase assay system, β-galactosidase assay system, TnT quick coupled transcription/translation system, and PCR master mix were purchased from Promega (Madison, WI). High-purity plasmid purification kits and DNeasy tissue kit were supplied by QIAGEN (Valencia, CA). Trypsin, penicillin, streptomycin, and cell culture minimal essential media were purchased from Invitrogen (Carlsbad, CA). Fetal bovine serum was purchased form HyClone (Logan, UT). Dimethyl sulfoxide and PHT were purchased from Sigma-Aldrich (St. Louis, MO). The transfection reagent Turbofectin 8, Yin Yang 1 (YY1) expression, and Nrf2 expression plasmids were procured from Origene (Rockville, MD). [γ-^32^P]ATP and [^35^S]methionine were supplied by PerkinElmer Life and Analytical Sciences (Waltham, MA). Exonuclease 1 and shrimp alkaline phosphatase were from USB (Cleveland, OH). FirstChoice RLM-RACE Kit was purchased from Ambion/Applied Biosystems (Austin, TX). Custom oligonucleotide synthesis and DNA sequencing was done by the Hartwell Center for Bioinformatics and Biotechnology at St. Jude Children's Research Hospital (Memphis, TN).

#### In Silico Analysis of the CYP2C9 and CYP2C19 Genes.

Databases screened for potential candidate SNPs in *CYP2C9*, which are in LD with *CYP2C19***2*: HapMap ([http://www.hapmap.org/](http://www.hapmap.org/)) and Perlegen ([http://genome.perlegen.com/](http://genome.perlegen.com/)). Web-based bioinformatic tools Nubiscan ([http://www.nubiscan.unibas.ch](http://www.nubiscan.unibas.ch)) and Transfac ([http://hc-transfac/cgi-bin/biobase/transfac/11.4/bin/start.cgi](http://hc-transfac/cgi-bin/biobase/transfac/11.4/bin/start.cgi)) were used to identify whether polymorphisms altered transcription factor binding sites.

#### Procurement of Liver Tissue.

Human liver studies were approved by the St. Jude Children's Research Hospital and University of Pittsburgh Institutional Review Boards. Human liver tissue (*n* = 28) was processed through the St. Jude Liver Resource at St. Jude Children's Research Hospital and was provided by the Liver Tissue Procurement and Distribution System and by the Cooperative Human Tissue Network. RNA from donor livers was used to analyze the expression of CYP2C9 and to prepare cDNA to study the polymorphic splicing events by amplifying full-length cDNA. DNA from donor livers was used to genotype for *CYP2C9***2*, *CYP2C9***3*, *CYP2C9-3089G*>*A, CYP2C9-2663delTG*, and *CYP2C19***2*.

#### mRNA Quantification.

Total RNA was isolated from liver or human hepatocytes with TRIzol. First-strand cDNA was prepared by use of oligo(dT) primers (Invitrogen, Superscript reverse transcription-PCR system). Real-time PCR quantitation of *CYP2C18* like Ex1/2C9 Ex2 chimera, *CYP2C9, CYP2C19, CYP3A4* mRNA, and the housekeeping control glyceraldehyde-3-phosphate dehydrogenase was carried out by use of the QuantiTect SYBR Green PCR kit (QIAGEN) according to the manufacturer's instructions. cDNA was analyzed in duplicate by quantitative real-time PCR on an ABI PRISM 7900HT Sequence Detection System (Applied Biosystems, Foster City, CA). Primers used for real-time quantification are provided in Supplemental Table 1. Specificity of amplification was confirmed in each case by performing melt curve analysis and agarose gel electrophoresis. The averaged Ct values were analyzed by the comparative Ct method to obtain relative mRNA expression levels.

#### Genotyping of CYP2C9 and CYP2C19 Alleles.

DNA from human livers was isolated by use of a DNeasy tissue kit. PCR was performed by use of 50 ng of genomic DNA template and 1 μM each primer in a 25-μl reaction volume with PCR master mix from Promega. Genotyping for *CYP2C9***2*, *CYP2C9***3*, *CYP2C9-3089G*>*A*, *CYP2C9-2663delTG*, and *CYP2C19***2* was performed by direct DNA sequencing of the specific PCR-amplified fragments. The primers were used for PCR amplification and DNA sequencing and are listed in Supplemental Table 1.

#### Epilepsy Study Population.

The patient population included adult individuals of primarily European ancestry enrolled in a retrospective observational study of adult and pediatric chronic epilepsy patients followed by epileptologists at Duke University Medical Center (*n* = 79), and the National Hospital for Neurology and Neurosurgery, University College London Hospital (*n* = 94). Studies were performed under protocols approved by the appropriate local institutional review boards. Informed consent was either obtained from each patient or from the patients' parent or caregiver when it was not possible to obtain consent from the patient directly. A DNA sample was obtained from each patient, and medical records were reviewed retrospectively. Selected clinical data including a comprehensive past medical history and demographic information, epilepsy diagnosis, seizure type and frequency, drug initiation and stop dates, and adverse effects from AEDs were recorded in a relational database (InForm). We tested whether concomitant antiepileptic medications, or any individual AED, in particular, was associated with PHT maintenance dose; however, these variables did not significantly predict PHT dose requirements. Thus, we treated all individuals achieving the maintenance dose definition as a group, and ignored the potential influence of concomitant medications as a practical measure in our analysis. Genotyping for *CYP2C9***2, CYP2C9***3*, and *CYP2C9-3089G*>*A* was performed by use of Taqman MGB genotyping assays (Applied Biosystems).

#### Phenotyping/Phenotype Definitions.

Maintenance dose was defined as any prescribed dose that remained unchanged for two consecutive clinic visits, not including the initial (referral) visit. If this definition was met more than once for the same drug and patient, the dose that was maintained for the longest period of time was selected as the maintenance dose. It was anticipated that the maintenance dose would tend to reflect, in a quantitative manner, the liability toward seizure control by a given drug for a particular patient, whereas the maximum dose will tend to be limited by the tolerability of the drug, although both maintenance dose and maximum are likely to be influenced by both efficacy and tolerability.

#### Chinese Patients on Warfarin.

This study received human subject approval from the Chinese University of Hong Kong-New Territories East Cluster Clinical Research Ethics Committee. After informed consent was obtained, a cohort of Chinese patients who had warfarin therapy initiated for indications of moderate intensity anticoagulation (target international normalized ratio = 2–3) were recruited at the anticoagulation clinic of the Prince of Wales Hospital in Hong Kong. Exclusion criteria included concurrent use of drugs known to interfere with warfarin metabolism, impaired hepatic function (>1.5 times upper normal limit of serum aminotransferases) or renal function (serum creatinine >120 μM), and patients who had received warfarin for less than 3 months. A 5-ml blood sample was drawn for genotyping during a regularly scheduled clinic visit. Complete warfarin-dosing information was retrieved from medical records to determine stable warfarin dose, defined as three international normalized ratio measurements within target range, at the same mean daily dose, after consecutive clinic visits.

#### Human Liver Microsomes.

Human liver microsomes were prepared from *CYP2C19***2* and *CYP2C9-3089G*>*A* genotyped samples (*n* = 14) and analyzed for the formation of (*S*)- and (*R*)-*p*-HPPH. Microsomal incubations consisted of 100 μl of protein (1.25 mg of protein/ml), 100 μl of 25 mM MgCl_2_, 100 μl of 250 mM Tris buffer, pH 7.4 at 37°C, and 100 μl of 50 μM PHT (10 μM final concentration). This mixture was preincubated at 37°C for 2 min in a shaking-water bath set at 30 rpm, and 100 μl of 5 mM NADPH was added to initiate the reaction. The reaction was stopped after 45 min by adding 100 μl of 20% trichloroacetic acid. Production of *p*-HPPH under these conditions was linear with respect to time and protein concentration. The samples were next placed on ice for 30 min to complete precipitation of the protein, followed by centrifugation at 13,000*g* for 10 min. An aliquot of the supernatant (500 μl) was then subjected to solid-phase extraction on Phenomenex Strata-X (30 mg) columns. Low-speed (50*g*) centrifugation for 30 s was used to elute the solutions and samples through the cartridge in a Forma Scientific (model 5681) swinging-bucket centrifuge (Thermo Fisher Scientific, Waltham, MA). The columns were activated with 1 ml of methanol, followed by conditioning with 1 ml of 10 mM ammonium formate buffer, pH 3. The sample was next added to the column and allowed to stand for 2 min before centrifugation. The cartridges were then washed with 1 ml of 2% methanol in buffer, followed by two washes of 1 ml of buffer solution. The cartridges were allowed to stand for 3 min before centrifugation. Sample elution was carried out with two 1-ml washes of acetonitrile. The final centrifugation was carried out at 75*g* for 1 min. All eluants were collected and dried under nitrogen at 40°C, followed by reconstitution in 100 μl of 50% acetonitrile and 50% ammonium acetate buffer (10 mM). The reconstituted samples were centrifuged at 10,000*g* for 10 min in a Beckman microcentrifuge, and a 30-μl sample was injected into the liquid chromatography-mass spectrometry for analysis. Separation of (*R*)- and (*S*)-*p*-HPPH was accomplished with a Nucleodex β-OH chiral column (200 mm × 4.6 mm) (Macherey-Nagel, Bethlehem, PA) with 1 ml/min of 75% ammonium formate (10 mM, pH 3) and 25% acetonitrile as mobile phase. The retention times for the (*R*)- and (*S*)-*p*-HPPH isomers were 15 and 17 min, respectively. Peak areas were directly measured and an (*S*)/(*R*) *p*-HPPH ratio was determined. Detection was by atmospheric pressure chemical ionization-mass spectrometry in the negative ion mode at an *m*/*z* of 266 on a Shimadzu 2010 liquid chromatography-mass spectrometry instrument (Shimadzu, Kyoto, Japan). The atmospheric pressure chemical ionization probe was maintained at 400°C, the heater block at 200°C, and the desolvation line at 200°C. The detector voltage was 1.6 kV, the microscan set to 0.3 amu, a scan time of 0.5 s, and nitrogen gas flow rate 2.5 liters/min.

#### CYP2C9 Reporter Plasmids.

Cloning of the −5470/+25 *CYP2C9* promoter luciferase reporter plasmid in the pGL3-basic vector (Promega) has been described previously ([Al-Dosari et al., 2006](#B1)). The plasmid, supplied by Dr. Dexi Liu (University of Pittsburgh), was resequenced to genotype for the *CYP2C9*-3089G>A and -2663delTG and found to be WT at these two positions. Site-directed mutagenesis was used to create the VAR *CYP2C9*-3089A plasmid by use of primers in Supplemental Table 1. The *CYP2C9*-2663delTG was generated commercially by GenScript Corp. The *CYP2C9***1B* haplotype was made by a performing site-directed mutagenesis on the *CYP2C9*-2663delTG plasmid. [*CYP2C9***1B* is a designation used by [Veenstra et al. (2005)](#B30), but identical to the *CYP2C9***1D* haplotype described by [Kramer et al. (2008)](#B17).] All nucleotide variations were confirmed by DNA sequencing.

#### Cell Culture and Transient Transfection Assays.

HepG2 human hepatoma cells were cultured in minimal essential media supplemented with 10% fetal bovine serum, 1% penicillin, and 1% streptomycin, and maintained in a humidified incubator at 37°C in an atmosphere of 5% CO_2_. For transfection studies, 3 × 10^5^ cells per well were seeded into 24-well culture dishes. After 24 h cells were transfected with 400 ng of *CYP2C9* promoter luciferase reporter plasmid + 40 ng of HNF-1α and HNF-4α luciferase expression plasmids ± 40 ng each of either pcDNA3-hPXR, pcDNA3-hCAR, pCMV6 Entry-YY1, or pCMV6 Entry-Nrf2 + pcDNA3-MafK. Each tube contained the appropriate amounts of pcDNA3 stuffer DNA to ensure equivalent amounts of total transfected plasmid and 50 ng of pSV40-LacZ to normalize LUC activity. HNF-1α and HNF-4α were always cotransfected because earlier observations demonstrated that these are required for optimal *CYP2C9* promoter activity ([Kramer et al., 2008](#B17)). Twenty-four hours later, transfection medium was replaced with normal growth medium, and cells were incubated for an additional 24 h. For induction experiments, transfection medium was replaced with medium supplemented with 50 μM PHT or dimethyl sulfoxide. After 24 h cell lysates were prepared and luciferase assays were performed according to the manufacturer's instructions. Luciferase activities were normalized with respect to β-galactosidase activity to correct for transfection efficiency and expressed as fold change with respect to vector control. Data are reported as mean ± S.D. of three determinations and were representative of multiple experiments. Statistical differences between groups was determined by use of the Student's *t* test.

#### Electrophoretic Mobility Shift Assay.

hPXR, hCAR, hRXR, hYY1, hNrf2 and Maf K proteins were generated by TnT in vitro transcription and translation per manufacturers' instructions. Proteins were incubated at 4°C for 1.5 h with ^32^P-labeled double-stranded oligonucleotides (600,000 cpm/reaction) in the presence or absence of 100- to 500-fold excess of unlabeled double-stranded oligonucleotide in a DNA-binding buffer [10 mM Tris, pH 8.0, 40 mM KCl, 0.05% Nonidet P40, 6% glycerol, 0.2 μg of poly(dI/dC), 200 μM ZnCl, 1 mM dithiothreitol]. Complexes were resolved by electrophoresis on a nondenaturing 6% polyacrylamide gel and analyzed using a PhosphorImager model 860 Storm (Molecular Dynamics, Sunnyvale, CA). Specific oligonucleotides sequences used are listed in Supplemental Table 1.

#### PHT Treatment of Primary Human Hepatocytes.

Studies in primary hepatocytes isolated from human liver were approved by the University of Pittsburgh Institutional Review Board. Livers were procured from donor organs and were flushed, in situ, and maintained with Belzar's UW solution (Barr Laboratories, Pomona, NY). Hepatocytes were isolated as described previously and were plated on collagen-coated six-well plates maintained in modified Williams' E medium for 48 h and then treated with PHT for 48 h. Media were then aspirated and TRIzol solution (Invitrogen) was added for RNA isolation. DNA was extracted from an aliquot of donor liver for *CYP2C9* genotyping.

#### 5′ Rapid Amplification of cDNA Ends.

5′ rapid amplification of cDNA ends (5′-RACE) was performed in control and PHT-treated human hepatocyte extracts according to the manufacturer's protocol. The 5′RACE and *CYP2C9* gene-specific primers are listed in Supplemental Table 1. RACE products were analyzed by use of 1% agarose gel electrophoresis, cloned into a TOPO TA vector, and sequenced by use of direct DNA sequencing.

#### Statistical Analysis.

Relationship of different genotypes with CYP2C9 mRNA expression, (*S*)/(*R*) *p*-HPPH ratio, and warfarin dose requirements was analyzed by use of the statistical program R:A Language and Environment for Statistical Analysis ([http://www.Rproject.org](http://www.Rproject.org)). Group differences were analyzed nonparametrically with use of the Wilcoxon rank-sum test to compare two genotype groups, whereas the Kruskal-Wallis test was used to compare three groups of genotypes with the phenotypes. Box plots indicate second and third quartiles, with the bold line within the box representing the median value; the whiskers represent the range after excluding the outliers. The outliers are defined by the R package as data points that fall outside the second and third quartiles by more than 1.5 times the interquartile range, and circles falling outside the whiskers represent outliers. Genetic association testing in the chronic epilepsy population was performed by use of multiple linear regression in StataIC10 (StataCorp LP, College Station, TX).

## Results

To test whether a previously unrecognized SNP in *CYP2C9* might explain the paradoxically low CYP2C9 phenytoin metabolite ratios in *CYP2C19***2* carriers, the CYP2C9 cDNA was PCR amplified and resequenced from 12 human livers with different *CYP2C19***2* genotypes (*CYP2C19***1/***1*, *n* = 8; *CYP2C19***1/***2*, *n* = 3; and *CYP2C19***2*/**2*, *n* = 1). Results demonstrated neither unique coding changes in the CYP2C9 cDNA nor alternative CYP2C9 transcripts that might be related to polymorphisms (e.g., intronic SNPs) (not shown).

### 

#### CYP2C9 Polymorphisms in LD with CYP2C19*2.

To identify *CYP2C9* polymorphisms in LD with *CYP2C19***2* we screened available databases and literature and identified at least eight linked *CYP2C9* polymorphisms: two intronic [intron 4 (470 bp downstream of Ex4/Int4 junction) and intron 8 (147 bp downstream of Ex8/Int8 junction)], and six in the 5′ region (−76 kb, −59 kb, −32 kb, −16 kb, and -3089G>A and -2663delTG). The complete haplotype has previously been designated *CYP2C9***1B* ([Veenstra et al., 2005](#B30)). The -3089G>A and -2663delTG polymorphisms were chosen for further study because they are in proximity to binding sites at −2898, −1839, and −1818 for other *CYP2C9* transcriptional regulators (e.g., CAR, constitutive androstane receptor and PXR, pregnane X receptor) that can be activated by PHT ([Fig. 1](#F1)B).

#### CYP2C9 Promoter Polymorphisms Are Associated with Maintenance PHT Dose and Steady-State Serum Levels in Patients with Epilepsy.

Having discovered the *CYP2C9***1B* haplotype through its association with *CYP2C19***2* and its paradoxical effect on urinary (*S*)/(*R*) *p*-HPPH ratio, we then tested independently whether these promoter polymorphisms influenced PHT dose requirements and steady-state PHT serum levels in patients with epilepsy receiving chronic PHT therapy (*n* = 173). After correcting for the effects of *CYP2C9***2* and *CYP2C9***3*, gender and population structure, we found that the *CYP2C9***1B* haplotype was associated with a significant reduction in PHT dose requirements ([Table 1](#T1)). The effect of *CYP2C9***1B* on phenytoin dose was independent of *CYP2C9***2* and *CYP2C9***3* genotype in the linear model ([Table 1](#T1)). The magnitude of the effect of *CYP2C9***1B* was similar to that for the previously identified *CYP2C9***2* and *CYP2C9***3* alleles, with a β-coefficient of −42 (i.e., 42 mg lower dose requirement per minor allele carried) and significantly improved the overall predictive power of *CYP2C9* genetic markers, from (adjusted) *r*^2^ = 0.186 in a model containing only *CYP2C9***2* and *CYP2C9***3* to (adjusted) *r*^2^ = 0.23 in a model containing these markers plus *CYP2C9***1B*, giving an absolute difference of 4.4% of the variation in phenytoin maintenance dose explained by *CYP2C9***1B*. The independent effect of *CYP2C9***1B* is shown graphically in [Fig. 2](#F2) in *n* = 123 patients who do not carry a *CYP2C9***2* or *CYP2C9***3* allele.

| Model | Independent Variables | Beta Coefficient | Adjusted R2 | F | p-Value |
| --- | --- | --- | --- | --- | --- |
| 1 | CYP2C9*2 | −44.6 | 0.113 | 3.44 | 0.004 |
| 2 | CYP2C9*3 | −89.0 | 0.143 | 4.20 | 6.71 × 10−5 |
| 3 | CYP2C9*1B | −21.7 | 0.080 | 2.66 | 0.121 |
| 4 | CYP2C9*2 + | −45.2 | 0.186 | 4.94 | 3.09 × 10−6 |
|   | CYP2C9*3 | −89.6 |   |   |   |
| 5 | CYP2C9*2 + | −105.4 | 0.230 | 5.67 | 1.11 × 10−7 |
|   | CYP2C9*3 + | −52.6 |   |   |   |
|   | CYP2C9*1B | −42.2 |   |   |   |
| 6 | TBW + | 2.52 | 0.371 | 5.06 | 2.70 × 10−5 |
|   | CYP2C9*2 + | −47.9 |   |   |   |
|   | CYP2C9*3 + | −93.2 |   |   |   |
| 7 | TBW + | 2.29 | 0.471 | 6.59 | 5.4 × 10−7 |
|   | CYP2C9*2 + | −65.5 |   |   |   |
|   | CYP2C9*3 + | −121 |   |   |   |
|   | CYP2C9*1B | −66.4 |   |   |   |

Table Caption: Association of CYP2C9 variants with PHT maintenance dose

![Fig. 2.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ad1b/2812115/785108ebeb00/zpt0021087890002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=2812115_zpt0021087890002.jpg)

CYP2C9*1B haplotype is associated with PHT maintenance dose in patients with epilepsy. The box plot shows only those patients found to be noncarriers of the reduced-function CYP2C9*2 and CYP2C9*3 alleles. A linear trend of decreased PHT dose requirements with increasing number of CYP2C9*1B alleles was observed. ∗, p = 0.017 versus CYP2C9*1/*1; ∗∗, p = 0.060 versus CYP2C9*1/*1B.

Because body weight is also a strong independent predictor of PHT dose requirements, we reassessed the *CYP2C9***1B* effect on PHT maintenance dose in a subset of *n* = 70 patients for whom total body weight at the time of PHT therapy was recorded. After correcting for total body weight, addition of *CYP2C9***1B* to the linear model improved the fit from (adjusted) *r*^2^ = 0.371 to (adjusted) *r*^2^ = 0.471, with approximately 10% of the variability in PHT dose explained by this haplotype alone ([Table 1](#T1)). We further inspected an overlapping subset of *n* = 78 patients to determine whether the effect of *CYP2C9***1B* was reflected in higher PHT serum levels in these patients. As expected, there was no relationship between the raw serum level and *CYP2C9***1B* haplotype, because the potential influence on serum level had already been corrected by the reduced maintenance dose (data not shown). However, we observed a significant effect on the ratio of serum level to maintenance dose; addition of *CYP2C9***1B* to a model including *CYP2C9***2*, **3*, gender, and population structure led to improved model fit (*p* = 0.072 without *CYP2C9***1B* versus *p* = 0.033 in the model including *CYP2C9***1B*) ([Table 2](#T2)).

| Model | Independent Variables | Adjusted R2 | p-Value |
| --- | --- | --- | --- |
| 1 | CYP2C9*2 | 0.081 | 0.084 |
| 2 | CYP2C9*3 | −0.063 | 0.899 |
| 3 | CYP2C9*1B | −0.071 | 0.936 |
| 4 | CYP2C9*2 + | 0.092 | 0.072 |
|   | CYP2C9*3 |   |   |
| 5 | CYP2C9*2 + | 0.129 | 0.033 |
|   | CYP2C9*3 + |   |   |
|   | CYP2C9*1B |   |   |

Table Caption: Association of CYP2C9 variants with PHT serum levels normalized to PHT maintenance dose

#### Neither the CYP2C9 Promoter Polymorphisms Nor the CYP2C19*2 Allele Were Associated with a Warfarin-Dosing Requirement in Vivo, Nor Hepatic CYP2C9 Expression or Activity in Vitro.

We next determined the generalizability of the effect of the *CYP2C19***2* genotype (and associated *CYP2C9* promoter polymorphisms) on CYP2C9-mediated drug clearance of another CYP2C9 substrate, warfarin. Because a higher percentage of Asians carry the *CYP2C19***2* allele (18–23%) compared with Whites (2–5%), we turned to a Chinese population (*n* = 67) with data on warfarin dose requirements necessary to achieve anticoagulation. The patients had not previously been genotyped for *CYP2C19***2* or these *CYP2C9* promoter rPMs ([Zhao et al., 2004](#B35)). In the Hong Kong Chinese cohort, the -3089G>A and -2663delTG polymorphisms were in complete LD. However, unlike Whites, although 89% of the Chinese *CYP2C19***2* subjects also had the CYP2C9 regulatory polymorphisms, 10.7% of the Chinese *CYP2C19***2* subjects did not. Therefore, we tested for the individual effects of each of the aforementioned genotypes on warfarin dosing after excluding those patients carrying the *CYP2C9***2*, **3*, and *VKORC1* variant alleles (leaving *n* = 46 patients). Warfarin dosing was not correlated with either the *CYP2C19***2* genotype (not shown) or the *CYP2C9* promoter polymorphisms ([Fig. 3](#F3)A).

![Fig. 3.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ad1b/2812115/641e589d663e/zpt0021087890003.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=2812115_zpt0021087890003.jpg)

A, effect of -3089G>A on warfarin dosing in a cohort of 46 Chinese patients on warfarin therapy after excluding samples with CYP2C9*2 and *3 and VKORC1 variant alleles. B, CYP2C9 mRNA expression in 28 human livers analyzed by quantitative real-time PCR. y-axis depicts mRNA expression each liver (normalized to its GAPDH and this value depicted relative to the normalized value obtained for one liver arbitrarily set at 100%). The x-axis represents visual genotyping results for each polymorphism. Light-gray squares represent homozygous major, dark-gray represent heterozygous, and black represents homozygous minor alleles. Box plot in inset depicts the effect of -3089G>A on mRNA expression after excluding CYP2C9*2 and *3 samples. C, effect of -3089G>A on the in vitro formation of (S)- and (R)-p-HPPH in 14 human liver microsome samples after excluding samples with CYP2C9*2 and *3. Group differences were analyzed nonparametrically using the Wilcoxon rank-sum test to compare two genotypes with the phenotype. Box plots indicate second and third quartiles. The bold line within the box represents median and whiskers represent the range after excluding the outliers.

A human liver resource was used to determine the effect of the *CYP2C9* promoter genotypes on CYP2C9 mRNA expression and phenytoin hydroxylation activity in vitro. Genotyping of 28 human livers from White donors confirmed that the *CYP2C9-3089G*>*A* and -*2663delTG* were in complete LD to *CYP2C19***2* in the White population ([Fig. 3](#F3)B). Quantitative real-time PCR of CYP2C9 mRNA levels showed no significant effect of the promoter polymorphisms on CYP2C9 basal mRNA expression. Next the (*S*)/(*R*) *p*-HPPH ratios were determined in microsomes from these same livers. There was an insignificant difference in hepatic CYP2C9 activity between livers with different *CYP2C19***2* genotypes ([Fig. 3](#F3)C), although we observed an interesting trend in this in vitro phenotype that was opposite to what was seen in vivo. *CYP2C19***2* heterozygote samples demonstrated a trend toward an increased (*S*)/(*R*) *p*-HPPH ratio which is actually what was originally expected in vivo ([Argikar et al., 2006](#B2)) because CYP2C9 is expected to contribute to PHT metabolism to a greater extent in *CYP2C19***2* poor metabolizers ([Fig. 1](#F1)A).

#### To Explain the in Vivo/in Vitro Discrepancy We Further Hypothesized That the CYP2C9 Promoter rPMs Differentially Affected PHT Induction of CYP2C9 Transcription.

PHT is a known activator of PXR and CAR and the subjects with the paradoxical phenotype were on maintenance PHT (induction) therapy. This suggested that the measurement of urinary phenytoin metabolites in vivo reflected induced CYP2C9 activity and the possibility of reduced PHT autoinduction when the two *CYP2C9* regulatory variant alleles were present. The -2663delTG and -3089G>A are in the vicinity of known CAR/PXR binding sites (−2898, −1839, and −1818). In silico bioinformatic analysis using Transfac suggested that the -3089G>A leads to the gain of a YY1 site and the -2663delTG leads to the loss of an Nrf2 site, and Nubiscan suggested possible PXR/CAR binding sites in those same regions ([Fig. 1](#F1)B).

#### Effect of Individual Promoter Polymorphisms on Constitutive and Phenytoin-Induced CYP2C9 Promoter Activity.

Neither the -2663delTG nor the -3089G>A TPMs, when singly engineered into a −5470/+25-*CYP2C9* luciferase reporter plasmid, decreased basal reporter activity ([Fig. 4](#F4), A and B). Compared with the WT reference reporter, both the individual variants decreased CAR, and CAR + PHT induced *CYP2C9* promoter activity ([Fig. 4](#F4), A and B). Because the -3089G>A created a putative YY1 site, we compared the -3089G and -3089A reporter plasmids in cells in which YY1 was overexpressed with an expression plasmid. YY1 overexpression decreased the -3089A variant promoter construct to a significantly greater extent than the WT promoter construct (*p* = 0.04) ([Fig. 4](#F4)A). Because the -2663delTG destroyed a putative Nrf2 site, we compared the two reporter plasmids in cells where Nrf2 (always cotransfected with the heterodimerization partner MafK) was overexpressed. Nrf2 cotransfection significantly increased (*p* = 0.02) the WT promoter construct's luciferase activity compared with the -2663delTG reporter construct, although the variant reporter was induced by PHT/Nrf2 ([Fig. 4](#F4)B).

![Fig. 4.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ad1b/2812115/e2c4be0786df/zpt0021087890004.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=2812115_zpt0021087890004.jpg)

Effect of individual CYP2C9 promoter polymorphisms on basal and PHT-inducible promoter activity. HepG2 cells were transfected with CYP2C9 reporters -3089A and -3089G (A), and -2663TG and -2663delTG (B) with or without cotransfected PXR, CAR, YY1, or NRF2 expression plasmids and luciferase expression measured after treatment with vehicle or PHT for 24 h. Luciferase activities were normalized to β-galactosidase. The relative luciferase normalized activity in vehicle-treated cells were each set as one, and the fold change in relative luciferase activity with respect to vector (w.r.t.) was graphed relative to this baseline. Values represent the mean ± S.D. measured in triplicate. p values indicate where there is a significant difference between the fold change in the variant allele compared with the corresponding wild-type allele for the identical condition. ∗, p = 0.05; ∗∗, p = 0.02; ∗∗∗, p = 0.03; #, p = 0.007; ##, p = 0.0005; ###, p = 0.005.

#### Effect of Promoter Haplotypes on Constitutive and Phenytoin-Induced CYP2C9 Promoter Activity.

Luciferase activity in cells with the wild-type haplotype (WT-HAP) was induced to a significantly greater extent (80% increase) compared with the variant haplotype (VAR-HAP) in cells that were cotransfected with PXR and treated with 50 μM PHT ([Fig. 5](#F5)A) (*p* = 0.002). In contrast, cotransfection with CAR with and without PHT induced luciferase activity from the WT-HAP and VAR-HAP to similar extents (not shown). Conversely, the VAR-HAP luciferase activity was 20% more repressed by YY1 overexpression alone versus the WT-HAP cells ([Fig. 5](#F5)B), and YY1 overexpression demonstrated a significantly greater (*p* = 0.007) repression of luciferase activity in the VAR-HAP cells when cotransfected with PXR and treated with PHT.

![Fig. 5.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ad1b/2812115/c161073aae18/zpt0021087890005.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=2812115_zpt0021087890005.jpg)

The CYP2C9 variant haplotype shows diminished responsiveness to PHT induction in vitro and in vivo. A–C, HepG2 cells were transfected with CYP2C9 WT and VAR-HAP reporters +/ PXR, YY1, or Nrf2 expression plasmids, and luciferase expression measured after treatment with vehicle or PHT for 24 h. Results were analyzed and graphed as in the legend to Fig. 4. p values indicate significant differences between the fold change in the VAR-HAP compared with the corresponding WT-HAP for the identical condition, ∗, p = 0.04; ∗∗, p = 0.02; #, p = 0.007; ##, p = 0.005; ###, p = 0.002.

PHT treatment of Nrf2-cotransfected cells increased luciferase promoter activity of the WT-HAP 43% more than the VAR-HAP (*p* = 0.02) ([Fig. 5](#F5)B). Nrf2 and PXR cotransfection further increased the basal (*p* = 0.04) and PHT induced (*p* = 0.008) promoter activity of the WT-HAP 37% more than the VAR-HAP.

Next, we compared the effect of cotransfection of PXR in combination with YY1 and Nrf2. YY1 repressed activation of the WT and VAR haplotypes by either PXR or Nrf2 cotransfected plasmids, with or without PHT, although the VAR-HAP was repressed to a greater extent (up to 33%) compared with the WT-HAP ([Fig. 5](#F5)C). In total, compared with the WT-HAP, the luciferase activity in cells containing the VAR-HAP was more significantly repressed by YY1 and less induced by PHT.

#### Electrophoretic Mobility Shift Assay Shows the −3089 Variant Allele Creates a YY1 Binding Element.

EMSAs were performed to evaluate any differential binding of YY1, Nrf2, PXR, or CAR to the WT and VAR alleles. The -3089G>A was predicted to create a YY1 binding site. Only the -3089A (VAR) oligonucleotide bound YY1 [lane 12 (VAR) versus lane 4 (WT)], and this binding was displaced by excess cold VAR oligonucleotide ([Fig. 6](#F6)). Neither the -3089G nor the -3089A allele bound CAR:RXR or PXR:RXR heterodimers. YY1 binding seemed to be somewhat diminished by coincubation with either heterodimer. Thus, the EMSA analysis ([Fig. 6](#F6)A) confirmed the in silico prediction that the -3089A VAR allele creates a YY1 binding site. In contrast, although in silico bioinformatic analysis predicted that the -2663delTG destroyed an Nrf2 binding site, neither the -2663TG WT nor -2663delTG oligonucleotides bound the Nrf2:MafK heterodimer, or CAR:RXR or PXR:RXR heterodimers (not shown).

![Fig. 6.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ad1b/2812115/733211826bda/zpt0021087890006.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=2812115_zpt0021087890006.jpg)

EMSA of transcription factor binding to the wild-type and polymorphic CYP2C9 YY1 promoter elements. 32P-labeled oligonucleotides representing the -3089G and A were incubated with products from in vitro synthesis with the empty expression vector (pcDNA3), or synthesized PXR:RXR, CAR:RXR, and YY1 proteins in the absence (no competitor) or presence of increasing amounts of unlabeled oligonucleotide cold competitor (CC). After electrophoresis, complex formation was assessed by a phosphoimager. Lanes 17, 18, and 19 represent binding of YY1, CAR, and PXR to positive control (+ve Ctrl) oligonucleotides, respectively. Sequences for all oligonucleotides are provided in Supplemental Table 1.

#### The Variant CYP2C9 Promoter Haplotype Alters CYP2C9 mRNA Induction by Phenytoin in Primary Human Hepatocytes.

Induction of CYP2C9 mRNA was compared in primary human hepatocytes from 12 donors (nine heterozygous for the VAR haplotype and three with a WT haplotype) treated with phenytoin. Phenytoin treatment resulted in a 3.52-fold induction of CYP2C9 mRNA in hepatocytes with the WT haplotype, but only a 2.07-fold induction in hepatocytes with the VAR haplotype (a 42% decrease relative to the WT haplotype) ([Fig. 7](#F7)). If the single outlier in the VAR haplotype group is removed, the difference in phenytoin induction of CYP2C9 between the two haplotypes reaches statistical significance (*p* = 0.02). No difference in induction of CYP3A4 or CYP2C19 mRNAs were seen between hepatocytes with the different *CYP2C9* promoter haplotypes ([Fig. 7](#F7)).

![Fig. 7.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ad1b/2812115/8cf3ca0d7c7c/zpt0021087890007.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=2812115_zpt0021087890007.jpg)

Effect of CYP2C9 promoter genotypes on CYP expression in human hepatocytes treated with PHT. Primary human hepatocytes were treated with vehicle or PHT for 48 h, RNA extracted and analyzed by quantitative real-time PCR. CYP2C9 and CYP2C18 like Ex1/ 2C9 Ex2 chimeric transcripts were amplified from cDNA using specific primers as shown. The fold change of each mRNA after PHT treatment is depicted on the y-axis and CYP2C9-3089G>A genotypes on the x-axis. Group differences were analyzed nonparametrically using the Wilcoxon rank-sum test as described in Fig. 3 legend.

#### Phenytoin Treatment Does Not Lead To Induction of Unique Alternative CYP2C9 mRNA Transcripts.

We also considered the possibility that *CYP2C9* 5′ regulatory polymorphisms could lead to genesis of PHT-inducible alternative CYP2C9 mRNAs. CYP2C9 alternative mRNAs have been described including a chimeric transcript (*CYP2C18* exon 1-like_*CYP2C9* chimera) that contains the *CYP2C18* exon 1-like sequence spliced through a segment of the CYP2C9 5′-untranslated region that is spliced to *CYP2C9* exons 2 to 9 ([Fig. 7](#F7)), but does not lead to an open reading frame ([Warner et al., 2001](#B32)). The *CYP2C18* exon 1-like sequence is located in the intergenic region between *CYP2C9* and *CYP2C19* genes and one of the SNPs in LD to the *CYP2C19***2* allele was located at −32kb, 320 bp downstream of *CYP2C18* like exon 1. Quantitation of this spliced transcript by quantitative real-time PCR in the phenytoin-treated primary human hepatocytes revealed no induction of the chimera in either haplotype ([Fig. 7](#F7)). To further exclude the possibility that some 5′ regulatory SNP created a unique TF binding site (e.g., CAR or PXR) that led to use of an alternative promoter generating a unique phenytoin-inducible CYP2C9 alternative mRNA, we performed 5′-RACE using cDNAs from vehicle- versus phenytoin-induced primary human hepatocytes. No 5′-RACE product unique to the phenytoin-treated hepatocytes was identified. Two transcripts of equal abundance in vehicle- and PHT-treated hepatocytes from both genotypes were identified by 5′-RACE. The transcripts started 25 bp (more abundant transcript) and 62 bp upstream of the translation start site. Furthermore, when we used 5′- and 3′-untranslated region primers to amplify the full-length CYP2C9 cDNA from control and PHT-treated human hepatocytes there was no evidence by sizing of PCR products on agarose gels of any PHT-inducible alternative CYP2C9 mRNAs in either genotypic group.

## Discussion

We undertook this study to determine why patients with *CYP2C9***1* genotypes, but heterozygous for *CYP2C19***2*, who would be expected to produce (*S*)/(*R*) *p*-HPPH metabolite ratios of 30 to 40:1, unexpectedly produced ratios of only 12.9 ([Argikar et al., 2006](#B2)). We hypothesized that an unrecognized SNP in *CYP2C9*, in LD with *CYP2C19***2*, could explain this surprising finding. In whites, we found *CYP2C19***2* was in complete LD with *CYP2C9***1B* containing the -3089G>A and -2663delTG polymorphisms. These polymorphisms decreased phenytoin-mediated induction of *CYP2C9* transcription in vitro and resulted in a diminished capacity to induce *CYP2C9* mRNA expression in primary human hepatocytes. We propose that this mechanism explains the clinically paradoxical finding of altered PHT clearance in *CYP2C19***2* white ([Argikar et al., 2006](#B2)) and Japanese patients ([Ieiri et al., 1997](#B15)) who were homozygous for *CYP2C9***1*.

In support of this, the current study demonstrates a strong effect of the *CYP2C9***1B* haplotype on PHT metabolism and dose requirements in patients with epilepsy on chronic PHT therapy. Among patients who did not carry the *CYP2C9***2* or *CYP2C9***3* alleles, the *CYP2C9***1B* haplotype was significantly associated with reduced PHT maintenance dose, consistent with reduced PHT induction of CYP2C9 metabolism in *CYP2C9***1B* carriers. It is noteworthy that the full effect of the *CYP2C9***1B* haplotype is not appreciated unless the independent effects of *CYP2C9***2* and *CYP2C9***3* are taken into account (and vice versa). A multivariate model that includes all *CYP2C9* genotype information in addition to gender, body weight, and genomic ancestry showed that nearly half of the variation in PHT maintenance dose can be explained by these predictors ([Table 1](#T1)), with up to 10% of the variation in dose attributable to *CYP2C9***1B* haplotype alone. The magnitude of the effect of *CYP2C9* alleles on PHT dose requirements seemed to strongly depend on the clinical setting; in an additional chronic epilepsy cohort managed with a less aggressive dosing strategy, no effect of any of the functional *CYP2C9* alleles was observed (C. Depondt, personal communication).

Given the magnitude of the effect of the promoter rPMs comprising the *CYP2C9***1B* haplotype on PHT maintenance dose observed here, it may be surprising that this allele has not previously been identified in candidate gene or genome-wide association studies such as those for warfarin, another CYP2C9 substrate. However, several lines of evidence conclusively demonstrate that the promoter rPMs affecting the PHT induction phenotype do not decrease basal CYP2C9 expression: neither hepatic CYP2C9 mRNA expression nor activity were affected by the *CYP2C19***1B* genotypes; and warfarin maintenance dose was not significantly affected by the -3089G>A and -2663delTG polymorphisms in Chinese patients ([Fig. 3](#F3)A). Likewise, [Veenstra et al. (2005](#B30)) previously found no difference in warfarin sensitivity (basal CYP2C9 activity) in white patients with the *CYP2C9***1B* allele.

Although we identified the -3089G>A and -2663delTG *CYP2C9* promoter rPMs by searching for SNPs in LD to *CYP2C19***2*, these rPMs were previously identified in the *CYP2C9***1B* promoter haplotype (allele frequency, 17.45% whites of European descent). Recently [Kramer et al. (2008](#B17)) identified common haplotypes in Hispanics and designated this same allele *CYP2C9***1D* (hereafter referred to as Kramer *1D when referencing results from that publication). It is noteworthy that in transient transfection studies the Kramer *1D haplotype, relative to the reference haplotype, exhibited no difference in constitutive promoter activity but showed a 50% decrease in PXR-mediated promoter activity upon treatment with rifampin. This result is similar to the effect observed here, that the *CYP2C9***1B* promoter had diminished activation by phenytoin. The Kramer *1D haplotype also includes a -1188T>C SNP. Although we did not include this SNP in our CYP2C9 variant reporters, [Kramer et al. (2008](#B17)) deduced that the -1188T>C transition was unlikely to be singly responsible for the differences in induction profiles because it was present in numerous other haplotypes that behaved no differently from the reference allele with respect to either basal or rifampin-inducible activity. Furthermore, a report by [Sandberg et al. (2004)](#B26) found no effect of the -1188T>C variant on gene expression in vitro. Although we cannot exclude the possibility that the -1188T>C might contribute to our phenotype, it would not negate the contribution of -3089G>A or -2663delTG, it would simply imply that these polymorphisms are not the only ones contributing to decreased phenytoin-mediated CYP2C9 induction.

Phenytoin has been shown to activate both PXR and CAR ([Luo et al., 2002](#B22); [Wang et al., 2004](#B31); [Yueh et al., 2005](#B33)). Although Nubiscan predicted possible PXR or CAR binding elements around −3089 and −2663 ([Fig. 1](#F1)), neither receptor bound to the −3089 or −2663 wild-type or variant oligonucleotides and hence cannot explain the decreased induction response. Nevertheless, the VAR-HAP demonstrated a significantly decreased PXR-mediated PHT induction of promoter activity compared with the WT-HAP ([Fig. 5](#F5)). It is also possible that CAR contributes to the differential activation of the CYP2C9 WT and VAR promoters by phenytoin because the CYP2C9 WT reporter was induced to a greater extent by cotransfected CAR compared with reporters with single CYP2C9 promoter variations ([Fig. 4](#F4)). Although it is unclear why the single variant reporters were more induced by CAR ± PHT but the haplotype reporters were more induced by PXR ± PHT, it does not negate the fact that the variant allele was consistently less inducible by PHT.

There are YY1 binding sites predicted in the *CYP2C9* 5′-flanking region and the -3089G>A polymorphism created an additional YY1 binding site. YY1 is a transcription factor that can either induce or repress expression, depending on the target gene. We found that YY1 is a repressor of PHT-inducible CYP2C9 transcription, but the magnitude of repression was always greater in the *CYP2C9* variant reporter with the additional YY1 site. The results suggest that YY1 contributes to the diminished PHT inducibility of the *1B haplotype.

Our data support the idea that PHT activation of Nrf2 could participate in CYP2C9 induction. PHT-activated Nrf2 induces mRNA expression of protective enzymes by binding to antioxidant response elements (ARE) in the promoter of target genes. Although Transfac predicted that the -2663delTG abrogated a putative Nrf2 binding site, EMSA failed to demonstrate Nrf2:MafK binding to oligonucleotides with the -2663TG or TG deletion sequences. Nevertheless, it is still possible that Nrf2 contributes to PHT induction of CYP2C9 because 1) mice treated with PHT showed Keap1-Nrf2-dependent induction of genes that included Cyp2b10 and Cyp2c29 ([Lu et al., 2008](#B20); [Lu and Uetrecht, 2008](#B21)); 2) Transfac analysis showed 21 high scoring putative ARE binding sites in the −5.3 kb *CYP2C9* 5′-flanking sequence, and 3) cotransfection with Nrf2/MafK induced luciferase activity from the *CYP2C9* reporter plasmids. If PHT generates an antioxidant response signal it could be due to PHT metabolites activating the Keap1-Nrf2-ARE signaling pathway.

The low (*S*)/(*R*) *p*-HPPH ratio in *CYP2C19***2* carriers seems to be due to the lower induction of the linked *CYP2C9***1B* allele, and resulting decreased metabolism of PHT to (*S*)-*p*-HPPH, even in the face of the greatly diminished formation of (*R*)-*p*-HPPH by *CYP2C19***2*. We could find no other polymorphism in the CYP2C9 cDNA of *CYP2C19***2/2C9***1B* carriers, nor generation of any novel PHT-induced alternative CYP2C9 transcripts in human hepatocytes with these genotypes that could explain the lower (*S*)-*p*-HPPH or greater (*R*)-metabolite formation. Likewise, 5′-RACE of PHT-treated primary human hepatocytes failed to identify any differentially activated promoter/CYP2C9 transcripts in *CYP2C19***2* subjects. Although we considered the possibility that PHT induction of other metabolic enzymes might contribute to the observed alteration in (*S*)/(*R*) *p*-HPPH ratios in *CYP2C19***2* patients, the literature strongly supports that *p*-HPPH is formed almost exclusively by either CYP2C9 or CYP2C19, with negligible activity by expressed CYP1A2, 2A6, 2B6, 2C8, 2D6, 2E1, or 3A4 ([Komatsu et al., 2000](#B16); [Giancarlo et al., 2001](#B14)). *p*-HPPH can undergo a second hydroxylation to form a catechol metabolite (Midha et al., 1977). This can also be formed via the diol metabolite by dihydrodiol dehydrogenase. Catechol formation from *p*-HPPH seems to be mainly catalyzed by CYP2C19 with additional contribution from CYP3A4 and CYP2C9 ([Cuttle et al., 2000](#B9)). In vivo, the catechol and methylcatechol metabolites are minor accounting for <5% of the dose in the urine. Although the amounts of these other metabolites were not measured in our study, the differential formation of these metabolites, even if they occurred, would seem insufficient to account for the altered *p*-HPPH ratios observed in *CYP2C19***2* patients.

The finding that the *CYP2C9***1B* allele has diminished PHT inducibility is clinically relevant because PHT has been reported to undergo autoinduction ([Dickinson et al., 1985](#B12); [Miller et al., 1989](#B23); [Chetty et al., 1998](#B8)). [Miller et al. (1989)](#B23) reported on two patients who demonstrated an 11 to 20% increase in *V*_max_ after 5 months to 1 year of treatment with PHT. Others have reported modest induction of PHT clearance and a decrease in AUC after repeated doses of PHT ([Yuen et al., 1983](#B34); [Chetty et al., 1998](#B8)). [Dickinson et al. (1985)](#B12) found that eight healthy individuals demonstrated a significant autoinduction of PHT metabolism and urinary excretion of *p*-HPPH increased with repeated administration of PHT. Although PHT autoinduction, in general, is not as appreciated clinically as is another antiepileptic drug, carbamazepine, this may potentially be explained by reduced CYP2C9 inducibility in patients carrying the *CYP2C9***1B* haplotype, which may obscure the PHT autoinduction in studies that have not accounted for this genetic effect.

Our findings, combined with those of [Kramer et al. (2008)](#B17), suggest that the *CYP2C9***1B* allele will not only affect the magnitude of PHT autoinduction of clearance, but also the magnitude of other CYP2C9 drug-drug interactions resulting from PXR/CAR activation. CYP2C9 metabolizes approximately 16% of clinically important drugs, including some with narrow therapeutic indices: sulfonylureas such as tolbutamide, anticonvulsants such as phenytoin, and the anticoagulant warfarin. Other substrates include the antiotensin II blocker losartan, the nonsteroidal anti-inflammatory drugs ibuprofen, naproxen, and flurbiprofen and drugs such as fluvastatin, fluoxetine, and rosiglitazone. Thus, it is possible that the extent of drug interactions after cotreatment with known PXR/CAR activators and CYP2C9 inducers such as rifampin or phenytoin or barbiturates might differ between individuals with the *CYP2C9***1B* versus other *CYP2C9***1* genotypes. Although CYP2C9 is not highly induced in human hepatocytes by phenytoin ([Sahi et al., 2009](#B25)) ([Fig. 7](#F7)), hepatic CYP2C9 is also induced by rifampin, phenobarbital, and avasimibe ([Chen et al., 2004](#B7); [Sahi et al., 2009](#B25)). Moreover, CYP2C9 is among the three most highly induced CYPs in human duodenum in patients treated with clinical doses of oral rifampin (E. Schuetz, unpublished observation). Our results would suggest that the magnitudes of drug interactions are likely to be different between the 17.4% of whites and 6.6% of Hispanics ([Kramer et al., 2008](#B17)) with the *CYP2C9***1B* allele compared with those with other *CYP2C9***1* alleles.

## Supplementary Material

## Acknowledgments

We thank the Hartwell Center for DNA sequencing and oligonucleotide synthesis, and Rinki Singh for assistance in the collection of clinical data.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Al-Dosari MS, Knapp JE, Liu D. (2006) Activation of human CYP2C9 promoter and regulation by CAR and PXR in mouse liver. Mol Pharm 3:322–328  [DOI](https://doi.org/10.1021/mp0500824) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16749864/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Pharm&title=Activation%20of%20human%20CYP2C9%20promoter%20and%20regulation%20by%20CAR%20and%20PXR%20in%20mouse%20liver&author=MS%20Al-Dosari&author=JE%20Knapp&author=D%20Liu&volume=3&publication_year=2006&pages=322-328&pmid=16749864&doi=10.1021/mp0500824&)

2. Argikar UA, Cloyd JC, Birnbaum AK, Leppik IE, Conway J, Kshirsagar S, Oetting WS, Klein EC, Remmel RP. (2006) Paradoxical urinary phenytoin metabolite (S)/(R) ratios in CYP2C19*1/*2 patients. Epilepsy Res 71:54–63  [DOI](https://doi.org/10.1016/j.eplepsyres.2006.05.015) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16815679/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Epilepsy%20Res&title=Paradoxical%20urinary%20phenytoin%20metabolite%20(S)/(R)%20ratios%20in%20CYP2C19*1/*2%20patients&author=UA%20Argikar&author=JC%20Cloyd&author=AK%20Birnbaum&author=IE%20Leppik&author=J%20Conway&volume=71&publication_year=2006&pages=54-63&pmid=16815679&doi=10.1016/j.eplepsyres.2006.05.015&)

3. Aynacioglu AS, Brockmöller J, Bauer S, Sachse C, Güzelbey P, Ongen Z, Nacak M, Roots I. (1999) Frequency of cytochrome P450 CYP2C9 variants in a Turkish population and functional relevance for phenytoin. Br J Clin Pharmacol 48:409–415  [DOI](https://doi.org/10.1046/j.1365-2125.1999.00012.x) | [PMC free article](/articles/PMC2014334/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10510154/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=Frequency%20of%20cytochrome%20P450%20CYP2C9%20variants%20in%20a%20Turkish%20population%20and%20functional%20relevance%20for%20phenytoin&author=AS%20Aynacioglu&author=J%20Brockm%C3%B6ller&author=S%20Bauer&author=C%20Sachse&author=P%20G%C3%BCzelbey&volume=48&publication_year=1999&pages=409-415&pmid=10510154&doi=10.1046/j.1365-2125.1999.00012.x&)

4. Bajpai M, Roskos LK, Shen DD, Levy RH. (1996) Roles of cytochrome P4502C9 and cytochrome P4502C19 in the stereoselective metabolism of phenytoin to its major metabolite. Drug Metab Dispos 24:1401–1403  [PubMed](https://pubmed.ncbi.nlm.nih.gov/8971149/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Roles%20of%20cytochrome%20P4502C9%20and%20cytochrome%20P4502C19%20in%20the%20stereoselective%20metabolism%20of%20phenytoin%20to%20its%20major%20metabolite&author=M%20Bajpai&author=LK%20Roskos&author=DD%20Shen&author=RH%20Levy&volume=24&publication_year=1996&pages=1401-1403&pmid=8971149&)

5. Bhasker CR, Miners JO, Coulter S, Birkett DJ. (1997) Allelic and functional variability of cytochrome P4502C9. Pharmacogenetics 7:51–58  [DOI](https://doi.org/10.1097/00008571-199702000-00007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9110362/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Allelic%20and%20functional%20variability%20of%20cytochrome%20P4502C9&author=CR%20Bhasker&author=JO%20Miners&author=S%20Coulter&author=DJ%20Birkett&volume=7&publication_year=1997&pages=51-58&pmid=9110362&doi=10.1097/00008571-199702000-00007&)

6. Browne TR, Leduc B. (2002). Phenytoin and other hydantoins: chemistry and biotransformation, in Antiepileptic Drugs (Levy RH, Mattson RH, Meldrum BS, Perucca E. eds.) 5th ed, pp 565–580, Lippincott Williams and Wilkins, Philadelphia:  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Antiepileptic%20Drugs&author=TR%20Browne&author=B%20Leduc&publication_year=2002&)

7. Chen Y, Ferguson SS, Negishi M, Goldstein JA. (2004) Induction of human CYP2C9 by rifampicin, hyperforin, and phenobarbital is mediated by the pregnane X receptor. J Pharmacol Exp Ther 308:495–501  [DOI](https://doi.org/10.1124/jpet.103.058818) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14600250/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacol%20Exp%20Ther&title=Induction%20of%20human%20CYP2C9%20by%20rifampicin,%20hyperforin,%20and%20phenobarbital%20is%20mediated%20by%20the%20pregnane%20X%20receptor&author=Y%20Chen&author=SS%20Ferguson&author=M%20Negishi&author=JA%20Goldstein&volume=308&publication_year=2004&pages=495-501&pmid=14600250&doi=10.1124/jpet.103.058818&)

8. Chetty M, Miller R, Seymour MA. (1998) Phenytoin auto-induction. Ther Drug Monit 20:60–62  [DOI](https://doi.org/10.1097/00007691-199802000-00011) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9485556/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ther%20Drug%20Monit&title=Phenytoin%20auto-induction&author=M%20Chetty&author=R%20Miller&author=MA%20Seymour&volume=20&publication_year=1998&pages=60-62&pmid=9485556&doi=10.1097/00007691-199802000-00011&)

9. Cuttle L, Munns AJ, Hogg NA, Scott JR, Hooper WD, Dickinson RG, Gillam EM. (2000) Phenytoin metabolism by human cytochrome P450: involvement of P450 3A and 2C forms in secondary metabolism and drug-protein adduct formation. Drug Metab Dispos 28:945–950  [PubMed](https://pubmed.ncbi.nlm.nih.gov/10901705/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Phenytoin%20metabolism%20by%20human%20cytochrome%20P450:%20involvement%20of%20P450%203A%20and%202C%20forms%20in%20secondary%20metabolism%20and%20drug-protein%20adduct%20formation&author=L%20Cuttle&author=AJ%20Munns&author=NA%20Hogg&author=JR%20Scott&author=WD%20Hooper&volume=28&publication_year=2000&pages=945-950&pmid=10901705&)

10. de Morais SM, Wilkinson GR, Blaisdell J, Nakamura K, Meyer UA, Goldstein JA. (1994) The major genetic defect responsible for the polymorphism of S-mephenytoin metabolism in humans. J Biol Chem 269:15419–15422  [PubMed](https://pubmed.ncbi.nlm.nih.gov/8195181/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=The%20major%20genetic%20defect%20responsible%20for%20the%20polymorphism%20of%20S-mephenytoin%20metabolism%20in%20humans&author=SM%20de%20Morais&author=GR%20Wilkinson&author=J%20Blaisdell&author=K%20Nakamura&author=UA%20Meyer&volume=269&publication_year=1994&pages=15419-15422&pmid=8195181&)

11. Desta Z, Zhao X, Shin JG, Flockhart DA. (2002) Clinical significance of the cytochrome P450 2C19 genetic polymorphism. Clin Pharmacokinet 41:913–958  [DOI](https://doi.org/10.2165/00003088-200241120-00002) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12222994/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacokinet&title=Clinical%20significance%20of%20the%20cytochrome%20P450%202C19%20genetic%20polymorphism&author=Z%20Desta&author=X%20Zhao&author=JG%20Shin&author=DA%20Flockhart&volume=41&publication_year=2002&pages=913-958&pmid=12222994&doi=10.2165/00003088-200241120-00002&)

12. Dickinson RG, Hooper WD, Patterson M, Eadie MJ, Maguire B. (1985) Extent of urinary excretion of p-hydroxyphenytoin in healthy subjects given phenytoin. Ther Drug Monit 7:283–289  [DOI](https://doi.org/10.1097/00007691-198507030-00008) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/4049465/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ther%20Drug%20Monit&title=Extent%20of%20urinary%20excretion%20of%20p-hydroxyphenytoin%20in%20healthy%20subjects%20given%20phenytoin&author=RG%20Dickinson&author=WD%20Hooper&author=M%20Patterson&author=MJ%20Eadie&author=B%20Maguire&volume=7&publication_year=1985&pages=283-289&pmid=4049465&doi=10.1097/00007691-198507030-00008&)

13. Garrard J, Harms SL, Eberly LE, Leppik IE. (2007) Use of antiepileptic medications in nursing homes. Int Rev Neurobiol 81:165–182  [DOI](https://doi.org/10.1016/S0074-7742(06)81010-X) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17433923/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20Rev%20Neurobiol&title=Use%20of%20antiepileptic%20medications%20in%20nursing%20homes&author=J%20Garrard&author=SL%20Harms&author=LE%20Eberly&author=IE%20Leppik&volume=81&publication_year=2007&pages=165-182&pmid=17433923&doi=10.1016/S0074-7742(06)81010-X&)

14. Giancarlo GM, Venkatakrishnan K, Granda BW, von Moltke LL, Greenblatt DJ. (2001) Relative contributions of CYP2C9 and 2C19 to phenytoin 4-hydroxylation in vitro: inhibition by sulfaphenazole, omeprazole, and ticlopidine. Eur J Clin Pharmacol 57:31–36  [DOI](https://doi.org/10.1007/s002280100268) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11372587/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Clin%20Pharmacol&title=Relative%20contributions%20of%20CYP2C9%20and%202C19%20to%20phenytoin%204-hydroxylation%20in%20vitro:%20inhibition%20by%20sulfaphenazole,%20omeprazole,%20and%20ticlopidine&author=GM%20Giancarlo&author=K%20Venkatakrishnan&author=BW%20Granda&author=LL%20von%20Moltke&author=DJ%20Greenblatt&volume=57&publication_year=2001&pages=31-36&pmid=11372587&doi=10.1007/s002280100268&)

15. Ieiri I, Mamiya K, Urae A, Wada Y, Kimura M, Irie S, Amamoto T, Kubota T, Yoshioka S, Nakamura K, et al. (1997) Stereoselective 4′-hydroxylation of phenytoin: relationship to (S)-mephenytoin polymorphism in Japanese. Br J Clin Pharmacol 43:441–445  [DOI](https://doi.org/10.1046/j.1365-2125.1997.00572.x) | [PMC free article](/articles/PMC2042765/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9146858/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=Stereoselective%204%E2%80%B2-hydroxylation%20of%20phenytoin:%20relationship%20to%20(S)-mephenytoin%20polymorphism%20in%20Japanese&author=I%20Ieiri&author=K%20Mamiya&author=A%20Urae&author=Y%20Wada&author=M%20Kimura&volume=43&publication_year=1997&pages=441-445&pmid=9146858&doi=10.1046/j.1365-2125.1997.00572.x&)

16. Komatsu T, Yamazaki H, Asahi S, Gillam EM, Guengerich FP, Nakajima M, Yokoi T. (2000) Formation of a dihydroxy metabolite of phenytoin in human liver microsomes/cytosol: roles of cytochromes P450 2C9, 2C19, and 3A4. Drug Metab Dispos 28:1361–1368  [PubMed](https://pubmed.ncbi.nlm.nih.gov/11038165/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Formation%20of%20a%20dihydroxy%20metabolite%20of%20phenytoin%20in%20human%20liver%20microsomes/cytosol:%20roles%20of%20cytochromes%20P450%202C9,%202C19,%20and%203A4&author=T%20Komatsu&author=H%20Yamazaki&author=S%20Asahi&author=EM%20Gillam&author=FP%20Guengerich&volume=28&publication_year=2000&pages=1361-1368&pmid=11038165&)

17. Kramer MA, Rettie AE, Rieder MJ, Cabacungan ET, Hines RN. (2008) Novel CYP2C9 promoter variants and assessment of their impact on gene expression. Mol Pharmacol 73:1751–1760  [DOI](https://doi.org/10.1124/mol.107.044149) | [PMC free article](/articles/PMC2413059/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18310303/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Pharmacol&title=Novel%20CYP2C9%20promoter%20variants%20and%20assessment%20of%20their%20impact%20on%20gene%20expression&author=MA%20Kramer&author=AE%20Rettie&author=MJ%20Rieder&author=ET%20Cabacungan&author=RN%20Hines&volume=73&publication_year=2008&pages=1751-1760&pmid=18310303&doi=10.1124/mol.107.044149&)

18. Küpfer A, Preisig R. (1984) Pharmacogenetics of mephenytoin: a new drug hydroxylation polymorphism in man. Eur J Clin Pharmacol 26:753–759  [DOI](https://doi.org/10.1007/BF00541938) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/6489416/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Clin%20Pharmacol&title=Pharmacogenetics%20of%20mephenytoin:%20a%20new%20drug%20hydroxylation%20polymorphism%20in%20man&author=A%20K%C3%BCpfer&author=R%20Preisig&volume=26&publication_year=1984&pages=753-759&pmid=6489416&doi=10.1007/BF00541938&)

19. Leppik IE. (2007) Epilepsy in the elderly: scope of the problem. Int Rev Neurobiol 81:1–14  [DOI](https://doi.org/10.1016/S0074-7742(06)81001-9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17433914/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20Rev%20Neurobiol&title=Epilepsy%20in%20the%20elderly:%20scope%20of%20the%20problem&author=IE%20Leppik&volume=81&publication_year=2007&pages=1-14&pmid=17433914&doi=10.1016/S0074-7742(06)81001-9&)

20. Lu W, Li X, Uetrecht JP. (2008) Changes in gene expression induced by carbamazepine and phenytoin: testing the danger hypothesis. J Immunotoxicol 5:107–113  [DOI](https://doi.org/10.1080/15476910802085723) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18569379/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Immunotoxicol&title=Changes%20in%20gene%20expression%20induced%20by%20carbamazepine%20and%20phenytoin:%20testing%20the%20danger%20hypothesis&author=W%20Lu&author=X%20Li&author=JP%20Uetrecht&volume=5&publication_year=2008&pages=107-113&pmid=18569379&doi=10.1080/15476910802085723&)

21. Lu W, Uetrecht JP. (2008) Peroxidase-mediated bioactivation of hydroxylated metabolites of carbamazepine and phenytoin. Drug Metab Dispos 36:1624–1636  [DOI](https://doi.org/10.1124/dmd.107.019554) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18463199/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Peroxidase-mediated%20bioactivation%20of%20hydroxylated%20metabolites%20of%20carbamazepine%20and%20phenytoin&author=W%20Lu&author=JP%20Uetrecht&volume=36&publication_year=2008&pages=1624-1636&pmid=18463199&doi=10.1124/dmd.107.019554&)

22. Luo G, Cunningham M, Kim S, Burn T, Lin J, Sinz M, Hamilton G, Rizzo C, Jolley S, Gilbert D, et al. (2002) CYP3A4 induction by drugs: correlation between a pregnane X receptor reporter gene assay and CYP3A4 expression in human hepatocytes. Drug Metab Dispos 30:795–804  [DOI](https://doi.org/10.1124/dmd.30.7.795) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12065438/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=CYP3A4%20induction%20by%20drugs:%20correlation%20between%20a%20pregnane%20X%20receptor%20reporter%20gene%20assay%20and%20CYP3A4%20expression%20in%20human%20hepatocytes&author=G%20Luo&author=M%20Cunningham&author=S%20Kim&author=T%20Burn&author=J%20Lin&volume=30&publication_year=2002&pages=795-804&pmid=12065438&doi=10.1124/dmd.30.7.795&)

23. Miller R, Bill PL, Du Toit J. (1989) Phenytoin auto-induction? Case reports. S Afr Med J 75:332–333  [PubMed](https://pubmed.ncbi.nlm.nih.gov/2928882/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=S%20Afr%20Med%20J&title=Phenytoin%20auto-induction?%20Case%20reports&author=R%20Miller&author=PL%20Bill&author=J%20Du%20Toit&volume=75&publication_year=1989&pages=332-333&pmid=2928882&)

24. Rettie AE, Haining RL, Bajpai M, Levy RH. (1999) A common genetic basis for idiosyncratic toxicity of warfarin and phenytoin. Epilepsy Res 35:253–255  [DOI](https://doi.org/10.1016/s0920-1211(99)00017-0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10413320/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Epilepsy%20Res&title=A%20common%20genetic%20basis%20for%20idiosyncratic%20toxicity%20of%20warfarin%20and%20phenytoin&author=AE%20Rettie&author=RL%20Haining&author=M%20Bajpai&author=RH%20Levy&volume=35&publication_year=1999&pages=253-255&pmid=10413320&doi=10.1016/s0920-1211(99)00017-0&)

25. Sahi J, Shord SS, Lindley C, Ferguson S, LeCluyse EL. (2009) Regulation of cytochrome P450 2C9 expression in primary cultures of human hepatocytes. J Biochem Mol Toxicol 23:43–58  [DOI](https://doi.org/10.1002/jbt.20264) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19202563/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biochem%20Mol%20Toxicol&title=Regulation%20of%20cytochrome%20P450%202C9%20expression%20in%20primary%20cultures%20of%20human%20hepatocytes&author=J%20Sahi&author=SS%20Shord&author=C%20Lindley&author=S%20Ferguson&author=EL%20LeCluyse&volume=23&publication_year=2009&pages=43-58&pmid=19202563&doi=10.1002/jbt.20264&)

26. Sandberg M, Johansson I, Christensen M, Rane A, Eliasson E. (2004) The impact of CYP2C9 genetics and oral contraceptives on cytochrome P450 2C9 phenotype. Drug Metab Dispos 32:484–489  [DOI](https://doi.org/10.1124/dmd.32.5.484) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15100169/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=The%20impact%20of%20CYP2C9%20genetics%20and%20oral%20contraceptives%20on%20cytochrome%20P450%202C9%20phenotype&author=M%20Sandberg&author=I%20Johansson&author=M%20Christensen&author=A%20Rane&author=E%20Eliasson&volume=32&publication_year=2004&pages=484-489&pmid=15100169&doi=10.1124/dmd.32.5.484&)

27. Stubbins MJ, Harries LW, Smith G, Tarbit MH, Wolf CR. (1996) Genetic analysis of the human cytochrome P450 CYP2C9 locus. Pharmacogenetics 6:429–439  [DOI](https://doi.org/10.1097/00008571-199610000-00007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8946475/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Genetic%20analysis%20of%20the%20human%20cytochrome%20P450%20CYP2C9%20locus&author=MJ%20Stubbins&author=LW%20Harries&author=G%20Smith&author=MH%20Tarbit&author=CR%20Wolf&volume=6&publication_year=1996&pages=429-439&pmid=8946475&doi=10.1097/00008571-199610000-00007&)

28. Sullivan-Klose TH, Ghanayem BI, Bell DA, Zhang ZY, Kaminsky LS, Shenfield GM, Miners JO, Birkett DJ, Goldstein JA. (1996) The role of the CYP2C9-Leu359 allelic variant in the tolbutamide polymorphism. Pharmacogenetics 6:341–349  [DOI](https://doi.org/10.1097/00008571-199608000-00007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8873220/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=The%20role%20of%20the%20CYP2C9-Leu359%20allelic%20variant%20in%20the%20tolbutamide%20polymorphism&author=TH%20Sullivan-Klose&author=BI%20Ghanayem&author=DA%20Bell&author=ZY%20Zhang&author=LS%20Kaminsky&volume=6&publication_year=1996&pages=341-349&pmid=8873220&doi=10.1097/00008571-199608000-00007&)

29. Tate SK, Depondt C, Sisodiya SM, Cavalleri GL, Schorge S, Soranzo N, Thom M, Sen A, Shorvon SD, Sander JW, et al. (2005) Genetic predictors of the maximum doses patients receive during clinical use of the anti-epileptic drugs carbamazepine and phenytoin. Proc Natl Acad Sci U S A 102:5507–5512  [DOI](https://doi.org/10.1073/pnas.0407346102) | [PMC free article](/articles/PMC556232/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15805193/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&title=Genetic%20predictors%20of%20the%20maximum%20doses%20patients%20receive%20during%20clinical%20use%20of%20the%20anti-epileptic%20drugs%20carbamazepine%20and%20phenytoin&author=SK%20Tate&author=C%20Depondt&author=SM%20Sisodiya&author=GL%20Cavalleri&author=S%20Schorge&volume=102&publication_year=2005&pages=5507-5512&pmid=15805193&doi=10.1073/pnas.0407346102&)

30. Veenstra DL, Blough DK, Higashi MK, Farin FM, Srinouanprachan S, Rieder MJ, Rettie AE. (2005) CYP2C9 haplotype structure in European American warfarin patients and association with clinical outcomes. Clin Pharmacol Ther 77:353–364  [DOI](https://doi.org/10.1016/j.clpt.2005.01.019) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15900281/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=CYP2C9%20haplotype%20structure%20in%20European%20American%20warfarin%20patients%20and%20association%20with%20clinical%20outcomes&author=DL%20Veenstra&author=DK%20Blough&author=MK%20Higashi&author=FM%20Farin&author=S%20Srinouanprachan&volume=77&publication_year=2005&pages=353-364&pmid=15900281&doi=10.1016/j.clpt.2005.01.019&)

31. Wang H, Faucette S, Moore R, Sueyoshi T, Negishi M, LeCluyse E. (2004) Human constitutive androstane receptor mediates induction of CYP2B6 gene expression by phenytoin. J Biol Chem 279:29295–29301  [DOI](https://doi.org/10.1074/jbc.M400580200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15123723/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Human%20constitutive%20androstane%20receptor%20mediates%20induction%20of%20CYP2B6%20gene%20expression%20by%20phenytoin&author=H%20Wang&author=S%20Faucette&author=R%20Moore&author=T%20Sueyoshi&author=M%20Negishi&volume=279&publication_year=2004&pages=29295-29301&pmid=15123723&doi=10.1074/jbc.M400580200&)

32. Warner SC, Finta C, Zaphiropoulos PG. (2001) Intergenic transcripts containing a novel human cytochrome P450 2C exon 1 spliced to sequences from the CYP2C9 gene. Mol Biol Evol 18:1841–1848  [DOI](https://doi.org/10.1093/oxfordjournals.molbev.a003726) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11557791/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Biol%20Evol&title=Intergenic%20transcripts%20containing%20a%20novel%20human%20cytochrome%20P450%202C%20exon%201%20spliced%20to%20sequences%20from%20the%20CYP2C9%20gene&author=SC%20Warner&author=C%20Finta&author=PG%20Zaphiropoulos&volume=18&publication_year=2001&pages=1841-1848&pmid=11557791&doi=10.1093/oxfordjournals.molbev.a003726&)

33. Yueh MF, Kawahara M, Raucy J. (2005) High volume bioassays to assess CYP3A4-mediated drug interactions: induction and inhibition in a single cell line. Drug Metab Dispos 33:38–48  [DOI](https://doi.org/10.1124/dmd.104.001594) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15466163/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=High%20volume%20bioassays%20to%20assess%20CYP3A4-mediated%20drug%20interactions:%20induction%20and%20inhibition%20in%20a%20single%20cell%20line&author=MF%20Yueh&author=M%20Kawahara&author=J%20Raucy&volume=33&publication_year=2005&pages=38-48&pmid=15466163&doi=10.1124/dmd.104.001594&)

34. Yuen GJ, Bell RD, Ludden TM. (1983) Phenytoin cumulation profiles. Res Commun Chem Pathol Pharmacol 42:355–368  [PubMed](https://pubmed.ncbi.nlm.nih.gov/6665297/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Res%20Commun%20Chem%20Pathol%20Pharmacol&title=Phenytoin%20cumulation%20profiles&author=GJ%20Yuen&author=RD%20Bell&author=TM%20Ludden&volume=42&publication_year=1983&pages=355-368&pmid=6665297&)

35. Zhao F, Loke C, Rankin SC, Guo JY, Lee HS, Wu TS, Tan T, Liu TC, Lu WL, Lim YT, et al. (2004) Novel CYP2C9 genetic variants in Asian subjects and their influence on maintenance warfarin dose. Clin Pharmacol Ther 76:210–219  [DOI](https://doi.org/10.1016/j.clpt.2004.05.005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15371982/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Novel%20CYP2C9%20genetic%20variants%20in%20Asian%20subjects%20and%20their%20influence%20on%20maintenance%20warfarin%20dose&author=F%20Zhao&author=C%20Loke&author=SC%20Rankin&author=JY%20Guo&author=HS%20Lee&volume=76&publication_year=2004&pages=210-219&pmid=15371982&doi=10.1016/j.clpt.2004.05.005&)
