# CYP4F2 genetic variant alters required warfarin dose

## Metadata
**Authors:** Michael D Caldwell, Tarif Awad, Julie A Johnson, Brian F Gage, Mat Falkowski, Paul Gardina, Jason Hubbard, Yaron Turpaz, Taimour Y Langaee, Charles Eby, Cristi R King, Amy Brower, John R Schmelzer, Ingrid Glurich, Humberto J Vidaillet, Steven H Yale, Kai Qi Zhang, Richard L Berg, James K Burmester
**Journal:** Blood
**Date:** 2008 Feb 4
**DOI:** [10.1182/blood-2007-11-122010](https://doi.org/10.1182/blood-2007-11-122010)
**PMID:** 18250228
**PMCID:** PMC2288721
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2288721/

## Abstract

Warfarin is an effective, commonly prescribed anticoagulant used to treat and prevent thrombotic events. Because of historically high rates of drug-associated adverse events, warfarin remains underprescribed. Further, interindividual variability in therapeutic dose mandates frequent monitoring until target anticoagulation is achieved. Genetic polymorphisms involved in warfarin metabolism and sensitivity have been implicated in variability of dose. Here, we describe a novel variant that influences warfarin requirements. To identify additional genetic variants that contribute to warfarin requirements, screening of DNA variants in additional genes that code for drug-metabolizing enzymes and drug transport proteins was undertaken using the Affymetrix drug-metabolizing enzymes and transporters panel. A DNA variant (rs2108622; V433M) in cytochrome P450 4F2 (CYP4F2) was associated with warfarin dose in 3 independent white cohorts of patients stabilized on warfarin representing diverse geographic regions in the United States and accounted for a difference in warfarin dose of approximately 1 mg/day between CC and TT subjects. Genetic variation of CYP4F2 was associated with a clinically relevant effect on warfarin requirement.

## Introduction

Warfarin is the only oral anticoagulant approved by the US Food and Drug Administration. Although it has been used for more than 50 years, initiation of therapy remains problematic because of interindividual variability in degree of anticoagulation achieved in response to the same warfarin dose. Thus, an appropriate warfarin dose in one patient may induce a hemorrhagic event in another

Warfarin initiation is associated with one of the highest adverse event rates for any single drug.^1–4^ Because of concerns regarding warfarin-induced bleeding, particularly in the elderly, physicians are often reluctant to initiate warfarin therapy in patients where it is warranted. Up to one-half of patients with atrial fibrillation and no contraindication to warfarin therapy, who are also at high risk of stroke (annual risk > 4%), are currently not receiving anticoagulation therapy because of the risk perceived to be associated with such therapy by both patients and healthcare providers.^5^ Thus, more reliable dosing strategies for warfarin initiation that approximate optimal maintenance dosing could improve the risk-benefit ratio, allowing a broader range of patients to safely benefit from warfarin treatment.

Modeling stable dose requirements based on clinical, physiologic, environmental, and genetic factors has shown promise as a strategic approach to predict individualized stable warfarin dose requirements.^6–12^ Patients who have variant alleles of cytochrome P450 (CYP) 2C9, the primary enzyme that metabolizes S-warfarin, require reduced maintenance doses compared with those having wild-type alleles.^13–15^ Warfarin dosing variability is also attributable to genetic polymorphisms in vitamin K 2,3 epoxide reductase complex 1 (VKORC1),^6,16–19^ the rate-limiting enzyme in the warfarin sensitive, vitamin K-dependent gamma carboxylation system. By incorporating these factors, it may be possible to decrease the time to achieve stable dose and risk of serious or life-threatening hemorrhagic events in patients with variant alleles compared with patients with the wild-type genotype. To date, however, most pharmacogenetic models explain just more than one-half of the variation in warfarin dose,^6–9^ suggesting that additional genetic, environmental, medical, or personal factors are important.

Genetic screening using the Affymetrix drug-metabolizing enzymes and transporters (DMET) chip was undertaken to explore whether additional polymorphisms of drug-metabolizing enzymes might contribute to warfarin dosing variability in a clinically important way. Here we provide evidence that a polymorphism in *CYP4F2* affects warfarin dose requirements, and we speculate on its mechanism of action in the vitamin K pathway.

## Methods

### Participating subjects

#### Screening/discovery cohort.

The initial cohort was recruited at Marshfield Clinic, a multispecialty group practice in Wisconsin as described.^6,15^ All patients recruited at Marshfield Clinic were established on a stable warfarin dose and had stable control of anticoagulation. Patients were excluded from the study if they were known to have cancer, renal or hepatic insufficiency, or congestive heart failure.

#### Validation cohorts.

A second cohort of patients stabilized on warfarin therapy was recruited at the University of Florida as described.^7^ Patients were excluded from this study if they had liver cirrhosis, advanced malignancy, hospitalization within the previous 4 weeks of the index visit, or febrile/diarrheal illness within 2 weeks of the index visit.

A third cohort of patients was recruited at Washington University in St Louis as described.^20^ Patients were excluded from this study if they had liver cirrhosis, advanced malignancy, or febrile/diarrheal illness within 2 weeks of the index visit.

Protocols at all 3 participating institutions were approved by the respective Institutional Review Board, and all subjects provided informed written consent in accordance with the Declaration of Helsinki for participation, and use of their genetic sample in studies aimed at understanding warfarin dose variability. To limit population stratification, all subjects in this study were white.

### DMET panel testing

The Targeted Human DMET 1.0 assay (Affymetrix, Santa Clara, CA) uses molecular inversion probes and GeneChip universal tag arrays to target 1228 single nucleotide polymorphism (SNP) positions within 170 DMETs.^21,22^ Polymorphisms were selected for the panel as known biomarkers or likely candidates to affect drug response, including SNPs targeting CYP proteins, non-CYP enzymes, and transport proteins ([Table 1](#T1)). Each molecular inversion probe recognizes a specific locus but lacks the polymorphic nucleotide at the SNP site. Separate incubations with each of the 4 deoxyribonucleotide triphosphates (dNTPs) led to polymerization and ligation only if the added dNTP is complementary to the base at the target site. If this gap-filling step is successful, the probe will subsequently be polymerase chain reaction (PCR)-amplified and labeled with a different fluorescent dye for each dNTP incubation. The specificity of the allele signal is therefore determined at the gap-filling step rather than by chip hybridization, because the latter is accomplished through generic tags on the probe constructs. In addition to the biallelic assays, there are also 27 insertion/deletion probes and one triallelic probe. The DMET assay has been shown to be highly accurate, at approximately 99.8% correctly called genotypes.^23^

### Table 1.

| DMET metabolism |  | DMET metabolism |  | DMET metabolism |  | DMET transport and other genes |  |
| --- | --- | --- | --- | --- | --- | --- | --- |
| Gene | SNP count | Gene | SNP count | Gene | SNP count | Gene | SNP count |
| ADH1B | 3 | CYP2C18 | 4 | GSTM4 | 2 | ABCB1 | 14 |
| ADH4 | 4 | CYP2C19 | 1 | GSTO1 | 1 | ABCB4 | 9 |
| ADH7 | 2 | CYP2D6 | 7 | GSTP1 | 3 | ABCB11 | 15 |
| ALDH1A1 | 1 | CYP2E1 | 7 | HNMT | 1 | ABCC1 | 10 |
| ALDH2 | 1 | CYP2F1 | 2 | MAOB | 1 | ABCC2 | 8 |
| ALDH3A1 | 2 | CYP2J2 | 1 | NAT1 | 7 | ABCC3 | 3 |
| AOX1 | 1 | CYP2S1 | 1 | NAT2 | 6 | ABCC4 | 12 |
| CDA | 2 | CYP39A1 | 2 | NQO1 | 4 | ABCC5 | 3 |
| CHST1 | 1 | CYP3A4 | 2 | POR | 1 | ABCC6 | 4 |
| CHST2 | 1 | CYP3A43 | 3 | PTGIS | 1 | ABCG2 | 1 |
| CHST3 | 13 | CYP3A5 | 1 | SULT1A1 | 1 | AHR | 2 |
| CHST5 | 4 | CYP3A7 | 1 | SULT1A2 | 1 | ATP7A | 1 |
| CHST7 | 1 | CYP4A11 | 1 | SULT1A3 | 1 | ATP7B | 10 |
| CHST10 | 5 | CYP4B1 | 6 | SULT1C1 | 1 | NR3C1 | 3 |
| CHST11 | 7 | CYP4F2 | 7 | SULT1C2 | 1 | PPARD | 38 |
| CHST13 | 2 | CYP4F8 | 4 | SULT1E1 | 2 | PPARG | 1 |
| CHST8 | 1 | CYP4F11 | 5 | SULT2A1 | 1 | RALBP1 | 3 |
| COMT | 2 | CYP4F12 | 4 | SULT2B1 | 2 | SLC10A1 | 1 |
| CYP11A1 | 1 | CYP4Z1 | 1 | SULT4A1 | 2 | SLC10A2 | 7 |
| CYP11B1 | 10 | CYP51A1 | 3 | TBXAS1 | 2 | SLC13A1 | 2 |
| CYP11B2 | 4 | CYP7A1 | 1 | TPMT | 3 | SLC15A1 | 5 |
| CYP17A1 | 3 | CYP8B1 | 1 | UGT1A1 | 5 | SLC15A2 | 5 |
| CYP19A1 | 5 | DPYD | 2 | UGT1A3 | 3 | SLC16A1 | 2 |
| CYP1A1 | 2 | EPHX1 | 7 | UGT1A4 | 1 | SLC19A1 | 2 |
| CYP1A2 | 6 | FMO1 | 4 | UGT1A6 | 2 | SLC22A1 | 7 |
| CYP1B1 | 1 | FMO2 | 10 | UGT1A7 | 1 | SLC22A2 | 2 |
| CYP20A1 | 2 | FMO3 | 6 | UGT2A1 | 2 | SLC22A3 | 2 |
| CYP24A1 | 4 | FMO5 | 1 | UGT2B4 | 2 | SLC22A4 | 3 |
| CYP2A13 | 3 | FMO6 | 4 | UGT2B11 | 1 | SLC22A5 | 3 |
| CYP2A6 | 13 | GSTA1 | 1 | UGT2B15 | 1 | SLC22A6 | 1 |
| CYP2A7 | 3 | GSTA2 | 4 | UGT2B28 | 1 | SLC22A8 | 2 |
| CYP2B6 | 8 | GSTA4 | 4 | UGT8 | 1 | SLC28A1 | 8 |
| CYP2C8 | 2 | GSTA5 | 1 | XDH | 4 | SLC28A2 | 1 |
| CYP2C9 | 2 | GSTM3 | 2 |   |   | SLC28A3 | 2 |
|   |   |   |   |   |   | SLC29A1 | 1 |
|   |   |   |   |   |   | SLC5A6 | 2 |
|   |   |   |   |   |   | SLC7A5 | 1 |
|   |   |   |   |   |   | SLC7A7 | 5 |
|   |   |   |   |   |   | SLCO1A2 | 3 |
|   |   |   |   |   |   | SLCO1B1 | 5 |
|   |   |   |   |   |   | SLCO1B3 | 4 |
|   |   |   |   |   |   | SLCO2B1 | 1 |
|   |   |   |   |   |   | SPG7 | 2 |

Table 1 Caption: Genes containing SNPs with more than 1% minor allele frequency in the Marshfield population

The laboratory protocol followed the standard process described in the Targeted Genotyping System User Guide,^24^ except for a preliminary PCR amplification step aimed at resolving genotypes from 29 SNPs in 14 regions containing pseudogenes and close homologs.^25^ The region containing *CYP4F2* was not included in this class of preamplified genes.

GeneChip Targeted Genotyping Analysis software was used to make genotype calls and generate quality control (QC) metrics, including call rates, average signal, cluster quality, Hardy-Weinberg equilibrium, and reproducibility. Of the 1228 SNP assays, 11 failed because of low call rate (< 70%) and were removed from the analysis. The average call rate of the remaining 1217 SNP assays was more than 98%. Genotypes represented by the passed SNP assays were called according to either predefined boundaries (derived from historical training sets) or dynamic clustering using the current dataset. From an initial sample population of 497 patients, 491 passed all QC metrics and produced useable genotypes. In the Marshfield population, 517 SNPs across 144 genes showed a minor allele frequency more than 1% and were used in the association study. The remaining polymorphisms target rare mutations or SNPs prevalent in other racial groups. Linkage disequilibrium estimates were generated with Haploview software (Broad Institute, Cambridge, MA).^26^

### Genotyping methods

*VKORC1* genotypes were determined on 436 Marshfield Clinic samples as described^6^ and on the remaining 61 samples using Invader chemistry in a laboratory test developed with analyte specific reagents manufactured by Third Wave Technologies (TWT, Madison, WI). The TWT assay used analyte-specific reagents that detect *CYP2C9**2, *CYP2C9**3, and *VKORC1* (position −1639).^27^

### Genotyping QC procedures

To confirm results from the DMET panel, *CYP4F2* genotypes were subsequently validated with 100% concordance for 9 samples using Big Dye Terminator, version 3.1 cycle sequencing with results read on an Applied Biosystems Prism 3100 Genetic Analyzer. Twenty-one additional samples were tested using Invader CYP4F2 research use only reagents provided by TWT with 100% concordance of genotype. This TWT assay was previously used in the Japanese Millennium project^28,29^ and the International Haplotype Map project. Analytic performance of the TWT research-use-only assay was validated using synthetic targets for both alleles and 4 sets of trios from the HapMap project obtained from Coriell Institute for Medical Research (Camden, NJ).

QC procedures for *CYP4F2* at the University of Florida and Washington University in St Louis included genotyping blinded to therapeutic dose. Approximately 5% of samples were duplicate genotyped at Washington University in St Louis with concordance. Genotyping of 15 samples of known genotype was also performed at the University of Florida.

### Statistical analysis

Candidate SNPs were screened for association with the residuals from the Marshfield pharmacogenetic model^6^ for 429 Marshfield subjects. The Kruskal-Wallis test was used to compare the distribution of residuals among observed SNP categories for each SNP. Because a large number of SNPs were to be screened, a Bonferroni correction to the usual 5% nominal level (*P* < .05/1228 = .00004) was set as a stringent initial threshold for statistical significance.

To validate the single SNP (rs2108622) that stood out strongly and was biologically plausible, the result was initially validated in 61 additional Marshfield subjects. The observed association was next validated in 2 independent cohorts from the University of Florida (295 white subjects, 3 missing *VKORC1*) and Washington University in St Louis (269 white subjects). These data were classified according to SNP genotype and evaluated relative to raw therapeutic dose and residuals from the Marshfield model. [Table 2](#T2) shows the allele frequencies at the 3 sites. Genetic data were tested for deviation from Hardy-Weinberg proportions using weighted least squares estimates of allele frequencies and χ^2^ goodness-of-fit tests. The rs2108622 SNP did not deviate from Hardy-Weinberg equilibrium (*P* = .37).

### Table 2.

|   | Marshfield, % | University of Florida, % | Washington University in St Louis, % |
| --- | --- | --- | --- |
| CYP2C9 |   |   |   |
| *1/*1 | 64.69 | 71.19 | 64.31 |
| *1/*2 | 18.98 | 15.93 | 20.07 |
| *1/*3 | 12.45 | 8.81 | 11.52 |
| *2/*2 | 1.43 | 3.39 | 1.49 |
| *2/*3 | 2.04 | 0.68 | 1.86 |
| *3/*3 | 0.41 | 0.00 | 0.74 |
| VKORC1 |   |   |   |
| GG | 38.78 | 36.99 | 37.92 |
| GC | 48.37 | 48.63 | 47.21 |
| CC | 12.86 | 14.38 | 14.87 |
| rs2108622 |   |   |   |
| CC | 49.59 | 47.80 | 52.42 |
| CT | 42.04 | 44.75 | 40.15 |
| TT | 8.37 | 7.46 | 7.43 |

Table 2 Caption: Genotype by site

In a final set of analyses, data from all 3 sites were pooled (n = 1051 total subjects) and used to fit a series of new regression models to fully evaluate the potential contribution of rs2108622 to interindividual variability in therapeutic warfarin dose and to investigate the consistency of the effect across study sites. All models were fit with the log of stable warfarin dose as the response. The association of *CYP2C9* and therapeutic warfarin dose has been well established with substantially reduced stable dose requirements in individuals with no wild-type *CYP2C9* alleles (ie, no *1 alleles). However, given the uncertain effect and relatively low frequency of multiple variant *CYP2C9* alleles, we did not try to quantify their effect. Instead, we conducted multiple regression analyses only on patients with at least one wild-type (*CYP2C9**1) allele (n = 1009) and used the observed mean dose (log scale) as the predicted value for those with 2 *CYP2C9* variants (ie, *2/*2, *2/*3, and *3/*3, n = 42). To assess the relative contribution of clinical and genetic factors, 3 models were developed using different sets of potential predictor variables: clinical factors only (sex, age, body surface area, and target international normalized ratio), clinical factors plus *CYP2C9* and *VKORC1*, and clinical factors *CYP2C9*, *VKORC1*, and rs2108622. A modified step-down approach^30^ was used, starting with a “full” model, which included all potential predictors pertinent to that model, effects to allow the 3 study sites to differ, 2-factor interactions (including those by study site), and terms allowing deviations from linearity for the genetic factors age, body surface area, and target international normalized ratio. Terms were then cautiously sequentially excluded from this full model in a hierarchical fashion (no interactions retained without main effects) if they showed very weak associations with dose (*P* < .5 in the initial steps, *P* < .1 for the final model). The regression models were finalized after evaluating residuals and excluding cases with large residuals as outliers (Studentized residual > 3). At most, 8 cases (< 1% of the cohort) were excluded as outliers during model fitting (these outliers were included for model assessment, as described in the next paragraph).

The explanatory power for the 3 final models was assessed graphically and by using the adjusted r-squared (R^2^_adj_). The R^2^_adj_ statistic measures the proportion of the total variability explained by the model with adjustment for the number of parameters in the model. The statistic was directly calculated to use all cases under analysis by pooling across both parts of the composite models and by applying the final regression model to any outliers dropped when estimating the parameters in the regression models.

## Results

### Discovery of novel SNPs affecting warfarin dose requirements

We used the DMET genotyping platform to assay 1228 SNPs. Of these, 517 were more than 1% polymorphic in the Marshfield (white) population. The 517 polymorphisms are distributed within 144 genes thought to be medically relevant ([Table 1](#T1)).

In our initial screening analysis, we tested each SNP for association with the residual variability from the Marshfield prediction model, evaluating each SNP for the proportion of residual variability explained by it after adjusting for known clinical factors and *CYP2C9* and *VKORC1*. One SNP, rs2108622, which represents a polymorphism in *CYP4F2*, stood out dramatically in the analysis ([Figure 1](#F1)). The estimated *P* value of 2.4 × 10^−7^ was well below our stringent initial threshold for statistical significance of 4.0 × 10^−5^.

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f4bb/2288721/6e30c64e8beb/zh80080817950001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=2288721_zh80080817950001.jpg)

Depiction of relative statistical relationships of SNPs with predicted warfarin therapeutic dose, Marshfield model. The P value (log scale) for each polymorphism, comparing residuals for warfarin dose, was plotted. Dashed line shows the adjusted threshold for significance.

Signals from SNP rs2108622 on the DMET panel produced 3 well-formed and distinct genotype clusters with a Hardy-Weinberg equilibrium *P* value of .83 and a 100% call rate. The minor allele frequency for rs2108622 in the Marshfield population was 29.1%. Minor allele frequencies for this SNP in the Centre d'Etude du Polymorphisme Humain (from various locations in the state of Utah), Han Chinese (Beijing, China), and Japanese (Tokyo, Japan) HapMap populations^31^ were 23.3%, 26.7%, and 23.3%, respectively, but only 5.8% in the Yoruban (Ibadan, Nigeria) population.

Three additional SNPs on the DMET panel, rs3093114, rs3093106, and rs3093105, were in partial linkage disequilibrium with rs2108622 with r^2^ values of 0.42, 0.49, and 0.48, respectively, in the Marshfield population. Each showed strong association with warfarin therapeutic dose, although none of these SNPs attained statistical significance under our stringent threshold. Several other SNPs in *CYP4F2* (rs3093216, rs3093211, rs3093209, rs3093207, rs3093199, rs3093193) from the PGA European Panel ([http://pga.gs.washington.edu](http://pga.gs.washington.edu)) were found in linkage disequilibrium, with rs2108622 with r^2^ values ranging from 0.70 to 0.75. Of these listed SNPs in linkage disequilibrium with rs2108622, only rs3093105 produces a nonsynonymous change in the coding sequence.

### Validation of SNP rs2108622

Results for 61 new Marshfield cases were obtained for initial confirmation of the association between rs2108622 and warfarin dose. In these new cases, the residuals showed association with rs2108622 (*P* = .023). Based on validation in this small Marshfield sample, the association was further tested in 2 independent cohorts from geographically distinct regions of the country.

[Figure 2](#F2) depicts the distribution of warfarin therapeutic dose by *CYP4F2* genotype for the full Marshfield cohort and the 2 validation cohorts. The raw data in [Figure 2](#F2) have not been adjusted for other clinical or genetic factors; but by inspection of the relationships between dose and *CYP4F2*, genetic variants are consistent across the 3 study groups with CC homozygotes requiring less warfarin. TT homozygotes required more warfarin and heterozygotes (CT) required intermediate warfarin doses. This pattern is repeated in [Figure 3](#F3) where we have controlled for clinical factors, as well as *CYP2C9* and *VKORC1* genotypes, by evaluating the distributions of residuals from the Marshfield model. Study site–specific analyses of these relationships yielded statistically significant results for the Marshfield (*P* < .001) and University of Florida (*P* = .027) study cohorts; results for the Washington University in St Louis cohort (*P* = .382) were not statistically significant, but the trend was consistent across sites.

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f4bb/2288721/852b7859ac78/zh80080817950002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=2288721_zh80080817950002.jpg)

Warfarin therapeutic dose by CYP4F2 genotype and study site. Box plots of dose by study site and CYP4F2 (all cases). Boxes extend from the 25th to the 75th percentiles, with a horizontal line at the median and vertical lines extending to the 10th and 90th percentiles.

### Figure 3.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f4bb/2288721/acff91f4eaf5/zh80080817950003.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=2288721_zh80080817950003.jpg)

Marshfield model residuals by CYP4F2 genotype and study site (*1 genotypes only). Model adjusts for age, gender, body surface area, indication for warfarin, VKORC1, and CYP2C9. Boxes extend from the 25th to the 75th percentiles, with a horizontal line at the median and vertical lines extending to the 10th and 90th percentiles.

Multiple regression models were fit to the combined data from all 3 study sites to better assess the overall contribution of *CYP4F2* to variability in dose. Results from the models are summarized by predictor contribution to variation in therapeutic warfarin dose in [Table 3](#T3). Clinical factors alone explained approximately 17% of the overall variability among patients in therapeutic warfarin dose. Clinical factors together with *CYP2C9* and *VKORC1* genotype explained approximately 54% of the variability, whereas the addition of *CYP4F2* genotype increased this R^2^_adj_ to 56%.

### Table 3.

| Predictor/predictor group | Adjusted R2 |
| --- | --- |
| Clinical only | 0.17 |
| Clinical plus CYP2C9 and VKORC1 | 0.54 |
| Clinical plus CYP2C9 and VKORC1 and CYP4F2 | 0.56 |

Table 3 Caption: Explained variation in therapeutic warfarin dose pooled models

Study site–specific effects were evaluated in the multiple regression models to examine the consistency of dosing and *CYP4F2* effects. Some difference in the general level of dosing was observed by study site ([Figure 2](#F2)), with somewhat higher doses at the University of Florida (*P* = .006) and slightly higher doses at Washington University in St Louis (*P* = .15) relative to Marshfield. With respect to *CYP4F2*, study site effects were significant at Washington University in St Louis (*P* = .029) but not the University of Florida (*P* = .19) relative to Marshfield. Study site–specific estimates of the *CYP4F2* effect in the pooled model showed a 12% increase with each T allele at Marshfield compared with 7% at the University of Florida and 4% at Washington University in St Louis.

## Discussion

We have identified a DNA variant in *CYP4F2* that has a clinically important impact on stable warfarin dose. In our pooled analyses, patients with 2 TT alleles require approximately 1 mg/day more warfarin than patients with 2 CC alleles.

Because it is frequently difficult to generalize the original association between a gene variant and phenotype,^32^ we replicated our original observation in 2 additional white cohorts of patients stabilized on warfarin. Importantly, all 3 cohorts demonstrated a consistent effect of *CYP4F2* rs2108622 on warfarin dose. There was a 4% to 12% increase in the warfarin dose per T allele. We postulate that the observed study site–related variation in effect may be the result of differences among cohorts with respect to other genetic and clinical factors not included in our current model. However, because no other studies have pooled data from multiple, independent study sites to estimate the effects of polymorphisms on warfarin stable dose, we do not know whether the effects we have observed are typical of pooled study site models compared with single study site models. This would more likely be the result if single study site models were developed on populations that were more genetically homogeneous as is the case with the Marshfield model.

Because of differences in the frequency of the underlying genetic variants among major racial groups, the potential clinical benefit from prospective *CYP4F2* genotyping varies by race. In whites and Asians, the minor allele frequency for *CYP4F2* is approximately 30% compared with approximately 7% in blacks ([http://www.ncbi.nlm.nih.gov/SNP/index.html](http://www.ncbi.nlm.nih.gov/SNP/index.html)). Accordingly, from a population perspective, the expected contribution of this polymorphism to stable warfarin dose in blacks is likely to be less than in whites and Asians.

The physiologic role of CYP4F2 in the vitamin K/warfarin pathway is unknown. It is known, however, that CYP4F2 hydroxylates the tocopherol phytyl side chain as the first step in the inactivation pathway of vitamin E.^33^ Given the similarity of the vitamin E and vitamin K side chains, CYP4F2 may hydroxylate the vitamin K phytyl side chain. Because CYP4F2 is the major cytochrome responsible for synthesis of 20-hydroxyeicosatetraenoic acid (20-HETE) in human kidney,^34^ the effect of CYP4F2 on warfarin dose could also be mediated through 20-HETE production.

The rs2108622 polymorphism in *CYP4F2* affects enzyme activity. It decreased 20-HETE production in a reconstituted recombinant protein system to approximately 60% of the wild-type enzyme.^35^ We postulate that individuals with T alleles have decreased function of the enzyme, thereby increasing the individual's requirement for warfarin.

Although recent studies^11,12,35,36^ demonstrate that using dosing models that include genetic testing may improve warfarin dosing, we have not presented a clinical model for use in prospectively dosing patients initiating warfarin therapy. However, we expect that such models will be forthcoming and that these models will continue to evolve as data from more patients at additional study sites become available. We also expect that, as those models improve with the discovery of additional genes or other factors, they will be important in clinical warfarin dosing.

## Acknowledgments

The authors thank Marshfield Clinic Research Foundation for its support through the assistance of Linda Weis and Alice Stargardt in the preparation of this manuscript. The authors also thank WaiWai Chang, Leewin Chern, Queeny Dong, Sai Htun, Richard Lao, Carlos Montesinos, Karen Tran, and Hanna Viernes of the Affymetrix Services Laboratory, and Duncan Foster of TWT for his assistance teaching the Third Wave assays.

This work was supported by Affymetrix Inc, Marshfield Clinic Research Foundation, and National Heart, Lung, and Blood Institute grant R01 HL074724.

## Footnotes

## Authorship

Contribution: M.D.C., T.A., P.G., R.L.B., J.K.B. designed the research; J.K.B. wrote the first draft of the manuscript; M.D.C., T.A., M.F., P.G., J.H., Y.T., J.A.J., T.Y.L., B.F.G., C.E., C.K., A.B., J.R.S., I.G., H.J.V., S.H.Y., K.Q.Z., and R.L.B. provided critical review and revisions to the manuscript; K.Q.Z. provided technical support.

Conflict-of-interest disclosure: T.A., M.F., P.G., J.H., and Y.T. are employees of Affymetrix Inc, the manufacturer of the DMET GeneChip. A.B. is an employee of TWT, the company that supplied reagents. All other authors declare no competing financial interests.

A provisional patent application for the use of CYP4F2 genetic testing has been filed.

Correspondence: James K. Burmester, Center for Human Genetics, Marshfield Clinic Research Foundation, 1000 North Oak Avenue, Marshfield, WI 54449; e-mail: burmester.jim@mcrf.mfldclin.edu.

## References

1. Petty GW, Brown RD, Jr, Whisnant JP, Sicks JD, O'Fallon WM, Wiebers DO. Frequency of major complications of aspirin, warfarin, and intravenous heparin for secondary stroke prevention: a population-based study. Ann Intern Med. 1999;130:14–22. doi: 10.7326/0003-4819-130-1-199901050-00004.  [DOI](https://doi.org/10.7326/0003-4819-130-1-199901050-00004) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9890845/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Intern%20Med&title=Frequency%20of%20major%20complications%20of%20aspirin,%20warfarin,%20and%20intravenous%20heparin%20for%20secondary%20stroke%20prevention:%20a%20population-based%20study.&author=GW%20Petty&author=RD%20Brown&author=JP%20Whisnant&author=JD%20Sicks&author=WM%20O'Fallon&volume=130&publication_year=1999&pages=14-22&pmid=9890845&doi=10.7326/0003-4819-130-1-199901050-00004&)

2. van der Meer FJ, Rosendaal FR, Vandenbroucke JP, Briet E. Bleeding complications in oral anticoagulant therapy: an analysis of risk factors. Arch Intern Med. 1993;153:1557–1562. doi: 10.1001/archinte.153.13.1557.  [DOI](https://doi.org/10.1001/archinte.153.13.1557) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8323419/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arch%20Intern%20Med&title=Bleeding%20complications%20in%20oral%20anticoagulant%20therapy:%20an%20analysis%20of%20risk%20factors.&author=FJ%20van%20der%20Meer&author=FR%20Rosendaal&author=JP%20Vandenbroucke&author=E%20Briet&volume=153&publication_year=1993&pages=1557-1562&pmid=8323419&doi=10.1001/archinte.153.13.1557&)

3. Cannegieter SC, Rosendaal FR, Wintzen AR, van der Meer FJ, Vandenbroucke JP, Briet E. Optimal oral anticoagulant therapy in patients with mechanical heart valves. N Engl J Med. 1995;333:11–17. doi: 10.1056/NEJM199507063330103.  [DOI](https://doi.org/10.1056/NEJM199507063330103) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7776988/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Optimal%20oral%20anticoagulant%20therapy%20in%20patients%20with%20mechanical%20heart%20valves.&author=SC%20Cannegieter&author=FR%20Rosendaal&author=AR%20Wintzen&author=FJ%20van%20der%20Meer&author=JP%20Vandenbroucke&volume=333&publication_year=1995&pages=11-17&pmid=7776988&doi=10.1056/NEJM199507063330103&)

4. Palareti G, Leali N, Coccheri S, et al. Bleeding complications of oral anticoagulant treatment: an inception-cohort, prospective collaborative study (ISCOAT): Italian Study on Complications of Oral Anticoagulant Therapy. Lancet. 1996;348:423–428. doi: 10.1016/s0140-6736(96)01109-9.  [DOI](https://doi.org/10.1016/s0140-6736(96)01109-9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8709780/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Bleeding%20complications%20of%20oral%20anticoagulant%20treatment:%20an%20inception-cohort,%20prospective%20collaborative%20study%20(ISCOAT):%20Italian%20Study%20on%20Complications%20of%20Oral%20Anticoagulant%20Therapy.&author=G%20Palareti&author=N%20Leali&author=S%20Coccheri&volume=348&publication_year=1996&pages=423-428&pmid=8709780&doi=10.1016/s0140-6736(96)01109-9&)

5. Evans A, Davis S, Kilpatrick C, Gerraty R, Campbell D, Greenberg P. The morbidity related to atrial fibrillation at a tertiary centre in one year: 9.0% of all strokes are potentially preventable. J Clin Neurosci. 2002;9:268–272. doi: 10.1054/jocn.2001.1018.  [DOI](https://doi.org/10.1054/jocn.2001.1018) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12093132/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Neurosci&title=The%20morbidity%20related%20to%20atrial%20fibrillation%20at%20a%20tertiary%20centre%20in%20one%20year:%209.0%%20of%20all%20strokes%20are%20potentially%20preventable.&author=A%20Evans&author=S%20Davis&author=C%20Kilpatrick&author=R%20Gerraty&author=D%20Campbell&volume=9&publication_year=2002&pages=268-272&pmid=12093132&doi=10.1054/jocn.2001.1018&)

6. Caldwell MD, Berg RL, Zhang KQ, et al. Evaluation of genetic factors for warfarin dose prediction. Clin Med Res. 2007;5:8–16. doi: 10.3121/cmr.2007.724.  [DOI](https://doi.org/10.3121/cmr.2007.724) | [PMC free article](/articles/PMC1855340/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17456829/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Med%20Res&title=Evaluation%20of%20genetic%20factors%20for%20warfarin%20dose%20prediction.&author=MD%20Caldwell&author=RL%20Berg&author=KQ%20Zhang&volume=5&publication_year=2007&pages=8-16&pmid=17456829&doi=10.3121/cmr.2007.724&)

7. Aquilante CL, Langaee TY, Lopez LM, et al. Influence of coagulation factor, vitamin K epoxide reductase complex subunit 1, and cytochrome P450 2C9 gene polymorphisms on warfarin dose requirements. Clin Pharmacol Ther. 2006;79:291–302. doi: 10.1016/j.clpt.2005.11.011.  [DOI](https://doi.org/10.1016/j.clpt.2005.11.011) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16580898/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Influence%20of%20coagulation%20factor,%20vitamin%20K%20epoxide%20reductase%20complex%20subunit%201,%20and%20cytochrome%20P450%E2%80%892C9%20gene%20polymorphisms%20on%20warfarin%20dose%20requirements.&author=CL%20Aquilante&author=TY%20Langaee&author=LM%20Lopez&volume=79&publication_year=2006&pages=291-302&pmid=16580898&doi=10.1016/j.clpt.2005.11.011&)

8. Zhu Y, Shennan M, Reynolds KK, et al. Estimation of warfarin maintenance dose based on VKORC1 (-1639 G>A) and CYP2C9 genotypes. Clin Chem. 2007;53:1199–1205. doi: 10.1373/clinchem.2006.078139.  [DOI](https://doi.org/10.1373/clinchem.2006.078139) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17510308/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Chem&title=Estimation%20of%20warfarin%20maintenance%20dose%20based%20on%20VKORC1%20(-1639%20G>A)%20and%20CYP2C9%20genotypes.&author=Y%20Zhu&author=M%20Shennan&author=KK%20Reynolds&volume=53&publication_year=2007&pages=1199-1205&pmid=17510308&doi=10.1373/clinchem.2006.078139&)

9. Gage BF. Pharmacogenetics-based coumarin therapy. Hematol Am Soc Hematol Educ Program. 2006:467–473. doi: 10.1182/asheducation-2006.1.467.  [DOI](https://doi.org/10.1182/asheducation-2006.1.467) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17124101/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hematol%20Am%20Soc%20Hematol%20Educ%20Program&title=Pharmacogenetics-based%20coumarin%20therapy.&author=BF%20Gage&publication_year=2006&pages=467-473&pmid=17124101&doi=10.1182/asheducation-2006.1.467&)

10. Millican EA, Lenzini PA, Milligan PE, et al. Genetic-based dosing in orthopedic patients beginning warfarin therapy. Blood. 2007;110:1511–1515. doi: 10.1182/blood-2007-01-069609.  [DOI](https://doi.org/10.1182/blood-2007-01-069609) | [PMC free article](/articles/PMC1975838/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17387222/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Genetic-based%20dosing%20in%20orthopedic%20patients%20beginning%20warfarin%20therapy.&author=EA%20Millican&author=PA%20Lenzini&author=PE%20Milligan&volume=110&publication_year=2007&pages=1511-1515&pmid=17387222&doi=10.1182/blood-2007-01-069609&)

11. Hillman MA, Wilke RA, Yale SH, et al. A prospective, randomized pilot trial of model-based warfarin dose initiation using CYP2C9 genotype and clinical data. Clin Med Res. 2005;3:137–145. doi: 10.3121/cmr.3.3.137.  [DOI](https://doi.org/10.3121/cmr.3.3.137) | [PMC free article](/articles/PMC1237155/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16160068/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Med%20Res&title=A%20prospective,%20randomized%20pilot%20trial%20of%20model-based%20warfarin%20dose%20initiation%20using%20CYP2C9%20genotype%20and%20clinical%20data.&author=MA%20Hillman&author=RA%20Wilke&author=SH%20Yale&volume=3&publication_year=2005&pages=137-145&pmid=16160068&doi=10.3121/cmr.3.3.137&)

12. Caraco Y, Blotnick S, Muszkat M. CYP2C9 genotype-guided warfarin prescribing enhances the efficacy and safety of anticoagulation: a prospective randomized controlled study. Clin Pharmacol Ther. 2007 Sep 12; doi: 10.1038/sj.clpt.6100316. Epub ahead of print.  [DOI](https://doi.org/10.1038/sj.clpt.6100316) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17851566/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=CYP2C9%20genotype-guided%20warfarin%20prescribing%20enhances%20the%20efficacy%20and%20safety%20of%20anticoagulation:%20a%20prospective%20randomized%20controlled%20study.&author=Y%20Caraco&author=S%20Blotnick&author=M%20Muszkat&publication_year=2007&pmid=17851566&doi=10.1038/sj.clpt.6100316&)

13. Aithal GP, Day CP, Kesteven PJ, Daly AK. Association of polymorphisms in the cytochrome P450 CYP2C9 with warfarin dose requirement and risk of bleeding complications. Lancet. 1999;353:717–719. doi: 10.1016/S0140-6736(98)04474-2.  [DOI](https://doi.org/10.1016/S0140-6736(98)04474-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10073515/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Association%20of%20polymorphisms%20in%20the%20cytochrome%20P450%20CYP2C9%20with%20warfarin%20dose%20requirement%20and%20risk%20of%20bleeding%20complications.&author=GP%20Aithal&author=CP%20Day&author=PJ%20Kesteven&author=AK%20Daly&volume=353&publication_year=1999&pages=717-719&pmid=10073515&doi=10.1016/S0140-6736(98)04474-2&)

14. Peyvandi F, Spreafico M, Siboni SM, Moia M, Mannucci PM. CYP2C9 genotypes and dose requirements during the induction phase of oral anticoagulant therapy. Clin Pharmacol Ther. 2004;75:198–203. doi: 10.1016/j.clpt.2003.09.015.  [DOI](https://doi.org/10.1016/j.clpt.2003.09.015) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15001971/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=CYP2C9%20genotypes%20and%20dose%20requirements%20during%20the%20induction%20phase%20of%20oral%20anticoagulant%20therapy.&author=F%20Peyvandi&author=M%20Spreafico&author=SM%20Siboni&author=M%20Moia&author=PM%20Mannucci&volume=75&publication_year=2004&pages=198-203&pmid=15001971&doi=10.1016/j.clpt.2003.09.015&)

15. Hillman MA, Wilke RA, Caldwell MD, Berg RL, Glurich I, Burmester JK. Relative impact of covariates in prescribing warfarin according to CYP2C9 genotype. Pharmacogenetics. 2004;14:539–547. doi: 10.1097/01.fpc.0000114760.08559.dc.  [DOI](https://doi.org/10.1097/01.fpc.0000114760.08559.dc) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15284536/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Relative%20impact%20of%20covariates%20in%20prescribing%20warfarin%20according%20to%20CYP2C9%20genotype.&author=MA%20Hillman&author=RA%20Wilke&author=MD%20Caldwell&author=RL%20Berg&author=I%20Glurich&volume=14&publication_year=2004&pages=539-547&pmid=15284536&doi=10.1097/01.fpc.0000114760.08559.dc&)

16. Rieder MJ, Reiner AP, Gage BF, et al. Effect of VKORC1 haplotypes on transcriptional regulation and warfarin dose. N Engl J Med. 2005;352:2285–2293. doi: 10.1056/NEJMoa044503.  [DOI](https://doi.org/10.1056/NEJMoa044503) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15930419/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Effect%20of%20VKORC1%20haplotypes%20on%20transcriptional%20regulation%20and%20warfarin%20dose.&author=MJ%20Rieder&author=AP%20Reiner&author=BF%20Gage&volume=352&publication_year=2005&pages=2285-2293&pmid=15930419&doi=10.1056/NEJMoa044503&)

17. D'Andrea G, D'Ambrosio RL, Di Perna P, et al. A polymorphism in the VKORC1 gene is associated with an interindividual variability in the dose-anticoagulant effect of warfarin. Blood. 2005;105:645–649. doi: 10.1182/blood-2004-06-2111.  [DOI](https://doi.org/10.1182/blood-2004-06-2111) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15358623/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=A%20polymorphism%20in%20the%20VKORC1%20gene%20is%20associated%20with%20an%20interindividual%20variability%20in%20the%20dose-anticoagulant%20effect%20of%20warfarin.&author=G%20D'Andrea&author=RL%20D'Ambrosio&author=P%20Di%20Perna&volume=105&publication_year=2005&pages=645-649&pmid=15358623&doi=10.1182/blood-2004-06-2111&)

18. Wadelius M, Chen LY, Downes K, et al. Common VKORC1 and GGCX polymorphisms associated with warfarin dose. Pharmacogenomics J. 2005;5:262–270. doi: 10.1038/sj.tpj.6500313.  [DOI](https://doi.org/10.1038/sj.tpj.6500313) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15883587/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Common%20VKORC1%20and%20GGCX%20polymorphisms%20associated%20with%20warfarin%20dose.&author=M%20Wadelius&author=LY%20Chen&author=K%20Downes&volume=5&publication_year=2005&pages=262-270&pmid=15883587&doi=10.1038/sj.tpj.6500313&)

19. Crawford DC, Ritchie MD, Rieder MJ. Identifying the genotype behind the phenotype: a role model found in VKORC1 and its association with warfarin dosing. Pharmacogenomics. 2007;8:487–496. doi: 10.2217/14622416.8.5.487.  [DOI](https://doi.org/10.2217/14622416.8.5.487) | [PMC free article](/articles/PMC3112050/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17465713/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Identifying%20the%20genotype%20behind%20the%20phenotype:%20a%20role%20model%20found%20in%20VKORC1%20and%20its%20association%20with%20warfarin%20dosing.&author=DC%20Crawford&author=MD%20Ritchie&author=MJ%20Rieder&volume=8&publication_year=2007&pages=487-496&pmid=17465713&doi=10.2217/14622416.8.5.487&)

20. Gage BF, Eby C, Milligan PE, Banet GA, Duncan JR, McLeod HL. Use of pharmacogenetics and clinical factors to predict the maintenance dose of warfarin. Thromb Haemost. 2004;91:87–94. doi: 10.1160/TH03-06-0379.  [DOI](https://doi.org/10.1160/TH03-06-0379) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14691573/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Thromb%20Haemost&title=Use%20of%20pharmacogenetics%20and%20clinical%20factors%20to%20predict%20the%20maintenance%20dose%20of%20warfarin.&author=BF%20Gage&author=C%20Eby&author=PE%20Milligan&author=GA%20Banet&author=JR%20Duncan&volume=91&publication_year=2004&pages=87-94&pmid=14691573&doi=10.1160/TH03-06-0379&)

21. Hardenbol P, Baner J, Jain M, et al. Multiplexed genotyping with sequence-tagged molecular inversion probes. Nat Biotechnol. 2003;21:673–678. doi: 10.1038/nbt821.  [DOI](https://doi.org/10.1038/nbt821) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12730666/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Biotechnol&title=Multiplexed%20genotyping%20with%20sequence-tagged%20molecular%20inversion%20probes.&author=P%20Hardenbol&author=J%20Baner&author=M%20Jain&volume=21&publication_year=2003&pages=673-678&pmid=12730666&doi=10.1038/nbt821&)

22. Hardenbol P, Yu F, Belmont J, et al. Highly multiplexed molecular inversion probe genotyping: over 10,000 targeted SNPs genotyped in a single tube assay. Genome Res. 2005;15:269–275. doi: 10.1101/gr.3185605.  [DOI](https://doi.org/10.1101/gr.3185605) | [PMC free article](/articles/PMC546528/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15687290/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res&title=Highly%20multiplexed%20molecular%20inversion%20probe%20genotyping:%20over%2010,000%20targeted%20SNPs%20genotyped%20in%20a%20single%20tube%20assay.&author=P%20Hardenbol&author=F%20Yu&author=J%20Belmont&volume=15&publication_year=2005&pages=269-275&pmid=15687290&doi=10.1101/gr.3185605&)

23. Daly TM, Dumaual CM, Miao X, et al. Multiplex assay for comprehensive genotyping of genes involved in drug metabolism, excretion, and transport. Clin Chem. 2007;53:1222–1230. doi: 10.1373/clinchem.2007.086348.  [DOI](https://doi.org/10.1373/clinchem.2007.086348) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17510302/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Chem&title=Multiplex%20assay%20for%20comprehensive%20genotyping%20of%20genes%20involved%20in%20drug%20metabolism,%20excretion,%20and%20transport.&author=TM%20Daly&author=CM%20Dumaual&author=X%20Miao&volume=53&publication_year=2007&pages=1222-1230&pmid=17510302&doi=10.1373/clinchem.2007.086348&)

24. Affymetrix GeneChip Scanner 3000 targeted genotyping system user guide. [Accessed July 17, 2007]; http://www.affymetrix.com/support/downloads/manuals/scanner3000_targeted_genotyping_user_guide.pdf.  [http://www.affymetrix.com/support/downloads/manuals/scanner3000_targeted_genotyping_user_guide.pdf](http://www.affymetrix.com/support/downloads/manuals/scanner3000_targeted_genotyping_user_guide.pdf)

25. Dumaual C, Miao X, Daly TM, et al. Comprehe-nsive assessment of metabolic enzyme and transporter genes using the Affymetrix Targeted Genotyping System. Pharmacogenomics. 2007;8:293–305. doi: 10.2217/14622416.8.3.293.  [DOI](https://doi.org/10.2217/14622416.8.3.293) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17324118/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Comprehe-nsive%20assessment%20of%20metabolic%20enzyme%20and%20transporter%20genes%20using%20the%20Affymetrix%20Targeted%20Genotyping%20System.&author=C%20Dumaual&author=X%20Miao&author=TM%20Daly&volume=8&publication_year=2007&pages=293-305&pmid=17324118&doi=10.2217/14622416.8.3.293&)

26. Barrett JC, Fry B, Maller J, Daly MJ. Haploview: analysis and visualization of LD and haplotype maps. Bioinformatics. 2005;21:263–265. doi: 10.1093/bioinformatics/bth457.  [DOI](https://doi.org/10.1093/bioinformatics/bth457) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15297300/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=Haploview:%20analysis%20and%20visualization%20of%20LD%20and%20haplotype%20maps.&author=JC%20Barrett&author=B%20Fry&author=J%20Maller&author=MJ%20Daly&volume=21&publication_year=2005&pages=263-265&pmid=15297300&doi=10.1093/bioinformatics/bth457&)

27. Koelbl JA, Mielke C, Stodola AJ, et al. Genotyping of CYP2C9 and VKORC1 polymorphisms using Invader® chemistry [abstract G14]. J Mol Diagn. 2006;8:625.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Mol%20Diagn&title=Genotyping%20of%20CYP2C9%20and%20VKORC1%20polymorphisms%20using%20Invader%C2%AE%20chemistry%20%5Babstract%20G14%5D.&author=JA%20Koelbl&author=C%20Mielke&author=AJ%20Stodola&volume=8&publication_year=2006&pages=625&)

28. Haga H, Yamada R, Ohnishi Y, Nakamura Y, Tanaka T. Gene-based SNP discovery as part of the Japanese Millennium Genome Project: identification of 190, 562 genetic variations in the human genome. Single-nucleotide polymorphism. J Hum Genet. 2002;47:605–610. doi: 10.1007/s100380200092.  [DOI](https://doi.org/10.1007/s100380200092) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12436197/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Hum%20Genet&title=Gene-based%20SNP%20discovery%20as%20part%20of%20the%20Japanese%20Millennium%20Genome%20Project:%20identification%20of%20190,%20562%20genetic%20variations%20in%20the%20human%20genome.%20Single-nucleotide%20polymorphism.&author=H%20Haga&author=R%20Yamada&author=Y%20Ohnishi&author=Y%20Nakamura&author=T%20Tanaka&volume=47&publication_year=2002&pages=605-610&pmid=12436197&doi=10.1007/s100380200092&)

29. Ohnishi Y, Tanaka T, Ozaki K, Yamada R, Suzuki H, Nakamura Y. A high-throughput SNP typing system for genome-wide association studies. J Hum Genet. 2001;46:471–477. doi: 10.1007/s100380170047.  [DOI](https://doi.org/10.1007/s100380170047) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11501945/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Hum%20Genet&title=A%20high-throughput%20SNP%20typing%20system%20for%20genome-wide%20association%20studies.&author=Y%20Ohnishi&author=T%20Tanaka&author=K%20Ozaki&author=R%20Yamada&author=H%20Suzuki&volume=46&publication_year=2001&pages=471-477&pmid=11501945&doi=10.1007/s100380170047&)

30. Harrell FE, Lee KL, Mark DB. Multivariable prognostic models: issues in developing models, evaluating assumptions and adequacy, and measuring and reducing errors. Stat Med. 1996;15:361–387. doi: 10.1002/(SICI)1097-0258(19960229)15:4<361::AID-SIM168>3.0.CO;2-4.  [DOI](https://doi.org/10.1002/(SICI)1097-0258(19960229)15:4<361::AID-SIM168>3.0.CO;2-4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8668867/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Stat%20Med&title=Multivariable%20prognostic%20models:%20issues%20in%20developing%20models,%20evaluating%20assumptions%20and%20adequacy,%20and%20measuring%20and%20reducing%20errors.&author=FE%20Harrell&author=KL%20Lee&author=DB%20Mark&volume=15&publication_year=1996&pages=361-387&pmid=8668867&doi=10.1002/(SICI)1097-0258(19960229)15:4<361::AID-SIM168>3.0.CO;2-4&)

31. International HapMap Consortium. The International HapMap Project. Nature. 2003;426:789–796. doi: 10.1038/nature02168.  [DOI](https://doi.org/10.1038/nature02168) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14685227/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=The%20International%20HapMap%20Project.&volume=426&publication_year=2003&pages=789-796&pmid=14685227&doi=10.1038/nature02168&)

32. NCI-NHGRI Working Group on Replication in Association Studies. Chanock SJ, Manolio T, et al. Replicating genotype-phenotype associations. Nature. 2007;447:655–660. doi: 10.1038/447655a.  [DOI](https://doi.org/10.1038/447655a) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17554299/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Replicating%20genotype-phenotype%20associations.&author=SJ%20Chanock&author=T%20Manolio&volume=447&publication_year=2007&pages=655-660&pmid=17554299&doi=10.1038/447655a&)

33. Sontag TJ, Parker RS. Cytochrome P450 omega-hydroxylase pathway of tocopherol catabolism: novel mechanism of regulation of vitamin E status. J Biol Chem. 2002;277:25290–25296. doi: 10.1074/jbc.M201466200.  [DOI](https://doi.org/10.1074/jbc.M201466200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11997390/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Cytochrome%20P450%20omega-hydroxylase%20pathway%20of%20tocopherol%20catabolism:%20novel%20mechanism%20of%20regulation%20of%20vitamin%20E%20status.&author=TJ%20Sontag&author=RS%20Parker&volume=277&publication_year=2002&pages=25290-25296&pmid=11997390&doi=10.1074/jbc.M201466200&)

34. Stec DE, Roman RJ, Flasch A, Rieder MJ. Functional polymorphism in human CYP4F2 decreases 20-HETE production. Physiol Genomics. 2007;30:74–81. doi: 10.1152/physiolgenomics.00003.2007.  [DOI](https://doi.org/10.1152/physiolgenomics.00003.2007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17341693/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Physiol%20Genomics&title=Functional%20polymorphism%20in%20human%20CYP4F2%20decreases%2020-HETE%20production.&author=DE%20Stec&author=RJ%20Roman&author=A%20Flasch&author=MJ%20Rieder&volume=30&publication_year=2007&pages=74-81&pmid=17341693&doi=10.1152/physiolgenomics.00003.2007&)

35. Voora D, Eby C, Linder MW, et al. Prospective dosing of warfarin based on cytochrome P-450 2C9 genotype. Thromb Haemost. 2005;93:700–705. doi: 10.1160/TH04-08-0542.  [DOI](https://doi.org/10.1160/TH04-08-0542) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15841315/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Thromb%20Haemost&title=Prospective%20dosing%20of%20warfarin%20based%20on%20cytochrome%20P-450%E2%80%892C9%20genotype.&author=D%20Voora&author=C%20Eby&author=MW%20Linder&volume=93&publication_year=2005&pages=700-705&pmid=15841315&doi=10.1160/TH04-08-0542&)

36. Anderson JL, Horne BD, Stevens SM, et al. Randomized trial of genotype-guided versus standard warfarin dosing in patients initiating oral anticoagulation. Circulation. 2007;116:2563–2570. doi: 10.1161/CIRCULATIONAHA.107.737312.  [DOI](https://doi.org/10.1161/CIRCULATIONAHA.107.737312) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17989110/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation&title=Randomized%20trial%20of%20genotype-guided%20versus%20standard%20warfarin%20dosing%20in%20patients%20initiating%20oral%20anticoagulation.&author=JL%20Anderson&author=BD%20Horne&author=SM%20Stevens&volume=116&publication_year=2007&pages=2563-2570&pmid=17989110&doi=10.1161/CIRCULATIONAHA.107.737312&)
