# Interleukin-6 (IL-6) haplotypes and the response to therapy of chronic hepatitis C virus infection

## Metadata
**Authors:** Leland J Yee, KyungAh Im, Brian Borg, Huiying Yang, T Jake Liang
**Journal:** Genes and immunity
**Date:** 2009 Apr 23
**DOI:** [10.1038/gene.2009.26](https://doi.org/10.1038/gene.2009.26)
**PMID:** 19387461
**PMCID:** PMC2810514
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2810514/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC2810514/pdf/nihms103946.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC2810514/pdf/nihms103946.pdf)

## Abstract

Chronic hepatitis C virus (HCV) infection affects nearly 170 million individuals worldwide. Treatment of HCV with pegylated interferon-α-2a is successful in eradicating virus from only 30%–80% of those treated. Interleukin-6 (IL-6) is an important cytokine involved in the immune response to infectious agents and in vitro studies suggest that host genetic variation, particularly haplotypes, may affect IL-6 expression. We examined the contribution of haplotypes in the IL-6 gene on sustained viral response (SVR) to therapy for chronic HCV infection. We observed the IL-6 T-T-G-G-G-G-C-A-G-A haplotype to be associated with a lower risk of achieving SVR among Caucasian Americans (CAs) (RR=0.80; 95%C.I.: 0.66– 0.98; p=0.0261). Using a sliding window approach, the rs1800797-(G)-rs1800796-(G)-rs1800795-(G) haplotype was associated with a reduced chance of SVR (RR=0.79; 95%C.I.: 0.66–0.94; p=0.0081), as was the rs1800796-(G)-rs1800795-(G)-rs2069830-(C) haplotype (RR=0.78; 95%C.I.: 0.66–0.94; p=0.0065) among CAs. Overall, the rs1800797-(G)-rs1800796-(G)-rs1800795-(G) haplotype was independently associated with a reduced chance of SVR (RR=0.78; 95% C.I.: 0.62–1.0; p=0.0489) after adjustment for potential confounding factors. Our findings further illustrate the complexity of IL-6 genetic regulation and the potential importance of haplotypes on IL-6 expression. Our findings provide additional support for the potential importance of genetic variation in the IL-6 gene and the response to HCV therapy.

## Introduction

Hepatitis C virus (HCV) infection affects an estimated 170 million individuals worldwide, and 5 million in the United States, where it is currently recognized as the most prevalent blood-borne infection and the leading indication for a liver transplant.[1](#R1)^, ^[2](#R2) Treatment of HCV with pegylated interferon-α-2a is successful in eradicating virus from only 30%–80% of those treated, with individuals infected with the more resilient genotype-1 virus having markedly lower response rates than those with non-genotype-1 infections.[3](#R3)^, ^[4](#R4) Additionally, differences in outcome have been described by race, with African Americans (AAs) having significantly lower response rates than Caucasian Americans (CAs).[5](#R5)^–^[7](#R7)

Expressed in a number of different cell types, including, hepatocytes, macrophages, B-cells and T-cells, Interleukin-6 (IL-6) is a pleiotropic cytokine important in the immunologic response to infections. IL-6 plays an important role in HCV infection as well as the response to IFN therapy. In addition to interacting with crucial components of the interferon response pathways, IL-6 is an activator of acute phase proteins in hepatocytes.[8](#R8) A recent study has suggested the potential importance of IL-6 in the treatment response of HCV patients to interferon-based therapy.[9](#R9)

Functional studies of *IL-6* genetics suggest that its expression is complex. A number of single nucleotide polymorphisms (SNPs) have been described within the IL-6 gene that form conserved haplotypes.[10](#R10)^, ^[11](#R11) While a number of these SNPs may have an impact on *IL-6* expression,[10](#R10)^, ^[11](#R11) studies suggest that IL-6 regulation is complex, with haplotypes playing a critical role in IL-6 expression.[11](#R11) In the present study we examined whether host genetic diversity in the *IL-6* gene is associated with the response to therapy for chronic HCV, with an emphasis on the role of *IL-6* haplotypes.

## Results

[Table 1](#T1) presents the baseline demographics of the participants in this study. Briefly, slight differences in age were observed in the NIDDK (p=0.034, Wilcoxon rank-sum test) and the overall combined cohort by race (p=0.027, Wilcoxon rank-sum test), indicating that AAs are slightly older than CAs in our cohorts. Additionally, slight differences in fibrosis score were observed in the NIDDK cohort by race (p=0.024, chi-square test), indicating higher proportion of AAs exhibited a worse baseline fibrosis category (>=3) than CAs. Whereas, the majority of AAs and CAs in Virahep-C cohort were in the milder fibrosis (<3) category. Nonetheless, we analyzed data by combining both cohort due to the small number of AA subjects in NIDDK cohort.

### Table 1.

|   | Virahep-C Cohort (n=374) |  |  | NIDDK Cohort (n=113) |  |  | Combined cohort (n=487) |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Factor | AA | CA | P* | AA | CA | P* | AA | CA | P* |
|   | n=180 | n=194 |   | n=22 | n=91 |   | n=202 | n=285 |   |
| Age (mean years ± SD) | 48.7 ± 7.1 | 47.0 ± 8.6 | 0.062 | 52.8 ± 7.6 | 48.3 ± 8.8 | 0.034 | 49.2 + 7.2 | 47.4 + 8.6 | 0.027 |
| Gender |  |  |  |  |  |  |  |  |  |
| Male | 118 (65.6%) | 126 (65.0%) | 0.902 | 6 (27.3%) | 41 (45.1%) | 0.129 | 68 (33.7%) | 109 (38.3%) | 0.3 |
| Female | 62 (34.4%) | 68 (35.1%) |   | 16 (72.7%) | 50 (55.0%) |   | 134 (66.3%) | 176 (61.8%) |   |
| Fibrosis Score (Ishak) |  |  |  |  |  |  |  |  |  |
| < 3 | 115 (64.3%) | 117 (60.3%) | 0.433 | 7 (33.3%) | 55 (60.4%) | 0.024 | 122 (61.0%) | 172 (60.4%) | 0.886 |
| ≥ 3 | 64 (35.8%) | 77 (39.7%) |   | 14 (66.7%) | 35 (39.6%) |   | 78 (39.0%) | 113 (39.7%) |   |
| Baseline viral level (log10 IU/mL) mean ± S.D. | 6.2 ± 0.7 | 6.3 ± 0.8 | 0.13 | 6.0 ± 0.4 | 5.7 ± 0.6 | 0.056 | 6.2 + 0.7 | 6.1 + 0.8 | 0.68 |
| Proportion of peginterferon taken in first 24 weeks (mean ± S.D.) | 0.82 ± 0.28 | 0.87 ± 0.26 | 0.02 | N/A | N/A | N/A | N/A | N/A | N/A |
| Response to therapy |  |  |  |  |  |  |  |  |  |
| NR | 131 (72.8%) | 90 (46.4%) | <.0001 | 20 (90.9%) | 62 (68.1%) | 0.032 | 151 (74.8%) | 152 (53.3%) | <.0001 |
| SVR | 49 (27.2%) | 104 (53.6%) |   | 2 (9.1%) | 29 (31.9%) |   | 51 (25.3%) | 133 (46.7%) |   |

Table 1 Caption: Baseline characteristics of the Virahep-C, NIDDK and the cohorts combined.

Among AA participants, 86 (47.8%) had HCV genotype 1a infections, 81 (45.0%) genotype 1b infections, and 13 (7.2%) a genotype-1 virus that could not be subgenotyped or mixed 1a/1b infection. Among CA participants, 111 (57.2%) had HCV genotype 1a, 55 (28.4%) genotype 1b, and 28 (14.4%) a genotype-1 virus that could not be subgenotyped or mixed 1a/1b infection. Participants in the NIDDK cohort were restricted to genotype 1-infected individuals.

Among the SNPs genotyped, rs13447445, rs13447446, and rs 2069829 were monomorphic and therefore not included in the analyses. [Figure 1.A](#F1). summarizes the SNPs genotyped in the present study and those included in haplotype construction. [Figure 1.B](#F1). presents the 10-SNP haplotypes that occurred with a frequency >5% in each race and their respective associations with SVR. The *IL-6 T-T-G-G-G-G-C-A-G-A* haplotype was associated with a lower risk of achieving SVR among CAs (RR=0.80; 95%C.I.: 0.66– 0.98; p=0.0261). A similar trend was observed among AAs for this haplotype (RR=0.70; 95% C.I.: 0.44–1.13), but the association was not statistically significant (p=0.1275).

### Figure 1. Targeted SNPs and IL-6 haplotypes.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/07fd/2810514/126fbebaf901/nihms103946f1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=2810514_nihms103946f1.jpg)

Part A. summarizes the single nucleotide polymorphisms (SNPs) targeted in the present study. Due to the fact that several additional SNPs were genotyped in the NIDDK cohort and not in Virahep-C, only SNPs common to both cohorts were used for haplotype estimation. SNPs not included in haplotype construction are in parenthesis. Note: the figure is not to scale. Part B. presents the haplotypes within each race with a frequency ≥5%, the observed frequencies, association with sustained viral response (SVR), the corresponding 95% confidence interval (95% C.I.), and p-value for the association.

[Figure 2.A](#F2). presents the associations for the 3-SNP sliding window analysis among AAs. None of the haplotypes tested were significantly associated with SVR. [Figure 2.B](#F2). presents the associations for the 3-SNP sliding window analysis among CAs. Frames containing SNP 4 (rs1800797), SNP 5 (rs1800796), and SNP 6 (rs1800795) had significant associations with SVR (sets 3, 4, and 5 in [Figure 2.B](#F2).). In particular, the rs1800797-(G)-rs1800796-(G)-rs1800795-(G) haplotype (set 4) was associated with a reduced chance of SVR (RR=0.79; 95%C.I.: 0.66–0.94; p=0.0081), as was the rs1800796-(G)-rs1800795-(G)-rs2069830-(C) haplotype (set 5) (RR=0.78; 95%C.I.: 0.66–0.94; p=0.0065).

### Figure 2. “Sliding window” analysis.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/07fd/2810514/ce632acf36ce/nihms103946f2a.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=2810514_nihms103946f2a.jpg)

Three-SNP sliding windows and the corresponding associations with sustained viralogic response (SVR) are presented. Part A. presents the results among African Americans (AAs), while Part B. presents the results among Caucasian Americans (CAs). For each 3-SNP block, the haplotype frequency, association with SVR (Relative Risk: (RR)), 95%C.I. interval and p-values are given. SNP1= rs1880242; SNP2= rs2056576; SNP3=rs2069827; SNP4=rs1800797; SNP5=rs1800796; SNP6=rs1800795; SNP7= rs2069830; SNP8=rs2069837; SNP9=rs1554606; SNP10=rs2069845. Only haplotypes with a frequency ≥5% are presented. Analyses was conducted using the Grasp program and SAS.

The rs1800797-(G)-rs1800796-(G)-rs1800795-(G) haplotype (set 4) was independently associated with a reduced chance of SVR (RR=0.78; 95% C.I.: 0.62–1.0; p=0.0489) after adjustment for potential confounding factors including race, baseline viral level, fibrosis score, gender and the interaction between race and baseline viral level ([Table 2](#T2)). [Table 3](#T3) presents the associations for individual SNPs.

### Table 2.

| Factor | RR** | 95% C.I.** | P† |
| --- | --- | --- | --- |
| IL-6 G-G-G haplotype carriage* | 0.78 | 0.62 – 1.0 | 0.0489 |
| CA race | 1.74 | 1.33 – 2.29 | <0.0001 |
| Baseline viral level (log10) | 0.58 | 0.43 – 0.78 | 0.0002 |
| Ishak fibrosis score (≥3 vs. <3) | 0.89 | 0.83 – 0.96 | 0.0041 |
| Male gender | 0.84 | 0.68 – 1.04 | 0.1147 |
| Interaction of CA race and baseline viral level | 1.55 | 1.12 – 2.15 | 0.0085 |

Table 2 Caption: Multivariable model of IL-6 haplotypes carriage adjusting for potential confounding factors.

### Table 3.

| Combined Virahep-C Study + NIDDK |  |  |  |  |  |  |
| --- | --- | --- | --- | --- | --- | --- |
| AA |  |  |  | CA |  |  |
|   |  |  |  |  |  |  |
| SNP | RR | 95%C.I. | P* | RR | 95%C.I. | P* |
| B.1. Allele Frequency |  |  |  |  |  |  |
| rs1880242-G | 0.92 | 0.58–1.46 | 0.7197 | 1.17 | 0.97–1.40 | 0.0972 |
| rs1880242-T | 0.97 | 0.84–1.13 | 0.7197 | 0.86 | 0.72–1.03 | 0.0972 |
| rs2056576-C | 1.14 | 0.81–1.61 | 0.4654 | 1.15 | 0.95–1.39 | 0.149 |
| rs2056576-T | 1.04 | 0.93–1.17 | 0.4654 | 0.87 | 0.72–1.05 | 0.149 |
| rs2069827-G | 0.60 | 0.25–1.44 | 0.3069 | 0.74 | 0.57–0.96 | 0.0532 |
| rs2069827-T | 0.77 | 0.40–1.47 | 0.3069 | 0.69 | 0.45–1.06 | 0.0532 |
| rs1800797-A | 1.01 | 0.52–1.95 | 0.9835 | 1.09 | 0.91–1.31 | 0.3713 |
| rs1800797-G | 1.00 | 0.80–1.26 | 0.9835 | 0.92 | 0.77–1.10 | 0.3713 |
| rs1800796-C | 0.58 | 0.26–1.33 | 0.1662 | 1.38 | 1.04–1.82 | 0.0558 |
| rs1800796-G | 0.87 | 0.74–1.02 | 0.1662 | 1.44 | 0.93–2.23 | 0.0558 |
| rs1800795-C | 0.91 | 0.47–1.79 | 0.7898 | 1.19 | 0.99–1.42 | 0.0578 |
| rs1800795-G | 0.97 | 0.79–1.19 | 0.7898 | 0.84 | 0.71–1.00 | 0.0578 |
| rs2069830-C | 0.80 | 0.49–1.29 | 0.3694 | N.C | N.C | N.C |
| rs2069830-T | 0.91 | 0.74–1.14 | 0.3694 | N.C | N.C | N.C |
| rs22069837-A | 0.85 | 0.53–1.36 | 0.5015 | 0.95 | 0.68–1.33 | 0.7831 |
| rs22069837-G | 0.94 | 0.77–1.14 | 0.5015 | 0.96 | 0.70–1.31 | 0.7831 |
| rs1554606-G | 1.04 | 0.72–1.51 | 0.8204 | 0.88 | 0.73–1.05 | 0.1537 |
| rs1554606-T | 1.01 | 0.90–1.15 | 0.8204 | 0.89 | 0.75–1.05 | 0.1537 |
| rs2069845-A | 1.01 | 0.71–1.45 | 0.9427 | 0.89 | 0.74–1.07 | 0.2137 |
| rs2069845-G | 1.00 | 0.89–1.14 | 0.9427 | 0.90 | 0.77–1.06 | 0.2137 |
| B.2. Allele carriage |  |  |  |  |  |  |
| rs1880242-G | 0.90 | 0.53–1.52 | 0.6899 | 1.28 | 0.94–1.74 | 0.1036 |
| rs1880242-T | 1.03 | 0.19–5.70 | 0.9769 | 0.84 | 0.62–1.13 | 0.2760 |
| rs2056576-C | 1.17 | 0.62–2.20 | 0.6253 | 1.31 | 0.86–2.00 | 0.1813 |
| rs2056576-T | 0.84 | 0.51–1.38 | 0.4997 | 0.87 | 0.67–1.12 | 0.2793 |
| rs2069827-G | N.C | N.C | N.C | N.C | N.C | N.C |
| rs2069827-T | 1.69 | 0.69–4.11 | 0.303 | 1.43 | 1.09–1.89 | 0.0274 |
| rs1800797-A | 1.01 | 0.51–2.00 | 0.9829 | 1.04 | 0.81–1.35 | 0.7387 |
| rs1800797-G | N.C | N.C | N.C | 0.78 | 0.57–1.08 | 0.1789 |
| rs1800796-C | 0.58 | 0.25–1.35 | 0.1761 | 1.48 | 1.10–1.20 | 0.0276 |
| rs1800796-G | N.C | N.C | N.C | 1.38 | 0.28–6.89 | 0.6604 |
| rs1800795-C | 0.94 | 0.47–1.89 | 0.8691 | 1.14 | 0.88–1.47 | 0.3278 |
| rs1800795-G | N.C | N.C | N.C | N.C | N.C | N.C |
| rs2069830-T | 1.39 | 0.82–2.34 | 0.2371 | N.C | N.C | N.C |
| rs22069837-A | N.C | N.C | N.C | 0.46 | 0.41–0.53 | 0.2844 |
| rs22069837-G | 1.25 | 0.74–2.10 | 0.4089 | 1.02 | 0.71–1.46 | 0.9182 |
| rs1554606-G | 0.75 | 0.37–1.51 | 0.4419 | 0.72 | 0.54–0.96 | 0.0441 |
| rs1554606-T | 0.84 | 0.52–1.34 | 0.4610 | 1.08 | 0.83–1.40 | 0.5678 |
| rs2069845-A | 0.66 | 0.35–1.26 | 0.2403 | 0.75 | 0.56–1.01 | 0.0893 |
| rs2069845-G | 0.83 | 0.52–1.33 | 0.4314 | 1.07 | 0.82–1.40 | 0.5927 |

Table 3 Caption: Associations between individual alleles and sustained viral response (SVR) in the combined Virahep-C and NIDDK cohort by race. Part B.1 presents the associations between allele frequency and SVR. Part B.2 presents the associations of allele carriage and SVR.

## Discussion

Using a systematic analysis of *IL-6* haplotypes, we observed consistent associations of haplotypes in this gene with a reduced likelihood of SVR. In particular, we observed the 10-SNP haplotype *IL-6 T-T-G-G-G-G-C-A-G-A* haplotype associated with a lower risk of achieving SVR among CAs (RR=0.80; 95%C.I.: 0.66– 0.98; p=0.0261). Using a sliding window approach, we observed frames containing rs1800797, rs1800796, and rs1800795 to be associated with SVR. In particular, the rs1800797-(G)-rs1800796-(G)-rs1800795-(G) haplotype was associated with a reduced chance of SVR (RR=0.79; 95%C.I.: 0.66–0.94; p=0.0081), as was the rs1800796-(G)-rs1800795-(G)-rs2069830-(C) haplotype (RR=0.78; 95%C.I.: 0.66–0.94; p=0.0065). All associations were only observed among CAs.

Our sliding window analyses further suggested that the region containing the SNPs rs1800797, rs1800796, rs1800795 and rs2069830 may be an important region in the IL-6 gene, with several haplotypes in this region associated with a lower likelihood of SVR. *In vitro* functional studies support the fact that this region may be of regulatory importance.[11](#R11)^, ^[12](#R12)

Previous studies of genetic variation in *IL-6* and the response to HCV therapy have focused on single nucleotide polymorphisms. Natterman and colleagues described an association between carriage of high producing genotypes (ie., the GG or GC genotypes)[10](#R10) of the *C-174G (rs1800795*) polymorphism and a greater likelihood of SVR.[12](#R12) In contrast, our haplotypic observations were of lowered likelihood of SVR. A functional study of IL-6 genetics by Terry et al., suggests that IL-6 regulation is complex and that small differences on the haplotype level may have a significant impact on IL-6 expression. For example, they described a (*−597G)+(−572G)+(−174G*) clone containing a 9/11 base pair A_n_T_n_ repeat to have greater levels of expression than one containing a 10/10 repeat.[11](#R11) These observations reinforce the importance of haplotypes in IL-6 expression.

The present study underscores the importance of examining genetic haplotypes in addition to individual SNPs. Our findings build upon those of Nattermann and colleagues and collectively, both studies suggest that host genetic variation in the *IL-6* gene may be important in the response to therapy for HCV. Additional studies are needed to fully elucidate the complex functional aspects of this gene.

## Methods

### Study Population and clinical data

This study utilized participants from the Study of Viral Resistance to Antiviral Therapy of Chronic Hepatitis C (Virahep-C), a National Institutes of Health (NIH)-funded multi-center study aimed at understanding the mechanisms of resistance to antiviral therapy for chronic HCV infection among genotype-1-infected interferon treatment-naïve individuals. Additional emphasis was placed on identifying factors that may contribute to differences in outcome by race among AAs and CAs.[5](#R5) All subjects were born in the United States (US) and race was determined by a self-administered questionnaire.

We also utilized participants who were attending the NIH Clinical Center (N-113). Individuals who were treatment-naïve, infected with HCV genotype-1, and underwent IFN-based therapy (pegylated interferon-α + ribavirin or IFN-α only) were included in the present analysis. Available demographic data including self-reported race, sex, patient age and response to interferon were included in multivariable analyses.

Sustained virologic response (SVR) was defined as having undetectable HCV-RNA 6 months after the discontinuation of therapy. The Roche Amplicor Assay version 2 with a lower limit of detection of 600 IU/mL was used in this study. Participants received a liver biopsy within 6 months prior to the start of therapy. Biopsies were scored by a single pathologist who was blinded to patient outcome using the Ishak modified histological activity index (HAI) score.[13](#R13) To facilitate categorical analysis, biopsy scores were dichotomized as ≥3 and <3. All participants from both the Virahep-C study and the NIH clinical center provided written consent to participate in host genetics studies.

### Statistical analysis

Proportions of SVR by baseline demographic and clinical characteristics were compared using the two-sided race-adjusted Mantel-Haenszel Chi-square test. The non-parametric Wilcoxon rank-sum test was used to compare racial differences in distributions of continuous data (such as age, baseline viral level, proportion of peginterferon). Unadjusted associations between individual haplotype (2N) and haplotype carrier (N) frequencies, as well as allele and allele carrier frequencies for examination of individual loci, with SVR were summarized using relative risk (RR) estimates, 95% confidence intervals (95% C.I.) and p-values.

Multiple Poisson regression models with sandwich estimators of the variance were utilized to adjust for potential confounding factors.[14](#R14) Potential confounding factors were selected based on previously published findings and data that were collected across both cohorts: included race, baseline viral levels, Ishak score, gender.[15](#R15) The SAS® program, version 9 was used for all analyses. Statistical significance was set at α=0.05.

### Genotyping

Among the 401 individuals treated in the Virahep-C Study, a subset of 373 consented to participate in host genetics studies and had DNA available for genotyping. Details of the Virahep-C host genetics ancillary study have been published previously.[16](#R16) Single nucleotide polymorphisms in the *IL-6* gene were selected using the HapMap database. Initially, rs1880242, rs2056576, rs2069827, and rs2069845 were selected using HapMap version 1 and genotyped using the Illumina Beadarray system (Illumina Corporation, San Diego, CA) for patients in both cohorts. Additionally, rs2069860, rs13306435 and rs2069850 were genotyped using allelic discrimination in the NIDDK cohort, but not the Virahep-C study.

To increase the density of SNP coverage for the *IL-6* gene, we further genotyped rs1554606, rs1800795, rs1800797, rs2069830, and rs2069837 by allelic discrimination using the ABI 7000 Sequence Detection System using TaqMan technology (Applied Biosystems, Foster City, CA) for participants from both cohorts. Additional genotyping for rs13447445, rs13447446, rs2069829 and rs1800796 was conducted using the fluorogenic 5’-nuclease TaqMan allelic discrimination assay on a 7900HT Real time PCR instrument.(Applied Biosystems, Foster City, CA) in both cohorts. All probes and reagents were purchased from Applied Biosystems (Foster City, CA). All genotype calls were determined by two independent investigators, and only concordant calls were used.

### Haplotype construction

We constructed haplotypes using the SAS Genetics module (SAS® v9.2) using a stepwise version of the EM algorithm (SAS® version 9.1; Cary, NC) separately by race. Due to the nominal sample size of the NIDDK cohort (N=22 AAs and N=91 CAs), participants from both cohorts were combined for haplotype construction and to improve statistical power for analyses. Since rs2069860, rs13306435 and rs2069850 were genotyped only in the NIDDK cohort and not the Virahep-C cohort, these SNPs were not included in haplotype construction to ensure that haplotypes were comparable across both cohorts. In addition, we conducted stratified analyses by cohort to determine whether the directions of associations are similar by cohort.

We further evaluated linkage disequilibrium (LD) among genotyped loci by examining the pair-wise LD between each locus within each race using the SAS® genetics module and the Grasp program.[17](#R17) Utilizing these data, we generated haplotypes consisting of 3, 4, and 5 consecutive, overlapping SNPs. Using a “sliding window” approach, we determined the association between carriage of smaller haplotype blocks and SVR by race in order to further refine specific regions of the *IL-6* gene that may be contributing to the association with SVR. Only haplotypes with a frequency >5% within a race were analyzed for that race.

### Evaluation of population structure

Using data from 161 ancestry-informative single nucleotide polymorphisms (SNPs), we derived estimates of individual admixture for participants in the genetics study, and utilized the structured association method to evaluate the population structure.[16](#R16)^, ^[18](#R18)^, ^[19](#R19) We observed two distinct ancestral groups that had a strong correlation with self-reported race.[16](#R16) Consequently, we used self-reported race in these analyses.

## Acknowledgements

This clinical study was a cooperative agreement funded by the NIDDK and co-funded by the National Center on Minority Health and Health Disparities (NCMHD), with a Cooperative Research and Development Agreement (CRADA) with Roche Laboratories, Inc. Grant numbers: U01 DK60329, U01 DK60340, U01 DK60324, U01 DK60344, U01 DK60327, U01 DK60335, U01 DK60352, U01 DK60342, U01 DK60345, U01 DK60309, U01 DK60346, U01 DK60349, U01 DK60341. Other support: National Center for Research Resources (NCRR), Intramural Research Program of the NIH, NIDDK, Center for Cancer Research, General Clinical Research Centers Program grants: M01 RR00645 (New York Presbyterian), M02 RR000079 (University of California, San Francisco), M01 RR16500 (University of Maryland), M01 RR000042 (University of Michigan), M01 RR00046 (University of North Carolina). Additional support for Dr. Leland J. Yee was provided by a National Institutes of Health Clinical Research Career Development Award Grant (1KL2 RR024154-02).

## Footnotes

## References

1. Armstrong GL, et al. The prevalence of hepatitis C virus infection in the United States, 1999 through 2002. Ann Intern Med. 2006;144:705–714. doi: 10.7326/0003-4819-144-10-200605160-00004.  [DOI](https://doi.org/10.7326/0003-4819-144-10-200605160-00004) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16702586/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Intern%20Med&title=The%20prevalence%20of%20hepatitis%20C%20virus%20infection%20in%20the%20United%20States,%201999%20through%202002&author=GL%20Armstrong&volume=144&publication_year=2006&pages=705-714&pmid=16702586&doi=10.7326/0003-4819-144-10-200605160-00004&)

2. Brown RS. Hepatitis C and liver transplantation. Nature. 2005;436:973–978. doi: 10.1038/nature04083.  [DOI](https://doi.org/10.1038/nature04083) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16107838/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Hepatitis%20C%20and%20liver%20transplantation&author=RS%20Brown&volume=436&publication_year=2005&pages=973-978&pmid=16107838&doi=10.1038/nature04083&)

3. Fried MW. Side effects of therapy of hepatitis C and their management. Hepatology. 2002;36:S237–S244. doi: 10.1053/jhep.2002.36810.  [DOI](https://doi.org/10.1053/jhep.2002.36810) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12407599/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hepatology&title=Side%20effects%20of%20therapy%20of%20hepatitis%20C%20and%20their%20management&author=MW%20Fried&volume=36&publication_year=2002&pages=S237-S244&pmid=12407599&doi=10.1053/jhep.2002.36810&)

4. Manns MP, et al. Peginterferon alfa-2b plus ribavirin compared with interferon alfa-2b plus ribavirin for initial treatment of chronic hepatitis C: a randomised trial. Lancet. 2001;358:958–965. doi: 10.1016/s0140-6736(01)06102-5.  [DOI](https://doi.org/10.1016/s0140-6736(01)06102-5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11583749/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Peginterferon%20alfa-2b%20plus%20ribavirin%20compared%20with%20interferon%20alfa-2b%20plus%20ribavirin%20for%20initial%20treatment%20of%20chronic%20hepatitis%20C:%20a%20randomised%20trial&author=MP%20Manns&volume=358&publication_year=2001&pages=958-965&pmid=11583749&doi=10.1016/s0140-6736(01)06102-5&)

5. Conjeevaram HS, et al. Peginterferon and ribavirin treatment in African American and Caucasian American patients with hepatitis C genotype 1. Gastroenterology. 2006;131:470–477. doi: 10.1053/j.gastro.2006.06.008.  [DOI](https://doi.org/10.1053/j.gastro.2006.06.008) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16890601/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastroenterology&title=Peginterferon%20and%20ribavirin%20treatment%20in%20African%20American%20and%20Caucasian%20American%20patients%20with%20hepatitis%20C%20genotype%201&author=HS%20Conjeevaram&volume=131&publication_year=2006&pages=470-477&pmid=16890601&doi=10.1053/j.gastro.2006.06.008&)

6. Jeffers LJ, Cassidy W, Howell CD, Hu S, Reddy KR. Peginterferon alfa-2a (40 kd) and ribavirin for black American patients with chronic HCV genotype 1. Hepatology. 2004;39:1702–1708. doi: 10.1002/hep.20212.  [DOI](https://doi.org/10.1002/hep.20212) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15185312/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hepatology&title=Peginterferon%20alfa-2a%20(40%20kd)%20and%20ribavirin%20for%20black%20American%20patients%20with%20chronic%20HCV%20genotype%201&author=LJ%20Jeffers&author=W%20Cassidy&author=CD%20Howell&author=S%20Hu&author=KR%20Reddy&volume=39&publication_year=2004&pages=1702-1708&pmid=15185312&doi=10.1002/hep.20212&)

7. Muir AJ, Bornstein JD, Killenberg PG. Peginterferon alfa-2b and ribavirin for the treatment of chronic hepatitis C in blacks and non-Hispanic whites. N Engl J Med. 2004;350:2265–2271. doi: 10.1056/NEJMoa032502.  [DOI](https://doi.org/10.1056/NEJMoa032502) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15163776/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Peginterferon%20alfa-2b%20and%20ribavirin%20for%20the%20treatment%20of%20chronic%20hepatitis%20C%20in%20blacks%20and%20non-Hispanic%20whites&author=AJ%20Muir&author=JD%20Bornstein&author=PG%20Killenberg&volume=350&publication_year=2004&pages=2265-2271&pmid=15163776&doi=10.1056/NEJMoa032502&)

8. Ramadori G, Christ B. Cytokines and the hepatic acute-phase response. Semin Liver Dis. 1999;19:141–155. doi: 10.1055/s-2007-1007106.  [DOI](https://doi.org/10.1055/s-2007-1007106) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10422197/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Semin%20Liver%20Dis&title=Cytokines%20and%20the%20hepatic%20acute-phase%20response&author=G%20Ramadori&author=B%20Christ&volume=19&publication_year=1999&pages=141-155&pmid=10422197&doi=10.1055/s-2007-1007106&)

9. Huang Y, et al. Defective hepatic response to interferon and activation of suppressor of cytokine signaling 3 in chronic hepatitis C. Gastroenterology. 2007;132:733–744. doi: 10.1053/j.gastro.2006.11.045.  [DOI](https://doi.org/10.1053/j.gastro.2006.11.045) | [PMC free article](/articles/PMC1853257/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17258724/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastroenterology&title=Defective%20hepatic%20response%20to%20interferon%20and%20activation%20of%20suppressor%20of%20cytokine%20signaling%203%20in%20chronic%20hepatitis%20C&author=Y%20Huang&volume=132&publication_year=2007&pages=733-744&pmid=17258724&doi=10.1053/j.gastro.2006.11.045&)

10. Fishman D, et al. The effect of novel polymorphisms in the interleukin-6 (IL-6) gene on IL-6 transcription and plasma IL-6 levels, and an association with systemic-onset juvenile chronic arthritis. J Clin Invest. 1998;102:1369–1376. doi: 10.1172/JCI2629.  [DOI](https://doi.org/10.1172/JCI2629) | [PMC free article](/articles/PMC508984/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9769329/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Invest&title=The%20effect%20of%20novel%20polymorphisms%20in%20the%20interleukin-6%20(IL-6)%20gene%20on%20IL-6%20transcription%20and%20plasma%20IL-6%20levels,%20and%20an%20association%20with%20systemic-onset%20juvenile%20chronic%20arthritis&author=D%20Fishman&volume=102&publication_year=1998&pages=1369-1376&pmid=9769329&doi=10.1172/JCI2629&)

11. Terry CF, Loukaci V, Green FR. Cooperative influence of genetic polymorphisms on interleukin 6 transcriptional regulation. J Biol Chem. 2000;275:18138–18144. doi: 10.1074/jbc.M000379200.  [DOI](https://doi.org/10.1074/jbc.M000379200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10747905/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Cooperative%20influence%20of%20genetic%20polymorphisms%20on%20interleukin%206%20transcriptional%20regulation&author=CF%20Terry&author=V%20Loukaci&author=FR%20Green&volume=275&publication_year=2000&pages=18138-18144&pmid=10747905&doi=10.1074/jbc.M000379200&)

12. Nattermann J, et al. Effect of the interleukin-6 C174G gene polymorphism on treatment of acute and chronic hepatitis C in human immunodeficiency virus coinfected patients. Hepatology. 2007;46:1016–1025. doi: 10.1002/hep.21778.  [DOI](https://doi.org/10.1002/hep.21778) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17668881/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hepatology&title=Effect%20of%20the%20interleukin-6%20C174G%20gene%20polymorphism%20on%20treatment%20of%20acute%20and%20chronic%20hepatitis%20C%20in%20human%20immunodeficiency%20virus%20coinfected%20patients&author=J%20Nattermann&volume=46&publication_year=2007&pages=1016-1025&pmid=17668881&doi=10.1002/hep.21778&)

13. Ishak K, et al. Histological grading and staging of chronic hepatitis. J Hepatol. 1995;22:696–699. doi: 10.1016/0168-8278(95)80226-6.  [DOI](https://doi.org/10.1016/0168-8278(95)80226-6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7560864/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Hepatol&title=Histological%20grading%20and%20staging%20of%20chronic%20hepatitis&author=K%20Ishak&volume=22&publication_year=1995&pages=696-699&pmid=7560864&doi=10.1016/0168-8278(95)80226-6&)

14. Zou G. A modified poisson regression approach to prospective studies with binary data. Am J Epidemiol. 2004;159:702–706. doi: 10.1093/aje/kwh090.  [DOI](https://doi.org/10.1093/aje/kwh090) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15033648/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Epidemiol&title=A%20modified%20poisson%20regression%20approach%20to%20prospective%20studies%20with%20binary%20data&author=G%20Zou&volume=159&publication_year=2004&pages=702-706&pmid=15033648&doi=10.1093/aje/kwh090&)

15. Conjeevaram HS, FM W, Jeffers LJ, Terrault NA, Wiley-Lucas TE, Afdhal N, Brown RS, Belle SH, Hoofnagle JH, Kleiner DE, Howell CD. Peg interferon and Ribavirin treatment in African Americans and Caucasian American Patients with Chronic Hepatitis C Genotype I. J. Gastroenterology in press. 2006 doi: 10.1053/j.gastro.2006.06.008.  [DOI](https://doi.org/10.1053/j.gastro.2006.06.008) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16890601/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Gastroenterology%20in%20press&title=Peg%20interferon%20and%20Ribavirin%20treatment%20in%20African%20Americans%20and%20Caucasian%20American%20Patients%20with%20Chronic%20Hepatitis%20C%20Genotype%20I&author=HS%20Conjeevaram&author=W%20FM&author=LJ%20Jeffers&author=NA%20Terrault&author=TE%20Wiley-Lucas&publication_year=2006&pmid=16890601&doi=10.1053/j.gastro.2006.06.008&)

16. Yee LJ, et al. Myxovirus-1 and protein kinase haplotypes and fibrosis in chronic hepatitis C virus. Hepatology. 2007;46:74–83. doi: 10.1002/hep.21636.  [DOI](https://doi.org/10.1002/hep.21636) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17526009/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hepatology&title=Myxovirus-1%20and%20protein%20kinase%20haplotypes%20and%20fibrosis%20in%20chronic%20hepatitis%20C%20virus&author=LJ%20Yee&volume=46&publication_year=2007&pages=74-83&pmid=17526009&doi=10.1002/hep.21636&)

17. Mathias RA, et al. A graphical assessment of p-values from sliding window haplotype tests of association to identify asthma susceptibility loci on chromosome 11q. BMC Genet. 2006;7:38. doi: 10.1186/1471-2156-7-38.  [DOI](https://doi.org/10.1186/1471-2156-7-38) | [PMC free article](/articles/PMC1526454/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16774684/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Genet&title=A%20graphical%20assessment%20of%20p-values%20from%20sliding%20window%20haplotype%20tests%20of%20association%20to%20identify%20asthma%20susceptibility%20loci%20on%20chromosome%2011q&author=RA%20Mathias&volume=7&publication_year=2006&pages=38&pmid=16774684&doi=10.1186/1471-2156-7-38&)

18. Pritchard JK, Stephens M, Donnelly P. Inference of population structure using multilocus genotype data. Genetics. 2000;155:945–959. doi: 10.1093/genetics/155.2.945.  [DOI](https://doi.org/10.1093/genetics/155.2.945) | [PMC free article](/articles/PMC1461096/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10835412/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genetics&title=Inference%20of%20population%20structure%20using%20multilocus%20genotype%20data&author=JK%20Pritchard&author=M%20Stephens&author=P%20Donnelly&volume=155&publication_year=2000&pages=945-959&pmid=10835412&doi=10.1093/genetics/155.2.945&)

19. Pritchard JK, Stephens M, Rosenberg NA, Donnelly P. Association mapping in structured populations. Am J Hum Genet. 2000;67:170–181. doi: 10.1086/302959.  [DOI](https://doi.org/10.1086/302959) | [PMC free article](/articles/PMC1287075/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10827107/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Association%20mapping%20in%20structured%20populations&author=JK%20Pritchard&author=M%20Stephens&author=NA%20Rosenberg&author=P%20Donnelly&volume=67&publication_year=2000&pages=170-181&pmid=10827107&doi=10.1086/302959&)
