# Thiopurine S-Methyltransferase Pharmacogenetics: Functional Characterization of a Novel Rapidly Degraded Variant Allozyme

## Metadata
**Authors:** Qiping Feng, Suda Vannaprasaht, Yi Peng, Susothorn Angsuthum, Yingyos Avihingsanon, Vivien C Yee, Wichittra Tassaneeyakul, Richard M Weinshilboum
**Journal:** Biochemical pharmacology
**Date:** 2009 Nov 27
**DOI:** [10.1016/j.bcp.2009.11.016](https://doi.org/10.1016/j.bcp.2009.11.016)
**PMID:** 19945438
**PMCID:** PMC2815203
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2815203/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC2815203/pdf/nihms161970.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC2815203/pdf/nihms161970.pdf)

## Abstract

A novel human thiopurine S-methyltransferase (TPMT) variant allele, (319 T>G, 107 Tyr>Asp, *27), was identified in a Thai renal transplantation recipient with reduced erythrocyte TPMT activity. The TPMT*27 variant allozyme showed a striking decrease in both immunoreactive protein level and enzyme activity after transient expression in a mammalian cell line. We set out to explore the mechanism(s) responsible for decreased expression of this novel variant of an important drug-metabolizing enzyme. We observed accelerated degradation of TPMT*27 protein in a rabbit reticulocyte lysate. TPMT*27 degradation was slowed by proteasome inhibition and involved chaperone proteins–similar to observations with regard to the degradation of the common TPMT*3A variant allozyme. TPMT*27 aggresome formation was also observed in transfected mammalian cells after proteasome inhibition. Inhibition of autophagy also decreased TPMT*27 degradation. Finally, structural analysis and molecular dynamics simulation indicated that TPMT*27 was less stable than was the wild type TPMT allozyme. In summary, TPMT*27 serves to illustrate the potential importance of protein degradation–both proteasome and autophagy-mediated degradation–for the pharmacogenetic effects of nonsynonymous SNPs.

Keywords: thiopurine S-methyltransferase, TPMT*27, pharmacogenetics, protein degradation, autophagy

## 1. Introduction

Thiopurine S-methyltransferase (TPMT) genetic polymorphisms represent a striking example of the clinical importance of pharmacogenetics [[1](#R1)–[3](#R3)]. TPMT also represents a model system for the study of pharmacogenetic functional mechanisms–especially mechanisms involving nonsynonymous SNPs–polymorphisms that alter the encoded amino acid sequence [[3](#R3)]. We pointed out several years ago that one important way by which nonsynonymous SNPs influence function is by altering the quantity of the encoded protein [[4](#R4)]. TPMT has proved to be a useful system in which to study mechanisms responsible for that effect. *TPMT***3A*, the most common variant allele in Caucasians [[5](#R5)], a variant allele with two nonsynonymous SNPs [[6](#R6)], results in striking decreases in both TPMT protein and enzyme activity [[7](#R7)–[11](#R11)]. A series of studies over the past decade has demonstrated that this process involves proteasome-mediated degradation, with the involvement of molecular chaperones [[9](#R9)–[11](#R11)] as well as autophagy [[8](#R8)]. Those processes for protein degradation are depicted schematically in [Fig. 1](#F1), a figure that may help readers to follow the sequence of experiments described subsequently. However, whether the mechanisms outlined in [Fig. 1](#F1) are unique to *TPMT***3A* remains unclear. In this study, we report a series of “bedside to bench” experiments in which we show that a novel TPMT variant allozyme undergoes a process of accelerated degradation similar to that responsible for the very low activity displayed by TPMT*3A. The experiments described subsequently also serve to illustrate our expanding understanding of the role of protein degradation in pharmacogenetics.

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6df1/2815203/1df5fdcbde35/nihms161970f1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=2815203_nihms161970f1.jpg)

Schematic representation of TPMT synthesis, folding and degradation pathways and processes. Modified with permission from Wang and Weinshilboum [3] and Weinshilboum and Wang [4].

Because of their potential clinical importance [[12](#R12)–[15](#R15)], TPMT genotypes and phenotypes have been determined in many patients. As a result, 29 different variant alleles have been identified [[12](#R12)] – more than half of which are associated with decreased TPMT activity [[6](#R6), [14](#R14)]. However, with the exception of a small number of allozymes such as TPMT*3A and TPMT*3C [[6](#R6), [8](#R8), [10](#R10), [11](#R11)], systematic in depth functional characterization of mechanisms responsible for these decreases in activity has generally been lacking.

In the present study, we set out to study mechanisms responsible for decreased enzyme activity associated with a nonsynonymous SNP in a novel *TPMT* variant allele discovered in a renal transplantation recipient in Thailand. Specifically, we tested the hypothesis that the decreased enzyme activity resulted from accelerated degradation of the variant allozyme encoded by this allele. Both immunoreactive protein and enzyme activity levels for the new allozyme were reduced after transient expression in mammalian cells. Our studies demonstrate that those decreases were due to rapid degradation as a result of both proteasome and autophagy-mediated process [[10](#R10), [11](#R11)]. These results demonstrated that increased understanding of protein degradation and of processes that target proteins for degradation have mechanistic significance for pharmacogenetics.

## 2. Material and methods

### 2.1. Patients and subjects

The renal transplant recipients studied were treated at Srinagarind Hospital, Khon Kaen, Thailand and Chulalongkorn Hospital, Bangkok, Thailand between and 2007. These patients received azathioprine as part of their standard immunosuppressive therapy program. DNA from 220 unrelated healthy native Thai subjects residing in different regions of Thailand were used to genotype TPMT*27 [[16](#R16)]. Written informed consent was obtained from all subjects. The study protocol was approved by the Ethics Committees on Human Research of Khon Kaen ([HE480937](https://www.ncbi.nlm.nih.gov/nuccore/HE480937)) and Chulalongkorn Universities (132/2006).

### 2.2. RBC TPMT activity assay

TPMT activity was determined in RBC lysates of renal transplantation patients as described previously [[17](#R17)]. Patients who had received allopurinol or blood transfusions within 3 months prior to sample collection were excluded. Specific activity was expressed as nmol 6-methylthioguanine (6-MTG) g^−1^ Hb. Hr^−1^.

### 2.3. TPMT enzyme activity assay for recombinant protein

TPMT enzyme activity in COS-1 cell lysates was measured as described previously with a radiochemical assay based on the methylation of 6-mercaptopurine (6-MP) with [^14^C-methyl]-AdoMet (PerkinElmer, Waltham, MA) as the methyl donor [[18](#R18)].

### 2.4. TPMT*3C and TPMT*6 genotyping

*TPMT***3C* was genotyped using a Custom Taqman^®^ SNP Genotyping Assay (Applied Biosystem, Carlsbad, CA), and TPMT*6 was genotyped using nested PCR as described previously [[19](#R19), [20](#R20)].

### 2.5. DNA sequence analysis and TPMT*27 genotyping

In samples with low TPMT activity that did not have *TPMT***3C* or **6* polymorphisms, the TPMT ORF was amplified as described previously [[21](#R21)]. PCR amplicons were purified and sequenced on both strands. To ensure that the novel *TPMT***27* allele was not an artifact introduced by PCR amplification, two independent amplifications were performed with DNA from this patient and sequencing was repeated.

*TPMT***27* genotyping was performed using a Custom Taqman^®^ SNP Genotyping Assay (Applied Biosystems) using an ABI PRISM 7500 Real-Time PCR System (Applied Biosystem). The sense primer was 5′-TGGAAATCAGTGAACTTGGGATACAA-3′ and the antisense primer was 5′-TTCCAGGAATTTCGGTGATTGGT-3′. The wild type (WT) probe was 5′-TCTTCTGAGTAAGAAAGAT-3′ with a 5′-fluoresceine (VIC) label, and the *TPMT***27* probe was 5′-CTGAGTCAGAAAGAT-3′ with a 6-carboxyfluoresceine (FAM) label. Control samples consisting of homozygous WT and samples heterozygous for *TPMT***27* were included to ensure genotyping accuracy.

### 2.6. TPMT site-directed mutagenesis and transient expression

Expression constructs for HA-tagged WT, *3A, *3C and *27 TPMT were created with the circular PCR. Sequences of mutated inserts were confirmed by DNA sequencing. These constructs were used to transfect COS-1 cells using Lipofectamine 2000 (Invitrogen, Carlsbad, CA). pSV-β-Galactosidase (Promega, Madison WI) was co-transfected to make it possible to correct for variation in transfection efficiency. Transfected cells were harvested and lysed, homogenates were centrifuged and supernatants were stored at −80°C.

### 2.7. Western blot analysis

Rabbit polyclonal antibodies directed against residues 40–59 at the amino terminus of human TPMT were used to perform quantitative Western blot analysis [[6](#R6)]. The ECL Western Blotting System (Amersham Pharmacia, Piscataway, NJ) was used to detect bound antibody, and results were expressed as a percentage of four samples of WT allozyme control run on the same gel.

### 2.8. RRL translation and degradation analysis

Transcription and translation of TPMT WT and variant constructs were performed with the TNT^®^ coupled RRL System (Promega), as described by Wang *et al* [[11](#R11)]. For the protein degradation experiments, ^35^S-methionine-labeled protein was added to an ATP generating system that contained “untreated” RRL, and aliquots were removed at 0, 2, 4 and 6 h for SDS-PAGE, followed by autoradiography.

### 2.9. RRL translation and immunoprecipitation (IP) analysis

TNT^®^ coupled RRL translation products were also used to perform IP analysis. Specifically, 10 μl aliquots of the reaction mixture were incubated with 200 μl untreated RRL at 4°C for 1 h. IP was then performed by conjugating immobilized protein G with anti-hsp70 (Abcam Inc., Cambridge, MA) and anti-hsp90 (Abcam) monoclonal antibodies. After incubation with the RRL reaction mixtures, the resin was washed five times with buffer. Bound proteins were then eluted and subjected to SDS-PAGE analysis, followed by autoradiography. The SDS-PAGE samples were transferred to PVDF membranes that were blotted with anti-hsp70 and anti-hsp90 antibodies, followed by ECL detection.

### 2.10. Immunofluorescence microscopy

COS-1 cells were grown on cover slips for 24 h prior to transfection with HA-tagged WT TPMT, TPMT*27 or TPMT*3A [[10](#R10)], and MG132, 20 μM, or DMSO were added. The cells were fixed in 4% paraformaldehyde at room temperature for 12–15 min, followed by the addition of 0.5% Triton X-100 (Sigma) at room temperature for 5 min. They were then washed with phosphate buffered saline (PBS), were “blocked” with 5% goat serum, were washed once again with PBS and were incubated with the primary anti-HA antibody (Covance, Berkeley, CA) diluted 1/200, followed by incubation with fluorescein isothiocyanate conjugated secondary antibody diluted 1/300 (Invitrogen). Finally, the cells were washed once again with PBS. The stained cells were visualized with a Zeiss LSM 510 Confocal Laser Scanning Microscope.

### 2.11. RNA interference studies

siRNA directed against ATG7 as well as control siRNA were purchased from Qiagen. HEK293 cells were plated in 6-well plates for one day and were co-transfected with ATG7 siRNA and HA-tagged TPMT constructs by using the X-tremeGENE siRNA Transfection Reagent (Roche Applied Science, Indianapolis, IN). The cells were harvested 48 h after transfection.

### 2.12. TPMT structural analysis

The crystal structure of TPMT bound to S-adenosyl-L-homocysteine (AdoHcy) (PDB accession code 2BZG) [[22](#R22)] was used for modeling and MD simulations. AdoHcy, bis-tris propane, and most solvent atoms were removed computationally, and selenomethionine residues were replaced with sulfur-containing methionine to generate a WT human TPMT model. Two water molecules in the Tyr107 hydrogen bonding network were retained. To generate a model of the TPMT*27 Asp107 variant, the Tyr107 in the WT structure was computationally replaced with aspartic acid. Parallel MD simulations of both WT TPMT (Tyr107) and TPMT*27 (Asp107) were performed using the GROMACS-3.3.3 software package and the GROMOS-87 force field [[23](#R23), [24](#R24)]. Using the SPC water model, the protein structures were solvated in a cubic periodic box with walls extending at least 9 Å from the protein molecules. Energy was minimized in 500 steps using a steepest descents minimization procedure. In order to solvate the system without distorting the protein molecule, 30 ps of position-restrained molecular dynamics was first calculated. Subsequently, an extended molecular dynamics simulation was carried out for 2 ns with coordinates saved for analysis every 1 ps. A time step of 2 fs was used, and the temperature of the system was coupled to a reference bath of 300K, with a coupling time constant of 0.1 ps. For analysis of the trajectory, the r.m.s.d. for the polypeptide backbone atoms at different time points was calculated relative to the starting structure.

## 3. Results

### 3.1. Identification of a novel TPMT variant allele

Red blood cell (RBC) TPMT activity was determined in 119 Thai renal transplantation recipients, all of whom had received azathioprine during immunosuppressive therapy. TPMT activity ranged from less than 6.24 nmol 6-methylthioguanine (6-MTG). g^−1^Hb. h^−1^ to 74.36 nmol 6-MTG. g^−1^Hb. h^−1^ ([Fig. 2](#F2)) [[20](#R20)]. DNA from these patients was also genotyped for TPMT*3C and TPMT*6, the two most common “low activity alleles” in Asian populations [[6](#R6), [7](#R7), [12](#R12)]. DNA samples from 9 patients who had RBC TPMT activity below 27 nmol 6-MTG. g^−1^Hb. h^−1^and who did not carry *TPMT***3C* or *TPMT***6* were used to amplify the entire *TPMT* ORF to determine whether novel polymorphisms might be detected by DNA sequencing. One patient (see the arrow in [Fig. 2](#F2)) with activity of 19.8 nmol 6-MTG. g^−1^Hb. h^−1^ had a novel nonsynonymous 319 T>G (107 Tyr>Asp) SNP in exon 5 of the *TPMT* gene. We have designated this new allele *TPMT***27*. An additional T>C nucleotide change was observed at *TPMT* ORF position 474 in the same patient. Therefore, this patient’s genotype was *TPMT***1S/***27*, although we could not determine whether the two variant nucleotides were on the same allele. The patient was a 59 year-old woman with end stage renal disease. She underwent living donor renal transplantation at King Chulalongkorn Memorial Hospital, Bangkok, Thailand, and was treated with cyclosporine, azathioprine and prednisolone. Her laboratory data are listed in [Supplementary Table 1](#SD1). The patient did not experience myelosuppression during azathioprine therapy.

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6df1/2815203/e3caaaece3a3/nihms161970f2.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=2815203_nihms161970f2.jpg)

Frequency distribution histogram of RBC TPMT activities for 119 renal transplantation recipients in Thailand. The arrow indicates the patient who carried the TPMT*27 allele.

Subsequently, 220 healthy Thai subjects were screened for the *TPMT* 319 T > G variant, but none carried this allele. Mechanisms responsible for decreased activity associated with the common *TPMT***3A* allele have been studied in detail [[8](#R8)–[11](#R11)]. We set out to determine whether similar mechanisms might apply to this novel allozyme.

### 3.2. TPMT*27 expression

Decreased protein quantity is a common mechanism responsible for the functional effects of genetic polymorphisms and mutations [[3](#R3), [4](#R4)]. Therefore, as the first step*,* we determined the level of immunoreactive protein expressed for the new TPMT variant allozyme. A mammalian expression construct for *TPMT***27* was generated by site-directed mutagenesis, and both human WT and TPMT*27 constructs were transfected into COS-1 cells. Data from Western blot analyses performed with an antibody directed against the N-terminal portion of the protein are shown in [Fig. 3A and 3B](#F3). The average level of TPMT*27 protein was only 17.2 ± 3.1% of the WT. Since this quantitation involved comparison with four samples of the WT allozyme run on the same gel, but without a “standard curve”, these data must be regarded as striking, but only semi-quantitative in nature. This same caution applies to all subsequent Western blot analyses – although the differences seen between WT and variant allozymes were uniformly striking. TPMT enzyme activity was also assayed for both WT and *27, and *27 activity was reduced to 7.6 ± 2.1% of that for the WT allozyme ([Fig. 3C](#F3)). These results were then compared with previous data published by our laboratory for a series of TPMT variant allozymes ([Fig. 3D](#F3)) [[6](#R6)]. The *TPMT***27* data correlated well with those data, data that showed a direct positive correlation between levels of activity and enzyme protein for all allozymes except *TPMT***5*, a variant with an amino acid alteration that disrupts the active site of the enzyme [[25](#R25)]. We next set out to determine possible mechanisms that might be responsible for the decreased level of TPMT*27 allozyme protein.

### Figure 3.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6df1/2815203/08e74567877d/nihms161970f3.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=2815203_nihms161970f3.jpg)

TPMT*27 protein expression and enzyme activity after transient expression in COS-1 cells. (A) Representative Western blot results. (B) TPMT*27 allozyme immunoreactive protein levels. Protein levels are expressed as a percentage of WT. Values shown are mean ± SEM for 6 independent determinations. (C) TPMT*27 allozyme enzyme activity levels. Activities are expressed as a percentage of WT. Values shown are mean ± SEM for 6 independent determinations. (D) Correlation of human TPMT allozyme enzyme activity and immunoreactive protein levels after transient expression in COS-1 cells. Modified with permission from Salavaggione et al. [6]. The allozyme described in the present study is indicated as a black dot. Correlation coefficients were calculated by omitting data for TPMT*5.

### 3.3. Rabbit reticulocyte lysate (RRL) degradation studies

Previous studies of *TPMT* **3A* showed that accelerated degradation was responsible for the decreased protein level observed for that allozyme [[4](#R4), [9](#R9), [11](#R11)]. Therefore, we set out to determine whether the change in amino acid 107 from Tyr to Asp in TPMT*27 might also be associated with accelerated degradation. As a first step, a RRL system was employed to compare the degradation of WT with TPMT*27. TPMT *3A was included as a positive control. Specifically, ^35^S-methionine labeled WT, *3A and *27 TPMT were generated using a treated RRL. Aliquots of the recombinant protein were then added to untreated RRL to initiate degradation. Both TPMT*3A and TPMT*27 were degraded rapidly, with only 38% and 48%, respectively, remaining after 6 h, while the level of the WT allozyme was decreased only 3% at the same time ([Fig. 4](#F4)). Therefore, accelerated degradation is one factor responsible for the decreased TPMT*27 enzyme activity and protein levels shown in [Fig. 3](#F3).

### Figure 4.

![Figure 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6df1/2815203/23547196b1d9/nihms161970f4.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=2815203_nihms161970f4.jpg)

Rabbit reticulocyte lysate (RRL) degradation studies. 35S-Methionine labeled WT, *3A and *27 TPMT allozymes were incubated for various times with an untreated RRL that included an ATP generating system. A representative result is shown at the bottom. Each point is the mean ± SEM for 3 independent experiments.

### 3.4. Proteasome-mediated degradation and aggresome formation

Wang *et al.* showed that alteration of the 2 variant amino acids in TPMT*3A caused protein misfolding [[10](#R10), [11](#R11)], which could trigger a series of events resulting in proteasome-mediated degradation [[9](#R9)–[11](#R11)]. Those authors also demonstrated that molecular chaperones such as hsp70 and hsp90 were involved in this process [[11](#R11)]. Therefore, we determined whether *TPMT***27* might also bind preferentially to chaperone proteins. We began by inhibiting proteasome-mediated degradation. Specifically, COS-1 cells were transfected with hemagglutinin (HA)-tagged WT, *3A or *27 TPMT constructs. After 24 h in culture, the cells were treated with either DMSO or 20 μM MG132, a proteasome inhibitor. [Fig. 5A](#F5) shows that average values of immunoreactive protein for both *3A and *27 were significantly elevated (P<0.01) after treatment with MG132, indicating proteasome involvement in TPMT*27 degradation.

### Figure 5.

![Figure 5](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6df1/2815203/2849e2a4fb43/nihms161970f5.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=2815203_nihms161970f5.jpg)

TPMT*27 degradation, chaperone protein interaction and aggresome formation. (A) Western blot analysis of TPMT*27 immunoreactive protein after MG132 treatment. Specifically, COS-1 cells were transfected with WT, *3A and *27 TPMT and were then incubated with DMSO or 20 μM MG132 in DMSO for 24 h, followed by Western blot analysis. Protein levels are expressed as a percentage of WT. Values shown are mean ± SEM for 3 independent determinations. The “insert” shows a representative Western blot result. (B) Interaction of chaperone proteins with TPMT*27. The figure at the top shows the translation of WT, *27, *3A and *3C TPMT in a RRL system. 35S-Methionine labeled WT, *27,*3A and *3C TPMT allozymes were then subjected to IP with immobilized protein G conjugated with anti-hsp70 or anti-hsp90 antibody. IP results are shown at the bottom of the figure. The top panels show the results for IP. The lower panels show the results of immunoblot (IB) analysis for each chaperone protein. (C) TPMT*27 aggresome formation. COS-1 cells were transfected with HA-tagged constructs with and without exposure to MG132. Aggresomes were visualized with confocal epifluorescence microscopy and are indicated by yellow arrows. A total of 200 cells were counted in each experiment. The bar graph shows data for aggresome formation in cells transfected with WT, *27 and *3A TPMT with or without MG132 treatment. Values shown are mean ± SEM for 3 independent determinations.

We also tested the possible involvement of chaperone proteins in *27 degradation. WT and TPMT*3A were included as negative and positive controls, respectively, as well as TPMT*3C, an allozyme with a lower rate of degradation than TPMT*3A [[8](#R8)–[11](#R11)]. Specifically, equal quantities of ^35^S-methionine labeled allozymes generated with the RRL system were incubated with untreated RRL and were then applied to Protein-G beads conjugated to anti-hsp70 or anti-hsp90 antibodies. While little or no binding to either hsp70 or hsp90 was seen for WT TPMT, strong binding was observed for *27 and *3A ([Fig. 5B](#F5)). There was much weaker binding of TPMT*3C by the two molecular chaperones ([Fig. 5B](#F5)). Therefore, hsp70 and hsp90 might also participate in TPMT*27 degradation – just as they do for TMPT*3A ([Fig. 1](#F1), Pathway #2) [[11](#R11)].

Aggresomes can be generated in cells when proteins misfold and/or when the capacity of proteasome-mediated degradation is exceeded [[26](#R26), [27](#R27)]. Aggresomes are perinuclear bodies composed of misfolded protein [[26](#R26), [27](#R27)]. Therefore, we performed immunofluorescence confocal microscopy after the transfection of COS-1 cells with HA-tagged WT, *3A and *27 TPMT. Representative confocal images are shown in [Fig. 5C](#F5). Large protein aggregates compatible with aggresome formation, indicated by the yellow arrows, were observed in cells transfected with TPMT*3A and TPMT*27 – but not with the WT allozyme – after treatment with MG132. Aggresome formation was observed in 20% of COS-1 cells transfected with TPMT*3A, and 18% of COS-1 cells transfected with TPMT*27 ([Fig. 5C](#F5), see Pathway #3 in [Fig. 1](#F1)).

### 3.5. Autophagy and TPMT*27 degradation

Recent studies performed with a yeast gene deletion library indicated that autophagy can also play a role in TPMT *3A degradation [[8](#R8)]. To determine whether autophagy might be included among mechanisms responsible for TPMT*27 clearance, we treated cells expressing TPMT with 3-methyladenine (3MA), an autophagy inhibitor [[28](#R28)]. We then performed Western blot analysis of WT, *3A, *3C and *27 TPMT. Western blot analysis showed elevated levels of TPMT*3A and *27 after 3-MA treatment ([Fig. 6A](#F6)), with only slight or no change for WT and TPMT*3C ([Fig. 6A](#F6)). We then used siRNA to knock down ATG7 ([Fig. 6B](#F6)), a gene that encodes a protein required for the biogenesis of autophagic vesicles [[29](#R29)]. After HEK293 cells were transfected with siRNA directed against ATG7 and expression constructs for TPMT allozymes, Western blot analysis showed that expression of both TPMT*3A and *27, but not WT TPMT, were increased ([Fig. 6C and 6D](#F6)) – supporting a role for autophagy in TPMT*27 clearance (see Pathway #4 in [Fig. 1](#F1)).

### Figure 6.

![Figure 6](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6df1/2815203/eebc764bfbf5/nihms161970f6.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=2815203_nihms161970f6.jpg)

Autophagy and TPMT*27 degradation. (A) Immunoblot analysis of HA-tagged WT, *27, *3A and *3C TPMT transfected into HEK293 cells. The cells were treated with 10 mM 3-methyladenosine (3MA) or water. Immunoblots were performed with anti-HA antibody (upper panel), or with anti-actin antibody (lower panel). (B) RT-PCR analysis of ATG7 in HEK293 cells transiently transfected with ATG7 siRNA. The values shown are mean ± SEM for 3 independent determinations. (C) Immunoblot analysis of WT, *27 and *3A TPMT transfected into HEK293 cells with or without siRNA “knockdown” of ATG7. (D) Immunoblot data for TPMT*27 and TPMT*3A with and without siRNA “knockdown” of ATG7. Value shown are mean ± SEM for 3 independent experiments.

These results indicated that TPMT *27, like TPMT*3A, displays decreased activity as a result of a decrease in the quantity of enzyme protein due to proteasome-mediated degradation as well as autophagy – with the involvement of chaperone proteins (see [Fig. 1](#F1)). All of these processes are thought to be related to protein misfolding. Since the crystal structure of human TPMT is known [[22](#R22), [30](#R30)], we used that information to apply structural analysis and molecular dynamics modeling to determine whether TPMT*27 misfolding might contribute to the cascade of events resulting in the accelerated clearance of this variant allozyme.

### 3.6. TPMT*27 structural analysis

The crystal structure of human WT TPMT showed that Tyr107 is located on the surface of the protein, distant from the active site [[22](#R22)]. The Tyr107 side chain participates in an extended hydrogen bonding network that includes the side chains of four other hydrophilic surface residues and two water molecules. This hydrogen bonding network tethers residues on two β-strands (Tyr107, Glu109, and Lys119) to two residues on an adjacent α-helix (Glu93 and Gln97) ([Fig. 7B](#F7)). The Asp107 variant introduces a hydrophilic residue that can be accommodated easily both sterically and chemically, but the shorter aspartic acid side chain would not be able to participate in the hydrogen bonding network. Therefore, the Tyr>Asp alteration in amino acid sequence could compromise this network of surface hydrogen bonds and, as a result, alter protein folding or stability.

### Figure 7.

![Figure 7](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6df1/2815203/c38af0f9c549/nihms161970f7.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=2815203_nihms161970f7.jpg)

TPMT*27 and WT TPMT structural analysis. (A) Deviations of WT TPMT (Try107) and the *27 Asp107 variant structures during molecular dynamics (MD) simulation. Differences in polypeptide mainchain backbone atom positions between the initial TPMT crystal structure and energy minimized WT (Tyr107) or variant (Asp107) structures during MD simulations were calculated for structures sampled at 1 ns increments. The trajectory for the WT Tyr107 MD simulation (purple) shows the r.m.s.d. plateaus, indicating a stable equilibrium for the energy minimization. The trajectory for the Asp107 variant MD simulation (blue) shows a higher r.m.s.d. than that for the WT Tyr107 which continues to increase at the end of the 2 ns simulation, indicating that the energy minimized structures are increasingly divergent and have not yet attained a stable equilibrium. (B) Structural distortion of the energy minimized Asp107 variant. The WT Tyr107 crystal structure is shown as a purple ribbon diagram with side chains of Tyr107 and other residues in the surface hydrogen bonding network, and with AdoHcy bound in the active site represented as ball-and-stick structures. Hydrogen bonds are drawn as black dashed lines, and oxygen atoms of the two water molecules participating in the hydrogen bonding are also shown. Superimposed is the ribbon diagram for the energy minimized variant Asp107 structure at the end of the 2 ns MD simulation, in transparent blue. Both the β-strand containing Tyr107/Asp 107 and the adjacent α-helix containing other residues in the hydrogen bonding network are distorted, together with a variety of other structural features near and distant from the Asp107 substitution site. The figure was prepared using PyMOL [41].

To probe the possible structural consequences of the 107 Tyr>Asp substitution, molecular dynamics (MD) simulations were performed with both the WT Tyr107 and the variant Asp107 molecular structures. Over the course of 2 ns simulations, molecular coordinates were saved for analysis every 1 ps. The difference between the structures at any time point from the starting coordinate was measured by calculating the root mean square deviation (r.m.s.d.) of the backbone atoms. A plot of the backbone r.m.s.d. vs. time for the WT Tyr107 MD simulation showed that these deviations reached a plateau at ~1.5 Å, indicating that the energy minimized structures reached a stable equilibrium ([Fig. 7A](#F7)). In contrast, the backbone r.m.s.d. plot for the Asp107 variant did not plateau after a 2 ns simulation and continued to rise beyond 2.0 Å, indicating that the energy minimized structures continued to increasingly deviate from the initial model and failed to achieve a stable equilibrium ([Fig. 7A](#F7)). A comparison of the trajectories for the two MD simulations indicated that the Asp107 substitution was less compatible with the WT TPMT protein fold than the WT Tyr107, supporting the hypothesis that the inability of the shorter aspartic acid side chain to contribute to the surface hydrogen bonding network might compromise the TPMT structure.

By superimposing the Asp107 coordinate obtained at the end of the 2ns MD simulation with the initial WT Tyr107 crystal structure, further details of the conformational deviation became apparent ([Fig. 7B](#F7)). Both the β-strand containing Tyr107Asp and the adjacent helix (residues 92–103) that contains residues in the same surface hydrogen bonding network were substantially shifted. Large shifts also occurred in other regions of the structure, both near and distant from the Tyr107Asp site, consistent with the large and increasing backbone r.m.s.d. calculated during our MD 2 ns simulation. This widespread distortion of structural features throughout the TPMT protein fold supports the conclusion that the Asp107 substitution is incompatible with the native structure of WT TPMT.

## 4. Discussion

Nonsynonymous SNPs are an important cause of inherited variation in drug response. Inherited alteration of only one or two amino acids often results in a decrease in the quantity of protein (see [Fig. 3D](#F3)) [[4](#R4)]. When the mechanism responsible has been studied, protein degradation has been the most frequent cause [[4](#R4), [6](#R6)–[11](#R11), [26](#R26), [27](#R27), [31](#R31)–[38](#R38)]. TPMT represents a striking example of both the clinical relevance of nonsynonymous SNPs for pharmacogenetics and the importance of protein degradation for these effects [[1](#R1), [2](#R2)]. TPMT genotyping studies have identified up to 30 variant alleles in different ethnic groups [[12](#R12)] since the TPMT cDNA and gene were first cloned and characterized in 1993 and 1996, respectively [[7](#R7), [39](#R39)]. *TPMT***27*, 319 T>G, 107 Tyr>Asp, was recently reported in a renal transplantation recipient in Thailand although it was not functionally characterized [[20](#R20)]. In the present study, we genotyped 220 healthy Thai subjects for *TPMT***27*, but failed to identify any additional carriers of this novel allele, making it likely that, in this population, it represents a mutation rather than a polymorphism. However, the purpose of the present study was neither to determine the population frequency of this variant nor to determine its clinical implications, but rather to study mechanisms responsible for the decreased level of enzyme activity associated with this variant allele.

A chaperone protein-dependent, proteasome-medicated pathway [[9](#R9)–[11](#R11)] and autophagy [[8](#R8)], have both been shown to contribute to the degradation of TPMT*3A, the most common variant allele in Caucasian subjects. We tested the hypothesis that protein degradation might also contribute to the decrease in enzyme protein that we observed for TPMT*27. Transient expression of TPMT*27 in COS-1 cells, followed by quantitative Western blot analysis and assay of enzyme activity showed a dramatic decrease in levels of both immunoreactive protein and enzyme activity. Those results also fit well with the correlation between protein and activity that we reported previously for 13 TPMT allozymes ([Fig. 3D](#F3)) [[6](#R6)]. Furthermore, TPMT*27 protein was rapidly degraded in a RRL system ([Fig. 4](#F4)). Immunoprecipitation demonstrated that two chaperone proteins which had been shown to be involved in targeting TPMT*3A for degradation, hsp70 and hsp 90 [[11](#R11)], also interacted with TPMT*27. In addition, degradation of TPMT*27 was inhibited by the proteasome inhibitor MG132 ([Figure 5A](#F5)), and TPMT*27, like TPMT*3A [[10](#R10)], formed aggresomes in COS-1 cells after proteasome inhibition ([Fig. 5C](#F5)). We also observed that TPMT*27 protein levels increased after the inhibition of autophagy with 3-MA ([Fig. 6A](#F6)), or after siRNA knockdown of ATG7 ([Fig. 6B, 6C and 6D](#F6)). Therefore, autophagy appeared to be another route for TPMT*27 degradation just as it is for TPMT*3A [[8](#R8)]. In summary, both proteasome-mediated degradation and autophagy contribute to TPMT*27 degradation. Finally, structural analysis and MD simulation suggested that substitution of Asp for Tyr at TPMT*27 position 107 might result in protein misfolding and resultant instability ([Fig. 7](#F7)).

It should be emphasized that not all TPMT variant allozymes with decreased activity can be explained by protein degradation. For example, TPMT*5 displays a moderately reduced level of immunoreactive protein together with nearly complete absence of enzyme activity [[6](#R6)]. In the case of TPMT*5, an alteration at the active site of the enzyme rather than protein degradation appears to account for the loss of function [[25](#R25)]. TPMT is not the only example for which inherited alteration in protein sequence results in reduced protein quantity. Other genes encoding enzymes with common nonsynonymous SNPs that are associated with functionally significant decreases in protein quantity include catechol O-methyltransferase (COMT), histamine N-methyltransferase (HNMT), a series of sulfotransferase (SULT) enzyme and quinone oxidoreductase 1 [[1](#R1), [17](#R17), [31](#R31)–[37](#R37), [40](#R40)]. Clearly, altered protein quantity is an important mechanism responsible for the functional effects of inherited variation in amino acid sequence, and this alteration, most often a decrease, frequently results from protein degradation.

In summary, we have studied mechanisms responsible for the decreased activity of TPMT*27, a novel variant allozyme for an important drug-metabolizing enzyme. We used this novel TPMT allele to test the hypothesis that protein clearance mechanisms might explain loss-of-function as a result of this genetic variation – perhaps as a result of protein misfolding. The mechanisms identified involved accelerated degradation mediated by both proteasome and autophagy pathways. Ultimately, it would be hoped that protein misfolding – and any related acceleration in degradation – might be predicted purely computationally, but that goal for pharmacogenetics still remains in the future.

## Supplementary Material

## Acknowledgments

We thank Luanne Wussow for her assistance with the preparation of this manuscript. Funded in part by National Institutes of Health (NIH) grants R01 GM28157, R01 CA132780, U01 GM61388 (The Pharmacogenetics Research Network) and a PhRMA Foundation “Center of Excellence in Clinical Pharmacology” Award. The TPMT study in renal transplant recipients in Thailand was funded by the National Center for Genetic Engineering and Biotechnology, Thailand.

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Weinshilboum R. Thiopurine pharmacogenetics: clinical and molecular studies of thiopurine methyltransferase. Drug Metab Dispos. 2001;29:601–5.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/11259360/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Thiopurine%20pharmacogenetics:%20clinical%20and%20molecular%20studies%20of%20thiopurine%20methyltransferase&author=R%20Weinshilboum&volume=29&publication_year=2001&pages=601-5&pmid=11259360&)

2. Weinshilboum R. Inheritance and drug response. N Engl J Med. 2003;348:529–37. doi: 10.1056/NEJMra020021.  [DOI](https://doi.org/10.1056/NEJMra020021) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12571261/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Inheritance%20and%20drug%20response&author=R%20Weinshilboum&volume=348&publication_year=2003&pages=529-37&pmid=12571261&doi=10.1056/NEJMra020021&)

3. Wang L, Weinshilboum R. Thiopurine S-methyltransferase pharmacogenetics: insights, challenges and future directions. Oncogene. 2006;25:1629–38. doi: 10.1038/sj.onc.1209372.  [DOI](https://doi.org/10.1038/sj.onc.1209372) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16550163/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncogene&title=Thiopurine%20S-methyltransferase%20pharmacogenetics:%20insights,%20challenges%20and%20future%20directions&author=L%20Wang&author=R%20Weinshilboum&volume=25&publication_year=2006&pages=1629-38&pmid=16550163&doi=10.1038/sj.onc.1209372&)

4. Weinshilboum R, Wang L. Pharmacogenetics: inherited variation in amino acid sequence and altered protein quantity. Clin Pharmacol Ther. 2004;75:253–8. doi: 10.1016/j.clpt.2003.12.002.  [DOI](https://doi.org/10.1016/j.clpt.2003.12.002) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15080131/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Pharmacogenetics:%20inherited%20variation%20in%20amino%20acid%20sequence%20and%20altered%20protein%20quantity&author=R%20Weinshilboum&author=L%20Wang&volume=75&publication_year=2004&pages=253-8&pmid=15080131&doi=10.1016/j.clpt.2003.12.002&)

5. Yan L, Zhang S, Eiff B, Szumlanski CL, Powers M, O’Brien JF, et al. Thiopurine methyltransferase polymorphic tandem repeat: genotype-phenotype correlation analysis. Clin Pharmacol Ther. 2000;68:210–9. doi: 10.1067/mcp.2000.108674.  [DOI](https://doi.org/10.1067/mcp.2000.108674) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10976552/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Thiopurine%20methyltransferase%20polymorphic%20tandem%20repeat:%20genotype-phenotype%20correlation%20analysis&author=L%20Yan&author=S%20Zhang&author=B%20Eiff&author=CL%20Szumlanski&author=M%20Powers&volume=68&publication_year=2000&pages=210-9&pmid=10976552&doi=10.1067/mcp.2000.108674&)

6. Salavaggione OE, Wang L, Wiepert M, Yee VC, Weinshilboum RM. Thiopurine S-methyltransferase pharmacogenetics: variant allele functional and comparative genomics. Pharmacogenet Genomics. 2005;15:801–15. doi: 10.1097/01.fpc.0000174788.69991.6b.  [DOI](https://doi.org/10.1097/01.fpc.0000174788.69991.6b) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16220112/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Thiopurine%20S-methyltransferase%20pharmacogenetics:%20variant%20allele%20functional%20and%20comparative%20genomics&author=OE%20Salavaggione&author=L%20Wang&author=M%20Wiepert&author=VC%20Yee&author=RM%20Weinshilboum&volume=15&publication_year=2005&pages=801-15&pmid=16220112&doi=10.1097/01.fpc.0000174788.69991.6b&)

7. Szumlanski C, Otterness D, Her C, Lee D, Brandriff B, Kelsell D, et al. Thiopurine methyltransferase pharmacogenetics: human gene cloning and characterization of a common polymorphism. DNA Cell Biol. 1996;15:17–30. doi: 10.1089/dna.1996.15.17.  [DOI](https://doi.org/10.1089/dna.1996.15.17) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8561894/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=DNA%20Cell%20Biol&title=Thiopurine%20methyltransferase%20pharmacogenetics:%20human%20gene%20cloning%20and%20characterization%20of%20a%20common%20polymorphism&author=C%20Szumlanski&author=D%20Otterness&author=C%20Her&author=D%20Lee&author=B%20Brandriff&volume=15&publication_year=1996&pages=17-30&pmid=8561894&doi=10.1089/dna.1996.15.17&)

8. Li F, Wang L, Burgess RJ, Weinshilboum RM. Thiopurine S-methyltransferase pharmacogenetics: autophagy as a mechanism for variant allozyme degradation. Pharmacogenet Genomics. 2008;18:1083–94. doi: 10.1097/FPC.0b013e328313e03f.  [DOI](https://doi.org/10.1097/FPC.0b013e328313e03f) | [PMC free article](/articles/PMC2583164/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18820593/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Thiopurine%20S-methyltransferase%20pharmacogenetics:%20autophagy%20as%20a%20mechanism%20for%20variant%20allozyme%20degradation&author=F%20Li&author=L%20Wang&author=RJ%20Burgess&author=RM%20Weinshilboum&volume=18&publication_year=2008&pages=1083-94&pmid=18820593&doi=10.1097/FPC.0b013e328313e03f&)

9. Tai HL, Krynetski EY, Schuetz EG, Yanishevski Y, Evans WE. Enhanced proteolysis of thiopurine S-methyltransferase (TPMT) encoded by mutant alleles in humans (TPMT*3A, TPMT*2): mechanisms for the genetic polymorphism of TPMT activity. Proc Natl Acad Sci U S A. 1997;94:6444–9. doi: 10.1073/pnas.94.12.6444.  [DOI](https://doi.org/10.1073/pnas.94.12.6444) | [PMC free article](/articles/PMC21069/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9177237/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&title=Enhanced%20proteolysis%20of%20thiopurine%20S-methyltransferase%20(TPMT)%20encoded%20by%20mutant%20alleles%20in%20humans%20(TPMT*3A,%20TPMT*2):%20mechanisms%20for%20the%20genetic%20polymorphism%20of%20TPMT%20activity&author=HL%20Tai&author=EY%20Krynetski&author=EG%20Schuetz&author=Y%20Yanishevski&author=WE%20Evans&volume=94&publication_year=1997&pages=6444-9&pmid=9177237&doi=10.1073/pnas.94.12.6444&)

10. Wang L, Nguyen TV, McLaughlin RW, Sikkink LA, Ramirez-Alvarado M, Weinshilboum RM. Human thiopurine S-methyltransferase pharmacogenetics: variant allozyme misfolding and aggresome formation. Proc Natl Acad Sci U S A. 2005;102:9394–9. doi: 10.1073/pnas.0502352102.  [DOI](https://doi.org/10.1073/pnas.0502352102) | [PMC free article](/articles/PMC1153717/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15967990/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&title=Human%20thiopurine%20S-methyltransferase%20pharmacogenetics:%20variant%20allozyme%20misfolding%20and%20aggresome%20formation&author=L%20Wang&author=TV%20Nguyen&author=RW%20McLaughlin&author=LA%20Sikkink&author=M%20Ramirez-Alvarado&volume=102&publication_year=2005&pages=9394-9&pmid=15967990&doi=10.1073/pnas.0502352102&)

11. Wang L, Sullivan W, Toft D, Weinshilboum R. Thiopurine S-methyltransferase pharmacogenetics: chaperone protein association and allozyme degradation. Pharmacogenetics. 2003;13:555–64. doi: 10.1097/01.fpc.0000054124.14659.99.  [DOI](https://doi.org/10.1097/01.fpc.0000054124.14659.99) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12972954/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Thiopurine%20S-methyltransferase%20pharmacogenetics:%20chaperone%20protein%20association%20and%20allozyme%20degradation&author=L%20Wang&author=W%20Sullivan&author=D%20Toft&author=R%20Weinshilboum&volume=13&publication_year=2003&pages=555-64&pmid=12972954&doi=10.1097/01.fpc.0000054124.14659.99&)

12. Garat A, Cauffiez C, Renault N, Lo-Guidice JM, Allorge D, Chevalier D, et al. Characterisation of novel defective thiopurine S-methyltransferase allelic variants. Biochem Pharmacol. 2008;76:404–15. doi: 10.1016/j.bcp.2008.05.009.  [DOI](https://doi.org/10.1016/j.bcp.2008.05.009) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18602085/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20Pharmacol&title=Characterisation%20of%20novel%20defective%20thiopurine%20S-methyltransferase%20allelic%20variants&author=A%20Garat&author=C%20Cauffiez&author=N%20Renault&author=JM%20Lo-Guidice&author=D%20Allorge&volume=76&publication_year=2008&pages=404-15&pmid=18602085&doi=10.1016/j.bcp.2008.05.009&)

13. Tamm R, Oselin K, Kallassalu K, Magi R, Anier K, Remm M, et al. Thiopurine S-methyltransferase (TPMT) pharmacogenetics: three new mutations and haplotype analysis in the Estonian population. Clin Chem Lab Med. 2008;46:974–9. doi: 10.1515/CCLM.2008.187.  [DOI](https://doi.org/10.1515/CCLM.2008.187) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18605963/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Chem%20Lab%20Med&title=Thiopurine%20S-methyltransferase%20(TPMT)%20pharmacogenetics:%20three%20new%20mutations%20and%20haplotype%20analysis%20in%20the%20Estonian%20population&author=R%20Tamm&author=K%20Oselin&author=K%20Kallassalu&author=R%20Magi&author=K%20Anier&volume=46&publication_year=2008&pages=974-9&pmid=18605963&doi=10.1515/CCLM.2008.187&)

14. Ujiie S, Sasaki T, Mizugaki M, Ishikawa M, Hiratsuka M. Functional characterization of 23 allelic variants of thiopurine S-methyltransferase gene (TPMT*2 – *24) Pharmacogenet Genomics. 2008;18:887–93. doi: 10.1097/FPC.0b013e3283097328.  [DOI](https://doi.org/10.1097/FPC.0b013e3283097328) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18708949/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Functional%20characterization%20of%2023%20allelic%20variants%20of%20thiopurine%20S-methyltransferase%20gene%20(TPMT*2%20%E2%80%93%20*24)&author=S%20Ujiie&author=T%20Sasaki&author=M%20Mizugaki&author=M%20Ishikawa&author=M%20Hiratsuka&volume=18&publication_year=2008&pages=887-93&pmid=18708949&doi=10.1097/FPC.0b013e3283097328&)

15. Schaeffeler E, Fischer C, Brockmeier D, Wernet D, Moerike K, Eichelbaum M, et al. Comprehensive analysis of thiopurine S-methyltransferase phenotype-genotype correlation in a large population of German-Caucasians and identification of novel TPMT variants. Pharmacogenetics. 2004;14:407–17. doi: 10.1097/01.fpc.0000114745.08559.db.  [DOI](https://doi.org/10.1097/01.fpc.0000114745.08559.db) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15226673/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Comprehensive%20analysis%20of%20thiopurine%20S-methyltransferase%20phenotype-genotype%20correlation%20in%20a%20large%20population%20of%20German-Caucasians%20and%20identification%20of%20novel%20TPMT%20variants&author=E%20Schaeffeler&author=C%20Fischer&author=D%20Brockmeier&author=D%20Wernet&author=K%20Moerike&volume=14&publication_year=2004&pages=407-17&pmid=15226673&doi=10.1097/01.fpc.0000114745.08559.db&)

16. Tassaneeyakul W, Mahatthanatrakul W, Niwatananun K, Na-Bangchang K, Tawalee A, Krikreangsak N, et al. CYP2C19 genetic polymorphism in Thai, Burmese and Karen populations. Drug Metab Pharmacokinet. 2006;21:286–90. doi: 10.2133/dmpk.21.286.  [DOI](https://doi.org/10.2133/dmpk.21.286) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16946555/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Pharmacokinet&title=CYP2C19%20genetic%20polymorphism%20in%20Thai,%20Burmese%20and%20Karen%20populations&author=W%20Tassaneeyakul&author=W%20Mahatthanatrakul&author=K%20Niwatananun&author=K%20Na-Bangchang&author=A%20Tawalee&volume=21&publication_year=2006&pages=286-90&pmid=16946555&doi=10.2133/dmpk.21.286&)

17. Ford LT, Berg JD. Determination of thiopurine S-methyltransferase activity in erythrocytes using 6-thioguanine as substrate and a non-extraction liquid chromatographic technique. J Chromatogr B Analyt Technol Biomed Life Sci. 2003;798:111–5. doi: 10.1016/j.jchromb.2003.09.017.  [DOI](https://doi.org/10.1016/j.jchromb.2003.09.017) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14630365/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Chromatogr%20B%20Analyt%20Technol%20Biomed%20Life%20Sci&title=Determination%20of%20thiopurine%20S-methyltransferase%20activity%20in%20erythrocytes%20using%206-thioguanine%20as%20substrate%20and%20a%20non-extraction%20liquid%20chromatographic%20technique&author=LT%20Ford&author=JD%20Berg&volume=798&publication_year=2003&pages=111-5&pmid=14630365&doi=10.1016/j.jchromb.2003.09.017&)

18. Weinshilboum RM, Raymond FA, Pazmino PA. Human erythrocyte thiopurine methyltransferase: radiochemical microassay and biochemical properties. Clin Chim Acta. 1978;85:323–33. doi: 10.1016/0009-8981(78)90311-x.  [DOI](https://doi.org/10.1016/0009-8981(78)90311-x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/657528/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Chim%20Acta&title=Human%20erythrocyte%20thiopurine%20methyltransferase:%20radiochemical%20microassay%20and%20biochemical%20properties&author=RM%20Weinshilboum&author=FA%20Raymond&author=PA%20Pazmino&volume=85&publication_year=1978&pages=323-33&pmid=657528&doi=10.1016/0009-8981(78)90311-x&)

19. Srimartpirom S, Tassaneeyakul W, Kukongviriyapan V, Tassaneeyakul W. Thiopurine S-methyltransferase genetic polymorphism in the Thai population. Br J Clin Pharmacol. 2004;58:66–70. doi: 10.1111/j.1365-2125.2004.02112.x.  [DOI](https://doi.org/10.1111/j.1365-2125.2004.02112.x) | [PMC free article](/articles/PMC1884540/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15206995/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=Thiopurine%20S-methyltransferase%20genetic%20polymorphism%20in%20the%20Thai%20population&author=S%20Srimartpirom&author=W%20Tassaneeyakul&author=V%20Kukongviriyapan&author=W%20Tassaneeyakul&volume=58&publication_year=2004&pages=66-70&pmid=15206995&doi=10.1111/j.1365-2125.2004.02112.x&)

20. Vannaprasaht S, Angsuthum S, Avihingsanon Y, Sirivongs D, Pongskul C, Makarawate P, et al. Impact of the heterozygous TPMT*1/*3C genotype on azathioprine-induced myelosuppression in kidney transplant recipients in Thailand. Clin Pharmacol Ther. 2009;31:1524–33. doi: 10.1016/j.clinthera.2009.07.008.  [DOI](https://doi.org/10.1016/j.clinthera.2009.07.008) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19695401/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Impact%20of%20the%20heterozygous%20TPMT*1/*3C%20genotype%20on%20azathioprine-induced%20myelosuppression%20in%20kidney%20transplant%20recipients%20in%20Thailand&author=S%20Vannaprasaht&author=S%20Angsuthum&author=Y%20Avihingsanon&author=D%20Sirivongs&author=C%20Pongskul&volume=31&publication_year=2009&pages=1524-33&pmid=19695401&doi=10.1016/j.clinthera.2009.07.008&)

21. Chang JG, Lee LS, Chen CM, Shih MC, Wu MC, Tsai FJ, et al. Molecular analysis of thiopurine S-methyltransferase alleles in South-east Asian populations. Pharmacogenetics. 2002;12:191–5. doi: 10.1097/00008571-200204000-00003.  [DOI](https://doi.org/10.1097/00008571-200204000-00003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11927834/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Molecular%20analysis%20of%20thiopurine%20S-methyltransferase%20alleles%20in%20South-east%20Asian%20populations&author=JG%20Chang&author=LS%20Lee&author=CM%20Chen&author=MC%20Shih&author=MC%20Wu&volume=12&publication_year=2002&pages=191-5&pmid=11927834&doi=10.1097/00008571-200204000-00003&)

22. Wu H, Horton JR, Battaile K, Allali-Hassani A, Martin F, Zeng H, et al. Structural basis of allele variation of human thiopurine-S-methyltransferase. Proteins. 2007;67:198–208. doi: 10.1002/prot.21272.  [DOI](https://doi.org/10.1002/prot.21272) | [PMC free article](/articles/PMC2750861/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17243178/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proteins&title=Structural%20basis%20of%20allele%20variation%20of%20human%20thiopurine-S-methyltransferase&author=H%20Wu&author=JR%20Horton&author=K%20Battaile&author=A%20Allali-Hassani&author=F%20Martin&volume=67&publication_year=2007&pages=198-208&pmid=17243178&doi=10.1002/prot.21272&)

23. Van Der Spoel D, Lindahl E, Hess B, Groenhof G, Mark AE, Berendsen HJ. GROMACS: fast, flexible, and free. J Comput Chem. 2005;26:1701–18. doi: 10.1002/jcc.20291.  [DOI](https://doi.org/10.1002/jcc.20291) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16211538/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Comput%20Chem&title=GROMACS:%20fast,%20flexible,%20and%20free&author=D%20Van%20Der%20Spoel&author=E%20Lindahl&author=B%20Hess&author=G%20Groenhof&author=AE%20Mark&volume=26&publication_year=2005&pages=1701-18&pmid=16211538&doi=10.1002/jcc.20291&)

24. van Gunsteren WF, Berendsen HJC. Biomos BV Nijenborgh. Vol. 4. AG Groningen; The Netherlands: 1987. Gromos-87 manual; p. 9747.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Biomos%20BV%20Nijenborgh&author=WF%20van%20Gunsteren&author=HJC%20Berendsen&publication_year=1987&)

25. Peng Y, Feng Q, Wilk D, Adjei AA, Salavaggione OE, Weinshilboum RM, et al. Structural basis of substrate recognition in thiopurine s-methyltransferase. Biochemistry. 2008;47:6216–25. doi: 10.1021/bi800102x.  [DOI](https://doi.org/10.1021/bi800102x) | [PMC free article](/articles/PMC2447621/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18484748/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochemistry&title=Structural%20basis%20of%20substrate%20recognition%20in%20thiopurine%20s-methyltransferase&author=Y%20Peng&author=Q%20Feng&author=D%20Wilk&author=AA%20Adjei&author=OE%20Salavaggione&volume=47&publication_year=2008&pages=6216-25&pmid=18484748&doi=10.1021/bi800102x&)

26. Wickner S, Maurizi MR, Gottesman S. Posttranslational quality control: folding, refolding, and degrading proteins. Science. 1999;286:1888–93. doi: 10.1126/science.286.5446.1888.  [DOI](https://doi.org/10.1126/science.286.5446.1888) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10583944/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Posttranslational%20quality%20control:%20folding,%20refolding,%20and%20degrading%20proteins&author=S%20Wickner&author=MR%20Maurizi&author=S%20Gottesman&volume=286&publication_year=1999&pages=1888-93&pmid=10583944&doi=10.1126/science.286.5446.1888&)

27. Kopito RR. Aggresomes, inclusion bodies and protein aggregation. Trends Cell Biol. 2000;10:524–30. doi: 10.1016/s0962-8924(00)01852-3.  [DOI](https://doi.org/10.1016/s0962-8924(00)01852-3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11121744/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Trends%20Cell%20Biol&title=Aggresomes,%20inclusion%20bodies%20and%20protein%20aggregation&author=RR%20Kopito&volume=10&publication_year=2000&pages=524-30&pmid=11121744&doi=10.1016/s0962-8924(00)01852-3&)

28. Seglen PO, Gordon PB. 3-Methyladenine: specific inhibitor of autophagic/lysosomal protein degradation in isolated rat hepatocytes. Proc Natl Acad Sci U S A. 1982;79:1889–92. doi: 10.1073/pnas.79.6.1889.  [DOI](https://doi.org/10.1073/pnas.79.6.1889) | [PMC free article](/articles/PMC346086/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/6952238/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&title=3-Methyladenine:%20specific%20inhibitor%20of%20autophagic/lysosomal%20protein%20degradation%20in%20isolated%20rat%20hepatocytes&author=PO%20Seglen&author=PB%20Gordon&volume=79&publication_year=1982&pages=1889-92&pmid=6952238&doi=10.1073/pnas.79.6.1889&)

29. Ohsumi Y. Molecular dissection of autophagy: two ubiquitin-like systems. Nat Rev Mol Cell Biol. 2001;2:211–6. doi: 10.1038/35056522.  [DOI](https://doi.org/10.1038/35056522) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11265251/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Mol%20Cell%20Biol&title=Molecular%20dissection%20of%20autophagy:%20two%20ubiquitin-like%20systems&author=Y%20Ohsumi&volume=2&publication_year=2001&pages=211-6&pmid=11265251&doi=10.1038/35056522&)

30. Scheuermann TH, Lolis E, Hodsdon ME. Tertiary structure of thiopurine methyltransferase from Pseudomonas syringae, a bacterial orthologue of a polymorphic, drug-metabolizing enzyme. J Mol Biol. 2003;333:573–85. doi: 10.1016/j.jmb.2003.08.039.  [DOI](https://doi.org/10.1016/j.jmb.2003.08.039) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14556746/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Mol%20Biol&title=Tertiary%20structure%20of%20thiopurine%20methyltransferase%20from%20Pseudomonas%20syringae,%20a%20bacterial%20orthologue%20of%20a%20polymorphic,%20drug-metabolizing%20enzyme&author=TH%20Scheuermann&author=E%20Lolis&author=ME%20Hodsdon&volume=333&publication_year=2003&pages=573-85&pmid=14556746&doi=10.1016/j.jmb.2003.08.039&)

31. Preuss CV, Wood TC, Szumlanski CL, Raftogianis RB, Otterness DM, Girard B, et al. Human histamine N-methyltransferase pharmacogenetics: common genetic polymorphisms that alter activity. Mol Pharmacol. 1998;53:708–17. doi: 10.1124/mol.53.4.708.  [DOI](https://doi.org/10.1124/mol.53.4.708) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9547362/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Pharmacol&title=Human%20histamine%20N-methyltransferase%20pharmacogenetics:%20common%20genetic%20polymorphisms%20that%20alter%20activity&author=CV%20Preuss&author=TC%20Wood&author=CL%20Szumlanski&author=RB%20Raftogianis&author=DM%20Otterness&volume=53&publication_year=1998&pages=708-17&pmid=9547362&doi=10.1124/mol.53.4.708&)

32. Shield AJ, Thomae BA, Eckloff BW, Wieben ED, Weinshilboum RM. Human catechol O-methyltransferase genetic variation: gene resequencing and functional characterization of variant allozymes. Mol Psychiatry. 2004;9:151–60. doi: 10.1038/sj.mp.4001386.  [DOI](https://doi.org/10.1038/sj.mp.4001386) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14966473/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Psychiatry&title=Human%20catechol%20O-methyltransferase%20genetic%20variation:%20gene%20resequencing%20and%20functional%20characterization%20of%20variant%20allozymes&author=AJ%20Shield&author=BA%20Thomae&author=BW%20Eckloff&author=ED%20Wieben&author=RM%20Weinshilboum&volume=9&publication_year=2004&pages=151-60&pmid=14966473&doi=10.1038/sj.mp.4001386&)

33. Freimuth RR, Eckloff B, Wieben ED, Weinshilboum RM. Human sulfotransferase SULT1C1 pharmacogenetics: gene resequencing and functional genomic studies. Pharmacogenetics. 2001;11:747–56. doi: 10.1097/00008571-200112000-00002.  [DOI](https://doi.org/10.1097/00008571-200112000-00002) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11740338/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Human%20sulfotransferase%20SULT1C1%20pharmacogenetics:%20gene%20resequencing%20and%20functional%20genomic%20studies&author=RR%20Freimuth&author=B%20Eckloff&author=ED%20Wieben&author=RM%20Weinshilboum&volume=11&publication_year=2001&pages=747-56&pmid=11740338&doi=10.1097/00008571-200112000-00002&)

34. Thomae BA, Eckloff BW, Freimuth RR, Wieben ED, Weinshilboum RM. Human sulfotransferase SULT2A1 pharmacogenetics: genotype-to-phenotype studies. Pharmacogenomics J. 2002;2:48–56. doi: 10.1038/sj.tpj.6500089.  [DOI](https://doi.org/10.1038/sj.tpj.6500089) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11990382/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Human%20sulfotransferase%20SULT2A1%20pharmacogenetics:%20genotype-to-phenotype%20studies&author=BA%20Thomae&author=BW%20Eckloff&author=RR%20Freimuth&author=ED%20Wieben&author=RM%20Weinshilboum&volume=2&publication_year=2002&pages=48-56&pmid=11990382&doi=10.1038/sj.tpj.6500089&)

35. Thomae BA, Rifki OF, Theobald MA, Eckloff BW, Wieben ED, Weinshilboum RM. Human catecholamine sulfotransferase (SULT1A3) pharmacogenetics: functional genetic polymorphism. J Neurochem. 2003;87:809–19. doi: 10.1046/j.1471-4159.2003.02027.x.  [DOI](https://doi.org/10.1046/j.1471-4159.2003.02027.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14622112/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Neurochem&title=Human%20catecholamine%20sulfotransferase%20(SULT1A3)%20pharmacogenetics:%20functional%20genetic%20polymorphism&author=BA%20Thomae&author=OF%20Rifki&author=MA%20Theobald&author=BW%20Eckloff&author=ED%20Wieben&volume=87&publication_year=2003&pages=809-19&pmid=14622112&doi=10.1046/j.1471-4159.2003.02027.x&)

36. Adjei AA, Thomae BA, Prondzinski JL, Eckloff BW, Wieben ED, Weinshilboum RM. Human estrogen sulfotransferase (SULT1E1) pharmacogenomics: gene resequencing and functional genomics. Br J Pharmacol. 2003;139:1373–82. doi: 10.1038/sj.bjp.0705369.  [DOI](https://doi.org/10.1038/sj.bjp.0705369) | [PMC free article](/articles/PMC1573968/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12922923/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Pharmacol&title=Human%20estrogen%20sulfotransferase%20(SULT1E1)%20pharmacogenomics:%20gene%20resequencing%20and%20functional%20genomics&author=AA%20Adjei&author=BA%20Thomae&author=JL%20Prondzinski&author=BW%20Eckloff&author=ED%20Wieben&volume=139&publication_year=2003&pages=1373-82&pmid=12922923&doi=10.1038/sj.bjp.0705369&)

37. Gaedigk A, Tyndale RF, Jurima-Romet M, Sellers EM, Grant DM, Leeder JS. NAD(P)H:quinone oxidoreductase: polymorphisms and allele frequencies in Caucasian, Chinese and Canadian Native Indian and Inuit populations. Pharmacogenetics. 1998;8:305–13. doi: 10.1097/00008571-199808000-00004.  [DOI](https://doi.org/10.1097/00008571-199808000-00004) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9731717/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=NAD(P)H:quinone%20oxidoreductase:%20polymorphisms%20and%20allele%20frequencies%20in%20Caucasian,%20Chinese%20and%20Canadian%20Native%20Indian%20and%20Inuit%20populations&author=A%20Gaedigk&author=RF%20Tyndale&author=M%20Jurima-Romet&author=EM%20Sellers&author=DM%20Grant&volume=8&publication_year=1998&pages=305-13&pmid=9731717&doi=10.1097/00008571-199808000-00004&)

38. US Department of Health and Human Services Food and Drug Administration. 2003  [Google Scholar](https://scholar.google.com/scholar_lookup?US%20Department%20of%20Health%20and%20Human%20Services%20Food%20and%20Drug%20Administration.%202003)

39. Honchel R, Aksoy IA, Szumlanski C, Wood TC, Otterness DM, Wieben ED, et al. Human thiopurine methyltransferase: molecular cloning and expression of T84 colon carcinoma cell cDNA. Mol Pharmacol. 1993;43:878–87.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/8316220/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Pharmacol&title=Human%20thiopurine%20methyltransferase:%20molecular%20cloning%20and%20expression%20of%20T84%20colon%20carcinoma%20cell%20cDNA&author=R%20Honchel&author=IA%20Aksoy&author=C%20Szumlanski&author=TC%20Wood&author=DM%20Otterness&volume=43&publication_year=1993&pages=878-87&pmid=8316220&)

40. Hildebrandt MA, Carrington DP, Thomae BA, Eckloff BW, Schaid DJ, Yee VC, et al. Genetic diversity and function in the human cytosolic sulfotransferases. Pharmacogenomics J. 2007;7:133–43. doi: 10.1038/sj.tpj.6500404.  [DOI](https://doi.org/10.1038/sj.tpj.6500404) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16801938/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Genetic%20diversity%20and%20function%20in%20the%20human%20cytosolic%20sulfotransferases&author=MA%20Hildebrandt&author=DP%20Carrington&author=BA%20Thomae&author=BW%20Eckloff&author=DJ%20Schaid&volume=7&publication_year=2007&pages=133-43&pmid=16801938&doi=10.1038/sj.tpj.6500404&)

41. DeLano WL. The PyMOL Molecular Graphics System. Palo Alto, CA, USA: DeLano Scientific; 2002.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=The%20PyMOL%20Molecular%20Graphics%20System&author=WL%20DeLano&publication_year=2002&)
