# Ethnic Stratification of the Association of RGS4 Variants with Antipsychotic Treatment Response in Schizophrenia

## Metadata
**Authors:** Daniel B Campbell, Philip J Ebert, Tara Skelly, T Scott Stroup, Jeffrey Lieberman, Pat Levitt, Patrick F Sullivan
**Journal:** Biological psychiatry
**Date:** 2007 Jun 22
**DOI:** [10.1016/j.biopsych.2007.04.018](https://doi.org/10.1016/j.biopsych.2007.04.018)
**PMID:** 17588543
**PMCID:** PMC2194758
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2194758/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC2194758/pdf/nihms36159.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC2194758/pdf/nihms36159.pdf)

## Abstract

**Background:** 
Genetic association studies, including a large meta-analysis, report association of Regulator of G Protein Signaling 4 (RGS4) with schizophrenia in the context of heterogeneity. The central role of RGS4 in regulating signaling via Gi/o coupled neurotransmitter receptors led us to hypothesize that there may be RGS4 genotypes predictive of specific disease phenotypes and antipsychotic treatment responses.

**Methods:** 
Subjects were 678 individuals with schizophrenia who participated in the Clinical Antipsychotic Trials of Intervention Effectiveness (CATIE). Among the 678 subjects, the inferred ancestries were: 198 (29%) ‘Africa only’, 397 (59%) ‘Europe only’ and 83 (12%) ‘Other’. Eight single nucleotide polymorphisms (SNPs) spanning RGS4 were genotyped. Multiple linear regression was used to analyze association of RGS4 markers with Positive and Negative Symptoms Scale (PANSS) scores at baseline and throughout antipsychotic treatment.

**Results:** 
Two consecutive markers within RGS4, rs2661319 and rs2842030, were associated with more severe baseline PANSS total score. Treatment with perphenazine was more effective than treatment with quetiapine (P=0.010) or ziprasidone (P=0.002) in individuals of inferred African ancestry and homozygous for the rs951439 ‘C’ allele.

**Conclusions:** 
RGS4 genotypes predicted both the severity of baseline symptoms and relative responsiveness to antipsychotic treatment. Although these analyses are exploratory and replication is required, these data provide support for RGS4 in schizophrenia pathogenesis and suggest a functional role for RGS4 in differential antipsychotic treatment efficacy of schizophrenia.

Keywords: Genetics, Candidate gene, Pharmacogenetics, G protein coupled receptors, Brain, Cerebral cortex

### Background

Genetic association studies, including a large meta-analysis, report association of Regulator of G Protein Signaling 4 (*RGS4*) with schizophrenia in the context of heterogeneity. The central role of RGS4 in regulating signaling via Gi/o coupled neurotransmitter receptors led us to hypothesize that there may be *RGS4* genotypes predictive of specific disease phenotypes and antipsychotic treatment responses.

### Methods

Subjects were 678 individuals with schizophrenia who participated in the Clinical Antipsychotic Trials of Intervention Effectiveness (CATIE). Among the 678 subjects, the inferred ancestries were: 198 (29%) ‘Africa only’, 397 (59%) ‘Europe only’ and 83 (12%) ‘Other’. Eight single nucleotide polymorphisms (SNPs) spanning *RGS4* were genotyped. Multiple linear regression was used to analyze association of *RGS4* markers with Positive and Negative Symptoms Scale (PANSS) scores at baseline and throughout antipsychotic treatment.

### Results

Two consecutive markers within *RGS4*, rs2661319 and rs2842030, were associated with more severe baseline PANSS total score. Treatment with perphenazine was more effective than treatment with quetiapine (P=0.010) or ziprasidone (P=0.002) in individuals of inferred African ancestry and homozygous for the rs951439 ‘C’ allele.

### Conclusions

*RGS4* genotypes predicted both the severity of baseline symptoms and relative responsiveness to antipsychotic treatment. Although these analyses are exploratory and replication is required, these data provide support for *RGS4* in schizophrenia pathogenesis and suggest a functional role for *RGS4* in differential antipsychotic treatment efficacy of schizophrenia.

## Introduction

Schizophrenia is a neurodevelopmental disorder with a substantial genetic component contributing to increased risk ([1](#R1)). Independent functional and genetic studies indicate that the regulator of G-protein signaling 4 (*RGS4*) gene is among a small group of promising disease vulnerability genes ([2](#R2), [3](#R3)).

The spatial expression pattern of *RGS4* in the adult brain matches closely the dopaminergic and muscarinic G-protein coupled receptor subtypes targeted by antipsychotic medications ([4](#R4)–[13](#R13)), suggesting a potential modulatory role for RGS4. The downstream effects of these receptors are regulated by the activity of RGS4 ([14](#R14)) and other regulators of G-protein signaling (RGS) proteins, which modulate G-protein coupled receptor signaling by accelerating the hydrolysis of GTP to GDP ([15](#R15)). The initial focus on *RGS4* came from gene microarray and in situ hybridization studies showing decreased levels of *RGS4* mRNA across cortical regions ([4](#R4)). Decreased levels of RGS4 protein were demonstrated subsequently in frontal cortex of patients with schizophrenia ([16](#R16)).

Moreover, the approximate genomic region containing *RGS4* was implicated in two linkage studies ([17](#R17), [18](#R18)) and ranked 13^th^ in a meta-analysis of schizophrenia genomewide linkage studies ([19](#R19)). The genetic association between *RGS4* and schizophrenia was detected initially in three different populations by Chowdari et al.([20](#R20)), and confirmed in five subsequent replication studies ([21](#R21)–[25](#R25)). However, there have been four reports of failures to replicate an association of *RGS4* and schizophrenia ([26](#R26)–[29](#R29)). Further, while the same single nucleotide polymorphisms (SNPs) in the 5′ region provide the strongest signal in association studies, the specific haplotypes have not been consistent across sample populations, which can be a hallmark of a false positive finding. However, a recent meta-analysis of more than 13,000 samples ([30](#R30)) evidenced additional support for the association of two common *RGS4* haplotypes with schizophrenia in the presence of etiological heterogeneity.

The well-known clinical heterogeneity of individuals with schizophrenia may be one particularly important source of the diversity of genetic associations for a single gene that is reported across multiple sample populations. Here, we performed a hypothesis-generating study to begin to dissect this heterogeneity by determining the association of specific alleles at the *RGS4* locus with clinical traits involved in the diagnosis of schizophrenia using the data generated by the CATIE trial ([31](#R31), [32](#R32)). Our analyses show that baseline Positive and Negative Symptoms Scale (PANSS) scores differed among *RGS4* marker genotypes. Further, our results indicate a significant interaction of *RGS4* genotype by treatment response that is specific to individuals of inferred African ancestry, suggesting a complex genetic influence in which the activity of RGS4 may predict the effectiveness of different pharmacologic treatments.

## Methods and Materials

### Subjects

The parent study has been described at length elsewhere ([31](#R31), [32](#R32)). Briefly, all subjects participated in CATIE (January 2001 to December 2004), a multi-phase randomized controlled trial of antipsychotic medications involving 1,460 persons with schizophrenia followed for up to 18 months. To maximize representativeness, subjects were ascertained from an array of clinical settings scattered across the United States. No subject was known to be related to any other subject. All subjects provided written informed consent (including an additional consent for genetic studies), and the full study protocol was reviewed by IRBs at the University of North Carolina and at participating study sites. Establishment of schizophrenia diagnosis, inclusion criteria and exclusion criteria have been described elsewhere ([33](#R33)).

### Phenotypes

Analyses were restricted to the length of time to discontinuation of treatment ([32](#R32)) and three measures derived from the Positive and Negative Symptoms Scale (PANSS), a broadly accepted measure for reliably ascertaining severity of schizophrenia symptoms ([34](#R34)). The time to discontinuation of treatment and change in PANSS score are the accepted endpoints of the CATIE trial (Lieberman et al 2005), and were selected to give the most accurate and reliable estimates of antipsychotic treatment response in schizophrenia. The PANSS is a 30-question evaluation of the core symptomatology of schizophrenia. Each question is scored on a scale of 1–7, with “1” indicating that the phenotype is absent and “7” indicating that the phenotype is extremely severe. Seven questions in the PANSS probe positive symptoms (e.g., delusions and hallucinations). The next 7 questions examine negative symptoms (e.g., blunted affect and emotional withdrawal). The final 16 questions concern general psychopathology (e.g., anxiety and depression). Responses to each question are categorical, but summations of the total score as well as the three subscales approach normal distributions. PANSS Total scores range from 30 to 210; scores for the PANSS positive and negative scales each range from 7 to 49.

The CATIE treatment protocol is described elsewhere ([31](#R31), [32](#R32)). Briefly, patients with schizophrenia were randomly assigned to one of five antipsychotic treatments in Phase 1. The five treatments in Phase 1 consisted of a first-generation antipsychotic, perphenazine, and several second-generation (atypical) antipsychotics, olanzapine, quetiaipine risperidone and ziprasidone. Randomization effectively matched age, sex, education, and baseline PANSS scores across the five antipsychotic treatment groups in Phase 1 ([32](#R32)). The analyses reported here are based on the 678 DNA samples available six months before the close of the CATIE trial.

The primary goal of the CATIE study was to compare the clinical effectiveness of a first-generation antipsychotic, perphenazine, to four newer antipsychotics ([32](#R32)). Therefore, our analyses focused on comparing the response to perphenazine versus each of the second-generation antipsychotics in phase 1 of CATIE. Thus all statistical comparisons are same-genotype perphenazine versus each of the second-generation antipsychotics. Two major measures were utilized to infer treatment response: time to discontinuation in Phase I and change in PANSS score from baseline as measured during Phase I. We recognize, however, that other variables of the study, such as a patient’s request to switch drug groups by discontinuing Phase I, may influence these domains in addition to response to the specific drug treatment.

High molecular weight DNA samples were obtained from all consenting subjects from cell lines established via EBV transformation at the Rutgers University Cell and DNA Repository ([http://www.rucdr.org/quality_control.html](http://www.rucdr.org/quality_control.html)). Sample DNA concentrations were quantified and normalized via the use of Picogreen dsDNA Quantitation Kits (Molecular Probes, Eugene, OR) ([33](#R33)).

### SNP Selection

SNP selection was based on ([20](#R20)) and the HapMap “CEU” Caucasian panel available in mid-2003 ([35](#R35)). Based on these data, we chose to genotype *RGS4* SNPs SNP4, SNP7 and SNP18 ([20](#R20)) (rs951436, rs951439, and rs2661319, respectively) plus an additional SNP in the same LD block (rs2842030). These four SNPs define a LD block including the putative *RGS4* promoter through intron 1, but do not cover the *RGS4* gene completely. To provide better coverage of the gene, we genotyped four additional SNPs (rs10759, rs2063142, rs2841959 and rs2841977).

### Comparison to previous reports of RGS4 association

We used the SNP genotype designations of the recent NCBI Genome Build 36.1 ([http://www.ncbi.nlm.nih.gov/SNP](http://www.ncbi.nlm.nih.gov/SNP)); all alleles are indicated in the (+) orientation of chromosome 1. Previous reports on *RGS4* association used the (−) orientation of chromosome 1, and thus identical alleles and haplotypes are referred to in the reverse complement. For example, we designate alleles at the rs951436 locus as A/C while Chowdari et al. ([20](#R20)) referred to the identical SNP (SNP4) as G/T. Thus, rs951436 AA = SNP4 GG, rs951436 CC = SNP4 TT, etc.

### Genotyping

Genotyping was performed with TaqMan 5′ exonuclease assays ([36](#R36)) as previously described ([33](#R33)). The overall no-call rate was 1.4%. Genotyping was conducted blind to all clinical data. Automated allele calls were made with SDS Data Collection software with all allele calls reviewed by an experienced operator according to protocol. In order for a SNP to be released for analysis, we required that: (a) 46 duplicated samples per SNP were in agreement, (b) call rates exceeded 95%, and (c) the clustering patterns were robust per review by an experienced technician.

Population stratification is a potential limitation of association studies ([37](#R37)) and is of particular concern in CATIE, given its explicit intent to ascertain a diverse, clinically representative sample ([31](#R31), [32](#R32)). There are two preconditions for population stratification to yield false positive results ([38](#R38), [39](#R39)). Because population stratification as a confounder exists only if the phenotypic distribution and the marker allele frequency differ across ancestral strata, we can determine the impact of stratification by measuring the marker and disease frequency and predict the extent of stratification. If there is substantial evidence that population substructure exists in the overall CATIE sample for the markers studied here, we default to stratified analysis. CATIE subjects were allowed to select multiple racial categories (White, Black/African-American, American Indian or Alaska native, Asian, Native Hawaiian or Pacific Islander, or Other) and were also asked if they were Hispanic or Latino. Inferred ancestry was defined as “Africa only” if a subject endorsed Black/African-American only, ”Europe only” if a subject endorsed White only, and “Other” if any other racial category was endorsed.

### Statistical Analysis

Data management and statistical analyses were performed with SAS version 9.1.3 (SAS Institute Inc). For baseline PANSS score analyses, multiple linear regressions were used to determine genotypic effect while controlling for ancestry, sex, and age. Analyses of time to discontinuation of treatment were performed using analysis of variance (ANOVA) followed by Bonferroni multiple comparisons test. Analyses of genotype X treatment interactions were performed across phase 1 using multiple linear regression with generalized estimating equations ([40](#R40)) to account for the dependence of measures on the same subject over time. The model included the covariates of baseline PANSS score, treatment, sex, and age. Haplotype associations were analyzed with haplo.score ([41](#R41)). HaploView (version 3.2) ([42](#R42)) was used to calculate Hardy-Weinberg Equilibrium (HWE) and |*D*′| and *r2* linkage disequilibrium estimates. LD blocks were defined by the Gabriel method ([43](#R43)) in HaploView.

In CATIE, the PANSS was administered at baseline and at multiple time points during antipsychotic treatment. All statistical analyses used multiple linear regression with GEE adjustment for non-independence of PANSS scores in the same individual over time to compare perphenazine with the second-generation antipsychotics. Significant differences represent altered PANSS score throughout the treatment period when comparing perphenazine response to olanzapine, quetiapine, risperidone and ziprasidone responses.

Because this is a hypothesis-generating study, we report uncorrected P values of global and post hoc multiple comparisons within each set of analyses. After all the hypothesis-generating analyses were complete, we performed exploratory corrections for multiple comparisons across all hypotheses using false discovery rates (FDR) ([44](#R44)). The FDR analyses included all 124 global comparisons: (8 SNPs * 2 ethnic groups * 3 baseline measures [PANSS total, positive and negative]) + (2 haplotypes * 2 ethnic groups * 3 baseline measures) + (8 SNPs * 2 ethnic groups * 4 treatment measures [PANSS total, positive and negative plus time to discontinuation]). Predictably, none of the uncorrected significant differences described here survived FDR correction for multiple comparisons.

### Project Context

*RGS4* was the third gene completed and analyzed in the CATIE sample, and all SNPs genotyped are reported. Additional investigations are underway (see [http://www.med.unc.edu/~pfsulliv/downloads.htm](http://www.med.unc.edu/~pfsulliv/downloads.htm)). We have attempted to follow published guidelines for association studies ([33](#R33), [37](#R37), [45](#R45), [46](#R46)).

## Results

### Sample Description

The sample for this report is 678 CATIE subjects who provided DNA samples. Of the 678 participants, the inferred ancestries were: 198 “Africa only” (29%), 397 “Europe only” (59%) and 83 “Other” (12%). Days to discontinuation of treatment during CATIE Phase 1 ranged from 6 to 611 days (mean ± standard deviation [sd] = 287 ± 199 days). The sample scores for the PANSS total ranged from 33 to 133 (mean ± sd = 73.5 ± 17.5) at baseline. The PANSS total scores during Phase 1 antipsychotic treatment ranged from 30 to 131 (mean ± sd = 68.8 ± 19.3). [Table 1](#T1) provides additional detail about the sample population.

### Table 1.

| Variable | n | n missing | mean | std | min | max |
| --- | --- | --- | --- | --- | --- | --- |
| Subject’s Age | 678 | 0 | 40.7 | 11.2 | 18 | 67 |
| Number of Years of Patient’s Education | 675 | 3 | 12.2 | 2.2 | 3 | 21 |
| Years Since First Prescribed an Antipsychotic Medi | 651 | 27 | 14.1 | 10.8 | 0 | 56 |
| Derived PANSS Total Score | 678 | 0 | 73.5 | 17.5 | 33 | 133 |
| Derived PANSS Positive Scale Score | 678 | 0 | 17.8 | 5.6 | 7 | 38 |
| Derived PANSS Negative Scale Score | 678 | 0 | 19.9 | 6.5 | 7 | 41 |
| PANSS General Psychopathology Scale | 678 | 0 | 35.9 | 9.0 | 16 | 69 |
| Ancestry | 198 | Africa only |   |   |   |   |
|   | 397 | Europe only |   |   |   |   |
|   | 83 | Other |   |   |   |   |
| Sex | 178 | Female |   |   |   |   |
|   | 500 | Male |   |   |   |   |

Table 1 Caption: Description of genotyped CATIE sample.

### Regional Map

[Figure 1a](#F1) depicts the genomic region of *RGS4* and the SNPs that were genotyped: rs951436, rs951439, rs2661319, rs2842030, rs10759, rs2063142, rs2841959 and rs2841977. The *RGS4* gene spans 7.2 kb of human chromosome 1q23.3.

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d64a/2194758/ef1c9bb85b61/nihms36159f1.jpg)

RGS4 genomic region, markers genotyped and linkage disequilibrium (LD) block structure. (a) RGS4 genomic structure. At the top are general genomic data along with an ideogram depicting chromosomal position. The topmost track shows RGS4 with the direction of transcription indicated by > (5′ → 3′), exons by vertical bars, and “U” indicates the presence of an untranslated region. The next track shows SNPs genotyped in the present study. (b) RGS4 LD block structure in the entire 678-individual sample. The genotyped SNPs define two LD blocks in the RGS4 gene: a 7-kb block including the promoter region and exon 1, and a 34-kb 3′ block. Pairwise D′ values (X 100) are indicated. (c) RGS4 LD block structure depicted with pairwise r2 values.

### Definition of Linkage Disequilibrium (LD) Blocks

The eight markers used to genotype the *RGS4* gene define two linkage disequilibrium (LD) blocks ([Figure 1 b,c](#F1)). The ~7-kb LD block 1 includes the putative promoter region, exon 1, and part of the first intron. The ~34-kb LD block 2 includes the 3′ untranslated region and downstream sequence. The LD block structure is similar in the African, European and Other ancestry groups ([Supplementary Table 1](#SD2)).

### RGS4 SNP Data and Ethnic Stratification

[Table 2](#T2) contains results for the SNPs genotyped along with reference data. Although no HWE tests were less than a significance level corrected for multiple comparisons, the allele frequencies for many of these SNPs were divergent in our sample as well as in reference data from dbSNP and HapMap. Six of the eight SNPs had no serious issues in regard to deviation from Hardy-Weinberg Equilibrium (HWE). There were no HWE problems with two SNPs (rs951436 and rs951439) and no serious problems for two other SNPs (rs2661319 and rs2842030). Two SNPs had HWE deviations when all subjects were considered together but had no meaningful departure when stratified by ancestry (rs2063142 and rs2841977); review of the genotype frequencies suggest that these SNPs have different genotype frequencies by ancestry so that HWE departures are expected if computed over all subjects.

### Table 2.

| SNP ID | Ancestry | n | Heterozyg | HWEProbExact | allele1 | Freq | Freq_95%CI_low | Freq_95%CI_high | allele2 | Freq | Freq_95%CI_low | Freq_95%CI_high | genotype1 | gfreq1 | genotype2 | gfreq2 | genotype3 | gfreq3 |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| rs951436 | _all_ | 657 | 0.46 | 0.102 | A | 0.58 | 0.56 | 0.61 | C | 0.42 | 0.39 | 0.44 | A/A | 0.35 | A/C | 0.46 | C/C | 0.19 |
| rs951436 | africa only | 194 | 0.35 | 0.564 | A | 0.76 | 0.71 | 0.80 | C | 0.24 | 0.20 | 0.29 | A/A | 0.58 | A/C | 0.35 | C/C | 0.07 |
| rs951436 | europe only | 380 | 0.52 | 0.470 | A | 0.52 | 0.49 | 0.56 | C | 0.48 | 0.44 | 0.51 | A/A | 0.26 | A/C | 0.52 | C/C | 0.22 |
| rs951436 | other | 83 | 0.41 | 0.121 | A | 0.46 | 0.38 | 0.54 | C | 0.54 | 0.46 | 0.62 | A/A | 0.25 | A/C | 0.41 | C/C | 0.34 |
|   |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
| rs951439 | _all_ | 662 | 0.49 | 1.000 | C | 0.57 | 0.55 | 0.60 | T | 0.43 | 0.40 | 0.45 | C/C | 0.33 | C/T | 0.49 | T/T | 0.18 |
| rs951439 | africa only | 196 | 0.48 | 0.774 | C | 0.54 | 0.49 | 0.59 | T | 0.46 | 0.41 | 0.51 | C/C | 0.30 | C/T | 0.48 | T/T | 0.21 |
| rs951439 | europe only | 386 | 0.49 | 1.000 | C | 0.57 | 0.54 | 0.61 | T | 0.43 | 0.39 | 0.46 | C/C | 0.33 | C/T | 0.49 | T/T | 0.18 |
| rs951439 | other | 80 | 0.49 | 0.622 | C | 0.66 | 0.58 | 0.73 | T | 0.34 | 0.27 | 0.42 | C/C | 0.41 | C/T | 0.49 | T/T | 0.10 |
|   |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
| rs2661319 | _all_ | 673 | 0.47 | 0.190 | C | 0.44 | 0.41 | 0.46 | T | 0.56 | 0.54 | 0.59 | C/C | 0.20 | C/T | 0.47 | T/T | 0.33 |
| rs2661319 | africa only | 195 | 0.36 | 0.573 | C | 0.25 | 0.21 | 0.30 | T | 0.75 | 0.70 | 0.79 | C/C | 0.07 | C/T | 0.36 | T/T | 0.56 |
| rs2661319 | europe only | 395 | 0.53 | 0.190 | C | 0.50 | 0.47 | 0.53 | T | 0.50 | 0.47 | 0.53 | C/C | 0.24 | C/T | 0.53 | T/T | 0.23 |
| rs2661319 | other | 83 | 0.39 | 0.045 | C | 0.54 | 0.46 | 0.62 | T | 0.46 | 0.38 | 0.54 | C/C | 0.35 | C/T | 0.39 | T/T | 0.27 |
|   |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
| rs2842030 | _all_ | 673 | 0.48 | 0.251 | G | 0.48 | 0.45 | 0.51 | T | 0.52 | 0.49 | 0.55 | G/G | 0.24 | G/T | 0.48 | T/T | 0.28 |
| rs2842030 | africa only | 197 | 0.40 | 0.019 | G | 0.59 | 0.54 | 0.64 | T | 0.41 | 0.36 | 0.46 | G/G | 0.39 | G/T | 0.40 | T/T | 0.21 |
| rs2842030 | europe only | 393 | 0.52 | 0.267 | G | 0.45 | 0.41 | 0.48 | T | 0.55 | 0.52 | 0.59 | G/G | 0.19 | G/T | 0.52 | T/T | 0.29 |
| rs2842030 | other | 83 | 0.43 | 0.486 | G | 0.37 | 0.30 | 0.45 | T | 0.63 | 0.55 | 0.70 | G/G | 0.16 | G/T | 0.43 | T/T | 0.41 |
|   |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
| rs10759 | _all_ | 672 | 0.35 | 0.003 | G | 0.73 | 0.70 | 0.75 | T | 0.27 | 0.25 | 0.30 | G/G | 0.55 | G/T | 0.35 | T/T | 0.10 |
| rs10759 | africa only | 197 | 0.31 | 0.470 | G | 0.82 | 0.79 | 0.86 | T | 0.18 | 0.14 | 0.21 | G/G | 0.67 | G/T | 0.31 | T/T | 0.02 |
| rs10759 | europe only | 392 | 0.36 | 0.003 | G | 0.69 | 0.66 | 0.73 | T | 0.31 | 0.27 | 0.34 | G/G | 0.51 | G/T | 0.36 | T/T | 0.13 |
| rs10759 | other | 83 | 0.39 | 0.317 | G | 0.67 | 0.60 | 0.75 | T | 0.33 | 0.25 | 0.40 | G/G | 0.48 | G/T | 0.39 | T/T | 0.13 |
|   |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
| rs2063142 | _all_ | 662 | 0.29 | 0.009 | A | 0.80 | 0.77 | 0.82 | G | 0.20 | 0.18 | 0.23 | A/A | 0.65 | A/G | 0.29 | G/G | 0.06 |
| rs2063142 | africa only | 191 | 0.18 | 0.254 | A | 0.89 | 0.86 | 0.92 | G | 0.11 | 0.08 | 0.14 | A/A | 0.80 | A/G | 0.18 | G/G | 0.02 |
| rs2063142 | europe only | 389 | 0.34 | 0.169 | A | 0.75 | 0.72 | 0.78 | G | 0.25 | 0.22 | 0.28 | A/A | 0.58 | A/G | 0.34 | G/G | 0.07 |
| rs2063142 | other | 82 | 0.29 | 0.206 | A | 0.78 | 0.71 | 0.84 | G | 0.22 | 0.16 | 0.29 | A/A | 0.63 | A/G | 0.29 | G/G | 0.07 |
|   |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
| rs2841959 | _all_ | 672 | 0.46 | 0.020 | C | 0.50 | 0.47 | 0.52 | T | 0.50 | 0.48 | 0.53 | C/C | 0.27 | C/T | 0.46 | T/T | 0.28 |
| rs2841959 | africa only | 196 | 0.54 | 0.384 | C | 0.50 | 0.45 | 0.54 | T | 0.50 | 0.46 | 0.55 | C/C | 0.23 | C/T | 0.54 | T/T | 0.23 |
| rs2841959 | europe only | 394 | 0.42 | 0.002 | C | 0.49 | 0.46 | 0.53 | T | 0.51 | 0.47 | 0.54 | C/C | 0.28 | C/T | 0.42 | T/T | 0.30 |
| rs2841959 | other | 82 | 0.43 | 0.194 | C | 0.51 | 0.42 | 0.59 | T | 0.49 | 0.41 | 0.58 | C/C | 0.29 | C/T | 0.43 | T/T | 0.28 |
|   |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
| rs2841977 | _all_ | 657 | 0.34 | 0.003 | C | 0.26 | 0.24 | 0.29 | T | 0.74 | 0.71 | 0.76 | C/C | 0.09 | C/T | 0.34 | T/T | 0.56 |
| rs2841977 | africa only | 193 | 0.20 | 0.730 | C | 0.12 | 0.09 | 0.15 | T | 0.88 | 0.85 | 0.91 | C/C | 0.02 | C/T | 0.20 | T/T | 0.78 |
| rs2841977 | europe only | 383 | 0.41 | 0.138 | C | 0.34 | 0.31 | 0.38 | T | 0.66 | 0.62 | 0.69 | C/C | 0.13 | C/T | 0.41 | T/T | 0.45 |
| rs2841977 | other | 81 | 0.35 | 0.390 | C | 0.26 | 0.19 | 0.33 | T | 0.74 | 0.67 | 0.81 | C/C | 0.09 | C/T | 0.35 | T/T | 0.57 |

Table 2 Caption: Ethnic stratification of genotyped RGS4 markers.

Finally, rs10759 and rs2841959 showed HWE deviations in subjects with European ancestries. We compared rs10759 to the HapMap CEU data and found that the minor allele frequencies (MAFs) were similar (0.31 versus 0.28) as were the observed heterozygosities (0.36 versus 0.43) and that this SNP is in a highly conserved recombination hotspot. rs2841959 again has similar MAF (0.49 versus 0.46) in comparison to HapMap CEU and is in a region with some evidence of selection in Europeans. Overall, there are multiple reasons for HWE deviation, including technical failures and population genetic phenomena ([47](#R47)), and these possibilities should be kept in mind when interpreting the findings from these two SNP.

### RGS4 genotype association with Baseline PANSS Scores

Analyses of baseline PANSS scores following ethnic stratification using sex and age as covariates indicated association of rs2842030 specifically in patients of European ancestry (P=0.021; [Table 3](#T3)). Similar baseline PANSS score trends across all ancestries prompted exploratory analyses of the combined sample using ancestry, age and sex as co-variates. A significant difference among genotypes was found at two consecutive LD block 1 markers, rs2661319 (P=0.049) and rs2842030 (P=0.015; [Table 3](#T3)).

### Table 3.

| SNP ID | Ancestry | n | DF | ProbF | LSMean1 (95% CI) | LSMean2 (95% CI) | LSMean3 (95% CI) |
| --- | --- | --- | --- | --- | --- | --- | --- |
| rs951436 | Combined | 657 | 2 | 0.1151 | A/A 75.406 (72.808, 78.003) | A/C 75.299 (73.005, 77.593) | C/C 71.629 (68.340, 74.919) |
| rs951436 | Africa only | 194 | 2 | 0.7090 | A/A 74.682 (71.419, 77.945) | A/C 74.723 (70.742, 78.705) | C/C 70.766 (61.648, 79.883) |
| rs951436 | Europe only | 380 | 2 | 0.2725 | A/A 71.318 (67.549, 75.087) | A/C 72.161 (69.503, 74.820) | C/C 68.451 (64.513, 72.390) |
| rs951436 | Other ancestry | 83 | 2 | 0.4979 | A/A 83.506 (75.203, 91.809) | A/C 78.625 (72.190, 85.060) | C/C 77.398 (69.948, 84.849) |
| rs951439 | Combined | 662 | 2 | 0.6154 | C/C 73.304 (70.749, 75.860) | C/T 74.701 (72.490, 76.913) | T/T 74.807 (71.407, 78.207) |
| rs951439 | Africa only | 196 | 2 | 0.9397 | C/C 74.598 (70.234, 78.962) | C/T 74.348 (70.828, 77.869) | T/T 73.456 (68.340, 78.573) |
| rs951439 | Europe only | 386 | 2 | 0.6015 | C/C 69.856 (66.563, 73.149) | C/T 71.693 (68.964, 74.421) | T/T 72.022 (67.651, 76.394) |
| rs951439 | Other ancestry | 80 | 2 | 0.8398 | C/C 77.053 (70.404, 83.701) | C/T 79.335 (73.341, 85.330) | T/T 80.114 (67.088, 93.141) |
| rs2661319 | Combined | 673 | 2 | 0.0489 | C/C 71.477 (68.321, 74.633) | C/T 75.932 (73.661, 78.203) | T/T 74.788 (72.170, 77.406) |
| rs2661319 | Africa only | 195 | 2 | 0.8292 | C/C 72.006 (63.196, 80.816) | C/T 74.287 (70.375, 78.198) | T/T 74.841 (71.566, 78.115) |
| rs2661319 | Europe only | 395 | 2 | 0.0540 | C/C 67.923 (64.216, 71.629) | C/T 72.972 (70.397, 75.548) | T/T 69.860 (65.965, 73.755) |
| rs2661319 | Other ancestry | 83 | 2 | 0.6972 | C/C 77.293 (70.009, 84.578) | C/T 79.772 (73.106, 86.438) | T/T 81.737 (73.555, 89.919) |
| rs2842030 | Combined | 673 | 2 | 0.0145 | G/G 75.171 (72.203, 78.140) | G/T 76.243 (74.028, 78.458) | T/T 71.628 (68.946, 74.310) |
| rs2842030 | Africa only | 197 | 2 | 0.4321 | G/G 75.823 (71.931, 79.715) | G/T 74.563 (70.825, 78.302) | T/T 71.612 (66.395, 76.830) |
| rs2842030 | Europe only | 393 | 2 | 0.0212 | G/G 69.857 (65.622, 74.091) | G/T 73.403 (70.831, 75.974) | T/T 67.929 (64.566, 71.293) |
| rs2842030 | Other ancestry | 83 | 2 | 0.8098 | G/G 81.140 (70.517, 91.763) | G/T 80.305 (73.975, 86.636) | T/T 77.862 (71.124, 84.599) |
| rs10759 | Combined | 672 | 2 | 0.4452 | G/G 73.836 (71.730, 75.942) | G/T 75.690 (73.185, 78.195) | T/T 74.359 (69.910, 78.809) |
| rs10759 | Africa only | 197 | 2 | 0.9714 | G/G 74.177 (71.188, 77.166) | G/T 74.339 (70.010, 78.668) | T/T 76.174 (59.875, 92.473) |
| rs10759 | Europe only | 392 | 2 | 0.2294 | G/G 69.756 (67.087, 72.424) | G/T 73.064 (69.948, 76.181) | T/T 71.287 (66.283, 76.291) |
| rs10759 | Other ancestry | 83 | 2 | 0.7758 | G/G 80.394 (74.085, 86.704) | G/T 79.784 (72.992, 86.577) | T/T 75.801 (64.487, 87.115) |
| rs2063142 | Combined | 662 | 2 | 0.5145 | A/A 74.048 (72.059, 76.037) | A/G 75.786 (73.019, 78.552) | G/G 75.322 (69.677, 80.966) |
| rs2063142 | Africa only | 191 | 2 | 0.2695 | A/A 73.576 (70.752, 76.400) | A/G 78.611 (72.952, 84.270) | G/G 72.476 (56.186, 88.765) |
| rs2063142 | Europe only | 389 | 2 | 0.7472 | A/A 70.418 (67.862, 72.974) | A/G 71.807 (68.640, 74.974) | G/G 70.151 (63.611, 76.690) |
| rs2063142 | Other ancestry | 82 | 2 | 0.6403 | A/A 79.851 (74.225, 85.477) | A/G 77.552 (69.868, 85.236) | G/G 85.532 (70.337, 100.73) |
| rs2841959 | Combined | 672 | 2 | 0.7620 | C/C 75.151 (72.364, 77.939) | C/T 74.078 (71.815, 76.340) | T/T 73.966 (71.201, 76.731) |
| rs2841959 | Africa only | 196 | 2 | 0.4793 | C/C 76.089 (71.174, 81.004) | C/T 72.946 (69.576, 76.317) | T/T 75.528 (70.574, 80.483) |
| rs2841959 | Europe only | 394 | 2 | 0.2720 | C/C 71.176 (67.665, 74.687) | C/T 72.599 (69.719, 75.478) | T/T 69.153 (65.756, 72.550) |
| rs2841959 | Other ancestry | 82 | 2 | 0.4607 | C/C 80.096 (72.635, 87.558) | C/T 75.465 (68.696, 82.234) | T/T 81.257 (73.312, 89.202) |
| rs2841977 | Combined | 657 | 2 | 0.6277 | C/C 73.088 (68.340, 77.835) | C/T 75.200 (72.567, 77.834) | T/T 74.065 (71.986, 76.145) |
| rs2841977 | Africa only | 193 | 2 | 0.6674 | C/C 81.096 (62.228, 99.965) | C/T 75.336 (70.014, 80.657) | T/T 73.766 (70.945, 76.587) |
| rs2841977 | Europe only | 383 | 2 | 0.5941 | C/C 69.274 (64.209, 74.338) | C/T 72.140 (69.203, 75.077) | T/T 71.089 (68.261, 73.917) |
| rs2841977 | Other ancestry | 81 | 2 | 0.9408 | C/C 77.067 (62.381, 91.753) | C/T 79.729 (72.253, 87.204) | T/T 78.726 (72.957, 84.495) |

Table 3 Caption: RGS4 genotype association with baseline PANSS total scores

### Haplotype association of baseline PANSS scores

Permutation-based analyses of haplotype-based baseline PANSS scores failed to show a global significant difference in strata defined by African (P=0.091) and European (P=0.065) reported ancestry. However, in an exploratory fashion, we noted that a LD block 1 haplotype – rs951436 allele A, rs951439 allele C, rs2661319 allele G and rs2842030 allele T (A-C-T-G) – showed a statistically significant PANSS total score in subjects of both African (P=0.048) and European ancestry (P=0.017) ([Supplementary Table 2](#SD2)). None of the baseline associations survive FDR correction for multiple comparisons of the 124 hypotheses tested concerning both baseline and treatment response association.

#### Treatment Response

Using GEE models to include all available longitudinal data, SNP by treatment analyses for PANSS total indicated a significant difference only among the group of African ancestry and *RGS4* marker rs951439 (P=0.024; [Supplementary Table 3](#SD2)). Exploratory analyses using PANSS positive and negative scores as the dependent variable revealed that both PANSS positive (P=0.037) and negative (P=0.049) responses contributed to the significant rs951439 PANSS total response difference ([Supplementary Table 3](#SD2)). In addition, the exploratory analyses of PANSS positive and negative response scores indicated association of the the rs2842030 (P=0.023) and rs10759 (P=0.025) markers specifically in PANSS positive response among patients of African ancestry ([Supplementary Table 3](#SD2)). The rs951439 and rs2842030 markers were sufficiently polymorphic in both the African and European ancestry samples to result in sample cell numbers ranging from 5 to 54. [Figures 2](#F2) and [3](#F3) summarize the *RGS4* marker by treatment response data in the Africa only and Europe only strata. [Supplementary Table 4](#SD2) presents additional *RGS4* marker data that are based on some sample cell numbers <5 but may represent important treatment response differences.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d64a/2194758/698ac3707704/nihms36159f2.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=2194758_nihms36159f2.jpg)

Association of RGS4 marker rs951439 genotype with antipsychotic treatment response. Among the 198 patients of African ancestry, rs951439 genotype CC continue on perphenazine treatment significantly longer than ziprasidone treatment (a) and have significantly lower PANSS scores (c, e, g) during treatment with perphenazine compared to ziprasidone. Patients of rs951439 CC genotype and African ancestry in the perphenazine group exhibits differences compared to patients in the quetiapine group when the PANSS total and PANSS negative scores are used as dependent variables (c, g). Among the 397 patients of European ancestry, no significant global differences between perphenazine and second-generation antipsychotics were observed (b, d, f, h). However, patients with rs951439 TT genotype in the risperidone group appeared to remain on risperidone longer (b) and exhibited less severe positive symptoms during risperidone treatment (f) than genotype CC. Each bar represents response data from 5–48 patients. Detailed analyses are presented in Supplementary Tables 3–7. * indicates significant difference (P<0.05) for both global and pairwise comparisons with perphenazine by multiple linear regression with GEE.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d64a/2194758/504d34eea823/nihms36159f3.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=2194758_nihms36159f3.jpg)

Association of RGS4 marker rs2842030 genotype with apparent antipsychotic treatment response. Among the 198 individuals of African ancestry, the rs2842030 genotype was significantly correlated with PANSS positive scores: genotype TT in the perphenazine group had better scores compared to quetiapine, risperidone and ziprasidone groups; there was no difference with between perphenazine and olanzapine (e). Trends similar to those observed for rs951439 (Figure 2) are evident in the African ancestry response associations for rs2842030 (a, c, g); however, only the associations using PANSS positive scores reached global significance. Among the 397 patients of European ancestry, no significant global differences between perphenazine and second-generation antipsychotics were observed (b, d, f, h). However, the group of individuals with the rs2842030 GG genotype on risperidone remained on treatment longer (b) and exhibit less severe positive symptoms during risperidone treatment (f) than genotype TT. Each bar represents response data from 5–54 patients. Detailed analyses are presented in Supplementary Tables 3–7. * indicates significant difference (P<0.05) for both global and pairwise comparisons with perphenazine by multiple linear regression with GEE.

### RGS4 genotype association with Phase 1 Time to Discontinuation of Treatment

When stratified by inferred ancestry, there was a significant rs951439 genotype by treatment interaction in subjects of African ancestry (P=0.013) but not in subjects of European ancestry only (P = 0.429). Patients of African ancestry with rs951439 genotype CC continued on perphenazine (391 ± 63 days) significantly longer than ziprasidone (124 ± 46 days) ([Figure 2a](#F2)). No other significant differences between perphenazine and the second-generation antipsychotics were present after stratification by *RGS4* genotype ([Supplementary Table 4](#SD2)).

### RGS4 genotype association with Phase 1 Drug Treatment: PANSS Total Scores

Patients of African ancestry and rs951439 genotype CC randomized to perphenazine treatment demonstrated significantly greater improvement than patients treated with quetiapine or risperidone. The average PANSS total score during perphenazine treatment (59.11 ± 3.45) was 29% better on average in comparison to ziprasidone treatment (76.48 ± 4.47; P=0.002) and 19% compared to quetiapine treatment (70.17 ± 2.41; P=0.010) ([Figure 2c](#F2)). Patients of African ancestry and rs951439 genotype CT responded significantly worse to perphenazine (70.61 ± 2.42) than olanzapine treatment (63.51 ± 1.55; P=0.007). There were no significant differences between perphenazine and second-generation antipsychotics among rs951439 genotypes in patients of European ancestry ([Supplementary Table 5](#SD2)). The associations with treatment response do not survive FDR correction for multiple comparisons of the 124 hypotheses tested here on baseline and treatment responses.

### RGS4 genotype association with Phase 1 Drug Treatment: PANSS Positive and Negative Scores

In a set of exploratory analyses, given the significant findings for PANSS total, we next analyzed the PANSS positive and negative symptoms using the same analytical strategies. Among patients of African ancestry and rs951439 CC genotype, the response to perphenazine (13.74 ± 1.44) was significantly more positive than the response to ziprasidone (18.76 ± 1.21; P=0.009) ([Figure 2e](#F2); [Supplementary Table 6](#SD2)). Among patients of African descent with the rs2842030 TT genotype, response to perphenazine was indistinguishable from response to olanzapine but significantly better than responses to quetiapine, risperidone and ziprasidone. The average PANSS positive score was decreased by more than 30% during perphenazine treatment (11.60 ± 1.64) and olanzapine treatment (11.60 ± 0.65) compared to treatments with quetiapine (16.53 ± 0.98; P=0.011), risperidone (16.70 ± 0.78; P=0.006) and ziprasidone (18.57 ± 0.52; P<0.001) ([Figure 3e](#F3)). Patients with rs2842030 genotype TT also appeared to respond more positively to both perphenazine and olanzapine than those of rs2842030 genotype GG or GT ([Figure 3e](#F3); [Supplementary Table 6](#SD2)). For PANSS negative scores, patients of African ancestry and rs951439 genotype CC had significantly greater improvement during treatment with perphenazine (17.27 ± 1.00) than quetiapine (20.39 ± 0.96; P=0.024) or ziprasidone (21.45 ± 0.88; P=0.002). Patients of African ancestry and rs951439 genotype CT responded with significantly higher PANSS negative scores during treatment with perphenazine (20.74 ± 0.76) than olanzapine (18.33 ± 0.78; P=0.017) or quetiapine (17.75 ± 0.90; P=0.010) ([Figure 2g](#F2); [Supplementary Table 7](#SD2)).

## Discussion

The present study revealed an association of genetic variants in the *RGS4* gene with alterations in the baseline PANSS total score and with differential responsiveness to antipsychotic treatment medications. The variants in the 5′ region of *RGS4* have not been analyzed yet for regulatory functions, and additional experiments will be required to identify the functional variants marked by association of the genotyped SNPs. However, the difference in baseline PANSS total score among *RGS4* markers in the 5′ region of the gene may be consistent with the hypothesis that differential regulation of *RGS4* contributes to the severity of schizophrenia symptoms ([30](#R30)). The different treatment responses among *RGS4* genotypes, based on changes in PANSS scores and time to discontinuation of treatment, is consistent with our working hypothesis that (a) the pharmacologic actions of antipsychotic treatments could be influenced by *RGS4* expression levels; and (b) certain *RGS4* genotypes may be useful for predicting the efficacy of a particular treatment regimen in specific, ethnically defined patient populations.

We caution that these are exploratory analyses and all results require confirmation. Nonetheless, the present findings, in combination with other data, provide additional evidence that *RGS4* variants correlate with certain phenotypic characteristics of individuals with schizophrenia. For example, reduction in the volume of the dorsolateral prefrontal cortex (DLPFC, a brain region implicated in schizophrenia) has been shown in first episode patients. Recently, Prasad et al ([48](#R48)) reported that this reduction correlates with *RGS4* SNP18 genotype GG. *RGS4* SNP18 is the reverse complement of rs2661319, for which we describe an association of the TT genotype (equivalent to the SNP18 GG genotype) with higher baseline PANSS Total scores. Taken together, Prasad et al ([48](#R48)) and the present study suggest that individuals with the rs2661319 TT genotype have quantifiably more severe disease symptoms, characterized by both PANSS total score and a neuroanatomical deficit in DLPFC.

There is a distinct ancestral difference in the correlations between *RGS4* genotype and treatment responses. In the sample of African ancestry patients, the rs951439 CC genotype correlated with: (a) longer time to discontinuation for those individuals in the perphenazine group compared to the quetiapine or ziprasidone groups; (b) decreased PANSS total and PANSS negative symptoms scores for individuals in the perphenazine treatment group compared to the quetiapine or ziprasidone treatment groups; and (c) decreased PANSS positive symptoms score for the individuals in the perphenazine treatment group compared to those patients in the ziprasidone treatment group ([Figure 2](#F2)). Nearly identical results were observed for individuals of African descent and rs2842030 genotype TT ([Figure 3](#F3)). In contrast, patients of European descent and in the risperidone group differed by *RGS4* genotype. Among European ancestry patients, rs951439 genotype CC or rs2842030 genotype TT correlated with: (a) decreased time to discontinuation of treatment compared to the opposite homozygous genotype; (b) a trend toward increased PANSS total scores; and (c) increased PANSS positive scores compared to same-genotype individuals treated with perphenazine ([Figures 2](#F2) and [3](#F3)). Particularly striking are the apparent differential responses to ziprasidone. Individuals of African ancestry and rs951439 CC genotype appeared to be poorer responders to ziprasidone compared to other rs951439 genotypes, as evidenced by decreased time to discontinuation and higher PANSS scores during treatment ([Figure 2](#F2)). In contrast, these features that characterize the group of patients on ziprasidone were indistinguishable among rs951439 genotypes in the European ancestry strata ([Figure 3](#F3)). A summary of the characteristics used to infer treatment responses stratified by ethnic descent and *RGS4* genotype is provided in [Table 4](#T4). Perphenazine and olanzapine appeared to be particularly effective in individuals of African ancestry and either rs951439 CC genotype or rs2842030 TT genotype. Among patients of European ancestry, risperidone was effective in those with rs951439 genotype TT or rs2842030 genotype GG but less effective in individuals of rs951439 genotype CC or rs2842030 genotype TT ([Table 4](#T4)).

### Table 4.

|   | Africa only |  | Europe only |  |
| --- | --- | --- | --- | --- |
| rs951439 | CC | TT | CC | TT |
| Perphenazine | + | 0 | 0 | 0 |
| Olanzapine | + | 0 | 0 | 0 |
| Quetiapine | − | + | 0 | 0 |
| Risperidone | + | 0 | − | + |
| Ziprasidone | − | + | 0 | 0 |
| rs2842030 | TT | GG | TT | GG |
| Perphenazine | + | 0 | 0 | 0 |
| Olanzapine | + | 0 | 0 | 0 |
| Quetiapine | − | 0 | 0 | 0 |
| Risperidone | − | 0 | − | + |
| Ziprasidone | − | + | 0 | 0 |

Table 4 Caption: Summary of ethnic stratification of RGS4-genotype dependent antipsychotic treatment effectiveness, defined by PANSS scores and time remaining on a specific drug treatment.

Antipsychotic medications act to modulate G-protein coupled receptors (GPCRs) stimulated by dopamine, acetylcholine and serotonin. RGS4 shortens the duration of signaling via GPCRs by acting as a GTPase activating protein (GAP), thereby accelerating the deactivation of the heterotrimeric G protein following receptor activation. Therefore, alterations in RGS4 availability or function could alter the effectiveness of antipsychotic medications. The association data presented here suggest that RGS4 plays an important and differential role in modulating the transduction of signals from GPCRs targeted by antipsychotic medications.

These results also emphasize the importance of including multiple ethnic groups in a study design and the importance of collecting substantially larger samples of under-represented ethnic groups than typically done in such studies. The present study of 678 individuals including 198 patients of African descent provides initial evidence that there may be more than one functional variant in *RGS4* that contributes to schizophrenia disease etiology and to characteristic phenotypes that may reflect in part antipsychotic treatment response. The data, while recognizing that these were obtained from a relatively small sample size, suggests further exploration to determine whether the variants derive from different ancestries. As with all association studies, replication of the findings in this first report is essential ([49](#R49)). However, together with numerous association analyses, these results provide additional support for the hypothesis that *RGS4* is part of a complex biological mechanism that underlies schizophrenia pathophysiology and continues to influence treatment outcomes throughout life by affecting antipsychotic treatments.

## Supplementary Material

## Acknowledgments

The CATIE project was supported by NIMH contract NO1 MH90001. The project reported here was funded by the Vanderbilt Kennedy Center P30 HD15052 (PL), and NIMH Conte Center MH45156 (D. Lewis, PI), Project 4 (PL, PI). PFS was supported by R01 MH-074027.

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Sullivan PF, Kendler KS, Neale MC. Schizophrenia as a complex trait: evidence from a meta-analysis of twin studies. Arch Gen Psychiatry. 2003;60:1187–1192. doi: 10.1001/archpsyc.60.12.1187.  [DOI](https://doi.org/10.1001/archpsyc.60.12.1187) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14662550/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arch%20Gen%20Psychiatry&title=Schizophrenia%20as%20a%20complex%20trait:%20evidence%20from%20a%20meta-analysis%20of%20twin%20studies&author=PF%20Sullivan&author=KS%20Kendler&author=MC%20Neale&volume=60&publication_year=2003&pages=1187-1192&pmid=14662550&doi=10.1001/archpsyc.60.12.1187&)

2. Harrison PJ, Weinberger DR. Schizophrenia genes, gene expression, and neuropathology: on the matter of their convergence. Mol Psychiatry. 2005;10:40–68. doi: 10.1038/sj.mp.4001558.  [DOI](https://doi.org/10.1038/sj.mp.4001558) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15263907/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Psychiatry&title=Schizophrenia%20genes,%20gene%20expression,%20and%20neuropathology:%20on%20the%20matter%20of%20their%20convergence&author=PJ%20Harrison&author=DR%20Weinberger&volume=10&publication_year=2005&pages=40-68&pmid=15263907&doi=10.1038/sj.mp.4001558&)

3. Norton N, Williams HJ, Owen MJ. An update on the genetics of schizophrenia. Curr Opin Psychiatry. 2006;19:158–164. doi: 10.1097/01.yco.0000214341.52249.59.  [DOI](https://doi.org/10.1097/01.yco.0000214341.52249.59) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16612196/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Opin%20Psychiatry&title=An%20update%20on%20the%20genetics%20of%20schizophrenia&author=N%20Norton&author=HJ%20Williams&author=MJ%20Owen&volume=19&publication_year=2006&pages=158-164&pmid=16612196&doi=10.1097/01.yco.0000214341.52249.59&)

4. Mirnics K, Middleton FA, Stanwood GD, Lewis DA, Levitt P. Disease-specific changes in regulator of G-protein signaling 4 (RGS4) expression in schizophrenia. Mol Psychiatry. 2001;6:293–301. doi: 10.1038/sj.mp.4000866.  [DOI](https://doi.org/10.1038/sj.mp.4000866) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11326297/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Psychiatry&title=Disease-specific%20changes%20in%20regulator%20of%20G-protein%20signaling%204%20(RGS4)%20expression%20in%20schizophrenia&author=K%20Mirnics&author=FA%20Middleton&author=GD%20Stanwood&author=DA%20Lewis&author=P%20Levitt&volume=6&publication_year=2001&pages=293-301&pmid=11326297&doi=10.1038/sj.mp.4000866&)

5. Erdely HA, Lahti RA, Lopez MB, Myers CS, Roberts RC, Tamminga CA, et al. Regional expression of RGS4 mRNA in human brain. Eur J Neurosci. 2004;19:3125–3128. doi: 10.1111/j.0953-816X.2004.03364.x.  [DOI](https://doi.org/10.1111/j.0953-816X.2004.03364.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15182322/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Neurosci&title=Regional%20expression%20of%20RGS4%20mRNA%20in%20human%20brain&author=HA%20Erdely&author=RA%20Lahti&author=MB%20Lopez&author=CS%20Myers&author=RC%20Roberts&volume=19&publication_year=2004&pages=3125-3128&pmid=15182322&doi=10.1111/j.0953-816X.2004.03364.x&)

6. Mash DC, White WF, Mesulam MM. Distribution of muscarinic receptor subtypes within architectonic subregions of the primate cerebral cortex. J Comp Neurol. 1988;278:265–274. doi: 10.1002/cne.902780209.  [DOI](https://doi.org/10.1002/cne.902780209) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/3230164/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Comp%20Neurol&title=Distribution%20of%20muscarinic%20receptor%20subtypes%20within%20architectonic%20subregions%20of%20the%20primate%20cerebral%20cortex&author=DC%20Mash&author=WF%20White&author=MM%20Mesulam&volume=278&publication_year=1988&pages=265-274&pmid=3230164&doi=10.1002/cne.902780209&)

7. Piggott M, Owens J, O’Brien J, Paling S, Wyper D, Fenwick J, et al. Comparative distribution of binding of the muscarinic receptor ligands pirenzepine, AF-DX 384, (R,R)-I-QNB and (R,S)-I-QNB to human brain. J Chem Neuroanat. 2002;24:211–223. doi: 10.1016/s0891-0618(02)00066-2.  [DOI](https://doi.org/10.1016/s0891-0618(02)00066-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12297267/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Chem%20Neuroanat&title=Comparative%20distribution%20of%20binding%20of%20the%20muscarinic%20receptor%20ligands%20pirenzepine,%20AF-DX%20384,%20(R,R)-I-QNB%20and%20(R,S)-I-QNB%20to%20human%20brain&author=M%20Piggott&author=J%20Owens&author=J%20O%E2%80%99Brien&author=S%20Paling&author=D%20Wyper&volume=24&publication_year=2002&pages=211-223&pmid=12297267&doi=10.1016/s0891-0618(02)00066-2&)

8. Liskowsky DR, Potter LT. D-2 dopamine receptors in the frontal cortex of rat and human. Life Sci. 1985;36:1551–1559. doi: 10.1016/0024-3205(85)90379-0.  [DOI](https://doi.org/10.1016/0024-3205(85)90379-0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/3982225/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Life%20Sci&title=D-2%20dopamine%20receptors%20in%20the%20frontal%20cortex%20of%20rat%20and%20human&author=DR%20Liskowsky&author=LT%20Potter&volume=36&publication_year=1985&pages=1551-1559&pmid=3982225&doi=10.1016/0024-3205(85)90379-0&)

9. Lidow MS, Goldman-Rakic PS, Rakic P, Innis RB. Dopamine D2 receptors in the cerebral cortex: distribution and pharmacological characterization with [3H]raclopride. Proc Natl Acad Sci U S A. 1989;86:6412–6416. doi: 10.1073/pnas.86.16.6412.  [DOI](https://doi.org/10.1073/pnas.86.16.6412) | [PMC free article](/articles/PMC297850/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2548214/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&title=Dopamine%20D2%20receptors%20in%20the%20cerebral%20cortex:%20distribution%20and%20pharmacological%20characterization%20with%20%5B3H%5Draclopride&author=MS%20Lidow&author=PS%20Goldman-Rakic&author=P%20Rakic&author=RB%20Innis&volume=86&publication_year=1989&pages=6412-6416&pmid=2548214&doi=10.1073/pnas.86.16.6412&)

10. Lidow MS, Goldman-Rakic PS, Gallager DW, Rakic P. Distribution of dopaminergic receptors in the primate cerebral cortex: quantitative autoradiographic analysis using [3H]raclopride, [3H]spiperone and [3H]SCH23390. Neuroscience. 1991;40:657–671. doi: 10.1016/0306-4522(91)90003-7.  [DOI](https://doi.org/10.1016/0306-4522(91)90003-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2062437/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neuroscience&title=Distribution%20of%20dopaminergic%20receptors%20in%20the%20primate%20cerebral%20cortex:%20quantitative%20autoradiographic%20analysis%20using%20%5B3H%5Draclopride,%20%5B3H%5Dspiperone%20and%20%5B3H%5DSCH23390&author=MS%20Lidow&author=PS%20Goldman-Rakic&author=DW%20Gallager&author=P%20Rakic&volume=40&publication_year=1991&pages=657-671&pmid=2062437&doi=10.1016/0306-4522(91)90003-7&)

11. Goldman-Rakic PS, Lidow MS, Smiley JF, Williams MS. The anatomy of dopamine in monkey and human prefrontal cortex. J Neural Transm Suppl. 1992;36:163–177. doi: 10.1007/978-3-7091-9211-5_8.  [DOI](https://doi.org/10.1007/978-3-7091-9211-5_8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1527516/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Neural%20Transm%20Suppl&title=The%20anatomy%20of%20dopamine%20in%20monkey%20and%20human%20prefrontal%20cortex&author=PS%20Goldman-Rakic&author=MS%20Lidow&author=JF%20Smiley&author=MS%20Williams&volume=36&publication_year=1992&pages=163-177&pmid=1527516&doi=10.1007/978-3-7091-9211-5_8&)

12. Ebert PJ, Campbell DB, Levitt P. Bacterial artificial chromosome transgenic analysis of dynamic expression patterns of regulator of G-protein signaling 4 during development. I. Cerebral cortex. Neuroscience. 2006;142:1145–1161. doi: 10.1016/j.neuroscience.2006.08.011.  [DOI](https://doi.org/10.1016/j.neuroscience.2006.08.011) | [PMC free article](/articles/PMC1780259/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16996696/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neuroscience&title=Bacterial%20artificial%20chromosome%20transgenic%20analysis%20of%20dynamic%20expression%20patterns%20of%20regulator%20of%20G-protein%20signaling%204%20during%20development.%20I.%20Cerebral%20cortex&author=PJ%20Ebert&author=DB%20Campbell&author=P%20Levitt&volume=142&publication_year=2006&pages=1145-1161&pmid=16996696&doi=10.1016/j.neuroscience.2006.08.011&)

13. Ebert PJ, Campbell DB, Levitt P. Bacterial artificial chromosome transgenic analysis of dynamic expression patterns of regulator of G-protein signaling 4 during development. II. Subcortical regions. Neuroscience. 2006;142:1163–1181. doi: 10.1016/j.neuroscience.2006.08.012.  [DOI](https://doi.org/10.1016/j.neuroscience.2006.08.012) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16989953/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neuroscience&title=Bacterial%20artificial%20chromosome%20transgenic%20analysis%20of%20dynamic%20expression%20patterns%20of%20regulator%20of%20G-protein%20signaling%204%20during%20development.%20II.%20Subcortical%20regions&author=PJ%20Ebert&author=DB%20Campbell&author=P%20Levitt&volume=142&publication_year=2006&pages=1163-1181&pmid=16989953&doi=10.1016/j.neuroscience.2006.08.012&)

14. Ding J, Guzman JN, Tkatch T, Chen S, Goldberg JA, Ebert PJ, et al. RGS4-dependent attenuation of M(4) autoreceptor function in striatal cholinergic interneurons following dopamine depletion. Nat Neurosci. 2006;9:832–842. doi: 10.1038/nn1700.  [DOI](https://doi.org/10.1038/nn1700) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16699510/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Neurosci&title=RGS4-dependent%20attenuation%20of%20M(4)%20autoreceptor%20function%20in%20striatal%20cholinergic%20interneurons%20following%20dopamine%20depletion&author=J%20Ding&author=JN%20Guzman&author=T%20Tkatch&author=S%20Chen&author=JA%20Goldberg&volume=9&publication_year=2006&pages=832-842&pmid=16699510&doi=10.1038/nn1700&)

15. Ross EM, Wilkie TM. GTPase-activating proteins for heterotrimeric G proteins: regulators of G protein signaling (RGS) and RGS-like proteins. Annu Rev Biochem. 2000;69:795–827. doi: 10.1146/annurev.biochem.69.1.795.  [DOI](https://doi.org/10.1146/annurev.biochem.69.1.795) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10966476/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annu%20Rev%20Biochem&title=GTPase-activating%20proteins%20for%20heterotrimeric%20G%20proteins:%20regulators%20of%20G%20protein%20signaling%20(RGS)%20and%20RGS-like%20proteins&author=EM%20Ross&author=TM%20Wilkie&volume=69&publication_year=2000&pages=795-827&pmid=10966476&doi=10.1146/annurev.biochem.69.1.795&)

16. Erdely HA, Tamminga CA, Roberts RC, Vogel MW. Regional alterations in RGS4 protein in schizophrenia. Synapse. 2006;59:472–479. doi: 10.1002/syn.20265.  [DOI](https://doi.org/10.1002/syn.20265) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16565965/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Synapse&title=Regional%20alterations%20in%20RGS4%20protein%20in%20schizophrenia&author=HA%20Erdely&author=CA%20Tamminga&author=RC%20Roberts&author=MW%20Vogel&volume=59&publication_year=2006&pages=472-479&pmid=16565965&doi=10.1002/syn.20265&)

17. Brzustowicz LM, Hodgkinson KA, Chow EW, Honer WG, Bassett AS. Location of a major susceptibility locus for familial schizophrenia on chromosome 1q21-q22. Science. 2000;288:678–682. doi: 10.1126/science.288.5466.678.  [DOI](https://doi.org/10.1126/science.288.5466.678) | [PMC free article](/articles/PMC3787922/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10784452/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Location%20of%20a%20major%20susceptibility%20locus%20for%20familial%20schizophrenia%20on%20chromosome%201q21-q22&author=LM%20Brzustowicz&author=KA%20Hodgkinson&author=EW%20Chow&author=WG%20Honer&author=AS%20Bassett&volume=288&publication_year=2000&pages=678-682&pmid=10784452&doi=10.1126/science.288.5466.678&)

18. Gurling HM, Kalsi G, Brynjolfson J, Sigmundsson T, Sherrington R, Mankoo BS, et al. Genomewide genetic linkage analysis confirms the presence of susceptibility loci for schizophrenia, on chromosomes 1q32.2, 5q33.2, and 8p21-22 and provides support for linkage to schizophrenia, on chromosomes 11q23.3-24 and 20q12.1-11.23. Am J Hum Genet. 2001;68:661–673. doi: 10.1086/318788.  [DOI](https://doi.org/10.1086/318788) | [PMC free article](/articles/PMC1274479/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11179014/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Genomewide%20genetic%20linkage%20analysis%20confirms%20the%20presence%20of%20susceptibility%20loci%20for%20schizophrenia,%20on%20chromosomes%201q32.2,%205q33.2,%20and%208p21-22%20and%20provides%20support%20for%20linkage%20to%20schizophrenia,%20on%20chromosomes%2011q23.3-24%20and%2020q12.1-11.23&author=HM%20Gurling&author=G%20Kalsi&author=J%20Brynjolfson&author=T%20Sigmundsson&author=R%20Sherrington&volume=68&publication_year=2001&pages=661-673&pmid=11179014&doi=10.1086/318788&)

19. Lewis CM, Levinson DF, Wise LH, DeLisi LE, Straub RE, Hovatta I, et al. Genome scan meta-analysis of schizophrenia and bipolar disorder, part II: Schizophrenia. Am J Hum Genet. 2003;73:34–48. doi: 10.1086/376549.  [DOI](https://doi.org/10.1086/376549) | [PMC free article](/articles/PMC1180588/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12802786/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Genome%20scan%20meta-analysis%20of%20schizophrenia%20and%20bipolar%20disorder,%20part%20II:%20Schizophrenia&author=CM%20Lewis&author=DF%20Levinson&author=LH%20Wise&author=LE%20DeLisi&author=RE%20Straub&volume=73&publication_year=2003&pages=34-48&pmid=12802786&doi=10.1086/376549&)

20. Chowdari KV, Mirnics K, Semwal P, Wood J, Lawrence E, Bhatia T, et al. Association and linkage analyses of RGS4 polymorphisms in schizophrenia. Hum Mol Genet. 2002;11:1373–1380. doi: 10.1093/hmg/11.12.1373.  [DOI](https://doi.org/10.1093/hmg/11.12.1373) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12023979/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=Association%20and%20linkage%20analyses%20of%20RGS4%20polymorphisms%20in%20schizophrenia&author=KV%20Chowdari&author=K%20Mirnics&author=P%20Semwal&author=J%20Wood&author=E%20Lawrence&volume=11&publication_year=2002&pages=1373-1380&pmid=12023979&doi=10.1093/hmg/11.12.1373&)

21. Williams NM, Preece A, Spurlock G, Norton N, Williams HJ, McCreadie RG, et al. Support for RGS4 as a susceptibility gene for schizophrenia. Biol Psychiatry. 2004;55:192–195. doi: 10.1016/j.biopsych.2003.11.002.  [DOI](https://doi.org/10.1016/j.biopsych.2003.11.002) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14732600/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biol%20Psychiatry&title=Support%20for%20RGS4%20as%20a%20susceptibility%20gene%20for%20schizophrenia&author=NM%20Williams&author=A%20Preece&author=G%20Spurlock&author=N%20Norton&author=HJ%20Williams&volume=55&publication_year=2004&pages=192-195&pmid=14732600&doi=10.1016/j.biopsych.2003.11.002&)

22. Morris DW, Rodgers A, McGhee KA, Schwaiger S, Scully P, Quinn J, et al. Confirming RGS4 as a susceptibility gene for schizophrenia. Am J Med Genet B Neuropsychiatr Genet. 2004;125:50–53. doi: 10.1002/ajmg.b.20109.  [DOI](https://doi.org/10.1002/ajmg.b.20109) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14755443/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Med%20Genet%20B%20Neuropsychiatr%20Genet&title=Confirming%20RGS4%20as%20a%20susceptibility%20gene%20for%20schizophrenia&author=DW%20Morris&author=A%20Rodgers&author=KA%20McGhee&author=S%20Schwaiger&author=P%20Scully&volume=125&publication_year=2004&pages=50-53&pmid=14755443&doi=10.1002/ajmg.b.20109&)

23. Chen X, Dunham C, Kendler S, Wang X, O’Neill FA, Walsh D, et al. Regulator of G-protein signaling 4 (RGS4) gene is associated with schizophrenia in Irish high density families. Am J Med Genet B Neuropsychiatr Genet. 2004;129:23–26. doi: 10.1002/ajmg.b.30078.  [DOI](https://doi.org/10.1002/ajmg.b.30078) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15274033/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Med%20Genet%20B%20Neuropsychiatr%20Genet&title=Regulator%20of%20G-protein%20signaling%204%20(RGS4)%20gene%20is%20associated%20with%20schizophrenia%20in%20Irish%20high%20density%20families&author=X%20Chen&author=C%20Dunham&author=S%20Kendler&author=X%20Wang&author=FA%20O%E2%80%99Neill&volume=129&publication_year=2004&pages=23-26&pmid=15274033&doi=10.1002/ajmg.b.30078&)

24. Zhang F, St Clair D, Liu X, Sun X, Sham PC, Crombie C, et al. Association analysis of the RGS4 gene in Han Chinese and Scottish populations with schizophrenia. Genes Brain Behav. 2005;4:444–448. doi: 10.1111/j.1601-183X.2005.00167.x.  [DOI](https://doi.org/10.1111/j.1601-183X.2005.00167.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16176390/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genes%20Brain%20Behav&title=Association%20analysis%20of%20the%20RGS4%20gene%20in%20Han%20Chinese%20and%20Scottish%20populations%20with%20schizophrenia&author=F%20Zhang&author=D%20St%20Clair&author=X%20Liu&author=X%20Sun&author=PC%20Sham&volume=4&publication_year=2005&pages=444-448&pmid=16176390&doi=10.1111/j.1601-183X.2005.00167.x&)

25. Fallin MD, Lasseter VK, Avramopoulos D, Nicodemus KK, Wolyniec PS, McGrath JA, et al. Bipolar I disorder and schizophrenia: a 440-single-nucleotide polymorphism screen of 64 candidate genes among Ashkenazi Jewish case-parent trios. Am J Hum Genet. 2005;77:918–936. doi: 10.1086/497703.  [DOI](https://doi.org/10.1086/497703) | [PMC free article](/articles/PMC1285177/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16380905/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Bipolar%20I%20disorder%20and%20schizophrenia:%20a%20440-single-nucleotide%20polymorphism%20screen%20of%2064%20candidate%20genes%20among%20Ashkenazi%20Jewish%20case-parent%20trios&author=MD%20Fallin&author=VK%20Lasseter&author=D%20Avramopoulos&author=KK%20Nicodemus&author=PS%20Wolyniec&volume=77&publication_year=2005&pages=918-936&pmid=16380905&doi=10.1086/497703&)

26. Sobell JL, Richard C, Wirshing DA, Heston LL. Failure to confirm association between RGS4 haplotypes and schizophrenia in Caucasians. Am J Med Genet B Neuropsychiatr Genet. 2005;139:23–27. doi: 10.1002/ajmg.b.30221.  [DOI](https://doi.org/10.1002/ajmg.b.30221) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16082709/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Med%20Genet%20B%20Neuropsychiatr%20Genet&title=Failure%20to%20confirm%20association%20between%20RGS4%20haplotypes%20and%20schizophrenia%20in%20Caucasians&author=JL%20Sobell&author=C%20Richard&author=DA%20Wirshing&author=LL%20Heston&volume=139&publication_year=2005&pages=23-27&pmid=16082709&doi=10.1002/ajmg.b.30221&)

27. Rizig MA, McQuillin A, Puri V, Choudhury K, Datta S, Thirumalai S, et al. Failure to confirm genetic association between schizophrenia and markers on chromosome 1q23.3 in the region of the gene encoding the regulator of G-protein signaling 4 protein (RGS4) Am J Med Genet B Neuropsychiatr Genet. 2006;141:296–300. doi: 10.1002/ajmg.b.30288.  [DOI](https://doi.org/10.1002/ajmg.b.30288) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16508931/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Med%20Genet%20B%20Neuropsychiatr%20Genet&title=Failure%20to%20confirm%20genetic%20association%20between%20schizophrenia%20and%20markers%20on%20chromosome%201q23.3%20in%20the%20region%20of%20the%20gene%20encoding%20the%20regulator%20of%20G-protein%20signaling%204%20protein%20(RGS4)&author=MA%20Rizig&author=A%20McQuillin&author=V%20Puri&author=K%20Choudhury&author=S%20Datta&volume=141&publication_year=2006&pages=296-300&pmid=16508931&doi=10.1002/ajmg.b.30288&)

28. Liu YL, Shen-Jang Fann C, Liu CM, Wu JY, Hung SI, Chan HY, et al. Evaluation of RGS4 as a candidate gene for schizophrenia. Am J Med Genet B Neuropsychiatr Genet. 2006;141:418–420. doi: 10.1002/ajmg.b.30286.  [DOI](https://doi.org/10.1002/ajmg.b.30286) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16526029/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Med%20Genet%20B%20Neuropsychiatr%20Genet&title=Evaluation%20of%20RGS4%20as%20a%20candidate%20gene%20for%20schizophrenia&author=YL%20Liu&author=C%20Shen-Jang%20Fann&author=CM%20Liu&author=JY%20Wu&author=SI%20Hung&volume=141&publication_year=2006&pages=418-420&pmid=16526029&doi=10.1002/ajmg.b.30286&)

29. Kampman O, Illi A, Hanninen K, Katila H, Anttila S, Rontu R, et al. RGS4 genotype is not associated with antipsychotic medication response in schizophrenia. J Neural Transm. 2006;113:1563–1568. doi: 10.1007/s00702-006-0445-2.  [DOI](https://doi.org/10.1007/s00702-006-0445-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16604300/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Neural%20Transm&title=RGS4%20genotype%20is%20not%20associated%20with%20antipsychotic%20medication%20response%20in%20schizophrenia&author=O%20Kampman&author=A%20Illi&author=K%20Hanninen&author=H%20Katila&author=S%20Anttila&volume=113&publication_year=2006&pages=1563-1568&pmid=16604300&doi=10.1007/s00702-006-0445-2&)

30. Talkowski ME, Seltman H, Bassett AS, Brzustowicz LM, Chen X, Chowdari KV, et al. Evaluation of a Susceptibility Gene for Schizophrenia: Genotype Based Meta-Analysis of RGS4 Polymorphisms from Thirteen Independent Samples. Biol Psychiatry. 2006;60:152–162. doi: 10.1016/j.biopsych.2006.02.015.  [DOI](https://doi.org/10.1016/j.biopsych.2006.02.015) | [PMC free article](/articles/PMC3133764/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16631129/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biol%20Psychiatry&title=Evaluation%20of%20a%20Susceptibility%20Gene%20for%20Schizophrenia:%20Genotype%20Based%20Meta-Analysis%20of%20RGS4%20Polymorphisms%20from%20Thirteen%20Independent%20Samples&author=ME%20Talkowski&author=H%20Seltman&author=AS%20Bassett&author=LM%20Brzustowicz&author=X%20Chen&volume=60&publication_year=2006&pages=152-162&pmid=16631129&doi=10.1016/j.biopsych.2006.02.015&)

31. Stroup TS, McEvoy JP, Swartz MS, Byerly MJ, Glick ID, Canive JM, et al. The National Institute of Mental Health Clinical Antipsychotic Trials of Intervention Effectiveness (CATIE) project: schizophrenia trial design and protocol development. Schizophr Bull. 2003;29:15–31. doi: 10.1093/oxfordjournals.schbul.a006986.  [DOI](https://doi.org/10.1093/oxfordjournals.schbul.a006986) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12908658/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Schizophr%20Bull&title=The%20National%20Institute%20of%20Mental%20Health%20Clinical%20Antipsychotic%20Trials%20of%20Intervention%20Effectiveness%20(CATIE)%20project:%20schizophrenia%20trial%20design%20and%20protocol%20development&author=TS%20Stroup&author=JP%20McEvoy&author=MS%20Swartz&author=MJ%20Byerly&author=ID%20Glick&volume=29&publication_year=2003&pages=15-31&pmid=12908658&doi=10.1093/oxfordjournals.schbul.a006986&)

32. Lieberman JA, Stroup TS, McEvoy JP, Swartz MS, Rosenheck RA, Perkins DO, et al. Effectiveness of antipsychotic drugs in patients with chronic schizophrenia. N Engl J Med. 2005;353:1209–1223. doi: 10.1056/NEJMoa051688.  [DOI](https://doi.org/10.1056/NEJMoa051688) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16172203/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Effectiveness%20of%20antipsychotic%20drugs%20in%20patients%20with%20chronic%20schizophrenia&author=JA%20Lieberman&author=TS%20Stroup&author=JP%20McEvoy&author=MS%20Swartz&author=RA%20Rosenheck&volume=353&publication_year=2005&pages=1209-1223&pmid=16172203&doi=10.1056/NEJMoa051688&)

33. Sullivan PF, Keefe RS, Lange LA, Lange EM, Stroup TS, Lieberman J, et al. NCAM1 and Neurocognition in Schizophrenia. Biol Psychiatry. 2006;61:902–910. doi: 10.1016/j.biopsych.2006.07.036.  [DOI](https://doi.org/10.1016/j.biopsych.2006.07.036) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17161382/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biol%20Psychiatry&title=NCAM1%20and%20Neurocognition%20in%20Schizophrenia&author=PF%20Sullivan&author=RS%20Keefe&author=LA%20Lange&author=EM%20Lange&author=TS%20Stroup&volume=61&publication_year=2006&pages=902-910&pmid=17161382&doi=10.1016/j.biopsych.2006.07.036&)

34. Kay SR, Fiszbein A, Opler LA. The positive and negative syndrome scale (PANSS) for schizophrenia. Schizophr Bull. 1987;13:261–276. doi: 10.1093/schbul/13.2.261.  [DOI](https://doi.org/10.1093/schbul/13.2.261) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/3616518/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Schizophr%20Bull&title=The%20positive%20and%20negative%20syndrome%20scale%20(PANSS)%20for%20schizophrenia&author=SR%20Kay&author=A%20Fiszbein&author=LA%20Opler&volume=13&publication_year=1987&pages=261-276&pmid=3616518&doi=10.1093/schbul/13.2.261&)

35. The International HapMap Consortium. The International HapMap Project. Nature. 2003;426:789–796. doi: 10.1038/nature02168.  [DOI](https://doi.org/10.1038/nature02168) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14685227/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=The%20International%20HapMap%20Project&volume=426&publication_year=2003&pages=789-796&pmid=14685227&doi=10.1038/nature02168&)

36. Livak KJ. Allelic discrimination using fluorogenic probes and the 5′ nuclease assay. Genet Anal. 1999;14:143–149. doi: 10.1016/s1050-3862(98)00019-9.  [DOI](https://doi.org/10.1016/s1050-3862(98)00019-9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10084106/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genet%20Anal&title=Allelic%20discrimination%20using%20fluorogenic%20probes%20and%20the%205%E2%80%B2%20nuclease%20assay&author=KJ%20Livak&volume=14&publication_year=1999&pages=143-149&pmid=10084106&doi=10.1016/s1050-3862(98)00019-9&)

37. Sullivan PF, Eaves LJ, Kendler KS, Neale MC. Genetic case-control association studies in neuropsychiatry. Arch Gen Psychiatry. 2001;58:1015–1024. doi: 10.1001/archpsyc.58.11.1015.  [DOI](https://doi.org/10.1001/archpsyc.58.11.1015) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11695947/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arch%20Gen%20Psychiatry&title=Genetic%20case-control%20association%20studies%20in%20neuropsychiatry&author=PF%20Sullivan&author=LJ%20Eaves&author=KS%20Kendler&author=MC%20Neale&volume=58&publication_year=2001&pages=1015-1024&pmid=11695947&doi=10.1001/archpsyc.58.11.1015&)

38. Heiman GA, Hodge SE, Gorroochurn P, Zhang J, Greenberg DA. Effect of population stratification on case-control association studies. I. Elevation in false positive rates and comparison to confounding risk ratios (a simulation study) Hum Hered. 2004;58:30–39. doi: 10.1159/000081454.  [DOI](https://doi.org/10.1159/000081454) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15604562/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Hered&title=Effect%20of%20population%20stratification%20on%20case-control%20association%20studies.%20I.%20Elevation%20in%20false%20positive%20rates%20and%20comparison%20to%20confounding%20risk%20ratios%20(a%20simulation%20study)&author=GA%20Heiman&author=SE%20Hodge&author=P%20Gorroochurn&author=J%20Zhang&author=DA%20Greenberg&volume=58&publication_year=2004&pages=30-39&pmid=15604562&doi=10.1159/000081454&)

39. Gorroochurn P, Hodge SE, Heiman G, Greenberg DA. Effect of population stratification on case-control association studies. II. False-positive rates and their limiting behavior as number of subpopulations increases. Hum Hered. 2004;58:40–48. doi: 10.1159/000081455.  [DOI](https://doi.org/10.1159/000081455) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15604563/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Hered&title=Effect%20of%20population%20stratification%20on%20case-control%20association%20studies.%20II.%20False-positive%20rates%20and%20their%20limiting%20behavior%20as%20number%20of%20subpopulations%20increases&author=P%20Gorroochurn&author=SE%20Hodge&author=G%20Heiman&author=DA%20Greenberg&volume=58&publication_year=2004&pages=40-48&pmid=15604563&doi=10.1159/000081455&)

40. Zeger SL, Liang KY, Albert PS. Models for longitudinal data: a generalized estimating equation approach. Biometrics. 1988;44:1049–1060.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/3233245/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biometrics&title=Models%20for%20longitudinal%20data:%20a%20generalized%20estimating%20equation%20approach&author=SL%20Zeger&author=KY%20Liang&author=PS%20Albert&volume=44&publication_year=1988&pages=1049-1060&pmid=3233245&)

41. Schaid DJ, Rowland CM, Tines DE, Jacobson RM, Poland GA. Score tests for association between traits and haplotypes when linkage phase is ambiguous. Am J Hum Genet. 2002;70:425–434. doi: 10.1086/338688.  [DOI](https://doi.org/10.1086/338688) | [PMC free article](/articles/PMC384917/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11791212/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Score%20tests%20for%20association%20between%20traits%20and%20haplotypes%20when%20linkage%20phase%20is%20ambiguous&author=DJ%20Schaid&author=CM%20Rowland&author=DE%20Tines&author=RM%20Jacobson&author=GA%20Poland&volume=70&publication_year=2002&pages=425-434&pmid=11791212&doi=10.1086/338688&)

42. Barrett JC, Fry B, Maller J, Daly MJ. Haploview: analysis and visualization of LD and haplotype maps. Bioinformatics. 2005;21:263–265. doi: 10.1093/bioinformatics/bth457.  [DOI](https://doi.org/10.1093/bioinformatics/bth457) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15297300/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=Haploview:%20analysis%20and%20visualization%20of%20LD%20and%20haplotype%20maps&author=JC%20Barrett&author=B%20Fry&author=J%20Maller&author=MJ%20Daly&volume=21&publication_year=2005&pages=263-265&pmid=15297300&doi=10.1093/bioinformatics/bth457&)

43. Gabriel SB, Schaffner SF, Nguyen H, Moore JM, Roy J, Blumenstiel B, et al. The structure of haplotype blocks in the human genome. Science. 2002;296:2225–2229. doi: 10.1126/science.1069424.  [DOI](https://doi.org/10.1126/science.1069424) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12029063/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=The%20structure%20of%20haplotype%20blocks%20in%20the%20human%20genome&author=SB%20Gabriel&author=SF%20Schaffner&author=H%20Nguyen&author=JM%20Moore&author=J%20Roy&volume=296&publication_year=2002&pages=2225-2229&pmid=12029063&doi=10.1126/science.1069424&)

44. Benjamini YHY. Controlling the false discovery rate: a practical and powerful approach to multiple testing. Journal of the Royal Statistical Society (Series B) 1995;57:289–300.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20the%20Royal%20Statistical%20Society%20(Series%20B)&title=Controlling%20the%20false%20discovery%20rate:%20a%20practical%20and%20powerful%20approach%20to%20multiple%20testing&author=YHY%20Benjamini&volume=57&publication_year=1995&pages=289-300&)

45. Colhoun HM, McKeigue PM, Davey Smith G. Problems of reporting genetic associations with complex outcomes. Lancet. 2003;361:865–872. doi: 10.1016/s0140-6736(03)12715-8.  [DOI](https://doi.org/10.1016/s0140-6736(03)12715-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12642066/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Problems%20of%20reporting%20genetic%20associations%20with%20complex%20outcomes&author=HM%20Colhoun&author=PM%20McKeigue&author=G%20Davey%20Smith&volume=361&publication_year=2003&pages=865-872&pmid=12642066&doi=10.1016/s0140-6736(03)12715-8&)

46. Little J, Bradley L, Bray MS, Clyne M, Dorman J, Ellsworth DL, et al. Reporting, appraising, and integrating data on genotype prevalence and gene-disease associations. Am J Epidemiol. 2002;156:300–310. doi: 10.1093/oxfordjournals.aje.a000179.  [DOI](https://doi.org/10.1093/oxfordjournals.aje.a000179) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12181099/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Epidemiol&title=Reporting,%20appraising,%20and%20integrating%20data%20on%20genotype%20prevalence%20and%20gene-disease%20associations&author=J%20Little&author=L%20Bradley&author=MS%20Bray&author=M%20Clyne&author=J%20Dorman&volume=156&publication_year=2002&pages=300-310&pmid=12181099&doi=10.1093/oxfordjournals.aje.a000179&)

47. Wittke-Thompson JK, Pluzhnikov A, Cox NJ. Rational inferences about departures from Hardy-Weinberg equilibrium. Am J Hum Genet. 2005;76:967–986. doi: 10.1086/430507.  [DOI](https://doi.org/10.1086/430507) | [PMC free article](/articles/PMC1196455/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15834813/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Rational%20inferences%20about%20departures%20from%20Hardy-Weinberg%20equilibrium&author=JK%20Wittke-Thompson&author=A%20Pluzhnikov&author=NJ%20Cox&volume=76&publication_year=2005&pages=967-986&pmid=15834813&doi=10.1086/430507&)

48. Prasad KM, Chowdari KV, Nimgaonkar VL, Talkowski ME, Lewis DA, Keshavan MS. Genetic polymorphisms of the RGS4 and dorsolateral prefrontal cortex morphometry among first episode schizophrenia patients. Mol Psychiatry. 2005;10:213–219. doi: 10.1038/sj.mp.4001562.  [DOI](https://doi.org/10.1038/sj.mp.4001562) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15381923/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Psychiatry&title=Genetic%20polymorphisms%20of%20the%20RGS4%20and%20dorsolateral%20prefrontal%20cortex%20morphometry%20among%20first%20episode%20schizophrenia%20patients&author=KM%20Prasad&author=KV%20Chowdari&author=VL%20Nimgaonkar&author=ME%20Talkowski&author=DA%20Lewis&volume=10&publication_year=2005&pages=213-219&pmid=15381923&doi=10.1038/sj.mp.4001562&)

49. Ioannidis JP. Commentary: grading the credibility of molecular evidence for complex diseases. Int J Epidemiol. 2006;35:572–578. doi: 10.1093/ije/dyl003.  [DOI](https://doi.org/10.1093/ije/dyl003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16540537/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Epidemiol&title=Commentary:%20grading%20the%20credibility%20of%20molecular%20evidence%20for%20complex%20diseases&author=JP%20Ioannidis&volume=35&publication_year=2006&pages=572-578&pmid=16540537&doi=10.1093/ije/dyl003&)
