# Peer Smoking and the Nicotinic Receptor Genes: An Examination of Genetic and Environmental Risks for Nicotine Dependence

## Metadata
**Authors:** Eric O Johnson, Li-Shiun Chen, Naomi Breslau, Dorothy Hatsukami, Tania Robbins, Nancy L Saccone, Richard A Grucza, Laura J Bierut
**Journal:** Addiction (Abingdon, England)
**Date:** 2010 Sep 15
**DOI:** [10.1111/j.1360-0443.2010.03074.x](https://doi.org/10.1111/j.1360-0443.2010.03074.x)
**PMID:** 20840187
**PMCID:** PMC2970633
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2970633/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC2970633/pdf/nihms207213.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC2970633/pdf/nihms207213.pdf)

## Abstract

**Background:** 
Peer smoking provides a socially reinforcing context of friends’ encouragement and approval that contributes to smoking behavior. Twin studies show correlations and interactions between peer substance use and genetic liability for substance use. However, none examined specific genes. Here we test the hypothesis that the nicotinic receptor genes CHRNA5 (rs16969968), CHRNA3 (rs578776), CHRNB3 (rs13277254), and CHRND (rs12466358) modify the risk for nicotine dependence (ND) associated with peer smoking.

**Methods:** 
Cases of current nicotine dependence (FTND ≥ 4) and smoking-exposed (smoked 100+ cigarettes lifetime), but non-dependent controls (lifetime FTND = 0) came from the Collaborative Genetic Study of Nicotine Dependence (n=2,038). Peer smoking was retrospectively assessed for grades 9–12.

**Results:** 
Peer smoking and the four SNPs were associated with ND. A statistically significant interaction was found between peer smoking and rs16969968 (p = 0.0077). Overall risk of ND was highest for the rs16969968 AA genotype. However, variance in ND attributable to peer smoking was substantially lower among those with the AA genotype at rs16969968 than the lower risk genotypes: AA = 2.5%, GA/AG = 11.2%, GG = 14.2%; p ≤ 0.004.

**Conclusions:** 
Peer smoking had a substantially lower effect on ND among those with the high risk AA genotype at the functional SNP rs16969968 (CHRNA5) than among those with lower risk genotypes. Such results highlight the possibility that given drug exposure those with specific genetic risks may be less affected by social contexts and intervention strategies focused on social factors could have less influence on those at highest genetic risk.

Keywords: nicotine dependence, peer smoking, gene-environmental interaction, nicotinic receptor genes, case control study

### Background

Peer smoking provides a socially reinforcing context of friends’ encouragement and approval that contributes to smoking behavior. Twin studies show correlations and interactions between peer substance use and genetic liability for substance use. However, none examined specific genes. Here we test the hypothesis that the nicotinic receptor genes *CHRNA5* (rs16969968), *CHRNA3* (rs578776), *CHRNB3* (rs13277254), and *CHRND* (rs12466358) modify the risk for nicotine dependence (ND) associated with peer smoking.

### Methods

Cases of current nicotine dependence (FTND ≥ 4) and smoking-exposed (smoked 100+ cigarettes lifetime), but non-dependent controls (lifetime FTND = 0) came from the Collaborative Genetic Study of Nicotine Dependence (n=2,038). Peer smoking was retrospectively assessed for grades 9–12.

### Results

Peer smoking and the four SNPs were associated with ND. A statistically significant interaction was found between peer smoking and rs16969968 (p = 0.0077). Overall risk of ND was highest for the rs16969968 AA genotype. However, variance in ND attributable to peer smoking was substantially lower among those with the AA genotype at rs16969968 than the lower risk genotypes: AA = 2.5%, GA/AG = 11.2%, GG = 14.2%; p ≤ 0.004.

### Conclusions

Peer smoking had a substantially lower effect on ND among those with the high risk AA genotype at the functional SNP rs16969968 (*CHRNA5*) than among those with lower risk genotypes. Such results highlight the possibility that given drug exposure those with specific genetic risks may be less affected by social contexts and intervention strategies focused on social factors could have less influence on those at highest genetic risk.

## Introduction

Peer smoking is a strong correlate of smoking behavior, predicting initiation, regular smoking and dependent smoking [[1](#R1)]. Evidence suggests that peer influence in established friendship networks and selection of peers in establishing new friendships both play roles in the observed similarity among peers for a number of risk behaviors [[2](#R2)–[4](#R4)] including smoking [[5](#R5)–[6](#R6)]. A variety of theories have been used to postulate and understand mechanisms by which peer smoking behaviors increase the risk of smoking onset and maintenance of smoking including Social Learning, Primary Socialization, Social Identity and Social Network Theories [[7](#R7)–[8](#R8)]. The key elements of these theories and empirical evidence suggest that, affiliation with smoking peers provides direct opportunities to initiate smoking and models for learning how to smoke (peer influence). Smoking peers also provide a socially reinforcing context of friends’ encouragement and approval [[9](#R9)–[10](#R10)] that maintains smoking behavior and a smoker social identity (peer influence and selection). Consistent with the reinforcement/maintenance effects of affiliating with smoking peers, peer smoking is associated with multiple stages of smoking and its influence extends beyond adolescence into adulthood [[1](#R1)].

However, peer smoking influence plays out against a backdrop of individual variability in genetic vulnerability to smoking and nicotine dependence. From a genetic perspective the interplay of peer and genetic influences on smoking may follow two mechanisms: gene-environment interactions (GxE) and/or gene-environment correlation (rGE) [[11](#R11)]. In the case of GxE the effects of peer smoking may differ (being lesser or greater) by individual genetic risk for smoking. Genes may also influence the degree to which individuals associate with smoking peers by influencing an individuals selection of peers (active rGE) or peer groups selection of an individual (evocative rGE) based on genetically influenced traits and behaviors [[11](#R11)–[12](#R12)]. Twin studies examining peer behaviors and genetic factors for substance use phenotypes have found evidence for both rGE and GxE [[12](#R12)–[16](#R16)]. These studies provide important insights into the interplay between the peer social context and a general genetic predisposition toward substance use and abuse symptoms. However, none of these studies specifically examined cigarette smoking or nicotine dependence, and there are no studies examining specific genetic variants associated with nicotine dependence and the influence of peer smoking.

In the first genome-wide association study of nicotine dependence [[17](#R17)] and a companion large scale candidate gene study [[18](#R18)] we identified several strong associations with variants in *CHRN* genes including the single nucleotide polymorphism (SNP) rs16969968 in *CHRNA5*. Independent studies have replicated the association between nicotine dependence and this specific SNP [[19](#R19)–[20](#R20)] and between nicotine dependence and proxy SNPs that are very highly correlated with rs16969968 (r^2^ = 0.975) [[21](#R21)–[22](#R22)]. Most recently we completed a dense coverage association study of the complete family of 16 nicotinic receptor subunit genes and nicotine dependence in which 226 SNPs were analyzed [[23](#R23)]. We identified four distinct loci associated with nicotine dependence after multiple test correction: two statistically independent SNPs in the *CHRNA5-CHRNA3-CHRNB4* gene cluster (rs16969968 and rs578776), one signal in *CHRNB3-CHRNA6* tagged by rs13277254, and a single-SNP association in the *CHRND* gene (rs12466358).

In this study we extend these genetic findings to test GxE between the nicotinic receptor gene SNPs (rs16969968, rs578776, rs13277254, rs12466358) and retrospectively reported peer smoking during grades 9 −12 for nicotine dependence in the Collaborative Genetic Study of Nicotine Dependence (COGEND) sample of European Americans, which is composed of 1,054 current nicotine-dependent smokers as cases (FTND ≥ 4) and 984 smoking-exposed, but non-dependent, controls (smoked ≥ 100 cigarettes lifetime but lifetime FTND = 0). Given these case/control definitions, this study tests the interplay of genes and H.S. peer smoking on the transition from having been cigarette smoking exposed to nicotine dependence.

## Methods and Materials

### Study design and sample

All study participants (N = 2,038) were recruited by the Collaborative Genetic Study of Nicotine Dependence (COGEND), a United States multi-site project based on community samples from St. Louis, MO and Detroit, MI [[17](#R17)–[18](#R18), [23](#R23)]. Cases and controls were required to have smoked at least 100 cigarettes lifetime, the threshold classically used to define a smoker. Cases were nicotine dependent, defined by current Fagerström Test for Nicotine Dependence (FTND) score of 4 or more [[24](#R24)]. Controls were defined as smokers (smoking at least 100 cigarettes lifetime), but who never had any symptoms of dependence (lifetime FTND=0). By selecting controls who smoked, we focused on those genetic and environmental effects that are specific to the development of nicotine dependence rather than smoking initiation. All selected COGEND participants were of European ancestry. See [Table 1](#T1) for sample characteristics.

### Table 1.

|   |  | Total |  | CaseCurrent FTND≥4 |  | ControlLifetime FTND=0 |  |
| --- | --- | --- | --- | --- | --- | --- | --- |
|   |  |  |  |  |  |  |  |
| Total: N |  | 2,038 |   | 1,054 |   | 984 |   |
|   |  |  |  |  |  |  |  |
| Age in years: Mean(sd) |  | 36.4 | (5.5) | 36.9 | (5.4) | 35.9 | (5.5) |
|   |  |  |  |  |  |  |  |
| Male: N, % |  | 792 | 38.9% | 489 | 61.7% | 303 | 38.3% |
|   |  |  |  |  |  |  |  |
| Number of peers who smoked: Mean(sd) |  | 4.6 | (2.7) | 5.2 | (2.5) | 3.8 | (2.7) |
|   |  | N | % | N | % | N | % |
| 0 |  | 212 | 10.8 | 60 | 6.0 | 152 | 15.8 |
| 1 |  | 123 | 6.3 | 42 | 4.2 | 81 | 8.4 |
| 2 |  | 198 | 10.1 | 72 | 7.2 | 126 | 13.1 |
| 3 |  | 168 | 8.5 | 75 | 7.4 | 93 | 10.0 |
| 4 |  | 255 | 13.0 | 134 | 13.3 | 121 | 12.6 |
| 5 |  | 163 | 8.3 | 85 | 8.4 | 78 | 8.1 |
| 6 |  | 257 | 13.1 | 147 | 14.6 | 110 | 11.5 |
| 7 |  | 164 | 8.3 | 99 | 9.8 | 65 | 6.8 |
| 8 |  | 427 | 21.7 | 293 | 29.1 | 134 | 14.0 |
|   |  |  |  |  |  |  |  |
| CHRN A5-A3-B4 Cluster Snps |  |   |   |   |   |   |   |
| rs16969968 |  | N | % | N | % | N | % |
| G/G |  | 870 | 42.7 | 412 | 39.1 | 458 | 46.6 |
| A/G |  | 905 | 44.5 | 469 | 44.5 | 436 | 44.4 |
| A/A |  | 261 | 12.8 | 172 | 16.3 | 89 | 9.0 |
| rs578776: N,% |  |   |   |   |   |   |   |
| C/C |  | 1,067 | 52.7 | 607 | 57.7 | 460 | 47.3 |
| T/C |  | 804 | 39.7 | 380 | 36.1 | 424 | 43.6 |
| T/T |  | 153 | 7.6 | 65 | 6.2 | 88 | 9.0 |
|   |  |  |  |  |  |  |  |
| CHRN B3–A6 Cluster SNP |  |   |   |   |   |   |   |
| rs13277254 |  | N | % | N | % | N | % |
| A/A |  | 1,268 | 62.5 | 688 | 65.5 | 580 | 59.2 |
| A/G |  | 673 | 33.1 | 333 | 31.7 | 340 | 34.7 |
| G/G |  | 89 | 4.4 | 30 | 2.8 | 59 | 6.0 |
|   |  |  |  |  |  |  |  |
| CHRND SNP |  |   |   |   |   |   |   |
| rs12466358 |  | N | % | N | % | N | % |
| T/T |  | 1,138 | 56.0 | 571 | 54.2 | 567 | 58.0 |
| T/G |  | 745 | 36.7 | 395 | 37.5 | 350 | 35.9 |
| G/G |  | 148 | 7.3 | 88 | 8.4 | 60 | 6.1 |

Table 1 Caption: Study Sample Characteristics

The study was carried out in compliance with the Code of Ethics of the World Medical Association and obtained informed consent from all participants and approval from the appropriate institutional review boards.

### Genotyping and quality control

Blood samples were collected for genetic analyses. Initial genotyping of 1608 subjects was performed by Perlegen Sciences using custom arrays and by the Center for Inherited Disease Research (CIDR) using Illumina Golden Gate technology as previously detailed [[17](#R17)–[18](#R18), [23](#R23)]. Genotyping of the additional 445 subjects was done at Washington University using Illumina Golden Gate technology and Sequenom MassArray iPLEX technology [[23](#R23)]. Self-reported race was verified using EIGENSTRAT [[25](#R25)]. For this study, we focused on four SNPs: rs16969968 and rs578776 located in the *CHRNA5-CHRNA3-CHRNB4* gene cluster, rs13277254 in *CHRNB3-CHRNA6*, and rs12466358 in the *CHRND* gene. All four SNPs had call rates of 98% or better. Allele correspondence was checked for the combined genotyped samples.

### High School peer smoking

The number of peers who smoked was assessed by two questions of the form “How many of your four best _____ friends smoked cigarettes during high school?”: one for male and the other for female best friends. The response to both questions was summed providing a count variable of number of best friends who smoked from 0–8. These measures where adapted from the Teenage Attitudes and Practices Survey (TAPS) [[26](#R26)] to reflect the specific period of adolescence and are widely used to characterize level of peer smoking [[27](#R27)].

### Statistical Analysis

#### Multivariable and Gene-Environment Interaction analyses

It is known from our prior work that the SNPs rs16969968, rs578776, rs13277254, rs12466358 are independently associated with nicotine dependence in the COGEND data [[23](#R23)]. The analyses for this study begins with multivariable logistic regression model including all four SNPs (additively coded as 0, 1, or 2 copies of the minor allele among those of European ancestry), number of H.S. peers who smoked (0–8) and the covariates gender and age. For each SNP, we tested for statistically significant multiplicative interaction between all model variables by adding a product term (e.g., rs16969968 × H.S. peer smoking) to the main effects model, retaining those that were statistically significant [[28](#R28)]. The Bonferroni corrected p-value used to test four hypothesized H.S. peer smoking by SNP interactions was 0.0125.

To evaluate the impact of GxE interactions the predicted probabilities of nicotine dependence for number of H.S. peers who smoked were plotted and the variance in nicotine dependence attributable to smoking peers estimated by genotype. Variance explained was estimated by Nagelkerke’s adjusted R^2^ from logistic regression models and the difference in genotype specific R^2^ tested by z-test [[29](#R29)].

We tested the GxE assumption of independence between genetic risk and environment exposures (no gene – environment correlation) by estimating the association between the SNPs rs16969968, rs578776, rs13277254, rs12466358 and peer smoking among controls only [[30](#R30)]. There was no evidence of such rGE between any SNP and H.S. peer smoking using a negative binomial regression model [[31](#R31)] (Incident Risk Ratios: 0.99 to 1.02, p-values > 0.30 for all SNPs). Although the lack of a correlation between these SNPs and H.S. peer smoking might suggest use of a case only approach to testing GxE interactions, we retained use of the case-control method to avoid the potential increase in Type 1 error [[30](#R30)]. The absence of rGE in the control only analyses provides some evidence that any observed GxE interaction is unlikely to be attributable to rGE in the population.

## Results

### Main Effects of Nicotinic Receptor Gene SNPs and H.S. Peer Smoking

[Table 2](#T2) presents the results of the multivariable logistic regression of the nicotinic receptor gene SNPs rs16969968, rs578776, rs13277254, and rs12466358 as well as number of H.S. peers who smoked on nicotine dependence, adjusted for age and gender. Each of these risk factors were independently and significantly associated nicotine dependence.

### Table 2.

| Main Effects Model |   | OR | 95% CI |  | p-value |
| --- | --- | --- | --- | --- | --- |
| LB | UB |  |  |  |  |
|   |  |   |  |  |  |
| Peer Smoking | Number ofpeers (0–8) | 1.26 | 1.21 | 1.31 | <0.0001 |
| CHRN A5-A3-B4 Cluster SNPs |   |   |   |   |   |
|   | rs16969968 | 1.23 | 1.05 | 1.45 | 0.010 |
|   | rs578776 | 0.79 | 0.66 | 0.93 | 0.006 |
| CHRN B3–A6 Cluster SNP |   |   |   |   |   |
|   | rs13277254 | 0.79 | 0.66 | 0.93 | 0.004 |
| CHRND SNP |   |   |   |   |   |
|   | rs12466358 | 1.18 | 1.01 | 1.37 | 0.036 |
| AGE |   | 1.03 | 1.02 | 1.05 | 0.0002 |
| Gender | Female | 1.0 |   |   |   |
|   | Male | 2.58 | 2.11 | 3.16 | <0.0001 |
|   |  |  |  |  |  |
| Interaction Model |   |   |   |   |   |
|   |  |  |  |  |  |
| Peer Smoking | Number ofpeers (0–8) | 1.27 | 1.19 | 1.34 | <0.0001 |
| CHRN A5-A3-B4 Cluster SNPs |   |   |   |   |   |
|   | rs16969968 | 1.70 | 1.28 | 2.28 | 0.0003 |
|   | rs578776 | 0.78 | 0.66 | 0.93 | 0.007 |
| CHRN B3–A6 Cluster SNP |   |   |   |   |   |
|   | rs13277254 | 0.78 | 0.66 | 0.92 | 0.004 |
| CHRND SNP |   |   |   |   |   |
|   | rs12466358 | 1.17 | 1.00 | 1.36 | 0.036 |
| AGE |   | 1.04 | 1.02 | 1.06 | 0.0002 |
| Gender | Female | 1.0 |   |   |   |
|   | Male | 1.52 | 1.03 | 2.24 | 0.03 |
| Peer smoking × rs16969968 |   | 0.93 | 0.88 | 0.98 | 0.0077 |
| Gender × rs16969968 |   | 1.14 | 1.05 | 1.23 | 0.002 |

Table 2 Caption: Multivariable Models of Peer Smoking and CHRN SNPs association with Nicotine Dependence adjust for Age and Gender

Treating number of H.S. peers who smoked and age as continuous variables in logistic regression models makes the assumption that they are linearly associated with nicotine dependence. Testing this assumption using fractional polynomial analysis [[28](#R28)], no higher power polynomial fit the association of peer smoking or age with nicotine dependence better than the linear model (p ≥ 0.49 and p ≥ 0.45 for models up to 4 power terms, respectively). Treating these variables as continuous is appropriate. The resulting odds ratios can be interpreted as the increased risk of nicotine dependence for every additional H.S. peer who smoked (26%) and for every year older a participant was (3%).

### Testing Interactions with H.S. Peer Smoking

Testing all possible interactions among variables in the main effects model, significant interactions were found between H.S. peer smoking and both rs16969968 and gender (p = 0.0077 & p=0.002, respectively; see [Table 2](#T2)). The two-way interaction rs16969968 × gender was not significant (p = 0.32) nor was the three-way interaction peer smoking × rs16969968 × gender (p = 0.67), and the point estimates of the peer smoking by rs16969968 were similar by gender (female OR =0.94; male OR=0.91). H.S. peer smoking appeared to have stronger effects on nicotine dependence for males than for females (OR = 1.37 95% CI 1.29 – 1.46 vs. OR = 1.20 95% CI 1.14 – 1.26). To illustrate the peer smoking by rs16969968 interaction [figure 1](#F1) presents the predicted probabilities of nicotine dependence for number of peer smokers by rs16969968 genotype, adjusted for gender and age. The overall risk of nicotine dependence is highest for the rs16969968 AA genotype but H.S. peer smoking had a substantially lower impact on those with this genotype compare to those with the GA or GG genotypes combined (p = 0.006). Indeed, although the addition of the rs16969968 × peer smoking interaction term increased the explain variance of the overall model (R^2^) by only 0.5% (18.9% vs. 18.4%) the variance in nicotine dependence attributable to H.S. peer smoking was 4–6 times lower among those with highest risk genotype at rs16969968 than the lower risk genotypes: AA = 2.5%, GA/AG = 11.2%, GG = 14.2%; z = 2.64, p=0.004 and z = 3.27, p = 0.0005 respectively.

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4956/2970633/59528c31cbe5/nihms207213f1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=2970633_nihms207213f1.jpg)

Predicted Probabilities of Nicotine Dependence for Number of Peer Smokers by rs16969968 Genotypes, Adjusted for Gender and Age

Adequate sample size across the range of each variable is a concern when testing interactions. [Table 1](#T1) provides descriptive information on peer smoking and the SNP rs16969968. We found no difference in the distribution of H.S. peer smoking by genotype of rs16969968 (*χ2*= 11.26, df=16, p=0.79). Additionally the smallest cell size included 15 subjects (AA genotype with one smoking peer). Thus it appears unlikely that the interaction of H.S. peer smoking and rs16969968 identified in these analyses is an artifact of small cell sizes. We also tested a model in which H.S. peer smoking was treated as an ordinal variable. These analyses found a statistically significant interaction between H.S. peer smoking and rs16969968 comparing the lowest to highest quartiles of number of smoking peers (p = 0.002). Thus the interaction finding appears robust to both the distribution of variables and scale used to model the peer smoking variable.

### Diplotype Analysis of rs16969968 – rs578776

Rs16969968 and rs578776 in the CHRN A5-A3-B4 gene cluster have statistically independent associations with nicotine dependence. However, these SNPs are not biologically independent [[23](#R23)]: the minor risk allele for rs16969968 (A) occurs on the background of the major allele for rs578776 (C), and not all possible joint genotypes are observed (see [Table 3](#T3)). Because of this pattern of linkage disequilibrium, the joint genotypes at these two loci are equivalent to diplotypes (paired haplotypes). With the minor allele of rs16969968 conferring risk for nicotine dependence and the minor allele of rs578776 (T) being protective ([Table 2](#T2)), analyses of the rs16969968/rs578776 diplotypes indicates significantly increased odds of nicotine dependence associated with the AA/CC diplotype and significantly reduced odds associated with the GG/TT diplotype compared with the GG/CC reference adjusting for age, gender and number of H.S. peers who smoked ([Table 3](#T3)).

### Table 3.

|   | rs578776 |  |  |
| --- | --- | --- | --- |
| rs16969968 | C/C | T/C | T/T |
| G/G | 1.0 (ref)146/127 | 0.74 (0.53–1.02)202/236 | 0.63 (0.41–0.96)64/88 |
| A/G | 1.06 (0.77–1.45)290/243 | 0.82 (0.58–1.15)178/188 | --0/0 |
| A/A | 1.61 (1.11–2.35)171/89 | --0/0 | --0/0 |

Table 3 Caption: Association of Nicotine Dependence with rs16969968-rs578776 Diplotype adjusted for Gender, Age and Number of Smoking Peers

In the diplotype regression model we found a H.S. peer smoking by the AA/CC rs16969968/rs578776 diplotype interaction (p = 0.01). As illustrated in [figure 2](#F2) H.S. peer smoking had a significantly lower impact on probability of nicotine dependence for those with AA/CC diplotype – the only observed diplotype that involves the high-risk AA genotype at rs16969968 – compared with other diplotypes. The variance in nicotine dependence attributable to H.S. peer smoking (R^2^) was more than four times lower among those with the AA/CC diplotype than all the other diplotypes combined: AA/CC = 2.7%, all other diplotypes = 12.5%; z = 2.98, p = 0.001. These results are consistent with the H.S. peer smoking by rs16969968 analysis and suggest that accounting for this SNP’s linkage disequilibrium relationship with rs578776 does not meaningfully alter the interpretation of the primary results.

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4956/2970633/e4db7fb19585/nihms207213f2.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=2970633_nihms207213f2.jpg)

Predicted Probabilities of Nicotine Dependence for Number of Peer Smokers by rs16969968 – rs578776 Diplotypes, Adjusted for Gender and Age

## Discussion

In this study we tested gene - environment interactions for risk of nicotine dependence between nicotinic receptor gene SNPs and reported number of H.S. peers who smoked. Significant main effects were observed for H.S. peer smoking and each of the four SNPs examined rs16969968 and rs578776 in the *CHRNA5-CHRNA3-CHRNB4* gene cluster, rs13277254 tagging *CHRNB3-CHRNA6* and rs12466358 in the *CHRND* gene. The statistically significant interaction between rs16969968 and H.S. peer smoking (p = 0.0077) indicated that the impact of H.S. peer smoking on the probability of nicotine dependence was substantially reduced among those with the highest risk genotype (AA).

Prior analyses of this [[23](#R23)] and other samples [[20](#R20)] found statistically significant and independent associations between nicotine dependence or heavy smoking and the four SNPs examined here or their high LD proxies. Of these SNPs only rs16969968 is known to be functional [[23](#R23)]. The minor allele at rs16969968 (A) results in an amino acid change (aspartic acid [D] to asparagine [N]) in the α5 neuronal nicotinic acetylcholine receptor subunit which reduces receptor function *in vitro* in response to a nicotinic agonist [[19](#R19)]. Finding a GxE interaction between this functional SNP and H.S. peer smoking is encouraging, suggesting that the interaction may be more than statistical and stand up to needed tests of replication in independent samples.

Although this is the first study to examine the interaction between specific genes and peer smoking for nicotine dependence, twin studies of related measures of peer influence and the total additive genetic effects on related phenotypes have been reported. These studies have found that the similarity in substance use and substance abuse symptoms between individuals and their peers has been largely attributable to additive genetic variance components [[12](#R12), [14](#R14)–[16](#R16)]. Harden et al. [[14](#R14)] and Dick et al. [[32](#R32)] observed that the genetic variance component for adolescent substance use or alcohol use was greater when peer use was greater, that is a GxE, adjusting for rGE. Testing a series of GxE models for perceived peer deviance Button et al. [[16](#R16)] found that the genetic variance component contributing to substance dependence symptoms during adolescence was greatest where peer deviance was high (social expression model) and where peer deviance was low (social distinction model). Our GxE finding between rs16969968 and H.S. peer smoking wherein the effects of the AA genotype on probability of nicotine dependence were most obvious at lower numbers of smoking peers is consistent with Button et al. [[16](#R16)] social distinction model. However, we interpret the GxE identified here as differing environmental effects by genotype given that those with AA genotype are at higher risk regardless of the level of peer smoking while those with high numbers of smoking peers without this risk genotype “catch-up” in probability of nicotine dependence to those with the high risk genotype but have this high probability for different reasons.

Another distinction between this and prior studies of the interplay between genetic risk and peer influence is this study’s focus on the transition between significant cigarette smoking exposure and becoming nicotine dependent by virtue of the case and control criteria: current smokers with an FTND score ≥ 4 versus having smoked at least 100 cigarettes but a lifetime FTND = 0. Because all study subjects have passed the threshold of smoking initiation the effect of peer smoking in this study represents that component of peer influence that has to do with maintenance of smoking: factors such as reinforcement of pro-smoking norms or connectedness to social networks encourage adoption of a smoker social identity rather than providing opportunities to initiate smoking or models of “how” to smoke. Thus our findings suggest that given exposure to 100 cigarettes or more those with the AA genotype at rs16969968 are only minimally influenced by peers as they transition into nicotine dependence.

Using the COGEND sample we recently reported another interaction with rs16969968 in which the risk of nicotine dependence associated with the AA genotype at rs16969968 was significantly greater among those with low parent monitoring compared to higher monitoring [[33](#R33)]. It is reasonable to expect that low parent monitoring may increase opportunities to affiliate with smoking peers and thereby increase risk of smoking. Indeed, the commentary accompanying Chen et al. [[33](#R33)], suggested this possibility [[34](#R34)]. In the COGEND sample we find a weak but statistically significant correlation between lower parent monitoring and a larger number of peers who smoked (r = 0.15, p < 0.0001). Thus it might be hypothesized that there would be greater risk of nicotine dependence associated with the A allele at rs16969968 when parent monitoring is lower and peer smoking higher. However, this was not what was observed; the probability of nicotine dependence remained high and relatively constant across numbers of smoking peers among those with the AA genotype. To explore these divergent findings we estimated models including both the parent monitoring scale and H.S. peer smoking, which did not significantly alter either measures’ main effect association with nicotine dependence, nor did parent monitoring change the interaction between peer smoking and rs16969968 results (available upon request). Additionally, there was no statistical interaction between H.S. peer smoking and parent monitoring in predicting nicotine dependence (p = 0.36), nor was there evidence of three-way interaction between rs16969968, parent monitoring, and peer smoking (p = 0.50). Thus our prior findings for rs16969968 with parent monitoring and those in the current study of peer smoking appear to be independent: increases in number of peers who smoke does not appear to be the mechanism by which low parent monitoring increases the expression of genetic risk associated with rs16969968.

In addition to the substantive relationship of our prior examination of parent monitoring there is a statistical consideration for the current study. Taking a broader perspective of correcting for multiple tests, results should be considered within the context of the number of tests done using a sample as a whole [[35](#R35)]. Having tested two hypothesized GxE interactions in the COGEND sample prior to the current study [[33](#R33)] plus the four tested here would lower the strict Bonferroni corrected p-value to 0.0083 (0.05/6). As the p-value for the H.S. peer smoking by rs16969968 interaction was 0.0077 this stricter correction does not alter this study’s conclusions.

### Limitations

There are several limitations of this study. First, reports of the number of H.S. peers who smoked were retrospective. Thus, temporal proximity and mechanisms of peer influence on the transition to nicotine dependence could not be examined, but is a direction for future research in genetically informative longitudinal samples. Additionally, recall bias may have affected the magnitude of the main effect of peer smoking on risk for nicotine dependence. However a strong association has been well demonstrated in longitudinal studies [[7](#R7), [32](#R32)]. Moreover, there is little reason to suspect that the level of recall bias would vary by rs16969968 genotype. Second, we were unable to distinguish different types of peer relationships (e.g. romantic versus friendship) which may have differential impact on risk of nicotine dependence. However, we did conduct parallel analyses for same and opposite sex peers. These results were entirely consistent with the presented results and are available upon request. Third, the case–control design of this study does not allow conclusions regarding risks of nicotine dependence in the population as a whole but limits the interpretation of results to the transition to dependent smoking among those exposed to cigarette smoking. Fourth, this study’s statistical power to detect an effect the size observed for the H.S. peer smoking by rs16969968 interaction is low (61% at the multiple testing corrected 2-sided p-value of 0.0125 [[36](#R36)]). As low power increases the risk of false negative tests, the study’s failure to identify other statistical interactions must be considered tentative. Low statistical power does not, however, compromise our primary finding of the H.S. peer smoking by rs16969968 interaction. Fifth, these results require replication in an independent sample. Although the current study does not have a replication sample, it does meet other criteria suggestive of a credible finding [[35](#R35)]: 1) evidence-based selection of genes and environmental risk with main effects on the phenotype; 2) the GxE interaction was found with a functional SNP; and 3) significant results after correcting for multiple testing. Thus the interaction between rs16969968 and H.S. peer smoking is worth follow-up by investigators with independent samples.

### Conclusion

In this first study of the interplay between specific genes and H.S. peer smoking we found both independent and interaction effects with SNPs in the nicotine receptor genes: rs16969968 and rs578776 in the *CHRNA5-CHRNA3-CHRNB4* gene cluster, rs13277254 tagging *CHRNB3-CHRNA6* and rs12466358 in the *CHRND* gene. Number of H.S. peers who smoked had little effect on the probability of nicotine dependence among those with the high risk AA genotype at the functional SNP rs16969968 but had substantial effects among those with lower risk genotypes. This result suggests that those with the AA genotype at rs16969968 may not need as supportive a social environment to make the transition from having initiated smoking to becoming nicotine dependent smokers as those with lower risk genotypes.

## Acknowledgements

In memory of Theodore Reich, founding Principal Investigator of COGEND, we are indebted to his leadership in the establishment and nurturing of COGEND and acknowledge with great admiration his seminal scientific contributions to the field. Lead investigators directing data collection are Laura Bierut, Naomi Breslau, Dorothy Hatsukami, and Eric Johnson. The authors thank Heidi Kromrei and Tracey Richmond for their assistance in data collection. This research was supported by NIH grants CA089392 from the National Cancer Institute, U01HG04422-02 from the National Human Genome Institute, and GA305231 from the Global Research Awards for Nicotine Dependence (GRAND) by Pfizer.

The COGEND project is a collaborative research group and part of the NIDA Genetics Consortium. Subject collection was supported by NIH grant CA89392 (PI - L Bierut) from the National Cancer Institute. Genotyping work at Perlegen Sciences was performed under NIDA Contract HHSN271200477471C. Phenotypic and genotypic data are stored in the NIDA Center for Genetic Studies (NCGS) at [http://zork.wustl.edu/](http://zork.wustl.edu/) under NIDA Contract HHSN271200477451C (PIs J Tischfield and J Rice). Genotyping services were also provided by the Center for Inherited Disease Research (CIDR). CIDR is fully funded through a federal contract from the National Institutes of Health to The Johns Hopkins University, contract number HHSN268200782096.

## Footnotes

## References

1. Hu MC, Davies M, Kandel DB. Epidemiology and correlates of daily smoking and nicotine dependence among young adults in the United States. Am J Public Health. 2006;96(2):299–308. doi: 10.2105/AJPH.2004.057232.  [DOI](https://doi.org/10.2105/AJPH.2004.057232) | [PMC free article](/articles/PMC1470478/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16380569/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Public%20Health&title=Epidemiology%20and%20correlates%20of%20daily%20smoking%20and%20nicotine%20dependence%20among%20young%20adults%20in%20the%20United%20States&author=MC%20Hu&author=M%20Davies&author=DB%20Kandel&volume=96&issue=2&publication_year=2006&pages=299-308&pmid=16380569&doi=10.2105/AJPH.2004.057232&)

2. Cohen J. Sources of peer group homogeneity. Sociology of Education. 1977;50:227–241.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sociology%20of%20Education&title=Sources%20of%20peer%20group%20homogeneity&author=J%20Cohen&volume=50&publication_year=1977&pages=227-241&)

3. Kandel DB. Homophily, selection, and socialization in adolescent friendships. American Journal of Sociology. 1978;84:427–436.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=American%20Journal%20of%20Sociology&title=Homophily,%20selection,%20and%20socialization%20in%20adolescent%20friendships&author=DB%20Kandel&volume=84&publication_year=1978&pages=427-436&)

4. Billy JOG, Udry JR. Patterns of adolescent friendship and effects on sexual behavior. Social Psychology Quarterly. 1985;48:27–41.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Social%20Psychology%20Quarterly&title=Patterns%20of%20adolescent%20friendship%20and%20effects%20on%20sexual%20behavior&author=JOG%20Billy&author=JR%20Udry&volume=48&publication_year=1985&pages=27-41&)

5. Ennett ST, Bauman KE. The contribution of influence and selection to adolescent peer group homogeneity: the case of adolescent cigarette smoking. J Pers Soc Psychol. 1994;67(4):653–663. doi: 10.1037//0022-3514.67.4.653.  [DOI](https://doi.org/10.1037//0022-3514.67.4.653) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7965611/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pers%20Soc%20Psychol&title=The%20contribution%20of%20influence%20and%20selection%20to%20adolescent%20peer%20group%20homogeneity:%20the%20case%20of%20adolescent%20cigarette%20smoking&author=ST%20Ennett&author=KE%20Bauman&volume=67&issue=4&publication_year=1994&pages=653-663&pmid=7965611&doi=10.1037//0022-3514.67.4.653&)

6. Engels RC, et al. Homogeneity of cigarette smoking within peer groups: influence or selection? Health Educ Behav. 1997;24(6):801–811. doi: 10.1177/109019819702400613.  [DOI](https://doi.org/10.1177/109019819702400613) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9408792/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Health%20Educ%20Behav&title=Homogeneity%20of%20cigarette%20smoking%20within%20peer%20groups:%20influence%20or%20selection?&author=RC%20Engels&volume=24&issue=6&publication_year=1997&pages=801-811&pmid=9408792&doi=10.1177/109019819702400613&)

7. Kobus K. Peers and adolescent smoking. Addiction. 2003;98 Suppl 1:37–55. doi: 10.1046/j.1360-0443.98.s1.4.x.  [DOI](https://doi.org/10.1046/j.1360-0443.98.s1.4.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12752361/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Addiction&title=Peers%20and%20adolescent%20smoking&author=K%20Kobus&volume=98&publication_year=2003&pages=37-55&pmid=12752361&doi=10.1046/j.1360-0443.98.s1.4.x&)

8. Hoffman BR, et al. Peer influences on adolescent cigarette smoking: a theoretical review of the literature. Subst Use Misuse. 2006;41(1):103–155. doi: 10.1080/10826080500368892.  [DOI](https://doi.org/10.1080/10826080500368892) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16393739/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Subst%20Use%20Misuse&title=Peer%20influences%20on%20adolescent%20cigarette%20smoking:%20a%20theoretical%20review%20of%20the%20literature&author=BR%20Hoffman&volume=41&issue=1&publication_year=2006&pages=103-155&pmid=16393739&doi=10.1080/10826080500368892&)

9. Duncan TE, et al. The consistency of family and peer influences on the development of substance use in adolescence. Addiction. 1995;90(12):1647–1660. doi: 10.1046/j.1360-0443.1995.901216477.x.  [DOI](https://doi.org/10.1046/j.1360-0443.1995.901216477.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8555956/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Addiction&title=The%20consistency%20of%20family%20and%20peer%20influences%20on%20the%20development%20of%20substance%20use%20in%20adolescence&author=TE%20Duncan&volume=90&issue=12&publication_year=1995&pages=1647-1660&pmid=8555956&doi=10.1046/j.1360-0443.1995.901216477.x&)

10. Flay BR, et al. Differential influence of parental smoking and friends' smoking on adolescent initiation and escalation of smoking. J Health Soc Behav. 1994;35(3):248–265.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/7983337/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Health%20Soc%20Behav&title=Differential%20influence%20of%20parental%20smoking%20and%20friends'%20smoking%20on%20adolescent%20initiation%20and%20escalation%20of%20smoking&author=BR%20Flay&volume=35&issue=3&publication_year=1994&pages=248-265&pmid=7983337&)

11. Plomin R, DeFries JC, Loehlin JC. Genotype-environment interaction and correlation in the analysis of human behavior. Psychol Bull. 1977;84(2):309–322.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/557211/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Psychol%20Bull&title=Genotype-environment%20interaction%20and%20correlation%20in%20the%20analysis%20of%20human%20behavior&author=R%20Plomin&author=JC%20DeFries&author=JC%20Loehlin&volume=84&issue=2&publication_year=1977&pages=309-322&pmid=557211&)

12. Cleveland HH, Wiebe RP, Rowe DC. Sources of exposure to smoking and drinking friends among adolescents: a behavioral-genetic evaluation. J Genet Psychol. 2005;166(2):153–169.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/15906929/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Genet%20Psychol&title=Sources%20of%20exposure%20to%20smoking%20and%20drinking%20friends%20among%20adolescents:%20a%20behavioral-genetic%20evaluation&author=HH%20Cleveland&author=RP%20Wiebe&author=DC%20Rowe&volume=166&issue=2&publication_year=2005&pages=153-169&pmid=15906929&)

13. Dick DM, et al. Gender differences in friends' influences on adolescent drinking: a genetic epidemiological study. Alcohol Clin Exp Res. 2007;31(12):2012–2019. doi: 10.1111/j.1530-0277.2007.00523.x.  [DOI](https://doi.org/10.1111/j.1530-0277.2007.00523.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17949469/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Alcohol%20Clin%20Exp%20Res&title=Gender%20differences%20in%20friends'%20influences%20on%20adolescent%20drinking:%20a%20genetic%20epidemiological%20study&author=DM%20Dick&volume=31&issue=12&publication_year=2007&pages=2012-2019&pmid=17949469&doi=10.1111/j.1530-0277.2007.00523.x&)

14. Harden KP, et al. Gene-environment correlation and interaction in peer effects on adolescent alcohol and tobacco use. Behav Genet. 2008;38(4):339–347. doi: 10.1007/s10519-008-9202-7.  [DOI](https://doi.org/10.1007/s10519-008-9202-7) | [PMC free article](/articles/PMC2898558/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18368474/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Behav%20Genet&title=Gene-environment%20correlation%20and%20interaction%20in%20peer%20effects%20on%20adolescent%20alcohol%20and%20tobacco%20use&author=KP%20Harden&volume=38&issue=4&publication_year=2008&pages=339-347&pmid=18368474&doi=10.1007/s10519-008-9202-7&)

15. Hill J, et al. Alcohol use in adolescent twins and affiliation with substance using peers. J Abnorm Child Psychol. 2008;36(1):81–94. doi: 10.1007/s10802-007-9161-0.  [DOI](https://doi.org/10.1007/s10802-007-9161-0) | [PMC free article](/articles/PMC2905232/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17665304/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Abnorm%20Child%20Psychol&title=Alcohol%20use%20in%20adolescent%20twins%20and%20affiliation%20with%20substance%20using%20peers&author=J%20Hill&volume=36&issue=1&publication_year=2008&pages=81-94&pmid=17665304&doi=10.1007/s10802-007-9161-0&)

16. Button TM, et al. Perceived peer delinquency and the genetic predisposition for substance dependence vulnerability. Drug Alcohol Depend. 2009;100(1–2):1–8. doi: 10.1016/j.drugalcdep.2008.08.014.  [DOI](https://doi.org/10.1016/j.drugalcdep.2008.08.014) | [PMC free article](/articles/PMC2663024/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19008053/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Alcohol%20Depend&title=Perceived%20peer%20delinquency%20and%20the%20genetic%20predisposition%20for%20substance%20dependence%20vulnerability&author=TM%20Button&volume=100&issue=1%E2%80%932&publication_year=2009&pages=1-8&pmid=19008053&doi=10.1016/j.drugalcdep.2008.08.014&)

17. Bierut LJ, et al. Novel genes identified in a high-density genome wide association study for nicotine dependence. Hum Mol Genet. 2007;16(1):24–35. doi: 10.1093/hmg/ddl441.  [DOI](https://doi.org/10.1093/hmg/ddl441) | [PMC free article](/articles/PMC2278047/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17158188/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=Novel%20genes%20identified%20in%20a%20high-density%20genome%20wide%20association%20study%20for%20nicotine%20dependence&author=LJ%20Bierut&volume=16&issue=1&publication_year=2007&pages=24-35&pmid=17158188&doi=10.1093/hmg/ddl441&)

18. Saccone SF, et al. Cholinergic nicotinic receptor genes implicated in a nicotine dependence association study targeting 348 candidate genes with 3713 SNPs. Hum Mol Genet. 2007;16(1):36–49. doi: 10.1093/hmg/ddl438.  [DOI](https://doi.org/10.1093/hmg/ddl438) | [PMC free article](/articles/PMC2270437/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17135278/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=Cholinergic%20nicotinic%20receptor%20genes%20implicated%20in%20a%20nicotine%20dependence%20association%20study%20targeting%20348%20candidate%20genes%20with%203713%20SNPs&author=SF%20Saccone&volume=16&issue=1&publication_year=2007&pages=36-49&pmid=17135278&doi=10.1093/hmg/ddl438&)

19. Bierut LJ, et al. Variants in nicotinic receptors and risk for nicotine dependence. Am J Psychiatry. 2008;165(9):1163–1171. doi: 10.1176/appi.ajp.2008.07111711.  [DOI](https://doi.org/10.1176/appi.ajp.2008.07111711) | [PMC free article](/articles/PMC2574742/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18519524/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Psychiatry&title=Variants%20in%20nicotinic%20receptors%20and%20risk%20for%20nicotine%20dependence&author=LJ%20Bierut&volume=165&issue=9&publication_year=2008&pages=1163-1171&pmid=18519524&doi=10.1176/appi.ajp.2008.07111711&)

20. Stevens VL, et al. Nicotinic receptor gene variants influence susceptibility to heavy smoking. Cancer Epidemiol Biomarkers Prev. 2008;17(12):3517–3525. doi: 10.1158/1055-9965.EPI-08-0585.  [DOI](https://doi.org/10.1158/1055-9965.EPI-08-0585) | [PMC free article](/articles/PMC2614129/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19029397/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Epidemiol%20Biomarkers%20Prev&title=Nicotinic%20receptor%20gene%20variants%20influence%20susceptibility%20to%20heavy%20smoking&author=VL%20Stevens&volume=17&issue=12&publication_year=2008&pages=3517-3525&pmid=19029397&doi=10.1158/1055-9965.EPI-08-0585&)

21. Berrettini W, et al. Alpha-5/alpha-3 nicotinic receptor subunit alleles increase risk for heavy smoking. Mol Psychiatry. 2008;13(4):368–373. doi: 10.1038/sj.mp.4002154.  [DOI](https://doi.org/10.1038/sj.mp.4002154) | [PMC free article](/articles/PMC2507863/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18227835/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Psychiatry&title=Alpha-5/alpha-3%20nicotinic%20receptor%20subunit%20alleles%20increase%20risk%20for%20heavy%20smoking&author=W%20Berrettini&volume=13&issue=4&publication_year=2008&pages=368-373&pmid=18227835&doi=10.1038/sj.mp.4002154&)

22. Thorgeirsson TE, et al. A variant associated with nicotine dependence, lung cancer and peripheral arterial disease. Nature. 2008;452(7187):638–642. doi: 10.1038/nature06846.  [DOI](https://doi.org/10.1038/nature06846) | [PMC free article](/articles/PMC4539558/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18385739/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=A%20variant%20associated%20with%20nicotine%20dependence,%20lung%20cancer%20and%20peripheral%20arterial%20disease&author=TE%20Thorgeirsson&volume=452&issue=7187&publication_year=2008&pages=638-642&pmid=18385739&doi=10.1038/nature06846&)

23. Saccone NL, et al. Multiple distinct risk loci for nicotine dependence identified by dense coverage of the complete family of nicotinic receptor subunit (CHRN) genes. Am J Med Genet B Neuropsychiatr Genet. 2009;150B(4):453–466. doi: 10.1002/ajmg.b.30828.  [DOI](https://doi.org/10.1002/ajmg.b.30828) | [PMC free article](/articles/PMC2693307/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19259974/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Med%20Genet%20B%20Neuropsychiatr%20Genet&title=Multiple%20distinct%20risk%20loci%20for%20nicotine%20dependence%20identified%20by%20dense%20coverage%20of%20the%20complete%20family%20of%20nicotinic%20receptor%20subunit%20(CHRN)%20genes&author=NL%20Saccone&volume=150B&issue=4&publication_year=2009&pages=453-466&pmid=19259974&doi=10.1002/ajmg.b.30828&)

24. Heatherton TF, et al. The Fagerstrom Test for Nicotine Dependence: a revision of the Fagerstrom Tolerance Questionnaire. Br J Addict. 1991;86(9):1119–1127. doi: 10.1111/j.1360-0443.1991.tb01879.x.  [DOI](https://doi.org/10.1111/j.1360-0443.1991.tb01879.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1932883/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Addict&title=The%20Fagerstrom%20Test%20for%20Nicotine%20Dependence:%20a%20revision%20of%20the%20Fagerstrom%20Tolerance%20Questionnaire&author=TF%20Heatherton&volume=86&issue=9&publication_year=1991&pages=1119-1127&pmid=1932883&doi=10.1111/j.1360-0443.1991.tb01879.x&)

25. Price AL, et al. Principal components analysis corrects for stratification in genome-wide association studies. Nat Genet. 2006;38(8):904–909. doi: 10.1038/ng1847.  [DOI](https://doi.org/10.1038/ng1847) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16862161/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Principal%20components%20analysis%20corrects%20for%20stratification%20in%20genome-wide%20association%20studies&author=AL%20Price&volume=38&issue=8&publication_year=2006&pages=904-909&pmid=16862161&doi=10.1038/ng1847&)

26. Allen KF, et al. Teenage tobacco use: Data estimates from the Teenage Attitudes and Practices Survey: United States. Hyattsville, MD: National Center for Health Statistics; 1989. p. 1992. (Advance data no. 224)  [DOI](https://doi.org/10.1037/e608902007-001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10125953/) | [Google Scholar](https://scholar.google.com/scholar_lookup?title=Teenage%20tobacco%20use:%20Data%20estimates%20from%20the%20Teenage%20Attitudes%20and%20Practices%20Survey:%20United%20States&author=KF%20Allen&publication_year=1989&)

27. Mayhew KP, Flay BR, Mott JA. Stages in the development of adolescent smoking. Drug Alcohol Depend. 2000;59 Suppl 1:S61–S81. doi: 10.1016/s0376-8716(99)00165-9.  [DOI](https://doi.org/10.1016/s0376-8716(99)00165-9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10773438/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Alcohol%20Depend&title=Stages%20in%20the%20development%20of%20adolescent%20smoking&author=KP%20Mayhew&author=BR%20Flay&author=JA%20Mott&volume=59&publication_year=2000&pages=S61-S81&pmid=10773438&doi=10.1016/s0376-8716(99)00165-9&)

28. Hosmer DW, Lemeshow S. Applied logistic regression. 2nd ed. xii. New York: Wiley: Wiley series in probability and statistics; 2000. p. 373. Texts and references section.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Applied%20logistic%20regression&author=DW%20Hosmer&author=S%20Lemeshow&publication_year=2000&)

29. Olkin I, Finn JD. Correlations Redux. Psychological Bulletin. 1995;118(1):155–164.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Psychological%20Bulletin&title=Correlations%20Redux&author=I%20Olkin&author=JD%20Finn&volume=118&issue=1&publication_year=1995&pages=155-164&)

30. Mukherjee B, et al. Tests for gene-environment interaction from case-control data: a novel study of type I error, power and designs. Genet Epidemiol. 2008;32(7):615–626. doi: 10.1002/gepi.20337.  [DOI](https://doi.org/10.1002/gepi.20337) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18473390/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genet%20Epidemiol&title=Tests%20for%20gene-environment%20interaction%20from%20case-control%20data:%20a%20novel%20study%20of%20type%20I%20error,%20power%20and%20designs&author=B%20Mukherjee&volume=32&issue=7&publication_year=2008&pages=615-626&pmid=18473390&doi=10.1002/gepi.20337&)

31. Long JS, Freese J. Regression Models for Categorical Dependent Variables using Stata. College Station, TX: Stata Press; 2001.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Regression%20Models%20for%20Categorical%20Dependent%20Variables%20using%20Stata&author=JS%20Long&author=J%20Freese&publication_year=2001&)

32. Dick DM, et al. Changing environmental influences on substance use across development. Twin Res Hum Genet. 2007;10(2):315–326. doi: 10.1375/twin.10.2.315.  [DOI](https://doi.org/10.1375/twin.10.2.315) | [PMC free article](/articles/PMC1905859/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17564520/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Twin%20Res%20Hum%20Genet&title=Changing%20environmental%20influences%20on%20substance%20use%20across%20development&author=DM%20Dick&volume=10&issue=2&publication_year=2007&pages=315-326&pmid=17564520&doi=10.1375/twin.10.2.315&)

33. Chen L, et al. Interplay of Genetic Risk Factors and Parent Monitoring in Risk for Nicotine Dependence. Addiction. 2009;104(10):1731–1740. doi: 10.1111/j.1360-0443.2009.02697.x.  [DOI](https://doi.org/10.1111/j.1360-0443.2009.02697.x) | [PMC free article](/articles/PMC2943646/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20871796/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Addiction&title=Interplay%20of%20Genetic%20Risk%20Factors%20and%20Parent%20Monitoring%20in%20Risk%20for%20Nicotine%20Dependence&author=L%20Chen&volume=104&issue=10&publication_year=2009&pages=1731-1740&pmid=20871796&doi=10.1111/j.1360-0443.2009.02697.x&)

34. Kaprio J. Commentary on Chen et al. (2009): Gene-environment interactions in nicotine dependence. Addiction. 2009;104(10):1741–1742. doi: 10.1111/j.1360-0443.2009.02750.x.  [DOI](https://doi.org/10.1111/j.1360-0443.2009.02750.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21265910/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Addiction&title=Commentary%20on%20Chen%20et%20al.%20(2009):%20Gene-environment%20interactions%20in%20nicotine%20dependence&author=J%20Kaprio&volume=104&issue=10&publication_year=2009&pages=1741-1742&pmid=21265910&doi=10.1111/j.1360-0443.2009.02750.x&)

35. Munafo MR. Reliability and replicability of genetic association studies. Addiction. 2009;104(9):1439–1440. doi: 10.1111/j.1360-0443.2009.02662.x.  [DOI](https://doi.org/10.1111/j.1360-0443.2009.02662.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19686515/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Addiction&title=Reliability%20and%20replicability%20of%20genetic%20association%20studies&author=MR%20Munafo&volume=104&issue=9&publication_year=2009&pages=1439-1440&pmid=19686515&doi=10.1111/j.1360-0443.2009.02662.x&)

36. Gauderman W, Morrison J. 2007 QUANTO 1.2.3: A computer program for power and sample size calculations for genetic-epidemiology studies, http://hydra.usc.edu/gxe.  [http://hydra.usc.edu/gxe](http://hydra.usc.edu/gxe)
