# Estimation of the Warfarin Dose with Clinical and Pharmacogenetic Data

## Metadata
**Authors:** The International Warfarin Pharmacogenetics Consortium
**Journal:** The New England journal of medicine
**Date:** 2009 Feb 19
**DOI:** [10.1056/NEJMoa0809329](https://doi.org/10.1056/NEJMoa0809329)
**PMID:** 19228618
**PMCID:** PMC2722908
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2722908/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC2722908/pdf/ukmss-5142.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC2722908/pdf/ukmss-5142.pdf)

## Abstract

**BACKGROUND:** 
Genetic variability among patients plays an important role in determining the dose of warfarin that should be used when oral anticoagulation is initiated, but practical methods of using genetic information have not been evaluated in a diverse and large population. We developed and used an algorithm for estimating the appropriate warfarin dose that is based on both clinical and genetic data from a broad population base.

**METHODS:** 
Clinical and genetic data from 4043 patients were used to create a dose algorithm that was based on clinical variables only and an algorithm in which genetic information was added to the clinical variables. In a validation cohort of 1009 subjects, we evaluated the potential clinical value of each algorithm by calculating the percentage of patients whose predicted dose of warfarin was within 20% of the actual stable therapeutic dose; we also evaluated other clinically relevant indicators.

**RESULTS:** 
In the validation cohort, the pharmacogenetic algorithm accurately identified larger proportions of patients who required 21 mg of warfarin or less per week and of those who required 49 mg or more per week to achieve the target international normalized ratio than did the clinical algorithm (49.4% vs. 33.3%, P<0.001, among patients requiring ≤21 mg per week; and 24.8% vs. 7.2%, P<0.001, among those requiring ≥49 mg per week).

**CONCLUSIONS:** 
The use of a pharmacogenetic algorithm for estimating the appropriate initial dose of warfarin produces recommendations that are significantly closer to the required stable therapeutic dose than those derived from a clinical algorithm or a fixed-dose approach. The greatest benefits were observed in the 46.2% of the population that required 21 mg or less of warfarin per week or 49 mg or more per week for therapeutic anticoagulation.

### BACKGROUND

Genetic variability among patients plays an important role in determining the dose of warfarin that should be used when oral anticoagulation is initiated, but practical methods of using genetic information have not been evaluated in a diverse and large population. We developed and used an algorithm for estimating the appropriate warfarin dose that is based on both clinical and genetic data from a broad population base.

### METHODS

Clinical and genetic data from 4043 patients were used to create a dose algorithm that was based on clinical variables only and an algorithm in which genetic information was added to the clinical variables. In a validation cohort of 1009 subjects, we evaluated the potential clinical value of each algorithm by calculating the percentage of patients whose predicted dose of warfarin was within 20% of the actual stable therapeutic dose; we also evaluated other clinically relevant indicators.

### RESULTS

In the validation cohort, the pharmacogenetic algorithm accurately identified larger proportions of patients who required 21 mg of warfarin or less per week and of those who required 49 mg or more per week to achieve the target international normalized ratio than did the clinical algorithm (49.4% vs. 33.3%, P<0.001, among patients requiring ≤21 mg per week; and 24.8% vs. 7.2%, P<0.001, among those requiring ≥49 mg per week).

### CONCLUSIONS

The use of a pharmacogenetic algorithm for estimating the appropriate initial dose of warfarin produces recommendations that are significantly closer to the required stable therapeutic dose than those derived from a clinical algorithm or a fixed-dose approach. The greatest benefits were observed in the 46.2% of the population that required 21 mg or less of warfarin per week or 49 mg or more per week for therapeutic anticoagulation.

## METHODS

### DATA COLLECTION AND STUDY COHORTS

The International Warfarin Pharmacogenetics Consortium comprises 21 research groups from 9 countries and 4 continents. The research groups contributed clinical and genetic data for a total of 5700 patients who were treated with warfarin. These data were curated (i.e., collected, formatted, and subjected to quality control) by staff at the Pharmacogenetics and Pharmacogenomics Knowledge Base (PharmGKB, [www.pharmgkb.org](http://www.pharmgkb.org)) and by members of the consortium. The cohort whose data were analyzed for this study included the subgroup of 5052 patients who had a target international normalized ratio (INR) of 2 to 3. The requirement for informed consent was waived because consent had been obtained previously by each participating center, and only de-identified data were used in the study.

We collected data on clinical factors that have previously been associated with warfarin therapy and that were available from the information received from all or most sites. These data included information on demographic characteristics, the primary indication for warfarin treatment, the stable therapeutic dose of warfarin, the treatment INR (the INR achieved with a stable warfarin dose), the target INR (the desired INR), the use of concomitant medications (grouped according to those that increase and those that decrease the INR), and the presence of genotype variants of *CYP2C9* (*1, *2 and *3) and *VKORC1* (at least one of seven single nucleotide polymorphisms [SNPs] in linkage disequilibrium[11](#R11)), as detailed in Section 1 in [Supplementary Appendix 1](#SD1), available with the full text of this article at [NEJM.org](http://NEJM.org). Information on race or ethnic group was reported by the patient or determined by the local investigator. Several potentially important variables (e.g., vitamin K intake and smoking status) were not consistently available and thus were not included. Data on adverse events such as thromboembolic events or bleeding or the need for repeated measurements of the INR before a stable dose was achieved were not available for this study. The outcome variable was the stable therapeutic dose of warfarin, defined as the steady-state dose that led to stable anticoagulation levels. Although the centers used different definitions for steady-state dose, most centers required stable levels of anticoagulation (i.e., INR) over a period during which the dose of warfarin was stable (Section 2 in [Supplementary Appendix 1](#SD1)).

### GENOTYPE QUALITY CONTROL

The National Genotyping Center at the Institute of Biomedical Sciences, Academia Sinica, Taipei, Taiwan, performed blinded regenotyping for quality control on DNA samples from 10% of the subjects from each site (except for sites 3, 9, and 14, which could not ship DNA samples internationally). DNA samples that were used for quality control were genotyped for the two polymorphisms in *CYP2C9* (*2=rs1799853, and *3=rs1057910) and seven SNPs in *VKORC1* (−1639 G→A=rs9923231, 1173 C→T=rs9934438, 497 T→G=rs2884737, 1542 G→C=rs8050894, 3730 G→A=rs7294, 2255 C→T=rs2359612, −4451 C→A=rs17880887). *CYP2C9* *2 and *3 and *VKORC1* rs9923231 were genotyped with the use of the TaqMan allelic discrimination assay (Applied Biosystems), whereas the remaining *VKORC1* SNPs were genotyped by mass spectrometry with the use of MassARRAY (Sequenom). Both methods had been validated previously at the National Genotyping Center with the use of direct sequencing and denaturing high-performance liquid chromatography, with 100% concordance on 200 samples. The complete data set of genotypes and clinical variables, as well as the full genotype quality-control data, is available to registered PharmGKB users at [www.pharmgkb.org](http://www.pharmgkb.org) (full data set accession number, PA162355460).

### STATISTICAL ANALYSIS

We randomly chose 80% of the eligible patients, (stratified according to site, for a total of 4043 patients who had a stable dose of warfarin and a target INR of 2 to 3) as the “derivation cohort” for developing all dose-prediction models. The remaining 20% of the patients (1009 patients, from all 21 sites) constituted the “validation cohort,” which was used for testing the final selected model. The investigators who performed the modeling and analysis did not have access to this validation set until after the final model was selected. A wide variety of numerical modeling methods were used for the data from the derivation cohort, including, but not limited to, support vector regression, regression trees, model trees, multivariate adaptive regression splines, least-angle regression, and Lasso, in addition to ordinary linear regression. Logarithmic and square-root transformations of doses were tested, in addition to a direct prediction of dose. Further details of the statistical modeling approaches that were tested and the evaluation methods that were used for selecting the best model from the derivation cohort are described in Section 3 in [Supplementary Appendix 1](#SD1).

Missing values for the *VKORC1* SNP rs9923231 were imputed on the basis of race and on the basis of the *VKORC1* SNP data at rs2359612, rs9934438, or rs8050894 (see Section 4 in [Supplementary Appendix 1](#SD1)). If the *VKORC1* genotype could not be imputed, it was treated as “missing” (a distinct variable) in the model.

The mean absolute error — that is, the mean of the absolute values for the difference between the predicted and actual maintenance doses — was used to evaluate each model's predictive accuracy. For models developed with the use of transformed data, the mean absolute error was computed in the original units rather than in the transformed units to allow a fair comparison of all models. We selected the final model as the one that had the lowest predictive mean absolute error as estimated by 10-fold cross-validation on the derivation cohort (detailed in Section 3 in [Supplementary Appendix 1](#SD1)). Using the validation data set, we compared dose predictions from the pharmacogenetic model with those from two other models: a clinical model that did not include genetic factors and a model with a fixed dose of 5 mg of warfarin per day. The clinical model was built with the use of the same methods as the pharmacogenetic model, but without the incorporation of genetic variables. The following assessment of clinical significance is based on this validation data set except where otherwise stated.

The mean absolute error and the coefficient of determination (R^2^) in the validation data set were our prespecified metrics for evaluating the pharmacogenetic, clinical, and fixed-dose models. These models were selected before the metrics were computed; thus, there were no multiple comparisons. We evaluated the potential clinical value of each algorithm by calculating the percentage of patients whose predicted dose of warfarin was within 20% of the actual stable therapeutic dose. In addition, we calculated the percentage of patients for whom the predicted dose according to each algorithm was at least 20% higher than the actual dose (overestimation) or at least 20% lower than the actual dose (underestimation). These values represent a difference of 1 mg per day relative to the traditional starting dose of 5 mg per day, a difference clinicians would be likely to define as clinically relevant. We also assessed the performance of the algorithms in three dose groups: participants requiring a low dose (≤21 mg per week), those requiring a high dose (≥49 mg per week), and those requiring intermediate doses (>21 and <49 mg per week) for stable therapeutic anticoagulation. These thresholds of 21 mg and 49 mg per week bracket the usual starting dose of 35 mg per week (5 mg per day). Patients requiring doses of less than 21 mg per week would be at risk for excessive anticoagulation if they were started on the standard dose of 35 mg per week. Conversely, patients requiring doses of more than 49 mg per week would be at risk for inadequate anticoagulation if they were started on a dose of 35 mg per week. We performed sensitivity analyses using other dose thresholds as well (see Section 5 in [Supplementary Appendix 1](#SD1)). We assessed the potential benefit of using the pharmacogenetic algorithm instead of the clinical algorithm or the fixed-dose model and computed the number needed to genotype (i.e., the number of patients who must be genotyped in order for one patient to have an improved dose estimate).[24](#R24) Finally, in a post hoc analysis, we assessed how well the algorithms predicted which patients actually required low or high doses.

## RESULTS

### CHARACTERISTICS OF THE PATIENTS AND QUALITY OF GENOTYPING DATA

The characteristics of the patients are shown in [Table 1](#T1). Genotype quality control was conducted on 480 samples (8.4% of the data set). The overall concordance for *CYP2C9* SNPs in the genotype quality control was 97.8%, and the overall concordance for *VKORC1* SNPs was 97.7%. Exclusion of the three sites that did not participate in the quality-control genotyping did not change the model. Each SNP was in Hardy–Weinberg equilibrium when it was tested in patients who were stratified according to race.

### Table 1.

| Variable | Derivation Cohort (N = 4043) | Validation Cohort (N = 1009) | P Value* |
| --- | --- | --- | --- |
| Warfarin dose — mg/wk |   |   | 0.40 |
| Median | 28.0 | 28.0 |   |
| Interquartile range | 19.0–38.5 | 21.0–38.5 |   |
| Genotype — no. (%) |   |   |   |
| VKORC1 rs9923231 |   |   | 0.97 |
| G/G | 1201 (29.7) | 302 (29.9) |   |
| A/G | 1444 (35.7) | 363 (36.0) |   |
| A/A | 1315 (32.5) | 326 (32.3) |   |
| Unknown | 83 (2.1) | 18 (1.8) |   |
| CYP2C9 † |   |   | 0.38 |
| *1/*1 | 2970 (73.5) | 749 (74.2) |   |
| *1/*2 | 509 (12.6) | 142 (14.1) |   |
| *1/*3 | 374 (9.3) | 76 (7.5) |   |
| *2/*2 | 36 (0.9) | 10 (1.0) |   |
| *2/*3 | 52 (1.3) | 10 (1.0) |   |
| *3/*3 | 15 (0.4) | 1 (0.1) |   |
| Unknown | 87 (2.2) | 21 (2.1) |   |
| Age — no. (%) |   |   | 0.88 |
| 10–19 yr | 11 (0.3) | 1 (0.1) |   |
| 20–29 yr | 80 (2.0) | 18 (1.8) |   |
| 30–39 yr | 145 (3.6) | 43 (4.3) |   |
| 40–49 yr | 363 (9.0) | 101 (10.0) |   |
| 50–59 yr | 753 (18.6) | 189 (18.7) |   |
| 60–69 yr | 1016 (25.1) | 239 (23.7) |   |
| 70–79 yr | 1151 (28.5) | 289 (28.6) |   |
| 80–89 yr | 497 (12.3) | 124 (12.3) |   |
| ≥90 yr | 27 (0.7) | 5 (0.5) |   |
| Height — m |   |   | 0.79 |
| Median | 1.68 | 1.68 |   |
| Interquartile range | 1.60–1.76 | 1.60–1.76 |   |
| Weight — kg |   |   | 0.52 |
| Median | 75.3 | 75.4 |   |
| Interquartile range | 62.0–89.4 | 63.0–90.0 |   |
| Race — no. (%)‡ |   |   | 0.68 |
| White | 2233 (55.2) | 562 (55.7) |   |
| Asian | 1229 (30.4) | 300 (29.7) |   |
| Black | 353 (8.7) | 97 (9.6) |   |
| Mixed, or missing data | 228 (5.6) | 50 (5.0) |   |
| Use of enzyme inducers — no. (%)§ | 41 (1.0) | 7 (0.7) | 0.35 |
| Use of amiodarone — no. (%) | 176 (4.4) | 56 (5.6) | 0.10 |

Table 1 Caption: Demographic and Clinical Characteristics of the Derivation and Validation Cohorts

### MODELING APPROACHES

An ordinary least-squares linear regression modeling method to develop a pharmacogenetic algorithm that predicts the square root of the dose and incorporates both genetic and clinical data proved to be the best modeling approach for these data (see Section 3 in [Supplementary Appendix 1](#SD1); the actual model is shown in Section 1 in [Supplementary Appendix 1](#SD1)). This approach performed best according to our criterion of the lowest mean absolute error but would also have been selected if the criterion had been the lowest root mean squared error. Although it may be somewhat surprising that ordinary least-squares regression would perform best according to the mean absolute error, the reason, in part, appears to be that minimizing the squared error in the prediction of the square root of the dose effectively minimizes the mean absolute error. This strong performance of ordinary least-squares regression was fortuitous because it yielded a simpler and more easily understood model than many of the more complex modeling approaches we used. Because the resulting dose algorithm computes the square root of the weekly dose, the output must be squared to compute the weekly dose. This algorithm is available at [www.warfarindosing.org](http://www.warfarindosing.org) and as a Microsoft Excel workbook (see [Supplementary Appendix 2](#SD2) and Section 6 in [Supplementary Appendix 1](#SD1)). With the use of the same linear regression approach, we also constructed an algorithm that included clinical variables only, with no incorporation of genetic data (see Section 1 in [Supplementary Appendix 1](#SD1)).

### DOSES PREDICTED BY THE THREE MODELS

The performance of the pharmacogenetic, clinical, and fixed-dose models in the derivation and validation cohorts is shown in [Table 2](#T2). The pharmacogenetic algorithm provided dose estimates that were significantly closer to the actual doses required than the estimates derived from the clinical algorithm or the fixed-dose approach, as evidenced by a mean absolute error that was lower than that for both the clinical algorithm and the fixed-dose approach (8.5±1.7 mg per week vs. 9.9±1.9 mg per week and 13.0±2.3 mg per week, respectively; P<0.001 for both comparisons). [Figure 1](#F1) shows comparisons of the predicted doses according to representative clinical or demographic characteristics, genotype combinations, race, and use or nonuse of amiodarone (an important interacting drug). These data show the way in which the addition of genetic information altered the dose prediction from the clinical model and suggest that most of the racial differences in dose requirements are explained by genotype. The addition of genetic information to clinical information decreased the absolute error in the dose estimate and increased the fraction of variability explained (R^2^). The R^2^ values and the P values for each factor in the two models are shown in Section 7 in [Supplementary Appendix 1](#SD1); a graph of the predicted warfarin dose based on the pharmacogenetic algorithm as compared with the observed dose in the validation cohort is shown in Section 8 in [Supplementary Appendix 1](#SD1).

### Table 2.

| Prediction Model | Derivation Cohort |  | Validation Cohort† |  |
| --- | --- | --- | --- | --- |
|   | Mean Absolute Error mg/wk | R2 % | Mean Absolute Error mg/wk | R2 % |
| Pharmacogenetic algorithm‡§ | 8.3±1.7 | 47 | 8.5±1.7 | 43 |
| Clinical algorithm§ | 10.0±2.0 | 27 | 9.9±1.9 | 26 |
| Fixed-dose approach¶ | 13.3±2.4 | 0 | 13.0±2.3 | 0 |

Table 2 Caption: Predicted Warfarin Doses with the Pharmacogenetic Algorithm, Clinical Algorithm, and Fixed-Dose Approach as Compared with the Actual Stable Dose in the Derivation and Validation Cohorts.*

### Figure 1. Comparisons of Warfarin Doses Predicted According to the Clinical Algorithm and the Pharmacogenetic Algorithm.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e60e/2722908/6569c405767f/ukmss-5142-f0001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=2722908_ukmss-5142-f0001.jpg)

Panel A shows the comparisons based on genotype and on use or nonuse of amiodarone. This example of a 50-year-old non-Asian, nonblack patient who is 1.75 m tall and weighs 80 kg shows that genotype can markedly change the recommended dose from more than 45 mg per week to less than 10 mg per week when all other factors are the same. Panel B shows the comparison based on race and genotype. This example of a 50-year-old patient who is 1.75 m tall and weighs 80 kg shows that racial differences in the estimated dose become insignificant when genetic information is added to the model and that the clinical algorithm, as compared with the pharmacogenetic algorithm, often produces an average value that may substantially overestimate or underestimate the dose.

We also derived specific models for different racial and ethnic groups, but in all cases, the general model that was adjusted for race performed better than these specific models. The performance at individual centers reflected the racial and ethnic makeup of the local patient population, with no influential outliers, a finding that was consistent with the genotype quality-control data. The pharmacogenetic model performed well for each of the three major ethnic groups and for each of the 21 sites (Section 9 in [Supplementary Appendix 1](#SD1)).

### CLINICAL RELEVANCE

The pharmacogenetic algorithm provided more accurate dose estimates than the clinical algorithm or the fixed-dose approach ([Table 2](#T2)). The differences in the performance of the three approaches in the low-dose (≤21 mg per week), intermediate-dose (>21 and <49 mg per week), and high-dose (≥49 mg per week) groups are shown in [Figure 2](#F2) and [Table 3](#T3). For patients who required 21 mg or less of warfarin per week (33.9% of the total cohort), the pharmacogenetic algorithm provided a significantly better prediction of dose than the clinical algorithm or the fixed-dose approach; 35% of the dose predictions fell within 20% of the actual dose (“ideal dose”) with the pharmacogenetic algorithm as compared with 24% with the clinical algorithm (P<0.001) and 0% with the fixed-dose approach (P<0.001). In addition, the pharmacogenetic algorithm provided significantly fewer overestimations of dose in the low-dose group (59.7%, vs. 74.8% with the clinical algorithm [P<0.001]; and 100% with the fixed-dose approach [P<0.001]) ([Table 3](#T3)). Similarly, for patients requiring 49 mg or more per week (12.4% of the total cohort), the pharmacogenetic algorithm predicted doses in the ideal range for significantly more patients than did the clinical algorithm or the fixed-dose approach (32.8% vs. 13.3% and 0%, respectively; P<0.001 for both comparisons), with significantly fewer dose underestimations (66.7% vs. 86.2% and 100%, respectively; P<0.001 for both comparisons). In the intermediate-dose group, the accuracy of the dose prediction was similar with the three approaches. A sensitivity analysis that used different dose thresholds (Section 5 in [Supplementary Appendix 1](#SD1)) highlighted the fact that the pharmacogenetic algorithm provided consistently better dose prediction. [Table 4](#T4) shows the comparison of the pharmacogenetic and clinical algorithms with respect to dose prediction for patients who required low or high doses — an important feature for clinicians when they are initiating treatment. Specifically, for the overall cohort, the pharmacogenetic algorithm correctly predicted low doses for 54% of all patients who required low doses, as compared with the clinical algorithm, which predicted low doses for 33% of these patients (P<0.001). Similarly, the pharmacogenetic algorithm accurately predicted high doses for 26% of patients who required high doses, as compared with the clinical algorithm, which predicted high doses for 9% of these patients (P<0.001). Thus, the pharmacogenetic algorithm significantly improved the dose prediction for patients who required either high or low doses of warfarin, a group that accounted for 46% of the entire cohort.

### Figure 2. Percentage of Patients with Dose Estimates within 20% of the Actual Dose, as Derived with the Use of a Pharmacogenetic Algorithm, a Clinical Algorithm, and a Fixed-Dose Approach.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e60e/2722908/553b4ee89662/ukmss-5142-f0002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=2722908_ukmss-5142-f0002.jpg)

The dose estimates are shown according to three actual-dose groups: low-dose (≤21 mg per week), intermediate-dose (>21 to <49 mg per week), and high-dose (≥49 mg per week). The fixed dose was 35 mg per week. With the fixed-dose approach, none of the estimates for the patients in the low-dose and high-dose groups were within 20% of the actual dose. Panel A shows data for the validation cohort (1009 patients), and Panel B for the derivation-plus-validation cohorts (5052 patients).

### Table 3.

| Actual Dose Required | No. of Patients | Ideal Dose |  | Underestimation |  | Overestimation |  |
| --- | --- | --- | --- | --- | --- | --- | --- |
|   |   | Percent | P Value† | Percent | P Value† | Percent | P Value† |
| Validation cohort only |  |  |  |  |  |  |  |
| ≤21 mg/wk | 324 |   |   |   |   |   |   |
| Pharmacogenetic approach |   | 33.0 | 0.008, <0.001 | 4.6 | 0.002, <0.001 | 62.3 | <0.001, <0.001 |
| Clinical approach |   | 25.9 | <0.001 | 0.6 | 0.50 | 73.5 | <0.001 |
| Fixed-dose approach |   | 0 |   | 0 |   | 100 |   |
| >21 mg/wk to <49 mg/wk | 560 |   |   |   |   |   |   |
| Pharmacogenetic approach |   | 54.6 | 0.72, 0.31 | 26.8 | 0.14, <0.001 | 18.6 | 0.25, <0.001 |
| Clinical approach |   | 53.6 | 0.55 | 29.8 | <0.001 | 16.6 | <0.001 |
| Fixed-dose approach |   | 51.6 |   | 9.1 |   | 39.3 |   |
| ≥49 mg/wk | 125 |   |   |   |   |   |   |
| Pharmacogenetic approach |   | 36.8 | <0.001, <0.001 | 63.2 | <0.001, <0.001 | 0 | 1.00, 1.00 |
| Clinical approach |   | 9.6 | <0.001 | 89.6 | <0.001 | 0.8 | 1.00 |
| Fixed-dose approach |   | 0 |   | 100 |   | 0 |   |
| Derivation-plus-validation cohort |  |  |  |  |  |  |  |
| ≤21 mg/wk | 1711 |   |   |   |   |   |   |
| Pharmacogenetic approach |   | 35.0 | <0.001, <0.001 | 5.4 | <0.001, <0.001 | 59.7 | <0.001, <0.001 |
| Clinical approach |   | 24.0 | <0.001 | 1.2 | <0.001 | 74.8 | <0.001 |
| Fixed-dose approach |   | 0 |   | 0 |   | 100 |   |
| >21 mg/wk to <49 mg/wk | 2716 |   |   |   |   |   |   |
| Pharmacogenetic approach |   | 55.9 | 0.02, 0.08 | 25.9 | <0.001, <0.001 | 18.2 | <0.001, <0.001 |
| Clinical approach |   | 53.3 | 0.80 | 31.0 | <0.001 | 15.6 | <0.001 |
| Fixed-dose approach |   | 53.7 |   | 8.9 |   | 37.4 |   |
| ≥49 mg/wk | 625 |   |   |   |   |   |   |
| Pharmacogenetic approach |   | 32.8 | <0.001, <0.001 | 66.7 | <0.001, <0.001 | 0.5 | 1.00, 0.25 |
| Clinical approach |   | 13.3 | <0.001 | 86.2 | <0.001 | 0.5 | 0.25 |
| Fixed-dose approach |   | 0 |   | 100 |   | 0 |   |

Table 3 Caption: Percentage of Patients in the Validation Cohort and in the Derivation-plus-Validation Cohort with an Ideal, Underestimated, or Overestimated Dose of Warfarin, as Estimated with the Pharmacogenetic Algorithm, Clinical Algorithm, and Fixed-Dose Approach in Patients Requiring Low, Intermediate, or High Actual Doses of Warfarin for a Therapeutic Effect.*

### Table 4.

| Actual or Predicted Dose Group | ≤21 mg/wk |  | ≥49 mg/wk |  |
| --- | --- | --- | --- | --- |
|   | Pharmacogenetic Algorithm | Clinical Algorithm | Pharmacogenetic Algorithm | Clinical Algorithm |
| Validation cohort |  |  |  |  |
| No. of patients predicted to require extreme dose | 232 | 162 | 48 | 15 |
| No. of patients actually requiring extreme dose | 324 | 324 | 125 | 125 |
| No. of patients correctly predicted to require extreme dose | 160* | 108 | 31* | 9 |
| Percent of patients correctly predicted to require extreme dose† | 49.4 | 33.3 | 24.8 | 7.2 |
| Derivation-plus-validation cohort |  |  |  |  |
| No. of patients predicted to require extreme dose | 1250 | 829 | 243 | 98 |
| No. of patients actually requiring extreme dose | 1711 | 1711 | 625 | 625 |
| No. of patients correctly predicted to require extreme dose | 929* | 571 | 165 | 57 |
| Percent of patients correctly predicted to require extreme dose† | 54.3 | 33.4 | 26.4 | 9.1 |

Table 4 Caption: Ability of the Pharmacogenetic and Clinical Algorithms to Correctly Identify Patients Requiring Low or High Doses of Warfarin.

Overall, estimates of dose derived with the use of the pharmacogenetic algorithm were closer to the actual dose than were estimates derived from the clinical algorithm for 60% of the patients and were closer than the fixed dose for 69% of the patients. To estimate the potential for a clinically meaningful improvement in dose prediction, we assessed the number of patients for whom one algorithm estimated a dose within 20% of the actual dose and the other algorithm did not — a stricter standard than simply the proximity to the actual dose. For the entire cohort, the number of patients needed to genotype in order to obtain such an improvement with the pharmacogenetic algorithm in one patient was 13.2 for the comparison with the clinical algorithm and 6.0 for the comparison with a fixed dose of 35 mg per week (Section 10 in [Supplementary Appendix 1](#SD1)).

## DISCUSSION

The pharmacogenetic algorithm we developed provided significantly better predictions of the appropriate dose of warfarin than either the clinical algorithm or a fixed-dose approach. The greatest differences among the dose-prediction approaches were noted among patients whose stable therapeutic warfarin doses were 21 mg or less per week and among those whose stable doses were 49 mg or more per week, representing 46% of the cohort. These are the patients for whom an underdose or an overdose could have adverse clinical consequences. Patients who require intermediate doses are likely to obtain little benefit from the use of a pharmacogenetic algorithm.

Our analysis does not address the issue of whether a precise initial dose of warfarin translates into improved clinical end points, such as a reduction in the time needed to achieve a stable therapeutic INR, fewer INRs that are out of range, and a reduced incidence of bleeding or thromboembolic events. However, our study lays important groundwork for a prospective trial and suggests that such a trial should be powered to detect the benefits of incorporating pharmacogenetic information into the dose algorithm for patients who require high or low doses — the subgroups in our study for whom dose estimates based on the pharmacogenetic algorithm differed significantly from those based on the clinical algorithm.

Visual inspection of the graph of pharmacogenetic dose prediction as compared with actual warfarin dose (Section 8 in [Supplementary Appendix 1](#SD1)) suggests that the pharmacogenetic algorithm performed less predictably among patients who required very high doses of warfarin (>70 mg per week). The common genetic markers explored in our study primarily explain increased sensitivity to warfarin, not increased resistance. Mutations of *VKORC1* have been associated with resistance to warfarin, but these mutations are rare except in Ethiopian and certain Jewish populations.[19](#R19)^,^[25](#R25)^-^[28](#R28) The discovery of additional genes (e.g., through genomewide association studies) may identify additional genetic variants that can improve predictability in patients who require high doses of warfarin.

Our study has other limitations. First, we did not have sufficient data across the 21 research groups to include potentially important factors such as smoking status, vitamin K intake, or alcohol consumption, as well as other genetic factors (e.g., cytochrome P450, family 4, subfamily F, polypeptide 2 [*CYP4F2*], apolipoprotein E [*ApoE*], or gamma-glutamyl carboxylase [*GGCX*])[4](#R4)^,^[5](#R5)^,^[16](#R16)^,^[17](#R17)^,^[29](#R29)^-^[31](#R31) or environmental factors[32](#R32) that could help predict the stable therapeutic dose of warfarin. However, the percentage of dose variability among patients that is explained by our model is similar to that in other published models, so the effect of these variables is probably small. Second, we did not have any information about adverse events that may have occurred before stable anticoagulation was achieved; our data set provided only the eventual stable therapeutic doses. Third, different sites genotyped different subgroups of *VKORC1* SNPs, requiring us to impute missing genotypes for some patients. Our imputation strategy, which is based on linkage disequilibrium in *VKORC1,* is generally reliable (see Section 4 in [Supplementary Appendix 1](#SD1)),[11](#R11) but it may have introduced some error; however, the error would probably have led to an underestimation of the benefit of adding genetic information. We also restricted our algorithm to patients who had a target INR of 2 to 3, so it provides no explicit guidance on dosage to achieve INRs outside this range. The population included in this study represents the typical population that is treated with warfarin — namely, the elderly. Only 6% of the cohort was younger than 40 years of age; therefore, additional research with respect to the use of these algorithms in children and younger adults is needed.

In conclusion, using data from a large and diverse cohort of patients, we developed a pharmacogenetic dose algorithm for warfarin that uses genotypes from two genes (*VKORC1* and *CYP2C9*) and clinical variables to predict the stable therapeutic dose. This pharmacogenetic algorithm predicts the stable therapeutic dose of warfarin better than a fixed-dose approach and better than a clinical algorithm built from the same large data set. With the use of this algorithm and a definition of the ideal estimated dose as a dose that differs by no more than 20% from the stable dose, the pharmacogenetic algorithm produced significantly better dose estimates, with the greatest benefit seen in patients ultimately requiring 21 mg or less of warfarin per week and in those requiring 49 mg or more per week. The pharmacogenetic algorithm thus provides a robust basis for a prospective clinical trial of the efficacy of genetically informed dose estimation for patients who require warfarin.

## Supplementary Material

## Acknowledgments

Supported by grants from the National Institutes of Health Pharmacogenetics Research Network (GM61374, GM074492, GM63340, and HL65962); National Institute of General Medical Sciences (GM68797 and GM32165); National Heart, Lung, and Blood Institute (HL092173, HL068834, HL066176, HL070936, HL074724, and HL71083); National Institute of Neurological Disorders and Stroke (NS45598 and NS053646); National Center for Research Resources (RR020741); National Research Program for Genomic Medicine, National Science Council, Taiwan (NSC95-3112-B-001-010 and NSC95-3112-B-001-011); Israeli Science Foundation (789/04); United States–Israel Binational Foundation (2003229); Ministry of Health, Korea (5745); Korea Science and Engineering Foundation and Korean Ministry of Education, Science, and Engineering (R13-2007-023-00000-0); Korea Health 21 Research and Development Project, Korean Ministry of Health and Welfare (A030001); Conselho Nacional de Desenvolvimento Científico e Tecnológico; Fundação de Amparo à Pesquisa do Estado do Rio de Janeiro; Financiadora de Estudos e Projetos; National Health Service Executive (Northern and Yorkshire); American Association of Colleges of Pharmacy; American Foundation for Pharmaceutical Education; American Heart Association; United Kingdom Department of Health; North Carolina Biotechnology Company (Multidisciplinary Research Grant 2006-MRG-1117); Leading Project of Ministry of Education, Culture, Sports, Science, and Technology, Japan; Centers for Disease Control and Prevention National Office of Public Health Genomics Seed Funding; University of Washington Drug Metabolism, Transport, and Pharmacogenomics Research Program; Swedish Society of Medicine; Foundation for Strategic Research, Sweden; Heart and Lung Foundation, Sweden; Söderberg, Thuréus, and Selander Foundations, Sweden; Clinical Research Support (Avtal om läkarutbildning och forskning), Sweden; and the Wellcome Trust, United Kingdom.

Dr. Klein reports receiving consulting fees from Affymetrix; Dr. Gage, receiving grant support from Osmetech; Dr. Altman, receiving consulting fees from and holding equity in 23andMe; Dr. Kimmel, receiving consulting fees from Pfizer, GlaxoSmithKline, Novartis, and Centocor and grant support from the Aetna Foundation, Shire, and Pfizer; Dr. Limdi, receiving grant support from UCB; Dr. Johnson, receiving consulting fees from Medco; and Dr. Caldwell, being a named coinventor on the Marshfield Clinic provisional patent (application number 60981186) on the association of polymorphisms in CYP4F2 with the stable therapeutic dose of warfarin (a patent that has been licensed by the Marshfield Clinic to Osmetech Molecular Diagnostics). No other potential conflict of interest relevant to this article was reported.

The complete data set of genotypes and clinical variables, as well as the full genotype quality-control data, is available to registered PharmGKB users at [www.pharmgkb.org](http://www.pharmgkb.org) (full data set accession number, PA162355460).

## APPENDIX

The members of the International Warfarin Pharmacogenetics Consortium are as follows (alphabetical listing by institution): **Writing Committee** — T.E. Klein, R.B. Altman, N. Eriksson, B.F. Gage, S.E. Kimmel, M.-T.M. Lee, N.A. Limdi, D. Page, D.M. Roden, M.J. Wagner, M.D. Caldwell, J.A. Johnson; **Data Contributors** — *Academic Sinica, Taipei, Taiwan*: M.-T.M. Lee, Y.-T. Chen; Chang Gung Memorial Hospital, Chang Gung University, Taipei, Taiwan: M.-S. Wen; China Medical University, Graduate Institute of Chinese Medical Science, Taichung, Taiwan: M.-T.M. Lee; *Hadassah Medical Organization, Jerusalem, Israel*: Y. Caraco, I. Achache, S. Blotnick, M. Muszkat; *Inje University, Busan, Korea*: J.-G. Shin, H.-S. Kim; *Instituto Nacional de Câncer, Rio de Janeiro*: G. Suarez-Kurtz, J. Alessandra Perini; Instituto Nacional de Cardiologia Laranjeiras, Rio de Janeiro: E. Silva-Assunção; *Intermountain Healthcare, Salt Lake City*: J.L. Anderson, B.D. Horne, J.F. Carlquist; *Marshfield Clinic, Marshfield, WI*: M.D. Caldwell, R.L. Berg, J.K. Burmester; *National University Hospital, Singapore*: B.C. Goh, S.-C. Lee; *Newcastle University, Newcastle, United Kingdom*: F. Kamali, E. Sconce, A.K. Daly; *University of Alabama, Birmingham*: N.A. Limdi; *University of California, San Francisco*: A.H.B. Wu; *University of Florida, Gainesville*: J.A. Johnson, T.Y. Langaee, H. Feng; *University of Illinois, Chicago*: L. Cavallari, K. Momary; *University of Liverpool, Liverpool, United Kingdom*: M. Pirmohamed, A. Jorgensen, C.H. Toh, P. Williamson; *University of North Carolina, Chapel Hill*: H. McLeod, J.P. Evans, K.E. Weck; *University of Pennsylvania, Philadelphia*: S.E. Kimmel, C. Brensinger; University of Tokyo and RIKEN Center for Genomic Medicine, Tokyo: Y. Nakamura, T. Mushiroda; *University of Washington, Seattle*: D. Veenstra, L. Meckley, M.J. Rieder, A.E. Rettie; *Uppsala University, Uppsala, Sweden*: M. Wadelius, N. Eriksson, H. Melhus; *Vanderbilt University, Nashville*: C.M. Stein, D.M. Roden, U. Schwartz, D. Kurnik; *Washington University in St. Louis, St. Louis*: B.F. Gage, E. Deych, P. Lenzini, C. Eby; Wellcome Trust Sanger Institute, Hinxton, United Kingdom: L.Y. Chen, P. Deloukas; **Statistical Analysis** — *University of Alabama, Birmingham*: N.A. Limdi; *Marshfield Clinic, Marshfield, WI*: M.D. Caldwell; *North Carolina State University, Raleigh*: A. Motsinger-Reif; *Stanford University, Palo Alto, CA*: R.B. Altman, H. Sagreiya, T.E. Klein, B.S. Srinivasan; Uppsala University, Uppsala Clinical Research Center, Uppsala, Sweden: N. Eriksson; *University of California, San Francisco*: A.H.B. Wu; *University of North Carolina, Chapel Hill*: M.J. Wagner; *University of Florida, Gainesville*: J.A. Johnson; *University of Pennsylvania, Philadelphia*: S.E. Kimmel; *University of Wisconsin–Madison*: D. Page, E. Lantz, T. Chang; *Vanderbilt University, Nashville*: M. Ritchie; *Washington University in St. Louis, St. Louis*: B.F. Gage, E. Deych; **Genotyping Quality Control of Consortium Samples** — *Academic Sinica, Taipei, Taiwan*: M.-T.M. Lee, L.-S. Lu; **Genotype and Phenotype Quality Control** — *Inje University, Busan, Korea*: J.-G. Shin; *Marshfield Clinic, Marshfield, WI*: M.D. Caldwell; *Stanford University, Palo Alto, CA*: T.E. Klein, R.B. Altman, B.S. Srinivasan; *University of Alabama, Birmingham*: N.A. Limdi; *University of Florida, Gainesville*: J.A. Johnson; *University of Pennsylvania, Philadelphia*: S.E. Kimmel; *University of North Carolina, Chapel Hill*: M.J. Wagner; *University of Wisconsin–Madison*: D. Page; *Washington University in St. Louis, St. Louis*: B.F. Gage; *Vanderbilt University, Nashville*: M. Ritchie; **Data Curation** — *Stanford University, Palo Alto, CA*: T.E. Klein, R.B. Altman, B.S. Srinivasan; *University of North Carolina, Chapel Hill*: M.J. Wagner; *Washington University in St. Louis, St. Louis*: E. Deych.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Wysowski DK, Nourjah P, Swartz L. Bleeding complications with warfarin use: a prevalent adverse effect resulting in regulatory action. Arch Intern Med. 2007;167:1414–9. doi: 10.1001/archinte.167.13.1414.  [DOI](https://doi.org/10.1001/archinte.167.13.1414) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17620536/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arch%20Intern%20Med&title=Bleeding%20complications%20with%20warfarin%20use:%20a%20prevalent%20adverse%20effect%20resulting%20in%20regulatory%20action&author=DK%20Wysowski&author=P%20Nourjah&author=L%20Swartz&volume=167&publication_year=2007&pages=1414-9&pmid=17620536&doi=10.1001/archinte.167.13.1414&)

2. Budnitz DS, Shehab N, Kegler SR, Richards CL. Medication use leading to emergency department visits for adverse drug events in older adults. Ann Intern Med. 2007;147:755–65. doi: 10.7326/0003-4819-147-11-200712040-00006.  [DOI](https://doi.org/10.7326/0003-4819-147-11-200712040-00006) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18056659/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Intern%20Med&title=Medication%20use%20leading%20to%20emergency%20department%20visits%20for%20adverse%20drug%20events%20in%20older%20adults&author=DS%20Budnitz&author=N%20Shehab&author=SR%20Kegler&author=CL%20Richards&volume=147&publication_year=2007&pages=755-65&pmid=18056659&doi=10.7326/0003-4819-147-11-200712040-00006&)

3. Anderson JL, Horne BD, Stevens SM, et al. Randomized trial of genotype-guided versus standard warfarin dosing in patients initiating oral anticoagulation. Circulation. 2007;116:2563–70. doi: 10.1161/CIRCULATIONAHA.107.737312.  [DOI](https://doi.org/10.1161/CIRCULATIONAHA.107.737312) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17989110/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation&title=Randomized%20trial%20of%20genotype-guided%20versus%20standard%20warfarin%20dosing%20in%20patients%20initiating%20oral%20anticoagulation&author=JL%20Anderson&author=BD%20Horne&author=SM%20Stevens&volume=116&publication_year=2007&pages=2563-70&pmid=17989110&doi=10.1161/CIRCULATIONAHA.107.737312&)

4. Aquilante CL, Langaee TY, Lopez LM, et al. Influence of coagulation factor, vitamin K epoxide reductase complex subunit 1, and cytochrome P450 2C9 gene polymorphisms on warfarin dose requirements. Clin Pharmacol Ther. 2006;79:291–302. doi: 10.1016/j.clpt.2005.11.011.  [DOI](https://doi.org/10.1016/j.clpt.2005.11.011) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16580898/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Influence%20of%20coagulation%20factor,%20vitamin%20K%20epoxide%20reductase%20complex%20subunit%201,%20and%20cytochrome%20P450%202C9%20gene%20polymorphisms%20on%20warfarin%20dose%20requirements&author=CL%20Aquilante&author=TY%20Langaee&author=LM%20Lopez&volume=79&publication_year=2006&pages=291-302&pmid=16580898&doi=10.1016/j.clpt.2005.11.011&)

5. Caldwell MD, Awad T, Johnson JA, et al. CYP4F2 genetic variant alters required warfarin dose. Blood. 2008;111:4106–12. doi: 10.1182/blood-2007-11-122010.  [DOI](https://doi.org/10.1182/blood-2007-11-122010) | [PMC free article](/articles/PMC2288721/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18250228/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=CYP4F2%20genetic%20variant%20alters%20required%20warfarin%20dose&author=MD%20Caldwell&author=T%20Awad&author=JA%20Johnson&volume=111&publication_year=2008&pages=4106-12&pmid=18250228&doi=10.1182/blood-2007-11-122010&)

6. D'Andrea G, D'Ambrosio RL, Di Perna P, et al. A polymorphism in the VKORC1 gene is associated with an interindividual variability in the dose-anticoagulant effect of warfarin. Blood. 2005;105:645–9. doi: 10.1182/blood-2004-06-2111.  [DOI](https://doi.org/10.1182/blood-2004-06-2111) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15358623/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=A%20polymorphism%20in%20the%20VKORC1%20gene%20is%20associated%20with%20an%20interindividual%20variability%20in%20the%20dose-anticoagulant%20effect%20of%20warfarin&author=G%20D'Andrea&author=RL%20D'Ambrosio&author=P%20Di%20Perna&volume=105&publication_year=2005&pages=645-9&pmid=15358623&doi=10.1182/blood-2004-06-2111&)

7. Herman D, Peternel P, Stegnar M, Breskvar K, Dolzan V. The influence of sequence variations in factor VII, gamma-glutamyl carboxylase and vitamin K epoxide reductase complex genes on warfarin dose requirement. Thromb Haemost. 2006;95:782–7.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/16676068/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Thromb%20Haemost&title=The%20influence%20of%20sequence%20variations%20in%20factor%20VII,%20gamma-glutamyl%20carboxylase%20and%20vitamin%20K%20epoxide%20reductase%20complex%20genes%20on%20warfarin%20dose%20requirement&author=D%20Herman&author=P%20Peternel&author=M%20Stegnar&author=K%20Breskvar&author=V%20Dolzan&volume=95&publication_year=2006&pages=782-7&pmid=16676068&)

8. Kimura R, Miyashita K, Kokubo Y, et al. Genotypes of vitamin K epoxide reductase, gamma-glutamyl carboxylase, and cytochrome P450 2C9 as determinants of daily warfarin dose in Japanese patients. Thromb Res. 2007;120:181–6. doi: 10.1016/j.thromres.2006.09.007.  [DOI](https://doi.org/10.1016/j.thromres.2006.09.007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17049586/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Thromb%20Res&title=Genotypes%20of%20vitamin%20K%20epoxide%20reductase,%20gamma-glutamyl%20carboxylase,%20and%20cytochrome%20P450%202C9%20as%20determinants%20of%20daily%20warfarin%20dose%20in%20Japanese%20patients&author=R%20Kimura&author=K%20Miyashita&author=Y%20Kokubo&volume=120&publication_year=2007&pages=181-6&pmid=17049586&doi=10.1016/j.thromres.2006.09.007&)

9. Millican EA, Lenzini PA, Milligan PE, et al. Genetic-based dosing in orthopedic patients beginning warfarin therapy. Blood. 2007;110:1511–5. doi: 10.1182/blood-2007-01-069609.  [DOI](https://doi.org/10.1182/blood-2007-01-069609) | [PMC free article](/articles/PMC1975838/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17387222/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Genetic-based%20dosing%20in%20orthopedic%20patients%20beginning%20warfarin%20therapy&author=EA%20Millican&author=PA%20Lenzini&author=PE%20Milligan&volume=110&publication_year=2007&pages=1511-5&pmid=17387222&doi=10.1182/blood-2007-01-069609&)

10. Momary KM, Shapiro NL, Viana MA, Nutescu EA, Helgason CM, Cavallari LH. Factors influencing warfarin dose requirements in African-Americans. Pharmacogenomics. 2007;8:1535–44. doi: 10.2217/14622416.8.11.1535.  [DOI](https://doi.org/10.2217/14622416.8.11.1535) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18034618/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Factors%20influencing%20warfarin%20dose%20requirements%20in%20African-Americans&author=KM%20Momary&author=NL%20Shapiro&author=MA%20Viana&author=EA%20Nutescu&author=CM%20Helgason&volume=8&publication_year=2007&pages=1535-44&pmid=18034618&doi=10.2217/14622416.8.11.1535&)

11. Rieder MJ, Reiner AP, Gage BF, et al. Effect of VKORC1 haplotypes on transcriptional regulation and warfarin dose. N Engl J Med. 2005;352:2285–93. doi: 10.1056/NEJMoa044503.  [DOI](https://doi.org/10.1056/NEJMoa044503) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15930419/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Effect%20of%20VKORC1%20haplotypes%20on%20transcriptional%20regulation%20and%20warfarin%20dose&author=MJ%20Rieder&author=AP%20Reiner&author=BF%20Gage&volume=352&publication_year=2005&pages=2285-93&pmid=15930419&doi=10.1056/NEJMoa044503&)

12. Schelleman H, Chen Z, Kealey C, et al. Warfarin response and vitamin K epoxide reductase complex 1 in African Americans and Caucasians. Clin Pharmacol Ther. 2007;81:742–7. doi: 10.1038/sj.clpt.6100144.  [DOI](https://doi.org/10.1038/sj.clpt.6100144) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17329985/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Warfarin%20response%20and%20vitamin%20K%20epoxide%20reductase%20complex%201%20in%20African%20Americans%20and%20Caucasians&author=H%20Schelleman&author=Z%20Chen&author=C%20Kealey&volume=81&publication_year=2007&pages=742-7&pmid=17329985&doi=10.1038/sj.clpt.6100144&)

13. Sconce EA, Khan TI, Wynne HA, et al. The impact of CYP2C9 and VKORC1 genetic polymorphism and patient characteristics upon warfarin dose requirements: proposal for a new dosing regimen. Blood. 2005;106:2329–33. doi: 10.1182/blood-2005-03-1108.  [DOI](https://doi.org/10.1182/blood-2005-03-1108) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15947090/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=The%20impact%20of%20CYP2C9%20and%20VKORC1%20genetic%20polymorphism%20and%20patient%20characteristics%20upon%20warfarin%20dose%20requirements:%20proposal%20for%20a%20new%20dosing%20regimen&author=EA%20Sconce&author=TI%20Khan&author=HA%20Wynne&volume=106&publication_year=2005&pages=2329-33&pmid=15947090&doi=10.1182/blood-2005-03-1108&)

14. Takahashi H, Wilkinson GR, Nutescu EA, et al. Different contributions of polymorphisms in VKORC1 and CYP2C9 to intra- and inter-population differences in maintenance dose of warfarin in Japanese, Caucasians and African-Americans. Pharmacogenet Genomics. 2006;16:101–10. doi: 10.1097/01.fpc.0000184955.08453.a8.  [DOI](https://doi.org/10.1097/01.fpc.0000184955.08453.a8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16424822/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Different%20contributions%20of%20polymorphisms%20in%20VKORC1%20and%20CYP2C9%20to%20intra-%20and%20inter-population%20differences%20in%20maintenance%20dose%20of%20warfarin%20in%20Japanese,%20Caucasians%20and%20African-Americans&author=H%20Takahashi&author=GR%20Wilkinson&author=EA%20Nutescu&volume=16&publication_year=2006&pages=101-10&pmid=16424822&doi=10.1097/01.fpc.0000184955.08453.a8&)

15. Tham LS, Goh BC, Nafziger A, et al. A warfarin-dosing model in Asians that uses single-nucleotide polymorphisms in vitamin K epoxide reductase complex and cytochrome P450 2C9. Clin Pharmacol Ther. 2006;80:346–55. doi: 10.1016/j.clpt.2006.06.009.  [DOI](https://doi.org/10.1016/j.clpt.2006.06.009) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17015052/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=A%20warfarin-dosing%20model%20in%20Asians%20that%20uses%20single-nucleotide%20polymorphisms%20in%20vitamin%20K%20epoxide%20reductase%20complex%20and%20cytochrome%20P450%202C9&author=LS%20Tham&author=BC%20Goh&author=A%20Nafziger&volume=80&publication_year=2006&pages=346-55&pmid=17015052&doi=10.1016/j.clpt.2006.06.009&)

16. Vecsler M, Loebstein R, Almog S, et al. Combined genetic profiles of components and regulators of the vitamin K-dependent gamma-carboxylation system affect individual sensitivity to warfarin. Thromb Haemost. 2006;95:205–11. doi: 10.1160/TH05-06-0446.  [DOI](https://doi.org/10.1160/TH05-06-0446) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16493479/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Thromb%20Haemost&title=Combined%20genetic%20profiles%20of%20components%20and%20regulators%20of%20the%20vitamin%20K-dependent%20gamma-carboxylation%20system%20affect%20individual%20sensitivity%20to%20warfarin&author=M%20Vecsler&author=R%20Loebstein&author=S%20Almog&volume=95&publication_year=2006&pages=205-11&pmid=16493479&doi=10.1160/TH05-06-0446&)

17. Wadelius M, Chen LY, Downes K, et al. Common VKORC1 and GGCX polymorphisms associated with warfarin dose. Pharmacogenomics J. 2005;5:262–70. doi: 10.1038/sj.tpj.6500313.  [DOI](https://doi.org/10.1038/sj.tpj.6500313) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15883587/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Common%20VKORC1%20and%20GGCX%20polymorphisms%20associated%20with%20warfarin%20dose&author=M%20Wadelius&author=LY%20Chen&author=K%20Downes&volume=5&publication_year=2005&pages=262-70&pmid=15883587&doi=10.1038/sj.tpj.6500313&)

18. Wu AH. Use of genetic and nongenetic factors in warfarin dosing algorithms. Pharmacogenomics. 2007;8:851–61. doi: 10.2217/14622416.8.7.851.  [DOI](https://doi.org/10.2217/14622416.8.7.851) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18240910/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Use%20of%20genetic%20and%20nongenetic%20factors%20in%20warfarin%20dosing%20algorithms&author=AH%20Wu&volume=8&publication_year=2007&pages=851-61&pmid=18240910&doi=10.2217/14622416.8.7.851&)

19. Loebstein R, Dvoskin I, Halkin H, et al. A coding VKORC1 Asp36Tyr polymorphism predisposes to warfarin resistance. Blood. 2007;109:2477–80. doi: 10.1182/blood-2006-08-038984.  [DOI](https://doi.org/10.1182/blood-2006-08-038984) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17110455/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=A%20coding%20VKORC1%20Asp36Tyr%20polymorphism%20predisposes%20to%20warfarin%20resistance&author=R%20Loebstein&author=I%20Dvoskin&author=H%20Halkin&volume=109&publication_year=2007&pages=2477-80&pmid=17110455&doi=10.1182/blood-2006-08-038984&)

20. Caldwell MD, Berg RL, Zhang KQ, et al. Evaluation of genetic factors for warfarin dose prediction. Clin Med Res. 2007;5:8–16. doi: 10.3121/cmr.2007.724. [Erratum, Clin Med Res 2007;5:142.]  [DOI](https://doi.org/10.3121/cmr.2007.724) | [PMC free article](/articles/PMC1855340/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17456829/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Med%20Res&title=Evaluation%20of%20genetic%20factors%20for%20warfarin%20dose%20prediction&author=MD%20Caldwell&author=RL%20Berg&author=KQ%20Zhang&volume=5&publication_year=2007&pages=8-16&pmid=17456829&doi=10.3121/cmr.2007.724&)

21. Bussey HI, Wittkowsky AK, Hylek EM, Walker MB. Genetic testing for warfarin dosing? Not yet ready for prime time. Pharmacotherapy. 2008;28:141–3. doi: 10.1592/phco.28.2.141.  [DOI](https://doi.org/10.1592/phco.28.2.141) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18225960/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacotherapy&title=Genetic%20testing%20for%20warfarin%20dosing?%20Not%20yet%20ready%20for%20prime%20time&author=HI%20Bussey&author=AK%20Wittkowsky&author=EM%20Hylek&author=MB%20Walker&volume=28&publication_year=2008&pages=141-3&pmid=18225960&doi=10.1592/phco.28.2.141&)

22. Caraco Y, Blotnick S, Muszkat M. CYP2C9 genotype-guided warfarin prescribing enhances the efficacy and safety of anticoagulation: a prospective randomized controlled study. Clin Pharmacol Ther. 2008;83:460–70. doi: 10.1038/sj.clpt.6100316.  [DOI](https://doi.org/10.1038/sj.clpt.6100316) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17851566/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=CYP2C9%20genotype-guided%20warfarin%20prescribing%20enhances%20the%20efficacy%20and%20safety%20of%20anticoagulation:%20a%20prospective%20randomized%20controlled%20study&author=Y%20Caraco&author=S%20Blotnick&author=M%20Muszkat&volume=83&publication_year=2008&pages=460-70&pmid=17851566&doi=10.1038/sj.clpt.6100316&)

23. Shurin SB, Nabel EG. Pharmacogenomics — ready for prime time? N Engl J Med. 2008;358:1061–3. doi: 10.1056/NEJMe0800801.  [DOI](https://doi.org/10.1056/NEJMe0800801) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18322288/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Pharmacogenomics%20%E2%80%94%20ready%20for%20prime%20time?&author=SB%20Shurin&author=EG%20Nabel&volume=358&publication_year=2008&pages=1061-3&pmid=18322288&doi=10.1056/NEJMe0800801&)

24. Cook RJ, Sackett DL. The number needed to treat: a clinically useful measure of treatment effect. BMJ. 1995;310:452–4. doi: 10.1136/bmj.310.6977.452. [Erratum, BMJ 1995;310:1056.]  [DOI](https://doi.org/10.1136/bmj.310.6977.452) | [PMC free article](/articles/PMC2548824/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7873954/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMJ&title=The%20number%20needed%20to%20treat:%20a%20clinically%20useful%20measure%20of%20treatment%20effect&author=RJ%20Cook&author=DL%20Sackett&volume=310&publication_year=1995&pages=452-4&pmid=7873954&doi=10.1136/bmj.310.6977.452&)

25. McWilliam A, Lutter A, Nardinelli C. Healthcare impact of personalized medicine using genetic testing: an exploratory analysis for warfarin. Personalized Med. 2008;5:279–84. doi: 10.2217/17410541.5.3.279.  [DOI](https://doi.org/10.2217/17410541.5.3.279) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29783488/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Personalized%20Med&title=Healthcare%20impact%20of%20personalized%20medicine%20using%20genetic%20testing:%20an%20exploratory%20analysis%20for%20warfarin&author=A%20McWilliam&author=A%20Lutter&author=C%20Nardinelli&volume=5&publication_year=2008&pages=279-84&pmid=29783488&doi=10.2217/17410541.5.3.279&)

26. Harrington DJ, Underwood S, Morse C, Shearer MJ, Tuddenham EG, Mumford AD. Pharmacodynamic resistance to warfarin associated with a Val66Met substitution in vitamin K epoxide reductase complex subunit 1. Thromb Haemost. 2005;93:23–6. doi: 10.1160/TH04-08-0540.  [DOI](https://doi.org/10.1160/TH04-08-0540) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15630486/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Thromb%20Haemost&title=Pharmacodynamic%20resistance%20to%20warfarin%20associated%20with%20a%20Val66Met%20substitution%20in%20vitamin%20K%20epoxide%20reductase%20complex%20subunit%201&author=DJ%20Harrington&author=S%20Underwood&author=C%20Morse&author=MJ%20Shearer&author=EG%20Tuddenham&volume=93&publication_year=2005&pages=23-6&pmid=15630486&doi=10.1160/TH04-08-0540&)

27. Rost S, Fregin A, Ivaskevicius V, et al. Mutations in VKORC1 cause warfarin resistance and multiple coagulation factor deficiency type 2. Nature. 2004;427:537–41. doi: 10.1038/nature02214.  [DOI](https://doi.org/10.1038/nature02214) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14765194/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Mutations%20in%20VKORC1%20cause%20warfarin%20resistance%20and%20multiple%20coagulation%20factor%20deficiency%20type%202&author=S%20Rost&author=A%20Fregin&author=V%20Ivaskevicius&volume=427&publication_year=2004&pages=537-41&pmid=14765194&doi=10.1038/nature02214&)

28. Scott SA, Edelmann L, Kornreich R, Desnick RJ. Warfarin pharmacogenetics: CYP2C9 and VKORC1 genotypes predict different sensitivity and resistance frequencies in the Ashkenazi and Sephardi Jewish populations. Am J Hum Genet. 2008;82:495–500. doi: 10.1016/j.ajhg.2007.10.002.  [DOI](https://doi.org/10.1016/j.ajhg.2007.10.002) | [PMC free article](/articles/PMC2427171/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18252229/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Warfarin%20pharmacogenetics:%20CYP2C9%20and%20VKORC1%20genotypes%20predict%20different%20sensitivity%20and%20resistance%20frequencies%20in%20the%20Ashkenazi%20and%20Sephardi%20Jewish%20populations&author=SA%20Scott&author=L%20Edelmann&author=R%20Kornreich&author=RJ%20Desnick&volume=82&publication_year=2008&pages=495-500&pmid=18252229&doi=10.1016/j.ajhg.2007.10.002&)

29. Kimmel SE, Christie J, Kealey C, et al. Apolipoprotein E genotype and warfarin dosing among Caucasians and African Americans. Pharmacogenomics J. 2008;8:53–60. doi: 10.1038/sj.tpj.6500445.  [DOI](https://doi.org/10.1038/sj.tpj.6500445) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17325732/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Apolipoprotein%20E%20genotype%20and%20warfarin%20dosing%20among%20Caucasians%20and%20African%20Americans&author=SE%20Kimmel&author=J%20Christie&author=C%20Kealey&volume=8&publication_year=2008&pages=53-60&pmid=17325732&doi=10.1038/sj.tpj.6500445&)

30. Sconce EA, Daly AK, Khan TI, Wynne HA, Kamali F. APOE genotype makes a small contribution to warfarin dose requirements. Pharmacogenet Genomics. 2006;16:609–11. doi: 10.1097/01.fpc.0000220567.98089.b5.  [DOI](https://doi.org/10.1097/01.fpc.0000220567.98089.b5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16847429/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=APOE%20genotype%20makes%20a%20small%20contribution%20to%20warfarin%20dose%20requirements&author=EA%20Sconce&author=AK%20Daly&author=TI%20Khan&author=HA%20Wynne&author=F%20Kamali&volume=16&publication_year=2006&pages=609-11&pmid=16847429&doi=10.1097/01.fpc.0000220567.98089.b5&)

31. Wadelius M, Chen LY, Eriksson N, et al. Association of warfarin dose with genes involved in its action and metabolism. Hum Genet. 2007;121:23–34. doi: 10.1007/s00439-006-0260-8.  [DOI](https://doi.org/10.1007/s00439-006-0260-8) | [PMC free article](/articles/PMC1797064/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17048007/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Genet&title=Association%20of%20warfarin%20dose%20with%20genes%20involved%20in%20its%20action%20and%20metabolism&author=M%20Wadelius&author=LY%20Chen&author=N%20Eriksson&volume=121&publication_year=2007&pages=23-34&pmid=17048007&doi=10.1007/s00439-006-0260-8&)

32. Holbrook AM, Pereira JA, Labiris R, et al. Systematic overview of warfarin and its drug and food interactions. Arch Intern Med. 2005;165:1095–106. doi: 10.1001/archinte.165.10.1095.  [DOI](https://doi.org/10.1001/archinte.165.10.1095) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15911722/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arch%20Intern%20Med&title=Systematic%20overview%20of%20warfarin%20and%20its%20drug%20and%20food%20interactions&author=AM%20Holbrook&author=JA%20Pereira&author=R%20Labiris&volume=165&publication_year=2005&pages=1095-106&pmid=15911722&doi=10.1001/archinte.165.10.1095&)
