# Genome-wide Association Study of Suicidal Ideation Emerging During Citalopram Treatment of Depressed Outpatients

## Metadata
**Authors:** Gonzalo Laje, Andrew S Allen, Nirmala Akula, Husseini Manji, A John Rush, Francis J McMahon
**Journal:** Pharmacogenetics and genomics
**Date:** 2009 Sep
**DOI:** [10.1097/FPC.0b013e32832e4bcd](https://doi.org/10.1097/FPC.0b013e32832e4bcd)
**PMID:** 19724244
**PMCID:** PMC2819190
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2819190/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC2819190/pdf/nihms171637.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC2819190/pdf/nihms171637.pdf)

## Abstract

Suicidal ideation is an uncommon but worrisome symptom than can emerge during antidepressant treatment. We have previously described association between treatment emergent suicidal ideation (TESI) and markers in genes encoding the glutamate receptors GRIK2 and GRIA3. The present genome-wide association study was conducted to identify additional genetic markers associated with TESI that may help identify individuals at high-risk who may benefit from closer monitoring, alternative treatments, and/or specialty care. A clinically-representative cohort of outpatients with non-psychotic major depressive disorder enrolled in the Sequenced Treatment Alternatives to Relieve Depression (STAR*D) trial were treated with citalopram under a standard protocol for up to 14 weeks. DNA samples from 90 white participants who developed TESI and a gender and race matched equal number of treated participants who denied any suicidal ideas were genotyped with 109,365 single nucleotide polymorphisms on the Illumina's Human-1 BeadChip. One marker was found to be associated with TESI in this sample at the experiment-wide adjusted p<0.05 level (marker rs11628713, allelic p= 6.2 × 10-7, OR = 4.7, permutation p=0.01). A second marker was associated at the experiment-wide adjusted p=0.06 level (rs10903034, allelic p = 3.02× 10-6, OR = 2.7, permutation p=0.06). These markers reside within the genes PAPLN and IL28RA, respectively. PAPLN encodes papilin, a protoglycan-like sulfated glycoprotein. IL28RA encodes an interleukin receptor. Together with our previous report, these findings may shed light on the biological basis of TESI and may help identify patients at increased risk of this potentially serious adverse event.

Keywords: treatment-emergent suicidal ideation, TESI, pharmacogenetics, citalopram, antidepressant, major depression, SSRI, STAR*D, genome-wide

## Introduction

Most suicides are attributable to major depressive episode. Effective treatment with antidepressant medication prevents suicide. There is, however, an ongoing controversy over whether antidepressants, especially the class known as selective serotonin reuptake inhibitors (SSRIs), can actually trigger suicidal thoughts or behavior (suicidality) in some people. [[1](#R1), [2](#R2)] The FDA determined that there was sufficient evidence to issue a “black box warning” highlighting the risk of treatment-emergent suicidality in those under the age of 25 yr prescribed antidepressants [[3](#R3)]. A subsequent decrease in the use of SSRIs, especially in children, [[4](#R4)] and a spike in the suicide rate among young people [[5](#R5), [6](#R6)] have only served to heighten the controversy.

Antidepressants decrease suicidality in most patients, but may increase it in others. Thus, it would be useful to identify at-risk groups who may benefit from closer monitoring or alternative therapy while decreasing unnecessary concern among others that may lead to underuse of an otherwise safe and effective class of medications.

Actual suicide attempts during antidepressant treatment are uncommon [[7](#R7)], but suicidal ideation is always a concern. Two prior studies suggest that genetic markers may identify patients who experience treatment-emergent suicidal ideation (TESI) [[8](#R8), [9](#R9)]. Both studies used data from the Sequenced Treatment Alternatives to Relieve Depression cohort (STAR*D) ([www.star-d.org](http://www.star-d.org)) (Rush, et al, Controlled Clinical Trials), but the two studies utilized somewhat different case definitions. Perlis et al. tested markers near CREB1 and detected evidence of association with onset or worsening of suicidal thoughts during citalopram treatment, but only in males. [[9](#R9)] In a study of 68 candidate genes, Laje et al. [[8](#R8)] detected an association between markers near GRIK2 and GRIA3 and the onset of thoughts of death or suicide during citalopram treatment. To our knowledge, no study has yet attempted to identify markers of TESI at the genome-wide level.

This report provides results of a genome-wide association study of TESI in the STAR*D cohort based on over 100,000 gene-based markers.

## Methods

### Sample

Ascertainment and evaluation for the STAR*D study are detailed elsewhere [[10](#R10), [11](#R11)]. Briefly, investigators implemented a standard study protocol at 18 primary care and 23 psychiatric care settings across the United States. Participants provided separate written informed consents for the treatment study and the DNA collection. Participation in DNA collection was optional. DNA collection was initiated approximately 12 months after initiation of the treatment study. Outpatients, 18-75 years of age, with a baseline 17-item Hamilton Depression Rating Scale [[12](#R12), [13](#R13)] (HRSD_17_) score of ≥14 who met DSM-IV [[14](#R14)] criteria for non-psychotic, major depressive disorder (MDD) were eligible. Patients with bipolar or psychotic disorders were excluded, as were those with a primary diagnosis of obsessive-compulsive or eating disorders, general medical conditions that contraindicated study medications, substance dependence requiring inpatient detoxification, clear non-response or intolerance to any protocol antidepressant during the current episode, or those who were pregnant or breast-feeding [[10](#R10), [11](#R11)].

At the first treatment step, all participants received the SSRI citalopram, typically starting at 20 mg/day, with dose increases following recommended procedures (up to 40mg/day by week 4 and 60mg/day by week 6) [[15](#R15)]. The protocol required an adequate dose of citalopram for a sufficient time to ensure that those who did not improve were most likely unresponsive to the medication [[15](#R15)]. No concomitant psychotropic medications were allowed, aside from benzodiazepines, hypnotics, or trazodone (up to 200 mg/d) for sleep, if needed. The sample characteristics were presented elsewhere [[15](#R15), [16](#R16)].

DNA samples were collected from 1953 participants. A sample of 20 ml of whole blood was collected and shipped to the Rutgers Cell Repository, where lymphocytes were extracted and cryopreserved using standard methods. DNA was extracted using Gene-Pure chemistry (Qiagen). Samples were arrayed robotically, then gender-verified with a set of three X-linked and two Y-linked markers. A CONSORT (Consolidated Standards of Reporting Trials) diagram of the current study sample is shown elsewhere [[8](#R8)].

Those who consented to DNA collection were similar to those in the full sample, but differed on some variables (for details, see [[16](#R16)]). These differences cannot affect the genetic association results, which derive from comparisons among the genotyped subjects, but do limit generalizability. Those who provided DNA did not differ in the frequency of TESI or in those variables that were associated with TESI in this sample [[8](#R8)].

All of the 90 white (including Hispanics), participants who developed TESI and a gender, race and ethnicity matched set of 90 treated participants who denied any history of suicide attempts and suicidal ideas during their citalopram treatment were used for the genome-wide scan. Due to limited resources we could not genotype the whole STAR*D sample, so to improve power controls were further selected based on a minimum of 3 visits, a baseline QIDS-C total score (CRC) >=10, adherence to medication, recurrent depression, no history of suicide attempts and further, no report of suicidal thoughts throughout citalopram treatment on QIDS-C_16_ or QIDS-SR_16_. An expanded set of 94 controls, selected on the same criteria, were added post-hoc to test the robustness of the findings.

### Phenotype Definition

Consistent with previous definitions of TESI [[8](#R8), [9](#R9)], we used the “Thoughts of Death or Suicide” question (item # 12) on the 16-item Quick Inventory of Depressive Symptomatology – Self-report (QIDS-SR_16_), a reliable and well-validated measure of symptom severity that has been shown to correlate well with the Hamilton Depression Rating Scale [[17](#R17)-[21](#R21)]. The QIDS-SR_16_ was chosen here over the clinician-rated version (the QIDS-C_16_) since TESI is a subjective phenomenon and we wished to avoid clinician bias and patient reluctance to verbally report the phenomenon to the clinician.

The QIDS-SR_16_ was administered at baseline and at each of the protocol-recommended clinic visits at baseline, weeks 2, 4, 6, 9, and 12. Possible responses to item #12 include: “I do not think of suicide or death” (=0), “I feel that life is empty or wonder if it's worth living”(=1), “I think of suicide or death several times a week for several minutes” (=2), and “I think of suicide or death several times a day in some detail, or I have made specific plans for suicide, or have actually tried to take my life (=3). Participants who scored “0” on this item before citalopram treatment and 1, 2 or 3 at least once during treatment were defined as TESI cases. Although a score is obtained from the scale's item this trait cannot be considered quantitative since the intervals between anchor points are not equal.

### Population structure

We evaluated the possibility of population structure leading to inflated association results by using as covariates in the primary allelic test the posterior probabilities calculated by STRUCTURE [[22](#R22), [23](#R23)] to evaluate the worst case scenario of 2 cryptic populations within the white subset (k=2). A set of 344 unlinked SNPs selected from the total dataset was run on STRUCTURE for 20,000 burn-in steps followed by 20,000 replications (admixture model) to assign to each individual a posterior probability of ancestry for each of the inferred populations. Additionally, to detect any residual bias we plotted the observed −log10 p-values against the expected −log10 p-values under the null hypothesis (Q-Q plot) ([figure 1](#F1)).

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b008/2819190/59db4398e04a/nihms171637f1.jpg)

Q-Q plot used for residual bias detection shows the observed –log10 p-values against the–log10 p-values expected under the null hypothesis.

### Selection of single nucleotide polymorphism (SNP) markers

We used the Illumina Human-1 BeadChip. This chip samples 109,365 single nucleotide polymorphisms, 48% fall within 10K of exons, 23% of the snps are in the transcripts, 15% fall in highly conserved regions, and 14% are uniformly spaced in the genome.

### Genotyping methods

A total of 90 white TESI cases and 90 gender, race and ethnicity matched controls were genotyped the Illumina Beadstation 500 according to manufacturer's recommended protocol. The markers of interest were further genotyped in the total sample using Taqman Assay (Applied Biosystems, Foster City, CA) followed by scanning the samples in Analyst HT (Molecular Devices, Sunnyvale, CA). In the 274 samples that were genotyped with both the Taqman and Illumina assays, there was 100% agreement at the genotype level. The genotypes are scored by a clustering algorithm (kindly provided by Sam Chen, Virginia Commonwealth University). The call rates for this sample were 99.5%. For quality control purposes 8501 SNPs were excluded if missingness exceeded 10%, if minor allele frequencies were lower than 5% and if Hardy –Weinberg deviation was lower than 1×10^-3^. No individuals were excluded due to gender mismatches.

### Statistical methods

Stratified score tests were developed from a retrospective logistic regression model adjusted for confounders via stratum specific intercepts (see appendix for details). In this analysis, autosomal markers were tested using a standard allelic dosing model in which each extra copy of the minor allele has a multiplicative effect on risk. In females, the genetic model used for X-linked markers also corresponded to an allelic dosing model. In males, however, the genetic model was parameterized so that the penetrance of a male with the minor allele would correspond to the penetrance of a female with two minor alleles. This parameterization was chosen because it corresponds to the allelic dosing model under X-inactivation. The score tests developed here are asymptotically equivalent to the “allelic” likelihood ratio test statistics given by the software program UNPHASED [[24](#R24)]. However, unlike the likelihood ratio tests, the score tests can be given in closed form (details in appendix) making for extremely rapid computation. All analyses were stratified on each individual's most likely subpopulation membership as output from the structure 2 subpopulation model detailed above.

We controlled the experiment-wide error rate by using a standard permutation scheme. This scheme aims at breaking any correlation between the SNP markers and the phenotype, reflecting a global null hypothesis, while maintaining the correlation structure among the markers. To implement, we randomly permuted the phenotypes between individuals to form permutation datasets. Since adjusted analyses were conducted, permutations were performed separately within each strata. On each resulting permutation dataset an entire genome scan was performed and the minimum p-value noted. By repeating the permutation/analysis steps a large number of times (50,000), we were able to estimate the permutation distribution of the minimum p-value under the global null hypothesis. By comparing our original raw p-values with the permutation distribution allowed us to assess their experiment-wide significance and compute permutation adjusted p-values.

Additional models, such as genotypic or haplotype tests, could have been performed but given the potential for correction of a large number of tests, no other models beyond the allelic were tested in our primary analyses. We ran a genotypic model on a post-hoc basis.

### Clinical parameters (risk) calculation

The effect of the two SNPs presented here (rs11628713 and rs10903034) as well as two previously published SNPs [[8](#R8)] found in GRIA3 (rs4825476) and GRIK2 (rs2518224) on an individual's risk of developing TESI was estimated, after supplemental genotyping of rs11628713 and rs10903034, from the total STAR*D sample (n=1915). Since this sample represents a prospective study of TESI development, risk estimation was conducted directly on this sample using a standard multivariable logistic regression model. The adequacy of model fit was assessed via the Hosmer and Lemeshow test [[25](#R25)].

## Results

### Descriptive variables

Demographic and clinical characteristics of the sample are presented elsewhere [[8](#R8)]. There were no significant demographic differences between cases (n=90) and controls (n=90), and no differences in several clinical variables that are known predictors of suicide [[26](#R26)]. Consistent with our previous report [[8](#R8)] in this subsample, TESI cases also received a significantly higher maximum citalopram dose, were significantly less likely to go into remission (full recovery, as defined previously [[15](#R15), [16](#R16)]), and were more likely to move to the secondary treatment step following citalopram [[15](#R15)]. As these variables could act as potential confounders, we performed a logistic regression that included these variables and the markers of interest. The results suggest these variables are not significant confounders ([Supplemental table 1](#SD2)).

### Association analysis

[Figure 2](#F2) presents results of the genetic association test. Two markers, rs11628713 and rs10903034 stand out. Both markers were in Hardy-Weinberg equilibrium in this sample.

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b008/2819190/db03bdb8d6b2/nihms171637f2.jpg)

Allelic comparisons for each of the 109,365 markers, ordered by physical position. The dashed lines represent permutation-corrected values.

One marker, rs11628713 (allelic p= 6.2 × 10^-7^, OR = 4.9, permutation p=0.01) was associated with TESI at the experiment-wise p<0.05 level. This is an intronic marker at about 50bp from exon 13 in the PAPLN gene in chromosome 14q24.2. This SNP is a C/T polymorphism in a non-conserved region where the ancestral C-allele is over-represented in TESI cases (allele frequency in cases 0.93 and controls 0.74). The association findings remained significant in the expanded reference set (total controls n=184), but the effect size was slightly lower (OR 4, 95%CI 2.08, 7.46).

The other marker, rs10903034, was associated with TESI at an experiment-wise p=0.06 level (allelic p =3.02 × 10^-6^, OR = 2.7, permutation = 0.06). This marker resides in chromosome 1p36.11 within the 3′UTR of the IL28RA gene. The ancestral T-allele is over-represented in TESI cases (allele frequency in cases 0.68 and controls 0.44).

One additional marker in PAPLN and four additional markers in IL28RA were associated with TESI in this sample ([Supplemental table 2](#SD3)). A post hoc genotypic test was conducted, but no additional markers were found to have experiment-wise significance (data not shown).

Finally, rs11628713 (PAPLN) and rs10903034 (IL28RA) were genotyped in the whole STAR*D sample (N=1915, cases n=120, controls n=1742, missing n=53). The PAPLN marker had an allele frequency in cases of 0.91 and 0.83 in controls (OR: 2.1, p=0.001). The IL28RA marker had an allele frequency in cases of 0.61 and 0.54 in controls (OR: 1.33, p=0.037).

### Association with previously described markers in GRIK2 and GRIA3

The markers previously associated with TESI are not included on the Illumina Infinium I chip used in the present study. As seen in [supplemental figure 1a](#SD1), there were no markers included on the Illumina Infinium I array that sampled rs2518224 through linkage disequilibrium in this population. However, marker rs5911588 does sample rs4825476 at an r^2^ value of 0.46 ([supplemental figure 1b](#SD1)), representing moderate correlation between these neighboring markers. As expected, rs5911588 is associated with TESI (p = 0.0004), consistent with the findings previously described in this sample.

### Additive effect of risk alleles

[Figure 3](#F3) presents risk estimates derived from a multiple logistic regression model which includes additive effects of four genotypes (rs11628713, rs10903034, rs4825476 and rs2518224) as well as the effect of sex in the whole STAR*D sample (n=1915). Race and ethnicity were not found to have significant main effects or to significantly modify genotypic effect parameters. Hence, race and ethnicity were excluded from the final model. The final model was found to provide a reasonable fit to the observed data (Hosmer and Lemeshow p=0.7; small p-values would indicate a poor model fit).

### Figure 3.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b008/2819190/348f4b4983d1/nihms171637f3.jpg)

Risk estimates derived from a multiple logistic regression model which includes additive effects of four genotypes (rs11628713, rs10903034, rs4825476 and rs2518224) by gender (males above). Figures are separated by gender (A: females, B: males) since one of the markers resides on a chromosome X and thus males are hemizygous for this variant. The x-axis represents PAPLN and IL28RA genotypes, the y-axis represents the risk estimate, and the z-axis represents GRIA3 and GRIK2 genotypes

The highest risk of developing TESI in the context of citalopram treatment would occur with one or two G alleles in GRIA3, the CC genotype in GRIK2, the CC genotype in PAPLN and the AA genotype in IL28RA, a very uncommon combination. All other possible allelic combinations and their predicted risks are shown in ([figure 3](#F3)).

## Discussion

To our knowledge this is the first genome-wide study to detect genetic markers related to TESI. We found two novel markers that were associated with TESI in this sample. Although the marker on IL28RA did not achieve experiment-wise significance at the 0.05 level, we chose to make it a part of the predictive model because of the strength of the association, the consistency observed in the signal of neighboring markers, and biological plausibility. However, until functional alleles are demonstrated or replication is shown in an independent sample, these findings must be viewed as preliminary.

Due to the relatively low frequency of TESI, replication will require a very large prospectively ascertained treatment sample or a retrospective case-control design. It remains unclear whether our findings extend to other antidepressants. However, due to the extensive use in psychiatry and primary care of citalopram, and its low risk for pharmacokinetic interactions at the cytochorme P-450-2D6 level, these findings have relevance even if confined to citalopram and related compounds. Moreover, the effect size estimates reported herein are likely to be inflated (winner's curse) and only after replication in independent samples we would be able to estimate the “true” effect size. Until such experiments have been carried out it is impossible to make statements about any clinical utility of the reported markers.

This study has several limitations. Although consistent with prior reports [[8](#R8), [9](#R9)], TESI was defined on the basis of a single item on a depression rating scale. Neither the instrument, the item, nor the STAR*D study itself, were designed primarily to address TESI. Due to the lack of a placebo group it is impossible to determine what fraction of TESI in this study is directly attributable to antidepressant treatment. These limitations are characteristic of all existing large antidepressant treatment samples, suggesting the need for future large, placebo-controlled studies.

Although the present study took a genome-wide approach, covering over 100,000 markers, current genotyping technology now allows for the study of at least 5 times more potentially relevant polymorphisms. Moreover, as described above, the design of the Illumina Infinium I chip is exon-centric. Therefore, the markers used did not sample potential regulatory regions between genes. Of course, factors other than sequence polymorphisms may be of relevance in TESI, such as copy number variation, methylation and acetylation patterns, and environmental influences.

As in all case-control genetic association studies, hidden population stratification is always a risk. When the trait of interest and marker allele frequencies differ among ancestral groups, spurious associations (and inflated test statistics) may arise. We have previously shown that there is no relationship between race and TESI in this sample. [[8](#R8)] We further adjusted the results for any differences in allele frequencies, using covariates. The final distribution of p-values was consistent with expectations under the null hypothesis ([Figure 1](#F1)), and the genomic inflation factor (lambda_GC_), based on the median of all the observed chi-square values, was 1.00. Thus, our findings are unlikely to be the result of population stratification.

PAPLN is a 37kb, 26 exon gene on chromosome 14 that encodes a proteoglycan-like sulfated glycoprotein called papilin. There is a paucity of information about papilin in humans.

The other gene implicated in this study, IL28RA, is a 33kb, 6 exon gene on chromosome 1 that encodes for a protein in the class II cytokine receptor family. Viral infection can induce the expression of interleukin 29 (IL29) (or interferon lambda 1), interleukin 28A (IL28A) (or interferon lambda 2) and interleukin 28B (IL28B) (or interferon lambda 3) [[27](#R27)]. While little is known about the role of this receptor or interleukin 28 (interferon lambda 2), or their relevance to mood disorders and suicidality, it is notable that the therapeutic use of interferon alpha is associated with a significant incidence of depression and suicidality [[28](#R28)].

In summary, we have identified markers in two novel genes that together with previous reports may shed light on the biological basis of suicidal ideation that emerges during antidepressant treatment. If replicated, these findings may take us a step closer toward developing markers with clinically meaningful predictive value for this potentially serious adverse event. Further work is needed to replicate these findings and uncover the functional variation that underlies the association signals we observe.

## Supplementary Material

## Acknowledgments

This study was funded by the Intramural Research Program of the National Institute of Mental Health, NIH, by a NARSAD Independent Investigator Award to FJM, Duke Clinical Research Institute and NIH #1K25-077663 to ASA. The authors thank the STAR*D research team for acquisition of clinical data and DNA samples and Forest Laboratories for providing citalopram at no cost for the STAR*D study. Data and sample collection were funded with federal funds from the NIMH, NIH, under contract N01MH90003 to University of Texas Southwestern Medical Center at Dallas (Principal Investigator, A. John Rush).

We thank Jo Steele for technical assistance, the Rutgers University Cell and DNA Repository for extracting DNA and providing samples to our laboratories, and most importantly, we thank the participants of the STAR*D trial, without whom this study would not be possible. The content of this publication does not necessarily reflect the views or policies of the DHHS, nor does mention of trade names, commercial products, or organizations imply endorsement by the U.S. Government.

## Appendix

### Retrospective score tests for autosomal SNP/trait association

Denote genotype by *G* and assume that it is coded as the number of minor alleles present so that *G* = 1 indicates a heterozygote while *G* = 2 indicates a homozygote for the minor allele. Let cases be indicated by *D* = 1 and controls by *D* = 0. Assume there are *t* strata indicated by the random variable *Z*. Let *djz* and *cjz* be the number of cases and controls, respectively, having genotype *j* in stratum *z*. Let *njz*_=_*cjz* + *djz* be the total number of individuals in the study having genotype *j* in stratum z. Let *dz* = *d*_0_*z* + *d*_1_*z* + *d*_2_*z* and *cz* = *c*_0_*z*_+_*c*_1_*z* + *c*_2_*z* We also define *pz*, the frequency of the minor allele in stratum *z*, by *pz* = exp (*γz*)/{1+exp (*γz*)}. Assume Hardy-Weinberg equilibrium among the controls in stratum *z* and that the odds of being a case are given by

| Pr(D=1\|G=j,Z=z)Pr(D=0\|G=j,Z=z)=exp(αz+XjTβ), |
| --- |
where *Xj* is a design vector that codes the genetic model. Then the likelihood, conditional on both disease status and strata, can be written as

| ∏z=1t∏j=02(exp{jγz+I(j=1)ln(2)}∑j∗=02exp{j∗γz+I(j∗=1)ln(2)})cjz×(exp{XjTβ+jγz+I(j=1)ln(2)}∑j∗=02exp{Xj∗Tβj∗γz+I(j∗=1)ln(2)})djz, |
| --- |
where *I*( ) is an indicator function. Note that the intercept parameter *αz* cancel out of the likelihood. From this likelihood and assuming an allelic dosing model in which *Xj* = *j*, it is easy to derive the score statistic *U* under the null hypothesis *β* = 0, i.e.,

| U=∑z=1t∑j=02dj(jz−2p^z), |
| --- |
where *pˆz* = (2*n*_2_*z* + *n*_1_*z*)/ 2*nz*. The variance of *Uβ* is given by

| V=∑t=1zdzcznz2p^z(1−p^z). |
| --- |
A score test is formed by *T* = *U*^2^/*V*. Asymptotically, *T* will have a *χ*^2^ distribution on 1 degree of freedom. Note that the only *djz* in *U* varies from permutation to permutation. The minor allele frequencies within strata do not vary and can be stored for computation of *U* The variance *V* is completely invariant to permuation and therefore does not need to be recomputed each iteration.

### Retrospective score tests for SNP/trait association on the X chromosome

We modify our notation by allowing the above quantities to be indexed by the sex of the patient. For example, we let *djsz* and *cjsz* be the number of cases and controls, respectively, having genotype *j* and sex *s* in stratum *z*. We assume Hardy-Weinberg equilibrium among the controls in stratum *z* (note that Hardy-Weinberg here implies Hardy-Weinberg frequencies in the females and similar allele frequencies between males and females). Let the odds of being a case are given by

| Pr(D=1\|G=j,Z=z,S=s)Pr(D=0\|G=j,Z=z,S=s)=exp(αzs+XjsTβ), |
| --- |
Then the likelihood can be written as

| ∏s=mf∏z=1t∏j=02I(s=f)(exp{jγz+I(j=1,s=f)In(2)}∑j∗=02exp{j∗γz+I(j∗=1,s=f)In(2)})cjz×(exp{XjsTβ+jγz+I(j=1,s=f)In(2)}∑j∗=02exp{Xjs∗Tβj∗γz+I(j∗=1,s=f)In(2)})djz. |
| --- |
From this likelihood and assuming a genetic model in which *Xjs* = *j*2*I*^(^*s*^=^*m*^)^, we can derive the score statistic *U* under the null hypothesis *β* = 0, i.e.,

| U=∑s=mf∑z=1t∑j=02I(s=f)djsz(j2I(s=m)−2p^z), |
| --- |
where *pˆz* (2*n*_2_*fz* + *nfz* +n_1_*mz*)/(2*fz* + *nmz*). The variance of *U* is given by

| V=∑t=1z2dmz+dfz−2dz2nmz+2nfz2p^z(1−p^z). |
| --- |
Again, a score test is formed by *T* =*U*^2^/*V* which will asymptotically be distributed as *χ*^2^ on 1 degree of freedom. Again, only *djsz* in *U* varies from permutation to permutation.

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Teicher MH, Glod C, Cole JO. Emergence of intense suicidal preoccupation during fluoxetine treatment. Am J Psychiatry. 1990;147:207–210. doi: 10.1176/ajp.147.2.207.  [DOI](https://doi.org/10.1176/ajp.147.2.207) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2301661/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Psychiatry&title=Emergence%20of%20intense%20suicidal%20preoccupation%20during%20fluoxetine%20treatment&author=MH%20Teicher&author=C%20Glod&author=JO%20Cole&volume=147&publication_year=1990&pages=207-210&pmid=2301661&doi=10.1176/ajp.147.2.207&)

2. Khan A, Khan S, Kolts R, Brown WA. Suicide rates in clinical trials of SSRIs, other antidepressants, and placebo: analysis of FDA reports. Am J Psychiatry. 2003;160:790–792. doi: 10.1176/appi.ajp.160.4.790.  [DOI](https://doi.org/10.1176/appi.ajp.160.4.790) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12668373/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Psychiatry&title=Suicide%20rates%20in%20clinical%20trials%20of%20SSRIs,%20other%20antidepressants,%20and%20placebo:%20analysis%20of%20FDA%20reports&author=A%20Khan&author=S%20Khan&author=R%20Kolts&author=WA%20Brown&volume=160&publication_year=2003&pages=790-792&pmid=12668373&doi=10.1176/appi.ajp.160.4.790&)

3. Food and Drug Administration. Antidepressant Use in Children, Adolescents and Adults. 2005 Online. 2-5-2006.  [Google Scholar](https://scholar.google.com/scholar_lookup?Food%20and%20Drug%20Administration.%20Antidepressant%20Use%20in%20Children,%20Adolescents%20and%20Adults.%202005%20Online.%202-5-2006.)

4. Murray ML, Thompson M, Santosh PJ, Wong IC. Effects of the committee on safety of medicines advice on antidepressant prescribing to children and adolescents in the UK. Drug Saf. 2005;28:1151–1157. doi: 10.2165/00002018-200528120-00009.  [DOI](https://doi.org/10.2165/00002018-200528120-00009) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16329717/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Saf&title=Effects%20of%20the%20committee%20on%20safety%20of%20medicines%20advice%20on%20antidepressant%20prescribing%20to%20children%20and%20adolescents%20in%20the%20UK&author=ML%20Murray&author=M%20Thompson&author=PJ%20Santosh&author=IC%20Wong&volume=28&publication_year=2005&pages=1151-1157&pmid=16329717&doi=10.2165/00002018-200528120-00009&)

5. Centers for Disease Control and Prevention. Suicide Trends Among Youths and Young Adults Aged 10--24 Years --- United States, 1990--2004. Online. 9-5-2007. 12-15-2007.  [Google Scholar](https://scholar.google.com/scholar_lookup?Centers%20for%20Disease%20Control%20and%20Prevention.%20Suicide%20Trends%20Among%20Youths%20and%20Young%20Adults%20Aged%2010--24%20Years%20---%20United%20States,%201990--2004.%20Online.%209-5-2007.%2012-15-2007.)

6. Gibbons RD, Brown CH, Hur K, Marcus SM, Bhaumik DK, Erkens JA, et al. Early evidence on the effects of regulators' suicidality warnings on SSRI prescriptions and suicide in children and adolescents. Am J Psychiatry. 2007;164:1356–1363. doi: 10.1176/appi.ajp.2007.07030454.  [DOI](https://doi.org/10.1176/appi.ajp.2007.07030454) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17728420/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Psychiatry&title=Early%20evidence%20on%20the%20effects%20of%20regulators'%20suicidality%20warnings%20on%20SSRI%20prescriptions%20and%20suicide%20in%20children%20and%20adolescents&author=RD%20Gibbons&author=CH%20Brown&author=K%20Hur&author=SM%20Marcus&author=DK%20Bhaumik&volume=164&publication_year=2007&pages=1356-1363&pmid=17728420&doi=10.1176/appi.ajp.2007.07030454&)

7. Food and Drug Administration. Suicidality in Children and Adolescents Being Treated With Antidepressant Medications. Online. 10-15-2004. 2-5-2006.  [Google Scholar](https://scholar.google.com/scholar_lookup?Food%20and%20Drug%20Administration.%20Suicidality%20in%20Children%20and%20Adolescents%20Being%20Treated%20With%20Antidepressant%20Medications.%20Online.%2010-15-2004.%202-5-2006.)

8. Laje G, Paddock S, Manji H, Rush AJ, Wilson AF, Charney D, et al. Genetic markers of suicidal ideation emerging during citalopram treatment of major depression. Am J Psychiatry. 2007;164:1530–1538. doi: 10.1176/appi.ajp.2007.06122018.  [DOI](https://doi.org/10.1176/appi.ajp.2007.06122018) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17898344/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Psychiatry&title=Genetic%20markers%20of%20suicidal%20ideation%20emerging%20during%20citalopram%20treatment%20of%20major%20depression&author=G%20Laje&author=S%20Paddock&author=H%20Manji&author=AJ%20Rush&author=AF%20Wilson&volume=164&publication_year=2007&pages=1530-1538&pmid=17898344&doi=10.1176/appi.ajp.2007.06122018&)

9. Perlis RH, Purcell S, Fava M, Fagerness J, Rush AJ, Trivedi MH, et al. Association between treatment-emergent suicidal ideation with citalopram and polymorphisms near cyclic adenosine monophosphate response element binding protein in the STAR*D study. Arch Gen Psychiatry. 2007;64:689–697. doi: 10.1001/archpsyc.64.6.689.  [DOI](https://doi.org/10.1001/archpsyc.64.6.689) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17548750/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arch%20Gen%20Psychiatry&title=Association%20between%20treatment-emergent%20suicidal%20ideation%20with%20citalopram%20and%20polymorphisms%20near%20cyclic%20adenosine%20monophosphate%20response%20element%20binding%20protein%20in%20the%20STAR*D%20study&author=RH%20Perlis&author=S%20Purcell&author=M%20Fava&author=J%20Fagerness&author=AJ%20Rush&volume=64&publication_year=2007&pages=689-697&pmid=17548750&doi=10.1001/archpsyc.64.6.689&)

10. Fava M, Rush AJ, Trivedi MH, Nierenberg AA, Thase ME, Sackeim HA, et al. Background and rationale for the sequenced treatment alternatives to relieve depression (STAR*D) study. Psychiatr Clin North Am. 2003;26:457–494. doi: 10.1016/s0193-953x(02)00107-7. x.  [DOI](https://doi.org/10.1016/s0193-953x(02)00107-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12778843/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Psychiatr%20Clin%20North%20Am&title=Background%20and%20rationale%20for%20the%20sequenced%20treatment%20alternatives%20to%20relieve%20depression%20(STAR*D)%20study&author=M%20Fava&author=AJ%20Rush&author=MH%20Trivedi&author=AA%20Nierenberg&author=ME%20Thase&volume=26&publication_year=2003&pages=457-494&pmid=12778843&doi=10.1016/s0193-953x(02)00107-7&)

11. Rush AJ, Fava M, Wisniewski SR, Lavori PW, Trivedi MH, Sackeim HA, et al. Sequenced Treatment Alternatives to Relieve Depression (STAR*D): rationale and design. Control Clin Trials. 2004;25:119–142. doi: 10.1016/s0197-2456(03)00112-0.  [DOI](https://doi.org/10.1016/s0197-2456(03)00112-0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15061154/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Control%20Clin%20Trials&title=Sequenced%20Treatment%20Alternatives%20to%20Relieve%20Depression%20(STAR*D):%20rationale%20and%20design&author=AJ%20Rush&author=M%20Fava&author=SR%20Wisniewski&author=PW%20Lavori&author=MH%20Trivedi&volume=25&publication_year=2004&pages=119-142&pmid=15061154&doi=10.1016/s0197-2456(03)00112-0&)

12. Hamilton M. A rating scale for depression. J Neurol Neurosurg Psychiatry. 1960;23:56–62. doi: 10.1136/jnnp.23.1.56.  [DOI](https://doi.org/10.1136/jnnp.23.1.56) | [PMC free article](/articles/PMC495331/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14399272/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Neurol%20Neurosurg%20Psychiatry&title=A%20rating%20scale%20for%20depression&author=M%20Hamilton&volume=23&publication_year=1960&pages=56-62&pmid=14399272&doi=10.1136/jnnp.23.1.56&)

13. Hamilton M. Development of a rating scale for primary depressive illness. Br J Soc Clin Psychol. 1967;6:278–296. doi: 10.1111/j.2044-8260.1967.tb00530.x.  [DOI](https://doi.org/10.1111/j.2044-8260.1967.tb00530.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/6080235/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Soc%20Clin%20Psychol&title=Development%20of%20a%20rating%20scale%20for%20primary%20depressive%20illness&author=M%20Hamilton&volume=6&publication_year=1967&pages=278-296&pmid=6080235&doi=10.1111/j.2044-8260.1967.tb00530.x&)

14. American Psychiatric Association. Diagnostic and Statistical Manual of Mental Disorders, Fourth Edition (DSM-IV) Washington, DC: American Psychiatric Association Press Inc.; 1994.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Diagnostic%20and%20Statistical%20Manual%20of%20Mental%20Disorders,%20Fourth%20Edition%20(DSM-IV)&publication_year=1994&)

15. Trivedi MH, Rush AJ, Wisniewski SR, Nierenberg AA, Warden D, Ritz L, et al. Evaluation of outcomes with citalopram for depression using measurement-based care in STAR*D: implications for clinical practice. Am J Psychiatry. 2006;163:28–40. doi: 10.1176/appi.ajp.163.1.28.  [DOI](https://doi.org/10.1176/appi.ajp.163.1.28) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16390886/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Psychiatry&title=Evaluation%20of%20outcomes%20with%20citalopram%20for%20depression%20using%20measurement-based%20care%20in%20STAR*D:%20implications%20for%20clinical%20practice&author=MH%20Trivedi&author=AJ%20Rush&author=SR%20Wisniewski&author=AA%20Nierenberg&author=D%20Warden&volume=163&publication_year=2006&pages=28-40&pmid=16390886&doi=10.1176/appi.ajp.163.1.28&)

16. McMahon FJ, Buervenich S, Charney D, Lipsky R, Rush AJ, Wilson AF, et al. Variation in the Gene Encoding the Serotonin 2A Receptor Is Associated with Outcome of Antidepressant Treatment. Am J Hum Genet. 2006;78:804–814. doi: 10.1086/503820.  [DOI](https://doi.org/10.1086/503820) | [PMC free article](/articles/PMC1474035/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16642436/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Variation%20in%20the%20Gene%20Encoding%20the%20Serotonin%202A%20Receptor%20Is%20Associated%20with%20Outcome%20of%20Antidepressant%20Treatment&author=FJ%20McMahon&author=S%20Buervenich&author=D%20Charney&author=R%20Lipsky&author=AJ%20Rush&volume=78&publication_year=2006&pages=804-814&pmid=16642436&doi=10.1086/503820&)

17. Rush AJ, Carmody TJ, Reimitz PE. The Inventory of Depressive Symptomatology (IDS): clinician (IDS-C) and self-report (IDS-SR) ratings of depressive symptoms. Int J Methods Psychiatr Res. 2000;9:45–59.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Methods%20Psychiatr%20Res&title=The%20Inventory%20of%20Depressive%20Symptomatology%20(IDS):%20clinician%20(IDS-C)%20and%20self-report%20(IDS-SR)%20ratings%20of%20depressive%20symptoms&author=AJ%20Rush&author=TJ%20Carmody&author=PE%20Reimitz&volume=9&publication_year=2000&pages=45-59&)

18. Rush AJ, Trivedi MH, Ibrahim HM, Carmody TJ, Arnow B, Klein DN, et al. The 16-Item Quick Inventory of Depressive Symptomatology (QIDS), clinician rating (QIDS-C), and self-report (QIDS-SR): a psychometric evaluation in patients with chronic major depression. Biol Psychiatry. 2003;54:573–583. doi: 10.1016/s0006-3223(02)01866-8.  [DOI](https://doi.org/10.1016/s0006-3223(02)01866-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12946886/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biol%20Psychiatry&title=The%2016-Item%20Quick%20Inventory%20of%20Depressive%20Symptomatology%20(QIDS),%20clinician%20rating%20(QIDS-C),%20and%20self-report%20(QIDS-SR):%20a%20psychometric%20evaluation%20in%20patients%20with%20chronic%20major%20depression&author=AJ%20Rush&author=MH%20Trivedi&author=HM%20Ibrahim&author=TJ%20Carmody&author=B%20Arnow&volume=54&publication_year=2003&pages=573-583&pmid=12946886&doi=10.1016/s0006-3223(02)01866-8&)

19. Trivedi MH, Rush AJ, Ibrahim HM, Carmody TJ, Biggs MM, Suppes T, et al. The Inventory of Depressive Symptomatology, Clinician Rating (IDS-C) and Self-Report (IDS-SR), and the Quick Inventory of Depressive Symptomatology, Clinician Rating (QIDS-C) and Self-Report (QIDS-SR) in public sector patients with mood disorders: a psychometric evaluation. Psychol Med. 2004;34:73–82. doi: 10.1017/s0033291703001107.  [DOI](https://doi.org/10.1017/s0033291703001107) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14971628/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Psychol%20Med&title=The%20Inventory%20of%20Depressive%20Symptomatology,%20Clinician%20Rating%20(IDS-C)%20and%20Self-Report%20(IDS-SR),%20and%20the%20Quick%20Inventory%20of%20Depressive%20Symptomatology,%20Clinician%20Rating%20(QIDS-C)%20and%20Self-Report%20(QIDS-SR)%20in%20public%20sector%20patients%20with%20mood%20disorders:%20a%20psychometric%20evaluation&author=MH%20Trivedi&author=AJ%20Rush&author=HM%20Ibrahim&author=TJ%20Carmody&author=MM%20Biggs&volume=34&publication_year=2004&pages=73-82&pmid=14971628&doi=10.1017/s0033291703001107&)

20. Rush AJ, Trivedi MH, Carmody TJ, Ibrahim HM, Markowitz JC, Keitner GI, et al. Self-reported depressive symptom measures: sensitivity to detecting change in a randomized, controlled trial of chronically depressed, nonpsychotic outpatients. Neuropsychopharmacology. 2005;30:405–416. doi: 10.1038/sj.npp.1300614.  [DOI](https://doi.org/10.1038/sj.npp.1300614) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15578008/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neuropsychopharmacology&title=Self-reported%20depressive%20symptom%20measures:%20sensitivity%20to%20detecting%20change%20in%20a%20randomized,%20controlled%20trial%20of%20chronically%20depressed,%20nonpsychotic%20outpatients&author=AJ%20Rush&author=MH%20Trivedi&author=TJ%20Carmody&author=HM%20Ibrahim&author=JC%20Markowitz&volume=30&publication_year=2005&pages=405-416&pmid=15578008&doi=10.1038/sj.npp.1300614&)

21. Rush AJ, Bernstein IH, Trivedi MH, Carmody TJ, Wisniewski S, Mundt JC, et al. An evaluation of the quick inventory of depressive symptomatology and the hamilton rating scale for depression: a sequenced treatment alternatives to relieve depression trial report. Biol Psychiatry. 2006;59:493–501. doi: 10.1016/j.biopsych.2005.08.022.  [DOI](https://doi.org/10.1016/j.biopsych.2005.08.022) | [PMC free article](/articles/PMC2929841/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16199008/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biol%20Psychiatry&title=An%20evaluation%20of%20the%20quick%20inventory%20of%20depressive%20symptomatology%20and%20the%20hamilton%20rating%20scale%20for%20depression:%20a%20sequenced%20treatment%20alternatives%20to%20relieve%20depression%20trial%20report&author=AJ%20Rush&author=IH%20Bernstein&author=MH%20Trivedi&author=TJ%20Carmody&author=S%20Wisniewski&volume=59&publication_year=2006&pages=493-501&pmid=16199008&doi=10.1016/j.biopsych.2005.08.022&)

22. Pritchard JK, Stephens M, Rosenberg NA, Donnelly P. Association mapping in structured populations. Am J Hum Genet. 2000;67:170–181. doi: 10.1086/302959.  [DOI](https://doi.org/10.1086/302959) | [PMC free article](/articles/PMC1287075/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10827107/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Association%20mapping%20in%20structured%20populations&author=JK%20Pritchard&author=M%20Stephens&author=NA%20Rosenberg&author=P%20Donnelly&volume=67&publication_year=2000&pages=170-181&pmid=10827107&doi=10.1086/302959&)

23. Pritchard JK, Stephens M, Donnelly P. Inference of population structure using multilocus genotype data. Genetics. 2000;155:945–959. doi: 10.1093/genetics/155.2.945.  [DOI](https://doi.org/10.1093/genetics/155.2.945) | [PMC free article](/articles/PMC1461096/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10835412/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genetics&title=Inference%20of%20population%20structure%20using%20multilocus%20genotype%20data&author=JK%20Pritchard&author=M%20Stephens&author=P%20Donnelly&volume=155&publication_year=2000&pages=945-959&pmid=10835412&doi=10.1093/genetics/155.2.945&)

24. Dudbridge F, Koeleman BP. Efficient computation of significance levels for multiple associations in large studies of correlated data, including genomewide association studies. Am J Hum Genet. 2004;75:424–435. doi: 10.1086/423738.  [DOI](https://doi.org/10.1086/423738) | [PMC free article](/articles/PMC1182021/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15266393/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Efficient%20computation%20of%20significance%20levels%20for%20multiple%20associations%20in%20large%20studies%20of%20correlated%20data,%20including%20genomewide%20association%20studies&author=F%20Dudbridge&author=BP%20Koeleman&volume=75&publication_year=2004&pages=424-435&pmid=15266393&doi=10.1086/423738&)

25. Hosmer D, Lemeshow S. Applied Logistic Regression. Second. New York: Wiley; 2000.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Applied%20Logistic%20Regression&author=D%20Hosmer&author=S%20Lemeshow&publication_year=2000&)

26. Hirschfeld RM, Russell JM. Assessment and treatment of suicidal patients. N Engl J Med. 1997;337:910–915. doi: 10.1056/NEJM199709253371307.  [DOI](https://doi.org/10.1056/NEJM199709253371307) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9302306/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Assessment%20and%20treatment%20of%20suicidal%20patients&author=RM%20Hirschfeld&author=JM%20Russell&volume=337&publication_year=1997&pages=910-915&pmid=9302306&doi=10.1056/NEJM199709253371307&)

27. Vilcek J. Novel interferons. Nat Immunol. 2003;4:8–9. doi: 10.1038/ni0103-8.  [DOI](https://doi.org/10.1038/ni0103-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12496969/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Immunol&title=Novel%20interferons&author=J%20Vilcek&volume=4&publication_year=2003&pages=8-9&pmid=12496969&doi=10.1038/ni0103-8&)

28. Janssen HL, Brouwer JT, van der Mast RC, Schalm SW. Suicide associated with alfa-interferon therapy for chronic viral hepatitis. J Hepatol. 1994;21:241–243. doi: 10.1016/s0168-8278(05)80402-7.  [DOI](https://doi.org/10.1016/s0168-8278(05)80402-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7989716/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Hepatol&title=Suicide%20associated%20with%20alfa-interferon%20therapy%20for%20chronic%20viral%20hepatitis&author=HL%20Janssen&author=JT%20Brouwer&author=RC%20van%20der%20Mast&author=SW%20Schalm&volume=21&publication_year=1994&pages=241-243&pmid=7989716&doi=10.1016/s0168-8278(05)80402-7&)
