# Germline genomic variations associated with childhood acute lymphoblastic leukemia

## Metadata
**Authors:** Lisa R Treviño, Wenjian Yang, Deborah French, Stephen Hunger, William L Carroll, Meenakshi Devidas, Cheryl Willman, Geoffrey Neale, James Downing, Susana Raimondi, Ching-Hon Pui, William E Evans, Mary V Relling
**Journal:** Nature genetics
**Date:** 2009 Aug 16
**DOI:** [10.1038/ng.432](https://doi.org/10.1038/ng.432)
**PMID:** 19684603
**PMCID:** PMC2762391
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2762391/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC2762391/pdf/nihms-134891.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC2762391/pdf/nihms-134891.pdf)

## Abstract

We identified germline single nucleotide polymorphisms (SNPs) associated with childhood acute lymphoblastic leukemia (ALL) and its subtypes. Using the Affymetrix 500K Mapping array and publicly available genotypes, we identified 18 SNPs whose allele frequency differed (P<1×10−5) between a pediatric ALL population (n=317) and non-ALL controls (n=17,958). Six of these SNPs differed (P≤0.05) in allele frequency among four ALL subtypes. Two SNPs in ARID5B not only differed between ALL and non-ALL groups (rs10821936, P=1.4×10−15, odds ratio[OR]=1.91; rs10994982, P=5.7×10−9, OR=1.62) but also distinguished B-hyperdiploid ALL from other subtypes (rs10821936, P=1.62 ×10−5, OR=2.17; rs10994982, P=0.003, OR 1.72). These ARID5B SNPs also distinguished B-hyperdiploid ALL from other subtypes in an independent validation cohort (n=124 children with ALL) (P=0.003 and P=0.0008, OR 2.45 and 2.86, respectively) and were associated with methotrexate accumulation and gene expression pattern in leukemic lymphoblasts. We conclude that germline genomic variations affect susceptibility to and characteristics of specific ALL subtypes.

## Methods

### Patients and DNA samples

We analyzed germline DNA from 441 children of European descent with the four most common ALL subtypes (see below). The discovery cohort consisted of 317 patients of European descent, including 262 patients from the St. Jude Children’s Research Hospital Total XIIIB (1994-1998) and XV (2000-2007) ALL protocols[34](#R34)^, ^[35](#R35) and 55 patients (all with B-precursor ALL subtypes) from the Children’s Oncology Group 9906 study.[36](#R36) The validation cohort consisted of the next 124 children of European descent enrolled on the St. Jude Total XV ALL protocol. This study was approved by the St. Jude Institutional Review Board and signed informed consent was obtained from patients, parents, or guardians, as appropriate.

### External data sets

We obtained SNP allele frequency and genotype data from three groups of European descent to serve as non-ALL controls. One group comprised participants in the Wellcome Trust Case Control Consortium (WTCCC; n=14,311; [http://www.wtccc.uk.org](http://www.wtccc.uk.org)),[37](#R37) excluding those with bipolar disease. The other two groups were the Genetic Association Informative Network (GAIN; [http://www.genome.gov/19518664](http://www.genome.gov/19518664)) schizophrenia (phs000021.v1.p1; n=2,601) and bipolar disorder (phs000017.v1.p1; n=1,046)[38](#R38)^-^[40](#R40) study cohorts. Because the prevalence of adult survivors of childhood ALL is less than 1:1,000, these three groups were considered non-ALL controls.

### Characterization of ALL molecular and immunophenotypic subgroups

The immunophenotyping and genotyping of leukemic lymphoblasts from St. Jude patients[34](#R34) and COG patients were previously described.[36](#R36)^, ^[41](#R41) We analyzed only patients with the four most common, non-overlapping ALL subtypes: B-lineage ALL with no defined genetic or chromosomal abnormalities (B-other; n=121 in discovery cohort, n=44 in validation cohort); B-lineage hyperdiploid ALL with more than 50 chromosomes (B-hyperdiploid; n=108 and n=36); B-lineage ALL bearing the t(12;21)/*ETV6-RUNX1* fusion (n=45 and n=20); and T-cell ALL (n=43 and n=24) ([Supplementary Table 8](#SD1) online).

### Genotyping and SNP Filtering Criteria

DNA was extracted from the blood of patients with ALL during complete remission. For patients in the discovery cohort and for 65 patients in the validation cohort, 500 ng of DNA was digested with *Nsp* and *Sty* restriction enzymes for the 500K Array Set chips (Affymetrix, Santa Clara, CA). DNA was amplified, labeled, and hybridized to chips as described.[30](#R30) The chips were scanned, and genotype calls were made by using the Bayesian Robust Linear Multichip with Mahalanobis Distance (BRLMM) algorithm for a total of 500,568 possible SNPs interrogated. Genotyping for 59 patients in the validation cohort was performed by using the Affymetrix Genome-Wide Human SNP Array 6.0 based on the Birdseed genotype-calling algorithm, which overlaps with 482,251 of the 500,568 SNPs on the 500K chip. Visual inspection of original allele-specific signal intensity plots (i.e. theta plots) was carried out to ensure that genotyping calls clustered distinctly. Genotyping for participants in the Wellcome Trust Case Control Consortium study was performed as reported, using the Affymetrix 500K Mapping Array sets.[37](#R37) The GAIN participants were analyzed by using the Affymetrix Genome-Wide Human SNP Array 6.0. The confirmation of *ARID5B* SNP genotypes in 386 ALL cases was performed using iPLEX from Sequenom, Inc., in the University of Chicago’s Genetic Services Laboratories.

SNPs with a genotyping call rate < 96% (n=84,032) in both the St. Jude and COG data sets were excluded in the discovery cohort. Of the remaining 416,536 SNPs interrogated, only those with adequate quality control measures in the WTCCC cohort [37](#R37) and both GAIN cohorts ([http://www.genome.gov/19518664](http://www.genome.gov/19518664)) were included in the genome-wide association analysis (n=307,944; [Supplementary Fig. 1](#SD1)).

### Sequencing of exon 3 and exon 4

Ten nanograms of genomic DNA from 63 of the CEPH HapMap samples was used to amplify and sequence exon 3 and exon 4 of the *ARID5B* gene ([Supplementary Table 9](#SD1)). Sequencing of the amplified product was performed using the dye-terminator chemistry (Applied Biosystems) in the St. Jude Children’s Research Hospitals’ Hartwell Center for Bioinformatics.

### Measurement of methotrexate polyglutamate accumulation

*In vivo* intracellular accumulation of methotrexate polyglutamate metabolites was measured as previously described[42](#R42)^-^[44](#R44) in bone marrow leukemic lymphoblasts from 118 patients. Bone marrow samples were obtained 42-44 hours after a single treatment with high-dose methotrexate (1 g/m^2^ given intravenously over 4 or 24 hours) during the “upfront window” before remission induction therapy.[34](#R34)^, ^[35](#R35)

### Gene expression profiling

Total RNA was extracted (TriReagent, MRC, Cincinnati, OH, USA) from cryopreserved mononuclear cell suspensions from bone marrow obtained from 156 patients of European descent at the time of diagnosis of ALL. The Affymetrix HG-U133A 2.0 Array (Affymetrix Inc, Santa Clara, CA, USA), comprising more than 22,283 probe sets, was used to interrogate the expression of RNA as described.[45](#R45)^, ^[46](#R46) Gene expression data were analyzed by using the Affymetrix MAS5.0 algorithm.

### Statistical analysis

The ancestry of patients with ALL in both the discovery and validation cohorts was inferred on the basis of the approximately 200,000 SNPs with a call rate > 99%. We used the genotypes of samples from the International HapMap project ([www.hapmap.org](http://www.hapmap.org); Phase II; 210 unrelated individuals of known ancestry) as a reference population and used STRUCTURE[47](#R47) to assess the percentage of European, African, and Asian ancestry for each patient. Patients whose ancestry was greater than 90% European were included in the study; however, almost identical SNP associations were observed when we used more stringent criteria (95% vs. 90%) to define European ancestry ([Supplementary Table 10](#SD1)).

R 2.6.1 statistical software ([http://www.r-project.org/](http://www.r-project.org/)) was used for analysis. For each SNP, logistic regression was used to compare the frequency of the B allele (e.g., for genotypes AA, AB, and BB, the frequency of the B allele is 0, 1, and 2) between the ALL patient population and the combined control group and between the ALL patient population and each of the three non-ALL control groups. A multinomial log-linear model was used to compare the allele frequencies of selected SNPs among the four ALL subtypes. Logistic regression was also used to compare allele frequencies between single ALL subtypes and all other subtypes combined (e.g., B-hyperdiploid ALL vs. non-B-hyperdiploid ALL; [Supplementary Fig. 1](#SD1)). All odds ratios reported are allelic odds ratios unless noted otherwise. All P values are reported as two-tailed P values.

The association between SNPs in *ARID5B* and leukemia-cell gene expression was analyzed by multiple linear regression for each of the 22,283 probe sets on the Affymetrix HG-U133A 2.0 Array (Affymetrix Inc, Santa Clara, CA, USA). The probe sets were rank-ordered by their significance in the regression model as assessed by analysis of variance (ANOVA). False discovery rates were estimated by using the q-value method. Multiple linear regression analysis was also used to compare SNP genotypes with methotrexate polyglutamate accumulation.

### Data analysis overview

In the first step, we compared the allele frequencies of 307,944 SNP genotypes in the discovery cohort of 317 patients with ALL and the non-ALL control groups ([Supplementary Fig. 1](#SD1)). For the second step, SNPs whose allele frequency differed between patients in the ALL and combined non-ALL group (P < 1.0 × 10^−5^) and between the ALL group and each of the three individual non-ALL groups (P < 0.01) were then compared among the four ALL subtypes. SNPs that distinguished among ALL subtypes were then tested in an independent validation cohort of 124 children of European descent with ALL. As a secondary analysis, we compared SNP genotypes only within ALL cases in the discovery cohort that distinguished among the four main ALL subtypes, without considering the non-ALL controls (Supplementary Tables [4](#SD1)-[7](#SD1)).

## Supplementary Material

## Acknowledgments

We thank our protocol co-investigators, clinical and research staff (particularly Ms. Pamela McGill, Dr. Jean Cai, Dr. Shih-Hsiang Chen, Ms. Nancy Kornegay and Dr. Jennifer Pauley), the Hartwell Center for Bioinformatics and Biotechnology, Dr. Soma Das at the Genetic Services Laboratory at the University of Chicago and the patients and their families who participated. We thank Dr. Robert Whitson from City of Hope Hospital for constructive discussions.

This study makes use of data generated by the Wellcome Trust Case-Control Consortium. A full list of the investigators who contributed to the generation of the data is available from [www.wtccc.org.uk](http://www.wtccc.org.uk). Funding for the project was provided by the Wellcome Trust under award 076113.

Funding support for the Genome-Wide Association of Schizophrenia Study was provided by the National Institute of Mental Health (R01 MH67257, R01 MH59588, R01 MH59571, R01 MH59565, R01 MH59587, R01 MH60870, R01 MH59566, R01 MH59586, R01 MH61675, R01 MH60879, R01 MH81800, U01 MH46276, U01 MH46289 U01 MH46318, U01 MH79469, and U01 MH79470) and the genotyping of samples was provided through the Genetic Association Information Network (GAIN). The datasets used for the analyses described in this manuscript were obtained from the database of Genotype and Phenotype database (dbGaP) found at [http://www.ncbi.nlm.nih.gov/gap](http://www.ncbi.nlm.nih.gov/gap) through dbGaP (accession number phs000021.v1.p1). Samples and associated phenotype data for the Genome-Wide Association of Schizophrenia Study were provided by the Molecular Genetics of Schizophrenia Collaboration (PI: Pablo V. Gejman, Evanston Northwestern Healthcare (ENH) and Northwestern University, Evanston, IL, USA).

Funding support for the Whole Genome Association Study of Bipolar Disorder was provided by the National Institute of Mental Health (NIMH) and the genotyping of samples was provided through the Genetic Association Information Network (GAIN). The datasets used for the analyses described in this manuscript were obtained from the database of Genotype and Phenotype database (dbGaP) found at [http://www.ncbi.nlm.nih.gov/gap](http://www.ncbi.nlm.nih.gov/gap) through dbGaP (accession number phs000017.v1.p1). Samples and associated phenotype data for the Collaborative Genomic Study of Bipolar Disorder were provided by The NIMH Genetics Initiative for Bipolar Disorder. Data and biomaterials were collected in four projects that participated in the NIMH Bipolar Disorder Genetics Initiative. From 1991-98, the Principal Investigators and Co-Investigators were: Indiana University, Indianapolis, IN, U01 MH46282, John Nurnberger, M.D., Ph.D., Marvin Miller, M.D., and Elizabeth Bowman, M.D.; Washington University, St. Louis, MO, U01 MH46280, Theodore Reich, M.D., Allison Goate, Ph.D., and John Rice, Ph.D.; Johns Hopkins University, Baltimore, MD U01 MH46274, J. Raymond DePaulo, Jr., M.D., Sylvia Simpson, M.D., MPH, and Colin Stine, Ph.D.; NIMH Intramural Research Program, Clinical Neurogenetics Branch, Bethesda, MD, Elliot Gershon, M.D., Diane Kazuba, B.A., and Elizabeth Maxwell, M.S.W. Data and biomaterials were collected as part of ten projects that participated in the NIMH Bipolar Disorder Genetics Initiative. From 1999-03, the Principal Investigators and Co-Investigators were: Indiana University, Indianapolis, IN, R01 MH59545, John Nurnberger, M.D., Ph.D., Marvin J. Miller, M.D., Elizabeth S. Bowman, M.D., N. Leela Rau, M.D., P. Ryan Moe, M.D., Nalini Samavedy, M.D., Rif El-Mallakh, M.D. (at University of Louisville), Husseini Manji, M.D. (at Wayne State University), Debra A. Glitz, M.D. (at Wayne State University), Eric T. Meyer, M.S., Carrie Smiley, R.N., Tatiana Foroud, Ph.D., Leah Flury, M.S., Danielle M. Dick, Ph.D., Howard Edenberg, Ph.D.; Washington University, St. Louis, MO, R01 MH059534, John Rice, Ph.D, Theodore Reich, M.D., Allison Goate, Ph.D., Laura Bierut, M.D. ; Johns Hopkins University, Baltimore, MD, R01 MH59533, Melvin McInnis M.D. , J. Raymond DePaulo, Jr., M.D., Dean F. MacKinnon, M.D., Francis M. Mondimore, M.D., James B. Potash, M.D., Peter P. Zandi, Ph.D, Dimitrios Avramopoulos, and Jennifer Payne; University of Pennsylvania, PA, R01 MH59553, Wade Berrettini M.D.,Ph.D. ; University of California at Irvine, CA, R01 MH60068, William Byerley M.D., and Mark Vawter M.D. ; University of Iowa, IA, R01 MH059548, William Coryell M.D. , and Raymond Crowe M.D. ; University of Chicago, IL, R01 MH59535, Elliot Gershon, M.D., Judith Badner Ph.D. , Francis McMahon M.D. , Chunyu Liu Ph.D., Alan Sanders M.D., Maria Caserta, Steven Dinwiddie M.D., Tu Nguyen, Donna Harakal; University of California at San Diego, CA, R01 MH59567, John Kelsoe, M.D., Rebecca McKinney, B.A.; Rush University, IL, R01 MH059556, William Scheftner M.D. , Howard M. Kravitz, D.O., M.P.H., Diana Marta, B.S., Annette Vaughn-Brown, MSN, RN, and Laurie Bederow, MA; NIMH Intramural Research Program, Bethesda, MD, 1Z01MH002810-01, Francis J. McMahon, M.D., Layla Kassem, PsyD, Sevilla Detera-Wadleigh, Ph.D, Lisa Austin,Ph.D, Dennis L. Murphy, M.D.

Supported by NCI grants CA 51001, CA 078224 , CA 36401 and CA 21765 and the NIH/NIGMS Pharmacogenetics Research Network and Database (U01 GM61393, U01 HL65899, U01GM61374 [http://pharmgkb.org](http://pharmgkb.org)), by a Center of Excellence grant from the State of Tennessee, and by the American Lebanese Syrian Associated Charities (ALSAC).

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Pui CH, Evans WE. Treatment of acute lymphoblastic leukemia. N. Engl. J Med. 2006;354:166–178. doi: 10.1056/NEJMra052603.  [DOI](https://doi.org/10.1056/NEJMra052603) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16407512/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%20Engl.%20J%20Med&title=Treatment%20of%20acute%20lymphoblastic%20leukemia&author=CH%20Pui&author=WE%20Evans&volume=354&publication_year=2006&pages=166-178&pmid=16407512&doi=10.1056/NEJMra052603&)

2. Schnakenberg E, et al. Polymorphisms of methylenetetrahydrofolate reductase (MTHFR) and susceptibility to pediatric acute lymphoblastic leukemia in a German study population. BMC. Med. Genet. 2005;6:23. doi: 10.1186/1471-2350-6-23.  [DOI](https://doi.org/10.1186/1471-2350-6-23) | [PMC free article](/articles/PMC1164414/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15921520/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC.%20Med.%20Genet&title=Polymorphisms%20of%20methylenetetrahydrofolate%20reductase%20(MTHFR)%20and%20susceptibility%20to%20pediatric%20acute%20lymphoblastic%20leukemia%20in%20a%20German%20study%20population&author=E%20Schnakenberg&volume=6&publication_year=2005&pages=23&pmid=15921520&doi=10.1186/1471-2350-6-23&)

3. Urayama KY, et al. MDR1 gene variants, indoor insecticide exposure, and the risk of childhood acute lymphoblastic leukemia. Cancer Epidemiol. Biomarkers Prev. 2007;16:1172–1177. doi: 10.1158/1055-9965.EPI-07-0007.  [DOI](https://doi.org/10.1158/1055-9965.EPI-07-0007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17548681/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Epidemiol.%20Biomarkers%20Prev&title=MDR1%20gene%20variants,%20indoor%20insecticide%20exposure,%20and%20the%20risk%20of%20childhood%20acute%20lymphoblastic%20leukemia&author=KY%20Urayama&volume=16&publication_year=2007&pages=1172-1177&pmid=17548681&doi=10.1158/1055-9965.EPI-07-0007&)

4. Wiemels JL, et al. Methylenetetrahydrofolate reductase (MTHFR) polymorphisms and risk of molecularly defined subtypes of childhood acute leukemia. Proceedings of the National Academy of Sciences of the United States of America. 2001;98:4004–4009. doi: 10.1073/pnas.061408298.  [DOI](https://doi.org/10.1073/pnas.061408298) | [PMC free article](/articles/PMC31169/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11274424/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20National%20Academy%20of%20Sciences%20of%20the%20United%20States%20of%20America&title=Methylenetetrahydrofolate%20reductase%20(MTHFR)%20polymorphisms%20and%20risk%20of%20molecularly%20defined%20subtypes%20of%20childhood%20acute%20leukemia&author=JL%20Wiemels&volume=98&publication_year=2001&pages=4004-4009&pmid=11274424&doi=10.1073/pnas.061408298&)

5. Stanulla M, et al. Polymorphisms within glutathione S-transferase genes (GSTM1, GSTT1, GSTP1) and risk of relapse in childhood B-cell precursor acute lymphoblastic leukemia: a case-control study. Blood. 2000;95:1222–1228.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/10666194/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Polymorphisms%20within%20glutathione%20S-transferase%20genes%20(GSTM1,%20GSTT1,%20GSTP1)%20and%20risk%20of%20relapse%20in%20childhood%20B-cell%20precursor%20acute%20lymphoblastic%20leukemia:%20a%20case-control%20study&author=M%20Stanulla&volume=95&publication_year=2000&pages=1222-1228&pmid=10666194&)

6. Healy J, et al. Promoter SNPs in G1/S checkpoint regulators and their impact on the susceptibility to childhood leukemia. Blood. 2007;109:683–692. doi: 10.1182/blood-2006-02-003236.  [DOI](https://doi.org/10.1182/blood-2006-02-003236) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17008550/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Promoter%20SNPs%20in%20G1/S%20checkpoint%20regulators%20and%20their%20impact%20on%20the%20susceptibility%20to%20childhood%20leukemia&author=J%20Healy&volume=109&publication_year=2007&pages=683-692&pmid=17008550&doi=10.1182/blood-2006-02-003236&)

7. Mathonnet G, Krajinovic M, Labuda D, Sinnett D. Role of DNA mismatch repair genetic polymorphisms in the risk of childhood acute lymphoblastic leukaemia. Br. J. Haematol. 2003;123:45–48. doi: 10.1046/j.1365-2141.2003.04551.x.  [DOI](https://doi.org/10.1046/j.1365-2141.2003.04551.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14510941/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Haematol&title=Role%20of%20DNA%20mismatch%20repair%20genetic%20polymorphisms%20in%20the%20risk%20of%20childhood%20acute%20lymphoblastic%20leukaemia&author=G%20Mathonnet&author=M%20Krajinovic&author=D%20Labuda&author=D%20Sinnett&volume=123&publication_year=2003&pages=45-48&pmid=14510941&doi=10.1046/j.1365-2141.2003.04551.x&)

8. Paulsson K, et al. Evidence for a single-step mechanism in the origin of hyperdiploid childhood acute lymphoblastic leukemia. Genes Chromosomes. Cancer. 2005;44:113–122. doi: 10.1002/gcc.20222.  [DOI](https://doi.org/10.1002/gcc.20222) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15942938/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genes%20Chromosomes.%20Cancer&title=Evidence%20for%20a%20single-step%20mechanism%20in%20the%20origin%20of%20hyperdiploid%20childhood%20acute%20lymphoblastic%20leukemia&author=K%20Paulsson&volume=44&publication_year=2005&pages=113-122&pmid=15942938&doi=10.1002/gcc.20222&)

9. Kager L, et al. Folate pathway gene expression differs in subtypes of acute lymphoblastic leukemia and influences methotrexate pharmacodynamics 10. J. Clin. Invest. 2005;115:110–117. doi: 10.1172/JCI22477.  [DOI](https://doi.org/10.1172/JCI22477) | [PMC free article](/articles/PMC539195/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15630450/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Invest&title=Folate%20pathway%20gene%20expression%20differs%20in%20subtypes%20of%20acute%20lymphoblastic%20leukemia%20and%20influences%20methotrexate%20pharmacodynamics%2010&author=L%20Kager&volume=115&publication_year=2005&pages=110-117&pmid=15630450&doi=10.1172/JCI22477&)

10. Nordgard SH, et al. Genome-wide analysis identifies 16q deletion associated with survival, molecular subtypes, mRNA expression, and germline haplotypes in breast cancer patients. Genes Chromosomes. Cancer. 2008;47:680–696. doi: 10.1002/gcc.20569.  [DOI](https://doi.org/10.1002/gcc.20569) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18398821/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genes%20Chromosomes.%20Cancer&title=Genome-wide%20analysis%20identifies%2016q%20deletion%20associated%20with%20survival,%20molecular%20subtypes,%20mRNA%20expression,%20and%20germline%20haplotypes%20in%20breast%20cancer%20patients&author=SH%20Nordgard&volume=47&publication_year=2008&pages=680-696&pmid=18398821&doi=10.1002/gcc.20569&)

11. Sobti RC, et al. Combined effect of GSTM1, GSTT1 and GSTP1 polymorphisms on histological subtypes of lung cancer. Biomarkers. 2008;13:282–295. doi: 10.1080/13547500701843437.  [DOI](https://doi.org/10.1080/13547500701843437) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18415801/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biomarkers&title=Combined%20effect%20of%20GSTM1,%20GSTT1%20and%20GSTP1%20polymorphisms%20on%20histological%20subtypes%20of%20lung%20cancer&author=RC%20Sobti&volume=13&publication_year=2008&pages=282-295&pmid=18415801&doi=10.1080/13547500701843437&)

12. Kim MS, et al. Frameshift mutation of UVRAG, an autophagy-related gene, in gastric carcinomas with microsatellite instability. Hum. Pathol. 2008;39:1059–1063. doi: 10.1016/j.humpath.2007.11.013.  [DOI](https://doi.org/10.1016/j.humpath.2007.11.013) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18495205/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum.%20Pathol&title=Frameshift%20mutation%20of%20UVRAG,%20an%20autophagy-related%20gene,%20in%20gastric%20carcinomas%20with%20microsatellite%20instability&author=MS%20Kim&volume=39&publication_year=2008&pages=1059-1063&pmid=18495205&doi=10.1016/j.humpath.2007.11.013&)

13. Campbell PJ. Somatic and germline genetics at the JAK2 locus. Nat. Genet. 2009;41:385–386. doi: 10.1038/ng0409-385.  [DOI](https://doi.org/10.1038/ng0409-385) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19338077/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet&title=Somatic%20and%20germline%20genetics%20at%20the%20JAK2%20locus&author=PJ%20Campbell&volume=41&publication_year=2009&pages=385-386&pmid=19338077&doi=10.1038/ng0409-385&)

14. Greaves M. Childhood leukaemia. BMJ. 2002;324:283–287. doi: 10.1136/bmj.324.7332.283.  [DOI](https://doi.org/10.1136/bmj.324.7332.283) | [PMC free article](/articles/PMC1122200/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11823363/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMJ&title=Childhood%20leukaemia&author=M%20Greaves&volume=324&publication_year=2002&pages=283-287&pmid=11823363&doi=10.1136/bmj.324.7332.283&)

15. Chen HC, et al. Genetic polymorphisms of metabolic enzymes CYP1A1, CYP2D6, GSTM1 and GSTT1 and leukemia susceptibility. Eur. J. Cancer Prev. 2008;17:251–258. doi: 10.1097/CEJ.0b013e3282b72093.  [DOI](https://doi.org/10.1097/CEJ.0b013e3282b72093) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18414197/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Cancer%20Prev&title=Genetic%20polymorphisms%20of%20metabolic%20enzymes%20CYP1A1,%20CYP2D6,%20GSTM1%20and%20GSTT1%20and%20leukemia%20susceptibility&author=HC%20Chen&volume=17&publication_year=2008&pages=251-258&pmid=18414197&doi=10.1097/CEJ.0b013e3282b72093&)

16. Pui CH, Evans WE. Acute lymphoblastic leukemia. N. Engl. J. Med. 1998;339:605–615. doi: 10.1056/NEJM199808273390907.  [DOI](https://doi.org/10.1056/NEJM199808273390907) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9718381/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%20Engl.%20J.%20Med&title=Acute%20lymphoblastic%20leukemia&author=CH%20Pui&author=WE%20Evans&volume=339&publication_year=1998&pages=605-615&pmid=9718381&doi=10.1056/NEJM199808273390907&)

17. Pui CH, Relling MV, Downing JR. Acute lymphoblastic leukemia. N. Engl. J. Med. 2004;350:1535–1548. doi: 10.1056/NEJMra023001.  [DOI](https://doi.org/10.1056/NEJMra023001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15071128/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%20Engl.%20J.%20Med&title=Acute%20lymphoblastic%20leukemia&author=CH%20Pui&author=MV%20Relling&author=JR%20Downing&volume=350&publication_year=2004&pages=1535-1548&pmid=15071128&doi=10.1056/NEJMra023001&)

18. Ziino O, et al. Acute lymphoblastic leukemia in children with associated genetic conditions other than Down’s syndrome. The AIEOP experience. Haematologica. 2006;91:139–140.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/16434385/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Haematologica&title=Acute%20lymphoblastic%20leukemia%20in%20children%20with%20associated%20genetic%20conditions%20other%20than%20Down%E2%80%99s%20syndrome.%20The%20AIEOP%20experience&author=O%20Ziino&volume=91&publication_year=2006&pages=139-140&pmid=16434385&)

19. Greaves MF. Speculations on the cause of childhood acute lymphoblastic leukemia. Leukemia. 1988;2:120–125.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/3278171/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Leukemia&title=Speculations%20on%20the%20cause%20of%20childhood%20acute%20lymphoblastic%20leukemia&author=MF%20Greaves&volume=2&publication_year=1988&pages=120-125&pmid=3278171&)

20. Perera FP. Environment and cancer: who are susceptible? Science. 1997;278:1068–1073. doi: 10.1126/science.278.5340.1068.  [DOI](https://doi.org/10.1126/science.278.5340.1068) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9353182/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Environment%20and%20cancer:%20who%20are%20susceptible?&author=FP%20Perera&volume=278&publication_year=1997&pages=1068-1073&pmid=9353182&doi=10.1126/science.278.5340.1068&)

21. Dong LM, et al. Genetic susceptibility to cancer: the role of polymorphisms in candidate genes. JAMA. 2008;299:2423–2436. doi: 10.1001/jama.299.20.2423.  [DOI](https://doi.org/10.1001/jama.299.20.2423) | [PMC free article](/articles/PMC2772197/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18505952/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA&title=Genetic%20susceptibility%20to%20cancer:%20the%20role%20of%20polymorphisms%20in%20candidate%20genes&author=LM%20Dong&volume=299&publication_year=2008&pages=2423-2436&pmid=18505952&doi=10.1001/jama.299.20.2423&)

22. Patsialou A, Wilsker D, Moran E. DNA-binding properties of ARID family proteins. Nucleic Acids Res. 2005;33:66–80. doi: 10.1093/nar/gki145.  [DOI](https://doi.org/10.1093/nar/gki145) | [PMC free article](/articles/PMC546134/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15640446/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=DNA-binding%20properties%20of%20ARID%20family%20proteins&author=A%20Patsialou&author=D%20Wilsker&author=E%20Moran&volume=33&publication_year=2005&pages=66-80&pmid=15640446&doi=10.1093/nar/gki145&)

23. Lahoud MH, et al. Gene targeting of Desrt, a novel ARID class DNA-binding protein, causes growth retardation and abnormal development of reproductive organs. Genome Res. 2001;11:1327–1334. doi: 10.1101/gr.168801.  [DOI](https://doi.org/10.1101/gr.168801) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11483573/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res&title=Gene%20targeting%20of%20Desrt,%20a%20novel%20ARID%20class%20DNA-binding%20protein,%20causes%20growth%20retardation%20and%20abnormal%20development%20of%20reproductive%20organs&author=MH%20Lahoud&volume=11&publication_year=2001&pages=1327-1334&pmid=11483573&doi=10.1101/gr.168801&)

24. Wilsker D, Patsialou A, Dallas PB, Moran E. ARID proteins: a diverse family of DNA binding proteins implicated in the control of cell growth, differentiation, and development. Cell Growth Differ. 2002;13:95–106.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/11959810/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Growth%20Differ&title=ARID%20proteins:%20a%20diverse%20family%20of%20DNA%20binding%20proteins%20implicated%20in%20the%20control%20of%20cell%20growth,%20differentiation,%20and%20development&author=D%20Wilsker&author=A%20Patsialou&author=PB%20Dallas&author=E%20Moran&volume=13&publication_year=2002&pages=95-106&pmid=11959810&)

25. Wilsker D, et al. The DNA-binding properties of the ARID-containing subunits of yeast and mammalian SWI/SNF complexes. Nucleic Acids Res. 2004;32:1345–1353. doi: 10.1093/nar/gkh277.  [DOI](https://doi.org/10.1093/nar/gkh277) | [PMC free article](/articles/PMC390273/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14982958/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=The%20DNA-binding%20properties%20of%20the%20ARID-containing%20subunits%20of%20yeast%20and%20mammalian%20SWI/SNF%20complexes&author=D%20Wilsker&volume=32&publication_year=2004&pages=1345-1353&pmid=14982958&doi=10.1093/nar/gkh277&)

26. Bourquin JP, et al. Identification of distinct molecular phenotypes in acute megakaryoblastic leukemia by gene expression profiling. Proc. Natl. Acad. Sci. U. S. A. 2006;103:3339–3344. doi: 10.1073/pnas.0511150103.  [DOI](https://doi.org/10.1073/pnas.0511150103) | [PMC free article](/articles/PMC1413912/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16492768/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A&title=Identification%20of%20distinct%20molecular%20phenotypes%20in%20acute%20megakaryoblastic%20leukemia%20by%20gene%20expression%20profiling&author=JP%20Bourquin&volume=103&publication_year=2006&pages=3339-3344&pmid=16492768&doi=10.1073/pnas.0511150103&)

27. Chang LW, et al. Computational identification of the normal and perturbed genetic networks involved in myeloid differentiation and acute promyelocytic leukemia. Genome Biol. 2008;9:R38. doi: 10.1186/gb-2008-9-2-r38.  [DOI](https://doi.org/10.1186/gb-2008-9-2-r38) | [PMC free article](/articles/PMC2374711/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18291030/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Biol&title=Computational%20identification%20of%20the%20normal%20and%20perturbed%20genetic%20networks%20involved%20in%20myeloid%20differentiation%20and%20acute%20promyelocytic%20leukemia&author=LW%20Chang&volume=9&publication_year=2008&pages=R38&pmid=18291030&doi=10.1186/gb-2008-9-2-r38&)

28. Molnar A, Georgopoulos K. The Ikaros gene encodes a family of functionally diverse zinc finger DNA-binding proteins. Mol. Cell Biol. 1994;14:8292–8303. doi: 10.1128/mcb.14.12.8292.  [DOI](https://doi.org/10.1128/mcb.14.12.8292) | [PMC free article](/articles/PMC359368/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7969165/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Cell%20Biol&title=The%20Ikaros%20gene%20encodes%20a%20family%20of%20functionally%20diverse%20zinc%20finger%20DNA-binding%20proteins&author=A%20Molnar&author=K%20Georgopoulos&volume=14&publication_year=1994&pages=8292-8303&pmid=7969165&doi=10.1128/mcb.14.12.8292&)

29. Molnar A, et al. The Ikaros gene encodes a family of lymphocyte-restricted zinc finger DNA binding proteins, highly conserved in human and mouse. J. Immunol. 1996;156:585–592.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/8543809/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Immunol&title=The%20Ikaros%20gene%20encodes%20a%20family%20of%20lymphocyte-restricted%20zinc%20finger%20DNA%20binding%20proteins,%20highly%20conserved%20in%20human%20and%20mouse&author=A%20Molnar&volume=156&publication_year=1996&pages=585-592&pmid=8543809&)

30. Mullighan CG, et al. Genome-wide analysis of genetic alterations in acute lymphoblastic leukaemia. Nature. 2007;446:758–764. doi: 10.1038/nature05690.  [DOI](https://doi.org/10.1038/nature05690) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17344859/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Genome-wide%20analysis%20of%20genetic%20alterations%20in%20acute%20lymphoblastic%20leukaemia&author=CG%20Mullighan&volume=446&publication_year=2007&pages=758-764&pmid=17344859&doi=10.1038/nature05690&)

31. Mullighan CG, et al. BCR-ABL1 lymphoblastic leukaemia is characterized by the deletion of Ikaros. Nature. 2008;453:110–114. doi: 10.1038/nature06866.  [DOI](https://doi.org/10.1038/nature06866) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18408710/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=BCR-ABL1%20lymphoblastic%20leukaemia%20is%20characterized%20by%20the%20deletion%20of%20Ikaros&author=CG%20Mullighan&volume=453&publication_year=2008&pages=110-114&pmid=18408710&doi=10.1038/nature06866&)

32. den Boer ML, et al. A subtype of childhood acute lymphoblastic leukaemia with poor treatment outcome: a genome-wide classification study. Lancet Oncol. 2009 doi: 10.1016/S1470-2045(08)70339-5.  [DOI](https://doi.org/10.1016/S1470-2045(08)70339-5) | [PMC free article](/articles/PMC2707020/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19138562/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet%20Oncol&title=A%20subtype%20of%20childhood%20acute%20lymphoblastic%20leukaemia%20with%20poor%20treatment%20outcome:%20a%20genome-wide%20classification%20study&author=ML%20den%20Boer&publication_year=2009&pmid=19138562&doi=10.1016/S1470-2045(08)70339-5&)

33. Mullighan CG, et al. Deletion of IKZF1 and Prognosis in Acute Lymphoblastic Leukemia. N. Engl. J. Med. 2009 doi: 10.1056/NEJMoa0808253.  [DOI](https://doi.org/10.1056/NEJMoa0808253) | [PMC free article](/articles/PMC2674612/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19129520/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%20Engl.%20J.%20Med&title=Deletion%20of%20IKZF1%20and%20Prognosis%20in%20Acute%20Lymphoblastic%20Leukemia&author=CG%20Mullighan&publication_year=2009&pmid=19129520&doi=10.1056/NEJMoa0808253&)

34. Pui CH, et al. Improved outcome for children with acute lymphoblastic leukemia: results of Total Therapy Study XIIIB at St Jude Children’s Research Hospital. Blood. 2004;104:2690–2696. doi: 10.1182/blood-2004-04-1616.  [DOI](https://doi.org/10.1182/blood-2004-04-1616) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15251979/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Improved%20outcome%20for%20children%20with%20acute%20lymphoblastic%20leukemia:%20results%20of%20Total%20Therapy%20Study%20XIIIB%20at%20St%20Jude%20Children%E2%80%99s%20Research%20Hospital&author=CH%20Pui&volume=104&publication_year=2004&pages=2690-2696&pmid=15251979&doi=10.1182/blood-2004-04-1616&)

35. Pui C-H, et al. Treating childhood acute lymphoblastic leukemia without cranial irradiation. New England Journal of Medicine. 2009;360:2730–2741. doi: 10.1056/NEJMoa0900386.  [DOI](https://doi.org/10.1056/NEJMoa0900386) | [PMC free article](/articles/PMC2754320/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19553647/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=New%20England%20Journal%20of%20Medicine&title=Treating%20childhood%20acute%20lymphoblastic%20leukemia%20without%20cranial%20irradiation&author=C-H%20Pui&volume=360&publication_year=2009&pages=2730-2741&pmid=19553647&doi=10.1056/NEJMoa0900386&)

36. Borowitz MJ, et al. Clinical significance of minimal residual disease in childhood acute lymphoblastic leukemia and its relationship to other prognostic factors: a Children’s Oncology Group study. Blood. 2008;111:5477–5485. doi: 10.1182/blood-2008-01-132837.  [DOI](https://doi.org/10.1182/blood-2008-01-132837) | [PMC free article](/articles/PMC2424148/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18388178/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Clinical%20significance%20of%20minimal%20residual%20disease%20in%20childhood%20acute%20lymphoblastic%20leukemia%20and%20its%20relationship%20to%20other%20prognostic%20factors:%20a%20Children%E2%80%99s%20Oncology%20Group%20study&author=MJ%20Borowitz&volume=111&publication_year=2008&pages=5477-5485&pmid=18388178&doi=10.1182/blood-2008-01-132837&)

37. The Wellcome Trust Case Control Consortium Genome-wide association study of 14,000 cases of seven common diseases and 3,000 shared controls. Nature. 2007;447:661–678. doi: 10.1038/nature05911.  [DOI](https://doi.org/10.1038/nature05911) | [PMC free article](/articles/PMC2719288/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17554300/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Genome-wide%20association%20study%20of%2014,000%20cases%20of%20seven%20common%20diseases%20and%203,000%20shared%20controls&volume=447&publication_year=2007&pages=661-678&pmid=17554300&doi=10.1038/nature05911&)

38. Dick DM, et al. Genomewide linkage analyses of bipolar disorder: a new sample of 250 pedigrees from the National Institute of Mental Health Genetics Initiative. Am. J. Hum. Genet. 2003;73:107–114. doi: 10.1086/376562.  [DOI](https://doi.org/10.1086/376562) | [PMC free article](/articles/PMC1180573/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12772088/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Hum.%20Genet&title=Genomewide%20linkage%20analyses%20of%20bipolar%20disorder:%20a%20new%20sample%20of%20250%20pedigrees%20from%20the%20National%20Institute%20of%20Mental%20Health%20Genetics%20Initiative&author=DM%20Dick&volume=73&publication_year=2003&pages=107-114&pmid=12772088&doi=10.1086/376562&)

39. McInnis MG, et al. Genome-wide scan and conditional analysis in bipolar disorder: evidence for genomic interaction in the National Institute of Mental Health genetics initiative bipolar pedigrees. Biol. Psychiatry. 2003;54:1265–1273. doi: 10.1016/j.biopsych.2003.08.001.  [DOI](https://doi.org/10.1016/j.biopsych.2003.08.001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14643094/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biol.%20Psychiatry&title=Genome-wide%20scan%20and%20conditional%20analysis%20in%20bipolar%20disorder:%20evidence%20for%20genomic%20interaction%20in%20the%20National%20Institute%20of%20Mental%20Health%20genetics%20initiative%20bipolar%20pedigrees&author=MG%20McInnis&volume=54&publication_year=2003&pages=1265-1273&pmid=14643094&doi=10.1016/j.biopsych.2003.08.001&)

40. Suarez BK, et al. Genomewide linkage scan of 409 European-ancestry and African American families with schizophrenia: suggestive evidence of linkage at 8p23.3-p21.2 and 11p13.1-q14.1 in the combined sample. Am. J. Hum. Genet. 2006;78:315–333. doi: 10.1086/500272.  [DOI](https://doi.org/10.1086/500272) | [PMC free article](/articles/PMC1380238/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16400611/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Hum.%20Genet&title=Genomewide%20linkage%20scan%20of%20409%20European-ancestry%20and%20African%20American%20families%20with%20schizophrenia:%20suggestive%20evidence%20of%20linkage%20at%208p23.3-p21.2%20and%2011p13.1-q14.1%20in%20the%20combined%20sample&author=BK%20Suarez&volume=78&publication_year=2006&pages=315-333&pmid=16400611&doi=10.1086/500272&)

41. Borowitz MJ, et al. Minimal residual disease detection in childhood precursor-B-cell acute lymphoblastic leukemia: relation to other risk factors. A Children’s Oncology Group study. Leukemia. 2003;17:1566–1572. doi: 10.1038/sj.leu.2403001.  [DOI](https://doi.org/10.1038/sj.leu.2403001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12886244/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Leukemia&title=Minimal%20residual%20disease%20detection%20in%20childhood%20precursor-B-cell%20acute%20lymphoblastic%20leukemia:%20relation%20to%20other%20risk%20factors.%20A%20Children%E2%80%99s%20Oncology%20Group%20study&author=MJ%20Borowitz&volume=17&publication_year=2003&pages=1566-1572&pmid=12886244&doi=10.1038/sj.leu.2403001&)

42. Fabre G, et al. In vitro formation of polyglutamyl derivatives of methotrexate and 7-hydroxymethotrexate in human lymphoblastic leukemia cells. Cancer Res. 1983;43:4648–4652.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/6192907/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=In%20vitro%20formation%20of%20polyglutamyl%20derivatives%20of%20methotrexate%20and%207-hydroxymethotrexate%20in%20human%20lymphoblastic%20leukemia%20cells&author=G%20Fabre&volume=43&publication_year=1983&pages=4648-4652&pmid=6192907&)

43. Masson E, et al. Accumulation of methotrexate polyglutamates in lymphoblasts is a determinant of antileukemic effects in vivo. A rationale for high- dose methotrexate. J. Clin. Invest. 1996;97:73–80. doi: 10.1172/JCI118409.  [DOI](https://doi.org/10.1172/JCI118409) | [PMC free article](/articles/PMC507064/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8550853/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Invest&title=Accumulation%20of%20methotrexate%20polyglutamates%20in%20lymphoblasts%20is%20a%20determinant%20of%20antileukemic%20effects%20in%20vivo.%20A%20rationale%20for%20high-%20dose%20methotrexate&author=E%20Masson&volume=97&publication_year=1996&pages=73-80&pmid=8550853&doi=10.1172/JCI118409&)

44. Synold TW, et al. Blast cell methotrexate-polyglutamate accumulation in vivo differs by lineage, ploidy, and methotrexate dose in acute lymphoblastic leukemia. J. Clin. Invest. 1994;94:1996–2001. doi: 10.1172/JCI117552.  [DOI](https://doi.org/10.1172/JCI117552) | [PMC free article](/articles/PMC294625/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7525652/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Invest&title=Blast%20cell%20methotrexate-polyglutamate%20accumulation%20in%20vivo%20differs%20by%20lineage,%20ploidy,%20and%20methotrexate%20dose%20in%20acute%20lymphoblastic%20leukemia&author=TW%20Synold&volume=94&publication_year=1994&pages=1996-2001&pmid=7525652&doi=10.1172/JCI117552&)

45. Ross ME, et al. Classification of pediatric acute lymphoblastic leukemia by gene expression profiling. Blood. 2003;102:2951–2959. doi: 10.1182/blood-2003-01-0338.  [DOI](https://doi.org/10.1182/blood-2003-01-0338) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12730115/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Classification%20of%20pediatric%20acute%20lymphoblastic%20leukemia%20by%20gene%20expression%20profiling&author=ME%20Ross&volume=102&publication_year=2003&pages=2951-2959&pmid=12730115&doi=10.1182/blood-2003-01-0338&)

46. Yeoh EJ, et al. Classification, subtype discovery, and prediction of outcome in pediatric acute lymphoblastic leukemia by gene expression profiling. Cancer Cell. 2002;1:133–143. doi: 10.1016/s1535-6108(02)00032-6.  [DOI](https://doi.org/10.1016/s1535-6108(02)00032-6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12086872/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Cell&title=Classification,%20subtype%20discovery,%20and%20prediction%20of%20outcome%20in%20pediatric%20acute%20lymphoblastic%20leukemia%20by%20gene%20expression%20profiling&author=EJ%20Yeoh&volume=1&publication_year=2002&pages=133-143&pmid=12086872&doi=10.1016/s1535-6108(02)00032-6&)

47. Pritchard JK, Stephens M, Donnelly P. Inference of population structure using multilocus genotype data. Genetics. 2000;155:945–959. doi: 10.1093/genetics/155.2.945.  [DOI](https://doi.org/10.1093/genetics/155.2.945) | [PMC free article](/articles/PMC1461096/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10835412/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genetics&title=Inference%20of%20population%20structure%20using%20multilocus%20genotype%20data&author=JK%20Pritchard&author=M%20Stephens&author=P%20Donnelly&volume=155&publication_year=2000&pages=945-959&pmid=10835412&doi=10.1093/genetics/155.2.945&)
