# Gefitinib for non-small-cell lung cancer patients with epidermal growth factor receptor gene mutations screened by peptide nucleic acid-locked nucleic acid PCR clamp

## Metadata
**Authors:** A Sutani, Y Nagai, K Udagawa, Y Uchida, N Koyama, Y Murayama, T Tanaka, H Miyazawa, M Nagata, M Kanazawa, K Hagiwara, K Kobayashi
**Journal:** British Journal of Cancer
**Date:** 2006 Nov 14
**DOI:** [10.1038/sj.bjc.6603466](https://doi.org/10.1038/sj.bjc.6603466)
**PMID:** 17106442
**PMCID:** PMC2360739
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2360739/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC2360739/pdf/95-6603466a.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC2360739/pdf/95-6603466a.pdf)

## Abstract

This study was prospectively designed to evaluate a phase II study of gefitinib for non-small-cell lung cancer (NSCLC) patients with epidermal growth factor receptor (EGFR) mutations. Clinical samples were tested for EGFR mutations by peptide nucleic acid-locked nucleic acid PCR clamp, and patients having EGFR mutations were given gefitinib 250 mg daily as the second treatment after chemotherapy. Poor PS patients omitted chemotherapy. Of 107 consecutive patients enrolled, samples from 100 patients were informative, and EGFR mutations were observed in 38 patients. Gefitinib was given to 27 patients with EGFR mutations, and the response rate was 78% (one complete response and 20 partial responses; 95% confidence interval: 58–93%). Median time to progression and median survival time (MST) from gefitinib treatment were 9.4 and 15.4 months, respectively. Grade 3 hepatic toxicity and skin toxicity were observed in one patient each. There were significant differences between EGFR mutations and wild-type patients in response rates (78 vs 14%, P=0.0017), and MST (15.4 vs 11.1 months, P=0.0135). A Cox proportional hazards model indicated that negative EGFR mutation was a secondary prognostic factor (hazards ratio: 2.259, P=0.036). This research showed the need for screening for EGFR mutations in NSCLC patients.

Keywords: lung neoplasms, EGFR, gefitinib, tyrosine kinase inhibitor, clinical trial, PNA-LNA PCR clamp

## PATIENTS AND METHODS

The two-step protocol of this phase II study, that is (i) testing for EGFR mutations by PNA-LNA PCR clamp, and (ii) administering gefitinib to NSCLC patients with EGFR mutations, were approved by the Institutional Review Board (IRB) of Saitama Medical University Hospital. This study was performed in accordance with the Declaration of Helsinki (1964, amended in 2000) of the World Medical Association.

### Primary entry criteria and testing for EGFR mutations

Consecutive NSCLC patients who were admitted in our single institution and gave written informed consent for testing for *EGFR* gene mutations by PNA-LNA PCR clamp, which was designed to detect 11 different EGFR mutations. Detection rate (sensitivity) by PNA-LNA PCR clamp is 89% and its accuracy (specificity) is 100%. In PNA-LNA PCR clamp, existence of other types of EGFR mutation is realised by seeing escape of the inhibition of amplification by the clamp primer, and, in this case, a direct sequencing method is employed to seek other types of EGFR mutation. Finally, overall sensitivity and specificity of this system is 97 and 100%, respectively, using clinical samples (submitting). The cytology specimens were divided into pathology samples (the main sample) and PNA-LNA PCR clamp samples (a small aliquot). When the pathologist confirmed a pathology sample to contain cancer cells (i.e. rated as classes IV or V), the cells in the PNA-LNA PCR clamp samples, which had been collected and stored in the AL buffer (a buffer containing protein denaturant: Qiagen, Hilden Germany), were then subjected to the analysis. While, the paraffin-embedded tissue specimens were serially thin-sliced: one slice was used to confirm the presence of cancer cells under microscopy, whereas the others were investigated by the PNA-LNA PCR reaction.

The PNA-LNA PCR clamp method has been described in detail ([Nagai et al, 2005](#bib10)). Briefly, primers used were

PCR primers were designed manually or by using Primer 3 software ([http://frodo.wi.mit.edu/cgi-bin/primer3/primer3.cgi](http://frodo.wi.mit.edu/cgi-bin/primer3/primer3.cgi)) so that the *T*_m_ values were between 55 and 60°C. Fluorogenic probes containing LNA were manually designed and confirmed by the LNA *T*_m_ prediction tool ([http://lna-tm.com/](http://lna-tm.com/)) to have *T*_m_ values between 54 and 56°C. Peptide nucleic acid clamp primers, 14- to 18- mer in length, were designed according to the guidelines ([Ugozzoli et al, 2004](#bib20)). LNA-containing oligos were synthesised by IDT (Coralville, IA, USA), and PNA oligos were synthesised by Greiner Japan, Tokyo, Japan. For PNA-LNA PCR clamp, PCR primers (200 nM each), fluorogenic probes (100 nM each) and a PNA clamp primer (5 *μ*M) were added to the Basic Mixture containing 25 mM TAPS pH 9.3, 50 mM KCl, 2 mM MgCl_2_, 1 mM 2-mercaptoethanol, 200 *μ*M each of dNTPs and 1.25 U of Takara Ex Taq HS (Takara Bio, Shiga, Japan). For PCR reactions, PCR and the real-time amplification monitoring for the PNA-LNA PCR clamp were performed using Smart Cycler II (Cepheid Sunnyvale, CA, USA). PCR cycling was a 30-s hold at 95°C followed by 45 cycles of 95°C for 3 s and 62°C (exons 18 and 19) or 56°C (exon 21) for 30 s.

### Secondary entry criteria and treatment schedule

After testing for EGFR mutations by PNA-LNA PCR clamp, patients who satisfied the following inclusion criteria were enrolled: (a) EGFR mutations, (b) inoperable stage III–IV and recurrence after operation, (c) measurable region(s), (d) adequate bone marrow (white blood cell count⩾4000 mm^−3^; platelet count⩾100 000 mm^−3^; hemoglobin⩾9.5 g dl^−1^), total bilirubin ⩽1.5 mg dl^−1^, transaminases less than twice the upper limit of normal, and serum creatinine level ⩽1.5 mg dl^−1^, (e) age 20 years, (f) no medical problems severe enough to prevent compliance with the study requirements, and (g) secondary informed consent to be treated by gefitinib.

Gefitinib (250 mg p.o. daily) was given as the second treatment after disease was on progression by cytotoxic chemotherapy for PS 0–2 patients with EGFR mutations. In the case of poorer PS owing to advanced disease, the first line chemotherapy was omitted and gefitinib was administered as the first therapy. The other patients not enrolled into the phase II study were clinically treated by appropriate therapies according to our institutional manual, and their data on EGFR mutation status and survival time were collected and analysed.

### Evaluation

Patients were evaluated by physical examination, chest X-ray, bone scintiscan, computed tomography (CT) of the head, chest and abdomen, and fiberoptic bronchoscopy, and clinical stages were then determined according to the tumour-node-metastasis system. Chest X-rays were assessed at least every 2 weeks after the initial evaluation, and a chest CT was planned to evaluate tumour response and tumour progression. Tumour response was classified in accordance with Response Evaluation Criteria in Solid Tumours.

Before the first course, each patient was subjected to a complete blood cell count (CBC), serum chemistry for renal and hepatic functions, electrolyte analysis and urinalysis. CBC, serum chemistry, electrolyte analysis and urinalysis were assessed at least once a week after the initial evaluation. The NCI Common Toxicity Criteria (version 3) was used to grade organ system damage.

### Statistical analysis

The primary end point of this study was the response to gefitinib for patients with EGFR mutations. Sample size was determined to be 25 patients with *EGFR* gene mutations. We chose a 75% response rate as a desirable target level and a 50% response rate as uninteresting. Our design had a power in excess of 90% and less than 10% type I error. A total number of patients to be tested by PNA-LNA PCR clamp was decided to be more than 100 patients because about 30% of Japanese NSCLC patients were reported to have EGFR mutations in previous articles ([Mitsudomi et al, 2005](#bib9); [Tokumo et al, 2005](#bib18)).

Secondary end points were survival, side effects and clinical usefulness of PNA-LNA PCR clamp. Differences in response to gefitinib and survival after gefitinib therapy between patients with wild *EGFR* genes and those with mutant *EGFR* genes were assessed to indicate a clinical usefulness of screening by PNA-LNA PCR clamp. Furthermore, differences in overall survival from the initial treatment between the groups, and whether EGFR mutations were a prognostic factor were also investigated. Survival curves were drawn using the Kaplan–Meier method, and Logrank was calculated for evaluating survival differences between the groups. A Cox proportional hazards model (multiple variate) using EGFR mutations, sex, stage and PS was also employed using the data from all the patients enrolled by the primary entry criteria. All the analyses were calculated by SPSS® 11.0J.

## RESULTS

From September 2004 to October 2005, samples from 107 Japanese NSCLC patients were tested by PNA-LNA PCR clamp but two patients refused consent for checking for EGFR mutations ([Figure 1](#fig1)). One hundred patients (93%) of the 107 patients provided adequate samples for evaluation of EGFR mutation status, and samples from seven patients did not provide enough DNA. PNA-LNA PCR clamp detected EGFR mutations in 38 patients (38%; 95% confidence interval (CI): 28–48%) who were 15 male and 23 female patients ([Table 1](#tbl1)). Their median age was 62 years old, and, of the 38 patients, 33 patients had adenocarcinoma. Exon 19 deletions, L858R and L861Q were found in 25 (66%), 12 (32%) and 1 (2%) patients, respectively ([Figure 1](#fig1)). On the other hand, 62 patients (51 men/11 women; median age: 66 years; adenocarcinoma: 43 patients) were judged to have wild-type EGFR. There were significant differences between EGFR mutation-positive and EGFR mutation-negative groups with regard to sex (male *vs* female: *P*=0.00001), histology (adenocarcinoma *vs* non-adenocarcinoma: *P*=0.02) and smoking (>20 pack-years *vs* <20 pack-years: *P*=0.003) ([Table 1](#tbl1)).

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/52e0/2360739/e99a8b81a419/95-6603466f1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=2360739_95-6603466f1.jpg)

Patients entered and source of specimen and type of EGFR mutation.

### Table 1. Patients' characteristics.

|   | Mutation | Wild-type |   |
| --- | --- | --- | --- |
|   | (n=38) | (n=62) | P-value |
| Male/female | 15/23 pts | 51/11 pts | 0.00001 |
| Median age (years) (s.d., range) | 62 (10.0, 44–79) | 66 (12.0, 32–81) | 0.09 |
|   |   |   |   |
| Stage |   |   |   |
| I | 1 pt (2.6%) | 3 pts (4.8%) | 0.175 |
| II | 1 pt (2.6%) | 2 pts (3.2%) |   |
| III | 9 pts (23.7%) | 23 pts (37.1%) |   |
| IV | 23 pts (60.5%) | 22 pts (35.5%) |   |
| Relapse | 4 pts (10.5%) | 12 pts (19.4%) |   |
|   |   |   |   |
| Histology |   |   |   |
| Adenocarcinoma | 33 pts (86.8%) | 43 pts (69.4%) | 0.02 |
| Squamous cell carcinoma | 2 pts (5.3%) | 12 pts (19.4%) |   |
| Adenosquamous | 1 pt (2.6%) | 1 pt (1.6%) |   |
| Large cell carcinoma | 0 pt (0.0%) | 1 pt (1.6%) |   |
| Undifferentiated | 2 pts (5.3%) | 5 pts (8.1%) |   |
|   |   |   |   |
| Smoking |   |   |   |
| >20 pack-years | 13 pts (34.2%) | 40 pts (64.5) | 0.003 |
|   |   |   |   |
| ECOG PS |   |   |   |
| 0–2 | 34 pts (89.5%) | 55 pts (88.7%) | 0.906 |
| 3–4 | 4 pts (10.5%) | 7 pts (11.3%) |   |
|   |   |   |   |
| Treatments a |   |   |   |
| Operation | 10 pts (26.3%) | 23 pts (37.1%) | 0.948 |
| Chemotherapy | 30 pts (78.9%) | 43 pts (69.4%) |   |
| Irradiation | 2 pts (5.3%) | 6 pts (9.7%) |   |

Table 1 Caption: ECOG PS=Eastern co-operative oncology group performance status; Pts=patients. aAll the treatments which were given to patients for the intervening periods of the diseases.

### Phase II study

Of the 38 patients with EGFR mutations, gefitinib was given to 27 patients. The other 11 patients were not treated by gefitinib because they did not meet the secondary entry criteria.

Four patients and 23 patients were given gefitinib as the first-line and the second-line treatment, respectively. All of the 27 patients were assessed for response. One patient showed a complete response (CR) and 20 patients showed partial responses (PRs). The overall response rate was 78% (95% CI: 62–94%) ([Table 2](#tbl2)). The response rate in the 23 patients treated by gefitinib after chemotherapy was 74% (95% CI: 56–92%). When patients were stratified by EGFR mutation types, response rates were 75% (15 out of 20 patients) for exon 19 deletions, and 86% (six out of seven patients) for L858R. There were no significant differences in the response rates between the mutation types (*χ*^2^ test: *P*=0.557).

### Table 2. Efficacy of gefitinib in patients with EGFR mutation.

|   | CR | PR | s.d. | PD | Response |
| --- | --- | --- | --- | --- | --- |
| Prior chemotherapy (+) | 1 | 16 | 5 | 1 | 17 pts/23 pts (74%) |
|   |   |   |   |   | (95% CI: 56–92%) |
| Prior chemotherapy (−) | 0 | 4 | 0 | 0 | 4 pts/4 pts (100%) |
|   |   |   |   |   |   |
| Exon 19 deletions | 1 | 14 | 4 | 1 | 15 pts/20 pts (75%) |
| L858R | 0 | 6 | 1 | 0 | 6 pts/7 pts (86%) |
|   |   |   |   |   |   |
| Total | 1 | 20 | 5 | 1 | 21 pts/27 pts (78%) |
|   |   |   |   |   | (95% CI: 62–94%) |

Table 2 Caption: CI=confidence interval; CR=complete response; EGFR=epidermal growth factor receptor; PD=progressive disease; PR=partial response; Pts=patients; s.d.= standard deviation.

For the 27 patients, median time to progression (TTP) from the gefitinib treatment was 9.4 months. And median survival time (MST) from the gefitinib treatment was 15.4 months ([Figure 2](#fig2)). There were also no significant differences in survival time after the gefitinib treatment between the patients with exon 19 deletions and L858R (Kaplan–Meier, logrank: *P*<0.455). The 21 patients with CR/PR had a longer TTP and overall survival (14.4+ and 19.1+ months, respectively) than patients with stable disease/progression (3.1 and 5.6 months, respectively).

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/52e0/2360739/009055467a1e/95-6603466f2.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=2360739_95-6603466f2.jpg)

Survival time curves after gefitinib treatment in patients with and without EGFR mutation are shown.

All 27 eligible patients were assessable for toxicity ([Table 3](#tbl3)). Grade 3 drug-related hepatic toxicity was observed in one patient (4%), and Grade 3 skin toxicity occurred in one patient (4%). Other gastrointestinal toxicities were mild and acceptable. No lung toxicities were observed.

### Table 3. Side effects of gefitinib in patients with EGFR mutation.

|   | No. of patients with CTC grade (n=27) |  |  |
| --- | --- | --- | --- |
|   | 2 | 3 | 4 |
| Haematologic toxicity |   |   |   |
| Neutropenia | 1 | 0 | 0 |
| Thrombocytopenia | 0 | 0 | 0 |
| Anaemia | 0 | 0 | 0 |
|   |   |   |   |
| Other toxicities |   |   |   |
| Diarrhoea | 5 | 0 | 0 |
| Nausea and vomiting | 2 | 0 | 0 |
| Acne/acneform | 9 | 1 | — |
| Abnormal liver function (AST, ALT) | 1 | 1 | 0 |
| Abnormal renal function | 0 | 0 | 0 |
| Acute lung injury | 0 | 0 | 0 |

Table 3 Caption: ALT=alanine aminotransferase; AST=aspartate aminotransferase; CTC=common toxicity criteria; EGFR=epidermal growth factor receptor.

### Clinical usefulness of PNA-LNA PCR clamp

To investigate the clinical usefulness of PNA-LNA PCR clamp screening, patients with EGFR mutations detected by the test were compared to those with wild EGFR. The response rates were significantly different between patients with EGFR mutations (78%) and patients with wild EGFR (14%) (*χ*^2^ test, *P*=0.0017, [Figure 2](#fig2)). Median survival time after the gefitinib treatment was significantly different between patients with EGFR mutations (15.4 months) and those with wild-type EGFR (11.1 months) (Kaplan–Meier, logrank: *P*=0.0135, [Figure 2](#fig2)).

Furthermore, to clarify whether EGFR mutation status tested by PNA-LNA PCR clamp could be a prognostic factor for NSCLC patients, the relationship between EGFR mutation status and overall survival were evaluated using 99 patients except for one patient who was lost in follow-up. [Figure 3](#fig3) shows the comparison of overall survival after the initial treatments between NSCLC patients with EGFR mutations and those with wild-type EGFR by the Kaplan–Meier method. Overall survival after the initial treatment was significantly different between the groups (EGFR mutations: 19.1 months and wild-type EGFR: 10.7 months, logrank: *P*<0.0108). The Cox proportional hazards model (multiple variate) was also applied using EGFR mutations, sex, stage and PS. The latter three factors are well known as prognostic factors in NSCLC patients ([Brundage et al, 2002](#bib3)). The Cox proportional hazards model indicated that detecting EGFR mutations was a secondary prognostic factor ([Table 4](#tbl4)).

### Figure 3.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/52e0/2360739/e339ec5a356d/95-6603466f3.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=2360739_95-6603466f3.jpg)

Overall survival curves after the initial treatments in patients with and without EGFR mutation are shown.

### Table 4. Cox proportional hazards analysis.

|   | Hazards ratio | P-value |
| --- | --- | --- |
| Mutation | 2.259 | 0.036 |
| Performance status | 1.542 | 0.002 |
| Male/female | 1.053 | 0.887 |
| Stage | 1.029 | 0.319 |

Table 4 Caption: A Cox proportional hazards model (multiple variate) using EGFR mutations, sex, stage and PS was employed using the data from all the patients (n=99*) enrolled by the primary entry criteria. *Data missing: one patient.

## DISCUSSION

With PNA-LNA PCR clamp, we were able to determine EGFR mutation status in a majority of the NSCLC patients using clinical samples such as sputum and BF cytology. To determine EGFR mutations, direct sequencing or PCR-single strand conformational polymorphism methods are frequently employed ([Lynch et al, 2004](#bib8); [Paez et al, 2004](#bib11); [Mitsudomi et al, 2005](#bib9); [Pao and Miller, 2005](#bib12); [Tokumo et al, 2005](#bib18); [Twombly, 2005](#bib19)). However, these methods are time-consuming and require specimens that consist mostly of cancer cells. Another approach that analysis of an increased *EGFR* gene copy number, based on fluorescence *in situ* hybridisation analysis, could be used as a predictive marker for sensitivity to gefitinib ([Bell et al, 2005](#bib1); [Hirsch and Witta, 2005](#bib6); [Takano et al, 2005](#bib15)). However, this method also needs specimens consisting mostly of cancer cells, significant operation time and skilful technicians who have intertechnician variability. Thus, these methods can be employed only at some academic medical centres. The preferred and practical method is one that can sensitively, specifically and quickly detect EGFR mutations from specimens used for the diagnosis of lung cancers without removing contaminating normal cells. Peptide nucleic acid-locked nucleic acid PCR clamp can rapidly (within 2 hours) detect EGFR mutations from all specimens used to diagnose lung cancers, that is, sputum, pleural effusion and bronchial washing which contain many normal cells. This method is able to sensitively and specifically detect all 11 types of EGFR mutations ([Nagai et al, 2005](#bib10)) in the presence of 100-fold wild-type EGFR background levels. These 11 mutations account for more than 95% of EGFR mutations found ([Lynch et al, 2004](#bib8); [Twombly, 2005](#bib19)).

PNA-LNA PCR clamp prospectively detected EGFR mutations in 38% (95% CI: 28–48%) of the consecutive patients with NSCLC. Patients who were EGFR mutation-positive were mostly women (61%) and had adenocarcinomas (87%), and significantly lower smoking index (34%). These results were consistent with the results of previous retrospective reports ([Mitsudomi et al, 2005](#bib9); [Tokumo et al, 2005](#bib18)). Some clinical studies are trying to select patients to gefitinib treatment by clinicopathologic features of adenocarcinoma and non-smoker without testing EGFR mutations. Our data indicate such an approach is not feasible. For example, when selecting patients with adenocarcinoma and smoking >20 pack-years, 15 of the 38 patients with EGFR mutations (39%) would be missed, whereas 13 of the 62 patients without EGFR mutations (21%) would be mistakenly included.

Furthermore, the presence of EGFR mutations detected by the PNA-LNA PCR clamp was found to be a prognostic factor in Japanese patients with NSCLC in this prospective screening. A Cox proportional hazards model indicated that detecting EGFR mutations was a significant prognostic factor and was superior to sex or stage, indicating that incorporating the PNA-LNA PCR clamp into clinical studies and clinical practice is critical.

This phase II study clearly showed the favourable response to gefinitib in NSCLC patients with EGFR mutations. The response rate was 78% and the lower limit of the 95% confidential interval of response was 62%. In contrast to previous retrospective analyses ([Riely et al, 2006](#bib13); [Hirsch et al, 2006](#bib5)), patients with exon 19 deletions were equally responsive compared to those with L858R in this study. This might be due to our small sample size, so these data need to be confirmed in a larger trial. In EGFR mutation-positive patients treated by gefitinib, TTP (9.4 months) of after the gefitinib treatment and MST (19.1 months) after the initial treatment were longer than in patients treated with the regimens using platinum doublet. Detection of EGFR mutations clearly differentiates gefinitib-sensitive patients from gefinitib-insensitive patients with regard to response rate and survival times.

Four patients with EGFR mutations received gefitinib as the first line treatment because they could not be given chemotherapy owing to poor PS. Two patients had meningitis carcinomatosa. One had multiple brain metastases. And one had repeated aspiration pneumonia owing to recurrent nerve palsy. All of these patients showed PR and obtained better PS. Their survival times were 190, 183+, 278+ and 296+ days, respectively, and all returned home. This experience taught us the usefulness of testing for EGFR mutations in patients with poor PS owing to advanced disease. Thus, even in Europe and the US where frequencies of EGFR mutations are low, incorporating testing for EGFR mutations in clinical practice may provide a huge benefit to some patients.

In conclusion, our study prospectively demonstrated the clinical benefit of gefitinib given to NSCLC patients with good PS as the second-line treatment harbouring EGFR mutations, and, also, gefitinib given to NSCLC patients with poor PS as the first-line treatment showed a favourable response. To attain this benefit, screening clinical samples at the time of diagnosis is imperative, and PNA-LNA PCR clamp is a good method to achieve this aim.

## Acknowledgments

This study was partly supported by grants from the Ministry of Education, Culture, Sports, Science and Technology of Japan. This work was presented at Poster Discussion in ASCO 2006.

## References

1. Bell DW, Lynch TJ, Haserlat SM, Harris PL, Okimoto RA, Brannigan BW, Sgroi DC, Muir B, Riemenschneider MJ, Iacona RB, Krebs AD, Johnson DH, Giaccone G, Herbst RS, Manegold C, Fukuoka M, Kris MJ, Baselga J, Ochs JS, Haber DA (2005) Epidermal growth factor receptor mutations and gene amplification in non-small-cell lung cancer: molecular analysis of the IDEAL/INTACT Gefitinib Trials. J Clin Oncol 23: 8081–8092  [DOI](https://doi.org/10.1200/JCO.2005.02.7078) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16204011/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Epidermal%20growth%20factor%20receptor%20mutations%20and%20gene%20amplification%20in%20non-small-cell%20lung%20cancer:%20molecular%20analysis%20of%20the%20IDEAL/INTACT%20Gefitinib%20Trials&volume=23&publication_year=2005&pages=8081&pmid=16204011&doi=10.1200/JCO.2005.02.7078&)

2. Brehmer D, Greff Z, Godl K, Blencke S, Kurtenbach A, Weber M, Müller S, Klebl S, Cotton M, Kéri G, Wissing J, Daub H (2005) Cellular targets of gefitinib. Cancer Res 65: 379–382  [PubMed](https://pubmed.ncbi.nlm.nih.gov/15695376/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=Cellular%20targets%20of%20gefitinib&volume=65&publication_year=2005&pages=379&pmid=15695376&)

3. Brundage MD, Davies D, Mackillop WJ (2002) Prognostic factors in non-small cell lung cancer: a decade of progress. Chest 122: 1037–1057  [DOI](https://doi.org/10.1378/chest.122.3.1037) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12226051/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chest&title=Prognostic%20factors%20in%20non-small%20cell%20lung%20cancer:%20a%20decade%20of%20progress&volume=122&publication_year=2002&pages=1037&pmid=12226051&doi=10.1378/chest.122.3.1037&)

4. Fukuoka M, Yano S, Giaccone G, Tamura T, Nakagawa K, Douillard JY, Nishiwaki Y, Vansteenkiste J, Kudoh S, Rischin D, Eek R, Horai T, Noda K, Takata I, Smit E, Averbuch S, Macleod A, Feyereislova A, Dong RP, Baselga J (2003) Multi-institutional randomized phase II trial of gefitinib for previously treated patients with advanced non-small-cell lung cancer (The IDEAL 1 Trial). J Clin Oncol 21: 2237–2246  [DOI](https://doi.org/10.1200/JCO.2003.10.038) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12748244/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Multi-institutional%20randomized%20phase%20II%20trial%20of%20gefitinib%20for%20previously%20treated%20patients%20with%20advanced%20non-small-cell%20lung%20cancer%20(The%20IDEAL%201%20Trial)&volume=21&publication_year=2003&pages=2237&pmid=12748244&doi=10.1200/JCO.2003.10.038&)

5. Hirsch FR, Franklin WA, McCoy J, Cappuzzo F, Varella-Garcia M, Witta SE, Gumerlock P, West H, Gandara DR, Bunn Jr PA (2006) Predicting clinical benefit from EGFR TKIs: not all EGFR mutations are equal. Proc Am Soc Clin Oncol 24(18S): 382s  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Am%20Soc%20Clin%20Oncol&title=Predicting%20clinical%20benefit%20from%20EGFR%20TKIs:%20not%20all%20EGFR%20mutations%20are%20equal&volume=24&publication_year=2006&pages=382s&)

6. Hirsch FR, Witta S (2005) Biomarkers for prediction of sensitivity to EGFR inhibitors in non-small cell lung cancer. Curr Opin Oncol 17: 118–122  [DOI](https://doi.org/10.1097/01.cco.0000155059.39733.9d) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15725915/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Opin%20Oncol&title=Biomarkers%20for%20prediction%20of%20sensitivity%20to%20EGFR%20inhibitors%20in%20non-small%20cell%20lung%20cancer&volume=17&publication_year=2005&pages=118&pmid=15725915&doi=10.1097/01.cco.0000155059.39733.9d&)

7. Kris MG, Natale RB, Herbst RS, Lynch TJ, Prager D, Belani CP, Schiller JH, Kelly K, Spiridonidis H, Sandler A, Albain KS, Cella D, Wolf MK, Averbuch SD, Ochs JJ, Kay AC (2003) Efficacy of gefitinib, an inhibitor of the epidermal growth factor receptor tyrosine kinase, in symptomatic patients with non-small cell lung cancer: a randomised trial. JAMA 290: 2149–2158  [DOI](https://doi.org/10.1001/jama.290.16.2149) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14570950/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA&title=Efficacy%20of%20gefitinib,%20an%20inhibitor%20of%20the%20epidermal%20growth%20factor%20receptor%20tyrosine%20kinase,%20in%20symptomatic%20patients%20with%20non-small%20cell%20lung%20cancer:%20a%20randomised%20trial&volume=290&publication_year=2003&pages=2149&pmid=14570950&doi=10.1001/jama.290.16.2149&)

8. Lynch TJ, Bell DW, Sordella R, Gurubhagavatula S, Okimoto RA, Brannigan BW, Harris PL, Haserlat SM, Supko JG, Haluska FG, Louis DN, Christiani DC, Settleman J, Haber DA (2004) Activating mutations in the epidermal growth factor receptor underlying responsiveness of non-small-cell lung cancer to gefitinib. New Engl J Med 350: 2129–2139  [DOI](https://doi.org/10.1056/NEJMoa040938) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15118073/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=New%20Engl%20J%20Med&title=Activating%20mutations%20in%20the%20epidermal%20growth%20factor%20receptor%20underlying%20responsiveness%20of%20non-small-cell%20lung%20cancer%20to%20gefitinib&volume=350&publication_year=2004&pages=2129&pmid=15118073&doi=10.1056/NEJMoa040938&)

9. Mitsudomi T, Kosaka T, Endoh H, Zorio Y, Hida T, Mori S, Hatooka S, Shinoda M, Takahashi T, Yatabe Y (2005) Mutations of the epidermal growth factor receptor gene predict prolonged survival after gefitinib treatment in patients with non-small-cell lung cancer with postoperative recurrence. J Clin Oncol 23: 2513–2520  [DOI](https://doi.org/10.1200/JCO.2005.00.992) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15738541/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Mutations%20of%20the%20epidermal%20growth%20factor%20receptor%20gene%20predict%20prolonged%20survival%20after%20gefitinib%20treatment%20in%20patients%20with%20non-small-cell%20lung%20cancer%20with%20postoperative%20recurrence&volume=23&publication_year=2005&pages=2513&pmid=15738541&doi=10.1200/JCO.2005.00.992&)

10. Nagai Y, Miyazawa H, Huqun, Tanaka T, Udagawa K, Kato M, Fukuyama S, Yokote A, Kobayashi K, Kanazawa M, Hagiwara K (2005) Genetic heterogeneity of the epidermal growth factor receptor in non-small cell lung cancer cell lines revealed by a rapid and sensitive detection system, the peptide nucleic acid-locked nucleic acid PCR clamp. Cancer Res 65: 7276–7282  [DOI](https://doi.org/10.1158/0008-5472.CAN-05-0331) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16105816/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=Genetic%20heterogeneity%20of%20the%20epidermal%20growth%20factor%20receptor%20in%20non-small%20cell%20lung%20cancer%20cell%20lines%20revealed%20by%20a%20rapid%20and%20sensitive%20detection%20system,%20the%20peptide%20nucleic%20acid-locked%20nucleic%20acid%20PCR%20clamp&volume=65&publication_year=2005&pages=7276&pmid=16105816&doi=10.1158/0008-5472.CAN-05-0331&)

11. Paez JG, Jänne PA, Lee JC, Tracy S, Greulich H, Gabriel S, Herman P, Kaye FJ, Lindeman N, Boggon TJ, Naoki K, Sasaki H, Fujii Y, Eck MJ, Sellers WR, Johnson BE, Meyerson M (2004) EGFR mutations in lung cancer: correlation with clinical response to gefitinib therapy. Science 304: 1497–1500  [DOI](https://doi.org/10.1126/science.1099314) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15118125/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=EGFR%20mutations%20in%20lung%20cancer:%20correlation%20with%20clinical%20response%20to%20gefitinib%20therapy&volume=304&publication_year=2004&pages=1497&pmid=15118125&doi=10.1126/science.1099314&)

12. Pao W, Miller VA (2005) Epidermal growth factor receptor mutations, small-molecule kinase inhibitors, and non-small-cell lung cancer: current knowledge and future directions. J Clin Oncol 23: 2556–2568  [DOI](https://doi.org/10.1200/JCO.2005.07.799) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15767641/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Epidermal%20growth%20factor%20receptor%20mutations,%20small-molecule%20kinase%20inhibitors,%20and%20non-small-cell%20lung%20cancer:%20current%20knowledge%20and%20future%20directions&volume=23&publication_year=2005&pages=2556&pmid=15767641&doi=10.1200/JCO.2005.07.799&)

13. Riely GJ, Pao W, Pham D, Li AR, Rizvi N, Venkatraman ES, Zakowski MF, Kris MG, Ladanyi M, Miller VA (2006) Clinical course of patients with non-small cell lung cancer and epidermal growth factor receptor exon 19 and exon 21 mutations treated with gefitinib or erlotinib. Clin Cancer Res 12: 839–844  [DOI](https://doi.org/10.1158/1078-0432.CCR-05-1846) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16467097/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=Clinical%20course%20of%20patients%20with%20non-small%20cell%20lung%20cancer%20and%20epidermal%20growth%20factor%20receptor%20exon%2019%20and%20exon%2021%20mutations%20treated%20with%20gefitinib%20or%20erlotinib&volume=12&publication_year=2006&pages=839&pmid=16467097&doi=10.1158/1078-0432.CCR-05-1846&)

14. Siegel-Lakhai WS, Beijnen JH, Schellens JHM (2005) Current knowledge and future directions of the selective epidermal growth factor receptor inhibitors erlotinib (Tarceva®) and gefitinib (Iressa®). Oncologist 10: 579–589  [DOI](https://doi.org/10.1634/theoncologist.10-8-579) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16177282/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncologist&title=Current%20knowledge%20and%20future%20directions%20of%20the%20selective%20epidermal%20growth%20factor%20receptor%20inhibitors%20erlotinib%20(Tarceva%C2%AE)%20and%20gefitinib%20(Iressa%C2%AE)&volume=10&publication_year=2005&pages=579&pmid=16177282&doi=10.1634/theoncologist.10-8-579&)

15. Takano T, Ohe Y, Sakamoto H, Tsuta K, Matsuno Y, Tateishi U, Yamamoto S, Nokihara H, Yamamoto N, Sekine I, Kunitoh H, Shibata T, Sakiyama T, Yoshida T, Tamura T (2005) Epidermal growth factor receptor gene mutations and increased copy numbers predict gefitinib sensitivity in patients with recurrent non-small-cell lung cancer. J Clin Oncol 23: 6829–6837  [DOI](https://doi.org/10.1200/JCO.2005.01.0793) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15998907/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Epidermal%20growth%20factor%20receptor%20gene%20mutations%20and%20increased%20copy%20numbers%20predict%20gefitinib%20sensitivity%20in%20patients%20with%20recurrent%20non-small-cell%20lung%20cancer&volume=23&publication_year=2005&pages=6829&pmid=15998907&doi=10.1200/JCO.2005.01.0793&)

16. Taron M, Ichinose Y, Rosell R, Mok T, Massuti B, Zamora L, Mate JL, Manegold C, Ono M, Queralt C, Jahan T, Sanchez JJ, Sanchez-Ronco M, Hsue V, Jablons D, Sanchez JM, Moran T (2005) Activating mutations in the tyrosine kinase domain of the epidermal growth factor receptor are associated with improved survival in gefitinib-treated chemorefractory lung adenocarcinomas. Clin Cancer Res 11: 5878–5885  [DOI](https://doi.org/10.1158/1078-0432.CCR-04-2618) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16115929/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=Activating%20mutations%20in%20the%20tyrosine%20kinase%20domain%20of%20the%20epidermal%20growth%20factor%20receptor%20are%20associated%20with%20improved%20survival%20in%20gefitinib-treated%20chemorefractory%20lung%20adenocarcinomas&volume=11&publication_year=2005&pages=5878&pmid=16115929&doi=10.1158/1078-0432.CCR-04-2618&)

17. Thatcher N, Chang A, Parikh P, Pereira JR, Ciuleanu T, Pawel J, Thongprasert S, Tan EH, Pemberton K, Archer V, Carroll K (2005) Gefitinib plus best supportive care in previously treated patients with refractory advanced non-small-cell lung cancer: results from a randomised, placebo-controlled, multicentre study (Iressa Survival Evaluation in Lung Cancer). Lancet 366: 1527–1537  [DOI](https://doi.org/10.1016/S0140-6736(05)67625-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16257339/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Gefitinib%20plus%20best%20supportive%20care%20in%20previously%20treated%20patients%20with%20refractory%20advanced%20non-small-cell%20lung%20cancer:%20results%20from%20a%20randomised,%20placebo-controlled,%20multicentre%20study%20(Iressa%20Survival%20Evaluation%20in%20Lung%20Cancer)&volume=366&publication_year=2005&pages=1527&pmid=16257339&doi=10.1016/S0140-6736(05)67625-8&)

18. Tokumo M, Toyooka S, Kiura K, Shigematsu H, Tomii K, Aoe M, Ichimura K, Tsuda T, Yano M, Tsukuda K, Tabata M, Ueoka H, Tanimoto M, Date H, Gazdar AF, Shimizu N (2005) The relationship between epidermal growth factor receptor mutations and clinicopathologic features in non-small cell lung cancers. Clin Cancer Res 11: 1167–1173  [PubMed](https://pubmed.ncbi.nlm.nih.gov/15709185/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=The%20relationship%20between%20epidermal%20growth%20factor%20receptor%20mutations%20and%20clinicopathologic%20features%20in%20non-small%20cell%20lung%20cancers&volume=11&publication_year=2005&pages=1167&pmid=15709185&)

19. Twombly R (2005) Failing survival advantage in crucial trial, future of Iressa in jeopardy. J Natl Cancer Inst 97: 249–250  [DOI](https://doi.org/10.1093/jnci/97.4.249) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15713955/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Natl%20Cancer%20Inst&title=Failing%20survival%20advantage%20in%20crucial%20trial,%20future%20of%20Iressa%20in%20jeopardy&volume=97&publication_year=2005&pages=249&pmid=15713955&doi=10.1093/jnci/97.4.249&)

20. Ugozzoli LA, Latorra D, Pucket R, Arar K, Hamby K (2004) Real-time genotyping with oligonucleotide probes containing locked nucleic acids. Anal Biochem 324: 143–152  [DOI](https://doi.org/10.1016/j.ab.2003.09.003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14654057/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anal%20Biochem&title=Real-time%20genotyping%20with%20oligonucleotide%20probes%20containing%20locked%20nucleic%20acids&volume=324&publication_year=2004&pages=143&pmid=14654057&doi=10.1016/j.ab.2003.09.003&)
