# Gemcitabine Metabolic and Transporter Gene Polymorphisms Are Associated with Drug Toxicity and Efficacy in Patients with Locally Advanced Pancreatic Cancer

## Metadata
**Authors:** Motofumi Tanaka, Milind Javle, Xiaoqun Dong, Cathy Eng, James L Abbruzzese, Donghui Li
**Journal:** Cancer
**Date:** 2010 Nov 15
**DOI:** [10.1002/cncr.25282](https://doi.org/10.1002/cncr.25282)
**PMID:** 20665488
**PMCID:** PMC2966859
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2966859/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC2966859/pdf/nihms190212.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC2966859/pdf/nihms190212.pdf)

## Abstract

**Background:** 
It has not been well established whether genetic variations can be biomarkers for clinical outcome of gemcitabine therapy. The purpose of this study was to identify single nucleotide polymorphisms (SNPs) of gemcitabine metabolic and transporter genes that are associated with toxicity and efficacy of gemcitabine-based therapy in patients with locally advanced pancreatic cancer (LAPC).

**Methods:** 
We evaluated 17 SNPs of the CDA, dCK, DCTD, RRM1, hCNT1-3, and hENT1 genes in 149 patients with LAPC who underwent gemcitabine-based chemoradiotherapy. The association of genotypes with neutropenia, tumor response to therapy, overall survival (OS), and progression-free survival (PFS) was analyzed by logistic regression, log-rank test, Kaplan-Meier plot, and Cox proportional hazards regression.

**Results:** 
The CDA A-76C, dCK C-1205T, RRM1 A33G, and hENT1 C913T genotypes were significantly associated with grade 3-4 neutropenia (P = .020, .015, .003, and .017, respectively). The CDA A-76C and hENT1 A-201G genotypes were significantly associated with tumor response to therapy (P = .017 and P = .019). A combined genotype effect of CDA A-76C, RRM1 A33G, RRM1 C-27A, and hENT1 A-201G on PFS was observed. Patients carrying 0–1 (n = 64), 2 (n = 50), or 3–4 (n = 17) at-risk genotypes had median PFS times of 8.3, 6.0, and 4.2 months, respectively (P = .002).

**Conclusions:** 
Our results indicate that some polymorphic variations of drug metabolic and transporter genes may be potential biomarkers for clinical outcome of gemcitabine-based therapy in patients with LAPC.

Keywords: gemcitabine metabolism, nucleoside transporter, single nucleotide polymorphism, locally advanced pancreatic cancer

### Background

It has not been well established whether genetic variations can be biomarkers for clinical outcome of gemcitabine therapy. The purpose of this study was to identify single nucleotide polymorphisms (SNPs) of gemcitabine metabolic and transporter genes that are associated with toxicity and efficacy of gemcitabine-based therapy in patients with locally advanced pancreatic cancer (LAPC).

### Methods

We evaluated 17 SNPs of the *CDA, dCK, DCTD, RRM1, hCNT1-3*, and *hENT1* genes in 149 patients with LAPC who underwent gemcitabine-based chemoradiotherapy. The association of genotypes with neutropenia, tumor response to therapy, overall survival (OS), and progression-free survival (PFS) was analyzed by logistic regression, log-rank test, Kaplan-Meier plot, and Cox proportional hazards regression.

### Results

The *CDA* A-76C, *dCK* C-1205T, *RRM1* A33G, and *hENT1* C913T genotypes were significantly associated with grade 3-4 neutropenia (*P* = .020, .015, .003, and .017, respectively). The *CDA* A-76C and *hENT1* A-201G genotypes were significantly associated with tumor response to therapy (*P* = .017 and *P* = .019). A combined genotype effect of *CDA* A-76C, *RRM1* A33G, *RRM1* C-27A, and *hENT1* A-201G on PFS was observed. Patients carrying 0–1 (n = 64), 2 (n = 50), or 3–4 (n = 17) at-risk genotypes had median PFS times of 8.3, 6.0, and 4.2 months, respectively (*P* = .002).

### Conclusions

Our results indicate that some polymorphic variations of drug metabolic and transporter genes may be potential biomarkers for clinical outcome of gemcitabine-based therapy in patients with LAPC.

## Introduction

Pancreatic cancer is the third most common gastrointestinal malignancy and the fourth leading cause of cancer deaths in the United States.[1](#R1) At diagnosis, only 20% of patients have a surgically resectable tumor, 30% have a locally advanced tumor, and 50% present with distant metastasis.[2](#R2) Over the past decade, gemcitabine (2′,2′-difluorodeoxycytidine, dFdC) has been the standard agent for first-line chemotherapy of advanced pancreatic cancer, producing limited clinical benefit and improved overall survival (OS) as compared with 5-fluorouracil (5-FU).[3](#R3) Recent studies have reported the efficacy of a combination therapy of gemcitabine plus radiation for unresectable locally advanced pancreatic cancer (LAPC).[4](#R4)^, ^[5](#R5) However, factors that can predict tumor response and survival have not been well elucidated.[6](#R6) In addition, although one major side effect caused by gemcitabine is hematological toxicity such as neutropenia, available biomarkers for severe toxicity have not yet been established.

Gemcitabine is a specific analogue of the native pyrimidine nucleotide deoxycytidine and a prodrug that requires cellular uptake and intracellular phosphorylation ([Fig. 1](#F1)).[7](#R7)^-^[9](#R9) Gemcitabine intracellular uptake is mediated mainly by human equilibrative nucleoside transporter (hENT1, aka solute carrier family 29 A1) and, to a lesser extent, by human concentrative nucleoside transporters (hCNT) 1 and hCNT3 (aka solute carrier family 28 A1 or A3).[9](#R9) Inside cells, gemcitabine is phosphorylated to its monophosphate form (dFdCMP) by deoxycytidine kinase (dCK) and this step is essential for further phosphorylation to its active triphosphate form (dFdCTP).[10](#R10) The active diphosphate metabolite of gemcitabine (dFdCDP) is also active and inhibits deoxyribonucleic acid (DNA) synthesis indirectly by inhibiting ribonucleotide reductase (RRM1).[8](#R8)^,^[11](#R11)^,^[12](#R12) Gemcitabine is inactivated primarily by deoxycytidine deaminase (CDA) into 2′,2′-difluorodeoxyuridine (dFdU), and gemcitabine monophosphate is inactivated by deoxycytidylate deaminase (DCTD) into dFdU monophosphate form (dFdUMP).[8](#R8)^,^[9](#R9)

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1fe2/2966859/8827f3dd8acb/nihms190212f1.jpg)

Schematic description of gemcitabine (dFdC) transportation and metabolism. The boxed letters indicate genes that are examined in this study.

Previous studies have demonstrated the relationship between gemcitabine metabolic or transport enzymes and clinical outcome. One study showed that low expression of CDA was associated with severe hematologic toxicity of gemcitabine.[13](#R13) Other studies in cell lines or tumor tissues have established the association between resistance to gemcitabine and decreased nucleoside transport into cells,[14](#R14)^-^[16](#R16) decreased expression of activation enzymes such as dCK,[17](#R17)^-^[20](#R20) increased expression of degradation enzymes such as CDA and DCTD,[21](#R21)^,^[22](#R22) as well as increased expression of RRM1.[23](#R23)^-^[26](#R26) In clinical studies of pancreatic cancer, high expression of hENT1 in tumors has been associated with improved survival in patients treated with gemcitabine.[15](#R15)^,^[16](#R16)^,^[23](#R23)^,^[27](#R27)

Single nucleotide polymorphisms (SNPs) of enzymes in gemcitabine's pharmacologic pathway have been previously identified.[8](#R8) The activity of these enzymes has been correlated with polymorphic gene variations by in vivo and in vitro studies.[9](#R9)^,^[28](#R28)^-^[30](#R30) However, only a few clinical studies have shown a positive association between the enzyme SNPs and gemcitabine toxicity.[31](#R31)^-^[33](#R33) We have previously shown that genetic variations in gemcitabine metabolism and transport are associated with drug toxicity and overall survival in patients with resectable pancreatic cancer[34](#R34). In the current study, we tried to validate the previous findings in 149 patients with LAPC who had undergone gemcitabine-based therapy.

## Materials and Methods

### Patient Recruitment and Data Collection

A single institution retrospective analysis was completed. We identified 149 patients with biopsy-confirmed LAPC at the time of diagnosis. LAPC was defined as unresectable tumors that extended to the celiac axis or the superior mesenteric artery or tumors that occluded the superior mesenteric venous (SMV)-portal venous confluence based on a review of the computed tomography (CT)[35](#R35). All patients were required to be treatment naïve and underwent gemcitabine-based chemotherapy as first-line therapy as a single agent or in combination at The University of Texas M. D. Anderson Cancer Center (Houston, Texas) from February 1999 to June 2007. The median dose of gemcitabine therapy was 750 mg/m^2^ (range, 450-1000 mg/m^2^). In 75 patients (50.3%), either cisplatin or oxaliplatin was administered with gemcitabine. In 125 patients (83.9%), gemcitabine therapy was followed by consolidative radiotherapy at a dose of 30 Gy. Patient observation continued through June 2009. Information on treatment provided, toxicity, tumor response to therapy, tumor progression, and survival time was collected by reviewing patients' medical records in an electronic database. This study was approved by the institutional review board of M. D. Anderson Cancer Center.

Neutropenia, the most common hematologic toxicity caused by gemcitabine, was graded according to the National Cancer Institute Common Terminology Criteria for Adverse Events (CTCAE), version 3.0. Tumor response to therapy was evaluated by comparing CT at the time of diagnosis with CT at 6-8 weeks after chemoradiotherapy or chemotherapy, and was defined according to the Response Evaluation Criteria in Solid Tumors (RECIST) as partial response (PR), stable disease (SD), or progressive disease (PD). OS and progression-free survival (PFS) were calculated from the date of diagnosis to the date of death and progression or last follow-up date, respectively. Twelve patients were excluded from PFS analysis because they were lost to follow-up on disease progression. Performance status was evaluated by Eastern Cooperative Oncology Group (ECOG) criteria.

### Extracting and Genotyping DNA

We selected 17 SNPs of the *CDA*, *dCK*, *RRM1*, *DCTD*, *hCNT1-3*, and *hENT1* genes according to the following criteria: 1) minor allele frequency of the SNP was greater than 10% among Caucasians, 2) coding SNPs including nonsynonymous or synonymous SNPs, and 3) SNPs that have been associated with cancer risk or clinical outcome in prior studies. [Table 1](#T1) summarizes the genes, nucleotide substitutions, function (such as encoding amino acid changes), reference SNP identification numbers, and minor allele frequencies of the 17 SNPs evaluated in this study.

### Table 1. SNPs evaluated.

|   |   |   |   | Minor allele frequency |  |
| --- | --- | --- | --- | --- | --- |
|   |   |   |   |   |  |
| Gene | SNP | Function | RS No. | observed* | reported† |
| CDA | Ex4 +111C>T | T145T | 1048977 | 0.34 | 0.28 |
|   | Ex2 -76A>C | K27Q | 2072671 | 0.36 | 0.44 |
| dCK | IVS6 -1205C>T | Intron | 4694362 | 0.42 | 0.45 |
|   | IVS2 +9846A>G | Intron | 12648166 | 0.43 | 0.43 |
| RRM1 | Ex19 +42G>A | A744A | 1042858 | 0.27 | 0.11 |
|   | Ex19 +33A>G | T741T | 3177016 | 0.47 | 0.47 |
|   | Ex9 -27C>A | R284R | 183484 | 0.50 | 0.48 |
| DCTD | Ex4 -47T>C | V116V | 7663494 | 0.28 | 0.33 |
| hCNT1 | Ex15 -16A>G | Q456Q | 2242048 | 0.16 | 0.15 |
|   | Ex9 -9C>A | Q237K | 8187758 | 0.33 | 0.19 |
| hCNT2 | Ex4 -38C>A | S75R | 1060896 | 0.47 | 0.33 |
|   | Ex2 -17C>T | P22L | 11854484 | 0.46 | 0.34 |
| hCNT3 | Ex14 -69C>T | L461L | 7853758 | 0.27 | 0.15 |
|   | Ex5 +25A>G | T89T | 7867504 | 0.40 | 0.39 |
| hENT1 | IVS12 -201A>G | Intron | 760370 | 0.33 | 0.35 |
|   | IVS2 -549T>C | Intron | 324148 | 0.30 | 0.30 |
|   | IVS2 +913C>T | Intron | 9394992 | 0.45 | 0.32 |

Table 1 Caption: Abbreviations: SNP, single-nucleotide polymorphism; RS No., reference SNP identification number. *The data observed in current study. †The reported minor allele frequency (Caucasian) was from SNP500 cancer database.

Peripheral blood lymphocytes before chemotherapy were obtained from 149 LAPC patients with informed consent, and DNA was extracted using Qiagen DNA isolation kits (Valencia, CA). Taqman 5′ nuclease assay was performed to determine all genetic variants. Primers and TaqMan MGB probes were provided by TaqMan SNP Genotyping Assay Services (Applied Biosystems, Foster City, CA, USA). The probes were labeled with the fluorescent dye VIC or FAM for each allele at the 5′ end. Polymerase chain reaction (PCR) was performed in a 5-μl total volume consisted of TaqMan Universal PCR Master Mix, 20 ng of genomic DNA (diluted with dH_2_O), and TaqMan SNP Genotyping Assay Mix. Allele discrimination was accomplished by running end point detection using ABI Prism 7900HT Sequence Detection System, and SDS 2.3 software (Applied Biosystems).

### Statistical Methods

The genotype distribution was tested for Hardy-Weinberg equilibrium using the goodness-of-fit χ^2^ test. The genotype association with grade 3-4 neutropenia toxicity and tumor response to therapy was analyzed by logistic regression. Gemcitabine dose intensity by genotype was compared using *t* test. OS and PFS were analyzed by log-rank test, Kaplan-Meier plot, and Cox proportional hazards regression model. The heterozygous and homozygous genotypes were combined in these analyses if the frequency of the homozygous mutant was low or if the homozygous and heterozygous genotypes had the same direction of effect on toxicity, tumor response, or survival. Multivariate analyses were performed with adjustment for clinical predictors that were statistically significant. All statistical testing was conducted with SPSS software, version 17.0 (SPSS Inc, Chicago, IL), and statistical significance and borderline significance were defined as *P* < .05 and *P* < .20, respectively.

We estimated the false-positive report probability (FPRP) for the observed statistically significant associations using the methods described by Wacholder et al.[36](#R36) FPRP is the probability of no true association between a genetic variant and a phenotype given a statistically significant finding. FPRP is determined not only by the observed P value but also by both the prior probability that the association between the genetic variant and the phenotype is real and the statistical power of the test. In the current study, odds ratio (OR) and hazard ratio (HR) values of 2.0 to 4.0 were considered as a likely threshold value. The prior probability employed was 0.25 for all SNPs. The FPRP value for noteworthiness was set at 0.2.

## Results

### Patients' Characteristics and Clinical Predictors

[Table 2](#T2) shows the patients' characteristics, clinical features of their tumors, and treatment. The median age of the 149 patients was 62 years (range, 38–86 years). Non-Hispanic whites comprised 92% of the patients. After a median follow-up of 16.8 months (range, 2-60 months), the median survival time (MST) of all patients was 15.2 ± 0.8 months [95% confidence interval (CI), 13.6–16.9]. Tumor response to therapy was significantly associated with OS (*P* < .001). ECOG performance status and presence of diabetes as a comorbidity had a borderline significant association with OS (*P* = .143 and *P* = .081) in log-rank test. Although 24 patients (16.1%) had not undergone radiotherapy, that factor was not associated with OS (*P* = .503). Concurrent therapy with a platinum drug also did not impact OS (*P* = .745).

### Table 2. Patient characteristics and overall survival (n = 149).

| Variable | No. of patients | No. of deaths (%) | MST (months) | Log-rank P |
| --- | --- | --- | --- | --- |
| Age (years) |   |   |   | .740 |
| ≤50 | 24 | 21 (87.5%) | 15.5 |   |
| 51-60 | 38 | 33 (86.8%) | 17.2 |   |
| 61-70 | 49 | 42 (85.7%) | 13.8 |   |
| >70 | 38 | 33 (86.8%) | 15.2 |   |
| Sex |   |   |   | .416 |
| Male | 90 | 78 (86.6%) | 14.2 |   |
| Female | 59 | 51 (86.4%) | 15.7 |   |
| Race |   |   |   | .241 |
| White | 136 | 118 (86.7%) | 14.5 |   |
| Hispanic | 9 | 8 (88.8%) | 19.8 |   |
| African American | 2 | 1 (50.0%) | 18.4 |   |
| Other | 2 | 2 (100%) | 8.8 |   |
| Performance status |   |   |   | .143 |
| 0 | 30 | 25 (83.3%) | 17.4 |   |
| 1 | 98 | 86 (87.8%) | 14.2 |   |
| 2 | 12 | 11 (91.6%) | 10.9 |   |
| Diabetes status |   |   |   | .081 |
| Negative | 112 | 95 (84.8%) | 15.5 |   |
| Positive | 37 | 34 (91.9%) | 15.2 |   |
| Tumor site |   |   |   | .644 |
| Head/neck | 117 | 103 (88.0%) | 14.8 |   |
| Body/tail | 32 | 26 (81.3%) | 17.2 |   |
| Tumor size (cm) |   |   |   | .398 |
| ≤5 | 114 | 99 (86.8%) | 15.3 |   |
| >5 | 31 | 27 (87.1%) | 13.6 |   |
| CA19-9 (units/ml) |   |   |   | .977 |
| ≤47 | 28 | 22 (78.6%) | 15.6 |   |
| 48-500 | 72 | 64 (88.9%) | 15.2 |   |
| 501-1,000 | 18 | 15 (83.3%) | 11.5 |   |
| >1,000 | 30 | 28 (93.3%) | 16.0 |   |
| Tumor response |   |   |   | <.001 |
| PR/SD | 86 | 70 (81.4%) | 19.3 |   |
| PD | 57 | 53 (93.0%) | 9.9 |   |
| Platinum drug use |   |   |   | .754 |
| yes | 75 | 63 (84.0%) | 16.2 |   |
| no | 74 | 66 (89.2%) | 13.6 |   |
| Radiotherapy |   |   |   | .503 |
| yes | 125 | 110 (88.0%) | 15.2 |   |
| no | 24 | 19 (79.2%) | 14.8 |   |

Table 2 Caption: Abbreviations: MST, median survival time; PR, partial response; SD, stable disease; PD, progressive disease.

### Genotype Frequencies

We successfully amplified the 17 genotypes in 97.3% to 100% of the samples. Approximately 10% of total samples were analyzed in duplicate, and no discrepancies were seen. Genotype frequencies of the 17 SNPs were found to be in Hardy-Weinberg equilibrium (χ^2^ = 0.001–2.097, *Ps* =.148–.973). No significant racial differences in genotype frequency were observed (data not shown). The two SNPs (IVS12 -201A>G and IVS2 -549T>C) of the *hENT1* gene were in linkage disequilibrium (|*D*′ |=0.774, *P* < 0.01).

### Association of Genotypes with Toxicity

None of the clinical factors including concurrent treatment with platinum drug (*P* = .457) or radiotherapy (*P* = .126) was associated with neutropenia, the most common hematologic toxicity caused by gemcitabine. The *CDA* A-76C, *dCK* C-1205T, *RRM1* A33G, and *hENT1* C913T genotypes, individually and jointly, were significantly associated with severe (grade 3-4) neutropenia ([Table 3](#T3)). For example, 39 (43.8%) of the *CDA* -76 AC/CC carriers compared with only 15 (25.0%) of the AA carriers had severe neutropenia *(P* = .020). Patients carrying 2 or 3–4 at-risk alleles had a significantly higher frequency of severe neutropenia than did patients carrying only 0-1 at-risk alleles (OR = 3.24, 95% CI = 1.19–8.82, *P* = .021; and OR = 11.0, 95% CI = 4.02–30.1, *P* < .001, respectively, [Table 3](#T3)). The FPRP was 0.02 for patients carrying 3–4 at-risk genotypes, indicating noteworthiness. No significant association of toxicity was observed in the remaining SNPs (data not shown).

### Table 3. Neutropenia toxicity and genotype (n = 149).

| Genotype | Grade1-2 | Grade 3-4 |   |   |
| --- | --- | --- | --- | --- |
|   |   |   |   |   |
|   | N (%) | N (%) | OR* (95% CI) | P |
| CDA A-76C (K27Q) |   |   |   |   |
| AA | 45 (75.0) | 15 (25.0) | 1.0 |   |
| AC/CC | 50 (56.2) | 39 (43.8) | 2.34 (1.14-4.80) | .020 |
| dCK C-1205T |   |   |   |   |
| CC/CT | 67 (71.3) | 27 (28.7) | 1.0 |   |
| TT | 27 (50.9) | 26 (49.1) | 2.39 (1.19-4.81) | .015 |
| RRM1 A33G (T741T) |   |   |   |   |
| AG/GG | 76 (72.4) | 29 (27.6) | 1.0 |   |
| AA | 18 (45.0) | 22 (55.0) | 3.20 (1.50-6.82) | .003 |
| hENT1 C913T |   |   |   |   |
| CC | 37 (77.1) | 11 (22.9) | 1.0 |   |
| CT/TT | 56 (56.6) | 43 (43.4) | 2.58 (1.18-5.64) | .017 |
|   |  |  |  |  |
| No. of at-risk genotypes† |   |   |   |   |
| 0-1 | 44 (86.3) | 7 (13.7) | 1.0 |   |
| 2 | 31 (66.0) | 16 (34.0) | 3.24 (1.19-8.82) | .021 |
| 3-4 | 16 (36.4) | 28 (63.6) | 11.00 (4.02-30.1) | <.001 |

Table 3 Caption: *Crude odds ratio. †CDA -76AC/CC, dCK -1205TT, RRM1 33AA, and hENT1 913CT/TT.

### Association of Genotypes with Tumor Response to Therapy

149 LAPC patients were analyzed on treatment effect. Radiation therapy and platinum drug use did not correlate with tumor response (*P* = .858 and *P* = .562). Two SNPs, *CDA* A-76C and *hENT1* A-201G, were significantly associated with tumor response in radiological evaluation after adjusting for age (*P* = .017 and *P* = .019, [Table 4](#T4)). For example, 41 (48.2%) of the *CDA* -76 AC/CC carriers compared with 16 (27.6%) of the AA carriers had a poor response to gemcitabine-based chemotherapy. Patients carrying 1–2 at-risk alleles had a significantly worse response to therapy than did patients carrying no at-risk alleles (OR = 3.40, 95% CI = 1.49–7.78, *P* = .004). The FPRP was 0.097 for patients carrying 1–2 at-risk genotypes, indicating noteworthiness. Gemcitabine dose intensity was slightly lower in *CDA* CC/AC variant carriers (683 ± 31 mg/m^2^) than that in the AA carriers (752 ± 46 mg/m^2^) but the difference was not statistically significant (*P* = .217).

### Table 4. Tumor response to therapy and genotype (n = 149).

| Genotype | PR/SD | PD |   |   |
| --- | --- | --- | --- | --- |
|   |   |   |   |   |
|   | N (%) | N (%) | OR* (95% CI) | P |
| CDA A-76C (K27Q) |   |   |   |   |
| AA | 42 (72.4) | 16 (27.6) | 1.0 |   |
| AC/CC | 44 (51.8) | 41 (48.2) | 2.50 (1.18-5.28) | .017 |
| hENT1 A-201G |   |   |   |   |
| AA/AG | 80 (65.0) | 43 (35.0) | 1.0 |   |
| GG | 6 (33.3) | 12 (66.7) | 3.63 (1.23-10.7) | .019 |
|   |  |  |  |  |
| No. of at-risk genotypes† |   |   |   |   |
| 0 | 38 (77.6) | 11 (22.4) | 1.0 |   |
| 1-2 | 48 (52.2) | 44 (47.8) | 3.40 (1.49-7.78) | .004 |

Table 4 Caption: Abbreviations: PR, partial response; SD, stable disease; PD, progressive disease. *OR was adjusted for age. †CDA -76AC/CC and hENT1 -201GG.

### Genotype Frequency and its Association with OS and PFS

None of the examined 17 SNPs was associated with OS (data not shown). The data of 137 LAPC patients were available for PFS analysis. Individually, two SNPs (*RRM1* A33G, *RRM1* C-27A) showed significant association with PFS (*P* = .048 and *P* = .042, respectively, [Table 5](#T5)). In addition, when the *CDA* A-76C and *hENT*1 A-201G variants were analyzed in combination with *RRM1* A33G and *RRM1* C-27A, a gene-dosage effect on PFS was observed. As the number of at-risk alleles increased, the PFS decreased ([Fig. 2](#F2)). Patients carrying 0–1 (n = 64), 2 (n = 50), or 3–4 (n = 17) at-risk alleles had median PFS times of 8.3, 6.0, and 4.2 months ([Table 5](#T5)), as well as 6-month progression-free rate of 76.5%, 52.0%, and 29.4%, respectively. The HR (95% CI) of progression was 1.79 (1.20–2.66) and 3.25 (1.79–5.90) for patients carrying 2 and 3–4 at-risk genotypes (*P* = .004 and *P* < .001, [Table 5](#T5)), after adjusting for performance status and tumor size. The FPRPs for patients carrying 2 and 3–4 at-risk genotypes were 0.017 and 0.006, respectively, indicating noteworthiness.

### Table 5. Progression-free survival and genotype (n = 137).

| Genotype | No. of Cases | No. of Events | TTP ± SE(months) | P(log-rank) | *HR (95% CI) | P |
| --- | --- | --- | --- | --- | --- | --- |
| CDA C111T (T145T) |   |   |   | .473 |   |   |
| CC | 56 | 52 | 7.6 ± 0.8 |   |   |   |
| CT | 67 | 53 | 7.1 ± 0.9 |   |   |   |
| TT | 14 | 14 | 6.8 ± 1.7 |   |   |   |
| CC vs. CT/TT |   |   |   |   | 1.22 (0.85-1.76) | .281 |
| CDA A-76C (K27Q) |   |   |   | .384 |   |   |
| AA | 52 | 48 | 8.2 ± 0.6 |   |   |   |
| AC | 67 | 63 | 6.5 ± 0.9 |   |   |   |
| CC | 18 | 18 | 5.5 ± 0.5 |   |   |   |
| AA vs. AC/CC |   |   |   |   | 1.27 (0.89-1.82) | .192 |
| dCK C-1205T |   |   |   | .839 |   |   |
| CC | 21 | 19 | 8.6 ± 1.2 |   |   |   |
| CT | 65 | 60 | 6.5 ± 0.4 |   |   |   |
| TT | 49 | 48 | 8.0 ± 0.4 |   |   |   |
| CC/TT vs. CT |   |   |   |   | 0.92 (0.64-1.32) | .653 |
| dCK A9846G |   |   |   | .866 |   |   |
| AA | 22 | 21 | 7.2 ± 0.8 |   |   |   |
| AG | 73 | 69 | 7.5 ± 0.8 |   |   |   |
| GG | 39 | 37 | 7.1 ± 1.4 |   |   |   |
| AG vs. AA/GG |   |   |   |   | 1.05 (0.73-1.50) | .811 |
| RRM1 G42A (A744A) |   |   |   | .462 |   |   |
| AA | 72 | 67 | 7.5 ± 0.5 |   |   |   |
| AG | 54 | 53 | 6.5 ± 1.3 |   |   |   |
| GG | 10 | 8 | 7.6 ± 1.1 |   |   |   |
| AA/GG vs. AG |   |   |   |   | 1.14 (0.79-1.63) | .486 |
| RRM1 A33G (T741T) |   |   |   | .339 |   |   |
| AA | 35 | 34 | 5.8 ± 0.6 |   |   |   |
| AG | 69 | 63 | 7.5 ± 0.5 |   |   |   |
| GG | 29 | 28 | 7.8 ± 1.0 |   |   |   |
| AG/GG vs. AA |   |   |   |   | 1.53 (1.00-2.34) | .048 |
| RRM1 C-27A (R284R) |   |   |   | .097 |   |   |
| CC | 30 | 25 | 7.5 ± 1.3 |   |   |   |
| AC | 73 | 72 | 6.8 ± 0.7 |   |   |   |
| AA | 33 | 31 | 8.0 ± 0.8 |   |   |   |
| AA/CC vs. AC |   |   |   |   | 1.46 (1.02-2.11) | .042 |
| DCTD T-47C (V116V) |   |   |   | .189 |   |   |
| TT | 77 | 72 | 7.1 ± 0.5 |   |   |   |
| CT | 50 | 47 | 8.0 ± 0.4 |   |   |   |
| CC | 10 | 10 | 5.1 ± 0.7 |   |   |   |
| CC/TT vs. CT |   |   |   |   | 1.07 (0.74-1.55) | .729 |
| hCNT1 A-16G (Q456Q) |   |   |   | .461 |   |   |
| AA | 6 | 6 | 6.4 ± 2.4 |   |   |   |
| AG | 32 | 28 | 7.8 ± 0.9 |   |   |   |
| GG | 98 | 94 | 7.2 ± 0.6 |   |   |   |
| AG vs. AA/GG |   |   |   |   | 1.18 (0.76-1.85) | .465 |
| hCNT1 C-9A (Q237K) |   |   |   | .787 |   |   |
| CC | 64 | 62 | 7.5 ± 0.7 |   |   |   |
| AC | 52 | 47 | 6.5 ± 1.5 |   |   |   |
| AA | 20 | 19 | 7.5 ± 0.4 |   |   |   |
| AA/AC vs. CC |   |   |   |   | 1.14 (0.80-1.62) | .482 |
| hCNT2 C-38A (S75R) |   |   |   | .559 |   |   |
| CC | 36 | 35 | 6.9 ± 1.6 |   |   |   |
| AC | 62 | 56 | 7.5 ± 0.8 |   |   |   |
| AA | 38 | 37 | 7.1 ± 0.7 |   |   |   |
| AC vs. AA/CC |   |   |   |   | 1.02 (0.71-1.47) | .923 |
| hCNT2 C-17T (P22L) |   |   |   | .874 |   |   |
| CC | 38 | 36 | 8.0 ± 0.8 |   |   |   |
| CT | 70 | 66 | 6.8 ± 0.7 |   |   |   |
| TT | 28 | 26 | 7.6 ± 1.2 |   |   |   |
| CC/TT vs. CT |   |   |   |   | 1.21 (0.85-1.74) | .288 |
| hCNT3 C-69T (L461L) |   |   |   | .154 |   |   |
| CC | 68 | 65 | 7.5 ± 0.8 |   |   |   |
| CT | 58 | 53 | 7.5 ± 0.6 |   |   |   |
| TT | 10 | 10 | 5.2 ± 0.6 |   |   |   |
| CT/TT vs. CC |   |   |   |   | 1.14 (0.81-1.62) | .455 |
| hCNT3 A25G (T89T) |   |   |   | .648 |   |   |
| AA | 54 | 52 | 7.5 ± 0.5 |   |   |   |
| AG | 59 | 54 | 7.2 ± 0.9 |   |   |   |
| GG | 24 | 23 | 5.6 ± 1.6 |   |   |   |
| AA/AG vs. GG |   |   |   |   | 1.16 (0.73-1.84) | .532 |
| hENT1 A-201G |   |   |   | .156 |   |   |
| AA | 61 | 57 | 8.0 ± 1.0 |   |   |   |
| AG | 57 | 54 | 8.0 ± 0.4 |   |   |   |
| GG | 17 | 16 | 5.1 ± 1.1 |   |   |   |
| AA/AG vs. GG |   |   |   |   | 1.70 (0.97-3.01) | .066 |
| hENT1 T-549C |   |   |   | .777 |   |   |
| CC | 63 | 59 | 6.9 ± 0.9 |   |   |   |
| CT | 61 | 58 | 7.5 ± 1.0 |   |   |   |
| TT | 11 | 11 | 8.3 ± 1.5 |   |   |   |
| TT/CT vs. CC |   |   |   |   | 1.12 (0.79-1.60) | .531 |
| hENT1 C913T |   |   |   | .987 |   |   |
| CC | 41 | 39 | 8.0 ± 0.8 |   |   |   |
| CT | 61 | 58 | 7.1 ± 0.6 |   |   |   |
| TT | 33 | 30 | 8.0 ± 1.6 |   |   |   |
| CC/TT vs. CT |   |   |   |   | 1.08 (0.74-1.57) | .704 |
|   |  |  |  |  |  |  |
| No. of at-risk genotypes† |   |   |   | .002 |   |   |
| 0-1 | 64 | 57 | 8.3 ± 0.5 |   | reference |   |
| 2 | 50 | 49 | 6.0 ± 0.8 |   | 1.79 (1.20-2.66) | .004 |
| 3-4 | 17 | 17 | 4.2 ± 1.5 |   | 3.25 (1.79-5.90) | <.001 |

Table 5 Caption: Abbreviations: TTP, time to progression; SE, standard error. *HR was adjusted for performance status and tumor size. †CDA -76AC/CC, RRM1 33AA, RRM1 -27AC, and hENT1 -201GG.

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1fe2/2966859/a56a2b491584/nihms190212f2.jpg)

Kaplan-Meier plot to assess the combined genotype effect of CDA -76AC/CC, RRM1 33AA, RRM1 -27AC, and hENT1 -201GG on progression-free survival. The number of 0 to 4 indicates the number of at-risk genotypes associated with reduced progression-free survival (Log-rank P = .002).

## Discussion

Our results in this study support the hypothesis that SNPs of gemcitabine metabolic and transporter genes are associated with clinical outcome in patients with LAPC. The gene variants of *CDA* A-76C, *dCK* C-1205T*, RRM1* A33G*, and hENT1* C913T correlated with severe neutropenia. In addition, the *CDA* A-76C and *hENT1* A-201G genotypes were significantly associated with tumor response to gemcitabine-based therapy and were marginally associated with PFS. These genotype effects remained significant after adjusting for clinical predictors in statistics.

CDA is involved in the salvage pathway of pyrimidine and plays a key role in detoxifying gemcitabine.[9](#R9) Three main SNPs have been identified in the *CDA* gene: C111T (T145T), A-76C (K27Q), and G208A (A70T).[8](#R8)^,^[37](#R37)^,^[38](#R38) Although the *CDA* 208AA homozygote allele and its related haplotype have been associated with severe drug toxicity in Japanese cancer patients treated with gemcitabine plus cisplatin, we excluded this SNP from our study because *CDA* G208A had not been detected in Caucasians.[29](#R29)^,^[31](#R31)^,^[32](#R32) The *CDA* A-76C variant C allele (Gln27) has been reported to have moderately or significantly lower deaminase activity for gemcitabine or cytosine arabinoside (ara-C) than wild-type genotype.[28](#R28)^,^[39](#R39) Our data showed significantly higher toxicity in the *CDA* -76 CC/AC variant than in the AA wild-type, suggesting lower deaminase activity of the C allele (Gln27) variant, which is consistent with previously reported data from in vitro studies.[28](#R28)^,^[39](#R39) Although our results indicated that the *CDA* -76 CC/AC variant was also associated with poorer tumor response, we do not feel this is due to dose reductions as there was no significant difference in the gemcitabine dose intensity in the *CDA* -76 CC/AC variant carriers as compared with the AA carriers. Nevertheless, there were controversial findings on this SNP in previous studies. The *CDA* A-76C variant A allele (Lys27) had significantly lower deaminase activity than the C allele (Gln27) in a study conducted in 90 patients with lung cancer.[40](#R40) The Lys27 haplotype did not show any significant effect on gemcitabine pharmacokinetics in a study of 256 Japanese patients.[32](#R32) Future studies are warranted to clarify the functional and clinical importance of this SNP in gemcitabine therapy.

dCK is the rate-limiting enzyme for intracellular activation of gemcitabine and was therefore thought to play an important role in sensitivity to gemcitabine.[9](#R9) Some studies have shown that the enzyme activity or expression level of dCK was associated with sensitivity to gemcitabine and survival of pancreatic cancer patients.[17](#R17)^,^[41](#R41) Shi et al reported that the haplotype containing *dCK* C-360G and C-201T had a significant association with higher levels of dCK mRNA and longer survival time of patients with acute myeloid leukemia treated with ara-C.[20](#R20) Our study showed a significantly higher toxicity in patients with the *dCK* -1205 TT variant than the CC/CT variant. Because this SNP is located in intronic region, it is not clear whether it directly affects dCK enzyme activity or whether it is in linkage disequilibrium with other functional SNPs or other genes.

RRM1 is essential for DNA synthesis and repair.[9](#R9) Davidson et al reported that the increased mRNA level of RRM1 resulted in drug resistance.[26](#R26) In a different study, Rha et al demonstrated a strong association between gemcitabine-induced neutropenia and the *RRM1* haplotype containing two SNPs (A2455G and G2464A).[33](#R33) Our data showed that the *RRM1* 33 AA variant was significantly associated with severe toxicity, suggesting a high susceptibility of this variant to gemcitabine. *RRM1* A33G is a synonymous SNP (T741T) that does not produce amino acid change. However, Kimchi-Sarfaty et al reported that a synonymous SNP in the *MDR*1 gene yielded a protein product with altered drug and inhibitor interactions.[42](#R42) Thus, the functional consequence of *RRM1* A33G SNP should be further investigated.

Nucleoside transporters have been thought to have an important role in gemcitabine cytotoxicity and efficacy.[16](#R16) Gemcitabine intracellular uptake is mediated mainly by hENT1 and, to a lesser extent, by hCNT1 and hCNT3,[9](#R9) supporting our current observations that the *hENT1* C913T genotype was significantly associated with neutropenia toxicity and the *hENT1* A-201G genotype with tumor response to gemcitabine and PFS. While two previous studies on the nonsynonymous SNPs of *hENT1* failed to demonstrate functional diversity,[43](#R43)^,^[44](#R44) it was reported that the CGG/CGC haplotypes of the *hENT1* promoter region containing the C-1345G, G-1050A, and G-706C SNPs showed moderately higher expression of *hENT1*.[45](#R45) The functional significance of the polymorphic variants investigated in our current study has not yet been demonstrated. Considering that *hENT1* expression has been associated with survival of patients with pancreatic cancer,[27](#R27) further genotype-phenotype analysis would be needed to clarify whether the *hENT1* genotype can be used as a surrogate marker for hENT1 activity.

In this study, we focused on LAPC because metastatic pancreatic cancer is associated with greater clinical and biological heterogeneity and in most instances, patients were seen in consultation at our institute but their primary treatments for metastatic disease were administered at other referring facilities. Comparing to findings of our previous study in patients with potentially resectable pancreatic cancer who underwent neoadjuvant gemcitabine-based chemoradiation[34](#R34), although the clinical characteristics of the two study populations are quite different, the association of *dCK -1205* T allele with severe gemcitabine toxicity and *hENT1* -201 A allele with better survival were observed in both studies, suggesting the robustness of these findings. In most LAPC cases, tissue samples are unavailable for measurement of protein expression. Therefore, if genotyping data from peripheral blood DNA is validated and found to be a reliable predictor for gemcitabine toxicity and efficacy, application of such data would be widely beneficial for patients with unresectable advanced pancreatic cancer.

In conclusion, genotypes of gemcitabine metabolic and transporter genes have potential as predictive biomarkers for toxicity and treatment effects of gemcitabine-based therapy in LAPC patients. Our observations still need to be confirmed in separate and larger patient populations. If confirmed, these findings may be helpful in stratifying patients to individualized therapy.

## Acknowledgments

**Grant Support:** Supported by the National Institutes of Health (NIH) R01 grant CA098380 (D.L.), SPORE P20 grant CA101936 (J.L.A.), NIH Cancer Center Core grant CA16672, Lockton Research Funds research grant (D.L.), and the Urbieta Family Cancer Fund (C.E.).

## Footnotes

## References

1. Jemal A, Siegel R, Ward E, Hao Y, Xu J, Thun MJ. Cancer statistics, 2009. CA Cancer J Clin. 2009;59:225–49. doi: 10.3322/caac.20006.  [DOI](https://doi.org/10.3322/caac.20006) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19474385/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=CA%20Cancer%20J%20Clin&title=Cancer%20statistics,%202009&author=A%20Jemal&author=R%20Siegel&author=E%20Ward&author=Y%20Hao&author=J%20Xu&volume=59&publication_year=2009&pages=225-49&pmid=19474385&doi=10.3322/caac.20006&)

2. Yeo TP, Hruban RH, Leach SD, Wilentz RE, Sohn TA, Kern SE, et al. Pancreatic cancer. Curr Probl Cancer. 2002;26:176–275. doi: 10.1067/mcn.2002.129579.  [DOI](https://doi.org/10.1067/mcn.2002.129579) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12399802/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Probl%20Cancer&title=Pancreatic%20cancer&author=TP%20Yeo&author=RH%20Hruban&author=SD%20Leach&author=RE%20Wilentz&author=TA%20Sohn&volume=26&publication_year=2002&pages=176-275&pmid=12399802&doi=10.1067/mcn.2002.129579&)

3. Burris HA, 3rd, Moore MJ, Andersen J, Green MR, Rothenberg ML, Modiano MR, et al. Improvements in survival and clinical benefit with gemcitabine as first-line therapy for patients with advanced pancreas cancer: a randomized trial. J Clin Oncol. 1997;15:2403–2413. doi: 10.1200/JCO.1997.15.6.2403.  [DOI](https://doi.org/10.1200/JCO.1997.15.6.2403) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9196156/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Improvements%20in%20survival%20and%20clinical%20benefit%20with%20gemcitabine%20as%20first-line%20therapy%20for%20patients%20with%20advanced%20pancreas%20cancer:%20a%20randomized%20trial&author=HA%20Burris&author=MJ%20Moore&author=J%20Andersen&author=MR%20Green&author=ML%20Rothenberg&volume=15&publication_year=1997&pages=2403-2413&pmid=9196156&doi=10.1200/JCO.1997.15.6.2403&)

4. Huguet F, Andre T, Hammel P, Artru P, Balosso J, Selle F, et al. Impact of chemoradiotherapy after disease control with chemotherapy in locally advanced pancreatic adenocarcinoma in GERCOR phase II and III studies. J Clin Oncol. 2007;25:326–331. doi: 10.1200/JCO.2006.07.5663.  [DOI](https://doi.org/10.1200/JCO.2006.07.5663) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17235048/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Impact%20of%20chemoradiotherapy%20after%20disease%20control%20with%20chemotherapy%20in%20locally%20advanced%20pancreatic%20adenocarcinoma%20in%20GERCOR%20phase%20II%20and%20III%20studies&author=F%20Huguet&author=T%20Andre&author=P%20Hammel&author=P%20Artru&author=J%20Balosso&volume=25&publication_year=2007&pages=326-331&pmid=17235048&doi=10.1200/JCO.2006.07.5663&)

5. Krishnan S, Rana V, Janjan NA, Varadhachary GR, Abbruzzese JL, Das P, et al. Induction chemotherapy selects patients with locally advanced, unresectable pancreatic cancer for optimal benefit from consolidative chemoradiation therapy. Cancer. 2007;110:47–55. doi: 10.1002/cncr.22735.  [DOI](https://doi.org/10.1002/cncr.22735) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17538975/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer&title=Induction%20chemotherapy%20selects%20patients%20with%20locally%20advanced,%20unresectable%20pancreatic%20cancer%20for%20optimal%20benefit%20from%20consolidative%20chemoradiation%20therapy&author=S%20Krishnan&author=V%20Rana&author=NA%20Janjan&author=GR%20Varadhachary&author=JL%20Abbruzzese&volume=110&publication_year=2007&pages=47-55&pmid=17538975&doi=10.1002/cncr.22735&)

6. Jimeno A, Hidalgo M. Molecular biomarkers: their increasing role in the diagnosis, characterization, and therapy guidance in pancreatic cancer. Mol Cancer Ther. 2006;5:787–796. doi: 10.1158/1535-7163.MCT-06-0005.  [DOI](https://doi.org/10.1158/1535-7163.MCT-06-0005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16648548/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Cancer%20Ther&title=Molecular%20biomarkers:%20their%20increasing%20role%20in%20the%20diagnosis,%20characterization,%20and%20therapy%20guidance%20in%20pancreatic%20cancer&author=A%20Jimeno&author=M%20Hidalgo&volume=5&publication_year=2006&pages=787-796&pmid=16648548&doi=10.1158/1535-7163.MCT-06-0005&)

7. Mini E, Nobili S, Caciagli B, Landini I, Mazzei T. Cellular pharmacology of gemcitabine. Ann Oncol. 2006;17(Suppl 5):v7–v12. doi: 10.1093/annonc/mdj941.  [DOI](https://doi.org/10.1093/annonc/mdj941) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16807468/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Oncol&title=Cellular%20pharmacology%20of%20gemcitabine&author=E%20Mini&author=S%20Nobili&author=B%20Caciagli&author=I%20Landini&author=T%20Mazzei&volume=17&issue=Suppl%205&publication_year=2006&pages=v7-v12&pmid=16807468&doi=10.1093/annonc/mdj941&)

8. Fukunaga AK, Marsh S, Murry DJ, Hurley TD, McLeod HL. Identification and analysis of single-nucleotide polymorphisms in the gemcitabine pharmacologic pathway. Pharmacogenomics J. 2004;4:307–314. doi: 10.1038/sj.tpj.6500259.  [DOI](https://doi.org/10.1038/sj.tpj.6500259) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15224082/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Identification%20and%20analysis%20of%20single-nucleotide%20polymorphisms%20in%20the%20gemcitabine%20pharmacologic%20pathway&author=AK%20Fukunaga&author=S%20Marsh&author=DJ%20Murry&author=TD%20Hurley&author=HL%20McLeod&volume=4&publication_year=2004&pages=307-314&pmid=15224082&doi=10.1038/sj.tpj.6500259&)

9. Ueno H, Kiyosawa K, Kaniwa N. Pharmacogenomics of gemcitabine: can genetic studies lead to tailor-made therapy? Br J Cancer. 2007;97:145–151. doi: 10.1038/sj.bjc.6603860.  [DOI](https://doi.org/10.1038/sj.bjc.6603860) | [PMC free article](/articles/PMC2360307/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17595663/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Cancer&title=Pharmacogenomics%20of%20gemcitabine:%20can%20genetic%20studies%20lead%20to%20tailor-made%20therapy?&author=H%20Ueno&author=K%20Kiyosawa&author=N%20Kaniwa&volume=97&publication_year=2007&pages=145-151&pmid=17595663&doi=10.1038/sj.bjc.6603860&)

10. Heinemann V, Schulz L, Issels RD, Plunkett W. Gemcitabine: a modulator of intracellular nucleotide and deoxynucleotide metabolism. Semin Oncol. 1995;22(4 Suppl 11):11–18.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/7481839/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Semin%20Oncol&title=Gemcitabine:%20a%20modulator%20of%20intracellular%20nucleotide%20and%20deoxynucleotide%20metabolism&author=V%20Heinemann&author=L%20Schulz&author=RD%20Issels&author=W%20Plunkett&volume=22&issue=4%20Suppl%2011&publication_year=1995&pages=11-18&pmid=7481839&)

11. Baker CH, Banzon J, Bollinger JM, Stubbe J, Samano V, Robins MJ, et al. 2′-Deoxy-2′-methylenecytidine and 2′-deoxy-2′,2′-difluorocytidine 5′-diphosphates: potent mechanism-based inhibitors of ribonucleotide reductase. J Med Chem. 1991;34:1879–1884. doi: 10.1021/jm00110a019.  [DOI](https://doi.org/10.1021/jm00110a019) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2061926/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Med%20Chem&title=2%E2%80%B2-Deoxy-2%E2%80%B2-methylenecytidine%20and%202%E2%80%B2-deoxy-2%E2%80%B2,2%E2%80%B2-difluorocytidine%205%E2%80%B2-diphosphates:%20potent%20mechanism-based%20inhibitors%20of%20ribonucleotide%20reductase&author=CH%20Baker&author=J%20Banzon&author=JM%20Bollinger&author=J%20Stubbe&author=V%20Samano&volume=34&publication_year=1991&pages=1879-1884&pmid=2061926&doi=10.1021/jm00110a019&)

12. Yen Y. Ribonucleotide reductase subunit one as gene therapy target. Clin Cancer Res. 2003;9:4304–4308.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/14555499/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=Ribonucleotide%20reductase%20subunit%20one%20as%20gene%20therapy%20target&author=Y%20Yen&volume=9&publication_year=2003&pages=4304-4308&pmid=14555499&)

13. Ganti AK, Parr A, Nguyen D, Grem JL. Gene expression profile of enzymes involved in gemcitabine (Gem) metabolism in bone marrow mononuclear cells as predictors of myelosuppression. J Clin Oncol. 2006;24 Abstract 2007.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Gene%20expression%20profile%20of%20enzymes%20involved%20in%20gemcitabine%20(Gem)%20metabolism%20in%20bone%20marrow%20mononuclear%20cells%20as%20predictors%20of%20myelosuppression&author=AK%20Ganti&author=A%20Parr&author=D%20Nguyen&author=JL%20Grem&volume=24&publication_year=2006&)

14. Achiwa H, Oguri T, Sato S, Maeda H, Niimi T, Ueda R. Determinants of sensitivity and resistance to gemcitabine: the roles of human equilibrative nucleoside transporter 1 and deoxycytidine kinase in non-small cell lung cancer. Cancer Sci. 2004;95:753–757. doi: 10.1111/j.1349-7006.2004.tb03257.x.  [DOI](https://doi.org/10.1111/j.1349-7006.2004.tb03257.x) | [PMC free article](/articles/PMC11158492/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15471562/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Sci&title=Determinants%20of%20sensitivity%20and%20resistance%20to%20gemcitabine:%20the%20roles%20of%20human%20equilibrative%20nucleoside%20transporter%201%20and%20deoxycytidine%20kinase%20in%20non-small%20cell%20lung%20cancer&author=H%20Achiwa&author=T%20Oguri&author=S%20Sato&author=H%20Maeda&author=T%20Niimi&volume=95&publication_year=2004&pages=753-757&pmid=15471562&doi=10.1111/j.1349-7006.2004.tb03257.x&)

15. Giovannetti E, Del Tacca M, Mey V, Funel N, Nannizzi S, Ricci S, et al. Transcription analysis of human equilibrative nucleoside transporter-1 predicts survival in pancreas cancer patients treated with gemcitabine. Cancer Res. 2006;66:3928–3935. doi: 10.1158/0008-5472.CAN-05-4203.  [DOI](https://doi.org/10.1158/0008-5472.CAN-05-4203) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16585222/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=Transcription%20analysis%20of%20human%20equilibrative%20nucleoside%20transporter-1%20predicts%20survival%20in%20pancreas%20cancer%20patients%20treated%20with%20gemcitabine&author=E%20Giovannetti&author=M%20Del%20Tacca&author=V%20Mey&author=N%20Funel&author=S%20Nannizzi&volume=66&publication_year=2006&pages=3928-3935&pmid=16585222&doi=10.1158/0008-5472.CAN-05-4203&)

16. Spratlin J, Sangha R, Glubrecht D, Dabbagh L, Young JD, Dumontet C, et al. The absence of human equilibrative nucleoside transporter 1 is associated with reduced survival in patients with gemcitabine-treated pancreas adenocarcinoma. Clin Cancer Res. 2004;10:6956–6961. doi: 10.1158/1078-0432.CCR-04-0224.  [DOI](https://doi.org/10.1158/1078-0432.CCR-04-0224) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15501974/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=The%20absence%20of%20human%20equilibrative%20nucleoside%20transporter%201%20is%20associated%20with%20reduced%20survival%20in%20patients%20with%20gemcitabine-treated%20pancreas%20adenocarcinoma&author=J%20Spratlin&author=R%20Sangha&author=D%20Glubrecht&author=L%20Dabbagh&author=JD%20Young&volume=10&publication_year=2004&pages=6956-6961&pmid=15501974&doi=10.1158/1078-0432.CCR-04-0224&)

17. Kroep JR, Loves WJ, van der Wilt CL, Alvarez E, Talianidis I, Boven E, et al. Pretreatment deoxycytidine kinase levels predict in vivo gemcitabine sensitivity. Mol Cancer Ther. 2002;1:371–376.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/12477049/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Cancer%20Ther&title=Pretreatment%20deoxycytidine%20kinase%20levels%20predict%20in%20vivo%20gemcitabine%20sensitivity&author=JR%20Kroep&author=WJ%20Loves&author=CL%20van%20der%20Wilt&author=E%20Alvarez&author=I%20Talianidis&volume=1&publication_year=2002&pages=371-376&pmid=12477049&)

18. Jordheim LP, Galmarini CM, Dumontet C. Gemcitabine resistance due to deoxycytidine kinase deficiency can be reverted by fruitfly deoxynucleoside kinase, DmdNK, in human uterine sarcoma cells. Cancer Chemother Pharmacol. 2006;58:547–554. doi: 10.1007/s00280-006-0195-8.  [DOI](https://doi.org/10.1007/s00280-006-0195-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16463058/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Chemother%20Pharmacol&title=Gemcitabine%20resistance%20due%20to%20deoxycytidine%20kinase%20deficiency%20can%20be%20reverted%20by%20fruitfly%20deoxynucleoside%20kinase,%20DmdNK,%20in%20human%20uterine%20sarcoma%20cells&author=LP%20Jordheim&author=CM%20Galmarini&author=C%20Dumontet&volume=58&publication_year=2006&pages=547-554&pmid=16463058&doi=10.1007/s00280-006-0195-8&)

19. Galmarini CM, Clarke ML, Jordheim L, Santos CL, Cros E, Mackey JR, et al. Resistance to gemcitabine in a human follicular lymphoma cell line is due to partial deletion of the deoxycytidine kinase gene. BMC Pharmacol. 2004;4:8. doi: 10.1186/1471-2210-4-8.  [DOI](https://doi.org/10.1186/1471-2210-4-8) | [PMC free article](/articles/PMC428575/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15157282/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Pharmacol&title=Resistance%20to%20gemcitabine%20in%20a%20human%20follicular%20lymphoma%20cell%20line%20is%20due%20to%20partial%20deletion%20of%20the%20deoxycytidine%20kinase%20gene&author=CM%20Galmarini&author=ML%20Clarke&author=L%20Jordheim&author=CL%20Santos&author=E%20Cros&volume=4&publication_year=2004&pages=8&pmid=15157282&doi=10.1186/1471-2210-4-8&)

20. Shi JY, Shi ZZ, Zhang SJ, Zhu YM, Gu BW, Li G, et al. Association between single nucleotide polymorphisms in deoxycytidine kinase and treatment response among acute myeloid leukaemia patients. Pharmacogenetics. 2004;14:759–768. doi: 10.1097/00008571-200411000-00007.  [DOI](https://doi.org/10.1097/00008571-200411000-00007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15564883/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Association%20between%20single%20nucleotide%20polymorphisms%20in%20deoxycytidine%20kinase%20and%20treatment%20response%20among%20acute%20myeloid%20leukaemia%20patients&author=JY%20Shi&author=ZZ%20Shi&author=SJ%20Zhang&author=YM%20Zhu&author=BW%20Gu&volume=14&publication_year=2004&pages=759-768&pmid=15564883&doi=10.1097/00008571-200411000-00007&)

21. Bergman AM, Pinedo HM, Peters GJ. Determinants of resistance to 2′,2′-difluorodeoxycytidine (gemcitabine) Drug Resist Updat. 2002;5:19–33. doi: 10.1016/s1368-7646(02)00002-x.  [DOI](https://doi.org/10.1016/s1368-7646(02)00002-x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12127861/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Resist%20Updat&title=Determinants%20of%20resistance%20to%202%E2%80%B2,2%E2%80%B2-difluorodeoxycytidine%20(gemcitabine)&author=AM%20Bergman&author=HM%20Pinedo&author=GJ%20Peters&volume=5&publication_year=2002&pages=19-33&pmid=12127861&doi=10.1016/s1368-7646(02)00002-x&)

22. Eliopoulos N, Cournoyer D, Momparler RL. Drug resistance to 5-aza-2′-deoxycytidine, 2′,2′-difluorodeoxycytidine, and cytosine arabinoside conferred by retroviral-mediated transfer of human cytidine deaminase cDNA into murine cells. Cancer Chemother Pharmacol. 1998;42:373–378. doi: 10.1007/s002800050832.  [DOI](https://doi.org/10.1007/s002800050832) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9771951/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Chemother%20Pharmacol&title=Drug%20resistance%20to%205-aza-2%E2%80%B2-deoxycytidine,%202%E2%80%B2,2%E2%80%B2-difluorodeoxycytidine,%20and%20cytosine%20arabinoside%20conferred%20by%20retroviral-mediated%20transfer%20of%20human%20cytidine%20deaminase%20cDNA%20into%20murine%20cells&author=N%20Eliopoulos&author=D%20Cournoyer&author=RL%20Momparler&volume=42&publication_year=1998&pages=373-378&pmid=9771951&doi=10.1007/s002800050832&)

23. Nakano Y, Tanno S, Koizumi K, Nishikawa T, Nakamura K, Minoguchi M, et al. Gemcitabine chemoresistance and molecular markers associated with gemcitabine transport and metabolism in human pancreatic cancer cells. Br J Cancer. 2007;96:457–463. doi: 10.1038/sj.bjc.6603559.  [DOI](https://doi.org/10.1038/sj.bjc.6603559) | [PMC free article](/articles/PMC2360025/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17224927/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Cancer&title=Gemcitabine%20chemoresistance%20and%20molecular%20markers%20associated%20with%20gemcitabine%20transport%20and%20metabolism%20in%20human%20pancreatic%20cancer%20cells&author=Y%20Nakano&author=S%20Tanno&author=K%20Koizumi&author=T%20Nishikawa&author=K%20Nakamura&volume=96&publication_year=2007&pages=457-463&pmid=17224927&doi=10.1038/sj.bjc.6603559&)

24. Rosell R, Danenberg KD, Alberola V, Bepler G, Sanchez JJ, Camps C, et al. Ribonucleotide reductase messenger RNA expression and survival in gemcitabine/cisplatin-treated advanced non-small cell lung cancer patients. Clin Cancer Res. 2004;10:1318–1325. doi: 10.1158/1078-0432.ccr-03-0156.  [DOI](https://doi.org/10.1158/1078-0432.ccr-03-0156) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14977831/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=Ribonucleotide%20reductase%20messenger%20RNA%20expression%20and%20survival%20in%20gemcitabine/cisplatin-treated%20advanced%20non-small%20cell%20lung%20cancer%20patients&author=R%20Rosell&author=KD%20Danenberg&author=V%20Alberola&author=G%20Bepler&author=JJ%20Sanchez&volume=10&publication_year=2004&pages=1318-1325&pmid=14977831&doi=10.1158/1078-0432.ccr-03-0156&)

25. Bepler G, Kusmartseva I, Sharma S, Gautam A, Cantor A, Sharma A, et al. RRM1 modulated in vitro and in vivo efficacy of gemcitabine and platinum in non-small-cell lung cancer. J Clin Oncol. 2006;24:4731–4737. doi: 10.1200/JCO.2006.06.1101.  [DOI](https://doi.org/10.1200/JCO.2006.06.1101) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16966686/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=RRM1%20modulated%20in%20vitro%20and%20in%20vivo%20efficacy%20of%20gemcitabine%20and%20platinum%20in%20non-small-cell%20lung%20cancer&author=G%20Bepler&author=I%20Kusmartseva&author=S%20Sharma&author=A%20Gautam&author=A%20Cantor&volume=24&publication_year=2006&pages=4731-4737&pmid=16966686&doi=10.1200/JCO.2006.06.1101&)

26. Davidson JD, Ma L, Flagella M, Geeganage S, Gelbert LM, Slapak CA. An increase in the expression of ribonucleotide reductase large subunit 1 is associated with gemcitabine resistance in non-small cell lung cancer cell lines. Cancer Res. 2004;64:3761–3766. doi: 10.1158/0008-5472.CAN-03-3363.  [DOI](https://doi.org/10.1158/0008-5472.CAN-03-3363) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15172981/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=An%20increase%20in%20the%20expression%20of%20ribonucleotide%20reductase%20large%20subunit%201%20is%20associated%20with%20gemcitabine%20resistance%20in%20non-small%20cell%20lung%20cancer%20cell%20lines&author=JD%20Davidson&author=L%20Ma&author=M%20Flagella&author=S%20Geeganage&author=LM%20Gelbert&volume=64&publication_year=2004&pages=3761-3766&pmid=15172981&doi=10.1158/0008-5472.CAN-03-3363&)

27. Farrell JJ, Elsaleh H, Garcia M, Lai R, Ammar A, Regine WF, et al. Human equilibrative nucleoside transporter 1 levels predict response to gemcitabine in patients with pancreatic cancer. Gastroenterology. 2009;136:187–195. doi: 10.1053/j.gastro.2008.09.067.  [DOI](https://doi.org/10.1053/j.gastro.2008.09.067) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18992248/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastroenterology&title=Human%20equilibrative%20nucleoside%20transporter%201%20levels%20predict%20response%20to%20gemcitabine%20in%20patients%20with%20pancreatic%20cancer&author=JJ%20Farrell&author=H%20Elsaleh&author=M%20Garcia&author=R%20Lai&author=A%20Ammar&volume=136&publication_year=2009&pages=187-195&pmid=18992248&doi=10.1053/j.gastro.2008.09.067&)

28. Gilbert JA, Salavaggione OE, Ji Y, Pelleymounter LL, Eckloff BW, Wieben ED, et al. Gemcitabine pharmacogenomics: cytidine deaminase and deoxycytidylate deaminase gene resequencing and functional genomics. Clin Cancer Res. 2006;12:1794–1803. doi: 10.1158/1078-0432.CCR-05-1969.  [DOI](https://doi.org/10.1158/1078-0432.CCR-05-1969) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16551864/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=Gemcitabine%20pharmacogenomics:%20cytidine%20deaminase%20and%20deoxycytidylate%20deaminase%20gene%20resequencing%20and%20functional%20genomics&author=JA%20Gilbert&author=OE%20Salavaggione&author=Y%20Ji&author=LL%20Pelleymounter&author=BW%20Eckloff&volume=12&publication_year=2006&pages=1794-1803&pmid=16551864&doi=10.1158/1078-0432.CCR-05-1969&)

29. Fitzgerald SM, Goyal RK, Osborne WR, Roy JD, Wilson JW, Ferrell RE. Identification of functional single nucleotide polymorphism haplotypes in the cytidine deaminase promoter. Hum Genet. 2006;119:276–283. doi: 10.1007/s00439-006-0142-0.  [DOI](https://doi.org/10.1007/s00439-006-0142-0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16446974/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Genet&title=Identification%20of%20functional%20single%20nucleotide%20polymorphism%20haplotypes%20in%20the%20cytidine%20deaminase%20promoter&author=SM%20Fitzgerald&author=RK%20Goyal&author=WR%20Osborne&author=JD%20Roy&author=JW%20Wilson&volume=119&publication_year=2006&pages=276-283&pmid=16446974&doi=10.1007/s00439-006-0142-0&)

30. Kwon WS, Rha SY, Choi YH, Lee JO, Park KH, Jung JJ, et al. Ribonucleotide reductase M1 (RRM1) 2464G>A polymorphism shows an association with gemcitabine chemosensitivity in cancer cell lines. Pharmacogenet Genomics. 2006;16:429–438. doi: 10.1097/01.fpc.0000204999.29924.da.  [DOI](https://doi.org/10.1097/01.fpc.0000204999.29924.da) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16708051/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Ribonucleotide%20reductase%20M1%20(RRM1)%202464G>A%20polymorphism%20shows%20an%20association%20with%20gemcitabine%20chemosensitivity%20in%20cancer%20cell%20lines&author=WS%20Kwon&author=SY%20Rha&author=YH%20Choi&author=JO%20Lee&author=KH%20Park&volume=16&publication_year=2006&pages=429-438&pmid=16708051&doi=10.1097/01.fpc.0000204999.29924.da&)

31. Yonemori K, Ueno H, Okusaka T, Yamamoto N, Ikeda M, Saijo N, et al. Severe drug toxicity associated with a single-nucleotide polymorphism of the cytidine deaminase gene in a Japanese cancer patient treated with gemcitabine plus cisplatin. Clin Cancer Res. 2005;11:2620–2624. doi: 10.1158/1078-0432.CCR-04-1497.  [DOI](https://doi.org/10.1158/1078-0432.CCR-04-1497) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15814642/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=Severe%20drug%20toxicity%20associated%20with%20a%20single-nucleotide%20polymorphism%20of%20the%20cytidine%20deaminase%20gene%20in%20a%20Japanese%20cancer%20patient%20treated%20with%20gemcitabine%20plus%20cisplatin&author=K%20Yonemori&author=H%20Ueno&author=T%20Okusaka&author=N%20Yamamoto&author=M%20Ikeda&volume=11&publication_year=2005&pages=2620-2624&pmid=15814642&doi=10.1158/1078-0432.CCR-04-1497&)

32. Sugiyama E, Kaniwa N, Kim SR, Kikura-Hanajiri R, Hasegawa R, Maekawa K, et al. Pharmacokinetics of gemcitabine in Japanese cancer patients: the impact of a cytidine deaminase polymorphism. J Clin Oncol. 2007;25:32–42. doi: 10.1200/JCO.2006.06.7405.  [DOI](https://doi.org/10.1200/JCO.2006.06.7405) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17194903/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Pharmacokinetics%20of%20gemcitabine%20in%20Japanese%20cancer%20patients:%20the%20impact%20of%20a%20cytidine%20deaminase%20polymorphism&author=E%20Sugiyama&author=N%20Kaniwa&author=SR%20Kim&author=R%20Kikura-Hanajiri&author=R%20Hasegawa&volume=25&publication_year=2007&pages=32-42&pmid=17194903&doi=10.1200/JCO.2006.06.7405&)

33. Rha SY, Jeung HC, Choi YH, Yang WI, Yoo JH, Kim BS, et al. An association between RRM1 haplotype and gemcitabine-induced neutropenia in breast cancer patients. Oncologist. 2007;12:622–630. doi: 10.1634/theoncologist.12-6-622.  [DOI](https://doi.org/10.1634/theoncologist.12-6-622) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17602053/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncologist&title=An%20association%20between%20RRM1%20haplotype%20and%20gemcitabine-induced%20neutropenia%20in%20breast%20cancer%20patients&author=SY%20Rha&author=HC%20Jeung&author=YH%20Choi&author=WI%20Yang&author=JH%20Yoo&volume=12&publication_year=2007&pages=622-630&pmid=17602053&doi=10.1634/theoncologist.12-6-622&)

34. Okazaki T, Javle M, Tanaka M, Abbruzzese JL, Li D. Single Nucleotide Polymorphisms of Gemcitabine Metabolic Genes and Pancreatic Cancer Survival and Drug Toxicity. Clin Cancer Res. doi: 10.1158/1078-0432.CCR-09-1555. in press.  [DOI](https://doi.org/10.1158/1078-0432.CCR-09-1555) | [PMC free article](/articles/PMC2802655/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20028759/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=Single%20Nucleotide%20Polymorphisms%20of%20Gemcitabine%20Metabolic%20Genes%20and%20Pancreatic%20Cancer%20Survival%20and%20Drug%20Toxicity&author=T%20Okazaki&author=M%20Javle&author=M%20Tanaka&author=JL%20Abbruzzese&author=D%20Li&pmid=20028759&doi=10.1158/1078-0432.CCR-09-1555&)

35. Tamm EP, Loyer EM, Faria S, Raut CP, Evans DB, Wolff RA, et al. Staging of pancreatic cancer with multidetector CT in the setting of preoperative chemoradiation therapy. Abdom Imaging. 2006;31:568–574. doi: 10.1007/s00261-005-0194-y.  [DOI](https://doi.org/10.1007/s00261-005-0194-y) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16465578/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Abdom%20Imaging&title=Staging%20of%20pancreatic%20cancer%20with%20multidetector%20CT%20in%20the%20setting%20of%20preoperative%20chemoradiation%20therapy&author=EP%20Tamm&author=EM%20Loyer&author=S%20Faria&author=CP%20Raut&author=DB%20Evans&volume=31&publication_year=2006&pages=568-574&pmid=16465578&doi=10.1007/s00261-005-0194-y&)

36. Wacholder S, Chanock S, Garcia-Closas M, El Ghormli L, Rothman N. Assessing the probability that a positive report is false: an approach for molecular epidemiology studies. J Natl Cancer Inst. 2004;96:434–442. doi: 10.1093/jnci/djh075.  [DOI](https://doi.org/10.1093/jnci/djh075) | [PMC free article](/articles/PMC7713993/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15026468/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Natl%20Cancer%20Inst&title=Assessing%20the%20probability%20that%20a%20positive%20report%20is%20false:%20an%20approach%20for%20molecular%20epidemiology%20studies&author=S%20Wacholder&author=S%20Chanock&author=M%20Garcia-Closas&author=L%20El%20Ghormli&author=N%20Rothman&volume=96&publication_year=2004&pages=434-442&pmid=15026468&doi=10.1093/jnci/djh075&)

37. Yue L, Saikawa Y, Ota K, Tanaka M, Nishimura R, Uehara T, et al. A functional single-nucleotide polymorphism in the human cytidine deaminase gene contributing to ara-C sensitivity. Pharmacogenetics. 2003;13:29–38. doi: 10.1097/00008571-200301000-00005.  [DOI](https://doi.org/10.1097/00008571-200301000-00005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12544510/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=A%20functional%20single-nucleotide%20polymorphism%20in%20the%20human%20cytidine%20deaminase%20gene%20contributing%20to%20ara-C%20sensitivity&author=L%20Yue&author=Y%20Saikawa&author=K%20Ota&author=M%20Tanaka&author=R%20Nishimura&volume=13&publication_year=2003&pages=29-38&pmid=12544510&doi=10.1097/00008571-200301000-00005&)

38. Schroder JK, Kirch C, Seeber S, Schutte J. Structural and functional analysis of the cytidine deaminase gene in patients with acute myeloid leukaemia. Br J Haematol. 1998;103:1096–1103. doi: 10.1046/j.1365-2141.1998.01084.x.  [DOI](https://doi.org/10.1046/j.1365-2141.1998.01084.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9886326/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Haematol&title=Structural%20and%20functional%20analysis%20of%20the%20cytidine%20deaminase%20gene%20in%20patients%20with%20acute%20myeloid%20leukaemia&author=JK%20Schroder&author=C%20Kirch&author=S%20Seeber&author=J%20Schutte&volume=103&publication_year=1998&pages=1096-1103&pmid=9886326&doi=10.1046/j.1365-2141.1998.01084.x&)

39. Kirch HC, Schroder J, Hoppe H, Esche H, Seeber S, Schutte J. Recombinant gene products of two natural variants of the human cytidine deaminase gene confer different deamination rates of cytarabine in vitro. Exp Hematol. 1998;26:421–425.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/9590659/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Exp%20Hematol&title=Recombinant%20gene%20products%20of%20two%20natural%20variants%20of%20the%20human%20cytidine%20deaminase%20gene%20confer%20different%20deamination%20rates%20of%20cytarabine%20in%20vitro&author=HC%20Kirch&author=J%20Schroder&author=H%20Hoppe&author=H%20Esche&author=S%20Seeber&volume=26&publication_year=1998&pages=421-425&pmid=9590659&)

40. Giovannetti E, Laan AC, Vasile E, Tibaldi C, Nannizzi S, Ricciardi S, et al. Correlation between cytidine deaminase genotype and gemcitabine deamination in blood samples. Nucleosides Nucleotides Nucleic Acids. 2008;27:720–725. doi: 10.1080/15257770802145447.  [DOI](https://doi.org/10.1080/15257770802145447) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18600531/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleosides%20Nucleotides%20Nucleic%20Acids&title=Correlation%20between%20cytidine%20deaminase%20genotype%20and%20gemcitabine%20deamination%20in%20blood%20samples&author=E%20Giovannetti&author=AC%20Laan&author=E%20Vasile&author=C%20Tibaldi&author=S%20Nannizzi&volume=27&publication_year=2008&pages=720-725&pmid=18600531&doi=10.1080/15257770802145447&)

41. Sebastiani V, Ricci F, Rubio-Viqueira B, Kulesza P, Yeo CJ, Hidalgo M, et al. Immunohistochemical and genetic evaluation of deoxycytidine kinase in pancreatic cancer: relationship to molecular mechanisms of gemcitabine resistance and survival. Clin Cancer Res. 2006;12:2492–2497. doi: 10.1158/1078-0432.CCR-05-2655.  [DOI](https://doi.org/10.1158/1078-0432.CCR-05-2655) | [PMC free article](/articles/PMC3619210/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16638857/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=Immunohistochemical%20and%20genetic%20evaluation%20of%20deoxycytidine%20kinase%20in%20pancreatic%20cancer:%20relationship%20to%20molecular%20mechanisms%20of%20gemcitabine%20resistance%20and%20survival&author=V%20Sebastiani&author=F%20Ricci&author=B%20Rubio-Viqueira&author=P%20Kulesza&author=CJ%20Yeo&volume=12&publication_year=2006&pages=2492-2497&pmid=16638857&doi=10.1158/1078-0432.CCR-05-2655&)

42. Kimchi-Sarfaty C, Oh JM, Kim IW, Sauna ZE, Calcagno AM, Ambudkar SV, et al. A “silent” polymorphism in the MDR1 gene changes substrate specificity. Science. 2007;315:525–528. doi: 10.1126/science.1135308.  [DOI](https://doi.org/10.1126/science.1135308) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17185560/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=A%20%E2%80%9Csilent%E2%80%9D%20polymorphism%20in%20the%20MDR1%20gene%20changes%20substrate%20specificity&author=C%20Kimchi-Sarfaty&author=JM%20Oh&author=IW%20Kim&author=ZE%20Sauna&author=AM%20Calcagno&volume=315&publication_year=2007&pages=525-528&pmid=17185560&doi=10.1126/science.1135308&)

43. Kim SR, Saito Y, Maekawa K, Sugiyama E, Kaniwa N, Ueno H, et al. Thirty novel genetic variations in the SLC29A1 gene encoding human equilibrative nucleoside transporter 1 (hENT1) Drug Metab Pharmacokinet. 2006;21:248–256. doi: 10.2133/dmpk.21.248.  [DOI](https://doi.org/10.2133/dmpk.21.248) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16858130/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Pharmacokinet&title=Thirty%20novel%20genetic%20variations%20in%20the%20SLC29A1%20gene%20encoding%20human%20equilibrative%20nucleoside%20transporter%201%20(hENT1)&author=SR%20Kim&author=Y%20Saito&author=K%20Maekawa&author=E%20Sugiyama&author=N%20Kaniwa&volume=21&publication_year=2006&pages=248-256&pmid=16858130&doi=10.2133/dmpk.21.248&)

44. Osato DH, Huang CC, Kawamoto M, Johns SJ, Stryke D, Wang J, et al. Functional characterization in yeast of genetic variants in the human equilibrative nucleoside transporter, ENT1. Pharmacogenetics. 2003;13:297–301. doi: 10.1097/00008571-200305000-00010.  [DOI](https://doi.org/10.1097/00008571-200305000-00010) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12724623/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Functional%20characterization%20in%20yeast%20of%20genetic%20variants%20in%20the%20human%20equilibrative%20nucleoside%20transporter,%20ENT1&author=DH%20Osato&author=CC%20Huang&author=M%20Kawamoto&author=SJ%20Johns&author=D%20Stryke&volume=13&publication_year=2003&pages=297-301&pmid=12724623&doi=10.1097/00008571-200305000-00010&)

45. Myers SN, Goyal RK, Roy JD, Fairfull LD, Wilson JW, Ferrell RE. Functional single nucleotide polymorphism haplotypes in the human equilibrative nucleoside transporter 1. Pharmacogenet Genomics. 2006;16:315–320. doi: 10.1097/01.fpc.0000189804.41962.15.  [DOI](https://doi.org/10.1097/01.fpc.0000189804.41962.15) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16609362/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Functional%20single%20nucleotide%20polymorphism%20haplotypes%20in%20the%20human%20equilibrative%20nucleoside%20transporter%201&author=SN%20Myers&author=RK%20Goyal&author=JD%20Roy&author=LD%20Fairfull&author=JW%20Wilson&volume=16&publication_year=2006&pages=315-320&pmid=16609362&doi=10.1097/01.fpc.0000189804.41962.15&)
