# Increased omeprazole metabolism in carriers of the CYP2C19*17 allele; a pharmacokinetic study in healthy volunteers

## Metadata
**Authors:** R Michael Baldwin, Staffan Ohlsson, Rasmus Steen Pedersen, Jessica Mwinyi, Magnus Ingelman-Sundberg, Erik Eliasson, Leif Bertilsson
**Journal:** British Journal of Clinical Pharmacology
**Date:** 2008 Feb 20
**DOI:** [10.1111/j.1365-2125.2008.03104.x](https://doi.org/10.1111/j.1365-2125.2008.03104.x)
**PMID:** 18294333
**PMCID:** PMC2432489
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2432489/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC2432489/pdf/bcp0065-0767.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC2432489/pdf/bcp0065-0767.pdf)

## Abstract

**AIMS:** 
To investigate the influence of the CYP2C19*17 allele on the pharmacokinetics of omeprazole, a commonly used CYP2C19 probe drug, in healthy volunteers.

**METHODS:** 
In a single-dose pharmacokinetic study, 17 healthy White volunteers genotyped as either CYP2C19*17/*17 or CYP2C19*1/*1 received an oral dose of 40 mg of omeprazole. Plasma was sampled for up to 10 h postdose, followed by quantification of omeprazole, 5-hydroxy omeprazole and omeprazole sulphone by high-performance liquid chromatography.

**RESULTS:** 
The mean omeprazole AUC∞ of 1973 h nmol l−1 in CYP2C19*17/*17 subjects was 2.1-fold lower [95% confidence interval (CI) 1.1, 3.3] than in CYP2C19*1/*1 subjects (4151 h nmol l−1, P = 0.04). A similar trend was observed for the sulphone metabolite with the CYP2C19*17/*17 group having a mean AUC∞ of 1083 h nmol l−1, 3.1-fold lower (95% CI 1.2, 5.5) than the CYP2C19*1/*1 group (3343 h nmol l−1, P = 0.03). A pronounced correlation (r2 = 0.95, P < 0.0001) was seen in the intraindividual omeprazole AUC∞ and omeprazole sulphone AUC∞ values.

**CONCLUSIONS:** 
The pharmacokinetics of omeprazole and omeprazole sulphone differ significantly between homozygous CYP2C19*17 and CYP2C19*1 subjects. For clinically important drugs that are metabolized predominantly by CYP2C19, the CYP2C19*17 allele might be associated with subtherapeutic drug exposure.

**WHAT IS ALREADY KNOWN ABOUT THIS SUBJECT:** 
The only existing study of CYP2C19*17-associated alterations in drug pharmacokinetics was retrospective and compared probe drug metabolic ratios.

The CYP2C19*17 allele had been associated with a two- and fourfold decrease in omeprazole and S/R-mephenytoin metabolic ratios.

**WHAT THIS STUDY ADDS:** 
This study characterized the single-dose pharmacokinetics of omeprazole, along with the 5-hydroxy and sulphone metabolites, in CYP2C19*17/*17 and CYP2C19*1/*1 subjects.

The observed differences in omeprazole AUC∞ suggest that the CYP2C19*17 allele is an important explanatory factor behind individual cases of therapeutic failure.

Keywords: 5-hydroxyomeprazole, depression, human, omeprazole sulphone, pantoprazole, pharmacogenetic, pharmacokinetics, SSRIs, ulcer

### AIMS

To investigate the influence of the *CYP2C19***17* allele on the pharmacokinetics of omeprazole, a commonly used CYP2C19 probe drug, in healthy volunteers.

### METHODS

In a single-dose pharmacokinetic study, 17 healthy White volunteers genotyped as either *CYP2C19***17/***17* or *CYP2C19***1/***1* received an oral dose of 40 mg of omeprazole. Plasma was sampled for up to 10 h postdose, followed by quantification of omeprazole, 5-hydroxy omeprazole and omeprazole sulphone by high-performance liquid chromatography.

### RESULTS

The mean omeprazole AUC_∞_ of 1973 h nmol l^−1^ in *CYP2C19***17/***17* subjects was 2.1-fold lower [95% confidence interval (CI) 1.1, 3.3] than in *CYP2C19***1/***1* subjects (4151 h nmol l^−1^, *P* = 0.04). A similar trend was observed for the sulphone metabolite with the *CYP2C19***17/***17* group having a mean AUC_∞_ of 1083 h nmol l^−1^, 3.1-fold lower (95% CI 1.2, 5.5) than the *CYP2C19***1/***1* group (3343 h nmol l^−1^, *P* = 0.03). A pronounced correlation (*r*^2^ = 0.95, *P* < 0.0001) was seen in the intraindividual omeprazole AUC_∞_ and omeprazole sulphone AUC_∞_ values.

### CONCLUSIONS

The pharmacokinetics of omeprazole and omeprazole sulphone differ significantly between homozygous *CYP2C19***17* and *CYP2C19***1* subjects. For clinically important drugs that are metabolized predominantly by CYP2C19, the *CYP2C19***17* allele might be associated with subtherapeutic drug exposure.

#### WHAT IS ALREADY KNOWN ABOUT THIS SUBJECT

#### WHAT THIS STUDY ADDS

#### WHAT IS ALREADY KNOWN ABOUT THIS SUBJECT

#### WHAT IS ALREADY KNOWN ABOUT THIS SUBJECT

#### WHAT THIS STUDY ADDS

#### WHAT THIS STUDY ADDS

## Introduction

Cytochrome P450 2C19 (CYP2C19) is an important enzyme in the metabolism of many clinically important drugs, including proton pump inhibitors (PPIs), antidepressants such as citalopram, amitriptyline and clomipramine, along with carisoprodol, diazepam, flunitrazepam, proguanil, phenytoin and mephenytoin [[1](#b1)]. Several genetic polymorphisms within the *CYP2C19* gene have been identified (cf. [http://www.cypalleles.ki.se/](http://www.cypalleles.ki.se/)) and the clinical implications of these polymorphisms have been the focus of much investigation. The first and most extensively described CYP2C19 polymorphic variants cause the poor metabolizer phenotype (PM). Of the 19 variant *CYP2C19* alleles, *CYP219***2* and *CYP2C19***3,* which encode for nonfunctional proteins, are responsible for the vast majority of PM phenotypes. The frequency of the PM phenotype varies significantly between populations, ranging from 2 to 5% in White and Black populations to 13–23% in Asian populations [[1](#b1)]. The impact of these polymorphisms on the efficacy of several commonly used drug classes is well established. For example, studies have demonstrated dramatic *CYP2C19*-associated differences in diazepam disposition, with systemic drug exposure (AUC) varying more than sixfold between individuals [[2](#b2), [3](#b3)]. A similar trend has been observed with certain tricyclic antidepressants and serotonin reuptake inhibitors such as citalopram with a near 50% reduction of systemic clearance observed in PM subjects [[4](#b4), [5](#b5)]. For the PPIs omeprazole, lansoprazole and pantoprazole, *CYP2C19* genotype is responsible for considerable interindividual differences in pharmacokinetics, with AUC values varying from five- to 12-fold [[6](#b6), [7](#b7)]. Furthermore, a prospective study of Japanese patients administered the standard 20-mg omeprazole dose in combination with amoxicillin for the treatment of *Helicobacter pylori* and peptic ulcer has demonstrated a clear association between genotype and treatment success. Cure rates for *H. pylori* infection were 29% in extensive metabolizer (EM) subjects compared with 100% in PM subjects, with a similar trend in the rates of ulcer healing [[8](#b8), [9](#b9)].

In contrast to the defective genes, we were able recently to identify a novel *CYP2C19* gene variant associated with a more rapid metabolism phenotype [[10](#b10)]. Two single nucleotide polymorphisms (SNPs) in the 5′-flanking region (−3402C→T and −806C→T) of the *CYP2C19* gene, which constitute the *CYP2C19***17* allele (cf. [http://www.cypalleles.ki.se/cyp2c19.htm](http://www.cypalleles.ki.se/cyp2c19.htm)), have been strongly linked with increased CYP2C19-dependent drug metabolism. The *CYP2C19***17* allele has a frequency of 18% in both Swedes and Ethiopians, but only 4% in Chinese populations [[10](#b10)]. Phenotypic analysis of Ethiopian populations using the CYP2C19 probe drug mephenytoin has revealed a fourfold difference in median S/R-mephenytoin ratio between subjects homozygous for *CYP2C19***1* and *CYP2C19***1*7. A similar twofold difference in metabolic ratio (MR) has been observed in Swedish populations using omeprazole as a probe drug [[10](#b10)]. Functional genomic studies indicated that the −806C→T mutation caused increased nuclear factor binding and increased reporter gene transcription in mouse transfection studies. These observations suggest that the pharmacokinetics and clinical efficacy of CYP2C19 substrate drugs could be significantly affected in individuals possessing the *CYP2C19***17* allele. The aim of the present study was to determine the pharmacokinetic differences of omeprazole and its two major metabolites 5-hydroxyomeprazole and omeprazole sulphone in subjects homozygous for the *CYP2C19***17* and *CYP2C19***1* alleles.

## Methods

### Design

The study was registered in the European Clinical Trials Database (EudraCT No2005-004717-15). The protocol was approved by the Regional Ethics Committee and the Swedish Medical Products Agency. The study was conducted in accordance with Good Clinical Practices and the Helsinki declaration. Previously genotyped volunteers were recruited from a database of former study participants at the Human Pharmacology Unit in Clinical Pharmacology, Karolinska University Hospital, Huddinge. All of the *CYP2C19***17/***17* subjects in the database (ten) were asked to participate and *CYP2C19***1/***1* subjects were recruited in a random fashion blinded to previous phenotypic data. Sample size calculations were based on the primary outcome represented by differences in AUC between *CYP2C19***1/***1* and *CYP2C19***17/***17* using our prior estimate of a 40% difference in omeprazole AUC between groups [[10](#b10)]. It was estimated that a true difference of 40% could be detected, given a one-sided level of significance (α) of 0.05 and a power (β) of 80%, using six *CYP2C19***17/***17* and 12 *CYP2C19***1/***1* subjects. Twelve volunteers previously genotyped as *CYP2C19***1/***1* were included, but only five *CYP2C19***17/***17* volunteers could participate. The study was conducted as an open label, one-phase trial where the volunteers were given a single oral dose of 40 mg omeprazole as two 20-mg tablets (Losec Mups®, AstraZeneca, Sweden). Peripheral blood samples were drawn from an i.v. cannula in a forearm vein into 10 ml sodium heparin collection tubes prior to drug administration and at the following times after omeprazole ingestion: 0.5, 1, 1.5, 2, 3, 4, 6, 8 and 10 h. After centrifugation for 10 min at 1500 *g*, plasma was separated and stored at −20°C until analysis.

### Subjects

After oral and written explanation of the study, subjects gave written informed consent. Prior to starting the study, blood was obtained for clinical chemistry and genotype confirmation along with urine for drug screening and urinalysis. No medications other than paracetamol were allowed within 2 weeks before omeprazole administration and all drugs were prohibited within 24 h of omeprazole administration. Additional exclusion criteria included the use of Saint John's wort within 2 months, foods containing poppy seeds within 3 days or foods and drinks containing grapefruit within 2 days of omeprazole administration. Female volunteers were required to abstain from hormonal contraceptives for a minimum of 3 weeks. Subjects were excluded based on clinically significant abnormal ECG, blood chemistry or urine analysis, urine drug detection, or a positive pregnancy test. The subjects' sex, age, weight and smoking status are presented in [Table 1](#tbl1). The volunteers were fasted from 8 h before to 2 h after omeprazole administration.

### Table 1.

|   | CYP2C19*17/*17 | CYP2C19*1/*1 |
| --- | --- | --- |
| Number of subjects | 5 | 11 |
| Sex (male/female) | 2/3 | 2/9 |
| Age (years) | 37 (24–45) | 33 (21–60) |
| Weight (kg) | 74 (63–107) | 69 (60–109) |
| Number of smokers | 0 | 1 |

Table 1 Caption: Demographic characteristics of the subjects

### Genotyping

Subjects were genotyped for CYP2C19*17 using a novel nested polymerase chain reaction (PCR) approach to introduce an allele-specific restriction site for restriction fragment length polymorphism analysis. Briefly, DNA was obtained from peripheral blood using a commercially available kit (DNeasy; Qiagen, West Sussex, UK). A 473 base pair (bp) fragment containing the *CYP2C19***17 –*806C→T mutation site was PCR-amplified using: 12.5 ng genomic DNA, 0.6 U Taq DNA polymerase (ABgene Epsom, UK), 2.5 μl 10× Buffer IV, 2 mM MgCl_2_, 0.2 mM dNTPs and 0.4 μM 2C19-1 forward primer (5′GCCCTTAGCACCAAATTCTC) and 2C19-1 reverse primer (5′ATTTAACCCCCTAAAAAAACACG). The thermoprofile consisted of an initial denaturation step at 95°C for 1 min followed by 35 cycles of 95°C for 30 s, 52°C for 30 s and 72°C for 30 s, followed by 72°C for 7 min. A 143-bp fragment with an allele-specific *Nsi*I restriction site was created with the following: 0.05 μl of the initial PCR amplification, 0.6 U *Taq* DNA polymerase (ABgene Epsom, UK), 2.5 μl 10× Buffer IV, 2 mM MgCl_2_, 0.2 mM dNTPs and 0.25 μM 2C19-2 forward primer (5′AAATTTGTGTCTTCTGT TCTCAATG) and 2C19-2 reverse primer (5′AGACCCTGG GAGAACAGGAC). The thermoprofile consisted of 95°C for 1 min followed by 25 cycles of 95°C for 30 s, 51°C for 30 s and 72°C for 30 s, followed by 72°C for 7 min. Restriction digestion of the resultant PCR product was performed at 37°C for 8 h and contained the following: 15 μl PCR reaction, 2.5 μl 10× NEB reaction buffer 3 and 0.8 μl *Nsi*I (New England BioLabs, Ipswich, MA, USA). Agarose gel electrophoresis revealed fragments of either 143 bp or 116 bp for the *CYP219***17* and *CYP2C19***1* alleles, respectively. Subjects were genotyped for the *CYP2C19***2* allele using a TaqMan Allelic Discrimination Assay (Applied Biosystems, Foster City, CA, USA) on an ABI PRISM 7700 according to the manufacturer's recommendations.

### Chemicals

Omeprazole, 5-hydroxyomeprazole, omeprazole sulphone, and internal standard H 259/36 were supplied by AstraZeneca (Mölndal, Sweden). All stock solutions were prepared in 5% methanol in sodium carbonate buffer 0.1 M, pH 9.3 and stored at −20°C. Unless otherwise stated, all other reagents were purchased from commercial vendors and were of reagent/analytical grade.

### Sample analysis

Omeprazole and the two metabolites, 5-hydroxyomeprazole and omeprazole sulphone, were quantified using a reversed-phase high-performance liquid chromatography (R-HPLC) method based on previously published methodologies [[11](#b11)–[13](#b13)]. Duplicate 100-μl plasma samples were each combined with 800 μl methylene chloride:acetonitrile (9 : 1, v/v) and 21 μl 250 mM disodium phosphate buffer (pH 9.7) containing 10 nmol of internal standard. For samples near the limit of quantification, 400 μl of plasma was combined with 1.4 ml methylene chloride:acetonitrile and 21 μl of internal standard. All samples were extracted for 10 min using a vortex mixer followed by centrifugation (2000 *g*, 10 min) and aspiration of the plasma/aqueous phase. The organic phase was transferred to a new tube and evaporated to dryness at 60°C. Samples were reconstituted in 30 μl of methanol followed by 100 μl of 10 mM disodium phosphate buffer (pH 9.3). To minimize potential particulate carryover to R-HPLC analysis, the reconstituted samples were filtered using a 0.2-μm microcentrifuge spin column (Ultrafree-MC; Millipore, Nödinge, Sweden) and 77 μl of the resultant filtrate injected into the R-HPLC system.

Chromatography was performed using a Varian Prostar R-HPLC system, consisting of a Model 240 solvent delivery module, Model 310 UV-Vis detector and a Model 410 autosampler, combined with a LiChrospher 60 RP-select B 125 × 4 mm, 5 μm particle, column and an identical 4 × 4 mm precolumn (Merck, Whitehouse Station, NJ, USA). The mobile phases were 62.5 mM ammonium acetate pH 7.0 (A) and acetonitrile (B). The solvent gradient was as follows: initially 80% A:20% B with a linear ramp to 55% A:45% B from 2 min to 18 min, a linear return to initial condition from 23 min to 27 min followed by a 5-min equilibration before injection of the next sample. The flow rate was 1 ml min^−1^. Absorbance was monitored at 302 nm and peak areas determined using the Varian Star version 5.51 software package. Retention times were 7, 11, 12 and 13 min for 5-hydroxyomeprazole, internal standard, omeprazole and the sulphone metabolite, respectively. Standard curves were generated from spiking drug-free plasma with omeprazole and the two metabolites in the concentration range of 10–2500 nmol l^−1^ combined with internal standard followed by extraction and quantification as described above. The limit of quantification for all three analytes was 10 nmol l^−1^. The intraday and interday coefficients of variation for omeprazole and the two major metabolites were <10% and <15%, respectively.

### Pharmacokinetic and statistical analysis

Pharmacokinetic parameters were determined using noncompartmental analysis and WinNonlin version 5.1 (Pharsight, Mountain View, CA, USA). The area under the concentration–time curve to infinity was calculated using the linear trapezoid rule and extrapolation to infinity using the log-linear terminal elimination phase (λ_Ζ_). Total body clearance was calculated as CL = molar dose/AUC_∞_ without normalization for the fraction of the administered dose reaching the central compartment (bioavailability). Data are reported as mean and 95% confidence interval (95% CI). Statistical calculations were performed using Graphpad Prism and Microsoft Excel. The pharmacokinetic parameters for *CYP2C19***17/***17* and *CYP2C19***1/***1* subjects were compared using an unpaired two-tailed heteroscedastic t-test.

## Results

### Omeprazole kinetics in healthy subjects

All subjects successfully completed the study in accordance with the protocol, and demographic information is shown in [Table 1](#tbl1). No statistically significant differences in age, weight or smoking status were observed between genotype groups (*P* > 0.5). One subject displayed extremely aberrant and drastically reduced absorption kinetics (222 nM *C*_max_*vs.* group mean 2109 nM *C*_max_) with first detectable drug observed at 6 h post administration (240 min *T*_lag_*vs.* group mean 28 min *T*_lag_) and was therefore excluded from the study.

The pharmacokinetic parameters for each study group are shown in [Table 2](#tbl2). The high sensitivity and resolution of the analytical method combined with the 10-h duration of blood sampling resulted in a very limited extrapolated area contribution to the area under the plasma–concentration time curve (AUC_∞_). The mean extrapolated area for omeprazole, 5-hydroxyomeprazole and omeprazole sulphone were 2%, 3% and 19% of the total AUC_∞_, respectively. The mean omeprazole AUC_∞_ of 1973 h nmol l^−1^ in *CYP2C19***17/***17* subjects was 2.1-fold lower (95% CI 1.1, 3.3) than in *CYP2C19***1/***1* subjects (4151 h nmol l^−1^, *P* = 0.04), as shown in [Figure 1](#fig01). A similar trend was observed with the sulphone metabolite, with the *CYP2C19***17/***17* group having a mean AUC_∞_ of 1083 h nmol l^−1^, 3.1-fold lower (95% CI 1.2, 5.5) than the *CYP2C19***1/***1* group (3343 h nmol l^−1^, *P* = 0.03). There was no statistical difference in AUC_∞_ values for the 5-hydroxy metabolite between the *CYP2C19***17/***17* and *CYP2C19***1/***1* groups (2989 *vs.* 3359 h nmol l^−1^, *P* = 0.33). Comparison of omeprazole AUC_∞_/5-hydroxyomeprazole AUC_∞_ ratios also demonstrated a statistically significant difference between *CYP2C19***17/***17* and *CYP2C19***1/***1* groups (0.66 *vs.* 1.2, *P* = 0.04). Comparison of intraindividual omeprazole and metabolite AUC_∞_ values yielded an excellent correlation with omeprazole sulphone (*r*^2^ = 0.95), but a significantly weaker correlation with 5-hydroxyomeprazole (*r*^2^ = 0.51) ([Figure 2](#fig02)).

### Table 2.

|   | Units | CYP2C19*1/*1 Mean (95% CI) | CYP2C19*17/*17 Mean (95% CI) | P-value |
| --- | --- | --- | --- | --- |
| Omeprazole |  |  |  |  |
| AUC∞ | nmol h l−1 | 4151 (2084, 6218) | 1973 (1476, 2469) | 0.04* |
| CL | l h−1 | 48 (22, 75) | 61 (44, 77) | 0.37† |
| Tmax | h | 2.1 (1.2, 3.1) | 2.1 (1.1, 3.1) | 0.95 |
| Cmax | nmol l−1 | 2109 (1169, 3049) | 1447 (1069, 1825) | 0.16 |
| λΖt1/2 | h | 1.2 (0.9, 1.5) | 0.9 (0.6, 1.2) | 0.18 |
| 5-Hydroxyomeprazole |  |  |  |  |
| AUC∞ | nmol h l−1 | 3359 (2803, 3915) | 2989 (2273, 3706) | 0.33 |
| Tmax | h | 2.4 (1.2, 3.7) | 2.1 (1.1, 3.1) | 0.65 |
| Cmax | nmol l−1 | 1534 (1081, 1988) | 1527 (1234, 1819) | 0.97 |
| λΖt1/2 | h | 1.3 (0.9, 1.6) | 1.1 (1.0, 1.3) | 0.33 |
| Omeprazole sulphone |  |  |  |  |
| AUC∞ | nmol h l−1 | 3343 (1301, 5384) | 1083 (690, 1476) | 0.03* |
| Tmax | h | 2.6 (1.4, 3.8) | 2.2 (1.3, 3.1) | 0.54 |
| Cmax | nmol l−1 | 563 (362, 765) | 336 (239, 433) | 0.04* |
| λΖt1/2 | h | 3.3 (1.7, 4.8) | 1.9 (1.4, 2.4) | 0.09 |
| OME AUC∞/5-OH AUC∞ |   | 1.2 (0.70, 1.6) | 0.66 (0.54, 0.79) | 0.04* |

Table 2 Caption: Pharmacokinetic parameters of omeprazole, 5-hydroxyomeprazole and the sulphone metabolite after a single oral 40-mg dose of omeprazole to 16 healthy volunteers

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1e12/2432489/ff36c72989cd/bcp0065-0767-f1.jpg)

(A) Distribution of area under the concentration–time curve (AUC∞) values for omeprazole, 5-hydroxyomeprazole (5-hydroxy) and omeprazole sulphone (sulphone) in relation to CYP2C19*17/*17 (n = 5) and CYP2C19*1/*1 (n = 11) genotype. (B) Distribution of the ratio of area under the concentration–time curves of omeprazole (OME AUC∞) and 5-hydroxy omeprazole (5-OH AUC∞) in relation to CYP2C19*17/*17 (n = 5) and CYP2C19*1/*1 (n = 11) genotype. Bars represent mean values. Statistical analyses were performed with an unpaired two-tailed heteroscedastic t-test

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1e12/2432489/a66ab20d5732/bcp0065-0767-f2.jpg)

Correlation of intraindividual omeprazole AUC∞ and metabolite AUC∞ values in CYP2C19*17/*17 (,n = 5) and CYP2C19*1/*1 (•, n = 11) subjects. (A) Relationship between omeprazole AUC∞ and omeprazole sulphone (Sulphone) AUC∞. Linear regression yields r2 = 0.9540 (P < 0.0001). (B) Relationship between omeprazole AUC∞ and 5-hydroxy omeprazole (5-OH) AUC∞. Linear regression yields r2 = 0.5100

Omeprazole and omeprazole sulphone *T*_max_, *C*_max_ and λ_Ζ_*t*_1/2_-values demonstrate a consistent trend towards an increased rate of elimination in the *CYP2C19***17* group; however, only the omeprazole sulphone *C*_max_ reached the level of statistical significance ([Table 2](#tbl2)). Conversely, there were few if any genotype-associated differences in the pharmacokinetics of the 5-hydroxy metabolite.

## Discussion

Our results indicate that individuals homozygous for the *CYP2C19***17* allele experience significantly reduced systemic exposure to the CYP2C19 probe drug omeprazole. On average, subjects with the *CYP2C19***17/***17* genotype have a 2.1-fold reduction in mean omeprazole AUC_∞_*vs. CYP2C19***1/***1* individuals. To our knowledge, this is the first prospective investigation of the impact of the *CYP2C19***17* allele on drug pharmacokinetics.

This study is in excellent accord with previous *CYP2C19***17* phenotyping studies utilizing either single time point evaluations of circulating drug levels or urinary metabolite ratios [[10](#b10), [14](#b14)]. In a very recent study, Rudberg *et al.*[[14](#b14)] have investigated both the frequency and clinical impact of the *CYP2C19***17* allele on escitalopram serum levels in 166 psychiatric patients during a routine patient drug monitoring programme. Significantly, they observed a 42% reduction in mean escitalopram serum concentrations in *CYP2C19***17/***17* subjects compared with *CYP2C19***1/***1* subjects. They therefore postulated that homozygous *CYP2C19***17* carriers would require a 50% increase in escitalopram dose to attain drug exposures similar to homozygous *CYP2C19***1* carriers. In our initial investigation of the *CYP2C19***17* allele, omeprazole and mephenytoin metabolic ratio determinations led us also to postulate that homozygous carriers for this allele could potentially experience ≈ 40% decline in systemic drug exposure [[10](#b10)]. These prior reports, combined with the 52% (2.1-fold) reduction in omeprazole AUC_∞_ observed in the present study, demonstrate a consistent relationship between the *CYP2C19***17/***17* genotype and a significant reduction in circulating drugs levels for these three CYP2C19 substrate drugs.

The impact of the *CYP2C19***17* allele was less obvious in a report published by Kurzakski *et al.*, as the authors were unable to observe any association between the presence of the *CYP2C19***17* allele and a decline in the effectiveness of *H. pylori* eradication in patients treated for peptic ulcer using a triple therapy comprised of pantoprazole, amoxicillin and metronidazole [[15](#b15)]. It is noteworthy that the majority of studies demonstrating a clear association between increased *H. pylori* eradication and the PM phenotype have used clarithromycin instead of metronidazole [[16](#b16)]. Clarithromycin is an established substrate and mechanism-based inactivator of CYP3A4 and co-administration would therefore decrease CYP3A4-mediated PPI metabolism and potentiate any existing PPI dependencies on CYP2C19-mediated metabolism [[17](#b17)]. This is supported by studies demonstrating clarithromycin to increase significantly the differences in systemic exposure (AUC) to either omeprazole or lansoprazole observed in EM, intermediate metabolizer and PM subjects [[18](#b18), [19](#b19)]. In contrast, metronidazole is not a substrate for CYP3A4 and is instead believed to be a substrate for CYP2C9, making a similar additive effective with the PM phenotype unlikely [[20](#b20), [21](#b21)]. Taken together, these factors could provide an explanation as to why the effectiveness of triple therapy using pantoprazole, amoxicillin and metronidazole for the eradication *H. pylori* is not significantly altered in individuals possessing the *CYP219***17* allele.

The implications of these findings on the therapeutic outcome for a specific therapeutic must, however, be evaluated within the context of both the therapeutic window and the magnitude and exclusivity for CYP2C19-mediated metabolism. These specific issues are especially relevant to the existing challenges of patient dose titration in the area of antidepressant drug therapy. To begin to provide a potential framework for genotype-based dosing adjustments, Kirchheiner *et al.* have analysed the available pharmacogenetic data pertinent to antidepressant and antipsychotic therapy, making the tentative recommendation to dose adjust PMs to 60% and EMs to 110% of the standard dose for certain selective serotonin reuptake inhibitors or tricyclic antidepressants [[22](#b22)]. Perhaps even more poignant examples of critical dose titration involve the chemotherapeutics cyclophosphamide and thalidomide, which are activated by CYP2C19. Not surprisingly, the *CYP2C19***2* allele has been linked to a decrease in side-effects but also a decrease in response to cyclophosphamide treatment [[23](#b23)]. Similarly, response rates following thalidomide treatment for multiple myeloma were nearly 50% lower in carriers of two defective CYP2C19 alleles compared with individuals with at least one functional CYP2C19 allele [[24](#b24)]. By extension, it could be expected that carriers of the *CYP2C19***17* allele would experience the converse, generating increased levels of the reactive metabolites and having an increased likelihood of treatment-related side-effects. Future clinical studies correlating *CYP2C19***17* with the efficacy of CYP2C19 substrates such as escitalopram, cyclophosphamide, thalidomide and omeprazole would extend the existing relationships between genotype and pharmacokinetics and provide the critical rationale and justification for drug dose adjustments in the future.

Although not a primary study objective, characterization of omeprazole sulphone kinetics revealed a slightly larger statistical difference in sulphone AUC_∞_ values between *CYP2C19***17* and *CYP2C19***1* homozygous individuals than similar comparisons using omeprazole AUC_∞_ values ([Figure 1](#fig01)). This is not entirely unexpected and can be explained as the result of two additive processes. Omeprazole is metabolized by two competing enzymes, CYP2C19, producing the 5-hydroxy metabolite, and CYP3A4, generating the sulphone and 3-hydroxymetabolites [[25](#b25)]. Therefore, subjects with elevated CYP2C19 activity will generate more of the 5-hydroxy and less of the sulphone metabolites. Additionally, prior studies of secondary omeprazole metabolism have shown omeprazole sulphone metabolism to be primarily mediated via CYP2C19 to generate the hydroxysulphone, thereby compounding the observed reduction in sulphone AUC_∞_ in *CYP2C19***17/***17* individuals [[26](#b26), [27](#b27)]. Correlation of intraindividual omeprazole and omeprazole sulphone AUC_∞_ values revealed a very strong linear relationship between subjects (*r*^2^ = 0.95), with the *CYP2C19***17/***17* subjects clustering closest to the origin ([Figure 2](#fig02)). Given that omeprazole sulphone has been previously employed as a measure of CYP3A4 activity, we would have therefore anticipated individual differences in CYP3A4 activity to have manifested more prominently in [Figure 2](#fig02)[[28](#b28)]. These observations serve as a second independent measure of CYP2C19 activity and lend further support to the relationship between the *CYP2C19***17* allele and increased CYP2C19-associated metabolism.

CYP2C19 metabolic activity can be assessed using different probe drugs and a variety of different end-points. CYP2C19-mediated metabolism of omeprazole is commonly quantified using either the AUC ratio of omeprazole/5-hydroxyomeprazole or, alternatively, the plasma concentration ratio between omeprazole and the 5-hydroxy metabolite 3 h after drug administration [[29](#b29)]. We have demonstrated a nearly equivalent statistically significant difference between genotypes using omeprazole AUC_∞_, omeprazole sulphone AUC_∞_ and the ratio of omeprazole AUC_∞_/5-hydroxyomeprazole AUC_∞_. The interindividual variability in absorption kinetics made a meaningful comparison of omeprazole plasma concentration ratios impossible.

As a group, *CYP2C19***17* homozygous individuals have a relatively homogeneous phenotype, and conversely, *CYP2C19***1/***1* individuals display significant heterogeneity in rates of CYP2C19-associated metabolism ([Figure 1](#fig01), *f* = 0.001). The scatter plots from both Rudberg *et al.*[[14](#b14)] and Sim *et al.*[[10](#b10)], along with [Figure 1](#fig01) from the present study, demonstrate a very similar distribution pattern for each genotype group irrespective of end-point. The explanation for both these phenomena could lie within the genetics and structure of the CYP2C gene locus. Previously, we have shown via electrophoretic mobility shift analysis that the −806C→T mutation directly affects the binding of nuclear proteins, indicating a direct effect on CYP2C19 transcription [[10](#b10)]. However, it stands to reason that the −3402C→T and −806C→T SNPs could also be genetic markers for a haplotype block extending significantly beyond the region from −3.5 kb to the 3′UTR of the CYP2C19 gene influencing the expression of the *CYP2C19* gene. Consistent with this, genotyping studies have shown *CYP2C19***17* and *CYP2C9***1* to be in 100% linkage, and additional information indicates that *CYP2C18*, *CYP2C19* and the exonic regions of *CYP2C9* reside within the same haplotype block [[14](#b14), [30](#b30)]. The existence of unidentified polymorphisms could thus account for the heterogeneity of CYP2C19-associated metabolic activity observed in the group currently defined as ‘*CYP2C19***1*’. It could also be postulated that drugs which are substrates for both CYP2C19 and CYP2C9 could be influenced by the suggested genetic linkage between these two genes.

In summary, we have further substantiated and also directly quantified the impact of the *CYP2C19***17* allele on the pharmacokinetics of omeprazole relative to the *CYP2C19***1* allele. Homozygous *CYP2C19***17* carriers experience a mean 2.1-fold reduction in omeprazole systemic drug exposure. Taken within the broader context of clinically important drugs that are metabolized predominantly by CYP2C19, the CYP2C19*17 allele has the potential to be a critical factor behind individual cases of therapeutic failure. Future studies will help to address the value of genotyping either prospectively or in instances of treatment resistance or failure.

## Competing interests

None to declare.

We thank Tommy Andersson, Kjell Andersson and Lars Weidolf from AstraZeneca for their assistance, expertise and for graciously providing analytical standards. We also thank Jolanta Widen for her assistance and technical expertise. We also wish to acknowledge Lilleba Bohman and Susanne Virding for their roles in patient genotyping, and Peter Johansson and Ilona Skilving for their invaluable assistance in the clinic. The clinical component of this work was supported by a grant from The Stockholm County Council (ALF 20050551/20060420). Additional funding was provided by The Swedish Research Council, The Danish Agency for Science, Technology and Innovation, The Lundbeck Foundation and from Torsten och Ragnar Söderbergs stiftelser. E.E. is an associate professor at Karolinska Institutet, but currently employed by AstraZeneca R & D Södertälje. However, at the time of the study E.E. was employed by Karolinska University Hospital. E.E. was the recipient of a clinical pharmacology research stipend from Pfizer Sweden AB.

## References

1. Desta Z, Zhao X, Shin JG, Flockhart DA. Clinical significance of the cytochrome P450 2C19 genetic polymorphism. Clin Pharmacokinet. 2002;41:913–58. doi: 10.2165/00003088-200241120-00002.  [DOI](https://doi.org/10.2165/00003088-200241120-00002) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12222994/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacokinet&title=Clinical%20significance%20of%20the%20cytochrome%20P450%202C19%20genetic%20polymorphism&author=Z%20Desta&author=X%20Zhao&author=JG%20Shin&author=DA%20Flockhart&volume=41&publication_year=2002&pages=913-58&pmid=12222994&doi=10.2165/00003088-200241120-00002&)

2. Bertilsson L, Henthorn TK, Sanz E, Tybring G, Sawe J, Villen T. Importance of genetic factors in the regulation of diazepam metabolism: relationship to S-mephenytoin, but not debrisoquin, hydroxylation phenotype. Clin Pharmacol Ther. 1989;45:348–55. doi: 10.1038/clpt.1989.40.  [DOI](https://doi.org/10.1038/clpt.1989.40) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2495208/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Importance%20of%20genetic%20factors%20in%20the%20regulation%20of%20diazepam%20metabolism:%20relationship%20to%20S-mephenytoin,%20but%20not%20debrisoquin,%20hydroxylation%20phenotype&author=L%20Bertilsson&author=TK%20Henthorn&author=E%20Sanz&author=G%20Tybring&author=J%20Sawe&volume=45&publication_year=1989&pages=348-55&pmid=2495208&doi=10.1038/clpt.1989.40&)

3. Qin XP, Xie HG, Wang W, He N, Huang SL, Xu ZH, Ou-Yang DS, Wang YJ, Zhou HH. Effect of the gene dosage of CYP2C19 on diazepam metabolism in Chinese subjects. Clin Pharmacol Ther. 1999;66:642–6. doi: 10.1016/S0009-9236(99)90075-9.  [DOI](https://doi.org/10.1016/S0009-9236(99)90075-9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10613621/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Effect%20of%20the%20gene%20dosage%20of%20CYP2C19%20on%20diazepam%20metabolism%20in%20Chinese%20subjects&author=XP%20Qin&author=HG%20Xie&author=W%20Wang&author=N%20He&author=SL%20Huang&volume=66&publication_year=1999&pages=642-6&pmid=10613621&doi=10.1016/S0009-9236(99)90075-9&)

4. Herrlin K, Yasui-Furukori N, Tybring G, Widen J, Gustafsson LL, Bertilsson L. Metabolism of citalopram enantiomers in CYP2C19/CYP2D6 phenotyped panels of healthy Swedes. Br J Clin Pharmacol. 2003;56:415–21. doi: 10.1046/j.1365-2125.2003.01874.x.  [DOI](https://doi.org/10.1046/j.1365-2125.2003.01874.x) | [PMC free article](/articles/PMC1884377/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12968986/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=Metabolism%20of%20citalopram%20enantiomers%20in%20CYP2C19/CYP2D6%20phenotyped%20panels%20of%20healthy%20Swedes&author=K%20Herrlin&author=N%20Yasui-Furukori&author=G%20Tybring&author=J%20Widen&author=LL%20Gustafsson&volume=56&publication_year=2003&pages=415-21&pmid=12968986&doi=10.1046/j.1365-2125.2003.01874.x&)

5. Sindrup SH, Brosen K, Hansen MG, Aaes-Jorgensen T, Overo KF, Gram LF. Pharmacokinetics of citalopram in relation to the sparteine and the mephenytoin oxidation polymorphisms. Ther Drug Monit. 1993;15:11–7. doi: 10.1097/00007691-199302000-00002.  [DOI](https://doi.org/10.1097/00007691-199302000-00002) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8451774/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ther%20Drug%20Monit&title=Pharmacokinetics%20of%20citalopram%20in%20relation%20to%20the%20sparteine%20and%20the%20mephenytoin%20oxidation%20polymorphisms&author=SH%20Sindrup&author=K%20Brosen&author=MG%20Hansen&author=T%20Aaes-Jorgensen&author=KF%20Overo&volume=15&publication_year=1993&pages=11-7&pmid=8451774&doi=10.1097/00007691-199302000-00002&)

6. Klotz U, Schwab M, Treiber G. CYP2C19 polymorphism and proton pump inhibitors. Basic Clin Pharmacol Toxicol. 2004;95:2–8. doi: 10.1111/j.1600-0773.2004.pto950102.x.  [DOI](https://doi.org/10.1111/j.1600-0773.2004.pto950102.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15245569/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Basic%20Clin%20Pharmacol%20Toxicol&title=CYP2C19%20polymorphism%20and%20proton%20pump%20inhibitors&author=U%20Klotz&author=M%20Schwab&author=G%20Treiber&volume=95&publication_year=2004&pages=2-8&pmid=15245569&doi=10.1111/j.1600-0773.2004.pto950102.x&)

7. Andersson T, Holmberg J, Rohss K, Walan A. Pharmacokinetics and effect on caffeine metabolism of the proton pump inhibitors, omeprazole, lansoprazole, and pantoprazole. Br J Clin Pharmacol. 1998;45:369–75. doi: 10.1046/j.1365-2125.1998.t01-1-00702.x.  [DOI](https://doi.org/10.1046/j.1365-2125.1998.t01-1-00702.x) | [PMC free article](/articles/PMC1873971/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9578184/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=Pharmacokinetics%20and%20effect%20on%20caffeine%20metabolism%20of%20the%20proton%20pump%20inhibitors,%20omeprazole,%20lansoprazole,%20and%20pantoprazole&author=T%20Andersson&author=J%20Holmberg&author=K%20Rohss&author=A%20Walan&volume=45&publication_year=1998&pages=369-75&pmid=9578184&doi=10.1046/j.1365-2125.1998.t01-1-00702.x&)

8. Furuta T, Ohashi K, Kamata T, Takashima M, Kosuge K, Kawasaki T, Hanai H, Kubota T, Ishizaki T, Kaneko E. Effect of genetic differences in omeprazole metabolism on cure rates for Helicobacter pylori infection and peptic ulcer. Ann Intern Med. 1998;129:1027–30. doi: 10.7326/0003-4819-129-12-199812150-00006.  [DOI](https://doi.org/10.7326/0003-4819-129-12-199812150-00006) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9867757/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Intern%20Med&title=Effect%20of%20genetic%20differences%20in%20omeprazole%20metabolism%20on%20cure%20rates%20for%20Helicobacter%20pylori%20infection%20and%20peptic%20ulcer&author=T%20Furuta&author=K%20Ohashi&author=T%20Kamata&author=M%20Takashima&author=K%20Kosuge&volume=129&publication_year=1998&pages=1027-30&pmid=9867757&doi=10.7326/0003-4819-129-12-199812150-00006&)

9. Klotz U. Clinical impact of CYP2C19 polymorphism on the action of proton pump inhibitors: a review of a special problem. Int J Clin Pharmacol Ther. 2006;44:297–302. doi: 10.5414/cpp44297.  [DOI](https://doi.org/10.5414/cpp44297) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16961157/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Clin%20Pharmacol%20Ther&title=Clinical%20impact%20of%20CYP2C19%20polymorphism%20on%20the%20action%20of%20proton%20pump%20inhibitors:%20a%20review%20of%20a%20special%20problem&author=U%20Klotz&volume=44&publication_year=2006&pages=297-302&pmid=16961157&doi=10.5414/cpp44297&)

10. Sim SC, Risinger C, Dahl ML, Aklillu E, Christensen M, Bertilsson L, Ingelman-Sundberg M. A common novel CYP2C19 gene variant causes ultrarapid drug metabolism relevant for the drug response to proton pump inhibitors and antidepressants. Clin Pharmacol Ther. 2006;79:103–13. doi: 10.1016/j.clpt.2005.10.002.  [DOI](https://doi.org/10.1016/j.clpt.2005.10.002) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16413245/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=A%20common%20novel%20CYP2C19%20gene%20variant%20causes%20ultrarapid%20drug%20metabolism%20relevant%20for%20the%20drug%20response%20to%20proton%20pump%20inhibitors%20and%20antidepressants&author=SC%20Sim&author=C%20Risinger&author=ML%20Dahl&author=E%20Aklillu&author=M%20Christensen&volume=79&publication_year=2006&pages=103-13&pmid=16413245&doi=10.1016/j.clpt.2005.10.002&)

11. Kobayashi K, Chiba K, Sohn DR, Kato Y, Ishizaki T. Simultaneous determination of omeprazole and its metabolites in plasma and urine by reversed-phase high-performance liquid chromatography with an alkaline-resistant polymer-coated C18 column. J Chromatogr. 1992;579:299–305. doi: 10.1016/0378-4347(92)80395-7.  [DOI](https://doi.org/10.1016/0378-4347(92)80395-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1429977/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Chromatogr&title=Simultaneous%20determination%20of%20omeprazole%20and%20its%20metabolites%20in%20plasma%20and%20urine%20by%20reversed-phase%20high-performance%20liquid%20chromatography%20with%20an%20alkaline-resistant%20polymer-coated%20C18%20column&author=K%20Kobayashi&author=K%20Chiba&author=DR%20Sohn&author=Y%20Kato&author=T%20Ishizaki&volume=579&publication_year=1992&pages=299-305&pmid=1429977&doi=10.1016/0378-4347(92)80395-7&)

12. Tybring G, Bottiger Y, Widen J, Bertilsson L. Enantioselective hydroxylation of omeprazole catalyzed by CYP2C19 in Swedish white subjects. Clin Pharmacol Ther. 1997;62:129–37. doi: 10.1016/S0009-9236(97)90060-6.  [DOI](https://doi.org/10.1016/S0009-9236(97)90060-6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9284848/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Enantioselective%20hydroxylation%20of%20omeprazole%20catalyzed%20by%20CYP2C19%20in%20Swedish%20white%20subjects&author=G%20Tybring&author=Y%20Bottiger&author=J%20Widen&author=L%20Bertilsson&volume=62&publication_year=1997&pages=129-37&pmid=9284848&doi=10.1016/S0009-9236(97)90060-6&)

13. Uno T, Niioka T, Hayakari M, Yasui-Furukori N, Sugawara K, Tateishi T. Absolute bioavailability and metabolism of omeprazole in relation to CYP2C19 genotypes following single intravenous and oral administrations. Eur J Clin Pharmacol. 2007;63:143–9. doi: 10.1007/s00228-006-0251-7.  [DOI](https://doi.org/10.1007/s00228-006-0251-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17203292/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Clin%20Pharmacol&title=Absolute%20bioavailability%20and%20metabolism%20of%20omeprazole%20in%20relation%20to%20CYP2C19%20genotypes%20following%20single%20intravenous%20and%20oral%20administrations&author=T%20Uno&author=T%20Niioka&author=M%20Hayakari&author=N%20Yasui-Furukori&author=K%20Sugawara&volume=63&publication_year=2007&pages=143-9&pmid=17203292&doi=10.1007/s00228-006-0251-7&)

14. Rudberg I, Mohebi B, Hermann M, Refsum H, Molden E. Impact of the ultrarapid CYP2C19*17 allele on serum concentration of escitalopram in psychiatric patients. Mol Ther. 2008;83:322–7. doi: 10.1038/sj.clpt.6100291.  [DOI](https://doi.org/10.1038/sj.clpt.6100291) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17625515/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Ther&title=Impact%20of%20the%20ultrarapid%20CYP2C19*17%20allele%20on%20serum%20concentration%20of%20escitalopram%20in%20psychiatric%20patients&author=I%20Rudberg&author=B%20Mohebi&author=M%20Hermann&author=H%20Refsum&author=E%20Molden&volume=83&publication_year=2008&pages=322-7&pmid=17625515&doi=10.1038/sj.clpt.6100291&)

15. Kurzawski M, Gawronska-Szklarz B, Wrzesniewska J, Siuda A, Starzynska T, Drozdzik M. Effect of CYP2C19*17 gene variant on Helicobacter pylori eradication in peptic ulcer patients. Eur J Clin Pharmacol. 2006;62:877–80. doi: 10.1007/s00228-006-0183-2.  [DOI](https://doi.org/10.1007/s00228-006-0183-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16912869/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Clin%20Pharmacol&title=Effect%20of%20CYP2C19*17%20gene%20variant%20on%20Helicobacter%20pylori%20eradication%20in%20peptic%20ulcer%20patients&author=M%20Kurzawski&author=B%20Gawronska-Szklarz&author=J%20Wrzesniewska&author=A%20Siuda&author=T%20Starzynska&volume=62&publication_year=2006&pages=877-80&pmid=16912869&doi=10.1007/s00228-006-0183-2&)

16. Padol S, Yuan Y, Thabane M, Padol IT, Hunt RH. The effect of CYP2C19 polymorphisms on H. pylori eradication rate in dual and triple first-line PPI therapies: a meta-analysis. Am J Gastroenterol. 2006;101:1467–75. doi: 10.1111/j.1572-0241.2006.00717.x.  [DOI](https://doi.org/10.1111/j.1572-0241.2006.00717.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16863547/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Gastroenterol&title=The%20effect%20of%20CYP2C19%20polymorphisms%20on%20H.%20pylori%20eradication%20rate%20in%20dual%20and%20triple%20first-line%20PPI%20therapies:%20a%20meta-analysis&author=S%20Padol&author=Y%20Yuan&author=M%20Thabane&author=IT%20Padol&author=RH%20Hunt&volume=101&publication_year=2006&pages=1467-75&pmid=16863547&doi=10.1111/j.1572-0241.2006.00717.x&)

17. Rodrigues AD, Roberts EM, Mulford DJ, Yao Y, Ouellet D. Oxidative metabolism of clarithromycin in the presence of human liver microsomes. Major role for the cytochrome P4503A (CYP3A) subfamily. Drug Metab Dispos. 1997;25:623–30.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/9152603/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Oxidative%20metabolism%20of%20clarithromycin%20in%20the%20presence%20of%20human%20liver%20microsomes.%20Major%20role%20for%20the%20cytochrome%20P4503A%20(CYP3A)%20subfamily&author=AD%20Rodrigues&author=EM%20Roberts&author=DJ%20Mulford&author=Y%20Yao&author=D%20Ouellet&volume=25&publication_year=1997&pages=623-30&pmid=9152603&)

18. Furuta T, Ohashi K, Kobayashi K, Iida I, Yoshida H, Shirai N, Takashima M, Kosuge K, Hanai H, Chiba K, Ishizaki T, Kaneko E. Effects of clarithromycin on the metabolism of omeprazole in relation to CYP2C19 genotype status in humans. Clin Pharmacol Ther. 1999;66:265–74. doi: 10.1016/S0009-9236(99)70034-2.  [DOI](https://doi.org/10.1016/S0009-9236(99)70034-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10511062/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Effects%20of%20clarithromycin%20on%20the%20metabolism%20of%20omeprazole%20in%20relation%20to%20CYP2C19%20genotype%20status%20in%20humans&author=T%20Furuta&author=K%20Ohashi&author=K%20Kobayashi&author=I%20Iida&author=H%20Yoshida&volume=66&publication_year=1999&pages=265-74&pmid=10511062&doi=10.1016/S0009-9236(99)70034-2&)

19. Saito M, Yasui-Furukori N, Uno T, Takahata T, Sugawara K, Munakata A, Tateishi T. Effects of clarithromycin on lansoprazole pharmacokinetics between CYP2C19 genotypes. Br J Clin Pharmacol. 2005;59:302–9. doi: 10.1111/j.1365-2125.2004.02329.x.  [DOI](https://doi.org/10.1111/j.1365-2125.2004.02329.x) | [PMC free article](/articles/PMC1884793/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15752376/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=Effects%20of%20clarithromycin%20on%20lansoprazole%20pharmacokinetics%20between%20CYP2C19%20genotypes&author=M%20Saito&author=N%20Yasui-Furukori&author=T%20Uno&author=T%20Takahata&author=K%20Sugawara&volume=59&publication_year=2005&pages=302-9&pmid=15752376&doi=10.1111/j.1365-2125.2004.02329.x&)

20. Roedler R, Neuhauser MM, Penzak SR. Does metronidazole interact with CYP3A substrates by inhibiting their metabolism through this metabolic pathway? Or should other mechanisms be considered? Ann Pharmacother. 2007;41:653–8. doi: 10.1345/aph.1H401.  [DOI](https://doi.org/10.1345/aph.1H401) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17374625/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Pharmacother&title=Does%20metronidazole%20interact%20with%20CYP3A%20substrates%20by%20inhibiting%20their%20metabolism%20through%20this%20metabolic%20pathway?%20Or%20should%20other%20mechanisms%20be%20considered?&author=R%20Roedler&author=MM%20Neuhauser&author=SR%20Penzak&volume=41&publication_year=2007&pages=653-8&pmid=17374625&doi=10.1345/aph.1H401&)

21. Blyden GT, Scavone JM, Greenblatt DJ. Metronidazole impairs clearance of phenytoin but not of alprazolam or lorazepam. J Clin Pharmacol. 1988;28:240–5. doi: 10.1002/j.1552-4604.1988.tb03139.x.  [DOI](https://doi.org/10.1002/j.1552-4604.1988.tb03139.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/3360972/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Pharmacol&title=Metronidazole%20impairs%20clearance%20of%20phenytoin%20but%20not%20of%20alprazolam%20or%20lorazepam&author=GT%20Blyden&author=JM%20Scavone&author=DJ%20Greenblatt&volume=28&publication_year=1988&pages=240-5&pmid=3360972&doi=10.1002/j.1552-4604.1988.tb03139.x&)

22. Kirchheiner J, Nickchen K, Bauer M, Wong ML, Licinio J, Roots I, Brockmoller J. Pharmacogenetics of antidepressants and antipsychotics: the contribution of allelic variations to the phenotype of drug response. Mol Psychiatry. 2004;9:442–73. doi: 10.1038/sj.mp.4001494.  [DOI](https://doi.org/10.1038/sj.mp.4001494) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15037866/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Psychiatry&title=Pharmacogenetics%20of%20antidepressants%20and%20antipsychotics:%20the%20contribution%20of%20allelic%20variations%20to%20the%20phenotype%20of%20drug%20response&author=J%20Kirchheiner&author=K%20Nickchen&author=M%20Bauer&author=ML%20Wong&author=J%20Licinio&volume=9&publication_year=2004&pages=442-73&pmid=15037866&doi=10.1038/sj.mp.4001494&)

23. Takada K, Arefayene M, Desta Z, Yarboro CH, Boumpas DT, Balow JE, Flockhart DA, Illei GG. Cytochrome P450 pharmacogenetics as a predictor of toxicity and clinical response to pulse cyclophosphamide in lupus nephritis. Arthritis Rheum. 2004;50:2202–10. doi: 10.1002/art.20338.  [DOI](https://doi.org/10.1002/art.20338) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15248218/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arthritis%20Rheum&title=Cytochrome%20P450%20pharmacogenetics%20as%20a%20predictor%20of%20toxicity%20and%20clinical%20response%20to%20pulse%20cyclophosphamide%20in%20lupus%20nephritis&author=K%20Takada&author=M%20Arefayene&author=Z%20Desta&author=CH%20Yarboro&author=DT%20Boumpas&volume=50&publication_year=2004&pages=2202-10&pmid=15248218&doi=10.1002/art.20338&)

24. Li Y, Hou J, Jiang H, Wang D, Fu W, Yuan Z, Chen Y, Zhou L. Polymorphisms of CYP2C19 gene are associated with the efficacy of thalidomide based regimens in multiple myeloma. Haematologica. 2007;92:1246–9. doi: 10.3324/haematol.11319.  [DOI](https://doi.org/10.3324/haematol.11319) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17666363/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Haematologica&title=Polymorphisms%20of%20CYP2C19%20gene%20are%20associated%20with%20the%20efficacy%20of%20thalidomide%20based%20regimens%20in%20multiple%20myeloma&author=Y%20Li&author=J%20Hou&author=H%20Jiang&author=D%20Wang&author=W%20Fu&volume=92&publication_year=2007&pages=1246-9&pmid=17666363&doi=10.3324/haematol.11319&)

25. Andersson T, Miners JO, Veronese ME, Tassaneeyakul W, Tassaneeyakul W, Meyer UA, Birkett DJ. Identification of human liver cytochrome P450 isoforms mediating omeprazole metabolism. Br J Clin Pharmacol. 1993;36:521–30. doi: 10.1111/j.1365-2125.1993.tb00410.x.  [DOI](https://doi.org/10.1111/j.1365-2125.1993.tb00410.x) | [PMC free article](/articles/PMC1364656/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12959268/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=Identification%20of%20human%20liver%20cytochrome%20P450%20isoforms%20mediating%20omeprazole%20metabolism&author=T%20Andersson&author=JO%20Miners&author=ME%20Veronese&author=W%20Tassaneeyakul&author=W%20Tassaneeyakul&volume=36&publication_year=1993&pages=521-30&pmid=12959268&doi=10.1111/j.1365-2125.1993.tb00410.x&)

26. Regardh CG, Andersson T, Lagerstrom PO, Lundborg P, Skanberg I. The pharmacokinetics of omeprazole in humans – a study of single intravenous and oral doses. Ther Drug Monit. 1990;12:163–72. doi: 10.1097/00007691-199003000-00010.  [DOI](https://doi.org/10.1097/00007691-199003000-00010) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2315973/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ther%20Drug%20Monit&title=The%20pharmacokinetics%20of%20omeprazole%20in%20humans%20%E2%80%93%20a%20study%20of%20single%20intravenous%20and%20oral%20doses&author=CG%20Regardh&author=T%20Andersson&author=PO%20Lagerstrom&author=P%20Lundborg&author=I%20Skanberg&volume=12&publication_year=1990&pages=163-72&pmid=2315973&doi=10.1097/00007691-199003000-00010&)

27. Andersson T, Miners JO, Veronese ME, Birkett DJ. Identification of human liver cytochrome P450 isoforms mediating secondary omeprazole metabolism. Br J Clin Pharmacol. 1994;37:597–604. doi: 10.1111/j.1365-2125.1994.tb04310.x.  [DOI](https://doi.org/10.1111/j.1365-2125.1994.tb04310.x) | [PMC free article](/articles/PMC1364821/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7917780/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=Identification%20of%20human%20liver%20cytochrome%20P450%20isoforms%20mediating%20secondary%20omeprazole%20metabolism&author=T%20Andersson&author=JO%20Miners&author=ME%20Veronese&author=DJ%20Birkett&volume=37&publication_year=1994&pages=597-604&pmid=7917780&doi=10.1111/j.1365-2125.1994.tb04310.x&)

28. Bottiger Y. Use of omeprazole sulfone in a single plasma sample as a probe for CYP3A4. Eur J Clin Pharmacol. 2006;62:621–5. doi: 10.1007/s00228-006-0156-5.  [DOI](https://doi.org/10.1007/s00228-006-0156-5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16791583/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Clin%20Pharmacol&title=Use%20of%20omeprazole%20sulfone%20in%20a%20single%20plasma%20sample%20as%20a%20probe%20for%20CYP3A4&author=Y%20Bottiger&volume=62&publication_year=2006&pages=621-5&pmid=16791583&doi=10.1007/s00228-006-0156-5&)

29. Chang M, Dahl ML, Tybring G, Gotharson E, Bertilsson L. Use of omeprazole as a probe drug for CYP2C19 phenotype in Swedish Caucasians: comparison with S-mephenytoin hydroxylation phenotype and CYP2C19 genotype. Pharmacogenetics. 1995;5:358–63. doi: 10.1097/00008571-199512000-00004.  [DOI](https://doi.org/10.1097/00008571-199512000-00004) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8747407/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Use%20of%20omeprazole%20as%20a%20probe%20drug%20for%20CYP2C19%20phenotype%20in%20Swedish%20Caucasians:%20comparison%20with%20S-mephenytoin%20hydroxylation%20phenotype%20and%20CYP2C19%20genotype&author=M%20Chang&author=ML%20Dahl&author=G%20Tybring&author=E%20Gotharson&author=L%20Bertilsson&volume=5&publication_year=1995&pages=358-63&pmid=8747407&doi=10.1097/00008571-199512000-00004&)

30. Walton R, Kimber M, Rockett K, Trafford C, Kwiatkowski D, Sirugo G. Haplotype block structure of the cytochrome P450 CYP2C gene cluster on chromosome 10. Nat Genet. 2005;37:915–6. doi: 10.1038/ng0905-915. author reply 916.  [DOI](https://doi.org/10.1038/ng0905-915) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16132042/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Haplotype%20block%20structure%20of%20the%20cytochrome%20P450%20CYP2C%20gene%20cluster%20on%20chromosome%2010&author=R%20Walton&author=M%20Kimber&author=K%20Rockett&author=C%20Trafford&author=D%20Kwiatkowski&volume=37&publication_year=2005&pages=915-6&pmid=16132042&doi=10.1038/ng0905-915&)
