# Germline Genetic Variation in an Organic Anion Transporter Polypeptide Associated With Methotrexate Pharmacokinetics and Clinical Effects

## Metadata
**Authors:** Lisa R Treviño, Noriko Shimasaki, Wenjian Yang, John C Panetta, Cheng Cheng, Deqing Pei, Diana Chan, Alex Sparreboom, Kathleen M Giacomini, Ching-Hon Pui, William E Evans, Mary V Relling
**Journal:** Journal of Clinical Oncology
**Date:** 2009 Nov 9
**DOI:** [10.1200/JCO.2008.20.4156](https://doi.org/10.1200/JCO.2008.20.4156)
**PMID:** 19901119
**PMCID:** PMC2793040
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2793040/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC2793040/pdf/zlj5972.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC2793040/pdf/zlj5972.pdf)

## Abstract

**Purpose:** 
Methotrexate plasma concentration is related to its clinical effects. Our aim was to identify the genetic basis of interindividual variability in methotrexate pharmacokinetics in children with newly diagnosed acute lymphoblastic leukemia (ALL).

**Patients and Methods:** 
We performed a genome-wide analysis of 500,568 germline single-nucleotide polymorphisms (SNPs) to identify how inheritance affects methotrexate plasma disposition among 434 children with ALL who received 3,014 courses of methotrexate at 2 to 5 g/m2. SNPs were validated in an independent cohort of 206 patients.

**Results:** 
Adjusting for age, race, sex, and methotrexate regimen, the most significant associations were with SNPs in the organic anion transporter polypeptide, SLCO1B1. Two SNPs in SLCO1B1, rs11045879 (P = 1.7 × 10−10) and rs4149081 (P = 1.7 × 10−9), were in linkage disequilibrium (LD) with each other (r2 = 1) and with a functional polymorphism in SLCO1B1, T521C (rs4149056; r2 > 0.84). rs11045879 and rs4149081 were validated in an independent cohort of 206 patients (P = .018 and P = .017), as were other SLCO1B1 SNPs residing in different LD blocks. SNPs in SLCO1B1 were also associated with GI toxicity (odds ratio, 15.3 to 16.4; P = .03 to .004).

**Conclusion:** 
A genome-wide interrogation identified inherited variations in a plausible, yet heretofore low-priority candidate gene, SLCO1B1, as important determinants of methotrexate's pharmacokinetics and clinical effects.

### Purpose

Methotrexate plasma concentration is related to its clinical effects. Our aim was to identify the genetic basis of interindividual variability in methotrexate pharmacokinetics in children with newly diagnosed acute lymphoblastic leukemia (ALL).

### Patients and Methods

We performed a genome-wide analysis of 500,568 germline single-nucleotide polymorphisms (SNPs) to identify how inheritance affects methotrexate plasma disposition among 434 children with ALL who received 3,014 courses of methotrexate at 2 to 5 g/m^2^. SNPs were validated in an independent cohort of 206 patients.

### Results

Adjusting for age, race, sex, and methotrexate regimen, the most significant associations were with SNPs in the organic anion transporter polypeptide, *SLCO1B1.* Two SNPs in *SLCO1B1*, rs11045879 (*P* = 1.7 × 10^−10^) and rs4149081 (*P* = 1.7 × 10^−9^), were in linkage disequilibrium (LD) with each other (*r*^2^ = 1) and with a functional polymorphism in *SLCO1B1*, T521C (rs4149056; *r*^2^ > 0.84). rs11045879 and rs4149081 were validated in an independent cohort of 206 patients (*P* = .018 and *P* = .017), as were other *SLCO1B1* SNPs residing in different LD blocks. SNPs in *SLCO1B1* were also associated with GI toxicity (odds ratio, 15.3 to 16.4; *P* = .03 to .004).

### Conclusion

A genome-wide interrogation identified inherited variations in a plausible, yet heretofore low-priority candidate gene, *SLCO1B1*, as important determinants of methotrexate's pharmacokinetics and clinical effects.

## INTRODUCTION

Whole-genome studies have identified genetic variations associated with risk of complex diseases.^1–5^ However, whole-genome pharmacogenetic studies are less common despite considerable speculation that such information will transform drug therapy.^6–10^ Methotrexate is used to treat malignancies, including acute lymphoblastic leukemia (ALL), as well as autoimmune disorders.^11–15^

The pharmacologic effects of methotrexate are well characterized, and its systemic exposure has been related to cure and toxicity in childhood ALL^16–19^ and other diseases.^20,21,19,22,23^ The genetic origins of interindividual pharmacokinetic and pharmacodynamic variability of methotrexate remain poorly understood, with conflicting results on candidate pharmacogenetic predictors for methotrexate.^24,25^ A genome-wide approach might identify new candidate genes whose polymorphisms forecast which patients might benefit from tailoring dosage on the basis of genomic variation. Herein, we performed a genome-wide association analysis of 434 patients with newly diagnosed ALL and surveyed 500,568 germline single-nucleotide polymorphisms (SNPs).

## PATIENTS AND METHODS

### Patients and Treatment

The discovery cohort included 434 children (median age, 5.92 years; range, 1.02 to 18.85 years) with ALL enrolled and treated on St. Jude Children's Research Hospital Total XIIIB and Total XV protocols (Data Supplement Table 1).^26–28^ The validation cohort consisted of the next set of 206 patients (median age, 5.10 years; range 1.17 to 18.67 years) treated on St. Jude's Total XV protocol (Data Supplement Table 1) who were enrolled after the initial analysis began. Total XIIIB included consolidation therapy of two weekly doses of methotrexate (2 g/m^2^ over 2 hours, followed by leucovorin) and 6-mercaptopurine (75 mg/m^2^ per night for 2 weeks).^26,27^ Subsequent therapy included identical methotrexate and 6-mercaptopurine doses administered every 8 weeks up to 1 year. Total XV included four doses of methotrexate during consolidation, each given over 24 hours with dosage adjusted to achieve a steady-state plasma concentration of 33 μmol/L (low-risk arm) or 65 μmol/L (standard/high-risk arm).^28^ Serum creatinine was measured in the 24 hours before methotrexate treatment. Parents and patients gave informed consent and assent as appropriate, and the study was approved by the institutional review board. All grade 3 and 4 toxicities were prospectively scored using the National Cancer Institute (NCI) Cancer Therapy Evaluation Program (CTEP) ([http://ctep.cancer.gov](http://ctep.cancer.gov)) toxicity criteria during the consolidation and continuation periods.^26^ The most common toxicities during consolidation, when only methotrexate and 6-mercaptopurine were given, were GI (mucositis) and infection. Detailed procedures for methotrexate administration (Data Supplement Table 2) and evaluations are provided in the Data Supplement.

### Pharmacokinetic Data

Clearance was estimated on the basis of three to four methotrexate plasma concentrations per course up to 48 hours from the start of infusion. Clearance was estimated (CL = k_e_ × V) using a two-compartmental linear pharmacokinetic model and a Bayesian approach, via ADAPT software (University of Southern California, Los Angeles, CA), as described.^26,29,30^ Population priors, mean (variance), were 9.0 L/m^2^ (22.1) for V_c_, 0.70 hours^−1^ (0.05) for K_e_, 0.08 hours^−1^ (0.002) for K_cp_, and 0.11 hours^−1^ (0.000014) for K_pc_ as described.^30^ The average methotrexate clearance per patient across courses was calculated using all available postremission courses.

### Genotyping and SNP Filtering Criteria

Germline DNA was extracted from blood after remission was achieved. DNA (500 ng) was applied to the 500K Array Set (Affymetrix, Santa Clara, CA). Chips were scanned and genotype calls were made using the Bayesian Robust Linear Multichip with Mahalanobis Distance (BRLMM) algorithm for 500,568 interrogated SNPs. Initial missing SNP genotypes were imputed using the fastPHASE software (Data Supplement; University of Washington, Seattle).^31^ SNPs were excluded for genotyping call rates less than 95% (n = 29,314), minor allele frequency less than 1% (n = 63,110), or if allele frequencies deviated from Hardy-Weinberg equilibrium (n = 10,944; *P* < 1 × 10^−7^). Some SNPs were excluded that met more than one criterion, leaving 398,699 SNPs in the analysis.

For a subset of patients in the validation cohort (n = 103), SNP genotypes were obtained using Affymetrix Genome-Wide Human SNP Array 6.0 (Affymetrix). The nonsynonymous SNP, *SLCO1B1* T521C (rs4149056) was not represented on the Affymetrix arrays and was independently genotyped (DNAPrint Genomics, Sarasota, FL); call rates were more than 97% for this SNP.

### Statistical Analysis

Analyses were conducted using R ([http://www.r-project.org/](http://www.r-project.org/)) and SAS (SAS Institute, Cary, NC). Associations between average methotrexate clearance and patient characteristics (sex, age, race, and treatment regimen) were tested using analysis of variance (ANOVA). Nongenetic factors (eg, hydration and alkalinization) were standardized for each treatment regimen (Data Supplement). Associations between germline SNP genotypes and methotrexate clearance were evaluated using a general linear model, treating genotype as a numerical variable (0 = AA, 1 = AB, and 2 = BB) and including age, race, sex, and treatment regimen as covariates. Age was coded in years and treated as a continuous variable. Treatment regimen was coded as a categoric variable, and Total XIIIB was considered the baseline covariate in the multivariate analysis. Ancestry was determined using the actual genotypes, computing each patient's probability of having European, African, or Asian ancestry (Data Supplement). Ancestry was alternatively characterized using Eigenstrat software (Harvard Medical School, Boston, MA) on the basis of a principal component analysis of SNP genotypes.^32^ We found good concordance (94%) between genomically determined ancestry and self-declared race. SNPs whose *P* value for association with methotrexate clearance was less than 1 × 10^−7^ were considered reaching genome-wide significance. Because only SNPs annotated to the *SLCO1B1* gene were tested for association with methotrexate clearance in the validation cohorts, a *P* value less than .05 was considered statistically significant. Multiple linear regression analysis was used to estimate percent variation explained by genotypes and other covariates.

A genome-wide linear mixed effect model^33^ was also conducted with each individual course of methotrexate clearance as a dependent variable, including course number (course 1 to 10 for patients on Total XIIIB and course 1 to 4 for patients on Total XV) and serum creatinine for each course as covariates. A linear mixed effect model was also used to estimate the intra- and interpatient variability in methotrexate clearance in the discovery cohort.

Logistic regression was used to analyze whether *SLCO1B1* genotypes were associated with toxicity (GI toxicity and infection) as a dichotomous variable (yes or no) during consolidation. Patients with grade 3 or 4 toxicities (mucositis or infection) were considered as having toxicity. Weighted logistic regression^26^ was used to analyze the genotype-toxicity association in the 120-week continuation phase of therapy for Total XIIIB, accounting for recurrent episodes and censoring. The cumulative incidences of GI toxicity were compared among genotypes using Gray's test with incorporation of competing risks.^34^ To compute the odds ratio estimated for toxicity during the consolidation phase of therapy, we added a continuity adjustment of 0.5 to each cell count to deal with the zero event count for those with common homozygous genotypes.

For the gene level analysis, 24,836 genes were annotated to the interrogated SNPs, including all SNPs located within 5,000 base pairs (ie, *cis*-SNPs) of each transcript. We tested for significant associations between all the SNP genotypes (per gene) and methotrexate clearance using multiple linear regression, and we assessed the percent variation explained by the genotypes using *r*^2^ test statistic, the Akaike information criterion, and permutation analyses (Data Supplement).

## RESULTS

In the discovery cohort, we estimated methotrexate clearance for 3,014 treatment courses given to 434 patients (Data Supplement Table 1). There was substantial interpatient variability in average (± standard deviation) methotrexate clearance ([Fig 1](#F1)), which differed among the 213 patients on the Total XIIIB (133 ± 25 mL/min/m^2^), the 114 patients on the Total XV low-risk (123 ± 28 mL/min/m^2^), and the 107 patients on the Total XV standard/high-risk (106 ± 22 mL/min/m^2^) treatment regimens (Data Supplement Fig 1; *P* < 2.2 × 10^−16^). The variability observed between treatment regimens is not surprising, given that dosage and the concomitant hydration and alkalinization (factors that can affect methotrexate clearance) differed (Data Supplement Table 2).^22^ Average methotrexate clearance differed by ancestral group (Data Supplement Fig 2; *P* = .005) with African American patients having the highest clearance. As previously noted,^19,22^ methotrexate clearance decreased with increasing age (Data Supplement Fig 3; *P* = 1.1 × 10^-6^). Thus, we adjusted all analyses for age, race, sex, and treatment regimen. Using a linear mixed effect model, the intrapatient variability in methotrexate clearance was 48%, while the interpatient variability was 52%.

### Fig 1.

![Fig 1.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6a1e/2793040/00b983d6353e/zlj9990990890001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=2793040_zlj9990990890001.jpg)

Average methotrexate clearance and linkage disequilibrium structure of the SLCO1B1 single-nucleotide polymorphisms (SNPs) in 434 children with acute lymphoblastic leukemia. The average methotrexate clearance for each of the 434 children in the discovery cohort is plotted from the highest to lowest (gray shaded area), and each patient's genotype at each of the SLCO1B1 SNP loci listed is indicated (colored bars below the x axis). Patients homozygous for the major allele are coded in blue, heterozygotes are coded in green, and those homozygous for the minor allele are coded in yellow.

We tested the association between 398,699 SNPs and average methotrexate clearance in 434 children in the discovery cohort. The strongest association signal was on chromosome 12 ([Fig 2](#F2)). The two top SNPs associated with methotrexate clearance were annotated to the transporter gene, *SLCO1B1*/OATP1B1, located on chromosome 12 ([Table 1](#T1); Data Supplement Tables 3-5). These two *SLCO1B1* SNPs remained significant (rs11045879, *P* = 1.7 × 10^−10^ and rs4149081, *P* = 1.7 × 10^−9^) even after correction for multiple testing. These two *SLCO1B1* SNPs were in complete LD (*r*^2^ = 1) with each other ([Fig 1](#F1)) and were associated with methotrexate clearance across regimens ([Fig 3](#F3)).

### Fig 2.

![Fig 2.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6a1e/2793040/3ab05562d6bf/zlj9990990890002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=2793040_zlj9990990890002.jpg)

Genome-wide P values showing the association of single-nucleotide polymorphisms (SNPs) with methotrexate clearance in the discovery cohort of 434 children with acute lymphoblastic leukemia. Shown is the distribution of P values (as –log10 values) for the association of 398,699 SNP genotypes with methotrexate clearance in the discovery cohort. The P value of 1.7 × 10−10 is for the most significant association identified between a single SNP (located in SLCO1B1 on chromosome 12) and methotrexate clearance.

### Table 1.

| dbSNP ID | Location | Genomic Position | Alleles (A/B) | Discovery Cohort (n = 434) |  |  |  | Validation Cohort (n = 206) |  |  |  | Combined Cohort (n = 640) |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| MAF | Coefficient* | 95% CI | P | MAF | Coefficient* | 95% CI | P | MAF | Coefficient* | 95% CI | P |  |  |  |  |
| rs11045879 | Intron | 21273886 | C/T | 0.17 | 13.1 | 9.1 to 17.1 | 1.7 × 10−10 | 0.17 | 6.05 | 1.0 to 10.9 | .018 | 0.16 | 10.8 | 7.6 to 14.0 | 8.2 × 10−11 |
| rs4149081 | Intron | 21269288 | A/G | 0.16 | 12.7 | 8.5 to 16.8 | 1.7 × 10−9 | 0.17 | 6.16 | 1.1 to 11.1 | .017 | 0.16 | 10.4 | 7.2 to 13.7 | 6.7 × 10−10 |
| rs11045818 | Synonymous | 21221028 | A/G | 0.14 | −9.2 | −13.8 to −4.5 | 2.2 × 10−4 | 0.13 | −11.7 | −17.1 to −6.3 | 3.7 × 10−5 | 0.14 | −9.3 | −12.9 to −5.7 | 6.3 × 10−7 |
| rs10841753 | Intron | 21212637 | C/T | 0.20 | −7.8 | −11.7 to −3.8 | 1.3 × 10−4 | 0.15 | −7.1 | −11.8 to −2.3 | .004 | 0.19 | −7.1 | −10.2 to −4.0 | 8.6 × 10−6 |
| rs11045872 | Intron | 21263611 | A/G | 0.16 | 8.1 | 3.7 to 12.5 | 5.0 × 10−4 | 0.15 | 7.1 | 2.0 to 12.2 | .007 | 0.16 | 7.3 | 3.9 to 10.7 | 3.4 × 10−5 |
| rs2900476 | Intron | 21227330 | C/T | 0.23 | 6.5 | 2.8 to 10.2 | 6.0 × 10−4 | 0.26 | 2.5 | −2.0 to 7.0 | .27 | 0.24 | 5.0 | 2.1 to 7.9 | .000796 |
| rs17328763 | 5′ upstream | 21173837 | C/T | 0.15 | −6.1 | −10.6 to −1.6 | .01 | 0.17 | −5.4 | −10.4 to −0.4 | .034 | 0.15 | −5.3 | −8.7 to −1.9 | .002676 |
| rs11045787 | Intron | 21191269 | G/T | 0.15 | −5.9 | −10.4 to −1.4 | .01 | 0.16 | −5.5 | −10.5 to −0.4 | .034 | 0.15 | −5.2 | −8.6 to −1.7 | .003227 |
| rs4149076 | Intron | 21262411 | C/T | 0.32 | 4.4 | 0.9 to 7.9 | .01 | 0.32 | −0.4 | −4.3 to 3.6 | .85 | 0.32 | 2.9 | 0.2 to 5.6 | .033823 |
| rs7966613 | intron | 21270899 | A/G | 0.32 | −3.6 | −7.0 to −0.1 | .03 | 0.33 | 0.8 | −3.1 to 4.8 | .67 | 0.32 | −2.3 | −5.0 to 0.4 | .093071 |

Table 1 Caption: SLCO1B1 SNPs and Methotrexate Clearance

### Fig 3.

![Fig 3.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6a1e/2793040/86c8c81f5670/zlj9990990890003.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=2793040_zlj9990990890003.jpg)

SLCO1B1 single-nucleotide polymorphism (SNP) rs11045879 is associated with methotrexate clearance and GI toxicity. (A) Association of SLCO1B1 SNP genotype (rs11045879) with methotrexate clearance in patients on Total XIIIB (TXIIIB), Total XV low-risk (TXVL), and Total XV standard/high-risk (TXVSH) treatment regimens. Numbers below each box plot indicate the sample size for each genotype. (B) Association of this SNP with GI toxicity. Logistic regression was used to analyze whether genotypes for SLCO1B1 SNPs were associated with toxicity as a dichotomous variable (yes or no) during the 2-week consolidation phase. The bar graphs display the percentage of patients (plotted on the y axis) per genotype (plotted on the x axis) who had grade 3 to 4 GI toxicity. Numbers above each bar represent the number of patients who had toxicity versus those who did not for the specified genotype. Similar relationships exist for SLCO1B1 SNP rs4149081 (data not shown).

We also used a linear mixed effects model to identify associations between SNP genotypes and methotrexate clearance; the results were similar to those assessing average clearance, with the same two top-ranked SNPs (*P* < 6.8 × 10^−9^) in *SLCO1B1* (Data Supplement Table 4). Only 6.0% of the intracourse variability in clearance was accounted for by course number, even when accounting for the premethotrexate serum creatinine (Data Supplement Table 6).

Eight additional *SLCO1B1* SNPs ([Table 1](#T1)) were associated with clearance (*P* < .05); these SNPs were not in LD (based on *r*^2^) with the top two *SLCO1B1* SNPs ([Fig 1](#F1)) and were encompassed by multiple haplotype blocks (Data Supplement Figs 4A and [4](#F4)B). In a gene-level analysis, *SLCO1B1* had a stronger association with methotrexate clearance (*P* < 1 × 10^−6^), with a higher *r*^2^ and a lower Akaike information criterion, than any other gene (Data Supplement Tables 7 and 11).

### Fig 4.

![Fig 4.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6a1e/2793040/f437a2114b35/zlj9990990890004.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=2793040_zlj9990990890004.jpg)

Incidence of toxicity based on SLCO1B1 rs11045879 genotype. Cumulative incidence of the first episode of GI toxicity significantly differed (P = .024) by SLCO1B1 rs11045879 genotype during the continuation phase of treatment during the Total XIIIB treatment regimen. Cumulative incidences of GI toxicity were compared among genotypes using Gray's test with incorporation of competing risks.

We genotyped a validation cohort of 206 additional patients ([Table 1](#T1)). Seven of the 10 interrogated *SLCO1B1* SNPs remained associated (*P* < .05) with methotrexate clearance ([Table 1](#T1)). Nine of the 10 *SLCO1B1* SNPs were associated with clearance (*P* < .05) when the two cohorts (discovery and validation) were combined ([Table 1](#T1)), and the strongest signal remained in *SLCO1B1* (Data Supplement Fig 5).

The percentage of methotrexate clearance variability that could be explained by *SLCO1B1* genotypes compared with the variability explained by factors such as race, sex, age, treatment regimen, or serum creatinine was estimated in a multivariate analysis for both the discovery and validation cohorts ([Table 2](#T2)). In a stepwise linear regression analysis, five SNPs were retained in the model ([Table 2](#T2)), such that *SLCO1B1* genetic variation accounted for 9.3% of the interpatient variability in the discovery cohort and 11.3% in the validation cohort. This superseded the proportion of variability contributed by the significant factors of race, serum creatinine, and sex; only treatment regimen accounted for a higher proportion (17.9%) of the variation in clearance ([Table 2](#T2)). Average serum creatinine concentrations explained 3.2% and 1.9% of the variability in methotrexate clearance in the discovery and validation cohorts, respectively; in a linear mixed effects model (Data Supplement Table 6), 2.4% of interpatient and 0.4% of intrapatient variability was accounted for by serum creatinine. Treatment regimen and *SLCO1B1* SNP genotypes accounted for 18.6% and 11.0% of variability in clearance, respectively. Results were similar for the first cycle only of methotrexate (Data Supplement Table 8).

### Table 2.

| Variables | Discovery Cohort (n = 434) |  |  |  | Validation Cohort With Serum Creatinine Data*(n = 125) |  |  |  | Validation Cohort† (n = 206) |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Coefficient‡ | 95% CI | P§ | r2 (%) | Coefficient‡ | 95% CI | P§ | r2 (%) | Coefficient‡ | 95% CI | P§ | r2 (%) |  |
| Ancestry¶ |   |   | .0007 | 2.1 |   |   | .32 | 1.6 |   |   | .021 | 4.1 |
| African | Ref‡ |   |   |   | Ref‡ |   |   |   | Ref‡ |   |   |   |
| European | −13.0 | −19.8 to −6.3 |   |   | −7.9 | −18.5 to 2.8 |   |   | −7.3 | −16.7 to 2.0 |   |   |
| Asian | −6.5 | −25.0 to 12.0 |   |   | 2.4 | −31.8 to 36.6 |   |   | 14.6 | −4.3 to 33.6 |   |   |
| Sex |   |   | .003 | 0.01 |   |   | .02 | 3.3 |   |   | .046 | 1.1 |
| Female | Ref‡ |   |   |   | Ref‡ |   |   |   | Ref‡ |   |   |   |
| Male | 6.6 | 2.2 to 10.9 |   |   | 7.7 | 1.1 to 14.3 |   |   | 5.6 | 0.2 to 11.0 |   |   |
| Serum creatinine, mg/dL‖ | −60.2 | −86.8 to −33.7 | 8.6 × 10−6 | 3.2 | −33.6 | −71.8 to 4.6 | .09 | 1.9 | NA |   | NA | NA |
| Age, years | 0.39 | −0.49 to 1.3 | .39 | 0.1 | 0.98 | −0.21 to 2.2 | .10 | 1.7 | −0.5 | −1.1 to 0.1 | .093 | 1.7 |
| Treatment regimen |   |   | 1.1 × 10−25 | 17.9 |   |   | .00013 | 9.3 |   |   | .001 | 2.8 |
| Total XV low-risk | Ref‡ |   |   |   | Ref‡ |   |   |   | Ref‡ |   |   |   |
| Total XV standard/high-risk | −15.9 | −22.1 to −9.7 |   |   | −15.0 | −22.8 to −7.3 |   |   | −7.2 | −13.1 to −1.4 |   |   |
| Total XIIIB | 14.3 | 8.7 to 19.8 |   |   | NA |   |   |   | NA |   |   |   |
| SLCO1B1 genotype# |   |   | 2.3 × 10−10 | 9.3 |   |   | .002 | 11.3 |   |   | 4.7 × 10−4 | 9.3 |
| rs4149081 (A/G) | 14.0 | 8.5 to 19.5 |   |   | 5.2 | −3.3 to 13.8 |   |   | 6.6 | 0 to 13.2 |   |   |
| rs10841753 (C/T) | −7.2 | −13.6 to −0.8 |   |   | 0.07 | −12.1 to 12.3 |   |   | 1.3 | −6.8 to 9.4 |   |   |
| rs11045818 (A/G) | −8.3 | −19.4 to −2.9 |   |   | −18.6 | −34.7 to −2.4 |   |   | −17.8 | −29.7 to −5.8 |   |   |
| rs2900476 (C/T) | 3.4 | −1.6 to 8.4 |   |   | 3.4 | −4.2 to 11.0 |   |   | 2.9 | −3.1 to 8.9 |   |   |
| rs17328763 (C/T) | −8.4 | −18.5 to 1.6 |   |   | −6.0 | −17.4 to 5.3 |   |   | −6.2 | −14.7 to 2.1 |   |   |

Table 2 Caption: Multivariate Analysis of Factors Associated With Average Methotrexate Clearance in the Discovery and Validation Cohorts

GI toxicity (grade 3 or 4 mucositis) and infection were the most common toxicities observed during the methotrexate-intensive consolidation and continuation phases of Total XIIIB therapy.^26^ SNPs in *SLCO1B1*, rs11045879 T allele (OR, 16.4; 95% CI, 8.7 to 26.7; *P* = .004) and the G allele at rs4149081 (OR, 15.3; 95% CI, 7.9 to 24.6; *P* = .03; data not shown) were each associated with GI toxicity during consolidation ([Fig 3](#F3)). The same two SNPs were also associated with GI toxicity during the continuation phase ([Fig 4](#F4)). During the consolidation phase of Total XV, GI toxicity was less frequent (5% *v* 18% in Total XIIIB), and we found no association with *SLCO1B1* SNP genotypes. Because all patients on Total XV had their methotrexate doses adjusted to achieve a common steady-state plasma concentration, relationships among toxicity and methotrexate exposure and *SLCO1B1* SNPs may have been attenuated on this trial. We found no associations between *SLCO1B1* SNP genotype and infectious toxicity in either trial (data not shown).

We also genotyped the nonsynonymous *SLCO1B1* T521C (rs4149056) polymorphism in a subset of 489 patients; this subset included 387 patients from the discovery cohort and 102 patients from the validation cohort (Data Supplement Table 1). *SLCO1B1* T521C was in high LD with the top *SLCO1B1* SNPs rs11045879 (*r*^2^ = 0.86, discovery cohort; *r*^2^ = 0.89, validation cohort) and rs4149081 (*r*^2^ = 0.86, discovery cohort; *r*^2^ = 0.89, validation cohort). *SLCO1B1* T521C was associated with methotrexate clearance in the discovery and the combined (discovery plus validation) patient cohorts (Data Supplement Table 9; *P* = 1.9 × 10^−7^ and *P* = 1.2 × 10^−7^, respectively). In addition, a marginal association (*P* = .07) was observed between this SNP genotype and the occurrence of GI toxicity during consolidation (Data Supplement Fig 6). In multivariate analysis, the T521C SNP was significantly associated with methotrexate clearance; however, when genotypes at T521C and at rs11045879 were allowed to compete, only the rs11045879 SNP remained in the model (Data Supplement Table 10).

## DISCUSSION

Using a genome-wide interrogation of germline variation, we uncovered genetic variation that affects the disposition and effects of methotrexate in children with ALL. The strongest genetic variation associated with methotrexate pharmacokinetics and dynamics resided in a reasonable candidate gene, *SLCO1B1*, although this transporter was not previously studied as a candidate gene in clinical pharmacogenetic studies of methotrexate.^35–38^

The *SLCO1B1* locus maintained significance after adjusting for covariates such as age, race, serum creatinine, sex, and treatment regimen and was verified in an independent validation set. The top-ranked SNPs (Data Supplement Tables 3-5), rs11045879 and rs4149081, were located in the gene (*SLCO1B1*) encoding an organic anion transporter (OATP1B1). Eight additional SNPs in *SLCO1B1* had *P* values less than .05 for their association with methotrexate clearance ([Table 1](#T1)). The fact that multiple SNPs in this locus, some of which were not in LD, were independently associated with methotrexate clearance provides further evidence of the functional importance of *SLCO1B1*.

*SLCO1B1* is localized at the sinusoidal membrane of hepatocytes, and its transcript has been detected in enterocytes.^39–41^ *SLCO1B1* mediates uptake of substrates from sinusoidal blood, resulting in their net excretion from blood (likely via biliary excretion). *SLCO1B1* has been shown to transport methotrexate in vitro,^42,43^ as well as other compounds such as bilirubin, bile acids, 3-hydroxy-3-methylglutaryl coenzyme A (HMG-CoA) reductase inhibitors,^44,45^ benzylpenicillin, rifampicin, angiotensin-converting enzyme inhibitors, and the active metabolite of irinotecan, SN-38.^46,47^

Previous studies have described functional consequences of nonsynonymous polymorphisms in *SLCO1B1* in vitro^43,48^ and in vivo (eg, statins).^49^ The *SLCO1B1* T521C SNP was in high LD (*r*^2^ = 0.84) with the SNPs interrogated on the genome-wide arrays we used, and our independent genotyping confirmed an association of lower methotrexate clearance (higher systemic exposure) with the C allele. This is consistent with prior studies that demonstrate greater plasma exposure to multiple *SLCO1B1* substrates in those carrying the 521C allele than those homozygous for the 521T allele.^49–51^ Nonetheless, most of the *SLCO1B1* SNPs that were associated with clearance ([Table 1](#T1)) were noncoding SNPs, not in LD with T521C, and in a multivariate model, SNPs other than T521C maintained an association with methotrexate clearance (Data Supplement Table 10). The fact that these SNPs resided throughout the gene suggest multiple mechanisms by which variations affect the function of *SLCO1B1*, perhaps by effects on transcription and post-transcriptional processing (eg, pre-mRNA splicing and mRNA translation). Moreover, noncoding SNPs may be in linkage with rare, nontyped coding SNPs. Thus it appears that the function of *SLCO1B1* may be affected by several different polymorphisms.

SNPs in *SLCO1B1* were associated not only with methotrexate clearance but also with GI toxicity ([Fig 3](#F3)) in Total XIIIB. Only 5% of patients in Total XV had toxicity, and this may in part be attributed to targeting of methotrexate to achieve a specific plasma concentration. The alleles that were associated with lower plasma exposure were always associated with increased GI toxicity. This finding is consistent with lower blood concentrations but higher GI tract or enterocyte concentrations of methotrexate being associated with GI toxicity. The fact that *SLCO1B1* polymorphisms were not associated with the other common toxicity,^26^ infection, is consistent with GI concentrations being a determinant of methotrexate GI toxicity^52,53^ and being of relatively minor importance for infectious toxicity. Because 6-mercaptopurine was given during consolidation, toxicity (particularly infection) may have been partly due to 6-mercaptopurine. However, GI toxicity is more closely linked to methotrexate^53^ than to thiopurines.^54,55^

*SLCO1B1* genotypes were associated with methotrexate clearance across three different treatment regimens ([Fig 3](#F3)) and were significant after adjusting for regimen differences and for sex, race, renal function, and age ([Table 2](#T2) and Data Supplement Table 6). Moreover, clearance was measured on multiple occasions, and thus the average value of clearance is a composite that already accounts for intrapatient variability that may have been due to nongenetic causes. Accounting for each course independently in a mixed effects model, and accounting for variation in serum creatinine,^56^ polymorphisms in *SLCO1B1* remained significant predictors of interpatient variability in clearance. The fact that serum creatinine explained a small fraction (2.4%) of the interpatient variability in methotrexate clearance may be because serum creatinine was monitored before administration of the drug, and renal dysfunction was a contraindication to high-dose methotrexate. The proportion (9.3%) of interpatient variability in drug clearance attributable to germline variation in a single gene is on par with or greater than the variability accounted for by a single gene for other complex phenotypes.^57^

Identifying patients at risk of low methotrexate clearance could be useful for monitoring and supportive care during high-dose methotrexate, particularly in settings in which rapid turnaround of plasma levels is not available. Moreover, identifying that *SLCO1B1* accounts for a substantial degree of interpatient variability in clearance highlights that drug interactions are likely to occur if a potent *SLCO1B1* substrate is given with methotrexate.

Methotrexate is metabolized to a 7-OH methotrexate metabolite.^58^ This metabolite could interact with *SLCO1B1*. Other genetic variations may influence methotrexate clearance, such as in *SLCO1A2* (Data Supplement Tables 3-5).^59^ Moreover, there may be important insertion/deletion or germline copy number variations that were not adequately interrogated on the array we used.

Using a whole-genome approach, we identified a plausible candidate gene, *SLCO1B1*, that is associated with methotrexate pharmacokinetics and pharmacodynamics. This illustrates the proof of principle that genome-wide tools can lead to the discovery of important pharmacogenetic links between inherited genomic variation and drug response in humans.

## Supplementary Material

## Acknowledgment

We thank our protocol coinvestigators, clinical and research staff, particularly Pamela McGill, Natalie Lowery, Sean Freeman, Nancy Kornegay, and Jennifer Pauley, PharmD, as well as patients and families for their participation. We thank the staff from the St Jude Hartwell Center for Bioinformatics and Biotechnology.

## Footnotes

## AUTHORS' DISCLOSURES OF POTENTIAL CONFLICTS OF INTEREST

The author(s) indicated no potential conflicts of interest.

## AUTHOR CONTRIBUTIONS

**Conception and design:** Mary V. Relling

**Financial support:** Mary V. Relling, William E. Evans

**Administrative support:** Ching-Hon Pui, William E. Evans, Mary V. Relling

**Collection and assembly of data:** Lisa R. Treviño, Noriko Shimasaki

**Data analysis and interpretation:** Lisa R. Treviño, Noriko Shimasaki, Wenjian Yang, John C. Panetta, Cheng Cheng, Deqing Pei, Diana Chan, Alex Sparreboom, Kathleen M. Giacomini, William E. Evans, Mary V. Relling

**Manuscript writing:** Lisa R. Treviño, Mary V. Relling

**Final approval of manuscript:** Lisa R. Treviño, Noriko Shimasaki, Wenjian Yang, John C. Panetta, Cheng Cheng, Deqing Pei, Diana Chan, Alex Sparreboom, Kathleen M. Giacomini, Ching-Hon Pui, William E. Evans, Mary V. Relling

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Gudmundsson J, Sulem P, Steinthorsdottir V, et al. Two variants on chromosome 17 confer prostate cancer risk, and the one in TCF2 protects against type 2 diabetes. Nat Genet. 2007;39:977–983. doi: 10.1038/ng2062.  [DOI](https://doi.org/10.1038/ng2062) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17603485/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Two%20variants%20on%20chromosome%2017%20confer%20prostate%20cancer%20risk,%20and%20the%20one%20in%20TCF2%20protects%20against%20type%202%20diabetes&author=J%20Gudmundsson&author=P%20Sulem&author=V%20Steinthorsdottir&volume=39&publication_year=2007&pages=977-983&pmid=17603485&doi=10.1038/ng2062&)

2. Hakonarson H, Grant SF, Bradfield JP, et al. A genome-wide association study identifies KIAA0350 as a type 1 diabetes gene. Nature. 2007;448:591–594. doi: 10.1038/nature06010.  [DOI](https://doi.org/10.1038/nature06010) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17632545/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=A%20genome-wide%20association%20study%20identifies%20KIAA0350%20as%20a%20type%201%20diabetes%20gene&author=H%20Hakonarson&author=SF%20Grant&author=JP%20Bradfield&volume=448&publication_year=2007&pages=591-594&pmid=17632545&doi=10.1038/nature06010&)

3. McPherson R, Pertsemlidis A, Kavaslar N, et al. A common allele on chromosome 9 associated with coronary heart disease. Science. 2007;316:1488–1491. doi: 10.1126/science.1142447.  [DOI](https://doi.org/10.1126/science.1142447) | [PMC free article](/articles/PMC2711874/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17478681/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=A%20common%20allele%20on%20chromosome%209%20associated%20with%20coronary%20heart%20disease&author=R%20McPherson&author=A%20Pertsemlidis&author=N%20Kavaslar&volume=316&publication_year=2007&pages=1488-1491&pmid=17478681&doi=10.1126/science.1142447&)

4. Samani NJ, Erdmann J, Hall AS, et al. Genomewide association analysis of coronary artery disease. N Engl J Med. 2007;357:443–453. doi: 10.1056/NEJMoa072366.  [DOI](https://doi.org/10.1056/NEJMoa072366) | [PMC free article](/articles/PMC2719290/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17634449/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Genomewide%20association%20analysis%20of%20coronary%20artery%20disease&author=NJ%20Samani&author=J%20Erdmann&author=AS%20Hall&volume=357&publication_year=2007&pages=443-453&pmid=17634449&doi=10.1056/NEJMoa072366&)

5. Yeager M, Orr N, Hayes RB, et al. Genome-wide association study of prostate cancer identifies a second risk locus at 8q24. Nat Genet. 2007;39:645–649. doi: 10.1038/ng2022.  [DOI](https://doi.org/10.1038/ng2022) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17401363/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Genome-wide%20association%20study%20of%20prostate%20cancer%20identifies%20a%20second%20risk%20locus%20at%208q24&author=M%20Yeager&author=N%20Orr&author=RB%20Hayes&volume=39&publication_year=2007&pages=645-649&pmid=17401363&doi=10.1038/ng2022&)

6. Goldstein DB. Pharmacogenetics in the laboratory and the clinic. N Engl J Med. 2003;348:553–556. doi: 10.1056/NEJMe020173.  [DOI](https://doi.org/10.1056/NEJMe020173) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12571264/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Pharmacogenetics%20in%20the%20laboratory%20and%20the%20clinic&author=DB%20Goldstein&volume=348&publication_year=2003&pages=553-556&pmid=12571264&doi=10.1056/NEJMe020173&)

7. Evans WE, McLeod HL. Pharmacogenomics–drug disposition, drug targets, and side effects. N Engl J Med. 2003;348:538–549. doi: 10.1056/NEJMra020526.  [DOI](https://doi.org/10.1056/NEJMra020526) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12571262/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Pharmacogenomics%E2%80%93drug%20disposition,%20drug%20targets,%20and%20side%20effects&author=WE%20Evans&author=HL%20McLeod&volume=348&publication_year=2003&pages=538-549&pmid=12571262&doi=10.1056/NEJMra020526&)

8. Evans WE, Relling MV. Moving towards individualized medicine with pharmacogenomics. Nature. 2004;429:464–468. doi: 10.1038/nature02626.  [DOI](https://doi.org/10.1038/nature02626) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15164072/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Moving%20towards%20individualized%20medicine%20with%20pharmacogenomics&author=WE%20Evans&author=MV%20Relling&volume=429&publication_year=2004&pages=464-468&pmid=15164072&doi=10.1038/nature02626&)

9. Meyer UA. Pharmacogenetics and adverse drug reactions. Lancet. 2000;356:1667–1671. doi: 10.1016/S0140-6736(00)03167-6.  [DOI](https://doi.org/10.1016/S0140-6736(00)03167-6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11089838/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Pharmacogenetics%20and%20adverse%20drug%20reactions&author=UA%20Meyer&volume=356&publication_year=2000&pages=1667-1671&pmid=11089838&doi=10.1016/S0140-6736(00)03167-6&)

10. Roden DM, Altman RB, Benowitz NL, et al. Pharmacogenomics: Challenges and opportunities. Ann Intern Med. 2006;145:749–757. doi: 10.7326/0003-4819-145-10-200611210-00007.  [DOI](https://doi.org/10.7326/0003-4819-145-10-200611210-00007) | [PMC free article](/articles/PMC5006954/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17116919/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Intern%20Med&title=Pharmacogenomics:%20Challenges%20and%20opportunities&author=DM%20Roden&author=RB%20Altman&author=NL%20Benowitz&volume=145&publication_year=2006&pages=749-757&pmid=17116919&doi=10.7326/0003-4819-145-10-200611210-00007&)

11. Jolivet J, Cowan KH, Curt GA, et al. The pharmacology and clinical use of methotrexate. N Engl J Med. 1983;309:1094–1104. doi: 10.1056/NEJM198311033091805.  [DOI](https://doi.org/10.1056/NEJM198311033091805) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/6353235/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=The%20pharmacology%20and%20clinical%20use%20of%20methotrexate&author=J%20Jolivet&author=KH%20Cowan&author=GA%20Curt&volume=309&publication_year=1983&pages=1094-1104&pmid=6353235&doi=10.1056/NEJM198311033091805&)

12. Pui CH, Relling MV, Downing JR. Acute lymphoblastic leukemia. N Engl J Med. 2004;350:1535–1548. doi: 10.1056/NEJMra023001.  [DOI](https://doi.org/10.1056/NEJMra023001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15071128/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Acute%20lymphoblastic%20leukemia&author=CH%20Pui&author=MV%20Relling&author=JR%20Downing&volume=350&publication_year=2004&pages=1535-1548&pmid=15071128&doi=10.1056/NEJMra023001&)

13. Olsen NJ, Stein CM. New drugs for rheumatoid arthritis. N Engl J Med. 2004;350:2167–2179. doi: 10.1056/NEJMra032906.  [DOI](https://doi.org/10.1056/NEJMra032906) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15152062/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=New%20drugs%20for%20rheumatoid%20arthritis&author=NJ%20Olsen&author=CM%20Stein&volume=350&publication_year=2004&pages=2167-2179&pmid=15152062&doi=10.1056/NEJMra032906&)

14. Wessels JA, van der Kooij SM, le Cessie S, et al. A clinical pharmacogenetic model to predict the efficacy of methotrexate monotherapy in recent-onset rheumatoid arthritis. Arthritis Rheum. 2007;56:1765–1775. doi: 10.1002/art.22640.  [DOI](https://doi.org/10.1002/art.22640) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17530705/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arthritis%20Rheum&title=A%20clinical%20pharmacogenetic%20model%20to%20predict%20the%20efficacy%20of%20methotrexate%20monotherapy%20in%20recent-onset%20rheumatoid%20arthritis&author=JA%20Wessels&author=SM%20van%20der%20Kooij&author=S%20le%20Cessie&volume=56&publication_year=2007&pages=1765-1775&pmid=17530705&doi=10.1002/art.22640&)

15. Niemeyer CM, Gelber RD, Tarbell NJ, et al. Low-dose versus high-dose methotrexate during remission induction in childhood acute lymphoblastic leukemia (protocol 81-01 update) Blood. 1991;78:2514–2519.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/1824248/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Low-dose%20versus%20high-dose%20methotrexate%20during%20remission%20induction%20in%20childhood%20acute%20lymphoblastic%20leukemia%20(protocol%2081-01%20update)&author=CM%20Niemeyer&author=RD%20Gelber&author=NJ%20Tarbell&volume=78&publication_year=1991&pages=2514-2519&pmid=1824248&)

16. Evans WE, Crom WR, Stewart CF, et al. Methotrexate systemic clearance influences probability of relapse in children with standard-risk acute lymphocytic leukaemia. Lancet. 1984;1:359–362. doi: 10.1016/s0140-6736(84)90411-2.  [DOI](https://doi.org/10.1016/s0140-6736(84)90411-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/6141424/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Methotrexate%20systemic%20clearance%20influences%20probability%20of%20relapse%20in%20children%20with%20standard-risk%20acute%20lymphocytic%20leukaemia&author=WE%20Evans&author=WR%20Crom&author=CF%20Stewart&volume=1&publication_year=1984&pages=359-362&pmid=6141424&doi=10.1016/s0140-6736(84)90411-2&)

17. Evans WE, Crom WR, Abromowitch M, et al. Clinical pharmacodynamics of high-dose methotrexate in acute lymphocytic leukemia. Identification of a relation between concentration and effect. N Engl J Med. 1986;314:471–477. doi: 10.1056/NEJM198602203140803.  [DOI](https://doi.org/10.1056/NEJM198602203140803) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/3456079/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Clinical%20pharmacodynamics%20of%20high-dose%20methotrexate%20in%20acute%20lymphocytic%20leukemia.%20Identification%20of%20a%20relation%20between%20concentration%20and%20effect&author=WE%20Evans&author=WR%20Crom&author=M%20Abromowitch&volume=314&publication_year=1986&pages=471-477&pmid=3456079&doi=10.1056/NEJM198602203140803&)

18. Camitta B, Leventhal B, Lauer S, et al. Intermediate-dose intravenous methotrexate and mercaptopurine therapy for non-T, non-B acute lymphocytic leukemia of childhood: A Pediatric Oncology Group study. J Clin Oncol. 1989;7:1539–1544. doi: 10.1200/JCO.1989.7.10.1539.  [DOI](https://doi.org/10.1200/JCO.1989.7.10.1539) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2778483/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Intermediate-dose%20intravenous%20methotrexate%20and%20mercaptopurine%20therapy%20for%20non-T,%20non-B%20acute%20lymphocytic%20leukemia%20of%20childhood:%20A%20Pediatric%20Oncology%20Group%20study&author=B%20Camitta&author=B%20Leventhal&author=S%20Lauer&volume=7&publication_year=1989&pages=1539-1544&pmid=2778483&doi=10.1200/JCO.1989.7.10.1539&)

19. Evans WE, Relling MV, Rodman JH, et al. Conventional compared with individualized chemotherapy for childhood acute lymphoblastic leukemia. N Engl J Med. 1998;338:499–505. doi: 10.1056/NEJM199802193380803.  [DOI](https://doi.org/10.1056/NEJM199802193380803) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9468466/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Conventional%20compared%20with%20individualized%20chemotherapy%20for%20childhood%20acute%20lymphoblastic%20leukemia&author=WE%20Evans&author=MV%20Relling&author=JH%20Rodman&volume=338&publication_year=1998&pages=499-505&pmid=9468466&doi=10.1056/NEJM199802193380803&)

20. Abelson HT, Fosburg MT, Beardsley GP, et al. Methotrexate-induced renal impairment: Clinical studies and rescue from systemic toxicity with high-dose leucovorin and thymidine. J Clin Oncol. 1983;1:208–216. doi: 10.1200/JCO.1983.1.3.208.  [DOI](https://doi.org/10.1200/JCO.1983.1.3.208) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/6607976/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Methotrexate-induced%20renal%20impairment:%20Clinical%20studies%20and%20rescue%20from%20systemic%20toxicity%20with%20high-dose%20leucovorin%20and%20thymidine&author=HT%20Abelson&author=MT%20Fosburg&author=GP%20Beardsley&volume=1&publication_year=1983&pages=208-216&pmid=6607976&doi=10.1200/JCO.1983.1.3.208&)

21. Jaffe N, Gorlick R. High-dose methotrexate in osteosarcoma: Let the questions surcease–time for final acceptance. J Clin Oncol. 2008;26:4365–4366. doi: 10.1200/JCO.2007.14.7793.  [DOI](https://doi.org/10.1200/JCO.2007.14.7793) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18802144/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=High-dose%20methotrexate%20in%20osteosarcoma:%20Let%20the%20questions%20surcease%E2%80%93time%20for%20final%20acceptance&author=N%20Jaffe&author=R%20Gorlick&volume=26&publication_year=2008&pages=4365-4366&pmid=18802144&doi=10.1200/JCO.2007.14.7793&)

22. Relling MV, Fairclough D, Ayers D, et al. Patient characteristics associated with high-risk methotrexate concentrations and toxicity. J Clin Oncol. 1994;12:1667–1672. doi: 10.1200/JCO.1994.12.8.1667.  [DOI](https://doi.org/10.1200/JCO.1994.12.8.1667) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8040679/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Patient%20characteristics%20associated%20with%20high-risk%20methotrexate%20concentrations%20and%20toxicity&author=MV%20Relling&author=D%20Fairclough&author=D%20Ayers&volume=12&publication_year=1994&pages=1667-1672&pmid=8040679&doi=10.1200/JCO.1994.12.8.1667&)

23. Woessmann W, Seidemann K, Mann G, et al. The impact of the methotrexate administration schedule and dose in the treatment of children and adolescents with B-cell neoplasms: A report of the BFM Group Study NHL-BFM95. Blood. 2005;105:948–958. doi: 10.1182/blood-2004-03-0973.  [DOI](https://doi.org/10.1182/blood-2004-03-0973) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15486066/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=The%20impact%20of%20the%20methotrexate%20administration%20schedule%20and%20dose%20in%20the%20treatment%20of%20children%20and%20adolescents%20with%20B-cell%20neoplasms:%20A%20report%20of%20the%20BFM%20Group%20Study%20NHL-BFM95&author=W%20Woessmann&author=K%20Seidemann&author=G%20Mann&volume=105&publication_year=2005&pages=948-958&pmid=15486066&doi=10.1182/blood-2004-03-0973&)

24. Laverdière C, Chiasson S, Costea I, et al. Polymorphism G80A in the reduced folate carrier gene and its relationship to methotrexate plasma levels and outcome of childhood acute lymphoblastic leukemia. Blood. 2002;100:3832–3834. doi: 10.1182/blood.V100.10.3832.  [DOI](https://doi.org/10.1182/blood.V100.10.3832) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12411325/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Polymorphism%20G80A%20in%20the%20reduced%20folate%20carrier%20gene%20and%20its%20relationship%20to%20methotrexate%20plasma%20levels%20and%20outcome%20of%20childhood%20acute%20lymphoblastic%20leukemia&author=C%20Laverdi%C3%A8re&author=S%20Chiasson&author=I%20Costea&volume=100&publication_year=2002&pages=3832-3834&pmid=12411325&doi=10.1182/blood.V100.10.3832&)

25. de Jonge R, Hooijberg JH, van Zelst BD, et al. Effect of polymorphisms in folate-related genes on in vitro methotrexate sensitivity in pediatric acute lymphoblastic leukemia. Blood. 2005;106:717–720. doi: 10.1182/blood-2004-12-4941.  [DOI](https://doi.org/10.1182/blood-2004-12-4941) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15797993/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Effect%20of%20polymorphisms%20in%20folate-related%20genes%20on%20in%20vitro%20methotrexate%20sensitivity%20in%20pediatric%20acute%20lymphoblastic%20leukemia&author=R%20de%20Jonge&author=JH%20Hooijberg&author=BD%20van%20Zelst&volume=106&publication_year=2005&pages=717-720&pmid=15797993&doi=10.1182/blood-2004-12-4941&)

26. Kishi S, Cheng C, French D, et al. Ancestry and pharmacogenetics of antileukemic drug toxicity. Blood. 2007;109:4151–4157. doi: 10.1182/blood-2006-10-054528.  [DOI](https://doi.org/10.1182/blood-2006-10-054528) | [PMC free article](/articles/PMC1885506/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17264302/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Ancestry%20and%20pharmacogenetics%20of%20antileukemic%20drug%20toxicity&author=S%20Kishi&author=C%20Cheng&author=D%20French&volume=109&publication_year=2007&pages=4151-4157&pmid=17264302&doi=10.1182/blood-2006-10-054528&)

27. Pui CH, Sandlund JT, Pei D, et al. Improved outcome for children with acute lymphoblastic leukemia: Results of Total Therapy Study XIIIB at St Jude Children's Research Hospital. Blood. 2004;104:2690–2696. doi: 10.1182/blood-2004-04-1616.  [DOI](https://doi.org/10.1182/blood-2004-04-1616) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15251979/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Improved%20outcome%20for%20children%20with%20acute%20lymphoblastic%20leukemia:%20Results%20of%20Total%20Therapy%20Study%20XIIIB%20at%20St%20Jude%20Children's%20Research%20Hospital&author=CH%20Pui&author=JT%20Sandlund&author=D%20Pei&volume=104&publication_year=2004&pages=2690-2696&pmid=15251979&doi=10.1182/blood-2004-04-1616&)

28. Pui CH, Campana D, Pei D, et al. Treating childhood acute lymphoblastic leukemia without cranial irradiation. N Engl J Med. 2009;360:2730–2741. doi: 10.1056/NEJMoa0900386.  [DOI](https://doi.org/10.1056/NEJMoa0900386) | [PMC free article](/articles/PMC2754320/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19553647/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Treating%20childhood%20acute%20lymphoblastic%20leukemia%20without%20cranial%20irradiation&author=CH%20Pui&author=D%20Campana&author=D%20Pei&volume=360&publication_year=2009&pages=2730-2741&pmid=19553647&doi=10.1056/NEJMoa0900386&)

29. Kishi S, Griener J, Cheng C, et al. Homocysteine, pharmacogenetics, and neurotoxicity in children with leukemia. J Clin Oncol. 2003;21:3084–3091. doi: 10.1200/JCO.2003.07.056.  [DOI](https://doi.org/10.1200/JCO.2003.07.056) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12915598/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Homocysteine,%20pharmacogenetics,%20and%20neurotoxicity%20in%20children%20with%20leukemia&author=S%20Kishi&author=J%20Griener&author=C%20Cheng&volume=21&publication_year=2003&pages=3084-3091&pmid=12915598&doi=10.1200/JCO.2003.07.056&)

30. Pauley JL, Panetta JC, Schmidt J, et al. Late-onset delayed excretion of methotrexate. Cancer Chemother Pharmacol. 2004;54:146–152. doi: 10.1007/s00280-004-0797-y.  [DOI](https://doi.org/10.1007/s00280-004-0797-y) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15148625/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Chemother%20Pharmacol&title=Late-onset%20delayed%20excretion%20of%20methotrexate&author=JL%20Pauley&author=JC%20Panetta&author=J%20Schmidt&volume=54&publication_year=2004&pages=146-152&pmid=15148625&doi=10.1007/s00280-004-0797-y&)

31. Scheet P, Stephens M. A fast and flexible statistical model for large-scale population genotype data: Applications to inferring missing genotypes and haplotypic phase. Am J Hum Genet. 2006;78:629–644. doi: 10.1086/502802.  [DOI](https://doi.org/10.1086/502802) | [PMC free article](/articles/PMC1424677/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16532393/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=A%20fast%20and%20flexible%20statistical%20model%20for%20large-scale%20population%20genotype%20data:%20Applications%20to%20inferring%20missing%20genotypes%20and%20haplotypic%20phase&author=P%20Scheet&author=M%20Stephens&volume=78&publication_year=2006&pages=629-644&pmid=16532393&doi=10.1086/502802&)

32. Price AL, Patterson NJ, Plenge RM, et al. Principal components analysis corrects for stratification in genome-wide association studies. Nat Genet. 2006;38:904–909. doi: 10.1038/ng1847.  [DOI](https://doi.org/10.1038/ng1847) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16862161/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Principal%20components%20analysis%20corrects%20for%20stratification%20in%20genome-wide%20association%20studies&author=AL%20Price&author=NJ%20Patterson&author=RM%20Plenge&volume=38&publication_year=2006&pages=904-909&pmid=16862161&doi=10.1038/ng1847&)

33. Pinheiro JC, Bates DM. Mixed-Effects Models in S and S-Plus. 2000  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mixed-Effects%20Models%20in%20S%20and%20S-Plus&author=JC%20Pinheiro&author=DM%20Bates&publication_year=2000&)

34. Gray RJ. A class of K-sample tests for comparing the cumulative incidence of a competing risk. Annals Stat. 1988;16:1141–1154.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annals%20Stat&title=A%20class%20of%20K-sample%20tests%20for%20comparing%20the%20cumulative%20incidence%20of%20a%20competing%20risk&author=RJ%20Gray&volume=16&publication_year=1988&pages=1141-1154&)

35. Krajinovic M, Moghrabi A. Pharmacogenetics of methotrexate. Pharmacogenomics. 2004;5:819–834. doi: 10.1517/14622416.5.7.819.  [DOI](https://doi.org/10.1517/14622416.5.7.819) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15469405/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Pharmacogenetics%20of%20methotrexate&author=M%20Krajinovic&author=A%20Moghrabi&volume=5&publication_year=2004&pages=819-834&pmid=15469405&doi=10.1517/14622416.5.7.819&)

36. Ansari M, Krajinovic M. Pharmacogenomics in cancer treatment defining genetic bases for inter-individual differences in responses to chemotherapy. Curr Opin Pediatr. 2007;19:15–22. doi: 10.1097/MOP.0b013e3280140613.  [DOI](https://doi.org/10.1097/MOP.0b013e3280140613) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17224657/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Opin%20Pediatr&title=Pharmacogenomics%20in%20cancer%20treatment%20defining%20genetic%20bases%20for%20inter-individual%20differences%20in%20responses%20to%20chemotherapy&author=M%20Ansari&author=M%20Krajinovic&volume=19&publication_year=2007&pages=15-22&pmid=17224657&doi=10.1097/MOP.0b013e3280140613&)

37. Cheok MH, Evans WE. Acute lymphoblastic leukaemia: A model for the pharmacogenomics of cancer therapy 2. Nat Rev Cancer. 2006;6:117–129. doi: 10.1038/nrc1800.  [DOI](https://doi.org/10.1038/nrc1800) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16491071/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Cancer&title=Acute%20lymphoblastic%20leukaemia:%20A%20model%20for%20the%20pharmacogenomics%20of%20cancer%20therapy%202&author=MH%20Cheok&author=WE%20Evans&volume=6&publication_year=2006&pages=117-129&pmid=16491071&doi=10.1038/nrc1800&)

38. Kager L, Evans WE. Pharmacogenomics of acute lymphoblastic leukemia. Curr Opin Hematol. 2006;13:260–265. doi: 10.1097/01.moh.0000231424.46148.f9.  [DOI](https://doi.org/10.1097/01.moh.0000231424.46148.f9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16755223/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Opin%20Hematol&title=Pharmacogenomics%20of%20acute%20lymphoblastic%20leukemia&author=L%20Kager&author=WE%20Evans&volume=13&publication_year=2006&pages=260-265&pmid=16755223&doi=10.1097/01.moh.0000231424.46148.f9&)

39. Abe T, Kakyo M, Tokui T, et al. Identification of a novel gene family encoding human liver-specific organic anion transporter LST-1. J Biol Chem. 1999;274:17159–17163. doi: 10.1074/jbc.274.24.17159.  [DOI](https://doi.org/10.1074/jbc.274.24.17159) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10358072/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Identification%20of%20a%20novel%20gene%20family%20encoding%20human%20liver-specific%20organic%20anion%20transporter%20LST-1&author=T%20Abe&author=M%20Kakyo&author=T%20Tokui&volume=274&publication_year=1999&pages=17159-17163&pmid=10358072&doi=10.1074/jbc.274.24.17159&)

40. Marzolini C, Tirona RG, Gervasini G, et al. A common polymorphism in the bile acid receptor farnesoid X receptor is associated with decreased hepatic target gene expression 10. Mol Endocrinol. 2007;21:1769–1780. doi: 10.1210/me.2007-0025.  [DOI](https://doi.org/10.1210/me.2007-0025) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17519356/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Endocrinol&title=A%20common%20polymorphism%20in%20the%20bile%20acid%20receptor%20farnesoid%20X%20receptor%20is%20associated%20with%20decreased%20hepatic%20target%20gene%20expression%2010&author=C%20Marzolini&author=RG%20Tirona&author=G%20Gervasini&volume=21&publication_year=2007&pages=1769-1780&pmid=17519356&doi=10.1210/me.2007-0025&)

41. Glaeser H, Bailey DG, Dresser GK, et al. Intestinal drug transporter expression and the impact of grapefruit juice in humans. Clin Pharmacol Ther. 2007;81:362–370. doi: 10.1038/sj.clpt.6100056.  [DOI](https://doi.org/10.1038/sj.clpt.6100056) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17215845/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Intestinal%20drug%20transporter%20expression%20and%20the%20impact%20of%20grapefruit%20juice%20in%20humans&author=H%20Glaeser&author=DG%20Bailey&author=GK%20Dresser&volume=81&publication_year=2007&pages=362-370&pmid=17215845&doi=10.1038/sj.clpt.6100056&)

42. Abe T, Unno M, Onogawa T, et al. LST-2, a human liver-specific organic anion transporter, determines methotrexate sensitivity in gastrointestinal cancers. Gastroenterology. 2001;120:1689–1699. doi: 10.1053/gast.2001.24804.  [DOI](https://doi.org/10.1053/gast.2001.24804) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11375950/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastroenterology&title=LST-2,%20a%20human%20liver-specific%20organic%20anion%20transporter,%20determines%20methotrexate%20sensitivity%20in%20gastrointestinal%20cancers&author=T%20Abe&author=M%20Unno&author=T%20Onogawa&volume=120&publication_year=2001&pages=1689-1699&pmid=11375950&doi=10.1053/gast.2001.24804&)

43. Tirona RG, Leake BF, Merino G, et al. Polymorphisms in OATP-C: Identification of multiple allelic variants associated with altered transport activity among European- and African-Americans. J Biol Chem. 2001;276:35669–35675. doi: 10.1074/jbc.M103792200.  [DOI](https://doi.org/10.1074/jbc.M103792200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11477075/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Polymorphisms%20in%20OATP-C:%20Identification%20of%20multiple%20allelic%20variants%20associated%20with%20altered%20transport%20activity%20among%20European-%20and%20African-Americans&author=RG%20Tirona&author=BF%20Leake&author=G%20Merino&volume=276&publication_year=2001&pages=35669-35675&pmid=11477075&doi=10.1074/jbc.M103792200&)

44. Niemi M, Schaeffeler E, Lang T, et al. High plasma pravastatin concentrations are associated with single nucleotide polymorphisms and haplotypes of organic anion transporting polypeptide-C (OATP-C, SLCO1B1) Pharmacogenetics. 2004;14:429–440. doi: 10.1097/01.fpc.0000114750.08559.32.  [DOI](https://doi.org/10.1097/01.fpc.0000114750.08559.32) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15226675/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=High%20plasma%20pravastatin%20concentrations%20are%20associated%20with%20single%20nucleotide%20polymorphisms%20and%20haplotypes%20of%20organic%20anion%20transporting%20polypeptide-C%20(OATP-C,%20SLCO1B1)&author=M%20Niemi&author=E%20Schaeffeler&author=T%20Lang&volume=14&publication_year=2004&pages=429-440&pmid=15226675&doi=10.1097/01.fpc.0000114750.08559.32&)

45. Pasanen MK, Neuvonen M, Neuvonen PJ, et al. SLCO1B1 polymorphism markedly affects the pharmacokinetics of simvastatin acid. Pharmacogenet Genomics. 2006;16:873–879. doi: 10.1097/01.fpc.0000230416.82349.90.  [DOI](https://doi.org/10.1097/01.fpc.0000230416.82349.90) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17108811/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=SLCO1B1%20polymorphism%20markedly%20affects%20the%20pharmacokinetics%20of%20simvastatin%20acid&author=MK%20Pasanen&author=M%20Neuvonen&author=PJ%20Neuvonen&volume=16&publication_year=2006&pages=873-879&pmid=17108811&doi=10.1097/01.fpc.0000230416.82349.90&)

46. Liu L, Cui Y, Chung AY, et al. Vectorial transport of enalapril by Oatp1a1/Mrp2 and OATP1B1 and OATP1B3/MRP2 in rat and human livers. J Pharmacol Exp Ther. 2006;318:395–402. doi: 10.1124/jpet.106.103390.  [DOI](https://doi.org/10.1124/jpet.106.103390) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16627748/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacol%20Exp%20Ther&title=Vectorial%20transport%20of%20enalapril%20by%20Oatp1a1/Mrp2%20and%20OATP1B1%20and%20OATP1B3/MRP2%20in%20rat%20and%20human%20livers&author=L%20Liu&author=Y%20Cui&author=AY%20Chung&volume=318&publication_year=2006&pages=395-402&pmid=16627748&doi=10.1124/jpet.106.103390&)

47. Nozawa T, Minami H, Sugiura S, et al. Role of organic anion transporter OATP1B1 (OATP-C) in hepatic uptake of irinotecan and its active metabolite, 7-ethyl-10-hydroxycamptothecin: In vitro evidence and effect of single nucleotide polymorphisms. Drug Metab Dispos. 2005;33:434–439. doi: 10.1124/dmd.104.001909.  [DOI](https://doi.org/10.1124/dmd.104.001909) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15608127/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Role%20of%20organic%20anion%20transporter%20OATP1B1%20(OATP-C)%20in%20hepatic%20uptake%20of%20irinotecan%20and%20its%20active%20metabolite,%207-ethyl-10-hydroxycamptothecin:%20In%20vitro%20evidence%20and%20effect%20of%20single%20nucleotide%20polymorphisms&author=T%20Nozawa&author=H%20Minami&author=S%20Sugiura&volume=33&publication_year=2005&pages=434-439&pmid=15608127&doi=10.1124/dmd.104.001909&)

48. Kameyama Y, Yamashita K, Kobayashi K, et al. Functional characterization of SLCO1B1 (OATP-C) variants, SLCO1B1*5, SLCO1B1*15 and SLCO1B1*15+ C1007G, by using transient expression systems of HeLa and HEK293 cells. Pharmacogenet Genomics. 2005;15:513–522. doi: 10.1097/01.fpc.0000170913.73780.5f.  [DOI](https://doi.org/10.1097/01.fpc.0000170913.73780.5f) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15970799/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Functional%20characterization%20of%20SLCO1B1%20(OATP-C)%20variants,%20SLCO1B1*5,%20SLCO1B1*15%20and%20SLCO1B1*15+%20C1007G,%20by%20using%20transient%20expression%20systems%20of%20HeLa%20and%20HEK293%20cells&author=Y%20Kameyama&author=K%20Yamashita&author=K%20Kobayashi&volume=15&publication_year=2005&pages=513-522&pmid=15970799&doi=10.1097/01.fpc.0000170913.73780.5f&)

49. Pasanen MK, Fredrikson H, Neuvonen PJ, et al. Different effects of SLCO1B1 polymorphism on the pharmacokinetics of atorvastatin and rosuvastatin. Clin Pharmacol Ther. 2007;82:726–733. doi: 10.1038/sj.clpt.6100220.  [DOI](https://doi.org/10.1038/sj.clpt.6100220) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17473846/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Different%20effects%20of%20SLCO1B1%20polymorphism%20on%20the%20pharmacokinetics%20of%20atorvastatin%20and%20rosuvastatin&author=MK%20Pasanen&author=H%20Fredrikson&author=PJ%20Neuvonen&volume=82&publication_year=2007&pages=726-733&pmid=17473846&doi=10.1038/sj.clpt.6100220&)

50. Ho RH, Choi L, Lee W, et al. Effect of drug transporter genotypes on pravastatin disposition in European- and African-American participants. Pharmacogenet Genomics. 2007;17:647–656. doi: 10.1097/FPC.0b013e3280ef698f.  [DOI](https://doi.org/10.1097/FPC.0b013e3280ef698f) | [PMC free article](/articles/PMC4063287/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17622941/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Effect%20of%20drug%20transporter%20genotypes%20on%20pravastatin%20disposition%20in%20European-%20and%20African-American%20participants&author=RH%20Ho&author=L%20Choi&author=W%20Lee&volume=17&publication_year=2007&pages=647-656&pmid=17622941&doi=10.1097/FPC.0b013e3280ef698f&)

51. Niemi M. Role of OATP transporters in the disposition of drugs. Pharmacogenomics. 2007;8:787–802. doi: 10.2217/14622416.8.7.787.  [DOI](https://doi.org/10.2217/14622416.8.7.787) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18240907/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Role%20of%20OATP%20transporters%20in%20the%20disposition%20of%20drugs&author=M%20Niemi&volume=8&publication_year=2007&pages=787-802&pmid=18240907&doi=10.2217/14622416.8.7.787&)

52. Egan LJ, Sandborn WJ, Mays DC, et al. Systemic and intestinal pharmacokinetics of methotrexate in patients with inflammatory bowel disease. Clin Pharmacol Ther. 1999;65:29–39. doi: 10.1016/S0009-9236(99)70119-0.  [DOI](https://doi.org/10.1016/S0009-9236(99)70119-0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9951428/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Systemic%20and%20intestinal%20pharmacokinetics%20of%20methotrexate%20in%20patients%20with%20inflammatory%20bowel%20disease&author=LJ%20Egan&author=WJ%20Sandborn&author=DC%20Mays&volume=65&publication_year=1999&pages=29-39&pmid=9951428&doi=10.1016/S0009-9236(99)70119-0&)

53. Evans WE, Tsiatis A, Crom WR, et al. Pharmacokinetics of sustained serum methotrexate concentrations secondary to gastrointestinal obstruction. J Pharm Sci. 1981;70:1194–1198. doi: 10.1002/jps.2600701103.  [DOI](https://doi.org/10.1002/jps.2600701103) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7299661/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharm%20Sci&title=Pharmacokinetics%20of%20sustained%20serum%20methotrexate%20concentrations%20secondary%20to%20gastrointestinal%20obstruction&author=WE%20Evans&author=A%20Tsiatis&author=WR%20Crom&volume=70&publication_year=1981&pages=1194-1198&pmid=7299661&doi=10.1002/jps.2600701103&)

54. Connell WR, Kamm MA, Ritchie JK, et al. Bone marrow toxicity caused by azathioprine in inflammatory bowel disease: 27 years of experience. Gut. 1993;34:1081–1085. doi: 10.1136/gut.34.8.1081.  [DOI](https://doi.org/10.1136/gut.34.8.1081) | [PMC free article](/articles/PMC1374358/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8174958/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gut&title=Bone%20marrow%20toxicity%20caused%20by%20azathioprine%20in%20inflammatory%20bowel%20disease:%2027%20years%20of%20experience&author=WR%20Connell&author=MA%20Kamm&author=JK%20Ritchie&volume=34&publication_year=1993&pages=1081-1085&pmid=8174958&doi=10.1136/gut.34.8.1081&)

55. Present DH, Meltzer SJ, Krumholz MP, et al. 6-Mercaptopurine in the management of inflammatory bowel disease: Short- and long-term toxicity. Ann Intern Med. 1989;111:641–649. doi: 10.7326/0003-4819-111-8-641.  [DOI](https://doi.org/10.7326/0003-4819-111-8-641) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2802419/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Intern%20Med&title=6-Mercaptopurine%20in%20the%20management%20of%20inflammatory%20bowel%20disease:%20Short-%20and%20long-term%20toxicity&author=DH%20Present&author=SJ%20Meltzer&author=MP%20Krumholz&volume=111&publication_year=1989&pages=641-649&pmid=2802419&doi=10.7326/0003-4819-111-8-641&)

56. Murry DJ, Synold TW, Pui CH, et al. Renal function and methotrexate clearance in children with newly diagnosed leukemia. Pharmacotherapy. 1995;15:144–149.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/7624260/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacotherapy&title=Renal%20function%20and%20methotrexate%20clearance%20in%20children%20with%20newly%20diagnosed%20leukemia&author=DJ%20Murry&author=TW%20Synold&author=CH%20Pui&volume=15&publication_year=1995&pages=144-149&pmid=7624260&)

57. Cohen JC, Boerwinkle E, Mosley TH, Jr, et al. Sequence variations in PCSK9, low LDL, and protection against coronary heart disease. N Engl J Med. 2006;354:1264–1272. doi: 10.1056/NEJMoa054013.  [DOI](https://doi.org/10.1056/NEJMoa054013) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16554528/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Sequence%20variations%20in%20PCSK9,%20low%20LDL,%20and%20protection%20against%20coronary%20heart%20disease&author=JC%20Cohen&author=E%20Boerwinkle&author=TH%20Mosley&volume=354&publication_year=2006&pages=1264-1272&pmid=16554528&doi=10.1056/NEJMoa054013&)

58. Fotoohi K, Jansen G, Assaraf YG, et al. Disparate mechanisms of antifolate resistance provoked by methotrexate and its metabolite 7-hydroxymethotrexate in leukemia cells: Implications for efficacy of methotrexate therapy. Blood. 2004;104:4194–4201. doi: 10.1182/blood-2004-04-1493.  [DOI](https://doi.org/10.1182/blood-2004-04-1493) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15308564/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Disparate%20mechanisms%20of%20antifolate%20resistance%20provoked%20by%20methotrexate%20and%20its%20metabolite%207-hydroxymethotrexate%20in%20leukemia%20cells:%20Implications%20for%20efficacy%20of%20methotrexate%20therapy&author=K%20Fotoohi&author=G%20Jansen&author=YG%20Assaraf&volume=104&publication_year=2004&pages=4194-4201&pmid=15308564&doi=10.1182/blood-2004-04-1493&)

59. Badagnani I, Castro RA, Taylor TR, et al. Interaction of methotrexate with organic-anion transporting polypeptide 1A2 and its genetic variants. J Pharmacol Exp Ther. 2006;318:521–529. doi: 10.1124/jpet.106.104364.  [DOI](https://doi.org/10.1124/jpet.106.104364) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16702441/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacol%20Exp%20Ther&title=Interaction%20of%20methotrexate%20with%20organic-anion%20transporting%20polypeptide%201A2%20and%20its%20genetic%20variants&author=I%20Badagnani&author=RA%20Castro&author=TR%20Taylor&volume=318&publication_year=2006&pages=521-529&pmid=16702441&doi=10.1124/jpet.106.104364&)
