# Polymorphisms in the VKORC1 gene are strongly associated with warfarin dosage requirements in patients receiving anticoagulation

## Metadata
**Authors:** T Li, L A Lange, X Li, L Susswein, B Bryant, R Malone, E M Lange, T‐Y Huang, D W Stafford, J P Evans
**Journal:** Journal of Medical Genetics
**Date:** 2006 Apr 12
**DOI:** [10.1136/jmg.2005.040410](https://doi.org/10.1136/jmg.2005.040410)
**PMID:** 16611750
**PMCID:** PMC2564574
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2564574/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC2564574/pdf/740.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC2564574/pdf/740.pdf)

## Abstract

**Background:** 
Warfarin is a mainstay of therapy for conditions associated with an increased risk of thromboembolic events. However, the use of this common agent is fraught with complications and little is known regarding inter‐individual variation in warfarin response.

**Objective:** 
We tested for association between single nucleotide polymorphisms (SNPs) in VKORC1 and CYP2C9 and average weekly warfarin dose required to maintain patients at their desired anticoagulation target.

**Methods:** 
The sample consisted of 93 European‐American patients from anticoagulation clinics at the University of North Carolina at Chapel Hill. Data on mean weekly warfarin dose were collected over a mean treatment period of 20.6 months. ANCOVA models were used and haplotype analysis was performed.

**Results:** 
Three of six VKORC1 SNPs were found to be very strongly associated with the average warfarin dose required to achieve the target international normalised ratio (INR; p<0.0001). The mean weekly dose by genotype ranged from approximately 27 to 47 mg. There was no evidence for an association between either of the two CYP2C9 polymorphisms studied, CYP2C9*2 and CYP2C9*3. CYP2C9*3 was significantly (p = 0.05) associated with average warfarin dosage after adjustment for VKORC1*1173.

**Conclusions:** 
These results are of considerable clinical interest and confirm recently published results regarding the role of these two genes in modifying warfarin metabolism and maintenance dosage. The consistent findings regarding the role of VKORC1 and CYP2C9 in warfarin metabolism and maintenance dosage represent a clinically useful proof of principal for the use of pharmacogenomic information in medicine and may lead to improved understanding of warfarin's actions.

Keywords: anticoagulation, CYP2C9, pharmacogenomics, VKORC1, warfarin

### Background

Warfarin is a mainstay of therapy for conditions associated with an increased risk of thromboembolic events. However, the use of this common agent is fraught with complications and little is known regarding inter‐individual variation in warfarin response.

### Objective

We tested for association between single nucleotide polymorphisms (SNPs) in *VKORC1* and *CYP2C9* and average weekly warfarin dose required to maintain patients at their desired anticoagulation target.

### Methods

The sample consisted of 93 European‐American patients from anticoagulation clinics at the University of North Carolina at Chapel Hill. Data on mean weekly warfarin dose were collected over a mean treatment period of 20.6 months. ANCOVA models were used and haplotype analysis was performed.

### Results

Three of six *VKORC1* SNPs were found to be very strongly associated with the average warfarin dose required to achieve the target international normalised ratio (INR; p<0.0001). The mean weekly dose by genotype ranged from approximately 27 to 47 mg. There was no evidence for an association between either of the two *CYP2C9* polymorphisms studied, *CYP2C9*2* and *CYP2C9*3*. *CYP2C9*3* was significantly (p = 0.05) associated with average warfarin dosage after adjustment for *VKORC1**1173.

### Conclusions

These results are of considerable clinical interest and confirm recently published results regarding the role of these two genes in modifying warfarin metabolism and maintenance dosage. The consistent findings regarding the role of *VKORC1* and *CYP2C9* in warfarin metabolism and maintenance dosage represent a clinically useful proof of principal for the use of pharmacogenomic information in medicine and may lead to improved understanding of warfarin's actions.

## Methods

This study was approved by the Biomedical Institutional Review Board at the University of North Carolina at Chapel Hill. Patients attending anticoagulation clinics at UNC who were being treated with warfarin were approached for participation in the current study. After obtaining informed consent, blood was obtained from 93 European‐American patients for DNA analysis. An extensive database is maintained on these patients that includes the indication for treatment, duration of treatment, average dose, dosage adjustments, and international normalised ratio (INR). Mean weekly warfarin dose required to achieve each patient's target INR was gathered from the database over a mean period of treatment of 20.6 months. Genomic DNAs were extracted from whole blood using the QIAamp DNA Blood Mini Kit (Qiagen, Valencia, CA).

### Genomic sequencing

Direct sequencing of 60 patients identified six single nucleotide polymorphisms (SNPs) in the *VKORC1* gene. Approximately 10 ng DNA was used for PCR reactions. The primers used to amplify the 5′‐UTR and exon 1 region were: CCAATCGCCGAGTCAGAGG and CCCAGTCCCCAGCACTGTCT; primers for exon 2 and flanking region were: AGGGGAGGATAGGGTCAGTG and CCTGTTAGTTACCTCCCCACA; and primers for exon 3 and the 3′‐UTR were: ATACGTGCGTAAGCCACCAC and ACCCAGATATGCCCCCTTAG. Sequencing was via high throughput capillary electrophoresis. Genotyping for these SNPs was then performed in the remaining patients.

Assay of known SNPs was via real time PCR. The assay reagents for SNP genotyping were obtained from the Assay‐by‐Design service (Applied Biosystems, Foster City, CA). The primers and probes (FAM and VIC dye‐labelled) were designed using Primer Express software and were synthesised in Applied Biosystems (table 1). PCR reactions used 2× TaqMan Universal PCR Master Mix, No AmpErase UNG (Applied Biosystems). The real time PCR reactions were performed in an Opticon II system (Bio‐Rad Laboratories/MJ Research, Waltham, MA). Conditions were as follows: 95°C 10 min preheat, 92°C 15 s, 60°C 1 min followed by a plate reading, 40 cycles. The results were read according to the signal value of FAM and VIC dye.

### Table 1 The primers and probes used in real time PCR genotyping.

| VKOR SNPs | VIC probe sequence | FAM probe sequence | Forward primer | Reverse primer |
| --- | --- | --- | --- | --- |
| VKOR 497 T>G | CCCCTTCaCCTGCGC | CCCCTTCcCCTGCGC | GCGGTAGAGATTGACGATGGT | GCAGCCATCGCCAACAC |
| VKOR 698 C>T | CAAGGCTgGTATAACG | CAAGGCTaGTATAACG | CTCTGATGCAAAACCGAGTGAAC | GCCCGGCCCTTAAGTAATTCTT |
| VKOR 1173 C>T | CCTAGTCCAAGgGTCGAT | CTAGTCCAAGaGTCGAT | CCCGGTGCCAGGAGATC | CACCTGGGCTATCCTCTGTTC |
| VKOR 1542 G>C | TCATCACgGAGCGTC | TCATCACcGAGCGTC | GGTGATCCACACAGCTGACA | CCTGTTAGTTACCTCCCCACATC |
| VKOR 2255 T>C | CCAGGACCaTGGTGC | CCAGGACCgTGGTGC | GCTCCAGAGAAGGCATCACT | GCCAAGTCTGAACCATGTGTCA |
| VKOR 3730 G>A | ATACCCgCACATGAC | CATACCCaCACATGAC | GTCCCTAGAAGGCCCTAGATGT | GTGTGGCACATTTGGTCCATT |
### Statistical methods

Each of the SNPs was first assessed to determine if the observed genotype frequencies were consistent with expected Hardy‐Weinberg proportions using Pearson's χ^2^ tests. Pair‐wise marker‐marker linkage disequilibrium was assessed using Lewontin's D′ statistic[14](#ref14) and Devlin and Risch's Δ^2^ statistic[15](#ref15) as implemented in the computer program GOLD ([www.sph.umich.edu/csg/abecasis/GOLD](http://www.sph.umich.edu/csg/abecasis/GOLD)).

To test for association between individual SNPs and the continuously distributed outcomes, average warfarin dose, and average INR, we performed analysis of covariance models (ANCOVA) using PROC GLM within the SAS software system (version 8.0, SAS Institute, Cary, NC). The outcomes were first examined for adherence to distributional assumptions (including approximate normality of error terms conditional on covariates and homoscedasticity); the natural log transformation was subsequently applied to average dosage to adhere to these assumptions. Covariate adjustment was made for age, gender, and target INR in all analytic models. Genotype was tested for general association (no mode of inheritance assumption) using a 2 df F test. A 1 df F test was used to test the null hypotheses of no association for the *CYP2C9* genotypes due to a lack of observed homozygotes for the *CYP2C9*2* or **3* alleles. We additionally examined a two locus model, in which we selected the most statistically significant polymorphism from each of the two genes and tested the effects of each while controlling for the other. Finally, a model testing the interaction effects between the two polymorphisms was performed.

Tests of haplotype effects were performed using a score test developed by Schaid, as implemented in the computer program HAPLO.STAT ([http://mayoresearch.mayo.edu/mayo/research/biostat/schaid.cfm](http://mayoresearch.mayo.edu/mayo/research/biostat/schaid.cfm)). HAPLO.STAT utilises a weighting scheme based on EM derived haplotype frequency estimates and weights every haplotype rather than assigning a “most likely” haplotype to an individual. Statistical differences in overall haplotype frequencies (excluding haplotypes with extremely low frequencies, for example 0.01) were tested, after adjustment for covariates (age, gender, and target INR), for association with the outcomes. Specific individual haplotype effects were also tested.

## Results

The sample consisted of 93 European‐American subjects with both genotype and phenotype data. The mean age was 63 years (SD 16, range 24–90 years) and 31% (n = 29) of the subjects were female. Mean treatment period for this study was 20.6 months (SD 11.0, range 12 days to 37.3 months). Indications for warfarin were predominantly atrial fibrillation or atrial flutter (45%, n = 47), with another large group (26%, n = 24) being treated or prophylaxed for venous thromboembolic events (VTE). Thirteen (14%) of the patients were receiving warfarin due to prosthetic mitral or aortic valves. The target INR for the majority of patients (83%, n = 77) was 2–3. Three of the prophylaxed VTE patients had a goal of 1.5–2.0. Thirteen patients had a goal of 2.5–3.5; this group included those with a prosthestic mitral valve as well as those with both a prosthetic atrial valve and atrial fibrillation. The positions and the corresponding observed allele frequencies of the identified SNPs in all 93 patients are given in table 2. The observed distributions of genotype data for the *VKORC1* and *CYP2C9* SNPs were consistent with expected Hardy‐Weinberg proportions (data not shown).

### Table 2 VKORC1 and CYP2C9 SNP major allele frequencies.

| SNPs | Position | NCBI dbSNP rs# | Frequency (%) of wild type allele |
| --- | --- | --- | --- |
| VKORC1*497 T>G | Intron 1 | Rs2884737 | 68.8 |
| VKORC1*698 C>T | Intron 1 | Rs17708472 | 79.6 |
| VKORC1*1173 C>T | Intron 1 | Rs9934438 | 54.8 |
| VKORC1*1542 G>C | Intron 2 | Rs8050894 | 56.5 |
| VKORC1*2255 T>C | Intron 2 | Rs2359612 | 44.1 |
| VKORC1*3730 G>A | 3′‐UTR | Rs7294 | 64.5 |
| CYP2C9*2 | Exon 3, C144R | Rs1799853 | 92.5 |
| CYP2C9*3 | Exon 7, L359I | Rs1057910 | 93.0 |

Table 2 Caption: Six SNPs were identified in the VKORC1 gene upon sequencing and real time PCR among 93 individuals. The location and allelic frequency of each SNP is shown.

Table 3 displays the association results of the six *VKORC1* polymorphisms with mean weekly warfarin dose and INR in these patients. As expected, the observed INRs cluster tightly around 2.5 since warfarin doses were continually adjusted clinically in order to achieve an appropriate level of anticoagulation, with an INR of 2–3 being the most common target. Five of the six *VKORC1* SNPs were found to be statistically associated with average warfarin dose required to achieve the target INR (p ranging from <0.0001 to 0.021). The *VKORC1**1173, *VKORC1**1542, and *VKORC1**2255 SNPs were found to be the most strongly associated with the average warfarin dose required to achieve the target INR (p<0.0001). The trend in mean warfarin dosage by all three genotypes is consistent with an additive mode of inheritance, where the mean for the heterozygote is roughly midway between the two homozygotes

### Table 3 Individual VKORC1 gene SNP association analysis results.

| VKORC1 gene SNPs | n | Average (least square means) weekly warfarin dose (mg) | INR |
| --- | --- | --- | --- |
| 497 T>G |   |   |   |
| TT | 43 | 43.6 | 2.5 |
| TG | 42 | 34.4 | 2.7 |
| GG | 8 | 24.9 | 2.7 |
| p value |   | 0.0015 | 0.12 |
| 698 C>T |   |   |   |
| CC | 59 | 35.2 | 2.7 |
| CT | 30 | 40.0 | 2.5 |
| TT | 4 | 54.7 | 2.4 |
| p value |   | 0.056 | 0.18 |
| 1173 C>T |   |   |   |
| CC | 31 | 47.1 | 2.5 |
| CT | 40 | 35.8 | 2.6 |
| TT | 22 | 26.9 | 2.7 |
| p value |   | <0.0001 | 0.10 |
| 1542 G>C |   |   |   |
| GG | 32 | 46.4 | 2.5 |
| GC | 41 | 36.0 | 2.6 |
| CC | 20 | 26.5 | 2.7 |
| p value |   | <0.0001 | 0.11 |
| 2255 T>C |   |   |   |
| TT | 20 | 26.5 | 2.7 |
| TC | 42 | 35.7 | 2.6 |
| CC | 31 | 47.1 | 2.5 |
| p value |   | <0.0001 | 0.11 |
| 3730 G>A |   |   |   |
| GG | 40 | 33.3 | 2.6 |
| GA | 40 | 39.7 | 2.6 |
| AA | 13 | 46.4 | 2.5 |
| p value |   | 0.021 | 0.63 |

Table 3 Caption: Least‐square means are on the untransformed data and p values were obtained using the log transformation. All models adjusted for age and gender. Models for average weekly dose of warfarin additionally adjusted for target INR.

The *VKORC1**1173, *1542, and *2255 SNPs were found to be in very strong linkage disequilibrium (LD) with each other, with estimated D′ values of 1.00 and estimated Δ^2^ values ranging from 0.94 to 0.98. Estimated D′ values ranged from 0.84 to 1.00 and estimated Δ^2 ^values ranged from 0.11 to 0.56 for all other SNP pair combinations. Haplotype analysis results are reported in table 4. Four haplotypes were observed with an estimated frequency greater than 3%. The permutation based global p value for any difference between haplotypes was 0.0004. All four of the individual haplotype specific tests were found to be statistically significant (p<0.05). The two haplotypes (EA1 and EA2) characterised by the T allele for *VKOR* 1173, the C allele for *VKOR* 1542, and the T allele for *VKOR* 2255 were associated with negative score statistics (indicating lower mean dosage), while the other two haplotypes were associated with positive score statistics (indicating higher mean dosage). Overall, the haplotype results are consistent with the single SNP results and suggest that the causal polymorphism is one of the identified SNPs (*VKORC1**1173, *1542, or ***2255) or another polymorphism in LD with these three SNPs.

### Table 4 Association between VKORC1 gene haplotypes and log average dose.

| Haplotype | VKORC1 gene SNPs |  |  |  |  |  |   | Score* | p† |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| 497 | 698 | 1173 | 1542 | 2255 | 3730 | Estimated frequency (%) |  |  |  |
| EA1 | G | C | T | C | T | G | 30.1 | −3.42 | 0.00063 |
| EA2 | T | C | T | C | T | G | 12.9 | −2.26 | 0.024 |
| EA3 | T | T | C | G | C | G | 19.3 | 2.47 | 0.013 |
| EA4 | T | C | C | G | C | A | 34.9 | 2.73 | 0.0063 |
| Global‡ |   |   |   |   |   |   |   |   | 0.0004 |

Table 4 Caption: *HAPLO.STAT (http://mayoresearch.mayo.edu/mayo/research/biostat/schaid.cfm) program statistic; †haplotype specific empirical p values comparing each specific haplotype to all others combined; ‡simulated p value for global association test.

The *CYP2C9* SNP association results are presented in table 5. Only 14 and 13 heterozygotes were observed for *CYP2C9*2* and *CYP2C9*3* polymorphisms, respectively. There was no evidence for an association between *CYP2C9* genotypes and average warfarin weekly dose before additional adjustment for *VKORC1* genotype. *CYP2C9*3* became marginally statistically significant (p = 0.050) after adjustment for *VKOR 1173*. The *CYP2C9*2* and *CYP2C9*3* SNPs were not found to be in significant LD with one another (D′ = 0.04, Δ^2^ = 0.00, p = 0.97). Given the lack of LD between the two SNPs, haplotype analyses were not performed.

### Table 5 Association between CYP2C9 genotypes and log average dose in European‐American subjects.

| CYP2C9 gene SNPs | n | Least square means |  | INR |
| --- | --- | --- | --- | --- |
| Average weekly dose (mg) | Average weekly dose (mg) after adjustment for VKORC1*1173 |  |  |  |
| CYP2C9*2 |   |   |   |   |
| 11 | 79 | 38.8 | 37.3 | 2.6 |
| 12 | 14 | 32.1 | 31.0 | 2.6 |
| 22 | 0 | – | – | – |
| p value |   | 0.19 | 0.16 | 0.93 |
| CYP2C9*3 |   |   |   |   |
| 11 | 80 | 38.5 | 37.1 | 2.6 |
| 12 | 13 | 33.0 | 31.5 | 2.7 |
| 22 | 0 | – | – | – |
| p value |   | 0.24 | 0.050 | 0.12 |

Table 5 Caption: All models adjusted for age and gender. Models for average weekly dose of warfarin were additionally adjusted for the target INR.

## Discussion

We have here shown that variants within *VKORC1* are strongly associated with the mean weekly dose of warfarin required to maintain a desired target INR in a sample of European‐American outpatients undergoing anticoagulation treatment. In addition to their robust statistical significance, the results presented here are potentially clinically meaningful given the notoriously narrow therapeutic window[16](#ref16) of the drug in question. The mean weekly dose of warfarin required to maintain the desired target INR is almost doubled (from 27 to 47 mg) for those who are homozygous C/C at position 1173, G/G at position 1542, or C/C at position 2255, as compared with the homozygotes for the opposite allele at each site. Also bolstering the significance of these results is the dose‐response curve observed in heterozygotes, who demonstrate intermediate warfarin sensitivity. Moreover, these alleles are common in this unselected sample with an estimated prevalence of 54.2% for the *VKORC1**1173 C allele, 55.7% for the *VKORC1**1542 G allele, and 54.7% for the *VKORC1**2255 C allele. These results are strongly consistent with recent findings regarding the effects of *VKORC1* genotype on warfarin dosage levels required to reach desired INR both in terms of overall statistical significance and mean warfarin dosage levels by genotype.[7](#ref7),[8](#ref8),[9](#ref9),[10](#ref10),[11](#ref11),[12](#ref12) For example, D'Andrea *et al*[7](#ref7) report an observed warfarin mean weekly dosage of 49.0, 35.7, and 25.9 (calculated by multiplying their reported mean daily dosage by 7) for *VKORC1**1173 genotypes C/C, C/T, and T/T, respectively, as compared with our observed weekly mean dosages of 47.1, 35.8, and 26.9, respectively, for the same genotypes. In addition, there is considerable agreement with the direction of the haplotype based associations. In our study, EA1 and EA2 are associated with lower mean levels of warfarin dosage and haplotypes EA3 and EA4 are associated with higher mean levels of warfarin dosage. Our haplotyes EA1 and EA2 have identical alleles as do haplotypes H3a and H3b in Wadelius *et al*[10](#ref10) for *VKORC1**1173, *2255, and *3730. Our haplotype EA3 corresponds to H2 and our haplotype EA4 corresponds to H1 in Wadelius *et al*,[10](#ref10) where H1 and H2 are also associated with higher warfarin dosage. Finally, our haplotypes EA1, EA2, EA3, and EA4 correspond to the sets of haplotypes {H2, H5}, {H1, H3}, {H4, H6, H9}, and {H7, H8}, respectively, in Rieder *et al*[8](#ref8); the direction of results between the two studies with respect to which haplotypes are associated with higher or lower warfarin dosage levels is also entirely consistent.

In addition to our *VKORC1* results, we did not detect a statistically significant association between the two *CYP2C9* polymorphisms and warfarin dosage before adjustment for *VKORC1*; however, results for *CYP2C9*3* did reach a marginal level of statistical significance (p = 0.05) after adjustment for *VKORC1**1173. Because the *CYP2C9* SNPs have lower minor allele frequencies than the *VKORC1* SNPs examined, the power to detect significant departures from the null hypothesis of no association for *CYP2C9* was less than that for *VKORC1*.

In previously reported work, Sconce *et al*[9](#ref9) demonstrated the best multivariable regression model for fitting warfarin dosage included both *CYP2C9* and *VKORC1*. Our multivariable model including age, gender, target INR, and *VKORC1**1173 genotype explained approximately 30% (R^2^ = 0.30) of our total observed variation. Our multivariable model including age, gender, target INR, and *CYP2C9*3* explained only 12% of the total variation. Including both *CYP2C9*3* and *VKORC1**1173 along with our covariates explained 34% of the total variation. Thus, our multivariable model appeared to explain less of the total variation in the mean warfarin dosage required to reach the target INR than the multivariable regression model of Sconce *et al*[9](#ref9) (R^2^ = 0.55). It should be noted that their model included height, a measure that we did not have available to us for this study. In addition, our results suggest, consistent with the findings of Wadelius *et al*[10](#ref10), that *VKORC1* polymorphisms explain a larger amount of the observed variation in mean warfarin dosage required to reach the target INR than do *CYP2C9* polymorphisms in the European‐American population.

During the course of our study, we also collected data on warfarin dosage requirements to reach the target INR for 18 African‐Americans. Of these 18 patients, one had a target INR of 1.5–2.0, 14 had a target INR of 2.0–3.0, and three had a target INR of 2.5–3.5. Marginally statistically significant results were observed for *VKORC1**1173 (p = 0.03) and *2255 (p = 0.03) and *CYP2C9*3* (p = 0.01) in this limited data set. These results could be entirely explained by the low observed warfarin dosages (average weekly dosage of 9.3 mg) of a single individual who was homozygous T/T at both *VKORC1**1173 and *2255 and heterozygous at *CYP2C9*3*. This individual's genotypes at these three markers were consistent with the genotypes that were shown to be associated with low warfarin dosage in the European‐American population. In addition, this individual had the lowest target INR (1.75) and was also the only African‐American individual in our sample with these aforementioned genotypes. We note that in our limited sample, there were strong differences in allele frequencies (data not shown) between the European‐American and African‐American samples. For example, *VKORC1**698 and *CYP2C9*2* were uninformative (completely homozygous for the wild type allele) in this small African‐American sample. We also note that we observed weaker marker‐marker LD and a larger diversity in haplotypes in this limited African‐American sample (data not shown). These findings are all consistent with the comparative results between the European and African‐Yoruban samples genotyped at *VKORC1* as part of the International HapMap Project ([www.hapmap.org](http://www.hapmap.org)). While these results based on 18 African‐American patients are limited and inconclusive, they do suggest that it is important to assess the role of *VKORC1* SNPs in warfarin response in other populations and that studying different populations with different haplotype patterns may be critical in ultimately identifying the functionally important variants.

It remains undetermined whether one or more of the SNPs identified here or elsewhere are causal in producing differential warfarin sensitivity or whether they are in LD with the actual causative SNP. Strong LD between *VKORC1* polymorphisms in the European‐American population hinders our ability to fine map/identify the functional loci. Further analysis of warfarin related gene(s) and surrounding sequence and the inclusion of larger numbers of patients, ideally in different populations, may lead to elucidation of the specific genotypic factors which influence warfarin sensitivity. In addition, prospective studies will ultimately be critical to establishing the clinical utility of assessing *VKORC1* genotype for determining optimal warfarin dosage. Nonetheless, combining genotype data from SNPs of the *VKORC1* gene with SNPs of the cytochrome *CYP2C9* gene in a clinical setting promises to result in a more robust assessment of warfarin sensitivity.

The results presented here support those from other recent reports[7](#ref7),[8](#ref8),[9](#ref9),[10](#ref10),[11](#ref11),[12](#ref12) and further demonstrate the need for prospective studies to evaluate the utility of *VKORC1* and *CYP2C9* genotype information with respect to outcomes such as length of time until the target INR is reached and adverse events. This current demonstration of common polymorphisms that predict warfarin sensitivity represents a potentially clinically useful proof of principal for the use of pharmacogenomic information in medicine and will likely play a critical role in future predictive models for warfarin dosage. Such models will have strong potential to guide physicians in quickly prescribing appropriate dosages and, ultimately, avoid complications.

## Electronic‐database information

Information about the computer program GOLD is at [www.sph.umich.edu/csg/abecasis/GOLD,](http://www.sph.umich.edu/csg/abecasis/GOLD,) the computer program HAPLO.STAT at [http://mayoresearch.mayo.edu/mayo/research/biostat/schaid.cfm](http://mayoresearch.mayo.edu/mayo/research/biostat/schaid.cfm), and the International HapMap Project at [www.hapmap.org](http://www.hapmap.org).

## Abbreviations

ANCOVA - analysis of covariance

INR - international normalised ratio

LD - linkage disequilibrium

SNP - single nucleotide polymorphism

VTE - venous thromboembolic events

## Footnotes

## References

1. Shulman S. Oral anticoagulation. In: Beutler E, Lichtman MA, Coller BS, Kipps TJ, Seligsohn U, eds. Williams hematology. 6th ed. New York: McGraw‐Hill, 20011777–1792.

2. Levine M N, Raskob G, Landefeld S, Kearon C. Hemorrhagic complications of anticoagulant treatment. Chest 2001119108S–21S.  [DOI](https://doi.org/10.1378/chest.119.1_suppl.108s) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11157645/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chest&volume=119&publication_year=2001&pages=108S-21S&pmid=11157645&doi=10.1378/chest.119.1_suppl.108s&)

3. Poller L, Shiach C R, MacCallum P K, Johansen A M, Munster A M, Magalhaes A, Jespersen J. Multicentre randomised study of computerised anticoagulant dosage. European Concerted Action on Anticoagulation. Lancet 19983521505–1509.  [DOI](https://doi.org/10.1016/s0140-6736(98)04147-6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9820298/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&volume=352&publication_year=1998&pages=1505-1509&pmid=9820298&doi=10.1016/s0140-6736(98)04147-6&)

4. Doecke C J, Cosh D G, Gallus A S. Standardised initial warfarin treatment: evaluation of initial treatment response and maintenance dose prediction by randomised trial, and risk factors for an excessive warfarin response. Aust N Z J Med 199121319–324.  [DOI](https://doi.org/10.1111/j.1445-5994.1991.tb04697.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1953510/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Aust%20N%20Z%20J%20Med&volume=21&publication_year=1991&pages=319-324&pmid=1953510&doi=10.1111/j.1445-5994.1991.tb04697.x&)

5. Li T, Chang C Y, Jin D Y, Lin P J, Khvorova A, Stafford D W. Identification of the gene for vitamin K epoxide reductase. Nature 2004427541–544.  [DOI](https://doi.org/10.1038/nature02254) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14765195/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&volume=427&publication_year=2004&pages=541-544&pmid=14765195&doi=10.1038/nature02254&)

6. Rost S, Fregin A, Ivaskevicius V, Conzelmann E, Hortnagel K, Pelz H J, Lappegard K, Seifried E, Scharrer I, Tuddenham E G, Muller C R, Strom T M, Oldenburg J. Mutations in VKORC1 cause warfarin resistance and multiple coagulation factor deficiency type 2. Nature 2004427537–541.  [DOI](https://doi.org/10.1038/nature02214) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14765194/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&volume=427&publication_year=2004&pages=537-541&pmid=14765194&doi=10.1038/nature02214&)

7. D'Andrea G, D'Ambrosio R L, Di Perna P, Chetta M, Santacroce R, Brancaccio V, Grandone E, Margaglione M. A polymorphism in the VKORC1 gene is associated with an interindividual variability in the dose‐anticoagulant effect of warfarin. Blood 2005105645–649.  [DOI](https://doi.org/10.1182/blood-2004-06-2111) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15358623/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&volume=105&publication_year=2005&pages=645-649&pmid=15358623&doi=10.1182/blood-2004-06-2111&)

8. Rieder M J, Reiner A P, Gage B F, Nickerson D A, Eby C S, McLeod H L, Blough D K, Thummel K E, Veenstra D L, Rettie A E. Effect of VKORC1 haplotypes on transcriptional regulation and warfarin dose. N Engl J Med 20053522285–2293.  [DOI](https://doi.org/10.1056/NEJMoa044503) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15930419/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&volume=352&publication_year=2005&pages=2285-2293&pmid=15930419&doi=10.1056/NEJMoa044503&)

9. Sconce E A, Khan T I, Wynne H A, Avery P, Monkhouse L, King B P, Wood P, Kesteven P, Daly A K, Kamali F. The impact of CYP2C9 and VKORC1 genetic polymorphism and patient characteristics upon warfarin dose requirements: proposal for a new dosing regimen. Blood 2005106(7)2329–2333.  [DOI](https://doi.org/10.1182/blood-2005-03-1108) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15947090/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&volume=106&publication_year=2005&pages=2329-2333&pmid=15947090&doi=10.1182/blood-2005-03-1108&)

10. Wadelius M, Chen L Y, Downes K, Ghori J, Hunt S, Eriksson N, Wallerman O, Melhus H, Wadelius C, Bentley D, Deloukas P. Common VKORC1 and GGCX polymorphisms associated with warfarin dose. Pharmacogenomics J 20055(4)262–270.  [DOI](https://doi.org/10.1038/sj.tpj.6500313) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15883587/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&volume=5&publication_year=2005&pages=262-270&pmid=15883587&doi=10.1038/sj.tpj.6500313&)

11. Yuan H Y, Chen J J, Lee M T, Wung J C, Chen Y F, Charng M J, Lu M J, Hung C R, Wei C Y, Chen C H, Wu J Y, Chen Y T. A novel functional VKORC1 promoter polymorphism is associated with inter‐individual and inter‐ethnic differences in warfarin sensitivity. Hum Mol Genet 2005141745–1751.  [DOI](https://doi.org/10.1093/hmg/ddi180) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15888487/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&volume=14&publication_year=2005&pages=1745-1751&pmid=15888487&doi=10.1093/hmg/ddi180&)

12. Lee S C, Ng S S, Oldenburg J, Chong P Y, Rost S, Guo J Y, Yap H L, Rankin S C, Khor H B, Yeo T C, Ng K S, Soong R, Goh B C. Interethnic variability of warfarin maintenance requirement is explained by VKORC1 genotype in an Asian population. Clin Pharmacol Ther 200679197–205.  [DOI](https://doi.org/10.1016/j.clpt.2005.11.006) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16513444/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&volume=79&publication_year=2006&pages=197-205&pmid=16513444&doi=10.1016/j.clpt.2005.11.006&)

13. Sanderson S, Emery J, Higgins J. CYP2C9 gene variants, drug dose, and bleeding risk in warfarin‐treated patients: a HuGEnet systematic review and meta‐analysis. Genet Med 2005797–104.  [DOI](https://doi.org/10.1097/01.gim.0000153664.65759.cf) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15714076/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genet%20Med&volume=7&publication_year=2005&pages=97-104&pmid=15714076&doi=10.1097/01.gim.0000153664.65759.cf&)

14. Lewontin R C. The interaction of selection and linkage. I. General considerations. Genetics 19644949–67.  [DOI](https://doi.org/10.1093/genetics/49.1.49) | [PMC free article](/articles/PMC1210557/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17248194/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genetics&volume=49&publication_year=1964&pages=49-67&pmid=17248194&doi=10.1093/genetics/49.1.49&)

15. Devlin B, Risch N. A comparison of linkage disequilibrium measures for fine‐scale mapping. Genomics 199529311–322.  [DOI](https://doi.org/10.1006/geno.1995.9003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8666377/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genomics&volume=29&publication_year=1995&pages=311-322&pmid=8666377&doi=10.1006/geno.1995.9003&)

16. Schulman S. Clinical practice. Care of patients receiving long‐term anticoagulant therapy. N Engl J Med 2003349675–683.  [DOI](https://doi.org/10.1056/NEJMcp025373) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12917305/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&volume=349&publication_year=2003&pages=675-683&pmid=12917305&doi=10.1056/NEJMcp025373&)
