# Interaction of Aminoglycosides with Human Mitochondrial 12S rRNA Carrying the Deafness-Associated Mutation

## Metadata
**Authors:** Yaping Qian, Min-Xin Guan
**Journal:** Antimicrobial Agents and Chemotherapy
**Date:** 2009 Aug 17
**DOI:** [10.1128/AAC.00965-08](https://doi.org/10.1128/AAC.00965-08)
**PMID:** 19687236
**PMCID:** PMC2772318
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2772318/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC2772318/pdf/0965-08.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC2772318/pdf/0965-08.pdf)

## Abstract

The mitochondrial 12S rRNA A1555G mutation is one of the important causes of aminoglycoside-induced and nonsyndromic hearing loss. Here we employed an RNA-directed chemical-modification approach to understanding the pathogenesis of aminoglycoside-induced hearing loss. The patterns of chemical modification of RNA oligonucleotides carrying the A1555G mutation by dimethyl sulfate (DMS) were distinct from those of the RNA oligonucleotides carrying wild-type sequence in the presence of aminoglycosides. In the RNA analogue carrying the A1555G mutation, reduced reactivity to DMS occurred in base G1555 as well as in bases C1556 and A1553 in the presence of paromomycin, neomycin, gentamicin, kanamycin, tobramycin, or streptomycin. In particular, base G1555 exhibited marked but similar levels of protection in the presence of 0.1 μM to 100 μM neomycin, gentamicin, or kanamycin. In contrast, the levels of protection in base G1555 appeared to be correlated with the concentration of paromycin, tobramycin, or steptomycin. Furthermore, increasing reactivities to DMS in the presence of these aminoglycosides were observed for bases A1492, C1493, C1494, and A1557 in the RNA analogue carrying the A1555G mutation. These data suggested that the A1555G mutation altered the binding properties of aminoglycosides at the A site of 12S rRNA and led to local conformational changes in 12S rRNA carrying the A1555G mutation. The interaction between aminoglycosides and 12S rRNA carrying the A1555G mutation provides new insight into the pathogenesis of aminoglycoside ototoxicity.

## MATERIALS AND METHODS

### Preparation of RNA oligonucleotide analogues.

To prepare RNA oligonucleotide analogues carrying the A site of human mitochondrial 12S rRNA, a DNA template (65 nt), as shown in Fig. [1A](#f1) and [2](#f2), was synthesized at a 250-nmol scale and purified on 10% (wt/vol) polyacrylamide gel electrophoresis (PAGE) (acrylamide-bisacrylamide, 19:1 [wt/wt])-7 M urea gels (Integrated DNA Technologies, Inc.). RNA oligonucleotide analogues (48 nt) consisted of wild-type or mutant human mtDNA sequences spanning the A site of 12S rRNA (bases 1488 to 1498 and 1551 to 1562 of mtDNA) ([22](#r22), [28](#r28)), two additional G-C base-pairs at the upper stem, and a tetraloop (UUCG) at the lower stem for stabilizing the structure, as well as an extended 17-base annealing site (5′-CCAACCGCCACCGAGCGC-3′) for the primer extension detection of the chemical modification ([23](#r23), [25](#r25)). RNA analogues were synthesized by in vitro transcription with T7 RNA polymerase (New England Biolabs) in the presence of 300 nM T7 primer (18 nt) (as shown in Fig. [2](#f2)) according to the manufacturer's protocol and were purified on 20% (wt/vol) PAGE (acrylamide-bisacrylamide, 19:1 [wt/wt])-7 M urea gels as detailed elsewhere ([23](#r23), [29](#r29)). Gel blocks with RNA fragments were eluted with an Elutrap electroelution apparatus (Schleicher & Schuell) and dialyzed with dialysis tubing (molecular weight cutoff, 1,000; Spectrum). Finally, RNA analogues were lyophilized and then dissolved in deionized water.

### FIG. 2.

![FIG. 2.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9eb7/2772318/80e247b8f6c7/zac0110985080002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=2772318_zac0110985080002.jpg)

Experimental schema for chemical modification.

### Chemical modification of RNA.

To determine if the A1555G mutation increases the binding of aminoglycosides to 12S rRNA and if these bound drugs cause conformational changes in the A site of 12S rRNA with respect to its wild-type counterpart, RNA was chemically modified by a method detailed elsewhere ([17](#r17), [23](#r23), [25](#r25)). Briefly, chemical modification reactions (100 μl) were performed in 80 mM K-HEPES (pH 7.0) with 10 pmol RNA oligonucleotide, and the reaction products were then denatured at 90°C for 1 min. Aminoglycoside antibiotics (Sigma-Aldrich) were added and then incubated at room temperature for 5 min. RNA modifications were conducted by addition of 2 μl dimethyl sulfate (DMS) (1:6 in ethanol), followed by incubation at room temperature for 10 min. Modification reactions were quenched by ethanol precipitation, and reaction products were resuspended in 10 μl of 1 M Tris-HCl (pH 8.1). After addition of 10 μl freshly prepared 0.2 M sodium borohydride (NaNH_4_), samples were incubated on ice in the dark for 30 min. The reaction was terminated by the addition of 300 μl of 0.3 M sodium acetate (NaAc), followed by ethanol precipitation. RNA pellets were redissolved in 20 μl of 1.0 M aniline-acetate (pH 4.5), followed by incubation in the dark at 60°C for 20 min. The reaction was stopped by the addition of 120 μl of 0.3 M NaAc and 120 μl of phenol-choloform-isoamyl alcohol (25:24:1), followed by vigorous mixing and centrifugation. Aqueous phases were precipitated by addition of 2.5 volume of absolute ethanol and 1/10 volume of 3 M NaAc, followed by a 70% ethanol wash and air drying. RNA pellets were then resuspended in 2.5 μl of deionized water.

### Primer extension.

To determine the nucleotide positions of chemical modification, a primer extension experiment was performed by a modified procedure detailed elsewhere ([26](#r26)). A DNA primer (5′-CGCGAGCCACGCCAACC-3′) complementary to the 3′ end of the annealing site was 5′ end labeled with [γ-^33^P]ATP (Perkin-Elmer, Inc.) using T4 polynucleotide kinase (New England Biolabs) at 37°C for 10 min, followed by inactivation of the enzyme at 90°C for 3 min. The DNA primer was annealed to 150 ng of modified RNA and 4 nmol of deoxynucleoside triphosphate in hybridization buffer (225 mM K-HEPES, 450 mM KCl [pH 7.0]) by heating at 90°C for 1 min, cooling to 47°C, and adding 5 nmol of one dideoxynucleoside triphosphate to each tube. Each extension reaction utilized 15 U of reverse transcriptase (Roche, Ltd.). The reaction mixture was incubated at 42°C for 1 h, and the reaction was stopped by addition of 20 μl of 0.1 M NaOH, followed by incubation for 10 min at 90°C to digest the remaining RNA. The labeled DNA was precipitated by 3 volumes of absolute ethanol and 0.1 volume of 3 M NaAc. Pellets were washed with 70% ethanol. The air-dried DNAs were resuspended in 5 μl of 7 M urea loading dyes. The sample was denatured at 90°C for 1 min and cooled on ice immediately; then it was loaded onto 20% PAGE (acrylamide-bisacrylamide, 19:1)-7 M urea gels. After electrophoresis, the gels were transferred to Whatman 3MM paper, dried, and autoradiographed 12 to 24 h at room temperature with Kodak XAr-5 film. Modification data were quantified by exposing the gel to a PhosphorImage screen (GE Healthcare) and were analyzed by Molecular Dynamics ImageQuant software.

### Data analysis.

The intensity of each band in the presence and absence of aminoglycosides was quantified with Kodak 1D Image Analysis software, version 3.6. To subtract the background variation, the average relative value of each band was normalized to the average relative value of bands located in the tetraloop area in each lane. The relative reactivities to DMS for each band in the RNA analogue carrying the A1555G mutation or in the wild-type 12S rRNA sequence in the presence of aminoglycosides were expressed as percentages of the average values for the corresponding band in the absence of drugs in each gel.

## RESULTS AND DISCUSSION

We employed an RNA-directed chemical-modification approach to determine if the A1555G mutation increases the binding of aminoglycosides to 12S rRNA and whether these bound drugs cause conformational changes in the A site of 12S rRNA with respect to its wild type counterpart. As shown in Fig. [1A](#f1), we designed RNA oligonucleotide analogues corresponding to the A site with wild-type 12S rRNA sequences and to a mutated version harboring the A1555G mutation as well as two additional G-C base pairs at the upper stem and a UUCG tetraloop at the lower stem to stabilize the structure ([24](#r24)). This RNA oligonucleotide model system, which mimics both the affinity and the specificity of the interaction of aminoglycosides with the small subunit of the ribosome, has been extensively used to investigate aminoglycoside-ribosome interaction in *E. coli* ([24](#r24), [25](#r25)). Aminoglycosides bind to the oligonucleotides encompassing the A site of 16S rRNA specifically and similarly to the way that they bind to the ribosome with the same affinity in *E. coli*. The RNAs, either alone or complexed with aminoglycosides, were incubated under suitable conditions with the chemical probe DMS, which modifies RNA at the guanine N-7, adenine N-1, and cytosine N-3 positions ([26](#r26)). If an antibiotic(s) binds RNA by stable interactions with specific nucleotide bases, it should be possible to identify such sites by diminished or reduced reactivity with DMS probes. These RNA analogues, as shown in Fig. [2](#f2), were first synthesized by in vitro transcription with T7 RNA polymerase using a DNA template. The binding of aminoglycosides to these RNA oligonucleotides was assayed by chemical probing with DMS, either alone, as naked RNA, or in the presence of increasing concentrations of various aminoglycosides. In particular, 150 ng of RNA oligonucleotides carrying the wild-type sequence or RNA oligonucleotides carrying the A1555G mutation in the presence of DMS was incubated with 0.1, 1, 10, or 100 μM paromomycin, neomycin, tobramycin, gentamicin, kanamycin, or streptomycin. Positions of chemical modification were determined by primer extension using synthetic DNA primers. Reverse transcriptase stops or pauses at modified bases, thus producing prematurely terminated cDNA chains. Sequencing reactions utilizing dideoxynucleotides and untreated RNA were performed in parallel, and transcripts were analyzed on a DNA sequencing gel, which was autoradiographed. The cDNA binding patterns produced by primer extension of probed uncomplexed RNA and probed complexed RNA were compared. In every case, the bases in the RNA analogues were shielded from chemical modification in the presence of bound drugs. Thus, protection was observed as decreased intensities of the relevant gel bands.

The patterns of chemical modifications by DMS in the RNA oligonucleotides carrying the A1555G mutation, as shown in Fig. [3](#f3), were distinct from those of wild-type RNA oligonucleotides in the presence and absence of paromomycin. Interestingly, bases A1492, C1493, and C1494 in the RNA analogue carrying the G1555 base migrated faster than the corresponding bases in the RNA analogue carrying the A1555 base. Higher reactivities to DMS in the absence or presence of paromomycin were observed for bases A1492, C1493, C1494, and A1557 in the RNA analogue carrying the G1555 base than those for the corresponding bases in the RNA analogue carrying the A1555 base. On the other hand, reduced reactivity to DMS in the presence of paromomycin was observed for bases G1555 and A1553 in the RNA analogue carrying the A1555G mutation. As an internal control, we also performed chemical modifications of the RNA oligonucleotides carrying the C1494U mutation in the presence of paromomycin. As shown in Fig. [3](#f3), reactivities to DMS in the presence of 0.1, 1, and 10 μM paromomycin were higher for bases A1492, C1493, U1494, and A1557 in the RNA analogue carrying the U1494 base than for the corresponding bases in the RNA analogue with the wild-type sequence. However, lower reactivities to DMS were observed in the presence of 100 μM paromomycin for bases A1492, C1493, U1494, A1555, C1556, and A1557 in the RNA analogue carrying the U1494 base than for the corresponding bases in the RNA analogue with the wild-type sequence. These results indicated that the A1555G and C1494T mutations increased the binding of paromomycin to the A site of 12S rRNA. These data also suggested that the A1555G mutation altered the local conformational changes of the A site of 12S rRNA and that such alterations were enhanced by the binding of aminoglycosides to the RNA analogue.

### FIG. 3.

![FIG. 3.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9eb7/2772318/c2f2b82dc2c3/zac0110985080003.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=2772318_zac0110985080003.jpg)

Autoradiographs of DMS probing reactions with human mitochondrial 12S rRNA oligonucleotides with the wild-type (WT) sequence or human mitochondrial 12S rRNA oligonucleotides carrying the A1555G or C1494T mutation. (A) Lanes a, b, c, and d show U, C, G, and A dideoxy sequencing reactions. (B) Lanes e and g, primer extension reactions using unmodified RNA analogues carrying base A1555 (WT) or G1555; lanes f and h, primer extension reactions using modified RNA analogues carrying base A1555 (WT) or G1555. (C) DMS probing reactions with RNA analogues in the presence of 0.1, 1, 10, and 100 μM paromomycin are shown in lanes 1 to 4 (WT), lanes 5 to 8 (A1555G mutation), and lanes 9 to 12 (C1494U mutation), respectively. The positions of bands corresponding to nucleotides of 12S rRNA are indicated to the right and left of the gels. An arrow indicates the nucleotide at position 1555.

Figure [4](#f4) shows the binding of paromomycin, neomycin, tobramycin, gentamicin, kanamycin, and streptomycin to RNA oligonucleotides with the wild-type sequence or those carrying the A1555G mutation, assayed by chemical modifications and monitored by primer extension. In agreement with the results for paromomycin, bases A1492, C1493, and C1494 in the RNA analogue carrying the G1555 base migrated faster than the corresponding bases in the RNA analogue carrying the A1555 base in the presence of the other five aminoglycosides. Various degrees of chemical modifications by DMS were detected in the RNA oligonucleotides carrying the A1555G mutation in the presence of different aminoglycosides. In particular, increasing reactivities to DMS in the absence or presence of aminoglycosides were observed for bases A1492, C1493, C1494, and A1557 in the RNA analogue carrying the A1555G mutation. In contrast, reduced reactivity to DMS was observed for bases G1555, A1556, and A1553 in the presence of 0.1 μM, 1 μM, 10 μM, and 100 μM paromomycin, neomycin, gentamicin, kanamycin, and tobramycin, as well as streptomycin. We have quantified these reactivities to DMS in the bases at positions 1555, 1556, and 1553 of RNA analogues in the presence of aminoglycosides by using Kodak 1D Image Analysis software, version 3.6. The relative values for each nucleotide in the RNA analogue carrying the A1555G mutation or wild-type 12S rRNA sequence in the presence of aminoglycosides were expressed as percentages of the average values for the corresponding band in the corresponding RNA analogue in the absence of drugs in each gel. Table [1](#t1) summarizes the relative reactivities to DMS for nucleotide at positions 1553, 1555, and 1556 in the RNA analogue carrying the A1555G mutation or wild-type 12S rRNA sequence in the presence of aminoglycosides relative to those in the absence of these drugs. In the RNA analogue carrying the A1555G mutation, the reactivities to DMS for base G1555 in the presence of aminoglycosides at 0.1 μM, 1 μM, 10 μM, and 100 μM (expressed as percentages of the reactivities in the absence of aminoglycosides) ranged from 74.5% to 59.1% with paromomycin, from 56.4% to 51.6% with neomycin, from 62.2% to 55.9% with gentamicin, from 77.7% to 53.6% with tobramycin, from 65% to 52.8% with kanamycin, and from 69.8% to 52.3% with streptomycin. In particular, base G1555 exhibited marked but similar levels of protection in the presence of 0.1 μM to 100 μM neomycin, gentamicin, and kanamycin. In contrast, the levels of protection for base G1555 appeared to be correlated with the concentrations of paromycin, tobramycin, and streptomycin. Apparently, these relative reactivities to DMS for base G1555 in the presence of aminoglycosides were lower than those for base A1555. These observations strongly indicated that the A1555G mutation increased the binding of aminoglycosides to base G1555 in the RNA analogues.

### FIG. 4.

![FIG. 4.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9eb7/2772318/7a4c4fdc3f4f/zac0110985080004.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=2772318_zac0110985080004.jpg)

Interactions of aminoglycosides with human mitochondrial 12S rRNA analogues carrying the wild-type sequence (WT) or the A1555G mutation (MT). DMS probing reactions with RNA analogues carrying base A1555 (WT) or G1555 (MT) in the presence of 0.1, 1, 10, and 100 μM paromomycin, neomycin, gentamicin, kanamycin, tobramycin, or streptomycin are shown in lanes 1 to 4 and lanes 5 to 8, respectively. The positions of bands corresponding to nucleotides of 12S rRNA are indicated to the right of each gel. An arrow indicates the nucleotide at position 1555.

### TABLE 1.

| Aminoglycoside | Nucleotide position | Relative reactivitya to DMS in the presence of the indicated concn of aminoglycoside |  |  |  |  |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| RNA oligonucleotide carrying 1555A |  |  |  | RNA oligonucleotide carrying 1555G |  |  |  |  |  |
| 0.1 μM | 1 μM | 10 μM | 100 μM | 0.1 μM | 1 μM | 10 μM | 100 μM |  |  |
| Paromomycin | 1553 | 122.8 | 109.0 | 95.2 | 79.2 | 78.1 | 73.9 | 60.5 | 50.8 |
| 1555 | 118.0 | 94.4 | 82.5 | 67.7 | 74.5 | 72.1 | 59.0 | 59.1 |  |
| 1556 | 142.1 | 138.5 | 139.5 | 113.9 | 115.1 | 95.1 | 76.8 | 73.9 |  |
| Neomycin | 1553 | 110.4 | 74.8 | 74.7 | 65.9 | 56.3 | 59.1 | 63.7 | 56.2 |
| 1555 | 101.6 | 85.5 | 62.9 | 62.7 | 56.4 | 56.0 | 55.0 | 51.6 |  |
| 1556 | 107.8 | 86.5 | 87.7 | 71.3 | 63.9 | 66.1 | 65.7 | 58.6 |  |
| Gentamicin | 1553 | 51.8 | 50.2 | 58.8 | 48.4 | 54.5 | 50.1 | 53.9 | 54.1 |
| 1555 | 85.0 | 75.1 | 77.9 | 69.0 | 62.2 | 58.2 | 56.2 | 55.9 |  |
| 1556 | 139.9 | 135.0 | 172.8 | 124.1 | 65.8 | 70.3 | 74.0 | 78.1 |  |
| Tobramycin | 1553 | 95.5 | 94.6 | 67.6 | 97.5 | 86.2 | 73.7 | 56.4 | 49.8 |
| 1555 | 86.7 | 71.9 | 59.5 | 59.0 | 77.7 | 60.5 | 48.9 | 53.6 |  |
| 1556 | 95.2 | 112.5 | 85.0 | 97.0 | 81.2 | 68.2 | 58.1 | 68.3 |  |
| Kanamycin | 1553 | 93.9 | 92.9 | 70.3 | 75.6 | 66.1 | 64.3 | 69.8 | 66.0 |
| 1555 | 89.4 | 84.7 | 70.3 | 62.3 | 65.0 | 59.9 | 56.6 | 52.8 |  |
| 1556 | 111.6 | 101.7 | 86.6 | 109.0 | 72.7 | 64.3 | 73.6 | 68.3 |  |
| Streptomycin | 1553 | 69.7 | 59.6 | 73.7 | 68.8 | 68.1 | 63.7 | 51.3 | 53.2 |
| 1555 | 79.4 | 69.5 | 72.9 | 66.9 | 69.8 | 67.1 | 56.2 | 52.3 |  |
| 1556 | 93.9 | 79.7 | 84.2 | 88.3 | 81.0 | 83.0 | 67.1 | 64.5 |  |

Table 1 Caption: Quantification of reactivity to DMS in RNA analogues in the presence of aminoglycosides

As shown in Table [1](#t1), the reactivities to DMS for bases A1553 and C1556 of RNA analogues carrying the A1555G mutation in the presence of aminoglycosides were reduced to various degrees, compared with those for the corresponding bases in the absence of aminoglycosides. As shown in Fig. [1A](#f1), the nucleotides at positions A1492, C1494, and A1557 of human mitochondrial 12S rRNA correspond to positions A1408, C1409, and A1493 of *E. coli* 16S rRNA, respectively. In *E. coli*, A1408 and A1493 are critical nucleotides for aminoglycoside binding to the A site of 16S rRNA ([16](#r16), [24](#r24)). In fact, these antibiotics bind in the major groove of the bacterial 16S rRNA with a pocket created by the A1408-A1493 base pair, a single bulged adenine (A1492), and the C1409-G1491 base pair. In particular, the C1409-G1491 base pair provides the floor for the antibiotic binding pocket ([7](#r7), [13](#r13), [27](#r27)). Here, the new G1555-C1494 base-pair created by the A1555G mutation altered the local conformation of the A site of mitochondrial 12S rRNA. Like its bacterial counterpart, the G1555-C1494 base-pair may produce a binding pocket of 12S rRNA for aminoglycosides. Hence, aminoglycosides bind to the A site of 12S rRNA carrying the A1555G mutation.

Figure [1B](#f1) summarizes the protection of nucleotides in human mitochondrial 12S rRNA analogues from DMS modification or their enhanced reactivities to DMS in the presence of various aminoglycosides. These data reveal how aminoglycosides interact with human mitochondrial 12S rRNA carrying the A1555G mutation. Our previous work has shown that the A1555G mutation led to a mild mitochondrial dysfunction but was itself insufficient to produce a deafness phenotype ([9](#r9), [10](#r10)). Exposure to aminoglycosides, which are concentrated in the perilymph and endolymph of the inner ear but are rapidly cleared from other tissues or organs ([12](#r12)), worsened mitochondrial dysfunctions caused by the A1555G mutation, leading to cell dysfunction or death in auditory systems ([8](#r8)). Therefore, exposure to aminoglycosides induces or worsens hearing loss in individuals carrying these mtDNA mutations. These data deepen our understanding of the pathogenesis of aminoglycoside ototoxicity and have a significant clinical impact.

## Acknowledgments

This work was supported by Public Health Service grants RO1DC05230 and RO1DC07696 from the National Institute on Deafness and Other Communication Disorders to M.-X.G.

We thank Scott Blanchard (Cornell University, New York, NY) for suggestions.

## Footnotes

## References

1. Chambers, H. F., and M. A. Sande. 1996. The aminoglycosides, p. 1103-1221. In J. G. Hardman, L. E. Limbird, P. B. Molinoff, R. W. Ruddon, and A. Gilman (ed.), Goodman & Gilman's the pharmacological basis of therapeutics, 9th ed. McGraw-Hill, New York, NY.

2. Davies, J., and B. D. Davis. 1968. Misreading of ribonucleic acid code words induced by aminoglycoside antibiotics. The effect of drug concentration. J. Biol. Chem. 243:3312-3316.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/5656371/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem.&volume=243&publication_year=1968&pages=3312&pmid=5656371&)

3. De Stasio, E. A., and A. E. Dahlberg. 1990. Effects of mutagenesis of a conserved base-paired site near the decoding region of Escherichia coli 16S ribosomal RNA. J. Mol. Biol. 212:127-133.  [DOI](https://doi.org/10.1016/0022-2836(90)90309-A) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1690811/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Mol.%20Biol.&volume=212&publication_year=1990&pages=127&pmid=1690811&doi=10.1016/0022-2836(90)90309-A&)

4. De Stasio, E. A., D. Moazed, H. F. Noller, and A. E. Dahlberg. 1989. Mutations in 16S ribosomal RNA disrupt antibiotic-RNA interactions. EMBO J. 8:1213-1216.  [DOI](https://doi.org/10.1002/j.1460-2075.1989.tb03494.x) | [PMC free article](/articles/PMC400937/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2472961/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=EMBO%20J.&volume=8&publication_year=1989&pages=1213&pmid=2472961&doi=10.1002/j.1460-2075.1989.tb03494.x&)

5. Estivill, X., N. Govea, A. Barcelo, E. Perello, C. Badenas, E. Romero, L. Moral, R. Scozzari, L. D'Urbano, M. Zeviani, and A. Torroni. 1998. Familial progressive sensorineural deafness is mainly due to the mtDNA A1555G mutation and is enhanced by treatment with aminoglycosides. Am. J. Hum. Genet. 62:27-35.  [DOI](https://doi.org/10.1086/301676) | [PMC free article](/articles/PMC1376822/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9490575/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Hum.%20Genet.&volume=62&publication_year=1998&pages=27&pmid=9490575&doi=10.1086/301676&)

6. Fischel-Ghodsian, N. 2005. Genetic factors in aminoglycoside toxicity. Pharmacogenomics 6:27-36.  [DOI](https://doi.org/10.1517/14622416.6.1.27) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15723603/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&volume=6&publication_year=2005&pages=27&pmid=15723603&doi=10.1517/14622416.6.1.27&)

7. Fourmy, D., M. I. Recht, S. C. Blanchard, and J. D. Puglisi. 1996. Structure of the A-site of Escherichia coli 16S ribosomal RNA complexed with an aminoglycoside antibiotic. Science 274:1367-1371.  [DOI](https://doi.org/10.1126/science.274.5291.1367) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8910275/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&volume=274&publication_year=1996&pages=1367&pmid=8910275&doi=10.1126/science.274.5291.1367&)

8. Guan, M. X. 2005. Prevalence of mitochondrial 12S rRNA mutations associated with aminoglycoside ototoxicity. Volta Rev. 105:211-237.  [DOI](https://doi.org/10.1016/j.mito.2010.10.006) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21047563/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Volta%20Rev.&volume=105&publication_year=2005&pages=211&pmid=21047563&doi=10.1016/j.mito.2010.10.006&)

9. Guan, M. X., N. Fischel-Ghodsian, and G. Attardi. 1996. Biochemical evidence for nuclear gene involvement in phenotype of nonsyndromic deafness associated with mitochondrial 12S rRNA mutation. Hum. Mol. Genet. 5:963-971.  [DOI](https://doi.org/10.1093/hmg/5.7.963) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8817331/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum.%20Mol.%20Genet.&volume=5&publication_year=1996&pages=963&pmid=8817331&doi=10.1093/hmg/5.7.963&)

10. Guan, M. X., N. Fischel-Ghodsian, and G. Attardi. 2001. Nuclear background determines biochemical phenotype in the deafness-associated mitochondrial 12S rRNA mutation. Hum. Mol. Genet. 10:573-580.  [DOI](https://doi.org/10.1093/hmg/10.6.573) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11230176/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum.%20Mol.%20Genet.&volume=10&publication_year=2001&pages=573&pmid=11230176&doi=10.1093/hmg/10.6.573&)

11. Hamasaki, K., and R. R. Rando. 1997. Specific binding of aminoglycosides to a human rRNA construct based on a DNA polymorphism, which causes aminoglycoside-induced deafness. Biochemistry 36:12323-12328.  [DOI](https://doi.org/10.1021/bi970962r) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9315872/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochemistry&volume=36&publication_year=1997&pages=12323&pmid=9315872&doi=10.1021/bi970962r&)

12. Henley, C. M., and J. Schacht. 1988. Pharmacokinetics of aminoglycoside antibiotics in blood, inner ear fluids and their relationship to ototoxicity. Audiology 27:137-146.  [DOI](https://doi.org/10.3109/00206098809081584) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/3044302/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Audiology&volume=27&publication_year=1988&pages=137&pmid=3044302&doi=10.3109/00206098809081584&)

13. Hobbie, S. N., P. Pfister, C. Bruell, P. Sander, B. François, E. Westhof, and E. C. Böttger. 2006. Binding of neomycin-class aminoglycoside antibiotics to mutant ribosomes with alterations in the A site of 16S rRNA. Antimicrob. Agents Chemother. 50:1489-1496.  [DOI](https://doi.org/10.1128/AAC.50.4.1489-1496.2006) | [PMC free article](/articles/PMC1426975/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16569869/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Antimicrob.%20Agents%20Chemother.&volume=50&publication_year=2006&pages=1489&pmid=16569869&doi=10.1128/AAC.50.4.1489-1496.2006&)

14. Li, R., G. Xing, M. Yan, X. Cao, X. Z. Liu, X. Bu, and M. X. Guan. 2004. Cosegregation of C-insertion at position 961 with A1555G mutation of mitochondrial 12S rRNA gene in a large Chinese family with maternally inherited hearing loss. Am. J. Med. Genet. 124A:113-117.  [DOI](https://doi.org/10.1002/ajmg.a.20305) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14699607/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Med.%20Genet.&volume=124A&publication_year=2004&pages=113&pmid=14699607&doi=10.1002/ajmg.a.20305&)

15. Matthijs, G., S. Claes, B. Longo-Bbenza, and J. J. Cassiman. 1996. Non-syndromic deafness associated with a mutation and a polymorphism in the mitochondrial 12S ribosomal RNA gene in a large Zairean pedigree. Eur. J. Hum. Genet. 4:46-51.  [DOI](https://doi.org/10.1159/000472169) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8800928/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Hum.%20Genet.&volume=4&publication_year=1996&pages=46&pmid=8800928&doi=10.1159/000472169&)

16. Moazed, D., and H. F. Noller. 1987. Interaction of antibiotics with functional sites in 16S ribosomal RNA. Nature 327:389-394.  [DOI](https://doi.org/10.1038/327389a0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2953976/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&volume=327&publication_year=1987&pages=389&pmid=2953976&doi=10.1038/327389a0&)

17. Moazed, D., S. Stern, and H. F. Noller. 1986. Rapid chemical probing of conformation in 16S ribosomal RNA and 30S ribosomal subunits using primer extension. J. Mol. Biol. 187:399-416.  [DOI](https://doi.org/10.1016/0022-2836(86)90441-9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2422386/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Mol.%20Biol.&volume=187&publication_year=1986&pages=399&pmid=2422386&doi=10.1016/0022-2836(86)90441-9&)

18. Morton, C. C. 2002. Genetics, genomics and gene discovery in the auditory system. Hum. Mol. Genet. 11:1229-1240.  [DOI](https://doi.org/10.1093/hmg/11.10.1229) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12015283/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum.%20Mol.%20Genet.&volume=11&publication_year=2002&pages=1229&pmid=12015283&doi=10.1093/hmg/11.10.1229&)

19. Morton, M. E. 1991. Genetic epidemiology of hearing impairment. Ann. N. Y. Acad. Sci. 630:16-31.  [DOI](https://doi.org/10.1111/j.1749-6632.1991.tb19572.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1952587/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann.%20N.%20Y.%20Acad.%20Sci.&volume=630&publication_year=1991&pages=16&pmid=1952587&doi=10.1111/j.1749-6632.1991.tb19572.x&)

20. Neefs, J. M., Y. Van de Peer, P. De Rijik, A. Goris, and R. De Wachter. 1991. Compilation of small ribosomal subunit RNA sequences. Nucleic Acids Res. 19(Suppl.):1987-2018.  [DOI](https://doi.org/10.1093/nar/19.suppl.1987) | [PMC free article](/articles/PMC331343/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2041797/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&volume=19&publication_year=1991&pages=1987&pmid=2041797&doi=10.1093/nar/19.suppl.1987&)

21. Noller, H. F. 1991. Ribosomal RNA and translation. Annu. Rev. Biochem. 60:191-227.  [DOI](https://doi.org/10.1146/annurev.bi.60.070191.001203) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1883196/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annu.%20Rev.%20Biochem.&volume=60&publication_year=1991&pages=191&pmid=1883196&doi=10.1146/annurev.bi.60.070191.001203&)

22. Prezant, T. R., J. V. Agapian, M. C. Bohlman, X. Bu, S. Oztas, W. Q. Qiu, K. S. Arnos, G. A. Cortopassi, L. Jaber, J. I. Rotter, M. Shohat, and N. Fischel-Ghodsian. 1993. Mitochondrial ribosomal RNA mutation associated with both antibiotic-induced and non-syndromic deafness. Nat. Genet. 4:289-294.  [DOI](https://doi.org/10.1038/ng0793-289) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7689389/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet.&volume=4&publication_year=1993&pages=289&pmid=7689389&doi=10.1038/ng0793-289&)

23. Puglisi, J. D., and J. R. Wyatt. 1995. Biochemical and NMR studies of RNA conformation with an emphasis on RNA pseudoknots. Methods Enzymol. 261:323-350.  [DOI](https://doi.org/10.1016/s0076-6879(95)61016-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8569502/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Methods%20Enzymol.&volume=261&publication_year=1995&pages=323&pmid=8569502&doi=10.1016/s0076-6879(95)61016-2&)

24. Purohit, P., and S. Stern. 1994. Interactions of a small RNA with antibiotic and RNA ligands of the 30S subunit. Nature 370:659-662.  [DOI](https://doi.org/10.1038/370659a0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8065453/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&volume=370&publication_year=1994&pages=659&pmid=8065453&doi=10.1038/370659a0&)

25. Recht, M. I., D. Fourmy, S. C. Blanchard, K. D. Dahlquist, and J. D. Puglisi. 1996. RNA sequence determinants for aminoglycoside binding to an A-site rRNA model oligonucleotide. J. Mol. Biol. 262:421-436.  [DOI](https://doi.org/10.1006/jmbi.1996.0526) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8893854/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Mol.%20Biol.&volume=262&publication_year=1996&pages=421&pmid=8893854&doi=10.1006/jmbi.1996.0526&)

26. Stern, S., D. Moazed, and H. F. Noller. 1988. Structural analysis of RNA using chemical and enzymatic probing monitored by primer extension. Methods Enzymol. 164:481-489.  [DOI](https://doi.org/10.1016/s0076-6879(88)64064-x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2468070/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Methods%20Enzymol.&volume=164&publication_year=1988&pages=481&pmid=2468070&doi=10.1016/s0076-6879(88)64064-x&)

27. Yoshizawa, S., D. Fourmy, and J. D. Puglisi. 1998. Structural origins of gentamicin antibiotic action. EMBO J. 17:6437-6448.  [DOI](https://doi.org/10.1093/emboj/17.22.6437) | [PMC free article](/articles/PMC1170992/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9822590/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=EMBO%20J.&volume=17&publication_year=1998&pages=6437&pmid=9822590&doi=10.1093/emboj/17.22.6437&)

28. Zhao, H., R. Li, Q. Wang, Q. Yan, J. H. Deng, D. Han, Y. Bai, W. Y. Young, and M. X. Guan. 2004. Maternally inherited aminoglycoside-induced and non-syndromic deafness is associated with the novel C1494T mutation in the mitochondrial 12S rRNA gene in a large Chinese family. Am. J. Hum. Genet. 74:139-152.  [DOI](https://doi.org/10.1086/381133) | [PMC free article](/articles/PMC1181901/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14681830/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Hum.%20Genet.&volume=74&publication_year=2004&pages=139&pmid=14681830&doi=10.1086/381133&)

29. Zimmermann, R. A., C. L. Thomas, and J. Wower. 1990. Structure and function of rRNA in the decoding domain and at the peptidyltransferase center, p. 331-347. In W. E. Hill, P. B. Moore, A. Dahlberg, D. Schlessinger, R. A. Garrett, and J. R. Warner (ed.), The ribosome: structure, function, and evolution. American Society for Microbiology, Washington, DC.
