# Phase II Trial of Pemetrexed Plus Bevacizumab for Second-Line Therapy of Patients With Advanced Non–Small-Cell Lung Cancer: NCCTG and SWOG Study N0426

## Metadata
**Authors:** Alex A Adjei, Sumithra J Mandrekar, Grace K Dy, Julian R Molina, Araba A Adjei, David R Gandara, Katie L Allen Ziegler, Philip J Stella, Kendrith M Rowland, Jr, Steven E Schild, Ralph G Zinner
**Journal:** Journal of Clinical Oncology
**Date:** 2009 Oct 19
**DOI:** [10.1200/JCO.2009.23.6406](https://doi.org/10.1200/JCO.2009.23.6406)
**PMID:** 19841321
**PMCID:** PMC2815996
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2815996/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC2815996/pdf/zlj614.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC2815996/pdf/zlj614.pdf)

## Abstract

**Purpose:** 
To evaluate the efficacy and toxicity of pemetrexed combined with bevacizumab as second-line therapy for patients with advanced non–small-cell lung cancer (NSCLC) and to correlate allelic variants in pemetrexed-metabolizing genes with clinical outcome.

**Patients and Methods:** 
Patients with previously treated NSCLC received pemetrexed (500 mg/m2 intravenous) combined with bevacizumab (15 mg/kg intravenous) every 3 weeks. The primary end point, evaluated using a one-stage Fleming design for detecting a true success rate of at least 70%, was the proportion of patients who were progression free and on treatment at 3 months. Polymorphisms in genes responsible for pemetrexed transport (reduced folate carrier [SLC19A1]) and metabolism (folylpolyglutamate synthase [FPGS] and gamma-glutamyl hydrolase [GGH]) evaluated in germline DNA (blood) were correlated with treatment outcome.

**Results:** 
Forty-eight evaluable patients (14 females and 34 males) received a median of four cycles (range, one to 20 cycles). The most common grade 3 or 4 nonhematologic adverse events (AEs) were fatigue (13%), dyspnea (10%), and thrombosis (10%). Grade 3 or 4 hematologic AEs were neutropenia (19%) and lymphopenia (13%). Twenty-four (57%; 95% CI, 41% to 72%) of the first 42 patients met the success criteria. Median overall survival (OS) and progression-free survival (PFS) times were 8.6 and 4.0 months, respectively. The exon 6 (2522)C→T polymorphism in SLC19A1 correlated with 3-month progression-free status (P = .01) and with PFS (P = .05). The IVS1(1307)C→T polymorphism in GGH correlated with OS (P = .04).

**Conclusion:** 
The study did not meet its primary end point. However, the median PFS time of 4 months is promising. Pharmacogenetic studies in larger cohorts are needed to definitively identify polymorphisms that predict for survival and toxicity of pemetrexed.

### Purpose

To evaluate the efficacy and toxicity of pemetrexed combined with bevacizumab as second-line therapy for patients with advanced non–small-cell lung cancer (NSCLC) and to correlate allelic variants in pemetrexed-metabolizing genes with clinical outcome.

### Patients and Methods

Patients with previously treated NSCLC received pemetrexed (500 mg/m^2^ intravenous) combined with bevacizumab (15 mg/kg intravenous) every 3 weeks. The primary end point, evaluated using a one-stage Fleming design for detecting a true success rate of at least 70%, was the proportion of patients who were progression free and on treatment at 3 months. Polymorphisms in genes responsible for pemetrexed transport (reduced folate carrier [*SLC19A1*]) and metabolism (folylpolyglutamate synthase [*FPGS*] and gamma-glutamyl hydrolase [*GGH*]) evaluated in germline DNA (blood) were correlated with treatment outcome.

### Results

Forty-eight evaluable patients (14 females and 34 males) received a median of four cycles (range, one to 20 cycles). The most common grade 3 or 4 nonhematologic adverse events (AEs) were fatigue (13%), dyspnea (10%), and thrombosis (10%). Grade 3 or 4 hematologic AEs were neutropenia (19%) and lymphopenia (13%). Twenty-four (57%; 95% CI, 41% to 72%) of the first 42 patients met the success criteria. Median overall survival (OS) and progression-free survival (PFS) times were 8.6 and 4.0 months, respectively. The exon 6 (2522)C→T polymorphism in *SLC19A1* correlated with 3-month progression-free status (*P* = .01) and with PFS (*P* = .05). The IVS1(1307)C→T polymorphism in *GGH* correlated with OS (*P* = .04).

### Conclusion

The study did not meet its primary end point. However, the median PFS time of 4 months is promising. Pharmacogenetic studies in larger cohorts are needed to definitively identify polymorphisms that predict for survival and toxicity of pemetrexed.

## INTRODUCTION

Only 25% to 30% of patients with non–small-cell lung cancer (NSCLC) have resectable disease (stage I or II) at the time of diagnosis.^1^ Treatment of recurrent NSCLC has been particularly difficult, with response rates and median survival times of all recently evaluated agents being approximately 9% and 8 months, respectively. There is a need for novel regimens to improve outcome.

Pemetrexed disodium is an antifolate whose primary mechanism of action is the inhibition of the enzyme thymidylate synthase (TS).^2^ Inhibition of TS results in a decrease in thymidine, a pyrimidine necessary for DNA synthesis.^3,4^ Pemetrexed disodium also inhibits dihydrofolate reductase and glycinamide ribonucleotide formyl transferase, a folate-dependent enzyme involved in purine synthesis.^5^ Pemetrexed disodium gains entry to the cell via the reduced folate carrier and, once localized, is an excellent substrate for folylpolyglutamate synthetase (FPGS). The pentaglutamate form of pemetrexed disodium is the predominant intracellular form and is greater than 60-fold more potent in its inhibition of TS than the monoglutamate.^5^

Pemetrexed has been approved by several licensing authorities worldwide for the first- and second-line therapy of NSCLC. Significant toxicity, including myelosuppression, skin rash, mucositis, and fatigue, was seen in initial studies with pemetrexed. The recognition that high serum homocysteine levels predicted for toxicity led investigators to administer folate and vitamin B_12_ supplementation in pemetrexed regimens. This has reduced but not completely abolished severe adverse events (AEs), especially in certain combination therapy regimens.^6^ In addition, the response rate of single-agent pemetrexed in NSCLC ranges from 17% to 23% in the first-line setting and is approximately 9% in the second-line setting.

The molecular mechanism underlying these variations in response rate and toxicity may be, in part, a result of genetic variations in genes that code for the proteins that act on pemetrexed, including those responsible for its transport (*SLC19A1*), activation (*FPGS*), and inactivation (*GGH*). Studies with methotrexate suggest that resistance to antifolates may be a result of decreased *SLC19A1* transport, decreased *FPGS* expression, and/or inactivating mutations or increased *GGH* expression.^7–12^ Therefore, we hypothesized that single nucleotide polymorphisms (SNPs) in these three genes influence the efficacy and toxicity of pemetrexed.

Finally, pemetrexed and antiangiogenic agents have been shown to be synergistic in preclinical models. Bevacizumab is a monoclonal antibody that targets circulating vascular endothelial growth factor A and has demonstrated clinical synergy with a number of chemotherapeutic agents. On the basis of this synergy, we evaluated the combination of pemetrexed and bevacizumab as second-line therapy of NSCLC and then evaluated the impact of the SNPs in *SLC19A1*, *FPGS*, and *GGH* on the clinical outcome.

## PATIENTS AND METHODS

### Patient Selection

Patients with histologic or cytologic evidence of measurable metastatic or stage IIIB (pleural effusion) nonsquamous NSCLC and only one prior systemic chemotherapy regimen were eligible for this study. Eligibility criteria also included age ≥ 18 years and Eastern Cooperative Oncology Group performance status ≤ 2. Prior radiation was not to encompass more than 30% of the bone marrow reserve, could not include the target lesions, and was completed at least 4 weeks before study enrollment. Adequate bone marrow (platelets ≥ 100,000/μL, absolute neutrophil count > 1,500/μL, and hemoglobin ≥ 10 g/dL), hepatic (total bilirubin ≤ 2× the upper limit of normal and AST ≤ 3× normal), and renal (serum creatinine ≤ 1.5× the upper limit of normal) function were required. Patients with hemoptysis or brain metastases, those receiving therapeutic anticoagulation, and those with bleeding diatheses were excluded. Patients who had previously received bevacizumab were not eligible. The protocol was approved by institutional review boards, and all patients were required to give written informed consent under federal and institutional guidelines ([Fig 1](#F1)).

### Fig 1.

![Fig 1.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7f5e/2815996/79288ed546a8/zlj9990993590001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=2815996_zlj9990993590001.jpg)

CONSORT diagram. (*) Patients were to be treated every 21 days until progessive disease, unacceptable toxicity, physician decision, patients refusal, or alternative treatment. (†) With the exception of the primary end point (based on the first 42 patients per protocol) and the genotype analysis (based on 33 patients who had complete genotype information).

### Experimental Treatment Schedule

Pemetrexed at a dose of 500 mg/m^2^ dissolved in 100 mL of 0.9% saline was administered intravenously over 10 minutes via an automatic dispensing pump, followed by bevacizumab 15 mg/kg infused intravenously over 90 minutes every 21 days until progression, unacceptable toxicity, patient refusal, alternate treatment, or investigator's decision to remove patient from study. Treatment with folic acid 400 μg daily started a week before pemetrexed treatment and continued until patients went off study. Treatment with vitamin B_12_ 1,000 μg intramuscularly was also started a week before pemetrexed administration and repeated every 9 weeks until patient went off study.

### Clinical Care of Patients

Complete patient histories, physical examinations, CBC counts, serum electrolytes, chemistries, and urinalysis were performed at baseline and before each cycle of treatment. Home blood pressure monitoring was performed to evaluate the incidence and severity of hypertension. Laboratory studies were repeated weekly. Radiologic studies (computed tomography and magnetic resonance imaging) were performed at baseline and after every two cycles of therapy to assess tumor response. Criteria for discontinuation from study included grade 3 or 4 drug-related toxicity requiring a treatment delay ≥ 2 weeks, progressive disease, withdrawal of consent, or change in patient's condition that made further treatment inappropriate.

### Pharmacogenetic Studies

#### Blood sampling and DNA extraction.

Ten milliliters of blood were drawn into EDTA tubes and shipped to the North Central Cancer Treatment Group central laboratory, where DNA was extracted and batched until the end of the study for genotyping, as described in the [Appendix](#app1) (online only).

#### SNP identification.

Resequencing data from Caucasian DNA samples^13^ and data from the National Center for Biotechnology Information and the HapMap Caucasian databases were used to obtain polymorphism (SNP) information on *FPGS*, *GGH*, and *SLC19A1*. SNPs in these three genes had been identified in our preliminary studies^14^ as potentially predicting for response and efficacy after therapy with pemetrexed.

The SNP genotype data sets were loaded into Haploview version 4.0 (Broad Institute, Cambridge, MA),^15^ and polymorphisms with frequencies greater than 5% were selected for haplotype analysis at an *r*^2^ threshold of 0.8. Haplotypes close to or greater than 2% were organized into single blocks from which haplotype tag SNPs (ht-SNPs) were derived. In addition, the tagger software was used to generate tag SNPs.

#### SNP genotyping.

All SNP genotyping was performed using the MassARRAY Compact system (Sequenom, San Diego, CA) and is described in detail in the [Appendix](#app1).

### Statistical Methods

This single-stage phase II study was designed to assess the 3-month progression-free survival (PFS) rate of the combination of pemetrexed disodium and bevacizumab as second-line treatment among patients with advanced NSCLC (stage IIIB with pleural effusions or stage IV). A one-stage Fleming design with an exact significance level of *P* = .09 and power of 90% was used to test the hypothesis that the true success rate (progression free at 3 months and on treatment) was at most 50% versus the alternative hypothesis that the true success rate is at least 70%. With a sample size of 42 evaluable patients, the regimen would be declared promising if at least 26 successes were observed.

Overall survival (OS) time was defined as the time from registration to death as a result of any cause. PFS was defined as the time from registration to the first date of disease progression or death as a result of any cause. PFS was censored at the date of the last contact for patients alive and progression free at the time of this analysis. Duration of response was defined as the interval between the date of first objective status assessment of a confirmed complete response or partial response (PR) and first date of documented disease progression. Exact binomial CIs for the proportion of successes were constructed. A secondary sensitivity analysis including such patients as successes was performed to assess the impact on the success rates. The distribution of PFS and OS time was estimated using the Kaplan-Meier method. Fisher's exact tests were used to compare the PFS status, AE patterns, and tumor response rates between the different tagged SNP subgroups. Kaplan-Meier curves and log-rank tests were also used to compare the OS and PFS distributions between the different tagged SNP subgroups. No adjustments for multiple comparisons were performed for the correlation of the polymorphisms with clinical outcomes because it was primarily an exploratory exercise. *P* ≤ .05 was considered statistically significant.

## RESULTS

### Patient and Treatment Characteristics

A total of 48 patients (all evaluable) were enrolled between May 2006 and February 2007. Data were frozen as of December 15, 2008. The baseline characteristics are listed in [Table 1](#T1). The median age was 65 years (range, 43 to 78 years), with 90% of patients having a performance status of 0 or 1 at baseline, 71% of patients being male, and 94% of patients having stage IV disease. Smoking history was not collected on these patients. All patients were off active treatment at the time of this analysis. The median follow-up time for surviving patients was 21 months (range, 3.9 to 25.4 months). The median number of treatment cycles delivered was four (range, one to 20 cycles). Sixty-seven percent of the patients ended treatment because of disease progression ([Table 1](#T1)). There were no dose reductions reported in the first two cycles for either drug; one dose reduction each for pemetrexed and bevacizumab was reported in the third cycle; and one dose reduction for pemetrexed was reported in cycle 4.

### Table 1.

| Characteristic | No. of Patients | % |
| --- | --- | --- |
| Baseline data |   |   |
| Age, years |   |  |
| Median | 65 |  |
| Range | 43-78 |  |
| Sex |   |   |
| Female | 14 | 29.2 |
| Male | 34 | 70.8 |
| Performance score |   |   |
| 0 | 17 | 35.4 |
| 1 | 26 | 54.2 |
| 2 | 5 | 10.4 |
| Lung stage |   |   |
| IIIB with pleural effusion | 3 | 6.3 |
| IV | 45 | 93.8 |
| Status of primary tumor |   |   |
| Resected with no residual | 7 | 14.6 |
| Unresected | 32 | 66.7 |
| Recurrent | 9 | 18.8 |
| Previous systemic cancer therapy |   |   |
| Platinum based | 42 | 87.5 |
| Other* | 6 | 12.5 |
| Follow-up data |   |   |
| No. of cycles of treatment |   |  |
| Median | 4 |  |
| Range | 1-20 |  |
| Progression status |   |   |
| No progression | 4 | 8.3 |
| Progression | 44 | 91.7 |
| Follow-up status |   |   |
| Alive | 10 | 20.8 |
| Dead | 38 | 79.2 |
| Ended treatment | 48 | 100.0 |
| Reason for ending treatment |   |   |
| Disease progression | 32 | 66.7 |
| Adverse event | 7 | 14.6 |
| Refused further treatment | 4 | 8.3 |
| Other medical problems | 1 | 2.1 |
| Died on study | 1 | 2.1 |
| Other† | 3 | 6.3 |

Table 1 Caption: Baseline Patient Demographic and Clinical Characteristics and Follow-Up Data

### AEs

All 48 patients were evaluable for AEs (summarized regardless of attribution, unless noted otherwise). A total of 32 patients (67%) and 11 patients (23%) reported grade 3 or higher and grade 4 or higher AEs, respectively ([Table 2](#T2)). Five grade 5 events (none deemed related to treatment) were reported on study (disease progression, n =3; disease progression/myocardial ventricular arrhythmia, n = 1; and acute respiratory distress syndrome, n = 1). In addition, one patient experienced a grade 5 sudden death (car accident) within 30 days of treatment completion.

### Table 2.

| AE | No. of Patients | % |
| --- | --- | --- |
| Patients with at least one of the following: |   |   |
| Grade 3+ AE | 32 | 66.7 |
| Grade 4+ AE | 11 | 22.9 |
| Grade 3+ hematologic AE | 14 | 29.2 |
| Grade 4+ hematologic AE | 4 | 8.3 |
| Grade 3+ nonhematologic AE | 28 | 58.3 |
| Grade 4+ nonhematologic AE | 8 | 16.7 |
| Grade 4 hematologic AEs |   |   |
| Anemia | 0 | 0.0 |
| Leukopenia | 2 | 4.2 |
| Lymphopenia | 1 | 2.1 |
| Neutropenia | 3 | 6.3 |
| Thrombocytopenia | 2 | 4.2 |
| Grade 4 nonhematologic AEs |   |   |
| Fatigue | 2 | 4.2 |
| Dyspnea | 2 | 4.2 |
| Thrombosis | 2 | 4.2 |
| Hypercalcemia | 1 | 2.1 |

Table 2 Caption: Summary of All Grade 3 or 4 AEs and Hematologic and Nonhematologic Grade 4 AEs

#### Hematologic AEs.

Fourteen patients (29%) reported a grade 3 or higher hematologic AE, with 8% reporting a grade 4 event ([Table 2](#T2)). The most common (occurred in ≥ 10% of patients, n = 5) grade 3 or 4 hematologic AEs were neutropenia (19%), leukopenia (17%), and lymphopenia (13%). [Table 2](#T2) lists all grade 4 hematologic AEs reported on this trial.

#### Nonhematologic AEs.

A total of 28 patients (58%) reported a grade 3 or higher nonhematologic AE, with 17% reporting a grade 4 or higher event ([Table 2](#T2)). The most common (occurred in ≥ 10% of patients, n = 5) grade 3 or 4 nonhematologic AEs were fatigue (13%), dyspnea (10%), and thrombosis (10%). [Table 2](#T2) lists all grade 4 nonhematologic AEs reported on this trial.

### Clinical Outcome

Twenty-four (57%; 95% CI, 41% to 72%) of the first 42 patients met the protocol-defined criteria for success (progression free and on treatment at 3 months). In addition, two patients (among the first 42 patients) who went off treatment before 3 months remained progression free at 3 months (26 of 42 patients; 62%; 95% CI, 46% to 76%).

Among all 48 patients, the clinical benefit rate was 50% (five confirmed PRs and 19 patients with stable disease for ≥ 6 weeks; 95% CI, 35% to 65%). The median response duration, among patients who had a PR, was 213 days (range, 84 to 460 days). The median PFS time was 4.0 months (95% CI, 2.8 to 4.7 months; [Fig 2](#F2)A), and the median OS time was 8.6 months (95% CI, 6.6 to 13.7 months; [Fig 2](#F2)B).

### Fig 2.

![Fig 2.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7f5e/2815996/e4177722ca32/zlj9990993590002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=2815996_zlj9990993590002.jpg)

(A) Progression-free survival (PFS) for patients with non–small-cell lung cancer (NSCLC) undergoing pemetrexed and bevacizumab combination therapy. Median PFS is 4 months (95% CI, 2.8 to 4.7 months; No. of events = 46). (B) Overall survival (OS) for patients with NSCLC undergoing pemetrexed and bevacizumab combination therapy. Median OS is 8.6 months (95% CI, 6.6 to 13.7 months; No. of events = 38).

### Genetic Polymorphisms and Treatment Outcomes

Forty-four of the 48 patients had blood samples available for DNA isolation and for the genotyping assay. Eighteen of 19 tag SNPs were successfully genotyped. Thirty-three (75%) of the 44 patients with DNA samples available had complete genotype information on all 18 SNPs. Distributions of patient genotypes and minor allele frequencies are listed in [Appendix Table A1](#app1) (online only).

Two polymorphisms in *GGH*, IVS1(1307)C→T and IVS5(1042)T→C, and one in *SLC19A1*, IVS2(4935)G→A, correlated significantly with toxicity. The SNP *GGH* IVS1(1307)C→T was associated with grade 3+ lymphopenia (CC *v* CT *v* TT: 0% *v* 5% *v* 50%, respectively; Fisher's exact test, *P* = .03), whereas *GGH* IVS5(1042)T→C was associated with any grade 4+ AE (TT *v* TC *v* CC: 35% *v* 0% *v* 25%, respectively; Fisher's exact test, *P* = .03). The *SLC19A1* IVS2(4935)G→A polymorphism correlated significantly with grade 3+ leukopenia (GG *v* AG *v* AA: 33% *v* 24% *v* 0%, respectively; Fisher's exact test, *P* = .05). Furthermore, the wild-type genotype in *FPGS*, 5′UTR(−63)G→A, correlated significantly with toxicity (grade 3+ neutropenia: GG *v* GA *v* AA: 40% *v* 0% *v* 0%, respectively; Fisher's exact test, *P* = .02; [Table 3](#T3)).

### Table 3.

| Adverse Event and Genotype | Frequency (% of patients) | P* |
| --- | --- | --- |
| Any grade 4: GGH IVS5(1042)T→C |   |   |
| TT | 35 | .03 |
| TC | 25 |   |
| CC | 0 |   |
| Grade 3+ neutropenia: FPGS 5′UTR(–63)G→A |   |   |
| GG | 40 | .02 |
| GA | 0 |   |
| AA | 0 |   |
| Grade 3+ leukopenia: SLC19A1 IVS2(4935)G→A |   |   |
| GG | 33 | .05 |
| AG | 24 |   |
| AA | 0 |   |
| Grade 3+ lymphopenia: GGH IVS1(1307)C→T |   |   |
| CC | 0 | .03 |
| CT | 5 |   |
| TT | 50 |   |

Table 3 Caption: Genotype and Adverse Events

A significant association was observed for the *GGH* polymorphism IVS1(1307)C→T and median OS (CC *v* CT *v* TT: 15.4 *v* 7.3 *v* 4.8 months, respectively; log-rank test, *P* = .04; [Fig 3](#F3)A). Additionally, the *GGH* exon 1 (91)C→T (Ala_31_Thr) polymorphism was marginally associated with median OS (CC *v* TC: 6.5 *v* 14.1 months, respectively; log-rank test, *P* = .06). The primary end point for this study was 3-month PFS, and only patients with the *SLC19A1* exon 6 (2522)C→T polymorphism showed a significant association with that end point (CC *v* CT *v* TT: 57% *v* 29% *v* 78%, respectively; Fisher's exact test, *P* = .01). Furthermore, the *SLC19A1* exon 6 (2522)C→T polymorphism was also associated significantly with median PFS (CC *v* CT *v* TT: 5.7 *v* 2.8 *v* 4.7 months, respectively; log-rank test, *P* = .05; [Fig 3](#F3)B). One other polymorphism in *GGH*, IVS7(1478)G→A, also exhibited a trend toward an association with 3-month PFS (GG *v* GA: 68% *v* 35%, respectively; Fisher's exact test, *P* = .06).

### Fig 3.

![Fig 3.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7f5e/2815996/fb7a79435b26/zlj9990993590003.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=2815996_zlj9990993590003.jpg)

(A) Kaplan-Meier estimates for overall survival (OS) for patients with non–small-cell lung cancer (NSCLC) undergoing pemetrexed and bevacizumab combination therapy. OS was plotted by GGH IVS1(1307)C→T polymorphism (rs3780126): CC CT v TT genotypes. (B) Kaplan-Meier estimates for progression-free survival (PFS) for patients with NSCLC undergoing pemetrexed and bevacizumab combination therapy. PFS was plotted by SLC19A1 exon 6 (2522)C→T polymorphism (rs1051298): CC v CT v TT genotypes.

## DISCUSSION

This multicenter, cooperative group, phase II study was performed to evaluate the clinical efficacy and toxicity of pemetrexed combined with bevacizumab administered every 3 weeks in the second-line therapy of patients with stage IV or IIIB (pleural effusion) NSCLC. To understand the pharmacogenetic implications of polymorphisms in three specific genes responsible for the transport and metabolism of pemetrexed, we evaluated the patients' genotypes and correlated them with response and toxicity. As typical with NSCLC studies, the patient population was older (median age, 65 years). However, this combination was well tolerated, with an incidence of grade ≥ 4 hematologic AEs of 8%. The incidence of grade ≥ 4 nonhematologic AEs was 17%, and these AEs included fatigue, thrombosis, hypercalcemia, and dyspnea. As these AEs are reviewed, it is important to note that they are reported regardless of investigator-attributed causality. This AE profile suggests that pemetrexed-bevacizumab–based combinations are worthy of further evaluation in NSCLC.

Twenty-four (57%; 95% CI, 41% to 72%) of the first 42 patients met the criteria for success. One patient remained progression free at 90 days but went off treatment (deep vein thrombosis 1 day short of the required 91 days) and experienced progression at day 278. Thus, the success criterion of at least 26 patients being progression free at 3 months was not met. Five patients achieved a PR. In these five responding patients, the median response duration was 213 days (range, 84 to 460 days). The median OS time was 8.6 months, with a median PFS time of 4.0 months.

Four tagged SNPs [5′UTR(−63)G→A in *FPGS*, IVS1(1307)C→T and IVS5(1042)T→C in *GGH*, and IVS2(4935)G→A in *SLC19A1*] were significantly correlated with AEs. In addition, two of these SNPs were associated with treatment efficacy. The IVS1(1307)C→T SNP in *GGH* that was associated with lymphopenia was significantly associated with OS, with patients with the wild-type CC genotype having a three-fold higher median OS time (15.4 months) compared with patients with the variant TT genotype (4.8 months). The exon 6 (2522)C→T polymorphism in *SLC19A1* correlated with median PFS time and 3-month PFS rate.

Thus, in this small study that analyzed relevant SNPs in the pemetrexed transport and metabolic pathway, five polymorphisms in the genes coding for pemetrexed transport and activating and inactivating proteins correlated with efficacy and toxicity. Limitations of the present study are the small sample size and the fact that patients received pemetrexed in combination with bevacizumab. The contribution of bevacizumab to the efficacy and toxicity of pemetrexed is a confounding variable. However, a number of similar pharmacogenetic studies have been performed with combinations of irinotecan^16^ and fluorouracil,^17^ for example, where SNPs relevant to irinotecan and fluorouracil have been correlated with toxicity and outcome.

Twenty-four patients met the predefined study criteria for success, instead of the 26 patients needed to declare this study successful. However, in this multicenter, dual cooperative group, second-line NSCLC study, the median PFS was 4.1 months. This represents an improvement in the median PFS compared with the PFS of 2.9 months reported for a number of single cytotoxic agents administered as second-line therapy for NSCLC.^18,19^ In a retrospective subgroup analysis of the pivotal phase III study of single-agent pemetrexed,^19^ the median PFS of nonsquamous patients (similar to the population treated in this study) was 3.1 months. Thus, although the 4.1-month median PFS time of this study is promising, a larger study incorporating confirmation of the pharmacogenetic data may yield information on specific genotypes of *SLC19A1*, *GGH*, and *FPGS* for which this combination may be particularly effective. There is an ongoing observational study evaluating these SNPs in a second-line single-agent pemetrexed study. Those results will provide information on the potential contribution of bevacizumab to the efficacy of pemetrexed in these genotypes.

## Acknowledgment

We thank the patients and their families as well as Raquel Ostby for expert secretarial assistance.

## Appendix

### Additional Participating Institutions

*Additional participating institutions include*: Siouxland Hematology-Oncology Associates, Sioux City, IA 51105 (Donald Wender, MD); Medcenter One Health Systems, Bismarck, ND 58506 (John T. Reynolds, MD); Metro-Minnesota Community Clinical Oncology Program, St Louis Park, MN 55416 (Patrick J. Flynn, MD); Cedar Rapids Oncology Project Community Clinical Oncology Program (CCOP), Cedar Rapids, IA 52403 (Martin Wiesenfeld, MD); Duluth CCOP, Duluth, MN 55805 (Daniel A. Nikcevich, MD); Hematology & Oncology of Dayton, Dayton, OH 45415 (Howard M. Gross, MD); Hawaii Minority-Based CCOP, Honolulu, HI 96813 (William S. Loui, MD); Heartland Cancer Research CCOP, St Louis, MO 63131 (Alan P. Lyss, MD); Minnesota Cooperative Group Outreach Program–Abbott Northwestern Hospital, Minneapolis, MN 55407 (Daniel M. Anderson, MD); Montana Cancer Consortium, Billings, MT 59101 (Benjamin T. Marchello, MD); Illinois Oncology Research Association CCOP, Peoria, IL 61615-7818 (John W. Kugler, MD); Upstate Carolina CCOP, Spartanburg, SC 29303 (James D. Bearden III, MD); Wichita Community Clinical Oncology Program, Wichita, KS 67214-3882 (Shaker R. Dakhil, MD).

### Supplemental Methods: Single Nucleotide Polymorphism Genotyping

All single nucleotide polymorphism (SNP) genotyping was performed using the MassARRAY Compact system (Sequenom, San Diego, CA). Briefly, the protocol involved polymerase chain reaction (PCR) amplification of the DNA samples using SNP-specific primers, followed by a base extension reaction using the iPLEX Gold chemistry (Sequenom). All SNP-specific and mass extend oligonucleotides were designed using RealSNP and MassARRAY Assay Designer (Sequenom) and purchased from Integrated DNA Technologies (Coralville, IA). HotStart Taq Polymerase (Qiagen, Santa Clarita, CA) was used for all PCR amplifications. Five nanograms of DNA were added to each 5-μL PCR reaction mixture in 384-well plates. PCR primers were assigned into appropriate multiplex groups by RealSNP and used at a 200 nmol/L final concentration. The PCR condition was set for 94°C for 15 minutes for hot start, followed by denaturing at 94°C for 20 seconds, annealing at 56°C for 30 seconds, extension at 72°C for 1 minute for 45 cycles, and final incubation at 72°C for 3 minutes. The PCR products were treated with 2-μL shrimp alkaline phosphatase (Sequenom) for 20 minutes at 37°C and then ramped to 85°C for 5 minutes to remove excess deoxyribonucleotide triphosphates (Ding C, Cantor CR. Proc Natl Acad Sci U S A 100:7449-7453, 2003) before performing the base extension reaction with Sequenom iplex enzyme and cleaning the products with SpectroCLEAN (Sequenom) resin. All reactions were carried out in a PTC225 thermal cycler (MJ Research, St Bruno, Quebec, Canada). The final PCR products were spotted to the appropriate locations on a 384-pad SpectroCHIP (Sequenom) using a ChipSpotter LT nanodispenser (Samsung, Seoul, South Korea) and analyzed to acquire data by mass spectrometry using the MassARRAY Analyzer Compact MALDI-TOF MS (Sequenom). Genotyping calls were performed in real time by the MassARRAY Typer Analyzer v3.4 (Sequenom).

For quality control purposes, each SNP assay was validated on DNA samples from the Coriell Cell Repository ([http://ccr.coriell.org/](http://ccr.coriell.org/)), SNP 500 DNA set, and three pedigree families to allow identification and removal of poor-quality assays before analyzing experimental samples. Furthermore, DNA samples with known genotypes were included in the sample plates to serve as positive controls during the genotyping assays.

### Table A1.

| Gene and SNP Location/Genotype | Gene ID/Chromosomal Location | dbSNP ID | No. of Patients | Genotype Frequency (%) |  |  | MAF* |
| --- | --- | --- | --- | --- | --- | --- | --- |
| Wild Type | Heterozygous | Variant |  |  |  |  |  |
| FPGS | 2356/9q34.1 |   |   |   |   |   |   |
| 5′-FR(−747)C→T† |   | —‡ | 36 | CC (94) | CT§ (6) | TT (0) | 0.028 |
| 5′-FR(−573)C→G |   | — | 38 | CC (71) | CG (24) | GG§ (5) | 0.171 |
| 5′-UTR(−63)G→A‖ |   | rs10760502 | 36 | GG (45) | GA (18) | AA (36) | 0.458 |
| IVS1(28)G→A† |   | — | 33 | GG (100) | GA (0) | AA (0) | 0.000 |
| IVS9(48)C→T† |   | rs41307463 | 36 | CC (100) | CT (0) | TT (0) | 0.000 |
| GGH¶ | 8836/8q12.3 |   |   |   |   |   |   |
| Exon 1(91)C→T Ala31Thr |   | rs11545077 | 37 | CC (54) | CT (43) | TT§ (3) | 0.243 |
| IVS1(1307)C→T |   | rs3780126 | 36 | CC (33) | CT (56) | TT (11) | 0.388 |
| IVS2(2442)T→C |   | rs11990678 | 37 | TT (84) | TC (16) | CC (0) | 0.081 |
| IVS3(1950)A→T |   | rs3780130 | 42 | AA (52) | AT (48) | TT (0) | 0.238 |
| IVS5(1042)T→C |   | rs7010484 | 36 | TT (47) | TC (42) | CC (11) | 0.319 |
| IVS7(1478)G→A |   | rs11995525 | 44 | GG (59) | GA (39) | AA§ (2) | 0.216 |
| IVS7(6177)C→A |   | rs12677953 | 36 | CC (83) | CA (17) | AA (0) | 0.083 |
| SLC19A1 | 6573/21q22.3 |   |   |   |   |   |   |
| IVS2(4935)G→A |   | rs914232 | 43 | GG (28) | GA (40) | AA (33) | 0.523 |
| IVS4(2117)C→T |   | rs2838958 | 36 | CC (39) | CT (47) | TT (14) | 0.375 |
| IVS5(391)C→T |   | rs2297291 | 42 | CC (17) | CT (48) | TT (36) | 0.595 |
| IVS5(707)C→T |   | rs2838956 | 44 | CC (50) | CT (32) | TT (18) | 0.341 |
| IVS5(9148)C→A |   | rs3788189 | 35 | CC (40) | CA (40) | AA (20) | 0.400 |
| Exon 6(2522)C→T (3′-UTR) |   | rs1051298 | 43 | CC (16) | CT (40) | TT (44) | 0.640 |

Table Caption: Distribution of Patient Genotype and Minor Allele Frequencies

## Footnotes

## AUTHORS' DISCLOSURES OF POTENTIAL CONFLICTS OF INTEREST

The author(s) indicated no potential conflicts of interest.

## AUTHOR CONTRIBUTIONS

**Conception and design:** Alex A. Adjei, Sumithra J. Mandrekar, Grace K. Dy, Julian R. Molina, David R. Gandara, Ralph G. Zinner

**Administrative support:** Alex A. Adjei, Sumithra J. Mandrekar, Grace K. Dy, Julian R. Molina, Katie L. Allen Ziegler, Steven E. Schild

**Provision of study materials or patients:** Grace K. Dy, Julian R. Molina, David R. Gandara, Philip J. Stella, Kendrith M. Rowland Jr, Ralph G. Zinner

**Collection and assembly of data:** Sumithra J. Mandrekar, Araba A. Adjei, Katie L. Allen Ziegler

**Data analysis and interpretation:** Alex A. Adjei, Sumithra J. Mandrekar, Araba A. Adjei, Katie L. Allen Ziegler

**Manuscript writing:** Alex A. Adjei, Sumithra J. Mandrekar, Araba A. Adjei, Katie L. Allen Ziegler, Steven E. Schild

**Final approval of manuscript:** Alex A. Adjei, Sumithra J. Mandrekar, Grace K. Dy, Julian R. Molina, Araba A. Adjei, David R. Gandara, Katie L. Allen Ziegler, Philip J. Stella, Kendrith M. Rowland Jr, Steven E. Schild, Ralph G. Zinner

## References

1. Jemal A, Siegel R, Ward E, et al. Cancer statistics 2008. CA Cancer J Clin. 2008;58:71–96. doi: 10.3322/CA.2007.0010.  [DOI](https://doi.org/10.3322/CA.2007.0010) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18287387/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=CA%20Cancer%20J%20Clin&title=Cancer%20statistics%202008&author=A%20Jemal&author=R%20Siegel&author=E%20Ward&volume=58&publication_year=2008&pages=71-96&pmid=18287387&doi=10.3322/CA.2007.0010&)

2. Schilsky RL. Antimetabolites. In: Perry MC, editor. The Chemotherapy Source Book. Baltimore, MD: Williams & Wilkin; 1992. pp. 301–315.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=The%20Chemotherapy%20Source%20Book&author=RL%20Schilsky&publication_year=1992&)

3. Schultz RM, Patel VF, Worzalla JF, et al. Role of thymidylate synthase in the antitumor activity of the multitargeted antifolate, LY231514. Anticancer Res. 1999;19:437–443.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/10226579/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anticancer%20Res&title=Role%20of%20thymidylate%20synthase%20in%20the%20antitumor%20activity%20of%20the%20multitargeted%20antifolate,%20LY231514&author=RM%20Schultz&author=VF%20Patel&author=JF%20Worzalla&volume=19&publication_year=1999&pages=437-443&pmid=10226579&)

4. Grem JL. Fluorinated pyrimidines. In: Chabner BA, Collins JM, editors. Cancer Chemotherapy: Principles and Practice. Philadelphia, PA: JB Lippincott; 1990.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Cancer%20Chemotherapy:%20Principles%20and%20Practice&author=JL%20Grem&publication_year=1990&)

5. Shih C, Chen VJ, Gossett LS, et al. LY231514, a pyrrolo[2,3-d]pyrimidine-based antifolate that inhibits multiple folate-requiring enzymes. Cancer Res. 1997;57:1116–1123.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/9067281/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=LY231514,%20a%20pyrrolo%5B2,3-d%5Dpyrimidine-based%20antifolate%20that%20inhibits%20multiple%20folate-requiring%20enzymes&author=C%20Shih&author=VJ%20Chen&author=LS%20Gossett&volume=57&publication_year=1997&pages=1116-1123&pmid=9067281&)

6. Molina JR, Adjei AA. The role of pemetrexed in lung cancer therapy. Clin Lung Cancer. 2003;5:21–27. doi: 10.3816/CLC.2003.n.017.  [DOI](https://doi.org/10.3816/CLC.2003.n.017) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14596699/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Lung%20Cancer&title=The%20role%20of%20pemetrexed%20in%20lung%20cancer%20therapy&author=JR%20Molina&author=AA%20Adjei&volume=5&publication_year=2003&pages=21-27&pmid=14596699&doi=10.3816/CLC.2003.n.017&)

7. Gorlick R, Cole P, Banerjee D, et al. Mechanisms of methotrexate resistance in acute leukemia: Decreased transport and polyglutamylation. Adv Exp Med Biol. 1999;457:543–550. doi: 10.1007/978-1-4615-4811-9_59.  [DOI](https://doi.org/10.1007/978-1-4615-4811-9_59) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10500832/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Adv%20Exp%20Med%20Biol&title=Mechanisms%20of%20methotrexate%20resistance%20in%20acute%20leukemia:%20Decreased%20transport%20and%20polyglutamylation&author=R%20Gorlick&author=P%20Cole&author=D%20Banerjee&volume=457&publication_year=1999&pages=543-550&pmid=10500832&doi=10.1007/978-1-4615-4811-9_59&)

8. Gorlick R, Goker E, Trippett T, et al. Defective transport is a common mechanism of acquired methotrexate resistance in acute lymphocytic leukemia and is associated with decreased reduced folate carrier expression. Blood. 1997;89:1013–1018.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/9028333/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Defective%20transport%20is%20a%20common%20mechanism%20of%20acquired%20methotrexate%20resistance%20in%20acute%20lymphocytic%20leukemia%20and%20is%20associated%20with%20decreased%20reduced%20folate%20carrier%20expression&author=R%20Gorlick&author=E%20Goker&author=T%20Trippett&volume=89&publication_year=1997&pages=1013-1018&pmid=9028333&)

9. Wong SC, Zhang L, Witt TL, et al. Impaired membrane transport in methotrexate-resistant CCRF-CEM cells involves early translation termination and increased turnover of a mutant reduced folate carrier. J Biol Chem. 1999;274:10388–10394. doi: 10.1074/jbc.274.15.10388.  [DOI](https://doi.org/10.1074/jbc.274.15.10388) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10187828/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Impaired%20membrane%20transport%20in%20methotrexate-resistant%20CCRF-CEM%20cells%20involves%20early%20translation%20termination%20and%20increased%20turnover%20of%20a%20mutant%20reduced%20folate%20carrier&author=SC%20Wong&author=L%20Zhang&author=TL%20Witt&volume=274&publication_year=1999&pages=10388-10394&pmid=10187828&doi=10.1074/jbc.274.15.10388&)

10. Rhee MS, Wang Y, Nair MG, et al. Acquisition of resistance to antifolates caused by enhanced gamma-glutamyl hydrolase activity. Cancer Res. 1993;53:2227–2230.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/7683570/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=Acquisition%20of%20resistance%20to%20antifolates%20caused%20by%20enhanced%20gamma-glutamyl%20hydrolase%20activity&author=MS%20Rhee&author=Y%20Wang&author=MG%20Nair&volume=53&publication_year=1993&pages=2227-2230&pmid=7683570&)

11. Liani E, Rothem L, Bunni MA, et al. Loss of folylpoly-gamma-glutamate synthetase activity is a dominant mechanism of resistance to polyglutamylation-dependent novel antifolates in multiple human leukemia sublines. Int J Cancer. 2003;103:587–599. doi: 10.1002/ijc.10829.  [DOI](https://doi.org/10.1002/ijc.10829) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12494465/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Cancer&title=Loss%20of%20folylpoly-gamma-glutamate%20synthetase%20activity%20is%20a%20dominant%20mechanism%20of%20resistance%20to%20polyglutamylation-dependent%20novel%20antifolates%20in%20multiple%20human%20leukemia%20sublines&author=E%20Liani&author=L%20Rothem&author=MA%20Bunni&volume=103&publication_year=2003&pages=587-599&pmid=12494465&doi=10.1002/ijc.10829&)

12. Zhao R, Babani S, Gao F, et al. The mechanism of transport of the multitargeted antifolate (MTA) and its cross-resistance pattern in cells with markedly impaired transport of methotrexate. Clin Cancer Res. 2000;6:3687–3695.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/10999762/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=The%20mechanism%20of%20transport%20of%20the%20multitargeted%20antifolate%20(MTA)%20and%20its%20cross-resistance%20pattern%20in%20cells%20with%20markedly%20impaired%20transport%20of%20methotrexate&author=R%20Zhao&author=S%20Babani&author=F%20Gao&volume=6&publication_year=2000&pages=3687-3695&pmid=10999762&)

13. Leil TA, Endo C, Adjei AA, et al. Identification and characterization of genetic variation in the folylpolyglutamate synthase gene. Cancer Res. 2007;67:8772–8782. doi: 10.1158/0008-5472.CAN-07-0156.  [DOI](https://doi.org/10.1158/0008-5472.CAN-07-0156) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17875718/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=Identification%20and%20characterization%20of%20genetic%20variation%20in%20the%20folylpolyglutamate%20synthase%20gene&author=TA%20Leil&author=C%20Endo&author=AA%20Adjei&volume=67&publication_year=2007&pages=8772-8782&pmid=17875718&doi=10.1158/0008-5472.CAN-07-0156&)

14. Adjei AA, Dy GK, Mandrekar S, et al. Pemetrexed pharmacogenomics: Haplotype analysis of transport and activation/inactivation gene polymorphisms and the therapeutic outcome in NSCLC, an NCCTG N0026 based study. J Thorac Oncol. 2007;2(suppl 4):pS445.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Thorac%20Oncol&title=Pemetrexed%20pharmacogenomics:%20Haplotype%20analysis%20of%20transport%20and%20activation/inactivation%20gene%20polymorphisms%20and%20the%20therapeutic%20outcome%20in%20NSCLC,%20an%20NCCTG%20N0026%20based%20study&author=AA%20Adjei&author=GK%20Dy&author=S%20Mandrekar&volume=2&issue=suppl%204&publication_year=2007&pages=pS445&)

15. Barrett JC, Fry B, Maller J, et al. Haploview: Analysis and visualization of LD and haplotype maps. Bioinformatics. 2005;21:263–265. doi: 10.1093/bioinformatics/bth457.  [DOI](https://doi.org/10.1093/bioinformatics/bth457) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15297300/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=Haploview:%20Analysis%20and%20visualization%20of%20LD%20and%20haplotype%20maps&author=JC%20Barrett&author=B%20Fry&author=J%20Maller&volume=21&publication_year=2005&pages=263-265&pmid=15297300&doi=10.1093/bioinformatics/bth457&)

16. Omura K. Clinical implications of dihydropyrimidine dehydrogenase (DPD) activity in 5-FU-based chemotherapy: Mutations in the DPD gene, and DPD inhibitory fluoropyrimidines. Int J Clin Oncol. 2003;8:132–138. doi: 10.1007/s10147-003-0330-z.  [DOI](https://doi.org/10.1007/s10147-003-0330-z) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12851836/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Clin%20Oncol&title=Clinical%20implications%20of%20dihydropyrimidine%20dehydrogenase%20(DPD)%20activity%20in%205-FU-based%20chemotherapy:%20Mutations%20in%20the%20DPD%20gene,%20and%20DPD%20inhibitory%20fluoropyrimidines&author=K%20Omura&volume=8&publication_year=2003&pages=132-138&pmid=12851836&doi=10.1007/s10147-003-0330-z&)

17. Côté JF, Kirzin S, Kramar A, et al. UGT1A1 polymorphism can predict hematologic toxicity in patients treated with irinotecan. Clin Cancer Res. 2007;13:3269–3275. doi: 10.1158/1078-0432.CCR-06-2290.  [DOI](https://doi.org/10.1158/1078-0432.CCR-06-2290) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17510208/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=UGT1A1%20polymorphism%20can%20predict%20hematologic%20toxicity%20in%20patients%20treated%20with%20irinotecan&author=JF%20C%C3%B4t%C3%A9&author=S%20Kirzin&author=A%20Kramar&volume=13&publication_year=2007&pages=3269-3275&pmid=17510208&doi=10.1158/1078-0432.CCR-06-2290&)

18. Fossella FV, DeVore R, Kerr RN, et al. Randomized phase III trial of docetaxel versus vinorelbine or ifosfamide in patients with advanced non–small-cell lung cancer previously treated with platinum containing chemotherapy regimens. J Clin Oncol. 2000;18:2354–2362. doi: 10.1200/JCO.2000.18.12.2354.  [DOI](https://doi.org/10.1200/JCO.2000.18.12.2354) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10856094/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Randomized%20phase%20III%20trial%20of%20docetaxel%20versus%20vinorelbine%20or%20ifosfamide%20in%20patients%20with%20advanced%20non%E2%80%93small-cell%20lung%20cancer%20previously%20treated%20with%20platinum%20containing%20chemotherapy%20regimens&author=FV%20Fossella&author=R%20DeVore&author=RN%20Kerr&volume=18&publication_year=2000&pages=2354-2362&pmid=10856094&doi=10.1200/JCO.2000.18.12.2354&)

19. Hanna N, Shepher FA, Fossella F, et al. Randomized phase III trial of pemetrexed versus docetaxel in patients with non–small-cell lung cancer previously treated with chemotherapy. J Clin Oncol. 2004;22:1589–1597. doi: 10.1200/JCO.2004.08.163.  [DOI](https://doi.org/10.1200/JCO.2004.08.163) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15117980/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Randomized%20phase%20III%20trial%20of%20pemetrexed%20versus%20docetaxel%20in%20patients%20with%20non%E2%80%93small-cell%20lung%20cancer%20previously%20treated%20with%20chemotherapy&author=N%20Hanna&author=FA%20Shepher&author=F%20Fossella&volume=22&publication_year=2004&pages=1589-1597&pmid=15117980&doi=10.1200/JCO.2004.08.163&)
