# A GENOME-WIDE ASSOCIATION STUDY OF BRONCHODILATOR RESPONSE IN LATINOS IMPLICATES RARE VARIANTS

## Metadata
**Authors:** Katherine A Drake, Dara G Torgerson, Christopher R Gignoux, Joshua M Galanter, Lindsey A Roth, Scott Huntsman, Celeste Eng, Sam S Oh, Sook Wah Yee, Lawrence Lin, Carlos D Bustamante, Andrés Moreno-Estrada, Karla Sandoval, Adam Davis, Luisa N Borrell, Harold J Farber, Rajesh Kumar, Pedro C Avila, Emerita Brigino-Buenaventura, Rocio Chapela, Jean G Ford, Michael A LeNoir, Fred Lurmann, Kelley Meade, Denise Serebrisky, Shannon Thyne, William Rodríguez-Cintrón, Saunak Sen, José R Rodríguez-Santana, Ryan D Hernandez, Kathleen M Giacomini, Esteban G Burchard
**Journal:** The Journal of allergy and clinical immunology
**Date:** 2013 Aug 29
**DOI:** [10.1016/j.jaci.2013.06.043](https://doi.org/10.1016/j.jaci.2013.06.043)
**PMID:** 23992748
**PMCID:** PMC3938989
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3938989/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC3938989/pdf/nihms522020.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC3938989/pdf/nihms522020.pdf)

## Abstract

**Rationale:** 
The primary rescue medication to treat acute asthma exacerbation is short-acting β2- adrenergic receptor (β2AR) agonists (SABAs), however there is variation in how well an individual responds to treatment. Although these differences may be due to environmental factors, there is mounting evidence for a genetic contribution to variability in bronchodilator drug response (BDR).

**Methods:** 
We performed a genome-wide association study (GWAS) for BDR in 1,782 Latino children with asthma using standard linear regression, adjusting for genetic ancestry and ethnicity, and performed replication studies in an additional 531 Latinos. We also performed admixture mapping across the genome by testing for an association between local European, African, and Native American ancestry and BDR, adjusting for genomic ancestry and ethnicity.

**Results:** 
We identified seven genetic variants associated with BDR at a genome-wide significant threshold (p<5×10−8), all of which had frequencies below 5%. Furthermore, we observed an excess of small p-values driven by rare variants (frequency < 5%), and by variants in the proximity of solute carrier (SLC) genes. Admixture mapping identified five significant peaks; fine mapping within these peaks identified two rare variants in SLC22A15 as being associated with increased BDR in Mexicans. Quantitative PCR and immunohistochemistry identified SLC22A15 as being expressed in the lung and bronchial epithelial cells.

**Conclusion:** 
Our results suggest that rare variation contributes to individual differences in response to albuterol in Latinos, notably in solute carrier genes that include membrane transport proteins involved in the transport of endogenous metabolites and xenobiotics. Resequencing in larger, multi-ethnic population samples and additional functional studies are required to further understand the role of rare variation in BDR.

Keywords: bronchodilator drug response, genome-wide association study, admixture mapping, Latinos, asthma, rare variants

### Rationale

The primary rescue medication to treat acute asthma exacerbation is short-acting β_2_- adrenergic receptor (β_2_AR) agonists (SABAs), however there is variation in how well an individual responds to treatment. Although these differences may be due to environmental factors, there is mounting evidence for a genetic contribution to variability in bronchodilator drug response (BDR).

### Methods

We performed a genome-wide association study (GWAS) for BDR in 1,782 Latino children with asthma using standard linear regression, adjusting for genetic ancestry and ethnicity, and performed replication studies in an additional 531 Latinos. We also performed admixture mapping across the genome by testing for an association between local European, African, and Native American ancestry and BDR, adjusting for genomic ancestry and ethnicity.

### Results

We identified seven genetic variants associated with BDR at a genome-wide significant threshold (p<5×10^−8^), all of which had frequencies below 5%. Furthermore, we observed an excess of small p-values driven by rare variants (frequency < 5%), and by variants in the proximity of solute carrier (SLC) genes. Admixture mapping identified five significant peaks; fine mapping within these peaks identified two rare variants in *SLC22A15* as being associated with increased BDR in Mexicans. Quantitative PCR and immunohistochemistry identified *SLC22A15* as being expressed in the lung and bronchial epithelial cells.

### Conclusion

Our results suggest that rare variation contributes to individual differences in response to albuterol in Latinos, notably in solute carrier genes that include membrane transport proteins involved in the transport of endogenous metabolites and xenobiotics. Resequencing in larger, multi-ethnic population samples and additional functional studies are required to further understand the role of rare variation in BDR.

## Introduction

Short-acting β_2_-adrenergic receptor (β_2_AR) agonists (SABAs) are the primary rescue medication for individuals having an asthma attack^1,2^. SABAs cause rapid bronchodilation, or smooth muscle relaxation in the airways, by stimulating β_2_AR. Response to SABAs is often measured by bronchodilator response (BDR), the percent change in forced expiratory volume in one second (FEV_1_) after administration of a SABA. There is wide inter-individual variability in BDR and not every patient responds to treatment ^2,3^. This variability may be caused by several factors including asthma severity, drug-drug interactions, environmental exposures, age, and genetic factors^2^.

A total of five candidate genes^4–10^ and one gene from a genome-wide association study (GWAS)^11^ have been reported as having variation associated with BDR. However, all together, these results provide limited information about the genetic contributions to variation in BDR. The most commonly studied candidate gene is adrenoceptor beta 2, surface (*ADRB2*), which encodes the direct target of SABAs (β_2_AR). However, evidence for the association of variation in *ADRB2* and BDR is inconsistent, which may be in part be due to variation in the single nucleotide polymorphisms (SNPs) studied, varying clinical endpoints/study conditions, and differences in the assumed genetic model^12^. Four other candidate genes have been studied to a lesser degree, including *CRHR2*, *ADCY9*, *ARG1*, and *THRB*^7–10^. Lastly, a recent GWAS of BDR in European Americans identified *SPATS2L* as a novel candidate gene for regulating β_2_AR levels^11^. However, there are likely additional genes that contribute to differences in BDR that remain to be identified, notably in non-European populations.

BDR has been shown to vary between four ethnic groups in the US. Despite having more severe asthma, Puerto Ricans have lower BDR as compared to African Americans or Mexicans^13,14^, and African Americans have lower BDR as compared to European Americans ^15^. Although these differences may be due to environmental factors, they may also be due to varying frequencies of genetic variants affecting BDR between populations. Specifically, these differences may reside in the ancestral populations of admixed individuals, including Latinos with varying degrees of African, European, and Native American ancestry^5,13,14^. Therefore, admixture mapping, which tests for a correlation between BDR and local genetic ancestry (rather than genotype at an individual SNP), is a complementary strategy to a traditional GWAS that may identify novel genetic loci associated with BDR.

In the current study, we performed a genome-wide association and admixture mapping study for BDR in 1,782 Latinos with asthma from across the US and Puerto Rico, and attempted to replicate our findings in an additional 266 Mexicans and 265 Puerto Ricans with asthma.

## Methods

### Study Subjects

#### GALA II

Patients with asthma in the discovery phase were recruited from the Genes- Environments & Admixture in Latino Americans (GALA II) study, which began in 2006 and is an ongoing, clinical study of children between the ages of 8–21 both with and without asthma. A total of 4,045 subjects including 1,976 patients with asthma were recruited from urban study centers across the mainland U.S. and Puerto Rico through June 2011 ([Table S1](#SD1)). All participants who met criteria for enrollment ([Table S2](#SD1)) completed in-person questionnaires related to their medical, asthma, allergic, social, environmental and demographic histories. In addition, all participants provided blood for genetic analysis and underwent spirometry. Each participant or parent was also required to identify all four grandparents as Latino. Local institutional review boards approved the studies and all subjects and legal guardians provided written informed assent/consent.

Bronchodilator response (BDR) was calculated as the percent change in forced expiratory volume (FEV_1_) following two doses of albuterol (post-FEV_1_) as compared to baseline prior to administering albuterol (pre-FEV_1_). Specifically, FEV_1_ was measured prior to administering albuterol and again after two doses of albuterol, with a 15-minute waiting period following each dose. A total of six (if <16 years of age) to eight (if >=16 years of age) total puffs of albuterol were administered. A total of 408 patients from the Centro de Neumologia Pediatrica in Puerto Rico were recruited based on having a BDR of at least 8%; of these, 121 patients were recruited based on having a BDR of at least 12%.

Self-reported ethnicity was divided into categories of Puerto Rican, Mexican, Mixed Latino, and Other Latino. Puerto Rican and Mexican ethnicities were available as selections on the questionnaire, whereas Mixed Latinos were defined as any individual who identified with more than one Latino ethnic group. Other Latinos were defined as those who chose only one Latino ethnic group from among Spanish/Hispanic/Latino, Cuban, Dominican, El Salvadorian, Guatemalan, Nicaraguan, Honduran, Colombian, Brazilian, and Argentinian. We performed our analyses in all of GALA II, and separately in the subsets of self-reported Puerto Ricans and Mexicans.

#### GALA I

Patients with asthma in the replication phase were participants in the Genetics of Asthma in Latino Americans (GALA I) Study^13^. Briefly, GALA I is a study of subjects with asthma (probands) age 8–40 and their biological parents recruited from schools, clinics, and hospitals across four sites: San Francisco Bay Area, New York City, Puerto Rico, and Mexico City. All probands who met criteria for enrollment ([Table S2](#SD1)) completed in-person questionnaires related to their medical, asthma, allergic, and demographic histories. One of these criteria was to have self-reported ethnicity of either Mexican or Puerto Rican in all four of the proband’s biological grandparents. All participants with asthma provided blood for genetic analysis and underwent spirometry before and after administration of albuterol. The Institutional Review Board at the University of California San Francisco and all recruitment sites approved the study. BDR was measured in a similar fashion as GALA II but with a lower dosage of albuterol. Specifically, post-FEV1 was measured after only a single dose of albuterol (as compared to two doses in GALA II). A total of two (if <16 years of age) to four (if >=16 years of age) total puffs of albuterol were administered (as compared to four [if <16 years of age] and six [if >=16 years of age] in GALA II).

### Genotyping and Quality Control

Genotyping of GALA II subjects was performed using the Axiom LAT1 array (World Array 4, Affymetrix, Santa Clara, CA) that contained 817,810 SNPs prior to quality control (QC). This array was specifically designed for Latino populations, and was shown to capture 80% of all variants with frequencies > 1% with an r*2* of 0.8 in the MXL population (Mexicans in Los Angeles, CA) and with an r^2^ of 0.7 in the PUR population (Puerto Ricans in Puerto Rico) from the 1000 Genomes interim June 2011 release^16^. We removed SNPs with <95% call rates and failed platform specific SNP quality criteria (n=63,328), those that deviated from Hardy-Weinberg equilibrium within their respective populations (Puerto Rican, Mexican, Other Latino) (p<10^−6^, n=1,845), and those not present in the Phase II HapMap (to create a consensus dataset to estimate local genetic ancestry, n=179,092). The total number of SNPs included in our analysis was 568,037. Subjects were filtered based on 97% call rates, discrepancy between genetic sex and reported gender, cryptic relatedness (identity by descent [IBD>0.3), and standard Affymetrix Axiom metrics. We also removed 136 individuals with missing BDR and 3 individuals who were outliers for BDR (BDR > 80 or < −50). Following QC, a total of 1,782 subjects with asthma were included in this study.

Genotyping and estimates of genetic ancestry for GALA I have been described previously^17^. Briefly, GALA I subjects were genotyped using the Affymetrix 6.0 GeneChip and after quality control filtering on markers and subjects, genotypes were available for 729,685 markers in 529 children with asthma (253 Mexican and 276 Puerto Rican subjects).

### Genetic Ancestry Estimation

We used genotypes from three populations to represent the ancestral haplotypes of Latinos for estimating local genetic ancestry: HapMap European (CEU), HapMap African (YRI) and 95 Native Americans. First, we combined the CEU and YRI genotypes across Phase II, Illumina Omni, and Affymetrix Axiom platforms for maximum coverage. In addition, we genotyped 95 Native American samples on the Axiom LAT1 array. Global ancestry was estimated using ADMIXTURE^18^, unsupervised and assuming 3 ancestral populations ([Figure S1](#SD1)). Local ancestry was estimated using LAMP under a 3-population model, assuming 20 generations of admixture^19,20^ after phasing the ancestral haplotypes using BEAGLE^21^.

### Replication

We attempted replication of significantly associated SNPs using *in silico* genotype imputation followed by direct genotyping of two SNPs using a Taqman SNP genotyping assay (Applied Biosystems, Carslbad, CA; rs1281748 and rs1281743). For *in silico* replication we first phased the data using the program SHAPE-IT^22^ accounting for relationships within trios. We then imputed 100kb regions around each SNP with a 20kb buffer on each side using the program IMPUTE2^16^ separately in the GALA I Mexicans and Puerto Ricans. Reference haplotypes for imputation were from Phase I version 3 of the 1000 Genomes Project^23^. Finally, we removed SNPs that had an info score < 0.5 and used the gene dosage output for all analyses to account for uncertainty in imputation.

### Genotype Association Testing

All statistical analyses were conducted using R (v2.14.1)^24^. For each SNP we used linear regression to test whether the number of minor alleles was associated with BDR after adjusting for ethnicity, local and global African and Native American ancestry. In addition, we performed the same analysis without adjusting for ethnicity separately in self-identified Puerto Ricans and Mexicans. Confidence bands on QQ plots were determined using a β(j, n-j+1) distribution for the j^th^ order statistic with n SNPs being tested^25^.

For each genome-wide significant SNP (p<5×10^−8^), we performed *in silico* fine mapping within 200 Kb up- and down-stream using IMPUTE2^16^, and using reference haplotypes from all available populations in the Phase I version 3 of the 1000 Genomes Project^23^. We also examined the dose-response relationship between the number of minor alleles and BDR, both before and after removing individuals with a BDR > 60 who were at the upper end of the BDR distribution (but were not considered statistical outliers, which were removed during QC). SNPs that were genome-wide significant and met three criteria were validated using Sanger sequencing and carried forward for replication in GALA I: 1) > 5 copies of the minor allele, 2) a consistent dose-response relationship after removing individuals with BDR > 60, and 3) high quality control measures and passed visual inspection of cluster plots. We also attempted replication at an additional imputed SNP in GALA II identified as genome-wide significant via *in silico* fine mapping. Replication was performed using linear regression to test for an association between minor allele dosage and BDR separately in Mexicans and Puerto Ricans, adjusting for local and global African and Native American ancestry.

### Admixture Mapping

We also tested for an association between local ancestry (African, Native American, and European) and BDR in Mexicans and Puerto Ricans using linear regression and adjusting for global ancestry. Traditional thresholds for genome-wide significance using allelic associations are too stringent for ancestry associations due to increased ancestry-LD relative to SNP-LD. Therefore, we ran 100 random permutations of BDR maintaining the haplotype structure and the link between BDR and ethnicity to establish a threshold for statistical significance of p<1.6×10^−4^ for African ancestry and p<1.0×10^−4^ for European and Native American ancestry. We attempted to replicate significant admixture mapping peaks in GALA I by testing for an association of BDR and local ancestry in the same manner as described above for GALA II.

### Quantification of Gene Expression

The expression of *SLC22A15* was quantified in 12 human tissues using real-time reverse transcription-PCR (rtPCR), and visualized in normal human lung tissue using immunohistochemistry available from the Human Protein Atlas ([http://www.proteinatlas.org/ENSG00000163393/normal/bronchus](http://www.proteinatlas.org/ENSG00000163393/normal/bronchus)). Total RNA of 13 human tissues (heart, adipose tissue, lung, placenta, total brain, small intestine, bladder, skeletal muscle, esophagus, peripheral blood monocyte, kidney and liver) was purchased from Clonetech. Each of these tissues was pooled from different individuals as specified in the product insert from Clonetech. Two micrograms of RNA were reverse transcribed to cDNA using a High Capacity cDNA Reverse Transcription kit (Applied Biosystems) using the following thermal cycler conditions: 10 min at 22 °C, 2 hours at 37 °C, 5 min at 4 °C, 10 min at 75 °C and 5 min at 4 °C. Exonuclease I enzyme (10U/mL) was added to each sample and the following incubation conditions were used to remove excess primers: 1 hour at 37 °C, 5 min at 4 °C and 10 min at 85 °C to inactivate the exonuclease enzyme. Samples were then stored at −20 °C. Real-time PCR was performed using Biotrove OpenArray technology (Life Technologies). The primers that were used to detect SLC22A15 transcripts are AACTCGCTTCCTGGTGGGCATGA (forward strand) and AGGCCGCCAATCGATCCTGCAA (reverse strand). The housekeeping genes used in this experiment were human beta-actin (*ACTB*), glyceraldehyde 3-phosphate dehydrogenase (*GAPDH*) and beta-2-microglobulin (*B2M*); primers for these genes were predesigned by Biotrove OpenArray technology. The mRNA expression level was normalized to the geometric mean of the housekeeping genes, and then the fold difference of *SLC22A15* transcription was determined using the delta-delta Ct method^26^ as described previously^27^. The fold expression in human liver tissue was normalized to 1 and used as a reference for relative quantification.

## Results

### Allelic Association Tests

After quality control, we tested for an association of BDR at a total of 568,037 SNPs in 1,782 children with asthma in all Latino individuals regardless of self-reported ethnicity, and also separately in subsets of 823 Puerto Ricans and 572 Mexicans. In all three analyses we observed an excess of small p-values compared to that expected by chance ([Figure 1](#F1), and [Supplementary Fig. 2](#SD1)). However, this excess disappeared after removing SNPs with a minor allele frequency (MAF) <5%, suggesting the excess of small p-values is being driven by rare variation. We also observed an excess of small p-values in variants in the proximity of solute carrier (SLC) genes ([Supplementary Fig. 3](#SD1)), which include membrane transport proteins involved in the transport of endogenous metabolites and xenobiotics.

### Figure 1. QQ plot for genome-wide allelic associations with bronchodilator response.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/79a3/3938989/24d28757cb9d/nihms522020f1.jpg)

The expected distribution of p-values is based on a uniform distribution. Black circles indicate all SNPs (inflation factor, (λ=1.005), whereas blue circles include only common SNPs with a MAF >= 5% (λ=1.004). The shaded region represents the 95% concentration band. QQ plots for the subset of Puerto Ricans and Mexicans are shown in Supplementary Figure 2.

A total of six SNPs were genome-wide significant in all subjects combined (p<5×10^−8^, [Figure 2A](#F2)). Of these SNPs, two had minor alleles that were present in at least five individuals (rs8191725 and rs77441273, [Table 1](#T1)), and both had a dose-response relationship suggesting the minor allele confers an increase in BDR ([Table 1](#T1)). This relationship remained consistent after removing individuals with BDR > 60, indicating the dose-response relationship is not being driven entirely by individuals with more extreme BDR. In the Puerto Rican and Mexican subsets, 7 and 6 rare SNPs were genome-wide significant, respectively ([Figure 2B and 2C](#F2)). Three of these SNPs in Puerto Ricans and two in Mexicans had minor alleles present in more than three individuals ([Table 1](#T1)). Only one of these variants was present in more than five individuals and taken forward for replication (rs77977790 in the Puerto Rican subset, [Table 1](#T1)). Three of the 7 SNPs that were genome-wide significant in the Puerto Ricans were also significant in all of GALA II; however, in all cases the association was driven by a single heterozygote with extreme BDR (BDR>60). Up to 7 carriers of the minor allele (when present) were validated for 6 of the variants listed in [Table 1](#T1) using Sanger sequencing. All genotypes were successfully validated with the exception of rs74973995 in a single individual that appeared homozygous. Rs8191725 failed PCR design.

### Figure 2. Manhattan plot of genome-wide association studies with bronchodilator response.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/79a3/3938989/f5280b9b1c88/nihms522020f2.jpg)

Association testing for BDR was performed using linear regression including ethnicity, local, and global African and Native American ancestry as covariates. SNPs meeting a genome-wide significance threshold of p < 5×10−8 are colored in red. Manhattan plots for the subset of Puerto Ricans and Mexicans are shown in Supplementary Figure 4.

### Table 1.

| Population | SNP ID | Gene | Annotation | MAF | p-value | Beta | Mean BDR by copies of minor allele (number of individuals) |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| 0 | 1 | 2 |  |  |  |  |  |  |  |
| All | rs8191725* | IGF2R | Intronic | 0.82% | 3.3×10−9 | 10.2 | 9.9 (1750) | 20.4 (29) | - |
| rs77441273* | SLC24A4 | Missense | 0.17% | 4.0×10−10 | 23.6 | 10 (1776) | 33.6 (6) | - |  |
|   |  |  |  |  |  |  |  |  |  |
| PR | rs77977790* | PAPPA2 | Intronic | 2.8% | 4.6 ×10−10 | 9.5 | 11.5 (776) | 20.6 (46) | - |
| rs77149876 | SPON1 | Intronic | 0.18% | 1.4 ×10−8 | 32.5 | 11.9 (819) | 44.1 (3) | - |  |
| rs115501901 | NCOA3 | 3′UTR | 0.18% | 4.2 ×10−8 | 31.5 | 11.9 (819) | 43.1 (3) | - |  |
|   |  |  |  |  |  |  |  |  |  |
| MX | rs116551936 | intergenic |   | 0.26% | 5.9 ×10−9 | 27.0 | 7.8 (566) | 36 (3) | - |
| rs74973995 | intergenic |   | 0.18% | 1.3 ×10−8 | 32.6 | 7.8 (569) | 39.9 (2) | - |  |

Table 1 Caption: Top signals of allelic association with BDR.

We found no significant associations at SNPs previously reported as being associated with BDR. However, two rare variants in the proximity of *ADCY9* and *CRHR2,* were significantly associated with BDR in our study after adjusting for multiple testing based on the number of SNPs within 50 Kb of each gene ([Table 2](#T2)). No individual SNP was associated with BDR within 50 Kb up- or downstream of *SPATS2L*, a gene identified through GWAS in European Americans^11^.

### Table 2. Top Signals of Allelic Association within 50 Kb of Previously Reported Candidate Genes.

| Gene | SNP ID | Position | p-value | Coefficient | MAF | # SNPs |
| --- | --- | --- | --- | --- | --- | --- |
| ADRB2 | rs10476900 | 148162955 | 0.040 | 1.32 | 10.3% | 56 |
| rs1042713 | 148206440 | 0.71 | −0.37 | 44% |   |  |
| rs1042714 | 148206473 | 0.76 | −0.13 | 22% |   |  |
| ADCY9 | rs144315541 | 4191522 | 0.0004* | −6.66 | 0.7% | 81 |
| rs2230739 | 4033436 | 0.33 | 0.98 | 27% |   |  |
| CRHR2 | rs73294475 | 30701596 | 0.0011* | 3.19 | 4.2% | 39 |
| rs7793837 | 30726777 | 0.98 | −0.0096 | 39% |   |  |
| ARG1 | rs79631493 | 131951607 | 0.077 | 2.00 | 2.6% | 24 |
| rs2781659 | 131891820 | 0.29 | −1.06 | 45% |   |  |
| SPATS2L | rs10203042 | 201354866 | 0.0016 | 4.10 | 2.1% | 71 |
| rs295137 | 201150040 | 0.48 | −0.71 | 38% |   |  |
| rs295114 | 201195602 | 0.73 | 0.12 | 65% |   |  |
| THRB | rs73038406 | 24573150 | 0.010 | −3.97 | 1.5% | 179 |
| rs892940 | 24538838 | 0.34 | 0.96 | 35% |   |  |

Table 2 Caption: Candidate SNPs associated with BDR in prior studies are in italics.

We attempted to replicate a total of three SNPs in GALA I that had been detected in >5 individuals in GALA II, including two SNPs identified in all of GALA II (rs8191725, rs77441273) and one identified in the subset of Puerto Ricans (rs77977790). We also attempted replication of an additional rare variant in the proximity of rs77977790 (rs143010317) that was genome-wide significantly associated with BDR using *in silico* fine mapping in Puerto Ricans in GALA II (β=−58, p=3.2×10^−10^). *In silico* replication via imputation in GALA I resulted in only three polymorphic SNPs with information scores > 0.5. None were significantly associated with BDR (p>0.05, [Table S3](#SD1)), however two of the SNPs showed an effect size in the same direction in Puerto Ricans (rs77441273, β=24.3, p=0.38; rs143010317, β=−12.6, p=0.24). Attempting replication by direct genotyping of three SNPs, we were unable to cluster any heterozygous genotypes for the minor alleles using custom TaqMan SNP Genotyping Assays.

### Admixture Mapping

At each of the 568,037 loci, we tested for an association of BDR with local genetic ancestry in 1,782 children with asthma in the subsets of 823 Puerto Rican and 572 Mexican children ([Table 3](#T3)). In total, we identified 5 admixture mapping peaks that were significantly associated with BDR (permutation p-value < 0.05, [Figure 3](#F3), [Supplemental Fig. 5](#SD1), [Table 3](#T3)). None of the admixture mapping peaks replicated in either Mexicans or Puerto Ricans in GALA I (p > 0.05).

### Table 3.

| Population | Ancestry* | p-value | Genome Coordinates (hg19) |  |  |
| --- | --- | --- | --- | --- | --- |
| Coefficient+ | (p<0.01) | Cytoband |  |  |  |
| Puerto Rican | NAM/EUR | 9.4×10−5a | −2.58/2.05 | chr1: 116204807–117505312 | 1p13 |
| Puerto Rican | NAM/EUR | 7.6×10−6b | 2.96/−2.56 | chr2: 235202022–236278203 | 2q37 |
| Puerto Rican | AFR | 1.7×10−5 | −2.83 | chr1: 157995576–158687163 | 1q23 |
| Puerto Rican | AFR | 1.5×10−5 | 2.69 | chr8: 5585682–6024650 | 8p23 |
| Mexican | AFR | 1.3×10−4 | 3.88 | chr14: 98812269–98932579 | 14q32 |

Table 3 Caption: Location of Admixture Mapping Peaks

### Figure 3. Admixture mapping in GALA II Puerto Ricans for (A) African, (B) Native American, and (C) European ancestry.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/79a3/3938989/24955582f7f4/nihms522020f3.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3938989_nihms522020f3.jpg)

Ancestry association testing for BDR was performed using linear regression including ethnicity, global African, and global Native American ancestry as covariates. Peaks obtaining statistical significance based on permutations are indicated in red. Results for Mexicans are shown in Supplementary Figures 5.

To perform fine mapping, we tested for allelic associations with BDR for SNPs within each peak. Because we restricted our analyses to candidate regions, we applied a Bonferonni correction for the number of SNPs within each peak. Two rare SNPs in the intron and 3′UTR of *SLC22A15* were significantly associated with increased BDR in Mexicans under the peak at 1p13 (rs1281748 and rs1281743, p = 8.8×10^−5^ and MAF=0.26% for both, [Figure 4](#F4)). Carrying one copy of the minor allele at these SNPs was associated with an increase of 18.8% in BDR. These allelic associations were specific to Mexicans in GALA II, despite the admixture mapping peak having been identified in Puerto Ricans. The SNPs were not significantly associated with BDR in the Puerto Rican subset of GALA II (p=0.28 for both), or GALA I (p=0.28 for rs1281748, and p=0.19 for rs1281743). The associations similarly failed to replicate in GALA I Mexicans, however only 1 and 2 individuals carried the minor allele at rs1281748 and rs1281743, respectively ([Table 4](#T4)).

### Figure 4. Top signals of allelic association under the admixture mapping peak at 1p3 in Mexicans.

![Figure 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/79a3/3938989/7996b1867bc5/nihms522020f4.jpg)

Plotted are the –log10(p-values) for tests of allelic association with BDR using linear regression adjusting for ethnicity, local, and genomic African and Native American ancestry. Significant associations were observed at two rare variants in the intron and 3′UTR of SLC22A15 in Mexicans after correcting for a reduced number of tests.

### Table 4.

| Study | rs1281748 |  |  | rs1281743 |  |  |
| --- | --- | --- | --- | --- | --- | --- |
| p-value | Coefficient | MAF | p-value | Coefficient | MAF |  |
| GALA II PR | 0.28 | −2.65 | 1.04% | 0.28 | −2.65 | 1.04% |
| GALA II MX | 8.8×10−5 | 18.8 | 0.26% | 8.8×10−5 | 18.8 | 0.26% |
| GALA I PR | 0.28 | 5.9 | 1.80% | 0.19 | 7.7 | 1.50% |
| GALA I MX | 0.56 | −5.25 | 0.39% | 0.47 | −9.2 | 0.19% |

Table 4 Caption: Top Signals of Allelic Association & Replication under 1p13 Admixture Mapping Peak

Transcription of *SLC22A15* was detected in the lung and additional tissues using RT-PCR ([Figure 5a](#F5)), and was consistent with previous findings that SLC22A15 (protein) is present in the lung^28^. Immunohistochemistry data from the Human Protein Atlas ([http://www.proteinatlas.org/ENSG00000163393/normal/bronchus](http://www.proteinatlas.org/ENSG00000163393/normal/bronchus)) confirmed our results, and demonstrates that *SLC22A15* is expressed within the cytoplasm and membrane of bronchial epithelial cells ([Figure 5b](#F5)).

### Figure 5. Expression levels of SLC22A15 in the lung.

![Figure 5](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/79a3/3938989/3f6e1b26c22a/nihms522020f5.jpg)

(A) Relative transcript abundance across human tissues assessed using quantitative rtPCR; SLC22A15 mRNA was detected at levels over 50,000x higher in the lung as compared to the liver. (B) Immunohistochemistry staining of lung tissue adapted from the Human Protein Atlas (http://www.proteinatlas.org/ENSG00000163393/normal/bronchus), showing the transcription of SLC22A15 (brown) in the cytoplasm and membrane of bronchial epithelial cells (indicated by a black arrow).

## Discussion

We performed genome-wide tests for allelic and local ancestry associations with BDR in 1,782 Latino children with asthma from across the US and Puerto Rico, and found 16 rare variants to be significantly associated with BDR in either Puerto Ricans, Mexicans, or in all of GALA II combined (including Puerto Ricans, Mexicans, and other Latinos) (p<5×10^−8^). All of the associated variants had frequencies at or below 5%, and no common variants were significantly associated with BDR following multiple testing correction. Although associations with rare variation are typically studied through the use of complete resequencing data, the Axiom LAT1 array used in our study was designed to have better coverage of rare variants as compared to other commercial genotyping arrays, and to have saturated coverage at 5,000 genes previously identified by GWAS including those for drug response^29^. Our use of this array may have allowed us to detect associations at rare variants not previously identified in genetic studies of BDR.

In addition, we replicated associations at two candidate genes for BDR (*ADCY9* and *CRHR2*) but at different variants than originally reported; both of the top associations were with rare rather than common variants. We failed to identify any associated SNPs in *SPATS2L,* looking both genome-wide and on a gene-based level. *SPATS2L* was identified as a top candidate in a recent GWAS for BDR in populations of European ancestry, where top signals of allelic associations were observed at two common variants and replicated in independent studies (rs295137 and rs295114, MAF > 40%, neither were genome-wide significant)^11^. One of the SNPs was genotyped in our study and we imputed the other (rs295114, information score=0.992). However, neither was associated with BDR in our study ([Table 2](#T2)), nor did we find any association with local ancestry and BDR at *SPATS2L* (p>0.5). The most significant association within 50 Kb of *SPATS2L* was at a rare variant (rs10203042, p=1.6×10^−3^, [Table 2](#T2)). However, this variant lies within an intron of the neighboring gene *KCTD18*, and was not statistically significant after correcting for 71 tests. There was little to no linkage disequilibrium between rs10203042 and either of the GWAS implicated SNPs (rs295137: r^2^=0.004, D′=0.33; rs295114: r^2^=0.003, D′=0.29). The difference between studies may be due to differences in genetic risk factors between European and Latino populations, or to differences in patterns of linkage disequilibrium at *SPATS2L* (*i.e.* differences in the coverage of variation between different genotyping arrays).

In addition to our allelic association results, we identified five admixture mapping peaks in Puerto Ricans and Mexicans, where local ancestry (European, Native American, or African ancestry) was significantly correlated with BDR. Given that rare variants are more likely than common variants to differ in frequency between the ancestral populations of modern Latinos^30^, the peaks we identified may be driven by differences in patterns of rare variation. In support of this, we identified two rare variants under one of the admixture mapping peaks (1p13) where the minor allele was significantly associated with higher BDR after correcting for a reduced number of tests, whereas we found no significant associations at any common variants.

Five of our top associated variants fell directly within an annotated gene ([Table 1](#T1)). All of these genes are plausible candidates for being associated with BDR, and include an ion transporter (*SLC24A4*), a gene involved in transcriptional regulation (*NCOA3*), and a gene involved in vascular smooth muscle cell adhesion (*SPON1*). Two of the genes, *IGF2R* and *PAPPA2,* are especially noteworthy. *IGF2R* activates *TGFB2*^31^ and is a member of the TGF-β pathway, which is well-studied in asthma susceptibility. *PAPPA2* cleaves IGF-binding protein 5 and may be a local regulator of IGF bioavailability ^32^. IGF is known to be involved in airway inflammation and remodeling^33^. In addition to these genes identified through allelic associations, two rare variants identified under the admixture mapping peak at 1p13 are in *SLC22A15*, a member of the organic ion transporter family (SLC22), which mediates the uptake of pharmaceutical drugs to different tissues used in the treatment of diseases such as cancer and diabetes^34^. These two SNPs are in complete LD; however, one of the SNPs lies within a promoter region identified through ChIP-seq in the ENCODE studies^35^. Although we currently do not know the substrates of SLC22A15, using sequence-based profiling with additional bioinformatics analysis shows that SLC22A15 is clustered closed to SLC22A7, an organic anion transporter^36^, which transports broad substrates of compounds such as antiviral nucleosides analogs and endogenous compounds such as cyclic GMP^27^. Our expression studies and data from the protein atlas demonstrate that *SLC22A15* is expressed in the lung and bronchial epithelial cells; however, additional studies are required to identify whether rare variation in *SLC22A15* contributes to differential transcription.

We attempted to replicate our findings in an independent sample of Latino children with asthma (the GALA I study), which, to the best of our knowledge, is the only other large study of BDR in Latinos. While two of the rare variants showed a consistent direction of effect in Puerto Ricans (rs77441273 and rs143010317), none of the rare variants were significantly associated with BDR in GALA I using *in silico* approaches at p<0.05. Imputation quality scores were generally low for the rare variants we attempted to replicate (0.19–0.88), and direct genotyping of three variants using Taqman assays suggests they may be absent in GALA I. These findings are not surprising given the smaller sample size of the replication study (N=531), and highlights the difficulty in replicating associations at individual rare variants. Apart from differences in statistical power, there are differences in the recruitment of patients between studies (see Methods), notably with respect to self-identified ethnicity with subjects in GALA II having more diverse ethnic backgrounds. Therefore, our results highlight the need for additional studies of BDR in larger samples of Latino populations to fully understand the contributions of individual rare variants to drug response.

In conclusion, we identified an enrichment of small p-values in a GWAS for BDR that was driven by rare variants, and implicated multiple individual rare variants as being significantly associated with BDR in Latino children with asthma. Additional studies involving complete resequencing and larger population samples are required to further understand the genetic contributions to differences in BDR. Future studies are also required to investigate the transport function of SLC22A15 and its relationship to bronchodilation.

## Supplementary Material

## Acknowledgments

**Sources of funding:** This work was supported by grants from the National Institute of Health (HL088133 and HL078885 to EGB, M01-RR00188 to Texas Children’s Hospital General Clinical Research Center), the National Institute On Minority Health And Health Disparities (P60MD006902), the National Institutes of Environmental Health Sciences (ES015794 to EGB), the National Institutes of Health Training Grant (T32 GM07175), the Flight Attendant Medical Research Institute (FAMRI), the Sandler Foundation, and an Ernest Bazley Grant (PCA).

The authors acknowledge the families and patients for their participation and thank the numerous health care providers and community clinics for their support and participation in GALA II. In particular, the authors thank the study coordinator Sandra Salazar; the recruiters who obtained the data: Duanny Alva, MD, Gaby Ayala-Rodriguez, Ulysses Burley, Lisa Caine, Elizabeth Castellanos, Jaime Colon, Denise DeJesus, Iliana Flexas, Blanca Lopez, Brenda Lopez, MD, Louis Martos, Vivian Medina, Juana Olivo, Mario Peralta, Esther Pomares, MD, Jihan Quraishi, Johanna Rodriguez, Shahdad Saeedi, Dean Soto, Ana Taveras, Emmanuel Viera. We would also like to thank Cheryl Winkler, Ph.D. and William Klitz, Ph.D. for providing data that was used to help estimate local ancestry.

## Abbreviations

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Nelson HS. Beta-adrenergic bronchodilators. N Engl J Med. 1995 Aug 24;333(8):499–506. doi: 10.1056/NEJM199508243330807.  [DOI](https://doi.org/10.1056/NEJM199508243330807) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7623883/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Beta-adrenergic%20bronchodilators&author=HS%20Nelson&volume=333&issue=8&publication_year=1995&pages=499-506&pmid=7623883&doi=10.1056/NEJM199508243330807&)

2. Palmer LJ, Silverman ES, Weiss ST, Drazen JM. Pharmacogenetics of asthma. American Journal of Respiratory and Critical Care Medicine. 2002 Apr 1;165(7):861–6. doi: 10.1164/ajrccm.165.7.2109096.  [DOI](https://doi.org/10.1164/ajrccm.165.7.2109096) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11934710/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=American%20Journal%20of%20Respiratory%20and%20Critical%20Care%20Medicine&title=Pharmacogenetics%20of%20asthma&author=LJ%20Palmer&author=ES%20Silverman&author=ST%20Weiss&author=JM%20Drazen&volume=165&issue=7&publication_year=2002&pages=861-6&pmid=11934710&doi=10.1164/ajrccm.165.7.2109096&)

3. Drazen JM, Silverman EK, Lee TH. Heterogeneity of therapeutic responses in asthma. Br Med Bull. 2000;56(4):1054–70. doi: 10.1258/0007142001903535.  [DOI](https://doi.org/10.1258/0007142001903535) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11359637/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20Med%20Bull&title=Heterogeneity%20of%20therapeutic%20responses%20in%20asthma&author=JM%20Drazen&author=EK%20Silverman&author=TH%20Lee&volume=56&issue=4&publication_year=2000&pages=1054-70&pmid=11359637&doi=10.1258/0007142001903535&)

4. Martinez FD, Graves PE, Baldini M, Solomon S, Erickson R. Association between genetic polymorphisms of the beta2-adrenoceptor and response to albuterol in children with and without a history of wheezing. J Clin Invest. 1997 Dec 15;100(12):3184–8. doi: 10.1172/JCI119874.  [DOI](https://doi.org/10.1172/JCI119874) | [PMC free article](/articles/PMC508532/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9399966/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Invest&title=Association%20between%20genetic%20polymorphisms%20of%20the%20beta2-adrenoceptor%20and%20response%20to%20albuterol%20in%20children%20with%20and%20without%20a%20history%20of%20wheezing&author=FD%20Martinez&author=PE%20Graves&author=M%20Baldini&author=S%20Solomon&author=R%20Erickson&volume=100&issue=12&publication_year=1997&pages=3184-8&pmid=9399966&doi=10.1172/JCI119874&)

5. Choudhry S, Ung N, Avila PC, Ziv E, Nazario S, Casal J, et al. Pharmacogenetic differences in response to albuterol between Puerto Ricans and Mexicans with asthma. American Journal of Respiratory and Critical Care Medicine. 2005 Mar 15;171(6):563–70. doi: 10.1164/rccm.200409-1286OC.  [DOI](https://doi.org/10.1164/rccm.200409-1286OC) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15557128/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=American%20Journal%20of%20Respiratory%20and%20Critical%20Care%20Medicine&title=Pharmacogenetic%20differences%20in%20response%20to%20albuterol%20between%20Puerto%20Ricans%20and%20Mexicans%20with%20asthma&author=S%20Choudhry&author=N%20Ung&author=PC%20Avila&author=E%20Ziv&author=S%20Nazario&volume=171&issue=6&publication_year=2005&pages=563-70&pmid=15557128&doi=10.1164/rccm.200409-1286OC&)

6. Silverman EK, Kwiatkowski DJ, Sylvia JS, Lazarus R, Drazen JM, Lange C, et al. Family-based association analysis of beta2-adrenergic receptor polymorphisms in the childhood asthma management program. Journal of Allergy and Clinical Immunology. 2003 Nov;112(5):870–6. doi: 10.1016/s0091-6749(03)02023-2.  [DOI](https://doi.org/10.1016/s0091-6749(03)02023-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14610472/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Allergy%20and%20Clinical%20Immunology&title=Family-based%20association%20analysis%20of%20beta2-adrenergic%20receptor%20polymorphisms%20in%20the%20childhood%20asthma%20management%20program&author=EK%20Silverman&author=DJ%20Kwiatkowski&author=JS%20Sylvia&author=R%20Lazarus&author=JM%20Drazen&volume=112&issue=5&publication_year=2003&pages=870-6&pmid=14610472&doi=10.1016/s0091-6749(03)02023-2&)

7. Poon AH, Tantisira KG, Litonjua AA, Lazarus R, Xu J, Lasky-Su J, et al. Association of corticotropin-releasing hormone receptor-2 genetic variants with acute bronchodilator response in asthma. Pharmacogenet Genomics. 2008 May;18(5):373–82. doi: 10.1097/FPC.0b013e3282fa760a.  [DOI](https://doi.org/10.1097/FPC.0b013e3282fa760a) | [PMC free article](/articles/PMC3208318/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18408560/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Association%20of%20corticotropin-releasing%20hormone%20receptor-2%20genetic%20variants%20with%20acute%20bronchodilator%20response%20in%20asthma&author=AH%20Poon&author=KG%20Tantisira&author=AA%20Litonjua&author=R%20Lazarus&author=J%20Xu&volume=18&issue=5&publication_year=2008&pages=373-82&pmid=18408560&doi=10.1097/FPC.0b013e3282fa760a&)

8. Tantisira KG, Small KM, Litonjua AA, Weiss ST, Liggett SB. Molecular properties and pharmacogenetics of a polymorphism of adenylyl cyclase type 9 in asthma: interaction between beta-agonist and corticosteroid pathways. Human Molecular Genetics. 2005 Jun 15;14(12):1671–7. doi: 10.1093/hmg/ddi175.  [DOI](https://doi.org/10.1093/hmg/ddi175) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15879435/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Human%20Molecular%20Genetics&title=Molecular%20properties%20and%20pharmacogenetics%20of%20a%20polymorphism%20of%20adenylyl%20cyclase%20type%209%20in%20asthma:%20interaction%20between%20beta-agonist%20and%20corticosteroid%20pathways&author=KG%20Tantisira&author=KM%20Small&author=AA%20Litonjua&author=ST%20Weiss&author=SB%20Liggett&volume=14&issue=12&publication_year=2005&pages=1671-7&pmid=15879435&doi=10.1093/hmg/ddi175&)

9. Litonjua AA, Lasky-Su J, Schneiter K, Tantisira KG, Lazarus R, Klanderman B, et al. ARG1 is a novel bronchodilator response gene: screening and replication in four asthma cohorts. American Journal of Respiratory and Critical Care Medicine. 2008 Oct 1;178(7):688–94. doi: 10.1164/rccm.200709-1363OC.  [DOI](https://doi.org/10.1164/rccm.200709-1363OC) | [PMC free article](/articles/PMC2556451/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18617639/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=American%20Journal%20of%20Respiratory%20and%20Critical%20Care%20Medicine&title=ARG1%20is%20a%20novel%20bronchodilator%20response%20gene:%20screening%20and%20replication%20in%20four%20asthma%20cohorts&author=AA%20Litonjua&author=J%20Lasky-Su&author=K%20Schneiter&author=KG%20Tantisira&author=R%20Lazarus&volume=178&issue=7&publication_year=2008&pages=688-94&pmid=18617639&doi=10.1164/rccm.200709-1363OC&)

10. Duan QL, Du R, Lasky-Su J, Klanderman BJ, Partch AB, Peters SP, et al. A polymorphism in the thyroid hormone receptor gene is associated with bronchodilator response in asthmatics. The Pharmacogenomics Journal. Nature Publishing Group. 2012 Jan 3;:1–7. doi: 10.1038/tpj.2011.56.  [DOI](https://doi.org/10.1038/tpj.2011.56) | [PMC free article](/articles/PMC3349771/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22212731/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=The%20Pharmacogenomics%20Journal.%20Nature%20Publishing%20Group&title=A%20polymorphism%20in%20the%20thyroid%20hormone%20receptor%20gene%20is%20associated%20with%20bronchodilator%20response%20in%20asthmatics&author=QL%20Duan&author=R%20Du&author=J%20Lasky-Su&author=BJ%20Klanderman&author=AB%20Partch&publication_year=2012&pages=1-7&pmid=22212731&doi=10.1038/tpj.2011.56&)

11. Himes BE, Jiang X, Hu R, Wu AC, Lasky-Su JA, Klanderman BJ, et al. Genome-Wide Association Analysis in Asthma Subjects Identifies SPATS2L as a Novel Bronchodilator Response Gene. PLoS Genetics. 2012 Jul;8(7):e1002824. doi: 10.1371/journal.pgen.1002824.  [DOI](https://doi.org/10.1371/journal.pgen.1002824) | [PMC free article](/articles/PMC3390407/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22792082/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genetics&title=Genome-Wide%20Association%20Analysis%20in%20Asthma%20Subjects%20Identifies%20SPATS2L%20as%20a%20Novel%20Bronchodilator%20Response%20Gene&author=BE%20Himes&author=X%20Jiang&author=R%20Hu&author=AC%20Wu&author=JA%20Lasky-Su&volume=8&issue=7&publication_year=2012&pages=e1002824&pmid=22792082&doi=10.1371/journal.pgen.1002824&)

12. Contopoulos-Ioannidis DG, Alexiou GA, Gouvias TC, Ioannidis JPA. An empirical evaluation of multifarious outcomes in pharmacogenetics: beta-2 adrenoceptor gene polymorphisms in asthma treatment. Pharmacogenet Genomics. 2006 Oct;16(10):705–11. doi: 10.1097/01.fpc.0000236332.11304.8f.  [DOI](https://doi.org/10.1097/01.fpc.0000236332.11304.8f) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17001289/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=An%20empirical%20evaluation%20of%20multifarious%20outcomes%20in%20pharmacogenetics:%20beta-2%20adrenoceptor%20gene%20polymorphisms%20in%20asthma%20treatment&author=DG%20Contopoulos-Ioannidis&author=GA%20Alexiou&author=TC%20Gouvias&author=JPA%20Ioannidis&volume=16&issue=10&publication_year=2006&pages=705-11&pmid=17001289&doi=10.1097/01.fpc.0000236332.11304.8f&)

13. Burchard EG, Avila PC, Nazario S, Casal J, Torres A, Rodriguez-Santana JR, et al. Lower bronchodilator responsiveness in Puerto Rican than in Mexican subjects with asthma. American Journal of Respiratory and Critical Care Medicine. 2004 Feb 1;169(3):386–92. doi: 10.1164/rccm.200309-1293OC.  [DOI](https://doi.org/10.1164/rccm.200309-1293OC) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14617512/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=American%20Journal%20of%20Respiratory%20and%20Critical%20Care%20Medicine&title=Lower%20bronchodilator%20responsiveness%20in%20Puerto%20Rican%20than%20in%20Mexican%20subjects%20with%20asthma&author=EG%20Burchard&author=PC%20Avila&author=S%20Nazario&author=J%20Casal&author=A%20Torres&volume=169&issue=3&publication_year=2004&pages=386-92&pmid=14617512&doi=10.1164/rccm.200309-1293OC&)

14. Naqvi M, Thyne S, Choudhry S, Tsai H-J, Navarro D, Castro RA, et al. Ethnic-Specific Differences in Bronchodilator Responsiveness Among African Americans, Puerto Ricans, and Mexicans with Asthma. J Asthma. 2007 Jan;44(8):639–48. doi: 10.1080/02770900701554441.  [DOI](https://doi.org/10.1080/02770900701554441) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17943575/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Asthma&title=Ethnic-Specific%20Differences%20in%20Bronchodilator%20Responsiveness%20Among%20African%20Americans,%20Puerto%20Ricans,%20and%20Mexicans%20with%20Asthma&author=M%20Naqvi&author=S%20Thyne&author=S%20Choudhry&author=H-J%20Tsai&author=D%20Navarro&volume=44&issue=8&publication_year=2007&pages=639-48&pmid=17943575&doi=10.1080/02770900701554441&)

15. Blake K, Madabushi R, Derendorf H, Lima J. Population Pharmacodynamic Model of Bronchodilator Response to Inhaled Albuterol in Children and Adults With Asthma. Chest. 2008 Nov 1;134(5):981–9. doi: 10.1378/chest.07-2991.  [DOI](https://doi.org/10.1378/chest.07-2991) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18583517/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chest&title=Population%20Pharmacodynamic%20Model%20of%20Bronchodilator%20Response%20to%20Inhaled%20Albuterol%20in%20Children%20and%20Adults%20With%20Asthma&author=K%20Blake&author=R%20Madabushi&author=H%20Derendorf&author=J%20Lima&volume=134&issue=5&publication_year=2008&pages=981-9&pmid=18583517&doi=10.1378/chest.07-2991&)

16. Howie BN, Donnelly P, Marchini J. A flexible and accurate genotype imputation method for the next generation of genome-wide association studies. PLoS Genetics. 2009 Jun;5(6):e1000529. doi: 10.1371/journal.pgen.1000529.  [DOI](https://doi.org/10.1371/journal.pgen.1000529) | [PMC free article](/articles/PMC2689936/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19543373/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genetics&title=A%20flexible%20and%20accurate%20genotype%20imputation%20method%20for%20the%20next%20generation%20of%20genome-wide%20association%20studies&author=BN%20Howie&author=P%20Donnelly&author=J%20Marchini&volume=5&issue=6&publication_year=2009&pages=e1000529&pmid=19543373&doi=10.1371/journal.pgen.1000529&)

17. Torgerson DG, Gignoux CR, Galanter JM, Drake KA, Roth LA, Eng C, et al. Case-control admixture mapping in Latino populations enriches for known asthma-associated genes. J Allergy Clin Immunol. 2012 Apr 12; doi: 10.1016/j.jaci.2012.02.040.  [DOI](https://doi.org/10.1016/j.jaci.2012.02.040) | [PMC free article](/articles/PMC3593143/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22502797/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Allergy%20Clin%20Immunol&title=Case-control%20admixture%20mapping%20in%20Latino%20populations%20enriches%20for%20known%20asthma-associated%20genes&author=DG%20Torgerson&author=CR%20Gignoux&author=JM%20Galanter&author=KA%20Drake&author=LA%20Roth&publication_year=2012&pmid=22502797&doi=10.1016/j.jaci.2012.02.040&)

18. Alexander DH, Novembre J, Lange K. Fast model-based estimation of ancestry in unrelated individuals. Genome Research. 2009 Sep;19(9):1655–64. doi: 10.1101/gr.094052.109.  [DOI](https://doi.org/10.1101/gr.094052.109) | [PMC free article](/articles/PMC2752134/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19648217/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Research&title=Fast%20model-based%20estimation%20of%20ancestry%20in%20unrelated%20individuals&author=DH%20Alexander&author=J%20Novembre&author=K%20Lange&volume=19&issue=9&publication_year=2009&pages=1655-64&pmid=19648217&doi=10.1101/gr.094052.109&)

19. Sankararaman S, Sridhar S, Kimmel G, Halperin E. Estimating local ancestry in admixed populations. Am J Hum Genet. 2008 Feb;82(2):290–303. doi: 10.1016/j.ajhg.2007.09.022.  [DOI](https://doi.org/10.1016/j.ajhg.2007.09.022) | [PMC free article](/articles/PMC2664993/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18252211/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Estimating%20local%20ancestry%20in%20admixed%20populations&author=S%20Sankararaman&author=S%20Sridhar&author=G%20Kimmel&author=E%20Halperin&volume=82&issue=2&publication_year=2008&pages=290-303&pmid=18252211&doi=10.1016/j.ajhg.2007.09.022&)

20. Baran Y, Pasaniuc B, Sankararaman S, Torgerson DG, Gignoux C, Eng C, et al. Fast and accurate inference of local ancestry in Latino populations. Bioinformatics. 2012 May 15;28(10):1359–67. doi: 10.1093/bioinformatics/bts144.  [DOI](https://doi.org/10.1093/bioinformatics/bts144) | [PMC free article](/articles/PMC3348558/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22495753/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=Fast%20and%20accurate%20inference%20of%20local%20ancestry%20in%20Latino%20populations&author=Y%20Baran&author=B%20Pasaniuc&author=S%20Sankararaman&author=DG%20Torgerson&author=C%20Gignoux&volume=28&issue=10&publication_year=2012&pages=1359-67&pmid=22495753&doi=10.1093/bioinformatics/bts144&)

21. Browning S. Multilocus Association Mapping Using Variable-Length Markov Chains. The American Journal of Human Genetics. 2006 Jun 1;78(6):903–13. doi: 10.1086/503876.  [DOI](https://doi.org/10.1086/503876) | [PMC free article](/articles/PMC1474089/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16685642/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=The%20American%20Journal%20of%20Human%20Genetics&title=Multilocus%20Association%20Mapping%20Using%20Variable-Length%20Markov%20Chains&author=S%20Browning&volume=78&issue=6&publication_year=2006&pages=903-13&pmid=16685642&doi=10.1086/503876&)

22. Delaneau O, Marchini J, Zagury J-F. A linear complexity phasing method for thousands of genomes. Nat Methods. 2012 Feb;9(2):179–81. doi: 10.1038/nmeth.1785.  [DOI](https://doi.org/10.1038/nmeth.1785) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22138821/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Methods&title=A%20linear%20complexity%20phasing%20method%20for%20thousands%20of%20genomes&author=O%20Delaneau&author=J%20Marchini&author=J-F%20Zagury&volume=9&issue=2&publication_year=2012&pages=179-81&pmid=22138821&doi=10.1038/nmeth.1785&)

23. Consortium T1GP, author C, committee S, Medicine PGBCO, BGI-Shenzhen, Broad Institute of MIT and Harvard, et al. . A map of human genome variation from population-scale sequencing. Nature. Nature Publishing Group. 2010 Oct 18;467(7319):1061–73. doi: 10.1038/nature09534.  [DOI](https://doi.org/10.1038/nature09534) | [PMC free article](/articles/PMC3042601/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20981092/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature.%20Nature%20Publishing%20Group&title=A%20map%20of%20human%20genome%20variation%20from%20population-scale%20sequencing&volume=467&issue=7319&publication_year=2010&pages=1061-73&pmid=20981092&doi=10.1038/nature09534&)

24. Team RDC. R: A language and environment for statistical computing [Internet] Vienna, Austria: R Foundation for Statistical Computing; 2010. Available from: http://www.R-project.org.  [http://www.R-project.org](http://www.R-project.org) | [Google Scholar](https://scholar.google.com/scholar_lookup?title=R:%20A%20language%20and%20environment%20for%20statistical%20computing%20%5BInternet%5D&publication_year=2010&)

25. Casella G, Berger RL. Statistical inference. Duxbury Pr; 2002. p. 660.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Statistical%20inference&author=G%20Casella&author=RL%20Berger&publication_year=2002&)

26. Livak KJ, Schmittgen TD. Analysis of relative gene expression data using real-time quantitative PCR and the 2(-Delta Delta C(T)) Method. Methods. 2001 Dec;25(4):402–8. doi: 10.1006/meth.2001.1262.  [DOI](https://doi.org/10.1006/meth.2001.1262) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11846609/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Methods&title=Analysis%20of%20relative%20gene%20expression%20data%20using%20real-time%20quantitative%20PCR%20and%20the%202(-Delta%20Delta%20C(T))%20Method&author=KJ%20Livak&author=TD%20Schmittgen&volume=25&issue=4&publication_year=2001&pages=402-8&pmid=11846609&doi=10.1006/meth.2001.1262&)

27. Cropp CD, Komori T, Shima JE, Urban TJ, Yee SW, More SS, et al. Organic anion transporter 2 (SLC22A7) is a facilitative transporter of cGMP. Mol Pharmacol. 2008 Apr;73(4):1151–8. doi: 10.1124/mol.107.043117.  [DOI](https://doi.org/10.1124/mol.107.043117) | [PMC free article](/articles/PMC2698938/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18216183/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Pharmacol&title=Organic%20anion%20transporter%202%20(SLC22A7)%20is%20a%20facilitative%20transporter%20of%20cGMP&author=CD%20Cropp&author=T%20Komori&author=JE%20Shima&author=TJ%20Urban&author=SW%20Yee&volume=73&issue=4&publication_year=2008&pages=1151-8&pmid=18216183&doi=10.1124/mol.107.043117&)

28. Eraly SA, Nigam SK. Novel human cDNAs homologous to Drosophila Orct and mammalian carnitine transporters. Biochem Biophys Res Commun. 2002 Oct 11;297(5):1159–66. doi: 10.1016/s0006-291x(02)02343-4.  [DOI](https://doi.org/10.1016/s0006-291x(02)02343-4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12372408/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20Biophys%20Res%20Commun&title=Novel%20human%20cDNAs%20homologous%20to%20Drosophila%20Orct%20and%20mammalian%20carnitine%20transporters&author=SA%20Eraly&author=SK%20Nigam&volume=297&issue=5&publication_year=2002&pages=1159-66&pmid=12372408&doi=10.1016/s0006-291x(02)02343-4&)

29. Hoffmann TJ, Zhan Y, Kvale MN, Hesselson SE, Gollub J, Iribarren C, et al. Design and coverage of high throughput genotyping arrays optimized for individuals of East Asian, African American, and Latino race/ethnicity using imputation and a novel hybrid SNP selection algorithm. Genomics. 2011 Dec;98(6):422–30. doi: 10.1016/j.ygeno.2011.08.007.  [DOI](https://doi.org/10.1016/j.ygeno.2011.08.007) | [PMC free article](/articles/PMC3502750/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21903159/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genomics&title=Design%20and%20coverage%20of%20high%20throughput%20genotyping%20arrays%20optimized%20for%20individuals%20of%20East%20Asian,%20African%20American,%20and%20Latino%20race/ethnicity%20using%20imputation%20and%20a%20novel%20hybrid%20SNP%20selection%20algorithm&author=TJ%20Hoffmann&author=Y%20Zhan&author=MN%20Kvale&author=SE%20Hesselson&author=J%20Gollub&volume=98&issue=6&publication_year=2011&pages=422-30&pmid=21903159&doi=10.1016/j.ygeno.2011.08.007&)

30. Gravel S, Henn BM, Gutenkunst RN, Indap AR, Marth GT, Clark AG, et al. Demographic history and rare allele sharing among human populations. PNAS. 2011 Jul 19;108(29):11983–8. doi: 10.1073/pnas.1019276108.  [DOI](https://doi.org/10.1073/pnas.1019276108) | [PMC free article](/articles/PMC3142009/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21730125/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PNAS&title=Demographic%20history%20and%20rare%20allele%20sharing%20among%20human%20populations&author=S%20Gravel&author=BM%20Henn&author=RN%20Gutenkunst&author=AR%20Indap&author=GT%20Marth&volume=108&issue=29&publication_year=2011&pages=11983-8&pmid=21730125&doi=10.1073/pnas.1019276108&)

31. Dennis PA, Rifkin DB. Cellular activation of latent transforming growth factor beta requires binding to the cation-independent mannose 6-phosphate/insulin-like growth factor type II receptor. PNAS. 1991 Jan 15;88(2):580–4. doi: 10.1073/pnas.88.2.580.  [DOI](https://doi.org/10.1073/pnas.88.2.580) | [PMC free article](/articles/PMC50855/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1846448/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PNAS&title=Cellular%20activation%20of%20latent%20transforming%20growth%20factor%20beta%20requires%20binding%20to%20the%20cation-independent%20mannose%206-phosphate/insulin-like%20growth%20factor%20type%20II%20receptor&author=PA%20Dennis&author=DB%20Rifkin&volume=88&issue=2&publication_year=1991&pages=580-4&pmid=1846448&doi=10.1073/pnas.88.2.580&)

32. Yan X, Baxter RC, Firth SM. Involvement of pregnancy-associated plasma protein-A2 in insulin-like growth factor (IGF) binding protein-5 proteolysis during pregnancy: a potential mechanism for increasing IGF bioavailability. J Clin Endocrinol Metab. 2010 Mar;95(3):1412–20. doi: 10.1210/jc.2009-2277.  [DOI](https://doi.org/10.1210/jc.2009-2277) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20103653/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Endocrinol%20Metab&title=Involvement%20of%20pregnancy-associated%20plasma%20protein-A2%20in%20insulin-like%20growth%20factor%20(IGF)%20binding%20protein-5%20proteolysis%20during%20pregnancy:%20a%20potential%20mechanism%20for%20increasing%20IGF%20bioavailability&author=X%20Yan&author=RC%20Baxter&author=SM%20Firth&volume=95&issue=3&publication_year=2010&pages=1412-20&pmid=20103653&doi=10.1210/jc.2009-2277&)

33. Yamashita N, Tashimo H, Ishida H, Matsuo Y, Arai H, Nagase H, et al. Role of insulin-like growth factor-I in allergen-induced airway inflammation and remodeling. Cell Immunol. 2005 Jun;235(2):85–91. doi: 10.1016/j.cellimm.2005.07.006.  [DOI](https://doi.org/10.1016/j.cellimm.2005.07.006) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16168397/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Immunol&title=Role%20of%20insulin-like%20growth%20factor-I%20in%20allergen-induced%20airway%20inflammation%20and%20remodeling&author=N%20Yamashita&author=H%20Tashimo&author=H%20Ishida&author=Y%20Matsuo&author=H%20Arai&volume=235&issue=2&publication_year=2005&pages=85-91&pmid=16168397&doi=10.1016/j.cellimm.2005.07.006&)

34. Giacomini K, Sugiyama Y. In: Goodman and Gilman’s The Pharmacological Basis of Therapeutics, Twelfth Edition. Brunton L, Lazo JS, Parker RL, editors. New York: McGraw-Hill; 2006. pp. 41–70.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Goodman%20and%20Gilman%E2%80%99s%20The%20Pharmacological%20Basis%20of%20Therapeutics,%20Twelfth%20Edition&author=K%20Giacomini&author=Y%20Sugiyama&publication_year=2006&)

35. ENCODE Project Consortium. A user’s guide to the encyclopedia of DNA elements (ENCODE) PLoS Biol. 2011 Apr;9(4):e1001046. doi: 10.1371/journal.pbio.1001046.  [DOI](https://doi.org/10.1371/journal.pbio.1001046) | [PMC free article](/articles/PMC3079585/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21526222/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Biol&title=A%20user%E2%80%99s%20guide%20to%20the%20encyclopedia%20of%20DNA%20elements%20(ENCODE)&volume=9&issue=4&publication_year=2011&pages=e1001046&pmid=21526222&doi=10.1371/journal.pbio.1001046&)

36. Schlessinger A, Matsson P, Shima JE, Pieper U, Yee SW, Kelly L, et al. Comparison of human solute carriers. Protein Sci. 2010 Mar;19(3):412–28. doi: 10.1002/pro.320.  [DOI](https://doi.org/10.1002/pro.320) | [PMC free article](/articles/PMC2866268/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20052679/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Protein%20Sci&title=Comparison%20of%20human%20solute%20carriers&author=A%20Schlessinger&author=P%20Matsson&author=JE%20Shima&author=U%20Pieper&author=SW%20Yee&volume=19&issue=3&publication_year=2010&pages=412-28&pmid=20052679&doi=10.1002/pro.320&)
