# Hypertension Susceptibility Loci and Blood Pressure Response to Antihypertensives – Results from the Pharmacogenomic Evaluation of Antihypertensive Responses (PEAR) Study

## Metadata
**Authors:** Yan Gong, Caitrin W McDonough, Zhiying Wang, Wei Hou, Rhonda M Cooper-DeHoff, Taimour Y Langaee, Amber L Beitelshees, Arlene B Chapman, John G Gums, Kent R Bailey, Eric Boerwinkle, Stephen T Turner, Julie A Johnson
**Journal:** Circulation. Cardiovascular genetics
**Date:** 2012 Oct 19
**DOI:** [10.1161/CIRCGENETICS.112.964080](https://doi.org/10.1161/CIRCGENETICS.112.964080)
**PMID:** 23087401
**PMCID:** PMC3529147
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3529147/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC3529147/pdf/nihms-423673.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC3529147/pdf/nihms-423673.pdf)

## Abstract

**Background:** 
To date, 39 SNPs have been associated with blood pressure (BP) or hypertension (HTN) in genome-wide association studies (GWAS) in Caucasians. Our hypothesis is that the loci/SNPs associated with BP/HTN are also associated with BP response to antihypertensive drugs.

**Methods and Results:** 
We assessed the association of these loci with BP response to atenolol or hydrochlorothiazide monotherapy in 768 hypertensive participants in the Pharmacogenomics Responses of Antihypertensive Responses (PEAR) study. Linear regression analysis was performed in Caucasians for each SNP in an additive model adjusting for baseline BP, age, gender and principal components for ancestry. Genetic scores were constructed to include SNPs with nominal associations and empirical p values were determined by permutation test. Genotypes of 37 loci were obtained from Illumina 50K cardiovascular or Omni1M GWAS chips. In Caucasians, no SNPs reached Bonferroni-corrected alpha of 0.0014, six reached nominal significance (p<0.05) and 3 were associated with atenolol BP response at p < 0.01. The genetic score of the atenolol BP lowering alleles was associated with response to atenolol (p =3.3*10−6 for SBP; p=1.6*10−6 for DBP). The genetic score of the HCTZ BP lowering alleles was associated with response to HCTZ (p = 0.0006 for SBP; p = 0.0003 for DBP). Both risk score p values were < 0.01 based on the empirical distribution from the permutation test.

**Conclusions:** 
These findings suggest selected signals from hypertension GWAS may predict BP response to atenolol and HCTZ when assessed through risk scoring.

Keywords: beta-blocker, diuretics, hypertension, pharmacogenetics, polymorphisms blood pressure

### Background

To date, 39 SNPs have been associated with blood pressure (BP) or hypertension (HTN) in genome-wide association studies (GWAS) in Caucasians. Our hypothesis is that the loci/SNPs associated with BP/HTN are also associated with BP response to antihypertensive drugs.

### Methods and Results

We assessed the association of these loci with BP response to atenolol or hydrochlorothiazide monotherapy in 768 hypertensive participants in the Pharmacogenomics Responses of Antihypertensive Responses (PEAR) study. Linear regression analysis was performed in Caucasians for each SNP in an additive model adjusting for baseline BP, age, gender and principal components for ancestry. Genetic scores were constructed to include SNPs with nominal associations and empirical p values were determined by permutation test. Genotypes of 37 loci were obtained from Illumina 50K cardiovascular or Omni1M GWAS chips. In Caucasians, no SNPs reached Bonferroni-corrected alpha of 0.0014, six reached nominal significance (p<0.05) and 3 were associated with atenolol BP response at p < 0.01. The genetic score of the atenolol BP lowering alleles was associated with response to atenolol (p =3.3*10^−6^ for SBP; p=1.6*10^−6^ for DBP). The genetic score of the HCTZ BP lowering alleles was associated with response to HCTZ (p = 0.0006 for SBP; p = 0.0003 for DBP). Both risk score p values were < 0.01 based on the empirical distribution from the permutation test.

### Conclusions

These findings suggest selected signals from hypertension GWAS may predict BP response to atenolol and HCTZ when assessed through risk scoring.

## Introduction

Hypertension (HTN) places over 76 million Americans (and approximately one billion individuals worldwide) at substantially increased risk for stroke, coronary heart disease (including heart attack), renal failure, and heart failure, and is the most common chronic disease for which medications are prescribed.^1^ Five drug classes are considered appropriate first line therapy for HTN and are often defined by their effect on the renin angiotensin system (RAS; β-blockers, ACE inhibitors and angiotensin receptor blockers) or sodium and calcium (thiazide diuretics and calcium channel blockers). ^2^ Response rates to monotherapy with any given antihypertensive drug are only about 50%,^3^ and those who fail to respond to one category (e.g. those affecting RAS) class are more likely to respond to a drug with an alternate mechanism (~75% response rate).^4, 5^ Selection of the initial antihypertensive therapy is essentially by trial- and-error. The difficulty in determining the most appropriate antihypertensive drug for a specific patient likely contributes to the fact that less than half of the hypertensive patients in the U.S. (and worldwide) currently have their blood pressure (BP) controlled.^6^ Pharmacogenomics offers the clinical promise of individualization of therapy based on a person’s genetic make-up.

To date, 39 single nucleotide polymorphisms (SNPs) have been associated with blood pressure (BP) or hypertension in genome-wide association studies (GWAS) and meta-analyses in Caucasians.^7-11^ Our hypothesis is that the loci/SNPs associated with BP/HTN are also associated with BP response to antihypertensive drugs. To test this hypothesis, we assessed in the Pharmacogenomic Evaluation of Antihypertensive Responses (PEAR) study the association of these loci with BP response to two first-line antihypertensives with contrasting pharmacological mechanisms: atenolol (a cardioselective β-blocker) and hydrochlorothiazide (HCTZ, a thiazide diuretic).

## Methods

### Participants

The Pharmacogenomic Evaluation of Antihypertensive Responses (PEAR) study ([clincaltrials.gov](http://clincaltrials.gov) identifier [NCT00246519](https://clinicaltrials.gov/ct2/show/NCT00246519)) (n=768) was a randomized multi-center clinical trial examining the role of genetic variability on BP response to HCTZ and/or atenolol. ^12^ Males and females of any race between the ages of 17 and 65 with essential hypertension (clinic diastolic BP ≥ 90 mmHg, ≤ 110 mmHg) were recruited to participate at University of Florida (Gainesville, FL), Mayo Clinic (Rochester, MN), and Emory University (Atlanta, GA). The institutional review boards at each institution approved the protocol and each participant provided informed written consent prior to entry into the study. Those currently treated had their antihypertensive drug therapy tapered (as necessary) and discontinued. The mean washout period was 28 days. This washout period was deemed to sufficiently balance risks of non-treatment with a sufficient period to allow return to untreated BP because post-washout BP in previously treated participants were nearly identical to baseline BP of those who entered the trial untreated.

Included participants were randomized to receive either HCTZ 12.5 mg daily or atenolol 50 mg daily for three weeks, followed by dose titration to 25 mg and 100 mg daily, respectively for SBP > 120 and DBP > 70. After nine weeks, BP response to monotherapy was assessed. The second study drug (alternate drug) was added in those patients with SBP > 120 mmHg or DBP > 70 mmHg (> 90% for both randomization arms), with dose titration after 3 weeks and response assessment after six more weeks, as in the first portion of the study. Participants were not given sodium restrictions, but were counseled to maintain consistent dietary intakes.

### Blood pressure Phenotype

The primary BP phenotype for PEAR was home BP measured with an automated device (MicroLife 3AC1-PC, Minneapolis MN) provided to each participant. The device was set to measure BP in triplicate with each activation and to store the average systolic and diastolic BPs and the time of each set of measurements. Participants were required to take readings daily in the seated position in triplicate upon rising and before retiring at least five of seven days prior to the three visits: baseline (prior to randomization), assessment visit after monotherapy and assessment visit after add-on therapy. All BP readings since the previous study visit were downloaded to the computer by study coordinators. Home BP response to monotherapy was the primary response phenotype because it exhibits less variability and is superior to office BP in prediction of prognosis.^13-15^

### Genotyping and QC steps

Thirty-nine SNPs/loci were identified from the BP GWAS studies and meta-analyses that have been published up to October 2011^7-11^ as signals that reached genome-wide significance of p<5*10^−8^. In PEAR participants, genotypes of 37 SNPs (exact SNPs for 26 loci and proxies for 11 loci) were obtained from Illumina Human 50K cardiovascular Beadchip or Omni1M quad GWAS Beadchip. Two SNPs (rs4590817 and rs319690) were not covered on either platform ([Table S1](#SD1)).

#### Illumina Human CVD 50K array

The HumanCVD50k SNP array ^16^ was a customized gene-centric array including ~2100 genes and ~50,000 SNPs genotyped using the Infinium II Assay (Illumina, San Diego, CA). Genotypes were called using GenomeStudio software version 2011.1 and the Genotyping Module version 1.9 calling algorithm (Illumina, San Diego, CA).

Participants were excluded if sample genotype call rates were below 95% and SNPs were excluded if genotype call rates were below 90%. The genotype data were not reclustered after QC filters, but the genotype and sample call rate was recalculated after QC. Sample contamination was detected by checking gender mismatches using X chromosome genotype data and cryptic relatedness was estimated by pairwise identity-by-descent (IBD) analysis implemented using PLINK ([http://pngu.mgh.harvard.edu/purcell/plink/](http://pngu.mgh.harvard.edu/purcell/plink/)). After the QC procedures, the total SNP call rate in the remaining individuals was 99.799%. Hardy-Weinberg equilibrium was assessed with chi-square test if one degree of freedom.

#### Illumina Omni1M quad GWAS chip

Human Omni1M quad GWAS chip also used Infinium II assay and genotypes were called using BeadStudio software and GenTrain2 calling algorithm. The same QC procedures were carried out for the GWAS genotype data. After the QC procedures, the total SNP call rate in the remaining individuals was 99.519%. Principal component analysis was performed to assess the ancestral background. Participant’s self-identified race information was confirmed with principal component analysis for genetic ancestry and 464 participants were confirmed as European descendants and 304 as African descendants. Seven hundred and fifty nine participants were included in this analysis, including 461 Caucasians and 298 African Americans. Nine participants were excluded due to genotype QC issues or missing genotypes.

### Statistical Methods

The primary analysis was performed in Caucasians since the GWAS signals selected for this analysis arose from studies in Caucasians. Associations of these SNPs with SBP and DBP responses to atenolol or HCTZ monotherapy were evaluated using linear regression, adjusting for baseline BP, age, gender and the first 2 principal components for ancestry which correspond to European and African ancestry respectively. Previous analysis ^17^ of BP response in PEAR established that age, gender and baseline BP were significant demographic predictors of BP response, while common covariates such as BMI and smoking were not associated with BP response and therefore not included as covariates. P values of <0.0014 (0.05/37) were considered statistically significant. Additive mode of inheritance was assumed where the SNPs were coded as 0, 1 and 2 in the linear regression model. For the SNPs with p<0.05 in Caucasians, the association analysis of the index SNPs and the surrounding 100 kb region was also performed in African American participants to assess if an association with the same direction was present in participants with a different ancestral background. Particular attention was also given to SNPs with association in the opposite direction for HCTZ and atenolol, since the counter balancing mechanisms targeted by these agents lead to good responders to one of the drugs being poor responders to the other.^18^ For the SNPs that reached nominal significance, genetic scores were constructed by summing the alleles with the best BP response for inclusion in regression models (using the BP lowering allele as the coded allele). A weighted score method was also assessed based on the coefficient of BP response for each SNP. The participants with any missing genotypes would not get a score, therefore were not included in the score analysis. Since the BP response alleles are different for atenolol and HCTZ, separate genetic scores were constructed for atenolol and HCTZ monotherapy. To validate the association of the scores with the BP responses, we generated the distribution of the regression test statistics empirically by repetitively permutating the BP responses. The relationship between the BP responses and the scores was randomly shuffled 1000 times, and a regression model was fit for each permutation. The raw p values of the scores predicting BP response of the raw data was then compared to the empirical distribution of 1000 p values from the permutations. Analysis were performed using PLINK ^19^ assuming an additive mode of inheritance. The genetic score analysis and permutation test were performed in SAS version 9.3 (Cary, NC).

## Results

The PEAR participants were Caucasian (60%) and African American (40%) hypertensive individuals, 53% were females. Participants had a mean age of 48.8 ± 9.2 years, and most were overweight or obese. The baseline characteristics of the 461 Caucasian and 298 African American participants assigned to atenolol and hydrochlorothiazide treatment are presented in [Table 1](#T1).

### Table 1.

| Characteristics* | Caucasians (n=461) |  | African Americans (n=298) |  |
| --- | --- | --- | --- | --- |
| Atenolol (n = 233) | HCTZ (n = 228) | Atenolol (n = 150) | HCTZ (n =148) |  |
| Age (mean ± SD) | 49.5 ± 9.5 | 50.0 ± 9.5 | 47.2 ± 8.5 | 47.4 ± 8.8 |
| Female gender (n, %) | 109 (46.8%) | 91 (40.0%) | 107 (71.3%) | 92 (62.2%) |
| BMI (kg/m2) | 30.3 ± 5.5 | 30.3 ± 4.9 | 31.6 ± 6.3 | 31.5 ± 5.4 |
| Duration of hypertension (years) | 7.0 ± 7.2 | 6.0 ± 7.1 | 6.4 ± 6.8 | 7.0 ± 7.7 |
| Taking at least 1 antihypertensive drug at entry | 180 (77.3%) | 164 (71.9%) | 105 (70.0%) | 110 (74.3%) |
| Smoking status |  |  |  |  |
| Current smoker | 26 (11.2%) | 31 (13.6%) | 23 (15.3%) | 29 (19.6%) |
| Ex-smoker | 65 (27.9%) | 65 (28.5%) | 29 (19.3%) | 20 (13.5%) |
| Ever smoker | 91 (39.1%) | 96 (42.1%) | 52 (34.1%) | 49 (33.1%) |
| Baseline mean home-recorded BP |  |  |  |  |
| Systolic (mmHg) | 145.0 ± 9.4 | 146.0 ± 10.0 | 145.1 ± 10.5 | 147.1 ± 11.5 |
| Diastolic (mmHg) | 92.8 ± 5.5 | 93.6 ± 5.5 | 93.9 ± 6.5 | 95.1 ± 6.6 |

Table 1 Caption: Baseline characteristics of PEAR participants.

After an average of 9 weeks of atenolol monotherapy, the mean blood pressure response was −11.0 ± 9.4/−9.9 ± 6.4 mmHg in Caucasians and −3.1± 10.6/−4.2 ± 6.7 mmHg in African American participants. Mean blood pressure response after HCTZ monotherapy was −7.7 ± 8.1/−4.3 ± 5.3 mmHg in Caucasians and −12.0 ± 9.7/7.1 ± 6.5 mmHg in African Americans. The distributions of BP response in [Figure S1](#SD1) demonstrate the large inter-individual variability in BP response to atenolol and HCTZ in both race groups.

Genotypes for 37 out of the 39 BP/HTN SNPs were available for PEAR participants from the two genotyping platforms. No SNPs deviated from Hardy-Weinberg equilibrium in Caucasians or African Americans.

When the association of these 37 SNPs with SBP and DBP response to atenolol or HCTZ monotherapy was assessed, none of the SNPs reached *a priori* alpha of 0.0014, although six SNPs achieved nominal significance (p<0.05). The associations of these SNPs with monotherapy BP response in Caucasians are summarized in [Table 2](#T2). The most interesting was an intergenic SNP rs1458038 near *FGF5,* which was associated with BP response to atenolol and HCTZ monotherapy, with genotype effects in the opposite direction ([Figure 1](#F1)). Participants with TT and TC genotypes of this common SNP responded better to atenolol than those with CC genotype (with SBP/DBP response of −12.9/−11.6 mmHg, −12.3/−10.6mmHg and −8.9/−8.5 mmHg for TT, TC and CC respectively, p =0.0067 for SBP, p = 0.018 for DBP), while the direction of association was the opposite for response to HCTZ, with −5.4/−2.6mmHg, −7.5/−4.3mmHg, −8.7/−4.8mmHg for TT, TC and CC respectively (p = 0.039 for SBP, p = 0.069 for DBP).

### Table 2.

| SNP | Nearest Gene | CHR | Position* | Alleles¶ | N | MAF | BP lowering allele§ | Atenolol |  |  |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| DBP |  |  | SBP |  |  |  |  |  |  |  |  |  |  |
| P | Beta | SE | P | Beta | SE |  |  |  |  |  |  |  |  |
| rs1458038 | FGF5 | 4 | 81,383,747 | T/C | 233 | 0.37 | T | 0.02 | −1.42 | 0.60 | 0.0067 | −2.48 | 0.91 |
| rs871606 | CHIC2 | 4 | 54,494,002 | C/T | 233 | 0.10 | T | 0.0037 | −2.57 | 0.87 | 0.017 | −3.23 | 1.34 |
| rs2932538 | MOV10 | 1 | 113,018,066 | C/T | 233 | 0.24 | T | 0.005 | −1.84 | 0.66 | 0.06 | −1.89 | 1.01 |
| rs1799945 | HFE | 6 | 26,199,158 | G/C | 233 | 0.12 | G | 0.15 | −1.36 | 0.95 | 0.02 | −3.38 | 1.44 |

Table 2 Caption: SNPs associated with BP response to atenolol or HCTZ monotherapy in Caucasians.

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0706/3529147/6b6853a96d96/nihms-423673-f0001.jpg)

Hypertension SNP rs1458038 near FGF5 associated with BP response to atenolol or hydrochlorothiazide monotherapy among Caucasian hypertensive patients.

The genetic score comprised of the 4 BP lowering alleles in response to atenolol monotherapy showed stronger association, with p = 3.3*10^−6^ for SBP response and p=1.6*10^−6^ for DBP response. The raw p values are smaller than the minimum p values from the 1000 permutations ([Table S2](#SD1)) and thus the significance was validated through the permutation tests. The BP responses to atenolol monotherapy by genetic score are shown in [Figure 2](#F2) (A for SBP response, B for DBP response). In Caucasian hypertensive patients with only one of the 4 BP lowering alleles, the average BP response to atenolol monotherapy was −4.0/−3.3 mmHg, while the average BP response was −18.6/−23.5 mmHg in those with six BP lowering alleles. This genetic score explained 8.5% of SBP response and 8.2% of DBP response to atenolol monotherapy, respectively.

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0706/3529147/d4d5543021b7/nihms-423673-f0002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3529147_nihms-423673-f0002.jpg)

Blood pressure response by genetic score of blood pressure lowering alleles in response to ATEN monotherapy. Blood pressure responses (mmHg) are presented in box plots where the whiskers indicating the minimum and maximum values. A. SBP; B. DBP

The association of the genetic score comprised of the 3 BP lowering alleles for HCTZ monotherapy was also stronger than those for the individual SNPs, with p of 0.0006 for SBP response and p of 0.0003 for DBP response to HCTZ monotherapy. The raw p values are < 0.01 based on the empirical distribution of the permutation test. The BP responses to HCTZ monotherapy by genetic score are shown in [Figure 3](#F3) (A for SBP responses and B for DBP responses). This genetic score explained 4.3% for SBP response and 5.3% for DBP response to HCTZ monotherapy. The weighted scores yielded almost identical results and due to the simplicity of the unweighted scores, this was considered the preferred model.

### Figure 3.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0706/3529147/334aa5bec664/nihms-423673-f0004.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3529147_nihms-423673-f0004.jpg)

Blood pressure response by genetic score of blood pressure lowering alleles in response to HCTZ monotherapy. Blood pressure responses (mmHg) are presented in box plots where the whiskers indicating the minimum and maximum values. A. SBP; B. DBP

Associations of these six SNPs and the SNPs in the same region with the BP response in African Americans were also evaluated. None of the Caucasian index SNPs achieved nominal significance in African Americans with a directional effect the same as in Caucasians. rs871606 near *CHIC2* was marginally associated with DBP response to atenolol monotherapy in African Americans (DBP, p = 0.06, beta=1.89), with the same direction of effect as in Caucasians. One SNP had a directionally opposite effect association in African Americans as in Caucasians: the C allele of rs3184504 in *SH2B3* (nonsynonymous, T>C, Trp > Arg) was associated with better BP reduction in Caucasians in response to HCTZ (show in [Table 2](#T2)). However in African Americans treated with HCTZ, the C allele of this SNP is associated with a slight increase in blood pressure (DBP, p = 0.047, beta=0.21).

When the 100kb regions flanking the index SNPs were evaluated in African Americans, two different SNPs (rs12069113, rs12076902) near *MOV10* were found to be nominally associated with BP response to atenolol (DBP, p=0.042 and 0.02, respectively. [Figure S2](#SD1)). These two SNP were in high LD with each other (r^2^ = 0.78 and 1 in Caucasians and African Americans, respectively) but in very low LD with the Caucasian index SNP, rs2932538, in both populations (r^2^ = 0.01 and 0.02 in Caucasians and African Americans, respectively). No stronger signals were found in the African Americans in the regions flanking the Caucasian index SNPs. There was also no evidence that the genetic scores for either atenolol or HCTZ BP lowering alleles in Caucasians are associated with BP response in African Americans, p = 0.49 for SBP and p = 0.49 for DBP in response to HCTZ and p = 0.59 for SBP and p = 0.42 for DBP in response to atenolol monotherapy, respectively.

## Discussion

Among 37 SNPs previously documented to influence BP, none achieved *a priori* alpha values for association with BP response to atenolol or HCTZ. However six SNPs were nominally associated with BP response in Caucasians. Two of these SNPs were also marginally associated with response in African Americans for the same drug, one with the same direction of effect and the other with the opposite direction of effect. Lack of association of these index SNPs in African Americans is not surprising due to different LD structures between these two populations.

Of interest is that gene score analyses for both atenolol and HCTZ BP lowering alleles revealed that the scores were more strongly associated with the BP response than the individual SNPs. This is not surprising for a phenotype with multiple genetic contributors, as is expected for the antihypertensive response. The percentage of variance of BP response explained by these genetic scores ranged from 4.3 ~ 8.5%, which are larger effect sizes than the genetic risk scores constructed to predict the untreated BP in the HTN GWAS studies (around 116 independent loci collectively explain ~ 2.2% of the phenotypic variance for SBP and DBP).^7^ This is consistent with numerous studies indicating that pharmacogenetic effect sizes are larger than disease genetic effect sizes. These data also provide insight into how genetic risk scores might be built to guide selection of an antihypertensive drug based on genetic information. In order to make sure the associations of these SNPs were not due to multicollinearity with the baseline BP, we also tested the association without adjusting for baseline BP and found that the results were essentially identical. In addition, we also tested the association of these SNPs with baseline BP and not surprisingly, given our small sample size, and limited BP range (DBP 90-110 mmHg) there was not a significant association with baseline BP. This indicates that the associations presented are not likely to be due to collinearity with the baseline BP.

The finding of SNP rs1458038 near *FGF5* is perhaps most intriguing single SNP finding since it was associated with BP responses to both ATEN and HCTZ, but with the effects in opposite directions. This represents a form of replication since the HCTZ-treated participants were an independent group, and the opposite direction of effect is consistent with our expectations on genetic associations for the two drugs. The effect sizes of BP response for this SNP ranged from 1.0 to 2.5 mmHg per allele (double that for variant homozygotes). Differences of 2 to 3 mmHg SBP reduction have been shown to reduce stroke risk by 20 ~ 30% in a meta-analysis of nine randomized controlled trials of antihypertensive drugs,^20^ suggesting the associations noted here would be considered clinically significant.

In a HTN GWAS study,^7^ the T allele of rs1458038 near *FGF5* was associated with higher BP and higher risk for HTN in Caucasian individuals. This SNP was also strongly associated with BP in East Asians and South Asians and only marginally in African Americans. Our study indicated that Caucasian hypertensive individuals with the risk allele for HTN (T) might respond better to atenolol, than to HCTZ. *FGF5* encodes fibroblast growth factor 5, which is involved in a variety of biological processes, including embryonic development, cell growth, morphogenesis, tissue repair, tumor growth and invasion. It is not immediately clear how this gene is involved in BP regulation, but the association with atenolol responsiveness suggests this gene might be a marker for high sympathetic nervous system and renin-angiotensin activity and unraveling the function of this gene may provide additional insights.

## Conclusions

We tested 37 SNPs/loci previously associated with BP in GWAS for association with antihypertensive response. Our results showed that the genetic loci associated with BP and/or HTN are not strongly associated with the BP response to commonly prescribed antihypertensives individually, with the most compelling data for a single SNP being one near *FGF5*. However, collectively a selection of these loci may contribute moderately to the BP response to the antihypertensives. This highlights the potential for building response scores for clinical prediction of response.

## Supplementary Material

## Acknowledgments

We acknowledge and thank the valuable contributions of the PEAR study participants, support staff, and study physicians: Drs. R. Whit Curry, Karen Hall, Frederic Rabari-Oskoui, Dan Rubin, and Siegfried Schmidt.

**Funding Sources:** PEAR was supported by the National Institute of Health Pharmacogenetics Research Network grant U01-GM074492 and the National Center for Advancing Translational Sciences under the award number UL1 TR000064 (University of Florida); UL1 TR000454 (Emory University) and UL1 TR000135 (Mayo Clinic) and funds from the Mayo Foundation. This research was also supported by K23 HL091120 (A.L.B.) and K23 HL086558 (R.M.C-D).

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Roger VL, Go AS, Lloyd-Jones DM, Adams RJ, Berry JD, Brown TM, et al. Heart disease and stroke statistics--2011 update: A report from the american heart association. Circulation. 2011;123:e18–e209. doi: 10.1161/CIR.0b013e3182009701.  [DOI](https://doi.org/10.1161/CIR.0b013e3182009701) | [PMC free article](/articles/PMC4418670/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21160056/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation&title=Heart%20disease%20and%20stroke%20statistics--2011%20update:%20A%20report%20from%20the%20american%20heart%20association&author=VL%20Roger&author=AS%20Go&author=DM%20Lloyd-Jones&author=RJ%20Adams&author=JD%20Berry&volume=123&publication_year=2011&pages=e18-e209&pmid=21160056&doi=10.1161/CIR.0b013e3182009701&)

2. Chobanian AV, Bakris GL, Black HR, Cushman WC, Green LA, Izzo JL, Jr., et al. The seventh report of the joint national committee on prevention, detection, evaluation, and treatment of high blood pressure: The jnc 7 report. JAMA. 2003;289:2560–2572. doi: 10.1001/jama.289.19.2560.  [DOI](https://doi.org/10.1001/jama.289.19.2560) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12748199/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA&title=The%20seventh%20report%20of%20the%20joint%20national%20committee%20on%20prevention,%20detection,%20evaluation,%20and%20treatment%20of%20high%20blood%20pressure:%20The%20jnc%207%20report&author=AV%20Chobanian&author=GL%20Bakris&author=HR%20Black&author=WC%20Cushman&author=LA%20Green&volume=289&publication_year=2003&pages=2560-2572&pmid=12748199&doi=10.1001/jama.289.19.2560&)

3. Materson BJ, Reda DJ, Cushman WC, Massie BM, Freis ED, Kochar MS, et al. Single-drug therapy for hypertension in men. A comparison of six antihypertensive agents with placebo. The department of veterans affairs cooperative study group on antihypertensive agents. N Engl J Med. 1993;328:914–921. doi: 10.1056/NEJM199304013281303.  [DOI](https://doi.org/10.1056/NEJM199304013281303) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8446138/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Single-drug%20therapy%20for%20hypertension%20in%20men.%20A%20comparison%20of%20six%20antihypertensive%20agents%20with%20placebo.%20The%20department%20of%20veterans%20affairs%20cooperative%20study%20group%20on%20antihypertensive%20agents&author=BJ%20Materson&author=DJ%20Reda&author=WC%20Cushman&author=BM%20Massie&author=ED%20Freis&volume=328&publication_year=1993&pages=914-921&pmid=8446138&doi=10.1056/NEJM199304013281303&)

4. Materson BJ, Reda DJ, Preston RA, Cushman WC, Massie BM, Freis ED, et al. Response to a second single antihypertensive agent used as monotherapy for hypertension after failure of the initial drug. Department of veterans affairs cooperative study group on antihypertensive agents. Arch Intern Med. 1995;155:1757–1762.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/7654109/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arch%20Intern%20Med&title=Response%20to%20a%20second%20single%20antihypertensive%20agent%20used%20as%20monotherapy%20for%20hypertension%20after%20failure%20of%20the%20initial%20drug.%20Department%20of%20veterans%20affairs%20cooperative%20study%20group%20on%20antihypertensive%20agents&author=BJ%20Materson&author=DJ%20Reda&author=RA%20Preston&author=WC%20Cushman&author=BM%20Massie&volume=155&publication_year=1995&pages=1757-1762&pmid=7654109&)

5. Dickerson JE, Hingorani AD, Ashby MJ, Palmer CR, Brown MJ. Optimisation of antihypertensive treatment by crossover rotation of four major classes. Lancet. 1999;353:2008–2013. doi: 10.1016/s0140-6736(98)07614-4.  [DOI](https://doi.org/10.1016/s0140-6736(98)07614-4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10376615/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Optimisation%20of%20antihypertensive%20treatment%20by%20crossover%20rotation%20of%20four%20major%20classes&author=JE%20Dickerson&author=AD%20Hingorani&author=MJ%20Ashby&author=CR%20Palmer&author=MJ%20Brown&volume=353&publication_year=1999&pages=2008-2013&pmid=10376615&doi=10.1016/s0140-6736(98)07614-4&)

6. Roger VL, Go AS, Lloyd-Jones DM, Benjamin EJ, Berry JD, Borden WB, et al. Heart disease and stroke statistics--2012 update: A report from the american heart association. Circulation. 2012;125:e2–e220. doi: 10.1161/CIR.0b013e31823ac046.  [DOI](https://doi.org/10.1161/CIR.0b013e31823ac046) | [PMC free article](/articles/PMC4440543/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22179539/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation&title=Heart%20disease%20and%20stroke%20statistics--2012%20update:%20A%20report%20from%20the%20american%20heart%20association&author=VL%20Roger&author=AS%20Go&author=DM%20Lloyd-Jones&author=EJ%20Benjamin&author=JD%20Berry&volume=125&publication_year=2012&pages=e2-e220&pmid=22179539&doi=10.1161/CIR.0b013e31823ac046&)

7. Ehret GB, Munroe PB, Rice KM, Bochud M, Johnson AD, Chasman DI, et al. Genetic variants in novel pathways influence blood pressure and cardiovascular disease risk. Nature. 2011;478:103–109. doi: 10.1038/nature10405.  [DOI](https://doi.org/10.1038/nature10405) | [PMC free article](/articles/PMC3340926/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21909115/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Genetic%20variants%20in%20novel%20pathways%20influence%20blood%20pressure%20and%20cardiovascular%20disease%20risk&author=GB%20Ehret&author=PB%20Munroe&author=KM%20Rice&author=M%20Bochud&author=AD%20Johnson&volume=478&publication_year=2011&pages=103-109&pmid=21909115&doi=10.1038/nature10405&)

8. Johnson AD, Newton-Cheh C, Chasman DI, Ehret GB, Johnson T, Rose L, et al. Association of hypertension drug target genes with blood pressure and hypertension in 86,588 individuals. Hypertension. 2011;57:903–910. doi: 10.1161/HYPERTENSIONAHA.110.158667.  [DOI](https://doi.org/10.1161/HYPERTENSIONAHA.110.158667) | [PMC free article](/articles/PMC3099407/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21444836/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hypertension&title=Association%20of%20hypertension%20drug%20target%20genes%20with%20blood%20pressure%20and%20hypertension%20in%2086,588%20individuals&author=AD%20Johnson&author=C%20Newton-Cheh&author=DI%20Chasman&author=GB%20Ehret&author=T%20Johnson&volume=57&publication_year=2011&pages=903-910&pmid=21444836&doi=10.1161/HYPERTENSIONAHA.110.158667&)

9. Wain LV, Verwoert GC, O’Reilly PF, Shi G, Johnson T, Johnson AD, et al. Genome-wide association study identifies six new loci influencing pulse pressure and mean arterial pressure. Nat Genet. 2011;43:1005–1011. doi: 10.1038/ng.922.  [DOI](https://doi.org/10.1038/ng.922) | [PMC free article](/articles/PMC3445021/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21909110/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Genome-wide%20association%20study%20identifies%20six%20new%20loci%20influencing%20pulse%20pressure%20and%20mean%20arterial%20pressure&author=LV%20Wain&author=GC%20Verwoert&author=PF%20O%E2%80%99Reilly&author=G%20Shi&author=T%20Johnson&volume=43&publication_year=2011&pages=1005-1011&pmid=21909110&doi=10.1038/ng.922&)

10. Levy D, Ehret GB, Rice K, Verwoert GC, Launer LJ, Dehghan A, et al. Genome-wide association study of blood pressure and hypertension. Nat Genet. 2009;41:677–687. doi: 10.1038/ng.384.  [DOI](https://doi.org/10.1038/ng.384) | [PMC free article](/articles/PMC2998712/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19430479/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Genome-wide%20association%20study%20of%20blood%20pressure%20and%20hypertension&author=D%20Levy&author=GB%20Ehret&author=K%20Rice&author=GC%20Verwoert&author=LJ%20Launer&volume=41&publication_year=2009&pages=677-687&pmid=19430479&doi=10.1038/ng.384&)

11. Newton-Cheh C, Johnson T, Gateva V, Tobin MD, Bochud M, Coin L, et al. Genome-wide association study identifies eight loci associated with blood pressure. Nat Genet. 2009;41:666–676. doi: 10.1038/ng.361.  [DOI](https://doi.org/10.1038/ng.361) | [PMC free article](/articles/PMC2891673/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19430483/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Genome-wide%20association%20study%20identifies%20eight%20loci%20associated%20with%20blood%20pressure&author=C%20Newton-Cheh&author=T%20Johnson&author=V%20Gateva&author=MD%20Tobin&author=M%20Bochud&volume=41&publication_year=2009&pages=666-676&pmid=19430483&doi=10.1038/ng.361&)

12. Johnson JA, Boerwinkle E, Zineh I, Chapman AB, Bailey K, Cooper-DeHoff RM, et al. Pharmacogenomics of antihypertensive drugs: Rationale and design of the pharmacogenomic evaluation of antihypertensive responses (pear) study. Am Heart J. 2009;157:442–449. doi: 10.1016/j.ahj.2008.11.018.  [DOI](https://doi.org/10.1016/j.ahj.2008.11.018) | [PMC free article](/articles/PMC2671287/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19249413/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20Heart%20J&title=Pharmacogenomics%20of%20antihypertensive%20drugs:%20Rationale%20and%20design%20of%20the%20pharmacogenomic%20evaluation%20of%20antihypertensive%20responses%20(pear)%20study&author=JA%20Johnson&author=E%20Boerwinkle&author=I%20Zineh&author=AB%20Chapman&author=K%20Bailey&volume=157&publication_year=2009&pages=442-449&pmid=19249413&doi=10.1016/j.ahj.2008.11.018&)

13. Ragot S, Genès N, Vaur L, Herpin D. Comparison of three blood pressure measurement methods for the evaluation of two antihypertensive drugs: Feasibility, agreement, and reproducibility of blood pressure response. Am J Hypertens. 2000;13:632–639. doi: 10.1016/s0895-7061(99)00258-7.  [DOI](https://doi.org/10.1016/s0895-7061(99)00258-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10912746/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hypertens&title=Comparison%20of%20three%20blood%20pressure%20measurement%20methods%20for%20the%20evaluation%20of%20two%20antihypertensive%20drugs:%20Feasibility,%20agreement,%20and%20reproducibility%20of%20blood%20pressure%20response&author=S%20Ragot&author=N%20Gen%C3%A8s&author=L%20Vaur&author=D%20Herpin&volume=13&publication_year=2000&pages=632-639&pmid=10912746&doi=10.1016/s0895-7061(99)00258-7&)

14. Turner ST, Schwartz GL, Chapman AB, Beitelshees AL, Gums JG, Cooper-Dehoff RM, et al. Power to identify a genetic predictor of antihypertensive drug response using different methods to measure blood pressure response. J Transl Med. 2012;10:47. doi: 10.1186/1479-5876-10-47.  [DOI](https://doi.org/10.1186/1479-5876-10-47) | [PMC free article](/articles/PMC3342146/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22413836/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Transl%20Med&title=Power%20to%20identify%20a%20genetic%20predictor%20of%20antihypertensive%20drug%20response%20using%20different%20methods%20to%20measure%20blood%20pressure%20response&author=ST%20Turner&author=GL%20Schwartz&author=AB%20Chapman&author=AL%20Beitelshees&author=JG%20Gums&volume=10&publication_year=2012&pages=47&pmid=22413836&doi=10.1186/1479-5876-10-47&)

15. Stergiou GS, Baibas NM, Gantzarou AP, Skeva II, Kalkana CB, Roussias LG, et al. Reproducibility of home, ambulatory, and clinic blood pressure: Implications for the design of trials for the assessment of antihypertensive drug efficacy. Am J Hypertens. 2002;15:101–104. doi: 10.1016/s0895-7061(01)02324-x.  [DOI](https://doi.org/10.1016/s0895-7061(01)02324-x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11863243/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hypertens&title=Reproducibility%20of%20home,%20ambulatory,%20and%20clinic%20blood%20pressure:%20Implications%20for%20the%20design%20of%20trials%20for%20the%20assessment%20of%20antihypertensive%20drug%20efficacy&author=GS%20Stergiou&author=NM%20Baibas&author=AP%20Gantzarou&author=II%20Skeva&author=CB%20Kalkana&volume=15&publication_year=2002&pages=101-104&pmid=11863243&doi=10.1016/s0895-7061(01)02324-x&)

16. Keating BJ, Tischfield S, Murray SS, Bhangale T, Price TS, Glessner JT, et al. Concept, design and implementation of a cardiovascular gene-centric 50 k snp array for large-scale genomic association studies. PLoS One. 2008;3:e3583. doi: 10.1371/journal.pone.0003583.  [DOI](https://doi.org/10.1371/journal.pone.0003583) | [PMC free article](/articles/PMC2571995/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18974833/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Concept,%20design%20and%20implementation%20of%20a%20cardiovascular%20gene-centric%2050%20k%20snp%20array%20for%20large-scale%20genomic%20association%20studies&author=BJ%20Keating&author=S%20Tischfield&author=SS%20Murray&author=T%20Bhangale&author=TS%20Price&volume=3&publication_year=2008&pages=e3583&pmid=18974833&doi=10.1371/journal.pone.0003583&)

17. Turner ST, Schwartz GL, Chapman AB, Beitelshees AL, Gums JG, Cooper-DeHoff RM, et al. Plasma renin activity predicts blood pressure responses to beta-blocker and thiazide diuretic as monotherapy and add-on therapy for hypertension. Am J Hypertens. 2010;23:1014–1022. doi: 10.1038/ajh.2010.98.  [DOI](https://doi.org/10.1038/ajh.2010.98) | [PMC free article](/articles/PMC2941699/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20725057/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hypertens&title=Plasma%20renin%20activity%20predicts%20blood%20pressure%20responses%20to%20beta-blocker%20and%20thiazide%20diuretic%20as%20monotherapy%20and%20add-on%20therapy%20for%20hypertension&author=ST%20Turner&author=GL%20Schwartz&author=AB%20Chapman&author=AL%20Beitelshees&author=JG%20Gums&volume=23&publication_year=2010&pages=1014-1022&pmid=20725057&doi=10.1038/ajh.2010.98&)

18. Laragh JH, Lamport B, Sealey J, Alderman MH. Diagnosis ex juvantibus. Individual response patterns to drugs reveal hypertension mechanisms and simplify treatment. Hypertension. 1988;12:223–226. doi: 10.1161/01.hyp.12.3.223.  [DOI](https://doi.org/10.1161/01.hyp.12.3.223) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/3049337/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hypertension&title=Diagnosis%20ex%20juvantibus.%20Individual%20response%20patterns%20to%20drugs%20reveal%20hypertension%20mechanisms%20and%20simplify%20treatment&author=JH%20Laragh&author=B%20Lamport&author=J%20Sealey&author=MH%20Alderman&volume=12&publication_year=1988&pages=223-226&pmid=3049337&doi=10.1161/01.hyp.12.3.223&)

19. Purcell S, Neale B, Todd-Brown K, Thomas L, Ferreira MA, Bender D, et al. Plink: A tool set for whole-genome association and population-based linkage analyses. Am J Hum Genet. 2007;81:559–575. doi: 10.1086/519795.  [DOI](https://doi.org/10.1086/519795) | [PMC free article](/articles/PMC1950838/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17701901/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Plink:%20A%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses&author=S%20Purcell&author=B%20Neale&author=K%20Todd-Brown&author=L%20Thomas&author=MA%20Ferreira&volume=81&publication_year=2007&pages=559-575&pmid=17701901&doi=10.1086/519795&)

20. Staessen JA, Wang JG, Thijs L. Cardiovascular protection and blood pressure reduction: A meta-analysis. Lancet. 2001;358:1305–1315. doi: 10.1016/S0140-6736(01)06411-X.  [DOI](https://doi.org/10.1016/S0140-6736(01)06411-X) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11684211/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Cardiovascular%20protection%20and%20blood%20pressure%20reduction:%20A%20meta-analysis&author=JA%20Staessen&author=JG%20Wang&author=L%20Thijs&volume=358&publication_year=2001&pages=1305-1315&pmid=11684211&doi=10.1016/S0140-6736(01)06411-X&)
