# Discovery of Regulatory Elements in Human ATP-Binding Cassette Transporters through Expression Quantitative Trait Mapping

## Metadata
**Authors:** Pär Matsson, Sook Wah Yee, Svetlana Markova, Kari Morrissey, Gerard Jenkins, Jiekun Xuan, Eric Jorgenson, Deanna L Kroetz, Kathleen M Giacomini
**Journal:** The pharmacogenomics journal
**Date:** 2011 Mar 8
**DOI:** [10.1038/tpj.2011.8](https://doi.org/10.1038/tpj.2011.8)
**PMID:** 21383772
**PMCID:** PMC3325368
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3325368/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC3325368/pdf/nihms-266943.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC3325368/pdf/nihms-266943.pdf)

## Abstract

ATP-Binding Cassette (ABC) membrane transporters determine the disposition of many drugs, metabolites and endogenous compounds. Coding region variation in ABC transporters is the cause of many genetic disorders, but much less is known about the genetic basis and functional outcome of ABC transporter expression level variation. We used genotype and mRNA transcript level data from human lymphoblastoid cell lines to assess population and gender differences in ABC transporter expression, and to guide the discovery of genomic regions involved in transcriptional regulation. Nineteen of 49 ABC genes were differentially expressed between individuals of African, Asian and European descent suggesting an important influence of race on expression level of ABC transporters. Twenty-four significant associations were found between transporter transcript levels and proximally located genetic variants. Several of the associations were experimentally validated in reporter assays. Through influencing ABC expression levels, these SNPs may affect disease susceptibility and response to drugs.

Keywords: ATP-Binding Cassette Transporters, Lymphoblastoid Cell Lines, Expression Quantitative Trait Mapping, Transcriptional Regulation, Population Differences

## Introduction

ATP-Binding Cassette (ABC) transporters have diverse and important functions in maintaining cellular homeostasis of endogenous solutes and various xenobiotics including many clinically important drugs.^1-2^ In humans, 49 ABC proteins have been identified and classified into seven subfamilies (ABCA-ABCG). Most of these are membrane transporters that utilize the energy from ATP hydrolysis to mediate efflux of solutes. The different members exhibit widely varying patterns of tissue and subcellular localization, and have substrate specificities that range from endogenous solutes such as cholesterol and bile salts (ABCA1, ABCB11, ABCG1, ABCG5 and ABCG8),^3-6^ phospholipids (ABCB4)^7^ and long-chain fatty acids (ABCD1 and ABCD2),^8-9^ to drugs and their metabolites (ABCB1, ABCG2, and several members of the ABCC family).^2, 10-11^

The important physiological functions of ABC transporters are highlighted by the many genetic disorders resulting from mutations in these proteins, e.g., Tangier’s disease (ABCA1),^4^ progressive familial intrahepatic cholestasis (ABCB4 and ABCB11),^12-13^ adrenoleukodystrophy (ABCD1),^9^ sitosterolemia (ABCG5 and ABCG8),^3, 6^ and cystic fibrosis (CFTR, also referred to as ABCC7).^14^ While such severe clinical phenotypes are typically attributable to loss-of-function or reduced-function mutations, natural variation in transporter expression levels may also result in detrimental clinical conditions. For instance, low expression of ABCA1 is associated with reduced plasma levels of high-density lipoprotein, although less extreme than that observed in Tangier’s disease patients.^15-16^ Similarly, expression level variation in drug transporting ABCs such as ABCB1, ABCC2 and ABCG2 can affect intestinal absorption, biliary and renal excretion, and tissue distribution of substrate drugs, which in turn can lead to loss of pharmacological effects, or to drug-induced toxicities. Identification of regulatory elements in ABC transporter genes is therefore essential for understanding the factors that control the expression levels of these important transporters.

Though many cis-regulatory elements have been identified in the genome, discovery of such elements is difficult. The most common approach to the discovery of cis-regulatory regions has involved comparative genomic methods as a first step to identify conserved sequence motifs followed by further computational analyses to discover transcription factor binding sites.^17-19^ While these methods have resulted in the identification of many regulatory regions, use of evolutionary conservation as a first step may not be reliable. In particular, most regulatory motifs are small, typically consisting of only a few base pairs, and are therefore difficult to identify using sequence comparisons.

Recently, several groups have used publicly available mRNA expression levels in lymphoblastoid cell lines (LBLs) along with genetic variation on a genome-wide scale to identify cis-regulatory variation in the human genome.^20-23^ These studies have revealed global patterns of expression quantitative trait loci (eQTLs), including population differences, and support an abundance of polymorphisms in and nearby genes, which associate with gene expression levels. In this study, we leverage these publicly available datasets, with the specific aims of determining population and individual expression level differences in ABC transporters and to discover and functionally annotate cis regulatory regions that may harbor enhancer or silencer elements that regulate the expression of ABC transporters.

## Materials and Methods

### ABC Gene Expression Data

Gene expression data were downloaded from the Wellcome Trust Sanger Institute web site ([http://www.sanger.ac.uk/humgen/genevar](http://www.sanger.ac.uk/humgen/genevar)). The published expression data had been log-normalized using a quantile normalization method^24^ across experimental replicates for each individual, and then by a median normalization method across all 270 HapMap individuals.^23^ Only data for unrelated individuals were used in the analyses, resulting in a total of 60 US Caucasians of Northern and Western European origin (CEU), 60 Yoruba from Ibadan, Nigeria (YRI), 45 Japanese from the Tokyo area (JPT), and 45 Chinese from the Beijing area (CHB) in the final dataset. For genes where multiple probes were available, analyses were performed both using individual probes and using the mean expression across all probes. The differences were generally negligible and, for clarity, only the results based on mean probe intensities are presented in the paper. Probes were considered detectably expressed in LBLs if their average intensity was above the 25^th^ percentile of all probes in that population.

### Population Dependent Expression Analyses

ANOVA was used to assess population and sex dependent differences in gene expression, using either a single-variable (population) or multi-variable (population+sex) general linear model fit in the R statistics package ([http://www.r-project.org](http://www.r-project.org)). A Bonferroni-corrected *p* < 0.001 was considered significant.

### Genotype Data and Tag SNP Determination

Genotype data from the r27 release of combined data from HapMap Phase II and III were downloaded from the International HapMap Consortium Web Site ([http://www.hapmap.org](http://www.hapmap.org)). For each population, local SNPs, i.e., SNPs located within a region spanning from 50 kb upstream of the transcription start site to 50 kb downstream of the end of the longest transcript (in the UCSC Genome Browser human genome release hg18; [http://genome.ucsc.edu](http://genome.ucsc.edu)) were extracted and used in subsequent analyses. Tag SNPs were determined for the CEU, YRI and combined JPT+CHB (ASI) populations separately, using the Tagger algorithm^25^ in HaploView 4.1 ([http://www.broadinstitute.org/haploview/haploview](http://www.broadinstitute.org/haploview/haploview)) with *r2* ≥ 0.8 and minor allele frequency ≥ 0.05.

### Association Analyses and Multiple Test Corrections

We used a linear model, as implemented in Plink v1.05^26^ ([http://pngu.mgh.harvard.edu/purcell/plink](http://pngu.mgh.harvard.edu/purcell/plink)), to fit normalized log scale expression data for each gene to each of its local tag SNPs. Genotypes were encoded as 0, 1 or 2 depending on the number of minor alleles present. Two types of multiple test corrections were used: permutation correction, in which the order of gene expression levels to genotypes was randomly permutated 25 000 times, and a per-gene Bonferroni correction. A corrected *p* < 0.05 was considered significant. The association analyses were performed in each of the three populations separately. To account for the Asian population being a combination of the Japanese and Han Chinese populations, conditional permutations were performed such that genotypes and phenotypes were only combined within each subpopulation.

### Multiple SNP associations using Partial Least Squares projection

Partial least squares (PLS) projection, as implemented in the PLS R package,^27^ was used to approximate the total expression level variance for each gene explained by its local SNPs (*R*^2^). To achieve comparable results for all genes and to minimize the risk of overfitting the regression models, we used only the first PLS component to estimate the total *R*^2^. The relative contribution of each local SNP to the total *R*^2^ was calculated as the fractional contribution of that SNP’s variable importance (VIP) to the total VIP summed over all SNPs.

### Prioritizing SNPs for Experimental Validation

All tag SNPs significantly associated with ABC gene expression levels were considered for experimental follow-up, as were all HapMap SNPs within their linkage disequilibrium (LD) blocks (*r2* ≥ 0.8). Multiple data sources were used to prioritize SNPs for experiments: a SNP was given a higher probability of being functional, i.e., driving the observed expression variation, if it was i) better correlated with the phenotype than were other SNPs in the LD block, ii) located in regions with higher *a priori* probabilities of being functional, such as proximal to the transcription start site, in the 3′-UTR, or in the first intron, iii) predicted to alter binding of transcription factors by the P-Match ([http://www.gene-regulation.com](http://www.gene-regulation.com)), TF-Search ([http://www.cbrc.jp/research/db/TFSEARCH.html](http://www.cbrc.jp/research/db/TFSEARCH.html)) or the DeltaMatch software^28^, or iv) located in a region conserved between humans and other placental mammals (based on the phyloP44wayPlacMammal track in the UCSC Genome Browser; [http://genome.ucsc.edu](http://genome.ucsc.edu)).

### Cloning of Putative Regulatory Elements

The regions surrounding SNPs selected for experimental validation were amplified by a PCR of pooled human genomic DNA (SOPHIE cohort, [http://pharmacogenetics.ucsf.edu](http://pharmacogenetics.ucsf.edu)), using previously reported PCR protocols.^29-30^ PCR primers were designed using the Primer 3 software ([http://frodo.wi.mit.edu/ primer3](http://frodo.wi.mit.edu/primer3)) by inputting 350 bp upstream and downstream sequence relative to each SNP. The primers used to construct the putative regulatory regions are listed in [Supplemental Table 3](#SD1). The PCR-amplified regions (approximately 500 bp centered on the SNP) were purified by gel extraction (QIAquick Gel Extraction Kit, Qiagen, Germantown, MD) and cloned into pENTR-dTOPO vectors (Invitrogen, Carlsbad, CA). Following plasmid DNA purification (QIAprep Spin Miniprep Kit (Qiagen), proper orientation was confirmed by sequencing. Clones in the correct orientation were subcloned into the multiple cloning site of the pGL4.23[*luc2*/minP] vector (Promega, Madison, WI), upstream of a minimal TATA promoter and a *luc2* firefly luciferase reporter gene. The orientation and sequence of the inserts were then re-verified by sequencing. The use of pooled genomic DNA from individuals of different ethnicities enabled us to obtain both the reference and variant alleles from the same PCR amplification for most regions. Site-directed mutagenesis (QuikChange Site-Directed Mutagenesis Kit, Stratagene, Santa Clara, CA) was used to obtain the variant allele for the remaining regions. For follow-up studies of negative regulatory elements, putative regulatory regions were re-amplified from the pGL4.23[*luc2*/minP] plasmids and subcloned into the pGL3 Promoter vector (Promega), which contains a strong SV40 promoter and a *luc+* firefly luciferase reporter gene downstream from the multiple cloning site.

### Reporter Assays for Determining Transcription Regulatory Effects

The human renal cell adenocarcinoma cell line ACHN was purchased from American Type Culture Collection (ATCC, Manassas, VA). The human hepatoblastoma cell lines HepG2 and Huh-7, human colorectal cell line HCT-116, and human renal adenocarcinoma cell line HEK293T were supplied by the UCSF Cell Culture Facility. All cell lines were maintained in a culture medium consisting of Dulbecco’s modified Eagle’s medium with 4 500 mg/L glucose, supplemented with 10% fetal bovine serum, 100 U/mL penicillin and 100 μg/mL streptomycin. In the reporter assay, the five cell lines were seeded into 48-well culture plates and were transfected 16 to 24 hours later using the Lipofectamine LTX transfection reagent (Invitrogen), according to the manufacturer’s protocols. The pGL4.23[*luc2*/minP] vectors (200 ng), with or without the region of interest, were co-transfected with pGL4.74[*hRluc*/TK] vector(5 ng; to normalize for transfection efficiency) into each well using 1.0 – 1.5 μL Lipofectamine LTX transfection reagent. Cells were lysed with passive lysis buffer 24 hours after transfection, and were assayed for firefly and renilla luciferase activity in a GloMax 96 Microplate Dual Injector Luminometer, using the Dual-Luciferase Reporter Assay system kit (Promega). To account for differences in cell type, transfection efficiency and cell density, reporter activities were normalized as log_2_ [(firefly luciferase/renilla luciferase)/average of the negative control], where negative control refers to cells transfected with the pGL4.23[*luc2*/minP] vector or pGL3-Promoter vector not containing the PCR-amplified regions.

### Accession Numbers

The expression data used in this study have been deposited previously at the Wellcome Trust Sanger Institute Web Site ([http://www.sanger.ac.uk/humgen/genevar](http://www.sanger.ac.uk/humgen/genevar)) and in the Gene Expression Omnibus (GEO) database ([http://www.ncbi.nlm.nih.gov/geo](http://www.ncbi.nlm.nih.gov/geo); accession [GSE6536](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE6536)).

## Results

### Population dependent expression of ABC transporters

Our analyses followed the steps outlined in [Fig. 1](#F1) and began with expression level analyses followed by association analyses of gene expression with genetic variants in and nearby each ABC transporter gene. For expression level analyses, first, we assessed the extent of inter-individual, population dependent and gender dependent variation in the expression levels of each ABC transporter ([Fig. 1A](#F1)). Genome-wide transcript levels, measured in LBLs from the 270 participants in the original HapMap European American (CEU), Han Chinese (CHB), Japanese (JPT) and Yoruba African (YRI) populations, were obtained from Stranger et al.^23^ Probes were available for all 49 human ABC transporters ([Table 1](#T1)), and 45 of these were detectably expressed in at least one of the HapMap populations.

### Fig. 1.

![Fig. 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e0ee/3325368/ad6e8e233987/nihms-266943-f0001.jpg)

Schematic of the workflow applied to identify regulatory regions and SNPs in ABC genes. The number of SNPs assessed in each analysis step is shown in the shaded panel to the right. A: Gene expression data for ABC genes were downloaded and extracted from the Wellcome Trust Sanger Institute GeneVar database. Expression levels for all 210 unrelated individuals were used to determine population differences in ABC gene expression. B: Genotype data was downloaded from the HapMap database, and all SNPs within 50 kb of each ABC gene was used in the further analyses. C: Tag SNPs were determined for each of the CEU, YRI and ASI (JPT+CHB) populations separately. D: Associations between ABC expression levels and SNPs were determined in each population. E: Tag SNPs significantly associated with expression levels, together with all HapMap SNPs in linkage disequilibrium (r2 ≥ 0.8) with them, were prioritized for experimental follow-up based on individual association significance, SNP location, species conservation, and predicted altered transcription factor binding. F: The ~500 bp regions surrounding each selected SNPs were cloned into a luciferase reporter vector (pGL4.23[luc2/minP] vector). G: Luciferase activity was assessed in five different cell lines (ACHN, HEK293T, HCT-116, HepG2 and Huh-7) for each reporter construct. Constructs resulting in decreased luciferase activity compared to the negative control were subsequently cloned into an alternative vector carrying a strong SV40 promoter (pGL3-Promoter vector). H: Allele-specific effects on luciferase activity was determined for all significant regulatory regions.

### Table 1.

| Family | n genesinterrogated(n in genome) | n genesexpressedin CEU1 | n genesexpressedin YRI1 | n genesexpressedin ASI1 | n genesexpressedin anypopulation1 |
| --- | --- | --- | --- | --- | --- |
| ABCA | 12 (12) | 11 | 10 | 10 | 11 |
| ABCB | 11 (11) | 10 | 10 | 9 | 10 |
| ABCC | 13 (13) | 12 | 12 | 12 | 12 |
| ABCD | 4 (4) | 4 | 4 | 4 | 4 |
| ABCE | 1 (1) | 1 | 1 | 1 | 1 |
| ABCF | 3 (3) | 3 | 3 | 3 | 3 |
| ABCG | 5 (5) | 4 | 3 | 4 | 4 |
|   |  |  |  |  |  |
| Total | 49 (49) | 45 | 43 | 43 | 45 |

Table 1 Caption: Number of genes expressed in lymphoblastoid cell lines from HapMap populations in the seven ABC transporter families.

The inter-individual variance of ABC transporter transcript levels was slightly higher than the average for all detectable transcripts, although not statistically significant (1.2-1.5 × genome average; *p* < 0.15-0.5, depending on the population). Compared to within-population variation, population differences contributed significantly less to the overall variability ([Supplemental Table 1](#SD2)). On average, 10% (range: 1-40%) of the total variance for the different transporters was related to population. Expression levels generally correlated well (r^2^ = 0.985-0.997) between pairs of populations ([Fig. 2](#F2)). To increase power, expression level data from the Japanese and Chinese populations were combined in subsequent analyses (ASI population).

### Fig. 2.

![Fig. 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e0ee/3325368/bb723d07fdac/nihms-266943-f0002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3325368_nihms-266943-f0002.jpg)

Comparisons of expression of ABC transporter genes in lymphoblastoid cell lines among four populations. The correlations of normalized probe intensities are shown across all pairs of populations. Probe intensities are shown as filled circles, and the standard error of the mean as grey ellipses. For most probes, the standard errors were smaller than the size of the plot symbols.

Of the 45 ABC transporters expressed in any population, 19 showed statistically significant population differences (Bonferroni-corrected *p* < 0.001, [Fig. 3](#F3)). However, these differences were generally modest, with an average difference in log normalized expression of 0.27±0.18 between the populations with the lowest and highest expression (corresponding to an average relative difference of 21%; range 5-81%) ([Fig. 3](#F3)). No major gender differences were observed ([Supplemental Table 1](#SD2)); the only ABC transporter with a statistically detectable gender effect was ABCG1, which was expressed on average at 22% higher levels in females than in males (Bonferroni-corrected *p* < 0.05).

### Fig. 3.

![Fig. 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e0ee/3325368/b6c9919770ca/nihms-266943-f0003.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3325368_nihms-266943-f0003.jpg)

ABC transporter genes exhibiting significant population differences in expression levels in lymphoblastoid cell lines. ANOVA was used to determine differential expression between the CEU, YRI and ASI populations. The plot shows ABC genes with significant differences after correcting for multiple tests (Bonferroni corrected p < 0.001). Bottom panels show log2 normalized expression values for each population. Top panels show relative differences compared to the overall mean across all populations, in original untransformed units.

### Associations between genotypes and gene expression

Next, we performed association analyses to identify local single nucleotide polymorphisms (SNPs) that associated with expression levels in each of the ABC transporters. (See [Fig. 1B-D](#F1)). Genome-wide analyses have suggested that the majority of regulatory polymorphisms are located within or proximal to the regulated genes,^20, 22-23, 31^ and based on this we decided to include SNPs in a ±50 000 bp region surrounding each transporter to balance coverage of putative regulatory regions against the statistical power to detect associations.

We used genotype data from the combined first, second and third phases of the HapMap project, which resulted in an increased coverage of genetic variation compared to previous genome-wide association analyses for this expression dataset.^23^ In total, 13 242, 13 120 and 13 335 SNPs with a population-specific minor allele frequency of at least 5% were examined in the CEU, YRI and ASI populations, respectively. The number of SNPs per gene varied between 79 and 963 depending on the transporter and the population, but when accounting for the large variation in gene size, the coverage was more similar among genes and populations: between 0.54 and 4.71 SNPs per kb, with an average of 1.46, 1.45 and 1.43 in the Asian, Caucasian, and African populations, respectively.

To increase our power to detect association signals, we exploited linkage disequilibrium patterns to limit the number of statistical tests. As generally observed, genetic diversity was largest in the Yoruba population, for which a total of 3 882 tag SNPs were needed to cover all common HapMap SNPs at an *r2*≥0.8. In the Caucasian and Asian populations, the corresponding numbers were 2 069 and 1 744. The genome-wide association signals obtained for these tag SNPs are shown in [Figure 4](#F4). Each population harbored some SNPs that were associated with expression levels of individual ABC transporters at extraordinarily low p-values. After correcting for multiple tests using a stringent Bonferroni correction or the permutation of phenotype data to determine empirical *p*-values, 24 tag SNPs were significantly associated with the expression levels of 10 different ABC transporters ([Table 2](#T2)). On average, the tag SNPs explained 21±8% (range: 11-38%) of the variation in the corresponding transcript levels. [Figure 5A](#F5) shows the most significant association for each of these 10 transporters.

### Fig. 4.

![Fig. 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e0ee/3325368/65e5a537616a/nihms-266943-f0004.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3325368_nihms-266943-f0004.jpg)

Genome-wide distribution of loci associated with ABC gene expression. The uncorrected significance of each SNP’s association with expression levels in lymphoblastoid cell lines in the CEU, YRI and ASI populations is shown as a function of relative genomic position, divided per chromosome. The relative locations of human ABC genes are shown in the top panel.

### Table 2.

| Gene | Chromo-some | Tag SNP | Relativeposition1 | ntaggedSNPs | Alleles(M/m)2, 3 | MAFCEU4 | MAFYRI4 | MAFASI4 | Mostsignificantin population5 | Directionality3 | r2 | Association(unadjusted p)5 | Association(Bonferroni p)5 |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| ABCA1 | 9 | rs2472507 | In gene | 4 | T/G | 0.20 | 0.06 | 0.35 | CEU | M < m | 0.38 | 8.3×10−7 | 9.2×10−5 |
| ABCA1 | 9 | rs2487030 | −15 752 | 10 | G/A | 0.20 | n. p. | 0.38 | CEU | M < m | 0.33 | 5.8×10−6 | 6.4×10−4 |
| ABCA1 | 9 | rs2515629 | In gene | 1 | T/C | 0.12 | 0.19 | 0.10 | CEU | M < m | 0.24 | 1.6×10−4 | 0.018 |
| ABCA1 | 9 | rs4149297 | In gene | 2 | T/C | 0.08 | 0.09 | n. p. | CEU | M < m | 0.23 | 2.8×10−4 | 0.031 |
| ABCA1 | 9 | rs2000068 | −27 872 | 2 | T/C | 0.38 | 0.17 | 0.36 | CEU | M < m | 0.21 | 4.3×10−4 | 0.048 |
| ABCA8 | 17 | rs11656365 | In gene | 3 | G/A | 0.45 | 0.08 | 0.16 | ASI | M > m | 0.11 | 1.9×10−3 | 0.049 |
| ABCA12 | 2 | rs2888327 | In gene | 1 | G/T | 0.13 | n. p. | n. p. | CEU | M < m | 0.22 | 2.5×10−4 | 0.013 |
| ABCA12 | 2 | rs10182702 | In gene | 1 | A/G | 0.16 | 0.42 | n. p. | YRI | M < m | 0.25 | 1.1×10−4 | 0.015 |
| ABCB3/TAP2 | 6 | rs2071543 | −5 082 | 3 | C/A | 0.16 | 0.09 | 0.18 | YRI | M < m | 0.34 | 3.4×10−6 | 2.4×10−4 |
| ABCB3/TAP2 | 6 | rs9357155 | −3 301 | 1 | C/T | 0.16 | 0.07 | 0.15 | YRI | M < m | 0.32 | 5.7×10−6 | 4.0×10−4 |
| ABCB9 | 12 | rs4148866 | In gene | 3 | G/A | 0.37 | n. p. | 0.44 | CEU | M < m | 0.18 | 1.3×10−3 | 0.016 |
| ABCB9 | 12 | rs12425009 | −42 292 | 7 | A/G | 0.34 | n. p. | 0.43 | CEU | M > m | 0.16 | 2.1×10−3 | 0.027 |
| ABCC4 | 13 | rs4773866 | In gene | 1 | G/A | 0.09 | 0.07 | n. p. | CEU | M < m | 0.22 | 3.0×10−4 | 0.049 |
| ABCC10 | 6 | rs2185631 | In gene | 1 | G/A | 0.49 | 0.13 | 0.44 | ASI | M > m | 0.14 | 3.5×10−4 | 0.0045 |
| ABCC10 | 6 | rs12195350 | −34 129 | 4 | A/G | 0.17 | 0.42 | 0.26 | ASI | M > m | 0.13 | 5.5×10−4 | 0.0072 |
| ABCC10 | 6 | rs2487663 | In gene | 1 | G/A | 0.20 | 0.49 | 0.31 | ASI | M > m | 0.13 | 8.0×10−4 | 0.010 |
| ABCC10 | 6 | rs9394952 | In gene | 9 | A/G | 0.46 | n. p. | 0.34 | ASI | M > m | 0.11 | 1.7×10−3 | 0.022 |
| ABCD1 | X | rs4898439 | 17 452 | 1 | C (A) | 0.19 | 0.31 | 0.46 | ASI | M < m | 0.13 | 7.9×10−4 | 0.0055 |
| ABCD1 | X | rs4898437 | 5 772 | 4 | T (C) | 0.18 | 0.31 | 0.42 | ASI | M < m | 0.12 | 1.4×10−3 | 0.0096 |
| ABCD1 | X | rs17091297 | −9 471 | 1 | G (C) | n. p. | 0.05 | n. p. | YRI | M < m | 0.18 | 1.4×10−3 | 0.043 |
| ABCD2 | 12 | rs4285917 | 44 681 | 20 | T (C) | 0.16 | 0.05 | 0.15 | ASI | M > m | 0.33 | 1.1×10−8 | 1.4×10−7 |
| ABCD2 | 12 | rs4284427 | In gene | 10 | A (G) | 0.33 | n. p. | 0.33 | ASI | M > m | 0.16 | 3.1×10−4 | 0.0040 |
| ABCD2 | 12 | rs10783969 | −39 188 | 13 | C (A) | 0.13 | 0.41 | 0.33 | CEU | M < m | 0.16 | 2.2×10−3 | 0.040 |
| ABCD3 | 1 | rs841698 | 34 435 | 1 | C (T) | n. p. | 0.32 | n. p. | YRI | M < m | 0.21 | 4.8×10−4 | 0.028 |

Table 2 Caption: Summary of SNPs significantly associated with ABC gene expression in lymphoblastoid cell lines from three populations.

### Fig. 5.

![Fig. 5](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e0ee/3325368/91cab5d735ba/nihms-266943-f0005.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3325368_nihms-266943-f0005.jpg)

SNPs that significantly associate with expression levels of ABC transporters in lymphoblastoid cell lines from three populations. A: The most significant association across all populations is shown for each ABC gene with a significant local SNP association. B: Population differences in ABCA12 expression (see Fig. 3) can be attributed to a private SNP in the CEU population. Genotypes are shown for the coding strand.

Notably, for some transporters, multiple significant tag SNPs were detected in a single population. For example, five significant associations were detected in ABCA1 in the CEU population, with *r2* ranging from 0.21 to 0.38. Since a cutoff of *r2* 0.8 was used to define LD blocks, intermediate intercorrelations often still exist among selected tags (e. g., the five significant tags in ABCA1 exhibit pairwise *r2* = 0.03-0.51 and *D*’=0.31-1.0). Consequently, the tag SNPs are not completely independent, and partial correlations can result in cases in which a true causal effect of one tag is detected as a weaker effect in another. We therefore adopted a multivariate approach, using partial least squares (PLS) projection, to estimate the overall contribution of all local SNPs to the expression variation.

For ABC transporters having at least one significant association, the full set of local SNPs explained 25±10% (range: 13-49%) of the total variance in expression levels ([Supplemental Table 2](#SD1)). Very similar results were obtained when only tag SNPs were included in the models, indicating that the PLS projection methodology accounts for SNP intercorrelation caused by linkage disequilibrium. The multi-SNP analyses typically recaptured the same relative importance of local SNPs as that determined through single-SNP associations: the average correlation between PLS-derived SNP weights and correlation coefficients derived from the single-SNP analyses was *r2*=0.92±0.06. In general, the total expression variance explained by all local SNPs was only moderately larger than that explained by the most significant single SNP ([Supplemental Fig. 1](#SD1)) suggesting that for most of the studied genes, a single regulatory locus explains all of the expression variation related to the interrogated local SNPs.

### Population dependent associations

We examined whether SNPs that associated with expression levels in any one population also explained the observed population differences. For seven of the SNPs associated with within-population expression variation, the allele frequencies were highly correlated with population average transcript levels (*r2*≥0.95; <0.1). In particular, the significantly higher expression of ABCA12 in the CEU compared to the YRI and ASI individuals could be attributed to a SNP (rs2888327) private to the Caucasian population ([Fig. 5B](#F5)). Similarly, population average expression levels of ABCC10 were well explained by the allele frequencies of rs12195350 (*r2*=1.0.; *p*<0.002), with expression levels CEU>ASI>YRI and allele frequencies CEU>ASI>YRI (data not shown).

### Detection of transcriptional regulatory regions

SNPs that associated with expression level of the transporters were used to identify putative regulatory regions. These regions were then experimentally validated in medium throughput functional studies using a transient transfection reporter system (see [Fig. 1E-H](#F1)). We prioritized all SNPs in LD with each significant tag based on multiple criteria, including their association with additional phenotypes, their location in genomic regions with higher *a priori* potential for regulatory function (e.g., near the transcription start site, in UTRs, or in the first few introns of the gene of interest), conservation to other placental mammals, or predicted alteration of transcription factor motifs ([Fig. 1E](#F1); Materials and Methods).

A total of 16 SNPs in 7 ABC transporters were selected for experimental verification. For each of these, a region of approximately 500 bp centered on the SNP was cloned from human genomic DNA into a luciferase reporter vector, upstream of a minimal TATA promoter. The constructs were transiently transfected on three to five separate occasions into five different cell lines, derived from human colorectal epithelium (HCT-116), renal epithelium (ACHN, HEK293T), or hepatocytes (HepG2, Huh-7) ([Fig. 6A](#F6)). For most constructs, similar results were obtained across the panel of cell lines. Five of the selected 16 variants resulted in a more than two-fold increased luciferase activity in multiple cell lines, three of which showed more than four-fold increased activity in specific cell types. Notably, several of the cloned regions resulted in a markedly reduced luciferase activity ([Fig. 6A](#F6)). Since the weak promoter is not specifically designed to study negative regulatory effects, we subcloned these six regions into an alternative vector upstream of the stronger SV40 promoter. Two of the regions gave a more than four-fold reduced luciferase activity also for the strong promoter ([Fig. 6B](#F6)), suggesting a binding of repressive regulatory factors to these regions. In summary, guided by statistical associations between genetic polymorphisms and expression levels, we identified and validated a total of five regions that were designated enhancer or silencer regions, and two that were designated moderate enhancers. Through affecting the expression levels of proximally located ABC transporters, these regulatory regions may influence diverse physiological and pharmacological functions including cholesterol efflux to high density lipoprotein,^3-4^ peroxisomal transport of very-long-chain fatty acids,^9, 32^ presentation of cytosolic peptide antigens to MHC class I molecules in the endoplasmatic reticulum,^33-34^ and resistance to anti-cancer drugs such as docetaxel and vinorelbine^35-36^ ([Table 3](#T3)).

### Fig. 6.

![Fig. 6](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e0ee/3325368/e9d3d20a761c/nihms-266943-f0006.jpg)

Regulation of luciferase reporter activity by genomic regions surrounding SNPs in or near ABC transporter genes. A: Putative regulatory elements were cloned into the weak promoter luciferase vector pGL4.23[luc2/minP]. B: Regions resulting in negative regulation in the weak promoter assay were assayed in an alternative vector containing the strong SV40 promoter (pGL3[Promoter]). Luciferase activity is shown relative to that of a vector only containing the promoter and the luciferase gene. The activity of each region was assessed in multiple cell lines (from left to right, for each construct: ACHN, HCT-116, HEK293T, HepG2 and Huh-7; the latter was only used in the weak promoter assay). Red, yellow, light blue, and dark blue bars: luciferase activity >4×, >2×, <0.5×, and <0.25× background promoter activity, respectively. Values are shown as mean ± S. E. M. from three to five separate experiments, each performed in triplicate.

### Table 3.

| Gene | Transporter function | SNP | Clonedregion1 | Designated as |
| --- | --- | --- | --- | --- |
| ABCA1 | Cholesterol transport | rs2487052 | −216 - +245 | Enhancer |
| ABCB3/TAP2 | Peptide transport, antigen presentation | rs2071543 | −230 - +232 | Moderate enhancer |
| ABCB9 | Lysosomal peptide transport | rs3741530 | −247 - +238 | Moderate enhancer |
| ABCC10 | Drug resistance | rs9349256 | −232 - +186 | Enhancer |
| ABCD1 | Peroxisomal fatty acid transport | rs17091297 | −233 - +256 | Silencer |
| ABCD2 | Peroxisomal fatty acid transport | rs12227668 | −275 - +189 | Silencer |
| ABCD2 | Peroxisomal fatty acid transport | rs12228617 | −303 - +194 | Enhancer |

Table 3 Caption: Regulatory regions in ABC transporter genes identified experimentally. Regions were designated enhancers and silencers if they resulted in a more than fourfold increased or decreased reporter activity compared to the negative control, or moderate enhancers if the increase was more than twofold (see Figure 6).

For the seven regulatory regions (five of which resulted in >2-fold enhanced luciferase activity and two of which resulted in a >4-fold reduced luciferase activity), we compared luciferase activities of the reference and variant alleles to determine the effect of the variants on transcription. Significant differences were observed for 5 of the allelic variants ([Table 4](#T4)). For most of these, statistical differences were specific to one or two of the tested cell lines. Four polymorphisms showed the same directionality as that observed in the association with mRNA expression in LBLs, although for two of these the difference did not reach statistical significance. Notably, the SNP rs17091297 in ABCD1 showed large differences across all tested cell lines, reproducing the directionality of the LBL association: the major allele resulted in an almost eight-fold lower reporter activity than the minor allele, mirroring the lower average LBL gene expression for the major allele ([Fig 7](#F7)). In contrast, the SNPs rs12227668 in ABCD2 and rs9349256 in ABCC10 showed the opposite directionality to that expected from LBL expression levels ([Table 4](#T4)); these results were consistent across multiple experiments and cell lines. Computationally predicted transcription factor binding affinities were consistently greater for the genetic variants that showed the higher reporter activity ([Table 4](#T4)), suggesting that the observed allelic differences are real, and that the different directionality could be the result of combinatorial regulation by additional regions in vivo, that were not included in the cloned regions.

### Table 4.

| Gene | SNP | Alleles(M/m)1, 2 | Association(unadjusted p) | Associationdirectionality2 | Association withexpression of flankinggene (Unadjusted p) | Reporter assaysignificantcell line3 | Reporterassaydirectionality2 | Predicted TFbindingdirectionality2, 4, 5 |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
| ABCA1 | rs2487052 | G/A | 8.3×10−7 | M < m | - | n. s. | M < m | n. a. |
| ABCB3/TAP2 | rs2071543 | C/A | 3.4×10−6 | M < m | ABCB2/TAP1 (p<0.02) | AC | M < m | n. a. |
| ABCB9 | rs3741530 | A/C | 2.1×10−3 | M > m | ARL6IP4 (p<0.009) | AC | M < m | n. a. |
| ABCC10 | rs9349256 | A/G | 1.9×10−3 | M > m | - | HG, HU | M < m | M < m (HNF-4) |
| ABCD1 | rs17091297 | G/C | 1.4×10−3 | M < m | IDH3G (p<0.05) | AC, HE, HC, HG | M < m | M < m (AP-4) |
| ABCD2 | rs12227668 | A/G | 1.1×10−8 | M > m | - | AC, HE, HC | M < m | M < m (HSF) |
| ABCD2 | rs12228617 | C/A | 3.0×10−7 | M > m | C12orf40 (p<0.04) | n. s. | M > m | n. a. |

Table 4 Caption: Direction of association of selected major (M) versus minor (m) alleles of ABC transporter genes with expression levels in lymphoblastoid cell lines and with lucifierase activities. In four cases, the direction of association of expression level agreed with the reporter assay, e.g., the major allele of SNP rs2071543 was associated with a lower expression level and reporter activity.

### Fig. 7.

![Fig. 7](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e0ee/3325368/2bd5b17cafae/nihms-266943-f0007.jpg)

Association of a SNP in ABCD1 with expression levels in lymphoblastoid cell lines from the YRI population, and with luciferase reporter activity. A: The SNP rs17091297 is correlated with ABCD1 mRNA levels in the YRI population (Bonferroni-corrected p < 0.05). B: The major allele results in decreased luciferase activity compared to the control, when inserted upstream of a strong SV40 promoter. Luciferase activity is shown relative to the background promoter activity. Values are shown as mean ± S. E. M. from four separate experiments, each performed in triplicate.

Thus, by using statistical associations between SNPs and expression levels to guide the experiments, five regions were identified that resulted in a markedly altered reporter activity, and an additional six having intermediate effects. To assess if this was more than expected by chance, we estimated the total number of regulatory regions in the examined ABC transporters based on a combination of species conservation data and experimental data for DNAse I hypersensitivity or histone 3-lysine 4 monomethylation (H3K4me1). Each of these measures have been shown to correlate with regulatory function,^37-40^ but used separately they are likely to overestimate the number of regulatory active sites. We used the fraction of the interrogated genomic DNA that had two or more of these characteristics as an estimate of true regulatory function, resulting in a background chance of hitting a regulatory region of 6.4% ([Supplemental Table 4](#SD1)). In contrast, our rate of 29-69%, depending on the cutoff used to call a significantly altered reporter activity, is significantly higher (*p* < 0.0002–1×10^−14^).

## Discussion

Predisposition to disease and the response to drugs are highly variable in human populations,^41-42^ with expression level differences in various genes contributing to interindividual differences in disease risk and drug response^43-44^. We assessed the influence of ethnicity, gender, and local genetic variation on the expression levels of ABC transporters in lymphoblastoid cell lines, and used the detected expression quantitative trait loci (eQTL) to guide experimental assays of putative regulatory genomic regions.

For 19 of the 49 ABC genes in the human genome, ethnicity was significantly associated with expression levels. Since ABC transporters are known to affect the disposition of many drugs, metabolites and endogenous ligands,^1-2^ such population-dependent expression can ultimately lead to differential susceptibility to disease or outcome of drug therapy among populations. For example, we observed that the expression level of the hepatic bile salt efflux pump (ABCB11/BSEP), which when mutated is associated with familial, drug-induced and pregnancy-induced intrahepatic cholestasis,^13, 45-46^ was significantly lower in the Asian than in the European and African samples. Though our data cannot be directly extrapolated to expression level differences in the liver, if such population differences in hepatic expression levels of ABCB11 occurred, our data would suggest that Asians may be more susceptible to intrahepatic cholestasis mediated by BSEP. Intriguingly, the incidence of xxx is higher in Asians than in European women, and the possible connection with population dependent BSEP expression thus warrants further studies.

The largest difference among populations was observed for ABCG1, with expression levels on average 1.8 fold higher in the Asian compared to the Caucasian samples. Notably, in contrast to the several cases of statistically significant population dependent expression, ABCG1 was the only ABC transporter for which gender was significantly associated with expression. ABCG1 mediates the efflux of cellular cholesterol to high density lipoprotein (HDL),^5, 47^ and the higher average expression levels in women, and in the Asian population, suggests that ABCG1 expression might contribute to the relatively higher HDL plasma levels observed in these populations.^48-49^ Again, some caution in extrapolation of our data in lymphoblastoid cell lines to liver must be exercised.

Previous studies have indicated that genetic differences account for a substantial part of observed gene expression differences among populations.^50-51^ This was also observed here for some of our genes, with population differences in allele frequencies generally well correlated with population average expression. Particularly, the SNP rs2888327 was only observed in the CEU population, which had a higher average expression of the proximally located ABCA12 gene than the ASI and YRI populations ([Figure 5B](#F5)). On the whole, however, the contribution of population and gender was small compared to the total ABC gene expression variation, consistent with other studies of population effects on gene expression,^50-52^ and also with the observation that 85-95% of human genetic variation is due to within-population variation.^53-54^

With current advances in sequencing techniques, access to near-complete personal genomes is not far from reality. However, for these large-scale datasets to be truly useful, much work is needed in functionally annotating the human genome. So far, our knowledge of non-coding regions is greatly lagging behind that of protein coding regions. Especially little is known about the extent and function of distantly located regulatory elements such as enhancers and silencers. Evolutionary constraints on non-coding sequences have been successfully used to identify regions with regulatory function.^17-19^ However, not all evolutionarily conserved regions have detectable functions in in vitro and in vivo enhancer assays.^18-19, 55^ Furthermore, many regulatory elements are likely located in regions not detectable using comparative genomics.^56-57^ Alternatively, DNAse I hypersensitivity,^37-38^ binding of enhancer-associated proteins such as the acetyltransferase and transcriptional coactivator p300,^40, 55^ and epigenetic modifications such as histone methylation and acetylation^39-40^ have been used to predict the genome-wide locations of regulatory regions. While highly promising, additional experimental validation of predicted regulatory regions is needed to determine the predictive value of these methods.

Here, we used a complementary approach to identify putative regulatory regions, through the association of genetic polymorphism with expression levels of ABC transporters. We identified 24 tag SNPs that were significantly associated with ABC transporter expression levels in lymphoblastoid cell lines. These were in linkage disequilibrium with a total of 104 interrogated SNPs with a minor allele frequency >5% in at least one population. After prioritizing the initial list of possible eQTLs based on association strength, genomic location, predicted alteration of transcription factor binding sites and species conservation, 16 regions within a ±50 000 bp range surrounding 7 ABC transporters were selected for experimental validation. This eQTL-centered approach led to the identification of five genomic regions that resulted in at least fourfold increased or decreased reporter activity, and six additional regions with at least twofold altered activities. Allele-specific regulatory function was shown for five of the regions. Genetic variation in these regions can thus contribute to differential expression among individuals, and can ultimately affect drug response or susceptibility to disease.

Previous analyses of expression quantitative traits in lymphoblastoid cell lines^20, 22^ have focused on global properties of regulatory polymorphisms, and have not included experimental verification of identified associations. In our gene family focused effort, 5 to 11 of 16 tested associations were experimentally replicated depending on the reporter assay cutoff used. However, five regions did not alter reporter activity. While significant signals in the reporter assay suggest causative regulatory effects of the examined regions, interpreting the lack of signal is less straight-forward. It could be a consequence of false positives in the association analysis, but could also result from linkage disequilibrium between the examined region and the actual causative site. In addition, combinatorial regulation of gene expression in vivo is difficult to replicate in an artificial system that does not include the larger genomic context.

We recently used a combination of comparative genomics and transcription factor binding site predictions to select 50 putative regulatory regions in important hepatic drug transporters from the ABC and SLC families.^58^ The enhancer activities of these regions were assayed in mice using hydrodynamic tail vein injection, which results in a highly localized expression in the liver. Using this technique, 12/50 (24%) of the prioritized regions showed experimental activity in vivo. Notably, however, none of the regions with experimental regulatory impact in the present study were detectably conserved among placental mammals, suggesting that eQTL mapping provides additional information compared to comparative genomics approaches.

The present work thus shows the general utility of expression level quantitative trait loci in defining genomic regions with regulatory potential, and, specifically, demonstrates transcriptional regulation of human ABC transporters by several hitherto uncharacterized genomic regions. Genetic and epigenetic variation in the identified regions can thus lead to altered transport of drugs and their metabolites, and of endogenous solutes such as cholesterol, fatty acids and peptide fragments. Ultimately, this can cause inter-individual variation in the pharmacological response to drugs and in important physiological processes, including cellular lipid metabolism, lysosomal protein degradation and in antigen presentation on the cell surface.

## Supplementary Material

## Acknowledgements

This work was supported by National Institutes of Health grant GM61390. PM was supported in part by a grant from the IF Foundation for Pharmaceutical Sciences, Sweden.

**Funding sources** This work was supported by National Institutes of Health grant GM61390. PM was supported in part by a grant from the IF Foundation for Pharmaceutical Sciences, Sweden.

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Glavinas H, Krajcsi P, Cserepes J, Sarkadi B. The role of ABC transporters in drug resistance, metabolism and toxicity. Curr Drug Deliv. 2004;1(1):27–42. doi: 10.2174/1567201043480036.  [DOI](https://doi.org/10.2174/1567201043480036) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16305368/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Drug%20Deliv&title=The%20role%20of%20ABC%20transporters%20in%20drug%20resistance,%20metabolism%20and%20toxicity&author=H%20Glavinas&author=P%20Krajcsi&author=J%20Cserepes&author=B%20Sarkadi&volume=1&issue=1&publication_year=2004&pages=27-42&pmid=16305368&doi=10.2174/1567201043480036&)

2. Schinkel AH, Jonker JW. Mammalian drug efflux transporters of the ATP binding cassette (ABC) family: an overview. Adv Drug Deliv Rev. 2003;55(1):3–29. doi: 10.1016/s0169-409x(02)00169-2.  [DOI](https://doi.org/10.1016/s0169-409x(02)00169-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12535572/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Adv%20Drug%20Deliv%20Rev&title=Mammalian%20drug%20efflux%20transporters%20of%20the%20ATP%20binding%20cassette%20(ABC)%20family:%20an%20overview&author=AH%20Schinkel&author=JW%20Jonker&volume=55&issue=1&publication_year=2003&pages=3-29&pmid=12535572&doi=10.1016/s0169-409x(02)00169-2&)

3. Berge KE, Tian H, Graf GA, Yu L, Grishin NV, Schultz J, et al. Accumulation of dietary cholesterol in sitosterolemia caused by mutations in adjacent ABC transporters. Science. 2000;290(5497):1771–1775. doi: 10.1126/science.290.5497.1771.  [DOI](https://doi.org/10.1126/science.290.5497.1771) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11099417/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Accumulation%20of%20dietary%20cholesterol%20in%20sitosterolemia%20caused%20by%20mutations%20in%20adjacent%20ABC%20transporters&author=KE%20Berge&author=H%20Tian&author=GA%20Graf&author=L%20Yu&author=NV%20Grishin&volume=290&issue=5497&publication_year=2000&pages=1771-1775&pmid=11099417&doi=10.1126/science.290.5497.1771&)

4. Brooks-Wilson A, Marcil M, Clee SM, Zhang LH, Roomp K, van Dam M, et al. Mutations in ABC1 in Tangier disease and familial high-density lipoprotein deficiency. Nat Genet. 1999;22(4):336–345. doi: 10.1038/11905.  [DOI](https://doi.org/10.1038/11905) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10431236/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Mutations%20in%20ABC1%20in%20Tangier%20disease%20and%20familial%20high-density%20lipoprotein%20deficiency&author=A%20Brooks-Wilson&author=M%20Marcil&author=SM%20Clee&author=LH%20Zhang&author=K%20Roomp&volume=22&issue=4&publication_year=1999&pages=336-345&pmid=10431236&doi=10.1038/11905&)

5. Klucken J, Buchler C, Orso E, Kaminski WE, Porsch-Ozcurumez M, Liebisch G, et al. ABCG1 (ABC8), the human homolog of the Drosophila white gene, is a regulator of macrophage cholesterol and phospholipid transport. Proc Natl Acad Sci U S A. 2000;97(2):817–822. doi: 10.1073/pnas.97.2.817.  [DOI](https://doi.org/10.1073/pnas.97.2.817) | [PMC free article](/articles/PMC15414/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10639163/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&title=ABCG1%20(ABC8),%20the%20human%20homolog%20of%20the%20Drosophila%20white%20gene,%20is%20a%20regulator%20of%20macrophage%20cholesterol%20and%20phospholipid%20transport&author=J%20Klucken&author=C%20Buchler&author=E%20Orso&author=WE%20Kaminski&author=M%20Porsch-Ozcurumez&volume=97&issue=2&publication_year=2000&pages=817-822&pmid=10639163&doi=10.1073/pnas.97.2.817&)

6. Lee MH, Lu K, Hazard S, Yu H, Shulenin S, Hidaka H, et al. Identification of a gene, ABCG5, important in the regulation of dietary cholesterol absorption. Nat Genet. 2001;27(1):79–83. doi: 10.1038/83799.  [DOI](https://doi.org/10.1038/83799) | [PMC free article](/articles/PMC1350991/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11138003/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Identification%20of%20a%20gene,%20ABCG5,%20important%20in%20the%20regulation%20of%20dietary%20cholesterol%20absorption&author=MH%20Lee&author=K%20Lu&author=S%20Hazard&author=H%20Yu&author=S%20Shulenin&volume=27&issue=1&publication_year=2001&pages=79-83&pmid=11138003&doi=10.1038/83799&)

7. Ruetz S, Gros P. Phosphatidylcholine translocase: a physiological role for the mdr2 gene. Cell. 1994;77(7):1071–1081. doi: 10.1016/0092-8674(94)90446-4.  [DOI](https://doi.org/10.1016/0092-8674(94)90446-4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7912658/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell&title=Phosphatidylcholine%20translocase:%20a%20physiological%20role%20for%20the%20mdr2%20gene&author=S%20Ruetz&author=P%20Gros&volume=77&issue=7&publication_year=1994&pages=1071-1081&pmid=7912658&doi=10.1016/0092-8674(94)90446-4&)

8. Lombard-Platet G, Savary S, Sarde CO, Mandel JL, Chimini G. A close relative of the adrenoleukodystrophy (ALD) gene codes for a peroxisomal protein with a specific expression pattern. Proc Natl Acad Sci U S A. 1996;93(3):1265–1269. doi: 10.1073/pnas.93.3.1265.  [DOI](https://doi.org/10.1073/pnas.93.3.1265) | [PMC free article](/articles/PMC40068/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8577752/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&title=A%20close%20relative%20of%20the%20adrenoleukodystrophy%20(ALD)%20gene%20codes%20for%20a%20peroxisomal%20protein%20with%20a%20specific%20expression%20pattern&author=G%20Lombard-Platet&author=S%20Savary&author=CO%20Sarde&author=JL%20Mandel&author=G%20Chimini&volume=93&issue=3&publication_year=1996&pages=1265-1269&pmid=8577752&doi=10.1073/pnas.93.3.1265&)

9. Watkins PA, Gould SJ, Smith MA, Braiterman LT, Wei HM, Kok F, et al. Altered expression of ALDP in X-linked adrenoleukodystrophy. Am J Hum Genet. 1995;57(2):292–301.  [PMC free article](/articles/PMC1801558/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7668254/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Altered%20expression%20of%20ALDP%20in%20X-linked%20adrenoleukodystrophy&author=PA%20Watkins&author=SJ%20Gould&author=MA%20Smith&author=LT%20Braiterman&author=HM%20Wei&volume=57&issue=2&publication_year=1995&pages=292-301&pmid=7668254&)

10. Kruh GD, Belinsky MG. The MRP family of drug efflux pumps. Oncogene. 2003;22(47):7537–7552. doi: 10.1038/sj.onc.1206953.  [DOI](https://doi.org/10.1038/sj.onc.1206953) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14576857/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncogene&title=The%20MRP%20family%20of%20drug%20efflux%20pumps&author=GD%20Kruh&author=MG%20Belinsky&volume=22&issue=47&publication_year=2003&pages=7537-7552&pmid=14576857&doi=10.1038/sj.onc.1206953&)

11. Vlaming ML, Lagas JS, Schinkel AH. Physiological and pharmacological roles of ABCG2 (BCRP): recent findings in Abcg2 knockout mice. Adv Drug Deliv Rev. 2009;61(1):14–25. doi: 10.1016/j.addr.2008.08.007.  [DOI](https://doi.org/10.1016/j.addr.2008.08.007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19118589/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Adv%20Drug%20Deliv%20Rev&title=Physiological%20and%20pharmacological%20roles%20of%20ABCG2%20(BCRP):%20recent%20findings%20in%20Abcg2%20knockout%20mice&author=ML%20Vlaming&author=JS%20Lagas&author=AH%20Schinkel&volume=61&issue=1&publication_year=2009&pages=14-25&pmid=19118589&doi=10.1016/j.addr.2008.08.007&)

12. Deleuze JF, Jacquemin E, Dubuisson C, Cresteil D, Dumont M, Erlinger S, et al. Defect of multidrug-resistance 3 gene expression in a subtype of progressive familial intrahepatic cholestasis. Hepatology. 1996;23(4):904–908. doi: 10.1002/hep.510230435.  [DOI](https://doi.org/10.1002/hep.510230435) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8666348/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hepatology&title=Defect%20of%20multidrug-resistance%203%20gene%20expression%20in%20a%20subtype%20of%20progressive%20familial%20intrahepatic%20cholestasis&author=JF%20Deleuze&author=E%20Jacquemin&author=C%20Dubuisson&author=D%20Cresteil&author=M%20Dumont&volume=23&issue=4&publication_year=1996&pages=904-908&pmid=8666348&doi=10.1002/hep.510230435&)

13. Strautnieks SS, Bull LN, Knisely AS, Kocoshis SA, Dahl N, Arnell H, et al. A gene encoding a liver-specific ABC transporter is mutated in progressive familial intrahepatic cholestasis. Nat Genet. 1998;20(3):233–238. doi: 10.1038/3034.  [DOI](https://doi.org/10.1038/3034) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9806540/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=A%20gene%20encoding%20a%20liver-specific%20ABC%20transporter%20is%20mutated%20in%20progressive%20familial%20intrahepatic%20cholestasis&author=SS%20Strautnieks&author=LN%20Bull&author=AS%20Knisely&author=SA%20Kocoshis&author=N%20Dahl&volume=20&issue=3&publication_year=1998&pages=233-238&pmid=9806540&doi=10.1038/3034&)

14. Kerem BS, Buchanan JA, Durie P, Corey ML, Levison H, Rommens JM, et al. DNA marker haplotype association with pancreatic sufficiency in cystic fibrosis. Am J Hum Genet. 1989;44(6):827–834.  [PMC free article](/articles/PMC1715674/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2567116/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=DNA%20marker%20haplotype%20association%20with%20pancreatic%20sufficiency%20in%20cystic%20fibrosis&author=BS%20Kerem&author=JA%20Buchanan&author=P%20Durie&author=ML%20Corey&author=H%20Levison&volume=44&issue=6&publication_year=1989&pages=827-834&pmid=2567116&)

15. Frikke-Schmidt R, Nordestgaard BG, Jensen GB, Tybjaerg-Hansen A. Genetic variation in ABC transporter A1 contributes to HDL cholesterol in the general population. J Clin Invest. 2004;114(9):1343–1353. doi: 10.1172/JCI20361.  [DOI](https://doi.org/10.1172/JCI20361) | [PMC free article](/articles/PMC524222/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15520867/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Invest&title=Genetic%20variation%20in%20ABC%20transporter%20A1%20contributes%20to%20HDL%20cholesterol%20in%20the%20general%20population&author=R%20Frikke-Schmidt&author=BG%20Nordestgaard&author=GB%20Jensen&author=A%20Tybjaerg-Hansen&volume=114&issue=9&publication_year=2004&pages=1343-1353&pmid=15520867&doi=10.1172/JCI20361&)

16. Slatter TL, Jones GT, Williams MJ, van Rij AM, McCormick SP. Novel rare mutations and promoter haplotypes in ABCA1 contribute to low-HDL-C levels. Clin Genet. 2008;73(2):179–184. doi: 10.1111/j.1399-0004.2007.00940.x.  [DOI](https://doi.org/10.1111/j.1399-0004.2007.00940.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18199144/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Genet&title=Novel%20rare%20mutations%20and%20promoter%20haplotypes%20in%20ABCA1%20contribute%20to%20low-HDL-C%20levels&author=TL%20Slatter&author=GT%20Jones&author=MJ%20Williams&author=AM%20van%20Rij&author=SP%20McCormick&volume=73&issue=2&publication_year=2008&pages=179-184&pmid=18199144&doi=10.1111/j.1399-0004.2007.00940.x&)

17. Nobrega MA, Ovcharenko I, Afzal V, Rubin EM. Scanning human gene deserts for longrange enhancers. Science. 2003;302(5644):413. doi: 10.1126/science.1088328.  [DOI](https://doi.org/10.1126/science.1088328) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14563999/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Scanning%20human%20gene%20deserts%20for%20longrange%20enhancers&author=MA%20Nobrega&author=I%20Ovcharenko&author=V%20Afzal&author=EM%20Rubin&volume=302&issue=5644&publication_year=2003&pages=413&pmid=14563999&doi=10.1126/science.1088328&)

18. Pennacchio LA, Ahituv N, Moses AM, Prabhakar S, Nobrega MA, Shoukry M, et al. In vivo enhancer analysis of human conserved non-coding sequences. Nature. 2006;444(7118):499–502. doi: 10.1038/nature05295.  [DOI](https://doi.org/10.1038/nature05295) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17086198/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=In%20vivo%20enhancer%20analysis%20of%20human%20conserved%20non-coding%20sequences&author=LA%20Pennacchio&author=N%20Ahituv&author=AM%20Moses&author=S%20Prabhakar&author=MA%20Nobrega&volume=444&issue=7118&publication_year=2006&pages=499-502&pmid=17086198&doi=10.1038/nature05295&)

19. Visel A, Prabhakar S, Akiyama JA, Shoukry M, Lewis KD, Holt A, et al. Ultraconservation identifies a small subset of extremely constrained developmental enhancers. Nat Genet. 2008;40(2):158–160. doi: 10.1038/ng.2007.55.  [DOI](https://doi.org/10.1038/ng.2007.55) | [PMC free article](/articles/PMC2647775/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18176564/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Ultraconservation%20identifies%20a%20small%20subset%20of%20extremely%20constrained%20developmental%20enhancers&author=A%20Visel&author=S%20Prabhakar&author=JA%20Akiyama&author=M%20Shoukry&author=KD%20Lewis&volume=40&issue=2&publication_year=2008&pages=158-160&pmid=18176564&doi=10.1038/ng.2007.55&)

20. Dixon AL, Liang L, Moffatt MF, Chen W, Heath S, Wong KC, et al. A genome-wide association study of global gene expression. Nat Genet. 2007;39(10):1202–1207. doi: 10.1038/ng2109.  [DOI](https://doi.org/10.1038/ng2109) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17873877/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=A%20genome-wide%20association%20study%20of%20global%20gene%20expression&author=AL%20Dixon&author=L%20Liang&author=MF%20Moffatt&author=W%20Chen&author=S%20Heath&volume=39&issue=10&publication_year=2007&pages=1202-1207&pmid=17873877&doi=10.1038/ng2109&)

21. Huang RS, Duan S, Shukla SJ, Kistner EO, Clark TA, Chen TX, et al. Identification of genetic variants contributing to cisplatin-induced cytotoxicity by use of a genomewide approach. Am J Hum Genet. 2007;81(3):427–437. doi: 10.1086/519850.  [DOI](https://doi.org/10.1086/519850) | [PMC free article](/articles/PMC1950832/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17701890/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Identification%20of%20genetic%20variants%20contributing%20to%20cisplatin-induced%20cytotoxicity%20by%20use%20of%20a%20genomewide%20approach&author=RS%20Huang&author=S%20Duan&author=SJ%20Shukla&author=EO%20Kistner&author=TA%20Clark&volume=81&issue=3&publication_year=2007&pages=427-437&pmid=17701890&doi=10.1086/519850&)

22. Stranger BE, Forrest MS, Clark AG, Minichiello MJ, Deutsch S, Lyle R, et al. Genome-wide associations of gene expression variation in humans. PLoS Genet. 2005;1(6):e78. doi: 10.1371/journal.pgen.0010078.  [DOI](https://doi.org/10.1371/journal.pgen.0010078) | [PMC free article](/articles/PMC1315281/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16362079/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet&title=Genome-wide%20associations%20of%20gene%20expression%20variation%20in%20humans&author=BE%20Stranger&author=MS%20Forrest&author=AG%20Clark&author=MJ%20Minichiello&author=S%20Deutsch&volume=1&issue=6&publication_year=2005&pages=e78&pmid=16362079&doi=10.1371/journal.pgen.0010078&)

23. Stranger BE, Nica AC, Forrest MS, Dimas A, Bird CP, Beazley C, et al. Population genomics of human gene expression. Nat Genet. 2007;39(10):1217–1224. doi: 10.1038/ng2142.  [DOI](https://doi.org/10.1038/ng2142) | [PMC free article](/articles/PMC2683249/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17873874/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Population%20genomics%20of%20human%20gene%20expression&author=BE%20Stranger&author=AC%20Nica&author=MS%20Forrest&author=A%20Dimas&author=CP%20Bird&volume=39&issue=10&publication_year=2007&pages=1217-1224&pmid=17873874&doi=10.1038/ng2142&)

24. Bolstad BM, Irizarry RA, Astrand M, Speed TP. A comparison of normalization methods for high density oligonucleotide array data based on variance and bias. Bioinformatics. 2003;19(2):185–193. doi: 10.1093/bioinformatics/19.2.185.  [DOI](https://doi.org/10.1093/bioinformatics/19.2.185) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12538238/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=A%20comparison%20of%20normalization%20methods%20for%20high%20density%20oligonucleotide%20array%20data%20based%20on%20variance%20and%20bias&author=BM%20Bolstad&author=RA%20Irizarry&author=M%20Astrand&author=TP%20Speed&volume=19&issue=2&publication_year=2003&pages=185-193&pmid=12538238&doi=10.1093/bioinformatics/19.2.185&)

25. Barrett JC, Fry B, Maller J, Daly MJ. Haploview: analysis and visualization of LD and haplotype maps. Bioinformatics. 2005;21(2):263–265. doi: 10.1093/bioinformatics/bth457.  [DOI](https://doi.org/10.1093/bioinformatics/bth457) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15297300/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=Haploview:%20analysis%20and%20visualization%20of%20LD%20and%20haplotype%20maps&author=JC%20Barrett&author=B%20Fry&author=J%20Maller&author=MJ%20Daly&volume=21&issue=2&publication_year=2005&pages=263-265&pmid=15297300&doi=10.1093/bioinformatics/bth457&)

26. Purcell S, Neale B, Todd-Brown K, Thomas L, Ferreira MA, Bender D, et al. PLINK: a tool set for whole-genome association and population-based linkage analyses. Am J Hum Genet. 2007;81(3):559–575. doi: 10.1086/519795.  [DOI](https://doi.org/10.1086/519795) | [PMC free article](/articles/PMC1950838/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17701901/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=PLINK:%20a%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses&author=S%20Purcell&author=B%20Neale&author=K%20Todd-Brown&author=L%20Thomas&author=MA%20Ferreira&volume=81&issue=3&publication_year=2007&pages=559-575&pmid=17701901&doi=10.1086/519795&)

27. Mevik BH, Wehrens R. The pls package: Principal component and partial least squares regression in R. J Stat Softw. 2007;18(2)  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Stat%20Softw&title=The%20pls%20package:%20Principal%20component%20and%20partial%20least%20squares%20regression%20in%20R&author=BH%20Mevik&author=R%20Wehrens&volume=18&issue=2&publication_year=2007&)

28. Williamson DW. PhD thesis. University of California San Francisco; San Francisco, CA: 2008. Genetic and bioinformatic approaches to identify polymorphic modulators of transcription factor binding and disease phenotypes including HIV-1 viremia.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=PhD%20thesis&author=DW%20Williamson&publication_year=2008&)

29. Tahara H, Yee SW, Urban TJ, Hesselson S, Castro RA, Kawamoto M, et al. Functional genetic variation in the basal promoter of the organic cation/carnitine transporters OCTN1 (SLC22A4) and OCTN2 (SLC22A5) J Pharmacol Exp Ther. 2009;329(1):262–271. doi: 10.1124/jpet.108.146449.  [DOI](https://doi.org/10.1124/jpet.108.146449) | [PMC free article](/articles/PMC2670604/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19141711/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacol%20Exp%20Ther&title=Functional%20genetic%20variation%20in%20the%20basal%20promoter%20of%20the%20organic%20cation/carnitine%20transporters%20OCTN1%20(SLC22A4)%20and%20OCTN2%20(SLC22A5)&author=H%20Tahara&author=SW%20Yee&author=TJ%20Urban&author=S%20Hesselson&author=RA%20Castro&volume=329&issue=1&publication_year=2009&pages=262-271&pmid=19141711&doi=10.1124/jpet.108.146449&)

30. Yee SW, Shima JE, Hesselson S, Nguyen L, De Val S, Lafond RJ, et al. Identification and characterization of proximal promoter polymorphisms in the human concentrative nucleoside transporter 2 (SLC28A2) J Pharmacol Exp Ther. 2009;328(3):699–707. doi: 10.1124/jpet.108.147207.  [DOI](https://doi.org/10.1124/jpet.108.147207) | [PMC free article](/articles/PMC2682268/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19098160/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacol%20Exp%20Ther&title=Identification%20and%20characterization%20of%20proximal%20promoter%20polymorphisms%20in%20the%20human%20concentrative%20nucleoside%20transporter%202%20(SLC28A2)&author=SW%20Yee&author=JE%20Shima&author=S%20Hesselson&author=L%20Nguyen&author=S%20De%20Val&volume=328&issue=3&publication_year=2009&pages=699-707&pmid=19098160&doi=10.1124/jpet.108.147207&)

31. Veyrieras JB, Kudaravalli S, Kim SY, Dermitzakis ET, Gilad Y, Stephens M, et al. High-resolution mapping of expression-QTLs yields insight into human gene regulation. PLoS Genet. 2008;4(10):e1000214. doi: 10.1371/journal.pgen.1000214.  [DOI](https://doi.org/10.1371/journal.pgen.1000214) | [PMC free article](/articles/PMC2556086/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18846210/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet&title=High-resolution%20mapping%20of%20expression-QTLs%20yields%20insight%20into%20human%20gene%20regulation&author=JB%20Veyrieras&author=S%20Kudaravalli&author=SY%20Kim&author=ET%20Dermitzakis&author=Y%20Gilad&volume=4&issue=10&publication_year=2008&pages=e1000214&pmid=18846210&doi=10.1371/journal.pgen.1000214&)

32. van Roermund CW, Visser WF, Ijlst L, van Cruchten A, Boek M, Kulik W, et al. The human peroxisomal ABC half transporter ALDP functions as a homodimer and accepts acyl-CoA esters. FASEB J. 2008;22(12):4201–4208. doi: 10.1096/fj.08-110866.  [DOI](https://doi.org/10.1096/fj.08-110866) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18757502/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=FASEB%20J&title=The%20human%20peroxisomal%20ABC%20half%20transporter%20ALDP%20functions%20as%20a%20homodimer%20and%20accepts%20acyl-CoA%20esters&author=CW%20van%20Roermund&author=WF%20Visser&author=L%20Ijlst&author=A%20van%20Cruchten&author=M%20Boek&volume=22&issue=12&publication_year=2008&pages=4201-4208&pmid=18757502&doi=10.1096/fj.08-110866&)

33. Bahram S, Arnold D, Bresnahan M, Strominger JL, Spies T. Two putative subunits of a peptide pump encoded in the human major histocompatibility complex class II region. Proc Natl Acad Sci U S A. 1991;88(22):10094–10098. doi: 10.1073/pnas.88.22.10094.  [DOI](https://doi.org/10.1073/pnas.88.22.10094) | [PMC free article](/articles/PMC52874/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1946428/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&title=Two%20putative%20subunits%20of%20a%20peptide%20pump%20encoded%20in%20the%20human%20major%20histocompatibility%20complex%20class%20II%20region&author=S%20Bahram&author=D%20Arnold&author=M%20Bresnahan&author=JL%20Strominger&author=T%20Spies&volume=88&issue=22&publication_year=1991&pages=10094-10098&pmid=1946428&doi=10.1073/pnas.88.22.10094&)

34. Perria CL, Rajamanickam V, Lapinski PE, Raghavan M. Catalytic site modifications of TAP1 and TAP2 and their functional consequences. J Biol Chem. 2006;281(52):39839–39851. doi: 10.1074/jbc.M605492200.  [DOI](https://doi.org/10.1074/jbc.M605492200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17068338/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Catalytic%20site%20modifications%20of%20TAP1%20and%20TAP2%20and%20their%20functional%20consequences&author=CL%20Perria&author=V%20Rajamanickam&author=PE%20Lapinski&author=M%20Raghavan&volume=281&issue=52&publication_year=2006&pages=39839-39851&pmid=17068338&doi=10.1074/jbc.M605492200&)

35. Bessho Y, Oguri T, Ozasa H, Uemura T, Sakamoto H, Miyazaki M, et al. ABCC10/MRP7 is associated with vinorelbine resistance in non-small cell lung cancer. Oncol Rep. 2009;21(1):263–268.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/19082471/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncol%20Rep&title=ABCC10/MRP7%20is%20associated%20with%20vinorelbine%20resistance%20in%20non-small%20cell%20lung%20cancer&author=Y%20Bessho&author=T%20Oguri&author=H%20Ozasa&author=T%20Uemura&author=H%20Sakamoto&volume=21&issue=1&publication_year=2009&pages=263-268&pmid=19082471&)

36. Hopper-Borge E, Chen ZS, Shchaveleva I, Belinsky MG, Kruh GD. Analysis of the drug resistance profile of multidrug resistance protein 7 (ABCC10): resistance to docetaxel. Cancer Res. 2004;64(14):4927–4930. doi: 10.1158/0008-5472.CAN-03-3111.  [DOI](https://doi.org/10.1158/0008-5472.CAN-03-3111) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15256465/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=Analysis%20of%20the%20drug%20resistance%20profile%20of%20multidrug%20resistance%20protein%207%20(ABCC10):%20resistance%20to%20docetaxel&author=E%20Hopper-Borge&author=ZS%20Chen&author=I%20Shchaveleva&author=MG%20Belinsky&author=GD%20Kruh&volume=64&issue=14&publication_year=2004&pages=4927-4930&pmid=15256465&doi=10.1158/0008-5472.CAN-03-3111&)

37. Boyle AP, Davis S, Shulha HP, Meltzer P, Margulies EH, Weng Z, et al. High-resolution mapping and characterization of open chromatin across the genome. Cell. 2008;132(2):311–322. doi: 10.1016/j.cell.2007.12.014.  [DOI](https://doi.org/10.1016/j.cell.2007.12.014) | [PMC free article](/articles/PMC2669738/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18243105/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell&title=High-resolution%20mapping%20and%20characterization%20of%20open%20chromatin%20across%20the%20genome&author=AP%20Boyle&author=S%20Davis&author=HP%20Shulha&author=P%20Meltzer&author=EH%20Margulies&volume=132&issue=2&publication_year=2008&pages=311-322&pmid=18243105&doi=10.1016/j.cell.2007.12.014&)

38. Sabo PJ, Humbert R, Hawrylycz M, Wallace JC, Dorschner MO, McArthur M, et al. Genomewide identification of DNaseI hypersensitive sites using active chromatin sequence libraries. Proc Natl Acad Sci U S A. 2004;101(13):4537–4542. doi: 10.1073/pnas.0400678101.  [DOI](https://doi.org/10.1073/pnas.0400678101) | [PMC free article](/articles/PMC384782/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15070753/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&title=Genomewide%20identification%20of%20DNaseI%20hypersensitive%20sites%20using%20active%20chromatin%20sequence%20libraries&author=PJ%20Sabo&author=R%20Humbert&author=M%20Hawrylycz&author=JC%20Wallace&author=MO%20Dorschner&volume=101&issue=13&publication_year=2004&pages=4537-4542&pmid=15070753&doi=10.1073/pnas.0400678101&)

39. Bernstein BE, Kamal M, Lindblad-Toh K, Bekiranov S, Bailey DK, Huebert DJ, et al. Genomic maps and comparative analysis of histone modifications in human and mouse. Cell. 2005;120(2):169–181. doi: 10.1016/j.cell.2005.01.001.  [DOI](https://doi.org/10.1016/j.cell.2005.01.001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15680324/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell&title=Genomic%20maps%20and%20comparative%20analysis%20of%20histone%20modifications%20in%20human%20and%20mouse&author=BE%20Bernstein&author=M%20Kamal&author=K%20Lindblad-Toh&author=S%20Bekiranov&author=DK%20Bailey&volume=120&issue=2&publication_year=2005&pages=169-181&pmid=15680324&doi=10.1016/j.cell.2005.01.001&)

40. Heintzman ND, Stuart RK, Hon G, Fu Y, Ching CW, Hawkins RD, et al. Distinct and predictive chromatin signatures of transcriptional promoters and enhancers in the human genome. Nat Genet. 2007;39(3):311–318. doi: 10.1038/ng1966.  [DOI](https://doi.org/10.1038/ng1966) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17277777/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Distinct%20and%20predictive%20chromatin%20signatures%20of%20transcriptional%20promoters%20and%20enhancers%20in%20the%20human%20genome&author=ND%20Heintzman&author=RK%20Stuart&author=G%20Hon&author=Y%20Fu&author=CW%20Ching&volume=39&issue=3&publication_year=2007&pages=311-318&pmid=17277777&doi=10.1038/ng1966&)

41. Yasuda SU, Zhang L, Huang SM. The role of ethnicity in variability in response to drugs: focus on clinical pharmacology studies. Clin Pharmacol Ther. 2008;84(3):417–423. doi: 10.1038/clpt.2008.141.  [DOI](https://doi.org/10.1038/clpt.2008.141) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18615002/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=The%20role%20of%20ethnicity%20in%20variability%20in%20response%20to%20drugs:%20focus%20on%20clinical%20pharmacology%20studies&author=SU%20Yasuda&author=L%20Zhang&author=SM%20Huang&volume=84&issue=3&publication_year=2008&pages=417-423&pmid=18615002&doi=10.1038/clpt.2008.141&)

42. Giacomini KM, Krauss RM, Roden DM, Eichelbaum M, Hayden MR, Nakamura Y. When good drugs go bad. Nature. 2007;446(7139):975–977. doi: 10.1038/446975a.  [DOI](https://doi.org/10.1038/446975a) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17460642/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=When%20good%20drugs%20go%20bad&author=KM%20Giacomini&author=RM%20Krauss&author=DM%20Roden&author=M%20Eichelbaum&author=MR%20Hayden&volume=446&issue=7139&publication_year=2007&pages=975-977&pmid=17460642&doi=10.1038/446975a&)

43. Gamazon ER, Huang RS, Cox NJ, Dolan ME. Chemotherapeutic drug susceptibility associated SNPs are enriched in expression quantitative trait loci. Proc Natl Acad Sci U S A. 2010;107(20):9287–9292. doi: 10.1073/pnas.1001827107.  [DOI](https://doi.org/10.1073/pnas.1001827107) | [PMC free article](/articles/PMC2889115/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20442332/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&title=Chemotherapeutic%20drug%20susceptibility%20associated%20SNPs%20are%20enriched%20in%20expression%20quantitative%20trait%20loci&author=ER%20Gamazon&author=RS%20Huang&author=NJ%20Cox&author=ME%20Dolan&volume=107&issue=20&publication_year=2010&pages=9287-9292&pmid=20442332&doi=10.1073/pnas.1001827107&)

44. Stark AL, Zhang W, Mi S, Duan S, O’Donnell PH, Huang RS, et al. Heritable and non-genetic factors as variables of pharmacologic phenotypes in lymphoblastoid cell lines. Pharmacogenomics J. 2010 doi: 10.1038/tpj.2010.3.  [DOI](https://doi.org/10.1038/tpj.2010.3) | [PMC free article](/articles/PMC2975793/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20142840/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Heritable%20and%20non-genetic%20factors%20as%20variables%20of%20pharmacologic%20phenotypes%20in%20lymphoblastoid%20cell%20lines&author=AL%20Stark&author=W%20Zhang&author=S%20Mi&author=S%20Duan&author=PH%20O%E2%80%99Donnell&publication_year=2010&pmid=20142840&doi=10.1038/tpj.2010.3&)

45. Pauli-Magnus C, Meier PJ, Stieger B. Genetic determinants of drug-induced cholestasis and intrahepatic cholestasis of pregnancy. Semin Liver Dis. 2010;30(2):147–159. doi: 10.1055/s-0030-1253224.  [DOI](https://doi.org/10.1055/s-0030-1253224) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20422497/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Semin%20Liver%20Dis&title=Genetic%20determinants%20of%20drug-induced%20cholestasis%20and%20intrahepatic%20cholestasis%20of%20pregnancy&author=C%20Pauli-Magnus&author=PJ%20Meier&author=B%20Stieger&volume=30&issue=2&publication_year=2010&pages=147-159&pmid=20422497&doi=10.1055/s-0030-1253224&)

46. Funk C, Ponelle C, Scheuermann G, Pantze M. Cholestatic potential of troglitazone as a possible factor contributing to troglitazone-induced hepatotoxicity: in vivo and in vitro interaction at the canalicular bile salt export pump (Bsep) in the rat. Mol Pharmacol. 2001;59(3):627–635.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/11179459/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Pharmacol&title=Cholestatic%20potential%20of%20troglitazone%20as%20a%20possible%20factor%20contributing%20to%20troglitazone-induced%20hepatotoxicity:%20in%20vivo%20and%20in%20vitro%20interaction%20at%20the%20canalicular%20bile%20salt%20export%20pump%20(Bsep)%20in%20the%20rat&author=C%20Funk&author=C%20Ponelle&author=G%20Scheuermann&author=M%20Pantze&volume=59&issue=3&publication_year=2001&pages=627-635&pmid=11179459&)

47. Vaughan AM, Oram JF. ABCG1 redistributes cell cholesterol to domains removable by high density lipoprotein but not by lipid-depleted apolipoproteins. J Biol Chem. 2005;280(34):30150–30157. doi: 10.1074/jbc.M505368200.  [DOI](https://doi.org/10.1074/jbc.M505368200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15994327/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=ABCG1%20redistributes%20cell%20cholesterol%20to%20domains%20removable%20by%20high%20density%20lipoprotein%20but%20not%20by%20lipid-depleted%20apolipoproteins&author=AM%20Vaughan&author=JF%20Oram&volume=280&issue=34&publication_year=2005&pages=30150-30157&pmid=15994327&doi=10.1074/jbc.M505368200&)

48. Japan Arteriosclerosis Longitudinal Study-Existing Cohorts Combine (JALS-ECC): rationale, design, and population characteristics. Circ J. 2008;72(10):1563–1568. doi: 10.1253/circj.cj-07-1049.  [DOI](https://doi.org/10.1253/circj.cj-07-1049) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18728337/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circ%20J&title=Japan%20Arteriosclerosis%20Longitudinal%20Study-Existing%20Cohorts%20Combine%20(JALS-ECC):%20rationale,%20design,%20and%20population%20characteristics&volume=72&issue=10&publication_year=2008&pages=1563-1568&pmid=18728337&doi=10.1253/circj.cj-07-1049&)

49. Lloyd-Jones D, Adams RJ, Brown TM, Carnethon M, Dai S, De Simone G, et al. Heart disease and stroke statistics--2010 update: a report from the American Heart Association. Circulation. 2010;121(7):e46–e215. doi: 10.1161/CIRCULATIONAHA.109.192667.  [DOI](https://doi.org/10.1161/CIRCULATIONAHA.109.192667) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20019324/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation&title=Heart%20disease%20and%20stroke%20statistics--2010%20update:%20a%20report%20from%20the%20American%20Heart%20Association&author=D%20Lloyd-Jones&author=RJ%20Adams&author=TM%20Brown&author=M%20Carnethon&author=S%20Dai&volume=121&issue=7&publication_year=2010&pages=e46-e215&pmid=20019324&doi=10.1161/CIRCULATIONAHA.109.192667&)

50. Spielman RS, Bastone LA, Burdick JT, Morley M, Ewens WJ, Cheung VG. Common genetic variants account for differences in gene expression among ethnic groups. Nat Genet. 2007;39(2):226–231. doi: 10.1038/ng1955.  [DOI](https://doi.org/10.1038/ng1955) | [PMC free article](/articles/PMC3005333/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17206142/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Common%20genetic%20variants%20account%20for%20differences%20in%20gene%20expression%20among%20ethnic%20groups&author=RS%20Spielman&author=LA%20Bastone&author=JT%20Burdick&author=M%20Morley&author=WJ%20Ewens&volume=39&issue=2&publication_year=2007&pages=226-231&pmid=17206142&doi=10.1038/ng1955&)

51. Zhang W, Duan S, Kistner EO, Bleibel WK, Huang RS, Clark TA, et al. Evaluation of genetic variation contributing to differences in gene expression between populations. Am J Hum Genet. 2008;82(3):631–640. doi: 10.1016/j.ajhg.2007.12.015.  [DOI](https://doi.org/10.1016/j.ajhg.2007.12.015) | [PMC free article](/articles/PMC2427223/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18313023/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Evaluation%20of%20genetic%20variation%20contributing%20to%20differences%20in%20gene%20expression%20between%20populations&author=W%20Zhang&author=S%20Duan&author=EO%20Kistner&author=WK%20Bleibel&author=RS%20Huang&volume=82&issue=3&publication_year=2008&pages=631-640&pmid=18313023&doi=10.1016/j.ajhg.2007.12.015&)

52. Storey JD, Madeoy J, Strout JL, Wurfel M, Ronald J, Akey JM. Gene-expression variation within and among human populations. Am J Hum Genet. 2007;80(3):502–509. doi: 10.1086/512017.  [DOI](https://doi.org/10.1086/512017) | [PMC free article](/articles/PMC1821107/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17273971/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Gene-expression%20variation%20within%20and%20among%20human%20populations&author=JD%20Storey&author=J%20Madeoy&author=JL%20Strout&author=M%20Wurfel&author=J%20Ronald&volume=80&issue=3&publication_year=2007&pages=502-509&pmid=17273971&doi=10.1086/512017&)

53. Excoffier L, Hamilton G. Comment on “Genetic structure of human populations”. Science. 2003;300(5627):1877. doi: 10.1126/science.1083411. author reply 1877.  [DOI](https://doi.org/10.1126/science.1083411) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12817127/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Comment%20on%20%E2%80%9CGenetic%20structure%20of%20human%20populations%E2%80%9D&author=L%20Excoffier&author=G%20Hamilton&volume=300&issue=5627&publication_year=2003&pages=1877&pmid=12817127&doi=10.1126/science.1083411&)

54. Rosenberg NA, Pritchard JK, Weber JL, Cann HM, Kidd KK, Zhivotovsky LA, et al. Genetic structure of human populations. Science. 2002;298(5602):2381–2385. doi: 10.1126/science.1078311.  [DOI](https://doi.org/10.1126/science.1078311) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12493913/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Genetic%20structure%20of%20human%20populations&author=NA%20Rosenberg&author=JK%20Pritchard&author=JL%20Weber&author=HM%20Cann&author=KK%20Kidd&volume=298&issue=5602&publication_year=2002&pages=2381-2385&pmid=12493913&doi=10.1126/science.1078311&)

55. Visel A, Blow MJ, Li Z, Zhang T, Akiyama JA, Holt A, et al. ChIP-seq accurately predicts tissue-specific activity of enhancers. Nature. 2009;457(7231):854–858. doi: 10.1038/nature07730.  [DOI](https://doi.org/10.1038/nature07730) | [PMC free article](/articles/PMC2745234/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19212405/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=ChIP-seq%20accurately%20predicts%20tissue-specific%20activity%20of%20enhancers&author=A%20Visel&author=MJ%20Blow&author=Z%20Li&author=T%20Zhang&author=JA%20Akiyama&volume=457&issue=7231&publication_year=2009&pages=854-858&pmid=19212405&doi=10.1038/nature07730&)

56. Cooper GM, Brown CD. Qualifying the relationship between sequence conservation and molecular function. Genome Res. 2008;18(2):201–205. doi: 10.1101/gr.7205808.  [DOI](https://doi.org/10.1101/gr.7205808) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18245453/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res&title=Qualifying%20the%20relationship%20between%20sequence%20conservation%20and%20molecular%20function&author=GM%20Cooper&author=CD%20Brown&volume=18&issue=2&publication_year=2008&pages=201-205&pmid=18245453&doi=10.1101/gr.7205808&)

57. McGaughey DM, Vinton RM, Huynh J, Al-Saif A, Beer MA, McCallion AS. Metrics of sequence constraint overlook regulatory sequences in an exhaustive analysis at phox2b. Genome Res. 2008;18(2):252–260. doi: 10.1101/gr.6929408.  [DOI](https://doi.org/10.1101/gr.6929408) | [PMC free article](/articles/PMC2203623/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18071029/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res&title=Metrics%20of%20sequence%20constraint%20overlook%20regulatory%20sequences%20in%20an%20exhaustive%20analysis%20at%20phox2b&author=DM%20McGaughey&author=RM%20Vinton&author=J%20Huynh&author=A%20Al-Saif&author=MA%20Beer&volume=18&issue=2&publication_year=2008&pages=252-260&pmid=18071029&doi=10.1101/gr.6929408&)

58. Kim MJ, Skewes-Cox P, Fukushima H, Hesselson S, Nguyen L, Eshragh JL, et al. In Vivo Pharmacogenomic Characterization of Enhancers of Liver Membrane Transporters. 2010. In Manuscript.
