# The Cyclin D1 (CCND1) A870G polymorphism predicts clinical outcome to lapatinib and capecitabine in HER2-positive metastatic breast cancer

## Metadata
**Authors:** M J LaBonte, P M Wilson, D Yang, W Zhang, R D Ladner, Y Ning, A Gerger, P O Bohanes, L Benhaim, R El-khoueiry, A El-khoueiry, H-J Lenz
**Journal:** Annals of Oncology
**Date:** 2011 Oct 11
**DOI:** [10.1093/annonc/mdr445](https://doi.org/10.1093/annonc/mdr445)
**PMID:** 21989330
**PMCID:** PMC3360546
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3360546/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC3360546/pdf/mdr445.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC3360546/pdf/mdr445.pdf)

## Abstract

**Background:** 
Lapatinib plus capecitabine emerged as an efficacious therapy in metastatic breast cancer (mBC). We aimed to identify germline single-nucleotide polymorphisms (SNPs) in genes involved in capecitabine catabolism and human epidermal receptor signaling that were associated with clinical outcome to assist in selecting patients likely to benefit from this combination.

**Patients and methods:** 
DNA was extracted from 240 of 399 patients enrolled in EGF100151 clinical trial (NCT00078572; clinicaltrials.gov) and SNPs were successfully evaluated in 234 patients. The associations between SNPs and clinical outcome were analyzed using Fisher’s exact test, Kaplan–Meier curves, log-rank tests, likelihood ratio test within logistic or Cox regression model, as appropriate.

**Results:** 
There were significant interactions between CCND1 A870G and clinical outcome. Patients carrying the A-allele were more likely to benefit from lapatinib plus capecitabine versus capecitabine when compared with patients harboring G/G (P = 0.022, 0.024 and 0.04, respectively). In patients with the A-allele, the response rate (RR) was significantly higher with lapatinib plus capecitabine (35%) compared with capecitabine (11%; P = 0.001) but not between treatments in patients with G/G (RR = 24% and 32%, respectively; P = 0.85). Time to tumor progression (TTP) was longer in patients with the A-allele treated with lapatinib plus capecitabine compared with capecitabine (median TTP = 7.9 and 3.4 months; P < 0.001), but not in patients with G/G (median TTP = 6.1 and 6.6 months; P = 0.92).

**Conclusion:** 
Our findings suggest that CCND1A870G may be useful in predicting clinical outcome in HER2-positive mBC patients treated with lapatinib plus capecitabine.

Keywords: capecitabine, cyclin D1, lapatinib, metastatic breast cancer, polymorphisms

### Background:

Lapatinib plus capecitabine emerged as an efficacious therapy in metastatic breast cancer (mBC). We aimed to identify germline single-nucleotide polymorphisms (SNPs) in genes involved in capecitabine catabolism and human epidermal receptor signaling that were associated with clinical outcome to assist in selecting patients likely to benefit from this combination.

### Patients and methods:

DNA was extracted from 240 of 399 patients enrolled in EGF100151 clinical trial ([NCT00078572](https://clinicaltrials.gov/ct2/show/NCT00078572); clinicaltrials.gov) and SNPs were successfully evaluated in 234 patients. The associations between SNPs and clinical outcome were analyzed using Fisher’s exact test, Kaplan–Meier curves, log-rank tests, likelihood ratio test within logistic or Cox regression model, as appropriate.

### Results:

There were significant interactions between *CCND1* A870G and clinical outcome. Patients carrying the A-allele were more likely to benefit from lapatinib plus capecitabine versus capecitabine when compared with patients harboring G/G (*P* = 0.022, 0.024 and 0.04, respectively). In patients with the A-allele, the response rate (RR) was significantly higher with lapatinib plus capecitabine (35%) compared with capecitabine (11%; *P* = 0.001) but not between treatments in patients with G/G (RR = 24% and 32%, respectively; *P* = 0.85). Time to tumor progression (TTP) was longer in patients with the A-allele treated with lapatinib plus capecitabine compared with capecitabine (median TTP = 7.9 and 3.4 months; *P* < 0.001), but not in patients with G/G (median TTP = 6.1 and 6.6 months; *P* = 0.92).

### Conclusion:

Our findings suggest that *CCND1*A870G may be useful in predicting clinical outcome in HER2-positive mBC patients treated with lapatinib plus capecitabine.

## introduction

Approximately 1 million new cases of breast cancer (BC) are reported each year worldwide and it remains the leading cause of cancer-related deaths in women [[1](#bib1), [2](#bib2)]. In the United States, the American Cancer Society estimates that in 2010 there will be ∼ 261 100 new cases of BC with 39 840 deaths [[3](#bib3)]. Approximately 15%–30% of BC cases demonstrate human epidermal receptor 2 (HER2) gene amplification and protein overexpression that is associated with aggressive disease, increased resistance to some chemotherapeutic agents and poor clinical outcome [[4](#bib4)–[7](#bib7)].

Lapatinib (Tykerb®/Tyverb®; GlaxoSmithKline, Research Triangle Park, NC) is an orally available, reversible small-molecule tyrosine kinase inhibitor that blocks both epidermal growth factor receptor (EGFR) and HER2 and their downstream signaling pathways [[8](#bib8), [9](#bib9)]. The Food and Drug Administration and the European Medicines Agency have both approved lapatinib for use in combination with the oral fluoropyrimidine, capecitabine (Xeloda; Roche, San Francicso, CA), for HER2-overexpressing metastatic breast cancer (mBC) patients who had previously failed an anthracycline, taxane and trastuzumab and in combination with letrozole (Femara; Novartis Pharmaceuticals, East Hanover, NJ) for hormone-positive, HER2-overexpressing postmenopausal women with mBC [[10](#bib10), [11](#bib11)]. The phase III trial evaluating the combination of lapatinib and capecitabine chemotherapy reported that the addition of lapatinib prolonged time to tumor progression (TTP) with a median of 6.2 versus 4.3 months and a hazard ratio (HR) of 0.57 [95% confidence interval (CI) 0.43–0.77. *P* < 0.001] and provided a trend toward improved overall survival (OS) (HR = 0.78, 95% CI 0.55–1.12, *P* = 0.177) [[11](#bib11), [12](#bib12)]. Lapatinib plus capecitabine resulted in a response rate (RR) of 23.7% compared with capecitabine alone at 13.9% (*P* = .017) [[11](#bib11)]. Despite a clear improvement in clinical benefit associated with the lapatinib and capecitabine combination versus capecitabine monotherapy (27% versus 18%), this clinical benefit remains limited to a subset of patients. As a result, there is a clear need to identify predictive markers in addition to HER2 positivity with the power to identify patients with a high likelihood of clinical benefit as candidates for lapatinib and capecitabine combination therapy [[13](#bib13)].

It is well established that heterogeneity within the patient population is a strong contributing factor to the observed interindividual variation in response to chemotherapeutic agents and subsequent clinical outcome. As such, the identification and validation of contributing genetic factors such as single-nucleotide polymorphisms (SNPs) represents a critical step in the advance toward personalized medicine. The purpose of the current study was to identify subgroups of HER2-positive mBC patients who may benefit from the addition of lapatinib to capecitabine chemotherapy.

Several studies have attempted to identify biomarkers that identify the patient population with a high likelihood of response to lapatinib treatment [[13](#bib13)–[15](#bib15)]. These studies have evaluated HER2 gene amplification, HER2 and EGFR messenger RNA (mRNA) and protein expression, serum transforming growth factor-α (TGF-α), epidermal growth factor (EGF), EGFR-extracellular domain (ECD) and HER2-ECD. Of these, only high serum TGF-α has been implicated in resistance to lapatinib and capecitabine treatment. While other studies have focused on investigating the influence of gene expression levels, this study hypothesized that polymorphisms within genes involved in both the capecitabine pathway (*TYMS* and *MTHFR*) and HER signaling cascade (*EGF*, EGFR, HER2, *CCND1*, *IL-8* *VEGF*) may predict RR, clinical benefit and/or TTP for patients treated with lapatinib plus capecitabine compared with capecitabine monotherapy.

## patients and methods

### study population and EGF100151 trial design

All patients included in this study participated in the EGF100151 clinical trial ([NCT00078572](https://clinicaltrials.gov/ct2/show/NCT00078572); clinicaltrials.gov), a phase III trial of 399 HER2-positive mBC patients who had been previously treated with an anthracycline, a taxane and trastuzumab. Patients were randomly assigned to receive capecitabine (2500 mg/m^2^/day, days 1–14, for 3 weeks) or capecitabine (2000 mg/m^2^/day, days 1–14, for 3 weeks) plus lapatinib (1250 mg/day) [[11](#bib11), [12](#bib12)]. The primary end point of the study was TTP and OS. The present analysis was conducted at the University of Southern California (USC)/Norris Comprehensive Cancer Center following approval by the USC Institutional Review Board for Medical Sciences. All patients provided written informed consent for tissue and blood collection to allow study of molecular correlates.

### genotyping and candidate polymorphisms

Genomic DNA was extracted from whole blood samples collected on study. The majority of SNPs were tested by the PCR–restriction fragment length polymorphism (PCR–RFLP) method as previously described [[16](#bib16)] and genotype agreed upon by the consensus of two independent investigators. The EGFR (CA)_n_ repeat polymorphism was tested by a 5′-end [γ-^33^P] ATP-labeled PCR protocol [[17](#bib17)–[19](#bib19)]. To ensure accuracy and specificity, a total of 10% positive and negative duplicate controls were matched for each polymorphism and were analyzed by direct DNA sequencing. Genotype concordance in these selected quality control samples was 100%. All SNP analyses were carried out blinded to the clinical data.

### selection of SNPs

The genes, reference SNP identification numbers, location, function, forward and reverse primer and restriction enzymes are summarized in [supplemental Table S1](http://www.annonc.oxfordjournals.org/lookup/suppl/doi:10.1093/annonc/mdr445/-/DC1) (available at *Annals of Oncology* online). The genes analyzed in this study were selected based upon stringent predefined criteria: (i) the gene was part of a pathway for which there was credible scientific basis to support its involvement in either capecitabine metabolism or HER activation and signaling; (ii) the gene has an established well-documented genetic polymorphism; and/or (iii) the SNP has potential to alter the function of the gene in a biologically relevant manner.

### statistical analysis

Allelic distribution of all SNPs was tested for deviation from Hardy–Weinberg equilibrium (HWE), and the fit to the equilibrium was evaluated utilizing the chi-square test with 1 *df*. The distribution of polymorphisms across baseline demographic, clinical and pathological characteristics was examined using Fisher’s exact test. The end points of the study included tumor RR, TTP and clinical benefit. The definition of the end points can be found in the article of the original trial [[11](#bib11)]. The associations between SNPs and TTP, RR and clinical benefit were examined using Kaplan–Meier curves, log-rank test, and Fisher’s exact test. The inheritance model for SNPs and clinical outcome was not established. A codominant model or dominant model was utilized whenever appropriate. We calculated that a total of 231 patients would be required to detect a significant interaction between treatment and an SNP on TTP with 80% power using a 0.05-significance level two-sided test ([http://www.swogstat.org/stat/public/int_survival.htm](http://www.swogstat.org/stat/public/int_survival.htm)). We assumed that the variant allele frequency was 50%, the variant allele carriers benefited from the combination treatment (HR = 0.4) and the patients carrying only the wild-type allele did not benefit from the combination treatment (HR = 1.0).

The interactions between treatment and SNPs on end points were tested using likelihood ratio test within logistic regression or Cox proportional hazards model for RR and clinical benefit and TTP, respectively. The classification and regression trees based on binary recursive partitioning (RP) were a multivariate used to predict end points by selecting predictors from treatment, SNPs and baseline patient characteristics [[20](#bib20), [21](#bib21)].

All statistical tests were two-sided and carried out using SAS statistical package version 9.2 (SAS Institute Inc., Cary, NC) and rPART for RP.

## results

### study population

Of the 399 patients enrolled in the EGF100151 trial, 240 patients (60%) had blood that was available for retrospective SNP analysis. Of these 240 blood samples that were available, DNA was successfully extracted and genotyping was successfully carried out in 234 patients. Of these 234 patients, 125 received capecitabine monotherapy and 109 received the lapatinib plus capecitabine combination. The clinical outcome of these 234 patients was representative of the entire study population. Specifically, the overall RR was not statistically different between the overall and the subset patient populations with 14% (95% CI 9% to 21%) in the overall patient population and 17% (95% CI 10% to 27%) in the subset of patients treated with capecitabine alone and 22% (95% CI 16% to 29%) in the overall patient population versus 33% (95% CI 23% to 43%) in the subset of patients treated with lapatinib plus capecitabine ([Table 1](#tbl1)). Furthermore, for TTP there was no statistical difference between the overall and the subset patient populations with 4.4 months in the overall patient population and 4.0 months in the subset of patients treated with capecitabine alone and 8.4 months (HR = 0.49; 95% CI 0.34–0.71) in the overall patient population versus 7.6 months (HR = 0.51; 95% CI 0.34–0.78) in the subset of patients treated with lapatinib plus capecitabine ([Table 1](#tbl1)). The baseline characteristics of these patients in both treatment arms were also comparable with the total study population ([Table 1](#tbl1)).

### Table 1.

|   | Subset population |  | Overall phase III Triala |  |
| --- | --- | --- | --- | --- |
| Characteristic | Capecitabine 2500 mg/m2, (n = 125) | Lapatinib 1250 mg plus capecitabine 2000 mg/m2 (n = 109) | Capecitabine 2500 mg/m2 (n = 161) | Lapatinib 1250 mg plus capecitabine 2000 mg/m2 (n = 163) |
| Age, year |   |   |   |   |
| Median | 50 | 53 | 51 | 54 |
| Range | 28–80 | 26–80 | 28–83 | 26–80 |
| ECOG performance status, n (%) |   |   |   |   |
| 0 | 85 (68) | 72 (66) | 89 (57) | 96 (61) |
| 1 | 40 (32) | 37 (34) | 68 (43) | 61 (39) |
| Hormone receptor status, n (%) |   |   |   |   |
| ER+ or PR+ | 60 (48%) | 47 (43%) | 75 (47%) | 78 (48%) |
| ER− and PR− | 58 (46) | 56 (51) | 80 (50) | 80 (49) |
| Unknown | 7 (6) | 6 (6) | 6 (4) | 5 (3) |
| Stage of disease, n (%) |   |   |   |   |
| IIIB or IIIC | 6 (5) | 4 (4) | 7 (4) | 7 (4) |
| Metastatic | 119 (95) | 105 (96) | 154 (96) | 156 (96) |
| N of advanced or metastatic sites, n (%) |   |   |   |   |
| <3 | 74 (59) | 59 (54) | 81 (50) | 84 (52) |
| ≥3 | 51 (41) | 50 (46) | 80 (50) | 79 (48) |
| Advanced or metastatic sites, n (%) |   |   |   |   |
| Visceral | 96 (77) | 78 (72) | 124 (77) | 125 (77) |
| Non-visceral only | 29 (23) | 31 (28) | 37 (23) | 38 (23) |
| Duration of trastuzumab therapy, week |   |   |   |   |
| Median | 47 | 49 | 44 | 42 |
| Range | 0–329 | 3–296 | 5–329 | 3–296 |
| Median overall response,% (95% CI) | 17 (10–27) | 33 (23–43) | 14 (9–21) | 22 (16–29) |
| Median time to tumor progression, months | 4.0 | 7.6 | 4.4 | 8.4 |
| HR (95% CI) |   | 0.51 (0.34–0.76) |   | 0.49 (0.34–0.71) |
| P-valueb |   | <0.001 |   | <0.001 |

Table 1 Caption: Baseline characteristics of the 234 women included in the biomarker analysis of EGF100151 clinical trial

### overall distribution of genotypes in BC patients

The genotype frequencies of the polymorphic variants of *CCND1*, *EGF*, EGFR, HER2, *IL-8*, *MTHFR*, *TYMS* and *VEGF* did not deviate significantly from the predicted distribution of HWE in either treatment group.

### univariate analysis of polymorphisms associated with tumor RR

Of the 10 SNPs analyzed in this study, *CCND1*A870G (rs17852153) was the only polymorphism significantly associated with clinical outcome ([Tables 2](#tbl2a) and [3](#tbl3a); [supplemental Table S2](http://www.annonc.oxfordjournals.org/lookup/suppl/doi:10.1093/annonc/mdr445/-/DC1), available at *Annals of Oncology* online). Out of 233 patients assessable for *CCND1* A870G analysis, 63 patients (27%) possessed the A/A genotype and 63 possessed the G/G genotype (27%), whereas 107 patients (46%) were heterozygous (A/G). When considering the RR in the patient population, there was a near statistically significant trend toward an interaction between the type of chemotherapy received and the *CCND1* A870G SNP ([Table 2](#tbl2a), *P* = 0.059). When patients possessing the A-allele were grouped together (A/A and A/G, *n* = 170; G/G, *n* = 63), patients with the A-allele had an RR of only 10.5% with capecitabine monotherapy (*n* = 83). However, when patients possessing the A-allele received capecitabine plus lapatinib (*n* = 87), the RR was significantly increased to 36.5% (*P* for interaction = 0.022). In contrast, patients with the G-allele did not demonstrate any statistically significant difference in RR between the capecitabine monotherapy and the combination treatment groups with RR of 32% and 24%, respectively ([Figure 1A](#fig1)).

### Table 2.

|   | Capecitabine 2500 mg/m2 |  |  |  |  |  | Lapatinib 1250 mg plus capecitabine 2000 mg/m2 |  |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Polymorphism | N | CR + PR, n (%) | SD, n (%) | PD, n (%) | P-valuea |  | N | CR + PR, n (%) | SD, n (%) | PD, n (%) | P-valuea |
| IL-8-251T>A |   |   |   |   | 0.32 |   |   |   |   |   | 1.00 |
| T/T | 42 | 6 (21) | 15 (54) | 7 (25) |   |   | 23 | 4 (25) | 9 (56) | 3 (19) |   |
| A/T | 56 | 6 (17) | 15 (42) | 15 (42) |   |   | 60 | 19 (37) | 26 (51) | 6 (12) |   |
| A/A | 27 | 3 (14) | 11 (50) | 8 (36) |   |   | 26 | 6 (27) | 12 (55) | 4 (18) |   |
| P for interaction |   |   |   |   |   | 0.27 |   |   |   |   |   |
| VEGF +936 C>T |   |   |   |   | 10.48 |   |   |   |   |   | 0.47 |
| C/C | 91 | 13 (20) | 30 (46) | 22 (34) |   |   | 84 | 24 (36) | 32 (48) | 10 (15) |   |
| C/T | 34 | 2 (10) | 11 (52) | 8 (38) |   |   | 25 | 5 (22) | 15 (65) | 3 (13) |   |
| P for interaction |   |   |   |   |   | 1.00 |   |   |   |   |   |
| TS 3′-UTR (+6 bp/−6 bp) |   |   |   |   | 0.65 |   |   |   |   |   | 0.26 |
| +/+ | 59 | 8 (19) | 18 (42) | 17 (40) |   |   | 49 | 10 (27) | 19 (51) | 8 (22) |   |
| +/− | 52 | 6 (18) | 17 (50) | 11 (32) |   |   | 45 | 15 (38) | 20 (51) | 4 (10) |   |
| −/− | 14 | 1 (11) | 6 (67) | 2 (22) |   |   | 15 | 4 (31) | 8 (62) | 1 (8) |   |
| P for interaction |   |   |   |   |   | 0.79 |   |   |   |   |   |
| TS 5′-UTR |   |   |   |   | 0.20 |   |   |   |   |   | 0.89 |
| 2R/2R, 2R/3C, 3C/3C | 80 | 10 (18) | 21 (38) | 25 (45) |   |   | 67 | 18 (34) | 25 (47) | 10 (19) |   |
| 2R/3G, 3G/3C | 37 | 5 (20) | 16 (64) | 4 (16) |   |   | 37 | 10 (32) | 19 (61) | 2 (6) |   |
| 3G/G | 7 | 0 (0) | 4 (80) | 1 (20) |   |   | 4 | 0 (0) | 3 (75) | 1 (25) |   |
| P for interaction |   |   |   |   |   | 0.38 |   |   |   |   |   |
| MTHFR +677 C>T |   |   |   |   | 1.00 |   |   |   |   |   | 1.00 |
| C/C | 50 | 6 (19) | 14 (45) | 11 (35) |   |   | 48 | 14 (35) | 20 (50) | 6 (15) |   |
| C/T | 54 | 5 (13) | 21 (54) | 13 (33) |   |   | 53 | 12 (29) | 23 (56) | 6 (15) |   |
| T/T | 20 | 4 (27) | 5 (33) | 6 (40) |   |   | 8 | 3 (38) | 4 (50) | 1 (13) |   |
| P for interaction |   |   |   |   |   | 0.98 |   |   |   |   |   |
| MTHFR +1298 A>C |   |   |   |   | 0.71 |   |   |   |   |   | 0.42 |
| A/A | 67 | 9 (18) | 25 (51) | 15 (31) |   |   | 57 | 14 (30) | 25 (54) | 7 (15) |   |
| A/C | 34 | 3 (17) | 7 (39) | 8 (44) |   |   | 31 | 7 (27) | 15 (58) | 4 (15) |   |
| C/C | 23 | 3 (17) | 9 (50) | 6 (33) |   |   | 21 | 8 (47) | 7 (41) | 2 (12) |   |
| P for interaction |   |   |   |   |   | 0.61 |   |   |   |   |   |
| EGF +61 A>G |   |   |   |   | 0.34 |   |   |   |   |   | 0.46 |
| A/A | 39 | 7 (28) | 11 (44) | 7 (28) |   |   | 34 | 9 (32) | 16 (57) | 3 (11) |   |
| A/G | 71 | 7 (14) | 22 (45) | 20 (41) |   |   | 60 | 19 (39) | 21 (43) | 9 (18) |   |
| G/G | 15 | 1 (8) | 8 (67) | 3 (25) |   |   | 15 | 1 (8) | 10 (83) | 1 (8) |   |
| P for interaction |   |   |   |   |   | 0.35 |   |   |   |   |   |
| EGFR +497 G>A |   |   |   |   | 1.00 |   |   |   |   |   | 0.21 |
| G/G | 71 | 8 (17) | 23 (48) | 17 (35) |   |   | 58 | 17 (37) | 24 (52) | 5 (11) |   |
| G/A | 44 | 7 (21) | 15 (45) | 11 (33) |   |   | 38 | 9 (28) | 18 (56) | 5 (16) |   |
| A/A | 10 | 0 (0) | 3 (60) | 2 (40) |   |   | 12 | 3 (27) | 5 (45) | 3 (27) |   |
| P for interaction |   |   |   |   |   | 0.67 |   |   |   |   |   |
| EGFR (CA)n |   |   |   |   | 0.28 |   |   |   |   |   | 0.34 |
| <20 | 60 | 5 (12) | 20 (49) | 16 (39) |   |   | 59 | 19 (39) | 23 (47) | 7 (14) |   |
| ≥20 | 65 | 10 (22) | 21 (47) | 14 (31) |   |   | 48 | 10 (26) | 23 (59) | 6 (15) |   |
| P for interaction |   |   |   |   |   | 0.12 |   |   |   |   |   |
| CCND1 +870 A>G |   |   |   |   | 0.16 |   |   |   |   |   | 0.10 |
| A/A | 30 | 2 (10) | 11 (52) | 8 (38) |   |   | 33 | 11 (41) | 13 (48) | 3 (11) |   |
| A/G | 57 | 4 (11) | 19 (53) | 13 (36) |   |   | 50 | 13 (32) | 24 (59) | 4 (10) |   |
| G/G | 37 | 9 (32) | 10 (36) | 9 (32) |   |   | 26 | 5 (24) | 10 (48) | 6 (29) |   |
| P for interaction |   |   |   |   |   | 0.059 |   |   |   |   |   |
| HER2 655 A>C |   |   |   |   | 0.64 |   |   |   |   |   | 0.63 |
| A/A | 75 | 9 (17) | 24 (45) | 20 (38) |   |   | 67 | 17 (32) | 27 (51) | 9 (17) |   |
| A/C | 50 | 6 (18) | 17 (52) | 10 (30) |   |   | 42 | 12 (33) | 20 (56) | 4 (11) |   |
| P for interaction |   |   |   |   |   | 0.89 |   |   |   |   |   |

Table 2 Caption: Univariate analysis of tumor response by treatment group and polymorphisms

### Table 3.

|   | Capecitabine 2500 mg/m2a |  |  |  | Lapatinib 1250 mg plus capecitabine 2000 mg/m2a |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- |
|   | Median TTP, months (95% CI) | Hazard ratio (95% CI) | P-valueb |  | Median TTP, months (95% CI) | Hazard ratio (95% CI) | P-valueb |
| IL-8-251T>A |   |   | 0.35 |   |   |   | 0.33 |
| T/T | 5.9 (3.1–8.3) | 1 (reference) |   |   | 17.1+ (7.9–17.1+) | 1 (reference) |   |
| A/T | 3.2 (2.8–4.0) | 1.34 (0.73––2.48) |   |   | 6.1 (5.1–11.5) | 1.91 (0.73–5.05) |   |
| A/A | 4.5 (3.9–11.2) | 0.88 (0.42––1.982 |   |   | 5.6 (4.5–10.8) | 2.07 (0.72–5.96) |   |
| P for interaction |   |   |   | 0.44 |   |   |   |
| VEGF +936 C>T |   |   | 0.81 |   |   |   | 0.12 |
| C/C | 4.1 (3.2–8.3) | 1 (reference) |   |   | 8.5 (5.6–19.7+) | 1 (reference) |   |
| C/T | 4.0 (2.7–6.6) | 1.07 (0.60–1.90) |   |   | 5.8 (4.2–10.8) | 1.67 (0.86–3.27) |   |
| P for interaction |   |   |   | 0.30 |   |   |   |
| TS 3′-UTR (+6bp/−6bp) |   |   | 0.14 |   |   |   | 0.81 |
| +/+ | 4.3 (2.7–6.6) | 1 (reference) |   |   | 10.8 (5.8–12.3+) | 1 (reference) |   |
| +/− | 4.5 (3.3–8.7) | 0.81 (0.46–1.44) |   |   | 5.8 (4.4–11.5) | 1.25 (0.62–2.50) |   |
| −/− | 3.2 (1.4–4.3) | 1.77 (0.79–3.96) |   |   | 5.6 (5.1–9.4) | 1.17 (0.47–2.90) |   |
| P for interaction |   |   |   | 0.33 |   |   |   |
| TS 5′-UTR |   |   | 0.11 |   |   |   | 0.67 |
| 2R/2R, 2R/3C, 3C/3C | 3.4 (2.8–4.5) | 1 (reference) |   |   | 7.9 (5.6–11.5) | 1 (reference) |   |
| 2R/3G, 3G/3C | 7.1 (3.6–11.2) | 0.54 (0.29–0.99) |   |   | 5.6 (4.5–17.6c) | 0.91 (0.45–1.87) |   |
| 3G/G | 4.1 (1.4–5.5c) | 1.04 (0.37–2.94) |   |   | 5.1 (4.2–9.4c) | 1.62 (0.49–5.41) |   |
| P for interaction |   |   |   | 0.60 |   |   |   |
| MTHFR +677 C>T |   |   | 0.90 |   |   |   | 0.61 |
| C/C | 4.0 (3.1–8.4) | 1 (reference) |   |   | 5.6 (4.4–17.6c) | 1 (reference) |   |
| C/T | 3.9 (2.8–6.6) | 1.09 (0.61–1.94) |   |   | 8.5 (5.8–19.7c) | 0.80 (0.41–1.56) |   |
| T/T | 5.9 (2.4–8.3) | 0.92 (0.43–1.97) |   |   | 5.6 (4.3–8.4c) | 1.28 (0.43–3.84) |   |
| P for interaction |   |   |   | 0.62 |   |   |   |
| MTHFR +1298 A>C |   |   | 0.64 |   |   |   | 0.43 |
| A/A | 4.5 (3.3–8.4) | 1 (reference) |   |   | 7.6 (5.8–19.7c) | 1 (reference) |   |
| A/C | 3.6 (2.6–7.1) | 1.35 (0.71–2.56) |   |   | 8.5 (4.0–9.4) | 1.58 (0.77–3.25) |   |
| C/C | 4.0 (2.7–8.4) | 1.15 (0.59–2.25) |   |   | 5.6 (5.4–17.1c) | 1.24 (0.56–2.77) |   |
| P for interaction |   |   |   | 0.94 |   |   |   |
| EGF +61 A>G |   |   | 0.24 |   |   |   | 0.96 |
| A/A | 7.1 (4.3–8.7) | 1 (reference) |   |   | 6.2 (4.5–17.1c) | 1 (reference) |   |
| A/G | 3.3 (2.8–4.1) | 1.57 (0.87–2.83) |   |   | 7.9 (5.6–11.5) | 0.92 (0.46–1.86) |   |
| G/G | 8.7+ (2.6–8.7c) | 1.11 (0.40–3.03) |   |   | 5.8 (4.3–19.7c) | 1.00 (0.37–2.68) |   |
| P for interaction |   |   |   | 0.46 |   |   |   |
| EGFR +497 G>A |   |   | 0.72 |   |   |   | 0.89 |
| G/G | 4.0 (3.1–5.9) | 1 (reference) |   |   | 7.6 (5.8–11.5) | 1 (reference) |   |
| G/A | 4.3 (2.8–9.4) | 0.80 (0.45–1.40) |   |   | 5.6 (5.1–19.7c) | 1.16 (0.58–2.31) |   |
| A/A | 3.3 (2.8–4.3c) | 0.95 (0.29–3.12) |   |   | 5.6 (3.3–17.1c) | 1.14 (0.45–2.84) |   |
| P for interaction |   |   |   | 0.63 |   |   |   |
| EGFR (CA)n |   |   | 0.16 |   |   |   | 0.38 |
| Both (CA)n <20 | 3.4 (2.8–4.5) | 1 (reference) |   |   | 7.6 (5.4–17.6c) | 1 (reference) |   |
| Any (CA)n ≥20 | 4.3 (3.3–8.4) | 0.70 (0.41–1.18) |   |   | 6.1 (5.6–9.4) | 1.31 (0.70–2.47) |   |
| P for interaction |   |   |   | 0.11 |   |   |   |
| CCND1 +870 A>G |   |   | 0.46 |   |   |   | 0.073 |
| A/A | 3.1 (1.9–5.6c) | 1 (reference) |   |   | 5.6 (4.4–10.9) | 1 (reference) |   |
| A/G | 4.0 (3.2–5.9) | 0.88 (0.43–1.81) |   |   | 8.5 (5.8–19.7c) | 0.58 (0.28–1.20) |   |
| G/G | 6.6 (3.3–8.4) | 0.67 (0.30–1.46) |   |   | 6.1 (2.7–9.4c) | 1.33 (0.60–2.96) |   |
| P for interaction |   |   |   | 0.045 |   |   |   |
| HER2 655 A>G |   |   | 0.40 |   |   |   | 0.87 |
| A/A | 4.1 (3.1–6.6) | 1 (reference) |   |   | 6.1 (5.4–19.7c) | 1 (reference) |   |
| A/G | 4.0 (3.1–8.4) | 0.80 (0.47–1.38) |   |   | 8.5 (5.6–10.9) | 0.95 (0.50–1.79) |   |
| P for interaction |   |   |   | 0.61 |   |   |   |

Table 3 Caption: Time to tumor progression (TTP) by treatment groups and polymorphisms

### Figure 1.

![Figure 1.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3836/3360546/f71141c06577/annoncmdr445f01_ht.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3360546_annoncmdr445f01_ht.jpg)

Interactions between CCND1 A870G polymorphism and treatment on response rate (RR), clinical benefit and time to tumor progression (TTP). The interactions between CCND1 A870G and treatment were analyzed by likelihood ratio test. There was a statistically significant interaction between CCND1 A870G and treatment in relation to (A) RR (P = 0.022) (B) clinical benefit, as defined as complete response, partial response or stable disease for at least 6 months (P = 0.040) and (C) TTP (P = 0.025), with those patients carrying the A-allele showing increased RR, clinical benefit and TTP with the addition of lapatinib compared with those patients who were homozygous for the G/G genotype.

### univariate analysis of polymorphisms associated with clinical benefit

The *CCND1* A870G was the only statistically significant SNP in the patient population with regard to clinical benefit that encompassed both tumor response and stable disease rate as defined in the EGF000151 protocol [[11](#bib11)]. Specifically, patients harboring the A-allele had a clinical benefit rate of only 9.5% with capecitabine monotherapy (*n* = 83); however, when patients received capecitabine plus lapatinib (*n* = 87), the clinical benefit rate was significantly higher at 30% ([supplemental Table 2](http://www.annonc.oxfordjournals.org/lookup/suppl/doi:10.1093/annonc/mdr445/-/DC1), available at *Annals of Oncology* online; *P* for interaction = 0.04). As observed previously with RR, patients with the G-allele did not demonstrate any statistically significant difference in the clinical benefit rate between the capecitabine monotherapy and the combination treatment groups with clinical benefit rates of 31.5% and 27%, respectively ([Figure 1B](#fig1)).

### univariate analysis of polymorphisms associated with TTP

The importance of the *CCND1* A870G SNP was further demonstrated during analysis of TTP where a statistically significant interaction between the type of chemotherapy received and the *CCND1* A870G SNP was observed ([Table 3](#tbl3a); *P* = 0.045). For patients with any A-allele, the median TTP in the capecitabine monotherapy group was 4.0 months (range: 3.1–4.8 months). The same genotypes in the lapatinib plus capecitabine combination arm had a significantly higher median TTP of 7.9 months (range: 5.6–11.5 months, *P* for interaction = 0.025). Patients possessing the G/G genotype, however, demonstrated a median TTP of 6.6 months (range: 3.3–8.4 months) in the capecitabine monotherapy group which did not differ in terms of statistical significance from the combination treatment groups that demonstrated a median TTP of 6.1 months and a similar range (range: 2.7–9.4 months) ([Figure 1C](#fig1)). The potential for any association between the duration of previous trastuzumab treatment and/or previous lines of chemotherapy and the influence of the *CCND1* A870G on TTP was analyzed. Neither the duration of previous trastuzumab treatment nor the number of lines of previous chemotherapy were significantly associated with the *CCND1* SNP and TTP (data not shown).

### multivariate RP

RP was utilized to construct a decision tree as a predictive model to classify patients based on the presence of these molecular markers and identify which patient subgroups benefited from the addition of lapatinib to capecitabine chemotherapy. This comprehensive RP analysis incorporated a total of 18 potential variables including clinicopathological data listed in [Table 1](#tbl1) and the panel of SNPs evaluated in this patient cohort. In the resultant decision tree, the most important factor that determined the TTP in these patients was their treatment assignment of capecitabine alone versus lapatinib plus capecitabine.

Within the patients who received lapatinib plus capecitabine, the important factors that determined patient outcome were the status of the *CCND1* A870G and *MTHFR* A1298C (rs1801131) SNPs. Patients with any *CCND1* A-allele and homozygous for the *MTHFR* A-allele demonstrated the most favorable outcome (95% CI 5.8–19.7; HR = 1). Patients carrying the *CCND1*G/G genotype had a less favorable outcome with a median TTP of 6.1 months (95% CI 2.7–9.4; HR =2.79). Patients carrying any *CCND1* A-allele could be further segregated with patients harboring the *MTHFR* 1298A/C or *MTHFR* 1298C/C genotypes having a median TTP of 5.6 months (95% CI 4.4–10.9; HR = 2.34) ([Figure 2A and B](#fig2)).

### Figure 2.

![Figure 2.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3836/3360546/e262765d23fa/annoncmdr445f02_ht.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3360546_annoncmdr445f02_ht.jpg)

Recursive partioning analysis. (A) This comprehensive recursive partitioning analysis for time to tumor progression (TTP) in metastatic breast cancer patients only incorporated a total of 18 potential markers to define six distinct patient groups (node 1–6) on the basis of TTP with treatment of capecitabine monotherapy or combination chemotherapy of lapatinib plus capecitabine. (B) Patients treated with lapatinib plus capecitabine that carry the CCND1 870 (rs17852153) A/A or A/G and the MTHFR 1298 (rs1801131) A/A genotype have a longer TTP (10.8 months) compared with those patients who carry the CCND1 870 A/A or A/G and MTHFR 1298 A/C or C/C genotypes [5.6 months, hazard ratio (HR) = 2.34, 95% confidence interval (CI) 1.08–5.054] or the CCND1 870 G/G genotype (6.1 months; HR = 2.79, 95% CI 1.14–6.78). (C) While patients who were treated with capecitabine alone carried the TS 5′UTR 2R/3G, 3G/3C or 3G/3G demonstrated a longer TTP of 7.1 months within this patient population compared with patients carrying the TS 5′UTR 2R/2R, 2R/3C or 3C/3C and either human epidermal receptor 2 655 A/G (3.1 months; HR = 2.25, 95% CI 1.21–4.16) or A/A (4.0 months; HR = 1.16, 95% CI 0.57–2.39) genotypes. CCND1, cyclin D1; MTHFR, methylenetetrahydrofolate reductase; TS, thymidylate synthase.

Within the patients who were given capecitabine monotherapy, the patients carrying the *TYMS* 5′-UTR polymorphisms with the 2R/2R, 2R/3C or 3C/3C genotypes demonstrated a decreased TTP when compared with the patients carrying the *TYMS* 5′-UTR polymorphisms 2R/3G; 2G/3C; 3G/3G with a median TTP of 7.1 months. The patients with the *TYMS* 5′-UTR polymorphisms of 2R/2R, 2R/3C or 3C/3C group can be further subdivided by the HER2 Ile655Val (rs1136201) polymorphisms, with patients harboring the A/A genotype having a TTP of 4.0 months (HR = 1.16; 95% CI 0.57–2.39) compared with those harboring the A/G genotype having a TTP of 3.1 months (HR = 2.25; 95% CI 1.21–4.16) ([Figure 2A and C](#fig2)).

## discussion

This study identified an SNP in the *CCND1* gene that was associated with increased RR, clinical benefit and TTP for patients receiving the combination of lapatinib plus capecitabine when compared with capecitabine monotherapy. Importantly, the *CCND1* 870A was consistently associated with all measures of clinical outcome in our study including RR, clinical benefit (which also considers stable disease rate) and TTP. The results of our study demonstrate a consistent and reproducible detrimental effect of the A-allele in mBC patients treated with capecitabine. Importantly, this association was independent of duration of previous trastuzumab treatment and previous lines of chemotherapy. Cyclin D1 is a member of the D-type cyclin family and an oncogene whose overexpression has been implicated in the etiology of a number of solid tumors including BC. As an essential regulator of cell cycle progression of the G_1_/S phase, cyclin D1 regulates the formation of active enzyme complexes and promotes S phase entry [[22](#bib22), [23](#bib23)]. Previous studies have demonstrated that cyclin D1 overexpression disrupts normal cell cycle control, thereby promoting the development and progression of many types of cancer, including breast, colon, lung, prostate and thyroid cancer [[24](#bib24)–[29](#bib29)]. In fact, cyclin D1 is one of the most commonly overexpressed oncogenes in BC and is reported in 45%–50% of primary ductal carcinomas [[27](#bib27)].

Two mRNA transcripts have been identified for cyclin D1, transcript-*a* and transcript-*b*. Transcript-*a* is the full-length 4.5-kb mRNA that includes exons 1–5, whereas transcript-*b* is shorter at 1.5- to 1.7-kb and is composed of exons 1–4 and intron 4 [[30](#bib30)–[33](#bib33)]. In the context of carcinogenesis, the presence of the polymorphic variant A-allele has been correlated with the expression of the variant transcript-*b* implicated in this study [[31](#bib31)]. Specifically, patients who are homozygous for the G-allele demonstrate predominant tumoral expression of the full-length cyclin D1 with normal activity and patients homozygous for the A-allele demonstrate elevated tumoral expression of the alternatively spliced mRNA with increased cyclin D1 activity and associated oncogenic functions. Heterozygous patients express both transcripts and have moderate cyclin D1 activity level within the cell. Betticher et al. [[30](#bib30)] functionally characterized the differing transcripts and demonstrated that the variant transcript-*b* lacks the proline-glutamic acid-serine-theonine (PEST)-rich region, a domain important in destabilizing *CCND1* mediated by the presence of the Thr-286 phosphorylation site necessary for cyclin D1 nuclear export. The loss of this domain leads to a significant increase in the protein half-life of transcript-*b* compared with the protein encoded by transcript-*a*. The SNP at 870 occurs directly at the splice donor sight between exon 4 and 5 and is reported to be an important determinant of successful *CCND1* mRNA splicing [[31](#bib31)]. The role of the A870G SNP and its association with cancer risk has reported inconsistently in the literature and appears to differ by cancer type. A recent meta-analysis that analyzed over 60 studies encompassing nine different cancers compared the significance of the A870G in 18 411 individuals who developed cancer and 22 209 healthy controls and concluded that there is strong evidence supporting an increased cancer risk associated with the *CCND1* A870G polymorphism in the human population [[34](#bib34)]. In addition, a growing number of reports have associated the A-allele with lack of response to chemotherapy. Preclinical studies utilizing cell line models, implicated cyclin D1 overexpression with resistance to a number of chemotherapeutics in gastric cancer cells and down-regulation of cyclin D1 by curcumin was reported to sensitize colorectal cancer cells to capecitabine in orthotopic mouse models [[35](#bib35)]. In non-small-cell lung cancer, the A-allele was strongly associated with increased risk of malignancy and lack of response to platinum-based chemotherapy [[36](#bib36)]. In colorectal cancer (CRC), the A-allele was associated with a highly significant decrease in OS in patients treated with the EGFR-targeted monoclonal antibody cetuximab [[37](#bib37)]. A recent study in CRC also reported that the *CCND1* A-allele was associated with significantly decreased TTP in patients receiving irinotecan-based chemotherapy but not in those who received EGFR-targeted therapy only, providing evidence of the role of the A-allele in conferring resistance particularly to DNA damaging therapeutics [[38](#bib38)]. The role of cyclin D1 in mediating response to DNA damage has been described by Zhiping et al. who reported differential roles for the alternate cyclin D1 isoforms in mediating the response to DNA damage. Specifically, transcript*-b* was associated with decreased DNA repair, decreased induction of the cell cycle inhibitor p21, decreased cell cycle arrest and decreased double-stranded DNA breaks following treatment with a DNA damaging agent [[38](#bib38)]. It is plausible that the reduced DNA repair ability of cyclin D1 transcript*-b* is a contributory factor to the increased cancer risk associated with the A-allele. In addition, the compromised ability of transcript*-b* to induce cell cycle arrest and apoptosis is a plausible explanation for the increased resistance to DNA damaging therapeutics observed with the A-allele in this and other studies.

In the current study, the *CCND1* A-allele that promotes the oncogenic transcript*-b* was strongly associated with poor clinical outcome in patients who received capecitabine alone. However, and of note, the A870G polymorphism was not associated with poor clinical outcome in patients who received the lapatinib plus capecitabine combination. It has been well established that the *CCND1* gene is a direct transcriptional target of the PI3K/AKT signaling pathway, which represents a primary mitogenic pathway activated by HER2. Furthermore, a recent microarray analysis identified *CCND1* as one of the most heavily down-regulated genes in response to lapatinib treatment in HER2-amplified cancer cell lines. Therefore, it is plausible that the inclusion of lapatinib in combination with capecitabine may have counteracted the negative effect of the A870G polymorphism in some patients as a result of HER2 inhibition and transcriptional suppression of the *CCND1* gene thereby inhibiting the transcription and translation of the detrimental oncogenic transcript*-b*. These results suggest that patients carrying the A-allele (A/A or A/G) are at a decreased probability of benefit from capecitabine monotherapy and would be candidates for the addition of lapatinib.

In conclusion, we analyzed a panel of germline SNPs involved in pathways governing the metabolism and mechanism of action of lapatinib and capecitabine in order to identify molecular markers that may identify patients with an increased probability of benefit from this combination. Only one SNP was significantly associated with clinical outcome in the EGF100151 patient cohort. Our study indicates the *CCND1* A870G SNP to be a predictive marker of clinical benefit to lapatinib plus capecitabine in patients with mBC. However, it is likely that *CCND1* A870G constitutes only one member of an as yet incomplete panel of molecular markers that will need to be considered in the treatment decision-making process in HER2-positive mBC.

Although this study is limited somewhat by its retrospective nature, the results and significance are strengthened by the reasonably-sized and randomized study design and the consistent results demonstrating the negative influence of the *CCND1* A-variant in multiple measures of clinical outcome. Furthermore, the patient cohort that was analyzed contained a control group that received capecitabine monotherapy and therefore provided a comparative group in which the influence of the polymorphism based on the therapy administered could be ascertained. Finally, based on the extensive literature that reports a detrimental role for the *CCND1* A-variant in multiple solid malignancies, and the oncogenic role of cyclin D1 frequently reported in BC, these results warrant further validation in larger prospective clinical trials.

## funding

This work was supported by the National Institute of Health (grant number 5 P30CA14089-27I) and carried out with the cooperation of GlaxoSmithKline. This study was completed in the Sharon A. Carpenter Laboratory at University of Southern California/Norris Comprehensive Cancer Center and in memory of Walter Norton.

## disclosure

The authors declare no conflict of interest.

## Supplementary Material

## Acknowledgments

Previous Presentation: ASCO Annual Meeting 2010 (Abstract ID #1015).

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. McPherson K, Steel CM, Dixon JM. ABC of breast diseases. Breast cancer-epidemiology, risk factors, and genetics. BMJ. 2000;321:624–628. doi: 10.1136/bmj.321.7261.624.  [DOI](https://doi.org/10.1136/bmj.321.7261.624) | [PMC free article](/articles/PMC1118507/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10977847/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMJ&title=ABC%20of%20breast%20diseases.%20Breast%20cancer-epidemiology,%20risk%20factors,%20and%20genetics&author=K%20McPherson&author=CM%20Steel&author=JM%20Dixon&volume=321&publication_year=2000&pages=624-628&pmid=10977847&doi=10.1136/bmj.321.7261.624&)

2. Parkin DM, Bray F, Ferlay J, Pisani P. Global cancer statistics, 2002. CA Cancer J Clin. 2005;55:74–108. doi: 10.3322/canjclin.55.2.74.  [DOI](https://doi.org/10.3322/canjclin.55.2.74) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15761078/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=CA%20Cancer%20J%20Clin&title=Global%20cancer%20statistics,%202002&author=DM%20Parkin&author=F%20Bray&author=J%20Ferlay&author=P%20Pisani&volume=55&publication_year=2005&pages=74-108&pmid=15761078&doi=10.3322/canjclin.55.2.74&)

3. American Cancer Society. Cancer Facts & Figures 2010. Atlanta, GA: American Cancer Society; 2010.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Cancer%20Facts%20&%20Figures%202010&publication_year=2010&)

4. McCann AH, Dervan PA, O’Regan M, et al. Prognostic significance of c-erbB-2 and estrogen receptor status in human breast cancer. Cancer Res. 1991;51:3296–3303.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/1674898/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=Prognostic%20significance%20of%20c-erbB-2%20and%20estrogen%20receptor%20status%20in%20human%20breast%20cancer&author=AH%20McCann&author=PA%20Dervan&author=M%20O%E2%80%99Regan&volume=51&publication_year=1991&pages=3296-3303&pmid=1674898&)

5. Slamon DJ, Clark GM, Wong SG, et al. Human breast cancer: correlation of relapse and survival with amplification of the HER-2/neu oncogene. Science. 1987;235:177–182. doi: 10.1126/science.3798106.  [DOI](https://doi.org/10.1126/science.3798106) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/3798106/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Human%20breast%20cancer:%20correlation%20of%20relapse%20and%20survival%20with%20amplification%20of%20the%20HER-2/neu%20oncogene&author=DJ%20Slamon&author=GM%20Clark&author=SG%20Wong&volume=235&publication_year=1987&pages=177-182&pmid=3798106&doi=10.1126/science.3798106&)

6. Menard S, Fortis S, Castiglioni F, et al. HER2 as a prognostic factor in breast cancer. Oncology. 2001;61(Suppl 2):67–72. doi: 10.1159/000055404.  [DOI](https://doi.org/10.1159/000055404) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11694790/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncology&title=HER2%20as%20a%20prognostic%20factor%20in%20breast%20cancer&author=S%20Menard&author=S%20Fortis&author=F%20Castiglioni&volume=61&issue=Suppl%202&publication_year=2001&pages=67-72&pmid=11694790&doi=10.1159/000055404&)

7. Gullick WJ, Love SB, Wright C, et al. c-erbB-2 protein overexpression in breast cancer is a risk factor in patients with involved and uninvolved lymph nodes. Br J Cancer. 1991;63:434–438. doi: 10.1038/bjc.1991.100.  [DOI](https://doi.org/10.1038/bjc.1991.100) | [PMC free article](/articles/PMC1971853/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1672253/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Cancer&title=c-erbB-2%20protein%20overexpression%20in%20breast%20cancer%20is%20a%20risk%20factor%20in%20patients%20with%20involved%20and%20uninvolved%20lymph%20nodes&author=WJ%20Gullick&author=SB%20Love&author=C%20Wright&volume=63&publication_year=1991&pages=434-438&pmid=1672253&doi=10.1038/bjc.1991.100&)

8. Xia W, Mullin RJ, Keith BR, et al. Anti-tumor activity of GW572016: a dual tyrosine kinase inhibitor blocks EGF activation of EGFR/erbB2 and downstream Erk1/2 and AKT pathways. Oncogene. 2002;21:6255–6263. doi: 10.1038/sj.onc.1205794.  [DOI](https://doi.org/10.1038/sj.onc.1205794) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12214266/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncogene&title=Anti-tumor%20activity%20of%20GW572016:%20a%20dual%20tyrosine%20kinase%20inhibitor%20blocks%20EGF%20activation%20of%20EGFR/erbB2%20and%20downstream%20Erk1/2%20and%20AKT%20pathways&author=W%20Xia&author=RJ%20Mullin&author=BR%20Keith&volume=21&publication_year=2002&pages=6255-6263&pmid=12214266&doi=10.1038/sj.onc.1205794&)

9. Konecny GE, Pegram MD, Venkatesan N, et al. Activity of the dual kinase inhibitor lapatinib (GW572016) against HER-2-overexpressing and trastuzumab-treated breast cancer cells. Cancer Res. 2006;66:1630–1639. doi: 10.1158/0008-5472.CAN-05-1182.  [DOI](https://doi.org/10.1158/0008-5472.CAN-05-1182) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16452222/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=Activity%20of%20the%20dual%20kinase%20inhibitor%20lapatinib%20(GW572016)%20against%20HER-2-overexpressing%20and%20trastuzumab-treated%20breast%20cancer%20cells&author=GE%20Konecny&author=MD%20Pegram&author=N%20Venkatesan&volume=66&publication_year=2006&pages=1630-1639&pmid=16452222&doi=10.1158/0008-5472.CAN-05-1182&)

10. Johnston S, Pippen J, Jr, Pivot X, et al. Lapatinib combined with letrozole versus letrozole and placebo as first-line therapy for postmenopausal hormone receptor-positive metastatic breast cancer. J Clin Oncol. 2009;27:5538–5546. doi: 10.1200/JCO.2009.23.3734.  [DOI](https://doi.org/10.1200/JCO.2009.23.3734) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19786658/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Lapatinib%20combined%20with%20letrozole%20versus%20letrozole%20and%20placebo%20as%20first-line%20therapy%20for%20postmenopausal%20hormone%20receptor-positive%20metastatic%20breast%20cancer&author=S%20Johnston&author=J%20Pippen&author=X%20Pivot&volume=27&publication_year=2009&pages=5538-5546&pmid=19786658&doi=10.1200/JCO.2009.23.3734&)

11. Geyer CE, Forster J, Lindquist D, et al. Lapatinib plus capecitabine for HER2-positive advanced breast cancer. N Engl J Med. 2006;355:2733–2743. doi: 10.1056/NEJMoa064320.  [DOI](https://doi.org/10.1056/NEJMoa064320) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17192538/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Lapatinib%20plus%20capecitabine%20for%20HER2-positive%20advanced%20breast%20cancer&author=CE%20Geyer&author=J%20Forster&author=D%20Lindquist&volume=355&publication_year=2006&pages=2733-2743&pmid=17192538&doi=10.1056/NEJMoa064320&)

12. Cameron D, Casey M, Press M, et al. A phase III randomized comparison of lapatinib plus capecitabine versus capecitabine alone in women with advanced breast cancer that has progressed on trastuzumab: updated efficacy and biomarker analyses. Breast Cancer Res Treat. 2008;112:533–543. doi: 10.1007/s10549-007-9885-0.  [DOI](https://doi.org/10.1007/s10549-007-9885-0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18188694/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Breast%20Cancer%20Res%20Treat&title=A%20phase%20III%20randomized%20comparison%20of%20lapatinib%20plus%20capecitabine%20versus%20capecitabine%20alone%20in%20women%20with%20advanced%20breast%20cancer%20that%20has%20progressed%20on%20trastuzumab:%20updated%20efficacy%20and%20biomarker%20analyses&author=D%20Cameron&author=M%20Casey&author=M%20Press&volume=112&publication_year=2008&pages=533-543&pmid=18188694&doi=10.1007/s10549-007-9885-0&)

13. Press MF, Finn RS, Cameron D, et al. HER-2 gene amplification, HER-2 and epidermal growth factor receptor mRNA and protein expression, and lapatinib efficacy in women with metastatic breast cancer. Clin Cancer Res. 2008;14:7861–7870. doi: 10.1158/1078-0432.CCR-08-1056.  [DOI](https://doi.org/10.1158/1078-0432.CCR-08-1056) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19047115/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=HER-2%20gene%20amplification,%20HER-2%20and%20epidermal%20growth%20factor%20receptor%20mRNA%20and%20protein%20expression,%20and%20lapatinib%20efficacy%20in%20women%20with%20metastatic%20breast%20cancer&author=MF%20Press&author=RS%20Finn&author=D%20Cameron&volume=14&publication_year=2008&pages=7861-7870&pmid=19047115&doi=10.1158/1078-0432.CCR-08-1056&)

14. Spector NL, Xia W, Burris H, III, et al. Study of the biologic effects of lapatinib, a reversible inhibitor of ErbB1 and ErbB2 tyrosine kinases, on tumor growth and survival pathways in patients with advanced malignancies. J Clin Oncol. 2005;23:2502–2512. doi: 10.1200/JCO.2005.12.157.  [DOI](https://doi.org/10.1200/JCO.2005.12.157) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15684311/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Study%20of%20the%20biologic%20effects%20of%20lapatinib,%20a%20reversible%20inhibitor%20of%20ErbB1%20and%20ErbB2%20tyrosine%20kinases,%20on%20tumor%20growth%20and%20survival%20pathways%20in%20patients%20with%20advanced%20malignancies&author=NL%20Spector&author=W%20Xia&author=H%20Burris&volume=23&publication_year=2005&pages=2502-2512&pmid=15684311&doi=10.1200/JCO.2005.12.157&)

15. Rhee J, Han SW, Cha Y, et al. High serum TGF-alpha predicts poor response to lapatinib and capecitabine in HER2-positive breast cancer. Breast Cancer Res Treat. 2010;125:107–114. doi: 10.1007/s10549-010-1200-9.  [DOI](https://doi.org/10.1007/s10549-010-1200-9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20936340/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Breast%20Cancer%20Res%20Treat&title=High%20serum%20TGF-alpha%20predicts%20poor%20response%20to%20lapatinib%20and%20capecitabine%20in%20HER2-positive%20breast%20cancer&author=J%20Rhee&author=SW%20Han&author=Y%20Cha&volume=125&publication_year=2010&pages=107-114&pmid=20936340&doi=10.1007/s10549-010-1200-9&)

16. Lurje G, Husain H, Power DG, et al. Genetic variations in angiogenesis pathway genes associated with clinical outcome in localized gastric adenocarcinoma. Ann Oncol. 2010;21:78–86. doi: 10.1093/annonc/mdp280.  [DOI](https://doi.org/10.1093/annonc/mdp280) | [PMC free article](/articles/PMC2795613/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19622587/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Oncol&title=Genetic%20variations%20in%20angiogenesis%20pathway%20genes%20associated%20with%20clinical%20outcome%20in%20localized%20gastric%20adenocarcinoma&author=G%20Lurje&author=H%20Husain&author=DG%20Power&volume=21&publication_year=2010&pages=78-86&pmid=19622587&doi=10.1093/annonc/mdp280&)

17. Lurje G, Zhang W, Schultheis AM, et al. Polymorphisms in VEGF and IL-8 predict tumor recurrence in stage III colon cancer. Ann Oncol. 2008;19:1734–1741. doi: 10.1093/annonc/mdn368.  [DOI](https://doi.org/10.1093/annonc/mdn368) | [PMC free article](/articles/PMC2735064/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18550579/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Oncol&title=Polymorphisms%20in%20VEGF%20and%20IL-8%20predict%20tumor%20recurrence%20in%20stage%20III%20colon%20cancer&author=G%20Lurje&author=W%20Zhang&author=AM%20Schultheis&volume=19&publication_year=2008&pages=1734-1741&pmid=18550579&doi=10.1093/annonc/mdn368&)

18. Lurje G, Nagashima F, Zhang W, et al. Polymorphisms in cyclooxygenase-2 and epidermal growth factor receptor are associated with progression-free survival independent of K-ras in metastatic colorectal cancer patients treated with single-agent cetuximab. Clin Cancer Res. 2008;14:7884–7895. doi: 10.1158/1078-0432.CCR-07-5165.  [DOI](https://doi.org/10.1158/1078-0432.CCR-07-5165) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19047118/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=Polymorphisms%20in%20cyclooxygenase-2%20and%20epidermal%20growth%20factor%20receptor%20are%20associated%20with%20progression-free%20survival%20independent%20of%20K-ras%20in%20metastatic%20colorectal%20cancer%20patients%20treated%20with%20single-agent%20cetuximab&author=G%20Lurje&author=F%20Nagashima&author=W%20Zhang&volume=14&publication_year=2008&pages=7884-7895&pmid=19047118&doi=10.1158/1078-0432.CCR-07-5165&)

19. Lurje G, Zhang W, Yang D, et al. Thymidylate synthase haplotype is associated with tumor recurrence in stage II and stage III colon cancer. Pharmacogenet Genomics. 2008;18:161–168. doi: 10.1097/FPC.0b013e3282f4aea6.  [DOI](https://doi.org/10.1097/FPC.0b013e3282f4aea6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18192902/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Thymidylate%20synthase%20haplotype%20is%20associated%20with%20tumor%20recurrence%20in%20stage%20II%20and%20stage%20III%20colon%20cancer&author=G%20Lurje&author=W%20Zhang&author=D%20Yang&volume=18&publication_year=2008&pages=161-168&pmid=18192902&doi=10.1097/FPC.0b013e3282f4aea6&)

20. Breiman L, Friedman JH, Olshen RA, Stone CJ. Classification and Regression Trees. New York: Chapman & Hall (Wadsworth, Inc); 1984.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Classification%20and%20Regression%20Trees&author=L%20Breiman&author=JH%20Friedman&author=RA%20Olshen&author=CJ%20Stone&publication_year=1984&)

21. Therneau TM, Atkinson EJ. An Introduction to Recursive Partitioning Using the RPART Routines. Rochester, MN: Mayo Clinic; 1997. Technical Report no 61.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=An%20Introduction%20to%20Recursive%20Partitioning%20Using%20the%20RPART%20Routines&author=TM%20Therneau&author=EJ%20Atkinson&publication_year=1997&)

22. Kato J, Matsushime H, Hiebert SW, et al. Direct binding of cyclin D to the retinoblastoma gene product (pRb) and pRb phosphorylation by the cyclin D-dependent kinase CDK4. Genes Dev. 1993;7:331–342. doi: 10.1101/gad.7.3.331.  [DOI](https://doi.org/10.1101/gad.7.3.331) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8449399/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genes%20Dev&title=Direct%20binding%20of%20cyclin%20D%20to%20the%20retinoblastoma%20gene%20product%20(pRb)%20and%20pRb%20phosphorylation%20by%20the%20cyclin%20D-dependent%20kinase%20CDK4&author=J%20Kato&author=H%20Matsushime&author=SW%20Hiebert&volume=7&publication_year=1993&pages=331-342&pmid=8449399&doi=10.1101/gad.7.3.331&)

23. Weinberg RA. The retinoblastoma protein and cell cycle control. Cell. 1995;81:323–330. doi: 10.1016/0092-8674(95)90385-2.  [DOI](https://doi.org/10.1016/0092-8674(95)90385-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7736585/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell&title=The%20retinoblastoma%20protein%20and%20cell%20cycle%20control&author=RA%20Weinberg&volume=81&publication_year=1995&pages=323-330&pmid=7736585&doi=10.1016/0092-8674(95)90385-2&)

24. Pirkmaier A, Yuen K, Hendley J, et al. Cyclin d1 overexpression sensitizes breast cancer cells to fenretinide. Clin Cancer Res. 2003;9:1877–1884.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/12738746/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=Cyclin%20d1%20overexpression%20sensitizes%20breast%20cancer%20cells%20to%20fenretinide&author=A%20Pirkmaier&author=K%20Yuen&author=J%20Hendley&volume=9&publication_year=2003&pages=1877-1884&pmid=12738746&)

25. Ratschiller D, Heighway J, Gugger M, et al. Cyclin D1 overexpression in bronchial epithelia of patients with lung cancer is associated with smoking and predicts survival. J Clin Oncol. 2003;21:2085–2093. doi: 10.1200/JCO.2003.03.103.  [DOI](https://doi.org/10.1200/JCO.2003.03.103) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12775733/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Cyclin%20D1%20overexpression%20in%20bronchial%20epithelia%20of%20patients%20with%20lung%20cancer%20is%20associated%20with%20smoking%20and%20predicts%20survival&author=D%20Ratschiller&author=J%20Heighway&author=M%20Gugger&volume=21&publication_year=2003&pages=2085-2093&pmid=12775733&doi=10.1200/JCO.2003.03.103&)

26. Khoo ML, Beasley NJ, Ezzat S, et al. Overexpression of cyclin D1 and underexpression of p27 predict lymph node metastases in papillary thyroid carcinoma. J Clin Endocrinol Metab. 2002;87:1814–1818. doi: 10.1210/jcem.87.4.8353.  [DOI](https://doi.org/10.1210/jcem.87.4.8353) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11932323/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Endocrinol%20Metab&title=Overexpression%20of%20cyclin%20D1%20and%20underexpression%20of%20p27%20predict%20lymph%20node%20metastases%20in%20papillary%20thyroid%20carcinoma&author=ML%20Khoo&author=NJ%20Beasley&author=S%20Ezzat&volume=87&publication_year=2002&pages=1814-1818&pmid=11932323&doi=10.1210/jcem.87.4.8353&)

27. Drobnjak M, Osman I, Scher HI, et al. Overexpression of cyclin D1 is associated with metastatic prostate cancer to bone. Clin Cancer Res. 2000;6:1891–1895.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/10815912/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=Overexpression%20of%20cyclin%20D1%20is%20associated%20with%20metastatic%20prostate%20cancer%20to%20bone&author=M%20Drobnjak&author=I%20Osman&author=HI%20Scher&volume=6&publication_year=2000&pages=1891-1895&pmid=10815912&)

28. Holley SL, Parkes G, Matthias C, et al. Cyclin D1 polymorphism and expression in patients with squamous cell carcinoma of the head and neck. Am J Pathol. 2001;159:1917–1924. doi: 10.1016/S0002-9440(10)63038-6.  [DOI](https://doi.org/10.1016/S0002-9440(10)63038-6) | [PMC free article](/articles/PMC1867042/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11696452/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Pathol&title=Cyclin%20D1%20polymorphism%20and%20expression%20in%20patients%20with%20squamous%20cell%20carcinoma%20of%20the%20head%20and%20neck&author=SL%20Holley&author=G%20Parkes&author=C%20Matthias&volume=159&publication_year=2001&pages=1917-1924&pmid=11696452&doi=10.1016/S0002-9440(10)63038-6&)

29. McKay JA, Douglas JJ, Ross VG, et al. Cyclin D1 protein expression and gene polymorphism in colorectal cancer. Aberdeen Colorectal Initiative. Int J Cancer. 2000;88:77–81. doi: 10.1002/1097-0215(20001001)88:1<77::aid-ijc12>3.0.co;2-o.  [DOI](https://doi.org/10.1002/1097-0215(20001001)88:1<77::aid-ijc12>3.0.co;2-o) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10964085/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Cancer&title=Cyclin%20D1%20protein%20expression%20and%20gene%20polymorphism%20in%20colorectal%20cancer.%20Aberdeen%20Colorectal%20Initiative&author=JA%20McKay&author=JJ%20Douglas&author=VG%20Ross&volume=88&publication_year=2000&pages=77-81&pmid=10964085&doi=10.1002/1097-0215(20001001)88:1<77::aid-ijc12>3.0.co;2-o&)

30. Betticher DC, Thatcher N, Altermatt HJ, et al. Alternate splicing produces a novel cyclin D1 transcript. Oncogene. 1995;11:1005–1011.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/7675441/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncogene&title=Alternate%20splicing%20produces%20a%20novel%20cyclin%20D1%20transcript&author=DC%20Betticher&author=N%20Thatcher&author=HJ%20Altermatt&volume=11&publication_year=1995&pages=1005-1011&pmid=7675441&)

31. Knudsen KE, Diehl JA, Haiman CA, Knudsen ES. Cyclin D1: polymorphism, aberrant splicing and cancer risk. Oncogene. 2006;25:1620–1628. doi: 10.1038/sj.onc.1209371.  [DOI](https://doi.org/10.1038/sj.onc.1209371) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16550162/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncogene&title=Cyclin%20D1:%20polymorphism,%20aberrant%20splicing%20and%20cancer%20risk&author=KE%20Knudsen&author=JA%20Diehl&author=CA%20Haiman&author=ES%20Knudsen&volume=25&publication_year=2006&pages=1620-1628&pmid=16550162&doi=10.1038/sj.onc.1209371&)

32. Solomon DA, Wang Y, Fox SR, et al. Cyclin D1 splice variants. Differential effects on localization, RB phosphorylation, and cellular transformation. J Biol Chem. 2003;278:30339–30347. doi: 10.1074/jbc.M303969200.  [DOI](https://doi.org/10.1074/jbc.M303969200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12746453/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Cyclin%20D1%20splice%20variants.%20Differential%20effects%20on%20localization,%20RB%20phosphorylation,%20and%20cellular%20transformation&author=DA%20Solomon&author=Y%20Wang&author=SR%20Fox&volume=278&publication_year=2003&pages=30339-30347&pmid=12746453&doi=10.1074/jbc.M303969200&)

33. Li Z, Jiao X, Wang C, et al. Alternative cyclin D1 splice forms differentially regulate the DNA damage response. Cancer Res. 2010;70:8802–8811. doi: 10.1158/0008-5472.CAN-10-0312.  [DOI](https://doi.org/10.1158/0008-5472.CAN-10-0312) | [PMC free article](/articles/PMC2970762/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20940395/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=Alternative%20cyclin%20D1%20splice%20forms%20differentially%20regulate%20the%20DNA%20damage%20response&author=Z%20Li&author=X%20Jiao&author=C%20Wang&volume=70&publication_year=2010&pages=8802-8811&pmid=20940395&doi=10.1158/0008-5472.CAN-10-0312&)

34. Pabalan N, Bapat B, Sung L, et al. Cyclin D1 Pro241Pro (CCND1-G870A) polymorphism is associated with increased cancer risk in human populations: a meta-analysis. Cancer Epidemiol Biomarkers Prev. 2008;17:2773–2781. doi: 10.1158/1055-9965.EPI-08-0169.  [DOI](https://doi.org/10.1158/1055-9965.EPI-08-0169) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18843022/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Epidemiol%20Biomarkers%20Prev&title=Cyclin%20D1%20Pro241Pro%20(CCND1-G870A)%20polymorphism%20is%20associated%20with%20increased%20cancer%20risk%20in%20human%20populations:%20a%20meta-analysis&author=N%20Pabalan&author=B%20Bapat&author=L%20Sung&volume=17&publication_year=2008&pages=2773-2781&pmid=18843022&doi=10.1158/1055-9965.EPI-08-0169&)

35. Kunnumakkara AB, Diagaradjane P, Anand P, et al. Curcumin sensitizes human colorectal cancer to capecitabine by modulation of cyclin D1, COX-2, MMP-9, VEGF and CXCR4 expression in an orthotopic mouse model. Int J Cancer. 2009;125:2187–2197. doi: 10.1002/ijc.24593.  [DOI](https://doi.org/10.1002/ijc.24593) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19623659/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Cancer&title=Curcumin%20sensitizes%20human%20colorectal%20cancer%20to%20capecitabine%20by%20modulation%20of%20cyclin%20D1,%20COX-2,%20MMP-9,%20VEGF%20and%20CXCR4%20expression%20in%20an%20orthotopic%20mouse%20model&author=AB%20Kunnumakkara&author=P%20Diagaradjane&author=P%20Anand&volume=125&publication_year=2009&pages=2187-2197&pmid=19623659&doi=10.1002/ijc.24593&)

36. Gautschi O, Hugli B, Ziegler A, et al. Cyclin D1 (CCND1) A870G gene polymorphism modulates smoking-induced lung cancer risk and response to platinum-based chemotherapy in non-small cell lung cancer (NSCLC) patients. Lung Cancer. 2006;51:303–311. doi: 10.1016/j.lungcan.2005.10.025.  [DOI](https://doi.org/10.1016/j.lungcan.2005.10.025) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16406195/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lung%20Cancer&title=Cyclin%20D1%20(CCND1)%20A870G%20gene%20polymorphism%20modulates%20smoking-induced%20lung%20cancer%20risk%20and%20response%20to%20platinum-based%20chemotherapy%20in%20non-small%20cell%20lung%20cancer%20(NSCLC)%20patients&author=O%20Gautschi&author=B%20Hugli&author=A%20Ziegler&volume=51&publication_year=2006&pages=303-311&pmid=16406195&doi=10.1016/j.lungcan.2005.10.025&)

37. Zhang W, Gordon M, Press OA, et al. Cyclin D1 and epidermal growth factor polymorphisms associated with survival in patients with advanced colorectal cancer treated with Cetuximab. Pharmacogenet Genomics. 2006;16:475–483. doi: 10.1097/01.fpc.0000220562.67595.a5.  [DOI](https://doi.org/10.1097/01.fpc.0000220562.67595.a5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16788380/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Cyclin%20D1%20and%20epidermal%20growth%20factor%20polymorphisms%20associated%20with%20survival%20in%20patients%20with%20advanced%20colorectal%20cancer%20treated%20with%20Cetuximab&author=W%20Zhang&author=M%20Gordon&author=OA%20Press&volume=16&publication_year=2006&pages=475-483&pmid=16788380&doi=10.1097/01.fpc.0000220562.67595.a5&)

38. Zhang W, Azuma M, Lurje G, et al. Molecular predictors of combination targeted therapies (cetuximab, bevacizumab) in irinotecan-refractory colorectal cancer (BOND-2 study) Anticancer Res. 2010;30:4209–4217.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/21036743/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anticancer%20Res&title=Molecular%20predictors%20of%20combination%20targeted%20therapies%20(cetuximab,%20bevacizumab)%20in%20irinotecan-refractory%20colorectal%20cancer%20(BOND-2%20study)&author=W%20Zhang&author=M%20Azuma&author=G%20Lurje&volume=30&publication_year=2010&pages=4209-4217&pmid=21036743&)
