# Do CYP2D6 genotypes reflect oxycodone requirements for cancer patients treated for cancer pain? A cross-sectional multicentre study

## Metadata
**Authors:** Trine Naalsund Andreassen, Ingrid Eftedal, Pål Klepstad, Andrew Davies, Kristin Bjordal, Staffan Lundström, Stein Kaasa, Ola Dale
**Journal:** European Journal of Clinical Pharmacology
**Date:** 2011 Jul 7
**DOI:** [10.1007/s00228-011-1093-5](https://doi.org/10.1007/s00228-011-1093-5)
**PMID:** 21735164
**PMCID:** PMC3249195
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3249195/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC3249195/pdf/228_2011_Article_1093.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC3249195/pdf/228_2011_Article_1093.pdf)

## Abstract

**Objective:** 
Opioids are recommended by the World Health Organization for moderate to severe cancer pain. Oxycodone is one of the most commonly used opioids and is metabolized in the liver by CYP3A4 and CYP2D6 enzymes. The aim of this cross-sectional study was to assess the relationship between oxycodone pharmacokinetics, pharmacodynamics and the CYP2D6 genotypes “poor metaboliser” (PM), “extensive metaboliser” (EM) and “ultra-rapid metaboliser” (URM) in a cohort of patients with cancer pain.

**Methods:** 
The patients were genotyped for the most common CYP2D6 variants and serum concentrations of oxycodone and metabolites were determined. Pain was assessed using the Brief Pain Inventory (BPI). The EORTC QLQ-C30 was used to assess the symptoms of tiredness and nausea. Cognitive function was assessed by the Mini Mental State (MMS) examination. Associations were examined by analyses of variance (ANOVA) and covariance (ANCOVA), or ordinal logistic regressions with and without covariates.

**Results:** 
The sample consisted of 27 PM, 413 EM (including heterozygotes) and 10 URM. PM had lower oxymorphone and noroxymorphone serum concentrations and oxymorphone to oxycodone ratios than EM and URM. No differences between PM, EM and URM in pain intensity, nausea, tiredness or cognitive function was found.

**Conclusion:** 
CYP2D6 genotypes caused expected differences in pharmacokinetics, but they had no pharmacodynamic consequence. CYP2D6 genotypes did not influence pain control, the adverse symptoms nausea and sedation or the risk for cognitive failure in this study of patients treated with oxycodone for cancer pain.

Keywords: Oxycodone, Metabolites, CYP2D6 genotypes, Pharmacokinetic, Pharmacodynamic, Cancer pain

### Objective

Opioids are recommended by the World Health Organization for moderate to severe cancer pain. Oxycodone is one of the most commonly used opioids and is metabolized in the liver by CYP3A4 and CYP2D6 enzymes. The aim of this cross-sectional study was to assess the relationship between oxycodone pharmacokinetics, pharmacodynamics and the CYP2D6 genotypes “poor metaboliser” (PM), “extensive metaboliser” (EM) and “ultra-rapid metaboliser” (URM) in a cohort of patients with cancer pain.

### Methods

The patients were genotyped for the most common CYP2D6 variants and serum concentrations of oxycodone and metabolites were determined. Pain was assessed using the Brief Pain Inventory (BPI). The EORTC QLQ-C30 was used to assess the symptoms of tiredness and nausea. Cognitive function was assessed by the Mini Mental State (MMS) examination. Associations were examined by analyses of variance (ANOVA) and covariance (ANCOVA), or ordinal logistic regressions with and without covariates.

### Results

The sample consisted of 27 PM, 413 EM (including heterozygotes) and 10 URM. PM had lower oxymorphone and noroxymorphone serum concentrations and oxymorphone to oxycodone ratios than EM and URM. No differences between PM, EM and URM in pain intensity, nausea, tiredness or cognitive function was found.

### Conclusion

*CYP2D6* genotypes caused expected differences in pharmacokinetics, but they had no pharmacodynamic consequence. *CYP2D6* genotypes did not influence pain control, the adverse symptoms nausea and sedation or the risk for cognitive failure in this study of patients treated with oxycodone for cancer pain.

## Introduction

The World Health Organisation (WHO) recommends opioids for the relief of moderate to severe cancer pain [[1](#CR1)]. Oxycodone is one of most commonly used opioids [[2](#CR2)–[4](#CR4)]. Oxycodone is metabolised in the liver mainly by CYP3A4 to noroxycodone, but also by CYP2D6 to oxymorphone, and via 6-keto reduction to α- and β-oxycodol. Noroxycodone and oxymorphone are further metabolised to noroxymorphone by CYP2D6 and CYP3A4 respectively [[5](#CR5)]. CYP3A4 and CYP2D6 belong to the cytochrome P450 system, the principal enzyme system for phase I metabolism. This system is present in virtually all tissues, but is most abundant in the liver and in the small intestine [[6](#CR6)]. The *CYP3A4* gene has many known polymorphisms, but no clinically important differences between genotypes have been observed [[7](#CR7), [8](#CR8)]. *CYP2D6* has several known polymorphisms that influence drug metabolism. Inactivating polymorphisms cause gene mutations and deletion(s) that result in a non-functional enzyme, whereas gene duplication(s) cause over-expression of active enzyme. Individuals with two non-functional alleles of *CYP2D6* are genotyped “poor metaboliser” (PM, 5–10% of Caucasians), whilst individuals with one decreased functional allele and one non-functional allele or two decreased functional alleles are “intermediate metaboliser” (IMs, 10–15% of Caucasians). Persons with two wild-type alleles (*CYP2D6*1*), or one functional and one non-functional allele, are referred to as “extensive metaboliser” (EM, 72–84% of Caucasians). Individuals with more than duplicates of the *CYP2D6* gene are “ultra-rapid metaboliser” (URM, 1–3% of Caucasians) [[9](#CR9), [10](#CR10)]. Because poor metabolisers are unable to metabolise *CYP2D6* substrates, a drug administered at normal dose may lead to high or toxic levels of the drug [[11](#CR11), [12](#CR12)]. On the other hand, an ultra-rapid metaboliser may experience reduced or no effect when given a drug that is a *CYP2D6* substrate, or they may experience adverse drug reactions [[13](#CR13), [14](#CR14)]. The *CYP2D6* genotype may therefore be of clinical importance for drugs that are metabolised by CYP2D6 enzymes.

The effect of the CYP2D6 genotype on the pharmacodynamics of oxycodone in a clinical setting of patients with cancer pain receiving chronic opioid administration has not previously been studied. This led us to the following research questions:

## Materials and methods

### Ethics

This multicentre study was performed according to the guidelines of the Helsinki Declaration and was approved by the relevant Research Ethics Committee of each study centre. Before entering the study, all participating patients gave their informed written consent.

### Patients

The patients were included from 2004 to 2008 in a multicentre cross-sectional study, the European Pharmacogenetic Opioid Study (EPOS) [[15](#CR15)], designed to explore hypotheses related to the pharmacogenetics of opioid analgesics. A convenience sample of 2,294 patients from 17 centres in 11 European countries were included in EPOS, with 461 patients (98% Caucasians) treated with oxycodone. Eleven patients were excluded; 8 because of a lack of DNA samples, and 3 because of incomplete CYP2D6 genotype analyses. Thus, 450 were included in the final analyses. Patients included in the present analysis were aged 18 years or more, had a verified malignant disease, and received scheduled oral, subcutaneous, or intravenous oxycodone treatment with a duration of treatment no less than 3 days. Patients who were not capable of speaking the language used at the study centre were excluded.

### Assessments

At the time of inclusion the following information was collected from each patient: age, gender, weight, height, ethnicity, medications and dosages, the time interval between last opioid administration and blood sampling, time since opioid treatment started, breakthrough pain, cancer diagnosis and time since diagnosis. Pain severity was assessed using the item “Pain on the average” from the Brief Pain Inventory (BPI), which has a numeric rating scale (NRS) from 0 (“No pain”) to 10 (“Pain as bad as you can imagine”) [[16](#CR16)]. The Mini Mental State (MMS) examination was used to assess cognitive function [[17](#CR17)]. Cognitive failure was defined as having a total MMS of 23 or less [[18](#CR18), [19](#CR19)]. Functional status was assessed by the Karnofsky performance status [[20](#CR20)]. The Karnofsky performance status has a linear scoring from 0 to 100%, with higher scores meaning better function. The European Organisation for Research and Treatment of Cancer’s health-related quality-of-life (QoL) questionnaire (EORTC QLQ-C30) version 3.0 was used to assess the patient’s self-reported QoL for the symptoms tiredness, nausea, constipation and depression. Tiredness was assessed using the item “Were you tired?” and depression was assessed using the item “Did you feel depressed?” with alternatives “not at all”, “a little”, “quite a bit” and “very much” for both items. Nausea and constipation were assessed using the symptom scale for nausea and vomiting, and constipation respectively. Scoring of the symptom scales were done by a linear transformation to a 0 to 100 scale. A score of 0–24 on these symptom scales corresponds to “not at all”, 25–49 corresponds to “a little”, 50–74 to “quite a bit” and 75–100 to “very much” [[21](#CR21), [22](#CR22)]. Standard analytical methods applied at each centre were used for haemoglobin, creatinine and albumin measurements. Body mass index (BMI) was calculated using the international system of units, BMI = weight (kg)/height^2^ (m^2^). Renal function was expressed as calculated glomerular filtration rate (GFR)/ 1.73 m^2^ body surface [[23](#CR23), [24](#CR24)].

Blood samples were obtained shortly prior to drug administration of the patients’ scheduled oral opioid medication (trough value). For practical reasons blood samples from out-patients (*n* = 68) were taken at the time of examination. Blood samples for opioid analyses in serum were collected in tubes with no additives and left at ambient temperature for 30–60 min before centrifugation at 2,500×g (approximately 3,000 rpm) for 10 min. Serum was then aliquoted and stored at −80°C prior to analysis.

### Serum concentration analyses

Details on handling of blood samples, analytical technique and instrumentation for serum concentration analyses of oxycodone, and the metabolites oxymorphone, noroxycodone and noroxymorphone have been described previously [[25](#CR25)]. Pharmacological analyses were carried out using a LC-MS/MS system. Correlation coefficients were *r* > 0.998 for all standard curves. Coefficients of variation (intra- and inter-day) for each analyte were 16.5 and 8.3% for oxycodone, 10.8 and 6.7% for noroxycodone, 10.0 and 7.5% for oxymorphone and 14.8 and 7.7% for noroxymorphone. The limits of quantification were oxycodone 0.32 nM (0.1 ng/ml), oxymorphone 0.07 nM (0.02 ng/ml), noroxycodone and noroxymorphone 0.17 nM (0.05 ng/ml).

### Genetic analyses

Blood samples for genetic analysis were collected at the time of inclusion in vacutainers containing EDTA (K2-EDTA, 5.4 mg/3 ml blood). The blood was aliquoted into cryotubes and frozen (−80°C) until isolation of DNA. DNA was isolated by a modified salting-out precipitation method for purification (Gentra Puregene Blood Kit, Gentra Systems, Mineapolis, MN, USA). Purified DNA was stored at −20°C prior to analysis.

### CYP2D6 genotyping

CYP2D6*2x2 (duplication) was detected according to the method of Lovlie et al. [[26](#CR26)], with some modifications. PCR was performed using the Gene Amp XL PCR kit (Roche/Applied Biosystems, Foster City, NJ, USA) in 50 μl reaction volumes with a hot start. The lower reaction mix layer contained 4.5 μl dH_2_O, 6 μl 3.3 XL Buffer II, 2.5 μl Mg(OAc)_2_ (25 mM), 4 μl dNTP Mix (10 mM), and 1.5 μl of each primer (10 mM). To separate the lower and upper reaction mix layer, wax was melted on top of the lower reaction mix (80°C, 3–4 min), and then cooled to room temperature. Then the upper reaction mix containing 18.5 μl dH2O, 9 μl 3.3 XL Buffer II, 1 μl r*Tth* DNA polymerase (2 U/μl) and 2.5 μl genomic DNA (5–10 ng/μl) was added.

Long PCR was carried out on a Techne TC-512 (Barloworld Scientific, Stone, Staffordshire, UK) with the following conditions: an initial denaturing step of 95°C for 1 min, followed by 35 cycles of 94°C for 1 min, 65°C for 30 s and 68°C for 2 min (+18 s increase for every new cycle), and a final elongation step of 72°C for 10 min.

The resulting long PCR products were separated and detected by electrophoresis (30 mA, 40 min) with 1.2% agarose gels containing ethidium bromide and tris-acetate buffer (Invitrogen, Carlsbad, CA, USA).

Detection of *CYP2D6*5* (deletion) was done in accordance with the method of Steen et al. [[27](#CR27)] and Hersberger et al. [[28](#CR28)], with some modifications. PCR was performed using the Gene Amp XL PCR kit (Roche/Applied Biosystems, Foster City, NJ, USA) in 50 μl reaction volumes with a hot start. To separate lower and upper reaction mix, heated wax was used (as described for detection of CYP2D6*2x2). The lower reaction mix contained 1.95 μl dH2O, 6 μl 3.3 XL Buffer II, 2.05 μl Mg(OAc)_2_ (25 mM), 4 μl dNTP Mix (10 mM) and 1.5 μl of each primer (10 mM; Table [1](#Tab1)). The upper reaction mix was as described for the detection of CYP2D6*2x2. The PCR conditions, and identification of the amplification products, were done in the same way as described for CYP2D6*2x2.

### Table 1.

| Genotype | n | Percentage of total population |
| --- | --- | --- |
| EM homozygote | 413 | 91.7 |
| *1/*1 (wild-type) heterozygote | 243 | 54.0 |
| *1/*5 (deletion) | 23 | 5.1 |
| *1/*3 | 12 | 2.7 |
| *1/*4 | 124 | 27.6 |
| *1/*6 | 11 | 2.4 |
| PM | 27 | 6.0 |
| *3/*4 | 2 | 0.4 |
| *4/*4 | 22 | 4.9 |
| *4/*6 | 3 | 0.7 |
| URM | 10 | 2.2 |
| *2/*2 (duplication) | 10 | 2.2 |

Table 1 Caption: Distribution of the CYP2D6 alleles and the corresponding genotype for the genetic groups: extensive metabolisers (EM), poor metabolisers (PM) and ultra-rapid metabolisers (URM) for the 450 patients with cancer pain

Long distance and multiplex PCR techniques were combined for the simultaneous detection of the five allele groups, *3, *4, *6, *7 and *8 in genomic DNA. Patients who lacked these mutations were categorised as having the CYP2D6*1 (functional) allele. With some modifications, these reactions were done in accordance with the method of Stüven et al. [[29](#CR29)].

First CYP2D6 was specifically amplified as a 4.7-kb fragment by a pre-multiplex long PCR with a hot start. To separate lower and upper reaction mix, heated wax was used (as described for the detection of CYP2D6*2 × 2). PCR was performed using the Gene Amp XL PCR kit (Roche/Applied Biosystems, Foster City, NJ, USA) in 50-μl reaction volumes. The lower reaction mix contained 4.5 μl dH2O, 6 μl 3.3 XL Buffer II, 2.5 μl Mg(OAc)2 (25 mM), 4 μl dNTP Mix (10 mM) and 1.5 μl of each primer (10 mM). The upper reaction mix was as described for the detection of CYP2D6*2 × 2. The pre-multiplex long PCR conditions, and identification of the 4.7-kb PCR product, were done in the same way as for CYP2D6*2x2.

The CYP2D6-specific 4.7-kb pre-amplification product was then used as a template for two separate PCR reactions with primers complementary to either the specific inactivating variant or the corresponding normal (wild-type) allele at each potential mutation site. Reactions (25 μl) contained 0.8 μl dH2O, 2.5 μl 10x PCR Gold Buffer, 1.5 μl MgCl2 (25 mM), 0.7 μl dNTP Mix (10 mM), 0.5 μl AmpliTaq Gold Polymerase (5 U/μl; all reagents from Applied Biosystems), 0.2 μl of the CYP2D6-specific 4.7 kb pre-amplification product and 3.76 μl of each of the following primers in the normal reaction; M (1.06 μM), A1 (0.10 μM), B1 (0.11 μM), E3 (0.12 μM) and T1 (0.64 μM), and for the mutation reaction 3.13 μl of each primer M (1.06 μM), A2 (0.10 μM), B2 (1.06 μM), E4 (0.10 μM), G2 (0.50 μM) and T2 (0.05 μM) was used.

Polymer chain reaction was carried out on a Techne TC-512 (Barloworld Scientific) with the following conditions: an initial denaturing step of 95°C for 5 min, followed by 14 cycles of 94°C for 1 min, 55°C for 30 s and 72°C for 2 min 20 s, and a final elongation step of 72°C for 5 min.

The resulting long PCR products were separated and detected by electrophoresis (25 mA, 45 min) with 4.0% agarose gels containing ethidium bromide and tris-acetate buffer (Invitrogen, Carlsbad, CA, USA).

The patients were divided into three genotype groups:

### Statistics

Descriptive group data are given as median (minimum–maximum) values. Comparisons between the genetic groups for the continuous descriptive data were explored with analyses of variance (one-way ANOVA). For the descriptive categorical data the comparisons were explored with logistic regression analyses. Median oxycodone and metabolite serum concentrations were calculated from all 450 patients independent of the time since the last dose to blood sample and opioid used as rescue medication.

Comparisons between the three genetic groups for the continuous variables (serum concentrations of oxycodone, noroxycodone, oxymorphone and noroxymorphone, and the oxymorphone/oxycodone ratio, pain intensity and cognitive function) were explored with analysis of variance (one-way ANOVA) and tested for homogeneity. For the variables where the overall F-test showed to be significant (*p* ≤ 0.05), the Games–Howell procedure [[30](#CR30)] was chosen for the post-hoc tests. The analyses were then repeated with non-genetic covariates previously found to influence the outcomes [[25](#CR25)]. For serum concentrations of oxycodone and the metabolites and the ratio oxymorphone/oxycodone these were: oxycodone total daily dose, use of CYP3A4 inhibitors or inducers, sex, time since last oxycodone dose, the number of medications other than opioids used in the last 24 h, albumin serum concentrations, use of steroids, BMI and glomerular filtration rate. Covariates in the analyses of pain intensity were oxymorphone serum concentrations, mixed pain, breakthrough pain, paracetamol medication, depression status, constipation status, female reproductive organ cancer and use of fluconazole. Covariates in the analyses of cognitive function were: age, Karnofsky and depression status, use of CYP2D6 inhibitors, steroid medication and breast cancer. Post-hoc ANCOVA comparisons between genetic groups were performed with Sidak [[31](#CR31)] corrected *p* values.

Comparisons between the three genetic groups and categorical data (nausea and tiredness) were explored by ordinal logistic regression without covariates. The analyses were then repeated with inclusion of non-genetic covariates previously known to influence the outcomes [[25](#CR25)]. For nausea these covariates were sex, depression status, prostate cancer or diagnosis of unknown origin, constipation status, use of anti-emetics and steroids. In the analysis of tiredness depression status, breast cancer, Karnofsky status and albumin status were included covariates. The statistical software SPSS for Windows v. 16.0 was used for all statistical analyses.

## Results

### Patients

We analysed 450 patients receiving oxycodone for cancer pain. Six percent (*n* = 27) were genotyped as poor metabolisers (PM), about 92% (*n* = 413) were extensive metabolisers (EM), while about 2% (*n* = 10) were genotyped as ultra-rapid metabolisers (URM). The distribution of the CYP2D6 genotypes is shown in Table [1](#Tab1). None of the patients had the *7 and *8 allelic variants. The allelic distributions followed the Hardy—Weinberg equation.

Descriptive data are shown in Table [2](#Tab2) and given as median (minimum to maximum) if not stated otherwise. Most of the demographic data were similar in the three genetic groups. However, the median Karnofsky performance status was 70% for EM and 50 and 55% for PM and URM respectively (*p* = 0.0004). Also, use of CYP3A4 inducer medications was statistically significantly different in PM (*n* = 0) compared with the other to genetic groups (*n* = 2 for both groups; *p* < 0.05).

### Table 2.

|   | PM (n = 27) | EM (n = 413) | URM (n = 10) |
| --- | --- | --- | --- |
| Gender: female/male (%) | 16/11 (59/41) | 180/233 (44/56) | 2/8 (20/80) |
| Age (years)a | 62 (19–84) | 62 (18–91) | 63 (44–81) |
| Karnofsky performance status (%)a* | 50 (20–90) | 70 (20–90) | 55 (30–70) |
| Body mass index (kg/m2)a | 23 (14–30) | 23 (14–41) | 23 (19–28) |
| Glomerular filtration rate (ml/min/1.73 m2)a | 77 (27–239) | 96 (24–261) | 115 (42–194) |
| Albumin serum (g/L)a | 31 (10–49) | 33 (11–91) | 29 (22–36) |
| Time since diagnosis (months)a | 31 (0–155) | 16 (0–286) | 31 (0–79) |
| Time since opioid treatment started (months)a | 1 (0–42) | 1 (0–97) | 2 (0–21) |
| Time since last dose (hours)a | 11 (1–13) | 10 (0.1–17) | 11 (0.8–12) |
| Number of medication ex. opioidsa | 7 (3–14) | 6 (0–17) | 9 (3–12) |
| CYP2D6 inhibitor medicationb | 2 | 35 | 1 |
| CYP3A4 inhibitor medicationb | 2 | 21 | 1 |
| CYP3A4 inducer medicationb* | 0 | 2 | 2 |
| Breakthrough pain: yes/no (%) | 18 / 9 (67/33) | 256/156 (62/38) | 8/2 (80/20) |
| Cancer diagnosis: |  |  |  |
| Gastrointestinal (inclusive pancreas, liver)c | 2 | 87 | 3 |
| Lung (inclusive mesothelioma)c | 6 | 70 | 1 |
| Prostatec | 4 | 70 | 3 |
| Other urologicalc | 0 | 26 | 1 |
| Breastc* | 8 | 56 | 0 |
| Female reproductive organsc | 1 | 33 | 0 |
| Haematologicalc | 1 | 29 | 0 |
| Head and neckc | 1 | 10 | 0 |
| Sarcomac | 1 | 8 | 1 |
| Skinc | 1 | 8 | 0 |
| Otherc | 2 | 23 | 1 |
| Unknown originc | 1 | 12 | 0 |
| More than one diagnosisc | 1 | 17 | 0 |

Table 2 Caption: Patient demographics for poor metabolisers (PM), extensive metabolisers (EM) and ultra-rapid metabolisers (URM )

#### Pharmacokinetics

Oxycodone total daily dose and serum concentrations for the three genetic groups are shown in Figs. [1](#Fig1) and [2](#Fig2).

![Fig. 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b747/3249195/63ee6f5449e8/228_2011_1093_Fig1_HTML.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3249195_228_2011_1093_Fig1_HTML.jpg)

Box and whisker plots of median oxycodone total daily dose (mg/24 h) with 25th and 75th percentiles among the three genetic groups poor metabolisers (PM, n = 27), extensive metabolisers (EM, n = 413) and ultra-rapid metabolisers (URM, n = 10). The whiskers represent the 5th and 95th percentiles

![Fig. 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b747/3249195/0f946ceacddc/228_2011_1093_Fig2_HTML.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3249195_228_2011_1093_Fig2_HTML.jpg)

Box and whisker plots of median serum concentrations (nMolar) of a oxycodone, b oxymorphone, c noroxycodone and d noroxymorphone with 25th and 75th percentiles between the three genetic groups poor metabolisers (PM, n = 27), extensive metabolisers (EM, n = 413) and ultra-rapid metabolisers (URM, n = 10). The whiskers represent the 5th and 95th percentile

Median oxycodone total daily dose were 80, 75 and 70 mg/24 h for the PM, EM, and URM respectively. Median serum concentrations of oxycodone were 110, 107 and 74 nM, noroxycodone 196, 101 and 118 nM, oxymorphone 0.2, 1.6 and 2.3 nM, and noroxymorphone 2.5, 18.0 and 34.0 nM for the PM, EM and URM respectively. There were no statistical differences among the three genetic groups with respect to oxycodone (F(2, 444) = 0.01, *p* = 0.99) and noroxycodone serum concentrations (F(2, 444) = 2.35, *p* = 0.10). There was a significant difference in serum concentrations of oxymorphone (F(2,444) = 30.79, *p* < 0.000) and noroxymorphone (F(2,444) = 43.43, *p *< 0.000) among the three genetic groups. The Games–Howell post-hoc analyses showed that there was a significant increase in serum concentrations of oxymorphone and noroxymorphone from PM to EM, and from PM to URM (all *p* < 0.001). Serum concentrations of oxymorphone were not statistically significantly different between EM and URM (*p* = 0.16). Noroxymorphone serum concentrations were statistically significantly lower (*p* = 0.05) in EM compared with URM, although the result was non-significant after analyses with covariates (*p* = 0.57); Table [3](#Tab3)).

| Dependent variablea | Groups compared | Mean difference between groups (MD) | Standard error MD | p value* | 95% CI for MD |
| --- | --- | --- | --- | --- | --- |
| Oxymorphone | PM vs EM | −0.84 | 0.15 | 0.000 | −1.21 −0.47 |
| PM vs URM | −1.12 | 0.20 | 0.000 | −1.62 −0.62 |  |
| EM vs URM | – | – | 0.16 | – |  |
| Noroxymorphone | PM vs EM | −1.04 | 0.18 | 0.000 | −1.50 −0.58 |
| PM vs URM | −1.38 | 0.22 | 0.000 | −1.91 −0.84 |  |
| EM vs URM | −0.34 | 0.12 | 0.05** | −0.68 −0.0005 |  |

Table Caption: Post-hoc analyses of variance (ANOVA) between the three genetic groups: poor metabolisers (PM), extensive metabolisers (EM) and ultra-rapid metabolisers (URM), for the outcomes that showed an overall statistical difference (F-test, both p = 0.000). The statistically significantly different groups are shown in italics

Except for the noroxymorphone serum concentrations, repeating all analyses with inclusion of covariates (ANCOVA analyses) gave similar results with respect to statistical significance (data not shown).

Median oxymorphone/oxycodone ratios were 0.0028, 0.0172 and 0.0244 for the PM, EM and URM respectively (Fig. [3](#Fig3)). There was a significant difference in the metabolic ratio oxymorphone/oxycodone (F(2,447) = 43,16 *p* < 0.000) among the three genetic groups. The Games–Howell post-hoc analyses showed that there was a significant increase in the ratio from PM to EM, and from PM to URM (all *p* < 0.000). The oxymorphone/oxycodone ratio was also statistically significant different between EM and URM (*p* = 0.017) in the ANOVA analysis. However, EM and URM (*p* = 0.121) were not statistically significant different after inclusion of the covariates “total daily oxycodone dose”, “use of CYP3A4 inducer drugs” and “number of co-medications used by the patient” (data not shown).

![Fig. 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b747/3249195/a342a06ec44c/228_2011_1093_Fig3_HTML.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3249195_228_2011_1093_Fig3_HTML.jpg)

Box and whisker plots of the median ratio of oxymorphone/oxycodone with 25th and 75th percentiles among the three genetic groups poor metabolisers (PM, n = 27), extensive metabolisers (EM, n = 413) and ultra-rapid metabolisers (URM, n = 10). The whiskers represent the 5th and 95th percentiles

#### Clinical outcomes

Median pain intensity was 4 on the NRS for PM and URM, and 3 for EM. The difference in pain intensity was non-significant between the groups (*p* = 0.8).

There were no difference between the groups with regard to tiredness (*p* = 0.7) and nausea (*p* = 0.6). All three genetic groups had a median score of 67 and 17 respectively, for the EORTC QLQ-C30 on tiredness and nausea. There were no differences in cognitive function among the groups (*p* = 0.8). PM and URM had a median cognitive function score on the Mini Mental State of 29, compared with a median of 28 for EM (Table [4](#Tab4)). PM had a median of 33, EM had median of 67 and URM had a median of 50 on the EORTC QLQ-C30 constipation scale (*p* = 0.9). All three genetic groups had a median score of 2 with regard to depression (data not shown).

|   | Poor metabolisers | Extensive metabolisers | Ultra-rapid metabolisers |
| --- | --- | --- | --- |
| Pain intensity | 4 (0–6) | 3 (0–10) | 4 (0–6) |
| Tiredness | 67 (0–100) | 67 (0–100) | 67 (0–100) |
| Nausea | 17 (0–83.33) | 17 (0–100) | 17 (0–50) |
| Cognitive function | 29 (20–30) | 28 (14–30) | 29 (20–30) |

Table Caption: Patient ratings of symptoms for the three genetic groups given as medians (minimum to maximum). Pain intensity from the Brief Pain Inventory, the tiredness and nausea score from the EORTC-QLQ-C30 and cognitive function from the Mini Mental State examine score. No statistically significant differences were found (one-way ANOVA and ordinal logistic regression analyses)

Seven EM and two URM used another regular opioid. The exclusion of these patients did not change the results (data not shown).

## Discussion

In this cohort of cancer patients it was observed that oxycodone metabolism, but not oxycodone efficacy, was influenced by *CYP2D6* genotypes.

CYP2D6 activity may have an impact on oxycodone efficacy because oxymorphone in relevant doses is an active analgesic [[32](#CR32)], and because noroxymorphone may exhibit an analgesic effect because of its abundance in serum and its μ-opioid receptor affinity [[33](#CR33)].

The PM had lower oxymorphone and noroxymorphone serum concentrations than the EM and URM, even when correcting for non-genetic covariates, previously shown to influence oxycodone pharmacokinetics [[34](#CR34)]. However, the observed difference in noroxymorphone concentrations between EM and URM was not observed in the analyses corrected for covariates, suggesting that the difference in noroxymorphone between the EM and URM was not related to *CYP2D6* genotypes, but caused by other factors.

Metabolic phenotyping by means of the oxymorphone/oxycodone ratio clearly separated the PM from the EM and URM in this sample of unselected, heterogeneous cancer patients as well. This confirms that CYP2D6 is the dominating metabolic pathway for the formation of oxymorphone from oxycodone. The observation that metabolic phenotyping did not separate EM from URM is also consistent with previous literature, as EM genotypes may display a metabolic pattern overlapping with URM genotypes [[35](#CR35)].

Significant differences in oxymorphone serum concentration levels for PM compared with EM were also demonstrated in a fairly large (*n *= 270) study in patients with post-operative pain [[36](#CR36)]. Moreover, Samer et al.’s [[37](#CR37)] study in healthy volunteers showed that PM had very low oxymorphone and noroxymorphone levels compared with EM and URM. Thus, all studies performed in humans consistently show that CYP2D6 genotypes alter the pharmacokinetics of oxycodone.

Despite the clear effects of *CYP2D6* genotypes on the pharmacokinetics of oxycodone, no difference was found between PM, EM and URM in comparing pain intensities, nausea, tiredness and cognitive function. Thus, this study suggests that *CYP2D6* genotyping and monitoring of oxycodone serum concentrations and its metabolites do not have any value in clinical routine practice. This is in accordance with the clinical study by Zwisler et al. [[36](#CR36)], who were unable to confirm an analgesic effect of oxymorphone or a difference in the efficacy of oxycodone between PM and EM, in 270 patients with post-operative pain treated with oxycodone. Further, no difference was found in the analgesic effect and adverse events between EM and URM patients with oxycodone administered for chronic non-malignant and malignant pain [[38](#CR38)].

In contrast, experimental pain studies in healthy volunteers [[39](#CR39), [40](#CR40)] observed differences among the three *CYP2D6* genotypes. In Zwisler et al.’s [[39](#CR39)] study there was a difference between EM and PM with regard to the analgesic effect of oxycodone on pain detection threshold, tolerance threshold and the cold pressor test. Samer et al. [[40](#CR40)] showed differences in pain tolerance- and subjective pain thresholds (URM > EM > PM) and differences between URM and EM in psychomotor tests (*p* < 0.05). However, an important shared limitation is that these two studies were single-dose oxycodone studies performed in healthy volunteers.

Our study suggests that oxymorphone and noroxymorphone do not contribute to the efficacy of oxycodone in a clinical setting with cancer patients. The reason for the lack of the pharmacodynamic effect of the potent compound oxymorphone could potentially be the very low level of this metabolite relative to oxycodone [[33](#CR33)]. The median oxymorphone serum concentrations in this study were 0.2, 1.5 and 3.1% of the median oxycodone serum concentrations in the PM, EM and URM respectively.

Median noroxymorphone serum concentrations constitute 2.3, 16.8 and 46% of the median oxycodone serum concentrations in the PM, EM and URM respectively. A difference in pain intensity or adverse events between PM and URM would be expected if noroxymorphone was an active metabolite, maybe also between PM and EM, because of the relatively large difference between the genotypes of noroxymorphone concentrations relative to oxycodone. This was not the case; there was no difference among PM, EM and URM with regard to effect or adverse events; thus, it seems unlikely that noroxymorphone is an important active metabolite of oxycodone.

The usage of CYP2D6 inhibitor medication was included in the covariate analyses of the efficacy of oxycodone. No association was found, which suggests that CYP2D6 inhibition does not affect the efficacy of oxycodone. The prevalence of co-medication with a CYP2D6 inhibitor was only about 8% and, therefore, a relevant difference could be undisclosed. However, this lack of impact from the use of CYP2D6 inhibitors is in accordance with other studies where inhibition of the CYP2D6 metabolic pathway with paroxetine or quinidine did not influence the efficacy of oxycodone in healthy volunteers [[40](#CR40)–[43](#CR43)], or patients with chronic pain [[38](#CR38)].

We recognise some limitations to this study. First, is the lack of observations before and after interventions. Therefore, conclusions about causal relationships should be applied with caution owing to its cross-sectional design. Second, with more than 80 known allelic variants of the *CYP2D6* gene, it was necessary to make a selection of which to study. We chose to use a panel of variant used by the Department of Pathology and Medical Genetics, St. Olav University Hospital, to determine PM, EM and URM. They have chosen variants that show clinically significant alteration of enzyme activity, omitting those that have no verified or insignificant effect on drug metabolism in vivo or those that are extremely rare. We chose to analyse for these routine allelic variants as the aim was to make judgements closely related to everyday clinical and laboratory practice. Finally, as in all clinical cancer pain cohorts the patients included in this study are heterogeneic with regard to characteristics that may affect pain intensity and other symptoms. In studies in healthy volunteers pain mechanisms are equal for all participants, and the participants are more homogeneous. However, experimental pain has little relevance in clinical practice, while our results reflect the clinical setting. Thus, in this cohort with chronic cancer pain, oxymorphone does not seem to contribute to the analgesic effect of oxycodone.

## Conclusion

Patients categorised as PMs of oxycodone have statistically significantly lower serum concentrations of oxymorphone and noroxymorphone and oxymorphone/oxycodone ratios than EM and URM. However, no difference was found among PM, EM and URM when comparisons of their pain intensities, nausea, tiredness and cognitive function were made. The *CYP2D6* genotype does not reflect oxycodone requirements and it is not associated with common adverse effects in this study of patients with cancer pain.

## Acknowledgements

We would like to thank the European Association for Palliative Care Research Network (EAPC-RN) for contributing to the organisation of the European Pharmacogenetic Opioid Study (EPOS). We also want to acknowledge the contribution of all investigators and each participating research centre: Andrew Davies, Sutton/Chelsea, UK; Augusto Caraceni, Alessandra Pigni, Milan, Italy; Danilo Miotti, Pavia, Italy; Eeva Salminen, Turku, Finland; Eriphili Argyra, Athens, Greece; Florian Strasser, St. Gallen, Switzerland; Irena Poviloniene, Vilnius, Lithuania; Jon Håvard Loge, Oslo, Norway; Kristin Bjordal, Oslo, Norway; Lucas Radbruch, Aachen, Germany; Marianne Kloke, Essen, Germany; Marco Maltoni, Forli, Italy; Per Sjøgren, Copenhagen, Denmark; Rainer Sabatowski, Dresden, Germany; Staffan Lundström, Stockholm, Sweden; Stein Kaasa, Trondheim, Norway; and Valgerdur Sigurdardottir, Reykjavik, Iceland. Gunnhild Jakobsen is thanked for organising the data collection and Turid Nilsen for organising shipping and storage of the biological samples. The Laboratory of Medical Genetics, Department of Pathology and Medical Genetics, St Olav’s University Hospital is thanked for providing chemicals and equipment needed for the genetic analyses. Hildegunn Pettersen is especially thanked for assistance and advice during the genetic analyses. This work was supported by grants from the Norwegian Research Council, the Norwegian Cancer Society, the EU 6th framework and Helse Midt-Norge RHF.

### Open Access

This article is distributed under the terms of the Creative Commons Attribution Noncommercial License which permits any noncommercial use, distribution, and reproduction in any medium, provided the original author(s) and source are credited.

## References

1. World Health Organization (2010) WHO Pain ladder. http://www.who.int/cancer/palliative/painladder/en/. Accessed March 2011  [http://www.who.int/cancer/palliative/painladder/en/](http://www.who.int/cancer/palliative/painladder/en/)

2. Jarlbaek L, Kehlet H, Sjogren P. Det legale opioidforbrug I Danmark (The licit opioid consumption in Denmark) Ugeskrift for Laeger. 2010;172(46):3173–3178.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/21073831/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ugeskrift%20for%20Laeger&title=Det%20legale%20opioidforbrug%20I%20Danmark%20(The%20licit%20opioid%20consumption%20in%20Denmark)&author=L%20Jarlbaek&author=H%20Kehlet&author=P%20Sjogren&volume=172&issue=46&publication_year=2010&pages=3173-3178&pmid=21073831&)

3. Hamunen K, Paakkari P, Kalso E. Trends in opioid consumption in the Nordic countries 2002–2006. Eur J Pain. 2009;13(9):954–962. doi: 10.1016/j.ejpain.2008.11.006.  [DOI](https://doi.org/10.1016/j.ejpain.2008.11.006) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19091608/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Pain&title=Trends%20in%20opioid%20consumption%20in%20the%20Nordic%20countries%202002%E2%80%932006&author=K%20Hamunen&author=P%20Paakkari&author=E%20Kalso&volume=13&issue=9&publication_year=2009&pages=954-962&pmid=19091608&doi=10.1016/j.ejpain.2008.11.006&)

4. Berg C, Furu K, Litleskare I, Rønning M, Sakshaug S, Selmer R, Skurtveit S, Strøm H. Reseptregisteret 2005–2009. In: Rønning M, editor. Legemiddelstatistikk 2010:2. Nydalen: Nasjonalt folkehelseinstitutt; 2010.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Legemiddelstatistikk%202010:2&author=C%20Berg&author=K%20Furu&author=I%20Litleskare&author=M%20R%C3%B8nning&author=S%20Sakshaug&publication_year=2010&)

5. Moore KA, Ramcharitar V, Levine B, Fowler D. Tentative identification of novel oxycodone metabolites in human urine. J Anal Toxicol. 2003;27(6):346–352. doi: 10.1093/jat/27.6.346.  [DOI](https://doi.org/10.1093/jat/27.6.346) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14516487/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Anal%20Toxicol&title=Tentative%20identification%20of%20novel%20oxycodone%20metabolites%20in%20human%20urine&author=KA%20Moore&author=V%20Ramcharitar&author=B%20Levine&author=D%20Fowler&volume=27&issue=6&publication_year=2003&pages=346-352&pmid=14516487&doi=10.1093/jat/27.6.346&)

6. Parkinson A, Ogilvie BW. Biotransformation of xenobiotics. In: Klaassen CD, editor. Casarett and Doull’s toxicology, vol 6. New York: McGraw-Hill; 2001. pp. 133–224.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Casarett%20and%20Doull%E2%80%99s%20toxicology,%20vol%206&author=A%20Parkinson&author=BW%20Ogilvie&publication_year=2001&)

7. Ball SE, Scatina J, Kao J, Ferron GM, Fruncillo R, Mayer P, Weinryb I, Guida M, Hopkins PJ, Warner N, Hall J. Population distribution and effects on drug metabolism of a genetic variant in the 5′ promoter region of CYP3A4. Clin Pharmacol Ther. 1999;66(3):288–294. doi: 10.1016/S0009-9236(99)70037-8.  [DOI](https://doi.org/10.1016/S0009-9236(99)70037-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10511065/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Population%20distribution%20and%20effects%20on%20drug%20metabolism%20of%20a%20genetic%20variant%20in%20the%205%E2%80%B2%20promoter%20region%20of%20CYP3A4&author=SE%20Ball&author=J%20Scatina&author=J%20Kao&author=GM%20Ferron&author=R%20Fruncillo&volume=66&issue=3&publication_year=1999&pages=288-294&pmid=10511065&doi=10.1016/S0009-9236(99)70037-8&)

8. Wandel C, Witte JS, Hall JM, Stein CM, Wood AJ, Wilkinson GR. CYP3A activity in African American and European American men: population differences and functional effect of the CYP3A4*1B5'-promoter region polymorphism. Clin Pharmacol Ther. 2000;68(1):82–91. doi: 10.1067/mcp.2000.108506.  [DOI](https://doi.org/10.1067/mcp.2000.108506) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10945319/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=CYP3A%20activity%20in%20African%20American%20and%20European%20American%20men:%20population%20differences%20and%20functional%20effect%20of%20the%20CYP3A4*1B5'-promoter%20region%20polymorphism&author=C%20Wandel&author=JS%20Witte&author=JM%20Hall&author=CM%20Stein&author=AJ%20Wood&volume=68&issue=1&publication_year=2000&pages=82-91&pmid=10945319&doi=10.1067/mcp.2000.108506&)

9. Spigset O. Cytokrom P-450-systemet (The cytochrome P-450 system) Tidsskr Nor Laegeforen. 2001;121(28):3296–3298.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/11826461/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Tidsskr%20Nor%20Laegeforen&title=Cytokrom%20P-450-systemet%20(The%20cytochrome%20P-450%20system)&author=O%20Spigset&volume=121&issue=28&publication_year=2001&pages=3296-3298&pmid=11826461&)

10. Zanger UM, Raimundo S, Eichelbaum M. Cytochrome P450 2D6: overview and update on pharmacology, genetics, biochemistry. Naunyn Schmiedebergs Arch Pharmacol. 2004;369(1):23–37. doi: 10.1007/s00210-003-0832-2.  [DOI](https://doi.org/10.1007/s00210-003-0832-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14618296/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Naunyn%20Schmiedebergs%20Arch%20Pharmacol&title=Cytochrome%20P450%202D6:%20overview%20and%20update%20on%20pharmacology,%20genetics,%20biochemistry&author=UM%20Zanger&author=S%20Raimundo&author=M%20Eichelbaum&volume=369&issue=1&publication_year=2004&pages=23-37&pmid=14618296&doi=10.1007/s00210-003-0832-2&)

11. Foster A, Mobley E, Wang Z. Complicated pain management in a CYP450 2D6 poor metabolizer. Pain Pract. 2007;7(4):352–356. doi: 10.1111/j.1533-2500.2007.00153.x.  [DOI](https://doi.org/10.1111/j.1533-2500.2007.00153.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17986163/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pain%20Pract&title=Complicated%20pain%20management%20in%20a%20CYP450%202D6%20poor%20metabolizer&author=A%20Foster&author=E%20Mobley&author=Z%20Wang&volume=7&issue=4&publication_year=2007&pages=352-356&pmid=17986163&doi=10.1111/j.1533-2500.2007.00153.x&)

12. Jannetto PJ, Bratanow NC. Utilization of pharmacogenomics and therapeutic drug monitoring for opioid pain management. Pharmacogenomics. 2009;10(7):1157–1167. doi: 10.2217/pgs.09.64.  [DOI](https://doi.org/10.2217/pgs.09.64) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19604091/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Utilization%20of%20pharmacogenomics%20and%20therapeutic%20drug%20monitoring%20for%20opioid%20pain%20management&author=PJ%20Jannetto&author=NC%20Bratanow&volume=10&issue=7&publication_year=2009&pages=1157-1167&pmid=19604091&doi=10.2217/pgs.09.64&)

13. Goryachkina K, Burbello A, Boldueva S, Babak S, Bergman U, Bertilsson L. CYP2D6 is a major determinant of metoprolol disposition and effects in hospitalized Russian patients treated for acute myocardial infarction. Eur J Clin Pharmacol. 2008;64(12):1163–1173. doi: 10.1007/s00228-008-0525-3.  [DOI](https://doi.org/10.1007/s00228-008-0525-3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18648788/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Clin%20Pharmacol&title=CYP2D6%20is%20a%20major%20determinant%20of%20metoprolol%20disposition%20and%20effects%20in%20hospitalized%20Russian%20patients%20treated%20for%20acute%20myocardial%20infarction&author=K%20Goryachkina&author=A%20Burbello&author=S%20Boldueva&author=S%20Babak&author=U%20Bergman&volume=64&issue=12&publication_year=2008&pages=1163-1173&pmid=18648788&doi=10.1007/s00228-008-0525-3&)

14. de Leon J, Dinsmore L, Wedlund P. Adverse drug reactions to oxycodone and hydrocodone in CYP2D6 ultrarapid metabolizers. J Clin Psychopharmacol. 2003;23(4):420–421. doi: 10.1097/01.jcp.0000085421.74359.60.  [DOI](https://doi.org/10.1097/01.jcp.0000085421.74359.60) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12920424/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Psychopharmacol&title=Adverse%20drug%20reactions%20to%20oxycodone%20and%20hydrocodone%20in%20CYP2D6%20ultrarapid%20metabolizers&author=J%20de%20Leon&author=L%20Dinsmore&author=P%20Wedlund&volume=23&issue=4&publication_year=2003&pages=420-421&pmid=12920424&doi=10.1097/01.jcp.0000085421.74359.60&)

15. Klepstad P, Fladvad T, Skorpen F, Bjordal K, Caraceni A, Dale O, Davies A, Kloke M, Lundström S, Maltoni M, Radbruch L, Sabatowski R, Sigurdardottir V, Strasser F, Fayers PM, Kaasa S. The influence from genetic variability on opioid use for cancer pain: a European study of 2294 cancer pain patients. Pain. 2011;152(5):1139–1145. doi: 10.1016/j.pain.2011.01.040.  [DOI](https://doi.org/10.1016/j.pain.2011.01.040) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21398039/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pain&title=The%20influence%20from%20genetic%20variability%20on%20opioid%20use%20for%20cancer%20pain:%20a%20European%20study%20of%202294%20cancer%20pain%20patients&author=P%20Klepstad&author=T%20Fladvad&author=F%20Skorpen&author=K%20Bjordal&author=A%20Caraceni&volume=152&issue=5&publication_year=2011&pages=1139-1145&pmid=21398039&doi=10.1016/j.pain.2011.01.040&)

16. Daut RL, Cleeland CS, Flanery RC. Development of the Wisconsin Brief Pain Questionnaire to assess pain in cancer and other diseases. Pain. 1983;17(2):197–210. doi: 10.1016/0304-3959(83)90143-4.  [DOI](https://doi.org/10.1016/0304-3959(83)90143-4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/6646795/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pain&title=Development%20of%20the%20Wisconsin%20Brief%20Pain%20Questionnaire%20to%20assess%20pain%20in%20cancer%20and%20other%20diseases&author=RL%20Daut&author=CS%20Cleeland&author=RC%20Flanery&volume=17&issue=2&publication_year=1983&pages=197-210&pmid=6646795&doi=10.1016/0304-3959(83)90143-4&)

17. Folstein MF, Folstein SE, McHugh PR. “Mini-mental state”: a practical method for grading the cognitive state of patients for the clinician. J Psychiatr Res. 1975;12(3):189–198. doi: 10.1016/0022-3956(75)90026-6.  [DOI](https://doi.org/10.1016/0022-3956(75)90026-6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1202204/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Psychiatr%20Res&title=%E2%80%9CMini-mental%20state%E2%80%9D:%20a%20practical%20method%20for%20grading%20the%20cognitive%20state%20of%20patients%20for%20the%20clinician&author=MF%20Folstein&author=SE%20Folstein&author=PR%20McHugh&volume=12&issue=3&publication_year=1975&pages=189-198&pmid=1202204&doi=10.1016/0022-3956(75)90026-6&)

18. Tombaugh TN, McIntyre NJ. The mini-mental state examination: a comprehensive review. J Am Geriatr Soc. 1992;40(9):922–935. doi: 10.1111/j.1532-5415.1992.tb01992.x.  [DOI](https://doi.org/10.1111/j.1532-5415.1992.tb01992.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1512391/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Am%20Geriatr%20Soc&title=The%20mini-mental%20state%20examination:%20a%20comprehensive%20review&author=TN%20Tombaugh&author=NJ%20McIntyre&volume=40&issue=9&publication_year=1992&pages=922-935&pmid=1512391&doi=10.1111/j.1532-5415.1992.tb01992.x&)

19. Anthony JC, LeResche L, Niaz U, von Korff MR, Folstein MF. Limits of the ‘Mini-Mental State’ as a screening test for dementia and delirium among hospital patients. Psychol Med. 1982;12(2):397–408. doi: 10.1017/S0033291700046730.  [DOI](https://doi.org/10.1017/S0033291700046730) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7100362/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Psychol%20Med&title=Limits%20of%20the%20%E2%80%98Mini-Mental%20State%E2%80%99%20as%20a%20screening%20test%20for%20dementia%20and%20delirium%20among%20hospital%20patients&author=JC%20Anthony&author=L%20LeResche&author=U%20Niaz&author=MR%20von%20Korff&author=MF%20Folstein&volume=12&issue=2&publication_year=1982&pages=397-408&pmid=7100362&doi=10.1017/S0033291700046730&)

20. Karnofsky DA, Abelmann WH, Craver LF, Burchenal JH. The use of the nitrogen mustards in the palliative treatment of carcinoma. Cancer. 1948;1(4):634–656. doi: 10.1002/1097-0142(194811)1:4<634::AID-CNCR2820010410>3.0.CO;2-L.  [DOI](https://doi.org/10.1002/1097-0142(194811)1:4&#x0003c;634::AID-CNCR2820010410&#x0003e;3.0.CO;2-L) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer&title=The%20use%20of%20the%20nitrogen%20mustards%20in%20the%20palliative%20treatment%20of%20carcinoma&author=DA%20Karnofsky&author=WH%20Abelmann&author=LF%20Craver&author=JH%20Burchenal&volume=1&issue=4&publication_year=1948&pages=634-656&doi=10.1002/1097-0142(194811)1:4&#x0003c;634::AID-CNCR2820010410&#x0003e;3.0.CO;2-L&)

21. Aaronson NK, Ahmedzai S, Bergman B, Bullinger M, Cull A, Duez NJ, Filiberti A, Flechtner H, Fleishman SB, de Haes JC. The European Organization for Research and Treatment of Cancer QLQ-C30: a quality-of-life instrument for use in international clinical trials in oncology. J Natl Cancer Inst. 1993;85(5):365–376. doi: 10.1093/jnci/85.5.365.  [DOI](https://doi.org/10.1093/jnci/85.5.365) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8433390/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Natl%20Cancer%20Inst&title=The%20European%20Organization%20for%20Research%20and%20Treatment%20of%20Cancer%20QLQ-C30:%20a%20quality-of-life%20instrument%20for%20use%20in%20international%20clinical%20trials%20in%20oncology&author=NK%20Aaronson&author=S%20Ahmedzai&author=B%20Bergman&author=M%20Bullinger&author=A%20Cull&volume=85&issue=5&publication_year=1993&pages=365-376&pmid=8433390&doi=10.1093/jnci/85.5.365&)

22. Fayers PM, Aaronson NK, Bjordal K, Groenvold M, Curran D, Bottomley A, on behalf of the EORTC Quality of Life Group . The EORTC QLQ C-30 Scoring Manual. 3. Brussels: European Organisation for Research and Treatment of Cancer; 2001.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=The%20EORTC%20QLQ%20C-30%20Scoring%20Manual&publication_year=2001&)

23. Levey AS, Eckardt KU, Tsukamoto Y, Levin A, Coresh J, Rossert J, De Zeeuw D, Hostetter TH, Lameire N, Eknoyan G. Definition and classification of chronic kidney disease: a position statement from Kidney Disease: Improving Global Outcomes (KDIGO) Kidney Int. 2005;67(6):2089–2100. doi: 10.1111/j.1523-1755.2005.00365.x.  [DOI](https://doi.org/10.1111/j.1523-1755.2005.00365.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15882252/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Kidney%20Int&title=Definition%20and%20classification%20of%20chronic%20kidney%20disease:%20a%20position%20statement%20from%20Kidney%20Disease:%20Improving%20Global%20Outcomes%20(KDIGO)&author=AS%20Levey&author=KU%20Eckardt&author=Y%20Tsukamoto&author=A%20Levin&author=J%20Coresh&volume=67&issue=6&publication_year=2005&pages=2089-2100&pmid=15882252&doi=10.1111/j.1523-1755.2005.00365.x&)

24. Levey AS, Coresh J, Greene T, Marsh J, Kusek J, Van Lente Medicine F. Estimating glomerular filtration rate with a new equation: MDRD4 revised equation for use with IDMS-traceable serum creatinine. J Am Soc Nephrol. 2005;11(Suppl):F-FC142.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Am%20Soc%20Nephrol&title=Estimating%20glomerular%20filtration%20rate%20with%20a%20new%20equation:%20MDRD4%20revised%20equation%20for%20use%20with%20IDMS-traceable%20serum%20creatinine&author=AS%20Levey&author=J%20Coresh&author=T%20Greene&author=J%20Marsh&author=J%20Kusek&volume=11&issue=Suppl&publication_year=2005&pages=F-FC142&)

25. Andreassen TN, Klepstad P, Davies A, Bjordal K, Lundstrom S, Kaasa S, Dale O. Influences on the pharmacokinetics of oxycodone: a multicentre cross-sectional study in 439 adult cancer patients. Eur J Clin Pharmacol. 2011;67(5):493–506. doi: 10.1007/s00228-010-0948-5.  [DOI](https://doi.org/10.1007/s00228-010-0948-5) | [PMC free article](/articles/PMC3076582/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21140139/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Clin%20Pharmacol&title=Influences%20on%20the%20pharmacokinetics%20of%20oxycodone:%20a%20multicentre%20cross-sectional%20study%20in%20439%20adult%20cancer%20patients&author=TN%20Andreassen&author=P%20Klepstad&author=A%20Davies&author=K%20Bjordal&author=S%20Lundstrom&volume=67&issue=5&publication_year=2011&pages=493-506&pmid=21140139&doi=10.1007/s00228-010-0948-5&)

26. Lovlie R, Daly AK, Molven A, Idle JR, Steen VM. Ultrarapid metabolizers of debrisoquine: characterization and PCR-based detection of alleles with duplication of the CYP2D6 gene. FEBS Lett. 1996;392(1):30–34. doi: 10.1016/0014-5793(96)00779-X.  [DOI](https://doi.org/10.1016/0014-5793(96)00779-X) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8769309/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=FEBS%20Lett&title=Ultrarapid%20metabolizers%20of%20debrisoquine:%20characterization%20and%20PCR-based%20detection%20of%20alleles%20with%20duplication%20of%20the%20CYP2D6%20gene&author=R%20Lovlie&author=AK%20Daly&author=A%20Molven&author=JR%20Idle&author=VM%20Steen&volume=392&issue=1&publication_year=1996&pages=30-34&pmid=8769309&doi=10.1016/0014-5793(96)00779-X&)

27. Steen VM, Andreassen OA, Daly AK, Tefre T, Borresen AL, Idle JR, Gulbrandsen AK. Detection of the poor metabolizer-associated CYP2D6(D) gene deletion allele by long-PCR technology. Pharmacogenetics. 1995;5(4):215–223. doi: 10.1097/00008571-199508000-00005.  [DOI](https://doi.org/10.1097/00008571-199508000-00005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8528268/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Detection%20of%20the%20poor%20metabolizer-associated%20CYP2D6(D)%20gene%20deletion%20allele%20by%20long-PCR%20technology&author=VM%20Steen&author=OA%20Andreassen&author=AK%20Daly&author=T%20Tefre&author=AL%20Borresen&volume=5&issue=4&publication_year=1995&pages=215-223&pmid=8528268&doi=10.1097/00008571-199508000-00005&)

28. Hersberger M, Marti-Jaun J, Rentsch K, Hanseler E. Rapid detection of the CYP2D6*3, CYP2D6*4, and CYP2D6*6 alleles by tetra-primer PCR and of the CYP2D6*5 allele by multiplex long PCR. Clin Chem. 2000;46(8 Pt 1):1072–1077.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/10926885/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Chem&title=Rapid%20detection%20of%20the%20CYP2D6*3,%20CYP2D6*4,%20and%20CYP2D6*6%20alleles%20by%20tetra-primer%20PCR%20and%20of%20the%20CYP2D6*5%20allele%20by%20multiplex%20long%20PCR&author=M%20Hersberger&author=J%20Marti-Jaun&author=K%20Rentsch&author=E%20Hanseler&volume=46&issue=8%20Pt%201&publication_year=2000&pages=1072-1077&pmid=10926885&)

29. Stuven T, Griese EU, Kroemer HK, Eichelbaum M, Zanger UM. Rapid detection of CYP2D6 null alleles by long distance- and multiplex-polymerase chain reaction. Pharmacogenetics. 1996;6(5):417–421. doi: 10.1097/00008571-199610000-00005.  [DOI](https://doi.org/10.1097/00008571-199610000-00005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8946473/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Rapid%20detection%20of%20CYP2D6%20null%20alleles%20by%20long%20distance-%20and%20multiplex-polymerase%20chain%20reaction&author=T%20Stuven&author=EU%20Griese&author=HK%20Kroemer&author=M%20Eichelbaum&author=UM%20Zanger&volume=6&issue=5&publication_year=1996&pages=417-421&pmid=8946473&doi=10.1097/00008571-199610000-00005&)

30. Games PA, Howell JF. Pairwise multiple comparison procedures with unequal N’s and/or variances: a Monte Carlo study. J Educ Stat. 1976;1:113–125. doi: 10.2307/1164979.  [DOI](https://doi.org/10.2307/1164979) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Educ%20Stat&title=Pairwise%20multiple%20comparison%20procedures%20with%20unequal%20N%E2%80%99s%20and/or%20variances:%20a%20Monte%20Carlo%20study&author=PA%20Games&author=JF%20Howell&volume=1&publication_year=1976&pages=113-125&doi=10.2307/1164979&)

31. Šidàk Z. Rectangular confidence region for the means of multivariate normal distributions. J Am Stat Assoc. 1967;62:626–633. doi: 10.2307/2283989.  [DOI](https://doi.org/10.2307/2283989) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Am%20Stat%20Assoc&title=Rectangular%20confidence%20region%20for%20the%20means%20of%20multivariate%20normal%20distributions&author=Z%20%C5%A0id%C3%A0k&volume=62&publication_year=1967&pages=626-633&doi=10.2307/2283989&)

32. Mayyas F, Fayers P, Kaasa S, Dale O. A systematic review of oxymorphone in the management of chronic pain. J Pain Symptom Manage. 2010;39(2):296–308. doi: 10.1016/j.jpainsymman.2009.07.010.  [DOI](https://doi.org/10.1016/j.jpainsymman.2009.07.010) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20152592/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pain%20Symptom%20Manage&title=A%20systematic%20review%20of%20oxymorphone%20in%20the%20management%20of%20chronic%20pain&author=F%20Mayyas&author=P%20Fayers&author=S%20Kaasa&author=O%20Dale&volume=39&issue=2&publication_year=2010&pages=296-308&pmid=20152592&doi=10.1016/j.jpainsymman.2009.07.010&)

33. Lalovic B, Kharasch E, Hoffer C, Risler L, Liu-Chen LY, Shen DD. Pharmacokinetics and pharmacodynamics of oral oxycodone in healthy human subjects: role of circulating active metabolites. Clin Pharmacol Ther. 2006;79(5):461–479. doi: 10.1016/j.clpt.2006.01.009.  [DOI](https://doi.org/10.1016/j.clpt.2006.01.009) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16678548/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Pharmacokinetics%20and%20pharmacodynamics%20of%20oral%20oxycodone%20in%20healthy%20human%20subjects:%20role%20of%20circulating%20active%20metabolites&author=B%20Lalovic&author=E%20Kharasch&author=C%20Hoffer&author=L%20Risler&author=LY%20Liu-Chen&volume=79&issue=5&publication_year=2006&pages=461-479&pmid=16678548&doi=10.1016/j.clpt.2006.01.009&)

34. Andreassen TN, Klepstad P, Davies A, Bjordal K, Lundström S, Kaasa S, Dale O. Influences on the pharmacokinetics of oxycodone: a multicentre cross-sectional study in 439 adult cancer patients. Eur J Clin Pharm. 2010;67:493–506. doi: 10.1007/s00228-010-0948-5.  [DOI](https://doi.org/10.1007/s00228-010-0948-5) | [PMC free article](/articles/PMC3076582/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21140139/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Clin%20Pharm&title=Influences%20on%20the%20pharmacokinetics%20of%20oxycodone:%20a%20multicentre%20cross-sectional%20study%20in%20439%20adult%20cancer%20patients&author=TN%20Andreassen&author=P%20Klepstad&author=A%20Davies&author=K%20Bjordal&author=S%20Lundstr%C3%B6m&volume=67&publication_year=2010&pages=493-506&pmid=21140139&doi=10.1007/s00228-010-0948-5&)

35. Lovlie R, Daly AK, Matre GE, Molven A, Steen VM. Polymorphisms in CYP2D6 duplication-negative individuals with the ultrarapid metabolizer phenotype: a role for the CYP2D6*35 allele in ultrarapid metabolism? Pharmacogenetics. 2001;11(1):45–55. doi: 10.1097/00008571-200102000-00006.  [DOI](https://doi.org/10.1097/00008571-200102000-00006) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11207030/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Polymorphisms%20in%20CYP2D6%20duplication-negative%20individuals%20with%20the%20ultrarapid%20metabolizer%20phenotype:%20a%20role%20for%20the%20CYP2D6*35%20allele%20in%20ultrarapid%20metabolism?&author=R%20Lovlie&author=AK%20Daly&author=GE%20Matre&author=A%20Molven&author=VM%20Steen&volume=11&issue=1&publication_year=2001&pages=45-55&pmid=11207030&doi=10.1097/00008571-200102000-00006&)

36. Zwisler ST, Enggaard TP, Mikkelsen S, Brosen K, Sindrup SH. Impact of the CYP2D6 genotype on post-operative intravenous oxycodone analgesia. Acta Anaesthesiol Scand. 2010;54(2):232–240. doi: 10.1111/j.1399-6576.2009.02104.x.  [DOI](https://doi.org/10.1111/j.1399-6576.2009.02104.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19719813/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Acta%20Anaesthesiol%20Scand&title=Impact%20of%20the%20CYP2D6%20genotype%20on%20post-operative%20intravenous%20oxycodone%20analgesia&author=ST%20Zwisler&author=TP%20Enggaard&author=S%20Mikkelsen&author=K%20Brosen&author=SH%20Sindrup&volume=54&issue=2&publication_year=2010&pages=232-240&pmid=19719813&doi=10.1111/j.1399-6576.2009.02104.x&)

37. Samer CF, Daali Y, Wagner M, Hopfgartner G, Eap CB, Rebsamen MC, Rossier MF, Hochstrasser D, Dayer P, Desmeules JA. The effects of CYP2D6 and CYP3A activities on the pharmacokinetics of immediate release oxycodone. Br J Pharmacol. 2010;160(4):907–918. doi: 10.1111/j.1476-5381.2010.00673.x.  [DOI](https://doi.org/10.1111/j.1476-5381.2010.00673.x) | [PMC free article](/articles/PMC2935997/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20590587/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Pharmacol&title=The%20effects%20of%20CYP2D6%20and%20CYP3A%20activities%20on%20the%20pharmacokinetics%20of%20immediate%20release%20oxycodone&author=CF%20Samer&author=Y%20Daali&author=M%20Wagner&author=G%20Hopfgartner&author=CB%20Eap&volume=160&issue=4&publication_year=2010&pages=907-918&pmid=20590587&doi=10.1111/j.1476-5381.2010.00673.x&)

38. Lemberg KK, Heiskanen TE, Neuvonen M, Kontinen VK, Neuvonen PJ, Dahl ML, Kalso EA. Does co-administration of paroxetine change oxycodone analgesia: an interaction study in chronic pain patients. Scand J Pain. 2010;1(1):24–33. doi: 10.1016/j.sjpain.2009.09.003.  [DOI](https://doi.org/10.1016/j.sjpain.2009.09.003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29913934/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Scand%20J%20Pain&title=Does%20co-administration%20of%20paroxetine%20change%20oxycodone%20analgesia:%20an%20interaction%20study%20in%20chronic%20pain%20patients&author=KK%20Lemberg&author=TE%20Heiskanen&author=M%20Neuvonen&author=VK%20Kontinen&author=PJ%20Neuvonen&volume=1&issue=1&publication_year=2010&pages=24-33&pmid=29913934&doi=10.1016/j.sjpain.2009.09.003&)

39. Zwisler ST, Enggaard TP, Noehr-Jensen L, Pedersen RS, Mikkelsen S, Nielsen F, Brosen K, Sindrup SH. The hypoalgesic effect of oxycodone in human experimental pain models in relation to the CYP2D6 oxidation polymorphism. Basic Clin Pharmacol Toxicol. 2009;104(4):335–344. doi: 10.1111/j.1742-7843.2009.00378.x.  [DOI](https://doi.org/10.1111/j.1742-7843.2009.00378.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19281600/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Basic%20Clin%20Pharmacol%20Toxicol&title=The%20hypoalgesic%20effect%20of%20oxycodone%20in%20human%20experimental%20pain%20models%20in%20relation%20to%20the%20CYP2D6%20oxidation%20polymorphism&author=ST%20Zwisler&author=TP%20Enggaard&author=L%20Noehr-Jensen&author=RS%20Pedersen&author=S%20Mikkelsen&volume=104&issue=4&publication_year=2009&pages=335-344&pmid=19281600&doi=10.1111/j.1742-7843.2009.00378.x&)

40. Samer CF, Daali Y, Wagner M, Hopfgartner G, Eap CB, Rebsamen MC, Rossier MF, Hochstrasser D, Dayer P, Desmeules JA. Genetic polymorphisms and drug interactions modulating CYP2D6 and CYP3A activities have a major effect on oxycodone analgesic efficacy and safety. Br J Pharmacol. 2010;160(4):919–930. doi: 10.1111/j.1476-5381.2010.00673.x.  [DOI](https://doi.org/10.1111/j.1476-5381.2010.00673.x) | [PMC free article](/articles/PMC2935998/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20590588/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Pharmacol&title=Genetic%20polymorphisms%20and%20drug%20interactions%20modulating%20CYP2D6%20and%20CYP3A%20activities%20have%20a%20major%20effect%20on%20oxycodone%20analgesic%20efficacy%20and%20safety&author=CF%20Samer&author=Y%20Daali&author=M%20Wagner&author=G%20Hopfgartner&author=CB%20Eap&volume=160&issue=4&publication_year=2010&pages=919-930&pmid=20590588&doi=10.1111/j.1476-5381.2010.00673.x&)

41. Heiskanen T, Olkkola KT, Kalso E. Effects of blocking CYP2D6 on the pharmacokinetics and pharmacodynamics of oxycodone. Clin Pharmacol Ther. 1998;64(6):603–611. doi: 10.1016/S0009-9236(98)90051-0.  [DOI](https://doi.org/10.1016/S0009-9236(98)90051-0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9871425/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Effects%20of%20blocking%20CYP2D6%20on%20the%20pharmacokinetics%20and%20pharmacodynamics%20of%20oxycodone&author=T%20Heiskanen&author=KT%20Olkkola&author=E%20Kalso&volume=64&issue=6&publication_year=1998&pages=603-611&pmid=9871425&doi=10.1016/S0009-9236(98)90051-0&)

42. Gronlund J, Saari TI, Hagelberg NM, Neuvonen PJ, Olkkola KT, Laine K. Exposure to oral oxycodone is increased by concomitant inhibition of CYP2D6 and 3A4 pathways, but not by inhibition of CYP2D6 alone. Br J Clin Pharmacol. 2010;70(1):78–87. doi: 10.1111/j.1365-2125.2010.03653.x.  [DOI](https://doi.org/10.1111/j.1365-2125.2010.03653.x) | [PMC free article](/articles/PMC2909810/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20642550/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=Exposure%20to%20oral%20oxycodone%20is%20increased%20by%20concomitant%20inhibition%20of%20CYP2D6%20and%203A4%20pathways,%20but%20not%20by%20inhibition%20of%20CYP2D6%20alone&author=J%20Gronlund&author=TI%20Saari&author=NM%20Hagelberg&author=PJ%20Neuvonen&author=KT%20Olkkola&volume=70&issue=1&publication_year=2010&pages=78-87&pmid=20642550&doi=10.1111/j.1365-2125.2010.03653.x&)

43. Kummer O, Hammann F, Moser C, Schaller O, Drewe J, Krahenbuhl S. Effect of the inhibition of CYP3A4 or CYP2D6 on the pharmacokinetics and pharmacodynamics of oxycodone. Eur J Clin Pharmacol. 2011;67(1):63–71. doi: 10.1007/s00228-010-0893-3.  [DOI](https://doi.org/10.1007/s00228-010-0893-3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20857093/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Clin%20Pharmacol&title=Effect%20of%20the%20inhibition%20of%20CYP3A4%20or%20CYP2D6%20on%20the%20pharmacokinetics%20and%20pharmacodynamics%20of%20oxycodone&author=O%20Kummer&author=F%20Hammann&author=C%20Moser&author=O%20Schaller&author=J%20Drewe&volume=67&issue=1&publication_year=2011&pages=63-71&pmid=20857093&doi=10.1007/s00228-010-0893-3&)
