# Novel variants of major drug-metabolising enzyme genes in diverse African populations and their predicted functional effects

## Metadata
**Authors:** Alice Matimba, Jurgen Del-Favero, Christine Van Broeckhoven, Collen Masimirembwa
**Journal:** Human Genomics
**Date:** 2009 Jan 1
**DOI:** [10.1186/1479-7364-3-2-169](https://doi.org/10.1186/1479-7364-3-2-169)
**PMID:** 19164093
**PMCID:** PMC3525272
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3525272/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC3525272/pdf/1479-7364-3-2-169.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC3525272/pdf/1479-7364-3-2-169.pdf)

## Abstract

Pharmacogenetics enables personalised therapy based on genetic profiling and is increasingly applied in drug discovery. Medicines are developed and used together with pharmacodiagnostic tools to achieve desired drug efficacy and safety margins. Genetic polymorphism of drug-metabolising enzymes such as cytochrome P450s (CYPs) and N-acetyltransferases (NATs) has been widely studied in Caucasian and Asian populations, yet studies on African variants have been less extensive. The aim of the present study was to search for novel variants of CYP2C9, CYP2C19, CYP2D6 and NAT2 genes in Africans, with a particular focus on their prevalence in different populations, their relevance to enzyme functionality and their potential for personalised therapy. Blood samples from various ethnic groups were obtained from the AiBST Biobank of African Populations. The nine exons and exon-intron junctions of the CYP genes and exon 2 of NAT2 were analysed by direct DNA sequencing. Computational tools were used for the identification, haplotype analysis and prediction of functional effects of novel single nucleotide polymorphisms (SNPs). Novel SNPs were discovered in all four genes, grouped to existing haplotypes or assigned new allele names, if possible. The functional effects of non-synonymous SNPs were predicted and known African-specific variants were confirmed, but no significant differences were found in the frequencies of SNPs between African ethnicities. The low prevalence of our novel variants and most known functional alleles is consistent with the generally high level of diversity in gene loci of African populations. This indicates that profiles of rare variants reflecting interindividual variability might become the most relevant pharmacodiagnostic tools explaining Africans' diversity in drug response.

Keywords: pharmacogenetics, cytochrome P450, N-acetyltransferase, single nucleotide polymorphisms, African populations

## Introduction

Pharmacogenetics describes patients' variation in response to therapy due to genetic factors. Pharmacogenetics-based therapy is of special interest for drugs with narrow therapeutic indices, where impairment in metabolic activity might cause difficulties in dose adjustment, resulting in increased susceptibility to adverse drug reactions (ADRs). The cytochrome P450 enzymes (CYPs) metabolise more than 80 per cent of clinically used drugs and most of them exhibit functionally significant genetic polymorphisms. The genes encoding CYP2C9, CYP2C19 and CYP2D6, as well as N- acetyltransferase 2 (NAT2), have been most extensively studied across various populations [[1](#B1),[2](#B2)]. The presence of novel variants remains to be ascertained in African populations, however, particularly rare (< 1 per cent frequency) single nucleotide polymorphisms (SNPs), which may contribute to a better understanding of interindividual variation in the metabolism of drugs.

The human *CYP2C *subfamily contains four highly homologous genes -- *2C8, 2C9, 2C18 *and *2C19 *-- which are located in a cluster on chromosome 10 [[3](#B3)]. CYP2C9 is the main CYP2C enzyme, constituting 20 per cent of total human liver microsomal P450 content [[4](#B4)].

*CYP2C9 *and *CYP2C19 *genes each contain nine exons and encode proteins of 490 amino acids in length. Although these genes are highly homologous (92 per cent), the enzymes differ in terms of substrate specificities [[5](#B5)]. Major variations in the occurrence of polymorphisms in both *CYP2C9 *and *CYP2C19 *genes have been reported in various populations. *CYP2C9 *variants *CYP2C9***2 *and *CYP2C9***3 *are the most common and occur at frequencies of 0.11 and 0.08, respectively, in Caucasians [[6](#B6)]. Population-based pharmacokinetics-pharmacodynamics modelling of their effects has been explored for revising labels of CYP2C9 substrate drugs [[7](#B7)].

Testing for *CYP2C9 *genotypes can be used to predict the starting dose of the anticoagulant drug warfarin to avoid excessive bleeding episodes [[8](#B8)]. Other drugs affected by *CYP2C9 *polymorphism are the antidiabetic agents glipizide and tolbutamide, the antiepileptic agent phenytoin, the antihypertensive drug losartan and non-steroidal anti-inflammatory drugs (NSAIDs) such as ibuprofen and diclofenac [[9](#B9)].

CYP2C19 metabolises omeprazole, diazepam and proguanil to a major extent. The common allelic variants, such as *CYP2C19***2 *and *CYP2C19***3*, cause reduced enzyme activity and contribute to the poor metabolism of substrate drugs [[10](#B10)]. A polymorphism in the promoter region has, however, been associated with increased enzyme activity [[11](#B11)]. Individuals carrying this variant may therefore require a higher dosage in order to achieve the therapeutic effect.

CYP2D6 metabolises a wide range of drugs, such as antiarrhythmic agents, tricyclic antidepressants, neuroleptics and anti-cancer agents [[12](#B12)]. *CYP2D6 *is the most polymorphic *CYP*, with alleles causing a spectrum of phenotypic responses. The presence of multiple copies of the gene results in individuals described as ultra-rapid metabolisers. For example, individuals carrying duplicated or multi-duplicated active *CYP2D6 *genes are very common among Ethiopians, compared with Caucasian, Oriental and other Black populations [[13](#B13)]. By contrast, whole gene deletions causing poor metaboliser phenotypes, have been observed across all populations. The African-specific alleles *CYP2D6***17 *and *CYP2D6***29 *cause reduced enzyme activity; individuals homozygous for these alleles are classified as intermediate metabolisers. Overall, Africans metabolise CYP2D6 substrates at a slower rate than Caucasians owing to the higher prevalence of these reduced-function alleles [[14](#B14)].

So far, NAT2 has been found to comprise 19 major known haplotypes. Important drugs metabolised by this enzyme include the anti-tuberculosis drug isoniazid and the antibiotic co-trimoxazole. Some polymorphisms of *NAT2 *have been shown to affect the acetylation of these drugs and this may result in toxic side effects [[15](#B15),[16](#B16)]. The most commonly known alleles are *NAT2***5, NAT2***6, NAT2***7 *and the African-specific *NAT2***14*. In addition, other SNPs have been discovered and are awaiting characterisation of their phenotypic effects.

In clinical pharmacogenetics, we aim to optimise therapeutic outcome by prescribing drugs to patients at doses that are predicted to be efficacious and safe. Knowledge of the types of genetic variants of major drug-metabolising enzymes and their frequency in the population is therefore important for the design and deployment of pharmacodiagnostic tools to guide drug prescription. Only a few studies on genotype-phenotype relationships of drug effects have been carried out in African populations [[16](#B16)-[20](#B20)]. Therefore, limited knowledge of polymorphisms and their impact in Africans may underestimate the importance of clinical applications of pharmacogenetics. Here, we report novel variants of the *CYP2C9, CYP2C19, CYP2D6 *and *NAT2 *genes found in African populations and their predicted functional effects.

## Materials and methods

### DNA samples

The study was carried out according to the Declaration of Helsinki (2000) of the World Medical Association and was approved by the Ethical Review Boards of Kenya, Nigeria, Tanzania and Zimbabwe. Informed consent was obtained from volunteers of the following ethnic groups: Hausa (20), Ibo (20), Luo (30), Maasai (13), San (40), Shona (23), Venda (9), Yoruba (20) and Tanzanian Mixed Bantu (12). Ethnicity was assigned based on the submission that parents and grandparents of the volunteers were of the same self-identified ethnic group. The exact numbers of samples analysed per gene are shown in Tables S1-S4 (Tables [2](#T2)-[5](#T5)) in the Appendix. DNA was extracted from whole blood samples stored in the AiBST Biobank of African Populations [[21](#B21)] using the QIAamp DNA Blood Mini Kit (Qiagen, KJ Venlo, The Netherlands).

### Table S1.

| NC_000010.9 | mRNAposition | SNP | mRNAfeature | Effect | dbSNP | Hausa(13) | Luo(12) | Maasai(11) | San(13) | Shona(23) | Venda(9) | TZ Bantu(12) | Total(93) |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| 96829291 | -375 | T > C | 5' UTR |   | rs9332103 | 0.04 | nd | 0 | 0 | 0 | nd | nd | 0.01 |
| 96829916 | 251 | T > C | Intron |   | rs9332104 | 0.08 | 0.10 | 0.27 | 0.12 | 0.22 | 0.17 | 0.11 | 0.16 |
| 96833076 | 3411 | T > C | Intron |   | rs9332120 | 0 | 0.10 | 0 | 0.04 | 0.13 | 0.17 | 0.14 | 0.08 |
| 96833152 | 3487 | A > G | Intron | splice site | rs12769205 | 0 | 0.04 | 0 | 0.17 | 0.11 | 0.06 | 0.09 | 0.07 |
| 96833165 | 3499 | T > A | Intron | splice site | rs9332121 | 0 | 0 | 0 | 0 | 0.03 | 0 | 0 | 0.01 |
| 96838628 | 8963 | T > C | Intron |   | nrs | 0 | 0.05 | 0 | 0 | 0 | 0 | 0 | 0.01 |
| 96838697 | 9032 | G > C | Intron |   | nrs | 0 | 0.10 | 0.14 | 0.15 | 0.15 | 0.13 | 0.14 | 0.12 |
| 96838734 | 9069 | G > A | Intron |   | Novel | 0 | 0.05 | 0.05 | 0 | 0 | 0 | 0.05 | 0.02 |
| 96839116 | 9451 | T > C | Intron |   | rs17443251 | 0 | 0.05 | 0 | 0 | 0 | 0 | 0.06 | 0.01 |
| 96839976 | 10311 | A > G | Intron |   | rs9332129 | 0 | 0.17 | 0.15 | 0.15 | 0.14 | 0.13 | 0.19 | 0.13 |
| 96840012 | 10347 | T > C | Intron |   | Novel | 0 | 0 | 0 | 0 | 0.03 | 0 | 0 | 0.01 |
| 96840200 | 10535 | A > G | Exon 5 | H251R (*9) | rs2256871 | 0.18 | 0.17 | 0.05 | 0.15 | 0.11 | 0.06 | 0 | 0.11 |
| 96840266 | 10601 | wt > delA | Exon 5 | K273 fs (*6) | nrs | 0.04 | 0 | 0 | 0 | 0 | 0.07 | 0 | 0.01 |
| 96863014 | 33349 | A > G | Intron |   | rs9332172 | 0.17 | 0.23 | 0.23 | 0.15 | 0.28 | 0.25 | 0.50 | 0.26 |
| 96863323 | 33658 | A > G | Intron |   | rs9332174 | 0.13 | 0.14 | 0.27 | 0.12 | 0.20 | 0.19 | 0.10 | 0.17 |
| 96872080 | 42415 | C > T | Intron |   | Novel | 0 | 0 | 0.06 | 0 | 0 | 0 | 0 | 0.01 |
| 96872134 | 42469 | T > C | Intron |   | rs9332197 | 0 | 0 | 0.05 | 0 | 0 | 0 | 0 | 0.01 |
| 96872184 | 42519 | T > C | Exon 7 | I327T (*31) | Novel | 0.04 | 0 | 0 | 0 | 0 | 0 | 0 | 0.01 |
| 96872284 | 42619 | G > C | Exon 7 | D360E (*5) | rs28371686 | 0 | 0 | 0 | 0 | 0.02 | 0.06 | 0 | 0.01 |
| 96877210 | 47545 | A > T | Intron |   | rs9332230 | 0 | 0 | 0 | 0 | 0 | 0 | 0.05 | 0.01 |
| 96877258 | 47593 | T > C | Intron |   | rs9332232 | 0.07 | 0.14 | 0.05 | 0.04 | 0.18 | 0.17 | 0.05 | 0.11 |
| 96877304 | 47639 | C > T | Intron |   | rs2298037 | 0 | 0 | 0 | 0 | 0.03 | 0.06 | 0 | 0.01 |
| 96879721 | 50056 | A > T | Intron |   | rs1934969 | 0.67 | 0.50 | 0.50 | 0.40 | 0.24 | 0.13 | 0.32 | 0.38 |
| 96879790 | 50125 | C > T | Intron |   | Novel | 0 | 0 | 0 | 0 | 0 | 0 | 0.05 | 0.01 |
| 96879861 | 50196 | C > T | Exon 9 | A441A | rs2017319 | 0.04 | 0.14 | 0.05 | 0.04 | 0.20 | 0.17 | 0.05 | 0.10 |
| 96879959 | 50294 | A > G | Exon 9 | N474S | Novel | 0 | 0.05 | 0 | 0 | 0 | 0 | 0.00 | 0.01 |
| 96879963 | 50298 | A > T | Exon 9 | G475G | rs1057911 | 0 | 0 | 0 | 0 | 0 | 0 | 0.05 | 0.01 |
| 96880006 | 50341 | G > T | Exon 9 | V490F (*32) | Novel | 0 | 0 | 0 | 0 | 0 | 0 | 0.05 | 0.01 |
| 96880078 | 50413 | C > T | 3' UTR |   | rs9332240 | 0 | 0.05 | 0 | 0 | 0.03 | 0 | 0.00 | 0.01 |
| 96880099 | 50434 | C > T | 3' UTR |   | rs9332241 | 0.08 | 0.05 | 0 | 0 | 0.02 | 0.13 | 0.14 | 0.05 |
| 96880166 | 50501 | C > T | 3' UTR |   | rs9332243 | 0 | 0.05 | 0 | 0 | 0.03 | 0.00 | 0.00 | 0.01 |

Table Caption: CYP2C9 single nucleotide polymorphism (SNP) frequencies

### Table S4.

| NC_000008.9 | mRNAposition | SNP | Effect | dbSNP | Hausa(20) | Yoruba(20) | Ibo(19) | Luo(16) | Maasai(12) | San(40) | Total(127) |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| 8950 | 191 | G > A | R64Q (*14) | rs1801279 | 0.03 | 0.08 | 0.13 | 0.20 | 0.08 | 0.09 | 0.10 |
| 9041 | 282 | C > T | Y94Y | rs1041983 | 0.4 | 0.44 | 0.55 | 0.44 | 0.38 | 0.29 | 0.39 |
| 9100 | 341 | T > C | I114T (*5) | rs1801280 | 0.33 | 0.14 | 0.34 | 0.27 | 0.5 | 0.2 | 0.27 |
| 9162 | 403 | C > G | L135V | nrs | 0 | 0.03 | 0 | 0.03 | 0 | 0 | < 0.01 |
| 9231 | 472 | A > C | I158L | Novel | 0 | 0 | 0 | 0.03 | 0 | 0 | < 0.01 |
| 9240 | 481 | C > T | L161L | rs1799929 | 0.25 | 0.14 | 0.34 | 0.27 | 0.46 | 0.14 | 0.24 |
| 9348 | 589 | C > T | R197X | Novel | 0 | 0 | 0 | 0 | 0 | 0.01 | < 0.01 |
| 9349 | 590 | G > A | R197Q (*6) | rs1799930 | 0.32 | 0.33 | 0.29 | 0.30 | 0.25 | 0.20 | 0.27 |
| 9400 | 641 | C > T | T214I | Novel | 0 | 0 | 0 | 0.03 | 0 | 0 | < 0.01 |
| 9442 | 683 | C > T | P228L | nrs | 0 | 0 | 0 | 0.03 | 0 | 0 | < 0.01 |
| 9525 | 766 | A > G | K256E | nrs | 0 | 0 | 0 | 0 | 0 | 0.03 | 0.01 |
| 9562 | 803 | A > G | K268R | rs1208 | 0.37 | 0.39 | 0.40 | 0.44 | 0.54 | 0.43 | 0.42 |
| 9568 | 809 | T > C | I270T | Novel | 0 | 0 | 0 | 0 | 0 | 0.13 | 0.04 |
| 9597 | 838 | G > A | V280M | nrs | 0.06 | 0.03 | 0.05 | 0.03 | 0 | 0 | 0.02 |
| 9616 | 857 | G > A | G286E (*7) | rs1799931 | 0.03 | 0.03 | 0.03 | 0.03 | 0.04 | 0.01 | 0.02 |

Table Caption: N-acetyltransferase 2 single nucleotide polymorphism (SNP) frequencies

### PCR and sequencing

Primers were designed using SNPBox and Primer3 software [[22](#B22),[23](#B23)]. Their specificity for each gene studied was confirmed by a BLAST analysis search and comparison of genomic sequences in the National Center for Biotechnology Information (NCBI) databases [http://blast.ncbi.nlm.nih.gov/Blast.cgi](http://blast.ncbi.nlm.nih.gov/Blast.cgi). Identical primers were used for the polymerase chain reaction (PCR) and sequencing, except where otherwise stated (Table S5 (Table [6](#T6))). First-step exon amplification mixtures (20 μl) contained 1x TiTaq buffer, 0.25 mM deoxyribonucleotide triphosphates, 0.5 μM of each primer and 0.25 units of TiTaq and DNA template (5 ng/μl). For PCR, an initial denaturation at 94°C for two minutes was followed by 35 cycles at 94°C for 30 seconds, 59°C for 30 seconds and 72°C for 60 seconds. Sequencing reactions were started at 96°C for one minute followed by 25 cycles at 96°C for ten seconds, 50°C for five seconds and 60°C for four minutes, and resolved on an ABI 3730 DNA Analyzer (Applied Biosystems, Brussels, Belgium).

### Table S5.

| Gene | Exon | First PCR primers | Sequencing primers |
| --- | --- | --- | --- |
| CYP2C9 | 5' UTR | cyp2C9-5'FLF ATCCTCAACTCAGTATGTCAGC | cyp2C9-5'FLSF1 ATCCTCAACTCAGTATGTCAGC |
|   |   | cyp2C9-5'FLR ATCACCTAGGTCCACTATATGC | cyp2C9-5'FLSR1 ACCTTTACCATTAAACCCCC |
|   |   |   | cyp2C9-5'FLSF2 CAATTCCTGCCTTCAGGA |
|   |   |   | cyp2C9-5'FLSR2 AAGGACTTTGACCCACTGAT |
|   | 1 | cyp2C9-1F GGAATGTACAGAGTGGACAATGG |   |
|   |   | cyp2C9-1R GATCCCACAATACCTTACCATTTAC |   |
|   | 2&3 | cyp2C9-2&3F |   |
|   |   | GACCTGCTGAATATGTTGATGTG | cyp2C9-2SF TCTTGAACTCCTGACCTTGT |
|   |   | cyp2C9-2&3R CCCGCTTCACATGAGCTAAC | cyp2C9-2SR GGAGCTCTGTAAGTCTCTGT |
|   |   |   | cyp2C9-3SF AGGAGTTTTCTGGAAGAGG |
|   |   |   | cyp2C9-3SR GGAAAAACACTGCTCTTTAACTC |
|   | 4* | cyp2C9-4F CAGCTAGGTTGTAATGGTCAACTC | *** |
|   |   | cyp2C9-4R GCTAATGGGCTTAGAAATCAGG |   |
|   | 5* | cyp2C9-5F TCATCTGGTTAGAATTGATCCTCTG | *** |
|   |   | cyp2C9-5R GCTATTAACTACCGCCTCAACTTC |   |
|   | 6* | cyp2C9-6F GAGGAAATGGACCTAGAGACCTTC | *** |
|   |   | cyp2C9-6R CCCATTGTAATCACCATTAGTTTG |   |
|   | 7* | cyp2C9-7F GTGCATCTGTAACCATCCTCTCT | *** |
|   |   | cyp2C9-7R |   |
|   |   | CAGACACTAGGACCTGTTACAAACC |   |
|   | 8* | cyp2C9-8F AGAAGGTTGCATCCAAGTATCC | *** |
|   |   | cyp2C9-8R GAGTTCTTGGGTACCTCACTGGT |   |
|   | 9 | cyp2c9-9F CTCATCCATCCATTCATTCATG | cyp2c9-9SF CTCATCCATCCATTCATTCATG |
|   |   | cyp2c9-9R CTCTAACACTCACCCAAAATAGC | cyp2c9-9SR CGAATGTTCACTAGATCTTCAG |
|   |   |   | cyp2c9-9S2F CTGCAGCTCTCTTTCCTC |
|   |   |   | cyp2c9-9S2FR CTCTAACACTCACCCAAAATAGC |
| CYP2C19 |   |   |   |
|   | 1 | cyp2C19-1F CAATTATGACGGTGCATTGG | *** |
|   |   | cyp2C19-1R CACTTCCCTTACTGTTTACCCTCA |   |
|   | 2&3 | cyp2C19-2&3F | cyp2C19-2SF AATTCAGAAATATTTGAGCCTGTGTG |
|   |   | GTTCTTGAAGCTGGGTATTTGTC | cyp2C19-2SR GGTTTTTCTCAACTCCTCCACAA |
|   |   | cyp2C19-2&3R | cyp2C19-3SF GCCTGGGATCTCCCTCCTAGTTT |
|   |   | AGCAAAGTTCAGGAGAACATAGG | cyp2C19-2&3R AAGCAAAGTTCAGGAGAACATAGG |
|   | 4 | cyp2C19-4F | *** |
|   |   | CAGCTAGGCTGTAATTGTTAATTCG |   |
|   |   | cyp2C19-4R |   |
|   |   | GAGTAATGGAAGACTCCAAAGTGC |   |
|   | 5 | cyp2C19-5F TTCAATTTCAGAGGCTGCTTG | *** |
|   |   | cyp2C19-5R |   |
|   |   | CTATGATGCTTACTGGATATTCATGC |   |
|   | 6 | cyp2C19-6F | *** |
|   |   | CAGCATATAAACAGAGCCAAAGAC |   |
|   |   | cyp2C19-6R |   |
|   |   | ACACCATTAAATTGGGACAGATTAC |   |
|   | 7 | cyp2C19-7F | *** |
|   |   | CCTAGCTTAAGGCACAGTTACACA |   |
|   |   | cyp2C19-7R |   |
|   |   | GAAAGACTCAAGGTGTCAAGATGTC |   |
|   | 8 | cyp2C19-8F | *** |
|   |   | GCCTTAAGCTCATGCCTCTTATTAC |   |
|   |   | cyp2C19-8R |   |
|   |   | GGCAGAATTCAACCAACCTATACTT |   |
|   | 9 | cyp2C19-9F TCATTGTTTAGTTGCCTATCCATC | *** |
|   |   | cyp2C19-9R CCATCTTCACCTTTGTCCTTTC |   |
| CYP2D6 |   |   |   |
|   | 1&2 | cyp2D6ex1_2F ACCAGGCCCCTCCACCGG | CYP2D6ex1_2F ACCAGGCCCCTCCACCGG |
|   |   | cyp2D6ex1_2R CTCTCTGCCCAGCTCGG | cyp2D6ex1SR |
|   |   |   | GTTTCACCCACCACCCATGTTT |
|   |   |   | cyp2D6ex2SF |
|   |   |   | CTTCCACCTGCTCACTCCTGGTA |
|   |   |   | cyp2D6ex2SR |
|   |   |   | CCTCCCTAGTGCAGGTGGTTTCT |
|   | 3&4 | cyp2D6ex3_4F ATTTCCCAGCTGGAATCC | cyp2D6ex3_4SF |
|   |   |   | GAGCATAGGGTTGGAGTGGGTG |
|   |   | cyp2D6ex3_4R GAGACTCCTCGGTCTCTC | cyp2D6ex3_4R GAGACTCCTCGGTCTCTC |
|   | 5&6 | cyp2D6ex5_6F GCCTGAGACTTGTCCAGG | cyp2D6ex5_6F GCCTGAGACTTGTCCAGG |
|   |   | cyp2D6ex5_6R CCGGCCCTGACACTCCTTCT | cyp2D6ex5_6R CCGGCCCTGACACTCCTTCT |
|   | 7,8,9 | cyp2D6ex7_9F GGATCCTGTAAGCCTGACCTC | cyp2D6ex7_9F GGATCCTGTAAGCCTGACCTC |
|   |   | cyp2D6ex7_9R | cyp2D6ex7SR |
|   |   | ACTGAGCCCTGGGAGGTAGGTAG | GTGGTGGCATTGAGGACTAGGTG |
|   |   |   | cyp2D6ex8SF |
|   |   |   | GTCCAGAGTATAGGCAGGGCTGG |
|   |   |   | cyp2D6ex8SR |
|   |   |   | AGCACAAAGCTCATAGGGGGATG |
|   |   |   | cyp2D6ex9SF |
|   |   |   | CTTCCTCTTCTTCACCTCCCTGC |
|   |   |   | cyp2D6ex9SR |
|   |   |   | AATATGGGCCTCCAGGCTGAGT |
| NAT2 | 2 | NAT2ex2F GAAGCATATTTTGAAAGAATTGG | NAT2ex2F GAAGCATATTTTGAAAGAATTGG |
|   |   | NAT2ex2R GCATTTTAAGGATGGCCTGT | NAT2SF1 TGCCAAAGAAGAAACACCAA |
|   |   |   | NAT2SR2 ACCTCGAACAATTGAAGATTTTGA |
|   |   |   | NAT2ex2R GCATTTTAAGGATGGCCTGT |

Table Caption: Polymerase chain reaction and sequencing primers

### Data analysis

Identification of SNPs was carried out using the novoSNP v2.1.9 software package [[24](#B24)]. Reference sequences were [NC_000010.9](https://www.ncbi.nlm.nih.gov/nuccore/NC_000010.9) for *CYP2C9*, [NC_000010.9](https://www.ncbi.nlm.nih.gov/nuccore/NC_000010.9) for *CYP2C19*, [M33388](https://www.ncbi.nlm.nih.gov/nuccore/M33388) for *CYP2D6 *and [NC_000008.9](https://www.ncbi.nlm.nih.gov/nuccore/NC_000008.9) for *NAT2*. All identified SNPs were compared with the NCBI Single Nucleotide Polymorphism database (dbSNP) [[25](#B25)]. As SNPs can cause the introduction of pre-microRNA (miRNA) sites, this was included as part of the annotation in the novoSNP analysis procedure. Frequencies of SNPs were calculated using Genepop [[26](#B26)].

HaploView v3.31 [[27](#B27)] was used to determine haplotypes from sequence genotype data. Linkage disequilibrium plots were generated to assess the extent to which SNPs were likely to be linked and hence likely to occur on the same haplotype with logarithm of odds (LOD) score > 3. The HaploView Tagger tool was used to estimate which SNPs were likely to be tagged by a single SNP in a predicted haplotype (threshold *r*^2 ^> 0.8).

### Prediction of functional effects of non-synonymous SNPs

Functional effects of non-synonymous SNPs were predicted using the Polyphen prediction programme [[28](#B28)] based on position-specific independent counts (PSIC) scores of multiple sequence alignments, as well as structural information, if available. The programme predicts the functional effects of SNPs based on occurrence in active/binding sites or in transmembrane regions, interference with disulphide or other bonds, compatibility with homologous sequences at that position, as well as mapping to known three-dimensional protein structures or validated homology models. Protein sequence accession numbers were obtained from Swiss-Prot [[29](#B29)] as CYP2C9: [P11712](https://www.uniprot.org/uniprotkb/P11712); CYP2C19: [P33261](https://www.uniprot.org/uniprotkb/P33261); CYP2D6: [P10635](https://www.uniprot.org/uniprotkb/P10635); NAT2: [P11245](https://www.uniprot.org/uniprotkb/P11245).

### Allele nomenclature

Allele nomenclature is assigned according to the Human Cytochrome P450 (CYP) Allele Nomenclature Committee [[30](#B30)] and the Arylamine N- acetyltransferase Gene Nomenclature Committee [[31](#B31)]. Alleles, as borne by specific SNPs, are assigned numbers -- for example, *CYP2C9***9 *to define the 10535A > G mutation, the presence of which results in the amino acid change H251R.

## Results

In our African populations, novel and known SNPs were found in all drug-metabolising enzyme genes studied (Tables S1-S4 (Tables [2](#T2)-[5](#T5))). Novel SNPs for *CYP2C9, CYP2C19 *and *CYP2D6 *were grouped and assigned to haplotypes or groups of other known mutations if possible (Table [1](#T1)). We mainly looked at the non-synonymous SNPs, since these are used to determine the eventual assignment of new functional alleles. *CYP2C9 *42519T > C (I327T) and 50341G > T (V490F) were assigned the new allele names *CYP2C9***31 *and *CYP2C9***32*, respectively. For *CYP2C19*, the 17869G > C/80161A > G (R186P/I331V) combination was assigned the new allele name *CYP2C19***22*. It was not possible to assign new haplotypes/alleles for *CYP2C9 *50294A > G (N474S) and *CYP2C19 *12690G > A (V113I) because their linkage with other alleles such as *CYP2C9***9 *(10535A > G; H251R) and *CYP2C19***2 *(19154G > A; P227P), respectively, could not be excluded. Known mutations also had some synonymous SNPs, as well as non-coding SNPs, grouped to them (eg *CYP2C9***9, CYP2C19***12 *and *CYP2C19***13*). It appears that the novel *CYP2D*61608 G > A (V119M) SNP is found on the known *CYP2D6***29 *allele, which is defined by 1659G > A (V136M) and 3183G > A (V338M). This haplotype group was therefore assigned the new name *CYP2D6***70*. New alleles were not assigned for *CYP2D6 *1621G > T (R123L) and 4057G > A (G445E), since further work is required fully to establish the haplotypes. The novel *NAT2 *SNPs did not appear to be linked to any other SNPs. HaploView-determined tag SNPs for *NAT2 *were used to determine the major haplotype frequencies (Figure [1](#F1)).

### Table 1.

| Gene | SNP grouping† | cDNA position | Amino acid change | Functional effect prediction (PSIC score) |
| --- | --- | --- | --- | --- |
| CYP2C9 | 42519T > C (*31) | 980 | I327T | Possible functional damage (2.761) |
|   |   |  |  |  |
|   | 50294A > G | 1421 | N474S | No functional damage (0.162) |
|   |   |  |  |  |
|   | 47545A > T |   |   |   |
|   | 50298A > T |   |   |   |
|   | 50341G > T (*32) | 1468 | V490F | Possible functional damage (1.806) |
|   |   |  |  |  |
|   | 10535A > G (*9) | 752 | H251R | Possible functional damage (2.239) |
|   | 50196C > T | 1323 | A441A |   |
| CYP2C19 | 12122G > A |   |   |   |
|   | 12690G > A | 337 | V113I | No functional damage (0.198) |
|   |   |  |  |  |
|   | 57453G > C |   |   |   |
|   | 90533C > T |   |   |   |
|   | 57575T > C |   |   |   |
|   | 87290C > T (*13) | 1228 | R410C |   |
|   |   |  |  |  |
|   | 90209A > C (*12) | 1473 | X491C | 26 extra amino acids |
|   | 90302C > T |   |   |   |
|   |   |  |  |  |
|   | 17869G > C (*22) | 557 | R186P | Possible functional damage (3.159) |
|   | 80161G > A | 991 | I331V |   |
|   |   |  |  |  |
| CYP2D6 | -175G > A |   |   |   |
|   | 310G > T |   |   |   |
|   | 843T > G |   |   |   |
|   | 1608G > A (*70) |   | V119M | No functional damage (0.054) |
|   | 1659G > A |   | V136M | Functional damage - reduced enzyme activity |
|   | 1661G > C |   |   |   |
|   | 3183G > A |   | V338M | Functional damage - reduced enzyme activity |
|   | 3384A > C |   |   |   |
|   | 4180G > C |   | S486T | No functional damage (0.267) |
|   | 4722T > G |   |   |   |
|   |   |  |  |  |
|   | 214G > C |   |   |   |
|   | 223C > G |   |   |   |
|   | 227T > C |   |   |   |
|   | 843T > G |   |   |   |
|   | 1621G > T |   | R123L | No functional damage (1.236) |
|   | 1661G > C |   |   |   |
|   | 2850C > T |   | R296C | No functional damage (0.254) |
|   | 3384A > C |   |   |   |
|   | 3584G > A |   |   |   |
|   | 3790C > T |   |   |   |
|   | 4180G > C |   | S486T | No functional damage (0.267) |
|   |   |  |  |  |
|   | 843T > G |   |   |   |
|   | 1661G > C |   |   |   |
|   | 2850C > T |   | R296C | No functional damage (0.254) |
|   | 3384A > C |   |   |   |
|   | 4057G > A |   | G445E | Possible functional damage, contact with functional site (3.063) |
|   | 4180G > C |   | S486T | No functional damage (0.267) |
|   |   |  |  |  |
| NAT2 | 10542A > C | 472 | I158L | No functional damage (0.615) |
|   | 10659C > T | 589 | R197X | No protein expressed |
|   | 10711C > T | 641 | T214I | Possible functional damage, involved in ligand binding (1.257) |
|   | 10879T > C | 809 | I270T | No functional damage (0.526) |

Table 1 Caption: Grouping of novel SNPs and functional effect prediction

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3694/3525272/888ec658e137/1479-7364-3-2-169-1.jpg)

N-acetyltransferase 2 haplotypes constructed from sequence and genotype data from the total population studied (n = 127). (a) Linkage disequilibrium plot with the genomic positions indicated at the top. In yellow are the tag single nucleotide polymorphisms (SNPs) which define the major known haplotypes and are able to capture other SNPs within the same haplotype. The amino acid changes at the various positions are shown. (b) Haplotype frequencies. Haplotypes and phenotypes (acetylators) were assigned according to Consensus Arylamine N-Acetyltransferase (NAT) Gene Nomenclature [31].

In *CYP2C9 *(Table S1 (Table [2](#T2))), three out of six non-synonymous SNPs -- 42519T > C (I327T), 50294A > G (N474S) and 50341G > T (V490F) -- were novel. Of these, I327T and V490F changes are predicted to have a functional effect (Table [1](#T1)); however, further inference of these amino acid changes with crystal structure information [[32](#B32)] and Gotoh's sequence alignments [[33](#B33)] indicates that they may not influence substrate recognition and binding. The most common non-synonymous *CYP2C9 *allele in this study was *CYP2C9***9 *(10535A > G; H251R), which is predicted to be damaging to enzyme function, although phenotypic studies in African individuals have shown no effect on the metabolism of the antiepileptic drug phenytoin. By contrast, the other known non-synonymous SNPs, such as *CYP2C9***5 *(42619C > G; D360E) and *CYP2C9***6 *(10601delA; K273fs) (Table S1 (Table [2](#T2))), did cause reduced enzyme activity [[17](#B17)].

The two novel non-synonymous SNPs discovered in *CYP2C19 *(Table S1 (Table [2](#T2))), 12690G > A (V113I) in exon 3 and 17869G > C (R186P) in exon 4, seem to cause very different effects on enzyme function, according to the physicochemical character of their amino acid changes (Table [1](#T1)). Whereas the effect of V113I may be negligible, the change from the basic arginine to proline at position 186 seems to be functionally damaging, as predicted (PSIC score = 3.159).

Three novel non-synonymous SNPs were found in *CYP2D6*: 1608G > A (V119M), 1621G > T (R123L) and 4057G > A (G445E) (Table S3([4](#T4))). Whereas the V119M and the R123L changes were predicted to have no effect on enzyme function (Table [1](#T1)), they are located in the substrate recognition site SRS1. The G445E substitution may be functionally important (PSIC score = 3.063) owing to its close proximity to the 443 site, which is critical for the heme ligand binding in this enzyme according to the crystal structure [[34](#B34)]. Consistent with other African data,[[35](#B35),[36](#B36)] the most common *CYP2D6 *haplotypes contributing to the variability of drug response were *CYP2D6***2 *(2850C > T; R296C and 4180G > C; S486T), *CYP2D6***17 *(1023C > T; T107I) and *CYP2D6***29 *(1659G > A; V136M and 3183G > A; V338M) (Table S3 (Table [4](#T4))).

### Table S3.

| M33388 | mRNAposition | SNP | mRNAfeature | Effect | dbSNP | Hausa(20) | Yoruba(20) | Ibo(20) | Luo(29) | Maasai(13) | Shona(15) | Venda(9) | TZ Bantu(10) | Total(136) |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| 1444 | -175 | G > A | 5' UTR |   | rs1080993 | 0.05 | 0.12 | 0.31 | 0.22 | 0.05 | 0.11 | 0.06 | 0.30 | 0.17 |
| 1469 | -150 | C > T | 5' UTR |   | nrs | 0.09 | 0.03 | 0 | 0 | 0 | 0 | 0 | 0 | 0.01 |
| 1534 | -85 | T > C | 5' UTR |   | nrs | 0.04 | 0 | 0 | 0 | 0 | 0.03 | 0 | 0 | 0.01 |
| 1577 | -42 | wt > insG | 5' UTR |   | rs28371695 | 0.19 | 0.35 | 0.13 | 0.21 | 0.05 | 0.20 | 0.13 | 0.10 | 0.19 |
| 1696 | 77 | G > A | Exon 1 | R26H (*43) | rs28371696 | 0.04 | 0 | 0.03 | 0 | 0 | 0.03 | 0 | 0.05 | 0.02 |
| 1701 | 82 | C > T | Exon 1 | R28C (*22) | nrs | 0 | 0 | 0 | 0 | 0 | 0 | 0 | 0.05 | < 0.01 |
| 1719 | 100 | C > T | Exon 1 | P34S (*10) | rs1065852 | 0.15 | 0.12 | 0.10 | 0.09 | 0.05 | 0 | 0.19 | 0.10 | 0.10 |
| 1833 | 214 | G > C | Intron |   | rs1080995 | 0.50 | 0.41 | 0.27 | 0.36 | 0.45 | 0.57 | 0.17 | 0.43 | 0.38 |
| 1840 | 221 | C > A | Intron |   | rs1080996 | 0.50 | 0.41 | 0.27 | 0.38 | 0.45 | 0.64 | 0.30 | 0.43 | 0.40 |
| 1842 | 223 | C > G | Intron |   | rs1080997 | 0.50 | 0.41 | 0.27 | 0.36 | 0.45 | 0.56 | 0.20 | 0.38 | 0.38 |
| 1846 | 227 | T > C | Intron |   | rs1080998 | 0.50 | 0.41 | 0.27 | 0.36 | 0.45 | 0.50 | 0.25 | 0.50 | 0.38 |
| 1851 | 232 | G > C | Intron |   | rs1080999 | 0.50 | 0.41 | 0.30 | 0.36 | 0.50 | 0.70 | 0.25 | 0.75 | 0.42 |
| 1852 | 233 | A > C | Intron |   | rs1080999 | 0.50 | 0.41 | 0.27 | 0.38 | 0.45 | 0.67 | 0.25 | 0.50 | 0.40 |
| 1864 | 245 | A > G | Intron |   | rs1081000 | 0.50 | 0.41 | 0.27 | 0.31 | 0.45 | 0.70 | 0.25 | 0.67 | 0.39 |
| 1929 | 310 | G > T | Intron |   | rs28371699 | 0 | 0.27 | 0.07 | 0.25 | 0.31 | 0.25 | 0 | nd | 0.18 |
| 2273 | 654 | C > T | Intron |   | Novel | nd | 0 | 0 | 0.07 | nd | 0.08 | 0.07 | 0.07 | 0.07 |
| 2365 | 746 | C > G | Intron |   | nrs | nd | nd | nd | 0.36 | nd | 0.40 | 0.31 | 0.50 | 0.40 |
| 2462 | 843 | T > G | Intron |   | rs28371702 | 0.14 | 0.40 | 0.20 | 0.33 | 0.38 | 0.33 | 0.13 | 0.20 | 0.29 |
| 2625 | 1006 | C > T | Exon 2 | R101R | Novel | 0.00 | 0.00 | 0.00 | 0.00 | 0.00 | 0.00 | 0.00 | 0.05 | 0.01 |
| 2642 | 1023 | C > T | Exon 2 | T107I (*17) | rs28371706 | 0.20 | 0.20 | 0.22 | 0.22 | 0.17 | 0.20 | 0.19 | 0.15 | 0.20 |
| 2658 | 1039 | C > T | Exon 2 | F112F | rs1081003 | 0.00 | 0.13 | 0.13 | 0.04 | 0.00 | 0.00 | 0.13 | 0.05 | 0.06 |
| 2686 | 1067 | T > G | Intron |   | Novel | 0.13 | 0.13 | 0.19 | 0.04 | 0.08 | 0.07 | 0.19 | 0.10 | 0.11 |
| 3227 | 1608 | G > A | Exon 3 | V119M (*70) | Novel | 0 | 0 | 0 | 0.02 | 0 | 0 | 0 | 0 | < 0.01 |
| 3240 | 1621 | G > T | Exon 3 | R123L | Novel | 0 | 0 | 0 | 0 | 0 | 0 | 0 | 0.05 | < 0.01 |
| 3278 | 1659 | G > A | Exon 3 | V136M (*29) | rs1058164 | 0.11 | 0.10 | 0.28 | 0.24 | 0.04 | 0.17 | 0.06 | 0.25 | 0.17 |
| 3280 | 1661 | G > C | Exon 3 | V136V | rs28371708 | 0.29 | 0.43 | 0.35 | 0.32 | 0.46 | 0.37 | 0.33 | 0.30 | 0.35 |
| 3335 | 1716 | G > A | Exon 3 | E155K (*45) | rs28371710 | 0.08 | 0 | 0 | 0.09 | 0.04 | 0 | 0.17 | 0 | 0.05 |
| 3465 | 1846 | G > A | Intron | Splicing defect (*4) | nrs | 0.03 | 0.08 | 0.08 | 0.04 | 0.04 | 0.00 | 0.00 | 0.05 | 0.04 |
| 3483 | 1863_1864 | ins(TTTCGCCCC)X2 | Exon 4 | 174_175ins(FRP)X2 | nrs | 0 | 0 | 0 | 0.04 | 0.08 | 0 | 0 | 0 | 0.02 |
| 3485 | 1866 | C > T | Exon 4 | N175N | nrs | 0 | 0 | 0 | 0.02 | 0.00 | 0 | 0 | 0 | < 0.01 |
| 3488 | 1869 | T > C | Exon 4 | G176G | nrs | 0 | 0.03 | 0 | 0.00 | 0.00 | 0 | 0 | 0 | < 0.01 |
| 3617 | 1998 | T > C | Exon 4 | F219F | novel | nd | nd | 0 | 0.00 | nd | 0.03 | 0 | 0.05 | 0.02 |
| 4194 | 2575 | C > A | Exon 5 | P267P | nrs | nd | nd | nd | 0.05 | nd | 0.03 | 0.22 | 0 | 0.07 |
| 4221 | 2602 | G > T | Exon 5 | L276L | novel | nd | nd | nd | 0.05 | nd | 0 | 0.06 | 0 | 0.02 |
| 4280 | 2661 | G > A | Intron |   | nrs | nd | nd | nd | 0.05 | nd | 0.03 | 0.11 | 0.05 | 0.06 |
| 4379 | 2760 | T > A | Intron |   | Novel | nd | nd | nd | 0.00 | nd | 0.10 | 0.06 | 0 | 0.04 |
| 4469 | 2850 | C > T | Exon 6 | R296C (*2) | nrs | nd | nd | nd | 0.55 | nd | 0.63 | 0.44 | 0.65 | 0.58 |
| 4607 | 2988 | G > A | Intron |   | nrs | nd | nd | nd | 0.00 | nd | 0.03 | 0 | 0 | 0.01 |
| 4802 | 3183 | G > A | Exon 7 | V338M (*29) | nrs | 0.13 | 0.10 | 0.29 | 0.20 | 0.04 | 0.17 | 0.06 | 0.13 | 0.16 |
| 4873 | 3254 | T > C | Exon 7 | H361H | rs2743457 | 0.09 | 0.00 | 0.00 | 0.07 | 0.08 | 0 | 0.13 | 0 | 0.04 |
| 4880 | 3259_3260 | wt > insTG | Exon 7 | 375 fs (*42) | nrs | 0 | 0 | 0 | 0 | 0 | 0.03 | 0 | 0 | < 0.01 |
| 5003 | 3384 | A > C | Intron |   | nrs | 0.30 | 0.45 | 0.34 | 0.28 | 0.42 | 0.37 | 0.25 | 0.38 | 0.65 |
| 5016 | 3397 | C > A | Intron |   | novel | 0 | 0 | 0 | 0 | 0 | 0 | 0.06 | 0 | < 0.01 |
| 5180 | 3561 | G > C | Intron |   | novel | 0 | 0 | 0 | 0.02 | 0 | 0 | 0.06 | 0.06 | 0.01 |
| 5201 | 3582 | A > G | Intron |   | nrs | 0.08 | 0.11 | 0.11 | 0.09 | 0.04 | 0.00 | 0.13 | 0.00 | 0.08 |
| 5203 | 3584 | G > A | Intron |   | nrs | 0.54 | 0.34 | 0.26 | 0.43 | 0.46 | 0.47 | 0.44 | 0.44 | 0.41 |
| 5326 | 3707 | G > A | Intron |   | nrs | 0 | 0 | 0.03 | 0 | 0 | 0.03 | 0 | 0 | 0.01 |
| 5349 | 3721 | wt > delGT | Intron |   | nrs | 0 | 0.03 | 0 | 0 | 0 | 0 | 0 | 0 | < 0.01 |
| 5409 | 3790 | C > T | Intron | Splice site | nrs | 0.53 | 0.34 | 0.26 | 0.44 | 0.54 | 0.47 | 0.44 | 0.44 | 0.42 |
| 5472 | 3853 | G > A | Exon 8 | E410K (*27) | nrs | 0 | 0 | 0 | 0.06 | 0.04 | 0 | 0 | 0.06 | 0.02 |
| 5652 | 4033 | C > T | Intron | Splice site | Novel | 0 | 0 | 0 | 0.02 | 0 | 0 | 0.06 | 0.06 | 0.01 |
| 5676 | 4057 | G > A | Exon 9 | G445E | Novel | 0 | 0 | 0 | 0.04 | 0 | 0 | 0 | 0 | 0.01 |
| 5799 | 4180 | G > C | Exon 9 | S486T | rs1135850 | 0.68 | 0.55 | 0.66 | 0.72 | 0.63 | 0.63 | 0.75 | 0.67 | 0.66 |
| 6013 | 4394 | wt > delAG | 3' UTR |   | Novel | 0 | 0 | 0 | 0 | 0 | 0.10 | 0.06 | 0 | 0.02 |
| 6020 | 4401 | C > T | 3' UTR |   | nrs | 0.09 | 0.10 | 0.11 | 0.07 | 0.04 | 0 | 0.25 | 0.06 | 0.08 |
| 6100 | 4481 | G > A | 3' UTR |   | nrs | 0.12 | 0.08 | 0.03 | 0.11 | 0.21 | 0.23 | 0.07 | 0.19 | 0.12 |
| 6275 | 4656 | wt > delACA | 3' UTR |   | nrs | 0.44 | 0.09 | 0.03 | 0.23 | 0.31 | 0.08 | 0.33 | 0.33 | 0.20 |
| 6341 | 4722 | T > G | 3' UTR |   | nrs | 0.63 | 0.57 | 0.73 | 0.57 | nd | nd | nd | 0.25 | 0.58 |

Table Caption: CYP2D6 single nucleotide polymorphism (SNP) frequencies

Four novel amino acid-changing SNPs were detected in *NAT2 *(Table S4 (Table [5](#T5))). The 641C > T (T214I) was predicted to have an effect on enzyme function (Table [1](#T1)) because the amino acid at this position was predicted to be involved in coenzyme A ligand binding as part of the acetylation process. The 589C > T (R197X) results in a stop codon being introduced and hence no protein is expressed. The most common alleles of *NAT2 *in this study were *NAT2***5 *(341T > C; I114T) and *NAT2***6 *(590G > A; R197Q) (Table S4 (Table [5](#T5))), which contribute largely to the slow acetylator phenotype in African populations. Figure [1](#F1) shows *NAT2 *haplotypes and their frequencies in the total population studied. The most common sub-haplotypes were *NAT2***6A *and *NAT2***5B*, which affect enzyme function, followed by the wild type *NAT2***4 *and *NAT2***12A*, which do not impair acetylation.

In addition to non-synonymous SNPs, numerous novel synonymous SNPs, SNPs in introns and at splice site junctions, were identified. SNPs at splice site junctions were investigated, but none of the novel SNPs were located within the most critical -1 to -2 positions of the acceptor sites or the -2 to +4 positions of the donor sites.

## Discussion

Major genetic variability in drug-metabolising enzymes has been reported in Caucasian and Asian populations [[37](#B37)]. The aim of this study was to search for novel variants of the highly polymorphic cytochrome P450 (*CYP2C9, CYP2C19, CYP2D6*) and N-acetyltransferase 2 (*NAT2*) genes in Africans, using representative samples from our newly established Biobank [[21](#B21)]. This analysis was focused on the occurrence of alleles in African populations, their potential effects on enzyme function and the applicability of such data to personalised therapy.

### African populations

Certain SNPs or haplotypes that have been reported as prevalent and functionally important in other populations are rare or have not yet been detected in African populations. For example, *CYP2C9***2 *(R144C) and *CYP2C9***3 *(I359L), while extensively studied in Asian and Caucasian populations and identified as rare alleles in African Americans (frequency ~1 per cent), were not found in Africans, neither in this study nor in a Beninese population [[38](#B38)]. By contrast, *CYP2C9***5 *(D360E) and *CYP2C9***6 *(K273fs) have been identified in African populations, although at low frequency (frequency = 0.01; Table S1 ([2](#T2))). *CYP2C9***5 *causes impaired enzyme activity,[[6](#B6)] and *CYP2C9***6*, first found in African Americans, is associated with phenytoin toxicity [[39](#B39)]. The importance of *CYP2C9***8 *(R150H), *CYP2C9***9 *(H251R) and *CYP2C9***11 *(R335W), which were detected in limited studies in Africans [[17](#B17)] (partly including the present study), and the distribution of poor metabolisers in African populations remain unclear.

The US Food and Drug Administration (FDA) has recommended genotyping for *CYP2C9***2 *and *CYP2C9***3 *with the aim of better use of warfarin [[40](#B40)]. Since these variants are practically absent in populations of African origin, their use in current pharmacodiagnostic kits that identify individuals carrying *CYP2C9***2 *and *CYP2C9***3 *may not be applicable in these populations. Test kits that detect *CYP2C9***5, CYP2C9***6, CYP2C9***8, CYP2C9***9 *and *CYP2C9***11*, as well as our novel SNPs, should be more predictive of the clinical response to CYP2C9 substrate drugs in Africans. Before such tools can be developed and deployed for clinical use, however, further studies are required to establish the frequencies of these alleles in larger African populations, in addition to genotype-phenotype studies to establish their functional relevance.

*CYP2C19***2 *(splicing defect) and *CYP2C19***3 *(W212X) have been recommended as biomarkers for the administration of certain CYP2C19 substrates [[41](#B41)]. The *CYP2C19 *poor-metaboliser phenotype is detected in two to four per cent of Caucasians and in about 20 per cent of Asians, and these two variants account for 99 per cent of these poor metaboliser phenotypes [[42](#B42),[43](#B43)]. Whereas *CYP2C19***2 *was the most frequent known defective variant in our study (frequency = 0.15; Table S2 (Table [3](#T3))), we and various genotype-phenotype correlation studies have found *CYP2C19***3 *to be rare in most African populations (frequency = 0.01; Table S2 (Table [3](#T3)) [[44](#B44)]). We also identified one individual in the Maasai ethnic group who was heterozygous for this allele, and a few heterozygous individuals have previously been reported in a Tanzanian population [[20](#B20)]. Earlier data show that *CYP2C19***2 *accounts for over 70 per cent of slow metabolisers of S-mephenytoin [[45](#B45)]. The missing 30 per cent might be made up by *CYP2C19***3 *and other variants such as *CYP2C19***12, CYP2C19***13 *and *CYP2C19***15*, which would make these SNPs important contenders to include in genotyping panels for diagnostic purposes in Africans.

### Table S2.

| NC_000010.9 | mRNAposition | SNP | mRNAfeature | Effect | dbSNP | Hausa(20) | Yoruba(20) | Ibo(20) | Luo(30) | Maasai(13) | Shona(15) | Venda(9) | TZ Bantu(10) | Total(137) |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| 96653591 | -97 | T > C | 5' UTR |   | rs4986894 | 0.13 | 0.18 | 0.33 | 0.07 | 0.08 | 0.2 | 0.11 | 0.15 | 0.15 |
| 96653743 | 55 | A > C | Exon 1 | I19L (*15) | rs17882687 | 0 | 0 | 0 | 0.05 | 0 | 0 | 0 | 0.05 | 0.02 |
| 96653787 | 99 | T > C | Exon 1 | P33P | rs17885098 | 0.05 | 0.08 | 0.15 | 0.18 | 0.15 | 0.17 | 0.17 | 0.30 | 0.15 |
| 96653871 | 183 | T > C | Intron |   | rs17882201 | 0 | 0 | 0 | 0 | 0 | 0 | 0.06 | 0 | < 0.01 |
| 96653876 | 188 | G > A | Intron |   | rs17881883 | 0 | 0 | 0.03 | 0 | 0 | 0.07 | 0.11 | 0.05 | 0.02 |
| 96653919 | 231 | A > C | Intron |   | Novel | 0 | 0 | 0 | 0.01 | 0 | 0 | 0 | 0 | < 0.01 |
| 96665810 | 12122 | A > G | Intron |   | rs7916649 | 0.5 | 0.37 | 0.29 | 0.58 | 0.50 | 0.25 | 0.28 | 0.33 | 0.41 |
| 96665994 | 12306 | G > A | Intron |   | rs17878649 | 0 | 0 | 0.03 | 0.08 | 0.04 | 0.10 | 0.06 | 0.10 | 0.05 |
| 96666148 | 12460 | G > C | Exon 2 | E92D | rs17878459 | 0 | 0 | 0 | 0 | 0 | 0.03 | 0 | 0 | < 0.01 |
| 96666295 | 12607 | wt > insC | Intron |   | Novel | 0 | 0 | 0 | 0.03 | 0.04 | 0 | 0 | 0 | 0.01 |
| 96666325 | 12637 | C > T | Intron |   | Novel | 0.03 | 0.1 | 0 | 0 | 0.04 | 0 | 0.06 | 0 | 0.03 |
| 96666350 | 12662 | A > G | Intron | Splice site | rs12769205 | 0.16 | 0.2 | 0.33 | 0.09 | 0.12 | 0.27 | 0.22 | 0.2 | 0.18 |
| 96666378 | 12690 | G > A | Exon 3 | V113I | Novel | 0 | 0 | 0 | 0 | 0 | 0 | 0.06 | 0 | < 0.01 |
| 96666472 | 12784 | G > A | Exon 3 | R144H (*9) | rs17884712 | 0 | 0 | 0 | 0 | 0 | 0 | 0.06 | 0 | < 0.01 |
| 96671557 | 17869 | G > T | Exon 4 | R186P (*22) | Novel | 0 | 0 | 0 | 0 | 0 | 0 | 0 | 0.06 | < 0.01 |
| 96671636 | 17948 | G > A | Exon 4 | W212X (*3) | rs4986893 | 0 | 0 | 0 | 0 | 0.04 | 0 | 0 | 0 | < 0.01 |
| 96671895 | 18207 | G > A | Intron |   | Novel | 0 | 0 | 0 | 0.02 | 0 | 0 | 0 | 0.06 | < 0.01 |
| 96671917 | 18229 | T > A | Intron |   | rs17884938 | 0.06 | 0.03 | 0.05 | 0.07 | 0.00 | 0.10 | 0 | 0 | 0.05 |
| 96671942 | 18254 | T > C | Intron |   | Novel | 0.03 | 0.03 | 0 | 0 | 0 | 0 | 0 | 0 | 0.01 |
| 96672506 | 18818 | T > C | Intron | Pre-miRNA(has-mir-139) | Novel | 0 | 0.03 | 0 | 0 | 0 | 0 | 0 | 0 | < 0.01 |
| 96672599 | 18911 | A > G | Intron |   | rs7088784 | 0.05 | 0.08 | 0.15 | 0.14 | 0.15 | 0.17 | 0.17 | 0.25 | 0.13 |
| 96672764 | 19076 | T > C | Intron | Splice site | Novel | 0 | 0 | 0 | 0 | 0 | 0.03 | 0 | 0 | < 0.01 |
| 96672842 | 19154 | G > A | Exon 5 | P227P (*2) | rs4244285 | 0.13 | 0.15 | 0.33 | 0.07 | 0.08 | 0.23 | 0.17 | 0.15 | 0.15 |
| 96673020 | 19332 | G > A | Intron | Pre-miRNA(has-mir-448) | Novel | 0 | 0 | 0 | 0 | 0.08 | 0 | 0 | 0 | < 0.01 |
| 96711141 | 57453 | G > C | Intron |   | Novel | 0 | 0 | 0.04 | 0.03 | 0 | 0.03 | 0 | 0 | 0.02 |
| 96711200 | 57512 | A > G | Intron |   | Novel | 0.03 | 0.05 | 0.08 | 0.03 | 0.04 | 0 | 0 | 0.05 | 0.04 |
| 96711255 | 57567 | A > T | Intron |   | Novel | 0 | 0 | 0.04 | 0.03 | 0 | 0.07 | 0.11 | 0.10 | 0.04 |
| 96711263 | 57575 | T > C | Intron |   | Novel | 0 | 0 | 0.04 | 0.03 | 0 | 0.03 | 0 | 0 | 0.02 |
| 96711325 | 57637 | wt > delG | Intron |   | Novel | 0.03 | 0.05 | 0.08 | 0.07 | 0.08 | 0.10 | 0.00 | 0.10 | 0.06 |
| 96711366 | 57678 | T > G | Intron |   | rs28399511 | 0 | 0 | 0 | 0 | 0.04 | 0 | 0 | 0 | < 0.01 |
| 96711428 | 57740 | G > C | Intron |   | rs4417205 | 0.14 | 0.15 | 0.21 | 0.09 | 0.13 | 0.23 | 0.22 | 0.20 | 0.15 |
| 96711677 | 57989 | G > C | Intron |   | Novel | 0 | 0 | 0 | 0.02 | 0 | 0.1 | 0 | 0 | 0.02 |
| 96733848 | 80160 | C > T | Exon 7 | V330V | rs3758580 | 0.12 | 0.18 | 0.25 | 0.07 | 0.08 | 0.17 | 0.17 | 0.15 | 0.13 |
| 96733849 | 80161 | G > A | Exon 7 | V331I | rs3758581 | 0.03 | 0.03 | 0 | 0 | 0.04 | 0 | 0 | 0 | 0.01 |
| 96734317 | 80629 | T > A | Intron |   | Novel | 0.03 | 0.05 | 0 | 0 | 0.05 | 0 | 0 | 0 | 0.01 |
| 96740794 | 87106 | T > C | Intron |   | rs4917623 | 0.38 | 0.13 | 0.21 | 0.31 | 0.23 | 0.03 | 0.06 | 0.15 | 0.22 |
| 96740978 | 87290 | T > C | Exon 8 | R410C (*13) | rs17879685 | 0 | 0 | 0.04 | 0.03 | 0 | 0.03 | 0 | 0 | 0.02 |
| 96741001 | 87313 | A > C | Exon 8 | G417G | rs17886522 | 0.03 | 0.05 | 0.08 | 0.07 | 0.04 | 0.10 | 0 | 0 | 0.06 |
| 96741110 | 87422 | A > G | Intron |   | Novel | 0.03 | 0 | 0 | 0 | 0.08 | 0 | 0 | 0 | 0.02 |
| 96741163 | 87475 | G > C | Intron |   | rs17880188 | 0.08 | 0.13 | 0 | 0.07 | 0.04 | 0.11 | 0.06 | 0 | 0.07 |
| 96741210 | 87522 | C > T | Intron |   | rs17885567 | 0.05 | 0.08 | 0.21 | 0.10 | 0.08 | 0.13 | 0.11 | 0.10 | 0.1 |
| 96743266 | 89578 | T > A | Intron |   | rs12779363 | 0 | 0 | 0.06 | 0.04 | 0 | 0.03 | 0 | 0 | 0.01 |
| 96743597 | 89909 | C > T | Intron |   | rs12268020 | 0.22 | 0.31 | 0.17 | 0.09 | 0.17 | 0.20 | 0.17 | 0.20 | 0.19 |
| 96741001 | 90011 | A > G | Intron | Splice site | rs4451645 | 0.13 | 0.14 | 0 | 0.07 | 0.04 | 0.13 | 0.11 | 0 | 0.08 |
| 96743897 | 90209 | A > C | Exon 9 | X491C; 26extra aa (*12) | nrs | 0 | 0 | 0 | 0.04 | 0 | 0.03 | 0 | 0 | 0.01 |
| 96743989 | 90301 | C > T | 3' UTR |   | Novel | 0.01 | 0 | 0 | 0 | 0 | 0 | 0 | 0 | < 0.01 |
| 96743990 | 90302 | C > T | 3' UTR |   | Novel | 0 | 0 | 0 | 0.04 | 0 | 0.03 | 0 | 0 | 0.01 |
| 96744221 | 90533 | C > T | 3' UTR |   | Novel | 0 | 0 | 0.05 | 0.04 | 0 | 0.04 | 0 | 0 | 0.02 |

Table Caption: CYP2C19 single nucleotide polymorphism (SNP) frequencies

Our analysis of diverse African populations confirmed that *CYP2D6***17 *(T107I) and *CYP2D6***29 *(V136M, V338M) remain the most important functional SNPs in the metabolism of CYP2D6 substrate drugs. Together with other less prevalent haplotypes, they explain why African populations generally have a larger number of intermediate metabolisers (~40 per cent) compared with Caucasian populations (~15 per cent) [[46](#B46)].

Based on the highly polymorphic *CYP2D6*, we used principal component analysis to investigate inter-ethnic variability. The fact that no significant differences were detected across ethnicities (data not shown) could be due to our small sample sizes; however, our data are consistent with a recent study illustrating that *CYP2D6 *shows a high frequency of altered activity variants but no clear population structure [[47](#B47)]. It may also imply that the phenotype status of those populations is not significantly different either.

It has been speculated that the variation in acetylator (NAT2) status across major world populations reflects differences in dietary habits or the environment. There is a high prevalence of slow and intermediate acetylators in African populations, however, due to the common *NAT2***5 *(I114T), *NAT2***6 *(R197Q) and *NAT2***14 *(R64Q) alleles, which contribute largely to the slow acetylator phenotype. This is consistent with our data (Figure [1](#F1)) and with a recent study of sub-Saharan populations which also indicates that the *NAT2***5B *and *NAT2***6A *haplotypes are more common than the wild-type haplotype *NAT2***4 *[[1](#B1),[48](#B48)].

### Enzyme function

Some mutations in coding regions cause amino acid changes that result in alterations of enzyme activity, substrate selectivity and, sometimes, protein stability. Ensuing functional differences cause different metaboliser phenotypes. So far, over 30 such variants have been reported for *CYP2C9*, approximately 20 for *CYP2C19 *and over 60 for *CYP2D6 *[[30](#B30)].

We have predicted functional effects of novel non-synonymous SNPs discovered in this study (Table [1](#T1)). These predictions were based on amino acid chemistry, conservation in the alignment of known sequences from the same protein families, and solved structures or homology modelling. Crystal structures of CYP2C9 and CYP2D6 have been reported [[32](#B32),[34](#B34)] and structures of the other enzymes have been approximated by homology modelling [[49](#B49),[50](#B50)]. It is assumed that such approximation is sufficiently accurate to predict functional effects in substrate recognition, binding and catalysis of reactions [[51](#B51)].

Amino acid changes with a PSIC score of less than 1 are assumed not to be involved in any functional sites and are predicted not to affect enzyme function (eg N474S in CYP2C9, V113I in CYP2C19, V119M in CYP2D6, and I158L and I270T in NAT2). Some changes with PSIC scores slightly above 1 may still have modest effects on enzyme function -- for example, R123L in CYP2D6 (PSIC score = 1.236). As shown in a previous study, in which the CYP2D6 sequence was aligned with Gotoh's sequence,[[33](#B33)] however, this residue is involved in the substrate recognition site SRS1 [[52](#B52)]. The T214I change in NAT2 (PSIC score = 1.257) seems to interfere with enzyme function because this residue is important for the interaction with the co-enzyme A ligand, according to homology model prediction.

The effect of the R186P change in CYP2C19 leads to a change in electrostatic charge and possibly geometry; hence, it is predicted to affect the protein dramatically, giving a high PSIC score. The high score observed for G445E in CYP2D6 might be due to its interaction with position 443, which is important for heme-ligand binding,[[34](#B34)] and therefore has a high probability of affecting enzyme function.

Whereas some defective splice site variants are well understood -- for example, *CYP2D6***4 *(1846G > A), which occurs at the zero acceptor position of exon 4 -- functional indications are less clear if mutations lie further away from splice site junctions. Rogan *et al *[[53](#B53)]. have used information theory analysis to show how other intronic and synonymous mutations may contribute to splice site effects in *CYP *genes [[53](#B53)]. For example, the defective allele *CYP2C19***2 *(19154G > A) results in a synonymous mutation (P227P), yet it has been associated with reduced enzyme activity. Further investigations showed that this mutation introduces a cryptic splice site 40 nucleotides downstream, resulting in a truncated protein. We used information theory to analyse novel synonymous SNPs and intronic SNPs within the splice sites (-25 to +2 for exon acceptor sites and -3 to +6 for exon donor sites) of *CYP2C9, CYP2C19 *and *CYP2D6 *but did not find any significant effects on splice site recognition (data not shown).

Pre-miRNA sequences are involved in the regulation of protein expression. Mutations in these sequences, as well as insertions of new pre-miRNA sequences, could affect enzyme expression, yet *CYP1B1 *is the only *CYP *that has been found to be miRNA regulated so far [[54](#B54)]. In the present study, we did not find any pre-miRNA sequences introduced in the 3' untranslated region (UTR) regions, yet in *CYP2C19*, 18818T > C in intron 4 and 19332G > A in intron 5 introduce miRNA binding sites for has-mir-139 and has-mir-448, respectively (Table S2 (Table [3](#T3))). Since miRNA binding sites mostly act within the 3' UTR, however, these mutations would not be expected to have any effects.

In summary, our data, in conjunction with other studies of sub-Saharan Africans and African Americans,[[17](#B17),[19](#B19),[55](#B55),[56](#B56)] indicate low heterogeneity in the frequency of functional mutations. In the genes studied, most functionally important SNPs have been found. What remains is to determine their prevalence across populations and to evaluate the functional effects of the novel SNPs. Expressing variant proteins and analysing their substrate turnover to show impaired enzymatic activity was beyond the scope of this study. We envisage that such analyses will strengthen our findings, however, and might become essential for the pharmacokinetic assessment of individual variants in order to meet regulatory requirements for diagnostic use.

### Personalised therapy

Our data indicate the importance of *CYP2C9, CYP2C19, CYP2D6 *and *NAT2 *for genotype assessment, including the identified novel SNPs, so that optimisation of drug use in African populations can be considered under appropriate clinical scenarios. This could enable correct dose adjustment for individuals who are likely to experience ADRs owing to poor metabolism or an inadequate therapeutic effect owing to ultra-rapid metabolism. It is noteworthy, however, that other factors, which are not related to the newly identified SNPs but affect the clinical pharmacology of prescribed medications, may play a role in clinical ADRs or therapeutic failure.

The incorporation of *CYP2C9 *genotyping as part of pre-prescription diagnosis for individuals being treated with drugs metabolised by this enzyme [[57](#B57)] indicates the immediate utility of pharmacogenetics. Likewise, pre-prescription genotyping has been recommended for CYP2D6-metabolised drugs with a narrow therapeutic window, such as some antipsychotic agents [[58](#B58)]. NAT2 genotype information can be used to predict the phenotypic status of individuals to enable dose adjustment of anti-tuberculosis drugs such as isoniazid.

## Conclusions

We have started to identify and catalogue novel variants (SNPs) of genes that are important in drug metabolism. We have confirmed African-specific variants but found modest variation between different African ethnicities, indicating similar metabolic profiles for most drugs, yet stressing inter-individual variability. The low frequency of our new *CYP2C9, CYP2C19, CYP2D6 *and *NAT2 *alleles seems to have reduced their impact at the population level. The generally high level of diversity in gene loci of African populations, however, indicates that rare variants (incidence of less than 1 per cent) and inter-individual variability might bear extra weight in explaining Africans' phenotypic diversity. As genome-wide association studies turn up new variants at high pace, the character of molecular diagnostics shifts from single genes to profiles, encompassing low frequency variants as their main constituents.

We have predicted the functional effects of non-synonymous SNPs and suggest genotype-phenotype studies to investigate the effects of these SNPs in individuals. Eventually, we recommend the genotyping of African populations to establish the prevalence of functionally important haplotypes towards the development of relevant pharmacodiagnostic tools for these populations.

## Appendix: Supplementary Tables

## Acknowledgements

Alice Matimba was a recipient of the Flemish International Council scholarship for her PhD studies in collaborating between the Department of Molecular Genetics, University of Antwerp (UA) and the African Institute of Biomedical Science and Technology (AiBST). The authors acknowledge the contribution of the personnel of the VIB Genetic Service Facility [http://www.vibgeneticservicefacility.be](http://www.vibgeneticservicefacility.be) for the genetic analysis. We also thank Anastasia Guantai, Margaret Oluka, Emmanuel Chigutsa, Oluseye Bolaji, Ben Ebeshi and the members of the Consortium for the Study of Pharmacogenetics of African populations (CoPhA) for the extensive sample collection in the AiBST Biobank.

## References

1. Sabbagh A, Langaney A, Darlu P, Gerard N, Krishnamoorthy R, Poloni ES. Worldwide distribution of NAT2 diversity: implications for NAT2 evolutionary history. BMC Genet. 2008;9:21. doi: 10.1186/1471-2156-9-21.  [DOI](https://doi.org/10.1186/1471-2156-9-21) | [PMC free article](/articles/PMC2292740/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18304320/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Genet&title=Worldwide%20distribution%20of%20NAT2%20diversity:%20implications%20for%20NAT2%20evolutionary%20history&author=A%20Sabbagh&author=A%20Langaney&author=P%20Darlu&author=N%20Gerard&author=R%20Krishnamoorthy&volume=9&publication_year=2008&pages=21&pmid=18304320&doi=10.1186/1471-2156-9-21&)

2. Wijnen PA, Op den Buijsch RA, Drent M. et al. Review article: The prevalence and clinical relevance of cytochrome P450 polymorphisms. Aliment Pharmacol Ther. 2007;26(Suppl 2):211–219. doi: 10.1111/j.1365-2036.2007.03490.x.  [DOI](https://doi.org/10.1111/j.1365-2036.2007.03490.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18081664/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Aliment%20Pharmacol%20Ther&title=Review%20article:%20The%20prevalence%20and%20clinical%20relevance%20of%20cytochrome%20P450%20polymorphisms&author=PA%20Wijnen&author=RA%20Op%20den%20Buijsch&author=M%20Drent&volume=26&issue=Suppl%202&publication_year=2007&pages=211-219&pmid=18081664&doi=10.1111/j.1365-2036.2007.03490.x&)

3. Gray IC, Nobile C, Muresu R, Ford S, Spurr NK. A 2.4-megabase physical map spanning the CYP2C gene cluster on chromosome 10q24. Genomics. 1995;28:328–332. doi: 10.1006/geno.1995.1149.  [DOI](https://doi.org/10.1006/geno.1995.1149) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8530044/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genomics&title=A%202.4-megabase%20physical%20map%20spanning%20the%20CYP2C%20gene%20cluster%20on%20chromosome%2010q24&author=IC%20Gray&author=C%20Nobile&author=R%20Muresu&author=S%20Ford&author=NK%20Spurr&volume=28&publication_year=1995&pages=328-332&pmid=8530044&doi=10.1006/geno.1995.1149&)

4. Shimada T, Yamazaki H, Mimura M, Inui Y, Guengerich FP. Interindividual variations in human liver cytochrome P-450 enzymes involved in the oxidation of drugs, carcinogens and toxic chemicals: Studies with liver microsomes of 30 Japanese and 30 Caucasians. J Pharmacol Exp Ther. 1994;270:414–423.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/8035341/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacol%20Exp%20Ther&title=Interindividual%20variations%20in%20human%20liver%20cytochrome%20P-450%20enzymes%20involved%20in%20the%20oxidation%20of%20drugs,%20carcinogens%20and%20toxic%20chemicals:%20Studies%20with%20liver%20microsomes%20of%2030%20Japanese%20and%2030%20Caucasians&author=T%20Shimada&author=H%20Yamazaki&author=M%20Mimura&author=Y%20Inui&author=FP%20Guengerich&volume=270&publication_year=1994&pages=414-423&pmid=8035341&)

5. Tsao CC, Wester MR, Ghanayem B. et al. Identification of human CYP2C19 residues that confer S-mephenytoin 4'-hydroxylation activity to CYP2C9. Biochemistry. 2001;40:1937–1944. doi: 10.1021/bi001678u.  [DOI](https://doi.org/10.1021/bi001678u) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11329260/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochemistry&title=Identification%20of%20human%20CYP2C19%20residues%20that%20confer%20S-mephenytoin%204'-hydroxylation%20activity%20to%20CYP2C9&author=CC%20Tsao&author=MR%20Wester&author=B%20Ghanayem&volume=40&publication_year=2001&pages=1937-1944&pmid=11329260&doi=10.1021/bi001678u&)

6. Yasar U, Eliasson E, Dahl ML, Johansson I, Ingelman-Sundberg M, Sjoqvist F. Validation of methods for CYP2C9 genotyping: Frequencies of mutant alleles in a Swedish population. Biochem Biophys Res Commun. 1999;254:628–631. doi: 10.1006/bbrc.1998.9992.  [DOI](https://doi.org/10.1006/bbrc.1998.9992) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9920790/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20Biophys%20Res%20Commun&title=Validation%20of%20methods%20for%20CYP2C9%20genotyping:%20Frequencies%20of%20mutant%20alleles%20in%20a%20Swedish%20population&author=U%20Yasar&author=E%20Eliasson&author=ML%20Dahl&author=I%20Johansson&author=M%20Ingelman-Sundberg&volume=254&publication_year=1999&pages=628-631&pmid=9920790&doi=10.1006/bbrc.1998.9992&)

7. Dickinson GL, Lennard MS, Tucker GT, Rostami-Hodjegan A. The use of mechanistic DM-PK-PD modelling to assess the power of pharmacogenetic studies -- CYP2C9 and warfarin as an example. Br J Clin Pharmacol. 2007;64:14–26. doi: 10.1111/j.1365-2125.2007.02850.x.  [DOI](https://doi.org/10.1111/j.1365-2125.2007.02850.x) | [PMC free article](/articles/PMC2000610/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17298479/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=The%20use%20of%20mechanistic%20DM-PK-PD%20modelling%20to%20assess%20the%20power%20of%20pharmacogenetic%20studies%20--%20CYP2C9%20and%20warfarin%20as%20an%20example&author=GL%20Dickinson&author=MS%20Lennard&author=GT%20Tucker&author=A%20Rostami-Hodjegan&volume=64&publication_year=2007&pages=14-26&pmid=17298479&doi=10.1111/j.1365-2125.2007.02850.x&)

8. Higashi MK, Veenstra DL, Kondo LM. et al. Association between CYP2C9 genetic variants and anticoagulation-related outcomes during warfarin therapy. JAMA. 2002;287:1690–1698. doi: 10.1001/jama.287.13.1690.  [DOI](https://doi.org/10.1001/jama.287.13.1690) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11926893/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA&title=Association%20between%20CYP2C9%20genetic%20variants%20and%20anticoagulation-related%20outcomes%20during%20warfarin%20therapy&author=MK%20Higashi&author=DL%20Veenstra&author=LM%20Kondo&volume=287&publication_year=2002&pages=1690-1698&pmid=11926893&doi=10.1001/jama.287.13.1690&)

9. Miners JO, Birkett DJ. Cytochrome P4502C9: An enzyme of major importance in human drug metabolism. Br J Clin Pharmacol. 1998;45:525–538. doi: 10.1046/j.1365-2125.1998.00721.x.  [DOI](https://doi.org/10.1046/j.1365-2125.1998.00721.x) | [PMC free article](/articles/PMC1873650/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9663807/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=Cytochrome%20P4502C9:%20An%20enzyme%20of%20major%20importance%20in%20human%20drug%20metabolism&author=JO%20Miners&author=DJ%20Birkett&volume=45&publication_year=1998&pages=525-538&pmid=9663807&doi=10.1046/j.1365-2125.1998.00721.x&)

10. Goldstein JA. Clinical relevance of genetic polymorphisms in the human CYP2C subfamily. Br J Clin Pharmacol. 2001;52:349–355. doi: 10.1046/j.0306-5251.2001.01499.x.  [DOI](https://doi.org/10.1046/j.0306-5251.2001.01499.x) | [PMC free article](/articles/PMC2014584/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11678778/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=Clinical%20relevance%20of%20genetic%20polymorphisms%20in%20the%20human%20CYP2C%20subfamily&author=JA%20Goldstein&volume=52&publication_year=2001&pages=349-355&pmid=11678778&doi=10.1046/j.0306-5251.2001.01499.x&)

11. Sim SC, Risinger C, Dahl ML. et al. A common novel CYP2C19 gene variant causes ultrarapid drug metabolism relevant for the drug response to proton pump inhibitors and antidepressants. Clin Pharmacol Ther. 2006;79:103–113. doi: 10.1016/j.clpt.2005.10.002.  [DOI](https://doi.org/10.1016/j.clpt.2005.10.002) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16413245/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=A%20common%20novel%20CYP2C19%20gene%20variant%20causes%20ultrarapid%20drug%20metabolism%20relevant%20for%20the%20drug%20response%20to%20proton%20pump%20inhibitors%20and%20antidepressants&author=SC%20Sim&author=C%20Risinger&author=ML%20Dahl&volume=79&publication_year=2006&pages=103-113&pmid=16413245&doi=10.1016/j.clpt.2005.10.002&)

12. Ingelman-Sundberg M. Genetic polymorphisms of cytochrome P450 2D6 (CYP2D6): Clinical consequences, evolutionary aspects and functional diversity. Pharmacogenomics J. 2005;5:6–13. doi: 10.1038/sj.tpj.6500285.  [DOI](https://doi.org/10.1038/sj.tpj.6500285) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15492763/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Genetic%20polymorphisms%20of%20cytochrome%20P450%202D6%20(CYP2D6):%20Clinical%20consequences,%20evolutionary%20aspects%20and%20functional%20diversity&author=M%20Ingelman-Sundberg&volume=5&publication_year=2005&pages=6-13&pmid=15492763&doi=10.1038/sj.tpj.6500285&)

13. Aklillu E, Persson I, Bertilsson L. et al. Frequent distribution of ultrarapid metabolizers of debrisoquine in an Ethiopian population carrying duplicated and multiduplicated functional CYP2D6 alleles. J Pharmacol Exp Ther. 1996;278:441–446.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/8764380/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacol%20Exp%20Ther&title=Frequent%20distribution%20of%20ultrarapid%20metabolizers%20of%20debrisoquine%20in%20an%20Ethiopian%20population%20carrying%20duplicated%20and%20multiduplicated%20functional%20CYP2D6%20alleles&author=E%20Aklillu&author=I%20Persson&author=L%20Bertilsson&volume=278&publication_year=1996&pages=441-446&pmid=8764380&)

14. Zanger UM, Raimundo S, Eichelbaum M. Cytochrome P450 2D6: Overview and update on pharmacology, genetics, biochemistry. Naunyn Schmiedebergs Arch Pharmacol. 2004;369:23–37. doi: 10.1007/s00210-003-0832-2.  [DOI](https://doi.org/10.1007/s00210-003-0832-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14618296/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Naunyn%20Schmiedebergs%20Arch%20Pharmacol&title=Cytochrome%20P450%202D6:%20Overview%20and%20update%20on%20pharmacology,%20genetics,%20biochemistry&author=UM%20Zanger&author=S%20Raimundo&author=M%20Eichelbaum&volume=369&publication_year=2004&pages=23-37&pmid=14618296&doi=10.1007/s00210-003-0832-2&)

15. Donald PR, Sirgel FA, Venter A. et al. The influence of human N-acetyltransferase genotype on the early bactericidal activity of isoniazid. Clin Infect Dis. 2004;39:1425–1430. doi: 10.1086/424999.  [DOI](https://doi.org/10.1086/424999) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15546075/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Infect%20Dis&title=The%20influence%20of%20human%20N-acetyltransferase%20genotype%20on%20the%20early%20bactericidal%20activity%20of%20isoniazid&author=PR%20Donald&author=FA%20Sirgel&author=A%20Venter&volume=39&publication_year=2004&pages=1425-1430&pmid=15546075&doi=10.1086/424999&)

16. Donald PR, Parkin DP, Seifart HI. et al. The influence of dose and N-acetyltransferase-2 (NAT2) genotype and phenotype on the pharmacokinetics and pharmacodynamics of isoniazid. Eur J Clin Pharmacol. 2007;63:633–639. doi: 10.1007/s00228-007-0305-5.  [DOI](https://doi.org/10.1007/s00228-007-0305-5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17505821/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Clin%20Pharmacol&title=The%20influence%20of%20dose%20and%20N-acetyltransferase-2%20(NAT2)%20genotype%20and%20phenotype%20on%20the%20pharmacokinetics%20and%20pharmacodynamics%20of%20isoniazid&author=PR%20Donald&author=DP%20Parkin&author=HI%20Seifart&volume=63&publication_year=2007&pages=633-639&pmid=17505821&doi=10.1007/s00228-007-0305-5&)

17. Allabi AC, Gala JL, Horsmans Y. et al. Functional impact of CYP2C95, CYP2C96, CYP2C98, and CYP2C911 in vivo among black Africans. Clin Pharmacol Ther. 2004;76:113–118. doi: 10.1016/j.clpt.2004.04.001.  [DOI](https://doi.org/10.1016/j.clpt.2004.04.001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15289788/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Functional%20impact%20of%20CYP2C95,%20CYP2C96,%20CYP2C98,%20and%20CYP2C911%20in%20vivo%20among%20black%20Africans&author=AC%20Allabi&author=JL%20Gala&author=Y%20Horsmans&volume=76&publication_year=2004&pages=113-118&pmid=15289788&doi=10.1016/j.clpt.2004.04.001&)

18. Bathum L, Skjelbo E, Mutabingwa TK, Madsen H, Horder M, Brosen K. Phenotypes and genotypes for CYP2D6 and CYP2C19 in a black Tanzanian population. Br J Clin Pharmacol. 1999;48:395–401. doi: 10.1046/j.1365-2125.1999.00019.x.  [DOI](https://doi.org/10.1046/j.1365-2125.1999.00019.x) | [PMC free article](/articles/PMC2014329/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10510152/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=Phenotypes%20and%20genotypes%20for%20CYP2D6%20and%20CYP2C19%20in%20a%20black%20Tanzanian%20population&author=L%20Bathum&author=E%20Skjelbo&author=TK%20Mutabingwa&author=H%20Madsen&author=M%20Horder&volume=48&publication_year=1999&pages=395-401&pmid=10510152&doi=10.1046/j.1365-2125.1999.00019.x&)

19. Gaedigk A, Bhathena A, Ndjountche L. et al. Identification and characterization of novel sequence variations in the cytochrome P4502D6 (CYP2D6) gene in African Americans. Pharmacogenomics J. 2005;5:173–182. doi: 10.1038/sj.tpj.6500305.  [DOI](https://doi.org/10.1038/sj.tpj.6500305) | [PMC free article](/articles/PMC1440720/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15768052/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Identification%20and%20characterization%20of%20novel%20sequence%20variations%20in%20the%20cytochrome%20P4502D6%20(CYP2D6)%20gene%20in%20African%20Americans&author=A%20Gaedigk&author=A%20Bhathena&author=L%20Ndjountche&volume=5&publication_year=2005&pages=173-182&pmid=15768052&doi=10.1038/sj.tpj.6500305&)

20. Herrlin K, Massele AY, Jande M. et al. Bantu Tanzanians have a decreased capacity to metabolize omeprazole and mephenytoin in relation to their CYP2C19 genotype. Clin Pharmacol Ther. 1998;64:391–401. doi: 10.1016/S0009-9236(98)90070-4.  [DOI](https://doi.org/10.1016/S0009-9236(98)90070-4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9797796/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Bantu%20Tanzanians%20have%20a%20decreased%20capacity%20to%20metabolize%20omeprazole%20and%20mephenytoin%20in%20relation%20to%20their%20CYP2C19%20genotype&author=K%20Herrlin&author=AY%20Massele&author=M%20Jande&volume=64&publication_year=1998&pages=391-401&pmid=9797796&doi=10.1016/S0009-9236(98)90070-4&)

21. Matimba A, Oluka MN, Ebeshi BU. et al. Establishment of a biobank and pharmacogenetics database of African populations. Eur J Hum Genet. 2008;7:780–783. doi: 10.1038/ejhg.2008.49.  [DOI](https://doi.org/10.1038/ejhg.2008.49) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18382479/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Hum%20Genet&title=Establishment%20of%20a%20biobank%20and%20pharmacogenetics%20database%20of%20African%20populations&author=A%20Matimba&author=MN%20Oluka&author=BU%20Ebeshi&volume=7&publication_year=2008&pages=780-783&pmid=18382479&doi=10.1038/ejhg.2008.49&)

22. Weckx S, De Rijk P, Van Broeckhoven C, Del Favero J. SNPbox: Web-based high-throughput primer design from gene to genome. Nucleic Acids Res. 2004;32:W170–W172. doi: 10.1093/nar/gkh369.  [DOI](https://doi.org/10.1093/nar/gkh369) | [PMC free article](/articles/PMC441507/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15215373/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=SNPbox:%20Web-based%20high-throughput%20primer%20design%20from%20gene%20to%20genome&author=S%20Weckx&author=P%20De%20Rijk&author=C%20Van%20Broeckhoven&author=J%20Del%20Favero&volume=32&publication_year=2004&pages=W170-W172&pmid=15215373&doi=10.1093/nar/gkh369&)

23. Rozen S, Skaletsky H. Primer3 on the WWW for general users and for biologist programmers. Methods Mol Biol. 2000;132:365–386. doi: 10.1385/1-59259-192-2:365.  [DOI](https://doi.org/10.1385/1-59259-192-2:365) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10547847/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Methods%20Mol%20Biol&title=Primer3%20on%20the%20WWW%20for%20general%20users%20and%20for%20biologist%20programmers&author=S%20Rozen&author=H%20Skaletsky&volume=132&publication_year=2000&pages=365-386&pmid=10547847&doi=10.1385/1-59259-192-2:365&)

24. Weckx S, Del Favero J, Rademakers R. et al. novoSNP, a novel computational tool for sequence variation discovery. Genome Res. 2005;15:436–442. doi: 10.1101/gr.2754005.  [DOI](https://doi.org/10.1101/gr.2754005) | [PMC free article](/articles/PMC551570/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15741513/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res&title=novoSNP,%20a%20novel%20computational%20tool%20for%20sequence%20variation%20discovery&author=S%20Weckx&author=J%20Del%20Favero&author=R%20Rademakers&volume=15&publication_year=2005&pages=436-442&pmid=15741513&doi=10.1101/gr.2754005&)

25. NCBI Single Nucleotide Polymorphism Database (dbSNP) The National Library of Medicine (NLM); http://www.ncbi.nlm.nih.gov/SNP [cited April 14 2008]  [http://www.ncbi.nlm.nih.gov/SNP](http://www.ncbi.nlm.nih.gov/SNP) | [Google Scholar](https://scholar.google.com/scholar_lookup?NCBI%20Single%20Nucleotide%20Polymorphism%20Database%20(dbSNP)%20The%20National%20Library%20of%20Medicine%20(NLM);%20http://www.ncbi.nlm.nih.gov/SNP%20%5Bcited%20April%2014%202008%5D)

26. Raymond M, Rousset F. GENEPOP (Version 1.2): Population Genetics Software for Exact Tests and Ecumenicism. Laboratoire de Genetique et Environment; 1995. http://genepop.curtin.edu.au [cited August 21 2007]  [http://genepop.curtin.edu.au](http://genepop.curtin.edu.au) | [Google Scholar](https://scholar.google.com/scholar_lookup?title=GENEPOP%20(Version%201.2):%20Population%20Genetics%20Software%20for%20Exact%20Tests%20and%20Ecumenicism&author=M%20Raymond&author=F%20Rousset&publication_year=1995&)

27. Barrett JC, Fry B, Maller J, Daly MJ. Haploview: Analysis and visualization of LD and haplotype maps. Bioinformatics. 2005;21:263–265. doi: 10.1093/bioinformatics/bth457.  [DOI](https://doi.org/10.1093/bioinformatics/bth457) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15297300/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=Haploview:%20Analysis%20and%20visualization%20of%20LD%20and%20haplotype%20maps&author=JC%20Barrett&author=B%20Fry&author=J%20Maller&author=MJ%20Daly&volume=21&publication_year=2005&pages=263-265&pmid=15297300&doi=10.1093/bioinformatics/bth457&)

28. Ramensky V, Bork P, Sunyaev S. Human non-synonymous SNPs: Server and survey. European Molecular Biology Laboratory; 2002. http://genetic.bwh.havard.edu/pph [cited September 1 2008]  [http://genetic.bwh.havard.edu/pph](http://genetic.bwh.havard.edu/pph) | [DOI](https://doi.org/10.1093/nar/gkf493) | [PMC free article](/articles/PMC137415/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12202775/) | [Google Scholar](https://scholar.google.com/scholar_lookup?title=Human%20non-synonymous%20SNPs:%20Server%20and%20survey&author=V%20Ramensky&author=P%20Bork&author=S%20Sunyaev&publication_year=2002&)

29. The Uniprot Consortium. The universal protein resource (UniProt) The Uniprot Consortium; 2008. http://www.expasy.uniprot.org [cited November 27 2007]  [http://www.expasy.uniprot.org](http://www.expasy.uniprot.org) | [Google Scholar](https://scholar.google.com/scholar_lookup?title=The%20universal%20protein%20resource%20(UniProt)&publication_year=2008&)

30. Home Page of the Human Cytochrome P450 (CYP) Allele Nomenclature Committee. Karolinska Institute; http://www.cypalleles.ki.se [cited February 21 2001]  [http://www.cypalleles.ki.se](http://www.cypalleles.ki.se) | [Google Scholar](https://scholar.google.com/scholar_lookup?Home%20Page%20of%20the%20Human%20Cytochrome%20P450%20(CYP)%20Allele%20Nomenclature%20Committee.%20Karolinska%20Institute;%20http://www.cypalleles.ki.se%20%5Bcited%20February%2021%202001%5D)

31. Consensus Arylamine N-Acetyltransferase (NAT) Gene Nomenclature. University of Lousiville School of Medicine; http://louisville.edu/medschool/pharmacology/NAT.html [cited October 14 2007]  [http://louisville.edu/medschool/pharmacology/NAT.html](http://louisville.edu/medschool/pharmacology/NAT.html) | [Google Scholar](https://scholar.google.com/scholar_lookup?Consensus%20Arylamine%20N-Acetyltransferase%20(NAT)%20Gene%20Nomenclature.%20University%20of%20Lousiville%20School%20of%20Medicine;%20http://louisville.edu/medschool/pharmacology/NAT.html%20%5Bcited%20October%2014%202007%5D)

32. Williams PA, Cosme J, Ward A, Angove HC, Matak VD, Jhoti H. Crystal structure of human cytochrome P450 2C9 with bound warfarin. Nature. 2003;424:464–468. doi: 10.1038/nature01862.  [DOI](https://doi.org/10.1038/nature01862) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12861225/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Crystal%20structure%20of%20human%20cytochrome%20P450%202C9%20with%20bound%20warfarin&author=PA%20Williams&author=J%20Cosme&author=A%20Ward&author=HC%20Angove&author=VD%20Matak&volume=424&publication_year=2003&pages=464-468&pmid=12861225&doi=10.1038/nature01862&)

33. Gotoh O. Substrate recognition sites in cytochrome P450 family 2 (CYP2) proteins inferred from comparative analyses of amino acid and coding nucleotide sequences. J Biol Chem. 1992;267:83–90.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/1730627/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Substrate%20recognition%20sites%20in%20cytochrome%20P450%20family%202%20(CYP2)%20proteins%20inferred%20from%20comparative%20analyses%20of%20amino%20acid%20and%20coding%20nucleotide%20sequences&author=O%20Gotoh&volume=267&publication_year=1992&pages=83-90&pmid=1730627&)

34. Rowland P, Blaney FE, Smyth MG. et al. Crystal structure of human cytochrome P450 2D6. J Biol Chem. 2006;281:7614–7622. doi: 10.1074/jbc.M511232200.  [DOI](https://doi.org/10.1074/jbc.M511232200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16352597/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Crystal%20structure%20of%20human%20cytochrome%20P450%202D6&author=P%20Rowland&author=FE%20Blaney&author=MG%20Smyth&volume=281&publication_year=2006&pages=7614-7622&pmid=16352597&doi=10.1074/jbc.M511232200&)

35. Masimirembwa C, Persson I, Bertilsson L, Hasler J, Ingelman-Sundberg M. A novel mutant variant of the CYP2D6 gene (CYP2D6*17) common in a black African population: Association with diminished debrisoquine hydroxylase activity. Br J Clin Pharmacol. 1996;42:713–719. doi: 10.1046/j.1365-2125.1996.00489.x.  [DOI](https://doi.org/10.1046/j.1365-2125.1996.00489.x) | [PMC free article](/articles/PMC2042718/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8971426/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=A%20novel%20mutant%20variant%20of%20the%20CYP2D6%20gene%20(CYP2D6*17)%20common%20in%20a%20black%20African%20population:%20Association%20with%20diminished%20debrisoquine%20hydroxylase%20activity&author=C%20Masimirembwa&author=I%20Persson&author=L%20Bertilsson&author=J%20Hasler&author=M%20Ingelman-Sundberg&volume=42&publication_year=1996&pages=713-719&pmid=8971426&doi=10.1046/j.1365-2125.1996.00489.x&)

36. Wennerholm A, Johansson I, Hidestrand M, Bertilsson L, Gustafsson LL, Ingelman-Sundberg M. Characterization of the CYP2D6*29 allele commonly present in a black Tanzanian population causing reduced catalytic activity. Pharmacogenetics. 2001;11:417–427. doi: 10.1097/00008571-200107000-00005.  [DOI](https://doi.org/10.1097/00008571-200107000-00005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11470994/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Characterization%20of%20the%20CYP2D6*29%20allele%20commonly%20present%20in%20a%20black%20Tanzanian%20population%20causing%20reduced%20catalytic%20activity&author=A%20Wennerholm&author=I%20Johansson&author=M%20Hidestrand&author=L%20Bertilsson&author=LL%20Gustafsson&volume=11&publication_year=2001&pages=417-427&pmid=11470994&doi=10.1097/00008571-200107000-00005&)

37. Mizutani T. PM frequencies of major CYPs in Asians and Caucasians. Drug Metab Rev. 2003;35:99–106. doi: 10.1081/DMR-120023681.  [DOI](https://doi.org/10.1081/DMR-120023681) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12959412/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Rev&title=PM%20frequencies%20of%20major%20CYPs%20in%20Asians%20and%20Caucasians&author=T%20Mizutani&volume=35&publication_year=2003&pages=99-106&pmid=12959412&doi=10.1081/DMR-120023681&)

38. Allabi AC, Gala JL, Desager JP, Heusterspreute M, Horsmans Y. Genetic polymorphisms of CYP2C9 and CYP2C19 in the Beninese and Belgian populations. Br J Clin Pharmacol. 2003;56:653–657. doi: 10.1046/j.1365-2125.2003.01937.x.  [DOI](https://doi.org/10.1046/j.1365-2125.2003.01937.x) | [PMC free article](/articles/PMC1884305/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14616425/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=Genetic%20polymorphisms%20of%20CYP2C9%20and%20CYP2C19%20in%20the%20Beninese%20and%20Belgian%20populations&author=AC%20Allabi&author=JL%20Gala&author=JP%20Desager&author=M%20Heusterspreute&author=Y%20Horsmans&volume=56&publication_year=2003&pages=653-657&pmid=14616425&doi=10.1046/j.1365-2125.2003.01937.x&)

39. Kidd RS, Curry TB, Gallagher S, Edeki T, Blaisdell J, Goldstein JA. Identification of a null allele of CYP2C9 in an African-American exhibiting toxicity to phenytoin. Pharmacogenetics. 2001;11:803–808. doi: 10.1097/00008571-200112000-00008.  [DOI](https://doi.org/10.1097/00008571-200112000-00008) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11740344/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Identification%20of%20a%20null%20allele%20of%20CYP2C9%20in%20an%20African-American%20exhibiting%20toxicity%20to%20phenytoin&author=RS%20Kidd&author=TB%20Curry&author=S%20Gallagher&author=T%20Edeki&author=J%20Blaisdell&volume=11&publication_year=2001&pages=803-808&pmid=11740344&doi=10.1097/00008571-200112000-00008&)

40. Thompson CA. FDA encourages genetics-aided warfarin dosing. Am J Health Syst Pharm. 2007;64:1994–1996. doi: 10.2146/news070085.  [DOI](https://doi.org/10.2146/news070085) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17893404/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Health%20Syst%20Pharm&title=FDA%20encourages%20genetics-aided%20warfarin%20dosing&author=CA%20Thompson&volume=64&publication_year=2007&pages=1994-1996&pmid=17893404&doi=10.2146/news070085&)

41. Furuta T, Shirai N, Kodaira MS. et al. Pharmacogenomics-based tailored versus standard therapeutic regimen for eradication of H. pylori. Clin Pharmacol Ther. 2007;81:521–528. doi: 10.1038/sj.clpt.6100043.  [DOI](https://doi.org/10.1038/sj.clpt.6100043) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17215846/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Pharmacogenomics-based%20tailored%20versus%20standard%20therapeutic%20regimen%20for%20eradication%20of%20H.%20pylori&author=T%20Furuta&author=N%20Shirai&author=MS%20Kodaira&volume=81&publication_year=2007&pages=521-528&pmid=17215846&doi=10.1038/sj.clpt.6100043&)

42. Goldstein JA, Ishizaki T, Chiba K. et al. Frequencies of the defective CYP2C19 alleles responsible for the mephenytoin poor metabolizer phenotype in various Oriental, Caucasian, Saudi Arabian and American black populations. Pharmacogenetics. 1997;7:59–64. doi: 10.1097/00008571-199702000-00008.  [DOI](https://doi.org/10.1097/00008571-199702000-00008) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9110363/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Frequencies%20of%20the%20defective%20CYP2C19%20alleles%20responsible%20for%20the%20mephenytoin%20poor%20metabolizer%20phenotype%20in%20various%20Oriental,%20Caucasian,%20Saudi%20Arabian%20and%20American%20black%20populations&author=JA%20Goldstein&author=T%20Ishizaki&author=K%20Chiba&volume=7&publication_year=1997&pages=59-64&pmid=9110363&doi=10.1097/00008571-199702000-00008&)

43. Ibeanu GC, Goldstein JA, Meyer U. et al. Identification of new human CYP2C19 alleles (CYP2C19*6 and CYP2C19*2B) in a Caucasian poor metabolizer of mephenytoin. J Pharmacol Exp Ther. 1998;286:1490–1495.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/9732415/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacol%20Exp%20Ther&title=Identification%20of%20new%20human%20CYP2C19%20alleles%20(CYP2C19*6%20and%20CYP2C19*2B)%20in%20a%20Caucasian%20poor%20metabolizer%20of%20mephenytoin&author=GC%20Ibeanu&author=JA%20Goldstein&author=U%20Meyer&volume=286&publication_year=1998&pages=1490-1495&pmid=9732415&)

44. Xie HG, Kim RB, Stein CM, Wilkinson GR, Wood AJ. Genetic polymorphism of (S)-mephenytoin 4'-hydroxylation in populations of African descent. Br J Clin Pharmacol. 1999;48:402–408. doi: 10.1046/j.1365-2125.1999.00009.x.  [DOI](https://doi.org/10.1046/j.1365-2125.1999.00009.x) | [PMC free article](/articles/PMC2014331/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10510153/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=Genetic%20polymorphism%20of%20(S)-mephenytoin%204'-hydroxylation%20in%20populations%20of%20African%20descent&author=HG%20Xie&author=RB%20Kim&author=CM%20Stein&author=GR%20Wilkinson&author=AJ%20Wood&volume=48&publication_year=1999&pages=402-408&pmid=10510153&doi=10.1046/j.1365-2125.1999.00009.x&)

45. Masimirembwa C, Bertilsson L, Johansson I, Hasler JA, Ingelman-Sundberg M. Phenotyping and genotyping of S-mephenytoin hydroxylase (cytochrome P450 2C19) in a Shona population of Zimbabwe. Clin Pharmacol Ther. 1995;57:656–661. doi: 10.1016/0009-9236(95)90228-7.  [DOI](https://doi.org/10.1016/0009-9236(95)90228-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7781265/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Phenotyping%20and%20genotyping%20of%20S-mephenytoin%20hydroxylase%20(cytochrome%20P450%202C19)%20in%20a%20Shona%20population%20of%20Zimbabwe&author=C%20Masimirembwa&author=L%20Bertilsson&author=I%20Johansson&author=JA%20Hasler&author=M%20Ingelman-Sundberg&volume=57&publication_year=1995&pages=656-661&pmid=7781265&doi=10.1016/0009-9236(95)90228-7&)

46. Gaedigk A, Simon SD, Pearce RE, Bradford LD, Kennedy MJ, Leeder JS. The CYP2D6 activity score: Translating genotype information into a qualitative measure of phenotype. Clin Pharmacol Ther. 2008;83:234–242. doi: 10.1038/sj.clpt.6100406.  [DOI](https://doi.org/10.1038/sj.clpt.6100406) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17971818/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=The%20CYP2D6%20activity%20score:%20Translating%20genotype%20information%20into%20a%20qualitative%20measure%20of%20phenotype&author=A%20Gaedigk&author=SD%20Simon&author=RE%20Pearce&author=LD%20Bradford&author=MJ%20Kennedy&volume=83&publication_year=2008&pages=234-242&pmid=17971818&doi=10.1038/sj.clpt.6100406&)

47. Sistonen J, Sajantila A, Lao O, Corander J, Barbujani G, Fuselli S. CYP2D6 worldwide genetic variation shows high frequency of altered activity variants and no continental structure. Pharmacogenet Genomics. 2007;17:93–101. doi: 10.1097/01.fpc.0000239974.69464.f2.  [DOI](https://doi.org/10.1097/01.fpc.0000239974.69464.f2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17301689/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=CYP2D6%20worldwide%20genetic%20variation%20shows%20high%20frequency%20of%20altered%20activity%20variants%20and%20no%20continental%20structure&author=J%20Sistonen&author=A%20Sajantila&author=O%20Lao&author=J%20Corander&author=G%20Barbujani&volume=17&publication_year=2007&pages=93-101&pmid=17301689&doi=10.1097/01.fpc.0000239974.69464.f2&)

48. Sabbagh A, Darlu P. SNP selection at the NAT2 locus for an accurate prediction of the acetylation phenotype. Genet Med. 2006;8:76–85. doi: 10.1097/01.gim.0000200951.54346.d6.  [DOI](https://doi.org/10.1097/01.gim.0000200951.54346.d6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16481889/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genet%20Med&title=SNP%20selection%20at%20the%20NAT2%20locus%20for%20an%20accurate%20prediction%20of%20the%20acetylation%20phenotype&author=A%20Sabbagh&author=P%20Darlu&volume=8&publication_year=2006&pages=76-85&pmid=16481889&doi=10.1097/01.gim.0000200951.54346.d6&)

49. Walraven JM, Trent JO, Hein DW. Computational and experimental analyses of mammalian arylamine N-acetyltransferase structure and function. Drug Metab Dispos. 2007;35:1001–1007. doi: 10.1124/dmd.107.015040.  [DOI](https://doi.org/10.1124/dmd.107.015040) | [PMC free article](/articles/PMC2085365/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17371801/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Computational%20and%20experimental%20analyses%20of%20mammalian%20arylamine%20N-acetyltransferase%20structure%20and%20function&author=JM%20Walraven&author=JO%20Trent&author=DW%20Hein&volume=35&publication_year=2007&pages=1001-1007&pmid=17371801&doi=10.1124/dmd.107.015040&)

50. Wang JF, Wei DQ, Li L, Zheng SY, Li YX, Chou KC. 3D structure modeling of cytochrome P450 2C19 and its implication for personalized drug design. Biochem Biophys Res Commun. 2007;355:513–519. doi: 10.1016/j.bbrc.2007.01.185.  [DOI](https://doi.org/10.1016/j.bbrc.2007.01.185) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17307149/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20Biophys%20Res%20Commun&title=3D%20structure%20modeling%20of%20cytochrome%20P450%202C19%20and%20its%20implication%20for%20personalized%20drug%20design&author=JF%20Wang&author=DQ%20Wei&author=L%20Li&author=SY%20Zheng&author=YX%20Li&volume=355&publication_year=2007&pages=513-519&pmid=17307149&doi=10.1016/j.bbrc.2007.01.185&)

51. de Graaf C, Oostenbrink C, Keizers PH. et al. Molecular modeling-guided site-directed mutagenesis of cytochrome P450 2D6. Curr Drug Metab. 2007;8:59–77. doi: 10.2174/138920007779315062.  [DOI](https://doi.org/10.2174/138920007779315062) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17266524/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Drug%20Metab&title=Molecular%20modeling-guided%20site-directed%20mutagenesis%20of%20cytochrome%20P450%202D6&author=C%20de%20Graaf&author=C%20Oostenbrink&author=PH%20Keizers&volume=8&publication_year=2007&pages=59-77&pmid=17266524&doi=10.2174/138920007779315062&)

52. Bapiro TE, Hasler JA, Ridderstrom M, Masimirembwa CM. The molecular and enzyme kinetic basis for the diminished activity of the cytochrome P450 2D6.17 (CYP2D6.17) variant. Potential implications for CYP2D6 phenotyping studies and the clinical use of CYP2D6 substrate drugs in some African populations. Biochem Pharmacol. 2002;64:1387–1398. doi: 10.1016/S0006-2952(02)01351-5.  [DOI](https://doi.org/10.1016/S0006-2952(02)01351-5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12392820/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20Pharmacol&title=The%20molecular%20and%20enzyme%20kinetic%20basis%20for%20the%20diminished%20activity%20of%20the%20cytochrome%20P450%202D6.17%20(CYP2D6.17)%20variant.%20Potential%20implications%20for%20CYP2D6%20phenotyping%20studies%20and%20the%20clinical%20use%20of%20CYP2D6%20substrate%20drugs%20in%20some%20African%20populations&author=TE%20Bapiro&author=JA%20Hasler&author=M%20Ridderstrom&author=CM%20Masimirembwa&volume=64&publication_year=2002&pages=1387-1398&pmid=12392820&doi=10.1016/S0006-2952(02)01351-5&)

53. Rogan PK, Svojanovsky S, Leeder JS. Information theory-based analysis of CYP2C19, CYP2D6 and CYP3A5 splicing mutations. Pharmacogenetics. 2003;13:207–218. doi: 10.1097/00008571-200304000-00005.  [DOI](https://doi.org/10.1097/00008571-200304000-00005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12668917/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Information%20theory-based%20analysis%20of%20CYP2C19,%20CYP2D6%20and%20CYP3A5%20splicing%20mutations&author=PK%20Rogan&author=S%20Svojanovsky&author=JS%20Leeder&volume=13&publication_year=2003&pages=207-218&pmid=12668917&doi=10.1097/00008571-200304000-00005&)

54. Tsuchiya Y, Nakajima M, Takagi S, Taniya T, Yokoi T. MicroRNA regulates the expression of human cytochrome P450 1B1. Cancer Res. 2006;66:9090–9098. doi: 10.1158/0008-5472.CAN-06-1403.  [DOI](https://doi.org/10.1158/0008-5472.CAN-06-1403) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16982751/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=MicroRNA%20regulates%20the%20expression%20of%20human%20cytochrome%20P450%201B1&author=Y%20Tsuchiya&author=M%20Nakajima&author=S%20Takagi&author=T%20Taniya&author=T%20Yokoi&volume=66&publication_year=2006&pages=9090-9098&pmid=16982751&doi=10.1158/0008-5472.CAN-06-1403&)

55. Blaisdell J, Mohrenweiser H, Jackson J. et al. Identification and functional characterization of new potentially defective alleles of human CYP2C19. Pharmacogenetics. 2002;12:703–711. doi: 10.1097/00008571-200212000-00004.  [DOI](https://doi.org/10.1097/00008571-200212000-00004) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12464799/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Identification%20and%20functional%20characterization%20of%20new%20potentially%20defective%20alleles%20of%20human%20CYP2C19&author=J%20Blaisdell&author=H%20Mohrenweiser&author=J%20Jackson&volume=12&publication_year=2002&pages=703-711&pmid=12464799&doi=10.1097/00008571-200212000-00004&)

56. Patin E, Harmant C, Kidd KK. et al. Sub-Saharan African coding sequence variation and haplotype diversity at the NAT2 gene. Hum Mutat. 2006;27:720. doi: 10.1002/humu.9438.  [DOI](https://doi.org/10.1002/humu.9438) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16786516/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mutat&title=Sub-Saharan%20African%20coding%20sequence%20variation%20and%20haplotype%20diversity%20at%20the%20NAT2%20gene&author=E%20Patin&author=C%20Harmant&author=KK%20Kidd&volume=27&publication_year=2006&pages=720&pmid=16786516&doi=10.1002/humu.9438&)

57. Gage BF, Lesko LJ. Pharmacogenetics of warfarin: Regulatory, scientific, and clinical issues. J Thromb Thrombolysis. 2008;25:45–51. doi: 10.1007/s11239-007-0104-y.  [DOI](https://doi.org/10.1007/s11239-007-0104-y) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17906972/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Thromb%20Thrombolysis&title=Pharmacogenetics%20of%20warfarin:%20Regulatory,%20scientific,%20and%20clinical%20issues&author=BF%20Gage&author=LJ%20Lesko&volume=25&publication_year=2008&pages=45-51&pmid=17906972&doi=10.1007/s11239-007-0104-y&)

58. Kirchheiner J. CYP2D6 phenotype prediction from genotype: Which system is the best? Clin Pharmacol Ther. 2008;83:225–227. doi: 10.1038/sj.clpt.6100455.  [DOI](https://doi.org/10.1038/sj.clpt.6100455) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18202689/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=CYP2D6%20phenotype%20prediction%20from%20genotype:%20Which%20system%20is%20the%20best?&author=J%20Kirchheiner&volume=83&publication_year=2008&pages=225-227&pmid=18202689&doi=10.1038/sj.clpt.6100455&)
