# Frequency and spectrum of mitochondrial 12S rRNA variants in 440 Han Chinese hearing impaired pediatric subjects from two otology clinics

## Metadata
**Authors:** Zhisen Shen, Jing Zheng, Bobei Chen, Guanghua Peng, Ting Zhang, Shasha Gong, Yi Zhu, Chuqin Zhang, Ronghua Li, Li Yang, Jianjin Zhou, Ting Cai, Lihua Jin, Jianxin Lu, Min-Xin Guan
**Journal:** Journal of Translational Medicine
**Date:** 2011 Jan 4
**DOI:** [10.1186/1479-5876-9-4](https://doi.org/10.1186/1479-5876-9-4)
**PMID:** 21205314
**PMCID:** PMC3029225
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3029225/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC3029225/pdf/1479-5876-9-4.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC3029225/pdf/1479-5876-9-4.pdf)

## Abstract

**Background:** 
Aminoglycoside ototoxicity is one of the common health problems. Mitochondrial 12S rRNA mutations are one of the important causes of aminoglycoside ototoxicity. However, the incidences of 12S rRNA mutations associated with aminoglycoside ototoxicity are less known.

**Methods:** 
A total of 440 Chinese pediatric hearing-impaired subjects were recruited from two otology clinics in the Ningbo and Wenzhou cities of Zhejiang Province, China. These subjects underwent clinical, genetic evaluation and molecular analysis of mitochondrial 12S rRNA. Resultant mtDNA variants were evaluated by structural and phylogenetic analysis.

**Results:** 
The study samples consisted of 227 males and 213 females. The age of all participants ranged from 1 years old to 18 years, with the median age of 9 years. Ninety-eight subjects (58 males and 40 females) had a history of exposure to aminoglycosides, accounting for 22.3% cases of hearing loss in this cohort. Molecular analysis of 12S rRNA gene identified 41 (39 known and 2 novel) variants. The incidences of the known deafness-associated 1555A > G, 1494C > T and 1095T > C mutations were 7.5%, 0.45% and 0.91% in this entire hearing-impaired subjects, respectively, and 21.4%, 2% and 2% among 98 subjects with aminoglycoside ototoxicity, respectively. The structural and phylogenetic evaluations showed that a novel 747A > G variant and known 839A > G, 1027A > G, 1310C > T and 1413T > C variants conferred increased sensitivity to aminoglycosides or nonsyndromic deafness as they were absent in 449 Chinese controls and localized at highly conserved nucleotides of this rRNA. However, other variants were polymorphisms. Of 44 subjects carrying one of definite or putative deafness-related 12S rRNA variants, only one subject carrying the 1413T > C variant harbored the 235DelC/299DelAT mutations in the GJB2 gene, while none of mutations in GJB2 gene was detected in other 43 subjects.

**Conclusions:** 
Mutations in mitochondrial 12S rRNA accounted for ~30% cases of aminoglycoside-induced deafness in this cohort. Our data strongly support the idea that the mitochondrial 12S rRNA is the hot spot for mutations associated with aminoglycoside ototoxicity. These data have been providing valuable information and technology to predict which individuals are at risk for ototoxicity, to improve the safety of aminoglycoside antibiotic therapy, and eventually to decrease the incidence of deafness.

### Background

Aminoglycoside ototoxicity is one of the common health problems. Mitochondrial 12S rRNA mutations are one of the important causes of aminoglycoside ototoxicity. However, the incidences of 12S rRNA mutations associated with aminoglycoside ototoxicity are less known.

### Methods

A total of 440 Chinese pediatric hearing-impaired subjects were recruited from two otology clinics in the Ningbo and Wenzhou cities of Zhejiang Province, China. These subjects underwent clinical, genetic evaluation and molecular analysis of mitochondrial 12S rRNA. Resultant mtDNA variants were evaluated by structural and phylogenetic analysis.

### Results

The study samples consisted of 227 males and 213 females. The age of all participants ranged from 1 years old to 18 years, with the median age of 9 years. Ninety-eight subjects (58 males and 40 females) had a history of exposure to aminoglycosides, accounting for 22.3% cases of hearing loss in this cohort. Molecular analysis of 12S rRNA gene identified 41 (39 known and 2 novel) variants. The incidences of the known deafness-associated 1555A > G, 1494C > T and 1095T > C mutations were 7.5%, 0.45% and 0.91% in this entire hearing-impaired subjects, respectively, and 21.4%, 2% and 2% among 98 subjects with aminoglycoside ototoxicity, respectively. The structural and phylogenetic evaluations showed that a novel 747A > G variant and known 839A > G, 1027A > G, 1310C > T and 1413T > C variants conferred increased sensitivity to aminoglycosides or nonsyndromic deafness as they were absent in 449 Chinese controls and localized at highly conserved nucleotides of this rRNA. However, other variants were polymorphisms. Of 44 subjects carrying one of definite or putative deafness-related 12S rRNA variants, only one subject carrying the 1413T > C variant harbored the 235DelC/299DelAT mutations in the *GJB2 *gene, while none of mutations in *GJB2 *gene was detected in other 43 subjects.

### Conclusions

Mutations in mitochondrial 12S rRNA accounted for ~30% cases of aminoglycoside-induced deafness in this cohort. Our data strongly support the idea that the mitochondrial 12S rRNA is the hot spot for mutations associated with aminoglycoside ototoxicity. These data have been providing valuable information and technology to predict which individuals are at risk for ototoxicity, to improve the safety of aminoglycoside antibiotic therapy, and eventually to decrease the incidence of deafness.

## Background

Aminoglycosides, such as gentamicin and tobramycin, are of great clinical importance for the treatment of bacterial infections. The use of these drugs can frequently lead to toxicity, which involves the renal, auditory and vestibular systems [[1](#B1),[2](#B2)]. The renal impairment is usually reversible, whereas the auditory and vestibular ototoxicity is usually irreversible. In familial cases of ototoxicity, aminoglycoside hypersensitivity is often maternally transmitted, suggesting that mutation(s) in mitochondrial DNA (mtDNA) is one of molecular bases for this susceptibility [[1](#B1),[2](#B2)]. As mitochondrial ribosomes share more similarities to bacterial ribosomes than do cytosolic counterparts, the human mitochondrial 12 rRNA was proposed to be the primary targeting site for aminoglycosides [[3](#B3),[4](#B4)]. The mutational analysis of mitochondrial genome in several Chinese and Arab-Israeli families with maternally transmitted aminoglycoside ototoxicity or/and nonsyndromic deafness led to the landmark discovery of the 12S rRNA 1555A > G mutation in 1993 [[3](#B3)]. Subsequently, the 1555A > G mutation has been found to be responsible for both aminoglycoside-induced and nonsyndromic hearing loss in many families worldwide [[4](#B4)-[10](#B10)]. On the other hand, the 12S rRNA 1494C > T mutation has been associated with both aminoglycoside-induced and nonsyndromic hearing loss only in some Chinese and Spanish families [[11](#B11)-[13](#B13)].

The 1555A > G and 1494C > T mutations are located at the highly conserved A-site of 12S rRNA [[4](#B4),[11](#B11)]. The A1555 and C1494 (equivalent to positions 1491 and 1409 of *Escherichia coli *16S rRNA, respectively) are in apposition to each other but do not form a base-pair. The 1555A > G or 1494C > T mutation creates a new G-C or A-U pair base-pair, thereby extending the adjacent stem by one nucleotide and making the secondary structure of mitochondrial 12S rRNA more closely resemble the corresponding region of *E. coli *16S rRNA and altering binding properties of aminoglycosides such as paromomycin, neomycin, gentamicin, and kanamycin at the A-site of 12S rRNA [[14](#B14)]. Thus, the administration of aminoglycosides can induce or worsen hearing loss in these subjects carrying the 1555A > G or 1494C > T mutation. In the absence of aminoglycosides, matrilineal relatives within and among families carrying the 1555A > G or 1494C > T mutation exhibited a considerable phenotypic variation with respect to severity and age-of-onset and penetrance of hearing loss [[4](#B4)-[13](#B13)]. Therefore, additional modifier factors such as aminoglycosides, nuclear and mitochondrial genetic modifiers contributed to the phenotypic variability of these mtDNA mutations [[11](#B11),[15](#B15)-[18](#B18)].

However, the incidences of the 1555A > G and 1494C > T mutations were only reported in the some cohorts of hearing-impaired subjects [[3](#B3),[19](#B19)-[24](#B24)]. As these mutations are only responsible for a portion of patients with hearing loss, it is anticipated that additional mutations causing hearing loss can be found in the same gene. In the present investigation, we carried out a systematic and extended mutational screening of 12S rRNA gene in a cohort of 440 hearing-impaired Han Chinese pediatric subjects from two otology clinics at Ningbo and Wenzhou, Zhejiang Province, China. Mutational analysis of 12S rRNA gene in these subjects identified the known 1555A > G and 1494C > T mutations as well as 39 other variants. Those variants have been further evaluated by phylogenetic analysis, structure-function relation and allelic frequency of these variants in the 449 Han Chinese controls from the same region. To examine if the *GJB2 *gene contributed to a deafness phenotype, we performed the mutational screening of *GJB2 *gene in 39 subjects carrying the known deafness-associated 12S rRNA mutations and 5 subjects carrying one of 5 putative 12S rRNA mutations.

## Methods

### Subjects and audiological examinations

A total of 440 unrelated hearing-impaired Chinese subjects, who were younger than 18 years old two otology clinics from Zhejiang Province, were enrolled in this study under an institutional review board-approved protocol of informed consent at the Cincinnati Children's Hospital Medical Center Institutional Review Board and Ethics Committee of Wenzhou Medical College, China. A comprehensive history and physical examination for these participating subjects were performed to identify any syndromic findings, the history of the use of aminoglycosides, genetic factors related to the hearing impairment. An age-appropriate audiological examination was performed and this examination included pure-tone audiometry (PTA) and/or auditory brainstem response (ABR), immittance testing and Distortion product otoacoustic emissions (DPOAE). The PTA was calculated from the average of the audiometric thresholds at 500, 1000, 2000, 4000 and 8000 Hz. The severity of hearing impairment was classified into five grades: normal <26 Decibel (dB); mild = 26-40 dB; moderate = 41-70 dB; severe = 71-90 dB; and profound >90 dB. The 449 control DNA used for screening for the presence of mtDNA variants were obtained from a panel of unaffected Han Chinese subjects from the same region.

### Mutational analysis of mitochondrial 12S rRNA gene

Genomic DNA was isolated from whole blood of participants using Puregene DNA Isolation Kits (Gentra Systems, Minneapolis, Minnesota, USA). Subject's DNA fragments spanning the 12S rRNA gene were amplified by PCR using oligodeoxynucleotides corresponding to positions 618-635 and 1988-2007 [[25](#B25)]. Each fragment was purified and subsequently analyzed by direct sequencing in an ABI 3700 automated DNA sequencer using the Big Dye Terminator Cycle (Applied Biosystems, Foster City, California, USA) sequencing reaction kit. The resultant sequence data were compared with the updated consensus Cambridge sequence (GenBank accession number: [NC_012920](https://ncbi.nlm.nih.gov/nucleotide/NC_012920)) [[26](#B26)]. The homoplasmy of the 1555A > G and 1494C > T mutations in these subjects were determined as detailed previously [[7](#B7),[11](#B11)]. The frequency of variants in the 12S rRNA gene in 449 Chinese control subjects was determined by direct sequencing of PCR products as described above.

### Mutational analysis of GJB2 gene

The DNA fragments spanning the entire coding region of *GJB2 *gene were amplified by PCR using the following oligodeoxynucleotides: forward-5'TATGACACTCCCCAGCACAG3' and reverse-5'GGGCAATGCTTAAACTGGC3'. PCR amplification and subsequent sequencing analysis were performed as detailed elsewhere [[10](#B10)]. The results were compared with the wild type *GJB2 *sequence (Version 1, GenBank accession number: [M86849](https://ncbi.nlm.nih.gov/nucleotide/M86849)) to identify the mutations.

### Structural analysis

The published secondary structures for the 12S rRNA [[27](#B27),[28](#B28)] were used to define the stem and loop structure. The secondary structure of human mitochondrial 12S rRNA was predicted by using the RnaViz program [[29](#B29)].

### Phylogenetic analysis

A total of 14 primate mitochondrial 12S rRNA sequences (Genbank), as shown in Table [1](#T1), were used in the interspecies analysis. These include *Homo sapiens, Gorilla gorilla, Pan paniscus, Pan troglodytes, Pongo pygmaeus, Pongo abelii, Hylobates lar, Macaca mulatta, Macaca sylvanus, Papio hamadryas, Cebus albifrons, Tarsius bancanus, Nycticebus coucang*, and *Lemur catta*. The conservation index (CI) was calculated by comparing the human nucleotide variants with other 13 primates. The CI was then defined as the percentage of species from the list of 14 different primates that have the wild-type nucleotide at that position.

### Table 1.

| Species name | GenBank accession number |
| --- | --- |
| Homo sapiens | NC_012920 |
|   |  |
| Gorilla gorilla | NC_001645 |
|   |  |
| Pan paniscus | NC_001644 |
|   |  |
| Pan troglodytes | NC_001643 |
|   |  |
| Pongo pygmaeus | NC_001646 |
|   |  |
| Pongo abelii | NC_002083 |
|   |  |
| Hylobates lar | NC_002082 |
|   |  |
| Macaca mulatta | NC_005943 |
|   |  |
| Macaca sylvanus | NC_002764 |
|   |  |
| Papio hamadryas | NC_001992 |
|   |  |
| Cebus albifrons | NC_002763 |
|   |  |
| Tarsius bancanus | NC_002811 |
|   |  |
| Nycticebus coucang | NC_002765 |
|   |  |
| Lemur catta | NC_004025 |

Table 1 Caption: mtDNA sequence data of 14 primate species

## Results

### Study samples

The study samples consisted of 227 males and 213 females. The age of all participants ranged from 1 years old to 18 years, with the median age of 9 years. All participants were Han Chinese recruited from ENT clinics at Ningbo and Wenzhou Cities of Zhejiang Province, China. Based on a clinician review of the medical record, 98 subjects (58 males and 40 females) had a history of exposure to aminoglycosides including gentamicin, streptomycin and kanamycin, accounting for 22.3% cases of hearing loss in this cohort. These subjects, due to infections or other illness, received a conventional daily dosage of aminoglycosides (3 5 mg/kg/dose every 8 h for gentamicin or 15 25 mg/kg/dose every 12 h for streptomycin, 15 mg/kg/dose every 8 h for kanamycin) at younger than 10 years old. Hearing impairment occurred from 3 days to three months after the administration of drugs. Audiological evaluation showed that 22 subjects had severe hearing loss and 76 individuals exhibited profound hearing loss. Furthermore, there was the wide range of severity of hearing loss in 342 affected subjects who did not have a history of exposure to aminoglycosides: 149 subjects exhibited profound hearing loss, 167 subjects had severe hearing loss and 26 individuals suffered from moderate hearing loss. The onset of the hearing loss ranged from congenital to 10 years old.

### Mutational analysis of mitochondrial 12S rRNA gene

Fragments spanning 12S rRNA gene were PCR-amplified from genomic DNA of 440 hearing-impaired Chinese subjects and each fragment was purified and subsequently analyzed by DNA sequencing. Comparison of the resultant sequence with the Cambridge consensus sequence [[26](#B26)] identified 41 nucleotide changes in the 12S rRNA gene as shown in Table [2](#T2). All the nucleotide changes were verified by sequence analysis of both strands and appeared to be homoplasmy. Of these, 2 subjects with profound hearing loss carried the 1494C > T mutation. Both subjects carrying the 1494C > T mutation had a history of exposure to aminoglycosides. These translate to a frequency of ~0.45% for the 1494C > T mutation in this Chinese pediatric deafness population. Among these, 33 hearing-impaired subjects carrying the 1555A > G mutation were composed of 21 subjects who had a history of exposure to aminoglycosides and 12 individuals who did not receive aminoglycoside treatment. These translate to a frequency of ~7.5% for the 1555A > G mutation in this entire Chinese pediatric deafness population, and approximately 21.4% in cases of aminoglycoside ototoxicity in this Chinese pediatric population. Furthermore, 4 subjects harbored the known deafness-associated 1095T > C mutation [[30](#B30),[31](#B31)] and 11 subjects carried the putative deafness-associated mutations at position of 961 (961insC and 961T > C) [[7](#B7),[21](#B21),[32](#B32),[33](#B33)], respectively.

### Table 2.

| Position | Replacement | Conservationindex (%)a | WCbase-pairsb | Previouslyreportedc | Number ofaffected subjects | Percentage (%) | Number ofcontrols (number/449) | Percentage (%) |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
| 663 | A to G | 78.6 | ↓A-U | Yes | 15 | 3.40 | 5 | 1.1 |
| 681 | T to C | 85.7 | ↓U-A | Yes | 5 | 1.13 | 8 | 1.8 |
| 709 | G to A | 64.3 | ↓G-C | Yes | 90 | 20.41 | 102 | 22.7 |
| 723 | A to G | 28.6 |   | Yes | 2 | 0.45 | 2 | 0.4 |
| 735 | A to G | 78.6 |   | Yes | 2 | 0.45 | 5 | 1.11 |
| 747 | A to G | 100 | ↓A-U | No | 1 | 0.23 | 0 | 0 |
| 752 | C to T | 100 |   | Yes | 26 | 6.12 | 17 | 3.8 |
| 789 | T to C | 85.7 |   | Yes | 1 | 0.23 | 1 | 0.2 |
| 813 | A to G | 28.6 |   | Yes | 1 | 0.23 | 0 | 0 |
| 827 | A to G | 92.9 |   | Yes | 16 | 3.63 | 12 | 2.7 |
| 839d | A to G | 78.6 | ↓A-U | Yes | 1 | 0.23 | 0 | 0 |
| 929 | A to T | 42.9 | ↓A-U | No | 1 | 0.23 | 0 | 0 |
| 942 | A to G | 64.3 |   | Yes | 1 | 0.23 | 0 | 0 |
| 951 | G to A | 92.9 | ↓G-C | Yes | 2 | 0.45 | 2 | 0.4 |
| 953 | T to C | 57.1 |   | Yes | 1 | 0.23 | 0 | 0 |
| 961 | insC | 42.9 |   | Yes | 9 | 2.04 | 14 | 3.1 |
| 961 | T to C | 42.9 |   | Yes | 2 | 0.23 | 4 | 0.9 |
| 980 | T to C | 64.3 | ↓U-A | Yes | 3 | 0.68 | 0 | 0 |
| 990 | T to C | 71.4 | ↓U-A | Yes | 1 | 0.23 | 0 | 0 |
| 1005 | T to C | 35.7 |   | Yes | 21 | 4.76 | 22 | 4.9 |
| 1009 | C to T | 21.4 |   | Yes | 6 | 1.36 | 8 | 1.8 |
| 1027 | A to G | 92.9 |   | Yes | 1 | 0.23 | 0 | 0 |
| 1041 | A to G | 42.9 |   | Yes | 2 | 0.45 | 4 | 0.9 |
| 1048 | C to T | 57.1 |   | Yes | 10 | 2.27 | 11 | 2.4 |
| 1095 | T to C | 92.9 | ↓U-A | Yes | 4 | 0.91 | 1 | 0.2 |
| 1107 | T to C | 85.7 |   | Yes | 36 | 8.39 | 25 | 5.6 |
| 1119 | T to C | 50.0 |   | Yes | 13 | 2.95 | 17 | 3.8 |
| 1187 | T to C | 57.1 |   | Yes | 1 | 0.23 | 0 | 0 |
| 1282 | G to A | 71.4 |   | Yes | 1 | 0.23 | 0 | 0 |
| 1310 | C to T | 85.7 | ↓G-C | Yes | 1 | 0.23 | 0 | 0 |
| 1382 | A to C | 92.9 | ↓A-U | Yes | 14 | 3.17 | 9 | 2.0 |
| 1391 | T to C | 64.3 |   | Yes | 1 | 0.23 | 1 | 0.2 |
| 1393 | G to A | 28.6 | ↑A-U | Yes | 2 | 0.45 | 0 | 0 |
| 1413 | T to C | 78.6 | ↑C-G | Yes | 1 | 0.23 | 0 | 0 |
| 1442 | G to A | 42.9 |   | Yes | 1 | 0.23 | 0 | 0 |
| 1462 | G to A | 50.0 |   | Yes | 1 | 0.23 | 0 | 0 |
| 1494 | C to T | 78.6 | ↑U-A | Yes | 2 | 0.45 | 0 | 0 |
| 1503 | G to A | 50.0 | ↑A-U | Yes | 1 | 0.23 | 0 | 0 |
| 1541 | T to C | 78.6 |   | Yes | 6 | 1.36 | 4 | 0.9 |
| 1555 | A to G | 85.7 | ↑A-U | Yes | 33 | 7.5 | 0 | 0 |
| 1598 | G to A | 50 |   | Yes | 12 | 2.72 | 9 | 2.0 |

Table 2 Caption: Variants in the mitochondrial 12S rRNA gene in 440 hearing-impaired Han Chinese subjects

In addition to the mutations mentioned above, there were 34 known and 2 novel variants in the 12S rRNA gene [[34](#B34)]. These variants were first evaluated by examining the allelic frequency in 449 Han Chinese control population. Nineteen out of 41 variants were absent in this Chinese control population. Of other 22 variants, the frequencies of 8 variants were <1% in 449 Chinese controls, while the allelic frequency of other 14 variants was >1% in this control population. Furthermore, we used the secondary structure of 12S rRNA [[29](#B29),[35](#B35)] to localize each variant with either a stem or a loop and to analyze if the base changes within stems alter classic Watson-Crick (WC) base pair [[29](#B29),[35](#B35)]. As shown in Figure [1](#F1), 23 variants were located at the loops, while 18 variants occurred in the stems of this rRNA. As shown in Table [2](#T2) and Figure [1](#F1), 5 variants 1393G > A, 1413T > C, 1494C > T, 1503G > A and 1555A > G created a putative base-pairing(s), while 12 variants 663A > G, 681T > C, 709G > A, 747A > G, 839A > G, 929A > T, 951G > A, 980T > C, 990T > C, 1095T > C, 1310C > T and 1382A > C abolished a putative base pairing(s). This suggested that the nucleotide variants were more frequent in loops than in stems. In addition, phylogenetic analysis was performed by comparing the human 12S rRNA nucleotide variants with other 13 primates. As shown in Table [2](#T2), conservation index (CI) among the variants ranged from 21.4% (1009C > T variant) to 100% (752C > T and 747A > G variants). In particular, CI of 18 variants including 1555A > G and 1494C > T mutations were >78%, CI of other 13 variants was between 78% and 50% and CI for the remaining variants was <50%. In addition to the 1555A > G and 1494C > T mutations, the novel 747A > G variant and the known 839A > G, 1027A > G, 1310C > T and 1413T > C variants [[22](#B22),[34](#B34)], which are absent in the 449 Chinese controls and whose CIs were >78%, were the putative deafness-associated variants. On the other hand, other variants such as 663A > G, 681T > C, 735A > G, 752C > T, 827A > G, 1107T > C and 1382A > C, whose CIs were >78%, which were present in the controls, appeared to be the polymorphisms.

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d5bc/3029225/54ce5ce373af/1479-5876-9-4-1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3029225_1479-5876-9-4-1.jpg)

Structure and sequence variants of human mitochondrial 12S rRNA. The secondary structure was predicted by using the RnaViz program (De Rijk and De Wachter, 1997). The variants were indicated by arrows.

### Clinical characterization of 39 hearing-impaired Chinese subjects carrying one of known or 12S rRNA mutations

Comprehensive medical evaluations of 33 probands carrying the 1555A > G mutation, two subjects harboring the 1494C > T mutation and four individuals carrying the 1095T > C mutation showed no other clinical abnormalities, including diabetes, muscular diseases, visual loss and neurological disorders. As shown in Table [3](#T3), audiological assessments of 33 subjects carrying the 1555A > G mutation showed that 15, 3 and 3 subjects with the aminoglycoside treatments exhibited profound, severe or moderate hearing loss, respectively. Moreover, 12 individuals, who did not have a history of exposure to aminoglycosides, exhibited a variety of severity and age-of-onset of hearing impairment. The age-of-onset of hearing loss in these subjects ranged from infant to 18 years, with an average of 6 years. Audiometric studies showed that 3 individuals suffered from profound hearing impairment, 5 subjects exhibited severe hearing impairment, 2 probands had moderate hearing impairment and 2 subjects exhibited mild hearing impairment. Furthermore, two subjects carrying the 1494C > T mutation exhibited severe or profound hearing loss, respectively. Among four subjects carrying the 1095T > C mutation, two subjects who was treated with aminoglycosides had profound and severe hearing loss, respectively, while two individuals who did not have a history to exposure exhibited profound and mild hearing impairment.

### Table 3.

| 12S rRNA mutation | GJB2 genemutation | Subjects | Gender | Audiometic configuration | Age-at-onset(years) | PTA a(dB)right ear | PTA (dB)left ear | Use ofdrugs | Level of hearingimpairment |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| 1555A > G | polymorphism | FE003-IV-1 | M | Slope | 1 | 98 | 98 | Yes | Profound |
| 1555A > G | polymorphism | FE007-IV-6 | M | Slope | 2 | 100 | 100 | Yes | Profound |
| 1555A > G | polymorphism | FE008-III-7 | F | Slope | 10 | 58 | 78 | No | Severe |
| 1555A > G | polymorphism | FE0128-IV-1 | F | Slope | 2 | 102 | 98 | Yes | Profound |
| 1555A > G | polymorphism | FE019-IV-1 | F | Slope | 2 | 67 | 82 | Yes | Severe |
| 1555A > G | polymorphism | FE020-III-15 | M | Slope | 8 | 81 | 74 | Yes | Severe |
| 1555A > G | polymorphism | FE036-III-1 | M | Slope | 10 | 58 | 49 | No | Moderate |
| 1555A > G | polymorphism | FE081-III-1 | M | Slope | 16 | 50 | 56 | Yes | Moderate |
| 1555A > G | polymorphism | FE122-III-2 | F | Slope | 2 | 71 | 53 | Yes | Moderate |
| 1555A > G | polymorphism | FE141-III-1 | F | Slope | 2 | 24 | 30 | No | Mild |
| 1555A > G | polymorphism | FE154-III-1 | F | Slope | 18 | 61 | 60 | No | Moderate |
| 1555A > G | polymorphism | FE160-III-1 | F | Slope | 5 | 61 | 74 | No | Severe |
| 1555A > G | polymorphism | FE163-III-3 | M | Flat | 2 | 110 | 99 | Yes | Profound |
| 1555A > G | polymorphism | FE300-II-12 | F | Slope | 3 | 110 | 105 | Yes | Profound |
| 1555A > G | polymorphism | FE304-II-2 | F | Slope | 3 | 100 | 80 | Yes | Profound |
| 1555A > G | polymorphism | FE317-III-10 | M | Slope | 4 | 94 | 93 | Yes | Profound |
| 1555A > G | polymorphism | FE350-III-1 | F | Flat | 1 | 100 | 100 | Yes | Profound |
| 1555A > G | polymorphism | NB038-III-1 | M | Flat | 1 | 90 | 87 | Yes | Profound |
| 1555A > G | polymorphism | NB048-III-2 | F | Slope | 1 | 78 | 81 | No | Severe |
| 1555A > G | polymorphism | NB052-III-2 | F | Flat | 1 | 120 | 102 | No | Profound |
| 1555A > G | polymorphism | NB076-III-1 | M | Flat | 1 | 118 | 118 | Yes | Profound |
| 1555A > G | polymorphism | NB078-III-2 | F | Flat | 6 | 110 | 117 | Yes | Profound |
| 1555A > G | polymorphism | NB079-III-1 | M | Flat | 1 | 102 | 102 | Yes | Profound |
| 1555A > G | polymorphism | NB094-III-2 | F | Flat | 1 | 117 | 117 | Yes | Profound |
| 1555A > G | polymorphism | NB111-III-2 | F | Slope | 3 | 83 | 86 | Yes | Severe |
| 1555A > G | polymorphism | NB126-III-2 | F | Slope | 2 | 84 | 92 | No | Profound |
| 1555A > G | polymorphism | NB137-III-1 | F | Slope | 2 | 111 | 115 | No | Profound |
| 1555A > G | polymorphism | ZX019-II-2 | F | Slope | 5 | 59 | 62 | Yes | Moderate |
| 1555A > G | polymorphism | ZX022-III-3 | M | Flat | 2 | 101 | 102 | Yes | Profound |
| 1555A > G | polymorphism | ZX025-III-14 | M | Flat | 1 | 113 | 108 | Yes | Profound |
| 1555A > G | polymorphism | ZX028-IV-1 | F | Slope | 3 | 87 | 87 | No | Severe |
| 1555A > G | polymorphism | ZX037-II-7 | M | Flat | 5 | 30 | 27 | No | Mild |
| 1555A > G | polymorphism | ZX047-III-1 | M | Slope | 6 | 78 | 79 | No | Severe |
| 1494C > T | polymorphism | FE247-III-1 | M | Flat | 3 | 100 | 100 | Yes | Profound |
| 1494C > T | polymorphism | NB133-II-1 | M | Slope | 2 | 86 | 88 | Yes | Severe |
| 1095T > C | polymorphism | FE312 | F | Slope | 9 | 82 | 80 | Yes | Severe |
| 1095T > C | polymorphism | NB021 | M | Slope | 10 | 36 | 37 | No | Mild |
| 1095T > C | polymorphism | NB067 | M | Flat | 1 | 93 | 93 | No | Profound |
| 1095T > C | polymorphism | NB100 | F | Flat | 5 | 100 | 95 | Yes | Profound |
| 747A > G | polymorphism | NS016-III-4 | M | Flat | 1 | 100 | 80 | Yes | Profound |
| 839A > G | polymorphism | NB005-III-1 | F | Flat | 1 | 78 | 81 | Yes | Severe |
| 1027A > G | polymorphism | FE239-II-1 | M | Slope | 18 | 82 | 85 | No | Severe |
| 1310C > T | polymorphism | NS071-IV-1 | M | Flat | 1 | 91 | 92 | No | Profound |
| 1413T > C | 235DelC/299DelAT | ZX039-IV-1 | F | Flat | 1 | 114 | 111 | No | Profound |

Table 3 Caption: Summary of clinical and molecular data for 44 Han Chinese subjects carrying the putative 12S rRNA mutations.

### Clinical and genetic characterization of 5 hearing-impaired Chinese subjects carrying one of 5 putative 12S rRNA mutation

Comprehensive medical histories of 5 probands carrying one of 5 putative 12S rRNA mutations and other members in these families showed no other clinical abnormalities, including diabetes, muscular diseases, visual loss and neurological disorders. As shown in Table [3](#T3), two subjects received a regular dose of gentamicin for various illnesses at the age of 1 year, while other three subjects did not have a history of exposure to aminoglycosides. There was no evidence that these subjects had any known cause to account for hearing loss. Audiological examination indicated that 2 subjects suffered from severe hearing loss and 3 subjects exhibited profound hearing loss. Variable patterns of audiometric configurations were detected in these subjects: 1 subject with slope-shaped pattern and 4 individuals with flat-shaped pattern. Besides the proband, no one of the NS016 pedigree carrying the 747A > G variant suffered from hearing loss. The pedigree FE239 with three matrilineal affected relatives carrying the 1027A > G mutation showed suggestively maternally transited hearing loss. Furthermore, two matrilineal relatives of 14 members in the pedigree NB005 carrying the 839A > G mutation, as shown in Figure [2](#F2), suffered from hearing loss. In addition, four of 16 members in the pedigree ZX039 carrying the 1413T > C variant experienced the loss of hearing.

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d5bc/3029225/705233e0dc40/1479-5876-9-4-2.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3029225_1479-5876-9-4-2.jpg)

Five Han Chinese pedigrees with aminoglycoside-induced and nonsyndromic hearing impairment. Hearing impaired individuals are indicated by filled symbols. Arrowhead denotes probands. Asterisks denote individuals who had a history of exposure to aminoglycosides.

### Mutational analysis of GJB2 gene

To examine if the *GJB2 *gene contributed to a deafness phenotype, we performed the mutational screening of *GJB2 *gene in 39 subjects carrying the known deafness-associated 12S rRNA mutations and 5 subjects carrying one of 5 putative 12S rRNA mutations. As shown in Table [3](#T3), the subject ZX039-IV-1 carrying the 12S rRNA 1413T > C mutation harbored the known 235DelC/299DelAT mutation in the *GJB2 *gene [[36](#B36),[37](#B37)], while none of other mutations in *GJB2 *gene was detected in other 43 affected subjects. Indeed, the absence of mutation in the *GJB2 *gene in those subjects with hearing impairment indicated that the *GJB2 *gene did not contribute to the deafness phenotype in those subjects.

## Discussion

The cohort of Chinese pediatric hearing-impaired subjects consisted of 98 subjects with aminoglycoside ototoxicity and 342 subjects, who did not have a history of exposure to aminoglycosides. Of known deafness-associated 12S rRNA mutations, the 1555A > G mutation accounted for 7.5% cases of this Chinese clinical population, while incidences of this mutation were 1.76% and 3.96% in two large cohorts of hearing impaired pediatric Han Chinese subjects from schools of deaf children [[22](#B22),[36](#B36)]. In the present study, the incidences of the 1555A > G mutation were 2.7% and 21.4% cases of nonsyndromic and aminoglycoside-induced hearing loss, respectively. In fact, the incidences of the 1555A > G mutation varied among different ethnic origins. With regard to the subjects with aminoglycoside ototoxicity, the incidences of the 1555A > G mutation were 33% in a small Japanese cohort [[19](#B19)] 13%, 10.4% and 5% in three Chinese cohorts [[3](#B3),[21](#B21),[22](#B22)] and ~17% in the two white cohorts from United States and Spain [[5](#B5),[32](#B32),[33](#B33)]. However, the incidence of 1555A > G mutation in nonsyndromic hearing loss was much lower than in those with aminoglycoside ototoxicity. In two white cohorts with nonsyndromic hearing loss, the frequency of the 1555A > G mutation varied from 0.6% to 2.5% [[20](#B20),[24](#B24)], while the incidence of the 1555A > G mutation in several Asian cohorts ranged from 2.9% to 5.3% [[19](#B19),[21](#B21)-[23](#B23)]. Thus, the large proportion of subjects with aminoglycoside ototoxicity in this cohort may contribute to higher incidence of the 1555A > G mutation than other cohorts. On the other hand, the incidences of the 1494C > T mutation appeared to be lower than those of the 1555A > G mutation. In this cohort, two subjects carrying the 1494C > T mutation had a history of exposure to aminoglycosides. This data appeared to be higher than the previous reports that three familial cases of 1340 sporadic Spanish hearing-impaired subjects carried the 1494C > T mutation [[12](#B12)] and three cases of 1642 pediatric deaf children [[22](#B22)]. Therefore, these two known 12S rRNA mutations account for from 4% to 8% cases among these Chinese hearing-impaired populations [[10](#B10)].

Of other known deafness-mutations, the frequency of the 1095T > C mutation was 0.91% in this cohort. The 1095T > C mutation, whose CI was 92.9%, occurred in one of 449 Chinese controls. This mutation has been found in several genetically-unrelated families with nonsyndromic and aminoglycoside-induced hearing loss [[21](#B21),[22](#B22),[30](#B30),[31](#B31)]. This T-to-C transition disrupted an evolutionarily conserved base-pair at stem loop of the helix 25 of 12S rRNA [[27](#B27)]. This nucleotide is also located at the P-site of ribosome, suggesting an important role in the initiation of mitochondrial protein synthesis [[31](#B31)]. Furthermore, the frequency of mutations at position 961 including 961insC and 961T > C was 2.27% in this pediatric population. Although mutations at this position have been implicated to be associated with hearing loss in different ethnic groups [[21](#B21),[22](#B22),[32](#B32),[33](#B33)], the lower CI (42.9%) and presence of 4% in the controls indicated that mutations in this position were polymorphisms.

A total of 41 (39 known and 2 novel) variants in 12S rRNA gene were identified in this cohort. Similar to other mtDNA variations, these variants can be grouped into three categories: neutral, adaptive and deleterious [[35](#B35)]. To identify putative deleterious mutation, these variants were further evaluated using following three criteria: 1). Absent in the 449 Chinese controls; 2). CI is >78%, proposed by Ruiz-Pesini and Wallace [[35](#B35)]; 3). Potential structural and functional alterations [[22](#B22)]. Among these variants, 19 variant were absent in the 449 Han Chinese controls, while the frequency of other variants ranged from 0.2% (13 variants such as 789T > C) to 22.7% (709G > A variant) in this Chinese control population. In particular, some of these variants occurring at high frequencies of both control and patient populations were the mitochondrial haplogroup specific variants [[36](#B36)]. These included the 663A > G variant of haplogroup A, the 827A > G and 1119T > C variants of haplogroup B4, the 709G > A and 1598G > A variants of haplogroup B5, the 1382A > C variant of haplogroup D4, the 681T > C, 752C > T, 1048C > T and 1107T > C variants of D5 haplogroup, the haplogroup F2 specific variant 1005T > C, the 1041A > G variant of haplogroup M9a, and the 1541T > C variant of haplogroup R5b [[38](#B38)]. Apparently, these haplogroup specific variants were adaptive or neutral but unlikely deleterious.

Phylogenetic analysis showed that CIs of 28 variants were more than 78%. Despite their higher CI, the 14 variants such as 663A > G, 681T > C, 752C > T, 735A > G, 827A > G, 1107T > C, 1382A > C and 1438A > G were present in the controls. On the other hand, the CIs for other 7 variants including 1555A > G and 1494C > T were at least 78% but these variants were absent in 449 Chinese controls. Based on the predicted secondary structure of mitochondrial 12S rRNA [[27](#B27),[35](#B35)], 23 variants were located at the loops and 18 variants occurred in the stems of this rRNA. Among these variants, 11 variants including the 1095T > C disrupted a WC base pairing(s) of 12S rRNA, while 5 variants including the 1555A > G and 1494C > T created a novel WC base-pairing(s) of this rRNA [[28](#B28),[29](#B29)]. In fact, the 1555A > G or 1494C > T mutation made the mitochondrial ribosome more bacteria-like [[4](#B4),[11](#B11),[14](#B14)]. Functional characterization demonstrated that the 1555A > G or 1494C > T mutation conferred sensitivity to aminoglycosides [[11](#B11),[15](#B15),[16](#B16),[18](#B18)]. Thus, individuals carrying either of mutations are predisposed to hearing loss. Indeed, the novel 747A > G variant and the known 839A > G, 1310C > T and 1413T > C variants [[22](#B22),[34](#B34)], which resided at the stems of 12S rRNA, were fitted with three criteria for the pathogenic mutations as described above. Furthermore, the 1027A > G variant, whose location was at a loop in the 12S rRNA and whose CI was 92.9%, was absent in 449 Han Chinese controls. Thus, alterations of the tertiary or quaternary structure of 12S rRNA by these putative variants may lead to significant effects on function, thereby contributing to the deafness phenotype. Genetic and clinical evaluations of these five hearing-impaired Chinese subjects carrying one of 5 putative 12S rRNA mutation were performed. The pedigree FE239 carrying the 1027A > G mutation exhibited suggestively maternally transited hearing loss, while other four pedigrees did not have a typically maternal inheritance of hearing loss. The presence of the known 235DelC/299DelAT mutation in the *GJB2 *gene in the subject ZX039-IV-1 carrying the 1413T > C mutation indicated its role in the deafness phenotype. The absence of mutation(s) in the *GJB2 *gene in other four subjects suggested the involvement of other modifier factors in the phenotypic manifestation of these putative deafness-associated 12S rRNA variants, as in the case of these families carrying the 1555A > G mutation [[39](#B39)]. Further genetic and biochemical characterizations were necessary for the understanding the pathophysiology of these putative deafness-associated 12S rRNA mutations. Moreover, approximately 70% of subjects with aminoglycoside-indece hearing loss in this cohort did not carry the pathogenic 12S rRNA 1555A > G and 1494C > T mutations as well as putative deafness-associated 12S rRNA mutations. These data implicated the involvement of other nuclear genes, besides mitochondrial 12S rRNA mutations, in development of hearing loss in these subjects.

## Conclusions

Mutations in mitochondrial 12S rRNA gene accounted for approximately 30% cases of aminoglycoside-induced hearing loss in this cohort. These results strongly support the idea that the mitochondrial 12S rRNA is the hot spot for mutations associated with aminoglycoside ototoxicity. These data have been providing valuable information and technology to predict which individuals are at risk for ototoxicity, to improve the safety of aminoglycoside antibiotic therapy, and eventually to decrease the incidence of deafness.

## Competing interests

The authors declare that they have no competing interests.

## Authors' contributions

The work presented here was carried out in collaboration between all authors. ZS, BC, GP, YZ, CZ, JZ, TC LJ participated in the clinical data collection. JZ, TZ, SG, RL, LY performed the mitochondrial 12S rRNA sequence analysis and data collection. JL participated in the design of the study. MXG conceived of the study, participated in its design and coordination and drafted the manuscript. All authors read and approved the final manuscript.

## Contributor Information

Zhisen Shen, Email: zhisen.sun@yahoo.com.cn.

Jing Zheng, Email: jing.zheng@wzmc.net.

Bobei Chen, Email: bobei.chen@wzmc.net.

Guanghua Peng, Email: Guanghua.peng@wzmc.net.

Ting Zhang, Email: Ting.zhang@wzmc.net.

Shasha Gong, Email: Shasha.gong@wzmc.net.

Yi Zhu, Email: Yizhu@wzmc.net.

Chuqin Zhang, Email: chuqin.zhang@wzmc.net.

Ronghua Li, Email: ronghua.li@cchmc.org.

Li Yang, Email: Li.yang@cchmc.org.

Jianjin Zhou, Email: jianjin.zhou@yahoo.com.cn.

Ting Cai, Email: ting.cai@yahoo.com.cn.

Lihua Jin, Email: lihuajin@yahoo.com.cn.

Jianxin Lu, Email: jianxin.lu@wzmc.net.

Min-Xin Guan, Email: min-xin.guan@cchmc.org.

## Acknowledgements

This work was supported by Public Health Service grants RO1DC05230 and RO1DC07696 from the National Institute on Deafness and Other Communication Disorders, and grants from National Basic Research Priorities Program of China 2004CCA02200, Ministry of Public Heath of Zhejiang Province 2006A100, Ministry of Science and Technology of Zhejiang Province 2007G50G2090026 and Zhejiang Provincial Program for the Cultivation of High-level Innovative Health talents to M.X.G. and Ministry of Science and Natural Science Foundation of Zhejiang Province Y207307 to Y.Z.

## References

1. Fischel-Ghodsian N. Genetic factors in aminoglycoside toxicity. Pharmacogenomics. 2005;6:27–36. doi: 10.1517/14622416.6.1.27.  [DOI](https://doi.org/10.1517/14622416.6.1.27) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15723603/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Genetic%20factors%20in%20aminoglycoside%20toxicity&author=N%20Fischel-Ghodsian&volume=6&publication_year=2005&pages=27-36&pmid=15723603&doi=10.1517/14622416.6.1.27&)

2. Guan MX. Prevalence of mitochondrial 12S rRNA mutations associated with aminoglycoside ototoxicity. Volta Review. 2005;105:211–237. doi: 10.1016/j.mito.2010.10.006.  [DOI](https://doi.org/10.1016/j.mito.2010.10.006) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21047563/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Volta%20Review&title=Prevalence%20of%20mitochondrial%2012S%20rRNA%20mutations%20associated%20with%20aminoglycoside%20ototoxicity&author=MX%20Guan&volume=105&publication_year=2005&pages=211-237&pmid=21047563&doi=10.1016/j.mito.2010.10.006&)

3. Hutchin T, Haworth I, Higashi K, Fischel-Ghodsian N, Stoneking M, Saha N, Arnos C, Cortopassi G. A molecular basis for human hypersensitivity to aminoglycoside antibiotics. Nuclear Acids Res. 1993;21:4174–4179. doi: 10.1093/nar/21.18.4174.  [DOI](https://doi.org/10.1093/nar/21.18.4174) | [PMC free article](/articles/PMC310046/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8414970/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nuclear%20Acids%20Res&title=A%20molecular%20basis%20for%20human%20hypersensitivity%20to%20aminoglycoside%20antibiotics&author=T%20Hutchin&author=I%20Haworth&author=K%20Higashi&author=N%20Fischel-Ghodsian&author=M%20Stoneking&volume=21&publication_year=1993&pages=4174-4179&pmid=8414970&doi=10.1093/nar/21.18.4174&)

4. Prezant TR, Agapian JV, Bohlman MC, Bu X, Oztas S, Qiu WQ, Arnos KS, Cortopassi GA, Jaber L, Rotter JI, Shohat M, Fischel-Ghodsian N. Mitochondrial ribosomal RNA mutation associated with both antibiotic-induced and non-syndromic deafness. Nature Genet. 1993;4:289–294. doi: 10.1038/ng0793-289.  [DOI](https://doi.org/10.1038/ng0793-289) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7689389/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature%20Genet&title=Mitochondrial%20ribosomal%20RNA%20mutation%20associated%20with%20both%20antibiotic-induced%20and%20non-syndromic%20deafness&author=TR%20Prezant&author=JV%20Agapian&author=MC%20Bohlman&author=X%20Bu&author=S%20Oztas&volume=4&publication_year=1993&pages=289-294&pmid=7689389&doi=10.1038/ng0793-289&)

5. Estivill X, Govea N, Barcelo E, Badenas C, Romero E, Moral L, Scozzari R, D'Urbano L, Zeviani M, Torroni A. Familial progressive sensorineural deafness is mainly due to the mtDNA A1555G mutation and is enhanced by treatment with aminoglycosides. Am J Hum Genet. 1998;62:27–35. doi: 10.1086/301676.  [DOI](https://doi.org/10.1086/301676) | [PMC free article](/articles/PMC1376822/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9490575/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Familial%20progressive%20sensorineural%20deafness%20is%20mainly%20due%20to%20the%20mtDNA%20A1555G%20mutation%20and%20is%20enhanced%20by%20treatment%20with%20aminoglycosides&author=X%20Estivill&author=N%20Govea&author=E%20Barcelo&author=C%20Badenas&author=E%20Romero&volume=62&publication_year=1998&pages=27-35&pmid=9490575&doi=10.1086/301676&)

6. Matthijs G, Claes S, Longo-Bbenza B, Cassiman J-J. Non-syndromic deafness associated with a mutation and a polymorphism in the mitochondrial 12S ribosomal RNA gene in a large Zairean pedigree. Eur J Hum Genet. 1996;4:46–51. doi: 10.1159/000472169.  [DOI](https://doi.org/10.1159/000472169) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8800928/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Hum%20Genet&title=Non-syndromic%20deafness%20associated%20with%20a%20mutation%20and%20a%20polymorphism%20in%20the%20mitochondrial%2012S%20ribosomal%20RNA%20gene%20in%20a%20large%20Zairean%20pedigree&author=G%20Matthijs&author=S%20Claes&author=B%20Longo-Bbenza&author=J-J%20Cassiman&volume=4&publication_year=1996&pages=46-51&pmid=8800928&doi=10.1159/000472169&)

7. Li R, Xing G, Yan M, Cao X, Liu XZ, Bu X, Guan MX. Cosegregation of C-insertion at position 961 with A1555G mutation of mitochondrial 12S rRNA gene in a large Chinese family with maternally inherited hearing loss. Am J Med Genet. 2004;124A:113–117. doi: 10.1002/ajmg.a.20305.  [DOI](https://doi.org/10.1002/ajmg.a.20305) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14699607/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Med%20Genet&title=Cosegregation%20of%20C-insertion%20at%20position%20961%20with%20A1555G%20mutation%20of%20mitochondrial%2012S%20rRNA%20gene%20in%20a%20large%20Chinese%20family%20with%20maternally%20inherited%20hearing%20loss&author=R%20Li&author=G%20Xing&author=M%20Yan&author=X%20Cao&author=XZ%20Liu&volume=124A&publication_year=2004&pages=113-117&pmid=14699607&doi=10.1002/ajmg.a.20305&)

8. Tang X, Yang L, Zhu Y, Liao Z, Wang J, Qian Y, Tao Z, Hu L, Wu G, Lan J, Wang X, Ji J, Wu J, Ji Y, Feng J, Chen J, Li Z, Zhang X, Lu J, Guan MX. Very low penetrance of hearing loss in seven Han Chinese pedigrees carrying the deafness-associated 12S rRNA A1555G mutation. Gene. 2007;393:11–19. doi: 10.1016/j.gene.2007.01.001.  [DOI](https://doi.org/10.1016/j.gene.2007.01.001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17341440/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gene&title=Very%20low%20penetrance%20of%20hearing%20loss%20in%20seven%20Han%20Chinese%20pedigrees%20carrying%20the%20deafness-associated%2012S%20rRNA%20A1555G%20mutation&author=X%20Tang&author=L%20Yang&author=Y%20Zhu&author=Z%20Liao&author=J%20Wang&volume=393&publication_year=2007&pages=11-19&pmid=17341440&doi=10.1016/j.gene.2007.01.001&)

9. Chen B, Sun D, Yang L, Zhang C, Yang A, Zhu Y, Zhao J, Chen Y, Guan M, Wang X, Li R, Tang X, Wang J, Tao Z, Lu J, Guan MX. Mitochondrial ND5 T12338C, tRNACys T5802C, and tRNAThr G15927A variants may have a modifying role in the phenotypic manifestation of deafness-associated 12S rRNA A1555G mutation in three Han Chinese pedigrees. Am J Med Genet. 2008;146A:1248–1258. doi: 10.1002/ajmg.a.32285.  [DOI](https://doi.org/10.1002/ajmg.a.32285) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18386806/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Med%20Genet&title=Mitochondrial%20ND5%20T12338C,%20tRNACys%20T5802C,%20and%20tRNAThr%20G15927A%20variants%20may%20have%20a%20modifying%20role%20in%20the%20phenotypic%20manifestation%20of%20deafness-associated%2012S%20rRNA%20A1555G%20mutation%20in%20three%20Han%20Chinese%20pedigrees&author=B%20Chen&author=D%20Sun&author=L%20Yang&author=C%20Zhang&author=A%20Yang&volume=146A&publication_year=2008&pages=1248-1258&pmid=18386806&doi=10.1002/ajmg.a.32285&)

10. Lu J, Qian Y, Li Z, Yang A, Zhu Y, Li R, Yang L, Tang X, Chen B, Ding Y, Li Y, You J, Zheng J, Tao Z, Zhao F, Wang J, Sun D, Zhao J, Meng Y, Guan MX. Mitochondrial haplotypes may modulate the phenotypic manifestation of the deafness-associated 12S rRNA 1555A > G mutation. Mitochondrion. 2010;10:69–81. doi: 10.1016/j.mito.2009.09.007.  [DOI](https://doi.org/10.1016/j.mito.2009.09.007) | [PMC free article](/articles/PMC2787746/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19818876/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mitochondrion&title=Mitochondrial%20haplotypes%20may%20modulate%20the%20phenotypic%20manifestation%20of%20the%20deafness-associated%2012S%20rRNA%201555A%20>%20G%20mutation&author=J%20Lu&author=Y%20Qian&author=Z%20Li&author=A%20Yang&author=Y%20Zhu&volume=10&publication_year=2010&pages=69-81&pmid=19818876&doi=10.1016/j.mito.2009.09.007&)

11. Zhao H, Li R, Wang Q, Yan Q, Deng JH, Han D, Bai Y, Young WY, Guan MX. Maternally inherited aminoglycoside-induced and nonsyndromic deafness is associated with the novel C1494T mutation in the mitochondrial 12S rRNA gene in a large Chinese family. Am J Hum Genet. 2004;74:139–152. doi: 10.1086/381133.  [DOI](https://doi.org/10.1086/381133) | [PMC free article](/articles/PMC1181901/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14681830/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Maternally%20inherited%20aminoglycoside-induced%20and%20nonsyndromic%20deafness%20is%20associated%20with%20the%20novel%20C1494T%20mutation%20in%20the%20mitochondrial%2012S%20rRNA%20gene%20in%20a%20large%20Chinese%20family&author=H%20Zhao&author=R%20Li&author=Q%20Wang&author=Q%20Yan&author=JH%20Deng&volume=74&publication_year=2004&pages=139-152&pmid=14681830&doi=10.1086/381133&)

12. Rodriguez-Ballesteros M, Olarte M, Aguirre LA, Galan F, Galan R, Vallejo LA, Navas C, Villamar M, Moreno-Pelayo MA, Moreno F, del Castillo I. Molecular and clinical characterisation of three Spanish families with maternally inherited non-syndromic hearing loss caused by the 1494C- > T mutation in the mitochondrial 12S rRNA gene. J Med Genet. 2006;43:e54. doi: 10.1136/jmg.2006.042440.  [DOI](https://doi.org/10.1136/jmg.2006.042440) | [PMC free article](/articles/PMC2563187/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17085680/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Med%20Genet&title=Molecular%20and%20clinical%20characterisation%20of%20three%20Spanish%20families%20with%20maternally%20inherited%20non-syndromic%20hearing%20loss%20caused%20by%20the%201494C-%20>%20T%20mutation%20in%20the%20mitochondrial%2012S%20rRNA%20gene&author=M%20Rodriguez-Ballesteros&author=M%20Olarte&author=LA%20Aguirre&author=F%20Galan&author=R%20Galan&volume=43&publication_year=2006&pages=e54&pmid=17085680&doi=10.1136/jmg.2006.042440&)

13. Chen J, Yang L, Yang A, Zhu Y, Zhao J, Sun D, Tao Z, Tang X, Wang J, Wang X, Tsushima A, Lan J, Li W, Wu F, Yuan Q, Ji J, Feng J, Wu C, Liao Z, Li Z, Greinwald JH, Lu J, Guan MX. Maternally inherited aminoglycoside-induced and nonsyndromic hearing loss is associated with the 12S rRNA C1494T mutation in three Han Chinese pedigrees. Gene. 2007;401:4–11. doi: 10.1016/j.gene.2007.06.009.  [DOI](https://doi.org/10.1016/j.gene.2007.06.009) | [PMC free article](/articles/PMC2014725/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17698299/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gene&title=Maternally%20inherited%20aminoglycoside-induced%20and%20nonsyndromic%20hearing%20loss%20is%20associated%20with%20the%2012S%20rRNA%20C1494T%20mutation%20in%20three%20Han%20Chinese%20pedigrees&author=J%20Chen&author=L%20Yang&author=A%20Yang&author=Y%20Zhu&author=J%20Zhao&volume=401&publication_year=2007&pages=4-11&pmid=17698299&doi=10.1016/j.gene.2007.06.009&)

14. Qian Y, Guan MX. Interaction of aminoglycosides with human mitochondrial 12S ribosomal RNA carrying the deafness-associated mutation. Antimicrob Agents Chemother. 2009;53:4612–4618. doi: 10.1128/AAC.00965-08.  [DOI](https://doi.org/10.1128/AAC.00965-08) | [PMC free article](/articles/PMC2772318/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19687236/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Antimicrob%20Agents%20Chemother&title=Interaction%20of%20aminoglycosides%20with%20human%20mitochondrial%2012S%20ribosomal%20RNA%20carrying%20the%20deafness-associated%20mutation&author=Y%20Qian&author=MX%20Guan&volume=53&publication_year=2009&pages=4612-4618&pmid=19687236&doi=10.1128/AAC.00965-08&)

15. Guan MX, Fischel-Ghodsian N, Attardi G. Biochemical evidence for nuclear gene involvement in phenotype of nonsyndromic deafness associated with mitochondrial 12S rRNA mutation. Hum Mol Genet. 1996;5:963–971. doi: 10.1093/hmg/5.7.963.  [DOI](https://doi.org/10.1093/hmg/5.7.963) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8817331/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=Biochemical%20evidence%20for%20nuclear%20gene%20involvement%20in%20phenotype%20of%20nonsyndromic%20deafness%20associated%20with%20mitochondrial%2012S%20rRNA%20mutation&author=MX%20Guan&author=N%20Fischel-Ghodsian&author=G%20Attardi&volume=5&publication_year=1996&pages=963-971&pmid=8817331&doi=10.1093/hmg/5.7.963&)

16. Guan MX, Fischel-Ghodsian N, Attardi G. A biochemical basis for the inherited susceptibility to aminoglycoside ototoxicity. Hum Mol Genet. 2000;9:1787–1793. doi: 10.1093/hmg/9.12.1787.  [DOI](https://doi.org/10.1093/hmg/9.12.1787) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10915767/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=A%20biochemical%20basis%20for%20the%20inherited%20susceptibility%20to%20aminoglycoside%20ototoxicity&author=MX%20Guan&author=N%20Fischel-Ghodsian&author=G%20Attardi&volume=9&publication_year=2000&pages=1787-1793&pmid=10915767&doi=10.1093/hmg/9.12.1787&)

17. Guan MX, Yan Q, Li X, Bykhovskaya Y, Gallo-Teran J, Hajek P, Umeda N, Zhao H, Garrido G, Mengesha E, Suzuki T, del Castillo I, Peters JL, Li R, Qian Y, Wang X, Ballana E, Shohat M, Lu J, Estivill X, Watanabe K, Fischel-Ghodsian N. Mutation in TRMU related to transfer RNA modification modulates the phenotypic expression of the deafness-associated mitochondrial 12S ribosomal RNA mutations. Am J Hum Genet. 2006;79:291–302. doi: 10.1086/506389.  [DOI](https://doi.org/10.1086/506389) | [PMC free article](/articles/PMC1559489/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16826519/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Mutation%20in%20TRMU%20related%20to%20transfer%20RNA%20modification%20modulates%20the%20phenotypic%20expression%20of%20the%20deafness-associated%20mitochondrial%2012S%20ribosomal%20RNA%20mutations&author=MX%20Guan&author=Q%20Yan&author=X%20Li&author=Y%20Bykhovskaya&author=J%20Gallo-Teran&volume=79&publication_year=2006&pages=291-302&pmid=16826519&doi=10.1086/506389&)

18. Zhao H, Young WY, Yan Q, Li R, Cao J, Wang Q, Li X, Peters JL, Han D, Guan MX. Functional characterization of the mitochondrial 12S rRNA C1494T mutation associated with aminoglycoside-induced and nonsyndromic hearing loss. Nucleic Acid Res. 2005;33:1132–1139. doi: 10.1093/nar/gki262.  [DOI](https://doi.org/10.1093/nar/gki262) | [PMC free article](/articles/PMC549421/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15722487/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acid%20Res&title=Functional%20characterization%20of%20the%20mitochondrial%2012S%20rRNA%20C1494T%20mutation%20associated%20with%20aminoglycoside-induced%20and%20nonsyndromic%20hearing%20loss&author=H%20Zhao&author=WY%20Young&author=Q%20Yan&author=R%20Li&author=J%20Cao&volume=33&publication_year=2005&pages=1132-1139&pmid=15722487&doi=10.1093/nar/gki262&)

19. Usami S, Abe S, Akita J, Namba A, Shinkawa H, Ishii M, Iwasaki S, Hoshino T, Ito J, Doi K, Kubo T, Nakagawa T, Komiyama S, Tono T, Komune S. Prevalence of mitochondrial gene mutations among hearing impaired patients. J Med Genet. 2000;37:38–40. doi: 10.1136/jmg.37.1.38.  [DOI](https://doi.org/10.1136/jmg.37.1.38) | [PMC free article](/articles/PMC1734443/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10633132/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Med%20Genet&title=Prevalence%20of%20mitochondrial%20gene%20mutations%20among%20hearing%20impaired%20patients&author=S%20Usami&author=S%20Abe&author=J%20Akita&author=A%20Namba&author=H%20Shinkawa&volume=37&publication_year=2000&pages=38-40&pmid=10633132&doi=10.1136/jmg.37.1.38&)

20. Li R, Greinwald JH, Yang L, Choo DI, Wenstrup RJ, Guan MX. Molecular analysis of mitochondrial 12S rRNA and tRNASer(UCN) genes in paediatric subjects with nonsyndromic hearing loss. J Med Genet. 2004;41:615–620. doi: 10.1136/jmg.2004.020230.  [DOI](https://doi.org/10.1136/jmg.2004.020230) | [PMC free article](/articles/PMC1735864/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15286157/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Med%20Genet&title=Molecular%20analysis%20of%20mitochondrial%2012S%20rRNA%20and%20tRNASer(UCN)%20genes%20in%20paediatric%20subjects%20with%20nonsyndromic%20hearing%20loss&author=R%20Li&author=JH%20Greinwald&author=L%20Yang&author=DI%20Choo&author=RJ%20Wenstrup&volume=41&publication_year=2004&pages=615-620&pmid=15286157&doi=10.1136/jmg.2004.020230&)

21. Li Z, Li R, Chen J, Liao Z, Zhu Y, Qian Y, Xiong S, Heman-Ackah S, Wu J, Choo DI, Guan MX. Mutational analysis of the mitochondrial 12S rRNA gene in Chinese pediatric subjects with aminoglycoside induced and non-syndromic hearing loss. Hum Genet. 2005;117:9–15. doi: 10.1007/s00439-005-1276-1.  [DOI](https://doi.org/10.1007/s00439-005-1276-1) | [PMC free article](/articles/PMC1484504/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15841390/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Genet&title=Mutational%20analysis%20of%20the%20mitochondrial%2012S%20rRNA%20gene%20in%20Chinese%20pediatric%20subjects%20with%20aminoglycoside%20induced%20and%20non-syndromic%20hearing%20loss&author=Z%20Li&author=R%20Li&author=J%20Chen&author=Z%20Liao&author=Y%20Zhu&volume=117&publication_year=2005&pages=9-15&pmid=15841390&doi=10.1007/s00439-005-1276-1&)

22. Lu J, Li Z, Zhu Y, Yang A, Li R, Zheng J, Cai Q, Peng G, Zheng W, Tang X, Chen B, Chen J, Liao Z, Yang L, Li Y, You J, Ding Y, Yu H, Wang J, Sun D, Zhao J, Xue L, Wang J, Guan MX. Mitochondrial 12S rRNA variants in 1642 Han Chinese pediatric subjects with aminoglycoside-induced and nonsyndromic hearing loss. Mitochondrion. 2010;10:380–390. doi: 10.1016/j.mito.2010.01.007.  [DOI](https://doi.org/10.1016/j.mito.2010.01.007) | [PMC free article](/articles/PMC2874659/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20100600/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mitochondrion&title=Mitochondrial%2012S%20rRNA%20variants%20in%201642%20Han%20Chinese%20pediatric%20subjects%20with%20aminoglycoside-induced%20and%20nonsyndromic%20hearing%20loss&author=J%20Lu&author=Z%20Li&author=Y%20Zhu&author=A%20Yang&author=R%20Li&volume=10&publication_year=2010&pages=380-390&pmid=20100600&doi=10.1016/j.mito.2010.01.007&)

23. Malik SG, Pieter N, Sudoyo H, Kadir A, Marzuki S. Prevalence of the mitochondrial DNA A1555G mutation in sensorineural deafness patients in island Southeast Asia. J Hum Genet. 2003;48:480–483. doi: 10.1007/s10038-003-0056-9.  [DOI](https://doi.org/10.1007/s10038-003-0056-9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12955586/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Hum%20Genet&title=Prevalence%20of%20the%20mitochondrial%20DNA%20A1555G%20mutation%20in%20sensorineural%20deafness%20patients%20in%20island%20Southeast%20Asia&author=SG%20Malik&author=N%20Pieter&author=H%20Sudoyo&author=A%20Kadir&author=S%20Marzuki&volume=48&publication_year=2003&pages=480-483&pmid=12955586&doi=10.1007/s10038-003-0056-9&)

24. Jacobs HT, Hutchin TP, Käppi T, Gillies G, Minkkinen K, Walker J, Thompson K, Rovio AT, Carella M, Melchionda S, Zelante L, Gasparini P, Pyykkö I, Shah ZH, Zeviani M, Mueller RF. Mitochondrial DNA mutations in patients with postlingual, nonsyndromic hearing impairment. Eur J Hum Genet. 2005;13:26–33. doi: 10.1038/sj.ejhg.5201250.  [DOI](https://doi.org/10.1038/sj.ejhg.5201250) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15292920/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Hum%20Genet&title=Mitochondrial%20DNA%20mutations%20in%20patients%20with%20postlingual,%20nonsyndromic%20hearing%20impairment&author=HT%20Jacobs&author=TP%20Hutchin&author=T%20K%C3%A4ppi&author=G%20Gillies&author=K%20Minkkinen&volume=13&publication_year=2005&pages=26-33&pmid=15292920&doi=10.1038/sj.ejhg.5201250&)

25. Rieder MJ, Taylor SL, Tobe VO, Nickerson DA. Automating the identification of DNA variations using quality-based fluorescence re-sequencing: analysis of the human mitochondrial genome. Nucleic Acids Res. 1998;26:967–973. doi: 10.1093/nar/26.4.967.  [DOI](https://doi.org/10.1093/nar/26.4.967) | [PMC free article](/articles/PMC147367/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9461455/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=Automating%20the%20identification%20of%20DNA%20variations%20using%20quality-based%20fluorescence%20re-sequencing:%20analysis%20of%20the%20human%20mitochondrial%20genome&author=MJ%20Rieder&author=SL%20Taylor&author=VO%20Tobe&author=DA%20Nickerson&volume=26&publication_year=1998&pages=967-973&pmid=9461455&doi=10.1093/nar/26.4.967&)

26. Andrews RM, Kubacka I, Chinnery PF, Lightowlers RN, Turnbull DM, Howell N. Reanalysis and revision of the Cambridge reference sequence for human mitochondrial DNA. Nat Genet. 1999;23:147. doi: 10.1038/13779.  [DOI](https://doi.org/10.1038/13779) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10508508/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Reanalysis%20and%20revision%20of%20the%20Cambridge%20reference%20sequence%20for%20human%20mitochondrial%20DNA&author=RM%20Andrews&author=I%20Kubacka&author=PF%20Chinnery&author=RN%20Lightowlers&author=DM%20Turnbull&volume=23&publication_year=1999&pages=147&pmid=10508508&doi=10.1038/13779&)

27. Neefs JM, Van de Peer Y, De Rijik P, Goris A, De Wachter R. Compilation of small ribosomal subunit RNA sequences. Nucleic Acids Res. 1991;19(Suppl):1987–2018. doi: 10.1093/nar/19.suppl.1987.  [DOI](https://doi.org/10.1093/nar/19.suppl.1987) | [PMC free article](/articles/PMC331343/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2041797/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=Compilation%20of%20small%20ribosomal%20subunit%20RNA%20sequences&author=JM%20Neefs&author=Y%20Van%20de%20Peer&author=P%20De%20Rijik&author=A%20Goris&author=R%20De%20Wachter&volume=19&issue=Suppl&publication_year=1991&pages=1987-2018&pmid=2041797&doi=10.1093/nar/19.suppl.1987&)

28. Springer MS, Douzery E. Secondary structure and patterns of evolution among mammalian mitochondrial 12S rRNA molecules. J Mol Evol. 1996;143:357–373. doi: 10.1007/BF02339010.  [DOI](https://doi.org/10.1007/BF02339010) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8798341/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Mol%20Evol&title=Secondary%20structure%20and%20patterns%20of%20evolution%20among%20mammalian%20mitochondrial%2012S%20rRNA%20molecules&author=MS%20Springer&author=E%20Douzery&volume=143&publication_year=1996&pages=357-373&pmid=8798341&doi=10.1007/BF02339010&)

29. De Rijk P, De Wachter R. RnaViz, a program for the visualisation of RNA secondary structure. Nucleic Acids Res. 1997;25:4679–4684. doi: 10.1093/nar/25.22.4679.  [DOI](https://doi.org/10.1093/nar/25.22.4679) | [PMC free article](/articles/PMC147081/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9358182/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=RnaViz,%20a%20program%20for%20the%20visualisation%20of%20RNA%20secondary%20structure&author=P%20De%20Rijk&author=R%20De%20Wachter&volume=25&publication_year=1997&pages=4679-4684&pmid=9358182&doi=10.1093/nar/25.22.4679&)

30. Tessa A, Giannotti A, Tieri L, Vilarinho L, Marotta G, Santorelli FM. Maternally inherited deafness associated with a T1095C mutation in the mDNA. Eur J Hum Genet. 2002;9:147–149. doi: 10.1038/sj.ejhg.5200601.  [DOI](https://doi.org/10.1038/sj.ejhg.5200601) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11313749/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Hum%20Genet&title=Maternally%20inherited%20deafness%20associated%20with%20a%20T1095C%20mutation%20in%20the%20mDNA&author=A%20Tessa&author=A%20Giannotti&author=L%20Tieri&author=L%20Vilarinho&author=G%20Marotta&volume=9&publication_year=2002&pages=147-149&pmid=11313749&doi=10.1038/sj.ejhg.5200601&)

31. Thyagarajan D, Bressman S, Bruno C, Przedborski S, Shanske S, Lynch T, Fahn S, DiMauro S. A novel mitochondrial 12SrRNA point mutation in parkinsonism, deafness, and neuropathy. Ann Neurol. 2002;48:730–736. doi: 10.1002/1531-8249(200011)48:5<730::AID-ANA6>3.0.CO;2-0.  [DOI](https://doi.org/10.1002/1531-8249(200011)48:5&#x0003c;730::AID-ANA6&#x0003e;3.0.CO;2-0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11079536/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Neurol&title=A%20novel%20mitochondrial%2012SrRNA%20point%20mutation%20in%20parkinsonism,%20deafness,%20and%20neuropathy&author=D%20Thyagarajan&author=S%20Bressman&author=C%20Bruno&author=S%20Przedborski&author=S%20Shanske&volume=48&publication_year=2002&pages=730-736&pmid=11079536&doi=10.1002/1531-8249(200011)48:5&#x0003c;730::AID-ANA6&#x0003e;3.0.CO;2-0&)

32. Bacino C, Prezant TR, Bu X, Fournier P, Fischel-Ghodsian N. Susceptibility mutations in the mitochondrial small ribosomal RNA gene in aminoglycoside induced deafness. Pharmacogenetics. 1995;5:165–172. doi: 10.1097/00008571-199506000-00005.  [DOI](https://doi.org/10.1097/00008571-199506000-00005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7550368/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Susceptibility%20mutations%20in%20the%20mitochondrial%20small%20ribosomal%20RNA%20gene%20in%20aminoglycoside%20induced%20deafness&author=C%20Bacino&author=TR%20Prezant&author=X%20Bu&author=P%20Fournier&author=N%20Fischel-Ghodsian&volume=5&publication_year=1995&pages=165-172&pmid=7550368&doi=10.1097/00008571-199506000-00005&)

33. Casano RA, Johnson DF, Bykhovskaya Y, Torricelli F, Bigozzi M, Fischel-Ghodsian N. Inherited susceptibility to aminoglycoside ototoxicity: genetic heterogeneity and clinical implications. Am J Otolaryngol. 1999;20:151–156. doi: 10.1016/S0196-0709(99)90062-5.  [DOI](https://doi.org/10.1016/S0196-0709(99)90062-5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10326749/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Otolaryngol&title=Inherited%20susceptibility%20to%20aminoglycoside%20ototoxicity:%20genetic%20heterogeneity%20and%20clinical%20implications&author=RA%20Casano&author=DF%20Johnson&author=Y%20Bykhovskaya&author=F%20Torricelli&author=M%20Bigozzi&volume=20&publication_year=1999&pages=151-156&pmid=10326749&doi=10.1016/S0196-0709(99)90062-5&)

34. Brandon MC, Lott MT, Nguyen KC, Spolim S, Navathe SB, Baldi P, Wallace DC. MITOMAP: a human mitochondrial genome database--2004 update. Nucleic Acids Res. 2005;33:D611–613. doi: 10.1093/nar/gki079.  [DOI](https://doi.org/10.1093/nar/gki079) | [PMC free article](/articles/PMC540033/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15608272/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=MITOMAP:%20a%20human%20mitochondrial%20genome%20database--2004%20update&author=MC%20Brandon&author=MT%20Lott&author=KC%20Nguyen&author=S%20Spolim&author=SB%20Navathe&volume=33&publication_year=2005&pages=D611-613&pmid=15608272&doi=10.1093/nar/gki079&)

35. Ruiz-Pesini E, Wallace DC. Evidence for adaptive selection acting on the tRNA and rRNA genes of human mitochondrial DNA. Hum Mutat. 2006;27:1072–1081. doi: 10.1002/humu.20378.  [DOI](https://doi.org/10.1002/humu.20378) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16947981/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mutat&title=Evidence%20for%20adaptive%20selection%20acting%20on%20the%20tRNA%20and%20rRNA%20genes%20of%20human%20mitochondrial%20DNA&author=E%20Ruiz-Pesini&author=DC%20Wallace&volume=27&publication_year=2006&pages=1072-1081&pmid=16947981&doi=10.1002/humu.20378&)

36. Yuan Y, You Y, Huang D, Cui J, Wang Y, Wang Q, Yu F, Kang D, Yuan H, Han D, Dai P. Comprehensive molecular etiology analysis of nonsyndromic hearing impairment from typical areas in China. J Transl Med. 2009;7:79. doi: 10.1186/1479-5876-7-79.  [DOI](https://doi.org/10.1186/1479-5876-7-79) | [PMC free article](/articles/PMC2754984/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19744334/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Transl%20Med&title=Comprehensive%20molecular%20etiology%20analysis%20of%20nonsyndromic%20hearing%20impairment%20from%20typical%20areas%20in%20China&author=Y%20Yuan&author=Y%20You&author=D%20Huang&author=J%20Cui&author=Y%20Wang&volume=7&publication_year=2009&pages=79&pmid=19744334&doi=10.1186/1479-5876-7-79&)

37. Liu XZ, Xia XJ, Ke XM, Ouyang XM, Du LL, Liu YH, Angeli S, Telischi FF, Nance WE, Balkany T, Xu LR. The prevalence of connexin 26 (GJB2) mutations in the Chinese population. Hum Genet. 2002;111:394–7. doi: 10.1007/s00439-002-0811-6.  [DOI](https://doi.org/10.1007/s00439-002-0811-6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12384781/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Genet&title=The%20prevalence%20of%20connexin%2026%20(GJB2)%20mutations%20in%20the%20Chinese%20population&author=XZ%20Liu&author=XJ%20Xia&author=XM%20Ke&author=XM%20Ouyang&author=LL%20Du&volume=111&publication_year=2002&pages=394-7&pmid=12384781&doi=10.1007/s00439-002-0811-6&)

38. Kong QP, Bandelt HJ, Sun C, Yao YG, Salas A, Achilli A, Wang CY, Zhong L, Zhu CL, Wu SF, Torroni A, Zhang YP. Updating the East Asian mtDNA phylogeny: a prerequisite for the identification of pathogenic mutations. Hum Mol Genet. 2006;15:2076–2086. doi: 10.1093/hmg/ddl130.  [DOI](https://doi.org/10.1093/hmg/ddl130) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16714301/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=Updating%20the%20East%20Asian%20mtDNA%20phylogeny:%20a%20prerequisite%20for%20the%20identification%20of%20pathogenic%20mutations&author=QP%20Kong&author=HJ%20Bandelt&author=C%20Sun&author=YG%20Yao&author=A%20Salas&volume=15&publication_year=2006&pages=2076-2086&pmid=16714301&doi=10.1093/hmg/ddl130&)

39. Guan MX. Mitochondrial 12S rRNA mutations associated with aminoglycoside ototoxicity. Mitochondrion. 2010.  [DOI](https://doi.org/10.1016/j.mito.2010.10.006) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21047563/)
