# Genome-wide mapping for clinically relevant predictors of lamotrigine- and phenytoin-induced hypersensitivity reactions

## Metadata
**Authors:** Mark McCormack, Thomas J Urban, Kevin V Shianna, Nicole Walley, Massimo Pandolfo, Chantal Depondt, Elijah Chaila, Gerard D O’Conner, Dalia Kasperavičiūtė, Rodney A Radtke, Erin L Heinzen, Sanjay M Sisodiya, Norman Delanty, Gianpiero L Cavalleri
**Journal:** Pharmacogenomics
**Date:** 2012 Mar
**DOI:** [10.2217/pgs.11.165](https://doi.org/10.2217/pgs.11.165)
**PMID:** 22379998
**PMCID:** PMC3428903
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3428903/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC3428903/pdf/ukmss-49236.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC3428903/pdf/ukmss-49236.pdf)

## Abstract

**Aims:** 
An association between carbamazepine-induced hypersensitivity and HLA-A*3101 has been reported in populations of both European and Asian descent. We aimed to investigate HLA-A*3101 and other common variants across the genome as markers for cutaneous adverse drug reactions (cADRs) attributed to lamotrigine and phenytoin.

**Materials & methods:** 
We recruited patients with lamotrigine-induced cADRs (n = 46) and patients with phenytoin-cADRs (n = 44) and the 1958 British birth cohort was used as a control (n = 1296). HLA-A*3101 was imputed from genome-wide association study data. We applied genome-wide association to study lamotrigine- and phenytoin-induced cADR, and total cADR cases combined.

**Results:** 
Neither HLA-A*3101 nor any other genetic marker significantly predicted lamotrigine- or phenytoin-induced cADRs.

**Conclusion:** 
HLA-A*3101 does not appear to be a predictor for lamotrigine- and phenytoin-induced cADRs in Europeans. Our genome-wide association study results do not support the existence of a clinically relevant common variant for the development of lamotrigine- or phenytoin-induced cADRs. As a predictive marker, HLA-A*3101 appears to be specific for carbamazepine-induced cADRs.

Keywords: epilepsy, GWAS, HLA-A*3101, hypersensitivity, lamotrigine, phenytoin

### Aims

An association between carbamazepine-induced hypersensitivity and *HLA-A*3101* has been reported in populations of both European and Asian descent. We aimed to investigate *HLA-A*3101* and other common variants across the genome as markers for cutaneous adverse drug reactions (cADRs) attributed to lamotrigine and phenytoin.

### Materials & methods

We recruited patients with lamotrigine-induced cADRs (n = 46) and patients with phenytoin-cADRs (n = 44) and the 1958 British birth cohort was used as a control (n = 1296). *HLA-A*3101* was imputed from genome-wide association study data. We applied genome-wide association to study lamotrigine- and phenytoin-induced cADR, and total cADR cases combined.

### Results

Neither *HLA-A*3101* nor any other genetic marker significantly predicted lamotrigine- or phenytoin-induced cADRs.

### Conclusion

*HLA-A*3101* does not appear to be a predictor for lamotrigine- and phenytoin-induced cADRs in Europeans. Our genome-wide association study results do not support the existence of a clinically relevant common variant for the development of lamotrigine- or phenytoin-induced cADRs. As a predictive marker, *HLA-A*3101* appears to be specific for carbamazepine-induced cADRs.

## Materials & methods

### Case subjects

We obtained approval for this study from the local research ethics committees of each site. Written, informed consent was obtained from all subjects prior to participation. Patients were recruited through centers affiliated with the EPIGEN Consortium; namely Duke University (NC, USA), University College London (London, UK), Université Libre de Bruxelles (Brussels, Belgium) and the Royal College of Surgeons in Ireland with Beaumont Hospital (Dublin, Ireland).

All patients were of European ancestry as determined by self-report and genetic-marker analysis. Cases were separated according to the causal drug and then phenotyped into distinct hypersensitivity states of MPE, HSS and SJS based on the extent and severity of their ADR and using disease criteria, as previously reported [[14](#R14)]. In total, there were 46 ADR cases relating to LTG, of which 42 were MPE cases, three were HSS and one was SJS. Of the 44 PHT-induced ADR cases, 40 were MPE cases and four were HSS cases. There were four patients who experienced MPE with both drugs on separate occasions and were thus included in each set of analyses. Given the low numbers of HSS and SJS patients and the evidence that a common marker may exist for all phenotypes (as evidence would suggest for *HLA-A*3101* and CBZ-induced cADRs) we analyzed all cases together as drug-specific cADR groups.

### Control subjects

The control group consisted of a homogeneous subgroup of 1296 subjects from the 1958 British birth cohort that were selected by principal component analysis of GWAS. Controls were unscreened for AED-related ADRs.

### Genome-wide analysis

Genome-wide data across >600,000 SNPs were available for 34 LTG and 42 PHT cases and all 1296 population control subjects. Genotyping for all cases was performed with the Illumina® Human610–610K Quad platform at the Duke University Center for Human Genome Variation (NC, USA). We removed subjects with a genotyping failure rate below 95%, SNPs with low minor allele frequency (<2%), those failing the Hardy–Weinberg equilibrium and any that had missing genotype information in >10% of all subjects.

### Imputation of HLA alleles

We imputed classic *HLA* alleles for all cases and controls using MACH 1.0 [[17](#R17)]. This method has been validated as an accurate method for imputing HLA subtypes in our cohort as described previously [[14](#R14)]. The HLA-typed 1958 British birth cohort constituted the reference population for imputation. For those samples for which GWAS data were not available we used PCR with sequence-specific primers to determine the presence of *HLA-A*3101*. The PCR conditions were as follows: the forward primer sequence was 5′-GATAGAGCAGGAGAGGCCT-3′, reverse primer sequence was 5′-AGCGCAGGTCCTCGTTCAA-3′ and the cycling parameters were 95°C for 3 min, five cycles of 95°C for 15 s, 70°C for 15 s and 72°C for 30 s, two cycles of 95°C for 15 s, 65°C for 15 s and 72°C for 30 s, four cycles of 95°C for 15 s, 55°C for 1 min and 72°C for 2 min, finally 72°C for 7 min.

### Statistical analysis

Statistical analyses were performed using PLINK (version 1.05) [[18](#R18)] and EIGENSTRAT [[19](#R19)] software packages. For the GWAS, we applied logistic regression (using an additive model) to each of the cADR case groups in turn versus the 1958 British birth cohort controls. We adjusted for population stratification by including significant principal components (generated using EIGENSTRAT) as covariates in the logistic-regression model and used a p-value of less than 5.0 × 10^−8^ as a threshold for genome-wide significance. [Supplementary Figure S1](#SD1) (see online, [www.futuremedicine.com/doi/suppl/10.2217/PGS.11.165](http://www.futuremedicine.com/doi/suppl/10.2217/PGS.11.165)) for a plot of the first and second principal components between cases and controls. Post-GWAS annotation analysis was performed with WGAViewer [[20](#R20)]. We calculated study power using the program Power for Genetic Association Analyses [[21](#R21)]. We estimated the relative risk (RR) required for 80% power to detect an allelic association (under an additive disease model) with the following parameters: a prevalence of 10% of the ADR in the population, a minor allele frequency of 5%, an experiment-wide α level of 0.05 and the assumption that a genetic marker is causal or in complete linkage with the causal variant.

## Results

### HLA-A*3101 as a predictor of LTG- and PHT-induced cADRs

[Table 1](#T1) illustrates the results of *HLA-A*3101* genotyping in LTG- and PHT-induced cADR cases and population controls. The allele does not appear to be enriched in any of the cADR patient groups. For each of the analyses considered, our power calculations indicate that we had reasonable power to detect effect sizes of clinical relevance (RR >2).

### Table 1. Distribution of HLA-A*3101 among lamotrigine and phenytoin case subjects compared with population control subjects.

| HLA-A*3101 | Lamotrigine–cADR cases | Phenytoin–cADR cases | Controls |
| --- | --- | --- | --- |
| Carriers (n; %) | 2 (4.35%) | 2 (4.55%) | 72 (5.56%) |
| Noncarriers (n; %) | 44 (95.65%) | 42 (95.45%) | 1224 (94.44%) |
| Total subjects (n) | 46 | 44 | 1296 |
| p-value | 0.698 | 0.746 |   |
| Odds ratio (95% CI) | 0.756 (0.183–3.129) | 0.791 (0.191–3.277) |   |
| Relative risk for 80%power | 2.204 | 2.229 |   |

Table 1 Caption: HLA-A*3101 was found at a low average frequency of 4.45% across all cADR cases and at 5.56% in our population controls, which is comparable with reported European population datasets [23]. cADR: Cutaneous adverse drug reaction.

### GWAS results

We next conducted a GWAS for each of the drug-specific cADR groups as well as a total combined case cohort. The results of the logistic regression analyses are depicted as Manhattan and quantile–quantile plots in [Figures 1](#F1) & [2](#F2). The ten most significant markers from the logistic regression analyses are listed in [Supplementary Tables S1-S3](#SD2). We did not detect any variants satisfying our threshold for genome-wide significance (p-value of 5.0 × 10^−8^), nor did we observe any significant deviations from the expected distribution in quantile–quantile plots (results shown in [Figures 1](#F1) & [2](#F2)). From power calculations ([Figure 3](#F3)) we estimated that we had 80% power to detect a variant with a minor allele frequency >10% and a RR >2.5 when considering all cases together. We feel this RR is within the realm of clinical relevance.

### Figure 1. Manhattan plots of association results.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1140/3428903/4ed59c7de3b5/ukmss-49236-f0001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3428903_ukmss-49236-f0001.jpg)

Manhattan plots of genome-wide association studies analysis of (A) lamotrigine, (B) phenytoin and (C) combined drug-induced cutaneous adverse drug reaction vs controls. Genome-wide significance was taken as 5.0 × 10−8. Figure generated in WGAViewer.

### Figure 2. Quantile–quantile plot of the association results.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1140/3428903/9ecc62eca396/ukmss-49236-f0002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3428903_ukmss-49236-f0002.jpg)

Quantile–quantile plots of p-values from genome-wide association studies of (A) lamotrigine, (B) phenytoin and (C) total drug-induced cutaneous adverse drug reactions vs controls calculated using logistic regression and including significant EIGENSTRAT axes as covariates. The expected chi distribution is represented along the x-axis, while the observed chi distribution is represented along the y-axis. Figure generated in WGAViewer.

### Figure 3. Power calculation for genome-wide association results.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1140/3428903/0bd06658fae5/ukmss-49236-f0003.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3428903_ukmss-49236-f0003.jpg)

The three analyses are represented with lamotrigine, phenytoin and combined drug cutaneous ADR groups with 1296 controls. Taking all cases together, we estimated we had 80% power to detect a variant with a minor allele frequency >10% and a relative risk >2.5. ADR: Adverse drug reaction.

## Discussion

The results presented here suggest that *HLA-A*3101* is not a clinically relevant predictor of cADRs caused by LTG and PHT in populations of European descent.

It has been shown in previous studies by ourselves and others that *HLA-A*3101* is a strong predictor for the clinical spectrum of CBZ-induced hypersensitivity and we reported an odds ratio of 8.33 for CBZ-induced MPE for a population of European descent, while the corresponding value for a population of Japanese descent was 10.8 [[15](#R15)].

We calculated that the current study had 80% power to detect a RR of 3.27 for cADRs based on a frequency of 5% for *HLA-A*3101*, which gives this study ample power to detect an effect size similar to that reported previously for CBZ-induced MPE. If *HLA-A*3101* is a predictor for either LTG- or PHT-induced cADRs, we were well powered to detect it. Of the 44 cases of PHT-induced cADRs, only two were positive for *HLA-A*3101*. Similarly, only two of the 46 cases of LTG-induced cADRs were positive for the allele. Incidentally, one of the subjects positive for *HLA-A*3101* in the LTG case group had a historical report of a rash with CBZ.

Although we included HSS and SJS, the overwhelming majority of cases were of the MPE type. Given the small number of HSS and SJS cases, we pooled all cases into a single group of hypersensitivity ADR cases per drug for a better powered genome-wide analysis. We chose to analyze all cases together as cADR groups as we did not have enough subjects to run a powered GWAS for HSS or SJS alone. The GWAS results remain negative when the HSS and SJS cases are removed. Further studies of PHT- and LTG-induced HSS and SJS in larger patient cohorts is warranted, especially given that the genetic predictors (e.g., *HLA-B*1502*) can be selective for a given hypersensitivity reaction [[5](#R5),[13](#R13),[22](#R22)], and the effect size correlates with the severity of the phenotype across the hypersensitivity spectrum (as appears to be the case for *HLA-A*3101* and CBZ [[14](#R14),[16](#R16)]).

Our GWAS results are not suggestive of a common, clinically relevant predictor of LTG- or PHT-induced cADRs in a European population. None of the variants tested satisfied the GWAS significance threshold, nor was there any clear biological candidate from within the pool of the most significant SNPs ([Supplementary Tables S1-S3](#SD2)). Our study was powered to detect a variant of minor allele frequency >10% carrying a clinically relevant genetic effect (RR > 2.5). We recognize that sample size is a major limitation of our study. This has limited our analysis to effect sizes in the realm of clinical relevance (for cADRs) and has prevented us from studying SJS/TEN specifically. We also cannot rule out genetic variants conveying smaller effects (RR <2.5) or rare variants of small or large effect, which were not captured by our GWAS chip.

With the increased availability of whole-exome and whole-genome sequencing, it should become possible and more affordable to identify rare variants not captured through GWAS. Discovering these genetic markers would further the understanding of the functional role of *HLA* in drug response, as well as helping clinicians screen patients susceptible to these ADRs. This study mainly focused on patients with the less severe MPE hypersensitivity reaction; however, there is a clear motivation to investigate whether *HLA-A*3101* extends itself as a genetic marker for HSS and SJS/TEN caused by LTG and PHT.

## Conclusion

*HLA-A*3101* does not appear to be a genetic marker for LTG- and PHT-induced MPE reactions in Europeans. The predictive application of *HLA-A*3101* as a clinically relevant genetic marker for cADRs in Europeans appears to be specific to CBZ.

## Supplementary Material

**Background**

**Results**

**Conclusion**

**Background**

**Results**

**Conclusion**

## Acknowledgments

**Financial & competing interests disclosure**

This work was supported by grants from the Medical Research Council (G0400126), the Wellcome Trust (084730), UCLH CRDC (F136), the National Institute for Health Research (08–08-SCC), Brainwave – the Irish Epilepsy Association/the Medical Research Charities Group of Ireland/Health Research Board award (2009/001), the National Society for Epilepsy, the Fonds National de la Recherche Scientifique and the Fonds Erasme pour la Recherche Médicale, Université Libre de Bruxelles (Belgium) and supported, in part, by Award Number RC2NS070344 from the National Institute of Neurological Disorders and Stroke. M McCormack was supported by a Health Research Board of Ireland’s Translational Research Scholars award. The authors acknowledge use of genotype data from the 1958 British birth cohort DNA collection, funded by the Medical Research Council grant G0000934 and the Wellcome Trust grant 068545/Z/02. The collection of the Belgian patients was supported by the Fonds National de la Recherche Scientifique grant number FC 63574/3.4.620.06 F, and the Fonds Erasme pour la Recherche Médicale, Université Libre de Bruxelles (Belgium).

No writing assistance was utilized in the production of this manuscript.

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Arif H, Buchsbaum R, Weintraub D, et al. Comparison and predictors of rash associated with 15 antiepileptic drugs. Neurology. 2007;68(20):1701–1709. doi: 10.1212/01.wnl.0000261917.83337.db.  [DOI](https://doi.org/10.1212/01.wnl.0000261917.83337.db) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17502552/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neurology&title=Comparison%20and%20predictors%20of%20rash%20associated%20with%2015%20antiepileptic%20drugs&author=H%20Arif&author=R%20Buchsbaum&author=D%20Weintraub&volume=68&issue=20&publication_year=2007&pages=1701-1709&pmid=17502552&doi=10.1212/01.wnl.0000261917.83337.db&)

2. Chung WH, Hung SI, Hong HS, et al. Medical genetics: a marker for Stevens-Johnson syndrome. Nature. 2004;428(6982):486. doi: 10.1038/428486a.  [DOI](https://doi.org/10.1038/428486a) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15057820/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Medical%20genetics:%20a%20marker%20for%20Stevens-Johnson%20syndrome&author=WH%20Chung&author=SI%20Hung&author=HS%20Hong&volume=428&issue=6982&publication_year=2004&pages=486&pmid=15057820&doi=10.1038/428486a&)

3. Hung SI, Chung WH, Liu ZS, et al. Common risk allele in aromatic antiepileptic-drug induced Stevens-Johnson syndrome and toxic epidermal necrolysis in Han Chinese. Pharmacogenomics. 2010;11(3):349–356. doi: 10.2217/pgs.09.162.  [DOI](https://doi.org/10.2217/pgs.09.162) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20235791/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Common%20risk%20allele%20in%20aromatic%20antiepileptic-drug%20induced%20Stevens-Johnson%20syndrome%20and%20toxic%20epidermal%20necrolysis%20in%20Han%20Chinese&author=SI%20Hung&author=WH%20Chung&author=ZS%20Liu&volume=11&issue=3&publication_year=2010&pages=349-356&pmid=20235791&doi=10.2217/pgs.09.162&)

4. Ding WY, Lee CK, Choon SE. Cutaneous adverse drug reactions seen in a tertiary hospital in Johor, Malaysia. Int. J. Dermatol. 49(7):834–841. doi: 10.1111/j.1365-4632.2010.04481.x.  [DOI](https://doi.org/10.1111/j.1365-4632.2010.04481.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20618508/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Dermatol&title=Cutaneous%20adverse%20drug%20reactions%20seen%20in%20a%20tertiary%20hospital%20in%20Johor,%20Malaysia&author=WY%20Ding&author=CK%20Lee&author=SE%20Choon&volume=49&issue=7&pages=834-841&pmid=20618508&doi=10.1111/j.1365-4632.2010.04481.x&)

5. Locharernkul C, Loplumlert J, Limotai C, et al. Carbamazepine and phenytoin induced Stevens-Johnson syndrome is associated with HLA-B*1502 allele in Thai population. Epilepsia. 2008;49(12):2087–2091. doi: 10.1111/j.1528-1167.2008.01719.x.  [DOI](https://doi.org/10.1111/j.1528-1167.2008.01719.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18637831/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Epilepsia&title=Carbamazepine%20and%20phenytoin%20induced%20Stevens-Johnson%20syndrome%20is%20associated%20with%20HLA-B*1502%20allele%20in%20Thai%20population&author=C%20Locharernkul&author=J%20Loplumlert&author=C%20Limotai&volume=49&issue=12&publication_year=2008&pages=2087-2091&pmid=18637831&doi=10.1111/j.1528-1167.2008.01719.x&)

6. Mehta TY, Prajapati LM, Mittal B, et al. Association of HLA-B*1502 allele and carbamazepine-induced Stevens-Johnson syndrome among Indians. Indian J. Dermatol. Venereol. Leprol. 2009;75(6):579–582. doi: 10.4103/0378-6323.57718.  [DOI](https://doi.org/10.4103/0378-6323.57718) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19915237/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Indian%20J.%20Dermatol.%20Venereol.%20Leprol&title=Association%20of%20HLA-B*1502%20allele%20and%20carbamazepine-induced%20Stevens-Johnson%20syndrome%20among%20Indians&author=TY%20Mehta&author=LM%20Prajapati&author=B%20Mittal&volume=75&issue=6&publication_year=2009&pages=579-582&pmid=19915237&doi=10.4103/0378-6323.57718&)

7. Shi YW, Min FL, Liu XR, et al. Hla-B alleles and lamotrigine-induced cutaneous adverse drug reactions in the Han Chinese population. Basic Clin. Pharmacol. Toxicol. 2011;109(1):42–46. doi: 10.1111/j.1742-7843.2011.00681.x.  [DOI](https://doi.org/10.1111/j.1742-7843.2011.00681.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21306565/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Basic%20Clin.%20Pharmacol.%20Toxicol&title=Hla-B%20alleles%20and%20lamotrigine-induced%20cutaneous%20adverse%20drug%20reactions%20in%20the%20Han%20Chinese%20population&author=YW%20Shi&author=FL%20Min&author=XR%20Liu&volume=109&issue=1&publication_year=2011&pages=42-46&pmid=21306565&doi=10.1111/j.1742-7843.2011.00681.x&)

8. An DM, Wu XT, Hu FY, Yan B, Stefan H, Zhou D. Association study of lamotrigine-induced cutaneous adverse reactions and HLA-B*1502 in a Han Chinese population. Epilepsy Res. 2010;92(2-3):226–230. doi: 10.1016/j.eplepsyres.2010.10.006.  [DOI](https://doi.org/10.1016/j.eplepsyres.2010.10.006) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21071176/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Epilepsy%20Res&title=Association%20study%20of%20lamotrigine-induced%20cutaneous%20adverse%20reactions%20and%20HLA-B*1502%20in%20a%20Han%20Chinese%20population&author=DM%20An&author=XT%20Wu&author=FY%20Hu&author=B%20Yan&author=H%20Stefan&volume=92&issue=2-3&publication_year=2010&pages=226-230&pmid=21071176&doi=10.1016/j.eplepsyres.2010.10.006&)

9. Hu FY, Wu XT, An DM, Yan B, Stefan H, Zhou D. Pilot association study of oxcarbazepine-induced mild cutaneous adverse reactions with HLA-B*1502 allele in Chinese Han population. Seizure. 2011;20(2):160–162. doi: 10.1016/j.seizure.2010.11.014.  [DOI](https://doi.org/10.1016/j.seizure.2010.11.014) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21169036/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Seizure&title=Pilot%20association%20study%20of%20oxcarbazepine-induced%20mild%20cutaneous%20adverse%20reactions%20with%20HLA-B*1502%20allele%20in%20Chinese%20Han%20population&author=FY%20Hu&author=XT%20Wu&author=DM%20An&author=B%20Yan&author=H%20Stefan&volume=20&issue=2&publication_year=2011&pages=160-162&pmid=21169036&doi=10.1016/j.seizure.2010.11.014&)

10. Alfirevic A, Jorgensen AL, Williamson PR, Chadwick DW, Park BK, Pirmohamed M. HLA-B locus in Caucasian patients with carbamazepine hypersensitivity. Pharmacogenomics. 2006;7(6):813–818. doi: 10.2217/14622416.7.6.813.  [DOI](https://doi.org/10.2217/14622416.7.6.813) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16981842/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=HLA-B%20locus%20in%20Caucasian%20patients%20with%20carbamazepine%20hypersensitivity&author=A%20Alfirevic&author=AL%20Jorgensen&author=PR%20Williamson&author=DW%20Chadwick&author=BK%20Park&volume=7&issue=6&publication_year=2006&pages=813-818&pmid=16981842&doi=10.2217/14622416.7.6.813&)

11. Lonjou C, Thomas L, Borot N, et al. A marker for Stevens-Johnson syndrome ethnicity matters. Pharmacogenomics J. 2006;6(4):265–268. doi: 10.1038/sj.tpj.6500356.  [DOI](https://doi.org/10.1038/sj.tpj.6500356) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16415921/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=A%20marker%20for%20Stevens-Johnson%20syndrome%20ethnicity%20matters&author=C%20Lonjou&author=L%20Thomas&author=N%20Borot&volume=6&issue=4&publication_year=2006&pages=265-268&pmid=16415921&doi=10.1038/sj.tpj.6500356&)

12. Gonzalez-Galarza FF, Christmas S, Middleton D, Jones AR. Allele frequency net: a database and online repository for immune gene frequencies in worldwide populations. Nucleic Acids Res. 2011;39(Database issue):D913–D919. doi: 10.1093/nar/gkq1128.  [DOI](https://doi.org/10.1093/nar/gkq1128) | [PMC free article](/articles/PMC3013710/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21062830/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=Allele%20frequency%20net:%20a%20database%20and%20online%20repository%20for%20immune%20gene%20frequencies%20in%20worldwide%20populations&author=FF%20Gonzalez-Galarza&author=S%20Christmas&author=D%20Middleton&author=AR%20Jones&volume=39&issue=Database%20issue&publication_year=2011&pages=D913-D919&pmid=21062830&doi=10.1093/nar/gkq1128&)

13. Hung SI, Chung WH, Jee SH, et al. Genetic susceptibility to carbamazepine-induced cutaneous adverse drug reactions. Pharmacogenet. Genomics. 2006;16(4):297–306. doi: 10.1097/01.fpc.0000199500.46842.4a.  [DOI](https://doi.org/10.1097/01.fpc.0000199500.46842.4a) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16538176/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet.%20Genomics&title=Genetic%20susceptibility%20to%20carbamazepine-induced%20cutaneous%20adverse%20drug%20reactions&author=SI%20Hung&author=WH%20Chung&author=SH%20Jee&volume=16&issue=4&publication_year=2006&pages=297-306&pmid=16538176&doi=10.1097/01.fpc.0000199500.46842.4a&)

14. Mccormack M, Alfirevic A, Bourgeois S, et al. HLA-A*3101 and carbamazepine-induced hypersensitivity reactions in Europeans. N. Engl. J. Med. 2011;364(12):1134–1143. doi: 10.1056/NEJMoa1013297.  [DOI](https://doi.org/10.1056/NEJMoa1013297) | [PMC free article](/articles/PMC3113609/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21428769/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%20Engl.%20J.%20Med&title=HLA-A*3101%20and%20carbamazepine-induced%20hypersensitivity%20reactions%20in%20Europeans&author=M%20Mccormack&author=A%20Alfirevic&author=S%20Bourgeois&volume=364&issue=12&publication_year=2011&pages=1134-1143&pmid=21428769&doi=10.1056/NEJMoa1013297&)

15. Ozeki T, Mushiroda T, Yowang A, et al. Genome-wide association study identifies HLA-A*3101 allele as a genetic risk factor for carbamazepine-induced cutaneous adverse drug reactions in Japanese population. Hum. Mol. Genet. 2010;20(5):1034–1041. doi: 10.1093/hmg/ddq537.  [DOI](https://doi.org/10.1093/hmg/ddq537) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21149285/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum.%20Mol.%20Genet&title=Genome-wide%20association%20study%20identifies%20HLA-A*3101%20allele%20as%20a%20genetic%20risk%20factor%20for%20carbamazepine-induced%20cutaneous%20adverse%20drug%20reactions%20in%20Japanese%20population&author=T%20Ozeki&author=T%20Mushiroda&author=A%20Yowang&volume=20&issue=5&publication_year=2010&pages=1034-1041&pmid=21149285&doi=10.1093/hmg/ddq537&)

16. Kim SH, Lee KW, Song WJ, et al. Carbamazepine-induced severe cutaneous adverse reactions and HLA genotypes in Koreans. Epilepsy Res. 2011;97(1-2):190–197. doi: 10.1016/j.eplepsyres.2011.08.010.  [DOI](https://doi.org/10.1016/j.eplepsyres.2011.08.010) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21917426/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Epilepsy%20Res&title=Carbamazepine-induced%20severe%20cutaneous%20adverse%20reactions%20and%20HLA%20genotypes%20in%20Koreans&author=SH%20Kim&author=KW%20Lee&author=WJ%20Song&volume=97&issue=1-2&publication_year=2011&pages=190-197&pmid=21917426&doi=10.1016/j.eplepsyres.2011.08.010&)

17. Li Y, Willer CJ, Ding J, Scheet P, Abecasis GR. MaCH: using sequence and genotype data to estimate haplotypes and unobserved genotypes. Genet. Epidemiol. 2010;34(8):816–834. doi: 10.1002/gepi.20533.  [DOI](https://doi.org/10.1002/gepi.20533) | [PMC free article](/articles/PMC3175618/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21058334/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genet.%20Epidemiol&title=MaCH:%20using%20sequence%20and%20genotype%20data%20to%20estimate%20haplotypes%20and%20unobserved%20genotypes&author=Y%20Li&author=CJ%20Willer&author=J%20Ding&author=P%20Scheet&author=GR%20Abecasis&volume=34&issue=8&publication_year=2010&pages=816-834&pmid=21058334&doi=10.1002/gepi.20533&)

18. Purcell S, Neale B, Todd-Brown K, et al. PLINK: a tool set for whole-genome association and population-based linkage analyses. Am. J. Hum. Genet. 2007;81(3):559–575. doi: 10.1086/519795.  [DOI](https://doi.org/10.1086/519795) | [PMC free article](/articles/PMC1950838/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17701901/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Hum.%20Genet&title=PLINK:%20a%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses&author=S%20Purcell&author=B%20Neale&author=K%20Todd-Brown&volume=81&issue=3&publication_year=2007&pages=559-575&pmid=17701901&doi=10.1086/519795&)

19. Price AL, Patterson NJ, Plenge RM, Weinblatt ME, Shadick NA, Reich D. Principal components analysis corrects for stratification in genome-wide association studies. Nat. Genet. 2006;38(8):904–909. doi: 10.1038/ng1847.  [DOI](https://doi.org/10.1038/ng1847) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16862161/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet&title=Principal%20components%20analysis%20corrects%20for%20stratification%20in%20genome-wide%20association%20studies&author=AL%20Price&author=NJ%20Patterson&author=RM%20Plenge&author=ME%20Weinblatt&author=NA%20Shadick&volume=38&issue=8&publication_year=2006&pages=904-909&pmid=16862161&doi=10.1038/ng1847&)

20. Ge D, Zhang K, Need AC, et al. WGAViewer: software for genomic annotation of whole genome association studies. Genome Res. 2008;18(4):640–643. doi: 10.1101/gr.071571.107.  [DOI](https://doi.org/10.1101/gr.071571.107) | [PMC free article](/articles/PMC2279251/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18256235/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res&title=WGAViewer:%20software%20for%20genomic%20annotation%20of%20whole%20genome%20association%20studies&author=D%20Ge&author=K%20Zhang&author=AC%20Need&volume=18&issue=4&publication_year=2008&pages=640-643&pmid=18256235&doi=10.1101/gr.071571.107&)

21. Menashe I, Rosenberg PS, Chen BE. PGA: power calculator for case-control genetic association analyses. BMC Genet. 2008;9:36. doi: 10.1186/1471-2156-9-36.  [DOI](https://doi.org/10.1186/1471-2156-9-36) | [PMC free article](/articles/PMC2387159/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18477402/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Genet&title=PGA:%20power%20calculator%20for%20case-control%20genetic%20association%20analyses&author=I%20Menashe&author=PS%20Rosenberg&author=BE%20Chen&volume=9&publication_year=2008&pages=36&pmid=18477402&doi=10.1186/1471-2156-9-36&)

22. Man CB, Kwan P, Baum L, et al. Association between HLA-B*1502 allele and antiepileptic drug-induced cutaneous reactions in Han Chinese. Epilepsia. 2007;48(5):1015–1018. doi: 10.1111/j.1528-1167.2007.01022.x.  [DOI](https://doi.org/10.1111/j.1528-1167.2007.01022.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17509004/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Epilepsia&title=Association%20between%20HLA-B*1502%20allele%20and%20antiepileptic%20drug-induced%20cutaneous%20reactions%20in%20Han%20Chinese&author=CB%20Man&author=P%20Kwan&author=L%20Baum&volume=48&issue=5&publication_year=2007&pages=1015-1018&pmid=17509004&doi=10.1111/j.1528-1167.2007.01022.x&)

23. Solberg OD, Mack SJ, Lancaster AK, et al. Balancing selection and heterogeneity across the classical human leukocyte antigen loci: a meta-analytic review of 497 population studies. Hum. Immunol. 2008;69(7):443–464. doi: 10.1016/j.humimm.2008.05.001.  [DOI](https://doi.org/10.1016/j.humimm.2008.05.001) | [PMC free article](/articles/PMC2632948/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18638659/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum.%20Immunol&title=Balancing%20selection%20and%20heterogeneity%20across%20the%20classical%20human%20leukocyte%20antigen%20loci:%20a%20meta-analytic%20review%20of%20497%20population%20studies&author=OD%20Solberg&author=SJ%20Mack&author=AK%20Lancaster&volume=69&issue=7&publication_year=2008&pages=443-464&pmid=18638659&doi=10.1016/j.humimm.2008.05.001&)
