# Limited contribution of common genetic variants to risk for liver injury due to a variety of drugs

## Metadata
**Authors:** Thomas J Urban, Yufeng Shen, Andrew Stolz, Naga Chalasani, Robert J Fontana, James Rochon, Dongliang Ge, Kevin V Shianna, Ann K Daly, M Isabel Lucena, Matthew R Nelson, Mariam Molokhia, Guruprasad P Aithal, Aris Floratos, Itsik Pe’er, Jose Serrano, Herbert Bonkovsky, Timothy J Davern, William M Lee, Victor J Navarro, Jayant A Talwalkar, David B Goldstein, Paul B Watkins
**Journal:** Pharmacogenetics and genomics
**Date:** 2012 Nov
**DOI:** [10.1097/FPC.0b013e3283589a76](https://doi.org/10.1097/FPC.0b013e3283589a76)
**PMID:** 22968431
**PMCID:** PMC3636716
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3636716/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC3636716/pdf/nihms-446262.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC3636716/pdf/nihms-446262.pdf)

## Abstract

**Background and aims:** 
Drug-induced liver injury (DILI) is a serious adverse drug event that is suspected to have a heritable component. We carried out a genome-wide association study of 783 individuals of European ancestry who experienced DILI due to more than 200 implicated drugs.

**Methods:** 
DILI patients from the US-based Drug-Induced Liver Injury Network (n = 401) and three international registries (n = 382) were genotyped with the Illumina 1Mduo BeadChip and compared with population controls (n = 3001). Potential associations were tested in 307 independent Drug-Induced Liver Injury Network cases.

**Results:** 
After accounting for known major histocompatibility complex risk alleles for flucloxacillin-DILI and amoxicillin/clavulanate-DILI, there were no genome-wide significant associations, including in the major histocompatibility complex region. Stratification of DILI cases according to clinical phenotypes (injury type, latency, age of onset) also did not show significant associations. An analysis of hepatocellular DILI (n = 285) restricted to 193 single-nucleotide polymorphisms previously associated with autoimmune disease showed a trend association for rs7574865, in the vicinity of signal transducer and activator of transcription 4 (STAT4) (P = 4.5 × 10−4). This association was replicated in an independent cohort of 168 hepatocellular DILI cases (P = 0.011 and 1.5 × 10−5 for combined cohorts). No significant associations were found with stratification by other clinical or demographic variables.

**Conclusion:** 
Although not significant at the genome-wide level, the association between hepatocellular DILI and STAT4 is consistent with the emerging role of the immune system in DILI. However, the lack of genome-wide association study findings supports the idea that strong genetic determinants of DILI may be largely drug-specific or may reflect rare genetic variations, which were not assessed in our study.

Keywords: drug-induced liver injury, genome-wide association study, pharmacogenetics

### Background and aims

Drug-induced liver injury (DILI) is a serious adverse drug event that is suspected to have a heritable component. We carried out a genome-wide association study of 783 individuals of European ancestry who experienced DILI due to more than 200 implicated drugs.

### Methods

DILI patients from the US-based Drug-Induced Liver Injury Network (*n* = 401) and three international registries (*n* = 382) were genotyped with the Illumina 1Mduo BeadChip and compared with population controls (*n* = 3001). Potential associations were tested in 307 independent Drug-Induced Liver Injury Network cases.

### Results

After accounting for known major histocompatibility complex risk alleles for flucloxacillin-DILI and amoxicillin/clavulanate-DILI, there were no genome-wide significant associations, including in the major histocompatibility complex region. Stratification of DILI cases according to clinical phenotypes (injury type, latency, age of onset) also did not show significant associations. An analysis of hepatocellular DILI (*n* = 285) restricted to 193 single-nucleotide polymorphisms previously associated with autoimmune disease showed a trend association for rs7574865, in the vicinity of signal transducer and activator of transcription 4 (*STAT4*) (*P* = 4.5 × 10^−4^). This association was replicated in an independent cohort of 168 hepatocellular DILI cases (*P* = 0.011 and 1.5 × 10^−5^ for combined cohorts). No significant associations were found with stratification by other clinical or demographic variables.

### Conclusion

Although not significant at the genome-wide level, the association between hepatocellular DILI and *STAT4* is consistent with the emerging role of the immune system in DILI. However, the lack of genome-wide association study findings supports the idea that strong genetic determinants of DILI may be largely drug-specific or may reflect rare genetic variations, which were not assessed in our study.

## Introduction

Idiosyncratic drug-induced liver injury (DILI) is a major cause of acute liver failure [[1](#R1)], but this statistic alone considerably underestimates the public health impact of DILI. For example, concern for DILI can limit patient access to drugs that might otherwise be beneficial. There are currently no tests available that can definitively establish the diagnosis of DILI or indicate which drug is responsible if a patient takes multiple medications. As a consequence, key drugs may be unnecessarily stopped, thereby creating a health risk to the patient. DILI is also the single, major adverse event that leads to termination of clinical drug development programs and to regulatory actions on approved drugs [[2](#R2)]. Concern for a DILI event during early clinical trials frequently leads to longer and larger clinical trials requested by the FDA, thus delaying patient access to important new medications and increasing the costs of bringing new drugs into the market. Improved understanding of underlying DILI mechanisms could lead to clinical tests that might be used to identify patients susceptible to DILI from a given drug, to confirm the diagnosis of DILI, and to help to identify the offending agent in a patient taking multiple drugs. This understanding should also lead to improved preclinical methodology to detect DILI potential in new drug candidates.

A strategy to identify the mechanisms underlying DILI has been genetic analysis of patients who have actually experienced DILI. The studies to date support the concept that DILI susceptibility results in part from genetic variation in at least four major classes of genes: drug metabolism including toxification and detoxification enzymes, drug and bile acid transporters, cellular stress response genes, and innate and adaptive immune response genes including the major histocompatibility complex (MHC) region genes [[3](#R3),[4](#R4)].

We hypothesized that some DILI susceptibility factors are shared across multiple drugs for several reasons. Individual drug-metabolizing enzymes and drug transporters are often able to transport or metabolize many different drug substrates. Thus, individual DILI risk factors at the level of enzymes and transporters could incur a risk for multiple drugs. Similarly, cellular responses to stress are finite and risk factors in these pathways would likely not be drug-specific. Furthermore, previous genetic associations support the concept that specific genetic susceptibility variants may be shared by different drugs [[3](#R3),[4](#R4)]. Finally, common risk factors for DILI are supported by the observation that in a cohort of Spanish DILI patients, nine had experienced more than one discrete episode of DILI due to different drugs [[5](#R5)], suggesting that some individuals may be particularly susceptible to DILI from multiple agents.

We carried out a genome-wide association study (GWAS) of DILI in 783 individuals of European ancestry who experienced DILI due to a large number of implicated drugs (> 200). Possible associations were then evaluated in a smaller replication cohort (*n* = 307). This report represents the largest investigation of DILI genetics to date.

## Methods

### Study participants

Cases (*n* = 783) were recruited from four separate studies [Drug-Induced Liver Injury Network (DILIN) [[6](#R6)], DILIGEN [[7](#R7)], EUDRAGENE [[8](#R8)], and the Spanish DILI Registry [[9](#R9)]]. A total of 565 cases were recruited from the DILIN network between August 2004 and April 2009 from eight DILIN clinical sites in the USA; 401 cases of European ancestry were included in the current study. The contribution of cases of European ancestry from the other networks was as follows: DILIGEN, 242 cases; EUDRAGENE, 89 cases; and Spanish DILI Registry, 51 cases. All participants provided written informed consent and each study was approved by the appropriate institutional review boards. Details of recruitment and the inclusion and exclusion criteria for these networks have been published previously [[10](#R10)]. For all cases in the DILIN, causality assessment was by expert consensus as described previously [[6](#R6)], whereas the Roussel-Uclaf causality assessment method (RUCAM) was used in the other three registries [[11](#R11)]. Notably, the cases studied here include 296 samples from DILI patients with injury attributed to flucloxacillin or amoxicillin/clavulanate that overlap almost completely with the samples used in previous reports [[7](#R7),[10](#R10)]. These were included here to identify the common variants contributing to DILI susceptibility across different drugs, and should not be taken as replicating previous published genetic associations. The replication cohort consisted of 307 cases from the DILIN network. Genotyped controls (*n* = 655) from the Population Reference Sample (POPRES, *n* = 655) [[12](#R12)] and the 1958 British Birth Cohort ([http://www.b58cgene.sgul.ac.uk](http://www.b58cgene.sgul.ac.uk)) (*n* = 2346) were used for comparison. Replication studies utilized genotype data from the National Blood Service cohort (*n* = 2249). Genotype data for the 1958 British Birth Cohort and the National Blood Service cohort were provided by the Wellcome Trust Case–Control Consortium ([http://www.wtccc.org.uk](http://www.wtccc.org.uk)).

### Secondary phenotypes

Cases were categorized as hepatocellular versus cholestatic versus mixed using the *R*-value [[11](#R11)]. Other secondary phenotypes included demographic or clinical strata (sex, age greater than or less than the median, time to onset greater than or less than the median, presence of eosinophilia, fever, or rash). Because experienced hepatologists enrolled each patient only after concluding that the patients probably had DILI, and because causality assessment protocols were not uniform across the different cohorts, our primary analyses were carried out on all cases where genotyping data were available. When potential associations were evident, post-hoc analyses were carried out by dichotomizing the sample into those with a causality score of at least probable (DILIN score of ‘probable’, ‘very likely’, or ‘definite’; RUCAM ≥ 6) versus those with causality less than probable (see [Supplementary Tables S1](#SD1) and [S2](#SD1), [http://links.lww.com/FPC/A505](http://links.lww.com/FPC/A505)).

### Genotyping

DNA was prepared as described previously [[10](#R10)]. Genome-wide genotyping of the European DILI cases and POPRES controls was carried out by Expression Analysis Inc. (Durham, North Carolina, USA) and of the USA cases by the Center for Human Genome Variation, Duke University. Genotype data for the 1958 British Birth Cohort controls were provided by the Wellcome Trust Case–Control Consortium 2. All patients were genotyped using the Illumina Human1M or 1MDuo BeadChip (Illumina Inc., San Diego, California, USA), each containing greater than 1 000 000 markers. A total of 800 769 markers, 783 cases (401 DILIN, 242 DILIGEN, 89 EUDRAGENE, and 51 Spanish DILI Registry), and 3001 controls passed quality control carried out as described previously [[13](#R13)].

### Statistical analysis

Population structure of the combined case and control sample was estimated by principal components analysis using a modification of EIGENSTRAT [[14](#R14)] as described previously [[13](#R13)]. Tests for association between the single-nucleotide polymorphism (SNP) genotype and DILI were carried out using logistic regression in PLINK [[15](#R15)] using the top 14 principal components emerging from the EIGENSTRAT analyses as covariates in the model. Where applicable, the top-associated SNPs in the MHC region for flucloxacillin-DILI (rs2395029) and amoxicillin/clavulanate-DILI (rs2523822, rs3135388) that were present in the quality control filtered dataset were also included as covariates to determine whether any risk for all-drug DILI may be explained by other variants after correcting for these known DILI risk alleles. Association test results were annotated and visualized using the WGAviewer software [[16](#R16)]. We also carried out hypothesis-guided exploratory analysis of genes involved in drug absorption, distribution, metabolism, and excretion (ADME) [[17](#R17)], or non-HLA genes known to be generally involved in autoimmune diseases [[18](#R18)], as described previously [[10](#R10)].

Replication genotyping was carried out using TaqMan SNP Genotyping Assays (Life Technologies, Carlsbad, California, USA). Replication testing was carried out using the χ^2^ or Fisher’s exact test of allele frequency differences between 307 European-ancestry DILI cases enrolled in the DILIN network not included in the GWAS and 2587 samples of European descent from the National Blood Service. Combined *P*-values for the discovery and replication sets were calculated using Stouffer’s weighted *Z* method. Power estimates were performed using the PGA software [[19](#R19)].

## Results

### Characteristics of the study population

The DILI case samples included a discovery cohort of 783 patients of primarily European ancestry, all of which were genotyped on the Illumina 1M or 1Mduo BeadChip; a replication cohort consisting of 307 US patients who were self-identified White, non-Hispanic, was available from the DILIN for follow-up of variants of interest emerging from the GWAS. The demographic and clinical characteristics of both cohorts are shown in [Table 1](#T1). The representation of implicated drugs ([Table 2](#T2)) was comparable between cohorts, with the exception of flucloxacillin, which was absent from the US-based (DILIN) replication cohort, and amoxicillin/clavulanate.

### Table 1.

|   | N (%) or median (range) |  |
| --- | --- | --- |
| Variables | Discovery sample | Replication sample |
| Sex |   |   |
| Male | 329 (42.0) | 136 (44.3) |
| Female | 454 (58.0) | 171 (55.7) |
| Age of onset (years) | 57.5 (1.8–98) | 54.2 (5.8–88) |
| Time to onset (days) | 24 (0–7045) | 39 (0–11463) |
| Injury type |   |   |
| Cholestatic | 187 (23.9) | 90 (29.3) |
| Mixed | 164 (20.9) | 64 (20.8) |
| Hepatocellular | 256 (32.7) | 139 (45.3) |
| Unknown/unclassified | 176 (22.5) | 14 (4.6) |
| Severity |   |   |
| Mild | 105 (13.4) | 50 (16.3) |
| Moderate | 80 (10.2) | 62 (20.2) |
| Moderate (hospitalized) | 124 (15.8) | 48 (15.6) |
| Severe | 47 (6.0) | 32 (10.4) |
| Fatal | 20 (2.6) | 16 (5.2) |
| Unknown/unclassified | 407 (52.0) | 99 (32.2) |
| Causality score |   |   |
| ‘Unlikely’ or ‘possible’ | 83 (10.6) | 35 (11.4) |
| ‘Probable’, ‘very likely’, ‘definite’ | 622 (79.4) | 173 (56.4) |
| Unknown/unclassified | 78 (10.0) | 99 (32.2) |
| Immunoallergic features |   |   |
| Eosinophilia | 46 (5.9) | NA |
| Fever | 161 (20.6) | NA |
| Rash | 125 (16.0) | NA |
| Any of above | 256 (32.7) | NA |

Table 1 Caption: Clinical characteristics of the study population in the initial discovery sample used for genome-wide association study (n=783) and in the replication cohort (n=307)

### Table 2.

|   | N (%) |  |
| --- | --- | --- |
| Drugs | Discoverysample | Replicationsample |
| Amoxicillin/clavulanate | 208 (26.6) | 42 (13.7) |
| Flucloxacillin | 75 (9.6) | 0 (0.0) |
| Diclofenac | 30 (3.8) | 4 (1.3) |
| Nitrofurantoin | 24 (3.1) | 15 (4.9) |
| Isoniazid | 17 (2.2) | 7 (2.3) |
| Sulfamethoxazole/trimethoprim | 14 (1.8) | 14 (4.6) |
| Valproic acid | 13 (1.7) | 2 (0.6) |
| Nimesulide | 12 (1.5) | 0 (0.0) |
| Minocycline | 10 (1.3) | 17 (5.5) |
| Celecoxib | 8 (1.0) | 1 (0.3) |
| Flucloxacillin and amoxicillin/ clavulanate | 8 (1.0) | 0 (0.0) |
| Ciprofloxacin | 7 (0.9) | 9 (2.9) |
| Phenytoin | 7 (0.9) | 1 (0.3) |
| Azathioprine | 6 (0.8) | 5 (1.6) |
| Duloxetine | 6 (0.8) | 0 (0.0) |
| Telithromycin | 6 (0.8) | 0 (0.0) |
| Atorvastatin | 5 (0.6) | 3 (1.0) |
| Simvastatin | 5 (0.6) | 4 (1.3) |
| All other drugs and combinations | 322 (41.1) | 183 (59.6) |

Table 2 Caption: The most common drugs implicated in drug-induced liver injury cases (drugs with n ≥ 5) in both the discovery sample and in the individuals available in the Drug-Induced Liver Injury Network replication cohort

### Genome-wide association study results: all drug-induced liver injury cases

Principal components analysis showed that the 783 DILI cases and 3001 controls were well matched ancestrally ([Supplementary Fig. S1](#SD1), [http://links.lww.com/FPC/A505](http://links.lww.com/FPC/A505)). When all case patients were included in the analysis, we observed a large number of SNPs in the MHC region on chromosome 6p with genome-wide significance ([Fig. 1a and d](#F1)). The MHC region has been shown previously to contain high-impact risk alleles for DILI due to flucloxacillin and amoxicillin/clavulanate, and in fact, these MHC associations for flucloxacillin and amoxicillin/clavulanate have been reported previously using DNA samples from individuals overlapping significantly with the participants in the current study [[7](#R7),[10](#R10)]. These cases were included here to improve power to detect novel genetic associations (anywhere in the genome) with DILI risk across all drugs. Given that these samples represent a large fraction of the case sample in the GWAS ([Table 2](#T2)) and may thus obscure other MHC associations with DILI due to other drugs, we also carried out a GWAS utilizing only DILI cases not explained by these two drugs. In the analysis of 487 DILI cases not attributed to flucloxacillin or amoxicillin/clavulanate, the observed *P*-value distribution genome-wide was not different from that expected under the null hypothesis, and no single SNP in the MHC region or elsewhere exceeded the typical threshold for genome-wide statistical significance (*P* < 10^−8^), ([Fig. 1b and e](#F1)). Similarly, when all drugs were considered, but the association tests were carried out conditional on the known MHC DILI risk SNPs for flucloxacillin and amoxicillin/clavulanate [[7](#R7),[10](#R10)], no genome-wide significant associations were detected ([Fig. 1c and f](#F1), [Table 3](#T3)).

### Fig. 1.

![Fig. 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e8d7/3636716/7c23938f3b3f/nihms-446262-f0001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3636716_nihms-446262-f0001.jpg)

Manhattan and Quantile–quantile plots of association study results including (a, d) or excluding (b, e) drug-induced liver injury (DILI) cases due to amoxicillin/clavulanate or flucloxacillin, for which strong genetic risk variants in the MHC region have been identified previously [7]. The genome-wide association study was also carried out including all cases, but conditioned on the known major histocompatibility complex single-nucleotide polymorphisms associated with flucloxacillin-induced or amoxicillin/clavulanate-induced DILI (c, f). The Manhattan plot shows the location along the genome on the x-axis and the –log10 P-value along the x-axis. A P-value less than 10−8 was considered significant. The quantile–quantile plot shows in rank order the observed P-values versus those expected under the null hypothesis, showing adequate correction for population structure (genomic inflation factor, λ = 1.02) and significant deviation from the null for the top-associated SNPs only in the analysis including amoxicillin/clavulanate and flucloxacillin cases.

### Table 3.

| SNP Sets | SNP | P-value | Chromosome:coordinate | Function | Nearest gene |
| --- | --- | --- | --- | --- | --- |
| Genome-wide | rs35709459 | 1.99E – 07 | X:91516677 | Intronic | PCDH11X |
|   | rs35925943 | 1.38E – 06 | X:91516339 | Intronic | PCDH11X |
|   | rs11767067 | 1.64E – 06 | 7:93022840 | Intergenic | CALCR |
|   | rs1187997 | 2.18E – 06 | 1:58455538 | Intronic | DAB1 |
|   | rs656437 | 4.24E – 06 | 11:95770378 | Intronic | MAML2 |
| ADME subset | rs3740065 | 0.0001 | 10:101605693 | Intronic | ABCC2 |
|   | rs2756113 | 0.0008 | 10:101569371 | Intronic | ABCC2 |
|   | rs2804397 | 0.0011 | 10:101559415 | Intronic | ABCC2 |
|   | rs8187710 | 0.0013 | 10:101611294 | Missense (C1515Y) | ABCC2 |
|   | rs11816708 | 0.0013 | 10:101615015 | Downstream | ABCC2 |
| Autoimmune subset | rs2476601 | 2.26E – 05 | 1:114377568 | Missense (R620W) | PTPN22 |
|   | rs6679677 | 4.18E – 05 | 1:114303808 | Upstream | RSBN1 |
|   | rs7202877 | 0.0002 | 16:75247245 | Intergenic | CTRB1 |
|   | rs1464510 | 0.003 | 3:188112554 | Intronic | LPP |
|   | rs12928822 | 0.0033 | 16:11403893 | Upstream | MIR548H2 |

Table 3 Caption: The top-associated variants in all drug-induced liver injury casesa for all single-nucleotide polymorphisms and selected subsets of single-nucleotide polymorphisms associated with absorption, distribution, metabolism, and excretion and autoimmune genes

As the MHC region has been implicated previously in DILI risk through GWAS, we determined whether the MHC region may be enriched for SNPs associated with DILI risk after the exclusion of the flucloxacillin and amoxicillin/clavulanate cases. Among the 487 non-flucloxacillin, non-amoxicillin/clavulanate-related DILI cases, we observed a lower *P*-value distribution among SNPs in the extended MHC region (chr6: 26 000 000–35 500 000) compared with all other autosomal SNPs ([Supplementary Fig. S4A, S4B](#SD1), [http://links.lww.com/FPC/A505](http://links.lww.com/FPC/A505)). To account for linkage disequilibrium prevalent in the MHC region, we randomly permuted the phenotype–genotype relationship among all individuals and repeated the association tests in the MHC region 1000 times. Among 1000 permutations, the observed median *P*-value in the observed (unpermuted) data (0.381) was in the lower 1.5 percentile of the median *P*-values in the MHC region (empirical *P* = 0.015, [Supplementary Fig. S4C](#SD1), [http://links.lww.com/FPC/A505](http://links.lww.com/FPC/A505)).

### Drug metabolism and transporter single-nucleotide polymorphism analysis

Because drug-metabolizing enzymes and transporters have been assumed to play critical roles in mechanisms underlying DILI, we tested whether any SNPs in the vicinity of 130 genes involved in ADME [[17](#R17)] showed a greater degree of association with DILI risk than expected by chance. We found that several SNPs in the region of *ABCC2* (encoding the multidrug resistance-associated protein MRP2) were the most associated SNPs in the ADME category ([Table 3](#T3)). The *P*-value corresponding to the highest associated SNP (rs3740065; *P* = 2.6 × 10^−5^) approached the specified threshold for significance after multiple test correction (*P* < 10^−5^). In addition, post-hoc analysis showed that the association with this SNP became stronger as the analysis was restricted to individuals with time to onset less than 30 days and causality scores of ‘probable’ or greater (RUCAM ICC ≥ 6 or DILIN causality of ‘probable’, ‘very likely’, or ‘definite’) (*P* = 2.5 × 10^−6^) ([Supplementary Table S4](#SD1), [http://links.lww.com/FPC/A505](http://links.lww.com/FPC/A505)). However, in a replication cohort of 304 independently genotyped DILI cases and 2249 controls from the National Blood Service cohort, there was no evidence of an association between rs3740065 and DILI risk (*P* = 0.49) ([Table 4](#T4)). Of note, no significant association with SNPs in the vicinity of the bile salt excretory protein gene (*ABCB11*) or manganese superoxide dismutase (*SOD2*) genes was found, although these have been implicated previously in DILI mechanisms [[3](#R3),[4](#R4)].

### Table 4.

|   |   |   |   | Discovery cohort |  |  | Replication cohort |  |  | Combined |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Phenotypes | SNP set | SNP | Nearest gene | Na | P-value | OR (95%CI) | Nb | P-value | OR (95%CI) | N | P-value |
| All DILI | ADME | rs3740065 | ABCC2 | 783 | 2.6×10−5 | 1.45 (1.22–1.72) | 304 | 0.49 | 0.90 (0.68–1.20) | 1087 | 1.9×10−3 |
| Cholestatic | AI and GWAS | rs2476601 | PTNP22 | 187 | 5.4×10−7 | 2.20 (1.62–3.00) | 105 | 0.26 | 0.74 (0.44–1.24) | 292 | 8.0×10−4 |
| Hepatocellular | AI | rs7574865 | STAT4 | 256 | 4.5×10−4 | 1.45 (1.18–1.79) | 168 | 0.011 | 1.39 (1.08–1.78) | 424 | 1.5×10−5 |
| Diclofenacc | GWAS | rs17036170 | PPARG | 30 | 1.0×10−8 | 11.3 (4.9–25.9) | 26 | 0.088 | 3.59 (0.83–15.6) | 56 | 2.0×10−8 |
| Fluoroquinolones | ADME | rs17862876 | UGT1A | 12 | 3.4×10−5 | 11.6 (3.6–37.4) | 10 | 1.00 | NA | 22 | 8.0×10−4 |
| NSAIDs | GWAS | rs9376256 | IL22RA2 | 68 | 6.7×10−8 | 3.42 (2.19–5.34) | 12 | 0.42 | 0.44 (0.06–3.23) | 80 | 1.0×10−4 |
| Statins | AI | rs7574865 | STAT4 | 27 | 4.9×10−5 | 3.35 (1.87–6.00) | 16 | 0.58 | 0.77 (0.31–1.91) | 43 | 3.5×10−3 |

Table 4 Caption: Top-associated variant and replication test statistics for each of the clinical strata and drug-specific or drug class-specific analyses

### Autoimmune single-nucleotide polymorphism analysis

We next tested whether any of the 193 SNPs associated previously with autoimmune diseases might have a stronger association with DILI risk than expected by chance. This analysis (conditioned on genotypes at the known MHC risk alleles for DILI due to flucloxacillin and amoxicillin/clavulanate) showed rs2476601 in the protein tyrosine phosphatase nonreceptor type 22 (*PTPN22*) gene to be the top-associated autoimmune SNP ([Table 3](#T3)). This association appeared to be driven solely by the cases classified as having a cholestatic injury type (see the section below).

### Genome-wide association study results: clinical strata and drug-specific or class-specific analyses

The results of the stratified analyses are shown in [Supplementary Figs S5](#SD1)–[S38](#SD1) ([http://links.lww.com/FPC/A505](http://links.lww.com/FPC/A505)) and [Supplementary Tables S5](#SD1)–[S38](#SD1) ([http://links.lww.com/FPC/A505](http://links.lww.com/FPC/A505)). Among the clinically defined strata, the most notable results were from the autoimmune-associated SNP sets, which showed experiment-wide significance (i.e. after correction for multiple testing of 193 autoimmune-associated SNPs) for an association between rs2476601, a nonsynonymous coding SNP in the *PTPN22* gene, and cholestatic injury (*P* = 5.4 × 10^−7^), and rs7574865 in the vicinity of signal transducer and activator of transcription 4 (*STAT4*), which showed an almost experiment-wide significant (*P* = 4.5 10^−4^) association with hepatocellular injury ([Table 4](#T4)). Although the association between rs2476601 and cholestatic injury was almost genome-wide significant and would have been declared significant within the autoimmune-restricted SNP analysis, this association was not replicated in the independent cohort of 105 cholestatic DILI cases (*P* = 0.26, with the trend in the opposite direction of that observed in the discovery cohort). The lack of replication could not be attributed to the lower representation of amoxicillin/clavulanate and the absence of flucloxacillin cases in the replication cohort as the other cholestatic DILI cases [*n* = 77, odds ratio (OR) 2.32 (1.45–3.70), *P* = 4.3 × 10^−4^] contributed to an equivalent degree to the initial apparent association as did the flucloxacillin and amoxicillin/clavulanate cases [*n* = 110, OR 2.14 (1.46–3.18), *P* = 1.2 × 10^−4^]. The association between rs7574865 located near the *STAT4* gene and hepatocellular injury was found to replicate in an independent set of 168 cases, although the estimated effect size and strength of association was reduced compared with the discovery cohort ([Table 4](#T4)).

No other associations approaching the specified levels of significance were observed, including examinations by time to onset, age of onset, presence of extrahepatic signs, or severity of injury.

Among the drug-specific or class-specific tests, the most notable results in the GWAS analyses were found for diclofenac (rs17036170, upstream of the *PPARG* gene, *P* = 1.07 × 10^−8^) and NSAIDs (rs9376256, an intergenic SNP in the region of *IL20RA* and *IL22RA2*, *P* = 6.7 × 10^−8^). In the ADME-restricted analysis of fluoroquinolone DILI cases, rs17862876 in the region of the UDP-glycosyltransferase-encoding *UGT1A* gene showed almost experiment-wide significance (3.4 × 10^−5^). In the autoimmune-restricted analysis of DILI due to statins, the rs7574865 polymorphism near *STAT4* reached experiment-wide significance (*P* = 4.9 × 10^−5^). However, for all drug and drug class-specific associations, attempts to replicate the observed findings in independent cohorts were unsuccessful (or ambiguous in the case of diclofenac/*PPARG*) ([Table 4](#T4)). The lack of replication was not explained by differences in the allele frequency between discovery and replication control cohorts (*P* ≥ 0.1 for the difference in frequency between control datasets for all SNPs, except rs17862876, which showed a slightly lower risk allele frequency in the replication controls (*P* = 0.018) and thus would be biased toward association in the same direction as in the discovery set). Neither could it be explained by platform differences in genotyping: among 401 patients of European ancestry from the DILIN cohort genotyped using both the Illumina 1Mduo BeadChip and single-site TaqMan assays, there was perfect concordance in genotype calls across platforms. No associations approaching significance were observed in other drug or drug class cohorts.

### Power calculations

As no genome-wide significant associations were found in our analysis, we then calculated what magnitude of effect [in terms of relative risk (RR)] from truly associated common SNPs would likely have been missed in our analyses. As shown in [Fig. 2a](#F2), the current sample size (793) provides sufficient power (> 80%) to detect SNPs conferring RR more than 2, and almost complete power to detect the effects of SNPs carrying RR greater than 2.5, even for a relatively low-frequency SNP [minor allele frequency (MAF) = 0.05, typically among the lowest frequency SNPs included in GWAS] ([Fig. 2a](#F2)). For the majority of the common SNP frequency spectrum (i.e. MAF > 10%), the current study was estimated to have almost complete power to detect the effects of SNPs carrying RR greater than 1.5 ([Fig. 2b](#F2)). However, analyses of less than 200 cases, which were typical in our analyses of clinical strata, would have fairly low power to detect RR less than 3, and the drug-specific or class-specific analyses utilizing case samples totaling 30 or fewer would generally be powered only to detect common variants with fairly large effects (RR > 5) ([Table 5](#T5)). Further, we estimated the influence of the use of unselected population controls relative to drug-exposed controls on statistical power to detect a genetic association. As suggested previously [[20](#R20)], for rare diagnoses such as DILI, the low rate of expected misclassification of control samples has a negligible effect on power ([Supplementary Table S39](#SD1), [http://links.lww.com/FPC/A505](http://links.lww.com/FPC/A505)).

### Fig. 2.

![Fig. 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e8d7/3636716/d5271a63baaa/nihms-446262-f0002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3636716_nihms-446262-f0002.jpg)

Power estimates for the identification of drug-induced liver injury risk variants given available sample sizes. (a) Power versus sample size over a range of relative risk, assuming a minor allele frequency of 0.05, sample size: n = 783 cases versus n = 3000 controls, population prevalence of 1 : 10 000, and significance threshold P < 10−8. (b) Detectable relative risk (RR) versus risk allele frequency over a range of power (1 – β), assuming r2 = 1 between the marker and the causal risk allele, and sample sizes and significance threshold as above.

### Table 5.

| Sample size | Detectable relative risk at 80% power |
| --- | --- |
| 783 | 1.7 |
| 200 | 2.3 |
| 100 | 2.9 |
| 30 | 5.1 |
| 10 | 10.8 |

Table 5 Caption: Detectable effect size for common variants (minor allele frequency = 10%) as a function of the number of case samples versus 3000 controls

## Discussion

If only a limited number of mechanisms were responsible for idiosyncratic DILI events, preclinical tests could reliably screen and limit drug candidates for DILI liability, and in principle, a ‘DILI chip’ could be used to screen patient populations to identify susceptible individuals and to potentially aid in the diagnosis of DILI. We therefore hoped to find strong and unambiguous associations between common variants and all-cause DILI at the genome-wide significance threshold. However, once the previously described MHC associations with flucloxacillin and amoxicillin/clavulanate-DILI were excluded, no significant genome-wide associations were found in our large cohort of DILI cases attributed to diverse drugs. Because the proposed mechanisms of DILI involve drug metabolism and disposition and immune response, we next restricted our analysis to SNPs in the vicinity of ADME genes and those associated previously with susceptibility to autoimmune diseases. None of the ADME SNP associations suggested in the test cohort were confirmed in the validation cohort.

For the autoimmune analysis only, the association between hepatocellular DILI and an SNP near the *STAT4* gene was also observed in the validation cohort; however, the effect appeared to be attenuated in the validation sample. *STAT4* is involved in inflammation through the regulation of several cytokines and has been implicated in T-cell maturation [[21](#R21)]. To our knowledge, this is the first reported association of an innate immune gene with DILI across a large number of implicated drugs, and supports a potential role of innate immunity in DILI. However, this association did not meet the typical rigorous threshold for significance in a genome-wide study. The reduced effect size observed in the validation cohort suggests the possibility of a ‘winner’s curse’ effect, but is also consistent with a chance association: given the seven SNPs carried forward for validation testing, it is not unlikely that one of them would continue to show an association in the validation cohort simply due to chance. We therefore cannot conclude that this is a true association, but we believe further study of the possible relationship between *STAT4* variation and hepatocellular DILI is warranted.

Although our sample sizes for each implicated drug were generally small, we found some interesting trends toward associations with specific drugs. For example, an SNP upstream of the *PPARG* gene was associated with diclofenac DILI at almost the genome-wide significance level and with an OR of 30 ([Table 4](#T4)). Diclofenac has been shown previously to interact with the PPARγ receptor at pharmacologic concentrations of the drug [[22](#R22)], and PPARγ activation has been shown to influence inflammation, specifically by diclofenac [[23](#R23),[24](#R24)]. Although this strong association was not replicated in our second cohort, the trend was in the same direction, the replication cohort was small, and the association in the combined cohorts remained almost genome-wide significant. Given the biological plausibility of this association, additional studies are warranted. For statin cases, an association with a *STAT4* SNP reached specified significance for the autoimmune-restricted analysis but also did not replicate in our second smaller cohort. Given the large population receiving statin medications and the current controversy about the DILI risk that statins pose [[25](#R25)], this observation also requires additional studies.

It seems unlikely that the lack of replication of tentative associations indicates differences in the characteristics of the discovery and replication samples. The larger proportion of amoxicillin/clavulanate and flucloxacillin-DILI cases in the discovery cohort did not appear to account for the failure to replicate, and other characteristics such as severity of injury, causality assessment, or patient characteristics appeared to be comparable between cohorts. Genotype calls between the GWAS chip and the targeted TaqMan assays were also perfectly concordant for all samples examined. The most likely reason for failure to replicate associations of interest ([Table 4](#T4)) therefore may be that these were simply false associations. This is consistent with the fact none of the variants carried forward to replication achieved genome-wide significance in the discovery cohort, and most of the variants selected on the basis of the ADME-restricted or autoimmune-restricted analysis were only approaching or barely surpassing a more lenient threshold for significance.

Recently reported GWAS have identified common alleles in the MHC region as risk factors for DILI [[7](#R7),[10](#R10),[26](#R26),[27](#R27),[28](#R28)]. If an adaptive immune attack on the liver is the final common event that causes significant liver injury, it is possible that only patients carrying certain MHC alleles are at risk of DILI from a specific drug, that is, ‘upstream’ events (e.g. hepatocyte accumulation of critical reactive metabolites or bile acids) may be necessary but not sufficient to produce DILI. We therefore carried out an exploratory analysis of the association signal in the MHC region compared with autosomal regions outside of the MHC after excluding DILI cases due to flucloxacillin or amoxicillin/clavulanate. Although the *P*-value distribution was lower in the MHC region compared with elsewhere ([Supplementary Fig. S4](#SD1), [http://links.lww.com/FPC/A505](http://links.lww.com/FPC/A505)), after permutation testing, the enrichment of the association signal in the MHC region only approached significance (empirical *P* = 0.015). The observed trend toward enrichment could be due to the extremely high degree of linkage disequilibrium among SNPs in the MHC region and/or an artifact of uncorrected population structure, as low-frequency subpopulation-specific variants in the MHC may not be accounted for by principal components analysis based largely on common variants outside of the MHC. Our analysis therefore does not support the concept of common MHC variation as a general risk factor for DILI, but does not exclude drug-specific or drug subset-specific effects.

On the basis of the power calculations carried out ([Fig. 2](#F2)), we estimate that common variants conferring the risk of a moderate effect size (i.e. RR ≥ 1.5) to all-cause DILI should have been discovered in our study. It is of note that in our initial GWAS analysis of all DILI cases, the MHC signals driven solely by the flucloxacillin and amoxicillin/clavulanate cases were appreciable at the genome-wide significance level ([Fig. 1](#F1)), although these cases account for only 38% of the cohort. As a further example of the ability to detect strong associations despite gross misclassification of phenotype, an analysis that included DILI cases due to flucloxacillin and all other drugs except amoxicillin/clavulanate identified the rs2395209 association (due to flucloxacillin) at the genome-wide significance level, despite the fact that flucloxacillin made up only 13.4% of the total cases ([Fig. 3](#F3)). Individuals carrying the HLA-B*5701 allele have about an 80-fold increased risk of developing DILI when treated with flucloxacillin [[7](#R7)]. Hence, if any combination of individuals comprising a similarly small fraction of our cohort had a common risk factor with comparable effect, we would have detected it in our GWAS analysis. The fact that no other risk variants were detected in the genome-wide analyses is therefore an important negative finding for identifying common SNPs with a very strong effect on DILI risk.

### Fig. 3.

![Fig. 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e8d7/3636716/0570cce2e95e/nihms-446262-f0003.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3636716_nihms-446262-f0003.jpg)

Manhattan (a) and Quantile–quantile (b) plots of genome-wide association study results for the analysis of all drug-induced liver injury (DILI) cases including flucloxacillin, but excluding amoxicillin/clavulanate-induced DILI cases. Despite the fact that flucloxacillin-DILI cases make up only 13.4% of the total case sample (75/560), there remains clear evidence of an HLA-B*5701 (rs2395029) association with DILI risk in the combined sample, suggesting that misclassification or heterogeneity of cases is unlikely to obscure the true DILI risk variants of a very large effect (the HLA-B*5701 allele confers an 80-fold increased risk of DILI from flucloxacillin [7]).

Our negative GWAS findings may suggest that future efforts should focus on collecting and analyzing DILI cases due to single drugs. However, collecting sufficient cases will be a challenge; although the *HLA-B**5701 association with flucloxacillin-DILI (OR~80) would have been detected by GWAS with as few as 12 cases, our data suggest that such strong associations with common variants are unusual. With amoxicillin/clavulanate-DILI, more than 50 cases would be required to demonstrate an HLA association by GWAS (OR~3) [[10](#R10)]. Test and replication cohorts of this size may simply not be feasible for most drugs implicated in the existing registries. If there in fact exist common genetic variants that underlie susceptibility to DILI, it may still be feasible to identify them with GWAS if a sufficient number of DILI cases due to diverse drugs could be ‘clustered’ by a common mechanism or surrogates for mechanism. For example, it may be possible to use chemoinformatic approaches or biological effects (e.g. induction of *STAT4* in cell culture or transport by MRP2) as surrogates for mechanisms to create subgroupings for GWAS.

Our use of population controls deserves comment. As DILI is such a rare event (occurring on average in fewer than one in 10 000 patients), the expected rate of misclassification of control individuals using unselected population controls is very low and should not appreciably impact statistical power to detect a genetic association ([[20](#R20)], [Supplementary Table S39](#SD1), [http://links.lww.com/FPC/A505](http://links.lww.com/FPC/A505)). However, this study design does not allow us to directly test for the effects of environmental risk factors for DILI, such as age, sex, or diagnosis/indication for the precipitant drug(s), among other factors. In particular, polypharmacy (i.e. patients taking a large number of concomitant drugs, even if only one is truly hepatotoxic) has been suggested as a risk factor for DILI [[29](#R29)], perhaps due to competition for, metabolism, or active transport out of hepatocytes or enzyme induction.

Only common genetic variants (predominantly variants with allele frequencies exceeding 5%) were genotyped in our patients. For an event frequency of less than one in 10 000, as is typical for DILI, it is not possible for common variants individually to have highly specific associations with susceptibility. Analytical approaches that could detect the synergistic effects of multiple SNPs are fraught with statistical (multiple comparison) and computational hurdles and such investigations would probably need to be directed by solid data on biological plausibility. It should also be acknowledged that effects not captured by GWAS or genome sequencing are likely to also contribute to DILI risk, including epigenetic or environmental effects as described above. Nonetheless, the availability of the DILI gene banks and the rapidly evolving genomic technology available currently make genetic investigation of DILI a high-priority area of research. Weak but detectable genetic contributions to risk should show important mechanistic pathways whose epigenetic or environmental influences could then be studied specifically.

## Conclusion

GWAS of the largest DILI cohort assembled to date was unable to identify any major genetic risk factors associated with DILI. Restricted analyses showed a replicated association between all-cause hepatocellular injury and *STAT4*, which should be further confirmed. Although the lack of significant GWAS findings does not rule out a highly polygenic model of many variants with a small effect on DILI risk, our data support the idea that very strong associations may be largely drug-specific or reflect rare genetic variations that may only be shown through ongoing advancements in next-generation sequencing. It also seems likely that the maximal yield from genetic interrogation will require parallel experimental approaches to define subgroups of drugs likely to have common mechanisms of DILI or to identify specific biological pathways implicated in DILI.

## Supplementary Material

## Acknowledgements

The authors thank the DILIGEN, EUDRAGENE, Spanish DILI, iSAEC, and DILIN collaborators for their help with case recruitment.

The authors acknowledge use of genotype data from the British 1958 Birth Cohort DNA collection and UK National Blood Service control cohorts. This study makes use of data generated by the Wellcome Trust Case–Control Consortium. A full list of the investigators who contributed to the generation of the data is available from [http://www.wtccc.org.uk](http://www.wtccc.org.uk). Funding for the project was provided by the Wellcome Trust under award 076113 and 085475.

The DILIN network is structured as a U01 cooperative agreement with funds provided by the National Institute of Diabetes and Digestive and Kidney Diseases (NIDDK) under grants: 2U01-DK065176-06 (Duke), 2U01-DK065201-06 (UNC), 2U01-DK065184-06 (Michigan), 2U01-DK065211-06 (Indiana), 5U01DK065193-04 (UConn), 5U01-DK065238-08 (UCSF/CPMC). Additional funding is provided by CTSA grants: UL1 RR025761 (Indiana), UL1 RR025747 (UNC), and UL1 UL1 RR024986 (UMich).

The DILIGEN sample collection was partly funded by the UK Department of Health (Pharmacogenetics Initiative ref PHGX10A), UK NIHR funding to the Nottingham Digestive Diseases Centre, and the iSAEC. The EUDRAGENE collaboration has received support from the EC 5th Framework program (QLRI-CT-2002-02757) and the iSAEC. M.M. has received funding from a UK NIHR postdoctoral award and is an investigator for the FP7 EU-ADR (ICT-215847) and ARITMO (HEALTH-241679) projects on drug safety. The Spanish DILI Registry is partly funded by the Spanish Medicine Agency and has received support from the iSAEC. CIBERehd is funded by Instituto de Salud Carlos III.

The genome-wide genotyping of DILIN cases was funded by the NIDDK under grant 2U01-DK065176-06. Genome-wide genotyping of the DILIGEN, Dundee, EUDRAGENE, Spanish DILI Registry and GSK POPRES controls was funded by iSAEC, with support from Abbott, Daiichi-Sankyo, GSK, J & J, Novartis, Pfizer, Roche, Sanofi-Aventis, Takeda, the Wellcome Trust and Wyeth.

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Ostapowicz G, Fontana RJ, Schiodt FV, Larson A, Davern TJ, Han SH, et al. Results of a prospective study of acute liver failure at 17 tertiary care centers in the United States. Ann Intern Med. 2002;137:947–954. doi: 10.7326/0003-4819-137-12-200212170-00007.  [DOI](https://doi.org/10.7326/0003-4819-137-12-200212170-00007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12484709/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Intern%20Med&title=Results%20of%20a%20prospective%20study%20of%20acute%20liver%20failure%20at%2017%20tertiary%20care%20centers%20in%20the%20United%20States&author=G%20Ostapowicz&author=RJ%20Fontana&author=FV%20Schiodt&author=A%20Larson&author=TJ%20Davern&volume=137&publication_year=2002&pages=947-954&pmid=12484709&doi=10.7326/0003-4819-137-12-200212170-00007&)

2. Watkins PB, Seligman PJ, Pears JS, Avigan MI, Senior JR. Using controlled clinical trials to learn more about acute drug-induced liver injury. Hepatology. 2008;48:1680–1689. doi: 10.1002/hep.22633.  [DOI](https://doi.org/10.1002/hep.22633) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18853438/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hepatology&title=Using%20controlled%20clinical%20trials%20to%20learn%20more%20about%20acute%20drug-induced%20liver%20injury&author=PB%20Watkins&author=PJ%20Seligman&author=JS%20Pears&author=MI%20Avigan&author=JR%20Senior&volume=48&publication_year=2008&pages=1680-1689&pmid=18853438&doi=10.1002/hep.22633&)

3. Andrade RJ, Robles M, Ulzurrun E, Lucena MI. Drug-induced liver injury: insights from genetic studies. Pharmacogenomics. 2009;10:1467–1487. doi: 10.2217/pgs.09.111.  [DOI](https://doi.org/10.2217/pgs.09.111) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19761370/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Drug-induced%20liver%20injury:%20insights%20from%20genetic%20studies&author=RJ%20Andrade&author=M%20Robles&author=E%20Ulzurrun&author=MI%20Lucena&volume=10&publication_year=2009&pages=1467-1487&pmid=19761370&doi=10.2217/pgs.09.111&)

4. Daly AK. Drug-induced liver injury: past, present and future. Pharmacogenomics. 2010;11:607–611. doi: 10.2217/pgs.10.24.  [DOI](https://doi.org/10.2217/pgs.10.24) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20415545/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Drug-induced%20liver%20injury:%20past,%20present%20and%20future&author=AK%20Daly&volume=11&publication_year=2010&pages=607-611&pmid=20415545&doi=10.2217/pgs.10.24&)

5. Lucena MI, Kaplowitz N, Hallal H, Castiella A, Garcia-Bengoechea M, Otazua P, et al. Recurrent drug-induced liver injury (DILI) with different drugs in the Spanish Registry. The dilemma of the relationship to autoimmune hepatitis. J Hepatol. 2011 doi: 10.1016/j.jhep.2010.12.041.  [DOI](https://doi.org/10.1016/j.jhep.2010.12.041) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21338638/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Hepatol&title=Recurrent%20drug-induced%20liver%20injury%20(DILI)%20with%20different%20drugs%20in%20the%20Spanish%20Registry.%20The%20dilemma%20of%20the%20relationship%20to%20autoimmune%20hepatitis&author=MI%20Lucena&author=N%20Kaplowitz&author=H%20Hallal&author=A%20Castiella&author=M%20Garcia-Bengoechea&publication_year=2011&pmid=21338638&doi=10.1016/j.jhep.2010.12.041&)

6. Fontana RJ, Watkins PB, Bonkovsky HL, Chalasani N, Davern T, Serrano J, et al. Drug-Induced Liver Injury Network (DILIN) prospective study: rationale, design and conduct. Drug Saf. 2009;32:55–68. doi: 10.2165/00002018-200932010-00005.  [DOI](https://doi.org/10.2165/00002018-200932010-00005) | [PMC free article](/articles/PMC3637941/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19132805/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Saf&title=Drug-Induced%20Liver%20Injury%20Network%20(DILIN)%20prospective%20study:%20rationale,%20design%20and%20conduct&author=RJ%20Fontana&author=PB%20Watkins&author=HL%20Bonkovsky&author=N%20Chalasani&author=T%20Davern&volume=32&publication_year=2009&pages=55-68&pmid=19132805&doi=10.2165/00002018-200932010-00005&)

7. Daly AK, Donaldson PT, Bhatnagar P, Shen Y, Péer I, Floratos A, et al. HLA-B*5701 genotype is a major determinant of drug-induced liver injury due to flucloxacillin. Nat Genet. 2009;41:816–819. doi: 10.1038/ng.379.  [DOI](https://doi.org/10.1038/ng.379) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19483685/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=HLA-B*5701%20genotype%20is%20a%20major%20determinant%20of%20drug-induced%20liver%20injury%20due%20to%20flucloxacillin&author=AK%20Daly&author=PT%20Donaldson&author=P%20Bhatnagar&author=Y%20Shen&author=I%20P%C3%A9er&volume=41&publication_year=2009&pages=816-819&pmid=19483685&doi=10.1038/ng.379&)

8. Molokhia M, McKeigue P. EUDRAGENE: European collaboration to establish a case-control DNA collection for studying the genetic basis of adverse drug reactions. Pharmacogenomics. 2006;7:633–638. doi: 10.2217/14622416.7.4.633.  [DOI](https://doi.org/10.2217/14622416.7.4.633) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16753010/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=EUDRAGENE:%20European%20collaboration%20to%20establish%20a%20case-control%20DNA%20collection%20for%20studying%20the%20genetic%20basis%20of%20adverse%20drug%20reactions&author=M%20Molokhia&author=P%20McKeigue&volume=7&publication_year=2006&pages=633-638&pmid=16753010&doi=10.2217/14622416.7.4.633&)

9. Andrade RJ, Lucena MI, Fernandez MC, Pelaez G, Pachkoria K, Garcia-Ruiz E, et al. Drug-induced liver injury: an analysis of 461 incidences submitted to the Spanish registry over a 10-year period. Gastroenterology. 2005;129:512–521. doi: 10.1016/j.gastro.2005.05.006.  [DOI](https://doi.org/10.1016/j.gastro.2005.05.006) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16083708/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastroenterology&title=Drug-induced%20liver%20injury:%20an%20analysis%20of%20461%20incidences%20submitted%20to%20the%20Spanish%20registry%20over%20a%2010-year%20period&author=RJ%20Andrade&author=MI%20Lucena&author=MC%20Fernandez&author=G%20Pelaez&author=K%20Pachkoria&volume=129&publication_year=2005&pages=512-521&pmid=16083708&doi=10.1016/j.gastro.2005.05.006&)

10. Lucena MI, Molokhia M, Shen Y, Urban TJ, Aithal GP, Andrade RJ, et al. Susceptibility to amoxicillin-clavulanate-induced liver injury is influenced by multiple HLA class I and II alleles. Gastroenterology. 2011 doi: 10.1053/j.gastro.2011.04.001.  [DOI](https://doi.org/10.1053/j.gastro.2011.04.001) | [PMC free article](/articles/PMC3129430/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21570397/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastroenterology&title=Susceptibility%20to%20amoxicillin-clavulanate-induced%20liver%20injury%20is%20influenced%20by%20multiple%20HLA%20class%20I%20and%20II%20alleles&author=MI%20Lucena&author=M%20Molokhia&author=Y%20Shen&author=TJ%20Urban&author=GP%20Aithal&publication_year=2011&pmid=21570397&doi=10.1053/j.gastro.2011.04.001&)

11. Danan G, Benichou C. Causality assessment of adverse reactions to drugs – I. a novel method based on the conclusions of international consensus meetings: application to drug-induced liver injuries. J Clin Epidemiol. 1993;46:1323–1330. doi: 10.1016/0895-4356(93)90101-6.  [DOI](https://doi.org/10.1016/0895-4356(93)90101-6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8229110/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Epidemiol&title=Causality%20assessment%20of%20adverse%20reactions%20to%20drugs%20%E2%80%93%20I.%20a%20novel%20method%20based%20on%20the%20conclusions%20of%20international%20consensus%20meetings:%20application%20to%20drug-induced%20liver%20injuries&author=G%20Danan&author=C%20Benichou&volume=46&publication_year=1993&pages=1323-1330&pmid=8229110&doi=10.1016/0895-4356(93)90101-6&)

12. Nelson MR, Bryc K, King KS, Indap A, Boyko AR, Novembre J, et al. The Population Reference Sample, POPRES: a resource for population, disease, and pharmacological genetics research. Am J Hum Genet. 2008;83:347–358. doi: 10.1016/j.ajhg.2008.08.005.  [DOI](https://doi.org/10.1016/j.ajhg.2008.08.005) | [PMC free article](/articles/PMC2556436/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18760391/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=The%20Population%20Reference%20Sample,%20POPRES:%20a%20resource%20for%20population,%20disease,%20and%20pharmacological%20genetics%20research&author=MR%20Nelson&author=K%20Bryc&author=KS%20King&author=A%20Indap&author=AR%20Boyko&volume=83&publication_year=2008&pages=347-358&pmid=18760391&doi=10.1016/j.ajhg.2008.08.005&)

13. Ge D, Fellay J, Thompson AJ, Simon JS, Shianna KV, Urban TJ, et al. Genetic variation in IL28B predicts hepatitis C treatment-induced viral clearance. Nature. 2009;461:399–401. doi: 10.1038/nature08309.  [DOI](https://doi.org/10.1038/nature08309) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19684573/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Genetic%20variation%20in%20IL28B%20predicts%20hepatitis%20C%20treatment-induced%20viral%20clearance&author=D%20Ge&author=J%20Fellay&author=AJ%20Thompson&author=JS%20Simon&author=KV%20Shianna&volume=461&publication_year=2009&pages=399-401&pmid=19684573&doi=10.1038/nature08309&)

14. Price AL, Patterson NJ, Plenge RM, Weinblatt ME, Shadick NA, Reich D. Principal components analysis corrects for stratification in genome-wide association studies. Nat Genet. 2006;38:904–909. doi: 10.1038/ng1847.  [DOI](https://doi.org/10.1038/ng1847) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16862161/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Principal%20components%20analysis%20corrects%20for%20stratification%20in%20genome-wide%20association%20studies&author=AL%20Price&author=NJ%20Patterson&author=RM%20Plenge&author=ME%20Weinblatt&author=NA%20Shadick&volume=38&publication_year=2006&pages=904-909&pmid=16862161&doi=10.1038/ng1847&)

15. Purcell S, Neale B, Todd-Brown K, Thomas L, Ferreira MA, Bender D, et al. PLINK: a tool set for whole-genome association and population-based linkage analyses. Am J Hum Genet. 2007;81:559–575. doi: 10.1086/519795.  [DOI](https://doi.org/10.1086/519795) | [PMC free article](/articles/PMC1950838/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17701901/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=PLINK:%20a%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses&author=S%20Purcell&author=B%20Neale&author=K%20Todd-Brown&author=L%20Thomas&author=MA%20Ferreira&volume=81&publication_year=2007&pages=559-575&pmid=17701901&doi=10.1086/519795&)

16. Ge D, Zhang K, Need AC, Martin O, Fellay J, Urban TJ, et al. WGAViewer: software for genomic annotation of whole genome association studies. Genome Res. 2008;18:640–643. doi: 10.1101/gr.071571.107.  [DOI](https://doi.org/10.1101/gr.071571.107) | [PMC free article](/articles/PMC2279251/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18256235/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res&title=WGAViewer:%20software%20for%20genomic%20annotation%20of%20whole%20genome%20association%20studies&author=D%20Ge&author=K%20Zhang&author=AC%20Need&author=O%20Martin&author=J%20Fellay&volume=18&publication_year=2008&pages=640-643&pmid=18256235&doi=10.1101/gr.071571.107&)

17. Ahmadi KR, Weale ME, Xue ZY, Soranzo N, Yarnall DP, Briley JD, et al. A single-nucleotide polymorphism tagging set for human drug metabolism and transport. Nat Genet. 2005;37:84–89. doi: 10.1038/ng1488.  [DOI](https://doi.org/10.1038/ng1488) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15608640/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=A%20single-nucleotide%20polymorphism%20tagging%20set%20for%20human%20drug%20metabolism%20and%20transport&author=KR%20Ahmadi&author=ME%20Weale&author=ZY%20Xue&author=N%20Soranzo&author=DP%20Yarnall&volume=37&publication_year=2005&pages=84-89&pmid=15608640&doi=10.1038/ng1488&)

18. Hindorff LA, Sethupathy P, Junkins HA, Ramos EM, Mehta JP, Collins FS, et al. Potential etiologic and functional implications of genome-wide association loci for human diseases and traits. Proc Natl Acad Sci USA. 2009;106:9362–9367. doi: 10.1073/pnas.0903103106.  [DOI](https://doi.org/10.1073/pnas.0903103106) | [PMC free article](/articles/PMC2687147/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19474294/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20USA&title=Potential%20etiologic%20and%20functional%20implications%20of%20genome-wide%20association%20loci%20for%20human%20diseases%20and%20traits&author=LA%20Hindorff&author=P%20Sethupathy&author=HA%20Junkins&author=EM%20Ramos&author=JP%20Mehta&volume=106&publication_year=2009&pages=9362-9367&pmid=19474294&doi=10.1073/pnas.0903103106&)

19. Menashe I, Rosenberg PS, Chen BE. PGA: power calculator for case-control genetic association analyses. BMC Genet. 2008;9:36. doi: 10.1186/1471-2156-9-36.  [DOI](https://doi.org/10.1186/1471-2156-9-36) | [PMC free article](/articles/PMC2387159/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18477402/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Genet&title=PGA:%20power%20calculator%20for%20case-control%20genetic%20association%20analyses&author=I%20Menashe&author=PS%20Rosenberg&author=BE%20Chen&volume=9&publication_year=2008&pages=36&pmid=18477402&doi=10.1186/1471-2156-9-36&)

20. Edwards BJ, Haynes C, Levenstien MA, Finch SJ, Gordon D. Power and sample size calculations in the presence of phenotype errors for case/control genetic association studies. BMC Genet. 2005;6:18. doi: 10.1186/1471-2156-6-18.  [DOI](https://doi.org/10.1186/1471-2156-6-18) | [PMC free article](/articles/PMC1131899/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15819990/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Genet&title=Power%20and%20sample%20size%20calculations%20in%20the%20presence%20of%20phenotype%20errors%20for%20case/control%20genetic%20association%20studies&author=BJ%20Edwards&author=C%20Haynes&author=MA%20Levenstien&author=SJ%20Finch&author=D%20Gordon&volume=6&publication_year=2005&pages=18&pmid=15819990&doi=10.1186/1471-2156-6-18&)

21. Korman BD, Kastner DL, Gregersen PK, Remmers EF. STAT4: genetics, mechanisms, and implications for autoimmunity. Curr Allergy Asthma Rep. 2008;8:398–403. doi: 10.1007/s11882-008-0077-8.  [DOI](https://doi.org/10.1007/s11882-008-0077-8) | [PMC free article](/articles/PMC2562257/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18682104/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Allergy%20Asthma%20Rep&title=STAT4:%20genetics,%20mechanisms,%20and%20implications%20for%20autoimmunity&author=BD%20Korman&author=DL%20Kastner&author=PK%20Gregersen&author=EF%20Remmers&volume=8&publication_year=2008&pages=398-403&pmid=18682104&doi=10.1007/s11882-008-0077-8&)

22. Adamson DJ, Frew D, Tatoud R, Wolf CR, Palmer CN. Diclofenac antagonizes peroxisome proliferator-activated receptor-gamma signaling. Mol Pharmacol. 2002;61:7–12. doi: 10.1124/mol.61.1.7.  [DOI](https://doi.org/10.1124/mol.61.1.7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11752200/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Pharmacol&title=Diclofenac%20antagonizes%20peroxisome%20proliferator-activated%20receptor-gamma%20signaling&author=DJ%20Adamson&author=D%20Frew&author=R%20Tatoud&author=CR%20Wolf&author=CN%20Palmer&volume=61&publication_year=2002&pages=7-12&pmid=11752200&doi=10.1124/mol.61.1.7&)

23. Kaur J, Sanyal SN. Modulation of inflammatory changes in early stages of colon cancer through activation of PPARgamma by diclofenac. Eur J Cancer Prev. 2010;19:319–327. doi: 10.1097/CEJ.0b013e32833ad385.  [DOI](https://doi.org/10.1097/CEJ.0b013e32833ad385) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20485182/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Cancer%20Prev&title=Modulation%20of%20inflammatory%20changes%20in%20early%20stages%20of%20colon%20cancer%20through%20activation%20of%20PPARgamma%20by%20diclofenac&author=J%20Kaur&author=SN%20Sanyal&volume=19&publication_year=2010&pages=319-327&pmid=20485182&doi=10.1097/CEJ.0b013e32833ad385&)

24. Zieleniak A, Wojcik M, Wozniak LA. Structure and physiological functions of the human peroxisome proliferator-activated receptor gamma. Arch Immunol Ther Exp (Warsz) 2008;56:331–345. doi: 10.1007/s00005-008-0037-y.  [DOI](https://doi.org/10.1007/s00005-008-0037-y) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18836859/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arch%20Immunol%20Ther%20Exp%20(Warsz)&title=Structure%20and%20physiological%20functions%20of%20the%20human%20peroxisome%20proliferator-activated%20receptor%20gamma&author=A%20Zieleniak&author=M%20Wojcik&author=LA%20Wozniak&volume=56&publication_year=2008&pages=331-345&pmid=18836859&doi=10.1007/s00005-008-0037-y&)

25. Bader T. Liver tests are irrelevant when prescribing statins. Lancet. 2010;376:1882–1883. doi: 10.1016/S0140-6736(10)62142-3.  [DOI](https://doi.org/10.1016/S0140-6736(10)62142-3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21109303/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Liver%20tests%20are%20irrelevant%20when%20prescribing%20statins&author=T%20Bader&volume=376&publication_year=2010&pages=1882-1883&pmid=21109303&doi=10.1016/S0140-6736(10)62142-3&)

26. Singer JB, Lewitzky S, Leroy E, Yang F, Zhao X, Klickstein L, et al. A genome-wide study identifies HLA alleles associated with lumiracoxib-related liver injury. Nat Genet. 2010;42:711–714. doi: 10.1038/ng.632.  [DOI](https://doi.org/10.1038/ng.632) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20639878/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=A%20genome-wide%20study%20identifies%20HLA%20alleles%20associated%20with%20lumiracoxib-related%20liver%20injury&author=JB%20Singer&author=S%20Lewitzky&author=E%20Leroy&author=F%20Yang&author=X%20Zhao&volume=42&publication_year=2010&pages=711-714&pmid=20639878&doi=10.1038/ng.632&)

27. Spraggs CF, Budde LR, Briley LP, Bing N, Cox CJ, King KS, et al. HLA-DQA1*02:01 Is a major risk factor for lapatinib-induced hepatotoxicity in women with advanced breast cancer. J Clin Oncol. 2011;29:667–673. doi: 10.1200/JCO.2010.31.3197.  [DOI](https://doi.org/10.1200/JCO.2010.31.3197) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21245432/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=HLA-DQA1*02:01%20Is%20a%20major%20risk%20factor%20for%20lapatinib-induced%20hepatotoxicity%20in%20women%20with%20advanced%20breast%20cancer&author=CF%20Spraggs&author=LR%20Budde&author=LP%20Briley&author=N%20Bing&author=CJ%20Cox&volume=29&publication_year=2011&pages=667-673&pmid=21245432&doi=10.1200/JCO.2010.31.3197&)

28. Kindmark A, Jawaid A, Harbron CG, Barratt BJ, Bengtsson OF, Andersson TB, et al. Genome-wide pharmacogenetic investigation of a hepatic adverse event without clinical signs of immunopathology suggests an underlying immune pathogenesis. Pharmacogenomics J. 2008;8:186–195. doi: 10.1038/sj.tpj.6500458.  [DOI](https://doi.org/10.1038/sj.tpj.6500458) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17505501/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Genome-wide%20pharmacogenetic%20investigation%20of%20a%20hepatic%20adverse%20event%20without%20clinical%20signs%20of%20immunopathology%20suggests%20an%20underlying%20immune%20pathogenesis&author=A%20Kindmark&author=A%20Jawaid&author=CG%20Harbron&author=BJ%20Barratt&author=OF%20Bengtsson&volume=8&publication_year=2008&pages=186-195&pmid=17505501&doi=10.1038/sj.tpj.6500458&)

29. Suzuki A, Yuen N, Walsh J, Papay J, Hunt CM, Diehl AM. Co-medications that modulate liver injury and repair influence clinical outcome of acetaminophen-associated liver injury. Clin Gastroenterol Hepatol. 2009;7:882–888. doi: 10.1016/j.cgh.2009.03.034.  [DOI](https://doi.org/10.1016/j.cgh.2009.03.034) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19362607/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Gastroenterol%20Hepatol&title=Co-medications%20that%20modulate%20liver%20injury%20and%20repair%20influence%20clinical%20outcome%20of%20acetaminophen-associated%20liver%20injury&author=A%20Suzuki&author=N%20Yuen&author=J%20Walsh&author=J%20Papay&author=CM%20Hunt&volume=7&publication_year=2009&pages=882-888&pmid=19362607&doi=10.1016/j.cgh.2009.03.034&)
