# Association between imatinib transporters and metabolizing enzymes genotype and response in newly diagnosed chronic myeloid leukemia patients receiving imatinib therapy

## Metadata
**Authors:** Sabrina Angelini, Simona Soverini, Gloria Ravegnini, Matt Barnett, Eleonora Turrini, Mark Thornquist, Fabrizio Pane, Timothy P Hughes, Deborah L White, Jerald Radich, Dong Wook Kim, Giuseppe Saglio, Daniela Cilloni, Ilaria Iacobucci, Giovanni Perini, Richard Woodman, Giorgio Cantelli-Forti, Michele Baccarani, Patrizia Hrelia, Giovanni Martinelli
**Journal:** Haematologica
**Date:** 2013 Feb
**DOI:** [10.3324/haematol.2012.066480](https://doi.org/10.3324/haematol.2012.066480)
**PMID:** 22875622
**PMCID:** PMC3561425
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3561425/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC3561425/pdf/0980193.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC3561425/pdf/0980193.pdf)

## Abstract

Imatinib has so far been the first-choice treatment in chronic myeloid leukemia with excellent results. However, only a proportion of patients achieve major molecular response – hence the need to find biological predictors of outcome to select the optimal therapeutic strategy now that more potent inhibitors are available. We investigated a panel of 20 polymorphisms in seven genes, potentially associated with the pharmacogenetics of imatinib, in a subset of 189 patients with newly diagnosed chronic myeloid leukemia enrolled in the TOPS trial. The analysis included polymorphisms in the transporters hOCT1, MDR1, ABCG2, OCTN1, and OATP1A2, and in the metabolizing genes CYP3A4 and CYP3A5. In the overall population, the OCTN1 C allele (rs1050152), a simple combination of polymorphisms in the hOCT1 gene and another combination in the genes involved in imatinib uptake were significantly associated with major molecular response. The combination of polymorphisms in imatinib uptake was also significantly associated with complete molecular response. Analyses restricted to Caucasians highlighted the significant association of MDR1 CC (rs60023214) genotype with complete molecular response. We demonstrate the usefulness of a pharmacogenetic approach for stratifying patients with chronic myeloid leukemia according to their likelihood of achieving a major or complete molecular response to imatinib. This represents an attractive opportunity for therapy optimization, worth testing in clinical trials.

## Introduction

The introduction of the Bcr-Abl tyrosine kinase inhibitor imatinib mesylate in the first-line treatment of Philadelphia chromosome-positive (Ph^+^) chronic myeloid leukemia (CML) revolutionized patients' outcome. Given the excellent hematologic and cytogenetic response rates that can be obtained in the chronic phase of the disease,[1](#b1-0980193) the goal of CML therapy has moved to the achievement of a major molecular response (MMR; defined as a 3-log reduction in Bcr-Abl transcript level from a standardized baseline value)[2](#b2-0980193) and possibly a complete molecular response [CMR; defined as at least a 4-log reduction corresponding to undetectable Bcr-Abl transcript by realtime reverse transcription polymerase chain reaction (PCR)],[2](#b2-0980193) which might represent an ‘operational cure’ as well as a prerequisite for discontinuation of imatinib.[3](#b3-0980193)

Research in CML has since focused on finding biological predictors of response allowing for treatment optimization. It is well recognized that interpatient variability in drug response reflects the systemic levels or intracellular concentrations of the drug, known to be associated with the pharmacokinetics (absorption, distribution, and metabolism) of the drug itself.[4](#b4-0980193) Imatinib is metabolized by the cytochrome-P450 - mostly CYP3A4 and 3A5 isoforms.[5](#b5-0980193) The active uptake of imatinib into cells is known to be mediated mainly by the hOCT1 transporter (encoded by the *SLC22A1* gene),[6](#b6-0980193)^-^[7](#b7-0980193) whereas its efflux is mediated by the ABC transporters, in particular ABCB1 (also known as MDR1) and, to a lesser extent, ABCG2.[6](#b6-0980193)^,^[8](#b8-0980193)^-^[10](#b10-0980193) Besides these, other transporters may be important in the absorption, distribution, and elimination of imatinib, including the families of organic cation transporters (OCT) and organic anion transporters (OAT). In particular, a recent study by Hu *et al.*[11](#b11-0980193) identified imatinib as a substrate of OATP1A2 (encoded by *SLCO1A2*), whereas the involvement of members of the OCTN family as imatinib transporters is still uncertain.

Interpatient variability in imatinib metabolism/transport is substantial and thus far unexplained. A possibility is that genetic polymorphisms in genes encoding imatinib-metabolizing enzymes and transporters may influence the extent to which imatinib is delivered to target cells. Accordingly, genetic polymorphisms of the candidate genes *CYP3A4/3A5*, *MDR1*, *ABCG2*, *OATP1A2*, *OCTN1* (encoded by *SLC22A4*) and *hOCT1* could affect expression of corresponding proteins and thus may predict differences in responses to imatinib. To explore this hypothesis, we selected a panel of polymorphisms in these genes and genotyped 189 newly diagnosed CML patients treated with imatinib in the framework of the Tyrosine kinase OPtimization and Selectivity (TOPS) phase III trial.[12](#b12-0980193)

## Design and Methods

### Study population

A total of 189 CML patients receiving imatinib were retrospectively enrolled in this pharmacogenetic study. These patients represented a subset of the population of the TOPS trial - a randomized, phase III study of imatinib 400 mg/day *versus* imatinib 800 mg/day in newly diagnosed and previously untreated CML patients in chronic phase.[12](#b12-0980193) The patients' selection was based exclusively on availability of written informed consent for correlative sub-studies, according to the Helsinki Declaration and its later amendments, and sufficient amount of archived material. The study was approved by all local ethical committees. The patients' characteristics and treatment protocol have been described elsewhere.[12](#b12-0980193) [Table 1](#t1-0980193) summarizes selected demographic characteristics of our study population.

### Table 1.

Table 1 Caption: Patients' and disease characteristics of the study population.

### Evaluation of response to imatinib

Molecular response (MR) was classified based on *BCR-ABL* to control gene transcript ratios, expressed on the International Scale.[13](#b13-0980193) MMR and CMR were defined as ratios ≤0.1% and ≤0.0032%, respectively. Cytogenetic response (CgR) based on bone marrow assessment was classified as complete (CCgR; 0% Ph^+^ cells), partial (PCgR; >0 to 35% Ph^+^ cells), minor (>35 to 65% Ph^+^ cells), minimal (> 65 to 95% Ph^+^ cells), and none (>95 to 100% Ph^+^ cells). A major cytogenetic response (MCgR) was defined as achieving either a CCgR or a PCgR. Time to response (MMR, CMR, MCgR, CCgR) was defined as the interval between the date of randomization and the first date of achieving a response. Patients who progressed to accelerated phase or blast crisis were classified as non-responders at all time points.

### Genotyping analysis

DNA was extracted from cryopreserved white blood cells using conventional methods. The characteristics of the studied polymorphisms (one insertion/deletion and 19 single nucleotide polymorphisms, from here on all referred to as SNP in the text) are reported in [Online Supplementary Table S1](#fn2-0980193). Genotypes were determined by PCR-based assays (restriction fragment length polymorphism and/or real-time) according to published methods,[14](#b14-0980193)^-^[20](#b20-0980193) or as recommended by the manufacturer ([Online Supplementary Table S1](#fn2-0980193)). Positive and negative controls were included in each reaction as quality controls. In addition, the accuracy of genotyping was confirmed by repetition of 100% of the samples. The replicates were 100% concordant.

### Statistical analysis

Genotype distribution – overall and by race – was tested for Hardy-Weinberg equilibrium (HWE) with exact tests.[21](#b21-0980193) The association of individual SNP with MMR, CMR, MCgR, and CCgR was assessed using the exact *P*-value based on a Cochran-Armitage trend test in an additive model with the number of minor alleles (0, 1, or 2) as the predictor. Survival analyses were also used to examine the relationship between SNP and MMR, CMR, MCgR, and CCgR. Time to response was defined as the interval between randomization and the date of first confirmed response for any of the given response variables under study. The follow-up time was censored at the date of last assessment for those not achieving a response. Hazard ratios were estimated with Cox proportional hazards models and evaluated with likelihood ratio tests. Dose of imatinib was included as a covariate in the models to adjust for possible differential response in the two imatinib dose groups. Statistically significant associations observed in the Cox regression analyses were further evaluated through plots based on the Kaplan-Meier method. The Kaplan-Meier curves were compared with log-rank tests of trend, assuming an additive genetic model. Haplotype frequencies were estimated using the expectation-maximization algorithm. Linkage disequilibrium between pairs of SP was estimated with Lewontin's D′, and likelihood ratio tests were performed to assess linkage disequilibrium among multiple loci. Empirical *P*-values obtained from permutation tests (*n* = 10,000 permutations) were used to evaluate the association between haplotypes and imatinib response at 12 months. All statistical tests were two-sided, with statistical significance defined as *P*<0.05. No adjustments were made for multiple comparisons. Statistical analyses were conducted using SAS version 9.1 (SAS Institute Inc., Cary, NC, USA).

The primary purpose of this study was to evaluate the effect of individual SNP on response to imatinib therapy. As an exploratory analysis, we also examined simple summary measures of functionally related (uptake and efflux) SNP. These were defined as the number of alleles, across the genes of interest, hypothesized or found in this study to be associated with a favorable response. According to the results of *in vitro* studies reported in the literature, we hypothesized a favorable response in the presence of *hOCT1* rs4646277, rs4646278, and rs2282143, *MDR1* rs1128503 (also known as *C1236T*), rs60023214 (also known as *C3435T*), rs1128501, and rs10245483 and *OATP1A2* rs11568563 major alleles and *ABCG2* rs2231137 and rs2231142 variant alleles.[21](#b21-0980193) No functional knowledge was available for the remaining SNP and the major allele was thus hypothesized to be associated with favorable response unless the present study suggested a different relationship.

## Results

### Genotype distribution

Genotype distributions of the 20 candidate SNP are summarized in [Online Supplementary Table S2A-C](#fn2-0980193). Five SNP (*hOCT1*: rs4646277, rs4646278; *MDR1*: rs1128501; *CYP3A4*: rs28371759 and *CYP3A5*: rs28365083) were homozygous for the major allele for all patients and were excluded from the analyses. The comparison of genotype frequencies according to the patients' demographic characteristics revealed differences between Asian and Caucasian subgroups ([Online Supplementary Table S2A, B](#fn2-0980193)). For this reason analyses of associations of SNP with response restricted to Caucasians were performed in addition to analyses based on all patients. A deviation from the HWE was observed for a single SNP (*MDR1* rs10245483) in the Caucasian population; no departures from the HWE were observed for any other SNP, either in the overall population or in Caucasians. When restricting the analysis to the Caucasian population, the genotype distributions of *MDR1* rs60023214 and rs2032582 (known as G2677T/A), *CYP3A4* rs2740574 and *CYP3A5* rs776746 differed significantly by geographical region. However, none of the four genotypes in the overall Caucasian group showed deviation from the HWE.

### Outcomes of imatinib therapy

Treatment outcomes for all patients and only the Caucasians are presented in [Figure 1](#f1-0980193). With a median total follow-up of 29 months among the entire study population, the 12-month post-randomization cumulative incidences of MMR, CMR, MCgR and CCgR were 67% (95% CI, 60%-73%), 9% (5%-14%), 82% (75%-87%), and 76% (70%-83%), respectively. Response rates in the Caucasian subgroup were similar.

### Figure 1.

![Figure 1.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5fb2/3561425/e74065753656/98193.fig1.jpg)

Rates of cytogenetic and molecular response to imatinib in the study population. Cumulative incidence of patients achieving major cytogenetic response (MCgR), complete cytogenetic response (CCgR), major molecular response (MMR) and complete molecular response (CMR) in the overall population (left panel) and in Caucasians (right panel).

### Outcomes of imatinib therapy according to candidate genotypes

Treatment outcomes were compared according to: (i) each candidate genotype, (ii) summary measures based on combinations of SNP in the same gene, and (iii) summary measures based on combinations of SNP in functionally related (uptake and efflux) genes. Results of associations with CgR and MR, adjusted for imatinib dose, among all patients are grouped in [Online Supplementary Table S3](#fn2-0980193). All the most relevant results estimated with Cox proportional hazard models and evaluated with likelihood ratio tests are presented in [Table 2](#t2-0980193).

### Table 2.

Table 2 Caption: Most relevant correlations between treatment outcomes and candidate genotypes or SNP combinations in the overall population.

As far as MR was concerned, presence of the C allele in *OCTN1* (rs1050152) had a favorable impact on MMR achievement (*P*=0.03). Nearly significant associations were observed for the presence of *MDR1* A (rs3213619) and *MDR1* C (rs60023214) alleles with achievement of CMR (*P*=0.07 and *P*=0.06 respectively). With respect to the summary measures, a combination of SNP in the *hOCT1* gene (see [Online Supplementary Table S4](#fn2-0980193) for combination design and distribution in the study population) was significantly correlated with MMR (*P*=0.03). The same combination had a weak effect on CMR achievement (*P*=0.08). When considering summary measures of uptake (see [Online Supplementary Table S5](#fn2-0980193) for combination design and distribution in the study population) and efflux, only the former was associated with both MMR and CMR (*P*=0.004 and *P*=0.01, respectively).

As far as CgR was concerned, *hOCT1* C (rs2282143) was significantly associated with MCgR (*P*=0.05); nearly significant associations were observed for the presence of *MDR1* A (rs3213619) and *OCTN1* C (rs1050152) alleles with achievement of MCgR (*P*=0.06 for both).

We also carried out analyses limited to Caucasian patients, our most representative sample set. All the results are presented in [Online Supplementary Table S6](#fn2-0980193), while the most relevant ones are grouped in [Table 3](#t3-0980193). The analyses yielded results similar to those observed in the overall population. In particular, the presence of *OCTN1* C (rs1050152) allele had a significantly favorable impact on achievement of MMR and a weak effect on CMR (*P*=0.03 and *P*=0.07, respectively). A combination of SNP in *hOCT1* ([Online Supplementary Table S4](#fn2-0980193)) was again associated with achievement of MMR (*P*=0.05). Similarly, achievement of MMR and CMR is associated with an increasing number of favorable alleles in genes functionally related to imatinib uptake ([Table 3](#t3-0980193); *P*=0.01 for both). In addition to these results, the analyses in the Caucasian subgroup pointed out the influence of two additional SNP on the achievement of MR. The CC genotype in *MDR1* (rs60023214) was significantly associated with MMR achievement (*P*=0.005), and a similar tendency was observed for the achievement of CMR (*P*=0.09). A nearly significant association between *CYP3A4* GG genotype (rs270574) and MMR achievement was also observed (*P*=0.07). None of the analyzed SNP or combination of SNP was associated with MCgR or CCgR.

### Table 3.

Table 3 Caption: Most relevant correlations between treatment outcomes and candidate genotypes or SNP combinations among Caucasians.

Plots of the estimated fraction of patients achieving response by time since treatment initiation based on the Kaplan-Meier method were analyzed in the overall population and separately for Caucasians. Presence of the minor allele in *OCTN1* (TT rs1050152), was associated with decreased rate of MMR in the overall population and Caucasians (*P*=0.028 and *P*=0.027, respectively; [Figure 2](#f2-0980193)). Results were similar for CMR, and in general the curves separated at later times compared to those for MMR ([Figure 2](#f2-0980193)). Similarly, the combination of favorable alleles in the genes involved in imatinib uptake was associated with increased rates of both MMR and CMR (*P*=0.004 and *P*=0.015, respectively, in the overall population, and *P*=0.005 and *P*=0.009, respectively, in Caucasians). In addition, an increase in the number of favorable alleles in the *hOCT1* gene was associated with an increased MMR rate (*P*=0.030 and *P*=0.043 in the overall population and Caucasians, respectively).

### Figure 2.

![Figure 2.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5fb2/3561425/4a6f076efad4/98193.fig2.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3561425_98193.fig2.jpg)

Molecular response by OCTN1 genotype. Cumulative incidence of patients achieving MMR and CMR according to OCTN1 rs1050152 genotype in the overall population (left panels) and in Caucasians (right panels). Given that the major allele is C and the minor allele is T, CC and CT carriers had significantly higher MMR and CMR rates with respect to TT carriers.

In addition to these results, the analyses in the Caucasian subgroup confirmed the influence of two additional SNP on MR. The major allele in *MDR1* (CCrs60023214) was significantly associated with a higher rate of CMR and had a weak effect on MMR (*P*=0.005 and *P*=0.081, respectively; [Figure 3A](#f3-0980193)). The major allele in CYP3A4 (AA rs2740574) was associated with a decreased rate of MMR (*P*=0.038; [Figure 3B](#f3-0980193)).

### Figure 3.

![Figure 3.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5fb2/3561425/0f7977dd54f7/98193.fig3.jpg)

Molecular response in Caucasians according to MDR1 and CYP3A4 genotypes. (A) Cumulative incidence of patients achieving CMR according to MDR1 rs60023214 genotype. Given that for rs60023214 the major allele is C and the minor allele is T, CC carriers had a significantly higher CMR rate with respect to CT and TT carriers. (B) Cumulative incidence of patients achieving MMR according to CYP3A4 rs2740574 genotype. Given that for rs2740574 the major allele is A and the minor allele is G, AA carriers had a significantly higher MMR rate with respect to AG carriers (no GG carriers were detected in our study population).

As previously reported, the three *MDR1* polymorphisms rs1129503 (C1236T), rs2032582 (G2677T/A), and rs60023214 (C3435T), were found to be in strong linkage disequilibrium (rs1129503 and rs2032582: D′=0.76; rs1129503 and rs60023214: D′=0.51; rs2032582 and rs60023214: D′=0.61; test for linkage disequilibrium among the three markers *P*<0.0001). However none of the resulting haplotypes (see [Online Supplementary Table S7](#fn2-0980193) for haplotype frequencies) was associated with response to imatinib.

Analogous analyses were performed for loss of MMR, MCgR, and CCgR. However, the loss of response rates was quite low and no suggestion of association could be detected.

Analyses were also performed for evidence of differences by drug dose (imatinib 400 mg *versus* 800 mg), but no statistically significant associations were observed.

## Discussion

The spectrum of therapeutic options for CML patients has recently been enriched by second-generation tyrosine kinase inhibitors that are more potent and/or more selective than imatinib in Bcr-Abl inhibition. Nilotinib and dasatinib had already demonstrated remarkable efficacy in imatinib-resistant cases[22](#b22-0980193)^,^[23](#b23-0980193) and phase II and III studies in newly diagnosed patients have shown even higher rates of responses, obtained earlier during therapy, than those achievable with imatinib.[24](#b24-0980193)^-^[27](#b27-0980193) Paradoxically, the availability of multiple therapeutic options is not paralleled by the availability of biological predictors of outcome enabling identification, at the time of diagnosis, of those patients who are more likely to benefit from nilotinib or dasatinib rather than from imatinib - hence there is still a need for CML treatment optimization. To this end, a refined stratification of patients in terms of likelihood of achieving MMR, or even better CMR, would be the necessary starting point. Despite intensive research efforts, for years the Sokal score has been the only reliable factor for risk prediction at the time of diagnosis. Only recently has pretherapy hOCT1 transporter activity been demonstrated to correlate with MMR and to predict for long-term risk of resistance to imatinib.[28](#b28-0980193)^,^[29](#b29-0980193)

In several pathological conditions, pharmacogenetics has proven to be a potential source of biomarkers given the known influence of polymorphisms in key genes encoding drug transporters and metabolizing enzymes on intracellular drug delivery and, therefore, on the effectiveness of the drugs. In CML, only three studies had so far explored this field, but they all suffered from the limitation of being conducted in heterogeneous populations including patients at different stages of disease, not all treated with imatinib first-line[30](#b30-0980193)^,^[31](#b31-0980193) and of having limited sample sizes.[32](#b32-0980193)^,^[33](#b33-0980193)

In the present study, we investigated a panel of polymorphisms that can be hypothesized to influence imatinib transport and metabolism in a large series of newly diagnosed, previously untreated CML patients receiving imatinib in the framework of the TOPS phase III trial. This is a worldwide trial that enrolled patients of different ethnic origin – which brings benefit to the search for common genetic traits of response. We found, using a multiple candidate gene-SNP approach, that there are genes in the imatinib metabolism and transport pathways that are associated with the degree of MR to imatinib therapy. Examining multiple candidate genes (20 SNP in 7 genes associated with imatinib metabolism and transport) also enables the potential effects of gene-gene interactions to be assessed.

In this study, a combination of SNP in *hOCT1* was significantly associated with MMR, regardless of ethnicity. This finding is biologically plausible, especially in the light of the fact that all the studied variants are in the coding region and lead to an amino acid change, assigning a putative functional role to these SNP. It has been demonstrated that hOCT1 activity is an important determinant of MR to imatinib in CML patients.[28](#b28-0980193)^,^[34](#b34-0980193) It is, therefore, reasonable to expect that a combination of SNP may influence hOCT1 expression and/or activity, thus playing a role in the achievement of MMR. It remains an open question whether *hOCT1* genotypes determine the levels of hOCT1 mRNA transcript or alter the transporter activity/selectivity – thus further studies are warranted in order to elucidate the correlation of *hOCT1* genotypes and activity. Another combination of SNP, all non-synonymous - in genes involved in imatinib uptake (*hOCT1*, *OCTN1*, *OATP1A2*) - was even more significantly associated with both MMR and CMR, regardless of ethnicity. This finding, also biologically plausible, shows the importance of individual pharmacokinetic differences for the response to imatinib therapy. Indeed, interindividual variation in imatinib uptake can be considered as a complex phenotypic trait, which can most probably be explained as a complex genotypic feature (i.e. one SNP *versus* a combination of SNP). However, little or no functional data are available in the literature for these SNP, so we can only hypothesize an effect on imatinib pharmacokinetics based on the fact that they all correspond to an amino acid variation in the protein sequence. An interesting finding was that the *OCTN1* rs1050152-C allele was significantly associated with MMR in both the overall population and Caucasians. As far as we are aware, none of the pharmacokinetic studies of imatinib has pointed out an involvement of the OCTN1 protein in imatinib transport; however, it cannot be excluded that this transporter is directly or indirectly involved in imatinib uptake. It is also possible that this association occurred by chance - making it essential to replicate the observation in an independent dataset, as well as to carry out further *in vitro* studies assessing what role OCTN1 may play in the cellular uptake of imatinib.

Our analyses in the Caucasian population also ascribed a role to an *MDR1* polymorphism in the achievement of CMR, confirming that this transporter may be a critical determinant of intracellular imatinib levels. Several pharmacogenetic association studies have focused on *MDR1* variants. In particular the role of three variants: rs1128503, rs60023214, and rs2032582 – better known from published studies as C1236T, C3435T and G2677T/A – has been extensively studied, with strikingly contrasting results. Dulucq *et al.*[30](#b30-0980193) were the first to report an association between *MDR1* rs2032582 (TT), rs1128503 (TT/TA) and higher MMR rates. They also found that the haplotype rs1128503 C, rs2032582 G, rs60023214 C was statistically linked to less frequent MMR. A subsequent study by Kim *et al.* could not replicate any of these observations – no association emerged between any of the *MDR1* gene polymorphisms analyzed and clinical outcome; however, no haplotype was assessed.[31](#b31-0980193) More recently, Ni *et al.*[32](#b32-0980193) reported that the rate of resistance to imatinib was higher in patients carrying the rs1128503 TT variant. Maffioli *et al.*[33](#b33-0980193) reported a protective effect from primary failure for the rs2032582 T allele, however the rs1128503, rs2032582, rs60023214 TGC haplotype - differing only by one allele from the one reported by Dulucq *et al.*[30](#b30-0980193) as favorable - was more frequently found in patients with primary resistance to imatinib.

In our data set, *MDR1* haplotypes were not associated with imatinib response. On the other hand, we found a statistically significant correlation between *MDR1* rs60023214 and CMR. Functional studies[35](#b35-0980193)^,^[36](#b36-0980193) have shown a key role for rs60023214 in the altered expression and *in vivo* activity of P-glycoprotein, product of the *MDR1* gene. However, our data showed a significantly higher CMR rate in carriers of the CC genotype than in carriers of the CT/TT genotypes, while results of these studies – the T allele has been reported to be associated with lower transcript levels as compared with the C allele – would suggest the opposite. Our observations are in agreement with the findings of Ni *et al.*[32](#b32-0980193) who reported that the rate of resistance was higher in CT/TT genotype carriers than in CC genotype carriers, and in disagreement with Maffioli *et al.*[33](#b33-0980193) In this latter study, the CC genotype was associated with a higher probability of primary treatment failure, but the results need to be considered with caution as only four patients experienced primary treatment failure. Another critical issue to bear in mind is that definitions of MR as well as of resistance to imatinib in CML have only recently been standardized; this unfortunately represents a confounding element, actually making it impossible to compare results in terms of associations across different pharmacogenetic studies. However, taken together these results suggest that MDR1 is, directly or indirectly, an important determinant of intracellular imatinib concentration.

Another interesting finding, restricted to the Caucasian group, is the significant association between the *CYP3A4* rs270574-GG minor allele and increased MMR rate. Cytochrome P450 isoform 3A4 is generally regarded as the major enzyme responsible for the metabolism of imatinib.[37](#b37-0980193)^,^[38](#b38-0980193) The rs270574 SNP is located in the regulatory region of the gene and might, therefore, alter transcriptional activity, ultimately influencing enzyme expression. To date, there have been no studies addressing the functional implications of this specific polymorphism *in vivo*; however, *in vitro* experiments have suggested that the variant allele may be associated with higher CYP3A4 expression.[39](#b39-0980193)^,^[40](#b40-0980193) Pharmacokinetic and pharmacodynamic assessments in the phase I study of imatinib in CML proposed that variability in CYP3A4 activity might, at least in part, account for the observed interpatient variability in drug exposure and that co-administration of drugs inducing CYP3A4 could increase imatinib metabolism and lead to low/ineffective imatinib plasma levels.[41](#b41-0980193) One would thus expect lower response rates to be associated with the rs270574 variant – which actually contrasts with our findings. On the other hand, a recent study in CML investigating the influence of *in vivo* CYP3A4 activity on the achievement of MR to imatinib found that higher activity significantly correlated with higher CMR rates at 12 months.[42](#b42-0980193) This observation, in line with our findings, might have a rationale in the fact that the main imatinib metabolite formed by CYP3A4, the N-desmethylated piperazine derivative ([CGP74588](https://www.ncbi.nlm.nih.gov/protein/CGP74588)), is pharmacologically active, with a potency and selectivity similar to those of imatinib,[37](#b37-0980193)^,^[38](#b38-0980193) and longer terminal half-life.[43](#b43-0980193)^,^[44](#b44-0980193) It may be hypothesized that the presence of the variant allele leads to a larger amount of [CGP74588](https://www.ncbi.nlm.nih.gov/protein/CGP74588). Another intriguing hypothesis is that [CGP74588](https://www.ncbi.nlm.nih.gov/protein/CGP74588) can be subjected to a different transport process – which would be in line with the observation of higher cellular uptake of [CGP74588](https://www.ncbi.nlm.nih.gov/protein/CGP74588) as compared to imatinib made by le Coutre *et al.*[44](#b44-0980193) Obviously no definitive explanation can be given and further studies on the pharmacokinetics of [CGP74588](https://www.ncbi.nlm.nih.gov/protein/CGP74588) are needed.

The study of combinations of SNP to assess response to imatinib therapy provides only a preliminary examination of the potential for creation of a panel of markers to drive treatment decisions. Creation of such a panel of markers will require larger sample sizes for development followed by testing in an independent set.

In conclusion, this study showed that genotyping should be taken into account in CML patients in an attempt to individualize treatment further, with the aim of enhancing efficacy in terms of achievement of MMR and CMR. On the basis of these findings, stratification of patients according to genotypes may be proposed for selection between imatinib and second-generation tyrosine kinase inhibitors, and represents an attractive opportunity for new clinical trials.

## Acknowledgments

**Funding**: This work was supported by Novartis Oncology, Clinical Development, TOPS Correlative Studies Network, the University of Bologna (“Progetti strategici”), and the Ministry of Education, University and Research of Italy (MIUR, grant number 2009TNXL9P to SA). The funders had no role in the study design, data collection and analysis, decision to publish, or preparation of the manuscript.

## Footnotes

## References

1. Druker BJ, Guilhot F, O'Brien SG, Gathmann I, Kantarjian H, Gattermann N, et al. Five-year follow-up of patients receiving imatinib for chronic myeloid leukemia. N Engl J Med. 2006;355(23):2408-17  [DOI](https://doi.org/10.1056/NEJMoa062867) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17151364/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Five-year%20follow-up%20of%20patients%20receiving%20imatinib%20for%20chronic%20myeloid%20leukemia&author=BJ%20Druker&author=F%20Guilhot&author=SG%20O'Brien&author=I%20Gathmann&author=H%20Kantarjian&volume=355&issue=23&publication_year=2006&pages=2408-17&pmid=17151364&doi=10.1056/NEJMoa062867&)

2. Hughes TP, Branford S. Monitoring disease response to tyrosine kinase inhibitor therapy in CML. Hematology Am Soc Hematol Educ Program. 2009;477-87  [DOI](https://doi.org/10.1182/asheducation-2009.1.477) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20008233/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hematology%20Am%20Soc%20Hematol%20Educ%20Program&title=Monitoring%20disease%20response%20to%20tyrosine%20kinase%20inhibitor%20therapy%20in%20CML&author=TP%20Hughes&author=S%20Branford&publication_year=2009&pages=477-87&pmid=20008233&doi=10.1182/asheducation-2009.1.477&)

3. Rousselot P, Huguet F, Rea D, Legros L, Cayuela JM, Maarek O, et al. Imatinib mesylate discontinuation in patients with chronic myelogenous leukemia in complete molecular remission for more than 2 years. Blood. 2007;109(1):58-60  [DOI](https://doi.org/10.1182/blood-2006-03-011239) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16973963/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Imatinib%20mesylate%20discontinuation%20in%20patients%20with%20chronic%20myelogenous%20leukemia%20in%20complete%20molecular%20remission%20for%20more%20than%202%20years&author=P%20Rousselot&author=F%20Huguet&author=D%20Rea&author=L%20Legros&author=JM%20Cayuela&volume=109&issue=1&publication_year=2007&pages=58-60&pmid=16973963&doi=10.1182/blood-2006-03-011239&)

4. Haiat S, Decleves X, Mittaine B, Legrand O, Francart S, Rio B, et al. Determination of imatinib plasma levels by high performance liquid chromatography (HPLC): evaluation of pharmacokinetic variability and haematological consequences. Blood. 2006;108 Abstract 1375.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Determination%20of%20imatinib%20plasma%20levels%20by%20high%20performance%20liquid%20chromatography%20(HPLC):%20evaluation%20of%20pharmacokinetic%20variability%20and%20haematological%20consequences&author=S%20Haiat&author=X%20Decleves&author=B%20Mittaine&author=O%20Legrand&author=S%20Francart&volume=108&publication_year=2006&)

5. Peng B, Lloyd P, Schran H. Clinical pharmacokinetics of imatinib. Clin Pharmacokinet. 2005;44(9):879-94  [DOI](https://doi.org/10.2165/00003088-200544090-00001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16122278/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacokinet&title=Clinical%20pharmacokinetics%20of%20imatinib&author=B%20Peng&author=P%20Lloyd&author=H%20Schran&volume=44&issue=9&publication_year=2005&pages=879-94&pmid=16122278&doi=10.2165/00003088-200544090-00001&)

6. Thomas J, Wang L, Clark RE, Pirmohamed M. Active transport of imatinib into and out of cells: implications for drug resistance. Blood. 2004;104(12):3739-45  [DOI](https://doi.org/10.1182/blood-2003-12-4276) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15315971/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Active%20transport%20of%20imatinib%20into%20and%20out%20of%20cells:%20implications%20for%20drug%20resistance&author=J%20Thomas&author=L%20Wang&author=RE%20Clark&author=M%20Pirmohamed&volume=104&issue=12&publication_year=2004&pages=3739-45&pmid=15315971&doi=10.1182/blood-2003-12-4276&)

7. White DL, Saunders VA, Dang P, Engler J, Zannettino AC, cambareri AC, et al. OCT-1-mediated influx is a key determinant of the intracellular uptake of imatinib but not nilotinib (AMN107): reduced OCT-1 activity is the cause of low in vitro sensitivity to imatinib. Blood. 2006;108(2):697-704  [DOI](https://doi.org/10.1182/blood-2005-11-4687) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16597591/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=OCT-1-mediated%20influx%20is%20a%20key%20determinant%20of%20the%20intracellular%20uptake%20of%20imatinib%20but%20not%20nilotinib%20(AMN107):%20reduced%20OCT-1%20activity%20is%20the%20cause%20of%20low%20in%20vitro%20sensitivity%20to%20imatinib&author=DL%20White&author=VA%20Saunders&author=P%20Dang&author=J%20Engler&author=AC%20Zannettino&volume=108&issue=2&publication_year=2006&pages=697-704&pmid=16597591&doi=10.1182/blood-2005-11-4687&)

8. Mahon FX, Belloc F, Lagarde V, Chollet C, Moreau-Gaudry F, Reiffers J, et al. MDR1 gene overexpression confers resistance to imatinib mesylate in leukemia cell line models. Blood. 2003;101(6):2368-73  [DOI](https://doi.org/10.1182/blood.V101.6.2368) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12609962/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=MDR1%20gene%20overexpression%20confers%20resistance%20to%20imatinib%20mesylate%20in%20leukemia%20cell%20line%20models&author=FX%20Mahon&author=F%20Belloc&author=V%20Lagarde&author=C%20Chollet&author=F%20Moreau-Gaudry&volume=101&issue=6&publication_year=2003&pages=2368-73&pmid=12609962&doi=10.1182/blood.V101.6.2368&)

9. Illmer T, Schaich M, Platzbecker U, Freiberg-Richter J, Oelschlägel U, von Bonin M, et al. P-glycoprotein-mediated drug efflux is a resistance mechanism of chronic myelogenous leukemia cells to treatment with imatinib mesylate. Leukemia. 2004;18(3):401-8  [DOI](https://doi.org/10.1038/sj.leu.2403257) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14724652/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Leukemia&title=P-glycoprotein-mediated%20drug%20efflux%20is%20a%20resistance%20mechanism%20of%20chronic%20myelogenous%20leukemia%20cells%20to%20treatment%20with%20imatinib%20mesylate&author=T%20Illmer&author=M%20Schaich&author=U%20Platzbecker&author=J%20Freiberg-Richter&author=U%20Oelschl%C3%A4gel&volume=18&issue=3&publication_year=2004&pages=401-8&pmid=14724652&doi=10.1038/sj.leu.2403257&)

10. Burger H, Nooter K. Pharmacokinetic resistance to imatinib mesylate: role of the ABC drug pumps ABCG2 (BCRP) and ABCB1 (MDR1) in the oral bioavailability of imatinib. Cell Cycle. 2004;3(12):1502-5  [DOI](https://doi.org/10.4161/cc.3.12.1331) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15611623/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Cycle&title=Pharmacokinetic%20resistance%20to%20imatinib%20mesylate:%20role%20of%20the%20ABC%20drug%20pumps%20ABCG2%20(BCRP)%20and%20ABCB1%20(MDR1)%20in%20the%20oral%20bioavailability%20of%20imatinib&author=H%20Burger&author=K%20Nooter&volume=3&issue=12&publication_year=2004&pages=1502-5&pmid=15611623&doi=10.4161/cc.3.12.1331&)

11. Hu S, Franke RM, Filipski KK, Hu C, Orwick SJ, de Bruijn EA, et al. Interaction of imatinib with human organic ion carriers. Clin Cancer Res. 2008;14(10):3141-8  [DOI](https://doi.org/10.1158/1078-0432.CCR-07-4913) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18483382/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=Interaction%20of%20imatinib%20with%20human%20organic%20ion%20carriers&author=S%20Hu&author=RM%20Franke&author=KK%20Filipski&author=C%20Hu&author=SJ%20Orwick&volume=14&issue=10&publication_year=2008&pages=3141-8&pmid=18483382&doi=10.1158/1078-0432.CCR-07-4913&)

12. Cortes JE, Baccarani M, Guilhot F, Druker BJ, Branford S, kim DW, et al. Phase III, randomized, open-label study of daily imatinib mesylate 400 mg versus 800 mg in patients with newly diagnosed, previously untreated chronic myeloid leukemia in chronic phase using molecular end points: tyrosine kinase inhibitor optimization and selectivity study. J Clin Oncol. 2010;28(3):424-30  [DOI](https://doi.org/10.1200/JCO.2009.25.3724) | [PMC free article](/articles/PMC4979244/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20008622/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Phase%20III,%20randomized,%20open-label%20study%20of%20daily%20imatinib%20mesylate%20400%20mg%20versus%20800%20mg%20in%20patients%20with%20newly%20diagnosed,%20previously%20untreated%20chronic%20myeloid%20leukemia%20in%20chronic%20phase%20using%20molecular%20end%20points:%20tyrosine%20kinase%20inhibitor%20optimization%20and%20selectivity%20study&author=JE%20Cortes&author=M%20Baccarani&author=F%20Guilhot&author=BJ%20Druker&author=S%20Branford&volume=28&issue=3&publication_year=2010&pages=424-30&pmid=20008622&doi=10.1200/JCO.2009.25.3724&)

13. Hughes T, Deininger M, Hochhaus A, Branford S, Radich J, Kaeda J, et al. Monitoring CML patients responding to treatment with tyrosine kinase inhibitors: review and recommendation for harmonizing current methodology for detecting BCR-ABL transcripts and kinase domain mutations and for expressing results. Blood. 2006;108(1):28-37  [DOI](https://doi.org/10.1182/blood-2006-01-0092) | [PMC free article](/articles/PMC1895821/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16522812/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Monitoring%20CML%20patients%20responding%20to%20treatment%20with%20tyrosine%20kinase%20inhibitors:%20review%20and%20recommendation%20for%20harmonizing%20current%20methodology%20for%20detecting%20BCR-ABL%20transcripts%20and%20kinase%20domain%20mutations%20and%20for%20expressing%20results&author=T%20Hughes&author=M%20Deininger&author=A%20Hochhaus&author=S%20Branford&author=J%20Radich&volume=108&issue=1&publication_year=2006&pages=28-37&pmid=16522812&doi=10.1182/blood-2006-01-0092&)

14. Goreva OB, Grishanova AY, Domnikova NP, Mukhin OV, Lyakhovich W. MDR1 gene C1236T and C6+139T polymorphisms in the Russian population: associations with predisposition to lymphoproliferative diseases and drug resistance. Bull Exp Biol Med. 2004;138(4):404-6  [DOI](https://doi.org/10.1007/s10517-005-0054-3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15665957/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bull%20Exp%20Biol%20Med&title=MDR1%20gene%20C1236T%20and%20C6+139T%20polymorphisms%20in%20the%20Russian%20population:%20associations%20with%20predisposition%20to%20lymphoproliferative%20diseases%20and%20drug%20resistance&author=OB%20Goreva&author=AY%20Grishanova&author=NP%20Domnikova&author=OV%20Mukhin&author=W%20Lyakhovich&volume=138&issue=4&publication_year=2004&pages=404-6&pmid=15665957&doi=10.1007/s10517-005-0054-3&)

15. Jamroziak K, Młynarski W, Balcerczak E, Mistygacz M, Trelinska J, Mirowski M, et al. Functional C3435T polymorphism of MDR1 gene: an impact on genetic susceptibility and clinical outcome of childhood acute lymphoblastic leukemia. Eur J Haematol. 2004;72(5):314-21  [DOI](https://doi.org/10.1111/j.1600-0609.2004.00228.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15059065/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Haematol&title=Functional%20C3435T%20polymorphism%20of%20MDR1%20gene:%20an%20impact%20on%20genetic%20susceptibility%20and%20clinical%20outcome%20of%20childhood%20acute%20lymphoblastic%20leukemia&author=K%20Jamroziak&author=W%20M%C5%82ynarski&author=E%20Balcerczak&author=M%20Mistygacz&author=J%20Trelinska&volume=72&issue=5&publication_year=2004&pages=314-21&pmid=15059065&doi=10.1111/j.1600-0609.2004.00228.x&)

16. Kurzawski M, Pawlik A, Gornik W, DroŸdzik M. Frequency of common MDR1 gene variants in Polish population. Pharmacol Rep. 2006;58(1):35-40  [PubMed](https://pubmed.ncbi.nlm.nih.gov/16531628/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacol%20Rep&title=Frequency%20of%20common%20MDR1%20gene%20variants%20in%20Polish%20population&author=M%20Kurzawski&author=A%20Pawlik&author=W%20Gornik&author=M%20Dro%C5%B8dzik&volume=58&issue=1&publication_year=2006&pages=35-40&pmid=16531628&)

17. Hu LL, Wang XX, Chen X, Chang J, Li C, Zhang Y, et al. BCRP gene polymorphisms are associated with susceptibility and survival of diffuse large B-cell lymphoma. Carcinogenesis. 2007;28(8):1740-4  [DOI](https://doi.org/10.1093/carcin/bgm113) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17494054/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Carcinogenesis&title=BCRP%20gene%20polymorphisms%20are%20associated%20with%20susceptibility%20and%20survival%20of%20diffuse%20large%20B-cell%20lymphoma&author=LL%20Hu&author=XX%20Wang&author=X%20Chen&author=J%20Chang&author=C%20Li&volume=28&issue=8&publication_year=2007&pages=1740-4&pmid=17494054&doi=10.1093/carcin/bgm113&)

18. Radriguez-Antona C, Sayi JG, Gustafsson LL, Bertilsson L, Ingelman-Sundberg M. Phenotype-genotype variability in the human CYP3A locus as assessed by the probe drug quinine and analysis of variant CYP3A4 alleles. Biochem Biophys Res Comm. 2005;338(1):299-305  [DOI](https://doi.org/10.1016/j.bbrc.2005.09.020) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16171783/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20Biophys%20Res%20Comm.&title=Phenotype-genotype%20variability%20in%20the%20human%20CYP3A%20locus%20as%20assessed%20by%20the%20probe%20drug%20quinine%20and%20analysis%20of%20variant%20CYP3A4%20alleles&author=C%20Radriguez-Antona&author=JG%20Sayi&author=LL%20Gustafsson&author=L%20Bertilsson&author=M%20Ingelman-Sundberg&volume=338&issue=1&publication_year=2005&pages=299-305&pmid=16171783&doi=10.1016/j.bbrc.2005.09.020&)

19. Hu YF, He J, Chen GL, Wang D, Liu ZQ, Zhang C, et al. CYP3A5*3 and CYP3A4*18 single nucleotide polymorphisms in a Chinese population. Clin Chim Acta. 2005; 353(1-2):187-92  [DOI](https://doi.org/10.1016/j.cccn.2004.11.005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15698606/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Chim%20Acta&title=CYP3A5*3%20and%20CYP3A4*18%20single%20nucleotide%20polymorphisms%20in%20a%20Chinese%20population&author=YF%20Hu&author=J%20He&author=GL%20Chen&author=D%20Wang&author=ZQ%20Liu&volume=353&issue=1-2&publication_year=2005&pages=187-92&pmid=15698606&doi=10.1016/j.cccn.2004.11.005&)

20. Van Schaik RHN, van der Heiden IP, van den Anker JN, Lindemans J. CYP3A5 variant allele frequencies in Dutch Caucasians. Clin Chem. 2002;48(10):1668-71  [PubMed](https://pubmed.ncbi.nlm.nih.gov/12324482/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Chem&title=CYP3A5%20variant%20allele%20frequencies%20in%20Dutch%20Caucasians&author=RHN%20Van%20Schaik&author=IP%20van%20der%20Heiden&author=JN%20van%20den%20Anker&author=J%20Lindemans&volume=48&issue=10&publication_year=2002&pages=1668-71&pmid=12324482&)

21. Guo SW, Thompson EA. Performing the exact test of Hardy-Weinberg proportion for multiple alleles. Biometrics. 1992;48(2):361-72  [PubMed](https://pubmed.ncbi.nlm.nih.gov/1637966/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biometrics&title=Performing%20the%20exact%20test%20of%20Hardy-Weinberg%20proportion%20for%20multiple%20alleles&author=SW%20Guo&author=EA%20Thompson&volume=48&issue=2&publication_year=1992&pages=361-72&pmid=1637966&)

22. Kantarjian HM, Giles F, Gattermann N, Bhalla K, Alimena G, Palandri F, et al. Nilotinib (formerly AMN107), a highly selective BCR-ABL tyrosine kinase inhibitor, is effective in patients with Philadelphia chromosome-positive chronic myelogenous leukemia in chronic phase following imatinib resistance and intolerance. Blood. 2007;110(10):3540-6  [DOI](https://doi.org/10.1182/blood-2007-03-080689) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17715389/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Nilotinib%20(formerly%20AMN107),%20a%20highly%20selective%20BCR-ABL%20tyrosine%20kinase%20inhibitor,%20is%20effective%20in%20patients%20with%20Philadelphia%20chromosome-positive%20chronic%20myelogenous%20leukemia%20in%20chronic%20phase%20following%20imatinib%20resistance%20and%20intolerance&author=HM%20Kantarjian&author=F%20Giles&author=N%20Gattermann&author=K%20Bhalla&author=G%20Alimena&volume=110&issue=10&publication_year=2007&pages=3540-6&pmid=17715389&doi=10.1182/blood-2007-03-080689&)

23. Hochhaus A, Kantarjian HM, Baccarani M, Lipton JH, Apperley J F,, Druker BJ, et al. Dasatinib induces notable hematologic and cytogenetic responses in chronic-phase chronic myeloid leukemia after failure of imatinib therapy. Blood. 2007;109(6):2303-9  [DOI](https://doi.org/10.1182/blood-2006-09-047266) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17138817/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Dasatinib%20induces%20notable%20hematologic%20and%20cytogenetic%20responses%20in%20chronic-phase%20chronic%20myeloid%20leukemia%20after%20failure%20of%20imatinib%20therapy&author=A%20Hochhaus&author=HM%20Kantarjian&author=M%20Baccarani&author=JH%20Lipton&author=J%20F,%20Apperley&volume=109&issue=6&publication_year=2007&pages=2303-9&pmid=17138817&doi=10.1182/blood-2006-09-047266&)

24. Rosti G, Palandri F, Castagnetti F, Breccia M, Levato L, Gugliotta G, et al. Nilotinib for the frontline treatment of Ph(+) chronic myeloid leukemia. Blood. 2009;114(24): 4933-8  [DOI](https://doi.org/10.1182/blood-2009-07-232595) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19822896/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Nilotinib%20for%20the%20frontline%20treatment%20of%20Ph(+)%20chronic%20myeloid%20leukemia&author=G%20Rosti&author=F%20Palandri&author=F%20Castagnetti&author=M%20Breccia&author=L%20Levato&volume=114&issue=24&publication_year=2009&pages=4933-8&pmid=19822896&doi=10.1182/blood-2009-07-232595&)

25. Cortes JE, Jones D, O'Brien S, Jabbour E, Konopleva M, Ferrajoli A, et al. Nilotinib as front-line treatment for patients with chronic myeloid leukemia in early chronic phase. J Clin Oncol. 2010;28(3):392-7  [DOI](https://doi.org/10.1200/JCO.2009.25.4896) | [PMC free article](/articles/PMC2815701/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20008621/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Nilotinib%20as%20front-line%20treatment%20for%20patients%20with%20chronic%20myeloid%20leukemia%20in%20early%20chronic%20phase&author=JE%20Cortes&author=D%20Jones&author=S%20O'Brien&author=E%20Jabbour&author=M%20Konopleva&volume=28&issue=3&publication_year=2010&pages=392-7&pmid=20008621&doi=10.1200/JCO.2009.25.4896&)

26. Kantarjian HM, Hochhaus A, Saglio G, De Souza C, Flinn IW, Stenke L, et al. Nilotinib versus imatinib for the treatment of patients with newly diagnosed chronic phase, Philadelphia chromosome-positive, chronic myeloid leukaemia: 24-month minimum follow-up of the phase 3 randomised ENESTnd trial. Lancet Oncol. 2011;12(9): 841-51  [DOI](https://doi.org/10.1016/S1470-2045(11)70201-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21856226/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet%20Oncol&title=Nilotinib%20versus%20imatinib%20for%20the%20treatment%20of%20patients%20with%20newly%20diagnosed%20chronic%20phase,%20Philadelphia%20chromosome-positive,%20chronic%20myeloid%20leukaemia:%2024-month%20minimum%20follow-up%20of%20the%20phase%203%20randomised%20ENESTnd%20trial&author=HM%20Kantarjian&author=A%20Hochhaus&author=G%20Saglio&author=C%20De%20Souza&author=IW%20Flinn&volume=12&issue=9&publication_year=2011&pages=841-51&pmid=21856226&doi=10.1016/S1470-2045(11)70201-7&)

27. Kantarjian HM, Shah NP, Cortes JE, Baccarani M, Agarwal MB, Undurraga MS, et al. Dasatinib or imatinib in newly diagnosed chronic-phase chronic myeloid leukemia: 2-year follow-up from a randomized phase 3 trial (DASISION). Blood 2012; 119(5):1123-9  [DOI](https://doi.org/10.1182/blood-2011-08-376087) | [PMC free article](/articles/PMC4916556/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22160483/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Dasatinib%20or%20imatinib%20in%20newly%20diagnosed%20chronic-phase%20chronic%20myeloid%20leukemia:%202-year%20follow-up%20from%20a%20randomized%20phase%203%20trial%20(DASISION)&author=HM%20Kantarjian&author=NP%20Shah&author=JE%20Cortes&author=M%20Baccarani&author=MB%20Agarwal&volume=119&issue=5&publication_year=2012&pages=1123-9&pmid=22160483&doi=10.1182/blood-2011-08-376087&)

28. White DL, Dang P, Engler J, frede A, Zrim S, Osborn M, et al. Functional activity of the OCT-1 protein is predictive of long-term outcome in patients with chronic-phase chronic myeloid leukemia treated with imatinib. J Clin Oncol. 2010;28(16):2761-7  [DOI](https://doi.org/10.1200/JCO.2009.26.5819) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20421539/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Functional%20activity%20of%20the%20OCT-1%20protein%20is%20predictive%20of%20long-term%20outcome%20in%20patients%20with%20chronic-phase%20chronic%20myeloid%20leukemia%20treated%20with%20imatinib&author=DL%20White&author=P%20Dang&author=J%20Engler&author=A%20frede&author=S%20Zrim&volume=28&issue=16&publication_year=2010&pages=2761-7&pmid=20421539&doi=10.1200/JCO.2009.26.5819&)

29. White DL, Radich J, Soverini S, saunders VA, Frede A, Dang P, et al. Chronic phase chronic myeloid leukemia patients with low OCT-1 activity randomised to highdose imatinib achieve better responses, and lower failure rates, than those randomized to standard-dose. Haematologica. 2012;97(6):907-14  [DOI](https://doi.org/10.3324/haematol.2011.056457) | [PMC free article](/articles/PMC3366658/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22207690/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Haematologica&title=Chronic%20phase%20chronic%20myeloid%20leukemia%20patients%20with%20low%20OCT-1%20activity%20randomised%20to%20highdose%20imatinib%20achieve%20better%20responses,%20and%20lower%20failure%20rates,%20than%20those%20randomized%20to%20standard-dose&author=DL%20White&author=J%20Radich&author=S%20Soverini&author=VA%20saunders&author=A%20Frede&volume=97&issue=6&publication_year=2012&pages=907-14&pmid=22207690&doi=10.3324/haematol.2011.056457&)

30. Dulucq S, Bouchet S, Turcq B, Lippert E, Etienne G, Reiffers J, et al. Multidrug resistance gene (MDR1) polymorphisms are associated with major molecular responses to standard-dose imatinib in chronic myeloid leukemia. Blood. 2008;112(5): 2024-7  [DOI](https://doi.org/10.1182/blood-2008-03-147744) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18524988/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Multidrug%20resistance%20gene%20(MDR1)%20polymorphisms%20are%20associated%20with%20major%20molecular%20responses%20to%20standard-dose%20imatinib%20in%20chronic%20myeloid%20leukemia&author=S%20Dulucq&author=S%20Bouchet&author=B%20Turcq&author=E%20Lippert&author=G%20Etienne&volume=112&issue=5&publication_year=2008&pages=2024-7&pmid=18524988&doi=10.1182/blood-2008-03-147744&)

31. Kim DH, Sriharsha L, Xu W, Kamel-Reid S, Liu X, Siminovitch K, et al. Clinical relevance of a pharmacogenetic approach using multiple candidate genes to predict response and resistance to imatinib therapy in chronic myeloid leukemia. Clin Cancer Res. 2009;15(14):4750-8  [DOI](https://doi.org/10.1158/1078-0432.CCR-09-0145) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19584153/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=Clinical%20relevance%20of%20a%20pharmacogenetic%20approach%20using%20multiple%20candidate%20genes%20to%20predict%20response%20and%20resistance%20to%20imatinib%20therapy%20in%20chronic%20myeloid%20leukemia&author=DH%20Kim&author=L%20Sriharsha&author=W%20Xu&author=S%20Kamel-Reid&author=X%20Liu&volume=15&issue=14&publication_year=2009&pages=4750-8&pmid=19584153&doi=10.1158/1078-0432.CCR-09-0145&)

32. Ni LN, Li JY, Miao KR, Qiao C, Zhang SJ, Qiu HR, et al. Multidrug resistance gene (MDR1) polymorphisms correlate with imatinib response in chronic myeloid leukemia. Med Oncol. 2011;28(1):265-9  [DOI](https://doi.org/10.1007/s12032-010-9456-9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20204543/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Med%20Oncol&title=Multidrug%20resistance%20gene%20(MDR1)%20polymorphisms%20correlate%20with%20imatinib%20response%20in%20chronic%20myeloid%20leukemia&author=LN%20Ni&author=JY%20Li&author=KR%20Miao&author=C%20Qiao&author=SJ%20Zhang&volume=28&issue=1&publication_year=2011&pages=265-9&pmid=20204543&doi=10.1007/s12032-010-9456-9&)

33. Maffioli M, Camós M, Gaya A, Hernández-Boluda Juan-Carlos, Álvarez-Larrán A, et al. Correlation between genetic polymorphims of the hOCT1 and MDR1 genes and the response to imatinib in patients newly diagnosed with chronicphase chronic myeloid leukemia. Leuk Res. 2011;35(8):1014-9  [DOI](https://doi.org/10.1016/j.leukres.2010.12.004) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21185600/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Leuk%20Res&title=Correlation%20between%20genetic%20polymorphims%20of%20the%20hOCT1%20and%20MDR1%20genes%20and%20the%20response%20to%20imatinib%20in%20patients%20newly%20diagnosed%20with%20chronicphase%20chronic%20myeloid%20leukemia&author=M%20Maffioli&author=M%20Cam%C3%B3s&author=A%20Gaya&author=Juan-Carlos%20Hern%C3%A1ndez-Boluda&author=A%20%C3%81lvarez-Larr%C3%A1n&volume=35&issue=8&publication_year=2011&pages=1014-9&pmid=21185600&doi=10.1016/j.leukres.2010.12.004&)

34. White DL, Saunders VA, Dang P, Engler J, Venables A, Zrim S, et al. Most CML patients who have a suboptimal response to imatinib have low OCT-1 activity: higher doses of imatinib may overcome the negative impact of low OCT-1 activity. Blood. 2007;110(12):4064-72  [DOI](https://doi.org/10.1182/blood-2007-06-093617) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17761829/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Most%20CML%20patients%20who%20have%20a%20suboptimal%20response%20to%20imatinib%20have%20low%20OCT-1%20activity:%20higher%20doses%20of%20imatinib%20may%20overcome%20the%20negative%20impact%20of%20low%20OCT-1%20activity&author=DL%20White&author=VA%20Saunders&author=P%20Dang&author=J%20Engler&author=A%20Venables&volume=110&issue=12&publication_year=2007&pages=4064-72&pmid=17761829&doi=10.1182/blood-2007-06-093617&)

35. Hoffmeyer S, Burk O, von Richter O, Arnold H P,, Brockmöller J, Johne A, et al. Functional polymorphisms of the human multidrug-resistance gene: multiple sequence variations and correlation of one allele with P-glycoprotein expression and activity in vivo. Proc Natl Acad Sci USA. 2000; 97(7):3473-8  [DOI](https://doi.org/10.1073/pnas.050585397) | [PMC free article](/articles/PMC16264/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10716719/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20USA&title=Functional%20polymorphisms%20of%20the%20human%20multidrug-resistance%20gene:%20multiple%20sequence%20variations%20and%20correlation%20of%20one%20allele%20with%20P-glycoprotein%20expression%20and%20activity%20in%20vivo&author=S%20Hoffmeyer&author=O%20Burk&author=O%20von%20Richter&author=H%20P,%20Arnold&author=J%20Brockm%C3%B6ller&volume=97&issue=7&publication_year=2000&pages=3473-8&pmid=10716719&doi=10.1073/pnas.050585397&)

36. Kimchi-Sarfaty C, Oh JM, Kim IW, Sauna ZE, Calcagno AM, Ambudkar SV, et al. A “silent” polymorphisms in the MDR1 gene changes substrates specificity. Science. 2007;315(5811):525-8  [DOI](https://doi.org/10.1126/science.1135308) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17185560/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=A%20%E2%80%9Csilent%E2%80%9D%20polymorphisms%20in%20the%20MDR1%20gene%20changes%20substrates%20specificity&author=C%20Kimchi-Sarfaty&author=JM%20Oh&author=IW%20Kim&author=ZE%20Sauna&author=AM%20Calcagno&volume=315&issue=5811&publication_year=2007&pages=525-8&pmid=17185560&doi=10.1126/science.1135308&)

37. Cohen MH, Williams G, Johnson JR, Duan J, Gobburu J, Rahman A, et al. Approval summary for imatinib mesylate capsules in the treatment of chronic myelogenous leukemia. Clin Cancer Res. 2002;8(5):935-42  [PubMed](https://pubmed.ncbi.nlm.nih.gov/12006504/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=Approval%20summary%20for%20imatinib%20mesylate%20capsules%20in%20the%20treatment%20of%20chronic%20myelogenous%20leukemia&author=MH%20Cohen&author=G%20Williams&author=JR%20Johnson&author=J%20Duan&author=J%20Gobburu&volume=8&issue=5&publication_year=2002&pages=935-42&pmid=12006504&)

38. Frye RF, Fitzgerald SM, Lagattuta TF, Hruska MW, Egorin MJ. Effect of St John's wort on imatinib mesylate pharmacokinetics. Clin Pharmacol Ther. 2004;76(4):323-9  [DOI](https://doi.org/10.1016/j.clpt.2004.06.007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15470331/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Effect%20of%20St%20John's%20wort%20on%20imatinib%20mesylate%20pharmacokinetics&author=RF%20Frye&author=SM%20Fitzgerald&author=TF%20Lagattuta&author=MW%20Hruska&author=MJ%20Egorin&volume=76&issue=4&publication_year=2004&pages=323-9&pmid=15470331&doi=10.1016/j.clpt.2004.06.007&)

39. Peng B, Hayes M, Resta D, Racine-Poon A, Druker BJ, Talpaz M, et al. Pharmacokinetics and pharmacodynamics of imatinib in a phase I trial with chronic myeloid leukemia patients. J Clin Oncol. 2004;22(5):935-42  [DOI](https://doi.org/10.1200/JCO.2004.03.050) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14990650/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Pharmacokinetics%20and%20pharmacodynamics%20of%20imatinib%20in%20a%20phase%20I%20trial%20with%20chronic%20myeloid%20leukemia%20patients&author=B%20Peng&author=M%20Hayes&author=D%20Resta&author=A%20Racine-Poon&author=BJ%20Druker&volume=22&issue=5&publication_year=2004&pages=935-42&pmid=14990650&doi=10.1200/JCO.2004.03.050&)

40. Gréen H, Skoglund K, Rommel F, Mirghani RA, Lotfi K. Cyp3A activity influences imatinib response in patients with chronic myeloid leukemia: a pilot study on in vivo CYP3A activity. Eur J Clin Parmacol. 2010; 66(4):383-6  [DOI](https://doi.org/10.1007/s00228-009-0772-y) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20054526/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Clin%20Parmacol&title=Cyp3A%20activity%20influences%20imatinib%20response%20in%20patients%20with%20chronic%20myeloid%20leukemia:%20a%20pilot%20study%20on%20in%20vivo%20CYP3A%20activity&author=H%20Gr%C3%A9en&author=K%20Skoglund&author=F%20Rommel&author=RA%20Mirghani&author=K%20Lotfi&volume=66&issue=4&publication_year=2010&pages=383-6&pmid=20054526&doi=10.1007/s00228-009-0772-y&)

41. Amirimani B, Walker AH, Weber BL, Rebbeck TP. Re: modification of clinical presentation of prostate tumors by a novel genetic variant in CYP3A4. J Natl Cancer Inst. 1999;91(18):1588-90  [DOI](https://doi.org/10.1093/jnci/91.18.1588) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10491443/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Natl%20Cancer%20Inst&title=Re:%20modification%20of%20clinical%20presentation%20of%20prostate%20tumors%20by%20a%20novel%20genetic%20variant%20in%20CYP3A4&author=B%20Amirimani&author=AH%20Walker&author=BL%20Weber&author=TP%20Rebbeck&volume=91&issue=18&publication_year=1999&pages=1588-90&pmid=10491443&doi=10.1093/jnci/91.18.1588&)

42. Amirimani B, Ning B, Deits AC, Weber BL, Kadlubar FF, Rebbeck TR. Increased transcriptional activity of the CYP3A4*1B promoter variant. Environ Mol Mutagen. 2003; 42(4):299-305  [DOI](https://doi.org/10.1002/em.10199) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14673875/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Environ%20Mol%20Mutagen&title=Increased%20transcriptional%20activity%20of%20the%20CYP3A4*1B%20promoter%20variant&author=B%20Amirimani&author=B%20Ning&author=AC%20Deits&author=BL%20Weber&author=FF%20Kadlubar&volume=42&issue=4&publication_year=2003&pages=299-305&pmid=14673875&doi=10.1002/em.10199&)

43. Larson RA, Druker BJ, Guilhot F, O'Brien SG, Riviere GJ, Krahnke T, et al. Imatinib pharmacokinetics and its correlation with response and safety in chronic-phase chronic myeloid leukemia: a subanalysis of the IRIS study. Blood. 2008;111(8):4022-8  [DOI](https://doi.org/10.1182/blood-2007-10-116475) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18256322/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Imatinib%20pharmacokinetics%20and%20its%20correlation%20with%20response%20and%20safety%20in%20chronic-phase%20chronic%20myeloid%20leukemia:%20a%20subanalysis%20of%20the%20IRIS%20study&author=RA%20Larson&author=BJ%20Druker&author=F%20Guilhot&author=SG%20O'Brien&author=GJ%20Riviere&volume=111&issue=8&publication_year=2008&pages=4022-8&pmid=18256322&doi=10.1182/blood-2007-10-116475&)

44. le Coutre P, Kreuzer KA, Pursche S, Bonin M, Leopold T, Baskaynak G, et al. Pharmacokinetics and cellular uptake of imatinib and its main metabolite CGP74588. Cancer Chemother Pharmacol. 2004;53(4):313-23  [DOI](https://doi.org/10.1007/s00280-003-0741-6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14658008/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Chemother%20Pharmacol&title=Pharmacokinetics%20and%20cellular%20uptake%20of%20imatinib%20and%20its%20main%20metabolite%20CGP74588&author=P%20le%20Coutre&author=KA%20Kreuzer&author=S%20Pursche&author=M%20Bonin&author=T%20Leopold&volume=53&issue=4&publication_year=2004&pages=313-23&pmid=14658008&doi=10.1007/s00280-003-0741-6&)
