# IL28B genotype is associated with differential expression of intrahepatic interferon-stimulated genes in chronic hepatitis C patients

## Metadata
**Authors:** Thomas J Urban, Alexander J Thompson, Shelton Bradrick, Jacques Fellay, Detlef Schuppan, Kenneth D Cronin, Linda Hong, Alexander McKenzie, Keyur Patel, Kevin V Shianna, John G McHutchison, David B Goldstein, Nezam Afdhal
**Journal:** Hepatology (Baltimore, Md.)
**Date:** 2010 Oct 7
**DOI:** [10.1002/hep.23912](https://doi.org/10.1002/hep.23912)
**PMID:** 20931559
**PMCID:** PMC3653303
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3653303/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC3653303/pdf/nihms227323.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC3653303/pdf/nihms227323.pdf)

## Abstract

**Conclusion:** 
The good response IL28B variant was strongly associated with lower level ISG expression. The results suggest that IL28B genotype may explain the relationship between hepatic ISG expression and HCV treatment outcome, and this is independent of miR-122 expression. IL28B-type was not associated with intrahepatic IL28B mRNA expression in vivo. Further investigation of the precise molecular mechanism(s) by which IL28B genetic variation influences HCV outcomes is warranted.

Keywords: interferon lambda, hepatitis C virus, gene expression

### Conclusion

The good response *IL28B* variant was strongly associated with lower level ISG expression. The results suggest that *IL28B* genotype may explain the relationship between hepatic ISG expression and HCV treatment outcome, and this is independent of miR-122 expression. *IL28B*-type was not associated with intrahepatic *IL28B* mRNA expression in vivo. Further investigation of the precise molecular mechanism(s) by which *IL28B* genetic variation influences HCV outcomes is warranted.

## Materials and Methods

### Patient Population

Sixty-one patients were recruited for this study; all attended the Liver Clinic at Beth Israel Deaconess Medical Center (Harvard University, Boston, MA, USA) for the management of HCV mono-infection, and had consented to storage of liver biopsy specimens and clinical data for research purposes. Liver biopsy in all patients was performed at this institution, and liver specimens were immediately placed in RNA later (Qiagen, Valencia, CA, USA) before storage at −80°C. Forty-one patients were treatment-naïve at the time of liver biopsy, and 20 were non-responders (NR) to prior IFN-based treatment regimens (standard / pegIFN ± RBV). Detailed description of initial viral response was not available for all patients, and since sample size was small, we grouped non-responders and relapsers as treatment failures and compared them to patients who attained SVR (responders). All patients had stopped antiviral therapy at least 6 months prior to biopsy. Written informed consent was obtained from all patients. The protocol was approved by the Institutional Review Board at Beth Israel Deaconess Medical Center.

At the time of liver biopsy, all patients had blood taken for serum liver function tests, a quantitative measure of serum/plasma HCV RNA performed using the Roche PCR assay, with a lower limit of detection of 50 IU/mL. HCV genotyping was performed using the line probe assay in all patients. All biopsy samples were evaluated by a single hepatopathologist and scored for stage of fibrosis and grade of inflammation using the METAVIR scoring system([12](#R12)).

### DNA / RNA extraction

Total RNA and genomic DNA were extracted from liver biopsies using the QIAgen RNeasy Mini Kit (QIAgen, Valencia, CA, USA) with a modification of the manufacturer’s protocol for DNA extraction after elution of RNA ([http://www.natureprotocols.com/2009/08/17/a_useful_procedure_to_isolate.php](http://www.natureprotocols.com/2009/08/17/a_useful_procedure_to_isolate.php)).

### IL28B genotyping

The region on chromosome 19 associated with HCV treatment response contains multiple SNPs in linkage disequilibrium around the *IL28B* gene([1](#R1)–[3](#R3)). We selected the most strongly associated SNP, rs12979860, which is a bi-allelic polymorphism (CC, CT, TT) that has been associated with treatment response in both Caucasians and African Americans ([1](#R1)). Genotyping was performed using the ABI TaqMan allelic discrimination kit and the ABI7900HT sequence Detection System (Applied Biosystems, Carlsbad, CA, USA) in 5uL reaction volume using standard Taqman Universal PCR conditions. Primer and probe sequences were: Forward Primer 5’-GCCTGTCGTGTACTGAACCA-3’, Reverse Primer 5’-GCGCGGAGTGCAATTCAAC-3’, Probe (C allele) 5’-VIC-TGGTTCGCGCCTTC-3’, Probe (T allele) 5’-FAM-CTGGTTCACGCCTTC-3’. Genotyping was conducted in a blinded fashion relative to HCV treatment status and other patient or treatment response characteristics.

### Expression Analysis

Approximately 200 ng total RNA was amplified and transcribed to cRNA, and hybridized to Illumina HT-12 Expression BeadChip per the manufacturer’s protocol (Illumina Inc., San Diego, CA, USA), targeting around 25,000 annotated genes with more than 48,000 probes.

### Quantitative PCR for Targeted Gene Expression Measurements

A custom Taqman MGB assay for gene expression was designed to achieve 1000-fold specificity for *IL28B* vs. the close homolog IL28A (see [Supplemental Figure S1](#SD1)). Primer and probe sequences are provided in the [Supplemental Materials](#SD1). Applied Biosystems Taqman Assays-on-Demand were used to detect expression of selected ISGs: IFI6, IFI27, and ISG15. miR-122 and RNA U6 determination was made using a specific Taqman MicroRNA Assay(Applied Biosystems, Foster City, CA, USA). Approximately 1 μg of total RNA from liver biopsies was reverse transcribed using the High-Capacity cDNA Synthesis Kit (Applied Biosystems, Foster City, CA, USA), using either an miRNA-specific primer (for miR-122 and U6) or random hexamer primers. Approximately 100 ng cDNA was used per reaction in a total volume of 10 μL, and duplicate reactions were performed for each sample. Data were transformed using the ΔΔCT method normalizing to GAPDH or beta-actin as reference genes(for miR-122, U6 RNA was used as a reference), and further normalized to the lowest-expressing sample for comparison.

### In Vitro Characterization of Coding IL28B Variants

Recombinant proteins were produced by cloning expression-optimized (Blue Heron Biotechnology) IL28B cDNA sequences corresponding to the two alternative alleles at the rs8103142 *IL28B* coding variant into pGEX-6P-2. GST fusion proteins were expressed in BL-21 cells and purified using glutathione-sepharose according to the manufacturer (GE Healthcare). Protein integrity and concentration were determined by SDS-PAGE.

For purification from human cells, the *IL28B* cDNA sequences corresponding to the two alternative alleles at the rs8103142 variant were cloned into pcDNA5/FRT/TO and stably integrated into Flp-in-Trex 293 cells(Invitrogen, Carlsbad, CA, USA). After 24 hours of culture of confluent stable cells in serum-free media containing 1 μg/ml doxycycline, the supernatant was collected and concentrated using centricon devices (Millipore, Billerica, MA , USA). Total protein was measured by the Bradford method, and serial dilutions of concentrate were used to treat Huh7.5 cells hosting an HCV replicon as described previously ([13](#R13)). HCV replicon RNA measurement was performed using a custom Taqman real-time PCR method, normalizing to GAPDH mRNA expression. Primer and probe sequences for HCV RNA determination were: Forward Primer 5’-TCAATAGGGTGGCTTCATGCCTCA-3’, Reverse Primer 5’-TGGAGTGAGTTTGAGCTTGGTCCT-3’, Probe 5’-VIC-TGGCAAGTACCTCTTCAACTGGGCA-3’. Measurement of *MX1* expression was performed in parallel using a specific Taqman Gene Expression Assay (Applied Biosystems, Foster City, CA, USA). Kinetics of HCV inhibition and *MX1* stimulation were estimated by least-squares regression using GraphPad Prism v5(GraphPad Software, Inc., La Jolla, CA, USA).

### Statistical Analysis

Microarray expression data were analyzed using the Partek Genomics Suite v6.4 (Partek Inc., St. Louis, MO, USA). Expression data were log_2_-transformed and normalized by quantile normalization prior to statistical analysis. Outlier samples identified by principal components analysis were removed from the study. A linear mixed model analysis was performed using ANCOVA. *IL28B* genotype, gender, age and ethnicity were included in the model. A false discovery rate threshold of 0.1 was used to correct for multiple testing, corresponding to a p-value< 4.3 × 10^−4^. Baseline HCV viral load was added to the linear model to determine viral load-independent relationships between *IL28B* genotype and gene expression. Enrichment of differentially expressed genes in particular biological pathways was determined using Ingenuity Pathway Analysis v8.0 (Ingenuity Systems Inc., Redwood City, CA, USA).A composite ISG metric was calculated as the average expression level of IFI6, IFI27, and ISG15 after linear (anti-log_2_) transformation and tested for association with SVR, IL28B genotype, and miR-122 expression by ANOVA. Differences in potency between IFN-λ3 variants were assessed by comparison of the 95% confidence intervals of the IC_50_ or EC_50_ parameter estimates.

## Results

### Patient Characteristics

The characteristics of the 61 patients included in the study are described in [Table 1](#T1). Patients represented multiple ethnicities including African American, Caucasian and Hispanic, with the majority infected with genotype 1 HCV([Table 1](#T1)). Most patients had mild-to-moderate hepatic necro-inflammatory activity and fibrosis. The distribution of *IL28B* genotype is listed in [Table 1](#T1). The C allele frequency was 0.55 and these genotypes were in Hardy-Weinberg equilibrium(P=0.76).

### Table 1. Clinical characteristics and genotype distributions in the study cohort (n = 61).

| Parameter | N=61 |   |
| --- | --- | --- |
|   |  |  |
| Age (years) | 51 | 43 – 56 |
| Male gender | 41 | 67.21% |
|   |  |  |
| Ethnicity |   |   |
| Caucasian | 38 | 62.30% |
| Hispanic | 5 | 8.20% |
| Asian | 5 | 8.20% |
| African-American | 12 | 19.67% |
|   |  |  |
| HCV RNA ( log10 IU/mL) | 6.08 | 5.61 – 6.62 |
|   |  |  |
| HCV genotype 1 | 55 | 90.16% |
|   |  |  |
| ALT (IU/mL) | 63 | 46 – 87 |
|   |  |  |
| METAVIR activity A2–3 (n,%) | 32 | 53.33% |
|   |  |  |
| METAVIR fibrosis F3–4 (n,%) | 2 | 3.33% |
| IL28B genotype |   |   |
| CC | 19 | 31.15% |
| CT | 29 | 47.54% |
| TT | 13 | 21.31% |
| C allele frequency | 0.55 |   |

Table 1 Caption: Data represent median (25 – 75 th centile) unless otherwise indicated.

### Gene Expression Profile vs IL28B genotype

Based on previous reports suggesting modest differences between CT and TT genotypes at rs12979860 in relation to SVR ([1](#R1)–[3](#R3)), genotypes were collapsed into a recessive model and comparisons were performed as CC vs. CT or TT (non-CC) genotypes. After applying a false discovery rate threshold of <0.1 to minimize false positive associations, a total of 164 genes were found to be differentially expressed between genotype groups. A full list of differentially expressed genes and the corresponding fold-change in expression and p-value is provided in [Supplemental Table S1](#SD1).

Further filtering to those genes with FDR <0.1 and absolute expression differences of at least ± 1.5-fold between CC vs. non-CC individuals gave a subset of 32 genes highly differentially expressed by genotype. Hierarchical clustering of this subset showed strong discrimination by *IL28B* genotype (see [Figure 1](#F1)), with a large number of ISGs having significantly lower expression in CC vs non-CC genotypes, including biologically relevant ISGs such as *MX1* (2.6-fold), *OAS1* (1.9-fold), *OAS2* (2.2-fold), *OAS3* (2.1-fold), *IFIT1* (2.3-fold), *IFIT2*, (1.7-fold), *IFIT3* (1.8-fold) and *ISG15* (3.9-fold). Notably, three genes in this subset showed the reverse effect, with higher expression in CC vs non-CC individuals: *FCN1* (1.8-fold), *CXCL9* (3.3-fold) and *CCL8* (1.6-fold).

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/571e/3653303/d59bb794c8e8/nihms227323f1.jpg)

Hierarchical clustering of samples according to genes showing the greatest differences in expression between IL28B genotypes. For clarity, only probes with FDR-corrected p-value <0.1 and a fold-change of >1.5 or of <−1.5 were included in the dendogram. IL28B-CC samples have higher expression than CT and TT samples when probe is red and lower expression when probe is blue.

Genes showing significant association with *IL28B* genotype were then analyzed for overrepresentation in known biological pathways using Ingenuity software. A number of immunological pathways were found to be significantly enriched for *IL28B*-associated genes, including interferon signaling as well as viral recognition pathways and a variety of adaptive and innate immune response pathways (see [Figure 2](#F2)). Further analysis of enrichment for biological functions showed a similar enrichment for inflammatory and antimicrobial response genes, among other functions such as cell-to-cell signaling ([Supplemental Figure S2](#SD1)).

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/571e/3653303/16acda76a6c8/nihms227323f2.jpg)

Pathway analysis showed that genes showing differential expression by IL28B-type clustered in the canonical interferon signaling pathway (FDR P value threshold=0.1). Several additional immune-related pathways reach significance in this analysis.

### Tests for independence of IL28B-associated and viral load-associated ISG expression

It was hypothesized that high baseline viral load may induce ISG expression in a manner unrelated to *IL28B* genotype. Thus, further analysis was performed to determine whether particular genes may be associated with *IL28B* genotype independently of viral load. Inclusion of baseline viral load in the ANOVA model revealed essentially the same set of genes associated with *IL28B* genotype, i.e., that the differences in gene expression by *IL28B* genotype were largely independent of HCV viral load at baseline. Hierarchical clustering of gene expression showed a similar pattern of low ISG expression in CC individuals, with only *FCN1* and *CXCL9* showing higher expression in the protective CC genotype group (see [Supplemental Figure S3](#SD1)).

### Tests for independence of IL28B genotype and ISG expression in prediction of treatment response

Data on pegIFN and RBV treatment response (sustained viral response, or SVR) was available for a small subset of patients (n=25, including five responders and twenty non-responders). 17 were prior non-responders to pegIFN and RBV therapy (all treatment had been stopped at least 6 months prior to liver biopsy); 8 were treated after the liver biopsy, and of these patients, 5 were responders and 3 were non-responders. This allowed for an exploratory analysis of the relationship between ISG expression and SVR conditioned on *IL28B* genotype. Despite the small sample size, a significant association between CC genotype and SVR was observed in this subset (p = 0.0054, CC vs. non-CC). In the same subset of samples, tests for association between SVR and 3 ISGs previously shown to be relevant to treatment outcome (*ISG15*, *IFI27* and *IFI6*) were shown to be significant. These genes were chosen on the basis of previous data linking them to SVR([7](#R7)), and all 3 showed at least a 1.5-fold difference in expression between responders and non-responders (p-value < 0.05) in the current dataset, and were confirmed by real-time quantitative PCR. A composite ISG expression variable based on the average value of the real-time PCR-based expression measures for these three ISGs similarly showed a significant association with SVR. Due to limitations of sample size, we lacked sufficient power to discriminate whether ISG expression or IL28B genotype had stronger association with SVR; however, an exploratory analysis including both IL28B genotype and mean ISG expression in the regression model suggested that IL28B genotype may be the predominant factor (see [Supplemental Table S2](#SD1)).

### miR-122 Expression and IL28B Genotype

Recent evidence indicates a necessary role of miR-122 expression in HCV replication ([14](#R14)). Therefore, we evaluated the relationship between *IL28B* type and intrahepatic expression levels of miR-122 using a targeted real-time PCR assay (miRNA expression levels were not tested on the microarray). miR-122 expression showed no correlation with rs12979860 genotype (see [Supplemental Figure S4](#SD1)). In relation to SVR, miR-122 expression showed significantly lower expression levels in liver biopsies from non-responders compared with responders ([Figure 3](#F3)), as described previously ([15](#R15)). Further, although there was a high degree of correlation among the targeted ISGs *IFI6*, *IFI27* and *ISG15*, these ISGs were not correlated with miR-122 expression ([Supplemental Figure S5](#SD1)), suggesting that the association between *IL28B*-type, ISG expression levels and SVR is independent of miR-122.

### Figure 3.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/571e/3653303/fc29ea5edabf/nihms227323f3.jpg)

Relationship between hepatic miR-122 expression and SVR. Real-time PCR measurement of miR-122 expression in liver biopsies from patients with known treatment outcome showed improved response rate in individuals with higher miR-122 expression levels (p = 0.024 by ANOVA).

### Intrahepatic Expression of IL28B

Expression of *IL28A* and *IL28B* mRNA was detectable in HCV infected liver tissue; however, expression of these transcripts showed no significant relationship with *IL28B* genotype. Due to the lack of specificity in the probe sequences for IL28A vs *IL28B* on the microarray, and the superior quantitative power of targeted real-time PCR assays, a specific quantitative PCR assay was designed for determination of *IL28B* mRNA expression ([Supplemental Figure S1](#SD1)). Consistent with the microarray results, *IL28B* mRNA expression as determined using the targeted assay showed no relationship with *IL28B* genotype ([Figure 4](#F4)). Additionally, no relationship was observed between *IL28B* genotype and liver *IL28A* mRNA expression ([Supplemental Figure S6](#SD1)).

### Figure 4.

![Figure 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/571e/3653303/bf201c4df054/nihms227323f4.jpg)

Relationship between IL28B genotype and IL28B mRNA expression in liver tissue. A real-time PCR assay specific for IL28B demonstrated the absence of association between IL28B genotype and IL28B mRNA expression (p = 0.58 by one-way ANOVA), consistent with the microarray expression data.

### Comparison of Variant IL28B Potencies In Vitro

We have previously identified through fine mapping strategies that the rs12979860 polymorphism is on a common haplotype block with a non-synonymous coding variant in exon 2 of *IL28B*, rs8103142, the only coding polymorphism in *IL28B* ([1](#R1)). As described previously, the set of polymorphisms identified in the *IL28B* gene region are highly correlated (r^2^ > 0.85 for all pairwise comparisons), and therefore it is not possible through genetic association alone to determine which if any of these variants is causal for the association, given current sample sizes. Since no significant differences were detected in *IL28B* mRNA expression levels in CC vs. non-CC individuals ([Figure 4](#F4)), we reasoned that the single conservative amino acid change (Lys70Arg, rs8103142) correlating with *IL28B* genotype may be responsible for differential anti-viral potency and induction of ISGs. To test this hypothesis, the recombinant IFN-λ3 variants were expressed in *E. Coli* as GST-fusion proteins and purified for functional assays. Treatment of Huh7.5 hepatoma cells harboring the HCV-Con1 subgenomic replicon ([13](#R13))with either IFN-λ3 variant caused similar dose-dependent declines in viral RNA abundances, with maximum inhibitory concentrations of ~10 ng/mL ([Figure 5](#F5)). Corresponding analysis of *MX1* mRNA levels demonstrated potent *MX1* induction after treatment with either variant, suggesting that the Lys to Arg amino acid difference does not affect IFN-λ3 potency.

### Figure 5.

![Figure 5](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/571e/3653303/12535544f4f3/nihms227323f5a.jpg)

Kinetics of inhibition of HCV replication (A) and stimulation of MX1 expression (B) by recombinant variant IFN-λ3 proteins in HCV replication-competent Huh7.5 cells.

IFN-λ3 is not known to undergo post-translational modifications that would be absent in recombinant proteins produced in *E. Coli*. Nevertheless, we wished to exclude the possibility that a hypothetical difference in potency due to Lys70Arg would be obscured due to expression in bacteria. We therefore established stable HEK-293 cell lines that inducibly express each *IL28B* variant and secrete the IFN-λ3 proteins into cell culture media. In order to test functionality of these factors, media from cells expressing *IL28B* were harvested and concentrated and, as a negative control, media from parental cells lacking a stable *IL28B* trans-gene was similarly concentrated. Western blot analysis revealed that IFN-λ3 was only detectable in media from over-expressing cells (data not shown). HCV replicon-harboring cells were treated with varying amounts of IFN-λ3 or control media and measurements of HCV and *MX1* RNA levels were again performed. As with results obtained with *E. Coli*-produced proteins, each IFN-λ3 variant generated in human cell culture significantly repressed HCV RNA levels and induced *MX1* expression with indistinguishable potencies(data not shown). Taken together, these data suggest that that other non-coding variant(s) dictate phenotypic response to anti-HCV therapy.

## Conclusions

We have investigated the relationship between *IL28B* genotype and patterns of intrahepatic gene expression in an ethnically diverse U.S.-based cohort of patients chronically infected with HCV. Previous reports have shown that high baseline ISG expression in liver tissue is associated with a poor response to treatment with IFN-based therapy ([7](#R7)–[11](#R11)). Here, we show that *IL28B* genotype is strongly associated with intrahepatic ISG expression, with the poor-response (non-CC) *IL28B* genotypes exhibiting higher ISG levels compared to patients with the favorable CC *IL28B* genotype. In a subgroup, viral clearance was associated with both *IL28B* genotype and lower level ISG expression. We also show that liver expression levels of *IL28B* mRNA are not associated with *IL28B* genotype, suggesting that the polymorphism might have an effect in biological potency of *IL28B* signaling. However, experimental analysis of the non-synonymous coding variant rs8103142 did not demonstrate any differences in anti-HCV or ISG induction potency between protein sequence variants.

The analysis of the intrahepatic transcriptome in patients with the poor response CT / TT *IL28B*-types compared to CC patients identified differential expression of > 150 genes. The list was dominated by canonical ISGs, including *MX1*, *OAS1/2/3*, *IFIT1/2/3*, *IFI6/27/44* and *ISG15*; all were up-regulated > 1.5-fold in CT/TT patients compared to CC patients. These genes have previously been shown to be among the main effector pathways of the IFN-mediated antiviral response ([16](#R16)). Pathway analysis confirmed that genes showing differential expression by *IL28B*-type clustered in the canonical interferon signaling pathway. That baseline induction of high level ISG expression is associated with poor therapeutic response to exogenous IFN therapy, perhaps by exhausting the IFN response pathway, has been suggested previously ([7](#R7)–[9](#R9), [11](#R11)). The data therefore confirms, in a North American population, the recent results from a study in a Japanese patient population reporting an association between intahepatic levels of ISG expression and pegIFN/RBV response as well as *IL28B* genotype (rs8099917)([17](#R17)).

Among the transcripts whose expression was highly differentiated by *IL28B* genotype, only three genes (*CCL8*, *CXCL9*, and *FCN1*) were shown to be significantly up-regulated in the protective CC genotype group. The chemotactic cytokine (CC) pathway, and *CXCL9* in particular, has previously been implicated in the control of HCV infection ([18](#R18)–[21](#R21)). The ficolin-1 (*FCN1*) gene, though not previously associated with anti-HCV activity, is suspected to play a role in innate immunity and also represents a plausible candidate for control of HCV ([22](#R22), [23](#R23)). The explanation for the discordant expression of these 3 genes relative to the general pattern of low level ISG expression remains unclear. A key question is whether the pattern of intrahepatic ISG expression is driven directly by IFN-λ3, or is a secondary effect of IFN-λ3 modulating a type 1 IFN effect, and this will require further investigation.

miR-122 has recently been identified to promote HCV replication, and an antisense inhibition of miR-122 has been shown to inhibit HCV ([24](#R24)). miR-122 is also involved in cholesterol metabolism, and inhibition profoundly reduced serum cholesterol levels in chimpanzees([24](#R24)). *IL28B* type has recently been associated with serum LDL levels in CHC patients ([25](#R25)). It was therefore plausible and important to investigate whether the *IL28B* polymorphism was associated with miR-122 expression levels. However, miR-122 was not observed to be associated with either *IL28B* type or ISG expression. Furthermore, in the treatment subgroup, we observed a significant relationship between miR-122 levels and treatment outcome, that was independent of *IL28B* genotype. This suggests that these two pathways may act independently to influence response to IFN-based treatment of HCV infection. The finding suggests that future treatment modalities for HCV infection may potentially take advantage of both IFN-dependent and miR-122-dependent modes of viral eradication, which may produce a synergistic effect on viral clearance.

The current study tests two of the most obvious hypotheses regarding the mechanism for the effect of *IL28B* genotype on HCV clearance: 1) differential intrahepatic expression of *IL28B* mRNA based on *IL28B* genotype, and 2) differential antiviral potency between protein sequence variants of the *IL28B* gene product, IFN-λ3. The relationship between *IL28B* genotype and *IL28B* mRNA in the literature has been controversial([1](#R1)–[3](#R3)).Previous studies have suggested that the effect of *IL28B* genotype on HCV treatment response may occur through differential expression of *IL28B* by genotype, with the low-response genotype showing lower expression of *IL28* in whole blood from healthy volunteers ([2](#R2)), or whole blood from HCV infected patients ([3](#R3)). More recently, Honda and colleagues showed that expression levels were similar in HCV-infected livers according to *IL28B* – type([17](#R17)). Here, we also show, using a highly specific assay for *IL28B* mRNA, that *IL28B* genotype is not associated with *IL28B* expression in liver biopsies from HCV infected patients. This suggested that the biological effect of the *IL28B* polymorphism might be to effect potency, which would require a non-synonymous causal variant. Previous work has suggested a possible causal role for a polymorphism in exon 2 (rs8103142, Arg70Lys). However, we rigorously investigated the potential for Arg70Lys to impact anti-viral potency of *IL28B* and found no discernible difference between variants. As fine mapping studies have not demonstrated any other coding variants to be highly linked with the discovery SNP, this suggests that the mechanism likely involves an explanation other than signaling potency. Chen and colleagues have recently shown that ISG expression levels may differ between different cell populations in the liver, where ISG upregulation was more pronounced in hepatocytes in non-responders to IFN therapy, and in Kupffer cells in responders ([26](#R26)). As microarray analysis does not differentiate the cell of origin of the observed ISG expression, it is possible that the *IL28B* polymorphism has cell-specific effects on expression that may not have been detected in the current study. Alternatively, genotype-specific polymorphisms could conceivably impact post-transcriptional regulation of *IL28B* expression in liver(i.e. mRNA splicing, translation, etc.) that would be missed by microarray and RT-PCR analyses.

The results presented here provide a paradox. The non-CC genotypes, associated with higher level ISG expression in the setting of chronic hepatitis C infection, have previously been associated with lower rates of spontaneous clearance. In contrast, the CC genotypes, in which low level ISG expression is observed, have been associated with high rates of spontaneous clearance ([4](#R4), [5](#R5)). It may be that the differential pattern of gene expression is specific to chronic HCV, and cannot be extrapolated to acute HCV. Alternatively, a more integrative hypothesis is that the effect of the poor-response *IL28B* variant is to ‘sabotage’ the liver’s response to type 1 IFN – perhaps by exhausting the common signaling pathway. In contrast, in the setting of the good-response CC variant, the liver remains relatively dormant, and ‘primed’ for type 1 IFN response. However, this hypothesis is speculative given current knowledge. Little is known about the interaction between type 1 and type 3 IFNs *in vivo*. Cell culture models of HCV replication suggest an additive antiviral effect of IFN-α and IFN-λ1/2, but no data is available yet for IFN-λ3; furthermore, these experimental models do not account for the host immune milieu. The precise pattern of interaction between type 1 IFN and type 3IFN signaling therefore remains unclear.

In conclusion, these data provide novel insights into the host genetic regulation of IFN signaling in the livers of patients chronically infected with HCV. The *IL28B* polymorphism is associated with differential intrahepatic gene expression profiles, where the good response *IL28B* variants are associated with lower levels of hepatic ISG expression. *IL28B* genotype was not associated with differences in intrahepatic *IL28B* gene expression, and protein sequence variants of IFN-λ3 do not appear to explain the differences in ISG expression or anti-HCV response by *IL28B* genotype. Thus, the biological mechanism underlying the influence of the *IL28B* polymorphism on IFN treatment response remains unclear. Further examination of the noncoding polymorphisms in the *IL28B* gene region, and of the dynamics of type 1 IFN sensitivity in the setting of sustained IFN-λ activity, both *in vitro* and *in vivo*, may provide a fuller understanding of the role of IFN-λ in natural clearance of HCV and its pharmacological treatment.

## Supplementary Material

## Acknowledgments

**Financial Support:**

Part of this work was supported by the National Institutes of Health grant NIH U19 AI066313 “Hepatitis C Cooperative Research Centers” to D.S. and N.H.A.

## Abbreviations

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Ge D, Fellay J, Thompson AJ, Simon JS, Shianna KV, Urban TJ, Heinzen EL, et al. Genetic variation in IL28B predicts hepatitis C treatment-induced viral clearance. Nature. 2009;461:399–401. doi: 10.1038/nature08309.  [DOI](https://doi.org/10.1038/nature08309) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19684573/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Genetic%20variation%20in%20IL28B%20predicts%20hepatitis%20C%20treatment-induced%20viral%20clearance&author=D%20Ge&author=J%20Fellay&author=AJ%20Thompson&author=JS%20Simon&author=KV%20Shianna&volume=461&publication_year=2009&pages=399-401&pmid=19684573&doi=10.1038/nature08309&)

2. Suppiah V, Moldovan M, Ahlenstiel G, Berg T, Weltman M, Abate ML, Bassendine M, et al. IL28B is associated with response to chronic hepatitis C interferon-alpha and ribavirin therapy. Nat Genet. 2009;41:1100–1104. doi: 10.1038/ng.447.  [DOI](https://doi.org/10.1038/ng.447) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19749758/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=IL28B%20is%20associated%20with%20response%20to%20chronic%20hepatitis%20C%20interferon-alpha%20and%20ribavirin%20therapy&author=V%20Suppiah&author=M%20Moldovan&author=G%20Ahlenstiel&author=T%20Berg&author=M%20Weltman&volume=41&publication_year=2009&pages=1100-1104&pmid=19749758&doi=10.1038/ng.447&)

3. Tanaka Y, Nishida N, Sugiyama M, Kurosaki M, Matsuura K, Sakamoto N, Nakagawa M, et al. Genome-wide association of IL28B with response to pegylated interferon-alpha and ribavirin therapy for chronic hepatitis C. Nat Genet. 2009;41:1105–1109. doi: 10.1038/ng.449.  [DOI](https://doi.org/10.1038/ng.449) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19749757/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Genome-wide%20association%20of%20IL28B%20with%20response%20to%20pegylated%20interferon-alpha%20and%20ribavirin%20therapy%20for%20chronic%20hepatitis%20C&author=Y%20Tanaka&author=N%20Nishida&author=M%20Sugiyama&author=M%20Kurosaki&author=K%20Matsuura&volume=41&publication_year=2009&pages=1105-1109&pmid=19749757&doi=10.1038/ng.449&)

4. Rauch A, Kutalik Z, Descombes P, Cai T, Di Iulio J, Mueller T, Bochud M, et al. Genetic variation in IL28B is associated with chronic hepatitis C and treatment failure: a genome-wide association study. Gastroenterology. 2010;138:1338–1345. 1345, e1331–1337. doi: 10.1053/j.gastro.2009.12.056.  [DOI](https://doi.org/10.1053/j.gastro.2009.12.056) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20060832/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastroenterology&title=Genetic%20variation%20in%20IL28B%20is%20associated%20with%20chronic%20hepatitis%20C%20and%20treatment%20failure:%20a%20genome-wide%20association%20study&author=A%20Rauch&author=Z%20Kutalik&author=P%20Descombes&author=T%20Cai&author=J%20Di%20Iulio&volume=138&publication_year=2010&pages=1338-1345&pmid=20060832&doi=10.1053/j.gastro.2009.12.056&)

5. Thomas DL, Thio CL, Martin MP, Qi Y, Ge D, O’Huigin C, Kidd J, et al. Genetic variation in IL28B and spontaneous clearance of hepatitis C virus. Nature. 2009;461:798–801. doi: 10.1038/nature08463.  [DOI](https://doi.org/10.1038/nature08463) | [PMC free article](/articles/PMC3172006/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19759533/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Genetic%20variation%20in%20IL28B%20and%20spontaneous%20clearance%20of%20hepatitis%20C%20virus&author=DL%20Thomas&author=CL%20Thio&author=MP%20Martin&author=Y%20Qi&author=D%20Ge&volume=461&publication_year=2009&pages=798-801&pmid=19759533&doi=10.1038/nature08463&)

6. Ghany MG, Strader DB, Thomas DL, Seeff LB. Diagnosis, management, and treatment of hepatitis C: an update. Hepatology. 2009;49:1335–1374. doi: 10.1002/hep.22759.  [DOI](https://doi.org/10.1002/hep.22759) | [PMC free article](/articles/PMC7477893/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19330875/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hepatology&title=Diagnosis,%20management,%20and%20treatment%20of%20hepatitis%20C:%20an%20update&author=MG%20Ghany&author=DB%20Strader&author=DL%20Thomas&author=LB%20Seeff&volume=49&publication_year=2009&pages=1335-1374&pmid=19330875&doi=10.1002/hep.22759&)

7. Asselah T, Bieche I, Narguet S, Sabbagh A, Laurendeau I, Ripault MP, Boyer N, et al. Liver gene expression signature to predict response to pegylated interferon plus ribavirin combination therapy in patients with chronic hepatitis C. Gut. 2008;57:516–524. doi: 10.1136/gut.2007.128611.  [DOI](https://doi.org/10.1136/gut.2007.128611) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17895355/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gut&title=Liver%20gene%20expression%20signature%20to%20predict%20response%20to%20pegylated%20interferon%20plus%20ribavirin%20combination%20therapy%20in%20patients%20with%20chronic%20hepatitis%20C&author=T%20Asselah&author=I%20Bieche&author=S%20Narguet&author=A%20Sabbagh&author=I%20Laurendeau&volume=57&publication_year=2008&pages=516-524&pmid=17895355&doi=10.1136/gut.2007.128611&)

8. Chen L, Borozan I, Feld J, Sun J, Tannis LL, Coltescu C, Heathcote J, et al. Hepatic gene expression discriminates responders and nonresponders in treatment of chronic hepatitis C viral infection. Gastroenterology. 2005;128:1437–1444. doi: 10.1053/j.gastro.2005.01.059.  [DOI](https://doi.org/10.1053/j.gastro.2005.01.059) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15887125/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastroenterology&title=Hepatic%20gene%20expression%20discriminates%20responders%20and%20nonresponders%20in%20treatment%20of%20chronic%20hepatitis%20C%20viral%20infection&author=L%20Chen&author=I%20Borozan&author=J%20Feld&author=J%20Sun&author=LL%20Tannis&volume=128&publication_year=2005&pages=1437-1444&pmid=15887125&doi=10.1053/j.gastro.2005.01.059&)

9. Feld JJ, Nanda S, Huang Y, Chen W, Cam M, Pusek SN, Schweigler LM, et al. Hepatic gene expression during treatment with peginterferon and ribavirin: Identifying molecular pathways for treatment response. Hepatology. 2007;46:1548–1563. doi: 10.1002/hep.21853.  [DOI](https://doi.org/10.1002/hep.21853) | [PMC free article](/articles/PMC2808168/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17929300/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hepatology&title=Hepatic%20gene%20expression%20during%20treatment%20with%20peginterferon%20and%20ribavirin:%20Identifying%20molecular%20pathways%20for%20treatment%20response&author=JJ%20Feld&author=S%20Nanda&author=Y%20Huang&author=W%20Chen&author=M%20Cam&volume=46&publication_year=2007&pages=1548-1563&pmid=17929300&doi=10.1002/hep.21853&)

10. Lanford RE, Guerra B, Bigger CB, Lee H, Chavez D, Brasky KM. Lack of response to exogenous interferon-alpha in the liver of chimpanzees chronically infected with hepatitis C virus. Hepatology. 2007;46:999–1008. doi: 10.1002/hep.21776.  [DOI](https://doi.org/10.1002/hep.21776) | [PMC free article](/articles/PMC2386986/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17668868/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hepatology&title=Lack%20of%20response%20to%20exogenous%20interferon-alpha%20in%20the%20liver%20of%20chimpanzees%20chronically%20infected%20with%20hepatitis%20C%20virus&author=RE%20Lanford&author=B%20Guerra&author=CB%20Bigger&author=H%20Lee&author=D%20Chavez&volume=46&publication_year=2007&pages=999-1008&pmid=17668868&doi=10.1002/hep.21776&)

11. Sarasin-Filipowicz M, Oakeley EJ, Duong FH, Christen V, Terracciano L, Filipowicz W, Heim MH. Interferon signaling and treatment outcome in chronic hepatitis C. Proc Natl Acad Sci U S A. 2008;105:7034–7039. doi: 10.1073/pnas.0707882105.  [DOI](https://doi.org/10.1073/pnas.0707882105) | [PMC free article](/articles/PMC2383932/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18467494/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&title=Interferon%20signaling%20and%20treatment%20outcome%20in%20chronic%20hepatitis%20C&author=M%20Sarasin-Filipowicz&author=EJ%20Oakeley&author=FH%20Duong&author=V%20Christen&author=L%20Terracciano&volume=105&publication_year=2008&pages=7034-7039&pmid=18467494&doi=10.1073/pnas.0707882105&)

12. Bedossa P, Poynard T. An algorithm for the grading of activity in chronic hepatitis C. The METAVIR Cooperative Study Group. Hepatology. 1996;24:289–293. doi: 10.1002/hep.510240201.  [DOI](https://doi.org/10.1002/hep.510240201) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8690394/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hepatology&title=An%20algorithm%20for%20the%20grading%20of%20activity%20in%20chronic%20hepatitis%20C.%20The%20METAVIR%20Cooperative%20Study%20Group&author=P%20Bedossa&author=T%20Poynard&volume=24&publication_year=1996&pages=289-293&pmid=8690394&doi=10.1002/hep.510240201&)

13. Blight KJ, McKeating JA, Marcotrigiano J, Rice CM. Efficient replication of hepatitis C virus genotype 1a RNAs in cell culture. J Virol. 2003;77:3181–3190. doi: 10.1128/JVI.77.5.3181-3190.2003.  [DOI](https://doi.org/10.1128/JVI.77.5.3181-3190.2003) | [PMC free article](/articles/PMC149761/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12584342/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Virol&title=Efficient%20replication%20of%20hepatitis%20C%20virus%20genotype%201a%20RNAs%20in%20cell%20culture&author=KJ%20Blight&author=JA%20McKeating&author=J%20Marcotrigiano&author=CM%20Rice&volume=77&publication_year=2003&pages=3181-3190&pmid=12584342&doi=10.1128/JVI.77.5.3181-3190.2003&)

14. Jopling CL, Yi M, Lancaster AM, Lemon SM, Sarnow P. Modulation of hepatitis C virus RNA abundance by a liver-specific MicroRNA. Science. 2005;309:1577–1581. doi: 10.1126/science.1113329.  [DOI](https://doi.org/10.1126/science.1113329) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16141076/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Modulation%20of%20hepatitis%20C%20virus%20RNA%20abundance%20by%20a%20liver-specific%20MicroRNA&author=CL%20Jopling&author=M%20Yi&author=AM%20Lancaster&author=SM%20Lemon&author=P%20Sarnow&volume=309&publication_year=2005&pages=1577-1581&pmid=16141076&doi=10.1126/science.1113329&)

15. Sarasin-Filipowicz M, Krol J, Markiewicz I, Heim MH, Filipowicz W. Decreased levels of microRNA miR-122 in individuals with hepatitis C responding poorly to interferon therapy. Nat Med. 2009;15:31–33. doi: 10.1038/nm.1902.  [DOI](https://doi.org/10.1038/nm.1902) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19122656/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Med&title=Decreased%20levels%20of%20microRNA%20miR-122%20in%20individuals%20with%20hepatitis%20C%20responding%20poorly%20to%20interferon%20therapy&author=M%20Sarasin-Filipowicz&author=J%20Krol&author=I%20Markiewicz&author=MH%20Heim&author=W%20Filipowicz&volume=15&publication_year=2009&pages=31-33&pmid=19122656&doi=10.1038/nm.1902&)

16. Sadler AJ, Williams BR. Interferon-inducible antiviral effectors. Nat Rev Immunol. 2008;8:559–568. doi: 10.1038/nri2314.  [DOI](https://doi.org/10.1038/nri2314) | [PMC free article](/articles/PMC2522268/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18575461/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Immunol&title=Interferon-inducible%20antiviral%20effectors&author=AJ%20Sadler&author=BR%20Williams&volume=8&publication_year=2008&pages=559-568&pmid=18575461&doi=10.1038/nri2314&)

17. Honda M, Sakai A, Yamashita T, Nakamoto Y, Mizukoshi E, Sakai Y, Nakamura M, et al. Hepatic Interferon-Stimulated Genes Expression Is Associated With Genetic Variation in Interleukin 28B and the Outcome of Interferon Therapy for Chronic Hepatitis C. Gastroenterology. 2010 doi: 10.1053/j.gastro.2010.04.049.  [DOI](https://doi.org/10.1053/j.gastro.2010.04.049) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20434452/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastroenterology&title=Hepatic%20Interferon-Stimulated%20Genes%20Expression%20Is%20Associated%20With%20Genetic%20Variation%20in%20Interleukin%2028B%20and%20the%20Outcome%20of%20Interferon%20Therapy%20for%20Chronic%20Hepatitis%20C&author=M%20Honda&author=A%20Sakai&author=T%20Yamashita&author=Y%20Nakamoto&author=E%20Mizukoshi&publication_year=2010&pmid=20434452&doi=10.1053/j.gastro.2010.04.049&)

18. Butera D, Marukian S, Iwamaye AE, Hembrador E, Chambers TJ, Di Bisceglie AM, Charles ED, et al. Plasma chemokine levels correlate with the outcome of antiviral therapy in patients with hepatitis C. Blood. 2005;106:1175–1182. doi: 10.1182/blood-2005-01-0126.  [DOI](https://doi.org/10.1182/blood-2005-01-0126) | [PMC free article](/articles/PMC1895193/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15860662/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Plasma%20chemokine%20levels%20correlate%20with%20the%20outcome%20of%20antiviral%20therapy%20in%20patients%20with%20hepatitis%20C&author=D%20Butera&author=S%20Marukian&author=AE%20Iwamaye&author=E%20Hembrador&author=TJ%20Chambers&volume=106&publication_year=2005&pages=1175-1182&pmid=15860662&doi=10.1182/blood-2005-01-0126&)

19. Florholmen J, Kristiansen MG, Steigen SE, Sorbye SW, Paulssen EJ, Kvamme JM, Konopski Z, et al. A rapid chemokine response of macrophage inflammatory protein (MIP)-1alpha, MIP-1beta and the regulated on activation, normal T expressed and secreted chemokine is associated with a sustained virological response in the treatment of chronic hepatitis C. Clin Microbiol Infect. 2010 doi: 10.1111/j.1469-0691.2010.03206.x.  [DOI](https://doi.org/10.1111/j.1469-0691.2010.03206.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20219081/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Microbiol%20Infect&title=A%20rapid%20chemokine%20response%20of%20macrophage%20inflammatory%20protein%20(MIP)-1alpha,%20MIP-1beta%20and%20the%20regulated%20on%20activation,%20normal%20T%20expressed%20and%20secreted%20chemokine%20is%20associated%20with%20a%20sustained%20virological%20response%20in%20the%20treatment%20of%20chronic%20hepatitis%20C&author=J%20Florholmen&author=MG%20Kristiansen&author=SE%20Steigen&author=SW%20Sorbye&author=EJ%20Paulssen&publication_year=2010&pmid=20219081&doi=10.1111/j.1469-0691.2010.03206.x&)

20. Zeremski M, Petrovic LM, Chiriboga L, Brown QB, Yee HT, Kinkhabwala M, Jacobson IM, et al. Intrahepatic levels of CXCR3-associated chemokines correlate with liver inflammation and fibrosis in chronic hepatitis C. Hepatology. 2008;48:1440–1450. doi: 10.1002/hep.22500.  [DOI](https://doi.org/10.1002/hep.22500) | [PMC free article](/articles/PMC2579317/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18798334/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hepatology&title=Intrahepatic%20levels%20of%20CXCR3-associated%20chemokines%20correlate%20with%20liver%20inflammation%20and%20fibrosis%20in%20chronic%20hepatitis%20C&author=M%20Zeremski&author=LM%20Petrovic&author=L%20Chiriboga&author=QB%20Brown&author=HT%20Yee&volume=48&publication_year=2008&pages=1440-1450&pmid=18798334&doi=10.1002/hep.22500&)

21. Zeremski M, Petrovic LM, Talal AH. The role of chemokines as inflammatory mediators in chronic hepatitis C virus infection. J Viral Hepat. 2007;14:675–687. doi: 10.1111/j.1365-2893.2006.00838.x.  [DOI](https://doi.org/10.1111/j.1365-2893.2006.00838.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17875002/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Viral%20Hepat&title=The%20role%20of%20chemokines%20as%20inflammatory%20mediators%20in%20chronic%20hepatitis%20C%20virus%20infection&author=M%20Zeremski&author=LM%20Petrovic&author=AH%20Talal&volume=14&publication_year=2007&pages=675-687&pmid=17875002&doi=10.1111/j.1365-2893.2006.00838.x&)

22. Lu J, Le Y, Kon OL, Chan J, Lee SH. Biosynthesis of human ficolin, an Escherichia coli-binding protein, by monocytes: comparison with the synthesis of two macrophage-specific proteins, C1q and the mannose receptor. Immunology. 1996;89:289–294. doi: 10.1046/j.1365-2567.1996.d01-732.x.  [DOI](https://doi.org/10.1046/j.1365-2567.1996.d01-732.x) | [PMC free article](/articles/PMC1456477/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8943728/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Immunology&title=Biosynthesis%20of%20human%20ficolin,%20an%20Escherichia%20coli-binding%20protein,%20by%20monocytes:%20comparison%20with%20the%20synthesis%20of%20two%20macrophage-specific%20proteins,%20C1q%20and%20the%20mannose%20receptor&author=J%20Lu&author=Y%20Le&author=OL%20Kon&author=J%20Chan&author=SH%20Lee&volume=89&publication_year=1996&pages=289-294&pmid=8943728&doi=10.1046/j.1365-2567.1996.d01-732.x&)

23. Lu J, Tay PN, Kon OL, Reid KB. Human ficolin: cDNA cloning, demonstration of peripheral blood leucocytes as the major site of synthesis and assignment of the gene to chromosome 9. Biochem J. 1996;313 ( Pt 2):473–478. doi: 10.1042/bj3130473.  [DOI](https://doi.org/10.1042/bj3130473) | [PMC free article](/articles/PMC1216931/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8573080/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20J&title=Human%20ficolin:%20cDNA%20cloning,%20demonstration%20of%20peripheral%20blood%20leucocytes%20as%20the%20major%20site%20of%20synthesis%20and%20assignment%20of%20the%20gene%20to%20chromosome%209&author=J%20Lu&author=PN%20Tay&author=OL%20Kon&author=KB%20Reid&volume=313&issue=Pt%202&publication_year=1996&pages=473-478&pmid=8573080&doi=10.1042/bj3130473&)

24. Lanford RE, Hildebrandt-Eriksen ES, Petri A, Persson R, Lindow M, Munk ME, Kauppinen S, et al. Therapeutic silencing of microRNA-122 in primates with chronic hepatitis C virus infection. Science. 2010;327:198–201. doi: 10.1126/science.1178178.  [DOI](https://doi.org/10.1126/science.1178178) | [PMC free article](/articles/PMC3436126/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19965718/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Therapeutic%20silencing%20of%20microRNA-122%20in%20primates%20with%20chronic%20hepatitis%20C%20virus%20infection&author=RE%20Lanford&author=ES%20Hildebrandt-Eriksen&author=A%20Petri&author=R%20Persson&author=M%20Lindow&volume=327&publication_year=2010&pages=198-201&pmid=19965718&doi=10.1126/science.1178178&)

25. Li JH, Lao XQ, Tillmann HL, Rowell J, Patel K, Thompson A, Suchindran S, et al. Interferon-lambda genotype and low serum low-density lipoprotein cholesterol levels in patients with chronic hepatitis C infection. Hepatology. 2010;51:1904–1911. doi: 10.1002/hep.23592.  [DOI](https://doi.org/10.1002/hep.23592) | [PMC free article](/articles/PMC2921623/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20235331/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hepatology&title=Interferon-lambda%20genotype%20and%20low%20serum%20low-density%20lipoprotein%20cholesterol%20levels%20in%20patients%20with%20chronic%20hepatitis%20C%20infection&author=JH%20Li&author=XQ%20Lao&author=HL%20Tillmann&author=J%20Rowell&author=K%20Patel&volume=51&publication_year=2010&pages=1904-1911&pmid=20235331&doi=10.1002/hep.23592&)

26. Chen L, Borozan I, Sun J, Guindi M, Fischer S, Feld J, Anand N, et al. Cell-type specific gene expression signature in liver underlies response to interferon therapy in chronic hepatitis C infection. Gastroenterology. 2010;138:1123–1133. e1121–1123. doi: 10.1053/j.gastro.2009.10.046.  [DOI](https://doi.org/10.1053/j.gastro.2009.10.046) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19900446/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastroenterology&title=Cell-type%20specific%20gene%20expression%20signature%20in%20liver%20underlies%20response%20to%20interferon%20therapy%20in%20chronic%20hepatitis%20C%20infection&author=L%20Chen&author=I%20Borozan&author=J%20Sun&author=M%20Guindi&author=S%20Fischer&volume=138&publication_year=2010&pages=1123-1133&pmid=19900446&doi=10.1053/j.gastro.2009.10.046&)
