# Genetic Variation in the Beta 2 Subunit of the Voltage-Gated Calcium Channel (CACNB2) and Pharmacogenetic Association with Adverse Cardiovascular Outcomes in the International Verapamil SR-Trandolapril Study-Genetic Substudy (INVEST-GENES)

## Metadata
**Authors:** Yuxin Niu, Yan Gong, Taimour Y Langaee, Heather M Davis, Hazem Elewa, Amber L Beitelshees, James I Moss, Rhonda M Cooper-DeHoff, Carl J Pepine, Julie A Johnson
**Journal:** Circulation. Cardiovascular genetics
**Date:** 2010 Dec 1
**DOI:** [10.1161/CIRCGENETICS.110.957654](https://doi.org/10.1161/CIRCGENETICS.110.957654)
**PMID:** 21156931
**PMCID:** PMC3060561
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3060561/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC3060561/pdf/nihms-272948.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC3060561/pdf/nihms-272948.pdf)

## Abstract

**Background:** 
Polymorphisms (SNPs) within the regulatory β2 subunit of the voltage-gated calcium channel (CACNB2) may contribute to variable treatment response to antihypertensive drugs and adverse cardiovascular outcomes.

**Methods and Results:** 
SNPs in CACNB2 from 60 ethnically diverse individuals were identified and characterized. Three common SNPs (rs2357928, rs7069292 and rs61839258) and a GWAS identified intronic SNP (rs11014166) were genotyped for a clinical association study in 5,598 hypertensive patients with coronary artery disease randomized to a beta-blocker (BB) or a calcium channel blocker (CCB) treatment strategy in INVEST-GENES. Reporter gene assays were conducted on the promoter SNP showing association with clinical outcomes. Twenty-one novel SNPs were identified. A promoter A>G SNP (rs2357928) was found to have significant interaction with treatment strategy for adverse cardiovascular outcomes (p for interaction = 0.002). In Caucasians, rs2357928 GG patients randomized to CCB were more likely to experience adverse outcome than those randomized to BB treatment strategy, with adjusted hazard ratio (CCB vs. BB) of 2.35 (1.19-4.66), p = 0.014. There was no evidence for such treatment difference in AG (1.16, 0.75-1.79, p = 0.69) and AA individuals (0.63, 0.36-1.11, p = 0.11). This finding was consistent in Hispanics and African Americans. CACNB2 rs11014166 showed similar pharmacogenetic effect in Hispanics, but not in Caucasians or African Americans. Reporter assay analysis of rs2357928 showed a significant increase in promoter activity for the G allele compared to the A allele.

**Conclusions:** 
These data suggest genetic variation within CACNB2 may influence treatment related outcomes in high risk hypertensive patients.

**Clinical Trial Registration Information:** 
Clinical trial identifier: NCT00133692, URL: http://clinicaltrials.gov/ct/gui/show/NCT00133692).

Keywords: Genetic variations, CACNB2, hypertension, cardiovascular outcomes, INVEST-GENES

### Background

Polymorphisms (SNPs) within the regulatory β2 subunit of the voltage-gated calcium channel (*CACNB2*) may contribute to variable treatment response to antihypertensive drugs and adverse cardiovascular outcomes.

### Methods and Results

SNPs in *CACNB2* from 60 ethnically diverse individuals were identified and characterized. Three common SNPs (rs2357928, rs7069292 and rs61839258) and a GWAS identified intronic SNP (rs11014166) were genotyped for a clinical association study in 5,598 hypertensive patients with coronary artery disease randomized to a beta-blocker (BB) or a calcium channel blocker (CCB) treatment strategy in INVEST-GENES. Reporter gene assays were conducted on the promoter SNP showing association with clinical outcomes. Twenty-one novel SNPs were identified. A promoter A>G SNP (rs2357928) was found to have significant interaction with treatment strategy for adverse cardiovascular outcomes (p for interaction = 0.002). In Caucasians, rs2357928 GG patients randomized to CCB were more likely to experience adverse outcome than those randomized to BB treatment strategy, with adjusted hazard ratio (CCB vs. BB) of 2.35 (1.19-4.66), p = 0.014. There was no evidence for such treatment difference in AG (1.16, 0.75-1.79, p = 0.69) and AA individuals (0.63, 0.36-1.11, p = 0.11). This finding was consistent in Hispanics and African Americans. *CACNB2* rs11014166 showed similar pharmacogenetic effect in Hispanics, but not in Caucasians or African Americans. Reporter assay analysis of rs2357928 showed a significant increase in promoter activity for the G allele compared to the A allele.

### Conclusions

These data suggest genetic variation within *CACNB2* may influence treatment related outcomes in high risk hypertensive patients.

### Clinical Trial Registration Information

Clinical trial identifier: [NCT00133692](https://clinicaltrials.gov/ct2/show/NCT00133692), URL: [http://clinicaltrials.gov/ct/gui/show/NCT00133692](http://clinicaltrials.gov/ct/gui/show/NCT00133692)).

## Introduction

High blood pressure is one of the major risk factors for cardiovascular disease (CVD) which is the leading cause of morbidity and mortality in the United States.[1](#R1) Beta blockers (BBs) and calcium channel blockers (CCBs) are among the most widely used antihypertensive drugs that are either used alone or in combination with other antihypertensive drugs to lower the blood pressure and the incidence of coronary events (myocardial infarction, death from coronary disease), strokes, and congestive heart failure.[2](#R2)^, ^[3](#R3) However, there exists limited pharmacogenetic data on CCBs in the literature. [4](#R4) [5](#R5)

Calcium channels mediate Ca^2+^ entry into cells in response to action potentials and sub-threshold depolarizing signals and are composed of four subunits: the pore forming α_1_ subunit, and the auxiliary α_2_, δ, β and γ subunits.[6](#R6) All CCBs work by binding to the α_1_ subunit of the voltage-gated L-type calcium channel (LTCC) to block transmembrane flow of Ca^2+^ through the channel. The β_2_ subunit of LTCC, encoded by *CACNB2* and expressed in cardiovascular tissue, is responsible for targeting the cell surface expression of the α_1_ pore forming subunit. Based on this physiologic role, we hypothesized *CACNB2* may contain variability important to response to antihypertensive drugs; particularly those with mechanisms derive from intracellular Ca^2+^, especially CCBs and BBs. Since the beginning of this study, a GWAS of blood pressure and hypertension reported *CACNB2* rs11014166 as one of the loci associated with both systolic and diastolic blood pressure and hypertension.[7](#R7) This finding strengthened our hypothesis about the potential influence of this gene on antihypertensive treatment response.

Selection of a hypertension treatment strategy in patients with stable coronary artery disease is largely empirical given that randomized trials have demonstrated equivalent outcomes with β-blocker- or calcium channel blocker-based treatment. In the context of one of these clinical trials, the INternational VErapamil SR Trandolapril Study, we have shown the regulatory β2 subunit of the voltage-gated calcium channel (*CACNB2*) to contain a polymorphism that associates with adverse cardiosvascular outcomes in a treatment specific manner. More specifically, Caucasians patients harboring the rs2357928 GG genotype randomized to calcium channel blocker-based treatment were more likely to experience an adverse cardiovascular outcome than those randomized to β-blocker-based treatment strategy. There was no evidence for such treatment difference in AG and AA individuals. This finding was consistent in Hispanics and African Americans. Reporter assay analysis of this polymorphism showed a significant increase in promoter activity for the G allele compared to the A allele. These data suggest that instead of empirical treatment, patients with the *CACNB2* rs2357928 GG genotype might benefit most from treatment with β blockers and that either β blockers or calcium channel blockers could be used in those with the AG and AA genotype. Importantly, *CACNB2* was among a small number of genes recently identified in hypertension GWAS. Taken together, these data suggest *CACNB2* is important in hypertension and treatment-associated outcomes in hypertensive patients. Replication of these findings in an independent clinical trial is warranted.

The purpose of this study was to investigate the pattern of genetic variation within *CACNB2* and explore whether this genetic variation plays a role in inter-patient variability in risk of adverse cardiovascular outcomes either independently or relative to antihypertensive drug treatment.

Selection of a hypertension treatment strategy in patients with stable coronary artery disease is largely empirical given that randomized trials have demonstrated equivalent outcomes with β-blocker- or calcium channel blocker-based treatment. In the context of one of these clinical trials, the INternational VErapamil SR Trandolapril Study, we have shown the regulatory β2 subunit of the voltage-gated calcium channel (*CACNB2*) to contain a polymorphism that associates with adverse cardiosvascular outcomes in a treatment specific manner. More specifically, Caucasians patients harboring the rs2357928 GG genotype randomized to calcium channel blocker-based treatment were more likely to experience an adverse cardiovascular outcome than those randomized to β-blocker-based treatment strategy. There was no evidence for such treatment difference in AG and AA individuals. This finding was consistent in Hispanics and African Americans. Reporter assay analysis of this polymorphism showed a significant increase in promoter activity for the G allele compared to the A allele. These data suggest that instead of empirical treatment, patients with the *CACNB2* rs2357928 GG genotype might benefit most from treatment with β blockers and that either β blockers or calcium channel blockers could be used in those with the AG and AA genotype. Importantly, *CACNB2* was among a small number of genes recently identified in hypertension GWAS. Taken together, these data suggest *CACNB2* is important in hypertension and treatment-associated outcomes in hypertensive patients. Replication of these findings in an independent clinical trial is warranted.

## Methods

### SNP Discovery

#### DNA samples

Genomic DNA isolated from lymphoblastic cell lines from 60 individuals of three racial/ethnic groups was obtained from Human Genetic Cell Repository sponsored by the National Institutes of Health housed at the Coriell Institute, Camden, NJ. [8](#R8) The 60 individuals included: 20 European Americans (EA), 20 African Americans (AA) and 20 Native Americans (NA). We also resequenced *CACNB2* from a chimpanzee (DNA obtained from Coriell) to assist in assessment of phylogenetic patterns associated with the discovered SNPs.

#### Resequencing and Detection of Variants

As the size of *CACNB2* is over 400 kb, we focused our resequencing effort on exons, intron/exon boundaries, 3’ and 5’ untranslated regions. These regions of *CACNB2* were amplified in fragments of approximately 217 to 658 bp. Four alternative promoter regions were amplified in fragments of approximately 1,973 to 2,879 bp. The purified PCR products were evaluated by direct sequencing with Amersham Biosciences ET-terminator chemistry method. DNA sequence data were compiled and polymorphic sites were identified using PolyPhred.[9](#R9) Sequence variations were called if both forward and reverse sequence reads were consistent.

Polymorphism Database Mining, Annotation Programs (PolyMAPr)[10](#R10) was used to predict the functional effects of nonsynonymous coding-region SNPs (PolyPhen) and any variants that might alter exon splicing enhancer sites (ESEfinder), putative transcription factor binding sites (JASPAR) or intron-exon splice sites (ASD: alternative splicing database, splice junction sequences).[10](#R10) SNPs were selected for analysis in the genetic association study based on PolyMAPr function score of >80% of the maximum possible, minor allele frequency of >0.05 in all three populations and at least 0.20 in one population.

#### Computational Methods

Comparative analysis of *CACNB2* sequences from multiple species (rat and mouse versus human) was performed with the VISTA genome browser tools ([http://pipeline.lbl.gov/cgibin/gateway2](http://pipeline.lbl.gov/cgibin/gateway2)).[11](#R11)^-^[13](#R13) The human (Human May 2004, UCSC hg17), mouse (May 2004, UCSC mm5) and rat (June 2003, UCSC rn3) genome sequences were obtained from the University of California Santa Cruz (UCSC) Genome Browser.

### Clinical Association Study

#### INVEST

The INternatioinal VErapamil SR Trandolapil Study (INVEST) evaluated adverse cardiovascular outcomes occurring following randomized treatment with either an atenolol-based BB treatment strategy or a verapamil SR-based CCB treatment strategy in 22,576 patients with documented coronary artery disease (CAD) and hypertension.[14](#R14) The design, protocol, and primary outcome have been published in detail elsewhere.[14](#R14)^, ^[15](#R15) Briefly, patients were eligible if they were aged 50 years or older and had documented CAD, with essential hypertension as defined by JNC VI[16](#R16) requiring drug therapy. Patients were randomly assigned to a verapamil SR- or an atenolol-based treatment strategy. Hydrochlorothiazide and/or trandolapril were added as needed to achieve JNC VI blood pressure targets.[16](#R16)

The primary outcome of INVEST was the first occurrence of all cause mortality, nonfatal myocardial infarction (MI), or nonfatal stroke. Secondary outcomes were the individual components of the primary outcome, i.e. all cause mortality, nonfatal MI and nonfatal stroke. Three members of the events committee, masked to treatment assignment, confirmed all outcome events by reviewing documentation and other pertinent patient records.

#### INVEST-GENES Cohort

Genetic samples were collected from 5,979 INVEST patients from 213 sites in the United States and Puerto Rico who provided genomic DNA samples and additional written informed consent for genetic studies. Genomic DNA was collected using buccal cells from mouthwash samples as previously described.[17](#R17) 5,598 samples still have sufficient quality and quantity DNA and were tested here. All patients provided written informed consent for participation in the main INVEST and in the genetic substudy and both studies were approved by the University of Florida Institutional Review Board. We conducted a nested case–control study including the 258 INVEST-GENES patients who experienced primary outcome event during study follow-up (cases) and 774 individuals who did not have an event during study follow-up, frequency-matched to cases for age (by decades), sex, and race/ethnicity in a ratio of approximately 3:1 (controls). We have previously documented the nested case-control approach to yield essentially identical results to analyses of the entire genetics cohort. [4](#R4)

#### Genotyping

The 4 SNPs selected for clinical association study (based on potential functional significance and minor allele frequencies of > 0.10): rs7069292, rs2357928, rs61839258 and rs7909119, were genotyped for INVEST-GENES case control samples by pyrosequencing.[18](#R18) One SNP (rs7909119) deviated from Hardy Weinberg Equilibrium, which appeared to be due to assay problems that could not be solved and was dropped from further consideration. The three SNPs included in the analysis were: rs61839258, rs2357928 and rs7069292. PCR reactions were carried out using HotStar Taq mix (Qiagen, Valencia, California USA) and 5 μl reactions were taken out to be carried out for pyrosequencing assays according to the manufacturer's recommendations.

During the final stages of our study, a GWAS study reported an intronic *CACNB2* SNP rs11014166 as being associated with blood pressure and hypertension.[7](#R7) We elected to genotype rs11014166 and rs2357928 (which showed pharmacogenetic association in the case-control sample analysis) in the entire INVEST-GENES cohort. Genotyping of the additional samples was performed with a Taqman SNP genotyping Assay. The following TaqMan probes were used for *CACNB2* rs11014166 and rs2357928 respectively: C_2740542_10, and TaqMan Custom Assay Part # 4331349. Genotype accuracy was verified by genotyping at least 5% randomly selected duplicate samples for each SNP on both TaqMan and Pyrosequencing genotyping platforms and results showed high concordance between two platforms.

### Reporter Assay for rs2357928

#### Construction of luciferase reporter vector constructs

Two luciferase reporter constructs, pGL4/+58G and pGL4/+58A, were generated by PCR using primer pairs including restriction sites *Kpn*I and *Hind*III for unidirectional subcloning. Details are shown in [supplemental materials](#SD1).

#### Transient transfection and luciferase assay

Human embryonic kidney (HEK) 293 cells and Chinese hamster ovary (CHO) cells were used for transient transfections according to the previously published procedures.[19](#R19) Details are reported in [supplemental materials](#SD1).

#### Fold luciferase activity

Fold luciferase activity was used for the relative promoter activity. The relative luciferase activity of the *CACNB2* reporter construct was represented as the ratio of the firefly luciferase activity to that of Renilla. The fold changes from negative control (pGL4/Basic) were calculated as the ratio of relative luciferase (pGL4/+58G and pGL4/+58A) to the mean relative luciferase (pGL4/Basic).

### Statistical analysis

Baseline characteristics were compared using chi-squared test or analysis of variance, as appropriate. Hardy–Weinberg Equilibrium was evaluated separately by race/ethnicity using chi-squared test with one degree of freedom. Linkage disequilibrium (LD) analysis was performed through use of the Haploview 3.32 program.[20](#R20)

To minimize the potential population stratification in our racially and ethnically diverse population in INVEST, all analyses were conducted by race/ethnicity separately. To control for potential of population admixture in each race/ethnicity group, we used a total panel of 87 ancestral informative markers, selected to show large allele frequency differences across three parental populations (West Africans, Indigenous Americans, and Europeans) selected from a panel of over 10,000 SNPs.[21](#R21) Maximum likelihood was used to estimate each patient's individual genomic ancestry proportions on these three axes and 2 of these 3 terms were included in the models to control for ancestry. These 87 AIMs were genotyped using either allele-specific PCR with universal energy transfer labeled primers[22](#R22) or competitive allele-specific PCR at Prevention Genetics, Marshfield, WI. To ensure accurate ancestry proportion estimates, at least 44 (50%) ancestry informative markers had to be genotyped successfully in each individual to be included in analyses.

For the case-control samples, unadjusted and adjusted odds ratios (ORs) and 95% confidence intervals (CIs) for occurrence of the primary outcome were calculated using logistic regression. The adjusted model controlled for the pre-specified covariates (age, gender, history of heart failure and history of MI) [23](#R23) and other potential confounders selected by a stepwise selection procedure (p<0.2 for entry, p<0.05 for stay): treatment strategy, ancestry, body mass index (BMI), mean on-treatment systolic blood pressure, diabetes, history of stroke or transient ischemic attack, renal insufficiency and smoking history. Interaction between genotype and treatment strategy was also evaluated. If a SNP showed a main effect of p < 0.10 for the composite outcome, secondary outcomes were also tested.

For the cohort analysis, Cox Proportional Hazard modeling was performed to assess the association of the SNPs and the risk for primary outcome and secondary outcomes adjusting for variables selected using the same model building process as mentioned above. In addition, time-varying exposure was used for verapamil, atenolol, HCTZ and trandolapril. The assumption of the proportion hazards was tested in each Cox regression model.[24](#R24) To account for multiple comparisons, we performed the false discovery rate (FDR) adjustment according to the method of Benjamini and Hochberg.[25](#R25)

Nonparametric Wilcoxon rank sum test was used to assess promoter activity indicated by luciferase activity ratios in the reporter assays. A two-sided *p*<0.05 was considered significant for the reporter assay analyses. All statistical analyses were conducted using SAS version 9.2 (Cary, North Carolina, USA).

## Results

### SNP Discovery

The genomic structure of the *CACNB2* gene is depicted in [Figure S1](#SD1). Exon 1A, 1B, 2C and 2D are the alternative first exons (AFE) in 11 transcript isoforms in the *CACNB2* gene. Exon 7A, 7B and 7C are mutually exclusive alternative exons in 8 transcript isoforms. The VISTA plot of the multiple species comparative analysis of *CACNB2* sequence from mouse and rat versus human is shown in [Figure S1D](#SD1). Comparison of the human *CACNB2* locus with those of mouse and rat genomic sequences showed strongly conserved (>75%) regions of the protein-coding (in blue) and 5’ untranslated region (in light green).

A total of 74 SNPs were identified in regions including exons and intron/exon boundaries, 3’ and 5’ UTR of *CACNB2* by direct sequencing ([Supplemental Table S1](#SD1)): 65 in African Americans (AA), and 45 both in European Americans (EA) and Native Americans (NA). Minor allele frequencies (MAF) of the *CACNB2* SNPs ranged from 0.025 to 0.5. Forty two SNPs were found in all three populations.

Fifty-three of the SNPs (72%) have been previously reported in the NCBI SNP database (dbSNP), while 21 (28%) were novel. Only 10 of the previously reported SNPs had been validated by multiple independent submissions with allele frequency information in public database. Detailed genotype data for each of the 60 individuals is available online at The Pharmacogenetics and Pharmacogenomics Knowledge Base (PharmGKB) website ([http://www.pharmgkb.org/](http://www.pharmgkb.org/)).

All SNPs in the three populations were in Hardy Weinberg Equilibrium. Pairwise LD of the 74 SNPs in *CACNB2* for the three race/ethnicity groups obtained from Haploview software are shown in [Figure S2](#SD1). A relative lack of strong LD was observed for all three groups, minimizing the value of a tag SNP approach in our association studies.

*In silico* tests of potential functional significance of the 74 SNPs of *CACNB2* were analyzed using PolyMAPr software. Fourteen SNPs suggested by PolyMapr to have potential functional significance (e.g. potential alternative promoter) are shown in [supplemental Table S2](#SD1). One of the potential functional SNP was located in the 3’-UTR region, and the other 13 were located in four candidate alternative promoter regions. Based on the PolyMapr function score of >80% of maximum possible, MAF > 0.05 in all three populations and with a MAF >0.20 in at least one, 4 SNPs (rs7069292, rs2357928, rs61839258 and rs7909119) were selected for clinical association study but rs7909119 was abandoned due to assay difficulties. SNP rs2357928 was found to have significant pharmacogenetic association in the case-control samples and therefore was genotyped in the rest of the INVEST-GENEs samples. Also genotyped in the INVEST GENES cohort samples was the newly reported hypertension/blood pressure GWAS SNP rs11014166, which was not found in our discovery effort due to its intronic location. Linkage disequilibrium between these two SNPs was minimal, with D’ and r^2^ of 0.12 and 0 in Caucasians, 0.20 and 0.02 in African Americans, 0.07 and 0.002 in Hispanics, respectively.

### Clinical Association Study

#### Study population and baseline characteristics

Baseline characteristics and past medical history for the INVEST-GENES participants are shown in [Table 1](#T1). Consistent with the overall INVEST participants, these hypertensive CAD patients were elderly (age: 66.1 ± 9.6 years), mostly overweight (BMI: 29.4 ± 5.5 kg/m^2^), racially diverse (41% Caucasians, 12% African Americans and 46% Hispanics), with slightly fewer men (45%) than women (55%). Patients in atenolol-based beta blocker (BB) and verapamil-based calcium channel blocker (CCB) treatment strategies were similar in baseline characteristic, medical history and non study medication use ([Table 1](#T1)).

| Characteristic | Case-control (n = 1,032) | Cohort (n = 5,598) | CCB (n = 2777) | BB (n = 2821) | p value† |
| --- | --- | --- | --- | --- | --- |
| Demographics |   |   |   |   |   |
| Age, mean (SD), years | 70.5 (9.5) | 66.1 (9.6) | 66.1 (9.6) | 66.2 (9.6) | 0.66 |
| Age > 70 | 566 (54.8) | 1902 (34.0) | 932 (33.6) | 970 (34.4) | 0.52 |
| Women | 524 (50.8) | 3090 (55.2) | 1537 (55.4) | 1553 (50.3) | 0.82 |
| Race/ethnicity |   |   |   |   | 0.56 |
| Caucasians | 616 (59.7) | 2315 (41.4) | 1174 (42.3) | 1141 (40.5) |   |
| African Americas | 131 (12.7) | 657 (11.7) | 323 (11.6) | 334 (11.8) |   |
| Hispanics | 261 (25.3) | 2589 (46.3) | 1261 (45.4) | 1328 (47.1) |   |
| Other/multiracial | 24 (2.3) | 37 (0.7) | 19 (0.7) | 18 (0.6) |   |
| BMI, mean (SD), kg/m2 | 28.6 (5.4) | 29.4 (5.5) | 29.3 (5.5) | 29.5 (5.6) | 0.14 |
| BP, mean (SD), mmHg |   |   |   |   |   |
| Systolic | 148.2 (19.0) | 148.0 (18.4) | 147.6 (18.6) | 148.4 (18.2) | 0.11 |
| Diastolic | 83.4 (11.1) | 85.4 (10.7) | 85.1 (10.8) | 85.7 (10.7) | 0.71 |
| Heart rate (beats per min) | 74.8 (9.3) | 74.8 (9.6) | 74.9 (9.6) | 74.7 (9.6) | 0.33 |
| Smoking history | 488 (47.3) | 2319 (41.4) | 1167 (42.0) | 1152 (40.8) | 0.37 |
| Past Medical History |   |   |   |   |   |
| Myocardial infarction | 326 (31.6) | 1317 (23.5) | 668 (24.1) | 649 (23.0) | 0.36 |
| Heart Failure (NYHC Class I-III) | 57 (5.5) | 193 (3.5) | 92 (3.3) | 101 (3.6) | 0.58 |
| Stroke or TIA | 107 (10.4) | 395 (7.1) | 222 (8.0) | 173 (6.1) | 0.0065 |
| Arrhythmia | 99 (9.6) | 187 (7.0) | 204 (7.4) | 187 (6.6) | 0.29 |
| Left Ventricular Hypertrophy | 182 (17.6) | 842 (15.0) | 401 (14.4) | 441 (15.6) | 0.21 |
| Peripheral Vascular Disease | 131 (12.7) | 620 (11.1) | 310 (11.2) | 310 (11.0) | 0.84 |
| Diabetes | 326 (31.6) | 1594 (28.5) | 782 (28.2) | 812 (28.8) | 0.6 |
| Hypercholoesterolemia | 646 (62.6) | 3068 (54.8) | 1536 (55.3) | 1532 (54.3) | 0.45 |
| Renal Insufficiency | 32 (3.1) | 88 (1.6) | 52 (1.9) | 36 (1.3) | 0.07 |
| Cancer | 66 (6.4) | 230 (4.1) | 125 (4.5) | 105 (3.7) | 0.14 |
| Medication |   |   |   |   |   |
| Aspirin/other antiplatelet agent | 613 (59.4) | 2590 (46.3) | 1291 (46.5) | 1299 (46.1) | 0.74 |
| Antidiabetic medication | 274 (26.5) | 1392 (24.9) | 666 (24.0) | 726 (25.7) | 0.13 |
| Lipid Lowering Drugs | 437 (42.3) | 2034 (36.3) | 1033 (37.2) | 1001 (35.5) | 0.18 |
| Nitrates | 324 (31.4) | 1568 (28.0) | 770 (27.7) | 798 (28.3) | 0.64 |

Table Caption: Baseline characteristics* for INVEST-GENES participants

### Genetic and pharmacogenetic association of CACNB2 polymorphism with primary outcome

Genotype data were complete for 1006 patients (97.5% of the case-control) for rs7069292, 1022 (99.0% of the case-control) for rs61839258, 5537 (98.9% of the cohort) for rs2357928 and 5460 (97.6% of the cohort) for rs11014166. All genotype frequencies were in Hardy-Weinberg Equilibrium in the three race/ethnicity groups. For all 4 SNPs, the genotype frequencies differed significantly by race/ethnicity ([Table 2](#T2)).

### Table 2.

|   | Caucasians | Hispanic | African Americans | p value for race |
| --- | --- | --- | --- | --- |
| rs2357928 | (n = 2284) | (n = 2563) | (n = 654) | < 0.0001 |
| A/A | 659 (28.9)* | 813 (31.7) | 296 (45.3) |   |
| A/G | 1124 (49.2) | 1261 (49.2) | 305 (46.6) |   |
| G/G | 501 (21.9) | 489 (19.1) | 53 (8.1) |   |
| G % | 46.5 | 43.7 | 31.4 |   |
|   |  |  |  |  |
| rs11014166 | (n = 2257) | (n = 2527) | (n = 642) | < 0.0001 |
| A/A | 955 (42.3) | 1195 (47.3) | 448 (69.8) |   |
| A/T | 1021 (45.2) | 1072 (42.4) | 175 (27.3) |   |
| T/T | 281 (12.4) | 260 (10.3) | 19 (3.0) |   |
| T % | 35.0 | 31.5 | 16.6 |   |
|   |  |  |  |  |
| rs7069292 | (n = 597) | (n = 257) | (n = 128) | 0.022 |
| T/T | 267 (44.7) | 120 (46.7) | 79 (61.7) |   |
| C/T | 273 (45.7) | 121 (47.1) | 42 (32.8) |   |
| C/C | 57 (9.6) | 16 (6.2) | 7 (5.5) |   |
| C % | 32.3 | 29.8 | 22 |   |
|   |  |  |  |  |
| rs61839258 | (n = 609) | (n = 259) | (n = 130) | 0.0004 |
| G/G | 422 (69.3) | 208 (80.3) | 112 (86.2) |   |
| T/G | 174 (28.6) | 49 (18.9) | 17 (13.1) |   |
| T/T | 13 (2.1) | 2 (0.8) | 1 (0.7) |   |
| T % | 16.1 | 10.2 | 07.7 |   |

Table 2 Caption: Genotype frequencies of rs2357928, rs11014166, rs7069292 and rs61839258 polymorphisms in INVEST-GENES participants.

SNP main effects of the 4 studied SNPs for the primary outcome are shown in [Table 3](#T3). With the exception of a borderline main effect for rs2357928 in African Americans, no main effect was observed. In African Americans, the minor allele (G) of rs2357928 was associated with lower risk for primary outcome regardless of treatment strategy, with adjusted hazard ratio (HR) of 0.45 (0.20-1.0) for each G allele, p = 0.049 (FDR- adjusted p = 0.29).

### Table 3.

|   | Caucasians |  |  | African Americans |  |  | Hispanics |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
|   | HR/OR§ | p value | SNP*Treatment p value | HR/OR | p value | SNP*Treatment p value | HR/OR | p value | SNP*Treatment p value |
| rs2357928 (G allele) | 0.72 (0.52-1.01) | 0.06 | 0.002* | 0.45 (0.20-1.0) | 0.049 | 0.32 | 0.58 (0.34-1.01) | 0.054 | 0.049 |
| rs11014166 (T allele) | 0.91 (0.57-1.45) | 0.68 | 0.4 | 1.03 (0.59-1.79) | 0.91 | 0.47 | 0.52 (0.24-1.13) | 0.097 | 0.16 |
| rs61839258 (T allele) | 1.13 (0.77-1.66) | 0.54 | 0.54 | 0.16 (0.03-1.04) | 0.055 | 0.98 | 1.36 (0.64-2.92) | 0.43 | 0.43 |
| rs7069292 (C allele) | 1.01 (0.74-1.38) | 0.97 | 0.24 | 0.75 (0.33-1.69) | 0.48 | 0.37 | 0.88 (0.51-1.52) | 0.65 | 0.46 |

Table 3 Caption: Adjusted hazard ratios/odds ratios and 95% confidence interval for primary outcome by race (adjusted for age, gender, history of MI, heart failure, ancestry) and the p-value for SNP*treatment (calcium channel blocker treatment or beta blocker treatment strategy) interaction. The alleles shown in parentheses are the minor alleles.

However in Caucasians, there was a significant interaction between rs2357928 and treatment strategy ([Table 3](#T3), p for interaction = 0.002, FDR-adjusted p=0.024). There was a marginally significant interaction between the treatment strategy and rs11014166 in Hispanics, with a p value of 0.049 (FDR-adjusted p=0.29). There was no evidence of SNP main effect or SNP*treatment interactions for rs61839258 and rs7069292 in any of the three race/ethnicity groups ([Table 3](#T3)).

#### Pharmacogenetic effect of rs2357928

For the SNPs with significant interaction with the treatment strategies, we performed analyses stratified by genotypes. In Caucasians, rs2357928 GG (minor allele homozygotes) patients randomized to CCB were more likely to experience an adverse outcome than those randomized to the BB treatment strategy, with adjusted HR (CCB vs. BB) of 2.35 (1.19-4.66), p=0.014 ([Figure 1](#F1)). There was no evidence for such treatment difference in AG (1.16, 0.75-1.79, p=0.69) and AA individuals (0.63, 0.36-1.11, p=0.11).

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/951e/3060561/e85920a0f9d0/nihms-272948-f0001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3060561_nihms-272948-f0001.jpg)

Pharmacogenetic association of rs2357928 and treatment strategies in three race/ethnicity groups. CCB: calcium channel blocker; BB: beta blocker; HR: Hazard ratios (shown in log scale); CI: confidence intervals.

Validation of this finding was provided through the data in Hispanics, where the same association was observed, with adjusted HR (CCB vs. BB) of 6.46 (1.23-33.93), p=0.028 in GG patients, 0.45 (0.11-1.85), p=0.27 for AG patients and 0.82 (0.35-1.88), p = 0.63 in AA patients, respectively ([Figure 1](#F1)). Since only one African American GG individual experienced the adverse cardiovascular outcome; HR for treatment strategy was not estimable. However, consistent with Caucasians and Hispanics, there was no evidence of treatment differences among AG and AA African American patients in terms of risk for primary outcome. These findings did not appear to be driven by a particular component of the primary outcome.

#### Pharmacogenetic effect of rs11014166

*CACNB2* rs11014166 showed marginally significant pharmacogenetic effect in Hispanics (p for interaction: 0.049). When stratified by genotype, for the minor allele (T) carriers, CCB treatment strategy was associated with higher risk for primary outcome than BB treatment strategy (adjusted HR, CCB vs. BB: 3.13, 1.39-7.06, p = 0.006, FDR-adjusted p=0.039). Such treatment difference was not observed for AA homozygotes (adjusted HR: 1.17, 0.61-2.24, p = 0.64). When the individual outcomes were evaluated, this pharmacogenetic effect was mainly driven by all-cause mortality (adjusted HR: 22.0, 2.63-184.17, p = 0.0043, FDR-adjusted p=0.028). There was no evidence of such pharmacogenetic association of rs11014166 in Caucasians and African Americans.

### Reporter gene assay

As shown in [Figure 2](#F2), promoter activity for rs2357928 (indicated by luciferase assay) of the minor allele G was significantly increased compared with allele A in CHO cells ([Figure 2A](#F2)) and HEK 293 cells ([Figure 2B](#F2)), with Wilcoxon rank sum test p value of 0.04 for both.

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/951e/3060561/e514c1893531/nihms-272948-f0002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3060561_nihms-272948-f0002.jpg)

Human CACNB2 reporter gene assay for rs2357928 in CHO cells (A) and HEK293 cells (B). The variant (G allele) of rs2357928 increased promoter activity of CACNB2 gene in CHO cells and HEK 293 and cells. Wilcoxon rank sum test p = 0.04 for both CHO cells and HEK 293 cells. The lines indicate the mean luciferase activity ratios.

## Discussion

*CACNB2* is a gene of increasing interest as it is one of the few that has recently been replicated as a hypertension gene. Here we report the common DNA sequence variations in the *CACNB2* coding region and flanking 5’ and 3’ UTR obtained from 20 EA and 20 AA and 20 NA samples. A total of 74 SNPs were identified in these *CACNB2* gene regions, of which 28% were novel SNPs not previously reported in dbSNP. In this study only 3 synonymous mutations were found in the last coding exon of *CACNB2* gene and we did not find any novel non-synonymous SNP in any of the three populations, indicating that *CACNB2* is highly conserved at the protein level. The comparison of the human *CACNB2* gene sequence with those of mouse and rat also showed strong conservation in the protein coding and 5’ UTR regions. Based on *in silico* functional analysis and minor allele frequencies, we genotyped three SNPs for study in our genetic association analysis, and later included the hypertension/SBP/DBP GWAS SNP (rs11014166) in our analysis.

In our clinical association study we identified one promoter SNP (rs2357928) in *CACNB2* that was associated with a significant SNP * treatment interaction in Caucasians, such that for minor allele (GG) homozygotes patients, verapamil SR-based CCB treatment was associated with significantly increased risk for the primary outcome compared to atenolol-based BB treatment strategy, while for AA and AG individuals, CCB and BB treatment strategies were equivalent. This effect was validated in Hispanics, while the GG genotype in African Americans was too infrequent to test. Power analyses indicated that we had greater than 80% power to detect HR of >1.8 in Caucasians GG individuals and >2.95 in Hispanics GG individuals.

The clinical association findings are further supported by *in vitro* functional studies that suggest rs2357928 may be a functional SNP. Specifically, the reporter assay data of promoter SNP rs2357928 showed a significant increase in luciferase activity for allele G compared to allele A in two different cell lines. These findings suggest this regulatory SNP, located within the second alternative promoter of the *CACNB2* gene, may affect basal transcriptional activation in this specific isoform.

The intronic GWAS SNP, rs11014166, which is in very low LD with the promoter SNP rs2357928, showed pharmacogenetic association in Hispanics only, with CCB treatment also associated with higher risk than BB for the minor allele (T) carriers, but not AA homozygotes. In the blood pressure and hypertension GWAS analysis of the CHARGE consortium,[7](#R7) the T allele of rs11014166 was associated with lower SBP, DBP and lower risk for hypertension. INVEST, a cardiovascular outcome study of hypertensive CAD patients, is not an appropriate replication cohort for the hypertension and BP association. Nevertheless, our data suggest this SNP may also be associated with adverse cardiovascular outcomes in hypertensive CAD patients in a treatment-specific manner.

Verapamil SR, like all marketed CCBs, binds directly to the α-subunit of L type calcium channel, which is modulated by the auxiliary β-subunit studied herein. Thus the validated association with rs2357928 including preliminary functional data suggest this gene (and SNP) deserve further study.

There are limitations worth noting in this study. First, the genetic association findings were not replicated in an independent sample of Caucasians. We did validate the finding in INVEST-GENES Hispanics, which is an independent sample from the Caucasians. However the number of events in Hispanic patients was small. This validation in a different ethnic group, along with the *in vitro* data that support a functional basis, suggest that rs2357928 is a functional SNP. Additionally, pharmacogenetic analyses conducted within a randomized controlled clinical trial, such as INVEST present clear advantages over cohort studies in that the randomization substantially reduces biases and confounding. The recent finding in the GWAS of the association of *CACNB2* with hypertension and blood pressure adds to the likelihood that this gene is very important in hypertension and hypertension related treatment outcomes.

While we did not observe this association in the African American samples, only one African American individual harboring the genotype with the most prominent effect (GG) experienced adverse cardiovascular outcome. However the pattern in heterozygotes and major allele homozygotes was consistent with Caucasians and Hispanics. So it is not clear whether the lack of association in this population is due to inadequate power, or an absence of genetic effect for this SNP in this population, or that the studied SNP is a tag SNP not in LD with the functional SNP in African Americans.

## Conclusions

Our association study suggested significant pharmacogenetic effects for a promoter SNP rs2357928 in *CACNB2,* such that for minor alleles homozygotes, treatment with verapamil SR-based CCB strategy was associated with substantially higher risk for adverse cardiovascular outcome as compared to an atenolol-based BB strategy. These findings were validated in a second ethnic group and further supported by *in vitro* studies suggesting differential transcriptional activity with this promoter SNP. Additional studies in other cohorts are required, but these data suggest this *CACNB2* SNP may have future potential for guiding selection of antihypertensive drug therapy among patients with CAD.

## Supplementary Material

## Acknowledgments

**Sources of Funding:** This project was funded by NIH grants HL074730, HL069758, GM074492 and RR017568, a grant from Abbott Laboratories and an Opportunity Fund Grant from the University of Florida.

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Lloyd-Jones D, Adams R, Carnethon M, De Simone G, Ferguson TB, Flegal K, Ford E, Furie K, Go A, Greenlund K, Haase N, Hailpern S, Ho M, Howard V, Kissela B, Kittner S, Lackland D, Lisabeth L, Marelli A, McDermott M, Meigs J, Mozaffarian D, Nichol G, O'Donnell C, Roger V, Rosamond W, Sacco R, Sorlie P, Stafford R, Steinberger J, Thom T, Wasserthiel-Smoller S, Wong N, Wylie-Rosett J, Hong Y. Heart disease and stroke statistics--2009 update: A report from the american heart association statistics committee and stroke statistics subcommittee. Circulation. 2009;119:480–486. doi: 10.1161/CIRCULATIONAHA.108.191259.  [DOI](https://doi.org/10.1161/CIRCULATIONAHA.108.191259) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19171871/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation&title=Heart%20disease%20and%20stroke%20statistics--2009%20update:%20A%20report%20from%20the%20american%20heart%20association%20statistics%20committee%20and%20stroke%20statistics%20subcommittee.&author=D%20Lloyd-Jones&author=R%20Adams&author=M%20Carnethon&author=G%20De%20Simone&author=TB%20Ferguson&volume=119&publication_year=2009&pages=480-486&pmid=19171871&doi=10.1161/CIRCULATIONAHA.108.191259&)

2. Jamerson K, Weber MA, Bakris GL, Dahlof B, Pitt B, Shi V, Hester A, Gupte J, Gatlin M, Velazquez EJ. Benazepril plus amlodipine or hydrochlorothiazide for hypertension in high-risk patients. N Engl J Med. 2008;359:2417–2428. doi: 10.1056/NEJMoa0806182.  [DOI](https://doi.org/10.1056/NEJMoa0806182) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19052124/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Benazepril%20plus%20amlodipine%20or%20hydrochlorothiazide%20for%20hypertension%20in%20high-risk%20patients.&author=K%20Jamerson&author=MA%20Weber&author=GL%20Bakris&author=B%20Dahlof&author=B%20Pitt&volume=359&publication_year=2008&pages=2417-2428&pmid=19052124&doi=10.1056/NEJMoa0806182&)

3. Turnbull F, Neal B, Algert C, Chalmers J, Chapman N, Cutler J, Woodward M, MacMahon S. Effects of different blood pressure-lowering regimens on major cardiovascular events in individuals with and without diabetes mellitus: Results of prospectively designed overviews of randomized trials. Arch Intern Med. 2005;165:1410–1419. doi: 10.1001/archinte.165.12.1410.  [DOI](https://doi.org/10.1001/archinte.165.12.1410) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15983291/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arch%20Intern%20Med&title=Effects%20of%20different%20blood%20pressure-lowering%20regimens%20on%20major%20cardiovascular%20events%20in%20individuals%20with%20and%20without%20diabetes%20mellitus:%20Results%20of%20prospectively%20designed%20overviews%20of%20randomized%20trials.&author=F%20Turnbull&author=B%20Neal&author=C%20Algert&author=J%20Chalmers&author=N%20Chapman&volume=165&publication_year=2005&pages=1410-1419&pmid=15983291&doi=10.1001/archinte.165.12.1410&)

4. Beitelshees AL, Gong Y, Wang D, Schork NJ, Cooper-Dehoff RM, Langaee TY, Shriver MD, Sadee W, Knot HJ, Pepine CJ, Johnson JA. Kcnmb1 genotype influences response to verapamil sr and adverse outcomes in the international verapamil sr/trandolapril study (invest). Pharmacogenet Genomics. 2007;17:719–729. doi: 10.1097/FPC.0b013e32810f2e3c.  [DOI](https://doi.org/10.1097/FPC.0b013e32810f2e3c) | [PMC free article](/articles/PMC2713584/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17700361/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Kcnmb1%20genotype%20influences%20response%20to%20verapamil%20sr%20and%20adverse%20outcomes%20in%20the%20international%20verapamil%20sr/trandolapril%20study%20(invest).&author=AL%20Beitelshees&author=Y%20Gong&author=D%20Wang&author=NJ%20Schork&author=RM%20Cooper-Dehoff&volume=17&publication_year=2007&pages=719-729&pmid=17700361&doi=10.1097/FPC.0b013e32810f2e3c&)

5. Langaee TY, Gong Y, Yarandi HN, Katz DA, Cooper-DeHoff RM, Pepine CJ, Johnson JA. Association of cyp3a5 polymorphisms with hypertension and antihypertensive response to verapamil. Clin Pharmacol Ther. 2007;81:386–391. doi: 10.1038/sj.clpt.6100090.  [DOI](https://doi.org/10.1038/sj.clpt.6100090) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17339868/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Association%20of%20cyp3a5%20polymorphisms%20with%20hypertension%20and%20antihypertensive%20response%20to%20verapamil.&author=TY%20Langaee&author=Y%20Gong&author=HN%20Yarandi&author=DA%20Katz&author=RM%20Cooper-DeHoff&volume=81&publication_year=2007&pages=386-391&pmid=17339868&doi=10.1038/sj.clpt.6100090&)

6. Arikkath J, Chen CC, Ahern C, Allamand V, Flanagan JD, Coronado R, Gregg RG, Campbell KP. Gamma 1 subunit interactions within the skeletal muscle l-type voltage-gated calcium channels. J Biol Chem. 2003;278:1212–1219. doi: 10.1074/jbc.M208689200.  [DOI](https://doi.org/10.1074/jbc.M208689200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12409298/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Gamma%201%20subunit%20interactions%20within%20the%20skeletal%20muscle%20l-type%20voltage-gated%20calcium%20channels.&author=J%20Arikkath&author=CC%20Chen&author=C%20Ahern&author=V%20Allamand&author=JD%20Flanagan&volume=278&publication_year=2003&pages=1212-1219&pmid=12409298&doi=10.1074/jbc.M208689200&)

7. Levy D, Ehret GB, Rice K, Verwoert GC, Launer LJ, Dehghan A, Glazer NL, Morrison AC, Johnson AD, Aspelund T, Aulchenko Y, Lumley T, Kottgen A, Vasan RS, Rivadeneira F, Eiriksdottir G, Guo X, Arking DE, Mitchell GF, Mattace-Raso FU, Smith AV, Taylor K, Scharpf RB, Hwang SJ, Sijbrands EJ, Bis J, Harris TB, Ganesh SK, O'Donnell CJ, Hofman A, Rotter JI, Coresh J, Benjamin EJ, Uitterlinden AG, Heiss G, Fox CS, Witteman JC, Boerwinkle E, Wang TJ, Gudnason V, Larson MG, Chakravarti A, Psaty BM, van Duijn CM. Genome-wide association study of blood pressure and hypertension. Nat Genet. 2009;41:677–687. doi: 10.1038/ng.384.  [DOI](https://doi.org/10.1038/ng.384) | [PMC free article](/articles/PMC2998712/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19430479/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Genome-wide%20association%20study%20of%20blood%20pressure%20and%20hypertension.&author=D%20Levy&author=GB%20Ehret&author=K%20Rice&author=GC%20Verwoert&author=LJ%20Launer&volume=41&publication_year=2009&pages=677-687&pmid=19430479&doi=10.1038/ng.384&)

8. Collins FS, Brooks LD, Chakravarti A. A DNA polymorphism discovery resource for research on human genetic variation. Genome Res. 1998;8:1229–1231. doi: 10.1101/gr.8.12.1229.  [DOI](https://doi.org/10.1101/gr.8.12.1229) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9872978/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res&title=A%20DNA%20polymorphism%20discovery%20resource%20for%20research%20on%20human%20genetic%20variation.&author=FS%20Collins&author=LD%20Brooks&author=A%20Chakravarti&volume=8&publication_year=1998&pages=1229-1231&pmid=9872978&doi=10.1101/gr.8.12.1229&)

9. Nickerson DA, Tobe VO, Taylor SL. Polyphred: Automating the detection and genotyping of single nucleotide substitutions using fluorescence-based resequencing. Nucleic Acids Res. 1997;25:2745–2751. doi: 10.1093/nar/25.14.2745.  [DOI](https://doi.org/10.1093/nar/25.14.2745) | [PMC free article](/articles/PMC146817/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9207020/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=Polyphred:%20Automating%20the%20detection%20and%20genotyping%20of%20single%20nucleotide%20substitutions%20using%20fluorescence-based%20resequencing.&author=DA%20Nickerson&author=VO%20Tobe&author=SL%20Taylor&volume=25&publication_year=1997&pages=2745-2751&pmid=9207020&doi=10.1093/nar/25.14.2745&)

10. Freimuth RR, Stormo GD, McLeod HL. Polymapr: Programs for polymorphism database mining, annotation, and functional analysis. Hum Mutat. 2005;25:110–117. doi: 10.1002/humu.20123.  [DOI](https://doi.org/10.1002/humu.20123) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15643605/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mutat&title=Polymapr:%20Programs%20for%20polymorphism%20database%20mining,%20annotation,%20and%20functional%20analysis.&author=RR%20Freimuth&author=GD%20Stormo&author=HL%20McLeod&volume=25&publication_year=2005&pages=110-117&pmid=15643605&doi=10.1002/humu.20123&)

11. Frazer KA, Pachter L, Poliakov A, Rubin EM, Dubchak I. Vista: Computational tools for comparative genomics. Nucleic Acids Res. 2004;32:W273–279. doi: 10.1093/nar/gkh458.  [DOI](https://doi.org/10.1093/nar/gkh458) | [PMC free article](/articles/PMC441596/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15215394/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=Vista:%20Computational%20tools%20for%20comparative%20genomics.&author=KA%20Frazer&author=L%20Pachter&author=A%20Poliakov&author=EM%20Rubin&author=I%20Dubchak&volume=32&publication_year=2004&pages=W273-279&pmid=15215394&doi=10.1093/nar/gkh458&)

12. Mayor C, Brudno M, Schwartz JR, Poliakov A, Rubin EM, Frazer KA, Pachter LS, Dubchak I. Vista : Visualizing global DNA sequence alignments of arbitrary length. Bioinformatics. 2000;16:1046–1047. doi: 10.1093/bioinformatics/16.11.1046.  [DOI](https://doi.org/10.1093/bioinformatics/16.11.1046) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11159318/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=Vista%20:%20Visualizing%20global%20DNA%20sequence%20alignments%20of%20arbitrary%20length.&author=C%20Mayor&author=M%20Brudno&author=JR%20Schwartz&author=A%20Poliakov&author=EM%20Rubin&volume=16&publication_year=2000&pages=1046-1047&pmid=11159318&doi=10.1093/bioinformatics/16.11.1046&)

13. Brudno M, Do CB, Cooper GM, Kim MF, Davydov E, Green ED, Sidow A, Batzoglou S. Lagan and multi-lagan: Efficient tools for large-scale multiple alignment of genomic DNA. Genome Res. 2003;13:721–731. doi: 10.1101/gr.926603.  [DOI](https://doi.org/10.1101/gr.926603) | [PMC free article](/articles/PMC430158/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12654723/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res&title=Lagan%20and%20multi-lagan:%20Efficient%20tools%20for%20large-scale%20multiple%20alignment%20of%20genomic%20DNA.&author=M%20Brudno&author=CB%20Do&author=GM%20Cooper&author=MF%20Kim&author=E%20Davydov&volume=13&publication_year=2003&pages=721-731&pmid=12654723&doi=10.1101/gr.926603&)

14. Pepine CJ, Handberg EM, Cooper-DeHoff RM, Marks RG, Kowey P, Messerli FH, Mancia G, Cangiano JL, Garcia-Barreto D, Keltai M, Erdine S, Bristol HA, Kolb HR, Bakris GL, Cohen JD, Parmley WW. A calcium antagonist vs a non-calcium antagonist hypertension treatment strategy for patients with coronary artery disease. The international verapamil-trandolapril study (invest): A randomized controlled trial. JAMA. 2003;290:2805–2816. doi: 10.1001/jama.290.21.2805.  [DOI](https://doi.org/10.1001/jama.290.21.2805) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14657064/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA&title=A%20calcium%20antagonist%20vs%20a%20non-calcium%20antagonist%20hypertension%20treatment%20strategy%20for%20patients%20with%20coronary%20artery%20disease.%20The%20international%20verapamil-trandolapril%20study%20(invest):%20A%20randomized%20controlled%20trial.&author=CJ%20Pepine&author=EM%20Handberg&author=RM%20Cooper-DeHoff&author=RG%20Marks&author=P%20Kowey&volume=290&publication_year=2003&pages=2805-2816&pmid=14657064&doi=10.1001/jama.290.21.2805&)

15. Pepine CJ, Handberg-Thurmond E, Marks RG, Conlon M, Cooper-DeHoff R, Volkers P, Zellig P. Rationale and design of the international verapamil sr/trandolapril study (invest): An internet-based randomized trial in coronary artery disease patients with hypertension. J Am Coll Cardiol. 1998;32:1228–1237. doi: 10.1016/s0735-1097(98)00423-9.  [DOI](https://doi.org/10.1016/s0735-1097(98)00423-9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9809930/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Am%20Coll%20Cardiol&title=Rationale%20and%20design%20of%20the%20international%20verapamil%20sr/trandolapril%20study%20(invest):%20An%20internet-based%20randomized%20trial%20in%20coronary%20artery%20disease%20patients%20with%20hypertension.&author=CJ%20Pepine&author=E%20Handberg-Thurmond&author=RG%20Marks&author=M%20Conlon&author=R%20Cooper-DeHoff&volume=32&publication_year=1998&pages=1228-1237&pmid=9809930&doi=10.1016/s0735-1097(98)00423-9&)

16. Kaplan NM. The 6th joint national committee report (jnc-6): New guidelines for hypertension therapy from the USA. Keio J Med. 1998;47:99–105. doi: 10.2302/kjm.47.99.  [DOI](https://doi.org/10.2302/kjm.47.99) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9659820/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Keio%20J%20Med&title=The%206th%20joint%20national%20committee%20report%20(jnc-6):%20New%20guidelines%20for%20hypertension%20therapy%20from%20the%20USA.&author=NM%20Kaplan&volume=47&publication_year=1998&pages=99-105&pmid=9659820&doi=10.2302/kjm.47.99&)

17. Andrisin TE, Humma LM, Johnson JA. Collection of genomic DNA by the noninvasive mouthwash method for use in pharmacogenetic studies. Pharmacotherapy. 2002;22:954–960. doi: 10.1592/phco.22.12.954.33598.  [DOI](https://doi.org/10.1592/phco.22.12.954.33598) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12173798/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacotherapy&title=Collection%20of%20genomic%20DNA%20by%20the%20noninvasive%20mouthwash%20method%20for%20use%20in%20pharmacogenetic%20studies.&author=TE%20Andrisin&author=LM%20Humma&author=JA%20Johnson&volume=22&publication_year=2002&pages=954-960&pmid=12173798&doi=10.1592/phco.22.12.954.33598&)

18. Langaee T, Ronaghi M. Genetic variation analyses by pyrosequencing. Mutat Res. 2005;573:96–102. doi: 10.1016/j.mrfmmm.2004.07.023.  [DOI](https://doi.org/10.1016/j.mrfmmm.2004.07.023) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15829240/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mutat%20Res&title=Genetic%20variation%20analyses%20by%20pyrosequencing.&author=T%20Langaee&author=M%20Ronaghi&volume=573&publication_year=2005&pages=96-102&pmid=15829240&doi=10.1016/j.mrfmmm.2004.07.023&)

19. Zhao LY, Liu J, Sidhu GS, Niu Y, Liu Y, Wang R, Liao D. Negative regulation of p53 functions by daxx and the involvement of mdm2. J Biol Chem. 2004;279:50566–50579. doi: 10.1074/jbc.M406743200.  [DOI](https://doi.org/10.1074/jbc.M406743200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15364927/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Negative%20regulation%20of%20p53%20functions%20by%20daxx%20and%20the%20involvement%20of%20mdm2.&author=LY%20Zhao&author=J%20Liu&author=GS%20Sidhu&author=Y%20Niu&author=Y%20Liu&volume=279&publication_year=2004&pages=50566-50579&pmid=15364927&doi=10.1074/jbc.M406743200&)

20. Barrett JC, Fry B, Maller J, Daly MJ. Haploview: Analysis and visualization of ld and haplotype maps. Bioinformatics. 2005;21:263–265. doi: 10.1093/bioinformatics/bth457.  [DOI](https://doi.org/10.1093/bioinformatics/bth457) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15297300/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=Haploview:%20Analysis%20and%20visualization%20of%20ld%20and%20haplotype%20maps.&author=JC%20Barrett&author=B%20Fry&author=J%20Maller&author=MJ%20Daly&volume=21&publication_year=2005&pages=263-265&pmid=15297300&doi=10.1093/bioinformatics/bth457&)

21. Shriver MD, Mei R, Parra EJ, Sonpar V, Halder I, Tishkoff SA, Schurr TG, Zhadanov SI, Osipova LP, Brutsaert TD, Friedlaender J, Jorde LB, Watkins WS, Bamshad MJ, Gutierrez G, Loi H, Matsuzaki H, Kittles RA, Argyropoulos G, Fernandez JR, Akey JM, Jones KW. Large-scale snp analysis reveals clustered and continuous patterns of human genetic variation. Hum Genomics. 2005;2:81–89. doi: 10.1186/1479-7364-2-2-81.  [DOI](https://doi.org/10.1186/1479-7364-2-2-81) | [PMC free article](/articles/PMC3525270/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16004724/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Genomics&title=Large-scale%20snp%20analysis%20reveals%20clustered%20and%20continuous%20patterns%20of%20human%20genetic%20variation.&author=MD%20Shriver&author=R%20Mei&author=EJ%20Parra&author=V%20Sonpar&author=I%20Halder&volume=2&publication_year=2005&pages=81-89&pmid=16004724&doi=10.1186/1479-7364-2-2-81&)

22. Myakishev MV, Khripin Y, Hu S, Hamer DH. High-throughput snp genotyping by allele-specific pcr with universal energy-transfer-labeled primers. Genome Res. 2001;11:163–169. doi: 10.1101/gr.157901.  [DOI](https://doi.org/10.1101/gr.157901) | [PMC free article](/articles/PMC311033/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11156625/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res&title=High-throughput%20snp%20genotyping%20by%20allele-specific%20pcr%20with%20universal%20energy-transfer-labeled%20primers.&author=MV%20Myakishev&author=Y%20Khripin&author=S%20Hu&author=DH%20Hamer&volume=11&publication_year=2001&pages=163-169&pmid=11156625&doi=10.1101/gr.157901&)

23. Pepine CJ, Kowey PR, Kupfer S, Kolloch RE, Benetos A, Mancia G, Coca A, Cooper-DeHoff RM, Handberg E, Gaxiola E, Sleight P, Conti CR, Hewkin AC, Tavazzi L. Predictors of adverse outcome among patients with hypertension and coronary artery disease. J Am Coll Cardiol. 2006;47:547–551. doi: 10.1016/j.jacc.2005.09.031.  [DOI](https://doi.org/10.1016/j.jacc.2005.09.031) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16458134/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Am%20Coll%20Cardiol&title=Predictors%20of%20adverse%20outcome%20among%20patients%20with%20hypertension%20and%20coronary%20artery%20disease.&author=CJ%20Pepine&author=PR%20Kowey&author=S%20Kupfer&author=RE%20Kolloch&author=A%20Benetos&volume=47&publication_year=2006&pages=547-551&pmid=16458134&doi=10.1016/j.jacc.2005.09.031&)

24. SCHOENFELD D. Partial residuals for the proportional hazards regression model. Biometrika. 1982;69:239–241.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biometrika&title=Partial%20residuals%20for%20the%20proportional%20hazards%20regression%20model.&author=D%20SCHOENFELD&volume=69&publication_year=1982&pages=239-241&)

25. Benjamini Y. HY. Controlling the false discovery rate: A practical and powerful approach to multiple testing. J Royal Statist Soc Ser B. 1995;57:289–300.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Royal%20Statist%20Soc%20Ser%20B&title=HY.%20Controlling%20the%20false%20discovery%20rate:%20A%20practical%20and%20powerful%20approach%20to%20multiple%20testing.&author=Y%20Benjamini&volume=57&publication_year=1995&pages=289-300&)
