# Multiple regulatory variants modulate expression of 5-hydroxytryptamine 2A receptors in human cortex

## Metadata
**Authors:** Ryan M Smith, Audrey C Papp, Amy Webb, Cara L Ruble, Leanne M Munsie, Laura K Nisenbaum, Joel E Kleinman, Barbara K Lipska, Wolfgang Sadee
**Journal:** Biological psychiatry
**Date:** 2012 Nov 13
**DOI:** [10.1016/j.biopsych.2012.09.028](https://doi.org/10.1016/j.biopsych.2012.09.028)
**PMID:** 23158458
**PMCID:** PMC3582836
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3582836/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC3582836/pdf/nihms413844.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC3582836/pdf/nihms413844.pdf)

## Abstract

**Background:** 
The 5-hydroxytryptamine 2A receptor, encoded by HTR2A, is a major post-synaptic target for serotonin in the human brain and a therapeutic drug target. Despite hundreds of genetic associations investigating HTR2A polymorphisms in neuropsychiatric disorders and therapies, the role of genetic HTR2A variability in health and disease remains uncertain.

**Methods:** 
To discover and characterize regulatory HTR2A variants, we sequenced whole transcriptomes from ten human brain regions with massively-parallel RNA sequencing and measured allelic expression of multiple HTR2A mRNA transcript variants. Following discovery of functional variants, we further characterized their impact on genetic expression in vitro.

**Results:** 
Three polymorphisms modulate the use of novel alternative exons and untranslated regions (UTRs), changing expression of RNA and protein. The frequent promoter variant rs6311, widely implicated in human neuropsychiatric disorders, decreases usage of an upstream transcription start site encoding a longer 5′UTR with greater translation efficiency. rs76665058, located in an extended 3′UTR and unique to individuals of African descent, modulates allelic HTR2A mRNA expression. The third SNP, unannotated and present in only a single subject, directs alternative splicing of exon 2. Targeted analysis of HTR2A in the Sequenced Treatment Alternatives to Relieve Depression (STAR*D) study reveals associations between functional variants and depression severity or citalopram response.

**Conclusions:** 
Regulatory polymorphisms modulate HTR2A mRNA expression in an isoform-specific manner, directing the usage of novel untranslated regions and alternative exons. These results provide a foundation for delineating the role of HTR2A and serotonin signaling in CNS disorders.

Keywords: serotonin, 5-HT2A, HTR2A, schizophrenia, depression, mRNA expression

### Background

The 5-hydroxytryptamine 2A receptor, encoded by *HTR2A*, is a major post-synaptic target for serotonin in the human brain and a therapeutic drug target. Despite hundreds of genetic associations investigating *HTR2A* polymorphisms in neuropsychiatric disorders and therapies, the role of genetic *HTR2A* variability in health and disease remains uncertain.

### Methods

To discover and characterize regulatory *HTR2A* variants, we sequenced whole transcriptomes from ten human brain regions with massively-parallel RNA sequencing and measured allelic expression of multiple *HTR2A* mRNA transcript variants. Following discovery of functional variants, we further characterized their impact on genetic expression *in vitro*.

### Results

Three polymorphisms modulate the use of novel alternative exons and untranslated regions (UTRs), changing expression of RNA and protein. The frequent promoter variant rs6311, widely implicated in human neuropsychiatric disorders, decreases usage of an upstream transcription start site encoding a longer 5′UTR with greater translation efficiency. rs76665058, located in an extended 3′UTR and unique to individuals of African descent, modulates allelic *HTR2A* mRNA expression. The third SNP, unannotated and present in only a single subject, directs alternative splicing of exon 2. Targeted analysis of *HTR2A* in the Sequenced Treatment Alternatives to Relieve Depression (STAR*D) study reveals associations between functional variants and depression severity or citalopram response.

### Conclusions

Regulatory polymorphisms modulate *HTR2A* mRNA expression in an isoform-specific manner, directing the usage of novel untranslated regions and alternative exons. These results provide a foundation for delineating the role of *HTR2A* and serotonin signaling in CNS disorders.

## Introduction

The 5-hydroxytryptamine 2A receptor (5-HT2A), encoded by *HTR2A*, is a widely-distributed post-synaptic target for serotonin (5-HT) in the human brain. The Genetic Association Database ([1](#R1)) reports 346 unique association studies between single nucleotide polymorphisms (SNPs) in *HTR2A* and human phenotypes. One hundred eighty-three (53%) of these studies find positive genotype-phenotype associations. Most are related to cognition or risk for neuropsychiatric disorders, supporting the presence of functional genetic variants in *HTR2A*, although many other studies fail to find associations ([2](#R2)). *HTR2A* variants also modulate drug response ([2](#R2)), as 5-HT2A is a target for atypical antipsychotics and antidepressants. Despite positive clinical associations and billions of dollars spent annually on drugs modulating 5-HT2A signaling ([3](#R3)), the role of genetic variants remain unclear.

Two widely studied SNPs in *HTR2A*, rs6311 (-1438G>A) and rs6313 (102C>T), are in high linkage disequilibrium (LD), do not change the encoded protein, and are used interchangeably in genetic association studies. rs6311, located in the *HTR2A* gene promoter, is suspected to yield positive clinical associations via changes in mRNA expression. Some studies find significant correlations between rs6311 or rs6313 genotype and mRNA or protein expression ([4](#R4)–[6](#R6)), although other *in vivo* or *ex vivo* studies contradict these findings ([7](#R7)–[13](#R13)), perhaps owing to environmental factors that alter *HTR2A* expression ([14](#R14)–[18](#R18)). Specifically, negative results from allelic *HTR2A* mRNA measurements argue against the presence of *cis*-acting SNPs modulating *HTR2A* mRNA expression ([7](#R7),[9](#R9)). A commonly studied non-synonymous SNP (rs6314, His452Tyr) unrelated in to mRNA expression is proposed to affect drug-related signaling ([19](#R19),[20](#R20)) and human memory ([2](#R2),[21](#R21)). Taken together, a clinical role for any of these variants remains equivocal.

Most disorders implicating *HTR2A* are complex genetic disorders; yet, known genetic variants account for only a small portion of the estimated disease risk or treatment outcome, leaving “missing heritability” ([22](#R22)). Full characterization of genetic variants with functional consequences in key risk genes is a critical step towards resolving missing heritability. Testing the hypothesis that *HTR2A* harbors regulatory genetic variants, we surveyed *HTR2A* mRNA expression in human brain, revealing novel alternative exons and untranslated regions (UTRs), which are modulated by common and rare functional SNPs that significantly associate with depression risk and SSRI treatment in the STAR*D Study ([23](#R23),[24](#R24)).

## Materials and Methods

### cDNA library construction

Demographics for post-mortem human tissues are listed in [Supplemental Table S1](#SD1). Samples with RNA Integrity Number (RIN) <5 (Agilent 2100 BioAnalyzer analysis, Agilent Technologies, Inc., Santa Clara, CA) were excluded from study. In total, 75 dorsolateral prefrontal cortex (BA46) samples from different individuals (cocaine abusers and controls) and nine additional brain regions (frontopolar cortex (BA10), Wernicke’s Area (BA22), ventral anterior cingulate cortex (BA24), insular cortex, amygdala, hippocampus, putamen, cerebellum, and pontine raphe nuclei) originating from the same individual were used in this study. Sample MB085 was excluded from group expression analyses, as it harbors a rare variant significantly affecting mRNA expression, described below. Genomic DNA (gDNA) and total RNA was isolated, as previously described ([25](#R25)). cDNA for transcriptome sequencing was reverse transcribed from 10ng of total RNA using the Ovation RNA-Seq System (NuGEN Technologies, Inc., San Carlos, CA). For all other *ex vivo* brain-related experiments, 500ng of total RNA was primed for reverse transcription with gene-specific primers ([Table S2](#SD1)) plus oligo-dT using SuperScript III (Life Technologies, Grand Island, NY).

### Massively parallel sequencing

Ten PFC BA46 Ovation RNA-Seq libraries were sequenced by SOLiD 4 Next-Generation Sequencing (Life Technologies), while libraries for the nine additional brain regions were sequenced on the 5500 SOLiD System (Life Technologies). Sequenced reads were mapped to NCBI Build 37/hg19 of the human genome using LifeScope Genomic Analysis Solutions software (Life Technologies), producing RPKM (Reads Per Kilobase of exon model per Million mapped reads) values for normalized mRNA expression. Mapped reads were visualized with the Broad Institute Integrative Genomics Viewer ([27](#R27)).

### Quantitative PCR (qPCR) and allelic mRNA expression

Gene expression was measured in triplicate in all 74 BA46 PFC gene-specific cDNA libraries via qPCR using primers listed in [Table S2](#SD1), as previously described ([25](#R25)). Step-wise linear regression was performed for C_T_ values, normalized to β-actin (*ACTB*), against *HTR2A* genotypes ([Figure 1](#F1)). Allelic expression imbalance (AEI) was measured, in duplicate for gDNA and triplicate for cDNA, in samples heterozygous for marker SNPs using SNaPshot ([28](#R28)). Significant AEI was defined as allelic differences greater than 2 standard deviations of the average within-sample error for the assay.

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2db6/3582836/84b366ccf36a/nihms413844f1.jpg)

Gene map of the HTR2A region and transcripts identified in the current study. Annotated SNPs were included in eQTL analyses. Alternative splicing regulates the usage of intron1/exon2 to create four splice isoforms (E2+, E2+(I1R), E2tr, and E2−). These transcripts can utilize multiple transcription start sites (arrows on E2+ 5′UTR) or alternative polyadenylation signals in the 3′UTR.

### In vitro characterization of rs6311 and 5′UTRs

Five chimeric 5′UTR-luciferase constructs were used for *in vitro* characterizations (no UTR, short: sUTR, intermediate: mUTR, and long with either rs6311 alleles: lUTR-rs6311/G and lUTR-rs6311/A; [supplemental methods](#SD1)). To measure allelic expression, lUTR-rs6311/G and lUTR-rs6311/A were co-transfected in triplicate (800ng/well of 12-well cell culture plates) using Lipofectamine 2000 (Life Technologies), along with no transfection controls. Antibiotic-treated media was added after 6 hours, to prevent infection. RNA and plasmid DNA was isolated with TRIzol (Life Technologies) by chloroform extraction and precipitation with isopropanol after 48 hours. mRNA AEI ratios were normalized to transfected plasmid DNA allele ratios.

To measure translation efficiency, each of the five 5′UTR luciferase constructs were transfected in duplicate (750ng/well) using Lipofectamine LTX with Plus Reagent (Life Technologies), including no-transfection controls. After 48 hours, protein and RNA were harvested using the Ambion PARIS system (Life Technologies). For each transfection, equal volumes of purified total cell RNA lysate (10ul) were used for cDNA synthesis and subsequent quantification of luciferase mRNA expression via qPCR, measured in triplicate. For luciferase protein expression, 25ul of purified protein lysate and 25ul of Dual-Glo Luciferase substrate (Promega) were added to a flat-bottom 96-well microplate, and fluorescent intensity was measured in triplicate by a Fusion fluorescent plate reader (PerkinElmer, Inc., Waltham, MA). Fluorescent signal intensity was divided by a transformed C_T_ value for the corresponding sample, yielding a luciferase protein activity measurement that is normalized to luciferase mRNA expression, reported as percentage of the no UTR construct ([Table S3](#SD1)).

### In vitro immunohistochemistry of HTR2A isoforms

Details are provided in [supplemental materials](#SD1).

### CpG methylation at the HTR2A locus in human prefrontal cortex

Details are provided in [supplemental materials](#SD1) and ([33](#R33)).

### Clinical associations with STAR*D

Details for the STAR*D study are published elsewhere ([23](#R23),[24](#R24); [Table S4](#SD1)). Covariates and clinical outcomes tested for genetic associations are listed in the [supplemental materials](#SD1). Step-wise linear regression or logistic regression was performed for demographic variables against each outcome variable separately and statistically significant covariates were included in subsequent analyses. SPSS v19.0 (IBM Corporation, Armonk, NY) was used for all statistical analyses.

## Results

### HTR2A mRNA characterization

All 10 BA46 PFC whole transcriptome samples exhibited coverage extending 1kb upstream of the NCBI *HTR2A* reference sequence ([NM_000621.4](https://www.ncbi.nlm.nih.gov/nuccore/NM_000621.4); GI:377520130) ([Supplemental Figure S1A](#SD1)). On average, coverage for this upstream region was 23% and 29% of the read depth observed for exon 1 and total *HTR2A* exonic depth, respectively. Transcriptome analysis also indicates that BA10, BA22, BA24, insular cortex, and putamen samples also express this upstream region. Expression of the upstream region is not correlated with total *HTR2A* expression, both measured with qPCR, in the BA46 tissue cohort (*n*=74, *r2*=0.01, *p*=0.365). However, PCR amplification of cDNA confirms that the upstream region represents an unspliced extension of the annotated 53UTR, with some mRNAs retaining intron 1 (E2^+(I1R)^) ([Figures S1B and S1C](#SD1)). Transcripts expressing the upstream 5′UTR constitute approximately 10% of total *HTR2A* mRNA in BA46 ([Table 1](#T1)). Differences in expression between the extended 5′UTR region and total *HTR2A* mRNA suggests the presence of an independent TSS with distinct regulatory control. We also detected low expression immediately downstream of the canonical polyadenylation signal (AATAAA) at the 3′ terminal end of *HTR2A*, possibly continuing 417 bases to a non-canonical polyadenylation signal (ATTAAA). This distal portion constitutes approximately 1% of the total transcript pool ([Table 1](#T1)) and is significantly correlated with exon 4 expression (*n*=74, *r2*=0.24, *p*=8.1×10^−6^).

### Table 1.

| Transcript | Percent of Total | S.D. |
| --- | --- | --- |
| E2+ (Total)a | - | 0.9 |
| Extended 5′UTR | 9.51 | 2.1 |
| E2+I1R | <5b | -c |
| E2tr | 0.05 | 3.6 |
| E2− | 0.18 | 2.3 |
| 3′UTR | 115 | 2.0 |
| Extended 3′UTR | 0.71 | 2.6 |
| Antisense | <1.0×10−9 | -c |

Table 1 Caption: Transcript Expression Levels in PFC (Relative to Total HTR2A)

*HTR2A* encodes a known alternatively spliced isoform lacking exon 2 ([NM_001165947.2](https://www.ncbi.nlm.nih.gov/nuccore/NM_001165947.2); GI:377520131; E2^−^). Targeted PCR amplification of exons 1–3 in cDNA confirmed the presence of E2^−^ in PFC and also uncovered a novel splice isoform generated from a splice acceptor site 544 nucleotides inside exon 2, yielding a truncated exon 2 (E2^tr^) ([Figure S2A](#SD1)). These two minor splice variants are also expressed in most other brain regions surveyed, although to varying degrees relative to the full-length (E2^+^) isoform ([Figure S2B](#SD1), [Table 1](#T1)). The genomic sequence adjacent to the E2^tr^ splice acceptor site encodes *cis-*regulatory components necessary for splicing, including a branch point and polypyrimidine tract ([Figure S2C](#SD1)). Transcripts identified in this study are displayed in [Figure 1](#F1), including the low-abundance nested antisense transcript in intron 3 ([34](#R34)) (expression levels in [Table 1](#T1)).

### Expression quantitative trait locus (eQTL) analysis of HTR2A mRNA

Step-wise linear regression performed with 74 PFCs between fifteen SNPs genotyped in *HTR2A* ([Figure 1](#F1)) and transcript expression measured with qPCR revealed a significant relationship between rs6311 and extended 5′UTR expression (*n*=74, *F*=14.08, *p*=7.1×10^−6^), when RNA Integrity Number (RIN) and cocaine condition were considered as significant covariates. rs6311 also significantly predicted E2^tr^ splice isoform expression (*n*=74, *F*=6.64, *p*=0.002) when cocaine condition was considered a significant covariate, and rs76665058 significantly predicted E2^−^ splice isoform expression (*n*=74, *F*=7.80, *p*=0.007).

To detect *cis*-eQTLs with greater sensitivity, we measured AEI in all PFC samples heterozygous for one or more of nine marker SNPs (59 of 74 samples) present in any mature mRNA transcript (rs1328685, rs6311, rs6312, rs6313, rs6314, rs3803189, rs7324017, rs73473857, rs61948307, and rs76665058; [Figure 1](#F1)). None of the SNPs residing in the major mRNA isoform (rs6313, rs6314, rs3803189, rs7324017, rs73473857, and rs61948307) displayed AEI significantly deviating from unity (average AEI ratio=1.06-fold ± 0.05).

Consistent with linear regression, significant AEI was observed for two SNPs in the extended 5′UTR, rs1328685 and rs6311. 10 of 15 samples heterozygous for rs1328685 displayed significant AEI (1.4 to 2.0-fold), while all 32 samples heterozygous for rs6311 showed significant AEI ranging from 1.3 to 2.2-fold allelic differences ([Figure 2A](#F2)). All samples displaying significant AEI for rs1328685 were also heterozygous for rs6311, and AEI ratios across the two SNPs were significantly correlated (*n*=10, *r2*=0.52, *p*=0.019). Only rs6311 fully accounted for the allelic expression phenotype (heterozygous in all AEI positive and homozygous in all AEI negative samples), with all other SNPs in high LD failing to fully match the AEI phenotype. Univariate ANOVA for 5′UTR mRNA expression (using RIN as a covariate) confirmed a significant effect of rs6311 genotype (*n*=74, *F*=6.76, *p*=0.002; [Figure 2B](#F2)), whereby tissues homozygous for the major rs6311/G expressed 2.5-fold more extended 5′UTR than homozygous minor rs6311/A tissues. These results show that the minor rs6311/A allele reduces extended 5′UTR expression.

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2db6/3582836/227b1a2bc1d6/nihms413844f2.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3582836_nihms413844f2.jpg)

AEI in the extended 5′UTR. A. All samples heterozygous for rs6311 (asterisks) display significant AEI, as determined by allelic differences >2 S.D. of the within-sample assay error (dotted lines). Ten of 15 samples heterozygous for rs1328685 show significant AEI. All samples where AEI is observed for rs1328685, AEI is also observed for rs6311 and the magnitude of AEI across SNPs is highly correlated (r2=0.52, p=0.019). B. Samples homozygous for the minor A/A allele of rs6311 express 2.5-fold less of the upstream 5′UTR relative to homozygous G/G samples. *p=0.002 C. Neuron-like SH-SY5Y cells show 1.7-fold AEI, similar to the PFC, while AEI is much attenuated in HEK293T cells. All data represented as mean ± S.D.

Allelic mRNA expression for splice variants lacking exon 2, measured in 14 samples heterozygous for rs6312 using primers specific for each splice variant, revealed a significant >2-fold AEI ratio in only one sample for E2^−^ and E2^tr^ isoforms (tissue MB085; [Figure S3A](#SD1)). Sequencing from exons 1–3 revealed a C>G SNP 650bp upstream of exon 3 (chr13:47,467,375 in GRCh37/hg19, no assigned rs#) as a candidate for explaining allelic splicing differences. *In silico* sequence analysis of this region using SpliceAid ([35](#R35)) shows that the “G” allele creates binding sites for the serine/arginine rich alternative splicing proteins SRSF1 (SF2/ASF) and SRSF5 (SRp40), supporting a role in alternative splicing. We sequenced additional samples but did not identify another carrier of this intronic SNP. No other AEI was observed for exon 2 splice variants, discounting rs6311 as a *cis*-acting contributor to E2^tr^ expression, as suggested by regression analysis.

Allelic mRNA measurements in the extended 3′UTR revealed significant AEI in all four samples heterozygous for rs76665058, located immediately downstream of the canonical polyadenylation signal. The minor “G” allele expressed 1.6 to 2.7-fold more mRNA than the major “A” allele ([Figure S3B](#SD1)). rs76665058 is only observed in individuals of African descent (Minor Allele Frequency (MAF)=0.059 in HapMap Yoruba in Ibadan, Nigeria (YRI) samples). Sanger sequencing of exon 4 through the entire 3′UTR region revealed no other suitable candidates accounting for allelic differences, leaving rs76665058 as the parsimonious *cis*-eQTL candidate. Linear regression suggests rs76665058 significantly predicts E2^−^ expression and subsequent univariate ANOVA indicates that minor “G” allele carriers express 2.9-fold more E2^−^ splice isoform mRNA than “A” allele homozygotes (*n*=74, *F*=7.72, *p*=0.007). This genotype effect remains significant after including sample MB085, as it also carries rs76665058, (*n*=75, *F*=13.62, *p*=0.0004), or when only African-American samples are considered for analysis (*n*=13, *F*=12.29, *p*=0.006), with post-mortem interval as a significant covariate.

### Extended 5′UTR in vitro analysis

*Cis*-eQTL analyses demonstrate rs6311 modulates expression of the previously unannotated extended 5′UTR. To test the effect of rs6311, we co-transfected wild-type and variant rs6311 UTR luciferase constructs (lUTR-rs6311/G and lUTR-rs6311/A) into SH-SY5Y neuroblastoma cells and non-neuronal HEK293T cells and measured AEI ratios. Allelic expression ratios were similar to those observed in PFCs only in SH-SY5Y cells, the WT rs6311/G construct expressing 1.7-fold more mRNA than rs6311/A ([Figure 2C](#F2)). This result indicates that rs6311 modulates expression of mRNA transcripts utilizing the upstream TSS and suggests tissue-selectivity.

We then tested whether rs6311 alters translation, which is sensitive to 5′UTR length and structure ([36](#R36)), using reporter gene constructs with short (sUTR), medium (mUTR), or long (lUTR-rs6311/G and A) *HTR2A* 5′UTRs and a no UTR control. Translation efficiency did not significantly differ between lUTR-rs6311/G and lUTR-rs6311/A, which were subsequently combined (lUTR) for statistical power. Translation efficiency was significantly increased for the lUTR *versus* the mUTR or sUTR in HEK293T cells (*ANOVA F*=6.9, *Bonferroni-corrected p*=0.017 and 0.022, respectively; [Figure 3](#F3)). In SH-SY5Y cells, both the lUTR and mUTR were translated more efficiently than the sUTR (*ANOVA F*=8.02, *Bonferroni-corrected p*=0.002 and 0.037, respectively; [Figure 3](#F3)).

### Figure 3.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2db6/3582836/9cea1c22bc88/nihms413844f3.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3582836_nihms413844f3.jpg)

Translation efficiency for UTR constructs. The long UTR construct (lUTR) translates significantly more efficiently than either the medium (mUTR) or short UTR (sUTR) in the HEK293T cells (*lUTR vs. mUTR p=0.022, lUTR vs. sUTR p=0.017). The lUTR and mUTR translated significantly more efficiently than the sUTR in SH-SY5Y cells (*lUTR vs. sUTR p=0.002, mUTR vs. sUTR p=0.037). Data represented as mean RLU% ± S.D.

### CpG methylation of the HTR2A gene locus

Allelic mRNA expression differences can result from increased transcription of rs6311/G or repression of rs6311/A. To assess the epigenetic status of *HTR2A* as an indicator of transcription regulation, we measured methylation of a CpG site in exon 2 of *HTR2A* (chr13:47,469,654 of build GRCh37/hg19) in 223 dorsolateral prefrontal cortex samples and subsequently correlated methylation status with *HTR2A* genotypes (as part of a larger study; 33). This CpG site was the most proximal informative site to the *HTR2A* locus available on the Infinium HumanMethylation27 BeadChip platform. Multiple SNPs, all in high LD with rs6311, strongly correlated with CpG methylation ([Figure 4](#F4) and [Table S5](#SD1)) with the minor “A” allele significantly associated with greater methylation ([Figure S4](#SD1); *Total p*=6.34×10^−7^: *Caucasian p*=8.27×10^−4^, *African-American p*=2.87×10^−4^) in the same brain region displaying AEI for the 5′UTR, associating higher CpG methylation with repressed extended 5′UTR expression.

### Figure 4.

![Figure 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2db6/3582836/8cfacc398c29/nihms413844f4.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3582836_nihms413844f4.jpg)

Correlation between SNPs and CpG methylation at HTR2A. Markers represent SNPs genotyped in this region, mapped across the x-axis corresponding to hg19 coordinate. The size for each marker represents r2 to rs6311 (marker 17; see also, Table S5). Data along the y-axis represents −log10 transformed p-value for genotype-methylation analysis. A. Multiple SNPs in HTR2A were highly correlated with CpG methylation at the HTR2A gene locus and in high LD with rs6311. Only SNPs with p<0.02 are reported. B. and C. rs6311 (marker 17) remained significantly correlated with methylation across ethnicity.

### Effect of functional SNPs in STAR*D

We tested genotype-phenotype relationships between functional *HTR2A* SNPs and depression severity, citalopram response, and drug side effects in the STAR*D study ([23](#R23),[24](#R24)). Previously genotyped surrogate markers ([23](#R23)) (rs6313, rs7323441, and rs585719) in high LD with functional SNPs identified in *HTR2A* (rs6311, rs6314, and rs76665058, respectively) were used to test clinical associations in STAR*D. Population-based linkage disequilibrium statistics (*D*′ and *r2*) between functional *HTR2A* SNPs and surrogate markers are listed in [Table S6](#SD1). Significant genotype-phenotype associations are listed in [Table 2](#T2), along with significant covariates included in the respective models.

### Table 2.

| Phenotype | SNP(s) | Samples (n) | Statistical Test | Significant Covariatesa | p |
| --- | --- | --- | --- | --- | --- |
| Depression Severity | rs7323441*rs6313 | All (1190) | Univariate ANOVA | current employment status, years of schooling, weight change prior to study | 0.011 |
| Depression Severity | rs6313b | All (1190) | Univariate ANOVA | current employment status, years of schooling, weight change prior to study | 0.049 |
| Responsec | rs585719 | African-American only (156) | Repeated-measures ANOVA | cumulative side effects, menopausal/post-hysterectomy | 0.021 |
| Heart Palpitations | rs6313 | All (1222) | Logistic regression | cumulative side effects, sex, menopausal/post-hysterectomy | 0.012 |

Table 2 Caption: Analysis of Functional SNPs for Clinical Phenotypes in STAR*D

Alone, no single marker was associated with depression severity, as measured by baseline QIDS score prior to SSRI treatment. However, rs6313 and rs7323441 interact to significantly affect depression severity (*n*=1190, *F*=3.26, *p*=0.011), with homozygotes for minor alleles of both SNPs (rs6313/T, rs7323441/A) displaying higher baseline QIDS scores versus all other genotypes (19.4 *vs.* 15.1–16.5). By accounting for rs7323441 genotype in this interaction model, a significant main effect for rs6313 is now evident for depression severity (*n*=1190 *F*=3.02, *p*=0.049). Homozygous minor rs6313/T samples have significantly higher baseline QIDS scores *versus* major allele homozygotes (17.3 *vs.* 15.6; *Bonferroni-corrected p*=0.042). To test whether the significant SNP interaction is unique to surrogate SNPs marking functional alleles, we examined SNP interactions between rs6313 or rs7323441 and 37 additional markers genotyped in the 100kb surrounding *HTR2A*. Only SNPs in high LD with rs6311 significantly interacted with rs7323441, and only rs7323441 significantly interacted with rs6313 to affect depression severity ([Table S7](#SD1)), strongly suggesting that significant interactions observed here are directly related to the functional alleles.

Logistic regression revealed a significant association between rs6313 and heart palpitations during citalopram treatment (*n*=1222, *Wald statistic*=6.32, *p*=0.012), with homozygous rs6313/T minor allele carriers reporting fewer instances during treatment as compared to major rs6313/C allele homozygotes (19.4% *vs.* 25.1%). No other treatment response or side effect phenotype was significantly associated with rs6313 or rs7323441 alone or with an interaction between these two SNPs.

African-American carriers of the rs585719 SNP “T” allele responded significantly better to citalopram versus homozygous major allele individuals (ΔQIDS=11.2 *vs.* 7.7; *n*=156, *F*=5.41, *p*=0.021). No samples within this cohort were homozygous for the minor “T” allele. As expected, including non-African-American samples in the analysis diminishes the association with response (*n*=1222, *F*=0.64, *p*=0.529), as this rs585719 only serves as a surrogate marker for rs76665058 in African-Americans.

## Discussion

We have undertaken an extensive analysis of the transcriptional profile of *HTR2A* in human brain, describing multiple mRNA transcripts including new splice variants and extended 53 and 3′UTRs, and revealing the presence of three regulatory variants. Understanding the biological consequences of these variants facilitated the interpretation of clinical association studies, in particular, response to citalopram in STAR*D.

### Transcription start site usage, protein translation, and rs6311

Multiple TSS’s appear to differentially express *HTR2A* 5′UTRs in various brain regions. TSS prediction by Eponine ([32](#R32)) generally supports earlier mapping studies ([30](#R30),[31](#R31)), but also identifies a TSS beginning approximately 100 nucleotides upstream of the mapped PFC transcripts generated by RNA-seq. We propose that the frequent minor rs6311/A allele located in the promoter region of the annotated short 5′UTR mRNA isoforms, but residing within the extended 5′UTR discovered here, decreases usage of the upstream TSS. The lack of AEI at SNPs residing in the major *HTR2A* transcript, previously reported by others ([7](#R7),[9](#R9)), supports the notion of TSS switching, whereby increased usage at one site concomitantly decreases usage at the other, balancing the allelic expression across the total transcript pool. Because increased CpG methylation at the *HTR2A* gene locus correlated with rs6311/A, we speculate that transcription factors preferentially binding rs6311/A recruit epigenetic modifier proteins that suppress usage of the upstream TSS in favor of the downstream TSS. MatInspector ([37](#R37)) predicts preferential binding of the Hand1/E47 heterodimer complex and the CCAAT-binding transcription factor nuclear factor I/C (NFIC) to rs6311/A, but additional studies are required to clarify these relationships.

Among the brain regions, the hippocampus is remarkable considering similar levels of *HTR2A* expression as compared to amygdala and putamen, but complete lack of evidence for extended 5′UTR expression ([Figure S1C](#SD1)). Consequently, phenotypes related to brain regions with low extended 5′UTR expression (hippocampus, cerebellum, raphe nuclei) may be less affected by rs6311 genotype.

We found more efficient translation of the long compared to short 5′UTRs, resulting in less protein production from the minor rs6311/A allele associated with decreased long 5′UTR isoform expression. These results contradict previous *in vitro* and *ex vivo* studies suggesting rs6311/A increases mRNA or protein ([4](#R4)–[6](#R6)), a persistent notion despite the inability to replicate these findings ([7](#R7)–[13](#R13)). *HTR2A* expression is susceptible to environmental factors, including drugs ([14](#R14)–[18](#R18)), confounding correlations with overall expression. Measuring allelic expression is a robust approach for locating *cis*-acting functional variants largely independent of *trans*-acting factors, uniquely identifying rs6311 as a modulator of TSS usage, expression of extended 5′UTR mRNAs, and consequently, translation.

Our *in vitro* analyses of transcriptional and translational differences related to rs6311 were performed in cell lines phenotypically distinct from pyramidal cortical neurons, where *HTR2A* is predominately expressed in the brain. However, these experiments compliment *ex vivo* tissue findings, mutually supporting evidence for regulatory functions of rs6311. Protein measurements will be required to assess the effects of rs6311 on 5-HT2A signaling *in vivo*.

### Alternative splicing, 3′UTR usage, and rs76665058

An unannotated SNP (absent from NCBI dbSNP Build 135 and the 1000 Genomes project) in intron 2 was significantly associated with E2^−^ splice variant expression. E2^−^ encodes a protein isoform differing at the *N*-terminus and first transmembrane domain. Considering the propensity of G-protein coupled receptors to homo- and heterodimerize, the ability of this SNP to alter the ratio between E2^+^ and E2^−^ mRNA isoforms could translate into differences in receptor complex function. Preliminary immunohistochemical analysis of these splice variants suggest that E2^−^ and E2^tr^ show greater cytoplasmic localization than E2^+^ following serum starvation ([Figure S5](#SD1)). This is consistent with a previous study demonstrating that membrane expression of 5-HT2A is dependent upon *N*-linked glycosylation sites ([38](#R38)) that are absent in the E2^−^ and E2^tr^ splice variants. However, further studies are needed to clarify the biological role of these splice variants.

rs76665058 significantly changed allelic expression of the extended 3′UTR in African-Americans. Located 40 nucleotides downstream of a canonical polyadenylation signal, rs76665058 increases expression of the extended 3′UTR, possibly by decreasing usage of the proximal polyadenylation site. It was also significantly associated with greater expression of E2^−^ mRNA. Gene expression, 3′-end processing, and alternative splicing are co-transcriptionally regulated ([39](#R39)), suggesting a possible mechanism by which a SNP near the canonical polyadenylation signal could affect E2^−^ splicing. The low mRNA expression of both the extended 3′UTR and E2^−^ relative to total *HTR2A* commands caution when evaluating the consequences of rs76665058 on 5-HT2A functions.

### Functional HTR2A polymorphisms in the context of disease

Our SNP-SNP analyses suggest that individuals carrying haplotypes with minor alleles at rs6311 (A) and rs6314 (T) experience greater depressive symptoms. As rs6311 and rs6314 are in low LD (*D3*=0.142, *r2*=0.001 in CEU), haplotypes in heterozygous carriers for both SNPs are ambiguous, and assessment of any effect of individual haplotypes in these subjects will require accurate phasing. In African-Americans, a surrogate marker for rs76665058 (rs585719) was significantly associated with greater reduction in QIDS score following citalopram treatment. This preliminary finding coupled with the functional characterization demands direct testing of rs76665058 in larger African-American cohorts, but until the clinical utility should be interpreted cautiously.

We compiled meta-analyses conducted for rs6311 and its surrogate marker, rs6313, in *HTR2A* ([Figure 5](#F5); also see references [40](#R40)–[53](#R53)), to obtain perspective on *HTR2A* function in other disorders. Results from this meta-analyses support the notion that rs6311 contributes to a multitude of disorders, with highly variable penetrance for different phenotypes. The observed associations for rs6311 are consistent with the expected effects resulting from either reduced or enhanced 5-HT2A activity, as predicted by our studies. Although the effects of *HTR2A* SNPs alone are modest, evidence for biological consequences of these SNPs enable study of interactions with other functionally validated SNPs in the 5-HT signaling pathway, such as those we previously uncovered in tryptophan hydroxylase 2 (*TPH2*) ([28](#R28)) and monoamine oxidase A (*MAOA*) ([54](#R54)), or the widely studied serotonin transporter promoter variable repeat polymorphism associated with early-life stress and depression ([55](#R55)).

### Figure 5.

![Figure 5](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2db6/3582836/bd1f31c121be/nihms413844f5.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3582836_nihms413844f5.jpg)

Compilation of meta-analyses performed for rs6311 and rs6313 in HTR2A. Higher odds ratios indicate greater risk for the minor alleles. For drug response (46,47), higher odds ratio indicates better therapeutic response for the minor alleles. #See references in text. aStudy estimated genotypic rather than allelic odds ratios. bReanalysis of study 50 with one additional population included.

Knowledge of the complete *HTR2A* transcript profile enabled the discovery of several regulatory polymorphisms through allelic *cis*-eQTL mRNA analysis, strengthening associations with phenotypes in STAR*D. These insights facilitate the study of complex genetic disorders involving serotonin signaling.

## Supplementary Material

## Acknowledgments

This work was supported by the National Institute of General Medical Sciences [U01GM092655 to W.S.]; the National Center for Research Resources [UL1RR025755 to Rebecca D. Jackson]; an Eli Lilly Collaborative Research Grant [to W.S.]; and the United States Air Force Department of Defense [FA7014-09-2-0004 to Gail E. Herman]. The authors thank Dr. Andy Fischer (Neuroscience, Ohio State) for his microscopy expertise and Dr. Deborah Mash (Neurology, University of Miami, FL) for providing brain tissues.

The views and opinions expressed in this article are those of the author(s) and do not reflect official policy or position of the United State Air Force, Department of Defense, or US Government. The use of product(s) and/or manufacturer name(s) is added for clarification only; in no way implies endorsement by the authors, USAF, or DoD of the product(s) or manufacturer(s).

Data and biomaterials for the clinical analysis were obtained from the limited access datasets distributed from the NIH-supported “Sequenced Treatment Alternatives to Relieve Depression” (STAR*D). STAR*D focused on non-psychotic major depressive disorder in adults seen in outpatient settings. The primary purpose of this research study was to determine which treatments work best if the first treatment with medication does not produce an acceptable response. The study was supported by NIMH Contract #N01MH90003 to the University of Texas Southwestern Medical Center. The ClinicalTrials.gov identifier is [NCT00021528](https://clinicaltrials.gov/ct2/show/NCT00021528).

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Becker KG, Barnes KC, Bright TJ, Wang SA. The genetic association database. Nat Genet. 2004;36:431–432. doi: 10.1038/ng0504-431.  [DOI](https://doi.org/10.1038/ng0504-431) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15118671/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=The%20genetic%20association%20database&author=KG%20Becker&author=KC%20Barnes&author=TJ%20Bright&author=SA%20Wang&volume=36&publication_year=2004&pages=431-432&pmid=15118671&doi=10.1038/ng0504-431&)

2. Serretti A, Drago A, De Ronchi D. HTR2A gene variants and psychiatric disorders: a review of current literature and selection of SNPs for future studies. Curr Med Chem. 2007;14:2053–2069. doi: 10.2174/092986707781368450.  [DOI](https://doi.org/10.2174/092986707781368450) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17691947/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Med%20Chem&title=HTR2A%20gene%20variants%20and%20psychiatric%20disorders:%20a%20review%20of%20current%20literature%20and%20selection%20of%20SNPs%20for%20future%20studies&author=A%20Serretti&author=A%20Drago&author=D%20De%20Ronchi&volume=14&publication_year=2007&pages=2053-2069&pmid=17691947&doi=10.2174/092986707781368450&)

3. Kleinrock M. The Use of Medicines in the United States: Review of 2010. Webinar from the IMS Institute for Healthcare Informatics. 2011 Apr; Available from: http://www.imshealth.com/deployedfiles/ims/Global/Content/Insights/IMS%20Institute%20for%20Healthcare%20Informatics/year_in_review_4_19_livemeeting.wmv.  [http://www.imshealth.com/deployedfiles/ims/Global/Content/Insights/IMS%20Institute%20for%20Healthcare%20Informatics/year_in_review_4_19_livemeeting.wmv](http://www.imshealth.com/deployedfiles/ims/Global/Content/Insights/IMS%20Institute%20for%20Healthcare%20Informatics/year_in_review_4_19_livemeeting.wmv)

4. Parsons MJ, D’Souza UM, Arranz MJ, Kerwin RW, Makoff AJ. The -1438A/G polymorphism in the 5-hydroxytryptamine type 2A receptor gene affects promoter activity. Biol Psychiatry. 2004;56:406–410. doi: 10.1016/j.biopsych.2004.06.020.  [DOI](https://doi.org/10.1016/j.biopsych.2004.06.020) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15364038/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biol%20Psychiatry&title=The%20-1438A/G%20polymorphism%20in%20the%205-hydroxytryptamine%20type%202A%20receptor%20gene%20affects%20promoter%20activity&author=MJ%20Parsons&author=UM%20D%E2%80%99Souza&author=MJ%20Arranz&author=RW%20Kerwin&author=AJ%20Makoff&volume=56&publication_year=2004&pages=406-410&pmid=15364038&doi=10.1016/j.biopsych.2004.06.020&)

5. Polesskaya OO, Sokolov BP. Differential expression of the “C” and “T” alleles of the 5-HT2A receptor gene in the temporal cortex of normal individuals and schizophrenics. J Neurosci Res. 2002;67:812–822. doi: 10.1002/jnr.10173.  [DOI](https://doi.org/10.1002/jnr.10173) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11891796/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Neurosci%20Res&title=Differential%20expression%20of%20the%20%E2%80%9CC%E2%80%9D%20and%20%E2%80%9CT%E2%80%9D%20alleles%20of%20the%205-HT2A%20receptor%20gene%20in%20the%20temporal%20cortex%20of%20normal%20individuals%20and%20schizophrenics&author=OO%20Polesskaya&author=BP%20Sokolov&volume=67&publication_year=2002&pages=812-822&pmid=11891796&doi=10.1002/jnr.10173&)

6. Turecki G, Brière R, Dewar K, Antonetti T, Lesage AD, Séguin M, et al. Prediction of level of serotonin 2A receptor binding by serotonin receptor 2A genetic variation in postmortem brain samples from subjects who did or did not commit suicide. Am J Psychiatry. 1999;156:1456–1458. doi: 10.1176/ajp.156.9.1456.  [DOI](https://doi.org/10.1176/ajp.156.9.1456) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10484964/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Psychiatry&title=Prediction%20of%20level%20of%20serotonin%202A%20receptor%20binding%20by%20serotonin%20receptor%202A%20genetic%20variation%20in%20postmortem%20brain%20samples%20from%20subjects%20who%20did%20or%20did%20not%20commit%20suicide&author=G%20Turecki&author=R%20Bri%C3%A8re&author=K%20Dewar&author=T%20Antonetti&author=AD%20Lesage&volume=156&publication_year=1999&pages=1456-1458&pmid=10484964&doi=10.1176/ajp.156.9.1456&)

7. Bray NJ, Buckland PR, Hall H, Owen MJ, O’Donovan MC. The serotonin-2A receptor gene locus does not contain common polymorphism affecting mRNA levels in adult brain. Mol Psychiatry. 2004;9:109–114. doi: 10.1038/sj.mp.4001366.  [DOI](https://doi.org/10.1038/sj.mp.4001366) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14699448/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Psychiatry&title=The%20serotonin-2A%20receptor%20gene%20locus%20does%20not%20contain%20common%20polymorphism%20affecting%20mRNA%20levels%20in%20adult%20brain&author=NJ%20Bray&author=PR%20Buckland&author=H%20Hall&author=MJ%20Owen&author=MC%20O%E2%80%99Donovan&volume=9&publication_year=2004&pages=109-114&pmid=14699448&doi=10.1038/sj.mp.4001366&)

8. Du L, Faludi G, Palkovits M, Demeter E, Bakish D, Lapierre YD, et al. Frequency of long allele in serotonin transporter gene is increased in depressed suicide victims. Biol Psychiatry. 1999;46:196–201. doi: 10.1016/s0006-3223(98)00376-x.  [DOI](https://doi.org/10.1016/s0006-3223(98)00376-x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10418694/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biol%20Psychiatry&title=Frequency%20of%20long%20allele%20in%20serotonin%20transporter%20gene%20is%20increased%20in%20depressed%20suicide%20victims&author=L%20Du&author=G%20Faludi&author=M%20Palkovits&author=E%20Demeter&author=D%20Bakish&volume=46&publication_year=1999&pages=196-201&pmid=10418694&doi=10.1016/s0006-3223(98)00376-x&)

9. Fukuda Y, Koga M, Arai M, Noguchi E, Ohtsuki T, Horiuchi Y, et al. Monoallelic and unequal allelic expression of the HTR2A gene in human brain and peripheral lymphocytes. Biol Psychiatry. 2006;60:1331–1335. doi: 10.1016/j.biopsych.2006.06.024.  [DOI](https://doi.org/10.1016/j.biopsych.2006.06.024) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17069769/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biol%20Psychiatry&title=Monoallelic%20and%20unequal%20allelic%20expression%20of%20the%20HTR2A%20gene%20in%20human%20brain%20and%20peripheral%20lymphocytes&author=Y%20Fukuda&author=M%20Koga&author=M%20Arai&author=E%20Noguchi&author=T%20Ohtsuki&volume=60&publication_year=2006&pages=1331-1335&pmid=17069769&doi=10.1016/j.biopsych.2006.06.024&)

10. Kouzmenko AP, Hayes WL, Pereira AM, Dean B, Burnet PW, Harrison PJ. 5-HT2A receptor polymorphism and steady state receptor expression in schizophrenia. Lancet. 1997;349:1815. doi: 10.1016/S0140-6736(05)61695-9.  [DOI](https://doi.org/10.1016/S0140-6736(05)61695-9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9269223/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=5-HT2A%20receptor%20polymorphism%20and%20steady%20state%20receptor%20expression%20in%20schizophrenia&author=AP%20Kouzmenko&author=WL%20Hayes&author=AM%20Pereira&author=B%20Dean&author=PW%20Burnet&volume=349&publication_year=1997&pages=1815&pmid=9269223&doi=10.1016/S0140-6736(05)61695-9&)

11. Kouzmenko AP, Scaffidi A, Pereira AM, Hayes WL, Copolov DL, Dean B. No correlation between A(-1438)G polymorphism in 5-HT2A receptor gene promoter and the density of frontal cortical 5-HT2A receptors in schizophrenia. Hum Hered. 1999;49:103–105. doi: 10.1159/000022853.  [DOI](https://doi.org/10.1159/000022853) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10077731/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Hered&title=No%20correlation%20between%20A(-1438)G%20polymorphism%20in%205-HT2A%20receptor%20gene%20promoter%20and%20the%20density%20of%20frontal%20cortical%205-HT2A%20receptors%20in%20schizophrenia&author=AP%20Kouzmenko&author=A%20Scaffidi&author=AM%20Pereira&author=WL%20Hayes&author=DL%20Copolov&volume=49&publication_year=1999&pages=103-105&pmid=10077731&doi=10.1159/000022853&)

12. Myers RL, Airey DC, Manier DH, Shelton RC, Sanders-Bush E. Polymorphisms in the regulatory region of the human serotonin 5-HT2A receptor gene (HTR2A) influence gene expression. Biol Psychiatry. 2007;61:167–173. doi: 10.1016/j.biopsych.2005.12.018.  [DOI](https://doi.org/10.1016/j.biopsych.2005.12.018) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16697352/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biol%20Psychiatry&title=Polymorphisms%20in%20the%20regulatory%20region%20of%20the%20human%20serotonin%205-HT2A%20receptor%20gene%20(HTR2A)%20influence%20gene%20expression&author=RL%20Myers&author=DC%20Airey&author=DH%20Manier&author=RC%20Shelton&author=E%20Sanders-Bush&volume=61&publication_year=2007&pages=167-173&pmid=16697352&doi=10.1016/j.biopsych.2005.12.018&)

13. Spurlock G, Heils A, Holmans P, Williams J, D’Souza UM, Cardno A, et al. A family based association study of T102C polymorphism in 5HT2A and schizophrenia plus identification of new polymorphisms in the promoter. Mol Psychiatry. 1998;3:42–49. doi: 10.1038/sj.mp.4000342.  [DOI](https://doi.org/10.1038/sj.mp.4000342) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9491812/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Psychiatry&title=A%20family%20based%20association%20study%20of%20T102C%20polymorphism%20in%205HT2A%20and%20schizophrenia%20plus%20identification%20of%20new%20polymorphisms%20in%20the%20promoter&author=G%20Spurlock&author=A%20Heils&author=P%20Holmans&author=J%20Williams&author=UM%20D%E2%80%99Souza&volume=3&publication_year=1998&pages=42-49&pmid=9491812&doi=10.1038/sj.mp.4000342&)

14. Burnet PW, Chen CP, McGowan S, Franklin M, Harrison PJ. The effects of clozapine and haloperidol on serotonin-1A, -2A and -2C receptor gene expression and serotonin metabolism in the rat forebrain. Neuroscience. 1996;73:531–540. doi: 10.1016/0306-4522(96)00062-0.  [DOI](https://doi.org/10.1016/0306-4522(96)00062-0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8783268/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neuroscience&title=The%20effects%20of%20clozapine%20and%20haloperidol%20on%20serotonin-1A,%20-2A%20and%20-2C%20receptor%20gene%20expression%20and%20serotonin%20metabolism%20in%20the%20rat%20forebrain&author=PW%20Burnet&author=CP%20Chen&author=S%20McGowan&author=M%20Franklin&author=PJ%20Harrison&volume=73&publication_year=1996&pages=531-540&pmid=8783268&doi=10.1016/0306-4522(96)00062-0&)

15. Hernandez I, Sokolov BP. Abnormalities in 5-HT2A receptor mRNA expression in frontal cortex of chronic elderly schizophrenics with varying histories of neuroleptic treatment. J Neurosci Res. 2000;59:218–225.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/10650880/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Neurosci%20Res&title=Abnormalities%20in%205-HT2A%20receptor%20mRNA%20expression%20in%20frontal%20cortex%20of%20chronic%20elderly%20schizophrenics%20with%20varying%20histories%20of%20neuroleptic%20treatment&author=I%20Hernandez&author=BP%20Sokolov&volume=59&publication_year=2000&pages=218-225&pmid=10650880&)

16. Huang XF, Han M, Storlien LH. Differential expression of 5-HT(2A) and 5-HT(2C) receptor mRNAs in mice prone, or resistant, to chronic high-fat diet-induced obesity. Brain Res Mol Brain Res. 2004;127:39–47. doi: 10.1016/j.molbrainres.2004.05.008.  [DOI](https://doi.org/10.1016/j.molbrainres.2004.05.008) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15306119/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Brain%20Res%20Mol%20Brain%20Res&title=Differential%20expression%20of%205-HT(2A)%20and%205-HT(2C)%20receptor%20mRNAs%20in%20mice%20prone,%20or%20resistant,%20to%20chronic%20high-fat%20diet-induced%20obesity&author=XF%20Huang&author=M%20Han&author=LH%20Storlien&volume=127&publication_year=2004&pages=39-47&pmid=15306119&doi=10.1016/j.molbrainres.2004.05.008&)

17. Oreland S, Pickering C, Gokturk C, Oreland L, Arborelius L, Nylander I. Two repeated maternal separation procedures differentially affect brain 5-hydroxytryptamine transporter and receptors in young and adult male and female rats. Brain Res. 2009;1305(Suppl):S37–49. doi: 10.1016/j.brainres.2009.08.069.  [DOI](https://doi.org/10.1016/j.brainres.2009.08.069) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19728999/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Brain%20Res&title=Two%20repeated%20maternal%20separation%20procedures%20differentially%20affect%20brain%205-hydroxytryptamine%20transporter%20and%20receptors%20in%20young%20and%20adult%20male%20and%20female%20rats&author=S%20Oreland&author=C%20Pickering&author=C%20Gokturk&author=L%20Oreland&author=L%20Arborelius&volume=1305&issue=Suppl&publication_year=2009&pages=S37-49&pmid=19728999&doi=10.1016/j.brainres.2009.08.069&)

18. Willins DL, Alsayegh L, Berry SA, Backstrom JR, Sanders-Bush E, Friedman L, et al. Serotonergic antagonist effects on trafficking of serotonin 5-HT2A receptors in vitro and in vivo. Ann N Y Acad Sci. 1998;861:121–127. doi: 10.1111/j.1749-6632.1998.tb10182.x.  [DOI](https://doi.org/10.1111/j.1749-6632.1998.tb10182.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9928248/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20N%20Y%20Acad%20Sci&title=Serotonergic%20antagonist%20effects%20on%20trafficking%20of%20serotonin%205-HT2A%20receptors%20in%20vitro%20and%20in%20vivo&author=DL%20Willins&author=L%20Alsayegh&author=SA%20Berry&author=JR%20Backstrom&author=E%20Sanders-Bush&volume=861&publication_year=1998&pages=121-127&pmid=9928248&doi=10.1111/j.1749-6632.1998.tb10182.x&)

19. Davies MA, Setola V, Strachan RT, Sheffler DJ, Salay E, Hufeisen SJ, et al. Pharmacologic analysis of non-synonymous coding h5-HT2A SNPs reveals alterations in atypical antipsychotic and agonist efficacies. Pharmacogenomics J. 2006;6:42–51. doi: 10.1038/sj.tpj.6500342.  [DOI](https://doi.org/10.1038/sj.tpj.6500342) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16314884/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Pharmacologic%20analysis%20of%20non-synonymous%20coding%20h5-HT2A%20SNPs%20reveals%20alterations%20in%20atypical%20antipsychotic%20and%20agonist%20efficacies&author=MA%20Davies&author=V%20Setola&author=RT%20Strachan&author=DJ%20Sheffler&author=E%20Salay&volume=6&publication_year=2006&pages=42-51&pmid=16314884&doi=10.1038/sj.tpj.6500342&)

20. Hazelwood LA, Sanders-Bush E. His452Tyr polymorphism in the human 5-HT2A receptor destabilizes the signaling conformation. Mol Pharmacol. 2004;66:1293–1300.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/15496511/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Pharmacol&title=His452Tyr%20polymorphism%20in%20the%20human%205-HT2A%20receptor%20destabilizes%20the%20signaling%20conformation&author=LA%20Hazelwood&author=E%20Sanders-Bush&volume=66&publication_year=2004&pages=1293-1300&pmid=15496511&)

21. de Quervain DJ, Henke K, Aerni A, Coluccia D, Wollmer MA, Hock C, et al. A functional genetic variation of the 5-HT2a receptor affects human memory. Nat Neurosci. 2003;6:1141–1142. doi: 10.1038/nn1146.  [DOI](https://doi.org/10.1038/nn1146) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14566344/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Neurosci&title=A%20functional%20genetic%20variation%20of%20the%205-HT2a%20receptor%20affects%20human%20memory&author=DJ%20de%20Quervain&author=K%20Henke&author=A%20Aerni&author=D%20Coluccia&author=MA%20Wollmer&volume=6&publication_year=2003&pages=1141-1142&pmid=14566344&doi=10.1038/nn1146&)

22. Manolio TA, Collins FS, Cox NJ, Goldstein DB, Hindorff LA, Hunter DJ, et al. Finding the missing heritability of complex diseases. Nature. 2009;461:747–753. doi: 10.1038/nature08494.  [DOI](https://doi.org/10.1038/nature08494) | [PMC free article](/articles/PMC2831613/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19812666/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Finding%20the%20missing%20heritability%20of%20complex%20diseases&author=TA%20Manolio&author=FS%20Collins&author=NJ%20Cox&author=DB%20Goldstein&author=LA%20Hindorff&volume=461&publication_year=2009&pages=747-753&pmid=19812666&doi=10.1038/nature08494&)

23. Garriock HA, Kraft JB, Shyn SI, Peters EJ, Yokoyama JS, Jenkins GD, et al. A genomewide association study of citalopram response in major depressive disorder. Biol Psychiatry. 2010;67:133–138. doi: 10.1016/j.biopsych.2009.08.029.  [DOI](https://doi.org/10.1016/j.biopsych.2009.08.029) | [PMC free article](/articles/PMC2794921/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19846067/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biol%20Psychiatry&title=A%20genomewide%20association%20study%20of%20citalopram%20response%20in%20major%20depressive%20disorder&author=HA%20Garriock&author=JB%20Kraft&author=SI%20Shyn&author=EJ%20Peters&author=JS%20Yokoyama&volume=67&publication_year=2010&pages=133-138&pmid=19846067&doi=10.1016/j.biopsych.2009.08.029&)

24. McMahon FJ, Buervenich S, Charney D, Lipsky R, Rush AJ, Wilson AF, et al. Variation in the gene encoding the serotonin 2A receptor is associated with outcome of antidepressant treatment. Am J Hum Genet. 2006;78:804–814. doi: 10.1086/503820.  [DOI](https://doi.org/10.1086/503820) | [PMC free article](/articles/PMC1474035/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16642436/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Variation%20in%20the%20gene%20encoding%20the%20serotonin%202A%20receptor%20is%20associated%20with%20outcome%20of%20antidepressant%20treatment&author=FJ%20McMahon&author=S%20Buervenich&author=D%20Charney&author=R%20Lipsky&author=AJ%20Rush&volume=78&publication_year=2006&pages=804-814&pmid=16642436&doi=10.1086/503820&)

25. Smith RM, Alachkar H, Papp AC, Wang D, Mash DC, Wang JC, et al. Nicotinic alpha5 receptor subunit mRNA expression is associated with distant 53 upstream polymorphisms. Eur J Hum Genet. 2011;19:76–83. doi: 10.1038/ejhg.2010.120.  [DOI](https://doi.org/10.1038/ejhg.2010.120) | [PMC free article](/articles/PMC2995013/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20700147/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Hum%20Genet&title=Nicotinic%20alpha5%20receptor%20subunit%20mRNA%20expression%20is%20associated%20with%20distant%2053%20upstream%20polymorphisms&author=RM%20Smith&author=H%20Alachkar&author=AC%20Papp&author=D%20Wang&author=DC%20Mash&volume=19&publication_year=2011&pages=76-83&pmid=20700147&doi=10.1038/ejhg.2010.120&)

26. Trapnell C, Williams BA, Pertea G, Mortazavi A, Kwan G, van Baren MJ, et al. Transcript assembly and quantification by RNA-Seq reveals unannotated transcripts and isoform switching during cell differentiation. Nat Biotechnol. 2010;28:511–515. doi: 10.1038/nbt.1621.  [DOI](https://doi.org/10.1038/nbt.1621) | [PMC free article](/articles/PMC3146043/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20436464/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Biotechnol&title=Transcript%20assembly%20and%20quantification%20by%20RNA-Seq%20reveals%20unannotated%20transcripts%20and%20isoform%20switching%20during%20cell%20differentiation&author=C%20Trapnell&author=BA%20Williams&author=G%20Pertea&author=A%20Mortazavi&author=G%20Kwan&volume=28&publication_year=2010&pages=511-515&pmid=20436464&doi=10.1038/nbt.1621&)

27. Robinson JT, Thorvaldsdóttir H, Winckler W, Guttman M, Lander ES, Getz G, et al. Integrative genomics viewer. Nat Biotechnol. 2011;29:24–26. doi: 10.1038/nbt.1754.  [DOI](https://doi.org/10.1038/nbt.1754) | [PMC free article](/articles/PMC3346182/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21221095/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Biotechnol&title=Integrative%20genomics%20viewer&author=JT%20Robinson&author=H%20Thorvaldsd%C3%B3ttir&author=W%20Winckler&author=M%20Guttman&author=ES%20Lander&volume=29&publication_year=2011&pages=24-26&pmid=21221095&doi=10.1038/nbt.1754&)

28. Lim JE, Pinsonneault J, Sadee W, Saffen D. Tryptophan hydroxylase 2 (TPH2) haplotypes predict levels of TPH2 mRNA expression in human pons. Mol Psychiatry. 2007;12:491–501. doi: 10.1038/sj.mp.4001923.  [DOI](https://doi.org/10.1038/sj.mp.4001923) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17453063/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Psychiatry&title=Tryptophan%20hydroxylase%202%20(TPH2)%20haplotypes%20predict%20levels%20of%20TPH2%20mRNA%20expression%20in%20human%20pons&author=JE%20Lim&author=J%20Pinsonneault&author=W%20Sadee&author=D%20Saffen&volume=12&publication_year=2007&pages=491-501&pmid=17453063&doi=10.1038/sj.mp.4001923&)

29. Kent WJ, Sugnet CW, Furey TS, Roskin KM, Pringle TH, Zahler AM, et al. The human genome browser at UCSC. Genome Res. 2002;12:996–1006. doi: 10.1101/gr.229102.  [DOI](https://doi.org/10.1101/gr.229102) | [PMC free article](/articles/PMC186604/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12045153/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res&title=The%20human%20genome%20browser%20at%20UCSC&author=WJ%20Kent&author=CW%20Sugnet&author=TS%20Furey&author=KM%20Roskin&author=TH%20Pringle&volume=12&publication_year=2002&pages=996-1006&pmid=12045153&doi=10.1101/gr.229102&)

30. Shih JC, Zhu Q, Chen K. Determination of transcription initiation sites and promoter activity of the human 5-HT2A receptor gene. Behav Brain Res. 1996;73:59–62. doi: 10.1016/0166-4328(96)00070-8.  [DOI](https://doi.org/10.1016/0166-4328(96)00070-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8788478/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Behav%20Brain%20Res&title=Determination%20of%20transcription%20initiation%20sites%20and%20promoter%20activity%20of%20the%20human%205-HT2A%20receptor%20gene&author=JC%20Shih&author=Q%20Zhu&author=K%20Chen&volume=73&publication_year=1996&pages=59-62&pmid=8788478&doi=10.1016/0166-4328(96)00070-8&)

31. Zhu QS, Chen K, Shih JC. Characterization of the human 5-HT2A receptor gene promoter. J Neurosci. 1995;15:4885–4895. doi: 10.1523/JNEUROSCI.15-07-04885.1995.  [DOI](https://doi.org/10.1523/JNEUROSCI.15-07-04885.1995) | [PMC free article](/articles/PMC6577879/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7623119/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Neurosci&title=Characterization%20of%20the%20human%205-HT2A%20receptor%20gene%20promoter&author=QS%20Zhu&author=K%20Chen&author=JC%20Shih&volume=15&publication_year=1995&pages=4885-4895&pmid=7623119&doi=10.1523/JNEUROSCI.15-07-04885.1995&)

32. Down TA, Hubbard TJ. Computational detection and location of transcription start sites in mammalian genomic DNA. Genome Res. 2002;12:458–461. doi: 10.1101/gr.216102.  [DOI](https://doi.org/10.1101/gr.216102) | [PMC free article](/articles/PMC155284/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11875034/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res&title=Computational%20detection%20and%20location%20of%20transcription%20start%20sites%20in%20mammalian%20genomic%20DNA&author=TA%20Down&author=TJ%20Hubbard&volume=12&publication_year=2002&pages=458-461&pmid=11875034&doi=10.1101/gr.216102&)

33. Numata S, Ye T, Hyde TM, Guitart-Navarro X, Tao R, Wininger M, et al. DNA methylation signatures in development and aging of the human prefrontal cortex. Am J Hum Genet. 2012;90:260–272. doi: 10.1016/j.ajhg.2011.12.020.  [DOI](https://doi.org/10.1016/j.ajhg.2011.12.020) | [PMC free article](/articles/PMC3276664/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22305529/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=DNA%20methylation%20signatures%20in%20development%20and%20aging%20of%20the%20human%20prefrontal%20cortex&author=S%20Numata&author=T%20Ye&author=TM%20Hyde&author=X%20Guitart-Navarro&author=R%20Tao&volume=90&publication_year=2012&pages=260-272&pmid=22305529&doi=10.1016/j.ajhg.2011.12.020&)

34. Liu W, Downing ACM, Munsie LM, Chen P, Reed MR, Ruble CL, et al. Pharmacogenetic analysis of the mGlu2/3 agonist LY2140023 monohydrate in the treatment of schizophrenia. Pharmacogenomics J. 2010 Dec 21; doi: 10.1038/tpj.2010.90.  [DOI](https://doi.org/10.1038/tpj.2010.90) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21173788/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Pharmacogenetic%20analysis%20of%20the%20mGlu2/3%20agonist%20LY2140023%20monohydrate%20in%20the%20treatment%20of%20schizophrenia&author=W%20Liu&author=ACM%20Downing&author=LM%20Munsie&author=P%20Chen&author=MR%20Reed&publication_year=2010&pmid=21173788&doi=10.1038/tpj.2010.90&)

35. Piva F, Giulietti M, Nocchi L, Principato G. SpliceAid: a database of experimental RNA target motifs bound by splicing proteins in humans. Bioinformatics. 2009;25:1211–1213. doi: 10.1093/bioinformatics/btp124.  [DOI](https://doi.org/10.1093/bioinformatics/btp124) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19261717/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=SpliceAid:%20a%20database%20of%20experimental%20RNA%20target%20motifs%20bound%20by%20splicing%20proteins%20in%20humans&author=F%20Piva&author=M%20Giulietti&author=L%20Nocchi&author=G%20Principato&volume=25&publication_year=2009&pages=1211-1213&pmid=19261717&doi=10.1093/bioinformatics/btp124&)

36. van der Velden AW, Thomas AA. The role of the 53 untranslated region of an mRNA in translation regulation during development. Int J Biochem Cell Biol. 1999;31:87–106. doi: 10.1016/s1357-2725(98)00134-4.  [DOI](https://doi.org/10.1016/s1357-2725(98)00134-4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10216946/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Biochem%20Cell%20Biol&title=The%20role%20of%20the%2053%20untranslated%20region%20of%20an%20mRNA%20in%20translation%20regulation%20during%20development&author=AW%20van%20der%20Velden&author=AA%20Thomas&volume=31&publication_year=1999&pages=87-106&pmid=10216946&doi=10.1016/s1357-2725(98)00134-4&)

37. Cartharius K, Frech K, Grote K, Klocke B, Haltmeier M, Klingenhoff A, et al. MatInspector and beyond: promoter analysis based on transcription factor binding sites. Bioinformatics. 2005;21:2933–2942. doi: 10.1093/bioinformatics/bti473.  [DOI](https://doi.org/10.1093/bioinformatics/bti473) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15860560/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=MatInspector%20and%20beyond:%20promoter%20analysis%20based%20on%20transcription%20factor%20binding%20sites&author=K%20Cartharius&author=K%20Frech&author=K%20Grote&author=B%20Klocke&author=M%20Haltmeier&volume=21&publication_year=2005&pages=2933-2942&pmid=15860560&doi=10.1093/bioinformatics/bti473&)

38. Maginnis MS, Haley SA, Gee GV, Atwood WJ. Role of N-linked glycosylation of the 5-HT2A receptor in JC virus infection. J Virol. 2010;84:9677–9684. doi: 10.1128/JVI.00978-10.  [DOI](https://doi.org/10.1128/JVI.00978-10) | [PMC free article](/articles/PMC2937807/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20660194/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Virol&title=Role%20of%20N-linked%20glycosylation%20of%20the%205-HT2A%20receptor%20in%20JC%20virus%20infection&author=MS%20Maginnis&author=SA%20Haley&author=GV%20Gee&author=WJ%20Atwood&volume=84&publication_year=2010&pages=9677-9684&pmid=20660194&doi=10.1128/JVI.00978-10&)

39. Maniatis T, Reed R. An extensive network of coupling among gene expression machines. Nature. 2002;416:499–506. doi: 10.1038/416499a.  [DOI](https://doi.org/10.1038/416499a) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11932736/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=An%20extensive%20network%20of%20coupling%20among%20gene%20expression%20machines&author=T%20Maniatis&author=R%20Reed&volume=416&publication_year=2002&pages=499-506&pmid=11932736&doi=10.1038/416499a&)

40. Lee YH, Choi SJ, Ji JD, Song GG. Candidate gene studies of fibromyalgia: a systematic review and meta-analysis. Rheumatol Int. 2012;32:417–426. doi: 10.1007/s00296-010-1678-9.  [DOI](https://doi.org/10.1007/s00296-010-1678-9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21120487/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Rheumatol%20Int&title=Candidate%20gene%20studies%20of%20fibromyalgia:%20a%20systematic%20review%20and%20meta-analysis&author=YH%20Lee&author=SJ%20Choi&author=JD%20Ji&author=GG%20Song&volume=32&publication_year=2012&pages=417-426&pmid=21120487&doi=10.1007/s00296-010-1678-9&)

41. Kling A, Seddighzadeh M, Ärlestig L, Alfredsson L, Rantapää-Dahlqvist S, Padyukov L. Genetic variations in the serotonin 5-HT2A receptor gene (HTR2A) are associated with rheumatoid arthritis. Ann Rheum Dis. 2008;67:1111–1115. doi: 10.1136/ard.2007.074948.  [DOI](https://doi.org/10.1136/ard.2007.074948) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18006541/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Rheum%20Dis&title=Genetic%20variations%20in%20the%20serotonin%205-HT2A%20receptor%20gene%20(HTR2A)%20are%20associated%20with%20rheumatoid%20arthritis&author=A%20Kling&author=M%20Seddighzadeh&author=L%20%C3%84rlestig&author=L%20Alfredsson&author=S%20Rantap%C3%A4%C3%A4-Dahlqvist&volume=67&publication_year=2008&pages=1111-1115&pmid=18006541&doi=10.1136/ard.2007.074948&)

42. Abdolmaleky HM, Faraone SV, Glatt SJ, Tsuang MT. Meta-analysis of association between the T102C polymorphism of the 5HT2a receptor gene and schizophrenia. Schizophr Res. 2004;67:53–62. doi: 10.1016/s0920-9964(03)00183-x.  [DOI](https://doi.org/10.1016/s0920-9964(03)00183-x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14741324/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Schizophr%20Res&title=Meta-analysis%20of%20association%20between%20the%20T102C%20polymorphism%20of%20the%205HT2a%20receptor%20gene%20and%20schizophrenia&author=HM%20Abdolmaleky&author=SV%20Faraone&author=SJ%20Glatt&author=MT%20Tsuang&volume=67&publication_year=2004&pages=53-62&pmid=14741324&doi=10.1016/s0920-9964(03)00183-x&)

43. Allen NC, Bagade S, McQueen MB, Ioannidis JP, Kavvoura FK, Khoury MJ, et al. Systematic meta-analyses and field synopsis of genetic association studies in schizophrenia: the SzGene database. Nat Genet. 2008;40:827–834. doi: 10.1038/ng.171.  [DOI](https://doi.org/10.1038/ng.171) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18583979/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Systematic%20meta-analyses%20and%20field%20synopsis%20of%20genetic%20association%20studies%20in%20schizophrenia:%20the%20SzGene%20database&author=NC%20Allen&author=S%20Bagade&author=MB%20McQueen&author=JP%20Ioannidis&author=FK%20Kavvoura&volume=40&publication_year=2008&pages=827-834&pmid=18583979&doi=10.1038/ng.171&)

44. Li D, Duan Y, He L. Association study of serotonin 2A receptor (5-HT2A) gene with schizophrenia and suicidal behavior using systematic meta-analysis. Biochem Biophys Res Commun. 2006;340:1006–1015. doi: 10.1016/j.bbrc.2005.12.101.  [DOI](https://doi.org/10.1016/j.bbrc.2005.12.101) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16405867/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20Biophys%20Res%20Commun&title=Association%20study%20of%20serotonin%202A%20receptor%20(5-HT2A)%20gene%20with%20schizophrenia%20and%20suicidal%20behavior%20using%20systematic%20meta-analysis&author=D%20Li&author=Y%20Duan&author=L%20He&volume=340&publication_year=2006&pages=1006-1015&pmid=16405867&doi=10.1016/j.bbrc.2005.12.101&)

45. Williams J, McGuffin P, Nöthen M, Owen MJ. Meta-analysis of association between the 5-HT2a receptor T102C polymorphism and schizophrenia. EMASS Collaborative Group. European Multicentre Association Study of Schizophrenia. Lancet. 1997;349:1221. doi: 10.1016/s0140-6736(05)62413-0.  [DOI](https://doi.org/10.1016/s0140-6736(05)62413-0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9130948/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Meta-analysis%20of%20association%20between%20the%205-HT2a%20receptor%20T102C%20polymorphism%20and%20schizophrenia.%20EMASS%20Collaborative%20Group.%20European%20Multicentre%20Association%20Study%20of%20Schizophrenia&author=J%20Williams&author=P%20McGuffin&author=M%20N%C3%B6then&author=MJ%20Owen&volume=349&publication_year=1997&pages=1221&pmid=9130948&doi=10.1016/s0140-6736(05)62413-0&)

46. Ramanathan S, Glatt SJ. Serotonergic system genes in psychosis of Alzheimer dementia: meta-analysis. Am J Geriatr Psychiatry. 2009;17:839–846. doi: 10.1097/JGP.0b013e3181ab8c3f.  [DOI](https://doi.org/10.1097/JGP.0b013e3181ab8c3f) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19910872/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Geriatr%20Psychiatry&title=Serotonergic%20system%20genes%20in%20psychosis%20of%20Alzheimer%20dementia:%20meta-analysis&author=S%20Ramanathan&author=SJ%20Glatt&volume=17&publication_year=2009&pages=839-846&pmid=19910872&doi=10.1097/JGP.0b013e3181ab8c3f&)

47. Lerer B, Segman RH, Tan EC, Basile VS, Cavallaro R, Aschauer HN, et al. Combined analysis of 635 patients confirms an age-related association of the serotonin 2A receptor gene with tardive dyskinesia and specificity for the non-orofacial subtype. Int J Neuropsychopharmacol. 2005;8:411–425. doi: 10.1017/S1461145705005389.  [DOI](https://doi.org/10.1017/S1461145705005389) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15857569/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Neuropsychopharmacol&title=Combined%20analysis%20of%20635%20patients%20confirms%20an%20age-related%20association%20of%20the%20serotonin%202A%20receptor%20gene%20with%20tardive%20dyskinesia%20and%20specificity%20for%20the%20non-orofacial%20subtype&author=B%20Lerer&author=RH%20Segman&author=EC%20Tan&author=VS%20Basile&author=R%20Cavallaro&volume=8&publication_year=2005&pages=411-425&pmid=15857569&doi=10.1017/S1461145705005389&)

48. Arranz MJ, Munro J, Sham P, Kirov G, Murray RM, Collier DA, et al. Meta-analysis of studies on genetic variation in 5-HT2A receptors and clozapine response. Schizophr Res. 1998;32:93–99. doi: 10.1016/s0920-9964(98)00032-2.  [DOI](https://doi.org/10.1016/s0920-9964(98)00032-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9713904/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Schizophr%20Res&title=Meta-analysis%20of%20studies%20on%20genetic%20variation%20in%205-HT2A%20receptors%20and%20clozapine%20response&author=MJ%20Arranz&author=J%20Munro&author=P%20Sham&author=G%20Kirov&author=RM%20Murray&volume=32&publication_year=1998&pages=93-99&pmid=9713904&doi=10.1016/s0920-9964(98)00032-2&)

49. Kato M, Serretti A. Review and meta-analysis of antidepressant pharmacogenetic findings in major depressive disorder. Mol Psychiatry. 2010;15:473–500. doi: 10.1038/mp.2008.116.  [DOI](https://doi.org/10.1038/mp.2008.116) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18982004/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Psychiatry&title=Review%20and%20meta-analysis%20of%20antidepressant%20pharmacogenetic%20findings%20in%20major%20depressive%20disorder&author=M%20Kato&author=A%20Serretti&volume=15&publication_year=2010&pages=473-500&pmid=18982004&doi=10.1038/mp.2008.116&)

50. Gorwood P, Kipman A, Foulon C. The human genetics of anorexia nervosa. Eur J Pharmacol. 2003;480:163–170. doi: 10.1016/j.ejphar.2003.08.103.  [DOI](https://doi.org/10.1016/j.ejphar.2003.08.103) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14623359/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Pharmacol&title=The%20human%20genetics%20of%20anorexia%20nervosa&author=P%20Gorwood&author=A%20Kipman&author=C%20Foulon&volume=480&publication_year=2003&pages=163-170&pmid=14623359&doi=10.1016/j.ejphar.2003.08.103&)

51. Martásková D, Šlachtová L, Kemlink D, Záhoráková D, Papežová H. Polymorphisms in serotonin-related genes in anorexia nervosa. The first study in Czech population and metaanalyses with previously performed studies. Folia Biol (Praha) 2009;55:192–197.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/19863848/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Folia%20Biol%20(Praha)&title=Polymorphisms%20in%20serotonin-related%20genes%20in%20anorexia%20nervosa.%20The%20first%20study%20in%20Czech%20population%20and%20metaanalyses%20with%20previously%20performed%20studies&author=D%20Mart%C3%A1skov%C3%A1&author=L%20%C5%A0lachtov%C3%A1&author=D%20Kemlink&author=D%20Z%C3%A1hor%C3%A1kov%C3%A1&author=H%20Pape%C5%BEov%C3%A1&volume=55&publication_year=2009&pages=192-197&pmid=19863848&)

52. Anguelova M, Benkelfat C, Turecki G. A systematic review of association studies investigating genes coding for serotonin receptors and the serotonin transporter: II. Suicidal behavior. Mol Psychiatry. 2003;8:646–653. doi: 10.1038/sj.mp.4001336.  [DOI](https://doi.org/10.1038/sj.mp.4001336) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12874600/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Psychiatry&title=A%20systematic%20review%20of%20association%20studies%20investigating%20genes%20coding%20for%20serotonin%20receptors%20and%20the%20serotonin%20transporter:%20II.%20Suicidal%20behavior&author=M%20Anguelova&author=C%20Benkelfat&author=G%20Turecki&volume=8&publication_year=2003&pages=646-653&pmid=12874600&doi=10.1038/sj.mp.4001336&)

53. Gizer IR, Ficks C, Waldman ID. Candidate gene studies of ADHD: a meta-analytic review. Hum Genet. 2009;126:51–90. doi: 10.1007/s00439-009-0694-x.  [DOI](https://doi.org/10.1007/s00439-009-0694-x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19506906/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Genet&title=Candidate%20gene%20studies%20of%20ADHD:%20a%20meta-analytic%20review&author=IR%20Gizer&author=C%20Ficks&author=ID%20Waldman&volume=126&publication_year=2009&pages=51-90&pmid=19506906&doi=10.1007/s00439-009-0694-x&)

54. Pinsonneault JK, Papp AC, Sadee W. Allelic mRNA expression of X-linked monoamine oxidase a (MAOA) in human brain: dissection of epigenetic and genetic factors. Hum Mol Genet. 2006;15:2636–2649. doi: 10.1093/hmg/ddl192.  [DOI](https://doi.org/10.1093/hmg/ddl192) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16893905/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=Allelic%20mRNA%20expression%20of%20X-linked%20monoamine%20oxidase%20a%20(MAOA)%20in%20human%20brain:%20dissection%20of%20epigenetic%20and%20genetic%20factors&author=JK%20Pinsonneault&author=AC%20Papp&author=W%20Sadee&volume=15&publication_year=2006&pages=2636-2649&pmid=16893905&doi=10.1093/hmg/ddl192&)

55. Caspi A, Sugden K, Moffitt TE, Taylor A, Craig IW, Harrington H, et al. Influence of life stress on depression: moderation by a polymorphism in the 5-HTT gene. Science. 2003;301:386–389. doi: 10.1126/science.1083968.  [DOI](https://doi.org/10.1126/science.1083968) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12869766/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Influence%20of%20life%20stress%20on%20depression:%20moderation%20by%20a%20polymorphism%20in%20the%205-HTT%20gene&author=A%20Caspi&author=K%20Sugden&author=TE%20Moffitt&author=A%20Taylor&author=IW%20Craig&volume=301&publication_year=2003&pages=386-389&pmid=12869766&doi=10.1126/science.1083968&)
