# ABCB1 (MDR1) polymorphisms and ovarian cancer progression and survival: A comprehensive analysis from the Ovarian Cancer Association Consortium and The Cancer Genome Atlas

## Metadata
**Authors:** Sharon E Johnatty, Jonathan Beesley, Bo Gao, Xiaoqing Chen, Yi Lu, Matthew H Law, Michelle J Henderson, Amanda J Russell, Ellen L Hedditch, Catherine Emmanuel, Sian Fereday, Penelope M Webb, Australian Ovarian Cancer Study Group, Ellen L Goode, Robert A Vierkant, Brooke L Fridley, Julie M Cunningham, Peter A Fasching, Matthias W Beckmann, Arif B Ekici, Estrid Hogdall, Susanne K Kjaer, Allan Jensen, Claus Hogdall, Robert Brown, Jim Paul, Sandrina Lambrechts, Evelyn Despierre, Ignace Vergote, Jenny Lester, Beth Y Karlan, Florian Heitz, Andreas du Bois, Philipp Harter, Ira Schwaab, Yukie Bean, Tanja Pejovic, Douglas A Levine, Marc T Goodman, Michael E Camey, Pamela J Thompson, Galina Lurie, Joellen Shildkraut, Andrew Berchuck, Kathryn L Terry, Daniel W Cramer, Murray D Norris, Michelle Haber, Stuart MacGregor, Anna deFazio, Georgia Chenevix-Trench
**Journal:** Gynecologic oncology
**Date:** 2013 Aug 1
**DOI:** [10.1016/j.ygyno.2013.07.107](https://doi.org/10.1016/j.ygyno.2013.07.107)
**PMID:** 23917080
**PMCID:** PMC3795832
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3795832/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC3795832/pdf/nihms511782.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC3795832/pdf/nihms511782.pdf)

## Abstract

**Objective:** 
ABCB1 encodes the multi-drug efflux pump P-glycoprotein (P-gp) and has been implicated in multi-drug resistance. We comprehensively evaluated this gene and flanking regions for an association with clinical outcome in epithelial ovarian cancer (EOC).

**Methods:** 
The best candidates from fine-mapping analysis of 21 ABCB1 SNPs tagging C1236T (rs1128503), G2677T/A (rs2032582), and C3435T (rs1045642) were analysed in 4,616 European invasive EOC patients from thirteen Ovarian Cancer Association Consortium (OCAC) studies and The Cancer Genome Atlas (TCGA). Additionally we analysed 1,562 imputed SNPs around ABCB1 in patients receiving cytoreductive surgery and either ‘standard’ first-line paclitaxel-carboplatin chemotherapy (n=1,158) or any first-line chemotherapy regimen (n=2,867). We also evaluated ABCB1 expression in primary tumours from 143 EOC patients.

**Result:** 
Fine-mapping revealed that rs1128503, rs2032582, and rs1045642 were the best candidates in optimally debulked patients. However, we observed no significant association between any SNP and either progression-free survival or overall survival in analysis of data from 14 studies. There was a marginal association between rs1128503 and overall survival in patients with nil residual disease (HR 0.88, 95% CI 0.77-1.01; p=0.07). In contrast, ABCB1 expression in the primary tumour may confer worse prognosis in patients with sub-optimally debulked tumours.

**Conclusion:** 
Our study represents the largest analysis of ABCB1 SNPs and EOC progression and survival to date, but has not identified additional signals, or validated reported associations with progression-free survival for rs1128503, rs2032582, and rs1045642. However, we cannot rule out the possibility of a subtle effect of rs1128503, or other SNPs linked to it, on overall survival.

### Objective

*ABCB1* encodes the multi-drug efflux pump P-glycoprotein (P-gp) and has been implicated in multi-drug resistance. We comprehensively evaluated this gene and flanking regions for an association with clinical outcome in epithelial ovarian cancer (EOC).

### Methods

The best candidates from fine-mapping analysis of 21 *ABCB1* SNPs tagging C1236T (rs1128503), G2677T/A (rs2032582), and C3435T (rs1045642) were analysed in 4,616 European invasive EOC patients from thirteen Ovarian Cancer Association Consortium (OCAC) studies and The Cancer Genome Atlas (TCGA). Additionally we analysed 1,562 imputed SNPs around *ABCB1* in patients receiving cytoreductive surgery and either ‘standard’ first-line paclitaxel-carboplatin chemotherapy (n=1,158) or any first-line chemotherapy regimen (n=2,867). We also evaluated *ABCB1* expression in primary tumours from 143 EOC patients.

### Result

Fine-mapping revealed that rs1128503, rs2032582, and rs1045642 were the best candidates in optimally debulked patients. However, we observed no significant association between any SNP and either progression-free survival or overall survival in analysis of data from 14 studies. There was a marginal association between rs1128503 and overall survival in patients with nil residual disease (HR 0.88, 95% CI 0.77-1.01; p=0.07). In contrast, *ABCB1* expression in the primary tumour may confer worse prognosis in patients with sub-optimally debulked tumours.

### Conclusion

Our study represents the largest analysis of *ABCB1* SNPs and EOC progression and survival to date, but has not identified additional signals, or validated reported associations with progression-free survival for rs1128503, rs2032582, and rs1045642. However, we cannot rule out the possibility of a subtle effect of rs1128503, or other SNPs linked to it, on overall survival.

## Introduction

Over the past three decades, significant advances have been made in chemotherapy for ovarian cancer, and the combination of cytoreductive surgery followed by the doublet of a taxane (paclitaxel 135 – 175 mg/m^2^) and carboplatin (AUC > 5), has been the most common regimen for primary treatment of this disease [[1](#R1)]. However, resistance to chemotherapy remains a major challenge to treatment, and although multidrug resistance (MDR) has been widely studied in *in vitro* models, the translational utility of this research remains elusive. The identification and analysis of genes relevant to MDR in patients may contribute to a better understanding of this phenomenon and potentially help circumvent this obstacle.

ABCB1, the most extensively studied ATP-binding cassette (ABC) transporter, encodes the multi-drug efflux pump P-glycoprotein (P-gp) and is involved in the transport of a wide range of anti-cancer drugs including paclitaxel [[2](#R2), [3](#R3)]. *ABCB1*, located at 7q21.12, is genetically very variable, and it is likely that single nucleotide polymorphisms (SNPs) in this gene may have significant effects on the expression and function of P-gp, and hence the absorption, metabolism and clearance of P-gp substrates. Of particular interest has been the nonsynonymous variant G2677T/A (rs2032582) which results in an amino acid change at codon 893 (Ala893Ser/Thr) in exon 21 and forms part of a common haplotype that includes two synonymous SNPs, C1236T (rs1128503) in exon 12 and C3435T (rs1045642) in exon 26. Studies evaluating allelic or haplotypic effects of these three SNPs on mRNA or protein expression in a wide range of human tissues have been extensively reviewed and found to be inconclusive [[4](#R4), [5](#R5)]. Likewise, association studies of *ABCB1* polymorphisms in EOC cases receiving taxane-based chemotherapy regimens have to date produced conflicting and inconclusive results [[6](#R6)-[11](#R11)].

We evaluated *ABCB1* SNPs and EOC disease progression and survival among 4,616 women participating in thirteen studies from the Ovarian Cancer Association Consortium (OCAC) [[12](#R12)] and The Cancer Genome Atlas (TCGA) ([http://cancergenome.nih.gov/](http://cancergenome.nih.gov/)). We first queried the SNP(s) that could account for our reported association between rs2032582 and rs1128503 and outcome in the AOCS study [[10](#R10), [11](#R11)] by fine-mapping the *ABCB1* locus in optimally debulked AOCS patients receiving standard first-line paclitaxel-carboplatin chemotherapy. We then analysed the best SNPs in a much larger cohort of patients with more mature clinical follow-up data from the OCAC and TCGA, and evaluated an additional 1,562 SNPs on chromosome 7 flanking *ABCB1*

## Methods

### Study subjects and ethics statement

EOC patients were from thirteen OCAC studies, and TCGA. European ancestry was determined using the program LAMP [[13](#R13)] to assign intercontinental ancestry based on the HapMap (release no. 22) genotype frequency data for European, or assumed to be of European ancestry based on self-report or geographic location where genetic markers were unavailable. Clinical definitions and criteria for progression across studies have been previously described [[14](#R14)]. All studies received approval from their respective human research ethics committees, and all OCAC participants provided written informed consent. Details of TCGA can be found at [http://cancergenome.nih.gov/](http://cancergenome.nih.gov/).

### DNA extraction, genotyping, and imputation

DNA extraction, genotyping methods and quality assurance for all samples have been previously described [[11](#R11)]. Data for TCGA patients was downloaded through the TCGA data portal and assessed for ancestral outliers to determine those of European descent. Imputation of SNPs at the chromosome 7 locus was performed using minimac [[15](#R15)] following pre-phasing by MACH algorithm 1.0 [[16](#R16)] utilizing information from 70 *ABCB1* SNPs genotyped in all European ovarian cancer patients on the Illumina Infinium iSelect array designed by the Collaborative Oncological Gene-environment Study (iCOGS) [[17](#R17)]. Genotypes were imputed to the European subset of the phased chromosomes from the 1000 Genome project (version 3). Genotyping methods and quality assurance of OCAC samples included on the iCOGs platform have been previously reported [[17](#R17)]. Imputation was performed for the *ABCB1* gene and flanking sequence extending out to the first recombination peaks with a recombination rate greater than 40, and spanning 85,570,519bp – 88,736,135bp of chromosome 7 (NCBI build 37). We evaluated only those with minor allele frequency (MAF) >0.05 and reasonable imputation quality (imputed r^2^>0.3).

### SNP selection and fine mapping

We used the Tagger program within Haploview [[18](#R18)] to select 18 *ABCB1* tagging SNPs (Hapmap Phase II release #24) within a 60 kb region encompassing rs2032582, and genotyped these and our three previously reported SNPs (rs1128503, rs2032582, and rs1045642) [[10](#R10), [11](#R11)] in optimally debulked AOCS patients that met our inclusion criteria (n=433). SNPs were analysed both independently and in forward and backward log-additive stepwise Cox proportional hazards (PH) models adjusted for tumor stage and level of residual disease, using a conservative p-value of 0.05 to enter or exit the model [[19](#R19)]. We also investigated haplotype frequencies for all patients genotyped for these 21 SNPs (n=615) using the Beagle Genetic Analysis Software package v.3.3.2 for inferring haplotype phase or sporadic missing genotype data in unrelated individuals [[20](#R20)]. The likelihood ratio test was used to compare regression models of the three most common *ABCB1* haplotypes (alternate models) versus the rs2032582 (null model) to test the likelihood that any observed association was due to haplotype effects.

### ABCB1 expression analysis

RNA was prepared from 143 AOCS serous ovarian tumors obtained at surgery prior to chemotherapy using the QIAGEN RNeasy extraction kit (QIAGEN, Australia) and reverse transcribed as previously described [[21](#R21)]. Quantification of expression utilising TaqMan® assays [*ABCB1* (ID Hs00184500_m1), *GUSB* (ID 4326320E), *HPRT* (ID 4326321E) and *PGK1* (ID 4326318E)] were conducted in duplicate using a 7900HT Fast Real-Time PCR system. Relative expression was calculated by the (delta-delta) Ct method using RQ Manager (Applied Biosystems) as the geometric mean of the expression values against three control genes [[22](#R22)]. Values were log transformed and *ABCB1* expression was analysed as a continuous variable using Cox regression. Expression values for significant associations were dichotomised into low vs. high for Kaplan-Meier analysis. We also analysed 374 TCGA serous EOC cases using publicly available data ([http://tcga-data.nci.nih.gov/](http://tcga-data.nci.nih.gov/)) [[23](#R23)].

### Statistical analysis

Progression-free survival (PFS) was defined as the interval between the date of histological diagnosis and the first confirmed sign of disease progression or death, as previously described [[14](#R14)]. Overall survival (OS) was the interval between the date of histological diagnosis and death from any cause. To control for ascertainment bias, prevalent cases, defined as having an interval >12 months between the date of histological diagnosis and DNA collection, were excluded from the analysis.

Analysis of PFS and OS was restricted to European/non-Hispanic white invasive EOC patients of all histologies classified as ‘standard’ if they had ≥4 cycles of paclitaxel and carboplatin intravenously (IV) at 3-weekly intervals, or ‘all chemo’ if they had any chemotherapy regimen following cytoreductive surgery; the majority of women in the ‘standard’ cohort were known to have had paclitaxel at 175 or 135 mg/m^2^ and carboplatin AUC 5 or 6. Patients included in our previous report [[11](#R11)] were not excluded from the current analysis because patient outcome and treatment data has since been updated. To maximize the statistical power for analysis of the best fine-mapping SNPs in OCAC and TCGA studies, we combined cases with actual and well-imputed (r^2^ >0.9) genotypes where available.

Cox PH models were used to estimate the hazard ratios (HRs) and 95% confidence intervals (CIs) for PFS and OS associated with *ABCB1* SNPs in EOC patients classified as ‘standard ’ and ‘all chemo’ for first-line chemotherapy. We evaluated potential confounders of PFS and observed a highly significant association between PFS and residual disease, tumor stage, histology and grade (log-rank p <0.0001; data not shown) and therefore included these covariates in all regression models. Additionally, OS analysis was adjusted for age at diagnosis. SNPs were modelled assuming log-additive effects by fitting the number or dosage score (imputed data) of rare alleles carried as a continuous covariate, and stratified by study. We observed a statistically significant interaction between the rs2032582 genotype and residual disease (p=0.001) and analysed SNPs for nil residual disease separately, as well as optimally (≤1 cm residual disease) and sub-optimally debulked (>1cm residual disease) patients. Between-study heterogeneity was assessed using the likelihood ratio test to compare regression models with and without a genotype-by-study interaction term. The median follow-up of the analysis cohort was estimated using the reverse Kaplan-Meier method [[24](#R24)]. All tests for association were two-tailed. The significance threshold for the fine-mapping study was p<0.05. However, a threshold of p<10^-3^ was used for the best fine-mapping candidate SNPs evaluated in the larger EOC cohort because we explored sixteen independent hypotheses according to chemotherapy regimen and debulking status. For the 1,562 imputed SNPs we used a modified Bonferroni adjustment that takes into account linkage disequilibrium (LD) between SNPs [[25](#R25), [26](#R26)], yielding a significance threshold of 4.6×10^-4^ required to keep the Type I error rate at 5%. All analyses were performed in STATA SE v. 11 (Stata Corp., USA), SPSS, and the R project for Statistical Computing v2.14 ([http://www.r-project.org/](http://www.r-project.org/)).

## Results

Details of study design, case ascertainment, and clinical characteristics of all women from participating OCAC studies have been described elsewhere [[27](#R27), [28](#R28)] and are summarized in [Tables S1 and S2](#SD2). A total of 4,942 European/non-Hispanic White women with invasive EOC and detailed clinical and treatment data were eligible for inclusion in the analysis. The median follow-up of the analysis cohort was 5.9 (95% CI 5.7 – 6.0) years. All genotype data included in this analysis conformed to quality assurance criteria for inclusion.

Progression-free survival (PFS) estimates from 18 *ABCB1* tagging SNPs selected for preliminary fine-mapping, and the three coding SNPs (rs1128503, rs2032582, and rs1045642) analysed in a subset of patients with optimal debulking from the AOCS (residual disease ≤ 1 cm, n=433) are shown in [Table 1](#T1). Nine SNPs that were significantly associated with PFS (p<0.05) in adjusted single-SNP analyses were then analysed by stepwise regression. Only rs2032582 remained significantly associated with PFS in forward stepwise modeling (p=0.03). Likewise, in the backward stepwise procedure, the sequential removal of each of the other eight SNPs successively improved the significance level of rs2032582 to that of the single SNP. The most informative model included one other SNP, rs4148738 (p=0.086) which is perfectly correlated with rs2032582 in the1000 Genomes reference population. We therefore concluded that rs2032582 best explains the observed association with PFS in this sample.

### Table 1.

| SNP | Gene Location | LD Block | bChr position | Unadjusted |  |  | aAdjusted |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
|   |  |  |  |  |  |  |  |  |  |
| HR | 95% CI | P | HR | 95% CI | P |  |  |  |  |
| rs7793196 | Intergenic | 1 | 86767498 | 1.06 | (0.84 -1.34) | 0.619 | 1.09 | (0.86 -1.37) | 0.47 |
| rs6946119 | 3’ Flank | 1 | 86773516 | 1.22 | (0.99 -1.51) | 0.064 | 1.15 | (0.94 -1.42) | 0.175 |
| rs6979885 | Intronic | 2 | 86782112 | 1.35 | (1.09 -1.68) | 0.007 | 1.26 | (1.02 -1.56) | 0.033 |
| rs2235048 | Intronic | 2 | 86783162 | 1.32 | (1.09 -1.61) | 0.004 | 1.26 | (1.05 -1.52) | 0.015 |
| rs1045642 | Exonic (sSNP) |   | 86783296 | 1.34 | (1.10 -1.62) | 0.003 | 1.25 | (1.03 -1.51) | 0.022 |
| rs2032582 | Exonic (nsSNP) | 3 | 86805269 | 0.74 | (0.61 -0.88) | 0.001 | 0.74 | (0.62 -0.89) | 0.001 |
| rs4148738 | Intronic | 3 | 86807700 | 0.76 | (0.63 -0.92) | 0.006 | 0.79 | (0.65 -0.96) | 0.015 |
| rs10276603 | Intronic | 3 | 86816178 | 1.01 | (0.77 -1.34) | 0.919 | 1.07 | (0.81 -1.41) | 0.642 |
| rs2091766 | Intronic | 3 | 86819155 | 1.23 | (1.00 -1.50) | 0.045 | 1.12 | (0.91 -1.38) | 0.272 |
| rs1128503 | Exonic (sSNP) |   | 86824252 | 0.8 | (0.66 -0.97) | 0.022 | 0.85 | (0.70 -1.04) | 0.119 |
| rs12704364 | Intronic | 3 | 86825826 | 1.24 | (1.01 -1.51) | 0.035 | 1.17 | (0.96 -1.42) | 0.13 |
| rs956825 | Intronic | 3 | 86836926 | 1.11 | (0.91 -1.36) | 0.292 | 1.04 | (0.85 -1.28) | 0.686 |
| rs10260862 | Intronic | 4 | 86846133 | 1.14 | (0.90 -1.45) | 0.271 | 1.15 | (0.91 -1.46) | 0.23 |
| rs10264990 | Intronic | singleton | 86847266 | 1.4 | (1.13 -1.74) | 0.002 | 1.33 | (1.07 -1.65) | 0.01 |
| rs1202174 | Intronic | 5 | 86854023 | 1.17 | (0.96 -1.43) | 0.12 | 1.19 | (0.97 -1.45) | 0.096 |
| rs17327442 | Intronic | 5 | 86857641 | 1.22 | (0.91 -1.62) | 0.186 | 1.15 | (0.86 -1.55) | 0.345 |
| rs1202184 | Intronic | 5 | 86858552 | 1.25 | (1.03 -1.52) | 0.025 | 1.24 | (1.02 -1.51) | 0.03 |
| rs1211152 | Intronic | 5 | 86859765 | 1.31 | (0.87 -1.97) | 0.195 | 1.3 | (0.86 -1.96) | 0.21 |
| rs17327624 | Intronic | singleton | 86861468 | 1.01 | (0.79 -1.30) | 0.925 | 0.98 | (0.76 -1.26) | 0.871 |
| rs2188526 | Intronic | 6 | 86865213 | 0.8 | (0.65 -0.97) | 0.022 | 0.77 | (0.63 -0.94) | 0.009 |
| rs13233308 | Intronic | singleton | 86889611 | 0.81 | (0.67 -0.98) | 0.028 | 0.78 | (0.64 -0.94) | 0.01 |

Table 1 Caption: Independent SNP effects on progression-free survival in ABCB1 tagging SNPs selected for fine-mapping in optimally debulked AOCS cases (n=433)

Haplotypes estimated for 1230 chromosomes in the 615 women with genotype data for the 21 SNPs revealed that the most common haplotypes contained the minor alleles of rs1128503, rs2032582, and rs1045642 ([Table S3](#SD2)). Likelihood ratio tests comparing regression models in optimally debulked carriers of the three most common haplotypes vs. carriers of the minor allele of rs2032582 only, showed no significant improvement in PFS estimates attributable to haplotypes (likelihood ratio p=0.8). We further expanded analysis of the rs1128503- rs2032582-rs1045642 haplotype to 3,437 invasive EOC cases from OCAC with genotype data and observed a similar distribution of haplotypes as seen in the preliminary fine mapping exercise of AOCS cases only. No significant haplotype effects were observed in any subset of cases in analysis of PFS ([Table S4](#SD2)).

Based on our fine-mapping and haplotype analysis, as well as a recent report that rs1045642 and rs1128503 may indeed be functionally relevant though silent [[29](#R29)], we evaluated the rs1128503, rs2032582, and rs1045642 SNPs in the largest available dataset of 4,461 European invasive EOC patients with actual (≥80%) and well-imputed genotypes (imputation r^2^>0.9) for these three SNPs. We observed only a marginal inverse association between the rare allele of rs1128503 and OS in patients with nil residual disease treated with any chemotherapy (HR 0.88, 95% CI 0.77-1.01; p=0.07) ([Table 2](#T2) & [Figure 1](#F1)). A similar effect was observed for rs2032582 (p=0.08; r^2^ =0.9 with rs1128503). We observed no significant association between any of the three SNPs and either PFS or OS in invasive EOC or any other subset for residual disease or treatment groups ([Table 2](#T2)). Additionally, we explored the dominant model and found no evidence to support dominant effects for these SNPs (data not shown). There was no evidence of between-study heterogeneity for any of these three SNPs (p≥0.2).

### Table 2.

| SNP | aAnalysis subset | bN | % Relapsed | % Died | cProgression-free survival |  |  | c,dOverall Survival |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
|   |  |  |  |  |  |  |  |  |  |  |
| HR | 95% CI | P | HR | 95% CI | P |  |  |  |  |  |
| G2677TA (rs2032582) | All Cases |  |  |  |  |  |  |  |  |  |
| Standard | 1,882 | 70.14 | 51.91 | 0.99 | (0.91 -1.08) | 0.83 | 1.00 | (0.90 -1.10) | 0.92 |  |
| All chemo | 4,450 | 67.30 | 52.09 | 1.02 | (0.96 -1.07) | 0.55 | 1.00 | (0.94 -1.07) | 0.93 |  |
| Nil Residual Disease |  |  |  |  |  |  |  |  |  |  |
| Standard | 687 | 48.18 | 31.15 | 1.03 | (0.87 -1.22) | 0.75 | 1.00 | (0.81 -1.23) | 0.99 |  |
| All chemo | 1,665 | 43.96 | 30.27 | 0.95 | (0.85 -1.06) | 0.37 | 0.89 | (0.77 -1.01) | 0.08 |  |
| Optimally debulked (≤1cm) |  |  |  |  |  |  |  |  |  |  |
| Standard | 1,118 | 61.72 | 46.15 | 1.00 | (0.90 -1.12) | 0.97 | 0.97 | (0.85 -1.10) | 0.61 |  |
| All chemo | 2,600 | 56.96 | 43.88 | 0.98 | (0.91 -1.06) | 0.68 | 0.95 | (0.87 -1.03) | 0.21 |  |
| Sub-optimally debulked (>1cm) |  |  |  |  |  |  |  |  |  |  |
| Standard | 672 | 84.52 | 60.86 | 1.01 | (0.89 -1.15) | 0.84 | 1.05 | (0.91 -1.22) | 0.49 |  |
| All chemo | 1,654 | 83.68 | 64.81 | 1.05 | (0.97 -1.14) | 0.22 | 1.05 | (0.96 -1.15) | 0.24 |  |
| C1236T (rs1128503) | All Cases |  |  |  |  |  |  |  |  |  |
| Standard | 1,873 | 70.10 | 51.79 | 1.02 | (0.94 -1.11) | 0.57 | 0.98 | (0.89 -1.08) | 0.75 |  |
| All chemo | 4,436 | 67.22 | 51.98 | 1.04 | (0.98 -1.10) | 0.19 | 1.00 | (0.94 -1.07) | 0.97 |  |
| Nil Residual Disease |  |  |  |  |  |  |  |  |  |  |
| Standard | 681 | 48.02 | 30.84 | 1.07 | (0.90 -1.26) | 0.46 | 0.95 | (0.77 -1.17) | 0.63 |  |
| All chemo | 1,659 | 43.88 | 30.14 | 1.00 | (0.89 -1.11) | 0.96 | 0.88 | (0.77 -1.01) | 0.07 |  |
| Optimally debulked (≤1cm) |  |  |  |  |  |  |  |  |  |  |
| Standard | 1,110 | 61.62 | 46.04 | 1.04 | (0.93 -1.16) | 0.50 | 0.94 | (0.83 -1.07) | 0.35 |  |
| All chemo | 2,592 | 56.87 | 43.79 | 1.01 | (0.94 -1.09) | 0.76 | 0.95 | (0.87 -1.03) | 0.20 |  |
| Sub-optimally debulked (>1cm) |  |  |  |  |  |  |  |  |  |  |
| Standard | 671 | 84.50 | 60.66 | 1.05 | (0.92 -1.19) | 0.49 | 1.08 | (0.93 -1.26) | 0.29 |  |
| All chemo | 1,648 | 83.62 | 64.68 | 1.07 | (0.98 -1.15) | 0.12 | 1.06 | (0.97 -1.16) | 0.23 |  |
| C3435T (rs1045642) | All Cases |  |  |  |  |  |  |  |  |  |
| Standard | 1,873 | 70.10 | 51.79 | 0.99 | (0.91 -1.07) | 0.81 | 0.97 | (0.88 -1.06) | 0.51 |  |
| All chemo | 4,435 | 67.24 | 51.97 | 0.99 | (0.94 -1.04) | 0.64 | 1.00 | (0.94 -1.06) | 0.99 |  |
| Nil Residual Disease |  |  |  |  |  |  |  |  |  |  |
| Standard | 681 | 48.02 | 30.84 | 0.94 | (0.80 -1.11) | 0.49 | 0.92 | (0.74 -1.14) | 0.44 |  |
| All chemo | 1,658 | 43.91 | 30.10 | 1.00 | (0.89 -1.11) | 0.98 | 1.06 | (0.93 -1.22) | 0.37 |  |
| Optimally debulked (≤1cm) |  |  |  |  |  |  |  |  |  |  |
| Standard | 1,110 | 61.62 | 46.04 | 0.98 | (0.88 -1.09) | 0.68 | 0.98 | (0.87 -1.11) | 0.77 |  |
| All chemo | 2,591 | 56.89 | 43.77 | 0.98 | (0.91 -1.05) | 0.51 | 1.01 | (0.93 -1.10) | 0.77 |  |
| Sub-optimally debulked (>1cm) |  |  |  |  |  |  |  |  |  |  |
| Standard | 671 | 84.50 | 60.66 | 0.98 | (0.87 -1.11) | 0.77 | 0.93 | (0.80 -1.08) | 0.33 |  |
| All chemo | 1,648 | 83.62 | 64.68 | 1.00 | (0.93 -1.08) | 0.99 | 1.00 | (0.91 -1.09) | 0.95 |  |

Table 2 Caption: Estimates for three ABCB1 coding SNPs in invasive ovarian cancer cases according to treatment group and debulking status.

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d533/3795832/88735b2fb4f8/nihms511782f1.jpg)

Forest plot of study-specific estimates for rs1128503 and overall survival for patients with nil residual disease receiving any chemotherapy regimen Estimates for HAW and NCO missing due to insufficient data for this subgroup

Finally, 1,562 of the 2,745 imputed chromosome 7 SNPs with MAF >0.05 and imputation r^2^>0.3 were analysed in ~3,000 European invasive EOC patients but no evidence of an association with PFS or OS at the pre-specified threshold of p<4.6 × 10^-4^ was observed in any of the chemotherapy subsets ([Tables S5 – S8](#SD2)).

ABCB1 mRNA expression levels in 143 AOCS serous tumors indicated that high-level *ABCB1* expression was associated with shorter PFS both overall (p=0.003) and in sub-optimally debulked patients (p=0.02; [Figure 2](#F2)), although the effect diminished upon adjustment for residual disease status and tumor stage ([Table 3](#T3)). The mean expression of *ABCB1* was higher in sub-optimally debulked patients compared to optimally debulked (1.48 ±0.1 vs 2.74 ±0.79, p=0.04) suggesting a possible link between ABCB1 and disease progression in patients unable to be optimally debulked. In the TCGA dataset, a trend for shorter OS was observed in sub-optimally debulked patients, which was substantially strengthened if patients were uniformly treated with regard to paclitaxel ([Table 3](#T3)).

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d533/3795832/137ce9873205/nihms511782f2.jpg)

Kaplan-Meier survival analysis of ABCB1 gene expression in serous ovarian cancer. In the AOCS cohort, high expression of ABCB1 is associated with shorter PFS, either (A) in the overall cohort, or (B) in the subgroup of patients with sub-optimally debulked tumors. Kaplan-Meier curves were generated following dichotomisation of gene expression to high or low expression with respect to the optimal cut-point, determined as the 15th percentile in the whole cohort, or the 30th percentile in the sub-optimally debulked subgroup. P-values were calculated according to the log-rank test.

### Table 3.

| Cohort | N | % Relapsed | % Died | Progression-free survival |  |  | Overall Survival |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
|   |  |  |  |  |  |  |  |  |  |
| HR | 95% CI | P | HR | 95% CI | P |  |  |  |  |
| AOCS |  |  |  |  |  |  |  |  |  |
| Univariate |   |   |   |   |   |   |   |   |   |
| ABCB1 expressiona | 143 | 81.1 | 56.6 | 1.32 | (1.03 -1.70) | 0.028 | 1.20 | (0.92 -1.55) | 0.173 |
| Multivariate model |   |   |   |   |   |   |   |   |   |
| ABCB1 expressiona | 143 | 81.1 | 56.6 | 1.26 | (0.99 -1.62) | 0.062 | 1.09 | (0.85 -1.4) | 0.50 |
| Residual disease >1cm |   |   |   | 2.00 | (1.36 -2.95) | <0.001 | 1.95 | (1.24 -3.07) | 0.004 |
| FIGO Stageb |   |   |   | 4.61 | (1.84 -11.49) | 0.001 | 10.53 | (1.45 -76.9) | 0.02 |
| TCGA Sub-optimally debulked patients (>1cm) |  |  |  |  |  |  |  |  |  |
| All chemo | 91 | 69.2 | 58.2 | 1.24 | (0.78 -1.97) | 0.36 | 1.58 | (0.98 -2.57) | 0.06 |
| Standard | 41 | 95.1 | 61 | 1.83 | (0.96 -3.49) | 0.066 | 3.39 | (1.51 -7.62) | 0.003 |

Table 3 Caption: Analysis of ABCB1 expression in serous EOC tumors

## Discussion

We have comprehensively analysed the largest study to date of *ABCB1* SNPs and clinical outcome among women with invasive EOC treated with paclitaxel-based chemotherapy. Fine-mapping analysis suggested that rs2032582 and the two silent SNPs (rs1128503 and rs1045642) were indeed the best candidates [[29](#R29), [30](#R30)]. However analysis of these SNPs in 4,616 European/non-Hispanic White invasive EOC patients from the OCAC studies and TCGA showed only a marginal association between rs1128503 (pair-wise r^2^ with rs2032582 = 0.9) and overall survival in patients with nil residual disease treated with any first-line chemotherapy regimen ([Table 2](#T2) & [Figure 1](#F1)). Although this association did not meet the threshold for multiple testing, we cannot rule out a small effect of this SNP or another linked to it on overall survival. It is likely that either SNP is modifying the effect of residual disease, a significant predictor of outcome, on overall survival independently of chemotherapy regimen, and warrants further investigation. There was no convincing evidence of an association between PFS or OS and any of the 1,562 SNPs analysed in either the ‘standard’ or ‘all chemo’ subsets ([Tables S5 – S8](#SD2)).

A recent systematic literature review of predictive and prognostic studies of ovarian cancer genetic markers including *ABCB1* highlighted the non-uniformity of outcome definitions and treatment approaches that limit interpretation and clinical utility, and the need for adequately powered, well-designed observational studies that might help to resolve the present heterogeneity of findings among pharmacogenetic studies [[31](#R31)]. Our patient data was subjected to stringent quality assurance measures with regard to chemotherapy data in an effort to avoid disparities in treatment exposure. Chemotherapy data for each study was obtained from patient charts according to standardized definitions, by data extractors who were blinded to our study hypothesis. Cases were included in the ‘standard’ chemotherapy subset if sufficient details of first-line chemotherapy confirmed that they had in fact received carboplatin and paclitaxel. Approximately 71% of women in this subset received a minimum of four cycles of paclitaxel at 175 or 135 mg/m^2^ and carboplatin AUC 5 or 6. The other 29% in this subgroup were presumed to have received ≥ 4 cycles of paclitaxel and carboplatin at these doses based on patient data that suggested that this was the planned treatment approach, with due consideration to toxicity data and other variables that might indicate that the doses or cycles of chemotherapy actually received were not as planned. Cases known to have received < 4 cycles of paclitaxel and carboplatin due to dose reduction or toxicity were included only in the ‘all chemo’ analyses. Analysis of the ‘all chemo’ dataset was designed to test whether any observed SNP associations from the ‘standard’ subsets would be diminished if indeed such associations were directly linked to exposure to a P-gp substrate.

To circumvent methodological flaws we restricted the analysis to European invasive EOC cases participating in the OCAC with standardized definitions of clinical and pathological characteristics. All analyses were stratified by study to account for unknown sources of variation, although baseline PFS by OCAC study has previously been shown to be comparable [[14](#R14)]. The median follow-up of the entire cohort of 4,942 eligible cases was 5.9 (95% CI 5.7 – 6.0) years. Given the prognosis of invasive ovarian cancer, it is highly likely that deaths in this cohort were due to ovarian cancer, and therefore represents a reasonable estimate of the association between SNPs and disease-specific survival.

Despite our rigorous approach to data quality and our null findings, we cannot rule out the possibility that some polymorphisms around the *ABCB1* locus may have a very subtle effect on outcomes. Expression studies suggest that variation in P-gp function is a putative mechanism for MDR in a wide range of disease [[4](#R4)] and may at least in part contribute to paclitaxel-based chemotherapy outcomes. It is possible that there are long range eQTLs that influence the expression of P-gp that were not genotyped in our fine mapping study of *ABCB1*. In terms of tumor-intrinsic ABCB1 function, whilst high levels of ABCB1 can clearly confer taxane resistance to cancer cells *in vitro*, the outcomes of numerous studies examining a potential clinical role are diverse and inconclusive. The current study suggests that *ABCB1* expression in primary tumors may influence the time to relapse, particularly in the subgroup of patients with significant residual tumor post-surgery. The relationship between *ABCB1* SNPs and expression has been intensively studied and sometimes weak associations observed between the T allele of rs1045642 and expression in cancer cells [[4](#R4)]. In the relatively large dataset of combined TCGA and AOCS patients (n=536), we found no association between the three coding SNPs, rs1045642, rs1128503 and rs2032582 and *ABCB1* expression levels ([Table S9](#SD2)).

There remains a great need to implement high-quality pharmacogenomic studies in EOC because the causes of differential treatment outcomes remain unknown. Promising new biological entities are being tested for concurrent administration with paclitaxel-based treatment regimens [[32](#R32)] which require prolonged administration, and the identification of genetic markers may enhance the clinical approaches and cost-effectiveness of these treatment approaches. However, large clinical trials or well-designed prospective cohort studies that take into account differential responses according to EOC tumor types are required to succeed in defining the role of genetics in treatment response.

## Supplementary Material

## Acknowledgments

This study would not have been possible without the contributions of the following: Per Hall (COGS); Douglas F. Easton (BCAC), Andrew Berchuck (OCAC), Rosalind A. Eeles, Douglas F. Easton, Ali Amin Al Olama, Zsofia Kote-Jarai (PRACTICAL), Georgia Chenevix-Trench, Antonis Antoniou, Fergus Couch and Ken Offit (CIMBA), Joe Dennis, Alison M. Dunning, Andrew Lee, and Ed Dicks (Cambridge), Javier Benitez, Anna Gonzalez-Neira and the staff of the CNIO genotyping unit, Jacques Simard and Daniel C. Tessier, Francois Bacot, Daniel Vincent, Sylvie LaBoissière and Frederic Robidoux and the staff of the McGill University and Génome Québec Innovation Centre, Stig E. Bojesen, Sune F. Nielsen, Borge G. Nordestgaard, and the staff of the Copenhagen DNA laboratory, and Julie M. Cunningham, Sharon A. Windebank, Christopher A. Hilker, Jeffrey Meyer and the staff of Mayo Clinic Genotyping Core Facility

The authors also acknowledge the cooperation of all participating institutions and the contributions of the women who participated in this study. The full AOCS Study Group is available at [http://www.aocstudy.org/](http://www.aocstudy.org/)

The results published here are in part based upon data generated by The Cancer Genome Atlas Pilot Project established by the National Cancer Institute and National Human Genome Research Institute. Information about TCGA can be found at [http://cancergenome.nih.gov/](http://cancergenome.nih.gov/).

Children’s Cancer Institute Australia is affiliated with the University of New South Wales and Sydney Children’s Hospital.

**Grant Support**

Funding for the iCOGS infrastructure came from: the European Community’s Seventh Framework Programme under grant agreement n° 223175 (HEALTH-F2-2009-223175) (COGS), Cancer Research UK (C1287/A10118, C1287/A 10710, C12292/A11174, C5047/A8384, C5047/A15007, C5047/A10692), the National Institutes of Health (CA128978) and Post-Cancer GWAS initiative (No. 1 U19 CA 148537 - the GAME-ON initiative), the Department of Defence (W81XWH-10-1-0341), the Canadian Institutes of Health Research (CIHR) for the CIHR Team in Familial Risks of Breast Cancer, Komen Foundation for the Cure, the Breast Cancer Research Foundation, and the Ovarian Cancer Research Fund.

The AOCS was supported by the U.S. Army Medical Research and Materiel Command under DAMD17-01-1-0729, the National Health and Medical Research Council (NHMRC) of Australia (400413, 400281), Cancer Council Victoria, Cancer Council Queensland, Cancer Council New South Wales, Cancer Council South Australia, The Cancer Foundation of Western Australia, and Cancer Council Tasmania. G. Chenevix-Trench is a Senior Principal Research fellow of the NHMRC, and this work was supported by NHMRC and Cancer Australia funding. Y. Li is funded by NHMRC grant 496675, and S. MacGregor is supported by an NHMRC career development award, B Gao was supported by an NHMRC Scholarship and a Cancer Institute NSW Research Scholar Award.

The Mayo Clinic study (MAY) was supported by R01 CA122443, P50 CA136393.

SCOTROC biological studies were supported by Cancer Research UK (grant C536/A6689).

The LAX study (Women’s Cancer Program) was supported by the American Cancer Society Early Detection Professorship (120950-SIOP-06-258-06-COUN) and Entertainment Industry Foundation.

The HAW study was supported by grant R01-CA58598 from the US National Institutes of Health.

The NEC study was supported by grant R01-CA54419 from the US national Institutes of Health and W81XWH-10-0280 from the Department of Defense.

The Danish Malignant Ovarian Tumor Study (MAL) was supported by a Danish Cancer Society research grant (94 222 52) and by the Mermaid I project

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Marchetti C, Pisano C, Facchini G, Bruni GS, Magazzino FP, Losito S, et al. First-line treatment of advanced ovarian cancer: current research and perspectives. Expert Rev Anticancer Ther. 2010;10:47–60. doi: 10.1586/era.09.167.  [DOI](https://doi.org/10.1586/era.09.167) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20014885/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Expert%20Rev%20Anticancer%20Ther&title=First-line%20treatment%20of%20advanced%20ovarian%20cancer:%20current%20research%20and%20perspectives&author=C%20Marchetti&author=C%20Pisano&author=G%20Facchini&author=GS%20Bruni&author=FP%20Magazzino&volume=10&publication_year=2010&pages=47-60&pmid=20014885&doi=10.1586/era.09.167&)

2. Juliano RL, Ling V. A surface glycoprotein modulating drug permeability in Chinese hamster ovary cell mutants. Biochim Biophys Acta. 1976;455:152–62. doi: 10.1016/0005-2736(76)90160-7.  [DOI](https://doi.org/10.1016/0005-2736(76)90160-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/990323/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochim%20Biophys%20Acta&title=A%20surface%20glycoprotein%20modulating%20drug%20permeability%20in%20Chinese%20hamster%20ovary%20cell%20mutants&author=RL%20Juliano&author=V%20Ling&volume=455&publication_year=1976&pages=152-62&pmid=990323&doi=10.1016/0005-2736(76)90160-7&)

3. Orina JN, Calcagno AM, Wu CP, Varma S, Shih J, Lin M, et al. Evaluation of current methods used to analyze the expression profiles of ATP-binding cassette transporters yields an improved drug-discovery database. Mol Cancer Ther. 2009;8:2057–66. doi: 10.1158/1535-7163.MCT-09-0256.  [DOI](https://doi.org/10.1158/1535-7163.MCT-09-0256) | [PMC free article](/articles/PMC2736804/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19584229/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Cancer%20Ther&title=Evaluation%20of%20current%20methods%20used%20to%20analyze%20the%20expression%20profiles%20of%20ATP-binding%20cassette%20transporters%20yields%20an%20improved%20drug-discovery%20database&author=JN%20Orina&author=AM%20Calcagno&author=CP%20Wu&author=S%20Varma&author=J%20Shih&volume=8&publication_year=2009&pages=2057-66&pmid=19584229&doi=10.1158/1535-7163.MCT-09-0256&)

4. Leschziner GD, Andrew T, Pirmohamed M, Johnson MR. ABCB1 genotype and PGP expression, function and therapeutic drug response: a critical review and recommendations for future research. Pharmacogenomics J. 2007;7:154–79. doi: 10.1038/sj.tpj.6500413.  [DOI](https://doi.org/10.1038/sj.tpj.6500413) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16969364/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=ABCB1%20genotype%20and%20PGP%20expression,%20function%20and%20therapeutic%20drug%20response:%20a%20critical%20review%20and%20recommendations%20for%20future%20research&author=GD%20Leschziner&author=T%20Andrew&author=M%20Pirmohamed&author=MR%20Johnson&volume=7&publication_year=2007&pages=154-79&pmid=16969364&doi=10.1038/sj.tpj.6500413&)

5. Sissung TM, Baum CE, Kirkland CT, Gao R, Gardner ER, Figg WD. Pharmacogenetics of membrane transporters: an update on current approaches. Mol Biotechnol. 2010;44:152–67. doi: 10.1007/s12033-009-9220-6.  [DOI](https://doi.org/10.1007/s12033-009-9220-6) | [PMC free article](/articles/PMC6362991/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19950006/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Biotechnol&title=Pharmacogenetics%20of%20membrane%20transporters:%20an%20update%20on%20current%20approaches&author=TM%20Sissung&author=CE%20Baum&author=CT%20Kirkland&author=R%20Gao&author=ER%20Gardner&volume=44&publication_year=2010&pages=152-67&pmid=19950006&doi=10.1007/s12033-009-9220-6&)

6. Green H, Soderkvist P, Rosenberg P, Horvath G, Peterson C. mdr-1 single nucleotide polymorphisms in ovarian cancer tissue: G2677T/A correlates with response to paclitaxel chemotherapy. Clin Cancer Res. 2006;12:854–9. doi: 10.1158/1078-0432.CCR-05-0950.  [DOI](https://doi.org/10.1158/1078-0432.CCR-05-0950) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16467099/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=mdr-1%20single%20nucleotide%20polymorphisms%20in%20ovarian%20cancer%20tissue:%20G2677T/A%20correlates%20with%20response%20to%20paclitaxel%20chemotherapy&author=H%20Green&author=P%20Soderkvist&author=P%20Rosenberg&author=G%20Horvath&author=C%20Peterson&volume=12&publication_year=2006&pages=854-9&pmid=16467099&doi=10.1158/1078-0432.CCR-05-0950&)

7. Marsh S, Paul J, King CR, Gifford G, McLeod HL, Brown R. Pharmacogenetic assessment of toxicity and outcome after platinum plus taxane chemotherapy in ovarian cancer: the Scottish Randomised Trial in Ovarian Cancer. J Clin Oncol. 2007;25:4528–35. doi: 10.1200/JCO.2006.10.4752.  [DOI](https://doi.org/10.1200/JCO.2006.10.4752) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17925548/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Pharmacogenetic%20assessment%20of%20toxicity%20and%20outcome%20after%20platinum%20plus%20taxane%20chemotherapy%20in%20ovarian%20cancer:%20the%20Scottish%20Randomised%20Trial%20in%20Ovarian%20Cancer&author=S%20Marsh&author=J%20Paul&author=CR%20King&author=G%20Gifford&author=HL%20McLeod&volume=25&publication_year=2007&pages=4528-35&pmid=17925548&doi=10.1200/JCO.2006.10.4752&)

8. Peethambaram P, Fridley BL, Vierkant RA, Larson MC, Kalli KR, Elliott EA, et al. Polymorphisms in ABCB1 and ERCC2 associated with ovarian cancer outcome. Int J Mol Epidemiol Genet. 2011;2:185–95.  [PMC free article](/articles/PMC3110393/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21686133/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Mol%20Epidemiol%20Genet&title=Polymorphisms%20in%20ABCB1%20and%20ERCC2%20associated%20with%20ovarian%20cancer%20outcome&author=P%20Peethambaram&author=BL%20Fridley&author=RA%20Vierkant&author=MC%20Larson&author=KR%20Kalli&volume=2&publication_year=2011&pages=185-95&pmid=21686133&)

9. Tian C, Ambrosone CB, Darcy KM, Krivak TC, Armstrong DK, Bookman MA, et al. Common variants in ABCB1, ABCC2 and ABCG2 genes and clinical outcomes among women with advanced stage ovarian cancer treated with platinum and taxane-based chemotherapy: A Gynecologic Oncology Group study. Gynecol Oncol. 2011 doi: 10.1016/j.ygyno.2011.11.022.  [DOI](https://doi.org/10.1016/j.ygyno.2011.11.022) | [PMC free article](/articles/PMC3278512/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22112610/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gynecol%20Oncol&title=Common%20variants%20in%20ABCB1,%20ABCC2%20and%20ABCG2%20genes%20and%20clinical%20outcomes%20among%20women%20with%20advanced%20stage%20ovarian%20cancer%20treated%20with%20platinum%20and%20taxane-based%20chemotherapy:%20A%20Gynecologic%20Oncology%20Group%20study&author=C%20Tian&author=CB%20Ambrosone&author=KM%20Darcy&author=TC%20Krivak&author=DK%20Armstrong&publication_year=2011&pmid=22112610&doi=10.1016/j.ygyno.2011.11.022&)

10. Johnatty SE, Beesley J, Paul J, Fereday S, Spurdle AB, Webb PM, et al. Correction: ABCB1 (MDR1) Polymorphisms and Progression-Free Survival among Women with Ovarian Cancer following Paclitaxel/Carboplatin Chemotherapy. Clin Cancer Res. 2012;18(1):319–20. doi: 10.1158/1078-0432.CCR-08-0606.  [DOI](https://doi.org/10.1158/1078-0432.CCR-08-0606) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18765553/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=Correction:%20ABCB1%20(MDR1)%20Polymorphisms%20and%20Progression-Free%20Survival%20among%20Women%20with%20Ovarian%20Cancer%20following%20Paclitaxel/Carboplatin%20Chemotherapy&author=SE%20Johnatty&author=J%20Beesley&author=J%20Paul&author=S%20Fereday&author=AB%20Spurdle&volume=18&issue=1&publication_year=2012&pages=319-20&pmid=18765553&doi=10.1158/1078-0432.CCR-08-0606&)

11. Johnatty SE, Beesley J, Paul J, Fereday S, Spurdle AB, Webb PM, et al. ABCB1 (MDR 1) polymorphisms and progression-free survival among women with ovarian cancer following paclitaxel/carboplatin chemotherapy. Clin Cancer Res. 2008;14:5594–601. doi: 10.1158/1078-0432.CCR-08-0606.  [DOI](https://doi.org/10.1158/1078-0432.CCR-08-0606) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18765553/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=ABCB1%20(MDR%201)%20polymorphisms%20and%20progression-free%20survival%20among%20women%20with%20ovarian%20cancer%20following%20paclitaxel/carboplatin%20chemotherapy&author=SE%20Johnatty&author=J%20Beesley&author=J%20Paul&author=S%20Fereday&author=AB%20Spurdle&volume=14&publication_year=2008&pages=5594-601&pmid=18765553&doi=10.1158/1078-0432.CCR-08-0606&)

12. Gayther SA, Song H, Ramus SJ, Kjaer SK, Whittemore AS, Quaye L, et al. Tagging single nucleotide polymorphisms in cell cycle control genes and susceptibility to invasive epithelial ovarian cancer. Cancer Res. 2007;67:3027–35. doi: 10.1158/0008-5472.CAN-06-3261.  [DOI](https://doi.org/10.1158/0008-5472.CAN-06-3261) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17409409/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=Tagging%20single%20nucleotide%20polymorphisms%20in%20cell%20cycle%20control%20genes%20and%20susceptibility%20to%20invasive%20epithelial%20ovarian%20cancer&author=SA%20Gayther&author=H%20Song&author=SJ%20Ramus&author=SK%20Kjaer&author=AS%20Whittemore&volume=67&publication_year=2007&pages=3027-35&pmid=17409409&doi=10.1158/0008-5472.CAN-06-3261&)

13. Sankararaman S, Sridhar S, Kimmel G, Halperin E. Estimating local ancestry in admixed populations. Am J Hum Genet. 2008;82:290–303. doi: 10.1016/j.ajhg.2007.09.022.  [DOI](https://doi.org/10.1016/j.ajhg.2007.09.022) | [PMC free article](/articles/PMC2664993/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18252211/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Estimating%20local%20ancestry%20in%20admixed%20populations&author=S%20Sankararaman&author=S%20Sridhar&author=G%20Kimmel&author=E%20Halperin&volume=82&publication_year=2008&pages=290-303&pmid=18252211&doi=10.1016/j.ajhg.2007.09.022&)

14. Huang RS, Johnatty SE, Gamazon ER, Im HK, Ziliak D, Duan S, et al. Platinum sensitivity-related germline polymorphism discovered via a cell-based approach and analysis of its association with outcome in ovarian cancer patients. Clin Cancer Res. 2011;17:5490–500. doi: 10.1158/1078-0432.CCR-11-0724.  [DOI](https://doi.org/10.1158/1078-0432.CCR-11-0724) | [PMC free article](/articles/PMC3160494/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21705454/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=Platinum%20sensitivity-related%20germline%20polymorphism%20discovered%20via%20a%20cell-based%20approach%20and%20analysis%20of%20its%20association%20with%20outcome%20in%20ovarian%20cancer%20patients&author=RS%20Huang&author=SE%20Johnatty&author=ER%20Gamazon&author=HK%20Im&author=D%20Ziliak&volume=17&publication_year=2011&pages=5490-500&pmid=21705454&doi=10.1158/1078-0432.CCR-11-0724&)

15. Howie B, Fuchsberger C, Stephens M, Marchini J, Abecasis GR. Fast and accurate genotype imputation in genome-wide association studies through pre-phasing. Nat Genet. 2012;44:955–9. doi: 10.1038/ng.2354.  [DOI](https://doi.org/10.1038/ng.2354) | [PMC free article](/articles/PMC3696580/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22820512/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Fast%20and%20accurate%20genotype%20imputation%20in%20genome-wide%20association%20studies%20through%20pre-phasing&author=B%20Howie&author=C%20Fuchsberger&author=M%20Stephens&author=J%20Marchini&author=GR%20Abecasis&volume=44&publication_year=2012&pages=955-9&pmid=22820512&doi=10.1038/ng.2354&)

16. Li Y, Willer C, Ding J, Scheet P, Abecasis G. MaCH: using sequence and genotype data to estimate haplotypes and unobserved genotypes. Genet Epidemiol. 2010;34:816–34. doi: 10.1002/gepi.20533.  [DOI](https://doi.org/10.1002/gepi.20533) | [PMC free article](/articles/PMC3175618/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21058334/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genet%20Epidemiol&title=MaCH:%20using%20sequence%20and%20genotype%20data%20to%20estimate%20haplotypes%20and%20unobserved%20genotypes&author=Y%20Li&author=C%20Willer&author=J%20Ding&author=P%20Scheet&author=G%20Abecasis&volume=34&publication_year=2010&pages=816-34&pmid=21058334&doi=10.1002/gepi.20533&)

17. Pharoah PD, Tsai YY, Ramus SJ, Phelan CM, Goode EL, Lawrenson K, et al. GWAS meta-analysis and replication identifies three new susceptibility loci for ovarian cancer. Nat Genet. 2013;45:362–70. doi: 10.1038/ng.2564.  [DOI](https://doi.org/10.1038/ng.2564) | [PMC free article](/articles/PMC3693183/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23535730/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=GWAS%20meta-analysis%20and%20replication%20identifies%20three%20new%20susceptibility%20loci%20for%20ovarian%20cancer&author=PD%20Pharoah&author=YY%20Tsai&author=SJ%20Ramus&author=CM%20Phelan&author=EL%20Goode&volume=45&publication_year=2013&pages=362-70&pmid=23535730&doi=10.1038/ng.2564&)

18. de Bakker PI, Yelensky R, Pe’er I, Gabriel SB, Daly MJ, Altshuler D. Efficiency and power in genetic association studies. Nat Genet. 2005;37:1217–23. doi: 10.1038/ng1669.  [DOI](https://doi.org/10.1038/ng1669) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16244653/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Efficiency%20and%20power%20in%20genetic%20association%20studies&author=PI%20de%20Bakker&author=R%20Yelensky&author=I%20Pe%E2%80%99er&author=SB%20Gabriel&author=MJ%20Daly&volume=37&publication_year=2005&pages=1217-23&pmid=16244653&doi=10.1038/ng1669&)

19. Cordell HJ, Clayton DG. A unified stepwise regression procedure for evaluating the relative effects of polymorphisms within a gene using case/control or family data: application to HLA in type 1 diabetes. Am J Hum Genet. 2002;70:124–41. doi: 10.1086/338007.  [DOI](https://doi.org/10.1086/338007) | [PMC free article](/articles/PMC384883/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11719900/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=A%20unified%20stepwise%20regression%20procedure%20for%20evaluating%20the%20relative%20effects%20of%20polymorphisms%20within%20a%20gene%20using%20case/control%20or%20family%20data:%20application%20to%20HLA%20in%20type%201%20diabetes&author=HJ%20Cordell&author=DG%20Clayton&volume=70&publication_year=2002&pages=124-41&pmid=11719900&doi=10.1086/338007&)

20. Browning SR, Browning BL. Rapid and accurate haplotype phasing and missing-data inference for whole-genome association studies by use of localized haplotype clustering. Am J Hum Genet. 2007;81:1084–97. doi: 10.1086/521987.  [DOI](https://doi.org/10.1086/521987) | [PMC free article](/articles/PMC2265661/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17924348/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Rapid%20and%20accurate%20haplotype%20phasing%20and%20missing-data%20inference%20for%20whole-genome%20association%20studies%20by%20use%20of%20localized%20haplotype%20clustering&author=SR%20Browning&author=BL%20Browning&volume=81&publication_year=2007&pages=1084-97&pmid=17924348&doi=10.1086/521987&)

21. Haber M, Smith J, Bordow SB, Flemming C, Cohn SL, London WB, et al. Association of high-level MRP1 expression with poor clinical outcome in a large prospective study of primary neuroblastoma. J Clin Oncol. 2006;24:1546–53. doi: 10.1200/JCO.2005.01.6196.  [DOI](https://doi.org/10.1200/JCO.2005.01.6196) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16575006/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Association%20of%20high-level%20MRP1%20expression%20with%20poor%20clinical%20outcome%20in%20a%20large%20prospective%20study%20of%20primary%20neuroblastoma&author=M%20Haber&author=J%20Smith&author=SB%20Bordow&author=C%20Flemming&author=SL%20Cohn&volume=24&publication_year=2006&pages=1546-53&pmid=16575006&doi=10.1200/JCO.2005.01.6196&)

22. Vandesompele J, De Preter K, Pattyn F, Poppe B, Van Roy N, De Paepe A, et al. Accurate normalization of real-time quantitative RT-PCR data by geometric averaging of multiple internal control genes. Genome Biol. 2002;3:RESEARCH0034. doi: 10.1186/gb-2002-3-7-research0034.  [DOI](https://doi.org/10.1186/gb-2002-3-7-research0034) | [PMC free article](/articles/PMC126239/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12184808/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Biol&title=Accurate%20normalization%20of%20real-time%20quantitative%20RT-PCR%20data%20by%20geometric%20averaging%20of%20multiple%20internal%20control%20genes&author=J%20Vandesompele&author=K%20De%20Preter&author=F%20Pattyn&author=B%20Poppe&author=N%20Van%20Roy&volume=3&publication_year=2002&pages=RESEARCH0034&pmid=12184808&doi=10.1186/gb-2002-3-7-research0034&)

23. Integrated genomic analyses of ovarian carcinoma. Nature. 2011;474:609–15. doi: 10.1038/nature10166.  [DOI](https://doi.org/10.1038/nature10166) | [PMC free article](/articles/PMC3163504/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21720365/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Integrated%20genomic%20analyses%20of%20ovarian%20carcinoma&volume=474&publication_year=2011&pages=609-15&pmid=21720365&doi=10.1038/nature10166&)

24. Clark TG, Altman DG, De Stavola BL. Quantification of the completeness of follow-up. Lancet. 2002;359:1309–10. doi: 10.1016/s0140-6736(02)08272-7.  [DOI](https://doi.org/10.1016/s0140-6736(02)08272-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11965278/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Quantification%20of%20the%20completeness%20of%20follow-up&author=TG%20Clark&author=DG%20Altman&author=BL%20De%20Stavola&volume=359&publication_year=2002&pages=1309-10&pmid=11965278&doi=10.1016/s0140-6736(02)08272-7&)

25. Li J, Ji L. Adjusting multiple testing in multilocus analyses using the eigenvalues of a correlation matrix. Heredity (Edinb) 2005;95:221–7. doi: 10.1038/sj.hdy.6800717.  [DOI](https://doi.org/10.1038/sj.hdy.6800717) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16077740/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Heredity%20(Edinb)&title=Adjusting%20multiple%20testing%20in%20multilocus%20analyses%20using%20the%20eigenvalues%20of%20a%20correlation%20matrix&author=J%20Li&author=L%20Ji&volume=95&publication_year=2005&pages=221-7&pmid=16077740&doi=10.1038/sj.hdy.6800717&)

26. Nyholt DR. A simple correction for multiple testing for single-nucleotide polymorphisms in linkage disequilibrium with each other. Am J Hum Genet. 2004;74:765–9. doi: 10.1086/383251.  [DOI](https://doi.org/10.1086/383251) | [PMC free article](/articles/PMC1181954/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14997420/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=A%20simple%20correction%20for%20multiple%20testing%20for%20single-nucleotide%20polymorphisms%20in%20linkage%20disequilibrium%20with%20each%20other&author=DR%20Nyholt&volume=74&publication_year=2004&pages=765-9&pmid=14997420&doi=10.1086/383251&)

27. Bolton KL, Tyrer J, Song H, Ramus SJ, Notaridou M, Jones C, et al. Common variants at 19p13 are associated with susceptibility to ovarian cancer. Nat Genet. 2010;42:880–4. doi: 10.1038/ng.666.  [DOI](https://doi.org/10.1038/ng.666) | [PMC free article](/articles/PMC3125495/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20852633/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Common%20variants%20at%2019p13%20are%20associated%20with%20susceptibility%20to%20ovarian%20cancer&author=KL%20Bolton&author=J%20Tyrer&author=H%20Song&author=SJ%20Ramus&author=M%20Notaridou&volume=42&publication_year=2010&pages=880-4&pmid=20852633&doi=10.1038/ng.666&)

28. Song H, Ramus SJ, Kjaer SK, DiCioccio RA, Chenevix-Trench G, Pearce CL, et al. Association between invasive ovarian cancer susceptibility and 11 best candidate SNPs from breast cancer genome-wide association study. Hum Mol Genet. 2009;18:2297–304. doi: 10.1093/hmg/ddp138.  [DOI](https://doi.org/10.1093/hmg/ddp138) | [PMC free article](/articles/PMC2685754/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19304784/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=Association%20between%20invasive%20ovarian%20cancer%20susceptibility%20and%2011%20best%20candidate%20SNPs%20from%20breast%20cancer%20genome-wide%20association%20study&author=H%20Song&author=SJ%20Ramus&author=SK%20Kjaer&author=RA%20DiCioccio&author=G%20Chenevix-Trench&volume=18&publication_year=2009&pages=2297-304&pmid=19304784&doi=10.1093/hmg/ddp138&)

29. Kimchi-Sarfaty C, Oh JM, Kim IW, Sauna ZE, Calcagno AM, Ambudkar SV, et al. A “silent” polymorphism in the MDR1 gene changes substrate specificity. Science. 2007;315:525–8. doi: 10.1126/science.1135308.  [DOI](https://doi.org/10.1126/science.1135308) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17185560/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=A%20%E2%80%9Csilent%E2%80%9D%20polymorphism%20in%20the%20MDR1%20gene%20changes%20substrate%20specificity&author=C%20Kimchi-Sarfaty&author=JM%20Oh&author=IW%20Kim&author=ZE%20Sauna&author=AM%20Calcagno&volume=315&publication_year=2007&pages=525-8&pmid=17185560&doi=10.1126/science.1135308&)

30. Sauna ZE, Kimchi-Sarfaty C, Ambudkar SV, Gottesman MM. Silent polymorphisms speak: how they affect pharmacogenomics and the treatment of cancer. Cancer Res. 2007;67:9609–12. doi: 10.1158/0008-5472.CAN-07-2377.  [DOI](https://doi.org/10.1158/0008-5472.CAN-07-2377) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17942888/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=Silent%20polymorphisms%20speak:%20how%20they%20affect%20pharmacogenomics%20and%20the%20treatment%20of%20cancer&author=ZE%20Sauna&author=C%20Kimchi-Sarfaty&author=SV%20Ambudkar&author=MM%20Gottesman&volume=67&publication_year=2007&pages=9609-12&pmid=17942888&doi=10.1158/0008-5472.CAN-07-2377&)

31. Diaz-Padilla I, Amir E, Marsh S, Liu G, Mackay H. Genetic polymorphisms as predictive and prognostic biomarkers in gynecological cancers: A systematic review. Gynecol Oncol. 2011 doi: 10.1016/j.ygyno.2011.10.034.  [DOI](https://doi.org/10.1016/j.ygyno.2011.10.034) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22063461/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gynecol%20Oncol&title=Genetic%20polymorphisms%20as%20predictive%20and%20prognostic%20biomarkers%20in%20gynecological%20cancers:%20A%20systematic%20review&author=I%20Diaz-Padilla&author=E%20Amir&author=S%20Marsh&author=G%20Liu&author=H%20Mackay&publication_year=2011&pmid=22063461&doi=10.1016/j.ygyno.2011.10.034&)

32. Burger RA, Brady MF, Bookman MA, Walker JL, Homesley HD, Fowler J, et al. Phase III trial of bevacizumab (BEV) in the primary treatment of advanced epithelial ovarian cancer (EOC), primary peritoneal cancer (PPC), or fallopian tube cancer (FTC): A Gynecologic Oncology Group study. ASCO Meeting Abstracts; 2010. LBA1.  [Google Scholar](https://scholar.google.com/scholar_lookup?Burger%20RA,%20Brady%20MF,%20Bookman%20MA,%20Walker%20JL,%20Homesley%20HD,%20Fowler%20J,%20et%20al.%20Phase%20III%20trial%20of%20bevacizumab%20(BEV)%20in%20the%20primary%20treatment%20of%20advanced%20epithelial%20ovarian%20cancer%20(EOC),%20primary%20peritoneal%20cancer%20(PPC),%20or%20fallopian%20tube%20cancer%20(FTC):%20A%20Gynecologic%20Oncology%20Group%20study.%20ASCO%20Meeting%20Abstracts;%202010.%20LBA1.)
