# A Single Nucleotide Polymorphism in cBIM Is Associated with a Slower Achievement of Major Molecular Response in Chronic Myeloid Leukaemia Treated with Imatinib

## Metadata
**Authors:** Vanessa Augis, Kelly Airiau, Marina Josselin, Béatrice Turcq, François-Xavier Mahon, Francis Belloc
**Journal:** PLoS ONE
**Date:** 2013 Nov 5
**DOI:** [10.1371/journal.pone.0078582](https://doi.org/10.1371/journal.pone.0078582)
**PMID:** 24223824
**PMCID:** PMC3818406
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3818406/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC3818406/pdf/pone.0078582.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC3818406/pdf/pone.0078582.pdf)

## Abstract

**Purpose:** 
BIM is essential for the response to tyrosine-kinase inhibitors (TKI) in chronic myeloid leukaemia (CML) patients. Recently, a deletion polymorphism in intron 2 of the BIM gene was demonstrated to confer an intrinsic TKI resistance in Asian patients. The present study aimed at identifying mutations in the BIM sequence that could lead to imatinib resistance independently of BCR-ABL mutations.

**Experimental Design:** 
BIM coding sequence analysis was performed in 72 imatinib-treated CML patients from a French population of our centre and in 29 healthy controls (reference population) as a case-control study. Real-time quantitative PCR (RT qPCR) was performed to assess Bim expression in our reference population.

**Results:** 
No mutation with amino-acid change was found in the BIM coding sequence. However, we observed a silent single nucleotide polymorphism (SNP) c465C>T (rs724710). A strong statistical link was found between the presence of the T allele and the high Sokal risk group (p = 0.0065). T allele frequency was higher in non responsive patients than in the reference population (p = 0.0049). Similarly, this T allele was associated with the mutation frequency on the tyrosine kinase domain of BCR-ABL (p<0.001) and the presence of the T allele significantly lengthened the time to achieve a major molecular response (MMR). Finally, the presence of the T allele was related to a decreased basal expression of the Bim mRNA in the circulating mononuclear cells of healthy controls.

**Conclusion:** 
These results suggest that the analysis of the c465C>T SNP of BIM could be useful for predicting the outcome of imatinib-treated CML patients.

### Purpose

BIM is essential for the response to tyrosine-kinase inhibitors (TKI) in chronic myeloid leukaemia (CML) patients. Recently, a deletion polymorphism in intron 2 of the BIM gene was demonstrated to confer an intrinsic TKI resistance in Asian patients. The present study aimed at identifying mutations in the BIM sequence that could lead to imatinib resistance independently of BCR-ABL mutations.

### Experimental Design

BIM coding sequence analysis was performed in 72 imatinib-treated CML patients from a French population of our centre and in 29 healthy controls (reference population) as a case-control study. Real-time quantitative PCR (RT qPCR) was performed to assess Bim expression in our reference population.

### Results

No mutation with amino-acid change was found in the BIM coding sequence. However, we observed a silent single nucleotide polymorphism (SNP) c465C>T (rs724710). A strong statistical link was found between the presence of the T allele and the high Sokal risk group (p = 0.0065). T allele frequency was higher in non responsive patients than in the reference population (p = 0.0049). Similarly, this T allele was associated with the mutation frequency on the tyrosine kinase domain of BCR-ABL (p<0.001) and the presence of the T allele significantly lengthened the time to achieve a major molecular response (MMR). Finally, the presence of the T allele was related to a decreased basal expression of the Bim mRNA in the circulating mononuclear cells of healthy controls.

### Conclusion

These results suggest that the analysis of the c465C>T SNP of BIM could be useful for predicting the outcome of imatinib-treated CML patients.

## Introduction

Chronic myeloid leukaemia (CML) is a myeloproliferative disorder characterized by the t(9;22) translocation leading to the fusion transcript BCR-ABL. This transcript encodes a deregulated chimeric tyrosine kinase. Small-molecules inhibiting Bcr-Abl tyrosine-kinase activity (TKI) such as imatinib mesylate have fundamentally improved the treatment of CML ; however, some patients do not undergo an optimal response to imatinib [[1]](#pone.0078582-Baccarani1). In chronic phase, this response failure can be explained in about 25% of cases by the growth of CML cells that exhibit point mutations in the BCR-ABL kinase domain [[2]](#pone.0078582-Lahaye1), [[3]](#pone.0078582-Baran1). The anti-leukemic mechanism of imatinib is to selectively inhibit the growth of CML cells and to induce apoptosis [[4]](#pone.0078582-Deininger1). RNA interference experiments have demonstrated that the pro-apoptotic protein BIM (a protein belonging to the Bcl-2 family proteins) is essential to this apoptotic signal [[5]](#pone.0078582-Shinjyo1)–[[7]](#pone.0078582-Belloc1). Thus, mutations in the BIM sequence could lead to imatinib resistance beside the appearance of the BCR-ABL kinase domain mutation. Recently, a deletion polymorphism in intron 2 of the BIM gene was demonstrated to confer a TKI resistance in Asian patients [[8]](#pone.0078582-Ng1).

In the current work, we performed BIM coding sequence analysis for imatinib responder and non responder CML patients. We did not find any mutation with amino-acid change in the coding sequence in any of the 72 patients analyzed. A single nucleotide polymorphism (SNP) located in exon 5 of the BIM gene was observed in our French population. The presence of the T allele in the c465C>T SNP was significantly associated with a longer delay to achieve a major molecular response (MMR) leading to more frequent mutations in the kinase domain of BCR-ABL and to TKI resistance.

## Materials and Methods

### Ethics Statement

Written informed consent was obtained in accordance with the Declaration of Helsinki from all patients and from parents or guardians on behalf of children who participated in this study, which was performed parallel to the molecular evaluation.

The study was approved by the local Ethics Committee : Comité Consultatif de Protection des Personnes dans la Recherche Biomédicale (CCPRB) de Bordeaux at the University of Bordeaux.

### Patients and healthy controls

All CML samples were supplied by the tumor bank of the Haut Lévêque Hospital (Pessac, Bordeaux, France) and obtained from patients who provided an informed consent. Among them, 56 samples from CML patients were presented to the Bordeaux Hospital between January 2005 and September 2008 for BCR-ABL tyrosine-kinase domain (TKD) sequencing. All patients with excess samples (46) were sequenced for the BIM gene. A randomly chosen group of 26 CML patients with optimal response to imatinib was also sequenced for the BIM gene afterwards. The BCR-ABL TKD could not be sequenced for these patients due to the low level of BCR-ABL mRNA available at the time of the analysis. Median age was 60 years (range 5 to 92), 30 patients were females and 42 males (*sex ratio*: 1.4). The disease progressed at least once for 25 patients in accelerated phase (AP, 21 patients) or in blastic phase (BP, 4 patients). All patients were treated with 400 mg imatinib per day at the time of their inclusion in the study.

Optimal response, suboptimal response, and failure to imatinib first line therapy were defined according to the European Leukaemia net recommendations [[1]](#pone.0078582-Baccarani1). We designated 2 groups of patients: the “non responders” *i.e*. patients with failure or suboptimal response and the “responders” *i.e.* patients with optimal response.

Additional cytogenetic anomalies were found in the progressive phases of the disease since there are criteria for acceleration. From the 46 patients analysed for a *BCR-ABL* mutation, 13 exhibited *BCR-ABL* mutations. They included the E355G (n = 3), T315I (n = 3), F317L (n = 3), M351L (n = 1), V299L (n = 2), H396R and F359V (n = 2) mutations. Two patients harboured a double mutation.

The follow-up of BCR-ABL expression was available for 64 patients and 39 obtained a MMR in less than 18 months and were considered as responders. Patients’ characteristics are summarized in [Table 1](#pone-0078582-t001).

### Table 1. Patients characteristics.

|   | Total (n = 72) | BIM 465 C/C (n = 32) | BIM 465 C/T (n = 34) | BIM 465 T/T (n = 6) |
| --- | --- | --- | --- | --- |
| Age in years |  |  |  |  |
| Median (range) | 60 (5–92) | 60 (14–79) | 60 (5–92) | 58 (35–71) |
| Sex |  |  |  |  |
| Sex ratio | 1.40 | 1.75 | 1.27 | 0.5 |
| Sokal risk scorea |  |  |  |  |
| Median | 0.90 | 0.90 | 0.89 | 0.88 |
| Low (n) | 19 | 10 | 7 | 2 |
| Intermediate (n) | 15 | 8 | 6 | 1 |
| High (n) | 14 | 3 | 11 | 0 |
| MMR b(n/available) |  |  |  |  |
| > 18 months (n) | 25/64 | 7 | 17 | 1 |
| ≤ b 18 months (n) | 39/64 | 22 | 13 | 4 |
| Additional cytogenetic anomaliesc (n) | 17 | 8 | 7 | 2 |
| Progressive phasesd (n) | 25 | 9 | 13 | 3 |
| TKD mutations (n/sequenced) | 13/46 | 2/19 | 9/23 | 2/4 |

Table 1 Caption: aThe Sokal score was known for 48 patients/72. bThe number of patients achieving or not a major molecular response (MMR) is shown. MMR was defined as a BCR-ABL/ABL ratio ≤ 0.1% on the international scale. cThe number of patients for whom one (or several) additional cytogenetic anomaly(ies) was (were) detected in Ph+ mitosis. dThe number of patients who underwent either the acceleration or blastic phase at least once during the study.

A cohort of 30 voluntary healthy controls (considered as a reference group) recruited among the hospital staff was also analysed for BIM sequencing (29 available) and for BIM expression by RT-qPCR.

### Amplification and sequencing

Whole leukocyte fraction was obtained from buffy coat followed by red cell lysis (using ammonium chloride). Total RNA extraction was conducted using the Trizol method (Invitrogen). Complementary DNA was generated using random primers and AMV reverse transcriptase (Roche) according to the manufacturer’s instructions.

- Molecular response to imatinib was assessed by quantifying the BCR-ABL transcript level using reverse-transcriptase quantitative polymerase chain reaction (RT qPCR), according to the recommendations proposed for result harmonization [[9]](#pone.0078582-Hughes1). MMR was defined as a BCR-ABL/ABL ratio ≤ 0.1% on the international scale [[1]](#pone.0078582-Baccarani1).

- BCR-ABL TKD sequencing was performed as previously described [[10]](#pone.0078582-Branford1).

- Complete *BIM* coding sequence was amplified using the Expand long template PCR system kit (Roche) and the following primers: *5′-TTCCCCCAAATGTCTGACTC-3′* (forward) and *5′-GTGCTGGGTCTTGTTGGTTT-3′* (reverse). BIM cDNA sequencing was not possible after this amplification (pink arrows, [Figure S1b](#pone.0078582.s001)), due to the presence of several isoforms that do not share the same sequence (See [Figure S1a](#pone.0078582.s001)). Thus, the sequencing strategy ([Figure S1b](#pone.0078582.s001)) focused on sequencing BIMEL, [[11]](#pone.0078582-OConnor1) the longest isoform generated by alternative splicing that contains coding information of the whole coding sequence. For this, primers were designed (yellow arrows) to anneal to exon 3 that is excised during alternative splicing of the other isoforms. The Big Dye terminator V 1.1 cycle sequencing kit (Applied Biosystems) was used for sequence reaction. Sequence primers were: *5′-CAGAGCCACAAGGTAATCCTG-3′* (forward) and *5′-TACCCACTGGAGGATCGAGA-3′* (reverse).

- Capillary electrophoresis was conducted using the 3130 Genetic Analyser (Applied Biosystems) and the sequences obtained were analysed using the DNA sequencing analysis software (Applied Biosystems). SNPs were detected with the Seqscape software (Applied Biosystems) and visually controlled.

### Analysis of BIM expression by RT-qPCR

cDNA was synthesized from 1 µg of total RNA with random hexamers in a final volume of 20 µL of RT reaction buffer (Roche). Quantitative RT-PCR was carried out in 96-well ABgene plates using the Mx3005P system (Stratagene) with the SYBR Green Master Mix reaction (Stratagene). All reactions were performed in a total volume of 18 µL and contained 2 µL of cDNA diluted 10 folds and 6.25 µM of each primer set. Triplicate samples were analysed for each data point. Negative controls without reverse transcriptase added were performed. The primer sequence information were: BIM forward primer *5′- GAAGGCAATCACGGAGGTGA-3′* and reverse primer *5′- AGGACTTGGGGTTTGTGTTG -3′*, *ß-actin* forward primer *5′-AGCATCGGGTGATGTTCATT-3′* and reverse primer *5′-ATTACAAGCATGCGTCACCA-3′*. Thermal cycling was performed at 95°C for 3 min followed by 40 cycles comprising each a denaturation step at 95°C for 15 s and an annealing/extension step at 60°C for 22 s. Amplification of the appropriate product was verified by continuous monitoring fluorescence through the dissociation temperature of the PCR product at a temperature transition rate of 0.1°C/s to generate a melting curve. PCR products of ß-actin and *BIM* were cloned into the pCR®2.1-Topo vector using the TOPO TA Cloning® kit (Invitrogen) and dilutions of these plasmids were used to generate a calibration curve. Triplicate assays of serial dilutions of these plasmids were included in each experiment to produce the calibration curves. *BIM* levels were expressed as the ratio of the number of transcripts to the number of *ß-actin* control transcripts in each sample.

### Cell lines

The K562 cell line, obtained from European collection of cell cultures (ECACC), comes from a CML patient and expresses the non mutated BCR-ABL fusion gene. Cell lines were maintained in RPMI 1640 medium (Biowest) supplemented with foetal calf serum (10% v/v), 10 mM Hepes, 100 units/ml penicillin, 50 µg/ml streptomycin in a humidified atmosphere containing 5% v/v CO_2_ at 37°C. Exponentially growing cells were used in all experiments.

The K562 imatinib-resistant cell line [[12]](#pone.0078582-Mahon1) was maintained in the presence of 1 µM imatinib in culture medium. It has no BCR-ABL mutation. BIM sequencing was conducted as described for the patients.

### Statistical analysis

To ensure statistical analysis and to avoid false associations, the Hardy-Weinberg exact equilibrium test and the analysis of SNP association with response status were performed with the online SNPstats software [[13]](#pone.0078582-SNPstats1). This web tool analyses associations by logistic regression and interactions by a global test and by a test for the integration in the linear trend of the nested variable [[14]](#pone.0078582-Sole1). Statistical analysis for Fisher’s exact, Chi square, Z, Mann-Whitney and Kaplan-Mayer tests were conducted using the MedCalc statistical analysis software. A p value <0.05 was considered as significant.

## Results

### Evidence for a polymorphism in the coding region of BIM mRNA

After cDNA amplification of BIM, we observed several isoform combinations (See [Figure S1a](#pone.0078582.s001)). In K562 cell lines (Lane 1 and 2), the 3 main isoforms BIM_EL_, BIM_L_ and BIM_S_ were present among others. K562 imatinib-resistant and -sensitive cells presented the same BIM isoforms. PCR products from patients in the different phases of the disease (Lane 3 to 5) also showed the 3 main isoforms but harboured more isoforms than the cell lines.

We did not find any mutation that led to amino-acid change in the coding sequence of BIM from any of the 72 patients sequenced. However, a silent polymorphism located in exon 5 was identified. This synonymous SNP changes the codon from ATC (Ile) to ATT (Ile) (See [figure S1c](#pone.0078582.s001)). The polymorphism c465C>T is located at the nucleotide position 465 (from the AUG of the starting codon in exon 2) and is indexed in the NCBI dbSNP database (rs724710) [[15]](#pone.0078582-International1), [[16]](#pone.0078582-dbSNP1). We observed 3 genotypes (electrophoregrams showing examples of the sequences obtained are presented in [figure S1c](#pone.0078582.s001)): homozygous C/C genotype, base C being the ancestral allele (upper chart), homozygous T/T genotype (middle chart) and heterozygous C/T genotype (lower chart). The overall frequencies for BIM 465 C/C, C/T and T/T genotypes found in CML patients (n = 72) were 44%, 47% and 8%, respectively and were significantly different than those of the local healthy control reference group (OR, 2.78; CI, 1.11–6.93; p = 0.024), for which the respective frequencies were 69, 23 and 8% ([Figure 1a](#pone-0078582-g001)). The genotype frequency distribution was in the Hardy-Weinberg equilibrium (p = 0.59 for the patients and p = 0.24 for the reference series). The minor allele frequency was greater than 1%. Thereafter, we analysed the impact of this SNP on CML patients in relation with the response to the treatment and progression of the disease.

### Figure 1. Relation between the c465C>T genotypes and the Sokal risk score or imatinib response.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e09a/3818406/fc59a9245baf/pone.0078582.g001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3818406_pone.0078582.g001.jpg)

a: BIMEL coding region was sequenced in the cDNA from 72 CML patients and 29 healthy donors. The c465C>T genotype frequency (in %) was plotted for the healthy controls and the whole patient cohort. In this cohort, 32 were responders (optimal response) while 40 “non responders” exhibited sub-optimal or failure to first line imatinib treatment. Each bar represents the frequency percentage of the corresponding genotype in each group. b: cDNAs from 48 patients were sequenced for the BIMEL coding region and the frequency of C/C or C/T+T/T genotypes was plotted for risk subcategories according to the Sokal score: <0.8 characterizes the low risk group, >1.2 the high risk group and 0.8<–<1.2 the intermediate group. The bars represent the frequency (in percentage) of the C/C (light grey) and C/T+T/T (dark grey) genotype. The number of cases was indicated above each bar?

### The c465C>T SNP is linked to the response to imatinib

The patients were classified as responder or non responder on the basis of their ability to achieve a MMR at 18 months of treatment with imatinib ([Table 1](#pone-0078582-t001)). The two groups did not differ in respect of age or sex ratio. However, the non-responder group presented more frequently a high Sokal risk score (p = 0.006) at diagnosis, additional cytogenetic anomalies (p = 0.002) and/or a progression (p = 0.003) during the follow-up than the responder group and included all the mutations in the BCR-ABL TDK.

The presence of the T allele was more frequent in the high Sokal risk group than in the low and intermediate risk groups ([Figure 1b](#pone-0078582-g001), OR = 0.197, 95% CI 0.05 to 0.85, p = 0.03 using the Fisher’s exact test). Moreover, interaction analysis of the BIM 465 SNP with the Sokal risk groups using the SNPstats web tool showed a p value of 0.0065. The test for interaction in the trend for the Sokal group within SNP gave a value p = 0.037 and reciprocally, SNP within the Sokal group gave a value p = 0.0022, confirming the strong interaction between both covariates. Interestingly, the 11 patients with a high risk score associated with a C/T genotype were found in suboptimal or failure response to imatinib.

When the frequencies of the C/C, C/T and T/T genotypes in the responsive and resistant groups were compared to those of the local reference group (healthy controls) ([Figure 1a](#pone-0078582-g001)), no significant difference was evidenced between the responder and reference groups. However, the distribution of the genotypes in the non responder group differed from that in the reference group (p = 0.016 using the chi square test) with an increased frequency of the T allele. This result was confirmed by the analysis of the non responder group compared to the local reference group using the SNPstats web tool showing an association of the genotype with the non responder phenotype in the dominant mode (OR, 4.13; CI, 1.49–11.45; p = 0.0049). We compared the c465C>T SNP genotype frequency in the patients obtaining a MMR in less than 18 months with those necessitating more time. [Figure 2a, left panel](#pone-0078582-g002), shows that the T allele was more frequently found in patients who do not reach a MMR in 18 months, i.e. patients in suboptimal response (chi square 3.88, p = 0.0488). The significance was better for a cut-off of 24 months to achieve a MMR ([figure 2a, right panel](#pone-0078582-g002), chi square 5.101, p = 0.0239). Moreover, a Kaplan-Meier analysis of the MMR frequency as a function of the treatment duration ([figure 2b](#pone-0078582-g002)) confirmed that the presence of the T allele lengthened the time for achieving a MMR (p = 0.0407).

### Figure 2. Relation between the C/C or T/T plus C/T BIM genotypes and the achievement of MMR.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e09a/3818406/de98a989dcdf/pone.0078582.g002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3818406_pone.0078582.g002.jpg)

a: cDNAs from 64 patients followed in the laboratory for their Bcr-Abl expression were sequenced for the BIMEL coding region. The number of patients achieving a MMR before or after 18 months (left panel) or 24 months (right panel) was plotted as a function of the c465C>T genotype: C/C (white bars) or C/T plus T/T (grey bars). p indicates significance with the chi-square test. b: Kaplan-Meier analysis of the achievement of a MMR. The figure represents the probability to achieve a MMR (in %) as a function of the treatment duration (in months) for patients with the C/C genotype (continuous line) or patients with the C/T or T/T genotype (dotted line). c: cDNAs from 37 resistant patients were sequenced for both the BCR-ABL tyrosine kinase domain (TKD) and BIMEL coding region. The frequency of patients with non mutated or mutated TKD was plotted as a function of the c465C>T genotype. The numbers above the bars indicate the sample size. The p value was calculated using the Chi-square test.

Because more than 30% (in our series) of non responder patients presented a mutation in the TKD of *ABL*, it was interesting to check if the c465C>T SNP genotype could be another predictive factor of resistance independently of the mutational status of the *BCR-ABL* gene. When the frequency of the genotype was plotted as a function of the mutational status in non responder patients, we observed ([Figure 2c](#pone-0078582-g002)) that the T allele was more frequent in mutated than in non mutated patients (p = 0.0250) or in reference group (p = 0.0013), while the frequency in non mutated patients did not differ from the local reference group (p = 0.16). Thus, we performed an association analysis of SNP with the mutational status using SNPstats and found a strong association (OR, 12.22; CI, 2.23–66.88; p<0.001) in a dominant mode. Conversely, no association was found between the SNP genotype and the frequency of additional chromosomal alterations.

We then wondered if the c465C>T SNP genotype could influence the expression level of Bim. If this is the case, it is likely independent of the Bcr-Abl expression and thus occurs in non leukemic cells as well. Bim_EL_ is the most regulated isoform during imatinib treatment [[7]](#pone.0078582-Belloc1) and thus, we analysed the expression level of its mRNA as a function of the c465C>T genotypes in normal mononuclear cells isolated from the blood of 29 healthy controls. It was found ([Figure 3](#pone-0078582-g003)) that the T allele was associated with a reduced expression of this mRNA (p = 0.0209). Moreover, only mononuclear cells expressing the C/C genotype were able to produce a Bim/β actin ratio greater than 100 (8/20). This suggests that cells expressing the C/C genotype had a better ability to respond to demand peaks in Bim than the cells expressing the C/T or T/T genotypes.

### Figure 3. Relation between the C/C or C/T plus T/T BIM genotypes and the basal expression of Bim.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e09a/3818406/f44135032c46/pone.0078582.g003.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3818406_pone.0078582.g003.jpg)

The cDNA from mononuclear cells of 29 healthy controls was analysed for BIM mRNA expression by quantitative RT-PCR and sequencing of the BIM coding region. The BIM expression (relative to β actin) was plotted as a function of the c465C>T genotype comparing the C/C to the C/T and T/T genotypes. The p value was calculated using the Mann-Whitney test.

## Discussion

In preliminary experiments, we sequenced the *BCR-ABL* fusion gene in a series of patients treated in the haematology centres of Bordeaux. Amongst them, 29% only of non responder patients in chronic phase exhibited a point mutation in the *BCR-ABL* gene. Thus, mutations in the *BCR-ABL* TKD were not involved in inadequate responses for most cases. The pro-apoptotic protein BIM plays a central role in apoptosis of *BCR-ABL* cells in response to tyrosine kinase inhibitors (TKI) [[5]](#pone.0078582-Shinjyo1)–[[7]](#pone.0078582-Belloc1). It has been proposed as a tumor suppressor in mouse B-cell leukemia and in human B-cell lymphomas [[17]](#pone.0078582-Egle1), [[18]](#pone.0078582-MestreEscorihuela1), and its dysfunction or altered expression could lead to TKI resistance in CML patients. However, we did not detect any point mutation or any other sequence anomaly susceptible to change the BIM amino-acid sequence in our local patient series. This does not exclude the occurrence of some qualitative acquired alterations in the BIM sequence but could only reflect the sensitivity limit of the method. Actually, the median relative expression of *BCR-ABL* was only 0.01% (range 0–65) in the responder group and 7.09% (range 0–100) in the non responder group at the time of *BIM* sequencing. If a *BIM* mutation occurred in *BCR-ABL* expressing cells, its detection would be impeded by the abundance of normal sequences in the non leukemic cells which also expressed *BIM*. In 13 samples, however, the relative *BCR-ABL* expression was higher than 20% and in these conditions, a mutation in *BIM* would have been detected if present but none of these patients had a mutation in the coding sequence. Consequently, on the basis of our results, the hypothesis that a qualitative deficiency of BIM provides an advantage to the cells during TKI treatment was not verified.

Nevertheless, our study showed that an intragenic SNP: c465C>T in the *BIM* gene generates three genotypes. Despite the relatively small sample size, the strength of this case-control study ensues from the fact that both patient and control participation was restricted to individuals residing in the Aquitaine region in France. The impact of this silent polymorphism c465C>T (rs724710) has never been studied before. The genotype distribution in CML patients differed from that found in a random population and this discrepancy was essentially due to the non responder patient group. The T allele was found here significantly associated with a higher risk of disease at diagnosis and, furthermore, with detrimental factors such as the appearance of mutations in the BCR-ABL TDK. The c465C>T SNP is located in the region coding for the structured and active part of the BIM protein (BH3). In the TT genotype, only the ATT (Ile) codon is used. This codon is supposed to be rare as described for the *MDR1* gene [[19]](#pone.0078582-KimchiSarfaty1) (RSCU values change from 20.9 for ATC to 15.8 for ATT) although its impact was evaluated in haplotypes of rare codons. Therefore, c465C>T SNP may have an impact on the speed of the BIM translation and consequently on the BIM folding and activity as previously suggested for the *MDR1* and *DRD2* genes [[19]](#pone.0078582-KimchiSarfaty1)–[[22]](#pone.0078582-Dulucq1). Additionally, although located in the coding sequence, this polymorphism could be in linkage disequilibrium with other point mutations in a non-coding part of the gene. For example, other SNPs linked to the c465C>T SNP could be located in the 3′ untranslated terminal region (UTR) of the *BIM* gene and some of them could interfere with the miRNA binding [[23]](#pone.0078582-Bermano1), [[24]](#pone.0078582-Xiao1). Other SNPs in intronic and UTR regions of the *BIM* gene were previously associated with a risk of developing non-Hodgkin lymphoma and some of them were in linkage disequilibrium with the c465C>T SNP [[25]](#pone.0078582-Kelly1). Similarly, SNPs in the promoter region could quantitatively reduce its expression. Twenty six SNPs located in the *BIM* promoter are currently indexed [[26]](#pone.0078582-SNPpromoter1). Unfortunately, the genomic DNA was not available for any of the CML patients studied here and thus the promoter region of the *BIM* gene could not be investigated in this series. However, this last hypothesis is supported by the finding here that the T allele in 465 was associated with a reduced basal expression of the *BIM* mRNA in normal blood cells. It is known [[7]](#pone.0078582-Belloc1) that leukemic cells deprived in BIM content are poor responder to TKI and it is possible that cells with decreased levels of BIM exhibit a slower response to these TKI. On an other hand, in patients, time response to TKI higher than 12 months are considered as favouring the occurrence of resistance and thus the detection of TKD mutations in BCR-ABL. Thus, it is possible that C/T and T/T genotypes, by lowering the BIM expression induced a lengthening of the response thus facilitating the occurrence and the selection of mutated clones in the leukemic stem cells population. Alternatively, these haplotypes could be related to polymorphisms in spliced elements like introns or in consensus splicing sequences leading to an accumulation of inactive or less active isoforms. Several inactive isoforms have previously been described for BIM [[27]](#pone.0078582-U1), [[28]](#pone.0078582-Miao1) and were found in the patients and cell lines studied (Data not shown and supporting [figure S1a](#pone.0078582.s001)). Thus, the quantitative analysis of mRNA for each of these isoforms could provide information on cellular ratios. Moreover, such a polymorphism leading to an inactive isoform was recently evidenced in Asian patients and was related to a TKI resistance [[8]](#pone.0078582-Ng1). However, this polymorphism common in East Asian individuals (5 to 12%) was not found in the European population. The SNP described here concerns far more people, namely 30% in our French population, and could be of great importance in the response to treatment.

Nevertheless, we report here a relationship between the *BIM* genotype and response of CML during imatinib treatment. SNPs are inherited genomic point mutations. This leads us to consider that some CML patients could have a genomic predisposition to poorly respond to treatment and thus could benefit from a personalized biological monitoring and treatment adaptation particularly associating BH3 mimetics to TKI as previously suggested [[8]](#pone.0078582-Ng1), [[29]](#pone.0078582-Airiau1) for compensating the Bim deficiency.

## Supporting Information

## Funding Statement

La Ligue contre le Cancer Aquitaine. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Baccarani M, Cortes J, Pane F, Niederwieser D, Saglio G, et al. (2009) Chronic myeloid leukemia: an update of concepts and management recommendations of European LeukemiaNet. J Clin Oncol 27: 6041–6051.  [DOI](https://doi.org/10.1200/JCO.2009.25.0779) | [PMC free article](/articles/PMC4979100/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19884523/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Chronic%20myeloid%20leukemia:%20an%20update%20of%20concepts%20and%20management%20recommendations%20of%20European%20LeukemiaNet&author=M%20Baccarani&author=J%20Cortes&author=F%20Pane&author=D%20Niederwieser&author=G%20Saglio&volume=27&publication_year=2009&pages=6041-6051&pmid=19884523&doi=10.1200/JCO.2009.25.0779&)

2. Lahaye T, Riehm B, Berger U, Paschka P, Muller MC, et al. (2005) Response and resistance in 300 patients with BCR-ABL-positive leukemias treated with imatinib in a single center: a 4.5-year follow-up. Cancer 103: 1659–1669.  [DOI](https://doi.org/10.1002/cncr.20922) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15747376/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer&title=Response%20and%20resistance%20in%20300%20patients%20with%20BCR-ABL-positive%20leukemias%20treated%20with%20imatinib%20in%20a%20single%20center:%20a%204.5-year%20follow-up&author=T%20Lahaye&author=B%20Riehm&author=U%20Berger&author=P%20Paschka&author=MC%20Muller&volume=103&publication_year=2005&pages=1659-1669&pmid=15747376&doi=10.1002/cncr.20922&)

3. Baran Y, Ural AU, Gunduz U (2007) Mechanisms of cellular resistance to imatinib in human chronic myeloid leukemia cells. Hematology: 1.  [DOI](https://doi.org/10.1080/10245330701384179) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17852433/)

4. Deininger MW, Goldman JM, Lydon N, Melo JV (1997) The tyrosine kinase inhibitor CGP57148B selectively inhibits the growth of BCR-ABL-positive cells. Blood 90: 3691–3698.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/9345054/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=The%20tyrosine%20kinase%20inhibitor%20CGP57148B%20selectively%20inhibits%20the%20growth%20of%20BCR-ABL-positive%20cells&author=MW%20Deininger&author=JM%20Goldman&author=N%20Lydon&author=JV%20Melo&volume=90&publication_year=1997&pages=3691-3698&pmid=9345054&)

5. Shinjyo T, Kuribara R, Inukai T, Hosoi H, Kinoshita T, et al. (2001) Downregulation of Bim, a proapoptotic relative of Bcl-2, is a pivotal step in cytokine-initiated survival signaling in murine hematopoietic progenitors. Mol Cell Biol 21: 854–864.  [DOI](https://doi.org/10.1128/MCB.21.3.854-864.2001) | [PMC free article](/articles/PMC86676/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11154272/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Cell%20Biol&title=Downregulation%20of%20Bim,%20a%20proapoptotic%20relative%20of%20Bcl-2,%20is%20a%20pivotal%20step%20in%20cytokine-initiated%20survival%20signaling%20in%20murine%20hematopoietic%20progenitors&author=T%20Shinjyo&author=R%20Kuribara&author=T%20Inukai&author=H%20Hosoi&author=T%20Kinoshita&volume=21&publication_year=2001&pages=854-864&pmid=11154272&doi=10.1128/MCB.21.3.854-864.2001&)

6. Kuribara R, Honda H, Matsui H, Shinjyo T, Inukai T, et al. (2004) Roles of Bim in apoptosis of normal and Bcr-Abl-expressing hematopoietic progenitors. Mol Cell Biol 24: 6172–6183.  [DOI](https://doi.org/10.1128/MCB.24.14.6172-6183.2004) | [PMC free article](/articles/PMC434248/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15226421/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Cell%20Biol&title=Roles%20of%20Bim%20in%20apoptosis%20of%20normal%20and%20Bcr-Abl-expressing%20hematopoietic%20progenitors&author=R%20Kuribara&author=H%20Honda&author=H%20Matsui&author=T%20Shinjyo&author=T%20Inukai&volume=24&publication_year=2004&pages=6172-6183&pmid=15226421&doi=10.1128/MCB.24.14.6172-6183.2004&)

7. Belloc F, Moreau-Gaudry F, Uhalde M, Cazalis L, Jeanneteau M, et al. (2007) Imatinib and nilotinib induce apoptosis of chronic myeloid leukemia cells through a Bim-dependant pathway modulated by cytokines. Cancer Biol Ther 6: 912–919.  [DOI](https://doi.org/10.4161/cbt.6.6.4101) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17538248/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Biol%20Ther&title=Imatinib%20and%20nilotinib%20induce%20apoptosis%20of%20chronic%20myeloid%20leukemia%20cells%20through%20a%20Bim-dependant%20pathway%20modulated%20by%20cytokines&author=F%20Belloc&author=F%20Moreau-Gaudry&author=M%20Uhalde&author=L%20Cazalis&author=M%20Jeanneteau&volume=6&publication_year=2007&pages=912-919&pmid=17538248&doi=10.4161/cbt.6.6.4101&)

8. Ng KP, Hillmer AM, Chuah CT, Juan WC, Ko TK, et al. (2012) A common BIM deletion polymorphism mediates intrinsic resistance and inferior responses to tyrosine kinase inhibitors in cancer. Nat Med 18: 521–528.  [DOI](https://doi.org/10.1038/nm.2713) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22426421/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Med&title=A%20common%20BIM%20deletion%20polymorphism%20mediates%20intrinsic%20resistance%20and%20inferior%20responses%20to%20tyrosine%20kinase%20inhibitors%20in%20cancer&author=KP%20Ng&author=AM%20Hillmer&author=CT%20Chuah&author=WC%20Juan&author=TK%20Ko&volume=18&publication_year=2012&pages=521-528&pmid=22426421&doi=10.1038/nm.2713&)

9. Hughes T, Deininger M, Hochhaus A, Branford S, Radich J, et al. (2006) Monitoring CML patients responding to treatment with tyrosine kinase inhibitors: review and recommendations for harmonizing current methodology for detecting BCR-ABL transcripts and kinase domain mutations and for expressing results. Blood 108: 28–37.  [DOI](https://doi.org/10.1182/blood-2006-01-0092) | [PMC free article](/articles/PMC1895821/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16522812/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Monitoring%20CML%20patients%20responding%20to%20treatment%20with%20tyrosine%20kinase%20inhibitors:%20review%20and%20recommendations%20for%20harmonizing%20current%20methodology%20for%20detecting%20BCR-ABL%20transcripts%20and%20kinase%20domain%20mutations%20and%20for%20expressing%20results&author=T%20Hughes&author=M%20Deininger&author=A%20Hochhaus&author=S%20Branford&author=J%20Radich&volume=108&publication_year=2006&pages=28-37&pmid=16522812&doi=10.1182/blood-2006-01-0092&)

10. Branford S, Hughes T (2006) Detection of BCR-ABL mutations and resistance to imatinib mesylate. Methods Mol Med 125: 93–106.  [DOI](https://doi.org/10.1385/1-59745-017-0:93) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16502579/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Methods%20Mol%20Med&title=Detection%20of%20BCR-ABL%20mutations%20and%20resistance%20to%20imatinib%20mesylate&author=S%20Branford&author=T%20Hughes&volume=125&publication_year=2006&pages=93-106&pmid=16502579&doi=10.1385/1-59745-017-0:93&)

11. O'Connor L, Strasser A, O'Reilly LA, Hausmann G, Adams JM, et al. (1998) Bim: a novel member of the Bcl-2 family that promotes apoptosis. EMBO J 17: 384–395.  [DOI](https://doi.org/10.1093/emboj/17.2.384) | [PMC free article](/articles/PMC1170389/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9430630/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=EMBO%20J&title=Bim:%20a%20novel%20member%20of%20the%20Bcl-2%20family%20that%20promotes%20apoptosis&author=L%20O'Connor&author=A%20Strasser&author=LA%20O'Reilly&author=G%20Hausmann&author=JM%20Adams&volume=17&publication_year=1998&pages=384-395&pmid=9430630&doi=10.1093/emboj/17.2.384&)

12. Mahon FX, Deininger MW, Schultheis B, Chabrol J, Reiffers J, et al. (2000) Selection and characterization of BCR-ABL positive cell lines with differential sensitivity to the tyrosine kinase inhibitor STI571: diverse mechanisms of resistance. Blood 96: 1070–1079.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/10910924/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Selection%20and%20characterization%20of%20BCR-ABL%20positive%20cell%20lines%20with%20differential%20sensitivity%20to%20the%20tyrosine%20kinase%20inhibitor%20STI571:%20diverse%20mechanisms%20of%20resistance&author=FX%20Mahon&author=MW%20Deininger&author=B%20Schultheis&author=J%20Chabrol&author=J%20Reiffers&volume=96&publication_year=2000&pages=1070-1079&pmid=10910924&)

13. SNPstats software. Available : http://bioinfo.iconcologia.net/snpstats/start.htm. Accessed 2012.  [http://bioinfo.iconcologia.net/snpstats/start.htm](http://bioinfo.iconcologia.net/snpstats/start.htm)

14. Sole X, Guino E, Valls J, Iniesta R, Moreno V (2006) SNPStats: a web tool for the analysis of association studies. Bioinformatics 22: 1928–1929.  [DOI](https://doi.org/10.1093/bioinformatics/btl268) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16720584/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=SNPStats:%20a%20web%20tool%20for%20the%20analysis%20of%20association%20studies&author=X%20Sole&author=E%20Guino&author=J%20Valls&author=R%20Iniesta&author=V%20Moreno&volume=22&publication_year=2006&pages=1928-1929&pmid=16720584&doi=10.1093/bioinformatics/btl268&)

15. International HapMap project website. Available http://hapmap.ncbi.nlm.nih.gov/ Accessed 2008  [http://hapmap.ncbi.nlm.nih.gov/](http://hapmap.ncbi.nlm.nih.gov/)

16. dbSNP website. Available http://www.ncbi.nlm.nih.gov/SNP/. Accessed 2008  [http://www.ncbi.nlm.nih.gov/SNP/](http://www.ncbi.nlm.nih.gov/SNP/)

17. Egle A, Harris AW, Bouillet P, Cory S (2004) Bim is a suppressor of Myc-induced mouse B cell leukemia. Proc Natl Acad Sci U S A 101: 6164–6169.  [DOI](https://doi.org/10.1073/pnas.0401471101) | [PMC free article](/articles/PMC395940/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15079075/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&title=Bim%20is%20a%20suppressor%20of%20Myc-induced%20mouse%20B%20cell%20leukemia&author=A%20Egle&author=AW%20Harris&author=P%20Bouillet&author=S%20Cory&volume=101&publication_year=2004&pages=6164-6169&pmid=15079075&doi=10.1073/pnas.0401471101&)

18. Mestre-Escorihuela C, Rubio-Moscardo F, Richter JA, Siebert R, Climent J, et al. (2007) Homozygous deletions localize novel tumor suppressor genes in B-cell lymphomas. Blood 109: 271–280.  [DOI](https://doi.org/10.1182/blood-2006-06-026500) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16960149/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Homozygous%20deletions%20localize%20novel%20tumor%20suppressor%20genes%20in%20B-cell%20lymphomas&author=C%20Mestre-Escorihuela&author=F%20Rubio-Moscardo&author=JA%20Richter&author=R%20Siebert&author=J%20Climent&volume=109&publication_year=2007&pages=271-280&pmid=16960149&doi=10.1182/blood-2006-06-026500&)

19. Kimchi-Sarfaty C, Oh JM, Kim IW, Sauna ZE, Calcagno AM, et al. (2007) A "silent" polymorphism in the MDR1 gene changes substrate specificity. Science 315: 525–528.  [DOI](https://doi.org/10.1126/science.1135308) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17185560/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=A%20%22silent%22%20polymorphism%20in%20the%20MDR1%20gene%20changes%20substrate%20specificity&author=C%20Kimchi-Sarfaty&author=JM%20Oh&author=IW%20Kim&author=ZE%20Sauna&author=AM%20Calcagno&volume=315&publication_year=2007&pages=525-528&pmid=17185560&doi=10.1126/science.1135308&)

20. Duan J, Wainwright MS, Comeron JM, Saitou N, Sanders AR, et al. (2003) Synonymous mutations in the human dopamine receptor D2 (DRD2) affect mRNA stability and synthesis of the receptor. Hum Mol Genet 12: 205–216.  [DOI](https://doi.org/10.1093/hmg/ddg055) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12554675/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=Synonymous%20mutations%20in%20the%20human%20dopamine%20receptor%20D2%20(DRD2)%20affect%20mRNA%20stability%20and%20synthesis%20of%20the%20receptor&author=J%20Duan&author=MS%20Wainwright&author=JM%20Comeron&author=N%20Saitou&author=AR%20Sanders&volume=12&publication_year=2003&pages=205-216&pmid=12554675&doi=10.1093/hmg/ddg055&)

21. Sauna ZE, Kimchi-Sarfaty C, Ambudkar SV, Gottesman MM (2007) Silent polymorphisms speak: how they affect pharmacogenomics and the treatment of cancer. Cancer Res 67: 9609–9612.  [DOI](https://doi.org/10.1158/0008-5472.CAN-07-2377) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17942888/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=Silent%20polymorphisms%20speak:%20how%20they%20affect%20pharmacogenomics%20and%20the%20treatment%20of%20cancer&author=ZE%20Sauna&author=C%20Kimchi-Sarfaty&author=SV%20Ambudkar&author=MM%20Gottesman&volume=67&publication_year=2007&pages=9609-9612&pmid=17942888&doi=10.1158/0008-5472.CAN-07-2377&)

22. Dulucq S, Bouchet S, Turcq B, Lippert E, Etienne G, et al. (2008) Multidrug resistance gene (MDR1) polymorphisms are associated with major molecular responses to standard-dose imatinib in chronic myeloid leukemia. Blood 112: 2024–2027.  [DOI](https://doi.org/10.1182/blood-2008-03-147744) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18524988/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Multidrug%20resistance%20gene%20(MDR1)%20polymorphisms%20are%20associated%20with%20major%20molecular%20responses%20to%20standard-dose%20imatinib%20in%20chronic%20myeloid%20leukemia&author=S%20Dulucq&author=S%20Bouchet&author=B%20Turcq&author=E%20Lippert&author=G%20Etienne&volume=112&publication_year=2008&pages=2024-2027&pmid=18524988&doi=10.1182/blood-2008-03-147744&)

23. Bermano G, Pagmantidis V, Holloway N, Kadri S, Mowat NA, et al. (2007) Evidence that a polymorphism within the 3'UTR of glutathione peroxidase 4 is functional and is associated with susceptibility to colorectal cancer. Genes Nutr 2: 225–232.  [DOI](https://doi.org/10.1007/s12263-007-0052-3) | [PMC free article](/articles/PMC2474949/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18850177/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genes%20Nutr&title=Evidence%20that%20a%20polymorphism%20within%20the%203'UTR%20of%20glutathione%20peroxidase%204%20is%20functional%20and%20is%20associated%20with%20susceptibility%20to%20colorectal%20cancer&author=G%20Bermano&author=V%20Pagmantidis&author=N%20Holloway&author=S%20Kadri&author=NA%20Mowat&volume=2&publication_year=2007&pages=225-232&pmid=18850177&doi=10.1007/s12263-007-0052-3&)

24. Xiao C, Srinivasan L, Calado DP, Patterson HC, Zhang B, et al. (2008) Lymphoproliferative disease and autoimmunity in mice with increased miR-17-92 expression in lymphocytes. Nat Immunol 9: 405–414.  [DOI](https://doi.org/10.1038/ni1575) | [PMC free article](/articles/PMC2533767/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18327259/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Immunol&title=Lymphoproliferative%20disease%20and%20autoimmunity%20in%20mice%20with%20increased%20miR-17-92%20expression%20in%20lymphocytes&author=C%20Xiao&author=L%20Srinivasan&author=DP%20Calado&author=HC%20Patterson&author=B%20Zhang&volume=9&publication_year=2008&pages=405-414&pmid=18327259&doi=10.1038/ni1575&)

25. Kelly JL, Novak AJ, Fredericksen ZS, Liebow M, Ansell SM, et al. (2010) Germline variation in apoptosis pathway genes and risk of non-Hodgkin's lymphoma. Cancer Epidemiol Biomarkers Prev 19: 2847–2858.  [DOI](https://doi.org/10.1158/1055-9965.EPI-10-0581) | [PMC free article](/articles/PMC2976783/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20855536/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Epidemiol%20Biomarkers%20Prev&title=Germline%20variation%20in%20apoptosis%20pathway%20genes%20and%20risk%20of%20non-Hodgkin's%20lymphoma&author=JL%20Kelly&author=AJ%20Novak&author=ZS%20Fredericksen&author=M%20Liebow&author=SM%20Ansell&volume=19&publication_year=2010&pages=2847-2858&pmid=20855536&doi=10.1158/1055-9965.EPI-10-0581&)

26. SNP@promoter website. Available : http://variome.kobic.re.kr/SNPatPromoter. Accessed 2008.  [http://variome.kobic.re.kr/SNPatPromoter](http://variome.kobic.re.kr/SNPatPromoter)

27. U M, Miyashita T, Shikama Y, Tadokoro K, Yamada M (2001) Molecular cloning and characterization of six novel isoforms of human Bim, a member of the proapoptotic Bcl-2 family. FEBS Lett 509: 135–141.  [DOI](https://doi.org/10.1016/s0014-5793(01)03145-3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11734221/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=FEBS%20Lett&title=Molecular%20cloning%20and%20characterization%20of%20six%20novel%20isoforms%20of%20human%20Bim,%20a%20member%20of%20the%20proapoptotic%20Bcl-2%20family&author=M%20U&author=T%20Miyashita&author=Y%20Shikama&author=K%20Tadokoro&author=M%20Yamada&volume=509&publication_year=2001&pages=135-141&pmid=11734221&doi=10.1016/s0014-5793(01)03145-3&)

28. Miao J, Chen GG, Yun JP, Chun SY, Zheng ZZ, et al. (2007) Identification and characterization of BH3 domain protein Bim and its isoforms in human hepatocellular carcinomas. Apoptosis 12: 1691–1701.  [DOI](https://doi.org/10.1007/s10495-007-0093-5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17503221/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Apoptosis&title=Identification%20and%20characterization%20of%20BH3%20domain%20protein%20Bim%20and%20its%20isoforms%20in%20human%20hepatocellular%20carcinomas&author=J%20Miao&author=GG%20Chen&author=JP%20Yun&author=SY%20Chun&author=ZZ%20Zheng&volume=12&publication_year=2007&pages=1691-1701&pmid=17503221&doi=10.1007/s10495-007-0093-5&)

29. Airiau K, Mahon FX, Josselin M, Jeanneteau M, Turcq B, et al.. (2012) ABT-737 increases tyrosine kinase inhibitor-induced apoptosis in chronic myeloid leukemia cells through XIAP downregulation and sensitizes CD34(+) CD38(-) population to imatinib. Exp Hematol 40: 367–378 e362.  [DOI](https://doi.org/10.1016/j.exphem.2012.01.004) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22240609/)
