# Response Prediction in Chronic Hepatitis C by Assessment of IP-10 and IL28B-Related Single Nucleotide Polymorphisms

## Metadata
**Authors:** Martin Lagging, Galia Askarieh, Francesco Negro, Stephanie Bibert, Jonas Söderholm, Johan Westin, Magnus Lindh, Ana Romero, Gabriele Missale, Carlo Ferrari, Avidan U Neumann, Jean-Michel Pawlotsky, Bart L Haagmans, Stefan Zeuzem, Pierre-Yves Bochud, Kristoffer Hellstrand, for the DITTO-HCV Study Group
**Journal:** PLoS ONE
**Date:** 2011 Feb 24
**DOI:** [10.1371/journal.pone.0017232](https://doi.org/10.1371/journal.pone.0017232)
**PMID:** 21390311
**PMCID:** PMC3044738
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3044738/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC3044738/pdf/pone.0017232.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC3044738/pdf/pone.0017232.pdf)

## Abstract

**Background:** 
High baseline levels of IP-10 predict a slower first phase decline in HCV RNA and a poor outcome following interferon/ribavirin therapy in patients with chronic hepatitis C. Several recent studies report that single nucleotide polymorphisms (SNPs) adjacent to IL28B predict spontaneous resolution of HCV infection and outcome of treatment among HCV genotype 1 infected patients.

**Methods and Findings:** 
In the present study, we correlated the occurrence of variants at three such SNPs (rs12979860, rs12980275, and rs8099917) with pretreatment plasma IP-10 and HCV RNA throughout therapy within a phase III treatment trial (HCV-DITTO) involving 253 Caucasian patients. The favorable SNP variants (CC, AA, and TT, respectively) were associated with lower baseline IP-10 (P = 0.02, P = 0.01, P = 0.04) and were less common among HCV genotype 1 infected patients than genotype 2/3 (P<0.0001, P<0.0001, and P = 0.01). Patients carrying favorable SNP genotypes had higher baseline viral load than those carrying unfavorable variants (P = 0.0013, P = 0.029, P = 0.0004 respectively). Among HCV genotype 1 infected carriers of the favorable C, A, or T alleles, IP-10 below 150 pg/mL significantly predicted a more pronounced reduction of HCV RNA from day 0 to 4 (first phase decline), which translated into increased rates of RVR (62%, 53%, and 39%) and SVR (85%, 76%, and 75% respectively) among homozygous carriers with baseline IP-10 below 150 pg/mL. In multivariate analyses of genotype 1-infected patients, baseline IP-10 and C genotype at rs12979860 independently predicted the first phase viral decline and RVR, which in turn independently predicted SVR.

**Conclusions:** 
Concomitant assessment of pretreatment IP-10 and IL28B-related SNPs augments the prediction of the first phase decline in HCV RNA, RVR, and final therapeutic outcome.

### Background

High baseline levels of IP-10 predict a slower first phase decline in HCV RNA and a poor outcome following interferon/ribavirin therapy in patients with chronic hepatitis C. Several recent studies report that single nucleotide polymorphisms (SNPs) adjacent to *IL28B* predict spontaneous resolution of HCV infection and outcome of treatment among HCV genotype 1 infected patients.

### Methods and Findings

In the present study, we correlated the occurrence of variants at three such SNPs (*rs12979860*, *rs12980275*, and *rs8099917*) with pretreatment plasma IP-10 and HCV RNA throughout therapy within a phase III treatment trial (HCV-DITTO) involving 253 Caucasian patients. The favorable SNP variants (CC, AA, and TT, respectively) were associated with lower baseline IP-10 (P = 0.02, P = 0.01, P = 0.04) and were less common among HCV genotype 1 infected patients than genotype 2/3 (P<0.0001, P<0.0001, and P = 0.01). Patients carrying favorable SNP genotypes had higher baseline viral load than those carrying unfavorable variants (P = 0.0013, P = 0.029, P = 0.0004 respectively). Among HCV genotype 1 infected carriers of the favorable C, A, or T alleles, IP-10 below 150 pg/mL significantly predicted a more pronounced reduction of HCV RNA from day 0 to 4 (first phase decline), which translated into increased rates of RVR (62%, 53%, and 39%) and SVR (85%, 76%, and 75% respectively) among homozygous carriers with baseline IP-10 below 150 pg/mL. In multivariate analyses of genotype 1-infected patients, baseline IP-10 and C genotype at *rs12979860* independently predicted the first phase viral decline and RVR, which in turn independently predicted SVR.

### Conclusions

Concomitant assessment of pretreatment IP-10 and *IL28B*-related SNPs augments the prediction of the first phase decline in HCV RNA, RVR, and final therapeutic outcome.

## Introduction

Hepatitis C virus (HCV) infects 170 million people worldwide [[1]](#pone.0017232-WHO1) and is a leading cause of chronic hepatitis, cirrhosis, and hepatocellular carcinoma [[2]](#pone.0017232-Saito1). Treatment with pegylated interferon-α (peg-IFN) and ribavirin results in a sustained viral response (SVR) in approximately 50% of patients infected with HCV of genotype 1 and 80% of those with HCV genotypes 2 or 3 [[3]](#pone.0017232-Fried1), [[4]](#pone.0017232-Hadziyannis1), [[5]](#pone.0017232-Manns1). Recently, several genome-wide association studies have revealed that single nucleotide polymorphisms (SNPs) in the *19q13* region, in close proximity to three genes (*IL28A*, *IL28B*, and *IL29*) encoding cytokines of the IFN-λ (i.e. type III IFN) family, predict spontaneous clearance of HCV infection [[6]](#pone.0017232-Rauch1), [[7]](#pone.0017232-Thomas1) as well as SVR following peg-IFN/ribavirin therapy among patients infected with HCV genotype 1 [[6]](#pone.0017232-Rauch1), [[8]](#pone.0017232-Ge1), [[9]](#pone.0017232-Suppiah1), [[10]](#pone.0017232-Tanaka1). Three of these SNPs are reportedly highly predictive of a favorable treatment response among HCV genotype 1 infected patients: *rs12979860* [[7]](#pone.0017232-Thomas1), [[8]](#pone.0017232-Ge1), *rs12980275* [[10]](#pone.0017232-Tanaka1), and *rs8099917* [[6]](#pone.0017232-Rauch1), [[9]](#pone.0017232-Suppiah1), [[10]](#pone.0017232-Tanaka1), with a strong linkage disequilibrium noted between *rs12979860* and *rs8099917* [[6]](#pone.0017232-Rauch1). Aside from differences in the assays utilized, SVR was the defined endpoint in the genome-wide association study (GWAS) identifying *rs12979860* [[8]](#pone.0017232-Ge1), whereas null responsiveness to interferon was used in the analysis that endorsed *rs12980275* and *rs8099917* [[10]](#pone.0017232-Tanaka1).

The C allele at *rs12979860* also is associated with higher baseline viral load [[8]](#pone.0017232-Ge1), [[11]](#pone.0017232-McCarthy1), which otherwise is an established negative predictor of response to peg-IFN/ribavirin therapy [[3]](#pone.0017232-Fried1), [[4]](#pone.0017232-Hadziyannis1), [[5]](#pone.0017232-Manns1). Similarly counterintuitive is the report that a C allele at *rs12979860* is more common in Caucasians with HCV genotype 2 and 3 infection than in genotype 1 infected or in HCV uninfected individuals [[11]](#pone.0017232-McCarthy1), [[12]](#pone.0017232-MontesCano1). In addition, two studies reported that homozygous carriage of GG at *rs8099917* was associated with slightly lower PBMC mRNA expression of *IL28* in 49 and 20 individuals, respectively [[9]](#pone.0017232-Suppiah1), [[10]](#pone.0017232-Tanaka1), although another study reported no difference in *IL28B* mRNA expression when stratified regarding *rs12979860* genotype in PBMC from 80 individuals [[8]](#pone.0017232-Ge1). In line with this latter finding, G allele carriage at *rs8099917* has been reported to be associated with elevated intrahepatic mRNA expression of a panel of 37 interferon-stimulated genes (ISGs) but not *IL28B* in 91 HCV infected patients [[13]](#pone.0017232-Honda1).

Interferon-gamma inducible protein 10 kDa (IP-10 or CXCL10) is a chemotactic CXC chemokine of 77 amino acids in its mature form [[14]](#pone.0017232-Luster1), [[15]](#pone.0017232-Neville1). IP-10 targets the CXCR3 receptor but, unlike other CXC chemokines, lacks chemotactic activity for neutrophils and instead attracts T lymphocytes, NK cells, and monocytes to sites of infection [[15]](#pone.0017232-Neville1), [[16]](#pone.0017232-Taub1), [[17]](#pone.0017232-Taub2). IP-10 is produced by a variety of cells, including hepatocytes, and levels of IP-10 at onset of therapy are reportedly elevated in patients infected with HCV of genotypes 1 or 4 who do not achieve SVR [[18]](#pone.0017232-Romero1). In difficult-to-treat genotype 1 infected HCV patients, cut-off levels of IP-10 in plasma of 150 pg/mL (approximately equal to 2 standard deviations above the mean IP-10 level of HCV seronegative blood donors) and 600 pg/mL have yielded positive and negative predictive values for SVR of 71% and 100%, respectively [[19]](#pone.0017232-Lagging1). IP-10 in plasma is mirrored by intrahepatic IP-10 mRNA, and strongly predicts the HCV RNA decline during the first days (“first phase decline”) during interferon/ribavirin therapy for all HCV genotypes [[20]](#pone.0017232-Askarieh1).

The impact of IP-10 on the elimination of HCV RNA during therapy in the setting of *IL28B* genetic variants is not known. We therefore assessed plasma IP-10 in relation to genetic variants at three major *IL28B* SNPs (*rs12979860*, *rs12980275*, and *rs8099917*) in patients chronically infected with HCV genotypes 1-4. Our results demonstrate a significant association between *IL28B* genetic variants and IP-10 in plasma and imply that combined assessment of these predictive factors may improve prognostication of the rate of first phase elimination of HCV RNA, as well as achieving a rapid virological response (RVR) and SVR.

## Methods

### Ethical Aspects

The treatment study conformed to the guidelines of the 1975 Declaration of Helsinki and was approved by ethical committees at each center (Medicinska fakultetens forskningsetikkommitté, Göteborgs Universitet, Gothenburg, Sweden, CPP-Ile-de-France IX, CHU Henri Mondor, Creteil, France, Comitato Etico Indipendente (IRB/IEC) dell'Azienda Ospedaliera di Parma, Parma, Italy, Comite d'Ethique du department de Medicine, Hopitaux Universitaires de Genève, Geneva, Switzerland, the Helsinki committee of the Kaplan Medical Center, Rehovot, Israel, the Ethics Committee of Hospital General Vall d'Hebron, Barcelona, Spain, the Ethics Committee of Aristotle University of Thessaloniki, Thessaloniki, Greece, the Ethics Committee of Klinikum der Johann Wolfgang Goethe-Universitat, Frankfurt, Germany, the Ethics Committee of University Hospital Rotterdam Dijkzigt, Rotterdam, Netherlands). Written informed consent was obtained from each patient included in this study.

### Patients

Between February 2001 and November 2003, 270 patients (180 men and 90 women) were recruited in a phase III, open-label, randomized, multicenter trial conducted by the DITTO-HCV study group at 9 centers in France, Germany, Greece, Israel, Italy, Netherlands, Spain, Sweden, and Switzerland, as previously reported [[21]](#pone.0017232-Zeuzem1). All patients were adults, had compensated liver disease, were treatment naïve for hepatitis C, and fulfilled the following inclusion criteria: a positive test for anti-HCV antibody, an HCV RNA level greater than 1000 IU/mL, and two serum alanine aminotransferase values above the upper limit of normal within 6 months of treatment initiation. Two hundred and fifty three Caucasian patients had samples available for *IL28B* analysis (baseline characteristics shown in [Table 1](#pone-0017232-t001)), and 252 of these patients had pretreatment plasma available for evaluation of IP-10.

### Table 1. Baseline Characteristics with Patients Grouped According to IL28B Genetic Variations.

| NCBI dbSNP ID | rs12979860 |  |  |  | rs12980275 |  |  |  | rs8099917 |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
|   | CC | CT | TT | P | AA | AG | GG | P | TT | TG | GG | P |
|   | (n = 93) | (n = 123) | (n = 37) |   | (n = 101) | (n = 115) | (n = 37) |   | (n = 153) | (n = 90) | (n = 10) |   |
| Age (years)a | 41.6 (10.1) | 41.9 (9.5) | 41.9 (11.4) | 0.85c | 41.7 (10.1) | 41.8 (10.3) | 41.0 (11.5) | 0.93c | 42.1 (9.2) | 40.5 (11.1) | 45.0 (12.5) | 0.23c |
| Gender (Male/Female)b | 64/29 | 77/46 | 28/8 | 0.18d | 65/36 | 78/37 | 26/10 | 0.62d | 102/51 | 60/29 | 7/3 | 0.97d |
| BMI (kg/m2)a | 25.1 (3.6) | 25.0 (3.5) | 25.0 (3.5) | 0.91c | 24.9 (3.7) | 25.3 (3.7) | 25.1 (3.3) | 0.69c | 25.1 (3.6) | 25.1 (3.5) | 26.3 (4.1) | 0.55c |
| HCV-RNA level (log10 IU/mL)a | 6.3 (0.8) | 6.1 (0.7) | 5.9 (0.8) | 0.0013c | 6.3 (0.8) | 6.1 (0.8) | 6.0 (0.6) | 0.029c | 6.3 (0.8) | 5.9 (0.7) | 5.8 (0.8) | 0.0004c |
| HCV Genotype (1/2/3/4)b | 44/13/33/3 | 96/7/15/5 | 30/3/1/3 | <0.0001d | 52/13/33/3 | 88/7/15/5 | 30/3/1/3 | <0.0001d | 97/15/38/3 | 66/7/11/6 | 7/1/0/2 | 0.01d |
| Fibrosis Stage (Ishak 0/1/2/3/4/5/6)b | 3/18/27/11/5/12/5 | 7/35/27/17/7/6/6 | 1/8/11/2/3/2/3 | 0.5d | 4/22/28/12/6/12/4 | 6/31/25/16/6/6/7 | 1/8/12/2/3/2/3 | 0.71d | 6/36/46/22/7/14/5 | 5/22/17/8/8/5/7 | 0/3/2/0/0/1/2 | 0.18d |
| Steatosis Grade (0/1/2/3)b | 31/33/12/5 | 54/35/14/1 | 13/13/3/1 | 0.35d | 35/35/13/5 | 48/33/13/2 | 15/13/3/0 | 0.54d | 60/53/16/7 | 35/24/12/0 | 3/4/1/0 | 0.43d |

Table 1 Caption: aData presented as mean (SD) or bn. cKruskal-Wallis Test. dChi Squared test.

### Treatment

All patients in the DITTO-HCV trial were treated for 6 weeks with 180 µg pegylated interferon-α2a sc once weekly (Pegasys, F. Hoffmann-LaRoche, Basel, Switzerland) and ribavirin orally twice daily (Copegus, F. Hoffmann-LaRoche) at a total daily dose of 1,000 mg for patients weighing less than 75 kg and 1,200 mg daily for above 75 kg. Thereafter, patients were randomized 1∶1 based on their viral kinetic classification to receive individualized therapy or to continue on standard combination therapy for a total of 48 weeks. There were no major differences in treatment outcome for patients receiving individualized or standard therapy [[21]](#pone.0017232-Zeuzem1).

### HCV Genotyping

Genotyping of HCV was performed using INNO-LiPA HCV II (Innogenetics N.V., Ghent, Belgium).

### HCV RNA Quantification

HCV RNA was determined by RT-PCR using Cobas Amplicor HCV Monitor version 2.0 (Roche Diagnostics, Branchburg, NJ), and quantified on days 0, 1, 4, 7, 8, 15, 22, 29, at end of treatment, and 24 months after the completion of treatment.

### Classification of Treatment Outcome

Patients were classified as having achieved RVR if HCV RNA was undetectable (<50 IU/mL) in plasma on treatment day 29, and were classified as having SVR if HCV RNA was undetectable in plasma 24 weeks after the completion of therapy.

### Liver Biopsies

Liver biopsies were obtained from patients in the DITTO-HCV trial within 12 months prior to inclusion in the study, and liver biopsy samples were processed for both histological evaluation (≥1.5 cm) and for RNA analysis (≥1 cm). Only biopsies with a length exceeding 1.5 cm and containing more than 6 portal tracts were evaluated. In total, liver biopsies from 228 infected patients in the DITTO-HCV trial were retrieved and evaluated. For each biopsy, a hematoxylin-eosin stain and a Sirius Red stain were centrally staged and graded by two independent observers experienced in pseudo-numerical scoring of liver biopsies in a blinded fashion according to the Ishak protocol [[22]](#pone.0017232-Ishak1). Equivocal issues were debated after the independent scores were noted, and a consensus score was obtained. In addition, steatosis was graded as follows: absent = 0, less than 30% of hepatocytes involved = 1, 30–70% of hepatocytes involved = 2, and more than 70% of hepatocytes involved = 3 [[23]](#pone.0017232-Westin1).

### IP-10 Quantification in Plasma

Quantification of IP-10 was performed using Quantikine (R&D SYSTEMS Minneapolis, MN), a solid-phase ELISA, on plasma samples obtained during the week prior to the start of therapy. All samples were stored at –70°C until assayed.

### DNA extraction

DNA from peripheral blood mononuclear cells was isolated using the QIAamp DNA mini kit (Qiagen) and quantitated on the NanoDrop 1000 spectrophotometer (Thermo Fischer Scientific).

### IL28-B genotyping

DNA samples from patients and controls were genotyped for the *IL28B rs8099917*, *rs12979860* and *rs12980275* polymorphism with TaqMan SNP genotyping assays (Applied Biosystems Inc, Foster City, CA), using the ABI 7500 Fast real time thermocycler, according to manufacturers recommended protocols. TaqMan probes and primers were designed and synthesized by Applied Biosystems Inc. Automated allele calling was performed using SDS software from Applied Biosystems Inc. Positive and negative controls were used in each genotyping assay. The primers and probes utilized were:

### Statistical Methods

Individual characteristics between groups were evaluated using the Wilcoxon-Mann-Whitney U-test, Kruskal-Wallis test and Chi squared (χ^2^) test, and Spearman's rank correlation coefficient *rs* test was utilized to evaluate relationships between variables. All abovementioned statistical analyses were performed using StatView for Macintosh (Version 5.0, SAS Institute Inc., Cary, NC, USA). After univariate analyses, multivariate analyses were performed among the HCV genotype 1 infected patients using Stata (version 9.1, StataCorp, College Station, TX, USA). For multivariate models, all variables associated with the endpoint (P<0.2) were entered, with age and gender forced into the model. Because of strong linkage disequilibrium noted between *rs12979860* and *rs12980275* (*R2* = 0.86 assuming Hardy-Weinberg equilibrium, as compared to *R2* = 0.42 for *rs12979860* and *rs8099917* as well as *R2* = 0.41 for *rs12980275* and *rs8099917*) and because *rs8099917* was not significantly associated with SVR, only SNP variants for *rs12979860* were included in the multivariate analyses. SNPs were tested using three models assuming one of the following modes of inheritance: dominant (comparing presence of one or two copies of the minor allele versus none), recessive (comparing presence of two copies of the minor allele versus none or one copy), and additive (none, one or two copies of the minor allele were coded 0, 1 and 2, respectively, assuming greater effect with increased copy number of the minor allele). Linkage disequilibrium was calculated using the pwld software implemented in Stata. All reported p-values are two-sided, and p-values <0.05 were considered significant.

## Results

A strong association was noted between the distribution of HCV genotypes and *IL28B* SNP variants (P<0.0001 for *rs12979860* and *rs12980275*, and P = 0.01 for *rs8099917*, Chi squared test; [Table 1](#pone-0017232-t001)) with the favorable CC at *rs12979860*, AA at *rs12980275*, and TT at *rs8099917* being significantly more common in patients with HCV genotype 2 or 3 infection than genotype 1 ([Figure 1](#pone-0017232-g001)). The 11 patients infected with HCV genotype 4 had a similar distribution of *IL28B* variants as patients infected with genotype 1. Patients who were homozygous for the favorable SNP genotypes had higher baseline viral load (mean 6.3 log_10_ IU/mL; [Table 1](#pone-0017232-t001)); heterozygous patients had intermediate (mean 6.1 log_10_ IU/mL for *rs12979860* and *rs12980275*, and 5.9 log_10_ IU/mL for TG at *rs8099917*); and patients who were homozygous for the risk alleles had lower (mean 5.9 log_10_ IU/mL for TT at *rs12979860*, 6.0 log_10_ IU/mL for GG at *rs12980275*, and 5.8 log_10_ IU/mL for GG at *rs8099917*, P = 0.0013, P = 0.029, and P = 0.0004 respectively; [Table 1](#pone-0017232-t001)).

### Figure 1. Frequency distribution of IL28B variants in relation to HCV genotypes 1-3.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8b8a/3044738/0d20b249d882/pone.0017232.g001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3044738_pone.0017232.g001.jpg)

Significantly lower baseline plasma IP-10 levels were observed in homozygous carriers of the favorable CC at *rs12979860* (median 189 vs. 258 pg/mL, P = 0.02, Mann-Whitney U-test; [Figure 2](#pone-0017232-g002)), AA at *rs12980275* (median 189 vs. 258 pg/mL, P = 0.01), and TT at *rs8099917* (median 224 vs. 288 pg/mL, P = 0.04), as compared with patients carrying one or two copies of the risk alleles.

### Figure 2. Tenth, 25th, 50th, 75th, and 90th percentiles of pretreatment IP-10 in relation to IL28B variants.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8b8a/3044738/5e16ab0efbc5/pone.0017232.g002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3044738_pone.0017232.g002.jpg)

Among genotype 1 infected patients, homozygous carriers of the three favorable *IL28B* alleles had significantly more pronounced first phase viral decline, as reflected by the reduction of HCV RNA from treatment day 0 to 4, when compared with patients carrying the risk alleles (mean 2.0, 0.9 and 0.6 log_10_ IU/mL for *rs12979860* CC, CT and TT, 1.8, 0.9 and 0.7 log_10_ IU/mL for *rs12980275* AA, AG and TT, and 1.4, 0.8 and 0.6 for *rs8099917* TT, TG and GG respectively, P<0.0001 for all 3 SNPs; Kruskal-Wallis test). Among homozygous or heterozygous carriers of the favorable alleles, IP-10 was highly significantly associated with the first phase reduction of HCV RNA (*rs* = −0.50, P = 0.001 and *rs* = −0.40, P<0.0001 for *rs12979860* CC and CT, *rs* = −0.29, P = 0.04 and *rs* = −0.39, P = 0.0003 for *rs12980275* AA and AG, and *rs* = −0.40, P<0.0001 and *rs* = −0.25, P = 0.046 for *rs8099917* TT and TG respectively; [Figure 3](#pone-0017232-g003)). This association was also significant for the maximum decline in HCV RNA from day 0 to 4 as well as the decline from day 0 to 1, thus emphasizing an association with the first phase decline in HCV RNA, and translated into a more rapid reduction of HCV RNA during the first 4 weeks of therapy among patients with lower baseline IP-10, and a slower decline among those with higher ([Figure 4](#pone-0017232-g004)). Similarly, among genotype 2/3 infected patients, CC carriers of *rs12979860* had significantly more pronounced first phase viral decline, as reflected by the reduction of HCV RNA from treatment day 0 to 4, when compared with patients carrying the risk allele (mean 2.7, 2.1 and 1.6 log_10_ IU/mL for CC, CT and TT, P = 0.04; Kruskal-Wallis test) and IP-10 was significantly correlated with the reduction in HCV RNA from day 0 to 4 in the 22 HCV genotype 2/3 infected patients with *rs12979860* CT (*rs* = −0.45, P = 0.04). In contrast to the first phase reduction in HCV RNA, none of the *IL28B*-related SNPs or baseline IP-10 predicted the second phase decline after stratification for the first phase decline.

### Figure 3. Correlation between IP-10 and first phase decline in IL28B variants among HCV genotype 1 patients.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8b8a/3044738/911cff78ff3b/pone.0017232.g003.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3044738_pone.0017232.g003.jpg)

### Figure 4. Mean HCV RNA reduction according to IP-10 in HCV genotype 1 with favorable IL28B genotype.

![Figure 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8b8a/3044738/af328d2163ab/pone.0017232.g004.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3044738_pone.0017232.g004.jpg)

Both *rs12979860* (P = 0.006) and *rs12980275* (P = 0.007) were significantly predictive of RVR, i.e. undetectable HCV RNA at week 4, whereas the allelic distribution of *rs8099917* was not (P = 0.18). Similarly, baseline plasma IP-10 was lower in patients achieving RVR than in those who did not (median 222 pg/mL vs. 401 pg/mL, P = 0.008; Mann-Whitney U-test), and, among homozygous carriers of the *rs12979860* and *rs8099917* alleles, a lower baseline IP-10 significantly predicted RVR ([Table 2](#pone-0017232-t002)). Interestingly none of the patients with baseline IP-10 above 600 pg/mL achieved RVR regardless of *IL28B* genotype.

### Table 2. The impact of IL28B genetic variations and baseline plasma IP-10 on the likelihood of achieving RVR among patients infected with HCV genotype 1.

| NCBI dbSNP ID | <150 pg/mL | 150–600 pg/mL | >600 pg/mL | P | Total | Pa |
| --- | --- | --- | --- | --- | --- | --- |
| rs12979860 |   |   |   |   |   |   |
| CC | 8/13 (62%) | 8/25 (32%) | 0/4 (0%) | 0.05 | 16/42 (38%) | 0.006 |
| CT | 4/22 (18%) | 11/55 (20%) | 0/16 (0%) | 0.15 | 15/93 (16%) |   |
| TT | 0/6 (0%) | 3/17 (18%) | 0/3 (0%) | 0.41 | 3/27 (11%) |   |
| rs12980275 |   |   |   |   |   |   |
| AA | 9/17 (53%) | 9/29 (31%) | 0/4 (0%) | 0.10 | 18/50 (36%) | 0.007 |
| AG | 3/19 (16%) | 10/51 (20%) | 0/15 (0%) | 0.18 | 13/85 (15%) |   |
| GG | 0/5 (0%) | 3/17 (18%) | 0/4 (0%) | 0.41 | 3/27 (11%) |   |
| rs8099917 |   |   |   |   |   |   |
| TT | 11/28 (39%) | 13/53 (25%) | 0/14 (0%) | 0.02 | 24/95 (25%) | 0.18 |
| TG | 1/12 (8%) | 9/40 (22%) | 0/8 (0%) | 0.20 | 10/61 (16%) |   |
| GG | 0/1 (0%) | 0/4 (0%) | 0/1 (0%) |   | 0/6 (0%) |   |
| Total | 12/41 (29%) | 22/97 (23%) | 0/23 (0%) | 0.02 |   |   |

Table 2 Caption: All P-values using Chi squared test. aP-values for SNP genotypes and frequency of RVR irrespective of baseline IP-10.

The distribution of variants at SNP *rs12979860* and *rs12980275* impacted significantly on SVR among HCV genotype 1 infected patients (66% vs. 48% for *rs12979860* CC and CT/TT, P = 0.04; 65% vs. 47% for *rs12980275* AA and AG/GG, P = 0.03; Chi squared test), but not among variants of *rs8099917*, although a non-significant trend towards lower SVR was noted among the 7 homozygous GG carriers as compared to 162 carriers of TT or TG (29% vs. 54%, P = 0.2). Again, IP-10 significantly augmented the prediction of SVR among genotype 1 infected for all three *IL28B* SNPs with the exception of G allele carriage at *rs8099917* ([Figure 5](#pone-0017232-g005)). In patients with baseline IP-10 below 150 pg/mL who were homozygous for favorable SNP genotypes, SVR was achieved in 85%, 75%, and 75% for *rs12979860* CC, *rs12980275* AA, and *rs8099917* TT, respectively. Notably, achieving RVR was slightly more predictive of achieving SVR (91%; [Table 3](#pone-0017232-t003)) than combinations of *IL28B* and baseline IP-10. This remained the case even when the analysis was restricted to the genotype 1 infected patients who had no dose reductions, i.e. per-protocol analysis (92% vs. 83% for RVR and *rs12979860* CC and baseline IP-10 below 150 pg/mL respectively; n = 126). In contrast to genotype 1, neither *IL28B* genotype distribution nor baseline IP-10 levels predicted SVR among the 71 HCV genotype 2/3 infected patients.

### Figure 5. SVR rates in HCV genotype 1 according to IL28B variants and baseline IP-10.

![Figure 5](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8b8a/3044738/620ef78ff133/pone.0017232.g005.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3044738_pone.0017232.g005.jpg)

### Table 3. Sensitivity, specificity, positive and negative predictive values of the likelihood achieving SVR among patients infected with HCV genotype 1 (n = 170).

|   | PPV | NPV | Sensitivity | Specificity |
| --- | --- | --- | --- | --- |
| rs12979860 CC (n = 44) | 66% | 52% | 32% | 81% |
| Baseline plasma IP-10 | 66% | 52% | 33% | 81% |
| <150 pg/mL (n = 44) |   |   |   |   |
| CC at rs12979860 or | 63% | 56% | 53% | 65% |
| IP-10<150 pg/mL (n = 75) |   |   |   |   |
| CC at rs12979860 and | 85% | 50% | 12% | 98% |
| IP-10<150 pg/mL (n = 13) |   |   |   |   |
| RVR (n = 34) | 91% | 59% | 37% | 96% |
In multivariate analyses, both IP-10 and C genotype at *rs12979860* were independent predictors of the first phase decline in HCV RNA (P = 0.009 and P<0.0001 respectively) as well as RVR (P = 0.048 and P = 0.016 respectively), and RVR was in turn the only independent predictor of SVR (P = 0.001). RVR remained the only predictor of SVR when the analysis was restricted to the fully compliant patients (P = 0.005) reiterating the importance of monitoring on-treatment response.

## Discussion

In spite of the pending introduction of direct antiviral agents (DAA) in routine clinical practice, interferon-α and ribavirin are likely to retain pivotal roles in the management of chronic HCV infection, and thus predicting responsiveness to interferon/ribavirin-based therapies will remain important. In this setting, the main finding in the present study was that pretreatment plasma levels of IP-10 increased the level of prediction of the first phase decline in HCV RNA among patients carrying *IL28B* SNP variants, which translated into improved prediction of RVR and SVR. Considering the high SVR rates among HCV genotype 1 infected homozygous carriers of CC at *rs12979860*, AA at *rs12980275*, or TT at *rs8099917* with baseline IP-10 levels below 150 pg/mL (85%, 76%, and 75% respectively), these patients, although few in number, should be encouraged to initiate therapy and may be candidates for shortened duration of therapy in line with current treatment guidelines considering the high likelihood of achieving RVR [[24]](#pone.0017232-Lagging2), [[25]](#pone.0017232-Sarrazin1). Additionally, these patients may be suitable for initial inclusion in pending trials with interferon-free DAA regimes, because of the favorable odds of successful salvage therapy with interferon in the event of development of resistance towards these new therapeutic agents.

The finding that *IL28B* variants primarily impact on the first phase decline of HCV RNA [[26]](#pone.0017232-Thompson1), as previously reported for IP-10 [[20]](#pone.0017232-Askarieh1), suggests that both *IL28B* variants and IP-10 are linked to the antiviral effectiveness of peg-IFN and ribavirin in the blocking of the production or release of infectious virions rather than on the clearance of HCV infected cells. The risk alleles, T at *rs12979860*, or G at *rs12980275* and *rs8099917*, were found to be significantly associated with modest elevations of baseline IP-10 levels. This finding is in line with the recent report by Honda et al. that G at *rs8099917* entails higher intrahepatic expression of a panel of 37 representative ISGs not including IP-10 [[13]](#pone.0017232-Honda1), which is typically observed in patients who respond less favorably to treatment [[27]](#pone.0017232-SarasinFilipowicz1). The elevated baseline induction of ISGs among risk allele carriers may explain why these alleles are associated with a lower viral load observed in our study among patients with chronic HCV infection, corroborating previous reports [[8]](#pone.0017232-Ge1), [[11]](#pone.0017232-McCarthy1).

Our finding that homozygous CC at *rs12979860*, AA at *rs12980275*, and TT at *rs8099917* were significantly more common in the setting of HCV genotype 2 or 3 infection than 1 in a population consisting only of Caucasian patients confirms and extends the findings reported by McCarthy et al. [[11]](#pone.0017232-McCarthy1). Indeed, the proportion of CC at *rs12979860* among HCV genotype 2 and 3 infected patients (56% and 67%, respectively) in our study is higher than the reported prevalence in HCV uninfected Caucasians (∼40%), suggesting that this SNP genotype entails a higher likelihood of development of chronic infection following exposure to HCV genotype 2 or 3. This observation, however, may be considered counter-intuitive since homozygous carriage of CC at *rs12979860* was associated with a significantly greater first phase decline of HCV RNA even in genotype 2/3 infected patients. A possible hypothesis for these seemingly contradictory findings is that carriage of the favorable allele C at *rs12979860*, A at *rs12980275* and T at *rs8099917*, being associated with a slightly diminished baseline activation of ISGs, is beneficial for clearance of all HCV genotypes as reflected by the association with a greater first phase decline in HCV RNA during therapy irrespective of HCV genotype, but more advantageous for genotype 1 in comparison to 2/3. In the event of continuous re-exposure to a variety of HCV genotypes following a possible initial spontaneous clearance of HCV, as is often the case among intravenous drug users in addition to the lack of a lasting protective immune response [[28]](#pone.0017232-Farci1), this skewness will exert selective pressure and over time lead to an under-representation of these favorable alleles among genotype 1 patients and an over-representation among 2/3 patients as compared to the non-infected population. This would be the case even if carriage of this SNP genotype were slightly beneficial in the event of a single point exposure to HCV genotype 2 or 3. Supportive of this concept is the non-significant trend towards lower frequency of homozygous CC at *rs12979860* (30%) observed among 27 HCV genotype 2/3 patients infected through exposure to contaminated blood products as compared to 322 genotype 2/3 patients with other routes of infection in another study (NORDynamIC trial) [[29]](#pone.0017232-Lagging3). Similarly, in an elderly population in southern Italy predominantly infected with HCV genotype 2 likely secondary to past iatrogenic exposure, a non-significant trend towards a lower proportion of CC at *rs12979860* was noted among genotype 2/3 infected patients than among non-infected controls (37% vs. 42%) [[30]](#pone.0017232-Mangia1).

In conclusion, baseline plasma IP-10 is significantly associated with *IL28B*-related SNPs, and augments the level of predictiveness of the first phase decline in HCV RNA, RVR, and final treatment outcome. Therefore, pre-treatment screening of *IL28B* genetic variants, together with measurement of IP-10 in plasma, may provide useful prognostic information prior to initiating antiviral therapy for HCV.

## Acknowledgments

We thank Elke Verhey-Hart, Marie-Louise Landelius and Ulla Gingsjö for expert technical assistance.

## Footnotes

## References

1. WHO Hepatitis C- globalprevalence (update). Weekly Epidemilogical Report. 1999;74:425–427.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Weekly%20Epidemilogical%20Report&title=Hepatitis%20C-%20globalprevalence%20(update).&author=%20WHO&volume=74&publication_year=1999&pages=425-427&)

2. Saito I, Miyamura T, Ohbayashi A, Harada H, Katayama T, et al. Hepatitis C virus infection is associated with the development of hepatocellular carcinoma. Proc Natl Acad Sci U S A. 1990;87:6547–6549. doi: 10.1073/pnas.87.17.6547.  [DOI](https://doi.org/10.1073/pnas.87.17.6547) | [PMC free article](/articles/PMC54573/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2168552/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&title=Hepatitis%20C%20virus%20infection%20is%20associated%20with%20the%20development%20of%20hepatocellular%20carcinoma.&author=I%20Saito&author=T%20Miyamura&author=A%20Ohbayashi&author=H%20Harada&author=T%20Katayama&volume=87&publication_year=1990&pages=6547-6549&pmid=2168552&doi=10.1073/pnas.87.17.6547&)

3. Fried MW, Shiffman ML, Reddy KR, Smith C, Marinos G, et al. Peginterferon alfa-2a plus ribavirin for chronic hepatitis C virus infection. N Engl J Med. 2002;347:975–982. doi: 10.1056/NEJMoa020047.  [DOI](https://doi.org/10.1056/NEJMoa020047) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12324553/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Peginterferon%20alfa-2a%20plus%20ribavirin%20for%20chronic%20hepatitis%20C%20virus%20infection.&author=MW%20Fried&author=ML%20Shiffman&author=KR%20Reddy&author=C%20Smith&author=G%20Marinos&volume=347&publication_year=2002&pages=975-982&pmid=12324553&doi=10.1056/NEJMoa020047&)

4. Hadziyannis SJ, Sette H, Jr, Morgan TR, Balan V, Diago M, et al. Peginterferon-alpha2a and ribavirin combination therapy in chronic hepatitis C: a randomized study of treatment duration and ribavirin dose. Ann Intern Med. 2004;140:346–355. doi: 10.7326/0003-4819-140-5-200403020-00010.  [DOI](https://doi.org/10.7326/0003-4819-140-5-200403020-00010) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14996676/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Intern%20Med&title=Peginterferon-alpha2a%20and%20ribavirin%20combination%20therapy%20in%20chronic%20hepatitis%20C:%20a%20randomized%20study%20of%20treatment%20duration%20and%20ribavirin%20dose.&author=SJ%20Hadziyannis&author=H%20Sette&author=TR%20Morgan&author=V%20Balan&author=M%20Diago&volume=140&publication_year=2004&pages=346-355&pmid=14996676&doi=10.7326/0003-4819-140-5-200403020-00010&)

5. Manns MP, McHutchison JG, Gordon SC, Rustgi VK, Shiffman M, et al. Peginterferon alfa-2b plus ribavirin compared with interferon alfa-2b plus ribavirin for initial treatment of chronic hepatitis C: a randomised trial. Lancet. 2001;358:958–965. doi: 10.1016/s0140-6736(01)06102-5.  [DOI](https://doi.org/10.1016/s0140-6736(01)06102-5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11583749/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Peginterferon%20alfa-2b%20plus%20ribavirin%20compared%20with%20interferon%20alfa-2b%20plus%20ribavirin%20for%20initial%20treatment%20of%20chronic%20hepatitis%20C:%20a%20randomised%20trial.&author=MP%20Manns&author=JG%20McHutchison&author=SC%20Gordon&author=VK%20Rustgi&author=M%20Shiffman&volume=358&publication_year=2001&pages=958-965&pmid=11583749&doi=10.1016/s0140-6736(01)06102-5&)

6. Rauch A, Kutalik Z, Descombes P, Cai T, Di Iulio J, et al. Genetic variation in IL28B is associated with chronic hepatitis C and treatment failure: a genome-wide association study. Gastroenterology. 2010;138:1338–1345, 1345 e1331–1337. doi: 10.1053/j.gastro.2009.12.056.  [DOI](https://doi.org/10.1053/j.gastro.2009.12.056) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20060832/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastroenterology&title=Genetic%20variation%20in%20IL28B%20is%20associated%20with%20chronic%20hepatitis%20C%20and%20treatment%20failure:%20a%20genome-wide%20association%20study.&author=A%20Rauch&author=Z%20Kutalik&author=P%20Descombes&author=T%20Cai&author=J%20Di%20Iulio&volume=138&publication_year=2010&pages=1338-1345,%201345%20e1331%E2%80%931337&pmid=20060832&doi=10.1053/j.gastro.2009.12.056&)

7. Thomas DL, Thio CL, Martin MP, Qi Y, Ge D, et al. Genetic variation in IL28B and spontaneous clearance of hepatitis C virus. Nature. 2009;461:798–801. doi: 10.1038/nature08463.  [DOI](https://doi.org/10.1038/nature08463) | [PMC free article](/articles/PMC3172006/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19759533/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Genetic%20variation%20in%20IL28B%20and%20spontaneous%20clearance%20of%20hepatitis%20C%20virus.&author=DL%20Thomas&author=CL%20Thio&author=MP%20Martin&author=Y%20Qi&author=D%20Ge&volume=461&publication_year=2009&pages=798-801&pmid=19759533&doi=10.1038/nature08463&)

8. Ge D, Fellay J, Thompson AJ, Simon JS, Shianna KV, et al. Genetic variation in IL28B predicts hepatitis C treatment-induced viral clearance. Nature. 2009;461:399–401. doi: 10.1038/nature08309.  [DOI](https://doi.org/10.1038/nature08309) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19684573/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Genetic%20variation%20in%20IL28B%20predicts%20hepatitis%20C%20treatment-induced%20viral%20clearance.&author=D%20Ge&author=J%20Fellay&author=AJ%20Thompson&author=JS%20Simon&author=KV%20Shianna&volume=461&publication_year=2009&pages=399-401&pmid=19684573&doi=10.1038/nature08309&)

9. Suppiah V, Moldovan M, Ahlenstiel G, Berg T, Weltman M, et al. IL28B is associated with response to chronic hepatitis C interferon-alpha and ribavirin therapy. Nat Genet. 2009;41:1100–1104. doi: 10.1038/ng.447.  [DOI](https://doi.org/10.1038/ng.447) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19749758/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=IL28B%20is%20associated%20with%20response%20to%20chronic%20hepatitis%20C%20interferon-alpha%20and%20ribavirin%20therapy.&author=V%20Suppiah&author=M%20Moldovan&author=G%20Ahlenstiel&author=T%20Berg&author=M%20Weltman&volume=41&publication_year=2009&pages=1100-1104&pmid=19749758&doi=10.1038/ng.447&)

10. Tanaka Y, Nishida N, Sugiyama M, Kurosaki M, Matsuura K, et al. Genome-wide association of IL28B with response to pegylated interferon-alpha and ribavirin therapy for chronic hepatitis C. Nat Genet. 2009;41:1105–1109. doi: 10.1038/ng.449.  [DOI](https://doi.org/10.1038/ng.449) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19749757/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Genome-wide%20association%20of%20IL28B%20with%20response%20to%20pegylated%20interferon-alpha%20and%20ribavirin%20therapy%20for%20chronic%20hepatitis%20C.&author=Y%20Tanaka&author=N%20Nishida&author=M%20Sugiyama&author=M%20Kurosaki&author=K%20Matsuura&volume=41&publication_year=2009&pages=1105-1109&pmid=19749757&doi=10.1038/ng.449&)

11. McCarthy JJ, Li JH, Thompson A, Suchindran S, Lao XQ, et al. Replicated association between an IL28B gene variant and a sustained response to pegylated interferon and ribavirin. Gastroenterology. 2010;138:2307–2314. doi: 10.1053/j.gastro.2010.02.009.  [DOI](https://doi.org/10.1053/j.gastro.2010.02.009) | [PMC free article](/articles/PMC2883666/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20176026/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastroenterology&title=Replicated%20association%20between%20an%20IL28B%20gene%20variant%20and%20a%20sustained%20response%20to%20pegylated%20interferon%20and%20ribavirin.&author=JJ%20McCarthy&author=JH%20Li&author=A%20Thompson&author=S%20Suchindran&author=XQ%20Lao&volume=138&publication_year=2010&pages=2307-2314&pmid=20176026&doi=10.1053/j.gastro.2010.02.009&)

12. Montes-Cano MA, Garcia-Lozano JR, Abad-Molina C, Romero-Gomez M, Barroso N, et al. Interleukin-28B genetic variants and hepatitis virus infection by different viral genotypes. Hepatology. 2010;52:33–37. doi: 10.1002/hep.23624.  [DOI](https://doi.org/10.1002/hep.23624) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20578254/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hepatology&title=Interleukin-28B%20genetic%20variants%20and%20hepatitis%20virus%20infection%20by%20different%20viral%20genotypes.&author=MA%20Montes-Cano&author=JR%20Garcia-Lozano&author=C%20Abad-Molina&author=M%20Romero-Gomez&author=N%20Barroso&volume=52&publication_year=2010&pages=33-37&pmid=20578254&doi=10.1002/hep.23624&)

13. Honda M, Sakai A, Yamashita T, Nakamoto Y, Mizukoshi E, et al. Hepatic ISG expression is associated with genetic variation in interleukin 28B and the outcome of IFN therapy for chronic hepatitis C. Gastroenterology. 2010;139:499–509. doi: 10.1053/j.gastro.2010.04.049.  [DOI](https://doi.org/10.1053/j.gastro.2010.04.049) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20434452/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastroenterology&title=Hepatic%20ISG%20expression%20is%20associated%20with%20genetic%20variation%20in%20interleukin%2028B%20and%20the%20outcome%20of%20IFN%20therapy%20for%20chronic%20hepatitis%20C.&author=M%20Honda&author=A%20Sakai&author=T%20Yamashita&author=Y%20Nakamoto&author=E%20Mizukoshi&volume=139&publication_year=2010&pages=499-509&pmid=20434452&doi=10.1053/j.gastro.2010.04.049&)

14. Luster AD, Unkeless JC, Ravetch JV. Gamma-interferon transcriptionally regulates an early-response gene containing homology to platelet proteins. Nature. 1985;315:672–676. doi: 10.1038/315672a0.  [DOI](https://doi.org/10.1038/315672a0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/3925348/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Gamma-interferon%20transcriptionally%20regulates%20an%20early-response%20gene%20containing%20homology%20to%20platelet%20proteins.&author=AD%20Luster&author=JC%20Unkeless&author=JV%20Ravetch&volume=315&publication_year=1985&pages=672-676&pmid=3925348&doi=10.1038/315672a0&)

15. Neville LF, Mathiak G, Bagasra O. The immunobiology of interferon-gamma inducible protein 10 kD (IP-10): a novel, pleiotropic member of the C-X-C chemokine superfamily. Cytokine Growth Factor Rev. 1997;8:207–219. doi: 10.1016/s1359-6101(97)00015-4.  [DOI](https://doi.org/10.1016/s1359-6101(97)00015-4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9462486/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cytokine%20Growth%20Factor%20Rev&title=The%20immunobiology%20of%20interferon-gamma%20inducible%20protein%2010%20kD%20(IP-10):%20a%20novel,%20pleiotropic%20member%20of%20the%20C-X-C%20chemokine%20superfamily.&author=LF%20Neville&author=G%20Mathiak&author=O%20Bagasra&volume=8&publication_year=1997&pages=207-219&pmid=9462486&doi=10.1016/s1359-6101(97)00015-4&)

16. Taub DD, Sayers TJ, Carter CR, Ortaldo JR. Alpha and beta chemokines induce NK cell migration and enhance NK-mediated cytolysis. J Immunol. 1995;155:3877–3888.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/7561094/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Immunol&title=Alpha%20and%20beta%20chemokines%20induce%20NK%20cell%20migration%20and%20enhance%20NK-mediated%20cytolysis.&author=DD%20Taub&author=TJ%20Sayers&author=CR%20Carter&author=JR%20Ortaldo&volume=155&publication_year=1995&pages=3877-3888&pmid=7561094&)

17. Taub DD, Lloyd AR, Conlon K, Wang JM, Ortaldo JR, et al. Recombinant human interferon-inducible protein 10 is a chemoattractant for human monocytes and T lymphocytes and promotes T cell adhesion to endothelial cells. J Exp Med. 1993;177:1809–1814. doi: 10.1084/jem.177.6.1809.  [DOI](https://doi.org/10.1084/jem.177.6.1809) | [PMC free article](/articles/PMC2191047/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8496693/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Exp%20Med&title=Recombinant%20human%20interferon-inducible%20protein%2010%20is%20a%20chemoattractant%20for%20human%20monocytes%20and%20T%20lymphocytes%20and%20promotes%20T%20cell%20adhesion%20to%20endothelial%20cells.&author=DD%20Taub&author=AR%20Lloyd&author=K%20Conlon&author=JM%20Wang&author=JR%20Ortaldo&volume=177&publication_year=1993&pages=1809-1814&pmid=8496693&doi=10.1084/jem.177.6.1809&)

18. Romero AI, Lagging M, Westin J, Dhillon AP, Dustin LB, et al. Interferon (IFN)-gamma-inducible protein-10: association with histological results, viral kinetics, and outcome during treatment with pegylated IFN-alpha 2a and ribavirin for chronic hepatitis C virus infection. J Infect Dis. 2006;194:895–903. doi: 10.1086/507307.  [DOI](https://doi.org/10.1086/507307) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16960776/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Infect%20Dis&title=Interferon%20(IFN)-gamma-inducible%20protein-10:%20association%20with%20histological%20results,%20viral%20kinetics,%20and%20outcome%20during%20treatment%20with%20pegylated%20IFN-alpha%202a%20and%20ribavirin%20for%20chronic%20hepatitis%20C%20virus%20infection.&author=AI%20Romero&author=M%20Lagging&author=J%20Westin&author=AP%20Dhillon&author=LB%20Dustin&volume=194&publication_year=2006&pages=895-903&pmid=16960776&doi=10.1086/507307&)

19. Lagging M, Romero AI, Westin J, Norkrans G, Dhillon AP, et al. IP-10 predicts viral response and therapeutic outcome in difficult-to-treat patients with HCV genotype 1 infection. Hepatology. 2006;44:1617–1625. doi: 10.1002/hep.21407.  [DOI](https://doi.org/10.1002/hep.21407) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17133471/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hepatology&title=IP-10%20predicts%20viral%20response%20and%20therapeutic%20outcome%20in%20difficult-to-treat%20patients%20with%20HCV%20genotype%201%20infection.&author=M%20Lagging&author=AI%20Romero&author=J%20Westin&author=G%20Norkrans&author=AP%20Dhillon&volume=44&publication_year=2006&pages=1617-1625&pmid=17133471&doi=10.1002/hep.21407&)

20. Askarieh G, Alsio A, Pugnale P, Negro F, Ferrari C, et al. Systemic and intrahepatic interferon-gamma-inducible protein 10 kDa predicts the first-phase decline in hepatitis C virus RNA and overall viral response to therapy in chronic hepatitis C. Hepatology. 2010;51:1523–1530. doi: 10.1002/hep.23509.  [DOI](https://doi.org/10.1002/hep.23509) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20186843/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hepatology&title=Systemic%20and%20intrahepatic%20interferon-gamma-inducible%20protein%2010%20kDa%20predicts%20the%20first-phase%20decline%20in%20hepatitis%20C%20virus%20RNA%20and%20overall%20viral%20response%20to%20therapy%20in%20chronic%20hepatitis%20C.&author=G%20Askarieh&author=A%20Alsio&author=P%20Pugnale&author=F%20Negro&author=C%20Ferrari&volume=51&publication_year=2010&pages=1523-1530&pmid=20186843&doi=10.1002/hep.23509&)

21. Zeuzem S, Pawlotsky JM, Lukasiewicz E, von Wagner M, Goulis I, et al. International, multicenter, randomized, controlled study comparing dynamically individualized versus standard treatment in patients with chronic hepatitis C. J Hepatol. 2005;43:250–257. doi: 10.1016/j.jhep.2005.05.016.  [DOI](https://doi.org/10.1016/j.jhep.2005.05.016) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16082736/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Hepatol&title=International,%20multicenter,%20randomized,%20controlled%20study%20comparing%20dynamically%20individualized%20versus%20standard%20treatment%20in%20patients%20with%20chronic%20hepatitis%20C.&author=S%20Zeuzem&author=JM%20Pawlotsky&author=E%20Lukasiewicz&author=M%20von%20Wagner&author=I%20Goulis&volume=43&publication_year=2005&pages=250-257&pmid=16082736&doi=10.1016/j.jhep.2005.05.016&)

22. Ishak K, Baptista A, Bianchi L, Callea F, De Groote J, et al. Histological grading and staging of chronic hepatitis. J Hepatol. 1995;22:696–699. doi: 10.1016/0168-8278(95)80226-6.  [DOI](https://doi.org/10.1016/0168-8278(95)80226-6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7560864/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Hepatol&title=Histological%20grading%20and%20staging%20of%20chronic%20hepatitis.&author=K%20Ishak&author=A%20Baptista&author=L%20Bianchi&author=F%20Callea&author=J%20De%20Groote&volume=22&publication_year=1995&pages=696-699&pmid=7560864&doi=10.1016/0168-8278(95)80226-6&)

23. Westin J, Nordlinder H, Lagging M, Norkrans G, Wejstal R. Steatosis accelerates fibrosis development over time in hepatitis C virus genotype 3 infected patients. J Hepatol. 2002;37:837–842. doi: 10.1016/s0168-8278(02)00299-4.  [DOI](https://doi.org/10.1016/s0168-8278(02)00299-4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12445426/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Hepatol&title=Steatosis%20accelerates%20fibrosis%20development%20over%20time%20in%20hepatitis%20C%20virus%20genotype%203%20infected%20patients.&author=J%20Westin&author=H%20Nordlinder&author=M%20Lagging&author=G%20Norkrans&author=R%20Wejstal&volume=37&publication_year=2002&pages=837-842&pmid=12445426&doi=10.1016/s0168-8278(02)00299-4&)

24. Lagging M, Wejstal R, Uhnoo I, Gerden B, Fischler B, et al. Treatment of hepatitis C virus infection: updated Swedish Consensus recommendations. Scand J Infect Dis. 2009;41:389–402. doi: 10.1080/00365540902998271.  [DOI](https://doi.org/10.1080/00365540902998271) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20001276/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Scand%20J%20Infect%20Dis&title=Treatment%20of%20hepatitis%20C%20virus%20infection:%20updated%20Swedish%20Consensus%20recommendations.&author=M%20Lagging&author=R%20Wejstal&author=I%20Uhnoo&author=B%20Gerden&author=B%20Fischler&volume=41&publication_year=2009&pages=389-402&pmid=20001276&doi=10.1080/00365540902998271&)

25. Sarrazin C, Berg T, Ross RS, Schirmacher P, Wedemeyer H, et al. [Prophylaxis, diagnosis and therapy of hepatitis C virus (HCV) infection: the German guidelines on the management of HCV infection]. Z Gastroenterol. 2010;48:289–351. doi: 10.1055/s-0028-1110008.  [DOI](https://doi.org/10.1055/s-0028-1110008) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20119896/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Z%20Gastroenterol&title=%5BProphylaxis,%20diagnosis%20and%20therapy%20of%20hepatitis%20C%20virus%20(HCV)%20infection:%20the%20German%20guidelines%20on%20the%20management%20of%20HCV%20infection%5D.&author=C%20Sarrazin&author=T%20Berg&author=RS%20Ross&author=P%20Schirmacher&author=H%20Wedemeyer&volume=48&publication_year=2010&pages=289-351&pmid=20119896&doi=10.1055/s-0028-1110008&)

26. Thompson AJ, Muir AJ, Sulkowski MS, Ge D, Fellay J, et al. Interleukin-28B polymorphism improves viral kinetics and is the strongest pretreatment predictor of sustained virologic response in genotype 1 hepatitis C virus. Gastroenterology. 2010;139:120–129 e118. doi: 10.1053/j.gastro.2010.04.013.  [DOI](https://doi.org/10.1053/j.gastro.2010.04.013) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20399780/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastroenterology&title=Interleukin-28B%20polymorphism%20improves%20viral%20kinetics%20and%20is%20the%20strongest%20pretreatment%20predictor%20of%20sustained%20virologic%20response%20in%20genotype%201%20hepatitis%20C%20virus.&author=AJ%20Thompson&author=AJ%20Muir&author=MS%20Sulkowski&author=D%20Ge&author=J%20Fellay&volume=139&publication_year=2010&pages=120-129%20e118&pmid=20399780&doi=10.1053/j.gastro.2010.04.013&)

27. Sarasin-Filipowicz M, Oakeley EJ, Duong FH, Christen V, Terracciano L, et al. Interferon signaling and treatment outcome in chronic hepatitis C. Proc Natl Acad Sci U S A. 2008;105:7034–7039. doi: 10.1073/pnas.0707882105.  [DOI](https://doi.org/10.1073/pnas.0707882105) | [PMC free article](/articles/PMC2383932/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18467494/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&title=Interferon%20signaling%20and%20treatment%20outcome%20in%20chronic%20hepatitis%20C.&author=M%20Sarasin-Filipowicz&author=EJ%20Oakeley&author=FH%20Duong&author=V%20Christen&author=L%20Terracciano&volume=105&publication_year=2008&pages=7034-7039&pmid=18467494&doi=10.1073/pnas.0707882105&)

28. Farci P, Alter HJ, Govindarajan S, Wong DC, Engle R, et al. Lack of protective immunity against reinfection with hepatitis C virus. Science. 1992;258:135–140. doi: 10.1126/science.1279801.  [DOI](https://doi.org/10.1126/science.1279801) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1279801/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Lack%20of%20protective%20immunity%20against%20reinfection%20with%20hepatitis%20C%20virus.&author=P%20Farci&author=HJ%20Alter&author=S%20Govindarajan&author=DC%20Wong&author=R%20Engle&volume=258&publication_year=1992&pages=135-140&pmid=1279801&doi=10.1126/science.1279801&)

29. Lagging M, Langeland N, Pedersen C, Farkkila M, Buhl MR, et al. Randomized comparison of 12 or 24 weeks of peginterferon alpha-2a and ribavirin in chronic hepatitis C virus genotype 2/3 infection. Hepatology. 2008;47:1837–1845. doi: 10.1002/hep.22253.  [DOI](https://doi.org/10.1002/hep.22253) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18454508/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hepatology&title=Randomized%20comparison%20of%2012%20or%2024%20weeks%20of%20peginterferon%20alpha-2a%20and%20ribavirin%20in%20chronic%20hepatitis%20C%20virus%20genotype%202/3%20infection.&author=M%20Lagging&author=N%20Langeland&author=C%20Pedersen&author=M%20Farkkila&author=MR%20Buhl&volume=47&publication_year=2008&pages=1837-1845&pmid=18454508&doi=10.1002/hep.22253&)

30. Mangia A, Thompson AJ, Santoro R, Piazzolla V, Tillmann HL, et al. An IL28B polymorphism determines treatment response of hepatitis C virus genotype 2 or 3 patients who do not achieve a rapid virologic response. Gastroenterology. 2010;139:821–827, 827 e821. doi: 10.1053/j.gastro.2010.05.079.  [DOI](https://doi.org/10.1053/j.gastro.2010.05.079) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20621700/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastroenterology&title=An%20IL28B%20polymorphism%20determines%20treatment%20response%20of%20hepatitis%20C%20virus%20genotype%202%20or%203%20patients%20who%20do%20not%20achieve%20a%20rapid%20virologic%20response.&author=A%20Mangia&author=AJ%20Thompson&author=R%20Santoro&author=V%20Piazzolla&author=HL%20Tillmann&volume=139&publication_year=2010&pages=821-827,%20827%20e821&pmid=20621700&doi=10.1053/j.gastro.2010.05.079&)
