# RHOA Is a Modulator of the Cholesterol-Lowering Effects of Statin

## Metadata
**Authors:** Marisa W Medina, Elizabeth Theusch, Devesh Naidoo, Frederick Bauzon, Kristen Stevens, Lara M Mangravite, Yu-Lin Kuang, Ronald M Krauss
**Journal:** PLoS Genetics
**Date:** 2012 Nov 15
**DOI:** [10.1371/journal.pgen.1003058](https://doi.org/10.1371/journal.pgen.1003058)
**PMID:** 23166513
**PMCID:** PMC3499361
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3499361/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC3499361/pdf/pgen.1003058.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC3499361/pdf/pgen.1003058.pdf)

## Abstract

Although statin drugs are generally efficacious for lowering plasma LDL-cholesterol levels, there is considerable variability in response. To identify candidate genes that may contribute to this variation, we used an unbiased genome-wide filter approach that was applied to 10,149 genes expressed in immortalized lymphoblastoid cell lines (LCLs) derived from 480 participants of the Cholesterol and Pharmacogenomics (CAP) clinical trial of simvastatin. The criteria for identification of candidates included genes whose statin-induced changes in expression were correlated with change in expression of HMGCR, a key regulator of cellular cholesterol metabolism and the target of statin inhibition. This analysis yielded 45 genes, from which RHOA was selected for follow-up because it has been found to participate in mediating the pleiotropic but not the lipid-lowering effects of statin treatment. RHOA knock-down in hepatoma cell lines reduced HMGCR, LDLR, and SREBF2 mRNA expression and increased intracellular cholesterol ester content as well as apolipoprotein B (APOB) concentrations in the conditioned media. Furthermore, inter-individual variation in statin-induced RHOA mRNA expression measured in vitro in CAP LCLs was correlated with the changes in plasma total cholesterol, LDL-cholesterol, and APOB induced by simvastatin treatment (40 mg/d for 6 wk) of the individuals from whom these cell lines were derived. Moreover, the minor allele of rs11716445, a SNP located in a novel cryptic RHOA exon, dramatically increased inclusion of the exon in RHOA transcripts during splicing and was associated with a smaller LDL-cholesterol reduction in response to statin treatment in 1,886 participants from the CAP and Pravastatin Inflamation and CRP Evaluation (PRINCE; pravastatin 40 mg/d) statin clinical trials. Thus, an unbiased filter approach based on transcriptome-wide profiling identified RHOA as a gene contributing to variation in LDL-cholesterol response to statin, illustrating the power of this approach for identifying candidate genes involved in drug response phenotypes.

## Introduction

Genome-wide association studies (GWAS) have been used to identify genetic contributors to a number of common diseases and traits [[1]](#pgen.1003058-Altshuler1). However, a major problem with this approach is that very large sample sizes are generally required to detect statistically significant associations [[2]](#pgen.1003058-Amos1). This is especially the case for pharmacogenomics, where identification of gene variants associated with drug response may require larger sample sizes than are generally available. Consequently, GWAS has had limited success in the identification of pharmacogenetically relevant single nucleotide polymorphisms (SNPs) that survive the stringency of genome-wide multiple testing [[3]](#pgen.1003058-Thompson1), [[4]](#pgen.1003058-Tanaka1). In the largest single statin clinical trial GWAS published to date (the JUPITER trial of ∼7000 individuals) only three loci (*ABCG2*, *APOE* and *LPA*) achieved genome-wide significance for association with the magnitude of LDL cholesterol reduction, and in total accounted for only a minor fraction of the overall variation in response [[5]](#pgen.1003058-Chasman1). Moreover, GWAS studies are limited by their ability to probe only common genetic variation and thus the limited findings suggest that association studies alone are unlikely to yield the basis for all or even the majority of the genetic variance associated with drug response.

In the present report, we describe the use of transcriptome-wide profiling to identify and prioritize genes that may contribute to inter-individual variation in statin-induced plasma LDL-cholesterol lowering. Statins inhibit HMG-CoA reductase (HMGCR), the enzyme that catalyzes the rate limiting step of cholesterol biosynthesis, thus lowering intracellular cholesterol levels [[6]](#pgen.1003058-Goldstein1). This in turn elicits an increase in expression of cellular LDL receptors that mediate plasma LDL clearance [[7]](#pgen.1003058-Goldstein2). Since the *HMGCR* gene is transcriptionally regulated by intracellular sterol content [[8]](#pgen.1003058-Brown1), the magnitude of induction of this gene is a cellular marker of *in vitro* statin response. We used expression array data from *in vitro* statin-exposed immortalized human hepatoma cell lines and lymphoblastoid cell lines established from participants of the Cholesterol and Pharmacogenetics (CAP) clinical trial of simvastatin treatment [[9]](#pgen.1003058-Simon1) to establish a set of “biological rules” for identifying genes whose expression characteristics qualified them as having biologically plausible effects on cholesterol metabolism. *RHOA* emerged from this analysis, and subsequent functional and genetic studies, as a novel candidate gene contributing to variation in LDL response to statins.

## Results

### Identification of RHOA as a candidate gene

We used a series of filters applied to genome-wide gene expression data from 480 human lymphoblastoid cell lines (LCLs) derived from participants in the Cholesterol and Pharmacogenetics study to identify genes that appeared to be biologically plausible candidates for modulating the effects of statins on cholesterol metabolism. The following filter criteria were used ([Table 1](#pgen-1003058-t001)): 1) expression in normal human liver; 2) change in transcript levels in HepG2 (n = 4) and Hep3B (n = 3) human hepatoma cell lines incubated with 2.0 µM activated simvastatin versus sham buffer for 24 hr, FDR<0.01, 3) change in transcript levels in CAP LCLs incubated with 2.0 µM activated simvastatin versus sham buffer for 24 hr (Q<0.05); 4) consistent directionality of statin-induced change in transcripts in hepatoma cell lines and LCLs; 5) correlation of statin-induced gene expression change in CAP LCLs with change in expression of *HMGCR*. After Bonferroni correction for multiple testing (p<1.17e-04) we identified 45 genes which passed all filter criteria ([Table 2](#pgen-1003058-t002)). When ranked in order of correlation, only two of the top thirteen genes did not encode enzymes in the cholesterol biosynthesis pathway: transmembrane protein 97 (*TMEM97*) and ras homolog gene family member A (*RHOA*). Although both had been previously implicated in lipid metabolism [[10]](#pgen.1003058-Guillaumot1), [[11]](#pgen.1003058-Okuhira1), [[12]](#pgen.1003058-Bartz1), neither had been shown to play a role in the cholesterol lowering effects of statin. However, *RHOA* was particularly intriguing since inhibition of RHOA signaling is thought to be a major mechanism by which statins exert pleiotropic (or non-lipid lowering) actions, such as anti-inflammatory effects. [Figure 1A](#pgen-1003058-g001) demonstrates the strong correlation between statin-induced change in *RHOA* and *HMGCR* transcript levels (p = 7.64E-16, r^2^ = 0.13).

### Table 1. Identification of candidate genes associated with statin.

| Expression Characteristics | Genes |
| --- | --- |
| Expressed in lymphoblastoid cell lines | 10,149 |
| Expressed in normal human liver, p<0.05 | 6629 |
| Statin responsive in hepatoma cell lines, FDR<0.01 | 845 |
| Statin responsive in LCLs, Q<0.05 | 725 |
| Same direction of statin response | 427 |
| Statin-induced expression correlated with statin-induced change in HMGCR, |   |
| Bonferroni (p<0.000117) | 45 |

Table 1 Caption: HepG2 (n = 4), Hep3B (n = 3) and 480 LCLs derived from participants of the Cholesterol and Pharmacogenetics (CAP) clinical trial were exposed to either 2 µM activated simvastatin or sham buffer for 24 hours, and gene expression was quantified on the Illumina HTref8v3 bead chip. Genes expressed in both LCLs and normal human liver (n = 60, GEO dataset GSE28893) were identified and tested for evidence of statin-induced changes in gene expression.

### Table 2. Expression characteristics of identified candidate genes.

|   | HepG2 or 3B (n = 7) |  | CAP LCLs (n = 480) |  | Correlation with HMGCR |  |
| --- | --- | --- | --- | --- | --- | --- |
| Gene Name | Fold Change | FDR | Fold Change | Q-value | p-value | r |
| HMGCS1 | 1.75±0.13 | 0 | 1.56±0.42 | 0 | 1.76E-40 | 0.50 |
| SQLE | 1.59±0.09 | 0 | 1.37±0.29 | 0 | 1.51E-26 | 0.46 |
| DHCR7 | 1.82±0.03 | 0 | 1.42±0.33 | 0 | 1.55E-26 | 0.36 |
| SC4MOL | 2.07±0.15 | 2.21E-03 | 1.53±0.70 | 0 | 2.27E-26 | 0.43 |
| TM7SF2 | 1.65±0.18 | 5.54E-03 | 1.65±0.49 | 0 | 4.33E-26 | 0.40 |
| LSS | 1.88±0.07 | 0 | 1.51±0.37 | 0 | 1.59E-23 | 0.35 |
| MVK | 1.80±0.17 | 2.21E-03 | 1.32±0.30 | 0 | 1.83E-19 | 0.37 |
| MVD | 2.15±0.15 | 4.56E-03 | 1.38±0.35 | 0 | 1.35E-17 | 0.31 |
| TMEM97 | 1.49±0.04 | 0 | 1.19±0.18 | 0 | 9.72E-17 | 0.29 |
| NSDHL | 1.53±0.07 | 0 | 1.19±0.25 | 0 | 5.68E-16 | 0.35 |
| RHOA | 1.47±0.09 | 2.67E-03 | 1.32±0.47 | 0 | 7.64E-16 | 0.39 |
| EBP | 1.42±0.09 | 8.81E-03 | 1.20±0.24 | 0 | 1.32E-14 | 0.30 |
| FDFT1 | 1.48±0.07 | 0 | 1.19±0.19 | 0 | 1.42E-14 | 0.32 |
| MSI2 | 0.83±0.02 | 0 | 0.96±0.14 | 0 | 3.68E-14 | −0.23 |
| FADS1 | 1.52±0.14 | 9.97E-03 | 1.24±0.31 | 0 | 2.96E-13 | 0.23 |
### Figure 1. Functional validation of RHOA.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c51b/3499361/c2dbce5cc22d/pgen.1003058.g001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3499361_pgen.1003058.g001.jpg)

(A) HMGCR and RHOA levels were quantified in 480 CAP LCLs after 24 hour incubation with 2 µM activated simvastatin or sham buffer. HepG2 (n = 12) and Huh7 cells (n = 6) cells were reverse transfected with Silencer Select siRNAs specific for RHOA or a negative control using the NeoSport transfection reagent. (B) After 48 hours, RHOA transcript levels were quantified by real time qPCR and compared to cells treated with the negative control to calculate percent knock-down of the RHOA transcript. (C) HepG2 cells (n = 2) were transfected and the cell lysate probed by immunoblot with antibodies to RHOA and β-actin. (D) HMGCR, SREBF2, and LDLR transcript levels were quantified by qPCR. APOB and APOAI accumulated in the cell culture media were quantified by ELISA. (E) Total and free intracellular cholesterol levels were quantified using the Amplex Red Cholesterol Assay Kit (Invitrogen), with cholesterol ester calculated as the difference of total minus free cholesterol. (F) HepG2 cells (n = 10) were incubated for 24 hours with 2 µM simvastatin +10% LPDS or sham buffer +10% FBS (control conditions), after which cells were harvested and RHOA and HMGCR mRNA were quantified by qPCR, and RHOA and HMGCR protein were quantified by immunoblot. Band densities were quantified using Alphaview SA version 3.4.0. The dashed line indicates a fold change of 1. (G) One representative immunoblot of HepG2 cells incubated as described in section D. (H) HepG2 and Huh7 cells (n = 6 each cell type) were transfected with the Silencer Select siRNA specific for SREBF1, SREBF2 or a non-targeting negative control. RHOA, HMGCR and CLPTM1 transcript levels were quantified after 48 hours. Statistically significant differences in gene expression, APOB, APOAI and intracellular cholesterol were evaluated by paired two-tailed t-test. All values shown are mean ± standard error. *p<0.05; **p<0.01, ***p<0.001.

### Effects of RHOA on intracellular cholesterol homeostasis

To determine if RHOA has a direct effect on markers of intracellular cholesterol homeostasis, we transfected HepG2 cells (n = 10) with siRNAs specific for *RHOA* or a non-targeting negative control and tested for changes in expression of *HMGCR*, low-density lipoprotein receptor (*LDLR*) and sterol response element binding transcription factor (*SREBF2* aka *SREBP2*) gene expression. Knock-down reduced *RHOA* transcript levels by 60-98% ([Figure 1B](#pgen-1003058-g001)), with no remaining detectable RHOA protein ([Figure 1C](#pgen-1003058-g001)), and also generated statistically significant reductions in expression of *HMGCR* (0.76±0.04 fold, p = 0.002), *SREBF2* (0.58±0.03 fold, p = 0.0003), and *LDLR* (0.73±0.13 fold, p = 0.03) [Figure 1D](#pgen-1003058-g001). *RHOA* knockdown-mediated reductions in expression of *HMGCR, LDLR*, and *SREBF2* were confirmed in a second hepatoma cell line, Huh7 (n = 6); however, the magnitude of the effect was less dramatic than that observed in the HepG2 transfections.

To further test the functional role of RHOA, we also measured levels of secreted APOB and APOA1, the major proteins on LDL and HDL particles respectively, in the culture media 48 hours after knock-down. APOB accumulation in the cell culture media was increased in HepG2 cells after *RHOA* knock-down (1.28±0.08 fold, p = 0.03, n = 12), while a similar but non-statistically significant trend was observed in Huh7 cells (1.08±0.06 fold, p = 0.10, n = 8), ([Figure 1D](#pgen-1003058-g001)). No significant changes in secreted levels of APOA1 were observed in either hepatoma cell line. Reduced *HMGCR*, *LDLR*, and *SREBF2* transcript levels together with increased APOB secretion with RHOA knock-down are all consistent with higher intracellular cholesterol levels, which was documented in the case of cholesterol esters (1.56±0.18 fold vs. controls, p = 0.004, n = 16), ([Figure 1E](#pgen-1003058-g001)). Although we also detected a trend for elevated free cholesterol after knock-down, this was not statistically significant. A trend of increased intracellular cholesterol ester and free cholesterol was also observed in Huh7 cells after *RHOA* knock-down (1.15±0.14 fold, p = 0.05 and 1.06±0.15 fold, p = 0.27, n = 8).

Lastly, since many genes involved in the maintenance of intracellular cholesterol are transcriptionally regulated in response to changes in intracellular sterol content through SREBF2, a transcription factor, we sought to test if *RHOA* was also subject to SREBF2 regulation. Sterol depletion activates SREBF2, thus stimulating expression of SREBF2 target genes. We confirmed that *RHOA* mRNA and protein levels were substantially increased by extreme sterol depletion in HepG2 cells with 2 µM simvastatin +10% lipoprotein deficient serum for 24 hr ([Figure 1F and 1G](#pgen-1003058-g001)). Induction of *HMGCR* mRNA and protein levels served as a positive control for the effects of cholesterol depletion. Finally, we found small but statistically significant reductions in *RHOA* transcript levels after *SREBF1* knock-down in HepG2 (0.83±0.07 fold, p = 0.05) and Huh7 (0.86±0.02 fold, p = 0.001) cell lines ([Figure 1H](#pgen-1003058-g001)).

### Correlation of RHOA transcript levels with cellular and clinical measures of statin response

Although statin-induced changes of *RHOA* and *LDLR* mRNA were positively correlated in the LCL panel ([Figure 2A](#pgen-1003058-g002)), change of the *RHOA* transcript was inversely correlated with level of LDLR cell surface protein ([Figure 2B](#pgen-1003058-g002)). Consistent with this relationship, we also identified an inverse correlation of *RHOA* transcript levels in statin- treated CAP LCLs with absolute changes in plasma total cholesterol (p = 0.02, r^2^ = 0.01), LDL cholesterol (p = 0.04, r^2^ = 0.01) and APOB (p = 0.007, r^2^ = 0.01), measured *in vivo* before and after simvastatin treatment of the individuals from whom these cell lines were derived ([Table S1](#pgen.1003058.s006)). In contrast, levels of *RHOA* in sham-treated LCLs were not significantly correlated with these measures at baseline ([Table S1](#pgen.1003058.s006)). Moreover, *RHOA* transcript levels in statin-treated LCLs were not significantly associated with statin-induced changes in plasma HDL cholesterol levels (data not shown).

### Figure 2. Association of variation in RHOA mRNA expression levels with other cellular phenotypes.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c51b/3499361/31cc5b5f9109/pgen.1003058.g002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3499361_pgen.1003058.g002.jpg)

480 CAP LCLs from Caucasian donors were incubated with 2 µM activated simvastatin or sham buffer for 24 hours, after which genome-wide gene expression was quantified using the Illumina human HT8v3 beadarray, and cell surface LDLR protein was measured by FACS. Greater induction of RHOA transcript levels was positively correlated with greater induction of both LDLR and HMGCR (shown in Figure 1) mRNA (A) and inversely correlated with induction of cell surface LDLR protein (B). Gene expression values and measures of LDLR protein were adjusted for cellular covariates as described in Figure 1, log transformed, the delta calculated as the statin-treated minus the sham-treated value, and quantile normalized.

### A RHOA cis-splicing QTL is associated with in vivo statin response and exhibits allele-specific expression

We next investigated the association of common genetic variation near *RHOA* with *in vivo* statin response. Analysis of HapMap3 CEU data [[13]](#pgen.1003058-Altshuler2) with Haploview [[14]](#pgen.1003058-Barrett1), revealed that *RHOA* fell within a large block of linkage disequilibrium spanning almost 500 kb and that there were four major haplotypes when considering markers within 10 kb of the gene, all with frequencies greater than 10% in the CEU population ([Figure S1](#pgen.1003058.s001); [Table 3](#pgen-1003058-t003)). Haplotypes were inferred based on directly genotyped SNPs ([Table S2](#pgen.1003058.s007)) or imputed genotypes (rs11716445 for H3B), and the number of copies of each haplotype were tested for association with change in LDL-cholesterol (delta log) in response to statin treatment of Caucasian participants in CAP (n = 580) and in the Pravastatin Inflammation CRP Evaluation (PRINCE: pravastatin 40 mg/day, 24 weeks, n = 1306) clinical trial, with adjustment for sex, age, BMI, smoking status, and study population. Of the four haplotypes, H3B showed the strongest association with statin response (p = 0.01), with homozygous H3B carriers having a 29% smaller reduction in the unadjusted percent change of LDL-cholesterol compared to non-carrier controls (−21.8±4.5% versus −30.7±0.4%, [Figure 3](#pgen-1003058-g003)). When the CAP and PRINCE cohorts were analyzed independently, the directionality of this association was consistent between the two populations ([Figure S2](#pgen.1003058.s002)). Haplotype H2 also demonstrated a modest association, with carriers having greater statin-induced changes in LDL-C (p = 0.04, n = 1886, [Figure 3](#pgen-1003058-g003)). There were no significant associations of H3B or H2 carrier status with baseline LDL-cholesterol (p = 0.3 for both).

### Table 3. RHOA haplotype definition.

| Haplotype | rs6446272 | rs1464569 | rs1464566 | rs1464567 | rs4855873 | rs1464568 | rs10640 | rs9814873 | rs6997 | rs6784820 | rs6779524 | rs940045 | rs13096474 | rs2140270 | rs6797765 | rs4955412 | rs7621003 | rs17595772 | rs11716445 | rs974495 | rs3448 | rs3811699 | rs17650792 | rs17080528 | N (CAP) | N (PRINCE) | % |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| H1 | C | C | G | C | G | T | C | T | G | C | G | T | G | T | C | G | G | T | C | G | G | A | C | G | 476 | 1154 | 42.6 |
| H2 | C | T | A | G | T | C | C | T | G | T | G | C | A | C | T | T | A | C | C | A | A | A | T | G | 311 | 671 | 25.6 |
| H3A | T | C | A | G | T | C | T | C | A | T | A | T | G | T | T | T | A | C | C | G | G | G | T | A | 282 | 576 | 22.4 |
| H3B | T | C | A | G | T | C | T | C | A | T | A | T | G | T | T | T | A | C | T | G | G | G | T | A | 105 | 255 | 9.4 |

Table 3 Caption: RHOA haplotypes were defined using common HapMap3 SNPs within 10 kb of the gene, with minor alleles in bold italics and the minor allele of rs11716445, the only RHOA SNP that differs between H3A and H3B, in underlined bold italic. Distributions of the haplotypes in the CAP and PRINCE clinical trial populations are also shown.

### Figure 3. Relationship between RHOA haplotypes H3B and H2 and statin-induced changes in LDL-cholesterol response in the CAP (N = 580) and PRINCE (N = 1306) clinical trials.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c51b/3499361/15ad148d7ddd/pgen.1003058.g003.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3499361_pgen.1003058.g003.jpg)

Linear regression of statin-induced changes (delta log) in LDL-cholesterol with adjustment age, sex, BMI, smoking status, and clinical trial was used to test for association with RHOA haplotypes. Graph depicts mean percent change in LDL-cholesterol +/− SE.

We found no association of either H3B or H2 with *RHOA* transcript levels in CAP LCLs after treatment with 2 µM statin or sham buffer (n = 115) ([Figure S3A](#pgen.1003058.s003)). However, rs11716445, the SNP that defines the H3B haplotype, is located in a rare 45 bp cryptic exon (referred to as *RHOA* exon 2.5) that we identified in multiple unique sequences during RNA-Seq analysis of three human hepatoma cell lines (HepG2, Hep3B and Huh7), and CAP LCLs (n = 3), [Figure 4A](#pgen-1003058-g004). Expression of the *RHOA 2.5* exon was validated by Sanger sequencing. Notably, we found that the H3B haplotype showed a very strong association with *RHOA* exon 2.5 levels under both sham (p = 2.7×10^−7^, n = 119) and statin (p = 9.1×10^−13^, n = 115) conditions, with carriers exhibiting the highest levels of exon 2.5 inclusion ([Figure 4B](#pgen-1003058-g004) and [Figure S3B](#pgen.1003058.s003)). The H2 haplotype also exhibited a more modest association with *RHOA* exon 2.5 levels in the opposite direction from H3B (p<0.01), consistent with their *in vivo* relationships.

### Figure 4. Association between RHOA haplotypes and expression of RHOA exon 2.5.

![Figure 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c51b/3499361/da42b02b62bc/pgen.1003058.g004.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3499361_pgen.1003058.g004.jpg)

(A) RHOA gene structure and location and sequence of exon 2.5. The 5′ and 3′ UTRs are indicated in solid grey, canonical exons are outlined in grey, and the novel exon is shown in red. The location of the three common RHOA exonic SNPs are shown. RNA-Seq libraries were prepared from HepG2, Hep3B, Huh7, and CAP LCLs (n = 3) and sequenced using 100 bp paired-end reads, and the novel exon was discovered by inspecting novel junction spanning reads. (B) Total RHOA expression levels and RHOA exon 2.5 levels were measured using qPCR, the percent containing exon 2.5 were calculated by dividing RHOA exon 2.5 levels by total RHOA levels, adjusted for batch effects using regression, and normal quantile transformed prior to association testing using linear regression. Graphs depict mean +/− SE. (C) CAP LCL cDNAs containing RHOA exon 2.5 were PCR-amplified and Sanger sequenced and the relative amounts of each allele of rs11716445 in exon 2.5 were estimated by measuring the height of the sequencing traces. (D) Representative sequencing traces of the LCL cDNA and genomic DNA of three heterozygous individuals from CAP illustrate that H3B is a cis-acting splicing QTL exhibiting allele-specific expression in RHOA exon 2.5 (rs11716445 and rs2878298) but not exon 5 (rs3448). H3B contains the minor allele [T] of rs11716445, H1 contains the minor allele [G] of rs2878298, and H2 contains the minor allele [A] of rs3448.

Using Sanger sequencing, we found evidence of allele-specific expression (ASE) at rs11716445 with over 90% of the exon 2.5-containing transcripts originating from the H3B chromosome ([Figure 4C and 4D](#pgen-1003058-g004)). We also observed evidence of ASE at rs2878298, another SNP found within exon 2.5 ([Figure 4D](#pgen-1003058-g004) and [Figure S3C](#pgen.1003058.s003)). There was no significant difference in the relative amount of ASE between the statin- and sham-treated states. Finally, to determine if rs11716445 was a general expression quantitative trait locus (eQTL) or a splicing QTL, we tested for ASE at rs3448, a SNP in the 3′ UTR of *RHOA*, in eight heterozygous carriers (or H2/H3B) and found no evidence that the ASE extended beyond exon 2.5 ([Figure 4D](#pgen-1003058-g004) and [Figure S3C](#pgen.1003058.s003)). These results strongly suggest that rs11716445 is a cis-acting splicing QTL.

## Discussion

We here present results of applying a set of biologically meaningful filters to identify and rank candidate genes associated with inter-individual variation in statin effects on cholesterol metabolism based on their gene expression characteristics. Using unbiased genome-wide screens, we identified genes that were normally expressed in the human liver and changed in response to statin treatment in a manner that was correlated with statin-induced change in *HMGCR* quantified as an *in vitro* marker of statin response. From these analyses we identified a number of genes not previously implicated in the lipid-lowering response to statin as potential candidates for future study. We selected *RHOA* since many of the non-lipid lowering benefits, or pleiotropic effects, of statin treatment have been attributed to its ability to inhibit RHOA activity. Our validation of RHOA as a modulator of cellular cholesterol metabolism, as well as the discovery that genetic variation within *RHOA* is associated with the magnitude of LDL-cholesterol response to statin treatment, support the continued studies of other novel candidate genes identified through this integrative genomics strategy.

RHOA has been previously implicated in cholesterol metabolism through the modulation of ABCA1-mediated cholesterol efflux via two distinct and opposing mechanisms. RHOA inhibition stimulates *ABCA1* gene expression via PPARγ and LXR activation [[15]](#pgen.1003058-Argmann1), while RHOA activation increases ABCA1 protein stability [[11]](#pgen.1003058-Okuhira1). Although excess intracellular levels of free cholesterol have been shown to increase RHOA activity [[16]](#pgen.1003058-Corey1), here we demonstrate that *RHOA* knock-down results in increased levels of secreted APOB, suggesting that RHOA may influence the pool of intracellular cholesterol available for lipoprotein production. Consistent with this hypothesis, we found that knock-down of *RHOA* in hepatoma cell lines resulted in increased intracellular content of cholesterol esters, the storage form of cellular cholesterol that can be mobilized for lipoprotein secretion. This occurred in conjunction with reduced expression of *HMGCR* and *LDLR*, presumably due to cholesterol-induced down-regulation of SREBF2. Very recently a novel protein, LAMTOR1 (also known as Pdro/p27RF-Rho), was found to both activate RHOA [[17]](#pgen.1003058-Hoshino1), and to regulate LDL-C uptake and intracellular cholesterol egress from the late endosome/lysosome [[10]](#pgen.1003058-Guillaumot1), further supporting a link between RHOA and cholesterol metabolism.

Additional evidence for such a link is provided by the strong correlations that we observed between statin-induced changes in *RHOA* mRNA levels and both *HMGCR* and *LDLR* transcripts. On the other hand, there was an inverse correlation between change in *RHOA* mRNA and cell surface LDLR protein. While this may appear to be at odds with the change in *LDLR* transcript level, it is consistent with our finding that greater statin-induced *RHOA* gene expression was associated with reduced *in vivo* response lipid response to statin treatment. It is possible that increased RHOA expression directly or indirectly reduces functional LDLR at the cell surface by altering post-translational processing or cellular trafficking, hypotheses that will be tested in future studies. Increased magnitude of this effect may contribute to attenuation of statin-induced plasma LDL lowering.

Based on its role in mediating the pleiotropic effects of statin response, *RHOA* has been proposed as a candidate gene for the study of statin pharmacogenetics; however, genetic variation within *RHOA* associated with statin response has not been previously identified [[18]](#pgen.1003058-Oliyarnyk1). Here, we report that a common *RHOA* haplotype, H3B, is associated with reduced LDL-cholesterol lowering in response to statin treatment in data derived from two independent clinical trials. Within *RHOA*, this haplotype was defined by a single SNP, rs11716445; however, since rs11716445 is in strong linkage disequilibrium with many SNPs in other genes up to 500 kb away from *RHOA*, it is possible that its association with statin response may also be due to genetic variation affecting other genes. rs11716445 explained less than 1% of the overall variation in LDL cholesterol response to statin, so neither the H3B haplotype or rs11716445 genotype alone would be a clinically useful diagnostic, but it could be included with other known markers of statin response to improve prediction algorithms.

Here we demonstrate that rs11716445 is a cis-acting splicing QTL also associated with allele-specific expression of *RHOA* exon 2.5, a rare exon found within *RHOA* intron 2. The presence of this exon does not disrupt the open reading frame and is predicted to cause a 14 amino acid inclusion in the B3 domain of the RHOA protein, a region with no known interactions [[19]](#pgen.1003058-Ihara1), [[20]](#pgen.1003058-Rittinger1). Although the functional impact of exon 2.5 inclusion is unknown, the fact that the two *RHOA* haplotypes associated with its expression levels, H3B and H2, are also the only two *RHOA* haplotypes found to be associated with *in vivo* variation in statin-induced change in LDL-cholesterol, strongly supports the likelihood that *RHOA* alternative splicing is functionally relevant.

*In silico* analysis with ESEfinder 3.0 identified SRSF2 (aka SC35), SRSF5 (aka SRp40), and SRSF1 (aka SF2/ASF) binding sites within 20 bp of the exon 2.5 splice donor [[21]](#pgen.1003058-Smith1). Notably, the rs11716445 “T” allele is predicted to disrupt an SRSF5 binding motif (TAGA[T/C]C) ([Figure S4](#pgen.1003058.s004)). This finding is consistent with previous reports demonstrating that SRSF5 and SRSF2 antagonize SRSF1 to promote exon exclusion, as the loss of the SRSF5 binding with the “T” allele would be predicted to result in exon 2.5 inclusion [[22]](#pgen.1003058-Chandradas1). Thus, these results strongly suggest that the rs11716445 “T” (minor) allele enhances expression of the *RHOA* 2.5 exon. We also found that the proportion of the expressed *RHOA* 2.5 exon containing the “T” allele was reduced in H1/H3B compared to H2/H3B and H3A/H3B heterozygotes ([Figure S3C](#pgen.1003058.s003)). Since the H1 haplotype contains the minor allele of the second common SNP within the 2.5 exon, rs2878298, which is predicted to generate a SRSF1 binding site, these findings suggest that there are multiple gene variants that regulate expression of this novel exon; however the functional effects of these SNPs (rs11716445 and rs2878298) as well as the expression of the cryptic *RHOA* exon remain to be tested.

In summary, we here report using a combination of expression array data, functional studies, and genetic analyses that *RHOA* is a novel candidate gene associated with variation in both *in vitro* and *in vivo* response to statin. Although additional studies of statin effects will be required to corroborate these findings, they demonstrate the value of using data from a variety of molecular techniques, including the combination of *in vivo* and *in vitro* genetically-regulated phenotypes, as a novel approach for identifying genes involved in drug response.

## Materials and Methods

### Cell culture

Lymphoblastoid cell lines (LCLs) from 480 Caucasian participants from the Cholesterol and Pharmacogeneomics (CAP) clinical trial [[9]](#pgen.1003058-Simon1) and HepG2 and Hep3B cell lines were grown under standard conditions and exposed to 2 µM simvastatin or sham buffer for 24 hours as previously described [[23]](#pgen.1003058-Medina1). Although much higher than normal circulating levels of plasma simvastatin, 2–40 nM [[24]](#pgen.1003058-Mason1), this concentration of simvastatin was selected based on previous dose response experiments that were used to determine the amount that elicits a consistent and significant induction of both *HMGCR* and *LDLR* mRNA ([Figure S5](#pgen.1003058.s005)) [[23]](#pgen.1003058-Medina1). Simvastatin was provided by Merck Inc. (Whitehouse Station, NJ) and activated to the β-hydroxyacid prior to use [[25]](#pgen.1003058-Brown2). Cell surface LDLR protein was measured in statin and sham treated CAP LCLs as previously described [[26]](#pgen.1003058-Mangravite1). To confirm statin regulation, HepG2 cells were grown in 6-well plates and incubated with 2 µM activated simvastatin +10% lipoprotein deficient serum (Hyclone) for 24 hours.

### Transcript quantification

Genome-wide gene expression was measured in RNA from CAP samples and statin and sham treated HepG2 and Hep3B cells. RNA was converted to biotin-labeled cRNA using the Illumina TotalPrep-96 RNA amplification kit (Applied Biosystems, Foster City, CA). cRNA was hybridized to Illumina HumanRef8v3 expression beadchips (Illumina, San Diego, CA). Data were analyzed using GenomeStudio (Illumina). All beadchips had a signal P95/P05>10. Significance analysis of microarrays (SAM) [[27]](#pgen.1003058-Tusher1) was performed on the 10,291 of 18,630 probed genes that were expressed in LCLs (FDR<0.05). Expression traits were adjusted for known covariates (age, gender, exposure batch, cell growth rate as determined by cell count on exposure day, and RNA labeling batch) and also for unknown sources of variation through adjustment for those principal components that described greater than 5% variance across the dataset [[28]](#pgen.1003058-Leek1). Adjusted data were quantile normalized across each gene to ensure normality.

Gene expression in human liver was determined using mean detection p-value as determined by GenomeStudio (Illumina, San Diego, CA) from expression profiles measured by Illumina Ref8v2 beadarray on 120 human liver samples (2 technical replicates each of 60 samples, GEO accession number: [GSE28893](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE28893) [[29]](#pgen.1003058-Innocenti1)). Mean detection p-values across all 120 samples was assessed, and genes with a p<0.05 were called expressed.

*HMGCR, LDLR*, *SREBF1, SREBF2, RHOA* (total), and *RHOAexon2.5* transcript levels were quantified by qPCR with gene expression normalized to *CLPTM* (TaqMan Assay number: Hs00171300_m1, Life Technologies) as previously described [[30]](#pgen.1003058-Medina2). Primers used for qPCR of total *RHOA* were F: *CGGAATGATGAGCACACAAG* and R: *TGCCTTCTTCAGGTTTCACC* and those used for qPCR of *RHOA* exon 2.5 were F: *TATCGAGGTGGATGGAAAGC* and R: *GCCAACTCTACCATAGTACATTGAAA*.

### siRNA transfections

*RHOA*, *SREBF1* and *SREBF2* knock-down was achieved by 48 hour transfection of 80,000 HepG2 or Huh7 cells/well in 12-well plates using either the Ambion Silence Select siRNA (s759) or non-targeting control according to the manufacturer's protocol. Cell culture media was collected from all samples at time of harvest, and APOB and APOAI were quantified in triplicate by sandwich-style ELISA. Samples with a coefficient of variation greater than 15% were subject to repeat measurement. Cholesterol was extracted from the cell pellets with hexane-isopropanol (3∶2, v/v) and dried under nitrogen. The extracted cholesterol was reconstituted with reaction buffer (0.5 M potassium phosphate, pH 7.4, 0.25 M NaCl, 25 mM cholic acid, 0.5% Triton X-100). Total cholesterol content was determined with the Amplex Red Cholesterol Assay Kit (Invitrogen) and normalized to total cellular protein quantified by the Pierce BCA Protein Assay Kit (Thermo Scientific). To quantify RHOA protein levels, cells were lysed in Cell Lytic lysis buffer (Sigma), loaded on a 4–12% Tris-Glycine Gel (Invitrogen), and proteins were transferred onto a PDVF membrane using the iBLOT gel transfer system (Invitrogen). The blot was then probed with antibodies diluted 1∶200 to RHOA (SC26C4), HMGCR (SCH300) and β-actin (SC ACTBD11B7), all purchased from Santa Cruz Biotechnology. Band densities were analyzed using the Mulitplex Band Analysis tool in Alphaview SA version 3.4.0.

### Imputation and haplotype reconstruction

Haplotypes H1, H2, and H3A were assigned using genotype data from tag SNPs ([Table S2](#pgen.1003058.s007)), while haplotype H3B was inferred using imputed rs11716445 genotypes. Imputation was performed in BIMBAM using 317K or 610K genotypes in a similar manner as previously described [[31]](#pgen.1003058-Barber1) except for use of the HapMap3 and 1KGP CEU pilot data as a reference population.

### In vivo association analysis

LDL-cholesterol was quantified in self-reported Caucasian American participants of the Cholesterol and Pharmacogenetics (CAP) clinical trial twice at baseline and after both 4 weeks and 6 weeks of simvastatin 40 mg/day and in the participants of the Pravastatin Inflammation and CRP Evaluation (PRINCE) clinical trial after 12 and 24 weeks of pravastatin 20 mg/day as previously described [[9]](#pgen.1003058-Simon1), [[32]](#pgen.1003058-Chasman2). Delta log LDL-cholesterol was calculated as the log average value of LDL-cholesterol on treatment minus the log average of the two baseline measurements, and percent change was the average on-statin value minus the average baseline value over the average baseline value. The CAP trial is registered at ClinicalTrials.gov ([NCT00451828](https://clinicaltrials.gov/ct2/show/NCT00451828)). Informed consent was obtained and approved by the institutional review boards of the sites of recruitment, University of California Los Angeles and San Francisco General Hospital. In addition, all research involving human participants was approved by the Children's Hospital Oakland Research Institute IRB. All haplotypes with a minor allele frequency greater than 5% were identified using Haploview [[14]](#pgen.1003058-Barrett1) with HapMap3 CEU data. Using an additive genetic model, haplotypes were tested for association with change in delta log LDL-cholesterol using combined results of both clinical trials with adjustment for age, sex, BMI, smoking status, and study population as well as for each trial separately with adjustment for age, sex, BMI, and smoking status.

### RNA–seq analysis

Hep3B, HepG2, and Huh7 cells were incubated in duplicate under either standard growth conditions (MEM supplemented with 10% FBS, 1% nonessential amino acids and 1% sodium pyruvate) or sterol depleted conditions (MEM supplemented with 1% nonessential amino acids, 1% sodium pyruvate, 2.0 µM simvastatin and 10% lipoprotein deficient serum) for 24 hours. RNA was extracted as previously described and samples from the duplicate experiments were pooled. Sequencing libraries were prepared by isolating mRNA from 7–10 µg total RNA using two rounds of the MicroPoly(A)Purist kit (Ambion), fragmenting the mRNA for 20 seconds, synthesizing cDNA using random primers, repairing ends, dA-tailing, ligating adapters, gel purifying fragments, amplifying libraries using indexed primers for 15 PCR cycles, and performing another round of gel purification. Libraries were sequenced to an average depth of 60 million 100 bp reads (30 million paired-end fragments). Expression of the novel *RHOA* exon was verified in independent samples through RT-PCR and Sanger sequencing.

### Sanger sequencing

DNA and RNA was isolated from CAP LCLs after 24 hours of exposure to sham buffer or 2 µM simvastatin. The DNA sequences of exon 2.5 and exon 5 were amplified using F: *CAAGGCAGGAGAATGGTGTG* and R: *CCACTGACGATGATTGCTTC* and F: *GGCCATATTACCCCTTTTCG* and R: *CCAGAGGGATCTAGGCTTCC*, respectively. RT-PCR was performed to amplify the transcript sequences of exon 2.5 and exon 5 (3′UTR) using F: *TCGTTAGTCCACGGTCTGGT* and R: *GCCAACTCTACCATAGTACATTGAAA* and F: *CGGAATGATGAGCACACAAG* and R: *TTGGAAAAATTAACTGGTACAGAAA*, respectively. PCR products were then subject to Sanger sequencing.

## Supporting Information

## Acknowledgments

The CAP LCLs for these studies were provided by Sheila Pressman and Dr. Jerome I. Rotter of Cedars-Sinai Medical Center. We would like to thank Vani Nimbal for her help in cell culture, Amir Mohamed and Mariya Kutilova for their assistance in the measurement of LDLR protein, Arnie Acosta for quantifying APOB and APOA1, and Caitlin Halitner for validation of the *RHOA* 2.5 exon.

## Funding Statement

This work was supported by NHLBI R01 HL104133-01 and NHLBI U19 HL69757-10. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Altshuler D, Daly MJ, Lander ES (2008) Genetic mapping in human disease. Science 322: 881–888.  [DOI](https://doi.org/10.1126/science.1156409) | [PMC free article](/articles/PMC2694957/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18988837/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Genetic%20mapping%20in%20human%20disease&author=D%20Altshuler&author=MJ%20Daly&author=ES%20Lander&volume=322&publication_year=2008&pages=881-888&pmid=18988837&doi=10.1126/science.1156409&)

2. Amos CI (2007) Successful design and conduct of genome-wide association studies. Hum Mol Genet 16 Spec No. 2: R220–225.  [DOI](https://doi.org/10.1093/hmg/ddm161) | [PMC free article](/articles/PMC2691963/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17597095/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=Successful%20design%20and%20conduct%20of%20genome-wide%20association%20studies&author=CI%20Amos&volume=16%20Spec%20No.%202&publication_year=2007&pages=R220-225&pmid=17597095&doi=10.1093/hmg/ddm161&)

3. Thompson JF, Hyde CL, Wood LS, Paciga SA, Hinds DA, et al. (2009) Comprehensive Whole-Genome and Candidate Gene Analysis for Response to Statin Therapy in the Treating to New Targets (TNT) Cohort. Circulation Cardiovascular Genetics 2: 173–181.  [DOI](https://doi.org/10.1161/CIRCGENETICS.108.818062) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20031582/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation%20Cardiovascular%20Genetics&title=Comprehensive%20Whole-Genome%20and%20Candidate%20Gene%20Analysis%20for%20Response%20to%20Statin%20Therapy%20in%20the%20Treating%20to%20New%20Targets%20(TNT)%20Cohort&author=JF%20Thompson&author=CL%20Hyde&author=LS%20Wood&author=SA%20Paciga&author=DA%20Hinds&volume=2&publication_year=2009&pages=173-181&pmid=20031582&doi=10.1161/CIRCGENETICS.108.818062&)

4. Tanaka Y, Nishida N, Sugiyama M, Kurosaki M, Matsuura K, et al. (2009) Genome-wide association of IL28B with response to pegylated interferon-alpha and ribavirin therapy for chronic hepatitis C. Nat Genet 41: 1105–1109.  [DOI](https://doi.org/10.1038/ng.449) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19749757/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Genome-wide%20association%20of%20IL28B%20with%20response%20to%20pegylated%20interferon-alpha%20and%20ribavirin%20therapy%20for%20chronic%20hepatitis%20C&author=Y%20Tanaka&author=N%20Nishida&author=M%20Sugiyama&author=M%20Kurosaki&author=K%20Matsuura&volume=41&publication_year=2009&pages=1105-1109&pmid=19749757&doi=10.1038/ng.449&)

5. Chasman DI, Giulianini F, Macfadyen J, Barratt BJ, Nyberg F, et al. (2012) Genetic Determinants of Statin-Induced Low-Density Lipoprotein Cholesterol Reduction: The Justification for the Use of Statins in Prevention: An Intervention Trial Evaluating Rosuvastatin (JUPITER) Trial. Circ Cardiovasc Genet 5: 257–264.  [DOI](https://doi.org/10.1161/CIRCGENETICS.111.961144) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22331829/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circ%20Cardiovasc%20Genet&title=Genetic%20Determinants%20of%20Statin-Induced%20Low-Density%20Lipoprotein%20Cholesterol%20Reduction:%20The%20Justification%20for%20the%20Use%20of%20Statins%20in%20Prevention:%20An%20Intervention%20Trial%20Evaluating%20Rosuvastatin%20(JUPITER)%20Trial&author=DI%20Chasman&author=F%20Giulianini&author=J%20Macfadyen&author=BJ%20Barratt&author=F%20Nyberg&volume=5&publication_year=2012&pages=257-264&pmid=22331829&doi=10.1161/CIRCGENETICS.111.961144&)

6. Goldstein JL, Brown MS (1990) Regulation of the mevalonate pathway. Nature 343: 425–430.  [DOI](https://doi.org/10.1038/343425a0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1967820/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Regulation%20of%20the%20mevalonate%20pathway&author=JL%20Goldstein&author=MS%20Brown&volume=343&publication_year=1990&pages=425-430&pmid=1967820&doi=10.1038/343425a0&)

7. Goldstein JL, Brown MS, Anderson RG, Russell DW, Schneider WJ (1985) Receptor-mediated endocytosis: concepts emerging from the LDL receptor system. Annu Rev Cell Biol 1: 1–39.  [DOI](https://doi.org/10.1146/annurev.cb.01.110185.000245) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2881559/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annu%20Rev%20Cell%20Biol&title=Receptor-mediated%20endocytosis:%20concepts%20emerging%20from%20the%20LDL%20receptor%20system&author=JL%20Goldstein&author=MS%20Brown&author=RG%20Anderson&author=DW%20Russell&author=WJ%20Schneider&volume=1&publication_year=1985&pages=1-39&pmid=2881559&doi=10.1146/annurev.cb.01.110185.000245&)

8. Brown MS, Goldstein JL (1985) The LDL receptor and HMG-CoA reductase–two membrane molecules that regulate cholesterol homeostasis. Curr Top Cell Regul 26: 3–15.  [DOI](https://doi.org/10.1016/b978-0-12-152826-3.50008-5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/4075823/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Top%20Cell%20Regul&title=The%20LDL%20receptor%20and%20HMG-CoA%20reductase%E2%80%93two%20membrane%20molecules%20that%20regulate%20cholesterol%20homeostasis&author=MS%20Brown&author=JL%20Goldstein&volume=26&publication_year=1985&pages=3-15&pmid=4075823&doi=10.1016/b978-0-12-152826-3.50008-5&)

9. Simon JA, Lin F, Hulley SB, Blanche PJ, Waters D, et al. (2006) Phenotypic predictors of response to simvastatin therapy among African-Americans and Caucasians: the Cholesterol and Pharmacogenetics (CAP) Study. Am J Cardiol 97: 843–850.  [DOI](https://doi.org/10.1016/j.amjcard.2005.09.134) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16516587/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Cardiol&title=Phenotypic%20predictors%20of%20response%20to%20simvastatin%20therapy%20among%20African-Americans%20and%20Caucasians:%20the%20Cholesterol%20and%20Pharmacogenetics%20(CAP)%20Study&author=JA%20Simon&author=F%20Lin&author=SB%20Hulley&author=PJ%20Blanche&author=D%20Waters&volume=97&publication_year=2006&pages=843-850&pmid=16516587&doi=10.1016/j.amjcard.2005.09.134&)

10. Guillaumot P, Luquain C, Malek M, Huber AL, Brugiere S, et al. (2010) Pdro, a protein associated with late endosomes and lysosomes and implicated in cellular cholesterol homeostasis. PLoS ONE 5: e10977 doi:10.1371/journal.pone.0010977  [DOI](http://dx.doi.org/10.1371/journal.pone.0010977) | [DOI](https://doi.org/10.1371/journal.pone.0010977) | [PMC free article](/articles/PMC2882324/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20544018/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&title=Pdro,%20a%20protein%20associated%20with%20late%20endosomes%20and%20lysosomes%20and%20implicated%20in%20cellular%20cholesterol%20homeostasis&author=P%20Guillaumot&author=C%20Luquain&author=M%20Malek&author=AL%20Huber&author=S%20Brugiere&volume=5&publication_year=2010&pages=e10977&pmid=20544018&doi=10.1371/journal.pone.0010977&)

11. Okuhira K, Fitzgerald ML, Tamehiro N, Ohoka N, Suzuki K, et al. (2010) Binding of PDZ-RhoGEF to ATP-binding cassette transporter A1 (ABCA1) induces cholesterol efflux through RhoA activation and prevention of transporter degradation. J Biol Chem 285: 16369–16377.  [DOI](https://doi.org/10.1074/jbc.M109.061424) | [PMC free article](/articles/PMC2871504/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20348106/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Binding%20of%20PDZ-RhoGEF%20to%20ATP-binding%20cassette%20transporter%20A1%20(ABCA1)%20induces%20cholesterol%20efflux%20through%20RhoA%20activation%20and%20prevention%20of%20transporter%20degradation&author=K%20Okuhira&author=ML%20Fitzgerald&author=N%20Tamehiro&author=N%20Ohoka&author=K%20Suzuki&volume=285&publication_year=2010&pages=16369-16377&pmid=20348106&doi=10.1074/jbc.M109.061424&)

12. Bartz F, Kern L, Erz D, Zhu M, Gilbert D, et al. (2009) Identification of cholesterol-regulating genes by targeted RNAi screening. Cell Metab 10: 63–75.  [DOI](https://doi.org/10.1016/j.cmet.2009.05.009) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19583955/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Metab&title=Identification%20of%20cholesterol-regulating%20genes%20by%20targeted%20RNAi%20screening&author=F%20Bartz&author=L%20Kern&author=D%20Erz&author=M%20Zhu&author=D%20Gilbert&volume=10&publication_year=2009&pages=63-75&pmid=19583955&doi=10.1016/j.cmet.2009.05.009&)

13. Altshuler DM, Gibbs RA, Peltonen L, Dermitzakis E, Schaffner SF, et al. (2010) Integrating common and rare genetic variation in diverse human populations. Nature 467: 52–58.  [DOI](https://doi.org/10.1038/nature09298) | [PMC free article](/articles/PMC3173859/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20811451/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Integrating%20common%20and%20rare%20genetic%20variation%20in%20diverse%20human%20populations&author=DM%20Altshuler&author=RA%20Gibbs&author=L%20Peltonen&author=E%20Dermitzakis&author=SF%20Schaffner&volume=467&publication_year=2010&pages=52-58&pmid=20811451&doi=10.1038/nature09298&)

14. Barrett JC, Fry B, Maller J, Daly MJ (2005) Haploview: analysis and visualization of LD and haplotype maps. Bioinformatics 21: 263–265.  [DOI](https://doi.org/10.1093/bioinformatics/bth457) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15297300/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=Haploview:%20analysis%20and%20visualization%20of%20LD%20and%20haplotype%20maps&author=JC%20Barrett&author=B%20Fry&author=J%20Maller&author=MJ%20Daly&volume=21&publication_year=2005&pages=263-265&pmid=15297300&doi=10.1093/bioinformatics/bth457&)

15. Argmann CA, Edwards JY, Sawyez CG, O'Neil CH, Hegele RA, et al. (2005) Regulation of macrophage cholesterol efflux through hydroxymethylglutaryl-CoA reductase inhibition: a role for RhoA in ABCA1-mediated cholesterol efflux. J Biol Chem 280: 22212–22221.  [DOI](https://doi.org/10.1074/jbc.M502761200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15817453/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Regulation%20of%20macrophage%20cholesterol%20efflux%20through%20hydroxymethylglutaryl-CoA%20reductase%20inhibition:%20a%20role%20for%20RhoA%20in%20ABCA1-mediated%20cholesterol%20efflux&author=CA%20Argmann&author=JY%20Edwards&author=CG%20Sawyez&author=CH%20O'Neil&author=RA%20Hegele&volume=280&publication_year=2005&pages=22212-22221&pmid=15817453&doi=10.1074/jbc.M502761200&)

16. Corey DA, Kelley TJ (2007) Elevated small GTPase activation influences the cell proliferation signaling control in Niemann-Pick type C fibroblasts. Biochim Biophys Acta 1772: 748–754.  [DOI](https://doi.org/10.1016/j.bbadis.2007.03.008) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17499483/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochim%20Biophys%20Acta&title=Elevated%20small%20GTPase%20activation%20influences%20the%20cell%20proliferation%20signaling%20control%20in%20Niemann-Pick%20type%20C%20fibroblasts&author=DA%20Corey&author=TJ%20Kelley&volume=1772&publication_year=2007&pages=748-754&pmid=17499483&doi=10.1016/j.bbadis.2007.03.008&)

17. Hoshino D, Tomari T, Nagano M, Koshikawa N, Seiki M (2009) A novel protein associated with membrane-type 1 matrix metalloproteinase binds p27(kip1) and regulates RhoA activation, actin remodeling, and matrigel invasion. J Biol Chem 284: 27315–27326.  [DOI](https://doi.org/10.1074/jbc.M109.041400) | [PMC free article](/articles/PMC2785659/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19654316/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=A%20novel%20protein%20associated%20with%20membrane-type%201%20matrix%20metalloproteinase%20binds%20p27(kip1)%20and%20regulates%20RhoA%20activation,%20actin%20remodeling,%20and%20matrigel%20invasion&author=D%20Hoshino&author=T%20Tomari&author=M%20Nagano&author=N%20Koshikawa&author=M%20Seiki&volume=284&publication_year=2009&pages=27315-27326&pmid=19654316&doi=10.1074/jbc.M109.041400&)

18. Oliyarnyk O, Renner W, Paulweber B, Wascher TC (2005) Interindividual differences of response to statin treatment cannot be explained by variations of the human gene for RhoA. Biochem Genet 43: 143–148.  [DOI](https://doi.org/10.1007/s10528-005-1507-0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15932062/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20Genet&title=Interindividual%20differences%20of%20response%20to%20statin%20treatment%20cannot%20be%20explained%20by%20variations%20of%20the%20human%20gene%20for%20RhoA&author=O%20Oliyarnyk&author=W%20Renner&author=B%20Paulweber&author=TC%20Wascher&volume=43&publication_year=2005&pages=143-148&pmid=15932062&doi=10.1007/s10528-005-1507-0&)

19. Ihara K, Muraguchi S, Kato M, Shimizu T, Shirakawa M, et al. (1998) Crystal structure of human RhoA in a dominantly active form complexed with a GTP analogue. J Biol Chem 273: 9656–9666.  [DOI](https://doi.org/10.1074/jbc.273.16.9656) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9545299/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Crystal%20structure%20of%20human%20RhoA%20in%20a%20dominantly%20active%20form%20complexed%20with%20a%20GTP%20analogue&author=K%20Ihara&author=S%20Muraguchi&author=M%20Kato&author=T%20Shimizu&author=M%20Shirakawa&volume=273&publication_year=1998&pages=9656-9666&pmid=9545299&doi=10.1074/jbc.273.16.9656&)

20. Rittinger K, Walker PA, Eccleston JF, Smerdon SJ, Gamblin SJ (1997) Structure at 1.65 A of RhoA and its GTPase-activating protein in complex with a transition-state analogue. Nature 389: 758–762.  [DOI](https://doi.org/10.1038/39651) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9338791/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Structure%20at%201.65%20A%20of%20RhoA%20and%20its%20GTPase-activating%20protein%20in%20complex%20with%20a%20transition-state%20analogue&author=K%20Rittinger&author=PA%20Walker&author=JF%20Eccleston&author=SJ%20Smerdon&author=SJ%20Gamblin&volume=389&publication_year=1997&pages=758-762&pmid=9338791&doi=10.1038/39651&)

21. Smith PJ, Zhang C, Wang J, Chew SL, Zhang MQ, et al. (2006) An increased specificity score matrix for the prediction of SF2/ASF-specific exonic splicing enhancers. Hum Mol Genet 15: 2490–2508.  [DOI](https://doi.org/10.1093/hmg/ddl171) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16825284/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=An%20increased%20specificity%20score%20matrix%20for%20the%20prediction%20of%20SF2/ASF-specific%20exonic%20splicing%20enhancers&author=PJ%20Smith&author=C%20Zhang&author=J%20Wang&author=SL%20Chew&author=MQ%20Zhang&volume=15&publication_year=2006&pages=2490-2508&pmid=16825284&doi=10.1093/hmg/ddl171&)

22. Chandradas S, Deikus G, Tardos JG, Bogdanov VY (2010) Antagonistic roles of four SR proteins in the biosynthesis of alternatively spliced tissue factor transcripts in monocytic cells. J Leukoc Biol 87: 147–152.  [DOI](https://doi.org/10.1189/jlb.0409252) | [PMC free article](/articles/PMC3210563/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19843576/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Leukoc%20Biol&title=Antagonistic%20roles%20of%20four%20SR%20proteins%20in%20the%20biosynthesis%20of%20alternatively%20spliced%20tissue%20factor%20transcripts%20in%20monocytic%20cells&author=S%20Chandradas&author=G%20Deikus&author=JG%20Tardos&author=VY%20Bogdanov&volume=87&publication_year=2010&pages=147-152&pmid=19843576&doi=10.1189/jlb.0409252&)

23. Medina MW, Gao F, Ruan W, Rotter JI, Krauss RM (2008) Alternative splicing of 3-hydroxy-3-methylglutaryl coenzyme A reductase is associated with plasma low-density lipoprotein cholesterol response to simvastatin. Circulation 118: 355–362.  [DOI](https://doi.org/10.1161/CIRCULATIONAHA.108.773267) | [PMC free article](/articles/PMC2720070/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18559695/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation&title=Alternative%20splicing%20of%203-hydroxy-3-methylglutaryl%20coenzyme%20A%20reductase%20is%20associated%20with%20plasma%20low-density%20lipoprotein%20cholesterol%20response%20to%20simvastatin&author=MW%20Medina&author=F%20Gao&author=W%20Ruan&author=JI%20Rotter&author=RM%20Krauss&volume=118&publication_year=2008&pages=355-362&pmid=18559695&doi=10.1161/CIRCULATIONAHA.108.773267&)

24. Mason RP, Walter MF, Day CA, Jacob RF (2005) Intermolecular differences of 3-hydroxy-3-methylglutaryl coenzyme a reductase inhibitors contribute to distinct pharmacologic and pleiotropic actions. Am J Cardiol 96: 11F–23F.  [DOI](https://doi.org/10.1016/j.amjcard.2005.06.008) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16126019/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Cardiol&title=Intermolecular%20differences%20of%203-hydroxy-3-methylglutaryl%20coenzyme%20a%20reductase%20inhibitors%20contribute%20to%20distinct%20pharmacologic%20and%20pleiotropic%20actions&author=RP%20Mason&author=MF%20Walter&author=CA%20Day&author=RF%20Jacob&volume=96&publication_year=2005&pages=11F-23F&pmid=16126019&doi=10.1016/j.amjcard.2005.06.008&)

25. Brown MS, Faust JR, Goldstein JL, Kaneko I, Endo A (1978) Induction of 3-hydroxy-3-methylglutaryl coenzyme A reductase activity in human fibroblasts incubated with compactin (ML-236B), a competitive inhibitor of the reductase. J Biol Chem 253: 1121–1128.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/624722/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Induction%20of%203-hydroxy-3-methylglutaryl%20coenzyme%20A%20reductase%20activity%20in%20human%20fibroblasts%20incubated%20with%20compactin%20(ML-236B),%20a%20competitive%20inhibitor%20of%20the%20reductase&author=MS%20Brown&author=JR%20Faust&author=JL%20Goldstein&author=I%20Kaneko&author=A%20Endo&volume=253&publication_year=1978&pages=1121-1128&pmid=624722&)

26. Mangravite LM, Medina MW, Cui J, Pressman S, Smith JD, et al. Combined Influence of LDLR and HMGCR Sequence Variation on Lipid-Lowering Response to Simvastatin. Arterioscler Thromb Vasc Biol 30 7: 1485–92.  [DOI](https://doi.org/10.1161/ATVBAHA.110.203273) | [PMC free article](/articles/PMC2909117/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20413733/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arterioscler%20Thromb%20Vasc%20Biol&title=Combined%20Influence%20of%20LDLR%20and%20HMGCR%20Sequence%20Variation%20on%20Lipid-Lowering%20Response%20to%20Simvastatin&author=LM%20Mangravite&author=MW%20Medina&author=J%20Cui&author=S%20Pressman&author=JD%20Smith&volume=30&issue=7&pages=1485-92&pmid=20413733&doi=10.1161/ATVBAHA.110.203273&)

27. Tusher VG, Tibshirani R, Chu G (2001) Significance analysis of microarrays applied to the ionizing radiation response. Proc Natl Acad Sci U S A 98: 5116–5121.  [DOI](https://doi.org/10.1073/pnas.091062498) | [PMC free article](/articles/PMC33173/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11309499/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&title=Significance%20analysis%20of%20microarrays%20applied%20to%20the%20ionizing%20radiation%20response&author=VG%20Tusher&author=R%20Tibshirani&author=G%20Chu&volume=98&publication_year=2001&pages=5116-5121&pmid=11309499&doi=10.1073/pnas.091062498&)

28. Leek JT, Storey JD (2008) A general framework for multiple testing dependence. Proc Natl Acad Sci U S A 105: 18718–18723.  [DOI](https://doi.org/10.1073/pnas.0808709105) | [PMC free article](/articles/PMC2586646/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19033188/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&title=A%20general%20framework%20for%20multiple%20testing%20dependence&author=JT%20Leek&author=JD%20Storey&volume=105&publication_year=2008&pages=18718-18723&pmid=19033188&doi=10.1073/pnas.0808709105&)

29. Innocenti F, Cooper GM, Stanaway IB, Gamazon ER, Smith JD, et al. (2011) Identification, replication, and functional fine-mapping of expression quantitative trait loci in primary human liver tissue. PLoS Genet 7: e1002078 doi:10.1371/journal.pgen.1002078  [DOI](http://dx.doi.org/10.1371/journal.pgen.1002078) | [DOI](https://doi.org/10.1371/journal.pgen.1002078) | [PMC free article](/articles/PMC3102751/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21637794/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet&title=Identification,%20replication,%20and%20functional%20fine-mapping%20of%20expression%20quantitative%20trait%20loci%20in%20primary%20human%20liver%20tissue&author=F%20Innocenti&author=GM%20Cooper&author=IB%20Stanaway&author=ER%20Gamazon&author=JD%20Smith&volume=7&publication_year=2011&pages=e1002078&pmid=21637794&doi=10.1371/journal.pgen.1002078&)

30. Medina MW, Gao F, Naidoo D, Rudel LL, Temel RE, et al. (2011) Coordinately regulated alternative splicing of genes involved in cholesterol biosynthesis and uptake. PLoS ONE 6: e19420 doi/10.1371/journal.pone.0019420.  [DOI](https://doi.org/10.1371/journal.pone.0019420) | [PMC free article](/articles/PMC3084847/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21559365/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&title=Coordinately%20regulated%20alternative%20splicing%20of%20genes%20involved%20in%20cholesterol%20biosynthesis%20and%20uptake&author=MW%20Medina&author=F%20Gao&author=D%20Naidoo&author=LL%20Rudel&author=RE%20Temel&volume=6&publication_year=2011&pages=e19420&pmid=21559365&doi=10.1371/journal.pone.0019420&)

31. Barber M, Mangravite L, Hyde CL, Chasman DI, Smith JD, et al. (2010) Genome-wide association of lipid-lowering response to statins in combined study populations. PLoS ONE 5: e9763 doi:10.1371/journal.pone.0009763  [DOI](http://dx.doi.org/10.1371/journal.pone.0009763) | [DOI](https://doi.org/10.1371/journal.pone.0009763) | [PMC free article](/articles/PMC2842298/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20339536/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&title=Genome-wide%20association%20of%20lipid-lowering%20response%20to%20statins%20in%20combined%20study%20populations&author=M%20Barber&author=L%20Mangravite&author=CL%20Hyde&author=DI%20Chasman&author=JD%20Smith&volume=5&publication_year=2010&pages=e9763&pmid=20339536&doi=10.1371/journal.pone.0009763&)

32. Chasman DI, Posada D, Subrahmanyan L, Cook NR, Stanton VP Jr, et al. (2004) Pharmacogenetic study of statin therapy and cholesterol reduction. Jama 291: 2821–2827.  [DOI](https://doi.org/10.1001/jama.291.23.2821) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15199031/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Jama&title=Pharmacogenetic%20study%20of%20statin%20therapy%20and%20cholesterol%20reduction&author=DI%20Chasman&author=D%20Posada&author=L%20Subrahmanyan&author=NR%20Cook&author=VP%20Stanton&volume=291&publication_year=2004&pages=2821-2827&pmid=15199031&doi=10.1001/jama.291.23.2821&)

33. Johnson AD, Handsaker RE, Pulit SL, Nizzari MM, O'Donnell CJ, et al. (2008) SNAP: a web-based tool for identification and annotation of proxy SNPs using HapMap. Bioinformatics 24: 2938–2939.  [DOI](https://doi.org/10.1093/bioinformatics/btn564) | [PMC free article](/articles/PMC2720775/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18974171/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=SNAP:%20a%20web-based%20tool%20for%20identification%20and%20annotation%20of%20proxy%20SNPs%20using%20HapMap&author=AD%20Johnson&author=RE%20Handsaker&author=SL%20Pulit&author=MM%20Nizzari&author=CJ%20O'Donnell&volume=24&publication_year=2008&pages=2938-2939&pmid=18974171&doi=10.1093/bioinformatics/btn564&)

34. Cartegni L, Wang J, Zhu Z, Zhang MQ, Krainer AR (2003) ESEfinder: A web resource to identify exonic splicing enhancers. Nucleic Acids Res 31: 3568–3571.  [DOI](https://doi.org/10.1093/nar/gkg616) | [PMC free article](/articles/PMC169022/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12824367/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=ESEfinder:%20A%20web%20resource%20to%20identify%20exonic%20splicing%20enhancers&author=L%20Cartegni&author=J%20Wang&author=Z%20Zhu&author=MQ%20Zhang&author=AR%20Krainer&volume=31&publication_year=2003&pages=3568-3571&pmid=12824367&doi=10.1093/nar/gkg616&)
