# Pharmacogenetics of MicroRNAs and MicroRNAs Biogenesis Machinery in Pediatric Acute Lymphoblastic Leukemia

## Metadata
**Authors:** Elixabet López-López, Ángela Gutiérrez-Camino, Maria Ángeles Piñán, José Sánchez-Toledo, Jose Javier Uriz, Javier Ballesteros, Purificación García-Miguel, Aurora Navajas, África García-Orad
**Journal:** PLoS ONE
**Date:** 2014 Mar 10
**DOI:** [10.1371/journal.pone.0091261](https://doi.org/10.1371/journal.pone.0091261)
**PMID:** 24614921
**PMCID:** PMC3948785
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3948785/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC3948785/pdf/pone.0091261.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC3948785/pdf/pone.0091261.pdf)

## Abstract

Despite the clinical success of acute lymphoblastic leukemia (ALL) therapy, toxicity is frequent. Therefore, it would be useful to identify predictors of adverse effects. In the last years, several studies have investigated the relationship between genetic variation and treatment-related toxicity. However, most of these studies are focused in coding regions. Nowadays, it is known that regions that do not codify proteins, such as microRNAs (miRNAs), may have an important regulatory function. MiRNAs can regulate the expression of genes affecting drug response. In fact, the expression of some of those miRNAs has been associated with drug response. Genetic variations affecting miRNAs can modify their function, which may lead to drug sensitivity. The aim of this study was to detect new toxicity markers in pediatric B-ALL, studying miRNA-related polymorphisms, which can affect miRNA levels and function. We analyzed 118 SNPs in pre-miRNAs and miRNA processing genes in association with toxicity in 152 pediatric B-ALL patients all treated with the same protocol (LAL/SHOP). Among the results found, we detected for the first time an association between rs639174 in DROSHA and vomits that remained statistically significant after FDR correction. DROSHA had been associated with alterations in miRNAs expression, which could affect genes involved in drug transport. This suggests that miRNA-related SNPs could be a useful tool for toxicity prediction in pediatric B-ALL.

## Introduction

Acute lymphoblastic leukemia (ALL) is the most common childhood cancer, accounting for 30% of all pediatric malignancies [[1]](#pone.0091261-Johnston1)]. During the last decades, survival has been increased due to advances in chemotherapy for childhood ALL and cure rates now exceed 80% [[2]](#pone.0091261-Pui1)]. However, despite the clinical success of therapy, patients often suffer from toxicity, requiring a dose reduction or cessation of treatment. Therefore, it would be very useful to identify predictors of these adverse effects [[3]](#pone.0091261-Pui2)].

In the last years, several studies have investigated the relationship between genetic variation and treatment-related toxicity in ALL [[4]](#pone.0091261-Kishi1)–[[10]](#pone.0091261-WanRosalina1). Nevertheless, most of these studies are focused in coding regions, which correspond only to about 1.5% of the entire genome.

Nowadays, it is known that regions that do not codify proteins have an important regulatory function. MiRNAs are small non-coding RNAs that regulate gene expression at the post-transcriptional level [[11]](#pone.0091261-Gong1). They are transcribed in the nucleus, as double stranded pri-miRNAs, which are processed to form the pre-miRNAs. The pre-miRNAs are exported to the cytoplasm and cleaved to produce two strands of miRNA [[12]](#pone.0091261-Ryan1), [[13]](#pone.0091261-Munker1). MiRNAs recognize their target mRNAs by binding to the 3′UTR of the target gene [[14]](#pone.0091261-Lewis1), which leads to an inhibition of translation or facilitated degradation of the target mRNA. All those steps are regulated by genes of the miRNA processing machinery.

MiRNAs can regulate genes involved in drug transport, metabolism and targets [[15]](#pone.0091261-Bartel1), affecting treatment response [[16]](#pone.0091261-Yokoi1). For example, upregulation of miR-125b, miR99a and miR-100 was related to resistance to vincristine and daunorubicin, and downregulation of miR-708 with resistance to glucocorticoids in pediatric B-ALL [[17]](#pone.0091261-Schotte1), [[18]](#pone.0091261-Han1). This data indicate that changes in the expression or function of miRNAs could affect response to treatment.

Variations in miRNA expression and function may occur through genetic variations [[12]](#pone.0091261-Ryan1), [[19]](#pone.0091261-Slaby1). Consequently, miRNA-related SNPs interfering with miRNA levels or function may lead to drug resistance or to drug sensitivity [[20]](#pone.0091261-Mishra1). In fact, response to methotrexate (MTX), one of the most important drugs in ALL treatment, has been associated with SNP 829C>T near the miR-24 binding site in the 3′UTR of *DHFR*, that causes increased DHFR expression [[21]](#pone.0091261-Mishra2). Also, in our group we observed that a polymorphism, that created a new miRNA binding site in *ABCC4* and could reduce ABCC4 expression, was associated with increased MTX plasma levels [[22]](#pone.0091261-LopezLopez2).

According to all those evidences, the aim of the present study was to determine if miRNA-related polymorphisms could be useful as new toxicity markers in pediatric B-ALL.

## Materials and Methods

### Ethics statement

University of the Basque Country (UPV/EHU) ethics committee board (CEISH) approval was obtained. Written Informed consent was obtained from all patients or their parents before sample collection.

### Patients

The patients included in this retrospective study were 152 children all diagnosed with B-ALL from 2000 to 2011 at the Pediatric Oncology Units of 4 Spanish hospitals (University Hospital Cruces; University Hospital Donostia; University Hospital Vall d'Hebrón and University Hospital La Paz).

### Treatment and toxicity evaluation

All patients were homogeneously treated with the LAL-SHOP 94/99 and 2005 protocols. The Induction phase consisted of treatment with daunorubicin, vincristine, prednisone, cyclophosphamide, asparaginase and triple intrathecal therapy. The consolidation phase consisted of high-dose methotrexate, mercaptopurine, cytarabine and triple intrathecal therapy [[22]](#pone.0091261-LopezLopez2).

Toxicity data were collected objectively, blinded to genotypes, from the patients' medical files. Toxicity was graded according to the Spanish Society of Pediatric Hematology and Oncology (SHOP) standards, adapted from the WHO criteria (grades 0–4). The highest grade of toxicity observed for each patient during the induction and consolidation therapy period was recorded.

### Genes and polymorphisms selection

We selected 21 genes in the pathway of miRNAs biogenesis and processing after literature review and using Patrocles database [[23]](#pone.0091261-Hiard1) ([Table 1](#pone-0091261-t001)). In each gene, we covered all the SNPs with potentially functional effects using F-SNP, Fast-SNP, polymirTS [[24]](#pone.0091261-Bao1), [[25]](#pone.0091261-Ziebarth1) and Patrocles [[23]](#pone.0091261-Hiard1) databases. We considered functional effects those causing amino acid changes, alternative splicing, in the promoter region in putative transcription factor binding sites, or disrupting/creating miRNAs targets. We also selected SNPs previously included in association studies in the literature. All SNPs were selected with a minor allele frequency greater than 5% (MAF≥0.05) in European/Caucasoid populations.

### Table 1. Genes involved in miRNA biogenesis and processing.

| COMPLEX | SUBGROUP | GENE |
| --- | --- | --- |
| DROSHA COMPLEX | DGCR8 | DGCR8 |
|   | DROSHA | DROSHA |
|   | SMAD5 | SMAD5 |
| NUCLEAR EXPORT COMPLEX | XPO5 | XPO5 |
|   | RAN | RAN |
| DICER COMPLEX | DICER1 | DICER1 |
|   | TARBP2P | TARBP2P |
| RISC COMPLEX | Argonaute Family | PIWIL1 |
|   | (AGO) | EIF2C1 |
|   |   | EIF2C2 |
|   | TNRC | TNRC6A |
|   |   | TNRC6B |
|   | SND1 | SND1 |
|   | GEMIN Complex | DDX20 |
|   |   | GEMIN4 |
|   |   | GEMIN5 |
|   | CCR4-NOT Complex | CNOT1 |
|   |   | CNOT2 |
|   |   | CNOT3 |
|   |   | CNOT4 |
|   |   | CNOT6 |
We searched for pre-miRNAs that had as putative targets genes involved in the pathways of the drugs used in LAL/SHOP protocol, using PharmGKB and mirWalk databases, and selected all the SNPs that had been described at the moment of the selection with a MAF>0.01 in European/Caucasic populations, using Patrocles and Ensembl databases and literature review.

### Genotyping

Genomic DNA was extracted with the phenol-chloroform method as previously described [[8]](#pone.0091261-LopezLopez1) from remission peripheral blood.

SNP genotyping was performed using TaqMan OpenArray Genotyping technology (Applied Biosystems, Life Technologies, Carlsbad, USA) according to the published Applied Biosystems protocol. The preliminary list of SNPs was filtered, using as criteria, suitability for the Taqman OpenArray platform.

Data were analyzed with Taqman Genotyper software for genotype clustering and calling. Duplicate samples were genotyped across the plates. In order to assess the Hardy-Weinberg equilibrium (HWE) status of each SNP, we genotyped in parallel 348 healthy adult individuals of Spanish origin.

### Statistical analysis

The χ2 or Fisher's exact test were used for HWE and toxicity analyses. The effect sizes of the associations were estimated by the odds ratios (OR's) from univariate logistic regression. The most significant test among dominant and recessive genetic models was selected. The results for each toxicity parameter were adjusted for multiple comparisons by the False Discovery Rate (FDR) [[26]](#pone.0091261-Benjamini1). In all cases the significance level was set at 5%. Analyses were performed by using R v2.11 software. Linkage disequilibrium analysis was performed with Haploview software v4.2.

## Results

### Patients' baseline characteristics

In this study, we have analyzed 152 B-ALL patients. Clinical data about MTX plasma concentration 72 h after infusion were available for 141 patients. Clinical data about other therapy-related toxicity in induction were available for 137 patients and in consolidation for 130 patients ([Table 2](#pone-0091261-t002)).

### Table 2. Characteristics of the study population.

| No. of patients, n | 152 |
| --- | --- |
| Mean age at diagnosis ± SD, years | 5.1456±3.41 |
| Sex, n (%) |   |
| Female | 65 (42.76) |
| Male | 87 (57.23) |
| Risk group, n (%) |   |
| Standard | 56 (40.57) |
| High | 56 (40.57) |
| Very high | 26 (18.84) |
| Treatment protocol, n (%) |   |
| LAL-SHOP 94/99 | 65 (43.05) |
| LAL-SHOP 2005 | 86 (56.95) |
| MTX dose in consolidation, n (%) |   |
| 3 g/m2 | 73 (48.34) |
| 5 g/m2 | 78 (51.66) |
| Toxicity during induction therapy, n (%) |   |
| Any toxicity | 79 (57.66) |
| Hepatic (Grade 2–4) | 45 (32.84) |
| Vomits (Grade 2–4) | 36 (26.28) |
| Diarrhea (Grade 2–4) | 16 (11.67) |
| Mucositis (Grade 2–4) | 29 (21.17) |
| Hyperbilirubinemia (Grade 1–4) | 21 (15.32) |
| Renal (Grade 1–4) | 5 (3.65) |
| Toxicity during consolidation therapy, n (%) |   |
| Any toxicity | 71 (54.61) |
| Hepatic (Grade 2–4) | 39 (30) |
| Vomits (Grade 2–4) | 31 (23.85) |
| Diarrhea (Grade 2–4) | 9 (6.92) |
| Mucositis (Grade 2–4) | 14 (10.77) |
| Hyperbilirubinemia (Grade 1–4) | 11 (8.46) |
| Renal (Grade 1–4) | 13 (10) |
| MTX concentration in plasma a, n (%) |   |
| Higher than 0.2 uM at 72 h | 51 (36.17) |

Table 2 Caption: aMTX levels were considered high if the concentration was over 0.2 µM at 72 h.

### Genotyping Results

We selected a total of 131 SNPs. After filtering for suitability for the Taqman Openarray platform, a final number of 118 SNPs (72 in 21 genes involved in miRNA biogenesis and 46 in 42 pre-miRNAs) was included in a Taqman Openarray Plate (Applied Biosystems) ([Table S1](#pone.0091261.s001) and [S2](#pone.0091261.s002)).

A successful genotyping was obtained in 145 DNA samples (95.39%). In the genotyping process, 106 SNPs out of 118 were genotyped satisfactorily (89.83%). The failures were due to no PCR amplification, insufficient intensity for cluster separation, or poor or no cluster definition). The average genotyping rate for all SNPs was 97.81%. Of those 106 SNPs, 14 were not in HWE in a population of 348 healthy controls and were not considered for further analysis. In total, 26 SNPs were excluded from the association study ([Table S3](#pone.0091261.s003)). The other 92 SNPs were used in the association studies.

### Analysis of the association with toxicity

In order to investigate if genetic variation may influence treatment toxicity, we tested the association between the 92 polymorphisms successfully genotyped that were in HWE in the control population and 15 different toxicity and pharmacokinetic parameters in the induction and consolidation phases ([Table 2](#pone-0091261-t002)) ([Table S4](#pone.0091261.s004) and [S5](#pone.0091261.s005)).

In the genes of the miRNA biogenesis machinery, the most significant association was found between rs639174 in *DROSHA* and vomits in consolidation (p-value = 0.0003). This association remained statistically significant after FDR correction (p-corrected = 0.028). Interestingly, in *DROSHA* gene, a total of 8 SNPs out of 14 analyzed were associated with toxicity. Rs639174, rs2287584, rs10035440, rs4867329 and rs3805500 in *DROSHA* were among the top 10 associated SNPs in the biogenesis machinery ([Table 3](#pone-0091261-t003)). Those SNPs are located along the whole gene and, in general, were not in high linkage disequilibrium among them (r^2^<0.8) ([Figure 1](#pone-0091261-g001)).

### Table 3. Most significant associations between polymorphisms in biogenesis machinery and toxicity parameters.

| Gene | SNP | Toxicity | Phase | Genotype | No toxicity, n (%) | Toxicity, n (%) | OR (95% CI) | p-value | p-corrected |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| DROSHA | rs639174 | Vomits | Cons | CC | 47 (92.2) | 4 (7.8) | 1.00 | 0.0003 | 0.0276 |
|   |   |   |   | CT/TT | 43 (65.2) | 23 (34.9) | 6.28 (2.01–19.64) |   |   |
|   | rs2287584 | Vomits | Cons | TT | 54 (88.5) | 7 (11.5) | 1.00 | 0.0026 | 0.1196 |
|   |   |   |   | CT/CC | 41 (66.1) | 21 (33.9) | 3.95 (1.53–10.18) |   |   |
|   | rs10035440 | Hyperbilirubinemia | Ind | TT | 43 (72.9) | 16 (27.1) | 1.00 | 0.0041 | 0.1886 |
|   |   |   |   | CT/CC | 65 (92.9) | 5 (7.1) | 0.23 (0.08–0.68) |   |   |
|   | rs4867329 | Vomits | Cons | AA/AC | 74 (72.6) | 28 (27.5) | 1.00 | 0.0088 | 0.2162 |
|   |   |   |   | CC | 21 (95.5) | 1 (4.6) | 0.13 (0.02–0.98) |   |   |
|   | rs3805500 | Hepatic toxicity | Cons | AA/AG | 70 (68.0) | 33 (32.0) | 1.00 | 0.0091 | 0.2392 |
|   |   |   |   | GG | 17 (94.4) | 1 (5.6) | 0.12 (0.02–0.98) |   |   |
| XPO5 | rs34324334 | Hyperbilirubinemia | Ind | CC | 101 (88.6) | 13 (11.4) | 1.00 | 0.0011 | 0.1012 |
|   |   |   |   | CT | 7 (50.0) | 7 (50.0) | 7.77 (2.35–25.7) |   |   |
| TNRC6A | rs6497759 | Hepatic toxicity | Ind | GG | 63 (76.8) | 19 (23.2) | 1.00 | 0.0013 | 0.1196 |
|   |   |   |   | AG/AA | 23 (48.9) | 24 (51.1) | 3.46 (1.60–7.46) |   |   |
| TNRC6B | rs9611280 | Hepatic toxicity | Cons | GG | 70 (67.3) | 34 (32.7) | 1.00 | 0.0042 | 0.2392 |
|   |   |   |   | AG | 19 (95.0) | 1 (5.0) | 0.11 (0.01–0.84) |   |   |
| CNOT1 | rs11866002 | Mucositis | Ind | CC/CT | 98 (81.7) | 22 (18.3) | 1.00 | 0.0056 | 0.3956 |
|   |   |   |   | TT | 4 (40.0) | 6 (60.0) | 6.68 (1.74–25.70) |   |   |
| DDX20 | rs197388 | Renal toxicity | Ind | AA/AT | 121 (97.6) | 3 (2.4) | 1.00 | 0.0068 | 0.4048 |
|   |   |   |   | TT | 3 (60.0) | 2 (40.0) | 26.89 (3.21–225) |   |   |
| GEMIN4 | rs3744741 | Hepatic toxicity | Ind | CC | 70 (73.7) | 25 (26.3) | 1.00 | 0.0080 | 0.2392 |
|   |   |   |   | CT/TT | 17 (48.6) | 18 (51.4) | 2.96 (1.33–6.63) |   |   |
| EIF2C1 | rs595961 | Vomits | Ind | AA | 67 (79.8) | 17 (20.2) | 1.00 | 0.0086 | 0.4017 |
|   |   |   |   | AG/GG | 24 (57.1) | 18 (42.9) | 2.96 (1.31–6.65) |   |   |

Table 3 Caption: Ind: induction. Cons: consolidation. p-corrected = p-value after FDR correction. N.S.: non-significant, p-value>0.05 after FDR correction.

### Figure 1. Linkage disequilibrium plot of the SNPs analyzed in DROSHA.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c34f/3948785/66d0d2b8ea0b/pone.0091261.g001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3948785_pone.0091261.g001.jpg)

White: r2 = 0, shades of grey: 0<r2<1, black: r2 = 1. Numbers in squares are D' values. Block definition is based on the Gabriel et al. method. The SNPs associated with toxicity are squared. Those among the top 10 associated SNPs are squared in black

Among the pre-miRNAs, the most significant SNPs were rs12894467 in mir-300, associated with hepatic toxicity and hyperbilirubinemia in induction, and rs56103835 in mir-453, associated with vomits and MTX plasma levels in consolidation ([Table 4](#pone-0091261-t004)).

### Table 4. Most significant associations between polymorphisms in pre-miRNAs and toxicity parameters.

| Gene | SNP | Toxicity | Phase | Genotype | No toxicity, n(%) | Toxicity, n(%) | OR (95% CI) | p-value | p-corrected |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| mir-300 | rs12894467 | Hepatic toxicity | Ind | CC/CT | 82 (71.9) | 32 (28.1) | 1.00 | 0.0038 | 0.1748 |
|   |   |   |   | TT | 5 (33.3) | 10 (66.8) | 5.12 (1.63–16.16) |   |   |
|   |   | Hyperbilirubinemia | Ind | CC/CT | 99 (86.8) | 15 (13.2) | 1.00 | 0.0174 | 0.2990 |
|   |   |   |   | TT | 9 (60.0) | 6 (40.0) | 4.40 (1.37–14.13) |   |   |
| mir-449b | rs10061133 | Renal toxicity | Ind | AA | 111 (98.2) | 2 (1.8) | 1.00 | 0.0132 | 0.4048 |
|   |   |   |   | GA/GG | 15 (83.3) | 3 (16.7) | 11.10 (1.71–71.9) |   |   |
| mir-453 | rs56103835 | Vomits | Cons | AA | 66 (83.5) | 13 (16.5) | 1.00 | 0.0141 | 0.2162 |
|   |   |   |   | GA/GG | 28 (63.6) | 16 (36.4) | 2.90 (1.23–6.82) |   |   |
|   |   | MTX clearance | Cons | AA | 62 (72.1) | 24 (27.9) | 1.00 | 0.0297 | 0.4554 |
|   |   |   |   | GA/GG | 25 (53.2) | 22 (46.8) | 2.27 (1.08–4.77) |   |   |
| mir-2053 | rs10505168 | Mucositis | Cons | TT | 43 (82.7) | 9 (17.3) | 1.00 | 0.0154 | 0.4298 |
|   |   |   |   | CT/CC | 68 (95.8) | 3 (4.2) | 0.21 (0.05–0.82) |   |   |
| mir-423 | rs6505162 | Diarrhea | Ind | CC | 25 (75.8) | 8 (24.2) | 1.00 | 0.0240 | 0.4618 |
|   |   |   |   | AC/AA | 88 (91.7) | 8 (8.3) | 0.28 (0.10–0.83) |   |   |
| mir-1307 | rs7911488 | Diarrhea | Ind | AA/AG | 100 (90.1) | 11 (9.9) | 1.00 | 0.0241 | 0.4618 |
|   |   |   |   | GG | 10 (66.7) | 5 (33.3) | 4.55 (1.31–15.72) |   |   |
| mir-618 | rs2682818 | Hyperbilirubinemia | Ind | CC/AC | 110 (85.4) | 19 (14.7) | 1.00 | 0.0247 | 0.3002 |
|   |   |   |   | AA | 0 (0.0) | 2 (100.0) | NE (NE- NE) |   |   |
| mir-146a | rs2910164 | Diarrhea | Ind | GG | 70 (93.3) | 5 (6.7) | 1.00 | 0.0251 | 0.4618 |
|   |   |   |   | CG/CC | 45 (80.4) | 11 (19.6) | 3.42 (1.11–10.50) |   |   |
| mir-1206 | rs2114358 | Mucositis | Cons | AA/AG | 96 (92.3) | 8 (7.7) | 1.00 | 0.0254 | 0.4298 |
|   |   |   |   | GG | 15 (75.0) | 5 (25.0) | 4.57 (1.28–16.28) |   |   |
| mir-577 | rs34115976 | Hyperbilirubinemia | Ind | CC | 67 (78.8) | 18 (21.2) | 1.00 | 0.0261 | 0.3002 |
|   |   |   |   | CG/GG | 41 (93.2) | 3 (6.8) | 0.27 (0.08–0.98) |   |   |
| mir-604 | rs2368393 | Renal toxicity | Cons | AA | 65 (95.6) | 3 (4.4) | 1.00 | 0.0271 | 0.6866 |
|   |   |   |   | AG/GG | 47 (83.9) | 9 (16.1) | 4.15 (1.07–16.15) |   |   |
| mir-492 | rs2289030 | Vomits | Ind | GG | 90 (76.3) | 28 (23.7) | 1.00 | 0.0282 | 0.5188 |
|   |   |   |   | CG | 6 (46.2) | 7 (53.9) | 3.75(1.16–12.08) |   |   |

Table 4 Caption: Ind: induction. Cons: consolidation. p-corrected = p-value after FDR correction. N.S.: non-significant, p-value>0.05 after FDR correction.

## Discussion

It is known that ALL treatment can cause toxicity and toxicity predictors are needed. It has been proposed that miRNA-related SNPs interfering with miRNA function may lead to drug resistance or to drug sensitivity [[20]](#pone.0091261-Mishra1). However, there are very few studies analyzing the role of polymorphisms in miRNA biogenesis genes and in miRNAs and none of them had been performed in pediatric ALL.

In this study, it is worth noting that we have found for the first time an association between rs639174 in *DROSHA* and vomits and this association remained statistically significant after FDR correction. In *DROSHA*, rs639174 is an intronic SNP with a putative role in transcriptional regulation (TR). This SNP had been previously associated with head and neck cancer recurrence, suggesting that in some way this SNP may have a functional effect on the gene [[27]](#pone.0091261-Zhang1). Interestingly, other 7 polymorphisms in *DROSHA* with a putative role in splicing and transcriptional regulation were associated with toxicity in induction and consolidation (rs10035440, rs2287584, rs4867329, rs3805500, rs6877842, rs10719 and rs7735863), although they did not remain statistically significant after FDR correction. The SNP rs3805500, which is associated with hepatic toxicity and vomits in our study, is in LD with the SNP rs640831, previously associated with reduced *DROSHA* mRNA expression and with expression changes in 56 miRNAs out of 199 analyzed [[28]](#pone.0091261-Rotunno1). This can be understood knowing that *DROSHA* (*RNASEN*) encodes an RNAse III enzyme, involved in pri-miRNAs maturation into pre-miRNAs [[29]](#pone.0091261-Phua1). This general alteration of miRNA expression could lead to changes in the expression of genes involved in response to treatment, which could explain the effect we have observed on toxicity during pediatric ALL treatment.

As far as we know, this is the first time that polymorphisms in miRNA processing genes have been associated with toxicity after treatment in cancer patients. Knowing that literature about the function of these genes and their implication in pharmacogenetics is scarce, our results indicate that these genes and polymorphisms could be of relevance in the study of drug response.

We also found associations between SNPs in pre-miRNAs and toxicity. Interestingly, SNPs associated with toxicity in induction were different from those associated with toxicity in consolidation, in which different drugs are given. This may mean that each miRNA regulates specific drug pathways. Although these associations did not remain significant after FDR correction and are currently of uncertain significance, we still consider that it is interesting to discuss them due to their putative roles in the regulation of drug pathways.

The most significant association between SNPs in pre-miRNAs and toxicity in induction was with rs12894467 in the premature mir-300, which could affect the structure and processing of this miRNA. Interestingly, among the predicted targets of mir-300, we can find the transporters *ABCC1* and *ABCB1*, with a role in vincristine detoxification, and the enzyme *ALDH5A1*, involved in cyclophosphamide inactivation. If the rs12894467 T risk allele caused an upregulation of mir-300, this could explain a downregulation of its targets, leading to an increased effect of the drugs used in the induction phase.

The most significant association in consolidation was between the SNP rs56103835 in the premature mir-453 (also known as mir-323b-5p) and both MTX plasma levels and vomits. This miRNA has as putative target genes *ABCC1*, *ABCB1*, *ABCC2* and *ABCC4*, which are involved in MTX transport. The SNP rs56103835, in which G allele is associated with higher risk of toxicity, is in the pre-miRNA, and thus could influence miRNA biogenesis and levels of mature mir-453. If mir-453 is up-regulated, it would decrease the activity of *ABCC1*, *ABCB1*, *ABCC2* and *ABCC4* genes, and the higher MTX plasma levels and toxicity observed could be explained. In fact, in a previous study carried out by our group, we showed the relevance of genetic variation in *ABCC2* and *ABCC4* genes for MTX toxicity [[22]](#pone.0091261-LopezLopez2).

In conclusion, we have found for the first time an association between rs639174 in DROSHA and vomits and other more uncertain associations between polymorphism in genes involved in miRNAs biogenesis and in pre-miRNAs and toxicity during pediatric ALL treatment. These results suggest that miRNA-related SNPs, which can be important in drug pharmacokinetics and dynamics, could be useful as toxicity markers in pediatric ALL. We open a new promising field of investigation, involving the study of miRNA-related polymorphisms in pediatric ALL treatment. Further studies are needed in order to assess the relevance of these SNPs in ALL pharmacogenetics.

## Supporting Information

## Acknowledgments

Support by SGIker at the University of the Basque Country (UPV/EHU) is gratefully acknowledged.

## Funding Statement

This project was supported by the Spanish Thematic Network for Cooperative Investigation in Cancer RTICC (RD/06/0020/0048), Basque Government (GIC10/71, SAI11/75), and University of the Basque Country UPV/EHU (UFI11/35 and GIU10/24). ELL was supported by a predoctoral grant from the Basque Government and by a “Fellowship for recent doctors until their integration in postdoctoral programs” by the Investigation Vice-rector's office of the University of the Basque Country UPV/EHU. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Johnston WT, Lightfoot TJ, Simpson J, Roman E (2010) Childhood cancer survival: a report from the United Kingdom Childhood Cancer Study. Cancer Epidemiol 34: 659–666.  [DOI](https://doi.org/10.1016/j.canep.2010.06.020) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20674536/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Epidemiol&title=Childhood%20cancer%20survival:%20a%20report%20from%20the%20United%20Kingdom%20Childhood%20Cancer%20Study&author=WT%20Johnston&author=TJ%20Lightfoot&author=J%20Simpson&author=E%20Roman&volume=34&publication_year=2010&pages=659-666&pmid=20674536&doi=10.1016/j.canep.2010.06.020&)

2. Pui CH, Robison LL, Look AT (2008) Acute lymphoblastic leukaemia. Lancet 371: 1030–1043.  [DOI](https://doi.org/10.1016/S0140-6736(08)60457-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18358930/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Acute%20lymphoblastic%20leukaemia&author=CH%20Pui&author=LL%20Robison&author=AT%20Look&volume=371&publication_year=2008&pages=1030-1043&pmid=18358930&doi=10.1016/S0140-6736(08)60457-2&)

3. Pui CH, Mullighan CG, Evans WE, Relling MV (2012) Pediatric acute lymphoblastic leukemia: where are we going and how do we get there? Blood 120: 1165–1174.  [DOI](https://doi.org/10.1182/blood-2012-05-378943) | [PMC free article](/articles/PMC3418713/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22730540/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Pediatric%20acute%20lymphoblastic%20leukemia:%20where%20are%20we%20going%20and%20how%20do%20we%20get%20there?&author=CH%20Pui&author=CG%20Mullighan&author=WE%20Evans&author=MV%20Relling&volume=120&publication_year=2012&pages=1165-1174&pmid=22730540&doi=10.1182/blood-2012-05-378943&)

4. Kishi S, Cheng C, French D, Pei D, Das S, et al. (2007) Ancestry and pharmacogenetics of antileukemic drug toxicity. Blood 109: 4151–4157.  [DOI](https://doi.org/10.1182/blood-2006-10-054528) | [PMC free article](/articles/PMC1885506/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17264302/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Ancestry%20and%20pharmacogenetics%20of%20antileukemic%20drug%20toxicity&author=S%20Kishi&author=C%20Cheng&author=D%20French&author=D%20Pei&author=S%20Das&volume=109&publication_year=2007&pages=4151-4157&pmid=17264302&doi=10.1182/blood-2006-10-054528&)

5. Huang L, Tissing WJ, de Jonge R, van Zelst BD, Pieters R (2008) Polymorphisms in folate-related genes: association with side effects of high-dose methotrexate in childhood acute lymphoblastic leukemia. Leukemia 22: 1798–1800.  [DOI](https://doi.org/10.1038/leu.2008.66) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18368069/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Leukemia&title=Polymorphisms%20in%20folate-related%20genes:%20association%20with%20side%20effects%20of%20high-dose%20methotrexate%20in%20childhood%20acute%20lymphoblastic%20leukemia&author=L%20Huang&author=WJ%20Tissing&author=R%20de%20Jonge&author=BD%20van%20Zelst&author=R%20Pieters&volume=22&publication_year=2008&pages=1798-1800&pmid=18368069&doi=10.1038/leu.2008.66&)

6. Liu SG, Li ZG, Cui L, Gao C, Li WJ, et al. (2011) Effects of methylenetetrahydrofolate reductase gene polymorphisms on toxicities during consolidation therapy in pediatric acute lymphoblastic leukemia in a Chinese population. Leuk Lymphoma 52: 1030–1040.  [DOI](https://doi.org/10.3109/10428194.2011.563883) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21534867/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Leuk%20Lymphoma&title=Effects%20of%20methylenetetrahydrofolate%20reductase%20gene%20polymorphisms%20on%20toxicities%20during%20consolidation%20therapy%20in%20pediatric%20acute%20lymphoblastic%20leukemia%20in%20a%20Chinese%20population&author=SG%20Liu&author=ZG%20Li&author=L%20Cui&author=C%20Gao&author=WJ%20Li&volume=52&publication_year=2011&pages=1030-1040&pmid=21534867&doi=10.3109/10428194.2011.563883&)

7. Treviño LR, Shimasaki N, Yang W, Panetta JC, Cheng C, et al. (2009) Germline genetic variation in an organic anion transporter polypeptide associated with methotrexate pharmacokinetics and clinical effects. J Clin Oncol 27: 5972–5978.  [DOI](https://doi.org/10.1200/JCO.2008.20.4156) | [PMC free article](/articles/PMC2793040/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19901119/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Germline%20genetic%20variation%20in%20an%20organic%20anion%20transporter%20polypeptide%20associated%20with%20methotrexate%20pharmacokinetics%20and%20clinical%20effects&author=LR%20Trevi%C3%B1o&author=N%20Shimasaki&author=W%20Yang&author=JC%20Panetta&author=C%20Cheng&volume=27&publication_year=2009&pages=5972-5978&pmid=19901119&doi=10.1200/JCO.2008.20.4156&)

8. Lopez-Lopez E, Martin-Guerrero I, Ballesteros J, Piñan MA, Garcia-Miguel P, et al. (2011) Polymorphisms of the SLCO1B1 gene predict methotrexate-related toxicity in childhood acute lymphoblastic leukemia. Pediatr Blood Cancer 57: 612–619.  [DOI](https://doi.org/10.1002/pbc.23074) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21387541/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pediatr%20Blood%20Cancer&title=Polymorphisms%20of%20the%20SLCO1B1%20gene%20predict%20methotrexate-related%20toxicity%20in%20childhood%20acute%20lymphoblastic%20leukemia&author=E%20Lopez-Lopez&author=I%20Martin-Guerrero&author=J%20Ballesteros&author=MA%20Pi%C3%B1an&author=P%20Garcia-Miguel&volume=57&publication_year=2011&pages=612-619&pmid=21387541&doi=10.1002/pbc.23074&)

9. Faganel Kotnik B, Grabnar I, Bohanec Grabar P, Dolžan V, Jazbec J (2011) Association of genetic polymorphism in the folate metabolic pathway with methotrexate pharmacokinetics and toxicity in childhood acute lymphoblastic leukaemia and malignant lymphoma. Eur J Clin Pharmacol 67: 993–1006.  [DOI](https://doi.org/10.1007/s00228-011-1046-z) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21509569/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Clin%20Pharmacol&title=Association%20of%20genetic%20polymorphism%20in%20the%20folate%20metabolic%20pathway%20with%20methotrexate%20pharmacokinetics%20and%20toxicity%20in%20childhood%20acute%20lymphoblastic%20leukaemia%20and%20malignant%20lymphoma&author=B%20Faganel%20Kotnik&author=I%20Grabnar&author=P%20Bohanec%20Grabar&author=V%20Dol%C5%BEan&author=J%20Jazbec&volume=67&publication_year=2011&pages=993-1006&pmid=21509569&doi=10.1007/s00228-011-1046-z&)

10. Wan Rosalina WR, Teh LK, Mohamad N, Nasir A, Yusoff R, et al. (2012) Polymorphism of ITPA 94C>A and risk of adverse effects among patients with acute lymphoblastic leukaemia treated with 6-mercaptopurine. J Clin Pharm Ther 37: 237–241.  [DOI](https://doi.org/10.1111/j.1365-2710.2011.01272.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21545474/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Pharm%20Ther&title=Polymorphism%20of%20ITPA%2094C>A%20and%20risk%20of%20adverse%20effects%20among%20patients%20with%20acute%20lymphoblastic%20leukaemia%20treated%20with%206-mercaptopurine&author=WR%20Wan%20Rosalina&author=LK%20Teh&author=N%20Mohamad&author=A%20Nasir&author=R%20Yusoff&volume=37&publication_year=2012&pages=237-241&pmid=21545474&doi=10.1111/j.1365-2710.2011.01272.x&)

11. Gong J, Tong Y, Zhang HM, Wang K, Hu T, et al. (2012) Genome-wide identification of SNPs in microRNA genes and the SNP effects on microRNA target binding and biogenesis. Hum Mutat 33: 254–263.  [DOI](https://doi.org/10.1002/humu.21641) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22045659/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mutat&title=Genome-wide%20identification%20of%20SNPs%20in%20microRNA%20genes%20and%20the%20SNP%20effects%20on%20microRNA%20target%20binding%20and%20biogenesis&author=J%20Gong&author=Y%20Tong&author=HM%20Zhang&author=K%20Wang&author=T%20Hu&volume=33&publication_year=2012&pages=254-263&pmid=22045659&doi=10.1002/humu.21641&)

12. Ryan BM, Robles AI, Harris CC (2010) Genetic variation in microRNA networks: the implications for cancer research. Nat Rev Cancer 10: 389–402.  [DOI](https://doi.org/10.1038/nrc2867) | [PMC free article](/articles/PMC2950312/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20495573/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Cancer&title=Genetic%20variation%20in%20microRNA%20networks:%20the%20implications%20for%20cancer%20research&author=BM%20Ryan&author=AI%20Robles&author=CC%20Harris&volume=10&publication_year=2010&pages=389-402&pmid=20495573&doi=10.1038/nrc2867&)

13. Munker R, Calin GA (2011) MicroRNA profiling in cancer. Clin Sci (Lond) 121: 141–158.  [DOI](https://doi.org/10.1042/CS20110005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21526983/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Sci%20(Lond)&title=MicroRNA%20profiling%20in%20cancer&author=R%20Munker&author=GA%20Calin&volume=121&publication_year=2011&pages=141-158&pmid=21526983&doi=10.1042/CS20110005&)

14. Lewis BP, Shih IH, Jones-Rhoades MW, Bartel DP, Burge CB (2003) Prediction of mammalian microRNA targets. Cell 115: 787–798.  [DOI](https://doi.org/10.1016/s0092-8674(03)01018-3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14697198/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell&title=Prediction%20of%20mammalian%20microRNA%20targets&author=BP%20Lewis&author=IH%20Shih&author=MW%20Jones-Rhoades&author=DP%20Bartel&author=CB%20Burge&volume=115&publication_year=2003&pages=787-798&pmid=14697198&doi=10.1016/s0092-8674(03)01018-3&)

15. Bartel DP (2004) MicroRNAs: genomics, biogenesis, mechanism, and function. Cell 116: 281–297.  [DOI](https://doi.org/10.1016/s0092-8674(04)00045-5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14744438/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell&title=MicroRNAs:%20genomics,%20biogenesis,%20mechanism,%20and%20function&author=DP%20Bartel&volume=116&publication_year=2004&pages=281-297&pmid=14744438&doi=10.1016/s0092-8674(04)00045-5&)

16. Yokoi T, Nakajima M (2013) microRNAs as mediators of drug toxicity. Annu Rev Pharmacol Toxicol 53: 377–400.  [DOI](https://doi.org/10.1146/annurev-pharmtox-011112-140250) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23189953/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annu%20Rev%20Pharmacol%20Toxicol&title=microRNAs%20as%20mediators%20of%20drug%20toxicity&author=T%20Yokoi&author=M%20Nakajima&volume=53&publication_year=2013&pages=377-400&pmid=23189953&doi=10.1146/annurev-pharmtox-011112-140250&)

17. Schotte D, De Menezes RX, Moqadam FA, Khankahdani LM, Lange-Turenhout E, et al. (2011) MicroRNA characterize genetic diversity and drug resistance in pediatric acute lymphoblastic leukemia. Haematologica 96: 703–711.  [DOI](https://doi.org/10.3324/haematol.2010.026138) | [PMC free article](/articles/PMC3084917/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21242186/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Haematologica&title=MicroRNA%20characterize%20genetic%20diversity%20and%20drug%20resistance%20in%20pediatric%20acute%20lymphoblastic%20leukemia&author=D%20Schotte&author=RX%20De%20Menezes&author=FA%20Moqadam&author=LM%20Khankahdani&author=E%20Lange-Turenhout&volume=96&publication_year=2011&pages=703-711&pmid=21242186&doi=10.3324/haematol.2010.026138&)

18. Han BW, Feng DD, Li ZG, Luo XQ, Zhang H, et al. (2011) A set of miRNAs that involve in the pathways of drug resistance and leukemic stem-cell differentiation is associated with the risk of relapse and glucocorticoid response in childhood ALL. Hum Mol Genet 20: 4903–4915.  [DOI](https://doi.org/10.1093/hmg/ddr428) | [PMC free article](/articles/PMC3221537/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21926415/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=A%20set%20of%20miRNAs%20that%20involve%20in%20the%20pathways%20of%20drug%20resistance%20and%20leukemic%20stem-cell%20differentiation%20is%20associated%20with%20the%20risk%20of%20relapse%20and%20glucocorticoid%20response%20in%20childhood%20ALL&author=BW%20Han&author=DD%20Feng&author=ZG%20Li&author=XQ%20Luo&author=H%20Zhang&volume=20&publication_year=2011&pages=4903-4915&pmid=21926415&doi=10.1093/hmg/ddr428&)

19. Slaby O, Bienertova-Vasku J, Svoboda M, Vyzula R (2012) Genetic polymorphisms and microRNAs: new direction in molecular epidemiology of solid cancer. J Cell Mol Med 16: 8–21.  [DOI](https://doi.org/10.1111/j.1582-4934.2011.01359.x) | [PMC free article](/articles/PMC3823089/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21692980/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Cell%20Mol%20Med&title=Genetic%20polymorphisms%20and%20microRNAs:%20new%20direction%20in%20molecular%20epidemiology%20of%20solid%20cancer&author=O%20Slaby&author=J%20Bienertova-Vasku&author=M%20Svoboda&author=R%20Vyzula&volume=16&publication_year=2012&pages=8-21&pmid=21692980&doi=10.1111/j.1582-4934.2011.01359.x&)

20. Mishra PJ, Bertino JR (2009) MicroRNA polymorphisms: the future of pharmacogenomics, molecular epidemiology and individualized medicine. Pharmacogenomics 10: 399–416.  [DOI](https://doi.org/10.2217/14622416.10.3.399) | [PMC free article](/articles/PMC2705205/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19290790/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=MicroRNA%20polymorphisms:%20the%20future%20of%20pharmacogenomics,%20molecular%20epidemiology%20and%20individualized%20medicine&author=PJ%20Mishra&author=JR%20Bertino&volume=10&publication_year=2009&pages=399-416&pmid=19290790&doi=10.2217/14622416.10.3.399&)

21. Mishra PJ, Humeniuk R, Longo-Sorbello GS, Banerjee D, Bertino JR (2007) A miR-24 microRNA binding-site polymorphism in dihydrofolate reductase gene leads to methotrexate resistance. Proc Natl Acad Sci U S A 104: 13513–13518.  [DOI](https://doi.org/10.1073/pnas.0706217104) | [PMC free article](/articles/PMC1948927/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17686970/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&title=A%20miR-24%20microRNA%20binding-site%20polymorphism%20in%20dihydrofolate%20reductase%20gene%20leads%20to%20methotrexate%20resistance&author=PJ%20Mishra&author=R%20Humeniuk&author=GS%20Longo-Sorbello&author=D%20Banerjee&author=JR%20Bertino&volume=104&publication_year=2007&pages=13513-13518&pmid=17686970&doi=10.1073/pnas.0706217104&)

22. Lopez-Lopez E, Ballesteros J, Piñan MA, Sanchez de Toledo J, Garcia de Andoin N, et al. (2013) Polymorphisms in the methotrexate transport pathway: a new tool for MTX plasma level prediction in pediatric acute lymphoblastic leukemia. Pharmacogenet Genomics 23: 53–61.  [DOI](https://doi.org/10.1097/FPC.0b013e32835c3b24) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23222202/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Polymorphisms%20in%20the%20methotrexate%20transport%20pathway:%20a%20new%20tool%20for%20MTX%20plasma%20level%20prediction%20in%20pediatric%20acute%20lymphoblastic%20leukemia&author=E%20Lopez-Lopez&author=J%20Ballesteros&author=MA%20Pi%C3%B1an&author=J%20Sanchez%20de%20Toledo&author=N%20Garcia%20de%20Andoin&volume=23&publication_year=2013&pages=53-61&pmid=23222202&doi=10.1097/FPC.0b013e32835c3b24&)

23. Hiard S, Charlier C, Coppieters W, Georges M, Baurain D (2010) Patrocles: a database of polymorphic miRNA-mediated gene regulation in vertebrates. Nucleic Acids Res 38: D640–651.  [DOI](https://doi.org/10.1093/nar/gkp926) | [PMC free article](/articles/PMC2808989/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19906729/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=Patrocles:%20a%20database%20of%20polymorphic%20miRNA-mediated%20gene%20regulation%20in%20vertebrates&author=S%20Hiard&author=C%20Charlier&author=W%20Coppieters&author=M%20Georges&author=D%20Baurain&volume=38&publication_year=2010&pages=D640-651&pmid=19906729&doi=10.1093/nar/gkp926&)

24. Bao L, Zhou M, Wu L, Lu L, Goldowitz D, et al. (2007) PolymiRTS Database: linking polymorphisms in microRNA target sites with complex traits. Nucleic Acids Res 35: D51–54.  [DOI](https://doi.org/10.1093/nar/gkl797) | [PMC free article](/articles/PMC1669716/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17099235/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=PolymiRTS%20Database:%20linking%20polymorphisms%20in%20microRNA%20target%20sites%20with%20complex%20traits&author=L%20Bao&author=M%20Zhou&author=L%20Wu&author=L%20Lu&author=D%20Goldowitz&volume=35&publication_year=2007&pages=D51-54&pmid=17099235&doi=10.1093/nar/gkl797&)

25. Ziebarth JD, Bhattacharya A, Chen A, Cui Y (2012) PolymiRTS Database 2.0: linking polymorphisms in microRNA target sites with human diseases and complex traits. Nucleic Acids Res 40: D216–221.  [DOI](https://doi.org/10.1093/nar/gkr1026) | [PMC free article](/articles/PMC3245163/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22080514/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=PolymiRTS%20Database%202.0:%20linking%20polymorphisms%20in%20microRNA%20target%20sites%20with%20human%20diseases%20and%20complex%20traits&author=JD%20Ziebarth&author=A%20Bhattacharya&author=A%20Chen&author=Y%20Cui&volume=40&publication_year=2012&pages=D216-221&pmid=22080514&doi=10.1093/nar/gkr1026&)

26. Benjamini Y, Hochberg Y (1995) Controlling the false discovery rate: A practical and powerful approach to multiple testing.: J Royal Stat Soc Series B.

27. Zhang X, Yang H, Lee JJ, Kim E, Lippman SM, et al. (2010) MicroRNA-related genetic variations as predictors for risk of second primary tumor and/or recurrence in patients with early-stage head and neck cancer. Carcinogenesis 31: 2118–2123.  [DOI](https://doi.org/10.1093/carcin/bgq177) | [PMC free article](/articles/PMC3105587/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20819778/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Carcinogenesis&title=MicroRNA-related%20genetic%20variations%20as%20predictors%20for%20risk%20of%20second%20primary%20tumor%20and/or%20recurrence%20in%20patients%20with%20early-stage%20head%20and%20neck%20cancer&author=X%20Zhang&author=H%20Yang&author=JJ%20Lee&author=E%20Kim&author=SM%20Lippman&volume=31&publication_year=2010&pages=2118-2123&pmid=20819778&doi=10.1093/carcin/bgq177&)

28. Rotunno M, Zhao Y, Bergen AW, Koshiol J, Burdette L, et al. (2010) Inherited polymorphisms in the RNA-mediated interference machinery affect microRNA expression and lung cancer survival. Br J Cancer 103: 1870–1874.  [DOI](https://doi.org/10.1038/sj.bjc.6605976) | [PMC free article](/articles/PMC3008605/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21102586/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Cancer&title=Inherited%20polymorphisms%20in%20the%20RNA-mediated%20interference%20machinery%20affect%20microRNA%20expression%20and%20lung%20cancer%20survival&author=M%20Rotunno&author=Y%20Zhao&author=AW%20Bergen&author=J%20Koshiol&author=L%20Burdette&volume=103&publication_year=2010&pages=1870-1874&pmid=21102586&doi=10.1038/sj.bjc.6605976&)

29. Phua SL, Sivakamasundari V, Shao Y, Cai X, Zhang LF, et al. (2011) Nuclear accumulation of an uncapped RNA produced by Drosha cleavage of a transcript encoding miR-10b and HOXD4. PLoS One 6: e25689.  [DOI](https://doi.org/10.1371/journal.pone.0025689) | [PMC free article](/articles/PMC3185001/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21991333/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Nuclear%20accumulation%20of%20an%20uncapped%20RNA%20produced%20by%20Drosha%20cleavage%20of%20a%20transcript%20encoding%20miR-10b%20and%20HOXD4&author=SL%20Phua&author=V%20Sivakamasundari&author=Y%20Shao&author=X%20Cai&author=LF%20Zhang&volume=6&publication_year=2011&pages=e25689&pmid=21991333&doi=10.1371/journal.pone.0025689&)
