# Deciphering the Interleukin 28B Variants That Better Predict Response to Pegylated Interferon-α and Ribavirin Therapy in HCV/HIV-1 Coinfected Patients

## Metadata
**Authors:** Montserrat de Castellarnau, Ester Aparicio, Mariona Parera, Sandra Franco, Cristina Tural, Bonaventura Clotet, Miguel Angel Martínez
**Journal:** PLoS ONE
**Date:** 2012 Feb 6
**DOI:** [10.1371/journal.pone.0031016](https://doi.org/10.1371/journal.pone.0031016)
**PMID:** 22328925
**PMCID:** PMC3273458
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3273458/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC3273458/pdf/pone.0031016.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC3273458/pdf/pone.0031016.pdf)

## Abstract

Previous works have documented the contribution of different IL28B-associated SNPs to spontaneous HCV clearance. This study investigated the effect of different interleukin (IL) 28B genetic variants on interferon (IFN)-based therapy response. We genotyped eight IL28B single-nucleotide polymorphisms (SNPs) in a cohort of 197 hepatitis C virus (HCV)/human immunodeficiency virus type 1 (HIV-1) coinfected patients from our clinic unit who received combined pegylated (peg)-IFN-α and ribavirin (RBV) therapy. This analysis included the two strongest tag predictors for HCV clearance, rs8099917 and rs12979860, and four causal variants (rs4803219, rs28416813, rs8103142, and rs4803217) located in the IL28B promoter, coding, and 3′-untranslated regions. Haplotypes carrying the major alleles at IL28B SNPs were highly associated with sustained virological responses (SVRs) after treatment with peg-IFN-α and RBV [odds ratio (OR) = 2.5, 95% confidence interval (CI) = 1.6–4.0, 4.0×10−5]. Three causal SNP genotypes (rs28416813, rs8103142, and rs4803217) displayed the highest association with SVRs (OR = 3.7, 95% CI = 2.0–6.7, p = 1.3×10−5). All four causal variants were in high linkage disequilibrium, both among themselves (r2≥0.94) and with the rs12979860 variant (r2≥0.92). In contrast, rs8099917 was in low linkage disequilibrium with the four causal variants (r2≤0.45) and with the rs12979860 variant (r2 = 0.45). These results demonstrate that rs12979860, compared to rs8099917, may be a better predictor of response to the peg-IFN/RBV treatment among HCV/HIV-1 coinfected patients. Moreover, causal IL28B variants are strongly associated with treatment SVRs.

## Introduction

The combination of pegylated interferon alpha (peg-IFN-α) with ribavirin (RBV) has been used to treat hepatitis C virus (HCV) infection. However, a sustained virological response (SVR; a negative hepatitis C polymerase chain reaction (PCR) test 6 months after cessation of therapy) for individuals infected with HCV genotypes 1 or 4 ranges between 40% and 50%. Patients infected with HCV genotypes 2 or 3 typically achieve SVRs of nearly 75% after only 6 months of therapy [[1]](#pone.0031016-Fried1), [[2]](#pone.0031016-Manns1), [[3]](#pone.0031016-Fried2), [[4]](#pone.0031016-Hadziyannis1). HCV genotype has been the most important predictive factor regarding the treatment response of HCV-infected patients. Nevertheless, host factors such as age, sex, race, liver fibrosis, and obesity have also been associated with peg-IFN-α/RBV therapy outcome [[5]](#pone.0031016-Gao1), [[6]](#pone.0031016-Walsh1). Four genome-wide association studies have demonstrated that several highly-correlated, common, single nucleotide polymorphisms (SNPs) located near the interleukin 28B gene (IL28B) strongly predict an SVR to peg-IFN-α/RBV therapy [[7]](#pone.0031016-Ge1), [[8]](#pone.0031016-Suppiah1), [[9]](#pone.0031016-Tanaka1), [[10]](#pone.0031016-Rauch1). Il28B polymorphisms have been also strongly associated with SVR in HCV/HIV-1 coinfected patients [[11]](#pone.0031016-Rallon1), [[12]](#pone.0031016-Pineda1), [[13]](#pone.0031016-Rallon2), [[14]](#pone.0031016-Labarga1), [[15]](#pone.0031016-Clausen1), [[16]](#pone.0031016-Dayyeh1). Two SNPs in particular (rs12979860 and rs8099917, located 3 and 7.5 kb upstream of the IL28B gene, respectively) were the strongest predictors for HCV clearance. The recent approval of direct-acting antiviral (DAA) molecules, the NS3 protease inhibitors telaprevir and bocebrevir, active on HCV will represent a major breakthrough for HCV infected patients. Because of the low genetic resistance of first-generation protease inhibitors most failures to a triple combination of peg-IFN-α/RBV and either telaprevir o boceprevir will be due to a poor response to peg-IFN-α and RBV. Predictors of SVR to former triple combination will be also included the IL28B genotype. To determine the best SNP to predict a response to peg-IFN-α/RBV treatment, we studied the effect of different IL28B genetic variants on IFN-based therapy response. Not only will this data refine IL28b based predictions of treatment response, it may also inform studies of IL28b mechanism in HCV response.

Although the rs12979860 and rs8099917 genotypes have been independently associated with HCV treatment outcome, whether these SNPs play a causal role or are merely tagging other unknown causal variants remains to be elucidated. IL28B (which encodes IFN-λ3) up-regulates interferon-stimulated genes, similar to IFN-α and IFN-β, but via a different receptor. There is also evidence that IFN-λ 3 affects the adaptive immune response [[17]](#pone.0031016-Robek1), [[18]](#pone.0031016-Marcello1). Moreover, IFN-λ molecules inhibit HCV replication in vitro, and trials of IFN-λ1 in HCV-infected patients have demonstrated promising results that suggest a mechanistic link between IL28B variants and HCV treatment outcome [[19]](#pone.0031016-Balagopal1). Recently, four SNPs located in the promoter (rs4803219 and rs28416813), coding (rs8103142), and 3′-untranslated (rs4803217) regions of IL28B have been shown to be highly associated with spontaneous HCV clearance [[20]](#pone.0031016-diIulio1). Therefore, we assessed the influence of four causal IL28B variants (rs4803219, rs28416813, rs8103142, and rs4803217) on SVR to IFN-based therapies, and compared the relationships of these four causal SNPs with the tag IL28B variants rs12979860 and rs8099917.

We previously established the strong relationships between the rs8099917 G allele and treatment failure in our cohort of HCV/HIV-1 coinfected patients [[21]](#pone.0031016-Aparicio1). In the present study, was examined the effect of eight different IL28B genetic variants on IFN-based therapeutic response in these patients.

## Results

In a recent study, four causal SNPs (rs4803219, rs28416813, rs8103142, and rs4803217) were associated with the two tagging SNPs that were most strongly associated with spontaneous HCV clearance (rs12979860 and rs8099917) [[20]](#pone.0031016-diIulio1). In this study, the four causal SNPs and two tagged SNPs were genotyped along with two additional SNPs, rs11881222 and rs8113007, which are located in the second IL28B intron and 7.5 kb upstream of the IL28B gene, respectively ([Figure 1](#pone-0031016-g001)). These eight SNPs were genotyped in a cohort of 197 HCV/HIV-1 coinfected patients from our clinic unit who received standard combined peg-IFN-α/RBV therapy. The clinical characteristics and HCV treatment responses of the197 HCV/HIV-1 coinfected patients included in the study are summarized in [Table 1](#pone-0031016-t001). Most patients were on antiretroviral therapy (n = 196, 99.5%), and most patients had controlled HIV-1 replication (n = 169, 86%). Eighty-three patients (42%) responded successfully to HCV treatment (i.e., achieved SVR). When the HIV-1 viral load was compared in the HIV-1 viremic patients (n = 28, 14%), no significant differences were observed between SVRs and patients failing therapy (p = 0.1612, Mann-Whitney U test). Similarly, the length of time patients were under antiretroviral treatment did not impact HCV SVR (p = 0.3439, Mann-Whitney U test). HCV genotype was significantly associated with SVR in our study cohort (p = 7.2×10^−5^) ([Table 1](#pone-0031016-t001)). Unexpectedly, older patients were more likely to achieve HCV SVR (p = 0.0223) ([Table 1](#pone-0031016-t001)). Nevertheless, the difference between the means was very limited (48.5 vs 47 years). A moderate association with SVR was also observed with patient AST liver enzyme level (p = 0.0353), and HCV viral load (p = 0.0120). These four clinical parameters are known to be associated with HCV treatment response.

### Figure 1. IL28B gene locus and position of single nucleotide polymorphisms (SNPs) analyzed in this study.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e4d7/3273458/50ddb0dd051b/pone.0031016.g001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3273458_pone.0031016.g001.jpg)

### Table 1. Clinical characteristics of patients with chronic HIV-1 and HCV coinfection who were treated with peg-IFN-α/RBV therapy.

|   | SVR | NSVR | p-value |
| --- | --- | --- | --- |
| Patients, n (%) | 83 (42%) | 114 (58%) | - |
| Age (mean ± SEM) | 48.57±0.5582 | 46.96±0.5523 | 0.0223 |
| Sex, n (%) |   |   | 1.0000 |
| Female | 27(44%) | 36 (56%) |   |
| Male | 56 (42%) | 78 (58%) |   |
| CD4+ (cell count/ml) (mean ± SEM) | 597.0±28.34 | 599.8±24.93 | 0.9866 |
| HCV genotype, n (%) |   |   | 7.2×10−5 |
| 1 | 35 (33%) | 70 (67%) |   |
| 3 | 41 (66%) | 21 (34%) |   |
| 4 | 11 (32%) | 23 (68%) |   |
| Other/Unknown | 0 | 2 |   |
| ALT (U/l) (mean ± SEM) | 87.78±8.22 | 95.19±7.30 | ns |
| AST (U/l) (mean ± SEM) | 59.31±4.95 | 67.88±4.04 | 0.0353 |
| HCV RNA (IU/ml) (mean ± SEM) | 5.819±0.079 | 6.043±0.048 | 0.0120 |
| HCV-RNA>5×105 IU/ml, n (%) | 43 (49%) | 83 (71%) | 0.0047 |
| Undetectable HIV-1 RNA, n (%) | 68 (78%) | 93 (80%) | ns |

Table 1 Caption: ALT: alanine aminotransferase; AST: aspartate aminotransferase; HCV: hepatitis C virus; HIV-1: human immunodeficiency virus type 1; NSVR: non-sustained virological response; SVR: sustained virological response. Age, Mann-Whitney U test; Sex, Chi-squared test; CD4+T cell count, Mann-Whitney U test; HCV genotype Chi-squared test; ALT and AST, Mann-Whitney U test; HCV RNA, unpaired t-test; Undetectable HIV-1, Chi-squared test.

All major IL28B single-allele SNPs were highly associated with SVRs ([Table 2](#pone-0031016-t002)). Three causal major alleles, rs28416813, rs8103142, and rs4803217, demonstrated the highest individual associations with SVRs (OR = 2.9, 95% CI = 1.8–4.6, p = 6.6×10^−6^) ([Table 2](#pone-0031016-t002)). As shown in [Table 3](#pone-0031016-t003), genotype distributions of all tested IL28B SNPs were significantly different among patients experiencing an SVR to peg-IFN-α/RBV treatment. Again, the causal SNPs rs28416813, rs8103142, and rs4803217 demonstrated the highest associations with SVRs (OR = 3.7, 95% CI = 2.0–6.7, p = 1.3×10^−5^) ([Table 3](#pone-0031016-t003)). The four causal variants were in high linkage disequilibrium among themselves (r^2^≥0.94) and with the tag rs12979860 variant (r^2^≥0.92) ([Figure 2](#pone-0031016-g002)). Importantly, rs8099917 was in low linkage disequilibrium with the four causal variants (r^2^≤0.45) and with the rs12979860 variant (r^2^ = 0.45), demonstrating that this SNP was only moderately linked to the causal SNPs in the IL28B gene. Notably, the tag rs8113007 variant (which is separated from the rs8099917 SNP by only 72 nucleotides) displayed intermediate linkage with the tag rs12979860 (r^2^ = 0.78) and the causal IL28B gene SNPs (r^2^ = 0.77–0.82). The rs11881222 SNP, located in the second IL28B intron ([Figure 1](#pone-0031016-g001)), was highly linked to both the causal IL28B SNPs (r^2^≥0.92) and the tag rs12979860 variant (r^2^ = 0.92) ([Figure 2](#pone-0031016-g002)).

### Table 2. Association of individual IL28B haplotypes with sustained virological response to peg-IFN-α/RBV therapy.

| SNP | Associated Allele | OR (95% CI) | p-value |
| --- | --- | --- | --- |
| rs4803217 | C | 2.9 (1.8–4.6) | 6.7×10−6 |
| rs11881222 | A | 2.8 (1.7–4.5) | 1.6×10−5 |
| rs8103142 | T | 2.9 (1.8–4.6) | 6.7×10−6 |
| rs28416813 | C | 2.9 (1.8–4.6) | 6.7×10−6 |
| rs4803219 | C | 2.8 (1.7–4.5) | 1.6×10−5 |
| rs12979860 | C | 2.8 (1.8–4.6) | 9.9×10−6 |
| rs8113007 | A | 2.3 (1.4–3.6) | 4.0×10−4 |
| rs8099917 | T | 2.7 (1.6–4.6) | 2.0×10−4 |
### Table 3. Association of IL28B genotype with sustained virological response to peg-IFN-α/RBV therapy.

|   | Allele | SVR |  |  | NSVR |  |  | MAF |  |   |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| SNP | (1/2) | 1/1 | 1/2 | 2/2 | 1/1 | 1/2 | 2/2 | SVR | NSVR | OR (95% CI) | p-value |
| rs4803217 | C/A | 56 | 23 | 4 | 41 | 55 | 18 | 0.19 | 0.40 | 3.7 (2.0–6.7) | 1.3×10−5 |
| rs11881222 | A/G | 57 | 22 | 4 | 44 | 53 | 17 | 0.18 | 0.38 | 3.5 (1.9–6.3) | 3.0×10−5 |
| rs8103142 | T/C | 56 | 23 | 4 | 41 | 55 | 18 | 0.19 | 0.40 | 3.7 (2.0–6.7) | 1.3×10−5 |
| rs28416813 | C/G | 56 | 23 | 4 | 41 | 55 | 18 | 0.19 | 0.40 | 3.7 (2.0–6.7) | 1.3×10−5 |
| rs4803219 | C/T | 57 | 22 | 4 | 44 | 53 | 17 | 0.18 | 0.38 | 3.5 (1.9–6.3) | 3.0×10−5 |
| rs12979860 | C/T | 56 | 23 | 4 | 42 | 54 | 18 | 0.19 | 0.39 | 3.6 (2.0–6.5) | 2.2×10−5 |
| rs8113007 | A/T | 55 | 20 | 8 | 43 | 54 | 17 | 0.22 | 0.39 | 3.2 (1.8–5.9) | 7.5×10−5 |
| rs8099917 | T/G | 66 | 13 | 4 | 60 | 44 | 10 | 0.13 | 0.28 | 3.5 (1.8–6.8) | 1.0×10−4 |

Table 3 Caption: MAF: minor allele frequency; NSVR: non-sustained virological response; SVR: sustained virological response. Numbers reflect the number of patients with indicated genotype.

### Figure 2. Pairwise linkage disequilibrium (LD) plot for the eight single nucleotide polymorphisms (SNPs) analyzed in this study.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e4d7/3273458/c70ca04427eb/pone.0031016.g002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3273458_pone.0031016.g002.jpg)

The linkage disequilibrium between the four candidate casual SNPs (rs4803217, rs8103142, rs28416813 and rs4803219) and the tagging SNPs is shown. An empty square represents r2 = 1.0.

Haplotypes carrying major alleles at IL28B were most strongly associated with an SVR to peg-IFN-α/RBV therapy (OR = 2.5, 95% CI = 1.6–4.0, p = 4.0×10^−5^) ([Table 4](#pone-0031016-t004)). These major haplotypes were visibly augmented in patients experiencing an SVR ([Tables 2](#pone-0031016-t002) and [4](#pone-0031016-t004)). Nevertheless, the prognostic value of the eight haplotypes together was not higher than the ORs obtained when the different haplotypes were analyzed individually ([Table 2](#pone-0031016-t002)). Importantly, homozygosity for the three of the four causal IL28B gene SNPs, rs28416813, rs8103142, and rs4803217, was completely linked to the rs12979860 major CC genotype (r^2^ = 1.0). In contrast, the rs8099917 major TT genotype demonstrated a lower correlation with homozygosity of these three causal IL28B gene SNPs (r^2^ = 0.83).

### Table 4. IL28B haplotype association with sustained virological response to peg-IFN-α/RBV therapy.

| rs4803217 | rs11881222 | rs8103142 | rs28416813 | rs4803219 | rs12979860 | rs8113007 | rs8099917 | OR (95% CI) | p-value |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| C | A | T | C | C | C | A | T | 2.5 (1.6–4.0) | 4.0×10−5 |
| A | G | C | G | T | T | T | G | 0.3 (0.1–0.5) | 1.1×10−5 |
| A | G | C | G | T | T | T | T | 0.9 (0.5–1.8) | 0.8368 |
| C | A | T | C | C | C | T | G | 2.8 (0.7–11.4) | 0.1351 |
| A | A | C | G | C | T | A | T | 0.3 (0.0–3.1) | 0.3131 |

Table 4 Caption: Major alleles are shown in bold.

## Discussion

Prior work has documented the contribution of different IL28B-associated SNPs to spontaneous HCV clearance [[7]](#pone.0031016-Ge1), [[8]](#pone.0031016-Suppiah1), [[9]](#pone.0031016-Tanaka1), [[10]](#pone.0031016-Rauch1), [[22]](#pone.0031016-Thomas1). Specifically, di Iulio et al. report the strong association of four SNPs located in the promoter, coding, and 3′-untranslated regions of IL28B with spontaneous HCV clearance [[20]](#pone.0031016-diIulio1). However, this study did not focus on patients treated with IFN-based therapies. In the present study, we tested the influence of four causal IL28B variants (rs4803219, rs28416813, rs8103142, and rs4803217) on SVR to peg-IFN-α/RBV therapy, and compared the relationships of these four causal SNPs with the tag IL28B variants rs12979860 and rs8099917.

In the present cohort of HCV/HIV-1 coinfected patients, three major causal IL28B variant haplotypes (rs28416813, rs8103142, and rs4803217, located in the promoter, coding, and 3′-untranslated regions of IL28B) were highly associated with SVR to peg-IFN-α/RBV therapy. Moreover, these three haplotypes, which were in perfect linkage equilibrium among themselves (r^2^ = 1.0), had a higher odds ratio regarding HCV clearance than the three tag IL28B SNPs tested here (rs8099917, rs8113007, and rs12979860). Two of these tag SNPs, rs8099917 and rs12979860, demonstrated the strongest associations with HCV clearance in the above-mentioned genome-wide association studies and are currently being introduced in clinical algorithms to predict SVR to IFN-based therapies. Another important finding of our study is the strong linkage of the tag rs12979860 variant with the IL28B causal SNPs. Homozygosity for the tag rs12979860 variant completely tagged (r^2^ = 1.0) homozygosity for three major causal SNPs: rs28416813, rs8103142, and rs4803217. This result suggests that rs12979860 may be a better marker for IL28B phenotype than rs8099917 or rs8113007. These findings extend those of di Iulio et al., confirming that rs12979860 can be a better predictor for HCV clearance, as well as underscoring the strong association of causal IL28B SNPs with HCV clearance [[20]](#pone.0031016-diIulio1). Most notably, to our knowledge this is the first study to investigate the association of IL28B tag and causal variants with patient response to peg-IFN-α/RBV therapy. These results provide compelling evidence for the relationship between IL28B tag and causal variants, and suggest possible involvement of the IL28B gene product, IFN-λ3, in SVR to IFN-based therapies. However, some limitations are worth noting. First, our study was restricted to an ethnically and epidemiologically homogeneous cohort of HCV/HIV-1 coinfected patients attending our clinic. Future work should include other ethnic and patient cohorts. Second, as previously suggested [[20]](#pone.0031016-diIulio1), functional studies are needed to precisely identify the host gene variants implicated in the control of HCV infection. Two studies found higher levels of IFN-λ2 and IFN-λ3 messenger RNAs in PBMCs from healthy individuals carrying the homozygous rs8099917 genotype associated with HCV clearance [[8]](#pone.0031016-Suppiah1), [[9]](#pone.0031016-Tanaka1). However, another study observed no association between hepatic expression of IFN-λ2 and IFN-λ3 and rs8099917 genotype [[23]](#pone.0031016-Honda1).

In short, the data shown here strongly support the association of tag and causal IL28B variants with HCV clearance after IFN-based therapy. Nevertheless, there were patients carrying alleles associated with clearance who did not clear the virus, as well as patients that carrying alleles not associated with clearance who were able to clear the virus. These results suggest that further studies should be conducted to investigate IL28B genetics, as well as other genetic factors associated with HCV clearance.

## Materials and Methods

### Patients

One hundred ninety-seven Caucasian HCV/HIV-1 coinfected patients from our HIV clinic unit who had a standard course of treatment with peg-IFN-α plus RBV with known virological response at 24 weeks post-treatment were included in this study. Treatment success (i.e., achieved SVR) was defined as undetectable plasma HCV RNA using a sensitive reverse transcriptase PCR assay 24 weeks after treatment cessation. HCV genotype, HCV viral load, HIV-1 viral load, CD4+ T cell count and liver enzyme levels.

Written informed consent was obtained from each patient who participated in the study. Similarly, ethical approval was obtained from our Institutional Review Board (Hospital Universitari Germans Trias i Pujol).

### DNA collection and extraction

Genomic DNA was extracted from peripheral blood mononuclear cells (PBMCs) using the QuickExtract DNA Extraction Protocol (EPICENTRE Biotechnologies) as previously described [[21]](#pone.0031016-Aparicio1).

### Genotyping of SNPs

SNP genotyping was performed by PCR amplification and direct PCR sequencing. The rs8099917 and rs8113007 variants were PCR amplified and sequenced with oligonucleotides rs8099917–128 (*5′-GTGCATATGTTTTCTGAC-3′*, sense) and rs8099917–556 (*5′-GAGGCCCCTCACCCATGC-3′*, antisense). The PCR amplification mixture contained 5 µl of PBMC genomic DNA solution, 10 pmol of each oligonucleotide, 200 µM deoxyribonucleoside triphosphates (dNTPs), 2 mM MgSO4, 1× high-fidelity PCR buffer (Invitrogen), and 0.25 U Platinum Taq DNA polymerase (Invitrogen) in a total reaction volume of 50 µl. Cycling parameters were one cycle of denaturation at 94°C for 2 min; and 40 cycles of denaturation at 95°C for 30 s, annealing at 55°C for 30 s, and extension at 68°C for 30 s. Extension was followed by a 7-min incubation at 68°C. The resulting 430-nucleotide PCR product was sequenced using two flanking PCR oligonucleotides (rs8099917–128 and rs8099917–556) in conjunction with the Big Dye v3.1 kit and the 3100 DNA sequencing system (Applied Biosystems) as described previously [[21]](#pone.0031016-Aparicio1). Sequence alignment and editing were performed using the Sequencer version 4.1 (GeneCodes) software program. The rs12979860 SNP was amplified and sequenced with oligonucleotides rs12979860–13 (*5′-GACGAGAGGGCGTTAGAGC-3′*, sense) and rs12979860–595 (*5′-GAGGGACCGCTACGTAAGTC-3′*, antisense). PCR and sequence conditions were identical to those described above. To genotype the rs4803219, rs28416813, rs8103142, rs11881222, and rs4803217 variants, a 3308-nucleotide genomic fragment was amplified with oligonucleotides IL-28-3308F (*5′-GAGCAGGTGGAATCCTCTTG-3′*, sense) and IL-28-3308R (*5′-AGCAGGCACCTTGAAATGTC-3′*, antisense). PCR conditions were as described above, but with PCR extension for 3 min at 68°C. Sequencing oligonucleotides for this fragment were IL-28-3308F, IL-28-3308R, rs8103142RV (*5′-CCATCCTCCCAGCAGTTAACCTCCC-3′*, sense), 2SNPRV (*5′-TCCCTCCAGCTGCTCATCTGGC-3′*, sense), and rs4803217RV (*5′-CCCAGACAGCCCCTGACCCA-3′*, sense).

### Statistical analysis

The Mann-Whitney U test, unpaired *t*-test, and Chi-squared tests were used to analyze treatment baseline covariates. Chi-squared tests and contingency tables were used to asses host genetic associations and to calculate the p-values, odds ratios, and 95% confidence intervals. The above Statistical analyses were performed using GraphPad Prism version 4.00 for Windows (San Diego, CA, USA). Linkage disequilibrium and haplotype analyses were performed using Haploview 4.2 [[24]](#pone.0031016-Barrett1).

## Acknowledgments

We thank Ester Ballana for helpful and fruitful discussions.

## Footnotes

## References

1. Fried MW, Shiffman M, Sterling RK, Weinstein J, Crippin J, et al. A multicenter, randomized trial of daily high-dose interferon-alfa 2b for the treatment of chronic hepatitis c: pretreatment stratification by viral burden and genotype. Am J Gastroenterol. 2000;95:3225–3229. doi: 10.1111/j.1572-0241.2000.03433.x.  [DOI](https://doi.org/10.1111/j.1572-0241.2000.03433.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11095346/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Gastroenterol&title=A%20multicenter,%20randomized%20trial%20of%20daily%20high-dose%20interferon-alfa%202b%20for%20the%20treatment%20of%20chronic%20hepatitis%20c:%20pretreatment%20stratification%20by%20viral%20burden%20and%20genotype.&author=MW%20Fried&author=M%20Shiffman&author=RK%20Sterling&author=J%20Weinstein&author=J%20Crippin&volume=95&publication_year=2000&pages=3225-3229&pmid=11095346&doi=10.1111/j.1572-0241.2000.03433.x&)

2. Manns MP, McHutchison JG, Gordon SC, Rustgi VK, Shiffman M, et al. Peginterferon alfa-2b plus ribavirin compared with interferon alfa-2b plus ribavirin for initial treatment of chronic hepatitis C: a randomised trial. Lancet. 2001;358:958–965. doi: 10.1016/s0140-6736(01)06102-5.  [DOI](https://doi.org/10.1016/s0140-6736(01)06102-5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11583749/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Peginterferon%20alfa-2b%20plus%20ribavirin%20compared%20with%20interferon%20alfa-2b%20plus%20ribavirin%20for%20initial%20treatment%20of%20chronic%20hepatitis%20C:%20a%20randomised%20trial.&author=MP%20Manns&author=JG%20McHutchison&author=SC%20Gordon&author=VK%20Rustgi&author=M%20Shiffman&volume=358&publication_year=2001&pages=958-965&pmid=11583749&doi=10.1016/s0140-6736(01)06102-5&)

3. Fried MW, Shiffman ML, Reddy KR, Smith C, Marinos G, et al. Peginterferon alfa-2a plus ribavirin for chronic hepatitis C virus infection. N Engl J Med. 2002;347:975–982. doi: 10.1056/NEJMoa020047.  [DOI](https://doi.org/10.1056/NEJMoa020047) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12324553/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Peginterferon%20alfa-2a%20plus%20ribavirin%20for%20chronic%20hepatitis%20C%20virus%20infection.&author=MW%20Fried&author=ML%20Shiffman&author=KR%20Reddy&author=C%20Smith&author=G%20Marinos&volume=347&publication_year=2002&pages=975-982&pmid=12324553&doi=10.1056/NEJMoa020047&)

4. Hadziyannis SJ, Sette H, Jr, Morgan TR, Balan V, Diago M, et al. Peginterferon-alpha2a and ribavirin combination therapy in chronic hepatitis C: a randomized study of treatment duration and ribavirin dose. Ann Intern Med. 2004;140:346–355. doi: 10.7326/0003-4819-140-5-200403020-00010.  [DOI](https://doi.org/10.7326/0003-4819-140-5-200403020-00010) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14996676/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Intern%20Med&title=Peginterferon-alpha2a%20and%20ribavirin%20combination%20therapy%20in%20chronic%20hepatitis%20C:%20a%20randomized%20study%20of%20treatment%20duration%20and%20ribavirin%20dose.&author=SJ%20Hadziyannis&author=H%20Sette&author=TR%20Morgan&author=V%20Balan&author=M%20Diago&volume=140&publication_year=2004&pages=346-355&pmid=14996676&doi=10.7326/0003-4819-140-5-200403020-00010&)

5. Gao B, Hong F, Radaeva S. Host factors and failure of interferon-alpha treatment in hepatitis C virus. Hepatology. 2004;39:880–890. doi: 10.1002/hep.20139.  [DOI](https://doi.org/10.1002/hep.20139) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15057887/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hepatology&title=Host%20factors%20and%20failure%20of%20interferon-alpha%20treatment%20in%20hepatitis%20C%20virus.&author=B%20Gao&author=F%20Hong&author=S%20Radaeva&volume=39&publication_year=2004&pages=880-890&pmid=15057887&doi=10.1002/hep.20139&)

6. Walsh MJ, Jonsson JR, Richardson MM, Lipka GM, Purdie DM, et al. Non-response to antiviral therapy is associated with obesity and increased hepatic expression of suppressor of cytokine signalling 3 (SOCS-3) in patients with chronic hepatitis C, viral genotype 1. Gut. 2006;55:529–535. doi: 10.1136/gut.2005.069674.  [DOI](https://doi.org/10.1136/gut.2005.069674) | [PMC free article](/articles/PMC1856185/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16299039/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gut&title=Non-response%20to%20antiviral%20therapy%20is%20associated%20with%20obesity%20and%20increased%20hepatic%20expression%20of%20suppressor%20of%20cytokine%20signalling%203%20(SOCS-3)%20in%20patients%20with%20chronic%20hepatitis%20C,%20viral%20genotype%201.&author=MJ%20Walsh&author=JR%20Jonsson&author=MM%20Richardson&author=GM%20Lipka&author=DM%20Purdie&volume=55&publication_year=2006&pages=529-535&pmid=16299039&doi=10.1136/gut.2005.069674&)

7. Ge D, Fellay J, Thompson AJ, Simon JS, Shianna KV, et al. Genetic variation in IL28B predicts hepatitis C treatment-induced viral clearance. Nature. 2009;461:399–401. doi: 10.1038/nature08309.  [DOI](https://doi.org/10.1038/nature08309) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19684573/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Genetic%20variation%20in%20IL28B%20predicts%20hepatitis%20C%20treatment-induced%20viral%20clearance.&author=D%20Ge&author=J%20Fellay&author=AJ%20Thompson&author=JS%20Simon&author=KV%20Shianna&volume=461&publication_year=2009&pages=399-401&pmid=19684573&doi=10.1038/nature08309&)

8. Suppiah V, Moldovan M, Ahlenstiel G, Berg T, Weltman M, et al. IL28B is associated with response to chronic hepatitis C interferon-alpha and ribavirin therapy. Nat Genet. 2009;41:1100–1104. doi: 10.1038/ng.447.  [DOI](https://doi.org/10.1038/ng.447) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19749758/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=IL28B%20is%20associated%20with%20response%20to%20chronic%20hepatitis%20C%20interferon-alpha%20and%20ribavirin%20therapy.&author=V%20Suppiah&author=M%20Moldovan&author=G%20Ahlenstiel&author=T%20Berg&author=M%20Weltman&volume=41&publication_year=2009&pages=1100-1104&pmid=19749758&doi=10.1038/ng.447&)

9. Tanaka Y, Nishida N, Sugiyama M, Kurosaki M, Matsuura K, et al. Genome-wide association of IL28B with response to pegylated interferon-alpha and ribavirin therapy for chronic hepatitis C. Nat Genet. 2009;41:1105–1109. doi: 10.1038/ng.449.  [DOI](https://doi.org/10.1038/ng.449) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19749757/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Genome-wide%20association%20of%20IL28B%20with%20response%20to%20pegylated%20interferon-alpha%20and%20ribavirin%20therapy%20for%20chronic%20hepatitis%20C.&author=Y%20Tanaka&author=N%20Nishida&author=M%20Sugiyama&author=M%20Kurosaki&author=K%20Matsuura&volume=41&publication_year=2009&pages=1105-1109&pmid=19749757&doi=10.1038/ng.449&)

10. Rauch A, Kutalik Z, Descombes P, Cai T, Di Iulio J, et al. Genetic variation in IL28B is associated with chronic hepatitis C and treatment failure: a genome-wide association study. Gastroenterology. 2010;138:1338–1345, 1345 e1331–1337. doi: 10.1053/j.gastro.2009.12.056.  [DOI](https://doi.org/10.1053/j.gastro.2009.12.056) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20060832/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastroenterology&title=Genetic%20variation%20in%20IL28B%20is%20associated%20with%20chronic%20hepatitis%20C%20and%20treatment%20failure:%20a%20genome-wide%20association%20study.&author=A%20Rauch&author=Z%20Kutalik&author=P%20Descombes&author=T%20Cai&author=J%20Di%20Iulio&volume=138&publication_year=2010&pages=1338-1345,%201345%20e1331%E2%80%931337&pmid=20060832&doi=10.1053/j.gastro.2009.12.056&)

11. Rallon NI, Naggie S, Benito JM, Medrano J, Restrepo C, et al. Association of a single nucleotide polymorphism near the interleukin-28B gene with response to hepatitis C therapy in HIV/hepatitis C virus-coinfected patients. AIDS. 2010;24:F23–29. doi: 10.1097/QAD.0b013e3283391d6d.  [DOI](https://doi.org/10.1097/QAD.0b013e3283391d6d) | [PMC free article](/articles/PMC4892373/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20389235/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=AIDS&title=Association%20of%20a%20single%20nucleotide%20polymorphism%20near%20the%20interleukin-28B%20gene%20with%20response%20to%20hepatitis%20C%20therapy%20in%20HIV/hepatitis%20C%20virus-coinfected%20patients.&author=NI%20Rallon&author=S%20Naggie&author=JM%20Benito&author=J%20Medrano&author=C%20Restrepo&volume=24&publication_year=2010&pages=F23-29&pmid=20389235&doi=10.1097/QAD.0b013e3283391d6d&)

12. Pineda JA, Caruz A, Rivero A, Neukam K, Salas I, et al. Prediction of response to pegylated interferon plus ribavirin by IL28B gene variation in patients coinfected with HIV and hepatitis C virus. Clin Infect Dis. 2010;51:788–795. doi: 10.1086/656235.  [DOI](https://doi.org/10.1086/656235) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20804372/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Infect%20Dis&title=Prediction%20of%20response%20to%20pegylated%20interferon%20plus%20ribavirin%20by%20IL28B%20gene%20variation%20in%20patients%20coinfected%20with%20HIV%20and%20hepatitis%20C%20virus.&author=JA%20Pineda&author=A%20Caruz&author=A%20Rivero&author=K%20Neukam&author=I%20Salas&volume=51&publication_year=2010&pages=788-795&pmid=20804372&doi=10.1086/656235&)

13. Rallon NI, Soriano V, Naggie S, Restrepo C, Goldstein D, et al. IL28B gene polymorphisms and viral kinetics in HIV/hepatitis C virus-coinfected patients treated with pegylated interferon and ribavirin. AIDS. 2011;25:1025–1033. doi: 10.1097/QAD.0b013e3283471cae.  [DOI](https://doi.org/10.1097/QAD.0b013e3283471cae) | [PMC free article](/articles/PMC4892378/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21505315/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=AIDS&title=IL28B%20gene%20polymorphisms%20and%20viral%20kinetics%20in%20HIV/hepatitis%20C%20virus-coinfected%20patients%20treated%20with%20pegylated%20interferon%20and%20ribavirin.&author=NI%20Rallon&author=V%20Soriano&author=S%20Naggie&author=C%20Restrepo&author=D%20Goldstein&volume=25&publication_year=2011&pages=1025-1033&pmid=21505315&doi=10.1097/QAD.0b013e3283471cae&)

14. Labarga P, Barreiro P, Mira JA, Vispo E, Rallon N, et al. Impact of IL28B Polymorphisms on Response to Peginterferon plus Ribavirin in HIV-HCV Coinfected Patients with Prior Non-Response or Relapse. AIDS. 2011;25:1131–1133. doi: 10.1097/QAD.0b013e3283471d83.  [DOI](https://doi.org/10.1097/QAD.0b013e3283471d83) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21537116/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=AIDS&title=Impact%20of%20IL28B%20Polymorphisms%20on%20Response%20to%20Peginterferon%20plus%20Ribavirin%20in%20HIV-HCV%20Coinfected%20Patients%20with%20Prior%20Non-Response%20or%20Relapse.&author=P%20Labarga&author=P%20Barreiro&author=JA%20Mira&author=E%20Vispo&author=N%20Rallon&volume=25&publication_year=2011&pages=1131-1133&pmid=21537116&doi=10.1097/QAD.0b013e3283471d83&)

15. Clausen LN, Weis N, Astvad K, Schonning K, Fenger M, et al. Interleukin-28B polymorphisms are associated with hepatitis C virus clearance and viral load in a HIV-1-infected cohort. J Viral Hepat. 2011;18:e66–74. doi: 10.1111/j.1365-2893.2010.01392.x.  [DOI](https://doi.org/10.1111/j.1365-2893.2010.01392.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21070502/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Viral%20Hepat&title=Interleukin-28B%20polymorphisms%20are%20associated%20with%20hepatitis%20C%20virus%20clearance%20and%20viral%20load%20in%20a%20HIV-1-infected%20cohort.&author=LN%20Clausen&author=N%20Weis&author=K%20Astvad&author=K%20Schonning&author=M%20Fenger&volume=18&publication_year=2011&pages=e66-74&pmid=21070502&doi=10.1111/j.1365-2893.2010.01392.x&)

16. Dayyeh BK, Gupta N, Sherman KE, de Bakker PI, Chung RT. IL28B Alleles Exert an Additive Dose Effect When Applied to HCV-HIV Coinfected Persons Undergoing Peginterferon and Ribavirin Therapy. PLoS One. 2011;6:e25753. doi: 10.1371/journal.pone.0025753.  [DOI](https://doi.org/10.1371/journal.pone.0025753) | [PMC free article](/articles/PMC3189209/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22003405/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=IL28B%20Alleles%20Exert%20an%20Additive%20Dose%20Effect%20When%20Applied%20to%20HCV-HIV%20Coinfected%20Persons%20Undergoing%20Peginterferon%20and%20Ribavirin%20Therapy.&author=BK%20Dayyeh&author=N%20Gupta&author=KE%20Sherman&author=PI%20de%20Bakker&author=RT%20Chung&volume=6&publication_year=2011&pages=e25753&pmid=22003405&doi=10.1371/journal.pone.0025753&)

17. Robek MD, Boyd BS, Chisari FV. Lambda interferon inhibits hepatitis B and C virus replication. J Virol. 2005;79:3851–3854. doi: 10.1128/JVI.79.6.3851-3854.2005.  [DOI](https://doi.org/10.1128/JVI.79.6.3851-3854.2005) | [PMC free article](/articles/PMC1075734/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15731279/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Virol&title=Lambda%20interferon%20inhibits%20hepatitis%20B%20and%20C%20virus%20replication.&author=MD%20Robek&author=BS%20Boyd&author=FV%20Chisari&volume=79&publication_year=2005&pages=3851-3854&pmid=15731279&doi=10.1128/JVI.79.6.3851-3854.2005&)

18. Marcello T, Grakoui A, Barba-Spaeth G, Machlin ES, Kotenko SV, et al. Interferons alpha and lambda inhibit hepatitis C virus replication with distinct signal transduction and gene regulation kinetics. Gastroenterology. 2006;131:1887–1898. doi: 10.1053/j.gastro.2006.09.052.  [DOI](https://doi.org/10.1053/j.gastro.2006.09.052) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17087946/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastroenterology&title=Interferons%20alpha%20and%20lambda%20inhibit%20hepatitis%20C%20virus%20replication%20with%20distinct%20signal%20transduction%20and%20gene%20regulation%20kinetics.&author=T%20Marcello&author=A%20Grakoui&author=G%20Barba-Spaeth&author=ES%20Machlin&author=SV%20Kotenko&volume=131&publication_year=2006&pages=1887-1898&pmid=17087946&doi=10.1053/j.gastro.2006.09.052&)

19. Balagopal A, Thomas DL, Thio CL. IL28B and the control of hepatitis C virus infection. Gastroenterology. 2010;139:1865–1876. doi: 10.1053/j.gastro.2010.10.004.  [DOI](https://doi.org/10.1053/j.gastro.2010.10.004) | [PMC free article](/articles/PMC3072961/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20950615/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastroenterology&title=IL28B%20and%20the%20control%20of%20hepatitis%20C%20virus%20infection.&author=A%20Balagopal&author=DL%20Thomas&author=CL%20Thio&volume=139&publication_year=2010&pages=1865-1876&pmid=20950615&doi=10.1053/j.gastro.2010.10.004&)

20. di Iulio J, Ciuffi A, Fitzmaurice K, Kelleher D, Rotger M, et al. Estimating the net contribution of interleukin-28B variation to spontaneous hepatitis C virus clearance. Hepatology. 2011;53:1446–1454. doi: 10.1002/hep.24263.  [DOI](https://doi.org/10.1002/hep.24263) | [PMC free article](/articles/PMC3128709/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21360716/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hepatology&title=Estimating%20the%20net%20contribution%20of%20interleukin-28B%20variation%20to%20spontaneous%20hepatitis%20C%20virus%20clearance.&author=J%20di%20Iulio&author=A%20Ciuffi&author=K%20Fitzmaurice&author=D%20Kelleher&author=M%20Rotger&volume=53&publication_year=2011&pages=1446-1454&pmid=21360716&doi=10.1002/hep.24263&)

21. Aparicio E, Parera M, Franco S, Perez-Alvarez N, Tural C, et al. IL28B SNP rs8099917 is strongly associated with pegylated interferon-alpha and ribavirin therapy treatment failure in HCV/HIV-1 coinfected patients. PLoS One. 2010;5:e13771. doi: 10.1371/journal.pone.0013771.  [DOI](https://doi.org/10.1371/journal.pone.0013771) | [PMC free article](/articles/PMC2966433/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21048934/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=IL28B%20SNP%20rs8099917%20is%20strongly%20associated%20with%20pegylated%20interferon-alpha%20and%20ribavirin%20therapy%20treatment%20failure%20in%20HCV/HIV-1%20coinfected%20patients.&author=E%20Aparicio&author=M%20Parera&author=S%20Franco&author=N%20Perez-Alvarez&author=C%20Tural&volume=5&publication_year=2010&pages=e13771&pmid=21048934&doi=10.1371/journal.pone.0013771&)

22. Thomas DL, Thio CL, Martin MP, Qi Y, Ge D, et al. Genetic variation in IL28B and spontaneous clearance of hepatitis C virus. Nature. 2009;461:798–801. doi: 10.1038/nature08463.  [DOI](https://doi.org/10.1038/nature08463) | [PMC free article](/articles/PMC3172006/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19759533/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Genetic%20variation%20in%20IL28B%20and%20spontaneous%20clearance%20of%20hepatitis%20C%20virus.&author=DL%20Thomas&author=CL%20Thio&author=MP%20Martin&author=Y%20Qi&author=D%20Ge&volume=461&publication_year=2009&pages=798-801&pmid=19759533&doi=10.1038/nature08463&)

23. Honda M, Sakai A, Yamashita T, Nakamoto Y, Mizukoshi E, et al. Hepatic ISG expression is associated with genetic variation in interleukin 28B and the outcome of IFN therapy for chronic hepatitis C. Gastroenterology. 2010;139:499–509. doi: 10.1053/j.gastro.2010.04.049.  [DOI](https://doi.org/10.1053/j.gastro.2010.04.049) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20434452/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastroenterology&title=Hepatic%20ISG%20expression%20is%20associated%20with%20genetic%20variation%20in%20interleukin%2028B%20and%20the%20outcome%20of%20IFN%20therapy%20for%20chronic%20hepatitis%20C.&author=M%20Honda&author=A%20Sakai&author=T%20Yamashita&author=Y%20Nakamoto&author=E%20Mizukoshi&volume=139&publication_year=2010&pages=499-509&pmid=20434452&doi=10.1053/j.gastro.2010.04.049&)

24. Barrett JC, Fry B, Maller J, Daly MJ. Haploview: analysis and visualization of LD and haplotype maps. Bioinformatics. 2005;21:263–265. doi: 10.1093/bioinformatics/bth457.  [DOI](https://doi.org/10.1093/bioinformatics/bth457) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15297300/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=Haploview:%20analysis%20and%20visualization%20of%20LD%20and%20haplotype%20maps.&author=JC%20Barrett&author=B%20Fry&author=J%20Maller&author=MJ%20Daly&volume=21&publication_year=2005&pages=263-265&pmid=15297300&doi=10.1093/bioinformatics/bth457&)
