# Rare versus common variants in pharmacogenetics: SLCO1B1 variation and methotrexate disposition

## Metadata
**Authors:** Laura B Ramsey, Gitte H Bruun, Wenjian Yang, Lisa R Treviño, Selina Vattathil, Paul Scheet, Cheng Cheng, Gary L Rosner, Kathleen M Giacomini, Yiping Fan, Alex Sparreboom, Torben S Mikkelsen, Thomas J Corydon, Ching-Hon Pui, William E Evans, Mary V Relling
**Journal:** Genome Research
**Date:** 2012 Jan
**DOI:** [10.1101/gr.129668.111](https://doi.org/10.1101/gr.129668.111)
**PMID:** 22147369
**PMCID:** PMC3246196
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3246196/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC3246196/pdf/1.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC3246196/pdf/1.pdf)

## Abstract

Methotrexate is used to treat autoimmune diseases and malignancies, including acute lymphoblastic leukemia (ALL). Inter-individual variation in clearance of methotrexate results in heterogeneous systemic exposure, clinical efficacy, and toxicity. In a genome-wide association study of children with ALL, we identified SLCO1B1 as harboring multiple common polymorphisms associated with methotrexate clearance. The extent of influence of rare versus common variants on pharmacogenomic phenotypes remains largely unexplored. We tested the hypothesis that rare variants in SLCO1B1 could affect methotrexate clearance and compared the influence of common versus rare variants in addition to clinical covariates on clearance. From deep resequencing of SLCO1B1 exons in 699 children, we identified 93 SNPs, 15 of which were non-synonymous (NS). Three of these NS SNPs were common, with a minor allele frequency (MAF) >5%, one had low frequency (MAF 1%–5%), and 11 were rare (MAF <1%). NS SNPs (common or rare) predicted to be functionally damaging were more likely to be found among patients with the lowest methotrexate clearance than patients with high clearance. We verified lower function in vitro of four SLCO1B1 haplotypes that were associated with reduced methotrexate clearance. In a multivariate stepwise regression analysis adjusting for other genetic and non-genetic covariates, SLCO1B1 variants accounted for 10.7% of the population variability in clearance. Of that variability, common NS variants accounted for the majority, but rare damaging NS variants constituted 17.8% of SLCO1B1's effects (1.9% of total variation) and had larger effect sizes than common NS variants. Our results show that rare variants are likely to have an important effect on pharmacogenetic phenotypes.

## Results

After our GWAS identified *SLCO1B1* as the only locus harboring SNPs associated with methotrexate clearance at the genome-wide significance level (Supplemental Fig. 1; [Trevino et al. 2009](#B39)), we sequenced the gene in a large group of well-phenotyped patients ([Table 1](#T1)). We used a combination of sequencing exons in the entire patient group and a strategy of focusing on the phenotypic outliers ([Cohen et al. 2006](#B8)) (highest vs. lowest methotrexate clearance patients) to detect important variants; we verified that a strategy of focusing on the outliers identified the important variants (Supplemental Fig. 3). We found 39 SNPs in exons 10–15 among all patients, and 21 SNPs in exons 1–9 among the phenotypic outliers; we genotyped all of these, in addition to all other published non-synonymous (NS) SNPs, in all patients (Supplemental Fig. 4). In total, we genotyped 104 SNPs, 93 of which were polymorphic in our 699 patients (Supplemental Table 1).

### Table 1.

Table 1 Caption: Patient characteristics

Of the 93 SNPs, 15 were NS, and seven of these were categorized as functionally damaging by at least three of four computational prediction algorithms ([Ng and Henikoff 2001](#B24); [Ferrer-Costa et al. 2004](#B12); [Yue et al. 2006](#B44); [Adzhubei et al. 2010](#B2)), whereas the other eight NS SNPs were categorized as tolerated ([Table 2](#T2)). Three of these NS SNPs were common, with a minor allele frequency (MAF) >5%, one had low frequency (MAF 1%–5%), and 11 were rare (MAF < 1%). Among patients with extreme phenotypes ([Fig. 1A](#F1)), we found an overrepresentation of damaging NS SNPs in those with the lowest decile for clearance compared with those with the average (45th–55th percentile) or highest deciles (*p* = 2.3 × 10^−8^) ([Fig. 1C](#F1)). Even after excluding patients with the damaging C alleles at the common rs4149056 polymorphism, the same trend holds for rare damaging NS SNPs (*p* = 0.026) ([Fig. 1D](#F1)). When all NS SNPs (irrespective of functional status) were considered, there was no evidence of overrepresentation of variant alleles in patients with extremely high or low clearances (*p* = 0.3) ([Fig. 1B](#F1)).

### Table 2.

Table 2 Caption: Functional effects of 15 non-synonymous (NS) SNPs

### Figure 1.

![Figure 1.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f1f8/3246196/a2a9d45cd09e/1fig1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3246196_1fig1.jpg)

Methotrexate clearance and presence of non-synonymous (NS) SNPs among outliers. (A) The average methotrexate clearance (adjusted for age, gender, ancestry, and treatment arm) is depicted for the lowest, middle, and highest tenth percentile groups (n = 70 patients per decile). The number of patients harboring NS SNPs did not differ among clearance decile groups (p = 0.3, Cochran-Armitage Trend Test) (B), whereas the number of patients harboring damaging NS SNPs differed by clearance decile group (p = 2.3 × 10−8, Cochran-Armitage Trend Test) (C), as did the number of patients harboring rare (MAF < 1%) damaging NS SNPs (p = 0.026, Cochran-Armitage Trend Test) (D).

Because it has already been established that the rs4149056 C allele confers reduced transporter function, we examined which other *SLCO1B1* polymorphisms were associated with methotrexate clearance. Among the 542 patients who were homozygous for the rs4149056 T allele, the 14 patients with rare damaging NS SNPs had lower methotrexate clearance than the 20 patients with rare tolerated NS SNPs (*p* = 0.001) ([Fig. 2A](#F2)) and the other 508 patients without any rare NS SNPs (*p* = 8 × 10^−4^). Effect size tended to be inversely related to MAF ([Fig. 2B](#F2)), although small sample sizes (inherent in rare variant analysis) precluded statistical significance in many cases. These results are consistent with the hypothesis that rare damaging SNPs may have more profound effects on phenotype than common SNPs ([Zhu et al. 2011](#B45)).

### Figure 2.

![Figure 2.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f1f8/3246196/7723bfdcff54/1fig2.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3246196_1fig2.jpg)

Patients with rare damaging NS SNPs have reduced methotrexate clearance. (A) Among the 542 patients who did not carry the variant C allele at rs4149056, median adjusted methotrexate clearance is lower in patients with rare damaging NS SNPs (n = 14) than those having no rare NS SNPs (n = 508, p = 8 × 10−4) or having rare tolerated NS SNPs (n = 20, p = 0.001, Wilcoxon rank-sum test). (B) SNPs with smaller MAFs have larger effect sizes. (Triangles) NS damaging SNPs; (squares) NS tolerated SNPs; (circles) other SNP types. Shapes filled in black indicate p < 0.05; others do not reach statistical significance. Effect size indicates the change in adjusted methotrexate clearance (in milliliters per minute per meter squared) attributable to each copy of the minor allele based on the general linear model, and error bars represent standard error within the model. Shading indicates area under a linear regression fit line for negative effect sizes (pink, lower clearance) or positive effect sizes (green, higher clearance) versus MAF.

Considering all 93 SNPs (adjusting for treatment arm, ancestry, gender, and age), 27 SNP genotypes (Supplemental Table 2) were associated with methotrexate clearance (*p* < 0.05). To identify which SNP genotypes were associated with clearance independent of their LD with the common rs4149056 NS SNP, we performed multivariate linear regression analysis with rs4149056 as a covariate: 23 of the 27 SNPs retained their association (*p* < 0.05) (Supplemental Table 2). Interestingly, an additional 12 SNPs were associated (*p* < 0.05) with clearance only after adjustment for rs4149056, indicating that the effect of these variants may be masked by the variation due to this common functional variant. Within the entire cohort of 699 patients, common low-function rs4149056 C alleles were present in 157 patients (22.4% of all children, MAF 12%), and an additional 15 (2.1% of children) had one rare allele associated with reduced methotrexate clearance (Exon3_560: one patient; Exon3_602: one patient; rs59502379: six patients; rs79661732: seven patients). Taken together, our results indicate that rare NS SNPs account for substantial variability in a small but non-negligible percentage of patients.

Of the seven damaging NS SNPs, the minor allele for three was related to lower methotrexate clearance in the entire group of 699 patients (*p* < 0.05) ([Table 2](#T2)). Conversely, the minor allele for three of the eight tolerated NS SNPs was associated with increased methotrexate clearance (*p* < 0.05; rs2306283 only after adjusting for rs4149056). The three damaging and three tolerated SNPs associated with clearance remain significant after adjusting for rs4149056. The rs2306283 SNP encodes an N130D substitution and becomes significant only after adjusting for rs4149056 (*p* = 0.34 before and *p* = 2.4 × 10^−5^ after adjustment) by conferring higher methotrexate clearance in the patients heterozygous for rs4149056 (Supplemental Fig. 5), which was confirmed in a classification and regression tree analysis (Supplemental Fig. 6).

To estimate the percentages of variability in methotrexate clearance accounted for by non-genetic and genetic variation, a multivariate analysis with forward variable selection revealed that the clinical covariates of treatment arm, age, ancestry, and gender accounted for 16.8%, 3.7%, 2%, and 0.2% of inter-patient variability in clearance, respectively. Variants in *SLCO1B1* accounted for 10.7% of inter-patient variability in clearance, with rare damaging and common coding SNPs accounting for 1.9% and 6.3%, respectively ([Fig. 3](#F3)). Of the clearance variation accounted for by *SLCO1B1* (10.7% of the total), rare damaging NS variants constituted 17.8% of *SLCO1B1*'s effect on clearance ([Fig. 3](#F3)). The rare damaging SNPs explained a significant portion of the variability in MTX clearance (*p* = 0.00014). Overall, clinical and genetic covariates accounted for one-third of the population variation in clearance.

### Figure 3.

![Figure 3.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f1f8/3246196/eae1464dbc7a/1fig3.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3246196_1fig3.jpg)

One-third of inter-patient variability in methotrexate clearance can be explained by clinical covariates and SLCO1B1 SNP genotypes. Forward variable selection was used to select covariates, including SNPs in each category (common NS, rare NS, synonymous [syn], 5′ upstream, or intronic) as input based on each covariate's increase in R2 for explaining variability in clearance (see Methods). SNPs included in this model are listed in Supplemental Table 2. Of the variation in clearance explained by SLCO1B1 polymorphisms, the percent apportioned to each category of SNP (common NS, rare NS, synonymous, 5′ upstream, or intronic) is illustrated in the pie chart at right.

Based on 15 NS SNPs, we observed 20 *SLCO1B1* haplotypes ([Fig. 4A](#F4)), five of which were previously reported ([Tirona et al. 2001](#B37); [Pasanen et al. 2008](#B31)). In a univariate linear regression model, the *5 (*p* = 6.4 × 10^−4^) and *15 (*p* = 9.2 × 10^−9^) haplotypes and two novel haplotypes (*23, *p* < 0.009 and *31, *p* < 0.03) were associated with low methotrexate clearance. The *14 haplotype (*p* = 1.5 × 10^−7^) and one novel haplotype (*35, *p* < 0.007) were associated with high clearance. All four haplotypes that were associated with low clearance contain an NS SNP predicted to be damaging, whereas the two haplotypes associated with high clearance only harbor NS SNPs predicted to be tolerated by all four prediction algorithms. The haplotypes harboring damaging NS SNPs are clustered with low-clearance patients ([Fig. 4B](#F4)), haplotypes harboring tolerated NS SNPs are clustered with high-clearance patients, and those heterozygous for high-clearance and low-clearance haplotypes are clustered with patients with no variant alleles (and intermediate clearance), suggesting that the low- and high-activity alleles offset each other's effects.

### Figure 4.

![Figure 4.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f1f8/3246196/99b649a3e24c/1fig4.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3246196_1fig4.jpg)

SLCO1B1 haplotypes affect average adjusted methotrexate clearance. (A) Haplotypes are based on 15 NS SNPs. (Red boxes with D) Damaging SNPs; (T) tolerated SNPs; (blue boxes) the ancestral allele; (orange boxes) the variant allele. The univariate P-value for the association with methotrexate clearance is indicated after adjustment for age, gender, ancestry, and treatment arm. (Yellow P-values) The haplotype confers significantly lower methotrexate clearance; (green P-values) significantly higher methotrexate clearance; (purple frequencies) singletons; (pink) those that are rare (frequency < 1%). (B) Diplotypes (x-axis) are sorted by median adjusted methotrexate clearance (y-axis). (Yellow haplotypes) Associated with lower methotrexate clearance (p < 0.05); (green haplotypes) associated with higher methotrexate clearance (p < 0.05). (Horizontal line) Median; (boxes) 25th to 75th percentiles; (whiskers) non-outlier range; (circles) outliers. (C) Methotrexate (mtx) uptake by HEK293 cells stably transfected with SLCO1B1 coding variants, plotted as percentage of SLCO1B1*1a-expressing cells, which averaged 30.8 pmol/mg. (Empty) HEK293 cells expressing the empty vector; (asterisks) transport that differs significantly (at p < 0.05, Student's t-test) compared with the SLCO1B1*1a haplotype.

To test for the functional consequences of novel variants discovered in our patients, we stably expressed *SLCO1B1* variants in mammalian HEK293 cells and measured the rate of methotrexate transport. The in vivo pharmacokinetic findings were supported by these in vitro functional studies: Cellular uptake of methotrexate in vitro was reduced in cells stably transfected with the *SLCO1B1* haplotypes associated with reduced methotrexate clearance in patients (*5, *15, *23, and *31) ([Fig. 4C](#F4)).

[](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3246196_1tbl1.jpg)

[](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3246196_1tbl2.jpg)

## Discussion

Our study demonstrates that both rare and common genomic variations contribute to meaningful phenotypic variation in drug disposition. Using computational tools for estimating functional effects of NS substitutions ([Carr et al. 2009](#B7); [Gonzalez-Perez and Lopez-Bigas 2011](#B14)), we found that damaging SNP genotypes concentrated among those with the lowest methotrexate clearance, with similar results when only rare damaging SNPs were considered ([Fig. 1](#F1)). Also important is that patients with intermediate clearance carried a burden of damaging SNPs in between the two outlying clearance groups, and in patients without the common damaging rs4149056 C allele, clearance was lower in those who harbored rare damaging SNP genotypes compared with those who harbored rare tolerated NS SNPs or no rare NS SNPs ([Fig. 2](#F2)). Despite some concerns about performance of computational tools to estimate functional effects of NS SNPs, the approach worked well in this study: Even combining the SNPs predicted to be damaging and tolerated into diplotypes yielded the expected intermediate level of methotrexate clearances ([Fig. 4B](#F4)). Moreover, our in vitro functional experiments validated the algorithms used to predict damaging SNPs and mimicked our in vivo assessments of clearance for each of the haplotypes with SNPs predicted to be damaging ([Fig. 4C](#F4)).

For example, one novel SNP (Exon_3_602) was observed only once among nearly 1400 chromosomes in our patients and resulted in the substitution of arginine at position 57 in the first extracellular loop ([Fig. 5](#F5)). In vitro mutagenesis had suggested that this site might be important for substrate binding, because it is a highly conserved, positively charged amino acid facing the putative binding pocket ([Weaver and Hagenbuch 2010](#B42)). We found that the *23 haplotype, which harbors a variant predicted to be damaging at position 71, has severely reduced methotrexate uptake in vitro ([Fig. 4C](#F4)), consistent with our in vivo assessments of clearance ([Fig. 4B](#F4)). Additionally, the *5 haplotype displayed a greater magnitude of a reduction in methotrexate transport in vitro than did the *15 haplotype, mimicking the patient data (i.e., *1a/*5 patients had lower clearance than *1a/*15 patients).

### Figure 5.

![Figure 5.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f1f8/3246196/a491e70332a0/1fig5.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3246196_1fig5.jpg)

Extracellular, transmembrane, and intracellular cytoplasmic domains predicted for SLCO1B1, indicating all 15 non-synonymous SNPs. Shown are SNPs associated with lower methotrexate clearance (p < 0.05) (yellow), higher clearance (green), or no association with clearance (black).

Epistasis is illustrated by the rs2306283 N130D SNP genotype in the second extracellular loop ([Fig. 5](#F5)), which was related to higher methotrexate clearance in patients carrying the lower-function rs4149056 T521C allele (which resides in the fourth transmembrane domain) (Supplemental Fig. 5; [Pasanen et al. 2008](#B31)). The G allele of rs2306283 is predicted by all four algorithms to be tolerated and is associated with increased clearance of methotrexate only after considering the rs4149056 genotype. Even though there was no statistically significant interaction between rs4149056 and rs2306283 (*p* = 0.08), the effect size of the G allele of rs2306283 was much greater in the rs4149056 TT/CC patients (36 ± 9.8 mL/min per meter squared) than in the patients with the rs4149056 TT genotype (8.3 ± 2.1 mL/min per meter squared) (Supplemental Table 4).

It is important that pharmacogenetic studies include all possible non-genetic covariates as part of the process for evaluating the quantitative contribution of genetic variation to the phenotype. Because all 699 patients were participating in clinical trials, we had outstanding ascertainment of all covariates. We could account for one-third of population variability in clearance, with 10.7% of all variation due to *SLCO1B1* variants. Of that 10.7%, rare variants accounted for 17.8% of the variation attributed to *SLCO1B1* SNPs. Thus, 15 of 699 unselected patients (2.1%) carried a rare NS allele in *SLCO1B1*, indicating that for the phenotype of methotrexate clearance, rare variation has an effect for a non-negligible proportion of patients.

The percentage of variation in methotrexate clearance accounted for by clinical covariates (22.7%) is comparable to the 19.7% described for a complex phenotype such has hypertriglyceridemia ([Johansen et al. 2010](#B17)), although for our pharmacogenetic phenotype, rare variants constituted a greater proportion of genetic variation (17.8%) than was true for rare variants associated with hypertriglyceridemia. Rare NS SNPs conferring either lower (if damaging) or higher (if tolerated) function for non-pharmacogenetic phenotypes, such as blood pressure ([Tomaszewski et al. 2010](#B38)) and other phenotypes, has been demonstrated ([Cohen et al. 2006](#B8); [Momozawa et al. 2010](#B23)). Thus, evidence is mounting that rare variants may be important to capture for the level of comprehensive interrogation of genetic variation that will be needed in clinical genetic testing.

Although we could explain a third of the inter-individual variation in methotrexate clearance based on *SLCO1B1* variation and non-genetic covariates, the other two-thirds of inter-patient variability remains unexplained. This variation could be due to epigenetic variation, genomic variation not assessed (in non-coding or coding regions), differences in regulatory miRNA, post-translational modification or gene expression, or other clinical or environmental variables not assessed. Moreover, additional studies of methotrexate disposition with low-dose methotrexate and with other regimens of high-dose methotrexate will be needed to assess the role of *SLCO1B1* variation in other clinical contexts.

Our study serves as a proof-of-principle for follow-up of a gene hit resulting from a GWAS in pharmacogenomics; here, a single gene (*SLCO1B1*) had common variants associated with drug clearance, but deep sequencing yielded additional rare variants that are likely to be functional and have penetrant effects. As is true for some disease phenotypes, our findings support the hypothesis that a combination of common and rare variants is likely to be important for pharmacogenetic phenotypes. There will be challenges when genotyping tests are deployed in the clinic, because it is likely that sequencing will be required for detection of rare variants for that small but non-negligible group of patients who harbor them.

## Methods

Methotrexate clearance was assessed for a total of 4184 courses administered at 2–5 g/m^2^ in 699 patients; doses for patients enrolled on the Total XV trial were adjusted based on pharmacokinetic parameters to achieve a desired plasma exposure, as described ([Pui et al. 2009](#B34)). Parents and patients gave informed consent or assent where appropriate, and the study was approved by the Institutional Review Board. Details of the two clinical trials and pharmacokinetic analysis have been previously published ([Pui et al. 2004](#B33), [2009](#B34); [Trevino et al. 2009](#B39)). The genome-wide array data were previously published on 640 of these patients ([Trevino et al. 2009](#B39)); the current cohort includes an additional 59 patients not reported in the original GWAS (Supplemental Fig. 7).

### Genotyping

In addition to all common variants (*n* = 24) interrogated on a genome-wide array ([Trevino et al. 2009](#B39)), we extensively sequenced and genotyped all *SLCO1B1* exons and surrounding introns and all reported non-synonymous variants in a group of 699 children with acute lymphoblastic leukemia (ALL) treated on St. Jude Children's Research Hospital Total XIIIB and Total XV protocols. Details of patients included in the current analysis and previous analysis ([Trevino et al. 2009](#B39)) are included in Supplemental Figure 7. Germline DNA was extracted from blood after remission was achieved. Genotypes were generated and calls made as described ([Trevino et al. 2009](#B39)), using GeneChip Human Mapping 500K Array sets or the Genome-Wide Human SNP Array 6.0 (Affymetrix). The *SLCO1B1* T521C (rs4149056) SNP was genotyped as described ([Trevino et al. 2009](#B39)). Genotyping and sequencing strategies are detailed in the Supplemental Material.

The methods used to generate each SNP genotype are included in Supplemental Table 1. The *P*-values (before and after adjustment for the T521C SNP, rs4149056), effect size, and SNPs included in the multivariate model are included in Supplemental Table 2. The Hardy–Weinberg equilibrium *P*-value and MAF in each ancestry group are included in Supplemental Table 3.

### Bioinformatics

We used four prediction algorithms—SIFT ([Ng and Henikoff 2001](#B24)), PMUT ([Ferrer-Costa et al. 2004](#B12)), SNPS3D ([Yue et al. 2006](#B44)), and Polyphen2 ([Adzhubei et al. 2010](#B2))—to categorize NS SNPs into damaging or tolerated groups. The SNPs in the damaging group (*n* = 7) were predicted to be damaging by three or four of the algorithms, while the other NS SNPs were included in the tolerated group ([Table 2](#T2)). Supplemental Table 5 contains the individual SNP scores for each algorithm.

### Cellular uptake assays

The plasmid pCMV6-XL4 vector, encoding *SLCO1B1*1b*, was purchased from Origene, and *SLCO1B1* was subsequently subcloned into pcDNA5/FRT/TO (Invitrogen). Mutations were introduced by the QuikChange II XL Site-Directed Mutagenesis Kit (Agilent Technologies) and confirmed by Sanger sequencing. Stable, isogenic HEK293 cell lines expressing inducible *SLCO1B1* variants were constructed using the Flp-In T-REx System according to the manufacturer's instructions (Invitrogen). Equal expression among variants was confirmed by RT-PCR (data not shown). Methotrexate uptake was performed using [^3^H]methotrexate as described in detail in the Supplemental Methods. The uptake of SLCO1B1 variant–expressing cells was normalized to the uptake of SLCO1B1*1a (WT)–expressing cells and expressed as the average uptake from two or three independent assays done in triplicate. Results were compared using a two-tailed *t*-test.

### Statistical analysis

Missing SNP genotypes among the 93 SNPs analyzed in 699 patients were imputed using the fastPHASE software ([Scheet and Stephens 2006](#B35)). Haplotypes were inferred using PHASE ([Stephens et al. 2001](#B36)). Genetic ancestries of patients were estimated using STRUCTURE ([Pritchard et al. 2000](#B32)) based on genotypes from the SNP array ([Yang et al. 2010](#B43)). Imputation of variants from the 1000 Genomes Project was performed using MACH1.0 ([Li et al. 2010](#B20)) based on HapMap CEU, YRI, CHB, and JPT reference populations (additional details in the Supplemental Methods).

The Cochran-Armitage Trend Test (prop.trend.test in R) was used to compare the proportion of patients in each clearance decile group harboring at least one NS, damaging NS, or rare damaging NS SNP ([Fig. 1](#F1)). The Wilcoxon rank-sum test was used to compare adjusted methotrexate clearance in patients (excluding carriers of the rs4149056 C allele) with rare damaging NS SNPs, rare tolerated NS SNPs, and those with no rare NS SNPs ([Fig. 2A](#F2)). This non-parametric test was used because the sample size was small (*n* = 20 patients with rare damaging NS SNPs, *n* = 14 patients with rare tolerated NS SNPs).

Average methotrexate clearance was used as the phenotype because linear mixed effect models considering the clearance of each course of methotrexate individually did not alter the association between genotypes and methotrexate clearance when compared with using the average methotrexate clearance across all courses ([Trevino et al. 2009](#B39)). Associations between germline SNP genotypes and methotrexate clearance were evaluated using a general linear model assuming an additive effect of allelic dosage, with all models including age, ancestry, gender, and treatment regimen as “clinical” covariates (Supplemental Table 2). Effect size refers to the beta from the linear regression ([Table 2](#T2); Supplemental Tables 2, 4). “Adjusted methotrexate clearance” refers to the residual from the linear regression including the clinical covariates. Age was coded in years and treated as a continuous variable. The treatment regimen was coded as a categorical variable. Clinical covariates were always included in any analyses of genetic (single SNP or multiple SNPs) association with clearance.

Multiple linear regression analysis was used to estimate percent variation explained by multiple genotypes and other covariates. All SNPs were grouped into the following categories: common NS, rare NS, synonymous, 5′ upstream, UTR, or intronic. Linear regression models were fit by including each category of SNPs sequentially (in the order above). We performed a forward selection strategy, adding SNPs that were significant at the 0.05 level in the presence of the SNPs and additional covariates already in the model. The significance of the SNP was determined using ANOVA comparing the models with the SNP versus without the SNP. The difference in *R*^2^ values between the models was used to calculate the percentage of variation attributable to each variable. Clinical covariates were included in all models. Statistical analyses were conducted using R ([http://www.r-project.org/](http://www.r-project.org/)) and Statistica 10 (StatSoft, Inc). All *P*-values presented are two-sided and were not adjusted for multiple testing.

## Data access

The data have been submitted to dbGaP under accession number phs000426.v1.p1 ([http://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000426.v1.p1](http://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi%3Fstudy_id%3Dphs000426.v1.p1)).

## Acknowledgments

We thank our protocol co-investigators, clinical and research staff, particularly Pamela McGill, Jeremy Hunt, Nancy Kornegay, and Xiangjun Cai, as well as patients and families for their participation. We thank the staff from the St. Jude Hartwell Center for Bioinformatics and Biotechnology. We thank Jun Yang for a critical review of the manuscript. This work is supported by NCI grants CA 36401 and CA 21765 and the NIH/NIGMS Pharmacogenomics Research Network (U01 GM92666 and U01 HL65899), and by the American Lebanese Syrian Associated Charities (ALSAC).

## Footnotes

## References

1. Abe T, Unno M, Onogawa T, Tokui T, Kondo TN, Nakagomi R, Adachi H, Fujiwara K, Okabe M, Suzuki T, et al. 2001. LST-2, a human liver-specific organic anion transporter, determines methotrexate sensitivity in gastrointestinal cancers. Gastroenterology 120: 1689–1699  [DOI](https://doi.org/10.1053/gast.2001.24804) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11375950/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastroenterology&title=LST-2,%20a%20human%20liver-specific%20organic%20anion%20transporter,%20determines%20methotrexate%20sensitivity%20in%20gastrointestinal%20cancers&author=T%20Abe&author=M%20Unno&author=T%20Onogawa&author=T%20Tokui&author=TN%20Kondo&volume=120&publication_year=2001&pages=1689-1699&pmid=11375950&doi=10.1053/gast.2001.24804&)

2. Adzhubei IA, Schmidt S, Peshkin L, Ramensky VE, Gerasimova A, Bork P, Kondrashov AS, Sunyaev SR 2010. A method and server for predicting damaging missense mutations. Nat Methods 7: 248–249  [DOI](https://doi.org/10.1038/nmeth0410-248) | [PMC free article](/articles/PMC2855889/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20354512/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Methods&title=A%20method%20and%20server%20for%20predicting%20damaging%20missense%20mutations&author=IA%20Adzhubei&author=S%20Schmidt&author=L%20Peshkin&author=VE%20Ramensky&author=A%20Gerasimova&volume=7&publication_year=2010&pages=248-249&pmid=20354512&doi=10.1038/nmeth0410-248&)

3. Altman RB, Kroemer HK, McCarty CA, Ratain MJ, Roden D 2011. Pharmacogenomics: Will the promise be fulfilled? Nat Rev Genet 12: 69–73  [DOI](https://doi.org/10.1038/nrg2920) | [PMC free article](/articles/PMC3098748/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21116304/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Genet&title=Pharmacogenomics:%20Will%20the%20promise%20be%20fulfilled?&author=RB%20Altman&author=HK%20Kroemer&author=CA%20McCarty&author=MJ%20Ratain&author=D%20Roden&volume=12&publication_year=2011&pages=69-73&pmid=21116304&doi=10.1038/nrg2920&)

4. Asimit J, Zeggini E 2011. Rare variant association analysis methods for complex traits. Annu Rev Genet 44: 293–308  [DOI](https://doi.org/10.1146/annurev-genet-102209-163421) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21047260/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annu%20Rev%20Genet&title=Rare%20variant%20association%20analysis%20methods%20for%20complex%20traits&author=J%20Asimit&author=E%20Zeggini&volume=44&publication_year=2011&pages=293-308&pmid=21047260&doi=10.1146/annurev-genet-102209-163421&)

5. Bansal V, Libiger O, Torkamani A, Schork NJ 2010. Statistical analysis strategies for association studies involving rare variants. Nat Rev Genet 11: 773–785  [DOI](https://doi.org/10.1038/nrg2867) | [PMC free article](/articles/PMC3743540/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20940738/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Genet&title=Statistical%20analysis%20strategies%20for%20association%20studies%20involving%20rare%20variants&author=V%20Bansal&author=O%20Libiger&author=A%20Torkamani&author=NJ%20Schork&volume=11&publication_year=2010&pages=773-785&pmid=20940738&doi=10.1038/nrg2867&)

6. Bodmer W, Bonilla C 2008. Common and rare variants in multifactorial susceptibility to common diseases. Nat Genet 40: 695–701  [DOI](https://doi.org/10.1038/ng.f.136) | [PMC free article](/articles/PMC2527050/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18509313/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Common%20and%20rare%20variants%20in%20multifactorial%20susceptibility%20to%20common%20diseases&author=W%20Bodmer&author=C%20Bonilla&volume=40&publication_year=2008&pages=695-701&pmid=18509313&doi=10.1038/ng.f.136&)

7. Carr DF, Whiteley G, Alfirevic A, Pirmohamed M 2009. Investigation of inter-individual variability of the one-carbon folate pathway: A bioinformatic and genetic review. Pharmacogenomics J 9: 291–305  [DOI](https://doi.org/10.1038/tpj.2009.29) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19581920/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Investigation%20of%20inter-individual%20variability%20of%20the%20one-carbon%20folate%20pathway:%20A%20bioinformatic%20and%20genetic%20review&author=DF%20Carr&author=G%20Whiteley&author=A%20Alfirevic&author=M%20Pirmohamed&volume=9&publication_year=2009&pages=291-305&pmid=19581920&doi=10.1038/tpj.2009.29&)

8. Cohen JC, Pertsemlidis A, Fahmi S, Esmail S, Vega GL, Grundy SM, Hobbs HH 2006. Multiple rare variants in NPC1L1 associated with reduced sterol absorption and plasma low-density lipoprotein levels. Proc Natl Acad Sci 103: 1810–1815  [DOI](https://doi.org/10.1073/pnas.0508483103) | [PMC free article](/articles/PMC1413637/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16449388/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci&title=Multiple%20rare%20variants%20in%20NPC1L1%20associated%20with%20reduced%20sterol%20absorption%20and%20plasma%20low-density%20lipoprotein%20levels&author=JC%20Cohen&author=A%20Pertsemlidis&author=S%20Fahmi&author=S%20Esmail&author=GL%20Vega&volume=103&publication_year=2006&pages=1810-1815&pmid=16449388&doi=10.1073/pnas.0508483103&)

9. Evans WE, Crom WR, Stewart CF, Bowman WP, Chen CH, Abromowitch M, Simone JV 1984. Methotrexate systemic clearance influences probability of relapse in children with standard-risk acute lymphocytic leukaemia. Lancet 1: 359–362  [DOI](https://doi.org/10.1016/s0140-6736(84)90411-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/6141424/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Methotrexate%20systemic%20clearance%20influences%20probability%20of%20relapse%20in%20children%20with%20standard-risk%20acute%20lymphocytic%20leukaemia&author=WE%20Evans&author=WR%20Crom&author=CF%20Stewart&author=WP%20Bowman&author=CH%20Chen&volume=1&publication_year=1984&pages=359-362&pmid=6141424&doi=10.1016/s0140-6736(84)90411-2&)

10. Evans WE, Crom WR, Abromowitch M, Dodge R, Look AT, Bowman WP, George SL, Pui CH 1986. Clinical pharmacodynamics of high-dose methotrexate in acute lymphocytic leukemia. Identification of a relation between concentration and effect. N Engl J Med 314: 471–477  [DOI](https://doi.org/10.1056/NEJM198602203140803) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/3456079/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Clinical%20pharmacodynamics%20of%20high-dose%20methotrexate%20in%20acute%20lymphocytic%20leukemia.%20Identification%20of%20a%20relation%20between%20concentration%20and%20effect&author=WE%20Evans&author=WR%20Crom&author=M%20Abromowitch&author=R%20Dodge&author=AT%20Look&volume=314&publication_year=1986&pages=471-477&pmid=3456079&doi=10.1056/NEJM198602203140803&)

11. Evans WE, Relling MV, Rodman JH, Crom WR, Boyett JM, Pui CH 1998. Conventional compared with individualized chemotherapy for childhood acute lymphoblastic leukemia. N Engl J Med 338: 499–505  [DOI](https://doi.org/10.1056/NEJM199802193380803) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9468466/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Conventional%20compared%20with%20individualized%20chemotherapy%20for%20childhood%20acute%20lymphoblastic%20leukemia&author=WE%20Evans&author=MV%20Relling&author=JH%20Rodman&author=WR%20Crom&author=JM%20Boyett&volume=338&publication_year=1998&pages=499-505&pmid=9468466&doi=10.1056/NEJM199802193380803&)

12. Ferrer-Costa C, Orozco M, de la Cruz X 2004. Sequence-based prediction of pathological mutations. Proteins 57: 811–819  [DOI](https://doi.org/10.1002/prot.20252) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15390262/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proteins&title=Sequence-based%20prediction%20of%20pathological%20mutations&author=C%20Ferrer-Costa&author=M%20Orozco&author=X%20de%20la%20Cruz&volume=57&publication_year=2004&pages=811-819&pmid=15390262&doi=10.1002/prot.20252&)

13. Franke RM, Gardner ER, Sparreboom A 2010. Pharmacogenetics of drug transporters. Curr Pharm Des 16: 220–230  [DOI](https://doi.org/10.2174/138161210790112683) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19835554/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Pharm%20Des&title=Pharmacogenetics%20of%20drug%20transporters&author=RM%20Franke&author=ER%20Gardner&author=A%20Sparreboom&volume=16&publication_year=2010&pages=220-230&pmid=19835554&doi=10.2174/138161210790112683&)

14. Gonzalez-Perez A, Lopez-Bigas N 2011. Improving the assessment of the outcome of nonsynonymous SNVs with a consensus deleteriousness score, Condel. Am J Hum Genet 88: 440–449  [DOI](https://doi.org/10.1016/j.ajhg.2011.03.004) | [PMC free article](/articles/PMC3071923/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21457909/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Improving%20the%20assessment%20of%20the%20outcome%20of%20nonsynonymous%20SNVs%20with%20a%20consensus%20deleteriousness%20score,%20Condel&author=A%20Gonzalez-Perez&author=N%20Lopez-Bigas&volume=88&publication_year=2011&pages=440-449&pmid=21457909&doi=10.1016/j.ajhg.2011.03.004&)

15. Hirano M, Maeda K, Shitara Y, Sugiyama Y 2004. Contribution of OATP2 (OATP1B1) and OATP8 (OATP1B3) to the hepatic uptake of pitavastatin in humans. J Pharmacol Exp Ther 311: 139–146  [DOI](https://doi.org/10.1124/jpet.104.068056) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15159445/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacol%20Exp%20Ther&title=Contribution%20of%20OATP2%20(OATP1B1)%20and%20OATP8%20(OATP1B3)%20to%20the%20hepatic%20uptake%20of%20pitavastatin%20in%20humans&author=M%20Hirano&author=K%20Maeda&author=Y%20Shitara&author=Y%20Sugiyama&volume=311&publication_year=2004&pages=139-146&pmid=15159445&doi=10.1124/jpet.104.068056&)

16. Iwai M, Suzuki H, Ieiri I, Otsubo K, Sugiyama Y 2004. Functional analysis of single nucleotide polymorphisms of hepatic organic anion transporter OATP1B1 (OATP-C). Pharmacogenetics 14: 749–757  [DOI](https://doi.org/10.1097/00008571-200411000-00006) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15564882/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Functional%20analysis%20of%20single%20nucleotide%20polymorphisms%20of%20hepatic%20organic%20anion%20transporter%20OATP1B1%20(OATP-C)&author=M%20Iwai&author=H%20Suzuki&author=I%20Ieiri&author=K%20Otsubo&author=Y%20Sugiyama&volume=14&publication_year=2004&pages=749-757&pmid=15564882&doi=10.1097/00008571-200411000-00006&)

17. Johansen CT, Wang J, Lanktree MB, Cao H, McIntyre AD, Ban MR, Martins RA, Kennedy BA, Hassell RG, Visser ME, et al. 2010. Excess of rare variants in genes identified by genome-wide association study of hypertriglyceridemia. Nat Genet 42: 684–687  [DOI](https://doi.org/10.1038/ng.628) | [PMC free article](/articles/PMC3017369/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20657596/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Excess%20of%20rare%20variants%20in%20genes%20identified%20by%20genome-wide%20association%20study%20of%20hypertriglyceridemia&author=CT%20Johansen&author=J%20Wang&author=MB%20Lanktree&author=H%20Cao&author=AD%20McIntyre&volume=42&publication_year=2010&pages=684-687&pmid=20657596&doi=10.1038/ng.628&)

18. Kameyama Y, Yamashita K, Kobayashi K, Hosokawa M, Chiba K 2005. Functional characterization of SLCO1B1 (OATP-C) variants, SLCO1B1*5, SLCO1B1*15 and SLCO1B1*15+C1007G, by using transient expression systems of HeLa and HEK293 cells. Pharmacogenet Genomics 15: 513–522  [DOI](https://doi.org/10.1097/01.fpc.0000170913.73780.5f) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15970799/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Functional%20characterization%20of%20SLCO1B1%20(OATP-C)%20variants,%20SLCO1B1*5,%20SLCO1B1*15%20and%20SLCO1B1*15+C1007G,%20by%20using%20transient%20expression%20systems%20of%20HeLa%20and%20HEK293%20cells&author=Y%20Kameyama&author=K%20Yamashita&author=K%20Kobayashi&author=M%20Hosokawa&author=K%20Chiba&volume=15&publication_year=2005&pages=513-522&pmid=15970799&doi=10.1097/01.fpc.0000170913.73780.5f&)

19. Konig J, Cui Y, Nies AT, Keppler D 2000. A novel human organic anion transporting polypeptide localized to the basolateral hepatocyte membrane. Am J Physiol Gastrointest Liver Physiol 278: G156–G164  [DOI](https://doi.org/10.1152/ajpgi.2000.278.1.G156) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10644574/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Physiol%20Gastrointest%20Liver%20Physiol&title=A%20novel%20human%20organic%20anion%20transporting%20polypeptide%20localized%20to%20the%20basolateral%20hepatocyte%20membrane&author=J%20Konig&author=Y%20Cui&author=AT%20Nies&author=D%20Keppler&volume=278&publication_year=2000&pages=G156-G164&pmid=10644574&doi=10.1152/ajpgi.2000.278.1.G156&)

20. Li Y, Willer CJ, Ding J, Scheet P, Abecasis GR 2010. MaCH: Using sequence and genotype data to estimate haplotypes and unobserved genotypes. Genet Epidemiol 34: 816–834  [DOI](https://doi.org/10.1002/gepi.20533) | [PMC free article](/articles/PMC3175618/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21058334/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genet%20Epidemiol&title=MaCH:%20Using%20sequence%20and%20genotype%20data%20to%20estimate%20haplotypes%20and%20unobserved%20genotypes&author=Y%20Li&author=CJ%20Willer&author=J%20Ding&author=P%20Scheet&author=GR%20Abecasis&volume=34&publication_year=2010&pages=816-834&pmid=21058334&doi=10.1002/gepi.20533&)

21. Liu DJ, Leal SM 2010. Replication strategies for rare variant complex trait association studies via next-generation sequencing. Am J Hum Genet 87: 790–801  [DOI](https://doi.org/10.1016/j.ajhg.2010.10.025) | [PMC free article](/articles/PMC2997372/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21129725/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Replication%20strategies%20for%20rare%20variant%20complex%20trait%20association%20studies%20via%20next-generation%20sequencing&author=DJ%20Liu&author=SM%20Leal&volume=87&publication_year=2010&pages=790-801&pmid=21129725&doi=10.1016/j.ajhg.2010.10.025&)

22. Lopez-Lopez E, Martin-Guerrero I, Ballesteros J, Pinan MA, Garcia-Miguel P, Navajas A, Garcia-Orad A 2011. Polymorphisms of the SLCO1B1 gene predict methotrexate-related toxicity in childhood acute lymphoblastic leukemia. Pediatr Blood Cancer 57: 612–619  [DOI](https://doi.org/10.1002/pbc.23074) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21387541/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pediatr%20Blood%20Cancer&title=Polymorphisms%20of%20the%20SLCO1B1%20gene%20predict%20methotrexate-related%20toxicity%20in%20childhood%20acute%20lymphoblastic%20leukemia&author=E%20Lopez-Lopez&author=I%20Martin-Guerrero&author=J%20Ballesteros&author=MA%20Pinan&author=P%20Garcia-Miguel&volume=57&publication_year=2011&pages=612-619&pmid=21387541&doi=10.1002/pbc.23074&)

23. Momozawa Y, Mni M, Nakamura K, Coppieters W, Almer S, Amininejad L, Cleynen I, Colombel JF, de Rijk P, Dewit O, et al. 2010. Resequencing of positional candidates identifies low frequency IL23R coding variants protecting against inflammatory bowel disease. Nat Genet 43: 43–47  [DOI](https://doi.org/10.1038/ng.733) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21151126/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Resequencing%20of%20positional%20candidates%20identifies%20low%20frequency%20IL23R%20coding%20variants%20protecting%20against%20inflammatory%20bowel%20disease&author=Y%20Momozawa&author=M%20Mni&author=K%20Nakamura&author=W%20Coppieters&author=S%20Almer&volume=43&publication_year=2010&pages=43-47&pmid=21151126&doi=10.1038/ng.733&)

24. Ng PC, Henikoff S 2001. Predicting deleterious amino acid substitutions. Genome Res 11: 863–874  [DOI](https://doi.org/10.1101/gr.176601) | [PMC free article](/articles/PMC311071/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11337480/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res&title=Predicting%20deleterious%20amino%20acid%20substitutions&author=PC%20Ng&author=S%20Henikoff&volume=11&publication_year=2001&pages=863-874&pmid=11337480&doi=10.1101/gr.176601&)

25. Niemi M, Schaeffeler E, Lang T, Fromm MF, Neuvonen M, Kyrklund C, Backman JT, Kerb R, Schwab M, Neuvonen PJ, et al. 2004. High plasma pravastatin concentrations are associated with single nucleotide polymorphisms and haplotypes of organic anion transporting polypeptide-C (OATP-C, SLCO1B1). Pharmacogenetics 14: 429–440  [DOI](https://doi.org/10.1097/01.fpc.0000114750.08559.32) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15226675/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=High%20plasma%20pravastatin%20concentrations%20are%20associated%20with%20single%20nucleotide%20polymorphisms%20and%20haplotypes%20of%20organic%20anion%20transporting%20polypeptide-C%20(OATP-C,%20SLCO1B1)&author=M%20Niemi&author=E%20Schaeffeler&author=T%20Lang&author=MF%20Fromm&author=M%20Neuvonen&volume=14&publication_year=2004&pages=429-440&pmid=15226675&doi=10.1097/01.fpc.0000114750.08559.32&)

26. Niemi M, Backman JT, Kajosaari LI, Leathart JB, Neuvonen M, Daly AK, Eichelbaum M, Kivisto KT, Neuvonen PJ 2005a. Polymorphic organic anion transporting polypeptide 1B1 is a major determinant of repaglinide pharmacokinetics. Clin Pharmacol Ther 77: 468–478  [DOI](https://doi.org/10.1016/j.clpt.2005.01.018) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15961978/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Polymorphic%20organic%20anion%20transporting%20polypeptide%201B1%20is%20a%20major%20determinant%20of%20repaglinide%20pharmacokinetics&author=M%20Niemi&author=JT%20Backman&author=LI%20Kajosaari&author=JB%20Leathart&author=M%20Neuvonen&volume=77&publication_year=2005a&pages=468-478&pmid=15961978&doi=10.1016/j.clpt.2005.01.018&)

27. Niemi M, Kivisto KT, Hofmann U, Schwab M, Eichelbaum M, Fromm MF 2005b. Fexofenadine pharmacokinetics are associated with a polymorphism of the SLCO1B1 gene (encoding OATP1B1). Br J Clin Pharmacol 59: 602–604  [DOI](https://doi.org/10.1111/j.1365-2125.2005.02354.x) | [PMC free article](/articles/PMC1884844/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15842561/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=Fexofenadine%20pharmacokinetics%20are%20associated%20with%20a%20polymorphism%20of%20the%20SLCO1B1%20gene%20(encoding%20OATP1B1)&author=M%20Niemi&author=KT%20Kivisto&author=U%20Hofmann&author=M%20Schwab&author=M%20Eichelbaum&volume=59&publication_year=2005b&pages=602-604&pmid=15842561&doi=10.1111/j.1365-2125.2005.02354.x&)

28. Nozawa T, Minami H, Sugiura S, Tsuji A, Tamai I 2005. Role of organic anion transporter OATP1B1 (OATP-C) in hepatic uptake of irinotecan and its active metabolite, 7-ethyl-10-hydroxycamptothecin: In vitro evidence and effect of single nucleotide polymorphisms. Drug Metab Dispos 33: 434–439  [DOI](https://doi.org/10.1124/dmd.104.001909) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15608127/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Role%20of%20organic%20anion%20transporter%20OATP1B1%20(OATP-C)%20in%20hepatic%20uptake%20of%20irinotecan%20and%20its%20active%20metabolite,%207-ethyl-10-hydroxycamptothecin:%20In%20vitro%20evidence%20and%20effect%20of%20single%20nucleotide%20polymorphisms&author=T%20Nozawa&author=H%20Minami&author=S%20Sugiura&author=A%20Tsuji&author=I%20Tamai&volume=33&publication_year=2005&pages=434-439&pmid=15608127&doi=10.1124/dmd.104.001909&)

29. Pasanen MK, Neuvonen M, Neuvonen PJ, Niemi M 2006. SLCO1B1 polymorphism markedly affects the pharmacokinetics of simvastatin acid. Pharmacogenet Genomics 16: 873–879  [DOI](https://doi.org/10.1097/01.fpc.0000230416.82349.90) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17108811/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=SLCO1B1%20polymorphism%20markedly%20affects%20the%20pharmacokinetics%20of%20simvastatin%20acid&author=MK%20Pasanen&author=M%20Neuvonen&author=PJ%20Neuvonen&author=M%20Niemi&volume=16&publication_year=2006&pages=873-879&pmid=17108811&doi=10.1097/01.fpc.0000230416.82349.90&)

30. Pasanen MK, Fredrikson H, Neuvonen PJ, Niemi M 2007. Different effects of SLCO1B1 polymorphism on the pharmacokinetics of atorvastatin and rosuvastatin. Clin Pharmacol Ther 82: 726–733  [DOI](https://doi.org/10.1038/sj.clpt.6100220) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17473846/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Different%20effects%20of%20SLCO1B1%20polymorphism%20on%20the%20pharmacokinetics%20of%20atorvastatin%20and%20rosuvastatin&author=MK%20Pasanen&author=H%20Fredrikson&author=PJ%20Neuvonen&author=M%20Niemi&volume=82&publication_year=2007&pages=726-733&pmid=17473846&doi=10.1038/sj.clpt.6100220&)

31. Pasanen MK, Neuvonen PJ, Niemi M 2008. Global analysis of genetic variation in SLCO1B1. Pharmacogenomics 9: 19–33  [DOI](https://doi.org/10.2217/14622416.9.1.19) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18154446/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Global%20analysis%20of%20genetic%20variation%20in%20SLCO1B1&author=MK%20Pasanen&author=PJ%20Neuvonen&author=M%20Niemi&volume=9&publication_year=2008&pages=19-33&pmid=18154446&doi=10.2217/14622416.9.1.19&)

32. Pritchard JK, Stephens M, Donnelly P 2000. Inference of population structure using multilocus genotype data. Genetics 155: 945–959  [DOI](https://doi.org/10.1093/genetics/155.2.945) | [PMC free article](/articles/PMC1461096/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10835412/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genetics&title=Inference%20of%20population%20structure%20using%20multilocus%20genotype%20data&author=JK%20Pritchard&author=M%20Stephens&author=P%20Donnelly&volume=155&publication_year=2000&pages=945-959&pmid=10835412&doi=10.1093/genetics/155.2.945&)

33. Pui CH, Sandlund JT, Pei D, Campana D, Rivera GK, Ribeiro RC, Rubnitz JE, Razzouk BI, Howard SC, Hudson MM, et al. 2004. Improved outcome for children with acute lymphoblastic leukemia: Results of Total Therapy Study XIIIB at St Jude Children's Research Hospital. Blood 104: 2690–2696  [DOI](https://doi.org/10.1182/blood-2004-04-1616) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15251979/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Improved%20outcome%20for%20children%20with%20acute%20lymphoblastic%20leukemia:%20Results%20of%20Total%20Therapy%20Study%20XIIIB%20at%20St%20Jude%20Children's%20Research%20Hospital&author=CH%20Pui&author=JT%20Sandlund&author=D%20Pei&author=D%20Campana&author=GK%20Rivera&volume=104&publication_year=2004&pages=2690-2696&pmid=15251979&doi=10.1182/blood-2004-04-1616&)

34. Pui CH, Campana D, Pei D, Bowman WP, Sandlund JT, Kaste SC, Ribeiro RC, Rubnitz JE, Raimondi SC, Onciu M, et al. 2009. Treating childhood acute lymphoblastic leukemia without cranial irradiation. N Engl J Med 360: 2730–2741  [DOI](https://doi.org/10.1056/NEJMoa0900386) | [PMC free article](/articles/PMC2754320/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19553647/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Treating%20childhood%20acute%20lymphoblastic%20leukemia%20without%20cranial%20irradiation&author=CH%20Pui&author=D%20Campana&author=D%20Pei&author=WP%20Bowman&author=JT%20Sandlund&volume=360&publication_year=2009&pages=2730-2741&pmid=19553647&doi=10.1056/NEJMoa0900386&)

35. Scheet P, Stephens M 2006. A fast and flexible statistical model for large-scale population genotype data: Applications to inferring missing genotypes and haplotypic phase. Am J Hum Genet 78: 629–644  [DOI](https://doi.org/10.1086/502802) | [PMC free article](/articles/PMC1424677/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16532393/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=A%20fast%20and%20flexible%20statistical%20model%20for%20large-scale%20population%20genotype%20data:%20Applications%20to%20inferring%20missing%20genotypes%20and%20haplotypic%20phase&author=P%20Scheet&author=M%20Stephens&volume=78&publication_year=2006&pages=629-644&pmid=16532393&doi=10.1086/502802&)

36. Stephens M, Smith NJ, Donnelly P 2001. A new statistical method for haplotype reconstruction from population data. Am J Hum Genet 68: 978–989  [DOI](https://doi.org/10.1086/319501) | [PMC free article](/articles/PMC1275651/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11254454/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=A%20new%20statistical%20method%20for%20haplotype%20reconstruction%20from%20population%20data&author=M%20Stephens&author=NJ%20Smith&author=P%20Donnelly&volume=68&publication_year=2001&pages=978-989&pmid=11254454&doi=10.1086/319501&)

37. Tirona RG, Leake BF, Merino G, Kim RB 2001. Polymorphisms in OATP-C: Identification of multiple allelic variants associated with altered transport activity among European- and African-Americans. J Biol Chem 276: 35669–35675  [DOI](https://doi.org/10.1074/jbc.M103792200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11477075/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Polymorphisms%20in%20OATP-C:%20Identification%20of%20multiple%20allelic%20variants%20associated%20with%20altered%20transport%20activity%20among%20European-%20and%20African-Americans&author=RG%20Tirona&author=BF%20Leake&author=G%20Merino&author=RB%20Kim&volume=276&publication_year=2001&pages=35669-35675&pmid=11477075&doi=10.1074/jbc.M103792200&)

38. Tomaszewski M, Debiec R, Braund PS, Nelson CP, Hardwick R, Christofidou P, Denniff M, Codd V, Rafelt S, van der Harst P, et al. 2010. Genetic architecture of ambulatory blood pressure in the general population: Insights from cardiovascular gene-centric array. Hypertension 56: 1069–1076  [DOI](https://doi.org/10.1161/HYPERTENSIONAHA.110.155721) | [PMC free article](/articles/PMC3035934/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21060006/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hypertension&title=Genetic%20architecture%20of%20ambulatory%20blood%20pressure%20in%20the%20general%20population:%20Insights%20from%20cardiovascular%20gene-centric%20array&author=M%20Tomaszewski&author=R%20Debiec&author=PS%20Braund&author=CP%20Nelson&author=R%20Hardwick&volume=56&publication_year=2010&pages=1069-1076&pmid=21060006&doi=10.1161/HYPERTENSIONAHA.110.155721&)

39. Trevino LR, Shimasaki N, Yang W, Panetta JC, Cheng C, Pei D, Chan D, Sparreboom A, Giacomini KM, Pui CH, et al. 2009. Germline genetic variation in an organic anion transporter polypeptide associated with methotrexate pharmacokinetics and clinical effects. J Clin Oncol 27: 5972–5978  [DOI](https://doi.org/10.1200/JCO.2008.20.4156) | [PMC free article](/articles/PMC2793040/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19901119/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Germline%20genetic%20variation%20in%20an%20organic%20anion%20transporter%20polypeptide%20associated%20with%20methotrexate%20pharmacokinetics%20and%20clinical%20effects&author=LR%20Trevino&author=N%20Shimasaki&author=W%20Yang&author=JC%20Panetta&author=C%20Cheng&volume=27&publication_year=2009&pages=5972-5978&pmid=19901119&doi=10.1200/JCO.2008.20.4156&)

40. van de Steeg E, van der Kruijssen CM, Wagenaar E, Burggraaff JE, Mesman E, Kenworthy KE, Schinkel AH 2009. Methotrexate pharmacokinetics in transgenic mice with liver-specific expression of human organic anion-transporting polypeptide 1B1 (SLCO1B1). Drug Metab Dispos 37: 277–281  [DOI](https://doi.org/10.1124/dmd.108.024315) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19022939/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Methotrexate%20pharmacokinetics%20in%20transgenic%20mice%20with%20liver-specific%20expression%20of%20human%20organic%20anion-transporting%20polypeptide%201B1%20(SLCO1B1)&author=E%20van%20de%20Steeg&author=CM%20van%20der%20Kruijssen&author=E%20Wagenaar&author=JE%20Burggraaff&author=E%20Mesman&volume=37&publication_year=2009&pages=277-281&pmid=19022939&doi=10.1124/dmd.108.024315&)

41. van de Steeg E, Wagenaar E, van der Kruijssen CM, Burggraaff JE, de Waart DR, Elferink RP, Kenworthy KE, Schinkel AH 2010. Organic anion transporting polypeptide 1a/1b-knockout mice provide insights into hepatic handling of bilirubin, bile acids, and drugs. J Clin Invest 120: 2942–2952  [DOI](https://doi.org/10.1172/JCI42168) | [PMC free article](/articles/PMC2912192/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20644253/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Invest&title=Organic%20anion%20transporting%20polypeptide%201a/1b-knockout%20mice%20provide%20insights%20into%20hepatic%20handling%20of%20bilirubin,%20bile%20acids,%20and%20drugs&author=E%20van%20de%20Steeg&author=E%20Wagenaar&author=CM%20van%20der%20Kruijssen&author=JE%20Burggraaff&author=DR%20de%20Waart&volume=120&publication_year=2010&pages=2942-2952&pmid=20644253&doi=10.1172/JCI42168&)

42. Weaver YM, Hagenbuch B 2010. Several conserved positively charged amino acids in OATP1B1 are involved in binding or translocation of different substrates. J Membr Biol 236: 279–290  [DOI](https://doi.org/10.1007/s00232-010-9300-3) | [PMC free article](/articles/PMC2989658/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20821001/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Membr%20Biol&title=Several%20conserved%20positively%20charged%20amino%20acids%20in%20OATP1B1%20are%20involved%20in%20binding%20or%20translocation%20of%20different%20substrates&author=YM%20Weaver&author=B%20Hagenbuch&volume=236&publication_year=2010&pages=279-290&pmid=20821001&doi=10.1007/s00232-010-9300-3&)

43. Yang W, Trevino LR, Yang JJ, Scheet P, Pui CH, Evans WE, Relling MV 2010. ARID5B SNP rs10821936 is associated with risk of childhood acute lymphoblastic leukemia in blacks and contributes to racial differences in leukemia incidence. Leukemia 24: 894–896  [DOI](https://doi.org/10.1038/leu.2009.277) | [PMC free article](/articles/PMC4607263/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20054350/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Leukemia&title=ARID5B%20SNP%20rs10821936%20is%20associated%20with%20risk%20of%20childhood%20acute%20lymphoblastic%20leukemia%20in%20blacks%20and%20contributes%20to%20racial%20differences%20in%20leukemia%20incidence&author=W%20Yang&author=LR%20Trevino&author=JJ%20Yang&author=P%20Scheet&author=CH%20Pui&volume=24&publication_year=2010&pages=894-896&pmid=20054350&doi=10.1038/leu.2009.277&)

44. Yue P, Melamud E, Moult J 2006. SNPs3D: Candidate gene and SNP selection for association studies. BMC Bioinformatics 7: 166 doi: 10.1186/1471-2105-7-166  [DOI](https://doi.org/10.1186/1471-2105-7-166) | [PMC free article](/articles/PMC1435944/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16551372/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Bioinformatics&title=SNPs3D:%20Candidate%20gene%20and%20SNP%20selection%20for%20association%20studies&author=P%20Yue&author=E%20Melamud&author=J%20Moult&volume=7&publication_year=2006&pages=166&pmid=16551372&doi=10.1186/1471-2105-7-166&)

45. Zhu Q, Ge D, Maia JM, Zhu M, Petrovski S, Dickson SP, Heinzen EL, Shianna KV, Goldstein DB 2011. A genome-wide comparison of the functional properties of rare and common genetic variants in humans. Am J Hum Genet 88: 458–468  [DOI](https://doi.org/10.1016/j.ajhg.2011.03.008) | [PMC free article](/articles/PMC3071924/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21457907/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=A%20genome-wide%20comparison%20of%20the%20functional%20properties%20of%20rare%20and%20common%20genetic%20variants%20in%20humans&author=Q%20Zhu&author=D%20Ge&author=JM%20Maia&author=M%20Zhu&author=S%20Petrovski&volume=88&publication_year=2011&pages=458-468&pmid=21457907&doi=10.1016/j.ajhg.2011.03.008&)
