# Common CYP2D6 polymorphisms affecting alternative splicing and transcription: long-range haplotypes with two regulatory variants modulate CYP2D6 activity

## Metadata
**Authors:** Danxin Wang, Ming J Poi, Xiaochun Sun, Andrea Gaedigk, J Steven Leeder, Wolfgang Sadee
**Journal:** Human Molecular Genetics
**Date:** 2013 Aug 28
**DOI:** [10.1093/hmg/ddt417](https://doi.org/10.1093/hmg/ddt417)
**PMID:** 23985325
**PMCID:** PMC3857955
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3857955/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC3857955/pdf/ddt417.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC3857955/pdf/ddt417.pdf)

## Abstract

Cytochrome P450 2D6 (CYP2D6) is involved in the metabolism of 25% of clinically used drugs. Genetic polymorphisms cause substantial variation in CYP2D6 activity and serve as biomarkers guiding drug therapy. However, genotype–phenotype relationships remain ambiguous except for poor metabolizers carrying null alleles, suggesting the presence of yet unknown genetic variants. Searching for regulatory CYP2D6 polymorphisms, we find that a SNP defining the CYP2D6*2 allele, rs16947 [R296C, 17–60% minor allele frequency (MAF)], previously thought to convey normal activity, alters exon 6 splicing, thereby reducing CYP2D6 expression at least 2-fold. In addition, two completely linked SNPs (rs5758550/rs133333, MAF 13–42%) increase CYP2D6 transcription more than 2-fold, located in a distant downstream enhancer region (>100 kb) that interacts with the CYP2D6 promoter. In high linkage disequilibrium (LD) with each other, rs16947 and the enhancer SNPs form haplotypes that affect CYP2D6 enzyme activity in vivo. In a pediatric cohort of 164 individuals, rs16947 alone (minor haplotype frequency 28%) was associated with reduced CYP2D6 metabolic activity (measured as dextromethorphan/metabolite ratios), whereas rs5758550/rs133333 alone (frequency 3%) resulted in increased CYP2D6 activity, while haplotypes containing both rs16947 and rs5758550/rs133333 were similar to the wild-type. Other alleles used in biomarker panels carrying these variants such as CYP2D6*41 require re-evaluation of independent effects on CYP2D6 activity. The occurrence of two regulatory variants of high frequency and in high LD, residing on a long haplotype, highlights the importance of gene architecture, likely shaped by evolutionary selection pressures, in determining activity of encoded proteins.

## RESULTS

### SNP rs16947 (CYP2D6*2, R296C, in exon 6) affects CYP2D6 exon 6 splicing and decreases CYP2D6 mRNA level

Upon genotyping 12 SNPs (11 coding region SNPs and 1 promoter SNP, [Supplementary Material, Fig. S1a](http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt417/-/DC1)), we measured the two major alternative SVs of *CYP2D6*, E3- SV lacking exon 3 and E6- SV lacking exon 6, in 58 human liver samples. rs16947 was the SNP most significantly associated with a 2-fold and a 3.3-fold increase in E3- and E6- SVs, respectively (Table [1](#DDT417TB1)). Since **41* has been reported to increase E6- SV formation ([12](#DDT417C12)), we compared E6- SV levels in livers carrying **2* (only rs16947) and *41 (rs16947 and rs28371725). Shown in Figure [1](#DDT417F1)A, *CYP2D6*2* carrier status revealed a significant gene dosage effect, with **2* homozygotes having highest E6- SV levels, followed by **2* heterozygotes and non-carriers. In contrast, CYP2D6**41* carrier status displayed a wider range of E6- SV levels in the non-carriers group and no significant differences between heterozygous and non-carriers (Fig. [1](#DDT417F1)B). Moreover, E6- SV levels did not differ significantly between **2* and **41* carriers regardless of whether they were either heterozygotes or homozygotes (*P* > 0.05), a result that differs from those reported previously ([12](#DDT417C12)). These results suggest that the *CYP2D6*2* allele (rs16947) alone accounts for altered E6- SV splicing, while the SNP characterizing *CYP2D6*41* (rs28371725) may play a minor role at the most.

### Table 1.

| Splice variants | Predictor | Regression β (level relative to the CYP2D6*1 reference) | P-value | R2 (%) |
| --- | --- | --- | --- | --- |
| E3- SV | 2850T (*2, rs16947) | 0.310 (2-fold) | 0.011 | 11 |
| 2988 A (*41, rs28371725) | 0.27 (1.8-fold) | 0.053 | 7 |  |
| E6- SV | 2850T (*2, rs16947) | 0.512 (3.3-fold) | <0.0001 | 25 |
| 1661 G (rs1058164) | −0.392 (0.41-fold) | 0.004 | 15 |  |
| 4180 G (rs1135840) | −0.392 (0.41-fold) | 0.004 | 15 |  |
| 2988 A (*41, rs28371725) | 0.418 (2.6-fold) | 0.007 | 13 |  |
| −1584 G (rs1080985) | 0.344 (2.4-fold) | 0.014 | 11 |  |

Table 1 Caption: Association of the CYP2D6 genotype with expression levels of SVs (log10 transformed) in 58 human liver samples

### Figure 1.

![Figure 1.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c9d3/3857955/236f3c71ce4d/ddt41701.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3857955_ddt41701.jpg)

Expression of CYP2D6 SV without exon 6 (E6-SV) grouped by CYP2D6*2 (rs16947, panel A) and CYP2D6*41 (rs16947 and rs28371725, panel B) haplotypes, in human liver samples. Samples with other genotypes were excluded. Heter, heterozygous carriers; homo, homozygous carriers. The differences between groups were analyzed by t-test.

We constructed *CYP2D6* genomic DNA expression plasmids (including all introns) with different genotypes/haplotypes ([Supplementary Material, Table S2a](http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt417/-/DC1)), known to yield full-length CYP2D6 mRNA and functional CYP2D6 protein in transfected cell culture ([12](#DDT417C12)), for transfection into HEK293 cells. Each haplotype/allele was represented by several constructs with additional SNPs considered not to affect function (see [Supplementary Material, Table S2a](http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt417/-/DC1) for sequence variations). Results obtained with four constructs representing the reference haplotype *CYP2D6*1* (clones 1a–d) were similar, and therefore, were combined. Consistent with the results obtained from liver samples, constructs containing rs16947 (clones 2a–c, 41a and 41b, [Supplementary Material, Table S2a](http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt417/-/DC1)) produced higher levels of E3- and E6- SVs than the reference *CYP2D6*1* (Fig. [2](#DDT417F2)A and B). Constructs representing *CYP2D6*4*, known to cause a splicing defect of exon 3 (rs3892097, 1846G>A, clones 4a–c, [Supplementary Material, Table S2a](http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt417/-/DC1)), decreased E3- SV, while increased formation of E6- SV was observed only with clone 4b, which also contains rs16947 (Fig. [2](#DDT417F2)A and B). *CYP2D6*41* (also carrying rs16947) had effects similar to those observed for *CYP2D6*2*, while *CYP2D6*10* (clone 10, P34S, 100C>T, rs1065852, [Supplementary Material, Table S2a](http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt417/-/DC1)) had no detectable effect (Fig. [2](#DDT417F2)). Similar results were obtained for E6- SV but not E3- SV upon transfection into HepG2 cells ([Supplementary Material, Fig. S2](http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt417/-/DC1)). These results also support the hypothesis that rs16947 rather than rs28371725 is mainly responsible for altered E6 and E3 splicing.

### Figure 2.

![Figure 2.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c9d3/3857955/7f09bf0bfc52/ddt41702.jpg)

Formation of SVs lacking exon 3 (E3-SV) (A) or exon 6 (E6-SV) (B) in HEK cells after transfection with CYP2D6 genomic DNA expression plasmids containing different genotypes/haplotypes. E3- SV and E6- SV were quantitated with PCR using fluorescently labeled primers, shown as percent of the total amount of CYP2D6 transcript. Data are mean ± SD of two independent experiments each performed with triplicates obtained from different clones of each construct: 1, combined results from four clones with CYP2D6*1 reference sequence; 2a–2c, 3 CYP2D6*2 clones; 4a–4b, 3 CYP2D6*4 clones; 41a–41b, 2 CYP2D6*41 clones; 10, 1 CYP2D6*10 clone (Supplementary Material, Table S2a lists sequence variation). Compared with CYP2D6*1, **P < 0.01, one-way ANOVA with Dunnett post-test.

E3- SV is an in-frame deletion of exon 3 shortening the translated protein product by 51 amino acids, while E6- SV encodes mRNA lacking 142 bases of exon 6 which results in a frame shift and creation of a premature termination codon 17 bases downstream. To test a role for nonsense-mediated RNA decay of the out-of-frame mRNA, we suppressed nonsense-mediated RNA decay with puromycin or cycloheximide, or with knockdown of nonsense-mediated RNA decay factor UPF1 using siRNA transfection ([19](#DDT417C19)), in HepG2 cells endogenously expressing the *CYP2D6*1* genotype. All three treatments increased E6- SV mRNA levels 2–3-fold but did not affect E3- SV ([Supplementary Material, Fig. S3a](http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt417/-/DC1)), indicating that E6- SV, but not E3- SVs, undergoes nonsense-mediated RNA decay. The half-life of full-length CYP2D6 mRNA and E6- SV, measured over 24 h, was estimated to be 4.8 and 2.3 h, respectively ([Supplementary Material, Fig. S3b](http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt417/-/DC1)). These results indicate that rs16947 reduces full-length CYP2D6 mRNA and increases the formation of E6- SV, which is then degraded via nonsense-mediated RNA decay.

To estimate the magnitude of the net effect of rs16947 and to compare the effects of rs16947 alone (**2*) versus rs16947 plus rs28371725 (**41*) on full-length CYP2D6 mRNA levels, we co-transfected *CYP2D6*1* and *CYP2D6*2 or CYP2D6*1* and *CYP2D6*41* genomic DNA expression constructs into HEK293 cells and measured allelic CYP2D6 mRNA expression using rs16947 as a marker SNP. Allelic RNA expression is a superior quantitative measure than total RNA for predicting the functional consequences of allelic variation as we have demonstrated in numerous publications ([20](#DDT417C20)–[23](#DDT417C23)), controlling for trans-acting factors that might affect mRNA expression in tissue samples or variations of inter-well transfection efficiency and amount of DNA being transfected in cell culture. Because rs16947 is located in exon 6, which is absent in E6- SV, the measured mRNA level represents full-length CYP2D6 mRNA when using rs16947 as marker. After normalization to the transfected plasmid DNA ratio, the allelic RNA ratios (C/T) were 2.8 ± 0.1 for CYP2D6*1/CYP2D6*2 co-transfection, and 2.4 ± 0.3 for CYP2D6*1/CYP2D6*41 co-transfection. This result supports the notion that rs16947 in **2* reduces CYP2D6 full-length mRNA at least 2-fold, and the combination of rs16947 and rs28371725 in **41* has similar effects on CYP2D6 mRNA expression compared with rs16947 alone.

### Enhancer region SNPs increase CYP2D6 mRNA transcription

Chromatin profiling in HepG2 cells (ENCODE data) ([24](#DDT417C24)) indicates that *CYP2D6* has a weak promoter, while two strong enhancer regions were identified at 126 kb upstream and 114 kb downstream of *CYP2D6*, respectively. Using chromatin immune-precipitation (CHIP) with an antibody against p300, a universal co-factor for transcription that binds to enhancer regions, both regions were enriched ∼10-fold with anti-p300 antibody (Fig. [3](#DDT417F3)A). Physical interactions between *CYP2D6* promoter and enhancers were tested with chromatin conformation capture (3C) assays ([25](#DDT417C25)). Screening a 300 kb region surrounding the *CYP2D6* promoter, the downstream enhancer showed the highest crosslinking frequency with the *CYP2D6* promoter (Fig. [3](#DDT417F3)B), indicating that it physically interacts with the *CYP2D6* promoter.

### Figure 3.

![Figure 3.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c9d3/3857955/aee8a149b4dd/ddt41703.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3857955_ddt41703.jpg)

Polymorphisms in a distal enhancer region affecting CYP2D6 transcription. (A) Chromatin immunoprecipitation (CHIP) revealed binding of p300 protein to upsteam (−126 kb) and downstream (+114 kb) enhancer regions. CHIP without p300 antibody served as negative control. (B) Chromatin conformation capture (3C) assay of long-range interactions between CYP2D6 promoter and adjacent regions, revealing a downstream (+114 kb) enhancer region. Relative crosslinking frequency of ligation products (12 HindIII fragments) was plotted according to distance (kb) from the CYP2D6 promoter. (C) Reporter gene assay testing downstream enhancer region activity. Variant alleles of rs5758550/rs133333 convey higher CYP2D6 promoter activity than the reference alleles in HepG2 cells. Data are mean ± SD, three independent experiments with duplicates. Compared with reference, ***P < 0.001 t-test.

Two completely linked SNPs, rs5758550 and rs133333, were found within the downstream enhancer region. Promoter reporter gene assays showed the minor SNP alleles conveyed substantially higher promoter activity compared with the reference allele (Fig. [3](#DDT417F3)C). SNPs rs5758550/rs133333 are in substantial LD with the promoter SNP rs1080985 (−1584C>G, [Supplementary Material, Fig. S1b](http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt417/-/DC1)), which had been tentatively associated with enhanced CYP2D6 mRNA/protein expression whereas reporter gene assays failed to confirm this ([12](#DDT417C12),[16](#DDT417C16)). Using a 4 kb upstream region (corresponding to 4–4462 bp in [AY545216](https://www.ncbi.nlm.nih.gov/nuccore/AY545216)) containing rs1080985 (−1584G) and additional SNPs ([Supplementary Material, Table S2b](http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt417/-/DC1)), we found that a haplotype containing rs1080985 (construct m3) had similar effects on transcription as *CYP2D6*1* ([Supplementary Material, Fig. S4](http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt417/-/DC1)). This finding is consistent with previous results suggesting a lack of effect of rs1080985 on gene transcription ([16](#DDT417C16)) and indicates that the observed association with increased CYP2D6 protein levels results from LD with the upstream enhancer SNPs.

### CYP2D6 mRNA expression is determined by both rs16947 and enhancer SNPs

Enhancer SNPs rs5758550/rs133333 (MAF 13–42% in different populations, [Supplemental Material, Table S3](http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt417/-/DC1)) are in high LD with rs16947 (MAF 33%) in Caucasian population ([Supplementary Material, Fig. S1b](http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt417/-/DC1) [and Table S3](http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt417/-/DC1)). Only combined consideration of both rs16947 and rs133333/rs5758550 can fully account for the pattern of allelic CYP2D6 mRNA expression imbalance we have observed in liver samples (all were from Caucasians except one) (Fig. [4](#DDT417F4)A). In livers heterozygous only for rs16947, the minor T allele was consistently associated with decreased mRNA levels (RNA ratio <1), while the haplotype carrying both rs16947 and enhancer SNPs increased RNA levels ∼2-fold compared with the *CYP2D6*1* reference (showing RNA ratios >1). Hence, the enhancer SNPs override the reduction in CYP2D6 mRNA expression conveyed by rs16947. One liver sample that did not carry either rs16947 (CC) or enhancer SNPs (AA) showed no allelic expression imbalance (RNA ratio close to 1), while a liver homozygous for rs16947 (TT) and heterozygous for the enhancer SNPs (GA) had the highest allelic RNA ratio (3-fold). Neither rs16947 nor the enhancer SNPs alone were associated with total CYP2D6 mRNA expression (*n* = 54, *P* > 0.05). Multiple linear regression analyses showed, however, that rs16947 was significantly associated with a 2.4-fold decrease (*P* = 0.011) and the enhancer SNPs with a 2-fold increase in RNA levels (*P* = 0.036) (Fig. [4](#DDT417F4)B and C). These results indicate that CYP2D6 mRNA expression is affected by both rs16947 and enhancer SNPs, in opposite directions.

### Figure 4.

![Figure 4.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c9d3/3857955/21ded7197c34/ddt41704.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3857955_ddt41704.jpg)

Allelic and total CYP2D6 mRNA expression in liver samples. (A) Allelic RNA or DNA ratios were measured in RNA and gDNA using markers (rs1065852, rs1058164 and rs16947), normalized to gDNA ratios, and expressed as ratio T/C (minor/major, rs16947) using a log10 scale. All samples were heterozygous for rs16947 (C/T) except two that were homozygous for reference (CC) or variant (TT) alleles as indicated. Livers carrying allele CYP2D6*4 were excluded. Data are mean ± SD, two independent experiments with duplicates. (B and C) Total CYP2D6 mRNA level grouped by rs16947 (B, adjusted by rs5758550/rs133333 genotype) or rs5758550/rs133333 (C, adjusted by rs16947 genotype) genotypes. The differences between groups were analyzed by multivariate regression.

### In vivo effects of rs16947 and enhancer SNPs

We tested genotype/haplotype effects on metabolic activity in a cohort of 164 children with/without attention deficit-hyperactivity disorder (ADHD) (see [Supplementary Material, Table S4](http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt417/-/DC1) for demographics). CYP2D6 enzyme activity is reflected by the urinary metabolic ratio of dextromethorphan/metabolites (DM/DX). Ten individuals carrying two null alleles constituting a poor metabolizer phenotype were removed from further analysis, leaving an effective study sample of 154. *CYP2D6* genotype frequencies and rs16947/enhancer SNP haplotypes are shown in [Supplementary Material, Table S5a and b](http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt417/-/DC1). Of all rs16947 containing alleles (excluding individuals with gene duplication), 68% also carried enhancer SNPs, while 28% carried rs16947 only. The majority of individuals carrying rs16947 but lacking the enhancer SNPs were *CYP2D6*41* or **29* alleles ([Supplemental Material, Table S5b](http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt417/-/DC1)). It is noted that four subjects (3% frequency) classified as *CYP2D6*35* or **45* carriers, typically assigned wild-type activity, carried the enhancer SNPs rs133333/rs5758550 but lacked rs16947. These subjects are predicted to have an ultra-rapid phenotype on the basis of the results presented in this report.

In individuals with a single functional *CYP2D6* allele (heterozygous for null alleles *CYP2D6*3*, **4*, *5, *6 or **40*), rs16947 carriers without enhancer SNPs rs133333/rs5758550 had higher log-transformed DM/DX ratios (lower enzyme activity), and rs16947 plus enhancer SNPs had similar ratio compared with reference genotype carriers (Fig. [5](#DDT417F5)). These results indicate that rs16947 reduced CYP2D6 enzyme activity while the enhancer SNPs compensated for the reduced function conveyed by rs16947. These results parallel the allelic mRNA ratios observed in the liver tissues of rs16947 only carriers. On the other hand, the *in vivo* enzyme activity of rs16947 and rs133333/rs5758550 haplotype carriers did not exceed *CYP2D6*1* activity, even though hepatic mRNA levels are elevated (Fig. [4](#DDT417F4)A). This result may be related to the reduced enzyme activity of the altered protein encoded by the non-synonymous SNP rs16947, as reported ([9](#DDT417C9),[10](#DDT417C10)).

### Figure 5.

![Figure 5.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c9d3/3857955/7ac0343775b7/ddt41705.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3857955_ddt41705.jpg)

Values of log (DM/DX) in individuals with haplotypes containing CYP2D6 rs16947 alone, reference genotypes or rs16947 in combination with the enhancer SNPs. Selected individuals had one null allele and one functional CYP2D6 gene to facilitate the detection of changes imparted by the enhancer SNPs. The differences between groups were analyzed by t-test. Data are presented as box plot. The box and horizontal line show the 25th and 75th percentiles and mean, and whiskers show the minimum and maximum values.

We then assigned a CYP2D6 metabolic activity score for each individual based on genotype using two models. In model 1, the CYP2D6 activity score was assigned according to guidelines based on accepted standards ([4](#DDT417C4)) ([Supplementary Material, Table S6](http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt417/-/DC1)). In model 2, we removed *CYP2D6*2, *41, *29, *35* and **45* while adding rs16947 and enhancer SNPs rs133333/rs5758550 as markers. As expected, the activity scores using both model 1 and model 2 were significantly associated with the log (DM/DX) (*P* < 0.001). In model 2, many individuals were reassigned to activity score groups predicting higher activity and thereby improving the linearized relationship with DM/DX ratios. Specifically, one individual designated as ultra-rapid metabolizer (activity score = 2.5) using model 1 was re-classified as extensive metabolizer (activity score = 2) and seven individuals (all with enhancer SNPs) are now designated as ultra-rapid metabolizer (score >2, Fig. [6](#DDT417F6)A and B), bringing the total number of ultra-rapid metabolizers to 12 subjects. Moreover, model 1 yielded a significant difference between IM versus EM phenotypes (*P* < 0.0001) but not EM versus UM groups (*P* = 0.10). In contrast, model 2 gave significant differences between all three groups (IM versus EM, *P* < 0.0001; EM versus UM, *P* = 0.021). Interestingly, two individuals with a *CYP2D6*1/*2* genotype but lacking the enhancer SNPs had the highest log DM/DX ratio (lowest enzyme activity) in the *CYP2D6*1/*2* group, while two individuals classified as *CYP2D6*1/*41* with enhancer SNPs showed the lowest log DM/DX ratio (highest enzyme activity) in the *CYP2D6*1/*41* group (Fig. [6](#DDT417F6)C). These results indicate that rs16947 and enhancer SNPs rs133333/rs5758550 predicted CYP2D6 activity more accurately than standard *CYP2D6* allele models.

### Figure 6.

![Figure 6.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c9d3/3857955/05904f9be00c/ddt41706.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3857955_ddt41706.jpg)

Relationship between log (DM/DX) and activity scores derived from genotype data for model 1 (A) and model 2 (B) (see Supplementary Material, Table S6 for additional details for models 1 and 2). Data are presented as box plot. The box and horizontal lines show the 25th and 75th percentiles and mean, and whiskers show the minimum and maximum values. (C) Log (DM/DX) for individuals with CYP2D6*1/*2 and *1/*41 genotypes. Open circles indicate the mean log (DM/DX). Note: all individuals with CYP2D6*1/*2 genotype also carried enhancer SNPs except two (highlighted by rectangle), while none of the individuals with *1/*41 genotype carried enhancer SNPs except two (oval circle). These outliers exemplify cases where the CYP2D6*2 and *41 classification fails accurately to predict metabolizer status.

## DISCUSSION

While the *CYP2D6* gene locus has been extensively studied, regulatory polymorphisms affecting activity have remained poorly characterized. We have now identified two distinct mechanisms regulating *CYP2D6* expression: rs16947 reduces CYP2D6 activity by altering exon 6 splicing and downstream enhancer SNPs rs5758550/rs133333 enhance transcription. The present study illustrates a robust approach to determine the effects of a distant enhancer region on its protein-coding target gene, involving associations with gene expression, analysis of allelic mRNA expression exclusively testing *cis-*acting factors, chromatin-immunoprecipitation and chromatin conformation capture (3C) assays. Separated by a large distance in the *CYP2D6* locus, these two regulatory variants are nevertheless in high LD with each other, a strong indicator of evolutionary selection pressures shaping this gene's architecture. If under positive selection, such two-allele relationships presented here are likely also to occur in other gene loci. CYP2D6 activity was determined by haplotypes containing either rs16947 alone (decreased activity, 28% minor haplotype frequency), rs16947 plus rs133333/rs5758550 (similar to the wild-type) or the enhancer SNPs alone (3% frequency, with enhanced activity). Each of these alleles may carry less frequent additional variants that need to be re-assessed as to their functional activity. Incorporating these findings into biomarker panels will enable predicting CYP2D6 activity more accurately compared with those currently utilized.

### Enhanced exon 6 splicing by rs16947

The *CYP2D6*2* allele is considered to be a fully functional allele ([4](#DDT417C4),[5](#DDT417C5)), although a recent pharmacokinetic study suggested that it is associated with reduced activity, at least with dextromethorphan as the probe substrate, possibly due to the R296C substitution caused by rs16947 ([26](#DDT417C26)). Our results, however, demonstrate that rs16947 decreases CYP2D6 mRNA levels by enhancing exon 6 skipping, which abrogates CYP2D6 activity. The location of rs16947 in exon 6 is consistent with reports demonstrating that disruption of exonic splicing enhancer sites promotes exon skipping ([27](#DDT417C27)). In other studies, however, rs16947 did not alter, or was associated with, increased mRNA expression *in vivo* compared with *CYP2D6*1* ([8](#DDT417C8),[12](#DDT417C12),[15](#DDT417C15)), suggesting the presence of yet unidentified additional regulatory variants.

### Far-downstream enhancer SNPs rs5758550/rs133333

Data from The Encyclopedia of DNA Elements (ENCODE) project indicate that distal enhancers can regulate gene expression via long-range interactions ([28](#DDT417C28)). We identified two distal enhancers, one upstream and one downstream of *CYP2D6*, but only the downstream enhancer physically interacts with the proximal *CYP2D6* promoter region in a 3C assay. To investigate the effect of such a distal enhancer sites on its target gene, we employed allelic RNA expression analysis, a powerful tool to identify *ci*s-acting regulatory variants ([20](#DDT417C20)–[23](#DDT417C23)), regardless of the distance between regulatory site and target gene. This approach identified two completely linked SNPs, rs5758550/rs133333, which fully accounted for increased CYP2D6 mRNA expression observed by allelic mRNA analysis in livers. More than half of the subjects carrying rs16947 present in *CYP2D6*2* also contained rs5758550/rs133333, accounting for increased mRNA expression *in vivo* reported previously ([12](#DDT417C12)) and in this study. On the other hand, the amino acid substitution R296C caused by rs16947 was found to be associated with 30–50% reduced enzyme activity ([9](#DDT417C9),[10](#DDT417C10),[29](#DDT417C29)). This could resolve the discrepancy between enhanced mRNA expression of *CYP2D6*2* alleles containing the enhancer SNPs, with unchanged enzyme activity *in vivo* compared with *CYP2D6*1*. Enhancer SNP carriers without rs16947 (3% frequency) have increased mRNA expression, and by implication enzyme activity, but because of the relatively small number of subjects with the downstream enhancer SNPs alone in our study, the *in vivo* effect requires further investigation.

### Regulatory variants shown not to alter CYP2D6 expression: rs8138080, rs1080985 and CYP2D6*41 SNP rs28371725

Previous reports have revealed a significant association between CYP2D6 mRNA levels and another far downstream SNP, rs8138080 ([17](#DDT417C17),[18](#DDT417C18)), which is in high LD (*D*′ = 1) with rs5758550/rs133333, located ∼10 kb further downstream, but present at higher MAF (∼40%). Discrepancies with the observed allelic mRNA ratios in livers rule out rs8138080 as a causative variant. Moreover, in agreement with previous studies ([12](#DDT417C12),[16](#DDT417C16)), we did not find changes in promoter activity with rs1080985 (−1584 C>G); this SNP was not significantly associated with CYP2D6 mRNA levels even after adjusting for rs16947. The higher CYP2D6 mRNA expression levels previously associated with −1584G ([15](#DDT417C15)) can be accounted for by partial LD with the rs5758550/rs133333 enhancer SNPs ([Supplementary Material, Fig. S1b](http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt417/-/DC1)), again attesting to the very long haplotypes characteristic of the *CYP2D6* locus.

*CYP2D6*41* containing rs16947 and another intron 6 SNP, rs28371725, were considered to convey reduced CYP2D6 mRNA/protein activity ([8](#DDT417C8),[12](#DDT417C12)). However, we show here that *CYP2D6*41* does not appear to have significant independent effects on CYP2D6 mRNA expression other than that exerted by rs16947. Previously, *CYP2D6*41* has been associated with an increased level of E6- SV resulting in reduced CYP2D6 expression and function ([12](#DDT417C12)). Our results provide evidence that this effect was largely caused by rs16947 alone rather than that the intron 6 SNP rs28371725 or the combination of rs16947 and rs28371725. Distinct from *CYP2D6**2 haplotypes that often contain enhancer SNPs, *CYP2D6*41* haplotypes largely exclude the downstream enhancer SNPs ([Supplementary Material, Table S5b](http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt417/-/DC1)). Therefore, *CYP2D6*41* can serve as a tagging SNP for identifying haplotypes containing only rs16947 without enhancer SNPs, thereby predicting an intermediate metabolizer phenotype. However, the LD between *CYP2D6*41* SNP rs28371725 and enhancer SNPs likely differs between populations, rendering this allele unsuitable as a surrogate marker. This contention is supported by our finding that two *CYP2D6*1/*41* subjects (both are African Americans) also carrying enhancer SNPs had the highest CYP2D6 enzyme activity in the *CYP2D6**1/*41 group, while two *CYP2D6*1/*2* subjects without enhancer SNPs had the lowest enzyme activity among all CYP2D6 **1/*2* subjects (Fig. [6](#DDT417F6)C).

### Relevance of gene loci with two or more functional variants with high LD in long-range haplotype blocks

The use of robust phenotypes, such as allelic mRNA ratios and drug levels, has enabled the discovery of hidden regulatory variants in *CYP2D6* that reveal a unique architecture of this gene locus: both regulatory variants are frequent, more than 100 kb apart, and yet in high LD. These characteristics are a hallmark of evolutionary selection pressures likely to impinge on CYP enzymes as a primary defense against environmental toxins ([30](#DDT417C30)). Such genes are likely to accumulate several adaptive mutations, and certainly, *CYP2D6* ranks as one of the most polymorphic genes. The example revealed here suggests selection of a haplotype of two regulatory variants with opposing effects in LD with each other but with different allele frequencies, consistent with co-evolution between these distant variants. Could this phenomenon be pervasive among genes under strong selection pressures that are modified by changing environmental factors? We have identified regulatory variants in a number of genes, discovering similar long-range allele interactions in two additional gene loci: *CETP* encoding cholesterylester transfer protein ([31](#DDT417C31)) and *CHRNA5* encoding the alpha 5 nicotinic receptor ([32](#DDT417C32)). In each of these gene loci, frequent distant enhancer region variants (6–15 kb upstream) occur in strong negative LD with a SNP in the coding region affecting splicing (*CETP*) or with a non-synonymous SNP altering protein function (*CHRNA5*), in each case separated by 20–30 kb. Understanding the dynamic interactions between these variants, either on the same or on opposite haplotypes, is critical to revealing the overall effect of a gene locus on relevant phenotypes. Occurrence of these interactions at three gene loci among less than 15 studied in sufficient detail in our group suggests that distal regulatory variants are frequent and may form haplotypes with unique properties, particularly in genes under evolutionary selection. Incorporating such gene architectures into GWAS analyses is critical for correctly estimating the contributions of genetic factors to human phenotypes.

### Clinical implications

Guidelines published by the Clinical Pharmacogenomics Implementation Consortium (CPIC) have been developed to predict metabolizer status for *CYP2D6* genetic tests for codeine and tricyclic antidepressants therapy ([4](#DDT417C4),[5](#DDT417C5)). The guideline includes several rs16947 containing alleles/haplotypes as biomarkers in addition to *CYP2D6*2* and **41*, namely *CYP2D *17, *29, *35* and **45. CYP2D6*35* and **45* are currently considered to convey normal enzyme activity with activity score of 1 ([4](#DDT417C4),[5](#DDT417C5)), but being in different LD with rs16947 and downstream enhancer SNPs ([Supplementary Material, Table S5b](http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt417/-/DC1)), their categorizations without taking the rs16947 and enhancer SNPs into account may lead to over- or under-predicting CYP2D6 activity. Similarly, *CYP2D6*2*9 (V136I and V338M) had little effect on CYP2D6 enzyme activity in cDNA transfected cell culture ([33](#DDT417C33)) but is considered a reduced function allele with activity score of 0.5 ([4](#DDT417C4),[5](#DDT417C5),[34](#DDT417C34)), possibly confounded by rs16947 and downstream enhancer SNPs. Most (86%) *CYP2D6*29* alleles carry rs16947 without the enhancer SNP, while 14% have both ([Supplementary Material, Table S5b](http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt417/-/DC1)). On the other hand, CYP2D6 activity conveyed by the *CYP2D6*17* allele appears to be mainly determined by its hallmark T107I substitution, because cDNA transfections have revealed over 70% decreased CYP2D6 enzyme activity ([9](#DDT417C9),[10](#DDT417C10),[29](#DDT417C29),[35](#DDT417C35)). Most *CYP2D6*17* haplotypes (93%) contain both rs16947 and the downstream enhancer SNPs ([Supplementary Material, Table S5b](http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt417/-/DC1)). The few *CYP2D6*17* alleles (7%) containing only rs16947 may cause even greater reduction in enzyme activity. We conclude that at present, *CYP2D6*2*, **41*, **29*, **35* and **45* are inadequate for inferring CYP2D6 activity *in vivo,* since CYP2D6 activity is affected by haplotypes containing rs16947 and the downstream enhancer SNPs rs5758550/rs133333. Therefore, we suggest that the clinical CYP2D6 biomarker panel be redesigned and simplified to enhance predictive accuracy. We propose to classify *CYP2D6*9, *10, *17* and alleles carrying only rs16947 as reduced function alleles, alleles carrying both rs16947 and enhancer SNPs as wild-type function alleles, and alleles carrying only the enhancer SNPs rs5758550/rs133333 as having increased function ([Supplementary Material, Table S6](http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt417/-/DC1)). Predictions from this model result in a better fit for the measured metabolic activities (Fig. [6](#DDT417F6)), and importantly, appear more accurately to identify subjects as ultra-rapid metabolizers with an overall activity score >2. Before this revised model is utilized, however, our findings warrant replication in larger cohorts.

In summary, we have identified and functionally characterized two regulatory variants in *CYP2D6*, the exon 6 SNP rs16947 and two downstream enhancer SNPs rs575550/rs133333. Haplotypes comprising these two loci determine the enzyme activity of CYP2D6 *in vivo,* suggesting that their inclusion into genotyping panels will enable predicting CYP2D6 activity with greater accuracy.

## MATERIALS AND METHODS

### Tissue, DNA and RNA samples

Fifty-eight human liver biopsy samples (55 Caucasians and 3 African Americans) were obtained from the Cooperative Human Network Midwestern and Western Division, under the approval of The Ohio State University Institutional Review Board. Preparation of genomic DNA and RNA from livers or cell cultures was performed as described ([20](#DDT417C20)). cDNA was prepared using gene specific primers and oligo(dT) ([20](#DDT417C20)).

### Genotyping

SNPs were genotyped using multiplex ([36](#DDT417C36)) or single primer extension assays using primers shown in [Supplementary Material, Table S7](http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt417/-/DC1). The pediatric study cohort was genotyped as described earlier ([3](#DDT417C3)) and included testing for gene duplications ([37](#DDT417C37)), the presence of hybrid genes ([38](#DDT417C38),[39](#DDT417C39)) and copy number variation analysis ([40](#DDT417C40)). Genotype analysis for rs5758550 and rs133333 was carried out using TaqMan assays obtained from Life Technologies (Foster City, CA, USA) using gDNA isolated with a Qiagen Blood Mini Kit.

### CYP2D6 total mRNA and SVs quantification

Total CYP2D6 mRNA or SVs were measured using real-time PCR with specific primers ([Supplementary Material, Table S7](http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt417/-/DC1)) and SYBR PCR master mix (Life Technologies) using β-Actin and GAPDH mRNA as internal controls ([21](#DDT417C21)). Relative amounts of SVs were measured by PCR amplification of cDNA using fluorescently labeled primers, followed by separation of PCR products on an ABI 3730 DNA analyzer (Life Technologies) ([41](#DDT417C41)).

### Quantitative analysis of allelic ratios in genomic DNA and mRNA using a primer extension method (SNaPShot)

SNaPShot assays were performed as described ([20](#DDT417C20)). Specific primers shown in [Supplementary Material, Table S7](http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt417/-/DC1) served to amplify nearly full-length CYP2D6 mRNA. The entire *CYP2D6* gene (5.1 kb) amplified from genomic DNA served as internal control to normalize allelic RNA ratios. *CYP2D6* copy number variations were not detectable in the livers tested by SNapShot assays and real-time PCR ([42](#DDT417C42)). Samples heterozygous for at least one of the three marker SNPs rs1065852 (100C>T, P34S), rs1058164 (1661G>C) and rs16947 (2850C>T, R296C) were selected for measuring allelic RNA expression. Results from each marker gave similar results for the same liver and were combined.

### CYP2D6 genomic DNA expression and reporter gene constructs

The entire *CYP2D6* gene (5.1 kb, including introns) was PCR amplified with high fidelity Advantage HD polymerase (Clontech, Mountain view, CA, USA) from liver DNA samples with different genotypes and cloned into pcDNA3 vector using Infusion cloning kit (Clontech) according to the manufacturer's protocol. Fragments of the promoter region (4.3 kb) and downstream enhancer (3.1 kb) regions were also PCR amplified from liver DNA samples and cloned into pGL3 vectors (see [Supplementary Material, Table S7](http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt417/-/DC1) for PCR primers). All constructs were sequenced to ensure absence of any random mutations during PCR amplification.

### Reporter gene assay

Cells were transfected with 1 µg plasmid DNA and luciferase activities measured 48 h post-transfection with Dual-Glo luciferase assay kit (Promega, Madison, WI, USA) on a fluorescence plate reader (PerkinElmer Life and Analytical Science, Waltham, MA, USA). As internal controls, *Renilla* luciferase constructs driven by a TK promoter were co-transfected with promoter constructs at 1:5 ratio.

### Cell culture and transfection

Cells were cultured at 37°C in a humidified incubator at 5% CO_2_ in DMEM (HepG2) or DMEM/F12 (HEK293) medium supplemented with 10% fetal bovine serum, 100 U/ml penicillin and 100 µg/ml streptomycin. The day before transfection, cells were plated into 12-well plates. Transfection of plasmid DNA was performed using lipofectamine 2000 (HEK) (Invitrogen Life Technologies, Carlsbad, CA, USA) or Fugene HD transfection reagent (HepG2). Transfection of UPF1 siRNA was performed using TransIT-TKO transfection reagent (Mirus, Madison, WI, USA). Cells were harvested 48 h after transfection for total RNA preparation. Some cells were treated with puromycin (30 µg/ml) or cycloheximide (10 µg/ml) for 6 h, or actinomycin D (20 µg/ml) for different time periods before harvesting, and RNA was prepared ([20](#DDT417C20)). In genomic *CYP2D6* DNA expression plasmid transfection experiments, mRNA was purified from total RNA using PolyATtract mRNA isolation systems (Promega) to avoid contamination of total RNA with plasmid DNA.

### Chromatin immuno-precipitation (CHIP) assay

CHIP assay was performed in HepG2 cells using Chip-IT Express Enzymatic kit (Active Motif, Carlsbad, CA, USA) and anti-P300 antibody. PCR primers are shown in [Supplementary Material, Table S7](http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt417/-/DC1).

### Chromatin conformation capture assay (3C assay)

The 3C assays followed by quantitative PCR were performed as described ([43](#DDT417C43)). A total of 12 *Hin*dIII cutting sites in a 300 kb region surrounding the *CYP2D6* promoter were assayed with TaqMan probes ([Supplementary Material, Table S7](http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt417/-/DC1)).

### In vivo dextromethorphan phenotyping study

CYP2D6 phenotype data were obtained for 164 children with and without ADHD participating in an ongoing longitudinal study assessing the effect of puberty on dextromethorphan metabolism. Subjects were administered a single 0.5 mg/kg dose of dextromethorphan (Children's Robitussin Cough, 7.5 mg/5 ml) after an overnight fasting, and urine was collected for 4 h after dosing. CYP2D6 phenotype is expressed as the urinary ratio of dextromethorphan to the CYP2D6-dependent *O-*demethylated product dextrorphan (DM/DX ratio) in the pooled urine sample. The data represent the average of seven measurements (obtained every 6 months for 3 years). The study was approved by the Children's Mercy Hospital Pediatric Institutional Review Board, and is registered as trial [NCT01118858](https://clinicaltrials.gov/ct2/show/NCT01118858) at ClinicalTrials.gov.

### Data analysis

Haplotype structure and LD plots were generated using Helix-Tree (Golden Helix, Bozeman, MT, USA) and Haploview. Associations between genotype and the level of CYP2D6 total mRNA expression or SVs were estimated with a linear regression model using Minitab software. Statistical analysis of allelic DNA and RNA ratios or difference in means was performed using Prism (GraphPad Software Inc., San Diego, CA, USA). Data are expressed as mean ± SD.

## SUPPLEMENTARY MATERIAL

[Supplementary Material is available at HMG online](http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt417/-/DC1).

*Conflict of Interest statement*. None declared.

## FUNDING

This work was supported by National Institute of Health (NIH) grants (U01 GM 092655 to D.W. and W.S., R01 HD 058556 to J.S.L.), and partially by grant from the National Center for Research Resources (UL1RR025755). The content is solely the responsibility of the authors and does not necessarily represent the official views of the National Center for Research Resources or the National Institutes of Health.

## Supplementary Material

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Phillips K.A., Veenstra D.L., Oren E., Lee J.K., Sadee W. Potential role of pharmacogenomics in reducing adverse drug reactions: a systematic review. JAMA. 2001;286:2270–2279. doi: 10.1001/jama.286.18.2270.  [DOI](https://doi.org/10.1001/jama.286.18.2270) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11710893/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA&title=Potential%20role%20of%20pharmacogenomics%20in%20reducing%20adverse%20drug%20reactions:%20a%20systematic%20review&author=K.A.%20Phillips&author=D.L.%20Veenstra&author=E.%20Oren&author=J.K.%20Lee&author=W.%20Sadee&volume=286&publication_year=2001&pages=2270-2279&pmid=11710893&doi=10.1001/jama.286.18.2270&)

2. Griese E.U., Zanger U.M., Brudermanns U., Gaedigk A., Mikus G., Morike K., Stuven T., Eichelbaum M. Assessment of the predictive power of genotypes for the in-vivo catalytic function of CYP2D6 in a German population. Pharmacogenetics. 1998;8:15–26. doi: 10.1097/00008571-199802000-00003.  [DOI](https://doi.org/10.1097/00008571-199802000-00003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9511177/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Assessment%20of%20the%20predictive%20power%20of%20genotypes%20for%20the%20in-vivo%20catalytic%20function%20of%20CYP2D6%20in%20a%20German%20population&author=E.U.%20Griese&author=U.M.%20Zanger&author=U.%20Brudermanns&author=A.%20Gaedigk&author=G.%20Mikus&volume=8&publication_year=1998&pages=15-26&pmid=9511177&doi=10.1097/00008571-199802000-00003&)

3. Gaedigk A., Simon S.D., Pearce R.E., Bradford L.D., Kennedy M.J., Leeder J.S. The CYP2D6 activity score: translating genotype information into a qualitative measure of phenotype. Clin. Pharmacol. Ther. 2008;83:234–242. doi: 10.1038/sj.clpt.6100406.  [DOI](https://doi.org/10.1038/sj.clpt.6100406) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17971818/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=The%20CYP2D6%20activity%20score:%20translating%20genotype%20information%20into%20a%20qualitative%20measure%20of%20phenotype&author=A.%20Gaedigk&author=S.D.%20Simon&author=R.E.%20Pearce&author=L.D.%20Bradford&author=M.J.%20Kennedy&volume=83&publication_year=2008&pages=234-242&pmid=17971818&doi=10.1038/sj.clpt.6100406&)

4. Crews K.R., Gaedigk A., Dunnenberger H.M., Klein T.E., Shen D.D., Callaghan J.T., Kharasch E.D., Skaar T.C. Clinical Pharmacogenetics Implementation Consortium (CPIC) guidelines for codeine therapy in the context of cytochrome P450 2D6 (CYP2D6) genotype. Clin. Pharmacol. Ther. 2012;91:321–326. doi: 10.1038/clpt.2011.287.  [DOI](https://doi.org/10.1038/clpt.2011.287) | [PMC free article](/articles/PMC3289963/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22205192/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Clinical%20Pharmacogenetics%20Implementation%20Consortium%20(CPIC)%20guidelines%20for%20codeine%20therapy%20in%20the%20context%20of%20cytochrome%20P450%202D6%20(CYP2D6)%20genotype&author=K.R.%20Crews&author=A.%20Gaedigk&author=H.M.%20Dunnenberger&author=T.E.%20Klein&author=D.D.%20Shen&volume=91&publication_year=2012&pages=321-326&pmid=22205192&doi=10.1038/clpt.2011.287&)

5. Hicks J.K., Swen J.J., Thorn C.F., Sangkuhl K., Kharasch E.D., Ellingrod V.L., Skaar T.C., Muller D.J., Gaedigk A., Stingl J.C. Clinical Pharmacogenetics Implementation Consortium (CPIC) guidelines for CYP2D6 and CYP2C19 genotypes and dosing of tricyclic antidepressants. Clin. Pharmacol. Ther. 2013;93:402–408. doi: 10.1038/clpt.2013.2.  [DOI](https://doi.org/10.1038/clpt.2013.2) | [PMC free article](/articles/PMC3689226/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23486447/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Clinical%20Pharmacogenetics%20Implementation%20Consortium%20(CPIC)%20guidelines%20for%20CYP2D6%20and%20CYP2C19%20genotypes%20and%20dosing%20of%20tricyclic%20antidepressants&author=J.K.%20Hicks&author=J.J.%20Swen&author=C.F.%20Thorn&author=K.%20Sangkuhl&author=E.D.%20Kharasch&volume=93&publication_year=2013&pages=402-408&pmid=23486447&doi=10.1038/clpt.2013.2&)

6. Scott S.A., Sangkuhl K., Gardner E.E., Stein C.M., Hulot J.S., Johnson J.A., Roden D.M., Klein T.E., Shuldiner A.R. Clinical Pharmacogenetics Implementation Consortium guidelines for cytochrome P450-2C19 (CYP2C19) genotype and clopidogrel therapy. Clin. Pharmacol. Ther. 2011;90:328–332. doi: 10.1038/clpt.2011.132.  [DOI](https://doi.org/10.1038/clpt.2011.132) | [PMC free article](/articles/PMC3234301/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21716271/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Clinical%20Pharmacogenetics%20Implementation%20Consortium%20guidelines%20for%20cytochrome%20P450-2C19%20(CYP2C19)%20genotype%20and%20clopidogrel%20therapy&author=S.A.%20Scott&author=K.%20Sangkuhl&author=E.E.%20Gardner&author=C.M.%20Stein&author=J.S.%20Hulot&volume=90&publication_year=2011&pages=328-332&pmid=21716271&doi=10.1038/clpt.2011.132&)

7. Bock K.W., Schrenk D., Forster A., Griese E.U., Morike K., Brockmeier D., Eichelbaum M. The influence of environmental and genetic factors on CYP2D6, CYP1A2 and UDP-glucuronosyltransferases in man using sparteine, caffeine, and paracetamol as probes. Pharmacogenetics. 1994;4:209–218. doi: 10.1097/00008571-199408000-00005.  [DOI](https://doi.org/10.1097/00008571-199408000-00005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7987405/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=The%20influence%20of%20environmental%20and%20genetic%20factors%20on%20CYP2D6,%20CYP1A2%20and%20UDP-glucuronosyltransferases%20in%20man%20using%20sparteine,%20caffeine,%20and%20paracetamol%20as%20probes&author=K.W.%20Bock&author=D.%20Schrenk&author=A.%20Forster&author=E.U.%20Griese&author=K.%20Morike&volume=4&publication_year=1994&pages=209-218&pmid=7987405&doi=10.1097/00008571-199408000-00005&)

8. Raimundo S., Toscano C., Klein K., Fischer J., Griese E.U., Eichelbaum M., Schwab M., Zanger U.M. A novel intronic mutation, 2988G>A, with high predictivity for impaired function of cytochrome P450 2D6 in white subjects. Clin. Pharmacol. Ther. 2004;76:128–138. doi: 10.1016/j.clpt.2004.04.009.  [DOI](https://doi.org/10.1016/j.clpt.2004.04.009) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15289790/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=A%20novel%20intronic%20mutation,%202988G>A,%20with%20high%20predictivity%20for%20impaired%20function%20of%20cytochrome%20P450%202D6%20in%20white%20subjects&author=S.%20Raimundo&author=C.%20Toscano&author=K.%20Klein&author=J.%20Fischer&author=E.U.%20Griese&volume=76&publication_year=2004&pages=128-138&pmid=15289790&doi=10.1016/j.clpt.2004.04.009&)

9. Yu A., Kneller B.M., Rettie A.E., Haining R.L. Expression, purification, biochemical characterization, and comparative function of human cytochrome P450 2D6.1, 2D6.2, 2D6.10, and 2D6.17 allelic isoforms. J. Pharmacol. Exp. Ther. 2002;303:1291–1300. doi: 10.1124/jpet.102.039891.  [DOI](https://doi.org/10.1124/jpet.102.039891) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12438554/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Pharmacol.%20Exp.%20Ther.&title=Expression,%20purification,%20biochemical%20characterization,%20and%20comparative%20function%20of%20human%20cytochrome%20P450%202D6.1,%202D6.2,%202D6.10,%20and%202D6.17%20allelic%20isoforms&author=A.%20Yu&author=B.M.%20Kneller&author=A.E.%20Rettie&author=R.L.%20Haining&volume=303&publication_year=2002&pages=1291-1300&pmid=12438554&doi=10.1124/jpet.102.039891&)

10. Marcucci K.A., Pearce R.E., Crespi C., Steimel D.T., Leeder J.S., Gaedigk A. Characterization of cytochrome P450 2D6.1 (CYP2D6.1), CYP2D6.2, and CYP2D6.17 activities toward model CYP2D6 substrates dextromethorphan, bufuralol, and debrisoquine. Drug Metab. Dispos. 2002;30:595–601. doi: 10.1124/dmd.30.5.595.  [DOI](https://doi.org/10.1124/dmd.30.5.595) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11950793/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab.%20Dispos.&title=Characterization%20of%20cytochrome%20P450%202D6.1%20(CYP2D6.1),%20CYP2D6.2,%20and%20CYP2D6.17%20activities%20toward%20model%20CYP2D6%20substrates%20dextromethorphan,%20bufuralol,%20and%20debrisoquine&author=K.A.%20Marcucci&author=R.E.%20Pearce&author=C.%20Crespi&author=D.T.%20Steimel&author=J.S.%20Leeder&volume=30&publication_year=2002&pages=595-601&pmid=11950793&doi=10.1124/dmd.30.5.595&)

11. Sakuyama K., Sasaki T., Ujiie S., Obata K., Mizugaki M., Ishikawa M., Hiratsuka M. Functional characterization of 17 CYP2D6 allelic variants (CYP2D6.2, 10, 14A-B, 18, 27, 36, 39, 47–51, 53–55, and 57) Drug Metab. Dispos. 2008;36:2460–2467. doi: 10.1124/dmd.108.023242.  [DOI](https://doi.org/10.1124/dmd.108.023242) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18784265/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab.%20Dispos.&title=Functional%20characterization%20of%2017%20CYP2D6%20allelic%20variants%20(CYP2D6.2,%2010,%2014A-B,%2018,%2027,%2036,%2039,%2047%E2%80%9351,%2053%E2%80%9355,%20and%2057)&author=K.%20Sakuyama&author=T.%20Sasaki&author=S.%20Ujiie&author=K.%20Obata&author=M.%20Mizugaki&volume=36&publication_year=2008&pages=2460-2467&pmid=18784265&doi=10.1124/dmd.108.023242&)

12. Toscano C., Klein K., Blievernicht J., Schaeffeler E., Saussele T., Raimundo S., Eichelbaum M., Schwab M., Zanger U.M. Impaired expression of CYP2D6 in intermediate metabolizers carrying the *41 allele caused by the intronic SNP 2988G>A: evidence for modulation of splicing events. Pharmacogenet. Genomics. 2006;16:755–766. doi: 10.1097/01.fpc.0000230112.96086.e0.  [DOI](https://doi.org/10.1097/01.fpc.0000230112.96086.e0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17001295/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet.%20Genomics&title=Impaired%20expression%20of%20CYP2D6%20in%20intermediate%20metabolizers%20carrying%20the%20*41%20allele%20caused%20by%20the%20intronic%20SNP%202988G>A:%20evidence%20for%20modulation%20of%20splicing%20events&author=C.%20Toscano&author=K.%20Klein&author=J.%20Blievernicht&author=E.%20Schaeffeler&author=T.%20Saussele&volume=16&publication_year=2006&pages=755-766&pmid=17001295&doi=10.1097/01.fpc.0000230112.96086.e0&)

13. Denson J., Wu Y., Yang W., Zhang J. Inter-individual variation of several cytochrome P450 2D6 splice variants in human liver. Biochem. Biophys. Res. Commun. 2005;330:498–504. doi: 10.1016/j.bbrc.2005.03.010.  [DOI](https://doi.org/10.1016/j.bbrc.2005.03.010) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15796910/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem.%20Biophys.%20Res.%20Commun.&title=Inter-individual%20variation%20of%20several%20cytochrome%20P450%202D6%20splice%20variants%20in%20human%20liver&author=J.%20Denson&author=Y.%20Wu&author=W.%20Yang&author=J.%20Zhang&volume=330&publication_year=2005&pages=498-504&pmid=15796910&doi=10.1016/j.bbrc.2005.03.010&)

14. Raimundo S., Fischer J., Eichelbaum M., Griese E.U., Schwab M., Zanger U.M. Elucidation of the genetic basis of the common ‘intermediate metabolizer’ phenotype for drug oxidation by CYP2D6. Pharmacogenetics. 2000;10:577–581. doi: 10.1097/00008571-200010000-00001.  [DOI](https://doi.org/10.1097/00008571-200010000-00001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11037799/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Elucidation%20of%20the%20genetic%20basis%20of%20the%20common%20%E2%80%98intermediate%20metabolizer%E2%80%99%20phenotype%20for%20drug%20oxidation%20by%20CYP2D6&author=S.%20Raimundo&author=J.%20Fischer&author=M.%20Eichelbaum&author=E.U.%20Griese&author=M.%20Schwab&volume=10&publication_year=2000&pages=577-581&pmid=11037799&doi=10.1097/00008571-200010000-00001&)

15. Zanger U.M., Fischer J., Raimundo S., Stuven T., Evert B.O., Schwab M., Eichelbaum M. Comprehensive analysis of the genetic factors determining expression and function of hepatic CYP2D6. Pharmacogenetics. 2001;11:573–585. doi: 10.1097/00008571-200110000-00004.  [DOI](https://doi.org/10.1097/00008571-200110000-00004) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11668217/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Comprehensive%20analysis%20of%20the%20genetic%20factors%20determining%20expression%20and%20function%20of%20hepatic%20CYP2D6&author=U.M.%20Zanger&author=J.%20Fischer&author=S.%20Raimundo&author=T.%20Stuven&author=B.O.%20Evert&volume=11&publication_year=2001&pages=573-585&pmid=11668217&doi=10.1097/00008571-200110000-00004&)

16. Gaedigk A., Bhathena A., Ndjountche L., Pearce R.E., Abdel-Rahman S.M., Alander S.W., Bradford L.D., Rogan P.K., Leeder J.S. Identification and characterization of novel sequence variations in the cytochrome P4502D6 (CYP2D6) gene in African Americans. Pharmacogenomics J. 2005;5:173–182. doi: 10.1038/sj.tpj.6500305.  [DOI](https://doi.org/10.1038/sj.tpj.6500305) | [PMC free article](/articles/PMC1440720/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15768052/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J.&title=Identification%20and%20characterization%20of%20novel%20sequence%20variations%20in%20the%20cytochrome%20P4502D6%20(CYP2D6)%20gene%20in%20African%20Americans&author=A.%20Gaedigk&author=A.%20Bhathena&author=L.%20Ndjountche&author=R.E.%20Pearce&author=S.M.%20Abdel-Rahman&volume=5&publication_year=2005&pages=173-182&pmid=15768052&doi=10.1038/sj.tpj.6500305&)

17. Schadt E.E., Molony C., Chudin E., Hao K., Yang X., Lum P.Y., Kasarskis A., Zhang B., Wang S., Suver C., et al. Mapping the genetic architecture of gene expression in human liver. PLoS Biol. 2008;6:e107. doi: 10.1371/journal.pbio.0060107.  [DOI](https://doi.org/10.1371/journal.pbio.0060107) | [PMC free article](/articles/PMC2365981/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18462017/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Biol.&title=Mapping%20the%20genetic%20architecture%20of%20gene%20expression%20in%20human%20liver&author=E.E.%20Schadt&author=C.%20Molony&author=E.%20Chudin&author=K.%20Hao&author=X.%20Yang&volume=6&publication_year=2008&pages=e107&pmid=18462017&doi=10.1371/journal.pbio.0060107&)

18. Yang X., Zhang B., Molony C., Chudin E., Hao K., Zhu J., Gaedigk A., Suver C., Zhong H., Leeder J.S., et al. Systematic genetic and genomic analysis of cytochrome P450 enzyme activities in human liver. Genome Res. 2010;20:1020–1036. doi: 10.1101/gr.103341.109.  [DOI](https://doi.org/10.1101/gr.103341.109) | [PMC free article](/articles/PMC2909567/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20538623/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res.&title=Systematic%20genetic%20and%20genomic%20analysis%20of%20cytochrome%20P450%20enzyme%20activities%20in%20human%20liver&author=X.%20Yang&author=B.%20Zhang&author=C.%20Molony&author=E.%20Chudin&author=K.%20Hao&volume=20&publication_year=2010&pages=1020-1036&pmid=20538623&doi=10.1101/gr.103341.109&)

19. Mendell J.T., ap Rhys C.M., Dietz H.C. Separable roles for rent1/hUpf1 in altered splicing and decay of nonsense transcripts. Science. 2002;298:419–422. doi: 10.1126/science.1074428.  [DOI](https://doi.org/10.1126/science.1074428) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12228722/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Separable%20roles%20for%20rent1/hUpf1%20in%20altered%20splicing%20and%20decay%20of%20nonsense%20transcripts&author=J.T.%20Mendell&author=C.M.%20ap%20Rhys&author=H.C.%20Dietz&volume=298&publication_year=2002&pages=419-422&pmid=12228722&doi=10.1126/science.1074428&)

20. Wang D., Guo Y., Wrighton S.A., Cooke G.E., Sadee W. Intronic polymorphism in CYP3A4 affects hepatic expression and response to statin drugs. Pharmacogenomics J. 2011;11:274–286. doi: 10.1038/tpj.2010.28.  [DOI](https://doi.org/10.1038/tpj.2010.28) | [PMC free article](/articles/PMC3248744/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20386561/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J.&title=Intronic%20polymorphism%20in%20CYP3A4%20affects%20hepatic%20expression%20and%20response%20to%20statin%20drugs&author=D.%20Wang&author=Y.%20Guo&author=S.A.%20Wrighton&author=G.E.%20Cooke&author=W.%20Sadee&volume=11&publication_year=2011&pages=274-286&pmid=20386561&doi=10.1038/tpj.2010.28&)

21. Wang D., Johnson A.D., Papp A.C., Kroetz D.L., Sadee W. Multidrug resistance polypeptide 1 (MDR1, ABCB1) variant 3435C>T affects mRNA stability. Pharmacogenet. Genomics. 2005;15:693–704.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/16141795/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet.%20Genomics&title=Multidrug%20resistance%20polypeptide%201%20(MDR1,%20ABCB1)%20variant%203435C>T%20affects%20mRNA%20stability&author=D.%20Wang&author=A.D.%20Johnson&author=A.C.%20Papp&author=D.L.%20Kroetz&author=W.%20Sadee&volume=15&publication_year=2005&pages=693-704&pmid=16141795&)

22. Wang D., Para M.F., Koletar S.L., Sadee W. Human N-acetyltransferase 1 *10 and *11 alleles increase protein expression through distinct mechanisms and associate with sulfamethoxazole-induced hypersensitivity. Pharmacogenet. Genomics. 2011;21:652–664. doi: 10.1097/FPC.0b013e3283498ee9.  [DOI](https://doi.org/10.1097/FPC.0b013e3283498ee9) | [PMC free article](/articles/PMC3172334/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21878835/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet.%20Genomics&title=Human%20N-acetyltransferase%201%20*10%20and%20*11%20alleles%20increase%20protein%20expression%20through%20distinct%20mechanisms%20and%20associate%20with%20sulfamethoxazole-induced%20hypersensitivity&author=D.%20Wang&author=M.F.%20Para&author=S.L.%20Koletar&author=W.%20Sadee&volume=21&publication_year=2011&pages=652-664&pmid=21878835&doi=10.1097/FPC.0b013e3283498ee9&)

23. Wang D., Chen H., Momary K.M., Cavallari L.H., Johnson J.A., Sadee W. Regulatory polymorphism in vitamin K epoxide reductase complex subunit 1 (VKORC1) affects gene expression and warfarin dose requirement. Blood. 2008;112:1013–1021. doi: 10.1182/blood-2008-03-144899.  [DOI](https://doi.org/10.1182/blood-2008-03-144899) | [PMC free article](/articles/PMC2515137/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18523153/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Regulatory%20polymorphism%20in%20vitamin%20K%20epoxide%20reductase%20complex%20subunit%201%20(VKORC1)%20affects%20gene%20expression%20and%20warfarin%20dose%20requirement&author=D.%20Wang&author=H.%20Chen&author=K.M.%20Momary&author=L.H.%20Cavallari&author=J.A.%20Johnson&volume=112&publication_year=2008&pages=1013-1021&pmid=18523153&doi=10.1182/blood-2008-03-144899&)

24. Dunham I., Kundaje A., Aldred S.F., Collins P.J., Davis C.A., Doyle F., Epstein C.B., Frietze S., Harrow J., Kaul R., et al. An integrated encyclopedia of DNA elements in the human genome. Nature. 2012;489:57–74. doi: 10.1038/nature11247.  [DOI](https://doi.org/10.1038/nature11247) | [PMC free article](/articles/PMC3439153/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22955616/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=An%20integrated%20encyclopedia%20of%20DNA%20elements%20in%20the%20human%20genome&author=I.%20Dunham&author=A.%20Kundaje&author=S.F.%20Aldred&author=P.J.%20Collins&author=C.A.%20Davis&volume=489&publication_year=2012&pages=57-74&pmid=22955616&doi=10.1038/nature11247&)

25. Dekker J., Rippe K., Dekker M., Kleckner N. Capturing chromosome conformation. Science. 2002;295:1306–1311. doi: 10.1126/science.1067799.  [DOI](https://doi.org/10.1126/science.1067799) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11847345/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Capturing%20chromosome%20conformation&author=J.%20Dekker&author=K.%20Rippe&author=M.%20Dekker&author=N.%20Kleckner&volume=295&publication_year=2002&pages=1306-1311&pmid=11847345&doi=10.1126/science.1067799&)

26. Abduljalil K., Frank D., Gaedigk A., Klaassen T., Tomalik-Scharte D., Jetter A., Jaehde U., Kirchheiner J., Fuhr U. Assessment of activity levels for CYP2D6*1, CYP2D6*2, and CYP2D6*41 genes by population pharmacokinetics of dextromethorphan. Clin. Pharmacol. Ther. 2010;88:643–651. doi: 10.1038/clpt.2010.137.  [DOI](https://doi.org/10.1038/clpt.2010.137) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20881950/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Assessment%20of%20activity%20levels%20for%20CYP2D6*1,%20CYP2D6*2,%20and%20CYP2D6*41%20genes%20by%20population%20pharmacokinetics%20of%20dextromethorphan&author=K.%20Abduljalil&author=D.%20Frank&author=A.%20Gaedigk&author=T.%20Klaassen&author=D.%20Tomalik-Scharte&volume=88&publication_year=2010&pages=643-651&pmid=20881950&doi=10.1038/clpt.2010.137&)

27. Liu H.X., Cartegni L., Zhang M.Q., Krainer A.R. A mechanism for exon skipping caused by nonsense or missense mutations in BRCA1 and other genes. Nat. Genet. 2001;27:55–58. doi: 10.1038/83762.  [DOI](https://doi.org/10.1038/83762) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11137998/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet.&title=A%20mechanism%20for%20exon%20skipping%20caused%20by%20nonsense%20or%20missense%20mutations%20in%20BRCA1%20and%20other%20genes&author=H.X.%20Liu&author=L.%20Cartegni&author=M.Q.%20Zhang&author=A.R.%20Krainer&volume=27&publication_year=2001&pages=55-58&pmid=11137998&doi=10.1038/83762&)

28. Sanyal A., Lajoie B.R., Jain G., Dekker J. The long-range interaction landscape of gene promoters. Nature. 2012;489:109–113. doi: 10.1038/nature11279.  [DOI](https://doi.org/10.1038/nature11279) | [PMC free article](/articles/PMC3555147/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22955621/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=The%20long-range%20interaction%20landscape%20of%20gene%20promoters&author=A.%20Sanyal&author=B.R.%20Lajoie&author=G.%20Jain&author=J.%20Dekker&volume=489&publication_year=2012&pages=109-113&pmid=22955621&doi=10.1038/nature11279&)

29. Oscarson M., Hidestrand M., Johansson I., Ingelman-Sundberg M. A combination of mutations in the CYP2D6*17 (CYP2D6Z) allele causes alterations in enzyme function. Mol. Pharmacol. 1997;52:1034–1040. doi: 10.1124/mol.52.6.1034.  [DOI](https://doi.org/10.1124/mol.52.6.1034) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9415713/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Pharmacol.&title=A%20combination%20of%20mutations%20in%20the%20CYP2D6*17%20(CYP2D6Z)%20allele%20causes%20alterations%20in%20enzyme%20function&author=M.%20Oscarson&author=M.%20Hidestrand&author=I.%20Johansson&author=M.%20Ingelman-Sundberg&volume=52&publication_year=1997&pages=1034-1040&pmid=9415713&doi=10.1124/mol.52.6.1034&)

30. Sadee W. The relevance of "missing heritability" in pharmacogenomics. Clin. Pharmacol. Ther. 2012;92:428–430. doi: 10.1038/clpt.2012.116.  [DOI](https://doi.org/10.1038/clpt.2012.116) | [PMC free article](/articles/PMC3638892/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22910439/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=The%20relevance%20of%20%22missing%20heritability%22%20in%20pharmacogenomics&author=W.%20Sadee&volume=92&publication_year=2012&pages=428-430&pmid=22910439&doi=10.1038/clpt.2012.116&)

31. Papp A.C., Pinsonneault J.K., Wang D., Newman L.C., Gong Y., Johnson J.A., Pepine C.J., Kumari M., Hingorani A.D., Talmud P.J., et al. Cholesteryl Ester Transfer Protein (CETP) polymorphisms affect mRNA splicing, HDL levels, and sex-dependent cardiovascular risk. PLoS ONE. 2012;7:e31930. doi: 10.1371/journal.pone.0031930.  [DOI](https://doi.org/10.1371/journal.pone.0031930) | [PMC free article](/articles/PMC3293889/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22403620/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&title=Cholesteryl%20Ester%20Transfer%20Protein%20(CETP)%20polymorphisms%20affect%20mRNA%20splicing,%20HDL%20levels,%20and%20sex-dependent%20cardiovascular%20risk&author=A.C.%20Papp&author=J.K.%20Pinsonneault&author=D.%20Wang&author=L.C.%20Newman&author=Y.%20Gong&volume=7&publication_year=2012&pages=e31930&pmid=22403620&doi=10.1371/journal.pone.0031930&)

32. Smith R.M., Alachkar H., Papp A.C., Wang D., Mash D.C., Wang J.C., Bierut L.J., Sadee W. Nicotinic alpha5 receptor subunit mRNA expression is associated with distant 5′ upstream polymorphisms. Eur. J. Hum. Genet. 2011;19:76–83. doi: 10.1038/ejhg.2010.120.  [DOI](https://doi.org/10.1038/ejhg.2010.120) | [PMC free article](/articles/PMC2995013/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20700147/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Hum.%20Genet.&title=Nicotinic%20alpha5%20receptor%20subunit%20mRNA%20expression%20is%20associated%20with%20distant%205%E2%80%B2%20upstream%20polymorphisms&author=R.M.%20Smith&author=H.%20Alachkar&author=A.C.%20Papp&author=D.%20Wang&author=D.C.%20Mash&volume=19&publication_year=2011&pages=76-83&pmid=20700147&doi=10.1038/ejhg.2010.120&)

33. Wennerholm A., Johansson I., Hidestrand M., Bertilsson L., Gustafsson L.L., Ingelman-Sundberg M. Characterization of the CYP2D6*29 allele commonly present in a black Tanzanian population causing reduced catalytic activity. Pharmacogenetics. 2001;11:417–427. doi: 10.1097/00008571-200107000-00005.  [DOI](https://doi.org/10.1097/00008571-200107000-00005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11470994/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Characterization%20of%20the%20CYP2D6*29%20allele%20commonly%20present%20in%20a%20black%20Tanzanian%20population%20causing%20reduced%20catalytic%20activity&author=A.%20Wennerholm&author=I.%20Johansson&author=M.%20Hidestrand&author=L.%20Bertilsson&author=L.L.%20Gustafsson&volume=11&publication_year=2001&pages=417-427&pmid=11470994&doi=10.1097/00008571-200107000-00005&)

34. Wennerholm A., Dandara C., Sayi J., Svensson J.O., Abdi Y.A., Ingelman-Sundberg M., Bertilsson L., Hasler J., Gustafsson L.L. The African-specific CYP2D617 allele encodes an enzyme with changed substrate specificity. Clin. Pharmacol. Ther. 2002;71:77–88. doi: 10.1067/mcp.2002.120239.  [DOI](https://doi.org/10.1067/mcp.2002.120239) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11823760/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=The%20African-specific%20CYP2D617%20allele%20encodes%20an%20enzyme%20with%20changed%20substrate%20specificity&author=A.%20Wennerholm&author=C.%20Dandara&author=J.%20Sayi&author=J.O.%20Svensson&author=Y.A.%20Abdi&volume=71&publication_year=2002&pages=77-88&pmid=11823760&doi=10.1067/mcp.2002.120239&)

35. Shen H., He M.M., Liu H., Wrighton S.A., Wang L., Guo B., Li C. Comparative metabolic capabilities and inhibitory profiles of CYP2D6.1, CYP2D6.10, and CYP2D6.17. Drug Metab. Dispos. 2007;35:1292–1300. doi: 10.1124/dmd.107.015354.  [DOI](https://doi.org/10.1124/dmd.107.015354) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17470523/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab.%20Dispos.&title=Comparative%20metabolic%20capabilities%20and%20inhibitory%20profiles%20of%20CYP2D6.1,%20CYP2D6.10,%20and%20CYP2D6.17&author=H.%20Shen&author=M.M.%20He&author=H.%20Liu&author=S.A.%20Wrighton&author=L.%20Wang&volume=35&publication_year=2007&pages=1292-1300&pmid=17470523&doi=10.1124/dmd.107.015354&)

36. Sistonen J., Fuselli S., Levo A., Sajantila A. CYP2D6 genotyping by a multiplex primer extension reaction. Clin. Chem. 2005;51:1291–1295. doi: 10.1373/clinchem.2004.046466.  [DOI](https://doi.org/10.1373/clinchem.2004.046466) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15905314/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Chem.&title=CYP2D6%20genotyping%20by%20a%20multiplex%20primer%20extension%20reaction&author=J.%20Sistonen&author=S.%20Fuselli&author=A.%20Levo&author=A.%20Sajantila&volume=51&publication_year=2005&pages=1291-1295&pmid=15905314&doi=10.1373/clinchem.2004.046466&)

37. Gaedigk A., Ndjountche L., Divakaran K., Dianne Bradford L., Zineh I., Oberlander T.F., Brousseau D.C., McCarver D.G., Johnson J.A., Alander S.W., et al. Cytochrome P4502D6 (CYP2D6) gene locus heterogeneity: characterization of gene duplication events. Clin. Pharmacol. Ther. 2007;81:242–251. doi: 10.1038/sj.clpt.6100033.  [DOI](https://doi.org/10.1038/sj.clpt.6100033) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17259947/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Cytochrome%20P4502D6%20(CYP2D6)%20gene%20locus%20heterogeneity:%20characterization%20of%20gene%20duplication%20events&author=A.%20Gaedigk&author=L.%20Ndjountche&author=K.%20Divakaran&author=L.%20Dianne%20Bradford&author=I.%20Zineh&volume=81&publication_year=2007&pages=242-251&pmid=17259947&doi=10.1038/sj.clpt.6100033&)

38. Gaedigk A., Jaime L.K., Bertino J.S., Jr, Berard A., Pratt V.M., Bradfordand L.D., Leeder J.S. Identification of novel CYP2D7-2D6 hybrids: non-functional and functional variants. Front. Pharmacol. 2010;1:121. doi: 10.3389/fphar.2010.00121.  [DOI](https://doi.org/10.3389/fphar.2010.00121) | [PMC free article](/articles/PMC3153001/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21833166/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front.%20Pharmacol.&title=Identification%20of%20novel%20CYP2D7-2D6%20hybrids:%20non-functional%20and%20functional%20variants&author=A.%20Gaedigk&author=L.K.%20Jaime&author=J.S.%20Bertino&author=A.%20Berard&author=V.M.%20Pratt&volume=1&publication_year=2010&pages=121&pmid=21833166&doi=10.3389/fphar.2010.00121&)

39. Gaedigk A., Fuhr U., Johnson C., Berard L.A., Bradford D., Leeder J.S. CYP2D7–2D6 hybrid tandems: identification of novel CYP2D6 duplication arrangements and implications for phenotype prediction. Pharmacogenomics. 2010;11:43–53. doi: 10.2217/pgs.09.133.  [DOI](https://doi.org/10.2217/pgs.09.133) | [PMC free article](/articles/PMC6556886/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20017671/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=CYP2D7%E2%80%932D6%20hybrid%20tandems:%20identification%20of%20novel%20CYP2D6%20duplication%20arrangements%20and%20implications%20for%20phenotype%20prediction&author=A.%20Gaedigk&author=U.%20Fuhr&author=C.%20Johnson&author=L.A.%20Berard&author=D.%20Bradford&volume=11&publication_year=2010&pages=43-53&pmid=20017671&doi=10.2217/pgs.09.133&)

40. Gaedigk A., Twist G.P., Leeder J.S. CYP2D6, SULT1A1 and UGT2B17 copy number variation: quantitative detection by multiplex PCR. Pharmacogenomics. 2012;13:91–111. doi: 10.2217/pgs.11.135.  [DOI](https://doi.org/10.2217/pgs.11.135) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22111604/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=CYP2D6,%20SULT1A1%20and%20UGT2B17%20copy%20number%20variation:%20quantitative%20detection%20by%20multiplex%20PCR&author=A.%20Gaedigk&author=G.P.%20Twist&author=J.S.%20Leeder&volume=13&publication_year=2012&pages=91-111&pmid=22111604&doi=10.2217/pgs.11.135&)

41. Wang D., Papp A.C., Binkley P.F., Johnson J.A., Sadee W. Highly variable mRNA expression and splicing of L-type voltage-dependent calcium channel alpha subunit 1C in human heart tissues. Pharmacogenet. Genomics. 2006;16:735–745. doi: 10.1097/01.fpc.0000230119.34205.8a.  [DOI](https://doi.org/10.1097/01.fpc.0000230119.34205.8a) | [PMC free article](/articles/PMC2688811/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17001293/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet.%20Genomics&title=Highly%20variable%20mRNA%20expression%20and%20splicing%20of%20L-type%20voltage-dependent%20calcium%20channel%20alpha%20subunit%201C%20in%20human%20heart%20tissues&author=D.%20Wang&author=A.C.%20Papp&author=P.F.%20Binkley&author=J.A.%20Johnson&author=W.%20Sadee&volume=16&publication_year=2006&pages=735-745&pmid=17001293&doi=10.1097/01.fpc.0000230119.34205.8a&)

42. Ramamoorthy A., Flockhart D.A., Hosono N., Kubo M., Nakamura Y., Skaar T.C. Differential quantification of CYP2D6 gene copy number by four different quantitative real-time PCR assays. Pharmacogenet. Genomics. 2010;20:451–454. doi: 10.1097/FPC.0b013e32833a1083.  [DOI](https://doi.org/10.1097/FPC.0b013e32833a1083) | [PMC free article](/articles/PMC4411953/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20421845/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet.%20Genomics&title=Differential%20quantification%20of%20CYP2D6%20gene%20copy%20number%20by%20four%20different%20quantitative%20real-time%20PCR%20assays&author=A.%20Ramamoorthy&author=D.A.%20Flockhart&author=N.%20Hosono&author=M.%20Kubo&author=Y.%20Nakamura&volume=20&publication_year=2010&pages=451-454&pmid=20421845&doi=10.1097/FPC.0b013e32833a1083&)

43. Hagege H., Klous P., Braem C., Splinter E., Dekker J., Cathala G., de Laat W., Forne T. Quantitative analysis of chromosome conformation capture assays (3C-qPCR) Nat. Protoc. 2007;2:1722–1733. doi: 10.1038/nprot.2007.243.  [DOI](https://doi.org/10.1038/nprot.2007.243) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17641637/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Protoc.&title=Quantitative%20analysis%20of%20chromosome%20conformation%20capture%20assays%20(3C-qPCR)&author=H.%20Hagege&author=P.%20Klous&author=C.%20Braem&author=E.%20Splinter&author=J.%20Dekker&volume=2&publication_year=2007&pages=1722-1733&pmid=17641637&doi=10.1038/nprot.2007.243&)
