# Rheumatoid Arthritis Risk Allele PTPRC Is Also Associated With Response to Anti–Tumor Necrosis Factor α Therapy

## Metadata
**Authors:** Jing Cui, Saedis Saevarsdottir, Brian Thomson, Leonid Padyukov, Annette H M van der Helm-van Mil, Joanne Nititham, Laura B Hughes, Niek de Vries, Soumya Raychaudhuri, Lars Alfredsson, Johan Askling, Sara Wedrén, Bo Ding, Candace Guiducci, Gert Jan Wolbink, J Bart A Crusius, Irene E van der Horst-Bruinsma, Marieke Herenius, Michael E Weinblatt, Nancy A Shadick, Jane Worthington, Franak Batliwalla, Marlena Kern, Ann W Morgan, Anthony G Wilson, John D Isaacs, Kimme Hyrich, the Biologics in Rheumatoid Arthritis Genetics and Genomics Study Syndicate, Michael F Seldin, Larry W Moreland, Timothy W Behrens, Cornelia F Allaart, Lindsey A Criswell, Tom W J Huizinga, Paul P Tak, S Louis Bridges, Jr, Rene E M Toes, Anne Barton, Lars Klareskog, Peter K Gregersen, Elizabeth W Karlson, Robert M Plenge
**Journal:** Arthritis and rheumatism
**Date:** 2010 Jul
**DOI:** [10.1002/art.27457](https://doi.org/10.1002/art.27457)
**PMID:** 20309874
**PMCID:** PMC3652476
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3652476/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC3652476/pdf/nihms453040.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC3652476/pdf/nihms453040.pdf)

## Abstract

**Objective:** 
Anti–tumor necrosis factor α (anti-TNF) therapy is a mainstay of treatment in rheumatoid arthritis (RA). The aim of the present study was to test established RA genetic risk factors to determine whether the same alleles also influence the response to anti-TNF therapy.

**Methods:** 
A total of 1,283 RA patients receiving etanercept, infliximab, or adalimumab therapy were studied from among an international collaborative consortium of 9 different RA cohorts. The primary end point compared RA patients with a good treatment response according to the European League Against Rheumatism (EULAR) response criteria (n = 505) with RA patients considered to be nonresponders (n = 316). The secondary end point was the change from baseline in the level of disease activity according to the Disease Activity Score in 28 joints (ΔDAS28). Clinical factors such as age, sex, and concomitant medications were tested as possible correlates of treatment response. Thirty-one single-nucleotide polymorphisms (SNPs) associated with the risk of RA were genotyped and tested for any association with treatment response, using univariate and multivariate logistic regression models.

**Results:** 
Of the 31 RA-associated risk alleles, a SNP at the PTPRC (also known as CD45) gene locus (rs10919563) was associated with the primary end point, a EULAR good response versus no response (odds ratio [OR] 0.55, P = 0.0001 in the multivariate model). Similar results were obtained using the secondary end point, the ΔDAS28 (P = 0.0002). There was suggestive evidence of a stronger association in autoantibody-positive patients with RA (OR 0.55, 95% confidence interval [95% CI] 0.39–0.76) as compared with autoantibody-negative patients (OR 0.90, 95% CI 0.41–1.99).

**Conclusion:** 
Statistically significant associations were observed between the response to anti-TNF therapy and an RA risk allele at the PTPRC gene locus. Additional studies will be required to replicate this finding in additional patient collections.

### Objective

Anti–tumor necrosis factor α (anti-TNF) therapy is a mainstay of treatment in rheumatoid arthritis (RA). The aim of the present study was to test established RA genetic risk factors to determine whether the same alleles also influence the response to anti-TNF therapy.

### Methods

A total of 1,283 RA patients receiving etanercept, infliximab, or adalimumab therapy were studied from among an international collaborative consortium of 9 different RA cohorts. The primary end point compared RA patients with a good treatment response according to the European League Against Rheumatism (EULAR) response criteria (n = 505) with RA patients considered to be nonresponders (n = 316). The secondary end point was the change from baseline in the level of disease activity according to the Disease Activity Score in 28 joints (ΔDAS28). Clinical factors such as age, sex, and concomitant medications were tested as possible correlates of treatment response. Thirty-one single-nucleotide polymorphisms (SNPs) associated with the risk of RA were genotyped and tested for any association with treatment response, using univariate and multivariate logistic regression models.

### Results

Of the 31 RA-associated risk alleles, a SNP at the *PTPRC* (also known as *CD45*) gene locus (rs10919563) was associated with the primary end point, a EULAR good response versus no response (odds ratio [OR] 0.55, *P* = 0.0001 in the multivariate model). Similar results were obtained using the secondary end point, the ΔDAS28 (*P* = 0.0002). There was suggestive evidence of a stronger association in autoantibody-positive patients with RA (OR 0.55, 95% confidence interval [95% CI] 0.39–0.76) as compared with autoantibody-negative patients (OR 0.90, 95% CI 0.41–1.99).

### Conclusion

Statistically significant associations were observed between the response to anti-TNF therapy and an RA risk allele at the *PTPRC* gene locus. Additional studies will be required to replicate this finding in additional patient collections.

## PATIENTS AND METHODS

### Patients

RA patients were selected from 9 different cohorts (as described below). The clinical features of the patients are listed in [Table 1](#T1). The diagnosis in all patients was defined by satisfaction of the American College of Rheumatology (formerly, the American Rheumatism Association) 1987 criteria for RA ([22](#R22)) or by confirmation from a board-certified rheumatologist. Inclusion criteria for our study were the presence of active disease (defined as a Disease Activity Score in 28 joints [DAS28] ≥3.2) prior to initiation of anti-TNF therapy (baseline) and available data on the DAS28 within 3–12 months after the start of the anti-TNF therapy (followup), as well as current treatment with the anti-TNF drug at the followup time point. From each cohort, we collected information on age, sex, disease duration, serotype status (anti–citrullinated protein antibody [ACPA] and/or rheumatoid factor [RF] positive), anti-TNF treatment duration (from start date to followup date), components of the DAS28 at anti-TNF start and followup, and other medications, including disease-modifying antirheumatic drugs (DMARDs) for RA. We defined seropositive (or autoantibody-positive) patients as those who were RF and/or ACPA positive, and seronegative patients as those who were negative for both (or negative for a single autoantibody, if only one was checked). We restricted our analysis to subjects with a self-reported white European ancestry, if that information was available from the cohort. Informed consent was obtained from each individual, and the institutional review board at each collection site approved the study protocol.

### Table 1.

|   | ABCoN | AMC | BeSt | BRAGGSS | BRASS† | EIRA‡ | ERA | KI | JBI |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Study design | Prospective | Prospective | RCT | Observational | Observational | Observational | RCT | Observational | Observational |
| Sample size |   |   |   |   |   |   |   |   |   |
| Total | 116 | 157 | 126 | 81 | 55 | 291 | 218 | 163 | 76 |
| EULAR good response‡ | 47 | 49 | 88 | 42 | 27 | 112 | 68 | 44 | 28 |
| EULAR moderate response‡ | 39 | 77 | ND | 7 | 11 | 104 | 102 | 89 | 33 |
| EULAR no response‡ | 30 | 31 | 38 | 32 | 17 | 75 | 48 | 30 | 15 |
| Anti-TNF drug | All | ada., inflix. | inflix. | inflix. | All | All | etan.§ | All | etan. |
| Disease activity measure | DAS28-CRP | DAS28-ESR | DAS28-ESR | DAS28-ESR | DAS28-CRP | DAS28-ESR | DAS28-CRP | DAS28-ESR | DAS28-ESR |
| Genotype method | SiP | SiP | SiP | SiP | Imputed | Imputed | SiP | SiP | SiP |
| Treatment duration, mean (range) months¶ | 3.4 (2.4–6.1) | 3.7 (1.8–4.2) | (6–12) | 6 | (3–12) | 3.5 (2.1–5) | 12 (10–13.5) | 3.7 (2.2–5) | 3.7 |
| Age, mean ± SD years | 54.6 ± 13.4 | 54.1 ± 12.8 | 50.7 ± 13.5 | 55.2 ± 10.8 | 56.6 ± 13.0 | 51.3 ± 12.5 | 50.6 ± 12.5 | 55.1 ± 13.5 | 52.9 ± 10.4 |
| Sex, % female | 79.3 | 77.7 | 65.1 | 72.8 | 85.5 | 73.9 | 72.5 | 84.7 | 81.6 |
| Seropositive status, % ACPA/RF positive | 89.8 | 79.9 | 81.3 | 92.3 | 100 | 81.8 | 87.2 | 85.9 | 71.1 |
| Concurrent methotrexate, % | 69.1 | 96.8 | 97.6 | 91.4 | 25.5 | 74.2 | 0 | 100 | 81.6 |
| Disease duration, mean ± SD years | 10.1 ± 9.7 | 10.6 ± 13.4 | 1.4 ± 1.2 | 11.6 ± 9.7 | 9.9 ± 9.8 | 3.3 ± 2.6 | 1.0 ± 0.9 | 13.1 ± 10.7 | 11.4 ± 8.9 |

Table 1 Caption: Characteristics of the 9 clinical cohorts of patients with rheumatoid arthritis receiving anti-TNF therapy*

### Cohorts

#### Autoimmune Biomarkers Collaborative Network (ABCoN)

The ABCoN study is a prospective clinical trial of 116 RA patients who were started on anti-TNF therapy (n = 51 receiving etanercept, n = 22 receiving adalimumab, and n = 43 receiving infliximab). Clinical data were obtained from evaluations at 5 time points: baseline (before therapy) as well as 6 weeks, 12 weeks, 6 months, and 1 year after the start of anti-TNF therapy ([23](#R23)).

#### Academic Medical Center (AMC) cohort

The AMC study enrolled and prospectively followed up RA patients who received anti-TNF therapy at the Department of Clinical Immunology and Rheumatology at the AMC of the University of Amsterdam. Of these patients, 55 received adalimumab ([24](#R24)) and 102 received infliximab ([8](#R8)). Clinical data obtained included the DAS28 at baseline and after 4, 8, 12, 16, and 24 weeks of treatment. All patients had a DAS28 of ≥3.2 at baseline, and the medication was kept stable during the study.

#### Treatment Strategies for Rheumatoid Arthritis (Behandel-strategieën voor Reumatoide Artritis [BeSt]) study

The BeSt study is a multicenter, randomized clinical trial of patients with new-onset RA ([25](#R25)). Patients had a high DAS28 at baseline and were assessed every 3 months. Medications were adjusted by a physician based on the DAS28. All patients had a disease duration of <2 years. All patients were being treated with infliximab at the time that the posttreatment response was measured. We included only a subset of 126 patients whose treatment response, according to the European League Against Rheumatism (EULAR) response criteria, was either a good response or no response.

#### Biologics in Rheumatoid Arthritis Genetics and Genomics Study Syndicate (BRAGGSS)

BRAGGSS is a prospective multicenter registry of RA patients who receive anti-TNF therapy in the United Kingdom. The treating physician assesses the treatment response at 6-month intervals. More than 1,000 patients have been enrolled to date, and the data have been used in published genetic studies ([26](#R26),[27](#R27)), but only a subset of 81 patients receiving infliximab and displaying either a EULAR good response or no response were included in the current study. There was no difference in the age, sex, or treatment history among the BRAGGSS patients selected for the current study and the remaining infliximab-treated BRAGGSS patients classified as EULAR good responders or EULAR nonresponders.

#### Brigham Rheumatoid Arthritis Sequential Study (BRASS)

The BRASS is a prospective observational registry of >1,000 RA patients receiving care at the Brigham and Women’s Hospital in Boston ([28](#R28)). Patients enrolled in this study include a subset (n = 6) from whom data on the DAS28 (with C-reactive protein [CRP]) were collected at baseline and 12 weeks after the start of anti-TNF therapy, and a subset (n = 49) who were followed up as part of routine care. For the latter, posttreatment disease activity was assessed in 3–12-month intervals following the initiation of anti-TNF therapy. In total, 24 patients were receiving etanercept, 26 were receiving adalimumab, and 5 were receiving infliximab.

#### Epidemiological Investigation of Rheumatoid Arthritis (EIRA)

The EIRA is a population-based study of incident RA that was initiated in 1996. Clinical followup data are registered in the Swedish Rheumatology and Biologics Register by the treating rheumatologist at each visit as part of a national surveillance system ([29](#R29)). Patients enrolled in the present study are a subset of those included in the EIRA study, comprising those patients who started anti-TNF therapy as the first biologic treatment during the followup period, with the DAS28 determined at baseline and at the 3-month followup visit (n = 100 receiving etanercept, n = 144 receiving infliximab, and n = 47 receiving adalimumab).

#### Immunex Early Rheumatoid Arthritis (ERA) study

The ERA study was a randomized clinical trial of 632 patients with early RA treated for 12 months with either etanercept (10 mg or 25 mg subcutaneously twice per week) or MTX (orally once per week) ([30](#R30)). More than 80% of patients continued receiving etanercept for 12 months, at which time disease activity and treatment response were assessed. At baseline, the 632 RA patients shared similar levels of disease activity and had RA for no more than 3 years. A subset of the patients consented to undergo genetic studies of treatment response (n = 457). Of these, 218 completed 12 months of etanercept treatment (n = 106 receiving 10 mg and n = 112 receiving 25 mg) and had a sufficient quantity of DNA, after whole-genome amplification, for direct genotyping. These samples have been used in a candidate gene study of treatment response ([31](#R31)).

#### Karolinska Institutet (KI) study

The KI study cohort consisted of RA patients from the outpatient clinics within the Department of Rheumatology at Karolinska University Hospital in Stockholm ([32](#R32)). The inclusion criteria included having received anti-TNF therapy as the first biologic treatment during 1999–2007, having provided a DNA sample to the Rheumatology Biobank, and having clinical followup data available as part of the Swedish Rheumatology and Biologics Register (as described above for the EIRA), including the DAS28 at baseline and at the 3-month followup visit. Patients were excluded if they were already enrolled in the EIRA. From a total of 632 RA patients in this group, 486 had clinical followup data available, and of these, 163 provided a DNA sample to the Rheumatology Biobank (n = 124 receiving infliximab, n = 31 receiving etanercept, and n = 8 receiving adalimumab). The baseline characteristics of those patients with an available DNA sample were similar to those who had not provided DNA.

#### Jan van Breemen Institute (JBI) study

Since 2005, all Dutch patients with RA starting treatment with etanercept have been enrolled in a cohort study at the JBI in Amsterdam. RA patients are enrolled in the study if they are eligible for anti-TNF therapy, in accordance with the Dutch consensus statement on TNF-blocking therapy. Eligibility criteria include the presence of active disease (DAS28 ≥3.2) and having failed treatment with at least 2 DMARDs, including MTX at the maximal or tolerable dosage. Exclusion criteria are active infection and pregnancy. Enrollment in the JBI cohort is still open, and the current genetic study includes the first 76 consecutive RA patients for whom genetic and clinical data were available. Disease activity (according to the DAS28) was assessed at baseline and after 4, 16, and 28 weeks of therapy. All patients were treated with etanercept at a dose of 50 mg subcutaneously every week or 25 mg twice a week.

### Definition of treatment response

The DAS28 was calculated directly from individual patient data, on the basis of the number of swollen and tender joints, the level of acute-phase response (using either the erythrocyte sedimentation rate or level of CRP), and patient’s general health assessment ([33](#R33)). [Table 1](#T1) lists the specific DAS28 versions used in each of the cohorts. Our primary analysis was to categorize patients according to the EULAR response criteria ([34](#R34)), in which a good response was defined as a followup DAS28 of <3.2 and improvement in the DAS28 of >1.2 from baseline, while nonresponse was defined as improvement in the DAS28 of <0.6 from baseline or improvement of ≤1.2 and a followup DAS28 of >5.1; moderate response was defined as those DAS28 values in between. In a secondary analysis, we used the change in DAS28 (ΔDAS28) from baseline to the time that treatment response was assessed (between 3 months and 12 months posttreatment).

### Clinical factors

We tested clinical factors for any association with treatment response (according to the EULAR response criteria), using logistic regression. Clinical factors included age, sex, disease duration, treatment duration, RF and ACPA status, concomitant medications, and the DAS28 prior to treatment. We also tested for a cohort effect in the logistic model by creating dummy variables for each cohort.

### Genotyping

We selected 31 validated or highly suggestive MHC and non-MHC RA risk alleles from recent large-scale genetic studies ([12](#R12)–[19](#R19),[35](#R35)–[42](#R42)). Tag SNPs for the MHC were selected from a high-density SNP genotyping study across the MHC ([42](#R42),[43](#R43)) and from the best independent tag SNPs in an RA genome-wide association study ([14](#R14)). Only 5 of the 9 cohorts had 4-digit *HLA–DRB1* genotype data (the ABCoN, BeSt, BRAGGSS, EIRA, and ERA studies). To assess population stratification, we genotyped 3 SNPs with highly differentiated allele frequencies across individuals of European ancestry ([44](#R44)) (for a complete list of the SNPs genotyped, see [Supplementary Table 1](#SD1), available on the *Arthritis & Rheumatism* Web site at [http://www3.interscience.wiley.com/journal/76509746/home](http://www3.interscience.wiley.com/journal/76509746/home)). As shown in [Table 1](#T1), genotype data were imputed from Affymetrix 6.0 (900K) data (for those patients in the BRASS cohort) or from Illumina 317K data (for those patients in the EIRA cohort), as previously described ([12](#R12)). Although IMPUTE provides probability scores ([45](#R45)), we used integer allele counts, since there is little difference between probability scores and counts (results not shown). For the remaining cohorts, we genotyped 31 SNPs (as well as proxy SNPs and European ancestry informative markers [AIMs] [[44](#R44)]) using Sequenom iPlex as previously described ([12](#R12)), which was performed at the Broad Institute. Within each cohort, we removed individuals with >10% missing genotypes (based on all available genotype data) and we removed SNPs that had >5% missing genotype data, that had a minor allele frequency of <1%, and that, on testing for Hardy-Weinberg equilibrium, were found to be significantly different (*P* < 0.001).

### Statistical analysis

In our primary analysis, we tested each SNP for an association with the anti-TNF response (EULAR good response versus no response) using logistic regression, assuming a log additive model. We controlled for age, sex, concurrent treatment with MTX, and cohort effect, as well as for the DAS28 at start of anti-TNF therapy, in a multivariate model that included SNPs potentially predictive of the anti-TNF response. In our secondary analysis, we modeled SNPs potentially predictive of the ΔDAS28, using univariate and multivariate linear regression models in which we adjusted for age, sex, concurrent treatment with MTX, cohort effect, and the DAS28 at start of anti-TNF therapy. We made no assumptions about the direction of effect on treatment response with respect to the risk of RA. We considered a Bonferroni-corrected *P* value of less than 0.05 as statistically significant, which, in our study of 31 SNPs, corresponded to *P* < 0.0016.

For each SNP, we tested for heterogeneity across the 9 cohorts using H statistics. We performed stratified analyses according to clinical characteristics to explore sources of heterogeneity. To compare the association among subgroups of RA patients for our primary outcome, we used a logistic regression model that included genotype, clinical category, and genotype × clinical category; the reported *P* value indicates the significance of the interaction term, which compares the odds ratio (OR) and 95% confidence interval (95% CI) for each clinical category. For our secondary outcome, the ΔDAS28, we used a general linear model that included genotype, clinical category, and genotype × clinical category, as well as the DAS28 at the start of anti-TNF therapy; the reported *P* value again indicates the significance of the interaction term, which, in this analysis, compares the beta values within each clinical category.

## RESULTS

### Patient characteristics

The clinical characteristics of the 1,283 RA patients with active disease from our 9 cohorts are shown in [Table 1](#T1). Patients were started on infliximab (n = 625), etanercept (n = 502), or adalimumab (n = 156), and most were treated concurrently with MTX (n = 955 [74.4%]). All patients were receiving anti-TNF therapy at the time that treatment response was assessed. Three of the cohorts comprised patients with early-onset RA (BeSt, EIRA, and ERA), while 2 of the studies were randomized controlled trials (BeSt and ERA). The percentage of patients who were seropositive for either RF or ACPAs was similar among the 9 cohorts (71.1–100%).

To build a multivariate clinical model for our genetic association study, we first tested for associations between each clinical variable and the EULAR response classification (good response, moderate response, or no response). As shown in [Table 2](#T2), younger age (*P* = 0.006) and male sex (*P* < 0.0001) were significantly correlated with better outcome. Concurrent treatment with MTX demonstrated a trend toward significance, with 73% of the patients treated concurrently with MTX classified as having a EULAR good response, as compared with 66% and 67% classified as EULAR moderate responders and EULAR nonresponders, respectively (each *P* = 0.07). The pretreatment (baseline) DAS28 was significantly correlated with treatment response, but the trend was not linear across the 3 EULAR categories (e.g., those in the moderate response category had the highest baseline DAS28, at a mean of 5.9). Based on these results, we included age, sex, concurrent treatment with MTX, baseline DAS28, and a cohort variable into the multivariate model.

### Table 2.

|   | Good response(n = 505) | Moderate response(n = 462) | No response(n = 316) | P |
| --- | --- | --- | --- | --- |
| Sex, % female | 71 | 77 | 84 | <0.0001 |
| Age, mean ± SD years | 51.5 ± 12.8 | 54.0 ± 12.3 | 53.3 ± 13.2 | 0.006 |
| Disease duration, mean ± SD years | 6.5 ± 9.9 | 6.9 ± 8.4 | 6.6 ± 8.8 | 0.71 |
| RF positive, % | 79 | 81 | 76 | 0.19 |
| ACPA positive, % | 77 | 77 | 74 | 0.77 |
| Concurrent methotrexate, % | 73 | 66 | 67 | 0.07 |
| Concurrent steroid, % | 33 | 39 | 35 | 0.21 |
| Concurrent NSAIDs, % | 57 | 60 | 55 | 0.45 |
| Treatment duration, mean ± SD months | 5.2 ± 3.4 | 5.6 ± 3.6 | 5.3 ± 3.5 | 0.29 |
| DAS28, mean ± SD |   |   |   |   |
| Baseline | 5.4 ± 1.0 | 5.9 ± 1.1 | 5.3 ± 1.3 | <0.0001 |
| Followup | 2.3 ± 0.6 | 4.1 ± 0.8 | 5.2 ± 1.2 | <0.0001 |

Table 2 Caption: Correlations of clinical characteristics with treatment response according to the 3 EULAR response categories among 1,283 patients with rheumatoid arthritis treated with anti–tumor necrosis factor α therapy*

### Genetic associations

We tested 31 RA risk alleles for associations with the response to anti-TNF therapy, using univariate and multivariate logistic regression analyses. The primary outcome was a EULAR good response versus no response (n = 505 versus n = 316, respectively). We chose this dichotomous outcome measure to minimize heterogeneity across the 9 cohorts, since the DAS28 is more accurate for patients with either high or low disease activity ([46](#R46)).

As shown in [Table 3](#T3), only a single SNP (rs10919563) showed a significant association (*P* < 0.01) with a EULAR good response to anti-TNF therapy. This SNP is in the *PTPRC* gene (also known as *CD45*), and was significantly associated with treatment response in both univariate and multivariate models (*P* = 0.0004 and *P* = 0.0001, respectively). The major allele (G allele), which is a known predictor of RA risk, is the same allele that was found to be a predictor of favorable response; in the presence of the major allele, the OR for the likelihood of a EULAR good response was 0.59 and 0.55 in univariate and multivariate models, respectively. When the data were corrected for multiple hypothesis testing, taking into account the number of SNPs tested (calculated as the *P* value divided by the number of SNPs, or 0.05/31), the association remained significant (*P* = 0.0016) (for more detailed data on all 31 SNPs tested in each of the 9 cohorts, see [Supplementary Table 2](#SD1), available on the *Arthritis & Rheumatism* Web site at [http://www3.interscience.wiley.com/journal/76509746/home](http://www3.interscience.wiley.com/journal/76509746/home)).

### Table 3.

|   |   |   |   |   | Univariate |  | Multivariate† |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Chr | SNP | Gene | POS (bp) | MAF | OR | P | OR | P |
| 1 | rs10919563 | PTPRC | 196,967,065 | 0.12 | 0.59 | 0.0004 | 0.55 | 0.0001 |
| 1 | rs11586238 | CD58 | 117,064,661 | 0.25 | 1.31 | 0.04 | 1.32 | 0.04 |
| 3 | rs4535211 | PLCL2 | 17,048,001 | 0.44 | 0.83 | 0.07 | 0.84 | 0.11 |
| 7 | rs11761231 | PODXL | 131,020,579 | 0.36 | 0.84 | 0.14 | 0.86 | 0.23 |
| 6 | rs1341239 | PRL | 22,412,183 | 0.37 | 1.17 | 0.16 | 1.21 | 0.09 |
| 11 | rs540386 | TRAF6 | 36,481,869 | 0.13 | 0.79 | 0.16 | 0.81 | 0.23 |
| 10 | rs2104286 | IL2RA | 6,139,051 | 0.24 | 0.85 | 0.21 | 0.84 | 0.18 |
| 6 | rs6920220 | TNFAIP3 | 138,048,197 | 0.23 | 0.87 | 0.30 | 0.87 | 0.32 |
| 6 | rs4895501 | TNFAIP3 | 138,329,253 | 0.39 | 1.11 | 0.32 | 1.11 | 0.33 |
| 1 | rs2476601 | PTPN22 | 114,179,091 | 0.16 | 1.14 | 0.35 | 1.15 | 0.32 |
| 1 | rs3890745 | TNFRSF14 | 2,543,484 | 0.31 | 1.11 | 0.35 | 1.13 | 0.30 |
| 6 | rs548234 | PRDM1 | 106,674,727 | 0.33 | 0.91 | 0.37 | 0.89 | 0.30 |
| 9 | rs2812378 | CCL21 | 34,700,260 | 0.38 | 1.09 | 0.40 | 1.06 | 0.57 |
| 2 | rs13031237 | REL | 60,989,633 | 0.37 | 0.92 | 0.43 | 0.94 | 0.56 |
| 20 | rs4810485 | CD40 | 44,181,354 | 0.21 | 0.91 | 0.46 | 0.94 | 0.66 |
| 10 | rs4750316 | PRKCQ | 6,433,266 | 0.18 | 0.91 | 0.50 | 0.92 | 0.54 |
| 2 | rs3087243 | CTLA4 | 204,447,164 | 0.39 | 1.07 | 0.52 | 1.13 | 0.26 |
| 6 | rs4947332 | HLA*0101 | 31,942,176 | 0.04 | 1.16 | 0.62 | 1.06 | 0.85 |
| 4 | rs231707 | TNIP2 | 2,664,183 | 0.19 | 0.93 | 0.64 | 0.89 | 0.46 |
| 9 | rs3761847 | TRAF1-C5 | 122,730,060 | 0.45 | 1.04 | 0.67 | 1.05 | 0.66 |
| 6 | rs6457617 | DR4 | 32,771,829 | 0.33 | 0.96 | 0.72 | 0.94 | 0.57 |
| 2 | rs1980421 | CD28 | 204,318,249 | 0.23 | 1.04 | 0.74 | 1.04 | 0.74 |
| 4 | rs6822844 | IL2-IL21 | 123,728,871 | 0.14 | 1.05 | 0.75 | 1.06 | 0.70 |
| 8 | rs13277113 | BLK | 11,386,595 | 0.29 | 1.04 | 0.76 | 0.99 | 0.95 |
| 6 | rs3817964 | HLA*0401 | 32,475,975 | 0.08 | 0.95 | 0.77 | 0.98 | 0.91 |
| 22 | rs3218253 | IL2RB | 35,874,756 | 0.27 | 1.04 | 0.78 | 1.09 | 0.51 |
| 6 | rs394581 | TAGAP | 159,402,509 | 0.26 | 1.03 | 0.79 | 1.04 | 0.71 |
| 6 | rs2621377 | HLA*DPB1 | 32,871,088 | 0.39 | 1.02 | 0.83 | 1.04 | 0.73 |
| 6 | rs13207033 | TNFAIP3 | 138,007,111 | 0.23 | 1.01 | 0.92 | 0.99 | 0.96 |
| 7 | rs42041 | CDK6 | 92,084,680 | 0.26 | 1.00 | 0.98 | 0.97 | 0.83 |
| 2 | rs11889341 | STAT4 | 191,651,987 | 0.23 | 1.00 | 1.00 | 1.03 | 0.85 |

Table 3 Caption: Associations of 31 SNPs with EULAR good response versus no response in univariate and multivariate models*

To determine whether the *PTPRC* SNP or any other SNP was associated with the secondary outcome measure, the ΔDAS28, as a continuous variable, we tested each SNP using univariate and multivariate linear regression models. This analysis included an additional 462 RA patients classified as having a EULAR moderate response, for a total of 1,283 RA patients who were started on anti-TNF therapy and evaluated for treatment response. Consistent with a true-positive result, the *PTPRC* SNP remained significantly associated with a favorable anti-TNF response (*P* = 0.0005 and *P* = 0.0002 in univariate and multivariate models, respectively). No other SNP was found to be significantly associated at *P* < 0.01, in either univariate or multivariate analyses (for detailed results on the association of each of the 31 SNPs with the ΔDAS28 in the 9 cohorts, see [Supplementary Tables 3 and 4](#SD1), available on the *Arthritis & Rheumatism* Web site at [http://www3.interscience.wiley.com/journal/76509746/home](http://www3.interscience.wiley.com/journal/76509746/home)).

We next examined whether the *PTPRC* SNP was correlated with treatment response across all 9 cohorts, or whether the correlation was specific to a single cohort (or subset of cohorts). As shown in [Figure 1](#F1), the OR for a favorable treatment response (EULAR good response versus no response) was relatively consistent across all cohorts. The KI cohort had the smallest point estimate of response (OR 0.12, 95% CI 0.04–0.42), whereas the point estimate for the AMC cohort was the highest (OR 2.88, 95% CI 0.78–10.7).

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/612d/3652476/66180f8aeb35/nihms453040f1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3652476_nihms453040f1.jpg)

Primary end point analysis. Shown are genotype counts and allele frequencies (freq.) in patients with rheumatoid arthritis (RA) according to European League Against Rheumatism (EULAR) treatment response (good response versus no response) to anti–tumor necrosis factor α (anti-TNF) therapy in each of the 9 different cohorts (left), and a forest plot of the results of association analyses of PTPRC in relation to a favorable EULAR response (right). The major allele (G allele) of PTPRC is the RA risk allele, and as the forest plot shows for each cohort and for the combined analysis (n = 821 patients), the same G allele is associated with a favorable response to anti-TNF therapy. The odds ratio (OR) point estimates and 95% confidence intervals (95% CIs) are shown for the major G allele relative to the minor A allele. For purposes of scale, the upper bound of the 95% CI for the Academic Medical Center (AMC) cohort is not shown (95% CI 0.78–10.70). ABCoN = Autoimmune Biomarkers Collaborative Network; BeSt = Treatment Strategies for Rheumatoid Arthritis (Behandelstrategieen voor Reumatoide Artritis); BRAGGSS = Biologics in Rheumatoid Arthritis Genetics and Genomics Study Syndicate; BRASS = Brigham Rheumatoid Arthritis Sequential Study; EIRA = Epidemiological Investigation of Rheumatoid Arthritis; eRA = Immunex Early Rheumatoid Arthritis; KI = Karolinska Institutet; JBI = Jan van Breemen Institute.

A formal test for heterogeneity among cohorts showed a nonsignificant trend across all cohorts (*P* = 0.06). After removal of the AMC cohort, heterogeneity was reduced (*P* = 0.19) and the association with *PTPRC* became more significant (*P* = 0.00003). After removal of the KI cohort, both heterogeneity and the *PTPRC* association became less significant (*P* = 0.21 and *P* = 0.0097, respectively). When we tested for heterogeneity in our secondary analysis of an association with the ΔDAS28 as a continuous trait, we observed modest evidence of differences across the 9 cohorts (*P* = 0.006) ([Figure 2](#F2)). (Results of tests for heterogeneity of associations with all 31 SNPs are listed in [Supplementary Table 5](#SD1), available on the *Arthritis & Rheumatism* Web site at [http://www3.interscience.wiley.com/journal/76509746/home](http://www3.interscience.wiley.com/journal/76509746/home)).

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/612d/3652476/0689232facff/nihms453040f2.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3652476_nihms453040f2.jpg)

Secondary end point analysis. The forest plot shows the results of the association analyses of PTPRC (major G allele and minor A allele) in relation to change in the Disease Activity Score in 28 joints from baseline to followup, as a quantitative trait, in patients with rheumatoid arthritis treated with anti-TNF therapy. The beta coefficient point estimates and 95% confidence intervals are shown for each cohort as well as for the combined analysis (n = 1,283 patients). See Figure 1 for definitions.

To determine whether the *PTPRC* SNP was associated with treatment response in a specific group of RA patients or in relation to a specific anti-TNF drug, we stratified the primary and secondary analyses by potentially important clinical categories, as follows: the RF and ACPA autoantibody status, the anti-TNF drug class, study design, sex, concurrent treatment with MTX, and disease duration ([Table 4](#T4)). The association was stronger among 1,037 seropositive patients with RA (defined as being either RF+ or ACPA+) than among 195 seronegative patients with RA. However, the ORs for an association of the *PTPRC* SNP with favorable treatment response were not statistically significantly different (*P* = 0.26) between seropositive and seronegative patients with RA (seropositive, OR 0.55, 95% CI 0.39–0.76; seronegative, OR 0.90, 95% CI 0.41–1.99). Similarly, a suggestive trend toward an association with serologic status was observed when the ΔDAS28 was used as the end point ([Table 4](#T4)).

### Table 4.

|   |   | EULAR categories |  |  |  |  | ΔDAS28 |  |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Clinical category | No. ofpatients | OR | LowerCI | UpperCI | P forassociation | P betweensubgroups† | Beta | LowerCI | UpperCI | P forassociation | P betweensubgroups† |
| ACPA/RF autoantibody status |   |   |   |   |   | 0.26 |   |   |   |   | 0.14 |
| Seropositive | 1,037 | 0.55 | 0.39 | 0.76 | 0.0003 |   | 0.30 | 0.14 | 0.46 | 0.0002 |   |
| Seronegative | 195 | 0.90 | 0.41 | 1.99 | 0.79 |   | −0.03 | −0.48 | 0.42 | 0.89 |   |
| Anti-TNF drug |   |   |   |   |   | 0.66 |   |   |   |   | 0.61 |
| Infliximab | 625 | 0.56 | 0.37 | 0.84 | 0.005 |   | 0.35 | 0.11 | 0.59 | 0.005 |   |
| Etanercept | 502 | 0.55 | 0.34 | 0.91 | 0.02 |   | 0.22 | 0.002 | 0.43 | 0.05 |   |
| Adalimumab | 156 | 0.86 | 0.36 | 2.09 | 0.74 |   | 0.16 | −0.24 | 0.55 | 0.44 |   |
| Study design |   |   |   |   |   | 0.21 |   |   |   |   | 0.01 |
| Prospective | 273 | 0.86 | 0.44 | 1.68 | 0.65 |   | −0.04 | −0.33 | 0.24 | 0.77 |   |
| RCT | 344 | 0.67 | 0.41 | 1.10 | 0.11 |   | 0.17 | −0.10 | 0.43 | 0.22 |   |
| Observational | 666 | 0.44 | 0.28 | 0.68 | 0.0003 |   | 0.51 | 0.28 | 0.75 | <0.0001 |   |
| Sex |   |   |   |   |   | 0.77 |   |   |   |   | 0.89 |
| Female | 975 | 0.57 | 0.41 | 0.81 | 0.001 |   | 0.26 | 0.08 | 0.43 | 0.004 |   |
| Male | 308 | 0.63 | 0.34 | 1.18 | 0.15 |   | 0.29 | −0.01 | 0.59 | 0.06 |   |
| Concurrent MTX |   |   |   |   |   | 0.24 |   |   |   |   | 0.20 |
| Yes | 880 | 0.52 | 0.37 | 0.75 | 0.0004 |   | 0.34 | 0.14 | 0.54 | 0.0006 |   |
| No | 397 | 0.76 | 0.45 | 1.29 | 0.31 |   | 0.12 | −0.12 | 0.36 | 0.31 |   |
| Disease duration by cohort |   |   |   |   |   | 0.67 |   |   |   |   | 0.78 |
| Early-onset cohort | 635 | 0.62 | 0.42 | 0.93 | 0.02 |   | 0.25 | 0.05 | 0.46 | 0.02 |   |
| Other cohorts | 648 | 0.55 | 0.35 | 0.85 | 0.007 |   | 0.29 | 0.07 | 0.52 | 0.01 |   |
| Disease duration by years since onset |   |   |   |   |   | 0.73 |   |   |   |   | 0.70 |
| Early onset (<2 years) | 499 | 0.62 | 0.39 | 0.99 | 0.05 |   | 0.24 | −0.002 | 0.48 | 0.05 |   |
| Not early onset | 784 | 0.56 | 0.38 | 0.82 | 0.003 |   | 0.29 | 0.10 | 0.49 | 0.004 |   |

Table 4 Caption: Stratified analyses of interactive associations of the PTPRC single-nucleotide polymorphism (rs10919563) with a EULAR good response and with the ΔDAS28*

When we restricted our analyses of the *PTPRC* SNP to the subcategory of seropositive patients with RA, we found less evidence of heterogeneity in the associations across the cohorts (heterogeneity in association with a EULAR good response versus no response, *P* = 0.11; heterogeneity in association with the ΔDAS28, *P* = 0.02). The association with the *PTPRC* SNP was also strongest within those subgroups that had the largest number of patients (e.g., infliximab-treated patients [n = 625], observational study design [n = 666], female sex [n = 975], and concurrent treatment with MTX [n = 955]). However, except for the study design variable in the analysis of association with the ΔDAS28, none of the clinical categories showed an OR that reached a convincing level of statistical significance (*P* < 0.05) when any other SNP was assessed (see [Supplementary Table 6](#SD1) [stratified by serologic status] and [Supplementary Table 7](#SD1) [stratified by anti-TNF drug] for detailed results on all of the RA risk SNPs, available on the *Arthritis & Rheumatism* Web site at [http://www3.interscience.wiley.com/journal/76509746/home](http://www3.interscience.wiley.com/journal/76509746/home)).

Finally, we assessed whether population stratification might account for our findings, despite the fact that the RA patients self-reported almost exclusively white European ancestry. We genotyped 3 AIMs and tested for an association with treatment outcome (see [Supplementary Figure 1](#SD1), available on the *Arthritis & Rheumatism* Web site at [http://www3.interscience.wiley.com/journal/76509746/home](http://www3.interscience.wiley.com/journal/76509746/home)). Only 1 of the 3 AIMs, rs7696175 (*TLR1* locus), showed an association with our primary end point, a EULAR good response versus no response (*P* = 0.001), and with our secondary end point, the ΔDAS28 (*P* = 0.002). However, there was no correlation between the *PTPRC* SNP association and the *TLR1* AIM association; the frequency of the *PTPRC* allele was similar across the 3 *TLR1* genotype classes. The cohort demonstrating the strongest association between the *PTPRC* SNP and treatment response (the KI cohort; OR 0.12) showed no association between the *TLR1* AIM and treatment response. Moreover, inclusion of the *TLR1* SNP in our multivariate model did not change the association of treatment response with the *PTPRC* SNP (results not shown). When we limited our analysis to seropositive patients with RA, we found only modest evidence of an association of treatment response with the *TLR1* AIM (*P* = 0.01), although there was continued evidence of an association with the *PTPRC* locus (*P* = 0.0003 for the association with the primary end point).

## DISCUSSION

In this study, we have identified an RA risk allele, rs10919563 in the *PTPRC/CD45* gene, that is also associated with the response to anti-TNF therapy in a large collection of RA patients. The association was found to be strongest among those patients who were seropositive for either RF or ACPA autoantibodies.

The statistical evidence in favor of a true-positive result indicating an association with the *PTPRC* SNP is very strong (*P* = 0.0001 in our multivariate model of EULAR good response versus no response), but not to the point where this finding can be classified as an unambiguous genetic biomarker of treatment response. In this study, we adjusted for multiple hypothesis testing according to the number of SNPs tested (n = 31), which yielded a *P* value of less than 0.0016, indicating statistical significance. The finding of an association at the *PTPRC* gene locus clearly surpasses this level of significance when assessed in relation to our primary and secondary end points. Nonetheless, independent replication is required to confirm definitively that the *PTPRC* SNP is associated with response to anti-TNF therapy.

There are important sources of heterogeneity that may confound our results. We combined the results across 9 different cohorts, each of which is quite different with regard to study design, ascertainment criteria, and duration of followup (as well as other clinical factors, as shown in [Table 1](#T1)). We chose to balance the gain in power achieved by the increase in sample size by the potential bias introduced through unmeasured confounding variables. To minimize any bias, we developed a clinical prediction model ([Table 2](#T2)) and restricted our primary analysis to the 2 most dichotomous EULAR categories of response. For our *PTPRC* finding, we found an association in autoantibody-positive (*P* = 0.0003), but not autoantibody-negative, patients, although the difference between the 2 categories was not statistically significant ([Table 4](#T4)).

Another potential source of heterogeneity is population ancestry. Although we did not have access to detailed ethnic information on all of the patients, and we do not yet have access to genome-wide SNP genotype data to match subjects using genetic data, the majority of patients from these collections are of European ancestry. In addition, we observed that the association with *PTPRC* stood out from the other 31 SNPs, as well from 3 SNPs that are highly differentiated across European populations (see [Supplementary Figure 1](#SD1) on the *Arthritis & Rheumatism* Web site at [http://www3.interscience.wiley.com/journal/76509746/home](http://www3.interscience.wiley.com/journal/76509746/home)). Taken together, these results suggest that population stratification alone does not account for the *PTPRC* result.

If the *PTPRC* association is confirmed in additional RA patient collections, then one of the most important applications will be in providing biologic insight into the mechanism by which some patients respond to treatment and others do not. The SNP associated with both RA and response to anti-TNF therapy lies within the *PTPRC* gene (also known as *CD45*). *PTPRC*, which is a transmembrane receptor–like molecule specifically expressed on the cell surface of all nucleated hematopoietic cells, is an essential regulator of T and B cell antigen receptor signaling ([47](#R47)) and a mediator of TNFα secretion from monocytes ([48](#R48)). Elegant studies in the mouse and human have demonstrated that *PTPRC* affects cellular responses by controlling the relative threshold of sensitivity to external stimuli, including secreted cytokines ([49](#R49)).

The application of our *PTPRC* finding to the clinical care of RA patients is not yet known. Given the modest effect on treatment response (OR of 0.55 in a multivariate model), this finding alone will likely have little impact on determining which patients should receive anti-TNF therapy. As additional genetic variants are identified, a composite genetic prediction score might provide sufficient discrimination between responders and nonresponders to be clinically useful. In addition, it will be important to test whether the *PTPRC* allele is associated with response to other medications used to treat RA, or whether the association is specific to anti-TNF therapy.

There are several strengths in our study. First, we have assembled one of the largest collections of RA patients treated with anti-TNF therapy available to date for pharmacogenetic studies. The largest previous anti-TNF pharmacogenetic study was in 1,070 RA patients ([27](#R27)). Our sample size improves the power to detect common variants of moderate effect size (OR >1.5, such as that observed for the *PTPRC* variant in our study), but is still underpowered to detect more modest effects, such as observed for most known RA risk loci outside of the MHC region. Second, we focused our primary analysis on more extreme response categories—EULAR good response versus no response, excluding moderate response patients—rather than on all 3 EULAR categories or ΔDAS28 as a quantitative trait. A dichotomous approach has conceptual appeal, given that the DAS28 is more accurate for patients with either high or low disease activity ([46](#R46)). Whether this approach truly improves power in RA pharmacogenetic studies will ultimately require additional true-positive associations for empirical comparison. Third, we have focused on SNPs with a higher prior probability of being functional and important in RA pathogenesis, the RA risk alleles. This approach facilitates interpretation of statistical significance and biologic plausibility.

Although we have strong statistical evidence to support an association with one RA risk allele (rs10919563 in the *PTPRC* gene), our study also demonstrates that the majority of RA risk alleles do not appear to be associated with treatment response. This finding is consistent with that of a recent study from the BRAGGSS, which showed no evidence for association between *PTPN22* or *HLA–DRB1* shared epitope alleles and treatment response ([26](#R26)). (Of note, only 5 of the 9 cohorts had 4-digit *HLA–DRB1* genotype data available, and therefore we were unable to test whether shared epitope alleles are associated with treatment response in our study.) It is possible that RA risk alleles have much more modest effects on response to anti-TNF therapy, and that our study is underpowered to detect these effects. However, we failed to see any trend toward significance based on directionality of the effect on RA risk and response to anti-TNF therapy (see [Supplementary Tables 2 and 3](#SD1) on the *Arthritis & Rheumatism* Web site at [http://www3.interscience.wiley.com/journal/76509746/home](http://www3.interscience.wiley.com/journal/76509746/home)). Alternatively, alleles associated with response to therapy may be different from those that are associated with disease risk.

In conclusion, we present strong statistical evidence in favor of a true-positive association between a SNP in the *PTPRC* gene and response to anti-TNF therapy in RA patients, especially among those seropositive for either ACPAs or RF autoantibodies. If additional studies confirm our findings, this will guide functional studies to understand how the *PTPRC* genetic variant provides biologic insight into why some patients respond to anti-TNF therapy while others do not. In addition, it is also clear that an unbiased scan of the human genome will be required to identify novel genetic factors associated with response to anti-TNF therapy.

## Supplementary Material

## ACKNOWLEDGMENTS

We thank Dr. Johan Bratt, head of the Department of Rheumatology at Karolinska University Hospital, and Dr. Jon Lampa for their help in the preparation of the KI study group, Dr. Staffan Lindblad from the Swedish Rheumatology Register, and the clinicians who recruited patients to EIRA and followed up the EIRA and KI patients throughout the study duration. We also acknowledge the help of Prof. B. A. C. Dijkmans, D. van Schaardenburg, A.S. Peña, P. L. Klarenbeek, M. T. Nurmohamed, W. F. Lems, R. R. J. van de Stadt, M. G. M. Bartelds, D. M. Gerlag, and C. A. Wijbrandts in gathering AMC and JBI subject samples and data.

Supported by the American College of Rheumatology Research and Education Foundation (Within Our Reach program grant to Dr. Plenge), the Netherlands Organisation for Health Research and Development (ZonMw grant 945-02-029), and the European Community Sixth Framework Programme funding (AutoCure). The Brigham Rheumatoid Arthritis Sequential Study Registry is supported by a grant from Biogen Idec and Crescendo Bioscience. The Epidemiological Investigation of Rheumatoid Arthritis study was supported by grants from the Swedish Medical Research Council, the Stockholm County Council, the Flight Attendant Medical Research Institute, the Swedish Council for Working Life and Social Research, King Gustaf V’s 80-Year Foundation, the Swedish Rheumatism Association, and the Swedish COMBINE project. The Karolinska Institutet study was supported by a grant from the Swedish Rheumatism Association. Dr. Cui is recipient of an American College of Rheumatology Research and Education Foundation Health Professional New Investigator award. Dr. Saevarsdottir’s work was supported by the Swedish Rheumatism Association, the Swedish Medical Research Council, and the Stockholm County Council. Dr. Padyukov’s work was supported by the Swedish Rheumatism Association and the Swedish Medical Research Council. Drs. de Vries, Herenius, and Tak’s work was supported by the European Community Sixth Framework Programme funding (AutoCure) and the Netherlands Organisation for Health Research and Development (ZonMw). Dr. Raychaudhuri’s work was supported by the NIH (grant K08-AR-055688). Dr. Alfredsson’s work was supported by the Swedish Medical Research Council, the Stockholm County Council, the Swedish Council for Working Life and Social Research, King Gustav V’s 80-Year Foundation, the Swedish COMBINE Project, and the Flight Attendant Medical Research Insitute. Drs. Worthington and Barton’s work was supported by Arthritis Research UK (ARC grant 17552). Drs. Seldin, Criswell, and Bridges’ work was supported by the NIH (grant R01-AI/AR-47487). Dr. Gregersen’s work was supported by the NIH (grant N01-AR-1-2256 to the Autoimmune Biomarkers Collaborative Network). Dr. Plenge’s work was supported by the NIH (National Institute of Arthritis and Musculoskeletal and Skin Diseases grants R01-AR-056768 and R01-AR-057108) and the William Randolph Hearst Fund of Harvard University; he is also recipient of a Career Award for Medical Scientists from the Burroughs Wellcome Fund.

Dr. Askling has received consulting fees, speaking fees, and/or honoraria from Wyeth, Bristol-Myers Squibb, and Schering-Plough (less than $10,000 each). Dr. Weinblatt has received consulting fees, from Biogen, Crescendo, Abbott, Amgen, Centocor, UCB, and Pfizer and has received research grants from Abbott, Crescendo Bioscience, and Biogen Idec. Dr. Shadick has received research grants from Crescendo Bioscience and Biogen Idec. Dr. Allaart has received consulting fees, speaking fees, and/or honoraria from Schering-Plough (less than $10,000). Dr. Huizinga has received consulting fees, speaking fees, and/or honoraria from Schering-Plough, Bristol-Myers Squibb, Biotest, Wyeth/Pfizer, Novartis, Roche, Sanofi-Aventis, Abbott, and Axis-Shield (less than $10,000 each) and received travel support from Roche and Abbott. Dr. Plenge has received consulting fees, speaking fees, and/or honoraria from Biogen Idec (less than $10,000).

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Klareskog L, Catrina AI, Paget S. Rheumatoid arthritis. Lancet. 2009;373:659–672. doi: 10.1016/S0140-6736(09)60008-8.  [DOI](https://doi.org/10.1016/S0140-6736(09)60008-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19157532/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Rheumatoid%20arthritis&author=L%20Klareskog&author=AI%20Catrina&author=S%20Paget&volume=373&publication_year=2009&pages=659-672&pmid=19157532&doi=10.1016/S0140-6736(09)60008-8&)

2. Finckh A, Liang MH, van Herckenrode CM, de Pablo P. Long-term impact of early treatment on radiographic progression in rheumatoid arthritis: a meta-analysis. Arthritis Rheum. 2006;55:864–872. doi: 10.1002/art.22353.  [DOI](https://doi.org/10.1002/art.22353) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17139662/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arthritis%20Rheum&title=Long-term%20impact%20of%20early%20treatment%20on%20radiographic%20progression%20in%20rheumatoid%20arthritis:%20a%20meta-analysis&author=A%20Finckh&author=MH%20Liang&author=CM%20van%20Herckenrode&author=P%20de%20Pablo&volume=55&publication_year=2006&pages=864-872&pmid=17139662&doi=10.1002/art.22353&)

3. Maini RN, Breedveld FC, Kalden JR, Smolen JS, Davis D, Macfarlane JD, et al. Therapeutic efficacy of multiple intravenous infusions of anti-tumor necrosis factor α monoclonal antibody combined with low-dose weekly methotrexate in rheumatoid arthritis. Arthritis Rheum. 1998;41:1552–1563. doi: 10.1002/1529-0131(199809)41:9<1552::AID-ART5>3.0.CO;2-W.  [DOI](https://doi.org/10.1002/1529-0131(199809)41:9<1552::AID-ART5>3.0.CO;2-W) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9751087/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arthritis%20Rheum&title=Therapeutic%20efficacy%20of%20multiple%20intravenous%20infusions%20of%20anti-tumor%20necrosis%20factor%20%CE%B1%20monoclonal%20antibody%20combined%20with%20low-dose%20weekly%20methotrexate%20in%20rheumatoid%20arthritis&author=RN%20Maini&author=FC%20Breedveld&author=JR%20Kalden&author=JS%20Smolen&author=D%20Davis&volume=41&publication_year=1998&pages=1552-1563&pmid=9751087&doi=10.1002/1529-0131(199809)41:9<1552::AID-ART5>3.0.CO;2-W&)

4. Klareskog L, van der Heijde D, de Jager JP, Gough A, Kalden J, Malaise M, et al. TEMPO (Trial of Etanercept and Methotrexate with Radiographic Patient Outcomes) study investigators Therapeutic effect of the combination of etanercept and methotrexate compared with each treatment alone in patients with rheumatoid arthritis: double-blind randomised controlled trial. Lancet. 2004;363:675–681. doi: 10.1016/S0140-6736(04)15640-7.  [DOI](https://doi.org/10.1016/S0140-6736(04)15640-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15001324/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=TEMPO%20(Trial%20of%20Etanercept%20and%20Methotrexate%20with%20Radiographic%20Patient%20Outcomes)%20study%20investigators%20Therapeutic%20effect%20of%20the%20combination%20of%20etanercept%20and%20methotrexate%20compared%20with%20each%20treatment%20alone%20in%20patients%20with%20rheumatoid%20arthritis:%20double-blind%20randomised%20controlled%20trial&author=L%20Klareskog&author=D%20van%20der%20Heijde&author=JP%20de%20Jager&author=A%20Gough&author=J%20Kalden&volume=363&publication_year=2004&pages=675-681&pmid=15001324&doi=10.1016/S0140-6736(04)15640-7&)

5. Hyrich KL, Watson KD, Silman AJ, Symmons DP for the British Society for Rheumatology Biologics Register. Predictors of response to anti-TNF-α therapy among patients with rheumatoid arthritis: results from the British Society for Rheumatology Biologics Register. Rheumatology (Oxford) 2006;45:1558–1565. doi: 10.1093/rheumatology/kel149.  [DOI](https://doi.org/10.1093/rheumatology/kel149) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16705046/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Rheumatology%20(Oxford)&title=Predictors%20of%20response%20to%20anti-TNF-%CE%B1%20therapy%20among%20patients%20with%20rheumatoid%20arthritis:%20results%20from%20the%20British%20Society%20for%20Rheumatology%20Biologics%20Register&author=KL%20Hyrich&author=KD%20Watson&author=AJ%20Silman&author=DP%20Symmons&volume=45&publication_year=2006&pages=1558-1565&pmid=16705046&doi=10.1093/rheumatology/kel149&)

6. Gibbons LJ, Hyrich KL. Biologic therapy for rheumatoid arthritis: clinical efficacy and predictors of response. BioDrugs. 2009;23:111–124. doi: 10.2165/00063030-200923020-00004.  [DOI](https://doi.org/10.2165/00063030-200923020-00004) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19489652/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BioDrugs&title=Biologic%20therapy%20for%20rheumatoid%20arthritis:%20clinical%20efficacy%20and%20predictors%20of%20response&author=LJ%20Gibbons&author=KL%20Hyrich&volume=23&publication_year=2009&pages=111-124&pmid=19489652&doi=10.2165/00063030-200923020-00004&)

7. Plenge RM, Criswell LA. Genetic variants that predict response to anti-tumor necrosis factor therapy in rheumatoid arthritis: current challenges and future directions. Curr Opin Rheumatol. 2008;20:145–152. doi: 10.1097/BOR.0b013e3282f5135b.  [DOI](https://doi.org/10.1097/BOR.0b013e3282f5135b) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18349743/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Opin%20Rheumatol&title=Genetic%20variants%20that%20predict%20response%20to%20anti-tumor%20necrosis%20factor%20therapy%20in%20rheumatoid%20arthritis:%20current%20challenges%20and%20future%20directions&author=RM%20Plenge&author=LA%20Criswell&volume=20&publication_year=2008&pages=145-152&pmid=18349743&doi=10.1097/BOR.0b013e3282f5135b&)

8. Wijbrandts CA, Dijkgraaf MG, Kraan MC, Vinkenoog M, Smeets TJ, Dinant H, et al. The clinical response to infliximab in rheumatoid arthritis is in part dependent on pretreatment tumour necrosis factor α expression in the synovium. Ann Rheum Dis. 2008;67:1139–1144. doi: 10.1136/ard.2007.080440.  [DOI](https://doi.org/10.1136/ard.2007.080440) | [PMC free article](/articles/PMC2564801/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18055470/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Rheum%20Dis&title=The%20clinical%20response%20to%20infliximab%20in%20rheumatoid%20arthritis%20is%20in%20part%20dependent%20on%20pretreatment%20tumour%20necrosis%20factor%20%CE%B1%20expression%20in%20the%20synovium&author=CA%20Wijbrandts&author=MG%20Dijkgraaf&author=MC%20Kraan&author=M%20Vinkenoog&author=TJ%20Smeets&volume=67&publication_year=2008&pages=1139-1144&pmid=18055470&doi=10.1136/ard.2007.080440&)

9. Plenge RM. Recent progress in rheumatoid arthritis genetics: one step towards improved patient care. Curr Opin Rheumatol. 2009;21:262–271. doi: 10.1097/BOR.0b013e32832a2e2d.  [DOI](https://doi.org/10.1097/BOR.0b013e32832a2e2d) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19365266/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Opin%20Rheumatol&title=Recent%20progress%20in%20rheumatoid%20arthritis%20genetics:%20one%20step%20towards%20improved%20patient%20care&author=RM%20Plenge&volume=21&publication_year=2009&pages=262-271&pmid=19365266&doi=10.1097/BOR.0b013e32832a2e2d&)

10. Bowes J, Barton A. Recent advances in the genetics of RA susceptibility. Rheumatology (Oxford) 2008;47:399–402. doi: 10.1093/rheumatology/ken005.  [DOI](https://doi.org/10.1093/rheumatology/ken005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18263596/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Rheumatology%20(Oxford)&title=Recent%20advances%20in%20the%20genetics%20of%20RA%20susceptibility&author=J%20Bowes&author=A%20Barton&volume=47&publication_year=2008&pages=399-402&pmid=18263596&doi=10.1093/rheumatology/ken005&)

11. Gregersen PK, Silver J, Winchester RJ. The shared epitope hypothesis: an approach to understanding the molecular genetics of susceptibility to rheumatoid arthritis. Arthritis Rheum. 1987;30:1205–1213. doi: 10.1002/art.1780301102.  [DOI](https://doi.org/10.1002/art.1780301102) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2446635/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arthritis%20Rheum&title=The%20shared%20epitope%20hypothesis:%20an%20approach%20to%20understanding%20the%20molecular%20genetics%20of%20susceptibility%20to%20rheumatoid%20arthritis&author=PK%20Gregersen&author=J%20Silver&author=RJ%20Winchester&volume=30&publication_year=1987&pages=1205-1213&pmid=2446635&doi=10.1002/art.1780301102&)

12. Raychaudhuri S, Remmers EF, Lee AT, Hackett R, Guiducci C, Burtt NP, et al. Common variants at CD40 and other loci confer risk of rheumatoid arthritis. Nat Genet. 2008;40:1216–1223. doi: 10.1038/ng.233.  [DOI](https://doi.org/10.1038/ng.233) | [PMC free article](/articles/PMC2757650/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18794853/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Common%20variants%20at%20CD40%20and%20other%20loci%20confer%20risk%20of%20rheumatoid%20arthritis&author=S%20Raychaudhuri&author=EF%20Remmers&author=AT%20Lee&author=R%20Hackett&author=C%20Guiducci&volume=40&publication_year=2008&pages=1216-1223&pmid=18794853&doi=10.1038/ng.233&)

13. Plenge RM, Cotsapas C, Davies L, Price AL, de Bakker PI, Maller J, et al. Two independent alleles at 6q23 associated with risk of rheumatoid arthritis. Nat Genet. 2007;39:1477–1482. doi: 10.1038/ng.2007.27.  [DOI](https://doi.org/10.1038/ng.2007.27) | [PMC free article](/articles/PMC2652744/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17982456/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Two%20independent%20alleles%20at%206q23%20associated%20with%20risk%20of%20rheumatoid%20arthritis&author=RM%20Plenge&author=C%20Cotsapas&author=L%20Davies&author=AL%20Price&author=PI%20de%20Bakker&volume=39&publication_year=2007&pages=1477-1482&pmid=17982456&doi=10.1038/ng.2007.27&)

14. Plenge RM, Seielstad M, Padyukov L, Lee AT, Remmers EF, Ding B, et al. TRAF1-C5 as a risk locus for rheumatoid arthritis: a genomewide study. N Engl J Med. 2007;357:1199–1209. doi: 10.1056/NEJMoa073491.  [DOI](https://doi.org/10.1056/NEJMoa073491) | [PMC free article](/articles/PMC2636867/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17804836/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=TRAF1-C5%20as%20a%20risk%20locus%20for%20rheumatoid%20arthritis:%20a%20genomewide%20study&author=RM%20Plenge&author=M%20Seielstad&author=L%20Padyukov&author=AT%20Lee&author=EF%20Remmers&volume=357&publication_year=2007&pages=1199-1209&pmid=17804836&doi=10.1056/NEJMoa073491&)

15. Thomson W, Barton A, Ke X, Eyre S, Hinks A, Bowes J, et al. Rheumatoid arthritis association at 6q23. Nat Genet. 2007;39:1431–1433. doi: 10.1038/ng.2007.32.  [DOI](https://doi.org/10.1038/ng.2007.32) | [PMC free article](/articles/PMC2674282/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17982455/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Rheumatoid%20arthritis%20association%20at%206q23&author=W%20Thomson&author=A%20Barton&author=X%20Ke&author=S%20Eyre&author=A%20Hinks&volume=39&publication_year=2007&pages=1431-1433&pmid=17982455&doi=10.1038/ng.2007.32&)

16. Kurreeman FA, Padyukov L, Marques RB, Schrodi SJ, Seddighzadeh M, Stoeken-Rijsbergen G, et al. A candidate gene approach identifies the TRAF1/C5 region as a risk factor for rheumatoid arthritis. PLoS Med. 2007;4:e278. doi: 10.1371/journal.pmed.0040278.  [DOI](https://doi.org/10.1371/journal.pmed.0040278) | [PMC free article](/articles/PMC1976626/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17880261/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Med&title=A%20candidate%20gene%20approach%20identifies%20the%20TRAF1/C5%20region%20as%20a%20risk%20factor%20for%20rheumatoid%20arthritis&author=FA%20Kurreeman&author=L%20Padyukov&author=RB%20Marques&author=SJ%20Schrodi&author=M%20Seddighzadeh&volume=4&publication_year=2007&pages=e278&pmid=17880261&doi=10.1371/journal.pmed.0040278&)

17. Raychaudhuri S, Thomson BP, Remmers EF, Eyre S, Hinks A, Guiducci C, et al. Genetic variants at CD28, PRDM1 and CD2/CD58 are associated with rheumatoid arthritis risk. Nat Genet. 2009;41:1313–1318. doi: 10.1038/ng.479.  [DOI](https://doi.org/10.1038/ng.479) | [PMC free article](/articles/PMC3142887/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19898481/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Genetic%20variants%20at%20CD28,%20PRDM1%20and%20CD2/CD58%20are%20associated%20with%20rheumatoid%20arthritis%20risk&author=S%20Raychaudhuri&author=BP%20Thomson&author=EF%20Remmers&author=S%20Eyre&author=A%20Hinks&volume=41&publication_year=2009&pages=1313-1318&pmid=19898481&doi=10.1038/ng.479&)

18. Plenge RM, Padyukov L, Remmers EF, Purcell S, Lee AT, Karlson EW, et al. Replication of putative candidate-gene associations with rheumatoid arthritis in >4,000 samples from North America and Sweden: association of susceptibility with PTPN22, CTLA4, and PADI4. Am J Hum Genet. 2005;77:1044–1060. doi: 10.1086/498651.  [DOI](https://doi.org/10.1086/498651) | [PMC free article](/articles/PMC1285162/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16380915/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Replication%20of%20putative%20candidate-gene%20associations%20with%20rheumatoid%20arthritis%20in%20>4,000%20samples%20from%20North%20America%20and%20Sweden:%20association%20of%20susceptibility%20with%20PTPN22,%20CTLA4,%20and%20PADI4&author=RM%20Plenge&author=L%20Padyukov&author=EF%20Remmers&author=S%20Purcell&author=AT%20Lee&volume=77&publication_year=2005&pages=1044-1060&pmid=16380915&doi=10.1086/498651&)

19. Gregersen PK, Amos CI, Lee AT, Lu Y, Remmers EF, Kastner DL, et al. REL, encoding a member of the NF-κB family of transcription factors, is a newly defined risk locus for rheumatoid arthritis. Nat Genet. 2009;41:820–823. doi: 10.1038/ng.395.  [DOI](https://doi.org/10.1038/ng.395) | [PMC free article](/articles/PMC2705058/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19503088/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=REL,%20encoding%20a%20member%20of%20the%20NF-%CE%BAB%20family%20of%20transcription%20factors,%20is%20a%20newly%20defined%20risk%20locus%20for%20rheumatoid%20arthritis&author=PK%20Gregersen&author=CI%20Amos&author=AT%20Lee&author=Y%20Lu&author=EF%20Remmers&volume=41&publication_year=2009&pages=820-823&pmid=19503088&doi=10.1038/ng.395&)

20. Altshuler D, Hirschhorn JN, Klannemark M, Lindgren CM, Vohl MC, Nemesh J, et al. The common PPARγ Pro12Ala polymorphism is associated with decreased risk of type 2 diabetes. Nat Genet. 2000;26:76–80. doi: 10.1038/79216.  [DOI](https://doi.org/10.1038/79216) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10973253/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=The%20common%20PPAR%CE%B3%20Pro12Ala%20polymorphism%20is%20associated%20with%20decreased%20risk%20of%20type%202%20diabetes&author=D%20Altshuler&author=JN%20Hirschhorn&author=M%20Klannemark&author=CM%20Lindgren&author=MC%20Vohl&volume=26&publication_year=2000&pages=76-80&pmid=10973253&doi=10.1038/79216&)

21. Kathiresan S, Musunuru K, Orho-Melander M. Defining the spectrum of alleles that contribute to blood lipid concentrations in humans. Curr Opin Lipidol. 2008;19:122–127. doi: 10.1097/MOL.0b013e3282f70296.  [DOI](https://doi.org/10.1097/MOL.0b013e3282f70296) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18388691/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Opin%20Lipidol&title=Defining%20the%20spectrum%20of%20alleles%20that%20contribute%20to%20blood%20lipid%20concentrations%20in%20humans&author=S%20Kathiresan&author=K%20Musunuru&author=M%20Orho-Melander&volume=19&publication_year=2008&pages=122-127&pmid=18388691&doi=10.1097/MOL.0b013e3282f70296&)

22. Arnett FC, Edworthy SM, Bloch DA, McShane DJ, Fries JF, Cooper NS, et al. The American Rheumatism Association 1987 revised criteria for the classification of rheumatoid arthritis. Arthritis Rheum. 1988;31:315–324. doi: 10.1002/art.1780310302.  [DOI](https://doi.org/10.1002/art.1780310302) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/3358796/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arthritis%20Rheum&title=The%20American%20Rheumatism%20Association%201987%20revised%20criteria%20for%20the%20classification%20of%20rheumatoid%20arthritis&author=FC%20Arnett&author=SM%20Edworthy&author=DA%20Bloch&author=DJ%20McShane&author=JF%20Fries&volume=31&publication_year=1988&pages=315-324&pmid=3358796&doi=10.1002/art.1780310302&)

23. Liu C, Batliwalla F, Li W, Lee A, Roubenoff R, Beckman E, et al. Genome-wide association scan identifies candidate polymorphisms associated with differential response to anti-TNF treatment in rheumatoid arthritis. Mol Med. 2008;14:575–581. doi: 10.2119/2008-00056.Liu.  [DOI](https://doi.org/10.2119/2008-00056.Liu) | [PMC free article](/articles/PMC2276142/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18615156/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Med&title=Genome-wide%20association%20scan%20identifies%20candidate%20polymorphisms%20associated%20with%20differential%20response%20to%20anti-TNF%20treatment%20in%20rheumatoid%20arthritis&author=C%20Liu&author=F%20Batliwalla&author=W%20Li&author=A%20Lee&author=R%20Roubenoff&volume=14&publication_year=2008&pages=575-581&pmid=18615156&doi=10.2119/2008-00056.Liu&)

24. Wijbrandts CA, Klaasen R, Dijkgraaf MG, Gerlag DM, van Eck-Smit BL, Tak PP. Bone mineral density in rheumatoid arthritis patients 1 year after adalimumab therapy: arrest of bone loss. Ann Rheum Dis. 2009;68:373–376. doi: 10.1136/ard.2008.091611.  [DOI](https://doi.org/10.1136/ard.2008.091611) | [PMC free article](/articles/PMC2945478/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18408246/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Rheum%20Dis&title=Bone%20mineral%20density%20in%20rheumatoid%20arthritis%20patients%201%20year%20after%20adalimumab%20therapy:%20arrest%20of%20bone%20loss&author=CA%20Wijbrandts&author=R%20Klaasen&author=MG%20Dijkgraaf&author=DM%20Gerlag&author=BL%20van%20Eck-Smit&volume=68&publication_year=2009&pages=373-376&pmid=18408246&doi=10.1136/ard.2008.091611&)

25. Goekoop-Ruiterman YP, de Vries-Bouwstra JK, Allaart CF, van Zeben D, Kerstens PJ, Hazes JM, et al. Clinical and radiographic outcomes of four different treatment strategies in patients with early rheumatoid arthritis (the BeSt study): a randomized, controlled trial. Arthritis Rheum. 2005;52:3381–3390. doi: 10.1002/art.21405.  [DOI](https://doi.org/10.1002/art.21405) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16258899/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arthritis%20Rheum&title=Clinical%20and%20radiographic%20outcomes%20of%20four%20different%20treatment%20strategies%20in%20patients%20with%20early%20rheumatoid%20arthritis%20(the%20BeSt%20study):%20a%20randomized,%20controlled%20trial&author=YP%20Goekoop-Ruiterman&author=JK%20de%20Vries-Bouwstra&author=CF%20Allaart&author=D%20van%20Zeben&author=PJ%20Kerstens&volume=52&publication_year=2005&pages=3381-3390&pmid=16258899&doi=10.1002/art.21405&)

26. Potter C, Hyrich KL, Tracey A, Lunt M, Plant D, Symmons DP, et al. Association of rheumatoid factor and anti-cyclic citrullinated peptide positivity, but not carriage of shared epitope or PTPN22 susceptibility variants, with anti-tumour necrosis factor response in rheumatoid arthritis. Ann Rheum Dis. 2009;68:69–74. doi: 10.1136/ard.2007.084715.  [DOI](https://doi.org/10.1136/ard.2007.084715) | [PMC free article](/articles/PMC2596303/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18375541/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Rheum%20Dis&title=Association%20of%20rheumatoid%20factor%20and%20anti-cyclic%20citrullinated%20peptide%20positivity,%20but%20not%20carriage%20of%20shared%20epitope%20or%20PTPN22%20susceptibility%20variants,%20with%20anti-tumour%20necrosis%20factor%20response%20in%20rheumatoid%20arthritis&author=C%20Potter&author=KL%20Hyrich&author=A%20Tracey&author=M%20Lunt&author=D%20Plant&volume=68&publication_year=2009&pages=69-74&pmid=18375541&doi=10.1136/ard.2007.084715&)

27. Bowes JD, Potter C, Gibbons LJ, Hyrich K, Plant D, Morgan AW, et al. for the BRAGGSS Investigation of genetic variants within candidate genes of the TNFRSF1B signalling pathway on the response to anti-TNF agents in a UK cohort of rheumatoid arthritis patients. Pharmacogenet Genomics. 2009;19:319–323. doi: 10.1097/FPC.0b013e328328d51f.  [DOI](https://doi.org/10.1097/FPC.0b013e328328d51f) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19262425/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=for%20the%20BRAGGSS%20Investigation%20of%20genetic%20variants%20within%20candidate%20genes%20of%20the%20TNFRSF1B%20signalling%20pathway%20on%20the%20response%20to%20anti-TNF%20agents%20in%20a%20UK%20cohort%20of%20rheumatoid%20arthritis%20patients&author=JD%20Bowes&author=C%20Potter&author=LJ%20Gibbons&author=K%20Hyrich&author=D%20Plant&volume=19&publication_year=2009&pages=319-323&pmid=19262425&doi=10.1097/FPC.0b013e328328d51f&)

28. Sato M, Schneeweiss S, Scranton R, Katz JN, Weinblatt ME, Avorn J, et al. The validity of a rheumatoid arthritis medical records-based index of severity compared with the DAS28. Arthritis Res Ther. 2006;8:R57. doi: 10.1186/ar1921.  [DOI](https://doi.org/10.1186/ar1921) | [PMC free article](/articles/PMC1526615/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16542499/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arthritis%20Res%20Ther&title=The%20validity%20of%20a%20rheumatoid%20arthritis%20medical%20records-based%20index%20of%20severity%20compared%20with%20the%20DAS28&author=M%20Sato&author=S%20Schneeweiss&author=R%20Scranton&author=JN%20Katz&author=ME%20Weinblatt&volume=8&publication_year=2006&pages=R57&pmid=16542499&doi=10.1186/ar1921&)

29. Askling J, Fored CM, Geborek P, Jacobsson LT, van Vollenhoven R, Feltelius N, et al. Swedish registers to examine drug safety and clinical issues in RA. Ann Rheum Dis. 2006;65:707–712. doi: 10.1136/ard.2005.045872.  [DOI](https://doi.org/10.1136/ard.2005.045872) | [PMC free article](/articles/PMC1798175/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16414975/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Rheum%20Dis&title=Swedish%20registers%20to%20examine%20drug%20safety%20and%20clinical%20issues%20in%20RA&author=J%20Askling&author=CM%20Fored&author=P%20Geborek&author=LT%20Jacobsson&author=R%20van%20Vollenhoven&volume=65&publication_year=2006&pages=707-712&pmid=16414975&doi=10.1136/ard.2005.045872&)

30. Bathon JM, Martin RW, Fleischmann RM, Tesser JR, Schiff MH, Keystone EC, et al. A comparison of etanercept and methotrexate in patients with early rheumatoid arthritis. N Engl J Med. 2000;343:1586–1593. doi: 10.1056/NEJM200011303432201.  [DOI](https://doi.org/10.1056/NEJM200011303432201) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11096165/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=A%20comparison%20of%20etanercept%20and%20methotrexate%20in%20patients%20with%20early%20rheumatoid%20arthritis&author=JM%20Bathon&author=RW%20Martin&author=RM%20Fleischmann&author=JR%20Tesser&author=MH%20Schiff&volume=343&publication_year=2000&pages=1586-1593&pmid=11096165&doi=10.1056/NEJM200011303432201&)

31. Criswell LA, Lum RF, Turner KN, Woehl B, Zhu Y, Wang J, et al. The influence of genetic variation in the HLA-DRB1 and LTA-TNF regions on the response to treatment of early rheumatoid arthritis with methotrexate or etanercept. Arthritis Rheum. 2004;50:2750–2756. doi: 10.1002/art.20469.  [DOI](https://doi.org/10.1002/art.20469) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15457442/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arthritis%20Rheum&title=The%20influence%20of%20genetic%20variation%20in%20the%20HLA-DRB1%20and%20LTA-TNF%20regions%20on%20the%20response%20to%20treatment%20of%20early%20rheumatoid%20arthritis%20with%20methotrexate%20or%20etanercept&author=LA%20Criswell&author=RF%20Lum&author=KN%20Turner&author=B%20Woehl&author=Y%20Zhu&volume=50&publication_year=2004&pages=2750-2756&pmid=15457442&doi=10.1002/art.20469&)

32. Padyukov L, Lampa J, Heimburger M, Ernestam S, Cederholm T, Lundkvist I, et al. Genetic markers for the efficacy of tumour necrosis factor blocking therapy in rheumatoid arthritis. Ann Rheum Dis. 2003;62:526–529. doi: 10.1136/ard.62.6.526.  [DOI](https://doi.org/10.1136/ard.62.6.526) | [PMC free article](/articles/PMC1754569/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12759288/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Rheum%20Dis&title=Genetic%20markers%20for%20the%20efficacy%20of%20tumour%20necrosis%20factor%20blocking%20therapy%20in%20rheumatoid%20arthritis&author=L%20Padyukov&author=J%20Lampa&author=M%20Heimburger&author=S%20Ernestam&author=T%20Cederholm&volume=62&publication_year=2003&pages=526-529&pmid=12759288&doi=10.1136/ard.62.6.526&)

33. Prevoo ML, van ’t Hof MA, Kuper HH, van Leeuwen MA, van de Putte LB, Van Riel PL. Modified disease activity scores that include twenty-eight–joint counts: development and validation in a prospective longitudinal study of patients with rheumatoid arthritis. Arthritis Rheum. 1995;38:44–48. doi: 10.1002/art.1780380107.  [DOI](https://doi.org/10.1002/art.1780380107) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7818570/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arthritis%20Rheum&title=Modified%20disease%20activity%20scores%20that%20include%20twenty-eight%E2%80%93joint%20counts:%20development%20and%20validation%20in%20a%20prospective%20longitudinal%20study%20of%20patients%20with%20rheumatoid%20arthritis&author=ML%20Prevoo&author=MA%20van%20%E2%80%99t%20Hof&author=HH%20Kuper&author=MA%20van%20Leeuwen&author=LB%20van%20de%20Putte&volume=38&publication_year=1995&pages=44-48&pmid=7818570&doi=10.1002/art.1780380107&)

34. van Gestel AM, Prevoo ML, van ’t Hof MA, Van Rijswijk MH, van de Putte LB, Van Riel PL. Development and validation of the European League Against Rheumatism response criteria for rheumatoid arthritis: comparison with the preliminary American College of Rheumatology and the World Health Organization/International League Against Rheumatism criteria. Arthritis Rheum. 1996;39:34–40. doi: 10.1002/art.1780390105.  [DOI](https://doi.org/10.1002/art.1780390105) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8546736/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arthritis%20Rheum&title=Development%20and%20validation%20of%20the%20European%20League%20Against%20Rheumatism%20response%20criteria%20for%20rheumatoid%20arthritis:%20comparison%20with%20the%20preliminary%20American%20College%20of%20Rheumatology%20and%20the%20World%20Health%20Organization/International%20League%20Against%20Rheumatism%20criteria&author=AM%20van%20Gestel&author=ML%20Prevoo&author=MA%20van%20%E2%80%99t%20Hof&author=MH%20Van%20Rijswijk&author=LB%20van%20de%20Putte&volume=39&publication_year=1996&pages=34-40&pmid=8546736&doi=10.1002/art.1780390105&)

35. Begovich AB, Carlton VE, Honigberg LA, Schrodi SJ, Chokkalingam AP, Alexander HC, et al. A missense single-nucleotide polymorphism in a gene encoding a protein tyrosine phosphatase (PTPN22) is associated with rheumatoid arthritis. Am J Hum Genet. 2004;75:330–337. doi: 10.1086/422827.  [DOI](https://doi.org/10.1086/422827) | [PMC free article](/articles/PMC1216068/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15208781/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=A%20missense%20single-nucleotide%20polymorphism%20in%20a%20gene%20encoding%20a%20protein%20tyrosine%20phosphatase%20(PTPN22)%20is%20associated%20with%20rheumatoid%20arthritis&author=AB%20Begovich&author=VE%20Carlton&author=LA%20Honigberg&author=SJ%20Schrodi&author=AP%20Chokkalingam&volume=75&publication_year=2004&pages=330-337&pmid=15208781&doi=10.1086/422827&)

36. Suzuki A, Yamada R, Chang X, Tokuhiro S, Sawada T, Suzuki M, et al. Functional haplotypes of PADI4, encoding citrullinating enzyme peptidylarginine deiminase 4, are associated with rheumatoid arthritis. Nat Genet. 2003;34:395–402. doi: 10.1038/ng1206.  [DOI](https://doi.org/10.1038/ng1206) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12833157/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Functional%20haplotypes%20of%20PADI4,%20encoding%20citrullinating%20enzyme%20peptidylarginine%20deiminase%204,%20are%20associated%20with%20rheumatoid%20arthritis&author=A%20Suzuki&author=R%20Yamada&author=X%20Chang&author=S%20Tokuhiro&author=T%20Sawada&volume=34&publication_year=2003&pages=395-402&pmid=12833157&doi=10.1038/ng1206&)

37. Barton A, Thomson W, Ke X, Eyre S, Hinks A, Bowes J, et al. Re-evaluation of putative rheumatoid arthritis susceptibility genes in the post-genome wide association study era and hypothesis of a key pathway underlying susceptibility. Hum Mol Genet. 2008;17:2274–2279. doi: 10.1093/hmg/ddn128.  [DOI](https://doi.org/10.1093/hmg/ddn128) | [PMC free article](/articles/PMC2465799/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18434327/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=Re-evaluation%20of%20putative%20rheumatoid%20arthritis%20susceptibility%20genes%20in%20the%20post-genome%20wide%20association%20study%20era%20and%20hypothesis%20of%20a%20key%20pathway%20underlying%20susceptibility&author=A%20Barton&author=W%20Thomson&author=X%20Ke&author=S%20Eyre&author=A%20Hinks&volume=17&publication_year=2008&pages=2274-2279&pmid=18434327&doi=10.1093/hmg/ddn128&)

38. Barton A, Thomson W, Ke X, Eyre S, Hinks A, Bowes J, et al. Rheumatoid arthritis susceptibility loci at chromosomes 10p15, 12q13 and 22q13. Nat Genet. 2008;40:1156–1159. doi: 10.1038/ng.218.  [DOI](https://doi.org/10.1038/ng.218) | [PMC free article](/articles/PMC2662493/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18794857/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Rheumatoid%20arthritis%20susceptibility%20loci%20at%20chromosomes%2010p15,%2012q13%20and%2022q13&author=A%20Barton&author=W%20Thomson&author=X%20Ke&author=S%20Eyre&author=A%20Hinks&volume=40&publication_year=2008&pages=1156-1159&pmid=18794857&doi=10.1038/ng.218&)

39. Suzuki A, Yamada R, Kochi Y, Sawada T, Okada Y, Matsuda K, et al. Functional SNPs in CD244 increase the risk of rheumatoid arthritis in a Japanese population. Nat Genet. 2008;40:1224–1229. doi: 10.1038/ng.205.  [DOI](https://doi.org/10.1038/ng.205) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18794858/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Functional%20SNPs%20in%20CD244%20increase%20the%20risk%20of%20rheumatoid%20arthritis%20in%20a%20Japanese%20population&author=A%20Suzuki&author=R%20Yamada&author=Y%20Kochi&author=T%20Sawada&author=Y%20Okada&volume=40&publication_year=2008&pages=1224-1229&pmid=18794858&doi=10.1038/ng.205&)

40. Zhernakova A, Alizadeh BZ, Bevova M, van Leeuwen MA, Coenen MJ, Franke B, et al. Novel association in chromosome 4q27 region with rheumatoid arthritis and confirmation of type 1 diabetes point to a general risk locus for autoimmune diseases. Am J Hum Genet. 2007;81:1284–1288. doi: 10.1086/522037.  [DOI](https://doi.org/10.1086/522037) | [PMC free article](/articles/PMC2276343/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17999365/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Novel%20association%20in%20chromosome%204q27%20region%20with%20rheumatoid%20arthritis%20and%20confirmation%20of%20type%201%20diabetes%20point%20to%20a%20general%20risk%20locus%20for%20autoimmune%20diseases&author=A%20Zhernakova&author=BZ%20Alizadeh&author=M%20Bevova&author=MA%20van%20Leeuwen&author=MJ%20Coenen&volume=81&publication_year=2007&pages=1284-1288&pmid=17999365&doi=10.1086/522037&)

41. Lee YC, Raychaudhuri S, Cui J, De Vivo MA, Ding B, Alfredsson L, et al. The PRL-1149 G/T polymorphism and rheumatoid arthritis susceptibility. Arthritis Rheum. 2009;60:1250–1254. doi: 10.1002/art.24468.  [DOI](https://doi.org/10.1002/art.24468) | [PMC free article](/articles/PMC2956274/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19404952/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arthritis%20Rheum&title=The%20PRL-1149%20G/T%20polymorphism%20and%20rheumatoid%20arthritis%20susceptibility&author=YC%20Lee&author=S%20Raychaudhuri&author=J%20Cui&author=MA%20De%20Vivo&author=B%20Ding&volume=60&publication_year=2009&pages=1250-1254&pmid=19404952&doi=10.1002/art.24468&)

42. Lee HS, Lee AT, Criswell LA, Seldin MF, Amos CI, Carulli JP, et al. Several regions in the major histocompatibility complex confer risk for anti-CCP-antibody positive rheumatoid arthritis, independent of the DRB1 locus. Mol Med. 2008;14:293–300. doi: 10.2119/2007-00123.Lee.  [DOI](https://doi.org/10.2119/2007-00123.Lee) | [PMC free article](/articles/PMC2255558/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18309376/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Med&title=Several%20regions%20in%20the%20major%20histocompatibility%20complex%20confer%20risk%20for%20anti-CCP-antibody%20positive%20rheumatoid%20arthritis,%20independent%20of%20the%20DRB1%20locus&author=HS%20Lee&author=AT%20Lee&author=LA%20Criswell&author=MF%20Seldin&author=CI%20Amos&volume=14&publication_year=2008&pages=293-300&pmid=18309376&doi=10.2119/2007-00123.Lee&)

43. De Bakker PI, McVean G, Sabeti PC, Miretti MM, Green T, Marchini J, et al. A high-resolution HLA and SNP haplotype map for disease association studies in the extended human. MHC Nat Genet. 2006;38:1166–1172. doi: 10.1038/ng1885.  [DOI](https://doi.org/10.1038/ng1885) | [PMC free article](/articles/PMC2670196/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16998491/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=MHC%20Nat%20Genet&title=A%20high-resolution%20HLA%20and%20SNP%20haplotype%20map%20for%20disease%20association%20studies%20in%20the%20extended%20human&author=PI%20De%20Bakker&author=G%20McVean&author=PC%20Sabeti&author=MM%20Miretti&author=T%20Green&volume=38&publication_year=2006&pages=1166-1172&pmid=16998491&doi=10.1038/ng1885&)

44. Wellcome Trust Case Control Consortium. Genome-wide association study of 14,000 cases of seven common diseases and 3,000 shared controls. Nature. 2007;447:661–678. doi: 10.1038/nature05911.  [DOI](https://doi.org/10.1038/nature05911) | [PMC free article](/articles/PMC2719288/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17554300/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Genome-wide%20association%20study%20of%2014,000%20cases%20of%20seven%20common%20diseases%20and%203,000%20shared%20controls&volume=447&publication_year=2007&pages=661-678&pmid=17554300&doi=10.1038/nature05911&)

45. Marchini J, Howie B, Myers S, McVean G, Donnelly P. A new multipoint method for genome-wide association studies by imputation of genotypes. Nat Genet. 2007;39:906–913. doi: 10.1038/ng2088.  [DOI](https://doi.org/10.1038/ng2088) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17572673/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=A%20new%20multipoint%20method%20for%20genome-wide%20association%20studies%20by%20imputation%20of%20genotypes&author=J%20Marchini&author=B%20Howie&author=S%20Myers&author=G%20McVean&author=P%20Donnelly&volume=39&publication_year=2007&pages=906-913&pmid=17572673&doi=10.1038/ng2088&)

46. Collier DS, Grant RW, Estey G, Surrao D, Chueh HC, Kay J. Physician ability to assess rheumatoid arthritis disease activity using an electronic medical record-based disease activity calculator. Arthritis Rheum. 2009;61:495–500. doi: 10.1002/art.24335.  [DOI](https://doi.org/10.1002/art.24335) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19333984/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arthritis%20Rheum&title=Physician%20ability%20to%20assess%20rheumatoid%20arthritis%20disease%20activity%20using%20an%20electronic%20medical%20record-based%20disease%20activity%20calculator&author=DS%20Collier&author=RW%20Grant&author=G%20Estey&author=D%20Surrao&author=HC%20Chueh&volume=61&publication_year=2009&pages=495-500&pmid=19333984&doi=10.1002/art.24335&)

47. Hermiston ML, Xu Z, Weiss A. CD45: a critical regulator of signaling thresholds in immune cells. Annu Rev Immunol. 2003;21:107–137. doi: 10.1146/annurev.immunol.21.120601.140946.  [DOI](https://doi.org/10.1146/annurev.immunol.21.120601.140946) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12414720/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annu%20Rev%20Immunol&title=CD45:%20a%20critical%20regulator%20of%20signaling%20thresholds%20in%20immune%20cells&author=ML%20Hermiston&author=Z%20Xu&author=A%20Weiss&volume=21&publication_year=2003&pages=107-137&pmid=12414720&doi=10.1146/annurev.immunol.21.120601.140946&)

48. Hayes AL, Smith C, Foxwell BM, Brennan FM. CD45-induced tumor necrosis factor α production in monocytes is phosphatidyl-inositol 3-kinase-dependent and nuclear factor-κB-independent. J Biol Chem. 1999;274:33455–33461. doi: 10.1074/jbc.274.47.33455.  [DOI](https://doi.org/10.1074/jbc.274.47.33455) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10559228/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=CD45-induced%20tumor%20necrosis%20factor%20%CE%B1%20production%20in%20monocytes%20is%20phosphatidyl-inositol%203-kinase-dependent%20and%20nuclear%20factor-%CE%BAB-independent&author=AL%20Hayes&author=C%20Smith&author=BM%20Foxwell&author=FM%20Brennan&volume=274&publication_year=1999&pages=33455-33461&pmid=10559228&doi=10.1074/jbc.274.47.33455&)

49. Irie-Sasaki J, Sasaki T, Matsumoto W, Opavsky A, Cheng M, Welstead G, et al. CD45 is a JAK phosphatase and negatively regulates cytokine receptor signalling. Nature. 2001;409:349–354. doi: 10.1038/35053086.  [DOI](https://doi.org/10.1038/35053086) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11201744/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=CD45%20is%20a%20JAK%20phosphatase%20and%20negatively%20regulates%20cytokine%20receptor%20signalling&author=J%20Irie-Sasaki&author=T%20Sasaki&author=W%20Matsumoto&author=A%20Opavsky&author=M%20Cheng&volume=409&publication_year=2001&pages=349-354&pmid=11201744&doi=10.1038/35053086&)
