# A Genome-Wide Association Study Identifies Novel Loci for Paclitaxel-Induced Sensory Peripheral Neuropathy in CALGB 40101

## Metadata
**Authors:** R Michael Baldwin, Kouros Owzar, Hitoshi Zembutsu, Aparna Chhibber, Michiaki Kubo, Chen Jiang, Dorothy Watson, Rachel J Eclov, Joel Mefford, Howard L McLeod, Paula N Friedman, Clifford A Hudis, Eric P Winer, Eric M Jorgenson, John S Witte, Lawrence N Shulman, Yusuke Nakamura, Mark J Ratain, Deanna L Kroetz
**Journal:** Clinical cancer research : an official journal of the American Association for Cancer Research
**Date:** 2012 Jul 27
**DOI:** [10.1158/1078-0432.CCR-12-1590](https://doi.org/10.1158/1078-0432.CCR-12-1590)
**PMID:** 22843789
**PMCID:** PMC3445665
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3445665/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC3445665/pdf/nihms396061.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC3445665/pdf/nihms396061.pdf)

## Abstract

**Purpose:** 
Sensory peripheral neuropathy is a common and sometimes debilitating toxicity associated with paclitaxel therapy. This study aims to identify genetic risk factors for development of this toxicity.

**Experimental Design:** 
A prospective pharmacogenetic analysis of primary breast cancer patients randomized to the paclitaxel arm of CALGB 40101 was used to identify genetic predictors of the onset and severity of sensory peripheral neuropathy. A genome-wide association study in 855 subjects of European ancestry was performed and findings were replicated in additional European (n = 154) and African American (n = 117) subjects.

**Results:** 
A single nucleotide polymorphism in FGD4 was associated with the onset of sensory peripheral neuropathy in the discovery cohort (rs10771973; HR, 1.57; 95% CI, 1.30–1.91; P = 2.6 × 10−6) and in a European (HR, 1.72; 95% CI, 1.06–2.80; P = 0.013) and African American (HR, 1.93; 95% CI, 1.13-3.28; P = 6.7 × 10−3) replication cohort. There is also evidence that markers in additional genes, including EPHA5 (rs7349683) and FZD3 (rs10771973), were associated with the onset or severity of paclitaxel-induced sensory peripheral neuropathy.

**Conclusions:** 
A genome-wide association study has identified novel genetic markers of paclitaxel-induced sensory peripheral neuropathy, including a common polymorphism in FGD4, a congenital peripheral neuropathy gene. These findings suggest that genetic variation may contribute to variation in development of this toxicity. Validation of these findings may allow for the identification of patients at increased risk of peripheral neuropathy and inform the use of an alternative to paclitaxel and/or the clinical management of this toxicity.

Keywords: paclitaxel, peripheral neuropathy, breast cancer, pharmacogenetics, genome-wide association study

### Purpose

Sensory peripheral neuropathy is a common and sometimes debilitating toxicity associated with paclitaxel therapy. This study aims to identify genetic risk factors for development of this toxicity.

### Experimental Design

A prospective pharmacogenetic analysis of primary breast cancer patients randomized to the paclitaxel arm of CALGB 40101 was used to identify genetic predictors of the onset and severity of sensory peripheral neuropathy. A genome-wide association study in 855 subjects of European ancestry was performed and findings were replicated in additional European (n = 154) and African American (n = 117) subjects.

### Results

A single nucleotide polymorphism in *FGD4* was associated with the onset of sensory peripheral neuropathy in the discovery cohort (rs10771973; HR, 1.57; 95% CI, 1.30–1.91; *P* = 2.6 × 10^−6^) and in a European (HR, 1.72; 95% CI, 1.06–2.80; *P* = 0.013) and African American (HR, 1.93; 95% CI, 1.13-3.28; *P* = 6.7 × 10^−3^) replication cohort. There is also evidence that markers in additional genes, including *EPHA5* (rs7349683) and *FZD3* (rs10771973), were associated with the onset or severity of paclitaxel-induced sensory peripheral neuropathy.

### Conclusions

A genome-wide association study has identified novel genetic markers of paclitaxel-induced sensory peripheral neuropathy, including a common polymorphism in *FGD4*, a congenital peripheral neuropathy gene. These findings suggest that genetic variation may contribute to variation in development of this toxicity. Validation of these findings may allow for the identification of patients at increased risk of peripheral neuropathy and inform the use of an alternative to paclitaxel and/or the clinical management of this toxicity.

## Introduction

Paclitaxel is a useful microtubule stabilizing agent with efficacy in the treatment of many cancers. It is effective for the treatment of breast cancer in the metastatic, adjuvant and neoadjuvant settings ([1](#R1), [2](#R2)). Sensory peripheral neuropathy remains a significant issue in the clinical utility of this agent. More than 50% of patients experience some degree of sensory peripheral neuropathy during their course of paclitaxel treatment, with 5–30% experiencing Grade 3 or 4 toxicity ([3](#R3), [4](#R4)). Paclitaxel-induced sensory peripheral neuropathy is dose-, treatment schedule- and infusion-time dependent ([3](#R3)). Cumulative dose is a significant predictor of sensory peripheral neuropathy, as is underlying diabetes and concurrent or previous administration of other drugs associated with this toxicity. A recent study suggests that mild to moderate symptoms of sensory peripheral neuropathy can persist for up to two years following completion of paclitaxel treatment ([5](#R5)). Long-term neuropathy is particularly concerning for patients with primary breast cancer, such as those evaluated in the current study, since more than 80% will be long-term survivors whose quality of life will be compromised. Significant sensory peripheral neuropathy during paclitaxel treatment can lead to dose reductions and treatment suspension, possibly resulting in sub-optimal disease treatment and the potential for an increased likelihood of relapse. A predictive marker for this dose-limiting toxicity would enable studies to identify if an individualized assessment of adverse event risk could be useful in the clinical decision-making process. It could also provide a possible target for therapeutic interventions.

Substantial inter-individual differences in the prevalence, reported and objective severity, and onset of peripheral neuropathy is consistent with an underlying genetic susceptibility to this toxicity. CALGB 40101 is a Phase III randomized study comparing cyclophosphamide and doxorubicin versus single agent paclitaxel as adjuvant therapy for breast cancer patients at relatively low risk for relapse. In addition, the study compared short versus longer therapy of each regimen as a 2×2 factorial design. A pharmacogenetic companion study (CALGB 60202) was included in this trial to prospectively evaluate germline determinants of interindividual differences in response and toxicity. An initial analysis of treatment outcome in CALGB 40101 has shown no difference in response between the four and six cycle treatment arms ([6](#R6)); additional analyses of response await complete follow-up data. The goal of this present study was to identify genetic markers predictive of sensory peripheral neuropathies in the paclitaxel treatment arm of CALGB 40101 and to further our understanding of the underlying mechanism of injury and repair. Herein we report the results of a genome-wide association study (GWAS) of 1,040 paclitaxel treated women to identify novel germline susceptibility loci associated with the development of sensory peripheral neuropathies. This represents the largest prospective breast cancer pharmacogenetic study of paclitaxel treatment toxicities to date and provides a paradigm for the identification of genetic markers with potential clinical application in personalized medicine.

## Materials and Methods

### Participants

All study participants were enrolled in CALGB 40101 and gave their additional consent to participate in the pharmacogenetic companion study (CALGB 60202). CALGB 40101 was open from May 15, 2002 until July 30, 2010. The final total accrual was 3,873 patients. Patients eligible for the treatment protocol were females with histologically confirmed invasive carcinoma of the breast and zero to three axillary nodes positive for cancer. ECOG performance status of 0–1, adequate organ function, and absence of CHF or myocardial infarction in the previous six months were required. Enrollment was required within 84 days of breast surgery (either modified radical mastectomy or lumpectomy) and treatment began within seven days of registration. Patients with locally advanced, inflammatory or metastatic breast cancer or involvement of dermal lymphatics were ineligible. Patients were disease-free from any prior malignancies for at least five years. Previous trastuzumab, chemotherapy or hormonal therapy, with the exception of tamoxifen, for the current malignancy was not permitted nor was anthracycline treatment for any previous disease. Patients who received tamoxifen or any other selective estrogen receptor modulators (SERM) for prevention or other indications (e.g. osteoporosis) were eligible. Treatment with tamoxifen, other SERMs or exogenous hormones (e.g. hormone replacement therapy, oral contraceptives, raloxifene) was discontinued prior to enrollment. Trastuzumab was recommended for patients with HER2 positive disease. Patients could also enroll in adjuvant studies of bisphosphonates or hormonal therapies (e.g. ovarian suppression concurrent with chemotherapy). All patients provided written informed consent for both the treatment and companion protocols that met state, federal and institutional guidelines.

### Treatment

Patients were randomly assigned with equal probability to four or six cycles of cyclophosphamide/doxorubicin (AC) or paclitaxel. A full description of the study design is included in a recent publication describing the initial analysis of treatment response ([6](#R6)). The first 570 patients were treated with AC every three weeks, or paclitaxel weekly for 12 or 18 weeks. Thereafter both regimens were administered every two weeks for four or six cycles. Pharmacogenetic samples were collected only from patients enrolled on the every two week regimens who received dose dense paclitaxel for four or six cycles. Paclitaxel was given over three hours at 175 mg/m^2^ when given every two weeks. The six cycle treatment arms for both drugs were closed after enrolling 3,172 patients. Arms were stratified by menopausal, estrogen receptor (ER), progesterone receptor (PgR) and HER2 status. Patient demographics are shown in [Table 1](#T1). Premedication recommendations for the initial dose were 12.5 – 50 mg diphenhydramine and either 50 mg ranitidine, 300 mg cimetidine, or 20 mg famotidine administered IV 30 – 60 minutes prior to paclitaxel. Dexamethasone was given as a 10 mg intravenous dose within 60 minutes of paclitaxel or alternatively, as a 10 mg or 20 mg oral dose more than one hour prior to paclitaxel. To facilitate the 14 day dosing schedule, filgrastim was recommended on days 3–10 of each cycle (5 µg/kg rounded to either 300 or 480 µg). Sargramostim (250–500 µg/m^2^, days 3–10) or pegfilgrastim (6 mg sc, 24–36 hours after paclitaxel) could be used in place of filgrastim. The treating physician could omit G-CSF treatment when confident neutrophils would recover within 14 days, however if treatment could not be delivered on schedule then a G-CSF was required in subsequent cycles. Erythropoetin was permitted at the discretion of the treating physician. Patients positive for HER2 by either IHC 3^+^ staining or gene amplification by FISH could initiate adjuvant trastuzumab concurrent with paclitaxel (weekly administration) or at the completion of paclitaxel (weekly or every 3 weeks). Weekly trastuzumab consisted of a 4 mg/kg intravenous loading dose followed by weekly doses of 2 mg/kg and the three week schedule of a loading dose of 8 mg/kg and 6 mg/kg every three weeks for a total duration of one year.

### Table 1.

|   |   | Randomizeda | Post QCb | Discoveryc | EuropeanReplication | AfricanAmericanReplication |
| --- | --- | --- | --- | --- | --- | --- |
| Sample Size |   | 1940 | 1023 | 855 | 154 | 117 |
| Age | Mean (SD) | 53.4 (9.6) | 53.4 (9.6) | 53.7 (9.6) | 55.2 (9.4) | 54.1 (9.2) |
| Self-ReportedRace and Ethnicity | White (Non-Hispanic/Non-Latino) | 1434 | 788 | 772 | 143 | 1d |
| White (Hispanic or Latino) | 63 | 15 | 1 | - | - |  |
| White (Unknown) | 115 | 77 | 72 | 11 | - |  |
| Black or African American (Non-Hispanic /Non-Latino) | 204 | 85 | - | - | 101 |  |
| Black or African American (Hispanic orLatino) | 10 | 2 | - | - | 1d |  |
| Black or African American (Unknown) | 17 | 13 | 1 | - | 13 |  |
| Asian | 29 | 10 | - | - | - |  |
| Native Hawaiian or Pacific Islander | 2 | 2 | - | - | - |  |
| American Indian or Alaska Native | 18 | 7 | 2 | - | - |  |
| Multiple | 2 | - | - | - | - |  |
| Unknown (Non-Hispanic /Non-Latino) | 9 | 4 | 4 | - | - |  |
| Unknown (Hispanic or Latino) | 24 | 13 | - | - | - |  |
| Unknown (Unknown) | 13 | 7 | 3 | - | 14 |  |
| MenopausalStatus | Post | 1176 (61%) | 609 (60%) | 513 (60%) | 99 (64%) | 81 (69%) |
| Pre | 764 (39%) | 414 (40%) | 342 (40%) | 55 (36%) | 36(31%) |  |
| ER/PgR Status | ER+/PR+ | 1059 (55%) | 546 (53%) | 475 (56%) | 108 (70%) | 35 (30%) |
| ER+ / PR− | 223 (11%) | 119 (12%) | 98 (11%) | 15 (10%) | 18 (15%) |  |
| ER+ / PR unknown | 3 (<1%) | 2 (<1%) | 2 (<1%) | 1 (1%) | 0 (<1%) |  |
| ER− / PR+ | 24 (1%) | 15 (1%) | 13 (2%) | 1 (1%) | 2 (2%) |  |
| ER− / PR− | 629 (32%) | 341 (33%) | 267 (31%) | 29 (19%) | 62 (53%) |  |
| ER unknown / PR unknown | 2 (<1%) | - | - | - | - |  |
| HER2 Status | Positive | 1505 (78%) | 790 (77%) | 660 (77%) | 120 (78%) | 88 (75%) |
| Negative | 361 (19%) | 195 (19%) | 163 (19%) | 27 (18%) | 26 (22%) |  |
| Unknown | 74 (4%) | 38 (4%) | 32 (4%) | 7 (5%) | 3 (3%) |  |
| Assigned Numberof Cycles | Four | 1151 (59%) | 572 (56%) | 471 (55%) | 139(90%)e | 75 (64%) |
| Six | 789 (41%) | 451 (44%) | 384 (45%) | 15 (10%) | 42 (36%) |  |

Table 1 Caption: Patient Demographics

### Genotyping and Quality Control

A summary of the steps included in sample and single nucleotide polymorphism (SNP) quality control and in principal components analysis (PCA) is illustrated in [Supplemental Figure 1](#SD1). A total of 1,040 paclitaxel-treated patients with informed consent and a DNA sample (obtained from peripheral blood) available as of July 1, 2009 were included in the primary study. Genomic DNA was genotyped using the HumanHap610-Quad Genotyping BeadChip (Illumina, CA, USA) which interrogated 592,532 SNPs. Subjects with call rates <0.98 (n=5) or with suboptimal genotype clustering performance (n=1) were excluded followed by reassessment of genotypes within the remaining subjects. SNPs with call rates <0.95, poor genotype clustering performance, >1 replicate or Mendelian discordance, relative minor allele frequency (MAF) < 0.005, non-diploid (e.g. Y or mitochondrial chromosomes) or deemed unreliable by Illumina (n=4,106; Tech Note: Infinium® Genotyping Data Analysis, 2007) were excluded, leaving 572,745 SNPs. Identity-by-descent (IBD) analysis verified the absence of closely related individuals (proportion IBD > 0.15) and identified one unintended duplicate pair which was removed and later confirmed to be due to a DNA plating error (PLINK version 1.07) ([7](#R7)). Evaluation of X-chromosome heterozygosity identified three genetic males which were also removed and similarly confirmed to be due to a DNA plating error ([8](#R8)). PCA as implemented by EIGENSOFT version 3.0 was used to visualize the genetic ancestry of the 1,029 individuals passing QC ([9](#R9)). PCA was performed using genotypes from study subjects combined with genotypes of unrelated individuals from the HapMap Project representing Northwest European (CEU, n=73), African (YRI, n=77) and Chinese (CHB, n=75) ancestries and genotyped using the same platform by Illumina ([Supplemental Figure 2](#SD1)) ([10](#R10)). To address the potential bias arising from population stratification, we chose to focus our primary analysis on individuals of Northern European descent. A second PCA was performed using only the 1,029 study subjects. Mean values for the first three eigenvectors within all patients self-declaring “White” race and “Non-Hispanic” ethnicity were determined. “Genetic Northwest Europeans” (herein called Europeans) were defined as individuals with each of their first three eigenvectors within two standard deviations of each mean value irrespective of self-declared race and ethnicity. A total of 859 individuals were identified and identical results were obtained when repeated with the inclusion of HapMap individuals (not shown). These 859 individuals were the focus of the primary analysis ([Supplemental Figure 2](#SD1)).

Imputation of genotypes was performed within the 859 Europeans using MACH 1.0 ([11](#R11)) and reference haplotypes from unrelated CEU individuals from either HapMap (r22) or the 1000 Genomes Project (June 2010 release). Prior to imputation, study genotypes were more stringently filtered and limited to autosomal SNPs with MAF ≥ 0.01 and exact Hardy-Weinberg *P*-values ≥ 0.001 in control subjects. To address any potential stranding inconsistencies between study genotypes and the reference haplotypes, all symmetric SNPs (A/T or C/G) with MAF > 0.40, and therefore difficult to resolve, were removed leaving 548,596 and 547,465 SNPs for imputation using the HapMap and 1000 Genomes reference haplotypes, respectively. Imputed SNPs with MAF < 0.01 or R^2^ < 0.5 were excluded. Genotyping within the replication cohorts (described below) was performed using TaqMan^®^ Allelic Discrimination assays (Applied Biosystems) and individual assays are shown in [Supplemental Tables 1 and 2](#SD1).

One hundred fifty nine self-declared “White” individuals with either “Non-Hispanic” or “Unknown” ethnicity who enrolled in the CALGB 40101 pharmacogenetic companion study subsequent to the genotyping of the original 1,040 subjects were used as a replication cohort. Within the discovery set, these criteria accurately identified 98.7% of the 859 Europeans with a false-positive rate of 2.4%. An additional 100 individuals of African ancestry were also identified from within the group of 1,029 individuals passing sample quality control. African ancestry was defined using individuals who self-declared “Black/African American” race with either “Non-Hispanic” or “Unknown” ethnicity. Any individual with their first three eigenvectors within three standard deviations of each eigenvector mean value were considered to be of African descent. These self declared race/ethnicity criteria identified 94.2% of individuals with African ancestry and incorrectly identified 2.0%. The final African American replication cohort consisted of the 100 patients of African descent with genome-wide data and an additional 20 self-declared “Black/African American” individuals with either “Non-Hispanic” or “Unknown” ethnicity who enrolled after the original genotyping.

### Statistical Analysis

The primary objective was the identification of SNPs associated with the occurrence of sensory peripheral neuropathy. The analyses were carried out using two complementary endpoints: 1) the cumulative dose level triggering the first grade 2 or higher treatment related sensory peripheral neuropathy episode and 2) the maximum observed treatment-related sensory peripheral neuropathy grade. The adverse events were graded according to the National Cancer Institute Common Toxicity Criteria for Adverse Events (NCI-CTCAE) version 2.0. The timing of sensory peripheral neuropathy was assessed with a time to event approach where an event was defined as the first incidence of a grade 2 or higher neuropathy and time as the cumulative paclitaxel exposure (mg/m^2^). For patients not experiencing any event, the total study paclitaxel drug exposure was used. These patients are effectively right-censored at the cumulative dose level. The marginal associations were tested using the Cox score test ([12](#R12)). The severity of sensory peripheral neuropathy, defined as the maximum grade neuropathy observed during paclitaxel treatment or within 30 days following the last dose, was evaluated using ordinal logistic regression. Cumulative dose (mg/m^2^) was log transformed and incorporated into the ordinal regression model. For both cases, the marginal null sampling distribution was approximated using asymptotics. These analyses were powered for an additive genetic model. To minimize Type I error due to sparseness, SNPs within the European discovery set were constrained to relative MAF ≥ 1% and the observation of a minimum of two minor allele homozygous genotypes leaving 521,600 evaluable SNPs. Imputed genotypes were represented as allele dosages bound between 0.0 and 2.0. All analyses were performed using the *R* statistical environment version 2.12 with the cumulative dose to event and ordinal analyses implemented using functions from the survival and MASS extension packages ([13](#R13)–[16](#R16)). Quantile-quantile plots of the marginal asymptotic *P*-values were evaluated for potential remaining population stratification or inflation of significance levels. Each SNP with a marginal *P*-value ≤ 10^−5^ was evaluated further for potential errors by checking its MAF (vs. HapMap), Hardy Weinberg Equilibrium (HWE) within unaffected subjects, and potentially informative missing rates; they were also visually inspected for genotype clustering performance.

Based on the combined results of the time to event and ordinal regression analyses of the 859 European patients, a replication plan delineating SNPs, regression model, genetic model (the most plausible model suggested from Kaplan-Meier estimates) and effect direction for one-sided testing was drafted *a priori* to any data collection within the replication cohorts. Three SNPs from the genes *FZD3, EPHA5* and *FGD4* (rs7001034, rs7349683, and rs10771973) were selected for replication based on marginal significance levels, biological plausibility and estimated effect size (as detailed in the Results). An additional ten SNPs with *P*-values < 10^−5^ and/or previously implicated in congenital sensory peripheral neuropathies (*NDRG1*) were also evaluated with the specified limitation of being constrained to exploratory analyses. Genotypes for the *FZD3* SNP rs7001034 were captured indirectly using a proxy SNP (rs7833751; R^2^ = 1.0 CEU HapMap r27) due to the absence of acceptable TaqMan assays to evaluate the locus directly. Due to the impracticality of capturing the *FZD3* linkage disequilibrium (LD) block to the same extent as the European group, this locus was not evaluated in the African ancestry replication group. Direct sequencing was used to capture the *FGD4* rs10771973 genotypes within the replication cohorts. To limit the overall Type I error rate for the validation study at the one-sided 0.05 level, we tested each of the three SNPs at the marginal 0.01 level. Since the *FGD4* locus replicated in both populations and there are significant differences in LD structure between the European and African American populations, an additional four coding region SNPs were chosen from the ~30 kb LD block containing rs10771973 to further extend this finding. In addition, to evaluate the independence of the identified association in rs10771973, the time to event analysis was repeated with rs10771973 as a covariate. This analysis was conducted using the R extension package GenABEL ([13](#R13)). A haplotype based association test was also conducted for the three genes containing the top hits (*EPHA5*, *FZD3* and *FGD4*), using all genotyped SNPs within 100 kb of the transcription start and stop sites for each gene. Phase for each SNP set was estimated using fastPHASE v 1.1 in all samples combined ([17](#R17)). Haplotype block boundaries using the method of Gabriel *et al.* were generated in Haploview v4.2 using HapMap v3 r2 CEU samples ([18](#R18)). For each haplotype block that included an allele with a per SNP association signal of < 10^−3^, individual haplotypes were extracted from fastPHASE output, and haplotypes with frequency less than 5% were combined. Association with outcome was analyzed on a per haplotype basis using time to event or maximum grade as described above.

## Results

Of the 859 individuals with European ancestry randomly assigned to paclitaxel treatment, four withdrew prior to study treatment and were therefore excluded ([Supplemental Figure 1](#SD1)). Patient characteristics of the CALGB 40101 paclitaxel treatment arm, the genotyped samples, and the discovery and replication cohorts are listed in [Table 1](#T1). The menopausal, ER, PgR and HER2 status, and the assigned number of cycles were not different between the genotyped paclitaxel cohort and the European discovery cohort. The genotyped sample was also representative of all patients randomized to paclitaxel treatment in CALGB 40101. One exception is a fewer number of samples from the six cycle paclitaxel arm in the European replication cohort, which reflects the early closure of the six cycle arm and the later study enrollment of this group of patients. Peripheral sensory neuropathy was the major dose limiting toxicity in the paclitaxel arm and the distribution of toxicity grades within the 855 patients in the primary analysis, stratified for number of treatment cycles assigned, is shown in [Table 2](#T2). Sensory peripheral neuropathy was dose dependent with 17% of the patients randomized to four cycles of paclitaxel experiencing a grade 2 or greater event as compared to 33% of those randomized to six cycles of treatment. The cumulative incidence of sensory peripheral neuropathy was similar between the entire cohort randomized to paclitaxel treatment and the discovery set ([Supplemental Figure 3](#SD1)), and between the discovery set and both replication groups (data not shown). There was no effect of age on cumulative dose triggering a grade 2 or greater peripheral neuropathy event (data not shown).

### Table 2.

|   |  | Sensory Peripheral Neuropathy Grade |  |  |  |  | EventRateb |
| --- | --- | --- | --- | --- | --- | --- | --- |
| 0 | 1 | 2 | 3 | 4 |  |  |  |
| Discovery Set |  |   |  |  |  |  |  |
| 4 Cycles |  | 181 (38%)a | 209 (44%) | 66 (14%) | 15 (3%) | 0 (0%) | 17% |
| 6 Cycles |  | 99 (26%) | 160 (42%) | 81 (21%) | 44 (11%) | 0 (0%) | 33% |
| European Replication |  |   |  |  |  |  |  |
| 4 Cycles |  | 44 (32%) | 67 (48%) | 24 (17%) | 4 (3%) | 0 (0%) | 20% |
| 6 Cycles |  | 6 (40%) | 4 (27%) | 3 (20%) | 2 (13%) | 0 (0%) | 33% |
| African American Replication |  |   |  |  |  |  |  |
| 4 Cycles |  | 23 (31%) | 33 (44%) | 12 (16%) | 7 (9%) | 0 (0%) | 25% |
| 6 Cycles |  | 9 (21%) | 19 (45%) | 7 (17%) | 6 (14%) | 1 (2%) | 33% |

Table 2 Caption: Incidence of Sensory Peripheral Neuropathies in Study Groups

Among the SNPs analyzed in the GWAS for association with the initial onset of sensory peripheral neuropathy, none reached genome-wide significance although seven had a marginal significance level of *P* < 10^−5^ ([Table 3](#T3) and [Supplemental Figure 4](#SD1)). Inspection of the quantile-quantile plot of the marginal *P*-values ([Supplemental Figure 5A](#SD1)) indicates the absence of any remaining population substructure (λ = 1.01). Of these top SNPs, biological relevance was apparent for polymorphisms in *EPHA5* (rs7349683; per allele HR, 1.63; 95% CI, 1.34 – 1.98; *P* = 9.6 × 10^7^; [Figure 1A](#F1)) and *FGD4* (rs10771973; per allele HR, 1.57; 95% CI, 1.30 – 1.91; *P* = 2.6 × 10^6^; [Figure 1B](#F1)). *EPHA5* encodes an ephrin receptor gene implicated in the process of neuronal regeneration following nerve injury and *FGD4* encodes a Rho-GTPase guanine nucleotide exchange factor previously implicated in congenital peripheral neuropathies ([19](#R19)–[22](#R22)). The *FGD4* ([Table 3](#T3); [Supplemental Figures 6A and 6B](#SD1)) and *EPHA5* ([Table 3](#T3); [Supplemental Figures 7A and 7B](#SD1)) SNPs were tested in replication cohorts and association for the former was confirmed in both the European and African American samples (Europeans: rs10771973; per allele HR, 1.72; 95% CI, 1.06 – 2.80; *P* = 0.013; African Americans: rs10771973; per allele HR, 1.93; 95% CI, 1.13 – 3.28; *P* = 6.7 × 10^−3^). Considering the high minor allele frequency of this risk allele in Europeans, 42% of patients are expected to have a 1.6-fold increased risk and 9% a 2.6-fold increased risk of peripheral neuropathy; in African Americans (MAF 17%) the increased risk is 1.9- and 3.7-fold, respectively. Inspection of the Kaplan-Meier genotype stratified time to neuropathy distributions suggests that an allele dose effect assumption for *FGD4* rs10771973 is appropriate ([Figure 1B](#F1)).

### Table 3.

| SNP | Chr | Genea | Alleles | Discovery(n=855) |  |  | EuropeanReplication (n=154)d |  |  | African AmericanReplication (n=117) |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| MAF b | HR(95% CI) | Pc | MAF | HR(95% CI) | P | MAF | HR(95% CI) | P |  |  |  |  |
| rs7349683 | 4 | EPHA5 | C/T | 0.36 | 1.63(1.34–1.98) | 9.6 × 10−7 | 0.32 | 0.96(0.57–1.60) | 0.43 | 0.13 | 1.16(0.55–2.42) | 0.35 |
| rs4737264 | 8 | XKR4 | A/C | 0.22 | 1.68(1.36–2.09) | 1.9×10−6 | 0.24 | 1.84(1.02–3.33) | 0.021 | 0.18 | 1.23(0.69–2.21) | 0.24 |
| rs10771973 | 12 | FGD4 | G/A | 0.31 | 1.57(1.30–1.91) | 2.6 × 10−6 | 0.33 | 1.72(1.06–2.80) | 0.013 | 0.17 | 1.93(1.13–3.28) | 6.7 × 10−3 |
| rs16948748 | 17 | PITPNA | T/G | 0.04 | 2.37(1.63–3.44) | 2.7 × 10−6 | 0.02 | 2.65e(0.63–11.1) | 0.083 | 0.07 | 1.07e(0.41–2.77) | 0.45 |
| rs16916932 | 10 | CACNB2 | C/T | 0.06 | 2.08(1.51–2.87) | 4.3 × 10−6 | 0.06 | 0.38e(0.09–1.58) | 0.082 | 0.08 | 1.13e(0.47–2.74) | 0.39 |
| rs17781082 | 12 | GRIP1/CAND1 | C/T | 0.42 | 1.60(1.31–1.96) | 4.3 × 10−6 | 0.43 | 1.22(0.74–1.99) | 0.22 | 0.21 | 1.32(0.76–2.30) | 0.16 |
| rs1903216 | 3 | BCL6/ | G/A | 0.48 | 1.59(1.30–1.95) | 5.6 × 10−6 | 0.41 | 2.08f(0.99–4.37) | 0.024 | 0.03 | 3.02f(1.04–8.73) | 0.016 |
| rs2233335 | 8 | NDRG1 | T/G | 0.38 | 0.65(0.52–0.80) | 5.2 × 10−5 | 0.39 | 0.94(0.58–1.53) | 0.41 | 0.23 | 1.40(075–2.60) | 0.14 |

Table 3 Caption: Top SNPs from Cumulative Dose to Event Analysis

### Figure 1. The EPHA5 rs7349683 C>T and FGD4 rs10771973 G>A polymorphisms are associated with an increased probability of developing paclitaxel-induced Grade 2 or greater sensory peripheral neuropathy.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f72a/3445665/669425b3c9b0/nihms396061f1.jpg)

The probability of the first instance of Grade 2 or greater neuropathy as a function of cumulative paclitaxel dose (corrected for body surface area) is shown for each genotype. Results are shown for A) rs7239683 (per allele HR = 1.63; 95% CI 1.34 – 1.98; P = 9.6 × 10−7) and B) rs10771973 (per allele HR, 1.57; 95% CI, 1.30 – 1.91; P = 2.6 × 10−6) in the discovery set. The number of individuals with each genotype is noted in parentheses.

No haplotypes in *FGD4* or *EPHA5* showed stronger association with time to sensory peripheral neuropathy than the single SNP analyses in these regions (data not shown). After conditioning the time to event analysis on rs10771973, no other genotyped markers at the *FGD4* locus showed association with time to peripheral neuropathy (data not shown). Using imputation to infer additional untyped markers and visualizing the linkage disequilibrium (LD) structure within the HapMap CEU population revealed a ~30 kb region of high LD within the *FGD4* locus showing a strong and reproducible association with the onset of sensory peripheral neuropathy ([Supplemental Figure 8](#SD1)). Approximately 16 SNPs are strongly linked (R^2^ ≥ 0.80) with rs10771973, five of which are synonymous variants within the coding region.

Ordinal logistic regression analyses were used to identify SNPs associated with the severity of sensory peripheral neuropathy. Four SNPs were associated with toxicity grade with a significance level of *P* < 1 × 10^−5^ ([Table 4](#T4) and [Supplemental Figure 4](#SD1)). As with the Cox analysis, a quantile-quantile plot of the normalized marginal *P*-values ([Supplemental Figure 5B](#SD1)) suggests the absence of any remaining population substructure (λ = 0.986). A SNP within the Frizzled 3 homolog *WNT* signaling receptor gene *(FZD3)* met the threshold of genome-wide significance (rs7001034; *P* = 3.1 × 10^−9^; OR, 0.57; 95% CI, 0.48 – 0.69) and demonstrated a clear relationship between allele dosage and sensory peripheral neuropathy grade ([Figure 2](#F2)). However, none of these top SNPs from the ordinal regression analysis replicated in either the European or African American populations ([Table 4](#T4)).

### Table 4.

| SNP | Chr | Genea | Alleles | Discovery(n=855) |  |  | EuropeanReplication (n=154)d |  |  | African AmericanReplication (n=117)d |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| MAFb | OR(95% CI) | Pc | MAF | OR(95% CI) | P | MAF | OR(95% CI) | P |  |  |  |  |
| rs7001034e | 8 | FZD3 | G/T | 0.40 | 0.57(0.48–0.69) | 3.1 ×10−9 |   |   |   |   |   |   |
| rs7833751f | 8 | FZD3 | G/T | 0.40 | 0.58(0.49–0.70) | 7.5 × 10−9 | 0.41 | 1.67f(0.88–3.19) | 0.058 |   |   |   |
| rs5934683 | X | /SHROOM2 | C/T | 0.36 | 1.61(1.33–1.93) | 6.0 × 10−7 | 0.33 | 0.99g(0.54–1.82) | 0.49 | 0.30 | 1.96g(0.49–7.89) | 0.17 |
| rs2941627 | 8 | /ZFPM2 | A/G | 0.13 | 1.91(1.40–2.51) | 3.5 × 10−6 | 0.15 | 1.37g(0.68–2.73) | 0.19 | 0.25 | 0.49g(0.24–0.97) | 0.021 |
| rs7973533 | 12 | /BCAT1 | T/G | 0.46 | 0.66(0.55–0.79) | 8.4 × 10−6 | 0.44 | 0.75(0.47–1.19) | 0.11 | 0.27 | 1.13(0.65–1.94) | 0.33 |

Table 4 Caption: Top SNPs from Ordinal Analysis

### Figure 2. Association of FZD3 SNP rs7001034 with sensory peripheral neuropathy.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f72a/3445665/c4a722858e25/nihms396061f2.jpg)

The minor allele frequency of rs7001034 in the European discovery cohort is expressed as a function of maximal grade of sensory peripheral neuropathy in 855 individuals.

## Discussion

A small subset of patients exposed to paclitaxel have significant and occasionally protracted neuropathy that has a major impact on quality of life. If we could prospectively identify these patients prior to administration of paclitaxel, they might be otherwise equally well served with alternative non-paclitaxel containing regimens. Using a genome-wide association study of CALGB 40101, we have identified several genetic loci associated with onset or severity of paclitaxel-induced sensory peripheral neuropathy. One of these novel markers associated with early-onset paclitaxel-induced sensory peripheral neuropathy (*FGD4*, rs10771973) was replicated in both Europeans and African Americans and resides within a gene with a clearly established role in the hereditary peripheral neuropathy Charcot-Marie-Tooth disease (CMT). These findings will inform studies to test the application of genetic markers for optimization of paclitaxel selection, dosing and adverse event management. Several features of the study design and analysis support the robustness of our findings, including the prospective design, a large cohort of patients with primary breast cancer who are chemotherapy naïve and treated with single agent paclitaxel, careful collection of sensory peripheral neuropathy and covariate data, strict censoring for dose and cycle reductions for other adverse reactions and preexisting neuropathy, and the use of cumulative dose to the initial incidence of Grade 2 toxicity to account for the established effect of total drug exposure on sensory peripheral neuropathy.

The current finding that *FGD4* plays a role in the development of paclitaxel-induced sensory peripheral neuropathy and/or the repair response of peripheral nerves following paclitaxel injury is consistent with the known functions of the gene. *FGD4* encodes for the protein FGD1-related F-actin binding protein (Frabin) and previous studies have shown specific point mutations in *FGD4* can cause the congenital peripheral neuropathy Charcot-Marie-Tooth disease (CMT4H) ([21](#R21)–[24](#R24)). The disease is characterized by a slow progressive demyelination of peripheral sensory and motor neurons accompanied by distal muscle weakness and atrophy, sensory loss, hyporeflexia and skeletal deformity ([25](#R25)). Paclitaxel-induced peripheral neuropathy shares some of these characteristics, including sensory loss and secondary demyelination ([26](#R26)–[28](#R28)). Frabin is a guanine nucleotide exchange factor for cdc42, a Rho-GTPase that regulates cellular morphogenesis, including myelination. Several hypotheses have been proposed to explain how mutations in *FGD4* might lead to demyelinating CMT4H disease, including disruption of the actin/microtubule cytoskeleton, loss of c-Jun-NH-terminal kinase (JNK) activation signals, and disruption of phosphoinositide signaling pathways, all of which could affect Schwann cell myelination and/or the bidirectional communication between Schwann cells and axons ([21](#R21)).

The observed association between the *FGD4* SNP rs10771973 and paclitaxel-induced sensory peripheral neuropathy is consistent with the hypothesis that common *FGD4* polymorphisms subtly affect the development and/or maintenance of Schwann cell function. In this case, carriers of common *FGD4* polymorphisms would have pre-existing subclinical abnormalities and a predisposition for toxicity. This is supported by increased risk for paclitaxel-induced sensory peripheral neuropathy in asymptomatic patients with diabetes, previous platinum drug exposures and alcohol use ([3](#R3)) and early Schwann cell activation in response to paclitaxel administration ([29](#R29)). Alternatively, *FGD4* polymorphisms could lead to impaired repair processes such as Schwann cell remyelination and/or axonal regeneration after paclitaxel exposure. Genetic variation in *FGD4* could also directly affect the response of Schwann cells to axonal injury via its ability to activate JNK ([30](#R30)). A neuronal protective role for activated JNK in cultured dorsal root ganglion cells exposed to oxaliplatin has been reported ([31](#R31)). Whether changes in frabin activity or expression lead to a decreased neuronal regenerative capacity and/or an increased sensitivity to paclitaxel-induced sensory peripheral neuropathy requires further study. Interestingly, *FGD4* was identified through a genome-wide siRNA screen in lung cancer cell lines as a paclitaxel chemosensitizer. The chemosensitizing properties of *FGD4* are related at least in part to its ability to prevent mitotic progression ([32](#R32)). Whether a similar mechanism is involved in the repair response to paclitaxel-induced peripheral neuropathy is unknown.

The *FGD4* rs10771973 SNP is located in the intronic region and is in tight LD with a number of other SNPs. Computational analysis of the genomic region surrounding this SNP found that rs10771972, another intronic SNP in high LD with rs10771973 in both the European and African populations, is predicted to alter conserved transcription factor binding sites for Myc-Max and USF (data not shown). One could speculate that disruption of either one or both of these transcription factor binding sites in patients carrying the rs10771973 SNP could lead to altered expression and therefore function of FGD4/Frabin.

The other two top hits from the genome-wide analysis are also of potential interest for the paclitaxel-induced sensory peripheral neuropathy phenotype. In the time to toxicity analysis, the most significant SNP was in *EPHA5*, which encodes for an ephrin receptor involved in axonal guidance and regeneration following injury. Recent studies have shown that in mice EphA5 mRNA is rapidly upregulated in response to a sciatic nerve lesion ([20](#R20)), and that EphA5 signaling during synaptogenesis is transduced via cdc42 ([19](#R19)), the Rho-GTPase involved in Frabin signaling. A common SNP in *FZD3* reached genome-wide significance in the ordinal analysis. *FZD3* encodes a Wnt receptor with reported roles in neurite outgrowth ([33](#R33)). In light of the biological relevance of *EPHA5* and *FZD3* and the limited size of the replication cohorts available for these studies, it will be necessary to further explore the role of these two genes in larger populations of paclitaxel-treated patients. Additional studies are also warranted for other top hits, including rs2233335 in the N-myc Downstream-Regulated Gene 1 (*NDRG1*; [Supplemental Table 1](#SD1)). Rare mutations in *NDRG1* are also associated with a different subtype of CMT (CMT4D) ([34](#R34)).

Until the availability of genome-wide approaches for identifying genetic predictors of paclitaxel-induced peripheral neuropathy, candidate gene approaches focused mostly on drug metabolizing enzymes and transporters implicated in paclitaxel exposure. These candidate gene studies yielded no replicated associations of SNPs with paclitaxel-induced sensory peripheral neuropathy, and most were complicated by a very small number of subjects, a retrospective analysis of toxicity, and chemotherapy with multiple agents ([35](#R35)–[38](#R38)). In the current analysis, no significant associations were observed for any SNPs residing in candidate genes known to influence paclitaxel exposure ([Supplemental Table 3](#SD1)), providing further evidence that factors contributing to the function and repair of peripheral nerves are more important than alterations in paclitaxel pharmacokinetics for determining genetic susceptibility to this toxicity. Interestingly, recent analyses of peripheral neuropathy induced by treatment with bortezomib, thalidomide and vincristine have provided evidence that genes involved in repair mechanisms, inflammation, peripheral nervous system development and mitochondrial dysfunction could influence an individual patient’s risk of developing toxicity ([39](#R39)–[42](#R42)). However there was no overlap of implicated genes with the current study ([Supplemental Table 3](#SD1)), suggesting that the mechanisms underlying this common toxicity might be drug specific.

To assess the potential translational implications of this finding to clinical practice, we estimated the cumulative dose level triggering an event for each *FGD4* rs10771973 genotype. Considering the data in [Figure 1B](#F1), to control the probability of experiencing a neuropathy event at a critical threshold of 33%, the tolerated cumulative dose level for patients with two copies of the risk allele is 710 mg/m^2^. The corresponding expected critical dose level for patients with one copy of the risk allele is increased to 877 mg/m^2^. Patients with no copies of the risk allele are expected to tolerate >1047 mg/m^2^, corresponding to the full dose of paclitaxel for six cycles. If these thresholds are prospectively validated and further refined in follow-up studies, they may be used to estimate tolerable dose levels based on genotype and to tailor the treatment regimen.

While this pharmacogenetic study has several advantages over previous studies on paclitaxel pharmacogenetics, including a large cohort of treatment naïve patients receiving single agent paclitaxel and a genome-wide approach to discovery, several limitations also exist. The most significant limitation is the sole use of the NCI-CTC for assessment of sensory peripheral neuropathy. It is widely recognized that detailed patient-reported symptom data and a quality of life assessment more accurately describes this phenotype and that physician reported NCI-CTC grading underreports peripheral neuropathy ([43](#R43)–[45](#R45)). However, it remains difficult to apply these techniques across the multiple sites and large sample sizes required for sufficient power for pharmacogenetic analyses. In a recent Phase III study of 1,060 women treated with taxanes, the Patient Neurotoxicity Questionnaire and the Functional Assessment of Cancer Therapy-General were administered to only the first 300 patients in the study ([46](#R46)). The only use of patient-reported toxicity data and symptom measurements for pharmacogenetic analysis of taxane peripheral neuropathy is limited by the very small sample size of the study ([38](#R38)). While it will be important in follow-up studies to validate these findings using additional instruments, it should be noted that despite its limitations the NCI-CTC scores are widely accepted for primary evaluation of treatment toxicity in large Phase III studies such as CALGB 40101. A second limitation of the current study is the small sample size of the replication cohorts, a common issue confronting almost all pharmacogenetic studies ([47](#R47)).

In summary, our findings support the use of prospective pharmacogenetic analyses of well phenotyped data sets collected under controlled clinical trial settings and unbiased genome-wide genetic approaches for the identification of novel genes involved in drug efficacy and toxicity. Using a prospective design for validation and replication and a well-controlled single agent clinical study we have identified a SNP in *FGD4* associated with increased risk of developing paclitaxel-induced sensory peripheral neuropathy. The involvement of *FGD4* in Charcot-Marie Tooth disease, a congenital peripheral neuropathy, provides strong evidence for the biological significance of this finding. The fact that a common *FGD4* SNP is associated with an increased risk of paclitaxel-induced sensory peripheral neuropathy in patients with both European and African ancestry makes it of potentially broad clinical significance. Additional SNPs in *EPHA5* and *FZD3* were also identified as potential risk factors for the onset and severity of sensory peripheral neuropathy. Additional samples for extension and validation of these findings are currently being collected in ongoing CALGB clinical trials of paclitaxel in the setting of metastatic breast cancer.

## Supplementary Material

Paclitaxel is widely used in the treatment of many cancers, including breast cancer. Treatment with paclitaxel is often limited by the development of peripheral neuropathies which can significantly impact a patient’s quality of life. Biomarkers for the prediction of paclitaxel-induced peripheral neuropathy could be used to optimize the use of paclitaxel. A genome-wide genotyping approach in women receiving single agent paclitaxel as adjuvant therapy for breast cancer identified several novel genetic loci implicated in paclitaxel-induced sensory peripheral neuropathy. In particular, a common genetic variant in *FGD4*, a causal gene for the congenital peripheral neuropathy Charcot-Marie-Tooth Disease, was associated with increased onset of neuropathy in both Europeans and African Americans. This variant and others identified in these studies could be validated as genetic predictors of paclitaxel-induced sensory peripheral neuropathy. The genetic variants identified in these studies will also lead to investigations into novel pathways for this common chemotherapy-induced toxicity.

Paclitaxel is widely used in the treatment of many cancers, including breast cancer. Treatment with paclitaxel is often limited by the development of peripheral neuropathies which can significantly impact a patient’s quality of life. Biomarkers for the prediction of paclitaxel-induced peripheral neuropathy could be used to optimize the use of paclitaxel. A genome-wide genotyping approach in women receiving single agent paclitaxel as adjuvant therapy for breast cancer identified several novel genetic loci implicated in paclitaxel-induced sensory peripheral neuropathy. In particular, a common genetic variant in *FGD4*, a causal gene for the congenital peripheral neuropathy Charcot-Marie-Tooth Disease, was associated with increased onset of neuropathy in both Europeans and African Americans. This variant and others identified in these studies could be validated as genetic predictors of paclitaxel-induced sensory peripheral neuropathy. The genetic variants identified in these studies will also lead to investigations into novel pathways for this common chemotherapy-induced toxicity.

## Acknowledgments

**Grant Support**

The research for CALGB 60202 and 40101 was supported, in part, by grants from the National Cancer Institute (CA31946) to the Cancer and Leukemia Group B (Monica M. Bertagnolli, M.D., Chair) and to the CALGB Statistical Center (Daniel J. Sargent, Ph.D., CA33601). The content of this manuscript is solely the responsibility of the authors and does not necessarily represent the official views of the National Cancer Institute. This work was also supported in part by NIH grants GM61390 and GM61393, and the Biobank Japan Project funded by the Japanese Ministry of Education, Culture, Sports, Science and Technology. This work is part of the NIH Pharmacogenomics Research Network-RIKEN Center for Genomic Medicine Global Alliance. Aparna Chhibber and Rachel J. Eclov were supported in part by NIH Training Grant T32 GM007175.

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Bishop JF, Dewar J, Toner GC, Smith J, Tattersall MH, Olver IN, et al. Initial paclitaxel improves outcome compared with cmfp combination chemotherapy as front-line therapy in untreated metastatic breast cancer. J Clin Oncol. 1999;17:2355–2364. doi: 10.1200/JCO.1999.17.8.2355.  [DOI](https://doi.org/10.1200/JCO.1999.17.8.2355) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10561297/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Initial%20paclitaxel%20improves%20outcome%20compared%20with%20cmfp%20combination%20chemotherapy%20as%20front-line%20therapy%20in%20untreated%20metastatic%20breast%20cancer&author=JF%20Bishop&author=J%20Dewar&author=GC%20Toner&author=J%20Smith&author=MH%20Tattersall&volume=17&publication_year=1999&pages=2355-2364&pmid=10561297&doi=10.1200/JCO.1999.17.8.2355&)

2. Buzdar AU, Singletary SE, Theriault RL, Booser DJ, Valero V, Ibrahim N, et al. Prospective evaluation of paclitaxel versus combination chemotherapy with fluorouracil, doxorubicin, and cyclophosphamide as neoadjuvant therapy in patients with operable breast cancer. J Clin Oncol. 1999;17:3412–3417. doi: 10.1200/JCO.1999.17.11.3412.  [DOI](https://doi.org/10.1200/JCO.1999.17.11.3412) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10550135/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Prospective%20evaluation%20of%20paclitaxel%20versus%20combination%20chemotherapy%20with%20fluorouracil,%20doxorubicin,%20and%20cyclophosphamide%20as%20neoadjuvant%20therapy%20in%20patients%20with%20operable%20breast%20cancer&author=AU%20Buzdar&author=SE%20Singletary&author=RL%20Theriault&author=DJ%20Booser&author=V%20Valero&volume=17&publication_year=1999&pages=3412-3417&pmid=10550135&doi=10.1200/JCO.1999.17.11.3412&)

3. Lee JJ, Swain SM. Peripheral neuropathy induced by microtubule-stabilizing agents. J Clin Oncol. 2006;24:1633–1642. doi: 10.1200/JCO.2005.04.0543.  [DOI](https://doi.org/10.1200/JCO.2005.04.0543) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16575015/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Peripheral%20neuropathy%20induced%20by%20microtubule-stabilizing%20agents&author=JJ%20Lee&author=SM%20Swain&volume=24&publication_year=2006&pages=1633-1642&pmid=16575015&doi=10.1200/JCO.2005.04.0543&)

4. Rowinsky EK, Donehower RC. Paclitaxel (taxol) N Engl J Med. 1995;332:1004–1014. doi: 10.1056/NEJM199504133321507.  [DOI](https://doi.org/10.1056/NEJM199504133321507) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7885406/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Paclitaxel%20(taxol)&author=EK%20Rowinsky&author=RC%20Donehower&volume=332&publication_year=1995&pages=1004-1014&pmid=7885406&doi=10.1056/NEJM199504133321507&)

5. Hershman DL, Weimer LH, Wang A, Kranwinkel G, Brafman L, Fuentes D, et al. Association between patient reported outcomes and quantitative sensory tests for measuring long-term neurotoxicity in breast cancer survivors treated with adjuvant paclitaxel chemotherapy. Breast Cancer Res Treat. 2011;125:767–774. doi: 10.1007/s10549-010-1278-0.  [DOI](https://doi.org/10.1007/s10549-010-1278-0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21128110/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Breast%20Cancer%20Res%20Treat&title=Association%20between%20patient%20reported%20outcomes%20and%20quantitative%20sensory%20tests%20for%20measuring%20long-term%20neurotoxicity%20in%20breast%20cancer%20survivors%20treated%20with%20adjuvant%20paclitaxel%20chemotherapy&author=DL%20Hershman&author=LH%20Weimer&author=A%20Wang&author=G%20Kranwinkel&author=L%20Brafman&volume=125&publication_year=2011&pages=767-774&pmid=21128110&doi=10.1007/s10549-010-1278-0&)

6. Shulman LN, Cirrincione CT, Berry DA, Becker HP, Perez EA, O’Regan RO, et al. Six cycles of doxorubicin and cyclophosphamide or paclitaxel are not superior to 4 cycles as adjuvant chemotherapy for breast cancer in women with 0-3 positive axillary nodes: Calgb 40101. J Clin Oncol. 2012 doi: 10.1200/JCO.2011.40.6405. in press.  [DOI](https://doi.org/10.1200/JCO.2011.40.6405) | [PMC free article](/articles/PMC3494835/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22826271/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Six%20cycles%20of%20doxorubicin%20and%20cyclophosphamide%20or%20paclitaxel%20are%20not%20superior%20to%204%20cycles%20as%20adjuvant%20chemotherapy%20for%20breast%20cancer%20in%20women%20with%200-3%20positive%20axillary%20nodes:%20Calgb%2040101&author=LN%20Shulman&author=CT%20Cirrincione&author=DA%20Berry&author=HP%20Becker&author=EA%20Perez&publication_year=2012&pmid=22826271&doi=10.1200/JCO.2011.40.6405&)

7. Purcell S, Neale B, Todd-Brown K, Thomas L, Ferreira MA, Bender D, et al. Plink: A tool set for whole-genome association and population-based linkage analyses. Am J Hum Genet. 2007;81:559–575. doi: 10.1086/519795.  [DOI](https://doi.org/10.1086/519795) | [PMC free article](/articles/PMC1950838/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17701901/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Plink:%20A%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses&author=S%20Purcell&author=B%20Neale&author=K%20Todd-Brown&author=L%20Thomas&author=MA%20Ferreira&volume=81&publication_year=2007&pages=559-575&pmid=17701901&doi=10.1086/519795&)

8. Weale ME. Quality control for genome-wide association studies. Methods Mol Biol. 2010;628:341–372. doi: 10.1007/978-1-60327-367-1_19.  [DOI](https://doi.org/10.1007/978-1-60327-367-1_19) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20238091/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Methods%20Mol%20Biol&title=Quality%20control%20for%20genome-wide%20association%20studies&author=ME%20Weale&volume=628&publication_year=2010&pages=341-372&pmid=20238091&doi=10.1007/978-1-60327-367-1_19&)

9. Price AL, Patterson NJ, Plenge RM, Weinblatt ME, Shadick NA, Reich D. Principal components analysis corrects for stratification in genome-wide association studies. Nat Genet. 2006;38:904–909. doi: 10.1038/ng1847.  [DOI](https://doi.org/10.1038/ng1847) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16862161/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Principal%20components%20analysis%20corrects%20for%20stratification%20in%20genome-wide%20association%20studies&author=AL%20Price&author=NJ%20Patterson&author=RM%20Plenge&author=ME%20Weinblatt&author=NA%20Shadick&volume=38&publication_year=2006&pages=904-909&pmid=16862161&doi=10.1038/ng1847&)

10. Frazer KA, Ballinger DG, Cox DR, Hinds DA, Stuve LL, Gibbs RA, et al. A second generation human haplotype map of over 3.1 million snps. Nature. 2007;449:851–861. doi: 10.1038/nature06258.  [DOI](https://doi.org/10.1038/nature06258) | [PMC free article](/articles/PMC2689609/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17943122/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=A%20second%20generation%20human%20haplotype%20map%20of%20over%203.1%20million%20snps&author=KA%20Frazer&author=DG%20Ballinger&author=DR%20Cox&author=DA%20Hinds&author=LL%20Stuve&volume=449&publication_year=2007&pages=851-861&pmid=17943122&doi=10.1038/nature06258&)

11. Li Y, Willer CJ, Ding J, Scheet P, Abecasis GR. Mach: Using sequence and genotype data to estimate haplotypes and unobserved genotypes. Genet Epidemiol. 2010;34:816–834. doi: 10.1002/gepi.20533.  [DOI](https://doi.org/10.1002/gepi.20533) | [PMC free article](/articles/PMC3175618/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21058334/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genet%20Epidemiol&title=Mach:%20Using%20sequence%20and%20genotype%20data%20to%20estimate%20haplotypes%20and%20unobserved%20genotypes&author=Y%20Li&author=CJ%20Willer&author=J%20Ding&author=P%20Scheet&author=GR%20Abecasis&volume=34&publication_year=2010&pages=816-834&pmid=21058334&doi=10.1002/gepi.20533&)

12. Cox DR. Regression models and life tables. J Royal Stat Soc Series B. 1972;34:187–220.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Royal%20Stat%20Soc%20Series%20B&title=Regression%20models%20and%20life%20tables&author=DR%20Cox&volume=34&publication_year=1972&pages=187-220&)

13. Aulchenko YS, Ripke S, Isaacs A, van Duijn CM. Genabel: An r library for genome-wide association analysis. Bioinformatics. 2007;23:1294–1296. doi: 10.1093/bioinformatics/btm108.  [DOI](https://doi.org/10.1093/bioinformatics/btm108) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17384015/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=Genabel:%20An%20r%20library%20for%20genome-wide%20association%20analysis&author=YS%20Aulchenko&author=S%20Ripke&author=A%20Isaacs&author=CM%20van%20Duijn&volume=23&publication_year=2007&pages=1294-1296&pmid=17384015&doi=10.1093/bioinformatics/btm108&)

14. Team RDC. R: A language and environment for statistical computing. Vienna, Austria: R Foundation for Statistical Computing; 2010.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=R:%20A%20language%20and%20environment%20for%20statistical%20computing&publication_year=2010&)

15. Therneau TM, Grambsch PM. Modeling survival data: Extending the cox model. New York: Springer-Verlag; 2000.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Modeling%20survival%20data:%20Extending%20the%20cox%20model&author=TM%20Therneau&author=PM%20Grambsch&publication_year=2000&)

16. Venables WN, Ripley BD. Modern applied statistics with S. Fourth edition. New York: Springer; 2002.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Modern%20applied%20statistics%20with%20S&author=WN%20Venables&author=BD%20Ripley&publication_year=2002&)

17. Scheet P, Stephens M. A fast and flexible statistical model for large-scale population genotype data: Applications to inferring missing genotypes and haplotypic phase. Am J Hum Genet. 2006;78:629–644. doi: 10.1086/502802.  [DOI](https://doi.org/10.1086/502802) | [PMC free article](/articles/PMC1424677/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16532393/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=A%20fast%20and%20flexible%20statistical%20model%20for%20large-scale%20population%20genotype%20data:%20Applications%20to%20inferring%20missing%20genotypes%20and%20haplotypic%20phase&author=P%20Scheet&author=M%20Stephens&volume=78&publication_year=2006&pages=629-644&pmid=16532393&doi=10.1086/502802&)

18. Gabriel SB, Schaffner SF, Nguyen H, Moore JM, Roy J, Blumenstiel B, et al. The structure of haplotype blocks in the human genome. Science. 2002;296:2225–2229. doi: 10.1126/science.1069424.  [DOI](https://doi.org/10.1126/science.1069424) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12029063/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=The%20structure%20of%20haplotype%20blocks%20in%20the%20human%20genome&author=SB%20Gabriel&author=SF%20Schaffner&author=H%20Nguyen&author=JM%20Moore&author=J%20Roy&volume=296&publication_year=2002&pages=2225-2229&pmid=12029063&doi=10.1126/science.1069424&)

19. Akaneya Y, Sohya K, Kitamura A, Kimura F, Washburn C, Zhou R, et al. Ephrin-a5 and epha5 interaction induces synaptogenesis during early hippocampal development. PLoS One. 2010;5:e12486. doi: 10.1371/journal.pone.0012486.  [DOI](https://doi.org/10.1371/journal.pone.0012486) | [PMC free article](/articles/PMC2930854/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20824214/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Ephrin-a5%20and%20epha5%20interaction%20induces%20synaptogenesis%20during%20early%20hippocampal%20development&author=Y%20Akaneya&author=K%20Sohya&author=A%20Kitamura&author=F%20Kimura&author=C%20Washburn&volume=5&publication_year=2010&pages=e12486&pmid=20824214&doi=10.1371/journal.pone.0012486&)

20. Barrette B, Calvo E, Vallieres N, Lacroix S. Transcriptional profiling of the injured sciatic nerve of mice carrying the wld(s) mutant gene: Identification of genes involved in neuroprotection, neuroinflammation, and nerve regeneration. Brain Behav Immun. 2010;24:1254–1267. doi: 10.1016/j.bbi.2010.07.249.  [DOI](https://doi.org/10.1016/j.bbi.2010.07.249) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20688153/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Brain%20Behav%20Immun&title=Transcriptional%20profiling%20of%20the%20injured%20sciatic%20nerve%20of%20mice%20carrying%20the%20wld(s)%20mutant%20gene:%20Identification%20of%20genes%20involved%20in%20neuroprotection,%20neuroinflammation,%20and%20nerve%20regeneration&author=B%20Barrette&author=E%20Calvo&author=N%20Vallieres&author=S%20Lacroix&volume=24&publication_year=2010&pages=1254-1267&pmid=20688153&doi=10.1016/j.bbi.2010.07.249&)

21. Delague V, Jacquier A, Hamadouche T, Poitelon Y, Baudot C, Boccaccio I, et al. Mutations in fgd4 encoding the rho gdp/gtp exchange factor frabin cause autosomal recessive charcot-marie-tooth type 4h. Am J Hum Genet. 2007;81:1–16. doi: 10.1086/518428.  [DOI](https://doi.org/10.1086/518428) | [PMC free article](/articles/PMC1950914/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17564959/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Mutations%20in%20fgd4%20encoding%20the%20rho%20gdp/gtp%20exchange%20factor%20frabin%20cause%20autosomal%20recessive%20charcot-marie-tooth%20type%204h&author=V%20Delague&author=A%20Jacquier&author=T%20Hamadouche&author=Y%20Poitelon&author=C%20Baudot&volume=81&publication_year=2007&pages=1-16&pmid=17564959&doi=10.1086/518428&)

22. Stendel C, Roos A, Deconinck T, Pereira J, Castagner F, Niemann A, et al. Peripheral nerve demyelination caused by a mutant rho gtpase guanine nucleotide exchange factor, frabin/fgd4. Am J Hum Genet. 2007;81:158–164. doi: 10.1086/518770.  [DOI](https://doi.org/10.1086/518770) | [PMC free article](/articles/PMC1950925/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17564972/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Peripheral%20nerve%20demyelination%20caused%20by%20a%20mutant%20rho%20gtpase%20guanine%20nucleotide%20exchange%20factor,%20frabin/fgd4&author=C%20Stendel&author=A%20Roos&author=T%20Deconinck&author=J%20Pereira&author=F%20Castagner&volume=81&publication_year=2007&pages=158-164&pmid=17564972&doi=10.1086/518770&)

23. Fabrizi GM, Taioli F, Cavallaro T, Ferrari S, Bertolasi L, Casarotto M, et al. Further evidence that mutations in fgd4/frabin cause charcot-marie-tooth disease type 4h. Neurology. 2009;72:1160–1164. doi: 10.1212/01.wnl.0000345373.58618.b6.  [DOI](https://doi.org/10.1212/01.wnl.0000345373.58618.b6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19332693/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neurology&title=Further%20evidence%20that%20mutations%20in%20fgd4/frabin%20cause%20charcot-marie-tooth%20disease%20type%204h&author=GM%20Fabrizi&author=F%20Taioli&author=T%20Cavallaro&author=S%20Ferrari&author=L%20Bertolasi&volume=72&publication_year=2009&pages=1160-1164&pmid=19332693&doi=10.1212/01.wnl.0000345373.58618.b6&)

24. Houlden H, Hammans S, Katifi H, Reilly MM. A novel frabin (fgd4) nonsense mutation p.R275x associated with phenotypic variability in cmt4h. Neurology. 2009;72:617–620. doi: 10.1212/01.wnl.0000342463.35089.cc.  [DOI](https://doi.org/10.1212/01.wnl.0000342463.35089.cc) | [PMC free article](/articles/PMC2677538/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19221294/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neurology&title=A%20novel%20frabin%20(fgd4)%20nonsense%20mutation%20p.R275x%20associated%20with%20phenotypic%20variability%20in%20cmt4h&author=H%20Houlden&author=S%20Hammans&author=H%20Katifi&author=MM%20Reilly&volume=72&publication_year=2009&pages=617-620&pmid=19221294&doi=10.1212/01.wnl.0000342463.35089.cc&)

25. Patzko A, Shy ME. Update on charcot-marie-tooth disease. Curr Neurol Neurosci Rep. 2011;11:78–88. doi: 10.1007/s11910-010-0158-7.  [DOI](https://doi.org/10.1007/s11910-010-0158-7) | [PMC free article](/articles/PMC3685483/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21080241/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Neurol%20Neurosci%20Rep&title=Update%20on%20charcot-marie-tooth%20disease&author=A%20Patzko&author=ME%20Shy&volume=11&publication_year=2011&pages=78-88&pmid=21080241&doi=10.1007/s11910-010-0158-7&)

26. Lipton RB, Apfel SC, Dutcher JP, Rosenberg R, Kaplan J, Berger A, et al. Taxol produces a predominantly sensory neuropathy. Neurology. 1989;39:368–373. doi: 10.1212/wnl.39.3.368.  [DOI](https://doi.org/10.1212/wnl.39.3.368) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2564647/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neurology&title=Taxol%20produces%20a%20predominantly%20sensory%20neuropathy&author=RB%20Lipton&author=SC%20Apfel&author=JP%20Dutcher&author=R%20Rosenberg&author=J%20Kaplan&volume=39&publication_year=1989&pages=368-373&pmid=2564647&doi=10.1212/wnl.39.3.368&)

27. Mielke S, Sparreboom A, Mross K. Peripheral neuropathy: A persisting challenge in paclitaxel-based regimes. Eur J Cancer. 2006;42:24–30. doi: 10.1016/j.ejca.2005.06.030.  [DOI](https://doi.org/10.1016/j.ejca.2005.06.030) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16293411/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Cancer&title=Peripheral%20neuropathy:%20A%20persisting%20challenge%20in%20paclitaxel-based%20regimes&author=S%20Mielke&author=A%20Sparreboom&author=K%20Mross&volume=42&publication_year=2006&pages=24-30&pmid=16293411&doi=10.1016/j.ejca.2005.06.030&)

28. Sahenk Z, Barohn R, New P, Mendell JR. Taxol neuropathy. Electrodiagnostic and sural nerve biopsy findings. Arch Neurol. 1994;51:726–729. doi: 10.1001/archneur.1994.00540190110024.  [DOI](https://doi.org/10.1001/archneur.1994.00540190110024) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7912506/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arch%20Neurol&title=Taxol%20neuropathy.%20Electrodiagnostic%20and%20sural%20nerve%20biopsy%20findings&author=Z%20Sahenk&author=R%20Barohn&author=P%20New&author=JR%20Mendell&volume=51&publication_year=1994&pages=726-729&pmid=7912506&doi=10.1001/archneur.1994.00540190110024&)

29. Cavaletti G, Tredici G, Braga M, Tazzari S. Experimental peripheral neuropathy induced in adult rats by repeated intraperitoneal administration of taxol. Exp Neurol. 1995;133:64–72. doi: 10.1006/exnr.1995.1008.  [DOI](https://doi.org/10.1006/exnr.1995.1008) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7601264/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Exp%20Neurol&title=Experimental%20peripheral%20neuropathy%20induced%20in%20adult%20rats%20by%20repeated%20intraperitoneal%20administration%20of%20taxol&author=G%20Cavaletti&author=G%20Tredici&author=M%20Braga&author=S%20Tazzari&volume=133&publication_year=1995&pages=64-72&pmid=7601264&doi=10.1006/exnr.1995.1008&)

30. Parkinson DB, Bhaskaran A, Arthur-Farraj P, Noon LA, Woodhoo A, Lloyd AC, et al. C-jun is a negative regulator of myelination. J Cell Biol. 2008;181:625–637. doi: 10.1083/jcb.200803013.  [DOI](https://doi.org/10.1083/jcb.200803013) | [PMC free article](/articles/PMC2386103/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18490512/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Cell%20Biol&title=C-jun%20is%20a%20negative%20regulator%20of%20myelination&author=DB%20Parkinson&author=A%20Bhaskaran&author=P%20Arthur-Farraj&author=LA%20Noon&author=A%20Woodhoo&volume=181&publication_year=2008&pages=625-637&pmid=18490512&doi=10.1083/jcb.200803013&)

31. Scuteri A, Galimberti A, Maggioni D, Ravasi M, Pasini S, Nicolini G, et al. Role of mapks in platinum-induced neuronal apoptosis. Neurotoxicology. 2009;30:312–319. doi: 10.1016/j.neuro.2009.01.003.  [DOI](https://doi.org/10.1016/j.neuro.2009.01.003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19428505/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neurotoxicology&title=Role%20of%20mapks%20in%20platinum-induced%20neuronal%20apoptosis&author=A%20Scuteri&author=A%20Galimberti&author=D%20Maggioni&author=M%20Ravasi&author=S%20Pasini&volume=30&publication_year=2009&pages=312-319&pmid=19428505&doi=10.1016/j.neuro.2009.01.003&)

32. Whitehurst AW, Bodemann BO, Cardenas J, Ferguson D, Girard L, Peyton M, et al. Synthetic lethal screen identification of chemosensitizer loci in cancer cells. Nature. 2007;446:815–819. doi: 10.1038/nature05697.  [DOI](https://doi.org/10.1038/nature05697) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17429401/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Synthetic%20lethal%20screen%20identification%20of%20chemosensitizer%20loci%20in%20cancer%20cells&author=AW%20Whitehurst&author=BO%20Bodemann&author=J%20Cardenas&author=D%20Ferguson&author=L%20Girard&volume=446&publication_year=2007&pages=815-819&pmid=17429401&doi=10.1038/nature05697&)

33. Endo Y, Beauchamp E, Woods D, Taylor WG, Toretsky JA, Uren A, et al. Wnt-3a and dickkopf-1 stimulate neurite outgrowth in ewing tumor cells via a frizzled3- and c-jun n-terminal kinase-dependent mechanism. Mol Cell Biol. 2008;28:2368–2379. doi: 10.1128/MCB.01780-07.  [DOI](https://doi.org/10.1128/MCB.01780-07) | [PMC free article](/articles/PMC2268413/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18212053/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Cell%20Biol&title=Wnt-3a%20and%20dickkopf-1%20stimulate%20neurite%20outgrowth%20in%20ewing%20tumor%20cells%20via%20a%20frizzled3-%20and%20c-jun%20n-terminal%20kinase-dependent%20mechanism&author=Y%20Endo&author=E%20Beauchamp&author=D%20Woods&author=WG%20Taylor&author=JA%20Toretsky&volume=28&publication_year=2008&pages=2368-2379&pmid=18212053&doi=10.1128/MCB.01780-07&)

34. Kalaydjieva L, Gresham D, Gooding R, Heather L, Baas F, de Jonge R, et al. N-myc downstream-regulated gene 1 is mutated in hereditary motor and sensory neuropathy-lom. Am J Hum Genet. 2000;67:47–58. doi: 10.1086/302978.  [DOI](https://doi.org/10.1086/302978) | [PMC free article](/articles/PMC1287101/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10831399/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=N-myc%20downstream-regulated%20gene%201%20is%20mutated%20in%20hereditary%20motor%20and%20sensory%20neuropathy-lom&author=L%20Kalaydjieva&author=D%20Gresham&author=R%20Gooding&author=L%20Heather&author=F%20Baas&volume=67&publication_year=2000&pages=47-58&pmid=10831399&doi=10.1086/302978&)

35. Bergmann TK, Green H, Brasch-Andersen C, Mirza MR, Herrstedt J, Holund B, et al. Retrospective study of the impact of pharmacogenetic variants on paclitaxel toxicity and survival in patients with ovarian cancer. Eur J Clin Pharmacol. 2011 doi: 10.1007/s00228-011-1007-6.  [DOI](https://doi.org/10.1007/s00228-011-1007-6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21327421/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Clin%20Pharmacol&title=Retrospective%20study%20of%20the%20impact%20of%20pharmacogenetic%20variants%20on%20paclitaxel%20toxicity%20and%20survival%20in%20patients%20with%20ovarian%20cancer&author=TK%20Bergmann&author=H%20Green&author=C%20Brasch-Andersen&author=MR%20Mirza&author=J%20Herrstedt&publication_year=2011&pmid=21327421&doi=10.1007/s00228-011-1007-6&)

36. Leskela S, Jara C, Leandro-Garcia LJ, Martinez A, Garcia-Donas J, Hernando S, et al. Polymorphisms in cytochromes p450 2c8 and 3a5 are associated with paclitaxel neurotoxicity. Pharmacogenomics J. 2011;11:121–129. doi: 10.1038/tpj.2010.13.  [DOI](https://doi.org/10.1038/tpj.2010.13) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20212519/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Polymorphisms%20in%20cytochromes%20p450%202c8%20and%203a5%20are%20associated%20with%20paclitaxel%20neurotoxicity&author=S%20Leskela&author=C%20Jara&author=LJ%20Leandro-Garcia&author=A%20Martinez&author=J%20Garcia-Donas&volume=11&publication_year=2011&pages=121-129&pmid=20212519&doi=10.1038/tpj.2010.13&)

37. Marsh S, Paul J, King CR, Gifford G, McLeod HL, Brown R. Pharmacogenetic assessment of toxicity and outcome after platinum plus taxane chemotherapy in ovarian cancer: The scottish randomised trial in ovarian cancer. J Clin Oncol. 2007;25:4528–4535. doi: 10.1200/JCO.2006.10.4752.  [DOI](https://doi.org/10.1200/JCO.2006.10.4752) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17925548/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Pharmacogenetic%20assessment%20of%20toxicity%20and%20outcome%20after%20platinum%20plus%20taxane%20chemotherapy%20in%20ovarian%20cancer:%20The%20scottish%20randomised%20trial%20in%20ovarian%20cancer&author=S%20Marsh&author=J%20Paul&author=CR%20King&author=G%20Gifford&author=HL%20McLeod&volume=25&publication_year=2007&pages=4528-4535&pmid=17925548&doi=10.1200/JCO.2006.10.4752&)

38. Sissung TM, Mross K, Steinberg SM, Behringer D, Figg WD, Sparreboom A, et al. Association of abcb1 genotypes with paclitaxel-mediated peripheral neuropathy and neutropenia. Eur J Cancer. 2006;42:2893–2896. doi: 10.1016/j.ejca.2006.06.017.  [DOI](https://doi.org/10.1016/j.ejca.2006.06.017) | [PMC free article](/articles/PMC1647318/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16950614/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Cancer&title=Association%20of%20abcb1%20genotypes%20with%20paclitaxel-mediated%20peripheral%20neuropathy%20and%20neutropenia&author=TM%20Sissung&author=K%20Mross&author=SM%20Steinberg&author=D%20Behringer&author=WD%20Figg&volume=42&publication_year=2006&pages=2893-2896&pmid=16950614&doi=10.1016/j.ejca.2006.06.017&)

39. Broyl A, Corthals SL, Jongen JL, van der Holt B, Kuiper R, de Knegt Y, et al. Mechanisms of peripheral neuropathy associated with bortezomib and vincristine in patients with newly diagnosed multiple myeloma: A prospective analysis of data from the hovon-65/gmmg-hd4 trial. Lancet Oncol. 2010;11:1057–1065. doi: 10.1016/S1470-2045(10)70206-0.  [DOI](https://doi.org/10.1016/S1470-2045(10)70206-0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20864405/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet%20Oncol&title=Mechanisms%20of%20peripheral%20neuropathy%20associated%20with%20bortezomib%20and%20vincristine%20in%20patients%20with%20newly%20diagnosed%20multiple%20myeloma:%20A%20prospective%20analysis%20of%20data%20from%20the%20hovon-65/gmmg-hd4%20trial&author=A%20Broyl&author=SL%20Corthals&author=JL%20Jongen&author=B%20van%20der%20Holt&author=R%20Kuiper&volume=11&publication_year=2010&pages=1057-1065&pmid=20864405&doi=10.1016/S1470-2045(10)70206-0&)

40. Cavaletti G, Alberti P, Marmiroli P. Chemotherapy-induced peripheral neurotoxicity in the era of pharmacogenomics. Lancet Oncol. 2011;12:1151–1161. doi: 10.1016/S1470-2045(11)70131-0.  [DOI](https://doi.org/10.1016/S1470-2045(11)70131-0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21719347/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet%20Oncol&title=Chemotherapy-induced%20peripheral%20neurotoxicity%20in%20the%20era%20of%20pharmacogenomics&author=G%20Cavaletti&author=P%20Alberti&author=P%20Marmiroli&volume=12&publication_year=2011&pages=1151-1161&pmid=21719347&doi=10.1016/S1470-2045(11)70131-0&)

41. Favis R, Sun Y, van de Velde H, Broderick E, Levey L, Meyers M, et al. Genetic variation associated with bortezomib-induced peripheral neuropathy. Pharmacogenet Genom. 2011;21:121–129. doi: 10.1097/FPC.0b013e3283436b45.  [DOI](https://doi.org/10.1097/FPC.0b013e3283436b45) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21228734/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genom&title=Genetic%20variation%20associated%20with%20bortezomib-induced%20peripheral%20neuropathy&author=R%20Favis&author=Y%20Sun&author=H%20van%20de%20Velde&author=E%20Broderick&author=L%20Levey&volume=21&publication_year=2011&pages=121-129&pmid=21228734&doi=10.1097/FPC.0b013e3283436b45&)

42. Johnson DC, Corthals SL, Walker BA, Ross FM, Gregory WM, Dickens NJ, et al. Genetic factors underlying the risk of thalidomide-related neuropathy in patients with multiple myeloma. J Clin Oncol. 2011;29:797–804. doi: 10.1200/JCO.2010.28.0792.  [DOI](https://doi.org/10.1200/JCO.2010.28.0792) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21245421/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Genetic%20factors%20underlying%20the%20risk%20of%20thalidomide-related%20neuropathy%20in%20patients%20with%20multiple%20myeloma&author=DC%20Johnson&author=SL%20Corthals&author=BA%20Walker&author=FM%20Ross&author=WM%20Gregory&volume=29&publication_year=2011&pages=797-804&pmid=21245421&doi=10.1200/JCO.2010.28.0792&)

43. Cleeland CS, Farrar JT, Hausheer FH. Assessment of cancer-related neuropathy and neuropathic pain. Oncologist. 2010;15(Suppl 2):13–18. doi: 10.1634/theoncologist.2009-S501.  [DOI](https://doi.org/10.1634/theoncologist.2009-S501) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20489192/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncologist&title=Assessment%20of%20cancer-related%20neuropathy%20and%20neuropathic%20pain&author=CS%20Cleeland&author=JT%20Farrar&author=FH%20Hausheer&volume=15&issue=Suppl%202&publication_year=2010&pages=13-18&pmid=20489192&doi=10.1634/theoncologist.2009-S501&)

44. Kuroi K, Shimozuma K. Neurotoxicity of taxanes: Symptoms and quality of life assessment. Breast Cancer. 2004;11:92–99. doi: 10.1007/BF02968010.  [DOI](https://doi.org/10.1007/BF02968010) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14718800/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Breast%20Cancer&title=Neurotoxicity%20of%20taxanes:%20Symptoms%20and%20quality%20of%20life%20assessment&author=K%20Kuroi&author=K%20Shimozuma&volume=11&publication_year=2004&pages=92-99&pmid=14718800&doi=10.1007/BF02968010&)

45. Postma TJ, Heimans JJ. Grading of chemotherapy-induced peripheral neuropathy. Ann Oncol. 2000;11:509–513. doi: 10.1023/a:1008345613594.  [DOI](https://doi.org/10.1023/a:1008345613594) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10907941/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Oncol&title=Grading%20of%20chemotherapy-induced%20peripheral%20neuropathy&author=TJ%20Postma&author=JJ%20Heimans&volume=11&publication_year=2000&pages=509-513&pmid=10907941&doi=10.1023/a:1008345613594&)

46. Shimozuma K, Ohashi Y, Takeuchi A, Aranishi T, Morita S, Kuroi K, et al. Taxane-induced peripheral neuropathy and health-related quality of life in postoperative breast cancer patients undergoing adjuvant chemotherapy: N-sas bc 02, a randomized clinical trial. Support Care Cancer. 2012 doi: 10.1007/s00520-012-1492-x.  [DOI](https://doi.org/10.1007/s00520-012-1492-x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22584733/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Support%20Care%20Cancer&title=Taxane-induced%20peripheral%20neuropathy%20and%20health-related%20quality%20of%20life%20in%20postoperative%20breast%20cancer%20patients%20undergoing%20adjuvant%20chemotherapy:%20N-sas%20bc%2002,%20a%20randomized%20clinical%20trial&author=K%20Shimozuma&author=Y%20Ohashi&author=A%20Takeuchi&author=T%20Aranishi&author=S%20Morita&publication_year=2012&pmid=22584733&doi=10.1007/s00520-012-1492-x&)

47. Motsinger-Reif AA, Jorgenson E, Relling MV, Kroetz DL, Weinshilboum R, Cox NJ, et al. Genome-wide association studies in pharmacogenomics: Successes and lessons. Pharmacogenet Genom. 2010 doi: 10.1097/FPC.0b013e32833d7b45.  [DOI](https://doi.org/10.1097/FPC.0b013e32833d7b45) | [PMC free article](/articles/PMC3003940/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20639796/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genom&title=Genome-wide%20association%20studies%20in%20pharmacogenomics:%20Successes%20and%20lessons&author=AA%20Motsinger-Reif&author=E%20Jorgenson&author=MV%20Relling&author=DL%20Kroetz&author=R%20Weinshilboum&publication_year=2010&pmid=20639796&doi=10.1097/FPC.0b013e32833d7b45&)
