# PACSIN2 polymorphism influences TPMT activity and mercaptopurine-related gastrointestinal toxicity

## Metadata
**Authors:** Gabriele Stocco, Wenjian Yang, Kristine R Crews, William E Thierfelder, Giuliana Decorti, Margherita Londero, Raffaella Franca, Marco Rabusin, Maria Grazia Valsecchi, Deqing Pei, Cheng Cheng, Steven W Paugh, Laura B Ramsey, Barthelemy Diouf, Joseph Robert McCorkle, Terreia S Jones, Ching-Hon Pui, Mary V Relling, William E Evans
**Journal:** Human Molecular Genetics
**Date:** 2012 Jul 30
**DOI:** [10.1093/hmg/dds302](https://doi.org/10.1093/hmg/dds302)
**PMID:** 22846425
**PMCID:** PMC3471396
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3471396/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC3471396/pdf/dds302.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC3471396/pdf/dds302.pdf)

## Abstract

Treatment-related toxicity can be life-threatening and is the primary cause of interruption or discontinuation of chemotherapy for acute lymphoblastic leukemia (ALL), leading to an increased risk of relapse. Mercaptopurine is an essential component of continuation therapy in all ALL treatment protocols worldwide. Genetic polymorphisms in thiopurine S-methyltransferase (TPMT) are known to have a marked effect on mercaptopurine metabolism and toxicity; however, some patients with wild-type TPMT develop toxicity during mercaptopurine treatment for reasons that are not well understood. To identify additional genetic determinants of mercaptopurine toxicity, a genome-wide analysis was performed in a panel of human HapMap cell lines to identify trans-acting genes whose expression and/or single-nucleotide polymorphisms (SNPs) are related to TPMT activity, then validated in patients with ALL. The highest ranking gene with both mRNA expression and SNPs associated with TPMT activity in HapMap cell lines was protein kinase C and casein kinase substrate in neurons 2 (PACSIN2). The association of a PACSIN2 SNP (rs2413739) with TPMT activity was confirmed in patients and knock-down of PACSIN2 mRNA in human leukemia cells (NALM6) resulted in significantly lower TPMT activity. Moreover, this PACSIN2 SNP was significantly associated with the incidence of severe gastrointestinal (GI) toxicity during consolidation therapy containing mercaptopurine, and remained significant in a multivariate analysis including TPMT and SLCO1B1 as covariates, consistent with its influence on TPMT activity. The association with GI toxicity was also validated in a separate cohort of pediatric patients with ALL. These data indicate that polymorphism in PACSIN2 significantly modulates TPMT activity and influences the risk of GI toxicity associated with mercaptopurine therapy.

## INTRODUCTION

Approximately 80% of children with acute lymphoblastic leukemia (ALL) can be cured with combination chemotherapy that almost always includes the purine anti-metabolite mercaptopurine ([1](#DDS302C1),[2](#DDS302C2)). However, treatment-related toxicity with this medication can be life-threatening and is the primary cause of interruption or discontinuation of chemotherapy, which can increase the risk of disease recurrence. Germline polymorphisms in genes encoding drug-metabolizing enzymes can significantly influence the efficacy and toxicity of antileukemic therapy ([3](#DDS302C3)). For mercaptopurine, methylation of the thiol moiety, which is catalyzed by the enzyme thiopurine *S*-methyltransferase (TPMT), yields inactive 6-methyl-mercaptopurine ([4](#DDS302C4),[5](#DDS302C5)). Although genetic polymorphisms in the gene encoding human *TPMT* are known to markedly influence mercaptopurine metabolism and toxicity, there are patients with wild-type *TPMT* who also develop toxicity for reasons that are not fully understood ([6](#DDS302C6),[7](#DDS302C7)).

Mercaptopurine is used together with high-dose methotrexate in post-remission consolidation therapy for pediatric ALL; this treatment regimen is associated with gastrointestinal (GI) toxicity that can be severe and constitutes an important adverse effect of ALL consolidation therapy ([2](#DDS302C2)).

In a previous study investigating the genetic basis of inter-individual differences in methotrexate pharmacokinetics in children with ALL, we showed that single-nucleotide polymorphisms (SNPs) in *solute carrier organic anion transporter family, member 1B1* (*SLCO1B1*) are associated with methotrexate clearance and GI toxicity during consolidation therapy ([8](#DDS302C8)).

Hapmap cell lines (from Utah residents with ancestry from northern and western Europe; CEU) are a model system suitable for genome-wide studies of genetic polymorphisms influencing pharmacological phenotypes that can be measured in cell lines, despite limited statistical power due to the relatively small sample size ([9](#DDS302C9),[10](#DDS302C10)). We previously used a genome-wide approach to rank 17 542 genes for their association with TPMT activity in HapMap cells: a TPMT haplotype significantly predicted TPMT phenotype; however, haplotypes of 96 genes ranked higher than TPMT, indicating the possibility that genetic polymorphism in other genes could influence TPMT activity ([11](#DDS302C11)). This prior genome-wide SNP analysis provided important insights into trans-acting genes influencing TPMT activity, but it is known that the incorporation of gene expression with genome-wide SNP analyses can reduce false positives and increase the robustness of discoveries ([12](#DDS302C12)), consistent with trait-associated SNPs being more likely to be expression quantitative trait loci ([13](#DDS302C13)).

Therefore, to identify additional genetic variability influencing TPMT activity, a genome-wide analysis was performed to identify trans-genes whose expression or SNPs are related to TPMT activity in a panel of human HapMap cell lines, with subsequent validation of their effects on TPMT activity and mercaptopurine toxicity in patients receiving mercaptopurine therapy.

## RESULTS

### TPMT activity in HapMap (CEU) cells

TPMT activity was measured in the 30 HapMap CEU trios (Fig. [1](#DDS302F1)). The median value was 28.5 U/10^9^ cells, with a range of 9.1–90.4. Genotype for SNP rs1142345, the most common SNP defining a non-functional *TPMT* allele in humans (*TPMT*3A* and *TPMT*3C* both contain rs1142345), was assessed in all HapMap CEU cell lines considered. As depicted in Figure [1](#DDS302F1) and as we have previously reported ([11](#DDS302C11)), the four cell lines from subjects heterozygous for the rs1142345 SNP (i.e. AG genotype) had significantly lower TPMT activity than the 83 cell lines with the AA wild-type genotype for this SNP (*P*-value Welch test = 0.00026).

### Figure 1.

![Figure 1.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/22c2/3471396/ebd4a334fb74/dds30201.jpg)

Distribution of TPMT activity measured in 87 HapMap CEU cell lines. Each rectangle represents the TPMT activity value measured in one cell line. Blue rectangles represent cell lines from individuals with wild-type TPMT, whereas orange rectangles represent cell lines from individuals with variant TPMT (rs1142342 AG genotype); the four cell lines from subjects heterozygous for the rs1142345 SNP (i.e. AG genotype) had significantly lower TPMT activity than the 83 cell lines with the AA wild-type genotype for this SNP (P-value Welch test = 0.00026) (11).

### Genes whose expression or SNPs were significantly associated with TPMT activity in HapMap cells

A summary of genes whose mRNA expression or SNPs were associated with TPMT activity in the HapMap cells is depicted in Figure [2](#DDS302F2). Association between gene expression and TPMT activity was analyzed for 15 661 genes: the expression of 38 genes was significantly correlated with TPMT activity with a *P*-value <5 × 10^−5^ (Table [1](#DDS302TB1)). Among these 38 genes, only one gene also had a SNP annotated to its sequence that was significantly associated with TPMT activity at a *P*-value <5 × 10^−4^: *protein kinase C and casein kinase substrate in neurons 2* (*PACSIN2*) mRNA expression (*P*-value linear model = 1.20 × 10^−5^, correlation estimate = 0.36) and SNP rs2413739 (*P*-value linear model = 4.6 × 10^−4^) were both significantly associated with TPMT activity.

### Figure 2.

![Figure 2.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/22c2/3471396/b8ca2fc0c247/dds30202.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3471396_dds30202.jpg)

Diagram describing the analysis strategy to identify the most important trans-genes whose expression and SNPs are associated with TPMT activity in the HapMap CEU population. Among the genes whose expression is highly associated with TPMT activity (false discovery rate 1.6%), PACSIN2 is the only one that has significant SNPs (P-value <0.0005) associated with TPMT activity.

### Table 1.

| Transcript ID | Chromosome | Gene ID | Gene symbol | Reference sequence | Correlation estimate | P-value |
| --- | --- | --- | --- | --- | --- | --- |
| 3764471 | chr17 | 9110 | MTMR4 | NM_004687 | 0.55 | 1.48E−07 |
| 3454006 | chr12 | 91010 | FMNL3 | NM_198900, NM_175736 | 0.52 | 5.27E−07 |
| 3496409 | chr13 | 2262 | GPC5 | NM_004466 | 0.48 | 6.91E−07 |
| 3290785 | chr10 | 8030 | CCDC6 | NM_005436 | 0.49 | 1.07E−06 |
| 3456212 | chr12 | 7786 | MAP3K12 | NM_006301 | 0.47 | 1.35E−06 |
| 3403595 | chr12 | 50856 | CLEC4A | NM_016184,NM_194450,NM_194447, NM_194448 | 0.39 | 1.85E−06 |
| 3275729 | chr10 | 3559 | IL2RA | NM_000417 | 0.44 | 2.46E−06 |
| 3662417 | chr16 | 1071 | CETP | NM_000078 | 0.39 | 2.91E−06 |
| 3832256 | chr19 | 10653 | SPINT2 | NM_021102 | 0.44 | 3.20E−06 |
| 3063795 | chr7 | 55262 | C7orf43 | NM_018275 | 0.46 | 5.96E−06 |
| 3156307 | chr8 | 5747 | PTK2 | NM_005607,NM_153831 | 0.43 | 6.57E−06 |
| 3883819 | chr20 | 22839 | DLGAP4 | NM_183006,NM_001042486,NM_014902 | 0.40 | 7.88E−06 |
| 3338968 | chr11 | 55191 | NADSYN1 | NM_018161 | 0.46 | 1.08E−05 |
| 3426257 | chr12 | 8835 | SOCS2 | NM_003877 | 0.44 | 1.11E−05 |
| 3962678 | chr22 | 11252 | PACSIN2 | NM_007229 | 0.38 | 1.20E−05 |
| 3965784 | chr22 | 5600 | MAPK11 | NM_002751 | 0.37 | 1.42E−05 |
| 3621080 | chr15 | 9333 | TGM5 | NM_201631, NM_004245 | 0.34 | 1.44E−05 |
| 3608638 | chr15 | 9899 | SV2B | NM_014848 | 0.39 | 1.47E−05 |
| 3802980 | chr18 | 1824 | DSC2 | NM_004949, NM_024422 | 0.41 | 1.50E−05 |
| 3470193 | chr12 | 1240 | CMKLR1 | NM_004072 | 0.40 | 1.53E−05 |
| 3479438 | chr12 | 55743 | CHFR | NM_018223 | 0.49 | 1.69E−05 |
| 2803329 | chr5 | 10409 | BASP1 | NM_006317 | 0.47 | 1.78E−05 |
| 3362263 | chr11 | 23258 | RAB6IP1 | NM_015213 | 0.45 | 2.27E−05 |
| 3830484 | chr19 | 2867 | FFAR2 | NM_005306 | 0.35 | 2.32E−05 |
| 2766492 | chr4 | 201895 | C4orf34 | NM_174921 | 0.39 | 2.72E−05 |
| 3476097 | chr12 | 8099 | CDK2AP1 | NM_004642 | 0.40 | 2.85E−05 |
| 3833793 | chr19 | 53916 | RAB4B | NM_016154 | 0.45 | 3.14E−05 |
| 3405032 | chr12 | 2120 | ETV6 | NM_001987 | 0.50 | 3.23E−05 |
| 3240987 | chr10 | 1326 | MAP3K8 | NM_005204 | 0.48 | 3.25E−05 |
| 2412312 | chr1 | 22996 | C1orf34 | NM_001080494 | 0.41 | 3.52E−05 |
| 3802924 | chr18 | 1825 | DSC3 | NM_001941, NM_024423 | 0.38 | 3.65E−05 |
| 2485688 | chr2 | 23177 | CEP68 | NM_015147 | 0.39 | 3.77E−05 |
| 3662612 | chr16 | 89970 | RSPRY1 | NM_133368 | 0.48 | 3.82E−05 |
| 2556752 | chr2 | 200734 | SPRED2 | NM_181784 | 0.47 | 3.84E−05 |
| 3753860 | chr17 | 256957 | C17orf66 | NM_152781 | 0.31 | 4.31E−05 |
| 2500275 | chr2 | 10018 | BCL2L11 | NM_138621, NM_006538 | -0.40 | 4.47E−05 |
| 3470597 | chr12 | 54434 | SSH1 | NM_018984 | 0.38 | 4.66E−05 |
| 3338192 | chr11 | 595 | CCND1 | NM_053056 | 0.36 | 4.72E−05 |

Table 1 Caption: Genes whose expression was statistically associated with TPMT activity (P-value lower than 5 × 10−5) in 87 Human HapMap CEU cell lines

### Relation of PACSIN2 SNP and TPMT activity in patients

The effect on erythrocyte TPMT activity of the *PACSIN2* SNP identified in the HapMap population (rs2413739) was assessed in ALL patients who either had a wild-type *TPMT* genotype or were heterozygous for the most common variant *TPMT* alleles. TPMT activity and genotype was evaluated in 299 patients; the *PACSIN2* genotype was successfully determined for 286 of the 299 patients for whom TPMT activity and the genotype were measured (for 13 patients, PACSIN2 genotyping could not be determined because of technical reasons). This analysis showed that both the *TPMT* and *PACSIN2* variants were significantly associated with TPMT activity (Fig. [3](#DDS302F3)). Indeed SNPs in both genes had a significant association with TPMT enzyme activity in a univariate analysis (*P*-values from linear model: *TPMT* = 6.0 × 10^−16^; *PACSIN2* = 0.027). Moreover, these SNPs (*TPMT*, *PACSIN2*) were also significantly related to TPMT activity in a multivariate analysis that combined the two SNPs in a multilocus genotype (*P*-values from linear model, *TPMT* = 3.0 × 10^−16^ and *PACSIN2* = 0.011); this multivariate model explained 22.4% of variability in TPMT activity. Effects of *TPMT* and *PACSIN2* genotypes were significant in patients after adjusting for ethnicity, evaluated as self-reported race, addressing the possibility of a potential confounding effect of ancestry (*P*-values from linear model adjusted for ethnicity, *TPMT* = 6.4 × 10^−16^; *PACSIN2* = 0.034). The *PACSIN2* SNP was significantly related to TPMT activity both in patients with the wild-type *TPMT* genotype (explaining 1.8% of variability, *P*-value from linear model = 0.031) and in those with the variant *TPMT* genotype (explaining 30.8% of variability, *P*-value from linear model = 0.0091); the difference in the median TPMT activity in patients with wild-type versus variant *PACSIN2* genotypes (SNP rs2413739) was 1.2 U/ml packed erythrocytes for patients with wild-type *TPMT* and 2.6 U/ml packed erythrocytes for those with variant *TPMT* (Fig. [3](#DDS302F3)).

### Figure 3.

![Figure 3.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/22c2/3471396/30f236ff7108/dds30203.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3471396_dds30203.jpg)

The box plot shows TPMT activity in 286 patients with ALL as a function of the TPMT rs1142345 [A719G]/PACSIN2 rs2413739 multilocus genotype. There were no GG genotypes for TPMT rs1142345 and therefore all variants genotypes for TPMT are heterozygous. SNPs in both genes had a significant association with TPMT enzyme activity in a univariate analysis (P-values from linear model with formula TPMT activity–genotype, considering an additive effect for the genotype: TPMT = 6.0 × 10−16; PACSIN2 P = 0.027). Moreover, these SNPs (TPMT, PACSIN2) were also significantly related to TPMT activity in a multivariate analysis that combined the two SNPs in a multilocus genotype (TPMT P-value from linear model = 3.0 × 10−16 and PACSIN2 P-value from linear model = 0.011). The PACSIN2 genotype had a significant effect even in the subgroup with the wild-type TPMT genotype (P-value from linear model = 0.031) and in the subgroup with the variant TPMT genotype (P-value from linear model = 0.0091).

### PACSIN2 SNP had a significant association with GI toxicity in patients treated with mercaptopurine

The entire cohort of Total 13B patients was considered for the genetic association discovery study for GI toxicity in patients: of 247 children with newly diagnosed ALL enrolled on the Total 13B study, 208 were evaluable herein. We excluded one patient who died shortly after diagnosis and did not have a remission DNA sample, five patients with Down syndrome, and one patient with cystic fibrosis because their underlying diseases can influence toxicity; we also excluded 32 patients who came off study for various reasons: induction failure (*n* = 1), ALL relapse (*n* = 12), second cancer (*n* = 3), stem cell transplantation (*n* = 3), non-compliance (*n* = 3), non-GI toxicity (*n* = 3), refused therapy (*n* = 2), death unrelated to disease or therapy (*n* = 2) or other reasons (*n* = 3) ([14](#DDS302C14)). Among the 208 children with ALL receiving therapy according to St Jude Total 13B protocol and evaluable for the toxicity during consolidation therapy, 16 (7.7%) developed severe (Grade 3–4) GI toxicity; the *TPMT* genotype was available for all 208 patients enrolled, whereas *PACSIN2* and *SLCO1B1* genotypes were determined in 189 (for 19 patients, PACSIN2 and/or SLCO1B1 could not be determined because of technical reasons). For these 189 children with complete genotyping data for *TPMT*, *SLCO1B1* and *PACSIN2*, the frequency of GI toxicity was 8.5%; among these patients, deficiency in TPMT activity predisposed to an increased incidence of severe GI toxicity during consolidation therapy which comprised methotrexate (2 g/m^2^/week) and mercaptopurine (75 mg/m^2^/day, in all patients during consolidation therapy, regardless of TPMT genotype). Indeed, among nine patients with a variant *TPMT* allele, the frequency of GI toxicity was 33%, compared with 7.2% in patients with wild-type *TPMT* (*P*-value from logistic regression model = 0.0058, Fig. [4](#DDS302F4)). *PACSIN2* SNP rs2413739 also had a significant association with GI toxicity during consolidation therapy: the frequency of toxicity was 2.1, 9.1 and 13.2%, respectively, for the CC, CT and TT allele (*P*-value from logistic regression model = 0.046, Fig. [4](#DDS302F4)). An *SLCO1B1* SNP (rs11045879) that we have previously reported to be associated with GI toxicity in ALL patients ([8](#DDS302C8)) was also associated with the incidence of GI toxicity: none of the patients with the *SLCO1B1* CC or CT genotype had GI toxicity, whereas 11.8% of the patients with the wild-type *SLCO1B1* TT genotype had GI toxicity (*P*-value from logistic regression model = 0.012, Fig. [4](#DDS302F4)).

### Figure 4.

![Figure 4.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/22c2/3471396/6de9769e7ece/dds30204.jpg)

The frequency of severe (Grade 3–4) GI toxicity during consolidation therapy in patients with ALL treated according to the St Jude Total 13B protocol as a function of TPMT rs1142345, PACSIN2 rs2413739, SLCO1B1 rs11045879 genotypes. Panel A shows barplots reporting the percentage of patients developing GI toxicity and the corresponding genotypes. Panel B represents cumulative incidence plots of GI toxicity and the corresponding genotypes for the same patients. There were no GG genotypes for TPMT rs1142345 and therefore all variants genotypes for TPMT are heterozygous. Among nine patients with a variant TPMT allele, the frequency of gastrointestinal toxicity was 33%, compared with 7.2% in patients with the wild-type TPMT (P-value from logistic regression model = 0.0058); PACSIN2 SNP rs2413739 also had a significant association with GI toxicity during consolidation therapy: the frequency of toxicity was 2.1, 9.1 and 13.2%, respectively, for the CC, CT and TT allele (P-value from logistic regression model = 0.046). An SLCO1B1 SNP (rs11045879) that we have previously reported to be associated with GI toxicity in ALL patients (9) was also associated with the incidence of GI toxicity: none of the patients with the SLCO1B1 CC or CT genotype had GI toxicity, whereas 11.8% of the patients with the wild-type SLCO1B1 TT genotype had GI toxicity (P-value from logistic regression model = 0.012).

### TPMT, SLCO1B1 and PACSIN2 multi-locus genotype and GI toxicity

Multivariate logistic regression models along with classification and regression trees analyses (CART) were used to assess the effects of *TPMT*, *SLCO1B1* and *PACSIN2* SNPs and their interaction in predisposing to GI toxicity. As summarized in Table [2](#DDS302TB2), all three SNPs had a significant relation with the GI toxicity in the multivariate analysis; the effect of the considered SNPs on the incidence of GI toxicity was confirmed after adjusting for ethnicity as defined by self-reported race, addressing the possibility of a potential confounding effect determined by ancestry (Table [2](#DDS302TB2)). CART analysis to evaluate interactions between the genotypes in reference to their relation to the GI toxicity (Fig. [5](#DDS302F5)) revealed that the *TPMT* genotype best distinguished GI toxicity probability (split 1 on CART, Fig. [5](#DDS302F5)), with the presence of the variant *TPMT* allele conferring an increased probability of GI toxicity (33.3%) compared with patients with wild-type *TPMT* (7.2%, *P*-value from logistic regression model on subgroup = 0.0058). Among the nine patients with variant *TPMT*, no additional effect of *PACSIN2* and *SLCO1B1* was evident, although the number of these patients was relatively low. Among the 180 patients with wild-type *TPMT*, the most important genotype to distinguish the probability of GI toxicity was the *SLCO1B1* SNP (split 2 on CART, Fig. [5](#DDS302F5)): none of the 52 patients with a C (variant) allele for the *SLCO1B1* SNP developed toxicity, whereas the probability among the 128 patients with the TT genotype was 10.2% (*P*-value from logistic regression model on subgroup = 0.017). Among the 128 patients with a wild-type allele for *TPMT* and TT *SCLO1B1* genotypes (split 3 on CART) the probability of GI toxicity in patients with the *PACSIN2* TT allele was two times higher than in those with a C allele (15.8 versus 7.8%), albeit not statistically significant given the small number of patients (*P*-value from a logistic regression model on subgroup = 0.17).

### Table 2.

|   | Multivariate logistic regression model for the incidence of GI toxicity |  |  | Multivariate logistic regression model for the incidence of GI toxicity, adjusted for ethnicitya |  |  |
| --- | --- | --- | --- | --- | --- | --- |
| Effectb | 95% Confidence interval | P-value | Effectb | 95% Confidence interval | P-value |  |
| PACSIN2 rs2413739 genotype (T versus C) | 2.06 | 1.01–4.56 | 0.040 | 2.09 | 1.03–4.58 | 0.037 |
| SLCO1B1 rs11045879 genotype (T versus C) | 12.61 | 1.69–1613.81 | 0.017 | 11.49 | 1.56–1466.31 | 0.0016 |
| TPMT rs1142345 genotype (A versus G) | 6.03 | 1.23–27.26 | 0.0085 | 6.58 | 1.11–33.33 | 0.045 |

Table 2 Caption: Multivariate logistic regression model for the effects of PACSIN2, SLCO1B1 and TPMT genotypes on the incidence of GI toxicity

### Figure 5.

![Figure 5.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/22c2/3471396/46c31ca2bcdf/dds30205.jpg)

Classification and regression tree analysis for genetic predictors influencing those patients at an increased risk for GI toxicity. The three SNPs have independent effects, with TPMT being ranked before SLCO1B1 and PACSIN2.

### Validation of TPMT, SLCO1B1 and PACSIN2 SNPs on mercaptopurine-related GI toxicity in an independent cohort of patients

To validate the effect of *TPMT, SLCO1B1* and *PACSIN2* SNPs on the incidence of severe GI toxicity during consolidation therapy of pediatric ALL, we assessed an independent set of patients who were treated on the Associazione Italiana di Ematologia ed Oncologia Pediatrica (Italy); Berlin-Frankfurt-Münster (Germany, Austria, Switzerland) ALL Study Group (AIEOP-BFM-ALL) 2000 protocol. *PACSIN2* SNP rs2413739 also showed a correlation with the incidence of GI toxicity in this cohort; the frequency of cases with GI toxicity was 28, 32 and 51% in the CC, CT and TT genotypes, respectively (*P*-value from logistic regression model = 0.032, Fig. [6](#DDS302F6)), as did SNP rs737782, in LD with rs2413739, with frequencies of GI toxicity of 26, 31 and 51% in the CC, CG and GG genotypes, respectively (*P*-value from a logistic regression model = 0.015, Fig. [6](#DDS302F6)). However, for SNP rs5996259, not in LD with rs2413739, there was not an association with toxicity (*P*-value from a logistic regression model = 0.32). The *TPMT* and *SLCO1B1* SNPs did not show a significant association with the incidence of GI toxicity in the Italian cohort, perhaps due to the lower dosage of mercaptopurine used in this protocol (25 versus 75 mg/m^2^/day).

### Figure 6.

![Figure 6.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/22c2/3471396/fb39fa5fc9c9/dds30206.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3471396_dds30206.jpg)

The frequency of severe (Grade 3–4) GI toxicity during consolidation therapy in the validation cohort, the Italian AIEOP-BFM-ALL 2000 and genotypes in PACSIN2; PACSIN2 SNP rs2413739 and rs737782 resulted associated with the incidence of GI toxicity in this cohort (P-values from logistic regression model, 0.032 and 0.015, respectively).

### Knock-down of PACSIN2 was associated with reduction in TPMT activity and mRNA in a human B-lineage leukemia cell line

Knock-down of PACSIN2 was performed in the NALM6 B-lineage ALL human cell line, heterologously expressing either *TPMT*1* or *TPMT*3A*. Through the heterologous expression of *TPMT*1*, enzymatic activity increased approximately six times in comparison with TPMT activity in native NALM6 (mean activity 176.8 versus 28.9 U/10^9^ cells, *P*-value Welch test = 1.1 × 10^−6^), whereas *TPMT*3A* expression lead to only a modest (<2-fold) increase in TPMT activity (41.5 versus 28.9 U/10^9^ cells, *P*-value Welch *t*-test = 0.028), due to the known rapid degradation of TPMT protein encoded by *TPMT*3A* in comparison with *TPMT*1* ([15](#DDS302C15)).

After knock-down of *PACSIN2*, a significant (30%) decrease in TPMT activity was observed in cells heterologously expressing TPMT*1 (123.8 U/10^9^ cells in *PACSIN2* knock-down versus 176.8 U/10^9^ in a knock-down scrambled vector control, *P*-value Welch *t*-test = 0.0036), but less so in cells expressing *TPMT*3A* (47.3 versus 41.5 U/10^9^ cells, *P*-value Welch *t*-test = 0.090) (Fig. [7](#DDS302F7)).

### Figure 7.

![Figure 7.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/22c2/3471396/5ade2956772c/dds30207.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3471396_dds30207.jpg)

TPMT activity in NALM6 cells heterologously expressing TPMT*1 or TPMT*3A and with knock-down of PACSIN2. After knock-down of PACSIN2, a significant (30%) decrease in TPMT activity was observed in cells heterologously expressing TPMT*1 (P-value Welch t-test = 0.0036).

Measurement of TPMT mRNA in these cells confirmed the presence of the knock-down of *PACSIN2* and showed an effect (reduction in 30%) on the expression of *TPMT* mRNA comparable with the effect on TPMT activity ([Supplementary Material, Fig. S1](http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/dds302/-/DC1)).

### Pathway analysis of genes modified after PACSIN2 knock-down identifies autophagy as significant process

For global gene expressions in NALM6 with *PACSIN2* knock-down in comparison with a scrambled vector control, 13 990 Affymetrix Human Genome U133 Plus 2.0 Array mRNA probe sets passed the filtering criteria based on signal intensity and probe sets present/absent call. Of these, 20 probe sets changed at least 50% [*P*-value LPE test <1 × 10^−3^, false discovery rate (FDR) = ∼65%] (Tables [3](#DDS302TB3)). To test the reliability of microarray data, we validated four genes [*PACSIN2*, *CSMD1*, *KIAA0226L*/*protein phosphatase 1, regulatory (inhibitor) subunit 2 pseudogene 4* (*PPP1R2P4*), *SH3-domain GRB2-like endophilin B1* (*SH3GLB1*)], by real-time PCR, confirming changes as assessed by microarray analysis ([Supplementary Material, Fig. S2](http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/dds302/-/DC1)).

### Table 3.

| Gene name | Gene symbol | Probe set ID | Ratio KD versus Ctrl | P-value | False discovery rate |
| --- | --- | --- | --- | --- | --- |
| CUB and Sushi multiple domains 1 | CSMD1 | 231223_at | 3.3 | 1.87E−10 | 0 |
| Not annotated to a gene | NA | 1562093_at | 19.2 | 2.28E−10 | 0 |
| Protein kinase C and casein kinase substrate in neurons 2 | PACSIN2 | 201651_s_at | 0.4 | 1.54E−07 | 7.00E−04 |
| Microtubule-associated protein 2 | MAP2 | 225540_at | 9.6 | 5.72E−07 | 0.002 |
| Not annotated to a gene | NA | 216532_x_at | 2.9 | 1.05E−05 | 0.0293 |
| Transcription elongation factor A (SII)-like 4 | TCEAL4 | 202371_at | 3.2 | 2.90E−05 | 0.0674 |
| KIAA0226L/protein phosphatase 1, regulatory (inhibitor) subunit 2 pseudogene 4 | KIAA0266L, PPP1R2P4 | 44790_s_at | 0.5 | 4.13E−05 | 0.0822 |
| Inhibitor of DNA binding 2, dominant negative helix-loop-helix protein | ID2 | 201565_s_at | 2.6 | 7.61E−05 | 0.1327 |
| Granzyme K (granzyme 3; tryptase II) | GZMK | 206666_at | 0.5 | 8.91E−05 | 0.1379 |
| Not annotated to a gene | NA | 241883_x_at | 4.9 | 0.000117 | 0.1632 |
| Secreted phosphoprotein 1 | SPP1 | 209875_s_at | 2.1 | 0.000166 | 0.2107 |
| Coiled-coil domain containing 88A | CCDC88A | 219387_at | 2.0 | 0.000205 | 0.2233 |
| Protein phosphatase 1, regulatory (inhibitor) subunit 2 pseudogene 4 | KIAA0266L, PPP1R2P4 | 219471_at | 0.5 | 0.000208 | 0.2233 |
| Not annotated to a gene | NA | 1560525_at | 3.5 | 0.000331 | 0.3293 |
| Intersectin 1 (SH3 domain protein) | ITSN1 | 209298_s_at | 3.6 | 0.000588 | 0.4962 |
| SH3-domain GRB2-like endophilin B1 | SH3GLB1 | 210101_x_at | 0.5 | 0.000596 | 0.4962 |
| Granzyme A (granzyme 1, cytotoxic T-lymphocyte-associated serine esterase 3) | GZMA | 205488_at | 0.4 | 0.00064 | 0.4962 |
| Zinc finger protein 75a | ZNF75A | 227670_at | 3.8 | 0.000641 | 0.4962 |
| tRNA methyltransferase 11 homolog (S. cerevisiae) | TRMT11 | 218877_s_at | 2.4 | 0.00074 | 0.5426 |
| odz, odd Oz/ten-m homolog 2 (Drosophila) | ODZ2 | 231867_at | 2.8 | 0.000927 | 0.646 |

Table 3 Caption: Genes whose mRNA was differentially expressed after knock-down of PACSIN2 in human NALM6 leukemia cells

Enrichment analysis using MetaCore GeneGO revealed that genes within certain functional pathways were over-represented among those differentially expressed after knock-down of *PACSIN2* (Table [4](#DDS302TB4)): the most highly ranked GeneGo pathways included vesicle formation (rank #1, *P*-value = 0.014), osteopontin and granzyme signaling (rank #2, *P*-value = 0.022) and autophagy (rank #3, *P*-value = 0.023). Highly ranked GeneOntology processes included cellular membrane organization (rank #1, *P*-value = 0.00010), membrane organization (rank #2, *P*-value = 0.00010) and cellular component organization (rank #3, *P*-value = 0.00016). GeneGo biological process networks included proteolysis in cell-cycle and apoptosis (rank #1, *P*-value = 0.010) and actin filaments in cytoskeleton (rank #2, *P*-value = 0.020); complete results for enrichment analysis are provided in [Supplementary Material, Table S1a–c](http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/dds302/-/DC1).

### Table 4.

| MetaCore GeneGo | Ranking | Pathway name | P-value | Ratio probe sets significant/in pathway | Significant mRNAs in pathway |
| --- | --- | --- | --- | --- | --- |
| GeneGo pathway maps | #1 | wtCFTR and delta508 traffic/clathrin-coated vesicles formation (norm and CF) | 1.36E−02 | 1/19 | SH3GLB1 |
| #2 | Development_Osteopontin signaling in osteoclasts and | 2.15E−02 | 1/30 | SPP1 |  |
| #2 | Apoptosis and survival_Granzyme A signaling | 2.15E−02 | 1/30 | GZMA |  |
| #3 | Autophagy | 2.29E−02 | 1/32 | SH3GLB1 |  |
| GeneGoProcess networks | #1 | Proteolysis: proteolysis in cell cycle and apoptosis | 1.01E−02 | 2/125 | GZMA, GZMK |
| #2 | Cytoskeleton: actin filaments | 1.95E−02 | 2/176 | PACSIN, PACSIN2 |  |
| GeneOntology Processes | #1 | Cellular membrane organization | 1.01E−04 | 5/653 | SH3GLB1, CCDC88A, ITSN1, PACSIN, PACSIN2 |
| #2 | Membrane organization | 1.03E−04 | 5/656 | SH3GLB1, CCDC88A, ITSN1, PACSIN, PACSIN2 |  |
| #3 | Negative regulation of cellular component organization | 1.61E−04 | 4/375 | MAP2, SPP1, PACSIN, PACSIN2 |  |

Table 4 Caption: Enrichment analysis of mRNA differentially expressed in human NALM6 leukemia cells, between PACSIN2 controls and knock-down

## DISCUSSION

This study has documented that *PACSIN2* modulates TPMT activity and that a common SNP in *PACSIN2* is significantly associated with the incidence of mercaptopurine-related GI toxicity in children with ALL. Although the importance of *TPMT* genetic polymorphisms in determining the mercaptopurine response and hematological toxicity in children with ALL is very well established (mechanistically and clinically) ([6](#DDS302C6),[16](#DDS302C16)), this is the first *in vivo* evidence that genetic variation in a second gene (*PACSIN2*) also influences TPMT activity. *PACSIN2* was identified by genome-wide SNP and gene expression analyses of the human HapMap cells ([9](#DDS302C9),[10](#DDS302C10),[12](#DDS302C12),[13](#DDS302C13)) for which TPMT activity was also determined ([11](#DDS302C11)), and the association was validated in patients. Indeed, the highest ranking gene for association with TPMT activity in the HapMap cell lines based on gene expression and SNPs (after adjusting for TPMT genotype) was *PACSIN2*, which carries a SNP (rs2413739) that is significantly associated with both TPMT activity and severe GI toxicity in children receiving mercaptopurine for the treatment of ALL. Moreover, knock-down of *PACSIN2* mRNA in human leukemia cells (NALM6) heterologously expressing *TPMT*1* resulted in significantly lower TPMT activity and mRNA.

PACSIN2, also called syndapin II, is a member of the ‘protein kinase C and casein kinase substrate in neurons’ family of proteins; they are involved in linking the actin cytoskeleton with vesicle formation by regulating tubulin polymerization and exert their function mainly through protein–protein interactions with different substrates, such as dynamin or N-WASP ([17](#DDS302C17)). There are data indicating that *PACSIN2* has a role in various biological processes, including endocytosis ([18](#DDS302C18)), cell-cycle control ([19](#DDS302C19)) and autophagy ([20](#DDS302C20)).

To identify additional genes and biological processes that are potentially altered by *PACSIN2*, we identified genes whose expression changed significantly in human leukemia cells in which the PACSIN2 had been knocked-down, revealing actin and cytoskeleton ([17](#DDS302C17),[21](#DDS302C21)), membrane organization ([22](#DDS302C22)) and autophagy ([20](#DDS302C20)) as processes/pathways significantly altered by knock-down of *PACSIN2*. Identification of autophagy was driven largely by the reduction in expression of *SH3GLB1* by *PACSIN2* knock-down. SH3GLB1, also known as endophilin B1/BIF-1, interacts with the UVRAG-Beclin 1, and contributes to the activation of autophagy ([23](#DDS302C23)); it is plausible that SH3GLB1 has a role in the fission of membranes from the Golgi during autophagosome formation ([24](#DDS302C24)). The effects of PACSIN2 on autophagy is further supported by significant changes we observed in a second gene *C13orf18* (also known as *KIAA0226-like*/*PPP1R2P4*), which has recently been shown to be involved in autophagy ([25](#DDS302C25)). The PACSIN2 role in autophagy may also be involved in the mechanism of its effects on TPMT activity, as autophagy has been linked to degradation of the TPMT protein codified by variant *TPMT*3A* and to a lesser extent wild-type *TPMT*1* ([26](#DDS302C26)). Taken together, our results indicate that PACSIN2 may influence TPMT activity through an effect on TPMT mRNA levels and/or TPMT protein degradation. Future studies are warranted to fully elucidate the molecular mechanisms by which *PACSIN2* alters TPMT activity.

The *PACSIN2* SNP rs2413739 is located in an intronic region of the gene and, therefore, it is not clear how this SNP influences *PACSIN2* mRNA and function. Interestingly, the UCSC Genome Browser ([27](#DDS302C27)) indicates that this SNP is located in an area enriched for a marker of acetylation of lysine 27 of the histone 3 protein (H3K27Ac histone mark) as determined by ChIP-Seq analyses ([27](#DDS302C27),[28](#DDS302C28)) ([Supplementary Material, Fig. S3](http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/dds302/-/DC1)). Therefore, the SNP rs2413739 may modulate transcription of the *PACSIN2* gene by influencing the level of acetylation or the effects of the acetylation of this marker, that is thought to enhance transcription possibly by blocking the spread of the repressive histone mark H3K27Me3 ([29](#DDS302C29)).

In addition to showing a significant association of this *PACSIN2* SNP on TPMT activity in cell lines and patients, we also show that patients with the *PACSIN2* rs2413739 C genotype had higher TPMT activity and lower GI toxicity than patients with the T genotype. Furthermore, the association of this *PACSIN2* SNP with GI toxicity during consolidation therapy of ALL with mercaptopurine was identified in patients treated according to the St Jude Total 13B protocol and validated in an independent cohort of Italian patients. In the St Jude Total 13B protocol, patients were treated for a short period of time (2 weeks) with relatively high doses of mercaptopurine (75 mg/m^2^/day, regardless of TPMT genotype) and methotrexate (2 g/m^2^ with folic acid rescue), which resulted in ∼8% of patients exhibiting Grade 3–4 GI toxicity; in this context, *PACSIN2* effects on GI toxicity were significant and independent from those of *TPMT* and *SLCO1B1*. In the validation cohort, the post-remission consolidation phase (protocol M) was quite different and involved lower mercaptopurine doses (25 mg/m^2^/day) and a longer treatment time (8 versus 2 weeks) and the overall incidence of severe GI toxicity was lower (∼4% of patients versus ∼8%); in this context the *PACSIN2* SNP was significantly related to the incidence of GI toxicity, whereas no effects of *TPMT* and *SLCO1B1* could be detected. Patients with high TPMT activity may be protected from mercaptopurine toxicity ([7](#DDS302C7),[30](#DDS302C30)) and in this study, the observation that *PACSIN2* was associated with GI toxicity in both cohorts may indicate that its effects on patients with high TPMT activity are clinically relevant and evident across different protocols of consolidation therapy for pediatric ALL: indeed the most common TPMT coding SNPs do not identify patients with high TPMT activity (only low TPMT activity), whereas the *PACSIN2* rs2413739 CC genotype does. Polymorphisms in the promoter region of TPMT have been previously associated with increased TPMT activity, although the clinical relevance of these observations is not clear ([31](#DDS302C31)). It is also plausible that PACSIN2 may influence mercaptopurine sensitivity independent of its effects on TPMT; indeed it has been shown recently to interact with Rac1 ([32](#DDS302C32)), which is implicated in the pharmacodynamics of thiopurines ([33](#DDS302C33)).

We have shown previously that during maintenance therapy of pediatric ALL according to Total 13B protocol, adjustment of mercaptopurine dose on the basis of TPMT phenotype or genotype mitigates the increased incidence of Grade 3–4 febrile neutropenia in patients with variant *TPMT* genotypes, allowing the effect of other genetic polymorphisms, such as those in inosine-triphosphate-pyrophosphatase to emerge ([34](#DDS302C34)). To explore whether the association of *PACSIN2* SNP with dose-dependent mercaptopurine toxicity is via its effects on TPMT activity, we evaluated whether the *PACSIN2* genotype was associated with the incidence of severe febrile neutropenia during Total 13B maintenance therapy and we could not find a significant association (*P*-value of a weighted logistic regression model considering time at risk = 0.41, [Supplementary Material, Fig. S4](http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/dds302/-/DC1)). This suggests that the association of the *PACSIN2* genotype with mercaptopurine-related dose-dependent toxicity occurs through modulation of TPMT activity, and not through a direct effect of PACSIN2 on cell sensitivity to mercaptopurine. However, this does not rule out tissue-specific effects of PACSIN2 on mercaptopurine sensitivity occurring in GI tissue and not in white blood cells.

In summary, the current study has identified a new genetic polymorphism influencing TPMT activity and mercaptopurine toxicity in children being treated for ALL, providing new insights into the pharmacogenomics of ALL chemotherapy.

## MATERIALS AND METHODS

### Cell culture

Eighty-nine Epstein–Barr virus-transformed lymphoblastoid cell lines, forming the 30 CEPH family trios genotyped in the International HapMap project, were obtained from the Coriell Institute of Medicine ([http://www.ccr.coriell.org](http://www.ccr.coriell.org)). Cell lines were cultured in RPMI-1640 media with 15% fetal bovine serum. Two cell lines with poor growth were excluded from the study.

### TPMT activity in cell lines

Aliquots were obtained for each cell line while in the log phase and cytosolic cell lysates were prepared, as previously reported in detail ([11](#DDS302C11)). TPMT activity was measured adapting a previously described radiochemical assay ([35](#DDS302C35)). Briefly, the level of TPMT activity for each cell line was determined by measuring the incorporation of radioactively labeled *S*-adenosyl-l-methionine into mercaptopurine's inactive metabolite, methylmercaptopurine ([11](#DDS302C11),[35](#DDS302C35)). One unit of enzyme activity represents the formation of 1 nmol of 6-methylmercaptopurine per hour of incubation. TPMT activity in cell lines was normalized per billion of cells evaluated.

### Genotypes and gene expression in cell lines

SNP genotypes for CEU cell lines were downloaded from release 22 on the International HapMap project Website ([http://www.HapMap.org](http://www.HapMap.org)). Gene expression measured by the Affymetrix GeneChip Human Exon 1.0 ST Array was downloaded from the website:

[http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE7761](http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE7761).

### Ethical

We studied children enrolled as patients in two single-institution (St Jude Total 13B and Total 15) and one multi-institution (AIEOP-BFM-ALL 2000) clinical protocol for the treatment of newly diagnosed ALL. All patients and/or their parents provided informed consent for the institutional review board-approved protocols.

### TPMT activity in patients

TPMT activity was measured in erythrocytes from pediatric patients with ALL enrolled at St Jude Children's Research Hospital in Total 13B or Total 15 protocols ([36](#DDS302C36),[37](#DDS302C37)) using a previously described method ([35](#DDS302C35)); one unit of enzyme activity represents the formation of 1 nmol of 6-methylmercaptopurine per hour of incubation. TPMT activity in patients was normalized per milliliter of packed erythrocytes. TPMT activity was measured after patients were in complete remission from ALL (i.e. at least 60 days from diagnosis) and at least 90 days from a red blood cell transfusion. There were a total of 323 measurements for 299 patients; for patients with multiple measurements, the median value was used.

### Toxicity in patients

Toxicity was evaluated during consolidation therapy for ALL according to St Jude Total 13B protocol and classified according to NCI criteria ([8](#DDS302C8),[36](#DDS302C36)): patients developing relevant GI toxicity during consolidation therapy (Grade 3–4 mucositis and diarrhea) were defined as cases and they were compared with all other patients in the cohort who did not develop mucositis and or diarrhea and served as controls in the genetic analyses; other GI toxicity (hepatotoxicity, vomiting and pancreatic toxicity) were uncommon (1 Grade 3 hepatotoxicity, 1 Grade 3 vomiting and no pancreatic toxicity) and therefore were not considered in the present analysis. Consolidation therapy for Total 13B consisted of 2 weekly doses of high-dose methotrexate (2 g/m^2^ over 2 h, followed by leucovorin rescue) and daily mercaptopurine (75 mg/m^2^/day for 2 weeks) ([38](#DDS302C38)). During consolidation therapy, the dosage of mercaptopurine therapy was the same for all patients, regardless of the TPMT genotype; mercaptopurine dosages were subsequently adjusted during continuation therapy based on the TPMT genotype ([6](#DDS302C6),[34](#DDS302C34)). The entire cohort of Total 13B patients was considered for the genetic association discovery study: of 247 children with newly diagnosed ALL enrolled on the Total 13B study, 208 were evaluable herein. We excluded one patient who died shortly after diagnosis and did not have a remission DNA sample, five patients with Down syndrome, and one patient with cystic fibrosis because their underlying diseases can influence toxicity; we also excluded 32 patients who came off study for various reasons: induction failure (*n* = 1), ALL relapse (*n* = 12), second cancer (*n* = 3), stem cell transplantation (*n* = 3), non-compliance (*n* = 3), non-GI toxicity (*n* = 3), refused therapy (*n* = 2), death unrelated to disease or therapy (*n* = 2) or other reasons (*n* = 3) ([14](#DDS302C14)). Validation cohort consisted of patients enrolled on the multi-institution AIEOP-BFM-ALL 2000 protocol. Post-remission consolidation therapy (protocol M) for the AIEOP-BFM-ALL 2000 protocol consisted of 8 weeks of treatment with four administrations of high-dose methotrexate (2 g/m^2^ over 24 h followed by leucovorin rescue) and daily mercaptopurine (25 mg/m^2^/day for 8 weeks) ([39](#DDS302C39)). Sixty-seven patients without Down syndrome who developed Grade 3–4 mucositis and/or diarrhea during consolidation therapy served as cases and each of them was matched with two controls from the same protocol based on sex, age, ALL lineage and ALL risk classification. The toxicity was classified by the NCI criteria ([39](#DDS302C39)).

### Genotypes in patients

Genotyping of St Jude patients was performed for the major non-functional variant alleles of TPMT (*TPMT*2*, *TPMT*3A* and *TPMT*3C*, defined by SNPs rs1142345, rs1800460 and rs1800462) using methods that we have previously described in detail ([6](#DDS302C6)). Genotyping of SNP rs11045879 in *SLCO1B1* in St Jude patients was obtained from Affymetrix 500K Array Set and Affymetrix Genome-Wide Human SNP Array 6.0. SNPs rs2413739 of *PACSIN2* for St Jude patients was genotyped independently with two different methods, by DNAPrint Genomics (Sarasota, FL) and by iPlex Sequenom assay done at the University of Chicago. Genotyping of the Italian validation cohort was performed at the University of Washington in St Louis, by Sequenom technology. Quality control of genotyping was verified by repeated analysis of the same samples, using different genotyping approaches; moreover, in the validation cohort, in addition to the SNPs in *TPMT, SLCO1B1* and *PACSIN2* genotyped in the St Jude cohort, two additional *PACSIN2* SNPs were genotyped: rs737782 and rs5996520, which were, respectively, in LD or not in LD with *PACSIN2* rs2413739.

### Knock-down of PACSIN2 in NALM6 heterologously expressing TPMT

The B lineage human leukemia cell line NALM6 was used to study effects of modulation of *PACSIN2* expression on TPMT activity.

The heterologous expression of *TPMT*1* and *TPMT*3A* proteins was obtained by transfection using Nucleofector (Lonza) of native NALM6 with the appropriate plasmid (Origene), according to the manufacturer instructions. For selection of stably transfected cells, after transfection, culture medium supplemented with G418 was used.

*PACSIN2* knockdown was achieved in NALM6 heterologously expressing TPMT by RNA interference using a lentiviral vector-based shRNA approach from the MISSION™ TRC-Hs library (Sigma). Lentiviral particles corresponding to the MISSION shRNA SHVRS-[NM_007229](https://www.ncbi.nlm.nih.gov/nuccore/NM_007229) target set were used as well as the MISSION Non-Target shRNA control. The specificity and efficacy of the shRNA PACSIN2 procedure were controlled by western blotting and real-time PCR, after transduction and puromycin selection.

TPMT activity was measured in these cell lines using the previously described radiochemical assay ([11](#DDS302C11),[35](#DDS302C35)). *PACSIN2* and *TPMT* mRNA were quantified by real-time PCR using the commercial TaqMan Gene Expression Assay (Hs00200589_m1 and Hs00909010_g1, respectively) from Applied Biosystems.

### Microarray analysis

Global gene expression in NALM6 stably transfected with shRNA sequences for the knock-down of PACSIN2 or scrambled sequences (control) were evaluated using microarray analysis. Total RNA of cells harvested during logarithmic growth was isolated using Trizol reagent. For quality control, RNA purity and integrity were evaluated by measuring the OD 260/280 ratio and analyzed on Agilent Bioanalyzer. Gene expression data was measured for 30 708 transcripts by the use of the Affymetrix Human Genome U133 Plus 2.0 Array, according to the manufacturer's instructions. Validation of a panel of differentially expressed genes was performed using gene expression real-time PCR assays from Applied Biosystem (Hs00899130_m1 for *CSMD1*, Hs00228336_m1 for *C13orf18/KIAA0266L/PPP1R2P4*, Hs00211220_m1 for *SH3GLB1*).

### Statistical analysis

Analysis was done using the software R. To identify trans-genetic factors associated with TPMT activity in the HapMap cell lines, a linear mixed effects model adjusted for relatedness was used, with log10 of TPMT activity as the dependent variable and gene expression and SNPs as the independent variables. Trans-genes were selected if they had SNPs associated with TPMT activity, using a cut-off of 5 × 10^−4^ for the *P*-value and then ranked according to the *P*-value for the association of gene expression with TPMT activity.

Effects of candidate SNPs on TPMT activity and GI toxicity in St Jude patients were evaluated by linear (for TPMT activity) or logistic regression (for toxicity) models, with formula phenotype–genotype; *P*-values are from the ANOVA test applied to these models, evaluating an additive effect of the genotype on the phenotype of interest. To check for the potential confounding effects of ancestry, analysis was performed even with models adjusted for ethnicity, determined as self-reported race [we have previously shown that in this population of patients, self-reported race was highly concordant with ethnicity determined using ancestry informative markers ([14](#DDS302C14))]. CART analysis was used to evaluate interaction between genetic factors associated with toxicity in St Jude patients.

Validation of the effect of genotypes on GI toxicity in the Italian cohort was done using logistic regression models, with formula phenotype–genotype and *P*-values are from the ANOVA test applied to these models.

Comparison of measurements (e.g. TPMT activity, gene mRNA expression) in different genetically modified NALM6 cell lines was performed by Welch's *t*-test and *P*-values were adjusted for multiple testing by Holm's method ([40](#DDS302C40)).

Gene expression data from the Affymetrix chip was filtered by setting detection >500 and considering only probe sets called present by the MAS5 algorithm in at least 50% of samples. Selected gene signal values were log-2 transformed and normalized by the IQR method (so that inter-quartile ranges on all chips are set to their widest range), as implemented in the pre-process function of the LPE R package. The significance of the expression data was determined using the LPE test from the same R package and fold change, in which the null hypothesis was that no difference exists among the two groups ([41](#DDS302C41)). The FDR was controlled by adjusting *P*-values using the Benjamini–Hochberg algorithm ([42](#DDS302C42)). Pathway analysis of the genes differentially expressed was performed using MetaCore/GeneGo; enrichment analysis was performed evaluating GeneGo pathways, GeneOntology processes and GeneGo process networks.

## SUPPLEMENTARY MATERIAL

[Supplementary Material is available at HMG online.](http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/dds302/-/DC1)

## ACKNOWLEDGEMENTS

We thank the patients and their parents for their participation in this study and our clinical staff for providing outstanding protocol-based patient care. We thank Natalya Lenchik, Yan Wang, Siamac Salehy, Margaret Needham, May Chung and Yaqin Chu for their outstanding technical assistance. We thank Nancy Kornegay and Mark Wilkinson for their computer and database expertise, the staff from the St Jude Hartwell Center for Bioinformatics and Biotechnology and in particular Jay Morris, Geoff Neale and Jennifer Peters for assistance with microarray and *in vitro* experiments, and the staff of St Jude Biomedical Communications and in particular Julie Groff for the preparation of the figures.

*Conflict of Interest statement.* W.E.E. is an inventor on a patent awarded to SJCRH for the molecular diagnosis of TPMT deficiency based on the TPMT genotype and has received patent royalties. M.V.R. has also received such royalties. The other authors declared no conflict of interest.

## FUNDING

Funding sources include grants from the National Institutes of Health (R37 CA36401 to W.E.E., M.V.R., C.-H.P.; R01 CA78224 to W.E.E., M.V.R., C.-H.P.); NIH Pharmacogenomics Research Grant (U01 GM 92666 to M.V.R., W.E.E., C.C., W.Y., C.-H.P.), NCI Cancer Center Support Grant to St Jude (CA21765), F32 CA141752 to S.W.P., the American Lebanese Syrian Associated Charities (ALSAC) and the Associazione Genitori Malati Emopatici Neoplastici Friuli Venezia Giulia (AGMEN).

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Pui C.H., Carroll W.L., Meshinchi S., Arceci R.J. Biology, risk stratification, and therapy of pediatric acute leukemias: an update. J. Clin. Oncol. 2012;29:551–565. doi: 10.1200/JCO.2010.30.7405.  [DOI](https://doi.org/10.1200/JCO.2010.30.7405) | [PMC free article](/articles/PMC3071256/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21220611/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Oncol.&title=Biology,%20risk%20stratification,%20and%20therapy%20of%20pediatric%20acute%20leukemias:%20an%20update&author=C.H.%20Pui&author=W.L.%20Carroll&author=S.%20Meshinchi&author=R.J.%20Arceci&volume=29&publication_year=2012&pages=551-565&pmid=21220611&doi=10.1200/JCO.2010.30.7405&)

2. Pui C.H., Evans W.E. Treatment of acute lymphoblastic leukemia. N. Engl. J. Med. 2006;354:166–178. doi: 10.1056/NEJMra052603.  [DOI](https://doi.org/10.1056/NEJMra052603) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16407512/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%20Engl.%20J.%20Med.&title=Treatment%20of%20acute%20lymphoblastic%20leukemia&author=C.H.%20Pui&author=W.E.%20Evans&volume=354&publication_year=2006&pages=166-178&pmid=16407512&doi=10.1056/NEJMra052603&)

3. Paugh S.W., Stocco G., McCorkle J.R., Diouf B., Crews K.R., Evans W.E. Cancer pharmacogenomics. Clin. Pharmacol. Ther. 2011;90:461–466. doi: 10.1038/clpt.2011.126.  [DOI](https://doi.org/10.1038/clpt.2011.126) | [PMC free article](/articles/PMC3646359/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21796115/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Cancer%20pharmacogenomics&author=S.W.%20Paugh&author=G.%20Stocco&author=J.R.%20McCorkle&author=B.%20Diouf&author=K.R.%20Crews&volume=90&publication_year=2011&pages=461-466&pmid=21796115&doi=10.1038/clpt.2011.126&)

4. Wang L., Weinshilboum R. Thiopurine S-methyltransferase pharmacogenetics: insights, challenges and future directions. Oncogene. 2006;25:1629–1638. doi: 10.1038/sj.onc.1209372.  [DOI](https://doi.org/10.1038/sj.onc.1209372) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16550163/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncogene&title=Thiopurine%20S-methyltransferase%20pharmacogenetics:%20insights,%20challenges%20and%20future%20directions&author=L.%20Wang&author=R.%20Weinshilboum&volume=25&publication_year=2006&pages=1629-1638&pmid=16550163&doi=10.1038/sj.onc.1209372&)

5. Krynetski E., Evans W.E. Drug methylation in cancer therapy: lessons from the TPMT polymorphism. Oncogene. 2003;22:7403–7413. doi: 10.1038/sj.onc.1206944.  [DOI](https://doi.org/10.1038/sj.onc.1206944) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14576848/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncogene&title=Drug%20methylation%20in%20cancer%20therapy:%20lessons%20from%20the%20TPMT%20polymorphism&author=E.%20Krynetski&author=W.E.%20Evans&volume=22&publication_year=2003&pages=7403-7413&pmid=14576848&doi=10.1038/sj.onc.1206944&)

6. Relling M.V., Gardner E.E., Sandborn W.J., Schmiegelow K., Pui C.H., Yee S.W., Stein C.M., Carrillo M., Evans W.E., Klein T.E. Clinical Pharmacogenetics Implementation Consortium guidelines for thiopurine methyltransferase genotype and thiopurine dosing. Clin. Pharmacol. Ther. 2011;89:387–391. doi: 10.1038/clpt.2010.320.  [DOI](https://doi.org/10.1038/clpt.2010.320) | [PMC free article](/articles/PMC3098761/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21270794/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Clinical%20Pharmacogenetics%20Implementation%20Consortium%20guidelines%20for%20thiopurine%20methyltransferase%20genotype%20and%20thiopurine%20dosing&author=M.V.%20Relling&author=E.E.%20Gardner&author=W.J.%20Sandborn&author=K.%20Schmiegelow&author=C.H.%20Pui&volume=89&publication_year=2011&pages=387-391&pmid=21270794&doi=10.1038/clpt.2010.320&)

7. Relling M.V., Hancock M.L., Rivera G.K., Sandlund J.T., Ribeiro R.C., Krynetski E.Y., Pui C.H., Evans W.E. Mercaptopurine therapy intolerance and heterozygosity at the thiopurine S-methyltransferase gene locus. J. Natl. Cancer. Inst. 1999;91:2001–2008. doi: 10.1093/jnci/91.23.2001.  [DOI](https://doi.org/10.1093/jnci/91.23.2001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10580024/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Natl.%20Cancer.%20Inst.&title=Mercaptopurine%20therapy%20intolerance%20and%20heterozygosity%20at%20the%20thiopurine%20S-methyltransferase%20gene%20locus&author=M.V.%20Relling&author=M.L.%20Hancock&author=G.K.%20Rivera&author=J.T.%20Sandlund&author=R.C.%20Ribeiro&volume=91&publication_year=1999&pages=2001-2008&pmid=10580024&doi=10.1093/jnci/91.23.2001&)

8. Trevino L.R., Shimasaki N., Yang W., Panetta J.C., Cheng C., Pei D., Chan D., Sparreboom A., Giacomini K.M., Pui C.H., et al. Germline genetic variation in an organic anion transporter polypeptide associated with methotrexate pharmacokinetics and clinical effects. J. Clin. Oncol. 2009;27:5972–5978. doi: 10.1200/JCO.2008.20.4156.  [DOI](https://doi.org/10.1200/JCO.2008.20.4156) | [PMC free article](/articles/PMC2793040/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19901119/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Oncol.&title=Germline%20genetic%20variation%20in%20an%20organic%20anion%20transporter%20polypeptide%20associated%20with%20methotrexate%20pharmacokinetics%20and%20clinical%20effects&author=L.R.%20Trevino&author=N.%20Shimasaki&author=W.%20Yang&author=J.C.%20Panetta&author=C.%20Cheng&volume=27&publication_year=2009&pages=5972-5978&pmid=19901119&doi=10.1200/JCO.2008.20.4156&)

9. Peters E.J., Kraja A.T., Lin S.J., Yen-Revollo J.L., Marsh S., Province M.A., McLeod H.L. Association of thymidylate synthase variants with 5-fluorouracil cytotoxicity. Pharmacogenet. Genomics. 2009;19:399–401. doi: 10.1097/FPC.0b013e328329fdec.  [DOI](https://doi.org/10.1097/FPC.0b013e328329fdec) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19339911/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet.%20Genomics&title=Association%20of%20thymidylate%20synthase%20variants%20with%205-fluorouracil%20cytotoxicity&author=E.J.%20Peters&author=A.T.%20Kraja&author=S.J.%20Lin&author=J.L.%20Yen-Revollo&author=S.%20Marsh&volume=19&publication_year=2009&pages=399-401&pmid=19339911&doi=10.1097/FPC.0b013e328329fdec&)

10. Wheeler H.E., Dolan M.E. Lymphoblastoid cell lines in pharmacogenomic discovery and clinical translation. Pharmacogenomics. 2012;13:55–70. doi: 10.2217/pgs.11.121.  [DOI](https://doi.org/10.2217/pgs.11.121) | [PMC free article](/articles/PMC3292907/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22176622/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Lymphoblastoid%20cell%20lines%20in%20pharmacogenomic%20discovery%20and%20clinical%20translation&author=H.E.%20Wheeler&author=M.E.%20Dolan&volume=13&publication_year=2012&pages=55-70&pmid=22176622&doi=10.2217/pgs.11.121&)

11. Jones T.S., Yang W., Evans W.E., Relling M.V. Using HapMap tools in pharmacogenomic discovery: the thiopurine methyltransferase polymorphism. Clin. Pharmacol. Ther. 2007;81:729–734. doi: 10.1038/sj.clpt.6100135.  [DOI](https://doi.org/10.1038/sj.clpt.6100135) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17329987/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Using%20HapMap%20tools%20in%20pharmacogenomic%20discovery:%20the%20thiopurine%20methyltransferase%20polymorphism&author=T.S.%20Jones&author=W.%20Yang&author=W.E.%20Evans&author=M.V.%20Relling&volume=81&publication_year=2007&pages=729-734&pmid=17329987&doi=10.1038/sj.clpt.6100135&)

12. Gamazon E.R., Huang R.S., Cox N.J., Dolan M.E. Chemotherapeutic drug susceptibility associated SNPs are enriched in expression quantitative trait loci. Proc. Natl. Acad. Sci. USA. 2010;107:9287–9292. doi: 10.1073/pnas.1001827107.  [DOI](https://doi.org/10.1073/pnas.1001827107) | [PMC free article](/articles/PMC2889115/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20442332/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&title=Chemotherapeutic%20drug%20susceptibility%20associated%20SNPs%20are%20enriched%20in%20expression%20quantitative%20trait%20loci&author=E.R.%20Gamazon&author=R.S.%20Huang&author=N.J.%20Cox&author=M.E.%20Dolan&volume=107&publication_year=2010&pages=9287-9292&pmid=20442332&doi=10.1073/pnas.1001827107&)

13. Nicolae D.L., Gamazon E., Zhang W., Duan S., Dolan M.E., Cox N.J. Trait-associated SNPs are more likely to be eQTLs: annotation to enhance discovery from GWAS. PLoS Genet. 2010;6:e1000888. doi: 10.1371/journal.pgen.1000888.  [DOI](https://doi.org/10.1371/journal.pgen.1000888) | [PMC free article](/articles/PMC2848547/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20369019/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet.&title=Trait-associated%20SNPs%20are%20more%20likely%20to%20be%20eQTLs:%20annotation%20to%20enhance%20discovery%20from%20GWAS&author=D.L.%20Nicolae&author=E.%20Gamazon&author=W.%20Zhang&author=S.%20Duan&author=M.E.%20Dolan&volume=6&publication_year=2010&pages=e1000888&pmid=20369019&doi=10.1371/journal.pgen.1000888&)

14. Kishi S., Cheng C., French D., Pei D., Das S., Cook E.H., Hijiya N., Rizzari C., Rosner G.L., Frudakis T., et al. Ancestry and pharmacogenetics of antileukemic drug toxicity. Blood. 2007;109:4151–4157. doi: 10.1182/blood-2006-10-054528.  [DOI](https://doi.org/10.1182/blood-2006-10-054528) | [PMC free article](/articles/PMC1885506/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17264302/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Ancestry%20and%20pharmacogenetics%20of%20antileukemic%20drug%20toxicity&author=S.%20Kishi&author=C.%20Cheng&author=D.%20French&author=D.%20Pei&author=S.%20Das&volume=109&publication_year=2007&pages=4151-4157&pmid=17264302&doi=10.1182/blood-2006-10-054528&)

15. Tai H.L., Fessing M.Y., Bonten E.J., Yanishevsky Y., d'Azzo A., Krynetski E.Y., Evans W.E. Enhanced proteasomal degradation of mutant human thiopurine S-methyltransferase (TPMT) in mammalian cells: mechanism for TPMT protein deficiency inherited by TPMT*2, TPMT*3A, TPMT*3B or TPMT*3C. Pharmacogenetics. 1999;9:641–650. doi: 10.1097/01213011-199910000-00011.  [DOI](https://doi.org/10.1097/01213011-199910000-00011) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10591545/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Enhanced%20proteasomal%20degradation%20of%20mutant%20human%20thiopurine%20S-methyltransferase%20(TPMT)%20in%20mammalian%20cells:%20mechanism%20for%20TPMT%20protein%20deficiency%20inherited%20by%20TPMT*2,%20TPMT*3A,%20TPMT*3B%20or%20TPMT*3C&author=H.L.%20Tai&author=M.Y.%20Fessing&author=E.J.%20Bonten&author=Y.%20Yanishevsky&author=A.%20d'Azzo&volume=9&publication_year=1999&pages=641-650&pmid=10591545&doi=10.1097/01213011-199910000-00011&)

16. Stanulla M., Schaeffeler E., Flohr T., Cario G., Schrauder A., Zimmermann M., Welte K., Ludwig W.D., Bartram C.R., Zanger U.M., et al. Thiopurine methyltransferase (TPMT) genotype and early treatment response to mercaptopurine in childhood acute lymphoblastic leukemia. J.A.M.A. 2005;293:1485–1489. doi: 10.1001/jama.293.12.1485.  [DOI](https://doi.org/10.1001/jama.293.12.1485) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15784872/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.A.M.A.&title=Thiopurine%20methyltransferase%20(TPMT)%20genotype%20and%20early%20treatment%20response%20to%20mercaptopurine%20in%20childhood%20acute%20lymphoblastic%20leukemia&author=M.%20Stanulla&author=E.%20Schaeffeler&author=T.%20Flohr&author=G.%20Cario&author=A.%20Schrauder&volume=293&publication_year=2005&pages=1485-1489&pmid=15784872&doi=10.1001/jama.293.12.1485&)

17. Kessels M.M., Qualmann B. The syndapin protein family: linking membrane trafficking with the cytoskeleton. J. Cell Sci. 2004;117:3077–3086. doi: 10.1242/jcs.01290.  [DOI](https://doi.org/10.1242/jcs.01290) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15226389/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Cell%20Sci.&title=The%20syndapin%20protein%20family:%20linking%20membrane%20trafficking%20with%20the%20cytoskeleton&author=M.M.%20Kessels&author=B.%20Qualmann&volume=117&publication_year=2004&pages=3077-3086&pmid=15226389&doi=10.1242/jcs.01290&)

18. Modregger J., Ritter B., Witter B., Paulsson M., Plomann M. All three PACSIN isoforms bind to endocytic proteins and inhibit endocytosis. J. Cell Sci. 2000;113(Pt 24):4511–4521. doi: 10.1242/jcs.113.24.4511.  [DOI](https://doi.org/10.1242/jcs.113.24.4511) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11082044/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Cell%20Sci.&title=All%20three%20PACSIN%20isoforms%20bind%20to%20endocytic%20proteins%20and%20inhibit%20endocytosis&author=J.%20Modregger&author=B.%20Ritter&author=B.%20Witter&author=M.%20Paulsson&author=M.%20Plomann&volume=113&issue=Pt%2024&publication_year=2000&pages=4511-4521&pmid=11082044&doi=10.1242/jcs.113.24.4511&)

19. Meng H., Tian L., Zhou J., Li Z., Jiao X., Li W.W., Plomann M., Xu Z., Lisanti M.P., Wang C., et al. PACSIN 2 represses cellular migration through direct association with cyclin D1 but not its alternate splice form cyclin D1b. Cell Cycle. 2011;10:73–81. doi: 10.4161/cc.10.1.14243.  [DOI](https://doi.org/10.4161/cc.10.1.14243) | [PMC free article](/articles/PMC3048077/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21200149/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Cycle&title=PACSIN%202%20represses%20cellular%20migration%20through%20direct%20association%20with%20cyclin%20D1%20but%20not%20its%20alternate%20splice%20form%20cyclin%20D1b&author=H.%20Meng&author=L.%20Tian&author=J.%20Zhou&author=Z.%20Li&author=X.%20Jiao&volume=10&publication_year=2011&pages=73-81&pmid=21200149&doi=10.4161/cc.10.1.14243&)

20. Szyniarowski P., Corcelle-Termeau E., Farkas T., Hoyer-Hansen M., Nylandsted J., Kallunki T., Jaattela M. A comprehensive siRNA screen for kinases that suppress macroautophagy in optimal growth conditions. Autophagy. 2011;7:892–903. doi: 10.4161/auto.7.8.15770.  [DOI](https://doi.org/10.4161/auto.7.8.15770) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21508686/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Autophagy&title=A%20comprehensive%20siRNA%20screen%20for%20kinases%20that%20suppress%20macroautophagy%20in%20optimal%20growth%20conditions&author=P.%20Szyniarowski&author=E.%20Corcelle-Termeau&author=T.%20Farkas&author=M.%20Hoyer-Hansen&author=J.%20Nylandsted&volume=7&publication_year=2011&pages=892-903&pmid=21508686&doi=10.4161/auto.7.8.15770&)

21. Qualmann B., Kelly R.B. Syndapin isoforms participate in receptor-mediated endocytosis and actin organization. J. Cell. Biol. 2000;148:1047–1062. doi: 10.1083/jcb.148.5.1047.  [DOI](https://doi.org/10.1083/jcb.148.5.1047) | [PMC free article](/articles/PMC2174535/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10704453/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Cell.%20Biol.&title=Syndapin%20isoforms%20participate%20in%20receptor-mediated%20endocytosis%20and%20actin%20organization&author=B.%20Qualmann&author=R.B.%20Kelly&volume=148&publication_year=2000&pages=1047-1062&pmid=10704453&doi=10.1083/jcb.148.5.1047&)

22. Wang Q., Navarro M.V., Peng G., Molinelli E., Goh S.L., Judson B.L., Rajashankar K.R., Sondermann H. Molecular mechanism of membrane constriction and tubulation mediated by the F-BAR protein Pacsin/Syndapin. Proc. Natl. Acad. Sci. U. S. A. 2009;106:12700–12705. doi: 10.1073/pnas.0902974106.  [DOI](https://doi.org/10.1073/pnas.0902974106) | [PMC free article](/articles/PMC2722316/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19549836/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A.&title=Molecular%20mechanism%20of%20membrane%20constriction%20and%20tubulation%20mediated%20by%20the%20F-BAR%20protein%20Pacsin/Syndapin&author=Q.%20Wang&author=M.V.%20Navarro&author=G.%20Peng&author=E.%20Molinelli&author=S.L.%20Goh&volume=106&publication_year=2009&pages=12700-12705&pmid=19549836&doi=10.1073/pnas.0902974106&)

23. Takahashi Y., Coppola D., Matsushita N., Cualing H.D., Sun M., Sato Y., Liang C., Jung J.U., Cheng J.Q., Mule J.J., et al. Bif-1 interacts with Beclin 1 through UVRAG and regulates autophagy and tumorigenesis. Nat. Cell. Biol. 2007;9:1142–1151. doi: 10.1038/ncb1634.  [DOI](https://doi.org/10.1038/ncb1634) | [PMC free article](/articles/PMC2254521/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17891140/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Cell.%20Biol.&title=Bif-1%20interacts%20with%20Beclin%201%20through%20UVRAG%20and%20regulates%20autophagy%20and%20tumorigenesis&author=Y.%20Takahashi&author=D.%20Coppola&author=N.%20Matsushita&author=H.D.%20Cualing&author=M.%20Sun&volume=9&publication_year=2007&pages=1142-1151&pmid=17891140&doi=10.1038/ncb1634&)

24. Takahashi Y., Meyerkord C.L., Hori T., Runkle K., Fox T.E., Kester M., Loughran T.P., Wang H.G. Bif-1 regulates Atg9 trafficking by mediating the fission of Golgi membranes during autophagy. Autophagy. 2011;7:61–73. doi: 10.4161/auto.7.1.14015.  [DOI](https://doi.org/10.4161/auto.7.1.14015) | [PMC free article](/articles/PMC3039731/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21068542/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Autophagy&title=Bif-1%20regulates%20Atg9%20trafficking%20by%20mediating%20the%20fission%20of%20Golgi%20membranes%20during%20autophagy&author=Y.%20Takahashi&author=C.L.%20Meyerkord&author=T.%20Hori&author=K.%20Runkle&author=T.E.%20Fox&volume=7&publication_year=2011&pages=61-73&pmid=21068542&doi=10.4161/auto.7.1.14015&)

25. Behrends C., Sowa M.E., Gygi S.P., Harper J.W. Network organization of the human autophagy system. Nature. 2010;466:68–76. doi: 10.1038/nature09204.  [DOI](https://doi.org/10.1038/nature09204) | [PMC free article](/articles/PMC2901998/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20562859/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Network%20organization%20of%20the%20human%20autophagy%20system&author=C.%20Behrends&author=M.E.%20Sowa&author=S.P.%20Gygi&author=J.W.%20Harper&volume=466&publication_year=2010&pages=68-76&pmid=20562859&doi=10.1038/nature09204&)

26. Li F., Wang L., Burgess R.J., Weinshilboum R.M. Thiopurine S-methyltransferase pharmacogenetics: autophagy as a mechanism for variant allozyme degradation. Pharmacogenet. Genomics. 2008;18:1083–1094. doi: 10.1097/FPC.0b013e328313e03f.  [DOI](https://doi.org/10.1097/FPC.0b013e328313e03f) | [PMC free article](/articles/PMC2583164/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18820593/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet.%20Genomics&title=Thiopurine%20S-methyltransferase%20pharmacogenetics:%20autophagy%20as%20a%20mechanism%20for%20variant%20allozyme%20degradation&author=F.%20Li&author=L.%20Wang&author=R.J.%20Burgess&author=R.M.%20Weinshilboum&volume=18&publication_year=2008&pages=1083-1094&pmid=18820593&doi=10.1097/FPC.0b013e328313e03f&)

27. Raney B.J., Cline M.S., Rosenbloom K.R., Dreszer T.R., Learned K., Barber G.P., Meyer L.R., Sloan C.A., Malladi V.S., Roskin K.M., et al. ENCODE whole-genome data in the UCSC genome browser (2011 update) Nucleic Acids Res. 2011;39:D871–D875. doi: 10.1093/nar/gkq1017.  [DOI](https://doi.org/10.1093/nar/gkq1017) | [PMC free article](/articles/PMC3013645/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21037257/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&title=ENCODE%20whole-genome%20data%20in%20the%20UCSC%20genome%20browser%20(2011%20update)&author=B.J.%20Raney&author=M.S.%20Cline&author=K.R.%20Rosenbloom&author=T.R.%20Dreszer&author=K.%20Learned&volume=39&publication_year=2011&pages=D871-D875&pmid=21037257&doi=10.1093/nar/gkq1017&)

28. Rosenbloom K.R., Dreszer T.R., Pheasant M., Barber G.P., Meyer L.R., Pohl A., Raney B.J., Wang T., Hinrichs A.S., Zweig A.S., et al. ENCODE whole-genome data in the UCSC Genome Browser. Nucleic Acids Res. 2010;38:D620–D625. doi: 10.1093/nar/gkp961.  [DOI](https://doi.org/10.1093/nar/gkp961) | [PMC free article](/articles/PMC2808953/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19920125/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&title=ENCODE%20whole-genome%20data%20in%20the%20UCSC%20Genome%20Browser&author=K.R.%20Rosenbloom&author=T.R.%20Dreszer&author=M.%20Pheasant&author=G.P.%20Barber&author=L.R.%20Meyer&volume=38&publication_year=2010&pages=D620-D625&pmid=19920125&doi=10.1093/nar/gkp961&)

29. Kent W.J., Sugnet C.W., Furey T.S., Roskin K.M., Pringle T.H., Zahler A.M., Haussler D. The human genome browser at UCSC. Genome Res. 2002;12:996–1006. doi: 10.1101/gr.229102.  [DOI](https://doi.org/10.1101/gr.229102) | [PMC free article](/articles/PMC186604/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12045153/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res.&title=The%20human%20genome%20browser%20at%20UCSC&author=W.J.%20Kent&author=C.W.%20Sugnet&author=T.S.%20Furey&author=K.M.%20Roskin&author=T.H.%20Pringle&volume=12&publication_year=2002&pages=996-1006&pmid=12045153&doi=10.1101/gr.229102&)

30. McLeod H.L., Coulthard S., Thomas A.E., Pritchard S.C., King D.J., Richards S.M., Eden O.B., Hall A.G., Gibson B.E. Analysis of thiopurine methyltransferase variant alleles in childhood acute lymphoblastic leukaemia. Br. J. Haematol. 1999;105:696–700. doi: 10.1046/j.1365-2141.1999.01416.x.  [DOI](https://doi.org/10.1046/j.1365-2141.1999.01416.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10354134/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Haematol&title=Analysis%20of%20thiopurine%20methyltransferase%20variant%20alleles%20in%20childhood%20acute%20lymphoblastic%20leukaemia&author=H.L.%20McLeod&author=S.%20Coulthard&author=A.E.%20Thomas&author=S.C.%20Pritchard&author=D.J.%20King&volume=105&publication_year=1999&pages=696-700&pmid=10354134&doi=10.1046/j.1365-2141.1999.01416.x&)

31. Roberts R.L., Gearry R.B., Bland M.V., Sies C.W., George P.M., Burt M., Marinaki A.M., Arenas M., Barclay M.L., Kennedy M.A. Trinucleotide repeat variants in the promoter of the thiopurine S-methyltransferase gene of patients exhibiting ultra-high enzyme activity. Pharmacogenet. Genomics. 2008;18:434–438. doi: 10.1097/FPC.0b013e3282f85e47.  [DOI](https://doi.org/10.1097/FPC.0b013e3282f85e47) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18408566/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet.%20Genomics&title=Trinucleotide%20repeat%20variants%20in%20the%20promoter%20of%20the%20thiopurine%20S-methyltransferase%20gene%20of%20patients%20exhibiting%20ultra-high%20enzyme%20activity&author=R.L.%20Roberts&author=R.B.%20Gearry&author=M.V.%20Bland&author=C.W.%20Sies&author=P.M.%20George&volume=18&publication_year=2008&pages=434-438&pmid=18408566&doi=10.1097/FPC.0b013e3282f85e47&)

32. de Kreuk B.J., Nethe M., Fernandez-Borja M., Anthony E.C., Hensbergen P.J., Deelder A.M., Plomann M., Hordijk P.L. The F-BAR domain protein PACSIN2 associates with Rac1 and regulates cell spreading and migration. J. Cell Sci. 2011;124:2375–2388. doi: 10.1242/jcs.080630.  [DOI](https://doi.org/10.1242/jcs.080630) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21693584/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Cell%20Sci.&title=The%20F-BAR%20domain%20protein%20PACSIN2%20associates%20with%20Rac1%20and%20regulates%20cell%20spreading%20and%20migration&author=B.J.%20de%20Kreuk&author=M.%20Nethe&author=M.%20Fernandez-Borja&author=E.C.%20Anthony&author=P.J.%20Hensbergen&volume=124&publication_year=2011&pages=2375-2388&pmid=21693584&doi=10.1242/jcs.080630&)

33. Tiede I., Fritz G., Strand S., Poppe D., Dvorsky R., Strand D., Lehr H.A., Wirtz S., Becker C., Atreya R., et al. CD28-dependent Rac1 activation is the molecular target of azathioprine in primary human CD4+ T lymphocytes. J. Clin. Invest. 2003;111:1133–1145. doi: 10.1172/JCI16432.  [DOI](https://doi.org/10.1172/JCI16432) | [PMC free article](/articles/PMC152932/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12697733/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Invest.&title=CD28-dependent%20Rac1%20activation%20is%20the%20molecular%20target%20of%20azathioprine%20in%20primary%20human%20CD4+%20T%20lymphocytes&author=I.%20Tiede&author=G.%20Fritz&author=S.%20Strand&author=D.%20Poppe&author=R.%20Dvorsky&volume=111&publication_year=2003&pages=1133-1145&pmid=12697733&doi=10.1172/JCI16432&)

34. Stocco G., Cheok M.H., Crews K.R., Dervieux T., French D., Pei D., Yang W., Cheng C., Pui C.H., Relling M.V., et al. Genetic polymorphism of inosine triphosphate pyrophosphatase is a determinant of mercaptopurine metabolism and toxicity during treatment for acute lymphoblastic leukemia. Clin. Pharmacol. Ther. 2009;85:164–172. doi: 10.1038/clpt.2008.154.  [DOI](https://doi.org/10.1038/clpt.2008.154) | [PMC free article](/articles/PMC2762405/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18685564/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Genetic%20polymorphism%20of%20inosine%20triphosphate%20pyrophosphatase%20is%20a%20determinant%20of%20mercaptopurine%20metabolism%20and%20toxicity%20during%20treatment%20for%20acute%20lymphoblastic%20leukemia&author=G.%20Stocco&author=M.H.%20Cheok&author=K.R.%20Crews&author=T.%20Dervieux&author=D.%20French&volume=85&publication_year=2009&pages=164-172&pmid=18685564&doi=10.1038/clpt.2008.154&)

35. McLeod H.L., Lin J.S., Scott E.P., Pui C.H., Evans W.E. Thiopurine methyltransferase activity in American white subjects and black subjects. Clin. Pharmacol. Ther. 1994;55:15–20. doi: 10.1038/clpt.1994.4.  [DOI](https://doi.org/10.1038/clpt.1994.4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8299312/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Thiopurine%20methyltransferase%20activity%20in%20American%20white%20subjects%20and%20black%20subjects&author=H.L.%20McLeod&author=J.S.%20Lin&author=E.P.%20Scott&author=C.H.%20Pui&author=W.E.%20Evans&volume=55&publication_year=1994&pages=15-20&pmid=8299312&doi=10.1038/clpt.1994.4&)

36. Pui C.H., Pei D., Sandlund J.T., Ribeiro R.C., Rubnitz J.E., Raimondi S.C., Onciu M., Campana D., Kun L.E., Jeha S., et al. Long-term results of St Jude Total Therapy Studies 11, 12, 13A, 13B, and 14 for childhood acute lymphoblastic leukemia. Leukemia. 2010;24:371–382. doi: 10.1038/leu.2009.252.  [DOI](https://doi.org/10.1038/leu.2009.252) | [PMC free article](/articles/PMC2820159/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20010620/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Leukemia&title=Long-term%20results%20of%20St%20Jude%20Total%20Therapy%20Studies%2011,%2012,%2013A,%2013B,%20and%2014%20for%20childhood%20acute%20lymphoblastic%20leukemia&author=C.H.%20Pui&author=D.%20Pei&author=J.T.%20Sandlund&author=R.C.%20Ribeiro&author=J.E.%20Rubnitz&volume=24&publication_year=2010&pages=371-382&pmid=20010620&doi=10.1038/leu.2009.252&)

37. Pui C.H., Campana D., Pei D., Bowman W.P., Sandlund J.T., Kaste S.C., Ribeiro R.C., Rubnitz J.E., Raimondi S.C., Onciu M., et al. Treating childhood acute lymphoblastic leukemia without cranial irradiation. N. Engl. J. Med. 2009;360:2730–2741. doi: 10.1056/NEJMoa0900386.  [DOI](https://doi.org/10.1056/NEJMoa0900386) | [PMC free article](/articles/PMC2754320/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19553647/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%20Engl.%20J.%20Med.&title=Treating%20childhood%20acute%20lymphoblastic%20leukemia%20without%20cranial%20irradiation&author=C.H.%20Pui&author=D.%20Campana&author=D.%20Pei&author=W.P.%20Bowman&author=J.T.%20Sandlund&volume=360&publication_year=2009&pages=2730-2741&pmid=19553647&doi=10.1056/NEJMoa0900386&)

38. Pui C.H., Sandlund J.T., Pei D., Campana D., Rivera G.K., Ribeiro R.C., Rubnitz J.E., Razzouk B.I., Howard S.C., Hudson M.M., et al. Improved outcome for children with acute lymphoblastic leukemia: results of Total Therapy Study XIIIB at St Jude Children's Research Hospital. Blood. 2004;104:2690–2696. doi: 10.1182/blood-2004-04-1616.  [DOI](https://doi.org/10.1182/blood-2004-04-1616) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15251979/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Improved%20outcome%20for%20children%20with%20acute%20lymphoblastic%20leukemia:%20results%20of%20Total%20Therapy%20Study%20XIIIB%20at%20St%20Jude%20Children's%20Research%20Hospital&author=C.H.%20Pui&author=J.T.%20Sandlund&author=D.%20Pei&author=D.%20Campana&author=G.K.%20Rivera&volume=104&publication_year=2004&pages=2690-2696&pmid=15251979&doi=10.1182/blood-2004-04-1616&)

39. Schrappe M., Valsecchi M.G., Bartram C.R., Schrauder A., Panzer-Grumayer R., Moricke A., Parasole R., Zimmermann M., Dworzak M., Buldini B., et al. Late MRD response determines relapse risk overall and in subsets of childhood T-cell ALL: results of the AIEOP-BFM-ALL 2000 study. Blood. 2011;118:2077–2084. doi: 10.1182/blood-2011-03-338707.  [DOI](https://doi.org/10.1182/blood-2011-03-338707) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21719599/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Late%20MRD%20response%20determines%20relapse%20risk%20overall%20and%20in%20subsets%20of%20childhood%20T-cell%20ALL:%20results%20of%20the%20AIEOP-BFM-ALL%202000%20study&author=M.%20Schrappe&author=M.G.%20Valsecchi&author=C.R.%20Bartram&author=A.%20Schrauder&author=R.%20Panzer-Grumayer&volume=118&publication_year=2011&pages=2077-2084&pmid=21719599&doi=10.1182/blood-2011-03-338707&)

40. Holm S. A simple sequentially rejective multiple test procedure. Scand. J. Stat. 1979;6:65–70.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Scand.%20J.%20Stat.&title=A%20simple%20sequentially%20rejective%20multiple%20test%20procedure&author=S.%20Holm&volume=6&publication_year=1979&pages=65-70&)

41. Jain N., Thatte J., Braciale T., Ley K., O'Connell M., Lee J.K. Local-pooled-error test for identifying differentially expressed genes with a small number of replicated microarrays. Bioinformatics. 2003;19:1945–1951. doi: 10.1093/bioinformatics/btg264.  [DOI](https://doi.org/10.1093/bioinformatics/btg264) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14555628/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=Local-pooled-error%20test%20for%20identifying%20differentially%20expressed%20genes%20with%20a%20small%20number%20of%20replicated%20microarrays&author=N.%20Jain&author=J.%20Thatte&author=T.%20Braciale&author=K.%20Ley&author=M.%20O'Connell&volume=19&publication_year=2003&pages=1945-1951&pmid=14555628&doi=10.1093/bioinformatics/btg264&)

42. Benjamini Y., Hochberg Y. Controlling the false discovery rate: a practical and powerful approach to multiple testing. J. R. Stat. Soc. Series B. 1995;57:289–300.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20R.%20Stat.%20Soc.%20Series%20B&title=Controlling%20the%20false%20discovery%20rate:%20a%20practical%20and%20powerful%20approach%20to%20multiple%20testing&author=Y.%20Benjamini&author=Y.%20Hochberg&volume=57&publication_year=1995&pages=289-300&)
