# Genome-wide association study of Stevens-Johnson Syndrome and Toxic Epidermal Necrolysis in Europe

## Metadata
**Authors:** Emmanuelle Génin, Martin Schumacher, Jean-Claude Roujeau, Luigi Naldi, Yvonne Liss, Rémi Kazma, Peggy Sekula, Alain Hovnanian, Maja Mockenhaupt
**Journal:** Orphanet Journal of Rare Diseases
**Date:** 2011 Jul 29
**DOI:** [10.1186/1750-1172-6-52](https://doi.org/10.1186/1750-1172-6-52)
**PMID:** 21801394
**PMCID:** PMC3173287
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3173287/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC3173287/pdf/1750-1172-6-52.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC3173287/pdf/1750-1172-6-52.pdf)

## Abstract

**Background:** 
Stevens-Johnson syndrome (SJS) and Toxic Epidermal Necrolysis (TEN) are rare but extremely severe cutaneous adverse drug reactions in which drug-specific associations with HLA-B alleles were described.

**Objectives:** 
To investigate genetic association at a genome-wide level on a large sample of SJS/TEN patients.

**Methods:** 
We performed a genome wide association study on a sample of 424 European cases and 1,881 controls selected from a Reference Control Panel.

**Results:** 
Six SNPs located in the HLA region showed significant evidence for association (OR range: 1.53-1.74). The haplotype formed by their risk allele was more associated with the disease than any of the single SNPs and was even much stronger in patients exposed to allopurinol (ORallopurinol = 7.77, 95%CI = [4.66; 12.98]). The associated haplotype is in linkage disequilibrium with the HLA-B*5801 allele known to be associated with allopurinol induced SJS/TEN in Asian populations.

**Conclusion:** 
The involvement of genetic variants located in the HLA region in SJS/TEN is confirmed in European samples, but no other locus reaches genome-wide statistical significance in this sample that is also the largest one collected so far. If some loci outside HLA play a role in SJS/TEN, their effect is thus likely to be very small.

### Background

Stevens-Johnson syndrome (SJS) and Toxic Epidermal Necrolysis (TEN) are rare but extremely severe cutaneous adverse drug reactions in which drug-specific associations with HLA-B alleles were described.

### Objectives

To investigate genetic association at a genome-wide level on a large sample of SJS/TEN patients.

### Methods

We performed a genome wide association study on a sample of 424 European cases and 1,881 controls selected from a Reference Control Panel.

### Results

Six SNPs located in the HLA region showed significant evidence for association (OR range: 1.53-1.74). The haplotype formed by their risk allele was more associated with the disease than any of the single SNPs and was even much stronger in patients exposed to allopurinol (OR_allopurinol _= 7.77, 95%CI = [4.66; 12.98]). The associated haplotype is in linkage disequilibrium with the HLA-B*5801 allele known to be associated with allopurinol induced SJS/TEN in Asian populations.

### Conclusion

The involvement of genetic variants located in the HLA region in SJS/TEN is confirmed in European samples, but no other locus reaches genome-wide statistical significance in this sample that is also the largest one collected so far. If some loci outside HLA play a role in SJS/TEN, their effect is thus likely to be very small.

## Background

Adverse drug reactions are a major public health issue as they represent an important cause of morbidity and mortality [[1](#B1)]. Skin lesions are frequent expressions of adverse drug reactions with, for some drugs, up to 10% of cutaneous reactions observed [[2](#B2),[3](#B3)]. SJS and TEN are severe cutaneous adverse reactions characterized by the development of acute exanthema which progresses towards limited (in SJS) or widespread (in TEN) blistering and erosion of the skin and mucous membranes [[4](#B4)]. SJS and TEN are thus considered to be two different forms of the same disease with TEN representing the most severe form [[5](#B5)]. The incidence of SJS/TEN is estimated to be of approximately 1-2 patients per million individuals per year [[6](#B6)]. It is thus a very rare disease but with a high morbidity and mortality (reaching up to 45% in TEN) that requires intensive treatment.

SJS/TEN is not associated with a single drug or a single group of drugs but several different drugs have been involved. A limited number of drugs however are more often associated with the disease: antibacterial sulfonamides (especially sulfamethoxazole), allopurinol, which is the most frequent drug involved [[7](#B7)], carbamazepine, lamotrigine, phenobarbital, phenytoine, non-steroidal anti-inflammatory drugs (NSAIDs) of the oxicam type and neviparine [[4](#B4),[8](#B8)]. Only a small number of individuals exposed to these "highly suspected" drugs develop the disease and a genetic susceptibility has been suggested [[9](#B9)-[12](#B12)].

An association with HLA was reported more than 20 years ago [[12](#B12),[13](#B13)]. More recently, studies in the Han Chinese population have involved the HLA-B locus with very strong drug-specific associations: the HLA-B*1502 allele was found in all carbamazepine-induced SJS/TEN patients [[14](#B14)] and the HLA-B*5801 in all allopurinol-induced SJS/TEN patients [[15](#B15)]. An investigation of HLA-B associations in European samples did not detect the association between the HLA-B*1502 allele and carbamazepine-induced SJS/TEN but did report a strong but not complete association between HLA-B*5801 and the allopurinol-induced disease [[16](#B16),[17](#B17)]. Furthermore, it has also been suggested that HLA genetic predisposition may not be the same for SJS and TEN and that it might thus be important to take into account the disease severity in association tests [[18](#B18)].

Apart from HLA, several other candidate genes have been tested for association with SJS/TEN, mostly genes involved in the immune response, in the inflammation process or in drug metabolism [[18](#B18)-[21](#B21)]. However, no association with these genes has been consistently found and it has been suggested that rather than focusing on candidate genes, a genome-wide association study might provide more insights into the genetic susceptibility of adverse drug reactions [[21](#B21)].

In the context of the RegiSCAR project (European Registry of Severe Cutaneous Adverse Reactions to Drugs and Collection of Biological Samples) DNA of 563 cases of SJS/TEN was collected. This is the largest available sample of SJS/TEN patients in the world with accurate medical information regarding the severity of the disease and the history of drug intake. These patients were genotyped at the Centre National de Genotypage (CNG) using Illumina 317 K chips and a Genome Wide Association Study (GWAS) was conducted against controls selected from the CNG European Reference Control Panel [[22](#B22)].

## Methods

### Patients

A total of 563 cases (226 males and 337 females, sex-ratio = 0.67) were included in this study. They were collected as part of the "European Registry of Severe Cutaneous Adverse Reactions" (RegiSCAR) (see [http://regiscar.uni-freiburg.de/](http://regiscar.uni-freiburg.de/)) in six countries (Austria (1 case), France (184 cases), Germany (331 cases), Israel (14), Italy (26) and The Netherlands (7)). All of them had a diagnosis of SJS (268 cases, 48%), SJS/TEN overlap (181 cases, 32%) or TEN (114 cases, 20%) validated as probable or definitive by the expert committee of RegiSCAR blindly from information on drug exposure.

For each patient, written informed consent was obtained and a blood sample was taken for genomic DNA extraction that was carried out at the CEPH-Fondation Jean Dausset (France) ([http://www.cephb.fr](http://www.cephb.fr)) until 2005 and at the biobank of the CIC-Henri Mondor / Créteil after 2005.

The 563 patients were genotyped on Illumina 317 K chips in France at the CNG ([http://www.cng.fr](http://www.cng.fr)) for a total of 318,127 SNPs (among which 309,091 were located on autosomes). Stringent quality-control (QC) was performed using Plink v1.06 [[23](#B23)] that led to the exclusion of 68 out of the 563 cases for low genotyping (MIND > 0.05) and 15,088 SNPs (14,343 autosomal SNPs) for missing data (GENO > 0.05). After QC, a total of 495 affected individuals were considered in the analysis for whom genotypes on 303,039 SNPs (294,748 located on autosomes) were available. The total genotyping rate in these 495 remaining individuals was 0.95.

### Controls

Controls were selected from the CNG European Reference Panel collected at the CNG [[22](#B22)]. This panel includes 5,847 unrelated individuals from 13 European countries who are all genotyped on Illumina 317 K chips. Since patients collected as part of the RegiSCAR project were mostly from France and Germany, we decided to consider only the 1,881 individuals from the Reference Control panel originating from these two countries (653 from Germany and 1,228 from France).

### Quality control and principal component analysis of genotype data

A second quality control of the data was performed where markers that were missing in more than 5% of the controls or had significant different missing rates between cases and controls were removed. A total of 25,834 autosomal markers were excluded.

To assess the level of population stratification in the sample, a principal component analysis (PCA) was performed using the genotypes of the 495 cases and 1,881 controls for a sub-panel of 35,232 markers obtained after pruning based on linkage disequilibrium using Plink (option indep-pairwise 50 5 0.1). The program smartPCA from the Eigenstrat package [[24](#B24),[25](#B25)] was used with the default options. A total of 71 patients were excluded from the study as they were found to be outliers. Those were mostly individuals with a suspected African or Asian ancestry. Plots of the first two principal components (PCs) are provided in Figure [1](#F1).

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/01da/3173287/faf9c2cc7efa/1750-1172-6-52-1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3173287_1750-1172-6-52-1.jpg)

Plots of top two principal components for cases (in red) and controls (in black).

As shown in the flowchart in Figure [2, a](#F2) total of 268,914 autosomal SNPs were kept for the association testing in a sample of 424 SJS/TEN patients and 1,881 controls.

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/01da/3173287/8d7411554475/1750-1172-6-52-2.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3173287_1750-1172-6-52-2.jpg)

Flowchart summarizing the different steps of the quality control of the data.

### Drug exposure

Information on drug exposure was collected by interviewing patients about their drug intakes and by consulting medical records. Reactions were considered to be potentially caused by allopurinol, carbamazepine or phenytoin if the drug has been taken between 4 and 10 days before the onset of the disease and if the drug has not been started more than 42 days before onset. This resulted in a group of 57 patients (13.4%) with potentially allopurinol-induced reactions, a group of 25 patients (5.9%) with potentially carbamazepine-induced reactions and a group of 19 patients (4.5%) with potentially phenytoin-induced reactions. Because of the low number in the latter two groups we decided to only compare the allopurinol group and the group with other drugs to the controls in this study. Note that because of the stringent time criterion used to determine allopurinol as culprit drug there might be some patients with an allopurinol-induced reaction left in the group of patients with a culprit drug other than allopurinol. For this reason, this analysis was further enforced by an additional analysis where any allopurinol exposed patient (irrelevant of time of usage) was assigned to the allopurinol group resulting in a higher sensitivity with respect to allopurinol-induced reaction while the specificity is decreased.

### Association testing

Association was tested using the PC-corrected Armitage-trend test implemented in Eigenstrat [[24](#B24)]. A sensitivity analysis was performed to determine the number of PCs that should be accounted for and we decided to adjust only on the first two PCs (the genomic control coefficient lambda [[26](#B26)] was 1.023 when adjusting on the top 2 PCs, 1.019 when adjusting on the top 10 PCs and 1.016 when adjusting on the top 20).

Odds-ratios (OR) and confidence intervals of the different genotypes were computed using the logistic regression model implemented in Plink v1.06 [[23](#B23)].

To study the influence of the drug involved in the disease, a second study was performed where the multinomial logistic model implemented in Stata v10 [[27](#B27)] was used as in [[28](#B28)]. The disease status D was coded in 3 classes: 0 unaffected (controls), 1 affected with a suspected allopurinol-induced disease, 2 affected with another suspected drug induced disease. At each SNP, an additive model was considered and two ORs were computed to model the effect of the SNP among allopurinol-induced affected individuals (OR_allopurinol_) and among other affected individuals (OR_otherdrugs_). The null hypothesis tested was one of no effect of the SNP in any of the strata, i.e. OR_allopurinol _= OR_otherdrugs _= 1. It was tested using a 2 degrees-of-freedom chi-square test. The significance of the difference between OR_allopurinol _and OR_otherdrugs _was also tested in this model using the corresponding one degree-of-freedom chi-square test.

In the same way, a multinomial logistic model was used to determine whether genetic differences exist depending on the disease severity. A four class disease status was used: 0 unaffected (controls), 1 SJS, 2 SJS/TEN overlap and 3 TEN.

Furthermore, haplotype association was tested using Plink v1.06 [[23](#B23)].

For all the logistic regression modelling and haplotype association testing, an adjustment on the first two PCs obtained from the PCA of the genotype data was used.

## Results

The clinical details of the 424 patients remaining after quality-control are summarized in Table [1](#T1): 61.3% are female, 47.2% have a SJS phenotype, 34.4% a SJS/TEN overlap phenotype and 18.4% a TEN phenotype. A majority of the patients was sampled from France (110 patients) and Germany (277 patients). History of drug intake was carefully monitored and a subgroup of 57 patients (13.4%) with potentially allopurinol-induced reactions was identified. There is no difference in phenotype distribution between males and females but there are some differences depending on the drug and the country where samples were collected. There is a lack of TEN among allopurinol-induced patients (test of homogeneity of the three disease forms in allopurinol induced versus non-allopurinol induced partients: chi-square = 7.89, p-value = 0.019) and a lack of TEN among patients from Germany as compared to patients from France (chi-square = 23.68, p-value = 7.2 10-6). The two variables however are not independent since allopurinol is more used in Germany than in some other countries and in particular France.

### Table 1.

| Number of patients | SJS | SJS/TEN | TEN | Total |
| --- | --- | --- | --- | --- |
| Total | 200 (47.2%) | 146 (34.4%) | 78 (18.4%) | 424 |
| Females | 124 | 89 | 47 | 260 (61.3%) |
| Allopurinol-induced | 33 | 21 | 3 | 57 (13.4%) |
| From France | 41 | 32 | 37 | 110 (25.9%) |
| From Germany | 145 | 97 | 35 | 277 (65.3%) |
| From Other countries | 14 | 17 | 6 | 37 (8.7%) |

Table 1 Caption: Description of the sub-sample of 424 patients selected for the analysis

For association testing, the 424 patients were compared to the 1,881 controls from the reference panel originating from France (1,228 individuals) and Germany (653 individuals) who were also the closest controls to the cases on the PCA plot (Figure [1](#F1)). We then tested each SNP for association controlling on the first two PCs of the PCA using Eigenstrat [[24](#B24)]. Results are presented in Figure [3](#F3). One genome-wide significant signal was detected in the MHC region on chromosome 6 where 6 SNPs showed False Discovery Rate adjusted q-values below 5% (see Figure [4](#F4) and Table [2](#T2)). The most significant SNP, rs9469003, is located at position 31,515,807, ~85 kb upstream of the HLA-B locus and has an OR of 1.73 (95%CI = [1.44;2.08]). The five other SNPs are all located in a more telomeric region, 250 kb apart from rs9469003, in between positions 31,114,834 and 31,250,224 (see Figure [5](#F5)). There is very limited Linkage Disequilibrium (LD) between these five SNPs and the top one and the six SNPs defined 13 haplotypes with a frequency of 1% or above. The haplotype CACGAC formed by the risk allele at each locus showed the strongest association with the disease with an OR of 2.84 (95%CI = [2.03; 3.98]) that is significantly higher than the one observed for rs9469003 only. To dissect further this haplotypic association, the haplotype conditioning test implemented in Plink v1.06 was used and we found that none of the 6 SNPs by itself has an independent effect nor explains the haplotype association.

### Figure 3.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/01da/3173287/c4508fb0722d/1750-1172-6-52-3.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3173287_1750-1172-6-52-3.jpg)

Results of the genome-wide association screen. The -log(p-value) of the PC-corrected Armitage-trend test implemented in Eigenstrat are plotted for the 268,818 autosomal SNPs that passed the QC with the different colors representing the different chromosome. The horizontal line represents the 10-6 p-value threshold that also corresponds to a False Discovery Rate q-value of 5%.

### Figure 4.

![Figure 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/01da/3173287/9fa4889fb18a/1750-1172-6-52-4.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3173287_1750-1172-6-52-4.jpg)

Results of the association test on chromosome 6. A The -log(p-value) of the PC-corrected Armitage-trend test implemented in Eigenstrat are plotted for the 18,1278 SNPs located on chromosome 6 as a function of their position on the chromosome in base pairs (Build 36). SNPs located in the HLA region are highlighted in red. B Detail of the association signals for the 706 SNPs located in the HLA region. The horizontal line represents the 10-6 p-value threshold.

### Table 2.

| SNP | Chr | Position Build 36 | RiskAllele (Other Allele) | Risk Allele Frequency in Cases | Risk Allele Frequency in Controls | PC corrected Trend testb | p-valueb | q-valuec | OddsRatiod | 95% CId | Pvalue of HW test | Annotationa |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| rs2844665 | 6 | 31114834 | C (T) | 0.72 | 0.62 | 26.46 | 2.69 10-7 | 1.48 10-2 | 1.54 | 1.30-1.82 | 0.77 | C6orf205flanking_3UTR |
| rs3815087 | 6 | 31201566 | A (G) | 0.31 | 0.21 | 26.32 | 2.89 10-7 | 1.54 10-2 | 1.53 | 1.29-1.80 | 0.08 | PSORS1C1UTR |
| rs3130931 | 6 | 31242867 | C (T) | 0.77 | 0.69 | 24.41 | 7.78 10-7 | 3.34 10-2 | 1.54 | 1.29-1.84 | 0.66 | POU5F1Intron |
| rs3130501 | 6 | 31244432 | G (A) | 0.83 | 0.74 | 31.78 | 1.73 10-8 | 0.22 10-2 | 1.74 | 1.43-2.13 | 0.59 | POU5F1Intron |
| rs3094188 | 6 | 31250224 | A (C) | 0.72 | 0.63 | 30.75 | 2.93 10-8 | 0.26 10-2 | 1.59 | 1.34-1.88 | 0.06 | POU5F1Flanking_5UTR |
| rs9469003 | 6 | 31515807 | C (T) | 0.24 | 0.15 | 36.41 | 1.60 10-9 | 0.04 10-2 | 1.73 | 1.44-2.08 | 0.003 | HCP5flanking_5UTR |

Table 2 Caption: List of most significant SNPs associated with SJS/TEN

### Figure 5.

![Figure 5](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/01da/3173287/9ce5a92b4aa4/1750-1172-6-52-5.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3173287_1750-1172-6-52-5.jpg)

Association results in the 401 kb region encompassing the top six signals. A The -log(p-value) of the PC-corrected Armitage-trend test are plotted against the chromosome 6 positions for the 126 SNPs located in the region. B Listing of genes located in this region based on the HapMap web-browser.

Using a multinomial logistic regression approach, we found that the association with haplotype CACGAC was much stronger in the subgroup of patients with an allopurinol induced disease (ORallopurinol = 7.77, 95%CI = [4.66; 12.98]) than in other patients (ORotherdrugs = 1.92, 95%CI = [1.40;2.64]). The equality of odds-ratio (ORallopurinol = ORotherdrugs) was strongly rejected (p = 6.56 × 10^-7^). Results were the same in France and Germany a shown in Additional File [1](#S1) Tables S1 and S2.

To determine how much of the observed signal could be explained by the known association between allopurinol-induced SJS/TEN and HLA-B*5801, we took advantage of the availability of HLA-B two-digit resolution genotypes for 74 of the 424 patients [[16](#B16)]. Among the 74 HLA-B genotyped patients, 11 were carriers of an HLA-B58 allele and had an allopurinol-induced disease. The frequency of the CACGAC haplotype is increased in HLA-B58 carriers as compared to non HLA-B58 carriers (32.44% versus 10.44%, p-value of the test adjusted on the top 2 PC = 0.0052) but the linkage disequilibrium is not complete. Note however that given the small sample size of HLA-B typed individuals, the power was quite low to detect this difference (power of 60% at a type-one error rate level of 5%).

## Discussion

Discovering genes involved in severe cutaneous adverse reactions and especially SJS/TEN is a major challenge for pharmacogenetics as these reactions, when not fatal, are a sword of Damocles for those who already had the disease and fear to take any drug. Despite important efforts, only genes located in the HLA region have been identified so far. One possible explanation for this lack of success is the limited sample sizes of patients available. Indeed SJS/TEN is fortunately a very rare disease and in most studies, sample sizes rarely exceed one or two hundreds of patients, making it difficult to investigate more than a few candidate genes. Through a collaborative effort, the RegiSCAR group was able to collect detailed medical information and DNA of more than half a thousand of patients from Europe who were genotyped on Illumina 317 K chips. By comparing their genotypes to the ones of 1,881 controls genetically matched for the country of origin, we were able to study the SJS/TEN association at a genome-wide level on European samples. Apart from six SNPs located in the HLA region, no other locus was found to be associated with the disease at a high enough significance level to ensure it is not false positive due to the multiple tests performed. Sample sizes of half a thousand are not enough to ensure a good power to detect common variants with small effects (OR in the range between 1.1 and 1.3) similar to those identified in several common diseases such as diabetes or different cancers [[29](#B29)] where larger samples are easier to collect. However, given the 424 patients remaining after Quality Control, the study was powerful enough to detect common variants with modest effects (the power exceeded 80% to detect variants with an allele frequency above 15% conferring ORs above 1.7 under a multiplicative model). The fact that only the HLA region is detected, suggests that there might not be any other common variant that confer a substantial increase in disease risk and could thus be of interest as a predictive factor. These power computations might however be too optimistic as they are based on the assumption that the same genetic variants might be involved in cases of SJS/TEN associated to different drugs. If we consider that there might exist some levels of genetic heterogeneity depending on the drugs involved as it is the case for HLA associations, the power can be dramatically reduced. For example, considering only the 57 allopurinol-exposed cases, the power to detect a variant of frequency 15% conferring an OR of 2 is then only 5% at a nominal type-one error rate value of 10^-6^. On the other hand, the power to detect a similar effect than the one observed at rs9469003 (i.e., risk allele of frequency of 15% with an associated OR of ~4 among allopurinol-induced SJS/TEN) is 99%, showing that the study was not underpowered to detect strong effects even if they were restricted to a small subset of patients.

It is true however that we might have missed some variants with important effects, especially those with minor allele frequencies below 5% that are not well covered by SNP-chips and it could thus be of interest to investigate the association with rare variants. If the "common disease-common variant hypothesis" was believed to explain the susceptibility to multifactorial diseases [[30](#B30),[31](#B31)], results from GWAS have shown that common variants only explain a minor part of most common disease heritabilities. It has been suggested that rare variants could explain at least part of this "missing heritability" and indeed, rare variants have been found in several diseases [[32](#B32)-[36](#B36)]. Interestingly these rare variants are often functional variants with a direct impact on the protein functionality and they usually confer a much stronger increase in disease risk than common variants. They are also more likely to be affected by some moderate levels of negative selection [[37](#B37),[38](#B38)]. For diseases such as adverse drug reactions, it is not unlikely that the genes and variants involved could be under selection. Indeed if most of the drugs have been introduced too recently to be directly responsible for selective pressures, they are often derived from nutrients that have been consumed by humans for a long time. Thus, genes involved in drug metabolism can potentially be involved in natural selection and this was confirmed by previous studies [[39](#B39)-[41](#B41)]. However, the implication of drug metabolism genes in SJS/TEN remains to be established.

Concerning the association with HLA detected in this sample, we confirmed that it is drug-specific with the strongest association found for the group of patients where allopurinol is suspected to be the cause of the disease. However the disease association was still detectable after exclusion of all patients exposed to allopurinol (whether considered as causing the reaction or not) (OR = 2.13, 95% CI = [1.41; 3.23]). This suggests that different HLA alleles are probably involved depending on the drug, and the 6 SNPs identified could be those that were in linkage disequilibrium (LD) with several of these HLA alleles. To further explore this hypothesis, we tried to impute HLA-B genotypes from the SNP data [[42](#B42)] but we were not successful because of the presence of multiple rare alleles at this locus such as HLA-B58 itself that is only present in one of the European families genotyped by de Bakker et al. [[43](#B43)] precluding the possibility to find appropriate tags for this allele using this sample. To overcome the problem, we tried to determine tag SNPs using our own HLA-B genotyped sample that was enriched in HLA-B58 but we did not find any SNP or group of SNPs that was correlated enough with the HLA-B58 allele (the maximum r^2 ^value was below 20%). This poor HLA-B tagging ability of the SNPs could certainly explain why the association found here in relation with allopurinol is much weaker than the one reported in studies on European samples using resolved HLA-B alleles [[16](#B16),[44](#B44)]. However, even in these two latter studies, the association with HLA-B*5801 is much weaker than the one found in Han Chinese [[15](#B15)] or Thai populations [[45](#B45)] where all patients with allopurinol-induced SJS/TEN are carriers of this allele. It is thus possible that the HLA genetic determinants involved in SJS/TEN are not the HLA-B alleles themselves but some loci in LD with them. Further studies in this genomic region would certainly be necessary to better understand the mechanisms involved.

On a methodological point of view, this study illustrates how Reference Control Panels could be used to test for association when only cases are genotyped. Only patients were genotyped here and controls were selected from the CNG European Control Panel [[22](#B22)] based on their country of origin that was either France and Germany, since those were the two main countries where the cases come from. We also tried to use the whole panel of controls and correct for population stratification using Principal Component Analysis (PCA) and an adjustment on up to 20 PCs instead of the 2 used here. We found that including controls that were more genetically distant to the cases does not really improve the signal in the HLA region and instead could lead to false positive results in other genomic regions. This is somewhat in contradiction with the results recently reported by Zhuang et al. [[46](#B46)] and will probably need to be further investigated. We also used some alternative approaches where controls were genetically matched to cases using either a distance based on the identity by state as described by Guan et al. [[47](#B47)] or a distance derived from the top PCs of the PCA performed to identify axes of variations similar to the one proposed by Luca et al. [[48](#B48)]. Tests accounting for the matching units were then performed and we found that only the HLA region pointed out with significance levels in the same order of magnitude as the ones obtained here. These matching strategies might however be of interest as they could allow the analysis of more heterogeneous samples by including for example those non-European RegiSCAR cases that were excluded here and matching them to individuals from the CEPH-Human Genome Diversity Panel [[49](#B49)].

## Conclusion

Our study confirms the involvement of genetic variants located in the HLA region in the susceptibility to SJS/TEN in European samples, especially in association with allopurinol. No difference is seen depending on the disease severity and no other locus reaches genome-wide association in this sample that is also the largest one collected so far.

## Competing interests

The authors declare that they have no competing interests.

## Authors' contributions

MM, MS, JCR, AH, ML designed the study, DZ, ML supervised the genotyping, LN, YL, JCR, MM, PS were responsible for clinical data collection, RK, EG performed the statistical analysis, EG wrote the manuscript and all authors contributed to and approved the final draft.

## Supplementary Material

## Contributor Information

Emmanuelle Génin, Email: emmanuelle.genin@inserm.fr.

Martin Schumacher, Email: ms@imbi.uni-freiburg.de.

Jean-Claude Roujeau, Email: jean-claude.roujeau@wanadoo.fr.

Luigi Naldi, Email: luigi.naldi@gised.it.

Yvonne Liss, Email: dzh@uniklinik-freiburg.de.

Rémi Kazma, Email: remi.kazma@gmail.com.

Peggy Sekula, Email: ps@imbi.uni-freiburg.de.

Alain Hovnanian, Email: alain.hovnanian@inserm.fr.

Maja Mockenhaupt, Email: dzh@uniklinik-freiburg.de.

## Acknowledgements

We are indebted to all patients whose participation made this study possible as well as to their treating physicians for the great cooperation. We thank the Centre National de Génotypage (CNG) and especially Mark Lathrop and Diana Zelenika for conducting the genotyping study and providing the control data. We also thank Alexis Sidoroff (Austria; data acquisition), Alexander Hellmer (Germany; data management), Konrad Bork, Uwe-Frithjof Haustein, Dieter Vieluf (Germany; clinical case review/expert committee), Davide Zenoni (Italy; data acquisition) and Jan Nico Bouwes Bavinck (The Netherlands; data acquisition) for their support in the study.

The RegiSCAR-study was funded by grants from the European Commission (QLRT-2002-01738), GIS-Institut des Maladies Rares and INSERM (4CH09G) in France, and by a consortium of pharmaceutical companies (Bayer Vital, Boehringer-Ingelheim, Cephalon, GlaxoSmithKline, MSD Sharp and Dohme, Merck, Novartis, Pfizer, Roche, Sanofi-Aventis, Servier). Maja Mockenhaupt received the Else Kröner Memorial Stipendium for support of clinical research through Else Kröner-Fresenius-Foundation. The Centre National de Génotypage (CNG), Paris, France, provided the funding for the genotyping.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Lazarou J, Pomeranz BH, Corey PN. Incidence of adverse drug reactions in hospitalized patients: a meta-analysis of prospective studies. Jama. 1998;279(15):1200–1205. doi: 10.1001/jama.279.15.1200.  [DOI](https://doi.org/10.1001/jama.279.15.1200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9555760/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Jama&title=Incidence%20of%20adverse%20drug%20reactions%20in%20hospitalized%20patients:%20a%20meta-analysis%20of%20prospective%20studies&author=J%20Lazarou&author=BH%20Pomeranz&author=PN%20Corey&volume=279&issue=15&publication_year=1998&pages=1200-1205&pmid=9555760&doi=10.1001/jama.279.15.1200&)

2. Svensson CK, Cowen EW, Gaspari AA. Cutaneous drug reactions. Pharmacol Rev. 2001;53(3):357–379.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/11546834/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacol%20Rev&title=Cutaneous%20drug%20reactions&author=CK%20Svensson&author=EW%20Cowen&author=AA%20Gaspari&volume=53&issue=3&publication_year=2001&pages=357-379&pmid=11546834&)

3. Wolverton SE. Update on cutaneous drug reactions. Adv Dermatol. 1997;13:65–84.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/9551141/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Adv%20Dermatol&title=Update%20on%20cutaneous%20drug%20reactions&author=SE%20Wolverton&volume=13&publication_year=1997&pages=65-84&pmid=9551141&)

4. Roujeau JC, Stern RS. Severe adverse cutaneous reactions to drugs. N Engl J Med. 1994;331(19):1272–1285. doi: 10.1056/NEJM199411103311906.  [DOI](https://doi.org/10.1056/NEJM199411103311906) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7794310/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Severe%20adverse%20cutaneous%20reactions%20to%20drugs&author=JC%20Roujeau&author=RS%20Stern&volume=331&issue=19&publication_year=1994&pages=1272-1285&pmid=7794310&doi=10.1056/NEJM199411103311906&)

5. Auquier-Dunant A, Mockenhaupt M, Naldi L, Correia O, Schroder W, Roujeau JC. Correlations between clinical patterns and causes of erythema multiforme majus, Stevens-Johnson syndrome, and toxic epidermal necrolysis: results of an international prospective study. Arch Dermatol. 2002;138(8):1019–1024. doi: 10.1001/archderm.138.8.1019.  [DOI](https://doi.org/10.1001/archderm.138.8.1019) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12164739/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arch%20Dermatol&title=Correlations%20between%20clinical%20patterns%20and%20causes%20of%20erythema%20multiforme%20majus,%20Stevens-Johnson%20syndrome,%20and%20toxic%20epidermal%20necrolysis:%20results%20of%20an%20international%20prospective%20study&author=A%20Auquier-Dunant&author=M%20Mockenhaupt&author=L%20Naldi&author=O%20Correia&author=W%20Schroder&volume=138&issue=8&publication_year=2002&pages=1019-1024&pmid=12164739&doi=10.1001/archderm.138.8.1019&)

6. Rzany B, Mockenhaupt M, Baur S, Schroder W, Stocker U, Mueller J, Hollander N, Bruppacher R, Schopf E. Epidemiology of erythema exsudativum multiforme majus, Stevens-Johnson syndrome, and toxic epidermal necrolysis in Germany (1990-1992): structure and results of a population-based registry. J Clin Epidemiol. 1996;49(7):769–773. doi: 10.1016/0895-4356(96)00035-2.  [DOI](https://doi.org/10.1016/0895-4356(96)00035-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8691227/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Epidemiol&title=Epidemiology%20of%20erythema%20exsudativum%20multiforme%20majus,%20Stevens-Johnson%20syndrome,%20and%20toxic%20epidermal%20necrolysis%20in%20Germany%20(1990-1992):%20structure%20and%20results%20of%20a%20population-based%20registry&author=B%20Rzany&author=M%20Mockenhaupt&author=S%20Baur&author=W%20Schroder&author=U%20Stocker&volume=49&issue=7&publication_year=1996&pages=769-773&pmid=8691227&doi=10.1016/0895-4356(96)00035-2&)

7. Halevy S, Ghislain PD, Mockenhaupt M, Fagot JP, Bouwes JN Bavinck, Sidoroff A, Naldi L, Dunant A, Viboud C, Roujeau JC. Allopurinol is the most common cause of Stevens-Johnson syndrome and toxic epidermal necrolysis in Europe and Israel. J Am Acad Dermatol. 2008;58(1):25–32. doi: 10.1016/j.jaad.2007.08.036.  [DOI](https://doi.org/10.1016/j.jaad.2007.08.036) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17919772/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Am%20Acad%20Dermatol&title=Allopurinol%20is%20the%20most%20common%20cause%20of%20Stevens-Johnson%20syndrome%20and%20toxic%20epidermal%20necrolysis%20in%20Europe%20and%20Israel&author=S%20Halevy&author=PD%20Ghislain&author=M%20Mockenhaupt&author=JP%20Fagot&author=JN%20Bavinck%20Bouwes&volume=58&issue=1&publication_year=2008&pages=25-32&pmid=17919772&doi=10.1016/j.jaad.2007.08.036&)

8. Mockenhaupt M, Viboud C, Dunant A, Naldi L, Halevy S, Bouwes JN Bavinck, Sidoroff A, Schneck J, Roujeau JC, Flahault A. Stevens-Johnson syndrome and toxic epidermal necrolysis: assessment of medication risks with emphasis on recently marketed drugs. The EuroSCAR-study. J Invest Dermatol. 2008;128(1):35–44. doi: 10.1038/sj.jid.5701033.  [DOI](https://doi.org/10.1038/sj.jid.5701033) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17805350/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Invest%20Dermatol&title=Stevens-Johnson%20syndrome%20and%20toxic%20epidermal%20necrolysis:%20assessment%20of%20medication%20risks%20with%20emphasis%20on%20recently%20marketed%20drugs.%20The%20EuroSCAR-study&author=M%20Mockenhaupt&author=C%20Viboud&author=A%20Dunant&author=L%20Naldi&author=S%20Halevy&volume=128&issue=1&publication_year=2008&pages=35-44&pmid=17805350&doi=10.1038/sj.jid.5701033&)

9. Pirmohamed M, Park BK. Genetic susceptibility to adverse drug reactions. Trends Pharmacol Sci. 2001;22(6):298–305. doi: 10.1016/S0165-6147(00)01717-X.  [DOI](https://doi.org/10.1016/S0165-6147(00)01717-X) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11395158/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Trends%20Pharmacol%20Sci&title=Genetic%20susceptibility%20to%20adverse%20drug%20reactions&author=M%20Pirmohamed&author=BK%20Park&volume=22&issue=6&publication_year=2001&pages=298-305&pmid=11395158&doi=10.1016/S0165-6147(00)01717-X&)

10. Melsom RD. Familial hypersensitivity to allopurinol with subsequent desensitization. Rheumatology (Oxford) 1999;38(12):1301. doi: 10.1093/rheumatology/38.12.1301.  [DOI](https://doi.org/10.1093/rheumatology/38.12.1301) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10587570/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Rheumatology%20(Oxford)&title=Familial%20hypersensitivity%20to%20allopurinol%20with%20subsequent%20desensitization&author=RD%20Melsom&volume=38&issue=12&publication_year=1999&pages=1301&pmid=10587570&doi=10.1093/rheumatology/38.12.1301&)

11. Pellicano R, Silvestris A, Iannantuono M, Ciavarella G, Lomuto M. Familial occurrence of fixed drug eruptions. Acta Derm Venereol. 1992;72(4):292–293.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/1357892/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Acta%20Derm%20Venereol&title=Familial%20occurrence%20of%20fixed%20drug%20eruptions&author=R%20Pellicano&author=A%20Silvestris&author=M%20Iannantuono&author=G%20Ciavarella&author=M%20Lomuto&volume=72&issue=4&publication_year=1992&pages=292-293&pmid=1357892&)

12. Roujeau JC, Huynh TN, Bracq C, Guillaume JC, Revuz J, Touraine R. Genetic susceptibility to toxic epidermal necrolysis. Arch Dermatol. 1987;123(9):1171–1173. doi: 10.1001/archderm.123.9.1171.  [DOI](https://doi.org/10.1001/archderm.123.9.1171) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/3477129/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arch%20Dermatol&title=Genetic%20susceptibility%20to%20toxic%20epidermal%20necrolysis&author=JC%20Roujeau&author=TN%20Huynh&author=C%20Bracq&author=JC%20Guillaume&author=J%20Revuz&volume=123&issue=9&publication_year=1987&pages=1171-1173&pmid=3477129&doi=10.1001/archderm.123.9.1171&)

13. Roujeau JC, Bracq C, Huyn NT, Chaussalet E, Raffin C, Duedari N. HLA phenotypes and bullous cutaneous reactions to drugs. Tissue Antigens. 1986;28(4):251–254. doi: 10.1111/j.1399-0039.1986.tb00491.x.  [DOI](https://doi.org/10.1111/j.1399-0039.1986.tb00491.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/3544335/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Tissue%20Antigens&title=HLA%20phenotypes%20and%20bullous%20cutaneous%20reactions%20to%20drugs&author=JC%20Roujeau&author=C%20Bracq&author=NT%20Huyn&author=E%20Chaussalet&author=C%20Raffin&volume=28&issue=4&publication_year=1986&pages=251-254&pmid=3544335&doi=10.1111/j.1399-0039.1986.tb00491.x&)

14. Chung WH, Hung SI, Hong HS, Hsih MS, Yang LC, Ho HC, Wu JY, Chen YT. Medical genetics: a marker for Stevens-Johnson syndrome. Nature. 2004;428(6982):486. doi: 10.1038/428486a.  [DOI](https://doi.org/10.1038/428486a) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15057820/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Medical%20genetics:%20a%20marker%20for%20Stevens-Johnson%20syndrome&author=WH%20Chung&author=SI%20Hung&author=HS%20Hong&author=MS%20Hsih&author=LC%20Yang&volume=428&issue=6982&publication_year=2004&pages=486&pmid=15057820&doi=10.1038/428486a&)

15. Hung SI, Chung WH, Liou LB, Chu CC, Lin M, Huang HP, Lin YL, Lan JL, Yang LC, Hong HS, Chen MJ, Lai PC, Wu MS, Chu CY, Wang KH, Chen CH, Fann CS, Wu JY, Chen YT. HLA-B*5801 allele as a genetic marker for severe cutaneous adverse reactions caused by allopurinol. Proc Natl Acad Sci USA. 2005;102(11):4134–4139. doi: 10.1073/pnas.0409500102.  [DOI](https://doi.org/10.1073/pnas.0409500102) | [PMC free article](/articles/PMC554812/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15743917/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20USA&title=HLA-B*5801%20allele%20as%20a%20genetic%20marker%20for%20severe%20cutaneous%20adverse%20reactions%20caused%20by%20allopurinol&author=SI%20Hung&author=WH%20Chung&author=LB%20Liou&author=CC%20Chu&author=M%20Lin&volume=102&issue=11&publication_year=2005&pages=4134-4139&pmid=15743917&doi=10.1073/pnas.0409500102&)

16. Lonjou C, Borot N, Sekula P, Ledger N, Thomas L, Halevy S, Naldi L, Bouwes-Bavinck JN, Sidoroff A, de Toma C, Schumacher M, Roujeau JC, Hovnanian A, Mockenhaupt M. RegiSCAR study group. A European study of HLA-B in Stevens-Johnson syndrome and toxic epidermal necrolysis related to five high-risk drugs. Pharmacogenetics and genomics. 2008;18(2):99–107. doi: 10.1097/FPC.0b013e3282f3ef9c.  [DOI](https://doi.org/10.1097/FPC.0b013e3282f3ef9c) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18192896/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics%20and%20genomics&title=A%20European%20study%20of%20HLA-B%20in%20Stevens-Johnson%20syndrome%20and%20toxic%20epidermal%20necrolysis%20related%20to%20five%20high-risk%20drugs&author=C%20Lonjou&author=N%20Borot&author=P%20Sekula&author=N%20Ledger&author=L%20Thomas&volume=18&issue=2&publication_year=2008&pages=99-107&pmid=18192896&doi=10.1097/FPC.0b013e3282f3ef9c&)

17. Lonjou C, Thomas L, Borot N, Ledger N, de Toma C, LeLouet H, Graf E, Schumacher M, Hovnanian A, Mockenhaupt M, Roujeau JC. RegiSCAR Group. A marker for Stevens-Johnson syndrome ...: ethnicity matters. Pharmacogenomics J. 2006;6(4):265–268. doi: 10.1038/sj.tpj.6500356.  [DOI](https://doi.org/10.1038/sj.tpj.6500356) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16415921/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=A%20marker%20for%20Stevens-Johnson%20syndrome%20...:%20ethnicity%20matters&author=C%20Lonjou&author=L%20Thomas&author=N%20Borot&author=N%20Ledger&author=C%20de%20Toma&volume=6&issue=4&publication_year=2006&pages=265-268&pmid=16415921&doi=10.1038/sj.tpj.6500356&)

18. Pirmohamed M, Arbuckle JB, Bowman CE, Brunner M, Burns DK, Delrieu O, Dix LP, Twomey JA, Stern RS. Investigation into the multidimensional genetic basis of drug-induced Stevens-Johnson syndrome and toxic epidermal necrolysis. Pharmacogenomics. 2007;8(12):1661–1691. doi: 10.2217/14622416.8.12.1661.  [DOI](https://doi.org/10.2217/14622416.8.12.1661) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18085998/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Investigation%20into%20the%20multidimensional%20genetic%20basis%20of%20drug-induced%20Stevens-Johnson%20syndrome%20and%20toxic%20epidermal%20necrolysis&author=M%20Pirmohamed&author=JB%20Arbuckle&author=CE%20Bowman&author=M%20Brunner&author=DK%20Burns&volume=8&issue=12&publication_year=2007&pages=1661-1691&pmid=18085998&doi=10.2217/14622416.8.12.1661&)

19. Miller JW. Of race, ethnicity, and rash: the genetics of antiepileptic drug-induced skin reactions. Epilepsy Curr. 2008;8(5):120–121. doi: 10.1111/j.1535-7511.2008.00263.x.  [DOI](https://doi.org/10.1111/j.1535-7511.2008.00263.x) | [PMC free article](/articles/PMC2566609/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18852831/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Epilepsy%20Curr&title=Of%20race,%20ethnicity,%20and%20rash:%20the%20genetics%20of%20antiepileptic%20drug-induced%20skin%20reactions&author=JW%20Miller&volume=8&issue=5&publication_year=2008&pages=120-121&pmid=18852831&doi=10.1111/j.1535-7511.2008.00263.x&)

20. Abe R. Toxic epidermal necrolysis and Stevens-Johnson syndrome: soluble Fas ligand involvement in the pathomechanisms of these diseases. J Dermatol Sci. 2008;52(3):151–159. doi: 10.1016/j.jdermsci.2008.06.003.  [DOI](https://doi.org/10.1016/j.jdermsci.2008.06.003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18657400/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Dermatol%20Sci&title=Toxic%20epidermal%20necrolysis%20and%20Stevens-Johnson%20syndrome:%20soluble%20Fas%20ligand%20involvement%20in%20the%20pathomechanisms%20of%20these%20diseases&author=R%20Abe&volume=52&issue=3&publication_year=2008&pages=151-159&pmid=18657400&doi=10.1016/j.jdermsci.2008.06.003&)

21. Pirmohamed M. Genetic factors in the predisposition to drug-induced hypersensitivity reactions. Aaps J. 2006;8(1):E20–26. doi: 10.1208/aapsj080103.  [DOI](https://doi.org/10.1208/aapsj080103) | [PMC free article](/articles/PMC2751420/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16584129/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Aaps%20J&title=Genetic%20factors%20in%20the%20predisposition%20to%20drug-induced%20hypersensitivity%20reactions&author=M%20Pirmohamed&volume=8&issue=1&publication_year=2006&pages=E20-26&pmid=16584129&doi=10.1208/aapsj080103&)

22. Heath SC, Gut IG, Brennan P, McKay JD, Bencko V, Fabianova E, Foretova L, Georges M, Janout V, Kabesch M. et al. Investigation of the fine structure of European populations with applications to disease association studies. Eur J Hum Genet. 2008;16(12):1413–1429. doi: 10.1038/ejhg.2008.210.  [DOI](https://doi.org/10.1038/ejhg.2008.210) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19020537/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Hum%20Genet&title=Investigation%20of%20the%20fine%20structure%20of%20European%20populations%20with%20applications%20to%20disease%20association%20studies&author=SC%20Heath&author=IG%20Gut&author=P%20Brennan&author=JD%20McKay&author=V%20Bencko&volume=16&issue=12&publication_year=2008&pages=1413-1429&pmid=19020537&doi=10.1038/ejhg.2008.210&)

23. Purcell S, Neale B, Todd-Brown K, Thomas L, Ferreira MA, Bender D, Maller J, Sklar P, de Bakker PI, Daly MJ, Sham PC. PLINK: a tool set for whole-genome association and population-based linkage analyses. American journal of human genetics. 2007;81(3):559–575. doi: 10.1086/519795.  [DOI](https://doi.org/10.1086/519795) | [PMC free article](/articles/PMC1950838/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17701901/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=American%20journal%20of%20human%20genetics&title=PLINK:%20a%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses&author=S%20Purcell&author=B%20Neale&author=K%20Todd-Brown&author=L%20Thomas&author=MA%20Ferreira&volume=81&issue=3&publication_year=2007&pages=559-575&pmid=17701901&doi=10.1086/519795&)

24. Price AL, Patterson NJ, Plenge RM, Weinblatt ME, Shadick NA, Reich D. Principal components analysis corrects for stratification in genome-wide association studies. Nature genetics. 2006;38(8):904–909. doi: 10.1038/ng1847.  [DOI](https://doi.org/10.1038/ng1847) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16862161/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature%20genetics&title=Principal%20components%20analysis%20corrects%20for%20stratification%20in%20genome-wide%20association%20studies&author=AL%20Price&author=NJ%20Patterson&author=RM%20Plenge&author=ME%20Weinblatt&author=NA%20Shadick&volume=38&issue=8&publication_year=2006&pages=904-909&pmid=16862161&doi=10.1038/ng1847&)

25. Patterson N, Price AL, Reich D. Population structure and eigenanalysis. PLoS genetics. 2006;2(12):e190. doi: 10.1371/journal.pgen.0020190.  [DOI](https://doi.org/10.1371/journal.pgen.0020190) | [PMC free article](/articles/PMC1713260/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17194218/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20genetics&title=Population%20structure%20and%20eigenanalysis&author=N%20Patterson&author=AL%20Price&author=D%20Reich&volume=2&issue=12&publication_year=2006&pages=e190&pmid=17194218&doi=10.1371/journal.pgen.0020190&)

26. Devlin B, Roeder K. Genomic control for association studies. Biometrics. 1999;55(4):997–1004. doi: 10.1111/j.0006-341X.1999.00997.x.  [DOI](https://doi.org/10.1111/j.0006-341X.1999.00997.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11315092/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biometrics&title=Genomic%20control%20for%20association%20studies&author=B%20Devlin&author=K%20Roeder&volume=55&issue=4&publication_year=1999&pages=997-1004&pmid=11315092&doi=10.1111/j.0006-341X.1999.00997.x&)

27. StataCorp. College Station, TX. StataCorp LP; 2007. Stata Statistical Software: Release 10.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=College%20Station,%20TX&publication_year=2007&)

28. Kazma R, Babron MC, Genin E. Genetic association and gene-environment interaction: a new method for overcoming the lack of exposure information in controls. Am J Epidemiol. pp. 225–235.  [DOI](https://doi.org/10.1093/aje/kwq352) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21084555/)

29. Manolio TA, Brooks LD, Collins FS. A HapMap harvest of insights into the genetics of common disease. J Clin Invest. 2008;118(5):1590–1605. doi: 10.1172/JCI34772.  [DOI](https://doi.org/10.1172/JCI34772) | [PMC free article](/articles/PMC2336881/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18451988/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Invest&title=A%20HapMap%20harvest%20of%20insights%20into%20the%20genetics%20of%20common%20disease&author=TA%20Manolio&author=LD%20Brooks&author=FS%20Collins&volume=118&issue=5&publication_year=2008&pages=1590-1605&pmid=18451988&doi=10.1172/JCI34772&)

30. Reich DE, Lander ES. On the allelic spectrum of human disease. Trends Genet. 2001;17(9):502–510. doi: 10.1016/S0168-9525(01)02410-6.  [DOI](https://doi.org/10.1016/S0168-9525(01)02410-6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11525833/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Trends%20Genet&title=On%20the%20allelic%20spectrum%20of%20human%20disease&author=DE%20Reich&author=ES%20Lander&volume=17&issue=9&publication_year=2001&pages=502-510&pmid=11525833&doi=10.1016/S0168-9525(01)02410-6&)

31. Lohmueller KE, Pearce CL, Pike M, Lander ES, Hirschhorn JN. Meta-analysis of genetic association studies supports a contribution of common variants to susceptibility to common disease. Nature genetics. 2003;33(2):177–182. doi: 10.1038/ng1071.  [DOI](https://doi.org/10.1038/ng1071) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12524541/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature%20genetics&title=Meta-analysis%20of%20genetic%20association%20studies%20supports%20a%20contribution%20of%20common%20variants%20to%20susceptibility%20to%20common%20disease&author=KE%20Lohmueller&author=CL%20Pearce&author=M%20Pike&author=ES%20Lander&author=JN%20Hirschhorn&volume=33&issue=2&publication_year=2003&pages=177-182&pmid=12524541&doi=10.1038/ng1071&)

32. Bodmer W, Bonilla C. Common and rare variants in multifactorial susceptibility to common diseases. Nature genetics. 2008;40(6):695–701. doi: 10.1038/ng.f.136.  [DOI](https://doi.org/10.1038/ng.f.136) | [PMC free article](/articles/PMC2527050/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18509313/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature%20genetics&title=Common%20and%20rare%20variants%20in%20multifactorial%20susceptibility%20to%20common%20diseases&author=W%20Bodmer&author=C%20Bonilla&volume=40&issue=6&publication_year=2008&pages=695-701&pmid=18509313&doi=10.1038/ng.f.136&)

33. Manolio TA, Collins FS, Cox NJ, Goldstein DB, Hindorff LA, Hunter DJ, McCarthy MI, Ramos EM, Cardon LR, Chakravarti A, Cho JH, Guttmacher AE, Kong A, Kruglyak L, Mardis E, Rotimi CN, Slatkin M, Valle D, Whittemore AS, Boehnke M, Clark AG, Eichler EE, Gibson G, Haines JL, Mackay TF, McCarroll SA, Visscher PM. Finding the missing heritability of complex diseases. Nature. 2009;461(7265):747–753. doi: 10.1038/nature08494.  [DOI](https://doi.org/10.1038/nature08494) | [PMC free article](/articles/PMC2831613/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19812666/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Finding%20the%20missing%20heritability%20of%20complex%20diseases&author=TA%20Manolio&author=FS%20Collins&author=NJ%20Cox&author=DB%20Goldstein&author=LA%20Hindorff&volume=461&issue=7265&publication_year=2009&pages=747-753&pmid=19812666&doi=10.1038/nature08494&)

34. Gorlov IP, Gorlova OY, Sunyaev SR, Spitz MR, Amos CI. Shifting paradigm of association studies: value of rare single-nucleotide polymorphisms. American journal of human genetics. 2008;82(1):100–112. doi: 10.1016/j.ajhg.2007.09.006.  [DOI](https://doi.org/10.1016/j.ajhg.2007.09.006) | [PMC free article](/articles/PMC2253956/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18179889/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=American%20journal%20of%20human%20genetics&title=Shifting%20paradigm%20of%20association%20studies:%20value%20of%20rare%20single-nucleotide%20polymorphisms&author=IP%20Gorlov&author=OY%20Gorlova&author=SR%20Sunyaev&author=MR%20Spitz&author=CI%20Amos&volume=82&issue=1&publication_year=2008&pages=100-112&pmid=18179889&doi=10.1016/j.ajhg.2007.09.006&)

35. McCarthy MI. Exploring the unknown: assumptions about allelic architecture and strategies for susceptibility variant discovery. Genome medicine. 2009;1(7):66. doi: 10.1186/gm66.  [DOI](https://doi.org/10.1186/gm66) | [PMC free article](/articles/PMC2717392/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19591663/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20medicine&title=Exploring%20the%20unknown:%20assumptions%20about%20allelic%20architecture%20and%20strategies%20for%20susceptibility%20variant%20discovery&author=MI%20McCarthy&volume=1&issue=7&publication_year=2009&pages=66&pmid=19591663&doi=10.1186/gm66&)

36. Fearnhead NS, Winney B, Bodmer WF. Rare variant hypothesis for multifactorial inheritance: susceptibility to colorectal adenomas as a model. Cell cycle (Georgetown, Tex. 2005;4(4):521–525. doi: 10.4161/cc.4.4.1591.  [DOI](https://doi.org/10.4161/cc.4.4.1591) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15753653/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20cycle%20(Georgetown,%20Tex&title=Rare%20variant%20hypothesis%20for%20multifactorial%20inheritance:%20susceptibility%20to%20colorectal%20adenomas%20as%20a%20model&author=NS%20Fearnhead&author=B%20Winney&author=WF%20Bodmer&volume=4&issue=4&publication_year=2005&pages=521-525&pmid=15753653&doi=10.4161/cc.4.4.1591&)

37. Boyko AR, Williamson SH, Indap AR, Degenhardt JD, Hernandez RD, Lohmueller KE, Adams MD, Schmidt S, Sninsky JJ, Sunyaev SR, White TJ, Nielsen R, Clark AG, Bustamante CD. Assessing the evolutionary impact of amino acid mutations in the human genome. PLoS genetics. 2008;4(5):e1000083. doi: 10.1371/journal.pgen.1000083.  [DOI](https://doi.org/10.1371/journal.pgen.1000083) | [PMC free article](/articles/PMC2377339/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18516229/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20genetics&title=Assessing%20the%20evolutionary%20impact%20of%20amino%20acid%20mutations%20in%20the%20human%20genome&author=AR%20Boyko&author=SH%20Williamson&author=AR%20Indap&author=JD%20Degenhardt&author=RD%20Hernandez&volume=4&issue=5&publication_year=2008&pages=e1000083&pmid=18516229&doi=10.1371/journal.pgen.1000083&)

38. Kryukov GV, Pennacchio LA, Sunyaev SR. Most rare missense alleles are deleterious in humans: implications for complex disease and association studies. American journal of human genetics. 2007;80(4):727–739. doi: 10.1086/513473.  [DOI](https://doi.org/10.1086/513473) | [PMC free article](/articles/PMC1852724/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17357078/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=American%20journal%20of%20human%20genetics&title=Most%20rare%20missense%20alleles%20are%20deleterious%20in%20humans:%20implications%20for%20complex%20disease%20and%20association%20studies&author=GV%20Kryukov&author=LA%20Pennacchio&author=SR%20Sunyaev&volume=80&issue=4&publication_year=2007&pages=727-739&pmid=17357078&doi=10.1086/513473&)

39. Luca F, Bubba G, Basile M, Brdicka R, Michalodimitrakis E, Rickards O, Vershubsky G, Quintana-Murci L, Kozlov AI, Novelletto A. Multiple advantageous amino acid variants in the NAT2 gene in human populations. PloS one. 2008;3(9):e3136. doi: 10.1371/journal.pone.0003136.  [DOI](https://doi.org/10.1371/journal.pone.0003136) | [PMC free article](/articles/PMC2527519/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18773084/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PloS%20one&title=Multiple%20advantageous%20amino%20acid%20variants%20in%20the%20NAT2%20gene%20in%20human%20populations&author=F%20Luca&author=G%20Bubba&author=M%20Basile&author=R%20Brdicka&author=E%20Michalodimitrakis&volume=3&issue=9&publication_year=2008&pages=e3136&pmid=18773084&doi=10.1371/journal.pone.0003136&)

40. Patin E, Barreiro LB, Sabeti PC, Austerlitz F, Luca F, Sajantila A, Behar DM, Semino O, Sakuntabhai A, Guiso N, Gicquel B, McElreavey K, Harding RM, Heyer E, Quintana-Murci L. Deciphering the ancient and complex evolutionary history of human arylamine N-acetyltransferase genes. American journal of human genetics. 2006;78(3):423–436. doi: 10.1086/500614.  [DOI](https://doi.org/10.1086/500614) | [PMC free article](/articles/PMC1380286/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16416399/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=American%20journal%20of%20human%20genetics&title=Deciphering%20the%20ancient%20and%20complex%20evolutionary%20history%20of%20human%20arylamine%20N-acetyltransferase%20genes&author=E%20Patin&author=LB%20Barreiro&author=PC%20Sabeti&author=F%20Austerlitz&author=F%20Luca&volume=78&issue=3&publication_year=2006&pages=423-436&pmid=16416399&doi=10.1086/500614&)

41. Wilson JF, Weale ME, Smith AC, Gratrix F, Fletcher B, Thomas MG, Bradman N, Goldstein DB. Population genetic structure of variable drug response. Nature genetics. 2001;29(3):265–269. doi: 10.1038/ng761.  [DOI](https://doi.org/10.1038/ng761) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11685208/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature%20genetics&title=Population%20genetic%20structure%20of%20variable%20drug%20response&author=JF%20Wilson&author=ME%20Weale&author=AC%20Smith&author=F%20Gratrix&author=B%20Fletcher&volume=29&issue=3&publication_year=2001&pages=265-269&pmid=11685208&doi=10.1038/ng761&)

42. Leslie S, Donnelly P, McVean G. A statistical method for predicting classical HLA alleles from SNP data. American journal of human genetics. 2008;82(1):48–56. doi: 10.1016/j.ajhg.2007.09.001.  [DOI](https://doi.org/10.1016/j.ajhg.2007.09.001) | [PMC free article](/articles/PMC2253983/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18179884/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=American%20journal%20of%20human%20genetics&title=A%20statistical%20method%20for%20predicting%20classical%20HLA%20alleles%20from%20SNP%20data&author=S%20Leslie&author=P%20Donnelly&author=G%20McVean&volume=82&issue=1&publication_year=2008&pages=48-56&pmid=18179884&doi=10.1016/j.ajhg.2007.09.001&)

43. de Bakker PI, McVean G, Sabeti PC, Miretti MM, Green T, Marchini J, Ke X, Monsuur AJ, Whittaker P, Delgado M, Morrison J, Richardson A, Walsh EC, Gao X, Galver L, Hart J, Hafler DA, Pericak-Vance M, Todd JA, Daly MJ, Trowsdale J, Wijmenga C, Vyse TJ, Beck S, Murray SS, Carrington M, Gregory S, Deloukas P, Rioux JD. A high-resolution HLA and SNP haplotype map for disease association studies in the extended human MHC. Nature genetics. 2006;38(10):1166–1172. doi: 10.1038/ng1885.  [DOI](https://doi.org/10.1038/ng1885) | [PMC free article](/articles/PMC2670196/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16998491/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature%20genetics&title=A%20high-resolution%20HLA%20and%20SNP%20haplotype%20map%20for%20disease%20association%20studies%20in%20the%20extended%20human%20MHC&author=PI%20de%20Bakker&author=G%20McVean&author=PC%20Sabeti&author=MM%20Miretti&author=T%20Green&volume=38&issue=10&publication_year=2006&pages=1166-1172&pmid=16998491&doi=10.1038/ng1885&)

44. Kazeem GR, Cox C, Aponte J, Messenheimer J, Brazell C, Nelsen AC, Nelson MR, Foot E. High-resolution HLA genotyping and severe cutaneous adverse reactions in lamotrigine-treated patients. Pharmacogenetics and genomics. 2009;19(9):661–665. doi: 10.1097/FPC.0b013e32832c347d.  [DOI](https://doi.org/10.1097/FPC.0b013e32832c347d) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19668019/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics%20and%20genomics&title=High-resolution%20HLA%20genotyping%20and%20severe%20cutaneous%20adverse%20reactions%20in%20lamotrigine-treated%20patients&author=GR%20Kazeem&author=C%20Cox&author=J%20Aponte&author=J%20Messenheimer&author=C%20Brazell&volume=19&issue=9&publication_year=2009&pages=661-665&pmid=19668019&doi=10.1097/FPC.0b013e32832c347d&)

45. Tassaneeyakul W, Jantararoungtong T, Chen P, Lin PY, Tiamkao S, Khunarkornsiri U, Chucherd P, Konyoung P, Vannaprasaht S, Choonhakarn C, Pisuttimarn P, Sangviroon A, Tassaneeyakul W. Strong association between HLA-B*5801 and allopurinol-induced Stevens-Johnson syndrome and toxic epidermal necrolysis in a Thai population. Pharmacogenetics and genomics. 2009;19(9):704–709. doi: 10.1097/FPC.0b013e328330a3b8.  [DOI](https://doi.org/10.1097/FPC.0b013e328330a3b8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19696695/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics%20and%20genomics&title=Strong%20association%20between%20HLA-B*5801%20and%20allopurinol-induced%20Stevens-Johnson%20syndrome%20and%20toxic%20epidermal%20necrolysis%20in%20a%20Thai%20population&author=W%20Tassaneeyakul&author=T%20Jantararoungtong&author=P%20Chen&author=PY%20Lin&author=S%20Tiamkao&volume=19&issue=9&publication_year=2009&pages=704-709&pmid=19696695&doi=10.1097/FPC.0b013e328330a3b8&)

46. Zhuang JJ, Zondervan K, Nyberg F, Harbron C, Jawaid A, Cardon LR, Barratt BJ, Morris AP. Optimizing the power of genome-wide association studies by using publicly available reference samples to expand the control group. Genetic epidemiology. 2010. Advance Online.  [DOI](https://doi.org/10.1002/gepi.20482) | [PMC free article](/articles/PMC2962805/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20088020/)

47. Guan W, Liang L, Boehnke M, Abecasis GR. Genotype-based matching to correct for population stratification in large-scale case-control genetic association studies. Genetic epidemiology. 2009;33(6):508–517. doi: 10.1002/gepi.20403.  [DOI](https://doi.org/10.1002/gepi.20403) | [PMC free article](/articles/PMC2732762/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19170134/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genetic%20epidemiology&title=Genotype-based%20matching%20to%20correct%20for%20population%20stratification%20in%20large-scale%20case-control%20genetic%20association%20studies&author=W%20Guan&author=L%20Liang&author=M%20Boehnke&author=GR%20Abecasis&volume=33&issue=6&publication_year=2009&pages=508-517&pmid=19170134&doi=10.1002/gepi.20403&)

48. Luca D, Ringquist S, Klei L, Lee AB, Gieger C, Wichmann HE, Schreiber S, Krawczak M, Lu Y, Styche A, Devlin B, Roeder K, Trucco M. On the use of general control samples for genome-wide association studies: genetic matching highlights causal variants. American journal of human genetics. 2008;82(2):453–463. doi: 10.1016/j.ajhg.2007.11.003.  [DOI](https://doi.org/10.1016/j.ajhg.2007.11.003) | [PMC free article](/articles/PMC2427172/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18252225/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=American%20journal%20of%20human%20genetics&title=On%20the%20use%20of%20general%20control%20samples%20for%20genome-wide%20association%20studies:%20genetic%20matching%20highlights%20causal%20variants&author=D%20Luca&author=S%20Ringquist&author=L%20Klei&author=AB%20Lee&author=C%20Gieger&volume=82&issue=2&publication_year=2008&pages=453-463&pmid=18252225&doi=10.1016/j.ajhg.2007.11.003&)

49. Cann HM, de Toma C, Cazes L, Legrand MF, Morel V, Piouffre L, Bodmer J, Bodmer WF, Bonne-Tamir B, Cambon-Thomsen A, Chen Z, Chu J, Carcassi C, Contu L, Du R, Excoffier L, Ferrara GB, Friedlaender JS, Groot H, Gurwitz D, Jenkins T, Herrera RJ, Huang X, Kidd J, Kidd KK, Langaney A, Lin AA, Mehdi SQ, Parham P, Piazza A, Pistillo MP, Qian Y, Shu Q, Xu J, Zhu S, Weber JL, Greely HT, Feldman MW, Thomas G, Dausset J, Cavalli-Sforza LL. A human genome diversity cell line panel. Science (New York, NY. 2002;296(5566):261–262. doi: 10.1126/science.296.5566.261b.  [DOI](https://doi.org/10.1126/science.296.5566.261b) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11954565/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science%20(New%20York,%20NY&title=A%20human%20genome%20diversity%20cell%20line%20panel&author=HM%20Cann&author=C%20de%20Toma&author=L%20Cazes&author=MF%20Legrand&author=V%20Morel&volume=296&issue=5566&publication_year=2002&pages=261-262&pmid=11954565&doi=10.1126/science.296.5566.261b&)

50. Strimmer K. fdrtool: a versatile R package for estimating local and tail area-based false discovery rates. Bioinformatics (Oxford, England) 2008;24(12):1461–1462. doi: 10.1093/bioinformatics/btn209.  [DOI](https://doi.org/10.1093/bioinformatics/btn209) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18441000/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics%20(Oxford,%20England)&title=fdrtool:%20a%20versatile%20R%20package%20for%20estimating%20local%20and%20tail%20area-based%20false%20discovery%20rates&author=K%20Strimmer&volume=24&issue=12&publication_year=2008&pages=1461-1462&pmid=18441000&doi=10.1093/bioinformatics/btn209&)
