# Plasma Letrozole Concentrations in Postmenopausal Women With Breast Cancer Are Associated With CYP2A6 Genetic Variants, Body Mass Index, and Age

## Metadata
**Authors:** Z Desta, Y Kreutz, AT Nguyen, L Li, T Skaar, LK Kamdem, NL Henry, DF Hayes, AM Storniolo, V Stearns, E Hoffmann, RF Tyndale, DA Flockhart
**Journal:** Clinical pharmacology and therapeutics
**Date:** 2011 Oct 5
**DOI:** [10.1038/clpt.2011.174](https://doi.org/10.1038/clpt.2011.174)
**PMID:** 21975350
**PMCID:** PMC3667657
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3667657/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC3667657/pdf/nihms-465721.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC3667657/pdf/nihms-465721.pdf)

## Abstract

The associations between plasma letrozole concentrations and CYP2A6 and CYP3A5 genetic variants were tested in the Exemestane and Letrozole Pharmacogenomics (ELPH) trial. ELPH is a multicenter, open-label prospective clinical trial in women randomly assigned (n ≈ 250 in each arm) to receive 2 years of treatment with either oral letrozole (2.5 mg/day) or oral exemestane (25 mg/day). CYP2A6 and CYP3A showed effects on letrozole metabolism in vitro. DNA samples were genotyped for variants in the CYP2A6 and CYP3A5 genes. plasma letrozole concentrations showed high interpatient variability (>10-fold) and were associated significantly with CYP2A6 genotypes (P < 0.0001), body mass index (BMI) (P < 0.0001), and age (P = 0.0035). However, CYP3A5 genotypes showed no association with plasma letrozole concentrations. These data suggest that CYP2A6 is the principal clearance mechanism for letrozole in vivo. CYP2A6 metabolic status, along with BMI and age, may serve as a biomarker of the efficacy of letrozole treatment or a predictor of adverse effects.

## RESUlTS

### In vitro studies

Incubation of 10 μmol/l letrozole with human liver microsomes (HLMs) and cofactors resulted in the formation of one main metabolite that was identified as 4,4′methanol-bisbenzonitrile. No other metabolites of letrozole, including the ketone metabolite mentioned in the product label,^11^ were identified after incubation of letrozole with HLMs (or expressed enzymes) and cofactors using high-performance liquid chromatography (HPLC) with ultraviolet or fluorescent detection and by liquid chromatography–tandem mass spectrometry approaches (see [Supplementary Methods](#SD1) online). Subsequent experiments showed that the formation rate of 4,4′-methanol-bisbenzonitrile (i) was characterized by a Hill equation: *K*_m_ = 37.5 ± 1.2 μmol/l, *V*_max_ = 18.6 ± 14.0 pmol/min/mg protein, and the Hill coefficient = 1.80 ± 0.42; (ii) showed ~26-fold variability among 21 HLMs and was significantly correlated with CYP3A (Spearman *r* = 0.72; *P* = 0.0002) and CYP2A6 (*r* = 0.45; *P* = 0.04); (iii) was inhibited by the CYP3A inhibitor ketoconazole (IC_50_ = 0.28μmol/l) and by the CYP2A6 inhibitor pilocarpine (IC_50_ = 8.2μmol/l) ([Figure 1](#F1)); and (iv) was formed by CYP2A6>CYP3A4 (and, to some extent, CYP2C19 and CYP2D6).

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4c8b/3667657/dc2dfdb68cdd/nihms-465721-f0001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3667657_nihms-465721-f0001.jpg)

Inhibition of the metabolism of letrozole to 4,4′-methanol-bisbenzonitrile in human liver microsomes (HLMs) by isoform specific inhibitors. (a) Letrozole (10 μmol/l) was incubated with HLMs (0.5 mg/ml) and cofactors with and without the following inhibitors: furafylline (20 μmol/l, CYP1A2), pilocarpine (50 μmol/l, CYP2A6), thioTEPA (50 μmol/l, CYP2B6), ticlopidine (5 μmol/l, CYP2B6, and CYP2C19), sulfaphenazole (25 μmol/l, CYP2C9), omeprazole (5 μmol/l, CYP2C19), quinidine (1 μmol/l, CYP2D6), diethyldithiocarbamate (50 μmol/l, CYP2E1), and ketoconazole (1 μmol/l, CYP3A); (b) IC50 values were determined by incubating letrozole with HLMs and cofactors with and without multiple inhibitor concentrations. The data represent either mean values ± SD in a or average of duplicate incubations in b. CYP, cytochrome P450.

### Clinical studies

#### Distribution of letrozole plasma concentrations

Plasma samples were available from 241 of the patients who had initially been randomized to the letrozole arm (*n* = 252) ([Figure 2](#F2)). Additional plasma samples were obtained from 43 patients who had initially been randomized to the exemestane arm but crossed over to letrozole before the 1-month (*n* = 7), 3-month (*n* = 35), or 6-month (*n* = 1) blood sample draw ([Figure 2](#F2)). The distribution of plasma concentrations of letrozole in the total cohort (*N* = 284) was characterized by large interpatient variability (median, 88.4 ng/ml; range, 0–349.2ng/ml) ([Figure 3](#F3)). In four of the patients, plasma letrozole concentrations were not detectable. Compliance with treatment regimens was not formally assessed in this trial, and the information obtained from patients’ self-reports in the charts indicates that two of the patients took their last letrozole dose within the 3 h prior to the blood draw, whereas there was no record of the timing with respect to the other two patients. The samples were assayed twice to rule out errors. The plasma letrozole concentration remained highly variable, with a median of 89.7 ng/ml (range: 28.4–349.2ng/ml; 12.3-fold difference) even after the data for the four aforementioned patients were excluded. The inset in [Figure 3](#F3) illustrates frequency distribution and probit analysis of letrozole concentrations, which was skewed toward high concentrations.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4c8b/3667657/2088647d30be/nihms-465721-f0002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3667657_nihms-465721-f0002.jpg)

Flow diagram illustrating enrollment of patients (Pts) and analyses of samples. ELPH, Exemestane and Letrozole Pharmacogenomics.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4c8b/3667657/2f99377f7fe7/nihms-465721-f0003.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3667657_nihms-465721-f0003.jpg)

Distribution of plasma letrozole concentrations in all (N = 284) patients with breast cancer. Plasma letrozole concentrations were measured using high-performance liquid chromatography at 1 month (n = 22), 3 months (n = 258), and 6 months (n = 4) after intake of 2.5 mg/day of letrozole. Among these, 43 subjects were initially randomized to exemestane and later crossed over to letrozole. Of the total number of subjects, the majority (88%) were whites (n = 250); 9.9% were blacks (n = 28), and 2.1% were Asians (n = 6). The frequency distribution is shown in the inset and was skewed to the right (high exposure).

#### Associations between plasma letrozole concentrations and demographic variables

The demographic characteristics of the patients are shown in [Supplementary Table S1](#SD2) online. As shown in [Figure 4](#F4) and [Table 1](#T1), plasma letrozole concentrations were positively correlated with age (Pearson *r* = 0.17; *P* = 0.0035) and negatively correlated with body mass index (BMI) (*r* = −0.26; *P* < 0.0001). Linear regression analysis revealed that age (*R*^2^ = 0.03; *P* = 0.0035) and BMI (*R*^2^ = 0.068; *P* = 8.6 × 10^−6^) were associated with plasma letrozole concentrations ([Figure 4](#F4); and [Table 1](#T1)). The sample size was too small (Asian, 6; black, 25) to test for associations with race ([Supplementary Table S1](#SD2) online).

![Figure 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4c8b/3667657/34c7b03457c5/nihms-465721-f0004.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3667657_nihms-465721-f0004.jpg)

Associations between plasma letrozole concentrations and (a) body mass index (BMI) and (b) age in patients with breast cancer.

| Independent variable | Adjusted r2 | β Coefficient (SE) | T value | P value |
| --- | --- | --- | --- | --- |
| Simple variate (simple linear regression) analysis |  |  |  |  |
|   |  |  |  |  |
| CYP2A6 gene score | 0.229 | −32.963 (3.76) | −8.77 | 2.49 × 10−16 |
| Normal |   |   |   |   |
| Intermediate |   |   |   |   |
| Slow |   |   |   |   |
|   |  |  |  |  |
| CYP3A5 gene score | 0.002 | −5.66 (7.72) | −0.73 | 0.47 |
| Expressor |   |   |   |   |
| Nonexpressor |   |   |   |   |
|   |  |  |  |  |
| Age | 0.0299 | 0.85 (0.29) | 2.95 | 0.0035 |
|   |  |  |  |  |
| BMI | 0.068 | −1.7261 (0.38) | −4.53 | 0.00000861 |
|   |  |  |  |  |
| Multivariate analysis |  |  |  |  |
|   |  |  |  |  |
| CYP2A6 gene score |   | −33.57 (3.55) | −9.45 | <2 × 10−16 |
| Normal |   |   |   |   |
| Intermediate |   |   |   |   |
| Slow |   |   |   |   |
|   |  |  |  |  |
| Age |   | 0.59 (0.246) | 2.4 | 0.0172 |
|   |  |  |  |  |
| BMI |   | −1.82 (0.327) | −5.57 | 0.000000064 |
|   |  |  |  |  |
| Age+BMI+CYP2A6genotype | 0.323 |   |   | <2.2 × 10−16 |

Table Caption: Associations of letrozole plasma concentrations with CYP2A6 and CYP3A5 genotypes and demographic variables

#### Associations between CYP2A6 genotype groups and plasma letrozole concentrations

DNA samples from 235 patients who had initially been randomized to receive letrozole, and from 40 other patients who had crossed over from the exemestane arm to the letrozole arm, were genotyped for *CYP2A6* variants (total *n* = 275). In 261 patients, data related to both genotype and plasma letrozole concentrations were available ([Figure 2](#F2)). Most of the subjects in this study were white ([Supplementary Table S1](#SD2) online). Data from 259 subjects were used in the final analyses. No significant differences in demographic characteristics were observed among the three genotype groups ([Supplementary Table S1](#SD2) online). The frequencies of the *CYP2A6* genotypes, depicted in [Figure 5](#F5), were in Hardy–Weinberg equilibrium. The genotype groups were categorized as normal, intermediate, or slow metabolizer groups (see Methods section) on the basis of genotype-predicted phenotype as shown by nicotine metabolism.^15,16^ The median (minimum to maximum) plasma letrozole concentrations in normal, intermediate, and slow metabolizers of CYP2A6 were 81.2 (0–217.5), 112.4 (52.7–1256.9), and 152.1 (50.0–268.6) ng/ml, respectively. Statistically significant differences in letrozole concentrations were observed among the three genotype groups (*P* < 0.0001; Kruskal–Wallis test) ([Figure 6a](#F6)). *Post hoc* analysis revealed significantly higher plasma letrozole concentrations in slow and intermediate metabolizers relative to those in normal metabolizers (*P* < 0.0001 for each) ([Figure 6a](#F6)). Large interindividual variability was seen within each genotype ([Figure 6a](#F6)). Simple linear regression analysis ([Table 1](#T1)) revealed that CYP2A6 genetic variation was significantly associated with plasma letrozole concentration (adjusted *R*^2^ = 0.229; *P* = 2.49 × 10^−16^) ([Table 1](#T1)).

![Figure 5](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4c8b/3667657/30df188682d8/nihms-465721-f0005.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3667657_nihms-465721-f0005.jpg)

Plasma letrozole concentrations in individual CYP2A6 genotypes. On the basis of genotype-predicted phenotypes, patients were grouped into normal (n = 200; *1/*1), intermediate (IM) (n = 40; *1/*9 and *1/*12), and slow (n = 21) metabolizer groups.

![Figure 6](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4c8b/3667657/246b56c844be/nihms-465721-f0006.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3667657_nihms-465721-f0006.jpg)

Plasma letrozole concentrations in patients with breast cancer are significantly associated with (a) CYP2A6 genetic variations but not with (b) CYP3A5 genotypes.

The data shown in [Figure 5](#F5) reveal that the phenotype predicted by *CYP2A6*1/*26* and *CYP2A6*1/*35* genotypes was consistent with normal metabolizers of letrozole rather than slow metabolizers, whereas *CYP2A6*12* was associated more with slow metabolizers than with intermediate metabolizers. These data are in contrast to the *CYP2A6* genotype–phenotype associations reported on the basis of nicotine metabolism.^15,16^ Given the ambiguity of the predictions regarding phenotype, and in order to ensure that no bias was introduced into estimates of the strength of genotype–phenotype associations on this account, additional analyses were performed after the two genotypes (*CYP2A6*1/*26* and *CYP2A6*1/*35*; *n* = 1 each) were excluded from the analysis, whereas data from *CYP2A6*1/*12* (*n* = 11) were analyzed on the assumption that they related to slow metabolizers.[Supplementary Table S2](#SD3) online shows that the data from both analyses were essentially similar to those listed in [Table 1](#T1), although the strength of the association was slightly stronger when the genotype-predicted phenotype was on the basis of letrozole concentrations (*R*^2^ = 0.265) than on the basis of nicotine metabolism (*R*^2^ = 0.229).

#### CYP3A5 genotypes and their associations with plasma letrozole concentrations

A total of 288 DNA samples were genotyped for *CYP3A5* variants (**3, *6,* and **7*). Letrozole concentrations and genotypes were available for only 270 of the patient samples. When the data were analyzed using the same approach as with CYP2A6 genotype-predicted phenotype (normal, intermediate, and slow metabolizers), no significant associations were observed (one-way analysis of variance; *P* = 0.16) among the three groups. The *r*^2^ derived from linear regression was 0.0048 (*P* = 0.256). Given that most of the patients studied were white, it is not surprising that the number of patients with the *1/*1 genotype was very small (*n* = 7) ([Supplementary Table S1](#SD2) online). We found no statistically significant differences in letrozole concentrations between *CYP3A5* expressors (*n* = 42: *1/*1, *n* = 7; *1/*3, *n* = 31; *1/*6, *n* = 3; and *n* = 1, *1/*7) and nonexpressors (*n* = 237: 3/*3, *n* = 228; *3/*6, *n* = 3; *3/*7, *n* = 4; *6/*6, *n* = 1; and *6/*7, *n* = 1) ([Figure 6b](#F6); [Table 1](#T1)).

#### Multivariate analyses

In the multivariate analyses, significant associations were noted between plasma letrozole concentrations and *CYP2A6* genotype (*P* < 2 × 10^−16^), BMI (6.4 × 10^−8^), and age (*P* = 0.017). The incorporation of age and BMI in the multiple regression analyses significantly improved the overall predictive value of *CYP2A6* genotype; the *R*^2^ value of 0.229 in the model containing *CYP2A6* genetic markers alone rose to an adjusted value of 0.323 in the model containing *CYP2A6* genotypes plus demographic variables. The absolute difference in the variation in letrozole concentrations attributable to age and BMI was ~9.4% ([Table 1](#T1)).

## DISCUSSION

In the present study, CYP2A6 and CYP3A enzymes were identified as the principal catalysts in the metabolism of letrozole into 4,4′-methanol-bisbenzonitrile *in vitro*. The genes coding for these enzymes were therefore selected for focused study of genotype–phenotype associations in a large-scale clinical trial in postmenopausal women with breast cancer. Letrozole concentrations exhibited considerable interpatient variability in this cohort of patients. This variability can be explained to a large extent by *CYP2A6* genetic variations and to a smaller extent by demographic factors, notably BMI and age; together, all these (*CYP2A6* variations plus demographic factors) explain ~32.3% of the interpatient variability. CYP3A5 genotype had no evident association with plasma letrozole concentrations *in vivo*. These data impart important information that may be applied to individualize letrozole therapy. In addition, they suggest that CYP2A6 may be involved in the principal clearance mechanism of letrozole *in vivo*.

Plasma letrozole concentrations varied more than 10-fold among patients with breast cancer who had quantifiable levels of letrozole. We estimate that ~23% of the letrozole variability is attributable to *CYP2A6* genetic variation. These *in vivo* data broadly concur with our *in vitro* findings showing the participation of CYP2A6 in letrozole metabolism (present data and ref. [13](#R13)). Other investigators have also reported that CYP2A6 and CYP3A represent the high- and low-affinity components, respectively, in the formation of the carbinol metabolite of letrozole.^14^ Although the results of these *in vitro* studies point toward a role for CYP2A6 in the *in vivo* metabolism of letrozole, it remains less clear whether this *in vivo* effect can be fully predicted from *in vitro* studies evaluating formation of 4,4′-methanol-bisbenzonitrile. According to the information provided in the product label, CYP2A6 is the sole catalyst in the metabolism of letrozole to a ketone metabolite, whereas CYP2A6 and CYP3A are involved in the formation of 4,4′-methanol-bisbenzonitrile.^11^ As proposed in our previous study, these two metabolites might be sequentially formed from letrozole, with the ketone being formed first, probably by CYP2A6, and subsequently being converted to 4,4′-methanol-bisbenzonitrile by CYP2A6 and CYP3A.^13^ If the latter model holds true, CYP2A6 could be the rate-limiting step in letrozole metabolism, as shown in our *in vivo* results; under this condition, assessing 4,4′-methanol-bisbenzonitrile alone would substantially underestimate the quantitative *in vivo* role of CYP2A6, given that there appears to be significant participation from CYP3A in the formation of this metabolite *in vitro* (present data and ref. [14](#R14)). Unfortunately, this suggestion, although plausible, remains speculative because no ketone metabolite standard was available to us and repeated attempts to identify the ketone metabolite in our *in vitro* and *in vivo* studies did not succeed. Regardless of the specific mechanisms involved, the findings of this study provide *in vivo* evidence that CYP2A6 is involved in the main clearance mechanism of letrozole and that this drug can now be added to the list of the few clinically important substrates of CYP2A6. The data suggest that letrozole may serve as a safe, alternative *in vivo* probe of CYP2A6 activity. Unfortunately, metabolite data were not obtained in this study to enable a proper evaluation of this suggestion. Attempts to measure concentrations of letrozole metabolites in plasma were not successful because these were below the limit of quantification of the assay, even after enzymatic hydrolysis by β-glucuronidase. This is not surprising because the quantity of letrozole metabolite in plasma is expected to be minimal given that letrozole has a very long elimination half-life (90–120 h) with very low oral clearance (1.2 l/h at steady state)^12^ and that any metabolites that are slowly produced undergo efficient phase II conjugation and rapid elimination through the kidney. Although a high abundance of conjugated letrozole metabolite is expected in urine,^12^ urine samples were not collected in these patients, and therefore these metabolic ratios could not be calculated. This study was not designed to address the utility of letrozole as a probe for CYP2A6. Formal pharmacokinetic studies, including determination of metabolite(s) in urine and comparison with established probes, are needed to validate the suggestion that letrozole can serve as an alternative probe for CYP2A6 *in vivo*.

Our data show that BMI and age are independent predictors of plasma letrozole concentrations. The inclusion of age and BMI in the multivariate model significantly improved the overall predictive value from 22.9% in a model containing only *CYP2A6* genetic markers to 32.3% in the model containing *CYP2A6* genetic markers plus demographic factors, yielding an absolute difference of ~9.4% of the variation in letrozole concentration as being attributable to age and BMI. The relatively lower plasma concentrations of letrozole in the peripheral compartment (i.e., plasma) may reflect the fact that letrozole is a highly lipophilic drug with a large volume of distribution (183 l)^12^ and that the volume of distribution increases with increasing BMI. The study finding that letrozole concentrations increase with increasing age concurs with previous reports showing an effect of age on CYP2A6 activity.^17,18^ One study reported no effect of age on letrozole exposure^12^; however, this result was possibly due to the small size of the study cohort.

Interestingly, ~68% of the variability in letrozole concentrations remains unaccounted for. First, although a relatively comprehensive genotyping was performed, it is possible that other (rare) *CYP2A6* variants that were not considered in this study, including gain-in-function alleles,^19^ may contribute to this variability. Second, drug interactions may influence CYP2A6 activity, given that patients with breast cancer often take other drugs concomitantly.^9^ Medication data were carefully collected in this study population, but their impact on letrozole concentrations was not assessed because the inhibitors and inducers of CYP2A6 are less characterized than those that influence other CYPs ([http://medicine.iupui.edu/clinpharm/ddis](http://medicine.iupui.edu/clinpharm/ddis)). Third, although our data show little effect of C*YP3A5* genotypes on letrozole concentrations, the possibility of the involvement of CYP3A4 in letrozole metabolism *in vivo* cannot be excluded. Finally, the possibility that other unidentified metabolic pathways may contribute cannot be fully ruled out, although this is unlikely given that metabolites other than the ketone and carbinole metabolites account for <5% of the dose in mass-balance studies.^11,12^ Although female sex is an independent predictor of increased CYP2A6 activity,^20^ this was not an issue in this study because all the patients were postmenopausal women.

We realize that genotype–phenotype associations may be prone to be false positives in the absence of replication, particularly when the analysis deals with one of many secondary end points of the study, as in this instance. Consequently, the lack of external replication is a limitation of this study. In addition, our suggestion that CYP2A6 might be involved in the main clearing mechanism for letrozole in *vivo* cannot be confirmed in the absence of metabolite data; such data could have provided further information on whether there is an inverse relationship between letrozole and metabolite levels and whether the metabolic ratios are related to the CYP2A6 status. Despite these limitations, there are findings that constitute compelling evidence implicating CYP2A6 as the dominant clearance mechanism of letrozole *in vivo*. These findings include: the robust association between CYP2A6 genetic variations and plasma letrozole concentrations (the present study); the available *in vitro* data supporting the hypothesis that CYP2A6 is involved in letrozole metabolism (the present study as well as refs. [11](#R11),[13](#R13),[14](#R14)); and the fact that the functional consequences of CYP2A6 variants, assessed from plasma letrozole concentrations, cosegregate for the most part with those assessed from nicotine metabolic status,^15,16,21^ supporting the proposition that nicotine and letrozole are metabolized by the same enzyme, i.e., CYP2A6. Therefore, we believe that the findings of our study have important implications. First, if letrozole plasma concentrations and/or *CYP2A6* genetic variants can predict letrozole efficacy and adverse effects, they may eventually be utilized to individualize breast cancer therapy using letrozole. In the Exemestane and Letrozole Pharmacogenomics (ELPH) trial, several biomarkers of efficacy and side effects of letrozole have been studied. Analyses of the trial data are ongoing to test the associations between letrozole concentrations and/or *CYP2A6* genetic variations with regard to a large number of phenotypic outcomes. The results of these analyses may also have relevance to other settings in which letrozole is widely used, especially as an inducer of ovulation in women with infertility.^22^ Second, given that CYP2A6 appears to be involved in the dominant clearance mechanism of letrozole *in vivo*, letrozole may serve as an alternative *in vivo* marker of CYP2A6 activity in postmenopausal women and in men. Third, because the letrozole metabolic pattern appears to diverge from nicotine metabolism with respect to a few *CYP2A6* variants (*12, *26, and *35), consistent with a previous report for variants such as *CYP2A6*17* between coumarin and nicotine,^23^ letrozole may be a valuable novel tool to resolve the issue of potential substrate-dependent effects of *CYPA6* genetic variants, particularly those that lead to structural changes. In conclusion, steady-state plasma concentrations of letrozole exhibit high interpatient variability in postmenopausal women with breast cancer, and this variability is partially explained by CYP2A6 metabolic status.

## METHODS

### In vitro studies

Kinetic, correlation,^24^ and inhibition^13,24,25^ analyses in HLMs and experiments in expressed CYPs were performed to identify letrozole metabolites and the specific CYPs involved (for details see [Supplementary Methods](#SD1) online).

### Clinical studies: the ELPH trial

#### Study patients

Postmenopausal women (18 years and older) with stage 0–III hormone receptor–positive breast cancer who were being started on aromatase inhibitor therapy, either “up-front” or after 1–5 years of tamoxifen treatment, were eligible. Eligible patients were recruited from August 2005 through July 2009 at the following COBRA member institutions: Indiana University Cancer Center, Sidney Kimmel Comprehensive Cancer Center at Johns Hopkins University, and the University of Michigan Comprehensive Cancer Center. Inclusion and exclusion criteria for participating in this study are detailed elsewhere.^26^ All indicated treatments (surgery, radiation therapy, and chemotherapy) for breast cancer were completed prior to study enrollment. The protocol was approved by the institutional review boards of all three participating study sites, and all the enrolled patients provided written informed consent. The clinical trial was reviewed on a biannual basis by an independent data and safety monitoring committee. The trial is registered at [http://www.clinicaltrials.gov](http://www.clinicaltrials.gov) (trial identifier [NCT00228956](https://clinicaltrials.gov/ct2/show/NCT00228956)).

#### Study design

The ELPH trial is a multicenter, open-label, randomized, prospective clinical trial. Patients were randomly assigned to receive either exemestane 25 mg orally daily or letrozole 2.5 mg orally daily for 2 years. DNA samples, plasma samples, and multiple phenotypic outcomes were prospectively collected at baseline and at multiple time points (up to 24 months) after drug administration for pharmacogenetic correlative analyses. The primary objective of the study was to correlate changes in breast density with aromatase (CYP19) genetic variation. The secondary objectives included testing for associations between various phenotypes (e.g., bone density and bone turnover metabolites, serum lipids, estrogen concentrations, hot flashes, quality-of-life measures, and drug concentrations) and variants in candidate genes. This report addresses a secondary hypothesis that genetic variants in drug-metabolizing enzymes predict letrozole concentrations. Briefly, blood samples (10 ml) were obtained at baseline and at 1, 3, 6, and 12 months after letrozole initiation. Plasma was separated from blood samples by centrifugation at 2,060*g* and stored at −80 °C until analysis. Plasma letrozole concentrations were determined from samples collected 1, 3, or 6 months after initiating letrozole. Plasma samples from patients who were randomized to the letrozole arm, and who stayed in this arm for at least 1 month, were analyzed. No plasma samples were available for those who discontinued the study or crossed over to the exemestane group before the 1-month blood draw. In addition, plasma samples from patients who were initially randomized to exemestane and were switched later to letrozole were also assayed for letrozole concentrations.

#### DNA samples

Genotyping was performed using genomic DNA extracted from whole blood using the QIAamp DNA Blood Maxi Kit–Spin Protocol in accordance with the manufacturer’s instructions (Qiagen, Valencia, CA). DNA samples from subjects who had initially been randomized to the letrozole arm, and from those who had crossed over from exemestane to letrozole, were analyzed.

#### CYP2A6 genotyping

All DNA samples were genotyped for *CYP2A6 *2, *4E* (detects **4A/D, C*, and *E*), **9*, and **12*, and African Americans were additionally genotyped for **17, *20, *23, *24, *25, *26, *27*, and **35* and Asians for **7, *8*, and **10*, using methods previously described.^15,21,27^ Subjects were then divided into three categories on the basis of previously published associations between *CYP2A6* genotypes and nicotine metabolism.^21^ Individuals with one copy of the decrease-of-function alleles (**9* and **12*) were categorized as intermediate metabolizers; those with two copies of the decrease-of-function alleles or one or two copies of loss-of-function alleles (CYP2A6*2, *4, *7, *10, *17, *20, *23, *24, *25,*26, *27, and **35*) or one decrease-of-function allele together with one loss-of-function allele as slow metabolizers; and those without these genetic variants as normal metabolizers (**1/*1*).

Despite the fact that individuals with alleles **26* and **35* have previously been categorized as slow metabolizers of nicotine,^15,16^ our study showed these alleles to be associated with relatively lower letrozole concentrations ([Figure 5](#F5)). The **1/*12* genotype, which codes for an intermediate nicotine metabolizer,^21^ was consistent with other loss-of-function alleles when assessed from the viewpoint of letrozole concentrations ([Figure 5](#F5)). Therefore, in addition to the analyses according to categorizations based on the effects of the genotypes on nicotine metabolism, secondary analyses were performed after recategorization of the **1/*12* (*n* = 11) genotype as slow metabolizer. The **1/*26* (*n* = 1) and **1/*35* (*n* = 1) genotypes were excluded from these analyses because of uncertainty regarding genotype-predicted phenotype and small sample sizes. In effect, regarding the genotype-predicted phenotypes for the alleles **12, *26*, and **35*, there were differences depending on whether nicotine^21^ or letrozole (the present study) was used as a substrate. If these results could be further confirmed, a novel categorization of substrate-dependent allele functionality and genotypic effect could be proposed on the basis of the results of our study.

#### CYP3A5 genotyping

DNA samples were genotyped for the alleles **3, *6*, and **7* using allelic-discrimination TaqMan SNP genotyping assays (Applied Biosystems, Foster City, CA).

#### HPLC assay

A new HPLC assay was developed to measure letrozole concentrations in plasma. To 100 μl of plasma, 50 μl of 1μg/ml internal standard and 25 μl of methanol were added; this was vortex-mixed and centrifuged at 14,000 rpm for 5 min in an Eppendorf model 5415C centrifuge (Brinkmann Instruments, Westbury, NY). The supernatant was extracted with 0.5 ml of 1mol/l NaOH/glycine buffer (pH = 11.3) and 6 ml ethyl acetate. The organic phase was evaporated to dryness. The residue was reconstituted with 150 μl of mobile phase, and 50 μl was injected into the HPLC system. The separation system included a Zorbax SB-C18 column (3.5 μm particle size, 150 × 4.6 mm; Phenomenex, Torrance, CA), a Nova-Pak C18 guard column (Waters, Milford, MA)(4-μm particle size), and a mobile phase consisting of 30% acetonitrile and 70% 10mmol/l KH_2_PO_4_ (adjusted to pH = 6.5), with a flow rate of 1 ml/min. The column elute was monitored by fluorescent detector (excitation, 230 nm; emission, 295 nm) (letrozole) or by ultraviolet detection set at 234 nm (internal standard). Letrozole was quantified using the ratio of peak area of letrozole to the peak area of internal standard, and calibration curves were constructed using known letrozole concentrations spiked to blank plasma. The limit of quantification of this assay was 7 ng/ml, with interday and intraday coefficients of variation of <6.5% and <10%, respectively.

#### Data and statistical analyses

*In vitro* data are provided in [Supplementary Methods](#SD1) online.

Continuous variables were summarized groupwise using descriptive statistics. The observed and expected frequencies of alleles and genotypes within populations were tested for deviation from Hardy–Weinberg equilibrium, using the χ^2^-test. Differences in letrozole concentrations among the three different genotype groups of *CYP2A6* were analyzed using nonparametric analysis of variance (Kruskal–Wallis test), with Dunnett’s multiple-comparison post-test. Simple linear regression analysis was used to analyze the correlations between letrozole concentrations and genetic (*CYP2A6, CYP3A5*) as well as demographic variables (age, race, BMI). The contributions of these variables to the overall intersubject variability of plasma letrozole concentrations were subsequently tested using multiple linear regression analyses. In particular, *CYP2A6* was coded in the categories normal, intermediate, and slow metabolizers. CYP3A5 was coded as expressors (1) and nonexpressors (0). Statistical analyses were performed using the statistics software packages SPSS, version 16.0 (SPSS, Chicago, IL) and GraphPad Prism (version 5.02, San Diego, CA). All tests were two sided. *P* < 0.05 was considered statistically significant.

## Supplementary Material

## ACKNOWLEDGMENTS

This study was supported in part by Pharmacogenetics Research Network grant U01-GM61373 (D.A.F.), Clinical Pharmacology training grant 5T32-GM08425 (D.A.F.), and grant 5RO1-GM078501-04 (Z.D.) from the National Institute of General Medical Sciences, National Institutes of Health (NIH), Bethesda, MD, and grants M01-RR000042 (University of Michigan), M01-RR00750 (Indiana University), and M01-RR00052 (Johns Hopkins University) from the National Center for Research Resources (NCRR), a component of the NIH. The contents of this article are solely the responsibility of the authors and do not necessarily represent the official views of the NCRR or the NIH. These studies were also supported by grants from Pfizer (D.F.H.), Novartis Pharma AG (D.F.H.), the Footwear Charitable Foundation of New York/QVC Presents Shoes on Sale (D.F.H.), and the Canadian Institutes of Health Research MOP 86471, CAMH, DA 020830 (NIH), and a CRC (R.F.T.).

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. RYAN KJ. Biological aromatization of steroids. J. Biol. Chem. 1959;234:268–272.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/13630892/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem&title=Biological%20aromatization%20of%20steroids&author=KJ%20RYAN&volume=234&publication_year=1959&pages=268-272&pmid=13630892&)

2. Bhatnagar AS, Häusler A, Schieweck K, Lang M, Bowman R. Highly selective inhibition of estrogen biosynthesis by CGS 20267, a new nonsteroidal aromatase inhibitor. J. Steroid Biochem. Mol. Biol. 1990;37:1021–1027. doi: 10.1016/0960-0760(90)90460-3.  [DOI](https://doi.org/10.1016/0960-0760(90)90460-3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2149502/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Steroid%20Biochem.%20Mol.%20Biol&title=Highly%20selective%20inhibition%20of%20estrogen%20biosynthesis%20by%20CGS%2020267,%20a%20new%20nonsteroidal%20aromatase%20inhibitor&author=AS%20Bhatnagar&author=A%20H%C3%A4usler&author=K%20Schieweck&author=M%20Lang&author=R%20Bowman&volume=37&publication_year=1990&pages=1021-1027&pmid=2149502&doi=10.1016/0960-0760(90)90460-3&)

3. Geisler J, Haynes B, Anker G, Dowsett M, Lønning PE. Influence of letrozole and anastrozole on total body aromatization and plasma estrogen levels in postmenopausal breast cancer patients evaluated in a randomized, cross-over study. J. Clin. Oncol. 2002;20:751–757. doi: 10.1200/JCO.2002.20.3.751.  [DOI](https://doi.org/10.1200/JCO.2002.20.3.751) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11821457/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Oncol&title=Influence%20of%20letrozole%20and%20anastrozole%20on%20total%20body%20aromatization%20and%20plasma%20estrogen%20levels%20in%20postmenopausal%20breast%20cancer%20patients%20evaluated%20in%20a%20randomized,%20cross-over%20study&author=J%20Geisler&author=B%20Haynes&author=G%20Anker&author=M%20Dowsett&author=PE%20L%C3%B8nning&volume=20&publication_year=2002&pages=751-757&pmid=11821457&doi=10.1200/JCO.2002.20.3.751&)

4. Geisler J, et al. Letrozole is superior to anastrozole in suppressing breast cancer tissue and plasma estrogen levels. Clin. Cancer Res. 2008;14:6330–6335. doi: 10.1158/1078-0432.CCR-07-5221.  [DOI](https://doi.org/10.1158/1078-0432.CCR-07-5221) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18829517/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Cancer%20Res&title=Letrozole%20is%20superior%20to%20anastrozole%20in%20suppressing%20breast%20cancer%20tissue%20and%20plasma%20estrogen%20levels&author=J%20Geisler&volume=14&publication_year=2008&pages=6330-6335&pmid=18829517&doi=10.1158/1078-0432.CCR-07-5221&)

5. Thürlimann B, et al. A comparison of letrozole and tamoxifen in postmenopausal women with early breast cancer. N. Engl. J. Med. 2005;353:2747–2757. doi: 10.1056/NEJMoa052258.  [DOI](https://doi.org/10.1056/NEJMoa052258) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16382061/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%20Engl.%20J.%20Med&title=A%20comparison%20of%20letrozole%20and%20tamoxifen%20in%20postmenopausal%20women%20with%20early%20breast%20cancer&author=B%20Th%C3%BCrlimann&volume=353&publication_year=2005&pages=2747-2757&pmid=16382061&doi=10.1056/NEJMoa052258&)

6. Coates AS, et al. Five years of letrozole compared with tamoxifen as initial adjuvant therapy for postmenopausal women with endocrine-responsive early breast cancer: update of study BIG 1-98. J. Clin. Oncol. 2007;25:486–492. doi: 10.1200/JCO.2006.08.8617.  [DOI](https://doi.org/10.1200/JCO.2006.08.8617) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17200148/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Oncol&title=Five%20years%20of%20letrozole%20compared%20with%20tamoxifen%20as%20initial%20adjuvant%20therapy%20for%20postmenopausal%20women%20with%20endocrine-responsive%20early%20breast%20cancer:%20update%20of%20study%20BIG%201-98&author=AS%20Coates&volume=25&publication_year=2007&pages=486-492&pmid=17200148&doi=10.1200/JCO.2006.08.8617&)

7. Henry NL, et al. A prospective study of aromatase inhibitor-associated musculoskeletal symptoms and abnormalities on serial high-resolution wrist ultrasonography. Cancer. 2010;116:4360–4367. doi: 10.1002/cncr.25385.  [DOI](https://doi.org/10.1002/cncr.25385) | [PMC free article](/articles/PMC2936686/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20549827/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer&title=A%20prospective%20study%20of%20aromatase%20inhibitor-associated%20musculoskeletal%20symptoms%20and%20abnormalities%20on%20serial%20high-resolution%20wrist%20ultrasonography&author=NL%20Henry&volume=116&publication_year=2010&pages=4360-4367&pmid=20549827&doi=10.1002/cncr.25385&)

8. Stearns V, et al. Active tamoxifen metabolite plasma concentrations after coadministration of tamoxifen and the selective serotonin reuptake inhibitor paroxetine. J. Natl. Cancer Inst. 2003;95:1758–1764. doi: 10.1093/jnci/djg108.  [DOI](https://doi.org/10.1093/jnci/djg108) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14652237/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Natl.%20Cancer%20Inst&title=Active%20tamoxifen%20metabolite%20plasma%20concentrations%20after%20coadministration%20of%20tamoxifen%20and%20the%20selective%20serotonin%20reuptake%20inhibitor%20paroxetine&author=V%20Stearns&volume=95&publication_year=2003&pages=1758-1764&pmid=14652237&doi=10.1093/jnci/djg108&)

9. Jin Y, et al. CYP2D6 genotype, antidepressant use, and tamoxifen metabolism during adjuvant breast cancer treatment. J. Natl. Cancer Inst. 2005;97:30–39. doi: 10.1093/jnci/dji005.  [DOI](https://doi.org/10.1093/jnci/dji005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15632378/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Natl.%20Cancer%20Inst&title=CYP2D6%20genotype,%20antidepressant%20use,%20and%20tamoxifen%20metabolism%20during%20adjuvant%20breast%20cancer%20treatment&author=Y%20Jin&volume=97&publication_year=2005&pages=30-39&pmid=15632378&doi=10.1093/jnci/dji005&)

10. Goetz MP, et al. Pharmacogenetics of tamoxifen biotransformation is associated with clinical outcomes of efficacy and hot flashes. J. Clin. Oncol. 2005;23:9312–9318. doi: 10.1200/JCO.2005.03.3266.  [DOI](https://doi.org/10.1200/JCO.2005.03.3266) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16361630/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Oncol&title=Pharmacogenetics%20of%20tamoxifen%20biotransformation%20is%20associated%20with%20clinical%20outcomes%20of%20efficacy%20and%20hot%20flashes&author=MP%20Goetz&volume=23&publication_year=2005&pages=9312-9318&pmid=16361630&doi=10.1200/JCO.2005.03.3266&)

11. Femara* (letrozole) [prescribing information] Novartis Pharmaceutical Canada; Quebec, Canada: 2011. http://www.novartis.ca/asknovartispharma/download.htm?res=famvir_scrip_e.pdf&resTitleId=125.  [http://www.novartis.ca/asknovartispharma/download.htm?res=famvir_scrip_e.pdf&resTitleId=125](http://www.novartis.ca/asknovartispharma/download.htm?res=famvir_scrip_e.pdf&resTitleId=125) | [Google Scholar](https://scholar.google.com/scholar_lookup?title=Femara*%20(letrozole)%20%5Bprescribing%20information%5D&publication_year=2011&)

12. Pfister CU, et al. Effect of age and single versus multiple dose pharmacokinetics of letrozole (Femara) in breast cancer patients. Biopharm. Drug Dispos. 2001;22:191–197. doi: 10.1002/bdd.273.  [DOI](https://doi.org/10.1002/bdd.273) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11745921/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biopharm.%20Drug%20Dispos&title=Effect%20of%20age%20and%20single%20versus%20multiple%20dose%20pharmacokinetics%20of%20letrozole%20(Femara)%20in%20breast%20cancer%20patients&author=CU%20Pfister&volume=22&publication_year=2001&pages=191-197&pmid=11745921&doi=10.1002/bdd.273&)

13. Jeong S, Woo MM, Flockhart DA, Desta Z. Inhibition of drug metabolizing cytochrome P450s by the aromatase inhibitor drug letrozole and its major oxidative metabolite 4,4′-methanol-bisbenzonitrile in vitro. Cancer Chemother. Pharmacol. 2009;64:867–875. doi: 10.1007/s00280-009-0935-7.  [DOI](https://doi.org/10.1007/s00280-009-0935-7) | [PMC free article](/articles/PMC2774495/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19198839/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Chemother.%20Pharmacol&title=Inhibition%20of%20drug%20metabolizing%20cytochrome%20P450s%20by%20the%20aromatase%20inhibitor%20drug%20letrozole%20and%20its%20major%20oxidative%20metabolite%204,4%E2%80%B2-methanol-bisbenzonitrile%20in%20vitro&author=S%20Jeong&author=MM%20Woo&author=DA%20Flockhart&author=Z%20Desta&volume=64&publication_year=2009&pages=867-875&pmid=19198839&doi=10.1007/s00280-009-0935-7&)

14. Murai K, Yamazaki H, Nakagawa K, Kawai R, Kamataki T. Deactivation of anti-cancer drug letrozole to a carbinol metabolite by polymorphic cytochrome P450 2A6 in human liver microsomes. Xenobiotica. 2009;39:795–802. doi: 10.3109/00498250903171395.  [DOI](https://doi.org/10.3109/00498250903171395) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19845430/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Xenobiotica&title=Deactivation%20of%20anti-cancer%20drug%20letrozole%20to%20a%20carbinol%20metabolite%20by%20polymorphic%20cytochrome%20P450%202A6%20in%20human%20liver%20microsomes&author=K%20Murai&author=H%20Yamazaki&author=K%20Nakagawa&author=R%20Kawai&author=T%20Kamataki&volume=39&publication_year=2009&pages=795-802&pmid=19845430&doi=10.3109/00498250903171395&)

15. Mwenifumbo JC, et al. Novel and established CYP2A6 alleles impair in vivo nicotine metabolism in a population of Black African descent. Hum. Mutat. 2008;29:679–688. doi: 10.1002/humu.20698.  [DOI](https://doi.org/10.1002/humu.20698) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18360915/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum.%20Mutat&title=Novel%20and%20established%20CYP2A6%20alleles%20impair%20in%20vivo%20nicotine%20metabolism%20in%20a%20population%20of%20Black%20African%20descent&author=JC%20Mwenifumbo&volume=29&publication_year=2008&pages=679-688&pmid=18360915&doi=10.1002/humu.20698&)

16. Al Koudsi N, Ahluwalia JS, Lin SK, Sellers EM, Tyndale RF. A novel CYP2A6 allele (CYP2A6*35) resulting in an amino-acid substitution (Asn438Tyr) is associated with lower CYP2A6 activity in vivo. Pharmacogenomics J. 2009;9:274–282. doi: 10.1038/tpj.2009.11.  [DOI](https://doi.org/10.1038/tpj.2009.11) | [PMC free article](/articles/PMC2922203/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19365400/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=A%20novel%20CYP2A6%20allele%20(CYP2A6*35)%20resulting%20in%20an%20amino-acid%20substitution%20(Asn438Tyr)%20is%20associated%20with%20lower%20CYP2A6%20activity%20in%20vivo&author=N%20Al%20Koudsi&author=JS%20Ahluwalia&author=SK%20Lin&author=EM%20Sellers&author=RF%20Tyndale&volume=9&publication_year=2009&pages=274-282&pmid=19365400&doi=10.1038/tpj.2009.11&)

17. Hukkanen J, Jacob P, 3rd, Benowitz NL. Metabolism and disposition kinetics of nicotine. Pharmacol. Rev. 2005;57:79–115. doi: 10.1124/pr.57.1.3.  [DOI](https://doi.org/10.1124/pr.57.1.3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15734728/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacol.%20Rev&title=Metabolism%20and%20disposition%20kinetics%20of%20nicotine&author=J%20Hukkanen&author=P%20Jacob&author=NL%20Benowitz&volume=57&publication_year=2005&pages=79-115&pmid=15734728&doi=10.1124/pr.57.1.3&)

18. Sinues B, et al. CYP2A6 activity in a healthy Spanish population: effect of age, sex, smoking, and oral contraceptives. Hum. Exp. Toxicol. 2008;27:367–372. doi: 10.1177/0960327107082224.  [DOI](https://doi.org/10.1177/0960327107082224) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18715882/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum.%20Exp.%20Toxicol&title=CYP2A6%20activity%20in%20a%20healthy%20Spanish%20population:%20effect%20of%20age,%20sex,%20smoking,%20and%20oral%20contraceptives&author=B%20Sinues&volume=27&publication_year=2008&pages=367-372&pmid=18715882&doi=10.1177/0960327107082224&)

19. Mwenifumbo JC, Zhou Q, Benowitz NL, Sellers EM, Tyndale RF. New CYP2A6 gene deletion and conversion variants in a population of Black African descent. Pharmacogenomics. 2010;11:189–198. doi: 10.2217/pgs.09.144.  [DOI](https://doi.org/10.2217/pgs.09.144) | [PMC free article](/articles/PMC2922202/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20136358/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=New%20CYP2A6%20gene%20deletion%20and%20conversion%20variants%20in%20a%20population%20of%20Black%20African%20descent&author=JC%20Mwenifumbo&author=Q%20Zhou&author=NL%20Benowitz&author=EM%20Sellers&author=RF%20Tyndale&volume=11&publication_year=2010&pages=189-198&pmid=20136358&doi=10.2217/pgs.09.144&)

20. Benowitz NL, Lessov-Schlaggar CN, Swan GE, Jacob P., 3rd Female sex and oral contraceptive use accelerate nicotine metabolism. Clin. Pharmacol. Ther. 2006;79:480–488. doi: 10.1016/j.clpt.2006.01.008.  [DOI](https://doi.org/10.1016/j.clpt.2006.01.008) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16678549/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther&title=Female%20sex%20and%20oral%20contraceptive%20use%20accelerate%20nicotine%20metabolism&author=NL%20Benowitz&author=CN%20Lessov-Schlaggar&author=GE%20Swan&author=P%20Jacob&volume=79&publication_year=2006&pages=480-488&pmid=16678549&doi=10.1016/j.clpt.2006.01.008&)

21. Ho MK, et al. Association of nicotine metabolite ratio and CYP2A6 genotype with smoking cessation treatment in African-American light smokers. Clin. Pharmacol. Ther. 2009;85:635–643. doi: 10.1038/clpt.2009.19.  [DOI](https://doi.org/10.1038/clpt.2009.19) | [PMC free article](/articles/PMC3698861/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19279561/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther&title=Association%20of%20nicotine%20metabolite%20ratio%20and%20CYP2A6%20genotype%20with%20smoking%20cessation%20treatment%20in%20African-American%20light%20smokers&author=MK%20Ho&volume=85&publication_year=2009&pages=635-643&pmid=19279561&doi=10.1038/clpt.2009.19&)

22. Kamath MS, Aleyamma TK, Chandy A, George K. Aromatase inhibitors in women with clomiphene citrate resistance: a randomized, double-blind, placebo-controlled trial. Fertil. Steril. 2010;94:2857–2859. doi: 10.1016/j.fertnstert.2010.06.008.  [DOI](https://doi.org/10.1016/j.fertnstert.2010.06.008) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20638059/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Fertil.%20Steril&title=Aromatase%20inhibitors%20in%20women%20with%20clomiphene%20citrate%20resistance:%20a%20randomized,%20double-blind,%20placebo-controlled%20trial&author=MS%20Kamath&author=TK%20Aleyamma&author=A%20Chandy&author=K%20George&volume=94&publication_year=2010&pages=2857-2859&pmid=20638059&doi=10.1016/j.fertnstert.2010.06.008&)

23. Ho MK, Mwenifumbo JC, Zhao B, Gillam EM, Tyndale RF. A novel CYP2A6 allele, CYP2A6*23, impairs enzyme function in vitro and in vivo and decreases smoking in a population of Black-African descent. Pharmacogenet. Genomics. 2008;18:67–75. doi: 10.1097/FPC.0b013e3282f3606e.  [DOI](https://doi.org/10.1097/FPC.0b013e3282f3606e) | [PMC free article](/articles/PMC2910083/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18216723/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet.%20Genomics&title=A%20novel%20CYP2A6%20allele,%20CYP2A6*23,%20impairs%20enzyme%20function%20in%20vitro%20and%20in%20vivo%20and%20decreases%20smoking%20in%20a%20population%20of%20Black-African%20descent&author=MK%20Ho&author=JC%20Mwenifumbo&author=B%20Zhao&author=EM%20Gillam&author=RF%20Tyndale&volume=18&publication_year=2008&pages=67-75&pmid=18216723&doi=10.1097/FPC.0b013e3282f3606e&)

24. Ogburn ET, Jones DR, Masters AR, Xu C, Guo Y, Desta Z. Efavirenz primary and secondary metabolism in vitro and in vivo: identification of novel metabolic pathways and cytochrome P450 2A6 as the principal catalyst of efavirenz 7-hydroxylation. Drug Metab. Dispos. 2010;38:1218–1229. doi: 10.1124/dmd.109.031393.  [DOI](https://doi.org/10.1124/dmd.109.031393) | [PMC free article](/articles/PMC2908985/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20335270/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab.%20Dispos&title=Efavirenz%20primary%20and%20secondary%20metabolism%20in%20vitro%20and%20in%20vivo:%20identification%20of%20novel%20metabolic%20pathways%20and%20cytochrome%20P450%202A6%20as%20the%20principal%20catalyst%20of%20efavirenz%207-hydroxylation&author=ET%20Ogburn&author=DR%20Jones&author=AR%20Masters&author=C%20Xu&author=Y%20Guo&volume=38&publication_year=2010&pages=1218-1229&pmid=20335270&doi=10.1124/dmd.109.031393&)

25. Jeong S, Nguyen PD, Desta Z. Comprehensive in vitro analysis of voriconazole inhibition of eight cytochrome P450 (CYP) enzymes: major effect on CYPs 2B6, 2C9, 2C19, and 3A. Antimicrob. Agents Chemother. 2009;53:541–551. doi: 10.1128/AAC.01123-08.  [DOI](https://doi.org/10.1128/AAC.01123-08) | [PMC free article](/articles/PMC2630638/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19029318/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Antimicrob.%20Agents%20Chemother&title=Comprehensive%20in%20vitro%20analysis%20of%20voriconazole%20inhibition%20of%20eight%20cytochrome%20P450%20(CYP)%20enzymes:%20major%20effect%20on%20CYPs%202B6,%202C9,%202C19,%20and%203A&author=S%20Jeong&author=PD%20Nguyen&author=Z%20Desta&volume=53&publication_year=2009&pages=541-551&pmid=19029318&doi=10.1128/AAC.01123-08&)

26. Henry NL, et al. Prospective characterization of musculoskeletal symptoms in early stage breast cancer patients treated with aromatase inhibitors. Breast Cancer Res. Treat. 2008;111:365–372. doi: 10.1007/s10549-007-9774-6.  [DOI](https://doi.org/10.1007/s10549-007-9774-6) | [PMC free article](/articles/PMC3081690/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17922185/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Breast%20Cancer%20Res.%20Treat&title=Prospective%20characterization%20of%20musculoskeletal%20symptoms%20in%20early%20stage%20breast%20cancer%20patients%20treated%20with%20aromatase%20inhibitors&author=NL%20Henry&volume=111&publication_year=2008&pages=365-372&pmid=17922185&doi=10.1007/s10549-007-9774-6&)

27. Mwenifumbo JC, Myers MG, Wall TL, Lin SK, Sellers EM, Tyndale RF. Ethnic variation in CYP2A6*7, CYP2A6*8 and CYP2A6*10 as assessed with a novel haplotyping method. Pharmacogenet. Genomics. 2005;15:189–192. doi: 10.1097/01213011-200503000-00008.  [DOI](https://doi.org/10.1097/01213011-200503000-00008) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15861044/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet.%20Genomics&title=Ethnic%20variation%20in%20CYP2A6*7,%20CYP2A6*8%20and%20CYP2A6*10%20as%20assessed%20with%20a%20novel%20haplotyping%20method&author=JC%20Mwenifumbo&author=MG%20Myers&author=TL%20Wall&author=SK%20Lin&author=EM%20Sellers&volume=15&publication_year=2005&pages=189-192&pmid=15861044&doi=10.1097/01213011-200503000-00008&)
