# A new CYP3A5 variant, CYP3A5*11, is shown to be defective in nifedipine metabolism in a recombinant cDNA expression system

## Metadata
**Authors:** Su-Jun Lee, Ilse P van der Heiden, Joyce A Goldstein, Ron HN van Schaik
**Journal:** Drug Metabolism and Disposition
**Date:** 2006 Oct 11
**DOI:** [10.1124/dmd.106.012310](https://doi.org/10.1124/dmd.106.012310)
**PMID:** 17035598
**PMCID:** PMC3302370
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3302370/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC3302370/pdf/nihms88281.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC3302370/pdf/nihms88281.pdf)

## Abstract

A new CYP3A5 variant, CYP3A5*11, was found in a single white European by DNA sequencing. The CYP3A5*11 allele contains a single nucleotide polymorphism (SNP) (g.3775 A>G) in exon 2 which results in a Tyr53Cys substitution and a g.6986A>G splice change, the latter SNP previously reported in the defective CYP3A5*3 allele. However, the CYP3A5*3 is not a null allele because this variant is associated with leaky splicing, resulting in small amounts of functional protein still being produced. We therefore constructed a cDNA coding for the newly identified CYP3A5.11 protein by site-directed mutagenesis, expressed it in Escherichia coli and partially purified it. While bacteria transformed with wild-type CYP3A5*1 cDNA expressed predominantly cytochrome P450, those transfected with CYP3A5*11 expressed a significant amount of denatured cytochrome P420 in addition to cytochrome P450, suggesting the protein to be unstable. CYP3A5.11 exhibited a 38% decrease in the Vmax for nifedipine metabolism, a 2.7-fold increase in the Km, and a 4.4-fold decrease in the CLint of nifedipine compared with CYP3A5.1. A polymerase chain reaction-restriction fragment length polymorphism (PCR-RFLP) genotyping procedure was developed, and used to genotyping DNA of 500 white individuals for CYP3A5*11. No additional examples of this allele were identified. In summary, individuals carrying the rare CYP3A5*11 allele are predicted to have lower metabolism of CYP3A5 substrates than individuals expressing CYP3A5*3.

## Introduction

Cytochrome P450s (CYPs) metabolize endogenous and xenobiotic compounds to more water-soluble products that are easily eliminated from the body. CYP3A is the most abundant cytochrome P450 subfamily in human liver and intestine ([Watkins et al., 1987](#R34); [Kolars et al., 1992](#R18); [Kolars et al., 1994](#R17); [Shimada et al., 1994](#R29)), accounting for 30% of total P450 in the human liver, and metabolizes approximately 50% of currently used clinical drugs ([Shimada et al., 1994](#R29); [Guengerich, 1999](#R6); [Rendic, 2002](#R25)). There are four human *CYP3A* genes: *CYP3A4, CYP3A5, CYP3A7* and *CYP3A43*. In postnatal human liver, CYP3A4 and CYP3A5 are the two major CYP3A enzymes, which have overlapping substrate specificities ([Guengerich, 1999](#R6); [Lamba et al., 2002](#R20); [Lee and Goldstein, 2005](#R22)). The *in vivo* observed variability in clearance of CYP3A-drugs can be caused by inhibition, induction, co-medication and/or the inheritance of *CYP3A* variant alleles encoding enzymes with altered activity ([Thummel and Wilkinson, 1998](#R32)). The CYP3A5 protein expression is highly variable in human liver due largely to the high frequency of an intronic SNP, *CYP3A5*3* (g.6986A>G), which creates a cryptic splice site which results in aberrant splicing ([Kuehl et al., 2001](#R19)). Although CYP3A5 may represent 50% of the total CYP3A content in livers of individuals homozygous for *CYP3A5*1*, CYP3A5 protein is detected at appreciable levels in only 10–30% of Caucasian and Asian livers due to the high frequency of *CYP3A5*3* ([Kuehl et al., 2001](#R19)). In contrast, CYP3A5 is highly expressed in livers of 50% of African-Americans, who have a high incidence of the *CYP3A5*1* allele ([Kuehl et al., 2001](#R19)). CYP3A5 is also expressed in extrahepatic tissues, such as lung ([Kivisto et al., 1996](#R15)), kidney ([Schuetz et al., 1992](#R27); [Haehner et al., 1996](#R7)), breast and leukocytes ([Huang et al., 1996](#R11); [Janardan et al., 1996](#R13)). The extent to which CYP3A5 contributes to total CYP3A activity has been a matter of debate. Some studies indicate that CYP3A5 accounts for up to 50% of total CYP3A protein in the liver ([Kuehl et al., 2001](#R19)), while other studies suggest it is a minor CYP3A form in liver ([Westlind-Johnsson et al., 2003](#R35)). However, the plasma concentration and dose requirement of the CYP3A-metabolized immunosuppressive drug tacrolimus could be clearly demonstrated correlated with the presence of the *CYP3A5*3* allele ([Hesselink et al., 2003](#R10); [Thervet et al., 2003](#R31), [Haufroid et al., 2004](#R9)), demonstrating that CYP3A5 activity is contributes substantially to the metabolism of specific drugs. Many additional *CYP3A5* SNPs have been described in recent reports ([Lee et al., 2003](#R23); [Saeki et al., 2003](#R26); [Solus et al., 2004](#R30)). Although *CYP3A5*3/*3* individuals have low expression of CYP3A5 protein ([Kuehl et al., 2001](#R19); [Lin et al., 2002](#R24)), at least one report suggests that livers of these individuals contain equal amounts of normally and abnormally spliced RNA due to leaky splicing ([Kuehl et al., 2001](#R19); [Lin et al., 2002](#R24)). This results in expression of some CYP3A5 protein, although at much lower amounts than in individuals containing the *CYP3A5*1* allele ([Hustert et al., 2001](#R12)). The *CYP3A5*3* allele has been associated with reduced clearance of lipid-lowering drugs such as simvastatin and lovastatin, as well as midazolam and tacrolimus ([Hesselink et al., 2003](#R10); [Thervet et al., 2003](#R31), [Haufroid et al., 2004](#R9); [Goto et al., 2004](#R5); [Kivisto et al., 2004](#R16); [Wong et al., 2004](#R38)). Although several other *CYP3A4* and *CYP3A5* alleles are predicted to be defective *in vitro,* a clear association between *CYP3A* genotypes and clinical phenotype remains unclear, largely because of the overlap in substrate specificity of CYP3A4 and CYP3A5. To date, the association between CYP3A4 and CYP3A5 genotype and variability in metabolism of CYP3A substrates remains an area of active study. The addition of a new allele, *CYP3A5*11* (characterized by the g.3775 A>G SNP), to the total known *CYP3A4* and *CYP3A5* alleles may be helpful in understanding these individual variations in future clinical studies.

## Materials and Methods

### Materials

Reduced form β-nicotinamide adenine dinucleotide phosphate (NADPH) nifedipine, δ-aminolevulinic acid (ALA), sodium cholate, phenylmethylsulfonyl fluoride (PMSF), leupeptin, aprotinin, lysozyme, ampicillin, L-α-dilauroyl-sn-glycero-3-phosphocholine, L-α-dioleoyl-sn-glycero-3-phosphocholine, and bovine brain phosphatidylserine were purchased from Sigma Chemical Co. (St. Louis, USA). Restriction enzymes and T4 DNA ligase were from New England Biolabs (Bevery, MA, USA). Oligonucleotide primers were obtained from Sigma Genosys (Woodlands, TX, USA). QuickChange mutagenesis kit and a proofreading Pfu DNA polymerase were from Stratagene (La Jolla, CA, USA). *E. coli* DH5α competent cells and isopropyl β-D-thiogalactopyranoside (IPTG) were from Invitrogen (Boston, MA, USA). Ni-NTA affinity resin was from Qiagen (Valencia, CA, USA). Imidazole was purchased from Calbiochem (La Jolla, CA, USA). Human NADPH-cytochrome P450 oxidoreductase (P450 reductase) and cytochrome b_5_ were from Oxford Biomedical Research (Oxford, MI, USA). All other chemicals and organic solvents for HPLC were of the highest grade from commercial sources.

### Identification of CYP3A5*11

DNA samples were from 500 healthy white Dutch individuals isolated as reported previously ([van Schaik et al., 2002](#R33)). The study was approved by the Medical Ethical Committee of the University Hospital Rotterdam.

### Construction, Expression, and Purification of CYP3A5.11

Wild type CYP3A5 cDNA was kindly provided by Dr. Frank Gonzales (National Cancer Institute, National Institutes of Health). The first eight amino acids in the N-terminus of *CYP3A5*1* cDNA were modified to MALLLAVF as described for the 17-alpha hydroxylase ([Barnes et al., 1991](#R1)) and a 5XHis-tag was added to the C-terminal region. Primers for the N- and C-terminal modifications were described previously ([Lee et al., 2003](#R23)). The modified CYP3A5 cDNA was cloned into the pCW vector via *Nde*I and *Hind*III sites and then utilized as the template for mutagenesis using a QuickChange kit (Stratagene, La Jolla, CA, USA). Mutagenesis primers for the construction of *CYP3A5*11* cDNA were 5′-ggaaatgttttgtcctgtcgtcaggtgagttg-3′ (forward) and 5′-caactcacctgacgacaggacaaaacatttcc-3′ (reverse). After construction, the entire *CYP3A5*11* cDNA construct was analyzed by sequencing in both directions. Expression of CYP3A5 wild type and CYP3A5.11 proteins in *E. coli* DH5α competent cells was induced with 0.5 mM IPTG and 0.5 mM δ-ALA and incubation proceeded at room temperature for 96 hr. To determine an optimal condition for maximal cytochrome P450 expression, CO difference spectra were monitored from 24 hr to 96 hr using a DW-2000/OLIS Spectrophotometer. Purification of P450 proteins was performed as previously described ([Lee et al., 2003](#R23); [Lee et al., 2005](#R21)). To minimize inter-experimental variations in expression and purification, *CYP3A5* wild-type and *CYP3A5*11* cDNA constructs were simultaneously expressed, solubilized by sonication with detergent, and purified under the same conditions twice. Two sets of purified recombinant P450s were used to verify results. To increase the purity and the recovery of cytochrome P450, solubilized membrane extract was purified on Ni-NTA affinity columns utilizing a histidine-tag. The eluted P450 was dialyzed for 48 hrs in dialysis buffer (100 mM potassium phosphate, pH 7.4 and 20% glycerol) and cytochrome P450 content was measured on a DW-2000/OLIS Spectrophotometer.

### Reconstitution and Enzyme Assays

The ratio of P450:P450 reductase:cytochrome b_5_ was 1:4:2 as optimized previously ([Lee et al., 2003](#R23)). Enzyme activity was reconstituted with incubation mixtures containing 4 pmol of P450 (determined spectrally), human NADPH-cytochrome P450 oxidoreductase (16 pmol), cytochrome b_5_ (8 pmol), 1:1:1 ratio of lipid mix (2 μg/reaction), sodium cholate (0.05 μmoles/reaction), and MgCl_2_ (30 mM in final) in a total reaction volume of 0.1 ml in 50 mM potassium buffer, pH 7.7. Nifedipine was protected from light during the experiment. Nifedipine concentrations for the kinetic analyses were 10, 20, 40, 60, 120, 240, and 480 μM. These substrate concentrations were above the lower limit of HPLC detection (6.25 μM) and were within the linear range for enzyme activity. Reactions were pre-incubated for 3 min at 37 °C and initiated with NADPH (1 mM in final) at 37 °C for 10 min. All assays were performed in triplicate. The reactions were stopped with the addition of 50 μl methanol and vigorously vortexed for 2 min followed by centrifugation at 10,000 × g for 15 min. The oxidized metabolite in the supernatant was analyzed by HPLC as described earlier ([Lee et al., 2003](#R23)). There was no catalytic activity in the absence of NADPH.

### Development of a Genotyping Test

A specific PCR-RFLP test was developed to detect this novel mutation. For a 50 μl PCR, 10 ng of genomic DNA was incubated in a PCR mixture containing 1X buffer [10 mM Tris-HCl, pH 8.3, 1.5 mM MgCl_2_, 50 mM KCl, 10 mg/L gelatin (Perkin-Elmer)], 0.2 mM each of the deoxynucleotide triphosphates (Roche), 1.25 U of Amplitaq Gold (Perkin-Elmer), and 40 pmol of primers P1 (5′-CATATTACCTCCCTCTgTcGAC-3′) and P2 (5′-CTCAAGCAACTCACCTGtCG). The underlined, lower case nucleotides are mismatches with the *CYP3A5* sequence, creating restriction sites in the PCR product for the detection of the *CYP3A5*11* SNP, but also creating an internal control for the digestion. PCR conditions were: 7 min at 94 °C; 30 cycles of 1 min at 94 °C, 1 min at 55 °C and 1 min at 72 °C; and finally 7 min at 72 °C. The size of the amplified product was 213 bp. The PCR product (10 μl) was digested with SalI (New England Biolabs) in a total volume of 15 μl for 2 h at 37°C and subsequently analyzed on a 3% agarose/Tris-borate-EDTA gel with ethidium bromide staining. The fragments obtained for the wild type allele were 196 and 17 bp; for the *CYP3A5*11* variant allele the expected fragments are 175, 21 and 17 bp. The sample identified as a heterozygote *CYP3A5*11* by sequencing was used as a positive control for the PCR-RFLP.

### Analysis of Enzyme kinetics and statistics

Nifedipine oxidation was analyzed by nonlinear regression using GraphPad Prism 4.0 (GraphPad Software Inc., San Diego, CA, USA). The goodness of fit was determined by the degrees of freedom, absolute sum of squares, and the standard error of the parameter estimates. Kinetics for nifedipine metabolism followed the Michaelis-Menten equation for hyperbolic kinetics as the best fit of the data as previously described ([Williams et al., 2002](#R36); [Lee et al., 2003](#R23)). Values represent the mean ± S.E of at least two independent experiments, each in triplicate. Statistical significance (p<0.001) was determined using a two-tailed Student’s t-test.

## Results

We found a novel SNP while validating a wild type sample for our *CYP3A5*8* PCR-RFLP assay by sequencing DNA samples from healthy Caucasians. One sample showed a heterozygous peak at a different site than expected for the *CYP3A5*8* polymorphism. The sample was resequenced (in both directions) to confirm the presence of the newly discovered g.3775 A>G SNP ([Fig. 1](#F1)). Analysis of this sample for the intronic *CYP3A5*3* SNP ([van Schaik et al., 2002](#R33)) revealed that this individual was homozygous for a g. 6986A>G SNP associated with the *CYP3A5*3/*3,* allele. Subsequent sequencing of all *CYP3A5* exons in both directions indicated that no other *CYP3A5* SNPs were present (results not shown).

### Fig. 1.

![Fig. 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/84c2/3302370/e8962cb6bf32/nihms88281f1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3302370_nihms88281f1.jpg)

Sequencing results of the sample heterozygous for g.3775 A>G SNP (CYP3A5*11), shown as the reverse sequence (T>C) (A) and the PCR-RFLP result of that sample (lane 1) and of 2 wild type controls as confirmed by sequencing (lanes 2 and 3) (B).

Maximal expression obtained for wild type CYP3A5.1 was 100–150 nmol/liter in *E. coli.* at 72 hr. CYP3A5.11 exhibited approximately 50% lower P450 expression of 40–70 nmol/liter at 72 hr with an additional peak at 420nm, suggesting that the protein may be unstable. CYP3A5.1 protein did not contain a 420 nm peak at 72 hr of expression in *E. coli*. Purification of P450s by Ni-NTA affinity column through the histidine-tag resulted in approximately 40% recovery for wild type and ~10% recovery for CYP3A5.11. Purified CYP3A5.1 protein contained only a small peak at 420nm, but CYP3A5.11 protein exhibited comparable amounts of denatured cytochrome P420 and cytochrome P450 despite two different rounds of expression and purification ([Fig. 2](#F2)). For enzyme assays, the reaction was normalized to equal amounts of cytochrome P450, not cytochrome P420. Since kinetic analysis requires large amounts of enzyme, we initially compared the statistical differences in turnover number by two aliquots of CYP3A5.1 and CYP3A5.11 proteins purified on separate days at two different concentrations of substrate. The first concentration of nifedipine, 60 μM, approximates the Km value of wild-type CYP3A5 as determined in our laboratory, and the second higher concentration of 240 μM approaches the Vmax of the enzyme. [Fig. 3](#F3) shows the results of two independent sets of assays. CYP3A5.11 had significantly lower (67% and 70% (P<0.001) turnover numbers for oxidation of nifedipine at the low substrate concentration and 44% and 50% (P<0.001) lower activity at the high concentration of nifedipine for the two preparations. There was no statistically significant difference in nifedipine oxidation by two preparations of CYP3A5.11 or CYP3A5.1, suggesting low inter-experimental variation in expression and purification. [Fig. 4](#F4) and [Table 1](#T1) compare kinetic parameters for nifedipine oxidation by CYP3A5.11 with that of wild type. Kinetic parameters for nifedipine metabolism by CYP3A5.11 protein exhibited a 39% decrease in the Vmax, a 2.7-fold increase in the Km, and a 4.3-fold decrease in the intrinsic clearance compared to wild type.

### Fig. 2.

![Fig. 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/84c2/3302370/46b0fd2031c0/nihms88281f2.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3302370_nihms88281f2.jpg)

Comparison of CO difference spectrum between CYP3A5.1 (A) and CYP3A5.11 (B). The recombinant cDNA containing CYP3A5*1 wild-type and CYP3A5*11 was constructed in pCW vector. The recombinant plasmid was transformed into E. coli DH5α competent cells. After 72 hr expression, the P450 was purified using a Ni-NTA affinity column utilizing the histidine-tag and reduced CO-spectrum was recorded between 400 and 500 nm using a DW-2000 spectrophotometer. The arrow indicates the P450 peak. CYP3A5.1 exhibited a major peak at 450 nm and a small peak at 420nm. The CYP3A5.11 protein showed two major peaks at 450 and at 420 nm.

### Fig. 3.

![Fig. 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/84c2/3302370/6ce3c533d246/nihms88281f3.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3302370_nihms88281f3.jpg)

Nifedipine metabolism by CYP3A5.1 and CYP3A5.11 proteins. Reactions include 4 pmol P450 protein, 16 pmol reductase, and 8 pmol cytochrome b5 (the ratio of P450:reductase:cytochrome b5 = 1:4:2) in 0.1 ml reaction buffer. All values are means of triplicate experiments ± S.E. The preparation of proteins and details of reaction mixtures are described under Materials and Methods. Two different concentrations were used, 60 μM which is near the Km value and 240 μM which is near the Vmax. *, significantly lower (p<0.001) than CYP3A5.1 as analyzed by two-tailed Student’s t-test.

### Fig. 4.

![Fig. 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/84c2/3302370/988abe62a7e0/nihms88281f4.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3302370_nihms88281f4.jpg)

Catalytic profiles of nifedipine metabolism by CYP3A5.1 (closed circles) and CYP3A5.11 (open circles). The preparation of proteins and details of reaction mixtures and kinetics are described under Materials and Methods. All reactions were performed with spectrally determined P450 (4 pmol), 16 pmol reductase, and 8 pmol cytochrome b5 (the ratio of P450:reductase:cytochrome b5 = 1:4:2) in 0.1 ml reaction buffer for 10 min. All values plotted are means of triplicate experiments ± S.E.

### Table 1.

| P450s | Vmax (nmol/min/nmol P450) | Km (μM) | CLint (ml/min/nmol P450) |
| --- | --- | --- | --- |
| CYP3A5.1 | 144.1±2.5 | 44.1±2.6 | 3.23 |
| CYP3A5.11 | 88.2±2.6 (↓39%)b | 118.7±8.8 (↑2.7×)b | 0.74 (↓4.3×)b |

Table 1 Caption: Kinetic properties of nifedipine metabolism between CYP3A5.1 and CYP3A5.11a

We screened 500 healthy white Dutch individuals for this novel *CYP3A5*11* polymorphism, but could not identify additional individuals with this single nucleotide polymorphism, thus resulting in an allelic frequency of 0.1% in this population.

## Discussion

Genetic variation in the CYP3A genes may influence the metabolism and elimination of CYP3A substrates in humans. Since CYP3A4 and CYP3A5 have overlapping substrates, it has been difficult to determine the effect of each CYP3A genotype on *in vivo* phenotype. The variant g.6986A>G (*CYP3A5*3)* is responsible for dramatic decreases in hepatic expression of CYP3A5 protein ([Kuehl et al., 2001](#R19)) and is a major determinant in low CYP3A5 protein expression in the liver due to its high frequency in many populations. However, individuals carrying the *CYP3A5*3/*3* genotype still express low amounts of CYP3A5 protein ([Hustert et al., 2001](#R12); [Westlind-Johnsson et al., 2003](#R35)). Several studies have identified protein coding variants of CYP3A5, as well as splice variants ([Jounaidi et al., 1996](#R14); [Chou et al., 2001](#R2); [Hustert et al., 2001](#R12); [Kuehl et al., 2001](#R19); [Lee et al., 2003](#R23); [Solus et al., 2004](#R30)). One such allele, *CYP3A5*10,* contains a F446S change in the heme-binding region in addition to the g.6986A>G change associated with the major splice variant ([Lee et al., 2003](#R23)), for which the recombinant CYP3A5.10 (F446S) protein exhibited a greater than 95% decrease in the intrinsic clearance for testosterone and nifedipine compared to wild type CYP3A5.1 ([Lee et al., 2003](#R23)).

In the present study, we identified a new *CYP3A5* variant, *CYP3A5*11*, in white Europeans containing a single base pair change coding for Tyr53Cys. We investigated metabolism of the anti-hypertensive drug nifedipine by recombinant CYP3A5.11 protein, showing a defective variant with a 38% decrease in the Vmax, a 2.7-fold increase in the Km and a 4.3-fold decrease in the intrinsic clearance compared to wild-type CYP3A5.1. Because this allele also contained the g.6986A>G SNP, lower amounts of CYP3A5 protein would be present in the liver ([Hustert et al., 2001](#R12); [Westlind-Johnsson et al., 2003](#R35)). The presence of the Tyr53Cys variant in conjuction with the g.6986A>G SNP is therefore expected to decrese this residual CYP3A5 activity even further. The exact importance of this new SNP, in the context of the *3 splice variant, requires further studies. We assume that the *3 and *11 SNPs would function independently, one affecting mRNA splicing, the other decreasing the activity of residual protein formed. However, the Tyr53Cys variant might occur on a haplotype lacking the g.6986A>G SNP, in which case it would then be expected to have a more profound effect on total CYP3A5 activity. Since polymorphisms can potentially affect metabolism of some substrtes more than others, it remains to be studied whether the magnitude of the decrease in activity of CYP3A5.11 protein toward nifedipine also holds true for other CYP3A5 substrates.

CYP3A4 and CYP3A5 have 84% amino acid sequence identity: alignment of CYP3A5 Cys-53 with CYP3A4 revealed that the CYP3A5 Cys-53 was not located in the known substrate recognition sites (SRSs) or in the active site cavity of CYP3A4 ([Harlow and Halpert, 1997](#R8); [Domanski et al., 1998](#R4); [Domanski et al., 2000](#R3); [Yano et al., 2004](#R39)). The region containing Cys-53, however, is well conserved in the CYP3A4 and CYP3A5 primary amino acid sequences. In the crystal structure of CYP3A4 protein, Cys-53 appears to be located around the N-terminal “A” helix ([Williams et al., 2004](#R37)). A notable hydrophobic region was found around the loop following helix “A” in CYP3A4, and this hydrophobic region was suggested to be mediated with the microsomal membrane interaction ([Williams et al., 2004](#R37); [Scott and Halpert, 2005](#R28)). The change from the tyrosine, which is a hydrophobic and aromatic amino acid, to the sulphur containing hydrophilic amino acid cysteine could cause structural changes in the membrane binding regions. The Tyr-53 in CYP3A4 was reported to be one of the amino acids involved in a hydrogen-bonding network with Arg-106, Glu-374, Asp-76, Arg-372, and Asp-61 ([Yano et al., 2004](#R39)). CYP3A4 and CYP3A5 have identical amino acids at all these locations except that CYP3A5 contains a Glu-76. This hydrogen-bonding network has been suggested to have an important role in the solvent access channel in an X-ray crystallography study ([Yano et al., 2004](#R39)). Therefore, the increased P420 and decreased catalytic activity for nifedipine by CYP3A5.11 protein is likely to result from conformational changes caused by membrane interaction and an altered hydrogen-bonding network. However, although we even used an excess 4:1 ratio of reductase to cytochrome P450, we cannot exclude that the presence of P420 interferes with reductase requirement of the remaining properly folded P450 protein, thereby lowering decreasing catalytic activity to some extent.

A PCR-RFLP genotyping test, developed for the detection of the g.3775 A>G SNP found in *CYP3A5*11,* detected no additional examples of this SNP in 500 genomic DNA samples from healthy Caucasians, suggesting that *CYP3A5*11* is a very rare allele in whites. One of our previous studies detected *CYP3A5*10* only in certain white ethnic groups (Adeyi) ([Lee et al., 2005](#R21)), and it is possible that also the *CYP3A5*11* allele is more frequent in other ethnic groups. The present genetic test with functional information will be useful for future clinical studies to link its association with other *CYP3A* SNPs, particularly in individuals who show defective metabolism of CYP3A substrates.

## Acknowledgments

This research was partially supported by the Intramural Research Program of NIH and NIEHS [Project Z01ES02124-27].

## Abbreviations

## References

1. Barnes HJ, Arlotto MP, Waterman MR. Expression and enzymatic activity of recombinant cytochrome P450 17 alpha-hydroxylase in Escherichia coli. Proc Natl Acad Sci U S A. 1991;88:5597–5601. doi: 10.1073/pnas.88.13.5597.  [DOI](https://doi.org/10.1073/pnas.88.13.5597) | [PMC free article](/articles/PMC51924/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1829523/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&title=Expression%20and%20enzymatic%20activity%20of%20recombinant%20cytochrome%20P450%2017%20alpha-hydroxylase%20in%20Escherichia%20coli&author=HJ%20Barnes&author=MP%20Arlotto&author=MR%20Waterman&volume=88&publication_year=1991&pages=5597-5601&pmid=1829523&doi=10.1073/pnas.88.13.5597&)

2. Chou FC, Tzeng SJ, Huang JD. Genetic polymorphism of cytochrome P450 3A5 in Chinese. Drug Metab Dispos. 2001;29:1205–1209.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/11502729/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Genetic%20polymorphism%20of%20cytochrome%20P450%203A5%20in%20Chinese&author=FC%20Chou&author=SJ%20Tzeng&author=JD%20Huang&volume=29&publication_year=2001&pages=1205-1209&pmid=11502729&)

3. Domanski TL, He YA, Harlow GR, Halpert JR. Dual role of human cytochrome P450 3A4 residue Phe-304 in substrate specificity and cooperativity. J Pharmacol Exp Ther. 2000;293:585–591.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/10773032/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacol%20Exp%20Ther&title=Dual%20role%20of%20human%20cytochrome%20P450%203A4%20residue%20Phe-304%20in%20substrate%20specificity%20and%20cooperativity&author=TL%20Domanski&author=YA%20He&author=GR%20Harlow&author=JR%20Halpert&volume=293&publication_year=2000&pages=585-591&pmid=10773032&)

4. Domanski TL, Liu J, Harlow GR, Halpert JR. Analysis of four residues within substrate recognition site 4 of human cytochrome P450 3A4: role in steroid hydroxylase activity and alpha-naphthoflavone stimulation. Arch Biochem Biophys. 1998;350:223–232. doi: 10.1006/abbi.1997.0525.  [DOI](https://doi.org/10.1006/abbi.1997.0525) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9473295/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arch%20Biochem%20Biophys&title=Analysis%20of%20four%20residues%20within%20substrate%20recognition%20site%204%20of%20human%20cytochrome%20P450%203A4:%20role%20in%20steroid%20hydroxylase%20activity%20and%20alpha-naphthoflavone%20stimulation&author=TL%20Domanski&author=J%20Liu&author=GR%20Harlow&author=JR%20Halpert&volume=350&publication_year=1998&pages=223-232&pmid=9473295&doi=10.1006/abbi.1997.0525&)

5. Goto M, Masuda S, Kiuchi T, Ogura Y, Oike F, Okuda M, Tanaka K, Inui K. CYP3A5*1-carrying graft liver reduces the concentration/oral dose ratio of tacrolimus in recipients of living-donor liver transplantation. Pharmacogenetics. 2004;14:471–478. doi: 10.1097/01.fpc.0000114747.08559.49.  [DOI](https://doi.org/10.1097/01.fpc.0000114747.08559.49) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15226679/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=CYP3A5*1-carrying%20graft%20liver%20reduces%20the%20concentration/oral%20dose%20ratio%20of%20tacrolimus%20in%20recipients%20of%20living-donor%20liver%20transplantation&author=M%20Goto&author=S%20Masuda&author=T%20Kiuchi&author=Y%20Ogura&author=F%20Oike&volume=14&publication_year=2004&pages=471-478&pmid=15226679&doi=10.1097/01.fpc.0000114747.08559.49&)

6. Guengerich FP. Cytochrome P-450 3A4: regulation and role in drug metabolism. Annu Rev Pharmacol Toxicol. 1999;39:1–17. doi: 10.1146/annurev.pharmtox.39.1.1.  [DOI](https://doi.org/10.1146/annurev.pharmtox.39.1.1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10331074/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annu%20Rev%20Pharmacol%20Toxicol&title=Cytochrome%20P-450%203A4:%20regulation%20and%20role%20in%20drug%20metabolism&author=FP%20Guengerich&volume=39&publication_year=1999&pages=1-17&pmid=10331074&doi=10.1146/annurev.pharmtox.39.1.1&)

7. Haehner BD, Gorski JC, Vandenbranden M, Wrighton SA, Janardan SK, Watkins PB, Hall SD. Bimodal distribution of renal cytochrome P450 3A activity in humans. Mol Pharmacol. 1996;50:52–59.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/8700118/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Pharmacol&title=Bimodal%20distribution%20of%20renal%20cytochrome%20P450%203A%20activity%20in%20humans&author=BD%20Haehner&author=JC%20Gorski&author=M%20Vandenbranden&author=SA%20Wrighton&author=SK%20Janardan&volume=50&publication_year=1996&pages=52-59&pmid=8700118&)

8. Harlow GR, Halpert JR. Alanine-scanning mutagenesis of a putative substrate recognition site in human cytochrome P450 3A4. Role of residues 210 and 211 in flavonoid activation and substrate specificity. J Biol Chem. 1997;272:5396–5402. doi: 10.1074/jbc.272.9.5396.  [DOI](https://doi.org/10.1074/jbc.272.9.5396) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9038138/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Alanine-scanning%20mutagenesis%20of%20a%20putative%20substrate%20recognition%20site%20in%20human%20cytochrome%20P450%203A4.%20Role%20of%20residues%20210%20and%20211%20in%20flavonoid%20activation%20and%20substrate%20specificity&author=GR%20Harlow&author=JR%20Halpert&volume=272&publication_year=1997&pages=5396-5402&pmid=9038138&doi=10.1074/jbc.272.9.5396&)

9. Haufroid V, Mourad M, Van Kerckhove V, Wawrzyniak J, De Meyer M, Eddour DC, Malaise J, Lison D, Squifflet JP, Wallemacq P. The effect of CYP3A5 and MDR1 (ABCB1) polymorphisms on cyclosporine and tacrolimus dose requirements and trough blood levels in stable renal transplant patients. Pharmacogenetics. 2004;14:147–54. doi: 10.1097/00008571-200403000-00002.  [DOI](https://doi.org/10.1097/00008571-200403000-00002) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15167702/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=The%20effect%20of%20CYP3A5%20and%20MDR1%20(ABCB1)%20polymorphisms%20on%20cyclosporine%20and%20tacrolimus%20dose%20requirements%20and%20trough%20blood%20levels%20in%20stable%20renal%20transplant%20patients&author=V%20Haufroid&author=M%20Mourad&author=V%20Van%20Kerckhove&author=J%20Wawrzyniak&author=M%20De%20Meyer&volume=14&publication_year=2004&pages=147-54&pmid=15167702&doi=10.1097/00008571-200403000-00002&)

10. Hesselink DA, van Schaik RH, van der Heiden IP, van der Werf M, Gregoor PJ, Lindemans J, Weimar W, van Gelder T. Genetic polymorphisms of the CYP3A4, CYP3A5, and MDR-1 genes and pharmacokinetics of the calcineurin inhibitors cyclosporine and tacrolimus. Clin Pharmacol Ther. 2003;74:245–54. doi: 10.1016/S0009-9236(03)00168-1.  [DOI](https://doi.org/10.1016/S0009-9236(03)00168-1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12966368/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Genetic%20polymorphisms%20of%20the%20CYP3A4,%20CYP3A5,%20and%20MDR-1%20genes%20and%20pharmacokinetics%20of%20the%20calcineurin%20inhibitors%20cyclosporine%20and%20tacrolimus&author=DA%20Hesselink&author=RH%20van%20Schaik&author=IP%20van%20der%20Heiden&author=M%20van%20der%20Werf&author=PJ%20Gregoor&volume=74&publication_year=2003&pages=245-54&pmid=12966368&doi=10.1016/S0009-9236(03)00168-1&)

11. Huang Z, Fasco MJ, Figge HL, Keyomarsi K, Kaminsky LS. Expression of cytochromes P450 in human breast tissue and tumors. Drug Metab Dispos. 1996;24:899–905.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/8869826/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Expression%20of%20cytochromes%20P450%20in%20human%20breast%20tissue%20and%20tumors&author=Z%20Huang&author=MJ%20Fasco&author=HL%20Figge&author=K%20Keyomarsi&author=LS%20Kaminsky&volume=24&publication_year=1996&pages=899-905&pmid=8869826&)

12. Hustert E, Haberl M, Burk O, Wolbold R, He YQ, Klein K, Nuessler AC, Neuhaus P, Klattig J, Eiselt R, Koch I, Zibat A, Brockmoller J, Halpert JR, Zanger UM, Wojnowski L. The genetic determinants of the CYP3A5 polymorphism. Pharmacogenetics. 2001;11:773–779. doi: 10.1097/00008571-200112000-00005.  [DOI](https://doi.org/10.1097/00008571-200112000-00005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11740341/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=The%20genetic%20determinants%20of%20the%20CYP3A5%20polymorphism&author=E%20Hustert&author=M%20Haberl&author=O%20Burk&author=R%20Wolbold&author=YQ%20He&volume=11&publication_year=2001&pages=773-779&pmid=11740341&doi=10.1097/00008571-200112000-00005&)

13. Janardan SK, Lown KS, Schmiedlin-Ren P, Thummel KE, Watkins PB. Selective expression of CYP3A5 and not CYP3A4 in human blood. Pharmacogenetics. 1996;6:379–385. doi: 10.1097/00008571-199610000-00001.  [DOI](https://doi.org/10.1097/00008571-199610000-00001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8946469/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Selective%20expression%20of%20CYP3A5%20and%20not%20CYP3A4%20in%20human%20blood&author=SK%20Janardan&author=KS%20Lown&author=P%20Schmiedlin-Ren&author=KE%20Thummel&author=PB%20Watkins&volume=6&publication_year=1996&pages=379-385&pmid=8946469&doi=10.1097/00008571-199610000-00001&)

14. Jounaidi Y, Hyrailles V, Gervot L, Maurel P. Detection of CYP3A5 allelic variant: a candidate for the polymorphic expression of the protein? Biochem Biophys Res Commun. 1996;221:466–470. doi: 10.1006/bbrc.1996.0618.  [DOI](https://doi.org/10.1006/bbrc.1996.0618) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8619878/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20Biophys%20Res%20Commun&title=Detection%20of%20CYP3A5%20allelic%20variant:%20a%20candidate%20for%20the%20polymorphic%20expression%20of%20the%20protein?&author=Y%20Jounaidi&author=V%20Hyrailles&author=L%20Gervot&author=P%20Maurel&volume=221&publication_year=1996&pages=466-470&pmid=8619878&doi=10.1006/bbrc.1996.0618&)

15. Kivisto KT, Griese EU, Fritz P, Linder A, Hakkola J, Raunio H, Beaune P, Kroemer HK. Expression of cytochrome P 450 3A enzymes in human lung: a combined RT-PCR and immunohistochemical analysis of normal tissue and lung tumours. Naunyn Schmiedebergs Arch Pharmacol. 1996;353:207–212. doi: 10.1007/BF00168759.  [DOI](https://doi.org/10.1007/BF00168759) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8717162/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Naunyn%20Schmiedebergs%20Arch%20Pharmacol&title=Expression%20of%20cytochrome%20P%20450%203A%20enzymes%20in%20human%20lung:%20a%20combined%20RT-PCR%20and%20immunohistochemical%20analysis%20of%20normal%20tissue%20and%20lung%20tumours&author=KT%20Kivisto&author=EU%20Griese&author=P%20Fritz&author=A%20Linder&author=J%20Hakkola&volume=353&publication_year=1996&pages=207-212&pmid=8717162&doi=10.1007/BF00168759&)

16. Kivisto KT, Niemi M, Schaeffeler E, Pitkala K, Tilvis R, Fromm MF, Schwab M, Eichelbaum M, Strandberg T. Lipid-lowering response to statins is affected by CYP3A5 polymorphism. Pharmacogenetics. 2004;14:523–525. doi: 10.1097/01.fpc.0000114762.78957.a5.  [DOI](https://doi.org/10.1097/01.fpc.0000114762.78957.a5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15284534/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Lipid-lowering%20response%20to%20statins%20is%20affected%20by%20CYP3A5%20polymorphism&author=KT%20Kivisto&author=M%20Niemi&author=E%20Schaeffeler&author=K%20Pitkala&author=R%20Tilvis&volume=14&publication_year=2004&pages=523-525&pmid=15284534&doi=10.1097/01.fpc.0000114762.78957.a5&)

17. Kolars JC, Lown KS, Schmiedlin-Ren P, Ghosh M, Fang C, Wrighton SA, Merion RM, Watkins PB. CYP3A gene expression in human gut epithelium. Pharmacogenetics. 1994;4:247–259. doi: 10.1097/00008571-199410000-00003.  [DOI](https://doi.org/10.1097/00008571-199410000-00003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7894497/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=CYP3A%20gene%20expression%20in%20human%20gut%20epithelium&author=JC%20Kolars&author=KS%20Lown&author=P%20Schmiedlin-Ren&author=M%20Ghosh&author=C%20Fang&volume=4&publication_year=1994&pages=247-259&pmid=7894497&doi=10.1097/00008571-199410000-00003&)

18. Kolars JC, Schmiedlin-Ren P, Schuetz JD, Fang C, Watkins PB. Identification of rifampin-inducible P450IIIA4 (CYP3A4) in human small bowel enterocytes. J Clin Invest. 1992;90:1871–1878. doi: 10.1172/JCI116064.  [DOI](https://doi.org/10.1172/JCI116064) | [PMC free article](/articles/PMC443248/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1430211/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Invest&title=Identification%20of%20rifampin-inducible%20P450IIIA4%20(CYP3A4)%20in%20human%20small%20bowel%20enterocytes&author=JC%20Kolars&author=P%20Schmiedlin-Ren&author=JD%20Schuetz&author=C%20Fang&author=PB%20Watkins&volume=90&publication_year=1992&pages=1871-1878&pmid=1430211&doi=10.1172/JCI116064&)

19. Kuehl P, Zhang J, Lin Y, Lamba J, Assem M, Schuetz J, Watkins PB, Daly A, Wrighton SA, Hall SD, Maurel P, Relling M, Brimer C, Yasuda K, Venkataramanan R, Strom S, Thummel K, Boguski MS, Schuetz E. Sequence diversity in CYP3A promoters and characterization of the genetic basis of polymorphic CYP3A5 expression. Nat Genet. 2001;27:383–391. doi: 10.1038/86882.  [DOI](https://doi.org/10.1038/86882) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11279519/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Sequence%20diversity%20in%20CYP3A%20promoters%20and%20characterization%20of%20the%20genetic%20basis%20of%20polymorphic%20CYP3A5%20expression&author=P%20Kuehl&author=J%20Zhang&author=Y%20Lin&author=J%20Lamba&author=M%20Assem&volume=27&publication_year=2001&pages=383-391&pmid=11279519&doi=10.1038/86882&)

20. Lamba JK, Lin YS, Schuetz EG, Thummel KE. Genetic contribution to variable human CYP3A-mediated metabolism. Adv Drug Deliv Rev. 2002;54:1271–1294. doi: 10.1016/s0169-409x(02)00066-2.  [DOI](https://doi.org/10.1016/s0169-409x(02)00066-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12406645/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Adv%20Drug%20Deliv%20Rev&title=Genetic%20contribution%20to%20variable%20human%20CYP3A-mediated%20metabolism&author=JK%20Lamba&author=YS%20Lin&author=EG%20Schuetz&author=KE%20Thummel&volume=54&publication_year=2002&pages=1271-1294&pmid=12406645&doi=10.1016/s0169-409x(02)00066-2&)

21. Lee SJ, Bell DA, Coulter S, Ghanayem B, Goldstein JA. Recombinant CYP3A4*17 is defective in metabolizing the hypertensive drug nifedipine and the CYP3A4*17 allele may occur on the same chromosome as CYP3A5*3 representing a new putative defective CYP3A haplotype. J Pharmacol Exp Ther. 2005;313:302–309. doi: 10.1124/jpet.104.078758.  [DOI](https://doi.org/10.1124/jpet.104.078758) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15634941/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacol%20Exp%20Ther&title=Recombinant%20CYP3A4*17%20is%20defective%20in%20metabolizing%20the%20hypertensive%20drug%20nifedipine%20and%20the%20CYP3A4*17%20allele%20may%20occur%20on%20the%20same%20chromosome%20as%20CYP3A5*3%20representing%20a%20new%20putative%20defective%20CYP3A%20haplotype&author=SJ%20Lee&author=DA%20Bell&author=S%20Coulter&author=B%20Ghanayem&author=JA%20Goldstein&volume=313&publication_year=2005&pages=302-309&pmid=15634941&doi=10.1124/jpet.104.078758&)

22. Lee SJ, Goldstein JA. Functionally defective or altered CYP3A4 and CYP3A5 single nucleotide polymorphisms and their detection with genotyping tests. Pharmacogenomics. 2005;6:357–371. doi: 10.1517/14622416.6.4.357.  [DOI](https://doi.org/10.1517/14622416.6.4.357) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16004554/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Functionally%20defective%20or%20altered%20CYP3A4%20and%20CYP3A5%20single%20nucleotide%20polymorphisms%20and%20their%20detection%20with%20genotyping%20tests&author=SJ%20Lee&author=JA%20Goldstein&volume=6&publication_year=2005&pages=357-371&pmid=16004554&doi=10.1517/14622416.6.4.357&)

23. Lee SJ, Usmani KA, Chanas B, Ghanayem B, Xi T, Hodgson E, Mohrenweiser HW, Goldstein JA. Genetic findings and functional studies of human CYP3A5 single nucleotide polymorphisms in different ethnic groups. Pharmacogenetics. 2003;13:461–472. doi: 10.1097/00008571-200308000-00004.  [DOI](https://doi.org/10.1097/00008571-200308000-00004) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12893984/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Genetic%20findings%20and%20functional%20studies%20of%20human%20CYP3A5%20single%20nucleotide%20polymorphisms%20in%20different%20ethnic%20groups&author=SJ%20Lee&author=KA%20Usmani&author=B%20Chanas&author=B%20Ghanayem&author=T%20Xi&volume=13&publication_year=2003&pages=461-472&pmid=12893984&doi=10.1097/00008571-200308000-00004&)

24. Lin YS, Dowling AL, Quigley SD, Farin FM, Zhang J, Lamba J, Schuetz EG, Thummel KE. Co-regulation of CYP3A4 and CYP3A5 and contribution to hepatic and intestinal midazolam metabolism. Mol Pharmacol. 2002;62:162–172. doi: 10.1124/mol.62.1.162.  [DOI](https://doi.org/10.1124/mol.62.1.162) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12065767/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Pharmacol&title=Co-regulation%20of%20CYP3A4%20and%20CYP3A5%20and%20contribution%20to%20hepatic%20and%20intestinal%20midazolam%20metabolism&author=YS%20Lin&author=AL%20Dowling&author=SD%20Quigley&author=FM%20Farin&author=J%20Zhang&volume=62&publication_year=2002&pages=162-172&pmid=12065767&doi=10.1124/mol.62.1.162&)

25. Rendic S. Summary of information on human CYP enzymes: human P450 metabolism data. Drug Metab Rev. 2002;34:83–448. doi: 10.1081/dmr-120001392.  [DOI](https://doi.org/10.1081/dmr-120001392) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11996015/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Rev&title=Summary%20of%20information%20on%20human%20CYP%20enzymes:%20human%20P450%20metabolism%20data&author=S%20Rendic&volume=34&publication_year=2002&pages=83-448&pmid=11996015&doi=10.1081/dmr-120001392&)

26. Saeki M, Saito Y, Nakamura T, Murayama N, Kim SR, Ozawa S, Komamura K, Ueno K, Kamakura S, Nakajima T, Saito H, Kitamura Y, Kamatani N, Sawada J. Single nucleotide polymorphisms and haplotype frequencies of CYP3A5 in a Japanese population. Hum Mutat. 2003;21:653. doi: 10.1002/humu.9147.  [DOI](https://doi.org/10.1002/humu.9147) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14961555/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mutat&title=Single%20nucleotide%20polymorphisms%20and%20haplotype%20frequencies%20of%20CYP3A5%20in%20a%20Japanese%20population&author=M%20Saeki&author=Y%20Saito&author=T%20Nakamura&author=N%20Murayama&author=SR%20Kim&volume=21&publication_year=2003&pages=653&pmid=14961555&doi=10.1002/humu.9147&)

27. Schuetz EG, Schuetz JD, Grogan WM, Naray-Fejes-Toth A, Fejes-Toth G, Raucy J, Guzelian P, Gionela K, Watlington CO. Expression of cytochrome P450 3A in amphibian, rat, and human kidney. Arch Biochem Biophys. 1992;294:206–214. doi: 10.1016/0003-9861(92)90159-t.  [DOI](https://doi.org/10.1016/0003-9861(92)90159-t) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1550347/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arch%20Biochem%20Biophys&title=Expression%20of%20cytochrome%20P450%203A%20in%20amphibian,%20rat,%20and%20human%20kidney&author=EG%20Schuetz&author=JD%20Schuetz&author=WM%20Grogan&author=A%20Naray-Fejes-Toth&author=G%20Fejes-Toth&volume=294&publication_year=1992&pages=206-214&pmid=1550347&doi=10.1016/0003-9861(92)90159-t&)

28. Scott EE, Halpert JR. Structures of cytochrome P450 3A4. Trends Biochem Sci. 2005;30:5–7. doi: 10.1016/j.tibs.2004.11.004.  [DOI](https://doi.org/10.1016/j.tibs.2004.11.004) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15653318/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Trends%20Biochem%20Sci&title=Structures%20of%20cytochrome%20P450%203A4&author=EE%20Scott&author=JR%20Halpert&volume=30&publication_year=2005&pages=5-7&pmid=15653318&doi=10.1016/j.tibs.2004.11.004&)

29. Shimada T, Yamazaki H, Mimura M, Inui Y, Guengerich FP. Interindividual variations in human liver cytochrome P-450 enzymes involved in the oxidation of drugs, carcinogens and toxic chemicals: studies with liver microsomes of 30 Japanese and 30 Caucasians. J Pharmacol Exp Ther. 1994;270:414–423.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/8035341/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacol%20Exp%20Ther&title=Interindividual%20variations%20in%20human%20liver%20cytochrome%20P-450%20enzymes%20involved%20in%20the%20oxidation%20of%20drugs,%20carcinogens%20and%20toxic%20chemicals:%20studies%20with%20liver%20microsomes%20of%2030%20Japanese%20and%2030%20Caucasians&author=T%20Shimada&author=H%20Yamazaki&author=M%20Mimura&author=Y%20Inui&author=FP%20Guengerich&volume=270&publication_year=1994&pages=414-423&pmid=8035341&)

30. Solus JF, Arietta BJ, Harris JR, Sexton DP, Steward JQ, McMunn C, Ihrie P, Mehall JM, Edwards TL, Dawson EP. Genetic variation in eleven phase I drug metabolism genes in an ethnically diverse population. Pharmacogenomics. 2004;5:895–931. doi: 10.1517/14622416.5.7.895.  [DOI](https://doi.org/10.1517/14622416.5.7.895) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15469410/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Genetic%20variation%20in%20eleven%20phase%20I%20drug%20metabolism%20genes%20in%20an%20ethnically%20diverse%20population&author=JF%20Solus&author=BJ%20Arietta&author=JR%20Harris&author=DP%20Sexton&author=JQ%20Steward&volume=5&publication_year=2004&pages=895-931&pmid=15469410&doi=10.1517/14622416.5.7.895&)

31. Thervet E, Anglicheau D, King B, Schlageter MH, Cassinat B, Beaune P, Legendre C, Daly AK. Impact of cytochrome p450 3A5 genetic polymorphism on tacrolimus doses and concentration-to-dose ratio in renal transplant recipients. Transplantation. 2003;76:1233–5. doi: 10.1097/01.TP.0000090753.99170.89.  [DOI](https://doi.org/10.1097/01.TP.0000090753.99170.89) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14578760/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Transplantation&title=Impact%20of%20cytochrome%20p450%203A5%20genetic%20polymorphism%20on%20tacrolimus%20doses%20and%20concentration-to-dose%20ratio%20in%20renal%20transplant%20recipients&author=E%20Thervet&author=D%20Anglicheau&author=B%20King&author=MH%20Schlageter&author=B%20Cassinat&volume=76&publication_year=2003&pages=1233-5&pmid=14578760&doi=10.1097/01.TP.0000090753.99170.89&)

32. Thummel KE, Wilkinson GR. In vitro and in vivo drug interactions involving human CYP3A. Annu Rev Pharmacol Toxicol. 1998;38:389–430. doi: 10.1146/annurev.pharmtox.38.1.389.  [DOI](https://doi.org/10.1146/annurev.pharmtox.38.1.389) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9597161/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annu%20Rev%20Pharmacol%20Toxicol&title=In%20vitro%20and%20in%20vivo%20drug%20interactions%20involving%20human%20CYP3A&author=KE%20Thummel&author=GR%20Wilkinson&volume=38&publication_year=1998&pages=389-430&pmid=9597161&doi=10.1146/annurev.pharmtox.38.1.389&)

33. van Schaik RH, van der Heiden IP, van den Anker JN, Lindemans J. CYP3A5 variant allele frequencies in Dutch Caucasians. Clin Chem. 2002;48:1668–1671.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/12324482/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Chem&title=CYP3A5%20variant%20allele%20frequencies%20in%20Dutch%20Caucasians&author=RH%20van%20Schaik&author=IP%20van%20der%20Heiden&author=JN%20van%20den%20Anker&author=J%20Lindemans&volume=48&publication_year=2002&pages=1668-1671&pmid=12324482&)

34. Watkins PB, Wrighton SA, Schuetz EG, Molowa DT, Guzelian PS. Identification of glucocorticoid-inducible cytochromes P-450 in the intestinal mucosa of rats and man. J Clin Invest. 1987;80:1029–1036. doi: 10.1172/JCI113156.  [DOI](https://doi.org/10.1172/JCI113156) | [PMC free article](/articles/PMC442342/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/3654968/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Invest&title=Identification%20of%20glucocorticoid-inducible%20cytochromes%20P-450%20in%20the%20intestinal%20mucosa%20of%20rats%20and%20man&author=PB%20Watkins&author=SA%20Wrighton&author=EG%20Schuetz&author=DT%20Molowa&author=PS%20Guzelian&volume=80&publication_year=1987&pages=1029-1036&pmid=3654968&doi=10.1172/JCI113156&)

35. Westlind-Johnsson A, Malmebo S, Johansson A, Otter C, Andersson TB, Johansson I, Edwards RJ, Boobis AR, Ingelman-Sundberg M. Comparitive analysis of CYP3A expression in human liver suggests only a minor role for CYP3A5 in drug metabolism. Drug Metab Dispos. 2003;31:755–761. doi: 10.1124/dmd.31.6.755.  [DOI](https://doi.org/10.1124/dmd.31.6.755) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12756208/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Comparitive%20analysis%20of%20CYP3A%20expression%20in%20human%20liver%20suggests%20only%20a%20minor%20role%20for%20CYP3A5%20in%20drug%20metabolism&author=A%20Westlind-Johnsson&author=S%20Malmebo&author=A%20Johansson&author=C%20Otter&author=TB%20Andersson&volume=31&publication_year=2003&pages=755-761&pmid=12756208&doi=10.1124/dmd.31.6.755&)

36. Williams JA, Ring BJ, Cantrell VE, Jones DR, Eckstein J, Ruterbories K, Hamman MA, Hall SD, Wrighton SA. Comparative metabolic capabilities of CYP3A4, CYP3A5, and CYP3A7. Drug Metab Dispos. 2002;30:883–891. doi: 10.1124/dmd.30.8.883.  [DOI](https://doi.org/10.1124/dmd.30.8.883) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12124305/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Comparative%20metabolic%20capabilities%20of%20CYP3A4,%20CYP3A5,%20and%20CYP3A7&author=JA%20Williams&author=BJ%20Ring&author=VE%20Cantrell&author=DR%20Jones&author=J%20Eckstein&volume=30&publication_year=2002&pages=883-891&pmid=12124305&doi=10.1124/dmd.30.8.883&)

37. Williams PA, Cosme J, Vinkovic DM, Ward A, Angove HC, Day PJ, Vonrhein C, Tickle IJ, Jhoti H. Crystal structures of human cytochrome P450 3A4 bound to metyrapone and progesterone. Science. 2004;305:683–686. doi: 10.1126/science.1099736.  [DOI](https://doi.org/10.1126/science.1099736) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15256616/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Crystal%20structures%20of%20human%20cytochrome%20P450%203A4%20bound%20to%20metyrapone%20and%20progesterone&author=PA%20Williams&author=J%20Cosme&author=DM%20Vinkovic&author=A%20Ward&author=HC%20Angove&volume=305&publication_year=2004&pages=683-686&pmid=15256616&doi=10.1126/science.1099736&)

38. Wong M, Balleine RL, Collins M, Liddle C, Clarke CL, Gurney H. CYP3A5 genotype and midazolam clearance in Australian patients receiving chemotherapy. Clin Pharmacol Ther. 2004;75:529–538. doi: 10.1016/j.clpt.2004.02.005.  [DOI](https://doi.org/10.1016/j.clpt.2004.02.005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15179407/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=CYP3A5%20genotype%20and%20midazolam%20clearance%20in%20Australian%20patients%20receiving%20chemotherapy&author=M%20Wong&author=RL%20Balleine&author=M%20Collins&author=C%20Liddle&author=CL%20Clarke&volume=75&publication_year=2004&pages=529-538&pmid=15179407&doi=10.1016/j.clpt.2004.02.005&)

39. Yano JK, Wester MR, Schoch GA, Griffin KJ, Stout CD, Johnson EF. The structure of human microsomal cytochrome P450 3A4 determined by X-ray crystallography to 2.05-A resolution. J Biol Chem. 2004;279:38091–38094. doi: 10.1074/jbc.C400293200.  [DOI](https://doi.org/10.1074/jbc.C400293200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15258162/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=The%20structure%20of%20human%20microsomal%20cytochrome%20P450%203A4%20determined%20by%20X-ray%20crystallography%20to%202.05-A%20resolution&author=JK%20Yano&author=MR%20Wester&author=GA%20Schoch&author=KJ%20Griffin&author=CD%20Stout&volume=279&publication_year=2004&pages=38091-38094&pmid=15258162&doi=10.1074/jbc.C400293200&)
